Team:NTU SINGAPORE/Improve
HOME
TEAM
Our Team Collaboration
PROJECT
Description Experiment Proof of Concept Demonstrate Improve
INTERLAB STUDY
PARTS
Basic Parts
HUMAN PRACTICE
Our Story Silver Gold
ATTRIBUTIONS CONTACT US
HOME RESEARCH INTERLAB STUDY PARTS COLLABORATION CONTACT
Improve
Our efforts in improving our project
In our attempt to improve our truncations of the part BBa_K213000 submitted by team NTU_SINGAPORE 2016, we explored 2 possible avenues of improvements. First, we considered if further truncations are possible, by looking at additional combinations and novel truncations. Second, we considered alternative functionalities for our truncated dCas9.
Further Truncations
To push the limits of truncations possible, ∆REC1-1 and ∆REC1-3 were added to our ∆3ple. ∆REC1-1 and ∆REC1-3 were chose as their individual truncations results in near WT gene activation. However, part attempt to combine them with HNH leads to drastic decrease in gene activation. Thus, it is no surprise that when ∆REC1-1 and ∆REC1-3 is combined with ∆3ple, gene activation dropped beyond zero. Curiously, these truncations resulted in MFI below negative control. Further experiments would be required to explore various hypothesis, such as potential CRISPRi activity.
Literature update: REC3 truncation
Janice et al (2017) reported that REC3 domain, a part of the REC1 domain, is responsible for HNH activation with on-target DNA binding. In addition, it was reported that REC3 truncation leads to a 1000-fold decrease in cleavage rate, but does not affect DNA binding affinity. Considering that the REC 3 domain has essentially no function in the nuclease null dCas9, and is likely to have minimal impact on the binding activity of our dCas9-VPR, the REC 3 truncation was replicated and characterized.
∆REC3 results in around 70% of WT gene activation – a poorer than expected result. More importantly, when ∆REC3 is combined with other truncations – even ∆HNH, gene activation drops drastically. Endogenous gene activation results for MIAT and TTN correlates exogenous data. This is a reminiscence of our prior work, where REC1 subdomain truncations were found to combine poorly with HNH and any further truncations. Indeed, in hindsight, this result was to be expected, since ∆REC3 is essentially both ∆REC1-1 and ∆REC1-3.
Our results for both further truncations and the novel REC3 truncation strongly suggests that ∆3ple is the best dCas9-VPR variant possible, at least within our current approach to dCas9 truncation. Further truncations generally lead to loss-of-function.
Novel functions for ∆3ple - CRISPRi
Based on our attempts at further combinations of truncations as well as truncation of newly characterized domains, we noted that ∆3ple, a combination of ∆HNH ∆REC2 and ∆RuvCIII-2, is the best combination of truncations we have. We noted that transcription activation functions of our ∆3ple are poor. Thus, we are interested if ∆3ple is functional in other applications, where poorer target DNA affinity may be less important. We decided to test the CRISPRi functionality of 3ple. As a proof of concept, we designed the experiment for a test in bacteria.
In bacterial CRISPRi, dCas9 is targeted to -35 or-10 of a promoter to block RNA polymerase binding, or to the non-template (NT) strand of the gene to block RNA polymerase transcription from proceeding. Transcription is blocked, resulting in expression interference.
For our construct, we decided to carry out CRISPRi on an endogenous GFP reporter E coli strain (a kind gift from Swaine Chen lab). BPK 65767 is adapted to non-T7 expression. The Cas9 promoter is replaced with Lac inducible promoter(Part BBa_K314103) while the sgRNA promoter was replaced with a high constitutive promoter(Part BBa_J23100). T7 terminators were maintained, as it has been shown that T7te can terminate normal promoters.
We tested targeting of -35, -10 and NT, using the original bacterial optimized Cas9 in BPK 65757 mutated at D10A and H849A to make dCas9. While -35 interference leads to partial repression of GFP expression, -10 and NT interference leads to robust CRISPRi of GFP. 2 negative controls were used, one with a scrambled gRNA target (emp), another with a target not available in the GFP reporter E coli strain (mcherryNT). Both negative controls confirmed no CRISPRi function against GFP gene with IPTG induction.
Thus, -10 target is chosen for our tests with truncated dCas9. WT-dCas9 and truncated dCas9 variants were cloned into BPK 65767, replacing Cas9 – to account for human optimized codons. The CRISPRi experiment is then repeated. Cells are immediately induced at OD0.2 with 0.5mM IPTG. Fluorescence (485nm/516nm) is normalized to cell density (OD600). At T=4hours, CRISPRi is observed for all samples. In both T=4 and T=8, induction uniformly led to robust CRISPRi of GFP expression for all truncated dCas9 variants – including ∆3ple.
Results suggest that multiple truncated ∆3ple can still be used to great effect for CRISPRi applications, at least in the context of bacterial genomes. The corresponding decrease in activity with increasing truncations seen in VPR fusion gene activation was not observed in CRISPRi. The poorer target DNA binding affinity of our multiply truncated dCas9 may be less important in CRISPRi, compared to in transcription activation, where considerably longer dwell time could be required for effective VPR fusion activity.
The results here could potentially mean that CRISPRi applications in mammalian cells can effectively utilize ∆3ple. As CRISPRi do not require any protein fusion, the much smaller size of ∆3ple meant that it can easily fit into rAAV, together with a strong promoter and polyadenylation signal for robust in vivo applications.
CRISPRi alone has many therapeutic applications, often similar to RNAi therapeutics. Unlike RNAi, however, CRISPRi acts at the genomic level – potentially achieving much more robust interference compared to the transcript level activity of RNAi.
While further tests of CRISPRi with ∆3ple, especially in mammalian models, is required, we have demonstrated the first steps towards novel applications of our ∆3ple dCas9. Such an improved functionality of our truncated dCas9 variants deserves an entire project to exploring its potentials and limitations – the robust interference even in ∆3ple could indicate that further truncations may be well tolerated. Depending on degree of truncations, CRISPRi activity may even be tuned – potentially allowing fine control of gene expression levels even without the highly inefficient RNP based dCas9-gRNA delivery.
In conclusion, we propose that ∆3ple is an improvement upon ∆RuvCIII-2 ∆HNH Sp-dCas9 (BBa_K2130000), both in smaller size, and in characterization of novel function for CRISPRi. ∆3ple (BBa_K2316000) has been submitted for gold requirement.
