<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU_SINGAPORE_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU SINGAPORE/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="w3-bar" id="myNavbar"><DIV class="w3-top w3-padding-large w3-transparent"><A href="https://2017.igem.org/Team:NTU_SINGAPORE" class="w3-bar-item w3-button"><FONT color="#800020"> HOME </FONT></A><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">TEAM</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Team" class="w3-bar-item w3-button"><FONT color="#800020">Our Team</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Collaborations" class="w3-bar-item w3-button"><FONT color="#800020">Collaboration</FONT></A></DIV></CLASS></DIV><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">PROJECT</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Description" class="w3-bar-item w3-button"><FONT color="#800020">Description</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Experiment" class="w3-bar-item w3-button"><FONT color="#800020">Experiment</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Results" class="w3-bar-item w3-button"><FONT color="#800020">Proof of Concept</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Demonstrate" class="w3-bar-item w3-button"><FONT color="#800020">Demonstrate</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Improve" class="w3-bar-item w3-button"><FONT color="#800020">Improve</FONT></A></DIV></CLASS></DIV><A href="https://2017.igem.org/Team:NTU_SINGAPORE/InterLab" class="w3-bar-item w3-button"><FONT color="#800020">INTERLAB STUDY</FONT></A><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">PARTS</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Basic_Part" class="w3-bar-item w3-button"><FONT color="#800020">Basic Parts</FONT></A></DIV></CLASS></DIV><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">HUMAN PRACTICE</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/OurStory" class="w3-bar-item w3-button"><FONT color="#800020">Our Story</FONT></A><FONT><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/Silver" class="w3-bar-item w3-button"><FONT color="#800020">Silver</FONT></A><FONT><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/Gold_Integrated" class="w3-bar-item w3-button"><FONT color="#800020">Gold</FONT></A></FONT></FONT></DIV></CLASS></DIV><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Attributions" class="w3-bar-item w3-button"><FONT color="#800020">ATTRIBUTIONS</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Contact Us" class="w3-bar-item w3-button"><FONT color="#800020"> CONTACT US</FONT></A><DIV id="navDemo" class="w3-bar-block w3-white w3-hide w3-hide-large w3-hide-medium"><A href="#home" class="w3-bar-item w3-button" onclick="toggleFunction()">HOME</A><A href="#research" class="w3-bar-item w3-button" onclick="toggleFunction()">RESEARCH</A><A href="#interlab study" class="w3-bar-item w3-button" onclick="toggleFunction()">INTERLAB STUDY</A><A href="#parts" class="w3-bar-item w3-button" onclick="toggleFunction()">PARTS</A><A href="#collaboration" class="w3-bar-item w3-button" onclick="toggleFunction()">COLLABORATION</A><A href="#contact" class="w3-bar-item w3-button" onclick="toggleFunction()">CONTACT</A></DIV></DIV><DIV class="ex w3-container w3-padding-32" id="Collaboration"><H3 class="w3-border-bottom w3-border-light-grey w3-padding-16">Collaboration</H3><PSTYLE> Collaborating with these teams allowed us to achieve our goals. It was our pleasure to help them to the best of our ability.
</PSTYLE></DIV><DIV class="content-overview" style="padding: 0px 66px; width: 872px;"><DIV class="circle-overview circle3"><A href="#NUS" style="top: -9px; width:200px; height:200px;" class="overview-content"><SPAN class="overview-text">NUS Singapore</SPAN></A></DIV><DIV class="circle-overview circle1"><A href="#macquarie" style="top: -9px; width:200px; height:200px;" class="overview-content"><SPAN class="overview-text">Macquarie Australia</SPAN></A></DIV></DIV></DIV><DIV class="ex column full_size"><H2><A href="https://2017.igem.org/Team:Macquarie_Australia"> Team Macquarie </A></H2><H4>How did we help them?</H4><P>Team Macquarie had constructed a hydrogenase gene capable of converting glucose to hydrogen gas. The plasmid <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2300001">BBa_K2300001</A> consists of the genes for hydrogenase enzyme (Hyd1), ferredoxin, ferredoxin-NADPH-reductase (FNR) and maturation enzyme (HydEF and HydG). Their expression levels are co-regulated under the pLac promoter.
</P><P>This year, our team helped Team Macquarie to evaluate each of the gene expression levels using RT-PCR.</P><P>The results of RT-PCR are shown below: </P><P> Formula: 
2^-(ΔΔCt) is used to calculate the fold change of gene expression after induction, in which</P><P>ΔCt(control)1= Ct(uninduced operon genes)- Ct(Chloramphenicol gene)</P><P>ΔCt(test)2 = Ct(Induced operon genes)-Ct(Induced Chloramphenicol gene)</P><P>Finally, use ΔCt(test)2- ΔCt(control)1 to get ΔΔCt.</P><P>IPTG and 20mM of glucose were used to induce gene expression. Comparison of 1mM IPTG and 20mM IPTG was made and it was found that 1mM IPTG has a higher gene expression level. It is worth noting that the RT-PCR efficiency might be different for different genes due to varied sizes of PCR products and the formula used above assumes that the PCR efficiency is 100% such that 2 copies of products are eventually obtained.</P><H4>How did they help us?</H4><P>In our gene activation experiments, we observed that our truncated dCas9 generally exhibit poorer gene activation compared to full-length wild type. We had hypothesized that this is likely due to poorer target DNA binding, and hence shorter dwell time at the promoter for the VPR transcriptional activator to work effectively. One key weakness with our experiments was that we were unable to directly characterize target DNA binding affinity by our truncated dCas9 variants. 
</P><P>For our collaboration with Macquarie iGEM team, we had a serendipitous meet-up with their advisor in Singapore. Noting their protein purification capabilities, we suggested an EMSA affinity binding assay. EMSA affinity binding assay is a well-documented method to characterize dCas9 binding to target DNA. Native gel PAGE is utilized, whereupon dCas9 binding to target DNA, a mobility shift upwards is observed due to increased size, and hence decreased migration. The intensity of the shifted band would indicate the amount of dCas9 binding to target DNA. We hypothesize that truncated dCas9 would have decreased binding, and hence lower band intensity. 
</P><P>Our dCas9 wild-type and truncated variants were cloned into BPK 65767 (gift from Keith Joung Lab, Addgene) with 6x His tag, using Gibson assembly. The original Cas9 is replaced – this is necessary as we are investigating binding without cutting with dCas9, and all dCas9 proteins were human codon optimized. Macquarie iGEM team utilized commercially synthesized sgRNA that they have previously shown to work with Cas9. 
</P><H2>Results</H2><P>In the EMSA gel experiment, Macquarie utilized the considerably purer commercial WT Cas9 to provide a reference point as to identity of bands. They confirmed that our WT dCas9 and all truncation variants were unable to cleave target DNA, as evidenced by singular uncut ferrochelatase PCR product in lane B9, D5, D8, and D11, compared to the cut products for commercial WT-Cas9 in lane B5.</P><P>Mobility shift was apparent for our WT-dCas when comparing the band size of ferrochelatase PCR target between lane 8 and 9, and this was reflected in commercial WT Cas9, albeit fainter due to most undergoing cleavage and dissociating. This shift appears to be due to gRNA binding, as the band is observed in the RNP lane with only dCas9 and gRNA. DNA binding appears to result in a very large RNPF complex that is unable to enter the gel, as evidenced by the bright DNA band in the well of B4. The same shift did not seem to appear with our WT-dCas9, although a high MW smear was observed. This phenomenon seems isolated as the same smear was not observed with the truncated dCas9 variants. </P><P>They demonstrated liberation of gRNA upon denaturation in Figure B lane B5. Unfortunately, the same gRNA bands were not observed in our wild-type dCas9. Similar gRNA bands were observed for truncated dCas9 in figure D. Based on the difference in intensity of the gRNA bands between RNP/RNPF lanes and denatured lane, results seem to suggest that HNH is able to bind to gRNA best, resulting in lesser free gRNA, followed by 3ple and 5ple. This corresponds to our gene activation tests, confirming that the poorer gene activation observe is not just due to poorer target DNA binding – it is likely that poor gRNA binding could be a contributing factor for the poor gene activation by truncated dCas9-VPR.  </P><P>The lack of supershifted RNFP in WT-dCas9 experiments coupled with a high MW smear led us to suspect potential DNase/RNase contamination. Such a contamination would also explain the loss of the gRNA band in the denaturation lane. Supershifted RNFP also adds doubt to the intensity of RNPF complex band. A lower percent native gel and a smaller target DNA PCR product would likely result in a smaller shift, and be more in line to established dCas9 EMSA experiments in literature. Noting these weaknesses, we look forward to further improving this protocol in the future. Such information would conclusively characterize the altered binding affinities for target DNA and gRNA by our truncated dCas9 variants.  </P><P>We are deeply thankful to Macquarie iGEM team for going above and beyond in this iGEM collaboration. We had initially proposed this experiment to merely observe DNA binding causing a supershift, but the results have taught us much more – such as about the variable gRNA binding by truncated dCas9. A special shoutout to Dr. Edward Moh, whose expertise has made the planning of this collaboration a fun process, despite us not being familiar with the experiment. Do check their <A href="https://2017.igem.org/Team:Macquarie_Australia">wiki</A> also!</P><DIV class="column full_size"><H2><A href="https://2017.igem.org/Team:NUS_Singapore"> Team NUS </A></H2><H4>How did we help them</H4><P>We helped the Team NUS to characterize their plasmid <A href="http://parts.igem.org/Part:BBa_K2447014">BBa_K2447014</A>. The protocol can be found <A href="https://2017.igem.org/Team:NUS_Singapore/Experiments">here</A></P><P>However, the results we obtained are not consistent with their expectations.</P><H4>How did they help us</H4><P>
In our collaboration with NUS, we noted their experience with well characterized bacterial plasmids. Thus, it was suggested that we leverage on their plasmid assets to test the CRISPRi potential of our truncated dCas9 in bacteria. 
</P><P>
In bacterial CRISPRi, dCas9 is targeted to -35 or -10 of promoter to block RNA polymerase binding, or to the non-template strand of the gene to block RNA polymerase transcription from proceeding. Transcription is blocked, resulting in expression interference. Once again, dCas9 affinity is studied, since strong affinity is required for effective binding and blocking. We hypothesize that truncated dCas9 would have decreased binding, and hence poorer interference of gene expression. In both NUS and our CRISPRi constructs, the target is GFP reporter gene. 
</P><P>
NUS provided 2 plasmids, a gRNA expressing plasmid (NUS-gRNA) and a low copy plasmid expressing dCas9 and GFP to be interfered with (NUS-dCas9-GFP). The gRNA targets -10 of the GFP promoter. These plasmids has previously been shown to work by the NUS lab’s pHD student. dCas9 expression is inducible with tetracycline. We performed Gibson assembly to move our truncated dCas9-VPR onto the NUS-dCas9-GFP plasmid, and subsequently carried out the CRISPRi experiments. 
</P><P>
The results in this experiment is different from that of our CRISPRi against endogenous genes – here, interference data more closely correlate to our gene activation data. WT and ∆HNH provides robust downregulation of genes, ∆REC3 provides intermediate while ∆3ple provide poor downregulation. Once again, similar to our mammalian model results, ∆3ple ∆R1-1 ∆R1-3 exhibits no downregulation. 
</P><P>
Such a result is to be expected: in this experiment, only the gRNA was induced. The dCas9 is under a strict tetracycline inducible promoter, and without tetracycline, expression level is likely to be very low. The low amount of dCas9 available for CRISPRi would mean strong binding affinity is especially more important. Indeed, preliminary tests performed with tetracycline induction alone appears to results in better CRISPRi activity for poor performers, likely due to abundance of dCas9 for CRISPRi.  
</P><P>
Not only that: the GFP gene in this case is on a p15A origin plasmid. Literature suggests that there should be 15-20 copies of this plasmid per cell – and hence 15-20 copies of GFP promoters for the dCas9 to bind to and interfere. On the other hand, there is only one genome bound GFP gene in our endogenous reporter E. coli strain for multiple dCas9 to bind to. 
</P><P>
Thus, the binding affinity of our truncated dCas9 variants are more strictly interrogated by this plasmid reporter system, especially when dCas9 expression is restricted. 
</P><P>
Based on these conclusions, we would expect to observe similar results with endogenous gene CRISPRi if we are to target a less effective site, for example the -35 of the promoter. In addition, we may also replicate the very low dCas9 level expression to achieve strict interrogation in endogenous CRISPRi. 
</P><P>
While plasmid reporter used here is an imperfect model of therapeutic CRISPRi, we have shown that it is a better method to interrogate our truncated dCas9 binding affinity – especially with regards to mammalian endogenous gene activation. Such a system may serve as a rapid intermediate to testing any future truncated dCas9, complementing other methods such as our mammalian exogenous gene activation and the EMSA assay established by Macquarie iGEM team. 
</P><P>
We would like to thank NUS iGEM team for providing a characterized CRISPRi dual plasmid system. Do check their wiki page also <A href="https://2017.igem.org/Team:NUS_Singapore">here</A>.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>