<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU_SINGAPORE_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU SINGAPORE/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="w3-bar" id="myNavbar"><DIV class="w3-top w3-padding-large w3-transparent"><A href="https://2017.igem.org/Team:NTU_SINGAPORE" class="w3-bar-item w3-button"><FONT color="#800020"> HOME </FONT></A><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">TEAM</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Team" class="w3-bar-item w3-button"><FONT color="#800020">Our Team</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Collaborations" class="w3-bar-item w3-button"><FONT color="#800020">Collaboration</FONT></A></DIV></CLASS></DIV><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">PROJECT</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Description" class="w3-bar-item w3-button"><FONT color="#800020">Description</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Experiment" class="w3-bar-item w3-button"><FONT color="#800020">Experiment</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Results" class="w3-bar-item w3-button"><FONT color="#800020">Proof of Concept</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Demonstrate" class="w3-bar-item w3-button"><FONT color="#800020">Demonstrate</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Improve" class="w3-bar-item w3-button"><FONT color="#800020">Improve</FONT></A></DIV></CLASS></DIV><A href="https://2017.igem.org/Team:NTU_SINGAPORE/InterLab" class="w3-bar-item w3-button"><FONT color="#800020">INTERLAB STUDY</FONT></A><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">PARTS</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Basic_Part" class="w3-bar-item w3-button"><FONT color="#800020">Basic Parts</FONT></A></DIV></CLASS></DIV><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">HUMAN PRACTICE</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/OurStory" class="w3-bar-item w3-button"><FONT color="#800020">Our Story</FONT></A><FONT><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/Silver" class="w3-bar-item w3-button"><FONT color="#800020">Silver</FONT></A><FONT><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/Gold_Integrated" class="w3-bar-item w3-button"><FONT color="#800020">Gold</FONT></A></FONT></FONT></DIV></CLASS></DIV><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Attributions" class="w3-bar-item w3-button"><FONT color="#800020">ATTRIBUTIONS</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Contact Us" class="w3-bar-item w3-button"><FONT color="#800020"> CONTACT US</FONT></A><DIV id="navDemo" class="w3-bar-block w3-white w3-hide w3-hide-large w3-hide-medium"><A href="#home" class="w3-bar-item w3-button" onclick="toggleFunction()">HOME</A><A href="#research" class="w3-bar-item w3-button" onclick="toggleFunction()">RESEARCH</A><A href="#interlab study" class="w3-bar-item w3-button" onclick="toggleFunction()">INTERLAB STUDY</A><A href="#parts" class="w3-bar-item w3-button" onclick="toggleFunction()">PARTS</A><A href="#collaboration" class="w3-bar-item w3-button" onclick="toggleFunction()">COLLABORATION</A><A href="#contact" class="w3-bar-item w3-button" onclick="toggleFunction()">CONTACT</A></DIV></DIV><DIV class="column full_size"><DIV class="ex w3-container w3-padding-32" id="Proof of Concept"><H3 class="w3-border-bottom w3-border-light-grey w3-padding-16">Proof of Concept</H3></DIV><DIV class="content-overview" style="padding: 0px 66px; width: 872px;"><DIV class="circle-overview circle1"><A href="#TP" style="top: -9px; width:200px; height:200px;" class="overview-content"><SPAN class="overview-text">Truncation Project</SPAN></A></DIV><DIV class="circle-overview circle2"><A href="#HDR" style="top: -9px; width:200px; height:200px;" class="overview-content"><SPAN class="overview-text">Homology Directed Repair</SPAN></A></DIV><DIV class="circle-overview circle3"><A href="#EGFR" style="top: -9px; width:200px; height:200px;" class="overview-content"><SPAN class="overview-text">EGFR Project</SPAN></A></DIV></DIV></DIV><DIV class="ex column full_size"><H2> Truncation Project </H2><U>Exogenous Gene Activation</U><P>Exogenous gene activation of ZSGreen fluorescent reporter gene is first performed. Quantitative assessment of tCas9 activity is acquired by flow cytometry. 3 plasmids are utilized in exogenous gene activation. The reporter plasmid contains ZSGreen under TREtight minimal promoter preceded by 2X protospacers. The sgRNA plasmid expresses sgRNA under U6 promoter. This plasmid also contains a constitutive mCherry expression, to report successfully transfected cells during flow cytometry. The dCas9-VPR plasmid expresses dCas9-VPR under constitutive CMV promoter. dCas9 can then utilized the sgRNA to bind to protospacer, while the fused VPR greatly upregulates expression of ZSGreen. Cells are transfected and incubated for 48H. Flow cytometry is then perfomed by gating for mcherry positive cells indicated successful transfection. The ZSGreen mean fluorescence is then acquired for these cells, reporting the gene actvation activity of the truncated dCas9-VPR. </P><U>Improvements to truncated dCas9-VPR</U><P>We have demonstrated in our Improve section that ∆3ple is the best set of truncations we have achieved with our current approach. While we demonstrated a new application of ∆3ple for bacterial CRISPRi, we are still interested in using it for mammalian gene activation. Thus, to expand on our truncation approach, we are interested in improving our truncated dCas9 protein for gene activation, especially for ∆3ple.</P><P>Truncation of dCas9 led to poorer gene activation activity as expected. The folding of protein is essentially to achieve the lowest free energy state possible with the amino acids in a polypeptide chain – that is the protein’s native, functional state. Natural selection in evolution would select for the best amino acids capable of folding into secondary and tertiary structures properly. Thus, truncations to domains and subdomains would likely disrupt an otherwise finely tuned molecular machine. </P><P>Consider the crystal structure of dCas9. In the following images, the 3 key truncations were highlighted. Note the space left behind when the domains are deleted. While the truncations in this project are designed to minimally impact secondary structures, the loss of structural support by these domains and subdomains is bound to have led to disruptions in tertiary structure. </P><P>Images above indicates before truncation and the right images, after truncation. Note that the domains/subdomains are only removed from the structure; no predictions on alteration to structure were performed. </P><P>In addition, consider the hydrophobicity of protein surfaces. Soluble proteins such as Cas9 has a largely hydrophilic outer surface, coupled with a hydrophobic core. Such an organization greatly decreases free energy at native state – the hydrophilic surface interacts with the aqueous environment in the cytoplasm, while the hydrophobic sidechains are sequestered within the protein. The truncation of domains and subdomains, especially those near the surface of the protein, is likely to reveal the hydrophobic core. Such an example could be seen in the following image of ∆RuvCIII-2 truncation, which reveals the dark red hydrophobic core (indicated by a black arrow). To achieve a lower free energy state, the hydrophilic sidechains in the vicinity would likely move in to cover the hydrophobic core, thus disrupting tertiary structure. </P><P>Surface hydrophobicity of dCas9 before and after RuvCIII truncation, as calculated by color_h plugin in Pymol. Red indicates hydrophobic, while white indicates hydrophilic. </P><P>Indeed, very poor gene activation activity by some truncations and when excessive truncations were combined is likely a result in a completely misfolded or even insoluble protein.</P><P>Removal of domains and subdomains is likely to result in a disturbed tertiary structure, with DNA binding residues shifted to suboptimal positions. The resulting drop in target DNA binding activity is likely the biggest contributor to poor gene activation activity, especially for smaller dCas9s with multiple truncations.</P><P>Examples of residues where mutations performed as indicated on the crystal structure of WT-Cas9 (above). Note its proximity to nucleic acid backbone (brown). </P><P>To test the hypothesis, as well as to improve the gene activation ability of our dCas9, mutation is done to improve target DNA binding affinity again. Crystal structure information of the wild type Cas9 were once again utilized to identify key residues close to DNA backbone, which were then mutated to positive charged side chains such as Arg or Lys. A total of 21 residues were identified and rapidly screened on ∆HNH ∆REC1-3 (indicated as ∆HNH ∆REC1-c in graph). Subsequent mentions of mutated residues are referred to as QC1-21.</P><H4>
Endogenous gene activation – a better model for in vivo applications 
</H4><P>Exogenous reporter gene activation is useful for quickly screening tCas9 created. However, it is a poor model for regulation of endogenous genes on chromosomes. Unlike on the reporter plasmid, chromosomal genes are controlled by promoters with varying levels of activity. These promoters are often already regulated by transcription factors, adding a level of variability of how well the gene may be upregulated. Lastly, the epigenetic landscape of endogenous genes may also play a role in impeding dCas9-VPR mediated gene regulation. Thus, there is a need to test our truncated dCas9 in the context of endogenous genes. </P><P>In this experiment, the reporter plasmid is no longer required. The sgRNA plasmid is retargeted to endogenous housekeeping genes such as ASCL1, TTN and MIAT. These genes were selected as parameters for upregulatory effects on these genes by dCas9-VPR has been previously established in literature. Cells are transfected and incubated for 48H, then sorted by flow-cytometry assisted cells sorting (FACS) for mCherry positive transfected cells. Quantitative assessment of tCas9 activity is then acquired by quantitative PCR (qPCR) for mRNA level of the housekeeping genes. </P><P>In continuation from last year’s project, better performing truncated dCas9s were tested on ASCL1 and MIAT. While data generally correlate with exogenous gene activation results, gene activation is by truncated dCas9-VPR is generally poorer than WT. This is even the case for ∆HNH, which had been shown to perform on par with WT in exogenous tests. ∆3ple and further truncations are unable to activate gene expression.</P><U>Extended incubation time</U><P>As the poor performance of truncated dCas9-VPR is likely due to poor affinity to target DNA, we posited that a longer incubation period may allow more time for the poorer binding truncated dCas9-VPR to interact with the promoters of genes, and better upregulate the genes. Indeed, prior experiments with longer incubation time in reporter experiments consistently demonstrated increased MFI. </P><P> Thus, a 72H incubation experiment for ASCL1 is performed. As seen in the results, gene activation by WT and well-performing truncated dCas9-VPR is only mildly affected by the longer incubation. However, poor performing truncated dCas9 such as ∆3ple, ∆3ple ∆REC1-3 and ∆3ple ∆REC1-1 ∆REC 1-3 appears to perform better.</P><P>Incubation time may thus be a consideration for therapeutic application – for example, repeated dosages may be required when utilizing extensively truncated dCas9-VPR. </P><U>Affinity enhanced truncated dCas9-VPR</U><P>Endogenous tests are also performed for truncated dCas9-VPR that has improved target DNA affinity by mutation of key residues close to target DNA.</P><P>Enhanced ∆REC1-3 ∆HNH results indicated that mutations to enhance target DNA binding do indeed improve gene activation in endogenous context, mirroring exogenous results. However, preliminary results for enhanced ∆3ple appears to only result in improvements for MIAT gene. Further replicates and tests with multiple mutations would be required to confirm results. </P><H4>Conclusion</H4><P>In conclusion, our team has built upon past work on truncated dCas9 for gene activation. We explored the truncation of a new subdomain, REC3. In addition, we also explored new combinations of truncations and its effect on gene activation. 
Multiple truncations led to decrease in gene activation, likely due to decreased target DNA affinity. We demonstrated that mutations to DNA backbone proximal residues to improve target DNA affinity can improve gene activation by multiply truncated dCas9-VPR. Results in exogenous gene activation has been replicated in endogenous gene activation – a better model for therapeutic applications.</P><P>
Multiple truncations led to decrease in gene activation, likely due to decreased target DNA affinity. We demonstrated that mutations to DNA backbone proximal residues to improve target DNA affinity can improve gene activation by multiply truncated dCas9-VPR. Results in exogenous gene activation have been replicated in endogenous gene activation – a better model for therapeutic applications. 
</P><P>While results suggest that multiply truncated dCas9-VPR perform much more poorly in endogenous tests, this is to be expected. Other than aforementioned differences in endogenous gene environment, regulatory feedback loops are likely to dampen the up-regulatory effects of dCas9-VPR. In addition, the genes tested are housekeeping genes, which already has a high basal expression level. In the context of therapeautic applications such as treatment of loss-of-function diseases, the upregulation effect would be more pronounced. Indeed, even a modest increase in gene expression could be helpful in some diseases.</P><P>Importantly, we have established that improving DNA binding specificity in our truncated dCas9 is a viable way to improve gene activation, both in exogenous and endogenous gene activation. Other methods to improve DNA binding can be explored. 
The truncations performed are likely to have negatively impacted specificity, in addition to affinity observed. It is worth noting that the mutations applied to improve DNA backbone affinity in our multiply truncated dCas9-VPR does not necessarily improves specificity. The mutations performed improves non-specific affinity to DNA backbone, and thus may even further decrease specificity by improving binding to off-target sequences. Future work would include characterization of off-target effects, and if necessary, improvements to specificity. 
</P><H2> HDR Project </H2><H4>Methodology</H4><P>In order to improve HDR efficiency, a fusion protein between SpCas9 and Rad52 was developed. Rad52 is one of the HDR proteins involved in mediating ssDNA annealing between homologous sequences and in assisting homologous pairing. The construct was tested in HEK293FT cells, targeting five genes. The construct was transfected concurrently with a GFP donor.</P><P>
Upon a successful HDR event, the donor will be integrated into the genome and GFP will be produced. Hence, the expression of GFP in the cells indicates a successful HDR event. Next, the cells are analyzed by flow cytometry to determine the frequency of HDR occurrence.
</P><P>
For this experiment, we fused Rad52, an HDR protein. Rad52 helps to repair DSB by promoting ssDNA annealing between homologous sequence, then assist in the homologous pairing process. Then, the plasmid was transfected into HEK293FT cells.
</P><P>
We used plasmid constructed by Dr. Yuan Ming. The plasmid expresses SpCas9 fused with Rad52 and then linked with P2A to a mCherry reporter. P2A is a self-cleaving protein. If this chain of protein is expressed, the P2A will self-cleave and release mCherry, hence the cell expressing it will fluoresce red. In other words, cells that are successfully transfected will fluoresce red. In the same plasmid, the sgRNA was cloned in and constitutively expressed under a human U6 promoter.
</P><H4>Schematics of the parts of the plasmid</H4><P>
HDR needs the presence of a donor template which contains the homologous sequence. In order to detect the presence of HDR, we used a GFP plasmid donor. In this plasmid, there is a GFP gene bearing homology sequence to the gene we targeted. This donor sequence was transfected with the plasmid with fusion protein and guide RNA. If there is a DSB resulting from the cutting action, the cell can use the GFP donor to repair the DSB, and consequently, GFP will be expressed. Hence, if we could measure HDR percentage indirectly by analyzing the percentage of cells expressing the GFP protein.
</P><P>
We used flow cytometry to analyze the cells expressing the fluorescent proteins (mCherry and GFP) seven days after transfection. We gated for living cells based on the forwards and side scatter, and the plotted in the two-dimensional plot with respect to signal detected by FITC and PE channel of the flow cytometer to capture GFP and mCherry fluorescence, respectively. The HDR efficiency was determined from the GFP knock-in percentage of successfully transfected cells (subsequently referred as knock-in).
</P><P>
A representative sample of how the knock-in event is calculated. Previously, the population had been gated for living and mCherry+ cells. The numbers are the percentage of cells falling under the gating. In this case, it is the GFP+ cells.
</P><P>
At first, we tested for GAPDH gene. We were happy because the result showed that there are more cells expressing GFP if transfected with the Rad52 fusion protein. This means that HDR level improved by expressing this protein. Moreover, the percentage of HDR using fusion protein is more than twice compared to the wild-type Cas9.
</P><P style="text-align:center">
Percentage of the knock-in event for GAPDH gene. The data were normalized to empty construct. Error bar is +- s.d.
</P><P>
In order to confirm our result, we also tested the construct targeting CLTA and GLUL gene. In these two genes, Rad52-fusion is also shown to improve HDR events.
</P><P style="text-align:center">
Percentage of the knock-in event for CLTA gene. The data were normalized to empty construct. Error bar is +- s.d.
</P><P style="text-align:center">
Percentage of the knock-in event for GLUL gene. The data were normalized to empty construct. Error bar is +- s.d.
</P><P>
Our results showed that the Rad52-SpCas9 fusion was able to promote HDR level in HEK293FT cells for all the genes tested. Although it is still unknown how this protein would improve the efficiency, we hypothesized that now Rad52 is available immediately after the DSB occurred, and hence outcompete NHEJ from repairing the DSB.
</P><H2>
EGFR Project 
</H2><H4>
Methodology
</H4><P>
A total of 10 oligonucleotide sequences, which differ by length and sequence, were designed to act as single-guide RNAs (sgRNAs). Four types of CRISPR plasmid constructs (SpCas9, SaCas9, AsCpf1, LbCpf1) were used in the cloning. In order to have a fair comparison amongst all endonucleases, we cloned the sgRNAs onto the same plasmid under the expression of two different promoters, CAG and EF1a.
</P><P>
After cloning, the plasmids were transfected into PC9 cells. Since the plasmids contain the mCherry reporter gene which codes for red fluorescence, successfully transfected cells were sorted via FACS (RFP+) after 24h and 48h. We then extracted the genomic DNA from these cells and PCR amplified the targeted genomic locus. The amplified products were tested for detection and quantification of indels using T7 endonuclease I (T7E1).
</P><P>
In a T7E assay, the PCR products are denatured and reannealed to allow heteroduplex formation between wild-type DNA and CRISPR/Cas9-mutated DNA. T7E1, which recognizes and cleaves mismatched DNA, is used to digest heteroduplexes. The resulting cleaved and full-length PCR products are visualized by agarose gel electrophoresis. The image of the gel is shown under the proof of concept. The diagram below shows the general workflow for T7E assay.
</P><P>
According to the T7E1 results, it is possible that Cpf1 endonucleases performed better than SpCas9 and SaCas9 because of the more intense bands in the gel image, as shown below. However, more quantitative tests are needed to be able to derive a firm conclusion. Besides that, another thing worth notice is that all the sgRNAs designed, ranging from 19-23 nucleotides, were able to bind to the target DNA and performed their work, as positive T7E1 results were observed for almost all the plasmids.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>