"
Page
Discussion
View source
History
teams
Log in
 
Team:Warsaw/Protocols
From 2013.igem.org
Project
Project description
Acrylamide detection
BiFC Toolbox
Cytotoxicity study
Safety
Achievements
Results
Parts
Cooperation with other iGEM Teams
Judging
Human Practice
Strategy overview
Cooperation with PWN
The Night of Biologists
Science Picnic 2013
Our cooperation with mass media
Extras
Protocols
Genetic lab journal
Cellular biology lab journal
Project glossary
Adventure with grants
Team
Team
Acknowledgments
Sponsors
Protocols
Contents
1 Synthetic biology protocols
1.1 Chemocompetent bacteria
1.2 Transformation  of chemocompetent bacteria
1.3 Transformation of electrocompetent bacteria
1.4 Alkaline Lysis
1.5 DNA Digestion
1.6 DNA Ligation
1.7 PCR
2 Cellular biology protocols
2.1 AlamarBlue assay
2.1.1 Protocol:
2.2 Flow Cytometry
2.2.1 Preparation of the samples
2.3 Hoechst stain
2.3.1 Hoechst 33342 stain protocol
2.3.2 Protocol
2.4 Micronucleus assay
2.5 Cell growth assay
Synthetic biology protocols
Chemocompetent bacteria
Inoculate 500 ml LB  with 5 ml of the overnight culture and incubate it shaking at 37 °C till the OD = 0.4.
Cool the culture for 10 min. in some ice.
Centifruge the sample at 6000 rpm at 4 °C for 3 minutes.
Suspend the precipitate gently in ~ 20 ml of cold solution of 0.1 M CaCl2, then add 0.1 M CaCl2 to a volume of 300 ml.
Centrifuge again as above.
Suspend the precipitate in 10 ml of cold 0.1 M CaCl2 and incubate it for 30 min.
Centrifuge again.
Suspend the sample in 6 ml of 0.1 M CaCl2 and 15% glycerol and Pipette 50 μl to an eppendorf (put them immediately in liquid nitrogen) and store at -80 °C.
Transformation  of chemocompetent bacteria
Put the bacteria into ice for 2-3 minutes (or until it melts)
Add a cooled plasmid or ligation in a volume not bigger than 20 μl and stir with a tip
Keep it in the ice for 20 to 30 minutes
Put it  to a heating block set for 42°C for 1.5 min
Put it back to ice for 2 min
Add 900 μl  of SOB and incubate at 37°C for 1 h
Sow on the appropriate selective deposit.
Transformation of electrocompetent bacteria
Put the bacteria into some ice for 2-3 minutes (or until it melts).
50 μl of bacteria pipete to a dialised and cooled plasmid DNA or ligation
Put the transformation mixture to a cuvette (also previously cooled on ice) - it is important that the sample must be on the bottom of the cuvette and free of air bubbles. You can tap the cuvette several times on the table.
Transform the bacteria in an electroporator set on 2500 V (time constant should be around 5)
Immediately add 900 μl  of SOB and incubate at 37°C for 1 h
Sow the appropriate selective deposit
Alkaline Lysis
Pick single colony and inoculate 3 ml of LB broth. Shake at 37°C overnight.
Centrifuge two times 1.5 ml cells in the same 1.5 ml Eppendorf tube at 6000 rpm for 1 minute. Aspirate supernatant.
Resuspend cell pellet in 100 μl of GTE buffer (50 mM Glucose, 25 mM Tris-HCl, 10 mM EDTA, pH 8).
Add 200 μl of buffer II (0.2 M NaOH, 1% SDS). Invert gently tube 6-8 times.
Add 150 μl of buffer III (3M potassium acetate, 2M acetic acid, pH 5.5). This solution neutralizes NaOH in the previous lysis step while precipitating the genomic DNA and SDS in an insoluble, white, amorphous precipitate. Incubate in ice for 5 min. Spin at top speed 10 min.
Transfer supernatant to a new tube, being careful not to pick up any white flakes.  Precipitate the nucleic acids with 1ml of 96% ethanol for 5 minutes at room temperature and centrifuge at top speed for 10 minutes. Aspirate supernatant, add 200 &mul 70% ethanol and centrifuge at top speed for 1 minute.
Aspirate off all the ethanol supernatant. Dissolve the pellet in 40 μl of water. Store at -20°C.
DNA Digestion
Add to Eppendorf tube  (for 20 μl):
15 μl water
2 μl buffer
2 μl DNA that you want digest (ex. plasmid)
0,5 μl enzyme X and 0,5 μl enzyme Y (or 1 μl enzyme Z)
(for 10 &mul [verification cloning]):
7 μl plasmid
1 μl buffer
1 μl water
0,5 μl enzyme X and 0,5 μl enzyme Y (or 1 μl enzyme Z)
Tubes incubate at 37°C for 1 h.
DNA Ligation
Add to Eppendorf tube
5 μl water
2 μl buffer
10 μl digest plasmids mix
1 μl T4 ligase
Incubate 2-3 h at room temperature or O/N at 16 degrees Celsius
PCR
Add to PCR tube (0,2 ml), do it on ice:
34 μl water
10 μl buffer with MgCl2
1 μl 10mM dNTPs
2 μl primers
2 μl template DNA (200x attenuate plasmid)
1 μl Phusion polymerase
PCR programme:
Initialization step – 98°C – 3 min
Denaturation step – 98°C – 30 sec
Annealing step – 55°C – 30 sec
Elongation step – 72°C – 40 sec
Final elongation – 72°C – 5 min
Final hold – 4°C – to end of time
30 cycles
Total time: about 1 h 40 minutes.
Cellular biology protocols
AlamarBlue assay
To indicate cell viability, we have decided to implement Alamar Blue assay. It bases on the ability of living cells to convert nonfluorescent resazurin to the highly fluorescent molecule- resorufin. While entering the cells, resazurin is being reduced to resorufin and the bright, red fluorescence is to be observed. Viable cells are continuing to convert resazurin to resorufin, so the strength of fluorescence increases proportionally and may be used to measure viability of the cells.
Protocol:
1st day Preparation of the 96-well plate:
Adding 80µl media with 10.000 cells to each well
Adding 20µl of properly concentrated acrylamide to each well
Row 1 and 2 -> no acrylamide: control
Row 3 and 4 -> 1mM  acrylamide (1,76µl of acrylamide + 350 µl of PBS)
Row 5 and 6 -> 2mM acrylamide (3,52µl of acrylamide + 348,48 µl of PBS)
Row 7 and 8 -> 3mM acrylamide (5,28µl of acrylamide + 346,72 µl of PBS)
Row 9 and 10 -> 4mM acrylamide (7,04µl of acrylamide + 344,96 µl of PBS)
Row 11 and 12 -> 5mM acrylamide (8,8µl of acrylamide + 343,2 µl of PBS)
2nd day
Adding 10µl of AlamarBlue to each well in rows A and B (24 hours incubation)
Incubate for 2 hours at 37°C, protected from light, in a cell culture incubator
Reading absorbance at 570 nm
3rd day
Adding 10µl of AlamarBlue to each well in rows C and D (48 hours incubation)
Incubate for 2 hours at 37°C, protected from light, in a cell culture incubator
Reading absorbance at 570 nm
4th day
Adding 10µl of AlamarBlue to each well in rows E and F (72 hours incubation)
Incubate for 2 hours at 37°C, protected from light, in a cell culture incubator
Reading absorbance at 570 nm
5th day
Adding 10µl of AlamarBlue to each well in rows G and H (96 hours incubation)
Incubate for 2 hours at 37°C, protected from light, in a cell culture incubator
Reading absorbance at 570 nm
Flow Cytometry
Preparation of the samples
Vacuum off media with floating cells and place it in different 15ml falcons.
Rinse each well of 6-well plate with 2 ml of PBS. Vacuum off PBS
Add 0,5 ml of Trypsin and 1 ml of PBS to each well.
Incubate the plate for 10 min at 37°C
Add 1 ml of media to each well to neutralize Trypsin
Suspend the cells, then pipette them off and add to previously prepared falcons.
Take a 50µl sample to perform a cell count
Centrifuge for 4 min at 400 RPM
Aspirate supernatant and suspend cells in PBS (1mln cells/1 ml)
Transfer 300 µl of suspension to the test tube
Centrifuge for 4 min at 400 RPM
Aspirate supernatant and suspend cells in 100µl of binding buffer
Add 4 μl of Annexin V or 5 μl of PI
Incubate for 20 min at room temperature, protected from light
Add 400µl of binding buffer, suspend, and transfer to test tubes used in your cytometer
Analyze by flow cytometry
Hoechst stain
Hoechst 33342 stain protocol
In order to be able to examine cell’s nuclei morphology we introduced staining with Hoechst dye. The dye binds to the minor groove of dsDNA preferentially to sequences rich in adenine and thymine. As the Hoechst dye may be excited by ultraviolet light at around 350 nm, and then emits blue fluorescent light (maximum at around 461 nm)  we’ll be able to observe changes in the cell’s nuclei using fluorescent microscopy. As UV light is harmful for the cells, they have to be fixed first in order to avoid excess death during the assay. Our aim is to study changes that occurred due to treating cells with the acrylamide: cell division, cell apoptosis and amount of micronuclei.
Protocol
Fixation of the cells:
Remove media
Rinse 3 times with PBS (1 ml per well of 6-well plate)
Add 1ml of 4% paraformaldehyde and 0,5% sucrose in PBS (per well of 6-well plate
Note: Paraformaldehyde is toxic! Always use gloves, safety glasses and work under the hood while using  solution.
Staining with Hoechst:
Remove the fixative with aspirator
Rinse 3 times with PBS
Add 1 ml of 10.000 times diluted stock solution of Hoechst
Incubate 30 minutes in darkness at 37°C
Remove stain
Rinse once with PBS then add 1 ml of PBS
Preparing a microscope preparations:
Prepare microscopic slides
Place a drop of mountant in the middle of the slide
Carefully place the round glass slide on the droplet (remember to put the slide side with cells to mountant down, so the cells stay immersed)
Remove excess liquid
Observations:
Once the microscope preparations are ready it is possible to examine them using fluorescent microscope. Hoechst dye emits blue  fluorescence light at  around 461 nm while excited by ultraviolet light at around 350 nm.
Micronucleus assay
As we would like to measure the genotoxity of acrylamide, we have decided to implement micronucleus assay. This test belongs to the most popular and widely used methods of searching for potentially mutagenic substances.
The assay bases on formation of the micronuclei, which are the erratic nuclei formed during the anaphase due to mutagen exposition. They may originate both from acentric chromosome fragments or whole chromosomes unable to migrate properly.
The modifications in chromosomes structure, observed during the test, indicate the genotoxic effect of tested substance.
In our lab we would treat different cell lines with various acrylamide concentrations and after dyeing with Hoechst we would count the micronuclei. To examine the mutagenic effect of acrylamide we would use suitable statistical tests.
Cell growth assay
As we aimed to obtain certain statistic concerning cells proliferation we have decided to implement cell growth assay.
To conduct the assay we have seeded cells from different cell lines (HeLa, HEK293 and 143b) in 6-wells plates adding proper concentration of acrylamide to each well and counted cells amount after 24, 48 and 72 hours.
Preparation of the 6-wells plates
adding 3 ml of fresh media to each well
adding proper amount of each cells to the wells
incubate at 37°C for 24 hours
add 333ml of certain acrylamide concentration to each well
count the cells from each well after 24, 48 and 72 hours using Trypan blue
This project is part of iGEM
Read more on the iGEM Main Page
Sponsors:
Click here for full list of sponsors.
Retrieved from " http://2013.igem.org/Team:Warsaw/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
