<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Warsaw_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Warsaw/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Warsaw/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Warsaw/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Warsaw/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="navigation"><DIV class="navigation"><DIV class="centering"><UL class="topnav" id="topnav"><LI class="project navitem3"><A href="/Team:Warsaw/Project_description">Project</A></LI><LI><A href="/Team:Warsaw/Project_description">Project description</A></LI><LI><A href="/Team:Warsaw/Acrylamide_detection">Acrylamide detection</A></LI><LI><A href="/Team:Warsaw/BiFC_Toolbox">BiFC Toolbox</A></LI><LI><A href="/Team:Warsaw/Cytotoxicity">Cytotoxicity study</A></LI><LI><A href="/Team:Warsaw/Safety">Safety</A></LI></UL><LI class="technical_stuff navitem4"><A href="/Team:Warsaw/Achievements">Achievements</A></LI><LI><A href="/Team:Warsaw/Results">Results</A></LI><LI><A href="/Team:Warsaw/Parts">Parts</A></LI><LI><A href="/Team:Warsaw/Cooperation">Cooperation with other iGEM Teams</A></LI><LI><A href="/Team:Warsaw/Judging">Judging</A></LI><LI class="background navitem5"><A href="/Team:Warsaw/Human_Practice">Human Practice</A></LI><LI><A href="/Team:Warsaw/Strategy_overview">Strategy overview</A></LI><LI><A href="/Team:Warsaw/Cooperation_with_PWN">Cooperation with PWN</A></LI><LI><A href="/Team:Warsaw/The night of Biologists">The Night of Biologists</A></LI><LI><A href="/Team:Warsaw/Science_Picnic_2013">Science Picnic 2013</A></LI><LI><A href="/Team:Warsaw/Our cooperation with mass media">Our cooperation with mass media </A></LI><LI class="navitem4"><A href="/Team:Warsaw/Extras" title="">Extras</A></LI><LI><A href="/Team:Warsaw/Protocols">Protocols</A></LI><LI><A href="/Team:Warsaw/Genetic_lab_journal">Genetic lab journal</A></LI><LI><A href="/Team:Warsaw/Cellular_biology_lab_journal">Cellular biology lab journal</A></LI><LI><A href="/Team:Warsaw/Glossary">Project glossary</A></LI><LI><A href="/Team:Warsaw/Adventure with grants">Adventure with grants</A></LI><LI class="team navitem6"><A href="/Team:Warsaw/Team" title="">Team</A></LI><LI><A href="/Team:Warsaw/Team">Team</A></LI><LI><A href="/Team:Warsaw/Acknowledgments">Acknowledgments</A></LI><LI><A href="/Team:Warsaw/Sponsors">Sponsors</A></LI></DIV></DIV></DIV><DIV id="heading"><DIV class="centering"><H1>

Protocols

            </H1></DIV></DIV><DIV id="innercontent"><DIV class="centering"><DIV class="whitebox article"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Synthetic_biology_protocols"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Synthetic biology protocols</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Chemocompetent_bacteria"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Chemocompetent bacteria</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Transformation_of_chemocompetent_bacteria"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Transformation  of chemocompetent bacteria</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Transformation_of_electrocompetent_bacteria"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Transformation of electrocompetent bacteria</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Alkaline_Lysis"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Alkaline Lysis</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#DNA_Digestion"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">DNA Digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#DNA_Ligation"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">DNA Ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#PCR"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">PCR</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#Cellular_biology_protocols"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Cellular biology protocols</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#AlamarBlue_assay"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">AlamarBlue assay</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Protocol:"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Protocol:</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Flow_Cytometry"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Flow Cytometry</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Preparation_of_the_samples"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Preparation of the samples</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Hoechst_stain"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Hoechst stain</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Hoechst_33342_stain_protocol"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Hoechst 33342 stain protocol</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Protocol"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Micronucleus_assay"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Micronucleus assay</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Cell_growth_assay"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Cell growth assay</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Synthetic_biology_protocols"><CENTER>Synthetic biology protocols</CENTER></SPAN></H2><H3><SPAN class="mw-headline" id="Chemocompetent_bacteria">Chemocompetent bacteria</SPAN></H3><OL><LI> Inoculate 500 ml LB  with 5 ml of the overnight culture and incubate it shaking at 37 °C till the OD = 0.4.
</LI><LI> Cool the culture for 10 min. in some ice.
</LI><LI> Centifruge the sample at 6000 rpm at 4 °C for 3 minutes.
</LI><LI> Suspend the precipitate gently in ~ 20 ml of cold solution of 0.1 M CaCl<SUB>2</SUB>, then add 0.1 M CaCl<SUB>2</SUB> to a volume of 300 ml.
</LI><LI> Centrifuge again as above.
</LI><LI> Suspend the precipitate in 10 ml of cold 0.1 M CaCl<SUB>2</SUB> and incubate it for 30 min.
</LI><LI> Centrifuge again.
</LI><LI> Suspend the sample in 6 ml of 0.1 M CaCl<SUB>2</SUB> and 15% glycerol and Pipette 50 μl to an eppendorf (put them immediately in liquid nitrogen) and store at -80 °C.
</LI></OL><H3><SPAN class="mw-headline" id="Transformation_of_chemocompetent_bacteria">Transformation  of chemocompetent bacteria</SPAN></H3><OL><LI> Put the bacteria into ice for 2-3 minutes (or until it melts)
</LI><LI> Add a cooled plasmid or ligation in a volume not bigger than 20 μl and stir with a tip
</LI><LI> Keep it in the ice for 20 to 30 minutes
</LI><LI> Put it  to a heating block set for 42°C for 1.5 min
</LI><LI> Put it back to ice for 2 min
</LI><LI> Add 900 μl  of SOB and incubate at 37°C for 1 h
</LI><LI> Sow on the appropriate selective deposit.
</LI></OL><H3><SPAN class="mw-headline" id="Transformation_of_electrocompetent_bacteria">Transformation of electrocompetent bacteria</SPAN></H3><OL><LI> Put the bacteria into some ice for 2-3 minutes (or until it melts).
</LI><LI> 50 μl of bacteria pipete to a dialised and cooled plasmid DNA or ligation
</LI><LI> Put the transformation mixture to a cuvette (also previously cooled on ice) - it is important that the sample must be on the bottom of the cuvette and free of air bubbles. You can tap the cuvette several times on the table.
</LI><LI> Transform the bacteria in an electroporator set on 2500 V (time constant should be around 5)
</LI><LI> Immediately add 900 μl  of SOB and incubate at 37°C for 1 h
</LI><LI> Sow the appropriate selective deposit
</LI></OL><H3><SPAN class="mw-headline" id="Alkaline_Lysis">Alkaline Lysis</SPAN></H3><OL><LI> Pick single colony and inoculate 3 ml of LB broth. Shake at 37°C overnight.
</LI><LI> Centrifuge two times 1.5 ml cells in the same 1.5 ml Eppendorf tube at 6000 rpm for 1 minute. Aspirate supernatant. 
</LI><LI> Resuspend cell pellet in 100 μl of GTE buffer (50 mM Glucose, 25 mM Tris-HCl, 10 mM EDTA, pH 8). 
</LI><LI> Add 200 μl of buffer II (0.2 M NaOH, 1% SDS). Invert gently tube 6-8 times.
</LI><LI> Add 150 μl of buffer III (3M potassium acetate, 2M acetic acid, pH 5.5). This solution neutralizes NaOH in the previous lysis step while precipitating the genomic DNA and SDS in an insoluble, white, amorphous precipitate. Incubate in ice for 5 min. Spin at top speed 10 min.
</LI><LI> Transfer supernatant to a new tube, being careful not to pick up any white flakes.  Precipitate the nucleic acids with 1ml of 96% ethanol for 5 minutes at room temperature and centrifuge at top speed for 10 minutes. Aspirate supernatant, add 200 &amp;mul 70% ethanol and centrifuge at top speed for 1 minute.
</LI><LI> Aspirate off all the ethanol supernatant. Dissolve the pellet in 40 μl of water. Store at -20°C.
</LI></OL><H3><SPAN class="mw-headline" id="DNA_Digestion">DNA Digestion</SPAN></H3><P>Add to Eppendorf tube 
(for 20 μl):
</P><OL><LI> 15 μl water
</LI><LI> 2 μl buffer
</LI><LI> 2 μl DNA that you want digest (ex. plasmid)
</LI><LI> 0,5 μl enzyme X and 0,5 μl enzyme Y (or 1 μl enzyme Z)
</LI></OL><P>(for 10 &amp;mul [verification cloning]):
</P><OL><LI> 7 μl plasmid
</LI><LI> 1 μl buffer
</LI><LI> 1 μl water
</LI><LI> 0,5 μl enzyme X and 0,5 μl enzyme Y (or 1 μl enzyme Z)
</LI></OL><P>Tubes incubate at 37°C for 1 h.
</P><H3><SPAN class="mw-headline" id="DNA_Ligation">DNA Ligation</SPAN></H3><P>Add to Eppendorf tube
</P><OL><LI> 5 μl water
</LI><LI> 2 μl buffer
</LI><LI> 10 μl digest plasmids mix
</LI><LI> 1 μl T4 ligase
</LI></OL><P>Incubate 2-3 h at room temperature or O/N at 16 degrees Celsius
</P><H3><SPAN class="mw-headline" id="PCR">PCR</SPAN></H3><P>Add to PCR tube (0,2 ml), do it on ice:
</P><OL><LI> 34 μl water
</LI><LI> 10 μl buffer with MgCl2
</LI><LI> 1 μl 10mM dNTPs
</LI><LI> 2 μl primers
</LI><LI> 2 μl template DNA (200x attenuate plasmid)
</LI><LI> 1 μl Phusion polymerase
</LI></OL><P>PCR programme:
</P><OL><LI> Initialization step – 98°C – 3 min
</LI><LI> Denaturation step – 98°C – 30 sec
</LI><LI> Annealing step – 55°C – 30 sec
</LI><LI> Elongation step – 72°C – 40 sec
</LI><LI> Final elongation – 72°C – 5 min
</LI><LI> Final hold – 4°C – to end of time
</LI></OL><P>30 cycles
</P><P>Total time: about 1 h 40 minutes.
</P><H2><SPAN class="mw-headline" id="Cellular_biology_protocols"><CENTER>Cellular biology protocols</CENTER></SPAN></H2><H3><SPAN class="mw-headline" id="AlamarBlue_assay">AlamarBlue assay</SPAN></H3><P>To indicate cell viability, we have decided to implement Alamar Blue assay. It bases on the ability of living cells to convert nonfluorescent resazurin to the highly fluorescent molecule- resorufin. While entering the cells, resazurin is being reduced to resorufin and the bright, red fluorescence is to be observed. Viable cells are continuing to convert resazurin to resorufin, so the strength of fluorescence increases proportionally and may be used to measure viability of the cells.
</P><H4><SPAN class="mw-headline" id="Protocol:">Protocol:</SPAN></H4><P><U>1st day</U>
Preparation of the 96-well plate:
</P><UL><LI> Adding 80µl media with 10.000 cells to each well
</LI><LI> Adding 20µl of properly concentrated acrylamide to each well 
</LI><LI> Row 1 and 2 -&gt; no acrylamide: control
</LI><LI> Row 3 and 4 -&gt; 1mM  acrylamide (1,76µl of acrylamide + 350 µl of PBS)
</LI><LI> Row 5 and 6 -&gt; 2mM acrylamide (3,52µl of acrylamide + 348,48 µl of PBS)
</LI><LI> Row 7 and 8 -&gt; 3mM acrylamide (5,28µl of acrylamide + 346,72 µl of PBS)
</LI><LI> Row 9 and 10 -&gt; 4mM acrylamide (7,04µl of acrylamide + 344,96 µl of PBS)
</LI><LI> Row 11 and 12 -&gt; 5mM acrylamide (8,8µl of acrylamide + 343,2 µl of PBS)
</LI></UL><P><U>2nd day</U></P><UL><LI> Adding 10µl of AlamarBlue to each well in rows A and B (24 hours incubation)
</LI><LI> Incubate for 2 hours at 37°C, protected from light, in a cell culture incubator
</LI><LI> Reading absorbance at 570 nm
</LI></UL><P><U>3rd day</U></P><UL><LI> Adding 10µl of AlamarBlue to each well in rows C and D (48 hours incubation)
</LI><LI> Incubate for 2 hours at 37°C, protected from light, in a cell culture incubator
</LI><LI> Reading absorbance at 570 nm
</LI></UL><P><U>4th day</U></P><UL><LI> Adding 10µl of AlamarBlue to each well in rows E and F (72 hours incubation)
</LI><LI> Incubate for 2 hours at 37°C, protected from light, in a cell culture incubator
</LI><LI> Reading absorbance at 570 nm
</LI></UL><P><U>5th day</U></P><UL><LI> Adding 10µl of AlamarBlue to each well in rows G and H (96 hours incubation) 
</LI><LI> Incubate for 2 hours at 37°C, protected from light, in a cell culture incubator
</LI><LI> Reading absorbance at 570 nm
</LI></UL><H3><SPAN class="mw-headline" id="Flow_Cytometry">Flow Cytometry</SPAN></H3><H4><SPAN class="mw-headline" id="Preparation_of_the_samples">Preparation of the samples</SPAN></H4><UL><LI>	Vacuum off media with floating cells and place it in different 15ml falcons. 
</LI></UL><UL><LI>	Rinse each well of 6-well plate with 2 ml of PBS. Vacuum off PBS
</LI></UL><UL><LI>	Add 0,5 ml of Trypsin and 1 ml of PBS to each well. 
</LI></UL><UL><LI>	Incubate the plate for 10 min at 37°C
</LI></UL><UL><LI>	Add 1 ml of media to each well to neutralize Trypsin
</LI></UL><UL><LI>	Suspend the cells, then pipette them off and add to previously prepared falcons.
</LI></UL><UL><LI>	Take a 50µl sample to perform a cell count
</LI></UL><UL><LI>	Centrifuge for 4 min at 400 RPM
</LI></UL><UL><LI>	Aspirate supernatant and suspend cells in PBS (1mln cells/1 ml)
</LI></UL><UL><LI>	Transfer 300 µl of suspension to the test tube
</LI></UL><UL><LI>	Centrifuge for 4 min at 400 RPM
</LI></UL><UL><LI>	Aspirate supernatant and suspend cells in 100µl of binding buffer 
</LI></UL><UL><LI>	Add 4 μl of Annexin V or 5 μl of PI
</LI></UL><UL><LI>	Incubate for 20 min at room temperature, protected from light
</LI></UL><UL><LI>	Add 400µl of binding buffer, suspend, and transfer to test tubes used in your cytometer
</LI></UL><UL><LI>	Analyze by flow cytometry
</LI></UL><H3><SPAN class="mw-headline" id="Hoechst_stain">Hoechst stain</SPAN></H3><H4><SPAN class="mw-headline" id="Hoechst_33342_stain_protocol">Hoechst 33342 stain protocol</SPAN></H4><P>In order to be able to examine cell’s nuclei morphology we introduced staining with Hoechst dye. The dye binds to the minor groove of dsDNA preferentially to sequences rich in adenine and thymine. As the Hoechst dye may be excited by ultraviolet light at around 350 nm, and then emits blue fluorescent light (maximum at around 461 nm)  we’ll be able to observe changes in the cell’s nuclei using fluorescent microscopy. As UV light is harmful for the cells, they have to be fixed first in order to avoid excess death during the assay. Our aim is to study changes that occurred due to treating cells with the acrylamide: cell division, cell apoptosis and amount of micronuclei. 
</P><H4><SPAN class="mw-headline" id="Protocol">Protocol</SPAN></H4><P><U>Fixation of the cells:</U></P><UL><LI> Remove media
</LI><LI> Rinse 3 times with PBS (1 ml per well of 6-well plate)
</LI><LI> Add 1ml of 4% paraformaldehyde and 0,5% sucrose in PBS (per well of 6-well plate
</LI></UL><P>Note: Paraformaldehyde is toxic! Always use gloves, safety glasses and work under the hood while using  solution.
</P><P><U>Staining with Hoechst:</U></P><UL><LI> Remove the fixative with aspirator
</LI><LI> Rinse 3 times with PBS
</LI><LI> Add 1 ml of 10.000 times diluted stock solution of Hoechst 
</LI><LI> Incubate 30 minutes in darkness at 37°C
</LI><LI> Remove stain
</LI><LI> Rinse once with PBS then add 1 ml of PBS
</LI></UL><P><U>Preparing a microscope preparations:</U></P><UL><LI> Prepare microscopic slides 
</LI><LI> Place a drop of mountant in the middle of the slide
</LI><LI> Carefully place the round glass slide on the droplet (remember to put the slide side with cells to mountant down, so the cells stay immersed)
</LI><LI> Remove excess liquid 
</LI></UL><P><U>Observations:</U></P><P>Once the microscope preparations are ready it is possible to examine them using fluorescent microscope. Hoechst dye emits blue  fluorescence light at  around 461 nm while excited by ultraviolet light at around 350 nm.
</P><H3><SPAN class="mw-headline" id="Micronucleus_assay">Micronucleus assay</SPAN></H3><P>As we would like to measure the genotoxity of acrylamide, we have decided to implement micronucleus assay. This test belongs to the most popular and widely used methods of searching for potentially mutagenic substances. 
</P><P>The assay bases on formation of the micronuclei, which are the erratic nuclei formed during the anaphase due to mutagen exposition. They may originate both from acentric chromosome fragments or whole chromosomes unable to migrate properly. 
</P><P>The modifications in chromosomes structure, observed during the test, indicate the genotoxic effect of tested substance. 
</P><P>In our lab we would treat different cell lines with various acrylamide concentrations and after dyeing with Hoechst we would count the micronuclei. To examine the mutagenic effect of acrylamide we would use suitable statistical tests.
</P><H3><SPAN class="mw-headline" id="Cell_growth_assay">Cell growth assay</SPAN></H3><P>As we aimed to obtain certain statistic concerning cells proliferation we have decided to implement cell growth assay.
</P><P>To conduct the assay we have seeded cells from different cell lines (HeLa, HEK293 and 143b) in 6-wells plates adding proper concentration of acrylamide to each well and counted cells amount after 24, 48 and 72 hours. 
</P><P><U>Preparation of the 6-wells plates</U></P><UL><LI>adding 3 ml of fresh media to each well
</LI><LI>adding proper amount of each cells to the wells
</LI><LI>incubate at 37°C for 24 hours
</LI><LI>add 333ml of certain acrylamide concentration to each well
</LI><LI>count the cells from each well after 24, 48 and 72 hours using Trypan blue
</LI></UL></DIV></DIV></DIV><DIV id="footer-panel"><DIV class="centering"><DIV class="footerBox"><H4>This project is part of iGEM</H4><P>Read more on the <A href="https://2013.igem.org/Main_Page">iGEM Main Page</A></P></DIV><DIV class="footerBox"><H4>Sponsors:</H4><P><A href="https://2013.igem.org/Team:Warsaw/Sponsors">Click here for full list of sponsors.</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Warsaw/Protocols">http://2013.igem.org/Team:Warsaw/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Warsaw/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Warsaw/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Warsaw/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Warsaw/Protocols&amp;oldid=313813" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>