"
Page
Discussion
View source
History
teams
Log in
 
Team:Uppsala-Sweden/Ethanol
From 2009.igem.org
Home
Vision
Project
Butanol Ethanol Pir Proteins Antisense RNA
Modelling
Modelling of ethanol production PLS model of ethanol concentration
Parts
Promoter Butanol Ethanol 1 Ethanol 2 Pir Antisense RNA
Team
About the Team Students Advisors
Notebook
Acknowledgements
Acknowledgements Sponsors
Contents
1 The Ethanol Project
2 The Pathway
3 The Constructs
3.1 : The Construct for Testing Purposes in E. Coli with adh2 from Z. mobilis
3.2 : The Final Construct for Synechocystis sp PCC6803 with adh2 from Z. mobilis
3.3 : The Construct for Testing Purposes in E. Coli with adh2 from S. cerevisiae
3.4 : The Final Construct for Synechocystis sp PCC6803 with adh2 from S. cerevisiae
4 Status
5 References
The Ethanol Project
Cyanobacteria have the capability to harvest the energy from the sun and convert it into other forms of energy. The natural way for these organisms is to store it as sugars or other carbohydrates in a way similar to plants. By introducing a casette of genes, we would like to obtain ethanol production with cyanobacteria, in particular Synechocystis sp PCC6803
The idea to produce ethanol with the use of cyanobacteria was originally proposed and already accomplished by Ming-De Deng and John R. Coleman. [1]
The source of carbon for the ethanol production is pyruvate which is also oxidative decarboxylated to be used in the Krebs Cycle. To obtain a higher ethanol yield, it might be sufficient to increase the pyruvate pool . By inhibiting the pyruvate dehydrogenase the usage of pyruvate for the Krebs cycle is reduced and the accumulating pyruvate could be used for ethanol production instead.
We from the Uppsala iGEM Team decided to try to improve the production of ethanol by interfering with the metabolic pathways of Synechocystis sp. PCC 6803 by an Antisense RNA and a protein-mediated approach. Hence we build a construct that encodes for ethanol production in our host organism.
The Pathway
Light energy is used in the Calvin cycle to transform water and carbon dioxide into organic compounds, among those is Ribulose-1,5-bisphosphate, which immediately splits into two 3-Phosphoglycerate molecules. [2] These two molecules can now be converted to 2-Phosphoglycerate. Enolase and pyruvatekinase finally catalyze the reaction over phosphoenolpyruvate to pyruvate. [3] The resulting pyruvate molecule can be now used as a substrate for ethanol production. A schematic view can be found further down .
Our first BioBrick for ethanol production  is the pyruvate decarboxylase from Zymomonas mobilis which decarboxylates pyruvate to form acetaldehyde (ethanal). [4] [5] Ethanal is then further catalyzed with a alcohol dehydrogenase to ethanol. [6] Initially we intended to use the adh2 from Z. mobilis but during the project we encountered some problems with our Z. mobilis strain, which made us change our plans as you will see further below.
The Constructs
We designed multiple constructs as we ran into some unexpected restriction site problem during the assembling process. The Z. mobilis strain which was source of the pdc and adh, most probably had a mutation inside the adh2 gene, which lead to a novel EcoRI restriction site. Thus we decided to build a back up construct using the adh2 from S. cerevisiae.
We build as well versions for testing purposes in E. Coli.
 : The Construct for Testing Purposes in E. Coli with adh2 from Z. mobilis
 : The Final Construct for Synechocystis sp PCC6803 with adh2 from Z. mobilis
 : The Construct for Testing Purposes in E. Coli with adh2 from S. cerevisiae
 : The Final Construct for Synechocystis sp PCC6803 with adh2 from S. cerevisiae
Click here to jump up again.
Status
as of 2009-1018
We have completed the construction of both the construct variants and they are currently being tested in E.coli employing the pLac promoter. The Z.mobilis variant has been transformed to to Synechocystis and is currently growing to pick able size on plate, this takes about one week.  The Yeast construct is lagging behind the Z.mobilis variant due to some failed transformations.
After transformation to Synechocystis it takes approximately two weeks till the culture has grown to enough volume to be used in tests.
References
[1] Ethanol synthesis by genetic engineering in cyanobacteria. Deng MD, Coleman JR. Appl Environ Microbiol. 1999 Feb;65(2):523-8
[2] Jeremy M. Berg, John L. Tymoczko, L. Stryer Biochemistry 5th Edition; Ch.21.1 p826-838
[3] Jeremy M. Berg, John L. Tymoczko, L. Stryer Biochemistry 5th Edition; Ch.16.1 p653
[4] Pyruvate decarboxylase of Zymomonas mobilis: isolation, properties, and genetic expression in Escherichia coli. Neale AD, Scopes RK, Wettenhall RE, Hoogenraad NJ. J Bacteriol. 1987 Mar;169(3):1024-8
[5] Structure and properties of pyruvate decarboxylase and site-directed mutagenesis of the Zymomonas mobilis enzyme. Judith M. Candy and Ronald G. Duggleby. Biochimicat Biophysica Acta (BBA) - Protein Structure and Molecular Enzymology Volume 1385, Issue 2, 16 June 1998, Pages 323-338
[6] Cloning and Sequencing of the Alcohol Dehydrogenase II Gene from Zymomonas mobilis. Conway et al. JOURNAL OF BACTERIOLOGY, June 1987, p. 2591-2597
Retrieved from " http://2009.igem.org/Team:Uppsala-Sweden/Ethanol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
