<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Nanjing_NFLS_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Nanjing NFLS/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>https://2016.igem.org/wiki/index.php?title=Team:Nanjing_NFLS/Project</TITLE></P><DIV class="header"><OL class="header_main" style="list-style-type:none;"><LI style="list-style-type:none;"><A href="https://2016.igem.org/wiki/index.php?title=Team:Nanjing_NFLS">HOME</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project">PROJECT</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project #overview_id">OVERVIEW</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project #design_id">DESIGN</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project #result_id">RESULT</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project #verification_id">VERIFICATION</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project #safety_id">SAFETY</A></LI></OL><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Teams">TEAMS</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Teams #members_id">MEMBERS</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Teams #instructors_id">INSTRUCTORS</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Attributions">ATTRIBUTIONS</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Collaborations">COLLABORATIONS</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/HP/Silver">HUMAN PRACTICE</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/wiki/index.php?title=Team:Nanjing_NFLS/Notebook ">NOTEBOOK</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/wiki/index.php?title=Team:Nanjing_NFLS/Notebook #protocol">PROTOCOL</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/wiki/index.php?title=Team:Nanjing_NFLS/Notebook #lab_pictures">LAB PICTURES</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Parts">PARTS</A></LI></DIV><DIV class="middle"><DIV class="middle_pic"><P style="width:1000px;
		height:680px;font-family: Times New Roman;color:#ff;font-size: 80px;
		font-weight: bolder;position: relative; left:300px;top:200px;">No More Algae Bloom.</P></DIV></DIV><DIV class="overview_title" id="overview_id"><DIV class="overview_title_text" style="text-align:center; font-size:60px;">Overview</DIV></DIV><DIV class="overview"><P class="overview_content" style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">
			The seasonal mass outbreak of green-blue algae poses serious threat to water environment. Gas vesicle protein, widely existing in all forms of planktonic cyanobacteria, provides the cyanobacteria with buoyancy and regulates their vertical distributions. GvpA protein acts as the main structural protein of gas vesicle. 
			Therefore, we plan to mutate GvpA1 gene in microcysis (main type of cyanobacteria causing algae boom) to limit the level of GvpA1 expression and change the conformation of gas vesicle protein---- a possible strategy to disrupt the distribution of microcysis in water and control algae bloom. The recombinant shuttle plasmid containing modified GvpA1 gene is transformed back into microcysis and western blot identification proves that the expression of GvpA1 after modification reduces significantly. The modified segment of GvpA1 and the shuttle plasmid we specially constructed to both replicate in E.coli and microcysis have been submitted as parts to be reused and analyzed by other teams.
		</P></DIV><DIV class="design_title1" id="design_id"><DIV class="design_title1_text" style="text-align:center; font-size:60px;">Design</DIV></DIV><DIV class="design"><DIV class="design_content"><P class="design_content_biaoti" style="font-weight: bold;
				font-size: 30px;
				height: 60px;
				line-height: 60px;
				color: #000027;font-family:Century Gothic;"> Gas vesicle protein and GvpA1 gene mutation</P><P style="width:1200px;
				margin:0px auto;
				font-size: 20px;
				line-height: 28px;
			    padding:10px;
				text-align: justify;
				color: #000027;
				font-family:Century Gothic;">
			The seasonal mass occurrences of green-blue algae (algae bloom) at water surfaces have been damaging the water environment for decades. Microcysis is a main type of green-blue algae that causes algae bloom. Floating and gathering together are the last and most critical stage in the formation of algae bloom. In other words, within short time period, change in relative vertical position (mainly upward) of mass amount of algae propagated and the resulted blockage of oxygen and sunlight on water surface are the main causes of algae bloom. Gas vesicle protein, which widely exists in all forms of planktonic cyanobacteria, provides cells with buoyancy by changing the density of cells and regulates their vertical distributions in natural waters, enabling them to achieve ideal vertical position in water for growth and subsequent niche colonization. Therefore, it is important to understand the characteristics of gas vesicles for water-bloom control. </P><P style="width:1200px;
				margin:0px auto;
				font-size: 20px;
				line-height: 28px;
			    padding:10px;
				text-align: justify;
				color: #000027;
				font-family:Century Gothic;">
			 GvpA protein forms the skeleton of gas vesicle and acts as the main structural protein. Therefore, we plans to mutate GvpA1 gene in order to limit the level of GvpA1 expression and change the conformation of gas vesicle protein---- a possible strategy to disrupt the distribution of microcysis in water and control algae bloom. By changing two base pairs, we mutate a serine to an alanine (hydrophilic amino acid to hydrophobic amino acid) and an alanine to a glycin. </P>

		     Gene sequence of GvpA1 before mutation:<P style="text-transform：Uppercase;font-size: 18px;line-height: 28px;font-family:Century Gothic;">atggcagtcgaaaaaaccaactcttcctctagcttagctgaagttatcgatcgtatccttgataaaggtatcgttattgatgcttgggctcgcgtatctctcgtcggaatcgaattattagcgattgaagctcgtgtagt<SPAN style="color:red;">C</SPAN>atcgcttc<SPAN style="color:red;">T</SPAN>gttgaaacctacctcaaatatgctgaagctgttggtctgactcaatctgcagcggttcctgcttaa</P>

			 Gene sequence of GvpA1 after mutation:<P style="text-transform：Uppercase;font-size: 18px;line-height: 28px;font-family:Century Gothic; ">atggcagtcgaaaaaaccaactcttcctctagcttagctgaagttatcgatcgtatccttgataaaggtatcgttattgatgcttgggctcgcgtatctctcgtcggaatcgaattattagcgattgaagctcgtgtagt<SPAN style="color:red;">T</SPAN>atcgcttc<SPAN style="color:red;">G</SPAN>gttgaaacctacctcaaatatgctgaagctgttggtctgactcaatctgcagcggttcctgcttaa</P>

			 Amino acid sequence after mutation：
			 MAVEKTNSS<SPAN class="design_content_red">A</SPAN> SL<SPAN class="design_content_red">G</SPAN>EVIDRIL DKGIVIDAWA RVSLVGIELL AIEARVVIAS VETYLKYAEA VGLTQSAAVPA<P class="design_content_biaoti" style="font-weight: bold;
				font-size: 30px;
				height: 60px;
				line-height: 60px;
				color: #000027;font-family:Century Gothic;"> Use of shuttle plasmid </P><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">After we recover gene segment of GvpA1 from cloning plasmid pET30a(+), we then link GvpA1 gene with shuttle plasmid pPKE2. We introduce shuttle plasmid into our experiments because we need to transform both E.coli and microcystis aeruginosa during experiments. （The plasmid of microcystis aeruginosa itself does not contain selectable marker and cannot replicate in E.coli. Traditional plasmid vector used to transform E.coli cannot transform microcystis either. ）The shuttle plasmid pPKE2 we constructed and used possesses the replication origins of E.coli plasmid and cyanobacterium plasmid, CaMV35S promoter, multiple cloning sites (MCS) and rbcS polyA terminator subcloned from plasmid pKYLX一71.35, which makes it convenient to insert and express target gene and to screen out the recombinants. Ori site of cyanobacterium plasmid comes from indigenous plasmids pPbs extracted from Plectonema boryanum. </P></DIV></DIV><DIV class="procedure_title"><DIV class="procedure_title_text" style="text-align:center; font-size:60px;">Procedure</DIV></DIV><DIV class="procedure" style="width:1200px;
		margin:0px auto;
		font-size: 20px;
		line-height: 28px;
	    padding:10px;
		text-align: justify;
		color: #000027;
		font-family:Century Gothic;"><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;"><B>1.DNA extraction from Microcystis：</B>
			Extract DNA from Microcystis aeruginosa using PowerSoil DNA Isolation Kit
			<B>2. Mutation PCR</B>
			Use mutation PCR to isolate and amplify the coding sequence of gene GvpA1 from genomic DNA and introduce restriction enzyme sites NdeI and HindIII.
			We have changed two separated bases in process, aiming to cause change of two <SPAN class="procedure_bold">amino acids in</SPAN> gene expression product of GvpA1 gene.<B>3. Plasmid pET vector construction and E.coli transformation</B>
			1) Take plasmid pET30a(+) as cloning vector, link gene segment of GvpA1 with plasmid pET30a(+) after enzyme digestion of NdeI and HindIII to get constructed vector.
			2) Prepare chemically competent E.coli cells and transform the recombinant plasmid containing GvpA1 into E.coli BL21(DE3) .
			3) Positive clones were screened by ampicillin and plasmid DNA extracted is identified with restriction endonucleases. Then use restriction enzyme digestion of HindIII and NdeI to recover gene segment of GvpA1 from plasmid pET30a(+)<B>4.  Shuttle plasmid pPKE2 vector construction and E.coli transformation</B>
			1) Take plasmid pPKE2 as expression vector and link gene segment of GvpA1 with the plasmid after enzyme digestion of HindIII and XbaI.
			2) Positive clones were screened by kanamycin and plasmid DNA extracted is identified with restriction endonucleases.<B>5. Mircrocystis transformation by shuttle plasmid vector</B>
			1) Basic kanamycin resistance test of Microcystis aeruginosa shows that the algae cannot resist concentration of 5μg/mL and above on BG11. Therefore， we choose 10-15μg/mL kanamycin for screening after shuttle plasmid pPKE2 containing modified GvpA1 gene is transformed into Microcysitis aeruginosa.
			2) Use western blot and SDS page to identify change in GvpA1 protein expression in Microcysitis aeruginosa after pPKE2 vector containing modified GvpA1 gene has been introduced.<B>6. Select algal strains grown well on BG11 medium containing Km(15μg/m )</B>    for extended culture. We then carried out controlled experiments between these strains containing recombinant plasmid and wild type to compare their growth conditions（survival competitaive and vertical distribution in water）</P></DIV><DIV class="result_title" id="result_id"><DIV class="result_title_text" style="text-align:center; font-size:60px;">Result</DIV></DIV><DIV class="result" style="width:1200px;
		margin:0px auto;
		font-size: 20px;
		line-height: 28px;
	    padding:10px;
		text-align: justify;
		color: #000027;
		font-family:Century Gothic;"><P class="result_content" style="font-weight: bold;
				font-size: 30px;
				height: 60px;
				line-height: 60px;
				color: #000027;font-family:Century Gothic;">Controlled experiments between mutation type and wild type</P><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">
			After extended culture of selected cyanobacterium (recombination group) and wild type (control group), we carried out controlled experiments to get comparison between mutation type and wild type. we mix the culture medium thoroughly, add 200uL formaldehyde to fix the cell of microcysis, count the number with blood cell counting chamber and repeat three times to obtain the average value per mL culture medium. The formula of computation are as follows:
			Counts/mL= average value×10^5×dilution times
		</P><P class="result_content2" style="font-size: 30px;line-height: 28px;">Growth curve</P><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;"> 
			As shown above figure, the total counts and growth rate of microcysis recombination are slightly less than the control when cultured in BG11 medium;</P><P class="result_content" style="font-size: 30px;line-height: 28px;">The shape and size of microcysis</P><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">
			The shape and diameter of microcysis are showed above. It could be seen from the picture that the mircrocystis of the control group is slightly larger. The size change is observed using a microscope and the diameter comparison is showed in the figure above. 

  		Through the comprehensive comparison of total counts, growth rate, shape and size between recombination group and control group, we arrive at the conclusion that the recombination group(microcystis transformed by recombination vector) has slightly lower level of survival competitive compared to control group.
  		
  	    However, we can observe that microcystis of recombination group acquires lower level of aggregation on the surface of the water, proving that the recombination group has less possibility to cause phenomenon leading to algae bloom.


		following picture showed the different percent of floating microcysis and submerged microcysis in recombination group. It can be seen that about 35% of the microcysis were submerged which might be the results of expression of the mutated gvpA in microcysis. 
		</P></DIV><DIV class="verification_title" id="verification_id"><DIV class="verification_title_text" style="text-align:center; font-size:60px;">Verification</DIV></DIV><DIV class="verification"><P class="verification_content" style="font-size: 30px;line-height: 28px;font-family:Century Gothic;">1. Verification of mutation PCR</P><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;"><SPAN class="veri_fig1" style="font-size: 20px;
			line-height: 28px; text-align: justify;
			color: #000027;
			font-family:Century Gothic;">Fig.1. Electropherogram of GvpA1 gene </SPAN>
			After extraction of DNA from microcystis aeruginosa，we have amplified and mutated the target gene GvpA1. With the knowledge of the length of target gene, we’ve verified its conformance to length requirements through electrophoresis.
		</P><P class="veri_fig2_text" style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">
			Fig.2. Sanger sequencing of GvpA1 gene after mutation PCR
			Sequence information：ATGGCAGTGG AAAAAACCAA CAGCTCAGCA TCTCTGGGTG AAGTGATTGA CCGCATCCTG GACAAAGGTA TCGTTATTGA CGCCTGGGCC CGTGTCAGCC TGGTGGGCAT TGAACTGCTG GCGATCGAAG CCCGCGTGGT TATTGCGAGC GTGGAAACCT ATCTGAAATA CGCTGAAGCA GTCGGTCTGA CGCAGTCGGC AGCGGTTCCG GCACACCATC ACCATCACCA TTGATAAAAG CTT</P><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">
			In order to ensure the planned mutation of GvpA1, we have verified through sanger sequencing.
		</P><P class="verification_content" style="font-size: 30px;line-height: 28px;font-family:Century Gothic;">2. Verification of pET30a(+) vector construction</P><P class="veri_fig3_text" style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">Fig.3. Electropherogram of plasmid pET30a(+) DNA
			1: plasmid pET30a(+) containing GvpA1; 2: Products after Mlu I and Hind III digestion</P><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">After transformation of E.coli BL21(DE3), we extracted the recombinant plasmid for restriction endonucleases identification. we’ve verified its conformance to length requirements through electrophoresis. </P><P class="verification_content" style="font-size: 30px;line-height: 28px;">3. Verification of pPKE2 vector construction</P><P class="veri_fig4_text" style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">Fig.4. Electropherogram of plasmid pPKE2 DNA
			A1:products after EcoRI digestion      
			A2: products after BamHI digestion     
			M: λDNA marker for Hind III digestion    
			B1: DNA of recombinant plasmid after EcoRI and SphI digestion    
			B2: Recombinant plasmid pPKE2   
			M: λDNA marker for EcRI and HindIII digestion</P><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">
			After transformation of E.coli BL21(DE3), we extracted the recombinant plasmid for restriction endonucleases identification.

			The plasmid should have a length of 9.8kb after ligation with target gene GvpA1, which is verified and showed in A1.

			Gene segment of GvpA1 and pPKE2 both include BamHI sites and two separated segments with length 2.8kb and 7.0kb each will show up after BamHI digestion， which is verified in A2.

			When gene segment of GvpA1 is inserted into the plasmid, a new SphI site will show up between original EcRI-SphI segment (2.8kb). Therefore, two separated segments with length 1.3kb and 1,5kb each will show up after EcoRI and SphI digestion, which is verified in B1.</P><P class="verification_content" style="font-size: 30px;line-height: 28px;">4. Verification of Microcystis transformation</P><P class="veri_fig5_text" style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">Fig.5. growth condition of microcystis a week after transformation</P><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">
			After shuttle plasmid vector is transformed into microsystis aeruginosa, transposon screening is carried out by 7-10 days of culturing on BG-11 medium which contains 15µg/mL of kanamycin.
			
			Left: control group (no recombinant plasmid introduced)
			Right: Microcystis of Recombination group appears on BG11 medium containing 
			Kanamycin </P><P class="verification_content" style="font-size: 30px;line-height: 28px; font-family:Century Gothic;">5. Identification of change in GvpA1 protein expression in Microcysitis</P><DIV class="verification_content_box"><DIV class="verification_content_box_text">
				M1: Protein marker
				M2: Western marker
	  			PC1: BSA(1 μg)
				PC2: BSA(2 μg)
				NC: Non-induced whole cell lysing reagent
				1: whole cell lysing reagent induced for 16h in 15°C
				2: whole cell lysing reagent induced for 4h in 37°C
				NC1: Supernatant of non-induced lysis buffer
				NC2: Precipitation of non-induced lysis buffer
				3: Supernatant of lysis buffer induced for 16h in 15°C
				4: Precipitation of lysis buffer induced for 15h in 15°C
				5: Supernatant of lysis buffer induced for 4h in 37°C
				6: Precipitation of lysis buffer induced for 4h in 37°C
			</DIV></DIV><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;">
			Western blotting result verifies that with microcystis transformed by recombinant plasmid, GvpA genes still express in microcystis but with limited expression level. </P></DIV><DIV class="safety_title" id="safety_id"><DIV class="safety_title_text" style="text-align:center; font-size:60px;">Safety</DIV></DIV><DIV class="safety"><DIV class="safety_content"><P class="safety_content_biaoti" style="font-size: 30px;line-height: 28px; padding-left:10px;">Materials and Facilities</P><P style="width:1200px;
				margin:0px auto;
				font-size: 20px;
				line-height: 28px;
			    padding:10px;
				text-align: justify;
				color: #000027;
				font-family:Century Gothic;
				margin-top:-40px;">
				Team Nanjing_NFLS attaches great importance to safety of synthetic biology and our team members are all aware of the consequences of irregular operations in labs. Therefore, our team strives to achieve a careful and comprehensive consideration in biosafety.

				The experimental materials we use includes microorganism of risk level one (Escherichia coli DH5a BL21(DE3)) and Microcystis aeruginosa , common laboratory reagents and other regular experimental materials. These are common biological materials which do not have any special harm or danger. Safety of experimenters can be guaranteed as long as we operate the facilities in a proper way, and take relevant preventive measures in advance.
				We conduct our experiments in the open workbench after considering the safety of experimental materials. All of the other experimental facilities meet the criterion of first-level labs. </P><P class="safety_content_biaoti" style="font-size: 30px;line-height: 28px;padding-left:10px;">Lab Rule</P><UL class="safety_content_ul" style="width:1200px;
						margin:0px auto;
						font-size: 20px;
						line-height: 28px;
					    padding-left:20px;
						text-align: justify;
						color: #000027;
						font-family:Century Gothic;
						margin-top:-40px;
						margin-left:20px;"><LI> Wear rubber gloves in all experiments.</LI><LI> Necessary steps should be performed in bio-safety cabinet.</LI><LI> Fire, electric heaters, microwave oven should not be left while being used.</LI><LI> Sterilize all liquid waste and solid waste containing living organism</LI></UL><P class="safety_content_biaoti" style="font-size: 30px;line-height: 28px;padding-left:10px;padding-left:10px;">Safety Training</P><P style="width:1200px;
				margin:0px auto;
				font-size: 20px;
				line-height: 28px;
			    padding:10px;
				text-align: justify;
				color: #000027;
				font-family:Century Gothic;
				margin-top:-40px;">Before operating in the lab, every member of Nanjing_NFLS has been involved in relevant training of lab safety. The training includes experimental safety knowledge of molecular biology, lab techniques, introductions to regular apparatus and software, safety of standard operation, personnel protection, common emergency response and handling, etc. We have ensured that every member of our team has a full awareness of experiment safety. Only in this way can we avoid various experimental dangers by taking preventive measures in advance. </P><P class="safety_content_biaoti" style="font-size: 30px;line-height: 28px;padding-left:10px;">Parts</P><P style="width:1200px;
			margin:0px auto;
			font-size: 20px;
			line-height: 28px;
		    padding:10px;
			text-align: justify;
			color: #000027;
			font-family:Century Gothic;
			margin-top:-40px;">Harmful parts are not allowed and we ensure that none of our parts would raise any safety issue according to the current professional knowledge.</P></DIV></DIV><DIV class="bottom"><OL class="bottom_main" style="list-style-type:none;"><LI style="list-style-type:none;"><A href="https://2016.igem.org/wiki/index.php?title=Team:Nanjing_NFLS" class="bottom_main_a">Home</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project" class="bottom_main_a">Project</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project #overview">Overview</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project #design_id">Design</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project #result_id">Result</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project #verification_id">Verification</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Project #safety_id">Safety</A></LI></OL><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Teams" class="bottom_main_a">Teams</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Teams #members_id">Members</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Teams #instructors_id">Instructors</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Attributions">Attributions</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Collaborations">Colliborations</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/HP/Silver" class="bottom_main_a">Human Practice</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/wiki/index.php?title=Team:Nanjing_NFLS/Notebook" class="bottom_main_a">Notebook</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/wiki/index.php?title=Team:Nanjing_NFLS/Notebook #protocol">Protocol</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/wiki/index.php?title=Team:Nanjing_NFLS/Notebook #lab_pictures">Lab pictures</A></LI><LI style="list-style-type:none;"><A href="https://2016.igem.org/Team:Nanjing_NFLS/Parts" class="bottom_main_a">Parts</A></LI><DIV class="contact_us"><P id="contact_us_title" style="font-size:25px;">Contact us:</P><P>Email:  NFLS_iGEM@126.com</P><P>Mobile: +86 13913850670</P><P>Address: 30 East Beijing Road, Nanjing,Jiangsu, China</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>