<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CSU_Fort_Collins_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CSU Fort Collins/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><SPAN style="color:#D7D392; margin-left:45px"><B>CSU iGEM</B></SPAN><UL id="accordion" class="accordion"><LI class="menu_item" style="border-top: 1px solid black"><A href="https://2017.igem.org/Team:CSU_Fort_Collins">HOME </A></LI><LI class="menu_item"><A href=" https://2017.igem.org/Team:CSU_Fort_Collins/Team"> TEAM   </A></LI><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2017.igem.org/Team:CSU_Fort_Collins/Description">   Description </A></LI><LI><A href="https://2017.igem.org/Team:CSU_Fort_Collins/InterLab">   Interlab Study</A></LI><LI><A href="https://2017.igem.org/Team:CSU_Fort_Collins/Experiments"> Experiments </A></LI><LI><A href="https://2017.igem.org/Team:CSU_Fort_Collins/Protocols"> Protocols </A></LI><LI><A href="https://2017.igem.org/Team:CSU_Fort_Collins/Results"> Results </A></LI></UL><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2017.igem.org/Team:CSU_Fort_Collins/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:CSU_Fort_Collins/Basic_Part"> Basic Parts </A></LI><LI class="menu_item"><A href="https://2017.igem.org/Team:CSU_Fort_Collins/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2017.igem.org/Team:CSU_Fort_Collins/Attributions">ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2017.igem.org/Team:CSU_Fort_Collins/HP/Silver">Silver </A></LI></DIV><DIV class="content_wrapper"><DIV class="column full_size"><H2>Experiments and daily lab notebook</H2><P>6.4.16</P><P>
We picked E. Coli “Stellar” cells from an LB/amp plate. The cells had been transformed with pLC71. 

We put the cells into test tubes with 5 mL of LB and 5 µL of ampicillin and then put those in the shaker at 37℃ overnight.
</P><P>Protocol for picking cells from plate to test tube:
<UL><LI>Materials</LI><LI>Autoclaved test tubes</LI><LI>100mL LB</LI><LI>100mg/mL ampicillin</LI><LI>Test tube rack</LI><LI>Bunsen Burner </LI><LI>P10 or P20 pipette</LI><LI>Latex gloves</LI></UL></P><P>Procedure</P><UL><LI>Place test tubes into the rack, label, date, initial.</LI><LI>Light bunsen burner for flame umbrella.</LI><LI>In each test tube: add 5mL of LB first, then, add 5 µL of AMP. Be sure that your pipette tip is submerged in the LB when adding the AMP.</LI><LI>Remove lid from LB/AMP plate. Use a pipette tip, gently pick a colony.</LI><LI>Eject pipet tip into test tube.</LI><LI>Flame the opening of the tube and recap.</LI><LI>Incubate tubes in 37℃ for approx. 16 hours. </LI></UL><P>
6.5.16</P><P>
We performed plasmid purification according to steps outlined in the S.O.P. in Tom’s lab.
After purification we had ≈120 µL of plasmid solution. 

Fluorimetry determined the concentration of DNA was 317 ng/µL.

Plasmid stored in iGEM box in fourth floor “VWR” fridge.
Now it’s in the 3rd floor fridge (igem limonene box) 8.30.16

Mini-Prep Protocol (Zymo Research):
Materials
<UL><LI>P1 buffer</LI><LI>P2 buffer</LI><LI>P3 buffer (refrigerated)</LI><LI>(3) 1.7mL eppes. </LI></UL>
Procedure
<OL><LI>Pour test tube contents (no more than 5mL) into (3) 1.7mL eppes.</LI><LI>Centrifuge the eppes for 3 minutes at max speed. </LI><LI>Discard supernatant, pour carefully into waste.</LI><LI>Centrifuge 1 minute. </LI><LI>Pipette and discard supernatant.</LI><LI>Add 200µL P1 buffer to 1st eppe.</LI><LI>Pipette up and down until the pellet is dissolved.</LI><LI>Pipette the buffer w/ dissolved pellet into 2nd eppe.</LI><LI>Pipette up and down until the pellet is dissolved.</LI><LI>Pipette the buffer w/ dissolved pellets into 3rd eppe.</LI><LI>Pipette up and down until the pellet is dissolved.</LI><LI>Eppe’s 1 &amp; 2 should be empty. Discard.</LI><LI>Add 200µL P2 by shooting in.</LI><LI>Add to all of your tubes quickly. </LI><LI>Snap all closed.</LI><LI>Inversion mix.</LI><LI>Leave tubes with only P1 &amp; P2 for no more than 5 minutes or DNA will be damaged. Be ready to add P3.</LI><LI>Add 400µL P3 (refrigerated) </LI><LI>Inversion mix until no pink remains.</LI><LI>Let sit for 1-2 minutes</LI><LI>Centrifuge full speed for 10 minutes.</LI><LI>Setup plasmid filter columns over large Zymo collection tubes.</LI><LI>Pipette supernatant into filter column. </LI><LI>Discard pellet</LI><LI>Spin column 1 minute.</LI><LI>Discard flowthrough </LI><LI>Add 200µL Endo Wash Buffer to filter</LI><LI>Spin 1 minute</LI><LI>Discard flowthrough</LI><LI>Add 400µL Plasmid DNA Wash Buffer</LI><LI>Spin 1 minute</LI><LI>Discard flowthrough</LI><LI>Spin 2 more minutes to dry (very important).</LI><LI>Move filter to labeled 1.7 mL eppes</LI><LI>Label</LI><LI>Add 16µL 10mM Tris HCl pH 8.0 to filter column</LI><LI>Let sit 2 minutes</LI><LI>Spin 1 minute</LI><LI>The flowthrough contains the DNA. Don’t throw it out.</LI><LI>(Repeat) Add 16µL 10mM Tris HCl pH 8.0 to filter column</LI><LI>Let sit 2 minutes</LI><LI>Spin 1 minute</LI><LI>The flowthrough contains the DNA. Don’t throw it out.</LI><LI>Throw away filter. Keep flowthrough in labeled eppe. </LI></OL></P><P>
6.6.16</P><P>
60 µL of plasmid were removed from our stock and digested with SAL1 and CutSmart buffer. 
</P><P>
6.7.16</P><P>We ran a gel purification of our cut plasmid, stained and imaged it on the third floor. 
File name: limonene plasmid 1 2016-06-07. George cut our plasmid out of the gel (the largest band on the gel). Our cut gel weighed 0.48g. We then used a kit to dissolve the gel and purify the plasmid. Fluorimetry showed the concentration to be 220 ng/µL.
</P><DIV><DIV class="column full_size"><P>6.8.16</P><P>First try at Infusion cloning. 
<OL><LI>35 µL Stellar E.coli  and 2.5 µL of post infusion pLC71</LI><LI>Cells plated and left to incubate around noon.</LI><LI>gene block: 100ng (μL/25ng) = 4μL</LI><LI>pLC71: 100 ng (μL/220ng) = 0.45μL</LI><LI>In-Fusion pre-mix: 2μL</LI><LI>Distilled water: 3.55μL</LI><LI>50°C for 15 minutes</LI></OL></P><P>
6.9.16</P><P>Took transformed cells out of 4th floor incubator at 11:00 AM. No growth on either plate. Re-transforming with the same infusion mix that we made yesterday</P><OL><LI>Stellar cells - 1501846A</LI><LI>Cell &amp; DNA mixed at 12:08 </LI><LI>The cells were kept on an ice/water miss the whole time. The tubes where cells &amp; DNA were mixed were also on ice before the mix.</LI><LI>The infusion mix was stored in the eppe block for about 5 minutes.</LI><LI>LB/amp plates by RZ 5/20/16</LI><LI>Heat shock at 12:34 at 44 celsius for 45s then directly back on ice.</LI><LI>Plated at 12:40</LI><LI>Incubate 4th floor 49°C at 12:47</LI></OL><P> 6.10.16</P><OL><LI>Took stellar cells transformed w/ infusion mix out of the incubator at 11:40</LI>
No growth, only bubbles.
Second try at infusion mix
Everything on ice for 10 minutes
Diluted plasmid w/ distilled water from sink.</OL><UL><LI>0.9uL 220ng/uL + 1.1uL dH2O</LI></UL>
Transformation:
<OL><LI>E. Coli was pulled out of the freezer at 1:50.</LI><LI>1:55, infusion mix was put in heat block w/ water at 50°C</LI><LI>Infusion mix was then put on ice and then mixed with E. Coli 2 minutes later.</LI><LI>We also transformed the same cells with a control of cut pLC71</LI><LI>Incubate all 3 tubes on ice for 30 minutes</LI><LI>Heat shock at 43°C for 45s</LI><LI>Plated after a 5 minutes back on ice (2:45)</LI><LI>LB/amp plates - AS 5/26/16</LI><LI>Incubated on 4th floor at 37°C overnight</LI></OL><P>6.11.16</P><P>No growth on any plates except for 2 colonies on the control (disgested pLC71)</P><P>6.21.16</P><P>Resuspending primers</P><OL><LI>Uncap LS p1, p1000 to add 292 μL distilled autoclaved H2O.</LI><LI>A little bit of water would fall out of the pipet tip. Tip was not full. Still, only a very small amount missing.</LI><LI>Recap.</LI></OL></DIV><DIV class="column full_size"><P>7.7.2016</P><P>These are the sequences for the two C. Limon gene blocks that are being used for our current experiments:</P><UL><LI>Blue/White screening allowed colonies with the gene block to be isolated</LI><LI>From the first plate with the C. Limon gene block 1, 3 colonies were picked and the miniprep protocol from the lab was followed, these samples are listed as 1A, 1B, 1C from here on out.  The same goes for the second C. Limon gene, except one of the culture tubes was broken so there are only two samples 2B and 2C.</LI><LI>Nano Dropped each of the following samples to determine the DNA concentration: 1A, 1B, 1C, 2B, 2C. Also, performed digestion on the concentrated plasmid using the NEB protocol with adjusted amounts of each material.</LI></UL><P>The nanodrop results are listed below:</P><UL><LI>1A: 325.5 ng/uL</LI><LI>1B: 325.9 ng/uL</LI><LI>1C: 381.6 ng/uL</LI><LI>2B: 243.9 ng/uL</LI><LI>2C: 332.7 ng/uL</LI></UL><P>The 260/280 ratio for each measurement was around 1.83, which means that the DNA is pretty pure.
After nano dropping each of the plasmids we used the following measurements for the digestion protocol:</P><P>For amount of DNA:</P><UL><LI>1A: 3.1uL</LI><LI>1B:2.1 uL</LI><LI>1C: 2.6 uL</LI><LI>2B: 4.1uL</LI><LI>2C: 3.0uL</LI></UL><P>10X Cutsmart:</P><UL><LI>5uL, making it 1X concentration</LI></UL><P>EcoR1:</P><UL><LI>1uL</LI></UL><P>Sterile DI Water:</P><UL><LI>1A: 40.9uL</LI><LI>1B: 40.9 uL</LI><LI>1C: 41.4 uL</LI><LI>2B: 39.9uL</LI><LI>2C: 41.0 uL</LI></UL><P>All of the reaction volumes came out to 50uL and were placed in the thermocycler following this program: 37°C for 1 hour and 30 minutes, then it is going to be heat inactivated at 65°C for 30 minutes, then held at 4°C until tomorrow.</P><P>7.10.16</P><P>Took digested plasmid from 7.8.16 and ran it through a gel with 1kb+ ladder to confirm that the plasmid had been cut in the correct place. Results are shown below, the order of the wells are:</P><UL><LI>LADDER</LI><LI>1A</LI><LI>1B</LI><LI>2A</LI><LI>2B</LI><LI>2C</LI></UL></DIV><DIV class="column full_size"><UL><LI>There seem to be bands at around 4,000 base pairs, which matches up with the StrataClone Blunt PCR Cloning Vector pSC-B-amp/kan containing 4.3 kb</LI><LI>There are also bands around 1298 for 1B only, which is the correct number of bp’s for C.Limon 1, and there are also two bands for 2B and 2C at around 817 base pairs, confirming that the gene blocks were cut a out of the Strataclone vector at the correct spot</LI></UL><P>7.12.16</P><P>Primers were ordered from IDT today, there are corresponding colors that can be matched with the genes listed above, also we went ahead and ordered the primers for the two C. Sten gene blocks listed below:</P><UL><LI>Limonene Synthase Forward Primer (6.20.16)</LI><LI>5’- ATGCGCGTTGCGGCAGTCGACCGGCAAAAGG-3’ ( GC%= 64.5%, Tm=77.8°C, length:31 bp )</LI><LI>Limonene Synthase Reverse Primer (6.20.16)</LI><LI>5’- GATTTTTTTTGTCGACTTTTTTGCGGCCGCT -3’</LI><LI>3’- AGCGGCCGCAAAAAAGTCGACAAAAAAAATC-5’(GC%=41.9%, Tm=70.2°C, length:31bp)</LI><LI>C.Limon 1-Reverse Primer</LI><LI>5’- GATCTTCACCGACGCCGTCGAGAGGTGGG -3’</LI><LI>3’-CCCACCTCTCGACGGCGTCGGTGAAGATC-5’ (GC%=65.5%, Tm=73.8°C, length: 29 bp) </LI><LI>C.Limon 2-Forward Primer</LI><LI>5’-CCGTCGAGAGGTGGGACATAAACTACGCCC-3’ (GC%=60%, Tm=72.2°C, length:30 bp)</LI><LI>C. Sten 1- Reverse Primer</LI><LI>5’-GATGGTGGAGGAATACCTGGTTCGTCGAGAA -3’</LI><LI>3’-TTCTCGACGAACCAGGTATTCCTCCACCATC-5’ (GC%=51.6%, Tm=70.6°C, length:31bp) </LI><LI>C.Sten 2- Forward Primer</LI><LI>5’-GGGCATGGTCCCAAGAAGGCAGCAC-3’ (GC%=64%, Tm=72.1°C, length: 25bp)</LI><LI>C.Sten gene block 1  (1198 bp)</LI><LI>ATGCGCGTTGCGGCAGTCGACCGGCAAAAGGCGAATTATGTGTAGGCATTAGGTTAAGCCTTCTTTTCATTTTTACGGCAATCAGCGAGCTTTTTTTGGCCGCTGAAAGGGTCTGAAAGCGAAAAGTATTTAAACCCCAAGTGGCCAGATAGGTATGACAACACTTAGTAGGGGCTAAAGCCCCAACAACCCAAGGAGGTGTTGTATGGCCCTCAAGATGACCAGCGCCGTCATGCAGATGGCGATACCGACCAAGCTCGCCAACTTCGTCAACAATAGCGACACCCACAAGCAAAGCCTGAAACTCCTGAGGAACGTTTCCACGATAAGCACCTCCGCGGCCGCGGCCACGCCGAGGCATAGACTCCCAGTCTGCTGCAGCGCGTCCAGCTCCAGCTCCAGCCAGCTGCCGACbCATAGAGAGGAGATCCGGCAATTACAAGCCAAGCAGGTGGGACGTTGATTTCATGCAATCCCTCAACAGCGACTACCAGGAGGAAAGACACAGGACGAAAGCCTCCGAGCTGATAACCCAAGTCAAGAATCTCCTGGAAAAAGAGACGAGCGATGACCCGATAAGACAGCTCGAACTGATCGATGACCTCCAAAGGCTGGGCCTCTCCGATCATTTCGAGCACGAATTTAAGGAGGTTCTGAACAGCATATACCTCGACAATAAATATTACAACATCAATATAATGAAGGAAACCACGTCCAGCAGAGATCTGTATTCCACCGCGCTCGCCTTCAGGCTGCTCAGAGAGCATGGCTTCCAGGTCGCGCAAGAAGTTTTTGACTGCTTCAAAAACGAGGAAGGCGAGTTTAAGGCCAGCCTGTCCGATGACCCAAGGGGCCTCCTGCAGCTCTACGAAGCGAGCTTCCTGTTTAAAGAGGGCGAAAATACGCTCGAGATCGCCAGAGAATTCGCGACCAAGCTGCTCCAAGAGAAAGTCAACTCCAGCGATGAAATAGACGATAATCTGCTCTCCAGCATCAGGTATTCCCTGGAGATACCGACGTACTGGAGCGTTATCAGACCAAACGTCTCCGTTTGGATAGACGCCTATAGGAAGAGACCGGATATGAATCCAGTCGTTCTCGAACTGGCGATCCTCGACGCCAACATAATGCAGGCGCAACTGCAGCAAGAGCTCAAAGAAGCCCTGGGATGGTGGAGGAATACCTGGTTCGTCGAGAA </LI><LI>C.Sten gene block 2  (897)</LI><LI>GGGCATGGTCCCAAGAAGGCAGCACAAAACCGCCAGACAACTCATGGCGAAGGTTATCGCCCTGATAACGGTCATGGACGATATCTATGACGTTTACGGAACCCTCGAGGAACTGGAGCTCTTCACGGATGCGTTTAGGAGATGGGACGTCAGCTCCATAGATCATCTGCCGACCTATATGCAGCTCTGCTTCCTGAGCATCAACAATTTTGTTGTTGACACGGCCTACAACATACTCAAAGAAACCGGCGTTAATGTCACGACCTATCTGGAGAAGTCCTGGGTTGATCAAGCGGAAAACTACCTCATGGAGAGCAAATGGTTCTACTCCGGACACAAGCCAAGCCTGGACGAATATCTCGAGAATTCCTGGATCAGCGTCTCCGGCCCGTGCGTTCTGACGCATGAATTCTTTGGAGTCACCGATAGCCTCGCCAAAGACACGCTGGATTCCCTCTACGAGTATCACGACATAGTTAGGTGGAGCTCCTACCTGCTCAGACTGGCGGATGACCTCGGCACCAGCGTCGAAGAGGTTTCCAGGGGAGATGTCCCAAAGAGCATCCAGTGCTATATGAACGACAATAACGCCTCCGAAGAGGAAGCGAGAGAGCATGTTAAAGGCCTGATAAGGGTCATGTGGAAGAAAATGAATGCCGAAAGAGTTAGCGAGGATTCCCCGTTCTGCAAGGACTTTATCAGGTGCTGCGAAGATCTCGGAAGAATGGCGCAATTCATGTACCACTATGGCGACGGACACGGCACGCAGCATGCCAAAATACACCAACAGATCACCGATTGCCTGTTCCAACCATTTGCCTACCCATACGACGTTCCGGACTACGCACACCATCACCATCACCATTGATTTTTTTTGTCGACTTTTTTGCGGCCGCT</LI><LI>C. Limon gene block 1(A,B,C), bp=1298</LI><LI>ATGCGCGTTGCGGCAGTCGACCGGCAAAAGGCGAATTATGTGTAGGCATTAGGTTAAGCCTTCTTTTCATTTTTACGGCAATCAGCGAGCTTTTTTTGGCCGCTGAAAGGGTCTGAAAGCGAAAAGTATTTAAACCCCAAGTGGCCAGATAGGTATGACAACACTTAGTAGGGGCTAAAGCCCCAACAACCCAAGGAGGTGTTGTATGAGCAGCTGCATAAACCCGAGCACCCTCGTCACCAGCGTCAACGCCTTCAAGTGCCTCCCGCTCGCCACCAACAAGGCCGCCATAAGGATAATGGCCAAGTACAAGCCGGTCCAGTGCCTCATAAGCGCCAAGTACGACAACCTCACCGTCGACAGGAGGAGCGCCAACTACCAGCCGAGCATATGGGACCACGACTTCCTCCAGAGCCTCAACAGCAACTACACCGACGAGGCCTACAAGAGGAGGGCCGAGGAGCTCAGGGGCAAGGTCAAGATAGCCATAAAGGACGTCATAGAGCCGCTCGACCAGCTCGAGCTCATAGACAACCTCCAGAGGCTCGGCCTCGCCCACAGGTTCGAGACCGAGATAAGGAACATCCTCAACAATATATACAACAATAACAAGGACTACAACTGGAGGAAGGAGAATCTCTATGCCACCAGCCTGGAGTTCAGGCTCCTGAGGCAGCACGGCTACCCGGTCAGCCAGGAGGTTTTCAACGGCTTCAAGGACGATCAGGGCGGATTCATATGCGACGATTTCAAGGGCATACTCAGCCTGCACGAGGCCAGCTACTATTCCCTCGAGGGCGAAAGCATAATGGAGGAAGCCTGGCAGTTCACCAGCAAGCACCTCAAAGAGGTCATGATAAGCAAGAACATGGAGGAAGACGTTTTCGTCGCCGAGCAGGCGAAGAGGGCCCTCGAGCTGCCGCTCCACTGGAAGGTCCCGATGCTCGAGGCCAGGTGGTTCATACACATCTACGAGAGGCGCGAAGACAAGAACCACCTCCTGCTCGAGCTGGCCAAAATGGAGTTCAACACCCTCCAGGCGATATACCAGGAGGAACTCAAGGAGATCAGCGGCTGGTGGAAAGACACCGGCCTGGGAGAGAAGCTCAGCTTCGCCAGGAACCGCCTCGTCGCGAGCTTCCTGTGGTCCATGGGCATAGCCTTCGAGCCGCAGTTCGCGTACTGCAGGCGCGTCCTCACCATAAGCATCGCCCTGATAACCGTTATCGACGATATATACGACGTCTATGGCACCCTCGACGAGCTGGAGATCTTCACCGACGCCGTCGAGAGGTGGG</LI><LI>C. Limon gene block 2(B,C), bp=817</LI><LI>CCGTCGAGAGGTGGGACATAAACTACGCCCTCAAGCACCTGCCGGGCTACATGAAGATGTGCTTCCTCGCGCTGTACAACTTCGTCAATGAGTTCGCCTACTATGTTCTCAAGCAGCAAGACTTCGATCTGCTCCTGAGCATAAAGAACGCCTGGCTCGGCCTGATACAGGCGTACCTCGTCGAGGCCAAGTGGTACCACAGCAAATACACCCCGAAGCTCGAGGAATATCTCGAGAACGGCCTCGTCAGCATAACCGGCCCGCTCATAATCACCATAAGCTACCTGTCCGGCACCAACCCGATAATCAAGAAAGAGCTCGAGTTCCTGGAAAGCAACCCGGACATAGTCCACTGGAGCTCCAAGATATTCAGGCTCCAGGACGACCTCGGCACCAGCAGCGACGAGATACAGAGGGGCGACGTCCCGAAGAGCATACAGTGCTACATGCACGAGACCGGCGCCAGCGAGGAGGTCGCCAGGCAGCACATAAAGGACATGATGAGGCAGATGTGGAAGAAGGTCAACGCCTACACCGCCGACAAGGACAGCCCGCTCACCGGCACCACCACCGAGTTCCTCCTCAACCTCGTCAGGATGAGCCACTTCATGTACCTCCACGGCGACGGCCACGGCGTCCAGAACCAGGAGACCATAGACGTCGGCTTCACCCTCCTCTTCCAGCCGATACCGCTCGAGGACAAGCACATGGCCTTCACCGCCAGCCCGGGCACCAAGGGCTACCCATACGACGTTCCGGACTACGCACACCATCACCATCACCATTGATTTTTTTTGTCGACTTTTTTGCGGCCGCT</LI></UL><P>7.14.16</P><UL><LI>Performed mini-prep on the prepared cultures from the white colonies on blunt cloning plate for: 1B, 2B, 2C (all of the plasmids that had bands on the gel)</LI><LI>Received all four primers today and resuspended each so that the concentration was 100uM</LI><LI>We are going to perform PCR with the phusion enzyme to amplify the C. Sten g-blocks before we try Blunt Cloning</LI><LI>For all four of the primers, we created a 10uM concentration by diluting 1ul of primer with 9ul of sterile diH2O</LI></UL><P>Component: 50μl reaction</P><P>Molecular grade H2O: Added first. 32.5μL </P><P>5x GC Buffer: 10μL</P><P>10 mM dNTPs: 1.0μL</P><P>L. Synthase forward (10μM): 2.5μL</P><P>C. Sten 1 - Reverse (10μM): 2.5μL</P><P>C. Sten g-block 1: 1.0μL </P><P>Tom’s Phusion DNA polymerase: Added last. 0.5μL</P><P>Component: 50μl reaction</P><P>Molecular grade H2O: Added first. 32.5μL </P><P>5x GC Buffer: 10μL</P><P>10 mM dNTPs: 1.0μL</P><P>L. synthase reverse (10μM): 2.5μL</P><P>Sten 2 - Forward (10μM): 2.5μL</P><P>Template DNA: 1.0μL </P><P>Tom’s Phusion DNA polymerase: Added last. 0.5μL</P><UL><LI>Ran the PCR reaction in the with Thermocycler Program as follows, for both of the C.Sten g-blocks</LI></UL><P>7.15.16</P><UL><LI>Sent in mini-prepped tubes containing Blunt cloning plasmid and C. Limon g-blocks  to quintarabio for sequencing results using the T3 and T7 primers that correspond with the blunt cloning kit primer binding sites, the concentrations for each of the g-blocks is listed below</LI><UL><LI>1B-264.0ng/ml, 260/280-1.75</LI><LI>2B-338.0ng/ml, 260/280-1.78</LI><LI>2C-304.5ng/ml, 260/280-1.70</LI></UL><LI>Ran the gel from the PCR performed on C. Sten 7.14.16 to see if the g-block was successfully amplified, results are shown below following this key: ladder, C. Sten 1, C. Sten 2, we are expecting 1198 bp’s for C.Sten 1, and 897 bp’s for C.Sten 2</LI></UL></DIV><DIV class="column full_size"><UL><LI>It seems like the C. Sten 1 was unsuccessful, so we will instead perform a two step PCR corresponding to the table below, with adjusted temperatures and extension time, also we will use more of the Phusion enzyme, since it is not commercial grade</LI><LI>C. Sten 2 seems to have bands at around 897 base pairs, which is the correct number, but there also seems to be some other bands, so we will run another gel on 7.16.16 with the C.Sten 1 Two-step PCR product and also, the 41ul of the rest of the C.Sten PCR product from 7.14.16, then we will cut it from the gel and purify it if there are bands at around 1198 base pairs (did not do this)</LI><LI>C. Sten 1, 2-step PCR</LI></UL><P>7.16.16</P><UL><LI>Performed PCR on C. Sten 1 again, since gel did not have correct band</LI><LI>Ran a gel with the 41ul of the rest of PCR product from C.Sten 2 and ran PCR product from today, ladder, 6 wells filled with C.Sten 1, 41ul of C.Sten 2 in last well</LI><LI>It was a failure so we are going to reorder the limonene synthase forward primer, because the melting temp. was so high (77.8°C)</LI><LI>This is the new primer we ordered: </LI><LI>ATG CGC GTT GCG GCA GTC GAC (GC=66.7%, melting temp=71.3°C, bp=21)</LI><LI>We will perform the two-step PCR again with both gene blocks, once the new forward primer arrives</LI></UL><P>7.19.16</P><P>Results from quintarabio for 1B, 2B, 2C were received today, results are as follows:</P><UL><LI>1B non conclusive, T3 and T7 sequences did not align</LI><LI>2B seemed to have worked, picture of aligned sequences is shown below</LI><LI>2C had some alignment, but a few couple of mismatched base pairs that proved to have non silent mutations when put into the Expasy program</LI><LI>I aligned the second part of the g-block with a spliced sequence I created from the T3 sequence and the RC of the T7 sequence, this is the spliced sequence I used, the yellow highlighted area was the sequence chosen that splits the two gene blocks apart:</LI><LI>Strataclone T3 sequence through PCR product,green is T3 primer binding site  yellow is discrepancy in the spliced gene and the starting sequence in the strataclone vector</LI><LI>CAATTAACCCTCACTAAAGGGAACAAAAGCTGGGTACCGGGCCCCCCCTCGAGGTCGACGGTATCGATAAGCTTGATATCCACTGTGGAATTCGCCCTT </LI><LI>Strataclone PCR product through T7 Primer binding site, green is T7 primer binding site, the sequence still has two EcoRI cut sites</LI><LI>AAGGGCGAATTCCACATTGGTCGCTGCAGCCCGGGGGATCCACTAGTTCTAGAGCGGCCGCACCGCGGGAGCTCCAATTCGCCCTATAGTGAGTCGTATTAC 
GGCATACTGGTACCGGGCCCCCCCTCGAGTCGACGGTATCGATAAGCTTGATATCCCTGTGGAATTCGCCCTTCCGTCGAGAGGTGGGACATAAACTACGCCCTCAAGCACCTGCCGGGCTACATGAAGATGTGCTTCCTCGCGCTGTACAACTTCGTCAATGAGTTCGCCTACTATGTTCTCAAGCAGCAAGACTTCGATCTGCTCCTGAGCATAAAGAACGCCTGGCTCGGCCTGATACAGGCGTACCTCGTCGAGGCCAAGTGGTACCACAGCAAATACACCCCGAAGCTCGAGGAATATCTCGAGAACGGCCTCGTCAGCATAACCGGCCCGCTCATAATCACCATAAGCTACCTGTCCGGCACCAACCCGATAATCAAGAAAGAGCTCGAGTTCCTGGAAAGCAACCCGGACATAGTCCACTGGAGCTCCAAGATATTCAGGCTCCAGGACGACCTCGGCACCAGCAGCGACGAGATACAGAGGGGCGACGTCCCGAAGAGCATACAGTGCTACATGCACGAGACCGGCGCCAGCGAGGAGGTCGCCAGGCAGCACATAAAGGACATGATGAGGCAGATGTGGAAGAAGGTCAACGCCTACACCGCCGACAAGGACAGCCCGCTCACCGGCACCACCACCGAGTTCCTCCTCAACCTCGTCAGGATGAGCCACTTCATGTACCTCCACGGCGACGGCCACGGCGTCCAGAACCAGGAGACCATAGACGTCGGCTTCACCCTCCTCTTCCAGCCGATACCGCTCGAGGACAAGCACATGGCCTTCACCGCCAGCCCGGGCACCAAGGGCTACCCATACGACGTTCCGGACTACGCACACCATCACCATCACCATTGATTTTTTTTGTCGACTTTTTTGCGGCCGCTAAGGGCGAATTCCACATTGGTCGCTGCAGCCCGGGGGATCCACTGTCTAGAGCGGCCGCACGCGAGTACAAGCC</LI><P>7.20.16</P><P>Casey &amp; Alec miniprep cultures from 7/19 streak plate. Cells from strataclone kit w/ limon inserts, these samples were sent for sequencing today</P><UL><LI>gBlock 1 (limon)</LI><LI>samples A &amp; B</LI><LI>gBlock 2 (limon)</LI><LI>samples A &amp; B</LI><LI>Fluorimetry to gather concentrations for sequencing</LI><LI>1A,Forgot to read error…-193.1 ng/uL</LI><LI>1B, Error: -7, 226 ng/uL</LI><LI>2B, Error: -10, 225 ng/uL</LI><LI>2A, Error: -2, 178.2 ng/uL</LI></UL><P>7.21.16</P><P>Sequencing results came in for miniprep done on 7.20 and I am thinking that the first gene block was inserted in a reverse orientation into the plasmid.</P><P>Same exact thing happened to the sequencing results from 7/15 for 1B, below is the C.Limon first part of the g-block aligned with the RC of the 1A spliced seq.</P><P>7.21.16</P><P>PCR with sten 1 &amp; 2; This time, using the new forward primer.</P><P>I’m thinking we should nanodrop to check for amplification instead of running a gel as we can save more of our sample if the result is good. Also, the gel won’t tell us a concentration, and we already know the gene block is in the reaction mix. We need to check the concentration of DNA. </P><P>After confirming that the gene blocks have been amplified, we should try the PCR assembly protocol or infusion cloning. We need to put the gene blocks together. </P><P>7.22.16</P><P>Nanodrop (both gave error warnings)
</P><P>gBlock 1</P><UL><LI>25 ng/uL</LI><LI>30 ng/uL</LI></UL><P>gBlock 2</P><UL><LI>21 ng/uL</LI></UL><P>Running gel</P><UL><LI>Poured last night</LI><LI>110 volts</LI><LI>Set 45 min</LI><LI>Well 1 = C.sten 1 PCR product</LI><LI>Well 2 = C.sten 2 PCR product</LI></UL><P>7.24.16</P><UL><LI>Re-did the PCR on C.Sten 1 and 2 using this mixture protocol:</LI><LI>I had to dilute all four of the primers, 1:10 and I also dilute the 25mM dntps to 10mM 2:5</LI></UL><P>7.25.16</P><UL><LI>Ran a gel with the PCR products from 7.24, pic is shown below</LI><LI>1A and 2B off of Madeline’s master plate, ladder c.sten 1 and c.sten 2</LI><LI>Confirmed with Sharon that the orientation of the gene block in the strataclone vector does not matter, so in this case, we can move forward with experimentation with the first gene block (miniprep, phusion PCR)</LI></UL><P>7.29.16</P><P>Redid the PCR with a 66-70°C gradient, and started a culture with 1A and 2B from madeline’s master plate</P><P>Diluted the 4 C. sten primers to 10mM and dNTPs to 10mM</P><P>Fixed gradient to do PCR with C. sten 1 and 2 TM + 3 and -3 (4 total)</P><P>C. Sten 1</P><P>C. Sten 2</P><P>8.8.16</P><P>Running gel with PCR products of 7.29.16</P><DIV class="column full_size"><P>8.9.16</P><P>Performed the gel purification protocol from the lab, including the following changes:</P><P><UL><LI>Only did 1 membrane wash </LI><LI>Did not pre heat the elution buffer</LI><LI>Did the final elution with 15uL of the 10mM tris-HCL ph 8.8</LI></UL></P><P>Both parts of the gene block will be nanodropped tomorrow to see how high the concentration of DNA is</P><P>8.10.16</P><P>PCR with minipreped stataclone vector + insert from culture started on 7.29.16</P><P>We have 4 samples, 1A, 1B, 2B, 2C </P><UL><LI>Two different annealing temperatures for each sample. 68 &amp; 72</LI><LI>So we will have 8 PCR reactions.</LI></UL><P>Ran a gel with the completed C.Limon PCR products listed above from 8.9.16, results are shown below: ladder, next three were sharon’s stuff, 1A-68, 1B-68, 2B-68, 2C-68, 1A-72, 1B-72, 2B-72, 2C-72</P><P>Nanodropped C. Sten 1 &amp; 2 gel purification products results are shown below:</P><UL><LI>C. Sten 1 green tube- 9.9 ng/ul, 260/280- 2.07</LI><LI>C. Sten 1 blue tube- 16.2 ng/ul, 260/280-1.92</LI><LI>C. Sten 2 orange tube- 14.1 ng/ul, 260/280-1.84</LI><LI>C. Sten 2 yellow tube- 17.4 ng/ul, 260/280-1.72</LI></UL><P>8.12.2016</P><P>Fluorimetry on C. limon gel purification</P><P>calibration : 99.76</P></DIV><DIV class="column half_size"><P>Concentration(ng/uL)</P><UL><LI>1A-68: 18.9k	</LI><LI>1A-72: 76.58</LI><LI>1B-68: 53.96</LI><LI>1B-72: 33.34</LI><LI>2B-68: 27.93</LI><LI>2B-72: 69.96</LI><LI>2C-68: 76.63</LI><LI>2C-72 : 13.27</LI></UL></DIV><DIV class="column half_size"><P>Error</P><UL><LI>-1.172</LI><LI>6.4</LI><LI>-2.9</LI><LI>-2.472</LI><LI>-5.44</LI><LI>-10.76</LI><LI>?</LI><LI>-11.73</LI></UL></DIV><DIV class="column full_size"><P>8.13.16</P><P>We tried infusion cloning with C.sten 1 and 2 g-blocks, these were the ones that were not in the strataclone vector, so after PCR amplification and running a gel to confirm amplification, we attempted to just directly clone both g-blocks into the pLC71 vector</P><P>We used a 1:1:1 molar ratio for 1st g-block:2nd g-block:pLC71 and the values for each are shown below, keep in mind we had two PCR products for each g-block, so they are separated by eppendorf tube color, in this case we only used C.sten 1 blue and C.sten 2 yellow</P><UL><LI>C.Sten 1 blue: 1.36 uL</LI><LI>C. Sten 2 yellow: 1.02uL</LI><LI>pLC71:0.88uL, dilution- 2uL pLC71 + 2uL sterile H2O, this way we could use 1.8uL instead of 0.88 which is hard to achieve with a pipette</LI><LI>3.8uL of water </LI><LI>We did a negative control that contained everything listed above, except for the infusion mix </LI></UL><P>After following the Infusion protocol, we followed the Interlab transformation protocol, except we added 2 uL of the plasmid instead of 1uL, also when we plated after the outgrowth period, we use 50uL on one plate, and the rest on another plate, ~200uL</P><P>9.2.16</P><P>Nanodropping: C. Limon 1B &amp; 2B in the strataclone vector</P><P><UL><LI>1B: 361.4 ng/μL	260/280: 1.84</LI><LI>2B: 368.9 ng/μL	260/280: 1.84</LI></UL></P><P>PCR:</P><UL><LI>M1V1 = M2V2</LI><LI>V1B = (2μL)(361.4 ng/μL) / (10ng/μL) = 72.28 μL</LI><LI>V1B = (2μL)(361.4ng/μL) / (10ng/μL) = 73.78 μL</LI><LI>So, to dilute the plasmids to 10 ng/μL..</LI><LI>1B: 2μL stock + 70.3μL dH2O</LI><LI>1B: 2μL stock + 71.8μL dH2O</LI></UL><P>Primers:</P><UL><LI>1B: Limonene Forward + C.limon1R</LI><LI>2B: C.limon2F + LSp2</LI></UL><P>We have 100μM primers, we want 10μM.</P><UL><LI>M1V1 = M2V2</LI><LI>V = (2μL)(100μM) / (10μM) = 20μL</LI><LI>So to dilute the primers…</LI><LI>2μL stock + 18μL dH2O</LI></UL><P>9.3.16</P><P>PCR failed. New idea; add DSMO (5% volume). Also try raising the annealing temperature. </P><P>Brett Burkart, an employee of the Santangelo lab, gave Casey this protocol for purifying PCR products.</P><P>PB buffer</P><P>Add 5x the volume of the combined PCR reactions. </P><UL><LI>Combined PCR reactions: 177µL</LI><LI>PB Buffer: 885µL</LI><LI>3M NaOAc pH 5.2</LI><LI>Add 1/100x the volume of the PCR reaction + PB buffer</LI><LI>Spin through column (may take two runs).</LI><LI>Discard flow through. </LI><LI>Load column with 750µL ethanol, spin.</LI><LI>Discard ethanol, spin 2 minutes to dry.</LI><LI>Label 1.7mL eppe.</LI><LI>Load 25µL 10mM tris HCl pH 8.5</LI><LI>Spin 2 mins. collect in labeled eppe.</LI><LI>Load 25µL 10mM tris HCl pH 8.5</LI><LI>Spin 2 mins. collect in same labeled eppe.</LI></UL><P>9.20.16</P><P>Dr. Peebles discovered the there was no overlap between our insert and vector. The infusion cloning could not have worked.</P><P>New primers:</P><P>Forward:</P><UL><LI>GGC CGC AAA AAA GTC GAA TGC GCG TTG CGG CAG TCG 
melt temp: 77.7 ºC</LI></UL><P>homologous region:</P><UL><LI>A TGC GCG TTG CGG CAG TCG 
melt temp: 70.6 ºC</LI></UL><P>Reverse:</P><UL><LI>CAA AAT GCG CGT TGC GGC AGA GCG GCC GCA AAA AAG TCG AC
melt temp: 78.4</LI></UL><P>homologous region:</P><UL><LI>AGCGGCCGCAAAAAAGTCGAC
melt temp: 68.3 ºC</LI></UL><P>9.22.16</P><P>Picked up new primers &amp; resuspended.</P><P>9.26.16</P><P>PCR with new primers, from strataclone vector.</P><P>7.25.17</P><P>Did a gDNA prep with non-confirmed strains:</P><UL><LI>pHF10: 1L, 4N, 2M, 3L, 2O, 1C, 4M, 4K, 1M, 2L</LI><LI>pLC71: 4E, 2E, 2A, 9E, 1C, 4P, 4D</LI><LI>Positive control: pHF10 3S</LI><LI>Negative control: pLC71</LI></UL></DIV><DIV class="column full_size"><P>05-16: Transformed pHF10 and pLC71 into TS559</P><LI>            Courtney </LI><P>5-311: Colonies Alive</P><LI> Courtney</LI><P>05-24: Followed spot plate protocol with pHF10</P><LI> Courtney, Kayla, Frankie, Cooper, Neil</LI><P>05-29: Transferred colonies to 40 individual 5uL flasks using lab stock ASW media and ASW Additives, Sulfur was not added. </P><LI> Carlos</LI><P>5-30: Sulfur Added to cultures to 20 of those cultures</P><LI> Carlos</LI><P>5-31: No growth on pHF10(1)</P><LI> Frankie, Kayla</LI><P>6-01: Four pHF10(1) cultures survived. Performed a gPrep on these cultures. Made new 25mM lovastatin stock. Used 6-24 pHF10(2) spot plates to make 20 5uL serum bottle cultures via ASW Additives Protocol. </P><LI> Frankie, Neil, Cooper, Kayla</LI><P>6-02: Ran a PCR and gel on the gprepped DNA samples from 6-01.</P><LI> Kayla, Cooper, Frankie, Melissa</LI><P>6-03:Serum bottles checked at 11:10am</P><LI> Melissa</LI><P>6-04: Serum bottles checked at 1:30</P><LI> Carlos</LI><P>6-05: Primers for 6-02 gel were wrong, redid gel with 6-01 DNA. Surviving pLC71(2) colonies removed from incubator. Made ASW-yt media. Added additives to 2 100 mL bottles of ASW-yt and inoculated from bottle 2C from pLC71(1).</P><LI> Robyn, Frankie Kayla, Melissa, Cooper</LI><P>6-06: Some oxidized sulfur, but not enough in 6-05 strain. Picked colonies from old PLC71 streak plate into 10 5 mL bottles</P><LI> Kayla, Carlos, Cooper, Neil</LI><P>6-07: No growth on pLC71</P><LI> Kayla, Neil, Carlos, Melissa</LI><P>6-08: Started a 100 mL culture of TS559 for transformation. Made minimal ASW plates.</P><LI> Frankie, Kayla</LI><P>6-09: Performed transformation protocol with pLC71 plasmid</P><LI> Frankie, Kayla</LI><P>6-14: Made rich ASW plates. Spotted 10 colonies per plate onto the rich ASW media</P><LI> Kayla, Courtney</LI><P>6-17: Checked plates, but needed more time</P><LI> 	Qianhui, Carlos</LI><P>6-19: Took 5 5mL serum bottles per plate, and placed in incubator at 11:00 am. </P><LI> Kayla</LI><P>6-20: At 2:30, 9 cultures had grown.</P><LI> Kayla</LI><P>6-21: Performed a gPREP of 3pLC71 cultures and 1 pHF10 culture. Performed pCR protocol.</P><LI> Cooper, Carlos, Jake, Robyn, Qianhui, Kayla</LI><P>6-23: Made a gel and and ran the gPrepped cultures</P><LI> Kayla, Jack</LI><P>6-26: Started 2 100 mL bottles from pLC71 cultures, 2 bottles from pHF10(1), and two bottles of TS559.</P><LI> Courtney, Skylar</LI><P>6-27: Removed cultures at 7:15 am. Performed sonication protocol. Stored protein lysate in fridge.</P><LI> Kayla</LI><P>7-6: Performed a Bradford Assay on protein lysate</P><LI> Neil, Kayla</LI><P>7-12: Performed a western blot analysis according to protocol. Blott was not clear, and no reasonable results obtained.</P><LI> Melissa, Carlos,Neil, Frankie, Robyn, Kayla</LI><P>7-17: Made ASW media.</P><LI> Melissa, Kayla</LI><P>7-19 No grotwth on current cultures, started 3 each serum bottles of pLC71 and 3 each of 4 pHF10 strains</P><LI> Kayla, Melissa, Frankie</LI><P>7-25: Performed a Gprep on pLC71 and pHF10 samples </P><LI> Kayla, Madeline</LI><P>8-8: Inoculated 3, 100 mL ASW bottles with 1 mL cultures each (1 TS559, 1 pLC71, 1 pHF10)</P><LI> Neil, Carlos</LI><P>8-9: No growth. Used a bottle of laboratory stock media and inoculated with 9 pHF10 strains, 10 pLC71 strains.</P><LI> Kayla </LI><P>8-10: 11:00 am No growth</P><LI> Kayla</LI><P>8-16: Most cultures showed signs of growth. </P><LI> Kayla</LI><P>9-14:Inoculated 4, 100 mL bottles of media ( 2 pLC71, 2 pHF10)</P><LI> Kayla</LI><P>9-16: Made new ASW media.   
	<LI> Neil, Kayla</LI></P><P>8-17: Inoculated 4 100 mL bottles of ASW media with 1 TS559, 2 pHF10, and 1 pLC71.</P><LI> Kayla</LI><P>8-19: No growth on cultures. Left in incubator. Made new, smaller cultures using 1 bottle ASW YT media according to protocol (3 bottles pLC71, 12 bottles pHF10)</P><LI> Kayla</LI><P>9-26: No growth in any culture. </P><LI> Kayla</LI></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>