Team:CSU Fort Collins/NoteBook/July
MENU ▤
CSU iGEM
HOME
TEAM
PROJECT
Description
Interlab Study
Experiments
Protocols
Results
PARTS
Parts
Basic Parts
SAFETY
ATTRIBUTIONS
HUMAN PRACTICES
Silver
7.7.2016
These are the sequences for the two C. Limon gene blocks that are being used for our current experiments: Blue/White screening allowed colonies with the gene block to be isolated From the first plate with the C. Limon gene block 1, 3 colonies were picked and the miniprep protocol from the lab was followed, these samples are listed as 1A, 1B, 1C from here on out.  The same goes for the second C. Limon gene, except one of the culture tubes was broken so there are only two samples 2B and 2C. Nano Dropped each of the following samples to determine the DNA concentration: 1A, 1B, 1C, 2B, 2C. Also, performed digestion on the concentrated plasmid using the NEB protocol with adjusted amounts of each material The nanodrop results are listed below: 1A: 325.5 ng/uL 1B: 325.9 ng/uL 1C: 381.6 ng/uL 2B: 243.9 ng/uL 2C: 332.7 ng/uL The 260/280 ratio for each measurement was around 1.83, which means that the DNA is pretty pure. After nano dropping each of the plasmids we used the following measurements for the digestion protocol: For amount of DNA: 1A: 3.1uL 1B:2.1 uL 1C: 2.6 uL 2B: 4.1uL 2C: 3.0uL 10X Cutsmart: 5uL, making it 1X concentration EcoR1: 1uL Sterile DI Water: 1A: 40.9uL 1B: 40.9 uL 1C: 41.4 uL 2B: 39.9uL 2C: 41.0 uL All of the reaction volumes came out to 50uL and were placed in the thermocycler following this program: 37°C for 1 hour and 30 minutes, then it is going to be heat inactivated at 65°C for 30 minutes, then held at 4°C until tomorrow                  7.10.16 Took digested plasmid from 7.8.16 and ran it through a gel with 1kb+ ladder to confirm that the plasmid had been cut in the correct place. Results are shown below, the order of the wells are: LADDER, 1A, 1B, 2A, 2B, 2C.    There seem to be bands at around 4,000 base pairs, which matches up with the StrataClone Blunt PCR Cloning Vector pSC-B-amp/kan containing 4.3 kb There are also bands around 1298 for 1B only, which is the correct number of bp’s for C.Limon 1, and there are also two bands for 2B and 2C at around 817 base pairs, confirming that the gene blocks were cut a out of the Strataclone vector at the correct spot  7.12.16 Primers were ordered from IDT today, there are corresponding colors that can be matched with the genes listed above, also we went ahead and ordered the primers for the two C. Sten gene blocks listed below Limonene Synthase Forward Primer (6.20.16) 5’- ATGCGCGTTGCGGCAGTCGACCGGCAAAAGG-3’ ( GC%= 64.5%, Tm=77.8°C, length:31 bp ) Limonene Synthase Reverse Primer (6.20.16) 5’- GATTTTTTTTGTCGACTTTTTTGCGGCCGCT -3’ 3’- AGCGGCCGCAAAAAAGTCGACAAAAAAAATC-5’(GC%=41.9%, Tm=70.2°C, length:31bp) C.Limon 1-Reverse Primer 5’- GATCTTCACCGACGCCGTCGAGAGGTGGG -3’ 3’-CCCACCTCTCGACGGCGTCGGTGAAGATC-5’ (GC%=65.5%, Tm=73.8°C, length: 29 bp)  C.Limon 2-Forward Primer 5’-CCGTCGAGAGGTGGGACATAAACTACGCCC-3’ (GC%=60%, Tm=72.2°C, length:30 bp) C. Sten 1- Reverse Primer 5’-GATGGTGGAGGAATACCTGGTTCGTCGAGAA -3’ 3’-TTCTCGACGAACCAGGTATTCCTCCACCATC-5’ (GC%=51.6%, Tm=70.6°C, length:31bp)  C.Sten 2- Forward Primer 5’-GGGCATGGTCCCAAGAAGGCAGCAC-3’ (GC%=64%, Tm=72.1°C, length: 25bp)  C.Sten gene block 1  (1198 bp) ATGCGCGTTGCGGCAGTCGACCGGCAAAAGGCGAATTATGTGTAGGCATTAGGTTAAGCCTTCTTTTCATTTTTACGGCAATCAGCGAGCTTTTTTTGGCCGCTGAAAGGGTCTGAAAGCGAAAAGTATTTAAACCCCAAGTGGCCAGATAGGTATGACAACACTTAGTAGGGGCTAAAGCCCCAACAACCCAAGGAGGTGTTGTATGGCCCTCAAGATGACCAGCGCCGTCATGCAGATGGCGATACCGACCAAGCTCGCCAACTTCGTCAACAATAGCGACACCCACAAGCAAAGCCTGAAACTCCTGAGGAACGTTTCCACGATAAGCACCTCCGCGGCCGCGGCCACGCCGAGGCATAGACTCCCAGTCTGCTGCAGCGCGTCCAGCTCCAGCTCCAGCCAGCTGCCGACbCATAGAGAGGAGATCCGGCAATTACAAGCCAAGCAGGTGGGACGTTGATTTCATGCAATCCCTCAACAGCGACTACCAGGAGGAAAGACACAGGACGAAAGCCTCCGAGCTGATAACCCAAGTCAAGAATCTCCTGGAAAAAGAGACGAGCGATGACCCGATAAGACAGCTCGAACTGATCGATGACCTCCAAAGGCTGGGCCTCTCCGATCATTTCGAGCACGAATTTAAGGAGGTTCTGAACAGCATATACCTCGACAATAAATATTACAACATCAATATAATGAAGGAAACCACGTCCAGCAGAGATCTGTATTCCACCGCGCTCGCCTTCAGGCTGCTCAGAGAGCATGGCTTCCAGGTCGCGCAAGAAGTTTTTGACTGCTTCAAAAACGAGGAAGGCGAGTTTAAGGCCAGCCTGTCCGATGACCCAAGGGGCCTCCTGCAGCTCTACGAAGCGAGCTTCCTGTTTAAAGAGGGCGAAAATACGCTCGAGATCGCCAGAGAATTCGCGACCAAGCTGCTCCAAGAGAAAGTCAACTCCAGCGATGAAATAGACGATAATCTGCTCTCCAGCATCAGGTATTCCCTGGAGATACCGACGTACTGGAGCGTTATCAGACCAAACGTCTCCGTTTGGATAGACGCCTATAGGAAGAGACCGGATATGAATCCAGTCGTTCTCGAACTGGCGATCCTCGACGCCAACATAATGCAGGCGCAACTGCAGCAAGAGCTCAAAGAAGCCCTGGGATGGTGGAGGAATACCTGGTTCGTCGAGAA  C.Sten gene block 2  (897) GGGCATGGTCCCAAGAAGGCAGCACAAAACCGCCAGACAACTCATGGCGAAGGTTATCGCCCTGATAACGGTCATGGACGATATCTATGACGTTTACGGAACCCTCGAGGAACTGGAGCTCTTCACGGATGCGTTTAGGAGATGGGACGTCAGCTCCATAGATCATCTGCCGACCTATATGCAGCTCTGCTTCCTGAGCATCAACAATTTTGTTGTTGACACGGCCTACAACATACTCAAAGAAACCGGCGTTAATGTCACGACCTATCTGGAGAAGTCCTGGGTTGATCAAGCGGAAAACTACCTCATGGAGAGCAAATGGTTCTACTCCGGACACAAGCCAAGCCTGGACGAATATCTCGAGAATTCCTGGATCAGCGTCTCCGGCCCGTGCGTTCTGACGCATGAATTCTTTGGAGTCACCGATAGCCTCGCCAAAGACACGCTGGATTCCCTCTACGAGTATCACGACATAGTTAGGTGGAGCTCCTACCTGCTCAGACTGGCGGATGACCTCGGCACCAGCGTCGAAGAGGTTTCCAGGGGAGATGTCCCAAAGAGCATCCAGTGCTATATGAACGACAATAACGCCTCCGAAGAGGAAGCGAGAGAGCATGTTAAAGGCCTGATAAGGGTCATGTGGAAGAAAATGAATGCCGAAAGAGTTAGCGAGGATTCCCCGTTCTGCAAGGACTTTATCAGGTGCTGCGAAGATCTCGGAAGAATGGCGCAATTCATGTACCACTATGGCGACGGACACGGCACGCAGCATGCCAAAATACACCAACAGATCACCGATTGCCTGTTCCAACCATTTGCCTACCCATACGACGTTCCGGACTACGCACACCATCACCATCACCATTGATTTTTTTTGTCGACTTTTTTGCGGCCGCT C. Limon gene block 1(A,B,C), bp=1298 ATGCGCGTTGCGGCAGTCGACCGGCAAAAGGCGAATTATGTGTAGGCATTAGGTTAAGCCTTCTTTTCATTTTTACGGCAATCAGCGAGCTTTTTTTGGCCGCTGAAAGGGTCTGAAAGCGAAAAGTATTTAAACCCCAAGTGGCCAGATAGGTATGACAACACTTAGTAGGGGCTAAAGCCCCAACAACCCAAGGAGGTGTTGTATGAGCAGCTGCATAAACCCGAGCACCCTCGTCACCAGCGTCAACGCCTTCAAGTGCCTCCCGCTCGCCACCAACAAGGCCGCCATAAGGATAATGGCCAAGTACAAGCCGGTCCAGTGCCTCATAAGCGCCAAGTACGACAACCTCACCGTCGACAGGAGGAGCGCCAACTACCAGCCGAGCATATGGGACCACGACTTCCTCCAGAGCCTCAACAGCAACTACACCGACGAGGCCTACAAGAGGAGGGCCGAGGAGCTCAGGGGCAAGGTCAAGATAGCCATAAAGGACGTCATAGAGCCGCTCGACCAGCTCGAGCTCATAGACAACCTCCAGAGGCTCGGCCTCGCCCACAGGTTCGAGACCGAGATAAGGAACATCCTCAACAATATATACAACAATAACAAGGACTACAACTGGAGGAAGGAGAATCTCTATGCCACCAGCCTGGAGTTCAGGCTCCTGAGGCAGCACGGCTACCCGGTCAGCCAGGAGGTTTTCAACGGCTTCAAGGACGATCAGGGCGGATTCATATGCGACGATTTCAAGGGCATACTCAGCCTGCACGAGGCCAGCTACTATTCCCTCGAGGGCGAAAGCATAATGGAGGAAGCCTGGCAGTTCACCAGCAAGCACCTCAAAGAGGTCATGATAAGCAAGAACATGGAGGAAGACGTTTTCGTCGCCGAGCAGGCGAAGAGGGCCCTCGAGCTGCCGCTCCACTGGAAGGTCCCGATGCTCGAGGCCAGGTGGTTCATACACATCTACGAGAGGCGCGAAGACAAGAACCACCTCCTGCTCGAGCTGGCCAAAATGGAGTTCAACACCCTCCAGGCGATATACCAGGAGGAACTCAAGGAGATCAGCGGCTGGTGGAAAGACACCGGCCTGGGAGAGAAGCTCAGCTTCGCCAGGAACCGCCTCGTCGCGAGCTTCCTGTGGTCCATGGGCATAGCCTTCGAGCCGCAGTTCGCGTACTGCAGGCGCGTCCTCACCATAAGCATCGCCCTGATAACCGTTATCGACGATATATACGACGTCTATGGCACCCTCGACGAGCTGGAGATCTTCACCGACGCCGTCGAGAGGTGGG  C. Limon gene block 2(B,C), bp=817 CCGTCGAGAGGTGGGACATAAACTACGCCCTCAAGCACCTGCCGGGCTACATGAAGATGTGCTTCCTCGCGCTGTACAACTTCGTCAATGAGTTCGCCTACTATGTTCTCAAGCAGCAAGACTTCGATCTGCTCCTGAGCATAAAGAACGCCTGGCTCGGCCTGATACAGGCGTACCTCGTCGAGGCCAAGTGGTACCACAGCAAATACACCCCGAAGCTCGAGGAATATCTCGAGAACGGCCTCGTCAGCATAACCGGCCCGCTCATAATCACCATAAGCTACCTGTCCGGCACCAACCCGATAATCAAGAAAGAGCTCGAGTTCCTGGAAAGCAACCCGGACATAGTCCACTGGAGCTCCAAGATATTCAGGCTCCAGGACGACCTCGGCACCAGCAGCGACGAGATACAGAGGGGCGACGTCCCGAAGAGCATACAGTGCTACATGCACGAGACCGGCGCCAGCGAGGAGGTCGCCAGGCAGCACATAAAGGACATGATGAGGCAGATGTGGAAGAAGGTCAACGCCTACACCGCCGACAAGGACAGCCCGCTCACCGGCACCACCACCGAGTTCCTCCTCAACCTCGTCAGGATGAGCCACTTCATGTACCTCCACGGCGACGGCCACGGCGTCCAGAACCAGGAGACCATAGACGTCGGCTTCACCCTCCTCTTCCAGCCGATACCGCTCGAGGACAAGCACATGGCCTTCACCGCCAGCCCGGGCACCAAGGGCTACCCATACGACGTTCCGGACTACGCACACCATCACCATCACCATTGATTTTTTTTGTCGACTTTTTTGCGGCCGCT   7.14.16  Performed mini-prep on the prepared cultures from the white colonies on blunt cloning plate for: 1B, 2B, 2C (all of the plasmids that had bands on the gel) Received all four primers today and resuspended each so that the concentration was 100uM We are going to perform PCR with the phusion enzyme to amplify the C. Sten g-blocks before we try Blunt Cloning For all four of the primers, we created a 10uM concentration by diluting 1ul of primer with 9ul of sterile diH2O Component 50μl reaction Molecular grade H2O Added first. 32.5μL  5x GC Buffer 10μL 10 mM dNTPs 1.0μL L. Synthase forward (10μM) 2.5μL C. Sten 1 - Reverse (10μM) 2.5μL C. Sten g-block 1  1.0μL  Tom’s Phusion DNA polymerase Added last. 0.5μL  Component 50μl reaction Molecular grade H2O Added first. 32.5μL  5x GC Buffer 10μL 10 mM dNTPs 1.0μL L. synthase reverse (10μM) 2.5μL Sten 2 - Forward (10μM) 2.5μL Template DNA  1.0μL  Tom’s Phusion DNA polymerase Added last. 0.5μL Ran the PCR reaction in the with Thermocycler Program as follows, for both of the C.Sten g-blocks, FROM NOW ON WE NEED TO USE EPPENDORF TEST FOLDER MAKE SURE TO LOG IN: Cycle Step 3-step protocol Cycles Temp (oC) Time Initial denaturation 98 5 min 1 Denaturation 98 10s   Annealing 67 15s 35 Extension 72 55s   Final Extension 72 5 min 1 Hold 4 hold    7.15.16 Sent in mini-prepped tubes containing Blunt cloning plasmid and C. Limon g-blocks  to quintarabio for sequencing results using the T3 and T7 primers that correspond with the blunt cloning kit primer binding sites, the concentrations for each of the g-blocks is listed below 1B-264.0ng/ml, 260/280-1.75 2B-338.0ng/ml, 260/280-1.78 2C-304.5ng/ml, 260/280-1.70 Ran the gel from the PCR performed on C. Sten 7.14.16 to see if the g-block was successfully amplified, results are shown below following this key: ladder, C. Sten 1, C. Sten 2, we are expecting 1198 bp’s for C.Sten 1, and 897 bp’s for C.Sten 2    It seems like the C. Sten 1 was unsuccessful, so we will instead perform a two step PCR corresponding to the table below, with adjusted temperatures and extension time, also we will use more of the Phusion enzyme, since it is not commercial grade C. Sten 2 seems to have bands at around 897 base pairs, which is the correct number, but there also seems to be some other bands, so we will run another gel on 7.16.16 with the C.Sten 1 Two-step PCR product and also, the 41ul of the rest of the C.Sten PCR product from 7.14.16, then we will cut it from the gel and purify it if there are bands at around 1198 base pairs (did not do this) C. Sten 1, 2-step PCR  Cycle Step 3-step protocol Cycles Temp (oC) Time Initial denaturation 98 5 min 1 Denaturation 98 10s   Extension 72 1.5 min  35 Hold 4 hold    Component 50μl reaction Molecular grade H2O Added first. 32.5μL  5x GC Buffer 10μL 10 mM dNTPs 1.0μL L. synthase reverse (10μM) 2.5μL C. Sten 2 - Forward (10μM) 2.5μL Template DNA  1.0μL  Tom’s Phusion DNA polymerase Added last 5uL   7.16.16 Performed PCR on C. Sten 1 again, since gel did not have correct band Ran a gel with the 41ul of the rest of PCR product from C.Sten 2 and ran PCR product from today, ladder, 6 wells filled with C.Sten 1, 41ul of C.Sten 2 in last well It was a failure so we are going to reorder the limonene synthase forward primer, because the melting temp. was so high (77.8°C) This is the new primer we ordered:  ATG CGC GTT GCG GCA GTC GAC (GC=66.7%, melting temp=71.3°C, bp=21) We will perform the two-step PCR again with both gene blocks, once the new forward primer arrives     7.19.16 Results from quintarabio for 1B, 2B, 2C were received today, results are as follows: 1B non conclusive, T3 and T7 sequences did not align 2B seemed to have worked, picture of aligned sequences is shown below 2C had some alignment, but a few couple of mismatched base pairs that proved to have non silent mutations when put into the Expasy program I aligned the second part of the g-block with a spliced sequence I created from the T3 sequence and the RC of the T7 sequence, this is the spliced sequence I used, the yellow highlighted area was the sequence chosen that splits the two gene blocks apart: Strataclone T3 sequence through PCR product,green is T3 primer binding site  yellow is discrepancy in the spliced gene and the starting sequence in the strataclone vector CAATTAACCCTCACTAAAGGGAACAAAAGCTGGGTACCGGGCCCCCCCTCGAGGTCGACGGTATCGATAAGCTTGATATCCACTGTGGAATTCGCCCTT  Strataclone PCR product through T7 Primer binding site, green is T7 primer binding site, the sequence still has two EcoRI cut sites    AAGGGCGAATTCCACATTGGTCGCTGCAGCCCGGGGGATCCACTAGTTCTAGAGCGGCCGCACCGCGGGAGCTCCAATTCGCCCTATAGTGAGTCGTATTAC  GGCATACTGGTACCGGGCCCCCCCTCGAGTCGACGGTATCGATAAGCTTGATATCCCTGTGGAATTCGCCCTTCCGTCGAGAGGTGGGACATAAACTACGCCCTCAAGCACCTGCCGGGCTACATGAAGATGTGCTTCCTCGCGCTGTACAACTTCGTCAATGAGTTCGCCTACTATGTTCTCAAGCAGCAAGACTTCGATCTGCTCCTGAGCATAAAGAACGCCTGGCTCGGCCTGATACAGGCGTACCTCGTCGAGGCCAAGTGGTACCACAGCAAATACACCCCGAAGCTCGAGGAATATCTCGAGAACGGCCTCGTCAGCATAACCGGCCCGCTCATAATCACCATAAGCTACCTGTCCGGCACCAACCCGATAATCAAGAAAGAGCTCGAGTTCCTGGAAAGCAACCCGGACATAGTCCACTGGAGCTCCAAGATATTCAGGCTCCAGGACGACCTCGGCACCAGCAGCGACGAGATACAGAGGGGCGACGTCCCGAAGAGCATACAGTGCTACATGCACGAGACCGGCGCCAGCGAGGAGGTCGCCAGGCAGCACATAAAGGACATGATGAGGCAGATGTGGAAGAAGGTCAACGCCTACACCGCCGACAAGGACAGCCCGCTCACCGGCACCACCACCGAGTTCCTCCTCAACCTCGTCAGGATGAGCCACTTCATGTACCTCCACGGCGACGGCCACGGCGTCCAGAACCAGGAGACCATAGACGTCGGCTTCACCCTCCTCTTCCAGCCGATACCGCTCGAGGACAAGCACATGGCCTTCACCGCCAGCCCGGGCACCAAGGGCTACCCATACGACGTTCCGGACTACGCACACCATCACCATCACCATTGATTTTTTTTGTCGACTTTTTTGCGGCCGCTAAGGGCGAATTCCACATTGGTCGCTGCAGCCCGGGGGATCCACTGTCTAGAGCGGCCGCACGCGAGTACAAGCC This is a picture of the alignment of the actual gene block sequence and this spliced sequence listed above, the top is the spliced sequence, the bottom is the g-block:         7.20.16 Casey & Alec miniprep cultures from 7/19 streak plate. Cells from strataclone kit w/ limon inserts, these samples were sent for sequencing today  gBlock 1 (limon) samples A & B gBlock 2 (limon) samples A & B Fluorimetry to gather concentrations for sequencing 1A,Forgot to read error…-193.1 ng/uL 1B, Error: -7, 226 ng/uL 2B, Error: -10, 225 ng/uL 2A, Error: -2, 178.2 ng/uL  7.21.16 -Sequencing results came in for miniprep done on 7.20 and I am thinking that the first gene block was inserted in a reverse orientation into the plasmid, below is a picture of the 1B spliced seq aligned with the C. Limon 1st part of the gene block:    Now here is a picture of the g-block aligned with the reverse complement of the spliced 1B seq from 7/20:                   Same exact thing happened to the sequencing results from 7/15 for 1B, below is the C.Limon first part of the g-block aligned with the RC of the 1A spliced seq:                   7.21.16  PCR with sten 1 & 2  This time, using the new forward primer.  Casey, Rusu, & Courtney prepared the reaction starting at 2:00 PM.  Casey - I came back to the lab later to run a gel and I realized we made a few mistakes.. Did not dilute the primers..  Did not add water to sten 2.. We only had 20 uL in the tube.. (this one is pretty bad you guys) Casey - 7:00 PM I’m re-doing the PCR.  Component 50μl reaction Molecular grade H2O Added first. 32.5μL  5x GC Buffer 10μL 10 mM dNTPs 1.0μL L. Synthase forward (10μM) (NEW) 2.5μL C. Sten 1 - Reverse (10μM) 2.5μL C. Sten g-block 1  1.0μL  Tom’s Phusion DNA polymerase Added last. 0.5μL  Component 50μl reaction Molecular grade H2O Added first. 32.5μL  5x GC Buffer 10μL 10 mM dNTPs 1.0μL L. synthase reverse (10μM) 2.5μL Sten 2 - Forward (10μM) 2.5μL Template DNA  1.0μL  Tom’s Phusion DNA polymerase Added last. 0.5μL  Cycle Step 3-step protocol Cycles Temp (oC) Time Initial denaturation 98 5 min 1 Denaturation 98 10s   Annealing 67 30s 30 Extension 72 2 min           Hold 4 hold    I’m thinking we should nanodrop to check for amplification instead of running a gel as we can save more of our sample if the result is good. Also, the gel won’t tell us a concentration, and we already know the gene block is in the reaction mix. We need to check the concentration of DNA.  After confirming that the gene blocks have been amplified, we should try the PCR assembly protocol or infusion cloning. We need to put the gene blocks together.  After that, we can run a gel.  2 bands around 1000bp = we suck 1 band around 2000 = we’re awesome  7.22.16 Nanodrop (both gave error warnings) gBlock 1 25 ng/uL 30 ng/uL gBlock 2 21 ng/uL Running gel Poured last night 110 volts Set 45 min Well 1 = C.sten 1 PCR product Well 2 = C.sten 2 PCR product  7.24.16 Re-did the PCR on C.Sten 1 and 2 using this mixture protocol: I had to dilute all four of the primers, 1:10 and I also dilute the 25mM dntps to 10mM 2:5 Component 50μl reaction Molecular grade H2O Added first. 32.5μL  5x GC Buffer 10μL 10 mM dNTPs 1.0μL L. synthase reverse/forward(10μM) 2.5μL C. Sten 2 - Forward/reverse (10μM) 2.5μL Template DNA 1.0μL  Tom’s Phusion DNA polymerase Added last 5uL  Cycle Step 3-step protocol Cycles Temp (oC) Time Initial denaturation 98 5 min 1 Denaturation 98 10s   Extension 68-72 (gradient) 1.5 min  35 Hold 4 hold   7.25.16 Ran a gel with the PCR products from 7.24, pic is shown below 1A and 2B off of Madeline’s master plate, ladder c.sten 1 and c.sten 2  Confirmed with Sharon that the orientation of the gene block in the strataclone vector does not matter, so in this case, we can move forward with experimentation with the first gene block (miniprep, phusion PCR) 7.29.16 -Redid the PCR with a 66-70°C gradient, and started a culture with 1A and 2B from madeline’s master plate -Diluted the 4 C. sten primers to 10mM and dNTPs to 10mM -Fixed gradient to do PCR with C. sten 1 and 2 TM + 3 and -3 (4 total)      C. Sten 1 Component 50μl reaction dH2O Added first. 28μL  5x GC Buffer 10μL 10 mM dNTPs 1.0μL L. Synthase forward (10μM) 2.5μL C. Sten 1 - Reverse (10μM) 2.5μL C. Sten g-block 1  1.0μL  Tom’s Phusion DNA polymerase (D) Added last. 5μL  C. Sten 2 Component 50μl reaction dH2O Added first. 28μL  5x GC Buffer 10μL 10 mM dNTPs 1.0μL C. Sten 2 forward (10μM) 2.5μL LS p2 (10μM) 2.5μL C. Sten g-block 2 1.0μL  Tom’s Phusion DNA polymerase (D) Added last. 5μL  Thermocycler setup:    Cycle step Phusion Cycles Temp (oC) Time Initial denaturation 98 30s 1 Denaturation 98 10s 35 Annealing TM+/- 3 30s           Extension     Hold         72        4 1min     ∞ 1               TM for C. Sten 1 Limonene Synthase forward primer - 77.8°C C. Sten 1 Reverse- 70.6°C TM for C. Sten 2 C. Sten 2 Forward- 72.1°C LS p2- 70.2°C  C. Sten 1 -/+ 3°C                                                LSF- 74.8/80.8°C C.Sten 1 R- 67.6/73.6°C         C. Sten 2 -/+3°C C. Sten 2 F- 69.1/75.1°C°C LS p2- 67.2/73.2    Placement in Thermocycler column #: CS2 (-3) = 2        CS2 (+3) = 10 CS1 (-3) = 3        CS1 (+3) = 11           8.8.16 Running gel with PCR products of 7.29.16  Looks good  8.9.16 Performed the gel purification protocol from the lab, including the following changes: Only did 1 membrane wash  Did not pre heat the elution buffer Did the final elution with 15uL of the 10mM tris-HCL ph 8.8 Both parts of the gene block will be nanodropped tomorrow to see how high the concentration of DNA is      8.10.16 PCR with minipreped stataclone vector + insert from culture started on 7.29.16 We have 4 samples, 1A, 1B, 2B, 2C  Two different annealing temperatures for each sample. 68 & 72 So we will have 8 PCR reactions. For C. Limon 1 Component 50μl reaction Molecular grade H2O Added first. 28 μL  5x GC Buffer 10μL 10 mM dNTPs 1.0μL L. synthase forward (10μM) 2.5μL C. Limon 1 or 2 middle primer (10μM) 2.5μL Template DNA  1.0μL  Tom’s Phusion DNA polymerase Added last 5uL So we’re going to need (4)(2.5μL) of each primer.  In the end we want (10μL)(10μM) so..  dilute(10μL)(10μM) = 10-4μmol = 10-4μmol(1L/25μmol) = 4*10-6L = 4μL stock  4μL stock + 6μL H2O = 10μL(10μM) primers & 8μL of dNTP’s stock 25 mM We want (8μL)(10mM) Dilute (8μL)(10mM) = 8*10-5mmol =  8*10-5mmol (1L/25mmol) = 3.2*10-6L = 3.2μL 3.2μL + 4.8μL H2O = 8μL(10mM) dNTP’s  For C. Limon 2 Component 50μl reaction Molecular grade H2O Added first. 32.5μL  5x GC Buffer 10μL 10 mM dNTPs 1.0μL LS p2(10μM) 2.5μL C. Limon 2 Forward (10μM) 2.5μL Template DNA  1.0μL  Tom’s Phusion DNA polymerase Added last 5uL So we’re going to need (4)(2.5μL) of each primer.  Cycle Step 3-step protocol Cycles Temp (oC) Time Initial denaturation 98 5 min 1 Denaturation 98 10s   Extension 68-72 (gradient) 1.5 min  35 Hold 4 hold      8.10.16 -Ran a gel with the completed C.Limon PCR products listed above from 8.9.16, results are shown below: ladder, next three were sharon’s stuff, 1A-68, 1B-68, 2B-68, 2C-68, 1A-72, 1B-72, 2B-72, 2C-72  -Nanodropped C. Sten 1 & 2 gel purification products results are shown below: C. Sten 1 green tube- 9.9 ng/ul, 260/280- 2.07 C. Sten 1 blue tube- 16.2 ng/ul, 260/280-1.92 C. Sten 2 orange tube- 14.1 ng/ul, 260/280-1.84 C. Sten 2 yellow tube- 17.4 ng/ul, 260/280-1.72  8.12.2016    Fluorimetry on C. limon gel purification calibration : 99.76  Concentration(ng/uL)  Error 1A-68: 18.9k    -1.172 1A-72: 76.58    6.4 1B-68: 53.96    -2.9 1B-72: 33.34    -2.472 2B-68: 27.93    -5.44 2B-72: 69.96    -10.76 2C-68: 76.63    ? 2C-72 : 13.27    -11.73   8.13.16  -We tried infusion cloning with C.sten 1 and 2 g-blocks, these were the ones that were not in the strataclone vector, so after PCR amplification and running a gel to confirm amplification, we attempted to just directly clone both g-blocks into the pLC71 vector We used a 1:1:1 molar ratio for 1st g-block:2nd g-block:pLC71 and the values for each are shown below, keep in mind we had two PCR products for each g-block, so they are separated by eppendorf tube color, in this case we only used C.sten 1 blue and C.sten 2 yellow  C.Sten 1 blue: 1.36 uL C. Sten 2 yellow: 1.02uL pLC71:0.88uL, dilution- 2uL pLC71 + 2uL sterile H2O, this way we could use 1.8uL instead of 0.88 which is hard to achieve with a pipette 3.8uL of water  We did a negative control that contained everything listed above, except for the infusion mix  -After following the Infusion protocol, we followed the Interlab transformation protocol, except we added 2 uL of the plasmid instead of 1uL, also when we plated after the outgrowth period, we use 50uL on one plate, and the rest on another plate, ~200uL  9.2.16  Nanodropping: C. Limon 1B & 2B in the strataclone vector  1B: 361.4 ng/μL 260/280: 1.84 2B: 368.9 ng/μL 260/280: 1.84  PCR:   M1V1 = M2V2  V1B = (2μL)(361.4 ng/μL) / (10ng/μL) = 72.28 μL  V1B = (2μL)(361.4ng/μL) / (10ng/μL) = 73.78 μL  So, to dilute the plasmids to 10 ng/μL..  1B: 2μL stock + 70.3μL dH2O  1B: 2μL stock + 71.8μL dH2O  Primers:   1B: Limonene Forward + C.limon1R 2B: C.limon2F + LSp2  We have 100μM primers, we want 10μM.  M1V1 = M2V2  V = (2μL)(100μM) / (10μM) = 20μL   So to dilute the primers…  2μL stock + 18μL dH2O     Component 50μl reaction MMix (5 rxns) - 41uL  Molecular grade H2O Added first. 27.5 μL  137.5µL 5x GC Buffer 10μL 50µL 25 mM dNTPs 0.5μL 2.5µL Primer 1(10μM) 2μL   Primer 2 (10μM) 2μL   Template DNA 10ng/uL 3μL  15µL Tom’s Phusion DNA polymerase Added last 5uL     Cycle Step 3-step protocol Cycles Temp (oC) Time Initial denaturation 98 2 min 1 Denaturation Anneal 98 60 10s 10s   Extension 72 2 min   40 Final Extension 72   1 Hold 4 hold    9.3.16  PCR failed. New idea; add DSMO (5% volume). Also try raising the annealing temperature.   DMSO:  We have 100% DMSO from NEB  (0.05)(50µL) = 2.5µL DMSO  With DMSO: Component 50μl reaction MMix (5 rxns) - 40uL  Molecular grade H2O Added first. 27μL  135µL 5x GC Buffer 10μL 50µL 25 mM dNTPs 0.5μL 2.5µL Primer 1(10μM) 2μL   Primer 2 (10μM) 2μL   100% DMSO 2.5µL 12.5µL Template DNA 10ng/uL 1μL    Tom’s Phusion DNA polymerase Added last 5uL    Cycle Step 3-step protocol Cycles Temp (oC) Time Initial denaturation 98 2 min 1 Denaturation Anneal 98 65 10s 10s   Extension 72 2 min   40 Final Extension 72   1 Hold 4 hold       Without:  Component 50μl reaction MMix (5 rxns) - 40uL  Molecular grade H2O Added first. 27.5 μL  147.5µL 5x GC Buffer 10μL 50µL 25 mM dNTPs 0.5μL 2.5µL Primer 1(10μM) 2μL   Primer 2 (10μM) 2μL   Template DNA 10ng/uL 3μL    Tom’s Phusion DNA polymerase Added last 5uL      Cycle Step 3-step protocol Cycles Temp (oC) Time Initial denaturation 98 2 min 1 Denaturation Anneal 98 65 10s 10s   Extension 72 2 min   40 Final Extension 72   1 Hold 4 hold    Brett Burkart, an employee of the Santangelo lab, gave Casey this protocol for purifying PCR products. PB buffer Add 5x the volume of the combined PCR reactions.  Combined PCR reactions PB Buffer 177µL 885µL  3M NaOAc pH 5.2 Add 1/100x the volume of the PCR reaction + PB buffer Working Volume  3M NaOAc pH 5.2 1062µL 10µL  Spin through column (may take two runs). Discard flow through.  Load column with 750µL ethanol, spin. Discard ethanol, spin 2 minutes to dry. Label 1.7mL eppe. Load 25µL 10mM tris HCl pH 8.5 Spin 2 mins. collect in labeled eppe. Load 25µL 10mM tris HCl pH 8.5 Spin 2 mins. collect in same labeled eppe.  9.20.16 Dr. Peebles discovered the there was no overlap between our insert and vector. The infusion cloning could not have worked. New primers Forward: GGC CGC AAA AAA GTC GAA TGC GCG TTG CGG CAG TCG  melt temp: 77.7 ºC  homologous region:  A TGC GCG TTG CGG CAG TCG  melt temp: 70.6 ºC  Reverse: CAA AAT GCG CGT TGC GGC AGA GCG GCC GCA AAA AAG TCG AC melt temp: 78.4  homologous region: AGCGGCCGCAAAAAAGTCGAC melt temp: 68.3 ºC 9.22.16 Picked up new primers & resuspended.  9.26.16     PCR with new primers, from strataclone vector.   Component 50μl reaction MMix (5 rxns) - 41uL  Molecular grade H2O Added first. 27.5 μL  137.5µL 5x GC Buffer 10μL 50µL 25 mM dNTPs 0.5μL 2.5µL Primer 1(10μM) 2μL   Primer 2 (10μM) 2μL   Template DNA 10ng/uL 3μL  15µL Tom’s Phusion DNA polymerase Added last 5uL     Cycle Step 3-step protocol Cycles Temp (oC) Time Initial denaturation 98 2 min 1 Denaturation Anneal 98 60 10s 10s   Extension 72 2 min   40 Final Extension 72   1 Hold 4 hold             7.25.17 Did a gDNA prep with non-confirmed strains: pHF10: 1L, 4N, 2M, 3L, 2O, 1C, 4M, 4K, 1M, 2L pLC71: 4E, 2E, 2A, 9E, 1C, 4P, 4D Positive control: pHF10 3S Negative control: pLC71  7.26.17 Performed PCR: Made a master mix with the following reagents 4.8 ul forward and reverse primers 12 ul dNTP 120 ul phusion 240 ul GC buffer 746.56 ul water 3 ul gDNA from extraction Ran a gel with the gDNA prep, but the gel was upside down so samples ran out           7.27.17  Ran the gel again correctly shown below, 120V for 50 minutes:  pHF10: 1L-1, 4N-2, 2M-3, 3L-4, 2O-5, 1C-6, 4M-7, 4K-8, 1M-9, 2L-10 pLC71: 4A-11, 2E-12, 2A-13, 9E-14, 1C-15, 4P-16, 4D-17 pHF10 positive control-18 pLC71 negative control-19 Confirmed pHF10: 1C, 2L, 4K, 1L, 4N, 3L, 2O
