<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Lethbridge_project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Lethbridge/project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Lethbridge/project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Lethbridge/project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Lethbridge/project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV style="background-color:#FFFFFF; color:white"><BLOCKQUOTE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Project"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project</SPAN></A></LI><LI class="toclevel-2"><A href="#Background_.26_Rationale"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Background &amp; Rationale</SPAN></A></LI><LI class="toclevel-2"><A href="#Objectives"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-2"><A href="#Research_Design_.26_Methods"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Research Design &amp; Methods</SPAN></A></LI><LI class="toclevel-2"><A href="#Significance_.26_Future_Directions"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Significance &amp; Future Directions</SPAN></A></LI><LI class="toclevel-2"><A href="#References"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Project"><FONT color="black">Project</FONT></SPAN></H1><H2><SPAN class="mw-headline" id="Background_.26_Rationale">Background &amp; Rationale</SPAN></H2><P>Astrocytes are an abundant form of glial cell located in the brain and spinal cord that serve to regulate cerebral blood flow and synapse function [1]. Central nervous system (CNS) injuries and neurodegenerative diseases are frequently accompanied by reactive astrogliosis, a process whereby astrocytes isolate damaged neural tissue, restrict inflammation, stimulate blood-brain barrier repair, and reduce further degeneration of surrounding brain tissue [2]. However, the formation of this obstructive “glial scar” also impedes future neural regeneration and axon remyelination and thus is a significant obstacle to rehabilitation in the CNS [3].</P><P>Interestingly, a recent study demonstrated the direct reprogramming of in vivo murine and cultured human reactive astrocytes to functional neurons using the retroviral expression of a single transcription factor, NeuroD1 [4]. Thus, NeuroD1 (if appropriately targeted to reactive astrocytes in damaged CNS tissue) has the potential to significantly improve current neural regeneration therapies. However, direct application of NeuroD1 to the human brain would not only be largely non-specific (and require the use of controversial viral delivery systems) but also excessively invasive. Consequently, the development of novel therapeutic delivery mechanisms is necessary to specifically target DNA, RNA, proteins, and other biologically relevant molecules to damaged or diseased neural cells. Microglia, another class of glial cell prevalent in the CNS, migrate to regions of neural damage and are the primary immune response in the CNS [5]. Their mobility, proximity to reactive astrocytes (and thus neural damage), and exosomal communication make them suitable candidates as potential delivery mechanisms for CNS injury-related rehabilitative therapies [6].</P><H2><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H2><P>Altogether, the aim of this study is to harness microglia as non-immunogenic carriers of neural tissue-specific therapeutic genes. Specifically, in an effort to promote functional recovery after CNS insult, these microglial cells will express a synthetic plasmid delivery system that exploits endogenous exosome production capabilities to deliver a neural reprogramming message. Our specific objectives include:</P><P>1) Generate microglial cells that package plasmids into exosomes and secrete them at regions of neural damage.</P><P>2) Use these engineered microglia to target stable plasmid DNA copies of the reprogramming factor gene NeuroD1 to regions of astrogliosis in a cell culture model of traumatic brain injury. NeuroD1 will be selectively expressed in reactive astrocytes, thereby directly converting obstructive glial scars to neurons (replacing at least some of those lost to damage or disease) and reducing the inhibitory environment produced by the reactive astrocytes.</P><P>3) Design a novel antibiotic-free plasmid selection system that will allow generation of the NeuroD1-carrying plasmids while avoiding the introduction of antibiotic-resistance coding genes into human cells. Our proposed system will also minimize the size of our plasmid for ease of exosomal packaging.</P><H2><SPAN class="mw-headline" id="Research_Design_.26_Methods">Research Design &amp; Methods</SPAN></H2><P>1) The rabies virus glycoprotein (RVG) is a small viral peptide that allows for CNS-specific uptake of exosomes. RVG, when fused to Lamp2B (an exosomal membrane protein), has proven to be a successful system for targeting siRNAs into CNS cells via exosomal delivery in mice [7]. Thus, to specifically target microglial exosomes (carrying therapeutic plasmids) to CNS cells, we will fuse RVG to the extra-exosomal N-terminus of Lamp2B and a Gal4 DNA-binding protein to the C-terminus to anchor the plasmid within newly formed exosomes [8]. Initial tests included the use of Clover (a green fluorescent protein) in place of the Gal4 module of our plasmid delivery system to examine exosomal packaging function and proper membrane orientation. </P><P><B>Figure 1. A schematic representation of the exosome localization fluorescence test construct. </B>This construct consists of a fusion between the RVG-Lamp2B protein and the fluorescent protein Clover available. The addition of clover allows for easy visualization and localization of expression. </P><P><B>Figure 2.  A schematic representation of the exosomal plasmid packaging construct.</B> This construct consists of a fusion between the RVG-Lamp2B protein and the Gal4 DNA binding domain available on the registry (K105007). When expressed in cells also harbouring a plasmid with the Gal4 UAS, this protein will bind and shuttle that plasmid into the exosome, enhancing therapeutic plasmid delivery to CNS cells.</P><P>2) The therapeutic plasmid will encode the reprogramming factor NeuroD1 under the regulation of a synthetic  astrocyte-specific promoter (gfaABC1D), expression of which is strongly enhanced in reactive astrocytes. Thus, transcription of NeuroD1 will be specifically restricted to reactive astrocytes [9]. </P><P><B>Figure 3. A schematic representation of the NeuroD1-IRES-Clover construct.</B> This construct is the NeuroD1 protein, an internal ribosome entry site (IRES), and the clover GFP variant. This will allow us to visualize the cells in which the NeuroD1 transcription factor is being expressed. </P><P><B>Figure 4. A schematic representation of the gfaABC1D regulated NeuroD1-IRES-Clover construct. </B> This is the same construct as above, but with astrocyte specific expression. This part in a plasmid containing the Gal4 UAS is the final therapeutic plasmid </P><P>3) mRNA encoding Tobacco Etch Virus (TEV) protease will also be targeted to the microglial exosomes to induce the production of TEV protease within the target cells [10-11]. The active TEV protease will cleave a specific protein sequence located between the Lamp2B and TALE proteins of the designed protein scaffold, thereby releasing the TALE and the bound therapeutic plasmid into the cytosol of the target cell. A nuclear localization sequence (located in proximity to the TALE sequence) will target the bound plasmid to the nucleus, where NeuroD1 will be transcribed if (and only if) the target cell is a reactive astrocyte. The expression of NeuroD1 will induce reprogramming of the astrocyte into a new, functional glutamatergic neuron.</P><P><B>Figure 5. Construct design for the TEV protease.</B> This construct with be transfected into microglia cells, and added alongside microglia transfected with the Lamp2B-Gal4 plasmid delivery system. Exosomes from both transfected microglia cell types will be targeted to reactive astrocytes whereby the TEV protease encoded by this part will cleave the Lamp2B-Gal4 releasing the nuclear localization sequence (NLS) and Gal4 binding domain to enter into the nucleus of the astrocyte.</P><P>4) Antibiotic resistance is commonly used as a laboratory method for selecting bacteria containing a plasmid of interest. Besides the obvious implications of transferring antibiotic resistance genes to human cells, the large size of plasmids carrying antibiotic resistance markers is non-ideal for exosomal packaging. To bypass the use of antibiotic resistance markers during plasmid selection and to reduce the overall size of the therapeutic plasmid, the RNA-IN/OUT system will be used to regulate cell lysis via the production of T4 Holin and Endolysis proteins [12-13]. The RNA-OUT mRNA sequence (transcribed from the plasmid) binds to any mRNA with the RNA-IN sequence upstream of the ribosomal binding site (RBS), thereby occluding ribosomal binding and preventing translation of the mRNA. In this study, the RNA-IN sequence will be located before the RBS of the T4 Holin-Endolysin mRNA which, when translated in the absence of the RNA-OUT containing plasmid, will cause cell death. </P><P><B>Figure 6. Lysis RNA-IN/OUT construct design.</B> On the left is an arabinose inducible lysis cassette with the S32 RNA sequence preceding the RBS. On the right is a constitutively expressed A32 RNA-OUT construct that has complementarity with the S32 RNA-IN in the sequence adjacent to the lysis cassette RBS. When expressed in the same cell, translation of the T4 Holin protein will be prevented and the cell would survive in arabinose.</P><P><B>Figure 7. Arabinose inducible lysis cassette .</B> When cells containing this part in grown in arabinose, expression of the Holin lysis cassette occurs and the cells die. This will be used in our antibiotic free plasmid selection system. </P><P><B>Figure 8. Universal RNA-IN/OUT selection system.</B> On the left is an improvement of the B0034 RBS part. Upstream of the RBS is the S32 RNA-IN sequence, and at the appropriate distance downstream from the RBS is a start codon and the <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> suffix. Having the start codon attached to the RBS part will add 3 amino acids to any protein expressed, but it will allow a single RNA-OUT part to regulate the translation of any protein that uses this RBS. On the right is the universal RNA-OUT, which contains the A32 antisense partner for the S32 RNA-IN sequence.</P><P><B>Figure 9. Heat-inducible promoter with universal RNA-IN RBS.</B> This part will allow expression of any protein placed downstream of it in a heat-dependent manner. That protein's translation can be prevented by the expression of the universal A32 RNA-OUT.</P><P>5) The engineered microglial cells will be introduced into CNS tissue cultures that imitate sites of neurological damage and neuron loss. Initial tests will include delivery of a GFP-producing plasmid with the RVG-Lamp2B-Gal4 system to test plasmid transfer and astrocyte-specific expression. Subsequently, the ability of the NeuroD1 reprogramming system described here to reprogram astrocytes will be tested via immunohistochemical detection of neuron-specific and glia-specific marker expression within the reprogrammed astrocytes. </P><P><B>Figure 10. Cell cultures used in this study as seen under a light microscope. </B> A) LADMAC cells. B) HEK-293 cells. C) EOC 13.31 (mouse microglia) cells. D) C8-D30 (mouse astrocyte type III) cells. </P><H2><SPAN class="mw-headline" id="Significance_.26_Future_Directions">Significance &amp; Future Directions</SPAN></H2><P>Every year, the medical expenses, lost wages, and decreased productivity associated with stroke costs the Canadian economy $3.6 billion [14]. By 2040, Alzheimer’s Disease and other neurodegenerative disorders are predicted to cost the Canadian economy $293 billion per year [15]. The long-term objective of this proposal is to significantly improve current neural regeneration therapies in an effort to overcome the rehabilitative obstacles associated with CNS insults such as stroke, Alzheimer’s Disease, and traumatic brain injury in a non-invasive, cost-effective manner [4]. Furthermore, with future discoveries of other novel tissue-specific tags (similar to RVG) coupled with targeted DNA transmission therapies such as that discussed in this study, this system can potentially be harnessed to combat other tissue-specific disorders.</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>Project header image adapted from Figure 6 of: Bi, F., Huang, C., Tong, J., Qiu, G., Huang, B., Wu, Q., Li, F., Xu, Z., Bowser, R., Xia, X-G., &amp; Zhou, H. (2013). Reactive astrocytes secrete Icn2 to promote neuron death. PNAS, 110, 4069-4074.</P><P>[1] Barres, B.A. (2008). The mystery and magic of glia: a perspective on their roles in health and disease. Neuron, 60, 430-440.</P><P>[2] Sofroniew, M.V. (2005).  Reactive astrocytes in neural repair and protection. The Neuroscientist, 11, 400-407.</P><P>[3] Fawcett, J.W. &amp; Asher, R.A. (1999). The glial scar and central nervous system repair. Brain Research Bulletin, 49, 377-391.</P><P>[4] Guo, Z., Zhang, L., Wu, Z., Chen, Y., Wang, F., &amp; Chen, G. (2014). In vivo direct reprogramming of reactive glial cells into functional neurons after brain injury and in an Alzheimer’s disease model.  Cell Stem Cell, 14, 188-202.</P><P>[5] Aloisi, F. (2001). Immune function of microglia. Glia, 36, 165-179.</P><P>[6] Turola, E., Furlan, R., Bianco, F., Matteoli, M., &amp; Verderio, C. (2012). Microglial microvesicle secretion and intercellular signaling. Frontiers in Physiology, 3, 149.</P><P>[7] Alvarez-Erviti, L., Seow, Y., Yin, H., Betts, C., Lakhal, S., &amp; Wood, M.J.A. (2011). Delivery of siRNA to the mouse brain by systemic injection of targeted exosomes. Nature Biotechnology, 29, 341-347.</P><P>[8] Bogdanove, A.J. &amp; Voytas, D.F. (2011). TAL effectors: Customizable proteins for DNA targeting. Science, 333, 1843-1846.</P><P>[9] Brenner, M., Kisseberth, W.C., Su, Y., Besnard, F., &amp; Messing, A. (1994). GFAP promoter directs astrocyte-specific expression in transgenic mice. The Journal of Neuroscience, 14, 1030-1037.</P><P>[10] Kapust, R.B. &amp; Waugh, D.S. (2000). Controlled intracellular processing of fusion proteins by TEV protease. Protein Expression and Purification, 19, 312-318.</P><P>[11] Batagov, A.O., Kuznetsov, V.A., &amp; Kurochkin, I.V. (2011). Identification of nucleotide patterns enriched in secreted RNAs as putative cis-acting elements targeting them to exosome nano-vesicles. BMC Genomics, 12(Suppl 3):S18</P><P>[12] Mutalik, V.K., Qi, L., Guimaraes, J.C., Lucks, J.B., &amp; Arkin, A.P. (2012). Rationally designed families of orthogaonal RNA regulators of translation. Nature Chem. Bio., 8, 447-454 </P><P>[13] Young, R. (2002). Bacteriophage holins: deadly diversity. Journal of Molecular Microbiology and Biotechnology, 4(1), 21–36.</P><P>[14] Public Health Agency of Canada. (2009). Tracking heart disease and stroke in Canada. Retrieved from <A href="http://www.phac-aspc.gc.ca/publicat/2009/cvd-avc/pdf/cvd-avs-2009-eng.pdf" class="external free" rel="nofollow">http://www.phac-aspc.gc.ca/publicat/2009/cvd-avc/pdf/cvd-avs-2009-eng.pdf</A></P><P>[15] Alzheimer Society of Canada. (2012). A new way of looking at the impact of dementia in Canada. Retrieved from <A href="http://www.alzheimer.ca/~/media/Files/national/Media-releases/asc_factsheet_new_data_09272012_en.pdf" class="external free" rel="nofollow">http://www.alzheimer.ca/~/media/Files/national/Media-releases/asc_factsheet_new_data_09272012_en.pdf</A></P></BLOCKQUOTE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Lethbridge/project">http://2014.igem.org/Team:Lethbridge/project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Lethbridge/project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Lethbridge/project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Lethbridge/project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Lethbridge/project&amp;oldid=392195" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>