"
Page
Discussion
View source
History
teams
Log in
 
Team:Minnesota/Parts Characterization
From 2009.igem.org
Home
The Team
The Project
SynBioSS Designer
Modeling
Experimental
Competition Requirements
Contents
1 Comments
2 Parts Characterization
2.1 Introduction
2.2 Sequencing and Narrowing Down Parts to Characterize
2.3 Characterization Experiments
2.3.1 Promoter I14032: IPTG regulation
2.3.1.1 Transfer Function
2.3.1.2 Stability
2.3.2 Promoter J13002: aTc regulation
2.3.2.1 Transfer Function
2.3.2.2 Stability
2.4 Challenges
2.5 Protocols
2.6 Acknowledgments
2.7 References
Comments
Although we will be judged only for the software we developed, an important aspect of our work is the combination of experiments and simulations. For detailed comparisons please look at the Notebook in the end of July.
We are submitting our constructs to the Registry.
We also characterized a handful of older BioBricks. Here are the details we are entering in the Registry.
Parts Characterization
Introduction
After an exhaustive search of the Registry of Standard Parts , we found five promoters that were included in the 2009 iGEM Kit as potential parts characterize. Using part BBa_F2620 , which was characterized by a group from MIT in 2004, as a template, we examined the following parts:
Part
Description
Regulators
People
I14032
Constitutive promoter classified as repressible
IPTG
Princeton 2004
J13002
Two TetR binding sites and RBS
aTc
UT Austin 2005
I14015
LasR, 3OC12HSL aTc regulated promoter
LasR, 3C12HSL, aTc
Princeton 2004
K091101
TTL AND gate
IPTG, aTc
Davidson Missouri-Western 2008
R0011
Inverting regulatory region controlled by LacI; for comparison since already characterized
IPTG
Registry
We chose these parts because they had the same regulators as the promoters we examined in our project. Part K091101 was particularly interesting because it was one of the constructs of the Tet and Lac operators that we examined for our research. Since Tet and Lac are commonly studied operators in synthetic biology, we wanted parts that involved them to be well-characterized to ensure the viability of future research. We decided to characterize these promoters by attaching them to part K081012 , which consists of a strong RBS and GFP. This 'PoPS generator' takes PoPS (Polymerase Per Second) as an input and gives GFP as an output, allowing us to indirectly measure PoPS and characterize our parts.
The figure at the right demonstrates how to combine standard biological parts to form a new composite part. In our case, the prefix part was each of our 5 promoters and these were digested with restriction enzymes EcoRI and SpeI. The suffix part was the PoPS generator in every case except part J13002 , which already contained an RBS. This RBS is defined as efficiency 1.0 while the RBS contained on our PoPS generator, which has an efficiency of 0.6. We decided to characterize the entire part that UT Austin 2005 submitted, which included their RBS. These suffix parts were cut using restriction enzymes XbaI and PstI. We ligated these parts to make a composite BioBrick part in pSB3K3 , a low-medium copy plasmid with kanamycin resistance. The 2009 iGEM Judging Criteria gives MIT's characterization of part BBa_F2620 as an exemplar of parts characterization. We gratefully acknowledge their pioneering work with parts characterization and hope that our work continues to maintain the high standard for characterization of parts. We characterized our 5 promoters included with the 2009 iGEM kit in terms of:
Transfer Function: the equilibrium relationship between the input and output
Stability: how transfer function changes across multiple rounds of cell division and culture
Sequencing and Narrowing Down Parts to Characterize
We utilized the software Sequencher through the Minnesota Supercomputing Institute (MSI) to trim the ends of sequences we received from the BMGC and align them. A chromatogram, shown below, told us the strength of the signal for each base pair. A good sequencing reaction will have a single peak for each base pair while problems with the primer or multiple products in the sequencing reaction will result in multiple peaks with equally strong signals for each base pair.  We wanted to compare both sequencing reactions we had as of 16 July 2009 to each other and to the sequences of the parts that were available on the Registry . We obtained successful alignments for parts I14032, J13002 and K091101, which were regulated by IPTG, aTc and both inducers, respectively. It is interesting to note that while our samples appeared to have multiple products, that is, on the chromatogram that accompanied our sequence output, we saw multiple signals for each base pair, the sequence products not only aligned with each other, but with the sequences on the Registry. So we ordered new primers for another sequencing reaction and, based on these alignments as well as the fact that part R0011 had already been characterized and I14013 was regulated by LasR as well as aTc, we reduced our number of parts to characterize to three.
Characterization Experiments
Our list of parts to characterize was then narrowed down to:
I14032
J13002
K091101
We inserted each promoter and either the PoPS generator or GFP into the pSB3K3 plasmid backbone in Top10 cells and confirmed our ligation by agarose gel and sequencing. Then, we transformed our new plasmid into DH5αPro cells, which constitutively express TetR and LacI for characterization. Therefore, even in our inducer media of 0 ng/ml, we observed some fluorescence. The plasmid also conferred kanamycin resistance upon the cells so we plated them on LB + Kan media in order to select for our successful transformants. In order to examine Transfer Function and Stability for each promoter, we grew 1ml cultures at increasing concentrations of either IPTG or aTc and kept them at OD595=0.2. We sampled each of our cultures every hour for 9 hours, fixing the samples in 4% PFA and resuspending in PBS in preparation for flow cytometry . Flow cytometry allowed us to pick out the population of living cells and determine the amount of GFP that each cell was producing.  We utilized the software FlowJo to analyze our flow cytometry data. We were interested in the mean GFP for 100,000 cells at each timepoint and inducer concentration and were able to extract these data from the plethora of information included in the data file.
Promoter I14032: IPTG regulation
The first promoter we looked at was I14032 , which was submitted to the Registry by Princeton in 2004.
In the Experience section on the Registry, it was stated that although this promoter was classified as repressible, it was actually constituitive. However, there were no data to support this statement so we decided to investigate the promoter. In addition to investigating the promoter inducibility status, we also wanted to examine Transfer Function and Stability. So we attached the PoPS generator to this promoter in a ligation reaction and induced it at various concentrations of IPTG: 0, 0.001, 0.01, 0.05 and 1 mM. We took samples over 9 hours and analyzed the products using flow cytometry.
Transfer Function
As you can see on the figure below, for each timepoint, the production of GFP is virtually identical. This confirmed the hypothesis that the promoter is actually constituitive rather than repressible. Regardless of IPTG concentration, we saw the same amount of GFP production over all the timepoints.
Stability
We also examined the GFP production at varying IPTG concentrations over multiple rounds of cell division. As you can see from the figure below, not only is the constituitive promoter reliably on, but it continues to produce GFP at the same rate regardless of the number of cell divisions.
Promoter J13002: aTc regulation
The second promoter we examined was J13002 , which was submitted to the Registry by UT Austin in 2005. The part consists of two TetR binding sites and RBS and is called a "TetR repressed PoPs/RIPS generator." Since the part already had a RBS, we attached GFP in order to indirectly measure PoPS. The results of our characterization can be found on the Experience page at the Registry for Part J13002. A photo of this part from the Registry of Standard Parts is displayed below:
<b3 />
Transfer Function
We also wanted to examine Transfer Function and Stability in this promoter. To do this, we grew 1 ml liquid cultures in different concentration of inducer media: 0, 1, 10, 50, 100 and 200 ng/ml of aTc. As you can see in the figure below, there was a general upward trend of GFP production with the concentration of aTc. The error bars in the graphs represent the standard error. Interestingly, we observed a spike in GFP production at 50 ng/ml of aTc. During the sampling, the culture growing in 50 ng/ml aTc inducer media did not grow as fast as the others, with the OD595 as low as 0.08 rather than the preferred 0.2. The cells were putting their energy towards GFP production rather than growing, so the lack of growth and spike of mean GFP is legitimate.  Since the inducer is an antibiotic, we hypothesized that at higher concentrations of 100 ng/ml and 200 ng/ml, the cells were using all their energy dealing with the aTc rather than GFP production. We did perform this experiment again but did not see any induction.
Stability
We also examined the stability of the device, which is how the production of GFP changes for each inducer concentration over multiple rounds of cell division. As you can see from the figure below, the device is clearly inducible with a general downward trend in GFP production over time. The graph of aTc concentration of 50 ng/ml again demonstrates the spike in GFP production that we observed at this concentration. This is the optimal inducer concentration for the device.
Unfortunately, we ran out of time for characterization of the third and final part, K091101 . However, we did characterize two existing BioBrick parts in terms of Transfer Function and Stability.
Challenges
A day by day catalog of what we did for parts characterization can be found on our Google calendar on the Experiments and Calendar page. Clearly, despite the summary above, some of these steps we had to redo over and over. Often, our DNA after plasmid preparation and restriction enzyme digest was not very pure and had a low concentration, which necessitated picking colonies, growing more cultures and prepping more plasmids. Sequencing, which was performed by the BMGC on the U of MN campus, provided us with a huge challenge because often, our sequences were not clean so we were not sure that we had the correct parts.
Running samples out on a gel as shown to the right, and specing samples can be very helpful in double-checking the size and purity of fragments, but sequencing was a very important step that really told us whether our procedure was viable. When we received wonky results-- that is, the signals were mixed and the samples appeared to have multiple products-- we hypothesized that this was due to the primer setting down in the wrong location during the sequence reaction or perhaps a hairpin loop structures getting in the way of the sequencing (we gratefully acknowledge John Barrett for his suggestions).
Protocols
Resuspend DNA for GFP and the promoter + RBS from the 2009 iGEM Parts Kit in 15 ul water
Transform DNA into Top10 Chemically Competent Cells
Pick colonies from transformation and grow in 2 ml cultures with appropriate antibiotic
Prep the plasmids
Determine DNA purity
For a further check, may perform PCR and run the products out on an agarose gel to see if amplification was successful. Use forward and reverse primers for BioBricks.
Sequence plasmids
Digest the promoter, GFP and plasmid backbone with the appropriate restriction enzymes for later ligation
Gel purify digest products
Ligate the promoter + RBS and GFP after restriction enzyme digest in the plasmid backbone pSB3K3 overnight
Sequence ligation products
Transform the ligation products into DH5alphaPro cells, which express TetR
Pick colonies and grow in 2ml liquid cultures plus kanamycin
Make up 5ml stock of inducer media at 0, 1, 10, 50, 100, 200 ng/ml aTc and one batch of 0.5 ml inducer media with twice the aTc concentration for the first induction
Spec cells at time 0
Inoculate 2x [aTc] media with 0.5 ml cells
Every hour, spec the DNA and, based on the OD595, which should be 0.2 dilute the cultures with inducer media and take 200 ul for sampling
Spin down 200 uL cells at 5K rpm for 5 minutes so as not to squash them
Remove supernatant and resuspend in 0.5 ml 4% sterile filter PFA
After 30 minutes, the cells have been fixed and can be spun down at 5K rpm for 5 minutes
Resuspend cells in 0.5 ml PBS and store in fridge until ready for flow cytometry
Sample for 9 hours
Run through flow cytometer and analyze using FlowJo software and Excel spreadsheet
Acknowledgments
To the MIT 2004 team for their pioneering work in Parts Characterization. Thanks also to the BMGC at UMN for sequencing our myriad of parts and the Masonic Cancer Center at UMN for allowing us to utilize the flow cytometers. We also acknowledge John Barrett for his help using FlowJo and suggestions on obtaining successful sequences.
References
Shetty, R.P.; Endy, D.; Knight, T.F. 2008. Engineering BioBrick vectors from BioBrick parts. Journal of Biological Engineering 2.
Retrieved from " http://2009.igem.org/Team:Minnesota/Parts_Characterization "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
