<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCSC_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCSC/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="button-custom"><DIV class="tiny-little-menu-word-above-button">MENU</DIV></DIV><UL><LI><A href="https://2017.igem.org/Team:UCSC/Judging">JUDGING</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Notebook">NOTEBOOK</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Human_Practices">INTRODUCTION</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Safety">SAFETY</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Applied_Design">APPLIED DESIGN</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Engagement">EDUCATION</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/HP/Silver">SILVER</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/HP/Gold_Integrated">GOLD INTEGRATED</A></LI></UL><LI><A href="https://2017.igem.org/Team:UCSC/Project">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Project">THE PROJECT</A></LI><LI><A href="#">METABOLIC ENGINEERING</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Acetaminophen">ACETAMINOPHEN</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/B-12">VITAMIN B<SUB>12</SUB></A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Model">MODELING</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Demonstrate">RESULTS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Target-Organism">TARGET ORGANISM</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Part_Collection">PARTS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/InterLab">INTERLAB STUDY</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Team">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Team">MEET THE TEAM</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC">HOME</A></LI><DIV id="page"><CENTER><H1>RESULTS</H1><DIV class="text-container"><P>Our project focuses on biosynthetically producing the molecules vitamin DMB B<SUB>12</SUB> and acetaminophen in <I>Synechococcus elongatus</I> PCC 7942. The secondary focuses of our project were geared towards analyzing the growth of <I>S. elongatus</I> PCC 7942 in seawater, creating a detection method for vitamin B<SUB>12</SUB> via a riboswitch/reporter system, and a proof of concept for the production of acetaminophen in <I>E. coli</I>.</P><H2 style="text-align: left; font-weight: 500;">I. Biosynthesis</H2><H4 style="text-align: left; font-weight: 400; font-size: 140%;">1a. Plasmid construction of DWB1, DWB2, DWB3, and DWB4</H4><P>Plasmids DWB1, DWB2, DWB3 and DWB4 were designed for gene integration into the chromosomal genome of <I>S. elongatus</I> PCC 7942. The plasmid AM2991 was constructed for homologous recombination into defined Neutral Integration Site 1 (NSI) in <I>S. elongatus</I> PCC 7492. The genes <I>ssuE</I> and <I>4ABH</I> were assembled into a AM2991 plasmid backbone constructing DWB1 and DWB2, respectively. The  plasmid AM1573 was used for recombination into Neutral Integration Site 2 (NSII). The genes <I>bluB</I> and <I>nhoA</I> were assembled into pAM1573 forming DWB3 and DWB4, respectively. All plasmids were successfully constructed and confirmed through sanger sequencing.</P><DIV class="container-3"><I>Plasmids were constructed through Gibson Assembly. Genes ssuE and 4ABH were inserted into pAM2991, while genes bluB and nhoA were inserted into pAM1573.</I></DIV><H4 style="text-align: left; font-weight: 400; font-size: 140%;">1b. Successful integration of <I>ssuE</I> into <I>S. elongatus</I> PCC 7942</H4><DIV class="diagram-image-center-large-1"><I>Homologous recombination of DWB1 plasmid into the chromosomal genome of <I>S. elongatus</I> PCC 7942.</I></DIV><DIV class="diagram-image-right-1"><I>Colony PCR of a single cell line transformed with DWB1. Amplification of the first neutral integration site in <I>S. elongatus</I> PCC 7942 shows two bands: the original unmodified genome at ~1690bp and ssuE integrated cassette into the chromosomal genome at ~6500bp. An isolated cell culture with successful integration into most of the genome is seen on Patch #4.</I></DIV><P>Integration of the <I>ssuE</I> gene into <I>S. elongatus</I> PCC 7942 genome was successful. Cells were inoculated with DWB1 plasmid and transformed in the dark, overnight. Transformed cells were selected for growth on BG-11 and streptomycin plates. Colonies were grown for 7-10 days and homologous recombination of the entire plasmid cassette into the genome was assayed through PCR of neutral site one.</P><P>Due to multiple copies of the chromosomal genome in the organism, <I>S. elongatus</I> PCC 7942 was patched onto antibiotic plates after colony picks, with the purpose of isolating cells that contain solely integrated gene constructs. After four rounds of patching a single cell line, specific amplification of a strand larger than that of the original genome indicates successful isolation of <I>ssuE</I> integrated cells.</P><P>Insertion of the gene <I>ssuE</I> into an organism expressing <I>bluB</I> has been documented to increase the expression of 5,6-dimethylbenzimidazole (5,6-DMB), the precursor for the mammalian usable version of vitamin B<SUB>12</SUB><SUP>[2]</SUP>. In order produce an abundance of 5,6-DMB within <I>S. elongatus</I> PCC 7942, DWB3 plasmid will be transformed and <I>bluB</I> will be integrated into cells that have been isolated with the optimized <I>ssuE </I>insert.</P><H2 style="text-align: left; font-weight: 500;">II. Growth</H2><DIV class="diagram-image-left-1"><I>Growth curve of S.elongatus PCC 7942 cultures grown in SCCB-1 (yellow) and BG-11(N+) (green). Absorbance with an OD<SUB>730</SUB> was taken daily.</I></DIV><H4 style="text-align: left; font-weight: 400; font-size: 140%;">2a. <I>S. elongatus</I> grown in local sea water, SCCB-1</H4><P><I>S. elongatus</I> PCC 7942 cultures are commercially grown in BG-11 media. However, seawater from a local beach was obtained, SCCB-1, to deduce whether locally sourced bodies of water were sufficient for the growth of <I>S. elongatus</I>. SCCB-1 was filtered, autoclaved and supplemented with a source of Nitrogen in the form of NaNO<SUB>2</SUB>.</P><P>To show the difference in growth with each media, SCCB-1 and BG-11(N+) were inoculated with <I>S. elongatus</I> PCC 7942 and cell density was measured daily for 10 circadian days. Cultures grown in SCCB-1 exhibit similar growth patterns to that of BG-11(N+) confirming the potential of locally sourcing a media for cyanobacterial growth.</P><H4 style="text-align: left; font-weight: 400; font-size: 140%;">2b. Growth Rates and Selection</H4><H4 style="text-align: center; font-weight: 300; font-size: 140%; width: 80%;">Growth Curves for Successive Inoculations of <I>S. elongatus</I> PCC 7942 at Various Wavelengths</H4><DIV class="readmore"><P style="text-align: center; font-size: 160% !important; background-color: white; width: 52%; border-radius: 24px;"><A href="#" id="example-show" class="showLink" onclick="showHide('example');return false;">Click here to view our growth data!</A></P><DIV id="example" class="more"><P style="text-align: center; font-size: 160% !important; background-color: white; width: 52%; border-radius: 24px;"><A href="#" id="example-hide" class="hideLink" onclick="showHide('example');return false;">Hide growth data</A></P><DIV class="text"><DIV class="container-3"><DIV class="row-3" style="margin-top: 30px;"><DIV class="col-md-12"><B style="font-size: 150%;"><I>Figure A</I></B><B style="font-size: 150%;"><I>Figure B</I></B><B style="font-size: 150%;"><I>Figure C</I></B><B style="font-size: 150%;"><I>Figure D</I></B></DIV></DIV><I>The observation of three consecutive cell cultures grown in BG-11(N+) showed a selection for cells that grew faster in BG-11(N+). <B>A.</B>) The average growth curves of a third passage inoculated in early August at OD<SUB>600</SUB>, OD<SUB>650</SUB>, OD<SUB>730</SUB>, and at OD<SUB>750</SUB> respectively. The cell culture is seen to enter log phase at day 7. <B>B.</B>) The average growth curves of a fourth passage inoculated in late August at OD<SUB>600</SUB>, OD<SUB>650</SUB>, OD<SUB>730</SUB>, and at OD<SUB>750</SUB> respectively. The cell culture is seen to enter log phase at day 6. <B>C.</B>) The average growth curves of a fourth passage inoculated in late October at OD<SUB>600</SUB>, OD<SUB>650</SUB>, OD<SUB>730</SUB>, and at OD<SUB>750</SUB> respectively. The cell culture is seen to enter log phase at day 4. <B>D.</B>) A Growth Curved generated from counting the cells in fourth passage cultures inoculated in mid and late October. All data taken for Graphs A - C were taken in the afternoon; however, Graph D data was collected in the early morning.</I></DIV></DIV></DIV></DIV><H4 style="text-align: left; font-weight: 400; font-size: 140%;">2c. Toxicity Tests</H4><P>A cell culture toxicity test suggested 13.23mM (10mg)  of acetaminophen to be detrimental to cell growth. While the amount of time at which the cell culture showed discoloration from green to nearly clear to brown varied in duplicate samples, both concluded with cell death and discoloration of cell sample. Cell culture containing 6.6mM (5mg) of acetaminophen showed inconsistent discoloration and cultures containing 1.32mM (1mg) of acetaminophen showed no significant color change.</P><P>Cultures were also tested with 1.8µM and 2.9µM of acetaminophen (to align with the expected amount based on a patent for the biosynthesis of acetaminophen in <I>E. coli</I><SUP>[1]</SUP>. No significant discoloration was observed. The discoloration and subsequent color change of the culture is suspected to be due to the degradation of acetaminophen into its precursor 4-aminophenol. Cell culture toxicity tests were performed with 4-aminophenol. Cell cultures were observed after the addition of 1.8µM, 9.1µM, and 18.3µM respectively. All samples with 4-aminophenol added turned from dark green to dark brown overnight. Further tests to determine the contents of the sample following the observed color change is recommended.</P><DIV class="container-2"><DIV class="row-3"><DIV class="col-md-6" style="padding-bottom: 25px;"><I><B>A.</B> Acetaminophen toxicity tests</I></DIV><DIV class="col-md-6" style="padding-bottom: 25px;"><I><B>B.</B> 4-aminophenol toxicity tests</I></DIV></DIV><I>Acetaminophen toxicity tests (<B>A</B>) showing color change at 13.23mM after 11 days and 4-aminophenol toxicity tests (<B>B</B>) yielding dramatic color change at 1.8µM, 9.1µM, and 18.3µM overnight.</I></DIV><H2 style="text-align: left; font-weight: 500;">III. Riboswitch</H2><DIV class="diagram-image-center-large-1"><I>Expression of the plasmid EN15 produces two strands of mRNA. In the absence of cobalamin, the Tet repressor is expressed hindering the expression of the reporter gene. In the presence of cobalamin, mRFP thus becomes expressed.</I></DIV><P>A detection mechanism for vitamin B<SUB>12</SUB>, specifically the molecule cobalamin, would allow for an instantaneous assay of its presence. The biobrick part, BBa_K1913011, from the Wageningen 2016 iGEM team was modified and tested for its affinity to 5,6-DMB-cobalamin (DMB B<SUB>12</SUB>) or adenine-cobalamin (B<SUB>12</SUB> analogs). The part consist of a riboswitch, btub, that binds specifically to cobalamin, sequestering the ribosomal binding site (RBS) upstream of the TetR gene, thereby hindering the expression of the repressor. In the absence of cobalamin, the TetR gene is constitutively expressed producing the Tet repressor which trans-regulates the Tet operator, TetO. The operator was designed upstream of the reporter gene, mRFP. Therefore, detection of the reporter RFP implies the presence of vitamin B<SUB>12</SUB>.</P><DIV class="diagram-image-right-1"><I>The expression of RFP is indicative of a functional riboswitch <B>or</B> the modified psbAI. psbAI is normally functional only in cyanobacteria and the modification of the -10 element allowed for transcription in <I>E coli</I>.</I></DIV><P>Plasmid EN15 was constructed through Gibson Assembly. Gene block fragments consisting of the riboswitch/reporter parts, NSII homologous sequences, and an ampicillin resistance were ligated onto an <I>E. coli</I> specific sequence, pBR322. The novel plasmid was designed for recombination into Neutral Integration Site 2 (NSII) within the chromosomal genome of <I>S. elongatus</I> PCC 7942. The riboswitch system was modified for expression within <I>S. elongatus</I> PCC 7942. A light induced psbAI promoter, documented to be constitutively expressed in cyanobacteria such as <I>S. elongatus</I> PCC 7942, was modified at the -10 element for expression in <I>E. coli</I>. The promoter was placed upstream of the biobrick parts, producing two separate mRNA strands.</P><P>The riboswitch system was modified to be expressed in <I>E. coli</I> as a proof that all parts work as expected. <I>E. coli</I> produces vitamin B<SUB>12</SUB> analogs which bind onto the riboswitch and express the mRFP reporter gene. Once the assembly of the plasmid EN15 was confirmed, it was then transformed into <I>E. coli</I> which further validated that all parts worked accordingly. Plasmid EN15 was confirmed via sanger sequencing.</P><H2 style="text-align: left; font-weight: 500;">VI. High Performance Liquid Chromatography (HPLC)</H2><P>To confirm the design of the plasmid DWB2, specifically the function of the <I>4ABH</I> gene, the plasmid was transformed into <I>E. coli</I>. However, since our gene block was codon optimized for <I>S. elongatus</I> PCC 7942, a Shine-Dalgarno sequence was inserted upstream of the <I>4ABH</I> gene.</P><P>HPLC (<A href="https://2017.igem.org/Team:UCSC/Notebook#verify-tag">conditions listed on the notebook page</A>) indicated that acetaminophen was not produced as expected. The positive control indicated that acetaminophen eluted at 4.7 minutes (Figure 1). The negative control indicated a baseline for the cell lysate and produced no peaks at 4.7 min (Figure 2). Chromatograms of cell cultures M1-M7 indicated no peak for acetaminophen at 4.7 minutes. The chromatogram of M7 was included as an example (Figure 3). The injection volume was 100µL for each sample.</P><DIV class="readmore"><P style="text-align: center; font-size: 160% !important; background-color: white; width: 52%; border-radius: 24px;"><A href="#" id="example1-show" class="showLink1" onclick="showHide1('example1');return false;">Click here to view our HPLC data!</A></P><DIV id="example1" class="more"><P style="text-align: center; font-size: 160% !important; background-color: white; width: 52%; border-radius: 24px;"><A href="#" id="example1-hide" class="hideLink1" onclick="showHide1('example1');return false;">Hide HPLC data</A></P><DIV class="text"><DIV class="container-3"><DIV class="row-3" style="margin-top: 30px;"><DIV class="col-md-12"><B style="font-size: 150%;"><I>Figure 1</I></B><B style="font-size: 150%;"><I>Figure 2</I></B><B style="font-size: 150%;"><I>Figure 3</I></B></DIV></DIV></DIV></DIV></DIV></DIV><P>These results indicate that the transformants did not produce acetaminophen. The transformants did not produce 4-aminophenol as expected. This could be due to a lack of <I>4ABH</I> expression, enzymatic activity to produce 4-aminophenol, or a lack of incorporation of the Shine-Dalgarno (SD) sequence into the DWB2 plasmid. Lack of incorporation of the SD sequence would result in lack of ribosomal binding sites and lead to no translation of the <I>4ABH</I> gene. Although the HPLC results showed no significant peak for acetaminophen, DWB2 was successfully transformed for <I>E. coli</I>. Additionally, acetaminophen and 4-aminophenol production was verified in <I>E. coli</I> through the insertion of the <I>4ABH</I> gene in other studies using similar methods<SUP>[1]</SUP>. Through these analyses, we were confident in moving forward with inserting out gene into <I>S. elongatus</I> PCC 7942.</P><P>In lieu of these results, the integration of the SD sequence into DWB2 will be re-done using a New England Biolabs<SUP>TM</SUP> (NEB<SUP>TM</SUP>) Q5<SUP>TM</SUP> Site-Directed Mutagenesis Kit. Primers have been re-designed for the process of integrating the SD sequence into the gene block in this method. <I>E. coli</I> DH5ɑ will be re-transformed and grown in M9M media, followed by HPLC to verify activity of the <I>4ABH</I> gene.</P><DIV class="container" style="width: 100%"><DIV class="row"><DIV class="col-md-4"><H3>P R O J E C T</H3><A href="https://2017.igem.org/Team:UCSC/Project" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">HOMEPAGE</DIV></DIV></A></DIV><DIV class="col-md-8"><H3>Click here to learn more!</H3><DIV class="row"><A href="https://2017.igem.org/Team:UCSC/Acetaminophen" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">ACETAMINOPHEN</DIV></DIV></A><A href="https://2017.igem.org/Team:UCSC/B-12" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">VITAMIN B<SUB>12</SUB></DIV></DIV></A><A href="https://2017.igem.org/Team:UCSC/Model" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">MODELING</DIV></DIV></A></DIV><DIV class="row"><A href="https://2017.igem.org/Team:UCSC/Target-Organism" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">TARGET ORGANISM</DIV></DIV></A><A href="https://2017.igem.org/Team:UCSC/Part_Collection" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">PARTS</DIV></DIV></A></DIV></DIV></DIV><DIV class="container"><DIV class="reference-list"><LI>[1] Anderson, J. C., HSIAU, T., Srivastava, S., RUAN, P., KOTKER, J. P. I., BODIK, R., &amp; Seshia, S. A. (2016). Method for biosynthesis of acetaminophen. Google Patents.</LI><LI>[2] Taga, M. E., Larsen, N. A., Howard-Jones, A. R., Walsh, C. T., &amp; Walker, G. C. (2007). BluB cannibalizes flavin to form the lower ligand of vitamin B12. Nature, 446(7134), 449–453. https://doi.org/10.1038/nature05611</LI></DIV></DIV></DIV></DIV></CENTER></DIV><FOOTER><DIV class="footer-box"><DIV class="col-md-5" style="padding-bottom: 40px;"><CENTER><DIV class="footer-section-title">S P O N S O R S</DIV><DIV class="col-md-2" style="padding-bottom: 40px;"><CENTER><DIV class="footer-section-title">S O C I A L </DIV></CENTER></DIV><DIV class="col-md-4" style="padding-bottom: 40px;"><CENTER><DIV class="footer-section-title">C O N T A C T</DIV><DIV class="address-of-our-team"><SPAN>Jack Baskin School of Engineering
    		University of California, Santa Cruz
    		1156 High Street
    		Santa Cruz, California, 95064</SPAN></DIV></CENTER></DIV></CENTER></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>