Team:UCSC/Demonstrate
MENU
JUDGING
NOTEBOOK
HUMAN PRACTICES
INTRODUCTION
SAFETY
APPLIED DESIGN
EDUCATION
COLLABORATIONS
SILVER
GOLD INTEGRATED
PROJECT
THE PROJECT
METABOLIC ENGINEERING
ACETAMINOPHEN
VITAMIN B12
MODELING
RESULTS
TARGET ORGANISM
PARTS
INTERLAB STUDY
TEAM
MEET THE TEAM
ATTRIBUTIONS
HOME
RESULTS
Our project focuses on biosynthetically producing the molecules vitamin DMB B12 and acetaminophen in Synechococcus elongatus PCC 7942. The secondary focuses of our project were geared towards analyzing the growth of S. elongatus PCC 7942 in seawater, creating a detection method for vitamin B12 via a riboswitch/reporter system, and a proof of concept for the production of acetaminophen in E. coli.
I. Biosynthesis
1a. Plasmid construction of DWB1, DWB2, DWB3, and DWB4
Plasmids DWB1, DWB2, DWB3 and DWB4 were designed for gene integration into the chromosomal genome of S. elongatus PCC 7942. The plasmid AM2991 was constructed for homologous recombination into defined Neutral Integration Site 1 (NSI) in S. elongatus PCC 7492. The genes ssuE and 4ABH were assembled into a AM2991 plasmid backbone constructing DWB1 and DWB2, respectively. The  plasmid AM1573 was used for recombination into Neutral Integration Site 2 (NSII). The genes bluB and nhoA were assembled into pAM1573 forming DWB3 and DWB4, respectively. All plasmids were successfully constructed and confirmed through sanger sequencing.
Plasmids were constructed through Gibson Assembly. Genes ssuE and 4ABH were inserted into pAM2991, while genes bluB and nhoA were inserted into pAM1573.
1b. Successful integration of ssuE into S. elongatus PCC 7942
Homologous recombination of DWB1 plasmid into the chromosomal genome of S. elongatus PCC 7942.
Colony PCR of a single cell line transformed with DWB1. Amplification of the first neutral integration site in S. elongatus PCC 7942 shows two bands: the original unmodified genome at ~1690bp and ssuE integrated cassette into the chromosomal genome at ~6500bp. An isolated cell culture with successful integration into most of the genome is seen on Patch #4.
Integration of the ssuE gene into S. elongatus PCC 7942 genome was successful. Cells were inoculated with DWB1 plasmid and transformed in the dark, overnight. Transformed cells were selected for growth on BG-11 and streptomycin plates. Colonies were grown for 7-10 days and homologous recombination of the entire plasmid cassette into the genome was assayed through PCR of neutral site one.
Due to multiple copies of the chromosomal genome in the organism, S. elongatus PCC 7942 was patched onto antibiotic plates after colony picks, with the purpose of isolating cells that contain solely integrated gene constructs. After four rounds of patching a single cell line, specific amplification of a strand larger than that of the original genome indicates successful isolation of ssuE integrated cells.
Insertion of the gene ssuE into an organism expressing bluB has been documented to increase the expression of 5,6-dimethylbenzimidazole (5,6-DMB), the precursor for the mammalian usable version of vitamin B12[2]. In order produce an abundance of 5,6-DMB within S. elongatus PCC 7942, DWB3 plasmid will be transformed and bluB will be integrated into cells that have been isolated with the optimized ssuE insert.
II. Growth
Growth curve of S.elongatus PCC 7942 cultures grown in SCCB-1 (yellow) and BG-11(N+) (green). Absorbance with an OD730 was taken daily.
2a. S. elongatus grown in local sea water, SCCB-1
S. elongatus PCC 7942 cultures are commercially grown in BG-11 media. However, seawater from a local beach was obtained, SCCB-1, to deduce whether locally sourced bodies of water were sufficient for the growth of S. elongatus. SCCB-1 was filtered, autoclaved and supplemented with a source of Nitrogen in the form of NaNO2.
To show the difference in growth with each media, SCCB-1 and BG-11(N+) were inoculated with S. elongatus PCC 7942 and cell density was measured daily for 10 circadian days. Cultures grown in SCCB-1 exhibit similar growth patterns to that of BG-11(N+) confirming the potential of locally sourcing a media for cyanobacterial growth.
2b. Growth Rates and Selection
Growth Curves for Successive Inoculations of S. elongatus PCC 7942 at Various Wavelengths
Click here to view our growth data!
Hide growth data
Figure A
Figure B
Figure C
Figure D
The observation of three consecutive cell cultures grown in BG-11(N+) showed a selection for cells that grew faster in BG-11(N+). A.) The average growth curves of a third passage inoculated in early August at OD600, OD650, OD730, and at OD750 respectively. The cell culture is seen to enter log phase at day 7. B.) The average growth curves of a fourth passage inoculated in late August at OD600, OD650, OD730, and at OD750 respectively. The cell culture is seen to enter log phase at day 6. C.) The average growth curves of a fourth passage inoculated in late October at OD600, OD650, OD730, and at OD750 respectively. The cell culture is seen to enter log phase at day 4. D.) A Growth Curved generated from counting the cells in fourth passage cultures inoculated in mid and late October. All data taken for Graphs A - C were taken in the afternoon; however, Graph D data was collected in the early morning.
2c. Toxicity Tests
A cell culture toxicity test suggested 13.23mM (10mg)  of acetaminophen to be detrimental to cell growth. While the amount of time at which the cell culture showed discoloration from green to nearly clear to brown varied in duplicate samples, both concluded with cell death and discoloration of cell sample. Cell culture containing 6.6mM (5mg) of acetaminophen showed inconsistent discoloration and cultures containing 1.32mM (1mg) of acetaminophen showed no significant color change.
Cultures were also tested with 1.8µM and 2.9µM of acetaminophen (to align with the expected amount based on a patent for the biosynthesis of acetaminophen in E. coli[1]. No significant discoloration was observed. The discoloration and subsequent color change of the culture is suspected to be due to the degradation of acetaminophen into its precursor 4-aminophenol. Cell culture toxicity tests were performed with 4-aminophenol. Cell cultures were observed after the addition of 1.8µM, 9.1µM, and 18.3µM respectively. All samples with 4-aminophenol added turned from dark green to dark brown overnight. Further tests to determine the contents of the sample following the observed color change is recommended.
A. Acetaminophen toxicity tests
B. 4-aminophenol toxicity tests
Acetaminophen toxicity tests (A) showing color change at 13.23mM after 11 days and 4-aminophenol toxicity tests (B) yielding dramatic color change at 1.8µM, 9.1µM, and 18.3µM overnight.
III. Riboswitch
Expression of the plasmid EN15 produces two strands of mRNA. In the absence of cobalamin, the Tet repressor is expressed hindering the expression of the reporter gene. In the presence of cobalamin, mRFP thus becomes expressed.
A detection mechanism for vitamin B12, specifically the molecule cobalamin, would allow for an instantaneous assay of its presence. The biobrick part, BBa_K1913011, from the Wageningen 2016 iGEM team was modified and tested for its affinity to 5,6-DMB-cobalamin (DMB B12) or adenine-cobalamin (B12 analogs). The part consist of a riboswitch, btub, that binds specifically to cobalamin, sequestering the ribosomal binding site (RBS) upstream of the TetR gene, thereby hindering the expression of the repressor. In the absence of cobalamin, the TetR gene is constitutively expressed producing the Tet repressor which trans-regulates the Tet operator, TetO. The operator was designed upstream of the reporter gene, mRFP. Therefore, detection of the reporter RFP implies the presence of vitamin B12.
The expression of RFP is indicative of a functional riboswitch or the modified psbAI. psbAI is normally functional only in cyanobacteria and the modification of the -10 element allowed for transcription in E coli.
Plasmid EN15 was constructed through Gibson Assembly. Gene block fragments consisting of the riboswitch/reporter parts, NSII homologous sequences, and an ampicillin resistance were ligated onto an E. coli specific sequence, pBR322. The novel plasmid was designed for recombination into Neutral Integration Site 2 (NSII) within the chromosomal genome of S. elongatus PCC 7942. The riboswitch system was modified for expression within S. elongatus PCC 7942. A light induced psbAI promoter, documented to be constitutively expressed in cyanobacteria such as S. elongatus PCC 7942, was modified at the -10 element for expression in E. coli. The promoter was placed upstream of the biobrick parts, producing two separate mRNA strands.
The riboswitch system was modified to be expressed in E. coli as a proof that all parts work as expected. E. coli produces vitamin B12 analogs which bind onto the riboswitch and express the mRFP reporter gene. Once the assembly of the plasmid EN15 was confirmed, it was then transformed into E. coli which further validated that all parts worked accordingly. Plasmid EN15 was confirmed via sanger sequencing.
VI. High Performance Liquid Chromatography (HPLC)
To confirm the design of the plasmid DWB2, specifically the function of the 4ABH gene, the plasmid was transformed into E. coli. However, since our gene block was codon optimized for S. elongatus PCC 7942, a Shine-Dalgarno sequence was inserted upstream of the 4ABH gene.
HPLC ( conditions listed on the notebook page ) indicated that acetaminophen was not produced as expected. The positive control indicated that acetaminophen eluted at 4.7 minutes (Figure 1). The negative control indicated a baseline for the cell lysate and produced no peaks at 4.7 min (Figure 2). Chromatograms of cell cultures M1-M7 indicated no peak for acetaminophen at 4.7 minutes. The chromatogram of M7 was included as an example (Figure 3). The injection volume was 100µL for each sample.
Click here to view our HPLC data!
Hide HPLC data
Figure 1
Figure 2
Figure 3
These results indicate that the transformants did not produce acetaminophen. The transformants did not produce 4-aminophenol as expected. This could be due to a lack of 4ABH expression, enzymatic activity to produce 4-aminophenol, or a lack of incorporation of the Shine-Dalgarno (SD) sequence into the DWB2 plasmid. Lack of incorporation of the SD sequence would result in lack of ribosomal binding sites and lead to no translation of the 4ABH gene. Although the HPLC results showed no significant peak for acetaminophen, DWB2 was successfully transformed for E. coli. Additionally, acetaminophen and 4-aminophenol production was verified in E. coli through the insertion of the 4ABH gene in other studies using similar methods[1]. Through these analyses, we were confident in moving forward with inserting out gene into S. elongatus PCC 7942.
In lieu of these results, the integration of the SD sequence into DWB2 will be re-done using a New England BiolabsTM (NEBTM) Q5TM Site-Directed Mutagenesis Kit. Primers have been re-designed for the process of integrating the SD sequence into the gene block in this method. E. coli DH5ɑ will be re-transformed and grown in M9M media, followed by HPLC to verify activity of the 4ABH gene.
P R O J E C T
HOMEPAGE
Click here to learn more!
ACETAMINOPHEN
VITAMIN B12
MODELING
TARGET ORGANISM
PARTS
[1] Anderson, J. C., HSIAU, T., Srivastava, S., RUAN, P., KOTKER, J. P. I., BODIK, R., & Seshia, S. A. (2016). Method for biosynthesis of acetaminophen. Google Patents.
[2] Taga, M. E., Larsen, N. A., Howard-Jones, A. R., Walsh, C. T., & Walker, G. C. (2007). BluB cannibalizes flavin to form the lower ligand of vitamin B12. Nature, 446(7134), 449–453. https://doi.org/10.1038/nature05611
S P O N S O R S
S O C I A L
C O N T A C T
Jack Baskin School of Engineering
University of California, Santa Cruz
1156 High Street
Santa Cruz, California, 95064
