<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCSC_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCSC/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="button-custom"><DIV class="tiny-little-menu-word-above-button">MENU</DIV></DIV><UL><LI><A href="https://2017.igem.org/Team:UCSC/Judging">JUDGING</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Notebook">NOTEBOOK</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Human_Practices">INTRODUCTION</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Safety">SAFETY</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Applied_Design">APPLIED DESIGN</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Engagement">EDUCATION</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/HP/Silver">SILVER</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/HP/Gold_Integrated">GOLD INTEGRATED</A></LI></UL><LI><A href="https://2017.igem.org/Team:UCSC/Project">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Project">THE PROJECT</A></LI><LI><A href="#">METABOLIC ENGINEERING</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Acetaminophen">ACETAMINOPHEN</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/B-12">VITAMIN B<SUB>12</SUB></A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Model">MODELING</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Demonstrate">RESULTS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Target-Organism">TARGET ORGANISM</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Part_Collection">PARTS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/InterLab">INTERLAB STUDY</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Team">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Team">MEET THE TEAM</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC">HOME</A></LI><DIV id="page"><CENTER><H1>MODELING</H1><DIV class="text-container"><DIV class="container" style="align-content: center;"><P style="text-align: center;"><SPAN class="quote">Predict and optimize yield</SPAN></P></DIV><H3>Background</H3><P>The purpose of modeling is to carefully examine the pathways of each intended biosynthetic product, look for ways to optimize production, and understand limiting factors. To accomplish these goals, we used available metabolic pathways for our target organism, and evaluated several different methods to model production of acetaminophen and B<SUB>12</SUB> in cyanobacteria. Each of these modeling methods has different assumptions which allow these data to be averaged; providing reasonable quantitative estimates of our biosynthetic products.</P><DIV class="container"><DIV class="row"><DIV class="col-md-6"><H3>ACETAMINOPHEN</H3></DIV></DIV></DIV><H2 style="text-align: left; font-weight: 500;">Acetaminophen Overview</H2><P>To predict acetaminophen biosynthesis, we analyzed the abundance of the acetaminophen's precursors anthranilate and PABA. Anthranilate comes from chorismate, which is primarily used by the cell to produce the aromatic amino acids phenylalanine, tyrosine, tryptophan, and folate<SUP>[1]</SUP>. We used publicly available tryptophan and folate concentrations, estimations from genomic sequences, and ribosomal protein sequences to estimate the amount of precursor available to be made into acetaminophen. Combining this information with enzyme kinetics, we were able to model how much product would be produced per gram biomass of <I>Arthrospira platensis</I>. Our results indicate that a backyard culture could produce 200 doses of acetaminophen every 10 days.</P><FIGCAPTION>Chorismate is processed into the aromatic amino acids, phenylalanine, tyrosine, tryptophan, and folate. Our inserted enzyme <I>4ABH</I> metabolize folate precursor PABA and tryptophan precursor anthranilate to make 4-aminophenol which is then processed by <I>nhoA</I> to make acetaminophen, top right.</FIGCAPTION><H2 style="text-align: left; font-weight: 500;">Method using Published Tryptophan Data</H2><P>We used published tryptophan and folate concentration data for <I>A. platensis</I> to calculate the moles of the acetaminophen precursors per gram dry mass<SUP>[7,8]</SUP>. Tryptophan and folate precursors anthranilate and PABA are the substrates for our enzyme <I>4ABH</I>, which converts those precursors to 4-aminophenol before <I>nhoA</I> finishes the conversion to acetaminophen. We show several different calculations below using different sources of data.</P><H2 style="text-align: left; font-weight: 400;">Assumptions</H2><UL style="font-family: 'objektiv-mk1'; font-size: inherit; text-align: left;"><LI>We assumed an even split of precursor at the branch points of PABA and anthranilate, even though enzyme K<SUB>m</SUB>s will affect the amount of precursor going down each pathway.</LI><LI>We assumed the moles of anthranilate produced are equal to the moles of tryptophan, because at least as many moles of anthranilate must be present to make the same ammount of moles tryptophan.</LI><LI>We assume that folate occurs in such small amounts, coming from 0.21nM of PABA, that its effect on acetaminophen production is neglegible<SUP>[7]</SUP>.</LI></UL>
          $$\frac{0.442\ µmol\ Trp}{1g\  biomass}\approx \frac{0.442\ µmol\ anth}{1g\ biomass}\times\frac{1\ mol\ acet}{3\ moles\ anth}\times\frac{151g\ acet.}{1 mol acet.}=\frac{2.3mg\ acet.}{1g\ biomass}$$
              <FIGCAPTION>This equation for acetaminophen (acet.) production using tryptophan (Trp) data in <I>Arthrospira platensis</I> gives us an approximation for acetaminophen's precursor anthranilate (anth.). The starting mass of tryptophan is the average of two literature values for tryptophan concentration<SUP>[7,8]</SUP> and assuming one third of the anthranilate precursor goes to our inserted enzyme, <I>4ABH</I>.</FIGCAPTION><H2 style="text-align: left; font-weight: 500;">Sequence Analysis Method</H2><P>To validate our organism's quantity of tryptophan precursor, we used a custom Python program<SUP>[9]</SUP> to convert DNA sequences to amino acids and calculate molar and mass percentages of tryptophan, which can be used to estimate the abundance of the acetaminophen precursor. </P><H2 style="text-align: left; font-weight: 400;">Assumptions</H2><UL style="font-family: 'objektiv-mk1'; font-size: inherit; text-align: left;"><LI>Using the entire genome sequence to approximate the cellular amino acid composition assumes each codon is expressed in equal concentrations. This does not account for some proteins being less expressed and the resulting skew in amino acid composition. </LI><LI>Using the ribosomal sequences assumes that the genes encoding ribosomal proteins are representative of the total cellular amino acids ratio.</LI><LI>The amount of anthranilate and PABA precursors produced are equal to that of their products tryptophan and folate.</LI><LI>Of the available precursors, one third will go down our pathway. This is based off the idea that the enzymes may have similar affinities for the precursors.</LI><LI>We assumed that folate occurs at a concentration of 0.21nM,<SUP>[7]</SUP> and comes from at least as many moles of PABA. Thus the small PABA concentration will be neglegible to the total acetaminophen produced.</LI></UL><H2 style="text-align: left; font-weight: 400;">Sequence Analysis Details</H2><P>We ran both the genome and all 55 listed ribosomal protein sequences through a custom Python program, which counted the instances of each amino acid in the sequence, totaled the mass, and calculated the percent tryptophan relative to total mass <SUP>[4]</SUP>. This program showed that tryptophan is 0.9% and 0.6% of total protein for the genome and ribosomal proteins respectively.</P>
          $$\frac{0.9\ g\ Trp}{100\ g\ cell\ protein}\times\frac{0.6\ g\ cell\ protein}{1\ g\ biomass}=\frac{0.054\ g\ Trp}{1g\ biomess}$$
          <FIGCAPTION>Trp is 0.9% by mass of total cellular protein, and cellular protein is 60% of dry biomass <SUP>[7]</SUP>. The percentage mass of Trp relative to cellular protein was converted to grams of trp per gram dry biomass. </FIGCAPTION>
          $$\frac{0.054\ g\ Trp}{1\ biomass}\rightarrow\frac{0.27\ mmol\ Trp}{1\ g\ biomass}\times\frac{1\ mol\ acet}{3\ mol\ Trp}\times\frac{151.163g\ acet}{1\ mol\ acet.}=\frac{13.7mg\ acet.}{1g\ biomass}$$
              <FIGCAPTION>This estimate is based on tryptophan quantities calculated by translating the organism's 3Mbp genome.</FIGCAPTION>
          $$\frac{0.6\ g\ Trp}{100\ g\ cell\ protein}\times\frac{0.6\ g\ cell\ protein}{1\ g\ biomass}=\frac{0.036\ g\ Trp}{1g\ biomess}$$
          <FIGCAPTION>In this ribosomal calculation, trp is 0.6% by mass of total cellular protein, and cellular protein is 60% of dry biomass <SUP>[7]</SUP>. The percentage mass of trp relative to cellular protein was converted to grams of trp per gram dry biomass. </FIGCAPTION>

          $$\frac{0.036 g\ Trp}{1 g\ biomass}\rightarrow \frac{0.175\ mmol\ Trp}{1\ g\ biomass} * \frac{1\ mol\ acet}{3\ mol\ chor} *\frac{151.163\ g}{1 mol\ acet} = \frac{8.8\ mg\ acet}{1 g\ biomass}$$
              <FIGCAPTION>This equation uses translated ribosomal amino acid composition to approximate total cellular amino acid composition since ribosomal proteins are highly expressed in cells, composing 9-22% of all proteins by mass<SUP>[10].</SUP></FIGCAPTION><H2 style="text-align: left; font-weight: 500;">Enzyme Competition</H2><P>Besides precursor concentrations, the next limiting factor would be how effective our enzyme <I>4ABH</I> is at converting precursors from the tryptophan and folate pathways into 4-aminophenol. We can either assume all three competing enzymes will have similar precursor affinity and produce all three products in equal quantities or use ratios of  competing enzymes K<SUB>m</SUB>s for the limiting intermediates. To make up for unknown enzyme rates and quantities, we made a chorismate metabolic simulation. Since the simulation used data based on K<SUB>m</SUB>s from completely different organisms, this enzyme kinetics comparison will be left for later troubleshooting. For that reason, all of the previous calulations were computed using a one third precursor to acetaminophen conversion rate.</P><H2 style="text-align: left; font-weight: 400;">Assumptions</H2><UL style="font-family: 'objektiv-mk1'; font-size: inherit; text-align: left;"><LI>We assume that our genes were designed, inserted, and translated successfully and in non-limiting quantities.</LI><LI>We assume K<SUB>m</SUB> values for <I>Arthrospira platensis</I> are the same as  other organisms' enzyme K<SUB>m</SUB>s.</LI><LI>We assume that enzyme rate and quantity is the same for each <I>4ABH, folp1, and TrpD</I>, else we could not compare K<SUB>m</SUB> ratios and calculate approximate the K<SUB>m</SUB> of <I>4ABH</I> for anthranilate.</LI></UL><H2 style="text-align: left; font-weight: 400;">Acetaminophen Conclusion</H2><P>These numbers show that there will probably be enough precursor to produce a useful, detectable quantity of acetaminophen.  Based on literature and sequence estimates of tryptophan, we can assume there would be at least that many moles of anthranilate precursor for our inserted pathway to push towards acetaminophen. The three calculations above can be averaged to finally predict 8.26mg ± 2.77mg acetaminophen per gram of <I>Arthrospira platensis</I> biomass or 8.26µg per mL. This would be significantly above the limit of detection for our HPLC, at 50ng per ml, and serve as a starting point for optimizing production. This means that one 325mg dose of acetaminophen could be obtained in ~39g of biomass, meaning a 3 by 12 foot round pool could produce enough acetaminophen for more than 200 doses every 10 days. While 39 grams isn't an ideal amount of medicine to consume, it does show that <I>Arthrospira platensis</I> has significant potential as a molecular factory for acetaminophen.</P><DIV class="container"><DIV class="row"><DIV class="col-md-6"><H3>VITAMIN B<SUB>12</SUB></H3></DIV></DIV></DIV><P>The quantity of DMB B<SUB>12</SUB> produced depends on a successful production and integration of the active B<SUB>12</SUB> lower ligand, composed of 5,6-dimethylbenzimidazole (5,6-DMB) and α-ribazole 5’-phosphate. For cyanobacteria in the wild, cobalt is often the limiting factor for growth and production of B<SUB>12</SUB><SUP>[26,27]</SUP>. Both <I>ssuE</I> and <I>bluB</I> genes are engineered to be regulated using a strong PtrC promoter, creating an abundance of the activating portion of the lower ligand, 5,6-DMB. Due to the abundant synthesis of 5,6-DMB, it should not act as the limiting factor for DMB B<SUB>12</SUB><SUP>[25]</SUP>. <I>Synechococcus elongatus</I> PCC 7942 and <I>Arthrospira platensis</I> C1 both have the <I>cobS</I> gene which codes for a protein that binds the α-ribazole within the 5,6-dimethylbenzimidazolyl nucleotide moiety to the cobalt-containing adenosylcobinamide-GDP complex<SUP>[14,15]</SUP>. Previous research involving the DMB B<SUB>12</SUB> pathway suggests that a lower ligand constructed with 5,6-DMB has a 100 fold higher binding affinity for cobalt than that of one constructed with adenine<SUP>[16, 17, 27]</SUP>. Due to the high binding affinity and abundance of 5,6-DMB as a lower ligand when compared to adenine, we assume a 100:1 ratio for DMB B<SUB>12</SUB> to adenine B<SUB>12</SUB>; for every 100 active DMB B<SUB>12</SUB> synthesized, there will be 1 inactive adenine B<SUB>12</SUB> synthesized.</P>
        $$\frac{2.5µg\ B_{12}\ analog}{1g\ drymass} * \frac{100\ DMB\ bindings}{100\ DMB\ bindings+1\ adenine\ binding} = \frac{2.47µg\ DMB\ B_{12}}{1g\ biomass}$$
    <FIGCAPTION>This equation shows an example of the calculation which models conversion from B<SUB>12</SUB> analogs to active DMB B<SUB>12</SUB>. This equation assumes the cobalt binding affinity ratio of <I>cobS</I> for DMB and adenine in <I>Arthrospira platensis</I> are the same as their orthologs present in the researched <I>Propionibacterium freudenreichii</I>.</FIGCAPTION><H2 style="text-align: left; font-weight: 400;">Assumptions</H2><UL style="font-family: 'objektiv-mk1'; font-size: inherit; text-align: left;"><LI>Cobalt is the primary limiting factor of DMB B<SUB>12</SUB> synthesis<SUP>[11,12]</SUP>.</LI><LI>The PtrC promoter is a strong promoter in <I>S. elongatus</I> 7942<SUP>[25]</SUP>.</LI><LI>Gene inserts will be expressed, oxidizing FMN to FMNH<SUB>2</SUB>, then catalyzing the synthesis of 5,6-DMB in excess<SUP>[13]</SUP>.</LI><LI>The <I>cobS</I> protein in <I>A. platensis</I> will attach the α-ribazole part of the 5,6-DMB lower ligand to cobalt-containing  adenosylcobinamide-GDP complex at rates similar to those assayed in <I>Propionibacterium freudenreichii</I><SUP>[14,15]</SUP>.</LI><LI>Cobalt will be provided in excess of 0.3mM according the BG-11 recipe, ensuring maximum precursor availability<SUP>[16,17]</SUP>.</LI><LI>There will be a 100:1 conversion of inactive adenine B<SUB>12</SUB> analogs to active DMB B<SUB>12</SUB> given the 100 fold higher binding affinity for 5,6-DMB as a lower ligand<SUP>[16, 17, 27]</SUP>.</LI></UL><H2 style="text-align: left; font-weight: 400;">B<SUB>12</SUB> Conclusion</H2><P>Published HPLC results show that <I>Arthrospira platensis</I> produces between 1.5-2.5µg B<SUB>12</SUB> analogs per gram dry weight<SUP>[18]</SUP>. Considering the assumed ratio for conversion of B<SUB>12</SUB> analogs to DMB B<SUB>12</SUB>, we predict 1.49-2.47µg DMB B<SUB>12</SUB> per gram dry weight of our engineered <I>A. platensis</I>, thus meeting the USDA’s recommended daily value of 6µg in one 2-4g serving.</P><DIV class="container" style="width: 100%"><DIV class="row"><DIV class="col-md-4"><H2>P R O J E C T</H2><A href="https://2017.igem.org/Team:UCSC/Project" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">HOMEPAGE</DIV></DIV></A></DIV><DIV class="col-md-8"><H2>Click here to learn more!</H2><DIV class="row"><A href="https://2017.igem.org/Team:UCSC/Acetaminophen" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">ACETAMINOPHEN</DIV></DIV></A><A href="https://2017.igem.org/Team:UCSC/B-12" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">VITAMIN B<SUB>12</SUB></DIV></DIV></A><A href="https://2017.igem.org/Team:UCSC/Demonstrate" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">RESULTS</DIV></DIV></A></DIV><DIV class="row"><A href="https://2017.igem.org/Team:UCSC/Target-Organism" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">TARGET ORGANISM</DIV></DIV></A><A href="https://2017.igem.org/Team:UCSC/Parts" class="proj-button"><DIV class="proj-button-desc"><DIV class="overlap-button-text">PARTS</DIV></DIV></A></DIV></DIV></DIV><DIV class="container"><DIV class="reference-list"><LI>[1] KEGG PATHWAY: Phenylalanine, tyrosine and tryptophan biosynthesis - Synechococcus elongatus PCC7942. (n.d.). Retrieved November 1, 2017, from http://www.genome.jp/kegg-bin/show_pathway?org_name=syf&amp;mapno=00400&amp;mapscale=&amp;show_description=hide</LI><LI>[2] Walsh, C. T., Haynes, S. W., &amp; Ames, B. D. (2012). Aminobenzoates as building blocks for natural product assembly lines. Nat. Prod. Rep., 29(1), 37–59. https://doi.org/10.1039/C1NP00072A</LI><LI>[3] Menezes, A. A., Cumbers, J., Hogan, J. A., &amp; Arkin, A. P. (2015). Towards synthetic biological approaches to resource utilization on space missions. Journal of The Royal Society Interface, 12(102), 20140715. https://doi.org/10.1098/rsif.2014.0715 </LI><LI>[4] Dschmelter. (2017). Chorismate acetaminophen simulation.py. Python. Retrieved from https://github.com/Dschmelter/bme160 (Original work published October 24, 2017)</LI><LI>[5] Tsuji, H., Ogawa, T., Bando, N., &amp; Sasaoka, K. (1986). Purification and properties of 4-aminobenzoate hydroxylase, a new monooxygenase from Agaricus bisporus. The Journal of Biological Chemistry, 261(28), 13203–13209.</LI><LI>[6] Rébeillé, F., Macherel, D., Mouillon, J. M., Garin, J., &amp; Douce, R. (1997). Folate biosynthesis in higher plants: purification and molecular cloning of a bifunctional 6-hydroxymethyl-7,8-dihydropterin pyrophosphokinase/7,8-dihydropteroate synthase localized in mitochondria. The EMBO Journal, 16(5), 947–957. https://doi.org/10.1093/emboj/16.5.947</LI><LI>[7] Food Composition Databases Show Foods -- Seaweed, spirulina, dried. (n.d.). Retrieved October 27, 2017, from https://ndb.nal.usda.gov/ndb/foods/show/3306?fgcd=&amp;manu=&amp;lfacet=&amp;format=Full&amp;count=&amp;max=50&amp;offset=&amp;sort=default\&amp;order=asc\&amp;qlookup=11667&amp;ds=&amp;qt=&amp;qp=&amp;qa=&amp;qn=&amp;q=&amp;ing=</LI><LI>[8] Narasimha, D. L. R., Venkataraman, G. S., Duggal, S. K., &amp; Eggum, B. O. (1982). Nutritional quality of the blue-green alga Spirulina platensis geitler. Journal of the Science of Food and Agriculture, 33(5), 456–460. https://doi.org/10.1002/jsfa.2740330511</LI><LI>[9] Dschmelter. (2017). AminoAcid Composition.py. Python. Retrieved from https://github.com/Dschmelter/bme160 (Original work published October 24, 2017)</LI><LI>[10] Dennis, P. P., &amp; Bremer, H. (1974). Macromolecular Composition During Steady-State Growth of Escherichia coli B/r. Journal of Bacteriology, 119(1), 270–281.</LI><LI>[11] Huang, H.-H., Camsund, D., Lindblad, P., &amp; Heidorn, T. (2010). Design and characterization of molecular tools for a Synthetic Biology approach towards developing cyanobacterial biotechnology. Nucleic Acids Research, 38(8), 2577–2593. https://doi.org/10.1093/nar/gkq164</LI><LI>[12] Deptula, P., Kylli, P., Chamlagain, B., Holm, L., Kostiainen, R., Piironen, V., … Varmanen, P. (2015). BluB/CobT2 fusion enzyme activity reveals mechanisms responsible for production of active form of vitamin B12 by Propionibacterium freudenreichii. Microbial Cell Factories, 14. https://doi.org/10.1186/s12934-015-0363-9 </LI><LI>[13] KEGG PATHWAY: Porphyrin and chlorophyll metabolism - Synechococcus elongatus PCC7942. (n.d.). Retrieved November 1, 2017, from http://www.genome.jp/kegg-bin/show_pathway?syf00860</LI><LI>[14] Anderson, P. J., Lango, J., Carkeet, C., Britten, A., Kräutler, B., Hammock, B. D., &amp; Roth, J. R. (2008). One pathway can incorporate either adenine or dimethylbenzimidazole as an alpha-axial ligand of B12 cofactors in Salmonella enterica. Journal of Bacteriology, 190(4), 1160–1171. https://doi.org/10.1128/JB.01386-07</LI><LI>[15] Stupperich, E., &amp; Nexø, E. (1991). Effect of the cobalt-N coordination on the cobamide recognition by the human vitamin B12 binding proteins intrinsic factor, transcobalamin and haptocorrin. European Journal of Biochemistry, 199(2), 299–303. https://doi.org/10.1111/j.1432-1033.1991.tb16124.x</LI><LI>[16] Bonnet, S., Webb, E. A., Panzeca, C., Karl, D. M., Capone, D. G., &amp; Wilhelmy, S. A. S. (2010). Vitamin B12 excretion by cultures of the marine cyanobacteria Crocosphaera and Synechococcus. Limnology and Oceanography, 55(5), 1959–1964. https://doi.org/10.4319/lo.2010.55.5.1959 </LI><LI>[17] Wang, K., Wommack, K. E., &amp; Chen, F. (2011). Abundance and Distribution of Synechococcus spp. and Cyanophages in the Chesapeake Bay▿. Applied and Environmental Microbiology, 77(21), 7459–7468. https://doi.org/10.1128/AEM.00267-11</LI><LI>[18] Watanabe, F., Katsura, H., Takenaka, S., Fujita, T., Abe, K., Tamura, Y., … Nakano, Y. (1999). Pseudovitamin B12 Is the Predominant Cobamide of an Algal Health Food, Spirulina Tablets. Journal of Agricultural and Food Chemistry, 47(11), 4736–4741. https://doi.org/10.1021/jf990541b</LI><LI>[20] Algae, U. C. C. of. (n.d.). BG-11 Trace Metals Solution Recipe. Retrieved November 1, 2017, from https://utex.org/products/bg-11-trace-metals-solution-recipe</LI><LI>[21] Deptula, P., Kylli, P., Chamlagain, B., Holm, L., Kostiainen, R., Piironen, V., … Varmanen, P. (2015). BluB/CobT2 fusion enzyme activity reveals mechanisms responsible for production of active form of vitamin B12 by Propionibacterium freudenreichii. Microbial Cell Factories, 14. https://doi.org/10.1186/s12934-015-0363-9</LI><LI>[22] Kuan, D., Duff, S., Posarac, D., &amp; Bi, X. (2015). Growth optimization of Synechococcus elongatus PCC7942 in lab flasks and a 2-D photobioreactor. The Canadian Journal of Chemical Engineering, 93(4), 640–647. https://doi.org/10.1002/cjce.22154</LI><LI>[23] Yan, R., Zhu, D., Zhang, Z., Zeng, Q., &amp; Chu, J. (2012). Carbon metabolism and energy conversion of Synechococcus sp. PCC 7942 under mixotrophic conditions: comparison with photoautotrophic condition. Journal of Applied Phycology, 24(4), 657–668. https://doi.org/10.1007/s10811-011-9683-2</LI><LI>[24] Moraes, I. de O., Arruda, R. de O. M., Maresca, N. R., Antunes, A. de O., &amp; Moraes, R. de O. (2013). Spirulina platensis: process optimization to obtain biomass. Food Science and Technology, 33, 179–183. https://doi.org/10.1590/S0101-20612013000500026</LI><LI>[25] Kim, W. J., Lee, S.-M., Um, Y., Sim, S. J., &amp; Woo, H. M. (2017). Development of SyneBrick Vectors As a Synthetic Biology Platform for Gene Expression in Synechococcus elongatus PCC 7942. Frontiers in Plant Science, 8, 293.</LI><LI>[26] Panzeca, C., Beck, A. J., Leblanc, K., Taylor, G. T., Hutchins, D. A., &amp; Sañudo-Wilhelmy, S. A. (2008). Potential cobalt limitation of vitamin B12 synthesis in the North Atlantic Ocean. Global Biogeochemical Cycles, 22(2), GB2029. https://doi.org/10.1029/2007GB003124</LI><LI>[27] Helliwell, K. E., Lawrence, A. D., Holzer, A., Kudahl, U. J., Sasso, S., Kräutler, B., … Smith, A. G. (2016). Cyanobacteria and Eukaryotic Algae Use Different Chemical Variants of Vitamin B12. Current Biology, 26(8), 999–1008. https://doi.org/10.1016/j.cub.2016.02.041 </LI></DIV></DIV></DIV></DIV></CENTER><FOOTER><DIV class="footer-box"><DIV class="col-md-5" style="padding-bottom: 40px;"><CENTER><DIV class="footer-section-title">S P O N S O R S</DIV><DIV class="col-md-2" style="padding-bottom: 40px;"><CENTER><DIV class="footer-section-title">S O C I A L </DIV></CENTER></DIV><DIV class="col-md-4" style="padding-bottom: 40px;"><CENTER><DIV class="footer-section-title">C O N T A C T</DIV><DIV class="address-of-our-team"><SPAN>Jack Baskin School of Engineering
    		University of California, Santa Cruz
    		1156 High Street
    		Santa Cruz, California, 95064</SPAN></DIV></CENTER></DIV></CENTER></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>