<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCSC_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCSC/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="button-custom"><DIV class="tiny-little-menu-word-above-button">MENU</DIV></DIV><UL><LI><A href="https://2017.igem.org/Team:UCSC/Judging">JUDGING</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Notebook">NOTEBOOK</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Human_Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Human_Practices">INTRODUCTION</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Safety">SAFETY</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Applied_Design">APPLIED DESIGN</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Engagement">EDUCATION</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/HP/Silver">SILVER</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/HP/Gold_Integrated">GOLD INTEGRATED</A></LI></UL><LI><A href="https://2017.igem.org/Team:UCSC/Project">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Project">THE PROJECT</A></LI><LI><A href="#">METABOLIC ENGINEERING</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Acetaminophen">ACETAMINOPHEN</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/B-12">VITAMIN B<SUB>12</SUB></A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Model">MODELING</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Demonstrate">RESULTS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Target-Organism">TARGET ORGANISM</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Part_Collection">PARTS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/InterLab">INTERLAB STUDY</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Team">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Team">MEET THE TEAM</A></LI><LI><A href="https://2017.igem.org/Team:UCSC/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2017.igem.org/Team:UCSC">HOME</A></LI><DIV id="page"><CENTER><H1>NOTEBOOK</H1></CENTER><DIV class="text-container"><CENTER><H2>TABLE OF CONTENTS</H2><DIV class="container"><OL><LI><A href="#modeling-tag">MODELING</A></LI><LI><A href="#growth-tag">GROWTH</A></LI><LI><A href="#plasmid-tag">PLASMID CONSTRUCTION</A></LI><UL><LI><A href="#DWB1-2-tag">DWB1 &amp; DWB2</A></LI><LI><A href="#DWB3-4-tag">DWB3 &amp; DWB4</A></LI></UL><LI><A href="#transform-tag"><I>S. ELONGATUS</I> 7942 TRANSFORMATIONS</A></LI><LI><A href="#riboswitch-tag">RIBOSWITCH</A></LI><LI><A href="#verify-tag">VERIFICATION OF <I>4ABH</I> GENE ACTIVITY IN <I>E. COLI</I></A></LI><LI><A href="#assay-tag">ASSAY</A></LI><LI><A href="#biobrick-tag">BIOBRICK ASSEMBLY</A></LI><LI><A href="#spirulina-tag"><I>ARTHROSPIRA PLATENSIS</I> SEQUENCING</A></LI><LI><A href="#interlab-tag">INTERLAB STUDY</A></LI></OL></DIV></CENTER></DIV><DIV class="text-container-wider"><H1 style="text-align: center; width: 100%;">MODELING</H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigators:</B> Daniel</H3><H3 style="text-align: center; width: 100%;"><B>Secondary Investigators:</B> Marissa, Tyler, Casidee, Evan, Khanh, McKenna, Tom, Xander, Brittney</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> The purpose of modeling is to predict product yield, determine feasibility of our project, and potential ways to increase production.</H3><DIV class="timeline"><DIV class="bubble-container left"><DIV class="bubble-content"><H2>J U N E</H2></DIV><DIV class="container-left-side"><UL><LI>Researched nutritional information for spirulina and <I>Synechococcus</I>, finding many different micronutrient levels which needed to be sorted through for quality.</LI><LI>Investigated disease burden associated with vitamin B<SUB>12</SUB>, iron, iodine, and vitamin A deficiency</LI><LI>Determined feasibility of bio-enriching our organisms to have increased iron, iodine, and beta-carotene content.</LI><LI>Researched protein production rate of our model organisms by referencing growth optimization papers, finding that <I>Synechococcus 7942</I> can reach maximum biomass of 1g/L in 7-10 days, while spirulina maximizes at 2g/L after approximately 15 days.</LI></UL></DIV></DIV><DIV class="bubble-container right"><DIV class="bubble-content"><H2>J U L Y</H2></DIV><DIV class="container-right-side"><UL><LI>Determined the factors that most impacted cyanobacteria are temperature, nitrogen, phosphorus, and iron.</LI><LI>Put those factors into an equation matching the approximate growth rate of our organism, formated this in Javascript and plotted interactively using Highcharts.</LI><LI>Contacted Abraxis Bioscience about getting our samples tested, finding that it would cost around $500 per sample, projecting costs of getting our B<SUB>12</SUB> conversion tested.</LI><LI>Found that chorismate products were primarily the aromatic amino acids, Phe, Trp, and Tyr by referencing the original NASA paper on pharmaceuticals in space.</LI></UL></DIV></DIV><DIV class="bubble-container left"><DIV class="bubble-content"><H2>A U G U S T</H2></DIV><DIV class="container-left-side"><UL><LI>We found several different data sets with amino acid concentrations for spirulina and tried to average the most reliable sources to estimate our aromatic amino acid concentration. With these numbers, we found how many moles of chorismate are available to make acetaminophen, giving our absolute ceiling of production.</LI><LI>Designed a python program to analyze the genomic and ribosomal sequences of <I>Synechococcus</I>, processing the data to find percentages of amino acids and relying on a papers describing macronutrient ratios to estimate chorismate mols per gram.</LI><LI>This gave us another estimate of acetaminophen, assuming a quarter of the chorismate goes to acetaminophen.</LI></UL></DIV></DIV><DIV class="bubble-container right"><DIV class="bubble-content"><H2>S E P T E M B E R</H2></DIV><DIV class="container-right-side"><UL><LI>To predict what proportion of our chorismate precursor would go to acetaminophen, we examined kinetic data, comparing K<SUB>M</SUB> ratios as an early estimate for how much of each product would be made.</LI><LI>Since no enzyme rates or quantities were available, we created a python program simulating the progress of each chorismate molecule along the metabolic pathway with and without our enzyme insertions.  We calibrated the python simulation to approximate actual percentages of rate times quantity of enzyme.</LI><LI>Calculated how much acetaminophen would be produced with the added genes, giving us another estimate using enzyme kinetics.</LI><LI>Reviewed the B<SUB>12</SUB> pathway and read papers on conversion of bacterial form B<SUB>12</SUB> to human-usable form.  Since 5,6-DMB has more than 100 times higher affinity for cobalamin than adenine, this means much of the bacterial B<SUB>12</SUB> would be converted to human-usable.  With several references for inactive B<SUB>12</SUB> in Spirulina, we got our first prediction of human-usable vitamin B<SUB>12</SUB>.</LI></UL></DIV></DIV><DIV class="bubble-container left"><DIV class="bubble-content"><H2>O C T O B E R</H2></DIV><DIV class="container-left-side"><UL><LI>Found folate to be a well regulated pathway, ensuring that our organism wouldn’t be growth limited by folate deficiency.</LI><LI>Found that cobalt is a major limiting factor for cyanobacterial growth in the ocean, suggesting that cobalt supplementation could make a our organisms produce more human usable B<SUB>12</SUB></LI><LI>More pathways research on PABA upregulation in <I>E. coli</I>. The research by Anderson et al encouraged us to pursue acetaminophen production in <I>E. coli</I> to provide evidence that our gene designs worked.</LI><LI>Discovered that our organism has the precursor anthranilate which can also be processed by our inserted gene <I>4ABH</I> into acetaminophen. This precursor of tryptophan is one step closer to the acetaminophen end product and is more abundant than the folate precursor, giving us another estimate of acetaminophen production.</LI></UL></DIV></DIV></DIV><H1 style="text-align: center; width: 100%;">GROWTH</H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigators:</B> Hailey, Sara, Khanh</H3><H3 style="text-align: center; width: 100%;"><B>Secondary Investigators:</B> Tom, Marissa</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> Monitoring the growth of <I>S. elongatus</I> PCC 7942 in specific and experimental conditions resulted in the collection of data for growth and modeling curves. Additionally, growth of <I>S. elongatus</I> PCC 7942 in local seawater proved for locally sourced media.</H3><DIV class="timeline"><DIV class="bubble-container left"><DIV class="bubble-content"><H2>J U N E</H2></DIV><DIV class="container-left-side"><UL><LI>Prepared BG-11 stock solutions</LI></UL></DIV></DIV><DIV class="bubble-container right"><DIV class="bubble-content"><H2>J U L Y</H2></DIV><DIV class="container-right-side"><UL><LI>Bought lights and timer to optimize 7942 growth conditions</LI></UL></DIV></DIV><DIV class="bubble-container left"><DIV class="bubble-content"><H2>S E P T E M B E R</H2></DIV><DIV class="container-left-side"><UL><LI>Observed effect of acetaminophen and 4-aminophenol in cell culture.</LI></UL></DIV></DIV><DIV class="bubble-container right"><DIV class="bubble-content"><H2>O C T O B E R</H2></DIV><DIV class="container-right-side"><UL><LI><B>October 31</B> - Wrapped up growth trials with cell counting and OD to monitor cell growth</LI></UL></DIV></DIV></DIV><H1 style="text-align: center; width: 100%;">DWB1 AND DWB2 PLASMID CONSTRUCTION</H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigators:</B> Jethro, McKenna, Casidee, Tyler</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> The goal of these experiments was to insert the <I>ssuE</I> and <I>4ABH</I> gene blocks into pAM2991. These new plasmids will be referred to as DWB1 and DWB2 respectively.</H3><DIV class="timeline"><DIV class="bubble-container left"><DIV class="bubble-content"><H2>J U L Y</H2></DIV><DIV class="container-left-side"><UL><LI>Designed gene blocks for <I>ssuE</I> and <I>4ABH</I>, as well as primers for linearization and sequencing</LI><LI>Ordered optimized gene insert</LI></UL></DIV></DIV><DIV class="bubble-container right"><DIV class="bubble-content"><H2>A U G U S T</H2></DIV><DIV class="container-right-side"><UL><LI>Linearization of pAM2991 via PCR</LI><LI>Gel extractions of plasmid linearizations</LI><LI>PCR on gel extractions to confirm bands</LI><LI>Gibson assembly of crude linear PCR product with gene blocks</LI><LI>Transformations of gibson reactions into <I>E. coli</I> Colony screening for successful DWB1 and DWB2 plasmid transformation via colony PCR</LI><LI>Completed Gibson assembly of DWB1 and DWB2</LI><LI>Confirmation of successful assembly via Sanger Sequence</LI><LI>Transformed constructed plasmids <I>S. elongatus</I> PCC 7942</LI></UL></DIV></DIV></DIV><H1 style="text-align: center; width: 100%;">DWB3 AND DWB4 PLASMID CONSTRUCTION</H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigators:</B> Mark, Sara</H3><H3 style="text-align: center; width: 100%;"><B>Secondary Investigators:</B> Alexander</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> The goal of these experiments was to insert the <I>bluB</I> and <I>nhoA</I> gene blocks into pAM1573. These new plasmids will be referred to as DWB3 and DWB4 respectively.</H3><DIV class="timeline"><DIV class="bubble-container left"><DIV class="bubble-content"><H2>J U L Y</H2></DIV><DIV class="container-left-side"><UL><LI>Designed DWB3 and DWB4, as well as primers for linearization and sequencing. Ordered <I>bluB</I> and <I>nhoA</I> gene blocks, primers, and pAM1573.</LI></UL></DIV></DIV><DIV class="bubble-container right"><DIV class="bubble-content"><H2>A U G U S T</H2></DIV><DIV class="container-right-side"><UL><LI><B>August 20</B> - The first step in constructing the DWB3 and DWB4 plasmids was linearizing pAM1573. This was done by performing PCR using primers that bound to positions 1,433bp and 1,710bp to create a 8,137bp PCR product. This was followed by a DpnI treatment to get rid of any methylated DNA (i.e. any circular plasmid that remained). To clean the PCR product we performed an ethanol precipitation.</LI><LI><B>August 23</B> - Performed a Gibson reaction using the <I>bluB</I> and <I>nhoA</I> gene blocks and then transformed DH5ɑ <I>E. coli</I> with it. The transformations were plated on LB + Chloramphenicol (170µg/mL) plates.</LI><LI><B>August 24</B> - There was colony growth and colony PCR showed band sizes larger than the PCR product of pAM1573 for two of the colonies (Figure 1). The colonies corresponding to lanes 5 and 9 were inoculated.</LI><DIV class="diagram-image-center-large-1"><I>Figure 1: Colony PCR results of E. coli transformed with Gibson reaction plasmids. Lane 1: 2-log ladder. Lane 2: E. coli transformed with untreated DWB4. Lanes 3-7: E. coli transformed with untreated DWB3. Lanes 8-9: E. coli transformed with DpnI-treated DWB4. Lane 10: pAM1573 control. Lane 11: no template control.</I></DIV><LI><B>August 24</B> - The inoculates corresponding to Figure 1 lanes 5 and 9 were mini-prepped and prepared for sequencing according to UC Berkeley’s sequencing specifications.</LI></UL></DIV></DIV><DIV class="bubble-container left"><DIV class="bubble-content"><H2>S E P T E M B E R</H2></DIV><DIV class="container-left-side"><UL><LI><B>September 1</B> - After receiving failed sequencing results on DWB3 and DWB4 we became suspicious of our results. Additional PCR on minipreps from liquid innoculations of colonies 5 and 9 did not reflect our previous findings. This prompted us to perform more Gibson reactions and transformations.</LI><LI><B>September 5 -- 20</B> - We performed several Gibson reactions followed by <I>E. coli</I> transformations and consistently got no colony growth for the <I>E. coli</I> transformed with the Gibson reactions; meanwhile, the positive control (<I>E. coli</I> transformed with pAM1573) always showed growth. This led us to believe that the chloramphenicol resistance gene in DWB3 and DWB4 was somehow defective.
          
          After taking a closer look at the sequence of pAM1573, we noticed that the terminators were incorrectly annotated in the reference maps AddGene makes available under “sequence information”; they were orientated in the wrong direction (Figure 2). Further exploration of the page identified the GenBank reference map underneath “Resource Information” contains the rrnB terminator labeled in the correct orientation.
          <DIV class="diagram-image-center-large-1"><I>Figure 2: Resolved annotation of pAM1573 (left) vs. AddGene “Full Sequences from Depository (1)” GenBank file that we used for reference (right). The rrnBT1 and rrnBT2 terminators are annotated in opposite directions. Both files are open source.</I></DIV>
          We believe our insert transcript was not terminated as expected since it was going in the opposite direction of the rrnB terminator, thus making a single transcript that included both our insert and <I>CmR</I>. This means that the transcript initiated at the AmpR promoter (at position 2,456bp) and ending at the rrnB terminator had complementarity to the transcript with our insert and <I>CmR</I>. It is possible that these two transcripts annealed to each other, resulting in double stranded RNA. Bacterial cells tend to destroy dsRNA longer than 30bp (since it signals a viral infection), thus destroying the transcript with <I>CmR</I> and causing the lack of colony growth of <I>E. coli</I> transformed with DWB3 and DWB4.</LI><LI><B>September 22</B> - To test our hypothesis of the destruction of <I>CmR</I>, we performed another Gibson and transformation, but this time we plated on LB + ampicillin plates (75µg/mL), since pAM1573 also has ampicillin resistance. There was colony growth on the plates containing transformed <I>E. coli</I> and no growth on the plate containing untransformed <I>E. coli</I> (i.e. <I>E. coli</I> without ampicillin resistance), suggesting our hypothesis was correct.</LI><LI><B>September 25</B> - Ordered new <I>bluB</I> and <I>nhoA</I> gene blocks containing a T0 terminator in the correct orientation.</LI></UL></DIV></DIV><DIV class="bubble-container right"><DIV class="bubble-content"><H2>O C T O B E R</H2></DIV><DIV class="container-right-side"><UL><LI><B>October 7</B> - Performed a Gibson reaction using the new <I>bluB</I> and <I>nhoA</I> gene blocks and then transformed DH5ɑ <I>E. coli</I> with it. The transformations were plated on LB + Chloramphenicol (170µg/mL) plates.</LI><LI><B>October 8</B> - There was colony growth and colony PCR showed band sizes larger than the PCR product of pAM1573 with no insert, suggesting the Gibson reaction was successful (Figure 3). The colonies corresponding to lanes 6, 7, and 8 were inoculated.</LI><DIV class="diagram-image-center-large-1"><I>Figure 3: Colony PCR of <I>E. coli</I> transformed with DWB3 (pAM1573 + <I>bluB</I> gene block containing T0 terminator) Gibson reaction. Lane 1: 2 log ladder. Lanes 2-5: colonies transformed with DWB4. Lanes 6-11: colonies transformed with DWB3. Lane 12: Negative control. Lane 13: PCR product of pAM1573 with no insert.</I></DIV><LI><B>October 9</B> - The inoculates corresponding to Figure 3 lanes 6, 7, and 8 were mini-prepped and prepared for sequencing according to UC Berkeley’s sequencing specifications.</LI><LI><B>October 13</B> - Sequencing results were received and analyzed. The colony from Figure 3 lane 6 had the best results, with only one mutation (Figure 4). Unfortunately, this mutation was problematic because it occurred in sequence encoding the <I>bluB</I> B12 binding site (Figure 5).
          
          To maximize our chances of obtaining an adequate DWB3 plasmid, we decided to perform another Gibson reaction as well as performing Q5 Site Directed Mutagenesis on the plasmid from the colony corresponding to lane 6.</LI><DIV class="diagram-image-center-large-1"><I>Figure 4: SNP found in sequence of <I>bluB</I> insert at position 516 for colony corresponding to Figure 3 lane 6. The G to A mutation would result in amino acid change from Aspartate (negatively charged R group) to Arginine (positively charged R group).</I></DIV><DIV class="diagram-image-center-large-1"><I>Figure 5: Structure of protein encoded by <I>bluB</I>. The SNP present in the sequence from the colony corresponding to Figure 3 lane 6 was problematic because it occurred at the position encoding the B<SUB>12</SUB> binding site, seen as the red spheres, and the change in amino acid charge would likely repel B<SUB>12</SUB>.</I></DIV><LI><B>October 15</B> - Performed a Gibson and transformation the same way as was done on 10/7/17. Also, designed and ordered primers for Q5 site directed mutagenesis.</LI><LI><B>October 16</B> - There was colony growth for the transformation performed with the new Gibson reaction and colony PCR showed band sizes larger than the PCR product of pAM1573 with no insert (Figure 6), suggesting the Gibson reaction was successful. The colonies corresponding to lanes 2, 9, 10 and 13 were inoculated.</LI><DIV class="diagram-image-center-large-1"><I>Figure 6: Colony PCR of <I>E. coli</I> transformed with new DWB3 and DWB4 (pAM1573 + gene blocks containing T0 terminator) Gibson reaction. Lane 1: 2 log ladder. Lanes 2-9: colonies transformed with DWB3. Lanes: 10-13: colonies transformed with DWB4. Lane 14: PCR product of pAM1573 with no insert.</I></DIV><LI><B>October 17</B> - The inoculates corresponding to Figure 3 lanes 2, 9, 10, and 13 were mini-prepped and prepared for sequencing according to UC Berkeley’s sequencing specifications.
          
          Performed Q5 Site directed mutagenesis on the plasmid from the colony corresponding to Figure 3 lane 6. This was used for transformation of <I>E. coli</I>.</LI><LI><B>October 18</B> - Sequencing results were received and analyzed. The colony from Figure 6 lane 9 had the best results for DWB3, with only one mutation (Figure 7). Fortunately, this mutation occurred in the rrnb terminator of the gene block insert, and thus the <I>bluB</I> transcript was unaffected. Both colonies for DWB4 had decent sequencing results (Figure 8). The colony corresponding to lane 10 only had one mutation, unfortunately since it was a deletion it would cause a frameshift. The colony corresponding to lane 13 showed several SNPs towards the end of the read, but fortunately they did not occur in the <I>nhoA</I> gene; also, the sequencing quality in this region was poor so it is possible that those SNPs are not actually there. These results show that DWB3 and DWB4 were successfully constructed.
          
          The plates with <I>E. coli</I> transformed with the Q5 site directed mutagenesis reaction showed growth, but nothing was done with those colonies since the sequencing results showed that we had constructed an adequate DWB3 plasmid.</LI><DIV class="diagram-image-center-large-1"><I>Figure 7: Sequencing result for the DWB3 colony corresponding to Figure 6 lane 9. SNP found in the sequence of the rrnb terminator of the <I>bluB</I> gene block insert. This mutation is unproblematic.</I></DIV><DIV class="diagram-image-center-large-1"><I>Figure 8: Sequencing results for the DWB4 colonies corresponding to Figure 6 lane 10 (top) and lane 13 (bottom).</I></DIV></UL></DIV></DIV></DIV><H1 style="text-align: center; width: 100%;"><I>S. ELONGATUS</I> 7942 TRANSFORMATIONS</H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigators:</B> Mckeñña, Casidee, Tyler, Jet</H3><H3 style="text-align: center; width: 100%;"><B>Secondary Investigators:</B> Sara, Mark</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> The goal of this is experiment was to isolate <I>S. elongatus</I> PCC 7942 that had successfully integrated genes into the genome.</H3><DIV class="timeline"><DIV class="bubble-container left"><DIV class="bubble-content"><H2>A U G U S T</H2></DIV><DIV class="container-left-side"><UL><LI>Began transformation of DWB1 &amp; DWB2 in <I>S. elongatus</I> PCC 7942</LI><LI>Colony PCR to screen for integration into 7942 genome</LI></UL></DIV></DIV><DIV class="bubble-container right"><DIV class="bubble-content"><H2>S E P T E M B E R</H2></DIV><DIV class="container-right-side"><UL><LI>Patch plated colonies for isolation of <I>ssuE</I> integrated cells</LI><LI>Perform colony PCR on streaks extracted from patch plates</LI><LI>Investigated agarose vs agar BG-11 plates</LI><LI>Investigated various antibiotic concentrations on BG-11 plates</LI></UL></DIV></DIV><DIV class="bubble-container left"><DIV class="bubble-content"><H2>O C T O B E R</H2></DIV><DIV class="container-left-side"><UL><LI>Establish <I>4ABH</I> as a toxic gene when integrated without <I>nhoA</I>; stop DWB2 transformations</LI><LI>Successfully isolated <I>ssuE</I> integrated <I>S. elongatus</I> PCC 7942 cells</LI><LI>Liquid inoculated <I>ssuE</I> integrated 7942</LI><LI>Began transforming <I>S. elongatus</I> PCC 7942 with DWB3, DWB1 + DWB3, and DWB4</LI></UL></DIV></DIV></DIV><H1 style="text-align: center; width: 100%;">RIBOSWITCH</H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigators:</B> Marissa, Jet</H3><H3 style="text-align: center; width: 100%;"><B>Secondary Investigators:</B> Mark</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> The goal of constructing a riboswitch and reporter system was for the purpose of having an instantaneous detection method for vitamin B<SUB>12</SUB>.</H3><DIV class="timeline"><DIV class="bubble-container right"><DIV class="bubble-content"><H2>S E P T E M B E R</H2></DIV><DIV class="container-right-side"><UL><LI>Research past iGEM biobrick parts</LI><LI>Investigate promoters usable for <I>S. elongatus</I> PCC 7942</LI><LI>Modified psbAI promoter for expression in <I>E. coli</I></LI></UL></DIV></DIV><DIV class="bubble-container left"><DIV class="bubble-content"><H2>O C T O B E R</H2></DIV><DIV class="container-left-side"><UL><LI>Gibson assembled riboswitch</LI><LI>Transformed riboswitch into <I>E. coli</I></LI><LI>Proved to detect vitamin B<SUB>12</SUB> analogs in <I>E. coli</I></LI></UL></DIV></DIV></DIV><H1 style="text-align: center; width: 100%;">VERIFICATION OF <I>4ABH</I> GENE ACTIVITY IN <I>E. COLI</I></H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigators:</B> Tyler, Jet, Xander, Tom</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> To confirm the design of the plasmid DWB2, specifically the function of the <I>4ABH</I> gene, the plasmid was transformed into <I>E. coli</I>. However, since our gene block was codon optimized for <I>S. elongatus</I> PCC 7942, a Shine-Dalgarno sequence was inserted upstream of the <I>4ABH</I> gene. The goal of this experiment was to verify the action of the 4-aminophenol production gene 4ABH as outlined in the Andersson patent.</H3><DIV class="timeline"><DIV class="bubble-container left"><DIV class="bubble-content"><H2>O C T O B E R</H2></DIV><DIV class="container-left-side"><UL><LI><B>References:</B></LI><LI>Anderson, J. C., HSIAU, T., Srivastava, S., RUAN, P., KOTKER, J. P. I., BODIK, R., &amp; Seshia, S. A. (2016). Method for biosynthesis of acetaminophen. Google Patents. Retrieved from <A href="https://www.google.com/patents/WO2016069155A2?cl=en">here</A></LI><LI>Daisuke Koma, Hayato Yamanaka, Kunihiko Moriyoshi, Kiyofumi Sakai, Takaya Masuda, Yoshihiro Sato, Kozo Toida &amp; Takashi Ohmoto (2014) Production of p- Aminobenzoic acid by metabolically engineered Escherichia coli, Bioscience, Biotechnology, and Biochemistry, 78:2, 350-357, DOI: <SPAN class="sequence-text">10.1080/09168451.2014.878222</SPAN></LI><LI><B>October 25</B> - A shine-dalgarno sequence was ligated to the <I>4ABH</I> gene block using a ligation kit. Resulting plasmid was transformed into DH5ɑ<I> E. coli</I>.</LI><LI><B>October 26</B> - Colonies were picked and inoculated into LB media as a pre cultures. These were named M1 through M10, and a positive and negative control.</LI><LI><B>October 27</B> - 12 Cell cultures of <I>E. coli</I> transformants were grown in LB media as pre-cultures.</LI><LI><B>October 30</B> - M9M media was prepared, and colonies from the preculture transformants were inoculated (5 mL).
            </LI><LI><B>M9M media is made with the following:</B></LI><OL><LI>10g/L glucose</LI><LI>6g/L Na<SUB>2</SUB>HPO<SUB>4</SUB></LI><LI>3 g/L KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>0.5 g/L NaCl</LI><LI>2g/L NH<SUB>4</SUB>Cl</LI><LI>246.5mg/L MgSO<SUB>4</SUB>·7H<SUB>2</SUB>O</LI><LI>14.7 mg/L CaCl<SUB>2</SUB>·2H<SUB>2</SUB>O</LI><LI>13.9 mg/L FeSO<SUB>4</SUB>·7H<SUB>2</SUB>O</LI><LI> 0.29 mg/L ZnSO<SUB>4</SUB>·7H<SUB>2</SUB>O</LI><LI>0.24 mg/L CoCl<SUB>2</SUB>·6H<SUB>2</SUB>O</LI></OL><LI><B>October 31</B> - HPLC was performed
            </LI><LI><B>Location:</B> This was done in the Macmillan lab at UCSC.</LI><LI><B>Purpose:</B> The purpose of running High-performance-liquid-chromatography was to identify the presence of acetaminophen as the product of the <I>4ABH</I> gene insert in <I>E. coli</I> DH5ɑ.</LI><LI><B>Column Conditions:</B></LI><LI>Column Description: Luna 5µm reverse-phase C18(2) 100A</LI><LI>Mobile Phase A: water</LI><LI>Mobile Phase B: methanol</LI><LI>UV detection: 254nm</LI><LI>Flow rate 1ml/min</LI><LI>Column temperature 23°C (not regulated).</LI></UL><LI><B><I>Preparation of samples and HPLC conditions:</I></B><B>Standards:</B></LI><LI>Acetaminophen (98%) was dissolved in methanol to make 10µg/mL, 500ng/mL, and 50ng/mL concentrations.</LI><LI>Injection volume: 10µL</LI><LI>Methylcobalamin (Me-cbl) was dissolved in 1:3 methanol:water.</LI><LI>Adenosylcobalamin (Ado-cbl) was dissolved in 1:1 methanol:water.</LI><LI>
                Conditions of the pumps were as follows:
                  </LI><LI>0.00 - 2.00 min ramp to 10% B</LI><LI>2.00 - 15.00 minutes ramp to 100% B</LI><LI>15.00 - 17.00 re-equilibriate to 10% B</LI><B>Cell cultures:</B><UL><LI><I>E. coli</I> DH5ɑ in M9M minimal media were lysed by bead beating (5 min), and prepared for HPLC by centrifugation (5000 rcf, 15 min) and filtration (0.22µm). The negative control was non-transformed <I>E. coli</I>. The positive control was 150 ng/mL acetaminophen in an aliquot of the negative control sample. Injection volumes were 100µL.</LI><LI>
                Phase conditions were phase A as water, phase B as methanol.
                  </LI><LI>0.00 - 2.00 min: ramp to 10% B</LI><LI>2.00 - 9.22 min: ramp to 100% B</LI><LI>9.30 - 11.30 min: hold at 100%</LI></UL><LI>The increase of 50% over 7.22 minutes was equivalent to the increase of 90% over 13 minutes.</LI></DIV></DIV></DIV><H1 style="text-align: center; width: 100%;">ASSAY</H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigators for B<SUB>12</SUB>:</B> Casidee, Jet, Marissa, McKenna</H3><H3 style="text-align: center; width: 100%;"><B>Primary Investigators for Acetaminophen:</B> Tyler, Xander</H3><H3 style="text-align: center; width: 100%;"><B>Secondary Investigators for B<SUB>12</SUB>:</B> Tyler, Xander</H3><H3 style="text-align: center; width: 100%;"><B>Secondary Investigators for Acetaminophen:</B> Khanh</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> To verify the success of our gene insertion by detecting for product in our model organisms.</H3><DIV class="timeline"><DIV class="bubble-container left"><DIV class="bubble-content"><H2>J U L Y</H2></DIV><DIV class="container-left-side"><UL><LI>Create assay protocol for acetaminophen and vitamin B<SUB>12</SUB> that will use the same column for HPLC.</LI></UL></DIV></DIV><DIV class="bubble-container right"><DIV class="bubble-content"><H2>S E P T E M B E R</H2></DIV><DIV class="container-right-side"><UL><LI>Order column</LI><LI>Order standards for vitamin B<SUB>12</SUB></LI></UL></DIV></DIV><DIV class="bubble-container left"><DIV class="bubble-content"><H2>O C T O B E R</H2></DIV><DIV class="container-left-side"><UL><LI>Extract vitamin B<SUB>12</SUB></LI><LI>Run standards of vitamin B<SUB>12</SUB> and purified acetaminophen</LI></UL></DIV></DIV></DIV><H1 style="text-align: center; width: 100%;">BIOBRICK ASSEMBLY</H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigators:</B> Tom</H3><H3 style="text-align: center; width: 100%;"><B>Secondary Investigators:</B> Pratibha, Khanh, Marissa</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> To submit our team registry parts for competition eligibility</H3><DIV class="timeline"><DIV class="bubble-container left"><DIV class="bubble-content"><H2>O C T O B E R</H2></DIV><DIV class="container-left-side"><UL><LI><B>I. Restriction Enzyme Digest</B>
          Both a double digest and a single digest were completed on pSB1C3. A double digest was completed on each of the gene inserts. The single digest was used to ensure correct banding of the double digest, the product of which we would use for the ligation of the pSB1C3 and DNA inserts.</LI><LI><B>Double Digest</B>
          The double digest protocol was adapted from NEB’s Single-temperature Double Digest protocol.
          <A href="https://www.protocols.io/view/Single-temperature-Double-Digest-imsuj5">(<SPAN class="sequence-text">dx.doi.org/10.17504/protocols.io.cjhuj5</SPAN>)</A></LI><DIV class="tg-wrap"><TABLE class="tg"><TBODY><TR><TH class="tg-hgcj">Components</TH><TH class="tg-amwm">Amounts</TH></TR><TR><TD class="tg-baqh">DNA</TD><TD class="tg-baqh">1µg</TD></TR><TR><TD class="tg-baqh">10x CutSmart Buffer</TD><TD class="tg-baqh">5µL</TD></TR><TR><TD class="tg-baqh">Xbal</TD><TD class="tg-baqh">1µL</TD></TR><TR><TD class="tg-baqh">Spel</TD><TD class="tg-baqh">1µL</TD></TR><TR><TD class="tg-baqh">H<SUB>2</SUB>O</TD><TD class="tg-baqh">Up to 50µL</TD></TR><TR><TD class="tg-baqh">Total Reaction</TD><TD class="tg-baqh">50µL</TD></TR></TBODY></TABLE></DIV><OL><LI>Mix reagents in a 1.5 mL microcentrifuge tube, adding the restriction enzyme last</LI><LI>Spin down the reaction in a microcentrifuge</LI><LI>Incubate the reaction at 37°C</LI></OL><LI><B>Single Digest</B>
            The single digest protocol was adapted from NEB’s Optimizing Restriction Endonuclease Reactions protocol. 
            <A href="https://www.neb.com/tools-and-resources/usage-guidelines/optimizing-restriction-endonuclease-reactions">(https://www.neb.com/tools-and-resources/usage-guidelines/optimizing-restriction-endonuclease-reactions)</A></LI><DIV class="tg-wrap"><TABLE class="tg"><TBODY><TR><TH class="tg-hgcj">Components</TH><TH class="tg-amwm">Amounts</TH></TR><TR><TD class="tg-baqh">DNA</TD><TD class="tg-baqh">1µg</TD></TR><TR><TD class="tg-baqh">10x CutSmart Buffer</TD><TD class="tg-baqh">5µL</TD></TR><TR><TD class="tg-baqh">Xbal/Spel</TD><TD class="tg-baqh">1µL</TD></TR><TR><TD class="tg-baqh">H<SUB>2</SUB>O</TD><TD class="tg-baqh">Up to 50µL</TD></TR><TR><TD class="tg-baqh">Total Reaction</TD><TD class="tg-baqh">50µL</TD></TR></TBODY></TABLE></DIV><OL><LI>Mix reagents in a 1.5 mL microcentrifuge tube, adding the restriction enzyme last</LI><LI>Spin down the reaction in a microcentrifuge</LI><LI>Incubate the reaction at 37°C</LI></OL><LI><B>II. Polymerase Chain Reaction</B>
          Primers made such that the flags contain the XbaI and SpeI sites on the gene for <I>ssuE</I></LI><LI><I>ssuE</I> fwd:
            <SPAN class="sequence-text">CATCATTCTAGATGAGTTATTTAATTATTGGTGCG</SPAN></LI><LI><I>ssuE</I> rev:
            <SPAN class="sequence-text">CATCATACTAGTAGGTACCTCAGCTAGTTATTGG</SPAN></LI><LI><I>4ABH</I> fwd:
            <SPAN class="sequence-text">CATCATTCTAGATGAGCCAGCAGGAACG</SPAN></LI><LI><I>4ABH</I> rev:
            <SPAN class="sequence-text">CATCATACTAGTAGGTACCTCAGCTAGCTACAG</SPAN></LI><LI>PCR Conditions</LI><DIV class="tg-wrap"><TABLE class="tg"><TBODY><TR><TH class="tg-yw4l">Component</TH><TH class="tg-yw4l">Amount</TH></TR><TR><TD class="tg-yw4l">10ng/μL</TD><TD class="tg-yw4l">1 μL</TD></TR><TR><TD class="tg-yw4l">10μM Forward primers</TD><TD class="tg-yw4l">2.5 μL</TD></TR><TR><TD class="tg-yw4l">10μM Reverse primers</TD><TD class="tg-yw4l">2.5 μL</TD></TR><TR><TD class="tg-yw4l">2x Q5 Master Mix</TD><TD class="tg-yw4l">25 μL</TD></TR><TR><TD class="tg-yw4l">H<SUB>2</SUB>O</TD><TD class="tg-yw4l">19 μL</TD></TR></TBODY></TABLE></DIV><LI>This PCR reaction was used to generate the gene inserts with the desired restriction digest sites to insert into pSB1C3.</LI><LI><B>III. Ligation</B></LI><LI>We adopted NEB’s protocol for ligation with T4 DNA Ligase
          <A href="https://www.protocols.io/view/Ligation-Protocol-WITH-T4-DNA-Ligase-M0202-imss4v">(<SPAN class="sequence-text">dx.doi.org/10.17504/protocols.io.cdks4v</SPAN>)</A>.
           
          This was used to insert the DNA parts into the pSB1C3 plasmid.</LI><DIV class="tg-wrap"><TABLE class="tg"><TBODY><TR><TH class="tg-9hbo">Component</TH><TH class="tg-9hbo">Amount (1:3 molar ratio)</TH><TH class="tg-9hbo">Amount (1:6 molar ratio)</TH></TR><TR><TD class="tg-yw4l">T4 DNA Ligase Buffer (10X)</TD><TD class="tg-yw4l">2 μL</TD><TD class="tg-yw4l">2 μL</TD></TR><TR><TD class="tg-yw4l">Vector DNA</TD><TD class="tg-yw4l">50 ng</TD><TD class="tg-yw4l">50 ng</TD></TR><TR><TD class="tg-yw4l">Insert DNA</TD><TD class="tg-yw4l">37.5 ng</TD><TD class="tg-yw4l">75 ng</TD></TR><TR><TD class="tg-yw4l">T4 ligase</TD><TD class="tg-yw4l">1 μL</TD><TD class="tg-yw4l">1 μL</TD></TR><TR><TD class="tg-yw4l">H<SUB>2</SUB>O</TD><TD class="tg-yw4l">Up to 20  μL</TD><TD class="tg-yw4l">Up to 20  μL</TD></TR><TR><TD class="tg-yw4l">Total</TD><TD class="tg-yw4l">20 μL</TD><TD class="tg-yw4l">20 μL</TD></TR></TBODY></TABLE></DIV><OL><LI>Mix reagents in a 1.5 mL microcentrifuge tube</LI><LI>Incubate at room temperature for 2 hours</LI><LI>Heat inactivate at 65°C for 10 minutes</LI></OL><LI><B>IV. Transformation—Chemi-competent with DH5-α <I>Escherichia coli</I></B></LI><OL><LI>Thaw DH5-α <I>E. coli</I> cells on ice</LI><LI>Mix 50μL of DH5-α <I>E. coli</I> cells with 2 μL of the ligation reaction</LI><LI>Ice for 30 minutes</LI><LI>Heat shock at 42°C for 30 seconds</LI><LI>Ice for 5 minutes</LI><LI>Add 350μL of room temperature SOC media</LI><LI>Incubate for 1 hour at 115 rpm and 37°C</LI><LI>Plate the contents of the microcentrifuge tube on a Luria Broth + Chloramphenicol (25ng/mL) plates</LI></OL><LI><B>V. Colony picks were completed</B></LI><LI><B>VI. Colony PCR</B></LI><LI>NEB’s Protocol for OneTaq 2X Master Mix with Standard Buffer was adapted for colony PCR. Colony PCR was used to identify picked colonies that had successfully taken up the desired plasmid.
          <A href="https://www.neb.com/protocols/2012/09/06/protocol-for-onetaq-2x-master-mix-with-standard-buffer-m0482">(<SPAN class="sequence-text">https://www.neb.com/protocols/2012/09/06/protocol-for-onetaq-2x-master-mix-with-standard-buffer-m0482</SPAN>)</A></LI><DIV class="tg-wrap"><TABLE class="tg"><TBODY><TR><TH>Components</TH><TH>Amount</TH></TR><TR><TD>OneTaq</TD><TD>5 μL</TD></TR><TR><TD>10 μM forward primer</TD><TD>1 μL</TD></TR><TR><TD>10 μM reverse primer</TD><TD>1μL</TD></TR><TR><TD>template</TD><TD>1 μL</TD></TR><TR><TD>H2O</TD><TD>2 μL</TD></TR><TR><TD>Total</TD><TD>10 μL</TD></TR></TBODY></TABLE></DIV><OL><LI>Mix the components for each reaction in a PCR tube</LI><LI>Transfer the PCR tubes into the thermocycler and thermocycle at the appropriate temperatures and times as specified in the protocol.</LI></OL><DIV class="diagram-image-center-large-1"><I>Figure 9: Gel for <I>ssuE</I> colony PCR</I></DIV><DIV class="diagram-image-center-large-1"><I>Figure 9: Gel for <I>4ABH</I> colony PCR</I></DIV><LI><B>VII. Inoculate successful colonies in Luria Broth and Chloramphenicol (25 μg/mL) media</B></LI><LI><B>VIII. Mini-prep the plasmid using the Zyppy<SUP>TM</SUP> Plasmid Miniprep Kit and standard protocol</B></LI></UL></DIV></DIV></DIV><H1 style="text-align: center; width: 100%;"><I>ARTHROSPIRA PLATENSIS</I> SEQUENCING</H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigator:</B> Marissa</H3><H3 style="text-align: center; width: 100%;"><B>Secondary Investigators:</B> Jet</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> An axenic strain of Spirulina, A. platensis UTEX 2340,  is being sequenced in an effort to help the scientific community gain a better understanding of A. platensis genetics. Sequencing the whole genome opens the possibility of applying the genetic modifications in PCC 7942 into A. platensis.</H3><DIV class="timeline"><DIV class="bubble-container left"><DIV class="bubble-content"><H2>A U G U S T</H2></DIV><DIV class="container-left-side"><UL><LI>Order <I>A. platensis</I> UTEX 2340</LI><LI>Grow the organism</LI></UL></DIV></DIV><DIV class="bubble-container right"><DIV class="bubble-content"><H2>S E P T E M B E R</H2></DIV><DIV class="container-right-side"><UL><LI>Filtration, pH treatment, antibiotic treatment, and serial dilutions</LI></UL></DIV></DIV><DIV class="bubble-container left"><DIV class="bubble-content"><H2>O C T O B E R</H2></DIV><DIV class="container-left-side"><UL><LI>Continuation of axenicity process/Troubleshooting contamination</LI></UL></DIV></DIV></DIV><H1 style="text-align: center; width: 100%;">INTERLAB STUDY</H1><H3 style="text-align: center; width: 100%;"><B>Primary Investigators:</B> Tom, Xander, Evan</H3><H3 style="text-align: center; width: 100%;"><B>Secondary Investigators:</B> Brittney</H3><H3 style="text-align: center; width: 100%;"><B>Purpose:</B> The purpose of this year’s InterLab Study is to address variations in fluorescence measurements, particularly with respect to green fluorescent protein (GFP).</H3><DIV class="timeline"><DIV class="bubble-container right"><DIV class="bubble-content"><H2>S E P T E M B E R</H2></DIV><DIV class="container-right-side"><UL><LI>See the <A href="https://2017.igem.org/Team:UCSC/InterLab">InterLab study main page</A> for more information on the processes cited below. All procedures were performed according to the recommended iGEM protocol.</LI><LI><B>September 20</B> - Day 1 of the <A href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">iGEM Plate Reader Protocol</A> was followed for calibrations of absorbance and fluorescence on the Perkin Elmer Envision 96-well format plate reader.
          
          Transformations of DH5α <I>E. coli</I> were performed according to the <A href="http://parts.igem.org/Help:Protocols/Transformation#Single_Tube_Transformation_Protocol">iGEM single tube transformation protocol</A>. Each device was transformed into DH5α <I>E. coli</I> and 2 concentrations of cells were plated for each device. These were called C1 and C2.
          
          The transformants were allowed to incubate overnight at 37°C.</LI><LI><B>September 21</B> - Day 2 of the <A href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">iGEM Plate Reader Protocol</A> for inoculation procedures of the transformants. One colony was picked each for C1 and C2 of each device. Colonies were inoculated in 10 mL LB medium + Chloramphenicol (25ug/mL).
          
          Cultures were incubated for 16 hrs at 37°C and 220 rpm.</LI><LI><B>September 22</B> - Day 3 of the <A href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">iGEM Plate Reader Protocol</A> was followed for cell measurements.
          
          This was done in the <A href="https://cscenter.pbsci.ucsc.edu/about/index.html">chemical screening center</A> on the UCSC campus.
          
          See the results posted in the <A href="https://2017.igem.org/Team:UCSC/InterLab"> UCSC interlab study page.</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><FOOTER><DIV class="footer-box"><DIV class="col-md-5" style="padding-bottom: 40px;"><CENTER><DIV class="footer-section-title">S P O N S O R S</DIV><DIV class="col-md-2" style="padding-bottom: 40px;"><CENTER><DIV class="footer-section-title">S O C I A L </DIV></CENTER></DIV><DIV class="col-md-4" style="padding-bottom: 40px;"><CENTER><DIV class="footer-section-title">C O N T A C T</DIV><DIV class="address-of-our-team"><SPAN>Jack Baskin School of Engineering
    		University of California, Santa Cruz
    		1156 High Street
    		Santa Cruz, California, 95064</SPAN></DIV></CENTER></DIV></CENTER></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>