"
Page
Discussion
View source
History
teams
Log in
 
Team:UANL Mty-Mexico/Modelling/Biphasic Switch
From 2011.igem.org
Team: UANL_Mty-Mexico
Home
Applications
Circuit
Mechanism
The Code
Overview
Project
Light Experiments
Integration
Circuit
Photocassette
Wet lab
Parameters
Quorum sensing
Biphasic Switch
Overview
Modelling
Parts
UANLBricks
Light Machine
Photochassis
Contributions
Safety
iGEM Lighters
Collaboration
Bio! Blog
Human Practices
Gallery
Hometown
Logo
Members
Team
Integration Protocol
Protocols
Notebook
Acknowledgments
Team: UANL_Mty-Mexico
Modelling: Biphasic switch
Biphasic Switch
One of the main elements of our project is the “Biphasic Switch”, as dubbed by Chris   A Voigt in his review Genetic Parts to Program a Bacteria (2006).
In our model, we include the affinity change due to the cooperativity effect and the dual behavior of cI. The affinity shift towards O2 and O3 is regulated by a logic rule regarding cI concentration, so when cI concentration is below its dissociation constant towards O1, the dissociation constant towards O2 and O3 remains at basal levels. But when cI concentration rises above its O1 dissociation constant, the cooperativity effect takes place and the dissociation constants towards O2 and O3 are lowered, increasing cI affinity.
Finally, we modeled the dual behavior of cI by considering operators O2 and O3 as if they were different promoters. Thus, their effect on the change of the regulated gene’s concentration is reflected in two different elements of the differential equations, one for activation at O2 and another for repression at O3. The different dissociation constants towards O1, O2 and O3, both for basal activity and for the cooperativity effect, were calculated from the total free energies for the binding of cI to the three operators published in Ackers GK, et al., (1982) and according to the formula presented there. The effect of cI on its regulated genes is then represented as follows:
The effect of cI on its regulated genes.
Where α is the maximum transcription rate; [cI] is the cI protein concentration; O2Kd is the dissociation constant of cI towards the O2 operator; O3Kd is the dissociation constant of cI towards operator O3; and µ is the degradation rate of the mRNA. Note that both the cI protein concentration and the dissociation constants have an exponent. This exponent is the Hill coefficient and equals two.
Chemically Induced Biphasic Switch
The figure 1 is a diagram of a Biphasic Circuit that responds to an IPTG input. The LacI gene is obviated.
Figure 1. Biphasic Circuit in response to an IPTG input. The LacI gene is obviated.
In brief, after the addition of IPTG to the culture, the LacI repression begins to cease, allowing cI to be expressed. This rise of cI concentration is supposed to be dependent on the IPTG input concentration, so that at low IPTG, the induction of cI is expected to be also low and in the range of the pRM activation state. On the other hand, when the IPTG input is high enough, the high cI induction will switch pRM into its repressed state.
The pRM is controlling the expression of two genes: the reporter GFP and another transcription factor cI434, which is negatively regulating the expression of another reporter gene, YFP. This reporter gene is also regulated by cI, but in a positive manner at any concentration considered. In this way, when the pRM promoter is active, GFP and cI434 will be expressed, giving rise to the first state of our Biphasic Switch, i. e., the green state. Conversely, when pRM is inhibited by the high concentration of cI, the inhibiton by cI434 will be gradually released and the second state will be observed, i. e., the yellow state (because of YFP).
Figure 2. Simulations of the IPTG induced Biphasic Switch at different IPTG concentrations. In blue is represent the simulations at IPTG 1300 nM; in red, IPTG 1950 nM; in green, IPTG 650 nM; and in cyan, 130 nM.
The time for the IPTG addition in all simulations is at 300 min. We considered a simulation time of 1000 min. just for the sake of clarity in the graphs, but the actual experimental times may be shorter. In all the graphs shown, the Y-axis represents protein concentration in nM units and the X-axis represents time in minutes.
The initial condition of the LacI concentration was determined in a simulation where, without any disturbance, the LacI gene was let to reach its saturation level. This saturation level is taken as an initial condition in order to avoid noisy behavior at the first minutes of simulation. Furthermore, in the wet-experiments, these simulated initial conditions may represent the incubation time previous the IPTG input, when the cells are allowed to grow, express LacI without disturbance and sustain the repression of the cI gene and of the Biphasic Switch as well. As seen, the green state is almost undetectable throughout the whole experiment, while the yellow one is activated with cI activation.
For cI to act as expected, i.e. a dual effect transcription factor, there must be some changes to the original circuitry, starting with modifications in LVA tags (all genes are considered to have one), RBS activity differences and perhaps some connectivity modifications.
The effect of RBS
The simulations in Figure 2 were performed considering that all genes have the same RBS efficiency. But in order to simulate the effect of different RBS, we assumed that protein translation is proportional to the activity of the specific RBS present in its mRNA. We decided to use the RBS Community Collection from the Registry of Standard Biological Parts ( http://partsregistry.org/Ribosome_Binding_Sites/Prokaryotic/Constitutive/Community_Collection ) because this set’s relative efficiencies have already been characterized.
We are aware that multiplying the maximum transcription rate times the relative RBS efficiency gives only a relative result, because the RBS characterization doesn’t report efficiencies in a nM concentration scale. The best approach would be to determine the ratio between the theoretical maximum translation rate and the translation rate for a given RBS, all in nM units. For this reason, the simulations of the RBS effects are reported in relative units, assuming that maximum transcription rate is achieved with RBS BBa_B0034, i.e. the reference RBS of the Community Collection. The RBS Relative Units are proposed as a unit measure for gene expression in our simulations, as the actual behavior of the protein concentration dynamics in nanomolar units can be also reflected in relative units, considering the basal signal as zero.
We decided to modify the RBS from cI, GFP and YFP. Protein cI was selected in order to maintain its concentration in a low range for some time, so that GFP inhibition can be delayed. Protein GFP was selected in order to increase its translation rate when using the most powerful RBS of the set. And finally, protein YFP was selected in order the noise it generates when the green state is turned on. Also, LacI LVA tag was removed, increasing its half-life. Furthermore, two LacI genes are considered in simulation D. The relative strengths of the RBS for the genes are the following:
Gene/RBS
Gene/RBS
Gene/RBS
Gene/RBS
Gene/RBS
LacI/1.124
LacI/1.124
LacI/1.124
2LacI/1.124
2LacI/1.124
cI/0.6
cI/0.3
cI/0.01
cI/0.01
cI/0.01
GFP/1.124
GFP/1.124
GFP/1.124
GFP/1.124
GFP/1.124
cI434/1.124
cI434/1.124
cI434/1.124
cI434/1.124
cI434/1.124
YFP/0.3
YFP/0.01
YFP/0.01
YFP/0.01
YFP/0.01
Simulations
Simulation E was performed considering an initial IPTG concentration of 285 nM, because, in a series of simulations, it was observed that GFP expression was greatly induced (8-10 nM steadily sustained) when working with IPTG in the 250-285 nM range.
Figure 3. RBS conditions A,B,C and D. In the four of them, the blue lines represent the 1300 nM IPTG simulation, the red line, the 650 nM IPTG and the green ones, 130 nM IPTG. In most cases, the blue and red lines hinder each other from sight, as they values may be very similar.
Figure 4. RBS conditions E. At time 0, the IPTG concentration is 0 nM; at minute 250, IPTG concentration is 285 nM; and finally, from minute 500 to the end of simulation, the IPTG concentration is 1300 nM.
The conditions of the simulation shown in figure 4 seem to fulfill for the expected behavior of the biphasic switch. That is, when IPTG concentration is null, the GFP and YFP expression remains basal and undetectable. But when it raises to a certain threshold (that was determined to be in the range of 250 – 285 nM IPTG by other, non-shown, simulations) and remains there, cI enters into its activator state and the GFP expression is turned on (to values of aprox. 10 RBS Relative Units), as well as cI434, while YFP remains basal. Then, when the IPTG concentration is incremented to a second threshold, cI starts acting as an inhibitor and turns off GFP and cI434 expression, while YFP expression is turned on to values around 10 RBS Relative Units, which are comparable to the GFP levels. Take into account that these values are calculated for a single cell, so they must be corrected for population density, as the fluorescence measured is the general signal from a sample.
In order to reset this chemically induced Biphasic Switch, the IPTG concentration must be set again to zero. This can be achieved by subculturing the IPTG induced culture into a series of fresh media cultures until no fluorescent signal is detected or by plating the culture and growing colonies in fresh media until, again, no fluorescence is detected. Another reset strategy is the Quorum Sensing connection by which a TetR gene is activated, so that it represses cI.
Red-Light Induced Biphasic Switch
Introduction
The red light system is based on a chimeric photoreceptor, which is composed by a photosensible domain and an EnvZ domain. This EnvZ domain is activated through darkness-stimulated autophosphorylation and then acts as a protein kinase, which in turn, phosphorylates and activates the transcription factor OmpR. Conversely, when red light is present, EnvZ will no longer exhibit a kinase activity, instead, it will act as a phosphatase that will inactivate the phosphorylated OmpR.
When phosphorylated, OmpR will turn on transcription on the PompC promoter, which in our system controls the expression of the repressor Mnt. Mnt is in turn repressing cI, so when red-light is present, Mnt expression will be decreased because the concentration of phosphorylated OmpR declines, and cI expression will be turned-on.
The expression of the chimeric photoreceptor and OmpR is given by the gene expression equations already shown. But the phosphorylation process is expressed by a set of equations and paramaters that were adapted from Srividhya and Krishnaswamy, (2004). These equations express the binding and dissociation of OmpR to EnvZ and, in our model, it is assumed that all the OmpR proteins that bind to EnvZ are phosphorylated. The parameters have been transformed from µM/second units to nM/minute.
After red-light exposure, all the EnvZ proteins in the membrane are assumed to be de-activated. Thus, these last equations are valid only when the light input is OFF.
When the cI gene is turned on using red light, the activity of the Biphasic Switch starts. The expression of the first state (GFP) is achieved through intermittent pulses of red light and far-red light; meanwhile, the second state is achieved through continuous exposure to red light.
Simulations
A number of simulations were performed in order to determine the appropriate time proportion for the cycles of red/far-red light exposure in the intermittent pulses. The next table shows the RBS strengths and the initial conditions that were considered:
Gene
RBS/Initial condition
(mRNA and protein)
Initial conditions
(mRNA) nM
nitial conditions
(protein) nM
Red photoreceptor
1
7.2109
1186.7
Mnt
1
56
24,500
cI
0.3
0
0
GFP
1
0
0
cI434
1
0
0
YFP
1
0
0
Figure 5 – A.- Biphasic Switch behavior in response to red-light stimuli. From min 0 to min 90, the system is in exposed to light, in order to reduce the phosphorylated OmpR levels as much as possible; from min 90 to min 600, the system is exposed to a series of cycles of 36 seconds of far-red light, followed by 59 min 24 seconds of red- light; finally, from min 600 to the end of the simulation, the red-light stimuli is continuously sustained. The light pulses only show the light activity after min 90. B.- Zoom to the cI graph from min 0 to 600; here, the oscillations due to the light pulses can be appreciated.
In brief, the red-light pulses induce the production of cI, but they do not last too long that the cI concentration rises to its maximum. In this way, cI production is induced during the red-light pulse, but then again repressed when the bacterial culture is exposed to far-red light. This periodic rises and declines of cI concentration cause, in turn, that the GFP reporter is expressed in this same periodic manner. On the other hand, when the red-light stimulus is continuously sustained, the cI gene then is also continuously expressed until it reaches its maximum, above the threshold for YFP activation. Thus, when light is continuously sustained, GFP oscillations are no longer observed, but YFP expression remains steady.
Note that it is assumed 100% efficiency for the light induction system. For this reason, we simulated a number of scenarios considering different efficiencies.
The system efficiency is defined as the ratio between 100 times the experimental cI mRNA levels and the theoretical ones:
The 100% efficiency cI mRNA levels are those obtained from the simulation on figure 5. In this way, the system behavior can be simulated at a given efficiency, no matter if the actual cI mRNA levels are not known.
Figure 6 – System behavior at different efficiencies, considering one Mnt gene. The green lines represent the GFP levels, while the red ones represent the YFP levels. The X-axis shows represents time in minutes and in the Y-axis the protein levels in RBS Relative Units. From min 90 to min 100, the system is exposed to cycles of 36 seconds far-red light and 59 min 24 seconds of red-light; from min 600 to the end of the simulation, the system is exposed to continuous red-light.
We performed a series of simulations for the red light-induced system either with one and two Mnt genes (LacI in the chemically-induced circuit and Mnt for the red light-induced circuit.
In the figure 6, it is shown a series of simulations considering different system efficiencies and a single Mnt gene. The lines representing GFP and YFP levels are shown in the same graph, so that they can be easily compared. During the light pulses (from min 90 to min 600) oscillating levels of the fluorescent proteins are observed. At system efficiencies ranging from 50 to 100%, the GFP oscillations can be quantitavely differentiated from the YFP noise. Below 40% efficiency, the cI levels are not enough to activate the first state of the Biphasic Switch.
Figure 7. System behaviour at different efficiencies, considering two Mnt genes. The green lines represent the GFP levels, while the red ones represent the YFP levels. The X-axis shows represents time in minutes and in the Y-axis the protein levels in RBS Relative Units. The light stimui are the same for figure 6.
On the other hand, for all efficiencies tested, the second state of the Biphasic Switch is achieved without GFP noise.
In figure 7, the simulations for the red light-induced system with two Mnt genes are shown. The state 1 of the Biphasic Switch was not observed for none of the system efficiencies tested; state 2, on the other hand, was observed for all of them.
Nevertheless, as the real system efficiency is not known and as an unknown efficiency reduction is expected, experiments for both the single- and double-Mnt systems are recommended.
References
Voigt CA (2006) Genetic parts to program bacteria. Curr Opin Biotechnol 17:548–557.
Srividhya KV, Krishnaswamy S (2004) A simulation model of Escherichia coli osmoregulatory switch using E-CELL system. BMC Microbiology 4:44.
Ackers GK, Johnson AD, Shea MA (1982) Quantitative model for gene regulation by A phage repressor. Proc Natl Acad Sci USA 79:1129-1133.
Biphasic Switch
Chemically Induced
Effect of RBS
Simulations
Red-light Induced
Simulations
References
Team: UANL_Mty-Mexico
Go back to top
Search
 
Contact us
Av. Manuel L. Barragan S/N, Cd. Universitaria. C.P.66450. San Nicolas de los Garza, Nuevo Leon, Mexico.
Retrieved from " http://2011.igem.org/Team:UANL_Mty-Mexico/Modelling/Biphasic_Switch "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
