<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_AHUT_China_Result skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:AHUT China/Result</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Royal Hotel</TITLE><TITLE>Royal Hotel</TITLE></P><DIV class="container"><NAV class="navbar navbar-expand-lg navbar-light"><DIV class="collapse navbar-collapse offset" id="navbarSupportedContent"><UL class="nav navbar-nav menu_nav ml-auto"><LI class="nav-item "><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/home">Home</A></LI><LI class="nav-item submenu dropdown"><A href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">TEAM</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Team
">Team Members</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Attributions">Attribution</A></LI></UL><LI class="nav-item submenu dropdown"><A href="#" style="color:#52C5FD" class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">PROJECT</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Overview">Overview</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Background">Background</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/protocol">Protocol</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/design">Design</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Demonstrate">Demonstrate</A></LI><LI class="nav-item"><A class="nav-link" href="
https://2018.igem.org/Team:AHUT_China/InterLab">Interlab</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Notebook">Notebook</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Result">Result</A></LI><LI class="nav-item submenu dropdown"><A href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">PARTS</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Improve">Improved parts</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/New_Parts">New Parts</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Model">Modeling</A></LI><LI class="nav-item submenu dropdown"><A href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Human_Practices">Overview</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/HP_FOR_SILVER">HP for Silver</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/INTEGRATED_FOR_GOLD">Integrated HP for gold</A></LI><LI class="nav-item"><A class="nav-link" href="
https://2018.igem.org/Team:AHUT_China/Public_Engagement">Public Engagement</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:AHUT_China/Collaborations">Collaboration</A></LI></DIV></NAV></DIV><SECTION class="latest_blog_area section_gap"><DIV class="container"><DIV class="section_title "><DIV align="center"><H2>Result</H2></DIV><H3>1. Molecular simulation</H3><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">Because wild-type human carbonic anhydrase 2 (CA2-WT) has the fastest reaction rate at 37 °C and loses its activity at 50 °C, so it may be not suitable for using wild type CA2 to capture CO<SPAN style="font-size: 13px">2</SPAN> under industrial operating conditions. Therefore, this project uses molecular simulation to design a new high-efficiency and stable carbonic anhydrase by improving its catalytic properties and biological stability for CO<SPAN style="font-size: 13px">2</SPAN> capture. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">First, with the help of computer-aided analysis software Discovery Visual Studio, we established the design principles of CA2 (Fig. 1 and Fig. 2): 1) Maintain the 3D structure of enzyme; 2) Modify the interactions between residues around active sites; 3) Improve the rigidity of active sites; 4) Shorten the distance of proton transfer. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;"> </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 1 human carbonic anhydrase 2 (CA2) </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 2 3D structure around the active site of CA2</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">Second, molecular docking of enzyme-substrate. Molecular docking with Autodock was performed to investigate the docking conformation of the substrate at the catalytic site and to analyze the interaction between the residues at the catalytic site and the substrate. Effects of the secondary and tertiary structure of the catalytic sites on the catalytic process were further investigated by using Autodock and Discovery Visual Studio. The mutation sites and substitution residues were set, and then the molecular docking of the recombinase was carried out to compare the enzyme-substrate docking conformation before and after recombination. Suitable mutation sites and replacement residues were selected to improve their catalytic properties. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">Third, enzyme-solvent kinetics simulation. Kinetic simulation was conducted by Gromacs software to investigate the conformation of the enzyme under aqueous solvent conditions at normal/high temperature conditions and to analyze the root mean square fluctuation of its individual residues. According to the results above, unstable residues were chosen to mutate, and the advanced structure of the enzyme and its rheology before and after recombination were further compared by Gromacs and Discovery Visual Studio software, then suitable mutation sites and replacement residues were selected to improve their stability. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">Basing on the simulation results above, we finally determined that the suitable mutation site of CA2 was L203K (the 203th Leucine mutated into lysine), as indicated in Fig. 3, the active site of mutant CA2 became compact at high temperature and gyration of radius was lower than wild-type CA2. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 3 Structure of Wild-type and mutant CA2 </P><H3>2. Engineered E. coli BL21(DE3)</H3><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">2.1 Construction of wild-type human carbonic anhydrase 2 (CA2-WT) and mutant human carbonic anhydrase 2 (CA2 (L203K)) expression plasmids 
The coding sequences of CA2-WT and CA2 (L203K) (<A href="
http://parts.igem.org/Part:BBa_K2547000">BBa_K2547000</A> and <A href="

http://parts.igem.org/Part:BBa_K2547004">BBa_K2547004</A>) were both synthesized, then cloned into the expression vector pET-30a(+), respectively. The correctness of the obtained recombinant vectors were identified by restriction enzyme digestion and sequencing (Fig. 4-7). </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 4 Map of CA2-WT recombinant vector</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 5 Agarose Gel Electrophoresis of CA2-WT recombinant plasmid and its identification by enzyme digestion. Lane M: DL marker; Lane 1: CA2-WT recombinant plasmid; Lane 2: enzyme digestion band of CA2-WT plasmid digested by MluⅠ, the length was 1028 bp (the arrow indicated).</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 6 Map of CA2 (L203K) recombinant vector</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 7 Agarose Gel Electrophoresis of CA2(L203K) recombinant plasmid and its identification by enzyme digestion (NdeⅠand Hind Ⅲ). Lane M: DNA marker; Lane 1: CA2 (L203K) recombinant plasmid; Lane 2: enzyme digestion band of CA2 (L203K), the length was 825 bp (the arrow indicated).</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">2.2 Induced expression of CA2-WT and CA2 (L203K) protein in E.coli BL21 (DE3) </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">Pilot expression of CA2-WT and CA2 (L203K) was carried out by inducing with isopropyl-1-thio-β-Dgalactopyrasonide (IPTG). Briefly, recombinant plasmids of the wild-type and mutant CA2 were transformed into E. coli BL21 (DE3), and positive clones were screened by kanamycin resistance. Then, the recombinant E. coli BL21 (DE3) were propagated and CA2 expression was induced with IPTG. Cells were lysed by sonication on ice, and the obtained crude extracts were centrifuged to separate supernatant and debris, and both fractions were subjected to SDS-PAGE and Western Blot. The results showed that both CA2-WT (Fig. 8 and Fig. 9) and CA2 (L203K) (Fig. 10) could be successfully expressed in BL21 (DE3) strain and existed in soluble form in the cell lysate supernatant. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 8 SDS-PAGE analysis for CA2-WT cloned in pET-30a(+) and expressed in BL21(DE3) strain.</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 9 Western blot analysis for CA2-WT cloned in pET-30a(+) and expressed in BL21(DE3) strain.</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 10 SDS-PAGE analysis for CA2 (L203K) cloned in pET-30a(+) vector and expressed in BL21(DE3) strain.</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">2.3 Purification of CA2-WT and CA2 (L203K) protein </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">After confirming that CA2-WT and CA2 (L203K) could be expressed in our chassis E. coli BL21 (DE3), protein of wild-type and mutant CA2 were further purified with nickel column for the following CO<SPAN style="font-size: 13px">2</SPAN> capture. As presented in Fig. 11 and 12, the results showed that both CA2-WT and CA2 (L203K) protein were purified with high purity as indicated by a significant single protein band after SDS-PAGE and Western blot. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;"> </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 11 Western blot analysis of CA2-WT protein. Lane M: Protein marker; Lane 1: Purified CA2-WT.</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 12 SDS-PAGE and Western blot analysis of CA2 (L203K) protein. Lane 1: Negative control; Lane 2: purified CA2 (L203K) protein.</P><H3>3. CO<STRONG>2</STRONG><STRONG> capture</STRONG></H3><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">3.1 Enzyme activity assay of CA2-WT and CA2 (L203K) protein </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">In order to confirm the forecast predicted by molecular simulation analysis, the enzyme activity of both wild-type and mutant CA2 protein were further tested experimentally by colorimetric and esterase assay. As indicated in Fig. 13, specific activity of mutant CA2 was about 2 times greater than that of wild-type enzyme. The kinetic constants (Km and Vmax) were calculated for esterase activity assay, and the result showed that CA2 (L203K) protein has a higher activity than CA2-WT (Fig. 14). </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 13 Colorimetric assay of CA2 activity</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 14 Esterase activity analysis of CA2 protein</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">3.2 Thermal stability studies of CA2-WT and CA2 (L203K) protein </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">In order to verify that our mutant CA2 can be applied to higher temperature for absorbing CO<SPAN style="font-size: 13px">2</SPAN> under industrial operating conditions. We investigated the effect of temperature on CA2 activity by esterase activity assay. As shown in Fig. 15, as the temperature increases, especially at 55 °C and 65 °C, the enzymatic activity of CA2-WT was significantly decreased, while the mutant CA2 still retain relatively high activity, indicating that CA2 (L203K) was more stable at high temperature and retained its activity. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 15 Activity of purified CA2-WT and CA2 (L203K) protein under indicated temperatures and time points.</P><H3>4. Improvement</H3><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">4.1 Characterization of an existing BioBrick Part 
<A href="http://parts.igem.org/Part:BBa_K2232000">BBa_K2232000</A>  (TSLV1-CA) </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">For characterization, we have demonstrated the output of this part <A href="http://parts.igem.org/Part:BBa_K2232000">BBa_K2232000</A> , the coding sequence (CDS) of Carbonic anhydrase (CA) from the polyextremophilic bacterium Bacillus halodurans TSLV1 (MTCC 10961, 16S rDNA Acc. No. HQ235051), in our chassis E. coli BL21(DE3). </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">The sequence of 
<A href="http://parts.igem.org/Part:BBa_K2232000">BBa_K2232000</A>  was synthesized and cloned it into the expression plasmid pET-30a(+) to obtain the recombinant expression vector (Fig. 16). </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 16 Agarose Gel Electrophoresis of TSLV1-CA recombinant plasmid and its identification by PCR. Lane M: DL marker; Lane 1: TSLV1-CA recombinant plasmid; Lane 2: PCR band of TSLV1-CA, the length was 894 bp.</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">Then, the TSLV1-CA expression plasmid was transformed into E. coli BL21 (DE3) strain, and positive clones were screened by kanamycin resistance. The positive clones were further propagated and induced with IPTG, followed by protein extraction from lysates of bacterial solution. The expression of TSLV1-CA was identified by Western blot analysis. The results are shown in Fig. 17, indicating that the coding sequence of <A href="http://parts.igem.org/Part:BBa_K2232000">BBa_K2232000</A>  can be expressed in our chassis E. coli BL21 (DE3). </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 17 Western blot analysis of protein extracted from lysates of TSLV1-CA expressed E.coli BL21(DE3) strain</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">4.2 Improve a Previous Part </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">We have changed the sequences of the existing part Carbonic anhydrase (csoS3) of the carboxysome of Halothiobacillus neapolitanus (<A href="http://parts.igem.org/Part:BBa_K1465205">BBa_K1465205</A>), and have generated a new Part 
<A href="http://parts.igem.org/Part:BBa_K2547003">BBa_K2547003</A> (Carbonic anhydrase (csoS3)-His) (Fig. 18) </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 18 Map of Carbonic anhydrase csoS3-His expression vector</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">Specifically, the coding sequence of Carbonic anhydrase csoS3 from original part was codon-optimized, and also a His tag was added to the end, to ensure that Carbonic anhydrase csoS3 could be expressed in E. coli BL21 (DE3) and retained potent carbonic anhydrase activity. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">First, the original coding sequence of csoS3 and the coding sequence with codon optimization were synthesized, and cloned into the pET-30a (+) expression vectors, respectively. The correctness of the two recombinant plasmids was verified by PCR (Fig. 19). </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 19 Agarose Gel Electrophoresis of Carbonic anhydrase csoS3 expression vectors and its identification by PCR. Lane M: DL marker; Lane 1: expression vector of csoS3 new part; Lane 2: PCR band of expression vector of csoS3 new part, the length was 1620 bp; Lane 3: expression vector of csoS3 original part; Lane 4: PCR band of expression vector of csoS3 original part, the length was 1620 bp.</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">Subsequently, the expression of two csoS3 plasmids in E. coli was detected via SDS-PAGE and Coomassie blue staining. As shown in Fig. 20, the result presented that the expression of csoS3 original part in E. coli was relatively low, and the expression of codon-optimized csoS3 new part in E. coli was higher than original part. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 20 SDS-PAGE and Coomassie blue staining of Carbonic anhydrase csoS3 plasmids expressed in E. coli BL21(DE3) strains. The arrow indicated was the bands of csoS3. Lane 1: Negative control (cell lysate without IPTG induction) of new part; Lane 2: Cell lysate with induction for 6 h at 37 ℃ of new part; Lane 3: Negative control (cell lysate without IPTG induction) of original part; Lane 4: Cell lysate with induction for 6 h at 37 ℃ of original part.</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">To further demonstrate the activity of our new part, new part of csoS3 carbonic anhydrase was purified through Ni-chelating affinity chromatography and detected by SDS-PAGE and Coomassie blue staining, as shown in Fig. 21. Then, the activity of csoS3 was measured via esterase method, and the enzyme activity was about 22.84 U/mL. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 14px;text-align: center;">Fig. 21 SDS-PAGE analysis of purified Carbonic anhydrase csoS3 protein.</P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">In conclusion, our results demonstrated that the function of csoS3 new part has been improved with higher expression and activity than original part. </P><H3>References</H3><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">[1] Savile C K, Lalonde J J. Biotechnology for the acceleration of carbon dioxide capture and sequestration[J]. Curr Opin Biotechnol, 2011, 22(6):818-82322. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">[2] Rahman F A, Aziz M M A, Saidur R, et al. Pollution to solution: Capture and sequestration of carbon dioxide (CO<SPAN style="font-size: 13px">2</SPAN>) and its utilization as a renewable energy source for a sustainable future[J]. Renewable &amp; Sustainable Energy Reviews, 2017, 71:112-126. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">[3] Hu G, Smith K H, Nicholas N J, et al. Enzymatic carbon dioxide capture using a thermally stable carbonic anhydrase as a promoter in potassium carbonate solvents[J]. Chemical Engineering Journal, 2017, 307:49-55. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">[4] Yong J K J, Stevens G W, Caruso F, et al. The use of carbonic anhydrase to accelerate carbon dioxide capture processes[J]. Journal of Chemical Technology &amp; Biotechnology, 2015, 90(1):3-10. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">[5] Capasso C, De L V, Carginale V, et al. Biochemical properties of a novel and highly thermostable bacterial α-carbonic anhydrase from Sulfurihydrogenibium yellowstonense YO3AOP1[J]. Journal of Enzyme Inhibition &amp; Medicinal Chemistry, 2012, 27(6):892. </P><P style="font-family: 'Arial Unicode MS', 'Microsoft YaHei UI', 'Microsoft YaHei UI Light', '华文细黑', '微软雅黑', '幼圆';  font-size: 18px;">[6] Deryanur KILIÇ, Orhan ERDOĞAN, Ömer İrfan KÜFREVİOĞLU. Eﬀect of mutation in active site residue Trp209 to Val, Leu, Ile and Pro on the catalytic activity and affinity for some benzenesulfonamides of human carbonic anhydrase II[J]. Turkish Journal of Biology, 2017, 41: 835-842. </P></DIV></DIV></SECTION></DIV><DIV id="index_footer"><DIV class="conentff"><UL id="index_foot_list"><LI>TEAM</LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Attributions">Attribution</A></LI></UL><LI>PROJECT</LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/protocol">Protocol</A></LI><LI><A href="href=" https:="">Design</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Result">Result</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Notebook">Notebook</A></LI><LI>Parts</LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Improve">Improved Parts</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/New_Parts">New Parts</A></LI><LI>Modeling</LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Model">Model</A></LI><LI>HP</LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/HP_FOR_SILVER">HP for Silver</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/INTEGRATED_FOR_GOLD">Integrated HP for Gold</A></LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Public_Engagement">Public Engagement</A></LI><LI>Collaboration</LI><LI><A href="https://2018.igem.org/Team:AHUT_China/Collaborations">Collaboration</A></LI><DIV id="index_foot_ahut_info"><P>Anhui University of Technology is located in the national civilized city, Nanjing metropolitan core cities - Ma'anshan City, Anhui Province. Anhui University of Technology is an engineering, metallurgical features, engineering, economics, management, arts, science, law and other disciplines coordinated development of key construction in Anhui multidisciplinary university, with &quot;The Master, Bo&quot; complete the multi-level education training system.</P></DIV><DIV align="center">Copyright 2018 @ AHUT iGEM Team , All rights reserved . </DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>