<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ITB_Indonesia_Data"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ITB_Indonesia/Data">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ITB_Indonesia/Data&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ITB_Indonesia/Data&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ITB_Indonesia/Data&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ITB_Indonesia/Data" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ITB Indonesia/Data</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="putih"><DIV id="center"><DIV id="blockNav"><UL><LI><A href="https://2014.igem.org/Team:ITB_Indonesia">HOME</A></LI><LI>TEAM
						</LI><LI><A href="https://igem.org/Team.cgi?id=1387">OFFICIAL TEAM PROFILE</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Team">TEAM MEMBER</A></LI></UL><LI>PROJECT
						</LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Project">OVERVIEW</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/DegMod">DEGRADATION MODULE</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/ConMod">CONVERTION MODULE</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/RepMod">REPORTER MODULE</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/SelfMod">SELF REGULATORY MODULE</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/FutureSystem">FUTURE SYSTEM</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Modeling">MODELING</A></LI><LI>WETLAB
						</LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/protocol">PROTOCOL</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Parts">PARTS</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Safety">SAFETY</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Data">DATA</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/ProteinModel">PROTEIN MODEL</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Achievement">ACHIEVEMENT</A></LI><LI>NOTEBOOK
						</LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/nb-wetlab">WETLAB</A></LI><LI>HUMAN PRACTICE
						</LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Unpas">INTRO SYNBIO TO UNPAS</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/SynBio">SYNBIO CLASS</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/SynbiGreen">SYNBIGREEN SURVEY</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Medsos">SOCIAL MEDIA</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/OnMedia">MEDIA</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Biosafety">BIOSAFETY SEMINAR</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/MeetUp">INDONESIA TEAMS MEET UP</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/Kids">FUN WITH KIDS</A></LI><LI><A href="https://2014.igem.org/Team:ITB_Indonesia/UPI">SHARING SYNBIO IN UPI</A></LI></DIV><H1>A. Scaning Electron Microscope (SEM) Result  Analysis</H1><P align="justify">For biodegrading experiments, we used 3x5 cm <SUP> 2 </SUP> bottle plastics with average weight 0.05 g. Then, we washed it several times with water and ethanol.</P><P align="justify">Plastic samples were applied just before we inoculated the bacteria on the medium. Bacterial culture was grown at 37<SUP>0</SUP>C in Luria-Broth medium. After 3 days of incubation, we washed the plastics with water and ethanol then dried for measurement of the weight loss.</P><P align="justify">Meanwhile, for UC Davis bacterial culture, we inoculated the bacteria in Luria-Broth medium supplemented with 170 ppm cloramphenicol. After an absorbance A600 nm of 0.6 was reached, we added 1% of arabinose and plastic samples. After 2 days of incubation, we washed the plastics with water and ethanol then dried for measurement of the weight loss.</P><P align="justify">We used medium supplemented with antibiotics and plastics sample as control on this experiment. The scaning electron microsccopy analysis of fractured surface of PET film was carried out using  Scaning Electron Microscope. The surface of the treated PET samples were coated with conductive heavy metals such as gold/palladium.</P><PALIGN>SEM image shows that cracks were observed at the surface of a plastic samples (PET) after incubation of the bacterial culture.  However, there was no significant weight decreased of the plastics samples.  From this SEM study  we concluded that both our LC Cut UC Davis and OmpA-LC-Cut Fussion ITB 2014 able to degrade PET plastics samples. Figure 1. shown the control sampel that we could not observe any cracks. Figure 2 and 3 shown the PET degradation by Lc cutinase from UC davis and from ITB_Indonesia.<DIV style="text-align: center;"><P><SMALL>Figure 1. Control sampel (PET plastic incubated in medium)</SMALL></P></DIV><DIV style="text-align: center;"><P><SMALL>Figure 2. PET plastic after treatment with ompA-LC-cutinase  from ITB_Indonesia construct (BBa_K1387006)</SMALL></P></DIV><DIV style="text-align: center;"><P><SMALL>Figure 3. PET plastic  after treatment with LC-cutinase UC Davis 2012 gene construct (Bba_K936020)</SMALL></P></DIV><H3>References</H3><OL style="text-decoration:none;"><LI>Sepperumal, Umaheswari, Murali Markandan, and Anbusaravanan Natarajan. 2013. electron microscopic studies of Polyethylene terepthalate degradation potential of Pseudomonas species. J. Microbiol. Biotech. Res. (1): 104-110</LI><LI>Mittal,Alok, R.K. Soni, Khrisna Dutt, and Swati Singh. 2010. Scaning electron microscopy study of hazardous waste flakes of polyethylene terephthalate (PET) by aminolysis and ammonolysis. Journal of Hazardous Materials Volume 178, Issues 1-3 </LI></OL><H1>B. Ethylene Glikol Chromic Acid Assay</H1><P align="justify">Poly(ethylene terephthalate) (PET) is a plastic that is a polymer of ethylene terephthalate (C10H8O4) units (Sarker et al., 2010). Polyethylene terephthalate (PET) can be degrade via limited enzymatic hydrolysis of the ester bond of the polymer backbone, one enzyme that have been assesed for this purpose is cutinase (Ribitsch et al., 2012). When polyethylene terephthalate degraded, it will produce terephthalic acid and ethylene glycol (Venkatachalam et al., 2012). Ethylene glycol is one of alcohol compound. Oxidation of alcohols by strong oxidants such as chromic acid (K2Cr2O7) in suphuric acid(H2SO4) is possible, but differs depending on the degree of alcohol. If a reaction has occurred using chromic acid in sulphuric acid, there is a color change from orange to green. In our project we use chromic acid to detect ethylene glycol as product of the PET degradation by LC cutinase enzyme.</P><P align="justify">Each sample used for measure the absorbance contain a fixed amount of chromic acid. The absorbance of the chromic acid will represent the total amount of ethylene glycol inversely. It means that if the absorbance is high, the amount of ethylene glycol in solution is low, and if the absorbance is low, the amount of ethylene glycol in solution is high. If the absorbance is high, the amount of chromic acid is high, which means that the amount of ethylene glycol being oxidized by chromic acid is low and vice versa. Both of LC Cutinase UC Davis and OmpA-LC Cutinase ITB_Indonesia shown the degrading activity of PET become ethylene glycol as once of the product.The optimum time to produce ethylene glycol is on the fourth hour after inoculation.</P><H3>References</H3><OL style="text-decoration:none;"><LI>Ribitsch D, Enrique HA, Katrin G, Anita D, Sabine Z, Annemarie M, Rosario DR, Georg S, Karl G, Helmut S and Georg MG. 2012. A New Esterase from Thermobifida halotolerans Hydrolyses Polyethylene Terephthalate (PET) and Polylactic Acid (PLA). <I>Polymers. 4: 617-629</I></LI><LI>Sarker M, Aminul K, Mohammad MR, Mohammed M and ASMD Mohammad. 2011. Waste Polyethylene Terephthalate (PETE-1) Conversion into Liquid Fuel. <I>Journal of Fundamentals of Renewable Energy and Applications. 1</I></LI><LI>Venkatachalam S, Shilpa GN, Jayprakash VL, Prashant RG, Krishna R and Anil KK. 2012. Degradation and Recyclability of Poly (Ethylene Terephthalate). <I>Intech</I></LI></OL><H1>C. pNPP assay</H1><P align="justify">Blastp program shows that LC cutinase shows high amino acid sequence similarity (54 to 60%) to lipase (Sulaiman, 2012). So, we tried to measure LC-cutinase activity using p-nitrophenylpalmitate (pNPP) as a substrate. pNPP typically used for measuring lipase activity.</P><P>In this assay, we used bacterial culture because in real application we would like to use bacterial culture to degrade PET directly. As negative control, we used <I>''E.coli'' </I> BL21(DE3) without plasmid. We also included LC-cutinase UC Davis 2012 gene construct (Bba_K936020) in this experiment. LC-cutinase UC Davis 2012 bacterial culture is induced after 2 hours of growth (after OD 600 nm ±0.6 was reached) using 1% of arabinose. All of the bacterial culture were harvested after 4 hours of growth.From ethylene glycol assay using chromic acid, we found that ompA-LC-cutinase from ITB_Indonesia construct (BBa_K1387006) has the highest activity after 4 hours of growth.</P><P align="justify">At first, we measured the activity at 60 degrees Celsius. We found that both ompA-LC-cutinase from ITB_Indonesia and LC-cutinase UC Davis 2012 still performed small activity. Then, we tried measure the activity at 25 degrees Celsius. The ompA-LC-cutinase from ITB_Indonesia’s activity was 0.001 U/mL. While, LC-cutinase UC Davis 2012 has ten fold higher (0.01 U/mL). The LC cutinase activity was determined based on the standard curve of p-nitrophenol. One unit of cutinase activity was defined as the amount of enzyme releasing 1 μmol pNP per minute under the assay conditions.</P><P align="justify">From this experiment, we are able to confirm that ompA-LC-cutinase from ITB_Indonesia successfully perform activity. But, its suggest that we still have to check the possible activity in different time of bacterial growth, expression rate,  LC-cutinase presentation by ompA and activity assay using different substrate.</P><DIV style="text-align: center;"><P><SMALL>Figure 1. LC Cutinase activity. The enzymatic activity was determined at 60C in assay condition using pNP-palmitate (C16) as a substrate. The experiment was carried out twice.</SMALL></P></DIV><DIV style="text-align: center;"><P><SMALL>Figure 2. LC Cutinase activity. The enzymatic activity was determined at 25 C in assay condition using pNP-palmitate (C16) as a substrate. The experiment was carried out twice.</SMALL></P></DIV><P align="">Small activity of LC-Cutinase maybe due to their oligomeric state. Tethering protein to cell surface may affect folding of protein and also their biological function (Park,2010). These suggest us to look futher detail on folding and displaying state of LC cutinase.</P><P align="justify">There are several things that should be noted when we designate fusion strategy of passanger protein or protein of interest with the outer membrane of protein. Linking the different passanger proteins to the  same outer membrane protein may result in different translocation. The characteristic of passanger protein affects the translocation. Its characteristic such as formation of disulfide bridge, total residue of hydrophobic aminoacid etc. </P><P align="justify">So there are four requirements have to be met for constructing well designed surface display. First is, it has to possesses efficient signal peptide. Second, it should have strong anchor to keep the fusion protein from detachment. Third, it should be compatible to the sequence that is inserted or fused. The last is it should resisstant to the attack of the protease from medium or periplasmic membrane.  </P><H3>Reference</H3><OL style="text-decoration:none;"><LI>Sulaiman, Sintawee, SayaYamat, EikoKanaya, Joong-Jae Kim, Yuichi Koga, Kazufumi Takano and Shigenori Kanaya. 2012. Isolation of a Novel Cutinase Homolog with Polyethylene Terephthalate-Degrading Activity from Leaf-Branch Compost by Using a Metagenomic Approach. <I>Applied and Environmental Microbiology p 1556-1562.</I></LI><LI>Lee, Sang Yup., Choi, Jong Hyun., Xu, Zhaohui. 2003. “Microbial Cell-Surface Display” <I>Elsevier</I></LI></OL></PALIGN></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ITB_Indonesia/Data">http://2014.igem.org/Team:ITB_Indonesia/Data</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ITB_Indonesia/Data" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ITB_Indonesia/Data" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ITB_Indonesia/Data&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ITB_Indonesia/Data&amp;oldid=365649" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>