<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NJU-China_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NJU-China/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Title</TITLE></P><DIV id="loader-wrapper"><DIV class="load_title">Now Loading...</DIV></DIV><CANVAS id="canvas" style="position: fixed"><P>your browser doesn't support canvas.</P></CANVAS><HEADER id="header" class="header"><DIV id="nav" class="nav"><UL class="nav-menu clearfix unstyled"><LI><A href="https://2018.igem.org/Team:NJU-China" class="three-d active">
                Home
                <SPAN class="three-d-box"><SPAN class="front">Home</SPAN><SPAN class="back">Home</SPAN></SPAN></A></LI><LI><A href="javascript:void(0)" class="three-d">
                Project
                <SPAN class="three-d-box"><SPAN class="front">Project</SPAN><SPAN class="back">Project</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/Project" class="three-d">
                        Background
                        <SPAN class="three-d-box"><SPAN class="front">Background</SPAN><SPAN class="back">Background</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/Design" class="three-d">
                        Design
                        <SPAN class="three-d-box"><SPAN class="front">Design</SPAN><SPAN class="back">Design</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/Results" class="three-d">
                        Results
                        <SPAN class="three-d-box"><SPAN class="front">Results</SPAN><SPAN class="back">Results</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/InterLab" class="three-d">
                        InterLab
                        <SPAN class="three-d-box"><SPAN class="front">InterLab</SPAN><SPAN class="back">InterLab</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/Notebook" class="three-d">
                        Notebook
                        <SPAN class="three-d-box"><SPAN class="front">Notebook</SPAN><SPAN class="back">Notebook</SPAN></SPAN></A></LI></UL><LI><A href="https://2018.igem.org/Team:NJU-China/Parts" class="three-d">
                Parts
                <SPAN class="three-d-box"><SPAN class="front">Parts</SPAN><SPAN class="back">Parts</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/Model" class="three-d">
                Model
                <SPAN class="three-d-box"><SPAN class="front">Model</SPAN><SPAN class="back">Model</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/Safety" class="three-d">
                Safety
                <SPAN class="three-d-box"><SPAN class="front">Safety</SPAN><SPAN class="back">Safety</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/Human_Practices" class="three-d">
                Human Practice
                <SPAN class="three-d-box"><SPAN class="front">Human Practice</SPAN><SPAN class="back">Human Practice</SPAN></SPAN></A></LI><LI><A href="javascript:void(0)" class="three-d">
                Team
                <SPAN class="three-d-box"><SPAN class="front">Team</SPAN><SPAN class="back">Team</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/Team" class="three-d" style="font-size:18px;">
                        Team
                        <SPAN class="three-d-box"><SPAN class="front" style="font-size:18px;">Team</SPAN><SPAN class="back" style="font-size:18px;">Team</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/Attributions" class="three-d" style="font-size:18px;">
                        Attributions
                        <SPAN class="three-d-box"><SPAN class="front" style="font-size:18px;">Attributions</SPAN><SPAN class="back" style="font-size:18px;">Attributions</SPAN></SPAN></A></LI><LI><A href="https://2018.igem.org/Team:NJU-China/Collaborations" class="three-d" style="font-size:17px;">
                        Collaborations
                        <SPAN class="three-d-box"><SPAN class="front" style="font-size:17px;">Collaborations</SPAN><SPAN class="back" style="font-size:17px;">Collaborations</SPAN></SPAN></A></LI></DIV></HEADER><DIV id="main" class="main"><H1 id="1">Neurite morphological proﬁle based on Sholl analysis and GED method</H1><P>PDF version can be downloaded <A href="https://static.igem.org/mediawiki/2018/3/35/T--NJU-China--2018--model-pdf.pdf">here</A>.</P><H2 id="1-1">I. Problem Review </H2><P> Based on peptide expression mechanism, with the purpose of delivering specified therapeutic molecules to certain position of the cell, biological molecular targeting systems are developed. In our experiment, two methods are introduced, each of which takes different RNA elements respectively, including the 5’-UTRs of Tick-borne encephalitis virus (TBEV) and the 3’-UTR of mouse β-Actin gene.
        According to our results, the former device performs better the latter one. The 5’-UTRs of TBEV could more precisely localize the targeted mRNA in dendrites of the neuron much more efficiently than 3'-UTR device and the control group.</P><H2 id="1-2">II. DFD and BFD distribution</H2><P>Furthermore, some experiment photographs has shown the difference between the distribution of the mRNA in the dendrites under each two circumstances (TBEV and Control). Biological molecules are localized more depthly in the dendrite with the 5'-UTR (TBEV) device, while more widthly by no device. The difference between two difference is very similar to comparision the BFS (Breadth-first search) algorithm and DFS (Depth-first search) algorithm. Therefore, these two diffusion behaviour could be vividly named, &quot;Breadth-first diffusion&quot;, and &quot;Depth-first diffusion&quot;.</P><P><I>Figure1</I> and <I>figure2</I> are simple illustrations of Depth-first diffusion and Breadth-first diffusion respectively, where orange circle denotes dendrite node, orange line denotes dendrite branch. In addition, red arrow means the direction in which biological molecules diffuse. In these two molecular diffusion pattern could be clearly oringinated, where in the Depth-first diffusion (abbreviate: DFD) strategy, moulecules tend to spread into more branches of the dendrite, while in the Breadth-first diffusion (abbreviate: BFD)  strategy, molecules focus on one certain branch of the dendrite, and cover a much more further distance than DFD.
        Our experiment results match these two diffusion pattern, however, these two diffusion could not be described quantitatively. Therefore, quantitative mathematical methods are needed in order to establish morphological profile for diffusion area in the dendrite.
        Therefore, in our model part, two mathematical model including Sholl analysis model and graph model are developed.</P><H2 id="1-3">Sholl analysis</H2><H3 id="1-3-1">i. Theoretical description </H3><P>Sholl analysis is a simple but useful model for morphological analysis of neurite.<SUP>[1]</SUP> It is oringinally intended for describing the relationship between the spatial position and the amount of dendrite branches, in the mathematical form similar to radial distribution function (abbreviate: RDF). </P><P>    Based on the photograph of the neurite, spherical coordinate centering at the cell body could be established. Then, shells whose radius vary at proper equal interval would be applied on this coordinate. For each shell, the amount of the intersection between the dendrite branches and the shell surface would be calculated.
        In the <I>figure3</I>, the realistic neurite is simplified as nodes and edges. Then, we apply two shells, (red one and purple one), on this simplified neurite model. There exist three intersections on the <I>shell1</I> (red shell), and only one intersection for the <I>shell2</I> (purple shell).
        If more shells and more intersections are calculated,  meaningful curves for branches versus radius could be derived. In some way, it could reflect the morphological feature of the neurite.
        However, the aim of our research is slightly different from its original utility. In our experiment, as for the area where biological molecules are not delivered, there exist no fluorescence signal, thus not being shown on the experiment photograph. Based on this feature, area where biological molecular diffused could be subtract from the whole neurite. Therefore, applying the Sholl analysis technique on the fluorescence area, could reflects the distribution feature of the biological molecules.
    </P><H3 id="1-3-2">ii. sholl analysis result  </H3><P>    We apply sholl analysis on the neurite with 5'-UTR(TBEV), 3'-UTR and Control Group respectively, where transfer capacities and different patterns of each device could be clearly originated as follow.<SUP>[2]</SUP></P><H4 id="1-3-2-1">ii.1 Transfer capacity</H4><P>In the <I>figure4</I> and <I>figure5</I>, the discrete points denotes the relationship of the shell at discrete interval and the branch-shell intersections, while the smooth curve is the fitting curve for these statistics based on polynomial fitting technique.</P><P>In the graph below, the TBEV 5'UTR device shows significant advantage over other two devices, which drives much more biological molecules into the dendrites.</P><H4 id="1-3-2-2">ii.2 Diffusion Pattern</H4><P>Furthermore, different diffusion patterns are quantitatively described through discribution based on Sholl Analysis.
        Red curve denotes the control group, while the black curve denotes the experiment group with 5'-UTR (TBEV) device.
        Under the 5'-UTR (TBEV) circumstances, the distribution curve covers a much longer range than the control group. What's else, the average intersections of the experiment group is obviously lower than the control group. In conclusion, red curve (control group) and black curve (experiment group) meet with the Breadth-first diffusion strategy, and Depth-first diffusion strategy respectively.</P><H2 id="1-4">Graph Edit Distance Method</H2><P>
        However, the basic model (originated from Sholl Analysis) has an obvious drawback that it cannot  describe the synapses’ relationships between parents and child vertexes and the similarity between different  basic models can't correspond to the similarity of images.
        Therefore, we introduced the graph model, which can further solve this problem. It defines each bifurcation point as a “vertex” and uses “edge” to represent relationship between bifurcation points. In a nutshell, the neuron can be modeled as the mathematical form of &quot;Graph&quot; or Computer Science concept &quot;Network&quot;.
        A universal Graph can be generally written as a two-tuple.
    </P><P>where, in detail:
        • V denotes the set of vertexes
        • E denotes the set of edges.</P><P>In addition, some other properties such as the weight of the edged (such as <I>w<SUB>ij</SUB></I>), or label map of nodes and edges (such as <I>ϕ</I>(v) : <I>V</I> → <I>T<SUB>V</SUB></I> and <I>φ(e)</I> : <I>E</I> → <I>T<SUB>E</SUB></I>) be discussed in some Bioinformatics Problem.
        In our neuron Graph model, the network can be simplified as unweighted Homogenous network. However, weight could be introduced as the length of the edge in the further research.</P><H3 id="1-4-1">i. GED definition</H3><P>Graph Edit Distance is an effective metric og the similarities between different graphs, based on which similarities between different neurons can be also calculated.<SUP>[3]</SUP>
        The general mathematical form of it can be written as:</P><P>The GED is the total cost of distortion that is needed to transform graph <I>g<SUB>1</SUB></I> to graph <I>g<SUB>2</SUB></I>,where P(<I>g<SUB>1</SUB></I>, <I>g<SUB>2</SUB></I>) denotes the set of edit operations including insertions, deletions and substitutions, and <I>e<SUB>i</SUB></I> is one certain operation.In our model, only the insertions and deletions are considered because of the absence of the labels of the nodels and edges.</P><P>In another word, it is kind of Error-tolerant Graph Imorphosim, where two graph <I>g<SUB>1</SUB></I> =&lt;<I>V</I> ∪ {<I>ϵ</I>}, <I>E</I> &gt; and <I>g<SUB>2</SUB></I> =&lt;<I>V′</I> ∪ {<I>ϵ</I>}, <I>E′</I> &gt;. {<I>ϵ</I>} denotes the empty node, just like &quot;vaccum&quot;. <I>V</I> ∪ {<I>ϵ</I>} of <I>g<SUB>1</SUB></I> can be mapped into <I>V′</I> ∪ {<I>ϵ</I>} of <I>g<SUB>2</SUB></I>.
        And mapping function of the edges is:
    </P><H3 id="1-4-2">ii. Numerical Results</H3><P>Based on the evolutionary graph edit distance algorithm, the GED between three kinds of neuron graphs.<SUP>[4]</SUP>
        The raw GED is set [8,0] and [16,0].
        After iterations, GED can be calculated:
    </P><P>The metric matrix can be derived:</P><H2 id="1-5">References</H2><P>[1] Ferreira T, Blackman A, Oyrer J, Jayabal A, Chung A, Watt A, Sjöström J, van Meyel D. (2014), Neuronal morphometry directly from bitmap images, Nature Methods 11(10): 982–984
[2] Schindelin J, Arganda-Carreras I, Frise E, Kaynig V, Longair M, Pietzsch T, Preibisch S, Rueden C, Saalfeld S, Schmid B, Tinevez JY, White DJ, Hartenstein V, Eliceiri K, Tomancak P, Cardona A. (2012) Fiji: an open-source platform for biological-image analysis, Nature Methods 9(7): 676-682.
[3] K. Riesen, Structural Pattern Recognition with Graph Edit Distance.
[4] Rashid Ibragimov, Maximilian Malek, Jiong Guo, Jan Baumbach: GEDEVO: An Evolutionary Graph Edit Distance Algorithm for Biological Network Alignment. GCB 2013:68-79
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>