Team:TAS Taipei/Experimental Summary
X
Project
Experiments
Modeling
Prototype
Human Practices
Safety
About Us
Attributions
Project
Experiment
Modeling
Prototype
Human Practice
Safety
About Us
Attributions
Experiments
Constructing DNA to make our ideas come to life.
PR
Construct Design
Can PR Bind Citrate-Capped NPs?
BIOFILM
Can Biofilm trap nanoparticles?
How Is Biofilm Production Regulated?
Construct Design
Hypothesis
Expression of CsgD and OmpR234 Increases Biofilm Formation
REFERENCES
LAB NOTEBOOK
EXPERIMENTAL SUMMARY
We use two approaches to capture nanoparticles (NPs): 1) target citrate-capped nanoparticles (CC-NPs) using a bacterial membrane protein Proteorhodopsin (PR), and 2) trap a wide variety of NPs using an E. coli biofilm. We show experimentally that PR binds to 60 nm citrate-capped silver nanoparticles (CC-AgNPs). We also show that an E. coli biofilm can trap gold and silver NPs ranging from 15-60 nm. To increase biofilm production, we overexpress CsgD and OmpR234, two positive regulators of biofilm synthesis. We demonstrate that E. coli carrying our constructs produce up to 8 times more biofilm compared to controls.
PROTEORHODOPSIN
Our goal is to remove as many NPs as possible from wastewater systems to minimize damage to the environment and our health. Due to the sheer variety of manufactured NPs and the significant chemical differences between them, we looked for common properties across different types of NPs. Capping agents are surface coatings frequently used to prevent aggregation of NPs (Niu & Li 2014). We specifically target citrate, the most commonly used NP capping agent (Levard et al. 2012).
Construct Design
PR is a transmembrane protein found in proteobacteria. Since PR has previously been shown to bind citrate through two positively charged lysine residues on its surface (figure 2-1; Béjà et al. 2000; Syed 2011), we hypothesized that PR may also bind citrate capping agents of CC-NPs.
We obtained the DNA sequence of pR (Syed 2011) and modified it to remove three internal cutting sites (EcoRI, PstI, and SpeI). The sequence of pR was then flanked by an upstream strong promoter and strong ribosome binding site (RBS) combination (BBa_K880005), and a downstream double terminator (BBa_B0015) to maximize expression of PR protein. This final construct (BBa_K2229400; figure 2-2) was ordered from IDT and cloned into pSB1C3, a biobrick backbone (BBa_K2229400; figure 2-4). We designed PCR primers to isolate the pR open reading frame (ORF), and then cloned it into pSB1C3 (BBa_K2229450; figure 2-3 and 2-5). Sequencing results from Tri-I Biotech confirmed that our final construct and the pR ORF are correct.
Figure 2-1 PR-citrate interaction (Syed 2011). Two lysine residues (blue) on the surface of PR can bind citrate (red).
Figure 2-2 BBa_K2229400: Proteorhodopsin expression. Our construct includes a strong promoter, strong RBS, the pR ORF, and a double terminator. Figure: Justin Y.
Figure 2-3 BBa_K2229450: pR ORF was isolated and cloned into pSB1C3. Figure: Justin Y.
Figure 2-4 PCR check for PR expression construct (BBa_K2229400) using VF2 and VR primers. The expected size of BBa_K2229400 is 1300 bp (green box). Cloning: Catherine Y., Dylan L.
Figure 2-5 PCR check for pR ORF (BBa_K2229450) using VF2 and VR primers. The expected size of BBa_K2229450 is 1100 bp (green box). Cloning: Dylan L.
Can Proteorhodopsin Bind Citrate-Capped Nanoparticles?
Using a solution containing 60 nm CC-AgNPs (from Sigma Aldrich), we tested PR’s ability to trap CC-NPs as we hypothesized. Because CC-AgNP solution is yellow in color, we can take absorbance measurements. Two groups were set up: E. coli carrying either BBa_K2229400 (PR expression construct; figure 2-2) or a negative control BBa_E0240 (GFP-generator) were grown in Luria-Bertani (LB) broth overnight. GFP-generator was used as a negative control because it does not express PR. The cultures were centrifuged, resuspended in distilled water to remove LB broth, and diluted to standardize bacterial population. Then, the cultures were mixed with CC-AgNP solution and shaken at 120 rpm. Every hour (for a total of 5 hours), one tube from each group was centrifuged at 4500 rpm to isolate the supernatant. At this speed, we observed that nearly all bacteria (and bound CC-AgNPs) were pulled down into the pellet while free CC-AgNPs remained in the supernatant, which was measured using a spectrophotometer at 430 nm.
Figure 2-6 Proteorhodopsin traps CC-AgNPs. A) Absorbance of the supernatant decreased markedly when PR bacteria was added to CC-AgNPs; the absorbance did not change significantly when GFP-Gen (negative control) bacteria was added. B) Over the 5 hour period, we observed a large orange region (aggregated CC-AgNPs) in the PR group. Experiment & Figure: Justin Y.
Over 5 hours, we found that absorbance values of the supernatant decreased much faster when PR bacteria was added, while the absorbance did not change significantly when GFP-generator bacteria was added (figure 2-6A). In addition, after centrifugation, we saw dark orange regions in the pellet of PR bacteria, but not in the GFP-generator bacteria (figure 2-6B). CC-AgNP solution is yellow in color, which suggests that the orange regions observed in the PR pellet are aggregated CC-AgNPs. In summary, our results suggest that PR is able to bind CC-AgNPs.
Using data from this experiment, our modeling team determined the rate at which PR binds to CC-AgNPs, a constant which was integrated into their model. Learn more about it here!
BIOFILM
Our biofilm approach was originally inspired by the ability of jellyfish mucus to bioaccumulate gold NPs (AuNPs) and quantum dots through electrostatic interactions (Patwa et al. 2015). A biofilm is a community of microbes embedded in a matrix of extracellular polymeric substances (EPS), which consist of different polysaccharides, proteins, and lipids (Donlan 2002). Recent studies show that the EPS in biofilms can also trap various NPs (Kaoru et al. 2015, Nevius et al. 2012; figure 3-1).
Figure 3-1 We envision using E. coli biofilms (green) to trap nanoparticles of different sizes and composition (pink).
Most wastewater treatment plants (WWTPs) already use microbes to break down and remove organic components in wastewater (Sehar & Naz 2016). The recent increase in NP contaminants entering WWTPs, however, poses a threat to these microbes and the treatment process. For example, AgNPs have antimicrobial effects and other metal oxide NPs can inhibit microbes from performing important processes such as nitrification (Yun & Lee 2017; Walden & Zhang 2016). Compared to planktonic bacteria, biofilms are nearly four times more resistant to NPs, increasing their tolerance to NPs in wastewater (Choi et al. 2010).
Can Biofilms Trap Nanoparticles?
The first step of this approach is to test if an E. coli biofilm can effectively trap NPs. E. coli liquid cultures grown in LB broth were transferred to petri dishes, with each containing glass coverslips to provide a surface for adherence. The dishes were incubated at 37˚C for 7 to 14 days, with 2 mL of LB added every two days to prevent the media from drying out. Viscous, film-like substances began to develop, indicating the production of EPS and biofilm (figure 3-2). At this point, we could extract and wash the biofilm.
Figure 3-2 Growing E. coli biofilms. A,B) Liquid cultures were plated with glass coverslips and incubated for up to 2 weeks. C) Biofilm can be washed and used. Experiment: Yvonne W.
Hoping to image the structure of biofilms up close using scanning electron microscopy (SEM), we tried different SEM sample fixation and preparation methods (figure 3-3 & table 1) (Fischer et al. 2012): A) freeze drying with a critical point dryer (CPD), B) fixation with glutaraldehyde (GA), and C) fixation with a combination of Alcian Blue, GA, and K4Fe(CN)6. We further improved image quality by taking multiple images of a field of view and stacking the images together to reduce noise (figure 3-4).
Table 1 Comparing different SEM sample preparation protocols. We tried each method and compiled the advantages and disadvantages.
Figure 3-3 Comparing different SEM sample preparation protocols. E. coli samples were prepared for SEM. A) Critical Point Drying seems to change cell morphology, whereas B,C) fixation with GA preserved both cell shape and biofilm structure. SEM Imaging & Figure: Justin Y.
Figure 3-4 Image stacking decreases noise. Here, 26 images of the same field of view were taken and stacked using Adobe Photoshop. SEM Imaging & Figure: Justin Y.
To test if biofilms can trap NPs, we used a solution containing 30 nm AuNPs (from Sigma Aldrich). Because the AuNP solution is purple in color, we can take absorbance measurements. Four experimental groups were set up (figure 3-5): a negative control containing only AuNPs, and three tubes containing AuNPs with either planktonic bacteria, biofilm, or “dead” biofilm (treated with antibiotics). If AuNPs are trapped, we would expect to see a decrease in absorbance (figure 3-5B). The four groups were shaken for 24 hours and centrifuged to isolate the supernatant (figure 3-5, C). The supernatant contains free AuNPs, which were measured using a spectrophotometer at 527 nm.
Figure 3-5 Biofilms trap nanoparticles. A,B) Experimental setup. C,D) Gold nanoparticles (AuNPs) were incubated with either bacteria or biofilm for 24 hours, and then centrifuged; free AuNPs were expected to remain in the supernatant. Shown are representative images and graph. Experiment: Yvonne W. Figure: Justin Y.
The negative control and the AuNP with planktonic bacteria groups did not change the purple color of the AuNP solution, indicating that bacteria alone cannot trap NPs (figure 3-5, C and D). The addition of biofilm, however, greatly reduced the amount of AuNP in the supernatant. Even with antibiotics, AuNP levels in the supernatant were still reduced, suggesting that the removal of AuNPs depends on the sticky and slimy extracellular components of biofilm and not on the bacteria itself. When we fixed and imaged the biofilm + AuNP sample by SEM (figure 3-6), we could see NPs in EPS areas. This was consistent with our idea that NPs are being trapped in the EPS layer of biofilms; therefore, we want to produce biofilm easily and efficiently.
Figure 3-6 SEM Image showing AuNPs trapped by biofilm. A biofilm+AuNP sample was fixed with GA. Some EPS is preserved (red) and AuNPs (white) seemed to aggregate and adhere onto the EPS. SEM Imaging: Justin Y.
How Is Biofilm Production Regulated?
Figure 3-7 Two curli operons—csgBA and csgDEFG—direct biofilm synthesis. Figure: Justin Y.
In E. coli, biofilm synthesis is mainly mediated through two curli operons (Barnhart & Chapman 2006). Curli fibers facilitate cell-surface and cell-cell adhesion, biofilm synthesis, and are the main protein components of the EPS (Reichhardt et al. 2015, Barnhart & Chapman 2006). The operons (csgBA and csgDEFG) control the expression of six proteins essential to biofilm formation (figure 3-7). CsgA and CsgB are curli monomers which can polymerize to form curli fibers; CsgD is an activator of csgBA transcription (figure 3-7). CsgE, CsgF, and CsgG help export CsgA and CsgB out of the cell. The operon csgDEFG can be activated by the protein OmpR, and the subsequent expression of all six proteins increase biofilm formation (Barnhart & Chapman 2006).
Construct Design
Three constructs were built to upregulate curli production by overexpressing CsgD, OmpR234 (a mutant form of OmpR which is constitutively active), or both. We acquired all parts from the iGEM distribution kit: a strong promoter and strong RBS combination (BBa_K880005) to maximize protein production, strong RBS (BBa_B0034), csgD (BBa_K805015), ompR234 (BBa_K342003), and a double terminator (BBa_B0015) to end transcription.
Figure 3-8 BBa_K2229100: CsgD expression. Our construct includes a strong promoter, strong RBS, csgD and double terminator. Figure: Justin Y.
Figure 3-9 BBa_K2229200: OmpR234 Expression. Our construct includes a strong promoter, strong RBS, ompR234 and double terminator. Figure: Justin Y.
For strong CsgD expression (figure 3-8, BBa_K2229100), csgD was inserted behind BBa_K880005 (figure 3-10, BBa_S05397), and then before BBa_B0015 (figure 3-11).
For strong OmpR234 expression (figure 3-9, BBa_K2229200), ompR234 was inserted before BBa_B0015 (figure 3-10, BBa_S05398) and then behind BBa_K880005 (figure 3-11).
Figure 3-10 PCR check for BBa_K880005 + CsgD and OmpR234 + BBa_B0015 using VF2 and VR primers. The expected size of BBa_K880005 +CsgD is 1000 bp (orange box) and OmpR234 +BBa_B0015 is 1100 bp (blue box). Cloning: Catherine Y., Dylan L., Justin Y.
Figure 3-11 PCR check for BBa_K2229100 & BBa_K2229200 using VF2 and VR primers. The expected size of BBa_K2229100 (CsgD expression) is 1100 bp (orange box), and BBa_K2229200 (OmpR234 expression) is 1200 bp (blue box). Cloning: Catherine Y., Dylan L., Justin Y.
Figure 3-12 BBa_K2229300: CsgD and OmpR234 Expression. Our construct includes a strong promoter, two strong RBS, csgD, ompR234 and double terminator. Figure: Justin Y.
To strongly express both CsgD and OmpR234 (figure 3-12, BBa_K2229300), a strong RBS (BBa_B0034) was inserted in front of the intermediate BBa_S05398 (ompR234+double terminator) to make BBa_S05399 (RBS+ompR234+double terminator). Finally, BBa_S05397 (K880005+csgD) was inserted before BBa_S05399 to complete the full construct (BBa_K2229300) (figure 3-13). Sequencing results from Tri-I Biotech confirmed that our final construct is correct.
Figure 3-13 PCR check for BBa_K2229300 using VF2 and VR primers. The expected size of BBa_K2229300 is 1900 bp (green box) Cloning: Catherine Y., Dylan L., Justin Y.
Hypothesis
Figure 3-14 Overexpression of CsgD and/or OmpR234 upregulates the curli operon to different degrees. We hypothesized that biofilm production would be upregulated (in increasing order) if we overexpress A) CsgD, B) OmpR234, or C) both. Figure: Justin Y.
We hypothesized that biofilm production would be upregulated (in increasing order) if we overexpress CsgD, OmpR234, or both (figure 3-14). Overexpression of CsgD would result in more curli monomers, but no transport proteins to carry those monomers out of the cell. Overexpression of OmpR234 would allow curli monomers to be exported and form curli fibers and biofilm. Finally, when both CsgD and OmpR234 are overexpressed, twice the amount of curli monomers should be made and exported to form even more curli fibers and biofilm.
Expression of CsgD and OmpR234 Increases Biofilm Formation
To test the expression of CsgD and OmpR234, we ran SDS-PAGE using transformed and lysed E. coli cultures (figure 3-15). Cultures transformed with the basic parts BBa_K805015 (csgD ORF alone) and BBa_K342003 (ompR234 ORF alone) were used as controls. We expected to see CsgD around 25 kDa and OmpR234 around 27 kDa (Brombacher et al. 2006; Martinez & Stock 1997). Compared to controls, thicker and darker bands at the expected sizes were observed in both BBa_K2229100 (CsgD overexpression) and BBa_K2229200 (OmpR234 overexpression) (figure 3-15; proteins bands are marked by asterisks).
In addition to the bands at 25 and 27 kDa, cultures carrying BBa_K2229300 (CsgD and OmpR234 expression) contained two extra bands at 15 kDa and 30 kDa, which were not observed in the controls. We looked into the product of other curli operon genes, and found that CsgG is around 30 kDa, whereas CsgA, B, C, E, and F are all around 15 kDa (Robinson et al. 2006; Uhlich et al. 2009; Shu et al. 2012). This suggests that, as expected, BBa_K2229300 stimulates the production of other curli proteins as well (predicted proteins and sizes are labeled in figure 3-15).
Figure 3-15 SDS-PAGE results show that BBa_K2229100, BBa_2229200, and BBa_K2229300 overexpress CsgD, OmpR234, or both proteins, respectively. Predicted proteins from the curli operons are listed on the right; E. coli expressing GFP was used as a positive control. Protein Gel: & Figure: Justin Y.
After confirming protein expression, we wanted to test if our constructs actually lead to faster and greater biofilm production. We used Congo Red (CR), a dye commonly used to measure biofilm production (Reinke & Gestwicki 2011). CR solution mixed with bacterial liquid cultures were transferred to 12-well microtiter plates, and incubated with glass coverslips at 37˚C for one day. The samples were then washed with Phosphate Buffered Saline (PBS) and dried at 60˚C. Any stained biofilm on the glass coverslips was solubilized in ethanol, and absorbance was measured at 500 nm (figures 3-16, 3-17, 3-19). If biofilms were present, the solution would appear red, which could be quantified by an absorbance value.
We find that overexpressing CsgD and/or OmpR234 increases biofilm production to different degrees, as we hypothesized (figure 3-19). Overexpression of CsgD (BBa_K2229100) doubles biofilm production compared to the control BBa_K805015 (figure 3-16), whereas overexpression of OmpR234 (BBa_K2229200) leads to about 8 times more biofilm compared to the control BBa_K342003 (figure 3-17 & figure 3-18, A). CGU_Taiwan helped us independently verify our OmpR234 overexpression results using a different dye, crystal violet, which is also commonly used to stain biofilms (figure 3-18, B). Interestingly, both biofilms characterized in our assay are found around the glass coverslip and do not seem to stick well to the glass surface (figures 3-16A & 3-17A).
Figure 3-16 Overexpression of CsgD (BBa_K2229100) doubles biofilm production. A) Congo red assay stains biofilm (red). B) Stained biofilm is solubilized in ethanol. C) Absorbance is measured at 500 nm. Experiment & Figure: Yvonne W.
Figure 3-17 Overexpression of OmpR234 (BBa_K2229200) leads to ~8 times more biofilm production than control. A) Congo red assay stains biofilm (red). B) Stained biofilm is solubilized in ethanol. C) Absorbance is measured at 500 nm. Experiment & Figure: Yvonne W.
Figure 3-18 Overexpression of OmpR234. A) OmpR234 overexpression (BBa_K2229200) produces more biofilm. B) CGU_Taiwan independently verified that BBa_K2229200 increases biofilm production through crystal violet staining. Experiment & Figure: Yvonne W.
When all three constructs were compared, we find that overexpression of both OmpR234 and CsgD (BBa_K2229300) increases biofilm production the most (figure 3-19). BBa_K2229300 also increased adhesion to our glass coverslips, and we could see a layer of biofilm which remained attached to the glass surface after the washing steps (figure 3-19, A).
Figure 3-19 Overexpression of both CsgD and OmpR234 (BBa_K2229300) increases biofilm production the most. A) Congo red assay stains biofilms. BBa_K2229300 increases adhesion to glass surfaces. B) Stained biofilm is solubilized in ethanol. C) Absorbance is measured at 500 nm. Experiment & Figure: Yvonne W.
In summary, we demonstrate that biofilms can trap NPs and our constructs function as hypothesized. Our collection of constructs (BBa_K2229100, BBa_K2229200, and BBa_K2229300) can successfully upregulate biofilm production to varying degrees.
*Details on any experimental setup can be found in our lab notebook .
REFERENCES
Barnhart, M. M., & Chapman, M. R. (2006). Curli Biogenesis and Function. Annual Review of Microbiology, 60, 131–147. http://doi.org/10.1146/annurev.micro.60.080805.142106
Béjà O., Aravind L., Koonin E. V., Suzuki M. T., Hadd A., Nguyen L. P., Jovanovich S. B., Gates C. M., Feldman R. A., Spudich J. L., Spudich E. N. and DeLong E. F., Bacterial rhodopsin: evidence for a new type of phototrophy in the sea, Science 289 (2000) 1902–1906.
Brombacher, E., Baratto, A., Dorel, C., & Landini, P. (2006). Gene Expression Regulation by the Curli Activator CsgD Protein: Modulation of Cellulose Biosynthesis and Control of Negative Determinants for Microbial Adhesion. Journal of Bacteriology, 188(6), 2027–2037. http://doi.org/10.1128/JB.188.6.2027-2037.2006
Choi,O.;Yu,C.-P.;EstebanFernańdez,G.;Hu,Z.Interactions of nanosilver with Escherichia coli cells in planktonic and biofilm cultures. Water Res. 2010, 44 (20), 6095−6103.
Donlan, R. M. (2002). Biofilms: Microbial Life on Surfaces. Emerging Infectious Diseases, 8(9), 881–890. http://doi.org/10.3201/eid0809.020063
Ferry J. L., Craig P., Hexel C., Sisco P., Frey R., Pennington P. L., et al. (2009). Transfer of gold nanoparticles from the water column to the estuarine food web. Nat. Nanotechnol. 4, 441–444. 10.1038/nnano.2009.157
Fischer, E. R., Hansen, B. T., Nair, V., Hoyt, F. H., & Dorward, D. W. (2012). Scanning Electron Microscopy. Current Protocols in Microbiology, CHAPTER, Unit2B.2. http://doi.org/10.1002/9780471729259.mc02b02s25
Flemming, H.-C., Neu, T. R., & Wozniak, D. J. (2007). The EPS Matrix: The “House of Biofilm Cells” . Journal of Bacteriology, 189(22), 7945–7947. http://doi.org/10.1128/JB.00858-07
Jiang, X.; Wang, X.; Tong, M.; Kim, H. Initial transport and retention behaviors of ZnO nanoparticles in quartz sand porous media coated with Escherichia coli biofilm. Environ. Pollut. 2013, 174, 38−49.
Ikuma, Kaoru, Alan W. Decho, and Boris L. T. Lau. “When Nanoparticles Meet Biofilms—interactions Guiding the Environmental Fate and Accumulation of Nanoparticles.” Frontiers in Microbiology 6 (2015): 591. PMC. Web. 2 Sept. 2017.
Levard, C., Hotze, E. M., Lowry, G. V., & Brown, G. E. (2012). Environmental Transformations of Silver Nanoparticles: Impact on Stability and Toxicity. Environmental Science & Technology, 46(13), 6900-6914. doi:10.1021/es2037405
Martinez-Hackert, E., & Stock, A. M. (1997). The DNA-binding domain of OmpR: crystal structures of a winged helix transcription factor. Chemistry & Biology, 4(2), 162. doi:10.1016/s1074-5521(97)90269-6 Nevius B. A., Chen Y. P., Ferry J. L., Decho A. W. (2012). Surface-functionalization effects on uptake of fluorescent polystyrene nanoparticles by model biofilms. Ecotoxicology 21, 2205–2213. 10.1007/s10646-012-0975-3
Niu, Z., & Li, Y. (2014). Removal and Utilization of Capping Agents in Nanocatalysis. Chemical Materials, (26), 1st ser., 72-83. doi:10.1021/cm4022479
Patwa, A., Thiéry, A., Lombard, F., Lilley, M. K. S., Boisset, C., Bramard, J.-F., … Barthélémy, P. (2015). Accumulation of nanoparticles in “jellyfish” mucus: a bio-inspired route to decontamination of nano-waste. Scientific Reports, 5, 11387. http://doi.org/10.1038/srep11387
Reichhardt, C., Jacobson, A. N., Maher, M. C., Uang, J., McCrate, O. A., Eckart, M., & Cegelski, L. (2015). Congo Red Interactions with Curli-Producing E. coli and Native Curli Amyloid Fibers. PLoS ONE, 10(10), e0140388. http://doi.org/10.1371/journal.pone.0140388
Reinke, A. A., & Gestwicki, J. E. (2011). Insight into Amyloid Structure Using Chemical Probes. Chemical Biology & Drug Design, 77(6), 399–411. http://doi.org/10.1111/j.1747-0285.2011.01110.x
Robinson, L. S., Ashman, E. M., Hultgren, S. J., & Chapman, M. R. (2006). Secretion of curli fibre subunits is mediated by the outer membrane-localized CsgG protein. Molecular Microbiology, 59(3), 870–881. http://doi.org/10.1111/j.1365-2958.2005.04997.x
Sehar, S., & Naz, I. (2016). Role of the Biofilms in Wastewater Treatment. DOI: 10.5772/63499
Shu, Q., Crick, S. L., Pinkner, J. S., Ford, B., Hultgren, S. J., & Frieden, C. (2012). The E. coli CsgB nucleator of curli assembles to β-sheet oligomers that alter the CsgA fibrillization mechanism. Proceedings of the National Academy of Sciences of the United States of America, 109(17), 6502–6507. http://doi.org/10.1073/pnas.1204161109
Slamovits, C. H., Okamoto, N., Burri, L., James, E. R., & Keeling, P. J. (2012). Corrigendum: A bacterial proteorhodopsin proton pump in marine eukaryotes. Nature Communications, 3, 822. doi:10.1038/ncomms1863
Syed, Farhana F., "Citrate Binding to the Membrane Protein Proteorhodopsin" (2011). Chemistry - Dissertations. Paper 182.
Uhlich, G. A., Gunther, N. W., Bayles, D. O., & Mosier, D. A. (2009). The CsgA and Lpp Proteins of an Escherichia coli O157:H7 Strain Affect HEp-2 Cell Invasion, Motility, and Biofilm Formation  . Infection and Immunity, 77(4), 1543–1552. http://doi.org/10.1128/IAI.00949-08
Connie Walden and Wen Zhang; Biofilms Versus Activated Sludge: Considerations in Metal and Metal Oxide Nanoparticle Removal from Wastewater. Environmental Science & Technology 2016 50 (16), 8417-8431, DOI: 10.1021/acs.est.6b01282
Yun, J., & Lee, D. G. (2017). Silver Nanoparticles: A Novel Antimicrobial Agent. Antimicrobial Nanoarchitectonics, 139-166. doi:10.1016/b978-0-323-52733-0.00006-9
