"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Costa Rica-TEC-UNA/Parts
From 2012.igem.org
Home
Team
Team Members and Advisors
Project
Description Notebook
Biobricks
Parts Submitted to the registry
Others
Safety Attributions Human Practice Official team profile
Biobricks designed
BBa_K836006
Lysis protein S from Enterobacteria phage lambda. This gene codifies for a protein that makes holes in the inner membrane of the bacterial cell. Between the RBS and the CDS a spacer consisting on a scar from a standard suffix and a prefix for CDS was inserted.
Source: http://www.uniprot.org/uniprot/P03705#section_seq it was optimized for R. opacus (avoiding standard restriction sites)
Part:BBa_K836005
Lysozyme from Rhodococcus phage RER2. Protein involved in the peptidoglycan degradation of the bacterial cell wall. A RBS for Gram positive bacteria (SpoVG) was added at the beginning.A spacer was put between the RBS and the start codon which corresponds to the scar between a normal suffix and the prefix used for CDS to avoid RBS-CDS problems. The sequence has no termination codon since we rely on the double terminators of the scar formed during assembly of biobricks.
Source: http://www.uniprot.org/uniprot/G9FHW6 ) and then it was optimized for R. opacus (avoiding standard restriction sites).
Part:BBa_K836007
Nitrilase regulator from R. rhodochrous. Protein in charge to regulate the expression of the genes put under control of nitA promoter. A spacer which consist of the scar left by the assembly of a normal suffix and a CDS prefix was inserted between RBS and CDS. The sequence has no termination codon since we rely on the double terminators of the scar formed during assembly of biobricks.
Source: The amino sequence was obtained from Uniprot ( http://www.uniprot.org/uniprot/P72312 ) and then the condon usage was optimized for expression in R. opacus.
Part:BBa_K836008
nitA promoter. Promoter inducible by nitrile and similar agents.
Source: http://www.ncbi.nlm.nih.gov/nucleotide/346420896?report=genbank&log$=nuclalign&blast_rank=1&RID=W0SH1PA7016
Part:BBa_K836009
Lysis device for R. opacus. Lysis construct based on the one presented by Berkeley's 2008 team BBa_K112022. It is based on the inducible expression of a lysozyme and lambda phage's holin in order to liberate internal products of the bacterial cell in an efficient way. The induction is achieved by the addition of nitrile or a similar agent since both genes are regulated by nitA promoter. Constitutive expression of nitR and a low constitutive expression of an antiholin are required to avoid undesired lysis.
Source:  The majority of the parts where made by backtranslating a protein. Others are existing biobricks while some are promoters obtained from databases as NCBI.
Part:BBa_K836002
O-acyltransferase WSD from Acinetobacter sp. This biobrick codifies for the protein DGAT (Diacylglycerol O-acyltransferase) which have dual activity, it can act as a wax ester synthase or as a diacylglycerol acyltransferase. In the latter case, its function consists in mediate the final step in TGAs synthesis using a fatty acyl-CoA and a DGA. This enzyme is vital in the storage and synthesis of lipids. Two transcriptional terminators (two biobricks BBa_B1006) were set at the end of the coding sequence.
Source: The amino sequence of this part was taken from Uniprot (wax-dgaT accesion Q8GGG1; http://www.uniprot.org/uniprot/Q8GGG1 ) and then codon optimization for R. opacus (avoiding restriction sites for the standard assembly method)
Part:BBa_K836003
lipA from B. cepacia. This biobrick codifies for the protein lipA from Burkholderia cepacia (Pseudomonas cepacia) which catalyzes the hydrolysis of triglycerides (Triacylglycerol + H2O = diacylglycerol + a carboxylate). In the presence of certain alcohols, it catalyzes the transesterification of TGAs to produce acyl esters and glycerol. It has only one PTM (Post-Translational Modification) which consists in a sulfide bond between 234 and 314 aa. Calcium serves as a cofactor for this enzyme (1 ion per subunit). The native signal peptide was not modified.  The sequence of a transcriptional terminator (biobrick BBa_B1006) was set at the end of the coding sequence.
Source: The amino sequence of this part was taken from Uniprot (lipA accesion P22088; http://www.uniprot.org/uniprot/P22088 ) and then codon optimization for E. coli (avoiding restriction sites for the standard assembly method).
Detailed parts
<groupparts>iGEM012 Costa_Rica-TEC-UNA</groupparts>
Retrieved from " http://2012.igem.org/Team:Costa_Rica-TEC-UNA/Parts "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
