<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ECUST_Quorum_Sensing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ECUST/Quorum Sensing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><UL class="navbar"><LI><A href="https://2018.igem.org/Team:ECUST" class="nav-text dropbtn">HOME</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Team" class="nav-text dropbtn">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Team" class="nav-text">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Attributions" class="nav-text">ATTRIBUTION</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Sponsors" class="nav-text">SPONSORS</A></LI></UL><LI><A href="https://2018.igem.org/Team:ECUST/Description" class="nav-text dropbtn">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Description" class="nav-text">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Background" class="nav-text">BACKGROUND</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Design" class="nav-text">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Experiments" class="nav-text">QUORUM SENSING</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Rust_Remover" class="nav-text">RUST REMOVER</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Biofilm_Remover" class="nav-text">BIOFILM REMOVER</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Fur_Inverter" class="nav-text">FUR INVERTER</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Lysin" class="nav-text">LYSIN</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Biocide" class="nav-text">BIOCIDE</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Light-on_Suicide" class="nav-text">LIGHT-ON SUICIDE</A></LI><LI><A href=" https://2018.igem.org/Team:ECUST/Demonstrate" class="nav-text">DEMONSTRATION</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Parts" class="nav-text dropbtn">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Parts#section1" class="nav-text"> BASIC PARTS</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Parts#section2" class="nav-text"> COMPOSITE PARTS</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Model" class="nav-text dropbtn">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Model" class="nav-text">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Quorum_Sensing" class="nav-text">QUORUM SENSING</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Iron_Sensing" class="nav-text">IRON SENSING</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Hardware" class="nav-text dropbtn">DEVICE</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Hardware" class="nav-text">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Software" class="nav-text">SOFTWARE</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Human_Practices" class="nav-text dropbtn">HUMAN PRACTICE</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Human_Practices" class="nav-text">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Integrated" class="nav-text">INTEGRATED HUMAN  PRACTICE</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Communication" class="nav-text">COMMUNICATION</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Public_Engagement" class="nav-text">EDUCATION &amp; PUBLIC ENGAGEMENT</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Applied_Design" class="nav-text">APPLIED DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Collaborations" class="nav-text">COLLABORATION</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Safety" class="nav-text dropbtn">BIOSAFETY</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Notebook" class="nav-text dropbtn">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/Protocol" class="nav-text">PROTOCOL</A></LI><LI><A href="https://igem.org/2018_Judging_Form?id=2737" class="nav-text dropbtn">JUDGING FORM</A></LI><LI><A href="https://2018.igem.org/Team:ECUST/InterLab" class="nav-text">INTERLAB</A></LI></DIV><DIV id="pagebanner" style="background-image:url(https://static.igem.org/mediawiki/2018/1/11/T--ECUST--model.png);"><DIV id="bannerspace"><DIV id="bannerquote">Quorum Sensing Model</DIV></DIV><MAIN><DIV class="contentbox"><H1 class="box-heading">1.Introduction</H1><P>   To understand, predict and ultimately control the behavior of our engineered microbial group effect, we have developed dynamic model of the system, based on transerential equations which describe and integrate the individual processes. This model involves several entities going from the molecular level (genes, RNAs, proteins, and metabolites) up to the cellular and population levels, distinct intracellular and extracellular compartments, and a wide range of biological and physical processes (transcription, translation, signalling, growth, transusion, etc). Here we can show the concentrate of DspB and Enterobactin produced by our engineered bacteria and the biofilm and rust removing time through calculating.</P></DIV><DIV class="contentbox"><H1 class="box-heading">2.Observations</H1><P>  Naturally, when there is a certain amount of HSL in the environment, HSL complex with afeR proteins and bind to afeR promoter which regulate positively the genes downstream (as shown on the Figure 1) and on that our sensing system relies to produce DspB and enterobactin.</P><FIGURE><FIGURE class="makeresponsive " style="margin-left: 20%; margin-right: 20%;width: 80%;"><FIGCAPTION><B style="margin-left: 20%; margin-right: 20%;width: 80%;">The mechanism of quorum sensing</B></FIGCAPTION></FIGURE></FIGURE></DIV><DIV class="contentbox"><H1 class="box-heading">3.Goals</H1><P>  Our goal of this model is to create a generic quorum sensing model so that:</P><P>  • We can determine the effect of afeR promoter and predict the production of DspB and enterobactin.</P><P>  • We can predict hao long our engineered bacteria would take to remove the biofilm and rust.</P></DIV><DIV class="contentbox"><H1 class="box-heading">4.Materials and Methods</H1><H2>4.1 HSL Transfer</H2><P>  HSL is produced by iron bacterias and realeased into the water environment. So the first step of our sensing is HSL transfering into our engineered <I>E.coli</I> from the water. And a passive transusion model is used for this process that the transfer rate of HSL can be described as this:</P><P>$$v_{diffuse,HSL,W-C}=K_{HSL,W-C}\left( \left[ HSL\right] _{W}-\left[ HSL\right] _{C}\right) $$</P><P>  • K<SUB>HSL,W-C</SUB> : transfer coefficient through the membrane (s<SUP>−1</SUP>)</P><P>  • We can predict hao long our engineered bacteria would take to remove the biofilm and rust.</P><H2>4.2 AfeR-HSL Complexation</H2><P>  AfeR is produced by engineered <I>E.coli</I> and functions in cell and its concentration is obtained approximating the number of protein per cell, using the <I>E.coli</I> concentration (cell/L) and the Avogadro number.</P><P>$$\left[ AfeR\right] _{C}=\left( Number of AfeR/cell\right) \cdot \dfrac {\left[ E.coli\right] }{N_{A}}$$</P><P>  The AfeR-HSL complexation is simply formed that way:</P><P>$$AfeR+HSL\leftrightarrow AfeR-HSL$$</P><P>  Assuming kinetics of AfeR-HSL complexation complexation is fast compared to the rest of the system, we assumed that the free and complexed forms are at equilibrum.</P><P>$$v_{complexation}=v_{dissociation}$$</P><P>$$k_{1}\cdot \left[ AfeR\right] _{C}\cdot \left[ HSL\right] _{C}=k_{2}\cdot \left[ AfeR-HSL\right] _{C}$$</P><P>$$\left[ AfeR-HSL\right] _{C}=\dfrac {\left[ AfeR\right] _{C}\cdot \left[ HSL\right] _{C}}{K_{eq,AfeR-HSL}}$$</P><P>$$K_{eq,AfeR-HSL}=k_{2}/k_{1}$$</P><P>  • K <SUB>eq, AfeR-HSL</SUB> : equilibrum constant of the AfeR-HSL complexation (mol/L)</P><H2>4.3 DspB Production</H2><P>  The production of the DspB from the DspB gene includes transcription and translation after activation. In addition, we should also consider its transport and degradation.</P><H3>4.3.1 DspB Gene Activation</H3><P>  This process is modeled using a Michaelian formalism depending on its activator (AfeR-HSL complexation) concentration. The promoter strength is also taken into account.</P><P>$$DspB_{DNA/cell}=DspB_{DNA0/cell}\cdot \dfrac {\left[ AfeR-HSL\right] _{C}}{K_{a,AfeR-HSL}+\left[ AfeR-HSL\right] _{C}}/cdot k_{p,afeR}$$</P><P>  • DspB <SUB>DNA,0/cell</SUB> : total number of DspB DNA per cell</P><P>  • DspB <SUB>DNA/cell</SUB> : number of activated DspB DNA per cell</P><P>  • K <SUB>a, AfeR-HSL</SUB> : activation constant of the AfeR-HSL complexation (mol/L)</P><P>  • k <SUB>p, afeR</SUB> : afeR promoter influence</P><H3>4.3.2 DspB Transcription</H3><P>  The DspB transcription depends on the transcription rate of the strain and the length of the DspB gene. The Avogadro number is used to express the transcription velocity in molar concentration in one cell per time unit.</P><P>$$v _{transcription,DspB mRNA}=\dfrac {DspB_{DNA/cell}\cdot k_{transcript}\cdot \left( RNA polymerase/gene\right) }{DNA length\cdot N_{A}\cdot V_{intracell}}$$</P><P>  • k<SUB>transcript</SUB> : <I>E.coli</I> transcription rate (nucleotides/s)</P><P>  • RNA polymerase/gene: number of RNA polymerase per gene</P><P>  • DNA length (DspB): number of nucleotides on the DspB gene</P><P>  • V <SUB>intracell</SUB>: volume of a bacterial cell (L)</P><P>  For the convenience of mathematical operation, we merged the ktranscript、RNA polymerase/gene and &quot;V&quot; intracell to a constant.</P><H3>4.3.3 DspB Translation</H3><P>  The DspB translation depends on the translation rate of the strain, the mRNA length and the quantity of mRNA. The translation velocity is expressed in molar concentration in one cell per time unit.</P><P>$$v _{translation,DspB}=\dfrac {\left[ DspB mRNA\right] \cdot k_{translation}\cdot \left( Ribosomes/RNA\right) }{RNA length}$$</P><P>  • k<SUB>translation</SUB> : <I>E.coli</I> translation rate (nucleotides/s)</P><P>  • Ribosomes/RNA: number of ribosomes per mRNA</P><P>  • RNA length (DspB): number of nucleotides on the DspB mRNA</P><P>  • [DspB mRNA] : DspB mRNA concentration in one <I>E.coli</I> cell</P><P>  For the convenience of mathematical operation, we merge the ktranslation and Ribosomes/RNA and to a constant.</P><H3>4.3.4 Degradation</H3><P>  Some of the DspB protein and mRNA are degraded. A degradation constant is used to model the degradation velocity.</P><P>$$v_{degradation,DspB}=K_{deg,DspB}\cdot \left[ DspB\right] _{C}$$</P><P>  • K<SUB>deg,DspB</SUB>: DspB degradation constant (s<SUP>−1</SUP>)</P><P>$$v_{degradation,DspB mRNA}=K_{deg,DspB mRNA}\cdot \left[ DspB mRNA\right] _{C}$$</P><P>  • K<SUB>deg,DspB mRNA</SUB>: DspB mRNA degradation constant (s<SUP>−1</SUP>)</P><H3>4.3.5 DspB Transfer</H3><P>  DspB protein needs to be transferred to the water environment to function. This process is taken into account through a passive transusion model.</P><P>$$v_{diffuse,DspB,C-W}=K_{DspB,C-W}\cdot \left( \left[ DspB\right] _{C}-\left[ DspB\right] _{W}\right) $$</P><P>  • K<SUB>DspB,C-W</SUB> : transfer coefficient through the membrane (s<SUP>−1</SUP>)</P><H2>4.4 Biofilm Removel</H2><P>  The biofilm is removed by the DspB and the process is modeled assuming a Michaelis-Menten kinetics.</P><P>$$v_{remo,biof}=k_{cat,DspB}\cdot \left[ DspB\right] _{W}\cdot \dfrac {\left[ Biof\right] }{k_{M,D}+\left[ Biof\right] }\cdot V_{intracell}\cdot \left[ E.coli\right] $$</P><P>  • k<SUB>cat,DspB</SUB> : catalytic constant of the DspB enzyme (s<SUP>−1</SUP>)</P><P>  • K<SUB>M,D</SUB>  : Michaelis constant of the DspB enzyme (mol/L)</P><H2>4.5 EntE Production</H2><P>  We treat enterobactin enzymes gene cluster as a whole gene (EntE gene). The production of the enterobactin enzymes from the EntE gene includes transcription and translation after activation. In addition, we should also consider its degradation. Because the enterobactin enzymes function in the cell, we don't need to consider its transport to the water environment.</P><H3>4.5.1 EntE Gene Activation</H3><P>  This process is modeled using a Michaelian formalism depending on its activator (AfeR-HSL complexation) concentration. The promoter strength is also taken into account.</P><P>$$EntE_{DNA/cell}=EntE_{DNA0/cell}\cdot \dfrac {\left[ AfeR-HSL\right] _{C}}{K_{a,AfeR-HSL}+\left[ AfeR-HSL\right] _{C}}\cdot k_{p,afeR}$$</P><P>  • EntE <SUB>DNA,0/cell</SUB> : total number of EntE DNA per cell</P><P>  • EntE <SUB>DNA/cell</SUB> : number of activated EntE DNA per cell</P><P>  • K a, <SUB>AfeR-HSL</SUB> : activation constant of the AfeR-HSL complexation (mol/L)</P><P>  • k p, <SUB>afeR</SUB> : afeR promoter influence</P><H3>4.5.2 EntE Transcription</H3><P>  The EntE transcription depends on the transcription rate of the strain and the length of the EntE gene. The Avogadro number is used to express the transcription velocity in molar concentration in one cell per time unit.</P><P>$$v_{transcription,EntE mRNA}=\dfrac {EntE_{DNA/cell}\cdot k_{transcript}\cdot \left( RNA polymerase/gene\right) }{DNA length\cdot N_{A}\cdot V_{intracell}}$$</P><P>  • EntE <SUB>DNA,/cell</SUB>  : number of EntE gene per cell</P><P>  • k<SUB>transcript</SUB> : <I>E.coli</I> transcription rate (nucleotides/s)</P><P>  • RNA polymerase/gene: number of RNA polymerase per gene</P><P>  • DNA length (EntE): number of nucleotides on the EntE gene</P><P>  • V<SUB>intracell</SUB> : volume of a bacterial cell (L)</P><P>  For the convenience of mathematical operation, we merged the k<SUB>transcript</SUB>、RNA polymerase/gene and V <SUB>intracell</SUB> to a constant.</P><H3>4.5.3 EntE Translation</H3><P>  The EntE translation depends on the translation rate of the strain, the mRNA length and the quantity of mRNA. The translation velocity is expressed in molar concentration in one cell per time unit.</P><P>$$v_{translation,EntE}=\dfrac {\left[ EntE mRNA\right] \cdot k_{translation}\cdot \left( Ribosomes/RNA\right) }{RNA length}$$</P><P>  • k<SUB>translation</SUB> : <I>E.coli</I> translation rate (nucleotides/s)</P><P>  • Ribosomes/RNA: number of ribosomes per mRNA</P><P>  • RNA length (EntE): number of nucleotides on the EntE mRNA</P><P>  • [EntE mRNA] : EntE mRNA concentration in one <I>E.coli</I> cell</P><P>  For the convenience of mathematical operation, we merge the k<SUB>translation</SUB> and Ribosomes/RNA and to a constant.</P><H3>4.5.4 Degradation</H3><P>  Some of the EntE protein and mRNA are degraded. A degradation constant is used to model the degradation velocity.</P><P>$$v_{degradation,EntE}=K_{deg,EntE}\cdot \left[ EntE\right] _{C}$$</P><P>  • K<SUB>deg,EntE</SUB>: EntE degradation constant (s<SUP>−1</SUP>)</P><P>$$v_{degradation,EntE mRNA}=K_{deg,EntE mRNA}\cdot \left[ EntE mRNA\right] _{C}$$</P><P>  • K<SUB>deg,EntE mRNA</SUB>: EntE mRNA degradation constant (s<SUP>−1</SUP>)</P><H2>4.6 Enterobactin Production</H2><H3>4.6.1 Enterobactin Production</H3><P>  Enterobactin is produced by <I>E.coli</I> through the reaction catalyzed by EntE and is modeled assuming a Michaelis-Menten kinetics.</P><P>$$v_{prod,EntE}=k_{cat,EntE}\cdot \left[ EntE\right] _{C}\cdot \dfrac {\left[ S\right] _{C}}{K_{M,E}+\left[ S\right] _{C}}\cdot V_{intracell}\cdot \left[ E.coli\right] $$</P><P>  • [EntE]<SUB>C</SUB> : EntE enzyme concentration in one <I>E.coli</I> cell (mol/L)</P><P>  • k <SUB>cat,EntE</SUB> : catalytic constant of the EntE enzyme (s<SUP>−1</SUP>)</P><P>  • [S]<SUB>C</SUB> : substrate concentration (mol/L)</P><P>  • K<SUB>M,E</SUB> : Michaelis constant of the EntE enzyme (mol/L)</P><H3>4.6.2 Enterobactin Transfer</H3><P>  Enterobactin needs to be transferred to the water environment to function. This process is taken into account through a passive transusion model.</P><P>$$v_{diffuse,Ent,C-W}=K_{Ent,C-W}\cdot \left( \left[ Ent\right] _{C}-\left[ Ent\right] _{W}\right) $$</P><P>  • K<SUB>DspB,C-W</SUB> : transfer coefficient through the membrane (s<SUP>−1</SUP>)</P><H2>4.7 Rust Removel</H2><P>  The rust is removed by the chelation of enterobactin.</P><P>$$Ent+Fe\left( OH\right) _{3}\rightarrow Ent-Fe^{3+}+3OH^{-}$$</P><P>  The equilibrum constant of this formula can be written as:</P><P>$$K=\dfrac {\left[ Ent-Fe^{3+}\right] \cdot \left[ OH^{-}\right] ^{3}}{\left[ Ent\right] }$$</P><P>$$=K_{Ent-Fe}\cdot K_{sp-Fe\left( OH\right) 3}$$</P><P>  • K<SUB>Ent-Fe</SUB> : chelation coefficient of enterobactin to Fe<SUP>3+</SUP> (M<SUP>−1</SUP>)</P><P>  • K<SUB>sp,Fe(OH)3</SUB> : precipitation coefficient of Fe(OH)<SUB>3</SUB> (s<SUP>−1</SUP>)</P><P>  And in this formula, </P><P>$$\left[ OH^{-}\right] =3\left[ Ent-Fe^{3+}\right] $$</P><P>  So the the concentration of Ent-Fe<SUP>3+</SUP> can be written as:</P><P>$$\left[ Ent-Fe^{3+}\right] =\left( K\cdot \left[ Ent\right] /27\right) ^{0.25}$$</P><P>  And amount of rust can be showed:</P><P>$$\left[ Rust\right] =\left[ Rust\right] _{0}-\left[ Ent-Fe^{3+}\right]$$</P></DIV><DIV class="contentbox"><H1 class="box-heading">5. Solver</H1><P>  The system of ODEs was solved using Matlab R2016a. And we used the ode15s solver.</P><P>  The complete set of ODEs is detailed here:</P><P>$$\dfrac {d\left[ HSL\right] _{C}}{dt}=v_{diffuse,HSL,W-C}$$</P><P>$$\dfrac {d\left[ DspB-mRNA\right] _{C}}{dt}=v _{transcription,DspB mRNA}-v_{degradation,DspB mRNA}$$</P><P>$$\dfrac {d\left[ DspB\right] _{C}}{dt}=v _{translation,DspB}-v_{degradation,DspB}$$</P><P>$$\dfrac {d\left[ Biof\right]}{dt}=-v_{remo,biof}$$</P><P>$$\dfrac {d\left[ EntE mRNA\right] _{C}}{dt}=v _{transcription,EntE mRNA}-v_{degradation,EntE mRNA}$$</P><P>$$\dfrac {d\left[ EntE\right] _{C}}{dt}=v _{translation,EntE}-v_{degradation,EntE}$$</P><P>$$\dfrac {d\left[ Ent\right] _{C}}{dt}=v_{prod,Ent}$$</P><P>$$\dfrac {d\left[ Ent\right] _{W}}{dt}=v_{diffuse,Ent,C-W}$$</P><P>You can freely re-use our code:<A target="_blank" style="color:white; text-decoration:underline;" href="https://static.igem.org/mediawiki/2018/1/1e/T--ECUST--QS_IGEM-MATLAB.zip"><I>General_resolution + System_of_ODEs.</I></A></P></DIV><DIV class="contentbox"><H1 class="box-heading">6. Result</H1><P>  At the beginning of the project, we needed to know if our quorum sensing would work in practice and if the information transmission between the transerent modules was possible and sufficiently fast. We thus carried out simulations by solving the ODEs system to have a first estimation of the dynamics of our synthetic system.</P><P>  The initial conditions, such as the concentrations of <I>E.coli</I> and HSL were set to biologically plausible values.</P><P>  [HSL]<SUB>W</SUB> = 10<SUP>-5</SUP> mol/L</P><P>  [E.coli] = 1.66*10<SUP>-12</SUP> mol/L (10<SUP>12</SUP> cell/L, OD<SUB>600</SUB>=1.5)</P><P>  [Biof]<SUB>0</SUB> = 1 (amount)</P><P>  [Rust]<SUB>0</SUB> = 1 (amount)</P><FIGURE><FIGURE class="makeresponsive floatleft" style="margin-left: 20%; margin-right: 20%;width: 80%;"><FIGCAPTION><B margin-left:="" 20="">The variety of biofilm by modeling.</B></FIGCAPTION></FIGURE><FIGURE><FIGURE class="makeresponsive floatleft" style="margin-left: 20%; margin-right: 20%;width: 80%;"><FIGCAPTION><B margin-left:="" 20="">The variety of rust by modeling.</B></FIGCAPTION></FIGURE><P>  The model result shows that usinng our engineered <I>E.coli</I> to remove a certain amount of biofilm needs about 5 days and to remove a certain amount of rust needs less than 4.5 days. The result is close to the real value which confirmes the feasibility of our project .</P><FIGURE><FIGURE class="makeresponsive floatleft" style="margin-left: 20%; margin-right: 20%;width: 80%;"><FIGCAPTION><B margin-left:="" 20="">The variety of DspB\EntE\Ent\Ent-Fe3+ by modeling.</B></FIGCAPTION></FIGURE><P>  This visual representation of the system's dynamics also allowed us to check that each variable evolves in a realistic range of concentrations, hence indicating the model predicts a consistent behavior.</P></FIGURE></FIGURE></FIGURE></DIV><DIV class="contentbox"><H1 class="box-heading">7. Addendum</H1><P>  Data and Parameter:</P><P><A target="_blank" style="color:white; text-decoration:underline;" href="https://static.igem.org/mediawiki/2018/7/76/T--ECUST--result--QUORUM_MODEL_constant.docx"><I>Click here to download the table.</I></A></P></DIV><DIV class="contentbox"><H1 class="box-heading">References</H1><P>1. QIAGEN, Origins of replication and copy numbers of various plasmids and cosmids In: Growth Of Bacterial Cultures, 2013 - 2017.</P><P>2. Esquerre T, Moisan A, Chiapello H, Arike L, Vilu R, Gaspin C, Cocaign-Bousquet M, Girbal L, Genome-wide investigation of mRNA lifetime determinants in <I>Escherichia coli</I> cells cultured at different growth rates. BMC Genomics. 2015, 16, 275,</P><P>3. Li Y C, Zhu J R. Role of N-acyl homoserine lactone (HSL)-based quorum sensing (QS) in aerobic sludge granulation.[J]. Applied Microbiology &amp; Biotechnology, 2014, 98(17):7623-7632.</P><P>4. Xiang Chen,Faming Zhu,Yunhe Cao,et al. Novel Expression Vector for Secretion of Cecropin AD in Bacillus subtilis with Enhanced Antimicrobial Activity[J]. Antimicrobial Agents and Chemotherapy,2009,53(9):3683-3689.</P><P>5. (Milo R, Jorgensen P, Moran U, Weber G, Springer M. BioNumbers—the database of key numbers in molecular and cell biology. Nucleic Acids Res. 2010;38(suppl 1):D750–D753. </P><P>Bionumber: 103021\107727\100197\114111\101440</P><P>6. Carrano CJ, KN R (1979) Ferric Ion Sequestering Agents. 2. Kinetics and Mechanism of Iron Removal from Transferrin by Enterobactin and Synthetic Tricatechols. J Am Chem Soc 101: 5401–5404.</P></DIV></MAIN></DIV><FOOTER><DIV id="ufo"><A class="nav-text back-to-top" href="#" style="align-self: flex-end; padding-bottom: 250px; font-size: 18px; color:#23d487; font-family:'eras';">Bring me up!</A></DIV><DIV class="footerbar" style="width:100%"><TABLE><TBODY><TR><TD><A style="text-decoration: none;
    color: white; href=" https:="">iGEM-Team ECUST<SMALL>EAST CHINA UNIVERSITY OF SCIENCE AND TECHNOLOGY Shanghai</SMALL>China</A></TD></TR></TBODY></TABLE></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>