Team:ECUST/Quorum Sensing
HOME
TEAM
TEAM
ATTRIBUTION
SPONSORS
PROJECT
OVERVIEW
BACKGROUND
DESIGN
QUORUM SENSING
RUST REMOVER
BIOFILM REMOVER
FUR INVERTER
LYSIN
BIOCIDE
LIGHT-ON SUICIDE
DEMONSTRATION
PARTS
BASIC PARTS
COMPOSITE PARTS
MODEL
OVERVIEW
QUORUM SENSING
IRON SENSING
DEVICE
HARDWARE
SOFTWARE
HUMAN PRACTICE
OVERVIEW
INTEGRATED HUMAN  PRACTICE
COMMUNICATION
EDUCATION & PUBLIC ENGAGEMENT
APPLIED DESIGN
COLLABORATION
BIOSAFETY
NOTEBOOK
PROTOCOL
JUDGING FORM
INTERLAB
Quorum Sensing Model
1.Introduction
   To understand, predict and ultimately control the behavior of our engineered microbial group effect, we have developed dynamic model of the system, based on transerential equations which describe and integrate the individual processes. This model involves several entities going from the molecular level (genes, RNAs, proteins, and metabolites) up to the cellular and population levels, distinct intracellular and extracellular compartments, and a wide range of biological and physical processes (transcription, translation, signalling, growth, transusion, etc). Here we can show the concentrate of DspB and Enterobactin produced by our engineered bacteria and the biofilm and rust removing time through calculating.
2.Observations
  Naturally, when there is a certain amount of HSL in the environment, HSL complex with afeR proteins and bind to afeR promoter which regulate positively the genes downstream (as shown on the Figure 1) and on that our sensing system relies to produce DspB and enterobactin.
The mechanism of quorum sensing
3.Goals
  Our goal of this model is to create a generic quorum sensing model so that:
  • We can determine the effect of afeR promoter and predict the production of DspB and enterobactin.
  • We can predict hao long our engineered bacteria would take to remove the biofilm and rust.
4.Materials and Methods
4.1 HSL Transfer
  HSL is produced by iron bacterias and realeased into the water environment. So the first step of our sensing is HSL transfering into our engineered E.coli from the water. And a passive transusion model is used for this process that the transfer rate of HSL can be described as this:
$$v_{diffuse,HSL,W-C}=K_{HSL,W-C}\left( \left[ HSL\right] _{W}-\left[ HSL\right] _{C}\right) $$
  • KHSL,W-C : transfer coefficient through the membrane (s−1)
  • We can predict hao long our engineered bacteria would take to remove the biofilm and rust.
4.2 AfeR-HSL Complexation
  AfeR is produced by engineered E.coli and functions in cell and its concentration is obtained approximating the number of protein per cell, using the E.coli concentration (cell/L) and the Avogadro number.
$$\left[ AfeR\right] _{C}=\left( Number of AfeR/cell\right) \cdot \dfrac {\left[ E.coli\right] }{N_{A}}$$
  The AfeR-HSL complexation is simply formed that way:
$$AfeR+HSL\leftrightarrow AfeR-HSL$$
  Assuming kinetics of AfeR-HSL complexation complexation is fast compared to the rest of the system, we assumed that the free and complexed forms are at equilibrum.
$$v_{complexation}=v_{dissociation}$$
$$k_{1}\cdot \left[ AfeR\right] _{C}\cdot \left[ HSL\right] _{C}=k_{2}\cdot \left[ AfeR-HSL\right] _{C}$$
$$\left[ AfeR-HSL\right] _{C}=\dfrac {\left[ AfeR\right] _{C}\cdot \left[ HSL\right] _{C}}{K_{eq,AfeR-HSL}}$$
$$K_{eq,AfeR-HSL}=k_{2}/k_{1}$$
  • K eq, AfeR-HSL : equilibrum constant of the AfeR-HSL complexation (mol/L)
4.3 DspB Production
  The production of the DspB from the DspB gene includes transcription and translation after activation. In addition, we should also consider its transport and degradation.
4.3.1 DspB Gene Activation
  This process is modeled using a Michaelian formalism depending on its activator (AfeR-HSL complexation) concentration. The promoter strength is also taken into account.
$$DspB_{DNA/cell}=DspB_{DNA0/cell}\cdot \dfrac {\left[ AfeR-HSL\right] _{C}}{K_{a,AfeR-HSL}+\left[ AfeR-HSL\right] _{C}}/cdot k_{p,afeR}$$
  • DspB DNA,0/cell : total number of DspB DNA per cell
  • DspB DNA/cell : number of activated DspB DNA per cell
  • K a, AfeR-HSL : activation constant of the AfeR-HSL complexation (mol/L)
  • k p, afeR : afeR promoter influence
4.3.2 DspB Transcription
  The DspB transcription depends on the transcription rate of the strain and the length of the DspB gene. The Avogadro number is used to express the transcription velocity in molar concentration in one cell per time unit.
$$v _{transcription,DspB mRNA}=\dfrac {DspB_{DNA/cell}\cdot k_{transcript}\cdot \left( RNA polymerase/gene\right) }{DNA length\cdot N_{A}\cdot V_{intracell}}$$
  • ktranscript : E.coli transcription rate (nucleotides/s)
  • RNA polymerase/gene: number of RNA polymerase per gene
  • DNA length (DspB): number of nucleotides on the DspB gene
  • V intracell: volume of a bacterial cell (L)
  For the convenience of mathematical operation, we merged the ktranscript、RNA polymerase/gene and "V" intracell to a constant.
4.3.3 DspB Translation
  The DspB translation depends on the translation rate of the strain, the mRNA length and the quantity of mRNA. The translation velocity is expressed in molar concentration in one cell per time unit.
$$v _{translation,DspB}=\dfrac {\left[ DspB mRNA\right] \cdot k_{translation}\cdot \left( Ribosomes/RNA\right) }{RNA length}$$
  • ktranslation : E.coli translation rate (nucleotides/s)
  • Ribosomes/RNA: number of ribosomes per mRNA
  • RNA length (DspB): number of nucleotides on the DspB mRNA
  • [DspB mRNA] : DspB mRNA concentration in one E.coli cell
  For the convenience of mathematical operation, we merge the ktranslation and Ribosomes/RNA and to a constant.
4.3.4 Degradation
  Some of the DspB protein and mRNA are degraded. A degradation constant is used to model the degradation velocity.
$$v_{degradation,DspB}=K_{deg,DspB}\cdot \left[ DspB\right] _{C}$$
  • Kdeg,DspB: DspB degradation constant (s−1)
$$v_{degradation,DspB mRNA}=K_{deg,DspB mRNA}\cdot \left[ DspB mRNA\right] _{C}$$
  • Kdeg,DspB mRNA: DspB mRNA degradation constant (s−1)
4.3.5 DspB Transfer
  DspB protein needs to be transferred to the water environment to function. This process is taken into account through a passive transusion model.
$$v_{diffuse,DspB,C-W}=K_{DspB,C-W}\cdot \left( \left[ DspB\right] _{C}-\left[ DspB\right] _{W}\right) $$
  • KDspB,C-W : transfer coefficient through the membrane (s−1)
4.4 Biofilm Removel
  The biofilm is removed by the DspB and the process is modeled assuming a Michaelis-Menten kinetics.
$$v_{remo,biof}=k_{cat,DspB}\cdot \left[ DspB\right] _{W}\cdot \dfrac {\left[ Biof\right] }{k_{M,D}+\left[ Biof\right] }\cdot V_{intracell}\cdot \left[ E.coli\right] $$
  • kcat,DspB : catalytic constant of the DspB enzyme (s−1)
  • KM,D : Michaelis constant of the DspB enzyme (mol/L)
4.5 EntE Production
  We treat enterobactin enzymes gene cluster as a whole gene (EntE gene). The production of the enterobactin enzymes from the EntE gene includes transcription and translation after activation. In addition, we should also consider its degradation. Because the enterobactin enzymes function in the cell, we don't need to consider its transport to the water environment.
4.5.1 EntE Gene Activation
  This process is modeled using a Michaelian formalism depending on its activator (AfeR-HSL complexation) concentration. The promoter strength is also taken into account.
$$EntE_{DNA/cell}=EntE_{DNA0/cell}\cdot \dfrac {\left[ AfeR-HSL\right] _{C}}{K_{a,AfeR-HSL}+\left[ AfeR-HSL\right] _{C}}\cdot k_{p,afeR}$$
  • EntE DNA,0/cell : total number of EntE DNA per cell
  • EntE DNA/cell : number of activated EntE DNA per cell
  • K a, AfeR-HSL : activation constant of the AfeR-HSL complexation (mol/L)
  • k p, afeR : afeR promoter influence
4.5.2 EntE Transcription
  The EntE transcription depends on the transcription rate of the strain and the length of the EntE gene. The Avogadro number is used to express the transcription velocity in molar concentration in one cell per time unit.
$$v_{transcription,EntE mRNA}=\dfrac {EntE_{DNA/cell}\cdot k_{transcript}\cdot \left( RNA polymerase/gene\right) }{DNA length\cdot N_{A}\cdot V_{intracell}}$$
  • EntE DNA,/cell : number of EntE gene per cell
  • ktranscript : E.coli transcription rate (nucleotides/s)
  • RNA polymerase/gene: number of RNA polymerase per gene
  • DNA length (EntE): number of nucleotides on the EntE gene
  • Vintracell : volume of a bacterial cell (L)
  For the convenience of mathematical operation, we merged the ktranscript、RNA polymerase/gene and V intracell to a constant.
4.5.3 EntE Translation
  The EntE translation depends on the translation rate of the strain, the mRNA length and the quantity of mRNA. The translation velocity is expressed in molar concentration in one cell per time unit.
$$v_{translation,EntE}=\dfrac {\left[ EntE mRNA\right] \cdot k_{translation}\cdot \left( Ribosomes/RNA\right) }{RNA length}$$
  • ktranslation : E.coli translation rate (nucleotides/s)
  • Ribosomes/RNA: number of ribosomes per mRNA
  • RNA length (EntE): number of nucleotides on the EntE mRNA
  • [EntE mRNA] : EntE mRNA concentration in one E.coli cell
  For the convenience of mathematical operation, we merge the ktranslation and Ribosomes/RNA and to a constant.
4.5.4 Degradation
  Some of the EntE protein and mRNA are degraded. A degradation constant is used to model the degradation velocity.
$$v_{degradation,EntE}=K_{deg,EntE}\cdot \left[ EntE\right] _{C}$$
  • Kdeg,EntE: EntE degradation constant (s−1)
$$v_{degradation,EntE mRNA}=K_{deg,EntE mRNA}\cdot \left[ EntE mRNA\right] _{C}$$
  • Kdeg,EntE mRNA: EntE mRNA degradation constant (s−1)
4.6 Enterobactin Production
4.6.1 Enterobactin Production
  Enterobactin is produced by E.coli through the reaction catalyzed by EntE and is modeled assuming a Michaelis-Menten kinetics.
$$v_{prod,EntE}=k_{cat,EntE}\cdot \left[ EntE\right] _{C}\cdot \dfrac {\left[ S\right] _{C}}{K_{M,E}+\left[ S\right] _{C}}\cdot V_{intracell}\cdot \left[ E.coli\right] $$
  • [EntE]C : EntE enzyme concentration in one E.coli cell (mol/L)
  • k cat,EntE : catalytic constant of the EntE enzyme (s−1)
  • [S]C : substrate concentration (mol/L)
  • KM,E : Michaelis constant of the EntE enzyme (mol/L)
4.6.2 Enterobactin Transfer
  Enterobactin needs to be transferred to the water environment to function. This process is taken into account through a passive transusion model.
$$v_{diffuse,Ent,C-W}=K_{Ent,C-W}\cdot \left( \left[ Ent\right] _{C}-\left[ Ent\right] _{W}\right) $$
  • KDspB,C-W : transfer coefficient through the membrane (s−1)
4.7 Rust Removel
  The rust is removed by the chelation of enterobactin.
$$Ent+Fe\left( OH\right) _{3}\rightarrow Ent-Fe^{3+}+3OH^{-}$$
  The equilibrum constant of this formula can be written as:
$$K=\dfrac {\left[ Ent-Fe^{3+}\right] \cdot \left[ OH^{-}\right] ^{3}}{\left[ Ent\right] }$$
$$=K_{Ent-Fe}\cdot K_{sp-Fe\left( OH\right) 3}$$
  • KEnt-Fe : chelation coefficient of enterobactin to Fe3+ (M−1)
  • Ksp,Fe(OH)3 : precipitation coefficient of Fe(OH)3 (s−1)
  And in this formula,
$$\left[ OH^{-}\right] =3\left[ Ent-Fe^{3+}\right] $$
  So the the concentration of Ent-Fe3+ can be written as:
$$\left[ Ent-Fe^{3+}\right] =\left( K\cdot \left[ Ent\right] /27\right) ^{0.25}$$
  And amount of rust can be showed:
$$\left[ Rust\right] =\left[ Rust\right] _{0}-\left[ Ent-Fe^{3+}\right]$$
5. Solver
  The system of ODEs was solved using Matlab R2016a. And we used the ode15s solver.
  The complete set of ODEs is detailed here:
$$\dfrac {d\left[ HSL\right] _{C}}{dt}=v_{diffuse,HSL,W-C}$$
$$\dfrac {d\left[ DspB-mRNA\right] _{C}}{dt}=v _{transcription,DspB mRNA}-v_{degradation,DspB mRNA}$$
$$\dfrac {d\left[ DspB\right] _{C}}{dt}=v _{translation,DspB}-v_{degradation,DspB}$$
$$\dfrac {d\left[ Biof\right]}{dt}=-v_{remo,biof}$$
$$\dfrac {d\left[ EntE mRNA\right] _{C}}{dt}=v _{transcription,EntE mRNA}-v_{degradation,EntE mRNA}$$
$$\dfrac {d\left[ EntE\right] _{C}}{dt}=v _{translation,EntE}-v_{degradation,EntE}$$
$$\dfrac {d\left[ Ent\right] _{C}}{dt}=v_{prod,Ent}$$
$$\dfrac {d\left[ Ent\right] _{W}}{dt}=v_{diffuse,Ent,C-W}$$
You can freely re-use our code: General_resolution + System_of_ODEs.
6. Result
  At the beginning of the project, we needed to know if our quorum sensing would work in practice and if the information transmission between the transerent modules was possible and sufficiently fast. We thus carried out simulations by solving the ODEs system to have a first estimation of the dynamics of our synthetic system.
  The initial conditions, such as the concentrations of E.coli and HSL were set to biologically plausible values.
  [HSL]W = 10-5 mol/L
  [E.coli] = 1.66*10-12 mol/L (1012 cell/L, OD600=1.5)
  [Biof]0 = 1 (amount)
  [Rust]0 = 1 (amount)
The variety of biofilm by modeling.
The variety of rust by modeling.
  The model result shows that usinng our engineered E.coli to remove a certain amount of biofilm needs about 5 days and to remove a certain amount of rust needs less than 4.5 days. The result is close to the real value which confirmes the feasibility of our project .
The variety of DspB\EntE\Ent\Ent-Fe3+ by modeling.
  This visual representation of the system's dynamics also allowed us to check that each variable evolves in a realistic range of concentrations, hence indicating the model predicts a consistent behavior.
7. Addendum
  Data and Parameter:
Click here to download the table.
References
1. QIAGEN, Origins of replication and copy numbers of various plasmids and cosmids In: Growth Of Bacterial Cultures, 2013 - 2017.
2. Esquerre T, Moisan A, Chiapello H, Arike L, Vilu R, Gaspin C, Cocaign-Bousquet M, Girbal L, Genome-wide investigation of mRNA lifetime determinants in Escherichia coli cells cultured at different growth rates. BMC Genomics. 2015, 16, 275,
3. Li Y C, Zhu J R. Role of N-acyl homoserine lactone (HSL)-based quorum sensing (QS) in aerobic sludge granulation.[J]. Applied Microbiology & Biotechnology, 2014, 98(17):7623-7632.
4. Xiang Chen,Faming Zhu,Yunhe Cao,et al. Novel Expression Vector for Secretion of Cecropin AD in Bacillus subtilis with Enhanced Antimicrobial Activity[J]. Antimicrobial Agents and Chemotherapy,2009,53(9):3683-3689.
5. (Milo R, Jorgensen P, Moran U, Weber G, Springer M. BioNumbers—the database of key numbers in molecular and cell biology. Nucleic Acids Res. 2010;38(suppl 1):D750–D753.
Bionumber: 103021\107727\100197\114111\101440
6. Carrano CJ, KN R (1979) Ferric Ion Sequestering Agents. 2. Kinetics and Mechanism of Iron Removal from Transferrin by Enterobactin and Synthetic Tricatechols. J Am Chem Soc 101: 5401–5404.
Bring me up!
iGEM-Team ECUST
EAST CHINA UNIVERSITY OF SCIENCE AND TECHNOLOGY
Shanghai
China
