<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_HP_FutureScenarios skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/HP/FutureScenarios</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="content_wrapper"><HEADER id="content-header"><SPAN id="logo-container" class="menu-item"><DIV class="menu-title"><A href="/Team:Groningen">CryptoGE®M</A></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Team</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Team">Members</A><A href="/Team:Groningen/Supervisors">Supervisors</A><A href="/Team:Groningen/Collaborations">Collaborations</A><A href="/Team:Groningen/Achievements">Achievements</A><A href="/Team:Groningen/Contact">Contact</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Project</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Tour"><STRONG>Tour</STRONG></A><A href="/Team:Groningen/Description">Description</A><A href="/Team:Groningen/Design">Design</A><A href="/Team:Groningen/Results">Results</A><A href="/Team:Groningen/Demonstrate">Demonstrate</A><A href="/Team:Groningen/Applications">Applications</A><A href="/Team:Groningen/FuturePerspectives">Future</A><A href="/Team:Groningen/Notebook">Notebook</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Biology</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Experiments">Experiments</A><A href="/Team:Groningen/Parts">Parts</A><A href="/Team:Groningen/Safety">Safety</A><A href="/Team:Groningen/Labjournal">Lab journal</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Computing</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Model">Modelling</A><A href="/Team:Groningen/Software">Software</A><A href="/Team:Groningen/Encoding">Encoding</A><A href="/Team:Groningen/Decoding">Decoding</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Human Practices</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Human_Practices">Human Practices</A><A href="/Team:Groningen/HP/Gold">Gold</A><A href="/Team:Groningen/HP/Silver">Silver</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Support</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Sponsors">Sponsors</A><A href="/Team:Groningen/Attributions">Attributions</A></DIV></DIV></SPAN></HEADER><DIV id="groningen-wrapper"><ARTICLE id="hp-future" class="collapse-h2 collapse-conditional"><SECTION><H2>Future scenarios</H2></SECTION><SECTION><BLOCKQUOTE> “Planning is bringing the future into the present so that 
		you can do something about it” - <STRONG> Alan Lakein </STRONG></BLOCKQUOTE><P>What would be the potential applications of CryptoGErM in the near 
		and far future?</P><P>In this part of Human Practices, we explore the future scenarios of 
		the introduction of the CryptoGErM system to the world. </P><P>With all these fictional scenarios we give a deep insight on the 
		repercussions, applications and consequences that CryptoGErM would have 
		if introduced as a daily life system. 
		</P><P>The scenarios are presented as fictional digital news and articles 
		from a futuristic and unreal city called <STRONG>New 
		Groningen</STRONG> in The Netherlands in the <STRONG>near and far 
		future,</STRONG> where the CryptoGErM system has been introduced, 
		followed by an explanation of the current reasons behind its 
		conceptualization. </P><P> We present them as digital news in a mobile device, based on the 
		prediction that the UN Intellectual property agency’s chief made in an 
		interview to La Tribune de Geneve, stating that newspapers will be 
		replaced by digital versions by 2040.</P><BLOCKQUOTE> &quot;In a few years, there will no longer be printed 
		newspapers as we know it today” “It’s an evolution. There’s no good or 
		bad about it. There are studies showing that they will disappear by 
		2040” - <STRONG> Francis Curry </STRONG></BLOCKQUOTE></SECTION><SECTION><H3 class="centrate">The last existing Public Library has disappeared!</H3><FIGURE class="centrate"><FIGCAPTION class="centrate"> First fictional scenario </FIGCAPTION></FIGURE><P>What if in the far future we start to run out of space to 
		accommodate the growing world population? Would we be able to find 
		suitable places to inhabit without damaging natural areas? What if we 
		use existent spaces to accommodate the growing population without the 
		necessity of building new human settlements protecting the green areas? 
		Have we thought about it? </P><DIV class="split"><DIV class="left flone"><P>According to the UN DESA report; “World Population Prospects: The 
				2015 Revision”, by 2050 the world population is expected to reach to 
				9.7 billion; the available living space could not be sustainable to 
				accommodate the rapidly-growing amount of people. Such raising in the 
				population, probably, will see humanity looking for more 
				space to make sustainable the living of new generations, without using 
				the green areas. </P><P>Within this fictional scenario, CryptoGErM could offer a potential 
				solution.</P><P>CryptoGErM is a system for the storing and safe-sending of 
				information. By making use of the first purpose, we could store the 
				whole current information contained in books, newspapers, articles and 
				other physical paper-based within bacteria to reduce the 
				space needed for the storing of such information. Bacteria, such 
				<EM>Bacillus subtilis</EM> is able to store up to 1.5 GB of 
				information; furthermore, 1 mm<SUP>3</SUP> of DNA can store up to 
				10<SUP>9</SUP> GB of information. <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP></P><P>Just to put an example of this fictional application; it is 
				estimated that the the information contained by the U.S. Library of 
				Congress is equivalent to 200 TB, currently it is stored in a physical 
				space of 186,000 square meter. If this information is stored into DNA, 
				it would only occupy 0.0002 mm<SUP>3</SUP>.</P><P>Furthermore, storing int DNA is more eco-friendly, because it does 
				not require any specific medium to keep it functional, in contrast with 
				current magnetic and optical storage medium that requires energetic 
				supply to continue working. Just in 2015, 416.2 terawatt hours of 
				electricity were used by world’s data centres. This consumption was 
				“significantly higher than the UK’s total consumption of about 300 
				terawatt hours.” <SUP id="ref-2"><A href="#cite-2">[2]</A></SUP></P></DIV><FIGURE class="centrate right flone"><FIGCAPTION class="centrate"> Storing information in the DNA of bacteria, will replace the current storing medium (Fictional scenario)</FIGCAPTION></FIGURE></DIV></SECTION><SECTION><H3 class="centrate">Spores of <EM>Bacillus subtilis</EM> from 2016 found in the New Groningen</H3><FIGURE class="centrate"><FIGCAPTION class="centrate"> Second fictional scenario </FIGCAPTION></FIGURE><P> This fictional scenario was created as an extreme case by using the 
		long-term stability of <EM>Bacillus subtilis </EM>spores. </P><P>One of the advantages of our system is that the spores can survive 
		for several years even if they are exposed to hard conditions. So far, 
		studies have shown that spores exhibit incredible longevity in a broad 
		range of environments on our planet. Actually, there are reports from 
		research groups that have recovered and revived spores from samples as 
		old as 10<SUP>5</SUP> years. This data shows the capability of our 
		system to store information for hundreds or thousands of years, even in 
		a hard-conditions environment.</P><P>This application seems promising in a near future, as we found out 
		after the interview we had at the Groninger Archives; Jan Jaap Hoogstins, 
		collection manager from the Groninger Archieven, explained us that now 
		most of the documents are being changed from analogous data storage to 
		digital data storage, and one of two most important things that a 
		storage system must fulfil is the durability, which means that the information 
		stored must be conserved for a long period of time. </P><P>Spores of <EM>Bacillus subtilis</EM><STRONG>are a reliable 
		option</STRONG> since they have shown its long-term resistance. 
		Actually, in 2000, a spore-forming <EM>Bacillus</EM> strain was 
		isolated from a 250 million years old salt crystals. Moreover, DNA itself is a 
		more stable data storage medium when compared to magnetic and optical 
		medial, because it can remain intact for at least 700 000 years at 
		-4˚C.<SUP id="ref-3"><A href="#cite-3">[3]</A></SUP> Even in harsh 
		environments, DNA has a half-life of over 500 years. <SUP id="ref-4"><A href="#cite-4">[4]</A></SUP> Contrastly, current magnetic storers of 
		information have lasts at most 30 years.<SUP id="ref-5"><A href="#cite-5">[5]</A></SUP></P><P>However, is the DNA that contains the encrypted information in bacteria able to mutate through this period of time? </P><P>A complete discussion about mutation in <EM>Bacillus subtilis</EM> 
		spores can be found in the <A href="/Team:Groningen/Model">Modelling Section</A>.</P></SECTION><SECTION><H3 class="centrate">A complete movie has been successfully stored in DNA</H3><FIGURE class="centrate"><FIGCAPTION class="centrate"> Third fictional scenario </FIGCAPTION></FIGURE><P>This fictional and extreme scenario provides an example of the 
		future applications of CryptoGErM based on the current files and 
		archives that have been stored successfully in DNA.</P><P>Currently, big companies such as Microsoft® have shown interest in the 
		storage of information in DNA. This gives us an idea that DNA storing 
		has started to become one of the most promising methods to store 
		information in the upcoming years. This is mainly given by the capacity of 
		storage that DNA posses; Church, Gao and Kosuri showed that 0.5 
		exabytes of information can be stored per gram of single-stranded 
		DNA.<SUP id="ref-6"><A href="#cite-6">[6]</A></SUP></P><P>A big breakthrough happened in April 2016, when Microsoft published an extensive 
		paper in a joint project with the University of Washington<SUP id="ref-7"><A href="#cite-7">[7]</A></SUP> in which they explore the 
		potential application for the use of DNA as storage medium. 
		According to this work, DNA storage has a higher 
		durability in comparison with current systems. Within the applications 
		of storing information in DNA, this group stored in a DNA sequence and 
		recovered succesfully three images that originally were in JPEG format. 
		This shows that nowadays, it is not only possible to store text and 
		information as we did in CryptoGErM; but DNA can be taken to another level 
		of storage.</P><P>If now it is possible to store images in DNA, in the near future we 
		might be able to store complete movies on DNA</P></SECTION><SECTION><H3 class="centrate">DNA sequence drops to $0.0002 per Megabase</H3><FIGURE class="centrate"><FIGCAPTION class="centrate"> Fourth fictional scenario </FIGCAPTION></FIGURE><P>CryptoGErM is designed primordially for information that requires to 
		be stored for a longer period of time and whose accessibility is not 
		required immediately. However, we are pretty sure that 
		this would change in the upcoming years due to the fact that DNA 
		sequencing is becoming faster and cheaper every year. The inclusion of 
		new and modern techniques, like massively parallel DNA sequencing,<SUP id="ref-8"><A href="#cite-8">[8]</A></SUP> have resulted in the 
		decrease of the price and time needed for the DNA sequencing 
		parallely to the increasing of the number of megabases that can be 
		read. </P><P>Since 2008,<SUP id="ref-9"><A href="#cite-9">[9]</A></SUP> it has 
		been predicted that DNA sequencing techniques would evolve till the 
		point that we will be able to obtain the DNA sequence in just a few 
		hours or minutes, even though many technical challenges have to be 
		overcome first. </P><P>The National Human Genome Research Institute (NHGRI), a division of 
		the National Institutes of Health (NIH), established to take part on 
		behalf of the NIH within the International Human Genome Project (HGP), 
		has been tracked the costs associated with the DNA sequencing (These 
		tack is done within the Centers founded by the National Institute). This 
		data can be consulted freely on the webpage of the NHGRI, in the 
		section “DNA Sequencing Costs: Data.<SUP id="ref-10"><A href="#cite-10">[10]</A></SUP> The following graph contains the data of 
		the costs of DNA Sequencing up to October 2015. </P><FIGURE class="centrate"><FIGCAPTION class="centrate"> Cost per Megabase of DNA Sequence </FIGCAPTION></FIGURE><P>The graph above shows how the cost for the sequencing of DNA has 
		plummeted since the beginning of the 21<SUP>st</SUP> century; this is mainly given by 
		the introduction of high-throughput techniques. 
		By October 2015, according to the data provided by 
		the National Human Genome Research Institute, <STRONG>the cost for 
		Megabase base of DNA sequencing was $0.0145.</STRONG> By this year 
		2016, the cost of the DNA sequencing is expecting to be further 
		reduced. However, until now, not official data has been released. </P><P>According to Moore’s Law, that states that the number of 
		transistors in a dense integrated circuit and its computational power 
		doubles approximately every two years, the cost per Megabase at 2015 
		was expected to be around $50; however, the computational equipment 
		associated with the DNA sequencing has growing exponentially decreasing 
		even further the price, besides the new techniques that allows a faster 
		and more precise analysis.</P><P>With all this information, our hypothetical scenario seems to be 
		possible in just a couple of years, in which novel upcoming techniques 
		are available on the market. </P><P><STRONG> However, CryptoGErM is not only a system for the storing 
		of information, our system goes beyond that; rather than just storing 
		encrypted information, our system can be used for physical sending. 
		</STRONG></P></SECTION><SECTION><H3>And if the system is in the wrong hands?</H3><P>What would happen if the system would end up in the hands of a criminal? Which impact would it have? </P><P>To give an answer to this question, we propose a computational 
		simulation to model the behaviors of some of the agents that could use 
		the CryptoGErM system, within these agents, a potential criminal was 
		included. Please consult the AI Agent Model in <A href="/Team:Groningen/Model">The Modelling Section</A> 
		for further information and discussion</P></SECTION><SECTION><H3>References</H3><UL><LI id="cite-1"><A href="#ref-1">[1]</A> Bornholt, J. et al. A DNA-Based Archival Storage System. ASPLOS’16. </LI><LI id="cite-2"><A href="#ref-2">[2]</A> Bawden, T. (January 23 2016) Global warming: Data centres to consume three times as much energy in next decadde, experts warm. Retrieved from http://www.independent.co.uk/environment/global-warming-data-centres-to-consume-three-times-as-much-energy-in-next-decade-experts-warn-a6830086.html </LI><LI id="cite-3"><A href="#ref-3">[3]</A> Ancient DNA: Towards a million-year-old genome - Nature 499, 34-35 (04 July 2013) DOI: 10.1038/nature12263</LI><LI id="cite-4"><A href="#ref-4">[4]</A> The half-life of DNA in bone: measuring decay kinetics in 158 dated fossils - Proc Biol Sci. 2012 Dec 7; 279 (1748)  </LI><LI id="cite-5"><A href="#ref-5">[5]</A> Bornholt, et al. A DNA-Based Archival Storage System. ASPLOS ’16. April, 2016. </LI><LI id="cite-6"><A href="#ref-6">[6]</A> Church, G. Gao, Y. Kosuri, S. Next-Generation Digital Information Storage in DNA. Science. 37, 2012, pp. 1628 </LI><LI id="cite-7"><A href="#ref-7">[7]</A> Bornholt, J., Lopez R., Carmean D., Ceze L., Seelig G., Strauss K. A DNA-Based Archival Storage System. ASPLOS ‘16. April 2-6. DOI: 10.1145/2872362.2872397 </LI><LI id="cite-8"><A href="#ref-8">[8]</A> Mardis, E. Next-Generation DNA Sequencing Methods. Annu. Rev. Genomics Hum. Genet. 9, ,2008, , pp. 238 </LI><LI id="cite-9"><A href="#ref-9">[9]</A> Shendure, J. Ji, H. Next-generation DNA sequencing. </LI><LI id="cite-10"><A href="#ref-10">[10]</A> Wetterstrand, KA. DNA Sequencing Costs: Data from the NHGRI Genome Sequencing Program (GSP) Available at: https://www.genome.gov/sequencingcostsdata/. Accessed September 18th 2016 </LI></UL></SECTION></ARTICLE></DIV><DIV id="page-footer" class="page-footer"><SECTION><H6>Team</H6><UL><LI><A href="/Team:Groningen/Team">Members</A></LI><LI><A href="/Team:Groningen/Supervisors">Supervisors</A></LI><LI><A href="/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A href="/Team:Groningen/Achievements">Achievements</A></LI><LI><A href="/Team:Groningen/Contact">Contact</A></LI><LI><A href="/Team:Groningen/Jamboree">Jamboree</A></LI></UL></SECTION><SECTION><H6>Project</H6><UL><LI><A href="/Team:Groningen/Tour">Tour</A></LI><LI><A href="/Team:Groningen/Description">Description</A></LI><LI><A href="/Team:Groningen/Design">Design</A></LI><LI><A href="/Team:Groningen/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Groningen/Summary">Summary</A></LI><LI><A href="/Team:Groningen/Applications">Applications</A></LI><LI><A href="/Team:Groningen/Results">Results</A></LI><LI><A href="/Team:Groningen/FuturePerspectives">Future Perspectives</A></LI></UL></SECTION><SECTION class="double"><H6>Biology</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Experiments">Experiments</A></LI><LI><A href="/Team:Groningen/Safety">Safety</A></LI><LI><A href="/Team:Groningen/Decoy">Decoy experiment</A></LI><LI><A href="/Team:Groningen/PhotoswitchableAntibiotics">Photoswitchable antibiotics</A></LI><LI><A href="/Team:Groningen/BrickCharacter">BioBrick Characterization</A></LI><LI><A href="/Team:Groningen/KeyDeletion">Key deletion</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/Proof">Proof of concept</A></LI><LI><A href="/Team:Groningen/Labjournal">Lab journal</A></LI><LI><A href="/Team:Groningen/PlasmidConstruction">Plasmid construction</A></LI><LI><A href="/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="/Team:Groningen/Labjournal/PrimerList">Primer list</A></LI><LI><A href="/Team:Groningen/Notebook">Notebook</A></LI></UL></DIV></SECTION><SECTION><H6>Modelling</H6><UL><LI><A href="/Team:Groningen/Model">Modelling</A></LI><LI><A href="/Team:Groningen/MutationRates">Mutation Rates</A></LI><LI><A href="/Team:Groningen/RandomMutation">Random Mutation</A></LI><LI><A href="/Team:Groningen/AIModel">AI Agent model</A></LI><LI><A href="/Team:Groningen/DecodingFidelity">Decoding Fidelity</A></LI><LI><A href="/Team:Groningen/SpiroModel">Optimal Strain</A></LI></UL></SECTION><SECTION><H6>Computing</H6><UL><LI><A href="/Team:Groningen/Software">Software</A></LI><LI><A href="/Team:Groningen/Encoding">Encoder</A></LI><LI><A href="/Team:Groningen/Decoding">Decoder</A></LI><LI><A href="/Team:Groningen/FullCoding">Full Coding</A></LI></UL></SECTION><SECTION class="double"><H6>Human Practices</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Groningen/HP/Outreach">Outreach</A></LI><LI><A href="/Team:Groningen/HP/Social">Social</A></LI><LI><A href="/Team:Groningen/HP/Legal">Legal</A></LI><LI><A href="/Team:Groningen/HP/FutureScenarios">Future Scenarios</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/HP/Education">Education</A></LI><LI><A href="/Team:Groningen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="/Team:Groningen/Engagement">Engagement</A></LI><LI><A href="/Team:Groningen/HP/Gold">Gold</A></LI><LI><A href="/Team:Groningen/HP/Silver">Silver</A></LI></UL></DIV></SECTION><SECTION><H6>Support</H6><UL><LI><A href="/Team:Groningen/Sponsors">Sponsors</A></LI><LI><A href="/Team:Groningen/Attributions">Attributions</A></LI></UL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>