<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_DecodingCosts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/DecodingCosts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="content_wrapper"><HEADER id="content-header"><SPAN id="logo-container" class="menu-item"><DIV class="menu-title"><A href="/Team:Groningen">CryptoGE®M</A></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Team</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Team">Members</A><A href="/Team:Groningen/Supervisors">Supervisors</A><A href="/Team:Groningen/Collaborations">Collaborations</A><A href="/Team:Groningen/Achievements">Achievements</A><A href="/Team:Groningen/Contact">Contact</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Project</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Tour"><STRONG>Tour</STRONG></A><A href="/Team:Groningen/Description">Description</A><A href="/Team:Groningen/Design">Design</A><A href="/Team:Groningen/Results">Results</A><A href="/Team:Groningen/Demonstrate">Demonstrate</A><A href="/Team:Groningen/Applications">Applications</A><A href="/Team:Groningen/FuturePerspectives">Future</A><A href="/Team:Groningen/Notebook">Notebook</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Biology</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Experiments">Experiments</A><A href="/Team:Groningen/Parts">Parts</A><A href="/Team:Groningen/Safety">Safety</A><A href="/Team:Groningen/Labjournal">Lab journal</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Computing</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Model">Modelling</A><A href="/Team:Groningen/Software">Software</A><A href="/Team:Groningen/Encoding">Encoding</A><A href="/Team:Groningen/Decoding">Decoding</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Human Practices</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Human_Practices">Human Practices</A><A href="/Team:Groningen/HP/Gold">Gold</A><A href="/Team:Groningen/HP/Silver">Silver</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Support</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Sponsors">Sponsors</A><A href="/Team:Groningen/Attributions">Attributions</A></DIV></DIV></SPAN></HEADER><DIV id="groningen-wrapper"><ARTICLE><SECTION><H2>Decoding – costs and reliability</H2><P>The cost of sequencing DNA has plummeted in the last two 
			decades. For instance, in the early 2000’s it took 13 years and $3 
			billion US dollars to sequence the entire human genome. With 
			current technologies, we have approached the $1000 dollars mark. In 
			fact, since 2007 you can have your whole DNA sequenced by less than 
			that! However, there are still issues with the fidelity and 
			accuracy of the readings obtained that would prevent them from 
			being used for our system (Zhu et. al., 2016).</P><FIGURE class="centrate"><FIGCAPTION>Price of sequencing one million of base pairs. 
				Prices dropped from 2007, when second generation techniques 
				were introduced in the marked. Source: National Human Genome 
				Research Institute</FIGCAPTION></FIGURE><P>Existing laboratory-level DNA sequencing technologies typically 
			allow for a reading error of ~1%. Thanks to optimization and 
			finne-tunning, traditional readings using Sanger biochemistry offer 
			now accuracies of up to 99.999% in a read length of 1,000 bp. With 
			modern, second generation or cyclic sequencing, higher reading 
			lengths have been achieved but with a decrement in accuracy (Wang 
			et. al., 2012).</P><P>Among the bioencryption layers, CryptoGERM prevents unauthorized 
			parties from reading the message by having a high ratio of decoy 
			spores that contain a useless sequence. In addition, if the right 
			growing conditions are not supplied our system prevents germination 
			and replication of spores that do contain the message. So, what 
			ratio of decoy:spores do I need to prevent brute sequencing and 
			message retrieval from a third party?</P><FIGURE class="centrate"><FIGCAPTION><STRONG>A)</STRONG> One of the bioencryption 
				bilayers is having a high proportion of decoy spores vs those 
				that do contain the intended useful sequence. If the right 
				conditions are not met (i.e. adding an X antibiotic to the 
				system) those spores that contain our message will die and be 
				outgrown by the decoy. <STRONG>B)</STRONG> Current sequencing 
				techniques will not be able to distinguish the hidden message 
				from noise.</FIGCAPTION></FIGURE><P>Fox et. al. (2014) reports that there is a 50% chance of 
			accurately distinguishing a true subclonal variant from a 
			sequencing artifact in an excess of 100 wild-type DNA sequences 
			using standard Q30 filter reads (error rate: 10<SUP>-2</SUP>). In agreement 
			with that result, in an experiment carried out to detect genomic 
			variations in marine pests, Pochon (2013) was able to detect one 
			variant out of 150 wild-type sequences.</P><P>Reading DNA has become increasingly more accurate. Schmidt 
			(2012) developed a method called Duplex Sequencing that uses both 
			strands of DNA to obtain a more precise consensus sequence yielding 
			an theoretical error of 3x10<SUP>-10</SUP>. That means that we could transmit 
			a message with a length in the order of Gigabytes without expecting 
			any lost! On the other hand, that also means that they allow a more 
			precise measurement in decoy-spores mixtures, and a 1:150 
			spore:decoy ratio might be insufficient in the future. In fact, 
			using Duplex Sequencing they were able to identify one mutant 
			sequence per 10,000 wild-type molecules.</P></SECTION><SECTION><H6>References</H6><UL><LI>Fox EJ, Reid-Bayliss KS, Emond MJ, Loeb LA (2014) Accuracy 
				of Next Generation Sequencing Platforms. Next Generat Sequenc 
				&amp; Applic 1: 106. doi:10.4172/jngsa.1000106 </LI><LI>Pochon (2013). Evaluating detection limits of 
				next-generation sequencing for the surveillance and monitoring 
				of international marine pests. PLos One 8(9):e73935</LI><LI>Schmitt MW, Kennedy SR, Salk JJ, Fox EJ, Hiatt JB, et al. 
				(2012) Detection of ultra-rare mutations by next-generation 
				sequencing. Proc Natl Acad Sci U S A 109: 14508-14513.</LI><LI>Wang, X., Blades, N., Ding, J., Sultana, R. and Parmigiani, 
				G. (2012). Estimation of sequencing error rates in short reads. 
				BMC Bioinformatics, 13:185</LI><LI>Zhu, X., Wang, J., Peng, B. and Shete, S. (2016). Empirical 
				estimation of sequencing error rates using smoothing splines. 
				BMC Bioinformatics, 17:177</LI></UL></SECTION></ARTICLE></DIV><DIV id="page-footer" class="page-footer"><SECTION><H6>Team</H6><UL><LI><A href="/Team:Groningen/Team">Members</A></LI><LI><A href="/Team:Groningen/Supervisors">Supervisors</A></LI><LI><A href="/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A href="/Team:Groningen/Achievements">Achievements</A></LI><LI><A href="/Team:Groningen/Contact">Contact</A></LI><LI><A href="/Team:Groningen/Jamboree">Jamboree</A></LI></UL></SECTION><SECTION><H6>Project</H6><UL><LI><A href="/Team:Groningen/Tour">Tour</A></LI><LI><A href="/Team:Groningen/Description">Description</A></LI><LI><A href="/Team:Groningen/Design">Design</A></LI><LI><A href="/Team:Groningen/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Groningen/Summary">Summary</A></LI><LI><A href="/Team:Groningen/Applications">Applications</A></LI><LI><A href="/Team:Groningen/Results">Results</A></LI><LI><A href="/Team:Groningen/FuturePerspectives">Future Perspectives</A></LI></UL></SECTION><SECTION class="double"><H6>Biology</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Experiments">Experiments</A></LI><LI><A href="/Team:Groningen/Safety">Safety</A></LI><LI><A href="/Team:Groningen/Decoy">Decoy experiment</A></LI><LI><A href="/Team:Groningen/PhotoswitchableAntibiotics">Photoswitchable antibiotics</A></LI><LI><A href="/Team:Groningen/BrickCharacter">BioBrick Characterization</A></LI><LI><A href="/Team:Groningen/KeyDeletion">Key deletion</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/Proof">Proof of concept</A></LI><LI><A href="/Team:Groningen/Labjournal">Lab journal</A></LI><LI><A href="/Team:Groningen/PlasmidConstruction">Plasmid construction</A></LI><LI><A href="/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="/Team:Groningen/Labjournal/PrimerList">Primer list</A></LI><LI><A href="/Team:Groningen/Notebook">Notebook</A></LI></UL></DIV></SECTION><SECTION><H6>Modelling</H6><UL><LI><A href="/Team:Groningen/Model">Modelling</A></LI><LI><A href="/Team:Groningen/MutationRates">Mutation Rates</A></LI><LI><A href="/Team:Groningen/RandomMutation">Random Mutation</A></LI><LI><A href="/Team:Groningen/AIModel">AI Agent model</A></LI><LI><A href="/Team:Groningen/DecodingFidelity">Decoding Fidelity</A></LI><LI><A href="/Team:Groningen/SpiroModel">Optimal Strain</A></LI></UL></SECTION><SECTION><H6>Computing</H6><UL><LI><A href="/Team:Groningen/Software">Software</A></LI><LI><A href="/Team:Groningen/Encoding">Encoder</A></LI><LI><A href="/Team:Groningen/Decoding">Decoder</A></LI><LI><A href="/Team:Groningen/FullCoding">Full Coding</A></LI></UL></SECTION><SECTION class="double"><H6>Human Practices</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Groningen/HP/Outreach">Outreach</A></LI><LI><A href="/Team:Groningen/HP/Social">Social</A></LI><LI><A href="/Team:Groningen/HP/Legal">Legal</A></LI><LI><A href="/Team:Groningen/HP/FutureScenarios">Future Scenarios</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/HP/Education">Education</A></LI><LI><A href="/Team:Groningen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="/Team:Groningen/Engagement">Engagement</A></LI><LI><A href="/Team:Groningen/HP/Gold">Gold</A></LI><LI><A href="/Team:Groningen/HP/Silver">Silver</A></LI></UL></DIV></SECTION><SECTION><H6>Support</H6><UL><LI><A href="/Team:Groningen/Sponsors">Sponsors</A></LI><LI><A href="/Team:Groningen/Attributions">Attributions</A></LI></UL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>