<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Labjournal skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Labjournal</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="content_wrapper"><HEADER id="content-header"><SPAN id="logo-container" class="menu-item"><DIV class="menu-title"><A href="/Team:Groningen">CryptoGE®M</A></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Team</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Team">Members</A><A href="/Team:Groningen/Supervisors">Supervisors</A><A href="/Team:Groningen/Collaborations">Collaborations</A><A href="/Team:Groningen/Achievements">Achievements</A><A href="/Team:Groningen/Contact">Contact</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Project</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Tour"><STRONG>Tour</STRONG></A><A href="/Team:Groningen/Description">Description</A><A href="/Team:Groningen/Design">Design</A><A href="/Team:Groningen/Results">Results</A><A href="/Team:Groningen/Demonstrate">Demonstrate</A><A href="/Team:Groningen/Applications">Applications</A><A href="/Team:Groningen/FuturePerspectives">Future</A><A href="/Team:Groningen/Notebook">Notebook</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Biology</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Experiments">Experiments</A><A href="/Team:Groningen/Parts">Parts</A><A href="/Team:Groningen/Safety">Safety</A><A href="/Team:Groningen/Labjournal">Lab journal</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Computing</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Model">Modelling</A><A href="/Team:Groningen/Software">Software</A><A href="/Team:Groningen/Encoding">Encoding</A><A href="/Team:Groningen/Decoding">Decoding</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Human Practices</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Human_Practices">Human Practices</A><A href="/Team:Groningen/HP/Gold">Gold</A><A href="/Team:Groningen/HP/Silver">Silver</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Support</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Sponsors">Sponsors</A><A href="/Team:Groningen/Attributions">Attributions</A></DIV></DIV></SPAN></HEADER><DIV id="groningen-wrapper"><ARTICLE id="labjournal" class="collapse-h2 collapse-conditional"><SECTION><H2>Labjournal</H2><P>Here you can find how we constructed our BioBricks and other plasmids which were used during our project, next you can find protocols which we followed during our experiments and finally you can see our primer list with every primer used. </P></SECTION><ARTICLE id="labjournal-plasmids" class="collapse-h2 collapse-conditional"><SECTION><H2>Plasmid construction</H2><P>The following plasmids were constructed mainly during summer/autumn 2016.</P></SECTION><SECTION id="Key-in-pDR111" class="collapse-h3"><H3>Key sequence in pDR111</H3><P>The key sequence was ordered as a gBlock from IDT to construct the key <EM>B. 
	subtilis</EM> strain. In a first approach this was done with the pDR111 <EM>B. 
	subtilis</EM> integration plasmid, which can also be amplified by <EM>E. coli</EM>. 
	Therefore the first step was to clone the key sequence into the pDR111 in <EM>E. coli</EM>. pDR111 is an integration plasmid which can be integrated into the <EM>B. subtilis</EM> genome. 
	It replaces <EM>amyE</EM> gene by double cross-over, which is necessary for production of alpha-amylase 
	(see <A href="http://subtiwiki.uni-goettingen.de/bank/index.php?gene=amyE&amp;action=Go">Subtiwiki.uni-goettingen.de</A>)       with desired insert which is located between amyE front flanking region and amyE back flanking region. See the plasmid map below. We used this plasmid for integration of our key sequence into the <EM>B. subtilis</EM> 168 sub+. The other approach made use of the BioBrick integration plasmid BBa_K823023 <A href="https://2016.igem.org/Team:Groningen/Experiments#Key-in-BBa_K823023">(Key sequence in BBa_K823023)</A>. </P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P>31/08/16: Amplification of the key sequence from the gBlock from IDT <A href="https://2016.igem.org/Team:Groningen/Experiments#gBlocks">(see gBlocks protocol)</A>
		with the primers F key only amplify and R key only amplify <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">(see primer list)</A> was done. The correct size of the     key PCR product of 144 bp was checked with DNA electrophoresis. </P><H5>PCR mixture:</H5><P>50 μl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gels see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 1. DNA electrophoresis of the key PCR product 
			sample 1-3. The correct size is 144 bp.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>PCR of the key sequence from the IDT gBlock was successful. We could see the correct band size which was 144 bp.</P><H5>Procedure after gel validation:</H5><P>PCR product was subsequently cleaned with the kit <A href="/Team:Groningen/Protocols#Clean-up-Jena">(PCR Purification Kit – Jena Bioscience)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>31/08/16: On this day restriction digestion of the PCR product of the key 
		sequence and pDR111 integration plasmid was performed. The key sequence 
		as an insert was cut with SalI and HindIII restriction enzymes. The 
		integration plasmid pDR111 as a vector was cut with exactly same 
		restriction enzymes.</P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following see 
		following <A href="/Team:Groningen/Protocols#rd-fast">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>The digestion mixture of backbone pDR111 was loaded on a gel to 
		extract the backbone. For detailed information on how to prepare and run 
		agarose gels see following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 2. DNA electrophoresis of digested pDR111 with SalI, HindIII.
			The correct size is 7,834 bp.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The restriction digestion was successful. We could see a band of 7,834 bp.</P><H5>Procedure after gel validation:</H5><P>Digested pDR111 was cut out from the gel and DNA was extracted by Gel 
		extraction kit <A href="/Team:Groningen/Protocols#Gel-extract-Jena">(Agarose Gel Extraction Kit – Jena Bioscience)</A>. The PCR product of the key was not 
		checked on the gel after the digestion but immediately cleaned up with 
		DNA Clean-up <A href="/Team:Groningen/Protocols#Clean-up-Nucleospin">(NucleoSpin® Gel and PCR Clean-up)</A>. </P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>31/08/16: The cut key was ligated to the SalI, HindIII cut pDR111.</P><H5>Ligation mixture:</H5><P>20 μl ligation assay was performed according to the following <A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>31/08/16: The ligation mix was heat shock transformed to competent Top10 <EM>E. coli</EM> cells following the <A href="/Team:Groningen/Protocols#transform-e-coli-standard">protocol</A>.
		Cells were plated on 100 μg/ml ampicillin LB agar to select the correct 
		construct. The next day colonies were picked to perform colony PCR to 
		find the correct construct with the primers F-amyE and R-amyE. Find 
		primers <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">here</A>.</P><FIGURE><FIGCAPTION>Figure 3. Transformation of the key sequence into the pDR111.</FIGCAPTION></FIGURE><H5>PCR mixture:</H5><P>25 μl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#Colony-PCR">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gels see 
		the following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 4. DNA electrophoresis of colony PCR for the key sequence in 
			pDR111. Sample 5,6,7 and 9 show the correct size of 
			504 bp.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>Transformation of pDR111+key into <EM>E. coli</EM> Top10 appeared to be 
		successful. The samples 5, 6, 7 and 9 showed the right size of band in 
		the colony PCR. These samples were used to obtain the plasmid from an overnight culture.</P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>02/09/16: Grown cultures of <EM>E. coli</EM> Top10 with pDR111+key were used to 
		obtain a <A href="/Team:Groningen/Protocols#GS">glycerol stocks</A> and plasmid isolation was performed 
		<A href="/Team:Groningen/Protocols#Mini-prep-QIA">(QIAprep® Spin Miniprep Kit)</A>. Firstly, concentration of the plasmids obtained was 
		measured on Nanodrop. Secondly, plasmids were sent for sequencing and 
		then stored at -20°C.</P><H5>Sequencing:</H5><P>Plasmids pDR111+key from colonies 5, 6, 7 and 9 were sent for 
		<A href="/Team:Groningen/Protocols#Sequencing">sequencing</A> with the sequencing primer F message 
		sequencing <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">(see primer list)</A>. </P><H5>Conclusion:</H5><P>Sequencing results showed that cloning of message sequence into the 
		integration plasmid pDR111 was successful. Sample 5, 6 and 9 show a 
		100% homology, so the key was successfully cloned into the pDR111 
		plasmid. See sequencing results in Figure 5. </P><FIGURE><FIGCAPTION>Figure 5. Sequencing result of the key in pDR111 for 
			sample 5, 6 and 9.</FIGCAPTION></FIGURE></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><H5>Experiments:</H5><P>See <A href="https://2016.igem.org/Team:Groningen/Experiments#proof-of-concept">Proof of concept</A>
experiment.</P></DIV></SECTION><SECTION id="Key-in-pSB1C3" class="collapse-h3"><H3>Key sequence in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K1930000">BBa_K1930000</A>)</H3><P>The iGEM team Groningen 2016 worked on bioencryption in order to safely store data in DNA. In our project we were mainly working with two sequences of DNA which were the encrypted message sequence and the encryption key sequence. The key sequence was created by our software. It is therefore artificial DNA. We ordered the encryption key sequence as gBlock from IDT (Integrated DNA technologies). To submit our key sequence as BioBrick (<A href="http://parts.igem.org/Part:BBa_K1930000">BBa_K1930000</A>) we cloned it in the pSB1C3 standard iGEM backbone. </P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P>27/09/16: The key was PCR amplified (see following 
		<A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>) from the pDR111+key plasmid with the 
		primers key prefix and key suffix <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">(see primer list)</A>. The correct size 187 bp of the 
		product was checked by DNA electrophoresis. The PCR product was stored 
		at 4°C.</P><H5>PCR mixture:</H5><P>50 μl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gels see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 1. DNA electrophoresis of amplified key sequence (187 bp in size).</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The key was successfully amplified with prefix and suffix from the 
		pDR111+key plasmid.</P><H5>Procedure after gel validation:</H5><P>PCR product was subsequently cleaned with <A href="/Team:Groningen/Protocols#Clean-up-Jena">(PCR Purification Kit – Jena Bioscience)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>28/09/16: The PCR product of the key sequence was cut with EcoRI and PstI. The backbone pSB1C3 <A href="http://parts.igem.org/Part:BBa_J04450">(BBa_J04450)</A> was digested with the same enzymes. This construct is carrying RFP reporter therefore it was used for easier screening after transformation. You could see self-ligations as red 
		colonies and the correct ones as white ones. </P><H5>RD mixture:</H5><P>30 μl RD assay was performed according to the following see 
		following <A href="/Team:Groningen/Protocols#rd-jena">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>The digestion mixture of backbone pSB1C3 - BBa_J04450 was loaded on a 
		gel to extract the digested backbone.</P><P>For detailed information on how to prepare and run 
		agarose gels see following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 2. DNA electrophoresis of EcoRI and PstI digested 
			pSB1C3 - BBa_J04450. The RFP insert is 1069 bp and the pSB1C3 backbone 
			is 2019 bp.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The digestion was successful. We could see bands for both expected 
		fragments on the gel, namely RFP insert is 1069bp and the 
		pSB1C3 backbone is 2019 bp.</P><H5>Procedure after gel validation:</H5><P>The upper band of 2019 bp was cut out from the gel 
		and DNA was extracted by <A href="/Team:Groningen/Protocols#Gel-extract-Jena">(Agarose Gel Extraction Kit – Jena Bioscience)</A>. The PCR product of the key sequence was not checked on the gel after the digestion but immediately 
		cleaned up with <A href="/Team:Groningen/Protocols#Clean-up-Nucleospin">(NucleoSpin® Gel and PCR Clean-up)</A>. The concentration of the digested and cleaned key sequence was measured with the Nanodrop, 10,3 ng/μl were obtained.</P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>07/10/16: The EcoRI, PstI cut pSB1C3 backbone was ligated with the 
		EcoRI, PstI cut key.</P><H5>Ligation mixture:</H5><P>20 μl ligation assay was performed according to the following <A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>08/10/16: The ligation mix was heat shock transformed to competent Top10 <EM>E. coli</EM> cells following the <A href="/Team:Groningen/Protocols#transform-e-coli standard">protocol</A>. 
		Cells were plated on 50 μg/ml chloramphenicol LB agar to select for the 
		correct constructs. The next day colonies were picked to perform colony 
		PCR to find the correct constructs with the primers key only prefix and 
		key only suffix. Find 
		primers<A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">here</A>.</P><FIGURE><FIGCAPTION>Figure 3. <EM>E. coli</EM> Top10 cells transformed with key in pSB1C3. 
			</FIGCAPTION></FIGURE><H5>PCR mixture:</H5><P>25 μl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#Colony-PCR">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s     </TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s     </TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>1:30 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gels see 
		the following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 4. Result of the colony PCR. Samples from colony 
			1-9 are showing the correct size of the key sequence which is 187 bp. C - is the water control. </FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The transformation of the key sequence in pSB1C3 to <EM>E. coli</EM> Top10 was 
		successful. We have obtained correct insert size when colony PCR was done. And that was 187 bp. </P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>09/10/16: Grown cultures of <EM>E. coli</EM> Top10 with the construct key in 
		pSB1C3 were used to obtain <A href="/Team:Groningen/Protocols#GS">glycerol stocks</A> and plasmid isolation 
		was performed <A href="/Team:Groningen/Protocols#Mini-prep-Jena">(Fast-n-Easy Plasmid Mini-Prep Kit - Jena Bioscience)</A>. Firstly, 
		concentration of the plasmids obtained was measured on Nanodrop. 
		Secondly, plasmids were sent for sequencing and then stored at 
		-20°C.</P><H5>Sequencing:</H5><P>The BioBrick key in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K1930000">BBa_K1930000</A>) from colonies 1 and 3 was sent for sequencing with the primers VF2 and VR, <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">(see primer list)</A>.</P><FIGURE><FIGCAPTION>Figure 5. Sequencing result for the key in pSB1C3 (BBa_K1930000) from 
			colony 1 with VF2. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 6. Sequencing result for the key in pSB1C3 (BBa_K1930000) from 
			colony 1 with VR.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The sequencing result proofed the successful integration of the key 
		into the pSB1C3. First BioBrick was made!</P></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><H5>Experiments:</H5><P>See integration of the key sequence in <EM>B. subtilis</EM> via the BioBrick <A href="http://parts.igem.org/Part:BBa_K823023">BBa_K823023</A> integration plasmid in <A href="https://2016.igem.org/Team:Groningen/Experiments#proof-of-concept">Proof of concept</A>
experiment. 
	</P></DIV></SECTION><SECTION id="Key-in-BBa_K823023" class="collapse"><H3>Key sequence in BBa_K823023</H3><P><A href="http://parts.igem.org/Part:BBa_K823023">BBa_K823023</A> is an available BioBrick from iGEM Munich 2012. It is an 
	integration plasmid for <EM>Bacillus subtilis</EM>, which can be used for 
	cloning in <EM>E. coli</EM> as well. An RFP is inserted in BBa_K823023 for more 
	efficient screening after transformation. It was chosen to 
	construct the <EM>Bacillus subtilis</EM> key strain. Construction was 
	performed as described in the following.</P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P>27/09/16: The key sequence was amplified <A href="https://2016.igem.org/Team:Groningen/Protocols#PCR-Q5">(see PCR protocol)</A> from the pDR111+key plasmid with the primers 
		key only + prefix and key only + suffix (primer sequences can be found <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">here</A>). The correct size of 187 bp of the product was checked by DNA electrophoresis. The PCR product was stored at 4°C.</P><H5>PCR mixture:</H5><P>50 µl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gels see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 1. DNA electrophoresis of amplified key sequence (187 bp in size).</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The key sequence with prefix and suffix was successfully amplified from the gBlock. Band of 187 bp could be seen.</P><H5>Procedure after gel validation:</H5><P>PCR product was subsequently cleaned with <A href="/Team:Groningen/Protocols#Clean-up-Jena">(PCR Purification Kit – Jena Bioscience)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>04/10/16: The key sequence PCR product was digested with EcoRI and PstI as well as the 
		BBa_K823023 plasmid. The digestion of the BBa_K823023 backbone was checked with DNA 
		electrophoresis. The digestion of the key sequence PCR product was immediately cleaned 
		with PCR Purification Kit – Jena Bioscience 
		<A href="https://2016.igem.org/Team:Groningen/Protocols#Clean-up-Jena">see protocol</A>.</P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following <A href="https://2016.igem.org/Team:Groningen/Protocols#rd-jena">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run 
		agarose gels see following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 2. BBa_K823023 cut with EcoRI and PstI. Backbone 
			~6,000 bp, RFP insert ~1,000 bp.</FIGCAPTION></FIGURE><H5> Conclusion: </H5><P>The digestion of BBa_K823023 showed the correct bands on 
		the gel for the 6,000 bp backbone and the 1,000 bp RFP insert. 
		Therefore the cloning procedure could proceed.</P><H5>Procedure after gel validation:</H5><P>Digested sample of the backbone ~6,000 bp were cut out from the 
		gel and DNA was extracted by Gel extraction kit (Nucleospin)
		<A href="https://2016.igem.org/Team:Groningen/Protocols#Gel-extract-Nucleospin">see protocol</A>. </P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>04/10/16: The EcoRI and PstI cut BBa_K823023 integration backbone was 
		ligated with the EcoRI, PstI cut key sequence.</P><H5>Ligation mixture:</H5><P>20 µl ligation assay was performed according to the following <A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>04/10/16: The ligation mix was heat shock transformed to competent 
		Top10 <EM>E. coli</EM> cells following the <A href="https://2016.igem.org/Team:Groningen/Protocols#transform-e-coli-standard">protocol</A>. Cells were plated on 100 μg/ml ampicillin LB agar to select for the correct constructs. The next day colonies were selected to perform colony PCR in order to find the correct constructs.</P><FIGURE><FIGCAPTION>Figure 3. Colonies of <EM>E.coli</EM> Top10 after 
			transformation with BBa_K823023 with the key sequence.</FIGCAPTION></FIGURE><H5>PCR mixture:</H5><P>25 µl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#Colony-PCR">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>1:30 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold</TD></TR></TBODY></TABLE><H5>DNA electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel <A href="https://2016.igem.org/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><H5>Conclusion:</H5><P>The transformation of the key sequence in BBa_K823023 to <EM>E. coli</EM> Top10 was 
		successful. We have obtained correct insert size when colony PCR was done. And that was 187 bp. </P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>05/10/16 Grown cultures of <EM>E. coli</EM> Top10 with the key sequence in BBa_K823023 were used for making the <A href="https://2016.igem.org/Team:Groningen/Protocols#GS&gt;glycerol stocks&lt;/a&gt; and plasmid isolation was performed&lt;a href=" https:="">(see Mini prep protocol)</A>. Firstly, concentration of the plasmids obtained was measured on Nanodrop. Secondly, plasmids were sent for sequencing and then stored at -20°C.</P><FIGURE><FIGCAPTION>Figure 4. Selecting colonies for glycerol stocks.</FIGCAPTION></FIGURE></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><H5>Experiments:</H5><P>See <A href="https://2016.igem.org/Team:Groningen/Experiments#proof-of-concept">Proof of concept</A>
experiment.</P></DIV></SECTION><SECTION id="Message-in-pDR111" class="collapse"><H3>Message sequence in pDR111</H3><P>The message sequence was ordered as a gBlock from IDT to construct the message <EM>B. 
	subtilis</EM> strain. In a first approach this was done with the pDR111 <EM>B. 
	subtilis</EM> integration plasmid, which can also be amplified by <EM>E. coli</EM>. 
	Therefore the first step was to clone the message sequence into the pDR111 in <EM>E. coli</EM>. pDR111 is an integration plasmid which can be integrated into the <EM>B. subtilis</EM> genome. 
	By double cross-over it replaces <EM>amyE</EM> gene, which is necessary for production of alpha-amylase 
	(see <A href="http://subtiwiki.uni-goettingen.de/bank/index.php?gene=amyE&amp;action=Go">Subtiwiki.uni-goettingen.de</A>)       with desired insert which is located between amyE front flanking region and amyE back flanking region. See the plasmid map below. We used this plasmid for integration of our message sequence into the <EM>B. subtilis</EM> 168 sub+. The other approach made use of the BioBrick integration plasmid BBa_K823023 <A href="https://2016.igem.org/Team:Groningen/Experiments#Message-in-BBa_K823023">(Message sequence in BBa_K823023)</A>. </P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P> 25/07/16: The message sequence was amplified from gBlock ordered from IDT (Integrated DNA technologies).Primers used for the amplification were F-message sequence and R-message sequence (primer sequences can be found <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">here</A>). </P><H5>PCR mixture:</H5><P>50 μl PCR assay was performed according to the following  
	<A href="https://2016.igem.org/Team:Groningen/Protocols#PCR-Q5">protocol.</A></P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(12X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(12X)</TD></TR><TR><TD>72ºC</TD><TD>1:30 min</TD><TD>(12X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold</TD></TR></TBODY></TABLE><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel <A href="https://2016.igem.org/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol.</A></P><FIGURE><FIGCAPTION>Figure 2. DNA electrophoresis with amplified message sequence 572 bp in size.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P> PCR of the message sequence from the IDT gBlock was successful. It was verified by DNA electrophoresis. 
	Correct size of a band could be seen and that was 572 bp.</P><H5>Procedure after gel validation:</H5><P>PCR product was subsequently cleaned with PCR Purification Kit – Jena Bioscience 
			<A href="https://2016.igem.org/Team:Groningen/Protocols#Clean-up-Jena">(see protocol).</A></P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>26/07/16: On this day restriction digestion of PCR product of the message sequence and pDR111 integration plasmid was done. Message sequence as an insert was cut with SalI and HindIII restriction enzymes. Integration plasmid pDR111 as a vector was cut with exactly same restriction enzymes. </P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following 
	<A href="https://2016.igem.org/Team:Groningen/Protocols#rd-fast">protocol.</A></P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel <A href="https://2016.igem.org/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol.</A></P><FIGURE><FIGCAPTION>Figure 3. DNA electrophoresis with samples from restriction digestion.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>RD of the PCR product of message sequence and integration plasmid pDR111 was successful. It was verified by DNA electrophoresis. Correct size of bands could be seen and that was 7,834 bp for pDR111 and 572 bp for message sequence.</P><H5>Procedure after gel validation:</H5><P>Digested samples were cut out from the gel and DNA was extracted by Gel extraction kit (Nucleospin)
	<A href="https://2016.igem.org/Team:Groningen/Protocols#Gel-extract-Nucleospin">(see protocol).</A></P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>26/07/16: Restriction digestion was followed by overnight ligation. Vector (integration plasmid pDR111) and insert (message sequence) was ligated in 1:5 molar ratio.</P><H5>Ligation mixture:</H5><P>20 μl ligation assay was performed according to the following 
	<A href="https://2016.igem.org/Team:Groningen/Protocols#ligation">protocol.</A></P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>27/07/16: This day transformation into <EM>E. coli</EM> Top10 and MC1061 cells was done. Correct clones were selected on 100 μg/ml ampicillin plates. Transformation protocol used can be found <A href="https://2016.igem.org/Team:Groningen/Protocols#transform-e-coli-standard">here</A>.</P><P> 28/07/16: Next day 12 colonies were obtained on the plates of <EM>E. coli</EM> MC1061 strain. To verify correct transformants colony PCR was performed <A href="https://2016.igem.org/Team:Groningen/Protocols#Colony-PCR">(see Colony PCR protocol).</A> Primers F-message sequence and R-message sequence were used for this colony PCR. Sequences of these primers can be found <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">here</A>.</P><H5>PCR mixture:</H5><P>25 μl PCR assay was performed according to the following 
	<A href="https://2016.igem.org/Team:Groningen/Protocols#PCR-Q5">protocol.</A></P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>1:30 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold</TD></TR></TBODY></TABLE><H5>DNA electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel see <A href="https://2016.igem.org/Team:Groningen/Protocols#DNA-electrophoresis">following protocol.</A></P><FIGURE><FIGCAPTION>Figure 4. DNA electrophoresis with samples from colony PCR.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 5. Transformation of pDR111+message into <EM>E. coli</EM> MC1061.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P> Transformation of pDR111+message into <EM>E. coli</EM> MC1061 appeared to be successful. Transformation efficiency was low, only 12 colonies were obtained, but 4 colonies seemed to be correct clones. And those were colonies 1, 8, 10 and 11 (see Figure 5). Correct sizes of the bands were obtained: 572 bp. Those colonies were grown overnight (see <A href="https://2016.igem.org/Team:Groningen/Protocols#cell-cultures-e-coli">cell culture protocol</A>) from pre-glycerol stocks made for colony PCR (see <A href="https://2016.igem.org/Team:Groningen/Protocols#Colony-PCR">colony PCR protocol</A>).
		
		 </P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>29/07/16: Grown cultures of <EM>E. coli</EM> MC1061 with pDR111+message were taken out from the incubator after overnight incubation. <A href="https://2016.igem.org/Team:Groningen/Protocols#GS">Glycerol stocks</A>) were made and plasmid isolation was performed (see <A href="https://2016.igem.org/Team:Groningen/Protocols#Mini-prep-QIA">Mini prep protocol</A>). 
Firstly, concentration of the plasmids was measured on Nanodrop. Secondly, plasmids were sent for sequencing and then stored at -20°C.</P><H5> Sequencing: </H5><P> Plasmids pDR111+message from colonies 1, 8, 10 and 11 were sent for <A href="https://2016.igem.org/Team:Groningen/Protocols#Sequencing">sequencing</A>)
	with the sequencing primer Gb_insert_F2 (<A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">(see primer list)</A>). </P><H5> Conclusion: </H5><P> Sequencing results showed that cloning of message sequence into integration plasmid pDR111 was successful. However just sequencing result 249 (Figure 9) has 100 % homology when compared to the reference (message sequence). Others have some bases missing or bases are not matching, it could be due to faulty sequencing. See sequencing results below.</P><FIGURE><FIGCAPTION>Figure 6. Sequencing result 246 of the message in pDR111.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 7. Sequencing result 247 of the message in pDR111.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 8. Sequencing result 248 of the message in pDR111.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 9. Sequencing result 249 of the message in pDR111.</FIGCAPTION></FIGURE></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><H5>Experiments:</H5><P>See <A href="https://2016.igem.org/Team:Groningen/Experiments#proof-of-concept">Proof of concept</A>
experiment.</P></DIV></SECTION><SECTION id="Message-in-pSB1C3" class="collapse-h3"><H3>Message sequence in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K1930001">BBa_K1930001</A>)</H3><P>The iGEM team Groningen 2016 worked on bioencryption in order to safely store data in DNA. In our project we were mainly working with two sequences of DNA which were the encrypted message sequence and the encryption key sequence. The message sequence was created by our software. It is therefore artificial DNA. We ordered the encryption message sequence as gBlock from IDT (Integrated DNA technologies). To submit our message sequence as BioBrick (<A href="http://parts.igem.org/Part:BBa_K1930001">BBa_K1930001</A>) we cloned it in the pSB1C3 standard iGEM backbone. </P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P>06/10/16: The message sequence was amplified from pDR111+message plasmid. 
		Primers used for the amplification were message prefix and message 
		suffix (primer sequences can be found <A href="https://2016.igem.org/Team:Groningen/Labjournal/PrimerList">here</A>). 
		</P><H5>PCR mixture:</H5><P>50 μl PCR assay was performed according to the following <A href="/Team:Groningen/Protocols#PCR-Taq">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold</TD></TR></TBODY></TABLE><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel 
		<A href="/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><FIGURE><FIGCAPTION>Figure 1. DNA electrophoresis with amplified message 
			sequence (M1 and M2) 599 bp in size. C - is water control.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>Amplification of the message sequence and addition of the prefix and 
		suffix to this sequence was successful. It was verified by DNA 
		electrophoresis. Correct size of a band could be seen and 
		that was 599 bp.</P><H5>Procedure after gel validation:</H5><P>PCR product was subsequently cleaned with PCR Purification Kit – <A href="/Team:Groningen/Protocols#Clean-up-Jena">(Jena Bioscience)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>07/10/16: On this day restriction digestion of PCR product of the 
		message sequence and pSB1C3 (BBa_J04450) was done. Message 
		sequence as an insert was cut with EcoRI and PstI restriction enzymes. 
		The backbone pSB1C3 <A href="http://parts.igem.org/Part:BBa_J04450">(BBa_J04450)</A> was digested with the same enzymes. This construct is carrying RFP reporter therefore it was used for easier screening after transformation. You could see self-ligations as red 
		colonies and the correct ones as white ones. </P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following <A href="/Team:Groningen/Protocols#rd-neb">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel 
		<A href="/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><FIGURE><FIGCAPTION>Figure 2. DNA electrophoresis with cut backbone pSB1C3 
			(BBa_J04450).</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>RD of the PCR product of the message sequence and pSB1C3 (BBa_J04450) 
		was successful. RD of pSB1C3 (BBa_J04450) was verified by DNA 
		electrophoresis. Correct size of band could be seen and 
		that was 2019 bp.</P><H5>Procedure after gel validation:</H5><P>The upper band was cut out from the gel and DNA was extracted by Gel 
		extraction kit <A href="/Team:Groningen/Protocols#Gel-extract-Nucleospin">(NucleoSpin® Gel and PCR Clean-up)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>07/10/16: Restriction digestion was followed by overnight 
		ligation. Vector (pSB1C3) and insert (message sequence) was ligated in 
		4:6 molar ratio.</P><H5>Ligation mixture:</H5><P>20 μl ligation assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>08/10/16: On this day transformation into <EM>E. coli</EM> Top10 cells was done. 
		Selection was made on 50 μg/ml chloramphenicol LB agar plates. 
		Transformation protocol used can be found <A href="/Team:Groningen/Protocols#transform-e-coli-standard">here</A>.</P><P>09/10/16: Next day we could assume that our transformation was 
		successful -&gt; colonies were obtained on the plates of <EM>E. coli</EM> Top10 
		strain. To verify correct transformants colony PCR was 
		performed (see colony PCR <A href="/Team:Groningen/Protocols#Colony-PCR">protocol</A>). Primers message 
		prefix and message suffix were used for this colony PCR. Sequences of 
		these primers can be found <A href="https://2016.igem.org/Team:Groningen/Labjournal/PrimerList">here</A>.</P><H5>PCR mixture:</H5><P>50 μl PCR assay was performed according to the following <A href="/Team:Groningen/Protocols#PCR-Taq">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold</TD></TR></TBODY></TABLE><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel 
		<A href="/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><FIGURE><FIGCAPTION>Figure 3. DNA electrophoresis with samples from colony 
			PCR.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 4. Transformation of pSB1C3+message into <EM>E. coli</EM>
			Top10.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>Transformation of pSB1C3+message into <EM>E. coli</EM> Top10 appeared to be 
		successful. Correct sizes of the bands from colony PCR were obtained: 599 bp. Correct clones were grown overnight (see cell culture <A href="/Team:Groningen/Protocols#cell-cultures-e-coli">protocol</A>) from pre-glycerol stocks 
		made for colony PCR (see colony PCR <A href="/Team:Groningen/Protocols#Colony-PCR">protocol</A>).</P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>10/10/16: Grown cultures of <EM>E. coli</EM> Top10 with pSB1C3+message were taken 
		out from the incubator after overnight incubation. <A href="/Team:Groningen/Protocols#GS">Glycerol stocks</A> were made and plasmid isolation was performed (see 
		<A href="/Team:Groningen/Protocols#Mini-prep-Jena">(Fast-n-Easy Plasmid Mini-Prep Kit - Jena Bioscience)</A>). Firstly, concentration of the plasmids 
		obtained was measured on Nanodrop. Secondly, plasmids were sent for 
		sequencing and then stored at -20°C.
		
		</P><H5>Sequencing:</H5><P>Plasmids pSB1C3+message from colonies 1, 2, 3, 4 and 5 were sent for <A href="/Team:Groningen/Protocols#Sequencing">sequencing</A> with the sequencing 
		primers VF2 and VR (primer sequence can be found <A href="https://2016.igem.org/Team:Groningen/Labjournal/PrimerList">here</A>).</P><H5>Conclusion:</H5><P>Sequencing results showed that cloning of message sequence into 
		pSB1C3 was successful. See sequencing results below.</P><FIGURE><FIGCAPTION>Figure 5. Sequencing result of the message in 
			pSB1C3 with VF2 primer.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 6. Sequencing result of the message in 
			pSB1C3 with VF2 and VR primers.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 7. Sequencing result of the message in 
			pSB1C3 with VF2 and VR primers.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 8. Sequencing result 249 of the message in 
			pDR111.</FIGCAPTION></FIGURE></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><H5>Experiments:</H5><P>See integration of the message sequence into <EM>B. subtilis</EM> via the BioBrick <A href="http://parts.igem.org/Part:BBa_K823023">BBa_K823023</A> integration plasmid in <A href="https://2016.igem.org/Team:Groningen/Experiments#proof-of-concept">Proof of concept</A>
experiment. 
	</P></DIV></SECTION><SECTION id="Message-in-K823023" class="collapse-h3"><H3>Message sequence in BBa_K823023</H3><P><A href="http://parts.igem.org/Part:BBa_K823023">BBa_K823023</A> is an available BioBrick from igem Munich 2012. It is an integration plasmid for <EM>Bacillus subtilis</EM>, which can be used for 
	cloning in <EM>E. coli</EM> as well. An RFP is inserted in BBa_K823023 for more 
	efficient screening after transformation. It was chosen to 
	construct the <EM>Bacillus subtilis</EM> message strain. Construction was 
	performed as described in the following.</P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P>06/10/16: Message sequence was amplified from pDR111+message plasmid. 
		Primers used for the amplification were message prefix and message 
		suffix (primer sequences can be found <A href="https://2016.igem.org/Team:Groningen/Labjournal/PrimerList">here</A>).</P><H5>PCR mixture:</H5><P>50 μl PCR assay was performed according to the following <A href="/Team:Groningen/Protocols#PCR-Taq">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gels see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 1. DNA electrophoresis with amplified message 
			sequence (M1 and M2) 599 bp in size. C - is water control.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>Amplification of the message sequence and addition of the prefix and 
		suffix to this sequence was successful. It was verified by DNA 
		electrophoresis (see above). Correct size of a band could be seen and 
		that was 599 bp.</P><H5>Procedure after gel validation:</H5><P>PCR product was subsequently cleaned with <A href="/Team:Groningen/Protocols#Clean-up-Jena">(PCR Purification Kit – Jena Bioscience)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>07/10/16: On this day restriction digestion of PCR product of the 
		message sequence and BBa_K823023 was done. Message sequence 
		as an insert was cut with EcoRI and PstI restriction enzymes. 
		BBa_K823023 as a vector was cut with exactly same restriction 
		enzymes.</P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following <A href="https://2016.igem.org/Team:Groningen/Protocols#rd-jena">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run 
		agarose gels see following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 2. DNA electrophoresis with cut backbone 
			BBa_K823023.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>RD of the PCR product of message sequence and BBa_K823023 was 
		successful. RD of BBa_K823023 was verified by DNA electrophoresis. Correct size of band could be seen and that was ∼6000 bp.</P><H5>Procedure after gel validation:</H5><P>The upper band was cut out from the gel and DNA was extracted by Agarose gel 
		extraction kit <A href="/Team:Groningen/Protocols#Gel-extract-Jena">(Jena Bioscience)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>07/10/16 Restriction digestion was followed by overnight 
		ligation. Vector (BBa_K823023) and insert (message sequence) was 
		ligated in 4:6 molar ratio.</P><H5>Ligation mixture:</H5><P>20 μl ligation assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>08/10/16: On this day transformation into <EM>E. coli</EM> Top10 was done. 
		Selection was made on 100 μg/ml ampicillin LB agar plates. 
		Transformation protocol used can be found <A href="/Team:Groningen/Protocols#transform-e-coli-standard">here</A>.</P><P>09/10/16: Next day we could assume that our transformation was 
		successful -&gt; colonies were obtained on the plates of <EM>E. coli</EM> Top10 
		strain. To verify correct transformants colony PCR was 
		performed (see colony PCR <A href="/Team:Groningen/Protocols#Colony-PCR">protocol</A>). Primers message 
		prefix and message suffix were used for this colony PCR. Sequences of 
		these primers can be found <A href="https://2016.igem.org/Team:Groningen/Labjournal/PrimerList">here</A>.</P><H5>PCR mixture:</H5><P>50 μl PCR assay was performed according to the following <A href="/Team:Groningen/Protocols#PCR-Taq">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel <A href="https://2016.igem.org/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><FIGURE><FIGCAPTION>Figure 3. DNA electrophoresis with samples from colony 
			PCR.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 4. Transformation of BBa_K823023+message into 
			<EM>E. coli</EM> Top10.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>Transformation of BBa_K823023+message into <EM>E. coli</EM> Top10 appeared to 
		be successful. Correct sizes of the bands from colony PCR (see Figure 
		3) were obtained: 599 bp. Correct clones were grown overnight (see cell 
		culture <A href="/Team:Groningen/Protocols#cell-cultures-e-coli">protocol</A>) from pre-glycerol stocks 
		made for colony PCR (see colony PCR <A href="/Team:Groningen/Protocols#Colony-PCR">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>10/10/16 Grown cultures of <EM>E. coli</EM> Top10 with BBa_K823023+message were 
		taken out from the incubator after overnight incubation. <A href="/Team:Groningen/Protocols#GS">Glycerol stocks</A> were made and plasmid isolation was 
		performed (see <A href="/Team:Groningen/Protocols#Mini-prep-Jena">(Fast-n-Easy Plasmid Mini-Prep Kit - Jena Bioscience)</A> protocol). Concentration of the 
		plasmids obtained was measured on Nanodrop and plasmids were stored at 
		-20°C.</P></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><H5>Experiments:</H5><P>See <A href="https://2016.igem.org/Team:Groningen/Experiments#proof-of-concept">Proof of concept</A>
experiment.</P></DIV></SECTION><SECTION id="sfGFP-Sp-in-pSB1C3" class="collapse-h3"><H3>sfGFP(Sp) in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K1930006">BBa_K1930006</A>)</H3><P>sfGFP(Sp) is a reporter gene originally optimized for <EM>Streptococcus pneumoniae</EM>. It has been shown when expressed in <EM>B. subtilis</EM> that the signal is even brighter than the one from sfGFP(Bs) <A href="http://aem.asm.org/content/79/20/6481.full">(Overkamp et al. 2013)</A>. To submit sfGFP(Sp) reporter gene as BioBrick (<A href="http://parts.igem.org/Part:BBa_K1930006">BBa_K1930006</A>) we cloned it in the pSB1C3 standard iGEM backbone. </P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P>06/10/16: sfGFP(Sp) was amplified from the pDR111+sfGFP(Sp) plasmid, that 
		we constructed during the project, with the primers prefix sfGFP(Sp) and 
		suffix sfGFP(Sp) (primer sequences can be found <A href="https://2016.igem.org/Team:Groningen/Labjournal/PrimerList">here</A>). </P><H5>PCR mixture:</H5><P>50 μl PCR assay was performed according to the following
		<A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 1. DNA electrophoresis with amplified sfGFP(Sp) 
			gene (763 bp in size).</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The sfGFP(Sp) was successfully amplified from the pDR111+sfGFP(Sp) plasmid. </P><H5>Procedure after gel validation:</H5><P>PCR product was subsequently cleaned with <A href="/Team:Groningen/Protocols#Clean-up-Jena">PCR Purification Kit – Jena Bioscience</A>. 
		
	</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>The sfGFP(Sp) was cut with EcoRI and PstI. The backbone pSB1C3 <A href="http://parts.igem.org/Part:BBa_J04450">(BBa_J04450)</A> was digested with the same enzymes. This construct is carrying RFP reporter therefore it was used for easier screening after transformation. You could see self-ligations as red 
		colonies and the correct ones as white ones. </P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#rd-neb">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 2. DNA electrophoresis of EcoRI and PstI digested 
			pSB1C3 - BBa_J04450. The RFP insert is 1069bp and the pSB1C3 backbone 
			is 2019 bp.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The digestion was successful because bands for both expected 
		fragments could be seen on the gel, namely RFP insert is 1069bp and the 
		pSB1C3 backbone is 2019 bp.</P><H5>Procedure after gel validation:</H5><P>The upper band of 2000 bp was cut out from the gel 
		and DNA was extracted with <A href="/Team:Groningen/Protocols#Gel-extract-Jena">Agarose Gel Extraction Kit – Jena Bioscience</A>. The PCR product was not 
		checked on the gel after the digestion but immediately cleaned up with 
		<A href="/Team:Groningen/Protocols#Clean-up-Nucleospin">NucleoSpin® Gel and PCR Clean-up</A>.</P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>The cut and cleaned sfGFP(Sp) was ligated to the cut and cleaned 
		pSB1C3 in a ratio of 2:1. </P><H5>Ligation mixture:</H5><P>20 μl ligation assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>07/10/16: The ligation mix was heat shock transformed to competent Top10 <EM>E. coli</EM> cells following the transformation <A href="/Team:Groningen/Protocols#transform-e-coli-standard">protocol</A>. 
		Cells were plated on 50 μg/ml chloramphenicol LB agar to select the 
		correct construct.</P><P>09/10/16: Colonies were picked to perform colony PCR to find the correct 
		constructs with the primers prefix sfGFP(Sp) and suffix sfGFP(Sp). Find primer sequences 
		<A href="/Team:Groningen/Labjournal/PrimerList">here</A>.</P><FIGURE><FIGCAPTION>Figure 3. Top10 <EM>E. coli</EM> colonies on 50 μg/ml chloramphenicol LB 
			agar plates after transformation with sfGFP(Sp) in pSB1C3.</FIGCAPTION></FIGURE><H5>PCR mixture:</H5><P>25 μl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>1:30 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 4. DNA electrophoresis of the colony PCR product. 
			C - is the water control. Samples 1-5 are from colony 1-5.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>All 5 samples show the right band by 763 bp. Therefore all of them 
		were grown overnight to harvest the plasmid the following day.</P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>10/10/16: Grown cultures of <EM>E. coli</EM> Top10 with the construct sfGFP(Sp) in 
		pSB1C3 were used to obtain (<A href="/Team:Groningen/Protocols#GS">glycerol stocks</A>) and plasmid 
		isolation was performed with <A href="/Team:Groningen/Protocols#Mini-prep-QIA">QIAprep® Spin Miniprep Kit</A>. Some of the overnight
		cultures seemed to already express the sfGFP (see Figure 5). Firstly, 
		concentration of the plasmids obtained was measured on Nanodrop. 
		Secondly, plasmids were sent for sequencing and then stored at 
		-20°C.</P><FIGURE><FIGCAPTION>Figure 5. Pellet of the overnight cultures from sample 
			2 and 5. Looks like the sfGFP(Sp) is also expressed in <EM>E. coli</EM> 
			Top10.</FIGCAPTION></FIGURE><H5>Sequencing:</H5><P> Sequencing results showed that sfGFP(Sp) gene is present (see Figure 6). However something happened with the prefix and the suffix most probably during cloning steps. It looks that prefix and also suffix was disrupted by insertion of few base pairs. We cannot really explain what happened and due to time limitation we could not fix this problem. However sfGFP(Sp) is clearly expressed in <EM>E. coli</EM> (see Figure 5).</P><FIGURE><FIGCAPTION>Figure 6. Sequencing result of sfGFP(Sp) in pSB1C3. GFP gene itself is underlined by red line.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The sfGFP(Sp) was cloned to the backbone pSB1C3. However from the sequencing results we could see that this part contains bad prefix and suffix (see Figure 6). GFP gene is underlined by red line on Figure 6, however you could also see the bad prefix and suffix. </P></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><P>See <A href="https://2016.igem.org/Team:Groningen/Experiments#decoy">decoy</A> experiments with sfGFP(Sp) in <EM>B. subtilis</EM>.</P></DIV></SECTION><SECTION id="sfGFP-Sp-in-pDR111-pDR111-message-plasmid" class="collapse-h3"><H3>sfGFP(Sp) in pDR111 and in pDR111+message plasmid</H3><P>sfGFP(Sp) is a reporter gene originally optimized for <EM>Streptococcus pneumoniae</EM>. It has been shown when expressed in <EM>B. subtilis</EM> that the signal is even brighter than the one from sfGFP(Bs) <A href="http://aem.asm.org/content/79/20/6481.full">(Overkamp et al. 2013)</A>. sfGFP(Sp) was cloned either only to pDR111 integration plasmid or to 
	pDR111+message plasmid. These plasmids were constructed with an interest to 
	have easier screening when integrated into the <EM>B. subtilis</EM> genome. pDR111 is an integration plasmid which can be integrated into the <EM>B. 
	subtilis</EM> genome. By double cross-over it replaces amyE gene, which is 
	necessary for production of <A href="http://subtiwiki.uni-goettingen.de/bank/index.php?gene=amyE&amp;action=Go">alpha-amylase</A>, 
	with desired insert which is located between amyE front flanking region and 
	amyE back flanking region. See the plasmid map below. </P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P>10/08/16: sfGFP(Sp) was amplified from pNW plasmid together with its 
		promoter (P<SUB>pta</SUB>), which is a constitutive promoter from <EM>Geobacillus</EM> 
		DSM2542 for phosphate acetyl transferase expression, and a triple 
		terminator (3TER). Primers used for the amplification were F-gfp insert 
		and R-gfp insert (primer sequences can be found <A href="https://2016.igem.org/Team:Groningen/Labjournal/PrimerList">here</A>).</P><H5>PCR mixture:</H5><P>25 μl PCR assay was performed according to the following <A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>57ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>1:30 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel 
		<A href="/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><FIGURE><FIGCAPTION>Figure 2. DNA electrophoresis with amplified 
			P<SUB>pta</SUB>-sfGFP(Sp)-3TER (1147 bp in size).</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>Amplification of the P<SUB>pta</SUB>-sfGFP(Sp)-3TER was successful. It was 
		verified by DNA electrophoresis. Correct size of a band 
		could be seen and that was 1147 bp. However second band could be seen, 
		that was probably caused by unspecific binding of the primers used. 
		Therefore lower band of correct size was cut out from the gel and DNA 
		was extracted.</P><H5>Procedure after gel validation:</H5><P>DNA extraction was done according to this <A href="/Team:Groningen/Protocols#Gel-extract-Jena">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>10/08/16: On this day restriction digestion of PCR product of the 
		sfGFP(Sp), pDR111 integration plasmid and pDR111+message 
		integration plasmid was done. sfGFP(Sp) as an insert was cut with BglII 
		and NheI restriction enzymes. Integration plasmid pDR111 and 
		pDR111+message as a vector was cut with BamHI and NheI. BglII could not 
		be used as a restriction enzyme for pDR111 due to multiple recognition 
		site of this cutter. BamHI as a compatible restriction enzyme to BglII 
		was used instead. </P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following <A href="/Team:Groningen/Protocols#rd-fast">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel 
		<A href="/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><FIGURE><FIGCAPTION>Figure 3. DNA electrophoresis with samples from restriction 
			digestion.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 4. DNA electrophoresis with samples from restriction 
			digestion.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>RD of the PCR product of sfGFP(Sp), integration plasmid pDR111 and 
		pDR111+message was successful. RD of pDR111 and pDR111+message was 
		verified by DNA electrophoresis. Correct sizes of bands could 
		be seen and that was 6479 bp and 7034 bp, respectively.</P><H5>Procedure after gel validation:</H5><P>The upper band was cut out from the gel and DNA was extracted by Gel 
		extraction kit <A href="/Team:Groningen/Protocols#Gel-extract-Nucleospin">(NucleoSpin® Gel and PCR Clean-up)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>10/08/16: Restriction digestion was followed by ligation at 
		room temperature for 45 min. Vector (integration plasmid pDR111 and 
		pDR111+message) and insert (sfGFP(Sp)) was ligated in 1:5 molar ratio.</P><H5>Ligation mixture:</H5><P>20 μl ligation assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>10/08/16: Ligation was followed by transformation into <EM>E. coli</EM> Top10. 
		Selection was made on 100 μg/ml ampicillin LB agar plates. 
		Transformation protocol used can be found <A href="/Team:Groningen/Protocols#transform-e-coli-standard">here</A>. </P><P>11/08/16: Next day we could assume that our transformation was 
		successful -&gt; colonies were obtained on the plates of <EM>E. coli</EM> Top10 
		strain. To verify correct transformants we grew some of the colonies 
		overnight in 3 ml LB with 100 μg/ml ampicillin (see cell culture 
		<A href="/Team:Groningen/Protocols#cell-cultures-e-coli">protocol</A>). On the next day plasmid 
		purification and restriction digestion control was done.</P><H5>Conclusion:</H5><P>Transformation of pDR111+sfGFP(Sp) and pDR111+message+sfGFP(Sp) into 
		<EM>E. coli</EM> Top10 was assumed to be successful. To see if we obtained 
		correct clones we did a restriction digestion control with EcoRI 
		restriction enzyme. </P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>12/08/16: Next day plasmid purification was done according to this 
		<A href="/Team:Groningen/Protocols#Mini-prep-QIA">protocol</A> and restriction digestion control with EcoRI 
		restriction enzyme was done to see if correct clones were obtained. 
		This enzyme has 2 recognition sites in our construct. Therefore two 
		bands (1535 bp and 6081 bp from pDR111+sfGFP(Sp) and 2090 bp and 6081 
		bp from pDR111+message+sfGFP(Sp)) should have been obtained on the 
		agarose gel. Moreover we sent this new constructed plasmid for 
		sequencing.</P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following <A href="/Team:Groningen/Protocols#rd-fast">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel 
		<A href="/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><FIGURE><FIGCAPTION>Figure 5. DNA electrophoresis with samples (1-4 from 
			pDR111+sfGFP(Sp)) from RD control with EcoRI.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 6. DNA electrophoresis with samples (1-6 from 
			pDR111++message+ sfGFP(Sp)) from RD control with EcoRI. 
			</FIGCAPTION></FIGURE><H5>Sequencing:</H5><P>Plasmid pDR111+sfGFP(Sp) and pDR111+message+sfGFP(Sp) was sent for 
		<A href="/Team:Groningen/Protocols#Sequencing">sequencing</A>. Primers used for 
		the sequencing were F-gfp insert for pDR111+sfGFP(Sp) and F-gfp insert 
		and F-message sequence for pDR111+message+sfGFP(Sp) (primer sequences 
		can be found <A href="https://2016.igem.org/Team:Groningen/Labjournal/PrimerList">here</A>). </P><H5>Conclusion:</H5><P>Figure 5 shows two correct bands (1535 bp and 6081 bp) when the 
		pDR111+sfGFP(Sp) was cut.  Figure 6 shows two correct bands (2090 bp 
		and 6081 bp) when the pDR111+message+sfGFP(Sp) was cut.  Therefore we 
		assumed that our cloning was successful and correct plasmids were 
		obtained. Sequencing results confirmed our assumption as you can see in 
		Figure 7, 8 and 9.</P><FIGURE><FIGCAPTION>Figure 7. Sequencing result 334 of the sfGFP(Sp) 
			construct in pDR111 (primer used: F-gfp 
			insert).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 8. Sequencing result 335 of the sfGFP(Sp) 
			construct in pDR111+message (primer used: F-gfp 
			insert).</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 9. Sequencing result 336 of the sfGFP(Sp) 
			construct in pDR111+message (primer used: F-message 
			sequence).</FIGCAPTION></FIGURE></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><H5>Experiment:</H5><P>The functionality of these constructs was checked by integration 
		into the <EM>B. subtilis</EM> genome 168 trp+ and imaging under the microscope. </P><H5>Integration into the genome of <EM>B. subtilis</EM> 168 trp+:</H5><P>sfGFP(Sp) in pDR111 and pDR111+message was integrated into the <EM>B. 
		subtilis</EM> 168 trp+ genome. pDR111 is an integration plasmid which can be 
		integrated into the <EM>B. subtilis</EM> genome. By double cross-over it 
		replaces amyE gene, with desired insert, in this case insert is 
		P<SUB>pta</SUB>-sfGFP(Sp)-3TER, which is located between amyE front 
		flanking region and amyE back flanking region. </P><H5>Experiment set-up:</H5><P>10 μl of sfGFP(Sp) in pDR111 and pDR111+message plasmids were 
		transformed into the <EM>B. subtilis</EM> 168 trp+ strain (see transformation 
		<A href="/Team:Groningen/Protocols#transform-b-subtilis">protocol</A>). Selection was made on 150 μg/ml 
		spectinomycin LB agar plates.</P><H5>Microscopy experiment set-up:</H5><P><EM>B. subtilis</EM> cells were grown overnight in 3 ml LB with 150 μg/ml 
		spectinomycin at 37°C and 220 rpm. On the next day microscopy slide was 
		prepared as explained in the following <A href="/Team:Groningen/Protocols#Microscopy">protocol</A>. Cells 
		were imaged using phase contrast: filter POL 50% 1 s exposure, green 
		fuorescence: FITC, 0.8 s exposure, objective: Olympus 100X/1.40, 
		camera: CoolSNAP_HQ/HQ2-ICX285 and software: Resolve3D softWoRx-Acquire 
		version.</P><FIGURE><FIGCAPTION>Figure 10. <EM>B. subtilis</EM> 168 trp+ cells with sfGFP 
			integrated into the amyE locus.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>Integration into the <EM>B. subtilis</EM> 168 trp+ strain of the plasmids 
		with the reporter sfGFP(Sp) was successful. Multiple colonies were 
		obtained on selection plates. <EM>B. subtilis</EM> cells were subsequently grown 
		and checked under the microscope to see if there is an expression of 
		GFP (see Figure 10). We could definitely see the GFP signal when 
		observing the cells under the microscope. This was big enough proof 
		that our plasmid construction followed by integration into the <EM>B. 
		subtilis</EM> genome was successful.</P></DIV></SECTION><SECTION id="PatpI-in-pSB1C3" class="collapse-h3"><H3>P<SUB>AtpI</SUB> in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K1930005">BBa_K1930005</A>)</H3><P>The promoter P<SUB>AtpI</SUB> has its origin in <EM>Bacillus subtilis</EM>. It is 
	responsible for the expression of <EM>atpA</EM> gene (ATP synthesis) during the first 30 
	min of the germination  of <EM>B. subtilis</EM><A href="https://www.ncbi.nlm.nih.gov/pubmed/25661487">(Sinai et al. 2015)</A>. <EM>atpA</EM> gene is part of an operon <EM>atpI-atpB-atpE-atpF-atpH-atpA-atpG-atpD-atpC</EM>, therefore the promoter region in front of the first protein coding gene (<EM>atpI</EM>) in this operon was chosen. The region was checked for the binding of sigma factors and 
	transcription factors with <A href="http://dbtbs.hgc.jp/">DBTBS</A>. No 
	binding factors were found with the highest significance level. In our 
	project we wanted to find a constitutive promoter for our ciprofloxacin 
	resistance casette. In the following part we put the promoter P<SUB>AtpI</SUB> in the 
	pSB1C3 backbone to make it available to other iGEM teams. </P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P>06/10/16: The promoter P<SUB>AtpI</SUB> was amplified (see <A href="/Team:Groningen/Protocols#PCR-Q5">PCR protocol</A>) from the 
		ciprofloxacin resistance cassette BioBrick <A href="http://parts.igem.org/Part:BBa_K1930004">BBa_K1930004</A>) with the primers pATPI+prefix and pATPI+suffix (primer sequences can be 
		found <A href="/Team:Groningen/Labjournal/PrimerList">here</A>). The 
		correct size of 372 bp for the PCR product was checked with DNA
		electrophoresis. The PCR product was stored at 4°C.</P><H5>PCR mixture:</H5><P>50 μl PCR assay was performed according to the following
		<A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 1. DNA electrophoresis with amplified P<SUB>AtpI</SUB> promoter 
			sequence 372 bp in size.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The P<SUB>AtpI</SUB> promoter was successfully amplified with prefix and suffix 
		from the plasmid.</P><H5>Procedure after gel validation:</H5><P>PCR product was subsequently cleaned with <A href="/Team:Groningen/Protocols#Clean-up-Jena">PCR Purification Kit – 
		Jena Bioscience</A>.</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>07/10/16: The P<SUB>AtpI</SUB> was cut with EcoRI and PstI. The backbone pSB1C3 <A href="http://parts.igem.org/Part:BBa_J04450">(BBa_J04450)</A> was digested with the same enzymes. This construct is carrying RFP reporter therefore it was used for easier screening after transformation. You could see self-ligations as red 
		colonies and the correct ones as white ones. </P><H5>RD mixture:</H5><P>30 μl RD assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#rd-jena">protocol</A>.</P><H5>DNA Electrophoresis: </H5><P>For detailed information on how to prepare 
		and run agarose gel see following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 2. DNA electrophoresis of EcoRI and PstI digested 
			pSB1C3 - BBa_J04450. The RFP insert is 1069bp and the pSB1C3 backbone 
			is 2019 bp.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The digestion was successful because bands for both expected 
		fragments could be seen on the gel, namely RFP insert is 1069bp and the 
		pSB1C3 backbone is 2019 bp.</P><H5>Procedure after gel validation:</H5><P>The upper band of 2000 bp was cut out from the gel 
		and DNA was extracted by <A href="/Team:Groningen/Protocols#Gel-extract-Jena">Gel extraction kit 
		(Nucleospin)</A>. The PCR product was not checked on the gel after the 
		digestion but immediately cleaned up with <A href="/Team:Groningen/Protocols#Clean-up-Nucleospin">(CR 
		Purification Kit – Jena Bioscience</A>.</P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>07/10/16: The EcoRI, PstI cut pSB1C3 backbone was ligated with the 
		EcoRI, PstI cut promoter P<SUB>AtpI</SUB>. </P><H5>Ligation mixture:</H5><P>20 μl ligation assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment: </H5><P>08/10/16: The ligation mix was heat shock transformed to competent Top10 <EM>E. coli</EM> cells following the <A href="/Team:Groningen/Protocols#transform-e-coli-standard">protocol</A>. 
		Cells were plated on 50 μg/ml chloramphenicol LB agar to select the 
		correct constructs. The next day colonies were picked to perform colony 
		PCR to find the correct constructs with the primers pATPI+prefix and pATPI+suffix (primer sequences can be found <A href="/Team:Groningen/Labjournal/PrimerList">here</A>).</P><FIGURE><FIGCAPTION>Figure 3. <EM>E. coli</EM> Top10 transformed with P<SUB>AtpI</SUB> in pSB1C3. </FIGCAPTION></FIGURE><H5>PCR mixture:</H5><P>25 μl PCR assay was performed according to the following <A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s     </TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s     </TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>1:30 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 4. Result of the colony PCR. Samples from colony 
			1-5 are showing the correct size of the promoter P<SUB>AtpI</SUB> of 372 bp. 
			C - is the water control. </FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The transformation of P<SUB>AtpI</SUB> in pSB1C3 to <EM>E. coli</EM> Top10 was 
		successful. </P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>09/10/16: Grown cultures of <EM>E. coli</EM> Top10 with the construct were used to 
		make <A href="/Team:Groningen/Protocols#GS">glycerol stocks</A> and 
		plasmid isolation was performed (see <A href="/Team:Groningen/Protocols#Mini-prep-Jena">Fast-n-Easy Plasmid Mini-prep kit</A>). 
		Firstly, concentration of the plasmids obtained was measured on 
		Nanodrop. Secondly, plasmids were sent for sequencing and then stored 
		at -20°C.</P><H5>Sequencing:</H5><P>The plasmid P<SUB>AtpI<SUB> in pSB1C3 from colonies 1 and 2 was sent for 
		sequencing with the primers VF2 and VR, see <A href="/Team:Groningen/Labjournal/PrimerList">primer list</A>.</SUB></SUB></P><FIGURE><FIGCAPTION>Figure 6. Sequencing result for P<SUB>AtpI</SUB> in pSB1C3 from 
			colony 1 with VR.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 5. Sequencing result for P<SUB>AtpI</SUB> in pSB1C3 from 
			colony 1 with VF2.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The sequencing result proofed the successful integration of the 
		P<SUB>AtpI</SUB> promoter in the pSB1C3. Another BioBrick was obtained!</P></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><P>For further experiments with this BioBrick see: <A href="https://2016.igem.org/Team:Groningen/Experiments">MIC value experiment</A>.</P></DIV></SECTION><SECTION id="Cipro-in-pSB1C3" class="collapse-h3"><H3>Ciprofloxacin resistance cassette in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K1930004">BBa_K1930004</A>)</H3><P>To design a <EM>qnrS1</EM> resistance cassette BioBrick we designed a gBlock that contains the <EM>Bacillus subtilis</EM> constitutive promoter P<SUB>AtpI</SUB>, which is active from a very early stage of germination and includes a ribosome binding site. The gBlock also contains the original <EM>qnrS1</EM> gene sequence from <EM>E. coli</EM>, the double terminator (<A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A>) from iGEM as well as the prefix and suffix for BioBricks. In summary the ciprofloxacin cassette consists of the following parts P<SUB>AtpI</SUB>+RBS+<EM>qnrS1</EM>+2TER, see plasmid map below.</P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P>The sequence of the <EM>qnrS1</EM> gene was amplified from the gBlock <EM>qnrS1</EM><EM>E. coli</EM> ordered from IDT. Primers used for the 
		amplification were F-qnrs1 <EM>E.coli</EM> and R-qnrs1 <EM>E.coli</EM> (primer sequences can be found <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">here</A>).

		</P><H5>PCR mixture:</H5><P>50 µl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(12X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(12X)</TD></TR><TR><TD>72ºC</TD><TD>1:30 min</TD><TD>(12X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gels see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 1. Gel electrophoresis of the PCR product.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The PCR of the  <EM>qnrS1</EM> sequence from the IDT gBlock was successful. It 
		was verified by DNA electrophoresis. A band with the correct 
		size of 1,194 bp could be seen.</P><H5>Procedure after gel validation:</H5><P>PCR product was subsequently cleaned with <A href="/Team:Groningen/Protocols#Clean-up-Jena">(PCR Purification Kit – Jena Bioscience)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>28/09/16 The <EM>qnrS1</EM> gene should have been cloned into the pSB1C3. For this case the vector <A href="http://parts.igem.org/Part:BBa_J04450">BBa_J04450</A> was used. The <EM>qnrS1</EM> gene and BBa_J04450 were cut with the restriction enzymes EcoRI and PstI.</P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following <A href="https://2016.igem.org/Team:Groningen/Protocols#rd-jena">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run 
		agarose gels see following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><P>Bands of the sizes 2,029 bp for the vector and 1,176 bp for the gene 
		were expected.</P><FIGURE><FIGCAPTION>Figure 2. DNA electrophoresis of EcoRI and PstI digested 
			pSB1C3 - BBa_J04450. The RFP insert is 1,069 bp and the pSB1C3 backbone 
			is 2019 bp.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The digestion was successful because bands for both expected 
		fragments could be seen on the gel, namely RFP insert is 1,069 bp and the 
		pSB1C3 backbone is 2,019 bp.</P><H5>Procedure after gel validation:</H5><P>The digested samples were cut out from the gel and the DNA was 
		extracted using the Agarose gel Extraction Kit <A href="/Team:Groningen/Protocols#Gel-extract-Jena">(Jena Bioscience)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>28/09/16: After restriction digestion ligation was performed. 6 µl 
		<EM>qnrS1</EM> insert DNA were ligated to 4 µl pBS1C3 vector DNA. The 
		ligation took place for 2 h at room temperature.</P><H5>Ligation mixture:</H5><P>20 µl ligation assay was performed according to the following <A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>28/09/16 After ligation the ciprofloxacin resistance cassette in pSB1C3 was transformed into <EM>E. coli</EM> Top10 cells. The transformation was plated on plates containing 50 µg/ml 
		chloramphenicol. The transformation was performed according to the 
		transformation <A href="/Team:Groningen/Protocols#transform-e-coli-standard">protocol</A>.</P><P>29/09/16: To analyze the success of the transformation 12 samples 
		were picked and a colony PCR was performed according to the following <A href="/Team:Groningen/Protocols#Colony-PCR">protocol</A>. The primers F-qnrs1 <EM>E.coli</EM> and R-qnrs1 
		<EM>E.coli</EM> were used. Primer sequences can be found <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">here</A>.

		</P><H5>DNA electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel <A href="https://2016.igem.org/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><H5>Conclusion:</H5><P>The colony PCR did not bring clear results. </P><H5>Experiment:</H5><P>30/09/16: To further analyze the success of the transformation 
		four colonies were grown as overnight cultures (started on 29/09/16) 
		in LB and 50 µg/ml chloramphenicol. On the next day plasmid isolation was performed and plasmids were digested using the enzymes EcoRI and PstI according to the following restriction digestion<A href="/Team:Groningen/Protocols#rd-neb">protocol</A>.</P><H5>DNA electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel <A href="https://2016.igem.org/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><H5>Conclusion:</H5><P>Two out of the four digested samples showed bands of the expected 
		sizes 2,029 bps for the vector and 1184 for the gene could be seen.</P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>The two samples that showed the correct digestion pattern in the restriction digestion control were sent for sequencing with the primers VF2 and VR, see <A href="/Team:Groningen/Labjournal/Primerlist">primer list</A>. These two samples showed the correct sequence (Figure 5.1-5.4, 5.1 mutant 1 forward, 5.2: mutant 2 forward, 5.3: mutant 1 reverse, 5.4: mutant 2 reverse ). </P><FIGURE><FIGCAPTION>Figure 5.1</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 5.2</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 5.3</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 5.4</FIGCAPTION></FIGURE></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><P> For further experiment see the construction of the plasmid <A href="https://2016.igem.org/Team:Groningen/Labjournal#Cipro-in-K823023">ciprofloxacin resistance cassette in BBa_K823023</A>.</P></DIV></SECTION><SECTION id="Cipro-in-K823023" class="collapse-h3"><H3>Ciprofloxacin resistance cassette in BBa_K823023</H3><P>To design a ciprofloxacin resistance cassette we designed a gBlock that contains the <EM>Bacillus subtilis</EM> promoter P<SUB>AtpI</SUB>, which is active from a very early stage of germination and includes a ribosome binding site. The gBlock also contains the original <EM>qnrS1</EM> gene sequence from <EM>E. coli</EM>. <EM>qnr</EM> genes code for pentapeptide repeat proteins. These proteins reduce susceptibility to quinolones by protecting the complex of DNA and DNA gyrase enzyme from the inhibitory effect of quinolones. Finally, this gBlock contains a double terminator <A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A> from iGEM as well as the prefix and suffix for BioBricks. In summary the ciprofloxacin cassette consists of the following parts P<SUB>AtpI</SUB>+RBS+<EM>qnrS1</EM>+2TER.<A href="https://2016.igem.org/Team:Groningen/Experiments#Key-in-BBa_K823023">BBa_K823023</A> is an available BioBrick from igem Munich 2012. It is an 
	integration plasmid for <EM>Bacillus subtilis</EM>, which can be used for 
	cloning in <EM>E. coli</EM> as well. An RFP is inserted in BBa_K823023 for more 
	efficient screening after transformation. Construction of ciprofloxacin resistance cassette in BBa_K823023 integration plasmid was performed as described in the following.</P><DIV class="collapse-h4"><H4 class="dna">PCR</H4><H5>Experiment:</H5><P>The sequence of the <EM>qnrS1</EM> gene was amplified from the gBlock <EM>qnrS1</EM>
		E. coli ordered from IDT. Primers used for the 
		amplification were F-qnrs1 <EM>E.coli</EM> and R-qnrs1 <EM>E.coli/</EM> (primer sequences can be found <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">here</A>).

		</P><H5>PCR mixture:</H5><P>50 µl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#PCR-Q5">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(12X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(12X)</TD></TR><TR><TD>72ºC</TD><TD>1:30 min</TD><TD>(12X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gels see 
		following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 1. DNA electrophoresis of the PCR product.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The PCR of the <EM>qnrS1</EM> sequence from the IDT gBlock was successful. It 
		was verified by DNA electrophoresis. A band with the correct 
		size of 1,194 bp could be seen.</P><H5>Procedure after gel validation:</H5><P>PCR product was subsequently cleaned with <A href="/Team:Groningen/Protocols#Clean-up-Jena">(PCR Purification Kit – Jena Bioscience)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>28/09/16: The <EM>qnrS1</EM> PCR product and BBa_K823023 were cut with the 
		restriction enzymes EcoRI and PstI.</P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following <A href="https://2016.igem.org/Team:Groningen/Protocols#rd-jena">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>For detailed information on how to prepare and run 
		agarose gels see following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 2. BBa_K823023 cut with EcoRI and PstI. Backbone ~6,000 
			bp, RFP insert 1000 bp.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The digestion of BBa_K823023 showed the correct bands on the gel. 6,000 bp backbone for the backbone and the 1000 bp for the RFP insert. Therefore the cloning procedure could proceed. </P><H5>Procedure after gel validation:</H5><P>Digested sample of the backbone ~6,000 bp were cut out from the 
		gel and DNA was extracted by Agarose gel extraction kit (Jena Bioscience)
		<A href="https://2016.igem.org/Team:Groningen/Labjournal#Gel-extract-Jena">(see protocol)</A>. </P><P>The digestion of the <EM>qnrS1</EM> PCR product was immediately cleaned with 
		the kit <A href="/Team:Groningen/Protocols#Clean-up-Jena">(PCR Purification Kit – Jena Bioscience)</A>.</P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><P>19/10 In this ligation the EcoRI, PstI cut gene <EM>qnrS1</EM> and the vector 
		BBa_K823023 were combined.</P><H5>Ligation mixture:</H5><P>20 µl ligation assay was performed according to the following <A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>29/09/16: The ligation mix was heat shock transformed to competent Top10 <EM>E. coli</EM> cells following the <A href="/Team:Groningen/Protocols#transform-e-coli-standard">protocol</A>. Cells 
		were plated on 100 μg/ml ampicillin LB agar to select for the correct 
		constructs. On the next day colonies were selected to perform colony PCR in 
		order to find the correct constructs using the primers F-qnrs1 <EM>E.coli</EM> 
		and R-qnrs1 <EM>E.coli</EM>. Primer sequences can be found <A href="https://2016.igem.org/Team:Groningen/Experiments#primer-list">here</A>.
		
		<FIGURE><FIGCAPTION>Figure 3. Transformation to <EM>E. coli</EM> Top10. </FIGCAPTION></FIGURE></P><H5>PCR mixture:</H5><P>25 µl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#Colony-PCR">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>1:30 min</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel <A href="https://2016.igem.org/Team:Groningen/Protocols#DNA-electrophoresis">see following protocol</A>.</P><FIGURE><FIGCAPTION>Figure 4. Result of the colony PCR. Sample indicated 
			with the star shows the correct sie of 1194 bp. C - is the water 
			control. </FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The transformation of qrnS1 (ciprofloxacin resistance cassette) in BBa_K823023 to <EM>E. coli</EM> Top10 was successful. </P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>To test if the construct would make <EM>B. subtilis</EM> resistant to 
		ciprofloxacin, the construct qrnS1 in BBa_K823023 was transformed into the <EM>B. 
		subtilis</EM> 168 tpr+.</P></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><H5>Experiment:</H5><P>13/10/16: The transformation to <EM>B. subtilis</EM> was performed according to the following <A href="/Team:Groningen/Protocols#transform-b-subtilis">protocol</A>. Colonies were selected on LB agar 
		with 5 μg/ml chloramphenicol.</P><FIGURE><FIGCAPTION>Figure 5. <EM>B. subtilis</EM> after transformation with 
			ciprofloxacin resistance cassette. </FIGCAPTION></FIGURE><P>14/10/16: Colonies were streaked out on agar with starch to perform the 
		starch test, which verifies the integration in the amyE locus in the 
		genome of <EM> B. subtilis</EM>. <A href="/Team:Groningen/Protocols#Starch-test">Integration check: Starch test</A></P><FIGURE><FIGCAPTION>Figure 6. Starch test. Colonies without a clear halo 
			are positive for integration.</FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The integration of the ciprofloxacin resistance cassette was 
		successful. </P><P>As a first check on the functionality of the ciprofloxacin 
		resistance cassette, we grew <EM>B. subtilis</EM> colonies from the starch test 
		with ciprofloxacin. As a control they were also grown with 
		chloramphenicol, the resistance on the backbone of the integration 
		vector. Figure 7 shows the result for 3 different colonies (tubes 
		indicated with 1 - 3). The tubes marked with Cm is the control with 
		chloramphenicol, which shows growth for all three colonies. The tubes 
		marked with Cipro were grown with ciprofloxacin. Colonies 2 and 3 showed 
		growth, whereas colony 1 did not grow. Seems like the resistance 
		cassette is working. To further explore if the ciprofloxacin cassette 
		is functional in <EM>B. subtilis</EM>, a MIC value test was performed. <A href="https://2016.igem.org/Team:Groningen/Experiments">See link</A>.</P><FIGURE><FIGCAPTION>Figure 7.</FIGCAPTION></FIGURE></DIV></SECTION><SECTION id="nucA-in-pSB1C3" class="collapse-h3"><H3>RBS+nucA in pSB1C3</H3><P>One approach to delete the key from the genome of <EM>B. subtilis</EM> is making 
	use of the <EM>nucA</EM> BioBrick (<A href="http://parts.igem.org/Part:BBa_K729004">BBa_K729004</A>). The <EM>nucA</EM> is a nuclease gene which has 
	its origin in the genome of <EM>Staphylococcus aureus</EM>. It is capable of 
	digesting genetic material. The RBS is from BioBrick (<A href="http://parts.igem.org/Part:BBa_B0030">BBa_B0030</A>). The 
	combination of these two BioBricks is the first step of achieving our <EM>nucA</EM> 
	killswitch, which consist of the tetR repressible promoter (<A href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A>), the RBS 
	controlling the  <EM>nucA</EM> gene (<A href="http://parts.igem.org/Part:BBa_B0030">BBa_B0030</A>) and double terminator (<A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A>). </P><DIV class="collapse-h4"><H4 class="dna">Obtaining RBS (<A href="http://parts.igem.org/Part:BBa_B0030">BBa_B0030</A>) and <EM>nucA</EM> (<A href="http://parts.igem.org/Part:BBa_K729004">BBa_K729004</A>)</H4><H5>Experiment:</H5><P>01/08/16: The RBS (<A href="http://parts.igem.org/Part:BBa_B0030">BBa_B0030</A>) was obtained from the part distribution 2016, 
		the plasmid was transformed to <EM>E. coli</EM> Top10 using this
		<A href="/Team:Groningen/Protocols#transform-e-coli-standard">protocol</A>. The <EM>nucA</EM> (<A href="http://parts.igem.org/Part:BBa_K729004">BBa_K729004</A>) was requested from 
		the iGEM headquarters. Colonies from these transformations were 
		cultured in LB medium. Grown cultures of <EM>E. coli</EM> Top10 with RBS and <EM>nucA</EM> 
		were used to obtain a <A href="/Team:Groningen/Protocols#GS">glycerol stocks</A> and plasmid 
		isolation was performed <A href="/Team:Groningen/Protocols#Mini-prep-QIA">(QIAprep® Spin Miniprep Kit)</A>. The concentration of the 
		plasmids obtained was measured on Nanodrop. The plasmids were stored at 
		-20°C.</P></DIV><DIV class="collapse-h4"><H4 class="restrict">Restriction digestion</H4><H5>Experiment:</H5><P>28/09/16: On this day restriction digestion of RBS (BBa_B0030) and <EM>nucA</EM> 
		(BBa_K729004) was performed. The backbone RBS (BBa_B0030) was 
		digested with SpeI and PstI. The insert <EM>nucA</EM> was digested with XbaI and 
		PstI. </P><H5>RD mixture:</H5><P>20 μl RD assay was performed according to the following
		<A href="/Team:Groningen/Protocols#rd-neb">protocol</A>.</P><H5>DNA Electrophoresis:</H5><P>The digestion mixture of backbone RBS in pSB1C3 and insert <EM>nucA</EM> was 
		loaded on a gel to extract both parts. For detailed information on how 
		to prepare and run agarose gel see following
		<A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure1 DNA electrophoresis of: 1. <EM>nucA</EM> in pSB1C3  cut 
			with XbaI and PstI. The fragments are supposed to be <EM>nucA/</EM> 561 bp and 
			pSB1C3 2,019 bp. 2. RBS in pSB1C3 cut with SpeI and PstI one fragment 
			of 2,070 bp </FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>The digestion was successful because the band of the <EM>nucA</EM> could be 
		seen: 500 bp and RBS in pSB1C3 showed a band of 2,000 bp. </P><H5>Procedure after gel validation:</H5><P>Digested <EM>nucA</EM> and RBS in pSB1C3 were cut out from the gel and DNA 
		was extracted by <A href="/Team:Groningen/Protocols#Gel-extract-Jena">(Agarose Gel Extraction Kit – Jena Bioscience)</A>. </P></DIV><DIV class="collapse-h4"><H4 class="ligation">Ligation</H4><H5>Experiment:</H5><P>28/09/16: The cut <EM>nucA</EM> was ligated to the SpeI and PstI cut RBS in 
		pSB1C3. </P><H5>Ligation mixture:</H5><P>20 μl ligation assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#ligation">protocol</A>.</P></DIV><DIV class="collapse-h4"><H4 class="transform">Transformation</H4><H5>Experiment:</H5><P>28/09/16: The ligation mix was heat shock transformed to competent Top10 
		<EM>E. coli</EM> cells following the transformation <A href="/Team:Groningen/Protocols#transform-e-coli-standard">protocol</A>. 
		Cells were plated on 50 μg/ml chloramphenicol LB agar to select the 
		correct constructs. The next day colonies were picked to perform colony 
		PCR to find the correct constructs with the primers VF2 and VR. Find 
		primers <A href="/Team:Groningen/Labjournal/PrimerList">here</A>.</P><FIGURE><FIGCAPTION>Figure 2. Transformation of <EM>nucA</EM> in RBS in pSB1C3.</FIGCAPTION></FIGURE><H5>PCR mixture:</H5><P>25 μl PCR assay was performed according to the following 
		<A href="/Team:Groningen/Protocols#Colony-PCR">protocol</A>.</P><H5>PCR set-up:</H5><TABLE><TBODY><TR><TD>95ºC</TD><TD>2:00 min</TD></TR><TR><TD>95ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>60ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>30s</TD><TD>(30X)</TD></TR><TR><TD>72ºC</TD><TD>2:00 min</TD></TR><TR><TD colspan="3">10ºC on hold </TD></TR></TBODY></TABLE><H5>DNA electrophoresis:</H5><P>For detailed information on how to prepare and run agarose gel see 
		the following <A href="/Team:Groningen/Protocols#DNA-electrophoresis">protocol</A>.</P><FIGURE><FIGCAPTION>Figure 3. DNA electrophoresis of colony PCR for samples 
			1-7 of RBS+<EM>nucA</EM> in pSB1C3 with primers VF2, VR. Expected product 896 
			bp. </FIGCAPTION></FIGURE><H5>Conclusion:</H5><P>Transformation of RBS+<EM>nucA</EM> in pSB1C3 in <EM>E. coli</EM> Top10 appeared to be 
		successful. The samples 2, 5, 6 and 7 showed the right size of band. 
		These samples were used to obtain the plasmid from an overnight culture.</P></DIV><DIV class="collapse-h4"><H4 class="validation">Validation</H4><H5>Experiment:</H5><P>29/09/16: Grown cultures of <EM>E. coli</EM> Top10 with RBS+<EM>nucA</EM> in pSB1C3 were 
		used to obtain the <A href="/Team:Groningen/Protocols#GS">glycerol stocks</A> and plasmid isolation was 
		performed <A href="/Team:Groningen/Protocols#Mini-prep-QIA">(QIAprep® Spin Miniprep Kit)</A>. Firstly, concentration of the plasmids 
		obtained was measured on Nanodrop. Secondly, plasmids were sent for 
		sequencing and then stored at -20°C.</P><H5>Sequencing:</H5><P>Plasmids RBS+<EM>nucA</EM> in pSB1C3 from colonies 2, 5, and 7 were sent for 
		<A href="/Team:Groningen/Protocols#Sequencing">sequencing</A> with the sequencing primers VF2 and VR. (See 
		<A href="/Team:Groningen/Labjournal/PrimerList">primer list</A>).</P><H5>Conclusion:</H5><P>Sequencing results showed that the <EM>nucA</EM> was cloned in the pSB1C3 
		backbone but the RBS was missing and the suffix was also incorrect. See 
		Figure 4. The <EM>nucA</EM> (BBa_K729004) received from the iGEM headquarters was 
		sent for sequencing as well. It turned out that the part BBa_K729004 
		does have the <EM>nucA</EM> gene but the prefix and suffix are incorrect . See 
		sequencing results in Figure 5 to 7. </P><FIGURE><FIGCAPTION>Figure 4. Sequencing result of the RBS+<EM>nucA</EM> in pSB1C3 
			for sample 5, 7 and 9. <EM>nucA</EM> can be seen in pink. The prefix is 
			marked in red. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 5. Sequencing result of the BBa_K729004 aligned 
			with the <EM>nucA</EM> gene.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 6. Sequencing result of the BBa_K729004 aligned 
			with the prefix.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 7. Sequencing result of the BBa_K729004 aligned 
			with the suffix. </FIGCAPTION></FIGURE></DIV><DIV class="collapse-h4"><H4 class="further">Experiments</H4><P> We wanted to improve the <A href="http://parts.igem.org/Part:BBa_K729004">BBa_K729004</A> part by giving it the correct prefix and suffix. Unfortunately we did not succeed to complete this task. </P></DIV></SECTION></ARTICLE><ARTICLE id="labjournal-protocols" class="collapse-h2 collapse-conditional"><SECTION><H2>Protocols</H2><P>  All protocols which were used throughout our project can 
		be found below or downloaded <A href="https://static.igem.org/mediawiki/2016/b/be/T--Groningen--Protocols.pdf?action=raw&amp;ctype=application/pdf">here</A>. 
		 </P></SECTION><SECTION class="collapse-h3"><H3 id="transform-b-subtilis">Transformation of <EM> B. subtilis </EM></H3><H4>Day 1</H4><P><OL><LI> Streak out desired strain and incubate the plate overnight at 37°C. </LI></OL></P><H4>Day 2</H4><P><OL><LI>Pick a nice big colony and drop it in 2 ml of completed 1X MC medium (see below).</LI><LI>Grow at 37°C for 5 hours (or more if culture is not really turbid).</LI><LI>Mix 400 µl of culture with DNA (usually 1 µg) in fresh tube. </LI><LI>Grow for an additional 2 hours at 37°C.</LI><LI>Plate on selective antibiotic plates.</LI><LI>Incubate overnight at 37°C.</LI></OL></P><H4>Preparation of MC medium</H4><TABLE><TBODY><TR><TH><B> Completed 1X MC Medium </B></TH></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>1.8 ml</TD></TR><TR><TD>10X MC Medium</TD><TD>200 µl</TD></TR><TR><TD>MgSO<SUB>4</SUB></TD><TD>6.7 µl</TD></TR><TR><TD>1 % tryptophan (for trp- strains)</TD><TD>10 µl</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH><B> 10X MC medium(for 100 ml) </B></TH></TR><TR><TD> K<SUB>2</SUB>HPO<SUB>4</SUB>.3H<SUB>2</SUB>O </TD><TD> 14 g </TD></TR><TR><TD> KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD> 5.2 g </TD></TR><TR><TD> Glucose </TD><TD> 20 g </TD></TR><TR><TD> 300 mM Tri-Na citrate* </TD><TD> 10 ml </TD></TR><TR><TD> Ferric NH<SUB>4</SUB> citrate** </TD><TD> 1 ml </TD></TR><TR><TD> Casein hydrolysate </TD><TD> 1 g </TD></TR><TR><TD> K glutamate </TD><TD> 1 g </TD></TR></TBODY></TABLE><P> *300 mM Tri-Na citrate = 8.8 g in 100 ml H<SUB>2</SUB>O (wrap in aluminium foil) </P><P> **Ferric NH<SUB>4</SUB> citrate = 2.2 g in 100 ml H<SUB>2</SUB>O (wrap in aluminium foil) </P><P><UL><LI> Mix everything in 40-50 ml H<SUB>2</SUB>O. </LI><LI> Then adjust to 100 ml. </LI><LI> Filter sterilize.</LI><LI> Freeze at -20°C. </LI></UL></P></SECTION><SECTION class="collapse-h3"><H3 id="transform-e-coli-standard">Transformation of <EM> E. coli </EM> (Standard protocol)</H3><P><OL><LI> Add ligation mixture to the tube of competent cells. </LI><LI> Leave 30 min on ice. </LI><LI> Heat shock for 5 min at 37°C or for 45 sec at 42°C. </LI><LI> Add 300 µl of LB medium. </LI><LI> Place at 37°C for 30 – 60 min. Shake vigorously (220 rpm).</LI><LI> Plate 30 µl on 1 selection plate, and 300 µl on another.</LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="transform-e-coli-neb">Transformation of <EM> E. coli </EM> (NEB<SUP>®</SUP> 5-alpha Competent <EM> E. coli </EM> High Efficiency Transformation Protocol)</H3><P><OL><LI>Thaw a tube of NEB 5-alpha Competent <EM> E. coli </EM> cells on ice until the last ice crystals disappear. </LI><LI> Mix gently and carefully pipette 50 µl of cells into a transformation tube on ice.</LI><LI>Add 1 – 5 µl containing 1 pg – 100 ng of plasmid DNA to the cell mixture.</LI><LI>Carefully flick the tube 4 – 5 times to mix cells and DNA. Do not vortex.</LI><LI>Place the mixture on ice for 30 min. Do not mix.</LI><LI>Heat shock at exactly 42°C for exactly 30 sec. Do not mix.</LI><LI>Place on ice for 5 min. Do not mix.</LI><LI>Pipette 950 µl of room temperature SOC into the mixture.</LI><LI>Place at 37°C for 60 min. Shake vigorously (220 rpm) or rotate.</LI><LI>Warm selection plates to 37°C.</LI><LI>Mix the cells thoroughly by flicking the tube and inverting, then perform several 10-fold serial dilutions in SOC.</LI><LI>Spread 50 – 100 µl of each dilution onto a selection plate and incubate overnight at 37°C. </LI><LI> Alternatively, incubate at 30°C for 24 – 36 hours or 25°C for 48 hours.</LI></OL></P><P><B> Reagents supplied: </B></P><P><UL><LI> 6 x 0.2 ml/tube of chemically competent NEB 5-alpha Competent <EM> E. coli </EM> cells </LI><LI> 25 ml of SOC Outgrowth Medium </LI><LI> 0.025 ml of 50 pg/µl pUC19 Control DNA </LI></UL></P></SECTION><SECTION class="collapse-h3"><H3 id="competent-cells">Calcium Chloride Competent cells</H3><P><UL><LI> Label 28 sterile 1.5 ml tubes with name of <EM> E. coli </EM> strain </LI><LI> All solutions, glass pipettes, pipette tips, 50 ml tubes and 1.5 ml tubes must be <B> sterile </B> and <B> pre-cooled </B></LI><LI> All work must be done quickly and ON ICE (fresh ice, not half melted!) </LI></UL></P><H4>Day 1</H4><P><OL><LI> Streak <EM> E. coli </EM> strain on LB agar plate and incubate overnight at 37°C. </LI></OL></P><H4>Day 2</H4><P><OL><LI> Pick one colony from fresh agar plate and inoculate 3 ml of LB. </LI><LI> Incubate overnight at 37°C and 220 rpm. </LI></OL></P><H4>Day 3</H4><P><OL><LI> Inoculate LB medium with 1:100 overnight culture (i.e. 1 ml for 100 ml LB) in 500 ml flask.</LI><LI> Incubate at 37°C and 220 rpm. </LI><LI> Closely follow OD<SUB>600</SUB> (very important!) </LI><LI> At OD<SUB>600</SUB> = 0.35 (max. 0.40) harvest cells (ca. 1.5 – 2 hours TOP10 / ca. 2 – 3 hours INV110). </LI><LI> Put culture in two 50 ml pre-cooled centrifuge tubes, leave on ice for 10 min. </LI><LI> Centrifuge cultures at 2400 x g (4°C) for 4 min.</LI><LI> Discard supernatant (drain last drops onto a paper towel).</LI><LI> Resuspend each pellet in 20 ml cold 0.1 M MgCl<SUB>2</SUB>, using pre-cooled pipette and keeping 50 ml tubes IN ice.</LI><LI> Combine both cell cultures in one 50 ml tube.</LI><LI> Centrifuge culture at 2400 x g (4°C) for 4 min. Discard supernatant.</LI><LI> Resuspend pellet in 20 ml cold 50 mM CaCl<SUB>2</SUB>, using pre-cooled pipette and keeping tube IN ice.</LI><LI> Leave the culture on ice for 20 min (fresh ice, not half melted).</LI><LI> Centrifuge culture at 2400 x g (4°C) for 4 min. Discard supernatant. </LI><LI> Resuspend pellet in 1 ml cold 50 mM CaCl<SUB>2</SUB> + 15 % (v/v) glycerol.</LI><LI> Leave the culture on ice for 10 min (fresh ice, not half melted).</LI><LI> Use pre-cooled tips and pipet 50 μl aliquots into pre-cooled 1.5 ml tubes.</LI><LI> Freeze immediately in liquid nitrogen.</LI><LI> Store at -80°C.</LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="rd-fast">Restriction digestion (FastDigest enzymes)</H3><P><OL><LI> Add 2 µl of 10X FastDigest buffer.</LI><LI> Add 1 µl of restriction enzyme 1.</LI><LI> Add 1 µl of restriction enzyme 2.</LI><LI> Add 400 ng of DNA (possible to calculate from the concentration of the sample).</LI><LI> Add MiliQ water up to 20 µl.</LI><LI> Mix the solution by flicking the tube.</LI><LI> Shortly centrifuge for 2-3 sec.</LI><LI> Incubate at 37°C for 30 min.</LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="rd-neb">Restriction digestion (NEB enzymes)</H3><P><OL><LI> Add 5 µl of NEB buffer.</LI><LI> Add 1 µl of restriction enzyme 1.</LI><LI> Add 1 µl of restriction enzyme 2.</LI><LI> Add 1 µg of DNA (possible to calculate from the concentration of the sample).</LI><LI> Add MiliQ water up to 50 µl.</LI><LI> Mix the solution by flicking the tube.</LI><LI> Shortly centrifuge for 2-3 sec.</LI><LI> Incubate at 37°C for 60 min.</LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="rd-jena">Restriction digestion (Jena Bioscience enzymes)</H3><P><OL><LI> Add 5 µl of 10X Universal buffer.</LI><LI> Add 1 µl of restriction enzyme 1.</LI><LI> Add 1 µl of restriction enzyme 2.</LI><LI> Add 1 µg of DNA (possible to calculate from the concentration of the sample).</LI><LI> Add MiliQ water up to 50 µl.</LI><LI> Mix the solution by flicking the tube.</LI><LI> Shortly centrifuge for 2-3 sec.</LI><LI> Incubate at 37°C for 60 min.</LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="ligation">Ligation</H3><P><OL><LI> Add 2 µl of ligation buffer.</LI><LI> Add 50 - 100 ng of vector DNA (possible to calculate from the concentration of the sample).</LI><LI> Add X ng of insert DNA. X is calculated using the length of both vector and insert and the molar ratio desired (<A href="https://nebiocalculator.neb.com/#!/ligation">NEB calculator</A>).</LI><LI> Add MiliQ water up to 20 µl.</LI><LI> Add 1 µl of T4 ligase.</LI><LI> Incubate overnight at 16°C or for 30 – 60 min at room temperature.</LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="cell-cultures-e-coli">Cell cultures (<EM> E. coli </EM>)</H3><P><OL><LI> Pick a single colony from the plate or from glycerol stock.</LI><LI> Put the colony in a 15 ml sterile tube and add 4 – 5 ml of LB medium.</LI><LI> (If needed, add selective antibiotics to the medium before adding a single colony) </LI><LI> Incubate the tube overnight at 37°C and 220 rpm.</LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="cell-cultures-b-subtilis">Cell cultures (<EM> B. subtilis </EM>)</H3><P><OL><LI> Pick a single colony from the plate or from glycerol stock.</LI><LI> Put the colony in a 15 ml sterile tube and add 3 ml of LB medium.</LI><LI> (If needed, add selective antibiotics to the medium before adding a single colony) </LI><LI> Incubate the tube overnight at 37°C and 220 rpm.</LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="GS">Glycerol stock</H3><P><OL><LI> Take 800 µl from a freshly grown culture and put it in a 1.5 ml tube. </LI><LI> Add 200 µl of 80 % glycerol.</LI><LI> Mix by vortexing.</LI><LI> Store at -80°C. </LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="DNA-electrophoresis">DNA electrophoresis</H3><H4>Gel preparation (for 400 ml)</H4><P><OL><LI> Measure out 4 g of agarose and add 400 ml of 1X TBE buffer (for 1 % gel). </LI><LI> Autoclave the solution, and let it cool down to ∼ 55°C. Be careful, agar solidifies at 32 – 40°C. </LI><LI> Add 4 µl of DNA stain.</LI><LI> Store at 55 - 60°C for repetitive use.</LI></OL></P><H4>Gel pouring</H4><P><OL><LI> Pour the solution into the mold including the combs.</LI><LI> Let it polymerize for 15 min. </LI><LI> Remove the combs.</LI><LI> Put prepared agarose gel in the electrophoresis container containing 1X TBE.</LI></OL></P><H4>Electrophoresis</H4><P><OL><LI> Add 1/6 of total volume of 6X loading buffer to every DNA sample.</LI><LI> Pipet DNA samples containing 6X loading buffer and DNA ladder to the wells.</LI><LI> Run at 150 – 170V for 20 – 30 min (depends on the fragments).</LI></OL></P><H4>Preparation of 5X TBE buffer</H4><TABLE><TBODY><TR><TH><B> 5X TBE buffer (for 1000 ml) </B></TH></TR><TR><TD> Tris-base </TD><TD> 54 g </TD></TR><TR><TD> Boric acid </TD><TD> 27.5 g </TD></TR><TR><TD> 0.5 M EDTA (pH 8.0) </TD><TD> 20 ml </TD></TR></TBODY></TABLE><P><UL><LI> Adjust pH to 8.3 by HCl. </LI><LI> 5X TBE buffer has to be diluted to 1X TBE buffer before using  in electrophoresis </LI></UL></P></SECTION><SECTION class="collapse-h3"><H3 id="PCR-Taq">PCR (Taq Core Kit – Jena Bioscience)</H3><H4>50 µl PCR assay</H4><TABLE><TBODY><TR><TH><B> 50 µl PCR assay </B></TH></TR><TR><TD> 10X Taq Reaction buffer complete </TD><TD> 5 µl </TD></TR><TR><TD> dNTP Mix </TD><TD> 1 µl </TD></TR><TR><TD> MgCl<SUB>2</SUB>* </TD><TD> 1 µl</TD></TR><TR><TD> 10 µM Forward Primer </TD><TD> 1 µl </TD></TR><TR><TD> 10 µM Reverse Primer </TD><TD> 1 µl </TD></TR><TR><TD> Template DNA </TD><TD> 2 - 50 ng </TD></TR><TR><TD> Taq polymerase </TD><TD> 0.2 – 0.5 µl </TD></TR><TR><TD> MiliQ water </TD><TD> up to 50 µl </TD></TR></TBODY></TABLE><P> *Adding of Mg<SUP>2+</SUP> is recommended for most applications.</P><H4>Cycling conditions</H4><TABLE><TBODY><TR><TH><B> Cycling conditions </B></TH></TR><TR><TD> initial denaturation</TD><TD> 94°C </TD><TD> 2 min </TD><TD> 1x </TD></TR><TR><TD> denaturation </TD><TD> 94°C </TD><TD> 30 sec </TD><TD> 30x </TD></TR><TR><TD> annealing* </TD><TD> 45 – 68°C </TD><TD> 30 sec </TD><TD> 30x </TD></TR><TR><TD> elongation** </TD><TD> 72°C </TD><TD> 30 sec – 4 min </TD><TD> 30x </TD></TR><TR><TD> final elongation </TD><TD> 72°C </TD><TD> 2 min </TD><TD> 1x </TD></TR><TR><TD> hold </TD><TD> 4 – 10°C </TD></TR></TBODY></TABLE><P> *The annealing temperature depends on the melting temperature of the primers used. </P><P> **The elongation time depends on the length of the fragments to be amplified (1 min/kb). </P></SECTION><SECTION class="collapse-h3"><H3 id="PCR-Q5">PCR (Q5<SUP>®</SUP> High-Fidelity)</H3><H4>50 µl PCR assay</H4><TABLE><TBODY><TR><TH><B> 50 µl PCR assay </B></TH></TR><TR><TD> Q5 High-Fidelity 2X Master Mix </TD><TD> 25 µl </TD></TR><TR><TD> 10 µM Forward Primer </TD><TD> 2.5 µl </TD></TR><TR><TD> 10 µM Reverse Primer </TD><TD> 2.5 µl </TD></TR><TR><TD> Template DNA </TD><TD> variable </TD></TR><TR><TD> MiliQ water </TD><TD> up to 50 µl </TD></TR></TBODY></TABLE><H4>25 µl PCR assay</H4><TABLE><TBODY><TR><TH><B> 25 µl PCR assay </B></TH></TR><TR><TD> Q5 High-Fidelity 2X Master Mix </TD><TD> 12.5 µl </TD></TR><TR><TD> 10 µM Forward Primer </TD><TD> 1.25 µl </TD></TR><TR><TD> 10 µM Reverse Primer </TD><TD> 1.25 µl </TD></TR><TR><TD> Template DNA </TD><TD> variable </TD></TR><TR><TD> MiliQ water </TD><TD> up to 25 µl </TD></TR></TBODY></TABLE><H4>Cycling conditions</H4><TABLE><TBODY><TR><TH><B> Cycling conditions </B></TH></TR><TR><TD> initial denaturation</TD><TD> 98°C </TD><TD> 30 sec </TD><TD> 1x </TD></TR><TR><TD> denaturation </TD><TD> 98°C </TD><TD> 5 - 10 sec </TD><TD> 25 - 30x </TD></TR><TR><TD> annealing* </TD><TD> 50 – 72°C </TD><TD> 10 - 30 sec </TD><TD> 25 - 30x </TD></TR><TR><TD> elongation** </TD><TD> 72°C </TD><TD> 20 – 30 sec/kb </TD><TD> 25 - 30x </TD></TR><TR><TD> final elongation </TD><TD> 72°C </TD><TD> 2 min </TD><TD> 1x </TD></TR><TR><TD> hold </TD><TD> 4 – 10°C </TD></TR></TBODY></TABLE><P> *The annealing temperature depends on the melting temperature of the primers used. </P></SECTION><SECTION class="collapse-h3"><H3 id="Colony-PCR">Colony PCR</H3><P><OL><LI> Make the Master Mix using the materials specified in PCR protocols above.</LI><LI> Prepare a pre-glycerol stock. </LI></OL></P><P><UL><LI> Pick a colony from the plate and add it to the mixture of 80 µl of LB medium and 20 µl of 80 % glycerol. Vortex it.</LI></UL></P><LI> Add 2 µl of pre-glycerol stock to the PCR mixture.</LI><LI> Run the PCR using the programs specified above.</LI></SECTION><SECTION class="collapse-h3"><H3 id="Mini-prep-Jena">Plasmid mini-prep (Fast-n-Easy Plasmid Mini-Prep Kit - Jena Bioscience)</H3><P><OL><LI> Harvest the bacterial cell culture (1 – 3 ml) by centrifugation.</LI><LI> Resuspend pelleted bacterial in 300 µl Lysis Buffer by pipetting or vortexing for 1 min.</LI><LI> Add 300 µl of Neutralization Buffer (containing RNase A) to sample.</LI><LI> Mix it gently by inverting the tube 4 – 6 times (do not vortex).</LI><LI> Centrifuge at 10,000 g for 5 min at room temperature in a micro-centrifuge.</LI><LI> The color of the binding mixture should change to bright yellow indicating a pH of 7.5 required for optimal DNA binding.</LI><LI> Place a Binding Column into a 2 ml collection tube.</LI><LI> Add 100 µl of Activation buffer into the Binding Column.</LI><LI> Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.</LI><LI> Apply the supernatant from steps 3 – 5 into the activated Binding Column by decanting or pipetting.</LI><LI> Centrifuge at 10,000 g for 30 sec.</LI><LI> Discard the flow-through.</LI><LI> Place the DNA loaded Binding Column into the used 2 ml tube.</LI><LI> Apply 500 µl of Washing Buffer (containing Ethanol) to the Binding Column.</LI><LI> Centrifuge at 10,000 g for 30 sec and discard the flow-through. </LI><LI><U> Optional Secondary Washing</U> (Recommended only for DNA &gt;200 bp, if highly purified DNA is required).</LI></OL></P><P><UL><LI> Add 700 µl of Washing Buffer to the Binding Column. </LI><LI> Centrifuge at 10,000 g for 30 sec and discard the flow-through.</LI><LI> Centrifuge again for 2 min to remove residual Washing Buffer.</LI></UL></P><LI> Place the Binding Column into a clean 1.5 ml tube (not provided in the kit).</LI><LI> Add 30 – 50 µl Elution Buffer or dd-water to the center of the column membrane. </LI><LI> Incubate for 1 min at room temperature.</LI><LI> Centrifuge at 10,000 g for 1 min to elute DNA.</LI></SECTION><SECTION class="collapse-h3"><H3 id="Mini-prep-QIA">Plasmid mini-prep (QIAprep<SUP>®</SUP> Spin Miniprep Kit)</H3><P><OL><LI> Pellet 1 – 5 ml bacterial overnight culture by centrifugation at 8000 rpm (6800 x g) for 3 min at room temperature (15 – 25°C). </LI><LI> Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a micro-centrifuge tube.</LI><LI> Add 250 µl Buffer P2 and mix thoroughly by inverting the tube 4 – 6 times until the solution becomes clear. </LI><LI> Do not allow the lysis reaction to proceed for more than 5 min. If using LyseBlue reagent, the solution will turn blue.</LI><LI> Add 350 µl Buffer N3 and mix immediately and thoroughly by inverting the tube 4 – 6 times. </LI></OL></P><P><EM> If using LyseBlue reagent, the solution will turn colorless.</EM></P><LI> Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table top micro-centrifuge.</LI><LI> Apply the supernatant from step 5 to the QIAprep spin column by decanting or pipetting. </LI><LI> Centrifuge for 30 – 60 sec at 13,000 rpm (~17,900 x g) and discard the flow-through. </LI><LI> Recommended:
        </LI><LI> Wash the QIAprep spin column by adding 500 µl Buffer PB. </LI><LI> Centrifuge for 30 – 60 sec at 13,000 rpm (~17,900 x g) and discard the flow-through.</LI><LI> Note: This step is only required when using <EM> endA<SUP>+</SUP></EM><SUP> strains or other bacterial strains with high nuclease activity or carbohydrate content.</SUP></LI><LI> Wash the QIAprep spin column by adding 750 µl Buffer PE. </LI><LI> Centrifuge for 30 – 60 sec at 13,000 rpm (~17,900 x g) and discard the flow-through. </LI><LI> Centrifuge for 1 min at 13,000 rpm (~17,900 x g) to remove residual wash buffer.</LI><LI> Place the QIAprep column in a clean 1.5 ml micro-centrifuge tube. </LI><LI> To elute DNA, add 50 µl Buffer EB (10 mM Tris-Cl, pH 8.5) or water to the center of the QIAprep spin column. </LI><LI> Let it stand for 1 min, and centrifuge for 1 min at 13,000 rpm (~17,900 x g).</LI></SECTION><SECTION class="collapse-h3"><H3 id="Clean-up-Jena">DNA Clean-up (PCR Purification Kit – Jena Bioscience)</H3><P><B> For DNA fragment sizes in the range of 200 bp to 5 kbp: </B></P><P><UL><LI>Add 5 volumes of Binding Buffer to 1 volume of DNA sample and mix well. </LI><LI> For example, if the volume of your DNA sample is 50 µl, add 250 µl Binding Buffer.</LI></UL></P><P><B> For DNA fragment sizes smaller than 200 bp or larger than 5 kbp: </B></P><P><UL><LI> Add 3 volumes Binding Buffer and 2 volumes of Isopropanol to the PCR sample. </LI><LI> For example, if the volume of your DNA sample is 50 µl, add 150 µl Binding Buffer and 100 µl Isopropanol.</LI></UL></P><P><B> Protocol </B></P><P><OL><LI> Place a Spin Column into a 2 ml collection tube.</LI><LI> Add 100 µl of Activation Buffer into the Spin Column.</LI><LI> Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.</LI><LI> Apply the sample mixture from step A or B into the activated Spin Column.</LI><LI> Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.</LI><LI> Discard the flow-through.</LI><LI> Place the DNA loaded Spin Colum into the used 2 ml tube.</LI><LI> Apply 700 µl of Washing Buffer to the Spin Column.</LI><LI> Centrifuge at 10,000 g for 30 sec and discard the flow-through.</LI><LI><U> Optional Secondary Washing</U> (Recommended only for DNA &gt;200 bp, if highly purified DNA is required).</LI></OL></P><P><UL><LI> Add 700 µl of Washing Buffer to the Binding Column. </LI><LI> Centrifuge at 10,000 g for 30 sec and discard the flow-through.</LI><LI> Centrifuge again for 2 min to remove residual Washing Buffer.</LI></UL></P><LI> Place the Binding Column into a clean 1.5 ml tube (not provided in the kit).</LI><LI> Add 30 – 50 µl Elution Buffer or dd-water to the center of the column membrane. </LI><LI> Incubate for 1 min at room temperature.</LI><LI> Centrifuge at 10,000 g for 1 min to elute DNA.</LI></SECTION><SECTION class="collapse-h3"><H3 id="Clean-up-Nucleospin">DNA Clean-up (NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up)</H3><P><OL><LI> Mix 1 volume of sample with 2 volumes of Buffer NTI (e.g. mix 100 μl PCR reaction and 200 μl Buffer NTI)*.</LI><LI> Place a NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up Column into a Collection Tube (2 ml) and load up to 700 μl sample.</LI><LI> Centrifuge for 30 sec at 11,000 x g. </LI><LI> Discard flow-through and place the column back into the collection tube.</LI><LI> Load remaining sample if necessary and repeat the centrifugation step.</LI><LI> Add 700 μl Buffer NT3 to the NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up Column.</LI><LI> Centrifuge for 30 sec at 11,000 x g. </LI><LI> Discard flow-through and place the column back into the collection tube.</LI><LI> Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and improve A<SUB>260</SUB>/A<SUB>230</SUB> values.</LI><LI> Centrifuge for 1 min at 11,000 x g to remove Buffer NT3 completely.</LI><LI> Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube**.</LI><LI> Place the NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up Column into a new 1.5 ml microcentrifuge tube (not provided).</LI><LI> Add 15–30 μl Buffer NE***.</LI><LI> Incubate at room temperature (18–25°C) for 1 min. </LI><LI> Centrifuge for 1 min at 11,000 x g.</LI></OL></P><P><EM> *For very small sample volumes &lt; 30 μl adjust the volume of the reaction mixture to 50–100 μl with water. </EM></P><P><EM> **Residual ethanol from Buffer NT3 might inhibit enzymatic reactions. Total removal of ethanol can be achieved by incubating the columns for 2–5 min at 70°C prior to elution.</EM></P><P><EM> ***DNA recovery of larger fragments (&gt; 1000 bp) can be increased by multiple elution steps with fresh buffer, heating to 70°C and incubation for 5 min. </EM></P></SECTION><SECTION class="collapse-h3"><H3 id="Gel-extract-Jena">Gel extraction (Agarose Gel Extraction Kit – Jena Bioscience)</H3><P><OL><LI> Cut the area of gel containing the DNA fragment.</LI><LI> Transfer the excised gel to a clean 1.5 ml microtube.</LI><LI> Add 3 volumes of Extraction Buffer to 1 volume of the sliced gel. For example, add 300 µl Extraction Buffer to each 100 mg (approx. 100 µl) gel. For gels containing &gt;2.5 % agarose, add 6 volumes of Extraction Buffer per gel volume.</LI><LI> Incubate at 60 °C for 10 min with occasional mixing to ensure gel dissolution.</LI><LI> For DNA fragment sizes smaller than 200 bp or larger than 5 kbp and to enhance yield add 1 volume Isopropanol per gel volume to the dissolved gel and mix well.</LI><LI> Place a Spin Column into a 2 ml collection tube.</LI><LI> Add 100 µl of Activation Buffer into the Spin Column.</LI><LI> Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.</LI><LI> Apply the sample mixture from steps 3 (5), 4 into the activated Spin Column.</LI><LI> Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.</LI><LI> Discard the flow-through.</LI><LI> Place the DNA loaded Spin Colum into the used 2 ml tube.</LI><LI> Apply 700 µl of Washing Buffer to the Spin Column.</LI><LI> Centrifuge at 10,000 g for 30 sec and discard the flow-through.</LI><LI><U> Optional Secondary Washing</U> (Recommended only for DNA &gt;200 bp, if highly purified DNA is required).</LI></OL></P><P><UL><LI> Add 700 µl of Washing Buffer to the Binding Column. </LI><LI> Centrifuge at 10,000 g for 30 sec and discard the flow-through.</LI><LI> Centrifuge again for 2 min to remove residual Washing Buffer.</LI></UL></P><LI> Place the Spin Column into a new 1.5 ml microcentrifuge tube (not provided).</LI><LI> Add 30 – 50 µl Elution Buffer or dd-water to the center of the column membrane.</LI><LI> Incubate for 1 min at room temperature.</LI><LI> Centrifuge at 10,000 g for 1 min to elute DNA.</LI></SECTION><SECTION class="collapse-h3"><H3 id="Gel-extract-Nucleospin">Gel extraction (NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up)</H3><P><OL><LI> Take a clean scalpel to excise the DNA fragment from an agarose gel. Remove all excess agarose.</LI><LI> Determine the weight of the gel slice and transfer it to a clean tube.</LI><LI> For each 100 mg of agarose gel &lt; 2 % add 200 μl Buffer NTI. (For gels containing &gt; 2 % agarose, double the volume of Buffer NTI).</LI><LI> Incubate sample for 5–10 min at 50°C. Vortex the sample briefly every 2–3 min until the gel slice is completely dissolved.</LI><LI> Place a NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up Column into a Collection Tube (2 ml) and load up to 700 μl sample.</LI><LI> Centrifuge for 30 sec at 11,000 x g. </LI><LI> Discard flow-through and place the column back into the collection tube.</LI><LI> Load remaining sample if necessary and repeat the centrifugation step.</LI><LI> Add 700 μl Buffer NT3 to the NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up Column. </LI><LI> Centrifuge for 30 sec at 11,000 x g. </LI><LI> Discard flow-through and place the column back into the collection tube.</LI><LI> Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and low A<SUB>260</SUB>/A<SUB>230</SUB> values.</LI><LI> Centrifuge for 1 min at 11,000 x g to remove Buffer NT3 completely. </LI><LI> Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube*.</LI><LI> Place the NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up Column into a new 1.5 ml microcentrifuge tube (not provided). </LI><LI> Add 15–30 μl Buffer NE**. </LI><LI> Incubate at room temperature (18–25°C) for 1 min. </LI><LI> Centrifuge for 1 min at 11,000 x g.</LI></OL></P><P><EM> *Residual ethanol from Buffer NT3 might inhibit enzymatic reactions. Total removal of ethanol can be achieved by incubating the columns for 2–5 min at 70°C prior to elution.</EM></P><P><EM> **DNA recovery of larger fragments (&gt; 1000 bp) can be increased by multiple elution steps with fresh buffer, heating to 70°C and incubation for 5 min. </EM></P></SECTION><SECTION class="collapse-h3"><H3 id="Spores-B-sub">Preparation of the spore stock of <EM> B. subtilis </EM></H3><P><OL><LI> Inoculate 20 ml of LB medium with cells from a fresh colony of <EM> B. subtilis </EM>. Always use flasks that comprise at least 5x times the volume of media used, and always use lids that are able to allow air passage.</LI><LI> Culture for 6 – 8 hours at 37°C with shaking at 200 rpm. <EM> B. subtilis </EM> grows best at 37°C and has a doubling time of 30 min.</LI><LI> Dilute 1:200 into 1000 ml DSM medium in a sterile 1 liter flask and grow at 37°C with shaking at 200 rpm. Add 5 ml of the culture to 1000 ml medium.</LI></OL></P></SECTION> Check samples for spores daily using phase contrast microscopy (see Phase Contrast Microscopy Protocol). Optional use of Gram stain (see <A href="http://www.uphs.upenn.edu/bugdrug/antibiotic_manual/Gram2.htm">Gram staining protocol</A>) to distinguish between vegetative cells (purple) and spores (transparent).<LI> After 2 – 3 days &gt;90% of the population should have sporulated.</LI><LI> Pellet cells at 9000 rpm for 20 min (keeping temperatures low), otherwise for small quantities a benchtop centrifuge will suffice.</LI><LI> Wash spores with ice-cold water 8 – 10 times to remove residual nutrients and lyse remaining vegetative cells. (Resuspend pellets in water, centrifuge, discard supernatant, repeat).</LI><LI> Store spores at -20°C for long-term storage or at 4°C, with weekly changes of water. </LI><H4>Difco Sporulation Medium (DSM)</H4><TABLE><TBODY><TR><TH><B> DSM (for 1000 ml) </B></TH></TR><TR><TD> Bacto nutrient broth (Difco) </TD><TD> 8 g </TD></TR><TR><TD> 10 % (w/v) KCl </TD><TD> 10 ml </TD></TR><TR><TD> 1.2 % (w/v) MgSO<SUB>4</SUB> . 7H<SUB>2</SUB>O </TD><TD> 10 ml </TD></TR><TR><TD> 1 M NaOH </TD><TD> 1.5 ml </TD></TR></TBODY></TABLE><P><UL><LI> Bring pH to 7.6. </LI><LI> Filter sterilize. </LI><LI> Add the following sterile (autoclaved) component solutions to 1000 ml of the cooled DSM medium prior to use:</LI></UL></P><TABLE><TBODY><TR><TD> 1 M Ca(NO<SUB>3</SUB>)<SUB>2</SUB></TD><TD> 1 ml </TD></TR><TR><TD> 0.01 M MnCl<SUB>2</SUB></TD><TD> 1 ml </TD></TR><TR><TD> 1 mM FeSO<SUB>4</SUB></TD><TD> 1 ml </TD></TR></TBODY></TABLE></ARTICLE><SECTION class="collapse-h3"><H3 id="Starch-test">Integration check: Starch test</H3><P><OL><LI> Inoculation: Use a fresh (16- to 18-hour) pure culture of test bacteria as an inoculation source. Pick a single isolated colony and either single streak or spot inoculate the surface of the agar medium. </LI><LI> Incubation: Incubate plates overnight at 37°C. </LI><LI> Starch Hydrolysis Test: After proper inoculation and incubation, flood the surface of the agar with Gram’s iodine solution. Record results immediately as the blue color formed with starch may fade giving a false-positive result of absence of starch. </LI><OL type="a"><LI> Appearance of a clear zone surrounding the bacterial growth indicates starch hydrolysis (+) by the organism due to its production of the extracellular enzymes. The clear zone will start out yellow (from the iodine) and becomes progressively lighter yellow and then clear -&gt; indicates wrong clones.</LI><LI> A blue/black or purple zone surrounding the growth indicates that starch is present and has not been hydrolyzed (-) and the organism did not produce the extracellular enzymes -&gt; indicates right clones.</LI></OL></OL></P><H4>Starch plates</H4><TABLE><TBODY><TR><TH> Solution of 1000 ml </TH></TR><TR><TD> Beef extract </TD><TD> 3 g </TD></TR><TR><TD> Soluble starch </TD><TD> 10 g </TD></TR><TR><TD> Agar </TD><TD> 12 g </TD></TR><TR><TD> Distilled water </TD><TD> 1000 ml </TD></TR></TBODY></TABLE><P><UL><LI> Suspend the first three ingredients in 1000 ml of distilled water. Mix thoroughly. </LI><LI> Heat with frequent agitation and carefully bring to just boiling (excessive boiling may hydrolyze starch).</LI><LI> Autoclave.</LI><LI> After sterilization pour the melted medium into sterilized petri plates (approximately 20-30 ml per plate) and let it solidify before use.</LI></UL></P></SECTION><SECTION class="collapse-h3"><H3 id="gBlocks">IDT gBlocks<SUP>®</SUP></H3><H4>Resuspending gBlocks Gene Fragment</H4><P><OL><LI> Centrifuge the tube for 3 – 5 sec at a minimum of 3000 x g to ensure the material is in the bottom of the tube.</LI><LI> Add 1X TE to reach a final concentration of 10 ng/µl.</LI><LI> Vortex briefly.</LI><LI> Incubate at 50°C for 20 min.</LI><LI> Briefly vortex and centrifuge.</LI></OL></P><H4>Amplifying gBlocks Gene Fragment</H4><P><OL><LI> For gBlocks Gene Fragments ≤ 1kb, amplification can be performed using a high fidelity polymerase. </LI><LI> To avoid sequence mutations due to amplification errors, limit cycle to 12 or fewer.</LI><LI> For gBlocks Gene Fragments &gt; 1kb, we do not recommend amplification.</LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="Microscopy">Time-lapse microscopy/Phase-contrast microscopy</H3><H4>Preparation of the 1.5% agarose</H4><P><OL><LI> Measure out 150 mg of agarose (for 1.5 % agarose) and add either 10 ml of LB medium (time-lapse) or 1X TBE buffer (phase-contrast images).</LI><LI> Dissolve the agarose.</LI><LI> Store at 55 - 60°C for repetitive use.</LI></OL></P><H4>Preparation of the cell cultures</H4><P><OL><LI> Pick a single colony from the plate or from glycerol stock.</LI><LI> Put the colony in a 15 ml sterile tube and add 3 ml of LB medium.</LI><LI> (If needed, add selective antibiotics to the medium before adding a single colony).</LI><LI> Incubate the tube overnight at 37°C and 220 rpm.</LI></OL></P><H4>Preparation of the microscope sample [<A href="http://www.jove.com/video/3145/live-cell-imaging-bacillus-subtilis-streptococcus-pneumoniae-using">1</A>]</H4><P><OL><LI> Clean two microscope glass slides with 70 % ethanol and water.</LI><LI> Take a gene frame and carefully remove one of the plastic foils from the gene frame without causing disassembly of the plastic cover on the other side of the gene frame.</LI><LI> Attach the gene frame in the middle of one of the glass slides by first facilitating contact on just one side, followed by guided attachment of the remaining gene frame with a fingernail. Prevent air bubbles while attaching the gene frame to the glass slide.</LI><LI> Transfer 500 μl of the warm agarose-LB or agarose-TBE in the middle of the gene frame. Make sure the whole area including (the borders) is fully covered.</LI><LI> The following steps have to be carried out quickly to prevent excessive drying of the agarose-LB or agarose-TBE.</LI><LI> Place the second glass slide on the agarose-LB or agarose-TBE filled gene frame. Try to avoid air bubbles. Place the sandwiched slides in a horizontal position for 45 min at 4°C in the refrigerator to allow the agarose-LB or agarose-TBE to solidify sufficiently.</LI><LI> Carefully slide off the upper glass slide. Use a razor blade to cut out agar strips of ~5 mm width within the gene frame, on which the cells will be grown.</LI><LI> Carefully remove the second and final plastic cover from the gene frame to expose the sticky side of the gene frame.</LI><LI> Load single cells on the solid medium without touching it with the pipet tip. Use 2.5 μl for a whole strip, or 1 μl for a small square. Always start on top of the agarose pad and allow the liquid to disperse equally on its assigned growth area by turning the slide up and down. The slide is ready, as soon as the edges of the liquid become corrugated and movement of the liquid is no longer visible when turning the slide.</LI><LI> Place a clean microscope slide cover slip on the gene frame from one side to the other (avoid air bubbles). Assure complete attachment by applying pressure on the cover slip along the gene frame with your fingernail. If the cover slip is placed on the cells without allowing them to dry long enough, cells tend to grow on top of each other during the experiment. Also be careful not to wait too long before applying the cover slip, since the agarose will then be too dry.</LI></OL></P><P> Reference: </P><P><A href="http://www.jove.com/video/3145/live-cell-imaging-bacillus-subtilis-streptococcus-pneumoniae-using">Jong, I. G. De, Beilharz, K., Kuipers, O. P., &amp; Veening, J.-W. (2011). Live Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy, (July), 1–6. http://doi.org/10.3791/3145 </A></P></SECTION><SECTION class="collapse-h3"><H3 id="Sequencing">Sequencing (Macrogen)</H3><P><OL><LI> Template DNA of 5 μl with either of following concentrations: </LI></OL></P><P><OL type="a"><LI> 50 ng/μl of PCR product. </LI><LI> 100 ng/μl of plasmid DNA. </LI></OL></P><P><OL><LI> Add 5 μl of primer with either of following concentrations: </LI></OL></P><P><OL type="a"><LI> 50 ng/μl of PCR product. </LI><LI> 100 ng/μl of plasmid DNA. </LI></OL></P></SECTION><SECTION class="collapse-h3"><H3 id="Photo-measurement">Photometric measurement of the OD<SUB>600</SUB> and mRFP fluorescence</H3><H4>Day 1</H4><P><OL><LI> Grow <EM>E. coli</EM> in 3 ml LB with 35 μg/ml chloramphenicol at 37°C, 220 rpm for 16 hours.</LI></OL></P><H4>Day 2</H4><P><B> Preparation of the 96 well plate </B></P><P><OL><LI> Prepare a transparent 96 well plate with a total volume of 250 μl. </LI><LI> Provide 175 μl of LB and 25 μl of the appropriate concentration of desired compound.</LI><LI> Dilute cells to OD<SUB>600</SUB> of 0.5 with LB and add 50 μl to each well.</LI></OL></P><P><B> Photometric measurement (Plate reader – Varioskan LUX Reader, Thermo Fischer) </B></P><P><OL><LI> To measure the fluorescence of the mRFP set the excitation and emission on 584 nm and 607 nm, respectively, with a bandwidth of 5 nm. </LI><LI> Measure the absorbance at 600 nm.</LI><LI> Measure every 20 min at 37°C with the plate reader.</LI></OL></P></SECTION></ARTICLE><ARTICLE id="primer-list" class="collapse-h2 collapse-conditional"><SECTION><H2>Primer List</H2><TABLE><TBODY><TR><TH>No.</TH><TH>Primer name</TH><TH>Sequence 5'-3'</TH><TH>T<SUB>m</SUB>[°C]</TH></TR><TR><TD>1</TD><TD>F-key sequence</TD><TD>CGGAATTCGCGGCCGCTTCTAGAGG</TD><TD>67.9</TD></TR><TR><TD>2</TD><TD>R-key sequence</TD><TD>GCCTGCAGCGGCCGCTACTAGTACC</TD><TD>68.0</TD></TR><TR><TD>3</TD><TD>F-message sequence</TD><TD>GCGTCGACGACCAAGCCTGCAAAAAC</TD><TD>65.4</TD></TR><TR><TD>4</TD><TD>R-message sequence</TD><TD>GCAAGCTTGTCGGTGGGTGCAATGC</TD><TD>65.2</TD></TR><TR><TD>5</TD><TD>F-qnrs1 e.coli</TD><TD>GCAAGCTTGAATTCGCGGCCGCTTC</TD><TD>67.6</TD></TR><TR><TD>6</TD><TD>R-qnrs1 e.coli</TD><TD>GCGGTACCCTGCAGCGGCCGCTACTAG</TD><TD>70.9</TD></TR><TR><TD>7</TD><TD>F-qnrs1 optimized</TD><TD>GCAAGCTTGAATTCGCGGCCGCTTC</TD><TD>67.6</TD></TR><TR><TD>8</TD><TD>R-qnrs1 optimized</TD><TD>GCGGTACCCTGCAGCGGCCGCTACTAG</TD><TD>70.9</TD></TR><TR><TD>9</TD><TD>amyE-prefix</TD><TD>GCG<U>GAATTCGCGGCCGCTTCTAG</U>ATGTTTGCAAAACGATTCAAAAC</TD><TD>73.6</TD></TR><TR><TD>10</TD><TD>amyE-suffix</TD><TD>CG<U>TACTAGTAGCGGCCGCTGCAG</U>TCAATGGGGAAGAGAACCGCTTAAGC</TD><TD>77.1</TD></TR><TR><TD>11</TD><TD>F-gfp insert</TD><TD>GCAGATCTAAATTGAATTCAACGCTCGAATGC</TD><TD>65.0</TD></TR><TR><TD>12</TD><TD>R-gfp insert</TD><TD>GCGCTAGCATTACGCCAAGCTTGAATCTTGCTTG</TD><TD>71.2</TD></TR><TR><TD>13</TD><TD>F message sequencing</TD><TD>GCGCGTACGATCTTTCAGCCGACTC</TD><TD>65.9</TD></TR><TR><TD>14</TD><TD>R message sequencing</TD><TD>GCGCCGCGTTTCGGTGATGAAGAT</TD><TD>65.3</TD></TR><TR><TD>15</TD><TD>F key only amplify</TD><TD>GCGGACCAGCTCATGATTCTCAC</TD><TD>60.9</TD></TR><TR><TD>16</TD><TD>R key only amplify</TD><TD>GCGAGGAGGCTTACTTGTCTGCTTTCTTC</TD><TD>65.9</TD></TR><TR><TD>17</TD><TD>amyE int check</TD><TD>CTCTGCCAAGTTGTTTTGATAGAGTG</TD><TD>59.2</TD></TR><TR><TD>18</TD><TD>key only + prefix</TD><TD><U>GAATTCGCGGCCGCTTCTAGAG</U>GACCAAGCCTGCAAAAC</TD><TD>73.6</TD></TR><TR><TD>19</TD><TD>key only + suffix</TD><TD><U>CTGCAGCGGCCGCTACTAGTA</U>GTCGGTGGGTGCTATGGAGCGACAGAA</TD><TD>78.3</TD></TR><TR><TD>20</TD><TD>message prefix</TD><TD><U>GAATTCGCGGCCGCTTCTAGAG</U>GACCAAGCCTGCAAAAACAAAG</TD><TD>74.4</TD></TR><TR><TD>21</TD><TD>message suffix</TD><TD><U>CTGCAGCGGCCGCTACTAGTA</U>GTCGGTGGGTGCAATGCTTC</TD><TD>76.0</TD></TR><TR><TD>22</TD><TD>pATPI+prefix</TD><TD>GC<U>GAATTCGCGGCCGCTTCTAGAG</U>TATAGGTGAAAATGTGAACATTC</TD><TD>72.9</TD></TR><TR><TD>23</TD><TD>pATPI+suffix</TD><TD><U>CTGCAGCGGCCGCTACTAGTA</U>CAATTATCTGTCTCCTGATGAA</TD><TD>73.0</TD></TR><TR><TD>24</TD><TD>prefix sfGFP(Sp)</TD><TD><U>GAATTCGCGGCCGCTTCTAG</U>ATGTCAAAAGGAGAAGAACTTTTTAC</TD><TD>71.3</TD></TR><TR><TD>25</TD><TD>suffix sfGFP(Sp)</TD><TD><U>CTGCAGCGGCCGCTACTAGTA</U>CATTATTATTTATAAAGTTCGTCCATAC</TD><TD>71.0</TD></TR><TR><TD>26</TD><TD>VF2</TD><TD>TGCCACCTGACGTCTAAGAA</TD><TD>55.4</TD></TR><TR><TD>27</TD><TD>VR</TD><TD>ATTACCGCCTTTGAGTGAGC</TD><TD>55.4</TD></TR><TR><TD>28</TD><TD>nucA+prefix</TD><TD><U>GAATTCGCGGCCGCTTCTAG</U>ATGAAAAAGATTTGGCTGGCGCTGGCTG</TD><TD>77.4</TD></TR><TR><TD>29</TD><TD>nucA+suffix</TD><TD><U>CTGCAGCGGCCGCTACTAGTA</U>TTATTGACCTGAATCAGCGTTGTC</TD><TD>74.2</TD></TR><TR><TD>30</TD><TD>Gb_insert_F2</TD><TD>GTACATCTTTGTAACTTTATTATACGACTGGGCCTTTCGTTTTATCTGTTG</TD><TD>72.0</TD></TR></TBODY></TABLE><P>*prefix and suffix are underlined</P></SECTION></ARTICLE><SECTION><H2>A day in the life of an iGEMer</H2></SECTION></DIV><DIV id="page-footer" class="page-footer"><SECTION><H6>Team</H6><UL><LI><A href="/Team:Groningen/Team">Members</A></LI><LI><A href="/Team:Groningen/Supervisors">Supervisors</A></LI><LI><A href="/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A href="/Team:Groningen/Achievements">Achievements</A></LI><LI><A href="/Team:Groningen/Contact">Contact</A></LI><LI><A href="/Team:Groningen/Jamboree">Jamboree</A></LI></UL></SECTION><SECTION><H6>Project</H6><UL><LI><A href="/Team:Groningen/Tour">Tour</A></LI><LI><A href="/Team:Groningen/Description">Description</A></LI><LI><A href="/Team:Groningen/Design">Design</A></LI><LI><A href="/Team:Groningen/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Groningen/Summary">Summary</A></LI><LI><A href="/Team:Groningen/Applications">Applications</A></LI><LI><A href="/Team:Groningen/Results">Results</A></LI><LI><A href="/Team:Groningen/FuturePerspectives">Future Perspectives</A></LI></UL></SECTION><SECTION class="double"><H6>Biology</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Experiments">Experiments</A></LI><LI><A href="/Team:Groningen/Safety">Safety</A></LI><LI><A href="/Team:Groningen/Decoy">Decoy experiment</A></LI><LI><A href="/Team:Groningen/PhotoswitchableAntibiotics">Photoswitchable antibiotics</A></LI><LI><A href="/Team:Groningen/BrickCharacter">BioBrick Characterization</A></LI><LI><A href="/Team:Groningen/KeyDeletion">Key deletion</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/Proof">Proof of concept</A></LI><LI><A href="/Team:Groningen/Labjournal">Lab journal</A></LI><LI><A href="/Team:Groningen/PlasmidConstruction">Plasmid construction</A></LI><LI><A href="/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="/Team:Groningen/Labjournal/PrimerList">Primer list</A></LI><LI><A href="/Team:Groningen/Notebook">Notebook</A></LI></UL></DIV></SECTION><SECTION><H6>Modelling</H6><UL><LI><A href="/Team:Groningen/Model">Modelling</A></LI><LI><A href="/Team:Groningen/MutationRates">Mutation Rates</A></LI><LI><A href="/Team:Groningen/RandomMutation">Random Mutation</A></LI><LI><A href="/Team:Groningen/AIModel">AI Agent model</A></LI><LI><A href="/Team:Groningen/DecodingFidelity">Decoding Fidelity</A></LI><LI><A href="/Team:Groningen/SpiroModel">Optimal Strain</A></LI></UL></SECTION><SECTION><H6>Computing</H6><UL><LI><A href="/Team:Groningen/Software">Software</A></LI><LI><A href="/Team:Groningen/Encoding">Encoder</A></LI><LI><A href="/Team:Groningen/Decoding">Decoder</A></LI><LI><A href="/Team:Groningen/FullCoding">Full Coding</A></LI></UL></SECTION><SECTION class="double"><H6>Human Practices</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Groningen/HP/Outreach">Outreach</A></LI><LI><A href="/Team:Groningen/HP/Social">Social</A></LI><LI><A href="/Team:Groningen/HP/Legal">Legal</A></LI><LI><A href="/Team:Groningen/HP/FutureScenarios">Future Scenarios</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/HP/Education">Education</A></LI><LI><A href="/Team:Groningen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="/Team:Groningen/Engagement">Engagement</A></LI><LI><A href="/Team:Groningen/HP/Gold">Gold</A></LI><LI><A href="/Team:Groningen/HP/Silver">Silver</A></LI></UL></DIV></SECTION><SECTION><H6>Support</H6><UL><LI><A href="/Team:Groningen/Sponsors">Sponsors</A></LI><LI><A href="/Team:Groningen/Attributions">Attributions</A></LI></UL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>