Team:Groningen/Protocols
﻿﻿
CryptoGE®M
Team
Members Supervisors Collaborations Achievements Contact
Project
Tour Description Design Results Demonstrate Applications Future Notebook
Biology
Experiments Parts Safety Lab journal
Computing
Modelling Software Encoding Decoding
Human Practices
Human Practices Gold Silver
Support
Sponsors Attributions
Protocols
All protocols which were used throughout our project can    be found below or downloaded here .
Transformation of B. subtilis
Day 1
Streak out desired strain and incubate the plate overnight at 37°C.
Day 2
Pick a nice big colony and drop it in 2 ml of completed 1X MC medium (see below).
Grow at 37°C for 5 hours (or more if culture is not really turbid).
Mix 400 µl of culture with DNA (usually 1 µg) in fresh tube.
Grow for an additional 2 hours at 37°C.
Plate on selective antibiotic plates.
Incubate overnight at 37°C.
Preparation of MC medium
Completed 1X MC Medium
H2O
1.8 ml
10X MC Medium
200 µl
MgSO4
6.7 µl
1 % tryptophan (for trp- strains)
10 µl
10X MC medium(for 100 ml)
K2HPO4.3H2O
14 g
KH2PO4
5.2 g
Glucose
20 g
300 mM Tri-Na citrate*
10 ml
Ferric NH4 citrate**
1 ml
Casein hydrolysate
1 g
K glutamate
1 g
*300 mM Tri-Na citrate = 8.8 g in 100 ml H2O (wrap in aluminium foil)
**Ferric NH4 citrate = 2.2 g in 100 ml H2O (wrap in aluminium foil)
Mix everything in 40-50 ml H2O.
Then adjust to 100 ml.
Filter sterilize.
Freeze at -20°C.
Transformation of E. coli (Standard protocol)
Add ligation mixture to the tube of competent cells.
Leave 30 min on ice.
Heat shock for 5 min at 37°C or for 45 sec at 42°C.
Add 300 µl of LB medium.
Place at 37°C for 30 – 60 min. Shake vigorously (220 rpm).
Plate 30 µl on 1 selection plate, and 300 µl on another.
Transformation of E. coli (NEB® 5-alpha Competent E. coli High Efficiency Transformation Protocol)
Thaw a tube of NEB 5-alpha Competent E. coli cells on ice until the last ice crystals disappear.
Mix gently and carefully pipette 50 µl of cells into a transformation tube on ice.
Add 1 – 5 µl containing 1 pg – 100 ng of plasmid DNA to the cell mixture.
Carefully flick the tube 4 – 5 times to mix cells and DNA. Do not vortex.
Place the mixture on ice for 30 min. Do not mix.
Heat shock at exactly 42°C for exactly 30 sec. Do not mix.
Place on ice for 5 min. Do not mix.
Pipette 950 µl of room temperature SOC into the mixture.
Place at 37°C for 60 min. Shake vigorously (220 rpm) or rotate.
Warm selection plates to 37°C.
Mix the cells thoroughly by flicking the tube and inverting, then perform several 10-fold serial dilutions in SOC.
Spread 50 – 100 µl of each dilution onto a selection plate and incubate overnight at 37°C.
Alternatively, incubate at 30°C for 24 – 36 hours or 25°C for 48 hours.
Reagents supplied:
6 x 0.2 ml/tube of chemically competent NEB 5-alpha Competent E. coli cells
25 ml of SOC Outgrowth Medium
0.025 ml of 50 pg/µl pUC19 Control DNA
Calcium Chloride Competent cells
Label 28 sterile 1.5 ml tubes with name of E. coli strain
All solutions, glass pipettes, pipette tips, 50 ml tubes and 1.5 ml tubes must be sterile and pre-cooled
All work must be done quickly and ON ICE (fresh ice, not half melted!)
Day 1
Streak E. coli strain on LB agar plate and incubate overnight at 37°C.
Day 2
Pick one colony from fresh agar plate and inoculate 3 ml of LB.
Incubate overnight at 37°C and 220 rpm.
Day 3
Inoculate LB medium with 1:100 overnight culture (i.e. 1 ml for 100 ml LB) in 500 ml flask.
Incubate at 37°C and 220 rpm.
Closely follow OD600 (very important!)
At OD600 = 0.35 (max. 0.40) harvest cells (ca. 1.5 – 2 hours TOP10 / ca. 2 – 3 hours INV110).
Put culture in two 50 ml pre-cooled centrifuge tubes, leave on ice for 10 min.
Centrifuge cultures at 2400 x g (4°C) for 4 min.
Discard supernatant (drain last drops onto a paper towel).
Resuspend each pellet in 20 ml cold 0.1 M MgCl2, using pre-cooled pipette and keeping 50 ml tubes IN ice.
Combine both cell cultures in one 50 ml tube.
Centrifuge culture at 2400 x g (4°C) for 4 min. Discard supernatant.
Resuspend pellet in 20 ml cold 50 mM CaCl2, using pre-cooled pipette and keeping tube IN ice.
Leave the culture on ice for 20 min (fresh ice, not half melted).
Centrifuge culture at 2400 x g (4°C) for 4 min. Discard supernatant.
Resuspend pellet in 1 ml cold 50 mM CaCl2 + 15 % (v/v) glycerol.
Leave the culture on ice for 10 min (fresh ice, not half melted).
Use pre-cooled tips and pipet 50 μl aliquots into pre-cooled 1.5 ml tubes.
Freeze immediately in liquid nitrogen.
Store at -80°C.
Restriction digestion (FastDigest enzymes)
Add 2 µl of 10X FastDigest buffer.
Add 1 µl of restriction enzyme 1.
Add 1 µl of restriction enzyme 2.
Add 400 ng of DNA (possible to calculate from the concentration of the sample).
Add MiliQ water up to 20 µl.
Mix the solution by flicking the tube.
Shortly centrifuge for 2-3 sec.
Incubate at 37°C for 30 min.
Restriction digestion (NEB enzymes)
Add 5 µl of NEB buffer.
Add 1 µl of restriction enzyme 1.
Add 1 µl of restriction enzyme 2.
Add 1 µg of DNA (possible to calculate from the concentration of the sample).
Add MiliQ water up to 50 µl.
Mix the solution by flicking the tube.
Shortly centrifuge for 2-3 sec.
Incubate at 37°C for 60 min.
Restriction digestion (Jena Bioscience enzymes)
Add 5 µl of 10X Universal buffer.
Add 1 µl of restriction enzyme 1.
Add 1 µl of restriction enzyme 2.
Add 1 µg of DNA (possible to calculate from the concentration of the sample).
Add MiliQ water up to 50 µl.
Mix the solution by flicking the tube.
Shortly centrifuge for 2-3 sec.
Incubate at 37°C for 60 min.
Ligation
Add 2 µl of ligation buffer.
Add 50 - 100 ng of vector DNA (possible to calculate from the concentration of the sample).
Add X ng of insert DNA. X is calculated using the length of both vector and insert and the molar ratio desired ( NEB calculator ).
Add MiliQ water up to 20 µl.
Add 1 µl of T4 ligase.
Incubate overnight at 16°C or for 30 – 60 min at room temperature.
Cell cultures ( E. coli )
Pick a single colony from the plate or from glycerol stock.
Put the colony in a 15 ml sterile tube and add 4 – 5 ml of LB medium.
(If needed, add selective antibiotics to the medium before adding a single colony)
Incubate the tube overnight at 37°C and 220 rpm.
Cell cultures ( B. subtilis )
Pick a single colony from the plate or from glycerol stock.
Put the colony in a 15 ml sterile tube and add 3 ml of LB medium.
(If needed, add selective antibiotics to the medium before adding a single colony)
Incubate the tube overnight at 37°C and 220 rpm.
Glycerol stock
Take 800 µl from a freshly grown culture and put it in a 1.5 ml tube.
Add 200 µl of 80 % glycerol.
Mix by vortexing.
Store at -80°C.
DNA electrophoresis
Gel preparation (for 400 ml)
Measure out 4 g of agarose and add 400 ml of 1X TBE buffer (for 1 % gel).
Autoclave the solution, and let it cool down to ∼ 55°C. Be careful, agar solidifies at 32 – 40°C.
Add 4 µl of DNA stain.
Store at 55 - 60°C for repetitive use.
Gel pouring
Pour the solution into the mold including the combs.
Let it polymerize for 15 min.
Remove the combs.
Put prepared agarose gel in the electrophoresis container containing 1X TBE.
Electrophoresis
Add 1/6 of total volume of 6X loading buffer to every DNA sample.
Pipet DNA samples containing 6X loading buffer and DNA ladder to the wells.
Run at 150 – 170V for 20 – 30 min (depends on the fragments).
Preparation of 5X TBE buffer
5X TBE buffer (for 1000 ml)
Tris-base
54 g
Boric acid
27.5 g
0.5 M EDTA (pH 8.0)
20 ml
Adjust pH to 8.3 by HCl.
5X TBE buffer has to be diluted to 1X TBE buffer before using  in electrophoresis
PCR (Taq Core Kit – Jena Bioscience)
50 µl PCR assay
50 µl PCR assay
10X Taq Reaction buffer complete
5 µl
dNTP Mix
1 µl
MgCl2*
1 µl
10 µM Forward Primer
1 µl
10 µM Reverse Primer
1 µl
Template DNA
2 - 50 ng
Taq polymerase
0.2 – 0.5 µl
MiliQ water
up to 50 µl
*Adding of Mg2+ is recommended for most applications.
Cycling conditions
Cycling conditions
initial denaturation
94°C
2 min
1x
denaturation
94°C
30 sec
30x
annealing*
45 – 68°C
30 sec
30x
elongation**
72°C
30 sec – 4 min
30x
final elongation
72°C
2 min
1x
hold
4 – 10°C
*The annealing temperature depends on the melting temperature of the primers used.
**The elongation time depends on the length of the fragments to be amplified (1 min/kb).
PCR (Q5® High-Fidelity)
50 µl PCR assay
50 µl PCR assay
Q5 High-Fidelity 2X Master Mix
25 µl
10 µM Forward Primer
2.5 µl
10 µM Reverse Primer
2.5 µl
Template DNA
variable
MiliQ water
up to 50 µl
25 µl PCR assay
25 µl PCR assay
Q5 High-Fidelity 2X Master Mix
12.5 µl
10 µM Forward Primer
1.25 µl
10 µM Reverse Primer
1.25 µl
Template DNA
variable
MiliQ water
up to 25 µl
Cycling conditions
Cycling conditions
initial denaturation
98°C
30 sec
1x
denaturation
98°C
5 - 10 sec
25 - 30x
annealing*
50 – 72°C
10 - 30 sec
25 - 30x
elongation**
72°C
20 – 30 sec/kb
25 - 30x
final elongation
72°C
2 min
1x
hold
4 – 10°C
*The annealing temperature depends on the melting temperature of the primers used.
Colony PCR
Make the Master Mix using the materials specified in PCR protocols above.
Prepare a pre-glycerol stock.
Pick a colony from the plate and add it to the mixture of 80 µl of LB medium and 20 µl of 80 % glycerol. Vortex it.
Add 2 µl of pre-glycerol stock to the PCR mixture.
Run the PCR using the programs specified above.
Plasmid mini-prep (Fast-n-Easy Plasmid Mini-Prep Kit - Jena Bioscience)
Harvest the bacterial cell culture (1 – 3 ml) by centrifugation.
Resuspend pelleted bacterial in 300 µl Lysis Buffer by pipetting or vortexing for 1 min.
Add 300 µl of Neutralization Buffer (containing RNase A) to sample.
Mix it gently by inverting the tube 4 – 6 times (do not vortex).
Centrifuge at 10,000 g for 5 min at room temperature in a micro-centrifuge.
The color of the binding mixture should change to bright yellow indicating a pH of 7.5 required for optimal DNA binding.
Place a Binding Column into a 2 ml collection tube.
Add 100 µl of Activation buffer into the Binding Column.
Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.
Apply the supernatant from steps 3 – 5 into the activated Binding Column by decanting or pipetting.
Centrifuge at 10,000 g for 30 sec.
Discard the flow-through.
Place the DNA loaded Binding Column into the used 2 ml tube.
Apply 500 µl of Washing Buffer (containing Ethanol) to the Binding Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Optional Secondary Washing (Recommended only for DNA >200 bp, if highly purified DNA is required).
Add 700 µl of Washing Buffer to the Binding Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Centrifuge again for 2 min to remove residual Washing Buffer.
Place the Binding Column into a clean 1.5 ml tube (not provided in the kit).
Add 30 – 50 µl Elution Buffer or dd-water to the center of the column membrane.
Incubate for 1 min at room temperature.
Centrifuge at 10,000 g for 1 min to elute DNA.
Plasmid mini-prep (QIAprep® Spin Miniprep Kit)
Pellet 1 – 5 ml bacterial overnight culture by centrifugation at 8000 rpm (6800 x g) for 3 min at room temperature (15 – 25°C).
Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a micro-centrifuge tube.
Add 250 µl Buffer P2 and mix thoroughly by inverting the tube 4 – 6 times until the solution becomes clear.
Do not allow the lysis reaction to proceed for more than 5 min. If using LyseBlue reagent, the solution will turn blue.
Add 350 µl Buffer N3 and mix immediately and thoroughly by inverting the tube 4 – 6 times.
If using LyseBlue reagent, the solution will turn colorless.
Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table top micro-centrifuge.
Apply the supernatant from step 5 to the QIAprep spin column by decanting or pipetting.
Centrifuge for 30 – 60 sec at 13,000 rpm (~17,900 x g) and discard the flow-through.
Recommended:
Wash the QIAprep spin column by adding 500 µl Buffer PB.
Centrifuge for 30 – 60 sec at 13,000 rpm (~17,900 x g) and discard the flow-through.
Note: This step is only required when using endA+ strains or other bacterial strains with high nuclease activity or carbohydrate content.
Wash the QIAprep spin column by adding 750 µl Buffer PE.
Centrifuge for 30 – 60 sec at 13,000 rpm (~17,900 x g) and discard the flow-through.
Centrifuge for 1 min at 13,000 rpm (~17,900 x g) to remove residual wash buffer.
Place the QIAprep column in a clean 1.5 ml micro-centrifuge tube.
To elute DNA, add 50 µl Buffer EB (10 mM Tris-Cl, pH 8.5) or water to the center of the QIAprep spin column.
Let it stand for 1 min, and centrifuge for 1 min at 13,000 rpm (~17,900 x g).
DNA Clean-up (PCR Purification Kit – Jena Bioscience)
For DNA fragment sizes in the range of 200 bp to 5 kbp:
Add 5 volumes of Binding Buffer to 1 volume of DNA sample and mix well.
For example, if the volume of your DNA sample is 50 µl, add 250 µl Binding Buffer.
For DNA fragment sizes smaller than 200 bp or larger than 5 kbp:
Add 3 volumes Binding Buffer and 2 volumes of Isopropanol to the PCR sample.
For example, if the volume of your DNA sample is 50 µl, add 150 µl Binding Buffer and 100 µl Isopropanol.
Protocol
Place a Spin Column into a 2 ml collection tube.
Add 100 µl of Activation Buffer into the Spin Column.
Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.
Apply the sample mixture from step A or B into the activated Spin Column.
Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.
Discard the flow-through.
Place the DNA loaded Spin Colum into the used 2 ml tube.
Apply 700 µl of Washing Buffer to the Spin Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Optional Secondary Washing (Recommended only for DNA >200 bp, if highly purified DNA is required).
Add 700 µl of Washing Buffer to the Binding Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Centrifuge again for 2 min to remove residual Washing Buffer.
Place the Binding Column into a clean 1.5 ml tube (not provided in the kit).
Add 30 – 50 µl Elution Buffer or dd-water to the center of the column membrane.
Incubate for 1 min at room temperature.
Centrifuge at 10,000 g for 1 min to elute DNA.
DNA Clean-up (NucleoSpin® Gel and PCR Clean-up)
Mix 1 volume of sample with 2 volumes of Buffer NTI (e.g. mix 100 μl PCR reaction and 200 μl Buffer NTI)*.
Place a NucleoSpin® Gel and PCR Clean-up Column into a Collection Tube (2 ml) and load up to 700 μl sample.
Centrifuge for 30 sec at 11,000 x g.
Discard flow-through and place the column back into the collection tube.
Load remaining sample if necessary and repeat the centrifugation step.
Add 700 μl Buffer NT3 to the NucleoSpin® Gel and PCR Clean-up Column.
Centrifuge for 30 sec at 11,000 x g.
Discard flow-through and place the column back into the collection tube.
Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and improve A260/A230 values.
Centrifuge for 1 min at 11,000 x g to remove Buffer NT3 completely.
Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube**.
Place the NucleoSpin® Gel and PCR Clean-up Column into a new 1.5 ml microcentrifuge tube (not provided).
Add 15–30 μl Buffer NE***.
Incubate at room temperature (18–25°C) for 1 min.
Centrifuge for 1 min at 11,000 x g.
*For very small sample volumes < 30 μl adjust the volume of the reaction mixture to 50–100 μl with water.
**Residual ethanol from Buffer NT3 might inhibit enzymatic reactions. Total removal of ethanol can be achieved by incubating the columns for 2–5 min at 70°C prior to elution.
***DNA recovery of larger fragments (> 1000 bp) can be increased by multiple elution steps with fresh buffer, heating to 70°C and incubation for 5 min.
Gel extraction (Agarose Gel Extraction Kit – Jena Bioscience)
Cut the area of gel containing the DNA fragment.
Transfer the excised gel to a clean 1.5 ml microtube.
Add 3 volumes of Extraction Buffer to 1 volume of the sliced gel. For example, add 300 µl Extraction Buffer to each 100 mg (approx. 100 µl) gel. For gels containing >2.5 % agarose, add 6 volumes of Extraction Buffer per gel volume.
Incubate at 60 °C for 10 min with occasional mixing to ensure gel dissolution.
For DNA fragment sizes smaller than 200 bp or larger than 5 kbp and to enhance yield add 1 volume Isopropanol per gel volume to the dissolved gel and mix well.
Place a Spin Column into a 2 ml collection tube.
Add 100 µl of Activation Buffer into the Spin Column.
Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.
Apply the sample mixture from steps 3 (5), 4 into the activated Spin Column.
Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.
Discard the flow-through.
Place the DNA loaded Spin Colum into the used 2 ml tube.
Apply 700 µl of Washing Buffer to the Spin Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Optional Secondary Washing (Recommended only for DNA >200 bp, if highly purified DNA is required).
Add 700 µl of Washing Buffer to the Binding Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Centrifuge again for 2 min to remove residual Washing Buffer.
Place the Spin Column into a new 1.5 ml microcentrifuge tube (not provided).
Add 30 – 50 µl Elution Buffer or dd-water to the center of the column membrane.
Incubate for 1 min at room temperature.
Centrifuge at 10,000 g for 1 min to elute DNA.
Gel extraction (NucleoSpin® Gel and PCR Clean-up)
Take a clean scalpel to excise the DNA fragment from an agarose gel. Remove all excess agarose.
Determine the weight of the gel slice and transfer it to a clean tube.
For each 100 mg of agarose gel < 2 % add 200 μl Buffer NTI. (For gels containing > 2 % agarose, double the volume of Buffer NTI).
Incubate sample for 5–10 min at 50°C. Vortex the sample briefly every 2–3 min until the gel slice is completely dissolved.
Place a NucleoSpin® Gel and PCR Clean-up Column into a Collection Tube (2 ml) and load up to 700 μl sample.
Centrifuge for 30 sec at 11,000 x g.
Discard flow-through and place the column back into the collection tube.
Load remaining sample if necessary and repeat the centrifugation step.
Add 700 μl Buffer NT3 to the NucleoSpin® Gel and PCR Clean-up Column.
Centrifuge for 30 sec at 11,000 x g.
Discard flow-through and place the column back into the collection tube.
Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and low A260/A230 values.
Centrifuge for 1 min at 11,000 x g to remove Buffer NT3 completely.
Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube*.
Place the NucleoSpin® Gel and PCR Clean-up Column into a new 1.5 ml microcentrifuge tube (not provided).
Add 15–30 μl Buffer NE**.
Incubate at room temperature (18–25°C) for 1 min.
Centrifuge for 1 min at 11,000 x g.
*Residual ethanol from Buffer NT3 might inhibit enzymatic reactions. Total removal of ethanol can be achieved by incubating the columns for 2–5 min at 70°C prior to elution.
**DNA recovery of larger fragments (> 1000 bp) can be increased by multiple elution steps with fresh buffer, heating to 70°C and incubation for 5 min.
Preparation of the spore stock of B. subtilis
Inoculate 20 ml of LB medium with cells from a fresh colony of B. subtilis . Always use flasks that comprise at least 5x times the volume of media used, and always use lids that are able to allow air passage.
Culture for 6 – 8 hours at 37°C with shaking at 200 rpm. B. subtilis grows best at 37°C and has a doubling time of 30 min.
Dilute 1:200 into 1000 ml DSM medium in a sterile 1 liter flask and grow at 37°C with shaking at 200 rpm. Add 5 ml of the culture to 1000 ml medium.
Check samples for spores daily using phase contrast microscopy (see Phase Contrast Microscopy Protocol). Optional use of Gram stain (see Gram staining protocol ) to distinguish between vegetative cells (purple) and spores (transparent).
After 2 – 3 days >90% of the population should have sporulated.
Pellet cells at 9000 rpm for 20 min (keeping temperatures low), otherwise for small quantities a benchtop centrifuge will suffice.
Wash spores with ice-cold water 8 – 10 times to remove residual nutrients and lyse remaining vegetative cells. (Resuspend pellets in water, centrifuge, discard supernatant, repeat).
Store spores at -20°C for long-term storage or at 4°C, with weekly changes of water.
Difco Sporulation Medium (DSM)
DSM (for 1000 ml)
Bacto nutrient broth (Difco)
8 g
10 % (w/v) KCl
10 ml
1.2 % (w/v) MgSO4 . 7H2O
10 ml
1 M NaOH
1.5 ml
Bring pH to 7.6.
Filter sterilize.
Add the following sterile (autoclaved) component solutions to 1000 ml of the cooled DSM medium prior to use:
1 M Ca(NO3)2
1 ml
0.01 M MnCl2
1 ml
1 mM FeSO4
1 ml
Integration check: Starch test
Inoculation: Use a fresh (16- to 18-hour) pure culture of test bacteria as an inoculation source. Pick a single isolated colony and either single streak or spot inoculate the surface of the agar medium.
Incubation: Incubate plates overnight at 37°C.
Starch Hydrolysis Test: After proper inoculation and incubation, flood the surface of the agar with Gram’s iodine solution. Record results immediately as the blue color formed with starch may fade giving a false-positive result of absence of starch.
Appearance of a clear zone surrounding the bacterial growth indicates starch hydrolysis (+) by the organism due to its production of the extracellular enzymes. The clear zone will start out yellow (from the iodine) and becomes progressively lighter yellow and then clear -> indicates wrong clones.
A blue/black or purple zone surrounding the growth indicates that starch is present and has not been hydrolyzed (-) and the organism did not produce the extracellular enzymes -> indicates right clones.
Starch plates
Solution of 1000 ml
Beef extract
3 g
Soluble starch
10 g
Agar
12 g
Distilled water
1000 ml
Suspend the first three ingredients in 1000 ml of distilled water. Mix thoroughly.
Heat with frequent agitation and carefully bring to just boiling (excessive boiling may hydrolyze starch).
Autoclave.
After sterilization pour the melted medium into sterilized petri plates (approximately 20-30 ml per plate) and let it solidify before use.
IDT gBlocks®
Resuspending gBlocks Gene Fragment
Centrifuge the tube for 3 – 5 sec at a minimum of 3000 x g to ensure the material is in the bottom of the tube.
Add 1X TE to reach a final concentration of 10 ng/µl.
Vortex briefly.
Incubate at 50°C for 20 min.
Briefly vortex and centrifuge.
Amplifying gBlocks Gene Fragment
For gBlocks Gene Fragments ≤ 1kb, amplification can be performed using a high fidelity polymerase.
To avoid sequence mutations due to amplification errors, limit cycle to 12 or fewer.
For gBlocks Gene Fragments > 1kb, we do not recommend amplification.
Time-lapse microscopy/Phase-contrast microscopy
Preparation of the 1.5% agarose
Measure out 150 mg of agarose (for 1.5 % agarose) and add either 10 ml of LB medium (time-lapse) or 1X TBE buffer (phase-contrast images).
Dissolve the agarose.
Store at 55 - 60°C for repetitive use.
Preparation of the cell cultures
Pick a single colony from the plate or from glycerol stock.
Put the colony in a 15 ml sterile tube and add 3 ml of LB medium.
(If needed, add selective antibiotics to the medium before adding a single colony).
Incubate the tube overnight at 37°C and 220 rpm.
Preparation of the microscope sample [ 1 ]
Clean two microscope glass slides with 70 % ethanol and water.
Take a gene frame and carefully remove one of the plastic foils from the gene frame without causing disassembly of the plastic cover on the other side of the gene frame.
Attach the gene frame in the middle of one of the glass slides by first facilitating contact on just one side, followed by guided attachment of the remaining gene frame with a fingernail. Prevent air bubbles while attaching the gene frame to the glass slide.
Transfer 500 μl of the warm agarose-LB or agarose-TBE in the middle of the gene frame. Make sure the whole area including (the borders) is fully covered.
The following steps have to be carried out quickly to prevent excessive drying of the agarose-LB or agarose-TBE.
Place the second glass slide on the agarose-LB or agarose-TBE filled gene frame. Try to avoid air bubbles. Place the sandwiched slides in a horizontal position for 45 min at 4°C in the refrigerator to allow the agarose-LB or agarose-TBE to solidify sufficiently.
Carefully slide off the upper glass slide. Use a razor blade to cut out agar strips of ~5 mm width within the gene frame, on which the cells will be grown.
Carefully remove the second and final plastic cover from the gene frame to expose the sticky side of the gene frame.
Load single cells on the solid medium without touching it with the pipet tip. Use 2.5 μl for a whole strip, or 1 μl for a small square. Always start on top of the agarose pad and allow the liquid to disperse equally on its assigned growth area by turning the slide up and down. The slide is ready, as soon as the edges of the liquid become corrugated and movement of the liquid is no longer visible when turning the slide.
Place a clean microscope slide cover slip on the gene frame from one side to the other (avoid air bubbles). Assure complete attachment by applying pressure on the cover slip along the gene frame with your fingernail. If the cover slip is placed on the cells without allowing them to dry long enough, cells tend to grow on top of each other during the experiment. Also be careful not to wait too long before applying the cover slip, since the agarose will then be too dry.
Reference:
Jong, I. G. De, Beilharz, K., Kuipers, O. P., & Veening, J.-W. (2011). Live Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy, (July), 1–6. http://doi.org/10.3791/3145
Sequencing (Macrogen)
Template DNA of 5 μl with either of following concentrations:
50 ng/μl of PCR product.
100 ng/μl of plasmid DNA.
Add 5 μl of primer with either of following concentrations:
50 ng/μl of PCR product.
100 ng/μl of plasmid DNA.
Photometric measurement of the OD600 and mRFP fluorescence
Day 1
Grow E. coli in 3 ml LB with 35 μg/ml chloramphenicol at 37°C, 220 rpm for 16 hours.
Day 2
Preparation of the 96 well plate
Prepare a transparent 96 well plate with a total volume of 250 μl.
Provide 175 μl of LB and 25 μl of the appropriate concentration of desired compound.
Dilute cells to OD600 of 0.5 with LB and add 50 μl to each well.
Photometric measurement (Plate reader – Varioskan LUX Reader, Thermo Fischer)
To measure the fluorescence of the mRFP set the excitation and emission on 584 nm and 607 nm, respectively, with a bandwidth of 5 nm.
Measure the absorbance at 600 nm.
Measure every 20 min at 37°C with the plate reader.
Team
Members
Supervisors
Collaborations
Achievements
Contact
Jamboree
Project
Tour
Description
Design
Demonstration
Summary
Applications
Results
Future Perspectives
Biology
Experiments
Safety
Decoy experiment
Photoswitchable antibiotics
BioBrick Characterization
Key deletion
Proof of concept
Lab journal
Plasmid construction
Protocols
Primer list
Notebook
Modelling
Modelling
Mutation Rates
Random Mutation
AI Agent model
Decoding Fidelity
Optimal Strain
Computing
Software
Encoder
Decoder
Full Coding
Human Practices
Human Practices
Outreach
Social
Legal
Future Scenarios
Education
Integrated Practices
Engagement
Gold
Silver
Support
Sponsors
Attributions
