Team:Groningen/Decoy
CryptoGE®M
Team
Members Supervisors Collaborations Achievements Contact
Project
Tour Description Design Results Demonstrate Applications Future Notebook
Biology
Experiments Parts Safety Lab journal
Computing
Modelling Software Encoding Decoding
Human Practices
Human Practices Gold Silver
Support
Sponsors Attributions
Decoy
The spores containing the DNA sequence that encodes our key will be    sent in a mixture with decoy spores. These strains were constructed    with our BioBricks BBa_K1930002 (key) and BBa_K1930006 (sfGFP) The high    ratio of decoy spores makes it hard for unauthorized parties to    retrieve the correct key if they try to sequence the entire sample by    brute force. In this experiment we tried to determine how fine our    system is in selecting the spores from the decoy once the right    treatment is applied.
We prepared a Bacillus subtilis strain containing a superfolder GFP    and a spectinomycin resistance cassette in the genome. Then we prepared    mixtures of that mutant with wild-type B. subtilis in different ratios.    In this experiment living cells were used instead of spores. After    growing in different conditions, the final mixture of mutant vs    wild-type strains was determined microscopically and in a flow    cytometer.
Due to the high ratio of decoy cells, any non-approved party trying    to sequence the entire sample will not be able to distinguish the    key-sequence from the background noise. In the scientific literature,    using standard sequencing techniques it has been possible to detect one    mutant out of 150 wild-type molecules [1] [2] . However, fine-tuned    technologies, such as Duplex Sequencing, have shown to increase that    number to one mutant in 10,000 wild-type cells. That same technique is theoretically    able to detect one mutant out of 10 million decoys [3] !
Experiment setup
We combined different ratios of sfGFP bacteria and decoy bacteria.    LB medium was inoculated from glycerol stocks of the sfGFP strain and    the wild-type strain which were grown overnight at 37 °C, shaking at 220 rpm    in a 3 ml culture. On the next day the corresponding dilutions were    made and grown again overnight at 37 °C in a shaking liquid 3 ml    culture. The antibiotic was added to both the preculture and the    diluted culture.
The next morning the cells were visualized in the microscope Time-lapse microscopy/Phase-contrast microscopy and additionally diluted 50 times in 1X PBS buffer to    analyze in the flow cytometer.
Figure no.
Concentration spectinomycin [µg/ml]
Initial ratio sfGFP:decoy
Final ratio sfGFP:decoy
1,2
0
0:1
0:1
3,4
0
1:0
25:1
5,6
150
0:1
0:1
7,8
150
1:0
160:1
9,10
0
1:1
12:1
11,12
150
1:1
230:1
13,14
0
1:150
10:1
15,16
150
1:150
70:1
17,18
0
1:10,000,000
0:1
19,20
150
1:10,000,000
0:1
Table 1. The initial ratio of mutant vs wild-type     strains was screened from 1 to 10 million. The final ratio was     measured as the relative (green:gray) area under the curve (AUC)     obtained in the flow cytometer (Figures 2, 4, 6, 8, 10, 12, 14, 16,     18 and 20). [Using: Flowing Software 2.5]
Results
The mutant strain containing the superfolder GFP can be seen green    in the microscopy images and is also marked green in the flow cytometer    graphs. The wild-type decoy cells are gray in both cases.
Figure 2 & 3: Wildtype without spectinomycin
Figure 4 & 5: sfGFP strain without spectinomyin
Figure 6 & 7: Wildtype with spectinomycin
Figure 8 & 9: sfGFP strain with spectinomyin
Figure 10 & 11: 1:1 without spectinomycin
Figure 12 & 13: 1:1 with spectinomycin
Figure 14 & 15: 1:150 without spectinomycin
Figure 16 & 17: 1:150 with spectinomycin
Figure 18 & 19: 1:10.000.000 without spectinomycin
Figure 20 & 21: 1:10.000.000 with spectinomycin
Discussion
As control groups we used samples that contained either only the    decoy wild-type strain or the spectinomycin-resistant sfGFP mutant.     While the wild type grew well without addition of spectinomycin    (Figures 1 and 2), no growth could be observed if the antibiotic was    added (Figures 5 and 6). On the other hand, the spectinomycin resistant    sfGFP strain grew well both in its presence or absence (Figures 3, 4, 7    and 8).  In both cases, growth of cells not expressing sfGFP was    observed. This could be due to not fully developed cells that do not    yet express sfGFP in their current cell cycle.
A mixed culture in the ratio 1:1 without the addition of    spectinomycin showed presence of both wild-type and sfGFP cells (Figures 9,    10, 11 and 12) as expected. However, the unexpected higher ratio of    sfGFP strain under conditions that do not give advantage over the    wild-type strain leads us to assume that the mutant generally grows    faster than the wild-type strain. Similarly, adding the antibiotic increases    20 times the fraction of sfGFP cells, in this case by killing the    non-resistant wild-type.
The samples with a 1:150 ratio showed consistent results (Figures    13, 14, 15 and 16) compared to the 1:1 ratios. Without the addition of    spectinomycin the mutant outgrew the wild-type strain (10:1) even    though the initial ratio was not in its favor (1:150).
We went to an extreme of using a ratio of 1 mutant in 10 million wild-type cells.    In this conditions, no growth of mutants was observed. The ratio is too    high to allow the mutant strain to grow even in the presence of    antibiotic that would give it an advantage over the wild-type strain.
Conclusion
Our experiment shows that a specific strain, in this case containing    a sfGFP and a spectinomycin resistance cassette can be selected from a    larger number of decoys. We could not determine the optimal ratio that    would strengthen this layer of biosecurity. For further experiments the    ratio of decoys should be fine tuned to determine the maximum ratio of    spores:decoys that could be used, thus reassuring that unauthorized    parties will not be able to recover the key by sequencing the whole    sample but the intended recipient will still be able to recover it.
References:
[1] Fox EJ, Reid-Bayliss KS,     Emond MJ, Loeb LA (2014) Accuracy of Next Generation Sequencing     Platforms. Next Generat Sequenc & Applic 1: 106.     doi:10.4172/jngsa.1000106
[2] Pochon (2013). Evaluating     detection limits of next-generation sequencing for the surveillance and monitoring of international marine pests. PLos One     8(9):e73935
[3] Schmitt MW, Kennedy SR,     Salk JJ, Fox EJ, Hiatt JB, et al. (2012) Detection of ultra-rare     mutations by next-generation sequencing. Proc Natl Acad Sci U S A     109: 14508-14513.
Team
Members
Supervisors
Collaborations
Achievements
Contact
Jamboree
Project
Tour
Description
Design
Demonstration
Summary
Applications
Results
Future Perspectives
Biology
Experiments
Safety
Decoy experiment
Photoswitchable antibiotics
BioBrick Characterization
Key deletion
Proof of concept
Lab journal
Plasmid construction
Protocols
Primer list
Notebook
Modelling
Modelling
Mutation Rates
Random Mutation
AI Agent model
Decoding Fidelity
Optimal Strain
Computing
Software
Encoder
Decoder
Full Coding
Human Practices
Human Practices
Outreach
Social
Legal
Future Scenarios
Education
Integrated Practices
Engagement
Gold
Silver
Support
Sponsors
Attributions
