<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_FuturePerspectives skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/FuturePerspectives</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="content_wrapper"><HEADER id="content-header"><SPAN id="logo-container" class="menu-item"><DIV class="menu-title"><A href="/Team:Groningen">CryptoGE®M</A></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Team</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Team">Members</A><A href="/Team:Groningen/Supervisors">Supervisors</A><A href="/Team:Groningen/Collaborations">Collaborations</A><A href="/Team:Groningen/Achievements">Achievements</A><A href="/Team:Groningen/Contact">Contact</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Project</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Tour"><STRONG>Tour</STRONG></A><A href="/Team:Groningen/Description">Description</A><A href="/Team:Groningen/Design">Design</A><A href="/Team:Groningen/Results">Results</A><A href="/Team:Groningen/Demonstrate">Demonstrate</A><A href="/Team:Groningen/Applications">Applications</A><A href="/Team:Groningen/FuturePerspectives">Future</A><A href="/Team:Groningen/Notebook">Notebook</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Biology</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Experiments">Experiments</A><A href="/Team:Groningen/Parts">Parts</A><A href="/Team:Groningen/Safety">Safety</A><A href="/Team:Groningen/Labjournal">Lab journal</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Computing</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Model">Modelling</A><A href="/Team:Groningen/Software">Software</A><A href="/Team:Groningen/Encoding">Encoding</A><A href="/Team:Groningen/Decoding">Decoding</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Human Practices</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Human_Practices">Human Practices</A><A href="/Team:Groningen/HP/Gold">Gold</A><A href="/Team:Groningen/HP/Silver">Silver</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Support</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Sponsors">Sponsors</A><A href="/Team:Groningen/Attributions">Attributions</A></DIV></DIV></SPAN></HEADER><DIV id="groningen-wrapper"><ARTICLE><SECTION><H2>Future Perspectives</H2><P>CryptoGErM is a highly robust and versatile system for secure 
			storage and transfer of data which exploits two key properties of 
			DNA; very high storage density and durability <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>. The system can 
			be developed further to make it even more secure, practical and 
			easy to use. In this section, we will discuss several possible 
			improvements for the CryptoGErM system.</P></SECTION><SECTION><H3>Additional Biological Security Layers</H3><DIV class="split"><DIV class="left flone"><P>Whether our system is used as a storage medium for blueprints of 
			military facilities or the script for a new Star Wars movie, 
			reliable biological security layers must be in place to prevent 
			unauthorized access. Retrieval of information from a DNA-based 
			archive should also be restricted to avoid alteration or deletion 
			of data. There are a number of biological lock systems to choose 
			from, and these can be combined to obtain a customizable security 
			system. Here we give a brief overview of future possibilities.</P><H4>Decoy strains</H4><P>A small amount of key carrying spores are mixed with a large 
			amount of decoy spores that do not contain the key or a biological 
			lock system. If they are not treated correctly, the decoy strains 
			outcompete the key strain, making it nearly impossible to find the 
			DNA sequence containing the key. This system is meant to be 
			combined with other biological security precautions described 
			below.</P></DIV></DIV><H4>Essential gene under inducible promoter</H4><P>An essential gene, such as <EM>atpA</EM> (ATP synthase subunit alpha) or 
			<EM>acsA</EM> (acetyl coenzyme A synthetase) can be placed under a 
			chemically regulated promoter, such as TetR. The constitutively 
			expressed copy of that gene must be deleted from the genome. 
			Addition of the inducer compound (in this case aTc) leads to 
			transcription of the essential gene, allowing the bacteria to grow. 
			<EM>atpA</EM> is a good choice for this system because it is expressed early 
			on in the cell growth cycle <SUP id="ref-2"><A href="#cite-2">[2]</A></SUP>, and is required for the cell to 
			live <SUP id="ref-2"><A href="#cite-2">[2]</A></SUP>. <EM>acsA</EM> is also a good choice, since it plays a central role 
			in metabolism <SUP id="ref-3"><A href="#cite-3">[3]</A></SUP> and is expressed early in the growth cycle <SUP id="ref-3"><A href="#cite-3">[3]</A></SUP>. 
			Additionally, since some acetyl coenzyme A is already present in 
			<EM>Bacillus subtilis</EM> spores <SUP id="ref-3"><A href="#cite-3">[3]</A></SUP>, the cells can be expected survive the 
			brief period after germination and before the induction of <EM>acsA</EM> 
			transcription. In this scenario the correct inducer compound must 
			be known in order to culture the strain containing the key. Thus, 
			the data cannot be accessed without the addition of this compound 
			to the growth medium. </P><H4>Competition system</H4><P>The key strain is resistant to a particular antibiotic, while a 
			competing strain (mixed in with the key strain) is not. The 
			competing strain produces an antimicrobial peptide (AMP) against 
			the key strain. Addition of the correct antibiotic to the growth 
			medium prior to culturing kills the competing strain, allowing the 
			key strain to grow. If no antibiotic is present, the competing 
			strain grows, produces the AMP and kills the key strain. If the 
			wrong antibiotic is added both strains die. Either way, without the 
			correct antibiotic, the key strain will not grow. </P><H4>Database of photoswitchable antibiotics and inducer compounds</H4><P>Although the photoswitchable antibiotic used in our project 
			responds only to UV light, it is possible to make photoswitches 
			respond to other wavelengths via molecular engineering techniques. 
			For example, a red light responsive azobenzene switch has been 
			developed by Woolley and co-workers, who substituted the four 
			positions ortho to the azo group with bulky, electron-rich 
			substituents <SUP id="ref-4"><A href="#cite-4">[4]</A></SUP>. </P><P>Molecular photoswitches can be conjugated to a variety of other 
			compounds in addition to antibiotics <SUP id="ref-5"><A href="#cite-5">[5]</A></SUP>. We envision a wide 
			variety of antibiotics and inducer compounds that respond to a 
			range of wavelengths, which could be selected from a database to 
			fit the user’s purpose. A collection of such photoresponsive 
			compounds could serve as a simple but effective method to protect 
			data encoded in the DNA of a micro-organism from unauthorized 
			access.</P><P>To simplify the process, it may eventually be possible to embed 
			a dry mixture of spores, LB powder and the photoresponsive compound 
			in filter paper, so that the user only has to add the right volume 
			of water and expose the mixture to the right wavelength of light in 
			order to retrieve the stored information. </P><H4>Biometric identification with toehold switches</H4><P>Germination of key strain spores is controlled by a toehold 
			switch, designed to respond to a particular RNA unique to the 
			recipient <SUP id="ref-6"><A href="#cite-6">[6]</A></SUP>. The switch represses growth by preventing the 
			translation of an essential gene, such as <EM>acsA</EM>. When the RNA is 
			added to the culture, the toehold switch no longer represses the 
			essential gene, allowing the bacteria to grow. The RNA could be 
			introduced into the culture by adding the recipient’s blood, or it 
			could purified from the blood and then added to the medium. This 
			method is a form of molecular biometric identification, which 
			ensures that the key can only be accessed by the recipient.</P></SECTION><SECTION><H3>Indexing for easy searching</H3><P>Archives are vast collections of information. Any archive, be it 
			on paper or in DNA, needs an indexing system to help find and 
			retrieve specific parts of the stored information. Without such a 
			system, it could take months or even years to sift through all the 
			data and find the relevant part. Thus, indexing systems are 
			essential in any archive platform. In simple terms, indexing 
			generates metadata about each file, which is then organized in a 
			searchable database. The index database can be stored separately, 
			or together with the data in the archive. In a DNA-based archival 
			system, the indices are coupled to primers designed to amplify 
			specific sections of the DNA sequence <SUP id="ref-7"><A href="#cite-7">[7]</A></SUP>. A database of primers 
			with a brief description of the data they access would be 
			indispensable for such an archival system.</P></SECTION><SECTION><H3>Naked DNA for greater storage capacity</H3><P>Although DNA inside spores has a longer shelf-life than naked 
			DNA, it suffers from a notable disadvantage that is not presented 
			by naked DNA: it contains sequences necessary for the replication 
			and metabolism of the host organism, and as a result, leaves less 
			room for data. If the data does not need to be preserved for 
			millions of years, it may be more feasible to encode it in naked 
			DNA as the storage capacity is far greater. If packaged and 
			preserved correctly, the DNA containing the data can last for 
			millennia <SUP id="ref-8"><A href="#cite-8">[8]</A></SUP>. </P></SECTION><SECTION><H3>Packaged DNA for greater durability </H3><P>If our system is adapted for use as a timecapsule, the 
			information should not be encrypted. Therefore, a second strain 
			carrying the decryption key is not needed, simplifying the system. 
			The shelf-life of the timecapsule can be greatly increased by 
			encasing the spores in silica spheres <SUP id="ref-9"><A href="#cite-9">[9]</A></SUP>. Additionally, genes 
			encoding intrinsically disordered proteins (IDPs) can be introduced 
			into the host’s genome, in such a way that they are expressed 
			during sporulation. These proteins originate from highly resilient 
			microscopic animals called tardigrades, and form a glass-like 
			substance when dehydrated <SUP id="ref-10"><A href="#cite-10">[10]</A></SUP>. This ‘bioglass’ protects DNA and 
			membrane integrity, and has been demonstrated to increase tolerance 
			to dehydration by 100% - 200% when expressed in <EM>E. coli</EM><SUP id="ref-10"><A href="#cite-10">[10]</A></SUP>. 
			Another interesting protein unique to tardigrades is the Damage 
			suppressor (Dsup) protein. This DNA-associating protein protects 
			DNA from ionizing radiation, and has been shown to increase 
			tolerance to X-rays by 40% when expressed in human cells <SUP id="ref-11"><A href="#cite-11">[11]</A></SUP>. 
			Addition of extra DNA repair systems in our host can further 
			increase the shelf-life by correcting any mutations that have 
			occurred <SUP id="ref-12"><A href="#cite-12">[12]</A></SUP>. If all of these measures are taken, the resulting 
			silica-encased spores can be placed in a lead container and locked 
			in an underground vault (such as the Svalbard Global Seed Vault) 
			preserving the information for millions of years.</P><FIGURE class="centrate"><FIGCAPTION>Svalbard Global Seed Vault</FIGCAPTION></FIGURE></SECTION><SECTION><H6>References</H6><UL><LI id="cite-1"><A href="#ref-1">[1]</A> Next-Generation 
				Digital Information Storage in DNA Science. 2012 Sep 
				28;337(6102):1628. Epub 2012 Aug 16.</LI><LI id="cite-2"><A href="#ref-2">[2]</A> The ATP synthase: the 
				understood, the uncertain and the unknown Biochemical Society 
				Transactions Feb 01, 2013, 41 (1) 1-16; DOI: 
				10.1042/BST20110773</LI><LI id="cite-3"><A href="#ref-3">[3]</A> Expression of the 
				<EM>Bacillus subtilis</EM><EM>acsA</EM> gene: position and sequence context 
				affect cre-mediated carbon catabolite repression. J Bacteriol. 
				1998 Dec; 180(24):6649-54.</LI><LI id="cite-4"><A href="#ref-4">[4]</A> Photoswitching azo 
				compounds in vivo with red light. J Am Chem Soc. 2013 Jul 
				3;135(26):9777-84. doi: 10.1021/ja402220t. Epub 2013 Jun 
				21.</LI><LI id="cite-5"><A href="#ref-5">[5]</A> Controlling the 
				activity of quorum sensing autoinducers with light Chem. Sci., 
				2015,6, 3593-3598 DOI: 10.1039/C5SC00215J</LI><LI id="cite-6"><A href="#ref-6">[6]</A> Toehold Switches: 
				De-Novo-Designed Regulators of Gene Expression Cell. 2014 Nov 
				6; 159 (4): 925-39. doi: 10.1016/j.cell.2014.10.002. Epub 2014 
				Oct 23.</LI><LI id="cite-7"><A href="#ref-7">[7]</A> A DNA-Based Archival 
				Storage System ASPLOS ’16 April 2–6, 2016, DOI: 
				http://dx.doi.org/10.1145/2872362.2872397</LI><LI id="cite-8"><A href="#ref-8">[8]</A> Towards practical, 
				high-capacity, low-maintenance information storage in 
				synthesized DNA Nature 494, 77–80 (07 February 2013) 
				doi:10.1038/nature11875</LI><LI id="cite-9"><A href="#ref-9">[9]</A> Robust Chemical 
				Preservation of Digital Information on DNA in Silica with 
				Error-Correcting Codes. Angew. Chem. Int. Ed.. DOI: 
				10.1002/anie.201411378 </LI><LI id="cite-10"><A href="#ref-10">[10]</A> Physicochemical 
				Properties of Cells and Their Effects on Intrinsically 
				Disordered Proteins (IDPs) Chem Rev. 2014 Jul 9; 114(13): 
				6661–6714.</LI><LI id="cite-11"><A href="#ref-11">[11]</A> Extremotolerant 
				tardigrade genome and improved radiotolerance of human cultured 
				cells by tardigrade-unique protein Nature Communications 7, 
				Article number: 12808 (2016) doi:10.1038/ncomms12808</LI><LI id="cite-12"><A href="#ref-12">[12]</A> The cutting edges 
				in DNA repair, licensing, and fidelity: DNA and RNA repair 
				nucleases sculpt DNA to measure twice, cut once. DNA Repair 
				Volume 19, July 2014, Pages 95–107 DOI: 
				10.1016/j.dnarep.2014.03.022</LI></UL></SECTION></ARTICLE></DIV><DIV id="page-footer" class="page-footer"><SECTION><H6>Team</H6><UL><LI><A href="/Team:Groningen/Team">Members</A></LI><LI><A href="/Team:Groningen/Supervisors">Supervisors</A></LI><LI><A href="/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A href="/Team:Groningen/Achievements">Achievements</A></LI><LI><A href="/Team:Groningen/Contact">Contact</A></LI><LI><A href="/Team:Groningen/Jamboree">Jamboree</A></LI></UL></SECTION><SECTION><H6>Project</H6><UL><LI><A href="/Team:Groningen/Tour">Tour</A></LI><LI><A href="/Team:Groningen/Description">Description</A></LI><LI><A href="/Team:Groningen/Design">Design</A></LI><LI><A href="/Team:Groningen/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Groningen/Summary">Summary</A></LI><LI><A href="/Team:Groningen/Applications">Applications</A></LI><LI><A href="/Team:Groningen/Results">Results</A></LI><LI><A href="/Team:Groningen/FuturePerspectives">Future Perspectives</A></LI></UL></SECTION><SECTION class="double"><H6>Biology</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Experiments">Experiments</A></LI><LI><A href="/Team:Groningen/Safety">Safety</A></LI><LI><A href="/Team:Groningen/Decoy">Decoy experiment</A></LI><LI><A href="/Team:Groningen/PhotoswitchableAntibiotics">Photoswitchable antibiotics</A></LI><LI><A href="/Team:Groningen/BrickCharacter">BioBrick Characterization</A></LI><LI><A href="/Team:Groningen/KeyDeletion">Key deletion</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/Proof">Proof of concept</A></LI><LI><A href="/Team:Groningen/Labjournal">Lab journal</A></LI><LI><A href="/Team:Groningen/PlasmidConstruction">Plasmid construction</A></LI><LI><A href="/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="/Team:Groningen/Labjournal/PrimerList">Primer list</A></LI><LI><A href="/Team:Groningen/Notebook">Notebook</A></LI></UL></DIV></SECTION><SECTION><H6>Modelling</H6><UL><LI><A href="/Team:Groningen/Model">Modelling</A></LI><LI><A href="/Team:Groningen/MutationRates">Mutation Rates</A></LI><LI><A href="/Team:Groningen/RandomMutation">Random Mutation</A></LI><LI><A href="/Team:Groningen/AIModel">AI Agent model</A></LI><LI><A href="/Team:Groningen/DecodingFidelity">Decoding Fidelity</A></LI><LI><A href="/Team:Groningen/SpiroModel">Optimal Strain</A></LI></UL></SECTION><SECTION><H6>Computing</H6><UL><LI><A href="/Team:Groningen/Software">Software</A></LI><LI><A href="/Team:Groningen/Encoding">Encoder</A></LI><LI><A href="/Team:Groningen/Decoding">Decoder</A></LI><LI><A href="/Team:Groningen/FullCoding">Full Coding</A></LI></UL></SECTION><SECTION class="double"><H6>Human Practices</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Groningen/HP/Outreach">Outreach</A></LI><LI><A href="/Team:Groningen/HP/Social">Social</A></LI><LI><A href="/Team:Groningen/HP/Legal">Legal</A></LI><LI><A href="/Team:Groningen/HP/FutureScenarios">Future Scenarios</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/HP/Education">Education</A></LI><LI><A href="/Team:Groningen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="/Team:Groningen/Engagement">Engagement</A></LI><LI><A href="/Team:Groningen/HP/Gold">Gold</A></LI><LI><A href="/Team:Groningen/HP/Silver">Silver</A></LI></UL></DIV></SECTION><SECTION><H6>Support</H6><UL><LI><A href="/Team:Groningen/Sponsors">Sponsors</A></LI><LI><A href="/Team:Groningen/Attributions">Attributions</A></LI></UL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>