<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_PhotoswitchableAntibiotics skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/PhotoswitchableAntibiotics</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="content_wrapper"><HEADER id="content-header"><SPAN id="logo-container" class="menu-item"><DIV class="menu-title"><A href="/Team:Groningen">CryptoGE®M</A></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Team</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Team">Members</A><A href="/Team:Groningen/Supervisors">Supervisors</A><A href="/Team:Groningen/Collaborations">Collaborations</A><A href="/Team:Groningen/Achievements">Achievements</A><A href="/Team:Groningen/Contact">Contact</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Project</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Tour"><STRONG>Tour</STRONG></A><A href="/Team:Groningen/Description">Description</A><A href="/Team:Groningen/Design">Design</A><A href="/Team:Groningen/Results">Results</A><A href="/Team:Groningen/Demonstrate">Demonstrate</A><A href="/Team:Groningen/Applications">Applications</A><A href="/Team:Groningen/FuturePerspectives">Future</A><A href="/Team:Groningen/Notebook">Notebook</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Biology</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Experiments">Experiments</A><A href="/Team:Groningen/Parts">Parts</A><A href="/Team:Groningen/Safety">Safety</A><A href="/Team:Groningen/Labjournal">Lab journal</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Computing</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Model">Modelling</A><A href="/Team:Groningen/Software">Software</A><A href="/Team:Groningen/Encoding">Encoding</A><A href="/Team:Groningen/Decoding">Decoding</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Human Practices</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Human_Practices">Human Practices</A><A href="/Team:Groningen/HP/Gold">Gold</A><A href="/Team:Groningen/HP/Silver">Silver</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Support</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Sponsors">Sponsors</A><A href="/Team:Groningen/Attributions">Attributions</A></DIV></DIV></SPAN></HEADER><DIV id="groningen-wrapper"><ARTICLE id="photo-biotics" class="collapse-h2 collapse-conditional"><SECTION><H2>Photoswitchable antibiotics</H2><P>Spirofloxacin is a conjugate of the broad-spectrum antibiotic 
			ciprofloxacin, and a spiropyran with photoswitching activity (see 
			Figures 1 and 2) <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>. The resulting compound has a much higher 
			antibiotic activity after irradiation with 365 nm light as a result 
			of a conformational change in the spiropyran moiety <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>. 
			Spirofloxacin quickly deactivates under natural light, and more 
			slowly in the dark <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>. This prevents the activation of the 
			antibiotic by sequential irradiation with all wavelengths. In 
			principle, photoswitchable antibiotics can be tuned to respond to 
			any wavelength using molecular engineering techniques <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>. </P><P>Spirofloxacin was kindly provided by the research group of Ben L. 
			Feringa. This antibiotic was used by us as a component in a 
			photoresponsive biological lock system. Spores of the key strain 
			(resistant to the antibiotic) are mixed with spores of 
			non-resistant, decoy strains. The antibiotic (in its ‘off’ state) 
			is given to the recipient together with the spores. The recipient 
			must activate the antibiotic with the ‘secret’ wavelength before 
			adding it to the culture medium. As a result, the resistant key 
			strain will grow, while the susceptible decoy strains will not. 
			When DNA is extracted from a correctly treated culture, it will be 
			predominantly the DNA of the key strain. If the culture is not 
			treated correctly, the decoy strains will outcompete the key 
			strain, making it nearly impossible to find the correct DNA 
			sequence.</P><FIGURE><FIGCAPTION>Figure 1 – Modification of ciprofloxacin with 
				spiropyran, resulting in spirofloxacin. Source: <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP> Ciprofloxacin−Photoswitch Conjugates: A 
				Facile Strategy for Photopharmacology</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 2 – Photoisomerization of spirofloxacin in response to UV light. 
				Source: <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP> Ciprofloxacin−Photoswitch Conjugates: A Facile 
				Strategy for Photopharmacology</FIGCAPTION></FIGURE></SECTION><SECTION><H3>Experiments</H3><DIV class="split"><DIV class="left flthree"><P>We carried out two pilot experiments to determine if <EM>Bacillus 
					subtilis</EM> 168 is resistant enough to UV to survive the exposure 
					needed to activate the antibiotic, and to demonstrate the switching 
					activity of the antibiotic.</P><H4>UV resistance test:</H4><UL><LI>Resuspend spores in 50mL LB medium</LI><LI>Transfer to sterile, open beaker (glass and plastic absorb UV) and place under 365nm UV light</LI><LI>Take a 1mL sample for plating and 1mL sample for OD600 measurement at t = 0, 10, 20, 30, 35, 40, 45, 50, 55, and 60 minutes.</LI><LI>Plate 100 µL, 10 µL and 1 µL in duplicate on LB agar. </LI><LI>Culture overnight at 37°C</LI><LI>Compare the growth on the plates</LI></UL></DIV><DIV class="right flone"><TABLE><TBODY><TR><TH>t(min)</TH><TH>OD600</TH></TR><TR><TD>0</TD><TD>0.055</TD></TR><TR><TD>10</TD><TD>0.055</TD></TR><TR><TD>20</TD><TD>0.054</TD></TR><TR><TD>30</TD><TD>0.053</TD></TR><TR><TD>35</TD><TD>0.053</TD></TR><TR><TD>40</TD><TD>0.053</TD></TR><TR><TD>45</TD><TD>0.051</TD></TR><TR><TD>50</TD><TD>0.052</TD></TR><TR><TD>55</TD><TD>0.049</TD></TR><TR><TD>60</TD><TD>0.050</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="split"><FIGURE class="left flone"><FIGCAPTION>Figure 3 – Agar plates from the UV resistance 
					test</FIGCAPTION></FIGURE><FIGURE class="right flone"><FIGCAPTION>Figure 4 – UV resistance test. 60 minute exposure, 
					plated 1 µL culture </FIGCAPTION></FIGURE></DIV><P>There is a lot of growth on all the plates, even the 60 minute 
			plate with 1 µL plated culture, as shown in Figures 3 and 4. This 
			result, along with the OD600 measurements suggest that <EM>Bacillus 
			subtilis</EM> 168 spores are resistant enough to UV (365nm) to survive 
			UV exposure necessary to activate a spiropyran-ciprofloxacin 
			conjugate.</P><H4>Antibiotic activity spot test:</H4><P>To demonstrate that spirofloxacin possesses antibiotic activity, 
			a spot test was carried out. <EM>B. subtilis</EM> 168 was plated on LB agar, 
			and a small amount of spirofloxacin powder was placed in the middle 
			of the plate. There is an area around the antibiotic that has no 
			growth (see Figure 5). This result suggests that even in its 
			non-irradiated state, spirofloxacin has some antibiotic activity.</P><FIGURE><FIGCAPTION>Figure 5 – Antibiotic activity spot test</FIGCAPTION></FIGURE><H4>Photoswitching activity test</H4><DIV class="split"><DIV class="left flone"><P>The activated (365nm irradiated merocyanin state, open ring) spirofloxacin has a 
					higher absorption peak at 590 nm than the deactivated (thermally adapted spiropyran 
					state, closed ring), as reported by Feringa and co-workers <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP> (see 
					Figure 6). </P><P>In order to demonstrate that spirofloxacin undergoes a change in 
					molecular conformation upon irradiation with 365 nm light, we set 
					up the following experiment.</P><UL><LI>Stock solution of spirofloxacin was prepared by dissolving 6.94 mg of spirofloxacin in 10 mL DMSO. Working solutions were prepared by diluting the stock solution in MilliQ H<SUB>2</SUB>O to the appropriate concentration.</LI><LI>Prepared 2 mL [100 µM] spirofloxacin in MilliQ H<SUB>2</SUB>O</LI><LI>Irradiated 1 mL of the solution for 60 minutes under 365 nm light</LI><LI>The other 1 mL was kept at ambient conditions</LI><LI>Measured OD590, using MilliQ H<SUB>2</SUB>O as a blank (open ring form of spirofloxacin absorbs at that wavelength).</LI></UL></DIV><DIV class="right flone"><FIGURE><FIGCAPTION>Figure 6 – Absorption spectrum of spirofloxacin 
						photoisomers. Source: <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP> Ciprofloxacin−Photoswitch Conjugates: 
						A Facile Strategy for Photopharmacology</FIGCAPTION></FIGURE></DIV></DIV><DIV class="split"><DIV class="right flone"><TABLE><TBODY><TR><TH>UV Irradiation </TH><TH>OD590</TH></TR><TR><TD>λ-             </TD><TD>0.014</TD></TR><TR><TD>λ+             </TD><TD>0.108</TD></TR></TBODY></TABLE></DIV><DIV class="left flthree"><P><STRONG>Conclusion:</STRONG> There is a clear difference in 
					absorption after irradiation (λ+). This suggests that a change in 
					molecular conformation has taken place as a result of irradiation 
					with 365 nm light. This result indicates that the photoswitch is 
					active, and undergoes photoisomerization under UV light. </P></DIV></DIV><H4>MIC and MBC tests – Ciprofloxacin, Wild-Type <EM>B. subtilis</EM> 168</H4><P>Minimal Inhibitory Concentration (MIC) is the lowest 
			concentration of antibiotic which prevents growth of bacteria. 
			Minimal Bactericidal Concentration (MBC) is the lowest 
			concentration that kills bacteria. Ciprofloxacin MIC and MBC tests 
			were carried out on wild-type <EM>Bacillus subtilis</EM> 168 to determine 
			the concentration of the antibiotic that this strain could 
			tolerate.</P><P>Two tests were carried out with ciprofloxacin. The growth of <EM>B. subtilis</EM> 168 in a 96-well plate with a range of ciprofloxacin concentrations were monitored for 18 and 24 hours. The positive control was used to construct the growth curve (see Figure 7). For the first test a range of 25 nM to 2,500 nM was used. Results showed that the MIC was somewhere between 100 nM and 200 nM (see Figure 8). The second time a range of 100 nM to 200 nM was tested to determine the MIC more precisely. Results showed that the MIC of ciprofloxacin on <EM>B. subtilis</EM> 168 was 160 - 170 nM (see Figures 9 and 10).</P><P>Now that we obtained the MIC value, we determined the MBC. 10 µL 
			of 160 nM, 170 nM and 180 nM cultures from the 96-well plate were 
			plated on LB agar in duplicate. These were incubated overnight at 37°C. 
			Only the 180 nM plate had no growth, so we take that to be the MBC.</P><H5>Results:</H5><DIV class="split"><FIGURE class="left flone"><FIGCAPTION>Figure 7 – <EM>B. subtilis</EM> 168 growth curve at 37°C</FIGCAPTION></FIGURE><FIGURE class="right flone"><FIGCAPTION>Figure 8 – Growth of <EM>B. subtilis</EM> 168 in the 
					presence of ciprofloxacin (25 nM- 2,500 nM)</FIGCAPTION></FIGURE></DIV><DIV class="split"><FIGURE class="left flone"><FIGCAPTION>Figure 9 – Growth of <EM>B. subtilis</EM> 168 in the 
					presence of ciprofloxacin (100 nM – 200 nM)</FIGCAPTION></FIGURE><FIGURE class="right flone"><FIGCAPTION>Figure 10 – <EM>B. subtilis</EM> - 130, 160, 170 nM 
					ciprofloxacin</FIGCAPTION></FIGURE></DIV><H4>Engineering ciprofloxacin resistance in <EM>Bacillus subtilis</EM> 168</H4><DIV class="split"><DIV class="left flone"><P>For our system, we needed a strain of <EM>B. subtilis</EM> that is 
					resistant to ciprofloxacin (and the spiropyran conjugate, 
					spirofloxacin). Such a strain was not found in literature, so we 
					engineered resistance via two parallel approaches.</P><H5>Quinolone resistance gene <EM>qnrS1</EM></H5><P>We introduced the gene <EM>qnrS1</EM> into our <EM>B. subtilis</EM> strain. The 
					product of this gene alters the surface topology of the enzyme DNA 
					gyrase, which is the target of ciprofloxacin. As a result, the 
					antibiotic can no longer bind to gyrase, and thus does not inhibit 
					DNA transcription. (Learn more about the design of our <A href="https://2016.igem.org/Team:Groningen/Labjournal#Cipro-in-pSB1C3">ciprofloxacin resistance BioBrick</A>)</P><H5>Directed evolution</H5><P>A single colony of <EM>B. subtilis</EM> 168 was found to be growing on a 
					190 nM ciprofloxacin agar plate. This was re-plated onto LB agar 
					medium with increasing antibiotic concentrations. After two months, 
					the strain was growing on 25 µM ciprofloxacin plates (see Figure 11).</P></DIV><DIV class="right flone"><FIGURE><FIGCAPTION>Figure 11 – <EM>B. subtilis</EM> - 130nM, 160 nM, 170 nM 
						ciprofloxacin</FIGCAPTION></FIGURE></DIV></DIV><H4>MIC tests – Ciprofloxacin, wild-type <EM>E. coli</EM> top10, <EM>E. coli</EM> 
			top10 (carrying <EM>qnrS1</EM>) and <EM>B. subtilis</EM> 168 (carrying <EM>qnrS1</EM>)</H4><P>MIC tests were carried out as done previously. The MIC values 
			for the wild-type strains was found to be between 100-130 nM for <EM>Escherichia 
			coli</EM> and 160 - 170 nM for <EM>B. subtilis</EM> (see Figures 12 and 10). MIC for <EM>E. coli</EM> top10 and <EM>B. 
			subtilis</EM> 168 carrying the <EM>qnrS1</EM> ciprofloxacin resistance gene was 
			determined to be between 1000 - 2000 nM, and 400 - 500 nM (see Figures 13 and 14). This is 
			a significant improvement in antibiotic tolerance (approximately 
			10x more resistance for <EM>E. coli</EM> and 2.3x more resistance for <EM>B. 
			subtilis</EM>). </P><H5>Results:</H5><DIV class="split"><FIGURE class="left flone"><FIGCAPTION>Figure 12 – Growth of wild-type <EM>E. coli</EM> top10 in 
					the Presence of Ciprofloxacin </FIGCAPTION></FIGURE><FIGURE class="right flone"><FIGCAPTION>Figure 13 – Growth of <EM>E. coli</EM> top10 carrying <EM>qnrS1</EM> 
					in the Presence of Ciprofloxacin </FIGCAPTION></FIGURE></DIV><P>A MIC test was carried out on three colonies of <EM>B. subtilis</EM> 168 
			transformed with the <EM>qnrS1</EM> resistance gene. Colony 1 was the most 
			resistant, with a MIC between 400 and 500 nM (see Figure 14).</P><FIGURE class="centrate"><FIGCAPTION>Figure 14 – Growth of <EM>B. subtilis</EM> 168 (with <EM>qnrS1</EM>1) 
				- Colony 1 With Ciprofloxacin</FIGCAPTION></FIGURE><DIV class="split"><FIGURE class="left flone"><FIGCAPTION>Figure 15 – Growth of <EM>B. subtilis</EM> 168 (with 
					<EM>qnrS1</EM>) - Colony 2 With Ciprofloxacin</FIGCAPTION></FIGURE><FIGURE class="right flone"><FIGCAPTION>Figure 16 – Growth of <EM>B. subtilis</EM> 168 (with <EM>qnrS1</EM>) 
					- Colony 3 With Ciprofloxacin</FIGCAPTION></FIGURE></DIV><P>A MIC test was carried out on ciprofloxacin resistant <EM>B. 
			subtilis</EM> 168 obtained via directed evolution (see Figure 11). The MIC value was in 
			excess of 2,000 nM (see Figure 17), so a second MIC test was set up using a higher 
			concentrations of ciprofloxacin. </P><FIGURE class="centrate"><FIGCAPTION>Figure 17 – Growth of <EM>B. subtilis</EM> 168 (Resistance 
				by Directed Evolution) With Ciprofloxacin</FIGCAPTION></FIGURE><DIV class="split"><DIV class="left flone"><P>Since the MIC of our ciprofloxacin-resistant <EM>B. subtilis</EM> 168 
					isolate could not be determined in our previous experiment, we set 
					up another MIC test with a higher range of ciprofloxacin 
					concentrations. We examined growth of this isolate in range of 
					ciprofloxacin concentrations from 500 nM to 30,000 nM and compared 
					it to wild-type <EM>B. subtilis</EM> 168 under the same conditions (see 
					Figure 18). The MIC for this isolate was determined to be between 
					20,000 and 30,000 nM (see Figure 19). The wild-type strain grew very poorly 
					even with the lowest concentration of ciprofloxacin (500 nM), never 
					surpassing OD600 = 0.11. In contrast, the resistant <EM>B. subtilis</EM> 
					isolate reached an OD600 = 0.3 in the presence of 500 nM after 
					approximately 12 hours of growth. Cultures with a concentration of 
					17,500 nM reached an OD600 = 0.17 after 18 hours of growth. Even 
					the cultures with 27,500 nM ciprofloxacin showed some growth after 
					18 hours, although very little, reaching an OD600 = 0.12 (which is 
					still more than the wild-type cultures with 500 nM ciprofloxacin). </P><P>To confirm that this was really <EM>Bacillus subtilis</EM>, a PCR was set 
					up, using primers that amplify the 16s rRNA region specific to <EM>B. 
					subtilis</EM>. We obtained a clear bands of identical size for both the resistant isolate and wild-type <EM>B. subtilis</EM> 168 (see Figure 20). To prove conclusively that our isolate is in fact <EM>B. subtilis</EM>, the PCR products were sent for sequencing. This isolate would serve 
					as a perfect starting point for the engineering of a highly 
					resistant <EM>B. subtilis</EM> strain.</P></DIV><DIV class="right flone"><FIGURE><FIGCAPTION>Figure 20 – Confirming the identity of our strain 
						with PCR</FIGCAPTION></FIGURE></DIV></DIV><DIV class="split"><FIGURE><FIGCAPTION>Figure 18 – Growth of <EM>B. subtilis</EM> 168 (wild-type) 
					With Ciprofloxacin</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 19 – Growth of <EM>B. subtilis</EM> 168 
					(Resistance by Directed Evolution) With 
					Ciprofloxacin</FIGCAPTION></FIGURE></DIV><P><EM>B. subtilis</EM> 168 appears to be naturally more resistant to 
			ciprofloxacin than <EM>E. coli</EM> top10, but not as resistant as <EM>E. coli</EM> 
			carrying the <EM>qnrS1</EM> resistance gene (see Figure 21).</P><FIGURE class="centrate"><FIGCAPTION>Figure 21 – Growth of <EM>E. coli</EM> top10 (wild-type and 
				with <EM>qnrS1</EM>) and <EM>B. subtilis</EM> 168 (wild-type) with 130 nM 
				ciprofloxacin</FIGCAPTION></FIGURE><H4>MIC tests – spirofloxacin</H4><H5>Test 1</H5><DIV class="split"><DIV class="left flone"><P>Nine tests were carried out to determine the MIC of 
					spirofloxacin on <EM>B. subtilis</EM> 168, <EM>E. coli</EM> BL4, <EM>E. coli</EM> DH5α and <EM>E. 
					coli</EM> CS1562. The test with <EM>E. coli</EM> were done due to the 
					inconsistent results we obtained with <EM>B. subtilis</EM>. Spirofloxacin 
					stock solution was diluted in LB to the working concentrations. 3 
					mL cultures with different concentrations of spirofloxacin were 
					inoculated and incubated overnight in the dark at 37°C in a shaking 
					waterbath. 	OD600 was measured the next day.</P><UL><LI>Strain: <EM>B. subtilis</EM> 168	</LI><LI>10 µL OD=0.001 spores per 2 mL culture	</LI><LI>UV (365nm) irradiation durations: 0, 30, 45, 60 minutes	</LI><LI>+ control: spores, 0 µM spirofloxacin</LI><LI>- control: no spores, 5 µM spirofloxacin</LI></UL><P><STRONG>Conclusion:</STRONG> Repeat with higher spirofloxacin 
					concentrations</P></DIV><DIV class="right flone"><TABLE><TBODY><TR><TH>[Spirofloxacin] nM</TH><TH>OD600= (λ-)       </TH><TH>OD600= (λ+)       </TH></TR><TR><TD>100      </TD><TD>overgrown</TD><TD>overgrown</TD></TR><TR><TD>200      </TD><TD>overgrown</TD><TD>overgrown</TD></TR><TR><TD>300      </TD><TD>overgrown</TD><TD>overgrown</TD></TR><TR><TD>400</TD><TD>overgrown</TD><TD>overgrown</TD></TR><TR><TD>500      </TD><TD>overgrown</TD><TD>overgrown</TD></TR><TR><TD>700      </TD><TD>overgrown</TD><TD>overgrown</TD></TR><TR><TD>1000     </TD><TD>overgrown</TD><TD>overgrown</TD></TR><TR><TD>1250     </TD><TD>overgrown</TD><TD>overgrown</TD></TR><TR><TD>1500     </TD><TD>overgrown</TD><TD>overgrown</TD></TR><TR><TD>2000     </TD><TD>overgrown</TD><TD>overgrown</TD></TR></TBODY></TABLE></DIV></DIV><H5>Test 2</H5><DIV class="split"><DIV class="left flone"><UL><LI>Strain: <EM>B. subtilis</EM> 168	                       </LI><LI>10 µL OD=0.001 spores per  2 mL culture	       </LI><LI>UV (365nm) irradiation durations: 0, 60 minutes</LI><LI>+ control: spores, 0 µM spirofloxacin          </LI><LI>- control: no spores, 5 µM spirofloxacin	   </LI></UL><P><STRONG>Conclusion:</STRONG></P><UL><LI>A lot of growth in all cultures except -controls.</LI><LI>Measured OD of 2 µM and 10 µM cultures. <EM>B. subtilis</EM> 168 appears to grow slightly better with spirofloxacin (which is counterintuitive, given that it's an antibiotic).</LI><LI>There is slightly less growth in the 10 µM cultures than in the 2 µM cultures.</LI><LI>There is a slight difference in activity between λ+ and λ- cultures. λ+ cultures had less growth, as expected, but the difference was not as big as expected.</LI><LI>Difference in + controls can be attributed to UV-induced damage of the cells.</LI></UL></DIV><DIV class="right flone"><TABLE><TBODY><TR><TH>[Spirofloxacin] µM</TH><TH>OD600= (λ-)</TH><TH>OD600= (λ+)</TH></TR><TR><TD> - control</TD><TD>0    </TD><TD>0.002</TD></TR><TR><TD>2         </TD><TD>1.512</TD><TD>1.364</TD></TR><TR><TD>2.5       </TD><TD> -   </TD><TD> -   </TD></TR><TR><TD>3         </TD><TD> -   </TD><TD> -   </TD></TR><TR><TD>3.5       </TD><TD> -   </TD><TD> -   </TD></TR><TR><TD>4         </TD><TD> -   </TD><TD> -   </TD></TR><TR><TD>4.5       </TD><TD> -   </TD><TD> -   </TD></TR><TR><TD>5         </TD><TD> -   </TD><TD> -   </TD></TR><TR><TD>5.5       </TD><TD> -   </TD><TD> -   </TD></TR><TR><TD>6         </TD><TD> -   </TD><TD> -   </TD></TR><TR><TD>10        </TD><TD>1.388</TD><TD>1.128</TD></TR><TR><TD> + control</TD><TD>1.06 </TD><TD>0.821</TD></TR></TBODY></TABLE></DIV></DIV><H5>Test 3</H5><DIV class="split"><DIV class="left flone"><UL><LI>Strains: <EM>B. subtilis</EM> 168, <EM>E. coli</EM> DH5α	                         </LI><LI>10 µL OD=0.001 <EM>B. subtilis</EM> spores per  2 mL culture	             </LI><LI>UV (365nm) irradiation durations: 0, 60 minutes	                 </LI><LI>+ control: spores, 0 µM spirofloxacin                            </LI><LI>- control: no spores, 10 µM spirofloxacin	                     </LI><LI>DMSO control: 10% DMSO + spores	                                 </LI><LI><EM>E. coli</EM> control: 2 µL overnight <EM>E. coli</EM> culture per  2 mL culture   i</LI></UL><P><STRONG>Conclusion:</STRONG></P><UL><LI>Growth is somewhat inhibited at 100 µM, indicating antibiotic activity.</LI><LI>There is slightly more growth after irradiation, which is unexpected and contradicts our previous result. </LI><LI>DMSO appears to affect bacterial growth at this concentration, but not to the extent observed with 100 µM.</LI><LI><EM>E. coli</EM> appears to respond differently to the antibiotic. Growth in both λ+ and λ- cultures, but less in the λ+ culture, as expected.</LI></UL></DIV><DIV class="right flone"><TABLE><TBODY><TR><TH>[Spirofloxacin] µM</TH><TH>OD600= (λ-)       </TH><TH>OD600= (λ+)</TH></TR><TR><TD>10             </TD><TD>0.885          </TD><TD>0.905          </TD></TR><TR><TD>20             </TD><TD>0.76           </TD><TD>0.68           </TD></TR><TR><TD>100            </TD><TD>0.015          </TD><TD>0.076          </TD></TR><TR><TD> - control     </TD><TD>0.002          </TD><TD>0.01           </TD></TR><TR><TD> + control     </TD><TD>0.89           </TD><TD>1.086          </TD></TR><TR><TD>DMSO control   </TD><TD>0.502          </TD><TD>0.218          </TD></TR><TR><TD><EM>E. coli</EM> control</TD><TD>0.772          </TD><TD>0.663          </TD></TR></TBODY></TABLE></DIV></DIV><H5>Test 4</H5><DIV class="split"><DIV class="left flone"><P>LB medium absorbs heavily in the UV region of the spectrum. To 
					overcome this issue we diluted the spirofloxacin in MilliQ H<SUB>2</SUB>O, and 
					irradiated it before to adding to the cultures.	</P><UL><LI>Strains: <EM>B. subtilis</EM> 168</LI><LI>10 µL OD=0.001 <EM>B. subtilis</EM> spores per  2 mL culture</LI><LI>UV (365nm) irradiation durations: 0, 60 minutes</LI><LI>+ control: spores, 0 µM spirofloxacin</LI><LI>- control: no spores, 10 µM spirofloxacin</LI></UL></DIV><DIV class="right flone"><TABLE><TBODY><TR><TH>[Spirofloxacin] µM</TH><TH>OD600= (λ-)       </TH><TH>OD600= (λ+)</TH></TR><TR><TD>0.25      </TD><TD>0.474</TD><TD>0.543</TD></TR><TR><TD>0.5       </TD><TD>0.585</TD><TD>0.703</TD></TR><TR><TD>1         </TD><TD>0.769</TD><TD>0.667</TD></TR><TR><TD>5         </TD><TD>0.775</TD><TD>0.718</TD></TR><TR><TD>10        </TD><TD>0.685</TD><TD>0.907</TD></TR><TR><TD> - control</TD><TD>0.155</TD><TD>0.217</TD></TR><TR><TD> + control</TD><TD>0.439</TD><TD>0.62 </TD></TR></TBODY></TABLE></DIV></DIV><H5>Test 5</H5><P>There was contamination in our last - control. This time, 
			additional steps were taken to minimize contamination and maximize 
			UV exposure. Every surface was thoroughly cleaned with ethanol. 
			Tips and LB were already autoclaved, but were autoclaved a second 
			time right before the experiment. Spirofloxacin was irradiated on a 
			sterile petri dish and filter sterilized with a 0.22 µm Whatman filter before being added to the 
			cultures.</P><DIV class="split"><DIV class="left flone"><UL><LI>Strains: <EM>B. subtilis</EM> 168, <EM>E. coli</EM> BL4	             </LI><LI>10 µL OD=0.001 <EM>B. subtilis</EM> spores per  2 mL culture	 </LI><LI>2 µL overnight <EM>E. coli</EM> culture per  2 mL culture	 </LI><LI>UV (365nm) irradiation durations: 0, 60 minutes	     </LI><LI>+ control: spores, 0 µM spirofloxacin                </LI><LI>- control: no spores, 10 µM spirofloxacin	         </LI><LI><EM>E. coli</EM> control: 10 µM spirofloxacin		         </LI></UL></DIV><DIV class="right flone"><TABLE><TBODY><TR><TH>[Spirofloxacin] µM</TH><TH>OD600= (λ-)</TH><TH>OD600= (λ+)</TH></TR><TR><TD>10        </TD><TD>2.094</TD><TD>2.03 </TD></TR><TR><TD> - control</TD><TD>0.01 </TD><TD>0.009</TD></TR><TR><TD> + control</TD><TD>2.746</TD><TD>2.8  </TD></TR><TR><TD><EM>E. coli</EM> control</TD><TD>0.021</TD><TD>0.012</TD></TR></TBODY></TABLE></DIV></DIV><P><STRONG>Conclusion: </STRONG></P><UL><LI>No significant difference in growth between λ- and λ+ cultures, or the +/- controls	</LI><LI>Growth of <EM>E. coli</EM> is inhibited, but appears to be slightly higher with λ-, which is an expected result.	</LI></UL><H5>Test 6</H5><P>Due to the inconsistent results we observed, we set up an 
			experiment to test the activity of spirofloxacin on <EM>E. coli</EM> 
			CS1562.	This is the strain used by the authors of the paper where 
			spirofloxacin was first reported <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>.	</P><DIV class="split"><DIV class="left flone"><UL><LI>Strain: <EM>E. coli</EM> CS1562	                           </LI><LI>7 µL overnight <EM>E. coli</EM> culture per  2 mL culture   </LI><LI>UV (365nm) irradiation durations: 0, 60 minutes	   </LI><LI>+ control: <EM>E. coli</EM>, 0 µM spirofloxacin             </LI><LI>- control: no cells, 1.25 µM spirofloxacin	       </LI><LI>Contamination control: 0 µM spirofloxacin, no cells</LI></UL><P><STRONG>Conclusion:</STRONG><UL><LI>No significant difference in growth between λ- and λ+ cultures, except with .25 µM (λ+ grows better) and 1.25 µM (opposite effect). Results are contradictory.</LI><LI>No significant difference in growth between different spirofloxacin concentrations, except at 10 µM where growth was inhibited.</LI><LI>+/- controls look as expected.</LI><LI>There is contamination in the contamination control cultures, so it is uncertain if these results are reliable.</LI></UL></P></DIV><DIV class="right flone"><TABLE><TBODY><TR><TH>[Spirofloxacin] µM</TH><TH>OD600= (λ-)</TH><TH>OD600= (λ+)</TH></TR><TR><TD>0.25</TD><TD>0.177</TD><TD>1.036</TD></TR><TR><TD>0.625</TD><TD>1.265</TD><TD>1.289</TD></TR><TR><TD>1</TD><TD>1.244</TD><TD>1.277</TD></TR><TR><TD>1.25</TD><TD>1.231</TD><TD>0.515</TD></TR><TR><TD>2</TD><TD>1.238</TD><TD>1.309</TD></TR><TR><TD>10</TD><TD>0.016</TD><TD>0.015</TD></TR><TR><TD> - control</TD><TD>0.009</TD><TD>1.131</TD></TR><TR><TD> + control</TD><TD>0.864</TD><TD>0.535</TD></TR><TR><TD>Contamination control</TD><TD>0.522</TD><TD>0.57</TD></TR></TBODY></TABLE></DIV></DIV><H5>Test 7</H5><DIV class="split"><DIV class="left flone"><UL><LI>Strain: <EM>E. coli</EM> CS1562	</LI><LI>1 µL overnight <EM>E. coli</EM> culture per  2 mL culture</LI><LI>UV (365nm) irradiation durations: 0, 60 minutes (filter sterilized after irradiation)	</LI><LI>+ control: <EM>E. coli</EM>, 0 µM spirofloxacin</LI><LI>- control: no cells, 5 µM spirofloxacin	</LI><LI>Contamination control: 0 µM spirofloxacin, no cells.</LI></UL><P><STRONG>Conclusion:</STRONG></P><UL><LI>Most λ+ cultures have slightly less growth than their counterpart λ- cultures, which is unexpected.</LI><LI>No significant difference in growth between different spirofloxacin concentrations</LI><LI>This time there is growth in the 10 µM cultures</LI><LI>λ-, 5 µM appears to be an anomaly. Perhaps we forgot to add spirofloxacin to that culture?</LI><LI>All controls look as expected.</LI></UL></DIV><DIV class="right flone"><TABLE><TBODY><TR><TH>[Spirofloxacin] µM</TH><TH>OD600= (λ-)</TH><TH>OD600= (λ+)</TH></TR><TR><TD>0.5  </TD><TD>1.104</TD><TD>0.766</TD></TR><TR><TD>0.625</TD><TD>0.28 </TD><TD>0.78 </TD></TR><TR><TD>1.25 </TD><TD>1.216</TD><TD>1.101</TD></TR><TR><TD>3    </TD><TD>1.207</TD><TD>1.094</TD></TR><TR><TD>5    </TD><TD>0.002</TD><TD>1.083</TD></TR><TR><TD>7    </TD><TD>1.24 </TD><TD>1.135</TD></TR><TR><TD>8    </TD><TD>1.134</TD><TD>1.102</TD></TR><TR><TD>9    </TD><TD>1.102</TD><TD>1.081</TD></TR><TR><TD>10   </TD><TD>0.929</TD><TD>0.89 </TD></TR><TR><TD> - control</TD><TD>0.002     </TD><TD>-0.002    </TD></TR><TR><TD> + control</TD><TD>0.711     </TD><TD>0.854     </TD></TR><TR><TD>Contamination control</TD><TD>0.015</TD><TD>0.007</TD></TR></TBODY></TABLE></DIV></DIV><H5>Test 8</H5><DIV class="split"><DIV class="left flone"><UL><LI>Strain: <EM>E. coli</EM> CS1562</LI><LI>1 µL overnight <EM>E. coli</EM> culture / mL	</LI><LI>UV (365nm) irradiation durations: 0, 60 minutes (filter sterilized after irradiation)</LI><LI>+ control: <EM>E. coli</EM>, 0 µM spirofloxacin</LI><LI>- control: no cells, 5 µM spirofloxacin</LI><LI>Contamination control: 0 µM spirofloxacin, no cells</LI></UL></DIV><DIV class="right flone"><TABLE><TBODY><TR><TH>[Spirofloxacin] µM</TH><TH>OD600= (λ-)       </TH><TH>OD600= (λ+)       </TH></TR><TR><TD>5         </TD><TD>0.003     </TD><TD>0.003     </TD></TR><TR><TD>10        </TD><TD>0.005     </TD><TD>0.741     </TD></TR><TR><TD> - control</TD><TD>-0.002    </TD><TD>0         </TD></TR><TR><TD> + control</TD><TD>0.999     </TD><TD>1.036     </TD></TR><TR><TD>Contamination control</TD><TD>0     </TD><TD>-0.004</TD></TR></TBODY></TABLE></DIV></DIV><P><STRONG>Conclusion: 	</STRONG></P><UL><LI>No growth in both 5 µM cultures, but there is growth in 10 µM, λ+ culture, which is unexpected.</LI><LI>There is growth in the 10 µM, λ+ culture, but not its counterpart λ- culture. This is the opposite of the expected result.</LI><LI>All controls look as expected.</LI></UL><H5>Test 9</H5><DIV class="split"><DIV class="left flone"><UL><LI>Strain: <EM>E. coli</EM> CS1562</LI><LI>1 µL overnight <EM>E. coli</EM> culture / mL</LI><LI>UV (365nm) irradiation durations: 0, 30 minutes (filter sterilized after irradiation)</LI><LI>+ control: <EM>E. coli</EM>, 0 µM spirofloxacin</LI><LI>- control: no cells, 5 µM spirofloxacin	</LI><LI>Contamination control: 0 µM spirofloxacin, no cells	</LI></UL><P><STRONG>Conclusion: 	</STRONG></P><UL><LI>No growth in any of the cultures, except with 0.625 µM spirofloxacin.</LI><LI>No significant difference in growth between .625 µM, λ- and .625 µM λ+ cultures</LI><LI>Controls do not look good.</LI></UL></DIV><DIV class="right flone"><TABLE><TBODY><TR><TH>[Spirofloxacin] µM</TH><TH>OD600= (λ-)</TH><TH>OD600= (λ+)</TH></TR><TR><TD>0.625     </TD><TD>0.294     </TD><TD>0.3       </TD></TR><TR><TD>1.25      </TD><TD>0.001     </TD><TD>0.004     </TD></TR><TR><TD>2         </TD><TD>0.001     </TD><TD>0         </TD></TR><TR><TD>4         </TD><TD>0.001     </TD><TD>0.002     </TD></TR><TR><TD>5         </TD><TD>0.001     </TD><TD>0.001     </TD></TR><TR><TD>7         </TD><TD>0         </TD><TD>0.001     </TD></TR><TR><TD>9         </TD><TD>0         </TD><TD>0.006     </TD></TR><TR><TD>10        </TD><TD>0.001     </TD><TD>0.003     </TD></TR><TR><TD> - control</TD><TD>0.366     </TD><TD>0.322     </TD></TR><TR><TD> + control</TD><TD>0.727     </TD><TD>0.541     </TD></TR><TR><TD>Contamination control</TD><TD>0.163</TD><TD>0.292</TD></TR></TBODY></TABLE></DIV></DIV><H4>Overall conclusion – spirofloxacin:</H4><P>Our results are inconsistent and sometimes contradictory. Some 
			of the difference between λ- and λ+ cultures could be attributed to 
			the difference in absorption of the two spirofloxacin photoisomers, 
			or due to the damaging effect of UV light on DNA. We were unable to 
			achieve a large enough difference in growth between cultures with 
			irradiated and non-irradiated spirofloxacin to be able to 
			incorporate it in our project. However, under the right conditions 
			(which would take longer to determine), or with further engineering 
			of the compound, we believe spirofloxacin could be used as a 
			component in a photoresponsive biological lock system.</P></SECTION><SECTION><H3>Conclusion and Discussion</H3><P>Photoswitchable antibiotics present an interesting option for a 
			biological lock system. Although we did not succeed in 
			incorporating spirofloxacin in our project due to the insufficient 
			difference in bioactivity between the two photoisomers, we believe 
			that it could be done with further optimization of our experimental 
			setup, or with further engineering of the compound. In order to 
			endow our <EM>B. subtilis</EM> 168 strain with resistance to ciprofloxacin 
			(and therefore, to spirofloxacin), we introduced the <EM>qnrS1</EM> 
			fluoroquinolone resistance gene. This gene was part of a resistance 
			cassette consisting of the pATPI promoter, a ribosomal binding 
			site, <EM>qnrS1</EM>, and a double terminator. The cassette was assembled in 
			<EM>E. coli</EM> top10, making it approximately 10 times more resistant than 
			the wild-type. When the cassette was introduced into <EM>B. subtilis</EM>, 
			resistance increased by a factor of 2.35. The difference in 
			observed resistance increase after transformation with <EM>qnrS1</EM> could 
			be attributed to the fact that <EM>qnrS1</EM> codon usage was optimized for 
			<EM>E. coli</EM>, and therefore did not function as well in <EM>B. subtilis</EM>. 
			Finally, a <EM>B. subtilis</EM> 168 isolate resistant to ciprofloxacin was 
			obtained in a parallel approach via directed evolution. This 
			isolate was found to tolerate ciprofloxacin concentrations well 
			over 20,000 nM, and is an ideal starting point for further 
			engineering of ciprofloxacin resistance. In the future, this 
			isolate could be transformed with the <EM>qnrS1</EM> and <EM>aac(6')-Ib-cr</EM> 
			resistance genes to obtain a strain of <EM>B. subtilis</EM> that is highly 
			resistant to ciprofloxacin, and possible other fluoroquinolone 
			antibiotics.</P></SECTION><SECTION><H6>References</H6><UL><LI id="cite-1"><A href="#ref-1">[1]</A> Ciprofloxacin−Photoswitch Conjugates: A Facile Strategy for Photopharmacology Velema, Hansen, Lerch, Driessen, Szymanski, Feringa  Bioconjugate Chem. 2015, 26, 2592−2597    DOI: 10.1021/acs.bioconjchem.5b00591 </LI><LI id="cite-2"><A href="#ref-2">[2]</A> Photoswitching azo compounds in vivo with red light. J Am Chem Soc. 2013 Jul 3;135(26):9777-84. doi: 10.1021/ja402220t. Epub 2013 Jun 21.</LI></UL></SECTION></ARTICLE></DIV><DIV id="page-footer" class="page-footer"><SECTION><H6>Team</H6><UL><LI><A href="/Team:Groningen/Team">Members</A></LI><LI><A href="/Team:Groningen/Supervisors">Supervisors</A></LI><LI><A href="/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A href="/Team:Groningen/Achievements">Achievements</A></LI><LI><A href="/Team:Groningen/Contact">Contact</A></LI><LI><A href="/Team:Groningen/Jamboree">Jamboree</A></LI></UL></SECTION><SECTION><H6>Project</H6><UL><LI><A href="/Team:Groningen/Tour">Tour</A></LI><LI><A href="/Team:Groningen/Description">Description</A></LI><LI><A href="/Team:Groningen/Design">Design</A></LI><LI><A href="/Team:Groningen/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Groningen/Summary">Summary</A></LI><LI><A href="/Team:Groningen/Applications">Applications</A></LI><LI><A href="/Team:Groningen/Results">Results</A></LI><LI><A href="/Team:Groningen/FuturePerspectives">Future Perspectives</A></LI></UL></SECTION><SECTION class="double"><H6>Biology</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Experiments">Experiments</A></LI><LI><A href="/Team:Groningen/Safety">Safety</A></LI><LI><A href="/Team:Groningen/Decoy">Decoy experiment</A></LI><LI><A href="/Team:Groningen/PhotoswitchableAntibiotics">Photoswitchable antibiotics</A></LI><LI><A href="/Team:Groningen/BrickCharacter">BioBrick Characterization</A></LI><LI><A href="/Team:Groningen/KeyDeletion">Key deletion</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/Proof">Proof of concept</A></LI><LI><A href="/Team:Groningen/Labjournal">Lab journal</A></LI><LI><A href="/Team:Groningen/PlasmidConstruction">Plasmid construction</A></LI><LI><A href="/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="/Team:Groningen/Labjournal/PrimerList">Primer list</A></LI><LI><A href="/Team:Groningen/Notebook">Notebook</A></LI></UL></DIV></SECTION><SECTION><H6>Modelling</H6><UL><LI><A href="/Team:Groningen/Model">Modelling</A></LI><LI><A href="/Team:Groningen/MutationRates">Mutation Rates</A></LI><LI><A href="/Team:Groningen/RandomMutation">Random Mutation</A></LI><LI><A href="/Team:Groningen/AIModel">AI Agent model</A></LI><LI><A href="/Team:Groningen/DecodingFidelity">Decoding Fidelity</A></LI><LI><A href="/Team:Groningen/SpiroModel">Optimal Strain</A></LI></UL></SECTION><SECTION><H6>Computing</H6><UL><LI><A href="/Team:Groningen/Software">Software</A></LI><LI><A href="/Team:Groningen/Encoding">Encoder</A></LI><LI><A href="/Team:Groningen/Decoding">Decoder</A></LI><LI><A href="/Team:Groningen/FullCoding">Full Coding</A></LI></UL></SECTION><SECTION class="double"><H6>Human Practices</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Groningen/HP/Outreach">Outreach</A></LI><LI><A href="/Team:Groningen/HP/Social">Social</A></LI><LI><A href="/Team:Groningen/HP/Legal">Legal</A></LI><LI><A href="/Team:Groningen/HP/FutureScenarios">Future Scenarios</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/HP/Education">Education</A></LI><LI><A href="/Team:Groningen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="/Team:Groningen/Engagement">Engagement</A></LI><LI><A href="/Team:Groningen/HP/Gold">Gold</A></LI><LI><A href="/Team:Groningen/HP/Silver">Silver</A></LI></UL></DIV></SECTION><SECTION><H6>Support</H6><UL><LI><A href="/Team:Groningen/Sponsors">Sponsors</A></LI><LI><A href="/Team:Groningen/Attributions">Attributions</A></LI></UL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>