Team:Groningen/KeyDeletion
CryptoGE®M
Team
Members Supervisors Collaborations Achievements Contact
Project
Tour Description Design Results Demonstrate Applications Future Notebook
Biology
Experiments Parts Safety Lab journal
Computing
Modelling Software Encoding Decoding
Human Practices
Human Practices Gold Silver
Support
Sponsors Attributions
Key deletion
nucA key deletion
Another biological security layer is provided by our key deletion    system. We have been working on two different approaches. Here we are    presenting the first approach, which makes use of the BioBricks in the    registry. Central to this approach is the nuclease nucA from BioBrick BBa_K729004 . This system will degrade the genomic DNA and thereby  the    key from the genome if no special treatment is applied. For the design    of a nucA key deletion system we made use of the following BioBricks: BBa_R0040 _tetR, BBa_K729004 _nuclease, BBa_B0030 _RBS and BBa_B0015 _terminator. During the cloning process we expected to obtain    the following BioBricks: RBS-nuclease, tetR-RBS-nuclease and    tetR-RBS-nuclease-terminator. The key deletion is controlled by the    constitutively on promoter PtetR. RNA polymerase binds to the DNA    sequence and leads to the expression of the nuclease. If tetracycline    is added, it functions as a repressor, binds to the promoter and thus    leads to the inhibition of the transcription of the nuclease. If an    unauthorized party tries to grow the key-containing spores in regular    growth medium without the addition of tetracycline, the kill switch    will be active and the DNA will be digested by the nuclease. Therefore    the key sequence will be lost and the encrypted message stays safe.    Only an authorized user of CryptoGErM knows that to grow the B. subtilis , tetracycline must be present, and will be able to get the key sequence. During the    construction of our nucA key deletion system we encountered the problem    that the BBa_K729004 has an incorrect prefix and suffix. See plasmid    construction . Therefore we wanted to improve this part with the    correct prefix and suffix. Sadly we failed to complete this task.
CRISPR key deletion
Another biological security layer is provided by our key deletion    system. We have been working on two different approaches. Here we are    presenting the second approach, which makes use of a CRISPR/Cas9 system    which will delete the key from the genome if no special treatment is    applied. Addition of atc or tetracycline will stop the Cas9 expression.
This system is highly flexible and biological layers can easily be    added or modified to the wishes of the user. An other goal of the    biological safety layer is to prevent the release of the genetically    modified bacteria carrying our encrypted message or key. The message    and key are integrated into the amyE locus of the Bacillus subtilis genome, with the use of the BBa_K823023 B. subtilis integration vector    . (See integration) This is advantageous, because it has been    demonstrated that the amyE locus can be deleted by using the    CRISPR/Cas9 system [1] .
The Clustered Regularly Interspaced Short Palindromic Repeat    (CRISPR)-associated (Cas) systems are a part of the bacterial adaptive    immune system. Recently a genome editing tool has been designed based    on a type II CRISPR/Cas9 system from Streptococcus pyogenes [2] . The    CRISPR-Cas system needs an additional sequence next to the protospacer    to distinguish the difference between it’s own DNA and “foreign” DNA.    This sequence is the protospacer adjacent motif, also known as PAM    sequence. [3] The PAM motif is essential for cleavage and makes sure    that nearly any DNA can be targeted by crRNA, assisted by    trans-activating crRNA (tracrRNA) cas9 to bind to the protospacer with    the PAM sequence and cleaves the double-strand DNA (dsDNA). The fusion    of crRNA and tracrRNA is sgRNA. [4]
Fig 1. CRISPR/Cas 9 Genome Editing tool. The Cas9     endonuclease (blue) is targeted to DNA by a guide RNA. Target     recognition is facilitated by the protospacer-adjacent motif (PAM).
Fig 2. CRISPR/Cas 9 Genome Editing Representation.
In our project we used a single-plasmid system to perform efficient    genome editing of Bacillus subtilis. The plasmid (pJOE8999) is used as    a shuttle vector with - a pUC origin of replication for E. coli, a    kanamycin resistance gene and a temperature-sensitive replication    origin from the plasmid pE194ts. [1] This plasmid carries the Cas9 gene    under the transcriptional regulation of a mannose-inducible promoter    PmanP. The sgRNA is transcribed under a semisynthetic promoter PvanP,    interrupted by the loop terminator and the T7 promoter (T7P). The Cas9    and sgRNA genetic elements allow the plasmid to be used a genome    editing tool, see Figure 2.
Fig 3. Plasmid pJOE8999 redesigned for our project.
For application in our project, the idea was to replace the PmanP    with the tetracycline-repressible promoter PtetR. The pTetR promoter is    constitutively on and is repressed by TetR. The repression can be    inhibited by the addition of tetracycline [5] .
With the help of the sgRNA Designer tool (Broad Institute) we were    able to design the 20-nucleotide spacer sequence of the sgRNA. This    sgRNA guides the Cas9 to its target, in our case the key or the    message [1] . The 700-bp amplified from B. subtilis serves as the DNA    template (PAM-motif) for amyE.
If the spores containing the message or the key escape into the    environment, the CRISPR/Cas9 system will be activated due to the    constitutively on pTetR and the targeted area, which contains the key    sequence in our case, will be deleted. The same applies if the spores    are in possession by unauthorized parties who are unaware of adding    tetracycline to inactivate the pTetR promoter. This addresses the    issues pertaining to access of the message and key by unauthorized    parties, and also prevents genetically modified bacteria from spreading    in the environment. Therefore, this system has potential to be applied    in any other genetically modified B. subtilis.
References
[1] Altenbucher J. Editing of the Bacillus Subtilis Genome by the CRISPR-Cas 9 system. Applied and Environmental Microbiology Vol 82, 17
[2] Selle K, Barrangou R. 2015. Harnessing CRISPR-Cas systems for bacterial genome editing. Trends Microbiol 23:225–232.
[3] Jiang W, Bikard D, Cox D, Zhang F, Marraffini LA. 2013. RNA-guided    editing of bacterial genomes using CRISPR-Cas systems. Nat Biotechnol    31:233–239.
[4] Jinek M, Chylinski K, Fonfara I, Hauer M, Doudna JA, Charpentier E.    2012. A programmable dual-RNA-guided DNA endonuclease in adaptive    bacterial immunity. Science 337:816–821.
[5] TetR Repressible Promoter: BBa_R0040
Team
Members
Supervisors
Collaborations
Achievements
Contact
Jamboree
Project
Tour
Description
Design
Demonstration
Summary
Applications
Results
Future Perspectives
Biology
Experiments
Safety
Decoy experiment
Photoswitchable antibiotics
BioBrick Characterization
Key deletion
Proof of concept
Lab journal
Plasmid construction
Protocols
Primer list
Notebook
Modelling
Modelling
Mutation Rates
Random Mutation
AI Agent model
Decoding Fidelity
Optimal Strain
Computing
Software
Encoder
Decoder
Full Coding
Human Practices
Human Practices
Outreach
Social
Legal
Future Scenarios
Education
Integrated Practices
Engagement
Gold
Silver
Support
Sponsors
Attributions
