Team:Groningen/Labjournal
CryptoGE®M
Team
Members Supervisors Collaborations Achievements Contact
Project
Tour Description Design Results Demonstrate Applications Future Notebook
Biology
Experiments Parts Safety Lab journal
Computing
Modelling Software Encoding Decoding
Human Practices
Human Practices Gold Silver
Support
Sponsors Attributions
Labjournal
Here you can find how we constructed our BioBricks and other plasmids which were used during our project, next you can find protocols which we followed during our experiments and finally you can see our primer list with every primer used.
Plasmid construction
The following plasmids were constructed mainly during summer/autumn 2016.
Key sequence in pDR111
The key sequence was ordered as a gBlock from IDT to construct the key B.   subtilis strain. In a first approach this was done with the pDR111 B.   subtilis integration plasmid, which can also be amplified by E. coli.   Therefore the first step was to clone the key sequence into the pDR111 in E. coli. pDR111 is an integration plasmid which can be integrated into the B. subtilis genome.   It replaces amyE gene by double cross-over, which is necessary for production of alpha-amylase   (see Subtiwiki.uni-goettingen.de )       with desired insert which is located between amyE front flanking region and amyE back flanking region. See the plasmid map below. We used this plasmid for integration of our key sequence into the B. subtilis 168 sub+. The other approach made use of the BioBrick integration plasmid BBa_K823023 (Key sequence in BBa_K823023) .
PCR
Experiment:
31/08/16: Amplification of the key sequence from the gBlock from IDT (see gBlocks protocol) with the primers F key only amplify and R key only amplify (see primer list) was done. The correct size of the     key PCR product of 144 bp was checked with DNA electrophoresis.
PCR mixture:
50 μl PCR assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gels see    following protocol .
Figure 1. DNA electrophoresis of the key PCR product     sample 1-3. The correct size is 144 bp.
Conclusion:
PCR of the key sequence from the IDT gBlock was successful. We could see the correct band size which was 144 bp.
Procedure after gel validation:
PCR product was subsequently cleaned with the kit (PCR Purification Kit – Jena Bioscience) .
Restriction digestion
Experiment:
31/08/16: On this day restriction digestion of the PCR product of the key    sequence and pDR111 integration plasmid was performed. The key sequence    as an insert was cut with SalI and HindIII restriction enzymes. The    integration plasmid pDR111 as a vector was cut with exactly same    restriction enzymes.
RD mixture:
20 μl RD assay was performed according to the following see    following protocol .
DNA Electrophoresis:
The digestion mixture of backbone pDR111 was loaded on a gel to    extract the backbone. For detailed information on how to prepare and run    agarose gels see following protocol .
Figure 2. DNA electrophoresis of digested pDR111 with SalI, HindIII.    The correct size is 7,834 bp.
Conclusion:
The restriction digestion was successful. We could see a band of 7,834 bp.
Procedure after gel validation:
Digested pDR111 was cut out from the gel and DNA was extracted by Gel    extraction kit (Agarose Gel Extraction Kit – Jena Bioscience) . The PCR product of the key was not    checked on the gel after the digestion but immediately cleaned up with    DNA Clean-up (NucleoSpin® Gel and PCR Clean-up) .
Ligation
Experiment:
31/08/16: The cut key was ligated to the SalI, HindIII cut pDR111.
Ligation mixture:
20 μl ligation assay was performed according to the following protocol .
Transformation
Experiment:
31/08/16: The ligation mix was heat shock transformed to competent Top10 E. coli cells following the protocol .   Cells were plated on 100 μg/ml ampicillin LB agar to select the correct    construct. The next day colonies were picked to perform colony PCR to    find the correct construct with the primers F-amyE and R-amyE. Find    primers here .
Figure 3. Transformation of the key sequence into the pDR111.
PCR mixture:
25 μl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
30s
(30X)
72ºC
2:00 min
10ºC on hold
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gels see    the following protocol .
Figure 4. DNA electrophoresis of colony PCR for the key sequence in     pDR111. Sample 5,6,7 and 9 show the correct size of     504 bp.
Conclusion:
Transformation of pDR111+key into E. coli Top10 appeared to be    successful. The samples 5, 6, 7 and 9 showed the right size of band in    the colony PCR. These samples were used to obtain the plasmid from an overnight culture.
Validation
Experiment:
02/09/16: Grown cultures of E. coli Top10 with pDR111+key were used to    obtain a glycerol stocks and plasmid isolation was performed (QIAprep® Spin Miniprep Kit) . Firstly, concentration of the plasmids obtained was    measured on Nanodrop. Secondly, plasmids were sent for sequencing and    then stored at -20°C.
Sequencing:
Plasmids pDR111+key from colonies 5, 6, 7 and 9 were sent for sequencing with the sequencing primer F message    sequencing (see primer list) .
Conclusion:
Sequencing results showed that cloning of message sequence into the    integration plasmid pDR111 was successful. Sample 5, 6 and 9 show a    100% homology, so the key was successfully cloned into the pDR111    plasmid. See sequencing results in Figure 5.
Figure 5. Sequencing result of the key in pDR111 for     sample 5, 6 and 9.
Experiments
Experiments:
See Proof of concept experiment.
Key sequence in pSB1C3 ( BBa_K1930000 )
The iGEM team Groningen 2016 worked on bioencryption in order to safely store data in DNA. In our project we were mainly working with two sequences of DNA which were the encrypted message sequence and the encryption key sequence. The key sequence was created by our software. It is therefore artificial DNA. We ordered the encryption key sequence as gBlock from IDT (Integrated DNA technologies). To submit our key sequence as BioBrick ( BBa_K1930000 ) we cloned it in the pSB1C3 standard iGEM backbone.
PCR
Experiment:
27/09/16: The key was PCR amplified (see following protocol ) from the pDR111+key plasmid with the    primers key prefix and key suffix (see primer list) . The correct size 187 bp of the    product was checked by DNA electrophoresis. The PCR product was stored    at 4°C.
PCR mixture:
50 μl PCR assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gels see    following protocol .
Figure 1. DNA electrophoresis of amplified key sequence (187 bp in size).
Conclusion:
The key was successfully amplified with prefix and suffix from the    pDR111+key plasmid.
Procedure after gel validation:
PCR product was subsequently cleaned with (PCR Purification Kit – Jena Bioscience) .
Restriction digestion
Experiment:
28/09/16: The PCR product of the key sequence was cut with EcoRI and PstI. The backbone pSB1C3 (BBa_J04450) was digested with the same enzymes. This construct is carrying RFP reporter therefore it was used for easier screening after transformation. You could see self-ligations as red    colonies and the correct ones as white ones.
RD mixture:
30 μl RD assay was performed according to the following see    following protocol .
DNA Electrophoresis:
The digestion mixture of backbone pSB1C3 - BBa_J04450 was loaded on a    gel to extract the digested backbone.
For detailed information on how to prepare and run    agarose gels see following protocol .
Figure 2. DNA electrophoresis of EcoRI and PstI digested     pSB1C3 - BBa_J04450. The RFP insert is 1069 bp and the pSB1C3 backbone     is 2019 bp.
Conclusion:
The digestion was successful. We could see bands for both expected    fragments on the gel, namely RFP insert is 1069bp and the    pSB1C3 backbone is 2019 bp.
Procedure after gel validation:
The upper band of 2019 bp was cut out from the gel    and DNA was extracted by (Agarose Gel Extraction Kit – Jena Bioscience) . The PCR product of the key sequence was not checked on the gel after the digestion but immediately    cleaned up with (NucleoSpin® Gel and PCR Clean-up) . The concentration of the digested and cleaned key sequence was measured with the Nanodrop, 10,3 ng/μl were obtained.
Ligation
Experiment:
07/10/16: The EcoRI, PstI cut pSB1C3 backbone was ligated with the    EcoRI, PstI cut key.
Ligation mixture:
20 μl ligation assay was performed according to the following protocol .
Transformation
Experiment:
08/10/16: The ligation mix was heat shock transformed to competent Top10 E. coli cells following the protocol .    Cells were plated on 50 μg/ml chloramphenicol LB agar to select for the    correct constructs. The next day colonies were picked to perform colony    PCR to find the correct constructs with the primers key only prefix and    key only suffix. Find    primers here .
Figure 3. E. coli Top10 cells transformed with key in pSB1C3.
PCR mixture:
25 μl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
1:30 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA electrophoresis:
For detailed information on how to prepare and run agarose gels see    the following protocol .
Figure 4. Result of the colony PCR. Samples from colony     1-9 are showing the correct size of the key sequence which is 187 bp. C - is the water control.
Conclusion:
The transformation of the key sequence in pSB1C3 to E. coli Top10 was    successful. We have obtained correct insert size when colony PCR was done. And that was 187 bp.
Validation
Experiment:
09/10/16: Grown cultures of E. coli Top10 with the construct key in    pSB1C3 were used to obtain glycerol stocks and plasmid isolation    was performed (Fast-n-Easy Plasmid Mini-Prep Kit - Jena Bioscience) . Firstly,    concentration of the plasmids obtained was measured on Nanodrop.    Secondly, plasmids were sent for sequencing and then stored at    -20°C.
Sequencing:
The BioBrick key in pSB1C3 ( BBa_K1930000 ) from colonies 1 and 3 was sent for sequencing with the primers VF2 and VR, (see primer list) .
Figure 5. Sequencing result for the key in pSB1C3 (BBa_K1930000) from     colony 1 with VF2.
Figure 6. Sequencing result for the key in pSB1C3 (BBa_K1930000) from     colony 1 with VR.
Conclusion:
The sequencing result proofed the successful integration of the key    into the pSB1C3. First BioBrick was made!
Experiments
Experiments:
See integration of the key sequence in B. subtilis via the BioBrick BBa_K823023 integration plasmid in Proof of concept experiment.
Key sequence in BBa_K823023
BBa_K823023 is an available BioBrick from iGEM Munich 2012. It is an   integration plasmid for Bacillus subtilis, which can be used for   cloning in E. coli as well. An RFP is inserted in BBa_K823023 for more   efficient screening after transformation. It was chosen to   construct the Bacillus subtilis key strain. Construction was   performed as described in the following.
PCR
Experiment:
27/09/16: The key sequence was amplified (see PCR protocol) from the pDR111+key plasmid with the primers    key only + prefix and key only + suffix (primer sequences can be found here ). The correct size of 187 bp of the product was checked by DNA electrophoresis. The PCR product was stored at 4°C.
PCR mixture:
50 µl PCR assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gels see    following protocol .
Figure 1. DNA electrophoresis of amplified key sequence (187 bp in size).
Conclusion:
The key sequence with prefix and suffix was successfully amplified from the gBlock. Band of 187 bp could be seen.
Procedure after gel validation:
PCR product was subsequently cleaned with (PCR Purification Kit – Jena Bioscience) .
Restriction digestion
Experiment:
04/10/16: The key sequence PCR product was digested with EcoRI and PstI as well as the    BBa_K823023 plasmid. The digestion of the BBa_K823023 backbone was checked with DNA    electrophoresis. The digestion of the key sequence PCR product was immediately cleaned    with PCR Purification Kit – Jena Bioscience see protocol .
RD mixture:
20 μl RD assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run    agarose gels see following protocol .
Figure 2. BBa_K823023 cut with EcoRI and PstI. Backbone     ~6,000 bp, RFP insert ~1,000 bp.
Conclusion:
The digestion of BBa_K823023 showed the correct bands on    the gel for the 6,000 bp backbone and the 1,000 bp RFP insert.    Therefore the cloning procedure could proceed.
Procedure after gel validation:
Digested sample of the backbone ~6,000 bp were cut out from the    gel and DNA was extracted by Gel extraction kit (Nucleospin) see protocol .
Ligation
Experiment:
04/10/16: The EcoRI and PstI cut BBa_K823023 integration backbone was    ligated with the EcoRI, PstI cut key sequence.
Ligation mixture:
20 µl ligation assay was performed according to the following protocol .
Transformation
Experiment:
04/10/16: The ligation mix was heat shock transformed to competent    Top10 E. coli cells following the protocol . Cells were plated on 100 μg/ml ampicillin LB agar to select for the correct constructs. The next day colonies were selected to perform colony PCR in order to find the correct constructs.
Figure 3. Colonies of E.coli Top10 after     transformation with BBa_K823023 with the key sequence.
PCR mixture:
25 µl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
1:30 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Conclusion:
The transformation of the key sequence in BBa_K823023 to E. coli Top10 was    successful. We have obtained correct insert size when colony PCR was done. And that was 187 bp.
Validation
Experiment:
05/10/16 Grown cultures of E. coli Top10 with the key sequence in BBa_K823023 were used for making the (see Mini prep protocol) . Firstly, concentration of the plasmids obtained was measured on Nanodrop. Secondly, plasmids were sent for sequencing and then stored at -20°C.
Figure 4. Selecting colonies for glycerol stocks.
Experiments
Experiments:
See Proof of concept experiment.
Message sequence in pDR111
The message sequence was ordered as a gBlock from IDT to construct the message B.   subtilis strain. In a first approach this was done with the pDR111 B.   subtilis integration plasmid, which can also be amplified by E. coli.   Therefore the first step was to clone the message sequence into the pDR111 in E. coli. pDR111 is an integration plasmid which can be integrated into the B. subtilis genome.   By double cross-over it replaces amyE gene, which is necessary for production of alpha-amylase   (see Subtiwiki.uni-goettingen.de )       with desired insert which is located between amyE front flanking region and amyE back flanking region. See the plasmid map below. We used this plasmid for integration of our message sequence into the B. subtilis 168 sub+. The other approach made use of the BioBrick integration plasmid BBa_K823023 (Message sequence in BBa_K823023) .
PCR
Experiment:
25/07/16: The message sequence was amplified from gBlock ordered from IDT (Integrated DNA technologies).Primers used for the amplification were F-message sequence and R-message sequence (primer sequences can be found here ).
PCR mixture:
50 μl PCR assay was performed according to the following protocol.
PCR set-up:
95ºC
2:00 min
95ºC
30s
(12X)
60ºC
30s
(12X)
72ºC
1:30 min
(12X)
72ºC
2:00 min
10ºC on hold
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol.
Figure 2. DNA electrophoresis with amplified message sequence 572 bp in size.
Conclusion:
PCR of the message sequence from the IDT gBlock was successful. It was verified by DNA electrophoresis.   Correct size of a band could be seen and that was 572 bp.
Procedure after gel validation:
PCR product was subsequently cleaned with PCR Purification Kit – Jena Bioscience (see protocol).
Restriction digestion
Experiment:
26/07/16: On this day restriction digestion of PCR product of the message sequence and pDR111 integration plasmid was done. Message sequence as an insert was cut with SalI and HindIII restriction enzymes. Integration plasmid pDR111 as a vector was cut with exactly same restriction enzymes.
RD mixture:
20 μl RD assay was performed according to the following protocol.
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol.
Figure 3. DNA electrophoresis with samples from restriction digestion.
Conclusion:
RD of the PCR product of message sequence and integration plasmid pDR111 was successful. It was verified by DNA electrophoresis. Correct size of bands could be seen and that was 7,834 bp for pDR111 and 572 bp for message sequence.
Procedure after gel validation:
Digested samples were cut out from the gel and DNA was extracted by Gel extraction kit (Nucleospin) (see protocol).
Ligation
Experiment:
26/07/16: Restriction digestion was followed by overnight ligation. Vector (integration plasmid pDR111) and insert (message sequence) was ligated in 1:5 molar ratio.
Ligation mixture:
20 μl ligation assay was performed according to the following protocol.
Transformation
Experiment:
27/07/16: This day transformation into E. coli Top10 and MC1061 cells was done. Correct clones were selected on 100 μg/ml ampicillin plates. Transformation protocol used can be found here .
28/07/16: Next day 12 colonies were obtained on the plates of E. coli MC1061 strain. To verify correct transformants colony PCR was performed (see Colony PCR protocol). Primers F-message sequence and R-message sequence were used for this colony PCR. Sequences of these primers can be found here .
PCR mixture:
25 μl PCR assay was performed according to the following protocol.
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
1:30 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol.
Figure 4. DNA electrophoresis with samples from colony PCR.
Figure 5. Transformation of pDR111+message into E. coli MC1061.
Conclusion:
Transformation of pDR111+message into E. coli MC1061 appeared to be successful. Transformation efficiency was low, only 12 colonies were obtained, but 4 colonies seemed to be correct clones. And those were colonies 1, 8, 10 and 11 (see Figure 5). Correct sizes of the bands were obtained: 572 bp. Those colonies were grown overnight (see cell culture protocol ) from pre-glycerol stocks made for colony PCR (see colony PCR protocol ).
Validation
Experiment:
29/07/16: Grown cultures of E. coli MC1061 with pDR111+message were taken out from the incubator after overnight incubation. Glycerol stocks ) were made and plasmid isolation was performed (see Mini prep protocol ).  Firstly, concentration of the plasmids was measured on Nanodrop. Secondly, plasmids were sent for sequencing and then stored at -20°C.
Sequencing:
Plasmids pDR111+message from colonies 1, 8, 10 and 11 were sent for sequencing )  with the sequencing primer Gb_insert_F2 ( (see primer list) ).
Conclusion:
Sequencing results showed that cloning of message sequence into integration plasmid pDR111 was successful. However just sequencing result 249 (Figure 9) has 100 % homology when compared to the reference (message sequence). Others have some bases missing or bases are not matching, it could be due to faulty sequencing. See sequencing results below.
Figure 6. Sequencing result 246 of the message in pDR111.
Figure 7. Sequencing result 247 of the message in pDR111.
Figure 8. Sequencing result 248 of the message in pDR111.
Figure 9. Sequencing result 249 of the message in pDR111.
Experiments
Experiments:
See Proof of concept experiment.
Message sequence in pSB1C3 ( BBa_K1930001 )
The iGEM team Groningen 2016 worked on bioencryption in order to safely store data in DNA. In our project we were mainly working with two sequences of DNA which were the encrypted message sequence and the encryption key sequence. The message sequence was created by our software. It is therefore artificial DNA. We ordered the encryption message sequence as gBlock from IDT (Integrated DNA technologies). To submit our message sequence as BioBrick ( BBa_K1930001 ) we cloned it in the pSB1C3 standard iGEM backbone.
PCR
Experiment:
06/10/16: The message sequence was amplified from pDR111+message plasmid.    Primers used for the amplification were message prefix and message    suffix (primer sequences can be found here ).
PCR mixture:
50 μl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
2:00 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Figure 1. DNA electrophoresis with amplified message     sequence (M1 and M2) 599 bp in size. C - is water control.
Conclusion:
Amplification of the message sequence and addition of the prefix and    suffix to this sequence was successful. It was verified by DNA    electrophoresis. Correct size of a band could be seen and    that was 599 bp.
Procedure after gel validation:
PCR product was subsequently cleaned with PCR Purification Kit – (Jena Bioscience) .
Restriction digestion
Experiment:
07/10/16: On this day restriction digestion of PCR product of the    message sequence and pSB1C3 (BBa_J04450) was done. Message    sequence as an insert was cut with EcoRI and PstI restriction enzymes.    The backbone pSB1C3 (BBa_J04450) was digested with the same enzymes. This construct is carrying RFP reporter therefore it was used for easier screening after transformation. You could see self-ligations as red    colonies and the correct ones as white ones.
RD mixture:
20 μl RD assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Figure 2. DNA electrophoresis with cut backbone pSB1C3     (BBa_J04450).
Conclusion:
RD of the PCR product of the message sequence and pSB1C3 (BBa_J04450)    was successful. RD of pSB1C3 (BBa_J04450) was verified by DNA    electrophoresis. Correct size of band could be seen and    that was 2019 bp.
Procedure after gel validation:
The upper band was cut out from the gel and DNA was extracted by Gel    extraction kit (NucleoSpin® Gel and PCR Clean-up) .
Ligation
Experiment:
07/10/16: Restriction digestion was followed by overnight    ligation. Vector (pSB1C3) and insert (message sequence) was ligated in    4:6 molar ratio.
Ligation mixture:
20 μl ligation assay was performed according to the following protocol .
Transformation
Experiment:
08/10/16: On this day transformation into E. coli Top10 cells was done.    Selection was made on 50 μg/ml chloramphenicol LB agar plates.    Transformation protocol used can be found here .
09/10/16: Next day we could assume that our transformation was    successful -> colonies were obtained on the plates of E. coli Top10    strain. To verify correct transformants colony PCR was    performed (see colony PCR protocol ). Primers message    prefix and message suffix were used for this colony PCR. Sequences of    these primers can be found here .
PCR mixture:
50 μl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
2:00 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Figure 3. DNA electrophoresis with samples from colony     PCR.
Figure 4. Transformation of pSB1C3+message into E. coli Top10.
Conclusion:
Transformation of pSB1C3+message into E. coli Top10 appeared to be    successful. Correct sizes of the bands from colony PCR were obtained: 599 bp. Correct clones were grown overnight (see cell culture protocol ) from pre-glycerol stocks    made for colony PCR (see colony PCR protocol ).
Validation
Experiment:
10/10/16: Grown cultures of E. coli Top10 with pSB1C3+message were taken    out from the incubator after overnight incubation. Glycerol stocks were made and plasmid isolation was performed (see (Fast-n-Easy Plasmid Mini-Prep Kit - Jena Bioscience) ). Firstly, concentration of the plasmids    obtained was measured on Nanodrop. Secondly, plasmids were sent for    sequencing and then stored at -20°C.
Sequencing:
Plasmids pSB1C3+message from colonies 1, 2, 3, 4 and 5 were sent for sequencing with the sequencing    primers VF2 and VR (primer sequence can be found here ).
Conclusion:
Sequencing results showed that cloning of message sequence into    pSB1C3 was successful. See sequencing results below.
Figure 5. Sequencing result of the message in     pSB1C3 with VF2 primer.
Figure 6. Sequencing result of the message in     pSB1C3 with VF2 and VR primers.
Figure 7. Sequencing result of the message in     pSB1C3 with VF2 and VR primers.
Figure 8. Sequencing result 249 of the message in     pDR111.
Experiments
Experiments:
See integration of the message sequence into B. subtilis via the BioBrick BBa_K823023 integration plasmid in Proof of concept experiment.
Message sequence in BBa_K823023
BBa_K823023 is an available BioBrick from igem Munich 2012. It is an integration plasmid for Bacillus subtilis, which can be used for   cloning in E. coli as well. An RFP is inserted in BBa_K823023 for more   efficient screening after transformation. It was chosen to   construct the Bacillus subtilis message strain. Construction was   performed as described in the following.
PCR
Experiment:
06/10/16: Message sequence was amplified from pDR111+message plasmid.    Primers used for the amplification were message prefix and message    suffix (primer sequences can be found here ).
PCR mixture:
50 μl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
2:00 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gels see    following protocol .
Figure 1. DNA electrophoresis with amplified message     sequence (M1 and M2) 599 bp in size. C - is water control.
Conclusion:
Amplification of the message sequence and addition of the prefix and    suffix to this sequence was successful. It was verified by DNA    electrophoresis (see above). Correct size of a band could be seen and    that was 599 bp.
Procedure after gel validation:
PCR product was subsequently cleaned with (PCR Purification Kit – Jena Bioscience) .
Restriction digestion
Experiment:
07/10/16: On this day restriction digestion of PCR product of the    message sequence and BBa_K823023 was done. Message sequence    as an insert was cut with EcoRI and PstI restriction enzymes.    BBa_K823023 as a vector was cut with exactly same restriction    enzymes.
RD mixture:
20 μl RD assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run    agarose gels see following protocol .
Figure 2. DNA electrophoresis with cut backbone     BBa_K823023.
Conclusion:
RD of the PCR product of message sequence and BBa_K823023 was    successful. RD of BBa_K823023 was verified by DNA electrophoresis. Correct size of band could be seen and that was ∼6000 bp.
Procedure after gel validation:
The upper band was cut out from the gel and DNA was extracted by Agarose gel    extraction kit (Jena Bioscience) .
Ligation
Experiment:
07/10/16 Restriction digestion was followed by overnight    ligation. Vector (BBa_K823023) and insert (message sequence) was    ligated in 4:6 molar ratio.
Ligation mixture:
20 μl ligation assay was performed according to the following protocol .
Transformation
Experiment:
08/10/16: On this day transformation into E. coli Top10 was done.    Selection was made on 100 μg/ml ampicillin LB agar plates.    Transformation protocol used can be found here .
09/10/16: Next day we could assume that our transformation was    successful -> colonies were obtained on the plates of E. coli Top10    strain. To verify correct transformants colony PCR was    performed (see colony PCR protocol ). Primers message    prefix and message suffix were used for this colony PCR. Sequences of    these primers can be found here .
PCR mixture:
50 μl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
2:00 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Figure 3. DNA electrophoresis with samples from colony     PCR.
Figure 4. Transformation of BBa_K823023+message into E. coli Top10.
Conclusion:
Transformation of BBa_K823023+message into E. coli Top10 appeared to    be successful. Correct sizes of the bands from colony PCR (see Figure    3) were obtained: 599 bp. Correct clones were grown overnight (see cell    culture protocol ) from pre-glycerol stocks    made for colony PCR (see colony PCR protocol .
Validation
Experiment:
10/10/16 Grown cultures of E. coli Top10 with BBa_K823023+message were    taken out from the incubator after overnight incubation. Glycerol stocks were made and plasmid isolation was    performed (see (Fast-n-Easy Plasmid Mini-Prep Kit - Jena Bioscience) protocol). Concentration of the    plasmids obtained was measured on Nanodrop and plasmids were stored at    -20°C.
Experiments
Experiments:
See Proof of concept experiment.
sfGFP(Sp) in pSB1C3 ( BBa_K1930006 )
sfGFP(Sp) is a reporter gene originally optimized for Streptococcus pneumoniae. It has been shown when expressed in B. subtilis that the signal is even brighter than the one from sfGFP(Bs) (Overkamp et al. 2013) . To submit sfGFP(Sp) reporter gene as BioBrick ( BBa_K1930006 ) we cloned it in the pSB1C3 standard iGEM backbone.
PCR
Experiment:
06/10/16: sfGFP(Sp) was amplified from the pDR111+sfGFP(Sp) plasmid, that    we constructed during the project, with the primers prefix sfGFP(Sp) and    suffix sfGFP(Sp) (primer sequences can be found here ).
PCR mixture:
50 μl PCR assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see    following protocol .
Figure 1. DNA electrophoresis with amplified sfGFP(Sp)     gene (763 bp in size).
Conclusion:
The sfGFP(Sp) was successfully amplified from the pDR111+sfGFP(Sp) plasmid.
Procedure after gel validation:
PCR product was subsequently cleaned with PCR Purification Kit – Jena Bioscience .
Restriction digestion
Experiment:
The sfGFP(Sp) was cut with EcoRI and PstI. The backbone pSB1C3 (BBa_J04450) was digested with the same enzymes. This construct is carrying RFP reporter therefore it was used for easier screening after transformation. You could see self-ligations as red    colonies and the correct ones as white ones.
RD mixture:
20 μl RD assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see    following protocol .
Figure 2. DNA electrophoresis of EcoRI and PstI digested     pSB1C3 - BBa_J04450. The RFP insert is 1069bp and the pSB1C3 backbone     is 2019 bp.
Conclusion:
The digestion was successful because bands for both expected    fragments could be seen on the gel, namely RFP insert is 1069bp and the    pSB1C3 backbone is 2019 bp.
Procedure after gel validation:
The upper band of 2000 bp was cut out from the gel    and DNA was extracted with Agarose Gel Extraction Kit – Jena Bioscience . The PCR product was not    checked on the gel after the digestion but immediately cleaned up with NucleoSpin® Gel and PCR Clean-up .
Ligation
Experiment:
The cut and cleaned sfGFP(Sp) was ligated to the cut and cleaned    pSB1C3 in a ratio of 2:1.
Ligation mixture:
20 μl ligation assay was performed according to the following protocol .
Transformation
Experiment:
07/10/16: The ligation mix was heat shock transformed to competent Top10 E. coli cells following the transformation protocol .    Cells were plated on 50 μg/ml chloramphenicol LB agar to select the    correct construct.
09/10/16: Colonies were picked to perform colony PCR to find the correct    constructs with the primers prefix sfGFP(Sp) and suffix sfGFP(Sp). Find primer sequences here .
Figure 3. Top10 E. coli colonies on 50 μg/ml chloramphenicol LB     agar plates after transformation with sfGFP(Sp) in pSB1C3.
PCR mixture:
25 μl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
1:30 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA electrophoresis:
For detailed information on how to prepare and run agarose gel see    following protocol .
Figure 4. DNA electrophoresis of the colony PCR product.     C - is the water control. Samples 1-5 are from colony 1-5.
Conclusion:
All 5 samples show the right band by 763 bp. Therefore all of them    were grown overnight to harvest the plasmid the following day.
Validation
Experiment:
10/10/16: Grown cultures of E. coli Top10 with the construct sfGFP(Sp) in    pSB1C3 were used to obtain ( glycerol stocks ) and plasmid    isolation was performed with QIAprep® Spin Miniprep Kit . Some of the overnight   cultures seemed to already express the sfGFP (see Figure 5). Firstly,    concentration of the plasmids obtained was measured on Nanodrop.    Secondly, plasmids were sent for sequencing and then stored at    -20°C.
Figure 5. Pellet of the overnight cultures from sample     2 and 5. Looks like the sfGFP(Sp) is also expressed in E. coli Top10.
Sequencing:
Sequencing results showed that sfGFP(Sp) gene is present (see Figure 6). However something happened with the prefix and the suffix most probably during cloning steps. It looks that prefix and also suffix was disrupted by insertion of few base pairs. We cannot really explain what happened and due to time limitation we could not fix this problem. However sfGFP(Sp) is clearly expressed in E. coli (see Figure 5).
Figure 6. Sequencing result of sfGFP(Sp) in pSB1C3. GFP gene itself is underlined by red line.
Conclusion:
The sfGFP(Sp) was cloned to the backbone pSB1C3. However from the sequencing results we could see that this part contains bad prefix and suffix (see Figure 6). GFP gene is underlined by red line on Figure 6, however you could also see the bad prefix and suffix.
Experiments
See decoy experiments with sfGFP(Sp) in B. subtilis.
sfGFP(Sp) in pDR111 and in pDR111+message plasmid
sfGFP(Sp) is a reporter gene originally optimized for Streptococcus pneumoniae. It has been shown when expressed in B. subtilis that the signal is even brighter than the one from sfGFP(Bs) (Overkamp et al. 2013) . sfGFP(Sp) was cloned either only to pDR111 integration plasmid or to   pDR111+message plasmid. These plasmids were constructed with an interest to   have easier screening when integrated into the B. subtilis genome. pDR111 is an integration plasmid which can be integrated into the B.   subtilis genome. By double cross-over it replaces amyE gene, which is   necessary for production of alpha-amylase ,   with desired insert which is located between amyE front flanking region and   amyE back flanking region. See the plasmid map below.
PCR
Experiment:
10/08/16: sfGFP(Sp) was amplified from pNW plasmid together with its    promoter (Ppta), which is a constitutive promoter from Geobacillus DSM2542 for phosphate acetyl transferase expression, and a triple    terminator (3TER). Primers used for the amplification were F-gfp insert    and R-gfp insert (primer sequences can be found here ).
PCR mixture:
25 μl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
57ºC
30s
(30X)
72ºC
1:30 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Figure 2. DNA electrophoresis with amplified     Ppta-sfGFP(Sp)-3TER (1147 bp in size).
Conclusion:
Amplification of the Ppta-sfGFP(Sp)-3TER was successful. It was    verified by DNA electrophoresis. Correct size of a band    could be seen and that was 1147 bp. However second band could be seen,    that was probably caused by unspecific binding of the primers used.    Therefore lower band of correct size was cut out from the gel and DNA    was extracted.
Procedure after gel validation:
DNA extraction was done according to this protocol .
Restriction digestion
Experiment:
10/08/16: On this day restriction digestion of PCR product of the    sfGFP(Sp), pDR111 integration plasmid and pDR111+message    integration plasmid was done. sfGFP(Sp) as an insert was cut with BglII    and NheI restriction enzymes. Integration plasmid pDR111 and pDR111+message as a vector was cut with BamHI and NheI. BglII could not    be used as a restriction enzyme for pDR111 due to multiple recognition    site of this cutter. BamHI as a compatible restriction enzyme to BglII    was used instead.
RD mixture:
20 μl RD assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Figure 3. DNA electrophoresis with samples from restriction     digestion.
Figure 4. DNA electrophoresis with samples from restriction     digestion.
Conclusion:
RD of the PCR product of sfGFP(Sp), integration plasmid pDR111 and    pDR111+message was successful. RD of pDR111 and pDR111+message was    verified by DNA electrophoresis. Correct sizes of bands could    be seen and that was 6479 bp and 7034 bp, respectively.
Procedure after gel validation:
The upper band was cut out from the gel and DNA was extracted by Gel    extraction kit (NucleoSpin® Gel and PCR Clean-up) .
Ligation
Experiment:
10/08/16: Restriction digestion was followed by ligation at    room temperature for 45 min. Vector (integration plasmid pDR111 and    pDR111+message) and insert (sfGFP(Sp)) was ligated in 1:5 molar ratio.
Ligation mixture:
20 μl ligation assay was performed according to the following protocol .
Transformation
Experiment:
10/08/16: Ligation was followed by transformation into E. coli Top10.    Selection was made on 100 μg/ml ampicillin LB agar plates.    Transformation protocol used can be found here .
11/08/16: Next day we could assume that our transformation was    successful -> colonies were obtained on the plates of E. coli Top10    strain. To verify correct transformants we grew some of the colonies    overnight in 3 ml LB with 100 μg/ml ampicillin (see cell culture protocol ). On the next day plasmid    purification and restriction digestion control was done.
Conclusion:
Transformation of pDR111+sfGFP(Sp) and pDR111+message+sfGFP(Sp) into E. coli Top10 was assumed to be successful. To see if we obtained    correct clones we did a restriction digestion control with EcoRI    restriction enzyme.
Validation
Experiment:
12/08/16: Next day plasmid purification was done according to this protocol and restriction digestion control with EcoRI    restriction enzyme was done to see if correct clones were obtained.    This enzyme has 2 recognition sites in our construct. Therefore two    bands (1535 bp and 6081 bp from pDR111+sfGFP(Sp) and 2090 bp and 6081    bp from pDR111+message+sfGFP(Sp)) should have been obtained on the    agarose gel. Moreover we sent this new constructed plasmid for    sequencing.
RD mixture:
20 μl RD assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Figure 5. DNA electrophoresis with samples (1-4 from     pDR111+sfGFP(Sp)) from RD control with EcoRI.
Figure 6. DNA electrophoresis with samples (1-6 from     pDR111++message+ sfGFP(Sp)) from RD control with EcoRI.
Sequencing:
Plasmid pDR111+sfGFP(Sp) and pDR111+message+sfGFP(Sp) was sent for sequencing . Primers used for    the sequencing were F-gfp insert for pDR111+sfGFP(Sp) and F-gfp insert    and F-message sequence for pDR111+message+sfGFP(Sp) (primer sequences    can be found here ).
Conclusion:
Figure 5 shows two correct bands (1535 bp and 6081 bp) when the    pDR111+sfGFP(Sp) was cut.  Figure 6 shows two correct bands (2090 bp    and 6081 bp) when the pDR111+message+sfGFP(Sp) was cut.  Therefore we    assumed that our cloning was successful and correct plasmids were    obtained. Sequencing results confirmed our assumption as you can see in    Figure 7, 8 and 9.
Figure 7. Sequencing result 334 of the sfGFP(Sp)     construct in pDR111 (primer used: F-gfp     insert).
Figure 8. Sequencing result 335 of the sfGFP(Sp)     construct in pDR111+message (primer used: F-gfp     insert).
Figure 9. Sequencing result 336 of the sfGFP(Sp)     construct in pDR111+message (primer used: F-message     sequence).
Experiments
Experiment:
The functionality of these constructs was checked by integration    into the B. subtilis genome 168 trp+ and imaging under the microscope.
Integration into the genome of B. subtilis 168 trp+:
sfGFP(Sp) in pDR111 and pDR111+message was integrated into the B.    subtilis 168 trp+ genome. pDR111 is an integration plasmid which can be    integrated into the B. subtilis genome. By double cross-over it    replaces amyE gene, with desired insert, in this case insert is    Ppta-sfGFP(Sp)-3TER, which is located between amyE front    flanking region and amyE back flanking region.
Experiment set-up:
10 μl of sfGFP(Sp) in pDR111 and pDR111+message plasmids were    transformed into the B. subtilis 168 trp+ strain (see transformation protocol ). Selection was made on 150 μg/ml    spectinomycin LB agar plates.
Microscopy experiment set-up:
B. subtilis cells were grown overnight in 3 ml LB with 150 μg/ml    spectinomycin at 37°C and 220 rpm. On the next day microscopy slide was    prepared as explained in the following protocol . Cells    were imaged using phase contrast: filter POL 50% 1 s exposure, green    fuorescence: FITC, 0.8 s exposure, objective: Olympus 100X/1.40,    camera: CoolSNAP_HQ/HQ2-ICX285 and software: Resolve3D softWoRx-Acquire    version.
Figure 10. B. subtilis 168 trp+ cells with sfGFP     integrated into the amyE locus.
Conclusion:
Integration into the B. subtilis 168 trp+ strain of the plasmids    with the reporter sfGFP(Sp) was successful. Multiple colonies were    obtained on selection plates. B. subtilis cells were subsequently grown    and checked under the microscope to see if there is an expression of    GFP (see Figure 10). We could definitely see the GFP signal when    observing the cells under the microscope. This was big enough proof    that our plasmid construction followed by integration into the B.    subtilis genome was successful.
PAtpI in pSB1C3 ( BBa_K1930005 )
The promoter PAtpI has its origin in Bacillus subtilis. It is   responsible for the expression of atpA gene (ATP synthesis) during the first 30   min of the germination of B. subtilis (Sinai et al. 2015) . atpA gene is part of an operon atpI-atpB-atpE-atpF-atpH-atpA-atpG-atpD-atpC, therefore the promoter region in front of the first protein coding gene (atpI) in this operon was chosen. The region was checked for the binding of sigma factors and   transcription factors with DBTBS . No   binding factors were found with the highest significance level. In our   project we wanted to find a constitutive promoter for our ciprofloxacin   resistance casette. In the following part we put the promoter PAtpI in the   pSB1C3 backbone to make it available to other iGEM teams.
PCR
Experiment:
06/10/16: The promoter PAtpI was amplified (see PCR protocol ) from the    ciprofloxacin resistance cassette BioBrick BBa_K1930004 ) with the primers pATPI+prefix and pATPI+suffix (primer sequences can be    found here ). The    correct size of 372 bp for the PCR product was checked with DNA   electrophoresis. The PCR product was stored at 4°C.
PCR mixture:
50 μl PCR assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gel see    following protocol .
Figure 1. DNA electrophoresis with amplified PAtpI promoter     sequence 372 bp in size.
Conclusion:
The PAtpI promoter was successfully amplified with prefix and suffix    from the plasmid.
Procedure after gel validation:
PCR product was subsequently cleaned with PCR Purification Kit –    Jena Bioscience .
Restriction digestion
Experiment:
07/10/16: The PAtpI was cut with EcoRI and PstI. The backbone pSB1C3 (BBa_J04450) was digested with the same enzymes. This construct is carrying RFP reporter therefore it was used for easier screening after transformation. You could see self-ligations as red    colonies and the correct ones as white ones.
RD mixture:
30 μl RD assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare    and run agarose gel see following protocol .
Figure 2. DNA electrophoresis of EcoRI and PstI digested     pSB1C3 - BBa_J04450. The RFP insert is 1069bp and the pSB1C3 backbone     is 2019 bp.
Conclusion:
The digestion was successful because bands for both expected    fragments could be seen on the gel, namely RFP insert is 1069bp and the    pSB1C3 backbone is 2019 bp.
Procedure after gel validation:
The upper band of 2000 bp was cut out from the gel    and DNA was extracted by Gel extraction kit    (Nucleospin) . The PCR product was not checked on the gel after the    digestion but immediately cleaned up with (CR    Purification Kit – Jena Bioscience .
Ligation
Experiment:
07/10/16: The EcoRI, PstI cut pSB1C3 backbone was ligated with the    EcoRI, PstI cut promoter PAtpI.
Ligation mixture:
20 μl ligation assay was performed according to the following protocol .
Transformation
Experiment:
08/10/16: The ligation mix was heat shock transformed to competent Top10 E. coli cells following the protocol .    Cells were plated on 50 μg/ml chloramphenicol LB agar to select the    correct constructs. The next day colonies were picked to perform colony    PCR to find the correct constructs with the primers pATPI+prefix and pATPI+suffix (primer sequences can be found here ).
Figure 3. E. coli Top10 transformed with PAtpI in pSB1C3.
PCR mixture:
25 μl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
1:30 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA electrophoresis:
For detailed information on how to prepare and run agarose gel see    following protocol .
Figure 4. Result of the colony PCR. Samples from colony     1-5 are showing the correct size of the promoter PAtpI of 372 bp.     C - is the water control.
Conclusion:
The transformation of PAtpI in pSB1C3 to E. coli Top10 was    successful.
Validation
Experiment:
09/10/16: Grown cultures of E. coli Top10 with the construct were used to    make glycerol stocks and    plasmid isolation was performed (see Fast-n-Easy Plasmid Mini-prep kit ).    Firstly, concentration of the plasmids obtained was measured on    Nanodrop. Secondly, plasmids were sent for sequencing and then stored    at -20°C.
Sequencing:
The plasmid PAtpI in pSB1C3 from colonies 1 and 2 was sent for    sequencing with the primers VF2 and VR, see primer list .
Figure 6. Sequencing result for PAtpI in pSB1C3 from     colony 1 with VR.
Figure 5. Sequencing result for PAtpI in pSB1C3 from     colony 1 with VF2.
Conclusion:
The sequencing result proofed the successful integration of the    PAtpI promoter in the pSB1C3. Another BioBrick was obtained!
Experiments
For further experiments with this BioBrick see: MIC value experiment .
Ciprofloxacin resistance cassette in pSB1C3 ( BBa_K1930004 )
To design a qnrS1 resistance cassette BioBrick we designed a gBlock that contains the Bacillus subtilis constitutive promoter PAtpI, which is active from a very early stage of germination and includes a ribosome binding site. The gBlock also contains the original qnrS1 gene sequence from E. coli, the double terminator ( BBa_B0015 ) from iGEM as well as the prefix and suffix for BioBricks. In summary the ciprofloxacin cassette consists of the following parts PAtpI+RBS+qnrS1+2TER, see plasmid map below.
PCR
Experiment:
The sequence of the qnrS1 gene was amplified from the gBlock qnrS1 E. coli ordered from IDT. Primers used for the    amplification were F-qnrs1 E.coli and R-qnrs1 E.coli (primer sequences can be found here ).
PCR mixture:
50 µl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(12X)
60ºC
30s
(12X)
72ºC
1:30 min
(12X)
72ºC
2:00 min
10ºC on hold
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gels see    following protocol .
Figure 1. Gel electrophoresis of the PCR product.
Conclusion:
The PCR of the qnrS1 sequence from the IDT gBlock was successful. It    was verified by DNA electrophoresis. A band with the correct    size of 1,194 bp could be seen.
Procedure after gel validation:
PCR product was subsequently cleaned with (PCR Purification Kit – Jena Bioscience) .
Restriction digestion
Experiment:
28/09/16 The qnrS1 gene should have been cloned into the pSB1C3. For this case the vector BBa_J04450 was used. The qnrS1 gene and BBa_J04450 were cut with the restriction enzymes EcoRI and PstI.
RD mixture:
20 μl RD assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run    agarose gels see following protocol .
Bands of the sizes 2,029 bp for the vector and 1,176 bp for the gene    were expected.
Figure 2. DNA electrophoresis of EcoRI and PstI digested     pSB1C3 - BBa_J04450. The RFP insert is 1,069 bp and the pSB1C3 backbone     is 2019 bp.
Conclusion:
The digestion was successful because bands for both expected    fragments could be seen on the gel, namely RFP insert is 1,069 bp and the    pSB1C3 backbone is 2,019 bp.
Procedure after gel validation:
The digested samples were cut out from the gel and the DNA was    extracted using the Agarose gel Extraction Kit (Jena Bioscience) .
Ligation
Experiment:
28/09/16: After restriction digestion ligation was performed. 6 µl qnrS1 insert DNA were ligated to 4 µl pBS1C3 vector DNA. The    ligation took place for 2 h at room temperature.
Ligation mixture:
20 µl ligation assay was performed according to the following protocol .
Transformation
Experiment:
28/09/16 After ligation the ciprofloxacin resistance cassette in pSB1C3 was transformed into E. coli Top10 cells. The transformation was plated on plates containing 50 µg/ml    chloramphenicol. The transformation was performed according to the    transformation protocol .
29/09/16: To analyze the success of the transformation 12 samples    were picked and a colony PCR was performed according to the following protocol . The primers F-qnrs1 E.coli and R-qnrs1 E.coli were used. Primer sequences can be found here .
DNA electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Conclusion:
The colony PCR did not bring clear results.
Experiment:
30/09/16: To further analyze the success of the transformation    four colonies were grown as overnight cultures (started on 29/09/16)    in LB and 50 µg/ml chloramphenicol. On the next day plasmid isolation was performed and plasmids were digested using the enzymes EcoRI and PstI according to the following restriction digestion protocol .
DNA electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Conclusion:
Two out of the four digested samples showed bands of the expected    sizes 2,029 bps for the vector and 1184 for the gene could be seen.
Validation
Experiment:
The two samples that showed the correct digestion pattern in the restriction digestion control were sent for sequencing with the primers VF2 and VR, see primer list . These two samples showed the correct sequence (Figure 5.1-5.4, 5.1 mutant 1 forward, 5.2: mutant 2 forward, 5.3: mutant 1 reverse, 5.4: mutant 2 reverse ).
Figure 5.1
Figure 5.2
Figure 5.3
Figure 5.4
Experiments
For further experiment see the construction of the plasmid ciprofloxacin resistance cassette in BBa_K823023 .
Ciprofloxacin resistance cassette in BBa_K823023
To design a ciprofloxacin resistance cassette we designed a gBlock that contains the Bacillus subtilis promoter PAtpI, which is active from a very early stage of germination and includes a ribosome binding site. The gBlock also contains the original qnrS1 gene sequence from E. coli. qnr genes code for pentapeptide repeat proteins. These proteins reduce susceptibility to quinolones by protecting the complex of DNA and DNA gyrase enzyme from the inhibitory effect of quinolones. Finally, this gBlock contains a double terminator BBa_B0015 from iGEM as well as the prefix and suffix for BioBricks. In summary the ciprofloxacin cassette consists of the following parts PAtpI+RBS+qnrS1+2TER. BBa_K823023 is an available BioBrick from igem Munich 2012. It is an   integration plasmid for Bacillus subtilis, which can be used for   cloning in E. coli as well. An RFP is inserted in BBa_K823023 for more   efficient screening after transformation. Construction of ciprofloxacin resistance cassette in BBa_K823023 integration plasmid was performed as described in the following.
PCR
Experiment:
The sequence of the qnrS1 gene was amplified from the gBlock qnrS1 E. coli ordered from IDT. Primers used for the    amplification were F-qnrs1 E.coli and R-qnrs1 E.coli/ (primer sequences can be found here ).
PCR mixture:
50 µl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(12X)
60ºC
30s
(12X)
72ºC
1:30 min
(12X)
72ºC
2:00 min
10ºC on hold
DNA Electrophoresis:
For detailed information on how to prepare and run agarose gels see    following protocol .
Figure 1. DNA electrophoresis of the PCR product.
Conclusion:
The PCR of the qnrS1 sequence from the IDT gBlock was successful. It    was verified by DNA electrophoresis. A band with the correct    size of 1,194 bp could be seen.
Procedure after gel validation:
PCR product was subsequently cleaned with (PCR Purification Kit – Jena Bioscience) .
Restriction digestion
Experiment:
28/09/16: The qnrS1 PCR product and BBa_K823023 were cut with the    restriction enzymes EcoRI and PstI.
RD mixture:
20 μl RD assay was performed according to the following protocol .
DNA Electrophoresis:
For detailed information on how to prepare and run    agarose gels see following protocol .
Figure 2. BBa_K823023 cut with EcoRI and PstI. Backbone ~6,000     bp, RFP insert 1000 bp.
Conclusion:
The digestion of BBa_K823023 showed the correct bands on the gel. 6,000 bp backbone for the backbone and the 1000 bp for the RFP insert. Therefore the cloning procedure could proceed.
Procedure after gel validation:
Digested sample of the backbone ~6,000 bp were cut out from the    gel and DNA was extracted by Agarose gel extraction kit (Jena Bioscience) (see protocol) .
The digestion of the qnrS1 PCR product was immediately cleaned with    the kit (PCR Purification Kit – Jena Bioscience) .
Ligation
19/10 In this ligation the EcoRI, PstI cut gene qnrS1 and the vector    BBa_K823023 were combined.
Ligation mixture:
20 µl ligation assay was performed according to the following protocol .
Transformation
Experiment:
29/09/16: The ligation mix was heat shock transformed to competent Top10 E. coli cells following the protocol . Cells    were plated on 100 μg/ml ampicillin LB agar to select for the correct    constructs. On the next day colonies were selected to perform colony PCR in    order to find the correct constructs using the primers F-qnrs1 E.coli and R-qnrs1 E.coli. Primer sequences can be found here .
Figure 3. Transformation to E. coli Top10.
PCR mixture:
25 µl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
1:30 min
(30X)
72ºC
2:00 min
10ºC on hold
DNA electrophoresis:
For detailed information on how to prepare and run agarose gel see following protocol .
Figure 4. Result of the colony PCR. Sample indicated     with the star shows the correct sie of 1194 bp. C - is the water     control.
Conclusion:
The transformation of qrnS1 (ciprofloxacin resistance cassette) in BBa_K823023 to E. coli Top10 was successful.
Validation
Experiment:
To test if the construct would make B. subtilis resistant to    ciprofloxacin, the construct qrnS1 in BBa_K823023 was transformed into the B.    subtilis 168 tpr+.
Experiments
Experiment:
13/10/16: The transformation to B. subtilis was performed according to the following protocol . Colonies were selected on LB agar    with 5 μg/ml chloramphenicol.
Figure 5. B. subtilis after transformation with     ciprofloxacin resistance cassette.
14/10/16: Colonies were streaked out on agar with starch to perform the    starch test, which verifies the integration in the amyE locus in the    genome of B. subtilis. Integration check: Starch test
Figure 6. Starch test. Colonies without a clear halo     are positive for integration.
Conclusion:
The integration of the ciprofloxacin resistance cassette was    successful.
As a first check on the functionality of the ciprofloxacin    resistance cassette, we grew B. subtilis colonies from the starch test    with ciprofloxacin. As a control they were also grown with    chloramphenicol, the resistance on the backbone of the integration    vector. Figure 7 shows the result for 3 different colonies (tubes    indicated with 1 - 3). The tubes marked with Cm is the control with    chloramphenicol, which shows growth for all three colonies. The tubes    marked with Cipro were grown with ciprofloxacin. Colonies 2 and 3 showed    growth, whereas colony 1 did not grow. Seems like the resistance    cassette is working. To further explore if the ciprofloxacin cassette    is functional in B. subtilis, a MIC value test was performed. See link .
Figure 7.
RBS+nucA in pSB1C3
One approach to delete the key from the genome of B. subtilis is making   use of the nucA BioBrick ( BBa_K729004 ). The nucA is a nuclease gene which has   its origin in the genome of Staphylococcus aureus. It is capable of   digesting genetic material. The RBS is from BioBrick ( BBa_B0030 ). The   combination of these two BioBricks is the first step of achieving our nucA killswitch, which consist of the tetR repressible promoter ( BBa_R0040 ), the RBS   controlling the nucA gene ( BBa_B0030 ) and double terminator ( BBa_B0015 ).
Obtaining RBS ( BBa_B0030 ) and nucA ( BBa_K729004 )
Experiment:
01/08/16: The RBS ( BBa_B0030 ) was obtained from the part distribution 2016,    the plasmid was transformed to E. coli Top10 using this protocol . The nucA ( BBa_K729004 ) was requested from    the iGEM headquarters. Colonies from these transformations were    cultured in LB medium. Grown cultures of E. coli Top10 with RBS and nucA were used to obtain a glycerol stocks and plasmid    isolation was performed (QIAprep® Spin Miniprep Kit) . The concentration of the    plasmids obtained was measured on Nanodrop. The plasmids were stored at    -20°C.
Restriction digestion
Experiment:
28/09/16: On this day restriction digestion of RBS (BBa_B0030) and nucA (BBa_K729004) was performed. The backbone RBS (BBa_B0030) was    digested with SpeI and PstI. The insert nucA was digested with XbaI and    PstI.
RD mixture:
20 μl RD assay was performed according to the following protocol .
DNA Electrophoresis:
The digestion mixture of backbone RBS in pSB1C3 and insert nucA was    loaded on a gel to extract both parts. For detailed information on how    to prepare and run agarose gel see following protocol .
Figure1 DNA electrophoresis of: 1. nucA in pSB1C3  cut     with XbaI and PstI. The fragments are supposed to be nucA/ 561 bp and     pSB1C3 2,019 bp. 2. RBS in pSB1C3 cut with SpeI and PstI one fragment     of 2,070 bp
Conclusion:
The digestion was successful because the band of the nucA could be    seen: 500 bp and RBS in pSB1C3 showed a band of 2,000 bp.
Procedure after gel validation:
Digested nucA and RBS in pSB1C3 were cut out from the gel and DNA    was extracted by (Agarose Gel Extraction Kit – Jena Bioscience) .
Ligation
Experiment:
28/09/16: The cut nucA was ligated to the SpeI and PstI cut RBS in    pSB1C3.
Ligation mixture:
20 μl ligation assay was performed according to the following protocol .
Transformation
Experiment:
28/09/16: The ligation mix was heat shock transformed to competent Top10 E. coli cells following the transformation protocol .    Cells were plated on 50 μg/ml chloramphenicol LB agar to select the    correct constructs. The next day colonies were picked to perform colony    PCR to find the correct constructs with the primers VF2 and VR. Find    primers here .
Figure 2. Transformation of nucA in RBS in pSB1C3.
PCR mixture:
25 μl PCR assay was performed according to the following protocol .
PCR set-up:
95ºC
2:00 min
95ºC
30s
(30X)
60ºC
30s
(30X)
72ºC
30s
(30X)
72ºC
2:00 min
10ºC on hold
DNA electrophoresis:
For detailed information on how to prepare and run agarose gel see    the following protocol .
Figure 3. DNA electrophoresis of colony PCR for samples     1-7 of RBS+nucA in pSB1C3 with primers VF2, VR. Expected product 896     bp.
Conclusion:
Transformation of RBS+nucA in pSB1C3 in E. coli Top10 appeared to be    successful. The samples 2, 5, 6 and 7 showed the right size of band.    These samples were used to obtain the plasmid from an overnight culture.
Validation
Experiment:
29/09/16: Grown cultures of E. coli Top10 with RBS+nucA in pSB1C3 were    used to obtain the glycerol stocks and plasmid isolation was    performed (QIAprep® Spin Miniprep Kit) . Firstly, concentration of the plasmids    obtained was measured on Nanodrop. Secondly, plasmids were sent for    sequencing and then stored at -20°C.
Sequencing:
Plasmids RBS+nucA in pSB1C3 from colonies 2, 5, and 7 were sent for sequencing with the sequencing primers VF2 and VR. (See primer list ).
Conclusion:
Sequencing results showed that the nucA was cloned in the pSB1C3    backbone but the RBS was missing and the suffix was also incorrect. See    Figure 4. The nucA (BBa_K729004) received from the iGEM headquarters was    sent for sequencing as well. It turned out that the part BBa_K729004    does have the nucA gene but the prefix and suffix are incorrect . See    sequencing results in Figure 5 to 7.
Figure 4. Sequencing result of the RBS+nucA in pSB1C3     for sample 5, 7 and 9. nucA can be seen in pink. The prefix is     marked in red.
Figure 5. Sequencing result of the BBa_K729004 aligned     with the nucA gene.
Figure 6. Sequencing result of the BBa_K729004 aligned     with the prefix.
Figure 7. Sequencing result of the BBa_K729004 aligned     with the suffix.
Experiments
We wanted to improve the BBa_K729004 part by giving it the correct prefix and suffix. Unfortunately we did not succeed to complete this task.
Protocols
All protocols which were used throughout our project can    be found below or downloaded here .
Transformation of B. subtilis
Day 1
Streak out desired strain and incubate the plate overnight at 37°C.
Day 2
Pick a nice big colony and drop it in 2 ml of completed 1X MC medium (see below).
Grow at 37°C for 5 hours (or more if culture is not really turbid).
Mix 400 µl of culture with DNA (usually 1 µg) in fresh tube.
Grow for an additional 2 hours at 37°C.
Plate on selective antibiotic plates.
Incubate overnight at 37°C.
Preparation of MC medium
Completed 1X MC Medium
H2O
1.8 ml
10X MC Medium
200 µl
MgSO4
6.7 µl
1 % tryptophan (for trp- strains)
10 µl
10X MC medium(for 100 ml)
K2HPO4.3H2O
14 g
KH2PO4
5.2 g
Glucose
20 g
300 mM Tri-Na citrate*
10 ml
Ferric NH4 citrate**
1 ml
Casein hydrolysate
1 g
K glutamate
1 g
*300 mM Tri-Na citrate = 8.8 g in 100 ml H2O (wrap in aluminium foil)
**Ferric NH4 citrate = 2.2 g in 100 ml H2O (wrap in aluminium foil)
Mix everything in 40-50 ml H2O.
Then adjust to 100 ml.
Filter sterilize.
Freeze at -20°C.
Transformation of E. coli (Standard protocol)
Add ligation mixture to the tube of competent cells.
Leave 30 min on ice.
Heat shock for 5 min at 37°C or for 45 sec at 42°C.
Add 300 µl of LB medium.
Place at 37°C for 30 – 60 min. Shake vigorously (220 rpm).
Plate 30 µl on 1 selection plate, and 300 µl on another.
Transformation of E. coli (NEB® 5-alpha Competent E. coli High Efficiency Transformation Protocol)
Thaw a tube of NEB 5-alpha Competent E. coli cells on ice until the last ice crystals disappear.
Mix gently and carefully pipette 50 µl of cells into a transformation tube on ice.
Add 1 – 5 µl containing 1 pg – 100 ng of plasmid DNA to the cell mixture.
Carefully flick the tube 4 – 5 times to mix cells and DNA. Do not vortex.
Place the mixture on ice for 30 min. Do not mix.
Heat shock at exactly 42°C for exactly 30 sec. Do not mix.
Place on ice for 5 min. Do not mix.
Pipette 950 µl of room temperature SOC into the mixture.
Place at 37°C for 60 min. Shake vigorously (220 rpm) or rotate.
Warm selection plates to 37°C.
Mix the cells thoroughly by flicking the tube and inverting, then perform several 10-fold serial dilutions in SOC.
Spread 50 – 100 µl of each dilution onto a selection plate and incubate overnight at 37°C.
Alternatively, incubate at 30°C for 24 – 36 hours or 25°C for 48 hours.
Reagents supplied:
6 x 0.2 ml/tube of chemically competent NEB 5-alpha Competent E. coli cells
25 ml of SOC Outgrowth Medium
0.025 ml of 50 pg/µl pUC19 Control DNA
Calcium Chloride Competent cells
Label 28 sterile 1.5 ml tubes with name of E. coli strain
All solutions, glass pipettes, pipette tips, 50 ml tubes and 1.5 ml tubes must be sterile and pre-cooled
All work must be done quickly and ON ICE (fresh ice, not half melted!)
Day 1
Streak E. coli strain on LB agar plate and incubate overnight at 37°C.
Day 2
Pick one colony from fresh agar plate and inoculate 3 ml of LB.
Incubate overnight at 37°C and 220 rpm.
Day 3
Inoculate LB medium with 1:100 overnight culture (i.e. 1 ml for 100 ml LB) in 500 ml flask.
Incubate at 37°C and 220 rpm.
Closely follow OD600 (very important!)
At OD600 = 0.35 (max. 0.40) harvest cells (ca. 1.5 – 2 hours TOP10 / ca. 2 – 3 hours INV110).
Put culture in two 50 ml pre-cooled centrifuge tubes, leave on ice for 10 min.
Centrifuge cultures at 2400 x g (4°C) for 4 min.
Discard supernatant (drain last drops onto a paper towel).
Resuspend each pellet in 20 ml cold 0.1 M MgCl2, using pre-cooled pipette and keeping 50 ml tubes IN ice.
Combine both cell cultures in one 50 ml tube.
Centrifuge culture at 2400 x g (4°C) for 4 min. Discard supernatant.
Resuspend pellet in 20 ml cold 50 mM CaCl2, using pre-cooled pipette and keeping tube IN ice.
Leave the culture on ice for 20 min (fresh ice, not half melted).
Centrifuge culture at 2400 x g (4°C) for 4 min. Discard supernatant.
Resuspend pellet in 1 ml cold 50 mM CaCl2 + 15 % (v/v) glycerol.
Leave the culture on ice for 10 min (fresh ice, not half melted).
Use pre-cooled tips and pipet 50 μl aliquots into pre-cooled 1.5 ml tubes.
Freeze immediately in liquid nitrogen.
Store at -80°C.
Restriction digestion (FastDigest enzymes)
Add 2 µl of 10X FastDigest buffer.
Add 1 µl of restriction enzyme 1.
Add 1 µl of restriction enzyme 2.
Add 400 ng of DNA (possible to calculate from the concentration of the sample).
Add MiliQ water up to 20 µl.
Mix the solution by flicking the tube.
Shortly centrifuge for 2-3 sec.
Incubate at 37°C for 30 min.
Restriction digestion (NEB enzymes)
Add 5 µl of NEB buffer.
Add 1 µl of restriction enzyme 1.
Add 1 µl of restriction enzyme 2.
Add 1 µg of DNA (possible to calculate from the concentration of the sample).
Add MiliQ water up to 50 µl.
Mix the solution by flicking the tube.
Shortly centrifuge for 2-3 sec.
Incubate at 37°C for 60 min.
Restriction digestion (Jena Bioscience enzymes)
Add 5 µl of 10X Universal buffer.
Add 1 µl of restriction enzyme 1.
Add 1 µl of restriction enzyme 2.
Add 1 µg of DNA (possible to calculate from the concentration of the sample).
Add MiliQ water up to 50 µl.
Mix the solution by flicking the tube.
Shortly centrifuge for 2-3 sec.
Incubate at 37°C for 60 min.
Ligation
Add 2 µl of ligation buffer.
Add 50 - 100 ng of vector DNA (possible to calculate from the concentration of the sample).
Add X ng of insert DNA. X is calculated using the length of both vector and insert and the molar ratio desired ( NEB calculator ).
Add MiliQ water up to 20 µl.
Add 1 µl of T4 ligase.
Incubate overnight at 16°C or for 30 – 60 min at room temperature.
Cell cultures ( E. coli )
Pick a single colony from the plate or from glycerol stock.
Put the colony in a 15 ml sterile tube and add 4 – 5 ml of LB medium.
(If needed, add selective antibiotics to the medium before adding a single colony)
Incubate the tube overnight at 37°C and 220 rpm.
Cell cultures ( B. subtilis )
Pick a single colony from the plate or from glycerol stock.
Put the colony in a 15 ml sterile tube and add 3 ml of LB medium.
(If needed, add selective antibiotics to the medium before adding a single colony)
Incubate the tube overnight at 37°C and 220 rpm.
Glycerol stock
Take 800 µl from a freshly grown culture and put it in a 1.5 ml tube.
Add 200 µl of 80 % glycerol.
Mix by vortexing.
Store at -80°C.
DNA electrophoresis
Gel preparation (for 400 ml)
Measure out 4 g of agarose and add 400 ml of 1X TBE buffer (for 1 % gel).
Autoclave the solution, and let it cool down to ∼ 55°C. Be careful, agar solidifies at 32 – 40°C.
Add 4 µl of DNA stain.
Store at 55 - 60°C for repetitive use.
Gel pouring
Pour the solution into the mold including the combs.
Let it polymerize for 15 min.
Remove the combs.
Put prepared agarose gel in the electrophoresis container containing 1X TBE.
Electrophoresis
Add 1/6 of total volume of 6X loading buffer to every DNA sample.
Pipet DNA samples containing 6X loading buffer and DNA ladder to the wells.
Run at 150 – 170V for 20 – 30 min (depends on the fragments).
Preparation of 5X TBE buffer
5X TBE buffer (for 1000 ml)
Tris-base
54 g
Boric acid
27.5 g
0.5 M EDTA (pH 8.0)
20 ml
Adjust pH to 8.3 by HCl.
5X TBE buffer has to be diluted to 1X TBE buffer before using  in electrophoresis
PCR (Taq Core Kit – Jena Bioscience)
50 µl PCR assay
50 µl PCR assay
10X Taq Reaction buffer complete
5 µl
dNTP Mix
1 µl
MgCl2*
1 µl
10 µM Forward Primer
1 µl
10 µM Reverse Primer
1 µl
Template DNA
2 - 50 ng
Taq polymerase
0.2 – 0.5 µl
MiliQ water
up to 50 µl
*Adding of Mg2+ is recommended for most applications.
Cycling conditions
Cycling conditions
initial denaturation
94°C
2 min
1x
denaturation
94°C
30 sec
30x
annealing*
45 – 68°C
30 sec
30x
elongation**
72°C
30 sec – 4 min
30x
final elongation
72°C
2 min
1x
hold
4 – 10°C
*The annealing temperature depends on the melting temperature of the primers used.
**The elongation time depends on the length of the fragments to be amplified (1 min/kb).
PCR (Q5® High-Fidelity)
50 µl PCR assay
50 µl PCR assay
Q5 High-Fidelity 2X Master Mix
25 µl
10 µM Forward Primer
2.5 µl
10 µM Reverse Primer
2.5 µl
Template DNA
variable
MiliQ water
up to 50 µl
25 µl PCR assay
25 µl PCR assay
Q5 High-Fidelity 2X Master Mix
12.5 µl
10 µM Forward Primer
1.25 µl
10 µM Reverse Primer
1.25 µl
Template DNA
variable
MiliQ water
up to 25 µl
Cycling conditions
Cycling conditions
initial denaturation
98°C
30 sec
1x
denaturation
98°C
5 - 10 sec
25 - 30x
annealing*
50 – 72°C
10 - 30 sec
25 - 30x
elongation**
72°C
20 – 30 sec/kb
25 - 30x
final elongation
72°C
2 min
1x
hold
4 – 10°C
*The annealing temperature depends on the melting temperature of the primers used.
Colony PCR
Make the Master Mix using the materials specified in PCR protocols above.
Prepare a pre-glycerol stock.
Pick a colony from the plate and add it to the mixture of 80 µl of LB medium and 20 µl of 80 % glycerol. Vortex it.
Add 2 µl of pre-glycerol stock to the PCR mixture.
Run the PCR using the programs specified above.
Plasmid mini-prep (Fast-n-Easy Plasmid Mini-Prep Kit - Jena Bioscience)
Harvest the bacterial cell culture (1 – 3 ml) by centrifugation.
Resuspend pelleted bacterial in 300 µl Lysis Buffer by pipetting or vortexing for 1 min.
Add 300 µl of Neutralization Buffer (containing RNase A) to sample.
Mix it gently by inverting the tube 4 – 6 times (do not vortex).
Centrifuge at 10,000 g for 5 min at room temperature in a micro-centrifuge.
The color of the binding mixture should change to bright yellow indicating a pH of 7.5 required for optimal DNA binding.
Place a Binding Column into a 2 ml collection tube.
Add 100 µl of Activation buffer into the Binding Column.
Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.
Apply the supernatant from steps 3 – 5 into the activated Binding Column by decanting or pipetting.
Centrifuge at 10,000 g for 30 sec.
Discard the flow-through.
Place the DNA loaded Binding Column into the used 2 ml tube.
Apply 500 µl of Washing Buffer (containing Ethanol) to the Binding Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Optional Secondary Washing (Recommended only for DNA >200 bp, if highly purified DNA is required).
Add 700 µl of Washing Buffer to the Binding Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Centrifuge again for 2 min to remove residual Washing Buffer.
Place the Binding Column into a clean 1.5 ml tube (not provided in the kit).
Add 30 – 50 µl Elution Buffer or dd-water to the center of the column membrane.
Incubate for 1 min at room temperature.
Centrifuge at 10,000 g for 1 min to elute DNA.
Plasmid mini-prep (QIAprep® Spin Miniprep Kit)
Pellet 1 – 5 ml bacterial overnight culture by centrifugation at 8000 rpm (6800 x g) for 3 min at room temperature (15 – 25°C).
Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a micro-centrifuge tube.
Add 250 µl Buffer P2 and mix thoroughly by inverting the tube 4 – 6 times until the solution becomes clear.
Do not allow the lysis reaction to proceed for more than 5 min. If using LyseBlue reagent, the solution will turn blue.
Add 350 µl Buffer N3 and mix immediately and thoroughly by inverting the tube 4 – 6 times.
If using LyseBlue reagent, the solution will turn colorless.
Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table top micro-centrifuge.
Apply the supernatant from step 5 to the QIAprep spin column by decanting or pipetting.
Centrifuge for 30 – 60 sec at 13,000 rpm (~17,900 x g) and discard the flow-through.
Recommended:
Wash the QIAprep spin column by adding 500 µl Buffer PB.
Centrifuge for 30 – 60 sec at 13,000 rpm (~17,900 x g) and discard the flow-through.
Note: This step is only required when using endA+ strains or other bacterial strains with high nuclease activity or carbohydrate content.
Wash the QIAprep spin column by adding 750 µl Buffer PE.
Centrifuge for 30 – 60 sec at 13,000 rpm (~17,900 x g) and discard the flow-through.
Centrifuge for 1 min at 13,000 rpm (~17,900 x g) to remove residual wash buffer.
Place the QIAprep column in a clean 1.5 ml micro-centrifuge tube.
To elute DNA, add 50 µl Buffer EB (10 mM Tris-Cl, pH 8.5) or water to the center of the QIAprep spin column.
Let it stand for 1 min, and centrifuge for 1 min at 13,000 rpm (~17,900 x g).
DNA Clean-up (PCR Purification Kit – Jena Bioscience)
For DNA fragment sizes in the range of 200 bp to 5 kbp:
Add 5 volumes of Binding Buffer to 1 volume of DNA sample and mix well.
For example, if the volume of your DNA sample is 50 µl, add 250 µl Binding Buffer.
For DNA fragment sizes smaller than 200 bp or larger than 5 kbp:
Add 3 volumes Binding Buffer and 2 volumes of Isopropanol to the PCR sample.
For example, if the volume of your DNA sample is 50 µl, add 150 µl Binding Buffer and 100 µl Isopropanol.
Protocol
Place a Spin Column into a 2 ml collection tube.
Add 100 µl of Activation Buffer into the Spin Column.
Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.
Apply the sample mixture from step A or B into the activated Spin Column.
Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.
Discard the flow-through.
Place the DNA loaded Spin Colum into the used 2 ml tube.
Apply 700 µl of Washing Buffer to the Spin Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Optional Secondary Washing (Recommended only for DNA >200 bp, if highly purified DNA is required).
Add 700 µl of Washing Buffer to the Binding Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Centrifuge again for 2 min to remove residual Washing Buffer.
Place the Binding Column into a clean 1.5 ml tube (not provided in the kit).
Add 30 – 50 µl Elution Buffer or dd-water to the center of the column membrane.
Incubate for 1 min at room temperature.
Centrifuge at 10,000 g for 1 min to elute DNA.
DNA Clean-up (NucleoSpin® Gel and PCR Clean-up)
Mix 1 volume of sample with 2 volumes of Buffer NTI (e.g. mix 100 μl PCR reaction and 200 μl Buffer NTI)*.
Place a NucleoSpin® Gel and PCR Clean-up Column into a Collection Tube (2 ml) and load up to 700 μl sample.
Centrifuge for 30 sec at 11,000 x g.
Discard flow-through and place the column back into the collection tube.
Load remaining sample if necessary and repeat the centrifugation step.
Add 700 μl Buffer NT3 to the NucleoSpin® Gel and PCR Clean-up Column.
Centrifuge for 30 sec at 11,000 x g.
Discard flow-through and place the column back into the collection tube.
Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and improve A260/A230 values.
Centrifuge for 1 min at 11,000 x g to remove Buffer NT3 completely.
Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube**.
Place the NucleoSpin® Gel and PCR Clean-up Column into a new 1.5 ml microcentrifuge tube (not provided).
Add 15–30 μl Buffer NE***.
Incubate at room temperature (18–25°C) for 1 min.
Centrifuge for 1 min at 11,000 x g.
*For very small sample volumes < 30 μl adjust the volume of the reaction mixture to 50–100 μl with water.
**Residual ethanol from Buffer NT3 might inhibit enzymatic reactions. Total removal of ethanol can be achieved by incubating the columns for 2–5 min at 70°C prior to elution.
***DNA recovery of larger fragments (> 1000 bp) can be increased by multiple elution steps with fresh buffer, heating to 70°C and incubation for 5 min.
Gel extraction (Agarose Gel Extraction Kit – Jena Bioscience)
Cut the area of gel containing the DNA fragment.
Transfer the excised gel to a clean 1.5 ml microtube.
Add 3 volumes of Extraction Buffer to 1 volume of the sliced gel. For example, add 300 µl Extraction Buffer to each 100 mg (approx. 100 µl) gel. For gels containing >2.5 % agarose, add 6 volumes of Extraction Buffer per gel volume.
Incubate at 60 °C for 10 min with occasional mixing to ensure gel dissolution.
For DNA fragment sizes smaller than 200 bp or larger than 5 kbp and to enhance yield add 1 volume Isopropanol per gel volume to the dissolved gel and mix well.
Place a Spin Column into a 2 ml collection tube.
Add 100 µl of Activation Buffer into the Spin Column.
Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.
Apply the sample mixture from steps 3 (5), 4 into the activated Spin Column.
Centrifuge at 10,000 g for 30 sec in a micro-centrifuge.
Discard the flow-through.
Place the DNA loaded Spin Colum into the used 2 ml tube.
Apply 700 µl of Washing Buffer to the Spin Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Optional Secondary Washing (Recommended only for DNA >200 bp, if highly purified DNA is required).
Add 700 µl of Washing Buffer to the Binding Column.
Centrifuge at 10,000 g for 30 sec and discard the flow-through.
Centrifuge again for 2 min to remove residual Washing Buffer.
Place the Spin Column into a new 1.5 ml microcentrifuge tube (not provided).
Add 30 – 50 µl Elution Buffer or dd-water to the center of the column membrane.
Incubate for 1 min at room temperature.
Centrifuge at 10,000 g for 1 min to elute DNA.
Gel extraction (NucleoSpin® Gel and PCR Clean-up)
Take a clean scalpel to excise the DNA fragment from an agarose gel. Remove all excess agarose.
Determine the weight of the gel slice and transfer it to a clean tube.
For each 100 mg of agarose gel < 2 % add 200 μl Buffer NTI. (For gels containing > 2 % agarose, double the volume of Buffer NTI).
Incubate sample for 5–10 min at 50°C. Vortex the sample briefly every 2–3 min until the gel slice is completely dissolved.
Place a NucleoSpin® Gel and PCR Clean-up Column into a Collection Tube (2 ml) and load up to 700 μl sample.
Centrifuge for 30 sec at 11,000 x g.
Discard flow-through and place the column back into the collection tube.
Load remaining sample if necessary and repeat the centrifugation step.
Add 700 μl Buffer NT3 to the NucleoSpin® Gel and PCR Clean-up Column.
Centrifuge for 30 sec at 11,000 x g.
Discard flow-through and place the column back into the collection tube.
Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and low A260/A230 values.
Centrifuge for 1 min at 11,000 x g to remove Buffer NT3 completely.
Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube*.
Place the NucleoSpin® Gel and PCR Clean-up Column into a new 1.5 ml microcentrifuge tube (not provided).
Add 15–30 μl Buffer NE**.
Incubate at room temperature (18–25°C) for 1 min.
Centrifuge for 1 min at 11,000 x g.
*Residual ethanol from Buffer NT3 might inhibit enzymatic reactions. Total removal of ethanol can be achieved by incubating the columns for 2–5 min at 70°C prior to elution.
**DNA recovery of larger fragments (> 1000 bp) can be increased by multiple elution steps with fresh buffer, heating to 70°C and incubation for 5 min.
Preparation of the spore stock of B. subtilis
Inoculate 20 ml of LB medium with cells from a fresh colony of B. subtilis . Always use flasks that comprise at least 5x times the volume of media used, and always use lids that are able to allow air passage.
Culture for 6 – 8 hours at 37°C with shaking at 200 rpm. B. subtilis grows best at 37°C and has a doubling time of 30 min.
Dilute 1:200 into 1000 ml DSM medium in a sterile 1 liter flask and grow at 37°C with shaking at 200 rpm. Add 5 ml of the culture to 1000 ml medium.
Check samples for spores daily using phase contrast microscopy (see Phase Contrast Microscopy Protocol). Optional use of Gram stain (see Gram staining protocol ) to distinguish between vegetative cells (purple) and spores (transparent).
After 2 – 3 days >90% of the population should have sporulated.
Pellet cells at 9000 rpm for 20 min (keeping temperatures low), otherwise for small quantities a benchtop centrifuge will suffice.
Wash spores with ice-cold water 8 – 10 times to remove residual nutrients and lyse remaining vegetative cells. (Resuspend pellets in water, centrifuge, discard supernatant, repeat).
Store spores at -20°C for long-term storage or at 4°C, with weekly changes of water.
Difco Sporulation Medium (DSM)
DSM (for 1000 ml)
Bacto nutrient broth (Difco)
8 g
10 % (w/v) KCl
10 ml
1.2 % (w/v) MgSO4 . 7H2O
10 ml
1 M NaOH
1.5 ml
Bring pH to 7.6.
Filter sterilize.
Add the following sterile (autoclaved) component solutions to 1000 ml of the cooled DSM medium prior to use:
1 M Ca(NO3)2
1 ml
0.01 M MnCl2
1 ml
1 mM FeSO4
1 ml
Integration check: Starch test
Inoculation: Use a fresh (16- to 18-hour) pure culture of test bacteria as an inoculation source. Pick a single isolated colony and either single streak or spot inoculate the surface of the agar medium.
Incubation: Incubate plates overnight at 37°C.
Starch Hydrolysis Test: After proper inoculation and incubation, flood the surface of the agar with Gram’s iodine solution. Record results immediately as the blue color formed with starch may fade giving a false-positive result of absence of starch.
Appearance of a clear zone surrounding the bacterial growth indicates starch hydrolysis (+) by the organism due to its production of the extracellular enzymes. The clear zone will start out yellow (from the iodine) and becomes progressively lighter yellow and then clear -> indicates wrong clones.
A blue/black or purple zone surrounding the growth indicates that starch is present and has not been hydrolyzed (-) and the organism did not produce the extracellular enzymes -> indicates right clones.
Starch plates
Solution of 1000 ml
Beef extract
3 g
Soluble starch
10 g
Agar
12 g
Distilled water
1000 ml
Suspend the first three ingredients in 1000 ml of distilled water. Mix thoroughly.
Heat with frequent agitation and carefully bring to just boiling (excessive boiling may hydrolyze starch).
Autoclave.
After sterilization pour the melted medium into sterilized petri plates (approximately 20-30 ml per plate) and let it solidify before use.
IDT gBlocks®
Resuspending gBlocks Gene Fragment
Centrifuge the tube for 3 – 5 sec at a minimum of 3000 x g to ensure the material is in the bottom of the tube.
Add 1X TE to reach a final concentration of 10 ng/µl.
Vortex briefly.
Incubate at 50°C for 20 min.
Briefly vortex and centrifuge.
Amplifying gBlocks Gene Fragment
For gBlocks Gene Fragments ≤ 1kb, amplification can be performed using a high fidelity polymerase.
To avoid sequence mutations due to amplification errors, limit cycle to 12 or fewer.
For gBlocks Gene Fragments > 1kb, we do not recommend amplification.
Time-lapse microscopy/Phase-contrast microscopy
Preparation of the 1.5% agarose
Measure out 150 mg of agarose (for 1.5 % agarose) and add either 10 ml of LB medium (time-lapse) or 1X TBE buffer (phase-contrast images).
Dissolve the agarose.
Store at 55 - 60°C for repetitive use.
Preparation of the cell cultures
Pick a single colony from the plate or from glycerol stock.
Put the colony in a 15 ml sterile tube and add 3 ml of LB medium.
(If needed, add selective antibiotics to the medium before adding a single colony).
Incubate the tube overnight at 37°C and 220 rpm.
Preparation of the microscope sample [ 1 ]
Clean two microscope glass slides with 70 % ethanol and water.
Take a gene frame and carefully remove one of the plastic foils from the gene frame without causing disassembly of the plastic cover on the other side of the gene frame.
Attach the gene frame in the middle of one of the glass slides by first facilitating contact on just one side, followed by guided attachment of the remaining gene frame with a fingernail. Prevent air bubbles while attaching the gene frame to the glass slide.
Transfer 500 μl of the warm agarose-LB or agarose-TBE in the middle of the gene frame. Make sure the whole area including (the borders) is fully covered.
The following steps have to be carried out quickly to prevent excessive drying of the agarose-LB or agarose-TBE.
Place the second glass slide on the agarose-LB or agarose-TBE filled gene frame. Try to avoid air bubbles. Place the sandwiched slides in a horizontal position for 45 min at 4°C in the refrigerator to allow the agarose-LB or agarose-TBE to solidify sufficiently.
Carefully slide off the upper glass slide. Use a razor blade to cut out agar strips of ~5 mm width within the gene frame, on which the cells will be grown.
Carefully remove the second and final plastic cover from the gene frame to expose the sticky side of the gene frame.
Load single cells on the solid medium without touching it with the pipet tip. Use 2.5 μl for a whole strip, or 1 μl for a small square. Always start on top of the agarose pad and allow the liquid to disperse equally on its assigned growth area by turning the slide up and down. The slide is ready, as soon as the edges of the liquid become corrugated and movement of the liquid is no longer visible when turning the slide.
Place a clean microscope slide cover slip on the gene frame from one side to the other (avoid air bubbles). Assure complete attachment by applying pressure on the cover slip along the gene frame with your fingernail. If the cover slip is placed on the cells without allowing them to dry long enough, cells tend to grow on top of each other during the experiment. Also be careful not to wait too long before applying the cover slip, since the agarose will then be too dry.
Reference:
Jong, I. G. De, Beilharz, K., Kuipers, O. P., & Veening, J.-W. (2011). Live Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy, (July), 1–6. http://doi.org/10.3791/3145
Sequencing (Macrogen)
Template DNA of 5 μl with either of following concentrations:
50 ng/μl of PCR product.
100 ng/μl of plasmid DNA.
Add 5 μl of primer with either of following concentrations:
50 ng/μl of PCR product.
100 ng/μl of plasmid DNA.
Photometric measurement of the OD600 and mRFP fluorescence
Day 1
Grow E. coli in 3 ml LB with 35 μg/ml chloramphenicol at 37°C, 220 rpm for 16 hours.
Day 2
Preparation of the 96 well plate
Prepare a transparent 96 well plate with a total volume of 250 μl.
Provide 175 μl of LB and 25 μl of the appropriate concentration of desired compound.
Dilute cells to OD600 of 0.5 with LB and add 50 μl to each well.
Photometric measurement (Plate reader – Varioskan LUX Reader, Thermo Fischer)
To measure the fluorescence of the mRFP set the excitation and emission on 584 nm and 607 nm, respectively, with a bandwidth of 5 nm.
Measure the absorbance at 600 nm.
Measure every 20 min at 37°C with the plate reader.
Primer List
No.
Primer name
Sequence 5'-3'
Tm[°C]
1
F-key sequence
CGGAATTCGCGGCCGCTTCTAGAGG
67.9
2
R-key sequence
GCCTGCAGCGGCCGCTACTAGTACC
68.0
3
F-message sequence
GCGTCGACGACCAAGCCTGCAAAAAC
65.4
4
R-message sequence
GCAAGCTTGTCGGTGGGTGCAATGC
65.2
5
F-qnrs1 e.coli
GCAAGCTTGAATTCGCGGCCGCTTC
67.6
6
R-qnrs1 e.coli
GCGGTACCCTGCAGCGGCCGCTACTAG
70.9
7
F-qnrs1 optimized
GCAAGCTTGAATTCGCGGCCGCTTC
67.6
8
R-qnrs1 optimized
GCGGTACCCTGCAGCGGCCGCTACTAG
70.9
9
amyE-prefix
GCGGAATTCGCGGCCGCTTCTAGATGTTTGCAAAACGATTCAAAAC
73.6
10
amyE-suffix
CGTACTAGTAGCGGCCGCTGCAGTCAATGGGGAAGAGAACCGCTTAAGC
77.1
11
F-gfp insert
GCAGATCTAAATTGAATTCAACGCTCGAATGC
65.0
12
R-gfp insert
GCGCTAGCATTACGCCAAGCTTGAATCTTGCTTG
71.2
13
F message sequencing
GCGCGTACGATCTTTCAGCCGACTC
65.9
14
R message sequencing
GCGCCGCGTTTCGGTGATGAAGAT
65.3
15
F key only amplify
GCGGACCAGCTCATGATTCTCAC
60.9
16
R key only amplify
GCGAGGAGGCTTACTTGTCTGCTTTCTTC
65.9
17
amyE int check
CTCTGCCAAGTTGTTTTGATAGAGTG
59.2
18
key only + prefix
GAATTCGCGGCCGCTTCTAGAGGACCAAGCCTGCAAAAC
73.6
19
key only + suffix
CTGCAGCGGCCGCTACTAGTAGTCGGTGGGTGCTATGGAGCGACAGAA
78.3
20
message prefix
GAATTCGCGGCCGCTTCTAGAGGACCAAGCCTGCAAAAACAAAG
74.4
21
message suffix
CTGCAGCGGCCGCTACTAGTAGTCGGTGGGTGCAATGCTTC
76.0
22
pATPI+prefix
GCGAATTCGCGGCCGCTTCTAGAGTATAGGTGAAAATGTGAACATTC
72.9
23
pATPI+suffix
CTGCAGCGGCCGCTACTAGTACAATTATCTGTCTCCTGATGAA
73.0
24
prefix sfGFP(Sp)
GAATTCGCGGCCGCTTCTAGATGTCAAAAGGAGAAGAACTTTTTAC
71.3
25
suffix sfGFP(Sp)
CTGCAGCGGCCGCTACTAGTACATTATTATTTATAAAGTTCGTCCATAC
71.0
26
VF2
TGCCACCTGACGTCTAAGAA
55.4
27
VR
ATTACCGCCTTTGAGTGAGC
55.4
28
nucA+prefix
GAATTCGCGGCCGCTTCTAGATGAAAAAGATTTGGCTGGCGCTGGCTG
77.4
29
nucA+suffix
CTGCAGCGGCCGCTACTAGTATTATTGACCTGAATCAGCGTTGTC
74.2
30
Gb_insert_F2
GTACATCTTTGTAACTTTATTATACGACTGGGCCTTTCGTTTTATCTGTTG
72.0
*prefix and suffix are underlined
A day in the life of an iGEMer
Team
Members
Supervisors
Collaborations
Achievements
Contact
Jamboree
Project
Tour
Description
Design
Demonstration
Summary
Applications
Results
Future Perspectives
Biology
Experiments
Safety
Decoy experiment
Photoswitchable antibiotics
BioBrick Characterization
Key deletion
Proof of concept
Lab journal
Plasmid construction
Protocols
Primer list
Notebook
Modelling
Modelling
Mutation Rates
Random Mutation
AI Agent model
Decoding Fidelity
Optimal Strain
Computing
Software
Encoder
Decoder
Full Coding
Human Practices
Human Practices
Outreach
Social
Legal
Future Scenarios
Education
Integrated Practices
Engagement
Gold
Silver
Support
Sponsors
Attributions
