Team:Groningen/Results
CryptoGEÂ®M
Team
Members Supervisors Collaborations Achievements Contact
Project
Tour Description Design Results Demonstrate Applications Future Notebook
Biology
Experiments Parts Safety Lab journal
Computing
Modelling Software Encoding Decoding
Human Practices
Human Practices Gold Silver
Support
Sponsors Attributions
Achievements
At the Jamboree we won the track award: Best Information Processing Project, gold medal and were nominated for:
Best Software Tool
Best Education & Public Engagement
Best Wiki
View our poster , presentation slides or other results .
Medals
Register and attend
Deliverables
Attributions
Parts/Contributions
Validated part/ contribution: BBa_K1930000 , BBa_K1930001 , BBa_K1930004
Collaborations
Human Practices
Integrated Human Practices
Characterization of a previous part: BBa_K823023
Proof of concept
Demonstrate your work!
Overview of constructed BioBricks
The BioBricks that were successfully constructed during our project:
BBa_K1930000: Key in pSB1C3
BBa_K1930001: Message in pSB1C3
BBa_K1930004: qnrS1 in pSB1C3
BBa_K1930005: pATPI in pSB1C3
BBa_K1930006: sfGFP in pSB1C3
Integration and recovery of messages in DNA
We integrated an encrypted message and a key sequence into the amyE locus of the genome of two different Bacillus subtilis strains. Both Bacillus strains were treated to form spores. Subsequently the spores were allowed to germinate, the DNA was isolated and the key and message sequences were amplified using corresponding primers. The amplified DNA was sequenced and we inserted the sequencing results into our encryption and decryption machine ( Encoder and Decoder ) . We were able to recover our key and with the key we could decrypt the sequencing results for the message. We could recover our message stored in DNA! To test out the system under real life conditions we sent the message-spores and the corresponding primers to two iGEM teams in the Netherlands. They also recieved the key from us in order to encrypt the message. The iGEM teams from Eindhoven and Wageningen were both able to decrypt our message. Our concept is working!
Read more about our proof of concept.
Read more about our collaboration with Eindhoven and Wageningen.
Watch how we send the spores to the iGEM teams Wageningen and Eindhoven!
CRISPR/Cas9 key deletion
We successfully annealed the sgRNA of the message and the key. The ligation of these with the vector pJOE8999 failed. We managed to amplify the two PAM sequences of 700 bp of the B. subtilis genome. We attempted to amplify the tetR from its plasmid backbone, unfortunately the PCR failed. This subproject should be repeated from the beginning.
Read more about our CRISPR/Cas9 key deletion.
Decoy experiment
In order to test the efficiency of selecting the correct key spores from decoy we conducted flow cytometer measurements. Additionally we observed the selection efficiency with a microscope. For this experiments we used the BioBrick BBa_K1930006 sfGFP in pSB1C3. It could clearly be observed that the antibiotic selection was successfull. Though, the expected outgrowing of the decoy cells in order to hide the key-cells could not be observed. In the future this experiment should be repeated to find a concentration of decoy cells that outgrows the key cells. Also the experiment should be repeated while using spores instead of living cells to adjust the experiment to real life conditions.
Read more about the decoy experiment.
MIC and MBC of ciprofloxacin.
The MIC and MBC of ciprofloxacin on wild-type Bacillus subtilis 168 were determined to be between 160-170 nM (MIC) and 180 nM (MBC). Subsequently, the MIC of E. coli top 10 and B. subtilis 168 carrying the qnrS1 ciprofloxacin resistance gene was determined to be between 1,000 - 2,000 nM, and 400 - 500 nM, respectively. This is a significant improvement in antibiotic tolerance when compared to the MIC values of the wild-type strains (100-130 nM for E. coli and 170-180 nM for B. subtilis). Finally, the MIC value of a B. subtilis 168 isolate resistant to ciprofloxacin (obtained via directed evolution) was determined to be greater than 20,000 nM, which is a huge improvement in ciprofloxacin resistance over both the wild-type strain, and the strain carrying qnrS1. In the future, the qnrS1 and aac(6')-Ib-cr quinolone resistance genes could be introduced into this highly resistant strain for even greater ciprofloxacin resistance.
Read more about the MIC and MBC determination.
Molecular Dynamics modelling
We used molecular dynamics methods, in particular umbrella sampling, to identify a priori which strains of B. subtilis would be more susceptible to spirofloxacin. Thus, when the antibiotic was in its active state it would have a higher wild-type vs mutant killing ratio. However, due to the lack of consistent results in its experimental phase we decided not to continue with the photoswitchable antibiotic approach.
Read more about the molecular dynamics modelling.
Characterization of the  BioBrick BBa_K823023
We managed to successfully calculate the transformation efficiency of BBa_K823023 . Transformation with 10 ng and with 100 ng DNA worked well for integration. BBa_K823023 could therefore be used as integration vector for the message and key in the genome of B. subtilis.
Read more about the transformation efficiency experiment.
Encryption and Decryption machine
The encryption and decryption machine can convert data into strings of DNA. Furthermore it can encrypt and decrypt the data using the Rijndael algorithm. It can also consider forbidden sequences such as restriction enzyme cutting sites. For now, extremely long datasets take up significantly longer. Further code optimization can fix this inconvenience.
Read more about the machine.
Belief-desire-intention model (BDI)
The BDI model simulates the behavior of the potential users of CryptoGErM. Computational Agents are programmed according to an Artificial Intelligence (A.I.) architecture and placed into a virtual scenario where they compete for saving and sending a top secret message. By observing their A.I. behavior we understood how to make CryptoGErM as safe and effective as possible. We obtained a concrete proof that creating the whole system around multiple security layers was the best strategy to follow.
Read more about the BDI model.
Random mutations modelling
This model explores the role of random mutations in the Bacillus subtilis bacteria. Considering how the encrypting and storing works, we have created a software which is able to virtually replace parts of the bacteria's DNA and check if the stored data would have been affected by a mutation. Out of 1,000 simulations this happened only 3 times, so we deduced that our system is 99.97% accurate on a 556 base pair message. The software is also able to compute the maximum amount of base pairs that can be modified in the bacteria which corresponds to 400,320 bp.
Read more about the random mutations modelling.
Collaborations
We collaborate with teams from all over the world. We send our message and key to the teams Wageningen and Eindhoven, we analyzed a BioBrick for Wageningen and we filled in a few surveys. In addition we also send a survey to a lot of teams, which has also been translated to Chinese, thanks to XMU-China. Finally, we helped translating protocols to Dutch for METU HS Ankara.
Read more about our collaborations.
Education
After a survey we found that 26% of the respondents did not know what bacteria are. We set it as our aim to enhance public awareness of bacteria and of genetic engineering to overcome this problem. Therefore, we invited more than 150 students into our lab to do some Bio-Art with colourful bacteria and gave a lecture about CryptoGErM. In addition, we guided/supervised several students in their high school thesis. They did their first cloning with GFP in E. coli and B. subtilis and experienced pros and cons of  working with the different bacteria. After the Giant Jamboree we will be giving lectures about our project during a science event in Groningen.
Read more about our educational part.
Outreach
To communicate our project to general society we gave an interview on the local radio, engaged with them at the night of art and science, had a feature in the university paper and in the magazine of the biology study association. We had more visual representation through a videoclip made by Unifocus and even reached the national news agency (NOS). More in-depth communication of our project to the scientific society was conducted at a barbecue  organised for the staff of the university, workshops at the Utrecht campus party and a presentation during the Dutch Biotechnology Conference and at the Groninger Biomolecular Sciences and Biotechnology Institute (GBB).
Read more about the outreach.
Having a fun summer
Team
Members
Supervisors
Collaborations
Achievements
Contact
Jamboree
Project
Tour
Description
Design
Demonstration
Summary
Applications
Results
Future Perspectives
Biology
Experiments
Safety
Decoy experiment
Photoswitchable antibiotics
BioBrick Characterization
Key deletion
Proof of concept
Lab journal
Plasmid construction
Protocols
Primer list
Notebook
Modelling
Modelling
Mutation Rates
Random Mutation
AI Agent model
Decoding Fidelity
Optimal Strain
Computing
Software
Encoder
Decoder
Full Coding
Human Practices
Human Practices
Outreach
Social
Legal
Future Scenarios
Education
Integrated Practices
Engagement
Gold
Silver
Support
Sponsors
Attributions
