<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="content_wrapper"><HEADER id="content-header"><SPAN id="logo-container" class="menu-item"><DIV class="menu-title"><A href="/Team:Groningen">CryptoGE®M</A></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Team</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Team">Members</A><A href="/Team:Groningen/Supervisors">Supervisors</A><A href="/Team:Groningen/Collaborations">Collaborations</A><A href="/Team:Groningen/Achievements">Achievements</A><A href="/Team:Groningen/Contact">Contact</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Project</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Tour"><STRONG>Tour</STRONG></A><A href="/Team:Groningen/Description">Description</A><A href="/Team:Groningen/Design">Design</A><A href="/Team:Groningen/Results">Results</A><A href="/Team:Groningen/Demonstrate">Demonstrate</A><A href="/Team:Groningen/Applications">Applications</A><A href="/Team:Groningen/FuturePerspectives">Future</A><A href="/Team:Groningen/Notebook">Notebook</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Biology</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Experiments">Experiments</A><A href="/Team:Groningen/Parts">Parts</A><A href="/Team:Groningen/Safety">Safety</A><A href="/Team:Groningen/Labjournal">Lab journal</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Computing</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Model">Modelling</A><A href="/Team:Groningen/Software">Software</A><A href="/Team:Groningen/Encoding">Encoding</A><A href="/Team:Groningen/Decoding">Decoding</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Human Practices</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Human_Practices">Human Practices</A><A href="/Team:Groningen/HP/Gold">Gold</A><A href="/Team:Groningen/HP/Silver">Silver</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Support</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Sponsors">Sponsors</A><A href="/Team:Groningen/Attributions">Attributions</A></DIV></DIV></SPAN></HEADER><DIV id="groningen-wrapper"><ARTICLE><SECTION><H2>Design</H2><P>We designed a novel and safe data storage and transmission 
			system by combining digital and biological safety precautions. The system 
			consists of the data which is to be transmitted or preserved, 
			further called: the message, and a key to the message. The key 
			functions as a password to retrieve the message. Whilst the message 
			is encrypted and thereby protected by the digital key, the key 
			itself also needs protection. Therefore we developed a system of 
			different biological safety layers. Our CryptoGErM system is an 
			unhackable bioencryption system. Even if cyber criminals would 
			intercept the system, they would not be able to get their hands on 
			the message because they lack the key. To access the key not only 
			general knowledge about molecular biology is required. The 
			recipient has to know specific information about the exact 
			biological procedure that has to be applied in order to access the 
			key.</P><FIGCAPTION>Watch this animation to learn about the CryptoGErM system!</FIGCAPTION></SECTION><SECTION><H3 class="apps">Applications:</H3><P>CryptoGErM has three main possibilities of application. Short 
			messages can be encrypted in the genome of <EM>Bacillus subtilis</EM> as 
			described in this chapter. The genome of <EM>B. subtilis</EM> consists of 
			approximately 4 million base pairs. Two bits are converted into one 
			base pair. Thus, 1 MB could be stored in the genome of one 
			bacterium. </P><P>For sending of larger datasets it would be possible to send the 
			encrypted data via conventional methods like email and only send 
			the key in spores.</P><P>Furthermore it is an option to divide the dataset over the 
			genome of several bacteria. This would also provide a way of long 
			term storage of large datasets.</P><UL class="linkylist"><LI><A href="/Team:Groningen/Applications">Learn more about the applications of CryptoGErM</A></LI></UL></SECTION><SECTION><H3 class="crypt">Encryption and Conversion</H3><P>The message that is to be sent and stored is encrypted 
			using the Rijndael algorithm. This 
			means the message is encoded so only an authorized person can read 
			it. The plain text is being converted to cipher text. In this state 
			the message cannot be read. The algorithm requires a key for the 
			encryption process. This key can be a word just like a password. In 
			theory it is possible to decrypt the message without the key but a 
			lot of computing power, knowledge and resources are required so 
			practical it is almost impossible. With the key the message can be 
			easily decrypted. The key itself is, of course, not encrypted.</P><P>At this point of our design we have a key and message. The 
			message is encrypted and reads out as nonsense.  At first key and 
			message consist out of plain text, numbers and symbols. Our 
			encryption and decryption machine then converts those into the 
			corresponding ASCII code. The number sequence of the ASCII code is 
			then converted into binary. And the binary code is converted into a 
			sequence of base pairs. Thus 00 stands for adenine, 01 for 
			cytosine, 10 for thymidine, and 11 for guanine. The machine also 
			adds header and footer sequences to the message and key sequences 
			for the recipient to be able to recover key and message from the 
			full genome. Figure 1 shows a conversion table of text to the ASCII code, to binary and to DNA.</P><P>The work with BioBricks requires some reserved sequences like 
			the restriction enzyme sequences for EcoRI, PstI, SpeI, XbaI and 
			ideally also NotI. Additionally our work with the <EM>B. subtilis</EM> 
			shuttle vector pDR111 required the reserved sequences SalI, HindIII 
			and BglII. Our encryption and decryption machine can consider those 
			reserved sequences while converting message and key. It is also 
			individually adjustable to other required reservations.</P><FIGURE><DIV class="split"><DIV class="left flthree"><TABLE><TBODY><TR><TH>Letter</TH><TD>H</TD><TD>e</TD><TD>l</TD><TD>l</TD><TD>o</TD><TD>w</TD><TD>o</TD><TD>r</TD><TD>l</TD><TD>d</TD></TR><TR><TH>ASCII</TH><TD>072</TD><TD>101</TD><TD>108</TD><TD>108</TD><TD>111</TD><TD>032</TD><TD>119</TD><TD>111</TD><TD>114</TD><TD>108</TD><TD>100</TD></TR><TR><TH>Binary</TH><TD><DIV>0100</DIV><DIV>1000</DIV></TD><TD><DIV>0110</DIV><DIV>0101</DIV></TD><TD><DIV>0110</DIV><DIV>1100</DIV></TD><TD><DIV>0110</DIV><DIV>1100</DIV></TD><TD><DIV>0110</DIV><DIV>1111</DIV></TD><TD><DIV>0010</DIV><DIV>0000</DIV></TD><TD><DIV>0111</DIV><DIV>0111</DIV></TD><TD><DIV>0110</DIV><DIV>1111</DIV></TD><TD><DIV>0111</DIV><DIV>0010</DIV></TD><TD><DIV>0110</DIV><DIV>1100</DIV></TD><TD><DIV>0110</DIV><DIV>0100</DIV></TD></TR><TR><TH>DNA</TH><TD>ATAC</TD><TD>CCTC</TD><TD>AGTC</TD><TD>AGTC</TD><TD>GGTC</TD><TD>AATA</TD><TD>GCGC</TD><TD>GGTC</TD><TD>TAGC</TD><TD>AGTC</TD><TD>ACTC</TD></TR></TBODY></TABLE></DIV></DIV><FIGCAPTION>Figure 1. Conversion from text to DNA</FIGCAPTION></FIGURE><UL class="linkylist"><LI><A href="/Team:Groningen/Software">Learn more about our software</A></LI></UL></SECTION><SECTION><H3 class="integrate">Integration:</H3><P>Our original message and our key have been converted into DNA 
			sequences. Synthesizing DNA is possible at constantly decreasing 
			costs <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>.</P><P>Data in DNA format has many advantages. The 
			right choice of organism can deliver even more advantages. We 
			chose <EM>Bacillus subtilis.</EM> It is not pathogenic for humans, it is 
			naturally competent and a well-researched model-organism. On top of 
			all that it can form highly resistant endospores. Viable spores of 
			<EM>B. subtilis</EM> could be recovered from a 250 million year old salt 
			crystal <SUP id="ref-2"><A href="#cite-2">[2]</A></SUP>. What else could we wish for as a highly safe 
			storage and transmission system? The sequences of key and message 
			are being integrated into the genome of two different <EM>B. subtilis</EM> 
			strains by making use of the <A href="http://parts.igem.org/Part:BBa_K823023">BBa_K823023</A><EM>B. subtilis</EM> integration 
			vector which integrates into the amyE locus. A scheme of the integration process is shown in Figure 3 and the sequences integrated in the plasmid is visible in figure 3. Like this they can be 
			shipped or stored separately what makes interception of our system 
			for unauthorized parties even harder.  Genomic integration of both 
			message and key is safer than plasmid integration because the 
			relevant sequences are hidden in a full genome. Whole genome 
			sequencing is more elaborated than sequencing of plasmid, 
			especially if unauthorized parties don’t know what they are looking 
			for.</P><FIGCAPTION>Figure 2. Workflow of the integration of message and key into the genomic DNA of <EM>Bacillus subtilis</EM>.</FIGCAPTION><FIGCAPTION>Figure 3. Sequences of message (<A href="http://parts.igem.org/Part:BBa_K1930001">BBa_K1930001</A>) and key (<A href="http://parts.igem.org/Part:BBa_K1930000">BBa_K1930000</A>) integrated into the <EM>amyE</EM> locus of the genomic DNA of <EM>B. subtilis</EM>.</FIGCAPTION><UL class="linkylist"><LI><A href="/Team:Groningen/Description">Learn more about the 
				advantages of using DNA and bacteria for data storage</A></LI><LI><A href="/Team:Groningen/Proof">Learn more about the 
				integration of the sequences in to the <EM>B. subtilis</EM> 
				genome</A></LI><LI><A href="/Team:Groningen/PlasmidConstruction">Learn more 
				about the plasmid construction</A></LI></UL></SECTION><SECTION><H3 class="transmit">Transmission</H3><P>Spores are highly stable at room temperature and they don’t 
			require and nutrients. The can stay in a test tube for weeks. This 
			grants enough time to send them to any spot on Earth. And who 
			would suspect highly secret data in a tiny amount of colorless 
			liquid in a reaction tube? A scheme of sporulation and shipping can seen in Figure 4.</P><UL class="linkylist"><LI><A href="/Team:Groningen/Description">Learn more about long 
				term storage of spores</A></LI></UL><FIGCAPTION>Figure 4: <EM>Bacillus subtilis</EM> forms highly resistant endospores which can easily be shipped.</FIGCAPTION></SECTION><SECTION><H3 class="treat">Treatment</H3><P>After the different spores, containing key and message, have 
			arrived at the recipient the key has to be recovered. Only with the 
			key it is possible to decode the message in the genome of the 
			message-spores. As visible in Figure 5, the spores have to recieve the right treatment during germination, otherwise the key sequence cannot be recovered.</P><DIV class="split"><FIGURE class="left flone"><FIGCAPTION>Figure 5: Medium with spores</FIGCAPTION></FIGURE><FIGURE class="left flone"><FIGCAPTION>Wrong treatment: spores died &amp; key lost <STRONG>forever</STRONG></FIGCAPTION></FIGURE><FIGURE class="right flone"><FIGCAPTION>Right treatment: CryptoGErM lives</FIGCAPTION></FIGURE></DIV><H4>1. Decoy key hiding</H4><P>To make it especially hard for unauthorized parties to access 
			our key, the key-spores will be sent in a mixture of decoy spores. 
			The decoy is present in a much higher ratio than the key-spores. 
			The key-spores have to be recovered using a specific selection 
			mechanism. We have developed a ciprofloxacin resistance cassette 
			that can be integrated into the <EM>B. subtilis</EM> genome. Only the 
			knowledge of this specific selection antibiotic will allow the 
			recovery. Even better: This resistance cassette also provides 
			resistance against spirofloxacin which is a conjugate of the 
			regular antibiotic ciprofloxacin and a spiropyran photoswitch. In 
			case of using the photoswitchable antibiotic, spirofloxacin (in its 
			inactivated state) could be already added to the spore mixture. The 
			recipient simply has to activate it by shining light of the 
			right wavelength on it.</P><P>To design a <EM>qnrS1</EM> resistance cassette BioBrick we designed a 
			gBlock that contains the <EM>Bacillus subtilis</EM> promoter PatpI, which is 
			active from a very early stage of germination and includes a 
			ribosome binding site. The gBlock also contains the original <EM>qnrS1 </EM>
			gene sequence from <EM>E. coli</EM>, the double terminator <A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A> from 
			iGEM as well as the prefix and suffix for BioBricks. The <EM>qnrS1</EM>
			resistance cassette was integrated into the amyE locus of the <EM>B. 
			subtilis</EM> genome using the <A href="http://parts.igem.org/Part:BBa_K823023">BBa_K823023</A><EM>B. subtilis</EM> integration 
			vector. The <EM>qnrS1</EM> plasmid and the selection process of the key-containing spores from the decoy can be seen in Figure 6.</P><DIV class="centrate"><FIGCAPTION>Figure 6. The <EM>qnrS1</EM> resistance cassette (<A href="http://parts.igem.org/Part:BBa_K1930007">BBa_K1930007</A>) is integrated into the <EM>amyE</EM> locus of the <EM>B. subtilis</EM> genome. Addition of the antibiotic ciprofloxacin will enable selection of the key-containing spores from the decoy.</FIGCAPTION></DIV><P>For testing the efficiency of the selection of the correct key 
			spores from the decoy a superfolder GFP was integrated into the 
			genome of the key spores. This can be seen in figure 7. Via microscopy and flow cytometry the 
			selection efficiency could be observed. The superfolder GFP was 
			integrated into the <EM>B. subtilis</EM> genome by using the pDR111 <EM>B. 
			subtilis</EM> shuttle vector.</P><FIGCAPTION>Figure 7: Integration of a <EM>sfGFP</EM> (<A href="http://parts.igem.org/Part:BBa_K1930006">BBa_K1930006</A>) into the genomic DNA of <EM>B. subtilis</EM>.</FIGCAPTION><H4>2. NucA key deletion</H4><P>Even more safety is provided by two key deletion approaches. In 
			this case, if the recipient doesn’t apply the correct treatment, 
			the key sequence will be destroyed.</P><P>For the design of a nucA key deletion we made use of the 
			following BioBricks: <A href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A>_tetR, <A href="http://parts.igem.org/Part:BBa_K729004">BBa_K729004</A>_nuclease, 
			<A href="http://parts.igem.org/Part:BBa_B0030">BBa_B0030</A>_RBS and <A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A>_terminator. During the cloning process 
			we expected to obtain the following BioBricks: RBS-nuclease, 
			tetR-RBS-nuclease and tetR-RBS-nuclease-terminator. The key deletion 
			is controlled by the constitutively on promoter PtetR. RNA 
			polymerase binds to the DNA sequence and leads to the expression of 
			the nuclease. If tetracycline is added, it functions as a 
			repressor, binds to the promoter and thus leads to the inhibition 
			of the transcription of the nuclease. If an unauthorized party 
			tries to grow the key-containing spores in regular growth medium 
			without the addition of tetracycline, the key deletion will be 
			active and the DNA will be digested by the nuclease. This process can be seen in figure 8.</P><FIGCAPTION>Figure 8. If the right repressor is not added a nuclease will cut the genome and thus delete the key sequence.</FIGCAPTION><H4>3. CRISPR key deletion</H4><P>Another key deletion approach makes use of the CRISPR/Cas9 
			system. The system can be used as a genome editing tool. The 
			plasmid pJOE8999 is a shuttle vector with a pUC origin of 
			replication for <EM>E. coli</EM>, a kanamycin resistance gene and a 
			temperature-sensitive replication origin of plasmid pE194ts. This 
			plasmid carries the Cas9 gene under the transcriptional regulation 
			of a mannose-inducible promoter PmanP, and a single guide RNA under 
			a strong constitutive promoter. For application in our project, the 
			idea was to replace the mannose-inducible promoter with a 
			tetracycline-repressible promoter PtetR. The pTetR promoter is 
			constitutively on and is repressed by TetR. The repression can be 
			inhibited by the addition of tetracycline.  If the key-spores 
			containing the CRISPR/Cas9 system are being revived by unauthorized 
			parties who are unaware of the correct treatment procedure, the 
			targeted area, in this case the key-sequence will be deleted as visible in Figure 9. This 
			way the message will be safe and secure.</P><FIGCAPTION>Figure 9. Without the addition of the correct repressor the CRISPR/Cas9 system will delete the key sequence from the genome of <EM>B. subtilis</EM>.</FIGCAPTION><UL class="linkylist"><LI><A href="/Team:Groningen/Decoy">Learn more about the selection of the correct spores from the decoy.</A></LI><LI><A href="/Team:Groningen/SpiroModel">Learn more about the modelling of the DNA-spirofloxacin-interaction.</A></LI><LI><A href="/Team:Groningen/PhotoswitchableAntibiotics">Learn more about the photoswitchable antibiotic.</A></LI><LI><A href="/Team:Groningen/KeyDeletion">Learn more about the key deletion</A></LI></UL></SECTION><SECTION><H3 class="crypt">Decoding</H3><P>Now that the receiver has received the message and was able to 
			successfully retrieve the correct key-spores, he can sequence both 
			of them. Key and message sequence have to be put into the 
			encryption and decryption machine. The machine is able to detect 
			the correct part of the full sequence by a header (GACCAAGCCTGC) 
			and a footer (GCACCCACCGAC) that have been added to the sequence 
			will converting it into a DNA string. The machine will apply the 
			key to the message and convert it back from DNA into the data set 
			that was chosen by the sender.</P><UL class="linkylist"><LI><A href="/Team:Groningen/Software">Learn more about our 
				software</A></LI><LI><A href="/Team:Groningen/Encoding">Try the encoding &amp; 
				encryption machine</A></LI><LI><A href="/Team:Groningen/Decoding">Try the decoding &amp; 
				decryption machine</A></LI></UL></SECTION><SECTION><H3 class="model">What if?</H3><P>Making use of artificial intelligence and computational 
			modelling we have designed different scenarios taking into account 
			a possible intruder of our system and also about how the mutation 
			rate of DNA influences our system.</P><P>We have also looked at the future perspectives of our project. 
			With the great flexibility of the BioBrick system, other methods 
			for securing the data inside DNA can be added or developed. 
			Finally, we have thought of ways to access the data in the DNA 
			faster.</P><UL class="linkylist"><LI><A href="/Team:Groningen/Model">Learn more about the 
				modelling involved in CryptoGErM</A></LI><LI><A href="/Team:Groningen/FuturePerspectives">Learn more about the future perspectives of CryptoGErM</A></LI></UL></SECTION><SECTION><H2>References</H2><UL><LI id="cite-1"><A href="#ref-1">[1]</A><A href="https://www.genome.gov/sequencingcosts/">NIH National Human Genome Research Institute.</A> [Online. Accessed October 19, 2016].</LI><LI id="cite-2"><A href="#ref-2">[2]</A> Vreeland, R.H., Rosenzweig, W.D. &amp; Powers, D.W., 2000. <A href="http://www.ncbi.nlm.nih.gov/pubmed/11057666">Isolation of a 250 million-year-old halotolerant bacterium from a primary salt crystal</A>. Nature, 407(6806), pp.897–900. [Online. Accessed September 26, 2016].</LI></UL></SECTION></ARTICLE></DIV><DIV id="page-footer" class="page-footer"><SECTION><H6>Team</H6><UL><LI><A href="/Team:Groningen/Team">Members</A></LI><LI><A href="/Team:Groningen/Supervisors">Supervisors</A></LI><LI><A href="/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A href="/Team:Groningen/Achievements">Achievements</A></LI><LI><A href="/Team:Groningen/Contact">Contact</A></LI><LI><A href="/Team:Groningen/Jamboree">Jamboree</A></LI></UL></SECTION><SECTION><H6>Project</H6><UL><LI><A href="/Team:Groningen/Tour">Tour</A></LI><LI><A href="/Team:Groningen/Description">Description</A></LI><LI><A href="/Team:Groningen/Design">Design</A></LI><LI><A href="/Team:Groningen/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Groningen/Summary">Summary</A></LI><LI><A href="/Team:Groningen/Applications">Applications</A></LI><LI><A href="/Team:Groningen/Results">Results</A></LI><LI><A href="/Team:Groningen/FuturePerspectives">Future Perspectives</A></LI></UL></SECTION><SECTION class="double"><H6>Biology</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Experiments">Experiments</A></LI><LI><A href="/Team:Groningen/Safety">Safety</A></LI><LI><A href="/Team:Groningen/Decoy">Decoy experiment</A></LI><LI><A href="/Team:Groningen/PhotoswitchableAntibiotics">Photoswitchable antibiotics</A></LI><LI><A href="/Team:Groningen/BrickCharacter">BioBrick Characterization</A></LI><LI><A href="/Team:Groningen/KeyDeletion">Key deletion</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/Proof">Proof of concept</A></LI><LI><A href="/Team:Groningen/Labjournal">Lab journal</A></LI><LI><A href="/Team:Groningen/PlasmidConstruction">Plasmid construction</A></LI><LI><A href="/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="/Team:Groningen/Labjournal/PrimerList">Primer list</A></LI><LI><A href="/Team:Groningen/Notebook">Notebook</A></LI></UL></DIV></SECTION><SECTION><H6>Modelling</H6><UL><LI><A href="/Team:Groningen/Model">Modelling</A></LI><LI><A href="/Team:Groningen/MutationRates">Mutation Rates</A></LI><LI><A href="/Team:Groningen/RandomMutation">Random Mutation</A></LI><LI><A href="/Team:Groningen/AIModel">AI Agent model</A></LI><LI><A href="/Team:Groningen/DecodingFidelity">Decoding Fidelity</A></LI><LI><A href="/Team:Groningen/SpiroModel">Optimal Strain</A></LI></UL></SECTION><SECTION><H6>Computing</H6><UL><LI><A href="/Team:Groningen/Software">Software</A></LI><LI><A href="/Team:Groningen/Encoding">Encoder</A></LI><LI><A href="/Team:Groningen/Decoding">Decoder</A></LI><LI><A href="/Team:Groningen/FullCoding">Full Coding</A></LI></UL></SECTION><SECTION class="double"><H6>Human Practices</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Groningen/HP/Outreach">Outreach</A></LI><LI><A href="/Team:Groningen/HP/Social">Social</A></LI><LI><A href="/Team:Groningen/HP/Legal">Legal</A></LI><LI><A href="/Team:Groningen/HP/FutureScenarios">Future Scenarios</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/HP/Education">Education</A></LI><LI><A href="/Team:Groningen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="/Team:Groningen/Engagement">Engagement</A></LI><LI><A href="/Team:Groningen/HP/Gold">Gold</A></LI><LI><A href="/Team:Groningen/HP/Silver">Silver</A></LI></UL></DIV></SECTION><SECTION><H6>Support</H6><UL><LI><A href="/Team:Groningen/Sponsors">Sponsors</A></LI><LI><A href="/Team:Groningen/Attributions">Attributions</A></LI></UL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>