<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_CRISPR skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/CRISPR</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="content_wrapper"><HEADER id="content-header"><SPAN id="logo-container" class="menu-item"><DIV class="menu-title"><A href="/Team:Groningen">CryptoGE®M</A></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Team</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Team">Members</A><A href="/Team:Groningen/Supervisors">Supervisors</A><A href="/Team:Groningen/Collaborations">Collaborations</A><A href="/Team:Groningen/Achievements">Achievements</A><A href="/Team:Groningen/Contact">Contact</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Project</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Tour"><STRONG>Tour</STRONG></A><A href="/Team:Groningen/Description">Description</A><A href="/Team:Groningen/Design">Design</A><A href="/Team:Groningen/Results">Results</A><A href="/Team:Groningen/Demonstrate">Demonstrate</A><A href="/Team:Groningen/Applications">Applications</A><A href="/Team:Groningen/FuturePerspectives">Future</A><A href="/Team:Groningen/Notebook">Notebook</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Biology</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Experiments">Experiments</A><A href="/Team:Groningen/Parts">Parts</A><A href="/Team:Groningen/Safety">Safety</A><A href="/Team:Groningen/Labjournal">Lab journal</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Computing</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Model">Modelling</A><A href="/Team:Groningen/Software">Software</A><A href="/Team:Groningen/Encoding">Encoding</A><A href="/Team:Groningen/Decoding">Decoding</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Human Practices</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Human_Practices">Human Practices</A><A href="/Team:Groningen/HP/Gold">Gold</A><A href="/Team:Groningen/HP/Silver">Silver</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Support</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Sponsors">Sponsors</A><A href="/Team:Groningen/Attributions">Attributions</A></DIV></DIV></SPAN></HEADER><DIV id="groningen-wrapper"><ARTICLE id="key-delete" class="collapse-h2 collapse-conditional"><SECTION><H2>Key deletion</H2></SECTION><SECTION><H3>NucA key deletion</H3><P>Another biological security layer is provided by our key deletion 
		system. We have been working on two different approaches. Here we are 
		presenting the first approach, which makes use of the BioBricks in the 
		registry. Central to this approach is the nuclease nucA from BioBriock 
		<A href="http://parts.igem.org/Part:BBa_K729004">BBa_K729004</A>. This system will degrade the genomic DNA and thereby  the 
		key from the genome if no special treatment is applied. For the design 
		of a nucA key deletion system we made use of the following BioBricks: 
		<A href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A>_tetR, <A href="http://parts.igem.org/Part:BBa_K729004">BBa_K729004</A>_nuclease, <A href="http://parts.igem.org/Part:BBa_B0030">BBa_B0030</A>_RBS and 
		<A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A>_terminator. During the cloning process we expected to obtain 
		the following BioBricks: RBS-nuclease, tetR-RBS-nuclease and 
		tetR-RBS-nuclease-terminator. The key deletion is controlled by the 
		constitutively on promoter PtetR. RNA polymerase binds to the DNA 
		sequence and leads to the expression of the nuclease. If tetracycline 
		is added, it functions as a repressor, binds to the promoter and thus 
		leads to the inhibition of the transcription of the nuclease. If an 
		unauthorized party tries to grow the key-containing spores in regular 
		growth medium without the addition of tetracycline, the kill switch 
		will be active and the DNA will be digested by the nuclease. Therefore 
		the key sequence will be lost and the encrypted message stays safe. 
		Only an authorized user of CryptoGErM knows how to grow the <EM>B. subtilis</EM> 
		with tetracycline and will be able to get the key sequence. During the 
		construction of our nucA key deletion system we encountered the problem 
		that the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K729004">BioBrick_K729004</A> 
		has an incorrect prefix and suffix. See <A href="Team:Groningen/Labjournal#nucA-in-pSB1C3">plasmid 
		construction</A>. Therefore we decided to improve this part with the 
		correct prefix and suffix. See <A href="Team:Groningen/Labjournal#improved-nucA-in-pSB1C3">improved nucA 
		in pSB1C3</A>. </P></SECTION><SECTION><H3>CRISPR key deletion</H3><P>Another biological security layer is provided by our key deletion 
		system. We have been working on two different approaches. Here we are 
		presenting the second approach, which makes use of a CRISPRcas system 
		which will delete the key from the genome if no special treatment is 
		applied. Addition of Atc or tetracycline will stop the cas9 expression.</P><P>This system is highly flexible and biological layers can easily be 
		added or modified to the wishes of the use. An other goal of the 
		biological safety layer is to prevent the release of the genetically 
		modified bacteria carrying our encrypted message or key. The message 
		and key are integrated into the AmyE locus of the <EM>Bacillus subtilis</EM> 
		genome, with the use of the <A href="http://parts.igem.org/Part:BBa_K823023">BBa_K823023</A><EM>B. subtilis</EM> integration vector 
		. (See integration) This is advantageous, because it has been 
		demonstrated that the amyE locus can be deleted by using the 
		CRISPR/Cas9 system <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>.  </P><P>The Clustered Regularly Interspaced Short Palindromic Repeat 
		(CRISPR)-associated (Cas) systems are a part of the bacterial adaptive 
		immune system. Recently a genome editing tool has been designed based 
		on a type II CRISPR/Cas9 system from Streptococcus pyogenes <SUP id="ref-2"><A href="#cite-2">[2]</A></SUP>. The 
		CRISPR-Cas system needs an additional sequence next to the protospacer 
		to distinguish the difference between it’s own DNA and “foreign” DNA. 
		This sequence is the protospacer adjacent motif, also known as PAM 
		sequence. <SUP id="ref-3"><A href="#cite-3">[3]</A></SUP> The PAM motif is essential for cleavage and makes sure 
		that nearly any DNA can be targeted by crRNA, assisted by 
		trans-activating crRNA (tracrRNA) cas9 to bind to the protospacer with 
		the PAM sequence and cleaves the double-strand DNA (dsDNA). The fusion 
		of crRNA and tracrRNA is sgRNA. <SUP id="ref-4"><A href="#cite-4">[4]</A></SUP></P><FIGURE><FIGCAPTION>Fig 1. CRISPR/Cas 9 Genome Editing tool. The Cas9 
			endonuclease (blue) is targeted to DNA by a guide RNA. Target 
			recognition is facilitated by the protospacer-adjacent motif (PAM). 
			</FIGCAPTION></FIGURE><P>In our project we used a single-plasmid system to perform efficient 
		genome editing of <EM>Bacillus subtilis</EM>. The plasmid (pJOE8999) is used as 
		a shuttle vector with - a pUC origin of replication for E. coli, a 
		kanamycin resistance gene and a temperature-sensitive replication 
		origin from the plasmid pE194ts. <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP> This plasmid carries the cas9 gene 
		under the transcriptional regulation of a mannose-inducible promoter 
		PmanP. The sgRNA is transcribed under a semisynthetic promoter PvanP, 
		interrupted by the oop terminator and the T7 promoter (T7P). The cas9 
		and sgRNA genetic elements allow the plasmid to be used a genome 
		editing tool, see figure 2.  </P><FIGURE><FIGCAPTION>Fig 2. Plasmid pJOE8999 redesigned for our project. </FIGCAPTION></FIGURE><P>For application in our project, the idea was to replace the PmanP 
		with the tetracycline-repressible promoter PtetR. The pTetR promoter is 
		constitutively on and is repressed by TetR. The repression can be 
		inhibited by the addition of tetracycline <SUP id="ref-5"><A href="#cite-5">[5]</A></SUP>. </P><P>With the help of the sgRNA Designer tool (Broad Institute) we were 
		able to design the 20-nucleotide spacer sequence of the sgRNA. This 
		sgRNA guides the Cas9 to its target, in our case the key or the 
		message<SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>. The 700-bp amplified from <EM>B. subtilis</EM> serves as the DNA 
		template (PAM-motif) for AmyE. </P><P>If the spores containing the message or the key escape into the 
		environment, the CRISPR/Cas9 system will be activated due to the 
		constitutively on pTetR and the targeted area, which contains the key 
		sequence in our case, will be deleted. The same applies if the spores 
		are in possession by unauthorized parties who are unaware of adding 
		tetracycline to inactivate the pTetR promoter. This addresses the 
		issues pertaining to access of the message and key by unauthorized 
		parties, and also prevents genetically modified bacteria from spreading 
		in the environment. Therefore, this system has potential to be applied 
		in any other genetically modified <EM>B. subtilis</EM>. </P></SECTION><SECTION><H6>References</H6><UL><LI id="cite-1"><A href="#ref-1">[1]</A> Altenbucher J. Editing of the <EM>Bacillus Subtilis</EM> Genome by the CRISPR-Cas 9 system. Applied and Environmental Microbiology Vol 82, 17 </LI><LI id="cite-2"><A href="#ref-2">[2]</A> Selle K, Barrangou R. 2015. Harnessing CRISPR-Cas systems for bacterial genome editing. Trends Microbiol 23:225–232. </LI><LI id="cite-3"><A href="#ref-3">[3]</A> Jiang W, Bikard D, Cox D, Zhang F, Marraffini LA. 2013. RNA-guided
			editing of bacterial genomes using CRISPR-Cas systems. Nat Biotechnol
			31:233–239.</LI><LI id="cite-4"><A href="#ref-4">[4]</A> Jinek M, Chylinski K, Fonfara I, Hauer M, Doudna JA, Charpentier E.
			2012. A programmable dual-RNA-guided DNA endonuclease in adaptive
			bacterial immunity. Science 337:816–821.</LI><LI id="cite-5"><A href="#ref-5">[5]</A> TetR Repressible Promoter: <A href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A></LI></UL></SECTION></ARTICLE></DIV><DIV id="page-footer" class="page-footer"><SECTION><H6>Team</H6><UL><LI><A href="/Team:Groningen/Team">Members</A></LI><LI><A href="/Team:Groningen/Supervisors">Supervisors</A></LI><LI><A href="/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A href="/Team:Groningen/Achievements">Achievements</A></LI><LI><A href="/Team:Groningen/Contact">Contact</A></LI><LI><A href="/Team:Groningen/Jamboree">Jamboree</A></LI></UL></SECTION><SECTION><H6>Project</H6><UL><LI><A href="/Team:Groningen/Tour">Tour</A></LI><LI><A href="/Team:Groningen/Description">Description</A></LI><LI><A href="/Team:Groningen/Design">Design</A></LI><LI><A href="/Team:Groningen/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Groningen/Summary">Summary</A></LI><LI><A href="/Team:Groningen/Applications">Applications</A></LI><LI><A href="/Team:Groningen/Results">Results</A></LI><LI><A href="/Team:Groningen/FuturePerspectives">Future Perspectives</A></LI></UL></SECTION><SECTION class="double"><H6>Biology</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Experiments">Experiments</A></LI><LI><A href="/Team:Groningen/Safety">Safety</A></LI><LI><A href="/Team:Groningen/Decoy">Decoy experiment</A></LI><LI><A href="/Team:Groningen/PhotoswitchableAntibiotics">Photoswitchable antibiotics</A></LI><LI><A href="/Team:Groningen/BrickCharacter">BioBrick Characterization</A></LI><LI><A href="/Team:Groningen/KeyDeletion">Key deletion</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/Proof">Proof of concept</A></LI><LI><A href="/Team:Groningen/Labjournal">Lab journal</A></LI><LI><A href="/Team:Groningen/PlasmidConstruction">Plasmid construction</A></LI><LI><A href="/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="/Team:Groningen/Labjournal/PrimerList">Primer list</A></LI><LI><A href="/Team:Groningen/Notebook">Notebook</A></LI></UL></DIV></SECTION><SECTION><H6>Modelling</H6><UL><LI><A href="/Team:Groningen/Model">Modelling</A></LI><LI><A href="/Team:Groningen/MutationRates">Mutation Rates</A></LI><LI><A href="/Team:Groningen/RandomMutation">Random Mutation</A></LI><LI><A href="/Team:Groningen/AIModel">AI Agent model</A></LI><LI><A href="/Team:Groningen/DecodingFidelity">Decoding Fidelity</A></LI><LI><A href="/Team:Groningen/SpiroModel">Optimal Strain</A></LI></UL></SECTION><SECTION><H6>Computing</H6><UL><LI><A href="/Team:Groningen/Software">Software</A></LI><LI><A href="/Team:Groningen/Encoding">Encoder</A></LI><LI><A href="/Team:Groningen/Decoding">Decoder</A></LI><LI><A href="/Team:Groningen/FullCoding">Full Coding</A></LI></UL></SECTION><SECTION class="double"><H6>Human Practices</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Groningen/HP/Outreach">Outreach</A></LI><LI><A href="/Team:Groningen/HP/Social">Social</A></LI><LI><A href="/Team:Groningen/HP/Legal">Legal</A></LI><LI><A href="/Team:Groningen/HP/FutureScenarios">Future Scenarios</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/HP/Education">Education</A></LI><LI><A href="/Team:Groningen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="/Team:Groningen/Engagement">Engagement</A></LI><LI><A href="/Team:Groningen/HP/Gold">Gold</A></LI><LI><A href="/Team:Groningen/HP/Silver">Silver</A></LI></UL></DIV></SECTION><SECTION><H6>Support</H6><UL><LI><A href="/Team:Groningen/Sponsors">Sponsors</A></LI><LI><A href="/Team:Groningen/Attributions">Attributions</A></LI></UL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>