Team:Groningen/Collaborations
CryptoGE®M
Team
Members Supervisors Collaborations Achievements Contact
Project
Tour Description Design Results Demonstrate Applications Future Notebook
Biology
Experiments Parts Safety Lab journal
Computing
Modelling Software Encoding Decoding
Human Practices
Human Practices Gold Silver
Support
Sponsors Attributions
Collaborations
As a communication system, the use of CryptoGErM requires     collaboration and so, to test our system, we asked the other iGEM     teams in the Netherlands to help us with this. However, we also    collaborated with iGEM teams abroad. This article describes all our    collaborations with the other iGEM teams.
Sending our message in spores to iGEM Wageningen and Eindhoven :
The idea of CryptoGErM is to send information integrated in the     sequence of DNA. Our system is using highly resistant spores in     which the DNA is stored. Spores are then transmitted to the     receiver who can get the DNA from them and therefore the stored     information as well.  We wanted to prove that this form of     information transmission works. Therefore we decided to cooperate     with two other iGEM teams from the Netherlands and send our     CryptoGErM spores with a secret message to Wageningen and Eindhoven . Read the accompanying letter here .     Follow this link to see how we constructed the message B. subtilis strain.
Figure 1. (A) Microscopy picture of the CryptoGErM      message spores aka B. subtilis spores. (B) Tubes that were send      via mail. (C) This map indicates how the CryptoGErM spores      traveled through the Netherlands.
We went through all the 6 steps of our project. The message was     encrypted with our program and converted to a DNA sequence. This     sequence was synthesized by IDT. When this synthesized DNA sequence     was obtained, we integrated the message sequence into the genome of Bacillus subtilis. Subsequently we sporulated B. subtilis and     spores were obtained. These spores together with primers were sent     to Wageningen and Eindhoven via mail under GMO conditions. Spores     and primers were in the mail from the 6th September till the 9th.     In this case we sent primers for PCR amplification of the message     because we assumed the other teams did not include a whole genome     sequencing in their finance plan. We sent this CryptoGErM protocol     along with the message to the mentioned  teams. The short version     is, they germinate the spores on spectinomycin LB agar plates and     from those colonies the message can be acquired via colony PCR.     Primers for the PCR are provided (primer sequences can be found here ). This PCR product is sent for     sequencing. The received sequence can be copy pasted to our     decoding program. Now they only need to know the digital key that     we used to encrypt the message. We were kind enough to tell them     the digital key beforehand.
Figure 2. (A) Members from iGEM team Eindhoven 2016      decrypting our message sent in spores. (B) Also the iGEM team Wageningen is happy to decrypt our message. Blue arrow indicates decrypted message.
It works! The iGEM team Eindhoven and Wageningen could read our     message which was: “The world is full of obvious things which     nobody by any chance ever observes.” The digital key to decrypt     this message was: ”Autoclave after reading!” We hope the teams did     this with our message after the decoding.
Characterization of the BioBricks ( BBa_K1913011 ) B12 biosensor and ( BBa_K1913012 ) guanine biosensor from the iGEM team Wageningen 2016
The aim of the Wageningen iGEM 2016 project is to construct an Escherichia coli strain that effectively kills the mite Varroa     destructor within a bee hive. One step towards this goal is the     design of a system that regulates gene expression upon presence of     the V. destructor mite, which is indicated with an increase of     vitamin B12 and guanine concentration. Vitamin B12 and guanine     riboswitches are used to design two constructs that provides E.     coli with a plasmid that regulates gene expression based on the     vitamin B12 or guanine concentration. For the design, the sequence     of the btuB gene vitamin B12 riboswitch of E. coli and the sequence     of the xpt-buX operon guanine riboswitch were used. The sequences     have been elongated with fifty amino acids downstream of the     riboswitch in the original genomic sequence of B. subtilis and E.     coli to potentially increase the     stability. It is followed by the sequence of the TetR system: a     quad-part inverter based on tetracycline that inverts the signal     since both riboswitches stop gene expression upon binding their     respective metabolite. The TetR QPI inverter reverses this process     and cause protein expression when the riboswitches have bound the     ligand of interest. This is the desired outcome in the overarching     project: a V. destructor sensing bacteria would ideally start with     the production of a toxin in presence of the mite. Instead of a     toxin, mRFP is used as a reporter gene.
iGEM Wageningen 2016 sent us these two BioBricks ( BBa_K1913011 and BBa_K1913012 )  they constructed, one to detect guanine and one     for vitamin B12. We helped them to show that the RFP expression is     controlled by the concentration of either vitamin B12 or guanine.     We could show an increasing RFP signal with increasing      concentration of vitamin B12 and guanine using a photometric     measurement with the plate reader (see protocol below). We also took     pictures of the cells at the microscope with and without vitamin     B12 or guanine.
Figure 3. BioBricks from Wageningen safely arrived      in Groningen and happily grow on our agar plates.
1. Photometric measurement of the OD600 and mRFP fluorescence
Grow E. coli in 3 ml LB with 35 μl/ml chloramphenicol  at 37°C     and 220 rpm for 16 h. Prepare a transparent 96-well plate with a     total volume of 250 μl. Provide 175 μl LB and  25 μl of the     appropriate concentration of guanine or vitamin B12. Dilute cells     to an OD600 of 0.5 with LB and add 50 μl of this dilutions to each     well. To measure the fluorescence the excitation and emission of     mRFP was set on 584 nm and 607 nm, respectively, with a bandwidth     of 5 nm. The absorbance was measured at 600 nm. Measurements were     taken every 20 min at 37°C with the plate reader (Varioskan LUX     Reader, Thermo Fischer). The results can be seen in Figure 5 and     6.
Figure 4. mRFP fluorescence and OD600 for E. coli with the vitamin B12 biosensor BioBrick BBa_K19130011 measured with       different concentrations of vitamin B12 over time.
The Figure 4 shows that E. coli expresses RFP in response to the     concentration of vitamin B12 in the medium.
Figure 5. mRFP fluorescence and OD600 for E. coli with the guanine biosensor BioBrick BBa_K1913012 measured with  different      concentrations of guanine over time.
Figure 5 does not show that the expression of RFP in E. coli with the biosensor for guanine is actually correlating with its     concentration.
Figure 6. 96-well plate loaded with E. coli containing either the vitamin B12 or the guanine sensing      BioBrick after 48 h. The RFP expression for the highest      concentration of vitamin B12 is even visible by eye. See well      B11 to C6 2 mg/L and C7 to D2 0,2 mg/L.
The difference in RFP expression in response to the     concentration could even be seen by eye. See Figure 6 and a photo     of the 96-well plate used for the plate reader experiment.
2. Microscopy of the E. coli with the biosensor
Grow E. coli in 3 ml LB with 35 μl/ml chloramphenicol  at 37°C     and 220 rpm for 16 h. The control grows in LB only. The sample is     grown with 2 mg/L vitamin B12 or 0,1 mg/L guanine. 1,5 μl of cell     culture is added to a microscopy slide covered with a thin layer of     1,5 % agarose (see microscopy protocol ). Microscopy pictures were     obtained with objective: Olympus 100X/1.40, camera: CoolSNAP_HQ /     HQ2-ICX285 and software: Resolve3D softWoRx-Acquire Version was     used. Phase contrast: Filter POL 50% 0,8 s exposure and red     fluorescence: mCherry 32%, 0,8 s exposure was applied.
Vitamin B12 sensing BioBrick BBa_K1913011:
Figure 7. Microscopy pictures of E. coli with the      vitamin B12 biosensor. (A) is grown in regular LB medium. (B)      is grown in LB medium with a concentration of 2mg/L vitamin      B12.
Guanine sensing BioBrick BBa_K1913012:
Figure 8. Microscopy pictures of E. coli with the      guanine biosensor. (A) is grown in regular LB medium. (B) is      grown in LB medium with a concentration of 0,1 mg/L      guanine.
Both experiments show that the RFP is already expressed in     regular LB. This is probably due to traces of guanine and vitamin     B12 in LB medium. Nevertheless the pictures also show that the RFP     signal increases with the addition of guanine and vitamin B12.
Dutch meet up with iGEM Wageningen, Eindhoven, Leiden:
Figure 9. Our presentation at the Dutch meet up
At this meeting we shared our first presentations and gave each     other feedback. It was very useful for starting up our iGEM     projects. Social snack was included. Special thanks to Wageningen     for hosting us!
iGEM Göttingen:
Team Göttingen is aiming for the optimizing of the     biotechnological production of vitamin B12. They sent us a survey     with an interest to know what we know about the 'red gold'.     Furthermore they wanted to evaluate the general understanding of     biotechnology and which chances we see in it. This survey contained     questions for example: „Are you aware of the physiological     functions of vitamin B12  in the human body?“, „What do you think,     in which of the following processes is vitamin B12 involved?“ and     „Are you aware of the biochemical functions of Vitamin B12 in any     living cell?“.
iGEM Paris Saclay:
We did their survey and earned this badge. The survey was about       Responsible Research and Innovation: This is an initiative from the       European Commission to bring society and science together. It means       that the scientists must work during the whole research process in       order to make sure that they fulfill the values, needs and       expectations of the European society.
iGEM METU HS Ankara:
The iGEM Team Metu HS Ankara 2016 contacted us about translation       iGEM Protocols (Protocols like transformation or making competent       cells) to our own language. They wanted to create a database       consisting of protocols in different languages.
Figure 10. Badge from Paris Saclay iGEM team 2016.
iGEM Tel Hai:
iGEM Tel Hai team 2016 is working on a new cure for Cystic     Fibrosis, and as part of their project, they wanted to collaborate     with the Cystic Fibrosis Foundation and with the Cystic Fibrosis     Foundation of Israel. They decided to run a campaign called 65     roses, of which people being photographed holding a rose in order     to raise awareness among the general public and raise money for     those great foundations, just like with the ALS ice bucket     challenge.
This campaign is worldwide. It is supporting the story of a girl     diagnosed with CF, who thought for the first time she heard about     the diagnosis that she has 65 roses .     Therefore we grew a rose for her in our lab.
Figure 11. A rose from the iGEM team Groningen      2016.
Our Survey
Nowadays, digital espionage is becoming a problem. Its main goal     is to steal information of private parties and use it in an illegal     way. Our project consists of the development of a system to store     encrypted information in bacteria. In addition, the information     will be secured with several biological layers that make it     digitally uncrackable. We would really like to know how society     sees our problem, therefore we made a survey. This survey contains     questions like; “Would you send an important highly-secret letter     to someone you trust by post or email?“, „Would you trust bacteria     to store your highly-secret information?“. More about this survey?     See this link .
This survey was sent to most of the participating iGEM Teams.     The teams who participated are: University of Westminster iGEM     2016, Paris-Saclay iGEM 2016, UPO-Sevilla iGEM 2016, Paris     Bettencourt iGEM 2016, iGEM Pasteur 2016, iGEM Sheffield 2016, iGEM     Leiden 2016 and TU Delft iGEM 2016. In exchange the teams got our     collaboration badge.
Figure 12. Our badge & the XMU-China badge
Very special thanks to iGEM XMU-China for the translation of our     survey into Chinese.
Team
Members
Supervisors
Collaborations
Achievements
Contact
Jamboree
Project
Tour
Description
Design
Demonstration
Summary
Applications
Results
Future Perspectives
Biology
Experiments
Safety
Decoy experiment
Photoswitchable antibiotics
BioBrick Characterization
Key deletion
Proof of concept
Lab journal
Plasmid construction
Protocols
Primer list
Notebook
Modelling
Modelling
Mutation Rates
Random Mutation
AI Agent model
Decoding Fidelity
Optimal Strain
Computing
Software
Encoder
Decoder
Full Coding
Human Practices
Human Practices
Outreach
Social
Legal
Future Scenarios
Education
Integrated Practices
Engagement
Gold
Silver
Support
Sponsors
Attributions
