<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="content_wrapper"><HEADER id="content-header"><SPAN id="logo-container" class="menu-item"><DIV class="menu-title"><A href="/Team:Groningen">CryptoGE®M</A></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Team</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Team">Members</A><A href="/Team:Groningen/Supervisors">Supervisors</A><A href="/Team:Groningen/Collaborations">Collaborations</A><A href="/Team:Groningen/Achievements">Achievements</A><A href="/Team:Groningen/Contact">Contact</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Project</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Tour"><STRONG>Tour</STRONG></A><A href="/Team:Groningen/Description">Description</A><A href="/Team:Groningen/Design">Design</A><A href="/Team:Groningen/Results">Results</A><A href="/Team:Groningen/Demonstrate">Demonstrate</A><A href="/Team:Groningen/Applications">Applications</A><A href="/Team:Groningen/FuturePerspectives">Future</A><A href="/Team:Groningen/Notebook">Notebook</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Biology</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Experiments">Experiments</A><A href="/Team:Groningen/Parts">Parts</A><A href="/Team:Groningen/Safety">Safety</A><A href="/Team:Groningen/Labjournal">Lab journal</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Computing</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Model">Modelling</A><A href="/Team:Groningen/Software">Software</A><A href="/Team:Groningen/Encoding">Encoding</A><A href="/Team:Groningen/Decoding">Decoding</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Human Practices</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Human_Practices">Human Practices</A><A href="/Team:Groningen/HP/Gold">Gold</A><A href="/Team:Groningen/HP/Silver">Silver</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Support</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Sponsors">Sponsors</A><A href="/Team:Groningen/Attributions">Attributions</A></DIV></DIV></SPAN></HEADER><DIV id="groningen-wrapper"><ARTICLE><SECTION><H2>Collaborations</H2><P>As a communication system, the use of CryptoGErM requires 
			collaboration and so, to test our system, we asked the other iGEM 
			teams in the Netherlands to help us with this. However, we also
			collaborated with iGEM teams abroad. This article describes all our
			collaborations with the other iGEM teams.
		</P></SECTION><SECTION><H3>Sending our message in spores to iGEM <A href="https://2016.igem.org/Team:Wageningen_UR/Collaboration">Wageningen</A> and <A href="https://2016.igem.org/Team:TU-Eindhoven/Collaborations">Eindhoven</A>:</H3><P>The idea of CryptoGErM is to send information integrated in the 
			sequence of DNA. Our system is using highly resistant spores in 
			which the DNA is stored. Spores are then transmitted to the 
			receiver who can get the DNA from them and therefore the stored 
			information as well.  We wanted to prove that this form of 
			information transmission works. Therefore we decided to cooperate 
			with two other iGEM teams from the Netherlands and send our 
			CryptoGErM spores with a secret message to <A href="https://2016.igem.org/Team:Wageningen_UR/Collaboration">Wageningen</A> and 
			<A href="https://2016.igem.org/Team:TU-Eindhoven/Collaborations">Eindhoven</A>. Read the accompanying letter <A href="https://static.igem.org/mediawiki/2016/7/7d/T--Groningen--Collaborations-spore-sending-proto.pdf">here</A>. 
			Follow this <A href="https://2016.igem.org/Team:Groningen/Labjournal#Message-in-K823023">link</A> to see how we constructed the message <EM>B. subtilis</EM> strain.</P><FIGURE><FIGCAPTION>Figure 1. (A) Microscopy picture of the CryptoGErM 
				message spores aka <EM>B. subtilis</EM> spores. (B) Tubes that were send 
				via mail. (C) This map indicates how the CryptoGErM spores 
				traveled through the Netherlands.</FIGCAPTION></FIGURE><P>We went through all the 6 steps of our project. The message was 
			encrypted with our program and converted to a DNA sequence. This 
			sequence was synthesized by IDT. When this synthesized DNA sequence 
			was obtained, we integrated the message sequence into the genome of 
			<EM>Bacillus subtilis</EM>. Subsequently we sporulated <EM>B. subtilis</EM> and 
			spores were obtained. These spores together with primers were sent 
			to Wageningen and Eindhoven via mail under GMO conditions. Spores 
			and primers were in the mail from the 6th September till the 9th. 
			In this case we sent primers for PCR amplification of the message 
			because we assumed the other teams did not include a whole genome 
			sequencing in their finance plan. We sent this CryptoGErM protocol 
			along with the message to the mentioned  teams. The short version 
			is, they germinate the spores on spectinomycin LB agar plates and 
			from those colonies the message can be acquired via colony PCR. 
			Primers for the PCR are provided (primer sequences can be found 
			<A href="https://2016.igem.org/Team:Groningen/Labjournal/PrimerList">here</A>). This PCR product is sent for 
			sequencing. The received sequence can be copy pasted to our 
			decoding program. Now they only need to know the digital key that 
			we used to encrypt the message. We were kind enough to tell them 
			the digital key beforehand.</P><FIGURE><FIGCAPTION>Figure 2. (A) Members from iGEM team Eindhoven 2016 
				decrypting our message sent in spores. (B) Also the iGEM team Wageningen is happy to decrypt our message. Blue arrow indicates decrypted message. </FIGCAPTION></FIGURE><P>It works! The iGEM team Eindhoven and Wageningen could read our 
			message which was: “The world is full of obvious things which 
			nobody by any chance ever observes.” The digital key to decrypt 
			this message was: ”Autoclave after reading!” We hope the teams did 
			this with our message after the decoding. </P></SECTION><SECTION><H3>Characterization of the BioBricks (<A href="http://parts.igem.org/Part:BBa_K1913011">BBa_K1913011</A>) B12 biosensor and (<A href="http://parts.igem.org/Part:BBa_K1913012">BBa_K1913012</A>) guanine biosensor from the iGEM team Wageningen 2016</H3><P>The aim of the Wageningen iGEM 2016 project is to construct an 
			<EM>Escherichia coli</EM> strain that effectively kills the mite <EM>Varroa 
			destructor</EM> within a bee hive. One step towards this goal is the 
			design of a system that regulates gene expression upon presence of 
			the <EM>V. destructor</EM> mite, which is indicated with an increase of 
			vitamin B12 and guanine concentration. Vitamin B12 and guanine 
			riboswitches are used to design two constructs that provides <EM>E. 
			coli</EM> with a plasmid that regulates gene expression based on the 
			vitamin B12 or guanine concentration. For the design, the sequence 
			of the <EM>btuB</EM> gene vitamin B12 riboswitch of <EM>E. coli</EM> and the sequence 
			of the <EM>xpt-buX</EM> operon guanine riboswitch were used. The sequences 
			have been elongated with fifty amino acids downstream of the 
			riboswitch in the original genomic sequence of <EM>B. subtilis</EM> and <EM>E. 
			coli</EM> to potentially increase the 
			stability. It is followed by the sequence of the TetR system: a 
			quad-part inverter based on tetracycline that inverts the signal 
			since both riboswitches stop gene expression upon binding their 
			respective metabolite. The TetR QPI inverter reverses this process 
			and cause protein expression when the riboswitches have bound the 
			ligand of interest. This is the desired outcome in the overarching 
			project: a <EM>V. destructor</EM> sensing bacteria would ideally start with 
			the production of a toxin in presence of the mite. Instead of a 
			toxin, <EM>mRFP</EM> is used as a reporter gene.</P><P>iGEM Wageningen 2016 sent us these two BioBricks (<A href="http://parts.igem.org/Part:BBa_K1913011">BBa_K1913011</A> and <A href="http://parts.igem.org/Part:BBa_K1913012">BBa_K1913012</A>)  they constructed, one to detect guanine and one 
			for vitamin B12. We helped them to show that the RFP expression is 
			controlled by the concentration of either vitamin B12 or guanine. 
			We could show an increasing RFP signal with increasing  
			concentration of vitamin B12 and guanine using a photometric 
			measurement with the plate reader (see protocol below). We also took 
			pictures of the cells at the microscope with and without vitamin 
			B12 or guanine. </P><FIGURE><FIGCAPTION>Figure 3. BioBricks from Wageningen safely arrived 
				in Groningen and happily grow on our agar plates.</FIGCAPTION></FIGURE><H4>1. Photometric measurement of the OD<SUB>600</SUB> and mRFP fluorescence</H4><P>Grow <EM>E. coli</EM> in 3 ml LB with 35 μl/ml chloramphenicol  at 37°C 
			and 220 rpm for 16 h. Prepare a transparent 96-well plate with a 
			total volume of 250 μl. Provide 175 μl LB and  25 μl of the 
			appropriate concentration of guanine or vitamin B12. Dilute cells 
			to an OD<SUB>600</SUB> of 0.5 with LB and add 50 μl of this dilutions to each 
			well. To measure the fluorescence the excitation and emission of 
			mRFP was set on 584 nm and 607 nm, respectively, with a bandwidth 
			of 5 nm. The absorbance was measured at 600 nm. Measurements were 
			taken every 20 min at 37°C with the plate reader (Varioskan LUX 
			Reader, Thermo Fischer). The results can be seen in Figure 5 and 
			6.</P><FIGURE><FIGCAPTION>Figure 4. mRFP fluorescence and OD<SUB>600</SUB> for <EM>E. coli</EM>
				with the vitamin B12 biosensor BioBrick BBa_K19130011 measured with  
				different concentrations of vitamin B12 over time.</FIGCAPTION></FIGURE><P>The Figure 4 shows that <EM>E. coli</EM> expresses RFP in response to the 
			concentration of vitamin B12 in the medium.</P><FIGURE><FIGCAPTION>Figure 5. mRFP fluorescence and OD<SUB>600</SUB> for <EM>E. coli</EM>
				with the guanine biosensor BioBrick BBa_K1913012 measured with  different 
				concentrations of guanine over time.</FIGCAPTION></FIGURE><P>Figure 5 does not show that the expression of RFP in <EM>E. coli</EM> 
			with the biosensor for guanine is actually correlating with its 
			concentration. </P><FIGURE><FIGCAPTION>Figure 6. 96-well plate loaded with <EM>E. coli</EM> 
				containing either the vitamin B12 or the guanine sensing 
				BioBrick after 48 h. The RFP expression for the highest 
				concentration of vitamin B12 is even visible by eye. See well 
				B11 to C6 2 mg/L and C7 to D2 0,2 mg/L.</FIGCAPTION></FIGURE><P>The difference in RFP expression in response to the 
			concentration could even be seen by eye. See Figure 6 and a photo 
			of the 96-well plate used for the plate reader experiment. </P><H4>2. Microscopy of the <EM>E. coli</EM> with the biosensor</H4><P>Grow <EM>E. coli</EM> in 3 ml LB with 35 μl/ml chloramphenicol  at 37°C 
			and 220 rpm for 16 h. The control grows in LB only. The sample is 
			grown with 2 mg/L vitamin B12 or 0,1 mg/L guanine. 1,5 μl of cell 
			culture is added to a microscopy slide covered with a thin layer of 
			1,5 % agarose (see microscopy <A href="https://2016.igem.org/Team:Groningen/Microscopy">protocol</A>). Microscopy pictures were 
			obtained with objective: Olympus 100X/1.40, camera: CoolSNAP_HQ / 
			HQ2-ICX285 and software: Resolve3D softWoRx-Acquire Version was 
			used. Phase contrast: Filter POL 50% 0,8 s exposure and red 
			fluorescence: mCherry 32%, 0,8 s exposure was applied. </P><P>Vitamin B12 sensing BioBrick BBa_K1913011:</P><FIGURE><FIGCAPTION>Figure 7. Microscopy pictures of <EM>E. coli</EM> with the 
				vitamin B12 biosensor. (A) is grown in regular LB medium. (B) 
				is grown in LB medium with a concentration of 2mg/L vitamin 
				B12.</FIGCAPTION></FIGURE><P>Guanine sensing BioBrick BBa_K1913012:</P><FIGURE><FIGCAPTION>Figure 8. Microscopy pictures of <EM>E. coli</EM> with the 
				guanine biosensor. (A) is grown in regular LB medium. (B) is 
				grown in LB medium with a concentration of 0,1 mg/L 
				guanine.</FIGCAPTION></FIGURE><P>Both experiments show that the RFP is already expressed in 
			regular LB. This is probably due to traces of guanine and vitamin 
			B12 in LB medium. Nevertheless the pictures also show that the RFP 
			signal increases with the addition of guanine and vitamin B12. </P></SECTION><SECTION><H3>Dutch meet up with iGEM Wageningen, Eindhoven, Leiden:</H3><FIGURE><FIGCAPTION>Figure 9. Our presentation at the Dutch meet up</FIGCAPTION></FIGURE><P>At this meeting we shared our first presentations and gave each 
			other feedback. It was very useful for starting up our iGEM 
			projects. Social snack was included. Special thanks to Wageningen 
			for hosting us!</P></SECTION><SECTION><H3>iGEM Göttingen:</H3><P>Team Göttingen is aiming for the optimizing of the 
			biotechnological production of vitamin B12. They sent us a survey 
			with an interest to know what we know about the 'red gold'. 
			Furthermore they wanted to evaluate the general understanding of 
			biotechnology and which chances we see in it. This survey contained 
			questions for example: „Are you aware of the physiological 
			functions of vitamin B12  in the human body?“, „What do you think, 
			in which of the following processes is vitamin B12 involved?“ and 
			„Are you aware of the biochemical functions of Vitamin B12 in any 
			living cell?“. </P></SECTION><DIV class="split"><DIV class="left fltwo"><SECTION><H3>iGEM Paris Saclay:</H3><P>We did their survey and earned this badge. The survey was about 
					Responsible Research and Innovation: This is an initiative from the 
					European Commission to bring society and science together. It means 
					that the scientists must work during the whole research process in 
					order to make sure that they fulfill the values, needs and 
					expectations of the European society. </P></SECTION><SECTION><H3>iGEM METU HS Ankara:</H3><P>The iGEM Team Metu HS Ankara 2016 contacted us about translation 
					iGEM Protocols (Protocols like transformation or making competent 
					cells) to our own language. They wanted to create a database 
					consisting of protocols in different languages. </P></SECTION></DIV><FIGURE class="right flone"><FIGCAPTION>Figure 10. Badge from Paris Saclay iGEM team 2016.</FIGCAPTION></FIGURE></DIV><SECTION><DIV class="split"><DIV class="left flone"><H3>iGEM Tel Hai:</H3><P>iGEM Tel Hai team 2016 is working on a new cure for Cystic 
			Fibrosis, and as part of their project, they wanted to collaborate 
			with the Cystic Fibrosis Foundation and with the Cystic Fibrosis 
			Foundation of Israel. They decided to run a campaign called 65 
			roses, of which people being photographed holding a rose in order 
			to raise awareness among the general public and raise money for 
			those great foundations, just like with the ALS ice bucket 
			challenge.</P><P>This campaign is worldwide. It is supporting the story of a girl 
			diagnosed with CF, who thought for the first time she heard about 
			the diagnosis that she has <A href="https://www.youtube.com/watch?v=qV30ZynAJfw">65 roses</A>. 
			Therefore we grew a rose for her in our lab.</P></DIV></DIV><FIGCAPTION>Figure 11. A rose from the iGEM team Groningen 
				2016.</FIGCAPTION></SECTION></ARTICLE><SECTION><H3>Our Survey</H3><P>Nowadays, digital espionage is becoming a problem. Its main goal 
			is to steal information of private parties and use it in an illegal 
			way. Our project consists of the development of a system to store 
			encrypted information in bacteria. In addition, the information 
			will be secured with several biological layers that make it 
			digitally uncrackable. We would really like to know how society 
			sees our problem, therefore we made a survey. This survey contains 
			questions like; “Would you send an important highly-secret letter 
			to someone you trust by post or email?“, „Would you trust bacteria 
			to store your highly-secret information?“. More about this survey? 
			See this <A href="https://2016.igem.org/Team:Groningen/HP/Social">link</A>.</P><P>This survey was sent to most of the participating iGEM Teams. 
			The teams who participated are: University of Westminster iGEM 
			2016, Paris-Saclay iGEM 2016, UPO-Sevilla iGEM 2016, Paris 
			Bettencourt iGEM 2016, iGEM Pasteur 2016, iGEM Sheffield 2016, iGEM 
			Leiden 2016 and TU Delft iGEM 2016. In exchange the teams got our 
			collaboration badge.</P><FIGURE><FIGCAPTION>Figure 12. Our badge &amp; the XMU-China badge</FIGCAPTION></FIGURE><P>Very special thanks to iGEM XMU-China for the translation of our 
			survey into Chinese.</P></SECTION></DIV><DIV id="page-footer" class="page-footer"><SECTION><H6>Team</H6><UL><LI><A href="/Team:Groningen/Team">Members</A></LI><LI><A href="/Team:Groningen/Supervisors">Supervisors</A></LI><LI><A href="/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A href="/Team:Groningen/Achievements">Achievements</A></LI><LI><A href="/Team:Groningen/Contact">Contact</A></LI><LI><A href="/Team:Groningen/Jamboree">Jamboree</A></LI></UL></SECTION><SECTION><H6>Project</H6><UL><LI><A href="/Team:Groningen/Tour">Tour</A></LI><LI><A href="/Team:Groningen/Description">Description</A></LI><LI><A href="/Team:Groningen/Design">Design</A></LI><LI><A href="/Team:Groningen/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Groningen/Summary">Summary</A></LI><LI><A href="/Team:Groningen/Applications">Applications</A></LI><LI><A href="/Team:Groningen/Results">Results</A></LI><LI><A href="/Team:Groningen/FuturePerspectives">Future Perspectives</A></LI></UL></SECTION><SECTION class="double"><H6>Biology</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Experiments">Experiments</A></LI><LI><A href="/Team:Groningen/Safety">Safety</A></LI><LI><A href="/Team:Groningen/Decoy">Decoy experiment</A></LI><LI><A href="/Team:Groningen/PhotoswitchableAntibiotics">Photoswitchable antibiotics</A></LI><LI><A href="/Team:Groningen/BrickCharacter">BioBrick Characterization</A></LI><LI><A href="/Team:Groningen/KeyDeletion">Key deletion</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/Proof">Proof of concept</A></LI><LI><A href="/Team:Groningen/Labjournal">Lab journal</A></LI><LI><A href="/Team:Groningen/PlasmidConstruction">Plasmid construction</A></LI><LI><A href="/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="/Team:Groningen/Labjournal/PrimerList">Primer list</A></LI><LI><A href="/Team:Groningen/Notebook">Notebook</A></LI></UL></DIV></SECTION><SECTION><H6>Modelling</H6><UL><LI><A href="/Team:Groningen/Model">Modelling</A></LI><LI><A href="/Team:Groningen/MutationRates">Mutation Rates</A></LI><LI><A href="/Team:Groningen/RandomMutation">Random Mutation</A></LI><LI><A href="/Team:Groningen/AIModel">AI Agent model</A></LI><LI><A href="/Team:Groningen/DecodingFidelity">Decoding Fidelity</A></LI><LI><A href="/Team:Groningen/SpiroModel">Optimal Strain</A></LI></UL></SECTION><SECTION><H6>Computing</H6><UL><LI><A href="/Team:Groningen/Software">Software</A></LI><LI><A href="/Team:Groningen/Encoding">Encoder</A></LI><LI><A href="/Team:Groningen/Decoding">Decoder</A></LI><LI><A href="/Team:Groningen/FullCoding">Full Coding</A></LI></UL></SECTION><SECTION class="double"><H6>Human Practices</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Groningen/HP/Outreach">Outreach</A></LI><LI><A href="/Team:Groningen/HP/Social">Social</A></LI><LI><A href="/Team:Groningen/HP/Legal">Legal</A></LI><LI><A href="/Team:Groningen/HP/FutureScenarios">Future Scenarios</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/HP/Education">Education</A></LI><LI><A href="/Team:Groningen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="/Team:Groningen/Engagement">Engagement</A></LI><LI><A href="/Team:Groningen/HP/Gold">Gold</A></LI><LI><A href="/Team:Groningen/HP/Silver">Silver</A></LI></UL></DIV></SECTION><SECTION><H6>Support</H6><UL><LI><A href="/Team:Groningen/Sponsors">Sponsors</A></LI><LI><A href="/Team:Groningen/Attributions">Attributions</A></LI></UL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>