<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="content_wrapper"><HEADER id="content-header"><SPAN id="logo-container" class="menu-item"><DIV class="menu-title"><A href="/Team:Groningen">CryptoGE®M</A></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Team</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Team">Members</A><A href="/Team:Groningen/Supervisors">Supervisors</A><A href="/Team:Groningen/Collaborations">Collaborations</A><A href="/Team:Groningen/Achievements">Achievements</A><A href="/Team:Groningen/Contact">Contact</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Project</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Tour"><STRONG>Tour</STRONG></A><A href="/Team:Groningen/Description">Description</A><A href="/Team:Groningen/Design">Design</A><A href="/Team:Groningen/Results">Results</A><A href="/Team:Groningen/Demonstrate">Demonstrate</A><A href="/Team:Groningen/Applications">Applications</A><A href="/Team:Groningen/FuturePerspectives">Future</A><A href="/Team:Groningen/Notebook">Notebook</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Biology</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Experiments">Experiments</A><A href="/Team:Groningen/Parts">Parts</A><A href="/Team:Groningen/Safety">Safety</A><A href="/Team:Groningen/Labjournal">Lab journal</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Computing</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Model">Modelling</A><A href="/Team:Groningen/Software">Software</A><A href="/Team:Groningen/Encoding">Encoding</A><A href="/Team:Groningen/Decoding">Decoding</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Human Practices</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Human_Practices">Human Practices</A><A href="/Team:Groningen/HP/Gold">Gold</A><A href="/Team:Groningen/HP/Silver">Silver</A></DIV></DIV></SPAN><SPAN class="menu-item"><DIV class="menu-title">Support</DIV><DIV class="sub-menu-wrap"><DIV class="sub-menu"><A href="/Team:Groningen/Sponsors">Sponsors</A><A href="/Team:Groningen/Attributions">Attributions</A></DIV></DIV></SPAN></HEADER><DIV id="groningen-wrapper"><ARTICLE><SECTION id="intro"><H2>Description</H2><P>Every one of us is creating, storing and transmitting lots of data every 
			day. Text messages, emails, online clouds are only a few examples. 
			Since 2002 the amount of digitally stored data has exceeded the 
			amount of data stored on analog media. By now less than 6 % of the 
			world’s data is still analog <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>.</P><P>It is not surprising that data breaches orchestrated by hackers 
			are on the rise as well. Financial and legal records, military and 
			government documents, these are examples of important information 
			that must be preserved for a long time, but could cause great 
			damage in the wrong hands. Moreover, the Dutch National Cyber 
			Security Center revealed that it is us citizens who are most likely to be
			attacked by cybercrime. We have become a civilization dependent on 
			information, and this information must be stored somewhere. As a 
			result, we are faced with two problems: where do we store all of 
			our data, and how do we keep it safe?</P><DIV class="split"><DIV class="left flone"><P>In addition to the low safety of digital and analog data storage, they 
			require a large amount of resources like storage space and 
			electricity. Data is stored in huge data centers as can be seen in Figure 1. In 2015, 416.2 TWh were used for storage of digital 
			data 3 costing 41 billion USD. This is higher than the annual 
			power consumption of the entire UK <SUP id="ref-2"><A href="#cite-2">[2]</A></SUP>, and is responsible for 
			approximately 2% of global greenhouse emissions, rivaling the 
			airline industry <SUP id="ref-4"><A href="#cite-4">[4]</A></SUP>. In 2015 about 2,500,000 TB of new data were 
			produced per day <SUP id="ref-1"><A href="#cite-1">[1]</A></SUP>. While the densest storage medium in use has a 
			capacity of 10 GB/mm³, data up to a density of up to 10<SUP>9</SUP> GB/ mm³ 
			can be stored in DNA <SUP id="ref-5"><A href="#cite-5">[5]</A></SUP>. It is furthermore expected that the 
			demand for silicon, which is required for flash memory, is to exceed 
			silicon supply by 2040 <SUP id="ref-6"><A href="#cite-6">[6]</A></SUP>. </P></DIV><DIV class="right flone"><FIGCAPTION>Figure 1. A data center. (Source: BalticServers)</FIGCAPTION></DIV></DIV><P>The iGEM team Groningen 2016 thinks it is about time to develop 
			a safe and novel data transmission system. Especially as scientists 
			we are in the urgent need of that. As our team consists of 
			biologists as well as computer scientists we developed a 
			multidisciplinary approach. Safe encryption of digital data, 
			conversion into a DNA sequence and integration into a bacterial 
			genome: This approach provides a system with multiple digital and 
			biological safety layers. DNA is an infinite resource and in a 
			spore it is safe from environmental influences.</P><P>Storage of data in DNA has been proposed as early as the 1960’s, 
			but has only recently become a hot topic <SUP id="ref-7"><A href="#cite-7">[7]</A></SUP>. This is in part due 
			to the ever-growing demand for data storage, as well as 
			advancements in DNA synthesis and sequencing technologies. Our goal 
			is to create a system for long-term data storage and data transfer 
			which cannot be hacked by digital means. Digital methods of 
			encrypting information and converting it into binary code are well 
			established, and data storage in DNA has already been demonstrated. 
			Our project combines these two approaches by first converting 
			information into binary code, encrypting it, and then storing it 
			safely in DNA.  Additional measures based on molecular biology will 
			prevent unauthorized access, ensuring the safety of the stored 
			information. </P><P>Our system will be useful for the kind of information that 
			should be stored and transferred in a very secure manner, but does 
			not have to be accessed within seconds. It will be possible to 
			obtain the message in about 24-48 hours; however, this timeframe is 
			likely to be reduced as new sequencing technologies are developed. </P><DIV class="split"><DIV class="right flone"><P>DNA is a far more stable data storage medium compared to 
			magnetic and optical media, remaining intact for at least 700,000 
			years at -4 °C <SUP id="ref-8"><A href="#cite-8">[8]</A></SUP>. Even in harsh environments, DNA has a half-life 
			of over 500 years <SUP id="ref-9"><A href="#cite-9">[9]</A></SUP>. In contrast, current storage technology is rated to
			last only up to 30 years <SUP id="ref-10"><A href="#cite-10">[10]</A></SUP>. Given the stability and compactness 
			of DNA, our system could also be adapted to serve as a time capsule 
			for human knowledge. Additionally, DNA storage will soon become a 
			cheaper alternative for data storage as DNA synthesis and 
			sequencing costs drop. It is estimated to become a cost-effective 
			method for long-term data storage within approximately ten years 
			<SUP id="ref-11"><A href="#cite-11">[11]</A></SUP>.</P></DIV><DIV class="left flone"><FIGCAPTION>Figure 2. DNA. (Source: Nogas1975)</FIGCAPTION></DIV></DIV><P>DNA data storage is an apocalypse-proof technology because DNA 
			will be relevant to future civilizations. As long as intelligent 
			DNA-based life exists, there will be compelling reasons to study 
			and manipulate DNA.</P><P>Our system is especially safe as it cannot be hacked by computer 
			scientists and it requires the specific knowledge of the recipient 
			to retrieve the stored data.</P><P>As our project is designed by BioBricks it is easily adjustable to individual wishes. Message and key are easy to implement and exchange, and biological safety layers can be customized. According to the iGEM values we worked with BioBricks that we designed ourselves as well as with BioBricks of previous teams. 

Read more about how we worked on the
			<A href="/Team:Groningen/BrickCharacter">characterization of the <EM>B. subtilis</EM> integration vector BBa_K823023.</A></P></SECTION><SECTION id="history"><H3>History of data storage</H3><DIV class="split"><DIV class="left flone"><P>The concept of storing data in DNA molecules was created and 
			published in the 1960s by the Soviet physicist Mikhail Samoilovich 
			Neiman <SUP id="ref-12"><A href="#cite-12">[12]</A></SUP>. He came up with the idea that digital data can be 
			stored in the base sequence of DNA. Because DNA exists of four 
			different nucleotides, the information density can be up to two 
			times higher compared to our familiar binary storage systems. Since 
			the transition from analog to digital storage devices the storage 
			half-life of our digital information has dropped a lot. Besides 
			this current efforts to guarantee longevity of digital data storage 
			are scarce <SUP id="ref-13"><A href="#cite-13">[13]</A></SUP>. Optical and magnetic storage devices are not 
			reliable for long-term data storage. When DNA is encapsulated 
			within silica spheres and stored at -18°C it is possible to 
			recover data more than 1 million years later. But if you don’t have 
			a freezer and live in central Europe your data will be safe for up to 2k 
			years <SUP id="ref-14"><A href="#cite-14">[14]</A></SUP>. So storing information in DNA can be done with higher 
			density and it can be kept much longer.</P><P>The first messages that had actually been stored in DNA had to 
			wait until 1988 where Davis managed to do so <SUP id="ref-15"><A href="#cite-15">[15]</A></SUP>. In 2010 
			scientists were able to encode 7920 bits in synthetic DNA <SUP id="ref-16"><A href="#cite-16">[16]</A></SUP>. Up 
			to this milestone it was difficult to write and read long perfect 
			DNA sequences. In 2012 Church <EM>et al.</EM> developed a new strategy to 
			encode arbitrary digital information with an encoding scheme that 
			uses better DNA synthesis and sequencing technologies. In this 
			research they were able to encode and decode a message containing a 
			little over 50k words and a few images <SUP id="ref-17"><A href="#cite-17">[17]</A></SUP>. To learn more about data storage we visited the archives of the city Groningen (figure 3).</P></DIV><DIV class="right flone"><FIGCAPTION>Figure 3. A visit to the archives of Groningen.</FIGCAPTION></DIV></DIV></SECTION><SECTION id="encryption"><H3>Encryption</H3><P>The original message is encrypted into a new message (the 
			ciphertext) by using the Rijndael algorithm <SUP id="ref-18"><A href="#cite-18">[18]</A></SUP>, which was 
			developed in 1998 by two Belgian cryptographers, Joan Daemen and 
			Vincent Rijmen. In November 2001 this algorithm was selected by the 
			U.S. National Institute of Standards and Technology (NIST) as the 
			new Advanced Encryption Standard (AES) <SUP id="ref-19"><A href="#cite-19">[19]</A></SUP>. Since then, it has 
			been adopted by the U.S. government to secure highly classified 
			data and has been used worldwide. The process of encryption is represented in figure 4.</P><P>After encryption, the message will be converted into a binary 
			message, by making use of the American Standard Code for 
			Information Interchange (ASCII). ASCII encodes characters into 
			integers, which can be represented as sets of binary digits.</P><P>The encrypted binary message is translated into a sequence of 
			the nucleotides ACTG by using the following translation scheme: the 
			binary pair 00 will be represented as A, 10 as T, 01 as C and 11 as 
			G (see table 1). Subsequently, the obtained string of nucleotides 
			is integrated into the DNA of <EM>Bacillus subtilis</EM>, which will serve 
			as the carrier organism for our secret message. For example, table 
			2 shows the translation of the plaintext “Hello world” into a 
			sequence of nucleotides (encryption is not applied in this 
			example).</P><P>The same strategy is applied to the encryption key, which is 
			integrated into the DNA of a separate <EM>Bacillus subtilis</EM> strain in 
			the same manner. In order to retrieve the original message, the 
			message needs to be decrypted by using the same key that was used 
			for encryption.</P><DIV class="split"><FIGURE class="left flone"><TABLE><TBODY><TR><TD>00</TD><TD>A</TD></TR><TR><TD>10</TD><TD>T</TD></TR><TR><TD>01</TD><TD>C</TD></TR><TR><TD>11</TD><TD>G</TD></TR></TBODY></TABLE><FIGCAPTION>Table 1. Translation scheme: pairs of binary digits 
					are represented by nucleotides.</FIGCAPTION></FIGURE><FIGURE class="right flthree"><TABLE><TBODY><TR><TH>Letter</TH><TD>H</TD><TD>e</TD><TD>l</TD><TD>l</TD><TD>o</TD><TD>w</TD><TD>o</TD><TD>r</TD><TD>l</TD><TD>d</TD></TR><TR><TH>ASCII</TH><TD>072</TD><TD>101</TD><TD>108</TD><TD>108</TD><TD>111</TD><TD>032</TD><TD>119</TD><TD>111</TD><TD>114</TD><TD>108</TD><TD>100</TD></TR><TR><TH>Binary</TH><TD><DIV>0100</DIV><DIV>1000</DIV></TD><TD><DIV>0110</DIV><DIV>0101</DIV></TD><TD><DIV>0110</DIV><DIV>1100</DIV></TD><TD><DIV>0110</DIV><DIV>1100</DIV></TD><TD><DIV>0110</DIV><DIV>1111</DIV></TD><TD><DIV>0010</DIV><DIV>0000</DIV></TD><TD><DIV>0111</DIV><DIV>0111</DIV></TD><TD><DIV>0110</DIV><DIV>1111</DIV></TD><TD><DIV>0111</DIV><DIV>0010</DIV></TD><TD><DIV>0110</DIV><DIV>1100</DIV></TD><TD><DIV>0110</DIV><DIV>0100</DIV></TD></TR><TR><TH>DNA</TH><TD>ATAC</TD><TD>CCTC</TD><TD>AGTC</TD><TD>AGTC</TD><TD>GGTC</TD><TD>AATA</TD><TD>GCGC</TD><TD>GGTC</TD><TD>TAGC</TD><TD>AGTC</TD><TD>ACTC</TD></TR></TBODY></TABLE><FIGCAPTION>Table 2. The message “Hello world” is 
					translated into a sequence of nucleotides ACTG.</FIGCAPTION></FIGURE></DIV><FIGURE><FIGCAPTION>Figure 4. The text “Hello world!” is encrypted by 
				using a key. Subsequently, both the key and the 
				encrypted binary message are translated into 
				nucleotide-sequences. </FIGCAPTION></FIGURE></SECTION><SECTION class="synthesis"><H3>DNA Synthesis</H3><P>DNA synthesis is the natural or artificial creation of 
			deoxyribonucleic acid (DNA) molecules. In the cell, each of the two 
			strands of the DNA molecule acts as a template for the synthesis of 
			a complementary strand. Based on a similar principle, polymerase 
			chain reaction (PCR) too is being used for DNA synthesis <EM>in vitro</EM>. 
			Further, with advances made in science, it is now possible to 
			create artificially synthesized novel nucleotide DNA sequences 
			<SUP id="ref-20"><A href="#cite-20">[20]</A></SUP></P><H4>DNA replication – the natural DNA biosynthesis (in vivo DNA 
			amplification)</H4><P>DNA and DNA replication mechanisms appeared late in early life 
			history. DNA traces its origin from RNA/Protein <SUP id="ref-21"><A href="#cite-21">[21]</A></SUP>, however, to 
			remain in the scope of our current project, the natural process of 
			DNA replication proceeds in an enzymatically catalyzed and 
			coordinated steps: initiation, elongation and termination.</P><P>The first step in DNA replication involves the unzipping of the 
			double helix structure of the DNA. This is carried out by an enzyme 
			called helicase, which breaks the hydrogen bonds between the 
			complementary bases pairs of DNA (A with T, C with G). This leads 
			to the separation of two single strands of DNA, creating a ‘Y’ 
			shaped replication fork. The two separated strands serve as 
			templates for making new strands of DNA subsequently. </P><P>One of the strands is oriented in the 3’ to 5’ direction 
			(towards the replication fork), whereas the other strand is 
			oriented in the 5’ to 3’ direction (away from the replication 
			fork).  Due to this difference in their orientation, the two 
			strands replicate differently. </P><OL><LI>A short piece of RNA sequence called primer (produced by an 
				enzyme called primase) binds to the end of the leading strand 
				(3’ to 5’). The primer acts as the starting point for DNA 
				synthesis. Thereafter DNA polymerase binds to the leading 
				strand and starts adding new complementary nucleotides to the 
				template DNA in the 5’ to 3’ direction. </LI><LI>The replication process for the lagging strand (5’ to 3’) 
				involves multiple RNA primers binding at random points along 
				the template DNA. This leads to the formation of short chunks 
				of DNA in the 5’ to 3’ direction, called Okazaki fragments.</LI></OL><P>Once the bases pairs are formed (A with T, C with G), another 
			enzyme called exonuclease dissociates the primer from the DNA 
			strand. The gaps are filled by complementary nucleotides and the 
			new strand of DNA is proofread by DNA polymerase. Finally, an 
			enzyme called DNA ligase seals the sequence of DNA into two 
			continuous double strands, following which the new DNA 
			automatically winds up into a double helix. </P><H4>Polymerase chain reaction - enzymatic DNA synthesis (<EM>in vitro</EM> 
			DNA amplification)</H4><DIV class="split"><DIV class="right flone"><P>Kary Mullis invented the PCR technique (Figure 5) in 1985 <SUP id="ref-22"><A href="#cite-22">[22]</A></SUP>. His work 
			revolutionized the process of making millions of copies of a scarce 
			sample of DNA, and was awarded the Nobel Prize for Chemistry, 1993. 
			The procedure follows the basic principle of DNA replication in 
			vivo. A small amount of the DNA containing the desired gene of 
			interest is aliquoted into a tube consisting nucleotides, primers 
			(pair of synthesized short DNA segments, that match segments on 
			each side of the desired gene), DNA polymerase enzyme and a buffer 
			that allows optimal activity of the polymerase enzyme. Thereafter 
			the tube containing the mix is subjected to cycles of repeated 
			heating and cooling which leads to amplification of the gene of 
			interest.</P><OL><LI>The first step in a regular cycling event involves heating 
				the mix to 94–98 °C for 20–30 seconds. It  disrupts the 
				hydrogen bonds between the complementary base pairs, yielding 
				single strands of DNA.</LI><LI>This is followed by lowering of temperature to 50–65 °C for 
				20–40 seconds, allowing the primers to form hydrogen bonds and 
				bind to the single-stranded DNA template. The polymerase binds 
				to the primer-template hybrid and begins DNA formation.</LI><LI>Following the binding of primers, the temperature is 
				increased to about 72 °C for the DNA polymerase to synthesizes 
				a new DNA strand complementary to the DNA template strand by 
				adding dNTPs that are complementary to the template in 5' to 3' 
				direction.</LI></OL><P>The processes of denaturation, annealing and elongation 
			constitute of one cycle. Multiple cycles are required to amplify 
			DNA.</P></DIV><DIV class="left flone"><FIGCAPTION>Figure 5. A PCR set up.</FIGCAPTION></DIV></DIV><H4>Gene synthesis - physically creating artificial gene sequences 
			(<EM>de novo</EM> DNA synthesis)</H4><P>For over 60 years, the synthetic production of new DNA sequences 
			has helped researchers understand and engineer biology. Gene 
			synthesis is also accelerating research in well-established 
			research fields by providing critical advantages over more laborious 
			traditional molecular cloning techniques.  <EM>De novo</EM> DNA synthesis 
			involves the chemical synthesis of relatively short but specific 
			fragments of nucleic acids. Chemical oligonucleotide synthesis does 
			not have the limitation of unidirectional nucleotide addition (5’ 
			to 3’), as compared to the naturally occurring DNA synthesis and 
			PCR. To obtain the desired oligonucleotide, loose nucleotides are 
			sequentially added to the growing oligonucleotide chain in the 
			required sequence. Typically, synthetic oligonucleotides are 
			single-stranded DNA or RNA molecules.</P><P>The synthesis starts with a non-nucleosidic linker being 
			attached to a solid support material. The oligonucleotide sequence 
			remains covalently bound to the support material over the entire 
			course of the chain assembly via its 3'-terminal hydroxy group. The 
			chain assembly is then continued until the completion, after which 
			the release of the oligonucleotides occurs by the hydrolytic 
			cleavage of a P-O bond that attaches the 3’-O of the 3’-terminal 
			nucleotide residue to the universal linker. </P><P>Synthetic genes offer several advantages over cloned native DNA. 
			These sequences are subjected to stringent quality checks to match 
			100% sequence verification by the private companies involved in 
			synthesis of synthetic DNA. Moreover, artificial DNA synthesis 
			allows the flexibility to researchers for changing enzyme 
			specificities and activities to suit the needs of their 
			experiments. Also, synthesis of specific sequences allows the 
			insertion of localization signals to target specific 
			protein/nucleic acid <EM>in vivo.</EM></P></SECTION><SECTION id="decoding"><H3>Decoding – costs and fidelity</H3><P>The cost of sequencing DNA has plummeted in the last two 
			decades. For instance, in the early 2000’s it took 13 years and $3 
			billion US dollars to sequence the entire human genome. With 
			current technologies, we have approached the $1,000 dollars mark. This development can be seen in figure 6. In 
			fact, since 2007 you can have your whole DNA sequenced for less 
			than that! However, there are still issues with the fidelity and 
			accuracy of the readings obtained that would prevent them from 
			being used for our system <SUP id="ref-27"><A href="#cite-27">[27]</A></SUP>.</P><P><FIGCAPTION>Figure 6. Price of sequencing one million of base pairs. Prices dropped 
			from 2007, when second generation techniques were introduced in the 
			marked. Source: National Human Genome Research Institute</FIGCAPTION></P><P>Existing laboratory-level DNA sequencing technologies typically 
			allow for a reading error of ~1%. Thanks to optimization and 
			fine-tunning, traditional readings using Sanger biochemistry offer 
			now accuracies of up to 99.999% in a read length of 1,000 bp. With 
			modern, second generation or cyclic sequencing, higher reading 
			lengths have been achieved but with a decrease in accuracy <SUP id="ref-26"><A href="#cite-26">[26]</A></SUP>.</P><P>Among the bioencryption layers, CryptoGErM prevents unauthorized 
			parties from reading the message by having a high ratio of decoy 
			spores that contain a useless sequence. In addition, if the right 
			growing conditions are not supplied our system prevents germination 
			and replication of spores that do contain the message. So, what 
			ratio of decoy:spores do I need to prevent brute force sequencing and 
			message retrieval from a third party? (see figure 7)</P><FIGURE><FIGCAPTION> Figure 7. A) One of the bioencryption bilayers is having a 
				high proportion of decoy spores vs those that do contain the 
				intended useful sequence. If the right conditions are not met 
				(i.e. adding an X antibiotic to the system) those spores that 
				contain our message will die and be outgrown by the decoy. B) 
				Current sequencing techniques will not be able to distinguish 
				the hidden message from noise.</FIGCAPTION></FIGURE><P>Fox <EM>et. al.</EM> (2014) reports that there is a 50% chance of 
			accurately distinguishing a true subclonal variant from a 
			sequencing artifact in an excess of 100 wild-type DNA sequences 
			using standard Q30 filter reads (error rate: 10-2) <SUP id="ref-23"><A href="#cite-23">[23]</A></SUP>. In 
			agreement with that result, in an experiment carried out to detect 
			genomic variations in marine pests, Pochon (2013) was able to 
			detect one variant out of 150 wild-type sequences <SUP id="ref-24"><A href="#cite-24">[24]</A></SUP>.</P><P>Reading DNA has become increasingly more accurate. Schmidt 
			(2012) developed a method called Duplex Sequencing that uses both 
			strands of DNA to obtain a more precise consensus sequence yielding 
			an theoretical error of 3x10<SUP>-10</SUP><SUP id="ref-25"><A href="#cite-25">[25]</A></SUP>. That means that we could 
			transmit a message with a length in the order of gigabytes without 
			expecting any loss! On the other hand, that also means that they 
			allow a more precise measurement in decoy-spores mixtures, and a 
			1:150 spore:decoy ratio might be insufficient in the future. In 
			fact, using Duplex Sequencing they were able to identify one mutant 
			sequence per 10,000 wild-type molecules.</P></SECTION><SECTION id="integration"><H3>Integration in <EM>B. subtilis</EM></H3>
			
			The chassis <EM>B. subtilis</EM> 168
			
			<P>The key and message sequence are stored in <EM>Bacillus subtilis</EM> (<EM>B. 
			subtilis</EM>), a gram-positive, catalase positive bacteria, usually 
			found in soil and gastrointestinal tracts. This rod-shaped 
			bacterium is about 4-10  µm in length and has a diameter of about 
			0.25–1.0  µm <SUP id="ref-28"><A href="#cite-28">[28]</A></SUP>. The cell is heavily flagellated, allowing the 
			microbe to move quickly in liquid medium. It is one of the most 
			well studied gram-positive microbe, and is one of the widely 
			adopted model organism to study bacterial cell differentiation and 
			sporulation. These cells are amenable to a wide array of genetic 
			manipulations, and the ease of transformation has allowed <EM>B. 
			subtilis</EM> to be used for selective protein expression to suit our 
			requirements. Moreover, the integration of DNA sequences into its 
			genome is well known, which has been an alternative to using 
			plasmids. The process of integration is demonstrate in figure 8.</P><FIGURE><FIGURE><FIGCAPTION>Figure 8. Integration of key and message sequence into the genomic DNA of <EM>Bacillus subtilis</EM>. </FIGCAPTION></FIGURE><SECTION id="storing-sending"><H3>Storing and sending</H3><H4>Spores</H4><P><EM>B. subtilis</EM> form endospores when environmental factors do not 
			favour survival or reproduction <SUP id="ref-29"><A href="#cite-29">[29]</A></SUP>. These endospores are highly 
			resistant and durable structures. They have a central cytoplasmic 
			core where the DNA and ribosomes are protected by an impermeable 
			and rigid coat. Spores, when released in the environment, can 
			survive extreme heat and freezing, lack of water, high pressure  
			exposure to many toxic chemicals and certain radiations. Compared 
			to the vegetative cells, endospores are contained in a thicker cell 
			wall along with additional layers that make them last long periods 
			in its dehydrated metabolically inactive state.</P><H4>Sporulation</H4><P>The process of endospore formation within a <EM>B. subtilis</EM> can take 
			up several hours to complete and is called sporulation or 
			sporogenesis. Sporulation can be induced manually in the lab by 
			limiting the availability of a key nutrient, such as the carbon or 
			nitrogen source.</P><P>Sporulation begins with a small portion of cytoplasm, along with 
			a newly replicated bacterial chromosome, is isolated by an 
			ingrowth of the plasma membrane, called spore septum. The spore 
			gets a double-layered membrane that will surround the chromosome 
			and cytoplasm. This structure is entirely enclosed within the 
			original cell, and is called forespore.</P><P>Peptidoglycan layers are laid down between the two membrane 
			layers. A thick spore coat is formed around the outside membrane, 
			which is responsible for the resistance of endospores. The last 
			stage of sporulation is the degradation of the original cell and 
			the release from the endospore <SUP id="ref-30"><A href="#cite-30">[30]</A></SUP>.</P><H4>Germination</H4><P>An endospore returns to its vegetative state by a process called 
			germination. This process is triggered by physical or chemical cell 
			damage to the endospore coat. The enzymes of the endospores then 
			break down the extra layers surrounding the endospore. The water 
			enters and the metabolism resumes. <SUP id="ref-30"><A href="#cite-30">[30]</A></SUP></P><H4>Natural Competence </H4><P>The natural competence of <EM>B. subtilis</EM> is one of its less used 
			advantages. Competent <EM>B. subtilis</EM> can actively pull DNA fragments 
			from their environment. These uptaken nucleotides change the 
			genotype by homologous recombination, also known as natural 
			transformation. In order for <EM>B. subtilis</EM> to integrate DNA from 
			medium, the cells can synthesize a specific DNA-binding and uptake 
			system as seen in figure 9. In this figure there are a few proteins 
			which form the translocation complex drawn (A, NucA; C, ComC; E, 
			ComE; F, ComF; G, ComG; CW, cell wall; CM, cell membrane; CYT, 
			cytoplasm). This system has no specifity for DNA, therefore <EM>B. 
			subtilis</EM> can take integrate plasmid DNA, phage DNA or chromosomal 
			DNA. <SUP id="ref-31"><A href="#cite-31">[31]</A></SUP></P><FIGCAPTION>Figure 9. Natural competence of <EM>B. subtilis</EM></FIGCAPTION><H4>Storing of spores</H4><P>Bacterial spores are tough, non-reproductive structures produced 
			by bacteria. They are highly resistant to aging, radiation, heat 
			and chemical damage. Endospores formed by <EM>Bacillus subtilis</EM> could 
			be found viable after millions of years <SUP id="ref-32"><A href="#cite-32">[32]</A></SUP>. These properties make 
			them the ideal storage medium for data in DNA. </P><P>During freeze-drying water is removed from a substance to 
			increase the storage-life and to make shipping easier. The most 
			effective method for long-term storage and thus for shipping of 
			<EM>Bacillus subtilis</EM> spores appears to be freeze-drying <SUP id="ref-33"><A href="#cite-33">[33]</A></SUP>. 
			Freeze-drying is commonly used for long-term storage of bacteria 
			<SUP id="ref-34"><A href="#cite-34">[34]</A></SUP> and spores of <EM>B. subtilis</EM>. On top of the fact that spores are 
			highly resistant under different harsh conditions, Fairhead et al. 
			showed that spores of <EM>B. subtilis</EM> are very resistant to several 
			cycles of freeze-drying <SUP id="ref-35"><A href="#cite-35">[35]</A></SUP>. The sending process is demonstrated in Figure 10.</P><FIGCAPTION>Figure 10. Sending of message and key in spores. </FIGCAPTION></SECTION><SECTION><H6>References:</H6><UL><LI id="cite-1"><A href="#ref-1">[1]</A><A href="http://www.nbcnews.com/id/41516959/ns/technology_and_science-innovation/t/worlds-shift-analog-digital-nearly-complete/#.V-0PS_CLTIV">“Shift from analog to digital is nearly complete</A> - Technology &amp; science - Innovation | NBC News.” [Online].</LI><LI id="cite-2"><A href="#ref-2">[2]</A><A href="http://www.independent.co.uk/environment/global-warming-data-centres-to-consume-three-times-as-much-energy-in-next-decade-experts-warn-a6830086.html">“Global warming: Data centres to consume three times as much energy in next decade, experts warn</A> | The Independent.” [Online].</LI><LI id="cite-4"><A href="#ref-4">[4]</A> GeSI SMARTer 2020: The Role of ICT in Driving a Sustainable Future. http://gesi.org/assets/js/lib/tinymce/jscripts/tiny_mce/plugins/ajaxfilemanager/uploaded/SMARTer2020-report.pdf</LI><LI id="cite-5"><A href="#ref-5">[5]</A> J. Bornholt, R. Lopez, D. M. Carmean, L. Ceze, G. Seelig, and K. Strauss, “A DNA-Based Archival Storage System.”</LI><LI id="cite-6"><A href="#ref-6">[6]</A> V. Zhirnov, R. M. Zadegan, G. S. Sandhu, G. M. Church, and W. L. Hughes, “Nucleic acid memory.,” Nat. Mater., vol. 15, no. 4, pp. 366–70, Apr. 2016.</LI><LI id="cite-7"><A href="#ref-7">[7]</A> Some fundamental issues of microminiaturization Radiotekhnika, 1964, No. 1, pp. 3-12</LI><LI id="cite-8"><A href="#ref-8">[8]</A> Ancient DNA: Towards a million-year-old genome. Nature 499, 34–35 (04 July 2013) DOI:10.1038/nature12263</LI><LI id="cite-9"><A href="#ref-9">[9]</A> The half-life of DNA in bone: measuring decay kinetics in 158 dated fossils. Proc Biol Sci. 2012 Dec 7;279(1748):4724-33DOI: 10.1098/rspb.2012.1745</LI><LI id="cite-10"><A href="#ref-10">[10]</A> A DNA-Based Archival Storage System. ASPLOS 2016 DOI: http://dx.doi.org/10.1145/2872362.2</LI><LI id="cite-11"><A href="#ref-11">[11]</A>Synthetic double-helix faithfully stores Shakespeare's sonnets. NatureDOI:10.1038/nature.2013.12279</LI><LI id="cite-12"><A href="#ref-12">[12]</A><A href="https://sites.google.com/site/msneiman1905/eng">“English - М.С. Нейман.”</A> [Online. Accessed: 22-Sep-2016].</LI><LI id="cite-13"><A href="#ref-13">[13]</A>M. Hilbert and P. López, “The world’s technological capacity to store, communicate, and compute information,” science, vol. 332, no. 6025, pp. 60–65, 2011.</LI><LI id="cite-14"><A href="#ref-14">[14]</A>R. N. Grass, R. Heckel, M. Puddu, D. Paunescu, and W. J. Stark, “Robust Chemical Preservation of Digital Information on DNA in Silica with Error-Correcting Codes,” Angew. Chem. Int. Ed., vol. 54, no. 8, pp. 2552–2555, Feb. 2015.</LI><LI id="cite-15"><A href="#ref-15">[15]</A>J. Davis, “Microvenus,” Art J., vol. 55, no. 1, pp. 70–74, 1996.</LI><LI id="cite-16"><A href="#ref-16">[16]</A>D. G. Gibson, J. I. Glass, C. Lartigue, V. N. Noskov, R.-Y. Chuang, M. A. Algire, G. A. Benders, M. G. Montague, L. Ma, M. M. Moodie, C. Merryman, S. Vashee, R. Krishnakumar, N. Assad-Garcia, C. Andrews-Pfannkoch, E. A. Denisova, L. Young, Z.-Q. Qi, T. H. Segall-Shapiro, C. H. Calvey, P. P. Parmar, C. A. Hutchison, H. O. Smith, and J. C. Venter, “Creation of a Bacterial Cell Controlled by a Chemically Synthesized Genome,” Science, vol. 329, no. 5987, pp. 52–56, Jul. 2010.</LI><LI id="cite-17"><A href="#ref-17">[17]</A>G. M. Church, Y. Gao, and S. Kosuri, “Next-Generation Digital Information Storage in DNA,” Science, vol. 337, no. 6102, pp. 1628–1628, Sep. 2012.</LI><LI id="cite-18"><A href="#ref-18">[18]</A> J. Daemen. V. Rijmen. &quot;AES Proposal: Rijndael&quot;. National Institute of Standards and Technology. p. 1. March 9, 2003.</LI><LI id="cite-19"><A href="#ref-19">[19]</A> &quot;Announcing the ADVANCED ENCRYPTION STANDARD (AES)&quot;. Federal Information Processing Standards Publication 197. United States National Institute of Standards and Technology (NIST). November 26, 2001.</LI><LI id="cite-20"><A href="#ref-20">[20]</A> Kosuri, S. and Church, G. 2014. Large-scale de novo DNA synthesis: technologies and applications. Nature Methods. 11, 5 (2014), 499–507.</LI><LI id="cite-21"><A href="#ref-21">[21]</A> Andras, P. and Andras, C. 2005. The origins of life – the “protein interaction world” hypothesis: protein interactions were the first form of self-reproducing life and nucleic acids evolved later as memory molecules. Medical Hypotheses. 64, 4 (2005), 678–688.</LI><LI id="cite-22"><A href="#ref-22">[22]</A> Mullis KB et al. &quot;Specific enzymatic amplification of DNA in vitro: the polymerase chain reaction.&quot; Cold Spring Harbor Symp. Quant. Biol. vol. 51 pp. 263–73 (1986)</LI><LI id="cite-23"><A href="#ref-23">[23]</A> Fox EJ, Reid-Bayliss KS, Emond MJ, Loeb LA (2014) Accuracy of Next Generation Sequencing Platforms. Next Generat Sequenc &amp; Applic 1: 106. doi:10.4172/jngsa.1000106</LI><LI id="cite-24"><A href="#ref-24">[24]</A> Pochon (2013). Evaluating detection limits of next-generation sequencing for the surveillance and monitoring of international marine pests. PLos One 8(9):e73935</LI><LI id="cite-25"><A href="#ref-25">[25]</A> Schmitt MW, Kennedy SR, Salk JJ, Fox EJ, Hiatt JB, et al. (2012) Detection of ultra-rare mutations by next-generation sequencing. Proc Natl Acad Sci U S A 109: 14508-14513.</LI><LI id="cite-26"><A href="#ref-26">[26]</A> Wang, X., Blades, N., Ding, J., Sultana, R. and Parmigiani, G. (2012). Estimation of sequencing error rates in short reads. BMC Bioinformatics, 13:185</LI><LI id="cite-27"><A href="#ref-27">[27]</A> Zhu, X., Wang, J., Peng, B. and Shete, S. (2016). Empirical estimation of sequencing error rates using smoothing splines. BMC Bioinformatics, 17:177</LI><LI id="cite-28"><A href="#ref-28">[28]</A>Yu, Allen Chi-Shing; Loo, Jacky Foo Chuen; Yu, Samuel; Kong, Siu Kai; Chan, Ting-Fung (2013). &quot;Monitoring bacterial growth using tunable resistive pulse sensing with a pore-based technique&quot;. Applied microbiology and biotechnology. 98 (2): 855–862.</LI><LI id="cite-29"><A href="#ref-29">[29]</A> Nicholson WL, Munakata N, Horneck G, Melosh HJ, Setlow P (2000). &quot;Resistance of Bacillus endospores to extreme terrestrial and extraterrestrial environments&quot;. Microbiology and Molecular Biology Reviews. 64 (3): 548–72.</LI><LI id="cite-30"><A href="#ref-30">[30]</A> Gerard J. Tortora Microbiology: An Introduction, Global Edition12th Revised Edition, July 2015, 9781292099149, Pearson Education Limited</LI><LI id="cite-31"><A href="#ref-31">[31]</A> Leendert W. Hamoen, Gerard Venema and Oscar P. Kuipers (2003). Controlling competence in Bacillus subtilis: shared use of regulators. Microbiology (2003), 149, 9–17 </LI><LI id="cite-32"><A href="#ref-32">[32]</A> Vreeland, R.H., Rosenzweig, W.D. &amp; Powers, D.W., 2000. <A href="http://www.ncbi.nlm.nih.gov/pubmed/11057666">Isolation of a 250 million-year-old halotolerant bacterium from a primary salt crystal</A>. Nature, 407(6806), pp.897–900. [Online. Accessed September 26, 2016].</LI><LI id="cite-33"><A href="#ref-33">[33]</A> Lacey, L.A., 2012. Manual of Techniques in Invertebrate Pathology., Elsevier Science.</LI><LI id="cite-34"><A href="#ref-34">[34]</A> Miyamoto-Shinohara, Y. et al., 2008. Survival of freeze-dried bacteria. The Journal of general and applied microbiology, 54(1), pp.9–24. Available at: http://www.ncbi.nlm.nih.gov/pubmed/18323678 [Accessed September 26, 2016].</LI><LI id="cite-35"><A href="#ref-35">[35]</A> Fairhead, H. et al., 1994. <A href="http://www.ncbi.nlm.nih.gov/pubmed/8074535">Small, acid-soluble proteins bound to DNA protect Bacillus subtilis spores from being killed by freeze-drying</A>. Applied and environmental microbiology, 60(7), pp.2647–9. [Online. Accessed September 26, 2016].</LI></UL></SECTION></FIGURE></SECTION></ARTICLE></DIV><DIV id="page-footer" class="page-footer"><SECTION><H6>Team</H6><UL><LI><A href="/Team:Groningen/Team">Members</A></LI><LI><A href="/Team:Groningen/Supervisors">Supervisors</A></LI><LI><A href="/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A href="/Team:Groningen/Achievements">Achievements</A></LI><LI><A href="/Team:Groningen/Contact">Contact</A></LI><LI><A href="/Team:Groningen/Jamboree">Jamboree</A></LI></UL></SECTION><SECTION><H6>Project</H6><UL><LI><A href="/Team:Groningen/Tour">Tour</A></LI><LI><A href="/Team:Groningen/Description">Description</A></LI><LI><A href="/Team:Groningen/Design">Design</A></LI><LI><A href="/Team:Groningen/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Groningen/Summary">Summary</A></LI><LI><A href="/Team:Groningen/Applications">Applications</A></LI><LI><A href="/Team:Groningen/Results">Results</A></LI><LI><A href="/Team:Groningen/FuturePerspectives">Future Perspectives</A></LI></UL></SECTION><SECTION class="double"><H6>Biology</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Experiments">Experiments</A></LI><LI><A href="/Team:Groningen/Safety">Safety</A></LI><LI><A href="/Team:Groningen/Decoy">Decoy experiment</A></LI><LI><A href="/Team:Groningen/PhotoswitchableAntibiotics">Photoswitchable antibiotics</A></LI><LI><A href="/Team:Groningen/BrickCharacter">BioBrick Characterization</A></LI><LI><A href="/Team:Groningen/KeyDeletion">Key deletion</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/Proof">Proof of concept</A></LI><LI><A href="/Team:Groningen/Labjournal">Lab journal</A></LI><LI><A href="/Team:Groningen/PlasmidConstruction">Plasmid construction</A></LI><LI><A href="/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="/Team:Groningen/Labjournal/PrimerList">Primer list</A></LI><LI><A href="/Team:Groningen/Notebook">Notebook</A></LI></UL></DIV></SECTION><SECTION><H6>Modelling</H6><UL><LI><A href="/Team:Groningen/Model">Modelling</A></LI><LI><A href="/Team:Groningen/MutationRates">Mutation Rates</A></LI><LI><A href="/Team:Groningen/RandomMutation">Random Mutation</A></LI><LI><A href="/Team:Groningen/AIModel">AI Agent model</A></LI><LI><A href="/Team:Groningen/DecodingFidelity">Decoding Fidelity</A></LI><LI><A href="/Team:Groningen/SpiroModel">Optimal Strain</A></LI></UL></SECTION><SECTION><H6>Computing</H6><UL><LI><A href="/Team:Groningen/Software">Software</A></LI><LI><A href="/Team:Groningen/Encoding">Encoder</A></LI><LI><A href="/Team:Groningen/Decoding">Decoder</A></LI><LI><A href="/Team:Groningen/FullCoding">Full Coding</A></LI></UL></SECTION><SECTION class="double"><H6>Human Practices</H6><DIV class="split"><UL class="left flone"><LI><A href="/Team:Groningen/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Groningen/HP/Outreach">Outreach</A></LI><LI><A href="/Team:Groningen/HP/Social">Social</A></LI><LI><A href="/Team:Groningen/HP/Legal">Legal</A></LI><LI><A href="/Team:Groningen/HP/FutureScenarios">Future Scenarios</A></LI></UL><UL class="right flone"><LI><A href="/Team:Groningen/HP/Education">Education</A></LI><LI><A href="/Team:Groningen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="/Team:Groningen/Engagement">Engagement</A></LI><LI><A href="/Team:Groningen/HP/Gold">Gold</A></LI><LI><A href="/Team:Groningen/HP/Silver">Silver</A></LI></UL></DIV></SECTION><SECTION><H6>Support</H6><UL><LI><A href="/Team:Groningen/Sponsors">Sponsors</A></LI><LI><A href="/Team:Groningen/Attributions">Attributions</A></LI></UL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>