Team:MIT/Repressor Group Experiment 1 Digestion Gels
BACKGROUND
EXPERIMENTS
Synthetic Promoters microRNA Recombinase Proof of Concept Collaborations
FUTURE WORK
Circuit Design Clinical Applications
HUMAN PRACTICES
Parts Toolbox Accessible Protocols Integrated Human Practices Public Outreach Interlab Study
NOTEBOOK
Lab Notebook Parts Mammalian Parts Collection Attributions Safety
AWARDS
HP Silver HP Gold Integrated HP Basic Part
TEAM
Experiment 1 Restriction Digestion for testing · Benchling
Experiment 1 Restriction Digestion for testing
Made with Benchling
Project: miRNA and Repressors subgroup
Authors: Wangui Mbuguiro
Dates: 2016-04-18 to 2016-05-07
Monday, 4/18/16
Goal:
Enzymes that work:
A
B
C
D
E
1
Tal14:meYFP
TRE:TALER14
phef1a:rtTA
2
Tal14
mEYFP
3
sacI
BlpI **BAD ENZYME
StuI
BsrGI
4
StuI
BsrGI
BspMI
5
Xho
ApaI
6
SacI
7
8
Final Choice:
sacI
BlpI
StuI
BspMI
Table1
A
B
1
Tal14:mEYFP 1
no enzyme
2
Tal14:mEYFP 1
BlpI
3
Tal14:mEYFP 1
sacI
4
Tal14:mEYFP 1
BlpI, sacI
5
Tal4:mEYFP 2
BlpI, SacI
6
Tal4:mEYFP 3
BlpI, SacI
7
Tal4:mEYFP 4
BlpI, SacI
8
TRE: TALER14 1
no enzyme
9
TRE: TALER14 2
StuI
10
TRE: TALER14 3
StuI
11
TRE: TALER14 4
StuI
12
TRE: TALER14 1
StuI
13
hef1a: rtTa 1
no enzyme
14
hef1a: rtTa 1
BspMI
15
hef1a: rtTa 2
BspMI
16
hef1a: rtTa 3
BspMI
17
hef1a: rtTa 4
BspMI
Table2
Notes for digest:
A
B
C
1
plasmid
buffer
temperature
2
Tal14:eYFP
cutsmart
37
3
TRE:Tal14
cutsmart
37
4
hef1a:rTTA
buffer 3.1
37
Table3
digestcalculations.jpg
Pouring an Agarose Gel: 10uL of SYBRSafe added to agarose solution after micorwaving and allowing it to cool until touchable
Results:
clipboard_2016-04-30_15:09:09.png
clipboard_2016-04-30_15:09:34.png
weisslab 2016-05-08 17hr 33min (1).jpg
Saturday, 5/7/16
Enzyme choices:
●
Tal14:mEYFP1: AseI, BspmI
●
TRE: TALER14: StuI
●
hef1a:rtTa:apaI
A
B
C
D
1
Lane
Sample
Enzyme
Expected?
2
1
Ladder
--
--
3
2
pDEST control
AseI BspmI
Yes
4
3
pDEST control
AseI
Yes
5
4
pDEST control
BspmI
kinda
6
5
Tal14:mEYFP 1a control
no enzyme
Yes
7
6
Tal14:mEYFP 1a
AseI
Yes
8
7
Tal14:mEYFP 1a
BspmI
Yes
9
8
Tal14:mEYFP 1a
AseI BspmI
Yes
10
9
Tal14:mEYFP 1b
AseI
Yes
11
10
Tal14:mEYFP 1b
BspmI
No
12
11
Tal14:mEYFP 1b
AseI BspmI
Yes
13
12
blank
--
--
14
13
TRE: TALER14 1a control
no enzyme
Yes
15
14
TRE: TALER14 1a
StuI
Yes
16
15
TRE: TALER14 1b
StuI
Yes
17
16
pDEST control
ApaI
Yes
18
17
hef1a: rtTa 1a control
no enzyme
Yes
19
18
hef1a: rtTa 1a
ApaI
Yes
20
19
hef1a: rtTa 1b
ApaI
Yes
Table4
MINIPREPS CHOSEN:
Tal14:mEYFP: 1a
TRE:Tal14: 1a
hef1a:rtTA: 1a
weisslab 2016-05-08 17hr 33min.jpg
Image with both side by side:
clipboard_2016-06-30_11:05:28.png
Notes for digest:
A
B
C
1
plasmid
buffer
temperature
2
Tal14:eYFP
3.1
37
3
TRE:Tal14
cutsmart
37
4
hef1a:rTTA
cutsmart
25
Table5
Diagnostic Restriction Digestion (Maya, Liz, Wangui, Kathryn 4/18/16)
Introduction
A diagnostic restriction digest helps identify correctly assembled clones from incorrect clones.
Materials
200 ul PCR strip tubes, one per reaction
Restriction enzyme (chosen below), 1 ul per reaction.
10X restriction enzyme buffer (chosen below), 1 ul per reaction.
Procedure
Choose A Good Restriction Enzyme
Using Benchling, choose a restriction enzyme that meets the following criteria:
- Cuts at least once in the insert (for a pEXPR, either the promoter or the gene.)
- Cuts at least once in the backbone.
- Gives bands that aren't too large (> 8 kb) or too small (< 200 bp).
- Gives a band pattern that is significantly different from the expected error mode.
* For LR reactions, the most common error is a pDEST that slipped through the selection.
- Begin in the "Brian's Favorites" list, then expand to the main Weiss lab list.
- If you can't find a single enzyme that gives an acceptable band pattern, choose two enzymes that give a acceptable band pattern when used together.
* This double-digest is subject to buffer compatibility, outlined below.
Record your enzyme choice on the plasmid's Description page.
Benchling will tell you the enzyme's buffer compatibility and active temperature. Record the buffer in which the enzyme is most active.
If there are multiple bufers in which the enzyme is equally active, choose in this order: Buffer 3.1, Buffer CS, Buffer 2.1, Buffer 1.1.
Set up the restriction digest
Retrieve the minipreps and the appropriate 10X buffer concentrate from the freezer.  Thaw on the benchtop or in your fingers.
Label the PCR tubes with your initials and an incrementing number.
ie: BT-1, -2, -3, -4
Vortex the minipreps and the 10X buffer concentrate briefly, then pulse down in the microfuge.
For each miniprep, set up a PCR tube containing the following in order:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
Remove the enzyme from the freezer for as little time as possible.
I have specified an "arbitrary" 3 µl volume of miniprep DNA; this should be fine as long as your miniprep concentration is >= 100 ng/ul.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
Incubate at the appropriate temperature for at least 1 hour and more more than 16 hours.
If the enzyme's active temperature is 37°C, use the 37°C plate incubator.
Stop the reaction by adding 2 ul of 6X NEB purple gel loading dye to each reaction.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
PAUSE POINT: The reaction can be stored almost indefinitely at room temperature once it's been stopped.
Proceed to gel electrophoresis.
Pouring an agarose gel
Introduction
Agarose gels are used to separate DNA fragments.  Their most common use is
Materials
TAE buffer, 50-100 ml
There is generally 1X TAE buffer at the iGEM bench in a 1L or 2L bottle.
If not, mix some more up from the carboy 10X stock by the main lab gel station. (Use the 1L graduated cylinder at the gel station.)
UltraPure agarose
Casting tray, casting stand, combs
Procedure
Mix and melt the agarose
Check to see if there is a gel waiting in the fridge.
Determine how much molten agarose you'll need.
A small gel cast takes 50 ml; a large gel cast is 100 ml.
Measure out the appropriate amount of TAE into a glass bottle or flask.
Add 5 µl SYBRSafe (0.5X ; stock solution is 10000X) per 100 ml of the solution and mix well.
Add UltraPure agarose to a final concentration of 1% (mass / volume)
So, if you're making 50 ml gel: 0.5 g.
Swirl the bottle or flask to distribute the agarose.
Heat the solution in the microwave with frequent stirring to dissolve the agarose homogenously. ~30 seconds/100ml solution
Let sit until cool enough to handle.
The agarose MUST cool some -- if it's too hot, it can warp the casting trays.
Don't allow it to get too cold!
PAUSE POINT - You can put the flask in the 55°C water bath almost indefinitely at this point.
Pour the gel
Set up the gel tray in the casting stand.  Make sure the rubber gaskets are flat up against the edges of the casting tray.
Set up the gel combs to form the wells.
Rinse the combs with water and wipe dry.
Note for combs: 15-well combs hold about 6 ul liquid per well, 12-well combs hold about 15 ul per well, 8-well combs hold about 20 ul per well
Taping two 8-well comb wells together results in a well that holds up to 100 ul
Taping three 8-well comb wells together result in a well that holds up to 200 ul
Pour the molten agarose into the casting tray.
If bubbles form around the combs, remove and re-insert.
Wait 30 mintues for the gel to solidify.
Use immediately, or place in a plastic zip-lock baggie with a little 1X TAE and store at 4°C.
Gel electrophoresis
Introduction
Gel electrophoresis separates pieces of DNA by length.
Materials
Agarose gel, 1 lane per sample, plus at least 1 lane for the ladder.
If you just poured the gel, make sure it has had at least 30 minutes to set.
If you have a large gel but only need a few lanes, cut out just a piece of it with a razor blade.  Cut straight!
Samples to run
6X NEB Purple Loading Dye
Molecular weight standard (aka "ladder")
Commonly available ladders are Hyperladder I and NEB 2-Log.
Gel box, lid, leads
Electrophoresis power supply
1X TAE buffer, enough to fill the gel box.
Procedure
Prepare your samples
If your samples are frozen, thaw them completely, flick or vortex to mix, then pulse down in the microfuge.
If your samples are not already in loading dye, mix them 1:6 with 6X NEB Purple Loading Dye
Add 1 µl of loading dye for every 5 µl of sample.
Pro tip: If you don't want to mix loading gel with your entire sample (PCR products, for example), cut off a strip of Parafilm; pipette 2-3 ul of sample onto the parafilm; add 1 ul loading dye; mix by pipetting and load directly.
Set up the gel box
Program the voltage on the power supply.  For the small gel box, use 100V; for the large gel box, use 150V.
Program the time on the power supply.  For a small analytical gel, set the timer for 30 minutes.
For larger gels, start at 45 minutes, and then check regularly!
Attach the leads to the gel box cover.  Make sure that the red (positive) lead is attached to the side of the box farthest away from the wells in the gel.
Remember, the DNA is negatively charged and will move toward the positive terminal. A useful neumonic for remembering this is "Run to Red."
If the gel box has not been used previously that day, empty it and rinse it out with DI water.
The TAE buffer can be re-used, but if it's been sitting out for too long it evaporates and the salt concentration (and conductivity) changes.  TAE is cheap; when in doubt, replace it.
Place the gel in the gel box.
Pour TAE into the gel box until it just barely covers the gel.
The TAE's purpose is to conduct electricity; over-filling the gel box results in a larger conductive path, more current flow, and more heating (which can screw up your run.)
Check the wells to see that they are free of bubbles.  If there are bubbles, blow them out by pipetting 100 µl of TAE from the gel box into the well.
Load your samples
Load your samples. For large combs, load 10 µl; for small combs, load 5 µl.
Work quickly. The samples begin to diffuse in the buffer, leading to smeared bands.
Load the ladder in the last lane.  Load 1/2 the volume of your samples: for large combs, load 5 µl, for small combs, load 2.5 µl.
Place the lid on the gel box. Make sure it is seated on the brass contacts.
Run the gel
Press the Start or Run button on the power supply.
Make sure the power supply doesn't complain about an open circuit.  If it does, re-seat the gel box lid and press Run again.
Double-check that there are bubbles forming on the platinum wires at either end of the gel box.
Double-check that the red (positive) lead is on the side farthest from the wells.  Remember, Run to Red.
Run until the pink band is 2/3 to 3/4 of the way down the gel.  For small analytical gels, this should take 30 minutes.  For larger gels, start at 45 minutes and check regularly!
00:30:00
Image the gel
If the power supply is still running, press the Stop button.
Lift the lid off of the gel box. Lift the lid straight up. If you try to "hinge" it up, the lid will break.
Transfer the gel to the GelDoc.
If it's not running, start QuantityOne from the toolbar.
If necessary, click the top button on the toolbox to select the scanner.
If necessary, reset the camera (as per instructions on the GelDoc computer.)
Press the Epifluorescent Illumination button on the GelDoc.  Check the gel's position, zoom and focus.
You want to be zoomed in so that the gel fills the field of view, and focussed so that the well edges are sharp.
Close the GelDoc door. Click Auto Expose.
The Auto Expose functionality generally over-exposes my gels a bit.  If your gel is over-exposed, remove 1/3 of the exposure time, type it into the Exposure box, then click Manually Expose.
In the File menu, select, Export as JPEG.... Save your gel to the iGEM folder on the desktop.
Open Benchling.  Log in, and copy the gel to the Description page for the plasmid you're building.
If there are multiple plasmids on the gel, save it to each plasmid's Description.
Discard the gel in the biowaste box. Wipe down the gel doc with a little water and a paper towel or Kimwipe.
Annotate the gel
Immediately, before you forget what's where, annotate the gel.
List what is in each lane.
Describe whether the pattern is what you expected or not. (You should have an in silico digestion to compare it to!)
Experiment 1: Restriction Digest and Gel Confirmation BM3R1 and Tal21 · Benchling
Experiment 1: Restriction Digest and Gel Confirmation BM3R1 and Tal21
Made with Benchling
Project: miRNA and Repressors subgroup
Authors: Elizabeth Strand
Dates: 2016-06-22 to 2016-06-23
Wednesday, 6/22
A
B
C
D
E
F
1
construct
enzymes
buffer
DNA concentration ng/ul (mini 1)
DNA concentration ng/ul (mini 2)
name on the PCR strip tube
2
TRE:Tal21
Stu1
CS
412.7
405.2
T:Tal21
3
TRE:BM3R1
Nhe1 + Stu1
CS
212.4
646.6
T:B
4
pBM3R1r:eYFP
Blp1 + Stu1
CS
120.7
112.5
B:E
5
pTALER21:eYFP
Blp1 + Stu1
CS
127.6
550.3
T21:E
Table1
A
B
C
D
E
F
G
H
I
1
Cap #
Bottle Label
*all controls used mini 1
2
1
TRE:Tal21 uncut
3
2
TRE:Tal21 Stu1 mini 1
4
3
TRE:Tal21 Stu 1 mini 2
5
4
TRE:BM3R1 uncut
6
5
TRE:BM3R1 Nhe1
7
6
TRE:BM3R1 Stu1
8
7
TRE:BM3R1 DD mini 1
9
8
TRE:BM3R1 DD mini 2
10
9
BM3R1: eYFP uncut
11
10
BM3R1: eYFP Blp1
12
11
BM3R1: eYFP Stu1
13
12
BM3R1: eYFP DD mini 1
14
13
BM3R1: eYFP DD mini 2
15
14
Tal21:eYFP uncut
16
15
Tal21:eYFP Blp1
17
16
Tal21:eYFP Stu1
18
17
Tal21:eYFP DD mini 1
19
18
Tal21:eYFP DD mini 2
20
Table2
A
B
C
D
E
F
G
H
I
J
K
L
M
N
O
P
Q
R
S
T
1
Well 1
Well 2
Well 3
Well 4
Well 5
Well 6
Well 7
Well 8
Well 9
Well 10
Well 11
Well 12
Well 13
Well 14
Well 15
Well 16
Well 17
Well 18
Well 19
Well 20
2
Ladder
TRE:Tal21 uncut
TRE:Tal21 mini 1
TRE:Tal21 mini 2
TRE:BM3R1 uncut
TRE:BM3R1 Nhe1
TRE:BM3R1 Stu1
TRE:BM3R1 mini 1
TRE:BM3R1 mini 2
DNA Ladder
BM3R1:EYFP uncut
BM3R1:EYFP Blp1
BM3R1:EYFP Stu1
BM3R1:EYFP mini 1
BM3R1:EYFP mini 2
Tal21: EYFP uncut
Tal21: EYFP Blp1
Tal21: EYFP Stu1
Tal21: EYFP mini 1
Tal21: EYFP mini 2
3
Table3
xp1_062216_repressorGroup.jpg
N3200_thumb.gif
@ https://www.neb.com/products/n3200-2-log-dna-ladder-01-100-kb
Thursday, 6/23
A
B
C
D
E
F
1
construct
enzymes
buffer
DNA concentration ng/ml (mini 1)
DNA concentration ng/ml (mini 2)
name on the PCR strip tube
2
pBM3R1r:eYFP
Nco1 + Nhe1
CS
120.7
112.5
T:Tal21
3
pTALER21:eYFP
Nco1 + Stu1
CS
127.6
550.3
T:B
4
pDEST
Blp1
CS
282 (no mini!)
Table4
A
B
1
pBM3R1r:eYFP (uncut)
B:E uncut
2
pBM3R1r:eYFP (Nco1)
B:E Nco1
3
pBM3R1r:eYFP (Nhe1)
B:E Nhe1
4
pBM3R1r:eYFP (Nco1+Nhe1)
B:E DD 1
5
pBM3R1r:eYFP (Nco1+Nhe1) mini2
B:E DD 2
6
pTALER21:eYFP (uncut)
T:E uncut
7
pTALER21:eYFP (Nco1)
T:E Nco1
8
pTALER21:eYFP (Stu1)
T:E Stu1
9
pTALER21:eYFP (Nco1+Stu1)
T:E DD 1
10
pTALER21:eYFP (Nco1+Stu1) mini2
T:E DD 2
11
pDest (uncut)
pDest uncut
12
pDest (Blp1)
pDEST cut
Table5
A
B
C
D
E
F
G
H
I
J
K
L
M
N
O
1
Well 1
Well 2
Well 3
Well 4
Well 5
Well 6
Well 7
Well 8
Well 9
Well 10
Well 11
Well 12
Well 13
Well 14
Well 15
2
Ladder
BM3R1:meYFP uncut
BM3R1:meYFP Nco1
BM3R1:meYFP Nhe1
BM3R1:meYFP mini 1
BM3R1:meYFP mini 2
Ladder
Taler21:meYFP uncut
Taler21:meYFP Nco1
Taler21:meYFP Stu1
Taler21:meYFP mini 1
Taler21:meYFP mini 2
Ladder
pDest uncut
pDest Cut
3
Table6
2ndgel.PNG
Diagnostic Restriction Digestion
Introduction
A diagnostic restriction digest helps identify correctly assembled clones from incorrect clones.
Materials
200 ul PCR strip tubes, one per reaction
Restriction enzyme (chosen below), 1 ul per reaction.
10X restriction enzyme buffer (chosen below), 1 ul per reaction.
Procedure
Choose A Good Restriction Enzyme
Using Benchling, choose a restriction enzyme that meets the following criteria:
- Cuts at least once in the insert (for a pEXPR, either the promoter or the gene.)
- Cuts at least once in the backbone.
- Gives bands that aren't too large (> 8 kb) or too small (< 200 bp).
- Gives a band pattern that is significantly different from the expected error mode.
* For LR reactions, the most common error is a pDEST that slipped through the selection.
- Begin in the "Brian's Favorites" list, then expand to the main Weiss lab list.
- If you can't find a single enzyme that gives an acceptable band pattern, choose two enzymes that give a acceptable band pattern when used together.
* This double-digest is subject to buffer compatibility, outlined below.
Record your enzyme choice on the plasmid's Description page.
Benchling will tell you the enzyme's buffer compatibility and active temperature. Record the buffer in which the enzyme is most active.
If there are multiple bufers in which the enzyme is equally active, choose in this order: Buffer 3.1, Buffer CS, Buffer 2.1, Buffer 1.1.
Set up the restriction digest
Retrieve the minipreps and the appropriate 10X buffer concentrate from the freezer.  Thaw on the benchtop or in your fingers.
Label the PCR tubes with your initials and an incrementing number.
ie: BT-1, -2, -3, -4
Vortex the minipreps and the 10X buffer concentrate briefly, then pulse down in the microfuge.
For each miniprep, set up a PCR tube containing the following in order:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
Remove the enzyme from the freezer for as little time as possible.
I have specified an "arbitrary" 3 µl volume of miniprep DNA; this should be fine as long as your miniprep concentration is >= 100 ng/ul.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
Incubate at the appropriate temperature for at least 1 hour and more more than 16 hours.
If the enzyme's active temperature is 37°C, use the 37°C plate incubator.
Stop the reaction by adding 2 ul of 6X NEB purple gel loading dye to each reacti on.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
PAUSE POINT: The reaction can be stored almost indefinitely at room temperature once it's been stopped.
Proceed to gel electrophoresis.
Gel electrophoresis
Introduction
Gel electrophoresis separates pieces of DNA by length.
Materials
Agarose gel, 1 lane per sample, plus at least 1 lane for the ladder.
If you just poured the gel, make sure it has had at least 30 minutes to set.
If you have a large gel but only need a few lanes, cut out just a piece of it with a razor blade.  Cut straight!
Samples to run
6X NEB Purple Loading Dye
Molecular weight standard (aka "ladder")
Commonly available ladders are Hyperladder I and NEB 2-Log.
Gel box, lid, leads
Electrophoresis power supply
1X TAE buffer, enough to fill the gel box.
Procedure
Prepare your samples
If your samples are frozen, thaw them completely, flick or vortex to mix, then pulse down in the microfuge.
If your samples are not already in loading dye, mix them 1:6 with 6X NEB Purple Loading Dye
Add 1 µl of loading dye for every 5 µl of sample.
Pro tip: If you don't want to mix loading gel with your entire sample (PCR products, for example), cut off a strip of Parafilm; pipette 2-3 ul of sample onto the parafilm; add 1 ul loading dye; mix by pipetting and load directly.
Set up the gel box
Program the voltage on the power supply.  For the small gel box, use 100V; for the large gel box, use 150V.
Program the time on the power supply.  For a small analytical gel, set the timer for 30 minutes.
For larger gels, start at 45 minutes, and then check regularly!
Attach the leads to the gel box cover.  Make sure that the red (positive) lead is attached to the side of the box farthest away from the wells in the gel.
Remember, the DNA is negatively charged and will move toward the positive terminal. A useful neumonic for remembering this is "Run to Red."
If the gel box has not been used previously that day, empty it and rinse it out with DI water.
The TAE buffer can be re-used, but if it's been sitting out for too long it evaporates and the salt concentration (and conductivity) changes.  TAE is cheap; when in doubt, replace it.
Place the gel in the gel box.
Pour TAE into the gel box until it just barely covers the gel.
The TAE's purpose is to conduct electricity; over-filling the gel box results in a larger conductive path, more current flow, and more heating (which can screw up your run.)
Check the wells to see that they are free of bubbles.  If there are bubbles, blow them out by pipetting 100 µl of TAE from the gel box into the well.
Load your samples
Load your samples. For large combs, load 10 µl; for small combs, load 5 µl.
Work quickly. The samples begin to diffuse in the buffer, leading to smeared bands.
Load the ladder in the last lane.  Load 1/2 the volume of your samples: for large combs, load 5 µl, for small combs, load 2.5 µl.
Place the lid on the gel box. Make sure it is seated on the brass contacts.
Run the gel
Press the Start or Run button on the power supply.
Make sure the power supply doesn't complain about an open circuit.  If it does, re-seat the gel box lid and press Run again.
Double-check that there are bubbles forming on the platinum wires at either end of the gel box.
Double-check that the red (positive) lead is on the side farthest from the wells.  Remember, Run to Red.
Run until the pink band is 2/3 to 3/4 of the way down the gel.  For small analytical gels, this should take 30 minutes.  For larger gels, start at 45 minutes and check regularly!
Image the gel
If the power supply is still running, press the Stop button.
Lift the lid off of the gel box. Lift the lid straight up. If you try to "hinge" it up, the lid will break.
Transfer the gel to the GelDoc.
If it's not running, start QuantityOne from the toolbar.
If necessary, click the top button on the toolbox to select the scanner.
If necessary, reset the camera (as per instructions on the GelDoc computer.)
Press the Epifluorescent Il lumination button on the GelDoc.  Check the gel's position, zoom and focus.
You want to be zoomed in so that the gel fills the field of view, and focussed so that the well edges are sharp.
Close the GelDoc door. Click Auto Expose.
The Auto Expose functionality generally over-exposes my gels a bit.  If your gel is over-exposed, remove 1/3 of the exposure time, type it into the Exposure box, then click Manually Expose.
In the File menu, select, Export as JPEG.... Save your gel to the iGEM folder on the desktop.
Open Benchling.  Log in, and copy the gel to the Description page for the plasmid you're building.
If there are multiple plasmids on the gel, save it to each plasmid's Description.
Discard the gel in the biowaste box. Wipe down the gel doc with a little water and a paper towel or Kimwipe.
Annotate the gel
Immediately, before you forget what's where, annotate the gel.
List what is in each lane.
Describe whether the pattern is what you expected or not. (You should have an in silico digestion to compare it to!)
RD and GC Tal21:eyfp/Tal14:eyfp · Benchling
RD and GC Tal21:eyfp/Tal14:eyfp
Made with Benchling
Project: miRNA and Repressors subgroup
Authors: Maya Kaul
Date: 2016-07-25
Monday, 7/25
Lanes:
nothing  nothing  uncut  pstI  EcoRI  Ladder  pDEST  A  B  Nothing...
Screen Shot 2016-07-25 at 7.21.01 PM.png
Screen Shot 2016-07-25 at 7.21.32 PM.png
Gel electrophoresis
Introduction
Gel electrophoresis separates pieces of DNA by length.
Materials
Agarose gel, 1 lane per sample, plus at least 1 lane for the ladder.
If you just poured the gel, make sure it has had at least 30 minutes to set.
If you have a large gel but only need a few lanes, cut out just a piece of it with a razor blade.  Cut straight!
Samples to run
6X NEB Purple Loading Dye
Molecular weight standard (aka "ladder")
Commonly available ladders are Hyperladder I and NEB 2-Log.
Gel box, lid, leads
Electrophoresis power supply
1X TAE buffer, enough to fill the gel box.
Procedure
Prepare your samples
If your samples are frozen, thaw them completely, flick or vortex to mix, then pulse down in the microfuge.
If your samples are not already in loading dye, mix them 1:6 with 6X NEB Purple Loading Dye
Add 1 µl of loading dye for every 5 µl of sample.
Pro tip: If you don't want to mix loading gel with your entire sample (PCR products, for example), cut off a strip of Parafilm; pipette 2-3 ul of sample onto the parafilm; add 1 ul loading dye; mix by pipetting and load directly.
Set up the gel box
Program the voltage on the power supply.  For the small gel box, use 100V; for the large gel box, use 150V.
Program the time on the power supply.  For a small analytical gel, set the timer for 30 minutes.
For larger gels, start at 45 minutes, and then check regularly!
Attach the leads to the gel box cover.  Make sure that the red (positive) lead is attached to the side of the box farthest away from the wells in the gel.
Remember, the DNA is negatively charged and will move toward the positive terminal. A useful neumonic for remembering this is "Run to Red."
If the gel box has not been used previously that day, empty it and rinse it out with DI water.
The TAE buffer can be re-used, but if it's been sitting out for too long it evaporates and the salt concentration (and conductivity) changes.  TAE is cheap; when in doubt, replace it.
Place the gel in the gel box.
Pour TAE into the gel box until it just barely covers the gel.
The TAE's purpose is to conduct electricity; over-filling the gel box results in a larger conductive path, more current flow, and more heating (which can screw up your run.)
Check the wells to see that they are free of bubbles.  If there are bubbles, blow them out by pipetting 100 µl of TAE from the gel box into the well.
Load your samples
Load your samples. For large combs, load 10 µl; for small combs, load 5 µl.
Work quickly. The samples begin to diffuse in the buffer, leading to smeared bands.
Load the ladder in the last lane.  Load 1/2 the volume of your samples: for large combs, load 5 µl, for small combs, load 2.5 µl.
Place the lid on the gel box. Make sure it is seated on the brass contacts.
Run the gel
Press the Start or Run button on the power supply.
Make sure the power supply doesn't complain about an open circuit.  If it does, re-seat the gel box lid and press Run again.
Double-check that there are bubbles forming on the platinum wires at either end of the gel box.
Double-check that the red (positive) lead is on the side farthest from the wells.  Remember, Run to Red.
Run until the pink band is 2/3 to 3/4 of the way down the gel.  For small analytical gels, this should take 30 minutes.  For larger gels, start at 45 minutes and check regularly!
Image the gel
If the power supply is still running, press the Stop button.
Lift the lid off of the gel box. Lift the lid straight up. If you try to "hinge" it up, the lid will break.
Transfer the gel to the GelDoc.
If it's not running, start QuantityOne from the toolbar.
If necessary, click the top button on the toolbox to select the scanner.
If necessary, reset the camera (as per instructions on the GelDoc computer.)
Press the Epifluorescent Illumination button on the GelDoc.  Check the gel's position, zoom and focus.
You want to be zoomed in so that the gel fills the field of view, and focussed so that the well edges are sharp.
Close the GelDoc door. Click Auto Expose.
The Auto Expose functionality generally over-exposes my gels a bit.  If your gel is over-exposed, remove 1/3 of the exposure time, type it into the Exposure box, then click Manually Expose.
In the File menu, select, Export as JPEG.... Save your gel to the iGEM folder on the desktop.
Open Benchling.  Log in, and copy the gel to the Description page for the plasmid you're building.
If there are multiple plasmids on the gel, save it to each plasmid's Description.
Discard the gel in the biowaste box. Wipe down the gel doc with a little water and a paper towel or Kimwipe.
Annotate the gel
Immediately, before you forget what's where, annotate the gel.
List what is in each lane.
Describe whether the pattern is what you expected or not. (You should have an in silico digestion to compare it to!)
Diagnostic Restriction Digestion
Introduction
A diagnostic restriction digest helps identify correctly assembled clones from incorrect clones.
Materials
200 ul PCR strip tubes, one per reaction
Restriction enzyme (chosen below), 1 ul per reaction.
10X restriction enzyme buffer (chosen below), 1 ul per reaction.
Procedure
Choose A Good Restriction Enzyme
Using Benchling, choose a restriction enzyme that meets the following criteria:
- Cuts at least once in the insert (for a pEXPR, either the promoter or the gene.)
- Cuts at least once in the backbone.
- Gives bands that aren't too large (> 8 kb) or too small (< 200 bp).
- Gives a band pattern that is significantly different from the expected error mode.
* For LR reactions, the most common error is a pDEST that slipped through the selection.
- Begin in the "Brian's Favorites" list, then expand to the main Weiss lab list.
- If you can't find a single enzyme that gives an acceptable band pattern, choose two enzymes that give a acceptable band pattern when used together.
* This double-digest is subject to buffer compatibility, outlined below.
Record your enzyme choice on the plasmid's Description page.
Benchling will tell you the enzyme's buffer compatibility and active temperature. Record the buffer in which the enzyme is most active.
If there are multiple bufers in which the enzyme is equally active, choose in this order: Buffer 3.1, Buffer CS, Buffer 2.1, Buffer 1.1.
Set up the restriction digest
Retrieve the minipreps and the appropriate 10X buffer concentrate from the freezer.  Thaw on the benchtop or in your fingers.
Label the PCR tubes with your initials and an incrementing number.
ie: BT-1, -2, -3, -4
Vortex the minipreps and the 10X buffer concentrate briefly, then pulse down in the microfuge.
For each miniprep, set up a PCR tube containing the following in order:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
Remove the enzyme from the freezer for as little time as possible.
I have specified an "arbitrary" 3 µl volume of miniprep DNA; this should be fine as long as your miniprep concentration is >= 100 ng/ul.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
Incubate at the appropriate temperature for at least 1 hour and more more than 16 hours.
If the enzyme's active temperature is 37°C, use the 37°C plate incubator.
Stop the reaction by adding 2 ul of 6X NEB purple gel loading dye to each reacti on.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
PAUSE POINT: The reaction can be stored almost indefinitely at room temperature once it's been stopped.
Proceed to gel electrophoresis.
Restriction Digestion and Gel Confirmation of TRE:BM3R1, TRE:Tal21, hef1a:Gal4VP16, Tal21:meYFP, BM3R1:meYFP · Benchling
Restriction Digestion and Gel Confirmation of TRE:BM3R1, TRE:Tal21, hef1a:Gal4VP16, Tal21:meYFP, BM3R1:meYFP
Made with Benchling
Project: miRNA and Repressors subgroup
Authors: Maya Kaul
Dates: 2016-07-05 to 2016-07-07
Tuesday, 7/5
n mTRE:BM3R1, TRE:Tal21, hef1a:Gal4VP16, Tal21:meYFP, BM3R1:meYFP
A
B
C
D
E
F
G
1
plasmid
TRE:BM3R1
TRE:Tal21
hef1a:Gal4VP16
Tal21:meYFP
BM3R1:meYFP
pDEST_mCherry
2
enzymes
Nhe1, Stu1
Stu1
BgIII, Nco1
Nco1, Stu1
Nco1, Nhe1
Nco1, Xho1
3
label
T:B
T:T21
H:G
T:E
B:E
D:M
4
5
Table1
-100ng of DNA in each 10ul digest reaction
Calculations:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
A
B
C
D
E
F
G
H
I
J
K
L
M
N
O
P
Q
R
S
T
U
V
W
X
Y
Z
AA
AB
1
LABEL
1
2
3
4
5
6
7
LABEL
8
9
10
11
12
LABEL
13
14
15
16
17
18
LABEL
19
20
21
22
23
24
2
TEMPLATE
TRE:BM3R1 - 1
TRE:BM3R1 -1
TRE:BM3R1 -1
TRE:BM3R1 -1
TRE:BM3R1 -3
TRE:BM3R1 -2
TRE:BM3R1 -4
TEMPLATE
TRE:Tal21 -1
TRE:Tal21 -1
TRE:Tal21 -2
TRE:Tal21 -3
TRE:Tal21 -4
TEMPLATE
hef1a: Gal4 -1
hef1a: Gal4 -1
hef1a: Gal4 -1
hef1a: Gal4 -1
hef1a: Gal4 -2
hef1a: Gal4 -3
TEMPLATE
Tal21:EYFP -1
Tal21:EYFP -1
Tal21:EYFP -1
Tal21:EYFP -1
Tal21:EYFP -2
Tal21:EYFP -3
3
H2O
6 ul
5 ul
5 ul
4 ul
4 ul
4 ul
4 ul
H2O
6 ul
5 ul
5 ul
5 ul
5 ul
H2O
3 ul
5 ul
5 ul
4 ul
4 ul
4 ul
H2O
6 ul
5 ul
5 ul
4 ul
4 ul
4 ul
4
BUFFER
1 ul
1 ul
1 ul
1 ul
1 ul
1 ul
1 ul
BUFFER
1 ul
1 ul
1 ul
1 ul
1 ul
BUFFER
1 ul
1 ul
1 ul
1 ul
1 ul
1 ul
BUFFER
1 ul
1 ul
1 ul
1 ul
1 ul
1 ul
5
DNA (100 ng/uL)
3 ul
3 ul
3 ul
3 ul
3 ul
3 ul
3 ul
DNA (100 ng/uL)
3 ul
3 ul
3 ul
3 ul
3 ul
DNA (100 ng/uL)
6 ul
6 ul
6 ul
6 ul
6 ul
6 ul
DNA (100 ng/uL)
3 ul
3 ul
3 ul
3 ul
3 ul
3 ul
6
ENZYME 1
0
1 ul nheI
0
1 ul nheI
1 ul nheI
1 ul nheI
1 ul nheI
ENZYME 1
0
1 ul stuI
1 ul stuI
1 ul stuI
1 ul stuI
ENZYME 1
0
1 ul bglII
0
1 ul bglII
1 ul bglII
1 ul bglII
ENZYME 1
0
1 ul NcoI
0
1 ul ncoI
1 ul ncoI
1 ul ncoI
7
ENZYME 2
0
0
1ul stuI
1 ul stuI
1 ul stuI
1 ul stuI
1 ul stuI
ENZYME 2
0
0
0
0
0
ENZYME 2
0
0
1ul ncoI
1 ul ncoI
1 ul ncoI
1 ul ncoI
ENZYME 2
0
0
1 ul stuI
1 ul stuI
1 ul stuI
1 ul stuI
Table2
A
B
C
D
E
F
G
H
I
J
K
L
M
N
O
P
Q
R
S
T
U
1
LABEL
25
26
27
28
29
30
LABEL
31
32
33
34
35
36
2
TEMPLATE
BM3R1:EYFP -1
BM3R1:EYFP -1
BM3R1:EYFP -1
BM3R1:EYFP -1
BM3R1:EYFP -2
BM3R1:EYFP -3
TEMPLATE
pDEST_mCherry -1
pDEST_mCherry -1
pDEST_mCherry -1
pDEST_mCherry -1
pDEST_mCherry -2
pDEST_mCherry -3
3
H2O
3 ul
2 ul
2 ul
1 ul
1 ul
1 ul
H2O
6 ul
5 ul
5 ul
4 ul
1 ul
4 ul
4
BUFFER
1 ul
1 ul
1 ul
1 ul
1 ul
1 ul
BUFFER
1 ul
1 ul
1 ul
1 ul
1 ul
1 ul
5
DNA (100 ng/uL)
6 ul
6 ul
6 ul
6 ul
6 ul
6 ul
DNA (100 ng/uL)
3 uL
3 uL
3 uL
3 uL
6 uL
3 uL
6
ENZYME 1
0
1 ul ncoI
0
1 ul ncoI
1 ul ncoI
1 ul ncoI
ENZYME 1
0
1 ul ncoI
0
1 ul ncoI
1 ul ncoI
1 ul ncoI
7
ENZYME 2
0
0
1 ul nheI
1 ul nheI
1 ul nheI
1 ul nheI
ENZYME 2
0
0
1ul xhoI
1 ul xhoI
1 ul xhoI
1 ul xhoI
Table3
GEL SET UP
A
B
C
D
E
F
G
H
I
J
K
L
M
N
O
P
Q
R
S
T
U
1
LANE
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
2
TUBE
LADDER
1
2
3
4
5
6
7
8
9
10
11
12
LADDER
x
13
14
15
16
17
3
EXPECTED?
-
y
y
y
y
n
y
n
y
y
y
y
n
-
-
y
?
?
n
n
4
LANE
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
5
TUBE
18
19
20
21
22
23
24
LADDER
25
26
27
28
29
30
31
32
33
34
35
36
6
EXPECTED?
n
?
n
n
n
n
n
-
?
n
n
n
n
n
n
n
n
n
n
n
Table4
Thursday, 7/7
Sequencing
A
B
C
1
Tube
DNA Template
Results
2
1
TRE:BM3R1 -2
fine
3
2
TRE:BM3R1 -2
-
4
3
TRE:Tal21 -1
shaky sequence but probably ok
5
4
TRE:Tal21 -1
-
6
5
hef1a:gal4 -1
fine
7
6
hef1a:gal4 -1
-
8
7
Tal21:eyfp -1
fine, **eyfp instead of mEYFP
9
8
Tal21:eyfp -1
-
10
9
BM3R1: EYFP -3
mostly fine, eyfp not efp. can ask about weirdness at att B2
11
10
BM3R1: EYFP -3
-
12
11
pDEST_mCherry -1
mCherry not there --> problem with purification or digestion sites?
13
12
pDEST_mCherry -1
-
Table5
pDEST-mCherry -- Notes from Brian
AsiSI is an enzyme Brian hasn't had experience with --> May be cause?
Could the purification
Do PCR again, scale to 50 uL. 2.5 uL primer.... Try gel purification
Cut for longer.. 2 hours
Likely don't need to purify
Diagnostic Restriction Digestion
Introduction
A diagnostic restriction digest helps identify correctly assembled clones from incorrect clones.
Materials
200 ul PCR strip tubes, one per reaction
Restriction enzyme (chosen below), 1 ul per reaction.
10X restriction enzyme buffer (chosen below), 1 ul per reaction.
Procedure
Choose A Good Restriction Enzyme
Using Benchling, choose a restriction enzyme that meets the following criteria:
- Cuts at least once in the insert (for a pEXPR, either the promoter or the gene.)
- Cuts at least once in the backbone.
- Gives bands that aren't too large (> 8 kb) or too small (< 200 bp).
- Gives a band pattern that is significantly different from the expected error mode.
* For LR reactions, the most common error is a pDEST that slipped through the selection.
- Begin in the "Brian's Favorites" list, then expand to the main Weiss lab list.
- If you can't find a single enzyme that gives an acceptable band pattern, choose two enzymes that give a acceptable band pattern when used together.
* This double-digest is subject to buffer compatibility, outlined below.
Record your enzyme choice on the plasmid's Description page.
Benchling will tell you the enzyme's buffer compatibility and active temperature. Record the buffer in which the enzyme is most active.
If there are multiple bufers in which the enzyme is equally active, choose in this order: Buffer 3.1, Buffer CS, Buffer 2.1, Buffer 1.1.
Set up the restriction digest
Retrieve the minipreps and the appropriate 10X buffer concentrate from the freezer.  Thaw on the benchtop or in your fingers.
Label the PCR tubes with your initials and an incrementing number.
ie: BT-1, -2, -3, -4
Vortex the minipreps and the 10X buffer concentrate briefly, then pulse down in the microfuge.
For each miniprep, set up a PCR tube containing the following in order:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
Remove the enzyme from the freezer for as little time as possible.
I have specified an "arbitrary" 3 µl volume of miniprep DNA; this should be fine as long as your miniprep concentration is >= 100 ng/ul.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
Incubate at the appropriate temperature for at least 1 hour and more more than 16 hours.
If the enzyme's active temperature is 37°C, use the 37°C plate incubator.
Stop the reaction by adding 2 ul of 6X NEB purple gel loading dye to each reacti on.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
PAUSE POINT: The reaction can be stored almost indefinitely at room temperature once it's been stopped.
Proceed to gel electrophoresis.
Gel electrophoresis
Introduction
Gel electrophoresis separates pieces of DNA by length.
Materials
Agarose gel, 1 lane per sample, plus at least 1 lane for the ladder.
If you just poured the gel, make sure it has had at least 30 minutes to set.
If you have a large gel but only need a few lanes, cut out just a piece of it with a razor blade.  Cut straight!
Samples to run
6X NEB Purple Loading Dye
Molecular weight standard (aka "ladder")
Commonly available ladders are Hyperladder I and NEB 2-Log.
Gel box, lid, leads
Electrophoresis power supply
1X TAE buffer, enough to fill the gel box.
Procedure
Prepare your samples
If your samples are frozen, thaw them completely, flick or vortex to mix, then pulse down in the microfuge.
If your samples are not already in loading dye, mix them 1:6 with 6X NEB Purple Loading Dye
Add 1 µl of loading dye for every 5 µl of sample.
Pro tip: If you don't want to mix loading gel with your entire sample (PCR products, for example), cut off a strip of Parafilm; pipette 2-3 ul of sample onto the parafilm; add 1 ul loading dye; mix by pipetting and load directly.
Set up the gel box
Program the voltage on the power supply.  For the small gel box, use 100V; for the large gel box, use 150V.
Program the time on the power supply.  For a small analytical gel, set the timer for 30 minutes.
For larger gels, start at 45 minutes, and then check regularly!
Attach the leads to the gel box cover.  Make sure that the red (positive) lead is attached to the side of the box farthest away from the wells in the gel.
Remember, the DNA is negatively charged and will move toward the positive terminal. A useful neumonic for remembering this is "Run to Red."
If the gel box has not been used previously that day, empty it and rinse it out with DI water.
The TAE buffer can be re-used, but if it's been sitting out for too long it evaporates and the salt concentration (and conductivity) changes.  TAE is cheap; when in doubt, replace it.
Place the gel in the gel box.
Pour TAE into the gel box until it just barely covers the gel.
The TAE's purpose is to conduct electricity; over-filling the gel box results in a larger conductive path, more current flow, and more heating (which can screw up your run.)
Check the wells to see that they are free of bubbles.  If there are bubbles, blow them out by pipetting 100 µl of TAE from the gel box into the well.
Load your samples
Load your samples. For large combs, load 10 µl; for small combs, load 5 µl.
Work quickly. The samples begin to diffuse in the buffer, leading to smeared bands.
Load the ladder in the last lane.  Load 1/2 the volume of your samples: for large combs, load 5 µl, for small combs, load 2.5 µl.
Place the lid on the gel box. Make sure it is seated on the brass contacts.
Run the gel
Press the Start or Run button on the power supply.
Make sure the power supply doesn't complain about an open circuit.  If it does, re-seat the gel box lid and press Run again.
Double-check that there are bubbles forming on the platinum wires at either end of the gel box.
Double-check that the red (positive) lead is on the side farthest from the wells.  Remember, Run to Red.
Run until the pink band is 2/3 to 3/4 of the way down the gel.  For small analytical gels, this should take 30 minutes.  For larger gels, start at 45 minutes and check regularly!
Image the gel
If the power supply is still running, press the Stop button.
Lift the lid off of the gel box. Lift the lid straight up. If you try to "hinge" it up, the lid will break.
Transfer the gel to the GelDoc.
If it's not running, start QuantityOne from the toolbar.
If necessary, click the top button on the toolbox to select the scanner.
If necessary, reset the camera (as per instructions on the GelDoc computer.)
Press the Epifluorescent Illumination button on the GelDoc.  Check the gel's position, zoom and focus.
You want to be zoomed in so that the gel fills the field of view, and focussed so that the well edges are sharp.
Close the GelDoc door. Click Auto Expose.
The Auto Expose functionality generally over-exposes my gels a bit.  If your gel is over-exposed, remove 1/3 of the exposure time, type it into the Exposure box, then click Manually Expose.
In the File menu, select, Export as JPEG.... Save your gel to the iGEM folder on the desktop.
Open Benchling.  Log in, and copy the gel to the Description page for the plasmid you're building.
If there are multiple plasmids on the gel, save it to each plasmid's Description.
Discard the gel in the biowaste box. Wipe down the gel doc with a little water and a paper towel or Kimwipe.
Annotate the gel
Immediately, before you forget what's where, annotate the gel.
List what is in each lane.
Describe whether the pattern is what you expected or not. (You should have an in silico digestion to compare it to!)
LR Reaction
Introduction
An LR reaction inserts one or more parts in pENTR vectors into a pDEST vector.  Used to assemble transcriptional units from promoters and genes.
Materials
Promoter pENTR plasmid: L4-Promoter-R1
Working concentration: 5 fmol/ul
Gene pENTR plasmid: L1-Gene-L2
Working concentration: 5 fmol/ul
Destination plasmid: pDEST
Working concentration: 10 fmol/ul
Nuclease-free TE
200 µl PCR strip tubes, 1 tube per rxn
5x LR Clonase II
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
Proteinase K
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
Procedure
LR Reaction Setup
For each LR you are doing, fill out a column in the following table:
A
B
C
D
E
F
1
Tube Label
2
Promoter pENTR
3
Gene pENTR
4
pDEST
5
6
Table1
For each LR, label a 200 µl strip tube with your initials and tube number.
Into each tube, pipette:
-- 1 µl of the promoter pENTR
-- 1 µl of the gene pENTR
-- 1 µl of the pDEST
Add 1 µl of TE to each tube
Retrieve an aliquot of LR Clonase from the -80.
Bring an razor blade with you, you'll need to cut a tube from the strip tubes.
Pulse the LR clonase tube in the microfuge to collect the clonase at the bottom.
Add 1 µl of the LR clonase to each LR reaction.
Be careful pipetting; LR clonase is viscous.
Cap the tubes.
Flick them several times to mix.
Pulse-spin the tubes in the microfuge to collect the liquid at the bottom.
Incubate at room temperature for at least 12 hours and not more than 24 hours.
A popular strategy is to tape the tubes to the shelves over the bench, with your initials and the date.
16-24 hours later: Proteinase K kill
Retrieve a 5 µl aliquot of proteinase K from the -80 freezer.m n
Thaw in your fingers, then pulse in the microfuge to collect at the bottom of the tube.
Pipette 1 ul into each of the LR reactions.
Flick several times to mix.
Pulse-spin the tubes in the microcentrifuge.
Incubate at 37° for 15 minutes, or room-temperature for an hour.
PAUSE POINT: You can store the reactions in the -20 indefinitely until the transformation.
00:15:00
Proceed to transformation.  Transform 2 µl.
Afterwards, cap the tubes. Write the date on the caps and store in the -20 (in case your transformation failed.)
