Team:MIT/Recombinase Group Notebook Week 5
BACKGROUND
EXPERIMENTS
Synthetic Promoters microRNA Recombinase Proof of Concept Collaborations
FUTURE WORK
Circuit Design Clinical Applications
HUMAN PRACTICES
Parts Toolbox Accessible Protocols Integrated Human Practices Public Outreach Interlab Study
NOTEBOOK
Lab Notebook Parts Mammalian Parts Collection Attributions Safety
AWARDS
HP Silver HP Gold Integrated HP Basic Part
TEAM
TN 07/02/16 Diagnostic digest · Benchling
TN 07/02/16 Diagnostic digest
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Maya Kaul
Date: 2016-07-02
Saturday, 7/2
pENTR SCRE
samples:
1: SCRE 1 uncut
2: SCRE 1 double digest BamHI + NcoI
3: SCRE 2 double digest BamHI + NcoI
4: SCRE 3 double digest BamHI + NcoI
clipboard_2016-07-05_09:56:52.png
07-02-2016.jpg
Diagnostic Restriction Digestion
Introduction
A diagnostic restriction digest helps identify correctly assembled clones from incorrect clones.
Materials
200 ul PCR strip tubes, one per reaction
Restriction enzyme (chosen below), 1 ul per reaction.
10X restriction enzyme buffer (chosen below), 1 ul per reaction.
Procedure
Choose A Good Restriction Enzyme
Using Benchling, choose a restriction enzyme that meets the following criteria:
- Cuts at least once in the insert (for a pEXPR, either the promoter or the gene.)
- Cuts at least once in the backbone.
- Gives bands that aren't too large (> 8 kb) or too small (< 200 bp).
- Gives a band pattern that is significantly different from the expected error mode.
* For LR reactions, the most common error is a pDEST that slipped through the selection.
- Begin in the "Brian's Favorites" list, then expand to the main Weiss lab list.
- If you can't find a single enzyme that gives an acceptable band pattern, choose two enzymes that give a acceptable band pattern when used together.
* This double-digest is subject to buffer compatibility, outlined below.
Record your enzyme choice on the plasmid's Description page.
Benchling will tell you the enzyme's buffer compatibility and active temperature. Record the buffer in which the enzyme is most active.
If there are multiple bufers in which the enzyme is equally active, choose in this order: Buffer 3.1, Buffer CS, Buffer 2.1, Buffer 1.1.
Set up the restriction digest
Retrieve the minipreps and the appropriate 10X buffer concentrate from the freezer.  Thaw on the benchtop or in your fingers.
Label the PCR tubes with your initials and an incrementing number.
ie: BT-1, -2, -3, -4
Vortex the minipreps and the 10X buffer concentrate briefly, then pulse down in the microfuge.
For each miniprep, set up a PCR tube containing the following in order:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
Remove the enzyme from the freezer for as little time as possible.
I have specified an "arbitrary" 3 µl volume of miniprep DNA; this should be fine as long as your miniprep concentration is >= 100 ng/ul.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
Incubate at the appropriate temperature for at least 1.5 hour and more more than 16 hours.
If the enzyme's active temperature is 37°C, use the 37°C plate incubator.
Stop the reaction by adding 2 ul of 6X NEB purple gel loading dye to each reacti on.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
PAUSE POINT: The reaction can be stored almost indefinitely at room temperature once it's been stopped.
Proceed to gel electrophoresis.
JG 7/5/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Maya Kaul
Date: 2016-07-05
Tuesday, 7/5
clipboard_2016-07-05_16:39:58.png
A
B
C
D
1
ng/ul
fmol/ul
2
pENTR SCRE 1
169
68
3
pENTR SCRE 2
4
pENTR FLPO 2
295
114
5
pENTR FLPO 3
6
pENTR hEF1a SloxP SV40 3
164.9
59
7
pENTR hEF1a SloxP SV40 4
8
pENTR hEF1a FRT SV40 1
228
82
9
pENTR hEF1a FRT SV40 4
Table1
LR Reaction
Introduction
An LR reaction inserts one or more parts in pENTR vectors into a pDEST vector.  Used to assemble transcriptional units from promoters and genes.
Materials
Promoter pENTR plasmid: L4-Promoter-R1
Working concentration: 5 fmol/ul
Gene pENTR plasmid: L1-Gene-L2
Working concentration: 5 fmol/ul
Destination plasmid: pDEST
Working concentration: 10 fmol/ul
Nuclease-free TE
200 µl PCR strip tubes, 1 tube per rxn
5x LR Clonase II
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
Proteinase K
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
Procedure
LR Reaction Setup
For each LR you are doing, fill out a column in the following table:
A
B
C
D
E
F
1
Tube Label
1
2
3
4
2
Promoter pENTR
EGSH
EGSH
hef1a-loxP-SV40-loxP (3-3)
hef1a-FRT-SV40-FRT (4-1)
3
Gene pENTR
Cre (1-1)
FLP (2-2)
eYFP
eYFP
4
pDEST
GTW6
GTW6
GTW6
GTW6
5
6
Table1
For each LR, label a 200 µl strip tube with your initials and tube number.
Into each tube, pipette:
-- 1 µl of the promoter pENTR
-- 1 µl of the gene pENTR
-- 1 µl of the pDEST
Add 1 µl of TE to each tube
Retrieve an aliquot of LR Clonase from the -80.
Bring an razor blade with you, you'll need to cut a tube from the strip tubes.
Pulse the LR clonase tube in the microfuge to collect the clonase at the bottom.
Add 1 µl of the LR clonase to each LR reaction.
Be careful pipetting; LR clonase is viscous.
Cap the tubes.
Flick them several times to mix.
Pulse-spin the tubes in the microfuge to collect the liquid at the bottom.
Incubate at room temperature for at least 12 hours and not more than 24 hours.
A popular strategy is to tape the tubes to the shelves over the bench, with your initials and the date.
16-24 hours later: Proteinase K kill
Retrieve a 5 µl aliquot of proteinase K from the -80 freezer.m n
Thaw in your fingers, then pulse in the microfuge to collect at the bottom of the tube.
Pipette 1 ul into each of the LR reactions.
Flick several times to mix.
Pulse-spin the tubes in the microcentrifuge.
Incubate at 37° for 15 minutes, or room-temperature for an hour.
PAUSE POINT: You can store the reactions in the -20 indefinitely until the transformation.
00:15:00
Proceed to transformation.  Transform 2 µl.
Afterwards, cap the tubes. Write the date on the caps and store in the -20 (in case your transformation failed.)
Transformation of E. coli
Introduction
Transformation is the process of inducing chemically competent E. coli to take up DNA.
Materials
Dry bath, set to 42°C
Fill the wells in the dry bath block 1/2 full with DI water.
Ice bucket, with ice
For thawing competent cells.
DNA to transform
Could be an assembly reaction (LR, Golden Gate, etc) or a miniprepped plasmid.
If you removed it from the freezer, make sure it's entirely thawed  out.
pUC19 Transformation Control, 1 pg/µl
The pUC19 control will tell you how efficient your transformations were.
SOC growth media, at room temperature
Check to make sure it's clear and NOT CLOUDY.
Antibiotic plates, one per transformation, plus 1 Amp plate for the pUC19 control
Make sure the plates you use match the resistance cassette of the plasmid!
Competent E. coli, one tube per transformation + one for the pUC19 control
These live in the -80 in 235.
Thaw on ice 3-4 minutes.
A timer, set for 30 seconds.
Procedure
Setup
Make sure the dry bath is set to 42°C and the wells in the block are 1/2 full of DI water
Remove selection plates from the refrigerator.  Double-check that they match the selection marker on your plasmid, then place them in the 37° incubator.
Retrieve the DNA to transform.
If frozen: thaw, completely, flick a few times to mix, then pulse down in the microfuge.
Fill an ice bucket with ice.  Retrieve one tube of competent E. coli per transformation from the -80 and thaw on ice, 3-4 minutes.
00:03:00
While the transformation tubes are thawing, label their tops with something descriptive.  Record the labels here:
A
B
C
D
E
F
G
H
1
EGSH-mKate
2
EGSH-GFP
3
control (puc19)
4
Table1
Transformation
Add 2 µl DNA from each reaction to a tube of competent cells.
Immediately after adding the DNA to each tube, stir the cells a few times with the pipette tip.
Add 1 µl of the pUC19 transformation control to the positive control tube.
Incubate on ice for 30 minutes.
00:30:00
Heat shock the cells for exactly 30 seconds in the 42° heat block. (Yes, set a timer.)
00:00:30
Place back on ice for 2 minutes.
00:02:00
Add 250 µl SOC to each tube.
Tape the tubes to the platform of a shaker at 37°C and shake at 270 RPM for 60 minutes.
01:00:00
Plating
Label the selection plates using the labels you recorded above.
Shake ~10 plating beads onto each plate.
Pipette 100 µl of each transformation onto the corresponding plates. NOTE: if you are using DNA from a golden gate reaction, see the golden gate protocol for instructions to dilute your sample.
-----Golden Gate: Plate only 10 ul of the outgrowth in a 200 ul puddle of water, or you will get a lawn of colonies.
Cover the plates and shake the beads around to spread the cells out.
Dispose of the beads by tapping them into the waste container.
Incubate the plates upside down overnight in the 37° incubator.
Don't incubate for more than 18-24 hours.
Compute transformation efficiency
Count the colonies on your positive transformation plate.
If there are many many colonies, then hooray!  You had a great transformation.  Just estimate.
Divide the number of colonies by the fraction of the transformation you plated.
So, if you resuspended your transformation in a total volume of 300 ul, then plated 100 ul, multiply the number of colonies by 3.
Transformation efficiency is expressed in colonies per microgram pUC19.  Multiply the number of colonies by the appropriate conversion factor.
So if you transformed 1 picogram of pUC19 DNA, multiply by 106.
Record your transformation efficiency in your (daily) lab notebook.
Sequencing protocol
Introduction
Preparing sample to send to Genewiz for sequencing Sequencing Pick-up Times: 9:30am, 3:00 pm, 5:30 pm
Materials
DNA stock
Nuclease-free water
Primer (5nM stock)
One microcentrifuge tube for each DNA plasmid
2 PCR tubes per plasmid (must be in strips of eight, round up if needed!)
Procedure
Diluting the DNA
For orders with <48 samples, use 8-strip PCR tubes. Label your tubes on the side vertically with "MI" and sample number (01, 02, 03, etc.). Use image below as guideline. Each plasmid you want to sequence will require 2 tubes, one for the forward primer and one for the reverse primer. Use "MI" instead of "GW"
Dilute your sequencing primer to 5 µM (pmol/µl) using water. You will need 5 µl for each sequencing reaction. Note, the primers in the tubes from IDT have not been diluted! Use already diluted stock (should be in regular microcentrifuge tubes).
Using the recorded concentration, dilute the DNA to the template concentration in 10 µl as detailed below. For example, a 5 kb plasmid would need a template concentration of ~50 ng/µl. Final volume for your working stock should 30 uL. Please make dilutions in water or Tris.
Add 10 µl of diluted template DNA to each tube.
Add 5 µl of diluted primer to each tube. Be careful about putting the forward and reverse primers in the correct tubes! Remember, for each DNA template, there should be two tubes - one with reverse primer and the other with forward primer
Vortex briefly
Log into genewiz. Username: igem-sequence@mit.edu Password: igem2014citrus
Click "create sequencing order"
Sevice Priority: click on "standard"
Create Order by: click on "online form"
Sample Type: click on "Pre-mixed"
Enter in how many samples you are sending in and click "Create new form"
Enter in descriptive names for your DNA (this is to help you remember when you get the results back), the DNA type, the DNA length (get this from your construct on benchling), the primer name (again, this is more for you). When you are done hit save and next.
It will give you an list of what you have entered and a price for each reaction. Hit "next step"
The payment information field should be autofilled. There has been an issue recently with the PO box number not being entered. If this is the case, click "credit card" (next to "payment info")  and then click "PO" to bring you back to the original form. Now you should be able to enter in a PO number. Open up a previous submitted order (there should be some saved to the desktop called "sequencing" or something like that) and copy over the PO number.
Hit "next step". Once it gives you a submitted order form, print 2 copies and pick them up from the printer (exit lab, through glass doors, on your left)
Put your PCR tubes in a large falcon tube.
Put the falcon tube in a ziploc bag (located in Qiagen column drawer) and put in one order form. Label the bag with something descriptive ("Weiss Lab MIT iGEM Recombinase 6/30/16" for example)
Put the bag in the Genewiz pickup box (located by the elevators, make sure you put in the genewiz box)
Open up the weiss lab orders spreadsheet (bookmarked on the computer). You need to request access to the spreadsheet if it is your first time ordering. Once you request access, email Brian so he knows that you have sent in a request.
Click the tab at the bottom called "genewiz".
Fill out the form. Make sure you indicate that the order is from iGEM so the right account can be charged.
You did it! YAY! Genewiz should get back to you pretty quickly, generally between 12-24 hrs.
TN 07/06/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Elizabeth Strand
Date: 2016-07-06
Wednesday, 7/6
pENTR SCRE: M13F(-21), M13R/T7
pENTR FLPO: M13F(-21), M13R/T7
pENTR hEF1a SloxP SV40: M13F(-21), M13R/T7
pENTR hEF1a FRT SV40: M13F(-21), M13R/T7
Other potential reverse primers: SV40pA-R/pFastBacR
JGPrimer3 + PrimerQ3mEYFP-R -5.99 hetero
PrimerQ4mEYFP-F + JGQ3mEYFP-R -5.19
Primer _ Q4 _ mEYFP _ F + DBPrimer - 4.41
JGPrimer3 + DBPrimer -3.55
A
B
C
D
E
F
G
H
1
Primer Name
Tm
Annealing T
Secondary Tm
Self-dimer
Hetero-dimer
2
PrimerQ4mEYFP-F
58
61
46
-6
-8
3
PrimerQ3mEYFP-R
61
61
45
-7
-6
4
Primer _ Q4 _ mEYFP _ F
59
58
34.2
-6.34
-7.07
5
Primer _ Q4 _ mEYFP _ R
55
58
39.3
-5.99
-7.07
6
Table1
Overnight liquid cultures (picking colonies)
Introduction
Overnight cultures are used to prepare miniprep DNA.
Materials
The plate from which you are picking colonies
15 ml round-bottom polystyrene tubes, one per culture
The ones with the snap caps, NOT conical tubes with screw caps
5 mL LB per culture
A container that can hold 5ml x the number of cultures
For a modest number of minipreps, a 50 ml conical tube works well.
For larger minipreps, use a sterile bottle (100 ml is frequently useful.)
Antibiotic stock, 1000X
Procedure
Materials Setup
Warm up the LB to at least room temperature (if it came from the fridge), but not warmer than 37°C
Label one round-bottom culture tube for each miniprep.  Use "NAME-1, NAME-2, ..."etc for the naming convention, where NAME is a shortened name of the plasmid (eg, "hEF1a:mKate").
Your impulse is to just use number, or initials and number, but trust me -- you will want to be able to identify this tube in three weeks when you've forgotten what you were doing.
Using a sterile pipette, transfer 5 ml of LB to the mixing container for each culture PLUS 5 ML.
Add antibiotic stock to a final concentration of 1X (1 µl stock for each 1 ml in the mixing container.)
Cap tightly and mix well.
Culture Setup
Using a sterile pipette, transfer 5 ml of LB+antibiotic to each round-bottom culture tube.
If you are making cultures with different antibiotics, take care that the right media goes in each tube.
Squirt ethanol on a pair of foreceps and wipe dry with a Kimwipe.
Use the foreceps to pick up a sterile 200µl pipette tip, scrape a colony off of the plate, and drop the pipette tip in the corresponding tube.
Repeat for each tube.
Transfer to an incubating shaker at 37°C and incubate 14-16 hours.
Don't over-grow too badly, or your yield will suffer.
If you need to grow longer, you can grow at 30°C instead for 20 hours.
Transformation of E. coli
Introduction
Transformation is the process of inducing chemically competent E. coli to take up DNA.
Materials
Dry bath, set to 42°C
Fill the wells in the dry bath block 1/2 full with DI water.
Ice bucket, with ice
For thawing competent cells.
DNA to transform
Could be an assembly reaction (LR, Golden Gate, etc) or a miniprepped plasmid.
If you removed it from the freezer, make sure it's entirely thawed  out.
pUC19 Transformation Control, 1 pg/µl
The pUC19 control will tell you how efficient your transformations were.
SOC growth media, at room temperature
Check to make sure it's clear and NOT CLOUDY.
Antibiotic plates, one per transformation, plus 1 Amp plate for the pUC19 control
Make sure the plates you use match the resistance cassette of the plasmid!
Competent E. coli, one tube per transformation + one for the pUC19 control
These live in the -80 in 235.
Thaw on ice 3-4 minutes.
A timer, set for 30 seconds.
Procedure
Setup
Make sure the dry bath is set to 42°C and the wells in the block are 1/2 full of DI water
Remove selection plates from the refrigerator.  Double-check that they match the selection marker on your plasmid, then place them in the 37° incubator.
Retrieve the DNA to transform.
If frozen: thaw, completely, flick a few times to mix, then pulse down in the microfuge.
Fill an ice bucket with ice.  Retrieve one tube of competent E. coli per transformation from the -80 and thaw on ice, 3-4 minutes.
00:03:00
While the transformation tubes are thawing, label their tops with something descriptive.  Record the labels here:
A
B
C
D
E
F
G
H
1
SCre
2
FLPO
3
hef1a_loxP_SV40_loxP
4
hef1a_FRT_SV40_loxP
5
puc-19TC
Table1
Transformation
Add 2 µl DNA from each reaction to a tube of competent cells.
Immediately after adding the DNA to each tube, stir the cells a few times with the pipette tip.
Add 1 µl of the pUC19 transformation control to the positive control tube.
Incubate on ice for 30 minutes.
00:30:00
Heat shock the cells for exactly 30 seconds in the 42° heat block. (Yes, set a timer.)
00:00:30
Place back on ice for 2 minutes.
00:02:00
Add 250 µl SOC to each tube.
Tape the tubes to the platform of a shaker at 37°C and shake at 270 RPM for 60 minutes.
01:00:00
Plating
Label the selection plates using the labels you recorded above.
Shake ~10 plating beads onto each plate.
Pipette 100 µl of each transformation onto the corresponding plates. NOTE: if you are using DNA from a golden gate reaction, see the golden gate protocol for instructions to dilute your sample.
-----Golden Gate: Plate only 10 ul of the outgrowth in a 200 ul puddle of water, or you will get a lawn of colonies.
Cover the plates and shake the beads around to spread the cells out.
Dispose of the beads by tapping them into the waste container.
Incubate the plates upside down overnight in the 37° incubator.
Don't incubate for more than 18-24 hours.
Compute transformation efficiency
Count the colonies on your positive transformation plate.
If there are many many colonies, then hooray!  You had a great transformation.  Just estimate.
Divide the number of colonies by the fraction of the transformation you plated.
So, if you resuspended your transformation in a total volume of 300 ul, then plated 100 ul, multiply the number of colonies by 3.
Transformation efficiency is expressed in colonies per microgram pUC19.  Multiply the number of colonies by the appropriate conversion factor.
So if you transformed 1 picogram of pUC19 DNA, multiply by 106.
Record your transformation efficiency in your (daily) lab notebook.
CF 7/7/16 Meeting with Deepak
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Julia Goupil
Date: 2016-07-07
Thursday, 7/7
●
spatial/temporal control is what we want
○
phosphorylation activated recombinase not possible
●
Small molecule that ends up dimerizing receptor (tango system, TEF cleavage site)
●
jak-stat - upon phosphorylation, they dimerize, lead to signal cascade. Uses cytokines as small molecule control
●
recombinases are large. Hard to fuse to something
●
GPCRs - respond to small molecules
○
form agregation of alpha and beta subunits
○
testing this sucks and will take two months
●
delta notch is cool, but uses proteins instead of small molecules
●
small molecule binders of notch do exist (a few of them)
●
can you get small molecule control w/ aptazyme at RNA level
○
deals with ribozymes, can cut off poly A tail to reduce basal expression
●
RNA control may be our best bet (lab already does some)
●
have RNA control with destabilization domain that can be stabilized with small molecule
●
aptamer control
●
is there a small molecule that can bind the recombination site and prevent the recombinase from binding?
●
make another Tal that sat on one of recombinase sites (DNA), under small molecule control
●
reduce basal expression of recombinase  is the ultimate goal NOT just nuclear localization
●
easiest way to reduce basal expression is sequential transfection
●
do all you can on the DNA level
●
2015 review weiss lab - first author is olivia andres
○
go over ways to regulate RNA (dd domains, RNA binding proteins, aptazymes, small molecules)
●
Nothing comes to mind at the protein level
○
this is why RNA control may be our best bet
●
have to create a delay of expression
○
another level of cascade helps a little
■
RNA regulation helps much more
●
small molecule gal4 vp16 - reduce amount of recombinase made until repression systems are at full force
●
RNA regulation!!!
●
Let's chat monday afternoon
●
UV is tricky in mammalian cells
○
need photo-cleavable linker
○
cleavage doesn't really work
●
2x change in nuclear localization in nature paper about UV light as control
○
really an enrichment vs. actually a photo-inducible NLS
●
Deepak is friends w/author of nature paper, can ask her what the latest is in the field (she's head of Heidelberg iGEM team)
●
Doesn't seem like this is the best route to go
○
NLS is an imperfect system
●
Protein level - options are much more limited because we're using recombinases
●
we should open it up to all regulation levels
●
DOX inducible isn't good enough
●
RNA REGULATION!!!!!!
●
send him an email wehn we figure out a time for Monday
JG 7/7/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Trinh Nguyen
Date: 2016-07-07
Thursday, 7/7
Nanodrop Results (in ng/uL):
A
B
C
D
E
1
1
2
3
4
2
1 (EGSH: mKate)
146.2
321
107.7
162.8
3
2 (EGSH: GFP)
209.8
188.4
175
188
Table1
clipboard_2016-07-07_17:31:23.png
recombinase EGSH mKate.jpg
Restriction digest for EGSH mKate
ladder
2-5 uncut (1,2,3,4)
ladder
7-11 double digest StuI + PvuII (1,2,3,4)
ladder
13-14 single digest StuI (1)
15-16 single digest PvuII (1)
recombinase EGSH mKate (2).jpg
Miniprep
Introduction
The miniprep uses silica gel to isolate plasmid DNA from an E. coli culture
Materials
Buffer P1 (resuspension buffer)
Retrieve from refrigerator.  If you are opening a new miniprep kit, add the RNAse and LyseBlue reagent and check the box on the cap.
Buffer P2 (lysis buffer)
Open the cap and look at the lysis buffer.  Swirl it around.  If it appears cloudy, the SDS has fallen out of solution; warm it for a few minutes in the 55°C water bath.
Buffer N3 (neutralization buffer)
Buffer PB (binding buffer)
Buffer PE (rinse buffer)
Make sure the "Ethanol added?" box has been checked.  If you are opening a new miniprep kit, add absolute ethanol as per the kit instructions and check the box on the cap.
Buffer EB (elution buffer)
Miniprep waste container
Miniprep buffers contain salts that can't go down the sink.
Per miniprep: two microcentrifuge tubes and one blue spin column, with collection tube.
Procedure
Harvest and resuspension
For each culture, label two microcentrifuge tubes on the cap and one blue spin column on the side.
The spin columns should be in their (cap-less) collection vials.
Pipette 1.6 mL of each culture into the corresponding microcentrifuge tubes.
(This is for a 1.7 ml microcentrifuge tube; set your pipettor to 800 ul and do two transfers.  If we ever go back to 2-ml tubes, do 1.9 ml (2x950 ul))
Centrifuge at maximum speed (10,000 or 13,000xg) for three minutes.
Aspirate the supernatant, or pour it off into the bleach bucket.
Pipette ANOTHER 1.8 ml of each culture into the corresponding microcentrifuge tubes.
Centrifuge at maximum speed for three minutes.
While the centrifuge is running, move the remaining cultures to 4degC.
Aspirate the supernatant off with the bench aspirator.  Be careful not to disturb the pelleted E. coli.
We use an aspirator here because the less extra salt and protein we put in the miniprep, the better the yield is.
Add 250 µl Buffer P1 to each tube.
Resuspend the E. coli pellet.  The preferred way is with the roto-mixer at the other end of the lab.
Alternately, if you have just a few tubes, you can resuspend on a vortex.
Make sure to resuspend fully and thoroughly. The resulting suspension should be smooth and cloudy; if there is particulate matter floating around, vortex some more.
Lysis
Add 250 µl Buffer P2 to each tube.
Work quickly; the lysis step should take less than 5 minutes.
Snap the tubes closed and invert them 4-6 times, until the tube is thoroughly mixed and the entire solution turns blue.
If you have many many tubes, you can stack a second tube rack on top of them and invert the entire thing.
Add 350 ul Buffer N3 to each tube.
Snap the tubes closed and invert 4-6 times, until the solution is thouroughly mixed and no longer blue.
The solution will become cloudy or flocculent.
Centrifuge on high speed for 10 minutes.
Separation
Remove the tubes from the microcentrifuge, being careful not to disturb the white pellet.
Using P-1000 micropipettor set to 850 ul, carefully transfer the supernatant from each centrifuge tube to the corresponding blue spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Don't forget to put the lid on the rotor! Some of the salts get aerosolized because the spin columns don't have caps.
Pour the flow-through from each column into the miniprep waste container.
Pipette 500 ul of Buffer PB onto each spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Pipette 750 ul of Buffer PE onto each spin column
Wait 1-3 minutes.
This allows some of the salt that's still bound to the silica matrix to resuspend in the buffer.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Return each spin column to its collection tube and centrifuge an additional 1 minute at high speed.
This removes every last trace of buffer PE; the ethanol can screw up downstream steps.
Transfer each spin column to a clean labelled microcentrifuge tube.
Pipette 50 ul of Buffer EB onto the center of each column.
The volume of EB is comparable to the volume of silica gel matrix; if you pipette down the side, you might not get the entire transfer to the matrix.
Wait 1-3 minutes.
This gives the DNA a chance to dissociate from the silica matrix.
Centrifuge the spin columns, in their collection tubes, for one minute at maximum speed.
Proceed directly to analyze the samples on the Nanodrop.
Diagnostic Restriction Digestion
Introduction
A diagnostic restriction digest helps identify correctly assembled clones from incorrect clones.
Materials
200 ul PCR strip tubes, one per reaction
Restriction enzyme (chosen below), 1 ul per reaction.
10X restriction enzyme buffer (chosen below), 1 ul per reaction.
Procedure
Choose A Good Restriction Enzyme
Using Benchling, choose a restriction enzyme that meets the following criteria:
- Cuts at least once in the insert (for a pEXPR, either the promoter or the gene.)
- Cuts at least once in the backbone.
- Gives bands that aren't too large (> 8 kb) or too small (< 200 bp).
- Gives a band pattern that is significantly different from the expected error mode.
* For LR reactions, the most common error is a pDEST that slipped through the selection.
- Begin in the "Brian's Favorites" list, then expand to the main Weiss lab list.
- If you can't find a single enzyme that gives an acceptable band pattern, choose two enzymes that give a acceptable band pattern when used together.
* This double-digest is subject to buffer compatibility, outlined below.
Record your enzyme choice on the plasmid's Description page.
Benchling will tell you the enzyme's buffer compatibility and active temperature. Record the buffer in which the enzyme is most active.
If there are multiple bufers in which the enzyme is equally active, choose in this order: Buffer 3.1, Buffer CS, Buffer 2.1, Buffer 1.1.
Set up the restriction digest
Retrieve the minipreps and the appropriate 10X buffer concentrate from the freezer.  Thaw on the benchtop or in your fingers.
Label the PCR tubes with your initials and an incrementing number.
ie: BT-1, -2, -3, -4
Vortex the minipreps and the 10X buffer concentrate briefly, then pulse down in the microfuge.
For each miniprep, set up a PCR tube containing the following in order:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
Remove the enzyme from the freezer for as little time as possible.
I have specified an "arbitrary" 3 µl volume of miniprep DNA; this should be fine as long as your miniprep concentration is >= 100 ng/ul.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
Incubate at the appropriate temperature for at least 1 hour and more more than 16 hours.
If the enzyme's active temperature is 37°C, use the 37°C plate incubator.
Stop the reaction by adding 2 ul of 6X NEB purple gel loading dye to each reacti on.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
PAUSE POINT: The reaction can be stored almost indefinitely at room temperature once it's been stopped.
Proceed to gel electrophoresis.
Overnight liquid cultures (picking colonies)
Introduction
Overnight cultures are used to prepare miniprep DNA.
Materials
The plate from which you are picking colonies
15 ml round-bottom polystyrene tubes, one per culture
The ones with the snap caps, NOT conical tubes with screw caps
5 mL LB per culture
A container that can hold 5ml x the number of cultures
For a modest number of minipreps, a 50 ml conical tube works well.
For larger minipreps, use a sterile bottle (100 ml is frequently useful.)
Antibiotic stock, 1000X
Procedure
Materials Setup
Warm up the LB to at least room temperature (if it came from the fridge), but not warmer than 37°C
Label one round-bottom culture tube for each miniprep.  Use "NAME-1, NAME-2, ..."etc for the naming convention, where NAME is a shortened name of the plasmid (eg, "hEF1a:mKate").
Your impulse is to just use number, or initials and number, but trust me -- you will want to be able to identify this tube in three weeks when you've forgotten what you were doing.
Using a sterile pipette, transfer 5 ml of LB to the mixing container for each culture PLUS 5 ML.
Add antibiotic stock to a final concentration of 1X (1 µl stock for each 1 ml in the mixing container.)
Cap tightly and mix well.
Culture Setup
Using a sterile pipette, transfer 5 ml of LB+antibiotic to each round-bottom culture tube.
If you are making cultures with different antibiotics, take care that the right media goes in each tube.
Squirt ethanol on a pair of foreceps and wipe dry with a Kimwipe.
Use the foreceps to pick up a sterile 200µl pipette tip, scrape a colony off of the plate, and drop the pipette tip in the corresponding tube.
Repeat for each tube.
Transfer to an incubating shaker at 37°C and incubate 14-16 hours.
Don't over-grow too badly, or your yield will suffer.
If you need to grow longer, you can grow at 30°C instead for 20 hours.
DB Miniprep 7/8/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Julia Goupil
Date: 2016-07-08
Friday, 7/8
Recombinase SCre(1U1S1D)PHOTO2.jpg
Recombinase234PHOTO3.jpg
Recombinase234PHOTO2.jpg
Recombinase234.jpg
Recombinase SCre(1U1S1D).jpg
Recombinase234PHOTO2.jpg
A
B
C
D
E
F
1
1-1 SCre
172.2
ng/ul
2
1-2 SCre
133.2
3
1-3 SCre
193.5
4
1-4 SCre
117.8
5
2-1 FlpO
114.1
6
3-1 hef1a-SloxP
164.1
7
4-1 hef1a-FRT
97
8
4-2 hef1a-FRT
190.2
9
4-3 hef1a-FRT
179.8
Table1
clipboard_2016-07-08_10:08:01.png
Gel loading order
Top row:
L   1-1U   1-2U   1-3U   1-4U L   1-1S   1-2S   1-3S   1-4S L   1-1DD   1-2DD   1-3DD   1-4DD
Bottom row
L   2-1U   3-1U   4-1U   4-2U   4-3U L   2-1S   3-1S   4-1S   4-2S   4-3S L   2-1D   3-1DD   4-1DD   4-2DD   4-3DD
Sample 1 single digested with ScaI
Sample 2 single digested with DraI
Sample 3 and 4  single digested with XhoI
Cutsmart buffer was used for all digested
Miniprep
Introduction
The miniprep uses silica gel to isolate plasmid DNA from an E. coli culture
Materials
Buffer P1 (resuspension buffer)
Retrieve from refrigerator.  If you are opening a new miniprep kit, add the RNAse and LyseBlue reagent and check the box on the cap.
Buffer P2 (lysis buffer)
Open the cap and look at the lysis buffer.  Swirl it around.  If it appears cloudy, the SDS has fallen out of solution; warm it for a few minutes in the 55°C water bath.
Buffer N3 (neutralization buffer)
Buffer PB (binding buffer)
Buffer PE (rinse buffer)
Make sure the "Ethanol added?" box has been checked.  If you are opening a new miniprep kit, add absolute ethanol as per the kit instructions and check the box on the cap.
Buffer EB (elution buffer)
Miniprep waste container
Miniprep buffers contain salts that can't go down the sink.
Per miniprep: two microcentrifuge tubes and one blue spin column, with collection tube.
Procedure
Harvest and resuspension
For each culture, label two microcentrifuge tubes on the cap and one blue spin column on the side.
The spin columns should be in their (cap-less) collection vials.
Pipette 1.6 mL of each culture into the corresponding microcentrifuge tubes.
(This is for a 1.7 ml microcentrifuge tube; set your pipettor to 800 ul and do two transfers.  If we ever go back to 2-ml tubes, do 1.9 ml (2x950 ul))
Centrifuge at maximum speed (10,000 or 13,000xg) for three minutes.
Aspirate the supernatant, or pour it off into the bleach bucket.
Pipette ANOTHER 1.8 ml of each culture into the corresponding microcentrifuge tubes.
Centrifuge at maximum speed for three minutes.
While the centrifuge is running, move the remaining cultures to 4degC.
Aspirate the supernatant off with the bench aspirator.  Be careful not to disturb the pelleted E. coli.
We use an aspirator here because the less extra salt and protein we put in the miniprep, the better the yield is.
Add 250 µl Buffer P1 to each tube.
Resuspend the E. coli pellet.  The preferred way is with the roto-mixer at the other end of the lab.
Alternately, if you have just a few tubes, you can resuspend on a vortex.
Make sure to resuspend fully and thoroughly. The resulting suspension should be smooth and cloudy; if there is particulate matter floating around, vortex some more.
Lysis
Add 250 µl Buffer P2 to each tube.
Work quickly; the lysis step should take less than 5 minutes.
Snap the tubes closed and invert them 4-6 times, until the tube is thoroughly mixed and the entire solution turns blue.
If you have many many tubes, you can stack a second tube rack on top of them and invert the entire thing.
Add 350 ul Buffer N3 to each tube.
Snap the tubes closed and invert 4-6 times, until the solution is thouroughly mixed and no longer blue.
The solution will become cloudy or flocculent.
Centrifuge on high speed for 10 minutes.
Separation
Remove the tubes from the microcentrifuge, being careful not to disturb the white pellet.
Using P-1000 micropipettor set to 850 ul, carefully transfer the supernatant from each centrifuge tube to the corresponding blue spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Don't forget to put the lid on the rotor! Some of the salts get aerosolized because the spin columns don't have caps.
Pour the flow-through from each column into the miniprep waste container.
Pipette 500 ul of Buffer PB onto each spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Pipette 750 ul of Buffer PE onto each spin column
Wait 1-3 minutes.
This allows some of the salt that's still bound to the silica matrix to resuspend in the buffer.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Return each spin column to its collection tube and centrifuge an additional 1 minute at high speed.
This removes every last trace of buffer PE; the ethanol can screw up downstream steps.
Transfer each spin column to a clean labelled microcentrifuge tube.
Pipette 50 ul of Buffer EB onto the center of each column.
The volume of EB is comparable to the volume of silica gel matrix; if you pipette down the side, you might not get the entire transfer to the matrix.
Wait 1-3 minutes.
This gives the DNA a chance to dissociate from the silica matrix.
Centrifuge the spin columns, in their collection tubes, for one minute at maximum speed.
Proceed directly to analyze the samples on the Nanodrop.
Gel electrophoresis
Introduction
Gel electrophoresis separates pieces of DNA by length.
Materials
Agarose gel, 1 lane per sample, plus at least 1 lane for the ladder.
If you just poured the gel, make sure it has had at least 30 minutes to set.
If you have a large gel but only need a few lanes, cut out just a piece of it with a razor blade.  Cut straight!
Samples to run
6X NEB Purple Loading Dye
Molecular weight standard (aka "ladder")
Commonly available ladders are Hyperladder I and NEB 2-Log.
Gel box, lid, leads
Electrophoresis power supply
1X TAE buffer, enough to fill the gel box.
Procedure
Prepare your samples
If your samples are frozen, thaw them completely, flick or vortex to mix, then pulse down in the microfuge.
If your samples are not already in loading dye, mix them 1:6 with 6X NEB Purple Loading Dye
Add 1 µl of loading dye for every 5 µl of sample.
Pro tip: If you don't want to mix loading gel with your entire sample (PCR products, for example), cut off a strip of Parafilm; pipette 2-3 ul of sample onto the parafilm; add 1 ul loading dye; mix by pipetting and load directly.
Set up the gel box
Program the voltage on the power supply.  For the small gel box, use 100V; for the large gel box, use 150V.
Program the time on the power supply.  For a small analytical gel, set the timer for 30 minutes.
For larger gels, start at 45 minutes, and then check regularly!
Attach the leads to the gel box cover.  Make sure that the red (positive) lead is attached to the side of the box farthest away from the wells in the gel.
Remember, the DNA is negatively charged and will move toward the positive terminal. A useful neumonic for remembering this is "Run to Red."
If the gel box has not been used previously that day, empty it and rinse it out with DI water.
The TAE buffer can be re-used, but if it's been sitting out for too long it evaporates and the salt concentration (and conductivity) changes.  TAE is cheap; when in doubt, replace it.
Place the gel in the gel box.
Pour TAE into the gel box until it just barely covers the gel.
The TAE's purpose is to conduct electricity; over-filling the gel box results in a larger conductive path, more current flow, and more heating (which can screw up your run.)
Check the wells to see that they are free of bubbles.  If there are bubbles, blow them out by pipetting 100 µl of TAE from the gel box into the well.
Load your samples
Load your samples. For large combs, load 10 µl; for small combs, load 5 µl.
Work quickly. The samples begin to diffuse in the buffer, leading to smeared bands.
Load the ladder in the last lane.  Load 1/2 the volume of your samples: for large combs, load 5 µl, for small combs, load 2.5 µl.
Place the lid on the gel box. Make sure it is seated on the brass contacts.
Run the gel
Press the Start or Run button on the power supply.
Make sure the power supply doesn't complain about an open circuit.  If it does, re-seat the gel box lid and press Run again.
Double-check that there are bubbles forming on the platinum wires at either end of the gel box.
Double-check that the red (positive) lead is on the side farthest from the wells.  Remember, Run to Red.
Run until the pink band is 2/3 to 3/4 of the way down the gel.  For small analytical gels, this should take 30 minutes.  For larger gels, start at 45 minutes and check regularly!
Image the gel
If the power supply is still running, press the Stop button.
Lift the lid off of the gel box. Lift the lid straight up. If you try to "hinge" it up, the lid will break.
Transfer the gel to the GelDoc.
If it's not running, start QuantityOne from the toolbar.
If necessary, click the top button on the toolbox to select the scanner.
If necessary, reset the camera (as per instructions on the GelDoc computer.)
Press the Epifluorescent Illumination button on the GelDoc.  Check the gel's position, zoom and focus.
You want to be zoomed in so that the gel fills the field of view, and focussed so that the well edges are sharp.
Close the GelDoc door. Click Auto Expose.
The Auto Expose functionality generally over-exposes my gels a bit. If your gel is over-exposed, remove 1/3 of the exposure time, type it into the Exposure box, then click Manually Expose.
In the File menu, select, Export as JPEG.... Save your gel to the iGEM folder on the desktop.
Open Benchling.  Log in, and copy the gel to the Description page for the plasmid you're building.
If there are multiple plasmids on the gel, save it to each plasmid's Description.
Discard the gel in the biowaste box. Wipe down the gel doc with a little water and a paper towel or Kimwipe.
Annotate the gel
Immediately, before you forget what's where, annotate the gel.
List what is in each lane.
Describe whether the pattern is what you expected or not. (You should have an in silico digestion to compare it to!)
