Team:MIT/Recombinase Group Notebook Week 6
BACKGROUND
EXPERIMENTS
Synthetic Promoters microRNA Recombinase Proof of Concept Collaborations
FUTURE WORK
Circuit Design Clinical Applications
HUMAN PRACTICES
Parts Toolbox Accessible Protocols Integrated Human Practices Public Outreach Interlab Study
NOTEBOOK
Lab Notebook Parts Mammalian Parts Collection Attributions Safety
AWARDS
HP Silver HP Gold Integrated HP Basic Part
TEAM
TN 07/10/16 · Benchling
TN 07/10/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Trinh Nguyen
Date: 2016-07-10
Sunday, 7/10
pDONOR L1_LacZ_R1 ran out. So, I can only do GG for SCRE and FLPO
I tried to vary the ratio of backbone and part
I diluted the gBlocks down to 5fmol/ul and used it to prepare the GG mixture
4 samples: (backbone:gBlock)
SCRE 1:1
SCRE 1:2
FLPO 1:1
FLPO 1:2
Golden Gate Assembly
Introduction
Golden Gate is an assembly method using Type II-S restriction enzymes and predefined overlap sequences to assemble multiple pieces of DNA in one reaction.  Golden Gate design is beyond the scope of this protocol; this protocol is the reaction setup only.
Materials
200 µl thin-wall PCR strip tubes, one per reaction
Backbone plasmid
Usually pDONR-GG L1_lacZ_L2 or pDONR-GG L4_lacZ_R1
Parts being assembled, either in plasmids or gBlocks
Nuclease-free water
T4 ligase buffer, 1.5 µl per reaction
Found in single-use aliquots of 10 (??) µl in the -20° freezer.  T4 ligase buffer does NOT like to be thawed and frozen again; once you use an aliquot, discard what's left!
10X BSA, 1.5 µl per reaction
Found in the -20° freezer.  ...and that's Bovine Serum Albumin, NOT the BsaI enzyme!
BsaI (or AarI) restriction enzyme, 1 µl per reaction
Leave the BsaI and T4 ligase in the freezer until the very end, and add after the rest of the reaction has been set up.
T4 ligase, 1 µl per reaction
Many protocols (both online and in the lab) call for high-concentration T4 ligase.  This is only necessary if you're doing VERY LARGE Golden Gate reactions (5-8 parts).  You do NOT need the HC ligase for most of the GGs you'll be doing.
Leave the BsaI and the T4 ligase in the freezer until the very end, and add after the rest of the reaction has been set up.
An empty 10µl or 200µl tip box
Ice in an ice bucket
Procedure
Record your reaction setup.
A Golden Gate works best when there are equimolar amounts of each part in the reaction.  By convention, we use 50 fmol of each part.  For each part (plasmid and gBlock), compute the concentration (fmol/ul) and the volume required for 50 fmol in the table below.  If you resuspended your gBlocks at 50 fmol/ul, you can omit them in this table.
A
B
C
D
E
1
Part
Size (kb)
Concentration, (ng/ul)
Concentration, (fmol/ul)
Volume for 50 fmol
2
pDONR-GG
3.4
100
44.6
1.1
3
4
5
Table1
Record the reaction setup for each reaction below.  For each reaction, include:
-- 50 fmol of each part
-- 1.5 ul T4 ligase buffer
-- 1.5 ul 10X BSA
-- 1 ul BsaI
-- 1 ul T4 ligase
-- Water to a total volume of 15 ul.
-- NOTE: If you're using AarI, you need to add the AarI oligo as well!
A
B
C
D
1
A19 L4_PhlF_R1
2
pDONR-GG L1_L2
1.1 ul
3
BT-01 gBlock
1 ul
4
BT-02 gBlock
1 ul
5
Water
6.9 ul
6
T4 ligase buffer
1.5 ul
7
10X BSA
1.5 ul
8
BsaI or AarI
1 ul
9
T4 ligase
1 ul
Table2
Set up your thermocycler
Program a thermocycler with the following program:
NOTE: If you are using AarI, use 50 cycles instead of 15.
A
B
1
Heat lid
70°
2
Start cycle
15X
3
--
37° for 1' 30"
4
--
16° for 3' 0"
5
Close Cycle
6
50° for 5'
7
80° for 10'
8
Store at 8°
Table3
Set up your reactions
Fill an empty tip box with ice, put the grey platform back on, then fill with water until just under the rack.
Label one thin-walled PCR tube per reaction.
Add the reaction components. Add the water first; add the restriction enzyme and buffer last.
Cap the tubes.  Flick a few times to mix, then pulse spin.
Load the tubes in the thermocycler and start the program.
Transform
Transform as usual. Plate only 10 ul of the outgrowth in a 200 ul puddle of water, or you will get a lawn of colonies.
TN 07/11/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Elizabeth Strand
Date: 2016-07-11
Monday, 7/11
Perform 2 sets of Golden Gate reactions
Set 1 included 5 samples with 1:1 ratio of backbone:part (50fmol of each part)
Thermocycler: ~2hrs
Sample 1: SCRE
Sample 2: FLPO
Sample 3: hEF1a - SloxP - SV40
Sample 4: hEF1a - FRT - SV40
Sample 5: hEF1a - SV40
Set 2 included 2 samples with 1:5 ratio of backbone:part (50fmol of each part and 10fmoll of backbone)
Thermocycler: ~8hrs (EBGG protocol)
Sample 1: SCRE
Sample 2: hEF1a - SloxP - SV40
EBGG thermocycler for GG
37 5min
Cycle 50X: { 37 2.5min - 4C 0.5min -  16C 5.5min }
37C 10 min
80C 20 min
4C hold
Golden Gate Assembly
Introduction
Golden Gate is an assembly method using Type II-S restriction enzymes and predefined overlap sequences to assemble multiple pieces of DNA in one reaction.  Golden Gate design is beyond the scope of this protocol; this protocol is the reaction setup only.
Materials
200 µl thin-wall PCR strip tubes, one per reaction
Backbone plasmid
Usually pDONR-GG L1_lacZ_L2 or pDONR-GG L4_lacZ_R1
Parts being assembled, either in plasmids or gBlocks
Nuclease-free water
T4 ligase buffer, 1.5 µl per reaction
Found in single-use aliquots of 10 (??) µl in the -20° freezer.  T4 ligase buffer does NOT like to be thawed and frozen again; once you use an aliquot, discard what's left!
10X BSA, 1.5 µl per reaction
Found in the -20° freezer.  ...and that's Bovine Serum Albumin, NOT the BsaI enzyme!
BsaI (or AarI) restriction enzyme, 1 µl per reaction
Leave the BsaI and T4 ligase in the freezer until the very end, and add after the rest of the reaction has been set up.
T4 ligase, 1 µl per reaction
Many protocols (both online and in the lab) call for high-concentration T4 ligase.  This is only necessary if you're doing VERY LARGE Golden Gate reactions (5-8 parts).  You do NOT need the HC ligase for most of the GGs you'll be doing.
Leave the BsaI and the T4 ligase in the freezer until the very end, and add after the rest of the reaction has been set up.
An empty 10µl or 200µl tip box
Ice in an ice bucket
Procedure
Record your reaction setup.
A Golden Gate works best when there are equimolar amounts of each part in the reaction.  By convention, we use 50 fmol of each part.  For each part (plasmid and gBlock), compute the concentration (fmol/ul) and the volume required for 50 fmol in the table below.  If you resuspended your gBlocks at 50 fmol/ul, you can omit them in this table.
A
B
C
D
E
1
Part
Size (kb)
Concentration, (ng/ul)
Concentration, (fmol/ul)
Volume for 50 fmol
2
pDONR-GG
3.4
100
44.6
1.1
3
4
5
Table1
Record the reaction setup for each reaction below.  For each reaction, include:
-- 50 fmol of each part
-- 1.5 ul T4 ligase buffer
-- 1.5 ul 10X BSA
-- 1 ul BsaI
-- 1 ul T4 ligase
-- Water to a total volume of 15 ul.
-- NOTE: If you're using AarI, you need to add the AarI oligo as well!
A
B
C
D
1
A19 L4_PhlF_R1
2
pDONR-GG L1_L2
1.1 ul
3
BT-01 gBlock
1 ul
4
BT-02 gBlock
1 ul
5
Water
6.9 ul
6
T4 ligase buffer
1.5 ul
7
10X BSA
1.5 ul
8
BsaI or AarI
1 ul
9
T4 ligase
1 ul
Table2
Set up your thermocycler
Program a thermocycler with the following program:
NOTE: If you are using AarI, use 50 cycles instead of 15.
A
B
1
Heat lid
70°
2
Start cycle
15X
3
--
37° for 1' 30"
4
--
16° for 3' 0"
5
Close Cycle
6
50° for 5'
7
80° for 10'
8
Store at 8°
Table3
Set up your reactions
Fill an empty tip box with ice, put the grey platform back on, then fill with water until just under the rack.
Label one thin-walled PCR tube per reaction.
Add the reaction components. Add the water first; add the restriction enzyme and buffer last.
Cap the tubes.  Flick a few times to mix, then pulse spin.
Load the tubes in the thermocycler and start the program.
Transform
Transform as usual. Plate only 10 ul of the outgrowth in a 200 ul puddle of water, or you will get a lawn of colonies.
Transformation of E. coli
Introduction
Transformation is the process of inducing chemically competent E. coli to take up DNA.
Materials
Dry bath, set to 42°C
Fill the wells in the dry bath block 1/2 full with DI water.
Ice bucket, with ice
For thawing competent cells.
DNA to transform
Could be an assembly reaction (LR, Golden Gate, etc) or a miniprepped plasmid.
If you removed it from the freezer, make sure it's entirely thawed  out.
pUC19 Transformation Control, 1 pg/µl
The pUC19 control will tell you how efficient your transformations were.
SOC growth media, at room temperature
Check to make sure it's clear and NOT CLOUDY.
Antibiotic plates, one per transformation, plus 1 Amp plate for the pUC19 control
Make sure the plates you use match the resistance cassette of the plasmid!
Competent E. coli, one tube per transformation + one for the pUC19 control
These live in the -80 in 235.
Thaw on ice 3-4 minutes.
A timer, set for 30 seconds.
Procedure
Setup
Make sure the dry bath is set to 42°C and the wells in the block are 1/2 full of DI water
Remove selection plates from the refrigerator.  Double-check that they match the selection marker on your plasmid, then place them in the 37° incubator.
Retrieve the DNA to transform.
If frozen: thaw, completely, flick a few times to mix, then pulse down in the microfuge.
Fill an ice bucket with ice.  Retrieve one tube of competent E. coli per transformation from the -80 and thaw on ice, 3-4 minutes.
00:03:00
While the transformation tubes are thawing, label their tops with something descriptive.  Record the labels here:
A
B
C
D
E
F
G
H
1
SCRE 1:1
2
SCRE 1:2
3
FLPO 1:1
4
FLPO 1:2
Table1
Transformation
Add 2 µl DNA from each reaction to a tube of competent cells.
Immediately after adding the DNA to each tube, stir the cells a few times with the pipette tip.
Add 1 µl of the pUC19 transformation control to the positive control tube.
Incubate on ice for 30 minutes.
00:30:00
Heat shock the cells for exactly 30 seconds in the 42° heat block. (Yes, set a timer.)
00:00:30
Place back on ice for 2 minutes.
00:02:00
Add 250 µl SOC to each tube.
Tape the tubes to the platform of a shaker at 37°C and shake at 270 RPM for 60 minutes.
01:00:00
Plating
Label the selection plates using the labels you recorded above.
Shake ~10 plating beads onto each plate.
Pipette 100 µl of each transformation onto the corresponding plates. NOTE: if you are using DNA from a golden gate reaction, see the golden gate protocol for instructions to dilute your sample.
-----Golden Gate: Plate only 10 ul of the outgrowth in a 200 ul puddle of water, or you will get a lawn of colonies.
Cover the plates and shake the beads around to spread the cells out.
Dispose of the beads by tapping them into the waste container.
Incubate the plates upside down overnight in the 37° incubator.
Don't incubate for more than 18-24 hours.
Compute transformation efficiency
Count the colonies on your positive transformation plate.
If there are many many colonies, then hooray!  You had a great transformation.  Just estimate.
Divide the number of colonies by the fraction of the transformation you plated.
So, if you resuspended your transformation in a total volume of 300 ul, then plated 100 ul, multiply the number of colonies by 3.
Transformation efficiency is expressed in colonies per microgram pUC19.  Multiply the number of colonies by the appropriate conversion factor.
So if you transformed 1 picogram of pUC19 DNA, multiply by 106.
Record your transformation efficiency in your (daily) lab notebook.
Overnight liquid cultures (picking colonies)
Introduction
Overnight cultures are used to prepare miniprep DNA.
Materials
The plate from which you are picking colonies
15 ml round-bottom polystyrene tubes, one per culture
The ones with the snap caps, NOT conical tubes with screw caps
5 mL LB per culture
A container that can hold 5ml x the number of cultures
For a modest number of minipreps, a 50 ml conical tube works well.
For larger minipreps, use a sterile bottle (100 ml is frequently useful.)
Antibiotic stock, 1000X
Procedure
Materials Setup
Warm up the LB to at least room temperature (if it came from the fridge), but not warmer than 37°C
Label one round-bottom culture tube for each miniprep.  Use "NAME-1, NAME-2, ..."etc for the naming convention, where NAME is a shortened name of the plasmid (eg, "hEF1a:mKate").
Your impulse is to just use number, or initials and number, but trust me -- you will want to be able to identify this tube in three weeks when you've forgotten what you were doing.
Using a sterile pipette, transfer 5 ml of LB to the mixing container for each culture PLUS 5 ML.
Add antibiotic stock to a final concentration of 1X (1 µl stock for each 1 ml in the mixing container.)
Cap tightly and mix well.
Culture Setup
Using a sterile pipette, transfer 5 ml of LB+antibiotic to each round-bottom culture tube.
If you are making cultures with different antibiotics, take care that the right media goes in each tube.
Squirt ethanol on a pair of foreceps and wipe dry with a Kimwipe.
Use the foreceps to pick up a sterile 200µl pipette tip, scrape a colony off of the plate, and drop the pipette tip in the corresponding tube.
Repeat for each tube.
Transfer to an incubating shaker at 37°C and incubate 14-16 hours.
Don't over-grow too badly, or your yield will suffer.
If you need to grow longer, you can grow at 30°C instead for 20 hours.
TN meeting with Linda
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Trinh Nguyen
Date: 2016-07-11
Monday, 7/11
Epithelial cells vs. stroma cells
Epithelial cells are harder to transfected and to grow
How to explain the circuit diagram?
explain in words first , with the estrogen high and low during menstruation cycle
explain each phase of the circuit
then put up the circuit
Estrogen responsive promoter in the output plasmid
how do we visualize implimenting this? Just thought experiments
delivery method? AV (~4.8kb)
Hugh challenge is heterogeniety
response to drug differently => feature for circuit
biopsy _ RNA profile, but not tell us the interaction and the phenotypic traits
Degradation domain fuses to protein, which is stabilize by a small molecule
progesterone fusing to degradation tag
Progesterone:
endogenous (screw up in endometriosis)
synthetic
LET'S GO BACK TO PROGESTERONE PHASE!!!
Test run on endometriosis cells for estrogene and progestrone level
Another option for practicality: in vitro diagnostic by testing on biopsy samples
vivo diagnostic
vivo therapeutic
platforms for vivo: AV, RNA
threshold to detect the size of lession
CF 7/12/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Colleen Foley
Date: 2016-07-12
Tuesday, 7/12
1.
SCre 1:1
2.
FLPO 1:1
3.
SloxP - SV40 1:1
4.
FRT - SV40 1:1
5.
SV40 1:1
6.
SCre 1:5
7.
SloxP - SV40 1:5
Transformation of E. coli
Introduction
Transformation is the process of inducing chemically competent E. coli to take up DNA.
Materials
Dry bath, set to 42°C
Fill the wells in the dry bath block 1/2 full with DI water.
Ice bucket, with ice
For thawing competent cells.
DNA to transform
Could be an assembly reaction (LR, Golden Gate, etc) or a miniprepped plasmid.
If you removed it from the freezer, make sure it's entirely thawed  out.
pUC19 Transformation Control, 1 pg/µl
The pUC19 control will tell you how efficient your transformations were.
SOC growth media, at room temperature
Check to make sure it's clear and NOT CLOUDY.
Antibiotic plates, one per transformation, plus 1 Amp plate for the pUC19 control
Make sure the plates you use match the resistance cassette of the plasmid!
Competent E. coli, one tube per transformation + one for the pUC19 control
These live in the -80 in 235.
Thaw on ice 3-4 minutes.
A timer, set for 30 seconds.
Procedure
Setup
Make sure the dry bath is set to 42°C and the wells in the block are 1/2 full of DI water
Remove selection plates from the refrigerator.  Double-check that they match the selection marker on your plasmid, then place them in the 37° incubator.
Retrieve the DNA to transform.
If frozen: thaw, completely, flick a few times to mix, then pulse down in the microfuge.
Fill an ice bucket with ice.  Retrieve one tube of competent E. coli per transformation from the -80 and thaw on ice, 3-4 minutes.
00:03:00
While the transformation tubes are thawing, label their tops with something descriptive.  Record the labels here:
A
B
C
D
E
F
G
H
1
puc-19L
2
puc-19TC
3
Pdest_mCherry
4
pDest
Table1
Transformation
Add 2 µl DNA from each reaction to a tube of competent cells.
Immediately after adding the DNA to each tube, stir the cells a few times with the pipette tip.
Add 1 µl of the pUC19 transformation control to the positive control tube.
Incubate on ice for 30 minutes.
00:30:00
Heat shock the cells for exactly 30 seconds in the 42° heat block. (Yes, set a timer.)
00:00:30
Place back on ice for 2 minutes.
00:02:00
Add 250 µl SOC to each tube.
Tape the tubes to the platform of a shaker at 37°C and shake at 270 RPM for 60 minutes.
01:00:00
Plating
Label the selection plates using the labels you recorded above.
Shake ~10 plating beads onto each plate.
Pipette 100 µl of each transformation onto the corresponding plates. NOTE: if you are using DNA from a golden gate reaction, see the golden gate protocol for instructions to dilute your sample.
-----Golden Gate: Plate only 10 ul of the outgrowth in a 200 ul puddle of water, or you will get a lawn of colonies.
Cover the plates and shake the beads around to spread the cells out.
Dispose of the beads by tapping them into the waste container.
Incubate the plates upside down overnight in the 37° incubator.
Don't incubate for more than 18-24 hours.
Compute transformation efficiency
Count the colonies on your positive transformation plate.
If there are many many colonies, then hooray!  You had a great transformation.  Just estimate.
Divide the number of colonies by the fraction of the transformation you plated.
So, if you resuspended your transformation in a total volume of 300 ul, then plated 100 ul, multiply the number of colonies by 3.
Transformation efficiency is expressed in colonies per microgram pUC19.  Multiply the number of colonies by the appropriate conversion factor.
So if you transformed 1 picogram of pUC19 DNA, multiply by 106.
Record your transformation efficiency in your (daily) lab notebook.
CF 7/13/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Trinh Nguyen
Date: 2016-07-13
Wednesday, 7/13
Meeting w/Deepak + Allen:
●
L7AE expression must be there constituitively
○
could have it with DD domain, but the problem with DD domain is that you actually have to wash out the ligand
●
if washing out is problematic, you could try a SMASH tag
○
unclear if we have L7AE-SMASH in lab
■
someone's probably made one - maybe Jake? Ron would know, or we could email the lab
●
put two K turns in front of recombinase, then add constitutive L7AE-SMASH
○
should be sufficient
○
this would be the easiest way
●
DD tag is more challenging - have to wash out ligand, have to wait a few days, change media, etc.
○
would be hard with transient transfection
●
try to keep output as low as possible with recombinase
○
50 ng is the lowest you can go and still get fluorescent response
○
lower than 50 you get into the noise of the pippetter
●
aptamers and aptazymes don't have as good fold difference
○
theofeline (spelling?) is probably the best but Deepak still says he can't really endorse it
●
"L7AE is a champ"
○
can get away with 2 K-turns
○
can do more K-turns if you want, but 2 should be good for our stuff
○
promoter K-turns recombinase L7AE SMASH
●
Talk to Jake, maybe Eli? Kevin (but he's on vacation)! Maybe Leila?
○
can provide sequences and some guidance
○
send email to Jake since he's in town
Miniprep Concentrations:
1-1: 182.1
1-2: 258.6
1-3: 254.0
1-4: 254.9
2-1: 258.0
2-2: 189.9
2-3: 216.4
2-4: 227.6
3-3: 336.1
3-4: 391.7
4-1: 328.2
4-2: 348.3
4-3: 251.9
4-4: 331.9
Gel Info:
A
B
C
D
E
F
G
H
I
J
K
L
M
N
O
P
Q
R
S
T
1
EMPTY
L
1-1
1-2
1-3
1-4
L
2-1
2-2
2-3
2-4
3-2
L
3-3
3-4
4-1
4-2
4-3
4-4
2-1 DD
2
2-2 DD
2-3 DD
2-4 DD
1-1 DD
1-2 DD
L
1-3 DD
1-4 DD
3-2 DD
3-3 DD
3-4 DD
L
4-1 DD
4-2 DD
4-3 DD
4-4 DD
BB 1/2
BB 3/4
L
EMPTY
Table1
Single digest for SCre: ScaI
Double digest for SCre:  ScaI + EcoRV-HF
Single digest for FLPO: DraI
Double digest for FLPO: DraI + ApaI-HF
Enzyme Buffer: Cutsmart
Temperature: 37 C
Incubation Time: 1.5 hr
Gel Results:
Recombinase1,2,3,4U2-1DDGOODQUALITY.jpg
Recombinase2-2,2-3,2-4,1,3,4DDB1B3.jpg
clipboard_2016-07-13_10:41:10.png
Samples that we sent in for sequencing:
1-2, 1-4
2-3, 2-4
3-3, 3-4
4-2, 4-3
Miniprep
Introduction
The miniprep uses silica gel to isolate plasmid DNA from an E. coli culture
Materials
Buffer P1 (resuspension buffer)
Retrieve from refrigerator.  If you are opening a new miniprep kit, add the RNAse and LyseBlue reagent and check the box on the cap.
Buffer P2 (lysis buffer)
Open the cap and look at the lysis buffer.  Swirl it around.  If it appears cloudy, the SDS has fallen out of solution; warm it for a few minutes in the 55°C water bath.
Buffer N3 (neutralization buffer)
Buffer PB (binding buffer)
Buffer PE (rinse buffer)
Make sure the "Ethanol added?" box has been checked.  If you are opening a new miniprep kit, add absolute ethanol as per the kit instructions and check the box on the cap.
Buffer EB (elution buffer)
Miniprep waste container
Miniprep buffers contain salts that can't go down the sink.
Per miniprep: two microcentrifuge tubes and one blue spin column, with collection tube.
Procedure
Harvest and resuspension
For each culture, label two microcentrifuge tubes on the cap and one blue spin column on the side.
The spin columns should be in their (cap-less) collection vials.
Pipette 1.6 mL of each culture into the corresponding microcentrifuge tubes.
(This is for a 1.7 ml microcentrifuge tube; set your pipettor to 800 ul and do two transfers.  If we ever go back to 2-ml tubes, do 1.9 ml (2x950 ul))
Centrifuge at maximum speed (10,000 or 13,000xg) for three minutes.
Aspirate the supernatant, or pour it off into the bleach bucket.
Pipette ANOTHER 1.8 ml of each culture into the corresponding microcentrifuge tubes.
Centrifuge at maximum speed for three minutes.
While the centrifuge is running, move the remaining cultures to 4degC.
Aspirate the supernatant off with the bench aspirator.  Be careful not to disturb the pelleted E. coli.
We use an aspirator here because the less extra salt and protein we put in the miniprep, the better the yield is.
Add 250 µl Buffer P1 to each tube.
Resuspend the E. coli pellet.  The preferred way is with the roto-mixer at the other end of the lab.
Alternately, if you have just a few tubes, you can resuspend on a vortex.
Make sure to resuspend fully and thoroughly. The resulting suspension should be smooth and cloudy; if there is particulate matter floating around, vortex some more.
Lysis
Add 250 µl Buffer P2 to each tube.
Work quickly; the lysis step should take less than 5 minutes.
Snap the tubes closed and invert them 4-6 times, until the tube is thoroughly mixed and the entire solution turns blue.
If you have many many tubes, you can stack a second tube rack on top of them and invert the entire thing.
Add 350 ul Buffer N3 to each tube.
Snap the tubes closed and invert 4-6 times, until the solution is thouroughly mixed and no longer blue.
The solution will become cloudy or flocculent.
Centrifuge on high speed for 10 minutes.
Separation
Remove the tubes from the microcentrifuge, being careful not to disturb the white pellet.
Using P-1000 micropipettor set to 850 ul, carefully transfer the supernatant from each centrifuge tube to the corresponding blue spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Don't forget to put the lid on the rotor! Some of the salts get aerosolized because the spin columns don't have caps.
Pour the flow-through from each column into the miniprep waste container.
Pipette 500 ul of Buffer PB onto each spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Pipette 750 ul of Buffer PE onto each spin column
Wait 1-3 minutes.
This allows some of the salt that's still bound to the silica matrix to resuspend in the buffer.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Return each spin column to its collection tube and centrifuge an additional 1 minute at high speed.
This removes every last trace of buffer PE; the ethanol can screw up downstream steps.
Transfer each spin column to a clean labelled microcentrifuge tube.
Pipette 50 ul of Buffer EB onto the center of each column.
The volume of EB is comparable to the volume of silica gel matrix; if you pipette down the side, you might not get the entire transfer to the matrix.
Wait 1-3 minutes.
This gives the DNA a chance to dissociate from the silica matrix.
Centrifuge the spin columns, in their collection tubes, for one minute at maximum speed.
Proceed directly to analyze the samples on the Nanodrop.
Diagnostic Restriction Digestion
Introduction
A diagnostic restriction digest helps identify correctly assembled clones from incorrect clones.
Materials
200 ul PCR strip tubes, one per reaction
Restriction enzyme (chosen below), 1 ul per reaction.
10X restriction enzyme buffer (chosen below), 1 ul per reaction.
Procedure
Choose A Good Restriction Enzyme
Using Benchling, choose a restriction enzyme that meets the following criteria:
- Cuts at least once in the insert (for a pEXPR, either the promoter or the gene.)
- Cuts at least once in the backbone.
- Gives bands that aren't too large (> 8 kb) or too small (< 200 bp).
- Gives a band pattern that is significantly different from the expected error mode.
* For LR reactions, the most common error is a pDEST that slipped through the selection.
- Begin in the "Brian's Favorites" list, then expand to the main Weiss lab list.
- If you can't find a single enzyme that gives an acceptable band pattern, choose two enzymes that give a acceptable band pattern when used together.
* This double-digest is subject to buffer compatibility, outlined below.
Record your enzyme choice on the plasmid's Description page.
Benchling will tell you the enzyme's buffer compatibility and active temperature. Record the buffer in which the enzyme is most active.
If there are multiple bufers in which the enzyme is equally active, choose in this order: Buffer 3.1, Buffer CS, Buffer 2.1, Buffer 1.1.
Set up the restriction digest
Retrieve the minipreps and the appropriate 10X buffer concentrate from the freezer.  Thaw on the benchtop or in your fingers.
Label the PCR tubes with your initials and an incrementing number.
ie: BT-1, -2, -3, -4
Vortex the minipreps and the 10X buffer concentrate briefly, then pulse down in the microfuge.
For each miniprep, set up a PCR tube containing the following in order:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
Remove the enzyme from the freezer for as little time as possible.
I have specified an "arbitrary" 3 µl volume of miniprep DNA; this should be fine as long as your miniprep concentration is >= 100 ng/ul.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
Incubate at the appropriate temperature for at least 1 hour and more more than 16 hours.
If the enzyme's active temperature is 37°C, use the 37°C plate incubator.
Stop the reaction by adding 2 ul of 6X NEB purple gel loading dye to each reacti on.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
PAUSE POINT: The reaction can be stored almost indefinitely at room temperature once it's been stopped.
Proceed to gel electrophoresis.
Gel electrophoresis
Introduction
Gel electrophoresis separates pieces of DNA by length.
Materials
Agarose gel, 1 lane per sample, plus at least 1 lane for the ladder.
If you just poured the gel, make sure it has had at least 30 minutes to set.
If you have a large gel but only need a few lanes, cut out just a piece of it with a razor blade.  Cut straight!
Samples to run
6X NEB Purple Loading Dye
Molecular weight standard (aka "ladder")
Commonly available ladders are Hyperladder I and NEB 2-Log.
Gel box, lid, leads
Electrophoresis power supply
1X TAE buffer, enough to fill the gel box.
Procedure
Prepare your samples
If your samples are frozen, thaw them completely, flick or vortex to mix, then pulse down in the microfuge.
If your samples are not already in loading dye, mix them 1:6 with 6X NEB Purple Loading Dye
Add 1 µl of loading dye for every 5 µl of sample.
Pro tip: If you don't want to mix loading gel with your entire sample (PCR products, for example), cut off a strip of Parafilm; pipette 2-3 ul of sample onto the parafilm; add 1 ul loading dye; mix by pipetting and load directly.
Set up the gel box
Program the voltage on the power supply.  For the small gel box, use 100V; for the large gel box, use 150V.
Program the time on the power supply.  For a small analytical gel, set the timer for 30 minutes.
For larger gels, start at 45 minutes, and then check regularly!
Attach the leads to the gel box cover.  Make sure that the red (positive) lead is attached to the side of the box farthest away from the wells in the gel.
Remember, the DNA is negatively charged and will move toward the positive terminal. A useful neumonic for remembering this is "Run to Red."
If the gel box has not been used previously that day, empty it and rinse it out with DI water.
The TAE buffer can be re-used, but if it's been sitting out for too long it evaporates and the salt concentration (and conductivity) changes.  TAE is cheap; when in doubt, replace it.
Place the gel in the gel box.
Pour TAE into the gel box until it just barely covers the gel.
The TAE's purpose is to conduct electricity; over-filling the gel box results in a larger conductive path, more current flow, and more heating (which can screw up your run.)
Check the wells to see that they are free of bubbles.  If there are bubbles, blow them out by pipetting 100 µl of TAE from the gel box into the well.
Load your samples
Load your samples. For large combs, load 10 µl; for small combs, load 5 µl.
Work quickly. The samples begin to diffuse in the buffer, leading to smeared bands.
Load the ladder in the last lane.  Load 1/2 the volume of your samples: for large combs, load 5 µl, for small combs, load 2.5 µl.
Place the lid on the gel box. Make sure it is seated on the brass contacts.
Run the gel
Press the Start or Run button on the power supply.
Make sure the power supply doesn't complain about an open circuit.  If it does, re-seat the gel box lid and press Run again.
Double-check that there are bubbles forming on the platinum wires at either end of the gel box.
Double-check that the red (positive) lead is on the side farthest from the wells.  Remember, Run to Red.
Run until the pink band is 2/3 to 3/4 of the way down the gel.  For small analytical gels, this should take 30 minutes.  For larger gels, start at 45 minutes and check regularly!
Image the gel
If the power supply is still running, press the Stop button.
Lift the lid off of the gel box. Lift the lid straight up. If you try to "hinge" it up, the lid will break.
Transfer the gel to the GelDoc.
If it's not running, start QuantityOne from the toolbar.
If necessary, click the top button on the toolbox to select the scanner.
If necessary, reset the camera (as per instructions on the GelDoc computer.)
Press the Epifluorescent Illumination button on the GelDoc.  Check the gel's position, zoom and focus.
You want to be zoomed in so that the gel fills the field of view, and focussed so that the well edges are sharp.
Close the GelDoc door. Click Auto Expose.
The Auto Expose functionality generally over-exposes my gels a bit.  If your gel is over-exposed, remove 1/3 of the exposure time, type it into the Exposure box, then click Manually Expose.
In the File menu, select, Export as JPEG.... Save your gel to the iGEM folder on the desktop.
Open Benchling.  Log in, and copy the gel to the Description page for the plasmid you're building.
If there are multiple plasmids on the gel, save it to each plasmid's Description.
Discard the gel in the biowaste box. Wipe down the gel doc with a little water and a paper towel or Kimwipe.
Annotate the gel
Immediately, before you forget what's where, annotate the gel.
List what is in each lane.
Describe whether the pattern is what you expected or not. (You should have an in silico digestion to compare it to!)
Sequencing protocol
Introduction
Preparing sample to send to Genewiz for sequencing Sequencing Pick-up Times: 9:30am, 3:00 pm, 5:30 pm
Materials
DNA stock
Nuclease-free water
Primer (5nM stock)
One microcentrifuge tube for each DNA plasmid
2 PCR tubes per plasmid (must be in strips of eight, round up if needed!)
Procedure
Diluting the DNA
For orders with <48 samples, use 8-strip PCR tubes. Label your tubes on the side vertically with "MI" and sample number (01, 02, 03, etc.). Use image below as guideline. Each plasmid you want to sequence will require 2 tubes, one for the forward primer and one for the reverse primer. Use "MI" instead of the "GW" in the picture.
Dilute your sequencing primer to 5 µM (pmol/µl) using water. You will need 5 µl for each sequencing reaction. Note, the primers in the tubes from IDT have not been diluted! Use already diluted stock (should be in regular microcentrifuge tubes).
Using the recorded concentration, dilute the DNA to the template concentration in 10 µl as detailed below. For example, a 5 kb plasmid would need a template concentration of ~50 ng/µl. Final volume for your working stock should 30 uL. Please make dilutions in water or Tris.
Add 10 µl of diluted template DNA to each tube.
Add 5 µl of diluted primer to each tube. Be careful about putting the forward and reverse primers in the correct tubes! Remember, for each DNA template, there should be two tubes - one with reverse primer and the other with forward primer
Vortex briefly
Log into genewiz. Username: igem-sequence@mit.edu Password: igem2014citrus
Click "create sequencing order"
Sevice Priority: click on "standard"
Create Order by: click on "online form"
Sample Type: click on "Pre-mixed"
Enter in how many samples you are sending in and click "Create new form"
Enter in descriptive names for your DNA (this is to help you remember when you get the results back), the DNA type, the DNA length (get this from your construct on benchling), the primer name (again, this is more for you). When you are done hit save and next.
It will give you an list of what you have entered and a price for each reaction. Hit "next step"
The payment information field should be autofilled. There has been an issue recently with the PO box number not being entered. If this is the case, click "credit card" (next to "payment info")  and then click "PO" to bring you back to the original form. Now you should be able to enter in a PO number. Open up a previous submitted order (there should be some saved to the desktop called "sequencing" or something like that) and copy over the PO number.
Hit "next step". Once it gives you a submitted order form, print 2 copies and pick them up from the printer (exit lab, through glass doors, on your left)
Put your PCR tubes in a large falcon tube.
Put the falcon tube in a ziploc bag (located in Qiagen column drawer) and put in one order form. Label the bag with something descriptive ("Weiss Lab MIT iGEM Recombinase 6/30/16" for example)
Put the bag in the Genewiz pickup box (located by the elevators, make sure you put in the genewiz box)
Open up the weiss lab orders spreadsheet (bookmarked on the computer). You need to request access to the spreadsheet if it is your first time ordering. Once you request access, email Brian so he knows that you have sent in a request.
Click the tab at the bottom called "genewiz".
Fill out the form. Make sure you indicate that the order is from iGEM so the right account can be charged.
You did it! YAY! Genewiz should get back to you pretty quickly, generally between 12-24 hrs.
DB 7/13/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Damon Berman
Date: 2016-07-13
Wednesday, 7/13
Things we still need to build
PCR:
Q4 - EYFP - Q3
Golden Gate:
pENTR L1 - TP901 - L2
pENTR L1 - attB - [upside-down]EYFP - attP - L2
pENTR L1 - L7Ae - SMASh - L2
pENTR L1 - 2x k-turn - EYFP - L2
pENTR L1 - 2x k-turn - SCre - L2
pENTR L1 - 2x k-turn - FLPO - L2
pENTR L1 - 2x k-turn - TP901 - L2
LR Cloning:
EGSH: SCre
EGSH: FLPO
hEF1a: EYFP
hEF1a: mKate
hEF1a - SloxP - SV40 - SloxP: EYFP
hEF1a - FRT - SV40 - FRT: EYFP
EGSH: TP901
hEF1a: L7Ae - SMASh
EGSH: 2x k-turn - EYFP
EGSH: 2x k-turn - SCre
EGSH: 2x k-turn - FLPO
EGSH: 2x k-turn - TP901
hEF1a: attB - [upside-down]EYFP - attP
CF 7/14/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Maya Kaul
Date: 2016-07-14
Thursday, 7/14
LR Reaction
Introduction
An LR reaction inserts one or more parts in pENTR vectors into a pDEST vector.  Used to assemble transcriptional units from promoters and genes.
Materials
Promoter pENTR plasmid: L4-Promoter-R1
Working concentration: 5 fmol/ul
Gene pENTR plasmid: L1-Gene-L2
Working concentration: 5 fmol/ul
Destination plasmid: pDEST
Working concentration: 10 fmol/ul
Nuclease-free TE
200 µl PCR strip tubes, 1 tube per rxn
5x LR Clonase II
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
Proteinase K
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
Procedure
LR Reaction Setup
For each LR you are doing, fill out a column in the following table:
A
B
C
D
E
F
1
Tube Label
1
2
3
4
2
Promoter pENTR
hEF1a
hEF1a
EGSH
EGSH
3
Gene pENTR
eYFP
mKate
SCre
FLPO
4
pDEST
gtw6
gtw6
gtw6
gtw6
5
6
Table1
For each LR, label a 200 µl strip tube with your initials and tube number.
Into each tube, pipette:
-- 1 µl of the promoter pENTR
-- 1 µl of the gene pENTR
-- 1 µl of the pDEST
Add 1 µl of TE to each tube
Retrieve an aliquot of LR Clonase from the -80.
Bring an razor blade with you, you'll need to cut a tube from the strip tubes.
Pulse the LR clonase tube in the microfuge to collect the clonase at the bottom.
Add 1 µl of the LR clonase to each LR reaction.
Be careful pipetting; LR clonase is viscous.
Cap the tubes.
Flick them several times to mix.
Pulse-spin the tubes in the microfuge to collect the liquid at the bottom.
Incubate at room temperature for at least 12 hours and not more than 24 hours.
A popular strategy is to tape the tubes to the shelves over the bench, with your initials and the date.
16-24 hours later: Proteinase K kill
Retrieve a 5 µl aliquot of proteinase K from the -80 freezer.m n
Thaw in your fingers, then pulse in the microfuge to collect at the bottom of the tube.
Pipette 1 ul into each of the LR reactions.
Flick several times to mix.
Pulse-spin the tubes in the microcentrifuge.
Incubate at 37° for 15 minutes, or room-temperature for an hour.
PAUSE POINT: You can store the reactions in the -20 indefinitely until the transformation.
00:15:00
Proceed to transformation.  Transform 2 µl.
Afterwards, cap the tubes. Write the date on the caps and store in the -20 (in case your transformation failed.)
JG 7/15/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Maya Kaul
Date: 2016-07-15
Friday, 7/15
Miniprep Nanodrop Results:
A
B
C
D
E
1
1
2
3
4
2
1: SloxP-Sv40
452.1
420
530.5
393.1
3
2: SloxP-Sv40
455.5
436.9
388.5
396.5
4
3: FRT-Sv40
427.3
473.6
497.1
476.7
5
4: Sv40
166.2
482.2
395.9
457.4
Table1
clipboard_2016-07-15_14:55:25.png
Gel:
L, 1-1, 1-2, 1-3, 1-4, 2-1, 2-2, 2-3, 2-4, L, 3-1, 3-2, 3-3, 3-4, 4-1, 4-2, 4-3, 4-4, B, L
1234BBRecombinase.jpg
LR Reaction
Introduction
An LR reaction inserts one or more parts in pENTR vectors into a pDEST vector.  Used to assemble transcriptional units from promoters and genes.
Materials
Promoter pENTR plasmid: L4-Promoter-R1
Working concentration: 5 fmol/ul
Gene pENTR plasmid: L1-Gene-L2
Working concentration: 5 fmol/ul
Destination plasmid: pDEST
Working concentration: 10 fmol/ul
Nuclease-free TE
200 µl PCR strip tubes, 1 tube per rxn
5x LR Clonase II
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
Proteinase K
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
Procedure
LR Reaction Setup
For each LR you are doing, fill out a column in the following table:
A
B
C
D
E
F
1
Tube Label
2
Promoter pENTR
3
Gene pENTR
4
pDEST
5
6
Table1
For each LR, label a 200 µl strip tube with your initials and tube number.
Into each tube, pipette:
-- 1 µl of the promoter pENTR
-- 1 µl of the gene pENTR
-- 1 µl of the pDEST
Add 1 µl of TE to each tube
Retrieve an aliquot of LR Clonase from the -80.
Bring an razor blade with you, you'll need to cut a tube from the strip tubes.
Pulse the LR clonase tube in the microfuge to collect the clonase at the bottom.
Add 1 µl of the LR clonase to each LR reaction.
Be careful pipetting; LR clonase is viscous.
Cap the tubes.
Flick them several times to mix.
Pulse-spin the tubes in the microfuge to collect the liquid at the bottom.
Incubate at room temperature for at least 12 hours and not more than 24 hours.
A popular strategy is to tape the tubes to the shelves over the bench, with your initials and the date.
16-24 hours later: Proteinase K kill
Retrieve a 5 µl aliquot of proteinase K from the -80 freezer.m n
Thaw in your fingers, then pulse in the microfuge to collect at the bottom of the tube.
Pipette 1 ul into each of the LR reactions.
Flick several times to mix.
Pulse-spin the tubes in the microcentrifuge.
Incubate at 37° for 15 minutes, or room-temperature for an hour.
PAUSE POINT: You can store the reactions in the -20 indefinitely until the transformation.
00:15:00
Proceed to transformation.  Transform 2 µl.
Afterwards, cap the tubes. Write the date on the caps and store in the -20 (in case your transformation failed.)
Miniprep
Introduction
The miniprep uses silica gel to isolate plasmid DNA from an E. coli culture
Materials
Buffer P1 (resuspension buffer)
Retrieve from refrigerator.  If you are opening a new miniprep kit, add the RNAse and LyseBlue reagent and check the box on the cap.
Buffer P2 (lysis buffer)
Open the cap and look at the lysis buffer.  Swirl it around.  If it appears cloudy, the SDS has fallen out of solution; warm it for a few minutes in the 55°C water bath.
Buffer N3 (neutralization buffer)
Buffer PB (binding buffer)
Buffer PE (rinse buffer)
Make sure the "Ethanol added?" box has been checked.  If you are opening a new miniprep kit, add absolute ethanol as per the kit instructions and check the box on the cap.
Buffer EB (elution buffer)
Miniprep waste container
Miniprep buffers contain salts that can't go down the sink.
Per miniprep: two microcentrifuge tubes and one blue spin column, with collection tube.
Procedure
Harvest and resuspension
For each culture, label two microcentrifuge tubes on the cap and one blue spin column on the side.
The spin columns should be in their (cap-less) collection vials.
Pipette 1.6 mL of each culture into the corresponding microcentrifuge tubes.
(This is for a 1.7 ml microcentrifuge tube; set your pipettor to 800 ul and do two transfers.  If we ever go back to 2-ml tubes, do 1.9 ml (2x950 ul))
Centrifuge at maximum speed (10,000 or 13,000xg) for three minutes.
Aspirate the supernatant, or pour it off into the bleach bucket.
Pipette ANOTHER 1.8 ml of each culture into the corresponding microcentrifuge tubes.
Centrifuge at maximum speed for three minutes.
While the centrifuge is running, move the remaining cultures to 4degC.
Aspirate the supernatant off with the bench aspirator.  Be careful not to disturb the pelleted E. coli.
We use an aspirator here because the less extra salt and protein we put in the miniprep, the better the yield is.
Add 250 µl Buffer P1 to each tube.
Resuspend the E. coli pellet.  The preferred way is with the roto-mixer at the other end of the lab.
Alternately, if you have just a few tubes, you can resuspend on a vortex.
Make sure to resuspend fully and thoroughly. The resulting suspension should be smooth and cloudy; if there is particulate matter floating around, vortex some more.
Lysis
Add 250 µl Buffer P2 to each tube.
Work quickly; the lysis step should take less than 5 minutes.
Snap the tubes closed and invert them 4-6 times, until the tube is thoroughly mixed and the entire solution turns blue.
If you have many many tubes, you can stack a second tube rack on top of them and invert the entire thing.
Add 350 ul Buffer N3 to each tube.
Snap the tubes closed and invert 4-6 times, until the solution is thouroughly mixed and no longer blue.
The solution will become cloudy or flocculent.
Centrifuge on high speed for 10 minutes.
Separation
Remove the tubes from the microcentrifuge, being careful not to disturb the white pellet.
Using P-1000 micropipettor set to 850 ul, carefully transfer the supernatant from each centrifuge tube to the corresponding blue spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Don't forget to put the lid on the rotor! Some of the salts get aerosolized because the spin columns don't have caps.
Pour the flow-through from each column into the miniprep waste container.
Pipette 500 ul of Buffer PB onto each spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Pipette 750 ul of Buffer PE onto each spin column
Wait 1-3 minutes.
This allows some of the salt that's still bound to the silica matrix to resuspend in the buffer.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Return each spin column to its collection tube and centrifuge an additional 1 minute at high speed.
This removes every last trace of buffer PE; the ethanol can screw up downstream steps.
Transfer each spin column to a clean labelled microcentrifuge tube.
Pipette 50 ul of Buffer EB onto the center of each column.
The volume of EB is comparable to the volume of silica gel matrix; if you pipette down the side, you might not get the entire transfer to the matrix.
Wait 1-3 minutes.
This gives the DNA a chance to dissociate from the silica matrix.
Centrifuge the spin columns, in their collection tubes, for one minute at maximum speed.
Proceed directly to analyze the samples on the Nanodrop.
Transformation of E. coli
Introduction
Transformation is the process of inducing chemically competent E. coli to take up DNA.
Materials
Dry bath, set to 42°C
Fill the wells in the dry bath block 1/2 full with DI water.
Ice bucket, with ice
For thawing competent cells.
DNA to transform
Could be an assembly reaction (LR, Golden Gate, etc) or a miniprepped plasmid.
If you removed it from the freezer, make sure it's entirely thawed  out.
pUC19 Transformation Control, 1 pg/µl
The pUC19 control will tell you how efficient your transformations were.
SOC growth media, at room temperature
Check to make sure it's clear and NOT CLOUDY.
Antibiotic plates, one per transformation, plus 1 Amp plate for the pUC19 control
Make sure the plates you use match the resistance cassette of the plasmid!
Competent E. coli, one tube per transformation + one for the pUC19 control
These live in the -80 in 235.
Thaw on ice 3-4 minutes.
A timer, set for 30 seconds.
Procedure
Setup
Make sure the dry bath is set to 42°C and the wells in the block are 1/2 full of DI water
Remove selection plates from the refrigerator.  Double-check that they match the selection marker on your plasmid, then place them in the 37° incubator.
Retrieve the DNA to transform.
If frozen: thaw, completely, flick a few times to mix, then pulse down in the microfuge.
Fill an ice bucket with ice.  Retrieve one tube of competent E. coli per transformation from the -80 and thaw on ice, 3-4 minutes.
00:03:00
While the transformation tubes are thawing, label their tops with something descriptive.  Record the labels here:
A
B
C
D
E
F
G
H
1
puc-19L
2
puc-19TC
3
Pdest_mCherry
4
pDest
Table1
Transformation
Add 2 µl DNA from each reaction to a tube of competent cells.
Immediately after adding the DNA to each tube, stir the cells a few times with the pipette tip.
Add 1 µl of the pUC19 transformation control to the positive control tube.
Incubate on ice for 30 minutes.
00:30:00
Heat shock the cells for exactly 30 seconds in the 42° heat block. (Yes, set a timer.)
00:00:30
Place back on ice for 2 minutes.
00:02:00
Add 250 µl SOC to each tube.
Tape the tubes to the platform of a shaker at 37°C and shake at 270 RPM for 60 minutes.
01:00:00
Plating
Label the selection plates using the labels you recorded above.
Shake ~10 plating beads onto each plate.
Pipette 100 µl of each transformation onto the corresponding plates. NOTE: if you are using DNA from a golden gate reaction, see the golden gate protocol for instructions to dilute your sample.
-----Golden Gate: Plate only 10 ul of the outgrowth in a 200 ul puddle of water, or you will get a lawn of colonies.
Cover the plates and shake the beads around to spread the cells out.
Dispose of the beads by tapping them into the waste container.
Incubate the plates upside down overnight in the 37° incubator.
Don't incubate for more than 18-24 hours.
Compute transformation efficiency
Count the colonies on your positive transformation plate.
If there are many many colonies, then hooray!  You had a great transformation.  Just estimate.
Divide the number of colonies by the fraction of the transformation you plated.
So, if you resuspended your transformation in a total volume of 300 ul, then plated 100 ul, multiply the number of colonies by 3.
Transformation efficiency is expressed in colonies per microgram pUC19.  Multiply the number of colonies by the appropriate conversion factor.
So if you transformed 1 picogram of pUC19 DNA, multiply by 106.
Record your transformation efficiency in your (daily) lab notebook.
Diagnostic Restriction Digestion
Introduction
A diagnostic restriction digest helps identify correctly assembled clones from incorrect clones.
Materials
200 ul PCR strip tubes, one per reaction
Restriction enzyme (chosen below), 1 ul per reaction.
10X restriction enzyme buffer (chosen below), 1 ul per reaction.
Procedure
Choose A Good Restriction Enzyme
Using Benchling, choose a restriction enzyme that meets the following criteria:
- Cuts at least once in the insert (for a pEXPR, either the promoter or the gene.)
- Cuts at least once in the backbone.
- Gives bands that aren't too large (> 8 kb) or too small (< 200 bp).
- Gives a band pattern that is significantly different from the expected error mode.
* For LR reactions, the most common error is a pDEST that slipped through the selection.
- Begin in the "Brian's Favorites" list, then expand to the main Weiss lab list.
- If you can't find a single enzyme that gives an acceptable band pattern, choose two enzymes that give a acceptable band pattern when used together.
* This double-digest is subject to buffer compatibility, outlined below.
Record your enzyme choice on the plasmid's Description page.
Benchling will tell you the enzyme's buffer compatibility and active temperature. Record the buffer in which the enzyme is most active.
If there are multiple bufers in which the enzyme is equally active, choose in this order: Buffer 3.1, Buffer CS, Buffer 2.1, Buffer 1.1.
Set up the restriction digest
Retrieve the minipreps and the appropriate 10X buffer concentrate from the freezer.  Thaw on the benchtop or in your fingers.
Label the PCR tubes with your initials and an incrementing number.
ie: BT-1, -2, -3, -4
Vortex the minipreps and the 10X buffer concentrate briefly, then pulse down in the microfuge.
For each miniprep, set up a PCR tube containing the following in order:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
Remove the enzyme from the freezer for as little time as possible.
I have specified an "arbitrary" 3 µl volume of miniprep DNA; this should be fine as long as your miniprep concentration is >= 100 ng/ul.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
Incubate at the appropriate temperature for at least 1 hour and no more than 16 hours.
If the enzyme's active temperature is 37°C, use the 37°C plate incubator.
Stop the reaction by adding 2 ul of 6X NEB purple gel loading dye to each reacti on.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
PAUSE POINT: The reaction can be stored almost indefinitely at room temperature once it's been stopped.
Proceed to gel electrophoresis.
Gel electrophoresis
Introduction
Gel electrophoresis separates pieces of DNA by length.
Materials
Agarose gel, 1 lane per sample, plus at least 1 lane for the ladder.
If you just poured the gel, make sure it has had at least 30 minutes to set.
If you have a large gel but only need a few lanes, cut out just a piece of it with a razor blade.  Cut straight!
Samples to run
6X NEB Purple Loading Dye
Molecular weight standard (aka "ladder")
Commonly available ladders are Hyperladder I and NEB 2-Log.
Gel box, lid, leads
Electrophoresis power supply
1X TAE buffer, enough to fill the gel box.
Procedure
Prepare your samples
If your samples are frozen, thaw them completely, flick or vortex to mix, then pulse down in the microfuge.
If your samples are not already in loading dye, mix them 1:6 with 6X NEB Purple Loading Dye
Add 1 µl of loading dye for every 5 µl of sample.
Pro tip: If you don't want to mix loading gel with your entire sample (PCR products, for example), cut off a strip of Parafilm; pipette 2-3 ul of sample onto the parafilm; add 1 ul loading dye; mix by pipetting and load directly.
Set up the gel box
Program the voltage on the power supply.  For the small gel box, use 100V; for the large gel box, use 150V.
Program the time on the power supply.  For a small analytical gel, set the timer for 30 minutes.
For larger gels, start at 45 minutes, and then check regularly!
Attach the leads to the gel box cover.  Make sure that the red (positive) lead is attached to the side of the box farthest away from the wells in the gel.
Remember, the DNA is negatively charged and will move toward the positive terminal. A useful neumonic for remembering this is "Run to Red."
If the gel box has not been used previously that day, empty it and rinse it out with DI water.
The TAE buffer can be re-used, but if it's been sitting out for too long it evaporates and the salt concentration (and conductivity) changes.  TAE is cheap; when in doubt, replace it.
Place the gel in the gel box.
Pour TAE into the gel box until it just barely covers the gel.
The TAE's purpose is to conduct electricity; over-filling the gel box results in a larger conductive path, more current flow, and more heating (which can screw up your run.)
Check the wells to see that they are free of bubbles.  If there are bubbles, blow them out by pipetting 100 µl of TAE from the gel box into the well.
Load your samples
Load your samples. For large combs, load 10 µl; for small combs, load 5 µl.
Work quickly. The samples begin to diffuse in the buffer, leading to smeared bands.
Load the ladder in the last lane.  Load 1/2 the volume of your samples: for large combs, load 5 µl, for small combs, load 2.5 µl.
Place the lid on the gel box. Make sure it is seated on the brass contacts.
Run the gel
Press the Start or Run button on the power supply.
Make sure the power supply doesn't complain about an open circuit.  If it does, re-seat the gel box lid and press Run again.
Double-check that there are bubbles forming on the platinum wires at either end of the gel box.
Double-check that the red (positive) lead is on the side farthest from the wells.  Remember, Run to Red.
Run until the pink band is 2/3 to 3/4 of the way down the gel.  For small analytical gels, this should take 30 minutes.  For larger gels, start at 45 minutes and check regularly!
Image the gel
If the power supply is still running, press the Stop button.
Lift the lid off of the gel box. Lift the lid straight up. If you try to "hinge" it up, the lid will break.
Transfer the gel to the GelDoc.
If it's not running, start QuantityOne from the toolbar.
If necessary, click the top button on the toolbox to select the scanner.
If necessary, reset the camera (as per instructions on the GelDoc computer.)
Press the Epifluorescent Illumination button on the GelDoc.  Check the gel's position, zoom and focus.
You want to be zoomed in so that the gel fills the field of view, and focussed so that the well edges are sharp.
Close the GelDoc door. Click Auto Expose.
The Auto Expose functionality generally over-exposes my gels a bit.  If your gel is over-exposed, remove 1/3 of the exposure time, type it into the Exposure box, then click Manually Expose.
In the File menu, select, Export as JPEG.... Save your gel to the iGEM folder on the desktop.
Open Benchling.  Log in, and copy the gel to the Description page for the plasmid you're building.
If there are multiple plasmids on the gel, save it to each plasmid's Description.
Discard the gel in the biowaste box. Wipe down the gel doc with a little water and a paper towel or Kimwipe.
Annotate the gel
Immediately, before you forget what's where, annotate the gel.
List what is in each lane.
Describe whether the pattern is what you expected or not. (You should have an in silico digestion to compare it to!)
Resuspending Primers
Introduction
Primers arrive from IDT dehydrated; to use them, you need to resuspend them in TE to a known concentration.  By convention, the stock concentration (in the blue-capped IDT tube) is 100 µM; the working concentration depends on the application:  - For PCR, the working concentration is usually 10 µM  - For sequencing, the working concentration is usually 5 µM
Materials
Nuclease-free TE
We resuspend in TE to help supress nuclease activity that would degrade the primers.
Dehydrated primers from IDT
One extra epi tube per primer
...for the working stock.
Procedure
Resuspend the dried primer
Label the top of the oligo tubes.  I recommend the group initials and a number.
In the little microfuge, spin the (dry) oligos briefly.
Sometimes the freeze-dried primers flake off the bottom of the tube.
Determine how many nanomoles of primer are in the tube.
This is the bottom line on the tube label.  As an example, the BT-01 primer says
7.3OD = 33.4 nmol = 0.23 mg. <-- the number of nanomoles is "33.4".
Add 10 µl of nuclease-free TE for each nanomole of primer.
For the BT-01 primer, this volume is 334 µl.
Vortex briefly.
Check the bottom of the tube to see if the primer is fully resuspended.  If not, vortex again.
Pulse spin the primers.
Make a working stock
Label the top of the epi tube the same as the oligo.
Determine the working stock concentration that you want.  Label the side of the epi tube with the working concentration.
Make 100 µl of working stock.
For example, if you're making a 10 µM working stock, dilute 10 µl of the concentrated stock into 90 µl of TE.
Vortex briefly.
Freeze both the concentrated and working stocks in the primers box.
TN 7/17/16 Overnight Culture
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Trinh Nguyen
Date: 2016-07-18
Monday, 7/18
All plates have white colonies
However, the plates were mislabel
Overnight liquid cultures (picking colonies)
Introduction
Overnight cultures are used to prepare miniprep DNA.
Materials
The plate from which you are picking colonies
15 ml round-bottom polystyrene tubes, one per culture
The ones with the snap caps, NOT conical tubes with screw caps
5 mL LB per culture
A container that can hold 5ml x the number of cultures
For a modest number of minipreps, a 50 ml conical tube works well.
For larger minipreps, use a sterile bottle (100 ml is frequently useful.)
Antibiotic stock, 1000X
Procedure
Materials Setup
Warm up the LB to at least room temperature (if it came from the fridge), but not warmer than 37°C
Label one round-bottom culture tube for each miniprep.  Use "NAME-1, NAME-2, ..."etc for the naming convention, where NAME is a shortened name of the plasmid (eg, "hEF1a:mKate").
Your impulse is to just use number, or initials and number, but trust me -- you will want to be able to identify this tube in three weeks when you've forgotten what you were doing.
Using a sterile pipette, transfer 5 ml of LB to the mixing container for each culture PLUS 5 ML.
Add antibiotic stock to a final concentration of 1X (1 µl stock for each 1 ml in the mixing container.)
Cap tightly and mix well.
Culture Setup
Using a sterile pipette, transfer 5 ml of LB+antibiotic to each round-bottom culture tube.
If you are making cultures with different antibiotics, take care that the right media goes in each tube.
Squirt ethanol on a pair of foreceps and wipe dry with a Kimwipe.
Use the foreceps to pick up a sterile 200µl pipette tip, scrape a colony off of the plate, and drop the pipette tip in the corresponding tube.
Repeat for each tube.
Transfer to an incubating shaker at 37°C and incubate 14-16 hours.
Don't over-grow too badly, or your yield will suffer.
If you need to grow longer, you can grow at 30°C instead for 20 hours.
