Team:MIT/miRNA Group Notebook pDEST mCherry
BACKGROUND
EXPERIMENTS
Synthetic Promoters microRNA Recombinase Proof of Concept Collaborations
FUTURE WORK
Circuit Design Clinical Applications
HUMAN PRACTICES
Parts Toolbox Accessible Protocols Integrated Human Practices Public Outreach Interlab Study
NOTEBOOK
Lab Notebook Parts Mammalian Parts Collection Attributions Safety
AWARDS
HP Silver HP Gold Integrated HP Basic Part
TEAM
PCR diary for pDEST mCherry with GG cassettes 2.0 · Benchling
PCR diary for pDEST mCherry with GG cassettes 2.0
Made with Benchling
Project: miRNA and Repressors subgroup
Authors: Elizabeth Strand
Dates: 2016-08-24 to 2016-10-09
Wednesday, 8/24
Starting Over, Primers
We realized the golden gate wouldn't work so we got new primers
A
B
C
1
08/12/16 Sent to Brian to order
2
#
Name
Primer Sequence (5'-3') - capitalize flap
3
55
iGEM 55 DONR-F-2.0
CCCGTGACTAGTGCTTATGTCTTCaattcacctgctgacaggttgag
4
56
iGEM 56 DONR-R-2.0
CTCGGTATCGATGTTGATGTCTTCcacctgcactgaagctgag
Table1
\
Primers only designed for restriction digest + ligation
PCR
Redoing for all --  even though DC RE should still be fine..
A
B
C
D
1
Nickname
Template
Forward Primer
Reverse Primer
2
DC
pDest_mCherry
iGEM 17 pDEST_mCherry Restriction Forward
iGEM 18 pDEST_mCherry Restriction Reverse
3
GG
pDONR_GG (L4, R1)
iGEM 55 DONR-F-2.0
iGEM 56 DONR-F-2.0
Table20
Prepare on Ice in this order
A
B
1
Volume per rxn
2
Ultrapure H2O
20
3
Template (dil to 2 ng/uL)
4
Fwd Primer (10 uM)
2.5
5
Rev Primer (10 uM)
2.5
6
Q5 Master Mix (2X)
25
7
Total
30
8
Table2
Protocol: Q5 repressors b
A
B
C
D
1
Step
Temperature (C)
Time (s)
Sample
2
Initial Denaturation
98
30
3
Cycle 30 x
4
Denature
98
10
5
Anneal
68
30
DC RE
6
67
30
GG RE
7
Extend
72
90
8
Cycle Ends
9
Final Extension
72
120
10
Store
4
forever
Table21
Thursday, 8/25
Gel Extraction
https://www.neb.com/products/n3200-2-log-dna-ladder-01-100-kb
A
B
C
1
Lane
Nickname
Expected Length
2
1
NEB Ladder 1 kb
--
3
2
DC
3836bp
4
3
DC
3836bp
5
4
skip
6
5
NEB Ladder 1 kb
--
7
6
GG RE
751bp
8
7
GG RE
751bp
Table22
Used wrong primers so i didn't get any DC :P can try to move forward with DC RE from before (should be fine).
A
B
C
D
E
F
1
Tube
Weight (mg)
QG Added (uL)
Isopropanol Added (uL)
Nanodrop (ng/uL)
2
DC RE
286
1716
286
32.9
Done 7/28, in diary 1.0
3
GG 1
128
768
128
4
GG2
143
858
143
Table23
Digestion (10 uL)
A
B
1
Name on Tube
Concentration (ng/uL)
2
DC RE
32.9
3
GG RE 1
53.1
Table24
A
B
C
D
E
1
"D+G RE"
Neg
2
DC RE DNA (100 ng)
3 uL
3
3
GG RE DNA (100 ng)
2 uL
0
4
10X Cut Smart
1 uL
1
5
SpeI
1 uL
1
6
ClaI
1 uL
1
7
Nuclease free water
2 uL
4
8
total volume
10 uL
10 uL
Table25
Digested for 1.5 hours at 37C
Heat inactivated for 40 mins at 80C (should have only been 20 mins.. ops)
Friday, 8/26
Ligation
Ligating: D+G as well as negative control (D alone)
A
B
C
1
"xp"
"t4"
2
Buffer
1 uL
1 uL
3
DNA
4 uL mixture
4 uL mixture
4
Insert
N/A
N/A
5
H2O
4 uL
4 uL
6
Ligase
1 uL express
1 uL t4
Table27
Gently mix the reaction by pipetting up and down, and microfuge briefly.
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
Chill on ice and transform 1-5 µl of the reaction into 50 µl competent cells.
Transformation
A
B
C
D
1
Label
What's in it
Amount (uL)
2
D+G XP
pDest_mCherry + GG with XP ligase
5
3
Neg XP
Dest alone with T4 ligase
5
4
D+G T4
pDest_mCherry + GG
5
Saving these
5
Neg T4
Dest alone
5
6
pUC 19 (1 pg/ul)
control
1
7
8
9
Table3
Plating with beads, 5 ul outgrowth + 200 ul water for some or 100 ul of outgrowth (plate labelled)
Saturday, 8/27
20160827_133826.jpg
20160827_133823.jpg
20160827_133817.jpg
Sunday, 8/28
Picked overnights and cultured in 4-5 ml of LB amp
Picked:
3 blue colonies
1 that looked mixed / both colors involved
1 red
Tuesday, 8/30
Overnights looked like this the next day:
20160828_095813.jpg
20160828_095849.jpg
20160828_095922.jpg
Pellets looked like this:
20160828_102316.jpg
20160828_102355.jpg
Miniprep Results
Lost Blue 1 :( RIP
A
B
1
Name on Tube
Concentration (ng/uL)
2
blue 2
459.7
3
blue 3
370.2
4
both
279.1
5
red
291.6
Table4
Confirmation Digest
Digest with BamHI, NcoI
A
B
C
D
E
F
G
H
1
Uncut (B2)
bamHI (B2)
ncoI  (B2)
B2
B3
Both
Red
2
DNA (~120 ng/ul)
3
3
3
3
3
3
3
3
10x NEB 3.1
1
1
1
1
1
1
1
4
BamHI
0
1
0
1
1
1
1
5
NcoI
0
0
1
1
1
1
1
6
Nuclease free water
6
5
5
4
4
4
4
7
total volume
10
10
10
10
10
10
10
Table5
Digested for 1 hours at 37C
Not enough NcoI to do all digests
Confirmation Gel
repmcherrygggel.JPG
weisslab 2016-08-28 14hr 26min.jpg
B2 and B3 look good!
Sunday, 10/9
Diagnostic Digest
Confirming:
Insertion of promoter/gene
presence of LacZ
CF 10/6 LR Code:
1: pBM3R1: 2kturn - TP901
2: pERE3: TP901
3: pERE5: TP901
4: pERE3: mKate
5: pERE5: mKate
6: pHybrid: BM3R1
(Used the 1 version for everything)
ONLY used "1" version for all
2: pERE3: TP901   3: pERE5: TP901
A
B
C
D
E
F
G
H
I
1
2 Uncut
2 Single (2B)
2 Single (2N)
2 Double
3 Double
2
DNA (~100 ng/ul)
3
3
3
3
3
10X Cut Smart
1
1
1
1
1
4
BamHI
0
0
1
1
1
5
NcoI
0
1
0
1
1
6
Nuclease free water
6
5
8
4
4
7
total volume
10
10
10
10
10
Table6
4: pERE3:mKate   5: pERE5:mKate
A
B
C
D
E
F
G
H
I
1
4 Uncut
4 Single
4 Single
4 Double
5 Double
2
DNA (~100 ng/ul)
3
3
3
3
3
3
10X Cut Smart
1
1
1
1
1
4
StuI
0
0
1
1
1
5
NcoI
0
1
0
1
1
6
Nuclease free water
6
5
5
4
4
7
total volume
10
10
10
10
10
Table7
Gel
Ran in the above order
clipboard_2016-10-09_15:33:26.png
weisslab 2016-10-09 15hr 46min.jpg
ERE5-TP901-GG-1 (3-1) andERE5-mKate-GG-1 (5-1) seem good
will attempt other golden gates with ERE3-TP901-GG-2 (2-2) and ERE-3-mKate-GG-2 (2-4)
PCR Diary for pDEST_mCherry and GG cassettes (06/27/16 - 8/24) · Benchling
PCR Diary for pDEST_mCherry and GG cassettes (06/27/16 - 8/24)
Made with Benchling
Project: miRNA and Repressors subgroup
Authors: Elizabeth Strand
Dates: 2016-06-27 to 2016-08-10
Monday, 6/27
Ran 4 PCR reactions
Set Up:
A
B
C
D
1
Nickname
Template
Forward Primer
Reverse Primer
2
DEST
pDEST-T
iGEM 09 pDEST-T R4_GTW6_R2 Forward
iGEM 10 pDEST-T R4_GTW6_R2 Reverse
3
RED
pL2f1 091
iGEM 11 pLacIq_mCherry Forward
iGEM 12 pLacIq_mCherry Reverse
4
GG cut
pDONR-GG L4_LacZa_L1
iGEM 19 pDONR_GG L4_LacZa_R1 Restriction Forward
iGEM 20 pDONR_GG L4_LacZa_R1 Restriction Forward
5
GG Gibson
pDONR-GG L4_LacZa_L1
iGEM 15 pDONR_GG L4_LacZa_R1 Gibson Forward
iGEM 16 pDONR_GG L4_LacZa_R1 Gibson Reverse
Table1
Protocol: "Q5 repressors1"
A
B
C
D
1
Step
Temperature (C)
Time (s)
Sample
2
Initial Denaturation
98
30
3
Cycle 30 x
4
Denature
98
10
5
Anneal
67
30
DEST
6
70
30
Red
7
72
30
GG cut and GG gibson
8
Extend
72
90
9
Cycle Ends
10
Final Extension
72
120
11
Store
4
forever
Table3
Tuesday, 6/28
Ran a Diagnostic Gel
A
B
C
1
Lane
Nickname
Expected Length
2
1
Bioline Ladder 1 kb
--
3
2
Bioline Ladder 1 kb
--
4
3
DEST
2972
5
4
RED
878
6
5
GG cut
738
7
6
GG Gibson
749
Table2
06-28-16 pcr products.jpg
Wednesday, 6/29
Checked concentration after PCR Purification
A
B
C
1
Name on Tube
Concentration (ng/uL)
2
DEST PCR
23.6
3
mCherry PCR (formerly known as RED)
28.3
4
GG cut PCR
13.1
5
GG gib PCR
23.9
Table4
Stored in -20C Repressor Box
Did Double Digest Protocol (In 10 uL total)
From NEB http://nebcloner.neb.com/#!/
Adjusting the NEB protocol per Brian's recommendations
A
B
C
1
DEST
mCherry
2
DNA (100 ng)
4 uL
3.5 uL
3
10X Cut Smart
1 uL
1 uL
4
AsiSI
1 uL
1 uL
5
XhoI
1 uL
1 uL
6
Nuclease free water
3 uL
3.5 uL
Table5
Positive Control for transformation:
A
B
1
pUC19 nondilute
2
DNA (100 ng)
2 uL
3
10X cut smart
1 uL
4
SphI
1 uL
5
Water to 10 uL
6 uL
6
Table6
Incubate all reactions at 37C per NEB protocol (1 hour)
Heat inactivated at 80C for 20 mins per NEB protocol.
Holding off on digesting GG cut and GG gibson until pDEST_mCherry is linearized
Stored in -20C Repressor Box
Ligation in 10 uL Reaction
Also used NEB T4 Ligase protocol for sticky ends + transformation into chemically competent cells. Changed amounts according to table below.
Want reaction to be 1:3 (backbone:insert).
Backbone = DEST
insert = mCherry
Note: T4 buffer is in aliquots in our -20C in a tip box. T4 Ligase is in enzyme box. Remove T4 Ligase only when ready to add to reaction
A
B
C
D
1
Tube: Dest +Cherry
Tube: DEST alone
Tube: PUC alone
2
Buffer
1 uL
1 uL
1 uL
3
Backbone
2 uL
2 uL
2 uL
4
Insert
2 uL
N/A
N/A
5
H2O
4 uL
6 uL
6 uL
6
T4 Ligase
1 uL
1 uL
1 uL
Table7
Stored in -20C Repressor Box
Transformation
Sample: DEST + mCherry
Negative Control: Ligation with backbone only (no insert)
Positive Control: pUC19 cut once and religated.
Positive Control: pUC19 never cut
Stored at 4C after incubation
Thursday, 6/30
Results of plating:
Controls fine, Sample only has one colony. Leaving sample at room temp to see if colony turns red.
Also retransforming ligation today.
Nothing turned red -->
Alot of plating and stuff happened that was kinda recorded in the experiment 1 folder Digest and Gel
Thursday, 7/7
REDO Double Digest Protocol (In 20 uL or 10 uL total)
From NEB http://nebcloner.neb.com/#!/
Brian suggested
A
B
1
One Pot
2
DEST DNA (100 ng)
4 uL
3
mCherry DNA (100 ng)
3.5 uL
4
10X Cut Smart
2 uL
5
AsiSI
1 uL
6
XhoI
1 uL
7
Nuclease free water
8.5 uL
Table8
A
B
C
1
DEST
mCherry
2
DNA (100 ng)
4 uL
3.5 uL
3
10X Cut Smart
1 uL
1 uL
4
AsiSI
1 uL
1 uL
5
XhoI
1 uL
1 uL
6
Nuclease free water
3 uL
3.5 uL
Table9
Incubate at 37C for 3 hours
Redo Ligation in 10 uL Reaction
Also used NEB T4 Ligase protocol for sticky ends + transformation into chemically competent cells. Changed amounts according to table below.
Want reaction to be 1:3 (backbone:insert).
Backbone = DEST
insert = mCherry
Note: T4 buffer is in aliquots in our -20C in a tip box. T4 Ligase is in enzyme box. Remove T4 Ligase only when ready to add to reaction
A
B
C
D
1
Tube: Dest + mCherry
Tube: DEST alone
Tube: One Pot
2
Buffer
1 uL
1 uL
1 uL
3
Backbone
2 uL
2 uL
4 uL mixture
4
Insert
2 uL
N/A
N/A
5
H2O
4 uL
6 uL
4 uL
6
T4 Ligase
1 uL
1 uL
1 uL
Table10
Gently mix the reaction by pipetting up and down, and microfuge briefly.
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
Chill on ice and transform 1-5 µl of the reaction into 50 µl competent cells.
Plating results: Very few colonies throughout, trashed
Friday, 7/8
Redid PCR reactions
per Brian's tube set up protocol
Set Up:
A
B
C
D
1
Nickname
Template
Forward Primer
Reverse Primer
2
DEST
pDEST-T
iGEM 09 pDEST-T R4_GTW6_R2 Forward
iGEM 10 pDEST-T R4_GTW6_R2 Reverse
3
RED
pL2f1 092
iGEM 11 pLacIq_mCherry Forward
iGEM 12 pLacIq_mCherry Reverse
4
GG cut
pDONR-GG L4_LacZa_L1
iGEM 19 pDONR_GG L4_LacZa_R1 Restriction Forward
iGEM 20 pDONR_GG L4_LacZa_R1 Restriction Forward
5
GG Gibson
pDONR-GG L4_LacZa_L1
iGEM 15 pDONR_GG L4_LacZa_R1 Gibson Forward
iGEM 16 pDONR_GG L4_LacZa_R1 Gibson Reverse
Table11
Protocol: "Q5 repressors1"
A
B
C
D
1
Step
Temperature (C)
Time (s)
Sample
2
Initial Denaturation
98
30
3
Cycle 30 x
4
Denature
98
10
5
Anneal
67
30
DEST
6
70
30
Red
7
72
30
GG cut and GG gibson
8
Extend
72
90
9
Cycle Ends
10
Final Extension
72
120
11
Store
4
forever
Table12
A
B
C
D
E
F
1
Top Lane
Nickname
Expected Length
Top Lane
Nickname
2
1
Bioline Ladder 1 kb
--
1
blank
3
2
1 kb Ladder
--
2
1 kb ladder
4
3
DEST
2972
3
Trinh 1
5
4
RED
878
4
Trinh 2
6
5
GG cut
738
7
6
GG Gibson
749
Table13
Protocol: Q5 repressors 1
Ran on gel, stored bands at 4C
Monday, 7/11
A
B
C
D
E
1
Tube
Weight (mg)
QG Added (uL)
Isopropanol Added (uL)
2
DEST
154
924
154
3
RED
160
960
160
4
GG gib
283
1698
283
5
GG cut
210
1260
210
Table14
REDO Double Digest Protocol (In 20 uL or 10 uL total)
From NEB http://nebcloner.neb.com/#!/
Brian suggested
A
B
C
1
Name on Tube
Concentration (ng/uL)
2
DEST PCR
10.8
3
mCherry PCR (formerly known as RED)
40.7
4
GG cut PCR
44
5
GG gib PCR
45.4
Table17
A
B
1
One Pot
2
DEST DNA (100 ng)
10 uL
3
mCherry DNA (100 ng)
2.5 uL
4
10X Cut Smart
2 uL
5
AsiSI
1 uL
6
XhoI
1 uL
7
Nuclease free water
3.5 uL
8
total volume
20 uL
Table15
A
B
C
1
DEST
mCherry
2
DNA (100 ng)
10 uL
2.5 uL
3
10X Cut Smart
1.5 uL
1 uL
4
AsiSI
1 uL
1 uL
5
XhoI
1 uL
1 uL
6
Nuclease free water
1.5 uL
4.5 uL
7
total volume
15 uL
10 uL
Table16
Incubated at 37C for 3 hours
Wednesday, 7/13
Redoing Ligation
blue = brian's ligation mixes
green = standard
D = pDest
DM = pDest mCherry (1 pot)
+ = pDest + mCherry
Redo Transformation
per Brian's benchling protocol --> with 5 uL of DNA added instead of 2 uL
Wednesday, 7/20
For the previous diagnostic gel digest (and sequencing) results of pDEST+mCherry, please refer to this note .
Second Diagnostic Gel Digest
Enzymes used for running the pDest+mCherry miniprep products on gel: Nco1 (single cut), Xho1 (single cut)
ALL MEASUREMENTS BELOW ARE ul.
A
B
C
D
E
F
G
H
I
J
K
L
M
N
O
P
Q
1
SAMPLE LABEL
LADDER
1
2
3
4
5
6
LADDER
7
8
9
10
11
12
LADDER
SAMPLE LABEL
2
PLASMID
pD+mC 1
pD+mC 1
pD+mC 1
pD+mC 1
pD+mC 2
pD+mC 3
pDmC 1
pDmC 1
pDmC 1
pDmC 1
pDmC 2
pDmC 3
PLASMID
3
H2O
6
5
5
4
4
4
6
5
5
4
4
4
H2O
4
BUFFER
1
1
1
1
1
1
1
1
1
1
1
1
BUFFER
5
DNA
3
3
3
3
3
3
3
3
3
3
3
3
DNA
6
ENZYME 1 (Nco1)
N/A
1 ul
N/A
1 ul
1 ul
1 ul
N/A
N/A
1 ul
1
1
1
(Nco1) ENZYME 1
7
ENZYME 2 (Xho1)
N/A
N/A
1 ul
1 ul
1 ul
1 ul
N/A
1 ul
N/A
1
1
1
(Xho1) ENZYME 2
8
WELL NUMBER
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
WELL NUMBER
Table18
why do we have 2 separate gels for the samples on the table above?
during the loading of the gel, due to a confusion we skipped 1 well between Sample 8 and Sample 9.
A
B
C
D
E
F
G
H
I
J
K
L
M
N
O
P
Q
1
well number
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
2
sample label (IDEAL)
LADDER
1
2
3
4
5
6
LADDER
7
8
9
10
11
12
LADDER
NO NEED
3
sample label (1st gel)
LADDER
1
2
3
4
5
6
LADDER+7
8
9
SKIP
9
10
11
12
LADDER
4
sample label (2nd gel)
SKIP
SKIP
LADDER
1
2
3
4
5
6
LADDER
7
8
10
11
LADDER
Table19
GEL 1
pdest-mcherry_16072_gel1_manualExposure.jpg
annotated_pdest-mcherry_160720_gel1_manualExposure.jpg
GEL 2
pdest-mcherry_16072_gel2_manualExposure2.jpg
annotated_pdest-mcherry_160720_gel2_manualExposure2.jpg
Monday, 7/25
Look at testing pDEST_mCherry to see optimal conditions for use.
Results: pDest_mCherry was successful in Tal21:EYFP LR reactions. Both the pDest + mCherry colony 1 and pDest_mCherry_one_pot colony 1 were used. Different of transformed cells were plated: 100uL of transformation, 20ul transformation/200uL water, and 10uL transformation/200 uL water.
The easiest plate to pick colonies was the 10uL/200uL plate using pDest + mCherry. Please see "Testing pDest mCherry (Making Tal21:EYFP)"
Thursday, 7/28
PCR Amplification
PCR for pDest_mCherry (previously referred to as pDest + mCherry colony 1 or 2) backbone, PCR Golden Gate Cassette to include new Q sites and either restricition enzyme sites or Gibson complementary sequence
A
B
C
D
1
Nickname
Template
Forward Primer
Reverse Primer
2
DC RE
pDest_mCherry
iGEM 17 pDEST_mCherry Restriction Forward
iGEM 18 pDEST_mCherry Restriction Reverse
3
DC Gib
pDest_mCherry
iGEM 13 pDEST_mCherry Gibson Forward
iGEM 14 pDEST_mCherry Gibson Reverse
4
GG RE
pDONR_GG
iGEM 39 pDONR-GG L4_LacZa_R1_RE_F
iGEM 40 pDONR-GG L4_LacZa_R1_RE_R
5
GG Gib
pDONR_GG
iGEM 37 pDONR-GG L4_LacZa_R1_Gib_F
iGEM 38 pDONR-GG L4_LacZa_R1_Gib_R
Table20
Prepare on Ice in this order
9uL Ultrapure Water
1uL Template DNA (Any Concentration less than 1 ug)-nanodrop verification.
1.25uL 10uM Forward Primer
1.25uL 10uM Reverse Primer
12.5uL 2X Q5 Master Mix
A
B
C
D
1
Step
Temperature (C)
Time (s)
Sample
2
Initial Denaturation
98
30
3
Cycle 30 x
4
Denature
98
10
5
Anneal
68
30
DC RE, DC Gib
6
67
30
GG RE, GG Gib
7
58
30
miR9, miR135
8
Extend
72
90
9
Cycle Ends
10
Final Extension
72
120
11
Store
4
forever
Table21
Gel Extraction
A
B
C
1
Lane
Nickname
Expected Length
2
1
3
2
Bioline Ladder 1 kb
--
4
3
Recombinse Group
--
5
4
DC RE
3836bp
6
5
DC Gib
3812bp
7
6
GG RE
751bp
8
7
GG Gib
774bp
Table22
A
B
C
D
E
1
Tube
Weight (mg)
QG Added (uL)
Isopropanol Added (uL)
Nanodrop (ng/uL)
2
DC RE
286
1716
286
32.9
3
DC Gib
239
1434
239
27.5
4
GG RE
329
1974
329
52.5
5
GG Gib
211
1266
211
40.5
Table23
Monday, 8/1
Digestion (10 uL rxn)
From NEB http://nebcloner.neb.com/#!/
Brian suggested
A
B
1
Name on Tube
Concentration (ng/uL)
2
DC RE
32.9
3
GG RE
52.5
Table24
A
B
1
"D+G RE"
2
DC RE DNA (100 ng)
3 uL
3
GG RE DNA (100 ng)
2 uL
4
10X Cut Smart
1 uL
5
SpeI
1 uL
6
ClaI
1 uL
7
Nuclease free water
2 uL
8
total volume
10 uL
Table25
Incubate at 37C for ~1.5 hr
A
B
C
1
"xp"
"t4"
2
Buffer
1 uL
1 uL
3
DNA
4 uL mixture
4 uL mixture
4
Insert
N/A
N/A
5
H2O
4 uL
4 uL
6
Ligase
1 uL express
1 uL t4
Table27
Gently mix the reaction by pipetting up and down, and microfuge briefly.
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
Chill on ice and transform 1-5 µl of the reaction into 50 µl competent cells.
Calculating for Gibson Assembly
Calculations
x = [(base pairs)(650)(0.2) ] /  [(1,000)(nanodrop count)].  If size is <200 bases, multiply x by 5 (see source beneath).
A
B
C
D
E
F
1
ng/uL
Dilution
base pairs
fmol in rxn
uL in rxn
2
DC gib
27.5
27.5
3812
40
3.60
3
GG gib
40.5
20
774
40
1.01
Table26
Up rxn to 40 uL???? --> Ask brian tomorrow -----> Don't
Tuesday, 8/2
Suggestions from Brian
To increase yield of gel extraction: up total pcr volume to 50 uL
Adjusting Gibson: his protol calls for 100-200 fmol. In reality, can scale reaction to as little as 7 fmol.
Try to keep Gibson total volume 10 ul, with up to 5 uL for DNA addition. keep DNA equimolar.
Gibson Assembly
A
B
C
1
"Gib"
"Ctrl"
2
Water
0.4
3
3
Master Mix
5
5
4
DC gib
3.6
1
5
GG gib
1
1
Table28
Tuesday, 8/9
Diagnostic Gel:
A
B
C
D
E
1
NEB quick load 2 log ladder
Nanodrop:
2
uncut
3
bamhI
4
NcoI
5
T4 A
6
T4 B
winner!!!!
146.4 ng/ul
7
XP A
8
XP B
9
Red A
10
Red B
Table29
repmcherrygggel.JPG
weisslab 2016-08-09 18hr 34min.jpg
Wednesday, 8/10
Golden Gate Assembly
To do::
Generate pDest-mCherry-GG construct on venchling
Secure bbsI enzyme
Testing pDEST mCherry (Making Tal21:EYFP) · Benchling
Testing pDEST mCherry (Making Tal21:EYFP)
Made with Benchling
Project: miRNA and Repressors subgroup
Authors: Elizabeth Strand
Dates: 2016-07-21 to 2016-07-26
Thursday, 7/21
All DEST's diluted to ~10 fmol/ul
A
B
C
D
E
1
Name on Tube
D
MD
+
2
DEST
"DEST-T"
"MD"
"+"
3
ENTR L4-R1
4xTal21
4xTal21
4xTal21
4
ENTR L1-L2
eyfp
eyfp
eyfp
5
Table1
Friday, 7/22
Transformed and plated.
Transformeed as usual, plated:
- 10 uL of transformation + 200 uL of water
- 20 uL of transformation + 200 uL of water
Monday, 7/25
Results of plating
20160723_154331.jpg
20160723_154342.jpg
20160723_154348.jpg
20160723_154352.jpg
20160723_154402.jpg
20160723_154413.jpg
20160723_154421.jpg
20160723_154430.jpg
20160723_154442.jpg
20160723_154450.jpg
Tuesday, 7/26
nothing
nothing
uncut
pst1
ecor1
ladder
pdest
a
b
nothing
nothing
nothing
weisslab 2016-07-25 18hr 48min.jpg
Testing pDEST_mChery_GG · Benchling
Testing pDEST_mChery_GG
Made with Benchling
Project: miRNA and Repressors subgroup
Authors: Elizabeth Strand
Date: 2016-09-23
Friday, 9/23
Suggested Approach
These are just some notes to help you get started.
*Our pDEST_mCherry_GG 2.0 has both an mCherry gene and a LacZ cassette. This means colonies may appear both red and blue. From my experience, the colonies containing mCherry AND LacZ look just straight up blue.
Thus, In order to test pDest_mCherry_GG, I would suggest the following steps
1.
Do an LR, plating on LB-amp plates (no xgal so colonies won't be blue), and then select for the non-red colonies FIRST. This will give you the colonies with the promoter and gene inserted that still have the LacZ cassette.
Destination: pDEST_mCherry_GG_2.0 ( Sequence )
Location: -20C, pink "iGEM Repressor + miRNA..." Box, in minipreps section. Label: DEST + GG B2 or B3 (choose either)
Promoter: hef1a
Location: -20C, working stocks box
Gene: choose between bfp, mkate, eyfp
Location: -20C, working stocks box
2.
Grow up overnight cultures then miniprep. Save a little of the culture (1-2ml for later)
3.
Design and run a gel digest & sequence to confirm gene and promoter have been inserted.
4.
Once confirmed, do golden gate with that tube. Follow protocol Golden Gate Protocol
5.
Mini and midiprep
6.
Design and run a restriction digest to ensure 451a sensor has been inserted. Could also try to reconfirm gene and promoter are also there.
Materials for Golden Gate
Follow materials listed in Golden Gate Protocol . These are just notes for a few of the materials/parts of the procedure.
●
"Backbone plasmid" --> Instead of using pDONR, we want to use miniprepped LR product that was created (as stated above)
Location: This is what you miniprepped and confirmed. You choose where to put it, but you should store it in our pink box in -20C, in miniprep section with distinct label.
●
Parts being assembled --> Choose the ultramer for 451 a . This is the only part necessary.
Location: -20C, "Glycerol + siRNA + ultramers box", Labelled 451a ultra (concentration is low, 17.2 ng/ul, you may need to calculate how much you need for golden gate early on and pcr amplify more if necessary.
**Don't leave this box out on the benchtop***
