Team:MIT/Recombinase Group Notebook Week 10
BACKGROUND
EXPERIMENTS
Synthetic Promoters microRNA Recombinase Proof of Concept Collaborations
FUTURE WORK
Circuit Design Clinical Applications
HUMAN PRACTICES
Parts Toolbox Accessible Protocols Integrated Human Practices Public Outreach Interlab Study
NOTEBOOK
Lab Notebook Parts Mammalian Parts Collection Attributions Safety
AWARDS
HP Silver HP Gold Integrated HP Basic Part
TEAM
CF 8/8/16 · Benchling
CF 8/8/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Elizabeth Strand
Date: 2016-08-08
Monday, 8/8
Transformation of E. coli
Introduction
Transformation is the process of inducing chemically competent E. coli to take up DNA.
Materials
Dry bath, set to 42°C
Fill the wells in the dry bath block 1/2 full with DI water.
Ice bucket, with ice
For thawing competent cells.
DNA to transform
Could be an assembly reaction (LR, Golden Gate, etc) or a miniprepped plasmid.
If you removed it from the freezer, make sure it's entirely thawed  out.
pUC19 Transformation Control, 1 pg/µl
The pUC19 control will tell you how efficient your transformations were.
SOC growth media, at room temperature
Check to make sure it's clear and NOT CLOUDY.
Antibiotic plates, one per transformation, plus 1 Amp plate for the pUC19 control
Make sure the plates you use match the resistance cassette of the plasmid!
Competent E. coli, one tube per transformation + one for the pUC19 control
These live in the -80 in 235.
Thaw on ice 3-4 minutes.
A timer, set for 30 seconds.
Procedure
Setup
Make sure the dry bath is set to 42°C and the wells in the block are 1/2 full of DI water
Remove selection plates from the refrigerator.  Double-check that they match the selection marker on your plasmid, then place them in the 37° incubator.
Retrieve the DNA to transform.
If frozen: thaw, completely, flick a few times to mix, then pulse down in the microfuge.
Fill an ice bucket with ice.  Retrieve one tube of competent E. coli per transformation from the -80 and thaw on ice, 3-4 minutes.
00:03:00
While the transformation tubes are thawing, label their tops with something descriptive.  Record the labels here:
A
B
C
D
E
F
G
H
1
puc-19L
2
puc-19TC
3
Pdest_mCherry
4
pDest
Table1
Transformation
Add 2 µl DNA from each reaction to a tube of competent cells.
Immediately after adding the DNA to each tube, stir the cells a few times with the pipette tip.
Add 1 µl of the pUC19 transformation control to the positive control tube.
Incubate on ice for 30 minutes.
00:30:00
Heat shock the cells for exactly 30 seconds in the 42° heat block. (Yes, set a timer.)
00:00:30
Place back on ice for 2 minutes.
00:02:00
Add 250 µl SOC to each tube.
Tape the tubes to the platform of a shaker at 37°C and shake at 270 RPM for 60 minutes.
01:00:00
Plating
Label the selection plates using the labels you recorded above.
Shake ~10 plating beads onto each plate.
Pipette 100 µl of each transformation onto the corresponding plates. NOTE: if you are using DNA from a golden gate reaction, see the golden gate protocol for instructions to dilute your sample.
-----Golden Gate: Plate only 10 ul of the outgrowth in a 200 ul puddle of water, or you will get a lawn of colonies.
Cover the plates and shake the beads around to spread the cells out.
Dispose of the beads by tapping them into the waste container.
Incubate the plates upside down overnight in the 37° incubator.
Don't incubate for more than 18-24 hours.
If LR was Done using pDEST_mCherry
When plating, use the plating wands to streak the plates. The plating wands are blue and should be found in the next to the PCR and eppendorf tubes in our bench.
Compute transformation efficiency
Count the colonies on your positive transformation plate.
If there are many many colonies, then hooray!  You had a great transformation.  Just estimate.
Divide the number of colonies by the fraction of the transformation you plated.
So, if you resuspended your transformation in a total volume of 300 ul, then plated 100 ul, multiply the number of colonies by 3.
Transformation efficiency is expressed in colonies per microgram pUC19.  Multiply the number of colonies by the appropriate conversion factor.
So if you transformed 1 picogram of pUC19 DNA, multiply by 106.
Record your transformation efficiency in your (daily) lab notebook.
CF 8/9/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Colleen Foley
Date: 2016-08-09
Tuesday, 8/9
Transfection experiment set up
A
B
C
D
1
Untransfected control
Single color (Y)       500ng hEF1a:eYFP  500ng pDONR
Single color (R)  500ng hEF1a:mKate  500ng pDONR
Single color (B) 500ng hEF1a:BFP  500ng pDONR
2
Three colors        300ng hEF1a:eYFP 300ng hEF1a:mKate 300ng hEF1a:BFP  100ng pDONR
300ng TRE:L7Ae    300ng TRE:mKate   100ng hEF1a:rTta     100ng hEF1a:eYFP    200ng pDONR            0nM Dox
300ng TRE:L7Ae 300ng TRE:mKate 100ng hEF1a:rTta 100ng hEF1a:eYFP 200ng pDONR           20 nM Dox
300ng TRE:L7Ae 300ng TRE:mKate 100ng hEF1a:rTta 100ng hEF1a:eYFP 200ng pDONR           50 nM Dox
3
300ng TRE:L7Ae 300ng TRE:mKate 100ng hEF1a:rTta 100ng hEF1a:eYFP 200ng pDONR           100 nM Dox
300ng TRE:L7Ae 300ng TRE:mKate 100ng hEF1a:rTta 100ng hEF1a:eYFP 200ng pDONR           200 nM Dox
300ng TRE:L7Ae 300ng TRE:mKate 100ng hEF1a:rTta 100ng hEF1a:eYFP 200ng pDONR           500 nM Dox
300ng TRE:L7Ae 300ng TRE:mKate 100ng hEF1a:rTta 100ng hEF1a:eYFP 200ng pDONR           1000 nM Dox
4
300ng TRE:L7Ae 300ng TRE:mKate 100ng hEF1a:rTta 100ng hEF1a:eYFP 200ng pDONR           2000 nM Dox
200ng hEF1a:flp-eYFP        300ng pDONR   100ng hEF1a:VgEcr 100ng hEF1a:BFP    300ng EGSH:mKate   0uM PonA
200ng hEF1a:flp-eYFP        300ng EGSH:TP901 100ng hEF1a:VgEcr 100ng hEF1a:BFP    300ng EGSH:mKate   0uM PonA
200ng hEF1a:flp-eYFP        300ng EGSH:TP901 100ng hEF1a:VgEcr 100ng hEF1a:BFP    300ng EGSH:mKate   0.1uM PonA
5
200ng hEF1a:flp-eYFP        300ng EGSH:TP901 100ng hEF1a:VgEcr 100ng hEF1a:BFP    300ng EGSH:mKate   0.5uM PonA
200ng hEF1a:flp-eYFP        300ng EGSH:TP901 100ng hEF1a:VgEcr 100ng hEF1a:BFP    300ng EGSH:mKate   1uM PonA
200ng hEF1a:flp-eYFP        300ng EGSH:TP901 100ng hEF1a:VgEcr 100ng hEF1a:BFP    300ng EGSH:mKate   2uM PonA
200ng hEF1a:flp-eYFP        300ng EGSH:TP901 100ng hEF1a:VgEcr 100ng hEF1a:BFP    300ng EGSH:mKate   5uM PonA
Table3
Mixture preparation: L7Ae toxicity
A
B
C
D
1
Amount (ng)
Amount (uL)
8 samples
2
TRE:L7Ae
300
3
24
3
TRE:mKate
300
3
24
4
hEF1a:rtTA
100
1
8
5
hEF1a:eYFP
100
1
8
6
pDONR
200
2
16
7
Viafect
2
16
8
Opti MEM
38
304
9
Total
1000
50
400
Table2
TP901 flipped eYFP
A
B
C
D
E
F
G
1
Amount (ng)
**Amount (uL)
Amount (ng)
Amount (uL)
**6 samples
2
hEF1a:flp-eYFP
200
2
200
2
12
3
EGSH:TP901
0
0
300
3
18
4
EGSH:mKate
300
3
300
3
18
5
hEF1a:VgEcr
100
1
100
1
6
6
hEF1a:BFP
100
1
100
1
6
7
pDONR
300
3
0
0
0
8
Viafect
2
2
12
9
Opti MEM
38
38
228
10
Total
50
1000
50
300
Table1
Controls
A
B
C
D
E
1
Single (Y)
Single (R)
Single (B)
Three
2
hEF1a:eYFP
5
0
0
3
3
hEF1a:mKate
0
5
0
3
4
hEF1a:BFP
0
0
5
3
5
pDONR
5
5
5
1
6
Viafect
2
2
2
2
7
Opti MEM
38
38
38
38
8
Total
50
50
50
50
Table5
DNA Transfection into HEK Cells
Introduction
Get started by giving your protocol a name and editing this introduction.
Materials
OptiMEM (Serum-free media)
Viafect
DNA from midipreps
Procedure
Phase I: Calculations
Select DNA to be transfected and determine the concentration through Nanodrop
Determine the volume of DNA required to have 0.5 ug in reaction mixture.
Ex. for a 521 ng/ul stock of DNA you would need approximately 1 uL. If the calculated volume you need is less than 0.5 uLr (i.e. hard to pipette) you may want to dilute your DNA solution so that you can get an appropriate volume.
Determine the amount of Viafect to use from the following table:
A
B
C
D
E
F
G
H
1
Volume of cells in media (per well)
Total volume transfection complex (per well)
Amount of DNA (per well)
Amt of Viafect Reagent for 1.5:1 Reagent:DNA
2:1
3:1
4:1
6:1
2
500 uL
50 uL
0.5 ug
0.75 uL
1 uL
1.5 uL
2 uL
3 uL
Table1
Calculate the amount of serum free DMEM media required to get a reaction volume of 50 uL
Ex. if you are adding 1 uL of DNA and 3 uL of Viafect, you would want to add 46 uL of DMEM
Phase II: Reaction Mixture
This phase consists of mixing the reactants required to create the transfection complex. Do so for all ratios of reagent to DNA above - add media first, then reagent, then DNA.
Let the reaction mixture sit for 10-20 minutes
Closer to 20 min is better
Phase III: Transfection
Retrieve cells from incubator. Make sure to leave this step for last because you don’t want to keep the cells at room temperature for too long.
Add entire 50 uL transfection complex to each well in the well plate and store your cells back in the incubator.
Phase IV: Checking for Fluorescence
Close off lights in room and turn on the microscope’s light.
Apply the appropriate fluorescence filter.
Check for fluorescence.
Overnight liquid cultures (picking colonies)
Introduction
Overnight cultures are used to prepare miniprep DNA.
Materials
The plate from which you are picking colonies
15 ml round-bottom polystyrene tubes, one per culture
The ones with the snap caps, NOT conical tubes with screw caps
5 mL LB per culture
A container that can hold 5ml x the number of cultures
For a modest number of minipreps, a 50 ml conical tube works well.
For larger minipreps, use a sterile bottle (100 ml is frequently useful.)
Antibiotic stock, 1000X
Procedure
Materials Setup
Warm up the LB to at least room temperature (if it came from the fridge), but not warmer than 37°C
Label one round-bottom culture tube for each miniprep.  Use "NAME-1, NAME-2, ..."etc for the naming convention, where NAME is a shortened name of the plasmid (eg, "hEF1a:mKate").
Your impulse is to just use number, or initials and number, but trust me -- you will want to be able to identify this tube in three weeks when you've forgotten what you were doing.
Using a sterile pipette, transfer 5 ml of LB to the mixing container for each culture PLUS 5 ML.
Add antibiotic stock to a final concentration of 1X (1 µl stock for each 1 ml in the mixing container.)
Cap tightly and mix well.
Culture Setup
Using a sterile pipette, transfer 5 ml of LB+antibiotic to each round-bottom culture tube.
If you are making cultures with different antibiotics, take care that the right media goes in each tube.
Squirt ethanol on a pair of foreceps and wipe dry with a Kimwipe.
Use the foreceps to pick up a sterile 200µl pipette tip, scrape a colony off of the plate, and drop the pipette tip in the corresponding tube.
Repeat for each tube.
Transfer to an incubating shaker at 37°C and incubate 14-16 hours.
Don't over-grow too badly, or your yield will suffer.
If you need to grow longer, you can grow at 30°C instead for 20 hours.
TN 8/10/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Damon Berman
Date: 2016-08-10
Wednesday, 8/10
Dox dilution
Dox Stock Concentration: 100uM
A
B
C
D
E
F
G
H
1
Steps order
3
1
2
2
2
Final Dox concentration in well (nM)
Final volume (uL)
Volume added to each well (uL)
Dox concentration in working stock (nM)
Volume of working stock (uL)
Dox stock concentration (A) (nM)
Volume added from Dox stock (A) (uL)
Volume of media (uL)
3
0
560
10
0
50
20000
0
50
4
20
560
10
1120
50
20000
2.8
47.2
5
50
560
10
2800
50
20000
7
43
6
100
560
10
5600
50
20000
14
36
7
200
560
10
11200
50
20000
28
22
8
500
560
10
28000
50
50000
28
22
9
1000
560
10
56000
50
100000
28
22
10
2000
570
20
57000
50
100000
28.5
21.5
Table1
Preparing Dox stock solution (A)
A
B
C
D
E
1
Dox stock concentration (nM)
Total volume (uL)
Dox Original Stock (OS) Concentration (nM)
OS volume (uL)
Media volume (uL)
2
20000
100
100000
20
80
3
50000
100
100000
50
50
4
100000
100
100000
100
0
Table2
PonA Induction
A
B
C
D
E
F
G
H
1
Steps order
3
1
2
2
2
Final Dox concentration in well (nM)
Final volume (uL)
Volume added to each well (uL)
Dox concentration in working stock (nM)
Volume of working stock (uL)
Dox stock concentration (A) (nM)
Volume added from Dox stock (A) (uL)
Volume of media (uL)
3
0
560
10
0
50
20000
0
50
4
100
560
10
5600
50
20000
14
36
5
500
560
10
28000
50
20000
70
-20
6
1000
560
10
56000
50
20000
140
-90
7
2000
560
10
112000
50
20000
280
-230
8
5000
560
10
280000
50
50000
280
-230
Table4
Miniprep Nanodrop Results:
A
B
1
1-1
500.4
2
1-2
280.8
3
1-3
254.3
4
1-4
251.2
5
2-1
216.1
6
2-2
188.9
7
2-3
220.3
8
2-4
354.1
Table3
Restriction Digest:
Screen Shot 2016-08-10 at 10.09.44 AM.png
recombinase_gel.jpg
LR Reactions
1.
pBM3R1: TP901
2.
pTal14: TP901
3.
pTal21: TP901
4.
pERE3: attB-flipped EYFP-attP
5.
pERE5: attB-flipped EYFP-attP
6.
pERE6: attB-flipped EYFP-attP
7.
pERE3: VP16-Gal4
8.
pERE5: VP16-Gal4
9.
pERE6: VP16-Gal4
NOTE: we ran out of LR clonase, so only reactions 1-4 and 7 have clonase added - the other four (5, 6, 8, and 9) are in our pink box in the -20 freezer and have the other four components mixed and ready to have clonase added when we get more
Miniprep
Introduction
The miniprep uses silica gel to isolate plasmid DNA from an E. coli culture
Materials
Buffer P1 (resuspension buffer)
Retrieve from refrigerator.  If you are opening a new miniprep kit, add the RNAse and LyseBlue reagent and check the box on the cap.
Buffer P2 (lysis buffer)
Open the cap and look at the lysis buffer.  Swirl it around.  If it appears cloudy, the SDS has fallen out of solution; warm it for a few minutes in the 55°C water bath.
Buffer N3 (neutralization buffer)
Buffer PB (binding buffer)
Buffer PE (rinse buffer)
Make sure the "Ethanol added?" box has been checked.  If you are opening a new miniprep kit, add absolute ethanol as per the kit instructions and check the box on the cap.
Buffer EB (elution buffer)
Miniprep waste container
Miniprep buffers contain salts that can't go down the sink.
Per miniprep: two microcentrifuge tubes and one blue spin column, with collection tube.
Procedure
Harvest and resuspension
For each culture, label two microcentrifuge tubes on the cap and one blue spin column on the side.
The spin columns should be in their (cap-less) collection vials.
Pipette 1.6 mL of each culture into the corresponding microcentrifuge tubes.
(This is for a 1.7 ml microcentrifuge tube; set your pipettor to 800 ul and do two transfers.  If we ever go back to 2-ml tubes, do 1.9 ml (2x950 ul))
Centrifuge at maximum speed (10,000 or 13,000xg) for three minutes.
Aspirate the supernatant, or pour it off into the bleach bucket.
Pipette ANOTHER 1.8 ml of each culture into the corresponding microcentrifuge tubes.
Centrifuge at maximum speed for three minutes.
While the centrifuge is running, move the remaining cultures to 4degC.
Aspirate the supernatant off with the bench aspirator.  Be careful not to disturb the pelleted E. coli.
We use an aspirator here because the less extra salt and protein we put in the miniprep, the better the yield is.
Add 250 µl Buffer P1 to each tube.
Resuspend the E. coli pellet.  The preferred way is with the roto-mixer at the other end of the lab.
Alternately, if you have just a few tubes, you can resuspend on a vortex.
Make sure to resuspend fully and thoroughly. The resulting suspension should be smooth and cloudy; if there is particulate matter floating around, vortex some more.
Lysis
Add 250 µl Buffer P2 to each tube.
Work quickly; the lysis step should take less than 5 minutes.
Snap the tubes closed and invert them 4-6 times, until the tube is thoroughly mixed and the entire solution turns blue.
If you have many many tubes, you can stack a second tube rack on top of them and invert the entire thing.
Add 350 ul Buffer N3 to each tube.
Snap the tubes closed and invert 4-6 times, until the solution is thouroughly mixed and no longer blue.
The solution will become cloudy or flocculent.
Centrifuge on high speed for 10 minutes.
Separation
Remove the tubes from the microcentrifuge, being careful not to disturb the white pellet.
Using P-1000 micropipettor set to 850 ul, carefully transfer the supernatant from each centrifuge tube to the corresponding blue spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Don't forget to put the lid on the rotor! Some of the salts get aerosolized because the spin columns don't have caps.
Pour the flow-through from each column into the miniprep waste container.
Pipette 500 ul of Buffer PB onto each spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Pipette 750 ul of Buffer PE onto each spin column
Wait 1-3 minutes.
This allows some of the salt that's still bound to the silica matrix to resuspend in the buffer.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Return each spin column to its collection tube and centrifuge an additional 1 minute at high speed.
This removes every last trace of buffer PE; the ethanol can screw up downstream steps.
Transfer each spin column to a clean labelled microcentrifuge tube.
Pipette 50 ul of Buffer EB onto the center of each column.
The volume of EB is comparable to the volume of silica gel matrix; if you pipette down the side, you might not get the entire transfer to the matrix.
Wait 1-3 minutes.
This gives the DNA a chance to dissociate from the silica matrix.
Centrifuge the spin columns, in their collection tubes, for one minute at maximum speed.
Proceed directly to analyze the samples on the Nanodrop.
Sequencing protocol
Introduction
Preparing sample to send to Genewiz for sequencing Sequencing Pick-up Times: 9:30am, 3:00 pm, 5:30 pm
Materials
DNA stock
Nuclease-free water
Primer (5nM stock)
One microcentrifuge tube for each DNA plasmid
2 PCR tubes per plasmid (must be in strips of eight, round up if needed!)
Procedure
Diluting the DNA
For orders with <48 samples, use 8-strip PCR tubes. Label your tubes on the side vertically with "MI" and sample number (01, 02, 03, etc.). Use image below as guideline. Each plasmid you want to sequence will require 2 tubes, one for the forward primer and one for the reverse primer. Use "MI" instead of the "GW" in the picture.
Dilute your sequencing primer to 5 µM (pmol/µl) using water. You will need 5 µl for each sequencing reaction. Note, the primers in the tubes from IDT have not been diluted! Use already diluted stock (should be in regular microcentrifuge tubes).
Using the recorded concentration, dilute the DNA to the template concentration in 10 µl as detailed below. For example, a 5 kb plasmid would need a template concentration of ~50 ng/µl. Final volume for your working stock should 30 uL. Please make dilutions in water or Tris.
Add 10 µl of diluted template DNA to each tube.
Add 5 µl of diluted primer to each tube. Be careful about putting the forward and reverse primers in the correct tubes! Remember, for each DNA template, there should be two tubes - one with reverse primer and the other with forward primer
Vortex briefly
Log into genewiz. Username: igem-sequence@mit.edu Password: igem2014citrus
Click "create sequencing order"
Sevice Priority: click on "standard"
Create Order by: click on "online form"
Sample Type: click on "Pre-mixed"
Enter in how many samples you are sending in and click "Create new form"
Enter in descriptive names for your DNA (this is to help you remember when you get the results back), the DNA type, the DNA length (get this from your construct on benchling), the primer name (again, this is more for you). When you are done hit save and next.
It will give you an list of what you have entered and a price for each reaction. Hit "next step"
The payment information field should be autofilled. There has been an issue recently with the PO box number not being entered. If this is the case, click "credit card" (next to "payment info")  and then click "PO" to bring you back to the original form. Now you should be able to enter in a PO number. Open up a previous submitted order (there should be some saved to the desktop called "sequencing" or something like that) and copy over the PO number. PO Number:  5510061907
Hit "next step". Once it gives you a submitted order form, print 2 copies and pick them up from the printer (exit lab, through glass doors, on your left)
Put your PCR tubes in a large falcon tube.
Put the falcon tube in a ziploc bag (located in Qiagen column drawer) and put in one order form. Label the bag with something descriptive ("Weiss Lab MIT iGEM Recombinase 6/30/16" for example)
Put the bag in the Genewiz pickup box (located by the elevators, make sure you put in the genewiz box)
Open up the weiss lab orders spreadsheet (bookmarked on the computer). You need to request access to the spreadsheet if it is your first time ordering. Once you request access, email Brian so he knows that you have sent in a request.
Click the tab at the bottom called "genewiz".
Fill out the form. Make sure you indicate that the order is from iGEM so the right account can be charged.
You did it! YAY! Genewiz should get back to you pretty quickly, generally between 12-24 hrs.
CF 8/12/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Maya Kaul
Date: 2016-08-12
Friday, 8/12
LR reactions
1.
pBM3R1: TP901
2.
pTal21: TP901
3.
pERE3: attB-flipped EYFP-attP
4.
pERE5: attB-flipped EYFP-attP
5.
pERE3: VP16-Gal4
6.
pERE5: VP16-Gal4
Pouring LB Agar Plates
Introduction
The media must be autoclaved before you pour plates.  This is not something for iGEMers to do; ask an advisor.
Materials
One sleeve of empty plates per 500 ml of media
Make sure you are using the Weiss lab's bacteriological plates, not tissue culture plates.
LB-Agar media, either solid or already melted
1000X stock solution of the antibiotic
(Amp)icillin, (Kan)amycin, (Chlor)amphenicol
Procedure
Making media
Make sure you don't already have a bunch of the plates already in the fridge
Fill a 1-liter bottle with 500 ml of water.(You can make up to 900mL for a 1-L bottle)
Weigh out XXXX of powdered LB-Agar (check the bottle for the weight.)
IF YOU MAKE A MESS OF THE WEIGH STATION, CLEAN UP AFTER YOURSELF.
Ask an instructor to autoclave your media.
Melting media
Solid media must be melted in the microwave. Be careful -- it gets very hot.
Loosen the cap to prevent the bottle from exploding. 1 full turn of the cap is sufficient.
Microwave 2-3 minutes, watching carefully, until the media begins to melt and bubble.
Continue microwaving in short bursts until the media is completely melted.
IF YOU MAKE A MESS OF THE MICROWAVE, CLEAN UP AFTER YOURSELF.
Cool the media to <= 60°C before adding the antibiotics.
You can either swirl it under a cold tap until you can handle it without burning yourself, or you can leave it for a few hours in the 55°C water bath.
To check the media temperature, swirl the bottle, wait ~30 seconds and measure with an IR thermometer. (Pew pew.)
Pouring plates
Add antibiotic.
Unless otherwise noted, it is 1000X -- so for 500 ml of media, add 500 µl of antibiotic.
Note: If you're pouring Golden Gate plates, XGal is 500X! (for 500 ml of media, add 1000 µl of XGal)
Pour all of the media you add these too. Refreezing and thawing has unpredictable results on the amount of antibiotic destroyed by the heat, so to avoid over/under dosing don't refreeze. 500 ml makes a sleeve of plates.
Tighten the lid to avoid spills and swirl the bottle to mix
Use a seriological pipette to add 20ml to each plate.
Use sterile practices
If you get bubbles, use the pipette to try and suck the air out of the bubbles, this removes them
20ml is a rough estimate, what is important is that you get a solid layer with no holes.
Mark the plates with a stripe of marker up the side, based on which antibiotic(s) you added:
Blue: Amp
Red: Kan
Green: Cm
Black: Non-antibiotic additive (like X-gal)
Dry on the bench overnight.
Re-stack the plates upside-down and label the sleeve with the antibiotics, your initials and the date.
LR Reaction
Introduction
An LR reaction inserts one or more parts in pENTR vectors into a pDEST vector.  Used to assemble transcriptional units from promoters and genes.
Materials
Promoter pENTR plasmid: L4-Promoter-R1
Working concentration: 5 fmol/ul
Gene pENTR plasmid: L1-Gene-L2
Working concentration: 5 fmol/ul
Destination plasmid: pDEST
Working concentration: 10 fmol/ul
Nuclease-free TE
200 µl PCR strip tubes, 1 tube per rxn
5x LR Clonase II
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
Proteinase K
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
Procedure
LR Reaction Setup
For each LR you are doing, fill out a column in the following table:
A
B
C
D
E
F
1
Tube Label
2
Promoter pENTR
3
Gene pENTR
4
pDEST
5
6
Table1
For each LR, label a 200 µl strip tube with your initials and tube number.
Into each tube, pipette:
-- 1 µl of the promoter pENTR
-- 1 µl of the gene pENTR
-- 1 µl of the pDEST
Add 1 µl of TE to each tube
Retrieve an aliquot of LR Clonase from the -80.
Bring an razor blade with you, you'll need to cut a tube from the strip tubes.
Pulse the LR clonase tube in the microfuge to collect the clonase at the bottom.
Add 1 µl of the LR clonase to each LR reaction.
Be careful pipetting; LR clonase is viscous.
Cap the tubes.
Flick them several times to mix.
Pulse-spin the tubes in the microfuge to collect the liquid at the bottom.
Incubate at room temperature for at least 12 hours and not more than 24 hours.
A popular strategy is to tape the tubes to the shelves over the bench, with your initials and the date.
16-24 hours later: Proteinase K kill
Retrieve a 5 µl aliquot of proteinase K from the -80 freezer.m n
Thaw in your fingers, then pulse in the microfuge to collect at the bottom of the tube.
Pipette 1 ul into each of the LR reactions.
Flick several times to mix.
Pulse-spin the tubes in the microcentrifuge.
Incubate at 37° for 15 minutes, or room-temperature for an hour.
PAUSE POINT: You can store the reactions in the -20 indefinitely until the transformation.
00:15:00
Proceed to transformation.  Transform 2 µl.
Afterwards, cap the tubes. Write the date on the caps and store in the -20 (in case your transformation failed.)
