Team:MIT/Experiments/miRNA
BACKGROUND
EXPERIMENTS
Synthetic Promoters microRNA Recombinase Proof of Concept Collaborations
FUTURE WORK
Circuit Design Clinical Applications
HUMAN PRACTICES
Parts Toolbox Accessible Protocols Integrated Human Practices Public Outreach Interlab Study
NOTEBOOK
Lab Notebook Parts Mammalian Parts Collection Attributions Safety
AWARDS
HP Silver HP Gold Integrated HP Basic Part
TEAM
microRNA: Detecting a Cell Specific Profile
How can we detect endometriosis using microRNA?
What is miRNA?
microRNA (miRNA) are small units of RNA found in all mammalian cells and play a regular role in gene regulation. miRNA is single stranded and only about 20 nucleotides long. miRNA forms a RNA-induced silencing complex (RISC) with other proteins and guides RISC to the complementary sequence on the mRNA (the miRNA target site). RISC cleaves messenger RNA and inhibits gene expression post transcriptionally.
Why is miRNA a good biomarker?
There are hundreds of different miRNAs that can make up a specific cell's miRNA profile. The profile and activity of miRNA varies between different cell types and different cells states.This makes it is possible to tell the difference between cells from the uterus and liver, and the difference between healthy and diseased cells based on miRNA profiles.
Found in all mammalian cells
Unique to cell type
Unique to cell states
What is the miRNA profile of Endometriosis?
Endometrial cells from women with endometriosis have been found to have dysregulated miRNA compared to endometrial cells from women who do not have endometriosis.( 1, 2, 3 ). In order to study the uses of these miRNAs in our circuit, eight candidates were chosen based on fold difference in miRNA level between the the endometrial biopsy from women with and without endometriosis. The eight candidates are shown in the table to the right:
(1) One of the graphs observed in order to choose miRNA candidates to characterize. The three miRNAs, miR-34c-5p, miR-9, miR-34-b are shown to be downregulated and have a significant fold difference (*p=0.05 **p<0.05, error bars= standard error of mean) compared to their respective levels in healthy eutopic endometrial cells. These measurements were from an endometrial biopsy and taken using quantitative polymerase chain reaction (qPCR), similar to the other sources we also studied.
Eight miRNAs candidates were chosen to be characterized in order to determine their gene regulatory ability for the purposes of increasing the specificity of our circuit.
How can our circuit sense miRNA activity?
Our circuit utilizes the natural function of miRNA to regulate gene expression. Depending on the miRNA activity in a cell, different levels of our desired gene will be expressed. This allows our circuit to produce a differential output depending on whether the cell has dysregulated miRNA. This is achieved by attaching 4 tandem sites complementary to the affected miRNA following a gene of interest. This is called the miRNA target site (miRNA-ts).
As miRNA activity increases, the output of the desired gene decreases.
Read more about our decision to use miRNA sensors
Are the target sites sensitive to miRNA levels?
These 4 tandem sites for miRNA binding were tested by inserting them after a red florescent gene (mKate) in the 3' untranslated region. We were able to see a ten fold repression upon increasing the concentration of siRNA from 0 to 1 nM. Saturation appeared at about 10 nM.
Read more about our experiment testing miRNA target site sensitivity
Does miRNA activity in endometrial cells respond to hormones?
The eight miRNA candidate target sites were characterized in TERT-immortalized Human Endometrial Stromal Cells (tHESC). Unlike more common cell lines like MCF7 and HEK293, tHESC is not a highly characterized cell line.This meant that the levels of our eight miRNA candidates were completely unknown in tHESC. We set out to characterize our miRNA target sites (miRNA-ts) in tHESC using a miRNA sensor.
What is tHESC?
Read more about our experiment to characterize miRNA targets sites in tHESC
Back to Experiments Home
References
1.Burney, RO, et al. Molecular Human Reproduction (2009). 15(10):625-631.
2.Pan, Q. Xiaoping, L. Toloubeydokhti, T. Chegini, N. Basic Science of Reproductive Medicine (2007). 13(11):797-806.
3.Joshi, NR, et al. Human Reproduction (2015). 30(12):2881-91.
