<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_Experiments_Recombinases skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/Experiments/Recombinases</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navdiv-wrapper"><DIV class="navdiv"><UL><LI><A href="https://2016.igem.org/Team:MIT/Description">BACKGROUND</A></LI><LI><A href="https://2016.igem.org/Team:MIT/Experiments">EXPERIMENTS</A><DIV class="submenu"><A href="https://2016.igem.org/Team:MIT/Experiments/Promoters ">Synthetic Promoters</A><A href="https://2016.igem.org/Team:MIT/Experiments/miRNA">microRNA</A><A href="https://2016.igem.org/Team:MIT/Experiments/Recombinases  ">Recombinase</A><A href="https://2016.igem.org/Team:MIT/Proof">Proof of Concept</A><A href="https://2016.igem.org/Team:MIT/Collaborations">Collaborations</A></DIV></LI><LI><A href="https://2016.igem.org/Team:MIT/Diagnosis_and_Future_Implications">FUTURE WORK</A><DIV class="submenu"><A href="https://2016.igem.org/Team:MIT/Genetic_Circuit">Circuit Design</A><A href="https://2016.igem.org/Team:MIT/Clinical_Applications">Clinical Applications</A></DIV></LI><LI><A href="https://2016.igem.org/Team:MIT/Human_Practices">HUMAN PRACTICES</A><DIV class="submenu"><A href="https://2016.igem.org/Team:MIT/Mammalian_synthetic_biology_kit">Parts Toolbox</A><A href="https://2016.igem.org/Team:MIT/accessible_protocols">Accessible Protocols</A><A href="https://2016.igem.org/Team:MIT/Integrated_Practices">Integrated Human Practices</A><A href="https://2016.igem.org/Team:MIT/Engagement">Public Outreach</A><A href="https://2016.igem.org/Team:MIT/InterLab">Interlab Study</A></DIV></LI><LI><A href="https://2016.igem.org/Team:MIT/Notebook">NOTEBOOK</A><DIV class="submenu"><A href="https://2016.igem.org/Team:MIT/Notebook">Lab Notebook</A><A href="https://2016.igem.org/Team:MIT/Parts">Parts</A><A href="https://2016.igem.org/Team:MIT/Part_Collection">Mammalian Parts Collection</A><A href="https://2016.igem.org/Team:MIT/Attributions">Attributions</A><A href="https://2016.igem.org/Team:MIT/Safety">Safety</A></DIV></LI><LI><A href="https://2016.igem.org/Team:MIT/Awards">AWARDS</A><DIV class="submenu"><A href="https://2016.igem.org/Team:MIT/HP/Silver">HP Silver</A><A href="https://2016.igem.org/Team:MIT/HP/Gold">HP Gold</A><A href="https://2016.igem.org/Team:MIT/Integrated_Practices">Integrated HP</A><A href="https://2016.igem.org/Team:MIT/Basic_Part">Basic Part</A></DIV></LI><LI><A href="https://2016.igem.org/Team:MIT/Team">TEAM</A></LI></UL></DIV></DIV><TITLE> Recombinases Background Information </TITLE><H1 style="color:#0f3d7f; text-align: center; font-size: 40px; line-height: 40px;">Recombinases:  Giving Memory to a Genetic Circuit</H1><H1 style="color:#ffffff; background-color:#0f3d7f; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: Trebuchet MS"> How can our circuit demonstrate temporal specificity?</H1><A style="text-decoration: none; color: #000000; float: left; margin: 15px"><DIV style="width: 340px; text-align: center;"><I>A recombinase excises a segment of DNA.<B> Source: </B>University of Rochester Introductory Biochemistry.</I></DIV></A><P> Endometriosis cells have distinct characteristics at different points in the menstrual cycle, presenting a major challenge in identifying diseased cells. Capturing chronological molecular traits is very important in the diagnosis of many diseases. For our project, we use DNA binding proteins called <B>recombinases</B> to achieve this <B>temporal specificity. </B></P><P>Recombinases are enzymes that can <B>recognize recombination sites</B>, and can either cut out the DNA between these recognition sites or invert the DNA sequence. There are two main families of recombinases: serine recombinases (also sometimes called serine integrases) and tyrosine recombinases. Serine integrases invert sequences, while tyrosine recombinases can either cut or flip sequences depending on the orientation of their recognition sites. Some recombinases exhibit <B>unidirectionality</B>, meaning that once they reverse or cut out the sequence, this action cannot be undone. This means that instead of behaving like a switch, capable of turning on or off, <B>unidirectional recombinases behave as latches</B>. Thus, unidirectional recombinases display higher efficacy in DNA modification than bidirectional recombinases. 
We can use recombinases as biological &quot;latches&quot; in our circuit to gain temporal specificity. Once the abnormal hormone level and the miRNA profile characteristic of a diseased cell have been identified during one phase of the menstrual cycle, the <B>first recombinase</B> can be activated to essentially <B>“lock in”</B> that information. When the second half of the circuit confirms the cell as being diseased in the second phase of the cycle, a <B>second recombinase latch</B> can be triggered, <B>activating the overall circuit</B>. 
</P><DIV style="text-decoration: none; color: #000000; float: center; margin: 15px;text-align:center"><DIV style="width: 599px; text-align: center;display:inline-block;"><I> Showing the mechanism of recombinase biological latches capturing temporal specificity during the estrogen and progesterone cycles. 1) Disease-related biological traits during the estrogen high phase activate the inducible promoter, leading to 2) expression of recombinase 1. Recombinase 1 would then 3) <B>&quot;lock-in&quot;</B> this information <B>by irreversible gene modification</B>. Similarly, 4) during the progesterone high phase, biomarkers associated with the disease would activate another promoter, leading to 5) expression of the second recombinase. <B>Two irreversible gene modification events</B> performed by recombinases caused by the presence of disease biomarkers at distinct time points lead to the production of an output.</I></DIV></DIV><H1 style="color:#ffffff; background-color:#0f3d7f; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: Trebuchet MS"> Do our recombinases work?</H1><P> 
We investigated <B>2 models of using recombinase to control gene expression</B>:</P><OL style="Verdana;"><LI>Using a unidirectional tyrosine recombinase (Cre or FLP) to excise a transcriptional stop signal, allowing a downstream gene to be expressed.</LI><LI>Using a unidirectional serine recombinase (<A href="http://parts.igem.org/Part:BBa_K2100019">TP901</A>) to flip gene from an off to an on orientation.</LI></OL><DIV style="text-decoration: none; color: #000000; float: center; margin: 15px;text-align:center"><DIV style="width: 499px; text-align: center;display:inline-block;"><I>Regulating gene expression using recombinases models.</I></DIV></DIV><P> Our experimental data showed that:</P><OL><LI>The <B>flipped gene system (2nd model)</B> successfully <B>knocked down the expression of the gene</B>, while the transcriptional stop signal (1st model) did not.</LI><LI>The expression level of the flipped gene can be indirectly controlled by expression of the <B>recombinase (TP901) under an inducible promoter</B>.</LI></OL><CENTER><I>  Recombinase TP901 showed approximately a two-fold increase between the off and on states under PonA-inducible promoter EGSH. </I></CENTER><A href="https://2016.igem.org/Team:MIT/Experiments/EGSH_TP901_Experiment"><P style="font-family: Trebuchet MS;font-color:#7ECEFD; font-size:18px;"><I><B>Read more about recombinase experiments here</B></I></P></A><H1 style="color:#ffffff; background-color:#0f3d7f; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: Trebuchet MS"> Challenges with High Efficiency of Recombinases</H1><P> Recombinases are highly efficient enzymes. When combined with a high-basal-activity promoter, this efficiency presents a challenge in our diagnostic tool: a few copies of the recombinase produced due to the promoter's leaky expression could lead to a significant amount of undesired output gene expression. In order to effectively use recombinases as biological latches, the basal expression must be reduced as much as possible. We need to use a strong repression system in order to reduce leaky expression. </P><H2 style="color: #000000; text-decoration:underline; font-family: Trebuchet MS;"> Repressible Promoters</H2><P> In order to gain tighter control of the recombinases, we paired them with repressible promoters that do not allow transcription of the recombinase to take place if the corresponding repressor protein is present. The three repressors we investigated were <A href="http://parts.igem.org/Part:BBa_K2100022">BM3R1</A>, <A href="http://parts.igem.org/Part:BBa_K2100024">TAL14</A>, and <A href="http://parts.igem.org/Part:BBa_K2100023">TAL21</A> because of their demonstrated success in literature. </P><A href="https://2016.igem.org/Team:MIT/Experiments/Repressors"><P style="font-family: Trebuchet MS;font-color:#7ECEFD; font-size:18px;"><I><B>Read more about repressor experiments here</B></I></P></A><H2 style="color: #000000; text-decoration:underline; font-family: Trebuchet MS;"> Translational Regulation: L7Ae/kink-turn</H2><P> We did a lot of research into effective high level repression systems, such as degradation domains and RNA-based gene regulation systems. After talking to experts in the Weiss Lab, we decided to test the L7Ae/k-turn system due to the system's efficiency and the availability of its components. 
</P><DIV style="text-decoration: none; color: #000000; float: center; margin: 15px;text-align:center"><DIV style="width: 90%; text-align: center;display:inline-block;"><I><B>Dox = 1000nM</B> activates the L7Ae/k-turn repressing system, and <B>PonA = 5uM</B> activates the expression of TP901 recombinase. The number of k-turn motifs at the 5'UTR can tune the expression of the regulated gene (<A href="http://parts.igem.org/Part:BBa_K2100019">TP901</A>). Doubling the k-turn repeats — from 2x k-turns to 4x kt-turns — reduces the expression level of TP901 by half (left figure). However, this ratio is not reflected in the expression level of the flipped eYFP, which is directly regulated by the recombinase and indirectly affected by the L7Ae/k-turn system. We do, however, observe a reduction in the activation of the output gene between  samples with and without k-turns (right figure).</I></DIV></DIV><A href="https://2016.igem.org/Team:MIT/L7AeRepressingSystem"><P style="font-family: Trebuchet MS;font-color:#7ECEFD; font-size:18px;"><I><B>Read more about our L7Ae k-turn experiments here</B></I></P></A><A style="text-decoration:none;" href="https://2016.igem.org/Team:MIT/Experiments"><H1 style="color:#FFFFFF; background-color:#0f3d7f; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: trebuchet MS"> Back to Experiments Home</H1></A></DIV></DIV></DIV></DIV></DIV></BODY></HTML>