Team:MIT/L7AeRepressingSystem
BACKGROUND
EXPERIMENTS
Synthetic Promoters microRNA Recombinase Proof of Concept Collaborations
FUTURE WORK
Circuit Design Clinical Applications
HUMAN PRACTICES
Parts Toolbox Accessible Protocols Integrated Human Practices Public Outreach Interlab Study
NOTEBOOK
Lab Notebook Parts Mammalian Parts Collection Attributions Safety
AWARDS
HP Silver HP Gold Integrated HP Basic Part
TEAM
L7Ae k-turn repressing system
L7Ae - Kink turn
RNA-Based Gene Regulation
L7Ae, an archaeal ribosomal protein, binds with high affinity to RNA motifs called kink-turns (k-turns), found in both archaeal and eukaryote RNAs [1][2][3]. The L7Ae protein sequence is divided into three structural regions, consisting of a highly conserved RNA-binding region (RBR) flanked by less conserved N-terminal and C-terminal regions [2]. Variation in the terminal regions could dictate RNA-binding specificity of different homologs of L7Ae protein [2]. When a k-turn motif is inserted into the target mRNA upstream of the open reading frame, L7Ae acts as a post-transcriptional regulator by preventing the ribosome machinery from performing translation[1][2][3]. The strength of the repression can be controlled by varying the distance between the k-turns and the 5’-end of the mRNA or by changing the number of the k-turn motifs [1].
Binding of L7Ae to kink-turn motifs preveting translation.
Toxicity concentration of L7Ae in mammalian cell
Purpose
We talked with graduate students and postdoctoral research associates in Prof. Weiss' lab to gain more information about the L7Ae/k-turn repressing system, and we learned from them that high concentration of L7Ae could potentially be harmful for mammalian cells since mammalian cell cultures have been showing unhealthy morphology under high concentrations of L7Ae. Thus, we designed an experiment to examine the toxicity of L7Ae to HEK293.
Experimental Setup
We put L7Ae under the Dox-inducible promoter pTRE . Higher amounts of Dox (a small molecule inducer) would increase the amount of L7Ae in the cell. The reporter gene for the amount L7Ae produced was pTRE:mKate . Yellow fluorescence ( hEF1a:eYFP ) was the transfection marker. We induced the cells with different amount of Dox concentration 24 hours after transfection. We trypsinized and harvested the cells for flow cytomertry analysis 24 hours after induction. During the preparation for flow cytometry, we stained the cells with SYTOX blue, a DNA-binding stain that cannot pass through an intact cell membrane, for live/dead cell analysis.
Click here to show/hide plate map of L7Ae toxicity experiment.
 
   
Untransfected Control
Single color (Y)
500ng hEF1a:eYFP
500ng pDONR
Single color (R)
500ng hEF1a:mKate
500ng pDONR
Single color (B)
500ng hEF1a:tagBFP
500ng pDONR
 
   
Three colors
300ng hEF1a:eYFP
300ng hEF1a:mKate
300ng hEF1a:BFP
100ng pDONR
300ng TRE:L7Ae
300ng TRE:mKate
100ng hEF1a:rtTA
100ng hEF1a:eYFP
200ng pDONR
0nM Dox
   
300ng TRE:L7Ae
300ng TRE:mKate
100ng hEF1a:rtTA
100ng hEF1a:eYFP
200ng pDONR
20 nM Dox
300ng TRE:L7Ae
300ng TRE:mKate
100ng hEF1a:rtTA
100ng hEF1a:eYFP
200ng pDONR
50 nM Dox
 
   
300ng TRE:L7Ae
300ng TRE:mKate
100ng hEF1a:rtTA
100ng hEF1a:eYFP
200ng pDONR
100 nM Dox
300ng TRE:L7Ae
300ng TRE:mKate
100ng hEF1a:rtTA
100ng hEF1a:eYFP
200ng pDONR
200 nM Dox
300ng TRE:L7Ae
300ng TRE:mKate
100ng hEF1a:rtTA
100ng hEF1a:eYFP
200ng pDONR
500 nM Dox
300ng TRE:L7Ae
300ng TRE:mKate
100ng hEF1a:rtTA
100ng hEF1a:eYFP
200ng pDONR
1000 nM Dox
   
300ng TRE:L7Ae
300ng TRE:mKate
100ng hEF1a:rtTA
100ng hEF1a:eYFP
200ng pDONR
2000 nM Dox
Result
Testing the toxicity of L7Ae in HEK293. Varying amount of  Dox would vary the expression of L7Ae (Dox concentraions: 1, 20, 50, 100, 200, 500, 1000, 2000 nM). X-axis: hEF1a:eYFP = transfection marker, Y-axis: Dead cells.
At low cell count, the amount of dead cells didn't change much for Dox concentrations less than 1000nM. When the Dox concentration was higher than 1000nM, the amount of dead cells increased linearly, indicating the cells' becoming unhealthy. At high cell count, the amount of cell dead increased with increasing concentration of Dox, and began to increase drastically between [Dox] = 1000nM and [Dox] = 2000nM. Thus, we decided to use 0-1000nM for the Dox concentration range to regulate the expression of L7Ae.
Recombinase and L7Ae-Kturn
Purpose
Using recombinases as biological 'latches' gives our genetic circuit the 'memory' necessary for identifying the disease's temporal specificity. Since the recombinase is controlled by an inducible promoter, however, leaky expression of the promoter (activation without input signals - disease biomarkers) could lead to unwanted activation of the output gene and thus a false positive diagnosis. By putting k-turn motifs in front of the recombinase gene, we hope to eliminate leaky expression of the recombinase when the system is not activated.
We designed an experiment to examine the repression level of the L7Ae - kink turn system on the expression of an output gene (eYFP - enhanced yellow flourescent protein), which is regulated by TP901 (a serine recombinase).
Experimental Setup
Diagram explaining experimental setup testing the effect of L7Ae/k-turn repressing system on basal expression of recombinase TP901.
We used two inducible promoter systems - pEGSH/PonA and pTRE/Dox - in this experiment to control the expression of two genes, L7Ae for tuning the repressing level and TP901, the recombinase. The pTRE/Dox system controled the expression of L7Ae. We induced the cells with Dox at the same time as transfection because the repressing system needed to be activated before TP901 recombinase was induced by PonA/pEGSH.
Additionally, including all the neccessary genes (TP901, flipped eYFP, L7Ae, VgEcr-RXR, and rtTA) and the reporter flourescent genes, the total number of plasmids went up to 7 (or 8 with the dummy DNA - pDONR). We were reaching the upper limit of the number of plasmids that can be cotransfected using lipofection. After asking for advice from other members of the Weiss lab, we increased the ratio of viafect:DNA from 1ul:500ng total DNA to 1.5ul:500ng total DNA.
Parts we used for this experiment: EGSH 4xk-turn:TP901 , EGSH 4xk-turn:mKate . EGSH 2xk-turn:TP901 , EGSH 2xk-turn:mKate , EGSH:TP901 , EGSH:mKate , hEF1a:flipped eYFP , TRE:L7Ae , hEF1a:VgEcr , hEF1a:rtTA
For each well:
Total amount of DNA: 1500ng
Viafect transfection reagent: 4.5ul
Click here to show/hide plate map of TP901 - L7Ae/k-turn experiment.
 
   
Untransfected Control
   
Single color (Y)
1000ng hEF1a:eYFP
500ng pDONR
   
Single color (R)
1000ng hEF1a:mKate
500ng pDONR
Single color (B)
1000ng hEF1a:tagBFP
500ng pDONR
 
Three colors
500ng hEF1a:eYFP
500ng hEF1a:mKate
500ng hEF1a:BFP
Control no L7Ae
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
0ng TRE: L7Ae
200ng hEF1a:BFP
300ng pDONR
1000nM Dox; 5uM PonA
   
Control no k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
1000nM Dox; 0uM PonA
Control no k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
1000nM Dox; 5uM PonA
Experiment 2x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
0nM Dox; 0uM PonA
Experiment 2x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
0nM Dox; 5uM PonA
Experiment 4x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
0nM Dox; 0uM PonA
Experiment 4x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
0nM Dox; 5uM PonA
Experiment 2x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
100nM Dox; 0uM PonA
Experiment 2x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
100nM Dox; 5uM PonA
Experiment 4x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
100nM Dox; 0uM PonA
Experiment 4x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
100nM Dox; 5uM PonA
Experiment 2x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
500nM Dox; 0uM PonA
Experiment 2x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
500nM Dox; 5uM PonA
Experiment 4x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
500nM Dox; 0uM PonA
Experiment 4x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
500nM Dox; 5uM PonA
Experiment 2x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
1000nM Dox; 0uM PonA
Experiment 2x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
1000nM Dox; 5uM PonA
Experiment 4x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
1000nM Dox; 0uM PonA
Experiment 4x k-turn
300ng EGSH-kturn: TP901
300ng EGSH-kturn:mKate
200ng hEF1a: flipped eYFP
100ng hEF1a: VgEcr
100ng hEF1a: rtTA
100ng TRE: L7Ae
200ng hEF1a:BFP
1000nM Dox; 5uM PonA
Read more about pEGSH/PonA inducible promoter system here.
Result
Testing the 2x k-turn L7Ae system with varied L7Ae expression level
Dox activates pTRE controlling the expressiong of L7Ae. PonA = 0uM: uninduced TP901 recombinase. PonA = 5uM: induced TP901 recombinase.
These graphs show that as the amount of Dox increased, there was smaller amount of eYFP expressed when TP901 expression was induced. When [Dox] = 0uM, at a high plasmid copy number, the amount of activated eYFP in induced TP901 sample was 2-fold higher than in uninduced TP901 sample (y-axis is in log scale). However, when Dox = [1000nM], the amount of activated EYFP in uninduced and induced TP901 samples stayed the same for a larger range of plasmid copy numbers. At high copy numbers, however, the amount of eYFP produced was still higher for induced TP901 than uninduced, and the amount difference is larger at lower concentrations of Dox. Thus, the L7Ae/k-turn RNA-based gene regulation system could reduce the basal expression of inactivated TP901 while still allowing the recombinase to perform its function when the whole system is activated.
Testing the effect of varying k-turn sequences
Dox = 1000nM - activate the L7Ae/k-turn repressing system, and PonA = 0uM - uninduced TP901.
Dox = 1000nM - activate the L7Ae/k-turn repressing system, and PonA = 5uM - induced TP901.
When we compared varying numbers of k-turns with constant amounts of Dox and PonA, we saw that the amount repression of mKate expression (and thus TP901 expression) increased significantly as we increased the number of k-turns for both the uninduced and induced states of pEGSH. While the difference between the amount of yellow fluorescence (and thus the amount of recombination) for 2 k-turns and 4 k-turns was less pronounced, we still observed a noticeable decrease in recombination when k-turns were present compared to when they were not present. We concluded that increasing the number of k-turns over the range we considered led to tighter repression by L7Ae.
Back to Recombinase Overview Page
REFERENCE:
Oliwia Andries, Tasuku Kitada, Katie Bodner, Niek N Sanders & Ron Weiss (2015) Synthetic biology devices and circuits for RNA-based ‘smart vaccines’: a propositional review, Expert Review of Vaccines, 14:2, 313-331
Gagnon KT, Zhang X, Qu G, et al. Signature amino acids enable the archaeal L7Ae box C/D RNP core protein to recognize and bind the K-loop RNA motif. Rna 2010;16(1):79-90
Stapleton JA, Endo K, Fujita Y, et al. Feedback control of protein expression in mammalian cells by tunable synthetic translational inhibition. ACS Synth Biol 2012;1(3):83-8
Wroblewska et al. Mammalian synthetic circuits with RNA binding proteins for RNA-only delivery. Nature Biotechnology 33, 839–841 (2015)
