Team:MIT/Recombinase Group Notebook Week 2
BACKGROUND
EXPERIMENTS
Synthetic Promoters microRNA Recombinase Proof of Concept Collaborations
FUTURE WORK
Circuit Design Clinical Applications
HUMAN PRACTICES
Parts Toolbox Accessible Protocols Integrated Human Practices Public Outreach Interlab Study
NOTEBOOK
Lab Notebook Parts Mammalian Parts Collection Attributions Safety
AWARDS
HP Silver HP Gold Integrated HP Basic Part
TEAM
TN 06/13/2016 LR · Benchling
TN 06/13/2016 LR
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Julia Goupil
Date: 2016-06-13
Monday, 6/13
LR Reaction
Introduction
An LR reaction inserts one or more parts in pENTR vectors into a pDEST vector.  Used to assemble transcriptional units from promoters and genes.
Materials
Promoter pENTR plasmid: L4-Promoter-R1
Working concentration: 5 fmol/ul
Gene pENTR plasmid: L1-Gene-L2
Working concentration: 5 fmol/ul
Destination plasmid: pDEST
Working concentration: 10 fmol/ul
Nuclease-free TE
200 µl PCR strip tubes, 1 tube per rxn
5x LR Clonase II
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
top shelf, far left metal rack, second column, bottom box, labled L
Proteinase K
Stored in ~5 µl aliquots in the -80 in room 235. Don't remove an aliquot until you're ready to use it.
top shelf, far left metal rack, third column, bottom box, labled K
Procedure
LR Reaction Setup
For each LR you are doing, fill out a column in the following table:
A
B
C
D
E
F
1
Tube Label
1
2
3
4
5
2
Promoter pENTR
EGSH
hEF1a
hEF1a
hEF1a
TRE
3
Gene pENTR
eYFP
BFP
VgEcr - 2A - RXR
rtTA
BFP
4
pDEST
GTW6
GTW6
GTW6
GTW6
GTW6
5
6
Table1
For each LR, label a 200 µl strip tube with your initials and tube number.
Into each tube, pipette:
-- 1 µl of the promoter pENTR
-- 1 µl of the gene pENTR
-- 1 µl of the pDEST
Add 1 µl of TE to each tube
Retrieve an aliquot of LR Clonase from the -80.
Bring an razor blade with you, you'll need to cut a tube from the strip tubes.
Pulse the LR clonase tube in the microfuge to collect the clonase at the bottom.
Add 1 µl of the LR clonase to each LR reaction.
Be careful pipetting; LR clonase is viscous.
Cap the tubes.
Flick them several times to mix.
Pulse-spin the tubes in the microfuge to collect the liquid at the bottom.
Incubate at room temperature for at least 12 hours and not more than 24 hours.
A popular strategy is to tape the tubes to the shelves over the bench, with your initials and the date.
16-24 hours later: Proteinase K kill
Retrieve a 5 µl aliquot of proteinase K from the -80 freezer.
Thaw in your fingers, then pulse in the microfuge to collect at the bottom of the tube.
Pipette 1 ul into each of the LR reactions.
Flick several times to mix.
Pulse-spin the tubes in the microcentrifuge.
Incubate at 37° for 15 minutes, or room-temperature for an hour.
PAUSE POINT: You can store the reactions in the -20 indefinitely until the transformation.
Proceed to transformation.  Transform 2 µl.
Afterwards, cap the tubes. Write the date on the caps and store in the -20 (in case your transformation failed.)
JG 6/14/16 Transformation
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Julia Goupil
Date: 2016-06-14
Tuesday, 6/14
Transformation of E. coli
Introduction
Transformation is the process of inducing chemically competent E. coli to take up DNA.
Materials
Dry bath, set to 42°C
Fill the wells in the dry bath block 1/2 full with DI water.
Ice bucket, with ice
For thawing competent cells.
DNA to transform
Could be an assembly reaction (LR, Golden Gate, etc) or a miniprepped plasmid.
If you removed it from the freezer, make sure it's entirely thawed  out.
pUC19 Transformation Control, 1 pg/µl
The pUC19 control will tell you how efficient your transformations were.
in iGEM reagent box
SOC growth media, at room temperature
Check to make sure it's clear and NOT CLOUDY.
Antibiotic plates, one per transformation, plus 1 Amp plate for the pUC19 control
Make sure the plates you use match the resistance cassette of the plasmid!
Competent E. coli, one tube per transformation + one for the pUC19 control
These live in the -80 in 235.
Thaw on ice 3-4 minutes.
A timer, set for 30 seconds.
Procedure
Setup
Make sure the dry bath is set to 42°C and the wells in the block are 1/2 full of DI water
Remove selection plates from the refrigerator.  Double-check that they match the selection marker on your plasmid, then place them in the 37° incubator.
Retrieve the DNA to transform.
If frozen: thaw, completely, flick a few times to mix, then pulse down in the microfuge.
Fill an ice bucket with ice.  Retrieve one tube of competent E. coli per transformation from the -80 and thaw on ice, 3-4 minutes.
While the transformation tubes are thawing, label their tops with something descriptive.  Record the labels here:
A
B
C
D
E
F
G
H
1
1
2
3
4
5
6
2
EGSH: EYFP
hEF1a: BFP
hEF1a: VgEcr
hEF1a: rtTA
TRE: BFP
pUC19 positive control
Table1
Transformation
Add 2 µl DNA from each reaction to a tube of competent cells.
Immediately after adding the DNA to each tube, stir the cells a few times with the pipette tip.
Add 1 µl of the pUC19 transformation control to the positive control tube.
Incubate on ice for 30 minutes.
Heat shock the cells for exactly 30 seconds in the 42° heat block. (Yes, set a timer.)
Place back on ice for 2 minutes.
Add 250 µl SOC to each tube.
Tape the tubes to the platform of a shaker at 37°C and shake at 270 RPM for 60 minutes.
Plating
Label the selection plates using the labels you recorded above.
Shake ~10 plating beads onto each plate. (Used innoculation loops instead of beads)
Pipette 100 µl of each transformation onto the corresponding plates.
Cover the plates and shake the beads around to spread the cells out.
Dispose of the beads by tapping them into the waste container.
Incubate the plates upside down overnight in the 37° incubator.
Don't incubate for more than 18-24 hours.
Compute transformation efficiency
Count the colonies on your positive transformation plate.
If there are many many colonies, then hooray!  You had a great transformation.  Just estimate.
Divide the number of colonies by the fraction of the transformation you plated.
So, if you resuspended your transformation in a total volume of 300 ul, then plated 100 ul, multiply the number of colonies by 3.
Transformation efficiency is expressed in colonies per microgram pUC19.  Multiply the number of colonies by the appropriate conversion factor.
So if you transformed 1 picogram of pUC19 DNA, multiply by 106.
Record your transformation efficiency in your (daily) lab notebook.
JG 6/15/16 Overnight culture
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Archis R. Bhandarkar
Date: 2016-06-15
Wednesday, 6/15
Overnight liquid cultures (picking colonies)
Introduction
Overnight cultures are used to prepare miniprep DNA.
Materials
The plate from which you are picking colonies
15 ml round-bottom polystyrene tubes, one per culture
The ones with the snap caps, NOT conical tubes with screw caps
5 mL LB per culture
A container that can hold 5ml x the number of cultures
For a modest number of minipreps, a 50 ml conical tube works well.
For larger minipreps, use a sterile bottle (100 ml is frequently useful.)
Antibiotic stock, 1000X
In the 4C fridge, Labeled iGem antibiotic
Procedure
Materials Setup
Warm up the LB to at least room temperature (if it came from the fridge), but not warmer than 37°C
Label one round-bottom culture tube for each miniprep.  Use "NAME-1, NAME-2, ..."etc for the naming convention, where NAME is a shortened name of the plasmid (eg, "hEF1a:mKate").
Your impulse is to just use number, or initials and number, but trust me -- you will want to be able to identify this tube in three weeks when you've forgotten what you were doing.
Using a sterile pipette, transfer 5 ml of LB to the mixing container for each culture PLUS 5 ML.
Add antibiotic stock to a final concentration of 1X (1 µl stock for each 1 ml in the mixing container.)
Cap tightly and mix well.
Culture Setup
Using a sterile pipette, transfer 5 ml of LB+antibiotic to each round-bottom culture tube.
If you are making cultures with different antibiotics, take care that the right media goes in each tube.
Squirt ethanol on a pair of foreceps and wipe dry with a Kimwipe.
Use the foreceps to pick up a sterile 200µl pipette tip, scrape a colony off of the plate, and drop the pipette tip in the corresponding tube.
Repeat for each tube.
Transfer to an incubating shaker at 37°C and incubate 14-16 hours.
Don't over-grow too badly, or your yield will suffer.
If you need to grow longer, you can grow at 30°C instead for 20 hours.
DB Miniprep 6/16/16 Results
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Archis R. Bhandarkar
Date: 2016-06-16
Thursday, 6/16
A
B
C
D
E
1
Colony 1
Colony 2
Colony 3
Colony 4
2
EGSH: eYFP
44.5 ng/uL
49.5 ng/uL
40.0 ng/uL
27.0 ng/uL
3
hEF1a: BFP
28.4 ng/uL
45.8 ng/uL
32.1 ng/uL
56.4 ng/uL
4
hEF1a: VgEcr
71.9 ng/uL
99.1 ng/uL
104.7 ng/uL
102.4 ng/uL
5
hEF1a: rtTA
52.9 ng/uL
289.9 ng/uL
67.4 ng/uL
37.0 ng/uL
6
TRE: BFP
33.4 ng/uL
32.3 ng/uL
24.6 ng/uL
106.9 ng/uL
Table1
JG 6/16/16 Miniprep
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Trinh Nguyen
Date: 2016-06-16
Thursday, 6/16
Miniprep
Introduction
The miniprep uses silica gel to isolate plasmid DNA from an E. coli culture
Materials
Buffer P1 (resuspension buffer)
Retrieve from refrigerator.  If you are opening a new miniprep kit, add the RNAse and LyseBlue reagent and check the box on the cap.
Buffer P2 (lysis buffer)
Open the cap and look at the lysis buffer.  Swirl it around.  If it appears cloudy, the SDS has fallen out of solution; warm it for a few minutes in the 55°C water bath.
Buffer N3 (neutralization buffer)
Buffer PB (binding buffer)
Buffer PE (rinse buffer)
Make sure the "Ethanol added?" box has been checked.  If you are opening a new miniprep kit, add absolute ethanol as per the kit instructions and check the box on the cap.
Buffer EB (elution buffer)
Miniprep waste container
Miniprep buffers contain salts that can't go down the sink.
Per miniprep: two microcentrifuge tubes and one blue spin column, with collection tube.
Procedure
Harvest and resuspension
For each culture, label two microcentrifuge tubes on the cap and one blue spin column on the side.
The spin columns should be in their (cap-less) collection vials.
Pipette 1.8 mL of each culture into the corresponding microcentrifuge tubes
Centrifuge at maximum speed (10,000 or 13,000xg) for three minutes.
Aspirate the supernatant, or pour it off into the bleach bucket.
Pipette ANOTHER 1.8 ml of each culture into the corresponding microcentrifuge tubes.
Centrifuge at maximum speed for three minutes.
While the centrifuge is running, move the remaining cultures to 4degC.
Aspirate the supernatant off with the bench aspirator.  Be careful not to disturb the pelleted E. coli.
We use an aspirator here because the less extra salt and protein we put in the miniprep, the better the yield is.
Add 250 µl Buffer P1 to each tube.
Resuspend the E. coli pellet.  The preferred way is with the roto-mixer at the other end of the lab.
Alternately, if you have just a few tubes, you can resuspend on a vortex.
Make sure to resuspend fully and thoroughly. The resulting suspension should be smooth and cloudy; if there is particulate matter floating around, vortex some more.
Lysis
Add 250 µl Buffer P2 to each tube.
Work quickly; the lysis step should take less than 5 minutes.
Snap the tubes closed and invert them 4-6 times, until the tube is thoroughly mixed and the entire solution turns blue.
If you have many many tubes, you can stack a second tube rack on top of them and invert the entire thing.
Add 350 ul Buffer N3 to each tube.
Snap the tubes closed and invert 4-6 times, until the solution is thouroughly mixed and no longer blue.
The solution will become cloudy or flocculent.
Centrifuge on high speed for 10 minutes.
Separation
Remove the tubes from the microcentrifuge, being careful not to disturb the white pellet.
Using P-1000 micropipettor set to 850 ul, carefully transfer the supernatant from each centrifuge tube to the corresponding blue spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Don't forget to put the lid on the rotor! Some of the salts get aerosolized because the spin columns don't have caps.
Pour the flow-through from each column into the miniprep waste container.
Pipette 500 ul of Buffer PB onto each spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Pipette 750 ul of Buffer PE onto each spin column
Wait 1-3 minutes.
This allows some of the salt that's still bound to the silica matrix to resuspend in the buffer.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Return each spin column to its collection tube and centrifuge an additional 1 minute at high speed.
This removes every last trace of buffer PE; the ethanol can screw up downstream steps.
Transfer each spin column to a clean labelled microcentrifuge tube.
Pipette 50 ul of Buffer EB onto the center of each column.
The volume of EB is comparable to the volume of silica gel matrix; if you pipette down the side, you might not get the entire transfer to the matrix.
Wait 1-3 minutes.
This gives the DNA a chance to dissociate from the silica matrix.
Centrifuge the spin columns, in their collection tubes, for one minute at maximum speed.
Proceed directly to analyze the samples on the Nanodrop.
Nanodrop
Introduction
Use the nanodrop to measure the DNA concentration in a sample.
Materials
The DNA to measure
Buffer EB (Elution Buffer)
Usually found at the Nanodrop station.
Procedure
Blank the Nanodrop
If it's not running, start the Nanodrop 2000 software.  Select "Nucleic Acids."
Ensure that the Type drop-down box on the right-hand side reads DNA.
Ensure that the Use cuvette box on the left-hand side is off.
Raise the Nanodrop arm.
Squirt a Kimwipe with a little water and gently wipe off both the measurement surfaces (the pedestal and the light aperture.)
Use a dry Kimwipe to gently wipe off both measurement surfaces.
Pipette 1.5 ul of Buffer EB onto the pedestal.
Gently lower the Nanodrop arm.
Click the Blank button.  Wait a few seconds for the instrument to blank.
Measure your samples
Gently wipe off both measurement surfaces.
Pipette 1.5 ul of your sample onto the pedestal.
Lower the Nanodrop arm.
Click the Measure button.
Record the concentration on the side of the tube and in the plasmid's notebook page.
Gently wipe off both measurement surfaces.
You do not need to use water to clean the surfaces between measurements; the measurement surfaces are hydrophobic and there is very littie sample careover.
Repeat steps 11-15 for each sample.
Clean the Nanodrop
Squirt a little water on a dry Kimwipe and wipe off both measurement surfaces.
Use a dry Kimwipe to wipe off both measurement surfaces.
Lower the arm of the Nanodrop before walking away from the instrument.
TN 06/16/2016 Literature Review
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Trinh Nguyen
Date: 2016-06-16
Thursday, 6/16
1.
Start with background on the authors
2.
Motivation + background information: Notch _ cell communication (intercellular communication)
○
Engineered system where cells working together
○
Signalling motifs: long-range and short-range
○
Short-range activation and long-range inhibition
○
Delta/Notch: initially discovered in fruit fly _ common cell-cell communication
Delta ligand: sending _ Notch: receiver
Intracellular domain is cleaved -> activiate transcripiton
Autoinhibition: when delta and notch express on the same cell
Delta pull on the notch -> creating cleavage
3.
Replace the intracellular domain of Notch with Gal4 -> activate a GFP
4.
Expansion: replace the extracellular and intracellular domain
○
Extracellular domain - notch core - intracellular domain
○
Extend the notch core -> tune the output
5.
What experiments to characterize the system?
○
Input - Ouput curve
○
Dynamic of the system (how fast?)
○
Check for cis inhibition
○
Cross talk between 2 different delta/notch systems on the same cell
○
Test on different cell lines (neuron, T cells, etc.)
6.
Application of these plastform
Therapeutic: engineered T-cells
Just work through some most improtant figures. Per experiments: why do they do it? what's the conclusion? What's next?
Set expectations -> fulfill expectations
Schedule for lab meeting:
Each subgroup once a week.
Regular check point
Write down every design decidions _ why did we do it?
TN 06/17/2016 diagnostic digest
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Maya Kaul
Date: 2016-06-17
Friday, 6/17
06-17-16 recombinase 1-2-5.1sc
06-17-16 recombinase 3-4-5.1sc
06-17-16 recombinase 1-2-5.jpg
1-4, 2-2,
06-17-16 recombinase 3-4-5.jpg
3-3, 4-4, 5-4
Diagnostic Restriction Digestion
Introduction
A diagnostic restriction digest helps identify correctly assembled clones from incorrect clones.
Materials
200 ul PCR strip tubes, one per reaction
Restriction enzyme (chosen below), 1 ul per reaction.
10X restriction enzyme buffer (chosen below), 1 ul per reaction.
Procedure
Choose A Good Restriction Enzyme
Using Benchling, choose a restriction enzyme that meets the following criteria:
- Cuts at least once in the insert (for a pEXPR, either the promoter or the gene.)
- Cuts at least once in the backbone.
- Gives bands that aren't too large (> 8 kb) or too small (< 200 bp).
- Gives a band pattern that is significantly different from the expected error mode.
* For LR reactions, the most common error is a pDEST that slipped through the selection.
- Begin in the "Brian's Favorites" list, then expand to the main Weiss lab list.
- If you can't find a single enzyme that gives an acceptable band pattern, choose two enzymes that give a acceptable band pattern when used together.
* This double-digest is subject to buffer compatibility, outlined below.
Record your enzyme choice on the plasmid's Description page.
Benchling will tell you the enzyme's buffer compatibility and active temperature. Record the buffer in which the enzyme is most active.
If there are multiple bufers in which the enzyme is equally active, choose in this order: Buffer 3.1, Buffer CS, Buffer 2.1, Buffer 1.1.
Set up the restriction digest
Retrieve the minipreps and the appropriate 10X buffer concentrate from the freezer.  Thaw on the benchtop or in your fingers.
Label the PCR tubes with your initials and an incrementing number.
ie: BT-1, -2, -3, -4
Vortex the minipreps and the 10X buffer concentrate briefly, then pulse down in the microfuge.
For each miniprep, set up a PCR tube containing the following in order:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
Remove the enzyme from the freezer for as little time as possible.
I have specified an "arbitrary" 3 µl volume of miniprep DNA; this should be fine as long as your miniprep concentration is >= 100 ng/ul.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
Incubate at the appropriate temperature for at least 1 hour and more more than 16 hours.
If the enzyme's active temperature is 37°C, use the 37°C plate incubator.
Stop the reaction by adding 2 ul of 6X NEB purple gel loading dye to each reacti on.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
PAUSE POINT: The reaction can be stored almost indefinitely at room temperature once it's been stopped.
Proceed to gel electrophoresis.
Pouring an agarose gel
Introduction
Agarose gels are used to separate DNA fragments.  Their most common use is
Materials
TAE buffer, 50-100 ml
There is generally 1X TAE buffer at the iGEM bench in a 1L or 2L bottle.
If not, mix some more up from the carboy 10X stock by the main lab gel station. (Use the 1L graduated cylinder at the gel station.)
UltraPure agarose
Casting tray, casting stand, combs
Procedure
Mix and melt the agarose
Check to see if there is a gel waiting in the fridge.
Determine how much molten agarose you'll need.
A small gel cast takes 50 ml; a large gel cast is 100 ml.
Measure out the appropriate amount of TAE into a glass bottle or flask.
Add 5 µl SYBRSafe (0.5X ; stock solution is 10000X) per 100 ml of the solution and mix well.
Add UltraPure agarose to a final concentration of 1% (mass / volume)
So, if you're making 50 ml gel: 0.5 g.
Swirl the bottle or flask to distribute the agarose.
Heat the solution in the microwave with frequent stirring to dissolve the agarose homogenously. ~30 seconds/100ml solution
Let sit until cool enough to handle.
The agarose MUST cool some -- if it's too hot, it can warp the casting trays.
Don't allow it to get too cold!
PAUSE POINT - You can put the flask in the 55°C water bath almost indefinitely at this point.
Pour the gel
Set up the gel tray in the casting stand.  Make sure the rubber gaskets are flat up against the edges of the casting tray.
Set up the gel combs to form the wells.
Rinse the combs with water and wipe dry.
Note for combs: 15-well combs hold about 6 ul liquid per well, 12-well combs hold about 15 ul per well, 8-well combs hold about 20 ul per well
Taping two 8-well comb wells together results in a well that holds up to 100 ul
Taping three 8-well comb wells together result in a well that holds up to 200 ul
Pour the molten agarose into the casting tray.
If bubbles form around the combs, remove and re-insert.
Wait 30 mintues for the gel to solidify.
Use immediately, or place in a plastic zip-lock baggie with a little 1X TAE and store at 4°C.
Splitting cells
Introduction
Culturing HEK-293
Materials
Lab coat should NOT be the same as the one used for bacterial work
Trypsin
p200 pipette tips (only use the ones from the orange boxes)
glass pipette (metal box found in iGEM drawer)
10cm culture dish
5mL conical tube
Media: should be warmed for ~45 minutes in the dry warmbath.
Procedure
If cell culture is in the incubator, remove and check under the microscope for confluence: under 20X magnification, should cover the bottom of the plate
Prepare the hood: fill in the sign-in sheet, turn on the regular light, wipe down with ethanol, and wipe everything down with ethanol before putting it under the hood
Put a glass tip on the vacuum tube.  To turn on the vacuum, there is a yellow knob on the left.  "On" means the knob is facing the hose.
With a plastic tip on the glass pipette, vacuum the media off the cell plate.
Quickly and gently use pipetter to add 3mL of trypsin to the plate and let sit 3-4 min
Add 10mL of media: pipette up and down to mix.  put into 50mL conical tube
Centrifuge (with balance!) at 2300rpm for 4 min.
Vacuum the media off the top and resuspend cells in 10 mL of media
Make the split: there should be 14mL of liquid in the culture dish.  a split refers to what amount of the 10mL cell culture is put into the culture dish
for 1:10 split: put 13 mL of media into culture dish, slowly add 1 mL of cell culture
for 1:5 split: 12mL of media, 2 mL of cell culture
for 1:2 split: 9mL of media, 5mL cell culture
Label plates with the cell type, initials, date, and the date of the next splitting (if 1:10, add 3 days, if 1:5, add 2 days, if 1:2, add 1 day).  Also put on passage number (how many times the cells have been split.)
Swish gently to mix, put in incubator
Clean up.  Suck up any media spilled, or any leftovers in the containers.  if the remainder of the media is not enough to use, bleach it.  Glass tips go in the sharps.  Close the hood, turn the UV on, sign out.  Take out the garbage, no matter how much/little there is.
Gel electrophoresis
Introduction
Gel electrophoresis separates pieces of DNA by length.
Materials
Agarose gel, 1 lane per sample, plus at least 1 lane for the ladder.
If you just poured the gel, make sure it has had at least 30 minutes to set.
If you have a large gel but only need a few lanes, cut out just a piece of it with a razor blade.  Cut straight!
Samples to run
6X NEB Purple Loading Dye
Molecular weight standard (aka "ladder")
Commonly available ladders are Hyperladder I and NEB 2-Log.
Gel box, lid, leads
Electrophoresis power supply
1X TAE buffer, enough to fill the gel box.
Procedure
Prepare your samples
If your samples are frozen, thaw them completely, flick or vortex to mix, then pulse down in the microfuge.
If your samples are not already in loading dye, mix them 1:6 with 6X NEB Purple Loading Dye
Add 1 µl of loading dye for every 5 µl of sample.
Pro tip: If you don't want to mix loading gel with your entire sample (PCR products, for example), cut off a strip of Parafilm; pipette 2-3 ul of sample onto the parafilm; add 1 ul loading dye; mix by pipetting and load directly.
Set up the gel box
Program the voltage on the power supply.  For the small gel box, use 100V; for the large gel box, use 150V.
Program the time on the power supply.  For a small analytical gel, set the timer for 30 minutes.
For larger gels, start at 45 minutes, and then check regularly!
Attach the leads to the gel box cover.  Make sure that the red (positive) lead is attached to the side of the box farthest away from the wells in the gel.
Remember, the DNA is negatively charged and will move toward the positive terminal. A useful neumonic for remembering this is "Run to Red."
If the gel box has not been used previously that day, empty it and rinse it out with DI water.
The TAE buffer can be re-used, but if it's been sitting out for too long it evaporates and the salt concentration (and conductivity) changes.  TAE is cheap; when in doubt, replace it.
Place the gel in the gel box.
Pour TAE into the gel box until it just barely covers the gel.
The TAE's purpose is to conduct electricity; over-filling the gel box results in a larger conductive path, more current flow, and more heating (which can screw up your run.)
Check the wells to see that they are free of bubbles.  If there are bubbles, blow them out by pipetting 100 µl of TAE from the gel box into the well.
Load your samples
Load your samples. For large combs, load 10 µl; for small combs, load 5 µl.
Work quickly. The samples begin to diffuse in the buffer, leading to smeared bands.
Load the ladder in the last lane.  Load 1/2 the volume of your samples: for large combs, load 5 µl, for small combs, load 2.5 µl.
Place the lid on the gel box. Make sure it is seated on the brass contacts.
Run the gel
Press the Start or Run button on the power supply.
Make sure the power supply doesn't complain about an open circuit.  If it does, re-seat the gel box lid and press Run again.
Double-check that there are bubbles forming on the platinum wires at either end of the gel box.
Double-check that the red (positive) lead is on the side farthest from the wells.  Remember, Run to Red.
Run until the pink band is 2/3 to 3/4 of the way down the gel.  For small analytical gels, this should take 30 minutes.  For larger gels, start at 45 minutes and check regularly!
Image the gel
If the power supply is still running, press the Stop button.
Lift the lid off of the gel box. Lift the lid straight up. If you try to "hinge" it up, the lid will break.
Transfer the gel to the GelDoc.
If it's not running, start QuantityOne from the toolbar.
If necessary, click the top button on the toolbox to select the scanner.
If necessary, reset the camera (as per instructions on the GelDoc computer.)
Press the Epifluorescent Illumination button on the GelDoc.  Check the gel's position, zoom and focus.
You want to be zoomed in so that the gel fills the field of view, and focussed so that the well edges are sharp.
Close the GelDoc door. Click Auto Expose.
The Auto Expose functionality generally over-exposes my gels a bit.  If your gel is over-exposed, remove 1/3 of the exposure time, type it into the Exposure box, then click Manually Expose.
In the File menu, select, Export as JPEG.... Save your gel to the iGEM folder on the desktop.
Open Benchling.  Log in, and copy the gel to the Description page for the plasmid you're building.
If there are multiple plasmids on the gel, save it to each plasmid's Description.
Discard the gel in the biowaste box. Wipe down the gel doc with a little water and a paper towel or Kimwipe.
Annotate the gel
Immediately, before you forget what's where, annotate the gel.
List what is in each lane.
Describe whether the pattern is what you expected or not. (You should have an in silico digestion to compare it to!)
