Team:MIT/Recombinase Group Notebook Week 8
BACKGROUND
EXPERIMENTS
Synthetic Promoters microRNA Recombinase Proof of Concept Collaborations
FUTURE WORK
Circuit Design Clinical Applications
HUMAN PRACTICES
Parts Toolbox Accessible Protocols Integrated Human Practices Public Outreach Interlab Study
NOTEBOOK
Lab Notebook Parts Mammalian Parts Collection Attributions Safety
AWARDS
HP Silver HP Gold Integrated HP Basic Part
TEAM
TN 07/25 lab meeting with Linda and Ron · Benchling
TN 07/25 lab meeting with Linda and Ron
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Elizabeth Strand
Date: 2016-07-25
Monday, 7/25
Everybody uses the Ishikawa cell lines
The only enddometrial cell line is 12Z
Both of these cell lines have estrogen receptors
Compare the Ishikawa (normal emdometrium epithelial) and 12Z (endometrial)
Start with the immortal stroma cell line (tHESC) and treat it with inflammatory factors.
Looke at the literature
Write up the protocol for cell culture; then, adding the inflammatory factor to change the cells
Go and talk to Kristy Cooke
transfect the tHESC and the Ishikawa to test for fluorescent
optimization for tHESC and ishikawa
try out all the sensors without any conditions to see the basal signals
try out 6 conditions on tHESC: (may want to do it with the Ishikawa)
+/-inflammation; ER-rich
+/-inflammation, PR-rich
ER/PR conditions (mimicing the cycles) (come up with the hormone conditions)
Transformation
1 - hEF1a: L7Ae
2 - TRE: L7Ae
3 - EGSH: TP901
4 - hEF1a: attB-flipped EYFP-attP
Transformation of E. coli
Introduction
Transformation is the process of inducing chemically competent E. coli to take up DNA.
Materials
Dry bath, set to 42°C
Fill the wells in the dry bath block 1/2 full with DI water.
Ice bucket, with ice
For thawing competent cells.
DNA to transform
Could be an assembly reaction (LR, Golden Gate, etc) or a miniprepped plasmid.
If you removed it from the freezer, make sure it's entirely thawed  out.
pUC19 Transformation Control, 1 pg/µl
The pUC19 control will tell you how efficient your transformations were.
SOC growth media, at room temperature
Check to make sure it's clear and NOT CLOUDY.
Antibiotic plates, one per transformation, plus 1 Amp plate for the pUC19 control
Make sure the plates you use match the resistance cassette of the plasmid!
Competent E. coli, one tube per transformation + one for the pUC19 control
These live in the -80 in 235.
Thaw on ice 3-4 minutes.
A timer, set for 30 seconds.
Procedure
Setup
Make sure the dry bath is set to 42°C and the wells in the block are 1/2 full of DI water
Remove selection plates from the refrigerator.  Double-check that they match the selection marker on your plasmid, then place them in the 37° incubator.
Retrieve the DNA to transform.
If frozen: thaw, completely, flick a few times to mix, then pulse down in the microfuge.
Fill an ice bucket with ice.  Retrieve one tube of competent E. coli per transformation from the -80 and thaw on ice, 3-4 minutes.
00:03:00
While the transformation tubes are thawing, label their tops with something descriptive.  Record the labels here:
A
B
C
D
E
F
G
H
1
puc-19L
2
puc-19TC
3
Pdest_mCherry
4
pDest
Table1
Transformation
Add 2 µl DNA from each reaction to a tube of competent cells.
Immediately after adding the DNA to each tube, stir the cells a few times with the pipette tip.
Add 1 µl of the pUC19 transformation control to the positive control tube.
Incubate on ice for 30 minutes.
00:30:00
Heat shock the cells for exactly 30 seconds in the 42° heat block. (Yes, set a timer.)
00:00:30
Place back on ice for 2 minutes.
00:02:00
Add 250 µl SOC to each tube.
Tape the tubes to the platform of a shaker at 37°C and shake at 270 RPM for 60 minutes.
01:00:00
Plating
Label the selection plates using the labels you recorded above.
Shake ~10 plating beads onto each plate.
Pipette 100 µl of each transformation onto the corresponding plates. NOTE: if you are using DNA from a golden gate reaction, see the golden gate protocol for instructions to dilute your sample.
-----Golden Gate: Plate only 10 ul of the outgrowth in a 200 ul puddle of water, or you will get a lawn of colonies.
Cover the plates and shake the beads around to spread the cells out.
Dispose of the beads by tapping them into the waste container.
Incubate the plates upside down overnight in the 37° incubator.
Don't incubate for more than 18-24 hours.
Compute transformation efficiency
Count the colonies on your positive transformation plate.
If there are many many colonies, then hooray!  You had a great transformation.  Just estimate.
Divide the number of colonies by the fraction of the transformation you plated.
So, if you resuspended your transformation in a total volume of 300 ul, then plated 100 ul, multiply the number of colonies by 3.
Transformation efficiency is expressed in colonies per microgram pUC19.  Multiply the number of colonies by the appropriate conversion factor.
So if you transformed 1 picogram of pUC19 DNA, multiply by 106.
Record your transformation efficiency in your (daily) lab notebook.
CF 7/26/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Elizabeth Strand
Date: 2016-07-26
Tuesday, 7/26
Plate layout:
A
B
C
D
1
Untransfected control
300ng EGSH:mKate 100ng pDONR 100ng hEF1a:BFP 0uM PonA
300ng EGSH:mKate 100ng hEF1a:VgEcr 100ng hEF1a:BFP 0uM PonA
300ng EGSH:mKate 100ng hEF1a:VgEcr 100ng hEF1a:BFP 0.5uM PonA
2
300ng EGSH:mKate 100ng hEF1a:VgEcr 100ng hEF1a:BFP 1uM PonA
300ng EGSH:mKate 100ng hEF1a:VgEcr 100ng hEF1a:BFP 5uM PonA
300ng EGSH:mKate 100ng hEF1a:VgEcr 100ng hEF1a:BFP 10uM PonA
Table1
Transfection Mixture (Control - well 2)
A
B
1
Media
44 uL
2
Viafect
1 uL
3
EGSH: mKate
3 uL
4
pDONR
1 uL
5
hEF1a: BFP
1 uL
6
TOTAL
50 uL
Table4
Transfection Mixture (Experiment - 5 wells)
A
B
1
Media
220 uL
2
Viafect
5 uL
3
EGSH: mKate
15 uL
4
hEF1a: VgEcr-P2A-RXR
5 uL
5
hEF1a: BFP
5 uL
6
TOTAL
250 uL
Table5
Pouring LB Agar Plates
Introduction
The media must be autoclaved before you pour plates.  This is not something for iGEMers to do; ask an advisor.
Materials
One sleeve of empty plates per 500 ml of media
Make sure you are using the Weiss lab's bacteriological plates, not tissue culture plates.
LB-Agar media, either solid or already melted
1000X stock solution of the antibiotic
(Amp)icillin, (Kan)amycin, (Chlor)amphenicol
Procedure
Making media
Make sure you don't already have a bunch of the plates already in the fridge
Fill a 1-liter bottle with 500 ml of water.
Weigh out XXXX of powdered LB-Agar (check the bottle for the weight.)
IF YOU MAKE A MESS OF THE WEIGH STATION, CLEAN UP AFTER YOURSELF.
Ask an instructor to autoclave your media.
Melting media
Solid media must be melted in the microwave. Be careful -- it gets very hot.
Loosen the cap to prevent the bottle from exploding. 1 full turn of the cap is sufficient.
Microwave 2-3 minutes, watching carefully, until the media begins to melt and bubble.
Continue microwaving in short bursts until the media is completely melted.
IF YOU MAKE A MESS OF THE MICROWAVE, CLEAN UP AFTER YOURSELF.
Cool the media to <= 60°C before adding the antibiotics.
You can either swirl it under a cold tap until you can handle it without burning yourself, or you can leave it for a few hours in the 55°C water bath.
To check the media temperature, swirl the bottle, wait ~30 seconds and measure with an IR thermometer. (Pew pew.)
Pouring plates
Add antibiotic.
Unless otherwise noted, it is 1000X -- so for 500 ml of media, add 500 µl of antibiotic.
Note: If you're pouring Golden Gate plates, XGal is 500X! (for 500 ml of media, add 1000 µl of XGal)
Pour all of the media you add these to. Refreezing and thawing has unpredictable results on the amount of antibiotic destroyed by the heat, so to avoid over/under dosing don't refreeze. 500 ml makes a sleeve of plates.
Tighten the lid to avoid spills and swirl the bottle to mix
Use a seriological pipette to add 20ml to each plate.
Use sterile practices
If you get bubbles, use the pipette to try and suck the air out of the bubbles, this removes them
20ml is a rough estimate, what is important is that you get a solid layer with no holes.
Mark the plates with a stripe of marker up the side, based on which antibiotic(s) you added:
Blue: Amp
Red: Kan
Green: Cm
Black: Non-antibiotic additive (like X-gal)
Dry on the bench overnight.
Re-stack the plates upside-down and label the sleeve with the antibiotics, your initials and the date.
DNA Transfection into HEK Cells
Introduction
Get started by giving your protocol a name and editing this introduction.
Materials
OptiMEM (Serum-free media)
Viafect
DNA from midipreps
Procedure
Phase I: Calculations
Select DNA to be transfected and determine the concentration through Nanodrop
Determine the volume of DNA required to have 0.5 ug in reaction mixture.
Ex. for a 521 ng/ul stock of DNA you would need approximately 1 uL. If the calculated volume you need is less than 0.5 uLr (i.e. hard to pipette) you may want to dilute your DNA solution so that you can get an appropriate volume.
Determine the amount of Viafect to use from the following table:
A
B
C
D
E
F
G
H
1
Volume of cells in media (per well)
Total volume transfection complex (per well)
Amount of DNA (per well)
Amt of Viafect Reagent for 1.5:1 Reagent:DNA
2:1
3:1
4:1
6:1
2
500 uL
50 uL
0.5 ug
0.75 uL
1 uL
1.5 uL
2 uL
3 uL
Table1
Calculate the amount of serum free DMEM media required to get a reaction volume of 50 uL
Ex. if you are adding 1 uL of DNA and 3 uL of Viafect, you would want to add 46 uL of DMEM
Phase II: Reaction Mixture
This phase consists of mixing the reactants required to create the transfection complex. Do so for all ratios of reagent to DNA above - add media first, then reagent, then DNA.
Let the reaction mixture sit for 10-20 minutes
Closer to 20 min is better
Phase III: Transfection
Retrieve cells from incubator. Make sure to leave this step for last because you don’t want to keep the cells at room temperature for too long.
Add entire 50 uL transfection complex to each well in the well plate and store your cells back in the incubator.
Phase IV: Checking for Fluorescence
Close off lights in room and turn on the microscope’s light.
Apply the appropriate fluorescence filter.
Check for fluorescence.
Overnight liquid cultures (picking colonies)
Introduction
Overnight cultures are used to prepare miniprep DNA.
Materials
The plate from which you are picking colonies
15 ml round-bottom polystyrene tubes, one per culture
The ones with the snap caps, NOT conical tubes with screw caps
5 mL LB per culture
A container that can hold 5ml x the number of cultures
For a modest number of minipreps, a 50 ml conical tube works well.
For larger minipreps, use a sterile bottle (100 ml is frequently useful.)
Antibiotic stock, 1000X
Procedure
Materials Setup
Warm up the LB to at least room temperature (if it came from the fridge), but not warmer than 37°C
Label one round-bottom culture tube for each miniprep.  Use "NAME-1, NAME-2, ..."etc for the naming convention, where NAME is a shortened name of the plasmid (eg, "hEF1a:mKate").
Your impulse is to just use number, or initials and number, but trust me -- you will want to be able to identify this tube in three weeks when you've forgotten what you were doing.
Using a sterile pipette, transfer 5 ml of LB to the mixing container for each culture PLUS 5 ML.
Add antibiotic stock to a final concentration of 1X (1 µl stock for each 1 ml in the mixing container.)
Cap tightly and mix well.
Culture Setup
Using a sterile pipette, transfer 5 ml of LB+antibiotic to each round-bottom culture tube.
If you are making cultures with different antibiotics, take care that the right media goes in each tube.
Squirt ethanol on a pair of foreceps and wipe dry with a Kimwipe.
Use the foreceps to pick up a sterile 200µl pipette tip, scrape a colony off of the plate, and drop the pipette tip in the corresponding tube.
Repeat for each tube.
Transfer to an incubating shaker at 37°C and incubate 14-16 hours.
Don't over-grow too badly, or your yield will suffer.
If you need to grow longer, you can grow at 30°C instead for 20 hours.
CF 7/27/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Maya Kaul
Date: 2016-07-27
Wednesday, 7/27
Sent for sequencing:
1,2 1-4, 2-1, 2-2, 3-1, 3-3, 4-2, 4-3
RecombinaseTOP.jpg
RecombinaseBOTTOM.jpg
Miniprep Nanodrop Concentrations:
A
B
1
1-1
576
2
1-2
522.9
3
1-3
493.7
4
1-4
420.4
5
2-1
445.6
6
2-2
753.8
7
2-3
388.5
8
2-4
348.6
9
3-1
521.7
10
3-2
466.5
11
3-3
378.8
12
3-4
104
13
4-1
499.2
14
4-2
546.3
15
4-3
461.3
16
4-4
237.5
Table1
recombinase_digest_july27.JPG
recombinase_digest2_july27.JPG
Miniprep
Introduction
The miniprep uses silica gel to isolate plasmid DNA from an E. coli culture
Materials
Buffer P1 (resuspension buffer)
Retrieve from refrigerator.  If you are opening a new miniprep kit, add the RNAse and LyseBlue reagent and check the box on the cap.
Buffer P2 (lysis buffer)
Open the cap and look at the lysis buffer.  Swirl it around.  If it appears cloudy, the SDS has fallen out of solution; warm it for a few minutes in the 55°C water bath.
Buffer N3 (neutralization buffer)
Buffer PB (binding buffer)
Buffer PE (rinse buffer)
Make sure the "Ethanol added?" box has been checked.  If you are opening a new miniprep kit, add absolute ethanol as per the kit instructions and check the box on the cap.
Buffer EB (elution buffer)
Miniprep waste container
Miniprep buffers contain salts that can't go down the sink.
Per miniprep: two microcentrifuge tubes and one blue spin column, with collection tube.
Procedure
Harvest and resuspension
For each culture, label two microcentrifuge tubes on the cap and one blue spin column on the side.
The spin columns should be in their (cap-less) collection vials.
Pipette 1.6 mL of each culture into the corresponding microcentrifuge tubes.
(This is for a 1.7 ml microcentrifuge tube; set your pipettor to 800 ul and do two transfers.  If we ever go back to 2-ml tubes, do 1.9 ml (2x950 ul))
Centrifuge at maximum speed (10,000 or 13,000xg) for three minutes.
Aspirate the supernatant, or pour it off into the bleach bucket.
Pipette ANOTHER 1.8 ml of each culture into the corresponding microcentrifuge tubes.
Centrifuge at maximum speed for three minutes.
While the centrifuge is running, move the remaining cultures to 4degC.
Aspirate the supernatant off with the bench aspirator.  Be careful not to disturb the pelleted E. coli.
We use an aspirator here because the less extra salt and protein we put in the miniprep, the better the yield is.
Add 250 µl Buffer P1 to each tube.
Resuspend the E. coli pellet.  The preferred way is with the roto-mixer at the other end of the lab.
Alternately, if you have just a few tubes, you can resuspend on a vortex.
Make sure to resuspend fully and thoroughly. The resulting suspension should be smooth and cloudy; if there is particulate matter floating around, vortex some more.
Lysis
Add 250 µl Buffer P2 to each tube.
Work quickly; the lysis step should take less than 5 minutes.
Snap the tubes closed and invert them 4-6 times, until the tube is thoroughly mixed and the entire solution turns blue.
If you have many many tubes, you can stack a second tube rack on top of them and invert the entire thing.
Add 350 ul Buffer N3 to each tube.
Snap the tubes closed and invert 4-6 times, until the solution is thouroughly mixed and no longer blue.
The solution will become cloudy or flocculent.
Centrifuge on high speed for 10 minutes.
Separation
Remove the tubes from the microcentrifuge, being careful not to disturb the white pellet.
Using P-1000 micropipettor set to 850 ul, carefully transfer the supernatant from each centrifuge tube to the corresponding blue spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Don't forget to put the lid on the rotor! Some of the salts get aerosolized because the spin columns don't have caps.
Pour the flow-through from each column into the miniprep waste container.
Pipette 500 ul of Buffer PB onto each spin column.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Pipette 750 ul of Buffer PE onto each spin column
Wait 1-3 minutes.
This allows some of the salt that's still bound to the silica matrix to resuspend in the buffer.
Centrifuge the spin columns for 30 seconds at maximum speed.
Pour the flow-through from each column into the miniprep waste container.
Return each spin column to its collection tube and centrifuge an additional 1 minute at high speed.
This removes every last trace of buffer PE; the ethanol can screw up downstream steps.
Transfer each spin column to a clean labelled microcentrifuge tube.
Pipette 50 ul of Buffer EB onto the center of each column.
The volume of EB is comparable to the volume of silica gel matrix; if you pipette down the side, you might not get the entire transfer to the matrix.
Wait 1-3 minutes.
This gives the DNA a chance to dissociate from the silica matrix.
Centrifuge the spin columns, in their collection tubes, for one minute at maximum speed.
Proceed directly to analyze the samples on the Nanodrop.
Diagnostic Restriction Digestion
Introduction
A diagnostic restriction digest helps identify correctly assembled clones from incorrect clones.
Materials
200 ul PCR strip tubes, one per reaction
Restriction enzyme (chosen below), 1 ul per reaction.
10X restriction enzyme buffer (chosen below), 1 ul per reaction.
Procedure
Choose A Good Restriction Enzyme
Using Benchling, choose a restriction enzyme that meets the following criteria:
- Cuts at least once in the insert (for a pEXPR, either the promoter or the gene.)
- Cuts at least once in the backbone.
- Gives bands that aren't too large (> 8 kb) or too small (< 200 bp).
- Gives a band pattern that is significantly different from the expected error mode.
* For LR reactions, the most common error is a pDEST that slipped through the selection.
- Begin in the "Brian's Favorites" list, then expand to the main Weiss lab list.
- If you can't find a single enzyme that gives an acceptable band pattern, choose two enzymes that give a acceptable band pattern when used together.
* This double-digest is subject to buffer compatibility, outlined below.
Record your enzyme choice on the plasmid's Description page.
Benchling will tell you the enzyme's buffer compatibility and active temperature. Record the buffer in which the enzyme is most active.
If there are multiple bufers in which the enzyme is equally active, choose in this order: Buffer 3.1, Buffer CS, Buffer 2.1, Buffer 1.1.
Set up the restriction digest
Retrieve the minipreps and the appropriate 10X buffer concentrate from the freezer.  Thaw on the benchtop or in your fingers.
Label the PCR tubes with your initials and an incrementing number.
ie: BT-1, -2, -3, -4
Vortex the minipreps and the 10X buffer concentrate briefly, then pulse down in the microfuge.
For each miniprep, set up a PCR tube containing the following in order:
- 5 ul enzyme-quality H2O
- 1 ul enzyme buffer
- 3 ul miniprep DNA
- 1 ul enzyme
Remove the enzyme from the freezer for as little time as possible.
I have specified an "arbitrary" 3 µl volume of miniprep DNA; this should be fine as long as your miniprep concentration is >= 100 ng/ul.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
Incubate at the appropriate temperature for at least 1 hour and more more than 16 hours.
If the enzyme's active temperature is 37°C, use the 37°C plate incubator.
Stop the reaction by adding 2 ul of 6X NEB purple gel loading dye to each reacti on.
Flick the strip tubes a few times to mix the reaction, then pulse down in the strip tube microfuge.
PAUSE POINT: The reaction can be stored almost indefinitely at room temperature once it's been stopped.
Proceed to gel electrophoresis.
Sequencing protocol
Introduction
Preparing sample to send to Genewiz for sequencing Sequencing Pick-up Times: 9:30am, 3:00 pm, 5:30 pm
Materials
DNA stock
Nuclease-free water
Primer (5nM stock)
One microcentrifuge tube for each DNA plasmid
2 PCR tubes per plasmid (must be in strips of eight, round up if needed!)
Procedure
Diluting the DNA
For orders with <48 samples, use 8-strip PCR tubes. Label your tubes on the side vertically with "MI" and sample number (01, 02, 03, etc.). Use image below as guideline. Each plasmid you want to sequence will require 2 tubes, one for the forward primer and one for the reverse primer. Use "MI" instead of the "GW" in the picture.
Dilute your sequencing primer to 5 µM (pmol/µl) using water. You will need 5 µl for each sequencing reaction. Note, the primers in the tubes from IDT have not been diluted! Use already diluted stock (should be in regular microcentrifuge tubes).
Using the recorded concentration, dilute the DNA to the template concentration in 10 µl as detailed below. For example, a 5 kb plasmid would need a template concentration of ~50 ng/µl. Final volume for your working stock should 30 uL. Please make dilutions in water or Tris.
Add 10 µl of diluted template DNA to each tube.
Add 5 µl of diluted primer to each tube. Be careful about putting the forward and reverse primers in the correct tubes! Remember, for each DNA template, there should be two tubes - one with reverse primer and the other with forward primer
Vortex briefly
Log into genewiz. Username: igem-sequence@mit.edu Password: igem2014citrus
Click "create sequencing order"
Sevice Priority: click on "standard"
Create Order by: click on "online form"
Sample Type: click on "Pre-mixed"
Enter in how many samples you are sending in and click "Create new form"
Enter in descriptive names for your DNA (this is to help you remember when you get the results back), the DNA type, the DNA length (get this from your construct on benchling), the primer name (again, this is more for you). When you are done hit save and next.
It will give you an list of what you have entered and a price for each reaction. Hit "next step"
The payment information field should be autofilled. There has been an issue recently with the PO box number not being entered. If this is the case, click "credit card" (next to "payment info")  and then click "PO" to bring you back to the original form. Now you should be able to enter in a PO number. Open up a previous submitted order (there should be some saved to the desktop called "sequencing" or something like that) and copy over the PO number. PO Number:  5510061907
Hit "next step". Once it gives you a submitted order form, print 2 copies and pick them up from the printer (exit lab, through glass doors, on your left)
Put your PCR tubes in a large falcon tube.
Put the falcon tube in a ziploc bag (located in Qiagen column drawer) and put in one order form. Label the bag with something descriptive ("Weiss Lab MIT iGEM Recombinase 6/30/16" for example)
Put the bag in the Genewiz pickup box (located by the elevators, make sure you put in the genewiz box)
Open up the weiss lab orders spreadsheet (bookmarked on the computer). You need to request access to the spreadsheet if it is your first time ordering. Once you request access, email Brian so he knows that you have sent in a request.
Click the tab at the bottom called "genewiz".
Fill out the form. Make sure you indicate that the order is from iGEM so the right account can be charged.
You did it! YAY! Genewiz should get back to you pretty quickly, generally between 12-24 hrs.
JG 7/28/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Maya Kaul
Date: 2016-07-28
Thursday, 7/28
Midiprep:
3-3: EGSH_TP901
4-3: hef1a-attb-flipped eyfp-attp
1-4: hef1a-L7ae
2-1 TRE-L7AE
Sample Preparation for Flow Cytometry
Introduction
Materials
Versene
Clear liquid on Brian's side of drawer
Trypsin
Remember to put Trypsin and cell media into dry bath to warm them up before starting
HEK cell media
Flow cytometry tubes
Tubes with Blue filters on top
One for each well
Ice Bucket
Glass pipette tips
P200 and P1000 pipette tips
Procedure
Preparing Samples
Retrieve cells from incubator
Look at the wells under the EVOS microscope by the Lu Bench to check the cells and look for fluorescence
For each well:
- Aspirate the media (make sure the tip does not touch the bottom of the well)
- Pipette 200 uL of versene slowly down the side of the well
Again, for each well:
- Aspirate the versene
-Add 3ml of PBS and shake over the media (let sit for a couple minutes), then aspirate off PBS.
- Pipette 200 uL of trypsin and wait three minutes
Add 500 uL of media to each well
Label flow cytometry tubes and put in an ice bucket
Pipette up and down in circles, and then add entire mixture of media and trypsin to a flow cytometry tube through the filter
Centrifuge at 2300 rpm for 4 min
Aspirate out the supernatant for each tube without disturbing the cell pellet
- Pellet is very small and hard to see
- Don't put tip lower than the line around the bottom of the tube so that you don't disturb the pellet
Add 500 uL of versene to each tube (making sure to avoid bubbles, as it can clog the cytometer)
Go find Brian, and he will load your samples into the cytometer
Glycerol Stock
Introduction
While minipreping, remaining miniprep culture should be have been kept in the 4 degree C fridge. After sequencing and confirmation, the miniprep culure of the cells you've chosen to midiprep should be made into a glycerol stock and put into the -80 iGEM glycerol box.
Materials
Cryo Tubes (orange caps, in bottom drawer)
50% Glycerol solution (on shelves above bench) 500uL (for each miniprep culture)
Miniprep 500uL
Label Printer and other components
P1000 and tips
Procedure
Initial Set Up
First make sure that the tape printer located on top of the computer tower has enough tape. If out of tape, contact Brian for more.
Take our your miniprep cultures and aspirate the ones that you will not be using for midiprep
Check that you have enough glycerol. If out, contact Brian.
Take out the number of cryo tubes equal to the amount of minipreps cultures you'll be using for midiprep. If out, check the Weiss Lab cabinets located in hall.
Labels
Labels should be made first before making the glycerol stock so you don't confuse tubes.
The label making program, P-touch Editor 5.0, can be found on the desktop of the computer
If the iGEM label template is not open, click File--->Open---->iGEM 2016 Template Label
Follow template of label. Fill in date, your initiails, plasmid name, cell type, and add 'iGEM 2016'
Once finished, print the labels using the printer named Brother PT-2430PC
The labels are peel stickers. Stick labels onto cryo tubes
Making Glycerol Stock
Pipette 500uL glycerol into each cryo tube.
Pipette 500ul of each miniprep culture into their respective cryo tubes. Remember to switch pipette tips each time
Cap tubes are store in -80 degree freezer in iGEM glycerol stock box. Should be located in the bottom of the freezer.
Store remaining miniprep cultures in 4 degree fridge until used to grow midiprep culture
JG 7/29/16
Made with Benchling
Project: iGem 2016 _ Recombinase subgroup
Authors: Maya Kaul
Date: 2016-07-29
Friday, 7/29
A
B
1
hEF1a: L7Ae
787.5
2
TRE: L7Ae
650.1
3
EGSH: TP901
1418.1
4
hEF1a: attB-flipped eYFP-attP
825.1
Table1
Q1-EGSH-Q2: 0.088 g
Q1-EGSH-Q2: 0.159 g
Midiprep
Introduction
Purifying large amount of DNA to tranfect mammalian cells
Materials
Qiagen MidiPrep
Procedure
Resuspending Primers
Introduction
Primers arrive from IDT dehydrated; to use them, you need to resuspend them in TE to a known concentration.  By convention, the stock concentration (in the blue-capped IDT tube) is 100 µM; the working concentration depends on the application:  - For PCR, the working concentration is usually 10 µM  - For sequencing, the working concentration is usually 5 µM
Materials
Nuclease-free TE
We resuspend in TE to help supress nuclease activity that would degrade the primers.
Dehydrated primers from IDT
One extra epi tube per primer
...for the working stock.
Procedure
Resuspend the dried primer
Label the top of the oligo tubes.  I recommend the group initials and a number.
In the little microfuge, spin the (dry) oligos briefly.
Sometimes the freeze-dried primers flake off the bottom of the tube.
Determine how many nanomoles of primer are in the tube.
This is the bottom line on the tube label.  As an example, the BT-01 primer says
7.3OD = 33.4 nmol = 0.23 mg. <-- the number of nanomoles is "33.4".
Add 10 µl of nuclease-free TE for each nanomole of primer.
For the BT-01 primer, this volume is 334 µl.
Vortex briefly.
Check the bottom of the tube to see if the primer is fully resuspended.  If not, vortex again.
Pulse spin the primers.
Make a working stock
Label the top of the epi tube the same as the oligo.
Determine the working stock concentration that you want.  Label the side of the epi tube with the working concentration.
Make 100 µl of working stock.
For example, if you're making a 10 µM working stock, dilute 10 µl of the concentrated stock into 90 µl of TE.
Vortex briefly.
Freeze both the concentrated and working stocks in the primers box.
PCR
Introduction
PCR (the polymerase chain reaction) uses a polymerase and DNA oligonucleotides to amplify a length of double-stranded DNA.  For general purpose PCR, I like to use the Q5 polymerase from NEB: the product documentation is here .
Materials
200 µl thin-wall PCR strip tubes, one per reaction
Forward and reverse primers, 10 µM working stock
Template DNA, 1 ng/ul
Make SURE that you have diluted out the template DNA appropriately!  Miniprep-concentration template will screw up your PCR!
Q5 master mix, thawed on ice
Ice bucket with ice
Empty 10 µl or 200 µl tip box
Gel for gel electrophoresis
Parafilm
6X NEB Purple Loading Dye
Procedure
Set up your PCR program
Use tmcalculator.neb.com to compute the annealing temperature Ta of your reaction based on your primers' melting temperatures.
Make sure to select Q5 2X Master Mix under "Polymerase/Kit"
Compute the extension time Tex based on your predicted product length: 30 seconds per kilobase.  So if my amplicon is 1500 bp, my extension time is 1'30"
Program the thermocycler with the following (there should be a "Q5" program you can easily edit):
A
1
Heated lid --> 105°C
2
HOLD at 98°C
3
98° for 30"
4
Cycle 30 times:
5
--  98° for 5 seconds
6
-- Ta for 15 seconds
7
-- 72° for time Tex
8
Close cycle
9
72° for 2'
10
Store at 8°C
Table1
Start the program on the thermocycler. It will heat the lid, heat the block to 98° then hold at that temperature.
Set up the PCR reactions
Use the following table to fill in your template and primers for each reaction you're running.  For a 20 µl reaction, use 1 ul of forward primer (10 uM), 1 ul of reverse primer (10 uM), 1 ul of template (1 ng/ul), 7 ul nuclease-free water, and 10 ul 2X Q5 master mix.
A
B
C
D
1
Q1_PhlF_QX
2
Fwd Primer
1 ul BT28 PhlF-F
3
Rev Primer
1 ul BT29 PhlF-R
4
Template
1 ul pENTR L1_PhlF_L2
5
Water
7 ul
6
2X Q5 MM
10 ul
Table2
Take the grey platform off of the tip box.  Fill the top box with ice, then replace the grey platform.  Fill the box with water until the level is just below the grey platform.
Label the PCR tubes.
Set up the PCR reactions.  Start with the water; end with the Q5 master mix.
Working quickly, re-cap the PCR reactions.  Flick several times to mix, then pulse spin in the tube microfuge.  Return the tubes to ice.
Carry the tubes to the pre-heated thermocycler.  Put the tubes in the block, close the lid and then press the button to continue through the hold and run the rest of the program.
Analyze the PCR reaction with gel electrophoresis
Prepare or secure a gel.
Slice a small piece of parafilm off of the roll.
Spot 4 ul of water in a droplet on the parafilm.  Add 1 ul of the PCR reaction and 1 ul of loading dye.
Pipette up and down with a 5 ul pipettor, then load in a well on the gel.
Run the gel.
If there is a single clean band, proceed to clean the reaction up with a QIAgen PCR cleanup column.
If there are multiple bands but you see the one you want, proceed to a gel extraction of the band you want.
Gel Extraction
Introduction
Get started by giving your protocol a name and editing this introduction.
Materials
QIAgen gel extraction kit
Purple columns
Buffer QG
Buffer PE
Buffer EB
Isopropanol
3M Sodium Acetate, pH=5.0
Microcentrifuge tubes
Procedure
Setup
Set a dry bath to 50°C and fill the holes in the metal block with water.  Transfer some Buffer EB (30 ul for each purification plus an extra 30 ul) to a microcentrifuge tube and place it in the dry back to warm.
Excise gel
On the blue-light transilluminator, cut out the gel band and put it in an eppendorf tube.
Weigh the gel band.
TARE the balance with an empty eppendorf tube.
Weigh the tube with the gel band in it.
Gel extraction
Add 6 volumes of Buffer QG to 1 volume of gel.  (Compute volume assuming 100 mg of gel ~ 100 ul.)
Incubate at 50°C until the gel is disolved, about 10 minutes; vortex briefly every 2-3 minutes.
Add 10 µl 3M sodium acetate pH 5.0.  Vortex briefly.
Add 1 gel volume of isopropanol.
Apply a maximum of 750 µl of the disolved gel to the QIAquick column and centrifuge for 30 seconds at maximum speed.
Pipette the flow-through back into the column and centrifuge again.  Discard the flow-through.
If needed, apply the rest of the disolved gel to the column.  Centrifuge; then re-apply and centrifuge again.  Discard flow-through.
Add 500 µl of buffer QG and centrifuge 30 seconds.  Discard flow-through.
Add 750 µl of Buffer PE to the column. Incubate at room temperature for 3-5 minutes, then centrifuge for 30 seconds.
Discard flow-through and centrifuge again for 1 minute.
Move column to a clean microcentrifuge tube.
Add 30 µl of the warm EB buffer to the center of the column.
Incubate in the 50-degree dry bath for 3 minutes.
Centrifuge for 1 minute.
Measure the concentration using the nanodrop (make sure to vortex your samples before).
