<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navdiv-wrapper"><DIV class="navdiv"><UL><LI><A href="https://2016.igem.org/Team:MIT/Description">BACKGROUND</A></LI><LI><A href="https://2016.igem.org/Team:MIT/Experiments">EXPERIMENTS</A><DIV class="submenu"><A href="https://2016.igem.org/Team:MIT/Experiments/Promoters ">Synthetic Promoters</A><A href="https://2016.igem.org/Team:MIT/Experiments/miRNA">microRNA</A><A href="https://2016.igem.org/Team:MIT/Experiments/Recombinases  ">Recombinase</A><A href="https://2016.igem.org/Team:MIT/Proof">Proof of Concept</A><A href="https://2016.igem.org/Team:MIT/Collaborations">Collaborations</A></DIV></LI><LI><A href="https://2016.igem.org/Team:MIT/Diagnosis_and_Future_Implications">FUTURE WORK</A><DIV class="submenu"><A href="https://2016.igem.org/Team:MIT/Genetic_Circuit">Circuit Design</A><A href="https://2016.igem.org/Team:MIT/Clinical_Applications">Clinical Applications</A></DIV></LI><LI><A href="https://2016.igem.org/Team:MIT/Human_Practices">HUMAN PRACTICES</A><DIV class="submenu"><A href="https://2016.igem.org/Team:MIT/Mammalian_synthetic_biology_kit">Parts Toolbox</A><A href="https://2016.igem.org/Team:MIT/accessible_protocols">Accessible Protocols</A><A href="https://2016.igem.org/Team:MIT/Integrated_Practices">Integrated Human Practices</A><A href="https://2016.igem.org/Team:MIT/Engagement">Public Outreach</A><A href="https://2016.igem.org/Team:MIT/InterLab">Interlab Study</A></DIV></LI><LI><A href="https://2016.igem.org/Team:MIT/Notebook">NOTEBOOK</A><DIV class="submenu"><A href="https://2016.igem.org/Team:MIT/Notebook">Lab Notebook</A><A href="https://2016.igem.org/Team:MIT/Parts">Parts</A><A href="https://2016.igem.org/Team:MIT/Part_Collection">Mammalian Parts Collection</A><A href="https://2016.igem.org/Team:MIT/Attributions">Attributions</A><A href="https://2016.igem.org/Team:MIT/Safety">Safety</A></DIV></LI><LI><A href="https://2016.igem.org/Team:MIT/Awards">AWARDS</A><DIV class="submenu"><A href="https://2016.igem.org/Team:MIT/HP/Silver">HP Silver</A><A href="https://2016.igem.org/Team:MIT/HP/Gold">HP Gold</A><A href="https://2016.igem.org/Team:MIT/Integrated_Practices">Integrated HP</A><A href="https://2016.igem.org/Team:MIT/Basic_Part">Basic Part</A></DIV></LI><LI><A href="https://2016.igem.org/Team:MIT/Team">TEAM</A></LI></UL></DIV></DIV><TITLE>Collaborations</TITLE><H1 style="color:#F20253; text-align: center; font-size: 40px; line-height: 40px;">Collaborations</H1><H1 style="color:#FFFFFF; background-color:#F20253; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: Trebuchet MS">Mammalian cell culture in microfluidic devices</H1><H2 style="color: #F20253; text-decoration:underline; font-family: Trebuchet MS;">Purpose</H2><A style="text-decoration: none; color: #000000; float: right; margin: 15px"><DIV style="width: 199px; text-align: center;"><I>Microfluidic device fabricated by BU Hardware iGEM Team.</I></DIV></A><P>
    Integration of microfluidic devices has advanced considerably in quantitative biological research [1][2]. The timing of the fluid flow can be regulated with high accuracy because of the short response time of in-chip membrane valves (as quick as 1 ms)[1]. Moreover, with the aid of hydrogel, cells growing in microfluidics can be structured in three-dimensional geometries [1]. Therefore, cell culture performed in microfluidic devices could resemble in vivo conditions due to tight control of the microenvironment [2].  On the other hand, automation is another advantage of microfluidic technology. Instead of manual intervention, programmable fluid flows can automatically change chemical conditions of the culture environment. This feature would be particularly useful to examine dynamically changing systems [1]. The ability to precisely control fluid flows and chemical species as well as the automation of microfluidic technology are highly beneficial to future development of our project. Thus, the MIT iGEM team decided to collaborate with the Boston University Hardware iGEM team to develop a protocol for mammalian cell culture in microfluidic devices .
    </P><P style="font-weight:bold">How does microfluidics benefit our project?</P><OLFV><LI>Automatically simulates estrogen and progesterone cycle in vitro cell culture</LI><LI>Resembles dynamically changing estrogen and progesterone concentrations</LI><LI>Dynamically monitors cell response and output gene expression</LI><H2 style="color: #F20253; text-decoration:underline; font-family: Trebuchet MS;">Collaboration</H2><P>
        We collaborated with the <B>Boston University Hardware iGEM Team</B> to culture mammalian cells in their microfluidic devices.  We started with the simplest experiment, which is <B>growing HEK293 (Human Embryonic Kidney) cells in a microfluidic device</B>. We performed the experiment with <B>two different device models</B>. 
    </P><A style="text-decoration: none; color: #000000; float: right; margin: 15px"><DIV style="width: 299px; text-align: center;"><I><B>Figure. </B>Microfluidic device model 1a = b = c = 400um ; Material: PDMS. Red for controlling air exchange. Blue for fluid channel. </I></DIV></A><H3 style="font-family: Trebuchet MS; font-weight:bold; font-size:15px;">Model 1</H3><P style="text-decoration:underline;">Microfluidic device</P><P>
    The device had 2 sets of channels, one for fluid flow (labeled blue in the diagram), and another one for air exchange (labeled red in the diagram). The first model did not have a chamber or specific region for cell culturing. The inlet and outlet were connected by a tube with uniform dimensions. This model, however, had rectangular cell traps along the channel.
    </P><P style="font-family: Trebuchet MS; text-decoration:underline;">Cell culturing protocol</P><OL><LI>Harvest the cells were using the standard culturing protocol, and then resuspend in media to achieve 2E4cells/cm^2 seeding density.</LI><LI>Sequentially sterilize and wash the channel with 1ml of 70% ethanol and 1xPBS by directly pipetting into the inlet.</LI><LI>Through the same inlet, pipet 1ml of cells solution (1E6 cells/ml).</LI><LI>Incubate the whole device with seeded cells at 37 degrees Celcius.</LI></OL><A style="text-decoration: none; color: #000000; float: right; margin: 15px"><DIV style="width: 299px; text-align: center;"><I><B>Figure. </B>Microfluidic device model 2.a = 3250um; b = 400um; volume of the chamber = ~ 4.4ul. Material: PDMS.</I></DIV></A><H3 style="font-family: Trebuchet MS; font-weight:bold; font-size:15px">Model 2</H3><P style="font-family: Trebuchet MS; text-decoration:underline;">Microfluidic device</P><P>Since we couldn't detect any cells under 10X light microscope (24 hrs post seeding) in the first model, we proposed several modifications to the device after doing some literature research, including:</P><OL><LI>Creating a <B>diamond-shape chamber</B> specifically for cell adhesion. The increase in cross-section area along the channel reduces the flow rate of the fluid, making it easier for the cells to stick to the surface of the chamber</LI><LI>Using a <B>syringe pump</B> to continuously <B>replenish media inside the chamber</B> after cell adhesion, preventing the media from drying out inside the chamber - which happened in model 1</LI><LI>Including a 3D-printed frame, fitting the size of a 96-well plate, so that the whole seeding and culturing process can be <B>monitored and captured using a microscope</B>.</LI></OL><P style="font-family: Trebuchet MS; text-decoration:underline;">Cell culturing protocol</P><OL><LI>Harvest the cells using the standard culturing protocol, and then resuspend in media to achieve 120E4 cells/ml seeding density.</LI><LI>Clean and wash the chamber with 70% ethanol and 1x PBS.</LI><LI>Use a 5mL syringe to load 1ml of media, 0.5ml of cell solution, and another media column to a 1.59mm diameter tube.</LI><LI>Use a syringe pump to load media and cell solution in the tube to the device.</LI></OL><DIV style="text-decoration: none; color: #000000; float: center; margin: 15px;text-align:center"><DIV style="width: 599px; text-align: center;display:inline-block;"><I><B>Figure. </B><B>A)</B> General setup for automatic cells seeding using a syringe pump. <B>B)</B> Input and output wires are connected to inlet and outlet of the microfluidic device. A 3D-printed frame is used to put the device on the microscope's placeholder, allowing dynamic monitoring. <B>C)</B> A syringe pump is set up to continuously pump the solution containing cells into the channel during seeding (seeding flow rate was: 20 ul/min, cell density: 120E4 cells/ml).</I></DIV></DIV><H2 style="color: #F20253; text-decoration:underline; font-family: Trebuchet MS;">Results</H2><H3 style="font-family: Trebuchet MS; font-weight:bold; font-size:15px;">Model 1</H3><DIV style="text-decoration: none; color: #000000; float: center; margin: 15px;text-align:center"><DIV style="width: 599px; text-align: center;display:inline-block;"><I><B>Figure. </B>Images of microfluidic channel, 24 hours after cells seeding.</I></DIV></DIV><P>
        After 24 hours, the channel was completely dried out. The images didn't show any cells inside the channel even though cells in the control culture dish grew normally (image not shown), which was seeded at the same time with as microfluidic device. Since we couldn't check for cells inside the device right after seeding, We couldn't narrow down the main reasons for <B>unsuccessful cell culturing</B> in the microfluidic device. Thus, we decided to <B>repeat the experiment (using model 2) with dynamic monitoring</B> right from the seeding stage.   
    </P><H3 style="font-family: Trebuchet MS; font-weight:bold; font-size:15px;">Model 2</H3><DIV style="text-decoration: none; color: #000000; float: center; margin: 15px;text-align:center"><DIV style="width: 699px; text-align: center;display:inline-block;"><I><B>Video. </B>Images taken inside the diamond chamber, first 340 minutes after seeding. Frame rate: 3fps. Each image frame was taken at 10 minute interval. <B>Red arrows:</B> indicate cells. <B>Blue arrows: </B> show evidence of poliferation</I></DIV></DIV><P>
        Right after seeding, the cells were still in suspension; therefore, they all had spherical shape. After about 20 frames (190 minutes), they started to elongate and shifted out of focal plane. At the 26th frame (250 minutes after seeding), the cells were out of focus, which could <B>indicate adhesion</B> of the cells to the chamber's surface. The blue arrows point at cells showing <B>evidience of mitosis</B>, including appearance of two nuclei within a single cell and formation of the metaphase plate. Unfortunately, at the end of the video (after 350 minutes), the media within the chamber dried out, which has been a major problem for our experimental setup.
    </P><A href="https://2016.igem.org/Team:BostonU_HW/Collaborations"><P style="font-family: Trebuchet MS;font-color:#7ECEFD; font-size:15px;"><I><B>Read more about our collaboration on the BU Hardware Team's website</B></I></P></A><H1 style="color:#FFFFFF; background-color:#F20253; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: Trebuchet MS">Testing pDEST mCherry</H1><H2 style="color: #F20253; text-decoration:underline; font-family: Trebuchet MS;">Purpose</H2><P>As our way of giving back to iGEM, we created a screening tool for use in Gateway cloning reactions.  In the past Gateway reactions have used the ccdB death gene as a negative selection tool to pick bacterial cultures that have successfully taken up a copy of the desired plasmid.  However, due to a patent on ccdB and negative selection tools in Gateway, iGEM had to remove all ccdB-containing parts from their registry.  Our screening tool, pDEST mCherry, uses red fluorescence to pick successfully transformed cultures.  With this, we hope that iGEM will be able to distribute the parts that were discontinued, and mammalian work that relies on Gateway reactions will be significantly easier for future teams.

While we used the pDEST mCherry successfully in our lab, we needed to test the clarity of the protocol we had written, and whether they would work in other labs. </P><H2 style="color: #F20253; text-decoration:underline; font-family: Trebuchet MS;">Collaboration</H2><P>We were thrilled to work with the <A href="https://2016.igem.org/Team:Tufts">Tufts University iGEM team</A> in order to test our protocol, and our construct.  </P><H2 style="color: #F20253; text-decoration:underline; font-family: Trebuchet MS;">Results</H2><P> The ligation reaction was successful.  As can be seen on the plate that Tufts sent us, there are white colonies, and small red colonies. </P><H1 style="color:#FFFFFF; background-color:#F20253; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: Trebuchet MS">Recombinases' sequences</H1><H2 style="color: #F20253; text-decoration:underline; font-family: Trebuchet MS;">Collaboration</H2><P>
    At the begining of the summer, we had a discussion with the <A href="https://2016.igem.org/Team:BostonU/Collaborations">BU Wetlab Team</A> to learn more about recombinases because some of their members and especially their graduate mentor - Ben Weinberg - had substantial experience working with recombinases. They specifically emphasized the issue of leaky expression of recombinase, resulting in high basal expression of output gene. Thus, during the conversation, we jotted down ideas for potential repressing systems, one of which were degradation tag and repressible promoters. However, the majority of their work with recombinase has been with tyrosine (Cre or FLP) family and done on E. coli. Thus, performing recombinase experiments in mammalian cell lines were unpredictable. Additionally, the BU Wetlab Team also helped us to obtain sequences for the recombinases and their equivalent recognition sites from Dr. Wilson Wong's lab. 
</P><A href="https://2016.igem.org/Team:MIT/Experiments"><H1 style="color:#FFFFFF; background-color:#F20253;; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: trebuchet MS"> Back to Experiments Home</H1></A><DIV style="font-style: italic;"><P style="text-decoration:underline; font-size: smaller;"><B>REFERENCE:</B></P><OL style="font-size: smaller;"><LI>Mehling Matthias, and Tay Savas. Microfluidic Cell Culture. Science Direct 2014; 25:95-102.</LI><LI>Komen Job, Wolbers Floor, Franke R. Henk, et al. Viability analysis and apoptosis induction of breast cancer cells in a microfluidic device: effect of cytostatic drugs. Biomed Microdevices(2008); 10:727–737</LI></OL></DIV></OLFV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>