"
Page
Discussion
View source
History
teams
Log in
 
Team:HIT-Harbin/project/part1
From 2012.igem.org
HIT-Harbin
Home
TEAM
OVERVIEW
UNDERGRADUATES
FACILITATORS
ABOUT THE UNIVERSITY
PROJECT
OVERVIEW
BIOSENSOR
BIOKILLER
BIOFILM
MODELING
APPLICATION
PARTS
SAFETY
NOTEBOOK
DIARY
PROTOCOL
ATTRIBUTION
HUMAN PRACTICE
LECTURE
SOFTRWARE
iGEM
Device1: Biosensor
    The detecting system is constructed to detect the existence of Staphylococcus aureus，which is based on the global regulator of virulence, agr quorum sensing system of S.aureus that modulates the expression of virulence factors in response to autoinducing peptides (AIPs)[1]. The detecting system we constructed is mainly composed of agrA and agrC.
    In the pathogenic species Staphylococcus aureus, the extracellular signal of the quorum sensing system is a thiolactone-containing cyclic peptides pheromone (AIP), whose sequence varies among the different staphylococcus strains. The polymorphism in the amino acid sequence of the AIP and of its corresponding receptor (AgrC) divides S.aureus strains into four major groups. The AIPs belonging to different groups are usually mutually inhibitory[
    AgrC is a transmembrane protein, which is the sensor molecule of a typical two-component signal system in S.aureus. AgrC possesses several key amino acid motifs typical of histidine protein kinase sensor. The AgrC sensor kinase can specifically binds to corresponding AIP, which secreted only from specific S.aureus, and the composite of AgrC and AIP then leads to phosphorylation of AgrA. AgrA in its phosphorylated sate activates transcription from both P2 and P3, leading to the production of GFP and 3OC6HSL. Thus we can detect the presence of S.aureus expediently by observing the expression of GFP. The figure(overview) shows the mechanism of our detecting system in E.coli.
    There is a trouble that the agr system belongs to S.aureus, but we hope this system works in E.coli, but . Therfore, we analyze the topology structure of AgrC and AgrA. Staphylococcus aureus AgrA, the transcriptional component of a quorum sensing system and global regulator of virulence that up-regulates secreted virulence factors and down-regulates cell wall-associated proteins, can bind in both the P2 and P3 promoter regions of the agr locus. The structure of AgrA, described by an online software PDB (Protein Data Bank), has ten β strands arranged into three antiparallel β sheets and a small α helix. The sheets are arranged roughly parallel to each other in an elongated β-β-β sandwich. A hydrophobic five-stranded β sheet (sheet 2: β3-β7) is at the center of the domain with two smaller amphipathic β sheets (sheet 1: β1-β2 and sheet 3: β8-β10) positioned on either side.
Fig 2. Structure of the Staphylococcus aureus AgrA bounding to DNA
     AgrC belongs to the histidine protein kinase (HPK) family and in particular to the HPK10 subfamily of QS peptide HPKs, which are predicted to consist of six or seven N-terminal transmembrane segments and a C-terminal cytoplasmic kinase domain. The 3D structures of HPK10 kinases have not been determined, but topology modeling using different prediction methods has indicated that AgrC has either five or six transmembrane segments. We have tried many different tools for modeling the structure of AgrC. Fig.2 shows the predicted topology of AgrC using the MEMSAT3 program, which indicates six transmembrane helices (18-24 amino acid sequence domains), three extracellular loops, with high probability of N-terminal in the cytoplasmic by TMHMM program.
Fig 3. Predicted transmembrane topology of the S. aureus AgrC protein..
    In S.aureus, there are four different agr groups, each with a distinct AIP capable of activating its cognate AgrC but inhibiting the AgrC of other groups. The AgrC-I and AgrC-IV are the most closely related due to the AIP-I and AIP-IV difference by a single amino acid residue. So we employed site-specific mutagenesis（AgrC-I A101T）to replace a amino acid residue in AgrC-I so that the mutated AgrC can bound to both AIP-I and AIP-IV. Fig.3 shows the sequence of AgrC-I after mutating and the mutation site of AgrC-I(highlighted)[3].
Fig 4. The mutation site of AgrC-I, as the red tag, we changed the 101th amino acid alanine into threonine.
Fig. 5 The schematic of the whole detecting device
    In order to observe if the detecting device senses the existence of S.aureus, we decided to clone green fluorescent protein (GFP) gene gfp under the control of P2（Fig. 5). The whole detecting device region(3163bp) was cut with EcorⅠand PstⅠto verify the sequence(Fig. 6).
Fig. 6 Double digestion analysis of the detecting device region(3163bp, lane2).
     Cultures of the E.coli cells carrying the pAgrC-AgrA-GFP (BL21) were grown in 50ml LB supplemented with 50μg/ml ampicillin at 37℃(160rpm) to exponential phase(A600=0.2). Then, the medium was treated with 5ml supernatant of S.aureus(A600=0.4). After 24h of treatment, cultures were placed under a fluorescence microscope(×1000) to observe the expression of GFP. The results confirmed that the detecting device cen be able to sense AIPs natively produced by S.aureus(Fig. 7).
Fig.7 A.Fluorescence microscopy of the engineered E.coli cells exposed to supernatant of S.aureus(A600=0.4). Circles indicate the green fluorescence. B.Visible light.
Reference
[1] Michael O, Roderich S, Cuong V, Gunther J, and Friedrich G. Inhibition of virulence factor expression in Staphylococcus aureus by the Staphylococcus epidermidis agr pheromone and derivatives[J]. FEBS Letters, 1999, 450:257-262.
[2] Sophie J, Christophe M, Jean T, Gerard L, et al. Relationships between Staphylococcus aureus Genetic Background, Virulence Factors, agr Groups (Alleles), and Human Disease[J]. Infection and Immunity, 2002, 70:631–641.
[3] Edward G, Elizabeth A. G, Tom W M, and Richard P N. Identification of Ligand Specificity Determinants in AgrC, the Staphylococcus aureus Quorum-sensing Receptor[J]. The journal of Biological Chemistry, 2008, 283:8930-8938.
Back to Top
Retrieved from " http://2012.igem.org/Team:HIT-Harbin/project/part1 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
