<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Leicester_Research skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Leicester/Research</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Live Preview</TITLE></P><DIV class="container" style="padding-right: 0px;padding-left: 0px;"><NAV class="navbar navbar-default"><DIV class="container"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Leicester">University of Leicester</A></DIV><DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Research">Research</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">About Us</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Sponsors">Sponsors</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Lab Work</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/labwork/methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/labwork/results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Measurement">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Safety">Safety</A></LI></DIV></DIV></DIV></NAV></DIV><DIV class="container" style="padding-bottom: 50px; padding-top:2px;"><DIV class="row"><DIV class="col-sm-2" style="padding-left:0px; padding right:30px;"><UL class="nav nav-pills nav-stacked" style="margin-left:0px;"><LI class="active"><A href="#">Research</A></LI><LI><A href="#Int">Introduction</A></LI><LI><A href="#Col">Colonisation</A></LI><LI><A href="#kswitch">Kill Switch</A></LI></UL></DIV><DIV class="col-sm-8"><DIV style="color:#fff; background-color: #3498db; border-radius: 50px; border-top-right-radius: 0px; border-bottom-left-radius: 0px; padding-left:30px; padding-right:30px; padding-top:30px; padding-bottom:20px; box-shadow: 10px 10px 50px #000000;"><H2 id="Int"> Introduction </H2><P>Due to the allotted lab time for the iGEM project, and various other constraints this project would not go to completion. Knowing this in advance required important decisions to be made in the early phases of the project's design, meaning sections that would be theoretical and those that were feasible for a lab based approach had to be defined. At this point decisions were made that determined the overall direction of the project, with the most important factor being what was possible within the timescale all the while ensuring the parts that were created were those which were most associated with the fundamental aim of the project. This page describes the theoretical sections of the project, taking into consideration why they would be required, ethical implications for their use  as well as the steps and process that would have taken place to create them were the time available for a complete product. The reason as to why these particular components could not advance onto lab work stage will also be described.</P></DIV><DIV style="color:#fff; background-color: #e74c3c; border-radius: 50px; border-top-right-radius: 0px; border-bottom-left-radius: 0px; padding-left:30px; padding-right:30px; padding-top:30px; padding-bottom:20px; box-shadow: 10px 10px 50px #000000;"><H2 id="Col"> Colonisation </H2><P>Due to the allotted lab time for the iGEM project, and various other constraints, this project 

would not go to completion. Knowing this in advance required important decisions to be made 

in the early phases of the project's design, meaning sections that would be theoretical and 

those that were feasible for a lab based approach had to be defined. At this point decisions 

were made that determined the overall direction of the project, with the most important factor 

being what was possible within the timescale, all the while ensuring the parts that were created 

were those which were most associated with the fundamental aim of the project. This page 

describes the theoretical sections of the project, taking into consideration why they would be 

required, ethical implications for their use as well as the steps and process that would have 

taken place to create them were the time available for a complete product. The reasons as to 

why these particular components could not advance onto the lab work stage will also be 

described.</P><H4> Escherichia coli Colonisation of The Intestinal Epithelial Layer </H4><P> Maximising NAD uptake by mammalian cells is an essential aim of the project and also a requirement for the efficacy of the final product. It was therefore essential that the genetically modified (GM) E. coli were capable of colonising the intestine. The reason being, so they could be maintained in the environment for a longer period of time, ideally permanently and therefore have a longer lasting effect. However colonisation should also minimise NAD waste by reducing the distance between the GM E. coli and intestinal cells, this in turn should reduce the diffusion distance for NAD before uptake occurs. Another point to consider is that by doing this it may also prevent the human gut microbiome from utilising this additional NAD pool. </P><P>There are various pathogenic E. coli strains that have had their colonisation mechanism thoroughly researched, all of which colonise or invade the epithelial layer of the intestine. These would therefore be ideal for optimal NAD uptake, however the fact that they are pathogenic would prevent them from being a suitable option. Although with consultation from Dr Richard Haigh who had previously carried out research on various E. coli strain colonising mechanisms, it was decided that it might be possible to utilise an operon or edit one that is involved with colonisation. The strain suggested was enteropathogenic E. coli (EPEC); it has two initial colonisation capabilities. One is a plasmid based type four pili known as bundle forming pili (BFP) and the second is a type three secretion system (TTSS) contained within the genome. Figure one below shows a simplified colonisation mechanism (Kaper et al., 2004). </P><P> The EPEC strain may be of use because they often contain a 70-100 kb plasmid known as the EPEC adherence factor (EAF), which encodes BFP and various other genes. BFP is associated with stage one of the colonisation mechanism referred to as localized adherence, however it is also responsible for a process known as autoaggregation (Nougayrède et al., 2003). This would be ideal for our project as it allows colonisation to the intestinal epithelial layer, but also enables E. coli to aggregate to one another at the cell surface. This should then allow increased uptake of NAD, however EAF is still a virulence factor, therefore it would require editing removing unnecessary virulence genes. Followed by verification that colonisation can still take place. </P><P><B>Figure one:</B> This diagram represents the EPEC colonisation mechanism. 1) Shows initial adhesion to the epithelial cell via BFP 2) Shows the type 3 secretion system initiating protein translocation 3) Represents pedestal formation. (Adapted from: Kaper et al., 2004: 124).</P><H4> A Comparison of BFP and The Type 3 Secretion System Colonisation Capabilities </H4><P> There are a number of other potential ways to colonise the intestinal epithelial cells, one example would be the use of the type three secretion system (TTSS) located in the 35 kb pathogenicity island known as the locus of enterocyte effacement (LEE). This is utilised by EPEC in the second stage of the colonisation mechanism shown in figure one, if EAF is present (Elliott et al., 2000). There are a number of differences between TTSS and EAF, which enable justification as to why one would be chosen over the other. The point that differentiates them the most is that TTSS is fundamental to EPEC’s ability to edit host cell function, in a way that is advantageous to the bacterium. To do this TTSS must efficiently bind to host cells, the eventual result is pedestal formation caused by actin build up and loss of microvilli, collectively known as attachment and effacing lesions (AEL) (Zhang et al., 2010).
</P><P> EAF’s function is significantly different; it is essentially an enhancer of TTSS. It contains regulatory genes PER A, B and C, that up regulate transcription of TTSS genes present on LEE enabling increased efficiency of colonisation and AEL (Trabulsi et al., 2002). The BFP gene cluster also enhances colonisation allowing for quicker anchorage to epithelial cells as well as triggering auto aggregation, allowing formation micro colonies (Frankel et al., 1998). However EAF on its own would produce no pathological outcome if transformed into K12 E. coli unless LEE is also present within its genome, this is a reason as to why BFP is a better option for the project, as it will still allow colonisation of epithelial cells without the resulting AEL triggered by TTSS. It will be less efficient than with the presence of LEE but it can trigger colonisation nonetheless, although unlike TTSS it does not disrupt the internal workings of host cells, making its use far more ethical. </P><P> However there is a range of additional reasons as to why TTSS cannot be used; as previously stated TTSS is contained within LEE, and with LEE being a 35 kb pathogenicity island this creates a significant problem. This large sequence is contained on the EPEC chromosome, where as BFP is contained on the EAF plasmid, meaning TTSS requires more preparation work, which isn’t possible within the time scale of the project.  TTSS is also more complex than the type four pilus system, LEE encodes 41 genes as stated by Kaper et al. (2004) were as BFP encodes 14, all of which are used to form the pili (Nara et al., 2010). Most of the 41 genes are for the formation of the TTSS, although some are effector proteins. These effector proteins are utilised by TTSS when it invades host cells, TTSS essentially acts like a needle which then proceeds to secrete effector proteins into the host cell having an array of effects on cellular function, which enable colonisation and ultimately causes AEL (Kaper et al., 2004). Most of these effector genes would therefore have to be identified and removed out of the 41 genes associated with LEE, which again isn’t possible within the time scale of the project.</P><P> What complicates matters further is the fact that one of these effector genes is fundamental to TTSS ability to colonise cells. The effector gene is known as translocated intimin receptor (TIR). TIR is secreted into host cells, it is then translocated into the host cell outer membrane. EPEC has a membrane bound receptor known as intimin that is then able to bind to TIR resulting in colonisation of the epithelial cell. However TIR is also responsible for actin remodelling triggered by the phosphorylation of one of its tyrosine residues 474, once it enters the cytoplasm, this causes pedestal formation and so the AEL effect (Nougayrède et al., 2003). This ultimately makes TTSS less ethical than BFP as its ability to colonise cells is dependent on one of these effecter proteins that must enter host cells. </P><P> BFP would be the most suitable for some additional reasons; first of all it is already available to us in an edited form containing the minimum number of genes required for BFP biogenesis and cell colonisation thanks to Dr Richard Haigh. Also BFP only interacts with the outer membrane it does not invade host cells like TTSS. BFP also enables aggregation of E. coli cells creating microcolonies, this is ideal for increasing NAD uptake and it is also something that TTSS alone is not capable of. During an EPEC infection TTSS remains attached to epithelial cells, BFP even though utilised to initially attach to epithelial cells also remains attached throughout prolonged infection, until treated with antibiotics (Saldan ̃a et al., 2009). This means this method could potentially be used as a prolonged treatment. </P><P> If BFP were to fail during the project assuming the Biobrick was constructed correctly, the potential alternative would most likely be a TTSS with all effectors bar TIR and other virulence genes removed. This would mean the TIR residue 474 would have to be changed to prevent phosphorylation and so prevention of AEL, but this is possible through IGT gene synthesis. However it would be interesting to consider the use of an edited BFP plasmid with the addition or PER A, B and C in combination with a plasmid based LEE containing only TTSS genes and the edited TIR. This would offer the most efficient colonisation method. It would also be worth noting the importance of investigating the requirements for effecter protein secretion into host cells, as it may be possible to provide NAD producing enzymes via TTSS creating an alternative way of increasing NAD output. However that is not possible for this project and ethical implications of this approach make it a less feasible overall. </P><H4> How BFP Functions </H4><P> If BFP is to be used it is essential to understand how it functions. The bfp operon contains fourteen genes and thirteen are required for biogenesis of BFP and the corresponding phenotypes (Hwang et al., 2003). Figure two below shows the genes contained within the bfp operon (Virulence Factors For Pathogenic Bacteria, 2014). </P><P> The following section will describe the function of the thirteen genes, providing detail as to why they would be required in the edited EAF plasmid or justifying reasons as to why they could be removed. The first of the genes contained within the operon is bfpA, this encodes prebundlin what will eventually be a major structural subunit of the BFP filament. However for prebundlin to become functional it must be cleaved by prepillin peptidase, which is encoded by bfpP. The protein removes the T4P signal sequence followed by N-methylating the N-terminus, forming mature bundlin (Nougayrède et al., 2003). Mature bundlin is situated in the cytoplasmic membrane, it has a globular C-terminal that is exposed to the periplasmic space and an N-terminal that makes up the filament's core, the N-terminals hydrophobic nature stabilises BFP structure when bundlin subunits come together (Donnenberg et al., 1997). </P><P>
The gene bfpE also encodes a protein that is situated in the cytoplasmic membrane and the structural determination of this protein made by Blank et al. (2001), resulted in a prediction that BfpE would have a major role in interacting between other components of the BFP complex. BfpE was found to be a polytopic protein because it has a large N-terminal cytoplasmic domain with three transmembrane domains, connected by loops. This is followed by another transmembrane domain producing a large C-terminal periplasmic domain. The BfpE protein was later shown to be a central component to the subassembly of three other proteins by crowther et al. (2004), by studying the protein interactions of BfpE, C, D and F using the yeast two-hybrid system. The gene bfpC encodes a biotopic protein meaning there is one region contained within the cytoplasm, this being the N-terminus and one region in the periplasm, this being the C-terminus. The BfpC protein closely associates with BfpE, to form a subcomplex (crowther et al., 2004). The function of which will be described later. </P><P> Two other genes bfpD and bfpF encode nucleotide binding proteins that each form hydrophilic homohexamers producing a ring shaped structure, these associate with BfpE and C in the cytoplasm (crowther et al., 2004). The function of BfpD and F has been determined and they are responsible for the way in which BFP triggers auto aggregation and eventual dispersal of the resulting microcolonies. These colonies disperse in a later phase of auto aggregation via twitching motility. Twitching motility is an energy requiring processes, when bfpF is knocked down twitching motility cannot occur and the outcome is increased localised adherence. This is due to the fact that BFP cannot be retracted (Bieber et al., 1998). Therefore this gene is required as part of the autoaggregation dispersal mechanism. The function of bfpD is antagonistic to bfpF, it's required for energy production that enables the extension of BFP, which is essential for initial colonisation and autoaggregation steps (Hwang et al., 2003). It's interesting to consider that as the bfpF gene is not necessary for BFP biogenesis but instead a mechanism of dispersal which increases virulence, that if it were to be knocked down an increased localised adherence with a reduction in dispersal may be beneficial to our project goals.</P><P> As stated previously a subassembly complex is formed between BfpE, C, D and F, as there functions and structures have now been described the importance of this protein complex can now be explained. BfpC and E closely associate in the cytoplasmic membrane, BfpC and E N-terminals are in close proximity to one another and interact, BfpD binds to BfpC and E N-terminals. When this occurs conformational changes take place that stabilise all three proteins and prevent degradation, a requirement for BFP biogenesis (crowther et al., 2004). Alongside this its known that BfpD is responsible for BFP extension, which is now known to act from the cytoplasm of cells. This occurs via interactions with BfpC and E, which are embedded in the cytoplasmic membrane. However it is BfpC that extends into the periplasmic space and has been hypothesised to interact with the outer membrane subassembly complex, therefore binding of BfpD to C may be responsible for triggering BFP extension. All the while BfpE acts as a scafhold and gatekeeper within the inner membrane (Ramer et al., 2002). However BfpF binds to a highly conserved cytoplasmic loop of BfpE and has been shown to be required for the retraction of the pilus, binding of BfpF to E may therefore trigger a conformational change that allows transport and retrieval of the filament, initiated from the cytoplasm (crowther et al., 2004). Therefore both BfpD and F exact their functions via different proteins situated within the cytoplasmic membrane, which may then trigger interactions with the gate on the outer membrane via BfpC or inner membrane via BfpE respectively, enabling extension or retraction of the pilus. </P><P> 
There are two more genes that function together bfpG and bfpB, which are the second and third genes in the bfp operon respectively. The bfpB gene is associated with the secretin superfamily and produces lipoproteins that are situated in the outer membrane; these form a multimer with a central channel. This channel is an incompletely gated secretion pathway for BFP. The fact the channel is incompletely gated means that once BFP activity has ceased and localised adherence is no longer required, the channel has a secondary function that allows secretion of other biological molecules (Schmidt et al., 2001). bfpG produces a second outer membrane protein that physically interacts with the monomers produced by bfpB, BfpG is not required for localisation of the monomers into the outer membrane or there stabilisation, but instead enables the formation of the multimer of BfpB proteins within the outer membrane (Schmidt et al., 2001).  An additional discovery by Daniel et al. (2006) was that BfpG exists in two forms, depending on weather cleavage occurs at signal peptidase one, site one or two. It was found that in the absence of BfpB, site one is cleaved resulting in a mature protein that is hydrophilic and present in the periplasm. However when BfpB is present in the outer membrane site two is cleaved resulting in localisation of BfpG in the outer membrane, this confirms BfpB is required for recruitment of BfpG to the outer membrane. Once the multimer has formed, BfpG may also be the protein responsible for forming the incomplete gate for the secretion channel.
</P><P> The gene bfpU has been found in the cytoplasm, cytoplasmic membrane and periplasm. However BfpU has been found to localise and physically interacts with BfpB and it may also interact with bundlin. In BfpB’s case BfpU binds to the N-terminus just like BfpG, BfpB may therefore have a similar effect on BfpU localisation as it does with BfpG. Due to these interactions it is suggested that BfpB, G and U form an outer membrane subassembly complex. It has also been found that BfpU and G do not interact even though they can be found in close proximity (Daniel et al., 2006). This may suggest that BfpU has no influence on gate control but instead has another role. With BfpU’s potential interaction with bundlin combined with BfpU’s distribution across other compartments of the cell, it has been suggested that BfpU could play a role in transporting the bundlin subunits from the cytoplasmic membrane to the outer membrane (Daniel et al., 2006). Bearing in mind that BfpB may also interact with BfpC, the discovery of this outer membrane subassembly complex and the knowledge that BfpC and U may span the periplasm, bridging the gap to both membrane complexes could suggest they have essential roles in pilus transport. However this is yet to be confirmed, although this information in combination with what Ramer et al. (2002) provided evidence for may support this hypothesis. They showed that deletion of bfpB, C or G reduced abundance of BfpU, meaning BfpU requires proteins associated with the sub assembly complexes for stabilisastion, this may suggest that BfpU bridges the periplasmic space although it may just mean that it interacts with both complexes independently. </P><P> The genes bfpI, J and K are all known as pilin-like proteins, just like BfpA the pillin-like proteins are processed by BfpP the prepilin peptidase enzyme, this occurs in the same way forming the mature proteins. The three proteins are also associated with the inner membrane assembly complex (Ramer et al., 2002). Deletion of both BfpC and U causes degradation of the three pilin-like proteins, suggesting the proteins associate with the two potential membrane bridging proteins for stabilisation. It is therefore suspected that BfpC, U, I, J and K may form a sub assembly complex that is situated on the periplasmic face of the inner membrane, however the exact way BfpI, J and K interact with BfpU and C is currently unknown (Ramer et al., 2002). It has been discovered that the pilin-like proteins are only required when BfpF is also present, which is responsible for pilus retraction. This suggests the pilin-like proteins have a role in preventing pilus retraction or aiding in extension (Masi et al., 2012). The gene bfpL has also been found to associate with this periplasmic complex via BfpC. BfpL is an inner membrane protein that only binds to the BfpC C-terminus in the periplasmic face. It has also been found that the deletion of BfpL causes a reduction in BfpK and vice versa. There has also been evidence that BfpI and J, are dependent on BfpL for there abundance, however there has been no definitive evidence that provides an indication of BfpL interacting with BfpI, J and K (Masi et al., 2012). Therefore if BfpI, J and K interact with BfpC it may be via different protein domains to BfpL, however the order and domain they associate with still requires confirmation. It suspected that BfpL may be responsible for extracting bundlin from the inner membrane for pilus formation (Masi et al., 2012). 
</P><P> The gene bfpH is predicted to be a transglycosylase via gene prediction software, however it is the only gene contained in the BFP operon that does not prevent BFP formation or other functions when knocked down. Its location is known to be in the periplasm (Hwang et al., 2003). As this gene does not affect BFP formation or phenotypes, further research does not appear to have been carried out.
</P><P> In summary there are three subassembly complexes that make up BFP, the inner membrane complex, the periplasmic bridging complex and the outer membrane complex. The inner membrane complex consists of BfpE a potential gate and scaffold for the other inner membrane proteins. BfpC, which associates with BfpE interacts with a range of other proteins from both the cytoplasm and periplasm producing a range of functions. It may also span the periplasm connecting the inner and out complexes, but this requires confirmation. BfpD acts from the cytoplasm and is an ATPase, which is responsible for pilus extension. This occurs via interaction with BfpC while joining too BfpE for stabilisation. BfpF also acts from the cytoplasm and is another ATPase that binds to a conserved loop of BfpE, associated with causing pilus retraction. </P><P> The periplasmic bridging complex is the most speculative complex. The periplasmic face of inner membrane complex enables binding of BfpU to BfpC and then the three pillin-like proteins BfpI, J and K bind to BfpU and C in close proximity to the inner membrane, these three proteins are associated with regulating pilus retraction and extension. The outer membrane complex is a multimer of BfpB proteins that form the outer membrane channel that enables pilus extension and retraction outside of the cell. BfpG is required for multimer formation and may be the gate to the channel, the gate however is incomplete and is suspected of having a secondary function enabling secretion of other molecules. BfpU also binds to BfpB but is not associated with chanel or gate formation and is suspected of connecting the two membrane complexes alongside BfpC, which can also associate with BfpB. </P><P> BfpA is the main filament subunit and may be transported across the membrane via BfpC and U with the energy for the process generated by BfpD or F depending on the required pilus action. BfpL proteins also bind to BfpC and are suspected of extracting BfpA subunits provided from the inner membrane protein BfpE, enabling their incorporation into the main pilus. Although currently this is speculation and requires further research. BfpP is a prepillin peptidase enzyme required for the maturation of BfpA, I, J and K before they can become functional. BfpH is the only gene in the BFP operon that is not required for BFP function and as a result its function has not been determined as of yet. </P><P> This section provides a general idea of how BFP may be constructed; however while some of the functions of the proteins have been determined, others remain speculative. Further research is required to determine the exact function and how they intimately interact with one another, to gain a definitive idea of how this type four pilus system functions. Nonetheless it is fundamental to the project to have an understanding of how this operon may operate to allow for the construction of a Biobrick that would be optimised for its role in colonisation. The importance of having a good theoretical understanding should enable effective ethical decisions, which would be required due to the fact that BFP is in fact a virulence factor.</P><H4> Why BFP is Only Theory </H4><P> The edited plasmid containing the required BFP genes could not be utilised as the lab space we were working in was classed as category one, as this edited plasmid originated from the virulence factor EAF it therefore required a category two lab. This is the sole reason as to why we could not use BFP in our lab project. The edited EAF plasmid contains only genes that are essential for BFP formation, that is the fourteen genes shown in figure two so all other unnecessary virulence factors are removed. This improves the efficacy of the edited EAF plasmid, which would mean its potential use in future applications becomes more probable. Even though the genes contained for BFP could still be considered virulence factors. However even in this edited state it could not be used in our lab.</P><H4> The Creation of a BFP Biobrick </H4><P> Although the edited EAF plasmid could not be converted into a Biobrick, the necessary steps to do so were considered. The first stage to constructing the Biobrick would be to obtain the sequence of the edited plasmid, the plasmid length should be ~14 kb, all restriction sites would then be identified or verified. Figure three below shows the restriction sites present on the BFP operon (Donnenberg et al., 1997). </P><P> The six restriction sites associated with standard assembly protocol RFC10 that are contained within the plasmid would be systematically removed via PRC amplification utilising internal primers. The removal of these restriction sites would then be verified at each step by restriction digest. Once all the disruptive restriction sites are removed the required restriction sites for the assembly protocol would then be attached via PCR. The product would then undergo gel electrophoresis followed by gel extraction. All associated steps required for digestion and ligation to the Psb1C3 backbone, or alternative such as bacterial artificial chromosome if the 14 kb size becomes an issue would then be carried out. Transformation and minipreps would then follow to enable plasmid amplification and purification respectively; some of the plasmid would then be digested and run on a gel to verify that the incorporated restriction sites function as they should. Finally if possible the construct would be sequence verified before being submitted as a part.   </P><H4> How to Verify That BFP functions </H4><P> Before submission can occur BFP’s function must be verified, therefore a test to confirm that the transformed E. coli can in fact colonise epithelial intestinal cells would be carried out. An experiment to confirm this would require tissue culture cells, suitable cells such as HeLa and HEp2 have been used by Nataro et al., (1985) to show EPEC does in fact adhere to host cells as part of its pathogenesis step. Another study carried out by Kavitha and Niranjali (2009) utilised HEp2 and INT 407 cell lines, this was to test if a particular compound inhibits EPEC adherence. Of the three cell lines INT 407 would be the best option to use for this verification test because INT 407 is the only one that consists of intestinal epithelial cells, therefore this would provide the most realistic result. </P><P> Acquisition of the required cell lines would either come from the ATCC, or appropriate department from the University of Leicester and grown under condition recommended by the specific provider, or ones which would be most appropriate for the following adherence assay. This adherence assay was utilised by Kavitha and Niranjali (2009) and is as follows. Cells would be grown to conﬂuency on a cover slip in tissue culture plates. The media would then be discarded and the monolayers would be washed with PBS. The antibiotic-free 2% gibco minimum essential medium would then be replaced and 50 µL our GM E. coli strain would be added to the appropriate plates. These would then be incubated for 3 hours at 37 °C. At the end of incubation, the media will be discarded and the cover slips washed three times with PBS. The cover slips would then be ﬁxed with Carnoy’s ﬁxative (1 part of glacialacetic acid and 2 parts absolute alcohol) for 15 minutes. The ﬁxed cells will then be air dried and stained with Gram safranin. The stained slides will then be imaged under a phase contrast microscope; verification of successful colonisation can be confirmed via comparison of a control lacking addition of GM E. coli. </P></DIV><DIV style="color:#fff; background-color: #2ecc71; border-radius: 50px; border-top-right-radius: 0px; border-bottom-left-radius: 0px; padding-left:30px; padding-right:30px; padding-top:30px; padding-bottom:20px; box-shadow: 10px 10px 50px #000000;"><H2 id="kswitch"> Kill Switch </H2><P>There are two kill switch systems in the genetically engineered bacteria: One, a kill switch which causes the bacteria to die upon addition of an inducer; the other, a maintenance kill switch whereupon the bacteria die if they leave the gut. This dual containment system allows for the reduction in bacterial escape and horizontal gene transfer as well as the death of the bacteria as a backup option if it goes wrong. </P><P>Potentially could use a toxin-antitoxin system but with a modified antitoxin protein which incorporates a nonstandard amino acid that is vital for the toxin function. Therefore upon induction of the nonstandard amino acid the toxin will have the right amino acids needed for its correct synthesis and thus will kill the cell. It is easier and simpler than the altering 22 essential enzymes like Rovner et al, 2015 but more effective than toxin-antitoxin systems. However, this approach will need to add a stop codon for the nonstandard amino acid in a key hydrophobic region (if the nonstandard amino acid is hydrophobic) (Rovner et al, 2015) so that without the non-natural amino acid the protein cannot fold correctly and thus be subject to proteolysis. This results in cell death upon the addition of the non-standard amino acid. However this would require the changing of the stop codon used to another stop codon in all other genes as well as engineering a tRNA synthase that charges the non-natural amino acid to an edited tRNA molecule that is cognate for the stop codon used.</P><P>The CCDA/CCDB kill switch is in an operon system with CCDA under the control of a temperature sensitive RBS (Part BBa_K115002) whereas the CCDB is under the control of a generic RBS (Part BBa_K581008). This means that at under 37<SUP>O</SUP>C (i.e. not in the human body) the translation rate  of the CCDA will dramatically decrease relative to other genes in the bacteria at the same temperature, whilst CCDB will remain the same in respect to the fundamental decrease in rate due to the lower temperature. Thus there will be a high enough ratio of CCDB (once translated) to kill the cell. Summed up: When the temperature is too cold, the antitoxin (CCDA) doesn’t work, so the toxin (CCDB) kills the cell.</P><P>However, CCDB is patented so another toxin/antitoxin system would be preferred such as MazF/MazE. If no antitoxin is available then the antitoxin in the system described above could be replaced with a polymerase gene (e.g. T7 Polymerase) and the desired toxin under the control of a promoter for that polymerase (i.e. a T7 consensus promoter). This would give higher levels of transcription and would be perhaps a more sure system. For our iGEM team however, due to time requirements, the simpler the system the better.</P><P>Another Kill Switch which could be used to selectively kill the bacteria would be to use the X and Y expansion of the genetic code by Malyshev, et al 2014. This would use a toxin (Such as MazF toxin) that is dependent on the bases X and Y. This can be done by using long-range PCR for the MazF inside pUC19 but leaving a few bases in between the forward and the reverse primer. Then synthesis and PCR amplify an oligonucleotide which contains the primers and the X (dNaM)/Y (d5SICS) codon (Malyshev, et al 2014) for the non-natural amino acid for that codon (assuming there is a tRNA synthase for this). This can then be inserted into the PCR amplified pUC19 plasmid (with the gaps) through Gibson Assembly to then be transformed (Malyshev, et al 2014) as can be seen in figure 1. Proof will be needed for the incorporation of the X/Y through testing whether the transformed cells only die through addition of the X and/or Y base in a medium lacking these. This would significantly reduce the likelihood of the bacterium kill switch being activated naturally in the human microbiome. However, like through the addition of a nonstandard amino acid via stop codons, a tRNA molecule will be needed that can recognise the synthetic bases as well as a tRNA synthase which can accurately charge the nonstandard amino acid to this tRNA. As such this method can only be used in theory for our iGEM team. This will be a very useful application of synthetic biology once these key tRNA’s and tRNA synthases are ready. </P></DIV><DIV style="color:#fff; background-color: #9b59b6; border-radius: 50px; border-top-right-radius: 0px; border-bottom-left-radius: 0px; padding-left:30px; padding-right:30px; padding-top:30px; padding-bottom:20px; box-shadow: 10px 10px 50px #000000;"><H2 id="furtherconsideration">Further Consideration</H2><H4><B>siRNA Inhibition</B></H4><P>Not practical as there is no viable/practical clinical application.</P><H4><B>Regulation of Cassette</B></H4><P>With further work we hope to fine-tune the levels of NAD+ output by inserting negative feedback promotor sequences. We would vary the consensus sequence until assays had confirmed an optimal level of NAD+ was being secreted, levels of which could be determined by extensive literature research and further experiments in vivo and in vitro.</P><H4> Further objectives </H4><UL><LI>Once the BFP Biobrick had been constructed and its functionality confirmed; it would then be necessary to see if it can work in tandem with the plasmid containing the genes responsible for increasing NAD output, as well as the killswitch. 
</LI><LI> However as each plasmid would require a different origin or replication, it would be more likely that the contents of each plasmid would be combined. This meaning the killswitch, NAD genes, and BFP genes would be combined as one cassette in a specific order. The length of the overall construct would be nearing 20 kb by this point although, for iGEM parts the three components to the project would be submitted independently. The significant size of the construct may require a bacterial artificial chromosome to enable it to function effectively. </LI><LI> However for this project to have real potential the three components required for effective treatment would need to be incorporated into the genome as this would prevent horizontal gene transfer, increasing efficacy </LI><DIV style="color:#fff; background-color: #3498db; border-radius: 50px; border-top-right-radius: 0px; border-bottom-left-radius: 0px; padding-left:30px; padding-right:30px; padding-top:30px; padding-bottom:20px; box-shadow: 10px 10px 50px #000000;"><H4> References: </H4><UL><LI> Bieber, D., Ramer, W.S., Wu, C., Murray., Willia, J., Tobe, T., Fernandez, R., 1998. Type IV Pili, transient bacterial aggregates, and virulence of enteropathogenic escherichia coli. Science. 280: 2114-2118.</LI><LI>Blank, T.E. and Donnenberg, S.M., 2001. Novel Topology of BfpE, a Cytoplasmic Membrane Protein Required for Type IV Fimbrial Biogenesis in Enteropathogenic Escherichia coli. Journal of Bacteriology. 183: 4435-4450.</LI><LI>Crowther, J.L., Anantha, P.R., Donnenberg, S.M., 2004. The inner membrane subassembly of the enteropathogenic Escherichia coli bundle-forming pilus machine. Molecular Microbiology. 52: 67-79.</LI><LI>Daniel, A., Singh, A., Crowther, J.L., Fernandes, J.P., Schreiber, W., Donnenberg, S.M., 2006. Interaction and localization studies of enteropathogenic Escherichia coli type IV bundle-forming pilus outer membrane components. Microbiology. 152: 2405-2420.</LI><LI>Donnenberg, S.M., Zhang, H., Stone, D.K., 1997. Biogenesis of the bundle-forming pilus of enteropathogenic Escherichia coli: reconstitution of fimbriae in recombinant E. coli and role of DsbA in pilin stability - a review. Gene. 192: 33-38.</LI><LI>Elliott, J.S., Sperandio, V., Giro ́n, A.J., Shin, S., Mellies, L.J., Wainwright, L., Hutcheson, W.S., Mcdaniel, K.T., Kaper, B.J., 2000. The Locus of Enterocyte Effacement (LEE)-Encoded Regulator Controls Expression of Both LEE- and Non-LEE-Encoded Virulence Factors in Enteropathogenic and Enterohemorrhagic Escherichia coli. Infection and Immunity. 68: 6115-6126.</LI><LI>Frankel, G., Phillips A.D., Rosenshine, I., Dougan, G., Kaper, J.B., Knutton, S., 1998. Enteropathogenic and enterohaemorrhagic Escherichia coli: more subversive elements. Molecular Microbiology. 30: 911-21.</LI><LI>Hwang, J., Bieber, D., Ramer, W.S., Wu, C., Schoolnik, K.G., 2003. Structural and Topographical Studies of the Type IV Bundle-Forming Pilus Assembly Complex of Enteropathogenic Escherichia coli. Journal of Bacteriology. 185: 6695-6701.</LI><LI>Kaper, B.J., Nataro, P.J., Mobley, T.L.H., 2004. Pathogenic Escherichia coli. Nature Reviews Microbiology. 2: 123-140. </LI><LI>Kavitha, D. and Niranjali, S., 2009. Inhibition of Enteropathogenic Escherichia coli Adhesion on Host Epithelial Cells by Holarrhena antidysenterica (L.) WALL. Phytotherapy Research. 23: 1229-1236.</LI><LI>Malyshev.D., Dhami, K., Lavergne, T., Chen, T., Dai,N., Foster,J., Correa,I. &amp; Romesberg, F. 2014. A semi-synthetic organism with an expanded genetic alphabet. Nature 509 (7500) 385-388. </LI><LI>Masi, D.L., Szmacinski, H., Schreiber, W., Donnenberg, S.M., 2012. BfpL is essential for type IV bundle-forming pilus biogenesis and interacts with the periplasmic face of BfpC. Microbiology. 158: 2515-2526.</LI><LI>Nataro, P.J Scaletsky, A.C.I., Kaper, B.J., Levine, M.M., Trabulsi, R.L., 1985. Plasmid-Mediated Factors Conferring Diffuse and Localized Adherence of Enteropathogenic Escherichia coli. Infection and Immunity. 48: 378-383.</LI><LI>Nougayrède, J., Fernandes J.P., Donnenberg S.M., 2003. Adhesion of enteropathogenic Escherichia coli to host cells. Cellular Microbiology. 5: 359-372.</LI><LI>Ramer, W.S., Schoolnik, K.G., Wu, C., Hwang, J., Schmidt, A.S., Bieber, D., 2002. The Type IV Pilus Assembly Complex: Biogenic Interactions among the Bundle-Forming Pilus Proteins of Enteropathogenic Escherichia coli. Journal of Bacteriology. 184: 3457-3465.</LI><LI>Rovner,A., Haimovich, A., Katz,S., Li, Z., Grome, M., Gassaway, B., Amiram,M., Patel,J., Gallagher,R., Rinehart,J. &amp; Isaacs, F. 2015. Recoded organisms engineered to depend on synthetic amino acids. Nature, <B>518</B> (7537), 89-93. </LI><LI>Saldan ̃a, Z., Erdem,  L.A., Schu ̈ller, S., Okeke, N.I., Lucas, M., Sivananthan, A., Phillips, D.A., Kaper, B.J., Puente, L.J., Giro ́n, A.J., 2009. The Escherichia coli Common Pilus and the Bundle-Forming Pilus Act in Concert during the Formation of Localized Adherence by Enteropathogenic E. coli. Journal of Bacteriology. 191: 3451-3461.</LI><LI>Schmidt, A.S., Bieber, D., Ramer, W.S., Hwang, J.,Wu, C., Schoolnik, K.G., 2001. Structure-Function Analysis of BfpB, a Secretin-Like Protein Encoded by the Bundle-Forming-Pilus Operon of Enteropathogenic Escherichia coli. 183: 4848-4859.</LI><LI>Trabulsi, R.L., Keller, R., Gomes, T.A.T., 2002. Typical and Atypical Enteropathogenic Escherichia coli. Emerging Infectious Diseases. 8: 508-513.</LI><LI>Virulence Factors For Pathogenic Bacteria., 2014. Identified Virulence Factors of EPEC : Adherence. Available at: www.mgc.ac.cn/cgi-bin/VFs/vfs.cgi?VFID=VF0174#VF0174 [Accessed 15th August 2015]. </LI><LI>Zhang, Q., Li , Q., Wang, C.,  Liu, X.,  Li, N., Li, J.,  2010. Enteropathogenic Escherichia coli changes distribution of occludin and ZO-1 in tight junction membrane microdomains in vivo. Microbial Pathogenesis. 48: 28-34. </LI></UL></DIV><DIV class="col-sm-2"><A class="twitter-timeline" data-widget-id="327381807010025474" href="https://twitter.com/iGEMLeicester" width="80" height="300">
Tweets by @iGEMLeicester
</A></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>