<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Leicester_labwork_results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Leicester/labwork/results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Live Preview</TITLE></P><DIV class="container" style="padding-right: 0px;padding-left: 0px;"><NAV class="navbar navbar-default"><DIV class="container"><DIV class="navbar-header"><A class="navbar-brand" href="https://2015.igem.org/Team:Leicester">University of Leicester</A></DIV><DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Research">Research</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">About Us</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Sponsors">Sponsors</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Lab Work</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/labwork/methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/labwork/results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Measurement">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Leicester/Safety">Safety</A></LI></DIV></DIV></DIV></NAV></DIV><DIV class="container" style="padding-bottom: 50px; padding-top:2px;"><DIV class="row"><DIV class="col-sm-2" style="padding-left:0px; padding right:30px;"><UL class="nav nav-pills nav-stacked" style="margin-left:0px;"><LI class="active"><A href="https://2015.igem.org/Team:Leicester/labwork/results">Results
</A></LI><LI><A href="#Discuss">Discussion</A></LI></UL></DIV><DIV class="col-sm-8"><DIV style="color:#fff; background-color: #3498db; border-radius: 50px; border-top-right-radius: 0px; border-bottom-left-radius: 0px; padding-left:30px; padding-right:30px; padding-top:30px; padding-bottom:20px; box-shadow: 10px 10px 50px #000000;"><H2> Results</H2><H4> PCR of nadD, nadE and PncB </H4><P>The three aforementioned genes were PCR amplified initially without the iGEM prefix and suffix, which would enable the correct hybridisation of the primer to the E.coli genomic DNA to amplify the genes. The forward and reverse primer sequences for these genes are: 

(see table)
</P><TABLE class="table table-bordered"><THEAD><TR><TH>nadD</TH><TH>nadE</TH><TH>PncB</TH><TH>Internal PncB</TH></TR></THEAD><TBODY><TR><TD>F:5'-ATGAAATCTT<P>TACAGGctctg-3'</P></TD><TD>F 5'-ATGACATTGC<P>AACAACaaataataaa-3'</P></TD><TD>F:5'-ATGACACAAT<P>TCGCTtctcc-3'</P></TD><TD>IF:5’-TTCATCATCCTG<P>GAGGCGCA-3’</P></TD></TR><TR><TD>R:5'-TCAGCGATAC<P>AAGccttg-3'</P></TD><TD>R:5'-TTACTTTTTC<P>CAGAAatcatcg-3'</P></TD><TD>5'-TTAACTGGCTT<P>Ttttaatatgcg-3'</P></TD><TD>IF:5’-TTCATCATCC<P>TGGAGGCGCA-3’</P></TD></TR></TBODY></TABLE><P>These primers were successful in amplifying these genes and was confirmed through gel electrophoresis. However as PncB has a Pst1 site which is incompatible with the biobrick system, internal primers were used which had a single base change (from a G to a C) to remove the restriction site. The Forward Internal and the Reverse Exterior primer were used for one PCR reaction and in another it was the Reverse Internal primer and the Forward Exterior primer. This resulted in two PCR products overlapping at the removed restriction site. This was confirmed by a gel. The forward and reverse exterior primers were then used to amplify these products into one PncB amplification and thus having a PncB without a restriction site. This was confirmed by digesting this PncB with Pst1 and resulted in one band, as can be seen in figure 1. The PCR product that was successful was purified and had concentrations of 8.2ng/µl on average. 

(see figure 1)
</P><P> To make these genes into biobricks the iGEM prefix and suffix (underlined) were added to primers (called fusion primers) which can be seen in the table below.  (see table) </P><TABLE class="table table-bordered"><THEAD><TR><TH>nadD</TH><TH>nadE</TH><TH>PncB</TH></TR></THEAD><TBODY><TR><TD>F:5’-<U>GTTTCTTCGAATTC<P>GCGGCCGCTT</P><P>CTAG</P></U>ATGAAATCTTTAC-3’</TD><TD>F:5'-<U>GTTTCTTCGAATTC<P>GCGGCCGCTT</P><P>CTAG</P></U>ATGACATTGCAAC-3'</TD><TD>F:5'-<U>GTTTCTTCGAATTC<P>GCGGCCGCTT</P><P>CTAG</P></U>ATGACACAATTCG-3'</TD></TR><TR><TD>R:5’-<U>GTTTCTTCCTGCA<P>GCGGCCGCTACTAG</P><P>TATTATTA</P></U>TCAGCCATA-3’</TD><TD>R:5'-<U>GTTTCTTCCTGCA<P>GCGGCCGCTACTAG</P><P>TATTA</P></U>TTACTTTTTCCAG-3'</TD><TD>R:5'-<U>GTTTCTTCCTGCA<P>GCGGCCGCTACTAG</P><P>TATTATTA</P></U>ACTGGCTTTT-3' </TD></TR></TBODY></TABLE><P> These primers were used for PCR amplification, using the previous PCR products as the target of amplification rather than genomic DNA. NadD and NadE amplification were successful. </P><H4> Transformation </H4><P> gBlocks were ordered from IDT for NadD, NadE, PncB; with a promoter, RBS and TAT signal peptide sequence immediately upstream and the Kill Switch. This was because ccdA was not a part and it was more time efficient to synthesis the Kill Switch than to assemble all the individual components. The NadD, NadE and PncB were ordered as it allowed for the fusion of the TAT signal peptide and our genes of interest, which cannot happen via the 3A assembly system. These sequences contained the iGEM prefix and suffix to be compatible with the 3A system. These were first digested with EcoR1-H1 and Pst1 and ligated into pSBC3 backbone, then transformed into NEB competent E.coli cells. The negative control for the transformations was cells that are not chloramphenicol resistant, the positive control being RFP as it has a clear visual conformation of correct transformation, as can be seen in figure 2. The transformation for the IDT ordered parts did not grow, although the controls worked. This happened multiple times, with different team members performing the transformation. The protocol was altered to increase transformation efficiency but did not produce any notable change. This also occurred from the IDT ordered NadD, NadE and PncB without promoters, RBS or TAT signal peptide sequence as well as the Kill switch part which has the ccdB containing a stop codon. Thus no biobricks were created. ( see figure) </P></DIV><DIV style="color:#fff; background-color: #e74c3c; border-radius: 50px; border-top-right-radius: 0px; border-bottom-left-radius: 0px; padding-left:30px; padding-right:30px; padding-top:30px; padding-bottom:20px; box-shadow: 10px 10px 50px #000000;"><H1 id="Discuss">Discussion</H1><P> One of the main difficulties was the fusion primer for the PncB collapsed multiple times, as can see in figure 3, resulting in the lack of PncB that could be digested, ligated and then transformed. The PCR amplification using the fusion primers for PncB was attempted multiple times, changing the initial annealing temperature to increase the chance of the primers annealing and thus amplifying PncB.  (insert figures) </P><P> The greatest difficulty was the lack of transformants for NadD, NadE, PncB or the TATSP versions, as well as the kill switch, despite the knowing that the transformation worked due to the controls, see figure 4. One problem could have been the restriction sites. It is difficult to tell if the digestion of the IDT sequences worked as there is only a difference of four bases between the digested and the undigested version, thus it is unknown whether the digestion did work. However as the sequences were the iGEM prefix and suffix sequences, this should not be a problem. One way to determine if this is the problem is to blunt end ligate the IDT synthesised DNA to a vector then cut the IDT synthesised DNA out of the vector using the EcoR1-H1 and the Pst1 restriction endonucleases. We did not have time to use this option however. </P><P> Another possible explanation for the lack of desired transformants could be due to the DNA being toxic which is considered highly unlikely as the genes are from E.coli and therefore would not be expected to be toxic. To tell this, we could have transformed the NadD and NadE produced from the PCR which contained the iGEM prefix and suffix to see whether the genes themselves were causing the lack of transformants, or the IDT sequence. However, this was realised too short of a time to the wiki freeze and so has not been done. We did however try the transformations multiple times with slight variations in an attempt to increase the transformation efficiency. This included adding more of the transformation product by adding 500ul instead of 200ul onto the plate. Another way to increase the transformation efficiency was to resuspend the transformants into 200ul of new SOC Media. This however did not have any notable effect on the transformation efficiency of the genes of interest. The volume of DNA added to the competent cells was also increased from 2ul to 20ul and then 100ul. (insert figure) </P><H4> Future Experimental Considerations </H4><P> We would advise transforming the PCR products of NadD and NadE into competent cells to see their effect and whether this is successful. Another consideration would be to lengthen the fusion primers for PncB to create a biobrick for PncB and then transform it into competent cells. </P><P>If the kill switch were to work, it would be advisable to test the kill switch through placing plates containing the bacteria in 37 C and a range of lower temperatures such as 35,30,25,20,10 and to compare the growth, or lack of, of these plates. </P><P>As we were unable to proceed with the colonisation aspect of our project, due to safety restrictions, working in a class 2 laboratory would be beneficial to make and test the biobricks for the colonisation. See the colonisation page for more information. 
</P></DIV></DIV><DIV class="col-sm-2"><A class="twitter-timeline" data-widget-id="327381807010025474" href="https://twitter.com/iGEMLeicester" width="80" height="300">
Tweets by @iGEMLeicester
</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>