<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_McGill_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:McGill/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:McGill"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:McGill/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:McGill/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:McGill/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:McGill/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:McGill/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:McGill/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:McGill/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:McGill/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:McGill/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><A href="https://2018.igem.org/Team:McGill/Safety"><DIV class="menu_item direct_link">
				SAFETY
			</DIV></A><A href="https://igem.org/2018_Judging_Form?team=McGill"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1>Design</H1><P>
To avoid having to test scFvs, which would significantly increase the amount of work needed, we will use
scFv sequences that have already been characterized. Our antigen of choice is the HER2 receptor, due to
the large body of research surrounding this biomarker and its overexpression in 20-25% of breast cancers
(Gianni et al., 2012).

The design of our BiME followed the standard pattern of design laid out for BiTE
using two ScFvs connected by a polypeptide linker (Baeuerle and Reinhardt, 2009). Thus, our first binding
domain is specific to the HER2 receptor. The other binding domain of our BiME antibody is specific to the
notch receptor domain, which in this case is populated by the GCN4 peptide sequence. 
The HER2 scFv was based off the pACgp67B-Her2 vector plasmid, a high affinity ScFv found on Addgene
which is designed to bind to human ERBB2 class molecules, which includes HER2. The scFv region was
identified and the polyhistidine tail removed from the sequence (Song et al., 2005). This was then used as
the first portion of the BiME antibody sequence. The linker sequence is a double length repeat of the
standard (GGGGS)3 linker, creating a (GGGGS)6 linker for greater flexibility between the two binding
domains (Trinh et al., 2004). The scFv for he GCN4 peptide sequence was then based off the pHR-scFvGCN4-sfGFP-GB1-NLS-dWPRE
plasmid
on
addgene,
which
contained
the
scFV
binding
domain
fused
to

GFP
and the
Large
T-Antigen
nuclear
localization
signal
(Tanenbaum
et
al.,
2014).
The
GFP
was
removed

through

cross-referencing with the DNA and protein sequences of GFP (Prasher et al., 1992), and the
nuclear localization signal was retained through the same manner (Hübner et al., 1997). The 6 histidine
polyhistidine tail from the HER2 scFv was added to the end of the GCN4 sequence, before the nuclear
localization signal. A diagram of the entire BiME structure can be seen below. 
 

The sequence for BiME has been optimized for production in E. coli, as both material scFvs were meant to
be produced in E. coli cultures. The HER2 scFv source paper recommended production in the DH5α strain
(Song et al., 2005) while the GCN4 scFv source recommended Stbl3 (Tanenbaum et al., 2014). For
production of BiMe, the DH5α has been chosen because it is readily available and suited to production of
sensitive proteins. Purification will be via IMAC, which our polyhistidine tagged protein will adhere to
with high affinity (Lindner et al., 1992). This allows cheap and relatively simple production of BiME in the
absence of protein synthesis services. The nuclear localization signal will also allow speedy analysis
through importin-α interaction assays (Fanara et al., 2000).
</P><P>The SynNotch protein has an intracellular rtTA system which cleaves from the receptor upon stimulation of the receptor with binding and physical pulling force. The rtTA system, also known as the Tet On/TetR system, was first created by Gossen et al (1995). In the presence of tetracycline, the released factor would be induced to promote transcription at the tight-TRE promoter. This will go on to transcribe GFP, our reporter protein, and through a P2A linker, more rtTA (encoded by the tta sequence) in order to generate a positive feedback loop. In clinical applications, the GFP can be substituted with any conceivable gene, including IFN gamma, TNFalpha, etc. In the continued presence of tetracycline, successfully bound cells will soon express detectable levels of the GFP reporter. </P><P> Our synnotch was an anti-CD19 synnotch construct with intracellular domain tTA. This was uploaded by the authors who had engineered SynNotch (Morsut et al, 2016) and ordered by the team from Addgene. The construct has its original ligand binding domain replaced with anti-CD19, a myc tag added for additional inducibility, and a rtTa intracellular domain that would cleave from the receptor upon activation and circulate in the nucleus. Upon induction with tetracycline, the rtTa transcription factor will bind to its promoter and induce expression. The benefit of this system is that the rtTA and its corresponding promoter are prokaryotic in nature and thus will not induce anything other than the intended target</P><H3> References </H3><P> Baeuerle, P.A., and Reinhardt, C. (2009). Bispecific T-Cell Engaging Antibodies for Cancer Therapy.
Cancer Res. 69, 4941–4944. </P><P>Fanara, P., Hodel, M.R., Corbett, A.H., and Hodel, A.E. (2000). Quantitative Analysis of Nuclear
Localization Signal (NLS)-Importin a Interaction through Fluorescence Depolarization EVIDENCE FOR
AUTO-INHIBITORY REGULATION OF NLS BINDING. J. Biol. Chem. 275, 21218–21223.</P><P>Gianni, L., Pienkowski, T., Im, Y.-H., Roman, L., Tseng, L.-M., Liu, M.-C., Lluch, A., Staroslawska, E.,
de la Haba-Rodriguez, J., Im, S.-A., et al. (2012). Efficacy and safety of neoadjuvant pertuzumab and
trastuzumab in women with locally advanced, inflammatory, or early HER2-positive breast cancer
(NeoSphere): a randomised multicentre, open-label, phase 2 trial. Lancet Oncol. 13, 25–32. </P><P>Hübner, S., Xiao, C.-Y., and Jans, D.A. (1997). The Protein Kinase CK2 Site (Ser111/112) Enhances
Recognition of the Simian Virus 40 Large T-antigen Nuclear Localization Sequence by Importin. J. Biol.
Chem. 272, 17191–17195. </P><P>Lindner, P., Guth, B., Wülfing, C., Krebber, C., Steipe, B., Müller, F., and Plückthun, A. (1992).
Purification of native proteins from the cytoplasm and periplasm of Escherichia coli using IMAC and
histidine tails: A comparison of proteins and protocols. Methods 4, 41–56. </P><P>Prasher, D.C., Eckenrode, V.K., Ward, W.W., Prendergast, F.G., and Cormier, M.J. (1992). Primary
structure of the Aequorea victoria green-fluorescent protein. Gene 111, 229–233.
Song, E., Zhu, P., Lee, S.-K., Chowdhury, D., Kussman, S., Dykxhoorn, D.M., Feng, Y., Palliser, D.,
Weiner, D.B., Shankar, P., et al. (2005). Antibody mediated in vivo delivery of small interfering RNAs via cell-surface receptors. Nat. Biotechnol. 23, 709–717.</P><P>Tanenbaum, M.E., Gilbert, L.A., Qi, L.S., Weissman, J.S., and Vale, R.D. (2014). A Protein-Tagging
System for Signal Amplification in Gene Expression and Fluorescence Imaging. Cell 159, 635–646. </P><P>Morsut, L., Roybal, K.T., Xiong, X., Gordley, R.M., Coyle, S.M., Thomson, M., and Lim, W.A. (2016). Engineering Customized Cell Sensing and Response Behaviors Using Synthetic Notch Receptors. Cell 164, 780–791.
</P><P> Transcriptional activation by tetracyclines in mammalian cells. Gossen M, Freundlieb S, Bender G, Müller G, Hillen W, &amp; Bujard H. Science . 1995 Jun 23;268(5218):1766-9.PubMed. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>