<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Fudan_pro"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Fudan/pro">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Fudan/pro&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Fudan/pro&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Fudan/pro&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Fudan/pro" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Fudan/pro</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>FD iGEM wiki</TITLE></P><DIV class="container_wapper"><DIV id="templatemo_banner_menu"><DIV class="container-fluid"><DIV class="col-xs-4 templatemo_logo"><A rel="nofollow" href="https://2014.igem.org/Team:Fudan"><H1 id="logo_text"><SPAN>iGEM</SPAN>age</H1></A></DIV><DIV class="col-sm-8 hidden-xs"><UL class="nav nav-justified"><LI><A href="https://2014.igem.org/Team:Fudan">Home</A></LI><LI><A class="current" href="#templatemo_about">Project</A></LI><LI><A href="https://2014.igem.org/Team:Fudan#templatemo_team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Fudan/outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Fudan#templatemo_timeline">Timeline</A></LI><LI><A href="#templatemo_contact">Contact</A></LI><LI class="igem_bar dropdown"><A href="javascript:void(0)" class="dropdown-toggle" data-toggle="dropdown">iGEM Bar </A></LI><LI style="height: 50px;"><A href="/wiki/index.php?title=Talk:Team:Fudan&amp;action=edit&amp;redlink=1">Discussion</A></LI><LI style="height: 50px;"><A href="/wiki/index.php?title=Team:Fudan&amp;action=edit">Edit</A></LI><LI style="height: 50px;"><A href="/wiki/index.php?title=Team:Fudan&amp;action=history">History</A></LI><LI style="height: 50px;"><A href="/Special:MovePage/Team:Fudan">Move</A></LI><LI style="height: 50px;"><A href="/wiki/index.php?title=Team:Fudan&amp;action=watch">Watch</A></LI><LI style="height: 50px;"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Fudan">Login</A></LI><LI style="height: 50px;"><A href="https://igem.org/User_Information">Account</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="templatemo_about" class="container_wapper " data-spy="scroll" data-target=".navbar-example"><DIV class="container-fluid"><UL id="nav"><LI class="current"><A href="#p1">Overview</A></LI><LI><A href="#p2">Design</A></LI><LI><A href="#p3">Protocol</A></LI><LI><A href="#p4">Summary&amp;Result</A></LI><LI><A href="#p5">Modeling</A></LI><LI><A href="#p6">Safety</A></LI></UL><H1 style="    margin-bottom: 20px;margin-top: 20px;">About Our Project</H1><DIV class="col-sm-6 col-md-8" style="border: dashed 2px #bd3e25;" id="p1"><H1 class=" text-left"><SMALL style="color:#bd3e25;">Overview&amp;Background </SMALL></H1><P><STRONG>Overview</STRONG></P><P>All levels of life make decisions sometimes, or rather frequently, about what to do in short term and long. Human beings make decisions through thinking, discussing and meetings. Cells don’t talk, they have no brain and don’t hold tea parties where they can gather and chat. Instead, they use chemical signals to communicate with each other, though intracellular network to “think”, and finally alter their expression profiles in response. The high complexity of intracellular signaling network enable cells to make correct decision in correct spatiotemporal condition, and that is crucial to many biological processes, i.e. stress response and metazoan morphogenesis.</P><P>Scientists have worked on this complicated intracellular signaling network for decades, and thanks to that, we now have unveiled many basic motifs that function as basic elements to construct complex network. By creating and combining elements, synthetic biologists have built several functional logic circuits. This summer, inspired by many splendid works in the area of biological circuit design, iGem team of Fudan University carried out a new design and tried hard to work it out. What we attempt to achieve is to create a circuit that can perform all different kinds of 16 Boolean calculation in mammalian cell to help it decide its fate in different drug-induce conditions.</P><P> </P><P><STRONG>Background</STRONG></P><P>Logic circuit design takes a great part in synthetic biology research, attempting to construct convenient modular parts which are capable of performing boolean calculation even within a complex network.thanks to many advances both theoretically and technically, such as the discovery of TALE and invention of Gibson assembly,scientists have made many encouraging progresses recently. Driven by success in general circuit design, more and more scientists start to concern about how to apply general parts and circuits to specific problems, which is widely agreed to have a very promising future. Here we would like to first go through some of the fantastic and inspiring works published these years, to know more about cutting edges in the field of logic circuit design and application before setting up to design our own circuit.</P><P> </P><P><STRONG>Cutting-edge in circuit design</STRONG></P><P>In the field of synthetic biology, we have discovered and constructed lots of information processing elements. By modify exist element and combine them together, scientists have constructed many functional logic circuits. There are several important features that a successful logic circuit should have, such as orthogonality, scaleability and editability. Circuit design in eukaryotic cells is generally more difficult than in bacterium, due to their intrinsic complexity which makes it difficult to meet the features mentioned above.</P><P> </P><P>Many big achievements in circuit design are made in prokaryotic cell, for example the biggest logic circuit report so far, consisting of 11 regulatory proteins and 38 additional genetic parts in total length of 21kb. No logic circuit in eukaryotic cell can be made this big yet. By using molecular chaperones to activate corresponding transcription factors, Voigt et.al.[1]</P><P>screened out 3 orthogonal AND gates and used them to construct a multi-layer circuit, demonstrating the high orthogonality and scaleability of their constructs. Some of the most flexible construct capable of performing multiple logic calculation are also made in prokaryotic cell. The construct Endy et.al.[2]</P><P> </P><P>reported in 2013 is based on intergrase to edit DNA sequence directly. When a single directional transcription terminator flanked by intergrase recognizing sequences is inverted by intergrase, the transcription machinery will be able to move on and express downstream gene. Though placing these terminators properly, users can build many different types of logic gates easily.</P><P> </P><P>Constructing a successful circuit in mammalian cell is a big challenge to synthetic biologists, but we don’t lack brilliant works.[3] To challenge the issue of proper circuit design in eukaryotic cell, many new and complex elements will be needed. RNA-based elements such as ribozyme, riboswitch and aptamer are intensively researched since early 21<SUP>st</SUP> century. Compare to regulatory protein, regulatory RNA have many unique advantages, such as quicker reacting time and easier de novo design. In 2008, Smolke et.al.[4]</P><P>successfully designed a powerful information-processing RNA device based on assembly of three components: sensor, actuator and transmitter, made of RNA aptamer, hammerhead ribozyme and linker sequence respectively. When aptamer sensor detect input drug, it will undergo a conformational change and affect the activity of ribozyme via a transmitter. This work is a very good example in RNA device construction because it provided a editable framework where we can make powerful parts through combining many different elements. Discovery of TALE DNA binding protein in 2009 provided us a very powerful tool to recognize arbitrary DNA sequence and boosted many fields of research, including synthetic biology. Early this year, R.Jerala et.al.[5] </P><P>reported a TALE-based system enable users to design almost any logic gate he want. By using TALE to block target promoter, we can build the NAND gate first, and then all other types of logic gates can be constructed by combining multiple NAND gates in correct order. The high orthogonality and editability of this system is really noteworthy, especially in the context of mammalian cell.</P><P> </P><DIV><DIV id="ftn5"><P>There are still many excellent works about circuit design, the examples we introduce there are only parts of them. Reviewing all these successful circuit construction, they all have very good orthogonality, scaleability and editability. If our team want to make a good design, then these features should be taken into consideration carefully.</P><P> </P><P>Reference:</P><OL><LI>Voigt, CA. et.al. (2012). “Genetic programs constructed from layered logic gates in single cells”. Nature 491(7423): 249-253.</LI><LI>Endy, D. et.al. (2013). “Amplifying genetic logic gates”. Science 340(6132): 599-603.</LI><LI>Fussenegger, M. et.al. (2013). “Biomedically relevant circuit-design strategies in mammalian synthetic biology”. Mol Syst Biol 9(691)</LI><LI>Smolke, CD. et.al. (2008). “Higher-Order Cellular Information Processing with Synthetic RNA Devices”. Science 322(5900): 456-460.</LI><LI>Jerala, R. et.al. (2014). “designable dna-binding domains enable construction of logic circuits in mammalian cells”. Nat Chem Biol. 10(3): 203-208.</LI><LI>Benson, J.D. et.al. (2009). “Single-vector inducible lentiviral RNAi system for oncology target validation”. Cell Cycle. 8: 498-504.</LI><LI>Nagy A (2000). &quot;Cre Recombinase:The Universal Reagent for Genome Tailoring&quot;. Genesis 26: 99–109.</LI><LI>Hayashi, S &amp; McMahon, AP. (2002). “Efficient Recombination in Diverse Tissues by a Tamoxifen-Inducible Form of Cre: A Tool for Temporally Regulated Gene Activation/Inactivation in the Mouse”. Dev Biol 244(2): 305-318.</LI><LI>Martick M. &amp; Scott WG. (2006). “Tertiary contacts distant from the active site prime a ribozyme for catalysis”. Cell 126(2):309-320</LI><LI>Smolke CD. et.al. (2010). “Genetic control of mammalian T-cell proliferation with synthetic RNA regulatory systems”. Proc Natl Acad Sci 107(19):8531-6.</LI><LI>Yokobayashi Y. et.al. (2009). “Conditional RNA interference mediated by allosteric ribozyme”. J Am Chem Soc 131(39):13906-7.</LI><LI>Scott WG. et.al. (2008). “A discontinuous hammerhead ribozyme embedded in a mammalian messenger RNA”. Nature 454(7206):899-902.</LI><LI>Holt RA. et.al. (2006). “A high-throughput screen identifying sequence and promiscuity characteristics of the loxP spacer region in Cre-mediated recombination”. BMC Genomics 7:73.</LI></OL></DIV></DIV><P><STRONG>Cell transdifferentiation</STRONG></P><P>Cell therapy is of significant importance to regenerative medicine, which needs through specific lineage cell to access. The most efficient way to access the target cell is to access through cell transdifferentiation or cell reprogramming, and there are normally three methods to achieve it. The first one is that four transcription factors OSKM(Oct4,Sox2,Klf4 and c-Myc) were transfected into somatic cells by using lenti virus as the carrier, then somatic cells were reprogrammed into iPScs. After reprogramming, the iPScs were further differentiated to become many different types of specialized cells. It is worth noting that Gurdon and Yamanaka got the 2012’s Nobel price for this ground-breaking work. The second way is using lenti virus to carry different DNA that can cause the cell transdifferentiation to transfect each lineage-specific transcription factors directly into somatic cells. In this case, it can achieve a direct cell transdifferentiation way to get all kinds of specific lineage cells. The last method is to reprogram the cells into multipotent stem cells first, and then transferred through a specific transcription factor, or chemical induction, to obtain various types of cells.</P><P>However, these three methods have their pros and cons. The first method, seems perfect to introduce four transcription factors into a differentiated cell and sufficiently return it into a pluripotent situation. However, it still requires many steps to differentiate this “most versatile cell” into specific types. Due to the existence of a strong oncogene c-Myc, the so-called omnipotent iPSc is not absolute security, its cells’ oncoenicity and fidelity remains to be tested. The second method, as early as 1987, Davis used the overexpressed transcription factor myoD’s cDNA successfully transdifferentiate MEF-mouse embryonic fibroblast into myoblast. Vierbuchen also successfully overexpressed Ascl1,Brn2 and Myt1l in MEF and got mature neuron in 2010. By knocking down PTB in MEF, Xue Y also got mature neuron in 2013. But over these years, it’s more difficult to get real lineage specific cell, which is similar to the mature cell, directly from somatic cell transdifferentiation. The reason is that the influence of few transcription factors is limited to cell transdifferentiation. In addition, the cells obtained by this method are too simple to make decisive assistance to the formation of mature tissue, and the cells barely have self renewing ability because of terminal differentiation.</P><P>The third method is the middle stage of the first two methods, it contains the advantages in the first two, but also obtain the disadvantages of them.</P><P>So we hope that through stably transfected a super plasmid named “iGemage” into mammalian cells, which contains four transcription factors OSKM （Oct4,Sox2,Klf4 and c-Myc）through connecting by T2A linker (used to induce reprogramming iPSc ) , with Sox2 gene ( used to induce reprogramming of inducible neural stem cells iNSc ) as well as with the PTB shRNA gene ( for transdifferentiation into neurons ) . It will be achieved the conversion of cells in various states by adding two different drugs --dox and tamoxifen.</P></DIV><DIV class="col-sm-6 col-md-8" style="border: dashed 2px #334d5c;" id="p2"><H1 class=" text-left"><SMALL style="color:#334d5c;">Design </SMALL></H1><P>The goal we want to achieve this year is to construct a logic circuit which can perform as many kinds of boolean calculation as possible while keeping the circuit as tiny as possible to make it much more useful when served as a brick to construct larger scale system.</P><P>Let’s draw a coordinate graph first, mark X-axis as Input1, Y-axis as Input2. There we can see four regions representing different input state: low [X] low [Y], high [X] low [Y], low [X] high [Y] and high [X] high [Y], named region A/B/C/D correspondingly. If a report signal only shine in region D in certain circuit, then we could say this circuit is a AND GATE of two input, and if report signal appear in both region B&amp;D in certain circuit, then this would be a XOR GATE. In similar way, we can list all common logic gates and its corresponding graph.</P><P>What if there exists a circuit that it can separate these four regions, in other word, each region within the circuit can decide its own report state? Obviously, that would be a circuit which is capable of performing all kinds of common boolean calculations ideally, and that is what we want!</P><P>So far we have set the goal to design a circuit take X/Y as two input and A/B/C/D as four output, namely a 2-input-4-output circuit where 4 outputs show relative high orthogonality. If we can achieve it successfully, more complex and multi-parallel boolean calculation could  be constructed within single layer using multiple kinds of outputs’ signal.</P><P>Now we have a goal, what we have to do next is to find a motif that can serve as blue print in circuit design.</P><P> </P><P><STRONG>Search for Motif</STRONG></P><P>Consider a motif or combination of several motifs that can perform all 2-input boolean calculation. In other words, what kind of motif would perform 2-input-4-output calculation? Our answer is the bi-fan motif. Given input X/Y and output A/B/C/D, through drawing following lines that X activate B/D while repress A/C, Y activate C/D while repress A/B, we can get a network which is derived from bi-fan motif containing 2 input elements , 4 output elements and  8 lines represent the interaction between these elements.</P><P>To figure out the motif which underline the boolean operator we need is essential for circuit design. Once we figure out the motif that serves as formula for our logic circuit design, the only thing remains to be done is to match certain biological processes to all corresponding lines in this network.</P><P>Before setting about puzzle solving, we have to be aware of one thing first. It needs 8 lines in total to construct this multi-functional logic circuit, which means lots of inter-elemental interactions should be packed within the circuit we design. But we have no intention of making one single circuit which is too big to scale up. A very compact array of functional elements is required.</P><P> </P><P><STRONG>Fill in Elements</STRONG></P><P>In order to match biological processes which corresponding to lines that represent some sorts of interactions, we must make it clear that what are the objects of these actions, namely, what X/Y/A/B/C/D represent. As the output of the system, A/B/C/D is less restricted; almost anything that can be inserted into a expression casette could be a candidate of A/B/C/D, such as protein, siRNA, ncRNA. For X/Y, in mammalian cell, generally we would think of two different inputs as two different drugs. So we screened mammalian inducible expression system, and finally chose Dox &amp; TM(tamoxifen, a kind of drug) as input X &amp; Y. Now let’s see what’s in the network: Tet activate B/D and repress A/C, while TM activate C/D and repress A/B.</P><P>It’s time to fill in the lines, but before that, we need to know more about how these expression system work to complete the puzzle.</P><P>For X, namely Tet, there are two distinct expression systems available: Tet-on and Tet-off. We chose Tet-on system because it’s relatively easy to manipulate in circuit design --- all you need to do about activation lines is to put B&amp;D behind a Tet-on expression promoter. As for repression lines, we managed to use Tet-inducible shRNA to knockdown A&amp;C.</P><P>For Y, namely TM, the core of TM expression system is the Cre recombinase, which can recognize loxP sequence and perform two distinct sequence-editing activities. When two loxP sequences share the same direction, Cre will remove the sequence between two loxP along with one loxP, and when two loxP sequences face toward each other, Cre will inverse the sequence between them and damage both loxP to make sure the intermediate sequence won’t be inverse back again. [picture, Cre bi-activity] In TM inducible expression system, The Cre recominase is modified by adding an ER binding domain, which will automatically adhere to ER, thus Cre will be located on ER where it has no activity. When TM is added to the system, ER binding domain will be blocked, thus Cre will then be able to drop off the ER and travel into nuclear to execute its recombinase activity. Bi-activity is indeed an interesting feature, and we managed to make use of it by designing two different backbones applying loxPs with different direction to them, named “excision model” and “inversion model” correspondingly.</P><P>In “excision model”, two loxPs face the same direction, intermediate sequences will be excised, which makes repression lines of TM relatively easy to draw --- you only need to put the repressed elements into the excised sequence. When we refer to activation lines, things start to be a little difficult: in traditional TM expression system, a barricading sequence is inserted into the intermediate sequence to prevent target protein from expression, but in our “excision model”, there would be a lot of things lie within the intermediate sequence, such as promoters and expression cassetes, which make it better not to transcribe the entire intermediate sequence. How to achieve TM-on when transcription have to be stopped somewhere between two loxP? Our solution is to use self-cleave ribozyme.</P><P>Ribozyme is a RNA sequence that can execute enzymatic activity, mostly endo-nuclease activity. There are many examples where scientists successfully use and modify drug-inducible ribozyme to perform boolean calculation. The ribozyme we use is a hammerhead self-cleave ribozyme, derived from 3’UTR of mouse Clec2 gene. If a mRNA strand contain a ribozyme, it will undergo degradation before it can be translated. Based on that, we plan to put this ribozyme into the intermediate sequence between two loxPs to degrade C&amp;D. When TM is added, sequence between two loxPs will then be excised, along side with ribozymes. Thus C&amp;D transcripts will be able to survive and translate.</P><P>Compare to “excision model”, things are less straight forward in “inversion model”, so we skip to show how we carry it out step by step, but give our answer to this puzzle directly later in next paragraph. </P><P>By the way, Cre recombinase actually recognizes not only loxP sequence, but also some of its variant, like lox2272. To be honest, circuit design would be a lot easier if we use several pairs of orthogonal loxP variants. Yet we challenged a tougher brainstorming that only one kind of loxP is allowed to use in both of our models.</P><P> </P><P><STRONG>Overview of our Design</STRONG></P><P>There we show the schematic diagrams of both Excision-model and inversion-model we designed. Both of these models are designed in a very compact way, involving transcription in both sense and anti-sense direction. Each model contains 4 MCSs representing 4 separated regions A/B/C/D we’ve discussed above. User can insert target gene selectively to construct desired logic gate. For example, if we insert RFP in all MCS_B/C/D as well as a GFP linked tandemly behind RFP in MCS D, then we can get a circuit calculating OR gate for RFP and AND gate for GFP simultaneously. Quite convenient, isn’t it!</P><P>Let’s move on and take a closer look at how these models operate, starting with excision-model.</P><P>In low-[Tet]-low-[TM] condition, pTRE promoters are silenced. And transcripts derived from MCS_C will undergo rapid degradation due to existent of ribozyme in their 3’UTR, leaving transcripts from MCS_A be the only active mRNA capable of translating.</P><P>In high-[Tet]-low-[TM] condition, all pTRE promoters start to work. Tet-induciable shRNA targeting A&amp;C starts to knock them down. In two kinds of remaining transcripts derive from pTRE, only C can survive while D carrying a ribozyme in its 3’UTR.</P><P>In low-[Tet]-high-[TM] condition, Cre reconbinase is activated by TM. Therefore sequence between two loxPs will be excised, while all pTRE will go for strike unless Tet is added, leaving C the only active transcription cassete free to express.</P><P>In high-[Tet]-high-[TM] condition, every pTRE that went for strike, back to its position due to present of Tet. Among them there is shRNA targeting C, thus D becomes the only survivor to report its signal.</P><P>excision-model is quite straight forward as explained in “Fill in Elements” section. Yet inversion-model is trickier. Let’s forget about region A and consider a 2-input-3-output inversion-model first.</P><P>In high-[Tet]-low-[TM] condition, only B is transcribed and translated. When it comes to low-[Tet]-high-[TM] condition where pTRE is off，activated Cre recombinase recognizes and inverses two loxP along with sequence bedded within, turning CMV promoter toward MCS_C and start transcription.</P><P>In high-[Tet]-high-[TM] condition, sequence between two loxPs is inverted and both CMV and pTRE are ready to transcribe their new down stream neighbor. Transcripts of C are then knocked down by Tet-induced shRNA, leaving D as final reporting signal.</P><P>Where should we add MCS_A into this 2-input-3-output sketch? We can simply put MCS_A ahead of pTRE, where it can be transcribed initially until CMV is inverted and face MCS_C.However, it is reported that polymerase collision may have effect on transcription, and this is what exactly going to happen in high-[Tet]-low-[TM] condition. To avoid polymerase collision, we designed a “plan B” by adding an orthogonal loxP variant, namely lox2272.</P><P>In planB, MCS_B and MCS_A is placed between one loxP and one lox2272 right downstream their corresponding promoters, theoretically avoid polymerase collision. Cre recombinase will execute an inversion followed by an excision when TM is added, presenting new neighbors to both pTRE promoter and CMV promoter.</P><P> </P><P><STRONG>Application</STRONG></P><P>
Here we have the circuit and next step is to figure out how to use it. The main feature of this circuit, whether in form of remove-model or inversion-model, is that it can make different response to 4 different permutations of 2 orthogonal input signals, namely separate 4 different input states. This feature will be very useful when operate properly. </P><P>The very first idea come to us is to make it a multi-output parallel logic gate. As described in [Design] section, the only thing we need to do to construct a single-output logic gate, such as an AND gate, is to insert target reporter gene into expression cassete D and do nothing to all other cassetes. Base on similar law, almost every common logic gate can be constructed. If we use some linker in expression cassetes, such as T2A linker, to produce multiple output reporter signal within single expression cassete, then we will get a circuit performing boolean calculation for multiple output simultaneously within only one layer. 
</P><P>Inducible cell differentiation:</P><P>As the “2 inputs and 4 outputs” logic circuit is completed, it will be applied to make a cell conversion machine named “iGemage”. In that machine, we replaced Gene A with OSKM(Oct4,Sox2,Klf4 and c-Myc), the four genes that can induce somatic cells to be reprogrammed into iPScs. Gene B was replaced with Sox2, which can induce somatic cells to be reprogrammed into iNScs.</P><P>When the substitution is made, cell transdifferentiation will be achieved by stably transfected the super plasmid “iGemage” with simply adding drugs. Dosing the cells with Tet/dox will make shSox2 and shPTB expressed, so that PTB gene (polypyrimidine tract binding protein 1 ) will be knockdown, inducing MEF cells to transdifferentiate to Neuron. Adding TM to the cell growth medium will make Sox2 overexpressed, inducing MEF cells to reprogramme to iNScs. Adding dox and TM together will make four transcription factors OSKM (Oct4,Sox2,Klf4 and c-Myc) through connecting by T2A linker overexpressed, inducing MEF cells to reprogramme to iPScs.</P><P align="center"> </P></DIV><DIV class="col-sm-6 col-md-8" style="border: dashed 2px #efc84a;" id="p3"><H1 class=" text-left"><SMALL style="color:#efc84a;">Protocol </SMALL></H1><P><A href="https://static.igem.org/mediawiki/2014/5/55/Fudan_prot.pdf">Protocol ：Direct conversion from MEF to neuron: download</A></P><P><STRONG>Biobrick Construction</STRONG></P><P><STRONG>Brief Introduction</STRONG></P><P>To create this biobrick, we utilized a combination of overlap PCR and thermostable ligase chain reaction (TLCR) to extract a sequence that contains all the functional parts to build our system. The brief introduction of this procedure is listed below.</P><P>PCR to generate gene fragments:</P><P>In this step, all separate fragments of the final system are amplified from several plasmids via PCR. We add necessary overlapping sites to the 5' and 3' ends when designing primers.</P><P>Details see PCR Method\Pfu PCR.</P><P>The PCR product is then purified.</P><P>Details see DNA purification\Xygen Gel Extraction.</P><P>Overlap PCR to make all fragments into three parts:</P><P>Every part is made from four or five fragments. In each part, every two fragments have complementary overlapping sites, and all of them are connected one by one through PCR.</P><P>details see PCR methods\overlap PCR.</P><P>The overlap PCR products are purified then.</P><P>Details see DNA purification\Exygen gel extraction.</P><P>Thermostable ligase chain reaction (TLCR) to insert each part into the vector.</P><P>All sticky ends of these three parts should be completed by T4 ligase first. Then each part is inserted into a vector pBluescriptIISk by TLCR, with the help of helpers. Helpers are DNA strains that contain the same sites of two to-be-connected ends of two DNA fragments.</P><P>We planned to insert all three parts in one vector through TLCR. However, it never works well, maybe due to too many repetitive sequences in part one. So we ended up in inserting every part in one vector separately and test separately.</P><P>Details see PCR methods\TLCR.</P><P>This method is originally created by Lu Daru’s lab. Thanks for professor Lu’s instruction.</P><P>Next, we cut the functional part down from plasmids made above by enzyme digestion; then insert them respectively into official backbone pSB1c3, and pcDNA3.1/myc-His (-) for eukaryotic expression test.</P><P>Details see Plasmid Editing\Double Digest、Plasmid Editing\T4 Ligase Ligation.</P><P> </P><P><STRONG>DNA purification</STRONG></P><P>Protocol for Agarose Gel Electrophoresis</P><P>1. Weigh agarose powder and TAE buffer according to a proper portion, and add them to a 100mL conical flask(we usually make 1.5% Agarose Gel);</P><P>2. Melt the mixture in a microwave until the solution becomes clear(don’t leave the microwave);</P><P>3. Let the solution cool down to about 40-50℃ and add DNA gel stain, pour the solution into the gel casting tray with appreciate comb;</P><P>4. Let the gel cool until it is solid;</P><P>5. Carefully pull out the comb;</P><P>6. Place the gel in the electrophoresis chamber;</P><P>7. Add enough TAE Buffer so that there is about 2-3 mm of buffer over the gel;</P><P>8. Pipette DNA samples mixed with appreciate amount of loading buffer and dye(GeneFinder) into wells on the gel;</P><P>9. Run the gel at 120V for about twenty minutes</P><P>Axygen gel extraction</P><P>1. Prepare agarose gel (1.5%), use thick-welled comb.</P><P>2. Run desired sample on gel.</P><P>3. Excise the agarose gel slice containing the DNA fragment of interest with a clean, sharp scalpel under ultraviolet illumination.</P><P>4. Weigh gel slice (tare with empty tube). Add 3 volumes of DE-A buffer per mg of gel (so a 100mg gel gets 300 uL of ADB buffer).</P><P>5. Resuspend the gel in Buffer DE-A by vortexing. Heat at 75°C until the gel is completely dissolved (typically, 6-8 minutes). Heat at 40°C if low-melt agarose gel is used. Intermittently vortexing (every 2-3 minutes) will accelerate gel solubilization.</P><P>Note: Gel must be completely dissolved or the DNA fragment recovery will be reduced. Do not heat the gel for longer than 10 minutes.</P><P>6. Add 0.5x Buffer DE-A volume of Buffer DE-B, mix. If the DNA fragment is less than 400 bp, supplement further with a 1x sample volume of isopropanol.</P><P>7.Place a Miniprep column into a 2 ml microfuge tube (provided). Transfer the solubilized agarose from Step 6 into the column. Centrifuge at 12,000xg for 1 minute.</P><P>8. Discard the filtrate from the 2 ml microfuge tube. Return the Miniprep column to the 2 ml microfuge tube and add 500 μl of Buffer W1. Centrifuge at 12,000xg for 30 seconds.</P><P>9. Discard the filtrate from the 2 ml microfuge tube. Return the Miniprep column to the 2 ml microfuge tube and add 700 μl of Buffer W2. Centrifuge at 12,000xg for 30 seconds.</P><P>10. Discard the filtrate from the 2 ml microfuge tube. Place the Miniprep column back into the 2 ml microfuge tube. Add a second 700 μl aliquot of Buffer W2 and centrifuge at 12,000xg for 1 minute.</P><P>11. Discard the filtrate from the 2 ml microfuge tube. Place the Miniprep column back into the 2 ml microfuge tube. Centrifugeat 12,000xg for 1 minute.</P><P>12. Transfer the Miniprep column into a clean 1.5ml microfuge tube (provided). To elute the DNA,add 25-30μl of Eluent or deionized water to the center of the membrane. Let it stand for 1 minuteat room temperature. Centrifuge at 12,000xg for 1 minute.</P><P>Note: Pre-warming the Eluent at 65°C will generally improve elution efficiency.</P><P>Measure the concentration on the nanodrop.</P><P>Tiangen Miniprep</P><P>1. Prewash the spin column by adding 0.5 ml Buffer BL and centrifuging for 30–60 s. Discard the flow-through.</P><P>2. Resuspend pelleted bacterial cells in 250 µl Buffer P1 (kept at 4 °C) and transfer to a microcentrifuge tube.</P><P>Note: Ensure that RNase A has been added to Buffer P1. No cell clumps should be visible after resuspension of the pellet.</P><P>3. Add 250 μl Buffer P2 and gently invert the tube 6-8 times to mix.</P><P>Mix gently by inverting the tube. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear. Do not allow the lysis reaction to proceed for more than 5 min.</P><P>4. Add 350 μl Buffer P3 and invert the tube immediately but gently 6-8 times.</P><P>To avoid localized precipitation, mix the solution gently but thoroughly, immediately after addition of Buffer P3. The solution should become cloudy.</P><P>5. Centrifuge for 10-12 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.</P><P>A compact white pellet will form.</P><P>6. Apply the supernatants from step 4 to the QIAprep spin column by decanting or pipetting.</P><P>7. Centrifuge for 30–60 s. Discard the flow-through.</P><P>Spinning for 60 seconds produces good results.</P><P>8. Wash spin column by adding 0.75 ml Buffer PE and centrifuging for 30–60 s.</P><P>Spinning for 60 seconds produces good results.</P><P>9. Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer.</P><P>10. Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 μl Buffer EBor water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.</P><P>Note:  Pre-warming the Eluent at 55°C will generally improve elution efficiency.</P><P>Measure the concentration on the nanodrop.</P><P> </P><P><STRONG>PCR methods</STRONG></P><P>Pfu PCR</P><P>1. Thaw Pfu, dNTP, primers, template DNA on ice.</P><P>2. To a new PCR tube, add:</P><P>10×Pfu Buffer: 5μl</P><P>Forward primer: 0.2-1.0 μM</P><P>Reverse primer: 0.2-1.0 μM</P><P>dNTP (10mM each) : 1.0 μl</P><P>Template DNA: 1-50ng (for plasmid)</P><P>                       10ng-1μg (for genome)</P><P>SinoBio Pfu: 0.25μl (1.25U)</P><P>ddH2O:  add up to 50 μl</P><P>The volumes vary according to our need under the instruction.</P><P>3. Mix solution well.</P><P>4. Place tube in PCR thermocycler. Set thermocycler program:</P><P>Initial Denaturation: 90s at 94°C ;</P><P>Loop (30-35 cycles), Denaturation: 30s at 94°C ,</P><P>Annealing: 30s at *(see below),</P><P>Elongation: ** (see below) at 72°C;</P><P>Final Elongation: 15 min at 68°C.</P><P>Store: 12°C. (not for too long)</P><P>*: Calculate Tm based on annealing region: (4(#G or C)+2(#A or T))C</P><P>**: Calculate Extension Time based on length of product: 60s/1kb</P><P>(Pfu DNA polymerase from Novoprotein Ltd. We use Pfu when we want high quality, such as low mutant rate, PCR products.)</P><P> </P><P>Taq PCR</P><P>1. Thaw Taq, dNTP, primers, template DNA on ice.</P><P>2. To a new PCR tube, add:</P><P>5×Taq Buffer: 10μl</P><P>Forward primer: 0.2-1.0 μM</P><P>Reverse primer: 0.2-1.0 μM</P><P>dNTP (10mM each) : 1.0 μl</P><P>Template DNA: 1-50ng (for plasmid)</P><P>                       10ng-1μg (for genome)</P><P>SinoBio Taq: 0.25μl (1.25U)</P><P>ddH2O:  add up to 50 μl</P><P>The volumes vary according to our need under the instruction.</P><P>3. Mix solution well.</P><P>4. Place tube in PCR thermocycler. Set thermocycler program:</P><P>Initial Denaturation: 90s at 94°C ;</P><P>Loop (30-35 cycles), Denaturation: 30s at 94°C ,</P><P>Annealing: 30s at *(see below),</P><P>Elongation: ** (see below) at 72°C;</P><P>Final Elongation: 15 min at 68°C.</P><P>Store: 12°C. (not for too long)</P><P>*: Calculate Tm based on annealing region: (4(#G or C)+2(#A or T))C</P><P>**: Calculate Extension Time based on length of product: 60s/1kb</P><P>(Taq DNA polymerase from Novoprotein Ltd. We use Taq when we are not sure about the best reaction condition.)</P><P> </P><P>HiFi PCR</P><P>1. Thaw Hifi, dNTP, primers, template DNA on ice.</P><P>2. To a new PCR tube, add:</P><P>5×KAPA HiFi Buffer: 5.0μl</P><P>Forward primer: 0.3 μM</P><P>Reverse primer: 0.3 μM</P><P>dNTP (10mM each) : 0.5 μl</P><P>Template DNA: 1-50ng (for plasmid)</P><P>                       10ng-1μg (for genome)</P><P>KAPA HiFi: 0.5μl (0.5U)</P><P>ddH2O:  add up to 25 μl</P><P>The volumes vary according to our need under the instruction.</P><P>3. Mix solution well.</P><P>4. Place tube in PCR thermocycler. Set thermocycler program:</P><P>Initial Denaturation: 2-5min at 95°C ;</P><P>Loop (30-35 cycles), Denaturation: 20s at 98°C ,</P><P>Annealing: 15s at *(see below),</P><P>Elongation: ** (see below) at 72°C;</P><P>Final Elongation: 1-5 min at 75°C.</P><P>Store: 12°C. (not for too long)</P><P>*: Calculate Tm based on annealing region: (4(#G or C)+2(#A or T))C</P><P>**: Calculate Extension Time based on length of product: 30s/1kb</P><P style="margin-left:21.0pt;">(KAPA HiFi DNA polymerase from Beijing Microread Genetics Co.,Ltd. We use HiFi when there are many CGs.)</P><P> </P><P>Overlap PCR</P><P>PCR amplify and connecting the necessary fragments, using proofreading polymerase enzyme.</P><P>They should have about 15-25 bp overlaps. Use oligo Tm calculators to figure out their annealing temp.</P><P>Clean up or gel extract the correct size band.Use cleaned up fragments as &quot;template&quot;. Unlike normal PCR, about 1/2 to 3/4 volume of the extension reaction should be template.</P><P>Use proofreading enzyme for extension.</P><P>Run 10-15 PCR cycles without end primers. (Template extension step)</P><P>Add end primers, then continue cycling for another 15-20 rounds.</P><P>Gel extract the correct fragment. Clone into a vector</P><P> </P><P>TLCR (Thermostable Ligase Chain Reaction)</P><P>All sticky ends should be completed by T4 ligase first.</P><P>As for primers when we mix PCR solution, only forward primer of the first DNA fragment and the reverse primer of the last DNA fragment are added. Then we additionally add helper DNA as well. (Helpers are DNA strains that contain the same sites of two to-be-connected ends of two DNA fragments.) The rest adding of the solution is almost the same with normal PCR.</P><P>Note: both forward and reverse primer should be phosphorylated by enzyme PNK, since T4 ligase would end up completion with no phosphate group at the end of DNA strains.</P><P>All rights about this reaction reserved by Lu Daru’s lab.</P><P>(both T4 ligase and PNK are from NEB)</P><P> </P><P><STRONG>Plasmid Editing</STRONG></P><P>    Plasmid Edting\Double Digest</P><P>We conduct double digest under the instruction of thermo scientific online, and the reaction condition varies according to different combination of enzymes.</P><P> </P><P>Plasmid Edting\T4 Ligase Ligation</P><P>1. Set up the following reaction in a microcentrifuge tube on ice.</P><P>(T4 DNA Ligase should be added last. Note that the table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.) </P><P>COMPONENT      20 μl REACTION</P><P>10X T4 DNA Ligase Buffer*       2 μl</P><P>Vector DNA (4 kb)                          50 ng (0.020 pmol)</P><P>Insert DNA (1 kb)                      37.5 ng (0.060 pmol)</P><P>T4 DNA Ligase                        1 μl</P><P>Nuclease-free water                      up to 20 μl</P><P>2. * The T4 DNA Ligase Buffer should be thawed and resuspended at room temperature.</P><P>3. Gently mix the reaction by pipetting up and down and microfuge briefly.</P><P>4. For sticky ends, incubate at 16°C overnight or room temperature for 10 minutes.</P><P>5. For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours (alternatively, high concentration T4 DNA Ligase can be used in a 10 minute ligation).</P><P>6. Heat inactivate at 65°C for 10 minutes.</P><P>7. Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells.</P><P>(T4 ligase from NEB)</P><P> </P><P><STRONG>Experimental Procedure</STRONG></P><P><STRONG>Mice</STRONG></P><P>We got an ICR mice from the SLAC laboratory animal(http://www.slaccas.com/product/Index.asp?MainClassId=10&amp;SubClassId=11).</P><P> </P><P><STRONG>Cell Culture</STRONG></P><P>iPS cells were maintained on feeder layer of MEFs. The iPS cells were maintained in DMEM contain 10% FBS, 1% penicillin/streptomycin, 1ng/mL doxycycline, 0.01% LIF.</P><P>We used the 13.5-days-pregnant mice to isolate MEFs from its uteri. Wash the embryos with phosphate-buffered saline(PBS). Cut the heads and mince the remained bodies with scissors. Add 0.25% trypsin and incubate in 37°C for 7mins in the tubes. After trypsinization, the medium(containing 10% FBS, 1% penicillin/streptomycin) was added and pipetted up and down to help the tissue dissociation. Cells were collected by centrifugation(1000rpm for 3mins at room temperature) and resuspended in fresh normal medium. We used the 150mm dishes at 37°C with 5% CO2 to culture MEFs.</P><P> </P><P><STRONG>Retroviral Infection</STRONG></P><P>We used the plasmid from Tongji iGEM team. It is TetO-FUW-OSKM from addgene. This is a lentiviral plasmid for tet-inducible expression of mouse Oct4, Sox2, Klf4 and Myc for iPS cell generation.</P><P>The protocol is similar with</P><P> iNSC and Neuron’s protocol.</P><P>0 d infected with lentivirus</P><P>0.5 d removed the transfection medium and add the normal medium</P><P>1.5 d removed the normal medium and add the normal medium with doxycycline</P><P>3.5 d removed normal medium and and mES medium with doxycycline and LIF</P><P align="center">Until the clones could be observed by naked eyes.</P></DIV><DIV class="col-sm-6 col-md-8" style="border: dashed 2px #334d5c;" id="p4"><H1 class=" text-left"><SMALL style="color:#334d5c;">Summary&amp;Result  </SMALL></H1><P>Cells of all organisms process numerous signals originating from the internal biological processes or from the environment to produce the appropriate cellular response<A href="#_ftn1" name="_ftnref1" title="">[1]</A>.Signal integration or pathway in cells are just like logic circuits in electronic or digital system. Previous work that designed gene regulatory circuits have already demonstrated the abilities to perform basic logic functions in both prokaryotic and eukaryotic cells.<A href="#_ftn2" name="_ftnref2" title="">[2]</A><A href="#_ftn3" name="_ftnref3" title="">[3]</A> However, these previous fantastic designs all have drawbacks or problems.</P><P>  This year, we Fudan iGem team tried to solve some problems of these work or designs. So we design a new kind of genetic logic circuit, which works only in eukaryotic cells, based on tet-on shRNA expression<A href="#_ftn4" name="_ftnref4" title="">[4]</A> system and Cre-ERTII system<A href="#_ftn5" name="_ftnref5" title="">[5]</A> with the help of ribozyme. This logic circuit can do all 16 boolean logic gates calculation ideally. We will test all three elements by using fluorescent proteins.</P><P>  For application, we considered using this design to make an “iGemage” ,which is a kind of cell conversion machine. By this machine, we can get three types of cells -- iPSc (inducible pluripotent stem cell),iNSc(inducible neural stem cell) and diffentiated neuron by dosing the cells with two kinds of drugs: Dox and Tamoxifen. We believe this machine will be beneficial to both cell therapies and regenerative medcine. </P><P><STRONG>Result</STRONG></P><P><STRONG>Picture</STRONG></P><P align="center"> </P><DIV> <DIV id="ftn1"><P><A href="#_ftnref1" name="_ftn1" title="">[1]</A> Nat Chem Biol. 2014 Mar;10(3):203-8.</P></DIV><DIV id="ftn2"><P><A href="#_ftnref2" name="_ftn2" title="">[2]</A> Nature. 2012 Nov 8;491(7423):249-53.</P></DIV><DIV id="ftn3"><P><A href="#_ftnref3" name="_ftn3" title="">[3]</A> Nat Chem Biol. 2014 Mar;10(3):203-8.</P></DIV><DIV id="ftn4"><P><A href="#_ftnref4" name="_ftn4" title="">[4]</A> Wiederschain D, Wee S, Chen L, et al.  Cell Cycle, 2009, 8(3): 498-504.</P></DIV><DIV id="ftn5"><P><A href="#_ftnref5" name="_ftn5" title="">[5]</A> Leone D P, Atanasoski S, Grausenburger R, et al. T 2003, 22(4): 430-440.</P></DIV></DIV></DIV><DIV class="col-sm-6 col-md-8" style="border: dashed 2px #58ad5f;" id="p5"><H1 class=" text-left"><SMALL style="color:#58ad5f;">Model </SMALL></H1><P><STRONG>Model</STRONG></P><P>In order to explain and model our final design—a “2-input-3-output” logic circuit, we introduce the neural network algorithm and establish our own neural network which consists of an input layer, a hidden layer and an output layer.</P><P>Firstly, the input layer is composed of two units. The value of each unit can be chosen between 0 and 1, and when the value is chosen as 1, it presents that a specific input signal is added into our system. Accordingly, 0 presents that the specific input signal has not been introduced to our logic circuit. Here, our input signals are different kinds of small molecular which can regulate gene expression or the activation of its target proteins. And based our final design, we define Input-1 as tetracycline (Tet) and Input-2 as Tamoxifen(TM).</P><P>Secondly, the hidden layer is comprised of seven units that are chosen and established dependent on their biological roles. It should be noted that there are only four functional elements (a pair of Cre-Loxp sites, a H1/T0 promoter and two tet-on promoters) regulated by our two input signals, so we have three “bias units” in the hidden layer, which represents two ribozymes and a constitutive promoter respectively. Therefore, we get a biological functions-dependant hidden layer and it help this model become vividly.</P><P>In the last layer of our neural network, we have three units respectively representing three different kinds of output results which depend on different input signals. After comparing the value of each unit in the last layer with the threshold we choose (Here, we choose 0.5 as our threhold), we can obtain the anticipation result, and in our design, the results correspond to cell fate.</P><P>In addition, we also entrust biological significance to both weight matrixes. The parameter a<SUP>i</SUP><SUB>jk </SUB>(i represents the no. of the weight matrix; j represents the no. of the unit in the former layer; k represents the no. of the unit in the former layer) is defined as a measurement of the effect that the specific unit in the former layer has on the given unit in the latter layer. In detail, a<SUP>i</SUP><SUB>jk </SUB>can be 1, 0 or -1, and 1 presents that the effect was positive, 0 presents that there is no relationship between the two units and -1 shows that the unit in the former layer is a negative regulator of the specific unit in the latter layer. For example, Tamoxifen has no effect on promoters induced by tetracycline, so we set a<SUP>1</SUP><SUB>24</SUB>, a<SUP>1</SUP><SUB>25</SUB>, a<SUP>1</SUP><SUB>26</SUB> as 0. The ribozyme-a in the hidden layer can lower the expression level of gene-A significantly. Therefore, a<SUP>2</SUP><SUB>11</SUB> equals -1. a<SUP>2</SUP><SUB>32</SUB> equals 1, because CMV is a constitutive promoter and promotes the expression of gene-B.</P><P>We use parameter f<SUP>i</SUP><SUB>jk </SUB>(both the meanings and the value of i, j, k are the same as those in a<SUP>i</SUP><SUB>jk</SUB>) to represent the efficiency of the act between the specific pair of units (The efficiency has been normalized by a constitutive promoter. We will show below. ). For instance, we insert ribozyme-a into the 3’UTR of gene-A and the cleavage efficiency of ribozymes is about 70%. We set f<SUP>2</SUP><SUB>11</SUB> as 0.7. If there is no relationship between two units set in the contiguous layers respectively, the corresponding f<SUP>i</SUP><SUB>jk</SUB> can be defined as 0. For the range of all of the parameters being in a small interval, there is no need to introduce a complicated function such as function sigmod which is often used in the field of machine learning to normalize the result and decrease the consumption of calculation to some extent therefore.</P><P>We can get the anticipation result of our design by some simple formulations listed below:</P><P>For the simplicity and flexibility of our model, everyone can use our model to test their designs or anticipate the results before taking pains to build their logic circuits. Furthermore, people can also use our model to explaining their own logic circuits directly just by training the parameters by their own training sets.</P><P>To conclude, we build a special neural network completely simulating and reflecting the true biological process of the logic circuit we establish. This design breaks the common configuration of the neural network and has low calculation consumption. In addition, our model is a good sample for modeling a such logic circuit or even a more complicated one, and we will continue developing this model to make it more accurate and more robust.</P><P>Note:</P><P>The method to measure the efficiency of all the elements and our system</P><P>We measure the efficiency of each of our functional elements by testing the activity of the fluorescent reporter (We first constructed plasmids containing a reporter gene for each functional element to test each one’s efficiency and used different fluorescent reporters to substitute gene-A, gene-B and gene-C to test the efficiency of the all logic circuit. For more detail, you can check other parts of our wiki). To normalize the efficiency of all the elements in our system, we utilize relative expression units (REUs)<SUP>【</SUP><SUP>1</SUP><SUP>】</SUP>, and the activities are normalized by the intensity of a GFP whose expression is controlled by a constitutive promoter. In detail, for different promoters, direct comparison is enough. For ribozymes and siRNAs which execute the cleavage of mRNA, we just need to do some simple calculation to get the efficiency. For the recombinant enzyme Cre, although it’s function is more complicated than the other functional elements, few steps of calculation and analysis are sufficient. For the efficiency of induction, we can introduce a concentration gradient in the cell culture to find the best condition.</P><P>For example:</P><P>Reference:</P><P align="left" style="margin-left:3pt;">(1) Moon, T. S. and C. Lou, et al. (2012). &quot;Genetic programs constructed from layered logic gates in single cells.&quot; Nature 491 (7423): 249-53.</P><P> </P></DIV><DIV class="col-sm-6 col-md-8" style="border: dashed 2px #4c9192;" id="p6"><H1 class=" text-left"><SMALL style="color:#4c9192;">Safety </SMALL></H1><P align="left">This year, we Fudan iGEM team worked on a project strongly associated with regenerative medicine and cell therapy.</P><P align="left">As is known to all, not before long when people get physiological damage, the only way to cure the damage and save the life is to expect a healthy organ donated by people who would like to offer it. However, studies shows that human body generate immunological rejection when exotic tissue or organ is transplanted. In addition, it is unpractical to cure neurodegenerative disease like Alzheimer's disease or Parkinson disease by cerebral transplantation. Developments in Regenerative Medicine are expected.</P><P>Each and every one of live plant cells is of totipotency, while most mammalian cells are even of no self-renewing ability. However, a kind of cell named ”stem cell” is of differentiation and division ability, which exists in mammalian organism almost everywhere. Stem cells not only help the organism get new fresh blood, but also save the tissue or organ when they are broken. What’s more, every organism comes from one stem cell—the most totipotent stem cell named oosperm. Stem cells are so important for they all have so much potential to both protect and develop the organism, as a result of which makes stem cells themselves be considered as a kind of therapy for regenerative medicine. </P><P align="left" style="margin-left:18pt;">Before 2006, scientists can only get mammalian cells of totipotency by obtaining the stem cells in the embryonic period during which stem cells get full totipetency. Experiments can be done with animal embryonic stem cells, but</P><P align="left" style="margin-left:18pt;">obtaining embryonic stem cells from human body is never allowed. It causes serious ethical and bioethical problems. In 2006, Yamanaka and his colleagues successfully make fibroblasts (somatic cells) reprogram into iPSC (inducible</P><P align="left" style="margin-left:18pt;">pluripotent stem cell) by stably transfecting four transcription factors (Oct4, Sox2, Klf4, c-Myc) with the help of lenti virus. This achievement not only made him get Nobel prize in 2012, but also arouse a new trend in researches of stem</P><P align="left" style="margin-left:18pt;">cell therapy, since severe bioethical and ethical problems avoided. What’s more, iPSc technology made the road of regenerative medicine more bright, for it could successfully solve the immunological problems too. What a great biological</P><P align="left" style="margin-left:18pt;">technology!</P><P align="left"> However, Things were far more than we expected. iPSc technology, limited by itself, not only for it needs lenti virus, which may be harmful to cells or organism, to help cells get four transcription factors stably overexpressed, but also for c-Myc, a strong oncogene, which made oncologists headache for many years, will also be powerfully overexpressed in this system. Unfortunately, iPSc did cause tumors successfully according to some recent reports. What’s more, iPSc also cause many teratoma’s formation, as a result of which makes iPSc not be safe anymore. Then many scientists decided to work on the project which can make cell conversion avoid of becoming tumors or causing cancers. They began to be focused on the research on direct somatic cell transdifferentiation or cell reprogramming to multipotent stem cells such as iNSc (inducible multipotent neural stem cell) etc. These two methods apparently are safer than iPSc technology. For direct somatic cell transdifferentiation, what we got were just well-differentiated cells, such as neuron, myoblast, which almost do not have any self-renewing ability. For cell reprogramming to multipotent stem cells, some reports insisted that these cells almost have no oncogenicity<A href="#_ftn1" name="_ftnref1" title="">[1]</A>.</P><P>For the reasons we talked about above, we Fudan iGem team tried to make a new kind of machine for cell conversion. Many choices for cells’ fate will be offered with the help of our machine “iGEmage”, and it will be safer for stem cell therapy because all the elements we designed will make this system controllable. In addition, there is no doubt that our machine will also evade many bioethical problems.</P><P>We offer a new machine for cell therapy ,and all we need is one of your somatic cells.</P><DIV> <DIV id="ftn1"><P><A href="#_ftnref1" name="_ftn1" title="">[1]</A> Ring K L, Tong L M, Balestra M E, et al. Direct reprogramming of mouse and human fibroblasts into multipotent neural stem cells with a single factor[J]. Cell stem cell, 2012, 11(1): 100-109.</P></DIV></DIV></DIV></DIV><DIV id="templatemo_contact" class="container_wapper"><DIV class="container-fluid"><H1>Contact</H1><DIV class="col-md-5 "><H2>For more information:</H2><P>Cao Xuanye
			Leader of 2014FudaniGEMage</P><P><STRONG>Email:</STRONG>12307110004@fudan.edu.cn<STRONG>Phone:</STRONG> 86-13818791002<STRONG>Website:</STRONG> www.igem.org<STRONG>Address:</STRONG> 137Room, Liren Bio-hall, Fudan University, 220# Handan Road, Yangpu Area,
 Shanghai City. </P></DIV></DIV></DIV><DIV id="templatemo_footer"><DIV><P>Copyright © 2014 Fudan Igem <EM>Zhu Liu</EM></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Fudan/pro">http://2014.igem.org/Team:Fudan/pro</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Fudan/pro" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Fudan/pro" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Fudan/pro&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Fudan/pro&amp;oldid=311336" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>