Team:Shenzhen SFLS/References
Home
Project
Background
Overview
References
Design
Protein Engineering
Confirmatory
Result
Parts
Modeling
Human Practice
Achivements
Practices
Social outreach
Collaborations
Notebook
Problems and Solutions
Protocals
Safety
Team
Shenzhen_SFLS
Members 2015
Attributions
（1）Gaurav Tiwari, Ruchi Tiwari, [...], and Saurabh K Bannerjee, Drug delivery systems: An updated review, Int J Pharm Investig. 2012 Jan-Mar; 2(1): 2–11. doi:  10.4103/2230-973X.96920 PMCID: PMC3465154 （2）Aaron C. Anselmo, Samir Mitragotri, An Overview of Clinical and Commercial Impact of Drug Delivery Systems, Journal of Controlled Release (2014), doi: 10.1016/j.jconrel.2014.03.053 （3）James T. Wu, c-erbB2 oncoprotein and its soluble ectodomain: a new potential tumor marker for prognosis early detection and monitoring patients undergoing Herceptin treatment, Clinica Chimica Acta (2002) （4）Image from the RCSB PDB (www.rcsb.org) of PDB ID 1BNA (H.R. Drew, R.M. Wing, T. Takano, C. Broka, S. Tanaka, K. Itakura, R.E.Dickerson (1981) Structure of a B-DNA dodecamer: conformation and dynamics Proc.Natl.Acad.Sci.USA 78: 2179-2183). （5）Zuhaida Asra Ahmad, Swee Keong Yeap,Abdul Manaf Ali, Wan Yong Ho, Noorjahan Banu Mohamed Alitheen, and Muhajir Hamid, scFv Antibody: Principles and Clinical Application, Clinical and Developmental Immunology, Volume 2012, Article ID 980250, 15 pages, doi:10.1155/2012/980250 （6）Gasteiger E., Hoogland C., Gattiker A., Duvaud S., Wilkins M.R., Appel R.D., Bairoch A.; Protein Identification and Analysis Tools on the ExPASy Server; (In) John M. Walker (ed): The Proteomics Protocols Handbook, Humana Press (2005).  pp. 571-607  Full text - Copyright Humana Press.  ProtParam tool （7）Gasteiger E., Hoogland C., Gattiker A., Duvaud S., Wilkins M.R., Appel R.D., Bairoch A.; Protein Identification and Analysis Tools on the ExPASy Server; (In) John M. Walker (ed): The Proteomics Protocols Handbook, Humana Press (2005).  pp. 571-607 (8)"The PredictProtein server." Rost B., Yachdav G., Liu J. Nucleic Acid Res. 32: 321-326 (2004) "Topology prediction for helical transmembrane proteins at 86% accuracy." Rost B., Fariselli P., Casadio R. Protein Sci. 5: 1704-1718 (1996) "Transmembrane Helix Prediction in Proteins." unpublished undefined: undefined-undefined (2014) "ISIS: interaction sites identified from sequence." Ofran Y., Rost B. Bioinformatics. 23: e13-e16 (2007) "Improvement of DNA- and RNA-Protein Binding Prediction" Peter Hönigschmid, Edda Kloppmann, Burkhard Rost undefined undefined: undefined-undefined (2012) "DISULFIND: a disulfide bonding state and cysteine connectivity prediction server." Ceroni A., Passerini A., Vullo A., Frasconi P. Nucleic Acids Res. 34: 177-181 (2006) "Finding nuclear localization signals." Cokol M., Nair R., Rost B. EMBO Rep. 1: 411-415 (2000) "Improved Disorder Prediction by Combination of Orthogonal Approaches." Schlessinger A., Punta M., Yachdav G., Kajan L., Rost B. PLoS ONE 4: e4433-e4433 (2009) "Predicting transmembrane beta-barrels in proteomes." Bigelow HR., Petrey DS., Liu J., Przybylski D., Rost B. Nucleic Acids Res. 32: 2566-2577 (2004) "Homology-based inference sets the bar high for protein function prediction." Hamp T., Kassner R., Seemayer S., Vicedo E., Schaefer C., Achten D., Auer F., Boehm A., Braun T., Hecht M., Heron M., Honigschmid P., Hopf TA., Kaufmann S., Kiening M., Krompass Denis., Landerer C., Mahlich Y., Roos M., Rost B BMC Bioinformatics (Suppl. Automated Function Prediction Meeting 2011). 14: Suppl. 3: S7-Suppl. 3: S7 (2013) "Gapped BLAST and PSI-BLAST: a new generation of protein database search programs." Altschul SF., Madden TL., Schäffer AA., Zhang J., Zhang Z., Miller W., Lipman DJ. Nucleic Acids Res. 25: 3389-3402 (1997) (9)Structural Insights into the Down-regulation of Overexpressed p185her2/neu Protein of Transformed Cells by the Antibody chA21, Huihao Zhou, Zhao Zha, Yang Liu, Hongtao Zhang, Juanjuan Zhu, Siyi Hu, Guodong Shen, Liansheng Cheng, Liwen Niu, Mark I. Greene, Maikun Teng and Jing Liu, doi: 10.1074/jbc.M111.235184 originally published online June 16, 2011, J. Biol. Chem. 2011, 286:31676-31683 (10)Benkert, Ku¨ nzli and Schwede, “QMEAN server for protein model quality estimation”, Nucleic Acids Research, 2009, Vol. 37, Web Server issue Published online 8 May 2009, doi:10.1093/nar/gkp322 (11)SWISS-MODEL Workspace: Marco Biasini, Stefan Bienert, Andrew Waterhouse, Konstantin Arnold, Gabriel Studer, Tobias Schmidt, Florian Kiefer, Tiziano Gallo Cassarino, Martino Bertoni, Lorenza Bordoli, Torsten Schwede (2014). SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information Nucleic Acids Research 2014 (1 July 2014) 42 (W1): W252-W258;   Arnold K, Bordoli L, Kopp J, and Schwede T (2006). The SWISS-MODEL Workspace: A web-based environment for protein structure homology modelling. Bioinformatics.,22,195-201.   Bordoli, L., Kiefer, F., Arnold, K., Benkert, P., Battey, J. and Schwede, T. (2009). Protein structure homology modelling using SWISS-MODEL Workspace. Nature Protocols, 4,1.  SWISS-MODEL Repository: Kiefer F, Arnold K, Künzli M, Bordoli L, Schwede T (2009). The SWISS-MODEL Repository and associated resources. Nucleic Acids Res. 37, D387-D392.   Kopp J, and Schwede T (2006). The SWISS-MODEL Repository: new features and functionalities. Nucleic Acids Res.,34, D315-D318. SWISS-MODEL and Swiss PdbViewer Guex, N., Peitsch, M.C. Schwede, T. (2009). Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective. Electrophoresis, 30(S1), S162-S173. (12)Förster, Theodor (1948). "Zwischenmolekulare Energiewanderung und Fluoreszenz" [Intermolecular energy migration and fluorescence]. Annalen der Physik (in German) 437: 55–75. Bibcode:1948AnP...437...55F. doi:10.1002/andp.19484370105. Claude Berney and Gaudenz Danuser, “FRET or No FRET: A Quantitative Comparison”, Biophysical Journal Volume 84 June 2003 3992–4010 (13)Harris, Daniel C. (2010). "Applications of Spectrophotometry". Quantitative Chemical Analysis (8th ed.). New York: W. H. Freeman and Co. pp. 419–44. ISBN 978-1-4292-1815-3.
