<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Project_Summary"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Project/Summary">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Project/Summary&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Summary&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Summary&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Project/Summary" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Project/Summary</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team">Team</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Members">Members</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Project</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Summary</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Highlights">Highlights</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Microviridin">     Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Phage_Display">     Phage Display</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#In_Vivo_Selection">     In Vivo Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Modeling">     Modeling</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Microviridin">Details Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">Details Phage</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">Details Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Modeling">Details Modeling</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">BioBricks</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/BioBricks">Details BioBricks</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Methods">Methods</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Meetings">Meetings</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics">Safety &amp; Ethics</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Assesment">Safety Assessment</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Questions">Safety Questions</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Ethics_Seminar">Ethics Seminar</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Survey">Survey</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Potsdam_Bioware_at_the_German_parliament">Parliament Visit</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Statement_of_the_German_government">     Statement</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Outreach">Outreach</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software">Software</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#BioLog_App">BioLog App</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#Download">Download</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Sponsors">Sponsors</A></DT></DL></LI></UL></DIV><DIV class="pageContent"><H2><SPAN class="mw-headline" id="Summary"> Summary </SPAN></H2><H3><SPAN class="mw-headline" id="Modification.2C_Selection_and_Production_of_Cyclic_Peptides_for_Therapy">Modification, Selection and Production of Cyclic Peptides for Therapy</SPAN></H3><P>One key task of biopharmaceuticals is the binding and blocking of deregulated proteins. Picking the right lead structure for biopharmaceuticals is very important for success. This year, we developed a novel system for the modification, selection and optimization of peptides showing potential for protease inhibition. These promising inhibitors are found in cyanobacteria and are called Microviridins. They belong to a class of peptides characterized by unusual ω-ester and ω-amide bonds between the amino-acid side chains, which are also referred to as depsipeptides. These modifications are introduced post translationally by a set of enzymes and result in an extraordinary tricyclic cage structure. 

Proteases are a large enzyme family and comprise about 647 human gene products. Therefore they are important drug targets. In addition, many harmful bacteria, viruses and fungi use proteases in their reproduction cycle and growth. The ability to block these proteases is highly relevant for therapy. Prominent examples of targets are the angiotensin-converting enzyme (ACE) and HIV proteases.
We chose a 6.5 kb Microviridin gene cluster named <I>mdnABCDE</I> from <I>Mycrocystis aeruginosa</I> NIES843. The <I>mdn</I> gene cluster comprises (i) a gene encoding a precursor peptide named MdnA, which is then modified to form the Microviridin, (ii) two genes encoding ATP-grasp-type ligases named <I>mdnB</I> and <I>mdnC</I>, (iii) an ABC- transporter encoding gene named <I>mdnE</I> as well as (iv) one gene encoding an N- acetyltransferase of the GNAT family named <I>mdnD</I> (Ziemert et al., 2008). Our major aim was to modify the MdnA precursor peptide and optimize its protease inhibiting properties. Towards this goal, we synthesized semi-rational <I>mdnA</I> gene libraries with partially randomized oligonucleotides. One library was cloned, verified by sequencing, and successfully used for selection.

To identify the best candidates inhibiting various given proteases, we established two different selection systems: Phage Display and a new developed <I>in-vivo</I> selection assay. Phage display is a frequently used technique in laboratories, yet to our knowledge, phage display of cellularly cyclized peptides is new. We constructed a fusion between the surface protein geneIII of the phage and the <I>mdnA</I> gene within the <I>mdnA</I> gene cluster. This system was verified by western blotting, Phage-ELISA and controlled phage panning experiments.

We also devised and constructed a recombinant <I>in-vivo</I> selection system linking protease degradation to antibiotic resistance. This system is divided into three parts: first a protease activity detector device, second a protease generator device, and third a protease blocking device. For the protease activity detector device we fused various protease cleavage sites between a signal sequence and the antibiotic resistance conferring enzyme β-lactamase. β-lactamase provides resistance when its transferred into the periplasm. Thus, when the protease cleaves the signal sequence from the β-lactamase antibiotic resistance is abolished and cells die under selective pressure. However, when our Microviridin variants inhibit the protease, cells survive, allowing for an easy and efficient selection of protease inhibitors. We demonstrated a wide dynamic range of the system between expressed and non-expressed protease in the presence of increasing ampicillin concentrations. Importantly, after co-transformation with our <I>mdnA</I>-libary we were able to isolate several clones, which are currently being characterized.

In addition to the practical work, we established a mathematical model of the <I>in-vivo</I> selection system. This model, which we were able to fit to experimental data, helped us to understand the selection process. We modeled the reaction kinetics with ordinary differential equations and simulated and fitted data with matlab.

Last but not least, we had several human practice projects. We send a survey to all members of the German parliament, visited a member of the parliament, held seminars on ethics and invited children to the lab.
</P><UL><LI>Ziemert, N., Ishida, K., Liaimer, A., Hertweck, C. &amp; Dittmann, E. (2008). Ribosomal synthesis of tricyclic depsipeptides in bloom-forming cyanobacteria. Angewandte Chemie (International ed. in English) 47, 7756-9
</LI></UL><H2><SPAN class="mw-headline" id="Highlights"> Highlights </SPAN></H2><H3><SPAN class="mw-headline" id="Microviridin"> Microviridin </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1:</B><B>HPLC chromatogram of heterologously expressed mdnA</B></DIV></DIV></DIV><P>
The microviridin sub-project aimed for modifications at the genetic level such that protease inhibiting  activities of the resulting peptides are enhanced. We used random mutagenesis or focused randomization of oligonucleotides for the creation of gene libraries, which can be screened for <I>mdnA</I>-variants with therapeutically promising mutations. For further experiments, we also fused mdnA to a myc-tag.
In addition, all coding genes of the <I>mdn</I>-cluster were converted to BioBricks. The microviridin from <I>mdnA</I> was purified and characterized by HPLC and cyclization was verified by mass spectrometry.
For the quick expression of several library clones, we also built auxiliary plasmid backbones with inducible promoters according to a proposed extension of the iGEM cloning standard.
<B><A href="/Team:Potsdam_Bioware/Project/Details_Microviridin" title="Team:Potsdam Bioware/Project/Details Microviridin">[more]</A></B></P><H3><SPAN class="mw-headline" id="Phage_Display"> Phage Display </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption"><B>Figure 2: Phage Display of microviridin.</B> Enrichment of MdnA carrying phages upon panning towards a protease was demonstrated.</DIV></DIV></DIV><P>
Phage display is a powerful tool for  selecting peptides or proteins that bind and regulate the function of target proteins. Phages robustly link the genotype to the phenotype of the presented peptide allowing for selection of protease binders under controlled <I>in vitro</I> conditions. We established genetics and panning of therapeutically interesting <I>mdnA</I>-libraries towards a panel of several purified proteases. First, the general suitability of phage display for this purpose was shown. An appropriate phagemid vector harboring an <I>mdnA-myc-geneIII</I> fusion gene, was constructed. The expression of the MdnA-myc-geneIII protein in <I>E. coli</I> was shown by western blot. Production of phage particles carrying MdnA were analyzed by Phage-ELISA. Finally, phage display procedures were optimized and an enrichment of MdnA modified phages over a control was shown. <B><A href="/Team:Potsdam_Bioware/Project/Details_Phage" title="Team:Potsdam Bioware/Project/Details Phage">[more]</A></B></P><H3><SPAN class="mw-headline" id="In_Vivo_Selection"> In Vivo Selection </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption"><B>Figure 3: Ampicillin resistance test.</B> Survival screen, without induced (blue) and with induced (magenta) protease. From left to right increasing ampicillin concentrations are shown.</DIV></DIV></DIV><P>A proper designed <I>in vivo</I> selection is very powerful, quick and entails an inherent selection against general toxicity. We designed and employed a multi-component system enabling inexpensive and time-saving selection of microviridin libraries blocking various proteases. Our genetic circuit links with high sensitivity and wide dynamic range a protease generator, a protease detector and a protease inhibitor device. This is achieved by coupling the export of the enzyme ß-lactamase to a signal sequence via an interspersed modularly cloned protease cleavage site. Inducible lactamase and protease generators are united on a single plasmid while the <I>mdnA</I> library is provided in trans. Functionality of the system was characterized under a multitude of conditions for different proteases. Ultimately, a one step selection of a genetic <I>mdnA</I> library against the protease TEV with increasing concentrations of the antibiotic ampicillin yielded inhibiting clones.  <B><A href="/Team:Potsdam_Bioware/Project/Details_Selection" title="Team:Potsdam Bioware/Project/Details Selection">[more]</A></B></P><H3><SPAN class="mw-headline" id="Modeling"> Modeling </SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4:</B><B>Plot predicting lactamase concentrations in the periplasm</B> (and thus the cell fittness) in dependence of the enzyme inhibition reaction coefficient K<SUB>D</SUB>.</DIV></DIV></DIV><P>
There is no synthetic biology without modeling, of course. We focused on systems modeling of our <I>in vivo</I> selection system in which reaction kinetics are analyzed and outcomes are predicted or fitted to experimental data. On one hand, predictions based on known parameters enable the rational choice of optimized conditions (e.g. induction levels of the protease generator). On the other hand, fitting to biological readouts (e.g. survival) enables deciphering of intrinsic parameters (e.g. interactions constants).
Reactions were coded as ordinary differential equations under consideration of sequential induction time points. Substance concentrations were numerically propagated through time. A useful dynamic range and also a certain robustness of our system were confirmed. We learned about correct time-scales for triggering and we predicted cell-division rates as a reference for the lab work. In a final step, we fitted our model to wet-lab measurements. <B><A href="/Team:Potsdam_Bioware/Project/Details_Modeling" title="Team:Potsdam Bioware/Project/Details Modeling">[more]</A></B></P><H3><SPAN class="mw-headline" id="Ethics"> Ethics </SPAN></H3><P>For information about an ethics seminar and a survey among politicians see <B><A href="/Team:Potsdam_Bioware/Safety_Ethics" title="Team:Potsdam Bioware/Safety Ethics">[here]</A></B>.
</P><H3><SPAN class="mw-headline" id="Software"> Software </SPAN></H3><P>Have a look at the features and screenshots of our BioLog app on this <B><A href="/Team:Potsdam_Bioware/Software" title="Team:Potsdam Bioware/Software">[page]</A></B>.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">http://2011.igem.org/Team:Potsdam_Bioware/Project/Summary</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Project/Summary" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Project/Summary" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Summary&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Summary&amp;oldid=261414" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>