<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Labjournal_October"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Labjournal/October">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Labjournal/October&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/October&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/October&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Labjournal/October" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Labjournal/October</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team">Team</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Members">Members</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Project</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Summary</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Highlights">Highlights</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Microviridin">     Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Phage_Display">     Phage Display</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#In_Vivo_Selection">     In Vivo Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Modeling">     Modeling</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Microviridin">Details Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">Details Phage</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">Details Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Modeling">Details Modeling</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">BioBricks</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/BioBricks">Details BioBricks</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Methods">Methods</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Meetings">Meetings</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics">Safety &amp; Ethics</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Assesment">Safety Assessment</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Questions">Safety Questions</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Ethics_Seminar">Ethics Seminar</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Survey">Survey</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Potsdam_Bioware_at_the_German_parliament">Parliament Visit</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Statement_of_the_German_government">     Statement</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Outreach">Outreach</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software">Software</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#BioLog_App">BioLog App</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#Download">Download</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Sponsors">Sponsors</A></DT></DL></LI></UL></DIV><DIV class="pageContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#98th_Labday_2011-09-24"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">98th Labday 2011-09-24</SPAN></A></LI><LI class="toclevel-2"><A href="#Colony_PCR_of_E.coli_XL1_blue_transformed_UP_SG2_.281.5.25.29"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Colony PCR of E.coli XL1 blue transformed UP_SG2 (1.5%)</SPAN></A></LI><LI class="toclevel-2"><A href="#Subcloning_of_14_3C_and_AraC_into_SG2_.281.5.25.29"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Subcloning of 14 3C and AraC into SG2 (1.5%)</SPAN></A></LI><LI class="toclevel-2"><A href="#Production_of_phages_containing_pPDV089_in_XL1-blue_cells"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Production of phages containing pPDV089 in XL1-blue cells</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#99th_Labday_2011-09-25"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">99th Labday 2011-09-25</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis_of_maxiprimer_I"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Agarose gel electrophoresis of maxiprimer I</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_phages_containing_pPDV089"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Purification of phages containing pPDV089</SPAN></A></LI><LI class="toclevel-2"><A href="#Maxiprimer_PCR_I_.E2.80.93_Amplification_of_the_end_of_the_mdn_cluster_.28Repetition.2C_usage_of_two_different_polymerases.29"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Maxiprimer PCR I – Amplification of the end of the mdn cluster (Repetition, usage of two different polymerases)</SPAN></A></LI><LI class="toclevel-1"><A href="#100th_Labday_2011-09-26"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">100th Labday 2011-09-26</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_Phage_Display_with_another_trypsin_protease"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Repeated Phage Display with another trypsin protease</SPAN></A></LI><LI class="toclevel-2"><A href="#ELISA_with_purified_phages"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">ELISA with purified phages</SPAN></A></LI><LI class="toclevel-2"><A href="#Control_of_Phage_Display"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Control of Phage Display</SPAN></A></LI><LI class="toclevel-1"><A href="#101th_Labday_2011-09-27"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">101th Labday 2011-09-27</SPAN></A></LI><LI class="toclevel-2"><A href="#SDS-_PAGE_for_coomassie_staining_with_purified_phages_carrying_mdnA_on_their_surface"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">SDS- PAGE for coomassie staining with purified phages carrying mdnA on their surface</SPAN></A></LI><LI class="toclevel-2"><A href="#Coomassie_staining"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Coomassie staining</SPAN></A></LI><LI class="toclevel-1"><A href="#102th_Labday_2011-10-06"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">102th Labday 2011-10-06</SPAN></A></LI><LI class="toclevel-2"><A href="#Amplification_of_megaprimers_for_site-directed_mutagenesis"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Amplification of megaprimers for site-directed mutagenesis</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis_and_DNA_extraction_of_megaprimers"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Agarose gel electrophoresis and DNA extraction of megaprimers</SPAN></A></LI><LI class="toclevel-2"><A href="#Megaprimer_PCR_for_site_directed_mutagenesis"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Megaprimer PCR for site directed mutagenesis</SPAN></A></LI><LI class="toclevel-1"><A href="#103th_Labday_2011-10-07"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">103th Labday 2011-10-07</SPAN></A></LI><LI class="toclevel-2"><A href="#DpnI_digestion_and_Transformation_of_mutated_pARW089"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">DpnI digestion and Transformation of mutated pARW089</SPAN></A></LI><LI class="toclevel-2"><A href="#Retransformation_of_XL1blue-cells_with_pPDV089"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Retransformation of XL1blue-cells with pPDV089</SPAN></A></LI><LI class="toclevel-1"><A href="#104th_Labday_2011-10-11"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">104th Labday 2011-10-11</SPAN></A></LI><LI class="toclevel-2"><A href="#Oligo-Fillin_for_Phage-display-mdnA-library_.28carrying_AgeI_restriction_site.29"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Oligo-Fillin for Phage-display-mdnA-library (carrying AgeI restriction site)</SPAN></A></LI><LI class="toclevel-2"><A href="#Troubleshooting:_Site-directed_mutagenesis_of_pARW089_.28for_use_as_biobrick.29"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Troubleshooting: Site-directed mutagenesis of pARW089 (for use as biobrick)</SPAN></A></LI><LI class="toclevel-2"><A href="#Megaprimer_PCR_for_site_directed_mutagenesis_2"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Megaprimer PCR for site directed mutagenesis</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_ligation_product_geneIII_into_pARW089_.28pARWIII.29_for_ligation_with_generated_mdnA_library"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">Digestion of ligation product geneIII into pARW089 (pARWIII) for ligation with generated mdnA library</SPAN></A></LI><LI class="toclevel-2"><A href="#DpnI_digestion_and_Transformation_of_mutated_pARW089_2"><SPAN class="tocnumber">7.5</SPAN><SPAN class="toctext">DpnI digestion and Transformation of mutated pARW089</SPAN></A></LI><LI class="toclevel-2"><A href="#Over_night_culture_of_retransformed_pPDV089"><SPAN class="tocnumber">7.6</SPAN><SPAN class="toctext">Over night culture of retransformed pPDV089</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_geneIII_for_ligation_with_generated_mdnA_library"><SPAN class="tocnumber">7.7</SPAN><SPAN class="toctext">Digestion of geneIII for ligation with generated mdnA library</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_mutated_pARW089_with_increased_amount"><SPAN class="tocnumber">7.8</SPAN><SPAN class="toctext">Transformation of mutated pARW089 with increased amount</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_mdnA_library_.28for_phage_display.29_using_AgeI"><SPAN class="tocnumber">7.9</SPAN><SPAN class="toctext">Digestion of mdnA library (for phage display) using AgeI</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_digested_mdnA_library_and_digested_gene_III"><SPAN class="tocnumber">7.10</SPAN><SPAN class="toctext">Purification of digested mdnA library and digested gene III</SPAN></A></LI><LI class="toclevel-2"><A href="#Triple_Ligation_with_pARWIII.2C_mdnA-library_and_gene_III"><SPAN class="tocnumber">7.11</SPAN><SPAN class="toctext">Triple Ligation with pARWIII, mdnA-library and gene III</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_triple_ligated_mdnA_library.2C_gene_III_and_pARWIII"><SPAN class="tocnumber">7.12</SPAN><SPAN class="toctext">Transformation of triple ligated mdnA library, gene III and pARWIII</SPAN></A></LI><LI class="toclevel-2"><A href="#Sidedirected_mutagenesis_of_HRV14_3C_protease"><SPAN class="tocnumber">7.13</SPAN><SPAN class="toctext">Sidedirected mutagenesis of HRV14 3C protease</SPAN></A></LI><LI class="toclevel-2"><A href="#Analytical_gelelctrophoresis_and_PCR_clean_up_of_sidedirected_mutated_fragments_P1.2C_P2.2CP3_and_P4_of_HRV14_3C"><SPAN class="tocnumber">7.14</SPAN><SPAN class="toctext">Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments P1, P2,P3 and P4 of HRV14 3C</SPAN></A></LI><LI class="toclevel-2"><A href="#Assembly_PCR_of_mutated_HRV14_3C_protease_fragments"><SPAN class="tocnumber">7.15</SPAN><SPAN class="toctext">Assembly PCR of mutated HRV14 3C protease fragments</SPAN></A></LI><LI class="toclevel-2"><A href="#Analytical_gelelctrophoresis_and_PCR_clean_up_of_sidedirected_mutated_fragments_P5_and_P6_of_HRV14_3C"><SPAN class="tocnumber">7.16</SPAN><SPAN class="toctext">Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments P5 and P6 of HRV14 3C</SPAN></A></LI><LI class="toclevel-2"><A href="#Assembly_PCR_of_mutated_HRV14_3C_protease_fragments_P5_and_P6"><SPAN class="tocnumber">7.17</SPAN><SPAN class="toctext">Assembly PCR of mutated HRV14 3C protease fragments P5 and P6</SPAN></A></LI><LI class="toclevel-1"><A href="#105th_Labday_2011-10-12"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">105th Labday 2011-10-12</SPAN></A></LI><LI class="toclevel-2"><A href="#Production_of_phages_containing_pPDV089_in_XL1-blue_cells_2"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Production of phages containing pPDV089 in XL1-blue cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Production_of_microviridin"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Production of microviridin</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_HRV14_3C.2C_AraC_and_Vector_SG11"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Digest of HRV14 3C, AraC and Vector SG11</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_digested_fragments_of_HRV14_3C_protease_and_vector_SG11_and_Transformation_of_XL1_blue_with_Ligation_products"><SPAN class="tocnumber">8.4</SPAN><SPAN class="toctext">Ligation of digested fragments of HRV14_3C protease and vector SG11 and Transformation of XL1 blue with Ligation products</SPAN></A></LI><LI class="toclevel-1"><A href="#106th_Labday_2011-10-13"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">106th Labday 2011-10-13</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation_of_samples_for_HPLC_analysis"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Preparation of samples for HPLC analysis</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_mutated_pARW089_using_super_competent_XL1_gold_cells"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Transformation of mutated pARW089 using super competent XL1 gold cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_phages_containing_pPDV089_2"><SPAN class="tocnumber">9.3</SPAN><SPAN class="toctext">Purification of phages containing pPDV089</SPAN></A></LI><LI class="toclevel-2"><A href="#Colony_PCR_of_plated_HRV14_3C_containing_colonies"><SPAN class="tocnumber">9.4</SPAN><SPAN class="toctext">Colony PCR of plated HRV14_3C containing colonies</SPAN></A></LI><LI class="toclevel-1"><A href="#107th_Labday_2011-10-14"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">107th Labday 2011-10-14</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_pARWIII_and_ligation_with_library"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Digest of pARWIII and ligation with library</SPAN></A></LI><LI class="toclevel-1"><A href="#108th_Labday_2011-10-15"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">108th Labday 2011-10-15</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_with_library_for_phage_display"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Transformation with library for phage display</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep.2C_Glycerol_stock_cultures_and_Survival-Test_of_HRV14_3C_clone.28SG24-P10.29"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Miniprep, Glycerol stock cultures and Survival-Test of HRV14 3C clone(SG24-P10)</SPAN></A></LI><LI class="toclevel-1"><A href="#109th_Labday_2011-10-16"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">109th Labday 2011-10-16</SPAN></A></LI><LI class="toclevel-2"><A href="#Overnight_culture_of_library"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Overnight culture of library</SPAN></A></LI><LI class="toclevel-2"><A href="#Survival-Test_of_one_HRV14_3C_clone.28_SG24-P10.29"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">Survival-Test of one HRV14_3C clone( SG24-P10)</SPAN></A></LI><LI class="toclevel-1"><A href="#110th_Labday_2011-10-17"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">110th Labday 2011-10-17</SPAN></A></LI><LI class="toclevel-2"><A href="#Amplification_of_megaprimer_1_.28site-directed_mutagenesis.29"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">Amplification of megaprimer 1 (site-directed mutagenesis)</SPAN></A></LI><LI class="toclevel-2"><A href="#Megaprimer_PCR_II_.28amplification_of_2nd_megaprimer_.282nd_part_of_mdn_cluster.2C_site-directed_mutagenesis.29"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">Megaprimer PCR II (amplification of 2nd megaprimer (2nd part of mdn cluster, site-directed mutagenesis)</SPAN></A></LI><LI class="toclevel-2"><A href="#Sequencing_of_mdnA_library_.28for_phage_display.29"><SPAN class="tocnumber">13.3</SPAN><SPAN class="toctext">Sequencing of mdnA library (for phage display)</SPAN></A></LI><LI class="toclevel-2"><A href="#Over_night_culture_with_created_mdnA_library_for_phage_display"><SPAN class="tocnumber">13.4</SPAN><SPAN class="toctext">Over night culture with created mdnA library for phage display</SPAN></A></LI><LI class="toclevel-1"><A href="#111th_Labday_2011-10-18"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">111th Labday 2011-10-18</SPAN></A></LI><LI class="toclevel-2"><A href="#Production_of_phages_containing_microviridin_library_on_their_surface_in_XL1-blue_cells"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Production of phages containing microviridin library on their surface in XL1-blue cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Sidedirected_mutagenesis_of_TEV_protease"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">Sidedirected mutagenesis of TEV protease</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_purification_and_analytical_GE"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">PCR purification and analytical GE</SPAN></A></LI><LI class="toclevel-2"><A href="#Assembly_PCR_of_TEV_protease"><SPAN class="tocnumber">14.4</SPAN><SPAN class="toctext">Assembly PCR of TEV protease</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_Purification.2C_analytical_GE.2C_Digest_and_Ligation_of_TEV_protease"><SPAN class="tocnumber">14.5</SPAN><SPAN class="toctext">PCR Purification, analytical GE, Digest and Ligation of TEV protease</SPAN></A></LI><LI class="toclevel-1"><A href="#112th_Labday_2011-10-19"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">112th Labday 2011-10-19</SPAN></A></LI><LI class="toclevel-2"><A href="#Amplification_of_megaprimer_I.2C_using_Phusion_.28Finnzymes.29"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">Amplification of megaprimer I, using Phusion (Finnzymes)</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gelelectrophoresis_and_purification_of_megaprimer_I"><SPAN class="tocnumber">15.2</SPAN><SPAN class="toctext">Agarose gelelectrophoresis and purification of megaprimer I</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_phages_containing_microviridin_library"><SPAN class="tocnumber">15.3</SPAN><SPAN class="toctext">Purification of phages containing microviridin library</SPAN></A></LI><LI class="toclevel-2"><A href="#Colony_PCR_of_TEV_in_vivo_selcetion"><SPAN class="tocnumber">15.4</SPAN><SPAN class="toctext">Colony PCR of TEV in vivo selcetion</SPAN></A></LI><LI class="toclevel-2"><A href="#Making_TEV_and_HRV14_3C_Biobricks"><SPAN class="tocnumber">15.5</SPAN><SPAN class="toctext">Making TEV and HRV14_3C Biobricks</SPAN></A></LI><LI class="toclevel-1"><A href="#113th_Labday_2011-10-20"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">113th Labday 2011-10-20</SPAN></A></LI><LI class="toclevel-2"><A href="#ELISA_with_purified_phages_carrying_unmodified_mdnA_on_their_surface"><SPAN class="tocnumber">16.1</SPAN><SPAN class="toctext">ELISA with purified phages carrying unmodified mdnA on their surface</SPAN></A></LI><LI class="toclevel-2"><A href="#SDS-_PAGE_with_purified_phages_carrying_mdnA_on_their_surface"><SPAN class="tocnumber">16.2</SPAN><SPAN class="toctext">SDS- PAGE with purified phages carrying mdnA on their surface</SPAN></A></LI><LI class="toclevel-2"><A href="#Western_Blot"><SPAN class="tocnumber">16.3</SPAN><SPAN class="toctext">Western Blot</SPAN></A></LI><LI class="toclevel-2"><A href="#Colony_PCR.2CMini-prep_and_Survival_screening"><SPAN class="tocnumber">16.4</SPAN><SPAN class="toctext">Colony PCR,Mini-prep and Survival screening</SPAN></A></LI><LI class="toclevel-2"><A href="#Amplification_of_megaprimer_II.2C_using_Phusion_.28Finnzymes.29"><SPAN class="tocnumber">16.5</SPAN><SPAN class="toctext">Amplification of megaprimer II, using Phusion (Finnzymes)</SPAN></A></LI><LI class="toclevel-2"><A href="#Amplification_of_mdn-cluster"><SPAN class="tocnumber">16.6</SPAN><SPAN class="toctext">Amplification of mdn-cluster</SPAN></A></LI><LI class="toclevel-1"><A href="#114th_Labday_2011-10-21"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">114th Labday 2011-10-21</SPAN></A></LI><LI class="toclevel-2"><A href="#Antibody_detection_with_blotted_membranes"><SPAN class="tocnumber">17.1</SPAN><SPAN class="toctext">Antibody detection with blotted membranes</SPAN></A></LI><LI class="toclevel-2"><A href="#Results_of_Survival_screening_at_37.C2.B0C_and_mini_prep_of_TEV_and_HRV14_3C_BioBricks"><SPAN class="tocnumber">17.2</SPAN><SPAN class="toctext">Results of Survival screening at 37°C and mini prep of TEV and HRV14_3C BioBricks</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_mdn-cluster_and_pSB1C3_and_ligation"><SPAN class="tocnumber">17.3</SPAN><SPAN class="toctext">Digest of mdn-cluster and pSB1C3 and ligation</SPAN></A></LI><LI class="toclevel-1"><A href="#115th_Labday_2011-10-22"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">115th Labday 2011-10-22</SPAN></A></LI><LI class="toclevel-2"><A href="#Results_of_survival_screening_at_30.C2.B0C"><SPAN class="tocnumber">18.1</SPAN><SPAN class="toctext">Results of survival screening at 30°C</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_with_pSB1C3.2Bmdn-cluster"><SPAN class="tocnumber">18.2</SPAN><SPAN class="toctext">Transformation with pSB1C3+mdn-cluster</SPAN></A></LI><LI class="toclevel-1"><A href="#116th_Labday_2011-10-23"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">116th Labday 2011-10-23</SPAN></A></LI><LI class="toclevel-2"><A href="#making_competent_cells_and_tranformation_with_Libary_of_SG26_carrying_cells"><SPAN class="tocnumber">19.1</SPAN><SPAN class="toctext">making competent cells and tranformation with Libary of SG26 carrying cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Production_of_phages_containing_microviridin_library_on_their_surface_in_ER2738_cells"><SPAN class="tocnumber">19.2</SPAN><SPAN class="toctext">Production of phages containing microviridin library on their surface in ER2738 cells</SPAN></A></LI><LI class="toclevel-1"><A href="#117th_Labday_2011-10-24"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">117th Labday 2011-10-24</SPAN></A></LI><LI class="toclevel-2"><A href="#Testing_mdnA_libary_with_SG26_carrying_cells_for_TEV_in_vivo_selection"><SPAN class="tocnumber">20.1</SPAN><SPAN class="toctext">Testing mdnA libary with SG26 carrying cells for TEV in vivo selection</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_phages_containing_microviridin_library_2"><SPAN class="tocnumber">20.2</SPAN><SPAN class="toctext">Purification of phages containing microviridin library</SPAN></A></LI><LI class="toclevel-1"><A href="#118th_Labday_2011-10-25"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">118th Labday 2011-10-25</SPAN></A></LI><LI class="toclevel-2"><A href="#Control_of_sequencing_results_of_sequenced_clones_from_created_library_for_phage_display"><SPAN class="tocnumber">21.1</SPAN><SPAN class="toctext">Control of sequencing results of sequenced clones from created library for phage display</SPAN></A></LI><LI class="toclevel-2"><A href="#Colony_PCR_of_pSB1C3.2Bmdn-cluster"><SPAN class="tocnumber">21.2</SPAN><SPAN class="toctext">Colony PCR of pSB1C3+mdn-cluster</SPAN></A></LI><LI class="toclevel-1"><A href="#119th_Labday_2011-10-26"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">119th Labday 2011-10-26</SPAN></A></LI><LI class="toclevel-2"><A href="#Phage_Display_with_wildtype_mdnA_and_different_proteases"><SPAN class="tocnumber">22.1</SPAN><SPAN class="toctext">Phage Display with wildtype mdnA and different proteases</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_pARWIII_and_ligation_with_library_2"><SPAN class="tocnumber">22.2</SPAN><SPAN class="toctext">Digest of pARWIII and ligation with library</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_triple_ligated_mdnA_library.2C_gene_III_and_pARWIII_2"><SPAN class="tocnumber">22.3</SPAN><SPAN class="toctext">Transformation of triple ligated mdnA library, gene III and pARWIII</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_pSB1C3.2Bmdn-cluster_A_clone_6_and_sending_for_sequencing"><SPAN class="tocnumber">22.4</SPAN><SPAN class="toctext">Miniprep of pSB1C3+mdn-cluster A clone 6 and sending for sequencing</SPAN></A></LI><LI class="toclevel-2"><A href="#SDS-_PAGE_with_lysated_cells_which_produced_the_mdnA-myc-geneIII_protein"><SPAN class="tocnumber">22.5</SPAN><SPAN class="toctext">SDS- PAGE with lysated cells which produced the mdnA-myc-geneIII protein</SPAN></A></LI><LI class="toclevel-2"><A href="#Western_Blot_2"><SPAN class="tocnumber">22.6</SPAN><SPAN class="toctext">Western Blot</SPAN></A></LI><LI class="toclevel-1"><A href="#120th_Labday_2011-10-27"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">120th Labday 2011-10-27</SPAN></A></LI><LI class="toclevel-2"><A href="#Antibody_detection_with_blotted_membranes_2"><SPAN class="tocnumber">23.1</SPAN><SPAN class="toctext">Antibody detection with blotted membranes</SPAN></A></LI><LI class="toclevel-2"><A href="#control_of_performed_phage_display"><SPAN class="tocnumber">23.2</SPAN><SPAN class="toctext">control of performed phage display</SPAN></A></LI></TD></TR></TBODY></TABLE><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="98th_Labday_2011-09-24">98th Labday 2011-09-24</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Colony_PCR_of_E.coli_XL1_blue_transformed_UP_SG2_.281.5.25.29"> Colony PCR of E.coli XL1 blue transformed UP_SG2 (1.5%)</SPAN></H3><P><B>Investigators:</B> Stefan, Paul, Sebastian</P><P><B>Aim:</B> checking for positive clones for further tasks</P><P><B>Methode:</B></P><P>Primer: 42,67</P><UL><LI> picked clones where incubated over night at 4°C in 1 ml LB media
</LI></UL><UL><LI> 5 µl Buffer S with 25 mM MgCl2 (purchased by Genaxxon)
</LI></UL><UL><LI> 2,5 µl each Primer (10 mM, see list below)
</LI></UL><UL><LI> 1 µl dNTP (each 10 mM)
</LI></UL><UL><LI> 0.5 µl Taq Polymerase (purchased by Genaxxon)
</LI></UL><UL><LI> 2 µl 25 mM MgCl2 (purchased by Genaxxon)
</LI></UL><UL><LI> 36.5 µl H2O
</LI></UL><UL><LI><B>Total volume:</B> 50 µl</LI></UL><P><B>PCR Program:</B></P><P>Initial denat = 6min 94°C
</P><P>30x
</P><P>denat: 30sec 94°C
</P><P>anneal: 30sec 55°C
</P><P>extend: 30sec 72°C
</P><P>final extend: 10min 72°C
</P><P><B>Results</B>:
</P><P>no positive clone probably due to rubbishy digestion
</P><P><B>Further tasks:</B></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Subcloning_of_14_3C_and_AraC_into_SG2_.281.5.25.29"> Subcloning of 14 3C and AraC into SG2 (1.5%)</SPAN></H3><P><B>Investigators:</B> Stefan, Paul, Sascha</P><P><B>Aim:</B> digestion of 14 3C protease</P><P><B>Methode:</B></P><UL><LI>14 µL H20
</LI></UL><UL><LI>2 µL buffer
</LI></UL><UL><LI>1 µL EcoRI
</LI></UL><UL><LI>1 µL PstI
</LI></UL><UL><LI>2 µL DNA
</LI></UL><UL><LI><B>Total volume:</B> 20 µl</LI></UL><P>digest over 2h or overnight depends how long Jessica is here
</P><P><B>Results</B>:
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Production_of_phages_containing_pPDV089_in_XL1-blue_cells">Production of phages containing pPDV089 in XL1-blue cells</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage and performing phage display</P><P><B>Method/Materials:</B></P><UL><LI> amplification of XL1-blue cells
</LI></UL><UL><LI> 50 ml over night culture will be inoculated with the cells, so that OD600 = 0.1
</LI></UL><UL><LI> add tetracyclin to the medium
</LI></UL><UL><LI> cells should be incubated 37 °C till OD600 = 0.3-0.5 (here: 0.356) is reached
</LI></UL><UL><LI> add 30 µl phages carrying mdnA-myc-gene III on their surface
</LI></UL><UL><LI> incubate for 10 min at 37 °C without shaking
</LI></UL><UL><LI> incubate for 50 min shaking at 37°C
</LI></UL><UL><LI> phage purification:
</LI></UL><UL><LI> centrifuge cell culture at 5000 x g/ 15 min
</LI></UL><UL><LI> fill supernatant in a new 50 ml falcon and centrifuge again (5000 g/15 min)
</LI></UL><UL><LI> 40 ml of the supernatant with 8 ml PEG-NaCl (20% (w/v) PEG-8000, 2,5 M NaCl)
</LI></UL><UL><LI> incubate over night at 4°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> go on with phage purification</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="99th_Labday_2011-09-25">99th Labday 2011-09-25</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_of_maxiprimer_I">Agarose gel electrophoresis of maxiprimer I</SPAN></H3><P><B>Time:</B> 2011-09-25, 20:00-23:00</P><P><B>Investigators:</B> Nicole </P><P><B>Idea:</B>Examination if PCR for Amplification of maxiprimer I works</P><P><B>Materials</B></P><UL><LI> Maxiprimer a and b, PCR done by Nadja, 2011-09-25
</LI></UL><UL><LI> Broad range agarose
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel red
</LI></UL><UL><LI> Gene Ruler 100 bp ladder (Fermentas)
</LI></UL><UL><LI> Gene Ruler DNA ladder mix (Fermentas)
</LI></UL><UL><LI> 6x loading dye (Promega)
</LI></UL><P><B>Methods:</B></P><UL><LI> 2 % agarose gel (1 g agarose and 50 ml 1x TAE buffer)
</LI></UL><UL><LI> addition of 2 µl gel red
</LI></UL><UL><LI> Load 50 µl sample and 8 µl loading dye
</LI></UL><UL><LI> 3.5 µl of each DNA ladder
</LI></UL><P><B>Results:</B></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Label
</TH><TH> Sample
</TH><TH> Expected size in bases
</TH></TR><TR><TD> 1
</TD><TD> 100bp DNA ladder
</TD></TR><TR><TD> 2
</TD><TD> A
</TD><TD> 129
</TD></TR><TR><TD> 3
</TD><TD> A
</TD><TD> 129
</TD></TR><TR><TD> 4
</TD><TD> B
</TD><TD> 129
</TD></TR><TR><TD> 5
</TD><TD> B
</TD><TD> 129
</TD></TR><TR><TD> 6
</TD><TD> DNA ladder mix
</TD></TR></TBODY></TABLE><P><A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=UP" class="new" title="UP">Media: UP_</A></P><P><B>Further tasks:</B></P><UL><LI> DNA extraction
</LI></UL><UL><LI> Maxiprimer PCR I using the extracted DNA as forward primer
</LI></UL><UL><LI> Gel electrophoresis and DNA extraction, expected size 431 bases
</LI></UL><UL><LI> Maxiprimer PCR II using the extracted DNA as forward primer
</LI></UL><UL><LI> Agarose gel electrophoresis and DNA extraction, expected size 6619 bases
</LI></UL><UL><LI> Restriction enzyme digestion using EcoRI and SpeI
</LI></UL><UL><LI> Ligation with pSB1C3
</LI></UL><UL><LI> Transformation
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Purification_of_phages_containing_pPDV089">Purification of phages containing pPDV089</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage, repeated phage display</P><P><B>Method/Materials:</B></P><UL><LI> after over night incubation:
</LI></UL><UL><LI> centrifuge precipitaed phages: 5000 x g/ 45 min
</LI></UL><UL><LI> discard supernatant
</LI></UL><UL><LI> centrifuge again, 5000 x g/ 5 min
</LI></UL><UL><LI> remove supernatant carefully
</LI></UL><UL><LI> take pellet in 1 ml TBS
</LI></UL><UL><LI> move in 1.5 ml Eppi
</LI></UL><UL><LI> centrifuge again: 17000 x g/10 min
</LI></UL><UL><LI> supernatant in a new Eppi with 200 µl PEG-NaCl (mix!)
</LI></UL><UL><LI> incubate on ice for 60 min
</LI></UL><UL><LI> centrifuge precipitated phages: 17.000 x g/ 10 min (4°C)
</LI></UL><UL><LI> resuspend pellet in 300 µl TBS
</LI></UL><UL><LI> centrifuge:17 000 x g/ 10 min (4°C)
</LI></UL><UL><LI> supernatant in a fresh eppi= purified phages
</LI></UL><P><B>Further tasks:</B></P><UL><LI> ELISA and phage display</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Maxiprimer_PCR_I_.E2.80.93_Amplification_of_the_end_of_the_mdn_cluster_.28Repetition.2C_usage_of_two_different_polymerases.29">Maxiprimer PCR I – Amplification of the end of the mdn cluster (Repetition, usage of two different polymerases)</SPAN></H3><P><B>Time:</B> 2011-09-25, 20:00-23:00</P><P><B>Investigators:</B> Nicole </P><P><B>Idea:</B>Checking out two different polymerase to obtain amplification</P><P><B>Materials</B></P><UL><LI> purified Maxiprimer a and b, PCR done by Nadja, 2011-09-25
</LI></UL><UL><LI> pARW089 clone 2, 85.7 ng/ µl
</LI></UL><UL><LI> PfuUltra
</LI></UL><UL><LI> PfuUltra HF Buffer
</LI></UL><UL><LI> Primer reverse: r_mdnABCDE_iGEM (10 µM)
</LI></UL><UL><LI> LongPCR Enzyme Mix (Fermentas)
</LI></UL><UL><LI> Long PCR Enzyme Mix buffer (Fermentas)
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><P><B>Methods:</B></P><P><I>1. PfuUltra Polymerase</I></P><P>PCR mixture, maxiprimer A
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Volume in µl
</TH></TR><TR><TD> PfuUltra Buffer HF
</TD><TD> 5
</TD></TR><TR><TD> dNTPs
</TD><TD> 1
</TD></TR><TR><TD> Template DNA, pARW089
</TD><TD> 1.2
</TD></TR><TR><TD> r_mdnABCDE_iGEM
</TD><TD> 1
</TD></TR><TR><TD> Maxiprimer A
</TD><TD> 5.9
</TD></TR><TR><TD> PfuUltra Polymerase
</TD><TD> 1
</TD></TR><TR><TD> H<SMALL>2</SMALL>O
</TD><TD> 34.9
</TD></TR></TBODY></TABLE><P>Labeled: P1
</P><P>PCR mixture, maxiprimer B
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Volume in µl
</TH></TR><TR><TD> PfuUltra Buffer HF
</TD><TD> 5
</TD></TR><TR><TD> dNTPs
</TD><TD> 1
</TD></TR><TR><TD> Template DNA, pARW089
</TD><TD> 1.2
</TD></TR><TR><TD> r_mdnABCDE_iGEM
</TD><TD> 1
</TD></TR><TR><TD> Maxiprimer B
</TD><TD> 8.2
</TD></TR><TR><TD> PfuUltra Polymerase
</TD><TD> 1
</TD></TR><TR><TD> H<SMALL>2</SMALL>O
</TD><TD> 32.6
</TD></TR></TBODY></TABLE><P>Labeled: P2
</P><P>Thermal cycling program</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Step
</TH><TH> Temperature in °C
</TH><TH> Time in seconds
</TH></TR><TR><TD> Hot start
</TD><TD> 95
</TD><TD> Hold
</TD></TR><TR><TD> Initial Denaturation
</TD><TD> 95
</TD><TD> 120
</TD></TR><TR><TD> Denaturation
</TD><TD> 95
</TD><TD> 30
</TD></TR><TR><TD> Annealing
</TD><TD> 54
</TD><TD> 30
</TD></TR><TR><TD> Extension
</TD><TD> 72
</TD><TD> 60
</TD></TR><TR><TD> Cycle number
</TD><TD> 30
</TD></TR><TR><TD> Final extension
</TD><TD> 72
</TD><TD> 600
</TD></TR></TBODY></TABLE><P><I>2. Long PCR enzyme mix</I></P><P>PCR mixture, maxiprimer A
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Volume in µl
</TH></TR><TR><TD> Long PCR enzyme mix buffer
</TD><TD> 5
</TD></TR><TR><TD> dNTPs
</TD><TD> 1
</TD></TR><TR><TD> Template DNA, pARW089
</TD><TD> 1.2
</TD></TR><TR><TD> r_mdnABCDE_iGEM
</TD><TD> 1
</TD></TR><TR><TD> Maxiprimer A
</TD><TD> 24.1
</TD></TR><TR><TD> Long PCR enzyme mix
</TD><TD> 0.3
</TD></TR><TR><TD> H<SMALL>2</SMALL>O
</TD><TD> 17.4
</TD></TR></TBODY></TABLE><P>Labeled: L1
</P><P>PCR mixture, maxiprimer B
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Volume in µl
</TH></TR><TR><TD> Long PCR enzyme mix buffer
</TD><TD> 5
</TD></TR><TR><TD> dNTPs
</TD><TD> 1
</TD></TR><TR><TD> Template DNA, pARW089
</TD><TD> 1.2
</TD></TR><TR><TD> r_mdnABCDE_iGEM
</TD><TD> 1
</TD></TR><TR><TD> Maxiprimer B
</TD><TD> 21.8
</TD></TR><TR><TD> Long PCR enzyme mix
</TD><TD> 0.3
</TD></TR><TR><TD> H<SMALL>2</SMALL>O
</TD><TD> 19.7
</TD></TR></TBODY></TABLE><P>Labeled: L2
</P><P>Thermal cycling program</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Step
</TH><TH> Temperature in °C
</TH><TH> Time in seconds
</TH></TR><TR><TD> Hot start
</TD><TD> 94
</TD><TD> Hold
</TD></TR><TR><TD> Initial Denaturation
</TD><TD> 94
</TD><TD> 300
</TD></TR><TR><TD> Denaturation
</TD><TD> 95
</TD><TD> 20
</TD></TR><TR><TD> Annealing
</TD><TD> 54
</TD><TD> 30
</TD></TR><TR><TD> Extension
</TD><TD> 68
</TD><TD> 22
</TD></TR><TR><TD> Cycle number
</TD><TD> 10
</TD></TR><TR><TD> Denaturation
</TD><TD> 94
</TD><TD> 20
</TD></TR><TR><TD> Annealing
</TD><TD> 68
</TD><TD> 30
</TD></TR><TR><TD> Extension
</TD><TD> 68
</TD><TD> 24
</TD></TR><TR><TD> Cycle number
</TD><TD> 15
</TD></TR><TR><TD> Final extension
</TD><TD> 68
</TD><TD> 600
</TD></TR></TBODY></TABLE><P><B>Output:</B></P><UL><LI> 2 samples (L1 and L2) in right Thermocycler
</LI></UL><UL><LI> 2 samples (P1 and P2) in left Thermocycler
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Gel electrophoresis and DNA extraction, expected size 431 bases
</LI></UL><UL><LI> Maxiprimer PCR II using the extracted DNA as reverse primer
</LI></UL><UL><LI> Gel electrophoresis and DNA extraction, expected size 6619 bases
</LI></UL><UL><LI> Restriction enzyme digestion using EcoRI and SpeI
</LI></UL><UL><LI> Ligation with pSB1C3
</LI></UL><UL><LI> Transformation
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="100th_Labday_2011-09-26">100th Labday 2011-09-26</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_Phage_Display_with_another_trypsin_protease">Repeated Phage Display with another trypsin protease</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine </P><P><B>Time:</B> 2011-09-16, 9:00-18:00</P><P><B>Aim:</B>check if unmodified mdnA on the surface of the phages binds to an immobilized protease. interactions of microviridin L with this protease were already detected</P><P><B>Material/Method:</B></P><UL><LI> in ELISA plate:
</LI></UL><UL><LI> coat wells with protease trypsin- EDTA in water for 2 h
</LI></UL><UL><LI> blocking in 3% milk powder for 2 h
</LI></UL><UL><LI> wash 6x with TBS-T (0,005%)
</LI></UL><UL><LI> panning with phages 1:1 helper phages and phages of interest (presenting mdnA-geneIII fusion protein on their surface, 1 h)
</LI></UL><UL><LI> wash 6x with TBS-T
</LI></UL><UL><LI> elute bound phages with 0,2 M Glycin-HCl, 10 min
</LI></UL><UL><LI> neutralize with 1 M Tris (pH 7-8)
</LI></UL><UL><LI> mix eluted phages with preparatory culture of XL1-blue cells
</LI></UL><UL><LI> store over night at 4°C
</LI></UL><P><B>Further tasks:</B></P><P>check binding of phages by plating them
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="ELISA_with_purified_phages">ELISA with purified phages</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Time:</B> 2011-09-16, 12:00-21:00</P><P><B>Aim:</B> control if mdnA-myc-geneIII will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI> ELISA plate coated with 5 µg/ml anti-c-myc antibody in phosohate buffer (pH 7,4), volume 100 µl/ well
</LI></UL><UL><LI> incubate for 4 h at room temperature
</LI></UL><UL><LI> blocking in 2 % milk powder in TBS (300 µl/ well)
</LI></UL><UL><LI> incubate for 2 h at room temperature
</LI></UL><UL><LI> add 100 µl phages, produced in XL1-blu cells diluted 1:1 in TBS-T (0,005%)
</LI></UL><UL><LI> incubate shaking for 60 min at room temperature
</LI></UL><UL><LI> wash 5 x with TBS-T (0,05%)
</LI></UL><UL><LI> add anti-gene-8-antibody (HRP coupled)
</LI></UL><UL><LI> incubate shaking for 60 min
</LI></UL><UL><LI> wash 5 x with TBS-T
</LI></UL><UL><LI> put substrate on the samples
</LI></UL><UL><LI> measure in plate photometer
</LI></UL><UL><LI> reference: helper phages
</LI></UL><P><B>Resluts:</B> positive but weak weak signal at wells with mdnA phages because of the low concentration of phages
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Control_of_Phage_Display">Control of Phage Display </SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine </P><P><B>Time:</B> 2011-09-2</P><P><B>Aim:</B>check if unmodified mdnA on the surface of the phages binds to an immobilized protease. interactions of microviridin L with this protease were already detected</P><P><B>Material/Method:</B></P><UL><LI> over night culture of XL1-blue cells was infected with eluted phages
</LI></UL><UL><LI> 1000 cells were infected with 10.000 phages and plated on agar with kanamycin and on agar with ampicillin
</LI></UL><UL><LI> control: 1:1 mix of helperphages and produced phages of interest carrying mdnA-geneIII plated also on agar with kanamycin and on agar with ampicillin</LI></UL><P><B>Further tasks:</B></P><UL><LI> check cell growth
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="101th_Labday_2011-09-27">101th Labday 2011-09-27</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="SDS-_PAGE_for_coomassie_staining_with_purified_phages_carrying_mdnA_on_their_surface">SDS- PAGE for coomassie staining with purified phages carrying mdnA on their surface</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> preparing geneIII-myc-mdnA-protein for mass spectrometry</P><P><B>Method/Materials:</B></P><UL><LI> seperating gel:
</LI></UL><UL><LI> acryl amide: 2,5 ml
</LI></UL><UL><LI> seperating gel buffer 4x: 1,5 ml
</LI></UL><UL><LI> water: 1,94
</LI></UL><UL><LI> APS: 60 µl
</LI></UL><UL><LI> stacking gel:
</LI></UL><UL><LI> acryl amide: 0,28 ml
</LI></UL><UL><LI> stacking gel buffer: 0,5 ml
</LI></UL><UL><LI> water: 1,22 ml
</LI></UL><UL><LI> APS: 10 µl
</LI></UL><UL><LI> loading dye: RotiLoad1
</LI></UL><UL><LI> incubate samples at 95°C for 20 min
</LI></UL><UL><LI> run at 50 V for half an hour, then switch to 20 mA
</LI></UL><P><B>Further tasks:</B></P><UL><LI> coomassie staining </LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Coomassie_staining">Coomassie staining</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> control if geneIII-myc-mdnA could be enriched for mass spectrometry</P><P><B>Method/Materials:</B></P><UL><LI> 20 ml water
</LI></UL><UL><LI> 20 ml methanol
</LI></UL><UL><LI> 60 ml coomassie stock solution
</LI></UL><UL><LI> gel storage over night in coomassie solution shaking </LI></UL><P><B>Further tasks:</B></P><UL><LI> destaining of the gel </LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="102th_Labday_2011-10-06">102th Labday 2011-10-06</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Amplification_of_megaprimers_for_site-directed_mutagenesis">Amplification of megaprimers for site-directed mutagenesis</SPAN></H3><P><B>Time:</B> 2011-10-06, 9:00-12:00</P><P><B>Investigators:</B> Nicole </P><P><B>Idea:</B>Site-directed mutagenesis of XbaI restriction sites within the mdn cluster (use as biobrick)</P><P><B>Materials</B></P><UL><LI> Primer:
</LI></UL><UL><LI> pf_muta_XbaI, 10 µM, 54°C (TATATTGAGGCATCAAGATCCAATTG)
</LI></UL><UL><LI> pr_muta_XbaI, 10 µM, 55°C (GCTTTCATCTTCTTGATAAAAATGTTATCA)
</LI></UL><UL><LI> pr_ohne_muta, 10 µM, 55°C (GCTTTCATCTTCTAGATAAAAATGTTATCA)
</LI></UL><UL><LI> pARW089 clone 2, 8.57 ng/ µl
</LI></UL><UL><LI> Genaxxopn Taq Polymerase S
</LI></UL><UL><LI> 10x Genaxxon Taq Polymerase Buffer (15 mM MgCl2)
</LI></UL><UL><LI> 25 mM MgCl2
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><P><B>Methods:</B></P><P><I>PCR mixture for A</I></P><P>A = mdn cluster without XbaI recognition sites at both sites
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Volume in µl
</TH></TR><TR><TD> dNTPs (10 mM)
</TD><TD> 1
</TD></TR><TR><TD> 10x Genaxxon Taq Polymerase Buffer
</TD><TD> 5
</TD></TR><TR><TD> 25 mM MgCl2
</TD><TD> 2
</TD></TR><TR><TD> pf_muta_XbaI (10 µM)
</TD><TD> 1.5
</TD></TR><TR><TD> pr_muta_XbaI
</TD><TD> 1.5
</TD></TR><TR><TD> Template DNA, pARW089, 8.57 ng/ µl
</TD><TD> 1
</TD></TR><TR><TD>Genaxxon Taq Polymerase S
</TD><TD> 0.5
</TD></TR><TR><TD> H<SMALL>2</SMALL>O
</TD><TD> 37.5
</TD></TR></TBODY></TABLE><P>Labeled: A1 and A2
</P><P><I>PCR mixture for B</I></P><P>A = mdn cluster without XbaI recognition sites at one sites</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Volume in µl
</TH></TR><TR><TD> dNTPs (10 mM)
</TD><TD> 1
</TD></TR><TR><TD> 10x Genaxxon Taq Polymerase Buffer
</TD><TD> 5
</TD></TR><TR><TD> 25 mM MgCl2
</TD><TD> 2
</TD></TR><TR><TD> pf_muta_XbaI (10 µM)
</TD><TD> 1.5
</TD></TR><TR><TD> pr_ohne_muta
</TD><TD> 1.5
</TD></TR><TR><TD> Template DNA, pARW089, 8.57 ng/ µl
</TD><TD> 1
</TD></TR><TR><TD> Genaxxon Taq Polymerase S
</TD><TD> 0.5
</TD></TR><TR><TD> H<SMALL>2</SMALL>O
</TD><TD> 37.5
</TD></TR></TBODY></TABLE><P>Labeled: B1 and B2
</P><P><I>Thermal cycling program</I></P><P>Program named MAXI1A
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Step
</TH><TH> Temperature in °C
</TH><TH> Time in seconds
</TH></TR><TR><TD> Hot start
</TD><TD> 94
</TD><TD> Hold
</TD></TR><TR><TD> Initial Denaturation
</TD><TD> 94
</TD><TD> 180
</TD></TR><TR><TD> Denaturation
</TD><TD> 94
</TD><TD> 45
</TD></TR><TR><TD> Annealing
</TD><TD> 51
</TD><TD> 45
</TD></TR><TR><TD> Extension
</TD><TD> 72
</TD><TD> 45
</TD></TR><TR><TD> Cycle number
</TD><TD> 25
</TD></TR><TR><TD> Final extension
</TD><TD> 72
</TD><TD> 600
</TD></TR></TBODY></TABLE><P><B>Output:</B></P><UL><LI> 2 samples (A1, A2, B1, B2) in left thermocycler
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Gel electrophoresis and DNA extraction, expected size 127 bases
</LI></UL><UL><LI> Megaprimer PCR using the extracted DNA as primer
</LI></UL><UL><LI> DpnI digestion with the half of each PCR reaction
</LI></UL><UL><LI> Transformation using XL1-Blue cells
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_and_DNA_extraction_of_megaprimers">Agarose gel electrophoresis and DNA extraction of megaprimers</SPAN></H3><P><B>Time:</B> 2011-10-06, 12:00-14:00</P><P><B>Investigators:</B> Nicole, Jessica </P><P><B>Idea:</B>Purification of megaprimers (used for site-directed mutagenesis)</P><P><B>Materials</B></P><UL><LI> Agarose
</LI></UL><UL><LI> Gel red
</LI></UL><UL><LI> 1x TAE Buffer
</LI></UL><UL><LI> PCR products (A1, A2, B1, B2)
</LI></UL><UL><LI> 6x Loadig Dye (Promega)
</LI></UL><UL><LI> 100bp DNA ladder (Fermentas)
</LI></UL><UL><LI> DNA ladder mix (Fermentas)
</LI></UL><UL><LI> NucleoSpin Extract Kit II (Machery Nagel)</LI></UL><P><B>Methods:</B></P><P><I>Preparation of 2% Agarose gel</I></P><UL><LI> 1 g Agarose in 50 ml TAE
</LI></UL><UL><LI> 2 µl Gel red</LI></UL><P><I>Loading of samples</I></P><UL><LI> 8 µl 6x Loading Dye and 50 µl of each PCR product
</LI></UL><UL><LI> 5 µl 100 bp DNA ladder
</LI></UL><UL><LI> 8 µl DNA ladder mix</LI></UL><P><I>Running agarose gel</I></P><P><I>DNA extraction and purification</I></P><UL><LI> using NucleoSpin Extract II (Machery Nagel)
</LI></UL><UL><LI> following maufacturer's protocol
</LI></UL><UL><LI> A1 and A2 resp. B1 and B2 using one column
</LI></UL><UL><LI> Elution buffer: H2O, volume: 35 µl, incubation 5 min at 70°C</LI></UL><P><B>Results:</B></P><P><I>Expected sizes and positions of samples</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Position
</TH><TH> Label
</TH><TH> Expected size in bases
</TH></TR><TR><TD> 1
</TD><TD> 100bp DNA ladder
</TD></TR><TR><TD> 2
</TD><TD> A1
</TD><TD> 127
</TD></TR><TR><TD> 3
</TD><TD> A2
</TD><TD> 127
</TD></TR><TR><TD> 4
</TD></TR><TR><TD> 5
</TD><TD> B1
</TD><TD> 127
</TD></TR><TR><TD> 6
</TD><TD> B2
</TD><TD> 127
</TD></TR><TR><TD> 7
</TD><TD> DNA ladder mix
</TD></TR></TBODY></TABLE><P><I> Gel picture</I></P><P><A href="/File:UP" title="File:UP">File:UP</A></P><P><I>DNA concentrations</I></P><UL><LI> A: c = 18.9 ng/ µl
</LI></UL><UL><LI> B: c = 23.0 ng/ µl</LI></UL><P><B>Output:</B></P><UL><LI> Megaprimer A (both mutations), stored PCR box
</LI></UL><UL><LI> Megaprimer B (one mutation, stored PCR box</LI></UL><P><B>Further tasks:</B></P><UL><LI> PCR using these megaprimers
</LI></UL><UL><LI> DpnI digestion
</LI></UL><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Megaprimer_PCR_for_site_directed_mutagenesis">Megaprimer PCR for site directed mutagenesis</SPAN></H3><P><B>Time:</B> 2011-10-06, 14:00-17:00</P><P><B>Investigators:</B> Nicole, Jessica </P><P><B>Idea:</B>Site-directed mutagenesis of XbaI restriction sites within the mdn cluster (use as biobrick)<B>Materials</B></P><UL><LI> Megaprimer (A and B), 2011-10-06, Jessica&amp;Nicole
</LI></UL><UL><LI> pARW089 clone 2, 85.7 ng/ µl
</LI></UL><UL><LI> PfuUltra Polymerase (Agilent)
</LI></UL><UL><LI> 10x PfuUltra Polymerase Buffer
</LI></UL><UL><LI> dNTPs (10 mM)</LI></UL><P><B>Methods:</B><I>PCR mixture</I></P><UL><LI> based on <A href="/File:UP_PfuUltra_HF_DNA_Polymerase.pdf" title="File:UP PfuUltra HF DNA Polymerase.pdf">File:UP PfuUltra HF DNA Polymerase.pdf</A></LI></UL><UL><LI> A = mdn cluster without XbaI recognition sites at both sites
</LI></UL><UL><LI> B = mdn cluster without XbaI recognition sites at one sites
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TD> dNTPs (10 mM)
</TD><TD> 1
</TD></TR><TR><TD> 10x PfuUltra Polymerase Buffer
</TD><TD> 5
</TD></TR><TR><TD> Megaprimer
</TD><TD> 0.5 (approx. 3.91 ng needed)
</TD></TR><TR><TD> 25 mM MgCl2
</TD><TD> 2
</TD></TR><TR><TD> Template DNA, pARW089, 85.7 ng/ µl
</TD><TD> 1.2
</TD></TR><TR><TD> PfuUltra Polymerase
</TD><TD> 2
</TD></TR><TR><TD> H<SMALL>2</SMALL>O
</TD><TD> 40.3
</TD></TR></TBODY></TABLE><P><I>Thermal cycling program</I></P><P>Program named MAXI5</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Step
</TH><TH> Temperature in °C
</TH><TH> Time in seconds
</TH></TR><TR><TD> Hot start
</TD><TD> 94
</TD><TD> Hold
</TD></TR><TR><TD> Initial Denaturation
</TD><TD> 94
</TD><TD> 300
</TD></TR><TR><TD> Denaturation
</TD><TD> 94
</TD><TD> 30
</TD></TR><TR><TD> Annealing
</TD><TD> 60
</TD><TD> 60
</TD></TR><TR><TD> Extension
</TD><TD> 68
</TD><TD> 20 min 50 sec
</TD></TR><TR><TD> Cycle number
</TD><TD> 7
</TD></TR><TR><TD> Denaturation
</TD><TD> 94
</TD><TD> 30
</TD></TR><TR><TD> Annealing
</TD><TD> 60
</TD><TD> 60
</TD></TR><TR><TD> Extension
</TD><TD> 68
</TD><TD> 20 min 50 sec + 10 sec per cycle
</TD></TR><TR><TD> Cycle number
</TD><TD> 8
</TD></TR></TBODY></TABLE><P><B>Output:</B></P><UL><LI> 2 samples in left thermocycler, labeled 2A and 2B</LI></UL><P><B>Further tasks:</B></P><UL><LI> DpnI digestion with the half of each PCR reaction
</LI></UL><UL><LI> Transformation using XL1-Blue cells
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="103th_Labday_2011-10-07">103th Labday 2011-10-07</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="DpnI_digestion_and_Transformation_of_mutated_pARW089">DpnI digestion and Transformation of mutated pARW089</SPAN></H3><P><B>Time:</B> 2011-10-07</P><P><B>Investigators:</B> Nicole für Sebastian ;), done by Jessica</P><P><B>Aim:</B>Site-directed mutagenesis of the mdn cluster (use as biobrick)</P><P><B>Materials:</B></P><UL><LI> DpnI (10 U/ µl)
</LI></UL><UL><LI> PCR products, 2A and 2B, stored in freezer (or thermal cylcer)
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> 4 agar plates with ampicillin
</LI></UL><P><B>Methods:</B></P><P><I>DpnI digestion</I></P><UL><LI> add 0.5 µl DpnI to 25 µl of the PCR product
</LI></UL><DL><DD><UL><LI>0.5 µl DpnI
</LI></UL></DD></DL><DL><DD><UL><LI>25 µl PCR product
</LI></UL></DD></DL><DL><DD><UL><LI>2.8 µl NEB 4 buffer
</LI></UL></DD></DL><UL><LI> mix gently and thoroughly by pipetting up and down or by vortexing
</LI></UL><UL><LI> incubate 1 hour at 37°C
</LI></UL><UL><LI> the other 25 µl of PCR product: undigested
</LI></UL><P><I>Transformation</I></P><UL><LI> add <B>2 µl PCR product </B>(one sample = DpnI digested, second sample = undigested) to competent cells
</LI></UL><UL><LI> incubate 30 min one ice
</LI></UL><UL><LI> heat pulse: 45 sec at 42°C
</LI></UL><UL><LI> 2 min on ice
</LI></UL><UL><LI> add 750 µl LB medium to each sample
</LI></UL><UL><LI> incubate 1 hour at 37°C and 850rpm
</LI></UL><UL><LI> cenrifugation and resuspend in 500 µl LB medium
</LI></UL><UL><LI> plate 250 µl on each of two plates with ampicillin or kanamycin antibiotic
</LI></UL><UL><LI> incubate the transformation plates at 37°C for <B>more than 16 hours</B></LI></UL><P><B>Output:</B></P><P>plates in incubator:
</P><UL><LI> Sample a (cluster without any XbaI recognition site), DpnI digested
</LI></UL><UL><LI> Sample a (cluster without any XbaI recognition site), undigested
</LI></UL><UL><LI> Sample b (cluster with one XbaI recognition site), DpnI digested
</LI></UL><UL><LI> Sample b (cluster with one XbaI recognition site), undigested
</LI></UL><P><B>Further tasks:</B></P><UL><LI> preparation of overnight cultures of several clones, especially from DpnI digested samples
</LI></UL><UL><LI> Miniprep
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Retransformation_of_XL1blue-cells_with_pPDV089">Retransformation of XL1blue-cells with pPDV089</SPAN></H3><P><B>Investigators:</B> Sandrina </P><P><B>Aim:</B> produce cells for phage display </P><P><B>Method/Materials:</B></P><UL><LI> addition of 0,3 µl ligated vector pPDV089 from clone 2S14 (after plasmid preperation) to competent XL1-blue cells</LI></UL><UL><LI> incubation 25 min on ice,</LI></UL><UL><LI> heat shock 45 sec at 42°C,</LI></UL><UL><LI> incubation 2 min on ice,</LI></UL><UL><LI> addition of 750 µl LB medium,</LI></UL><UL><LI> incubation 60 min at 37 °C and 750 rpm</LI></UL><UL><LI> plating on agar plates containing 100 µg/ml ampicillin</LI></UL><UL><LI> storage over night at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> over night culture, amplification of cells for phage display, test digestion</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="104th_Labday_2011-10-11">104th Labday 2011-10-11</SPAN></H2><H3 style="background-color: rgb(254, 122, 122);"><SPAN class="mw-headline" id="Oligo-Fillin_for_Phage-display-mdnA-library_.28carrying_AgeI_restriction_site.29">Oligo-Fillin for Phage-display-mdnA-library (carrying AgeI restriction site)</SPAN></H3><P><B>Investigators:</B> Nicole
</P><P><B>Time:</B> 2011-10-11, 9:00-14:00</P><P><B>Aim: </B> Production of a library carrying mutated mdnA for phage-display (carrying ageI restriction site)</P><P><B>Material: </B></P><UL><LI> oligos (25 mM, HPLC purified, ordered by Sigma Aldrich):
</LI></UL><UL><LI> o_mdnA_library, 76
</LI></UL><UL><LI> o_focused_library_AgeI, 85
</LI></UL><UL><LI> 10x Klenow buffer
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><UL><LI>H<SMALL>2</SMALL>O
</LI></UL><UL><LI> Machery Nagel NucleoSpin Extract II Kit
</LI></UL><P><B>Method: </B></P><UL><LI> Reaction mix (total volume: 50 µl)
</LI></UL><UL><LI> 3 µl fw-oligonucleotide
</LI></UL><UL><LI> 3 µl rev-oligonucleotide
</LI></UL><UL><LI> 5 µl dNTPs
</LI></UL><UL><LI> 5 µl Klenow buffer
</LI></UL><UL><LI> 34 µl H<SMALL>2</SMALL>O
</LI></UL><UL><LI> Reaction conditions
</LI></UL><UL><LI> Program: ORIGAMI1
</LI></UL><UL><LI> after finishing program ORIGAMI1:
</LI></UL><UL><LI> addition of 1 µl Klenow-fragment
</LI></UL><UL><LI> incubation 1 hour, 37°C
</LI></UL><UL><LI> PCR purification using Machery Nagel NucleoSpin Extract II Kit, following manufacturer's protocol
</LI></UL><P><B>Results: </B></P><UL><LI> purified hybridized oligos for production of mdnA-library
</LI></UL><P><B>Output:</B></P><UL><LI> hybridization product: purified AgeI-library, named AgeI 2
</LI></UL><UL><LI> stored in box: library, freezer
</LI></UL><P><B>Further tasks:</B></P><UL><LI> restriction enzyme digestion and ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122);"><SPAN class="mw-headline" id="Troubleshooting:_Site-directed_mutagenesis_of_pARW089_.28for_use_as_biobrick.29">Troubleshooting: Site-directed mutagenesis of pARW089 (for use as biobrick)</SPAN></H3><P><B>Investigators:</B> Nicole
</P><P><B>Time:</B> 2011-10-11, 11:00</P><P><B>Aim: </B> Site-directed mutagenesis of XbaI recognition site within the mdn-cluster</P><P><B>Ideas:</B></P><P>Based on QuickChange II Site-Directed mutagenesis Kit, Instruction manual</P><UL><LI> 2nd Transformation: increased DpnI treated DNA -&gt; 4 µl
</LI></UL><UL><LI> Agarose gel --&gt; at least 80% of DNA supercoiled
</LI></UL><UL><LI> Ethanol precipitation of DpnI digested PCR product, resuspension in decreased water volume and transformation
</LI></UL><UL><LI> 2nd PCR using LongPCR enzyme mix (Fermentas)
</LI></UL><P><B>Further tasks:</B></P><P>see ideas
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Megaprimer_PCR_for_site_directed_mutagenesis_2">Megaprimer PCR for site directed mutagenesis</SPAN></H3><P><B>Time:</B> 2011-10-06, 11:30-13:00</P><P><B>Investigators:</B> Nicole, Jessica </P><P><B>Idea:</B>Site-directed mutagenesis of XbaI restriction sites within the mdn cluster (use as biobrick)<B>Materials</B></P><UL><LI> Megaprimer (A and B), 2011-10-06, Jessica&amp;Nicole
</LI></UL><UL><LI> pARW089 clone 2, 85.7 ng/ µl
</LI></UL><UL><LI> LongPCR enzyme mix (Fermentas)
</LI></UL><UL><LI> 10x Long PCR enzyme mix buffer (15 mM MgCl2)
</LI></UL><UL><LI> dNTPs (10 mM)</LI></UL><P><B>Methods:</B><I>PCR mixture</I></P><UL><LI> based on [[File: ]]</LI></UL><UL><LI> A = mdn cluster without XbaI recognition sites at both sites
</LI></UL><UL><LI> B = mdn cluster without XbaI recognition sites at one sites
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TD> dNTPs (10 mM)
</TD><TD> 1
</TD></TR><TR><TD> 10x LOng PCR enzyme mix (Fermentas)
</TD><TD> 5
</TD></TR><TR><TD> Megaprimer
</TD><TD> 0.5 (approx. 50 pmol)
</TD></TR><TR><TD> Template DNA, pARW089, 85.7 ng/ µl
</TD><TD> 1.2
</TD></TR><TR><TD> PfuUltra Polymerase
</TD><TD> 0.3
</TD></TR><TR><TD> H<SMALL>2</SMALL>O
</TD><TD> 42
</TD></TR></TBODY></TABLE><P><I>Thermal cycling program</I></P><P>Program named MAXI5</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Step
</TH><TH> Temperature in °C
</TH><TH> Time in seconds
</TH></TR><TR><TD> Hot start
</TD><TD> 94
</TD><TD> Hold
</TD></TR><TR><TD> Initial Denaturation
</TD><TD> 94
</TD><TD> 300
</TD></TR><TR><TD> Denaturation
</TD><TD> 94
</TD><TD> 20
</TD></TR><TR><TD> Annealing
</TD><TD> 60
</TD><TD> 60
</TD></TR><TR><TD> Extension
</TD><TD> 68
</TD><TD> 8 min 50 sec
</TD></TR><TR><TD> Cycle number
</TD><TD> 7
</TD></TR><TR><TD> Denaturation
</TD><TD> 94
</TD><TD> 20
</TD></TR><TR><TD> Annealing
</TD><TD> 60
</TD><TD> 60
</TD></TR><TR><TD> Extension
</TD><TD> 68
</TD><TD> 8 min 50 sec + 5 sec per cycle
</TD></TR><TR><TD> Cycle number
</TD><TD> 8
</TD></TR></TBODY></TABLE><P><B>Output:</B></P><UL><LI> 2 samples in left thermocycler, labeled A and B</LI></UL><P><B>Further tasks:</B></P><UL><LI> DpnI digestion with the half of each PCR reaction
</LI></UL><UL><LI> Transformation using XL1-Blue cells
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_ligation_product_geneIII_into_pARW089_.28pARWIII.29_for_ligation_with_generated_mdnA_library">Digestion of ligation product geneIII into pARW089 (pARWIII) for ligation with generated mdnA library</SPAN></H3><P><B>Investigator:</B> Sandrina</P><P><B>Aim:</B> get digested pARW089 (without mdnA) and geneIII for ligation with generated mdnA library</P><P><B>Material/Method:</B></P><UL><LI> 30 µl/160 µg/ml pARWIII
</LI></UL><UL><LI> 1 µl restriction enzyme SfoI
</LI></UL><UL><LI> 1 µl restriction enzyme AatII
</LI></UL><UL><LI> 4 µl NEB 10x buffer 4
</LI></UL><UL><LI> 2 µl water
</LI></UL><UL><LI> 3 h, 37°C
</LI></UL><P><B>Further Task:</B></P><UL><LI> purification
</LI></UL><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="DpnI_digestion_and_Transformation_of_mutated_pARW089_2">DpnI digestion and Transformation of mutated pARW089</SPAN></H3><P><B>Time:</B> 2011-10-11</P><P><B>Investigators:</B> Jessica, Nicole </P><P><B>Aim:</B>Site-directed mutagenesis of the mdn cluster (use as biobrick)</P><P><B>Materials:</B></P><UL><LI> DpnI (10 U/ µl)
</LI></UL><UL><LI> PCR products, A and B
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> 4 agar plates with ampicillin
</LI></UL><P><B>Methods:</B></P><P><I>DpnI digestion</I></P><UL><LI> add 0.5 µl DpnI to 25 µl of the PCR product
</LI></UL><DL><DD><UL><LI>0.5 µl DpnI
</LI></UL></DD></DL><DL><DD><UL><LI>25 µl PCR product
</LI></UL></DD></DL><UL><LI> mix gently and thoroughly by pipetting up and down or by vortexing
</LI></UL><UL><LI> incubate 1 hour at 37°C
</LI></UL><UL><LI> the other 25 µl of PCR product: undigested
</LI></UL><P><I>Transformation</I></P><UL><LI> add 2 µl PCR product (one sample = DpnI digested, second sample = undigested) to competent cells
</LI></UL><UL><LI> incubate 30 min one ice
</LI></UL><UL><LI> heat pulse: 45 sec at 42°C
</LI></UL><UL><LI> 2 min on ice
</LI></UL><UL><LI> add 750 µl LB medium to each sample
</LI></UL><UL><LI> incubate 1 hour at 37°C and 850rpm
</LI></UL><UL><LI> cenrifugation and resuspend in 500 µl LB medium
</LI></UL><UL><LI> plate 250 µl on each of two plates with ampicillin or kanamycin antibiotic
</LI></UL><UL><LI> incubate the transformation plates at 37°C for <B>more than 16 hours</B></LI></UL><P><B>Output:</B></P><P>plates in incubator:
</P><UL><LI> Sample a (cluster without any XbaI recognition site), DpnI digested
</LI></UL><UL><LI> Sample a (cluster without any XbaI recognition site), undigested
</LI></UL><UL><LI> Sample b (cluster with one XbaI recognition site), DpnI digested
</LI></UL><UL><LI> Sample b (cluster with one XbaI recognition site), undigested
</LI></UL><P><B>Further tasks:</B></P><UL><LI> preparation of overnight cultures of several clones, especially from DpnI digested samples
</LI></UL><UL><LI> Miniprep
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Over_night_culture_of_retransformed_pPDV089">Over night culture of retransformed pPDV089</SPAN></H3><P><B>Investigator:</B> Sandrina</P><P><B>Aim:</B> production of phages carrying mdnA on their surface</P><P><B>Material/Method:</B></P><UL><LI> LB medium with ampicillin (1:1000)
</LI></UL><UL><LI> incubate shaking over night at 37°C
</LI></UL><P><B>Further Task:</B></P><P>production of phages
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_geneIII_for_ligation_with_generated_mdnA_library">Digestion of geneIII for ligation with generated mdnA library</SPAN></H3><P><B>Investigator:</B> Sandrina</P><P><B>Aim:</B> get digested geneIII for ligation with generated mdnA library</P><P><B>Material/Method:</B></P><UL><LI> 50 µl/10 µg/ml geneIII
</LI></UL><UL><LI> 1 µl restriction enzyme NgoMIV
</LI></UL><UL><LI> 6 µl NEB 10x buffer 1
</LI></UL><UL><LI> 3 µl water
</LI></UL><UL><LI> 1 h, 37°C
</LI></UL><P><B>Further Task:</B></P><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_mutated_pARW089_with_increased_amount">Transformation of mutated pARW089 with increased amount</SPAN></H3><P><B>Time:</B> 2011-10-11, 17:00-20:00</P><P><B>Investigators:</B> Jessica, Nicole </P><P><B>Aim:</B>Site-directed mutagenesis of the mdn cluster (use as biobrick)</P><P><B>Materials:</B></P><UL><LI> PCR products, 2A and 2B
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> 2 agar plates with ampicillin
</LI></UL><P><B>Methods:</B></P><UL><LI> add 2 µl PCR product (one sample = DpnI digested, second sample = undigested) to competent cells
</LI></UL><UL><LI> incubate 30 min one ice
</LI></UL><UL><LI> heat pulse: 45 sec at 42°C
</LI></UL><UL><LI> 2 min on ice
</LI></UL><UL><LI> add 750 µl LB medium to each sample
</LI></UL><UL><LI> incubate 1 hour at 37°C and 850rpm
</LI></UL><UL><LI> cenrifugation and resuspend in 500 µl LB medium
</LI></UL><UL><LI> plate 250 µl on each of two plates with ampicillin or kanamycin antibiotic
</LI></UL><UL><LI> incubate the transformation plates at 37°C for <B>more than 16 hours</B></LI></UL><P><B>Output:</B></P><P>plates in incubator:
</P><UL><LI> Sample 2A (cluster without any XbaI recognition site), DpnI digested
</LI></UL><UL><LI> Sample 2B (cluster with one XbaI recognition site), DpnI digested
</LI></UL><P><B>Further tasks:</B></P><UL><LI> preparation of overnight cultures of several clones, especially from DpnI digested samples
</LI></UL><UL><LI> Miniprep
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_mdnA_library_.28for_phage_display.29_using_AgeI">Digestion of mdnA library (for phage display) using AgeI</SPAN></H3><P><B>Investigator:</B> Jessica, Nicole</P><P><B>Time:</B> 2011-10-11, 17:00-18:00</P><P><B>Aim:</B> Production of pUPIII (carrying mdnA-library and gene III) for screening by phage display</P><P><B>Materials:</B></P><UL><LI> NEB Buffer 1
</LI></UL><UL><LI> AgeI (NEB)
</LI></UL><UL><LI> purified oligos, lib AgeI, Nic, 2011-10-11
</LI></UL><P><B>Methods:</B></P><UL><LI> 25 µl purified PCR product (lib AgeI)
</LI></UL><UL><LI> 3 µl NEB Buffer 1
</LI></UL><UL><LI> 2 µl AgeI
</LI></UL><P><B>Result/ Output:</B></P><UL><LI> digested library for phage display, digested lib AgeI, stored freezer (box library)
</LI></UL><P><B>Further Task:</B></P><UL><LI> purification of digested lib AgeI
</LI></UL><UL><LI> tripple ligation with gene III and pARWIII
</LI></UL><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Purification_of_digested_mdnA_library_and_digested_gene_III">Purification of digested mdnA library and digested gene III</SPAN></H3><P><B>Investigator:</B> Sandrina, Jessica, Nicole</P><P><B>Time:</B> 2011-10-11, 18:00-19:00</P><P><B>Aim:</B> Production of pUPIII (carrying mdnA-library and gene III) for screening by phage display</P><P><B>Materials:</B></P><UL><LI> Machery Nagel Nucleo Spin Extract II Kit
</LI></UL><P><B>Methods:</B></P><UL><LI> following Machery Nagel Nucleo Spin Extract II Kit
</LI></UL><UL><LI> Exceptions:
</LI></UL><UL><LI> Elution buffer: H2O
</LI></UL><UL><LI> Elution volume: 25 µl
</LI></UL><UL><LI> Incubation 5 min, 70°C (before 1 min centrifugation)
</LI></UL><P><B>Results:</B></P><UL><LI> concentrations:
</LI></UL><UL><LI> mdnA library: c = 174 ng/ µl
</LI></UL><UL><LI> gene III: c = 20.7 ng/ µl
</LI></UL><P><B>Output:</B></P><UL><LI> purified and digested
</LI></UL><UL><LI> gene III
</LI></UL><UL><LI> mdnA library
</LI></UL><P><B>Further Task:</B></P><UL><LI> tripple ligation with gene III and pARWIII
</LI></UL><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Triple_Ligation_with_pARWIII.2C_mdnA-library_and_gene_III">Triple Ligation with pARWIII, mdnA-library and gene III</SPAN></H3><P><B>Investigator:</B> Sandrina, Jessica, Nicole</P><P><B>Time:</B> 2011-10-11, 19:00-20:00</P><P><B>Aim:</B> Production of pUPIII (carrying mdnA-library and gene III) for screening by phage display</P><P><B>Materials:</B></P><UL><LI> T4 DNA ligase buffer (Fermentas)
</LI></UL><UL><LI> T4 DNA ligase
</LI></UL><UL><LI> purified and digested gene III, 2011-10-11, Nic
</LI></UL><UL><LI> purified and digested mdnA library, 2011-10-11, Nic
</LI></UL><UL><LI> purified and digested pARWIII, 2011-10-11, San
</LI></UL><P><B>Methods:</B></P><P><I>Concentrations and size of inserts and backbone</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Concentration in ng/ µl
</TH><TH> Size in bases
</TH></TR><TR><TD> gene III
</TD><TD> 20.7
</TD><TD> 541
</TD></TR><TR><TD> mdnA library
</TD><TD> 174
</TD><TD> 161
</TD></TR><TR><TD> pARWIII
</TD><TD> 34.8
</TD><TD> 9989
</TD></TR></TBODY></TABLE><P><I>Ligation mix</I></P><UL><LI> 2 µl T4 DNA ligase buffer
</LI></UL><UL><LI> 1 µl T4 DNA ligase
</LI></UL><UL><LI> 13 µl pARWIII
</LI></UL><UL><LI> 3.5 µl gene III
</LI></UL><UL><LI> 1 µl mdnA library
</LI></UL><UL><LI> for transformation control: instead of geneIII and mdnA library 4.5 µl H2O
</LI></UL><P><B>Results/ OUtput:</B></P><UL><LI> ligation product, named geneIII mdnA lib pARWIII, 2011-10-11, stored in freezer (libary box)
</LI></UL><UL><LI> transformation control, named H2O, 2011-10-11, stored in freezer (library box)
</LI></UL><P><B>Further Task:</B></P><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_triple_ligated_mdnA_library.2C_gene_III_and_pARWIII">Transformation of triple ligated mdnA library, gene III and pARWIII</SPAN></H3><P><B>Time:</B> 2011-10-11, 19:45-22:00</P><P><B>Investigators:</B> Sandrina, Jessica, Nicole </P><P><B>Aim:</B>Site-directed mutagenesis of the mdn cluster (use as biobrick)</P><P><B>Materials:</B></P><UL><LI> XL1-Blue chemocompetent cells
</LI></UL><UL><LI> ligation product, named geneIII mdnA lib pARWIII, 2011-10-11
</LI></UL><UL><LI> transformation control, named H2O, 2011-10-11
</LI></UL><UL><LI> add 2 µl PCR product to competent cells (for ligation product 5 samples overall)
</LI></UL><UL><LI> incubate 25 min one ice
</LI></UL><UL><LI> heat pulse: 45 sec at 42°C
</LI></UL><UL><LI> 2 min on ice
</LI></UL><UL><LI> add 750 µl LB medium to each sample
</LI></UL><UL><LI> incubate 45 min at 37°C and 850rpm
</LI></UL><UL><LI> cenrifugation and resuspend in 200 µl LB medium
</LI></UL><UL><LI> plate 200 µl on plates with ampicillin
</LI></UL><UL><LI> incubate the transformation plates at 37°C for approx. 16 hours
</LI></UL><P><B>Output:</B></P><P>plates in incubator:
</P><UL><LI> pUPIII 1-5, 2011-10-11, San&amp;Nic
</LI></UL><UL><LI> transformation control, 2011-10-11, San&amp;Nic
</LI></UL><P><B>Further tasks:</B></P><UL><LI> preparation of overnight cultures of several clones
</LI></UL><UL><LI> Miniprep
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Sidedirected_mutagenesis_of_HRV14_3C_protease">Sidedirected mutagenesis of HRV14 3C protease</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[1]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P>Aim: Sidedirected mutagenesis of TEV and PreSciccion protease</P><P><B>Materials:</B></P><UL><LI> HRV14 3C protease vector (P10 by Gunther Stier)
</LI></UL><UL><LI> different Primers
</LI></UL><UL><LI> Phusion polymerase kit by NEB
</LI></UL><P><B>Methode:</B></P><P>PCR</P><UL><LI> Template: 1 µL
</LI></UL><UL><LI> Nucleotides: 1 µL of 10 mM ready to use dNTP mix</LI></UL><UL><LI> 10 µL 5 x Phusion GC Buffer</LI></UL><UL><LI> 2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)</LI></UL><UL><LI> 32,5 µL of pure water</LI></UL><UL><LI>0,5 µL Phusion HF polymerase</LI></UL><P>Program:
</P><UL><LI>Denat: 30 sec 98°C
</LI></UL><UL><LI>30x
</LI></UL><P>Denat: 10 sec 98°C
</P><P>Annealing+Extend: 25 sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P>Used Primers:
</P><P>Fragment 1 of HRV14 3C protease (P1):
</P><UL><LI> f_HRV14 3C_AraFusion_NgoMIV
</LI></UL><UL><LI> r_HRV14 3C_ACCAGC
</LI></UL><P>Fragment 2 of HRV14 3Cprotease (P2):
</P><UL><LI> f_HRV14 3C_ACCAGC
</LI></UL><UL><LI> r_HRV14 3C_tm_Xba208_A-T
</LI></UL><P>Fragment 3 of HRV14 3Cprotease (P3):
</P><UL><LI> f_HRV14 3C_tm_Xba208_A-T
</LI></UL><UL><LI> r_HRV14 3C_tm_Xba280_A-T
</LI></UL><P>Fragment 4 of HRV14 3Cprotease (P4):
</P><UL><LI> f_HRV14 3C_tm_Xba280_A-T
</LI></UL><UL><LI> r_HRV14 3C_iGEM_BamHI
</LI></UL><P><B>Further Tasks:</B></P><P>PCR purification, analytical Gel to check sizes of the amplified fragments, assembly PCR of Fragment </P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Analytical_gelelctrophoresis_and_PCR_clean_up_of_sidedirected_mutated_fragments_P1.2C_P2.2CP3_and_P4_of_HRV14_3C">Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments P1, P2,P3 and P4 of HRV14 3C</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[2]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P><B>Aim:</B> Check the size of amplified fragments and clean them up after PCR</P><P><B>Materials:</B></P><UL><LI> PCR fragments P1-P4
</LI></UL><UL><LI> Fermentas 6x loading dye
</LI></UL><UL><LI> 100bp ladder (Gene Ruler by Fermentas)
</LI></UL><UL><LI> Macherey-Nagel Nucleo Spin II Kit for clean up
</LI></UL><P><B>Methode:</B></P><UL><LI> Clean up after manual for PCR purification of the used Kit
</LI></UL><UL><LI> gel electophoresis (1.5 % agarose gel with analytical gel pockets): 30 min at 110 V
</LI></UL><P><B>Results:</B></P><P>The sizes of the fragments of the PCR products are matching the fragment sizes extracted with Geneious.
</P><P><B>Further Tasks:</B> Merge of PCR fragments P5 and P6 into PreSciccion protease, digestion of TEV protease with NgoMIV and BamHI, Ligation of TEV protease with digested AraC induction system (NgoMIV and HindIII) and digested pJC354_ssTorA_XhoI_CS-TEV_NheI_BlaFL (HindIII and BamHI)
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Assembly_PCR_of_mutated_HRV14_3C_protease_fragments">Assembly PCR of mutated HRV14 3C protease fragments</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[3]</A>]</B></P><P><B>Investigators:</B> Sascha, Stefan, Sebastian</P><P>Aim: Assembly of mutated TEV and PreSciccion protease fragments</P><P><B>Materials:</B></P><UL><LI> HRV14 3C protease fragments P1 and P2 --&gt; P5
</LI></UL><UL><LI> HRV14 3C protease fragments P3 and P4 --&gt; P6
</LI></UL><UL><LI> different Primers
</LI></UL><UL><LI> Phusion polymerase kit by NEB
</LI></UL><P><B>Methode:</B></P><P>PCR</P><UL><LI> Template: 2 µL (1µl of each fragment after sizes were checked and fragments got cleaned up with PCR clean up kit from Macherey-Nagel (Nucleo Spin Kit))
</LI></UL><UL><LI> Nucleotides: 1 µL of 10 mM ready to use dNTP mix</LI></UL><UL><LI> 10 µL 5 x Phusion HF Buffer</LI></UL><UL><LI> 2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)</LI></UL><UL><LI> 31,5 µL of pure water</LI></UL><UL><LI>0,5 µL Phusion HF polymerase</LI></UL><P>Program:
</P><UL><LI>Denat: 30 sec 98°C
</LI></UL><UL><LI>30x
</LI></UL><P>Denat: 10 sec 98°C
</P><P>Annealing+Extend: 35 sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P>Used Primers:
</P><P>Assembly of fragment 1 and fragment 2 of HRV14 3C protease (P1+P2)--&gt; mutated HRV14 3C protease fragment 5 (P5):
</P><UL><LI> f_HRV14 3C_AraFusion_NgoMIV
</LI></UL><UL><LI> r_HRV14 3C_tm_Xba208_A-T
</LI></UL><P>Assembly of fragment 3 and fragment 4 of HRV14 3C protease (P3+P4)--&gt; mutated HRV14 3C protease fragment 6 (P6):
</P><UL><LI> f_HRV14 3C_tm_Xba208_A-T
</LI></UL><UL><LI> r_HRV14 3C_iGEM_BamHI
</LI></UL><P><B>Further Tasks:</B></P><P>PCR purification, analytical Gel to check sizes of the amplified fragments, assembly PCR of Fragment </P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Analytical_gelelctrophoresis_and_PCR_clean_up_of_sidedirected_mutated_fragments_P5_and_P6_of_HRV14_3C">Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments P5 and P6 of HRV14 3C</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[4]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P><B>Aim:</B> Check the size of amplified fragments and clean them up after PCR</P><P><B>Materials:</B></P><UL><LI> PCR fragments P5 and P6
</LI></UL><UL><LI> Fermentas 6x loading dye
</LI></UL><UL><LI> 100bp ladder (Gene Ruler by Fermentas)
</LI></UL><UL><LI> Macherey-Nagel Nucleo Spin II Kit for clean up
</LI></UL><P><B>Methode:</B></P><UL><LI> Clean up after manual for PCR purification of the used Kit
</LI></UL><UL><LI> gel electophoresis (1.5 % agarose gel with analytical gel pockets): 30 min at 110 V
</LI></UL><P><B>Results:</B></P><P>The sizes of the fragments of the PCR products are matching the fragment sizes extracted with Geneious.
</P><P><B>Further Tasks:</B> Merge of PCR fragments P5 and P6 into PreSciccion protease, digestion of TEV protease with NgoMIV and BamHI, Ligation of TEV protease with digested AraC induction system (NgoMIV and HindIII) and digested pJC354_ssTorA_XhoI_CS-TEV_NheI_BlaFL (HindIII and BamHI)
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Assembly_PCR_of_mutated_HRV14_3C_protease_fragments_P5_and_P6">Assembly PCR of mutated HRV14 3C protease fragments P5 and P6</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[5]</A>]</B></P><P><B>Investigators:</B> Sascha, Stefan, Sebastian</P><P>Aim: Assembly of mutated TEV and PreSciccion protease fragments</P><P><B>Materials:</B></P><UL><LI> PreSciccion protease fragment P5
</LI></UL><UL><LI> PreSciccion protease fragment P6
</LI></UL><UL><LI> different Primers
</LI></UL><UL><LI> Phusion polymerase kit by NEB
</LI></UL><P><B>Methode:</B></P><P>PCR</P><UL><LI> Template: 2 µL (1µl of each fragment after sizes were checked and fragments got cleaned up with PCR clean up kit from Macherey-Nagel (Nucleo Spin Kit))
</LI></UL><UL><LI> Nucleotides: 1 µL of 10 mM ready to use dNTP mix</LI></UL><UL><LI> 10 µL 5 x Phusion HF Buffer</LI></UL><UL><LI> 2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)</LI></UL><UL><LI> 31,5 µL of pure water</LI></UL><UL><LI>0,5 µL Phusion HF polymerase</LI></UL><P>Program:
</P><UL><LI>Denat: 30 sec 98°C
</LI></UL><UL><LI>30x
</LI></UL><P>Denat: 10 sec 98°C
</P><P>Anneal+Extend: 35 sec 72°C
</P><UL><LI>Final Extend: 5 min 72°C
</LI></UL><P>Used Primers:
</P><P>Assembly of fragment 5 and fragment 6 of HRV14 3C protease (P3+P4)--&gt; mutated PreSciccion portease fragment 6 (P6):
</P><UL><LI> f_HRV14 3C_AraFusion_NgoMIV
</LI></UL><UL><LI> r_HRV14 3C_iGEM_BamHI
</LI></UL><P><B>Further Tasks:</B></P><P>PCR purification, analytical Gel to check sizes of the amplified fragments, digest and ligation with other fragments </P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="105th_Labday_2011-10-12">105th Labday 2011-10-12</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Production_of_phages_containing_pPDV089_in_XL1-blue_cells_2">Production of phages containing pPDV089 in XL1-blue cells</SPAN></H3><P><B>Investigators:</B>Sandrina</P><P><B>Aim:</B> production of phages carrying unmodified mdnA on their surface</P><P><B>Method/Materials:</B></P><UL><LI><B>first step</B>: amplification of cells containing pPDV089 (clone: 2S14):
</LI></UL><UL><LI> 500 ml LB medium will be inoculated with the cells, so that OD600 = 0.1
</LI></UL><UL><LI> add antibiotics tetracyclin and ampicillin to the medium
</LI></UL><UL><LI> cells should be incubated 37 °C till OD 600 = 0.3-0.5 (here: 0.356) is reached
</LI></UL><UL><LI><B>second step</B>: infection with helper phages
</LI></UL><UL><LI> add helper phages 10^11 phages/50 ml (3,5 µl)
</LI></UL><UL><LI> incubate for 10 min at 37°C (without shaking!)
</LI></UL><UL><LI> add 0,5 mM IPTG
</LI></UL><UL><LI> incubate 50 min at 28°C and rpm
</LI></UL><UL><LI> add 70 µg/ml kanamycin and incubate for 5 h at 28°C (...)
</LI></UL><UL><LI><B>third step</B>: phage purification
</LI></UL><UL><LI> centrifuge cell culture at 5000 x g/ 15 min
</LI></UL><UL><LI> fill supernatant in a new 50 ml falcon and centrifuge again (5000 g/15 min)
</LI></UL><UL><LI> 40 ml of the supernatant with 8 ml PEG-NaCl (20% (w/v) PEG-8000, 2,5 M NaCl)
</LI></UL><UL><LI> incubate over night at 4°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> go on with phage purification</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Production_of_microviridin">Production of microviridin</SPAN></H3><P><B>Time:</B> 2011-10-12, 09:30-19:30</P><P><B>Investigators:</B> Sandrina, Jessica, Nicole </P><P><B>Aim:</B> Culturing cells to produce microviridin</P><P><B>Materials:</B></P><UL><LI> LB medium
</LI></UL><UL><LI> overnight cultures of pARW089
</LI></UL><UL><LI> 500mM Tris-HCL
</LI></UL><P><B>Method:</B></P><UL><LI> 17 ml overnight culture + 374 ml LB medium
</LI></UL><UL><LI> incubation for 7.5 h
</LI></UL><DL><DD><UL><LI> one @ 30°C
</LI></UL></DD></DL><DL><DD><UL><LI> one @ 37°C
</LI></UL></DD></DL><UL><LI> centrifuged @ 4000g for 15 min
</LI></UL><UL><LI> washed with 500mM Tris-HCL
</LI></UL><UL><LI> centrifuged
</LI></UL><UL><LI> supernatant discarded
</LI></UL><UL><LI> pellet stored @ -20°C
</LI></UL><P><B>Output:</B></P><UL><LI> four 50 ml falcons (2x 30°C, 2x 37°C)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> preparing samples for HPLC
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_HRV14_3C.2C_AraC_and_Vector_SG11"> Digest of HRV14 3C, AraC and Vector SG11 </SPAN></H3><P><B>Investigator:</B> Paul Stefan Sebastian</P><P><B>Material:</B></P><P>Enzymes: (all pruchased from NEB)</P><UL><LI> EcoRI-HF
</LI></UL><UL><LI> PstI-HF
</LI></UL><UL><LI> NgoMIV
</LI></UL><P>DNA-Fragments:</P><UL><LI> mutated HRV14 3Cprotease
</LI></UL><UL><LI> AraC
</LI></UL><UL><LI> SG11
</LI></UL><P>NEBuffer 4</P><P><B>Methode:</B></P><P>All digestion batches got a total volume of 10µl and were incubated for 2 h at 37°C:
</P><UL><LI>1µl Buffer
</LI></UL><UL><LI>1µl each Enzyme
</LI></UL><UL><LI>0,1 µl BSA if required
</LI></UL><UL><LI>7 or 6,9 µl DNA
</LI></UL><P>HRV14 3C got digested with PstI-HF+NgoMIV, AraC was digested with NgoMIV+EcoRI-HF. Vector SG11 which containing the cleavage site for the HRV14 3C got digested with EcoRI-HF+PstI-HF.
</P><P><B>Output:</B></P><P>Digested fragements:
</P><UL><LI>NgoMIV_HRV14 3C_PstI
</LI></UL><UL><LI>EcoRI_AraC_NgoMIV
</LI></UL><UL><LI>PstI_SG11_EcoRI
</LI></UL><P><B>Further Tasks:</B></P><P>Clean up of all fragments with PCR CleanUp Kit by M&amp;N, ligation of fragments and transformation into E.coli XL1 blue.
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_digested_fragments_of_HRV14_3C_protease_and_vector_SG11_and_Transformation_of_XL1_blue_with_Ligation_products"> Ligation of digested fragments of HRV14_3C protease and vector SG11 and Transformation of XL1 blue with Ligation products</SPAN></H3><P><B>Investigators:</B> Stefan, Sascha, Paul</P><P><B>Materials:</B></P><P>Fragments:</P><UL><LI>NgoMIV_PreSciccion_PstI
</LI></UL><UL><LI>EcoRI_AraC_NgoMIV
</LI></UL><UL><LI>PstI_SG11_EcoRI
</LI></UL><P>QuickLigase purchased by Fermentas</P><P>QuickLigase 10x Buffer purchased by Fermentas</P><P><B>Methodes:</B></P><P>Standard ligation protocol from Fermantas for QuickLigase Kit. Ratio between vector and both fragments (protease and AraC-induction system) where 3:3:1.</P><P>Batch was incubated for 10 min at 37°C. After 10 min, all batches where put on ice.
</P><P><B>Output:</B></P><UL><LI> ligation batch
</LI></UL><P><B>Next Step:</B></P><P>Transformation of XL1 blue <I>E.coli</I> with ligation batch using the standard protocol for heat shock transformation. Transformed cells were platted on agar plates containing cm as antibiotic.</P><P><B>Further tasks:</B></P><UL><LI> colony PCR to get to know, which clones are positive and for all positive clones: mini-prep of plasmid DNA, sequencing and preparation of glycerol stock clutures.
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="106th_Labday_2011-10-13">106th Labday 2011-10-13</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparation_of_samples_for_HPLC_analysis">Preparation of samples for HPLC analysis</SPAN></H3><P><B>Time:</B> 2011-10-13</P><P><B>Investigators:</B> Jessica, Nicole</P><P><B>Material:</B></P><UL><LI> pellets of pARW089 samples (from 2011-10-13)
</LI></UL><UL><LI> 100% methanol
</LI></UL><UL><LI> 5% methanol
</LI></UL><UL><LI> sterile water
</LI></UL><UL><LI> Sep-Pak Cartridges
</LI></UL><UL><LI> speed vac
</LI></UL><P><B>Method:</B></P><UL><LI> pellet was resuspended in 5 ml of sterile water
</LI></UL><UL><LI> the resuspended sample was sonicated for 4 min (2 sec on, 2 sec off)
</LI></UL><UL><LI> centrifugation for 15 min @ 4000 rpm in eppendorf Centrifuge 5810 R
</LI></UL><UL><LI> supernatant was transfered to Sep-Pak Plus C18 Cartridges, that were equilibrated with 2 ml of 100% methanol
</LI></UL><UL><LI> cartridges were washed with 2 ml water
</LI></UL><UL><LI> samples were loaded
</LI></UL><UL><LI> cartridges were washed with 2 ml of 5% methanol
</LI></UL><UL><LI> samples were eluted with 2 ml of 100% methanol
</LI></UL><UL><LI> samples were put in a speedvac so that the methanol can evaporate
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_mutated_pARW089_using_super_competent_XL1_gold_cells">Transformation of mutated pARW089 using super competent XL1 gold cells</SPAN></H3><P><B>Time:</B> 2011-10-13, 16:30-19:00</P><P><B>Investigators:</B> Jessica, Nicole</P><P><B>Aim:</B> Site-directed mutagenesis of pARW089 (carrying XbaI recognition sites) for use as biobrick</P><P><B>Material:</B></P><UL><LI> super competent XL1 gold cells (tet and cm resistance), given by Site directed mutagenesis lightening kit (stratagene)
</LI></UL><UL><LI> PCR product of site-directed mutagenesis, DpnI digested, Nic&amp;Jes 2011-10-07
</LI></UL><UL><LI> ß-Mercaptoethanol
</LI></UL><UL><LI> SOC medium
</LI></UL><UL><LI> Agarplate containing Amp
</LI></UL><P><B>Method:</B></P><P>Followed by manufacturer's protocol <A href="/File:UP" title="File:UP">File:UP</A></P><UL><LI> use 45 µl cells (thawn at ice)
</LI></UL><UL><LI> add 2 µl ß-mercaptoethanol
</LI></UL><UL><LI> incubate 2 min on ice
</LI></UL><UL><LI> add 2 µl DpnI-treated sample (mixing)
</LI></UL><UL><LI> incubate 30 min on ice
</LI></UL><UL><LI> heat impulse: 42°C 30 sec
</LI></UL><UL><LI> incubate 2 min on ice
</LI></UL><UL><LI> add 500 µl SOC medium
</LI></UL><UL><LI> incubate 1 hour at 37°C and 250 rpm
</LI></UL><UL><LI> plate 500 µl on one agarplate (Amp)
</LI></UL><UL><LI> incubate over night at 37°C
</LI></UL><P><B>Results/ Output:</B></P><UL><LI> agar plate with mutated pARW089, named DpnI dig. pARW089 2011-10-13
</LI></UL><UL><LI> stored in 37°C incubator
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking over night cultures
</LI></UL><UL><LI> miniprep
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Purification_of_phages_containing_pPDV089_2">Purification of phages containing pPDV089</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> phages carrying unmodified mdnA on their surface</P><P><B>Method/Materials:</B></P><UL><LI> after over night incubation:
</LI></UL><UL><LI> centrifuge precipitaed phages: 5000 x g/ 45 min
</LI></UL><UL><LI> discard supernatant
</LI></UL><UL><LI> centrifuge again, 5000 x g/ 5 min
</LI></UL><UL><LI> remove supernatant carefully
</LI></UL><UL><LI> take pellet in 1 ml TBS
</LI></UL><UL><LI> move in 1.5 ml Eppi
</LI></UL><UL><LI> centrifuge again: 17000 x g/10 min
</LI></UL><UL><LI> supernatant in a new Eppi with 200 µl PEG-NaCl (mix!)
</LI></UL><UL><LI> incubate on ice for 60 min
</LI></UL><UL><LI> centrifuge precipitated phages: 17.000 x g/ 10 min (4°C)
</LI></UL><UL><LI> resuspend pellet in 300 µl TBS
</LI></UL><UL><LI> centrifuge:17 000 x g/ 10 min (4°C)
</LI></UL><UL><LI> supernatant in a fresh eppi= purified phages
</LI></UL><P><B>Further tasks:</B></P><UL><LI> ELISA and phage display</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Colony_PCR_of_plated_HRV14_3C_containing_colonies"> Colony PCR of plated HRV14_3C containing colonies </SPAN></H3><P><B>Investigators:</B> Stefan, Paul, Sebastian
</P><P><B>Aim:</B></P><P>Checking for positive clones</P><P><B>Used Primer:</B></P><P>f_HRV14_3C_iGEM, r_HRV14_3C_iGem:BamHI
</P><P><B> Method: </B></P><UL><LI>2,5µl of each primer (10µM) = 5µl
</LI></UL><UL><LI>5µl 10y polymerase buffer
</LI></UL><UL><LI>2µl 25mM MgCl2
</LI></UL><UL><LI>1µl dNTP mix
</LI></UL><UL><LI>0,5µl Taq-Polymerase (GenAxxon)
</LI></UL><UL><LI>36,5 µl H2O
</LI></UL><P>=50µl
</P><P>Colonies were picked with a 20µl tip, dipped into PCR batch, and then plunged into 1ml LB to grew cells from picked colony (from these 1ml batches overnight cultures will be inoculated in case of positive clones).
</P><UL><LI> 20 colonies were picked HRV14 3C protease (see entry from 01.09.2011) = 20 PCR batches.
</LI></UL><P><B>PCR Program:</B></P><P>Initial denat = 3min 94°C
</P><P>25x
</P><P>denat: 2min 10sec 94°C
</P><P>anneal: 2min 10sec 70°C
</P><P>extend: 1min 72°C
</P><P>final extend: 10min 72°C
</P><UL><LI> The PCR products were resolved on a 1% analytical agarose gel
</LI></UL><P><B>Results: </B></P><P>all clones were positive, 2 were selected for further studies.
</P><P><B>Further Tasks: </B></P><P>overnight culture, mini prep, test digest, sequencing of clones and survival screening
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="107th_Labday_2011-10-14">107th Labday 2011-10-14</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_pARWIII_and_ligation_with_library">Digest of pARWIII and ligation with library</SPAN></H3><P><B>Time:</B> 2011-10-14</P><P><B>Investigators:</B> Sandrina, Jessica, Nicole</P><P><B>Material:</B></P><P><B>Method:</B></P><P>Digest of pARWIII
</P><P>Gel electrophoresis
</P><P>Gel purification
</P><P>Digest of geneIII
</P><P>Purification
</P><P>Ligation
</P><UL><LI> 2 µl T4 ligase buffer
</LI></UL><UL><LI> 1 µl T4 ligase
</LI></UL><UL><LI> 6 µl pARW089
</LI></UL><UL><LI> 10 µl geneIII
</LI></UL><UL><LI> 1 µl mdnA library
</LI></UL><P><B>Output:</B></P><UL><LI> 3 ligation products stored @ -20°C
</LI></UL><P><B>Further task:</B></P><UL><LI> transformation of one ligation product and control
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="108th_Labday_2011-10-15">108th Labday 2011-10-15</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_with_library_for_phage_display">Transformation with library for phage display</SPAN></H3><P><B>Time:</B> 2011-10-15</P><P><B>Investigators:</B> Sabine, Jessica</P><P><B>Material:</B></P><P><B>Method:</B></P><P><B>Output:</B></P><P><B>Further task:</B></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep.2C_Glycerol_stock_cultures_and_Survival-Test_of_HRV14_3C_clone.28SG24-P10.29">Miniprep, Glycerol stock cultures and Survival-Test of HRV14 3C clone(SG24-P10)</SPAN></H3><P><B> Investigators: </B> Paul, Stefan, Sascha
</P><P><B> Aim: </B> Getting Plasmid DNA, establish a glycerol stock culture and test whether TorA-detection device is working</P><P><B>Mateials:</B>:
</P><UL><LI> Mini prep Kit purchased by M&amp;N
</LI></UL><UL><LI> steril Glycerol
</LI></UL><UL><LI> different Agarplates
</LI></UL><UL><LI>overnight culture of SG24
</LI></UL><P><B> Plates: </B></P><UL><LI>1x Cm (25 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM IPTG
</LI></UL><UL><LI>1x Cm + 2% Ara
</LI></UL><UL><LI>1x Cm + 2% Ara +1mM IPTG
</LI></UL><UL><LI>1x Cm + Amp (100 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (25 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (50 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (75 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (100 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (150 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (200 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (300 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (400 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (600 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (800 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (25 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (50 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (75 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (100 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (150 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (200 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (300 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (400 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (600 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (800 µg/ml)
</LI></UL><P><B> Method: </B></P><UL><LI> for glycerol stock, cells were mixed in a 1:1 ratio with steril glycerol
</LI></UL><UL><LI> plasmid isolation based on the manual of M&amp;N
</LI></UL><UL><LI>for survival screening 50µl of cell dilution with OD=0.002 were plated on prepared plates
</LI></UL><P><B> Results: </B></P><P>No Cell death visible, repeat with different Temperature!
</P><P><B> Further Tasks:</B></P><UL><LI> repeat survival test at 30 °C
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="109th_Labday_2011-10-16">109th Labday 2011-10-16</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Overnight_culture_of_library">Overnight culture of library</SPAN></H3><P><B>Time:</B> 2011-10-16</P><P><B>Investigators:</B> Jessica</P><P><B>Material:</B></P><P><B>Method:</B></P><P><B>Output:</B></P><P><B>Further task:</B></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Survival-Test_of_one_HRV14_3C_clone.28_SG24-P10.29">Survival-Test of one HRV14_3C clone( SG24-P10)</SPAN></H3><P><B> Investigators: </B> Paul, Stefan, Sascha
</P><P><B> Aim: </B> Test whether TorA-detection device functions
</P><P><B>Mateial</B></P><P>different agaplates, overnight culture of SG24
</P><P><B> Plates: </B></P><UL><LI>1x Cm (25 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM IPTG
</LI></UL><UL><LI>1x Cm + 2% Ara
</LI></UL><UL><LI>1x Cm + 2% Ara +1mM IPTG
</LI></UL><UL><LI>1x Cm + Amp (100 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (25 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (50 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (75 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (100 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (150 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (200 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (300 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (400 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (600 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (800 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (25 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (50 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (75 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (100 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (150 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (200 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (300 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (400 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (600 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (800 µg/ml)
</LI></UL><P><B> Method: </B></P><UL><LI>for survival screening 50µl of cell dilution with OD=0.002 were plated on prepared plates and incubated for 2 days at 30°c
</LI></UL><P><B> Results: </B></P><P>No Cell death visible!
</P><P><B> Further Tasks:</B></P><UL><LI> thinking in progress...
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="110th_Labday_2011-10-17">110th Labday 2011-10-17</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Amplification_of_megaprimer_1_.28site-directed_mutagenesis.29">Amplification of megaprimer 1 (site-directed mutagenesis)</SPAN></H3><P><B>Time:</B> 2011-10-17</P><P><B>Investigators:</B> Nadja, Jessica</P><P><B>Aim: </B>Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used</P><P><B>Material:</B></P><UL><LI> primers 86, 87, 88
</LI></UL><UL><LI> pARW089, 8.57 ng/ µl
</LI></UL><P><B>Method:</B></P><P>Samples and used primers
</P><UL><LI>a) pARW089 with both mutations: pf_muta_XbaI (86), pr_muta_XbaI (87)
</LI></UL><UL><LI>b) pARW089 with one mutation: pf_muta_XbaI (86), pr_ohne_muta (88)
</LI></UL><P>PCR mixture
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Volume in µl
</TH></TR><TR><TD> dNTPs (10 mM)</TD><TD> 1
</TD></TR><TR><TD> 10x Genaxxon buffer S</TD><TD> 5
</TD></TR><TR><TD>25 mM MgCL2</TD><TD>2
</TD></TR><TR><TD>Primer forward (10 µM)</TD><TD>1.5
</TD></TR><TR><TD>Primer reverse (10 µM)</TD><TD>1.5
</TD></TR><TR><TD>DNA pARW089 (8.57 ng/ul)</TD><TD>1
</TD></TR><TR><TD>Genaxxon Taq Polymerase S</TD><TD>0.5
</TD></TR><TR><TD>H2O</TD><TD> 37.5
</TD></TR></TBODY></TABLE><P>PCR conditions (Thermal cycler)</P><P>Programm IGMI1
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Time</TH><TH> Temp.
</TH></TR><TR><TD>Hold</TD><TD>94°C
</TD></TR><TR><TD>Initial denaturation </TD><TD>3 min</TD><TD>94°C
</TD></TR><TR><TD>Denaturation </TD><TD>45 sec</TD><TD> 94°C
</TD></TR><TR><TD>Annealing</TD><TD> 45 sec</TD><TD> 51°C
</TD></TR><TR><TD>Extension </TD><TD>45 sec</TD><TD> 72°C
</TD></TR><TR><TD>Number of cycles</TD><TD> 30 x</TD></TR><TR><TD>Final extension</TD><TD> 10 min </TD><TD>72°C
</TD></TR></TBODY></TABLE><P><B>Output:</B></P><UL><LI> PCR products, A and B
</LI></UL><P><B>Further task:</B></P><UL><LI> Agarose gel ellectrophoresis and purification
</LI></UL><UL><LI> Megaprimer PCR 2 (with this purified primers)
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Megaprimer_PCR_II_.28amplification_of_2nd_megaprimer_.282nd_part_of_mdn_cluster.2C_site-directed_mutagenesis.29">Megaprimer PCR II (amplification of 2nd megaprimer (2nd part of mdn cluster, site-directed mutagenesis)</SPAN></H3><P><B>Time:</B> 2011-10-18</P><P><B>Investigators:</B> Nadja, Jessica, Nicole</P><P><B>Aim: </B>Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used</P><P><B>Material:</B></P><UL><LI> primers: megaprimer A and B, Jes&amp;Nad 2011-10-17, r_mdn_iGEM
</LI></UL><UL><LI> pARW089, 8.57 ng/ µl
</LI></UL><UL><LI> LongPCR enzyme mix + buffer (fermentas)
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><P><B>Methods:</B></P><P><I>Samples and used primers</I></P><UL><LI>A) pARW089 with both mutations: Megaprimer 1 - A and pr_mdn_iGEM
</LI></UL><UL><LI>B) pARW089 with one mutation: Megaprimer 1 - B and pr_mdn_iGEM
</LI></UL><P><I>PCR mixture</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Volume in µl
</TH></TR><TR><TD> dNTPs (10 mM)</TD><TD> 1
</TD></TR><TR><TD> Long PCR enzyme mix </TD><TD> 5
</TD></TR><TR><TD>Primer forward, megaprimer (32 ng/ µl)(</TD><TD>20
</TD></TR><TR><TD>Primer reverse (10 µM)</TD><TD> 1
</TD></TR><TR><TD>DNA pARW089 (8.57 ng/ul)</TD><TD> 5
</TD></TR><TR><TD>Long PCR enzyme mix </TD><TD>0.3
</TD></TR><TR><TD>H2O</TD><TD> 17.7
</TD></TR></TBODY></TABLE><P><I>PCR conditions (Thermal cycler)</I></P><P>Programm IGMAXI3
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Time</TH><TH> Temp.
</TH></TR><TR><TD>Hold</TD><TD>94°C
</TD></TR><TR><TD>Initial denaturation </TD><TD>5 min</TD><TD>94°C
</TD></TR><TR><TD>Denaturation </TD><TD>30 sec</TD><TD> 94°C
</TD></TR><TR><TD>Annealing</TD><TD> 30 sec</TD><TD> 55°C
</TD></TR><TR><TD>Extension </TD><TD>30 sec</TD><TD> 68°C
</TD></TR><TR><TD>Number of cycles</TD><TD> 15 x</TD></TR><TR><TD>Final extension</TD><TD> 10 min </TD><TD>68°C
</TD></TR></TBODY></TABLE><P><B>Output:</B></P><UL><LI> PCR products, A and B
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Gel electrophoresis and purification
</LI></UL><UL><LI> megaprimer PCR 3
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Sequencing_of_mdnA_library_.28for_phage_display.29">Sequencing of mdnA library (for phage display)</SPAN></H3><P><B>Time:</B> 2011-10-18</P><P><B>Investigators:</B> Sandrina, Nicole</P><P><B>Aim: </B>Producation of mdnA library (for phage display), confirmation of diversity</P><P><B>Material:</B></P><UL><LI> Library clone 1 and 5, San, 2011-10-18
</LI></UL><UL><LI> sf_mdnA_1
</LI></UL><P><B>Methods:</B></P><P><I>Concentration of DNA and used DNA amount</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> DNA concentration in ng/ µl </TH><TH> Volume DNA in µl </TH><TH> Volume H2O in µl
</TH></TR><TR><TD> Library clone 1 </TD><TD> 60.8 </TD><TD> 20 </TD><TD> 0
</TD></TR><TR><TD> Library clone 5 </TD><TD> 98.5 </TD><TD> 14 </TD><TD> 6
</TD></TR></TBODY></TABLE><P>Primer used: sf_mdnA_1
</P><P><B>Output:</B></P><UL><LI> samples sent for sequencing
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Analysis of sequencing
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Over_night_culture_with_created_mdnA_library_for_phage_display">Over night culture with created mdnA library for phage display</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B>Phage display with microviridin library</P><P><B>Method/Materials:</B></P><UL><LI> LB medium
</LI></UL><UL><LI> ampicillin and tetracyclin
</LI></UL><P><B>Further tasks:</B></P><UL><LI> phage production and phage display</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="111th_Labday_2011-10-18">111th Labday 2011-10-18</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Production_of_phages_containing_microviridin_library_on_their_surface_in_XL1-blue_cells">Production of phages containing microviridin library on their surface in XL1-blue cells</SPAN></H3><P><B>Investigators:</B>Sandrina, Sabine</P><P><B>Aim:</B> production of phages carrying modified mdnA on their surfac for screening by phage display</P><P><B>Method/Materials:</B></P><UL><LI><B>first step</B>: amplification of cells containing mutated mdnA:
</LI></UL><UL><LI> 500 ml LB medium will be inoculated with the cells, so that OD600 = 0.1
</LI></UL><UL><LI> add antibiotics tetracyclin and ampicillin to the medium
</LI></UL><UL><LI> cells should be incubated 37 °C till OD 600 = 0.3-0.5 (here: 0.356) is reached
</LI></UL><UL><LI><B>second step</B>: infection with helper phages
</LI></UL><UL><LI> add helper phages 10^11 phages/50 ml (3,5 µl)
</LI></UL><UL><LI> incubate for 10 min at 37°C (without shaking!)
</LI></UL><UL><LI> add 0,5 mM IPTG
</LI></UL><UL><LI> incubate 50 min at 28°C and rpm
</LI></UL><UL><LI> add 70 µg/ml kanamycin and incubate for 5 h at 28°C (...)
</LI></UL><UL><LI><B>third step</B>: phage purification
</LI></UL><UL><LI> centrifuge cell culture at 5000 x g/ 15 min
</LI></UL><UL><LI> fill supernatant in a new 50 ml falcon and centrifuge again (5000 g/15 min)
</LI></UL><UL><LI> 40 ml of the supernatant with 8 ml PEG-NaCl (20% (w/v) PEG-8000, 2,5 M NaCl)
</LI></UL><UL><LI> incubate over night at 4°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> go on with phage purification</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Sidedirected_mutagenesis_of_TEV_protease">Sidedirected mutagenesis of TEV protease</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[6]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P>Aim: Sidedirected mutagenesis of TEV protease</P><P><B>Materials:</B></P><UL><LI> TEV protease vector (P9 by Gunther Stier)
</LI></UL><UL><LI> different Primers
</LI></UL><UL><LI> Phusion polymerase kit by NEB
</LI></UL><P><B>Methode:</B></P><P>PCR</P><UL><LI> Template: 1 µL
</LI></UL><UL><LI> Nucleotides: 1 µL of 10 mM ready to use dNTP mix</LI></UL><UL><LI> 10 µL 5 x Phusion GC Buffer</LI></UL><UL><LI> 2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)</LI></UL><UL><LI> 32,5 µL of pure water</LI></UL><UL><LI>0,5 µL Phusion HF polymerase</LI></UL><P>Program:
</P><UL><LI>Denat: 30 sec 98°C
</LI></UL><UL><LI>30x
</LI></UL><P>Denat: 10 sec 98°C
</P><P>Anneal+Extend: 25 sec 72°C
</P><UL><LI>Final Extend: 5min 72°C
</LI></UL><P>Used Primers:
</P><P>Fragment 1 of TEV protease (T1):
</P><UL><LI> f_TEV_AraFusion_NgoMIV
</LI></UL><UL><LI> r_TEV_ACCAGC
</LI></UL><P>Fragment 2 of TEV protease (T2):
</P><UL><LI> f_TEV_ACCAGC
</LI></UL><UL><LI> r_TEV_iGEM_BamHI
</LI></UL><P><B>Further Tasks:</B></P><P>PCR purification, analytical Gel to check sizes of the amplified fragments, assembly PCR of Fragment </P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="PCR_purification_and_analytical_GE">PCR purification and analytical GE</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[7]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P>Aim: Clean up PCR Fragments T1 and T2 and check sizes of fragments</P><P><B>Materials:</B></P><UL><LI> M&amp;N PCR puri Kit
</LI></UL><P><B>Methode:</B></P><P>Kit was used after manual of M&amp;N</P><P>Sizes of fragments were checked in 1% agarose gel, 5µl were mixed with 1µl 6xLoading Dye.</P><P><B>Output</B></P><P>Fragments with the right size and clean fractions of T1 and T2
</P><P><B>Further Tasks:</B></P><P>Assembly PCR, Digest of Fragments, Ligation and Transformation
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Assembly_PCR_of_TEV_protease">Assembly PCR of TEV protease</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[8]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P>Aim: Sidedirected mutagenesis of TEV protease</P><P><B>Materials:</B></P><UL><LI> Fragments T1 and T2
</LI></UL><UL><LI> different Primers
</LI></UL><UL><LI> Phusion polymerase kit by NEB
</LI></UL><P><B>Methode:</B></P><P>PCR</P><UL><LI> Template: 2µl mix of T1 and T2 in a molar ratio of 1:1
</LI></UL><UL><LI> Nucleotides: 1 µL of 10 mM ready to use dNTP mix</LI></UL><UL><LI> 10 µL 5 x Phusion GC Buffer</LI></UL><UL><LI> 2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)</LI></UL><UL><LI> 31,5 µL of pure water</LI></UL><UL><LI>0,5 µL Phusion HF polymerase</LI></UL><P>Program:
</P><UL><LI>Denat: 30 sec 98°C
</LI></UL><UL><LI>30x
</LI></UL><P>Denat: 10 sec 98°C
</P><P>Anneal+Extend: 35 sec 72°C
</P><UL><LI>Final Extend: 5min 72°C
</LI></UL><P>Used Primers:
</P><UL><LI> f_TEV_AraFusion_NgoMIV
</LI></UL><UL><LI> r_TEV_iGEM_BamHI
</LI></UL><P><B>Further Tasks:</B></P><P>PCR purification, analytical Gel to check sizes of the amplified fragments, assembly PCR of Fragment </P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="PCR_Purification.2C_analytical_GE.2C_Digest_and_Ligation_of_TEV_protease">PCR Purification, analytical GE, Digest and Ligation of TEV protease</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[9]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P>Aim: Getting the TEV into its test vector</P><P><B>Materials:</B></P><UL><LI> mutated TEV protease
</LI></UL><UL><LI> T4 Ligase, different Restriction Enzymes
</LI></UL><P><B>Methode:</B></P><UL><LI>5µl of the PCR product of mutated TEV protease were checked with anayltical GE on a 1% agarosegel
</LI></UL><UL><LI>PCR products got cleaned up by use of the M&amp;N kit for PCR purification
</LI></UL><UL><LI>Mutated TEV and Vector SG6 containing the cleavage site for TEV protease and AraC were digested with different enzymes and cleaned up again.
</LI></UL><UL><LI>TEV with PstI and NgoMIV
</LI></UL><UL><LI>AraC with EcoRI and NgoMIV
</LI></UL><UL><LI>SG6 with PstI and EcoRI
</LI></UL><UL><LI>the concentration of the fragments were estimated with NanoDrop
</LI></UL><UL><LI>Ligation of all Fragment with T4 Ligase by Fermentas (see manual of T$ ligase by Fermentas)
</LI></UL><UL><LI>Ligation Batch is used for Transformation of XL1 blue
</LI></UL><UL><LI>transformed cells were platted on agarplates containing cm
</LI></UL><P><B>Further Tasks:</B></P><P>colony PCR, plasmid prep survival screening and sequencing</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="112th_Labday_2011-10-19">112th Labday 2011-10-19</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Amplification_of_megaprimer_I.2C_using_Phusion_.28Finnzymes.29">Amplification of megaprimer I, using Phusion (Finnzymes)</SPAN></H3><P><B>Time:</B> 2011-10-19</P><P><B>Investigators:</B> Jessica, Nicole</P><P><B>Aim: </B>Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used</P><P><B>Material:</B></P><UL><LI> primers:
</LI></UL><UL><LI> pf_muta_XbaI
</LI></UL><UL><LI> pr_muta_XbaI
</LI></UL><UL><LI> pr_ohne_muta
</LI></UL><UL><LI> pARW089, 8.57 ng/ µl
</LI></UL><UL><LI> Phusion HF Buffer (Finnzymes)
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><P><B>Methods:</B></P><P><I>Samples and used primers</I></P><P>A. pARW089 mutation: pf_muta_XbaI and pr_muta_XbaI </P><P>B. pARW089 without mutations: pf_muta_XbaI and pr_ohne_muta</P><P><I>PCR mixture</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Volume in µl
</TH></TR><TR><TD> dNTPs (10 mM)</TD><TD> 1
</TD></TR><TR><TD> Phusion HF Buffer </TD><TD> 10
</TD></TR><TR><TD>Primer forward, megaprimer (10 µM)</TD><TD> 2.5
</TD></TR><TR><TD>Primer reverse (10 µM)</TD><TD> 2.5
</TD></TR><TR><TD>DNA pARW089 (8.57 ng/ul)</TD><TD> 1
</TD></TR><TR><TD>Phusion polymerase </TD><TD>0.5
</TD></TR><TR><TD>H2O</TD><TD> 32.5
</TD></TR></TBODY></TABLE><P><I>PCR conditions (Thermal cycler)</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Time</TH><TH> Temp.
</TH></TR><TR><TD>Hold</TD><TD>98°C
</TD></TR><TR><TD>Initial denaturation </TD><TD>30 sec</TD><TD>98°C
</TD></TR><TR><TD>Denaturation </TD><TD>10 sec</TD><TD> 98°C
</TD></TR><TR><TD>Annealing</TD><TD> 20 sec</TD><TD> 58°C
</TD></TR><TR><TD>Extension </TD><TD>20 sec</TD><TD> 72°C
</TD></TR><TR><TD>Number of cycles</TD><TD> 30 x</TD></TR><TR><TD>Final extension</TD><TD> 7 min </TD><TD>72°C
</TD></TR></TBODY></TABLE><P><B>Output:</B></P><UL><LI> PCR products, A1, A2, A3 and B1, B2, B3
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Gel electrophoresis and purification
</LI></UL><UL><LI> megaprimer PCR 2
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gelelectrophoresis_and_purification_of_megaprimer_I">Agarose gelelectrophoresis and purification of megaprimer I</SPAN></H3><P><B>Time:</B> 2011-10-19</P><P><B>Investigators:</B> Jessica, Nicole</P><P><B>Aim: </B>Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used</P><P><B>Material:</B></P><UL><LI> Agarose
</LI></UL><UL><LI> 6x Loading dye
</LI></UL><UL><LI> DNA Ladder (NEB)
</LI></UL><UL><LI> Machery Nagel Nucleo Spin Extract II
</LI></UL><P><B>Methods:</B></P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Purification_of_phages_containing_microviridin_library">Purification of phages containing microviridin library</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B> phages carrying modified mdnA on their surface</P><P><B>Method/Materials:</B></P><UL><LI> after over night incubation:
</LI></UL><UL><LI> centrifuge precipitaed phages: 5000 x g/ 45 min
</LI></UL><UL><LI> discard supernatant
</LI></UL><UL><LI> centrifuge again, 5000 x g/ 5 min
</LI></UL><UL><LI> remove supernatant carefully
</LI></UL><UL><LI> take pellet in 1 ml TBS
</LI></UL><UL><LI> move in 1.5 ml Eppi
</LI></UL><UL><LI> centrifuge again: 17000 x g/10 min
</LI></UL><UL><LI> supernatant in a new Eppi with 200 µl PEG-NaCl (mix!)
</LI></UL><UL><LI> incubate on ice for 60 min
</LI></UL><UL><LI> centrifuge precipitated phages: 17.000 x g/ 10 min (4°C)
</LI></UL><UL><LI> resuspend pellet in 300 µl TBS
</LI></UL><UL><LI> centrifuge:17 000 x g/ 10 min (4°C)
</LI></UL><UL><LI> supernatant in a fresh eppi= purified phages
</LI></UL><P><B>Further tasks:</B></P><UL><LI> repeat procedure with ER2738 cells, because no phage pellet could be obserevd after precipitation
</LI></UL><UL><LI> phage display</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Colony_PCR_of_TEV_in_vivo_selcetion">Colony PCR of TEV in vivo selcetion</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[10]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P>Aim: Getting the positiv clones for TEV in vivo selction</P><P><B>Materials:</B></P><UL><LI> XL1 clonies containing TorA-BlaFl construct with TEV cleavage site and TEV protease
</LI></UL><UL><LI> Phusion Polymerase set
</LI></UL><UL><LI> different primers
</LI></UL><P><B>Methode:</B></P><P>PCR: 3min intial Dneaturation 98°C, 25 cycles: 10 sec denaturation 98°C, 35 sec annealing+elongation at 72°C. Final elongation for 5min at 72°C.</P><P>used primers:
</P><UL><LI>f_TEV_iGEM
</LI></UL><UL><LI>r_TEV_iGEM_BamHI
</LI></UL><P>Colonies were picked with a 10µl tip, some bacterias were used for PCR reaction, the rest were used as pre culture of colnes. After Colony PCR and check via GE in 1% agarose gel positive clones were used for overnight cultures.</P><P><B>Output:</B></P><P>2 positive clones SG25 and SG26</P><P><B>Further Tasks:</B></P><P>plasmid prep, survival screening and sequencing</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Making_TEV_and_HRV14_3C_Biobricks">Making TEV and HRV14_3C Biobricks</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[11]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Stefan</P><P>Aim: Getting the positiv clones for TEV in vivo selction</P><P><B>Materials:</B></P><UL><LI> Phusion Polymerase set
</LI></UL><UL><LI> different primers
</LI></UL><UL><LI> different restriction enzymes
</LI></UL><UL><LI> pSB1C3 vector
</LI></UL><P><B>Methode:</B></P><P>PCR: 3min intial Dneaturation 98°C, 25 cycles: 10 sec denaturation 98°C, 35 sec annealing+elongation at 72°C. Final elongation for 5min at 72°C.</P><P>used primers for TEV:
</P><UL><LI>f_TEV_iGEM
</LI></UL><UL><LI>r_TEV_iGEM_BamHI
</LI></UL><P>used primers for HRV14_3C:
</P><UL><LI>f_HRV14_3C_iGEM
</LI></UL><UL><LI>r_HRV14_3C_iGEM_BamHI
</LI></UL><P>Mutated TEV and HRV14_3C were amplified out of vector containing mutated proteased (SG25 for TEV, SG24 for HRV14_3C), cleaned up with PCR purification kit form M&amp;N, digested with EcoRI-HF, PstI-HF. Also pSB1C3 vector was digested with EcoRI and PstI. Both, proteases and vector were cleaned up via GE on 1% agarose gel followed by PCR clean up with M&amp;N kit. The different proteases were ligated with pSB1C3 using T4 ligase purchased by Fermentas and the ligation products were transformed into XL1 blue cells using standard heat shock protocol.
</P><P><B>Further Tasks:</B></P><P>plasmid prep and sequencing</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="113th_Labday_2011-10-20">113th Labday 2011-10-20</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="ELISA_with_purified_phages_carrying_unmodified_mdnA_on_their_surface">ELISA with purified phages carrying unmodified mdnA on their surface</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Time:</B> 2011-09-16, 12:00-21:00</P><P><B>Aim:</B> recontrol if mdnA-myc-geneIII will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI> ELISA plate coated with 5 µg/ml anti-c-myc antibody in phosohate buffer (pH 7,4), volume 100 µl/ well
</LI></UL><UL><LI> incubate for 4 h at room temperature
</LI></UL><UL><LI> blocking in 2 % milk powder in TBS (300 µl/ well)
</LI></UL><UL><LI> incubate for 2 h at room temperature
</LI></UL><UL><LI> add 100 µl phages, produced in XL1-blu cells diluted 1:1 in TBS-T (0,005%)
</LI></UL><UL><LI> incubate shaking for 60 min at room temperature
</LI></UL><UL><LI> wash 5 x with TBS-T (0,05%)
</LI></UL><UL><LI> add anti-gene-8-antibody (HRP coupled)
</LI></UL><UL><LI> incubate shaking for 60 min
</LI></UL><UL><LI> wash 5 x with TBS-T
</LI></UL><UL><LI> put substrate on the samples
</LI></UL><UL><LI> measure in plate photometer
</LI></UL><UL><LI> reference: helper phages
</LI></UL><P><B>Resluts:</B> positive signal at wells with mdnA phages
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="SDS-_PAGE_with_purified_phages_carrying_mdnA_on_their_surface">SDS- PAGE with purified phages carrying mdnA on their surface</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> preparing geneIII-myc-mdnA-protein for western blot</P><P><B>Method/Materials:</B></P><UL><LI> seperating gel:
</LI></UL><UL><LI> acryl amide: 2,5 ml
</LI></UL><UL><LI> seperating gel buffer 4x: 1,5 ml
</LI></UL><UL><LI> water: 1,94
</LI></UL><UL><LI> APS: 60 µl
</LI></UL><UL><LI> stacking gel:
</LI></UL><UL><LI> acryl amide: 0,28 ml
</LI></UL><UL><LI> stacking gel buffer: 0,5 ml
</LI></UL><UL><LI> water: 1,22 ml
</LI></UL><UL><LI> APS: 10 µl
</LI></UL><UL><LI> loading dye: RotiLoad1
</LI></UL><UL><LI> incubate samples at 95°C for 20 min
</LI></UL><UL><LI> run at 50 V for half an hour, then switch to 20 mA
</LI></UL><P><B>Further tasks:</B></P><UL><LI> western blot </LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Western_Blot">Western Blot</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA is expressed on the phages produced in XL1-blue cells</P><P><B>Method/Materials:</B></P><UL><LI> membrane impregnated in methanol
</LI></UL><UL><LI> whatman paper
</LI></UL><UL><LI> blotting buffer
</LI></UL><UL><LI> blotting chamber
</LI></UL><UL><LI> 1 h at 200 mA
</LI></UL><UL><LI> blocking over night in TBS with 5% milk powder
</LI></UL><P><B>Further tasks:</B></P><UL><LI> antibody detection </LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Colony_PCR.2CMini-prep_and_Survival_screening">Colony PCR,Mini-prep and Survival screening</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[12]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P>Aim: Getting the TEV into its test vector</P><P><B>Materials:</B></P><UL><LI>clones containing TEV and TorA-bla construct with TEV protease
</LI></UL><UL><LI>colonies of TEV and HRV14_3C protease
</LI></UL><UL><LI>Phusion polymerase Kit
</LI></UL><UL><LI>different agar plates
</LI></UL><P><B>Methode:</B></P><P>standard colony PCR protocol were used for TEV and HRV14_3C with same used for ampilication, followed by analytical GE with 1% agarose gel. all positiv clones were used for over night culture for plasmid prep. </P><P>The two positive clones SG25 and SG26 were grown over night in LB media, diluted next day and grown again to an OD of 0.3 to get them into exponential growth phase and diluted again to an OD of 0.002. 100µl of the dilution of each clone gots plated on 2 of each different set of agar plates (see below).</P><P><B> used Plates: </B></P><UL><LI>4x Cm (25 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM IPTG
</LI></UL><UL><LI>4x Cm + 2% Ara
</LI></UL><UL><LI>4x Cm + 2% Ara +1mM IPTG
</LI></UL><UL><LI>4x Cm + Amp (100 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM ITPG + Amp (25 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM ITPG + Amp (50 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM ITPG + Amp (75 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM ITPG + Amp (100 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM ITPG + Amp (150 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM ITPG + Amp (200 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM ITPG + Amp (300 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM ITPG + Amp (400 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM ITPG + Amp (600 µg/ml)
</LI></UL><UL><LI>4x Cm + 1mM ITPG + Amp (800 µg/ml)
</LI></UL><UL><LI>4x Cm + 2% Ara + 1mM IPTG + Amp (25 µg/ml)
</LI></UL><UL><LI>4x Cm + 2% Ara + 1mM IPTG + Amp (50 µg/ml)
</LI></UL><UL><LI>4x Cm + 2% Ara + 1mM IPTG + Amp (75 µg/ml)
</LI></UL><UL><LI>4x Cm + 2% Ara + 1mM IPTG + Amp (100 µg/ml)
</LI></UL><UL><LI>4x Cm + 2% Ara + 1mM IPTG + Amp (150 µg/ml)
</LI></UL><UL><LI>4x Cm + 2% Ara + 1mM IPTG + Amp (200 µg/ml)
</LI></UL><UL><LI>4x Cm + 2% Ara + 1mM IPTG + Amp (300 µg/ml)
</LI></UL><UL><LI>4x Cm + 2% Ara + 1mM IPTG + Amp (400 µg/ml)
</LI></UL><UL><LI>4x Cm + 2% Ara + 1mM IPTG + Amp (600 µg/ml)
</LI></UL><UL><LI>4x Cm + 2% Ara + 1mM IPTG + Amp (800 µg/ml)
</LI></UL><P>Each clone was incubated at 30 and 37°C.
</P><P><B>Further Tasks:</B></P><P>See results of in vivo selection system, plasmid prep of TEV and HRV14_3C Biobricks and sequencing.
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Amplification_of_megaprimer_II.2C_using_Phusion_.28Finnzymes.29">Amplification of megaprimer II, using Phusion (Finnzymes)</SPAN></H3><P><B>Time:</B> 2011-10-20</P><P><B>Investigators:</B> Jessica, Nicole</P><P><B>Aim: </B>Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used</P><P><B>Material:</B></P><UL><LI> primers:
</LI></UL><DL><DD>a) Megaprimer A and r_mdn_iGEM
</DD></DL><DL><DD>b) Megaprimer B and r_mdn_iGEM
</DD></DL><UL><LI> pARW089, 8.57 ng/ µl
</LI></UL><UL><LI> Phusion HF Buffer (Finnzymes)
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><P><B>Methods:</B></P><P><I>PCR mixture</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> volume in µl
</TH></TR><TR><TD> dNTPs (10 mM)</TD><TD> 1
</TD></TR><TR><TD> Phusion HF Buffer </TD><TD> 10
</TD></TR><TR><TD>Primer reverse r_mdnABCDE_iGEM (10 µM)</TD><TD> 2.5
</TD></TR><TR><TD>DNA pARW089 (8.57 ng/ul)</TD><TD> 1
</TD></TR><TR><TD>Phusion polymerase </TD><TD>0.5
</TD></TR></TBODY></TABLE><P>varied volume of megaprimer I
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> A1a (0.2 µM)</TH><TH> A1b (0.5 µM)</TH><TH> B1a (0.2 µM)</TH><TH>B1a (0.2 µM)</TH><TH>A2a (0.2 µM)</TH><TH>A2b (0.5 µM)</TH><TH>B2a (0.2 µM)</TH><TH>B2b (0.5 µM)
</TH></TR><TR><TD> Primer forward, Megaprimer I</TD><TD> 11.6</TD><TD>29.0</TD><TD>9.0</TD><TD>22.0</TD><TD>7.2</TD><TD>18</TD><TD>7.4</TD><TD>18.5
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 23.4</TD><TD>6.0</TD><TD>26.0</TD><TD>13.0</TD><TD>27.8</TD><TD>17</TD><TD>27.6</TD><TD>16.5
</TD></TR></TBODY></TABLE><P><I>PCR conditions (Thermal cycler)</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Time</TH><TH> Temp.
</TH></TR><TR><TD>Hold</TD><TD>98°C
</TD></TR><TR><TD>Initial denaturation </TD><TD>30 sec</TD><TD>98°C
</TD></TR><TR><TD>Denaturation </TD><TD>30 sec</TD><TD> 98°C
</TD></TR><TR><TD>Annealing</TD><TD> 20 sec</TD><TD> 72°C
</TD></TR><TR><TD>Extension </TD><TD>20 sec</TD><TD> 72°C
</TD></TR><TR><TD>Number of cycles</TD><TD> 30 x</TD></TR><TR><TD>Final extension</TD><TD> 7 min </TD><TD>72°C
</TD></TR></TBODY></TABLE><UL><LI>Gelelectrophoresis and Purification
</LI></UL><P><B>Output:</B></P><UL><LI> megaprimer A2 and B2
</LI></UL><P><B>Further tasks:</B></P><UL><LI> amplification of whole cluster
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Amplification_of_mdn-cluster">Amplification of mdn-cluster</SPAN></H3><P><B>Time:</B> 2011-10-20</P><P><B>Investigators:</B> Jessica, Nicole</P><P><B>Aim: </B>Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used</P><P><B>Material:</B></P><UL><LI> primers:
</LI></UL><DL><DD>a) Megaprimer A2 and Pf_mdnABCDET7
</DD></DL><DL><DD>b) Megaprimer B2 and Pf_mdnABCDET7
</DD></DL><UL><LI> pARW089, 8.57 ng/ µl
</LI></UL><UL><LI> Long Enzyme Mix (Fermentas)
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><P><B>Methods:</B></P><P><I>PCR mixture</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> volume in µl
</TH></TR><TR><TD> dNTPs (10 mM)</TD><TD> 1
</TD></TR><TR><TD> Long enzyme mix (Fermentas) Buffer </TD><TD> 5
</TD></TR><TR><TD>Primer forward Pf_mdnABCDET7 (10 µM)</TD><TD> 1
</TD></TR><TR><TD>DNA pARW089 (8.57 ng/ul)</TD><TD> 1.2
</TD></TR><TR><TD>Long PCR Polymerase </TD><TD>0.3
</TD></TR></TBODY></TABLE><P>varied volume of megaprimer II
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> 50 pmol</TH><TH> 125 ng</TH><TH> 250 ng
</TH></TR><TR><TD> Primer forward, Megaprimer II</TD><TD> 25</TD><TD>0.5</TD><TD>1.0
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 16.5</TD><TD>41</TD><TD>40.5
</TD></TR></TBODY></TABLE><P><I>PCR conditions (Thermal cycler)</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Time</TH><TH> Temp.
</TH></TR><TR><TD>Hold</TD><TD>98°C
</TD></TR><TR><TD>Initial denaturation </TD><TD>3 min</TD><TD>94°C
</TD></TR><TR><TD>Denaturation </TD><TD>20 sec</TD><TD> 94°C
</TD></TR><TR><TD>Annealing</TD><TD> 30 sec</TD><TD> 68°C
</TD></TR><TR><TD>Extension </TD><TD>7 min</TD><TD> 68°C
</TD></TR><TR><TD>Number of cycles</TD><TD> 7 x</TD></TR><TR><TD>Denaturation </TD><TD>20 sec</TD><TD> 94°C
</TD></TR><TR><TD>Annealing</TD><TD> 30 sec</TD><TD> 68°C
</TD></TR><TR><TD>Extension </TD><TD>7 min + 3s/cycle</TD><TD> 68°C
</TD></TR><TR><TD>Number of cycles</TD><TD> 10 x</TD></TR><TR><TD>Final extension</TD><TD> 7 min </TD><TD>68°C
</TD></TR></TBODY></TABLE><UL><LI>Gelelectrophoresis and Purification
</LI></UL><P><B>Output:</B></P><UL><LI> mdn-cluster A and B
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Digest and ligation with pSB1C3
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="114th_Labday_2011-10-21">114th Labday 2011-10-21</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Antibody_detection_with_blotted_membranes">Antibody detection with blotted membranes</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B> control if geneIII-myc-mdnA is presented on the phages</P><P><B>Method/Materials:</B></P><UL><LI> incubate blocked membrane for 1 h with primary antibody (anti -c-myc- antibody) in TBS with 5% milk powder
</LI></UL><UL><LI> wash 3x 10 min with TBS buffer
</LI></UL><UL><LI> incubate for 1 h with secondary antibody
</LI></UL><UL><LI> wash 3x 10 min with TBS buffer
</LI></UL><UL><LI> develop membranes with ECL- Kit</LI></UL><P><B>Results:</B></P><UL><LI> signals were seen at positive controls and a very weak signal at mdnA-myc-geneIII lane
</LI></UL><P><B>Further tasks:</B></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Results_of_Survival_screening_at_37.C2.B0C_and_mini_prep_of_TEV_and_HRV14_3C_BioBricks">Results of Survival screening at 37°C and mini prep of TEV and HRV14_3C BioBricks</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[13]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P>Aim: Getting the TEV and HRV14_3C Biobrick, see if invivo selection at 37°C works</P><P><B>Materials:</B></P><UL><LI> plasmid prep Kit of M&amp;N
</LI></UL><P><B>Methode:</B></P><UL><LI>plasmids were isolated as described in the manual of the Kit
</LI></UL><UL><LI>looking for clones on plates
</LI></UL><P><B>Output</B></P><UL><LI>plasmids for sequencing P1,P2,P4,P7+T1,T2,T6
</LI></UL><UL><LI>no results for in vivo selection
</LI></UL><P><B>Further Tasks:</B></P><P>sequencing</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_mdn-cluster_and_pSB1C3_and_ligation">Digest of mdn-cluster and pSB1C3 and ligation</SPAN></H3><P><B>Time:</B> 2011-10-21</P><P><B>Investigators:</B> Jessica, Nicole</P><P><B>Material:</B></P><UL><LI> EcoRI-HF, PstI-HF
</LI></UL><UL><LI> pSB1C3 (#3)
</LI></UL><UL><LI> PCR products: mdn-cluster A and B
</LI></UL><P><B>Methods:</B></P><P>Digest pSB1C3
</P><UL><LI>4 µl NEB bufer 4
</LI></UL><UL><LI>15 µl pSB1C3 (250 ng/µl)
</LI></UL><UL><LI>2 µl EcoRI-HF
</LI></UL><UL><LI>2 µl PstI-HF
</LI></UL><UL><LI>17 µl water
</LI></UL><P>Digest mdn-cluster
</P><UL><LI>5 µl NEB buffer 4
</LI></UL><UL><LI>33 µl PCR product
</LI></UL><UL><LI>2 µl EcoRI-HF
</LI></UL><UL><LI>2 µl PstI-HF
</LI></UL><UL><LI>8 µl water
</LI></UL><P>Gel purification of pSB1C3</P><P>PCR purification of mdn-cluster</P><P>Ligation
</P><UL><LI>1 µl T4 ligase buffer
</LI></UL><UL><LI>1 µl T4 ligase
</LI></UL><UL><LI>1 µl pSB1C3
</LI></UL><UL><LI>7 µl mdn-cluster
</LI></UL><P><B>Output:</B></P><UL><LI> ligations: pSB1C3+mdn-cluster A, pSB1C3+mdn-cluster B
</LI></UL><P><B>Further tasks:</B></P><UL><LI> transformation
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="115th_Labday_2011-10-22">115th Labday 2011-10-22</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Results_of_survival_screening_at_30.C2.B0C">Results of survival screening at 30°C</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[14]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P>Aim: see if in vivo selection could work</P><P><B>Materials:</B></P><P>none
</P><P><B>Method:</B></P><P>counting colonies on plates</P><P><B>Results</B></P><P>For Both; Sg25 and SG26, cells survival up to 800 µg/ml ampicillin without active protease. Only for SG26 we were able to see cell death, if protease is activated, cells die early --&gt; no clones on plates with 50µg/ml ampicillin.</P><P>over night culture of SG26 carrying XL1 blue cells.
</P><P><B>Further Tasks:</B></P><P>making comptent cell out of SG26
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_with_pSB1C3.2Bmdn-cluster">Transformation with pSB1C3+mdn-cluster</SPAN></H3><P><B>Investigators:</B> Jessica</P><P><B>Time:</B> 2011-10-22</P><P><B>Method:</B></P><UL><LI> addition of 4 µl ligation reaction to XL1-blue cells</LI></UL><UL><LI> incubation 25 min on ice,</LI></UL><UL><LI> heat shock 45 sec at 42°C,</LI></UL><UL><LI> incubation 2 min on ice,</LI></UL><UL><LI> addition of 750 µl LB medium,</LI></UL><UL><LI> incubation 60 min at 37 °C and 750 rpm</LI></UL><UL><LI> plating on agar plates containing 100 µg/ml tetracyclin and 100 µg/µl ampicillin</LI></UL><UL><LI> storage over night at 37°C
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="116th_Labday_2011-10-23">116th Labday 2011-10-23</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="making_competent_cells_and_tranformation_with_Libary_of_SG26_carrying_cells"> making competent cells and tranformation with Libary of SG26 carrying cells</SPAN></H3><P><B>Investigators:</B> Stefan, Paul, Sascha</P><P><B>Aim:</B> transformig XL1 blue cells SG26 with mdnA libary, </P><P><B>Methode:</B></P><P>Standard protocol for chemical competent cells followed by standard heat shock protocol for transformation. Transformed cells were plated on 10 agar plates containing cm and kan as antibiotics.
</P><P><B>Results</B>:
</P><P>Libary transformed into cells with the TEV in vivo selection system.
</P><P><B> further Tasks:</B></P><P>Test cells for surviving at higher ampicillin concentration than 25 µg/ml.
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Production_of_phages_containing_microviridin_library_on_their_surface_in_ER2738_cells">Production of phages containing microviridin library on their surface in ER2738 cells</SPAN></H3><P><B>Investigators:</B>Sandrina, Sabine</P><P><B>Aim:</B> production of phages carrying modified mdnA on their surfac for screening by phage display</P><P><B>Method/Materials:</B></P><UL><LI><B>first step</B>: amplification of cells containing mutated mdnA:
</LI></UL><UL><LI> 500 ml LB medium will be inoculated with the cells, so that OD600 = 0.1
</LI></UL><UL><LI> add antibiotics tetracyclin and ampicillin to the medium
</LI></UL><UL><LI> cells should be incubated 37 °C till OD 600 = 0.3-0.5 (here: 0.356) is reached
</LI></UL><UL><LI><B>second step</B>: infection with helper phages
</LI></UL><UL><LI> add helper phages 10^11 phages/50 ml (3,5 µl)
</LI></UL><UL><LI> incubate for 10 min at 37°C (without shaking!)
</LI></UL><UL><LI> add 0,5 mM IPTG
</LI></UL><UL><LI> incubate 50 min at 28°C and rpm
</LI></UL><UL><LI> add 70 µg/ml kanamycin and incubate for 5 h at 28°C (...)
</LI></UL><UL><LI><B>third step</B>: phage purification
</LI></UL><UL><LI> centrifuge cell culture at 5000 x g/ 15 min
</LI></UL><UL><LI> fill supernatant in a new 50 ml falcon and centrifuge again (5000 g/15 min)
</LI></UL><UL><LI> 40 ml of the supernatant with 8 ml PEG-NaCl (20% (w/v) PEG-8000, 2,5 M NaCl)
</LI></UL><UL><LI> incubate over night at 4°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> go on with phage purification</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="117th_Labday_2011-10-24">117th Labday 2011-10-24</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Testing_mdnA_libary_with_SG26_carrying_cells_for_TEV_in_vivo_selection"> Testing mdnA libary with SG26 carrying cells for TEV in vivo selection</SPAN></H3><P><B>Investigators:</B> Stefan, Paul, Sascha</P><P><B>Aim:</B> see if there is any mdnA which leads to cell survival </P><P><B>Methode:</B></P><P>library was harvest from agar plates, put into a preculture containing cm and kan and grown for 3 h at 37°C, deluted to an OD of 0.02 and plated on different agar plates.
</P><UL><LI>1x Cm (25 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM IPTG
</LI></UL><UL><LI>1x Cm + 2% Ara
</LI></UL><UL><LI>1x Cm + 2% Ara +1mM IPTG
</LI></UL><UL><LI>1x Cm + Amp (100 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (20 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (30 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (40 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (50 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (100 µg/ml)
</LI></UL><UL><LI>1x Cm + 1mM ITPG + Amp (400 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (20 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (30 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (40 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (50 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (100 µg/ml)
</LI></UL><UL><LI>1x Cm + 2% Ara + 1mM IPTG + Amp (400 µg/ml)
</LI></UL><P>all plates were incubated at 30°c for two days
</P><P><B>Results</B>:
</P><P>Libary transformed into cells with the TEV in vivo selection system.
</P><P><B> further Tasks:</B></P><P>Test cells for surviving at higher ampicillin concentration than 25 µg/ml.
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Purification_of_phages_containing_microviridin_library_2">Purification of phages containing microviridin library</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B> phages carrying modified mdnA on their surface</P><P><B>Method/Materials:</B></P><UL><LI> after over night incubation:
</LI></UL><UL><LI> centrifuge precipitaed phages: 5000 x g/ 45 min
</LI></UL><UL><LI> discard supernatant
</LI></UL><UL><LI> centrifuge again, 5000 x g/ 5 min
</LI></UL><UL><LI> remove supernatant carefully
</LI></UL><UL><LI> take pellet in 1 ml TBS
</LI></UL><UL><LI> move in 1.5 ml Eppi
</LI></UL><UL><LI> centrifuge again: 17000 x g/10 min
</LI></UL><UL><LI> supernatant in a new Eppi with 200 µl PEG-NaCl (mix!)
</LI></UL><UL><LI> incubate on ice for 60 min
</LI></UL><UL><LI> centrifuge precipitated phages: 17.000 x g/ 10 min (4°C)
</LI></UL><UL><LI> resuspend pellet in 300 µl TBS
</LI></UL><UL><LI> centrifuge:17 000 x g/ 10 min (4°C)
</LI></UL><UL><LI> supernatant in a fresh eppi= purified phages
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="118th_Labday_2011-10-25">118th Labday 2011-10-25</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Control_of_sequencing_results_of_sequenced_clones_from_created_library_for_phage_display">Control of sequencing results of sequenced clones from created library for phage display</SPAN></H3><P><B>Investigators:</B> Sandrina, Jessica</P><P><B>Aim:</B>check if mdnA library, which could be screened by phage display was produced</P><P><B>Results:</B></P><UL><LI> all clones contained the wildtype mdnA-myc-geneIII-gene
</LI></UL><P><B>Further tasks:</B></P><UL><LI> repetition of cloning the mdnA library
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Colony_PCR_of_pSB1C3.2Bmdn-cluster">Colony PCR of pSB1C3+mdn-cluster</SPAN></H3><P><B>Investigators:</B> Jessica</P><P><B>Time:</B> 2011-10-25</P><P><B>Method:</B></P><P><I>PCR mixture</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> volume in µl
</TH></TR><TR><TD> dNTPs (10 mM)</TD><TD> 1
</TD></TR><TR><TD> Buffer </TD><TD> 4
</TD></TR><TR><TD>Primer reverse (10 µM)</TD><TD>1
</TD></TR><TR><TD>Primer forward (10 µM)</TD><TD>1
</TD></TR><TR><TD>DNA pARW089 (8.57 ng/ul)</TD><TD> 1
</TD></TR><TR><TD>polymerase </TD><TD>0.2
</TD></TR><TR><TD>water</TD><TD>7.8
</TD></TR></TBODY></TABLE><P>Gel electrophoresis</P><P>Overnight culture of pSB1C3+mdn-cluster A clone 6</P><P><B>Further tasks:</B></P><UL><LI> miniprep and sequencing
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="119th_Labday_2011-10-26">119th Labday 2011-10-26</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Phage_Display_with_wildtype_mdnA_and_different_proteases">Phage Display with wildtype mdnA and different proteases</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine, Jessica </P><P><B>Time:</B> 2011-09-14, 9:00-22:00</P><P><B>Aim:</B>check if unmodified mdnA on the surface of the phages binds to an immobilized protease. interactions of microviridin L with the proteases trypsin, chymotrypsin and elastase were already detected</P><P><B>Material/Method:</B></P><UL><LI> in ELISA plate:
</LI></UL><UL><LI> coat wells with proteases trypsin, elastase, chymotrypsin, papain, proteinase K, pepsin and mycolysin in 0,2 M PBS with ph 5.0 , for 2 h
</LI></UL><UL><LI> blocking in TBS with 3% BSA for 2 h
</LI></UL><UL><LI> wash 6x with TBS-T (0,005%)
</LI></UL><UL><LI> panning with phages 1:1 helper phages and phages of interest (presenting mdnA-geneIII fusion protein on their surface, 1 h)
</LI></UL><UL><LI> wash 6x with TBS-T
</LI></UL><UL><LI> elute bound phages with 0,2 M Glycin-HCl, 10 min
</LI></UL><UL><LI> neutralize with 1 M Tris (pH 7-8)
</LI></UL><UL><LI> mix eluted phages with preparatory culture of ER2738 cells
</LI></UL><UL><LI> incubate 10 min at 37°C without shaking
</LI></UL><UL><LI> incubate 50 min at 37°C shaking
</LI></UL><UL><LI> plate on amp plates and on kan plates
</LI></UL><P><B>Further tasks:</B></P><P>check cell growing
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_pARWIII_and_ligation_with_library_2">Digest of pARWIII and ligation with library</SPAN></H3><P><B>Time:</B> 2011-10-26</P><P><B>Investigators:</B> Sandrina, Jessica, Sabine</P><P><B>Material:</B></P><P><B>Method:</B></P><P>Digest of pARWIII
</P><P>Gel electrophoresis
</P><P>Gel purification
</P><P>Digest of geneIII
</P><P>Purification
</P><P>Ligation
</P><UL><LI> 2 µl T4 ligase buffer
</LI></UL><UL><LI> 1 µl T4 ligase
</LI></UL><UL><LI> 6 µl pARW089
</LI></UL><UL><LI> 10 µl geneIII
</LI></UL><UL><LI> 1 µl mdnA library
</LI></UL><P><B>Output:</B></P><UL><LI> 3 ligation products stored @ -20°C
</LI></UL><P><B>Further task:</B></P><UL><LI> transformation of one ligation product and control
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_triple_ligated_mdnA_library.2C_gene_III_and_pARWIII_2">Transformation of triple ligated mdnA library, gene III and pARWIII</SPAN></H3><P><B>Time:</B> 2011-10-26, 19:45-22:00</P><P><B>Investigators:</B> Sandrina, Jessica, Sabine, Nadja </P><P><B>Aim:</B>Phage display with created mdnA library</P><P><B>Materials:</B></P><UL><LI> XL1-Blue chemocompetent cells
</LI></UL><UL><LI> ligation product, named geneIII mdnA lib pARWIII, 2011-10-11
</LI></UL><UL><LI> transformation control, named H2O, 2011-10-11
</LI></UL><UL><LI> add 2 µl PCR product to competent cells (for ligation product 5 samples overall)
</LI></UL><UL><LI> incubate 25 min one ice
</LI></UL><UL><LI> heat pulse: 45 sec at 42°C
</LI></UL><UL><LI> 2 min on ice
</LI></UL><UL><LI> add 750 µl LB medium to each sample
</LI></UL><UL><LI> incubate 45 min at 37°C and 850rpm
</LI></UL><UL><LI> cenrifugation and resuspend in 200 µl LB medium
</LI></UL><UL><LI> plate 200 µl on plates with ampicillin
</LI></UL><UL><LI> incubate the transformation plates at 37°C for approx. 16 hours
</LI></UL><P><B>Further tasks:</B></P><UL><LI> preparation of overnight cultures of several clones
</LI></UL><UL><LI> Miniprep
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_pSB1C3.2Bmdn-cluster_A_clone_6_and_sending_for_sequencing">Miniprep of pSB1C3+mdn-cluster A clone 6 and sending for sequencing</SPAN></H3><P><B>Time:</B> 2011-10-26</P><P><B>Investigators:</B> Sandrina, Jessica, Sabine, Nadja </P><P><B>Materials/Method:</B></P><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><DL><DD><UL><LI> Protocol for high-copy plasmids
</LI></UL></DD></DL><DL><DD><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL></DD></DL><P><B>Results:</B> (from 2011-10-28)</P><UL><LI> only part of the cluster could be found in pSB1C3
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="SDS-_PAGE_with_lysated_cells_which_produced_the_mdnA-myc-geneIII_protein">SDS- PAGE with lysated cells which produced the mdnA-myc-geneIII protein</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> preparing the cells which expressed geneIII-myc-mdnA-protein for detecting the expression of this protein by western blot</P><P><B>Method/Materials:</B></P><UL><LI> seperating gel:
</LI></UL><UL><LI> acryl amide: 2,5 ml
</LI></UL><UL><LI> seperating gel buffer 4x: 1,5 ml
</LI></UL><UL><LI> water: 1,94
</LI></UL><UL><LI> APS: 60 µl
</LI></UL><UL><LI> stacking gel:
</LI></UL><UL><LI> acryl amide: 0,28 ml
</LI></UL><UL><LI> stacking gel buffer: 0,5 ml
</LI></UL><UL><LI> water: 1,22 ml
</LI></UL><UL><LI> APS: 10 µl
</LI></UL><UL><LI> loading dye: RotiLoad1
</LI></UL><UL><LI> incubate samples at 95°C for 20 min
</LI></UL><UL><LI> run at 50 V for half an hour, then switch to 20 mA
</LI></UL><P><B>Further tasks:</B></P><UL><LI> western blot </LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Western_Blot_2">Western Blot</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA is expressed ER2738 cells</P><P><B>Method/Materials:</B></P><UL><LI> membrane impregnated in methanol
</LI></UL><UL><LI> whatman paper
</LI></UL><UL><LI> blotting buffer
</LI></UL><UL><LI> blotting chamber
</LI></UL><UL><LI> 1 h at 200 mA
</LI></UL><UL><LI> blocking over night in TBS with 5% milk powder
</LI></UL><P><B>Further tasks:</B></P><UL><LI> antibody detection </LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="120th_Labday_2011-10-27">120th Labday 2011-10-27</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Antibody_detection_with_blotted_membranes_2">Antibody detection with blotted membranes</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B> control if geneIII-myc-mdnA is expressed in the cells</P><P><B>Method/Materials:</B></P><UL><LI> incubate blocked membrane for 1 h with primary antibody (anti -c-myc- antibody) in TBS with 5% milk powder
</LI></UL><UL><LI> wash 3x 10 min with TBS buffer
</LI></UL><UL><LI> incubate for 1 h with secondary antibody
</LI></UL><UL><LI> wash 3x 10 min with TBS buffer
</LI></UL><UL><LI> develop membranes with ECL- Kit</LI></UL><P><B>Results:</B></P><UL><LI> signals were seen at lanes loaded with cell lysate --&gt; it can be concluded that mdnA-myc-gneeIII protein is expressed in the cells
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="control_of_performed_phage_display">control of performed phage display</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B> control if phage display in general is possible with mdnA-myc-geneIII </P><P><B>Method/Materials:</B></P><UL><LI> count growed clones </LI></UL><P><B>Results:</B></P><UL><LI> phages carriying mdnA on their surface could be enriched, when chymotrypsin and elastase acted as targets
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/October">http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/October</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Labjournal/October" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Labjournal/October" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/October&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/October&amp;oldid=254214" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>