<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_BioBricks"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/BioBricks">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/BioBricks&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/BioBricks&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/BioBricks&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/BioBricks" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/BioBricks</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team">Team</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Members">Members</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Project</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Summary</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Highlights">Highlights</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Microviridin">     Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Phage_Display">     Phage Display</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#In_Vivo_Selection">     In Vivo Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Modeling">     Modeling</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Microviridin">Details Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">Details Phage</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">Details Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Modeling">Details Modeling</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">BioBricks</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/BioBricks">Details BioBricks</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Methods">Methods</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Meetings">Meetings</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics">Safety &amp; Ethics</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Assesment">Safety Assessment</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Questions">Safety Questions</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Ethics_Seminar">Ethics Seminar</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Survey">Survey</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Potsdam_Bioware_at_the_German_parliament">Parliament Visit</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Statement_of_the_German_government">     Statement</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Outreach">Outreach</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software">Software</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#BioLog_App">BioLog App</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#Download">Download</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Sponsors">Sponsors</A></DT></DL></LI></UL></DIV><DIV class="pageContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#BioBricks"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">BioBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#BioBrick_mdnED"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">BioBrick mdnED</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#BioBrick_mdnA"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">BioBrick mdnA</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#BioBrick_mdnB"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">BioBrick mdnB</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#BioBrick_mdnC"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">BioBrick mdnC</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#BioBrick_mdnD"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">BioBrick mdnD</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#BioBrick_mdnE"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">BioBrick mdnE</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#BioBrick_mdnA-myc-gene_III"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">BioBrick mdnA-myc-gene III</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#BioBrick_myc-gene_III"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">BioBrick myc-gene III</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#BioBrick_AraC-TEV"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">BioBrick AraC-TEV</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#BioBrick_AraC-14_3C_protease"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">BioBrick AraC-14_3C protease</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#BioBrick_ssTorA_CS-TEV_blaFL"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">BioBrick ssTorA_CS-TEV_blaFL</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#BioBrick_ssTorA_CS-14_3C_blaFL"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">BioBrick ssTorA_CS-14_3C_blaFL</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#pSB1A3_Ara_and_pSB1A3_lac"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">pSB1A3 Ara and pSB1A3 lac</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#BioBrick_TEV_protease"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">BioBrick TEV protease</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#BioBrick_HRV14_3C_protease"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">BioBrick HRV14 3C protease</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="BioBricks"> BioBricks </SPAN></H2><TABLE class="wikitable" border="1"><TBODY><TR><TH> Label number </TH><TH> BioBrick Nickname </TH><TH> Tube label
</TH></TR><TR><TD> BBa_K627000 </TD><TD> mdnED </TD><TD> 1
</TD></TR><TR><TD> BBa_K627001 </TD><TD> mdnA </TD><TD> 2
</TD></TR><TR><TD> BBa_K627002 </TD><TD> mdnB </TD><TD> 3
</TD></TR><TR><TD> BBa_K627003 </TD><TD> mdnC </TD><TD> 4
</TD></TR><TR><TD> BBa_K627004 </TD><TD> mdnD </TD><TD> 5
</TD></TR><TR><TD> BBa_K627005 </TD><TD> mdnE </TD><TD> 6
</TD></TR><TR><TD> BBa_K627006 </TD><TD> mdnA c-myc gene III </TD><TD> 7
</TD></TR><TR><TD> BBa_K627007 </TD><TD> c-myc gene III </TD><TD> 8
</TD></TR><TR><TD> BBa_K627008 </TD><TD> AraC TEV protease 1 </TD><TD> 9
</TD></TR><TR><TD> BBa_K627009 </TD><TD> AraC TEV protease 2 </TD><TD> 10
</TD></TR><TR><TD> BBa_K627010 </TD><TD> AraC TEV protease 3 </TD><TD> 11
</TD></TR><TR><TD> BBa_K627011 </TD><TD> AraC 14_3C protease </TD><TD> 12
</TD></TR><TR><TD> BBa_K627012 </TD><TD> ssTorA CS-TEV BlaFL </TD><TD> 13
</TD></TR><TR><TD> BBa_K627013 </TD><TD> ssTorA CS-14_3C BlaFL </TD><TD> 14
</TD></TR><TR><TD> BBa_K627014 </TD><TD> A3_Ara_YFP </TD><TD> 15
</TD></TR><TR><TD> BBa_K627015 </TD><TD> A3_lac_YFP </TD><TD> 16
</TD></TR><TR><TD> BBa_K627016 </TD><TD> TEV protease </TD><TD> 17
</TD></TR><TR><TD> BBa_K627017 </TD><TD> HRV14 3C protease </TD><TD> 18
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="BioBrick_mdnED"> BioBrick mdnED </SPAN></H3><P><B>Part name:</B> BBa_K627000<B>Part type:</B> Coding<B>Short description:</B> ABC transporter and N-acetyltransferase from  the mdn-cluster<B>Full description:</B>
The BioBrick mdnED is a part of the whole microviridin gene (mdn) cluster, which encodes the protease 
inhibitor microviridin L. Microviridins are tricyclic depsipeptides, which are ribosomally synthesized 
by the cyanobacteria Microcystis aeruginosa (Ziemert et al., 2010). They have a promising potential for therapy as they can block 
disease-relevant proteases (Ziemert et al., 2008).

Microviridins are synthesized from a ribosomal precursor peptide (MdnA). Additionally, the microviridin L 
biosynthesis gene cluster consists of genes encoding an ATP-grasp-type ligase (mdnB and mdnC) and genes, 
which encode an ABC transporter (mdnE) and a N-acetyltransferase of the GNAT family (mdnD) (Ziemert et al., 
2008). 

In the following BioBrick mdnED the genes mdnD (N-acetyltransferase of the GNAT family) and mdnE (ABC 
transporter) is encoded (Ziemert et al., 2008).
<B>Source of the part:</B>
The BioBrick mdnDE as a part of the microviridin gene (mdn) cluster was isolated from Microcystis aeruginosa strain NIES-843.<B>Design information:</B>
This BioBrick was built by PCR using the following PCR primers
Forward primer: TAAATGAATTCGCGGCCGCTTCTAGATGCCTCAATATACTACTAAAC
Reverse primer: ATTTCTGCAGCGGCCGCTACTAGTATCAGCAAACCCTACTTAATTTC
To insert mdnED in the vector pSB1C3, the resulting PCR product and the vector were digested with the restriction enzymes EcoRI and SpeI.
Because this BioBrick is an expression part, the adenine of mdnE gene's start codon is part of the XbaI 
recognition site.<B>References:</B>
Ziemert, N., Ishida, K., Liaimer, A., Hertweck, C. &amp; Dittmann, E. (2008). Ribosomal synthesis of tricyclic depsipeptides in bloom-forming cyanobacteria. Angewandte Chemie (International ed. in English) 47, 7756-9
Ziemert, N., Ishida, K., Weiz, A., Hertweck, C. &amp; Dittmann, E. (2010). Exploiting the natural diversity of microviridin gene clusters for discovery of novel tricyclic depsipeptides. Applied and environmental microbiology 76, 3568-74<B>Genbank file:</B><A href="/wiki/images/5/55/UP_BioBrick_mdnED.gb" class="internal" title="UP BioBrick mdnED.gb">Media: UP_BioBrick_mdnED.gb</A></P><H3><SPAN class="mw-headline" id="BioBrick_mdnA"> BioBrick mdnA </SPAN></H3><P><B>Part name:</B> BBa_K627001<B>Part type:</B> Coding<B>Short description:</B> mdnA gene encoding the precursor peptide of the tricyclic microviridin<B>Full description:</B>
The BioBrick mdnA is a part of the whole microviridin gene (mdn) cluster, which encodes the protease inhibitor microviridin L. Microviridins are tricyclic depsipeptides, which are ribosomally synthesized by Microcystis aeruginosa (Ziemert et al., 2010). They have a promising potential for therapy as they can block disease-relevant proteases (Ziemert et al., 2008).

Microviridins are synthesized from a ribosomal precursor peptide (MdnA). Additionally, the microviridin L biosynthesis gene cluster consists of genes encoding an ATP-grasp-type ligase (mdnB and mdnC) and genes, which encode an ABC transporter (mdnE) and a N-acetyltransferase of the GNAT family (mdnD) (Ziemert et al., 2008).

The following BioBrick mdnA encodes the ribosomal precursor peptide (MdnA), which is essential for microviridin production (Ziemert et al., 2008).

Because this BioBrick is a RFC10 expression part, the adenine of mdnB gene`s start codon is part of the XbaI recognition site.
<B>Usage and Biology:</B>
Microviridin production in <I>E. coli</I> cells expressing the mdn genes was monitored by reverse phase HPLC. Qualitative analysis involves running a standard that contains the target analytes. The vectors pARW071 and pARW089 served as a control in our experiments because these vectors contain the original mdn-cluster. We could use the retention time as a way to determine the presence of the microviridin production in other samples. HPLC analysis of the purified compound yielded a high peak with a retention time of approximately 5 min. Minor peaks could be detected during the following 7 min (Fig. 1). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 1:</B> HPLC chromatogram of mdnA</DIV></DIV></DIV></DIV><P>All HPLC chromatograms of the isolated mdnA  showed reliable peaks with a retention time of 5 min together with a number of following minor peaks.
</P><P>Fractions of the peaks were sampled and the identity of microviridin and also the presence of cyclization, which is important for the activity, was affirmed with mass spectrometry (Fig. 2). By using a mass spectrometer it is possible to determine both the elemental composition of a fraction and the chemical structure of molecules.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:UP_modularization_MSII.gif" title="File:UP modularization MSII.gif">File:UP modularization MSII.gif</A><DIV class="thumbcaption"><B>Figure 2:</B> Mass Spectrometry Data</DIV></DIV></DIV></DIV><P><B>Source of the part:</B>
The BioBrick mdnA as a part of the microviridin gene (mdn) cluster was isolated from Microcystis aeruginosa strain NIES-843.
<B>Design information:</B>
This BioBrick was built by PCR using the following PCR primers
Forward primer: TTCCATGGCGCCAGAGGAATCTAGATGGCATATCCCAACGATC 
Reverse primer: CTTCTGACTGGGAAGATTATACCGGTTAATACTAGTAGCGGCCGCTGCAGGACGTC
To insert mdnA in the vector pSB1C3, the resulting PCR product and the vector were digested with the restriction enzymes EcoRI and SpeI.

Because this BioBrick is an expression part, the adenine of mdnA gene`s start codon is part of the XbaI recognition site. Further the sequence contains a AgeI recognition site after mdnA.
<B>Experiences:</B><B>References:</B>
Ziemert, N., Ishida, K., Liaimer, A., Hertweck, C. &amp; Dittmann, E. (2008). Ribosomal synthesis of tricyclic depsipeptides in bloom-forming cyanobacteria. Angewandte Chemie (International ed. in English) 47, 7756-9

Ziemert, N., Ishida, K., Weiz, A., Hertweck, C. &amp; Dittmann, E. (2010). Exploiting the natural diversity of microviridin gene clusters for discovery of novel tricyclic depsipeptides. Applied and environmental microbiology 76, 3568-74
<B>Genbank file:</B><A href="/wiki/images/a/ab/UP_Biobrick_mdnA.gb" class="internal" title="UP Biobrick mdnA.gb">Media: UP_Biobrick_mdnA.gb</A></P><H3><SPAN class="mw-headline" id="BioBrick_mdnB"> BioBrick mdnB </SPAN></H3><P><B>Part name:</B> BBa_K627002<B>Part type:</B> Coding<B>Short description:</B> ATP-grasp-type ligase from mdn-cluster<B>Full description:</B>
The BioBrick mdnB is a part of the whole microviridin gene (mdn) cluster, which encodes the protease inhibitor microviridin L. Microviridins are tricyclic depsipeptides, which are ribosomally synthesized by Microcystis aeruginosa (Ziemert et al., 2010). They have a promising potential for therapy as they can block disease-relevant proteases (Ziemert et al., 2008).

Microviridins are synthesized from a ribosomal precursor peptide (MdnA). Additionally, the microviridin L biosynthesis gene cluster consists of genes encoding an ATP-grasp-type ligase (mdnB and mdnC) and genes, which encode an ABC transporter (mdnE) and a N-acetyltransferase of the GNAT family (mdnD) (Ziemert et al., 2008).

The following BioBrick mdnB encodes a ATP-grasp-type ligase (Ziemert et al., 2008).

Because this BioBrick is an expression part, the adenine of mdnB gene`s start codon is part of the XbaI recognition site.
<B>Source of the part:</B>
The BioBrick mdnB as a part of the microviridin gene (mdn) cluster was isolated from Microcystis aeruginosa strain NIES-843.
<B>Design information:</B>
This BioBrick was built by PCR using the following PCR primers
Forward primer: ATTATGAATTCGCGGCCGCTTCTAGATGAAAGAATCGCCTAAAGTTG
Reverse primer: TAATCTGCAGCGGCCGCTACTAGTATCAACCGAAGACTAAAAAATCAGCG
To insert mdnB in the vector pSB1C3, the resulting PCR product and the vector were digested with the restriction enzymes EcoRI and SpeI.

Because this BioBrick is an expression part, the adenine of mdnE gene's start codon is part of the XbaI recognition site.
<B>References:</B>
Ziemert, N., Ishida, K., Liaimer, A., Hertweck, C. &amp; Dittmann, E. (2008). Ribosomal synthesis of tricyclic depsipeptides in bloom-forming cyanobacteria. Angewandte Chemie (International ed. in English) 47, 7756-9

Ziemert, N., Ishida, K., Weiz, A., Hertweck, C. &amp; Dittmann, E. (2010). Exploiting the natural diversity of microviridin gene clusters for discovery of novel tricyclic depsipeptides. Applied and environmental microbiology 76, 3568-74
<B>Genbank file:</B><A href="/wiki/images/e/e1/UP_BioBrick_mdnB.gb" class="internal" title="UP BioBrick mdnB.gb">Media: UP_BioBrick_mdnB.gb</A></P><H3><SPAN class="mw-headline" id="BioBrick_mdnC"> BioBrick mdnC </SPAN></H3><P><B>Part name:</B> BBa_K627003<B>Part type:</B> Coding<B>Short description:</B> ATP-grasp-type ligase from the mdn-cluster<B>Full description:</B>
The BioBrick mdnC is a part of the whole microviridin gene (mdn) cluster, which encodes the protease 
inhibitor microviridin L. Microviridins are tricyclic depsipeptides, which are ribosomally synthesized 
by Microcystis (Ziemert et al., 2010). They have a promising potential for therapy as they can block 
disease-relevant proteases (Ziemert et al., 2008).

Microviridins are synthesized from a ribosomal precursor peptide (MdnA). Additionally, the microviridin L 
biosynthesis gene cluster consists of genes encoding an ATP-grasp-type ligase (mdnB and mdnC) and genes, 
which encode an ABC transporter (mdnE) and a N-acetyltransferase of the GNAT family (mdnD) (Ziemert et al., 
2008). 

The following BioBrick mdnC encodes the ATP-grasp-type ligase (Ziemert et al., 2008).

Because this BioBrick is a RF10 expression part, the adenine of mdnC gene`s start codon is part of the XbaI 
recognition site.<B>Source of the part:</B>
The BioBrick mdnC as a part of the microviridin gene (mdn) cluster was isolated from Microcystis aeruginosa strain NIES-843.<B>Design information:</B>
This BioBrick was built by PCR using the following PCR primers
Forward primer: TATTTGAATTCGCGGCCGCTTCTAGATGACCGTTTTAATTGTTAC
Reverse primer: ATTTCTGCAGCGGCCGCTACTAGTATTATGAGTTAACTAGGATTTC 
To insert mdnC in the vector pSB1C3, the resulting PCR product and the vector were digested with the restriction enzymes EcoRI and SpeI.

Because this BioBrick is a RF10 expression part, the adenine of mdnA gene`s start codon is part of the XbaI recognition site.
<B>References:</B>
Ziemert, N., Ishida, K., Liaimer, A., Hertweck, C. &amp; Dittmann, E. (2008). Ribosomal synthesis of tricyclic depsipeptides in bloom-forming cyanobacteria. Angewandte Chemie (International ed. in English) 47, 7756-9
Ziemert, N., Ishida, K., Weiz, A., Hertweck, C. &amp; Dittmann, E. (2010). Exploiting the natural diversity of microviridin gene clusters for discovery of novel tricyclic depsipeptides. Applied and environmental microbiology 76, 3568-74<B>Genbank file:</B><A href="/wiki/images/f/f4/UP_BioBrick_mdnC.gb" class="internal" title="UP BioBrick mdnC.gb">Media: UP_BioBrick_mdnC.gb</A></P><H3><SPAN class="mw-headline" id="BioBrick_mdnD"> BioBrick mdnD </SPAN></H3><P><B>Part name:</B> BBa_K627004<B>Part type:</B> Coding<B>Short description:</B> N-acetyltransferase from the mdn-cluster<B>Full description:</B>
The BioBrick mdnD is a part of the whole microviridin gene (mdn) cluster, which encodes the protease 
inhibitor microviridin L. Microviridins are tricyclic depsipeptides, which are ribosomally synthesized 
by Microcystis (Ziemert et al., 2010). They have a promising potential for therapy as they can block 
disease-relevant proteases (Ziemert et al., 2008).

Microviridins are synthesized from a ribosomal precursor peptide (MdnA). Additionally, the microviridin L 
biosynthesis gene cluster consists of genes encoding an ATP-grasp-type ligase (mdnB and mdnC) and genes, 
which encode an ABC transporter (mdnE) and a N-acetyltransferase of the GNAT family (mdnD) (Ziemert et al., 
2008). 

The following BioBrick mdnD encodes a N-acetyltransferase of the GNAT family (Ziemert et al., 2008).

Because this BioBrick is a RF10 expression part the adenine of mdnD gene's start codon is part of the XbaI 
recognition site.<B>Source of the part:</B>
The BioBrick mdnD as a part of the microviridin gene (mdn) cluster was isolated from Microcystis aeruginosa strain NIES-843.<B>Design information:</B>
This BioBrick was built by PCR using the following PCR primers
Forward primer: TATATGAATTCGCGGCCGCTTCTAGATGAAAGCACTGGAAAAACTG
Reverse primer: ATTTCTGCAGCGGCCGCTACTAGTATCAGCAAACCCTACTTAATTTC 
To insert mdnD in the vector pSB1C3, the resulting PCR product and the vector were digested with the restriction enzymes EcoRI and SpeI.

Because this BioBrick is a RF10 expression part the adenine of mdnD gene`s start codon is part of the XbaI recognition site.
<B>References:</B>
Ziemert, N., Ishida, K., Liaimer, A., Hertweck, C. &amp; Dittmann, E. (2008). Ribosomal synthesis of tricyclic depsipeptides in bloom-forming cyanobacteria. Angewandte Chemie (International ed. in English) 47, 7756-9
Ziemert, N., Ishida, K., Weiz, A., Hertweck, C. &amp; Dittmann, E. (2010). Exploiting the natural diversity of microviridin gene clusters for discovery of novel tricyclic depsipeptides. Applied and environmental microbiology 76, 3568-74<B>Genbank file</B><A href="/wiki/images/3/36/UP_BioBrick_mdnD.gb" class="internal" title="UP BioBrick mdnD.gb">Media: UP_BioBrick_mdnD.gb</A></P><H3><SPAN class="mw-headline" id="BioBrick_mdnE"> BioBrick mdnE </SPAN></H3><P><B>Part name:</B> BBa_K627005<B>Part type:</B> Coding<B>Short description:</B> ABC transporter from the mdn-cluster<B>Full description:</B>
The BioBrick mdnE is a part of the whole microviridin gene (mdn) cluster, which encodes the protease 
inhibitor microviridin L. Microviridins are tricyclic depsipeptides, which are ribosomally synthesized 
by Microcystis (Ziemert et al., 2010). They have a promising potential for therapy as they can block 
disease-relevant proteases (Ziemert et al., 2008).

Microviridins are synthesized from a ribosomal precursor peptide (MdnA). Additionally, the microviridin L 
biosynthesis gene cluster consists of genes encoding an ATP-grasp-type ligase (mdnB and mdnC) and genes, 
which encode an ABC transporter (mdnE) and a N-acetyltransferase of the GNAT family (mdnD) (Ziemert et al., 
2008). 

The following BioBrick mdnE encodes an ABC transporter (Ziemert et al., 2008).

Because this BioBrick is a RF10 expression part the adenine of mdnE gene`s start codon is part of the XbaI 
recognition site.<B>Source of the part:</B>
The BioBrick mdnE as a part of the microviridin gene (mdn) cluster was isolated from Microcystis aeruginosa strain NIES-843.<B>Design information:</B>
This BioBrick was built by PCR using the following PCR primers
Forward primer: TAAATGAATTCGCGGCCGCTTCTAGATGCCTCAATATACTACTAAAC
Reverse primer: ATTTCTGCAGCGGCCGCTACTAGTACTATATTCTCACCCATTTTAAG
To insert mdnE in the vector pSB1C3, the resulting PCR product and the vector were digested with the restriction enzymes EcoRI and SpeI.

Because this BioBrick is a RF10 expression part the adenine of mdnE gene`s start codon is part of the XbaI recognition site.
<B>References:</B>
Ziemert, N., Ishida, K., Liaimer, A., Hertweck, C. &amp; Dittmann, E. (2008). Ribosomal synthesis of tricyclic depsipeptides in bloom-forming cyanobacteria. Angewandte Chemie (International ed. in English) 47, 7756-9
Ziemert, N., Ishida, K., Weiz, A., Hertweck, C. &amp; Dittmann, E. (2010). Exploiting the natural diversity of microviridin gene clusters for discovery of novel tricyclic depsipeptides. Applied and environmental microbiology 76, 3568-74<B>Genbank file</B><A href="/wiki/images/2/2e/UP_BioBrick_mdnE.gb" class="internal" title="UP BioBrick mdnE.gb">Media: UP_BioBrick_mdnE.gb</A></P><H3><SPAN class="mw-headline" id="BioBrick_mdnA-myc-gene_III"> BioBrick mdnA-myc-gene III </SPAN></H3><P><B>Part name:</B> BBa_K627006<B>Part type:</B> Coding<B>Short description:</B> Fusion part of mdnA gene (from mdn-cluster) with myc-tag and gene III<B>Full description</B>
This BioBrick is important for testing the suitability of a phage display system as an appropriate screening method for recombinant mdnA libraries.
The mdnA gene encodes the precursor peptide from the mdn-cluster, which is an essential building block for production of the protease inhibitor microviridin from Microcystis aeruginosa (Ziemert et al., 2008 and 2010). This peptide is known to be able to bind and inhibit proteases. Thus, variants of mdnA have a high potential for therapy as they can block disease-relevant proteases.
The gene III protein is a coat protein from the filamentous bacteriophage M13 (Smith, 1985). It appears only three to five times on the tip of the phage and is responsible for infection of bacterial cells. In phage display, genes from a DNA library are usually fused to the gene III and are displayed as fusion protein on the surface of the phages. Because it has been shown that the proteins of interest, which are fused to the C-terminus of gene-III can be functionally displayed (Krebber et al. 1997, Fuh and Sidhu, 2000, Rakonjac et al., 1999) only the sequence for this part has to be used.
The inserted sequence of the myc-tag enables easy detection and/or purification of the proteins of interest. In our case, the tag was used to verify display of the microviridin on the phage particle.
After transformation of E. coli with a vector containing the mdnA-myc-geneIII-fusion gene (in the context of the mdnA gene cluster) and co-infection with helper phages, E. coli cells were able to produce phage particles, which carry the microviridin peptide on their surface as confirmed by ELISA. These experiments proved the suitability of phage display as a screening method for variants of microviridin. 
<B>Control of expression of mdnA-myc-gene III in <I>E. coli</I></B></P><P>To control expression of the mdnA-myc-gene III fusion gene was analyzed by western blotting. E. coli cells transformed with the phagemid pPDV089 were harvested and lysated. The cell proteins were electrophoretically separated and transferred to a membrane. The mdnA-myc-gene III-fusion proteins were detected using specific anti-myc-antibodies and horseradish peroxidase (HRP)-linked antibodies as second antibodies. Enhanced chemiluminescence (ECL) was used to visualize the protein. ECL is based on the emission of light during the HRP -catalyzed oxidation of luminol, which was captured by a camera. The western blot analysis resulted in a band of a size just below the 30 kDa mark representing the mdnA-myc-gene III-fusion protein (24 kDa).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 3: Control of expression of mdnA-myc-gene III in <I>E. coli</I> by western blotting.</B> For detection anti-myc-antibodies and secondary HRP-linked antibodies were used. The resulting band represents the mdnA-myc-gene III-fusion protein.</DIV></DIV></DIV></DIV><P><B>Detection of phages carrying mdnA on their surface by ELISA</B>
The next step was the detection of the expression of the mdnA-myc-gene III-fusion gene on the surface of the phage. So <I>E. coli</I> cells strain XL1-Blue were first transformed with the phagemid pPDV089 before they were infected with helper phages. <I>E. coli</I> cells containing both plasmids were selected. An ELISA test was performed to determine whether these cells are able to produce phage particles carrying the mdnA peptide on their surface. To perform this test anti-c-myc-antibodies were immobilized on ELISA plates and incubated with purified phages. For detection a second antibody coupled with horse radish peroxidase (HRP) was used which binds the gene VIII protein of the phages. The HRP substrate o-phenyldiamine (OPD) was added and in case of binding a color reaction was expected. The color shift from achromatic to yellow in wells incubated with phages produced in XL1-Blue cells showed the successful expression of mdnA-c-myc-gene III-fusion protein on the phages.
For more precise results the absorption at 492 nm was measured. The data were presented in a bar plot. As a negative control helper phages were added instead of produced phages. Furthermore two wells were prepared were the secondary antibody was not added.
The graphic shows clearly the much higher absorption measured in wells, which were incubated with phage particles of interest produced in XL1-Blue cells. As has already pointed out this shows the succeeded expression of mdnA-c-myc-gene III-fusion protein on the surface of the phages.
</P><DIV align="center"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 4: Detection of phages carrying mdnA on their surface by ELISA.</B> The bar plot shows the absorption at 492 nm. Anti-myc-antibodies were immobilized. For detection a second antibody coupled with horse radish peroxidase (HRP) was used which binds the gene VIII coat protein of the phages. The left bar shows the absorption of the wells containing helper phages (negative control), the right bar shows the absorption of wells containing mdnA carrying phages</DIV></DIV></DIV></DIV></DIV><P><B>Testing phage display with unmodified mdnA to examine its suitability as screening method </B>
To test the suitability of this screening method, phages representing unmodified mdnA on their surface and phages not representing mdnA (helper phages) in a ratio of one to one were incubated with immobilized trypsin which is known as a target of mdnA. The display was conducted in ELISA plates. The bound phages were eluted using a buffer with low pH value and neutralized afterwards. To check how many phages interacted with trypsin, <I>E. coli</I> cells XL1-Blue were re-infected with eluted phages and plated on agar with different antibiotics. Cells infected with phages carrying mdnA are able to grow on agar with ampicillin whereas cells infected with helper phages are able to grow on agar with kanamycin. To control the success of the panning round additionally <I>E. coli</I> cells were infected with phage mix before panning and plated on agar. Subsequent the number of clones grew on ampicillin and kanamycin before and after panning was compared. During the running of this step it was noticed that much more cells were infected with helper phages than with phages carrying mdnA despite of the engaged 1:1 ratio. This was surprising and indicated that mdnA on the surface of the phages may inhibit their infectivity. After controlling the plates an  infection ratio of phages carrying mdnA to helper phages of 1:400 was calculated. This fact should be analyzed in further experiments.
The results of the first phage display are plotted in the figure below. After one panning round an enrichment of phages carrying mdnA was expected. This is attributable to the fact that phage particles carrying mdnA c-myc geneIII fusion protein on their surface are expected to bind specifically to the immobilized trypsin. Unfortunately this was not observed in this experiment. The ratio of cells growing on kanamycin agar before (4000) to cells growing on kanamycin agar (cells containing helper phages) after panning (750) was determined as 5:1. The ratio of cells growing on ampicillin agar before (12) to cells growing on ampicillin agar (cells containing mdnA carrying phages) after panning (2) was nearly equal. Thus no enrichment of mdnA carrying phages occurred in the first experiment. So it was decided to repeat this experiment under improved conditions. Therefor the number of washing steps during the described experimental procedure was increased. Here the ratio of cells growing on kanamycin agar before (3000) to cells growing on kanamycin agar (cells containing helper phages) after panning (29) was determined as 103:1. The ratio of cells growing on ampicillin agar before (26) to cells growing on ampicillin agar (cells containing mdnA carrying phages) after panning (2) was determined as 13:1. Thus an enrichment factor of eight was reached for the phages displaying mdnA on their surface. 
These results indicate that the unmodified mdnA expressed on the phages binds specifically to the immobilized trypsin. Therefore it can be deduced that mdnA is presented in a functional 3D structure. These findings suggest that phage display in general is an appropriate method for screening a recombinant mdnA library. Further experiments are required to optimize this system.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 5: Optimization of phage display.</B> After optimized conditions (right) a clear concentration of phages carrying mdnA after one panning round was noted. <I>E. coli</I> cells were infected with phage mix (helper phages and phages carrying mdnA) before and after panning and plated on agar containing kanamycin or ampicillin. The ratio of cells growing on ampicillin or kanamycin agar before panning to cells growing on ampicillin or kanamycin agar after was calculated. Helper phages which acted as negative control have a kanamycine resistance whereby phages carrying mdnA have an ampicillin resistance.</DIV></DIV></DIV></DIV><P><B>Testing phage display with unmodified mdnA against further proteases</B>
Furthermore the interaction of unmodified mdnA with other proteases was determined. From the literature (Ziemert, 2010) the high inhibitory activity of microviridin L, besides trypsin, against chymotrypsin and elastase is also known. So a phage display with these enzymes was performed. Additionally papain, proteinase K, mycolysin and pepsin were tested for which no interaction was shown yet. For all enzymes an equal amount of E. coli cells and phages were used. In agreement with data from the literature interaction of microviridin with chymotrypsin and elastase was confirmed. Additionally an interaction with papain was noted. All other proteases were not bound by microviridin. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 6: Phage display against different proteases.</B> After panning the number of clones was counted. In agreement with data from the literature interaction of microviridin with chymotrypsin and elastase was confirmed. Additionally an interaction with papain was noted.</DIV></DIV></DIV></DIV><P><B>Source of the part:</B>
The BioBrick mdnA is a part of the microviridin gene (mdn) cluster which was isolated from Microcystis aeruginosa strain NIES-843.
The gene III protein is a coat protein from the filamentous bacteriophage M13.
<B>Design information:</B>
Gene III was amplified from the vector pak100blaKDIR using the following primers
Forward:TAAGCTTCTAGATGGCCGGCGAGCAGAAGCTGATCTCTGAGGAAGACCTGGGTGGTGGCTCTGGTTCC
Reverse: TGCTTAGACGTCCTGCAGCGGCCGCTACTAGTATTAACCGGTAGACTCCTTATTACGCAGTA
The mdnA gene  was amplified from the vector pARW089 using the following primers
Forward: TTCCATGGCGCCAGAGGAATCTAGATGGCATATCCCAACGATC
Reverse: CTTCTGACTGGGAAGATTATACCGGTTAATACTAGTAGCGGCCGCTGCAGGACGTC
Gene III was amplified from pak100blaKDIR and mdnA from pARW089 by PCR. The primers were designed to enable the introduction of iGEM. The PCR product gene III was digested by whereas the PCR product mdnA was digested by AgeI. Thus a mdnA-gene III fusion part according to RFC25 was generated whereby AgeI and NgoMIV overhangs are compatible and  placed in frame with the protein sequence. The ligation of AgeI and NgoMIV overhangs resulted in a scar coding for the threonine and glycine. Because the introduction of restriction sites before mdnA leaded to a great distance between ribosome binding site and mdnA a second  RBS was inserted among SfoI and XbaI recognition sites to ensure a sufficiently expression rate of the mdnA-gene III-fusion gene. Between mdnA and gene III myc sequence was inserted.

Because this BioBrick is an RFC25 expression part, the adenine of mdnA gene's start codon is part of the XbaI recognition site. Further the sequence contains a AgeI recognition site after gene III.
<B>References:</B></P><UL><LI>Fuh G., Sidhu S.S. (2000). Efficient phage display of polypeptides fused to the carboxy-terminus of the M13 gene-3 minor coat protein. FEBS Lett. 480(2-3):231-4
</LI></UL><UL><LI>Krebber, A., Bornhauser, S., Burmester, J., Honegger, A., Willuda, J., Bosshard, H. R., Plückthun, A. (1997) Reliable cloning of functional antibody variable domains from hybridomas and spleen cell repertoires employing a reengineered phage display system. J. Immunol. Meth. 201(1):35-55 
</LI></UL><UL><LI>Rakonjac J., Feng J., Model P. (1999). Filamentous phage are released from the bacterial membrane by a two-step mechanism involving a short C-terminal fragment of pIII. J Mol Biol. 289(5):1253-65
</LI><LI>Smith, G.P. (1985). Filamentous fusion phage: Novel expression vectors that display cloned antigens on the virus surface. Science 228: 1315-17
</LI></UL><UL><LI>Ziemert, N., Ishida, K., Liaimer, A., Hertweck, C. &amp; Dittmann, E. (2008). Ribosomal synthesis of tricyclic depsipeptides in bloom-forming cyanobacteria. Angewandte Chemie (International ed. in English) 47, 7756-9
</LI></UL><UL><LI>Ziemert, N., Ishida, K., Weiz, A., Hertweck, C. &amp; Dittmann, E. (2010). Exploiting the natural diversity of microviridin gene clusters for discovery of novel tricyclic depsipeptides. Applied and environmental microbiology 76, 3568-74 
</LI></UL><P><B>Genbank file:</B><A href="/wiki/images/f/fc/UP_Biobrick_mdnA_c-myc_geneIII.gb" class="internal" title="UP Biobrick mdnA c-myc geneIII.gb">Media: UP_Biobrick_mdnA_c-myc_geneIII.gb</A></P><H3><SPAN class="mw-headline" id="BioBrick_myc-gene_III"> BioBrick myc-gene III </SPAN></H3><P><B>Part name:</B> BBa_K627007<B>Part type:</B> Coding<B>Short description:</B> Fusion of c-myc-tag and gene III <B>Full description:</B>
The gene III protein is a coat protein from the filamentous bacteriophage M13 (Smith, 1985). It appears only 3-5 times on the tip of the phage and is responsible for infection of bacterial cells. It is usually fused to the gene of interest to determine protein-protein-interactions of its expression product using M13 phage display system. The gene III-protein consists of three domains: one C-termial and two N-terminal domains. Gene III was generated as biobrick several times for instance M13003 (2006) and M31537 (2007). It has been shown that proteins of interest fused to the carboxy-terminus of gene-III-proteins can be functionally displayed on the surface of phages (Fuh and Sidhu, 2000, Rakonjac et al., 1999). In the course of our project a phagemid was constructed carrying truncated gene III (only encoding C-terminal domain) as biobrick. To enable easy purification and detection of gene-III-fused proteins additionally a myc-tag-sequence was merged before gene –III-sequence.  
<B>Source of the part:</B>
The gene-III-protein is a coat protein from the filamentous bacteriophage M13.
<B>Design information:</B>
Gene III was amplified from the vector pak100blaKDIR using the following primers
Forward:TAAGCTTCTAGATGGCCGGCGAGCAGAAGCTGATCTCTGAGGAAGACCTGGGTGGTGGCTCTGGTTCC
Reverse: TGCTTAGACGTCCTGCAGCGGCCGCTACTAGTATTAACCGGTAGACTCCTTATTACGCAGTA
Because myc-gene III is a fusion part (RFC25), no start codon is needed. Before the myc sequence a NgoMIV restriction site is located which has compatible overhangs to AgeI restriction sites. The ligation of AgeI and NgoMIV overhangs will result in a scar coding for threonine and glycine. So genes of interest containing AgeI restriction sites can be easily fused to gene III without frame shift. The gene III sequence contains no stop codon because there is one located between AgeI and SpeI recognition site directly merged behind gene III sequence. 
<B>References:</B>
Fuh G., Sidhu S.S. (2000). Efficient phage display of polypeptides fused to the carboxy-terminus of the M13 gene-3 minor coat protein. FEBS Lett. 480(2-3):231-4
</P><P>Rakonjac J., Feng J., Model P. (1999). Filamentous phage are released from the bacterial membrane by a two-step mechanism involving a short C-terminal fragment of pIII. J Mol Biol. 289(5):1253-65
</P><P>Smith, G.P. (1985). Filamentous fusion phage: Novel expression vectors that display cloned antigens on the virus surface. Science 228: 1315-17 
<B>Genbank file:</B><A href="/wiki/images/9/9e/UP_Biobrick_myc_gene_III.gb" class="internal" title="UP Biobrick myc gene III.gb">Media: UP_Biobrick_myc_gene III.gb</A></P><H3><SPAN class="mw-headline" id="BioBrick_AraC-TEV"> BioBrick AraC-TEV </SPAN></H3><P><B>Part name:</B> BBa_K627008, BBa_K627009, BBa_K627010 <B>Part type:</B> Device<B>Short description:</B> Fusion part of arabinose-inducible induction system and the TEV protease<B>RFC:</B> 10, 12, 23<B>Full description:</B></P><P><B>Introduction</B>
This BioBrick is a 2 part fusion of arabinose-inducible induction system and the TEV protease.<B>General Information of the single sub parts:</B><B>TEV protease</B>
TEV protease is the common name for the 27 kDa catalytic domain of the Nuclear Inclusion a endopeptidase (NIa) encoded by the tobacco etch virus (TEV). It recognizes a linear epitope of the general form E-Xaa-Xaa-Y -Xaa-Q-(G/S), with cleavage occurring between Q and G or Q and S. In TEV protease the serine nucleophile of the conventional Ser-Asp-His triad is a cysteine instead. This probably explains why TEV protease is resistant to many commonly used protease inhibitors.
At 37°C, the TEV protease forms inclusion body, which leads to an inactive form. Incubated at 30 °C, the protease is expressed as soluble type and is highly active.<B>Arabinose-inducible induction system</B>
The arabinose-inducible induction system was amplified via PCR from the pBAD_iGEMexpress vector. Based on the inhibitionary funtion of the catabolic activator protein (CAP) it got a very low expression rate without induction by arabinose. For the induction concentration between 2 mM up to 50 mM arabinose can be used to get an high expression rate.<B>General Function:</B>
When arabinose is added to the media (2mM up to 50 mM), the CAP lost its inhibitory function and the arabinose-inducible induction system gets activated. The protein, here the TEV protease, after the T7 RBS gets express at a very high rate. So we can assume that this system is an effectiv protease generator.<B>Performance and Summary:</B>
In combination with our created ssTorA_CS-TEV_blaFL device (BBa_K627012, <A href="http://partsregistry.org/Part:BBa_K627012" class="external text" rel="nofollow">More Information</A>) this protease generator was able to mediate the cell death of our transformed <I>E.coli</I> cells at ampicillin concentration up to 100 µg/ml.<B>Source of the part:</B>
Arabinose inducible operon form pBAD_iGEM_express, TEV protease from Gunther Stier et al.<B>Design Notes</B>
This BioBrick was built by PCR using the following PCR primers:<B>Primer used for site directed mutagenesis:</B><I>Fragment 1:</I></P><UL><LI>r_TEV_ACCAGC: GGAATGTGCAGCTGGTGTCTGACACC</LI><LI>f_TEV_iGEM: ATATAGAATTCGCGGCCGCTTCTAGATGGGAGAAAGCTTGTTTAAGGGA</LI></UL><P><I>Fragment 2:</I></P><UL><LI>r_TEV_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTATTGCGAGTACACCAATTCATTCAT</LI><LI>f_TEV_ACCAGC: GGTGTCAGACACCAGCTGCACATTCC</LI></UL><P><B>Primer used for assembly PCR of mutated fragments:</B></P><UL><LI>f_TEV_iGEM: ATATAGAATTCGCGGCCGCTTCTAGATGGGAGAAAGCTTGTTTAAGGGA</LI><LI>r_TEV_iGEM_BamHI: TATAGGATCCACTGCAGCGGCCGCTACTAGTTTATTGCGAGTACACCAATTCATTCAT</LI></UL><P><B>Primers used for amplification of pBAD arabinose-inducible induction system:</B></P><UL><LI>f_AraC_iGEM_HindIII: TATAAGCTTGAATTCGCGGCCGCTTCTAGATTATGACAACTTGACGGCTACATCATT</LI><LI>r_AraC_NgoMIV: ATAGCCGGCCTCCTTCTTAAAGTTAAACAAAATTATTTCTAGCCC</LI></UL><P><B>Primers used for amplification and modification of TEV protease:</B></P><UL><LI>f_TEV_AraFusion_NgoMIV: ATATTGCCGGCATGGGAGAAAGCCTGTTTAAGGGA</LI><LI>r_TEV_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTATTGCGAGTACACCAATTCATTCAT</LI></UL><P><B>Experience:</B><B>References:</B>
Cabrita, L. D., Gilis, D., Robertson, A. L., Dehouck, Y., Rooman, M. and Bottomley, S. P. (2007). Enhancing the stability and solubility of TEV protease using in silico design. Protein Sci. 16: 2360-2367

Kapust, R. B., Tözsér, J., Fox, J. D., Anderson, D. E., Cherry, S., Copeland, T. D., and Waugh, D. S. (2001). Tobacco etch virus protease: Mechanism of autolysis and rational design of stable mutants with wild-type catalytic proficiency. Prot. Eng. 14: 993-1000.

Lucast, L. J., Batey, R. T., and Doudna, J. A. (2001). Large-scale purification of a stable form of recombinant tobacco etch virus protease. Biotechniques 30: 544-550.<B>Genbank file</B><A href="/wiki/images/a/a8/BioBrick_AraC-TEV_comp.gb" class="internal" title="BioBrick AraC-TEV comp.gb">Media:BioBrick_AraC-TEV_comp.gb</A>
Three parts are existing.
</P><H3><SPAN class="mw-headline" id="BioBrick_AraC-14_3C_protease"> BioBrick AraC-14_3C protease</SPAN></H3><P><B>Part name:</B> BBa_K627011<B>Part type:</B>Device<B>Short description:</B> Fusion part of arabinose-inducible induction system and the HRV 14_3C protease<B>RFC:</B> 10, 12, 23<B>Full description:</B><B>Introduction</B>
This BioBrick is a 2 part fusion of arabinose-inducible induction system and the HRV 14_3C protease.<B>General Information of the single sub parts:</B><B>HRV 14_3C</B>
The recombinant type 14_3C protease from human rhinovirus (HRV 3C) recognizes the same cleavage site as the native enzyme: LeuGluValLeuPheGln/GlyPro.
The 22 kDa protease got its optimal activity at 4°C but also shows a high cleavage rate at 37°C. The 14_3C works with a catalytic triade, containing the amino acid residues Ser-Asp-His at its active site.<B>Arabinose-inducible induction system</B>
The arabinose-inducible induction system was amplified via PCR from the pBAD_iGEMexpress vector. Based on the inhibitionary funtion of the catabolic activator protein (CAP) it got a very low expression rate without induction by arabinose. For the induction concentration between 2 mM up to 50 mM arabinose can be used to get an high expression rate.<B>General Function:</B>
When arabinose is added to the media (2mM up to 50 mM), the CAP lost its inhibitory function and the arabinose-inducible induction system gets activated. The protein, here the HRV 14_3C protease, after the T7 RBS gets express at a very high rate. So we can assume that this system is an effectiv protease generator.<B>Performance and Summary:</B>
In combination with our created ssTorA_CS-14_3C_blaFL device (BBa_K627013, <A href="http://partsregistry.org/Part:BBa_K627013" class="external text" rel="nofollow">More Information</A>) this protease generator was able to mediate the cell death of our transformed <I>E.coli</I> cells at ampicillin concentration up to 100 µg/ml.<B>Source of the part:</B> 
Arabinose inducible operon form pBAD_iGEM_express was ampilied via PCR, 14_3C protease was isolated from the genome of human rhinovirus 14_3C and mutated with primers listed in part description BBa: to remove all iGEM restriction sites inside the protease gene.<B>Design Notes:</B>
This BioBrick was built by PCR using the following PCR primers:<B>Site directed mutagenesis of HRV 14 3C protease:</B><I>Fragment 1:</I></P><UL><LI> f_14_3C_iGEM: ATATAGAATTCGCGGCCGCTTCTAGATGGGACCAAACACAGAATTTGCACTATCC
</LI><LI> r_14_3C_ACCAGC: ACCATTTGCTGGTACATCATCACCAGG
</LI></UL><P><I>Fragment 2:</I></P><UL><LI> f_14_3C_ACCAGC: CCTGGTGATGATGTACCAGCAAATGGT
</LI><LI> r_14_3C_tm_XbaI208_A-T: CACTGTAAGCTCAAGATTAATGTTCTC
</LI></UL><P><I>Fragment 3:</I></P><UL><LI> f_14_3C_tm_Xba208_A-T: GAGAACATTAATCTTGAGCTTACAGTG
</LI><LI> r_14_3C_tm_XbaI280_A-T: ATCCACACCTTCGAGATCTTCTGATAT
</LI></UL><P><I>Fragment 4:</I></P><UL><LI> f_14_3C_tm_Xba280_A-T: ATATCAGAAGATCTCGAAGGTGTGGAT
</LI><LI> r_14_3C_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTTGTTTCTCTACAAAATATTGTTTTTTAAGTTGAGCTGA
</LI></UL><P><B>Primers used for first assembly PCR of mutated fragments:</B><I>Fragment 5, containing Fragment 1 and 2:</I></P><UL><LI> f_14_3C_iGEM: ATATAGAATTCGCGGCCGCTTCTAGATGGGACCAAACACAGAATTTGCACTATCC
</LI><LI> r_14_3C_tm_Xba208_A-T: CACTGTAAGCTCAAGATTAATGTTCTC
</LI></UL><P><I>Fragment 6, containing Fragment 3 and 4:</I></P><UL><LI> f_14_3C_tm_Xba208_A-T: GAGAACATTAATCTTGAGCTTACAGTG
</LI><LI> r_14_3C_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTTGTTTCTCTACAAAATATTGTTTTTTAAGTTGAGCTGA
</LI></UL><P><B>Primer used for final assembly PCR of mutated fragments:</B><I>Complete mutated 14_3C protease, containing Fragment 5 and 6:</I></P><UL><LI> f_14_3C_iGEM: ATATAGAATTCGCGGCCGCTTCTAGATGGGACCAAACACAGAATTTGCACTATCC
</LI><LI> r_14_3C_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTTGTTTCTCTACAAAATATTGTTTTTTAAGTTGAGCTGA
</LI></UL><P><I>Primers used for amplification of the pBAD arabinose-inducible induction system:</I></P><UL><LI> f_AraC_iGEM_HindIII: TATAAGCTTGAATTCGCGGCCGCTTCTAGATTATGACAACTTGACGGCTACATCATT
</LI><LI> r_AraC_NgoMIV: ATAGCCGGCCTCCTTCTTAAAGTTAAACAAAATTATTTCTAGCCC
</LI></UL><P><I>Primers used for amplification and modification of HRV 14 3C protease:</I></P><UL><LI> f_14_3C_AraFusion_NgoMIV: ATATTGCCGGCATGGGACCAAACACAGAATTTGCACTATCC
</LI><LI> r_14_3C_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTTGTTTCTCTACAAAATATTGTTTTTTAAGTTGAGCTGA
</LI></UL><P><B>Experience:</B><B>References:</B>
Combination of Two Separate Binding Domains Defines Stoichiometry between Type III Secretion System Chaperone IpgC and Translocator Protein IpaB, Ravi Kumar Lokareddy, Michele Lunelli, Björn Eilers, Vivien Wolter, and Michael Kolbe, December 17, 2010 The Journal of Biological Chemistry, 285, 39965-39975.

Cleavage of Small Peptides In Vitro by Human Rhinovirus 14 3C Protease  expressed in Escherichia coli, MICHAEL G. CORDINGLEY,* R. BRUCE REGISTER, PIA L. CALLAHAN, VICTOR M. GARSKY,AND RICHARD J. COLONNO JOURNAL OF VIROLOGY, Dec. 1989, p. 5037-5045
<B>Genbank file:</B><A href="/wiki/images/4/49/BioBrick_AraC-14_3C.gb" class="internal" title="BioBrick AraC-14 3C.gb">Media:BioBrick_AraC-14_3C.gb</A></P><H3><SPAN class="mw-headline" id="BioBrick_ssTorA_CS-TEV_blaFL"> BioBrick ssTorA_CS-TEV_blaFL </SPAN></H3><P><B>Part name:</B> BBa_K627012<B>Part type:</B> Composite Part<B>Short description:</B> Fusion of TorA sig-seq, TEV protease cleavage site and b-lactamase<B>RFC:</B> 10, 23, 25 <B>Full description:</B><B>Introduction</B>
This device is an detector for proteolytic activity of the matching protease (see cleavage site, here for TEV protease) inside the cytoplasm of <I>E.coli</I>. If there is an active protease, than the resistance capacity against ampicillin is lowered and the cells won't survive on higher antibiotic concentrations.<B>General Information of the single sub parts:</B><B>ssTorA</B>
The TorA signal-sequence originate from the enzyme Trimethylamin-N-Oxid-Reduktase, which needs to be folded in the cytoplasm.<B>Lactamase:</B>
The beta lactamase is an enzyme which cleaves lactam rings and makes bacteria resistant to beta lactam antibiotics like ampicillin and penicillin.<B>Proteolytic cleavage site for TEV protease:</B>
The cleavage site for the TEV protease was created via oligo hybridization and ligated into the construct with cleavage site for XhoI and NheI, which makes this part modular and easy adaptable for any other protease.<B>Setting up the right order...</B>
This construct contains the cleavage site for the TEV protease flanked by the sequence of beta-lactamase at the end and the TorA signal sequence for the TAT (twin arginine transporter) pathway infront of the cleavage site.<B>General Function</B>
This protease activity detector device provides the signal sequence from TorA for the TAT pathway, called ssTorA. These signal sequence is used for the export of folded proteins into the periplasm of gram-negative bacteria like <I>E.coli</I>.
The modular cleavage site, which can be exchanged with the enzymes XhoI and NheI, can be adapted to any protease. This feature allows the user to check for specific proteolytic activity in the cytoplasm.
The beta lactamase, a popular resistance marker, is used to check the activity of the desired protease inside the cytoplasm. By using the TAT pathway to export the lactamase into the periplasm the survival of bacteria cells at high ampicillin concentrations shows a low protease activity. In contrast, the death of all colony forming units indicates a high proteolytic activity inside the cytoplasm, which leads to a reduced export capacity of beta lactamase via the TAT pathway.
Also the modular construction of the cleavage site, which can be exchanged with the restriction enzymes XhoI and NheI, makes this protease activity detector highly adaptable to any desired protease.<B>Performance and Summary:</B>
With co-transformation of the TEV protease on a second plasmid, we can show the desired proof of principal: when the protease expression was activated, the transformed <I>E.coli</I> cells did not grow at ampicillin concentrations up to 100 µg/ml. The control colonies without induction of the protease survived up to concentrations of 400 µg/ml ampicillin.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 7:</B> Survival test at different ampicillin concentrations: A) approx. 1000 cfu platet on agar plates containing chloramphenicol; B) Reduced amount of <I>E.coli</I> colonies, approx. 1000 cfu were platet on agar plates containing chloramphenicol, 1mM IPTG to induce ampicillin resistance via BBa_K627012 and 400µg/ml ampicillin; C)no culture growth of <I>E.coli</I> colonies, approx. 1000 cfu were platet on agar plates containing chloramphenicol, 1mM IPTG to induce ampicillin resistance via BBa_K627012 and 600µg/ml ampicillin</DIV></DIV></DIV></DIV><P><B>Source</B>
TorA signal sequence and the lactamase was amplified via PCR from the vector pJC354, which originates from two iGEM compatible BioBricks: BBa_K208005 and BBa_I757010. The cleavage site was created, based on the information of the Brenda enzymes database, via 2 oligonucleotides ordered from SigmaAldrich.<B>Design Notes:</B>
This BioBrick was built by PCR using the following PCR primers:</P><UL><LI> p_TorA-f: CTTCTAGATGAACAATAACGATCTCTTTCAGGCATC
</LI><LI> o_Bla_igem_r: CTACTAGTATTAACCGGTCCAATGCTTAATCAGTGAGGCAC
</LI></UL><P><B>References</B>
ssTorA - <A href="http://partsregistry.org/Part:BBa_K208005" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_K208005</A>
blaFL - <A href="http://partsregistry.org/Part:BBa_I757010" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_I757010</A><B>Genbank file</B><A href="/wiki/images/d/d0/BioBrick_ssTorA_CS-TEV_blaFL.gb" class="internal" title="BioBrick ssTorA CS-TEV blaFL.gb">Media:BioBrick_ssTorA_CS-TEV_blaFL.gb</A></P><H3><SPAN class="mw-headline" id="BioBrick_ssTorA_CS-14_3C_blaFL"> BioBrick ssTorA_CS-14_3C_blaFL </SPAN></H3><P><B>Part name:</B> BBa_K627013<B>Part type:</B> Composite Part<B>Short description:</B> Fusion of ssTorA, a cleavage site for HRV 14_3C protease and beta lactamase<B>RFC:</B> 10, 23, 25 <B>Full description:</B><B>Introduction</B>
This device is an detector for proteolytic activity of the matching protease (see cleavage site, here for HRV 14_3C protease) inside the cytoplasm of <I>E.coli</I>. If there is an active protease, than the resistance capacity against ampicillin is lowered and the cells won't survive on higher antibiotic concentrations.<B>General Information of the single sub parts:</B><B>ssTorA</B>
The TorA signal-sequence originate from the enzyme Trimethylamin-N-Oxid-Reduktase, which needs to be folded in the cytoplasm.<B>Lactamase:</B>
The beta lactamase is an enzyme which cleaves lactam rings and makes bacteria resistant to beta lactam antibiotics like ampicillin and penicillin.<B>Proteolytic cleavage site for HRV 14_3C protease:</B>
The cleavage site for the TEV protease was created via oligo hybridization and ligated into the construct with cleavage site for XhoI and NheI, which makes this part modular and easy adaptable for any other protease.<B>Setting up the right order...</B>
This construct contains the cleavage site for the HRV 14_3C protease flanked by the sequence of beta-lactamase at the end and the TorA signal sequence for the TAT (twin arginine transporter) pathway infront of the cleavage site.<B>General Function</B>
This protease activity detector device provides the signal sequence from TorA for the TAT pathway, called ssTorA. These signal sequence is used for the export of folded proteins into the periplasm of gram-negative bacteria like <I>E.coli</I>.
The modular cleavage site, which can be exchanged with the enzymes XhoI and NheI, can be adapted to any protease. This feature allows the user to check for specific proteolytic activity in the cytoplasm.
The beta lactamase, a popular resistance marker, is used to check the activity of the desired protease inside the cytoplasm. By using the TAT pathway to export the lactamase into the periplasm the survival of bacteria cells at high ampicillin concentrations shows a low protease activity. In contrast, the death of all colony forming units indicates a high proteolytic activity inside the cytoplasm, which leads to a reduced export capacity of beta lactamase via the TAT pathway.
Also the modular construction of the cleavage site, which can be exchanged with the restriction enzymes XhoI and NheI, makes this protease activity detector highly adaptable to any desired protease.<B>Performance and Summary:</B>
With co-transformation of the HRV 14_3C protease on a second plasmid, we can show the desired proof of principal: when the protease expression was activated, the transformed <I>E.coli</I> cells did not grow at ampicillin concentrations up to 100 µg/ml. The control colonies without induction of the protease survived up to concentrations of 400 µg/ml ampicillin.<B>Source</B>
TorA signal sequence and the lactamase was amplified via PCR from the vector pJC354, which originates from two iGEM compatible BioBricks: BBa_K208005 and BBa_I757010. The cleavage site was created, based on the information of the Brenda enzymes database, via 2 oligonucleotides ordered from SigmaAldrich.<B>Design Notes:</B>
This BioBrick was built by PCR using the following PCR primers:</P><UL><LI> p_TorA-f: CTTCTAGATGAACAATAACGATCTCTTTCAGGCATC
</LI><LI> o_Bla_igem_r: CTACTAGTATTAACCGGTCCAATGCTTAATCAGTGAGGCAC
</LI></UL><P><B>Experience</B><B>References</B>
ssTorA - <A href="http://partsregistry.org/Part:BBa_K208005" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_K208005</A>
blaFL - <A href="http://partsregistry.org/Part:BBa_I757010" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_I757010</A><B>Genbank file</B><A href="/wiki/images/7/77/BioBrick_ssTorA_CS-14_3C_blaFL.gb" class="internal" title="BioBrick ssTorA CS-14 3C blaFL.gb">Media:BioBrick_ssTorA_CS-14_3C_blaFL.gb</A></P><H3><SPAN class="mw-headline" id="pSB1A3_Ara_and_pSB1A3_lac"> pSB1A3 Ara and pSB1A3 lac</SPAN></H3><P><B>Part name:</B> BBa_K627014<B>Part type:</B> Plasmid Backbone<B>Short description:</B> Plasmid backbone pSB1A3 carrying the arabinose induction system<B>Part name:</B> BBa_K627015<B>Part type:</B> Plasmid Backbone<B>Short description:</B> Plasmid backbone pSB1A3 carrying the lac operon induction system</P><P><B>Full description:</B>
In a subtask of our work we wanted to construct auxiliary expression backbones with inducible promotors.  In contrast to constitutive systems inducible systems only express the protein of interest after adding an inducer, which can be for example AHL or arabinose.
We decided to construct IPTG- and arabinose-inducible systems. Therefore we amplified the promotor region and fused it to a reporter gene, which is constituted by YFP. Using this we have the option to verify the presence of the inducible promotor and also the function of the induction process by fluorescence. Our idea was to use the reporter gene as a placeholder. Via restriction enzyme digestion using the iGEM restriction enzyme site you will be able to replace the reporter gene with your gene of interest. As vector backbone we made use of the pSB1A3, which has ampicillin resistance (Fig. 8). The constructs using pSB1K3 (kanamycin resistance) and pSB1C3 (chloramphenicol resistance) are still on the anvil. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 8:</B> Cloning Scheme of Expression Backbones. <I>amp</I> - gene for ampicillin resistance</DIV></DIV></DIV></DIV><P>We successfully tested the IPTG- and arabinose-inducible system (Fig 9). Using fluorescence microscopy we were able to detect the YFP-expression after an induction time of approximately 1.5 hrs. To prove the hypothesis we opposed an induced sample with a non-induced control for each promotor. By use of brightfield combined with differential interference contrast microscopy we traced the E. coli cells and switched then to the YFP detecting channel to investigate the fluorescence of these cells.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 9:</B> Fluorescenc Microscopy. control -  not induced; induced - induction with IPTG or Arabinose, respectively</DIV></DIV></DIV></DIV><P>We also tested the inducible systems by using fluorescence spectroscopy.  For this experiment we induced both systems at a time when the cultures were located in phase of exponential growth. After inducing, the cultures were analyzed by fluorescence spectroscopy.  In this analyze the cells were excitated by 500 nm.  The resultant emission was measured in a spectrum between 510 and 580 nm. (Fig 10;11)
</P><TABLE><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:412px;"><DIV class="thumbcaption"><B>Figure 10:</B> Emission (526nm) of pSB1A3_Ara_YFP and control –  E.coli cells (including pSB1A3_Ara_YFP) were induced and measured at certain moments.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:412px;"><DIV class="thumbcaption"><B>Figure 11:</B> Emission (526nm) of induced pSB1A3_lac_YFP and control –  E.coli cells (including pSB1A3_lac_YFP) were induced and measured at certain moments.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>In contrast to the control of the arabinose inducible system the Lac-control also shows fluorescence. This demonstrates the leaky Lac-promotor, what is also a scientifically proven fact. A leaky promotor means that the promotor is not regulated in a leakproof manner. Even in repressed conditions the transcript can arise.
</P><P>The lacking of the Lac-I gene on the vector is the reason for expression of YFP in both controls. For further research in this field it is necessary to clone the Lac-I gene inside the constructed vector.
<B>Source</B></P><UL><LI> pGA14mVenus Geneart
</LI><LI> Linearized BioBrick backbones
</LI></UL><P><B>Design Notes:</B>
We decided to construct IPTG- and arabinose-inducible systems. Therefore we amplified the promotor region and fused it to a reporter gene, which is constituted by YFP. Using this we have the option to verify the presence of the inducible promotor and also the function of the induction process by fluorescence. Our idea was to use the reporter gene as a placeholder. Via restriction enzyme digestion using the iGEM restriction enzyme site you will be able to replace the reporter gene with your gene of interest. As vector backbone we made use of the pSB1A3, which has ampicillin resistance (Fig. 8). The constructs using pSB1K3 (kanamycin resistance) and pSB1C3 (chloramphenicol resistance) are still on the anvil. 
</P><H3><SPAN class="mw-headline" id="BioBrick_TEV_protease">BioBrick TEV protease</SPAN></H3><P><B>Part name:</B> BBa_K627016<B>Part type:</B> Coding part<B>Short description:</B> mutated TEV protease without iGEM restriction sites<B>General Informations</B>
TEV protease is the common name for the 27 kDa catalytic domain of the Nuclear Inclusion a endopeptidase (NIa) encoded by the tobacco etch virus (TEV). It recognizes a linear epitope of the general form E-Xaa-Xaa-Y -Xaa-Q-(G/S), with cleavage occurring between Q and G or Q and S. In TEV protease the serine nucleophile of the conventional Ser-Asp-His triad is a cysteine instead. This probably explains why TEV protease is resistant to many commonly used protease inhibitors.
At 37°C, the TEV protease forms inclusion body, which leads to an inactive form. Incubated at 30 °C, the protease is expressed as soluble type and is highly active.<B>Primer used for site directed mutagenesis:</B><I>Fragment 1:</I></P><UL><LI>r_TEV_ACCAGC: GGAATGTGCAGCTGGTGTCTGACACC</LI><LI>f_TEV_iGEM: ATATAGAATTCGCGGCCGCTTCTAGATGGGAGAAAGCTTGTTTAAGGGA</LI></UL><P><I>Fragment 2:</I></P><UL><LI>r_TEV_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTATTGCGAGTACACCAATTCATTCAT</LI><LI>f_TEV_ACCAGC: GGTGTCAGACACCAGCTGCACATTCC</LI></UL><P><B>Primer used for assembly PCR of mutated fragments:</B></P><UL><LI>f_TEV_iGEM: ATATAGAATTCGCGGCCGCTTCTAGATGGGAGAAAGCTTGTTTAAGGGA</LI><LI>r_TEV_iGEM_BamHI: TATAGGATCCACTGCAGCGGCCGCTACTAGTTTATTGCGAGTACACCAATTCATTCAT</LI></UL><P><B>Primers used for amplification of pBAD arabinose-inducible induction system:</B></P><UL><LI>f_AraC_iGEM_HindIII: TATAAGCTTGAATTCGCGGCCGCTTCTAGATTATGACAACTTGACGGCTACATCATT</LI><LI>r_AraC_NgoMIV: ATAGCCGGCCTCCTTCTTAAAGTTAAACAAAATTATTTCTAGCCC</LI></UL><P><B>Primers used for amplification and modification of TEV protease:</B></P><UL><LI>f_TEV_AraFusion_NgoMIV: ATATTGCCGGCATGGGAGAAAGCCTGTTTAAGGGA</LI><LI>r_TEV_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTATTGCGAGTACACCAATTCATTCAT</LI></UL><P><B>References:</B>
Cabrita, L. D., Gilis, D., Robertson, A. L., Dehouck, Y., Rooman, M. and Bottomley, S. P. (2007). Enhancing the stability and solubility of TEV protease using in silico design. Protein Sci. 16: 2360-2367

Kapust, R. B., Tözsér, J., Fox, J. D., Anderson, D. E., Cherry, S., Copeland, T. D., and Waugh, D. S. (2001). Tobacco etch virus protease: Mechanism of autolysis and rational design of stable mutants with wild-type catalytic proficiency. Prot. Eng. 14: 993-1000.

Lucast, L. J., Batey, R. T., and Doudna, J. A. (2001). Large-scale purification of a stable form of recombinant tobacco etch virus protease. Biotechniques 30: 544-550.
</P><H3><SPAN class="mw-headline" id="BioBrick_HRV14_3C_protease">BioBrick HRV14 3C protease</SPAN></H3><P><B>Part name:</B> BBa_K627017<B>Part type:</B> Coding part<B>Short description:</B> HRV14 3C protease<B>HRV 14_3C</B>
The recombinant type 14_3C protease from human rhinovirus (HRV 3C) recognizes the same cleavage site as the native enzyme: LeuGluValLeuPheGln/GlyPro.
The 22 kDa protease got its optimal activity at 4°C but also shows a high cleavage rate at 37°C. The 14_3C works with a catalytic triade, containing the amino acid residues Ser-Asp-His at its active site.<B>General Function:</B>
When arabinose is added to the media (2mM up to 50 mM), the CAP lost its inhibitory function and the arabinose-inducible induction system gets activated. The protein, here the HRV 14_3C protease, after the T7 RBS gets express at a very high rate. So we can assume that this system is an effectiv protease generator.<B>Design Notes:</B>
This BioBrick was built by PCR using the following PCR primers:<B>Site directed mutagenesis of HRV 14 3C protease:</B><I>Fragment 1:</I></P><UL><LI> f_14_3C_iGEM: ATATAGAATTCGCGGCCGCTTCTAGATGGGACCAAACACAGAATTTGCACTATCC
</LI><LI> r_14_3C_ACCAGC: ACCATTTGCTGGTACATCATCACCAGG
</LI></UL><P><I>Fragment 2:</I></P><UL><LI> f_14_3C_ACCAGC: CCTGGTGATGATGTACCAGCAAATGGT
</LI><LI> r_14_3C_tm_XbaI208_A-T: CACTGTAAGCTCAAGATTAATGTTCTC
</LI></UL><P><I>Fragment 3:</I></P><UL><LI> f_14_3C_tm_Xba208_A-T: GAGAACATTAATCTTGAGCTTACAGTG
</LI><LI> r_14_3C_tm_XbaI280_A-T: ATCCACACCTTCGAGATCTTCTGATAT
</LI></UL><P><I>Fragment 4:</I></P><UL><LI> f_14_3C_tm_Xba280_A-T: ATATCAGAAGATCTCGAAGGTGTGGAT
</LI><LI> r_14_3C_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTTGTTTCTCTACAAAATATTGTTTTTTAAGTTGAGCTGA
</LI></UL><P><B>Primers used for first assembly PCR of mutated fragments:</B><I>Fragment 5, containing Fragment 1 and 2:</I></P><UL><LI> f_14_3C_iGEM: ATATAGAATTCGCGGCCGCTTCTAGATGGGACCAAACACAGAATTTGCACTATCC
</LI><LI> r_14_3C_tm_Xba208_A-T: CACTGTAAGCTCAAGATTAATGTTCTC
</LI></UL><P><I>Fragment 6, containing Fragment 3 and 4:</I></P><UL><LI> f_14_3C_tm_Xba208_A-T: GAGAACATTAATCTTGAGCTTACAGTG
</LI><LI> r_14_3C_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTTGTTTCTCTACAAAATATTGTTTTTTAAGTTGAGCTGA
</LI></UL><P><B>Primer used for final assembly PCR of mutated fragments:</B><I>Complete mutated 14_3C protease, containing Fragment 5 and 6:</I></P><UL><LI> f_14_3C_iGEM: ATATAGAATTCGCGGCCGCTTCTAGATGGGACCAAACACAGAATTTGCACTATCC
</LI><LI> r_14_3C_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTTGTTTCTCTACAAAATATTGTTTTTTAAGTTGAGCTGA
</LI></UL><P><I>Primers used for amplification of the pBAD arabinose-inducible induction system:</I></P><UL><LI> f_AraC_iGEM_HindIII: TATAAGCTTGAATTCGCGGCCGCTTCTAGATTATGACAACTTGACGGCTACATCATT
</LI><LI> r_AraC_NgoMIV: ATAGCCGGCCTCCTTCTTAAAGTTAAACAAAATTATTTCTAGCCC
</LI></UL><P><I>Primers used for amplification and modification of HRV 14 3C protease:</I></P><UL><LI> f_14_3C_AraFusion_NgoMIV: ATATTGCCGGCATGGGACCAAACACAGAATTTGCACTATCC
</LI><LI> r_14_3C_iGEM_BamHI: ATATAGGATCCACTGCAGCGGCCGCTACTAGTTTTGTTTCTCTACAAAATATTGTTTTTTAAGTTGAGCTGA
</LI></UL><P><B>References:</B>
Combination of Two Separate Binding Domains Defines Stoichiometry between Type III Secretion System Chaperone IpgC and Translocator Protein IpaB, Ravi Kumar Lokareddy, Michele Lunelli, Björn Eilers, Vivien Wolter, and Michael Kolbe, December 17, 2010 The Journal of Biological Chemistry, 285, 39965-39975.

Cleavage of Small Peptides In Vitro by Human Rhinovirus 14 3C Protease  expressed in Escherichia coli, MICHAEL G. CORDINGLEY,* R. BRUCE REGISTER, PIA L. CALLAHAN, VICTOR M. GARSKY,AND RICHARD J. COLONNO JOURNAL OF VIROLOGY, Dec. 1989, p. 5037-5045
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Potsdam_Bioware/BioBricks">http://2011.igem.org/Team:Potsdam_Bioware/BioBricks</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/BioBricks" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/BioBricks" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/BioBricks&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/BioBricks&amp;oldid=257935" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>