<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Labjournal_August_part_1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Labjournal/August_part_1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Labjournal/August_part_1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/August_part_1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/August_part_1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Labjournal/August_part_1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Labjournal/August part 1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team">Team</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Members">Members</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Project</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Summary</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Highlights">Highlights</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Microviridin">     Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Phage_Display">     Phage Display</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#In_Vivo_Selection">     In Vivo Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Modeling">     Modeling</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Microviridin">Details Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">Details Phage</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">Details Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Modeling">Details Modeling</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">BioBricks</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/BioBricks">Details BioBricks</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Methods">Methods</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Meetings">Meetings</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics">Safety &amp; Ethics</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Assesment">Safety Assessment</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Questions">Safety Questions</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Ethics_Seminar">Ethics Seminar</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Survey">Survey</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Potsdam_Bioware_at_the_German_parliament">Parliament Visit</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Statement_of_the_German_government">     Statement</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Outreach">Outreach</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software">Software</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#BioLog_App">BioLog App</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#Download">Download</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Sponsors">Sponsors</A></DT></DL></LI></UL></DIV><DIV class="pageContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#50th_Labday_2011-08-01"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">50th Labday 2011-08-01</SPAN></A></LI><LI class="toclevel-2"><A href="#Sequencing_of_mutated_mdnA_genes"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Sequencing of mutated mdnA genes</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_mdnA_and_GeneIII_for_phage_display_.28strategy_2.29"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Ligation of mdnA and GeneIII for phage display (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_mdnA.2FGeneIII-fusion_gene_into_pARW089_for_phage_display_.28strategy_2.29"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Ligation of mdnA/GeneIII-fusion gene into pARW089 for phage display (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#Mutagenesis_of_14_3C_and_TEV_proteases_to_remove_iGEM_restriction_sites_from_the_proteases_and_introduction_of_iGEM_restriction_sites"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Mutagenesis of 14_3C and TEV proteases to remove iGEM restriction sites from the proteases and introduction of iGEM restriction sites</SPAN></A></LI><LI class="toclevel-2"><A href="#Amplification_of_Arabinose_Induction_System_from_pBAD_iGEMexpress"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Amplification of Arabinose Induction System from pBAD_iGEMexpress</SPAN></A></LI><LI class="toclevel-2"><A href="#Annealing_of_TEV_mut_fragI_and_TEV_mut_fragII_.2F14_3C_mut_fragI_and_14_3C_mut_fragII"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Annealing of TEV_mut_fragI and TEV_mut_fragII /14_3C_mut_fragI and 14_3C_mut_fragII</SPAN></A></LI><LI class="toclevel-2"><A href="#Planning_and_accomplishing_the_PCRs_of_pBAD-mYFP_Venus_with_Arabinosis_and_pEX_HisII_with_Lac"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Planning and accomplishing the PCRs of pBAD-mYFP Venus with Arabinosis and pEX_HisII with Lac</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#51th_Labday_2011-08-02"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">51th Labday 2011-08-02</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_BBa_I763007_in_E._coli_XL1-Blue"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Transformation of BBa_I763007 in E. coli XL1-Blue</SPAN></A></LI><LI class="toclevel-2"><A href="#Repetition_of_the_PCRs_of_pBAD-mYFP_Venus_with_Arabinosis_and_pEX_HisII_with_Lac_because_of_less_yield_further_planning_and_accomplishing_the_PCR_of_BBa_I763007_as_it_is_a_constitutive_one"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Repetition of the PCRs of pBAD-mYFP Venus with Arabinosis and pEX_HisII with Lac because of less yield further planning and accomplishing the PCR of BBa_I763007 as it is a constitutive one</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparing_of_linearized_backbones_.28pSB1A3.2C_pSB1K3_and_pSB1T3.29_to_produce_vectors_for_further_applications"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Preparing of linearized backbones (pSB1A3, pSB1K3 and pSB1T3) to produce vectors for further applications</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_linearized_plasmid_backbones_.28pSB1A3.2C_pSB1K3_and_pSB1T3.29"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Restriction enzyme digestion of linearized plasmid backbones (pSB1A3, pSB1K3 and pSB1T3)</SPAN></A></LI><LI class="toclevel-2"><A href="#Gel_electrophoresis_of_digested_.28linearized.29_plasmid_backbones_.28pSB1A3.2C_pSB1K3_and_pSB1T3.29"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Gel electrophoresis of digested (linearized) plasmid backbones (pSB1A3, pSB1K3 and pSB1T3)</SPAN></A></LI><LI class="toclevel-1"><A href="#52th_Labday_2011-08-03"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">52th Labday 2011-08-03</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_generated_pPDV089_.28strategy_2.29"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Transformation of generated pPDV089 (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_pBAD-mYFP_Venus_and_pEX_HisII_with_pSB1_K3_and_pSB1_A3"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Ligation of pBAD-mYFP Venus and pEX_HisII with pSB1_K3 and pSB1_A3</SPAN></A></LI><LI class="toclevel-1"><A href="#53th_Labday_2011-08-04"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">53th Labday 2011-08-04</SPAN></A></LI><LI class="toclevel-2"><A href="#digestion_of_vector_pARW089_.28strategy_2.29"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">digestion of vector pARW089 (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#digestion_of_PCR_products_vector_pARW089_.28strategy_2.29"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">digestion of PCR products vector pARW089 (strategy 2)</SPAN></A></LI><LI class="toclevel-1"><A href="#54th_Labday_2011-08-05"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">54th Labday 2011-08-05</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_fragments_and_pJC354-NheI-143C-Xho_blaFL_GGH5_vector"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Digestion of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV fragments and pJC354-NheI-143C-Xho_blaFL_GGH5 vector</SPAN></A></LI><LI class="toclevel-2"><A href="#digest_pSB1A3.2C_pSB1K3_.28clone_1-16.29"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">digest pSB1A3, pSB1K3 (clone 1-16)</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_pARW089_for_Phage_Display_strategy_II"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Restriction enzyme digestion pARW089 for Phage Display strategy II</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_overnight_cultures_from_ligation_of_pARW089_.2B_mutated_mdnA_and_test_digest"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Miniprep of overnight cultures from ligation of pARW089 + mutated mdnA and test digest</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_overnight_cultures_from_ligation_of_pBAD-mYFP_Venus_and_pEX_HisII_with_pSB1_K3_and_pSB1_A3"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">Miniprep of overnight cultures from ligation of pBAD-mYFP Venus and pEX_HisII with pSB1_K3 and pSB1_A3</SPAN></A></LI><LI class="toclevel-1"><A href="#55th_Labday_2011-08-06"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">55th Labday 2011-08-06</SPAN></A></LI><LI class="toclevel-2"><A href="#2nd_Mutagenesis_of_TEV_proteases_to_remove_iGEM_restriction_sites_from_the_proteases_and_introduction_of_iGEM_restriction_sites"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">2nd Mutagenesis of TEV proteases to remove iGEM restriction sites from the proteases and introduction of iGEM restriction sites</SPAN></A></LI><LI class="toclevel-2"><A href="#2nd_Annealing_of_TEV_mut_fragI_and_TEV_mut_fragII"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">2nd Annealing of TEV_mut_fragI and TEV_mut_fragII</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI_and_of_the_unexpected_band_from_the_2nd_annealing_TEV_PCR"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Digestion of NgoMIV_iGEM_TEV-Protease_iGEM_BamHI and of the unexpected band from the 2nd annealing TEV PCR</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_PCR_of_mdnA_and_gene_III_for_phage_display_.28strategy_2.29"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">Repeated PCR of mdnA and gene III for phage display (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#digestion_of_PCR_products_.28strategy_2.29"><SPAN class="tocnumber">6.5</SPAN><SPAN class="toctext">digestion of PCR products (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#gel_electrophoresis_of_digested_fragments_and_digested_pARW089"><SPAN class="tocnumber">6.6</SPAN><SPAN class="toctext">gel electrophoresis of digested fragments and digested pARW089</SPAN></A></LI><LI class="toclevel-2"><A href="#prepare_samples_for_sequencing_.28pARW089_.2B_mutated_mdnA.29"><SPAN class="tocnumber">6.7</SPAN><SPAN class="toctext">prepare samples for sequencing (pARW089 + mutated mdnA)</SPAN></A></LI><LI class="toclevel-1"><A href="#56th_Labday_2011-08-07"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">56th Labday 2011-08-07</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_purification_of_TEV_mut_fragI_und_II"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">PCR purification of TEV_mut_fragI und II</SPAN></A></LI><LI class="toclevel-2"><A href="#gel_electrophoresisTEV_mut_fragI_and_II"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">gel electrophoresisTEV_mut_fragI and II</SPAN></A></LI><LI class="toclevel-2"><A href="#Annealing_PCR_TEV_mut_fragI_and_II"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Annealing PCR TEV_mut_fragI and II</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_and_pJC354-NheI-143C-Xho_blaFL_GGH5_vector"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV and pJC354-NheI-143C-Xho_blaFL_GGH5 vector</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_Ligation_of_pJC_AraC_and_14_3C_in_E._coli_XL1_blue"><SPAN class="tocnumber">7.5</SPAN><SPAN class="toctext">Transformation of Ligation of pJC AraC and 14_3C in E. coli XL1 blue</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_PCR_gene_III_for_phage_display_.28strategy_2.29"><SPAN class="tocnumber">7.6</SPAN><SPAN class="toctext">Repeated PCR gene III for phage display (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#digestion_of_PCR_products_and_vector_.28strategy_2.29"><SPAN class="tocnumber">7.7</SPAN><SPAN class="toctext">digestion of PCR products and vector (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#prepare_samples_for_sequencing_.28pSB1A3.2C_pSB1K3.29"><SPAN class="tocnumber">7.8</SPAN><SPAN class="toctext">prepare samples for sequencing (pSB1A3, pSB1K3)</SPAN></A></LI><LI class="toclevel-1"><A href="#57th_Labday_2011-08-08"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">57th Labday 2011-08-08</SPAN></A></LI><LI class="toclevel-2"><A href="#test_digest_of_pSB1K3"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">test digest of pSB1K3</SPAN></A></LI><LI class="toclevel-2"><A href="#sequencing_pARW089_.2B_mutated_mdnA"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">sequencing pARW089 + mutated mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Assembly_PCR_of_TEV_frag_I_and_TEV_frag_II"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Assembly PCR of TEV_frag_I and TEV_frag_II</SPAN></A></LI><LI class="toclevel-2"><A href="#design_primer_for_biobrick_-_mdnABC.2CmdnB.2CmdnC.2CmdnDE.2CmdnD.2CmdnE"><SPAN class="tocnumber">8.4</SPAN><SPAN class="toctext">design primer for biobrick - mdnABC,mdnB,mdnC,mdnDE,mdnD,mdnE</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_mdnA.2FGeneIII-fusion_gene_into_pARW089_for_phage_display_and_transformation_of_competent_cells.28strategy_2.29"><SPAN class="tocnumber">8.5</SPAN><SPAN class="toctext">Ligation of mdnA/GeneIII-fusion gene into pARW089 for phage display and transformation of competent cells(strategy 2)</SPAN></A></LI><LI class="toclevel-1"><A href="#58th_Labday_2011-08-09"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">58th Labday 2011-08-09</SPAN></A></LI><LI class="toclevel-2"><A href="#test_digest_of_pSB1K3.2BYFP_or_CFP_and_pSB1A3.2BYFP_or_CFP"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">test digest of pSB1K3+YFP or CFP and pSB1A3+YFP or CFP</SPAN></A></LI><LI class="toclevel-2"><A href="#Gel_purification_of_amplificated_TEV_protease"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Gel purification of amplificated TEV protease</SPAN></A></LI><LI class="toclevel-2"><A href="#Ampilifcation_of_NgoMIV_TEV_iGEM_BamHI_protease_via_PCR"><SPAN class="tocnumber">9.3</SPAN><SPAN class="toctext">Ampilifcation of NgoMIV_TEV_iGEM_BamHI protease via PCR</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_-Purification_of_amplified_NgoMIV_TEV_iGEM_BamHI_DNA-Fragements"><SPAN class="tocnumber">9.4</SPAN><SPAN class="toctext">PCR -Purification of amplified NgoMIV_TEV_iGEM_BamHI DNA-Fragements</SPAN></A></LI><LI class="toclevel-2"><A href="#Ampilifcation_of_NgoMIV_TEV_iGEM_BamHI_protease_via_PCR_2"><SPAN class="tocnumber">9.5</SPAN><SPAN class="toctext">Ampilifcation of NgoMIV_TEV_iGEM_BamHI protease via PCR</SPAN></A></LI><LI class="toclevel-2"><A href="#ELISA_to_test_the_anti_myc-tag_antibodies_9E10"><SPAN class="tocnumber">9.6</SPAN><SPAN class="toctext">ELISA to test the anti myc-tag antibodies 9E10</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparing_linearized_backbones_pSB1K3.2C_pSB1A3_and_pSB1T3_for_ligation_w.2F_YFP_and_CFP"><SPAN class="tocnumber">9.7</SPAN><SPAN class="toctext">Preparing linearized backbones pSB1K3, pSB1A3 and pSB1T3 for ligation w/ YFP and CFP</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_YFP_from_pGA14mVenusGeneart_and_CFP_from_pGA14-Cerulean"><SPAN class="tocnumber">9.8</SPAN><SPAN class="toctext">Digestion of YFP from pGA14mVenusGeneart and CFP from pGA14-Cerulean</SPAN></A></LI><LI class="toclevel-2"><A href="#over_night_culture_from_PDV089"><SPAN class="tocnumber">9.9</SPAN><SPAN class="toctext">over night culture from PDV089</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_PCR_of_mdnA_for_phage_display_.28strategy_1.29_repeated_with_new_ordered_reversed_primer_and_of_mdnA_with_rfc25_restriction_sites_.28strategy_2.29.2C_gel_electrophoresis_and_purification_of_PCR_products"><SPAN class="tocnumber">9.10</SPAN><SPAN class="toctext">Repeated PCR of mdnA for phage display (strategy 1) repeated with new ordered reversed primer and of mdnA with rfc25 restriction sites (strategy 2), gel electrophoresis and purification of PCR products</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_mdnA_with_sfiI_over_night"><SPAN class="tocnumber">9.11</SPAN><SPAN class="toctext">Digestion of mdnA with sfiI over night</SPAN></A></LI><LI class="toclevel-2"><A href="#design_primer_for_biobrick_-_redesign_of_pf_mdnE.2C_pf_mdnD"><SPAN class="tocnumber">9.12</SPAN><SPAN class="toctext">design primer for biobrick - redesign of pf_mdnE, pf_mdnD</SPAN></A></LI><LI class="toclevel-1"><A href="#59th_Labday_2011-08-10"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">59th Labday 2011-08-10</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI_fragments_.28produced_on_09.08.2011.29_and_pJC354-NheI-TEV-Xho_blaFL_GGH5_vector"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Digestion of NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragments (produced on 09.08.2011) and pJC354-NheI-TEV-Xho_blaFL_GGH5 vector</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_E._coli_XL1-Blue_with_pGA14mVenusGeneart_resp._pGA14-Cerulean"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">Transformation of E. coli XL1-Blue with pGA14mVenusGeneart resp. pGA14-Cerulean</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_Gel_of_digested_pSB1K3.2C_pSB1A3_and_pSB1T3_and_digested_CFP_and_YFP_fragments"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">Agarose Gel of digested pSB1K3, pSB1A3 and pSB1T3 and digested CFP and YFP fragments</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_YFP_from_pGA14mVenusGeneart_and_CFP_from_pGA14-Cerulean_2"><SPAN class="tocnumber">10.4</SPAN><SPAN class="toctext">Digestion of YFP from pGA14mVenusGeneart and CFP from pGA14-Cerulean</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_E._coli_overnight_culture_containing_pPDV089"><SPAN class="tocnumber">10.5</SPAN><SPAN class="toctext">Miniprep of E. coli overnight culture containing pPDV089</SPAN></A></LI><LI class="toclevel-2"><A href="#Test_digestion_of_ligations_for_strategy_2_after_Plasmid_preperation"><SPAN class="tocnumber">10.6</SPAN><SPAN class="toctext">Test digestion of ligations for strategy 2 after Plasmid preperation</SPAN></A></LI><LI class="toclevel-2"><A href="#cultivation_of_cells_containing_pAK100_bla_KDIR_from_E._coli_glycerol_stocks"><SPAN class="tocnumber">10.7</SPAN><SPAN class="toctext">cultivation of cells containing pAK100 bla KDIR from E. coli glycerol stocks</SPAN></A></LI><LI class="toclevel-1"><A href="#60th_Labday_2011-08-11"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">60th Labday 2011-08-11</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_Gel_digested_CFP_and_YFP_fragments_.28from_2011-08-10.29"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Agarose Gel digested CFP and YFP fragments (from 2011-08-10)</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_digested_and_purified_CFP_and_YFP_fragments_.28from_2011-08-11.29_w.2F_digested_and_purified_pSB1A3.2C_pSB1K3.2C_pSB1T3_.28from_2011-08-10.29"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Ligation of digested and purified CFP and YFP fragments (from 2011-08-11) w/ digested and purified pSB1A3, pSB1K3, pSB1T3 (from 2011-08-10)</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI_or_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_into_pJC354-NheI-143C-Xho_blaFL_GGH5_or_pJC354-NheI-TEV-Xho_blaFL_GGH5_vector"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI or NgoMIV_iGEM_TEV-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV into pJC354-NheI-143C-Xho_blaFL_GGH5 or pJC354-NheI-TEV-Xho_blaFL_GGH5 vector</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_E._Coli_XL1_Blue_Cells_with_ligation_products_.28NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI_or_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_into_pJC354-NheI-143C-Xho_blaFL_GGH5_or_pJC354-NheI-TEV-Xho_blaFL_GGH5_vectors.29"><SPAN class="tocnumber">11.4</SPAN><SPAN class="toctext">Transformation of E. Coli XL1 Blue Cells with ligation products (NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI or NgoMIV_iGEM_TEV-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV into pJC354-NheI-143C-Xho_blaFL_GGH5 or pJC354-NheI-TEV-Xho_blaFL_GGH5 vectors)</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation"><SPAN class="tocnumber">11.5</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_PCR_gene_III_for_phage_display_.28strategy_2.29_2"><SPAN class="tocnumber">11.6</SPAN><SPAN class="toctext">Repeated PCR gene III for phage display (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_E._coli_overnight_culture_containing_pakblaKDIR"><SPAN class="tocnumber">11.7</SPAN><SPAN class="toctext">Miniprep of E. coli overnight culture containing pakblaKDIR</SPAN></A></LI><LI class="toclevel-2"><A href="#digestion_of_pak_blaKDIR_.28strategy_1.29"><SPAN class="tocnumber">11.8</SPAN><SPAN class="toctext">digestion of pak blaKDIR (strategy 1)</SPAN></A></LI><LI class="toclevel-2"><A href="#sending_clone_12.2C_13_and_17_from_ligation_strategy_2_.28see_2011-08-10.29_to_MWG_for_sequencing"><SPAN class="tocnumber">11.9</SPAN><SPAN class="toctext">sending clone 12, 13 and 17 from ligation strategy 2 (see 2011-08-10) to MWG for sequencing</SPAN></A></LI><LI class="toclevel-2"><A href="#Gel_electrophoresis_of_digested_PAK100_bla_KDIR_vector.28strategy_1.29_and_PCR_of_geneIII"><SPAN class="tocnumber">11.10</SPAN><SPAN class="toctext">Gel electrophoresis of digested PAK100 bla KDIR vector(strategy 1) and PCR of geneIII</SPAN></A></LI><LI class="toclevel-2"><A href="#ligation_of_mdnA_into_pak100blaKDIR_.28stategy_1.29"><SPAN class="tocnumber">11.11</SPAN><SPAN class="toctext">ligation of mdnA into pak100blaKDIR (stategy 1)</SPAN></A></LI><LI class="toclevel-1"><A href="#61th_Labday_2011-08-12"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">61th Labday 2011-08-12</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI_or_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI_with_HindIII_iGEM_AraC_NgoMIV"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI or NgoMIV_iGEM_TEV-Protease_iGEM_BamHI with HindIII_iGEM_AraC_NgoMIV</SPAN></A></LI><LI class="toclevel-2"><A href="#Colony_PCR_of_TEV_clones_obtained_from_transformation"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">Colony PCR of TEV clones obtained from transformation</SPAN></A></LI><LI class="toclevel-2"><A href="#Results_of_Transformation_from_2011-8-11_and_Overnight_Cultures_of_marked_colonies"><SPAN class="tocnumber">12.3</SPAN><SPAN class="toctext">Results of Transformation from 2011-8-11 and Overnight Cultures of marked colonies</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_generated_pPDV100_from_over_night_ligation_.28strategy_1.29"><SPAN class="tocnumber">12.4</SPAN><SPAN class="toctext">Transformation of generated pPDV100 from over night ligation (strategy 1)</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_PCR_gene_III_for_phage_display_.28strategy_2.29_3"><SPAN class="tocnumber">12.5</SPAN><SPAN class="toctext">Repeated PCR gene III for phage display (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_overnight_cultures_of_Cerulean_and_mVenus_Geneart"><SPAN class="tocnumber">12.6</SPAN><SPAN class="toctext">Miniprep of overnight cultures of Cerulean and mVenus Geneart</SPAN></A></LI><LI class="toclevel-2"><A href="#gel_electrophoresis_of_colony_PCR.28TEV.29"><SPAN class="tocnumber">12.7</SPAN><SPAN class="toctext">gel electrophoresis of colony PCR(TEV)</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_pSB1A3.2BYFP_in_XL1-Blue_and_pSB1T3.2BYFPII_in_RV308"><SPAN class="tocnumber">12.8</SPAN><SPAN class="toctext">Transformation of pSB1A3+YFP in XL1-Blue and pSB1T3+YFPII in RV308</SPAN></A></LI><LI class="toclevel-2"><A href="#Design_and_ordering_of_primers_to_produce_BioBricks_of_the_mdn_genes"><SPAN class="tocnumber">12.9</SPAN><SPAN class="toctext">Design and ordering of primers to produce BioBricks of the mdn genes</SPAN></A></LI><LI class="toclevel-1"><A href="#62th_Labday_2011-08-13"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">62th Labday 2011-08-13</SPAN></A></LI><LI class="toclevel-2"><A href="#Mini-Prep_of_pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV_and_creation_of_glycerol_stock_culture"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">Mini-Prep of pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV and creation of glycerol stock culture</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_pSB1X3_.2B_Y_.28X_-_A.2FT.2FK_.3B_Y_-_YFP.2FCFP.29"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">Miniprep of pSB1X3 + Y (X - A/T/K ; Y - YFP/CFP)</SPAN></A></LI><LI class="toclevel-1"><A href="#63th_Labday_2011-08-14"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">63th Labday 2011-08-14</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_PCR_of_mdnA_and_gene_III_for_phage_display_.28strategy_1.2B2.29"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Repeated PCR of mdnA and gene III for phage display (strategy 1+2)</SPAN></A></LI><LI class="toclevel-2"><A href="#digestion_of_pak_blaKDIR_.28strategy_1.29_2"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">digestion of pak blaKDIR (strategy 1)</SPAN></A></LI><LI class="toclevel-2"><A href="#overnight_culture_of_10_picked_clones_of_XL_blue_cells_transformed_with_pPDV100"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">overnight culture of 10 picked clones of XL blue cells transformed with pPDV100</SPAN></A></LI><LI class="toclevel-2"><A href="#generate_over_night_culture_of"><SPAN class="tocnumber">14.4</SPAN><SPAN class="toctext">generate over night culture of</SPAN></A></LI><LI class="toclevel-1"><A href="#64th_Labday_2011-08-15"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">64th Labday 2011-08-15</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_pSB1A3_.2B_YFP_clone_I.2C_pSB1A3_.2B_YFP_clone_II.2C_pSB1A3_.2B_YFP_clone_III"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">Miniprep of pSB1A3 + YFP clone I, pSB1A3 + YFP clone II, pSB1A3 + YFP clone III</SPAN></A></LI><LI class="toclevel-2"><A href="#Test_digest_of_minipreps_of_pSB1X3.2BYFP.2FCPF_from_2011-08-13_and_2011-08-15"><SPAN class="tocnumber">15.2</SPAN><SPAN class="toctext">Test digest of minipreps of pSB1X3+YFP/CPF from 2011-08-13 and 2011-08-15</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_E._coli_RV308_with_pSB1T3.2BYFPII"><SPAN class="tocnumber">15.3</SPAN><SPAN class="toctext">Transformation of E. coli RV308 with pSB1T3+YFPII</SPAN></A></LI><LI class="toclevel-2"><A href="#mini_prep_and_test_digestion_of_phage_display_vector_pPDV100_from_over_night_cultures_.28strategy_1.29"><SPAN class="tocnumber">15.4</SPAN><SPAN class="toctext">mini prep and test digestion of phage display vector pPDV100 from over night cultures (strategy 1)</SPAN></A></LI><LI class="toclevel-2"><A href="#digestion_of_vector_pARW089_.28strategy_2.29_2"><SPAN class="tocnumber">15.5</SPAN><SPAN class="toctext">digestion of vector pARW089 (strategy 2)</SPAN></A></LI><LI class="toclevel-2"><A href="#Survival_test_for_E._coli_XL1_blue_transformed_with_pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV"><SPAN class="tocnumber">15.6</SPAN><SPAN class="toctext">Survival test for E. coli XL1 blue transformed with pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV</SPAN></A></LI><LI class="toclevel-2"><A href="#Survival_test_for_E.coli_XL1_blue_transformed_with_pUP_SG2_ssTorA_CS-Pre_bla"><SPAN class="tocnumber">15.7</SPAN><SPAN class="toctext">Survival test for E.coli XL1 blue transformed with pUP_SG2_ssTorA_CS-Pre_bla</SPAN></A></LI><LI class="toclevel-1"><A href="#65th_Labday_2011-08-16"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">65th Labday 2011-08-16</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_YFP_from_pGA14mVenusGeneart_and_CFP_from_pGA14-Cerulean_3"><SPAN class="tocnumber">16.1</SPAN><SPAN class="toctext">Digestion of YFP from pGA14mVenusGeneart and CFP from pGA14-Cerulean</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_Gel_digested_CFP_and_YFP_fragments"><SPAN class="tocnumber">16.2</SPAN><SPAN class="toctext">Agarose Gel digested CFP and YFP fragments</SPAN></A></LI><LI class="toclevel-2"><A href="#Sequencing_of_pSB1A3_and_pSB1K3_carrying_Cerulean_resp._mVenus"><SPAN class="tocnumber">16.3</SPAN><SPAN class="toctext">Sequencing of pSB1A3 and pSB1K3 carrying Cerulean resp. mVenus</SPAN></A></LI><LI class="toclevel-2"><A href="#Planning_and_accomplishing_the_control_PCRs_of_pSB1A3_and_pSB1K3_carrying_mVenus_resp._Cerulean"><SPAN class="tocnumber">16.4</SPAN><SPAN class="toctext">Planning and accomplishing the control PCRs of pSB1A3 and pSB1K3 carrying mVenus resp. Cerulean</SPAN></A></LI><LI class="toclevel-2"><A href="#DNA_extraction_of_mVenus_and_Cerulean"><SPAN class="tocnumber">16.5</SPAN><SPAN class="toctext">DNA extraction of mVenus and Cerulean</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis_test_digested_pPDV100_.28strategy_1.29"><SPAN class="tocnumber">16.6</SPAN><SPAN class="toctext">Agarose gel electrophoresis test digested pPDV100 (strategy 1)</SPAN></A></LI><LI class="toclevel-2"><A href="#Sequencing_of_pPDV100_of_clone_2_.28strategy_1.29"><SPAN class="tocnumber">16.7</SPAN><SPAN class="toctext">Sequencing of pPDV100 of clone 2 (strategy 1)</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_pSB1A3.2FpSB1K3_with_CFP.2FYFP_and_PCR_fragments_of_the_promoters_.28Ara.2C_Lac.2C_constitutive.29"><SPAN class="tocnumber">16.8</SPAN><SPAN class="toctext">Digestion of pSB1A3/pSB1K3 with CFP/YFP and PCR fragments of the promoters (Ara, Lac, constitutive)</SPAN></A></LI></TD></TR></TBODY></TABLE><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="50th_Labday_2011-08-01">50th Labday 2011-08-01</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Sequencing_of_mutated_mdnA_genes">Sequencing of mutated mdnA genes</SPAN></H3><P><B>Investigators:</B> Steffi, Vanessa, Nicole</P><P><B>Aim:</B> Determination of mutation rate employing sequencing of mutated mdnA</P><P><B>Time:</B> 2011-08-01,10:00-13:00</P><P><B>Materials:</B></P><UL><LI> Miniprep of mutated mdnA and restricted for 2 resp. 3 hours
</LI></UL><UL><LI> Sequencing Primer: sf_mdna_1
</LI></UL><UL><LI> Freelabels for Value ReadTube (MWG Eurofins)
</LI></UL><P><B>Method:</B></P><UL><LI> DNA concentration (for sequencing): 100 ng/ µl
</LI></UL><UL><LI> Total volume: 15 µl
</LI></UL><UL><LI> Primer concentration: 2 pmol/ µl
</LI></UL><UL><LI> Total volume: 200 µl (approx. 15 µl per sequencing reaction)
</LI></UL><UL><LI> sent to MWG Eurofins with the aid of Free sample bags
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Analyzing sequencing results
</LI></UL><UL><LI> Determination of mutation rates
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_mdnA_and_GeneIII_for_phage_display_.28strategy_2.29">Ligation of mdnA and GeneIII for phage display (strategy 2)</SPAN></H3><P><B>Investigators:</B> Leif </P><P><B>Aim:</B>ligation of mdnA and GenIII to get a fusion gene for phage display</P><P><B>Time:</B> 11:00-13:00</P><P><B>Method:</B></P><UL><LI> ligation-samples from 2011-07-27;</LI></UL><P><B>Protocol: </B></P><UL><LI> 5 µl (ca 25 ng) digested geneIII (NgoMIV, AatII) </LI></UL><UL><LI> 3 µl (ca 30 ng) digested mdnA (NarI, AgeI) </LI></UL><UL><LI> 2 µl 10x T4 Ligase Buffer </LI></UL><UL><LI> 1 µl T4 Ligase </LI></UL><UL><LI> 9 µl water </LI></UL><P>2 h, room temperature
</P><P><B>Further tasks:</B> ligation into vector
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_mdnA.2FGeneIII-fusion_gene_into_pARW089_for_phage_display_.28strategy_2.29">Ligation of mdnA/GeneIII-fusion gene into pARW089 for phage display (strategy 2)</SPAN></H3><P><B>Investigators:</B> Leif </P><P><B>Aim:</B> ligation of mdnA and GenIII to get a fusion gene for phage display</P><P><B>Time:</B> 11:00-13:00</P><P><B>Method:</B></P><UL><LI> ligation-samples from 2011-08-01;</LI></UL><P><B>Protocol: </B></P><UL><LI> 6 µl (ca 70 ng) digested vector pARW089 (NarI, AatII)
</LI></UL><UL><LI> 1 µl (ca 5 ng) fusion gene mdnA/geneIII
</LI></UL><UL><LI> 2 µl 10x T4 Ligase Buffer
</LI></UL><UL><LI> 1 µl T4 Ligase
</LI></UL><UL><LI> 10 µl water
</LI></UL><P>2 h, room temperature, then over night in the freezer
</P><P><B>Further tasks:</B> transformation of <I>E. coli</I></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Mutagenesis_of_14_3C_and_TEV_proteases_to_remove_iGEM_restriction_sites_from_the_proteases_and_introduction_of_iGEM_restriction_sites">Mutagenesis of 14_3C and TEV proteases to remove iGEM restriction sites from the proteases and introduction of iGEM restriction sites</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[1]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian </P><P><B>Aim:</B></P><UL><LI> Removal of iGEM restriction sites from proteases, amplifying protease fragments with iGEM restriction sites</LI></UL><P><B>Materials:</B></P><UL><LI>Plasmid 1: pET9d_Thrombin-CS_XbaI-TEV-Protease_BamHI
</LI></UL><UL><LI>Primers: (1) f_TEV_ACCAGC , r_TEV_iGEM_BahmHI (2) r_TEV_ACCAGC , f_TEV_AraFusion_NgoMIV
</LI></UL><UL><LI>Plasmid 2: pGEX-3_14_3C
</LI></UL><UL><LI>Primers: (1) f_14_3C_ACCAGC, r_14_3C_iGEM_BamHI (2) r_14_3C_ACCAGC, f_14_3C_AraFusion_NgoMIV
</LI></UL><P><B> Used method: </B></P><P>PCR
</P><UL><LI>Template: 1µl = 3,6 ng
</LI></UL><UL><LI>Nucleotides: 1µl of 10mM ready to use dNTP mix
</LI></UL><UL><LI>5µl 10x Amplification buffer S
</LI></UL><UL><LI>5µl 25mM MgCl2
</LI></UL><UL><LI>2,5µl primers = 25pmol absolute (2,5µl of each primer = 5µl per tube)
</LI></UL><UL><LI>32,5µl of pure water
</LI></UL><UL><LI>0,5µl TaqPol
</LI></UL><P>Program: iGEM001
</P><UL><LI>Denat: 3min 94°C
</LI></UL><UL><LI>5x:
</LI></UL><P>Denat: 45sec 94°C
</P><P>Anneal:45sec 53°C
</P><P>Extend:45sec 72°C
</P><UL><LI>25x:
</LI></UL><P>Denat: 45sec 60°C
</P><P>Anneal:45sec 60°C
</P><P>Extend:45sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P><B> Result: </B></P><P>Resolving of PCR products (10µl) on 2% agarose gel
</P><P>Expected Fragments:
</P><P>TEV:
</P><UL><LI>TEV_mut_fragI: 360bp
</LI></UL><UL><LI>TEV_mut_fragII: 400bp
</LI></UL><P>14_3C:
</P><UL><LI>14_3C_mut_fragI: 420bp
</LI></UL><UL><LI>14_3C_mut_fragII: 153bp
</LI></UL><P><B> Further going: </B></P><UL><LI>40µl of PCR products left:*20µl for preparative agarose gel (2%)
</LI></UL><UL><LI>20µl for PCR purification Kit
</LI></UL><UL><LI>Assembly PCR of purificated products to produce NgoMIV_iGEM_TEV-Protease_iGEM_BamHI and NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Amplification_of_Arabinose_Induction_System_from_pBAD_iGEMexpress">Amplification of Arabinose Induction System from pBAD_iGEMexpress</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[2]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian </P><P><B>Aim:</B></P><UL><LI> Amplificarion of an arabinose induction system (AraC) from pBAD_iGEMexpress plasmid, produces a 1273bp fragment</LI></UL><P><B>Materials:</B></P><UL><LI>Plasmid: pBAD_iGEMexpress (Nr.4)
</LI></UL><UL><LI>Primers: f_AraC_HindIII_iGEM , r_AraC_NgoMIV
</LI></UL><P><B> Used method: </B></P><P>PCR
</P><UL><LI>Template: 1µl = 7,2 ng
</LI></UL><UL><LI>Nucleotides: 1 µl of 10mM ready to use dNTP mix
</LI></UL><UL><LI>5µl 10x Amplification buffer S
</LI></UL><UL><LI>5µl 25mM MgCl2
</LI></UL><UL><LI>2,5µl primers = 25pmol absolute (2,5µl of each primer = 5µl per tube)
</LI></UL><UL><LI>32,5µl of pure water
</LI></UL><UL><LI>0,5µl TaqPol
</LI></UL><P>Program: iGEM002
</P><UL><LI>Denat: 3min 94°C
</LI></UL><UL><LI>5x:
</LI></UL><P>Denat: 60sec 94°C
</P><P>Anneal:60sec 53°C
</P><P>Extend:60sec 72°C
</P><UL><LI>25x:
</LI></UL><P>Denat: 60sec 60°C
</P><P>Anneal:60sec 60°C
</P><P>Extend:60sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P><B> Result: </B></P><P>Resolving of PCR products (10µl) on 1% agarose gel
</P><P>Expected Fragments: HindIII_iGEM_AraC_NgoMIV 1273bp
</P><P><B> Further tasks: </B></P><UL><LI>30µl of PCR product left: (1) Preparative Agarose Gel + Extraction(2) Digestion of fragment with HindIII and NgoMIV and gel purification (3) Ligation with (digested) NgoMIV_14_3C-Protease_iGEM_BamHI or NgoMIV_TEV_iGEM_BamHI fragments (see entry above).
</LI></UL><P><B> EDIT: </B></P><P>The HindIII_iGEM_AraC_NgoMIV fragment was purificated from a preparative gel (1.5%)
</P><P>Concentration:12,5ng/µl
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Annealing_of_TEV_mut_fragI_and_TEV_mut_fragII_.2F14_3C_mut_fragI_and_14_3C_mut_fragII">Annealing of TEV_mut_fragI and TEV_mut_fragII /14_3C_mut_fragI and 14_3C_mut_fragII</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[3]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian </P><P><B>Materials:</B></P><P>TEV:
</P><UL><LI>TEV_mut_fragI: 360bp (40µl)
</LI></UL><UL><LI>TEV_mut_fragII: 400bp (40µl)
</LI></UL><P>Primers for PCR:
</P><UL><LI>f_TEV_AraFusion_NgoMIV
</LI></UL><UL><LI>r_TEV_iGEM_BamHI
</LI></UL><P>HRV 14_3C:
</P><UL><LI>14_3C_mut_fragI: 420bp (40µl)
</LI></UL><UL><LI>14_3C_mut_fragII: 153bp (40µl)
</LI></UL><P>Primers for PCR:
</P><UL><LI>f_14_3C_AraFusion_NgoMIV
</LI></UL><UL><LI>r_14_3C_iGEM_BamHI
</LI></UL><P><B> Used method: </B></P><P>1. 20µl of EACH fragment were purificated using a preparative agarose gel followed by extraction with &quot;NucleoSpin ExtractII&quot;-KIT and the other 20µl of EACH fragment were purificated with PCR-purification unsing &quot;NucleoSpin ExtractII&quot;-KIT.
</P><P>2. Annealing of purificated primers using PCR (program iGEM002):
</P><UL><LI>Template: Everything from purification (~15µl) = 4 reaction batches: 2x for TEV fragments, 2x 14_3C fragments from each purification method, respectively.
</LI></UL><UL><LI>Nucleotides: 1 µl of 10mM ready to use dNTP mix
</LI></UL><UL><LI>5µl 10x Amplification buffer S
</LI></UL><UL><LI>5µl 25mM MgCl2
</LI></UL><UL><LI>2,5µl primers = 25pmol absolute (2,5µl of each primer = 5µl per tube)
</LI></UL><UL><LI>17,5µl of pure water
</LI></UL><UL><LI>0,5µl TaqPol
</LI></UL><P>Program: iGEM0002
</P><UL><LI>Denat: 3min 94°C
</LI></UL><UL><LI>5x:
</LI></UL><P>Denat: 60sec 94°C
</P><P>Anneal:60sec 53°C
</P><P>Extend:60sec 72°C
</P><UL><LI>25x:
</LI></UL><P>Denat: 60sec 60°C
</P><P>Anneal:60sec 60°C
</P><P>Extend:60sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P><B> Result: </B></P><P>Resolving of PCR products (5µl) on 1,5% analytical agarose gel
</P><P>Expected Fragments
</P><P>for TEV:
</P><P>NgoMIV_iGEM_TEV-Protease_iGEM_BamHI - 760bp
</P><P>for 14_3C:
</P><P>NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI - 573bp
</P><P>Summary:
</P><P>Ligation of TEV_mut_fragI and TEV_mut_fragII did NOT work.
</P><P>Ligation of 14_3C_mut_fragI and 14_3C_mut_fragII did work.
</P><P>The two fractions of ligated 14_3C fragments were combined and PCR-purificated using &quot;NucleoSpin ExtractII&quot;-KIT (Concentration: 25ng/µl)
</P><P><B> Further tasks: </B></P><P>Digestion of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI fragment with NgoMIV and BamHI
</P><P>Starting a new PCR for to produce new TEV-mut_fragI and TEV_mut_fragII fragments!!
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Planning_and_accomplishing_the_PCRs_of_pBAD-mYFP_Venus_with_Arabinosis_and_pEX_HisII_with_Lac"> Planning and accomplishing the PCRs of pBAD-mYFP Venus with Arabinosis and pEX_HisII with Lac </SPAN></H3><P><B>Investigators:</B> Nicole, Nadja</P><P><B>Aim:</B>To get Biobricks with Arabinosis and IPTG induction</P><P><B>Time:</B> 2011-08-01,14:00-19:00</P><P><B>Materials:</B></P><UL><LI> Primer: 1. pr_Ara_Xba1, pf_Ara_EcoR1 and 2. pf_IPTG_EcoR1, pf_IPTG_Xba1
</LI></UL><UL><LI> Plasmids: 1. pBAD-mYFP Venus and 2. pEX_HisII
</LI></UL><P><B>Method:</B>PCR</P><P>1.pBAD-mYFP Venus with Ara
</P><UL><LI> 1,25 µl pBAD-mYFP Venus (1:10, 10,8 ng/µl)
</LI></UL><UL><LI> 1,00 µl dNTPs
</LI></UL><UL><LI> 5,00 µl Buffer
</LI></UL><UL><LI> 2,50 µl pr_Ara_XbaI
</LI></UL><UL><LI> 2,50 µl pf_Ara_EcoRI
</LI></UL><UL><LI> 2,00 µl MgCl<SUB>2</SUB></LI></UL><UL><LI> 0,50 µl Polymerase S
</LI></UL><UL><LI> 35,25 µl H<SUB>2</SUB>O
</LI></UL><P>2. pEX_HisII with Lac
</P><UL><LI> 1,00 µl pEX_HisII(1:20, 9,6 ng/µl)
</LI></UL><UL><LI> 1,00 µl dNTPs
</LI></UL><UL><LI> 5,00 µl Buffer
</LI></UL><UL><LI> 2,50 µl pf_IPTG_EcoRI
</LI></UL><UL><LI> 2,50 µl pf_IPTG_XbaI
</LI></UL><UL><LI> 2,00 µl MgCl2
</LI></UL><UL><LI> 0,50 µl Polymerase S
</LI></UL><UL><LI> 35,50 µl H<SUB>2</SUB>O
</LI></UL><P>3. Program for both: IGBIOB1 (30 cycles)
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Step
</TH><TH> Temperature
</TH><TH> Time
</TH></TR><TR><TD> Hot start
</TD><TD> 94°C
</TD><TD> hold
</TD></TR><TR><TD> Initial Denaturation
</TD><TD> 94°C
</TD><TD> 3min (180s)
</TD></TR><TR><TD> Denaturation
</TD><TD> 94°C
</TD><TD> 20s
</TD></TR><TR><TD> Annealing
</TD><TD> 44°C
</TD><TD> 40s
</TD></TR><TR><TD> Extension
</TD><TD> 72°C
</TD><TD> 73s
</TD></TR><TR><TD> Final Extension
</TD><TD> 72°C
</TD><TD> 600s
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> See PCR results on agarose gel and do a gel extraction
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="51th_Labday_2011-08-02">51th Labday 2011-08-02</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_BBa_I763007_in_E._coli_XL1-Blue">Transformation of BBa_I763007 in <I>E. coli</I> XL1-Blue</SPAN></H3><P><B>Investigators:</B> Jessica, Steffi, Vanessa</P><P><B>Aim:</B>Transformation of plasmid BBa_I763007 (containg Lambda promoter, RBS and RFP) in E. coli XL1-Blue cells to produce glycerol stocks for further use </P><P><B>Time:</B> 2011-08-02, </P><P><B>Materials:</B></P><P><B>Method:</B></P><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culturing
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Repetition_of_the_PCRs_of_pBAD-mYFP_Venus_with_Arabinosis_and_pEX_HisII_with_Lac_because_of_less_yield_further_planning_and_accomplishing_the_PCR_of_BBa_I763007_as_it_is_a_constitutive_one">Repetition of the PCRs of pBAD-mYFP Venus with Arabinosis and pEX_HisII with Lac because of less yield further planning and accomplishing the PCR of BBa_I763007 as it is a constitutive one </SPAN></H3><P><B>Investigators:</B> Nadja, Nicole</P><P><B>Aim:</B>To increase the yield and to get a constitutive Biobrick</P><P><B>Time:</B>2011-08-02</P><P><B>Materials:</B></P><UL><LI>Primer: 1. pr_Ara_Xba1, pf_Ara_EcoR1 and 2. pf_IPTG_EcoR1, pf_IPTG_Xba1 and 3. Pr_constitutive_XbaI, pf_constitutive_EcoRI
</LI></UL><UL><LI> Plasmids: 1. pBAD-mYFP Venus and 2. pEX_HisII and 3. BBa_1763007
</LI></UL><P><B>Method:</B>PCR</P><P>1. pBAD-mYFP Venus with Ara
</P><UL><LI> 1,25 µl pBad (1:10, 10,8ng/µl)
</LI></UL><UL><LI> 1,00 µl dNTPs
</LI></UL><UL><LI> 5,00 µl Buffer
</LI></UL><UL><LI> 2,50 µl pr_Ara_Xba1
</LI></UL><UL><LI> 2,50 µl pf_Ara_EcoR1
</LI></UL><UL><LI> 2,00 µl MgCl2
</LI></UL><UL><LI> 0,50 µl Polymerase S
</LI></UL><UL><LI> 35,25µl H<SUB>2</SUB>O
</LI></UL><P>2. pEX_HisII with Lac
</P><UL><LI> 1,00 µl pEX (1:20, 9,6 ng/µl)
</LI></UL><UL><LI> 1,00 µl dNTPs
</LI></UL><UL><LI> 5,00 µl Buffer
</LI></UL><UL><LI> 2,50 µl pf_IPTG_EcoR1
</LI></UL><UL><LI> 2,50 µl pf_IPTG_Xba1
</LI></UL><UL><LI>2,00 µl MgCl<SUB>2</SUB></LI></UL><UL><LI>0,50 µl Polymerase S
</LI></UL><UL><LI> 35,50 µl H<SUB>2</SUB>O
</LI></UL><P>1. BBa_1763007 -constititive
</P><UL><LI> 1,00 µl BBa_1763007
</LI></UL><UL><LI> 1,00 µl dNTPs
</LI></UL><UL><LI> 5,00 µl Buffer
</LI></UL><UL><LI> 2,50 µl pr_constitutive_XbaI
</LI></UL><UL><LI> 2,50 µl pf_ constitutive_EcoR1
</LI></UL><UL><LI> 2,00 µl MgCl2
</LI></UL><UL><LI> 0,50 µl Polymerase S
</LI></UL><UL><LI> 35,5 µl H<SUB>2</SUB>O
</LI></UL><P>3. Program for 1. pBAD-mYFP Venus and 2. pEX_HisII: IGBIOB1 (30 cycles)
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Step
</TH><TH> Temperature
</TH><TH> Time
</TH></TR><TR><TD> Hot start
</TD><TD> 94°C
</TD><TD> hold
</TD></TR><TR><TD> Initial Denaturation
</TD><TD> 94°C
</TD><TD> 3min (180s)
</TD></TR><TR><TD> Denaturation
</TD><TD> 94°C
</TD><TD> 20s
</TD></TR><TR><TD> Annealing
</TD><TD> 44°C
</TD><TD> 40s
</TD></TR><TR><TD> Extension
</TD><TD> 72°C
</TD><TD> 73s
</TD></TR><TR><TD> Final Extension
</TD><TD> 72°C
</TD><TD> 600s
</TD></TR></TBODY></TABLE><P>4. Program for BBa_1763007 -constititive: IGBIOC1 (30 cycles: 2-step PCR, first run10x, second run 20x)
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Step
</TH><TH> Temperature
</TH><TH> Time
</TH></TR><TR><TD> Hot start
</TD><TD> 94°C
</TD><TD> hold
</TD></TR><TR><TD> Initial Denaturation
</TD><TD> 94°C
</TD><TD> 3min (180s)
</TD></TR><TR><TD> Denaturation
</TD><TD> 94°C
</TD><TD> 20s
</TD></TR><TR><TD> Annealing
</TD><TD> 43°C
</TD><TD> 40s
</TD></TR><TR><TD> Extension
</TD><TD> 72°C
</TD><TD> 45s
</TD></TR><TR><TD> Denaturation
</TD><TD> 94°C
</TD><TD> 20s
</TD></TR><TR><TD> Annealing
</TD><TD> 54°C
</TD><TD> 40s
</TD></TR><TR><TD> Extension
</TD><TD> 72°C
</TD><TD> 45s
</TD></TR><TR><TD> Final Extension
</TD><TD> 72°C
</TD><TD> 600s
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> See PCR results on agarose gel and do a gel extraction
</LI></UL><P><B>Agarose gel electrophoresis:</B></P><P><I>Gel 1</I></P><UL><LI> 5 µl PCR + 1 µl 6x loading dye
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> M </TD><TD> GeneRuler™ DNA Ladder Mix (diluted 1:10) </TD><TD> 10 </TD></TR><TR><TD> 1 </TD><TD> - </TD><TD> - </TD><TD> -
</TD></TR><TR><TD> 2 </TD><TD> PCR Arabinose promotor </TD><TD> 6 </TD><TD> ~1200
</TD></TR><TR><TD> 3 </TD><TD> PCR Arabinose promotor </TD><TD> 6 </TD><TD> ~1200
</TD></TR><TR><TD> 4 </TD><TD> - </TD><TD> - </TD><TD> -
</TD></TR><TR><TD> 5 </TD><TD> PCR Lac promotor </TD><TD> 6 </TD><TD> ~1400
</TD></TR><TR><TD> 6 </TD><TD> PCR Lac promotor </TD><TD> 6 </TD><TD> ~1400
</TD></TR></TBODY></TABLE><P><I>Gel 2</I></P><UL><LI> 5 µl PCR + 1 µl 6x loading dye
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> 1 </TD><TD> PCR Lambda promotor from BBa_I763007 </TD><TD> - </TD><TD> ?
</TD></TR><TR><TD> 2 </TD><TD> PCR Lambda promotor from BBa_I763007 </TD><TD> - </TD><TD> ?
</TD></TR><TR><TD> 3 </TD><TD> - </TD><TD> - </TD></TR><TR><TD> M </TD><TD> GeneRuler™ DNA Ladder Mix (diluted 1:10) </TD><TD> 10 </TD></TR></TBODY></TABLE><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparing_of_linearized_backbones_.28pSB1A3.2C_pSB1K3_and_pSB1T3.29_to_produce_vectors_for_further_applications">Preparing of linearized backbones (pSB1A3, pSB1K3 and pSB1T3) to produce vectors for further applications</SPAN></H3><P><B>Investigators:</B> Nadja, Nicole</P><P><B>Time:</B> 2011-08-02, </P><P><B>Aim:</B> Using linearized backbones for further transformations, iGEM restrictions sites are necessary </P><P><B>Idea:</B>Producing vectors with different antibiotic resistances (ampicillin, kanamycin and tetracyclin) and iGEM restriction sites to </P><P>1. have the choice which resistance you want and then the possibility to clone each gene of interest easily in the chosen vector </P><P>2. to clone inducible and constitutive promoter systems in it and use this as expression vector for further experiments
</P><P><B>Steps:</B></P><P>1. Restriction enzyme digestion</P><P>2. Ligation with reporter gene</P><P>3. Transformation and miniprep afterwards</P><P><B>Materials:</B></P><UL><LI> Linearized backbones from iGEM Registry of Standard biological parts (part of Spring 2010 DNA distribution kits) - only with EcoRI and PstI restriction sites
</LI></UL><DL><DD><UL><LI> pSB1A3 - ampicillin resistance
</LI></UL></DD></DL><DL><DD><UL><LI> pSB1T3 - tetracycline resistance
</LI></UL></DD></DL><DL><DD><UL><LI> pSB1K3 - kanamycin resistance
</LI></UL></DD></DL><DL><DD><UL><LI> pSB1C3 - we already got from KUK lab
</LI></UL></DD></DL><UL><LI> Manual ‘Spring 2011 DNA distribution’, see: <A href="http://partsregistry.org/Help:Spring_2011_DNA_distribution" class="external free" rel="nofollow">http://partsregistry.org/Help:Spring_2011_DNA_distribution</A> or <A href="/File:UP_Spring_2011_DNA_distribution.pdf" title="File:UP Spring 2011 DNA distribution.pdf">File:UP Spring 2011 DNA distribution.pdf</A></LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_linearized_plasmid_backbones_.28pSB1A3.2C_pSB1K3_and_pSB1T3.29">Restriction enzyme digestion of linearized plasmid backbones (pSB1A3, pSB1K3 and pSB1T3) </SPAN></H3><P><B>Investigators:</B> Nadja, Nicole</P><P><B>Time:</B> 2011-08-02, </P><P><B>Aim:</B>Producing vectors, which carry tetracycline, kanamycin and ampicillin resistance genes and have all iGEM restriction sites </P><P><B>Materials:</B></P><UL><LI> Linearized backbones from iGEM Registry of Standard biological parts (part of Spring 2010 DNA distribution kits) à only with EcoRI and PstI restriction sites
</LI></UL><DL><DD>pSB1A3, pSB1T3, pSB1K3
</DD></DL><UL><LI> Manual ‘Spring 2011 DNA distribution’, see: <A href="http://partsregistry.org/Help:Spring_2011_DNA_distribution" class="external free" rel="nofollow">http://partsregistry.org/Help:Spring_2011_DNA_distribution</A> or <A href="/File:UP_Spring_2011_DNA_distribution.pdf" title="File:UP Spring 2011 DNA distribution.pdf">File:UP Spring 2011 DNA distribution.pdf</A></LI></UL><UL><LI> NEB Buffer 2
</LI></UL><UL><LI> Purified BSA (NEB)
</LI></UL><UL><LI> EcoRI (NEB)
</LI></UL><UL><LI> pstI (NEB)
</LI></UL><UL><LI>dH<SMALL>2</SMALL>O
</LI></UL><P><B>Method:</B></P><P><B>1. Enzyme master mix</B></P><UL><LI> 5 µl NEB Buffer 2
</LI></UL><UL><LI> 0.5 µl BSA
</LI></UL><UL><LI> 0.5 µl of each EcoRI, PstI
</LI></UL><UL><LI> 18.5 µl dH<SMALL>2</SMALL>O
</LI></UL><UL><LI> mix 4 µl linearized plasmid backbone (25 ng/ µl) with 4 µl enzyme master mix
</LI></UL><P><B>2. Reaction conditions</B></P><UL><LI> 30 min at 37°C by 750 rpm
</LI></UL><UL><LI> 20 min at 80°C by 750 rpm
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Gel electrophoresis
</LI></UL><UL><LI> DNA extraction
</LI></UL><UL><LI> Ligation with reporter genes CFP and YFP
</LI></UL><UL><LI> Transformation
</LI></UL><UL><LI> Miniprep
</LI></UL><UL><LI> Sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Gel_electrophoresis_of_digested_.28linearized.29_plasmid_backbones_.28pSB1A3.2C_pSB1K3_and_pSB1T3.29">Gel electrophoresis of digested (linearized) plasmid backbones (pSB1A3, pSB1K3 and pSB1T3) </SPAN></H3><P><B>Investigators:</B> Nadja, Nicole</P><P><B>Time:</B> 2011-08-02, </P><P><B>Aim:</B>Producing vectors, which carry tetracycline, kanamycin and ampicillin resistance genes and have all iGEM restriction sites, through cloning CFP resp. YFP in the linearized plasmid backbones. </P><P><B>Materials:</B></P><UL><LI> linearized plasmids backbones (pSB1A3, pSB1K3, pSB1T3) digested with EcoRI and PstI
</LI></UL><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> DNA Ladder GeneRuler, 100bp plus (1:10) (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas)
</LI></UL><P><B>Method:</B></P><P><B>1. Production of one 1 % and one 1.5 % agarose gels</B></P><UL><LI> 1 % gel: 0.5 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> 1.5 % gel: 0.75 g in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to each gel
</LI></UL><P><B>2. Loading gels and running</B></P><UL><LI> Add 6 µl Loading dye to each 30 µl sample
</LI></UL><UL><LI> Gene Ruler DNA ladder
</LI></UL><UL><LI> Running conditions: 100 V, approx. 45 min
</LI></UL><P><B>3. Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH colspan="3"> gel 1 (1 %) </TH><TH colspan="3"> gel 2 (1.5 %)
</TH></TR><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> 1 </TD><TD> marker </TD><TD> marker </TD><TD> 6 </TD></TR><TR><TD> 2 </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> -</TD><TD> - </TD><TD> -
</TD></TR><TR><TD> 3 </TD><TD> pSB1T3 </TD><TD> 36 </TD><TD> 2206 </TD><TD> CFP </TD><TD> 36 </TD></TR><TR><TD> 4 </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> -
</TD></TR><TR><TD> 5 </TD><TD> pSB1A3 </TD><TD> 36 </TD><TD> 1862 </TD><TD> YFP </TD><TD> 36 </TD></TR><TR><TD> 6 </TD><TD> - </TD><TD> - </TD><TD> -</TD><TD> - </TD><TD> - </TD><TD> -
</TD></TR><TR><TD> 7 </TD><TD> pSB1K3 </TD><TD> 36 </TD><TD> 1978 </TD><TD> - </TD><TD> - </TD><TD> -
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><P>[[File:]] [[File:]]
</P><P>The bands were excised and purified using the NucleoSpin Extract II (Macherey-Nagel) extraction Kit.
</P><P><B>Further Tasks:</B></P><UL><LI> Ligation of CFP resp. YFP in each linearized backbone
</LI></UL><UL><LI> Transformation
</LI></UL><UL><LI> Minprep and production of glycerol stocks
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="52th_Labday_2011-08-03">52th Labday 2011-08-03</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_generated_pPDV089_.28strategy_2.29">Transformation of generated pPDV089 (strategy 2)</SPAN></H3><P><B>Investigators:</B> Leif</P><P><B>Aim:</B>amplification of pPDV089</P><P><B>Time:</B> 13:30-15:00</P><P><B>Method:</B></P><UL><LI> addition of 10 µl ligation reaction to XL1-blue cells</LI></UL><UL><LI> incubation 25 min on ice,</LI></UL><UL><LI> heat shock 45 sec at 42°C,</LI></UL><UL><LI> incubation 2 min on ice,</LI></UL><UL><LI> addition of 750 µl LB medium,</LI></UL><UL><LI> incubation 60 min at 37 °C and 750 rpm</LI></UL><UL><LI> plating on agar plates containing 100 µg/ml tetracyclin and 100 µg/µl ampicillin</LI></UL><UL><LI> storage over night at 37°C</LI></UL><P><B>Further tasks:</B> control cell clones
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_pBAD-mYFP_Venus_and_pEX_HisII_with_pSB1_K3_and_pSB1_A3">Ligation of pBAD-mYFP Venus and pEX_HisII with pSB1_K3 and pSB1_A3</SPAN></H3><P><B>Investigators:</B>Nadja, Nicole</P><P><B>Aim:</B>Build Biobricks inducible with IPTG or Arabinosis and detectible with YFP or CFP</P><P><B>Time:</B>2011-08-02,10:00-13:00</P><P><B>Materials:</B></P><UL><LI> T4 ligase
</LI></UL><UL><LI> 10x ligase buffer
</LI></UL><UL><LI> vectors: pSB1_K3 and pSB1_A3
</LI></UL><UL><LI> insert: pBAD-mYFP Venus and pEX_HisII
</LI></UL><P><B>Method:</B></P><UL><LI> total volume of 20µl
</LI></UL><UL><LI> 1µl T4 ligase
</LI></UL><UL><LI> 2µl 10x ligase buffer
</LI></UL><UL><LI> 5µl vector
</LI></UL><UL><LI> 3µl insert
</LI></UL><UL><LI> fill up to 20µl with H20
</LI></UL><UL><LI> incubation over night at 18°C</LI></UL><P><B>Further tasks:</B></P><UL><LI> over night culture
</LI></UL><UL><LI> miniprep
</LI></UL><UL><LI> sequenzing
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="53th_Labday_2011-08-04">53th Labday 2011-08-04</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="digestion_of_vector_pARW089_.28strategy_2.29">digestion of vector pARW089 (strategy 2)</SPAN></H3><P><B>Investigators:</B> Leif</P><P><B>Aim:</B> digestion of pARW089</P><P><B>Time:</B> 2011-08-04,10:00-12:00</P><P>Digestion of vector pARW089 with NarI (EheI isoschizomere) and AatII
</P><UL><LI> 20 µl sample
</LI></UL><UL><LI> NEB 10x buffer (2 µl)
</LI></UL><UL><LI> Buffer M 10x buffer (2 µl)
</LI></UL><UL><LI> 1 µl restriction enzyme NarI
</LI></UL><UL><LI> 1 µ restriction enzyme AatII
</LI></UL><UL><LI> 13 µl H2O
</LI></UL><UL><LI> 2 h at 37°C, then over night in the fridge
</LI></UL><P>Wrong buffer, so the experiment was rerun.
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="digestion_of_PCR_products_vector_pARW089_.28strategy_2.29">digestion of PCR products vector pARW089 (strategy 2)</SPAN></H3><P><B>Investigators:</B> Leif</P><P><B>Aim:</B> digestion of pARW089</P><P><B>Time:</B> 2011-08-04,10:00-12:00</P><P>Digestion of vector pARW089 with NarI (EheI isoschizomere) and AatII
</P><UL><LI> 20 µl sample
</LI></UL><UL><LI> NEB 10x buffer (2 µl)
</LI></UL><UL><LI> 1 µl restriction enzyme NarI
</LI></UL><UL><LI> 1 µ restriction enzyme AatII
</LI></UL><UL><LI> 13 µl H2O
</LI></UL><UL><LI> 2 h at 37°C, then over night in the fridge
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gel electrophoresis and purification of the two digested fragments
</LI></UL><UL><LI> Problem: Shaker was on, so no digest possible!
</LI></UL><P><B>Investigators:</B> Stefan</P><P><B>Aim:</B>digestion of pARW089</P><P><B>Time:</B> 2011-08-04,10:00-12:00</P><P>Digestion of vector pARW089 with NarI (EheI isoschizomere) and AatII
</P><UL><LI> 40 µl sample
</LI></UL><UL><LI> NEB 4 10x buffer (4 µl)
</LI></UL><UL><LI> 1 µl restriction enzyme NarI
</LI></UL><UL><LI> 1 µ restriction enzyme AatII
</LI></UL><UL><LI> 32 µl H2O
</LI></UL><UL><LI> 2 µl pARW089 (undiluted sample)
</LI></UL><P><I>2 h at 37°C, then heat over night in the fridge</I></P><P><B>Further Tasks:</B></P><UL><LI> gel electrophoresis and purification of the two digested fragments
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="54th_Labday_2011-08-05">54th Labday 2011-08-05</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_fragments_and_pJC354-NheI-143C-Xho_blaFL_GGH5_vector">Digestion of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV fragments and pJC354-NheI-143C-Xho_blaFL_GGH5 vector</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[4]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian </P><P><B>Materials:</B></P><P>NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI - 573bp
</P><P>HindIII_iGEM_AraC_NgoMIV - 1273bp
</P><P>pJC354-NheI-143C-Xho_blaFL_GGH5 vector (contains 14_3C cleavage site)
</P><P><B> Digestion protocol: </B></P><P>1: NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI (25ng/µl):
</P><UL><LI>30µl NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI
</LI></UL><UL><LI>1µl NgoMIV
</LI></UL><UL><LI>1µl BamHI
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>12.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>2: HindIII_iGEM_AraC_NgoMIV (12.5ng/µl)
</P><UL><LI>40µl HindIII_iGEM_AraC_NgoMIV
</LI></UL><UL><LI>1µl HindIII
</LI></UL><UL><LI>1µl NgoMIV
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>1.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>3: pJC354-NheI-143C-Xho_blaFL_GGH5 vector (270ng/µl)
</P><UL><LI>4µl pJC354-NheI-143C-Xho_blaFL_GGH5 vector (contains 14_3C cleavage site)
</LI></UL><UL><LI>1µl HindIII
</LI></UL><UL><LI>1µl BamHI
</LI></UL><UL><LI>5µl 10x buffer
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>38.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>--&gt;The reaction was allowed to proceed for 2h!
</P><P><B> Result: </B></P><P>Resolving of digested fragments (50µl) and digested vector (50µl) on 1.5% and 1% preparative agarose gels, respectively.
</P><P>1: NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI (band 5) and HindIII_iGEM_AraC_NgoMIV (band 3) fragments
</P><P>2: pJC354-NheI-143C-Xho_blaFL_GGH5 vector
</P><P><B>Summary:</B></P><P>The bands corresponding to digested fragments were excissed and purificated with PCR-purification unsing &quot;NucleoSpin ExtractII&quot;-KIT.
</P><P><B> Further tasks: </B></P><P>Triple ligation of digested fragments
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="digest_pSB1A3.2C_pSB1K3_.28clone_1-16.29">digest pSB1A3, pSB1K3 (clone 1-16) </SPAN></H3><P><B>Investigators:</B> Niels </P><P><B> Aim: </B> prove of Insert
</P><P><B> Digestion protocol: 16x </B></P><UL><LI>5µl DNA - pSB1A3,pSB1K3
</LI></UL><UL><LI>0,5µl XbaI
</LI></UL><UL><LI>0,5µl EcoRI
</LI></UL><UL><LI>2µl 10x buffer = NEB 4
</LI></UL><UL><LI>12.5µl pure water
</LI></UL><UL><LI>total: 20µl
</LI></UL><P>37°C for 1h
</P><P><B> Result: </B></P><P><B> Further tasks: </B> sequencing
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_pARW089_for_Phage_Display_strategy_II">Restriction enzyme digestion pARW089 for Phage Display strategy II</SPAN></H3><P><B>Investigators:</B> Leif</P><P><B>Time:</B> 2011-08-05,10:00-12:30 </P><P><B>Aim:</B> Restriction enzyme digestion of pARW089 according to the protocol by Nadine from the 2011-07-22 </P><P><B>Materials:</B></P><UL><LI> DNA: 5 µL
</LI></UL><UL><LI> NEB Buffer 4 (10x): 4 µL
</LI></UL><UL><LI> Enzyme AatII: 0.8 µL
</LI></UL><UL><LI> Enzyme NarI: 2 µL
</LI></UL><UL><LI> H2O: 28.2 µL
</LI></UL><P>Total Volume: 40 µL</P><P><B>Results:</B> Incubation of the digestion for 2 h with 750 rpm caused a breakdown of the restriction enzymes. The expreriment has to be repeated.</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_from_ligation_of_pARW089_.2B_mutated_mdnA_and_test_digest">Miniprep of overnight cultures from ligation of pARW089 + mutated mdnA and test digest</SPAN></H3><P><B>Investigators:</B> Jessica</P><P><B>Time:</B> 2011-08-05, 10:00-18:30</P><P><B> Aim: </B> DNA for sequencing and confirmation of insert </P><P><B>1. Miniprep:</B></P><UL><LI> 20 overnight cultures (1A-2a(2 h digest, clone a),1A-2b,2A-2a,...,5A-3b)
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> 1A-2a </TD><TD> 435.9
</TD></TR><TR><TD> 2A-2a </TD><TD> 480.1
</TD></TR><TR><TD> 3A-2a </TD><TD> 464.3
</TD></TR><TR><TD> 4A-2a </TD><TD> 362.7
</TD></TR><TR><TD> 5A-2a </TD><TD> 433.9
</TD></TR><TR><TD> 1A-3a </TD><TD> 528.3
</TD></TR><TR><TD> 2A-3a </TD><TD> 489.5
</TD></TR><TR><TD> 3A-3a </TD><TD> 487.3
</TD></TR><TR><TD> 4A-3a </TD><TD> 422.7
</TD></TR><TR><TD> 5A-3a </TD><TD> 537.3
</TD></TR><TR><TD> 1A-2b </TD><TD> 422
</TD></TR><TR><TD> 2A-2b </TD><TD> 297.4
</TD></TR><TR><TD> 3A-2b </TD><TD> 538.2
</TD></TR><TR><TD> 4A-2b </TD><TD> 427
</TD></TR><TR><TD> 5A-2b </TD><TD> 452
</TD></TR><TR><TD> 1A-3b </TD><TD> 375.5
</TD></TR><TR><TD> 2A-3b </TD><TD> 549.9
</TD></TR><TR><TD> 3A-3b </TD><TD> 307.8
</TD></TR><TR><TD> 4A-3b </TD><TD> 450.1
</TD></TR><TR><TD> 5A-3b </TD><TD> 462.3
</TD></TR></TBODY></TABLE><P><B>2. Preparation of glycerol stocks:</B></P><UL><LI> adding 300 µl glycerol to 700 µl culture</LI></UL><P><B>3. Digest:</B></P><UL><LI>2µl DNA (10 samples, only clone a)
</LI></UL><UL><LI>0,5µl NarI
</LI></UL><UL><LI>0,5µl AatII
</LI></UL><UL><LI>2µl 10x buffer NEB 4
</LI></UL><UL><LI>15µl H<SUB>2</SUB>O
</LI></UL><UL><LI>total: 20µl
</LI></UL><UL><LI>37°C for 1h
</LI></UL><P><B>4. Agarose gel electrophoresis:</B></P><UL><LI> 1% agarose gel
</LI></UL><UL><LI> 1 h at 115 V</LI></UL><P><I>Samples</I></P><UL><LI> 20 µl digest + 4 µl 6x loading dye
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> M </TD><TD> GeneRuler™ DNA Ladder Mix (diluted 1:10) </TD><TD> 10 </TD></TR><TR><TD> 1 </TD><TD> 1A-2a </TD><TD> 24 </TD><TD> 10296, 116
</TD></TR><TR><TD> 2 </TD><TD> 2A-2a </TD><TD> 24 </TD><TD> 10296, 116
</TD></TR><TR><TD> 3 </TD><TD> 3A-2a </TD><TD> 24 </TD><TD> 10296, 116
</TD></TR><TR><TD> 4 </TD><TD> 4A-2a </TD><TD> 24 </TD><TD> 10296, 116
</TD></TR><TR><TD> 5 </TD><TD> 1A-3a </TD><TD> 24 </TD><TD> 10296, 116
</TD></TR><TR><TD> 6 </TD><TD> 5A-2a </TD><TD> 24 </TD><TD> 10296, 116
</TD></TR><TR><TD> 7 </TD><TD> 2A-3a </TD><TD> 24 </TD><TD> 10296, 116
</TD></TR><TR><TD> 8 </TD><TD> 3A-3a </TD><TD> 24 </TD><TD> 10296, 116
</TD></TR><TR><TD> 9 </TD><TD> 4A-3a </TD><TD> 24 </TD><TD> 10296, 116
</TD></TR><TR><TD> 10 </TD><TD> 5A-3a </TD><TD> 24 </TD><TD> 10296, 116
</TD></TR></TBODY></TABLE><P><B> Result: </B></P><UL><LI> Inserts could be confirmed for samples from clone a</LI></UL><P><B> Further tasks: </B></P><UL><LI> sequencing to determine mutation rate
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_from_ligation_of_pBAD-mYFP_Venus_and_pEX_HisII_with_pSB1_K3_and_pSB1_A3">Miniprep of overnight cultures from ligation of pBAD-mYFP Venus and pEX_HisII with pSB1_K3 and pSB1_A3</SPAN></H3><P><B>Investigators:</B> Nicole, Nadja</P><P><B>Time:</B> 2011-08-05</P><P><B> Aim: </B> DNA for sequencing and confirmation of insert </P><P><B>Materials/Methods:</B></P><P><B>1. Miniprep:</B></P><UL><LI> 16 overnight cultures
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><P><B>2. Preparation of glycerol stocks:</B></P><UL><LI> adding 300 µl glycerol to 700 µl culture </LI></UL><P><B>Further tasks:</B></P><UL><LI> sequenzing
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="55th_Labday_2011-08-06">55th Labday 2011-08-06</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="2nd_Mutagenesis_of_TEV_proteases_to_remove_iGEM_restriction_sites_from_the_proteases_and_introduction_of_iGEM_restriction_sites">2nd Mutagenesis of TEV proteases to remove iGEM restriction sites from the proteases and introduction of iGEM restriction sites</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[5]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul</P><P><B>Aim:</B></P><UL><LI> Removal of iGEM restriction sites from proteases, amplifying protease fragments with iGEM restriction sites</LI></UL><P><B>Materials:</B></P><UL><LI>Plasmid 1: pET9d_Thrombin-CS_XbaI-TEV-Protease_BamHI
</LI></UL><UL><LI>Primers: (1) f_TEV_ACCAGC , r_TEV_iGEM_BahmHI (2) r_TEV_ACCAGC , f_TEV_AraFusion_NgoMIV
</LI></UL><P><B> Used method: </B></P><P>PCR
</P><UL><LI>Template: 1µl = 3,6 ng
</LI></UL><UL><LI>Nucleotides: 1µl of 10mM ready to use dNTP mix
</LI></UL><UL><LI>5µl 10x Amplification buffer S
</LI></UL><UL><LI>5µl 25mM MgCl2
</LI></UL><UL><LI>2,5µl primers = 25pmol absolute (2,5µl of each primer = 5µl per tube)
</LI></UL><UL><LI>32,5µl of pure water
</LI></UL><UL><LI>0,5µl TaqPol
</LI></UL><P>Program: iGEM001
</P><UL><LI>Denat: 3min 94°C
</LI></UL><UL><LI>5x:
</LI></UL><P>Denat: 45sec 94°C
</P><P>Anneal:45sec 53°C
</P><P>Extend:45sec 72°C
</P><UL><LI>25x:
</LI></UL><P>Denat: 45sec 94°C
</P><P>Anneal:45sec 60°C
</P><P>Extend:45sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P><B> Result: </B></P><P>Resolving of PCR products (50µl) on 2% preparative agarose gel
</P><P>Expected Fragments:
</P><P>TEV:
</P><UL><LI>TEV_mut_fragI: 360bp
</LI></UL><UL><LI>TEV_mut_fragII: 400bp
</LI></UL><P><B> Further going: </B></P><UL><LI>gel extraction with NucleoSpin Extract II
</LI></UL><UL><LI>Assembly PCR of purificated products to produce NgoMIV_iGEM_TEV-Protease_iGEM_BamHI
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="2nd_Annealing_of_TEV_mut_fragI_and_TEV_mut_fragII">2nd Annealing of TEV_mut_fragI and TEV_mut_fragII</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[6]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul</P><P><B>Materials:</B></P><P>TEV:
</P><UL><LI>TEV_mut_fragI: 360bp (40µl)
</LI></UL><UL><LI>TEV_mut_fragII: 400bp (40µl)
</LI></UL><P>Primers for PCR:
</P><UL><LI>f_TEV_AraFusion_NgoMIV
</LI></UL><UL><LI>r_TEV_iGEM_BamHI
</LI></UL><P><B> Used method: </B></P><P>1. 50µl of EACH fragment were purificated using a preparative agarose gel followed by extraction with &quot;NucleoSpin ExtractII&quot;-KIT.
</P><P>2. Annealing of purificated primers using PCR (program iGEMMED):
</P><UL><LI>Template: 5µl TEV_mut_fragI and 10µl TEV_mut_fragII = 1 reaction batch.
</LI></UL><UL><LI>Nucleotides: 1 µl of 10mM ready to use dNTP mix
</LI></UL><UL><LI>5µl 10x Amplification buffer S
</LI></UL><UL><LI>5µl 25mM MgCl2
</LI></UL><UL><LI>2,5µl primers = 25pmol absolute (2,5µl of each primer = 5µl per tube)
</LI></UL><UL><LI>18,5µl of pure water
</LI></UL><UL><LI>0,5µl TaqPol
</LI></UL><P>Program: iGEMMED
</P><UL><LI>Denat: 3min 94°C
</LI></UL><UL><LI>3x:
</LI></UL><P>Denat: 60sec 94°C
</P><P>Anneal:60sec 53°C
</P><P>Extend:60sec 72°C
</P><UL><LI>28x:
</LI></UL><P>Denat: 60sec 94°C
</P><P>Anneal:60sec 65°C
</P><P>Extend:60sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P><B> Result: </B></P><P>Resolving of PCR products (50µl) on 1,5% preparative agarose gel
</P><P>Expected Fragments
</P><P>for TEV:
</P><P>NgoMIV_iGEM_TEV-Protease_iGEM_BamHI - 760bp
</P><P>GelDoc documentation was not possible. 3 bands were visible: </P><P>1 band (1) at 700 - 800 bp (NgoMIV_iGEM_TEV-Protease_iGEM_BamHI),</P><P>1 band (2) at 500 - 600 bp (unexpected,) </P><P>1 band (3) at 300 - 400 bp (TEV_mut_fragI and TEV_mut_fragII)
</P><P>Summary:
</P><P>Ligation of TEV_mut_fragI and TEV_mut_fragII did work.
</P><P>Unexpected band at 500 - 600 bp.
</P><P><B> Further tasks: </B></P><P>Digestion of NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment and of the unexpected band with NgoMIV and BamHI.
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI_and_of_the_unexpected_band_from_the_2nd_annealing_TEV_PCR">Digestion of NgoMIV_iGEM_TEV-Protease_iGEM_BamHI and of the unexpected band from the 2nd annealing TEV PCR </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[7]</A>]</B></P><P><B>Investigators:</B>Sascha</P><P><B>Materials:</B></P><P>NgoMIV_iGEM_TEV-Protease_iGEM_BamHI - 760bp
</P><P>unexpected band - 500-600 bp
</P><P><B> Digestion protocol: </B></P><UL><LI>30µl NgoMIV_iGEM_TEV-Protease_iGEM_BamHI / 30µl unexpected band
</LI></UL><UL><LI>1µl NgoMIV
</LI></UL><UL><LI>1µl BamHI
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>12.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><UL><LI>5µl vector
</LI></UL><P>--&gt;The reaction was allowed to proceed for 2h and was resolving (50µl) on 1,5% preparative agarose gel.
</P><P><B> Result: </B></P><P>No visible bands on 1.5% preparative agarose gels.
</P><P><B> Further tasks: </B></P><P>New PCR for to produce new TEV-mut_fragI and TEV_mut_fragII fragments!!
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_PCR_of_mdnA_and_gene_III_for_phage_display_.28strategy_2.29">Repeated PCR of mdnA and gene III for phage display (strategy 2)</SPAN></H3><P><B>Investigator:</B> Sabine, Sandrina </P><P><B>Time:</B> 2011-08-06,12:00-14:00 </P><P><B>Aim:</B></P><UL><LI> amplification of mdnA with NarI and AgeI restriction sites (strategy 2)</LI></UL><UL><LI> amplificate GeneIII with NgoMIV and AatII restriction sites (strategy 2)</LI></UL><P><B>Primer:</B></P><UL><LI> primer: pf_mdnA_iGEM_EheI and pr_mdnA_iGEM_AatII (mdnaA, strategy 2)</LI></UL><UL><LI> primer: pf_geneIII_NgoMIV and pr_geneIII_iGEM_AatII (geneIII, strategy 2)</LI></UL><P><B>Reaction Components:</B></P><UL><LI> 5 µl Vector pARW089/Vector 100blaKDIR
</LI></UL><UL><LI> 0,25 µl Taq Polymerase S (BioScience)
</LI></UL><UL><LI> 1 µl dNTPs
</LI></UL><UL><LI> 1 µl per primer
</LI></UL><UL><LI> 5 µl 10x PCR Buffer S
</LI></UL><UL><LI> 37,75 µl DNase free water
</LI></UL><P><B>Further tasks:</B></P><UL><LI> purification
</LI></UL><UL><LI> digestion
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="digestion_of_PCR_products_.28strategy_2.29">digestion of PCR products (strategy 2)</SPAN></H3><P><B>Investigator:</B> Sabine, Sandrina</P><P><B>Aim:</B></P><P>digestion of mdnA and gene III for getting an mdnA-geneIII fusion gene with rfc25 restrition sites after ligation
</P><P><B>Time:</B> 2011-08-06,14:00-15:30</P><P><B>digestion enzymes:</B></P><UL><LI> digestion of mdnA (strategy 2) with NarI and AgeI
</LI></UL><UL><LI> digestion of geneIII (strategy 2) with NgoMIV and AatII
</LI></UL><P><B>reaction components:</B></P><UL><LI> 4 µl NEB 10x buffer
</LI></UL><UL><LI> 1 µl per restriction enzyme
</LI></UL><UL><LI> 30 µl PCR product
</LI></UL><UL><LI> 4 µl H²O
</LI></UL><P><B>reaction conditions:</B></P><UL><LI> 1 h for PCR fragments
</LI></UL><UL><LI> 37°C NarI, AgeI, NgoMIV and AatII digestion
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gel electrophoresis for purification of the digested fragments and vectors
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="gel_electrophoresis_of_digested_fragments_and_digested_pARW089">gel electrophoresis of digested fragments and digested pARW089</SPAN></H3><P><B>Investigator:</B> Sandrina</P><P><B>Aim:</B></P><UL><LI> control and purification of digested PCR fragments
</LI></UL><UL><LI> control and purification of digested pARW089 (2011-08-05)
</LI></UL><P><B>Time:</B> 2011-08-06,14:00-18:00</P><P><B>Results:</B></P><UL><LI> mdnA (NarI and AgeI, stategy 2): ca 200 bp, but estimation difficult, because no GelDoc available (weekend)
</LI></UL><UL><LI> geneIII (NgoMIV and AatII): no band
</LI></UL><UL><LI> pARW089 : ca 10 kb, but estimation difficult, because no GelDoc available (weekend)
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> repeat PCR of geneIII
</LI></UL><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="prepare_samples_for_sequencing_.28pARW089_.2B_mutated_mdnA.29"> prepare samples for sequencing (pARW089 + mutated mdnA) </SPAN></H3><P><B>Investigators:</B> Niels </P><P><B> Aim: </B> mdnA_1 sequencing by eurofins mwg|operon
</P><P><B> guidelines </B> (eurofins mwg|operon)
</P><P><B> Primer :2 pmol/µl </B> (10 µl each sample)
</P><UL><LI> 2,4 µl (100 µM) sf_mdnA_1
</LI></UL><UL><LI> 117,6 µl water
</LI></UL><P><B> Sample: 70 ng/µl </B> (15 µl total)
</P><UL><LI> sample (cDNA ng/µl) - DNA µl ( ad water 15 µl)
</LI></UL><UL><LI> 1A-2a (435,9) - 2,4
</LI></UL><UL><LI> 2A-2a (480,1) - 2,2
</LI></UL><UL><LI> 3A-2a (464,3) - 2,3
</LI></UL><UL><LI> 4A-2a (362,7) - 2,9
</LI></UL><UL><LI> 5A-2a (433,9) - 2,4
</LI></UL><UL><LI> 1A-3a (528,3) - 1,99
</LI></UL><UL><LI> 2A-3a (489,5) - 2,15
</LI></UL><UL><LI> 3A-3a (487,3) - 2,15
</LI></UL><UL><LI> 4A-3a (422,7) - 2,48
</LI></UL><UL><LI> 5A-3a (537,3) - 1,95
</LI></UL><P><B> Legend: </B></P><UL><LI><B> example: 1A-2a </B></LI></UL><UL><LI><B>1</B>A-2a
</LI></UL><UL><LI> sample 1-5 : different Mn-concentration (error-pone PCR)
</LI></UL><UL><LI> 1<B>A</B>-2a
</LI></UL><UL><LI> sample A-C : different error-pone PCR- appendage
</LI></UL><UL><LI> 1A-<B>2</B>a
</LI></UL><UL><LI> sample 2 or 3: 2h(or 3h) digest of pARW089 with AatII / NarI
</LI></UL><UL><LI> 1A-2<B>a</B></LI></UL><UL><LI> sample a or b : 1. clone(a) or 2. clone(b) from the same plate with: <I>E. coli</I> XL1 blue transformed with ligation of vector pARW089 and insert (error pone - PCR )
</LI></UL><P><B> Further tasks: </B> sequencing
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="56th_Labday_2011-08-07">56th Labday 2011-08-07</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="PCR_purification_of_TEV_mut_fragI_und_II">PCR purification of TEV_mut_fragI und II</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[8]</A>]</B></P><P><B>Investigators:</B> Stefan</P><P><B>Aim:</B></P><UL><LI> PCR purification</LI></UL><P><B>Materials:</B></P><UL><LI> Promega PCR clean-Up System
</LI></UL><UL><LI> TEV_mut_fragI (360bp), TEV_mut_fragII (400bp)
</LI></UL><P><B> Used method: </B></P><P><B> Further going: </B></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="gel_electrophoresisTEV_mut_fragI_and_II">gel electrophoresisTEV_mut_fragI and II </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[9]</A>]</B></P><P><B>Investigators:</B> Stefan</P><P><B>Aim:</B></P><UL><LI> TEV_mut_fragI (360bp), TEV_mut_fragII (400bp)</LI></UL><P><B>Materials:</B></P><UL><LI> use small raq at 80V
</LI></UL><P><B> Used method: </B></P><UL><LI>no picture could be taken due to the weak UV signal
</LI></UL><P><B> Further going: </B></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Annealing_PCR_TEV_mut_fragI_and_II"> Annealing PCR TEV_mut_fragI and II </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[10]</A>]</B></P><P><B>Investigators:</B> Stefan</P><P><B>Aim:</B></P><UL><LI> TEV_mut_fragI (360bp), TEV_mut_fragII (400bp)</LI></UL><P><B>Materials:</B></P><P>1. 50µl of EACH fragment were purificated using a preparative agarose gel followed by extraction with &quot;NucleoSpin ExtractII&quot;-KIT.
</P><P>2. Annealing of purificated primers using PCR (program iGEMMED):
</P><UL><LI>Template: 5µl TEV_mut_fragI and 10µl TEV_mut_fragII = 1 reaction batch.
</LI></UL><UL><LI>Nucleotides: 1 µl of 10mM ready to use dNTP mix
</LI></UL><UL><LI>5µl 10x Amplification buffer S
</LI></UL><UL><LI>5µl 25mM MgCl2
</LI></UL><UL><LI>2,5µl primers = 25pmol absolute (2,5µl of each primer = 5µl per tube)
</LI></UL><UL><LI>18,5µl of pure water
</LI></UL><UL><LI>0,5µl TaqPol
</LI></UL><P>Program: iGEMMED
</P><UL><LI>Denat: 3min 94°C
</LI></UL><UL><LI>3x:
</LI></UL><P>Denat: 60sec 94°C
</P><P>Anneal:60sec 53°C
</P><P>Extend:60sec 72°C
</P><UL><LI>28x:
</LI></UL><P>Denat: 60sec 94°C
</P><P>Anneal:60sec 65°C
</P><P>Extend:60sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P><B> Used method: </B></P><P><B> Further going: </B></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_and_pJC354-NheI-143C-Xho_blaFL_GGH5_vector"> Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV and pJC354-NheI-143C-Xho_blaFL_GGH5 vector </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[11]</A>]</B></P><P><B>Investigators:</B> Stefan</P><P><B>Aim:</B>Triple-ligation of
</P><P>NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI (573bp), HindIII_iGEM_AraC_NgoMIV (1273bp) and pJC354-NheI-143C-Xho_blaFL_GGH5 vector
</P><P><B>Materials:</B></P><UL><LI> 3 µL 14_3C fragment
</LI></UL><UL><LI> 3 µL AraC fragment
</LI></UL><UL><LI> 1 µL pJC354
</LI></UL><UL><LI> 2 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> 10 µL H20
</LI></UL><P><B> Used method: </B></P><P>ligation at room temperatur for 3h
</P><P><B> Results: </B></P><P>band was very blurry on the gel, probably of contaminated running buffer
</P><P><B> Further going: </B></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_Ligation_of_pJC_AraC_and_14_3C_in_E._coli_XL1_blue">Transformation of Ligation of pJC AraC and 14_3C in <I>E. coli</I> XL1 blue </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[12]</A>]</B></P><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[13]</A>]</B></P><P><B>Investigators:</B> Stefan</P><P><B>Aim:</B>transform the ligation into <I>E. coli</I> XL1 blue</P><P><B>Materials:</B></P><P>protocol:
</P><P>addition of 10 µl ligation reaction to cells (XL1-blue, tet-resistance) in 1.5 ml Eppi,</P><P>incubation 25 min on ice,</P><P>heat shock 45 sec at 42°C,</P><P>incubation 2 min on ice,</P><P>addition of 750 µl LB medium,</P><P>incubation at 37 °C for 60 min in Eppendorf thermomixer at 750 rpm,</P><P>plating on LB medium with 1,5 % agar, 100 µg/ml chloramphenicol, </P><P>storage over night at 37°C</P><P><B> Result: </B></P><P>no colonies
</P><P><B> Further going: </B></P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_PCR_gene_III_for_phage_display_.28strategy_2.29">Repeated PCR gene III for phage display (strategy 2)</SPAN></H3><P><B>Investigator:</B> Sabine </P><P><B>Time:</B> 2011-08-07,11:00-13:00 </P><P><B>Aim:</B></P><UL><LI> amplificate GeneIII with NgoMIV and AatII restriction sites (strategy 2)</LI></UL><P><B>Primer:</B></P><UL><LI> primer: pf_geneIII_NgoMIV and pr_geneIII_iGEM_AatII (geneIII, strategy 2)</LI></UL><P><B>Reaction Components:</B></P><UL><LI> 5 µl Vector pARW089 / pak100blaKDIR
</LI></UL><UL><LI> 0,25 µl Taq Polymerase S (BioScience)
</LI></UL><UL><LI> 1 µl dNTPs
</LI></UL><UL><LI> 1 µl per primer
</LI></UL><UL><LI> 5 µl 10x PCR Buffer S
</LI></UL><UL><LI> 37,75 µl DNase free water
</LI></UL><UL><LI> purification of PCR fragments with QIAquick Gel Extraction Kit (250)
</LI></UL><P><B>Further tasks:</B> digestion
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="digestion_of_PCR_products_and_vector_.28strategy_2.29">digestion of PCR products and vector (strategy 2)</SPAN></H3><P><B>Investigator:</B> Sabine</P><P><B>Aim:</B></P><UL><LI> digestion of mdnA and gene III for getting an mdnA-geneIII fusion gene with rfc25 restrition sites after ligation
</LI></UL><UL><LI> digestion of pARW089 for ligation of mdnA/geneIII fusion gene into it (strategy 2)</LI></UL><P><B>Time:</B> 2011-08-06,12:30-15:00</P><P><B>digestion enzymes:</B></P><UL><LI> digestion of mdnA (strategy 2) with NarI and AgeI
</LI></UL><UL><LI> digestion of geneIII (strategy 2) with NgoMIV and AatII
</LI></UL><UL><LI> digestion of pARW089 (strategy 2) with NarI and AatII
</LI></UL><P><B>reaction components:</B></P><UL><LI> 5 µl NEB 10x buffer
</LI></UL><UL><LI> 1 µl per restriction enzyme
</LI></UL><UL><LI> 40 µl PCR product / 5 µl vector
</LI></UL><UL><LI> ad 50 µl water
</LI></UL><P><B>reaction conditions:</B></P><UL><LI> 1 h for PCR fragments
</LI></UL><UL><LI> 3 h for plasmids
</LI></UL><UL><LI> 37°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gel electrophoresis for purification of the digested fragments and vectors
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="prepare_samples_for_sequencing_.28pSB1A3.2C_pSB1K3.29"> prepare samples for sequencing (pSB1A3, pSB1K3) </SPAN></H3><P><B>Investigators:</B> Niels </P><P><B> Aim: </B> sequencing by eurofins mwg|operon
</P><P><B> guidelines </B> (eurofins mwg|operon)
</P><P><B> Primer :2 pmol/µl </B> (10 µl each sample)
</P><P><B> Sample: 70 ng/µl </B> (15 µl total)
</P><UL><LI> sample (cDNA ng/µl) - DNA µl ( ad water 15 µl)
</LI></UL><UL><LI> 1A3 CFP (204,5) - 5,13
</LI></UL><UL><LI> 1K3 CFP (216,2) - 4,86
</LI></UL><UL><LI> 1A3 YFP (235,7) - 4,45
</LI></UL><UL><LI> 1K3 YFP (213,6) - 4,92
</LI></UL><P><B> Further tasks: </B> sequencing
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="57th_Labday_2011-08-08">57th Labday 2011-08-08</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="test_digest_of_pSB1K3">test digest of pSB1K3 </SPAN></H3><P><B>Time:</B> 2011-08-08,11:00-16:00</P><P><B>Investigators:</B> Vanessa, Steffi, Katharina, Nadine </P><P><B> Aim: </B> prove of Insert (CFP or YFP)</P><P><B>Materials:</B></P><UL><LI> pSB1K3 (clone 1 and 4 from last week ???)
</LI></UL><UL><LI> EcoRI, PstI, XbaI, HindIII, AatII
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> DNA Ladder Mix (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas)
</LI></UL><P><B> Digestion protocol (XbaI, EcoRI) </B></P><UL><LI>5 µl DNA - pSB1K3 YFP or CFP (clone 4 or 1, respectively)
</LI></UL><UL><LI>0.5 µl XbaI
</LI></UL><UL><LI>0.5 µl EcoRI
</LI></UL><UL><LI>2 µl 10x buffer = NEB 4
</LI></UL><UL><LI>12.5 µl pure water
</LI></UL><P><B> Digestion protocol (PstI, EcoRI) </B></P><UL><LI>5 µl DNA - pSB1K3+YFP or CFP (clone 4 or 1, respectively)
</LI></UL><UL><LI>0.5 µl EcoRI
</LI></UL><UL><LI>0.5 µl PstI
</LI></UL><UL><LI>2 µl 10x buffer = NEB 4
</LI></UL><UL><LI>12.5 µl pure water
</LI></UL><P><B> Digestion protocol (AatII, HindIII) </B></P><UL><LI>5 µl DNA - pSB1K3 YFP or CFP (clone 4 or 1, respectively)
</LI></UL><UL><LI>0.5 µl AatII
</LI></UL><UL><LI>0.5 µl HindIII
</LI></UL><UL><LI>2 µl 10x buffer = NEB 4
</LI></UL><UL><LI>12.5 µl pure water
</LI></UL><UL><LI>total: 20µl
</LI></UL><P>37°C for 3h
</P><P><B> Production of one 1 % </B></P><UL><LI> 1 % gel: 0.5 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to each gel
</LI></UL><P><B>Loading gels and running</B></P><UL><LI> Add 4 µl Loading dye to each 20 µl sample
</LI></UL><UL><LI> 15 µl DNA Ladder Mix
</LI></UL><UL><LI> Running conditions: 100 V, approx. 45 min
</LI></UL><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> 1 </TD><TD> marker </TD></TR><TR><TD> 2 </TD><TD> pSB1K3+CFP (clone1), EcoRI, PstI </TD><TD> 24 </TD><TD> 812, 2163
</TD></TR><TR><TD> 3 </TD><TD> pSB1K3+CFP (clone1), XbaI, EcoRI </TD><TD> 24</TD><TD> 7, 2968
</TD></TR><TR><TD> 4 </TD><TD> pSB1K3+CFP (clone1), AatII, HindIII </TD><TD> 24</TD><TD> 721, 2254
</TD></TR><TR><TD> 2 </TD><TD> pSB1K3+YFP (clone4), EcoRI, PstI </TD><TD> 24 </TD><TD> 789, 2163
</TD></TR><TR><TD> 3 </TD><TD> pSB1K3+YFP (clone4), XbaI, EcoRI </TD><TD> 24</TD><TD> 7, 2945
</TD></TR><TR><TD> 4 </TD><TD> pSB1K3+YFP (clone4), AatII, HindIII </TD><TD> 24</TD><TD> 721, 2231
</TD></TR></TBODY></TABLE><P><B> Result: </B></P><P><A href="/File:UP_AG_digest_pSB1K3_YFP_CFP_EcoRI_XbaI_2011-08-08_JE_001.jpg" title="File:UP AG digest pSB1K3 YFP CFP EcoRI XbaI 2011-08-08 JE 001.jpg">300px</A></P><P><B> Further tasks: </B> repeat the test digest tomorrow
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="sequencing_pARW089_.2B_mutated_mdnA"> sequencing pARW089 + mutated mdnA </SPAN></H3><P><B>Investigators:</B> Niels,Steffi </P><P><B> Aim: </B> sequencing by eurofins mwg|operon
</P><P><B> samples </B> prepared at 06.08.2011
</P><P><B> Primer :2 pmol/µl </B> 120 µl
</P><P>sf_mdnA_1
</P><P><B> pARW089 + mutated mdnA </B> (15 µl total)
</P><UL><LI> sample ID : 2h
</LI></UL><UL><LI> 1A-2a - AKM001W020
</LI></UL><UL><LI> 2A-2a - AKM001W021
</LI></UL><UL><LI> 3A-2a - AKM001W022
</LI></UL><UL><LI> 4A-2a - AKM001W023
</LI></UL><UL><LI> 5A-2a - AKM001W024
</LI></UL><UL><LI> sample ID : 3h
</LI></UL><UL><LI> 1A-3a - AKM001W025
</LI></UL><UL><LI> 2A-3a - AKM001W026
</LI></UL><UL><LI> 3A-3a - AKM001W027
</LI></UL><UL><LI> 4A-3a - AKM001W028
</LI></UL><UL><LI> 5A-3a - AKM001W029
</LI></UL><P><B> Results (arrived on 2011-08-11): </B></P><UL><LI> no mutations could be found in mdnA
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Assembly_PCR_of_TEV_frag_I_and_TEV_frag_II"> Assembly PCR of TEV_frag_I and TEV_frag_II </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[14]</A>]</B></P><P><B>Investigators:</B> Sebastian, Stefan</P><P><B>Aim:</B> Assembly PCR to mutate the EcoRI site
</P><P><B>Materials:</B></P><UL><LI>Primer:
</LI></UL><UL><LI> 2,5 µLf_TEV_AraFusion_NgoMIV (0.5 µmol stock)
</LI></UL><UL><LI> 2,5 µLr_TEV_iGEM_BamHI (0.5 µmol stock)
</LI></UL><UL><LI> 1 µL Fragment TEV I (approx. 2.5 ng)
</LI></UL><UL><LI> 1 µL Fragment TEV II (approx. 2.5 ng)
</LI></UL><UL><LI> 5 µL 10x polymerase buffer
</LI></UL><UL><LI> 5 µL 25 mM MgCl2
</LI></UL><UL><LI> 1 µL dNTP
</LI></UL><UL><LI> 0.5 µL T4 polymerase(Fermentas)
</LI></UL><UL><LI> 18.5 µL H20
</LI></UL><P><B> Used method: </B></P><P>Program: iGEMMED
</P><UL><LI>Denat: 3min 94°C
</LI></UL><UL><LI>3x:
</LI></UL><P>Denat: 60sec 94°C
</P><P>Anneal:60sec 53°C
</P><P>Extend:60sec 72°C
</P><UL><LI>28x:
</LI></UL><P>Denat: 60sec 94°C
</P><P>Anneal:60sec 65°C
</P><P>Extend:60sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P><B> Further going: </B></P><P>analytical gel electrophoresis to confirm the correct size of bands
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="design_primer_for_biobrick_-_mdnABC.2CmdnB.2CmdnC.2CmdnDE.2CmdnD.2CmdnE"> design primer for biobrick - mdnABC,mdnB,mdnC,mdnDE,mdnD,mdnE </SPAN></H3><P><B>Investigators:</B> Katharina, Niels </P><P><B> Aim: </B> design and order primer
</P><P><B> Primer sequence </B></P><UL><LI> pf_mdnABC_EcoRI_NotI_XbaI
</LI></UL><P>GAATTCGCGGCCGCTTCTAGATGGCATATCCCAACGATC
</P><UL><LI> pf_mdnB_EcoRI_NotI_XbaI
</LI></UL><P>GAATTCGCGGCCGCTTCTAGATGAAAGAATCGCCTAAAGTTG
</P><UL><LI> pf_mdnC_EcoRI_NotI_XbaI
</LI></UL><P>GAATTCGCGGCCGCTTCTAGATGACCGTTTTAATTGTTAC
</P><UL><LI> pf_mdnE_EcoRI_NotI_XbaI
</LI></UL><P>CAATCATCATATAACTCCGTAGATCTTCGCCGGCGCTTAAG
</P><UL><LI> pf_mdnD_EcoRI_NotI_XbaI
</LI></UL><P>GTCAAAAAGGTCACGAAAGTAGATCTTCGCCGGCGCTTAAG
</P><UL><LI> pr_mdnABC_SpeI_NotI_PstI
</LI></UL><P>GAAATCCTAGTTAACTCATAATACTAGTAGCGGCCGCTGCAG
</P><UL><LI> pr_mdnE_SpeI_NotI_PstI
</LI></UL><P>CTGCAGCGGCCGCTACTAGTAGATATAAGAGTGGGTAAAATTC
</P><UL><LI> pr_mdnDE_SpeI_NotI_PstI
</LI></UL><P>CTGCAGCGGCCGCTACTAGTATCAGCAAACCCTACTTAATTTC
</P><UL><LI> pr_mdnB_SpeI_NotI_PstI
</LI></UL><P>GCGATCGCTGATTTTTTAGTTACTAGTAGCGGCCGCTGCAG
</P><P><B> ordered my sigma </B></P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_mdnA.2FGeneIII-fusion_gene_into_pARW089_for_phage_display_and_transformation_of_competent_cells.28strategy_2.29">Ligation of mdnA/GeneIII-fusion gene into pARW089 for phage display and transformation of competent cells(strategy 2)</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine </P><P><B>Aim:</B></P><UL><LI>ligation of mdnA-geneIII fusiongene into pARW089 with digested fragments (see 2011-08-07)</LI></UL><UL><LI>amplification of generated plasmids by transformation
</LI></UL><P><B>Time:</B> 10:00-18:00</P><P><B>Method:</B></P><UL><LI> ligation-samples from 2011-08-07;</LI></UL><P><B>Protocol: </B></P><UL><LI> 5 µl (ca 11 ng) digested vector pARW089 (NarI, AatII)
</LI></UL><UL><LI> 1 µl (ca 270 ng) fusion gene mdnA/geneIII
</LI></UL><UL><LI> 2 µl 10x T4 Ligase Buffer
</LI></UL><UL><LI> 1 µl T4 Ligase
</LI></UL><UL><LI> 10 µl water
</LI></UL><P>5 h at 16°C and 1 h at room temperature
</P><UL><LI> two bands were observed after mdnA-geneIII ligation (red boxes)--&gt; ligation with pARW089 was tried with both bands
</LI></UL><P>transformation:
</P><P>protocol:
</P><P>addition of 10 µl ligation reaction to cells (XL1-blue, tet-resistance) in 1.5 ml Eppi,</P><P>incubation 25 min on ice,</P><P>heat shock 45 sec at 42°C,</P><P>incubation 2 min on ice,</P><P>addition of 750 µl LB medium,</P><P>incubation at 37 °C for 60 min in Eppendorf thermomixer at 750 rpm,</P><P>plating on LB medium with 1,5 % agar, 100 µg/ml ampicillin, 100 µg/µl tetracyclin</P><P>storage over night at 37°C</P><P><B>Further tasks:</B> test digestion
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="58th_Labday_2011-08-09">58th Labday 2011-08-09</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="test_digest_of_pSB1K3.2BYFP_or_CFP_and_pSB1A3.2BYFP_or_CFP">test digest of pSB1K3+YFP or CFP and pSB1A3+YFP or CFP </SPAN></H3><P><B>Time:</B> 2011-08-09,9:00-15:00</P><P><B>Investigators:</B> Nadine, Vanessa, Steffi, Laura </P><P><B> Aim: </B> prove of Insert (CFP or YFP)
</P><P><B>Materials:</B></P><UL><LI> pSB1K3 (clone 1 and 4 from last week ???), pSB1AK (clone 11 and 12 from last week ???)
</LI></UL><UL><LI> EcoRI, XbaI, HindIII, AatII, HaeII, BglI
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> Buffer 2
</LI></UL><UL><LI>100x BSA
</LI></UL><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> DNA Ladder Mix (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas)
</LI></UL><P><B> Digestion protocol (XbaI, EcoRI) (4x)</B></P><UL><LI>5 µl DNA - pSB1K3+YFP or CFP (clone 4 or 1, respectively) or pSB1A3+YFP or CFP (clone 12 or 11, respectively)
</LI></UL><UL><LI>0.5 µl XbaI
</LI></UL><UL><LI>0.5 µl EcoRI
</LI></UL><UL><LI>2 µl 10x buffer = NEB 4
</LI></UL><UL><LI>12.5 µl pure water
</LI></UL><P><B> Digestion protocol (AatII, HindIII) </B></P><UL><LI>5 µl DNA - pSB1K3 YFP or CFP (clone 4 or 1, respectively) (2x)
</LI></UL><UL><LI>0.5 µl AatII
</LI></UL><UL><LI>0.5 µl HindIII
</LI></UL><UL><LI>2 µl 10x buffer = NEB 4
</LI></UL><UL><LI>12.5 µl pure water
</LI></UL><UL><LI>total: 20µl
</LI></UL><P><B> Digestion protocol (HaeII, BglI) (4x)</B></P><UL><LI>5 µl DNA pSB1A3+YFP or CFP (clone 12 or 11, respectively)
</LI></UL><UL><LI>0.5 µl HaeII
</LI></UL><UL><LI>0.5 µl BglI
</LI></UL><UL><LI>2 µl 10x buffer = NEB 2
</LI></UL><UL><LI> 0.2 µl 100xBSA
</LI></UL><UL><LI>12.3 µl pure water
</LI></UL><P>37°C for 2h
</P><P><B> Production of one 1 % agarose gel</B></P><UL><LI> 1 % gel: 0.5 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to each gel
</LI></UL><P><B>Loading gels and running</B></P><UL><LI> Add 4 µl Loading dye to each 20 µl sample
</LI></UL><UL><LI> 15 µl DNA Ladder Mix
</LI></UL><UL><LI> Running conditions: 100 V, approx. 45 min
</LI></UL><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> 1 </TD><TD> marker </TD></TR><TR><TD> 2 </TD><TD> pSB1K3+CFP (clone1), not digested</TD><TD> 24</TD></TR><TR><TD> 3 </TD><TD> pSB1K3+CFP (clone1), XbaI, EcoRI </TD><TD> 24</TD><TD> 7, 2968
</TD></TR><TR><TD> 4 </TD><TD> pSB1K3+CFP (clone1), AatII, HindIII </TD><TD> 24</TD><TD> 721, 2254
</TD></TR><TR><TD> 5 </TD><TD> pSB1K3+YFP (clone4), XbaI, EcoRI </TD><TD> 24</TD><TD> 7, 2945
</TD></TR><TR><TD> 6 </TD><TD> pSB1K3+YFP (clone4), AatII, HindIII </TD><TD> 24</TD><TD> 721, 2231
</TD></TR><TR><TD> 7 </TD><TD> pSB1K3+YFP (clone4), not digested </TD><TD> 24</TD></TR><TR><TD> 8 </TD><TD> pSB1A3+CFP (clone11), not digested </TD><TD> 24</TD></TR><TR><TD> 9 </TD><TD> pSB1A3+CFP (clone11), XbaI, EcoRI </TD><TD> 24</TD><TD> 15, 2911
</TD></TR><TR><TD> 10 </TD><TD> pSB1A3+CFP (clone11), HaeII, BglI </TD><TD> 24</TD><TD> 1359, 1567
</TD></TR><TR><TD> 11 </TD><TD> pSB1A3+YFP (clone12), XbaI, EcoRI </TD><TD> 24</TD><TD> 15, 2888
</TD></TR><TR><TD> 12 </TD><TD> pSB1A3+YFP (clone12), HaeII, BglI </TD><TD> 24</TD><TD> 765, 924, 1188
</TD></TR></TBODY></TABLE><P><B> Result: </B></P><P><B> Further tasks: </B> repeat from beginning: w/ linearized backbones pSB1K3, pSB1A3 and pSB1T3
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Gel_purification_of_amplificated_TEV_protease"> Gel purification of amplificated TEV protease </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[15]</A>]</B></P><P><B>Investigators:</B> Sebastian</P><P><B>Aim:</B> Clean up a pure fraction of TEV protease without iGEM-RS in the nucleotide sequence
</P><P><B>Materials:</B></P><P>PCR and gel purification kit purchased by Promega (Wizard SV GEL and PCR Clean-Up System)
</P><P><B> Used method: </B></P><P>Done as described in the manual</P><P><B>Results:</B></P><P>NgoMIV_TEV_iGEM_BamHI with a concentration of 6,9 ng/µl</P><P><B> Further going: </B></P><P>Amplification of TEV protease via PCR and digest with BamHI and NgoMIV
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Ampilifcation_of_NgoMIV_TEV_iGEM_BamHI_protease_via_PCR"> Ampilifcation of NgoMIV_TEV_iGEM_BamHI protease via PCR </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[16]</A>]</B></P><P><B>Investigators:</B> Sebastian, Niels</P><P><B>Aim:</B> Amplification of the TEV protease for digest with BamHI and NgoMIV</P><P><B>Materials:</B></P><P>1 µl Template - NgoMIV_TEV_iGEM_BamHI (6,9 ng/µl)</P><P>2,5 µl Primer 1 - f_TEV_AraFusion_NgoMIV (0,5 µM)</P><P>2,5 µl Primer 2 - r_TEV_iGEM_BamHI (0,5 µM)</P><P>5 µl 10x Reaction Buffer (Fermentas)</P><P>1 µl 10 mM dNTP's (Fermentas)</P><P>5 µl 25 µM MgCl2</P><P>0,5 µl DNA-Polymerase (Fermentas)</P><P>32,5 µl water</P><P><B> Used method: </B></P><P>Program: iGEM004
</P><UL><LI>Denat: 3min 94°C
</LI></UL><UL><LI>30x:
</LI></UL><P>Denat: 60sec 94°C
</P><P>Anneal:60sec 65°C
</P><P>Extend:60sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P><B> Further going: </B></P><P>PCR-Purification of the amplified DNA-Fragments and digest for ligation
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="PCR_-Purification_of_amplified_NgoMIV_TEV_iGEM_BamHI_DNA-Fragements"> PCR -Purification of amplified NgoMIV_TEV_iGEM_BamHI DNA-Fragements</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[17]</A>]</B></P><P><B>Investigators:</B> Sebastian</P><P><B>Aim:</B> Clean UP of the amplified fragments</P><P><B>Materials:</B></P><P><B> PCR-Clean up kit purchased by Promega(Wizard SV Gel and PCR-Up System)</B></P><P><B> Used method: </B></P><P><B> As described in the Manual of the kit </B></P><P><B>Results:</B></P><P>25 µl NgoMIV_TEV_iGEM_BamHI with 68,5 ng/µl
</P><P><B> Further going: </B>digest of the fragments and ligation for transformation
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Ampilifcation_of_NgoMIV_TEV_iGEM_BamHI_protease_via_PCR_2"> Ampilifcation of NgoMIV_TEV_iGEM_BamHI protease via PCR </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[18]</A>]</B></P><P><B>Investigators:</B> Sebastian</P><P><B>Aim:</B> 2nd amplification of the NgoMIV_TEV_iGEM_BamHI protease for digest with BamHI and NgoMIV</P><P><B>Materials:</B></P><P>1 µl Template - NgoMIV_TEV_iGEM_BamHI (6,9 ng/µl)</P><P>2,5 µl Primer 1 - f_TEV_AraFusion_NgoMIV (0,5 µM)</P><P>2,5 µl Primer 2 - r_TEV_iGEM_BamHI (0,5 µM)</P><P>5 µl 10x Reaction Buffer (Fermentas)</P><P>1 µl 10 mM dNTP's (Fermentas)</P><P>5 µl 25 µM MgCl2</P><P>0,5 µl DNA-Polymerase (Fermentas)</P><P>32,5 µl water</P><P><B> Used method: </B></P><P>Program: iGEM005
</P><UL><LI>Denat: 3min 94°C
</LI></UL><UL><LI>30x:
</LI></UL><P>Denat: 60sec 94°C
</P><P>Anneal:60sec 63°C
</P><P>Extend:60sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P><B> Further going: </B></P><P>PCR-Purification of the amplified DNA-Fragments and digest for ligation
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="ELISA_to_test_the_anti_myc-tag_antibodies_9E10"> ELISA to test the anti myc-tag antibodies 9E10 </SPAN></H3><P><B>Investigators:</B> Sebastian</P><P><B>Aim:</B> Testing the reactivity of the 9E10 antibody fractions</P><P><B>Method:</B></P><P>Coating of an ELISA (96-well microtiter plate) with 50 µl/well 5 mg/ml FITC-BSA and incubation over night</P><P>Blocking of the free binding sites with 50 µl/well PBS-5% NKS 0,0025% phenolred for 1 hour</P><P>Incubation with different ScFv tagged with myc (unkown concentration)for 1 hour</P><UL><LI> A1-H4 - anti FITC ScFv </LI></UL><UL><LI> A5-H8 - Z6.1 (anti-FITC ScFV) </LI></UL><UL><LI> A9-H12 - GST-tagged protein </LI></UL><P>Incubation with the tracer antibody 9E10 (different fractions in different lines (A-H) for 1 hour</P><P>Incubation with goat anti mice antibody labeled with HRP for 1 hour</P><P>Addition of the HRP-substrate (1 mg/ml OPD, 0,01 % H2O2, in 0,1 M Na-Citrate Buffer pH 5) for 30 min</P><P>Stopping of the reaction with 100 µl/well 1 M H2SO4 and 50 mM Na2SO3</P><P>measurement of the wavelength at 490 nm and 690 nm af reference</P><P><B>Materials:</B></P><P>1 µg/ml 9E10 antibody (from different fractions)</P><P>0,5 µg/ml 9E10 antibody (from different fractions)</P><P>several stock solutions (Blocking Solution, Substrate, PBS-NKS-Phenolred)</P><P><B>Results:</B></P><P>2 active fractions of 9E10 for coulping to column NHS-activated sepharose material</P><P><B>Further Tasks:</B></P><P>Coupling of active antibodies to NHS-activated sepharose for purification of myc-tagged proteins </P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparing_linearized_backbones_pSB1K3.2C_pSB1A3_and_pSB1T3_for_ligation_w.2F_YFP_and_CFP">Preparing linearized backbones pSB1K3, pSB1A3 and pSB1T3 for ligation w/ YFP and CFP </SPAN></H3><P><B>Time:</B> 2011-08-09,15:00-20:00</P><P><B>Investigators:</B> Vanessa, Jessica, Nadine </P><P><B> Motivation: </B> first step of expression backbone creation: digest linearized backbones with EcoRI and PstI</P><P><B>Materials:</B></P><UL><LI> pSB1K3, pSB1A3 and pSB1T3
</LI></UL><UL><LI> EcoRI, PstI
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> 100x BSA
</LI></UL><UL><LI> DpnI
</LI></UL><P><B>Protocol:</B></P><P><B> Digestion protocol (following iGEM distribution protocol for linearized backbones):</B></P><UL><LI>Mastermix
</LI></UL><UL><LI>4 µl NEB Buffer 4
</LI></UL><UL><LI>0.4 µl BSA
</LI></UL><UL><LI>0.4 µl EcoRI
</LI></UL><UL><LI>0.4 µl PstI
</LI></UL><UL><LI>0.4 µl DpnI
</LI></UL><UL><LI>14.4 µl pure water
</LI></UL><UL><LI>total: 20 µl
</LI></UL><UL><LI> reaction mix:
</LI></UL><UL><LI>4 µl mastermix + 4 µl linearized backbone (pSB1K3, pSB1A3 or pSB1T3) from distribution
</LI></UL><UL><LI> Incubation:
</LI></UL><UL><LI>37°C for 30 min</LI></UL><UL><LI> Heat deactivation:
</LI></UL><UL><LI> 80°C for 20 min
</LI></UL><UL><LI>stored in fridge 4°C
</LI></UL><P><B>Further Task: </B></P><UL><LI>Digestion of YFP from pGA14mVenusGeneart and CFP from pGA14-Cerulean
</LI></UL><UL><LI> Ligation of digested linearized backbones from today with digested YFP and CFP <B>(results: pSB1K3+YFP, pSB1T3+YFP, pSB1A2+YFP, pSB1K3+CFP, pSB1T3+CFP, pSB1A2+CFP)</B></LI></UL><UL><LI>Transformation w/ ligation products, <B>NOTE: !!! Don´t use XL1-Blue for pSB1T3+YFP and pSB1T3+CFP!!! Cells contain Tet-R already!!!</B></LI></UL><UL><LI> Picking of colonies
</LI></UL><UL><LI>over-night cultures for mini-prep
</LI></UL><UL><LI> mini-prep of over-night cultures
</LI></UL><UL><LI>test-digest
</LI></UL><UL><LI> pSB1A3+YFP/CFP:
</LI></UL><UL><LI> HaeII and BglI (protocol see 2011-8-9), expected: 3 fragments
</LI></UL><UL><LI> pSB1K3+YFP/CFP
</LI></UL><UL><LI>HaeII (develop protocol, calculate exact expected bp) expected: 2 fragments
</LI></UL><UL><LI> pSB1T3+YFP/CFP
</LI></UL><UL><LI> ClaI and ApaLI (develop protocol, calculate exact expected bp) expected: 3 fragments
</LI></UL><UL><LI> also: digested, not ligated linearized backbones (pSB1K3, pSB1A3 and pSB1T3)
</LI></UL><UL><LI> also: undigested pSB1K3+YFP, pSB1T3+YFP, pSB1A2+YFP, pSB1K3+CFP, pSB1T3+CFP, pSB1A2+CFP
</LI></UL><UL><LI> if test-digest positive: digestion w/ EcoRI and XbaI
</LI></UL><UL><LI> digestion of PCR products form ??? (Ara, Lac and constitutive promotor)
</LI></UL><UL><LI> ligation of digested vectors w/digested PCR products from ???? (Ara, Lac and constitutive promotor)<B>(results: pSB1K3+YFP+Ara, pSB1T3+YFP+Ara, pSB1A2+YFP+Ara, pSB1K3+CFP+Ara, pSB1T3+CFP+Ara, pSB1A2+CFP+Ara, pSB1K3+YFP+Lac, pSB1T3+YFP+Lac, pSB1A2+YFP+LAc, pSB1K3+CFP+Lac, pSB1T3+CFP+Lac, pSB1A2+CFP+Lac, pSB1K3+YFP+const, pSB1T3+YFP+const, pSB1A2+YFP+const, pSB1K3+CFP+const, pSB1T3+CFP+const, pSB1A2+CFP+const)</B></LI></UL><UL><LI>test digestion, sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_YFP_from_pGA14mVenusGeneart_and_CFP_from_pGA14-Cerulean">Digestion of YFP from pGA14mVenusGeneart and CFP from pGA14-Cerulean </SPAN></H3><P><B>Time:</B> 2011-08-09,15:00-20:00</P><P><B>Investigators:</B> Jessica, Nadine, Vanessa </P><P><B>Materials:</B></P><P>Preparation for ligation of YFP/CFP (insert) into pSB1A3, pSB1K3 or pSB1T3 (vectors)
</P><P><B>Materials:</B></P><UL><LI> pGA14mVenusGeneart and pGA14-Cerulean
</LI></UL><UL><LI> EcoRI, PstI
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> 100x BSA
</LI></UL><P><B>Protocol:</B></P><P><B> Digestion protocol (following iGEM distribution protocol for linearized backbones):</B></P><UL><LI>Mastermix
</LI></UL><UL><LI>2 µl NEB Buffer 4
</LI></UL><UL><LI>0.2 µl BSA
</LI></UL><UL><LI>0.2 µl EcoRI
</LI></UL><UL><LI>0.2 µl PstI
</LI></UL><UL><LI>7.4 µl pure water
</LI></UL><UL><LI>total: 10 µl
</LI></UL><UL><LI> reaction mix:
</LI></UL><UL><LI>4 µl mastermix + 4 µl 1:2 diluted DNA
</LI></UL><UL><LI> Incubation:
</LI></UL><UL><LI>37°C for 30 min
</LI></UL><UL><LI> Heat deactivation:
</LI></UL><UL><LI> 80°C for 20 min
</LI></UL><UL><LI><B>YFP dig., 9.8.11, Nad &amp; Jes</B> stored in fridge 4°C
</LI></UL><UL><LI><B>CFP dig., 9.8.11, Nad &amp; Jes</B> stored in fridge 4°C
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="over_night_culture_from_PDV089">over night culture from PDV089</SPAN></H3><P><B>Investigators:</B> Sandrina </P><P><B>Aim:</B> control plasmid ligation (pARW089, mdnA, geneIII --&gt; PDV089, strategy 2)
</P><P><B>Time:</B> 2011-08-09,16.00-17:00 </P><P><B>Materials/Methods:</B></P><UL><LI> LB-medium with tet and amp
</LI></UL><UL><LI> cell clones from over night plate
</LI></UL><UL><LI> incubate over night at 37°C and 750 rpm
</LI></UL><P><B>Further tasks:</B></P><UL><LI> plasmid preparation and analytic digestion
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_PCR_of_mdnA_for_phage_display_.28strategy_1.29_repeated_with_new_ordered_reversed_primer_and_of_mdnA_with_rfc25_restriction_sites_.28strategy_2.29.2C_gel_electrophoresis_and_purification_of_PCR_products">Repeated PCR of mdnA for phage display (strategy 1) repeated with new ordered reversed primer and of mdnA with rfc25 restriction sites (strategy 2), gel electrophoresis and purification of PCR products</SPAN></H3><P><B>Investigators:</B> Sandrina </P><P><B>Time:</B> 2011-06-30,11:00-15:00 </P><P><B>Aim:</B></P><UL><LI> amplification of mdnA with SfiI restriction sites (vector pARW089) to clone it into pAk100 bla KDIR </LI></UL><UL><LI> amplification of mdnA with rfc25 restriction sites to fuse it with geneIII and clone it into pARW089 </LI></UL><P><B>Materials/Methods:</B></P><P>see 2011-06-10
</P><P>changes:
</P><UL><LI> program: 123, Thermo Hybrid, PX2
</LI></UL><P><B>Results:</B></P><P>expected bands (ca. 260 bp for strategy 1 and ca. 230 bp for strategy 2) were observed after gel electrophoresis
</P><P><B>further tasks:</B></P><P>digestion of mdnA fragment with sfiI (strategy 1) and NarI and AgeI (strategy 2)
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_mdnA_with_sfiI_over_night">Digestion of mdnA with sfiI over night</SPAN></H3><P><B>Investigators:</B> Sandrina </P><P><B>Time:</B> 2011-08-09,16:30-17:00 </P><P><B>Aim:</B></P><P>digestion of apmlificated mdnA to clone it into PAK100 bla KDIR </P><P><B>Materials/Methods:</B></P><P>50 µl sample:
</P><UL><LI> 0,2 µl BSA
</LI></UL><UL><LI> 5 µl 10x Puffer 4
</LI></UL><UL><LI> 0,5 µl sfiI
</LI></UL><UL><LI> 40 µl PCR product
</LI></UL><UL><LI> 4,3 µl H2O
</LI></UL><P>incubation over night at 50°C
</P><P><B>further tasks:</B></P><P>ligation with digested PAK100 bla KDIR
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="design_primer_for_biobrick_-_redesign_of_pf_mdnE.2C_pf_mdnD"> design primer for biobrick - redesign of pf_mdnE, pf_mdnD </SPAN></H3><P><B>Investigators:</B> Niels, Nadine </P><P><B> Aim: </B> design and order primer
</P><P><B> Primer sequence </B></P><UL><LI> pf_mdnE_EcoRI_NotI_XbaI
</LI></UL><P>CAATCATCATATAACTCCGTAGATCTTCGCCGGCGCTTAAG
</P><UL><LI> pf_mdnD_EcoRI_NotI_XbaI
</LI></UL><P>GTCAAAAAGGTCACGAAAGTAGATCTTCGCCGGCGCTTAAG
</P><P><B> ordered my sigma </B></P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="59th_Labday_2011-08-10">59th Labday 2011-08-10</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI_fragments_.28produced_on_09.08.2011.29_and_pJC354-NheI-TEV-Xho_blaFL_GGH5_vector">Digestion of NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragments (produced on 09.08.2011) and pJC354-NheI-TEV-Xho_blaFL_GGH5 vector</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[19]</A>]</B></P><P><B>Investigators:</B> Paul</P><P><B>Materials:</B></P><P>NgoMIV_iGEM_TEV_iGEM_BamHI:
</P><UL><LI>three samples: one sample produced on 09.08.2011 and already PCR purificated; two samples produced on 09.08.2011 not PCR purificated --&gt; PCR purification of the two unpurificated samples using &quot;NucleoSpin ExtractII&quot;-KIT!
</LI></UL><P>TEVI:67,9ng/µl (2ml eppi in long screening reck)
</P><P>TEVII:56ng/µl (2ml eppi in long screening reck)
</P><P>TEVIII:68,5ng/µl (1,5 ml eppi in logn screening reck, eppi named: 2. TEV muta PCR...)
</P><P>pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (contains TEV cleavage site): 470ng/µl
</P><P><B> Digestion protocol: </B></P><P>1: NgoMIV_iGEM_TEV-Protease_iGEM_BamHI (3x, each sample 1x):
</P><UL><LI>20µl NgoMIV_iGEM_TEV-Protease_iGEM_BamHI
</LI></UL><UL><LI>1µl NgoMIV
</LI></UL><UL><LI>1µl BamHI-HF
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>22.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>2: pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (470ng/µl)
</P><UL><LI>3µl pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (contains TEV cleavage site)
</LI></UL><UL><LI>1µl HindIII
</LI></UL><UL><LI>1µl BamHI-HF
</LI></UL><UL><LI>5µl 10x buffer = NEB4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>39.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>--&gt;The reaction was allowed to proceed for 2h at 37°C!
</P><P><B> Result: </B></P><P>pJC354-NheI-TEV-Xho_blaFL_GGH5 vector:
</P><P>Expected band: 4700bp --&gt; Excission of band and pruification unsing &quot;NucleoSpin ExtractII&quot;-KIT
</P><P>NgoMIV_iGEM_TEV-Protease_iGEM_BamHI:
</P><P>Expected bands: 760bp --&gt; Excission of red marked areas and pruification unsing &quot;NucleoSpin ExtractII&quot;-KIT
</P><P><B> Further tasks: </B></P><P>Triple ligation of digested fragments and HindIII_iGEM_AraC_NgoMIV
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_E._coli_XL1-Blue_with_pGA14mVenusGeneart_resp._pGA14-Cerulean">Transformation of <I>E. coli</I> XL1-Blue with pGA14mVenusGeneart resp. pGA14-Cerulean</SPAN></H3><P><B>Investigators:</B> Nadine </P><P><B>Aim:</B>Transformation of <I>E. coli</I> XL1-Blue cells with pGA14mVenusGeneart resp. pGA14-Cerulean to produce glycerol stocks for further use if necessary </P><P><B>Time:</B> 2011-08-10, 9:20-? </P><P><B>Materials:</B></P><UL><LI> pGA14mVenusGeneart resp. pGA14-Cerulean
</LI></UL><UL><LI><I>E. coli</I> XL1-Blue cells
</LI></UL><UL><LI> LB medium
</LI></UL><P><B>Method:</B></P><UL><LI> addition of 1 µl plasmid to XL1-blue cells</LI></UL><UL><LI> incubation 30 min on ice,</LI></UL><UL><LI> heat shock 45 sec at 42°C,</LI></UL><UL><LI> incubation 2 min on ice,</LI></UL><UL><LI> addition of 750 µl LB medium,</LI></UL><UL><LI> incubation 60 min at 37 °C and 750 rpm</LI></UL><UL><LI> plating on agar plates containing 100 µg/µl ampicillin</LI></UL><UL><LI> storage over night at 37°C</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_Gel_of_digested_pSB1K3.2C_pSB1A3_and_pSB1T3_and_digested_CFP_and_YFP_fragments">Agarose Gel of digested pSB1K3, pSB1A3 and pSB1T3 and digested CFP and YFP fragments</SPAN></H3><P><B>Investigators:</B> Jessica, Vanessa, Nadine </P><P><B>Aim:</B>Purification of insert and vector for the first ligation of the expression backbone</P><P><B>Time:</B> 2011-08-10, 9:20-16:00 </P><P><B>Materials:</B></P><UL><LI> digested pSB1K3, pSB1A3 and pSB1T3 from 2011-09-08
</LI></UL><UL><LI> digested CFP and YFP from 2011-09-08 (<B>YFP dig. and CFP dig., 9.8.11, Nad &amp; Jes</B>)
</LI></UL><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> DNA Ladder Mix (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas)
</LI></UL><P><B> Production of one 1 % agarose gel</B></P><UL><LI> 2 x 1 % gel: 0.5 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to each gel
</LI></UL><P><B>Loading gels and running</B></P><UL><LI> Add 1.2 µl Loading dye to each 8 µl sample
</LI></UL><UL><LI> 12 µl DNA Ladder Mix
</LI></UL><UL><LI> Running conditions: 110 V, approx. 45 min
</LI></UL><P><B> Loading of gel 1</B></P><UL><LI>gel 1: pSB1K3, pSB1A3 and pSB1T3
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> marker </TD><TD>15 </TD></TR><TR><TD> 1 </TD><TD> pSB1A3, EcoRI, PstI</TD><TD> 9.2</TD><TD> 2114, 41
</TD></TR><TR><TD> 2 </TD><TD> - </TD><TD> -</TD><TD> -
</TD></TR><TR><TD> 3 </TD><TD> pSB1K3, EcoRI, PstI </TD><TD> 9.2</TD><TD> 2163, 41
</TD></TR><TR><TD> 4 </TD><TD> - </TD><TD> -</TD><TD> -
</TD></TR><TR><TD> 5 </TD><TD> pSB1T3, EcoRI, PstI </TD><TD> -</TD><TD> 2422, 41
</TD></TR></TBODY></TABLE><P><B> Result gel 1: </B></P><UL><LI> bigger bands appear as expected
</LI></UL><UL><LI> 41 bp band is to small to appear in the gel
</LI></UL><UL><LI> gel extraction with Wizard SV Gel and PCR Clean-Up System (Promega):
</LI></UL><UL><LI><B>pSB1A3 dig. pur. 10.08.11 Nad &amp; Jes</B> : 9.8 ng/µl
</LI></UL><UL><LI><B>pSB1K3 dig. pur. 10.08.11 Nad &amp; Jes</B> : 10.7 ng/µl
</LI></UL><UL><LI><B>pSB1T3 dig. pur. 10.08.11 Nad &amp; Jes</B> : 11.3 ng/µl
</LI></UL><UL><LI> stored in freezer (-20°C), red box: expression backbones
</LI></UL><P><B> Loading of gel 2 </B></P><UL><LI>gel 2: CFP and YFP
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> 1 </TD><TD> marker </TD></TR><TR><TD> 2 </TD><TD> from Screening (TEV Vector)</TD><TD> 42</TD></TR><TR><TD> 3 </TD><TD> -</TD><TD> -</TD><TD>-
</TD></TR><TR><TD> 4 </TD><TD> CFP </TD><TD> 9.2</TD><TD> 771, 2873
</TD></TR><TR><TD> 5 </TD><TD> - </TD><TD> -</TD><TD> -
</TD></TR><TR><TD> 6 </TD><TD> YFP </TD><TD> 9.2</TD><TD> 763, 2881
</TD></TR></TBODY></TABLE><P><B> Result gel 2: </B></P><UL><LI> in lane 4 and 6 are too many bands
</LI></UL><UL><LI> troubleshooting: it seems that the mastermixes were mixed up! In one mastermix was DpnI. DpnI digests methylated DNA. The vectors are from <I>E. coli</I> and therfore methylated. This could be an explanation for these band patterns.
</LI></UL><P><B>Further tasks:</B></P><UL><LI> repeat digest of YFP from pGA14mVenusGeneart and CFP from pGA14-Cerulean
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_YFP_from_pGA14mVenusGeneart_and_CFP_from_pGA14-Cerulean_2">Digestion of YFP from pGA14mVenusGeneart and CFP from pGA14-Cerulean </SPAN></H3><P><B>Time:</B> 2011-08-10,15:00-17:00</P><P><B>Investigators:</B> Vanessa, Jessica, Nadine </P><P><B>Materials:</B></P><UL><LI> pGA14mVenusGeneart and pGA14-Cerulean
</LI></UL><UL><LI> EcoRI, PstI
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> 100x BSA
</LI></UL><P><B>Protocol:</B></P><P><B> Digestion protocol (following iGEM distribution protocol for linearized backbones):</B></P><UL><LI>Mastermix
</LI></UL><UL><LI>2 µl NEB Buffer 4
</LI></UL><UL><LI>0.2 µl BSA
</LI></UL><UL><LI>0.2 µl EcoRI
</LI></UL><UL><LI>0.2 µl PstI
</LI></UL><UL><LI>7.4 µl pure water
</LI></UL><UL><LI>total: 10 µl
</LI></UL><UL><LI> reaction mix:
</LI></UL><UL><LI>4 µl mastermix + 4 µl DNA (<B>not diluted!!!</B>)
</LI></UL><UL><LI> Incubation:
</LI></UL><UL><LI>37°C for 30 min
</LI></UL><UL><LI> Heat deactivation:
</LI></UL><UL><LI> 80°C for 20 min
</LI></UL><UL><LI><B>YFP dig., 10.8.11, Van &amp; Jes</B> stored in fridge 4°C
</LI></UL><UL><LI><B>CFP dig., 10.8.11, Van &amp; Jes</B> stored in fridge 4°C
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_E._coli_overnight_culture_containing_pPDV089">Miniprep of <I>E. coli</I> overnight culture containing pPDV089 </SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B> purification of pPDV089 for test digestion and sequencing (15 clones)</P><P><B>Time:</B> 10:00-13:00</P><P><B>Method/Materials:</B> see protocol 5.1 of the NucleoSpin Plasmid Kit</P><P><B>Further tasks:</B> test digestion</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Test_digestion_of_ligations_for_strategy_2_after_Plasmid_preperation">Test digestion of ligations for strategy 2 after Plasmid preperation </SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina </P><P><B>Aim:</B>control if liagation of geneIII and mdnA into pARW089 (strategy 2) worked</P><P><B>Time:</B> 14:00-16:00</P><P><B>Materials/Methods:</B></P><P>Strategy 2:
</P><UL><LI> 0,5 µl XbaI
</LI></UL><UL><LI> 0,5 µl SpeI
</LI></UL><UL><LI> 2 µl 10x buffer 4 (NEB)
</LI></UL><UL><LI> 0,2 µl BSA
</LI></UL><UL><LI> 10 µl vector DNA
</LI></UL><UL><LI> 12,8 µl H2O
</LI></UL><P>incubate for 1 h at 37°C
</P><P><B>Results:</B></P><UL><LI>expected size for all samples: strategy 2: ca. 5000 bp, 4000 bp, 600 bp and 200
</LI></UL><UL><LI>three different pARW089 vectors (1,2,3) were used for ligation, they were digested with the same enzymes but this was done from three different persons
</LI></UL><UL><LI>after digestion of geneIII PCR product with AatII and NgoMIV two bands were seen after gel electrophoresis, ligations were done with both samples (called here: &quot;upper band&quot; and &quot;lower band&quot;)
</LI></UL><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> marker, DNA ladder mix Fermentas </TD></TR><TR><TD> 1 </TD><TD> free </TD></TR><TR><TD> 2 </TD><TD> vector 1, upper band, clone 1 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 3 </TD><TD> vector 1, upper band, clone 2</TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 4 </TD><TD> vector 1, upper band, clone 3 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 5 </TD><TD> vector 1, lower band, clone 1 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 6 </TD><TD> vector 1, lower band, clone 2 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 7 </TD><TD> vector 1, lower band, clone 3 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 8 </TD><TD> free </TD></TR><TR><TD> 9 </TD><TD> vector 2, upper band, clone 1 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 10 </TD><TD> vector 2, upper band, clone 2 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 11 </TD><TD> vector 2, upper band, clone 3 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 12 </TD><TD> vector 2, lower band, clone 1 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 13 </TD><TD> vector 2, lower band, clone 2 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 14 </TD><TD> vector 2, lower band, clone 3 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 15 </TD><TD> free </TD></TR><TR><TD> 16 </TD><TD> vector 3, upper band, clone 1 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 17 </TD><TD> vector 3, upper band, clone 2 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR><TR><TD> 18 </TD><TD> vector 3, upper band, clone 3 </TD><TD> 20</TD><TD> ca. 5000, 4000, 600, 200
</TD></TR></TBODY></TABLE><P><A href="/File:UP_test_dig._3_PDV089.png%3F" title="File:UP test dig. 3 PDV089.png?">350px</A></P><P><B>Further tasks:</B></P><UL><LI> sequence clones 12, 13 and 17
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="cultivation_of_cells_containing_pAK100_bla_KDIR_from_E._coli_glycerol_stocks">cultivation of cells containing pAK100 bla KDIR from <I>E. coli</I> glycerol stocks</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina </P><P><B>Aim:</B> cultivate cells conatining pAK100 bla KDIR (vector) to purify it and use it for phage display.
</P><P><B>Time:</B> 2011-06-14,16:00-17.30</P><P><B>Materials/Methods:</B></P><UL><LI>glycerol stock nr. 15, pAK100 bla KDIR, XL1-blu, 2003-07-31
</LI></UL><UL><LI>LB-medium with chloramphenicol (1:1000)
</LI></UL><P><B>Further tasks:</B> plasmid preperation und digestion with SfiI
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="60th_Labday_2011-08-11">60th Labday 2011-08-11</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_Gel_digested_CFP_and_YFP_fragments_.28from_2011-08-10.29">Agarose Gel digested CFP and YFP fragments (from 2011-08-10)</SPAN></H3><P><B>Investigators:</B> Jessica, Nadine, Vanessa </P><P><B>Aim:</B>Purification of YFP and CFP fragment</P><P><B>Time:</B> 2011-08-11, 8:00-11:00 </P><P><B>Materials:</B></P><UL><LI> digested CFP and YFP from 2011-09-08 (<B>YFP dig. and CFP dig., 10.8.11, Van &amp; Jes</B>)
</LI></UL><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> DNA Ladder Mix (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas)
</LI></UL><P><B> Production of one 1 % agarose gel</B></P><UL><LI> 1 % gel: 0.5 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to each gel
</LI></UL><P><B>Loading gels and running</B></P><UL><LI> Add 1.2 µl Loading dye to each 8 µl sample
</LI></UL><UL><LI> 12 µl DNA Ladder Mix
</LI></UL><UL><LI> Running conditions: 100 V, approx. 45 min
</LI></UL><P><B> Loading of gel </B></P><UL><LI>gel: CFP and YFP
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> 1 </TD><TD> marker </TD><TD> 10</TD></TR><TR><TD> 2 </TD><TD> -</TD><TD> -</TD><TD> -
</TD></TR><TR><TD> 3</TD><TD> CFP </TD><TD> 9.2</TD><TD> 771, 2873
</TD></TR><TR><TD> 4 </TD><TD> -</TD><TD> -</TD><TD>-
</TD></TR><TR><TD> 5 </TD><TD> YFP </TD><TD> 9.2</TD><TD> 763, 2881
</TD></TR><TR><TD> 6 </TD><TD> - </TD><TD> -</TD><TD> -
</TD></TR></TBODY></TABLE><P><B> Result: </B></P><UL><LI> bands appear as expected
</LI></UL><UL><LI> lower bands (red box) in lane 3 and 4 were excised and DNA was extracted w/ Wizard SV Gel and PCR Clean-Up System (Promega):
</LI></UL><UL><LI><B>CFP dig. pur. Jes &amp; VB 11.8.2011</B>, 5.7 ng/µl
</LI></UL><UL><LI><B>YFP dig. pur. Jes &amp; VB 11.8.2011</B>, 5.4 ng/µl
</LI></UL><UL><LI> stored in freezer (-20°C), red box: expression backbones
</LI></UL><P><B>Further tasks:</B></P><UL><LI> ligation of digested and purified YFP and CFP with digested and purified pSB1A3, pSB1K3 and pSB1T3
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_digested_and_purified_CFP_and_YFP_fragments_.28from_2011-08-11.29_w.2F_digested_and_purified_pSB1A3.2C_pSB1K3.2C_pSB1T3_.28from_2011-08-10.29">Ligation of digested and purified CFP and YFP fragments (from 2011-08-11) w/ digested and purified pSB1A3, pSB1K3, pSB1T3 (from 2011-08-10)</SPAN></H3><P><B>Investigators:</B> Nadine, Vanessa, Jessica </P><P><B>Aim:</B>Ligation</P><P><B>Time:</B> 2011-08-11, 11:00-14:30 </P><P><B>Materials:</B></P><UL><LI> T4 Ligase
</LI></UL><UL><LI> 10x T4 Ligase buffer
</LI></UL><UL><LI>inserts
</LI></UL><UL><LI><B>CFP dig. pur. Jes &amp; VB 11.8.2011</B>, 5.7 ng/µl
</LI></UL><UL><LI><B>YFP dig. pur. Jes &amp; VB 11.8.2011</B>, 5.4 ng/µl
</LI></UL><UL><LI>vectors:
</LI></UL><UL><LI><B>pSB1A3 dig. pur. 10.08.11 Nad &amp; Jes</B> : 9.8 ng/µl,
</LI></UL><UL><LI><B>pSB1K3 dig. pur. 10.08.11 Nad &amp; Jes</B> : 10.7 ng/µl
</LI></UL><UL><LI><B>pSB1T3 dig. pur. 10.08.11 Nad &amp; Jes</B> : 11.3 ng/µl
</LI></UL><UL><LI> pure sterile water
</LI></UL><P><B> Protocols</B></P><UL><LI> to calculate the volumes <A href="http://old.gibthon.org/" class="external free" rel="nofollow">http://old.gibthon.org/</A> was used
</LI></UL><P><B> pSB1K3 (Volumes in µl)</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>CFP</B></TD><TD><B>YFP</B></TD><TD><B>Control</B></TD></TR><TR><TD> 10x T4 ligase buffer </TD><TD> 1 </TD><TD> 1</TD><TD>1
</TD></TR><TR><TD> T4 ligase </TD><TD> 1</TD><TD> 1</TD><TD> 1
</TD></TR><TR><TD> vector</TD><TD> 2.7 </TD><TD> 2.6</TD><TD> 2.6
</TD></TR><TR><TD> insert </TD><TD> 5.3</TD><TD> 5.4</TD><TD>-
</TD></TR><TR><TD> water </TD><TD> - </TD><TD> -</TD><TD> 5.4
</TD></TR></TBODY></TABLE><P><B> pSB1A3 (Volumes in µl)</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>CFP</B></TD><TD><B>YFP</B></TD><TD><B>Control</B></TD></TR><TR><TD> 10x T4 ligase buffer </TD><TD> 1 </TD><TD> 1</TD><TD>1
</TD></TR><TR><TD> T4 ligase </TD><TD> 1</TD><TD> 1</TD><TD> 1
</TD></TR><TR><TD> vector</TD><TD> 2.8 </TD><TD> 2.8</TD><TD> 2.8
</TD></TR><TR><TD> insert </TD><TD> 5.2</TD><TD> 5.2</TD><TD>-
</TD></TR><TR><TD> water </TD><TD> - </TD><TD> -</TD><TD> 5.2
</TD></TR></TBODY></TABLE><P><B> pSB1T3 (Volumes in µl)</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>CFP</B></TD><TD><B>YFP</B></TD><TD><B>Control</B></TD></TR><TR><TD> 10x T4 ligase buffer </TD><TD> 1 </TD><TD> 1</TD><TD>1
</TD></TR><TR><TD> T4 ligase </TD><TD> 1</TD><TD> 1</TD><TD> 1
</TD></TR><TR><TD> vector</TD><TD> 2.6 </TD><TD> 2.5</TD><TD> 2.5
</TD></TR><TR><TD> insert </TD><TD> 5.4</TD><TD> 5.5</TD><TD>-
</TD></TR><TR><TD> water </TD><TD> - </TD><TD> -</TD><TD> 5.5
</TD></TR></TBODY></TABLE><UL><LI> Incubation: 2 hrs at room temperature
</LI></UL><P><B> Result: </B></P><UL><LI><B>pSB1A3+YFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1A3+CFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1A3+control, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1K3+YFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1K3+CFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1K3+control, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1A3+YFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1A3+CFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1A3+control, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1T3+YFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1T3+CFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1T3+control, lig, VB, 11.8.2011</B></LI></UL><UL><LI> all stored in freezer (-20°V in red box: expression backbone)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Transformation in XL1-blue, except for pSB1T3+YFP/CFP: here use RV308
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI_or_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_into_pJC354-NheI-143C-Xho_blaFL_GGH5_or_pJC354-NheI-TEV-Xho_blaFL_GGH5_vector"> Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI or NgoMIV_iGEM_TEV-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV into pJC354-NheI-143C-Xho_blaFL_GGH5 or pJC354-NheI-TEV-Xho_blaFL_GGH5 vector </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[20]</A>]</B></P><P><B>Investigators:</B> Paul, Sebastian</P><P><B>Aim</B>:</P><UL><LI><B>1. </B>Triple-ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI (573bp), HindIII_iGEM_AraC_NgoMIV (1273bp) and pJC354-NheI-143C-Xho_blaFL_GGH5 vector (~4700bp) to create pUP_SG2_TorA_CS-14_3C_bla_AraC-14_3C</LI></UL><UL><LI><B>2.</B>Triple-ligation of NgoMIV_iGEM_TEV_iGEM_BamHI (760bp), HindIII_iGEM_AraC_NgoMIV (1273bp) and pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (~4700bp)to create pUP_SG1_TorA_CS-TEV_bla_AraC-TEV</LI></UL><P>Calculation of volumes to be used with: <A href="http://www.gibthon.org/ligate.html" class="external text" rel="nofollow">ligation calculator</A> with 1:1 molar ratio
</P><P><B>Materials:</B></P><P>14_3C: 2 reaction batches (we have two digested pJC354-NheI-143C-Xho_blaFL_GGH5 vector fractions)
</P><P>1:
</P><UL><LI> 2 µL NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI fragment (573bp, 3.2ng/µl)
</LI></UL><UL><LI> 1,5 µL AraC fragment
</LI></UL><UL><LI> 4,5 µL pJC354-NheI-143C-Xho_blaFL_GGH5 vector (8.3ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P>2:
</P><UL><LI> 1,2 µL NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI (573bp, 3.2ng/µl) fragment
</LI></UL><UL><LI> 0,8 µL AraC fragment
</LI></UL><UL><LI> 6 µL pJC354-NheI-143C-Xho_blaFL_GGH5 vector (6.7ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><UL><LI> 2 controls: As 1 and 2 but with water instead of fragment
</LI></UL><P>TEV: 3 reaction batches (we have three digested NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment fractions)
</P><P>1:
</P><UL><LI> 0.8 µL NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment (760bp, 18.5ng/µl) (TEVI, see 10.08.2011)
</LI></UL><UL><LI> 2,6 µL AraC fragment
</LI></UL><UL><LI> 4,6 µL pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (19.4ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P>2:
</P><UL><LI> 1.1 µL NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment (760bp, 12.5ng/µl) (TEVII, see 10.08.2011)
</LI></UL><UL><LI> 2,5 µL AraC fragment
</LI></UL><UL><LI> 4,4 µL pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (19.4ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P>3:
</P><UL><LI> 0.7 µL NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment (760bp, 20.4ng/µl) (TEVIII, see 10.08.2011)
</LI></UL><UL><LI> 2,6 µL AraC fragment
</LI></UL><UL><LI> 4,7 µL pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (19.4ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><UL><LI> 1 control: As 2 but with water instead of fragment
</LI></UL><P><B> Used method: </B></P><P>ligation at room temperatur for 1h
</P><P><B> Further going:Transformation of XL1blue cells with ligation products </B></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_E._Coli_XL1_Blue_Cells_with_ligation_products_.28NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI_or_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_into_pJC354-NheI-143C-Xho_blaFL_GGH5_or_pJC354-NheI-TEV-Xho_blaFL_GGH5_vectors.29"> Transformation of <I>E. Coli</I> XL1 Blue Cells with ligation products (NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI or NgoMIV_iGEM_TEV-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV into pJC354-NheI-143C-Xho_blaFL_GGH5 or pJC354-NheI-TEV-Xho_blaFL_GGH5 vectors)</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[21]</A>]</B></P><P><B>Investigators:</B>Paul, Sascha</P><P><B>Aim:</B>transform the ligation into <I>E. coli</I> XL1 blue</P><P><B>Materials:</B></P><P>8x XL1 blue cells from -80 stock, 2x 14_3C-ligations + 2x controls ; 3x TEV-ligations + 1x control</P><P><B>protocol:</B></P><UL><LI>Taw cells on ice
</LI></UL><UL><LI>addition of 10 µl ligation reaction to cells (XL1-blue, tet-resistance, 60µl) in 2 ml Eppi,
</LI></UL><UL><LI>incubation 25 min on ice,
</LI></UL><UL><LI>heat shock 45 sec at 42°C,
</LI></UL><UL><LI>incubation 2 min on ice,
</LI></UL><UL><LI>addition of 750 µl LB medium,
</LI></UL><UL><LI>incubation at 37 °C for 60 min in Eppendorf thermomixer at 600 rpm
</LI></UL><UL><LI>plating on LB medium with 1,5 % agar, 100 µg/ml chloramphenicol,
</LI></UL><UL><LI>storage over night at 37°C
</LI></UL><P><B>Results</B></P><P>No colonies in case of 14_3C Protease, no colonies in controls of 14_3C
</P><P>Tev: three colonies on each plate, including control plate
</P><P><B>Further tasks</B></P><P>Picking colonies, making precultures and colony-PCRs and isolation/sequencing of pUP_SG1_TorA_CS-TEV_bla_AraC-TEV
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation">Transformation </SPAN></H3><P><B>Time:</B> 2011-8-11, 14:30 - 17:30 </P><P><B>Investigators:</B> Vanessa, Jessica, Nadine, Niels</P><P><B>Material:</B></P><UL><LI><B>pSB1A3+YFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1A3+CFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1A3+control, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1K3+YFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1K3+CFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1K3+control, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1A3+YFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1A3+CFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1A3+control, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1T3+YFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1T3+CFP, lig, VB, 11.8.2011</B></LI></UL><UL><LI><B>pSB1T3+control, lig, VB, 11.8.2011</B></LI></UL><UL><LI>6 x XL1-blue cells (competent) for pSB1A3 and pSB1K3 variants
</LI></UL><UL><LI>3 x RV380 cells (competent) for pSB1T3 variants
</LI></UL><P><B>Method:</B></P><UL><LI> addition of 2 µl ligation reaction to XL1-blue cells or RV380 cells</LI></UL><UL><LI> incubation 25 min on ice,</LI></UL><UL><LI> heat shock 45 sec at 42°C,</LI></UL><UL><LI> incubation 2 min on ice,</LI></UL><UL><LI> addition of 750 µl LB medium,</LI></UL><UL><LI> incubation 60 min at 37 °C and 750 rpm</LI></UL><UL><LI> plating on agar plates containing 100 µg/ml tetracyclin
</LI></UL><UL><LI> plating on agar plates containing 100 µg/ml kanamycin
</LI></UL><UL><LI> plating on agar plates containing 100 µg/µl ampicillin</LI></UL><UL><LI> incubation over night at 37°C</LI></UL><P><B>NOTE:</B></P><UL><LI><B>pSB1T3+YFP/CFP have the same label; we have to check the insert with test digestion</B></LI></UL><P><B>Further tasks:</B> control cell clones tomorrow morning
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_PCR_gene_III_for_phage_display_.28strategy_2.29_2">Repeated PCR gene III for phage display (strategy 2)</SPAN></H3><P><B>Investigator:</B> Sabine </P><P><B>Time:</B> 2011-08-11,1:00-12:30 </P><P><B>Aim:</B></P><UL><LI> amplificate GeneIII with NgoMIV and AatII restriction sites (strategy 2)</LI></UL><P><B>Primer:</B></P><UL><LI> primer: pf_geneIII_NgoMIV_XbaI_myc and pr_geneIII_iGEM_AatII (geneIII, strategy 2)</LI></UL><P><B>Reaction Components:</B></P><UL><LI> 5 µl pak100blaKDIR
</LI></UL><UL><LI> 0,25 µl Taq Polymerase S (BioScience)
</LI></UL><UL><LI> 1 µl dNTPs
</LI></UL><UL><LI> 1 µl per primer
</LI></UL><UL><LI> 5 µl 10x PCR Buffer S
</LI></UL><UL><LI> 37,75 µl DNase free water
</LI></UL><P><B>Purification:</B></P><UL><LI> NucleoSpin Extract II
</LI></UL><P><B>Further tasks:</B></P><UL><LI> digestion
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_E._coli_overnight_culture_containing_pakblaKDIR">Miniprep of <I>E. coli</I> overnight culture containing pakblaKDIR </SPAN></H3><P><B>Investigators:</B> Sabine</P><P><B>Aim:</B> purification of pakblaKDIR</P><P><B>Time:</B> 10:00-11:00</P><P><B>Method/Materials:</B> see protocol 5.1 of the NucleoSpin Plasmid Kit</P><P><B>Further tasks:</B></P><UL><LI> digestion with Sfi (strategy I)
</LI></UL><UL><LI> PCR of geneIII (strategy II)
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="digestion_of_pak_blaKDIR_.28strategy_1.29">digestion of pak blaKDIR (strategy 1)</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine</P><P><B>Aim:</B> ligation of digested mdnA into pak blaKDIR to get phage display vector pPDV100 (strategy I)
</P><P><B>Time:</B> 2011-08-11, 11:00-14:00</P><P><B>reaction components:</B></P><UL><LI> 15 µl pak blaKDIR (ca 1 µg)
</LI></UL><UL><LI> 2 µl NEB 10x buffer
</LI></UL><UL><LI> 0,2 µl 100x BSA
</LI></UL><UL><LI> 1 µl restriction enzyme SfiI
</LI></UL><UL><LI> 1,8 µl water
</LI></UL><P><B>reaction conditions:</B></P><UL><LI> 3 h, 37°C
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="sending_clone_12.2C_13_and_17_from_ligation_strategy_2_.28see_2011-08-10.29_to_MWG_for_sequencing">sending clone 12, 13 and 17 from ligation strategy 2 (see 2011-08-10) to MWG for sequencing</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Time:</B> 2011-8-11, 12:00 - 15:00 </P><P><B>Aim:</B> get sequence of generated phage display vector pPDV089 (strategy 2) to control the ligation of digested pARW089 with digested mdnA and gene III</P><P><B>Method/Materials:</B></P><UL><LI>50-100 ng DNA in 15 µl sample</LI></UL><UL><LI>used primer: sf_mdna_1 (nr. 6)
</LI></UL><P><B>Further tasks:</B></P><P>perform alignment
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Gel_electrophoresis_of_digested_PAK100_bla_KDIR_vector.28strategy_1.29_and_PCR_of_geneIII">Gel electrophoresis of digested PAK100 bla KDIR vector(strategy 1) and PCR of geneIII</SPAN></H3><P><B>Investigators:</B> Sandrina </P><P><B>Aim:</B>Purification of PAK100 bla KDIR (strategy 1) and analysis if PCR worked </P><P><B>Time:</B> 2011-08-11,15:00-16:30</P><P><B>Materials/Methods:</B></P><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> digested vector PAK100 bla KDIR
</LI></UL><UL><LI> DNA Ladder Mix(Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas) </LI></UL><P><B>Method:</B></P><P><B>1. Production of a 1 % agarose gel</B></P><UL><LI> adding 2 µl gel red </LI></UL><P><B>2. Run</B></P><UL><LI> 100 V
</LI></UL><UL><LI> time: 00:45 h </LI></UL><P><B>Results:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> M </TD><TD> marker </TD></TR><TR><TD> 1 </TD><TD> geneIII PCR </TD><TD> 5 </TD><TD> ca. 400
</TD></TR><TR><TD> 2 </TD><TD> digested PAK100 bla KDIR vector </TD><TD> 30 </TD><TD> ca. 5500, 800
</TD></TR></TBODY></TABLE><P>one band was cutted out of the gel for purifacation
</P><P><B>Further tasks:</B></P><P>ligation of mdnA with pAK100 bla KDIR
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="ligation_of_mdnA_into_pak100blaKDIR_.28stategy_1.29">ligation of mdnA into pak100blaKDIR (stategy 1)</SPAN></H3><P><B>Investigator:</B> Niels, Sandrina</P><P><B>Aim:</B> generate phage display vector pPDV100 (strategy 1)</P><P><B>Time:</B> 2011-07-20,16:30-18:00</P><P><B>Method/Materials:</B></P><UL><LI> 3 µl (ca 90 ng) Sfi-digested vector pak100
</LI></UL><UL><LI> 1 µl (ca 1000 ng) Sfi-digested PCR fragment mdnA from over night digested sample (2011-08-09)
</LI></UL><UL><LI> 2 µl 10x T4 Ligase Buffer
</LI></UL><UL><LI> 1 µl T4 Ligase
</LI></UL><UL><LI> 13 µl water
</LI></UL><UL><LI> incubate over night at 14°C
</LI></UL><P><B>Further Tasks:</B> transformation of competent cells
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="61th_Labday_2011-08-12">61th Labday 2011-08-12</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI_or_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI_with_HindIII_iGEM_AraC_NgoMIV"> Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI or NgoMIV_iGEM_TEV-Protease_iGEM_BamHI with HindIII_iGEM_AraC_NgoMIV </SPAN></H3><P><B>Investigators:</B> Sebastian, Paul
</P><P><B>Aim</B>:
</P><P>1. Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI (573bp) and HindIII_iGEM_AraC_NgoMIV (1273bp)</P><P>2. Ligation of NgoMIV_iGEM_TEV_iGEM_BamHI (760bp) and HindIII_iGEM_AraC_NgoMIV (1273bp)</P><P>3. Amplification of ligated fragments via PCR
</P><P>Calculation of volumes to be used with: <A href="http://www.gibthon.org/ligate.html" class="external text" rel="nofollow">ligation calculator</A> with 1:1 molar ratio
</P><P><B>Materials:</B></P><P>1:
</P><UL><LI> 4.3 µL NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI fragment (573bp, 3.2ng/µl)
</LI></UL><UL><LI> 3.7 µL AraC fragment
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P>2:
</P><UL><LI> 1.1 µL NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment (760bp, 18.5ng/µl) (TEVI, see 10.08.2011)
</LI></UL><UL><LI> 4 µL AraC fragment
</LI></UL><UL><LI> 2.9 µL pure water
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P><B> Used method: </B></P><P>ligation at room temperatur for 1h
</P><P>3:
</P><UL><LI>10µl of ligation reaction batch
</LI></UL><UL><LI>1µl dNTP
</LI></UL><UL><LI>2.5µl of primer1
</LI></UL><UL><LI>2.5µl of primer2
</LI></UL><UL><LI>5µl 25mM MgCl2
</LI></UL><UL><LI>5µl Amplification Buffer 10x (genaxxon)
</LI></UL><UL><LI>0.5µl Taq-Polymerase (genaxxon)
</LI></UL><P>Primer:
</P><P>TEV+AraC:
</P><P>r_Tev_iGEM_BamHI
</P><P>f_AraC_HindIII_iGEM
</P><P>14_3C+AraC:
</P><P>r_14_3C_iGEM_BamHI
</P><P>f_AraC_HindIII_iGEM
</P><P><B> Used method: </B></P><UL><LI>Initial Denat: 3min 94°C
</LI></UL><UL><LI>25x
</LI></UL><UL><LI>Denat: 2.02min 94°C
</LI></UL><UL><LI>Anneal: 2.02min 70°C
</LI></UL><UL><LI>Extension: 2.02min 72°C
</LI></UL><UL><LI>final extension: 10min 72°C
</LI></UL><P><B>Results:</B></P><P>No results, no expected bands were visible
</P><P><B> Further Tasks: </B></P><P>resolving of PCR products on preparative 1% agarose gel and exciccion of correspoinding bands (1:~1800bp, 2:~2000bp)
</P><P><A href="/File:UP_AG_1%25_2011-08-12_prep-TEV_AraC.jpg" title="File:UP AG 1% 2011-08-12 prep-TEV AraC.jpg">File:UP AG 1% 2011-08-12 prep-TEV AraC.jpg</A></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Colony_PCR_of_TEV_clones_obtained_from_transformation"> Colony PCR of TEV clones obtained from transformation </SPAN></H3><P><B>Stefan, Sebastian</B></P><P><B>Aim:</B></P><P>get a positive clone
</P><P><B>Materials:</B></P><UL><LI>Primer:
</LI></UL><UL><LI>f_AraC_HindIII_iGEM
</LI></UL><UL><LI>r_TEV_iGEM_BamHI
</LI></UL><UL><LI> 2.5 µL forward Primer (25 pmol)
</LI></UL><UL><LI> 2.5 µL reverse Primer (25 pmol)
</LI></UL><UL><LI> 5 µL 10x Polymerase buffer (Genaxxiom)
</LI></UL><UL><LI> 5 µL MgCL2
</LI></UL><UL><LI> 1 µL dNTPs
</LI></UL><UL><LI> 0.5 µL Taq Polymerase (Genaxxiom)
</LI></UL><UL><LI> 33.5 µL H20
</LI></UL><P>10 colonies were picked
</P><P><B> Used method: </B></P><UL><LI>Initial Denat: 3 min 94°C
</LI></UL><UL><LI>25x
</LI></UL><UL><LI>Denat: 2.02 min 94°C
</LI></UL><UL><LI>Anneal: 2.02 min 70°C
</LI></UL><UL><LI>Extension: 2.02 min 72°C
</LI></UL><UL><LI>final extension: 10min 72°C
</LI></UL><P><B> Further Tasks: </B></P><P>gel electrophoresis of PCR products 1% agarose gel of corresponding bands (1:~1800bp, 2:~2000bp)
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Results_of_Transformation_from_2011-8-11_and_Overnight_Cultures_of_marked_colonies"> Results of Transformation from 2011-8-11 and Overnight Cultures of marked colonies</SPAN></H3><P><B>Time:</B> 2011-8-12, 9:00-9:30</P><P><B>Investigators :</B> Jessica, Nadine, Katharina</P><P><B>Aim:</B> check Transformation</P><P><B>Results:</B></P><P><B>Preparing Overnight Cultures of marked colonies</B></P><P><B>Time:</B> 2011-8-12, 18:00-18:30</P><P><B>Investigators :</B> Jessica</P><P><B>Materials/Method:</B></P><UL><LI> 5 ml LB-Medium with 5 µl antibiotic (either Tet, Amp or Kan)
</LI></UL><UL><LI> cultures:
</LI></UL><UL><LI><B>pSB1T3+YFP I clone I 12.08.11 Jes, pSB1T3+YFP I clone II 12.08.11 Jes, pSB1T3+YFP I clone III 12.08.11 Jes</B></LI></UL><UL><LI><B>pSB1K3+YFP clone I 12.08.11 Jes, pSB1T3+YFP clone II 12.08.11 Jes, pSB1T3+YFP clone III 12.08.11 Jes</B></LI></UL><UL><LI><B>pSB1K3+CFP clone I 12.08.11 Jes, pSB1T3+CFP clone II 12.08.11 Jes, pSB1T3+CFP clone III 12.08.11 Jes</B></LI></UL><UL><LI><B>pSB1A3+CFP clone I 12.08.11 Jes, pSB1A3+CFP clone II 12.08.11 Jes</B></LI></UL><P><B>Further Task:</B></P><UL><LI> repeat Transformation of pSB1T3+YFPII and pSB1A3+YFP
</LI></UL><UL><LI> do miniprep of overnight cultures and confirm insert by digest (see 2011-08-09)</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_generated_pPDV100_from_over_night_ligation_.28strategy_1.29">Transformation of generated pPDV100 from over night ligation (strategy 1)</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B>amplification of pPDV100</P><P><B>Time:</B> 10:00-12:00</P><P><B>Method:</B></P><UL><LI> addition of 10 µl ligation reaction to XL1-blue cells</LI></UL><UL><LI> incubation 25 min on ice,</LI></UL><UL><LI> heat shock 45 sec at 42°C,</LI></UL><UL><LI> incubation 2 min on ice,</LI></UL><UL><LI> addition of 750 µl LB medium,</LI></UL><UL><LI> incubation 60 min at 37 °C and 750 rpm</LI></UL><UL><LI> plating on agar plates containing 100 µg/ml tetracyclin and 100 µg/µl ampicillin</LI></UL><UL><LI> storage over night at 37°C</LI></UL><P><B>Further tasks:</B> control cell clones through test digestion with sfiI
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_PCR_gene_III_for_phage_display_.28strategy_2.29_3">Repeated PCR gene III for phage display (strategy 2)</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine </P><P><B>Time:</B> 2011-08-12,12:30-17:00 </P><P><B>Aim:</B></P><UL><LI> amplificate GeneIII with NgoMIV and AatII restriction sites (strategy 2)</LI></UL><P><B>Primer:</B></P><UL><LI> primer: pf_geneIII_NgoMIV_XbaI_myc and pr_geneIII_iGEM_AatII (geneIII, strategy 2)</LI></UL><P><B>Reaction Components:</B></P><UL><LI> 5 µl pak100blaKDIR
</LI></UL><UL><LI> 0,25 µl Taq Polymerase S (BioScience)
</LI></UL><UL><LI> 1 µl dNTPs
</LI></UL><UL><LI> 1 µl per primer
</LI></UL><UL><LI> 5 µl 10x PCR Buffer S
</LI></UL><UL><LI> 37,75 µl DNase free water
</LI></UL><P><B>Purification:</B></P><UL><LI> NucleoSpin Extract II
</LI></UL><P><B>Further tasks:</B></P><UL><LI> digestion with AatII and NgoMIV
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_Cerulean_and_mVenus_Geneart">Miniprep of overnight cultures of Cerulean and mVenus Geneart</SPAN></H3><P><B>Investigators:</B> Katharina</P><P><B>Time:</B> 2011-08-12, 9:00-10:00</P><P><B> Aim: </B></P><P><B>1. Miniprep:</B></P><UL><LI> 2 overnight cultures
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 30 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> Cerulean </TD><TD> 530.1
</TD></TR><TR><TD> mVenus Geneart</TD><TD> 573.6
</TD></TR></TBODY></TABLE><UL><LI> stored in -20°C (red box, expression backbones)</LI></UL><P><B>2. Preparation of glycerol stocks:</B></P><UL><LI> adding 300 µl glycerol to 700 µl culture</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="gel_electrophoresis_of_colony_PCR.28TEV.29">gel electrophoresis of colony PCR(TEV)</SPAN></H3><P><B>Investigators:</B> Sascha, Paul, Sebastian
</P><P><B>Aim:</B> screen for positiv clone
</P><P><B>Result:</B></P><P>Gel electrophoresis of PCR products (5µl) on 1.5% and agarose gels, respectively.
</P><P>Expected band (AraC + TEV): ~ 2000 bp.
</P><P>One positive clone TEV 3 III.
</P><P><A href="/File:UP_AG_1_5%25_2011-08-12_colPCR_AraC-TEV2.jpg" title="File:UP AG 1 5% 2011-08-12 colPCR AraC-TEV2.jpg">600px</A></P><P>Set up a pre-culter of clon TEV 3 III.
</P><P><B>Further tasks:</B></P><P>Stock culture, plasmid preparation</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_pSB1A3.2BYFP_in_XL1-Blue_and_pSB1T3.2BYFPII_in_RV308">Transformation of pSB1A3+YFP in XL1-Blue and pSB1T3+YFPII in RV308</SPAN></H3><P><B>Investigators:</B> Katharina</P><P><B>Aim:</B> Transformation of Ligations</P><P><B>Time:</B> 2011-08-12, 10:00-13:00</P><P><B>Materials:</B></P><UL><LI> competent <I>E. coli</I> cells (XL1-Blue and RV308, respectively)
</LI></UL><UL><LI> ligation products: pSB1A3+YFP, lig, VB, 11.8.2011 and pSB1T3+YFP II, lig, VB, 11.8.2011
</LI></UL><P><B>Method:</B></P><UL><LI>addition of 2 µl ligation reaction to cells (XL1-blue, RV) in 1.5 ml Eppi,
</LI></UL><UL><LI>incubation 25 min on ice,
</LI></UL><UL><LI>heat shock 45 sec at 42°C,
</LI></UL><UL><LI>incubation 5 min on ice,
</LI></UL><UL><LI>addition of 750 µl LB medium,
</LI></UL><UL><LI>incubation at 37 °C shaking for 80 min,
</LI></UL><UL><LI>plating on LB medium with appropriate antibiotic (Amp and Tet,respectively
</LI></UL><UL><LI>storage over night at 37°C
</LI></UL><UL><LI>2 plates: <B>pSB1T3+YFP II Jes</B> and <B>pSB1A3+YFP Jes</B></LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Design_and_ordering_of_primers_to_produce_BioBricks_of_the_mdn_genes">Design and ordering of primers to produce BioBricks of the mdn genes</SPAN></H3><P><B>Investigators:</B> Jessica, Nadine, Katharina</P><P><B>Time:</B> 2011-08-12, 11:00-14:00</P><P><B>Materials:</B></P><UL><LI> Geneious
</LI></UL><P><B>Results:</B></P><UL><LI>pf_mdnABC_EcoRI_NotI_XbaI 12.08. : TTAATGAATTCGCGGCCGCTTCTAGATGGCATATCCCAACGATC
</LI></UL><UL><LI>pf_mdnB_EcoRI_NotI_XbaI 12.08.: ATTATGAATTCGCGGCCGCTTCTAGATGAAAGAATCGCCTAAAGTTG
</LI></UL><UL><LI>pf_mdnC_EcoRI_NotI_XbaI 12.08.: TATTTGAATTCGCGGCCGCTTCTAGATGACCGTTTTAATTGTTAC
</LI></UL><UL><LI>pf_mdnD_EcoRI_NotI_XbaI 12.08.: TATATGAATTCGCGGCCGCTTCTAGATGAAAGCACTGGAAAAACTG
</LI></UL><UL><LI>pf_mdnE_EcoRI_NotI_XbaI 12.08.: TAAATGAATTCGCGGCCGCTTCTAGATGCCTCAATATACTACTAAAC
</LI></UL><UL><LI>pr_mdnABC_SpeI_NotI_PstI 12.08.: ATTTCTGCAGCGGCCGCTACTAGTATTATGAGTTAACTAGGATTTC
</LI></UL><UL><LI>pr_mdnB_SpeI_NotI_PstI 12.08.: TAATCTGCAGCGGCCGCTACTAGTAACTAAAAAATCAGCGATCGC
</LI></UL><UL><LI>pr_mdnD_SpeI_NotI_PstI 12.08.: ATTTCTGCAGCGGCCGCTACTAGTATCAGCAAACCCTACTTAATTTC
</LI></UL><UL><LI>pr_mdnE_SpeI_NotI_PstI 12.08.: ATTTCTGCAGCGGCCGCTACTAGTACTATATTCTCACCCATTTTAAG
</LI></UL><P><B>Further tasks:</B></P><UL><LI> develop PCR program
</LI></UL><UL><LI> PCR
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="62th_Labday_2011-08-13">62th Labday 2011-08-13</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Mini-Prep_of_pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV_and_creation_of_glycerol_stock_culture">Mini-Prep of pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV and creation of glycerol stock culture</SPAN></H3><P><B>Investigator:</B> Sebastian</P><P><B> Aim: </B></P><UL><LI> Isolation of created plasmid pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV and establishing of an <I>E. coli</I> Xl1 blue glycerol stock culture for later use</LI></UL><P><B> Materials:</B></P><UL><LI>Overnight culture of<I>E. coli</I> XL1 Blue transformed with pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV plasmid</LI></UL><UL><LI>NucleoSpin Plasmid (NoLid) Kit purchased by Macherey-Nagel</LI></UL><P><B>Method:</B></P><P><I>preparing the stock culture </I></P><UL><LI>350 µl sterile glycerol where mixed with 350 µl form the overnight culture of <I>E. coli</I> XL1 blue transformed with plasmid
</LI></UL><P>pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV, vortexed and stored at -80°C (stock number: G1)</P><P><I>Mini-prep of pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV
</I></P><UL><LI>preparation was performed as described in the manual of the used Kit </LI></UL><P><B> Results: </B></P><UL><LI>glycerol stock culture G1: <I>E. coli</I> (XL1 blue) transformed with pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV </LI></UL><UL><LI>isolated plasmid pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV</LI></UL><P><B> Further Tasks: </B></P><UL><LI>sequencing of the plasmid pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV</LI></UL><UL><LI>growth test for the stock culture at different conditions (with induction by IPTG and arabinose at increasing ampicillin concentrations) --&gt; &quot;survival test&quot; </LI></UL><UL><LI>preparing of competent cells, which are transformed with pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_pSB1X3_.2B_Y_.28X_-_A.2FT.2FK_.3B_Y_-_YFP.2FCFP.29">Miniprep of pSB1X3 + Y (X - A/T/K ; Y - YFP/CFP) </SPAN></H3><P><B>Investigators:</B> Niels</P><P><B>Aim:</B> isolate plasmid of pSB1X3 + Y (X - A/T/K ; Y - YFP/CFP)</P><P><B>Material:</B> 5 ml / over night culture</P><UL><LI> pSB1T3+YFP I clone I 12.08.11 Jes
</LI></UL><UL><LI> pSB1T3+YFP I clone II 12.08.11 Jes
</LI></UL><UL><LI> pSB1T3+YFP I clone III 12.08.11 Jes
</LI></UL><UL><LI> pSB1K3+YFP clone I 12.08.11 Jes
</LI></UL><UL><LI> pSB1T3+YFP clone II 12.08.11 Jes
</LI></UL><UL><LI> pSB1T3+YFP clone III 12.08.11 Jes
</LI></UL><UL><LI> pSB1K3+CFP clone I 12.08.11 Jes
</LI></UL><UL><LI> pSB1T3+CFP clone II 12.08.11 Jes
</LI></UL><UL><LI> pSB1T3+CFP clone III 12.08.11 Jes
</LI></UL><UL><LI> pSB1A3+CFP clone I 12.08.11 Jes
</LI></UL><UL><LI> pSB1A3+CFP clone II 12.08.11 Jes
</LI></UL><P>&lt;b&gt;Method:</P><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl elution buffer<SUB>2</SUB>O
</LI></UL><P>Check concentration with NanoDrop</P><UL><LI> pSB1T3+YFP I clone I 12.08.11 Jes 10,3 ng/µl
</LI></UL><UL><LI> pSB1T3+YFP I clone II 12.08.11 Jes 16,6 ng/µl
</LI></UL><UL><LI> pSB1T3+YFP I clone III 12.08.11 Jes 24,2 ng/µl
</LI></UL><UL><LI> pSB1K3+YFP clone I 12.08.11 Jes 29,7 ng/µl
</LI></UL><UL><LI> pSB1K3+YFP clone II 12.08.11 Jes 22,9 ng/µl
</LI></UL><UL><LI> pSB1K3+YFP clone III 12.08.11 Jes 25,7 ng/µl
</LI></UL><UL><LI> pSB1K3+CFP clone I 12.08.11 Jes 23,4 ng/µl
</LI></UL><UL><LI> pSB1K3+CFP clone II 12.08.11 Jes 37,6 ng/µl
</LI></UL><UL><LI> pSB1K3+CFP clone III 12.08.11 Jes 27,8 ng/µl
</LI></UL><UL><LI> pSB1A3+CFP clone I 12.08.11 Jes 5,8 ng/µl
</LI></UL><UL><LI> pSB1A3+CFP clone II 12.08.11 Jes 21,7 ng/µl
</LI></UL><P>&lt;b&gt;NOTE:</P><UL><LI><B>pSB1T3+YFP/CFP have the same label; we have to check the insert with test digestion</B></LI></UL><P><B>Further tasks:</B></P><UL><LI> digest
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="63th_Labday_2011-08-14">63th Labday 2011-08-14</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_PCR_of_mdnA_and_gene_III_for_phage_display_.28strategy_1.2B2.29">Repeated PCR of mdnA and gene III for phage display (strategy 1+2)</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine </P><P><B>Time:</B> 2011-08-14,11:00-14:00 </P><P><B>Aim:</B></P><UL><LI> amplification of mdnA with SfiI restriction sites (strategy 1)</LI></UL><UL><LI> amplification of mdnA with NarI and AgeI restriction sites (strategy 2)</LI></UL><UL><LI> amplification of GeneIII with NgoMIV and AatII restriction sites (strategy 2)</LI></UL><UL><LI> 5 PCRs of every gene for testing different digestion and ligation conditions
</LI></UL><P><B>Primer:</B></P><UL><LI> primer 31 and 56: pf_sfi-mdnA_2 and pr_sfi_mdnA_myc-2 (mdnA, strategy 1)</LI></UL><UL><LI> primer 19 and 32: pf_mdnA_iGEM_EheI and pr_mdnA_iGEM_AatII (mdnaA, strategy 2)</LI></UL><UL><LI> primer 54 and 55: pf_geneIII_NgoMIV_XbaI_myc and pr_geneIII_iGEM_AatII (geneIII, strategy 2)</LI></UL><P><B>Reaction Components:</B></P><UL><LI> 2 µl (ca 16 ng) Vector pARW089 (mdnA) or pakblaKDIR (geneIII)
</LI></UL><UL><LI> 0,25 µl Taq Polymerase S (BioScience)
</LI></UL><UL><LI> 1 µl dNTPs
</LI></UL><UL><LI> 1 µl per primer
</LI></UL><UL><LI> 5 µl 10x PCR Buffer S
</LI></UL><UL><LI> 39,75 µl water
</LI></UL><P><B>PCR condition changes for geneIII:</B></P><UL><LI> higher annealing temperature: 60°C
</LI></UL><UL><LI> Stage 1: only 10 cycles (instead of 15)
</LI></UL><UL><LI> purification of PCR fragments with QIAquick Gel Extraction Kit (250)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> digestion
</LI></UL><P><B>Results:</B></P><P>PCRs of mdnA and mdnA were ok, PCR of geneIII: only oligo band
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="digestion_of_pak_blaKDIR_.28strategy_1.29_2">digestion of pak blaKDIR (strategy 1)</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine</P><P><B>Aim:</B> cut geneIII out of pak100 blaKDIR to use it as template for PCR of geneIII
</P><P><B>Time:</B> 2011-08-14, 12:00-15:00</P><P><B>reaction components:</B></P><UL><LI> 5 µl pak blaKDIR (ca 350 ng)
</LI></UL><UL><LI> 2 µl NEB 10x buffer 2
</LI></UL><UL><LI> 1 µl restriction enzyme AvaI
</LI></UL><UL><LI> 1 µl restriction enzyme HindIII
</LI></UL><UL><LI> 11 µl water
</LI></UL><P><B>reaction conditions:</B></P><UL><LI> 3 h, 37°C
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="overnight_culture_of_10_picked_clones_of_XL_blue_cells_transformed_with_pPDV100">overnight culture of 10 picked clones of XL blue cells transformed with pPDV100</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B> amplification and purification of generated phage display vector pPDV100 for test digestion and sequencing</P><P><B>Time:</B> 16:00-16:30</P><P><B>Method/Materials:</B></P><P>5 ml LB medium per clone containining 100 µg/ml chloramphenicol
</P><P>storage over night at 37°C and 800 rpm</P><P><B>Further tasks:</B></P><P>plasmid preparation, test digestion and sequencing</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="generate_over_night_culture_of"> generate over night culture of </SPAN></H3><P><B>Investigators:</B> Niels</P><P><B>Aim:</B> generate a over night culture for isolating the plasmid </P><P><B>Material:</B></P><UL><LI> 5 ml LB Medium
</LI></UL><UL><LI> 5 µl ampicilin (20 mg/ml)
</LI></UL><UL><LI> plate pSB1A3+YFP
</LI></UL><UL><LI> clone I 12.08.11 Jes
</LI></UL><UL><LI> clone II 12.08.11 Jes
</LI></UL><UL><LI> clone III 12.08.11 Jes
</LI></UL><P><B>further tasks </B></P><UL><LI> isolation of plasmid
</LI></UL><UL><LI> digest with
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="64th_Labday_2011-08-15">64th Labday 2011-08-15</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_pSB1A3_.2B_YFP_clone_I.2C_pSB1A3_.2B_YFP_clone_II.2C_pSB1A3_.2B_YFP_clone_III">Miniprep of pSB1A3 + YFP clone I, pSB1A3 + YFP clone II, pSB1A3 + YFP clone III </SPAN></H3><P><B>Investigators:</B> Steffi</P><P><B>Time:</B> 2011-08-15, 07:00-10:00 </P><P><B>Aim:</B> isolate plasmid of pSB1A3 + YFP clone I, pSB1A3 + YFP clone II, pSB1A3 + YFP clone III </P><P><B>Material:</B> 5 ml over night culture
</P><UL><LI> pSB1A3+YFP I clone I 12.08.11 Jes
</LI></UL><UL><LI> pSB1A3+YFP I clone II 12.08.11 Jes
</LI></UL><UL><LI> pSB1A3+YFP I clone III 12.08.11 Jes
</LI></UL><P><B>Method:</B></P><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl elution buffer<SUB>2</SUB>O
</LI></UL><P>Check concentration with NanoDrop and Agarosegel
</P><UL><LI> pSB1A3+YFP I clone I: 134.3 ng/µl
</LI></UL><UL><LI> pSB1A3+YFP I clone II: 113.8 ng/µl
</LI></UL><UL><LI> pSB1A3+YFP I clone III: 179.2 ng/µl
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> M </TD><TD> GeneRuler™ DNA Ladder Mix (diluted 1:10) </TD><TD> 6 </TD></TR><TR><TD> 1 </TD><TD> - </TD><TD> - </TD></TR><TR><TD> 2 </TD><TD> pSB1A3+YFP I clone I 12.08.11 Jes </TD><TD> 1 </TD></TR><TR><TD> 3 </TD><TD> pSB1A3+YFP I clone II 12.08.11 Jes </TD><TD> 1 </TD></TR><TR><TD> 4 </TD><TD> pSB1A3+YFP I clone III 12.08.11 Jes </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> digest
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Test_digest_of_minipreps_of_pSB1X3.2BYFP.2FCPF_from_2011-08-13_and_2011-08-15">Test digest of minipreps of pSB1X3+YFP/CPF from 2011-08-13 and 2011-08-15</SPAN></H3><P><B>Investigators:</B> Jessica, Laura, Steffi, Vanessa </P><P><B> Time: </B> 2011-08-15, 10:00-15:00</P><P><B> Aim: </B> prove of Insert YFP/CPF</P><P><B>Materials:</B></P><P>pSB1T3+YFP I clone I 12.08.11 Jes pSB1T3+YFP I clone II 12.08.11 Jes pSB1T3+YFP I clone III 12.08.11 Jes pSB1K3+YFP clone I 12.08.11 Jes pSB1T3+YFP clone II 12.08.11 Jes pSB1T3+YFP clone III 12.08.11 Jes pSB1K3+CFP clone I 12.08.11 Jes pSB1T3+CFP clone II 12.08.11 Jes pSB1T3+CFP clone III 12.08.11 Jes pSB1A3+CFP clone I 12.08.11 Jes pSB1A3+CFP clone II 12.08.11 Jes
</P><UL><LI> ApaLI, ClaI, HaeII, BglI
</LI></UL><UL><LI> NEB Buffer 4
</LI></UL><UL><LI> NEB Buffer 2
</LI></UL><UL><LI> 100x BSA
</LI></UL><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> DNA Ladder Mix (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Promega)
</LI></UL><P><B> Digestion protocol for pSB1K3+YFP/CFP (with HaeII) </B></P><UL><LI>2 µl DNA
</LI></UL><UL><LI>0.5 µl HaeII
</LI></UL><UL><LI>2 µl 10x buffer = NEB 4
</LI></UL><UL><LI>15.8 µl pure water
</LI></UL><UL><LI>total: 20µl
</LI></UL><P><B> Digestion protocol for pSB1A3+YFP/CFP (with HaeII, BglI) </B></P><UL><LI>2 µl DNA
</LI></UL><UL><LI>0.5 µl HaeII
</LI></UL><UL><LI>0.5 µl BglI
</LI></UL><UL><LI>2 µl 10x buffer = NEB 2
</LI></UL><UL><LI>0.2 µl 100xBSA
</LI></UL><UL><LI>15.3 µl pure water
</LI></UL><P><B> Digestion protocol for pSB1T3+YFPI (with ApaLI, ClaI) </B></P><UL><LI>2 µl DNA
</LI></UL><UL><LI>0.5 µl ApaLI
</LI></UL><UL><LI>0.5 µl ClaI
</LI></UL><UL><LI>2 µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.2 µl 100xBSA
</LI></UL><UL><LI>15.3 µl pure water
</LI></UL><P>37°C for ~1h
</P><P><B> Production of one 1 % agarose gel</B></P><UL><LI> 1 % gel: 0.5 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to each gel
</LI></UL><P><B>Loading gels and running</B></P><UL><LI> Add 4 µl Loading dye to each 20 µl sample
</LI></UL><UL><LI> 12 µl DNA Ladder Mix
</LI></UL><UL><LI> Running conditions: 100 V, approx. 45 min
</LI></UL><P><B> Loading of gels</B></P><P><I>Gel 1</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> GeneRuler™ DNA Ladder Mix (diluted 1:10) </TD><TD> 12 </TD></TR><TR><TD> 1 </TD><TD> pSB1A3+YFP clone I </TD><TD> 24 </TD><TD> 765, 924, 1188
</TD></TR><TR><TD> 2 </TD><TD> - </TD></TR><TR><TD> 3 </TD><TD> pSB1A3+YFP clone II </TD><TD> 24</TD><TD> 765, 924, 1188
</TD></TR><TR><TD> 4 </TD><TD> pSB1A3+YFP clone III </TD><TD> 24 </TD><TD> 765, 924, 1188
</TD></TR><TR><TD> 5 </TD><TD> pSB1A3+CFP clone I</TD><TD> 24</TD><TD> 765, 924, 1188
</TD></TR><TR><TD> 6 </TD><TD> pSB1A3+CFP clone II</TD><TD> 24</TD><TD> 765, 924, 1188
</TD></TR></TBODY></TABLE><P><I>Gel 2</I></P><TABLE class="wikitable" border="1"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> GeneRuler™ DNA Ladder Mix (diluted 1:10) </TD><TD> 12 </TD></TR><TR><TD> 1 </TD><TD> pSB1K3+YFP clone I </TD><TD> 24 </TD><TD> 924, 2010
</TD></TR><TR><TD> 2 </TD><TD> pSB1K3+YFP clone II </TD><TD> 24 </TD><TD> 924, 2010
</TD></TR><TR><TD> 3 </TD><TD> pSB1K3+CFP clone I </TD><TD> 24 </TD><TD> 924, 2010
</TD></TR><TR><TD> 4 </TD><TD> pSB1K3+CFP clone II </TD><TD> 24 </TD><TD> 924, 2010
</TD></TR><TR><TD> 5 </TD><TD> pSB1K3+CFP clone III </TD><TD> 24 </TD><TD> 924, 2010
</TD></TR><TR><TD> 6 </TD><TD> pSB1K3+YFP clone III </TD><TD> 24 </TD><TD> 924, 2010
</TD></TR><TR><TD> 7 </TD><TD> - </TD></TR><TR><TD> 8 </TD><TD> pSB1T3+YFP I clone I </TD><TD> 24 </TD><TD> 536, 869, 1788
</TD></TR><TR><TD> 9 </TD><TD> pSB1T3+YFP I clone II </TD><TD> 24 </TD><TD> 536, 869, 1788
</TD></TR><TR><TD> 10 </TD><TD> pSB1T3+YFP I clone III </TD><TD> 24 </TD><TD> 536, 869, 1788
</TD></TR></TBODY></TABLE><P><B> Result: </B></P><UL><LI> Inserts couldn't be confirmed
</LI></UL><UL><LI> possible explanation: concentration of insert to low before ligation
</LI></UL><P><B> Further tasks: </B></P><UL><LI> repeat digest of pGA14mVenusGeneart and pGA14-Cerulean to get YFP and CFP
</LI></UL><UL><LI> repeat ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_E._coli_RV308_with_pSB1T3.2BYFPII">Transformation of <I>E. coli</I> RV308 with pSB1T3+YFPII</SPAN></H3><P><B>Investigators:</B> Niels, Jessica, Steffi </P><P><B>Aim:</B>Transformation of <I>E. coli</I> RV - cells with </P><P><B>Materials:</B></P><UL><LI>ligation of pSB1T3+YFPII from 2011-08-11
</LI></UL><UL><LI><I>E. coli</I> RV308
</LI></UL><UL><LI> LB medium
</LI></UL><P><B>Method:</B></P><UL><LI> addition of 2 µl plasmid to RV - cells</LI></UL><UL><LI> incubation 30 min on ice,</LI></UL><UL><LI> heat shock 90 sec at 42°C,</LI></UL><UL><LI> incubation 2 min on ice,</LI></UL><UL><LI> addition of 750 µl LB medium,</LI></UL><UL><LI> incubation 75 min at 37 °C and 700 rpm</LI></UL><UL><LI> plating on agar plates containing 100 µg/µl tetracycline</LI></UL><UL><LI> storage over night at 37°C</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="mini_prep_and_test_digestion_of_phage_display_vector_pPDV100_from_over_night_cultures_.28strategy_1.29">mini prep and test digestion of phage display vector pPDV100 from over night cultures (strategy 1)</SPAN></H3><P><B>Investigators:</B> Laura, Sabine</P><P><B>Aim:</B> test digestion of pPDV100 to control mdnA cloning into pak blaKDIR</P><P><B>Time:</B> 2011-08-15,12:00-14:15 and 16.00-18.00</P><P>Digestion of vector pPDV100 with Sfi (10 clones)
</P><UL><LI> 10 µl sample
</LI></UL><UL><LI> 2 µl NEB 10x buffer 4
</LI></UL><UL><LI> 1 µ restriction enzyme SfiI
</LI></UL><UL><LI> 7 µl water
</LI></UL><UL><LI> over night at 50°C
</LI></UL><P>Digestion of vector pPDV100 with PvuII (10 clones)
</P><UL><LI> 10 µl sample
</LI></UL><UL><LI> 2 µl NEB 10x buffer 4
</LI></UL><UL><LI> 1 µ restriction enzyme PvuII
</LI></UL><UL><LI> 7 µl water
</LI></UL><UL><LI> over night at 37°C
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="digestion_of_vector_pARW089_.28strategy_2.29_2">digestion of vector pARW089 (strategy 2)</SPAN></H3><P><B>Investigators:</B> Laura, Sabine</P><P><B>Aim:</B> cloning od mdnA/geneIII fusion gene into pARW089</P><P><B>Time:</B> 2011-08-15,17:30-18:00 </P><P><B>Reaction components:</B></P><UL><LI> 5 µl pARW089
</LI></UL><UL><LI> 2 µl NEB 10x buffer 4
</LI></UL><UL><LI> 1 µ restriction enzyme NarI
</LI></UL><UL><LI> 1 µ restriction enzyme AatII
</LI></UL><UL><LI> 11 µl water
</LI></UL><P><B>Reaction conditions:</B></P><UL><LI> over night at 37°C
</LI></UL><P><B>Results:</B></P><UL><LI> no of the expected bands (10,2 kb and 160 bp)
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Survival_test_for_E._coli_XL1_blue_transformed_with_pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV"> Survival test for <I>E. coli</I> XL1 blue transformed with pUP_SG1_ssTorA_CS-TEV_bla_AraC-TEV </SPAN></H3><P><B>Investigator:</B> Sebastian, Stefan, Sascha</P><P><B>Aim:</B></P><UL><LI>testing the influence of the induction with IPTG and arabinose (different concentrations)</LI></UL><UL><LI>capacity of resistence vs. ampicillin after induction of TorA_CS-TEV_bla with IPTG</LI></UL><P><B>Materials:</B></P><UL><LI> LB-Agar</LI></UL><UL><LI> deluted overnight culture of <I>E. coli</I> XL1-blue transformed with pUP_SG1_TorA_CS-TEV_bla_AraC-TEV</LI></UL><UL><LI> LB-Media</LI></UL><UL><LI> Stock solutions of 1M IPTG, 1M arabinose (ara), 100mg/ml ampicillin (amp) and 25 mg/ml chloramphenicol (cm)</LI></UL><P><B>Methode:</B></P><P>100µl of deluted overnight culture of <I>E. coli</I> XL1-blue transformed with pUP_SG1_TorA_CS-TEV_bla_AraC-TEV (OD (600 nm)=0.002) were plated on different perpared agar plates and incubated over night at 30°C.</P><UL><LI><I>used plates:</I></LI></UL><UL><LI>total plates: 15</LI></UL><UL><LI> 1 plate with cm (25µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG </LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, amp (50 µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, amp (100 µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, amp (200 µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, amp (400 µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, amp (800 µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM ara</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 5 mM ara</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 10 mM ara</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 20 mM ara</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, 1 mM ara</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, 5 mM ara</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, 10 mM ara</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, 20 mM ara</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Survival_test_for_E.coli_XL1_blue_transformed_with_pUP_SG2_ssTorA_CS-Pre_bla"> Survival test for E.coli XL1 blue transformed with pUP_SG2_ssTorA_CS-Pre_bla </SPAN></H3><P><B>Investigator:</B> Sebastian, Stefan, Sascha</P><P><B>Aim:</B></P><UL><LI>testing the influence of the induction with IPTG and arabinose (different concentrations)</LI></UL><UL><LI>capacity of resistence vs. ampicillin after induction of TorA_CS-TEV_bla with IPTG</LI></UL><P><B>Materials:</B></P><UL><LI> LB-Agar</LI></UL><UL><LI> deluted overnight culture of E.coli XL1-blue transformed with pUP_SG1_TorA_CS-TEV_bla_AraC-TEV</LI></UL><UL><LI> LB-Media</LI></UL><UL><LI> Stock solutions of 1M IPTG, 1M arabinose (ara), 100mg/ml ampicillin (amp) and 25 mg/ml chloramphenicol (cm)</LI></UL><P><B>Methode:</B></P><P>100µl of deluted overnight culture of E.coli XL1-blue transformed with pUP_SG1_TorA_CS-TEV_bla_AraC-TEV (OD (600 nm)=0.002) were plated on different perpared agar plates and incubated over night at 30°C.</P><UL><LI><I>used plates:</I></LI></UL><UL><LI>total plates: 15</LI></UL><UL><LI> 1 plate with cm (25µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG </LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, amp (50 µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, amp (100 µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, amp (200 µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, amp (400 µg/ml)</LI></UL><UL><LI> 1 plate with cm (25µg/ml), 1 mM IPTG, amp (800 µg/ml)</LI></UL><P><B>Results:</B></P><P><B>Further tasks:</B><B>Results:</B></P><P><B>Further tasks:</B></P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="65th_Labday_2011-08-16">65th Labday 2011-08-16</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_YFP_from_pGA14mVenusGeneart_and_CFP_from_pGA14-Cerulean_3">Digestion of YFP from pGA14mVenusGeneart and CFP from pGA14-Cerulean </SPAN></H3><P><B>Time:</B> 2011-08-16,08:00-14:00</P><P><B>Investigators:</B> Steffi </P><P><B>Materials:</B></P><P>Preparation for ligation of YFP/CFP (insert) into pSB1A3, pSB1K3 or pSB1T3 (vectors)
</P><P><B>Materials:</B></P><UL><LI> pGA14mVenusGeneart and pGA14-Cerulean
</LI></UL><UL><LI> EcoRI-HF, PstI-HF
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> 100x BSA
</LI></UL><P><B>Protocol:</B></P><P><B> Digestion protocol (following iGEM distribution protocol for linearized backbones):</B></P><UL><LI>Mastermix
</LI></UL><UL><LI>2 µl NEB Buffer 4
</LI></UL><UL><LI>0.2 µl BSA
</LI></UL><UL><LI>1 µl EcoRI-HF
</LI></UL><UL><LI>1 µl PstI-HF
</LI></UL><UL><LI>10,8 µl pure water
</LI></UL><UL><LI>total: 15 µl
</LI></UL><UL><LI> reaction mix:
</LI></UL><UL><LI>15 µl mastermix + 5 µl DNA
</LI></UL><UL><LI> Incubation:
</LI></UL><UL><LI>37°C for 1 h
</LI></UL><UL><LI> Heat deactivation:
</LI></UL><UL><LI> 80°C for 20 min
</LI></UL><P><B>Further Task: </B></P><UL><LI> gel electrophoresis
</LI></UL><UL><LI> gel extraction and purification
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_Gel_digested_CFP_and_YFP_fragments">Agarose Gel digested CFP and YFP fragments </SPAN></H3><P><B>Investigators:</B> Steffi, Jessica, Nicole </P><P><B>Aim:</B>Purification of YFP and CFP fragment</P><P><B>Time:</B> 2011-08-16, 11:00-12:00 </P><P><B>Materials:</B></P><UL><LI> digested CFP and YFP
</LI></UL><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> DNA Ladder Mix (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas)
</LI></UL><P><B> Production of one 1 % agarose gel</B></P><UL><LI> 1 % gel: 0.5 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red gel
</LI></UL><P><B>Loading gel and running</B></P><UL><LI> Add 5 µl Loading dye to each 20 µl sample
</LI></UL><UL><LI> 6 µl DNA Ladder Mix
</LI></UL><UL><LI> Running conditions: 85 V, approx. 1 h
</LI></UL><P><B> Loading of gel </B></P><UL><LI>gel: CFP and YFP
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> 1 </TD><TD> marker </TD><TD> 6</TD></TR><TR><TD> 2 </TD><TD> -</TD><TD> -</TD><TD> -
</TD></TR><TR><TD> 3</TD><TD> CFP </TD><TD> 20</TD><TD> 771, 2873
</TD></TR><TR><TD> 4 </TD><TD> -</TD><TD> -</TD><TD>-
</TD></TR><TR><TD> 5 </TD><TD> YFP </TD><TD> 20</TD><TD> 763, 2881
</TD></TR><TR><TD> 6 </TD><TD> - </TD><TD> -</TD><TD> -
</TD></TR></TBODY></TABLE><P><B> Result: </B></P><UL><LI> bands appear as expected
</LI></UL><UL><LI> lower bands (red box) in lane 3 and 4 were excised and DNA was extracted w/ Wizard SV Gel and PCR Clean-Up System (Promega):
</LI></UL><UL><LI><B>CFP dig. pur. Jes &amp; VB 11.8.2011</B>, ???? ng/µl
</LI></UL><UL><LI><B>YFP dig. pur. Jes &amp; VB 11.8.2011</B>, ???? ng/µl
</LI></UL><P><B>Further tasks:</B></P><UL><LI> ligation of digested and purified YFP and CFP with digested and purified pSB1A3, pSB1K3 and pSB1T3
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Sequencing_of_pSB1A3_and_pSB1K3_carrying_Cerulean_resp._mVenus">Sequencing of pSB1A3 and pSB1K3 carrying Cerulean resp. mVenus </SPAN></H3><P><B>Investigators:</B>Nicole </P><P><B>Time:</B> 2011-08-16, 13:00-14:00 </P><P><B>Aim:</B> Confirmation of ligation of linearized backbones pSB1A3 and pSB1K3 with mVenus and Cerulean as dummies</P><P><B>Materials:</B></P><UL><LI> pSB1A3 carrying Cerulean, miniprep done by Nadja/Nicole, c<SMALL>DNA</SMALL> = 164,1 ng/ µl
</LI></UL><UL><LI> pSb1A3 carrying mVenus, miniprep done by Steffi, c<SMALL>DNA</SMALL> = 179,2 ng/ µl
</LI></UL><UL><LI> pSB1K3 carrying Cerulean, miniprep done by Nadja/Nicole, c<SMALL>DNA</SMALL> = 216,2 ng/ µl
</LI></UL><UL><LI> pSb1K3 carrying mVenus, miniprep done by Nadja/Nicole, c<SMALL>DNA</SMALL> = 178,9 ng/ µl
</LI></UL><UL><LI> Sequencing primer: VF2
</LI></UL><UL><LI> Freelabels for Value Read Tube (MWG Eurofins)
</LI></UL><P><B>Method:</B></P><P><A href="/File:UP_2011-08-16_pSB1A3_pSB1K3_CFP_YFP_VF2.jpg" title="File:UP 2011-08-16 pSB1A3 pSB1K3 CFP YFP VF2.jpg">File:UP 2011-08-16 pSB1A3 pSB1K3 CFP YFP VF2.jpg</A></P><UL><LI> DNA concentration (for sequencing): 70 ng/ µl
</LI></UL><UL><LI> Total volume: 15 µl
</LI></UL><UL><LI> Primer concentration: 2 pmol/ µl
</LI></UL><UL><LI> Total volume: 50 µl
</LI></UL><UL><LI> sent to MWG Eurofins
</LI></UL><P><B>Results:</B></P><UL><LI> expected 2011-08-18, afternoon
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Analyzing sequencing results
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Planning_and_accomplishing_the_control_PCRs_of_pSB1A3_and_pSB1K3_carrying_mVenus_resp._Cerulean">Planning and accomplishing the control PCRs of pSB1A3 and pSB1K3 carrying mVenus resp. Cerulean </SPAN></H3><P><B>Investigators:</B> Jessica, Nicole </P><P><B>Time:</B> 2011-08-16, 14:00-16:00 </P><P><B>Aim:</B> Confirmation of ligation of linearized backbones pSB1A3 and pSB1K3 with mVenus and Cerulean as dummies; therefore perfoming PCRs using VF2 and VR2 primers, which bind in the backbone and lead to amplification of insert
</P><P><B>Materials:</B></P><P>1. Plasmids </P><UL><LI> pSB1A3 carrying Cerulean, miniprep done by Nadja/Nicole, c<SMALL>DNA</SMALL> = 164,1 ng/ µl
</LI></UL><UL><LI> pSb1A3 carrying mVenus, miniprep done by Steffi, c<SMALL>DNA</SMALL> = 179,2 ng/ µl
</LI></UL><UL><LI> pSB1K3 carrying Cerulean, miniprep done by Nadja/Nicole, c<SMALL>DNA</SMALL> = 216,2 ng/ µl
</LI></UL><UL><LI> pSb1K3 carrying mVenus, miniprep done by Nadja/Nicole, c<SMALL>DNA</SMALL> = 178,9 ng/ µl
</LI></UL><P>2. Primer </P><UL><LI> VF2
</LI></UL><UL><LI> VR2
</LI></UL><UL><LI> bind in the backbone and lead to amplification of insert
</LI></UL><P>3. Other materials
</P><UL><LI> Polymerase S and corresponding buffer
</LI></UL><P><B>Method:</B></P><P>1. Reaction mix (volumes in µl)
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH><B>Ingredient</B></TH><TH><B>pSB1A3+CFP</B></TH><TH><B>pSB1A3+YFP</B></TH><TH><B>pSB1K3+CFP</B></TH><TH><B>pSB1K3+YFP</B></TH></TR><TR><TD>DNA (1:100)</TD><TD> 6.0</TD><TD> 5.6</TD><TD> 4.6</TD><TD> 5.6
</TD></TR><TR><TD> Buffer (15 mM MgCl<SMALL>2</SMALL>)</TD><TD> 5.0 </TD><TD> 5.0</TD><TD> 5.0</TD><TD> 5.0
</TD></TR><TR><TD> dNTPs (10 mM each)</TD><TD> 1.0 </TD><TD> 1.0</TD><TD> 1.0</TD><TD> 1.0
</TD></TR><TR><TD> MgCl<SMALL>2</SMALL> (25 mM)</TD><TD> 2.0</TD><TD> 2.0</TD><TD> 2.0</TD><TD> 2.0
</TD></TR><TR><TD> VF2 (10 mM)</TD><TD> 1.0</TD><TD> 1.0</TD><TD> 1.0</TD><TD> 1.0
</TD></TR><TR><TD>VR2 (10 mM)</TD><TD> 1.0</TD><TD> 1.0</TD><TD> 1.0</TD><TD> 1.0
</TD></TR><TR><TD> Genaxxon Polymerase S </TD><TD> 0.3 </TD><TD> 0.3 </TD><TD> 0.3 </TD><TD> 0.3
</TD></TR><TR><TD> H<SMALL>2</SMALL>O </TD><TD> 33.7 </TD><TD> 34.1</TD><TD> 35.1</TD><TD> 34.1
</TD></TR></TBODY></TABLE><P>2. PCR program (IGCONT1)
</P><TABLE class="wikitable" border="1"><TBODY><TR><TH><B>Step</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>Hot Start</TD><TD> 94°C </TD><TD> Hold
</TD></TR><TR><TD> Initial denaturation</TD><TD> 94°C </TD><TD> 3 min
</TD></TR><TR><TD> Denaturation</TD><TD> 94°C</TD><TD> 45 s
</TD></TR><TR><TD> Annealing</TD><TD> 50°C</TD><TD> 45 s
</TD></TR><TR><TD>Extension</TD><TD> 72°C</TD><TD> 70 s
</TD></TR><TR><TD> Final extension </TD><TD> 72°C </TD><TD> 10 min
</TD></TR></TBODY></TABLE><UL><LI> 30 cycles of denaturation, annealing and extension
</LI></UL><P><B>Agarose gel electrophoresis</B></P><TABLE class="wikitable" border="1"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> GeneRuler™ DNA Ladder Mix (diluted 1:10) </TD><TD> 12 </TD></TR><TR><TD> 1 </TD><TD> - </TD></TR><TR><TD> 2 </TD><TD> PCR of pSB1A3+CFP </TD><TD> 6 </TD><TD> ~1000
</TD></TR><TR><TD> 3 </TD><TD> PCR of pSB1A3+YFP </TD><TD> 6 </TD><TD> ~1000
</TD></TR><TR><TD> 4 </TD><TD> PCR of pSB1K3+CFP </TD><TD> 6 </TD><TD> ~1000
</TD></TR><TR><TD> 5 </TD><TD> PCR of pSB1K3+YFP </TD><TD> 6 </TD><TD> ~1000
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> according to PCR all 4 plasmids carry the insert
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="DNA_extraction_of_mVenus_and_Cerulean">DNA extraction of mVenus and Cerulean</SPAN></H3><P><B>Investigators:</B> Nicole</P><P><B>Time:</B> 2011-08-16, 15:00-16:00</P><P><B> Aim: </B> DNA of reporter genes mVenus and Cerulean </P><P><B>Materials</B></P><UL><LI> Agarose gel electrophoresis of mVenus and Cerulean, done by Steffi previously
</LI></UL><UL><LI> Machery-Nagel NucleoSpin Extract II, protocol for DNA extraction from agarose gels
</LI></UL><P><B>Method:</B></P><UL><LI> extraction done based on manufacturer's protocol
</LI></UL><P><B>Results:</B></P><P>DNA concentrations measured by Nanodrop
</P><UL><LI> c<SMALL>mVenus</SMALL> = 6.9 ng/ µl
</LI></UL><UL><LI> c<SMALL>Cerulean</SMALL> = 10.6 ng/ µl
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_test_digested_pPDV100_.28strategy_1.29">Agarose gel electrophoresis test digested pPDV100 (strategy 1) </SPAN></H3><P><B>Investigators:</B> Sabine</P><P><B>Aim:</B> control of created pPDV100</P><P><B>Time:</B> 2011-08-16,10:00-12:00</P><P><B>Materials and Method:</B></P><UL><LI> 0,75 % agarose gel&lt;/b&gt;</LI></UL><UL><LI> 100 V
</LI></UL><UL><LI> 1 h </LI></UL><P><B>expected bands:</B></P><UL><LI> SfiI digestion pak bla KDIR with mdnA insert: 200 bp and 4,3 kb
</LI></UL><UL><LI> SfiI digetion pak bla KDIR without mdnA insert: 800 bp and 4,3 kb
</LI></UL><UL><LI> PvuII digestion pak bla KDIR with mdnA insert: 100 bp, 1860 bp and 2550 bp
</LI></UL><UL><LI> PvuII digestion pak bla KDIR with mdnA insert: 100 bp, 2480 bp and 2550 bp
</LI></UL><P><B>Results:</B></P><UL><LI> clone 2 may be a positive clone</LI></UL><P><B>Further tasks:</B></P><UL><LI> send pPDV of clone 2 to Eurofins for sequencing
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Sequencing_of_pPDV100_of_clone_2_.28strategy_1.29">Sequencing of pPDV100 of clone 2 (strategy 1)</SPAN></H3><P><B>Investigators:</B> Sabine</P><P><B>Aim:</B> Control of created pPDV100</P><P><B>Time:</B> 2011-08-16,13:00-14:00</P><P><B>Material/Method:</B></P><UL><LI> Miniprep of clone 2
</LI></UL><UL><LI> Sequencing Primer: sf_mdna_1
</LI></UL><UL><LI> Freelabels for Value ReadTube (MWG Eurofins)
</LI></UL><UL><LI> DNA concentration 70 ng/ µl in a total volume of 15 µl
</LI></UL><UL><LI> Primer concentration: 15 pmol/µl in the total volume of 15 µl (mix)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Analyzing sequencing results
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_pSB1A3.2FpSB1K3_with_CFP.2FYFP_and_PCR_fragments_of_the_promoters_.28Ara.2C_Lac.2C_constitutive.29">Digestion of pSB1A3/pSB1K3 with CFP/YFP and PCR fragments of the promoters (Ara, Lac, constitutive)</SPAN></H3><P><B>Time:</B> 2011-08-16</P><P><B>Investigators:</B> Nicole, Niels, Jessica</P><P><B>Aim:</B></P><P>Preparation for ligation of promoters into pSB1A3, pSB1K3 carrying CFP/YFP
</P><P><B>Materials:</B></P><UL><LI> minipreps of:
</LI></UL><UL><LI> pSB1A3+YFP I clone III: 179.2 ng/µl(from 2011-08-15)
</LI></UL><UL><LI> pSB1A3+CFP, pSB1K3+CFP, pSB1K3+YFP from 2011-08-05
</LI></UL><UL><LI> purified PCR fragments (Ara, Lac, constitutive) from 2011-08-02
</LI></UL><UL><LI> EcoRI-HF, XbaI
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> 100x BSA
</LI></UL><P><B>Method:</B></P><P><B> Digestion protocol (following iGEM distribution protocol for linearized backbones):</B></P><UL><LI>Mastermix
</LI></UL><UL><LI>2 µl NEB Buffer 4
</LI></UL><UL><LI>0.2 µl BSA
</LI></UL><UL><LI>1 µl EcoRI-HF
</LI></UL><UL><LI>1 µl PstI-HF
</LI></UL><UL><LI>10,8 µl pure water
</LI></UL><UL><LI>total: 15 µl
</LI></UL><UL><LI> reaction mix:
</LI></UL><UL><LI>15 µl mastermix + 5 µl DNA
</LI></UL><UL><LI> Incubation:
</LI></UL><UL><LI>37°C for 1 h
</LI></UL><UL><LI> Heat deactivation:
</LI></UL><UL><LI> 80°C for 20 min
</LI></UL><P><B>Further Task: </B></P><UL><LI> gel electrophoresis
</LI></UL><UL><LI> gel extraction and purification
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/August_part_1">http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/August_part_1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Labjournal/August_part_1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Labjournal/August_part_1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/August_part_1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/August_part_1&amp;oldid=165104" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>