<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Labjournal_September_part_1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Labjournal/September_part_1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Labjournal/September_part_1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/September_part_1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/September_part_1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Labjournal/September_part_1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Labjournal/September part 1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team">Team</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Members">Members</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Project</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Summary</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Highlights">Highlights</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Microviridin">     Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Phage_Display">     Phage Display</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#In_Vivo_Selection">     In Vivo Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Modeling">     Modeling</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Microviridin">Details Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">Details Phage</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">Details Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Modeling">Details Modeling</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">BioBricks</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/BioBricks">Details BioBricks</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Methods">Methods</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Meetings">Meetings</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics">Safety &amp; Ethics</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Assesment">Safety Assessment</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Questions">Safety Questions</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Ethics_Seminar">Ethics Seminar</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Survey">Survey</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Potsdam_Bioware_at_the_German_parliament">Parliament Visit</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Statement_of_the_German_government">     Statement</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Outreach">Outreach</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software">Software</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#BioLog_App">BioLog App</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#Download">Download</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Sponsors">Sponsors</A></DT></DL></LI></UL></DIV><DIV class="pageContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#82th_Labday_2011-09-01"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">82th Labday 2011-09-01</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_pUP089_.28vector.29"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Digest of pUP089 (vector)</SPAN></A></LI><LI class="toclevel-2"><A href="#miniprep_of_overnight_cultures_-_mdnDE.2C_mdnABC"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">miniprep of overnight cultures - mdnDE, mdnABC</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_miniprep_-_mdnDE.2C_mdnABC"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Digest of miniprep - mdnDE, mdnABC</SPAN></A></LI><LI class="toclevel-2"><A href="#Testdigest_of_miniprep_-_mdnDE.2C_mdnABC"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Testdigest of miniprep - mdnDE, mdnABC</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_pUP089_and_Lib-2"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Ligation of pUP089 and Lib-2</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_library2"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Transformation of library2</SPAN></A></LI><LI class="toclevel-2"><A href="#Picking_clones_for_overnight_cultures_of_pSB1C3_.2B_mdnABC_and_DE.2C_respectively"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Picking clones for overnight cultures of pSB1C3 + mdnABC and DE, respectively</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_NgoMIV_TEV-Protease_iGEM_BamHI.2C_NgoMIV_14_3C-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_fragments_and_pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5_and_pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5_vector"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Digestion of NgoMIV_TEV-Protease_iGEM_BamHI, NgoMIV 14_3C-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV fragments and pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5 and pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5 vector</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI_or_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_into_pJC354-NheI-143C-Xho_blaFL_GGH5_or_pJC354-NheI-TEV-Xho_blaFL_GGH5_vector"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI or NgoMIV_iGEM_TEV-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV into pJC354-NheI-143C-Xho_blaFL_GGH5 or pJC354-NheI-TEV-Xho_blaFL_GGH5 vector</SPAN></A></LI><LI class="toclevel-2"><A href="#plasmid_pereperation_and_test_digestion_pPDV089_to_control_deletion_of_kanamycin_gene.2C_pARW089_containing_only_geneIII_.28no_mdnA.29.2C_pSB1C3_containing_mdnA.2C_geneIII_or_mdnA.2FgeneIII"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">plasmid pereperation and test digestion pPDV089 to control deletion of kanamycin gene, pARW089 containing only geneIII (no mdnA), pSB1C3 containing mdnA, geneIII or mdnA/geneIII</SPAN></A></LI><LI class="toclevel-2"><A href="#overnight_culture_of_picked_E._coli_clones_transformed_with_pARW089_containing_only_geneIII_.28no_mdnA.29"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">overnight culture of picked E. coli clones transformed with pARW089 containing only geneIII (no mdnA)</SPAN></A></LI><LI class="toclevel-2"><A href="#Production_of_phages_containing_pPDV089"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">Production of phages containing pPDV089</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#83th_Labday_2011-09-02"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">83th Labday 2011-09-02</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_overnight_cultures_from_ligation_of_pSB1C3_.2B_mdnABC_and_mdnDE.2C_respectively"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Miniprep of overnight cultures from ligation of pSB1C3 + mdnABC and mdnDE, respectively</SPAN></A></LI><LI class="toclevel-2"><A href="#test_digest_of_pSB1C3_.2B_mdnABC_and_DE.2C_respectively"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">test digest of pSB1C3 + mdnABC and DE, respectively</SPAN></A></LI><LI class="toclevel-2"><A href="#Colony_PCR_of_plated_cells_from_01.09.2011_.28see_entry_from_01.09.2011_for_picked_colonies.29"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Colony PCR of plated cells from 01.09.2011 (see entry from 01.09.2011 for picked colonies)</SPAN></A></LI><LI class="toclevel-2"><A href="#over_night_cultures_of_pSB1C3.2BmdnABC_clone_2_and_P1_for_screening_team"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">over night cultures of pSB1C3+mdnABC clone 2 and P1 for screening team</SPAN></A></LI><LI class="toclevel-2"><A href="#plate_transformation_of_pUP089A_and_B_.28XL1-blue.29"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">plate transformation of pUP089A and B (XL1-blue)</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_of_mdnA_with_N-terminal_myc_tag_.28mcyN.29"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">PCR of mdnA with N-terminal myc tag (mcyN)</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_XL1blue-cells_with_pPDV089"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Transformation of XL1blue-cells with pPDV089</SPAN></A></LI><LI class="toclevel-2"><A href="#Send_pSB1C3_vectors.2C_containing_geneIII.2C_mdnA_and_geneIII.2FmdnA_fusion_gene.2C_for_sequencing"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Send pSB1C3 vectors, containing geneIII, mdnA and geneIII/mdnA fusion gene, for sequencing</SPAN></A></LI><LI class="toclevel-2"><A href="#Plasmid_preperation_and_test_digestion_of_pARW089_containing_only_geneIII_.28no_mdnA.29"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">Plasmid preperation and test digestion of pARW089 containing only geneIII (no mdnA)</SPAN></A></LI><LI class="toclevel-1"><A href="#84th_Labday_2011-09-03"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">84th Labday 2011-09-03</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR-clean_up_of_mdnA_with_N-terminal_myc_tag_.28mcyN.29_.28Jessica_2011-09-02.29"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">PCR-clean up of mdnA with N-terminal myc tag (mcyN) (Jessica 2011-09-02)</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_mycN.2C_mycC_and_pSB1C3"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Restriction enzyme digestion of mycN, mycC and pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_mycC_and_mycN.2C_respectively.2C_and_pSB1C3"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Ligation of mycC and mycN, respectively, and pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_XL1_with_mycC.2FmycN_in_pSB1C3"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Transformation of XL1 with mycC/mycN in pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Oligo-Fillin_for_mdnA-Library_2_.28repetition.29"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Oligo-Fillin for mdnA-Library 2 (repetition)</SPAN></A></LI><LI class="toclevel-2"><A href="#Gel_purification_of_Oligo-Fillin_for_mdnA-Library_2_.28repetition.29"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">Gel purification of Oligo-Fillin for mdnA-Library 2 (repetition)</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_Oligo-Fillin_for_mdnA-Library_2_.28repetition.29_and_pUP089"><SPAN class="tocnumber">3.7</SPAN><SPAN class="toctext">Restriction enzyme digestion of Oligo-Fillin for mdnA-Library 2 (repetition) and pUP089</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_Lib2_.28repetition.29_and_pUP089"><SPAN class="tocnumber">3.8</SPAN><SPAN class="toctext">Ligation of Lib2 (repetition) and pUP089</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_XL1_with_Lib2_in_pUP089"><SPAN class="tocnumber">3.9</SPAN><SPAN class="toctext">Transformation of XL1 with Lib2 in pUP089</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_XL1_with_K3_and_A3_expression_backbones"><SPAN class="tocnumber">3.10</SPAN><SPAN class="toctext">Transformation of XL1 with K3 and A3 expression backbones</SPAN></A></LI><LI class="toclevel-2"><A href="#Picking_clones_for_overnight_cultures_of_pSB1C3_.2B_DE_clone_2_and_clone_3"><SPAN class="tocnumber">3.11</SPAN><SPAN class="toctext">Picking clones for overnight cultures of pSB1C3 + DE clone 2 and clone 3</SPAN></A></LI><LI class="toclevel-2"><A href="#overnight_culture_of_picked_E._coli_clones_transformed_with_of_XL1blue-cells_with_pPDV089"><SPAN class="tocnumber">3.12</SPAN><SPAN class="toctext">overnight culture of picked E. coli clones transformed with of XL1blue-cells with pPDV089</SPAN></A></LI><LI class="toclevel-1"><A href="#85th_Labday_2011-09-04"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">85th Labday 2011-09-04</SPAN></A></LI><LI class="toclevel-2"><A href="#Production_of_phages_containing_pPDV089_in_XL1_blue_cells"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Production of phages containing pPDV089 in XL1 blue cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_cells_picked_from_plates_on_02.09.2011_and_inoculated_into_ON_precultures"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Miniprep of cells picked from plates on 02.09.2011 and inoculated into ON precultures</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_overnight_cultures_from_ligation_of_pSB1C3.2BmdnDE_clone_2_and_clone3"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Miniprep of overnight cultures from ligation of pSB1C3+mdnDE clone 2 and clone3</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_XL1_with_mdnBC_in_pSB1C3"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Transformation of XL1 with mdnBC in pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Control_experiment:_Transformation_of_XL1_with_pARW089_and_pARW071.2C_respectively"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">Control experiment: Transformation of XL1 with pARW089 and pARW071, respectively</SPAN></A></LI><LI class="toclevel-2"><A href="#Picking_clones_for_overnight_cultures_of_pSB1C3.2BmycC_and_pSB1C3.2BmycN"><SPAN class="tocnumber">4.6</SPAN><SPAN class="toctext">Picking clones for overnight cultures of pSB1C3+mycC and pSB1C3+mycN</SPAN></A></LI><LI class="toclevel-2"><A href="#Picking_clones_for_overnight_cultures_of_pARW089_and_pARW071"><SPAN class="tocnumber">4.7</SPAN><SPAN class="toctext">Picking clones for overnight cultures of pARW089 and pARW071</SPAN></A></LI><LI class="toclevel-1"><A href="#86th_Labday_2011-09-05"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">86th Labday 2011-09-05</SPAN></A></LI><LI class="toclevel-2"><A href="#Test_digest_of_isolated_plasmids_.284.9.2011.29_from_clones_that_were_positive_in_colony_PCR_.282.09.2011.29"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Test digest of isolated plasmids (4.9.2011) from clones that were positive in colony PCR (2.09.2011)</SPAN></A></LI><LI class="toclevel-2"><A href="#competent_cells_-_ER2738"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">competent cells - ER2738</SPAN></A></LI><LI class="toclevel-2"><A href="#check_resistance_of_competent_cells_-_E._coli_ER2738"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">check resistance of competent cells - E. coli ER2738</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_overnight_cultures_of_pSB1C3.2BmycC_and_pSB1C3.2BmycN"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Miniprep of overnight cultures of pSB1C3+mycC and pSB1C3+mycN</SPAN></A></LI><LI class="toclevel-2"><A href="#Cultures_of_pARW089_and_pARW071_for_HPLC-Analysis"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">Cultures of pARW089 and pARW071 for HPLC-Analysis</SPAN></A></LI><LI class="toclevel-2"><A href="#Oligo-Fillin_for_mdnA-Library_2_.28repetition.29_2"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">Oligo-Fillin for mdnA-Library 2 (repetition)</SPAN></A></LI><LI class="toclevel-2"><A href="#Control_restriction_enzyme_digestion_of_pSB1C3_mdnA_mycC_and_pSB1C3_mdnA_mycN"><SPAN class="tocnumber">5.7</SPAN><SPAN class="toctext">Control restriction enzyme digestion of pSB1C3_mdnA_mycC and pSB1C3_mdnA_mycN</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR:_mdnB.2C_mdnABCDE"><SPAN class="tocnumber">5.8</SPAN><SPAN class="toctext">PCR: mdnB, mdnABCDE</SPAN></A></LI><LI class="toclevel-2"><A href="#Different_overnight_cultures"><SPAN class="tocnumber">5.9</SPAN><SPAN class="toctext">Different overnight cultures</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_phages_containing_pPDV089"><SPAN class="tocnumber">5.10</SPAN><SPAN class="toctext">Purification of phages containing pPDV089</SPAN></A></LI><LI class="toclevel-2"><A href="#Glycerol_stocks_of_phages_containing_pPDV089"><SPAN class="tocnumber">5.11</SPAN><SPAN class="toctext">Glycerol stocks of phages containing pPDV089</SPAN></A></LI><LI class="toclevel-2"><A href="#Concentration_of_phages_containing_pPDV089_measured_by_nanodrop"><SPAN class="tocnumber">5.12</SPAN><SPAN class="toctext">Concentration of phages containing pPDV089 measured by nanodrop</SPAN></A></LI><LI class="toclevel-1"><A href="#87th_Labday_2011-09-06"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">87th Labday 2011-09-06</SPAN></A></LI><LI class="toclevel-2"><A href="#Glycerol_stocks_of_pSB1C3_mdnBC_and_pUP089"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Glycerol stocks of pSB1C3_mdnBC and pUP089</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_overnight_cultures_of_pUP089_.26_psB1C3_.2B_mdnBC"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Miniprep of overnight cultures of pUP089 &amp; psB1C3 + mdnBC</SPAN></A></LI><LI class="toclevel-2"><A href="#check_plates_-_resistance_of_competent_E._coli_ER2738_cells"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">check plates - resistance of competent E. coli ER2738 cells</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_mdnABCDE_w.2F_Long_PCR_Enzyme_mix_from_Fermentas"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">PCR mdnABCDE w/ Long PCR Enzyme mix from Fermentas</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_purification_of_mdnB"><SPAN class="tocnumber">6.5</SPAN><SPAN class="toctext">PCR purification of mdnB</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_purified_PCR_product_mdnB_and_pSB1C3"><SPAN class="tocnumber">6.6</SPAN><SPAN class="toctext">Digest of purified PCR product mdnB and pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR:_Agarose_Gel:_mdnB_digest_verification.3B_pSBIC3_B_digest_verification_and_PCR_mdnABCDE_with_Long_Enzyme_Mix_verification"><SPAN class="tocnumber">6.7</SPAN><SPAN class="toctext">PCR: Agarose Gel: mdnB digest verification; pSBIC3_B digest verification and PCR mdnABCDE with Long Enzyme Mix verification</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_purification_of_PCR_fragment_mdnABCDE_.28T7.2C_89.29_and_digested_mdnB"><SPAN class="tocnumber">6.8</SPAN><SPAN class="toctext">PCR purification of PCR fragment mdnABCDE (T7, 89) and digested mdnB</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR:_Agarose_Gel:_pSBIC3_A.2C_C.2C_D.2C_E"><SPAN class="tocnumber">6.9</SPAN><SPAN class="toctext">PCR: Agarose Gel: pSBIC3_A, C, D, E</SPAN></A></LI><LI class="toclevel-2"><A href="#Gel_Extraction_of_pSB1C3_E"><SPAN class="tocnumber">6.10</SPAN><SPAN class="toctext">Gel Extraction of pSB1C3_E</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_pSB1C3_E_and_mdnB"><SPAN class="tocnumber">6.11</SPAN><SPAN class="toctext">Ligation of pSB1C3_E and mdnB</SPAN></A></LI><LI class="toclevel-2"><A href="#prepare_samples_for_sequencing_.28created_pARWIII.29"><SPAN class="tocnumber">6.12</SPAN><SPAN class="toctext">prepare samples for sequencing (created pARWIII)</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_pSB1C3"><SPAN class="tocnumber">6.13</SPAN><SPAN class="toctext">Digestion of pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#digestion_of_vector_pARWIII"><SPAN class="tocnumber">6.14</SPAN><SPAN class="toctext">digestion of vector pARWIII</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_PCR_products_geneIII_and_mdnA_for_cloning_into_pSB1C3"><SPAN class="tocnumber">6.15</SPAN><SPAN class="toctext">Digestion of PCR products geneIII and mdnA for cloning into pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis_for_purification_of_digested_PCR_products_and_vectors"><SPAN class="tocnumber">6.16</SPAN><SPAN class="toctext">Agarose gel electrophoresis for purification of digested PCR products and vectors</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_geneIII_and_mdnA_into_pSB1C3"><SPAN class="tocnumber">6.17</SPAN><SPAN class="toctext">Ligation of geneIII and mdnA into pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Re-ligation_of_NsiI-digested_pARWIII"><SPAN class="tocnumber">6.18</SPAN><SPAN class="toctext">Re-ligation of NsiI-digested pARWIII</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_created_vectors_in_E._coli"><SPAN class="tocnumber">6.19</SPAN><SPAN class="toctext">Transformation of created vectors in E. coli</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_of_TorA-bla_with_clevage_site_of_14_3C_and_TEV_for_BioBricks"><SPAN class="tocnumber">6.20</SPAN><SPAN class="toctext">PCR of TorA-bla with clevage site of 14_3C and TEV for BioBricks</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR:_mdnABCDE"><SPAN class="tocnumber">6.21</SPAN><SPAN class="toctext">PCR: mdnABCDE</SPAN></A></LI><LI class="toclevel-2"><A href="#Cultures_of_pARW071_and_pARW089_for_HPLC"><SPAN class="tocnumber">6.22</SPAN><SPAN class="toctext">Cultures of pARW071 and pARW089 for HPLC</SPAN></A></LI><LI class="toclevel-2"><A href="#Overnight_cultures_of_expression_backbones"><SPAN class="tocnumber">6.23</SPAN><SPAN class="toctext">Overnight cultures of expression backbones</SPAN></A></LI><LI class="toclevel-1"><A href="#88th_Labday_2011-09-07"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">88th Labday 2011-09-07</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_and_Ligation_of_TorA-bla_with_cleavage_site_of_14_3C_and_TEV_for_BioBricks_and_TEV_.2F_14_3C_biobricks"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Digestion and Ligation of TorA-bla with cleavage site of 14_3C and TEV for BioBricks and TEV / 14_3C biobricks</SPAN></A></LI><LI class="toclevel-2"><A href="#Oligo-Fillin_for_mdnA-Library_.28repetition.292"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Oligo-Fillin for mdnA-Library (repetition)2</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparing_samples_mdnC.2C_mdnD.2C_mdnE_and_mdnABC_for_sequencing"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Preparing samples mdnC, mdnD, mdnE and mdnABC for sequencing</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_Fillin_mdnA-Lib2"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">Purification of Fillin mdnA-Lib2</SPAN></A></LI><LI class="toclevel-2"><A href="#digest_of_purified_Fillin_mdnA-Lib2_and_pUP089"><SPAN class="tocnumber">7.5</SPAN><SPAN class="toctext">digest of purified Fillin mdnA-Lib2 and pUP089</SPAN></A></LI><LI class="toclevel-2"><A href="#purification_of_digested_pSB1C3_and_pUP089"><SPAN class="tocnumber">7.6</SPAN><SPAN class="toctext">purification of digested pSB1C3 and pUP089</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_Fillin_mdnA-Lib2_and_pUP089"><SPAN class="tocnumber">7.7</SPAN><SPAN class="toctext">Ligation of Fillin mdnA-Lib2 and pUP089</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_mdnABCDE_in_pSB1C3"><SPAN class="tocnumber">7.8</SPAN><SPAN class="toctext">Ligation of mdnABCDE in pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_purified_PCR_product_mdnABCDE_and_pSB1C3"><SPAN class="tocnumber">7.9</SPAN><SPAN class="toctext">Digest of purified PCR product mdnABCDE and pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#test_digest_of_mdnBC"><SPAN class="tocnumber">7.10</SPAN><SPAN class="toctext">test digest of mdnBC</SPAN></A></LI><LI class="toclevel-2"><A href="#test_pSB1A3_Ara_YFP_and_pSB1A3_Lac_YFP_.28expression_backbones.29"><SPAN class="tocnumber">7.11</SPAN><SPAN class="toctext">test pSB1A3_Ara_YFP and pSB1A3_Lac_YFP (expression backbones)</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR:_mdnABCDE_2"><SPAN class="tocnumber">7.12</SPAN><SPAN class="toctext">PCR: mdnABCDE</SPAN></A></LI><LI class="toclevel-2"><A href="#Overnight_cultures_of_expression_backbones_and_pUP089"><SPAN class="tocnumber">7.13</SPAN><SPAN class="toctext">Overnight cultures of expression backbones and pUP089</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_test_digest_of_mdnBC"><SPAN class="tocnumber">7.14</SPAN><SPAN class="toctext">Repeated test digest of mdnBC</SPAN></A></LI><LI class="toclevel-2"><A href="#Production_of_phages_containing_pPDV089_in_ER2738_cells"><SPAN class="tocnumber">7.15</SPAN><SPAN class="toctext">Production of phages containing pPDV089 in ER2738 cells</SPAN></A></LI><LI class="toclevel-2"><A href="#SDS-_PAGE_for_Western_Blotting_and_coomassie_staining_of_purified_phages"><SPAN class="tocnumber">7.16</SPAN><SPAN class="toctext">SDS- PAGE for Western Blotting and coomassie staining of purified phages</SPAN></A></LI><LI class="toclevel-2"><A href="#Western_Blot"><SPAN class="tocnumber">7.17</SPAN><SPAN class="toctext">Western Blot</SPAN></A></LI><LI class="toclevel-2"><A href="#Coomassie_staining"><SPAN class="tocnumber">7.18</SPAN><SPAN class="toctext">Coomassie staining</SPAN></A></LI><LI class="toclevel-2"><A href="#overnight_culture_of_picked_E._coli_clones_transformed_with_pARWIII_.28pARW089_containing_geneIII.29_to_control_deletion_of_kanamycin_gene.2C_pSB1C3_containing_mdnA.2C_geneIII_or_mdnA.2FgeneIII"><SPAN class="tocnumber">7.19</SPAN><SPAN class="toctext">overnight culture of picked E. coli clones transformed with pARWIII (pARW089 containing geneIII) to control deletion of kanamycin gene, pSB1C3 containing mdnA, geneIII or mdnA/geneIII</SPAN></A></LI><LI class="toclevel-1"><A href="#89th_Labday_2011-09-08"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">89th Labday 2011-09-08</SPAN></A></LI><LI class="toclevel-2"><A href="#Oligo-Fillin_for_mdnA-library_2_.28repetition.29_and_for_Phage-display-mdnA-library_.28carrying_AgeI_restriction_site.29"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Oligo-Fillin for mdnA-library 2 (repetition) and for Phage-display-mdnA-library (carrying AgeI restriction site)</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_XL1_with_different_constructs"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Transformation of XL1 with different constructs</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_Fillin_mdnA-Library_2_.28carrying_either_AgeI_or_AatII_restriction_site.29_and_PCR_product_of_mdnABCDE_from_pARW071"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Purification of Fillin mdnA-Library_2 (carrying either AgeI or AatII restriction site) and PCR product of mdnABCDE from pARW071</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_mdnA_focused_library_2_carrying_AatII_restriction_site_and_pUP089"><SPAN class="tocnumber">8.4</SPAN><SPAN class="toctext">Restriction enzyme digestion of mdnA focused library 2 carrying AatII restriction site and pUP089</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_mdnABCDE71_and_pSB1C3"><SPAN class="tocnumber">8.5</SPAN><SPAN class="toctext">Restriction enzyme digestion of mdnABCDE71 and pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_mdnA_focused_library_2_carrying_AgeI_restriction_site_and_pARWIII"><SPAN class="tocnumber">8.6</SPAN><SPAN class="toctext">Restriction enzyme digestion of mdnA focused library 2 carrying AgeI restriction site and pARWIII</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_of_TorA-bla_with_clevage_site_of_14_3C_and_TEV_for_BioBricks_2"><SPAN class="tocnumber">8.7</SPAN><SPAN class="toctext">PCR of TorA-bla with clevage site of 14_3C and TEV for BioBricks</SPAN></A></LI><LI class="toclevel-2"><A href="#Test_digest_of_pSB1C3.2BmdnDE"><SPAN class="tocnumber">8.8</SPAN><SPAN class="toctext">Test digest of pSB1C3+mdnDE</SPAN></A></LI><LI class="toclevel-2"><A href="#Overnight_cultures_of_pSB1C3.2Bmyc"><SPAN class="tocnumber">8.9</SPAN><SPAN class="toctext">Overnight cultures of pSB1C3+myc</SPAN></A></LI><LI class="toclevel-2"><A href="#Sidedirected_mutagenesis_of_TEV_protease_and_14_3C_protease"><SPAN class="tocnumber">8.10</SPAN><SPAN class="toctext">Sidedirected mutagenesis of TEV protease and 14_3C protease</SPAN></A></LI><LI class="toclevel-2"><A href="#Analytical_gelelctrophoresis_and_PCR_clean_up_of_sidedirected_mutated_fragments_T1.2C_T2.2C_P1.2C_P2.2C_P3_and_P4_.5B.5B7.5D.5D"><SPAN class="tocnumber">8.11</SPAN><SPAN class="toctext">Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments T1, T2, P1, P2, P3 and P4 [[7]]</SPAN></A></LI><LI class="toclevel-2"><A href="#Assembly_PCR_of_mutated_TEV_protease_and_14_3C_protease_fragments"><SPAN class="tocnumber">8.12</SPAN><SPAN class="toctext">Assembly PCR of mutated TEV protease and 14_3C protease fragments</SPAN></A></LI><LI class="toclevel-2"><A href="#Analytical_gelelctrophoresis_and_PCR_clean_up_of_sidedirected_mutated_fragments_TEV_protease.2C_P5_and_P6_.5B.5B10.5D.5D"><SPAN class="tocnumber">8.13</SPAN><SPAN class="toctext">Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments TEV protease, P5 and P6 [[10]]</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparing_glycerol_stocks_of_pUP089"><SPAN class="tocnumber">8.14</SPAN><SPAN class="toctext">Preparing glycerol stocks of pUP089</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_mdnA-lib2_and_mdnA-Age"><SPAN class="tocnumber">8.15</SPAN><SPAN class="toctext">Ligation of mdnA-lib2 and mdnA-Age</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_phages_containing_pPDV089_2"><SPAN class="tocnumber">8.16</SPAN><SPAN class="toctext">Purification of phages containing pPDV089</SPAN></A></LI><LI class="toclevel-2"><A href="#Glycerol_stocks_of_phages_containing_pPDV089_2"><SPAN class="tocnumber">8.17</SPAN><SPAN class="toctext">Glycerol stocks of phages containing pPDV089</SPAN></A></LI><LI class="toclevel-2"><A href="#Concentration_of_phages_containing_pPDV089_measured_by_nanodrop_2"><SPAN class="tocnumber">8.18</SPAN><SPAN class="toctext">Concentration of phages containing pPDV089 measured by nanodrop</SPAN></A></LI><LI class="toclevel-2"><A href="#plasmid_pereperation_of_pARWIII_to_control_deletion_of_kanamycin_gene.2C_pSB1C3_containing_mdnA.2C_geneIII_or_mdnA.2FgeneIII"><SPAN class="tocnumber">8.19</SPAN><SPAN class="toctext">plasmid pereperation of pARWIII to control deletion of kanamycin gene, pSB1C3 containing mdnA, geneIII or mdnA/geneIII</SPAN></A></LI><LI class="toclevel-2"><A href="#Antibody_detection_with_blotted_membranes"><SPAN class="tocnumber">8.20</SPAN><SPAN class="toctext">Antibody detection with blotted membranes</SPAN></A></LI><LI class="toclevel-1"><A href="#90th_Labday_2011-09-09"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">90th Labday 2011-09-09</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation_of_samples_for_HPLC_analysis"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Preparation of samples for HPLC analysis</SPAN></A></LI><LI class="toclevel-2"><A href="#Counting_of_colonies_on_plates_.28pUP089.2Blib2.29"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Counting of colonies on plates (pUP089+lib2)</SPAN></A></LI><LI class="toclevel-2"><A href="#Glycerol_stocks_of_pSB1C3.2BmdnDE_K1.2C_K2_and_K5"><SPAN class="tocnumber">9.3</SPAN><SPAN class="toctext">Glycerol stocks of pSB1C3+mdnDE K1, K2 and K5</SPAN></A></LI><LI class="toclevel-2"><A href="#Sequencing_of_pSB1C3.2BmdnD.2FABC"><SPAN class="tocnumber">9.4</SPAN><SPAN class="toctext">Sequencing of pSB1C3+mdnD/ABC</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_XL1_with_different_constructs_2"><SPAN class="tocnumber">9.5</SPAN><SPAN class="toctext">Transformation of XL1 with different constructs</SPAN></A></LI><LI class="toclevel-2"><A href="#Survival-Test_of_one_14_3C_clone.28_SG9-P10.29_and_one_Tev-clone_.28SG13-T6.29"><SPAN class="tocnumber">9.6</SPAN><SPAN class="toctext">Survival-Test of one 14_3C clone( SG9-P10) and one Tev-clone (SG13-T6)</SPAN></A></LI><LI class="toclevel-2"><A href="#Assembly_PCR_of_mutated_14_3C_protease"><SPAN class="tocnumber">9.7</SPAN><SPAN class="toctext">Assembly PCR of mutated 14_3C protease</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_NgoMIV_TEV-Protease_iGEM_BamHI.2C_NgoMIV_14_3C-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_fragments_and_pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5_and_pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5_vector_2"><SPAN class="tocnumber">9.8</SPAN><SPAN class="toctext">Digestion of NgoMIV_TEV-Protease_iGEM_BamHI, NgoMIV_14_3C-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV fragments and pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5 and pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5 vector</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI_or_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_into_pJC354-NheI-143C-Xho_blaFL_GGH5_or_pJC354-NheI-TEV-Xho_blaFL_GGH5_vector_2"><SPAN class="tocnumber">9.9</SPAN><SPAN class="toctext">Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI or NgoMIV_iGEM_TEV-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV into pJC354-NheI-143C-Xho_blaFL_GGH5 or pJC354-NheI-TEV-Xho_blaFL_GGH5 vector</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_from_overnight_cultures_T_I.2FII.2C_14_3C_CS.2CPC.2C_TEV-CS-1.2C_P1"><SPAN class="tocnumber">9.10</SPAN><SPAN class="toctext">Miniprep from overnight cultures T I/II, 14_3C CS,PC, TEV-CS-1, P1</SPAN></A></LI><LI class="toclevel-2"><A href="#Sequencing_of_miniprep_.2809-09-2011.29_-_T_I.2FII.2C_14_3C_CS.2CPC.2C_TEV-CS-1.2C_P1"><SPAN class="tocnumber">9.11</SPAN><SPAN class="toctext">Sequencing of miniprep (09-09-2011) - T I/II, 14_3C CS,PC, TEV-CS-1, P1</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_TEV_and_14_3C"><SPAN class="tocnumber">9.12</SPAN><SPAN class="toctext">Transformation of TEV and 14_3C</SPAN></A></LI><LI class="toclevel-2"><A href="#Test_digest_of_pSB1C3.2BmycN.2FC"><SPAN class="tocnumber">9.13</SPAN><SPAN class="toctext">Test digest of pSB1C3+mycN/C</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_overnight_cultures_.2B_pSB1C3.2BmycN.2FmycC"><SPAN class="tocnumber">9.14</SPAN><SPAN class="toctext">Miniprep of overnight cultures + pSB1C3+mycN/mycC</SPAN></A></LI><LI class="toclevel-2"><A href="#Sequencing_of_pSB1C3.2BmdnDE.2FmycN.2FmycC"><SPAN class="tocnumber">9.15</SPAN><SPAN class="toctext">Sequencing of pSB1C3+mdnDE/mycN/mycC</SPAN></A></LI><LI class="toclevel-2"><A href="#Picking_clones_for_overnight_cultures_of_Lib_2_for_confirmation"><SPAN class="tocnumber">9.16</SPAN><SPAN class="toctext">Picking clones for overnight cultures of Lib 2 for confirmation</SPAN></A></LI><LI class="toclevel-2"><A href="#Prepare_liquid_culture_for_mdnA_foc2"><SPAN class="tocnumber">9.17</SPAN><SPAN class="toctext">Prepare liquid culture for mdnA foc2</SPAN></A></LI><LI class="toclevel-2"><A href="#Picking_clones_for_overnight_cultures_of_pSB1C3.2BmdnABCDE_with_and_without_T7-promotor"><SPAN class="tocnumber">9.18</SPAN><SPAN class="toctext">Picking clones for overnight cultures of pSB1C3+mdnABCDE with and without T7-promotor</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparing_overnight_cultures_of_expression_backbone-clones"><SPAN class="tocnumber">9.19</SPAN><SPAN class="toctext">Preparing overnight cultures of expression backbone-clones</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR:_mdnABCDE_3"><SPAN class="tocnumber">9.20</SPAN><SPAN class="toctext">PCR: mdnABCDE</SPAN></A></LI><LI class="toclevel-2"><A href="#test_digestion_to_control_if_pSB1C3_is_containing_mdnA.2C_geneIII_or_mdnA.2FgeneIII"><SPAN class="tocnumber">9.21</SPAN><SPAN class="toctext">test digestion to control if pSB1C3 is containing mdnA, geneIII or mdnA/geneIII</SPAN></A></LI><LI class="toclevel-2"><A href="#Send_clones_2-6_.28geneIII-mdnA_in_pSB.29_for_sequencing"><SPAN class="tocnumber">9.22</SPAN><SPAN class="toctext">Send clones 2-6 (geneIII-mdnA in pSB) for sequencing</SPAN></A></LI><LI class="toclevel-2"><A href="#test_digestion_to_control_if_kanamycin_resistance_gene_is_knocked_out_of_pARWIII"><SPAN class="tocnumber">9.23</SPAN><SPAN class="toctext">test digestion to control if kanamycin resistance gene is knocked out of pARWIII</SPAN></A></LI><LI class="toclevel-1"><A href="#91th_Labday_2011-09-10"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">91th Labday 2011-09-10</SPAN></A></LI><LI class="toclevel-2"><A href="#miniprep_of_Lib2_clones_for_confirmation"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">miniprep of Lib2 clones for confirmation</SPAN></A></LI><LI class="toclevel-2"><A href="#miniprep_of_mdnA_foc2"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">miniprep of mdnA foc2</SPAN></A></LI><LI class="toclevel-2"><A href="#miniprep_of_pSB1C3.2BmdnABCDE_with_and_without_T7_promotor"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">miniprep of pSB1C3+mdnABCDE with and without T7 promotor</SPAN></A></LI><LI class="toclevel-2"><A href="#Counting_of_colonies_on_plates_.28pUP089.2Blib2.29_2"><SPAN class="tocnumber">10.4</SPAN><SPAN class="toctext">Counting of colonies on plates (pUP089+lib2)</SPAN></A></LI><LI class="toclevel-2"><A href="#New_test_digest_of_pSB1C3.2BmycN.2FC"><SPAN class="tocnumber">10.5</SPAN><SPAN class="toctext">New test digest of pSB1C3+mycN/C</SPAN></A></LI><LI class="toclevel-2"><A href="#Test_Expression_Backbones:_pSB1K3_Ara.2FLac_CFP"><SPAN class="tocnumber">10.6</SPAN><SPAN class="toctext">Test Expression Backbones: pSB1K3_Ara/Lac_CFP</SPAN></A></LI><LI class="toclevel-2"><A href="#Glycerol_stocks:_Expression_Backbones"><SPAN class="tocnumber">10.7</SPAN><SPAN class="toctext">Glycerol stocks: Expression Backbones</SPAN></A></LI><LI class="toclevel-2"><A href="#Oligo-Fillin_for_mdnA-library_2_.28repetition.29_and_for_Phage-display-mdnA-library_.28carrying_AgeI_restriction_site.29_2"><SPAN class="tocnumber">10.8</SPAN><SPAN class="toctext">Oligo-Fillin for mdnA-library 2 (repetition) and for Phage-display-mdnA-library (carrying AgeI restriction site)</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_ParWIII_and_fill-in-reaction_of_Libary_2"><SPAN class="tocnumber">10.9</SPAN><SPAN class="toctext">Restriction enzyme digestion of ParWIII and fill-in-reaction of Libary 2</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_pSB1C3_for_ligation_with_mycN.2FC"><SPAN class="tocnumber">10.10</SPAN><SPAN class="toctext">Restriction enzyme digestion of pSB1C3 for ligation with mycN/C</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_pSB1C3_for_control"><SPAN class="tocnumber">10.11</SPAN><SPAN class="toctext">Restriction enzyme digestion of pSB1C3 for control</SPAN></A></LI><LI class="toclevel-2"><A href="#Picking_clones_for_overnight_cultures_of_pSB1C3.2BmdnABCDE"><SPAN class="tocnumber">10.12</SPAN><SPAN class="toctext">Picking clones for overnight cultures of pSB1C3+mdnABCDE</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparing_overnight_cultures_of_expression_backbone-clones_2"><SPAN class="tocnumber">10.13</SPAN><SPAN class="toctext">Preparing overnight cultures of expression backbone-clones</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_mdnABCDE_.2B_pSB1C3"><SPAN class="tocnumber">10.14</SPAN><SPAN class="toctext">Transformation of mdnABCDE + pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Colony_PCR_of_plated_TEV_and_14_3C_containing_colonies"><SPAN class="tocnumber">10.15</SPAN><SPAN class="toctext">Colony PCR of plated TEV and 14_3C containing colonies</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_TEV.2C_14_3C.2C_AraC_and_different_Vectors_.28pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL.2C_pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL_and_pSB1C3.29"><SPAN class="tocnumber">10.16</SPAN><SPAN class="toctext">Digest of TEV, 14_3C, AraC and different Vectors (pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL, pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL and pSB1C3)</SPAN></A></LI><LI class="toclevel-2"><A href="#Gelextraction_of_digested_fragements.2C_ligation_and_transfomration_of_ligation_products_into_E._coli_XL1_blue"><SPAN class="tocnumber">10.17</SPAN><SPAN class="toctext">Gelextraction of digested fragements, ligation and transfomration of ligation products into E. coli XL1 blue</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_digested_fragments_of_TEV_protease.2C_14_3C_protease_and_different_vectors"><SPAN class="tocnumber">10.18</SPAN><SPAN class="toctext">Ligation of digested fragments of TEV protease, 14_3C protease and different vectors</SPAN></A></LI><LI class="toclevel-2"><A href="#Transfromation_of_E._coli_XL1_blue_with_ligation_products"><SPAN class="tocnumber">10.19</SPAN><SPAN class="toctext">Transfromation of E. coli XL1 blue with ligation products</SPAN></A></LI><LI class="toclevel-2"><A href="#Plating_of_transformed_E._coli_XL1_blue"><SPAN class="tocnumber">10.20</SPAN><SPAN class="toctext">Plating of transformed E. coli XL1 blue</SPAN></A></LI></TD></TR></TBODY></TABLE><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="82th_Labday_2011-09-01">82th Labday 2011-09-01</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_pUP089_.28vector.29">Digest of pUP089 (vector) </SPAN></H3><P><B>Investigator:</B> Niels, Katharina</P><P><B>Aim:</B> generate vector for ligation and transformation to generate a libary
</P><P><B>Materials/Methods:</B></P><UL><LI> pub089 (A)
</LI></UL><UL><LI> 5 µl DNA (~4000ng)
</LI></UL><UL><LI> 0,5 µl Sfo I
</LI></UL><UL><LI> 0,5 µl Aat II
</LI></UL><UL><LI> 3 µl 10x Buffer 4 (Biolabs)
</LI></UL><UL><LI> 21 µl water
</LI></UL><UL><LI> total volume: 30µl
</LI></UL><P><B>Purification</B></P><UL><LI> by NucleoSpin Extract II Kit, protocol for PCR purification
</LI></UL><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> backseat driver asks: why do you use only 5 µl? what about gel purification?</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> dephosphorylation
</LI></UL><UL><LI> ligation
</LI></UL><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="miniprep_of_overnight_cultures_-_mdnDE.2C_mdnABC">miniprep of overnight cultures - mdnDE, mdnABC </SPAN></H3><P><B>Investigator:</B> Niels, Jessica</P><P><B>Aim:</B> Isolate DNA from over night cultures from 31.08.2011 (Katharina)
</P><P><B>Materials/Methods:</B></P><P><B>1.miniprep:</B></P><UL><LI> over night culture
</LI></UL><UL><LI> mdnABC
</LI></UL><UL><LI> clone 3
</LI></UL><UL><LI> clone 4
</LI></UL><UL><LI> clone 5
</LI></UL><UL><LI> clone 6
</LI></UL><UL><LI> clone 7
</LI></UL><UL><LI> mdnDE
</LI></UL><UL><LI> clone 1
</LI></UL><UL><LI> clone 2
</LI></UL><UL><LI> clone 3
</LI></UL><UL><LI> clone 4
</LI></UL><UL><LI> clone 5
</LI></UL><UL><LI>using Kit NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> elution in 50µl Elution buffer
</LI></UL><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> backseat driver: almost there, but a real plasmid name would be better.</TD></TR></TBODY></TABLE><P><B>2. Preparation of glycerol stocks:</B></P><UL><LI> adding 300 µl glycerol to 700 µl culture</LI></UL><P><B> Result: </B></P><P><B>Further tasks:</B></P><UL><LI> test-digest
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_miniprep_-_mdnDE.2C_mdnABC">Digest of miniprep - mdnDE, mdnABC </SPAN></H3><P><B>Investigator:</B> Niels, Jessica</P><P><B>Aim:</B> prove of isolated DNA from over night cultures from 2011-09-01 (Niels)
</P><P><B>Materials/Methods:</B></P><UL><LI> DNA (miniprep)
</LI></UL><UL><LI> SpeI
</LI></UL><UL><LI> EcoRI
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> BSA
</LI></UL><UL><LI> Water
</LI></UL><P>'<I>1.Digest:</I></P><UL><LI> DNA (µl) / Water (µl)
</LI></UL><UL><LI> mdnABC
</LI></UL><UL><LI> clone 3 -
</LI></UL><UL><LI> clone 4 -
</LI></UL><UL><LI> clone 5 -
</LI></UL><UL><LI> clone 6 -
</LI></UL><UL><LI> clone 7 -
</LI></UL><UL><LI> mdnDE
</LI></UL><UL><LI> clone 1 -
</LI></UL><UL><LI> clone 2 -
</LI></UL><UL><LI> clone 3 -
</LI></UL><UL><LI> clone 4 -
</LI></UL><UL><LI> clone 5 -
</LI></UL><UL><LI> 3µl Buffer 4 (Neblab)
</LI></UL><UL><LI> 0,3µl BSA
</LI></UL><UL><LI> 1 µl Spe I
</LI></UL><UL><LI> 1 µl EcoRI
</LI></UL><UL><LI>incubate 2h at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> analyze by gel
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Testdigest_of_miniprep_-_mdnDE.2C_mdnABC">Testdigest of miniprep - mdnDE, mdnABC </SPAN></H3><P><B>Investigator:</B> Niels, Katharina</P><P><B>Aim:</B> prove of plasmid
</P><P><B>Materials:</B></P><UL><LI> DNA (miniprep)
</LI></UL><UL><LI> SpeI
</LI></UL><UL><LI> EcoRI
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> BSA
</LI></UL><UL><LI> Water
</LI></UL><P><B>Digest:</B>:
</P><UL><LI>mdnABC
</LI></UL><UL><LI> 1µl DNA
</LI></UL><UL><LI> 2 µl Buffer 2 (Neblab)
</LI></UL><UL><LI> 0,5 µl HindIII
</LI></UL><UL><LI> 0,5 µl AvaI
</LI></UL><UL><LI> 16 µl Water
</LI></UL><UL><LI> incubate 2h at 37°C
</LI></UL><UL><LI>mdnDE
</LI></UL><UL><LI> 1µl DNA
</LI></UL><UL><LI> 2 µl 10x Buffer 4 (Neblab)
</LI></UL><UL><LI> 0,5µl HpaI
</LI></UL><UL><LI> 16,5 µl Water
</LI></UL><UL><LI> incubate 2h at 37°C
</LI></UL><P><B>Further tasks:</B></P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_pUP089_and_Lib-2">Ligation of pUP089 and Lib-2</SPAN></H3><P><B>Investigators:</B> Steffi, Niels, Katharina</P><P><B>Material</B></P><UL><LI> purified pUP089 (vector)
</LI></UL><UL><LI> purified Lib-2 (insert)
</LI></UL><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> backseat driver: do these poor pieces of DNA have no date or other referral?</TD></TR></TBODY></TABLE><P><B>Method</B></P><UL><LI> 1 µl 10x Buffer
</LI></UL><UL><LI> 1 µl T4 Ligase
</LI></UL><UL><LI> 6 µl vector
</LI></UL><UL><LI> 2 µl insert
</LI></UL><UL><LI>1h @ RT </LI></UL><P><B>Further tasks:</B></P><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_library2">Transformation of library2</SPAN></H3><P><B>Investigators:</B>Katharina, Steffi</P><P><B>Aim:</B> Transformation of Ligation </P><P><B>Materials:</B></P><UL><LI> competent <I>E. coli</I> cells (XL1-Blue, 2011-08-29)
</LI></UL><UL><LI> ligation products: Lib2 + ligation control (2011-09-01, Nie, Kat, Ste)
</LI></UL><P><B>Method:</B></P><UL><LI>addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI>incubation 30 min on ice,
</LI></UL><UL><LI>heat shock 45 sec at 42°C,
</LI></UL><UL><LI>incubation 3 min on ice,
</LI></UL><UL><LI>addition of 750 µl LB medium,
</LI></UL><UL><LI>incubation at 37 °C shaking for 60 min,
</LI></UL><UL><LI>plating on LB medium with appropriate antibiotic (Kan)
</LI></UL><UL><LI>storage over night at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><P><B>Output:</B></P><UL><LI> 2 plates w/ Kan: mdn-lib2 + control
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Picking_clones_for_overnight_cultures_of_pSB1C3_.2B_mdnABC_and_DE.2C_respectively">Picking clones for overnight cultures of pSB1C3 + mdnABC and DE, respectively</SPAN></H3><P><B>Investigators:</B>Steffi, Katharina</P><P><B>Aim:</B> check colonies for correct plasmid</P><P><B>Materials:</B></P><UL><LI> agar plates from 2011-08-31, (fridge):
</LI></UL><UL><LI> pSB1C3+mdnABC
</LI></UL><UL><LI> pSB1C3+mdnDE
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> chloramphenicol (25 mg/ml)
</LI></UL><P><B>Protocol:</B></P><UL><LI> 6 x: 5 ml LB medium + 5 µl chloramphenicol
</LI></UL><UL><LI> pick colony from plate (from each plate 3 colonies)
</LI></UL><UL><LI> transfer to LB medium
</LI></UL><UL><LI> incubate over night in 37°C shaker (200 rpm)
</LI></UL><P><B>Output:</B></P><UL><LI>over night cultures:
</LI></UL><UL><LI> pSB1C3+mdnABC
</LI></UL><UL><LI> clone 1
</LI></UL><UL><LI> clone 2
</LI></UL><UL><LI> clone 3
</LI></UL><UL><LI>pSB1C3+mdnDE
</LI></UL><UL><LI> clone 1
</LI></UL><UL><LI> clone 2
</LI></UL><UL><LI> clone 3
</LI></UL><P><B>Further tasks:</B></P><UL><LI> miniprep
</LI></UL><UL><LI> test digest
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_NgoMIV_TEV-Protease_iGEM_BamHI.2C_NgoMIV_14_3C-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_fragments_and_pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5_and_pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5_vector">Digestion of NgoMIV_TEV-Protease_iGEM_BamHI, NgoMIV 14_3C-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV fragments and pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5 and pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5 vector</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[1]</A>]</B></P><P><B>Investigators:</B>Paul, Sascha, Stefan, Sebastian</P><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR>&lt;td style=&quot;vertical-align: top; background-color: rgb(204, 204, 204);&quot;.&lt;/td&gt;</TR>&lt;/table&gt;<B>Materials:</B>

4 samples NgoMIV_TEV-Protease_iGEM_BamHI:

<UL><LI>TEV 3: 71,7 ng/µl
</LI></UL><UL><LI>TEV 7: 65,3 ng/µl
</LI></UL><UL><LI>TEV 10: 58,6 ng/µl
</LI></UL><UL><LI>TEV 16: 62,0 ng/µl
</LI></UL>

2 samples NgoMIV_14_3C-Protease_iGEM_BamHI

<UL><LI>P1: 91,5 ng/µl
</LI></UL><UL><LI>P2: 82,5 ng/µl
</LI></UL>

2 samples HindIII_iGEM_AraC_NgoMIV

<UL><LI>A1: 103,7 ng/µl
</LI></UL><UL><LI>A2: 127,4 ng/µl
</LI></UL>

pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (contains TEV cleavage site): 470 ng/µl

pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5 vector (contains 14_3C cleavage site): 270,2 ng/µl

<TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> backseat driver: nice list, but dates would be really cool.</TD></TR></TBODY></TABLE><P><B> Digestion protocol: </B></P><P>1: NgoMIV_TEV-Protease_iGEM_BamHI (4x, each sample 1x):
</P><UL><LI>10µl NgoMIV_iGEM_TEV-Protease_iGEM_BamHI
</LI></UL><UL><LI>1µl NgoMIV
</LI></UL><UL><LI>1µl BamHI-HF
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>32.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>2: NgoMIV_14_3C-Protease_iGEM_BamHI (2x, each sample 1x):
</P><UL><LI>10µl NgoMIV_14_3C-Protease_iGEM_BamHI
</LI></UL><UL><LI>1µl NgoMIV
</LI></UL><UL><LI>1µl BamHI-HF
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>32.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>3: HindIII_iGEM_AraC_NgoMIV (2x, each sample 1x):
</P><UL><LI>10µl HindIII_iGEM_AraC_NgoMIV
</LI></UL><UL><LI>1µl HindIII
</LI></UL><UL><LI>1µl NgoMIV
</LI></UL><UL><LI>5µl 10x buffer = NEB4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>32.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>4: pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5:
</P><UL><LI>3µl pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5
</LI></UL><UL><LI>1µl HindIII
</LI></UL><UL><LI>1µl BamHI-HF
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>39.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>5: pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5:
</P><UL><LI>5µl pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5
</LI></UL><UL><LI>1µl HindIII
</LI></UL><UL><LI>1µl BamHI-HF
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>37.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>--&gt;The reaction was allowed to proceed for 2h at 37°C!
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI_or_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_into_pJC354-NheI-143C-Xho_blaFL_GGH5_or_pJC354-NheI-TEV-Xho_blaFL_GGH5_vector"> Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI or NgoMIV_iGEM_TEV-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV into pJC354-NheI-143C-Xho_blaFL_GGH5 or pJC354-NheI-TEV-Xho_blaFL_GGH5 vector </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[2]</A>]</B></P><P><B>Investigators:</B> Paul, Sebastian, Sascha, Stefan</P><P><B>Aim</B>:</P><UL><LI><B>1. </B>Triple-ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI (573bp), HindIII_iGEM_AraC_NgoMIV (1273bp) and pJC354-NheI-143C-Xho_blaFL_GGH5 vector (~4700bp) to create pUP_SG2_TorA_CS-14_3C_bla_AraC-14_3C</LI></UL><UL><LI><B>2.</B>Triple-ligation of NgoMIV_iGEM_TEV_iGEM_BamHI (760bp), HindIII_iGEM_AraC_NgoMIV (1273bp) and pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (~4700bp)to create pUP_SG1_TorA_CS-TEV_bla_AraC-TEV</LI></UL><P>Calculation of volumes to be used with: <A href="http://www.gibthon.org/ligate.html" class="external text" rel="nofollow">ligation calculator</A> with 1:1 molar ratio
</P><P><B>Materials:</B></P><P>14_3C: 2 reaction batches (we have two digested 14_3C-fractions)
</P><P>1:
</P><UL><LI> 2,9 µL NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI fragment (573bp, 5,4ng/µl)
</LI></UL><UL><LI> 2 µL AraC fragment (12.6 ng/µl)
</LI></UL><UL><LI> 3,1 µL pJC354-NheI-143C-Xho_blaFL_GGH5 vector (30.8 ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P>2:
</P><UL><LI> 2,5 µL NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI (573bp, 6,6 ng/µl) fragment
</LI></UL><UL><LI> 2,2 µL AraC fragment
</LI></UL><UL><LI> 3,3 µL pJC354-NheI-143C-Xho_blaFL_GGH5 vector (30.8 ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><UL><LI> 1 controls: As 2 but with water instead of fragment
</LI></UL><P>TEV: 4 reaction batches (we have three digested NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment fractions)
</P><P>1:
</P><UL><LI> 1.8 µL NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment (760bp, 5,8 ng/µl)
</LI></UL><UL><LI> 1.4 µL AraC fragment (12,6 ng/µl)
</LI></UL><UL><LI> 4.9 µL pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (12,9 ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P>2:
</P><UL><LI> 2.4 µL NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment (760bp, 3,7 ng/µl)
</LI></UL><UL><LI> 1.2 µL AraC fragment (12,6 ng/µl)
</LI></UL><UL><LI> 4.3 µL pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (12,9 ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P>3:
</P><UL><LI> 3.2 µL NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment (760bp, 2.5 ng/µl)
</LI></UL><UL><LI> 1.1 µL AraC fragment (12,6 ng/µl)
</LI></UL><UL><LI> 3.8 µL pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (12,9 ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P>4:
</P><UL><LI> 2.4 µL NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment (760bp, 3,9 ng/µl)
</LI></UL><UL><LI> 1.2 µL AraC fragment (12,6 ng/µl)
</LI></UL><UL><LI> 4.4 µL pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (12,9 ng/µl)
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> backseat driver: are you doing the same thing four times? why do you fractionate your DNA?</TD></TR></TBODY></TABLE><UL><LI> 1 control: As 4 but with water instead of fragment
</LI></UL><P><B> Used method: </B></P><P>ligation at room temperatur for 1h
</P><P><B> Further task: Transformation of XL1blue cells with ligation products </B></P><P><B> Edit: </B></P><P>Competent cells were transformed with the complete ligation batches and plated on Cm containing plates!
</P><P>Results:
</P><P>1: 14_3C containing colnes, 10 colonies picked
</P><P>2: TEV containing clones, 10 colonies picked
</P><P>Note: &quot;Rest&quot; means, that after plating 100µl on a plate remaining cells in a tube were centrifugated, resuspended and plated again on a new plate.
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="plasmid_pereperation_and_test_digestion_pPDV089_to_control_deletion_of_kanamycin_gene.2C_pARW089_containing_only_geneIII_.28no_mdnA.29.2C_pSB1C3_containing_mdnA.2C_geneIII_or_mdnA.2FgeneIII">plasmid pereperation and test digestion pPDV089 to control deletion of kanamycin gene, pARW089 containing only geneIII (no mdnA), pSB1C3 containing mdnA, geneIII or mdnA/geneIII </SPAN></H3><P><B>Investigators:</B> Sandrina, Laura</P><P><B>Aim:</B> control deletion of kanamycin gene in created pPDV089, control ligation of geneIII into PARW089, control ligation of mdnA, geneIII and mdnA/geneIII into pSB1C3</P><P><B>Method/Materials:</B></P><P>plasmid preperation:
</P><P>protocol 5.1 of the NucleoSpin Plasmid Kit
</P><P>test digestion:
</P><UL><LI> 5 clones from pPDV089_2S14 (ampicillin):
</LI></UL><UL><LI>4 µl (132-256 ng/µl) vector DNA
</LI></UL><UL><LI>0.5 µl NsiI
</LI></UL><UL><LI>0.5 µl SpHI
</LI></UL><UL><LI>2 µl buffer 2
</LI></UL><UL><LI>13 µl water
</LI></UL><P>incubate for 1 h at 37°C
</P><UL><LI> 5 clones from pARW089 with geneIII (kanamycin):
</LI></UL><P>test digestion could not be done, because cells did not grow over night
</P><UL><LI> 5 clones from pSB1C3 with mdnA (chloramphenicol), 5 clones from pSB1C3 with geneIII (chloramphenicol), 5 clones from pSB1C3 with mdnA/geneIII (chloramphenicol):
</LI></UL><UL><LI>4 µl vector DNA (58- 120 ng/µl)
</LI></UL><UL><LI>2 µl buffer 3
</LI></UL><UL><LI>0,2 µl BSA
</LI></UL><UL><LI>0.5 µl XbaI
</LI></UL><UL><LI>0.5 µl PstI
</LI></UL><P>12.8 µl water
</P><P>incubate for 1 h at 37°C </P><P><B>Results:</B></P><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected insert size in bp</B></TD></TR><TR><TD> M </TD><TD> marker, DNA ladder mix Fermentas </TD></TR><TR><TD> 1 </TD><TD> geneIII in pSB1C3, clone 1 </TD><TD> 10 </TD><TD> ca. 500
</TD></TR><TR><TD> 2 </TD><TD> geneIII in pSB1C3, clone 2 </TD><TD> 10 </TD><TD> ca. 500
</TD></TR><TR><TD> 3 </TD><TD> geneIII in pSB1C3, clone 3 </TD><TD> 10 </TD><TD> ca. 500
</TD></TR><TR><TD> 4 </TD><TD> geneIII in pSB1C3, clone 4 </TD><TD> 10 </TD><TD> ca. 500
</TD></TR><TR><TD> 5 </TD><TD> geneIII in pSB1C3, clone 5 </TD><TD> 10 </TD><TD> ca. 500
</TD></TR><TR><TD> 6 </TD><TD> free </TD></TR><TR><TD> 7 </TD><TD> mdnA in pSB1C3, clone 1 </TD><TD> 12</TD><TD> ca. 180
</TD></TR><TR><TD> 8 </TD><TD> mdnA in pSB1C3, clone 2 </TD><TD> 12</TD><TD> ca. 180
</TD></TR><TR><TD> 9 </TD><TD> mdnA in pSB1C3, clone 3 </TD><TD> 12</TD><TD> ca. 180
</TD></TR><TR><TD> 10 </TD><TD> mdnA in pSB1C3, clone 4 </TD><TD> 12</TD><TD> ca. 180
</TD></TR><TR><TD> 11 </TD><TD> mdnA in pSB1C3, clone 5 </TD><TD> 12</TD><TD> ca. 180
</TD></TR><TR><TD> 12 </TD><TD> pPDV089, clone 1 </TD><TD> 12</TD><TD> ca. 1400
</TD></TR><TR><TD> 13 </TD><TD> pPDV089, clone 2 </TD><TD> 12</TD><TD> ca. 1400
</TD></TR><TR><TD> 14 </TD><TD> pPDV089, clone 3 </TD><TD> 12</TD><TD> ca. 1400
</TD></TR><TR><TD> 15 </TD><TD> pPDV089, clone 4 </TD><TD> 12</TD><TD> ca. 1400
</TD></TR><TR><TD> 16 </TD><TD> pPDV089, clone 5 </TD><TD> 12</TD><TD> ca. 1400
</TD></TR><TR><TD> 17 </TD><TD> free </TD></TR><TR><TD> 18 </TD><TD> mdnA-geneIII fusion gene in pSB1C3, clone 1 </TD><TD> 12</TD><TD> ca. 700
</TD></TR><TR><TD> 19 </TD><TD> mdnA-geneIII fusion gene in pSB1C3, clone 2 </TD><TD> 12</TD><TD> ca. 700
</TD></TR><TR><TD> 20 </TD><TD>mdnA-geneIII fusion gene in pSB1C3, clone 3 </TD><TD> 12</TD><TD> ca. 700
</TD></TR><TR><TD> 21 </TD><TD> mdnA-geneIII fusion gene in pSB1C3, clone 4 </TD><TD> 12</TD><TD> ca. 700
</TD></TR><TR><TD> 22 </TD><TD> mdnA-geneIII fusion gene in pSB1C3, clone 5 </TD><TD> 12</TD><TD> ca. 700
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> sequncing of geneIII in pSB1C3, clone 3, mdnA-geneIII fusion gene in pSB1C3, clone 4 and mdnA in pSB1C3, clone 4
</LI></UL><UL><LI> phage display with pPDV089
</LI></UL><UL><LI> repeat ligation of geneIII in pARW089
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="overnight_culture_of_picked_E._coli_clones_transformed_with_pARW089_containing_only_geneIII_.28no_mdnA.29">overnight culture of picked <I>E. coli</I> clones transformed with pARW089 containing only geneIII (no mdnA)</SPAN></H3><P><B>Investigators:</B> Sandrina, Laura</P><P><B>Aim:</B> control ligation of geneIII into PARW089</P><P><B>Method/Materials:</B></P><UL><LI> 4 clones from pARW089 with geneIII (kanamycin)
</LI></UL><UL><LI> 5 ml LB medium per clone
</LI></UL><UL><LI> storage over night at 37°C and 750 rpm</LI></UL><P><B>Further tasks:</B></P><UL><LI> test digestion</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Production_of_phages_containing_pPDV089">Production of phages containing pPDV089</SPAN></H3><P><B>Investigators:</B> Sandrina, Laura</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI> first: amplification of cells containing pPDV089
</LI></UL><UL><LI> 50 ml DYT medium will be inoculated with the cells, so that OD600 = 0.1
</LI></UL><UL><LI> antibiotics tetracyclin and ampicillin should be geiven to the medium
</LI></UL><UL><LI> cells should be incubated 37 °C till OD600 = 0.3-0.5 is reached
</LI></UL><UL><LI>upcomming problem: cells died
</LI></UL><P><B>Further tasks:</B></P><UL><LI> repeat this procedure</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="83th_Labday_2011-09-02">83th Labday 2011-09-02</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_from_ligation_of_pSB1C3_.2B_mdnABC_and_mdnDE.2C_respectively">Miniprep of overnight cultures from ligation of pSB1C3 + mdnABC and mdnDE, respectively</SPAN></H3><P><B>Investigators:</B> Vanessa, Katharina</P><P><B>Time:</B> 2011-09-02</P><P><B> Aim: </B> DNA for sequencing and confirmation of insert </P><P><B>Materials:</B></P><UL><LI> 6 overnight cultures
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> mdnABC clone 1 </TD><TD> 745.7
</TD></TR><TR><TD> mdnABC clone 2</TD><TD> 1121.8
</TD></TR><TR><TD> mdnABC clone 3</TD><TD> 974.6
</TD></TR><TR><TD> mdnDE clone 1</TD><TD> 873.2
</TD></TR><TR><TD> mdnDE clone 2</TD><TD> 665.5
</TD></TR><TR><TD> mdnDE clone 3</TD><TD> 665.7
</TD></TR></TBODY></TABLE><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="test_digest_of_pSB1C3_.2B_mdnABC_and_DE.2C_respectively">test digest of pSB1C3 + mdnABC and DE, respectively </SPAN></H3><P><B>Time:</B> 2011-09-02</P><P><B>Investigators:</B> Steffi, Katharina</P><P><B> Aim: </B> prove of Insert (mdnABC or mdnDE)</P><P><B>Materials:</B></P><P><B> Digestion protocol for pSB1C3+mdnABC</B></P><UL><LI> 1 µl DNA
</LI></UL><UL><LI> 0.3 µl BSA
</LI></UL><UL><LI> 1 µl HincII
</LI></UL><UL><LI> 3 µl 10x buffer = NEB 3
</LI></UL><UL><LI> 24.7 µl pure water
</LI></UL><UL><LI> 37°C for 1h
</LI></UL><P><B> Digestion protocol pSB1C3+mdnDE </B></P><UL><LI> 1 µl DNA
</LI></UL><UL><LI> 1 µl HpaI
</LI></UL><UL><LI> 3 µl 10x buffer = NEB 4
</LI></UL><UL><LI> 25 µl pure water
</LI></UL><UL><LI> 37°C for 1h
</LI></UL><P><B> Production of one 1 % </B></P><UL><LI> 1 % gel: 0.5 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to each gel
</LI></UL><P><B>Loading gels and running</B></P><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> marker </TD></TR><TR><TD> 1 </TD><TD> mndABC clone 1 </TD><TD> 12 </TD><TD> 486, 1482, 2884
</TD></TR><TR><TD> 2 </TD><TD> mdnABC clone 2 </TD><TD> 12 </TD><TD> 486, 1482, 2884
</TD></TR><TR><TD> 3 </TD><TD> mdnABC clone 3 </TD><TD> 12 </TD><TD> 486, 1482, 2884
</TD></TR><TR><TD> 4 </TD><TD> mdnDE clone 1 </TD><TD> 12 </TD><TD> 1329, 3825
</TD></TR><TR><TD> 5 </TD><TD> mdnDE clone 2 </TD><TD> 12 </TD><TD> 1329, 3825
</TD></TR><TR><TD> 6 </TD><TD> mdnDE clone 3 </TD><TD> 12 </TD><TD> 1329, 3825
</TD></TR></TBODY></TABLE><P><B> Results:</B></P><P><B> Conclusions:</B></P><UL><LI> test digest seems to be correct for:
</LI></UL><UL><LI> lane 2, 5 and 6
</LI></UL><P><B> Further task:</B></P><UL><LI> send positive plasmids for sequencing
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Colony_PCR_of_plated_cells_from_01.09.2011_.28see_entry_from_01.09.2011_for_picked_colonies.29"> Colony PCR of plated cells from 01.09.2011 (see entry from 01.09.2011 for picked colonies) </SPAN></H3><P><B>Investigators:</B> Sascha, Paul, Sebastian
</P><P><B>Aim:</B></P><P>Proove whether triple ligation of TEV and 14_3C into backbone was succesful.
</P><P><B>Used Primer:</B></P><P>Tev:
</P><P>f_AraC_HindIII_iGEM, r_TEV_iGEM_BamHI
</P><P>14_3C:
</P><P>f_AraC_HindIII_iGEM, r_14_3C_iGem:BamHI
</P><P><B> Method: </B></P><UL><LI>2,5µl of each primer (10µM) = 5µl
</LI></UL><UL><LI>5µl 10y polymerase buffer
</LI></UL><UL><LI>2µl 25mM MgCl2
</LI></UL><UL><LI>1µl dNTP mix
</LI></UL><UL><LI>0,5µl Taq-Polymerase (GenAxxon)
</LI></UL><UL><LI>36,5 µl H2O
</LI></UL><P>=50µl
</P><P>Colonies were picked with a 20µl tip, dipped into PCR batch, and then plunged into 1ml LB to grew cells from picked colony (from these 1ml batches overnight cultures will be inoculated in case of positive clones).
</P><UL><LI> 10 colonies were picked for TEV and 10 colonies for 14_3C protease (see entry from 01.09.2011) = 20 PCR batches.
</LI></UL><P><B>PCR Program:</B></P><P>Initial denat = 3min 94°C
</P><P>25x
</P><P>denat: 2min 10sec 94°C
</P><P>anneal: 2min 10sec 70°C
</P><P>extend: 2min 10sec 72°C
</P><P>final extend: 10min 72°C
</P><UL><LI> The PCR products were resolved on a 1% analytical agarose gel
</LI></UL><P>Expected fragments:
</P><P>Tev+AraC ~ 2033 bp
</P><P>14_3C+AraC ~ 1846 bp
</P><P><B>Results: </B></P><P><A href="/File:UP_AG_ColonyPCR_P_T_20110902.jpg" title="File:UP AG ColonyPCR P T 20110902.jpg">400px</A></P><UL><LI> Based on the results from the agarose gel overnight precultures from the 1 ml batches were prepared.
</LI></UL><P>Following clones were used:
</P><P>Tev:
</P><P>T2, T4, T5, T6, T7, T8, T9, T10
</P><P>14_3C:
</P><P>P1, P3, P4, P6; P8, P9, P10
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="over_night_cultures_of_pSB1C3.2BmdnABC_clone_2_and_P1_for_screening_team">over night cultures of pSB1C3+mdnABC clone 2 and P1 for screening team</SPAN></H3><P><B>time: </B> 2011-9-2, 18:00 </P><P><B>Investigators:</B> Nadine, Jessica </P><P><B>Aim:</B> preparation of gycerol stock from pSB1C3+mdnABC transformed cells (cultures were placed instable in the incubator)</P><P><B>Materials:</B></P><UL><LI> over night culture from Katharina (2011-9-1)
</LI></UL><UL><LI> pSB1C3+mdnABC clone 2
</LI></UL><UL><LI> over night culture from Paul (2011-9-2):
</LI></UL><UL><LI> P1
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> chloramphenicol (25 mg/ml)
</LI></UL><P><B>Protocol:</B></P><UL><LI> 2 x: 10 ml LB medium + 10 µl chloramphenicol
</LI></UL><UL><LI> use left cell suspension for inoculation
</LI></UL><UL><LI> transfer to LB medium
</LI></UL><UL><LI> incubate over night in 37°C shaker (200 rpm)
</LI></UL><P><B>Output:</B></P><UL><LI>over night cultures:
</LI></UL><UL><LI> pSB1C3+mdnABC
</LI></UL><UL><LI> clone 3
</LI></UL><UL><LI> P1 for screening team
</LI></UL><P><B>Further tasks:</B></P><UL><LI> mdnABC: glycerol stocks
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="plate_transformation_of_pUP089A_and_B_.28XL1-blue.29">plate transformation of pUP089A and B (XL1-blue)</SPAN></H3><P><B>time: </B> 2011-9-2, 17:00 </P><P><B>Investigators:</B> Nadine, Jessica </P><P><B>Aim:</B> get new pUP089 vector that is not contaminated, check new plasmid for antibiotic resistance</P><P><B>Materials:</B></P><UL><LI> Trafo from Jessica (2011-9-2)
</LI></UL><UL><LI> pUP089 A
</LI></UL><UL><LI> pUP089 B
</LI></UL><UL><LI> Agar plates w/:
</LI></UL><UL><LI> 100 µg/ml Cm (2x)
</LI></UL><UL><LI> 100 µg/ml Amp (1x)
</LI></UL><UL><LI> 100 µg/ml Kan (2x)
</LI></UL><P><B>Protocol:</B></P><UL><LI> centrifuge incubated cells
</LI></UL><UL><LI> resuspend in 16 µl LB
</LI></UL><UL><LI> plate 50 µl
</LI></UL><UL><LI> incubate over night in 37°C
</LI></UL><P><B>Output:</B></P><UL><LI>Agar plates:
</LI></UL><UL><LI> Cm pUP089 A
</LI></UL><UL><LI> Cm pUP089 B
</LI></UL><UL><LI> Kan pUP089 A
</LI></UL><UL><LI> Kan pUP089 B
</LI></UL><UL><LI> Amp pUP089 A
</LI></UL><P><B>Further tasks:</B></P><UL><LI> check plates
</LI></UL><UL><LI> over night cultures
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR_of_mdnA_with_N-terminal_myc_tag_.28mcyN.29">PCR of mdnA with N-terminal myc tag (mcyN) </SPAN></H3><P><B>Investigators:</B> Jessica </P><P><B>Time:</B> 2011-09-02, 14:00</P><P><B>Material:</B></P><UL><LI> pARW089 (~8 ng/µl)
</LI></UL><UL><LI> dNTPs
</LI></UL><UL><LI> primer 23, 24
</LI></UL><UL><LI> HF Phusion Buffer 5x
</LI></UL><UL><LI> Phusion Polymerase
</LI></UL><P><B>Method:</B></P><UL><LI> 1 µl pARW089
</LI></UL><UL><LI> 1 µl dNTPs (10 mM each)
</LI></UL><UL><LI> 2.5 µl forward primer (10µM)
</LI></UL><UL><LI> 2.5 µl reverse primer (10µM)
</LI></UL><UL><LI> 10 µl HF Phusion Buffer 5x
</LI></UL><UL><LI> 0.5 µl Phusion Polymerase
</LI></UL><UL><LI> 32.5 µl water
</LI></UL><UL><LI>total 50 µl
</LI></UL><UL><LI> edited programm IGBIO2
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>Step</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>Hot Start</TD><TD> 98°C </TD><TD> Hold </TD></TR><TR><TD> Initial denaturation</TD><TD> 98°C </TD><TD> 30 sec</TD></TR><TR><TD> Denaturation</TD><TD> 98°C</TD><TD> 10 s </TD><TD rowspan="3"> 10x
</TD></TR><TR><TD> Annealing</TD><TD> 58°C</TD><TD> 20 s
</TD></TR><TR><TD>Extension</TD><TD> 72°C</TD><TD> 20 s
</TD></TR><TR><TD> Denaturation</TD><TD> 98°C</TD><TD> 10 s</TD><TD rowspan="3"> 20x
</TD></TR><TR><TD> Annealing</TD><TD> 72°C</TD><TD> 20 s
</TD></TR><TR><TD>Extension</TD><TD> 72°C</TD><TD> 20 s
</TD></TR><TR><TD> Final extension </TD><TD> 72°C </TD><TD> 10 min </TD></TR><TR><TD> 4°C </TD><TD> Hold</TD></TR></TBODY></TABLE><P><B>Result:</B></P><UL><LI> PCR product mycN (stored in fridge, violett reck)
</LI></UL><P><B>NOTE:</B> PCR mycC done in july is stored at -20°C in blue PCR box (gel purified)
</P><P><B>Further Tasks:</B></P><UL><LI> gel electrophoresis
</LI></UL><UL><LI> PCR clean up
</LI></UL><UL><LI> restriction enzyme digestion
</LI></UL><UL><LI> ligation
</LI></UL><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_XL1blue-cells_with_pPDV089">Transformation of XL1blue-cells with pPDV089</SPAN></H3><P><B>Investigators:</B> Sandrina </P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI> addition of 0,3 µl ligated vector pPDV089 from clone 2S14 (after plasmid preperation) to competent XL1-blue cells</LI></UL><UL><LI> incubation 25 min on ice,</LI></UL><UL><LI> heat shock 45 sec at 42°C,</LI></UL><UL><LI> incubation 2 min on ice,</LI></UL><UL><LI> addition of 750 µl LB medium,</LI></UL><UL><LI> incubation 60 min at 37 °C and 750 rpm</LI></UL><UL><LI> plating on agar plates containing 100 µg/ml tetracyclin and 100 µg/ml ampicillin</LI></UL><UL><LI> storage over night at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> over night culture, amplification of cells for phage display, test digestion</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Send_pSB1C3_vectors.2C_containing_geneIII.2C_mdnA_and_geneIII.2FmdnA_fusion_gene.2C_for_sequencing">Send pSB1C3 vectors, containing geneIII, mdnA and geneIII/mdnA fusion gene, for sequencing</SPAN></H3><P><B>Investigators:</B> Sandrina </P><P><B>Aim:</B> control if mdnA, geneIII and mdnA/geneIII fusion gene were fully ligated in pSB1C3 to generate biobricks</P><P><B>Method/Materials:</B></P><UL><LI>sending clones geneIII in pSB1C3, clone 3, mdnA-geneIII fusion gene in pSB1C3, clone 4 and mdnA in pSB1C3, clone 4 to GATC
</LI></UL><UL><LI> 20 µl, 50-100 ng/µl
</LI></UL><UL><LI>primer:
</LI></UL><P><B>Further tasks:</B></P><UL><LI> sequence alignment</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Plasmid_preperation_and_test_digestion_of_pARW089_containing_only_geneIII_.28no_mdnA.29">Plasmid preperation and test digestion of pARW089 containing only geneIII (no mdnA)</SPAN></H3><P><B>Investigators:</B> Sandrina </P><P><B>Aim:</B> control if ligation of geneIII in pARW089 worked</P><P><B>Method/Materials:</B></P><P>plasmid preperation:
</P><UL><LI>protocol 5.1 of the NucleoSpin Plasmid Kit
</LI></UL><UL><LI>test digestion:
</LI></UL><UL><LI>4 clones from pARW089 with geneIII:
</LI></UL><UL><LI>4 µl (132-256 ng/µl) vector DNA
</LI></UL><UL><LI>0.5 µl SfoI
</LI></UL><UL><LI>0.5 µl AatII
</LI></UL><UL><LI>2 µl buffer 4
</LI></UL><UL><LI>13 µl water
</LI></UL><UL><LI>incubate for 1 h at 37°C
</LI></UL><UL><LI>glycerol stocks of clone 1, 2 and 4 (stored at -80°C)
</LI></UL><P><B>Results:</B></P><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> marker, DNA ladder mix Fermentas </TD></TR><TR><TD> 1 </TD><TD> geneIII in pARW089, clone 1 </TD><TD> 10 </TD><TD> ca. 10000 and 550
</TD></TR><TR><TD> 2 </TD><TD> geneIII in pARW089, clone 2 </TD><TD> 10 </TD><TD> ca. 10000 and 550
</TD></TR><TR><TD> 3 </TD><TD> geneIII in pARW089, clone 3 </TD><TD> 10 </TD><TD> ca. 10000 and 550
</TD></TR><TR><TD> 4 </TD><TD> geneIII in pARW089, clone 4 </TD><TD> 10 </TD><TD> ca. 10000 and 550
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> sequence vector</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="84th_Labday_2011-09-03">84th Labday 2011-09-03</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR-clean_up_of_mdnA_with_N-terminal_myc_tag_.28mcyN.29_.28Jessica_2011-09-02.29">PCR-clean up of mdnA with N-terminal myc tag (mcyN) (Jessica 2011-09-02) </SPAN></H3><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Time:</B> 2011-09-03, 9:00</P><P><B>Material:</B></P><UL><LI> PCR product of mycN
</LI></UL><UL><LI> NucleoSpin Extract II Kit from Macherey-Nagel
</LI></UL><UL><LI> elution in 50 µl
</LI></UL><P><B>Result:</B></P><UL><LI> concentration of product:
</LI></UL><UL><LI> 40.0 ng/µl
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_mycN.2C_mycC_and_pSB1C3">Restriction enzyme digestion of mycN, mycC and pSB1C3</SPAN></H3><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Time:</B> 2011-09-03, 10:00</P><P><B>Material:</B></P><UL><LI> purified PCR products
</LI></UL><UL><LI> mycC (4.8 ng/µl (2011-07-05 Jessica))
</LI></UL><UL><LI> mycN (40.0 ng/µl)
</LI></UL><UL><LI> NEB Buffer 4
</LI></UL><UL><LI> XbaI
</LI></UL><UL><LI> PstI
</LI></UL><UL><LI> BSA
</LI></UL><UL><LI> digestion of mycN and mycC
</LI></UL><UL><LI> 30 µl DNA
</LI></UL><UL><LI> 5 µl Buffer 4
</LI></UL><UL><LI> 1.5 µl XbaI
</LI></UL><UL><LI> 2.0 µl PstI
</LI></UL><UL><LI> 0.5 µl BSA
</LI></UL><UL><LI> 12 µl water
</LI></UL><UL><LI> digestion of pSB1C3
</LI></UL><UL><LI> 6 µl DNA (235.0 ng/µl (2011-06-22))
</LI></UL><UL><LI> 3 µl Buffer 4
</LI></UL><UL><LI> 1.5 µl XbaI
</LI></UL><UL><LI> 2.0 µl PstI
</LI></UL><UL><LI> 0.5 µl BSA
</LI></UL><UL><LI> 11.5 µl water
</LI></UL><UL><LI> concentration after digestion
</LI></UL><UL><LI> mycC: 4.2 ng/µl
</LI></UL><UL><LI> mycN: 15.2 ng/µl
</LI></UL><UL><LI> pSB1C3: 16.4 ng/µl
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_mycC_and_mycN.2C_respectively.2C_and_pSB1C3">Ligation of mycC and mycN, respectively, and pSB1C3</SPAN></H3><P><B>Time:</B> 2011-09-03, 17:30</P><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Materials</B></P><UL><LI> T4 DNA Ligase Buffer (Fermentas)
</LI></UL><UL><LI> T4 DNA Ligase
</LI></UL><UL><LI> purified samples of Lib2 and pUP089
</LI></UL><P><B>Method mycC</B></P><UL><LI> 1µl T4 DNA Ligase Buffer
</LI></UL><UL><LI> 1µl T4 DNA Ligase
</LI></UL><UL><LI> 5µl pSB1C3
</LI></UL><UL><LI> 3µl purified mycC
</LI></UL><UL><LI> also preparing ligation control (3µl water instead of mycC)
</LI></UL><UL><LI> incubation for 1h at roomtemperature
</LI></UL><P><B>Method mycN</B></P><UL><LI> 1µl T4 DNA Ligase Buffer
</LI></UL><UL><LI> 1µl T4 DNA Ligase
</LI></UL><UL><LI> 7µl pSB1C3
</LI></UL><UL><LI> 1µl purified mycN
</LI></UL><UL><LI> incubation for 1h at roomtemperature
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_XL1_with_mycC.2FmycN_in_pSB1C3">Transformation of XL1 with mycC/mycN in pSB1C3</SPAN></H3><P><B>Time:</B> 2011-09-03, 19:00</P><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Materials:</B></P><UL><LI> XL1-Blue
</LI></UL><UL><LI> ligation products: pSB1C3+mycC and pSB1C3+mycN
</LI></UL><P><B>Method:</B></P><UL><LI> addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI> incubation 25 min on ice,
</LI></UL><UL><LI> heat shock 90 sec at 42°C,
</LI></UL><UL><LI> incubation 5 min on ice,
</LI></UL><UL><LI> addition of 750 µl LB medium,
</LI></UL><UL><LI> incubation at 37 °C shaking for 60 min,
</LI></UL><UL><LI> plating on LB medium with appropriate antibiotic (Kan)
</LI></UL><UL><LI> storage over night at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><P><B>Output:</B></P><UL><LI> 2 plates with Cm: mycC, mycN
</LI></UL><UL><LI> 1 plate with Cm: control myc
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Oligo-Fillin_for_mdnA-Library_2_.28repetition.29">Oligo-Fillin for mdnA-Library 2 (repetition)</SPAN></H3><P><B>Time:</B> 2011-09-03, 11:00</P><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Materials</B></P><UL><LI> Primer, 25 µM :
</LI></UL><UL><LI> # 74: o_foc_library_2
</LI></UL><UL><LI> Klenow-Buffer 10X
</LI></UL><UL><LI> Klenow Fragment
</LI></UL><UL><LI> dNTPs
</LI></UL><UL><LI> water
</LI></UL><P><B>Protocol:</B></P><UL><LI>Reaction mix 1
</LI></UL><UL><LI> 1 µl fw-Oligonucleotide (#76)
</LI></UL><UL><LI> 1 µl rev-Oligonucleotide (#74)
</LI></UL><UL><LI> 2 µl dNTPs (10 mM)
</LI></UL><UL><LI> 2 µl Klenow-Buffer
</LI></UL><UL><LI> 0.5 µl Klenow Fragment
</LI></UL><UL><LI> 14 µl water
</LI></UL><UL><LI> total volume: 20.5 µl
</LI></UL><UL><LI> Reaction mix 2
</LI></UL><UL><LI> 2 µl fw-Oligonucleotide (#76)
</LI></UL><UL><LI> 2 µl rev-Oligonucleotide (#74)
</LI></UL><UL><LI> 2 µl dNTPs (10 mM)
</LI></UL><UL><LI> 2 µl Klenow-Buffer
</LI></UL><UL><LI> 0.5 µl Klenow Fragment
</LI></UL><UL><LI> 12 µl water
</LI></UL><UL><LI> total volume: 20.5 µl
</LI></UL><UL><LI> reaction mix 3
</LI></UL><UL><LI> 3 µl fw-Oligonucleotide (#76)
</LI></UL><UL><LI> 3 µl rev-Oligonucleotide (#74)
</LI></UL><UL><LI> 2 µl dNTPs (10 mM)
</LI></UL><UL><LI> 2 µl Klenow-Buffer
</LI></UL><UL><LI> 0.5 µl Klenow Fragment
</LI></UL><UL><LI> 10 µl water
</LI></UL><UL><LI> total volume: 20.5 µl
</LI></UL><P>2. PCR program
</P><UL><LI> name: Fillin
</LI></UL><UL><LI> 3 min 94 °C
</LI></UL><UL><LI> 0.3°C per s (94°C-37°C)
</LI></UL><UL><LI> addition of 0.5 µl Klenow Fragment
</LI></UL><UL><LI> press enter
</LI></UL><UL><LI> 1hr 37°C
</LI></UL><P>Agarose gel:
</P><P>1 %: 0.5 g in 50 ml TAE
</P><P><B>Results:</B></P><UL><LI> Output:
</LI></UL><UL><LI> mdnA-Lib2 1µl
</LI></UL><UL><LI> mdnA-Lib2 2µl
</LI></UL><UL><LI> mdnA-Lib2 3µl
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Gel purification
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Gel_purification_of_Oligo-Fillin_for_mdnA-Library_2_.28repetition.29">Gel purification of Oligo-Fillin for mdnA-Library 2 (repetition)</SPAN></H3><P><B>Time:</B> 2011-09-03, 13:00</P><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Materials</B></P><UL><LI> Macherey Nagel - NucleoSpin Extract II, protocol for DNA extraction from agarose gels
</LI></UL><UL><LI> Eluation with 50µl NE-Buffer</LI></UL><P><B>NanoDrop: Concentrations</B></P><UL><LI> Lib2 (1 µl) purified: 13.8 ng/µl
</LI></UL><UL><LI> Lib2 (2 µl) purified: 14.9 ng/µl
</LI></UL><UL><LI> Lib2 (3 µl) purified: 9.0 ng/µl
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_Oligo-Fillin_for_mdnA-Library_2_.28repetition.29_and_pUP089">Restriction enzyme digestion of Oligo-Fillin for mdnA-Library 2 (repetition) and pUP089</SPAN></H3><P><B>Time:</B> 2011-09-03, 13:00</P><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Materials</B></P><UL><LI> purified Lib2 samples
</LI></UL><UL><LI> pUP089 (2011-06-22)
</LI></UL><UL><LI> AatII
</LI></UL><UL><LI> NEB Buffer 4
</LI></UL><P><B>Method</B></P><P><B>Digestion protocol for Lib2</B></P><UL><LI> 50 µl purified Lib2 sample
</LI></UL><UL><LI> 2.0 µl AatII
</LI></UL><UL><LI> 6 µl Buffer 4
</LI></UL><UL><LI> 2.0 µl water
</LI></UL><UL><LI> 37°C for 1 h
</LI></UL><P><B>Digestion protocol for pUP089:</B></P><UL><LI> pUP089
</LI></UL><UL><LI> NEB Buffer 4
</LI></UL><UL><LI> SfoI
</LI></UL><UL><LI> AatII
</LI></UL><UL><LI> digestion protocol
</LI></UL><UL><LI> 4 µl DNA
</LI></UL><UL><LI> 3µl Buffer 4
</LI></UL><UL><LI> 1.5 µl AatII
</LI></UL><UL><LI> 1.5 SfoI
</LI></UL><UL><LI> 20 µl water
</LI></UL><UL><LI> incubation @ 37°C for 4 h
</LI></UL><UL><LI> control digestion 1
</LI></UL><UL><LI> 2 µl DNA
</LI></UL><UL><LI> 3 µl Buffer 4
</LI></UL><UL><LI> 1.5 µl AatII
</LI></UL><UL><LI> 23.5 µl water
</LI></UL><UL><LI> incubation @ 37°C for 4 h
</LI></UL><UL><LI> control digestion 2
</LI></UL><UL><LI> 2 µl DNA
</LI></UL><UL><LI> 3 µl Buffer 4
</LI></UL><UL><LI> 1.5 µl SfoI
</LI></UL><UL><LI> 23.5 µl water
</LI></UL><UL><LI> incubation @ 37°C for 1h
</LI></UL><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD></TR><TR><TD> M </TD><TD> marker, DNA ladder mix Fermentas </TD></TR><TR><TD> 1 </TD><TD> Lib2 (1µl) digested
</TD></TR><TR><TD> 2 </TD><TD> Lib2 (2µl) digested
</TD></TR><TR><TD> 3 </TD><TD> Lib2 (3µl) digested
</TD></TR><TR><TD> 4 </TD><TD> pUP SfoI/AatII
</TD></TR><TR><TD> 5 </TD><TD> pUP AatII (control)
</TD></TR><TR><TD> 6 </TD><TD> pUP SfoI (control)
</TD></TR></TBODY></TABLE><UL><LI> fragments were excised and purified using Macherey-Nagel Nucleo SpinII Extract Kit
</LI></UL><UL><LI> concentrations after purification
</LI></UL><UL><LI> Lib2 (1µl): 6.8 ng/µl
</LI></UL><UL><LI> Lib2 (2µl): 4.7 ng/µl
</LI></UL><UL><LI> Lib2 (3µl): 4.9 ng/µl
</LI></UL><UL><LI> pUP089: 14.9 ng/µl
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_Lib2_.28repetition.29_and_pUP089">Ligation of Lib2 (repetition) and pUP089</SPAN></H3><P><B>Time:</B> 2011-09-03, 17:30</P><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Materials</B></P><UL><LI> T4 DNA Ligase Buffer (Fermentas)
</LI></UL><UL><LI> T4 DNA Ligase
</LI></UL><UL><LI> purified samples of Lib2 and pUP089
</LI></UL><P><B>Method</B></P><UL><LI> 1µl T4 DNA Ligase Buffer
</LI></UL><UL><LI> 1µl T4 DNA Ligase
</LI></UL><UL><LI> 7µl pUP089
</LI></UL><UL><LI> 1µl Lib2 (1µl)
</LI></UL><UL><LI> also preparing ligation control (1µl water instead of Lib2)
</LI></UL><UL><LI> incubation for 1h at roomtemperature
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_XL1_with_Lib2_in_pUP089">Transformation of XL1 with Lib2 in pUP089</SPAN></H3><P><B>Time:</B> 2011-09-03, 19:00</P><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Materials:</B></P><UL><LI> XL1-Blue
</LI></UL><UL><LI> ligation products: pUP089+Lib2
</LI></UL><P><B>Method:</B></P><UL><LI> addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI> incubation 25 min on ice,
</LI></UL><UL><LI> heat shock 90 sec at 42°C,
</LI></UL><UL><LI> incubation 5 min on ice,
</LI></UL><UL><LI> addition of 750 µl LB medium,
</LI></UL><UL><LI> incubation at 37 °C shaking for 60 min,
</LI></UL><UL><LI> plating on LB medium with appropriate antibiotic (Kan)
</LI></UL><UL><LI> storage over night at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><P><B>Output:</B></P><UL><LI> 10 plates with Kan: Lib2
</LI></UL><UL><LI> 1 plate with Kan: control Lib2
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_XL1_with_K3_and_A3_expression_backbones">Transformation of XL1 with K3 and A3 expression backbones</SPAN></H3><P><B>Time:</B> 2011-09-03, 19:00</P><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Materials:</B></P><UL><LI> XL1-Blue
</LI></UL><UL><LI> DNA of:
</LI></UL><UL><LI> pSB1A3_YFP_Ara clone A
</LI></UL><UL><LI> pSB1A3_YFP_Lac clone B
</LI></UL><UL><LI> pSB1K3_CFP_Ara clone A
</LI></UL><UL><LI> pSB1K3_CFP_Lac clone B
</LI></UL><UL><LI> pSB1K3_YFP_Lac clone C
</LI></UL><P><B>Method:</B></P><UL><LI> addition of 2 µl DNA to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI> incubation 25 min on ice,
</LI></UL><UL><LI> heat shock 45 sec at 42°C,
</LI></UL><UL><LI> incubation 2 min on ice,
</LI></UL><UL><LI> addition of 750 µl LB medium,
</LI></UL><UL><LI> incubation at 37 °C shaking for 60 min,
</LI></UL><UL><LI> plating on LB medium with appropriate antibiotic (Kan/Amp)
</LI></UL><UL><LI> storage over night at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Picking_clones_for_overnight_cultures_of_pSB1C3_.2B_DE_clone_2_and_clone_3">Picking clones for overnight cultures of pSB1C3 + DE clone 2 and clone 3</SPAN></H3><P><B>Investigators:</B>Sebastian, Katharina</P><P><B>Time:</B> 2011-09-04, 16:00</P><P><B>Materials:</B></P><UL><LI> liquid cultures of mdnDE clone 2 and mdnDE clone 3
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> chloramphenicol (25 mg/ml)
</LI></UL><P><B>Method:</B></P><UL><LI> inoculate 1 ml of liquid culture to 4 ml LB medium+ Cm
</LI></UL><UL><LI> incubate over night in 37°C shaker (200 rpm)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> miniprep
</LI></UL><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="overnight_culture_of_picked_E._coli_clones_transformed_with_of_XL1blue-cells_with_pPDV089">overnight culture of picked <I>E. coli</I> clones transformed with of XL1blue-cells with pPDV089</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI> 3 clones from pPDV089, clone 2S14, (ampicillin and tetracyclin)
</LI></UL><UL><LI> 5 ml LB medium per clone
</LI></UL><UL><LI> storage over night at 37°C and 750 rpm</LI></UL><P><B>Further tasks:</B></P><UL><LI> amplification for phage display and test digestion</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="85th_Labday_2011-09-04">85th Labday 2011-09-04</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Production_of_phages_containing_pPDV089_in_XL1_blue_cells">Production of phages containing pPDV089 in XL1 blue cells</SPAN></H3><P><B>Investigators:</B> Sandrina, Katharina, Nadine</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI><B>first step</B>: amplification of cells containing pPDV089( clone: 2S14):
</LI></UL><UL><LI> 50 ml DYT medium will be inoculated with the cells, so that OD600 = 0.1
</LI></UL><UL><LI> add antibiotics tetracyclin and ampicillin to the medium
</LI></UL><UL><LI> cells should be incubated 37 °C till OD600 = 0.3-0.5 (here: 0.332) is reached
</LI></UL><UL><LI><B>second step</B>: infection with helper phages
</LI></UL><UL><LI> add helper phages 10^11 phages/50 ml (...)
</LI></UL><UL><LI> incubate for 10 min at 37°C (without shaking!)
</LI></UL><UL><LI> add 0,5 mM IPTG
</LI></UL><UL><LI> incubate 50 min at 28°C and rpm
</LI></UL><UL><LI> add 70 µg/ml kanamycin and incubate for 5 h at 28°C (...)
</LI></UL><UL><LI><B>third step</B>: phage purification
</LI></UL><UL><LI> centrifuge cell culture at 5000 x g/ 15 min
</LI></UL><UL><LI> fill supernatant in a new 50 ml falcon and centrifuge again (5000 g/15 min)
</LI></UL><UL><LI> 40 ml of the supernatant with 8 ml PEG-NaCl (20% (w/v) PEG-8000, 2,5 M NaCl)
</LI></UL><UL><LI> incubate over night at 4°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> go on with phage purification</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_cells_picked_from_plates_on_02.09.2011_and_inoculated_into_ON_precultures"> Miniprep of cells picked from plates on 02.09.2011 and inoculated into ON precultures </SPAN></H3><P><B>Investigators:</B> Sandrina, Katharina</P><P><B>Time:</B> 2011-09-04, 11:00</P><P><B>1. Miniprep:</B></P><UL><LI> 15 overnight cultures (pUP SG3- pUP SG17)
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> pUP SG3 </TD><TD> 1410.2
</TD></TR><TR><TD> pUP SG4 </TD><TD> 1146.2
</TD></TR><TR><TD> pUP SG5 </TD><TD> 1289.7
</TD></TR><TR><TD> pUP SG6 </TD><TD> 940.1
</TD></TR><TR><TD> pUP SG7 </TD><TD> 1418.5
</TD></TR><TR><TD> pUP SG8 </TD><TD> 1244.4
</TD></TR><TR><TD> pUP SG9 </TD><TD> 1287.6
</TD></TR><TR><TD> pUP SG10 </TD><TD> 1188.7
</TD></TR><TR><TD> pUP SG11 </TD><TD> 1276.1
</TD></TR><TR><TD> pUP SG12 </TD><TD> 1193.9
</TD></TR><TR><TD> pUP SG13 </TD><TD> 1255.1
</TD></TR><TR><TD> pUP SG14 </TD><TD> 1474.7
</TD></TR><TR><TD> pUP SG15 </TD><TD> 1143.5
</TD></TR><TR><TD> pUP SG16 </TD><TD> 1268.0
</TD></TR><TR><TD> pUP SG17 </TD><TD> 1218.7
</TD></TR></TBODY></TABLE><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_from_ligation_of_pSB1C3.2BmdnDE_clone_2_and_clone3">Miniprep of overnight cultures from ligation of pSB1C3+mdnDE clone 2 and clone3</SPAN></H3><P><B>Investigators:</B> Sandrina, Katharina</P><P><B>Time:</B> 2011-09-04, 11:00</P><P><B>Material:</B></P><UL><LI> 2 overnight cultures (mdnDE clone 2 and mdnDE clone 3)
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><P><B>Results:</B></P><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> mdnDE clone 2 </TD><TD> 601.9
</TD></TR><TR><TD> mdnDE clone 3 </TD><TD> 571.7
</TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_XL1_with_mdnBC_in_pSB1C3">Transformation of XL1 with mdnBC in pSB1C3</SPAN></H3><P><B>Time:</B> 2011-09-04, 14:00</P><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Materials:</B></P><UL><LI> XL1-Blue
</LI></UL><UL><LI> ligation products: pSB1C3+mdnBC (Niels 2011-08-24)
</LI></UL><P><B>Method:</B></P><UL><LI> addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI> incubation 25 min on ice,
</LI></UL><UL><LI> heat shock 45 sec at 42°C,
</LI></UL><UL><LI> incubation 2 min on ice,
</LI></UL><UL><LI> addition of 750 µl LB medium,
</LI></UL><UL><LI> incubation at 37 °C shaking for 60 min,
</LI></UL><UL><LI> plating on LB medium with appropriate antibiotic (Cm)
</LI></UL><UL><LI> storage over night at 37°C
</LI></UL><UL><LI><B>no ligation control available</B></LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Control_experiment:_Transformation_of_XL1_with_pARW089_and_pARW071.2C_respectively">Control experiment: Transformation of XL1 with pARW089 and pARW071, respectively</SPAN></H3><P><B>Time:</B> 2011-09-04, 14:00</P><P><B>Investigators:</B> Nadine, Katharina </P><P><B>Materials:</B></P><UL><LI> XL1-Blue
</LI></UL><UL><LI> DNA of pARW089 and pARW071 (2011-05-24)
</LI></UL><P><B>Method:</B></P><UL><LI> addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI> incubation 25 min on ice,
</LI></UL><UL><LI> heat shock 45 sec at 42°C,
</LI></UL><UL><LI> incubation 2 min on ice,
</LI></UL><UL><LI> addition of 750 µl LB medium,
</LI></UL><UL><LI> incubation at 37 °C shaking for 60 min,
</LI></UL><UL><LI> plating on LB medium with appropriate antibiotic (Cm)
</LI></UL><UL><LI> storage over night at 37°C
</LI></UL><UL><LI><B>no ligation control available</B></LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Picking_clones_for_overnight_cultures_of_pSB1C3.2BmycC_and_pSB1C3.2BmycN">Picking clones for overnight cultures of pSB1C3+mycC and pSB1C3+mycN</SPAN></H3><P><B>Investigators:</B>Katharina</P><P><B>Time:</B> 2011-09-04, 16:00</P><P><B>Materials:</B></P><UL><LI> agar plates of pSB1C3+mycC and pSB1C3+mycN
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> chloramphenicol (25 mg/ml)
</LI></UL><P><B>Method:</B></P><UL><LI> picking 5 clones for each construct
</LI></UL><UL><LI> incubate over night in 37°C shaker (200 rpm)
</LI></UL><P><B>Output:</B></P><UL><LI> 5 liquid cultures of mycC (1-5)
</LI></UL><UL><LI> 5 liquid cultures of mycN (1-5)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> miniprep
</LI></UL><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Picking_clones_for_overnight_cultures_of_pARW089_and_pARW071">Picking clones for overnight cultures of pARW089 and pARW071</SPAN></H3><P><B>Investigators:</B>Katharina</P><P><B>Time:</B> 2011-09-04, 16:00</P><P><B>Materials:</B></P><UL><LI> Glycerolstocks of pARW089 and pARW071
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> Kan
</LI></UL><P><B>Method:</B></P><UL><LI> preparing one liquid culture for each construct
</LI></UL><UL><LI> incubate over night in 37°C shaker (200 rpm)
</LI></UL><P><B>Further tasks:</B></P><UL><LI><B>preparation of 400ml culture for HPLC analysis on 2011-09-05</B></LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="86th_Labday_2011-09-05">86th Labday 2011-09-05</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Test_digest_of_isolated_plasmids_.284.9.2011.29_from_clones_that_were_positive_in_colony_PCR_.282.09.2011.29">Test digest of isolated plasmids (4.9.2011) from clones that were positive in colony PCR (2.09.2011)</SPAN></H3><P><B> Investigators: </B> Paul, Stefan, Sebastian
</P><P><B> Aim: </B> Prove whether plasmids contain all expected inserts.
</P><P><B> Method: </B> Digestion of plasmids with HincII. Produces 3 fragments from both, the 14_3C and the TEV containing plasmids.
</P><UL><LI>Expected fragments:
</LI></UL><UL><LI>Tev: 550 bp, 2700 bp, 3400 bp
</LI></UL><UL><LI>14_3C: 550 bp, 2300 bp, 3600 bp
</LI></UL><UL><LI>5µl 10x buffer 4 (NEB)
</LI></UL><UL><LI>0,5µl 100x BSA
</LI></UL><UL><LI>1µl Sample
</LI></UL><UL><LI>0,5µl HincII Enzyme (NEB)
</LI></UL><UL><LI>43µl water
</LI></UL><UL><LI>15 reaction batches, 7 for 14_3C, 8 for TEV:
</LI></UL><P>following colnes were used:
</P><P>2 pUP_SG3_ssTorA_CS-14_3C_blaFL_AraC-14_3C1
</P><P>3 pUP_SG4_ssTorA_CS-14_3C_blaFL_AraC-14_3C3
</P><P>4 pUP_SG5_ssTorA_CS-14_3C_blaFL_AraC-14_3C4
</P><P>5 pUP_SG6_ssTorA_CS-14_3C_blaFL_AraC-14_3C6
</P><P>6 pUP_SG7_ssTorA_CS-14_3C_blaFL_AraC-14_3C8
</P><P>7 pUP_SG8_ssTorA_CS-14_3C_blaFL_AraC-14_3C9
</P><P>8 pUP_SG9_ssTorA_CS-14_3C_blaFL_AraC-14_3C10
</P><P>11 pUP_SG10_ssTorA_CS-TEV_blaFL_AraC-TEV2
</P><P>12 pUP_SG11_ssTorA_CS-TEV_blaFL_AraC-TEV4
</P><P>13 pUP_SG12_ssTorA_CS-TEV_blaFL_AraC-TEV5
</P><P>14 pUP_SG13_ssTorA_CS-TEV_blaFL_AraC-TEV6
</P><P>15 pUP_SG14_ssTorA_CS-TEV_blaFL_AraC-TEV7
</P><P>16 pUP_SG15_ssTorA_CS-TEV_blaFL_AraC-TEV8
</P><P>17 pUP_SG16_ssTorA_CS-TEV_blaFL_AraC-TEV9
</P><P>18 pUP_SG17_ssTorA_CS-TEV_blaFL_AraC-TEV10
</P><P><B>Results:</B></P><P><A href="/File:UP_AG_destdigest_P_T_20110905.JPG" title="File:UP AG destdigest P T 20110905.JPG">400px</A></P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="competent_cells_-_ER2738">competent cells - ER2738 </SPAN></H3><P><B>Investigator:</B> Katharina, Niels, Sandrina, Steffi</P><P><B>Aim:</B> produce competent cells
</P><P><B>Materials/Methods:</B></P><TABLE class="wikitable" border="1"><TBODY><TR><TD><B>TFB I</B></TD><TD><B>1000ml</B></TD><TD><B>200ml</B></TD></TR><TR><TD> 100mM Rubidium Chloride </TD><TD> 12.1 </TD><TD> 2.42g
</TD></TR><TR><TD> 30mM Potassium Acetate</TD><TD> 2.944 </TD><TD> 0.59g
</TD></TR><TR><TD> 10mM Calcium Chloride </TD><TD> 1.47 </TD><TD> 0.29g
</TD></TR><TR><TD> 15% w/v Glycerol (87%) </TD><TD> 150 </TD><TD> 34.5g
</TD></TR></TBODY></TABLE><P>Adjust pH to 5.8 with acetic acid</P><P>Filter sterilize the solution</P><TABLE class="wikitable" border="1"><TBODY><TR><TD><B>TFB II</B></TD><TD><B>500ml</B></TD><TD><B>100ml</B></TD></TR><TR><TD> 50mM Rubidium Chloride </TD><TD> 0.6 </TD><TD> 0.121g
</TD></TR><TR><TD> 10mM MOPS</TD><TD> 1.05 </TD><TD> 0.210g
</TD></TR><TR><TD> 75mM Calcium Chloride </TD><TD> 5.51 </TD><TD> 1.100g
</TD></TR><TR><TD> 15% w/v Glycerol (87%) </TD><TD> 75 </TD><TD> 17.24g
</TD></TR></TBODY></TABLE><P>Adjust pH to 7.0 with KOH</P><P>Filter sterilize the solution</P><P>Work always sterile and cold and speedy!
</P><UL><LI> All volumes deal with the common cellline!
</LI></UL><UL><LI> Prepare 80 Eppis (1,5?l)
</LI></UL><UL><LI> get liquid nitrogen
</LI></UL><UL><LI> prepare 5 ml LB-Medium with the specific antibiotic (for ER2738: Tet), inoculate and incubate over night
</LI></UL><UL><LI> prepare 200 ml LB-Medium with the specific antibiotic, inoculate with 2 ml of the over-night-culture
</LI></UL><UL><LI> grow while shaking at 37°C, 190 rpm to an OD600 at 0,4-0,6
</LI></UL><UL><LI> keep cell suspension in sterile falcons (50 ml) 10 min on ice, then centrifuge for 5 min, 4°C, 4000 rpm
</LI></UL><UL><LI> discard supernatant, carefully resuspend on ice with 40 ml icecold TFB I and keep 10 min on ice
</LI></UL><UL><LI> centrifuge for 5 min, 4°C, 4000 rpm
</LI></UL><UL><LI> discard supernatant, carefully resuspend pellet in 8 ml TFB II
</LI></UL><UL><LI> aliquot in Eppis: 50?l per tube and store immediately at liquid nitrogen and afterwards at -80 °C
</LI></UL><P><B>Results:</B></P><UL><LI> 80 tubes ER2738 for transformation(50 µl competent cells at -80°C)
</LI></UL><P><B>Further tasks:</B> check by transformation and check the resistance on agar plates with different antibiotics
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="check_resistance_of_competent_cells_-_E._coli_ER2738">check resistance of competent cells - <I>E. coli</I> ER2738</SPAN></H3><P><B>Investigator:</B> Niels, Katharina, Sandrina, Steffi</P><P><B>Aim:</B> check resistance of competent ER2738-cells on agar plates with different antibiotics
</P><P><B>Materials/Methods:</B></P><UL><LI>competent ER2738-cells from 2011-09-05 (Niels, Katharina, Sandrina, Steffi)
</LI></UL><UL><LI>LB-plates with Tetracycline, Chloramphenicol, Kanamycine, Ampicillin, without antibiotics
</LI></UL><UL><LI>plating 50 µl on agar plates
</LI></UL><UL><LI>incubate at 37°C over night
</LI></UL><P><B>Further tasks:</B> control agar plates
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_pSB1C3.2BmycC_and_pSB1C3.2BmycN">Miniprep of overnight cultures of pSB1C3+mycC and pSB1C3+mycN</SPAN></H3><P><B>Investigators:</B>Nadine, Nicole, Katharina</P><P><B>Time:</B> 2011-09-04, 9:00</P><P><B>Materials:</B></P><UL><LI> liquid culture of pSB1C3+mycC and pSB1C3+mycN
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> mycN clone 1 </TD><TD> 400.6
</TD></TR><TR><TD> mycN clone 2 </TD><TD> 343.0
</TD></TR><TR><TD> mycN clone 3 </TD><TD> 395.4
</TD></TR><TR><TD> mycN clone 4 </TD><TD> 618.0
</TD></TR><TR><TD> mycN clone 5 </TD><TD> 551.1
</TD></TR><TR><TD> mycC clone 1 </TD><TD> 519.7
</TD></TR><TR><TD> mycC clone 2 </TD><TD> 453.3
</TD></TR><TR><TD> mycC clone 3 </TD><TD> 631.5
</TD></TR><TR><TD> mycC clone 4 </TD><TD> 689.5
</TD></TR><TR><TD> mycC clone 5 </TD><TD> 897.3
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> test digestion for confirmation
</LI></UL><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Cultures_of_pARW089_and_pARW071_for_HPLC-Analysis">Cultures of pARW089 and pARW071 for HPLC-Analysis</SPAN></H3><P><B>Investigators:</B>Katharina, Nadine, Nicole, Steffi</P><P><B>Materials:</B></P><UL><LI> overnight cultures of pARW089 and pARW071 from 2011-09-04 (Katharina)
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> Kan
</LI></UL><P><B>Method:</B></P><UL><LI> preparing one liquid culture for each construct with Kanamycine
</LI></UL><UL><LI> incubate for 6 hrs in 37°C shaker (190 rpm)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> HPLC analysis
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Oligo-Fillin_for_mdnA-Library_2_.28repetition.29_2">Oligo-Fillin for mdnA-Library 2 (repetition)</SPAN></H3><P><B>Time:</B> 2011-09-05, 10:00</P><P><B>Investigators:</B> Nadine, Katharina, Jessica, Nicole, Steffi </P><P><B>Aim:</B> Repetition of fillin reaction for library with new fillin program
</P><P><B>Materials</B></P><UL><LI> Primer, 25 µM :
</LI></UL><UL><LI> # 76
</LI></UL><UL><LI> # 74
</LI></UL><UL><LI> Klenow-Buffer 10X
</LI></UL><UL><LI> Klenow Fragment
</LI></UL><UL><LI> dNTPs
</LI></UL><UL><LI> water
</LI></UL><P><B>Protocol:</B></P><UL><LI> Reaction mix (50 µl)
</LI></UL><UL><LI> 3 µl fw-Oligonucleotide (#76)
</LI></UL><UL><LI> 3 µl rev-Oligonucleotide (#74)
</LI></UL><UL><LI> 5 µl dNTPs (10 mM)
</LI></UL><UL><LI> 5 µl Klenow-Buffer
</LI></UL><UL><LI> 34 µl water
</LI></UL><UL><LI> total volume: 50 µl </LI></UL><P>(2. Fillin program
</P><UL><LI> name: Fillin
</LI></UL><UL><LI> 3 min 94 °C
</LI></UL><UL><LI> 0.3°C per s (94°C-37°C)
</LI></UL><UL><LI> addition of 0.5 µl Klenow Fragment
</LI></UL><UL><LI> press enter
</LI></UL><UL><LI> 1hr 37°C)
</LI></UL><P><B>Results:</B></P><UL><LI> Output:
</LI></UL><UL><LI> mdnA-Lib2, Nad, 2011-9-5
</LI></UL><P><B>Further tasks:</B></P><UL><LI> digest w/ AatII
</LI></UL><UL><LI> ligation w/ pUP089 (digested w/ AatII and SfoI)
</LI></UL><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122);"><SPAN class="mw-headline" id="Control_restriction_enzyme_digestion_of_pSB1C3_mdnA_mycC_and_pSB1C3_mdnA_mycN">Control restriction enzyme digestion of pSB1C3_mdnA_mycC and pSB1C3_mdnA_mycN</SPAN></H3><P><B>Investigators:</B> Nadine, Nicole, Katharina, Steffi
</P><P><B>Aim:</B> Confirmation of the insertion of mdnA with myc-tag (N-terminal and C-terminal) in pSB1C3 vector
</P><P><B>Time:</B> 2011-09-05, 10:00-11:30</P><P><B>Material: </B></P><UL><LI> clones:
</LI></UL><UL><LI> pSB1C3_mdnA_mycC, clone 1, Katharina, 2011-09-05
</LI></UL><UL><LI> pSB1C3_mdnA_mycC, clone 2, Katharina, 2011-09-05
</LI></UL><UL><LI> pSB1C3_mdnA_mycC, clone 3, Katharina, 2011-09-05
</LI></UL><UL><LI> pSB1C3_mdnA_mycC, clone 4, Katharina, 2011-09-05
</LI></UL><UL><LI> pSB1C3_mdnA_mycC, clone 5, Katharina, 2011-09-05
</LI></UL><UL><LI> pSB1C3_mdnA_mycN, clone 1, Katharina, 2011-09-05
</LI></UL><UL><LI> pSB1C3_mdnA_mycN, clone 2, Katharina, 2011-09-05
</LI></UL><UL><LI> pSB1C3_mdnA_mycN, clone 3, Katharina, 2011-09-05
</LI></UL><UL><LI> pSB1C3_mdnA_mycN, clone 4, Katharina, 2011-09-05
</LI></UL><UL><LI> pSB1C3_mdnA_mycN, clone 5, Katharina, 2011-09-05
</LI></UL><UL><LI> DraI (Fermentas)
</LI></UL><UL><LI> Tango buffer (Fermentas)
</LI></UL><P><B>Method: </B></P><P>Reaction mix
</P><UL><LI> 2 µl DNA
</LI></UL><UL><LI> 2 µl Tango buffer
</LI></UL><UL><LI> 2 µl DraI (cuts three times)
</LI></UL><UL><LI> 14 µl H<SMALL>2</SMALL>O
</LI></UL><P>Reaction conditions
</P><UL><LI> 37°C, 2 hours
</LI></UL><P><B>Expected results: </B></P><UL><LI> four fragments for each plasmid
</LI></UL><UL><LI> fragment 1: 771 bases
</LI></UL><UL><LI> fragment 2: 692 bases
</LI></UL><UL><LI> fragment 3: 19 bases
</LI></UL><UL><LI> fragment 4: 1114 bases
</LI></UL><UL><LI> pSB1C3_mdnA_mycC (four fragments)
</LI></UL><UL><LI> fragment 1: 804 bases
</LI></UL><UL><LI> fragment 2: 692 bases
</LI></UL><UL><LI> fragment 3: 19 bases
</LI></UL><UL><LI> fragment 4: 1082 bases
</LI></UL><P><B>Conclusion: </B></P><P><B>Output: </B>clone_name, cell type, stored fridge/freezer/-80°C; model saved as <I>name</I> in <I>folder</I></P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR:_mdnB.2C_mdnABCDE">PCR: mdnB, mdnABCDE </SPAN></H3><P><B>Time:</B> 2011-09-05</P><P><B>Investigators:</B> Nicole, Nadja</P><P><B>Materials</B></P><UL><LI> vector: pARW089 (8,6 ng/µl), pARW071 (6,3 ng/µl), pARW089_T7 (8,6 ng/µl),
</LI></UL><UL><LI> 1-Phusion HF Polymerase, NEB
</LI></UL><UL><LI> 1-Phusion HF Buffer
</LI></UL><UL><LI> 2-PFU- Ultra HF Polymerase, NEB
</LI></UL><UL><LI> 2-PFU- Ultra HF Buffer, NEB
</LI></UL><UL><LI> dNTPs
</LI></UL><UL><LI> water
</LI></UL><UL><LI> Primer:
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>#</B></TH><TH><B>Primer</B></TH></TR><TR><TD> 58</TD><TD> pf_mdnB_EcoRI_NotI_XbaI 12.08.
</TD></TR><TR><TD>81</TD><TD> pr_mdnB_SpeI_NotI_PstI_30.08.
</TD></TR><TR><TD> 84</TD><TD> pf_mdnABCDE89_EcoRI_NotI_XbaI
</TD></TR><TR><TD>82</TD><TD> r_mdnABCDE_iGEM
</TD></TR></TBODY></TABLE><P><B>Protocol:</B></P><UL><LI>mdnB
</LI></UL><UL><LI> 1 µl pARW089 (diluted 1:100)
</LI></UL><UL><LI> 1 µl dNTPs (10 mM)
</LI></UL><UL><LI> 2.5 µl Primer forward (10 µM)- 81
</LI></UL><UL><LI> 2.5 µl Primer backward (10 µM)- 58
</LI></UL><UL><LI> 10 µl Phusion buffer HF
</LI></UL><UL><LI> 0.5 µl Phusion HF Ploymerase (2U/µl)
</LI></UL><UL><LI> 32.5 µl water
</LI></UL><UL><LI> total volume: 50 µl
</LI></UL><UL><LI>mdnABCDE
</LI></UL><UL><LI> 1 µl pARW089, 1 µl pARW089_T7, 1,3 µl pARW071
</LI></UL><UL><LI> 1 µl dNTPs (10 mM)
</LI></UL><UL><LI> 1 µl 2-PFU- Ultra HF Polymerase, NEB
</LI></UL><UL><LI> 5 µl 2-PFU- Ultra HF Buffer, NEB
</LI></UL><UL><LI> 40 µl (pARW089 pARW089_T7) water, 39,7 µl (pARW071) water
</LI></UL><UL><LI> 1 µl Primer forward (10 mM) - 84
</LI></UL><UL><LI> 1 µl Primer backward (10 mM)- 82
</LI></UL><P>2. PCR programs
</P><UL><LI>IGMDNBB for mdnB
</LI></UL><UL><LI> first steps: 10x
</LI></UL><UL><LI> second steps: 20x
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>Step</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>Hot Start</TD><TD> 98°C </TD><TD> Hold
</TD></TR><TR><TD> Initial denaturation</TD><TD> 98°C </TD><TD> 30 sec
</TD></TR><TR><TD>Denaturation</TD><TD> 98°C</TD><TD> 10 s
</TD></TR><TR><TD>Annealing</TD><TD> 64°C</TD><TD> 20 s
</TD></TR><TR><TD> Elongation</TD><TD> 72°C</TD><TD> 25 s
</TD></TR><TR><TD>DenaturationII </TD><TD> 98°C</TD><TD> 10 s
</TD></TR><TR><TD>AnnealingII </TD><TD> 72°C</TD><TD> 20 s
</TD></TR><TR><TD>ElongationII </TD><TD> 72°C</TD><TD> 30 s
</TD></TR><TR><TD> Final Elongation </TD><TD> 72°C </TD><TD> 10 min
</TD></TR></TBODY></TABLE><UL><LI> IGMDNAB
</LI></UL><UL><LI> first steps: 10x
</LI></UL><UL><LI> second steps: 20x
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>Step</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>Hot Start</TD><TD> 98°C </TD><TD> Hold
</TD></TR><TR><TD> Initial denaturation</TD><TD> 98°C </TD><TD> 30 sec
</TD></TR><TR><TD>Denaturation</TD><TD> 98°C</TD><TD> 10 s
</TD></TR><TR><TD>Annealing</TD><TD> 69°C</TD><TD> 20 s
</TD></TR><TR><TD>Elongation</TD><TD> 72°C</TD><TD> 2 min 10 s
</TD></TR><TR><TD>DenaturationII</TD><TD> 98°C</TD><TD> 30 s
</TD></TR><TR><TD>AnnealingII</TD><TD> 72°C</TD><TD> 10 s
</TD></TR><TR><TD>ElongationII</TD><TD> 72°C</TD><TD> 2min 30 s
</TD></TR><TR><TD> Final extension </TD><TD> 72°C </TD><TD> 10 min
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> M: DNA ladder mix (Fermentas)
</LI></UL><UL><LI> 1: mdnABCDE from pARW089, exp. size: ~ 6500 bp
</LI></UL><UL><LI> 2: mdnABCDE+T7 from pARW089, exp. size: ~ 6500 bp
</LI></UL><UL><LI> 3: mdnABCDE from pARW071, exp. size: ~ 6500 bp
</LI></UL><UL><LI> 4: -
</LI></UL><UL><LI> 5: mdnB, exp. size: ~1000 bp
</LI></UL><UL><LI> Output:
</LI></UL><UL><LI> mdnB worked out
</LI></UL><UL><LI>mdnABCDE did not work out
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Different_overnight_cultures">Different overnight cultures</SPAN></H3><P><B>Investigators:</B> Jessica, Nadja</P><P><B>Time:</B> 2011-09-04, 18:00</P><P><B>Materials:</B></P><UL><LI> glycerol stocks of pARW089 and pARW071
</LI></UL><UL><LI> plates of pSB1C3 + mdnBC (from 2011-09-04), pUP089 A and B, pSB1A3_Ara_YFP, pSB1A3_Lac_YFP, pSB1K3_Ara_CFP, pSB1K3_Lac_CFP, pSB1K3_Lac_YFP (from 2011-09-03)
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> Kan, Amp, Cm
</LI></UL><P><B>Method:</B></P><UL><LI> number of tubes: 5x mdnBC, 4x pUP089, 1x for the others
</LI></UL><UL><LI> incubate over night in 37°C shaker (200 rpm)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> pSB1C3 + mdnBC: test digest
</LI></UL><UL><LI> pUP089: miniprep and glycerol stocks?
</LI></UL><UL><LI> expression backbones: glycerol stocks, preculture for expression test
</LI></UL><UL><LI> pARW089 and pARW071: cultures for HPLC
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Purification_of_phages_containing_pPDV089">Purification of phages containing pPDV089</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI> after over night incubation:
</LI></UL><UL><LI> centrifuge precipitaed phages: 5000 x g/ 45 min
</LI></UL><UL><LI> discard supernatant
</LI></UL><UL><LI> centrifuge again, 5000 x g/ 5 min
</LI></UL><UL><LI> remove supernatant carefully
</LI></UL><UL><LI> take pellet in 1 ml TBS
</LI></UL><UL><LI> move in 1.5 ml Eppi
</LI></UL><UL><LI> centrifuge again: 17000 x g/10 min
</LI></UL><UL><LI> supernatant in a new Eppi with 200 µl PEG-NaCl (mix!)
</LI></UL><UL><LI> incubate on ice for 60 min
</LI></UL><UL><LI> centrifuge precipitated phages: 17.000 x g/ 10 min (4°C)
</LI></UL><UL><LI> resuspend pellet in 300 µl TBS
</LI></UL><UL><LI> centrifuge:17 000 x g/ 10 min (4°C)
</LI></UL><UL><LI> supernatant in a fresh eppi= purified phages
</LI></UL><P><B>Further tasks:</B></P><UL><LI> measure concentration of phages</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Glycerol_stocks_of_phages_containing_pPDV089">Glycerol stocks of phages containing pPDV089</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI> 300 µl phages in TBS
</LI></UL><UL><LI> 750 µl 86 % glycerol
</LI></UL><UL><LI> store at -80°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> measure concentration of phages</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Concentration_of_phages_containing_pPDV089_measured_by_nanodrop">Concentration of phages containing pPDV089 measured by nanodrop</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Results:</B></P><UL><LI> OD 269: 0,095
</LI></UL><UL><LI> OD 320: 0,003
</LI></UL><UL><LI> phages per ml:
</LI></UL><P>Ph/ml = (Abs 269- Abs 320) * 6*10^16/ bp (phagemid)
</P><UL><LI> bp (phagemid) = 10945
</LI></UL><P>Ph/ml= 5*10^11
</P><P><B>Further tasks:</B></P><UL><LI> measure concentration of phages by serial dilution</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="87th_Labday_2011-09-06">87th Labday 2011-09-06</SPAN></H2><H3 style="background-color: rgb(254, 122, 122);"><SPAN class="mw-headline" id="Glycerol_stocks_of_pSB1C3_mdnBC_and_pUP089">Glycerol stocks of pSB1C3_mdnBC and pUP089</SPAN></H3><P><B>Investigators:</B> Nadine, Nicole, Niels
</P><P><B>Aim:</B> Preparing of glycerol stocks of pSB1C3_mdnBC and pUP089 for further use
</P><P><B>Time:</B> 2011-09-06, 8:30-9:15 </P><P><B>Material: </B></P><P>Clones:
</P><UL><LI> pSB1C3_mdnBC, clone A, source, date
</LI></UL><UL><LI> pSB1C3_mdnBC, clone B, source, date
</LI></UL><UL><LI> pSB1C3_mdnBC, clone C
</LI></UL><UL><LI> pSB1C3_mdnBC, clone D
</LI></UL><UL><LI> pSB1C3_mdnBC, clone E
</LI></UL><UL><LI> pUP089 A clone A
</LI></UL><UL><LI> pUP089 A clone B
</LI></UL><UL><LI> pUP089 B clone A
</LI></UL><UL><LI> pUP089 B clone B
</LI></UL><P>Glycerol
</P><P><B>Method: </B></P><UL><LI> adding of each 700 µl over night culture to 300 µl glycerol
</LI></UL><UL><LI> gently inverting
</LI></UL><P><B>Results: </B></P><UL><LI> glycerol stocks
</LI></UL><P><B>Output: </B></P><P>stored in freezer (-21°C); glycerolstock box
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Number </TH><TH> Description </TH><TH> Strain </TH><TH> Resistance
</TH></TR><TR><TD> G34 </TD><TD> pSB1C3_mdnBC clone A </TD><TD> XL1-blue </TD><TD> cm
</TD></TR><TR><TD> G35 </TD><TD> pSB1C3_mdnBC clone B </TD><TD> XL1-blue </TD><TD> cm
</TD></TR><TR><TD> G36 </TD><TD> pSB1C3_mdnBC clone C </TD><TD> XL1-blue </TD><TD> cm
</TD></TR><TR><TD> G37 </TD><TD> pSB1C3_mdnBC clone D </TD><TD> XL1-blue </TD><TD> cm
</TD></TR><TR><TD> G38 </TD><TD> pSB1C3_mdnBC clone E </TD><TD> XL1-blue </TD><TD> cm
</TD></TR><TR><TD> G39 </TD><TD> pUP089 A clone A </TD><TD> XL1-blue </TD><TD> kana
</TD></TR><TR><TD> G40 </TD><TD> pUP089 A clone B </TD><TD> XL1-blue </TD><TD> kana
</TD></TR><TR><TD> G41 </TD><TD> pUP089 B clone A </TD><TD> XL1-blue </TD><TD> kana
</TD></TR><TR><TD> G42 </TD><TD> pUP089 B clone B </TD><TD> XL1-blue </TD><TD> kana
</TD></TR></TBODY></TABLE><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_pUP089_.26_psB1C3_.2B_mdnBC">Miniprep of overnight cultures of pUP089 &amp; psB1C3 + mdnBC</SPAN></H3><P><B>Investigators: </B>Niels, Steffi</P><P><B>Materials:</B></P><UL><LI> liquid culture of pSB1C3+ mdnBC &amp; puP089
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> puP089 A a </TD><TD> 508 ng/µl
</TD></TR><TR><TD> puP089 A b </TD><TD> 504 ng/µl
</TD></TR><TR><TD> puP089 B a </TD><TD> 305 ng/µl
</TD></TR><TR><TD> puP089 B b </TD><TD> 602 ng/µl
</TD></TR><TR><TD> psB1C3 + mdnBC a </TD><TD> 444 ng/µl
</TD></TR><TR><TD> psB1C3 + mdnBC b </TD><TD> 665 ng/µl
</TD></TR><TR><TD> psB1C3 + mdnBC c </TD><TD> 661 ng/µl
</TD></TR><TR><TD> psB1C3 + mdnBC d </TD><TD> 625 ng/µl
</TD></TR><TR><TD> psB1C3 + mdnBC e </TD><TD> 609 ng/µl
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> test digestion for confirmation
</LI></UL><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="check_plates_-_resistance_of_competent_E._coli_ER2738_cells">check plates - resistance of competent <I>E. coli</I> ER2738 cells</SPAN></H3><P><B>Investigators:</B> Niels, Katharina, Sandrina, Steffi</P><P><B>Aim:</B> check resistance of competent cells – ER2738 (from 2011-09-05, Niels, Katharina, Sandrina, Steffi)</P><P><B>Materials:</B></P><UL><LI> agar plates from competent cells - <I>E. coli</I> ER2738 from 2011-09-05 (San/ Ste)</LI></UL><P><B>Results:</B></P><UL><LI> all competent cells - <I>E. coli</I> ER2738 grow on agar plates with Tetracycline and without antibiotics
</LI></UL><UL><LI> no <I>E. coli</I> ER2738 clones on agar plates with Ampicillin, Kanamycine, Chloramphenicol
</LI></UL><P><B>Conclusions:</B></P><UL><LI> competent cells - <I>E. coli</I> ER2738 work and can be used for transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR_mdnABCDE_w.2F_Long_PCR_Enzyme_mix_from_Fermentas">PCR mdnABCDE w/ Long PCR Enzyme mix from Fermentas</SPAN></H3><P><B>Time:</B> 2011-09-06</P><P><B>Investigators:</B> Nicole, Nadine</P><P><B>Aim:</B> generate BioBrick of mdnABCDE, amplify mdnABCDE for ligation in pSB1C3</P><P><B>Materials</B></P><UL><LI> vector: pARW089 (8,6 ng/µl), pARW071 (6,3 ng/µl)
</LI></UL><P><B>Materials:</B></P><UL><LI> DNA polymerases: Long PCR Enzyme Mix (Fermentas)
</LI></UL><UL><LI> Template DNA: pARW071 (vector)
</LI></UL><UL><LI> dNTPs
</LI></UL><UL><LI> water
</LI></UL><UL><LI> Primer:
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>#</B></TH><TH><B>Primer</B></TH></TR><TR><TD> 84</TD><TD> pf_mdnABCDE89_EcoRI_NotI_XbaI (froward for pARW071 and pARW089)
</TD></TR><TR><TD> 82</TD><TD> r_mdnABCDE_iGEM (reverse for all)
</TD></TR><TR><TD> 83</TD><TD> pf_mdnABCDE89+T7_EcoRI_NotI_XbaI (froward for pARW089, generating mdnABCDE+T7)
</TD></TR></TBODY></TABLE><P><B>Protocol:</B></P><UL><LI>mdnABCDE
</LI></UL><UL><LI> 2 µl vector
</LI></UL><UL><LI> 1 µl dNTPs (10 mM)
</LI></UL><UL><LI> 3 µl forward Primer
</LI></UL><UL><LI> 3 µl reverse Primer
</LI></UL><UL><LI> 0.3 µl Polymerase
</LI></UL><UL><LI> 5 µl Buffer Fermentas Long Enzyme Mix (+MgCl2)
</LI></UL><UL><LI> 35.3 µl water
</LI></UL><UL><LI> total volume: 50 µl
</LI></UL><P>2. PCR program
</P><UL><LI>IGMDNBB for mdnB
</LI></UL><UL><LI> first steps: 10x
</LI></UL><UL><LI> second steps: 20x
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>Step</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>Hot Start</TD><TD> 94°C </TD><TD> Hold
</TD></TR><TR><TD> Initial denaturation</TD><TD> 94°C </TD><TD> 180 sec
</TD></TR><TR><TD>Denaturation</TD><TD> 94°C</TD><TD> 15 s
</TD></TR><TR><TD>Annealing</TD><TD> 59 </TD><TD> 30 s
</TD></TR><TR><TD> Elongation</TD><TD> 68°C</TD><TD> 300 s
</TD></TR><TR><TD>DenaturationII </TD><TD> 94°C</TD><TD> 15 s
</TD></TR><TR><TD>AnnealingII </TD><TD> 72°C</TD><TD> 30 s
</TD></TR><TR><TD>ElongationII </TD><TD> 72°C</TD><TD> 300 s + 2 s per each cycle s
</TD></TR><TR><TD> Final Elongation </TD><TD> 68°C </TD><TD> 600 s
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> Output:
</LI></UL><UL><LI> mdnABCDE from pARW071, Nad/Nic, 6.9.11 (lane 2)
</LI></UL><UL><LI> mdnABCDE from pARW089, Nad/Nic, 6.9.11 (lane 3)
</LI></UL><UL><LI> mdnABCDE+T7 from pARW089, Nad/Nic, 6.9.11 (lane 4)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> PCR purification
</LI></UL><UL><LI> digest
</LI></UL><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122);"><SPAN class="mw-headline" id="PCR_purification_of_mdnB">PCR purification of mdnB</SPAN></H3><P><B>Investigators:</B> Nadine, Nicole
</P><P><B>Aim:</B> Purification of mdnB (PCR product)
</P><P><B>Time:</B> 2011-09-06, 9:30-10:30 </P><P><B>Material: </B></P><UL><LI> PCR product of mdnB, PCR 2011-09-06, Nadja, Nicole, Jessica
</LI></UL><UL><LI> Machery-Nagel Nucleo Spin Extract II, PCR purification protocol
</LI></UL><P><B>Method: </B></P><UL><LI> done as described by manufacturer's protocol
</LI></UL><UL><LI> elution buffer
</LI></UL><UL><LI> elution volume 50 µl
</LI></UL><P><B>Results: </B></P><UL><LI> purified PCR product of mdnB
</LI></UL><UL><LI> c<SMALL>DNA</SMALL> = 44.6 ng/ µl
</LI></UL><P><B>Output: </B></P><UL><LI> stored in fridge, named: PCR product mdnB, pur., 2011-09-06
</LI></UL><P><B>Further tasks:</B></P><UL><LI> restriction enzyme digestion using SpeI and EcoRI
</LI></UL><UL><LI> Ligation in pSB1C3 (also digested with SpeI and EcoRI)
</LI></UL><UL><LI> Transformation in XL1-blue
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_purified_PCR_product_mdnB_and_pSB1C3">Digest of purified PCR product mdnB and pSB1C3</SPAN></H3><P><B>Investigators: </B>Niels</P><P><B>Materials:</B></P><UL><LI> PCR product mdnB from 2011-9-5, Nad
</LI></UL><UL><LI> pSB1C3+mdnBC (a, b, c, d, e)
</LI></UL><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> backseat driver asks: why do you digest our almost finished biobricks (pSB1C3 carrying mdnBC)? Why don't you use the vector pSB1C3 from the plasmid box?</TD></TR></TBODY></TABLE><UL><LI> EcoRI, SpeI
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> BSA
</LI></UL><UL><LI> H<SUB>2</SUB>O
</LI></UL><P><B>Protocol:</B></P><UL><LI> PCR product:
</LI></UL><UL><LI> 48 µl PCR product mdnB
</LI></UL><UL><LI> 1 µl EcoRI
</LI></UL><UL><LI> 1 µl SpeI
</LI></UL><UL><LI> 6 µl Buffer 4
</LI></UL><UL><LI> 0.6 µl BSA
</LI></UL><UL><LI> 3.4 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> total volume: 60µl
</LI></UL><UL><LI> pSB1C3+mdnBC:
</LI></UL><UL><LI> 25 µl H<SUB>2</SUB>O and DNA (see below)
</LI></UL><UL><LI> 1 µl EcoRI
</LI></UL><UL><LI> 1 µl SpeI
</LI></UL><UL><LI> 3 µl Buffer 4
</LI></UL><UL><LI> 0.3 µl BSA
</LI></UL><UL><LI> total volume: 30µl
</LI></UL><UL><LI> pSB1C3+mdnBC dilutions
</LI></UL><UL><LI> a: 9 µl vector + 16 water
</LI></UL><UL><LI> b: 6 µl vector + 19 water
</LI></UL><UL><LI> c: 6 µl vector + 19 water
</LI></UL><UL><LI> d: 6.5 µl vector + 18.5 water
</LI></UL><UL><LI> e: ´6.5 µl vector + 18.5 water
</LI></UL><UL><LI> incubation 5 hrs at 37°C
</LI></UL><P><B>Output:</B></P><UL><LI> PCR mdnB, pur., dig., Nie, 6.9.11
</LI></UL><UL><LI> pSB1C3+mdnBC a, dig., Nie, 6.9.11
</LI></UL><UL><LI> pSB1C3+mdnBC b, dig., Nie, 6.9.11
</LI></UL><UL><LI> pSB1C3+mdnBC c, dig., Nie, 6.9.11
</LI></UL><UL><LI> pSB1C3+mdnBC d, dig., Nie, 6.9.11
</LI></UL><UL><LI> pSB1C3+mdnBC e, dig., Nie, 6.9.11
</LI></UL><P><B>Further tasks:</B></P><UL><LI> purification
</LI></UL><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR:_Agarose_Gel:_mdnB_digest_verification.3B_pSBIC3_B_digest_verification_and_PCR_mdnABCDE_with_Long_Enzyme_Mix_verification">PCR: Agarose Gel: mdnB digest verification; pSBIC3_B digest verification and PCR mdnABCDE with Long Enzyme Mix verification </SPAN></H3><P><B>Time:</B> 2011-09-06</P><P><B>Investigators:</B>Nadja</P><P><B>Materials</B></P><UL><LI> digested mdnB
</LI></UL><UL><LI> digested pSBIC3_B
</LI></UL><UL><LI> PCR outcome of mdnABCDE (89, T7, 71)
</LI></UL><P><B> Production of one 0,8 % agarose gel </B></P><UL><LI> 0,8 % gel: 0.4 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to the gel
</LI></UL><P><B>Loading gels and running</B></P><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix </TD><TD> 10µl (1:100)</TD></TR><TR><TD> 1 </TD><TD> 71</TD><TD> 2+ 3H2O+ 1 6x loading dye </TD><TD> 6600
</TD></TR><TR><TD> 2 </TD><TD> T7 </TD><TD> 2+ 3H2O+ 1 6x loading dye </TD><TD> 6600
</TD></TR><TR><TD> 3 </TD><TD> 89 </TD><TD> 2+ 3H2O+ 1 6x loading dye </TD><TD> 6600
</TD></TR><TR><TD> 4 </TD><TD> mdnB</TD><TD> 10+1 6x loading dye </TD><TD> 1031
</TD></TR><TR><TD> 5 </TD><TD> pSB1C3_B </TD><TD> 30+ 6µl loading dye </TD><TD> 2390, 243
</TD></TR></TBODY></TABLE><P><B> Results:</B></P><P>Photos are already assigned to their appropriate experiments</P><P><B> Conclusions:</B></P><UL><LI> test digest seems to be correct for mdnb (line 4), also the PCR was successful (line 1,2,3), only the pSB1C3_B digest did not worked out (line 5)
</LI></UL><P><B> Further task:</B></P><UL><LI> pSB1C3_A, C,D,E digest verification
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR_purification_of_PCR_fragment_mdnABCDE_.28T7.2C_89.29_and_digested_mdnB">PCR purification of PCR fragment mdnABCDE (T7, 89) and digested mdnB</SPAN></H3><P><B>Time:</B> 2011-09-06</P><P><B>Investigators:</B> Jessica, Nicole</P><P><B>Materials</B></P><UL><LI> mdnABCDE+T7 from pARW089, Nad/Nic, 6.9.11
</LI></UL><UL><LI> mdnABCDE from pARW089, Nad/Nic, 6.9.11
</LI></UL><UL><LI>PCR mdnB, pur., dig., Nie, 6.9.11
</LI></UL><UL><LI> Macherey-Nagel Nucleo Spin Extract II kit</LI></UL><P><B> Protocol </B></P><UL><LI>Protocol for PCR clean-up
</LI></UL><UL><LI> elution with 35 µl NE buffer (incubation for 2 min at 70°C before centrifuging)
</LI></UL><P><B> Output:</B></P><UL><LI> mdnABCDE+T7 from pARW089, pur., Nadja/Jessica, 6.9.11
</LI></UL><UL><LI> mdnABCDE from pARW089, pur., Nadja/Jessica, 6.9.11
</LI></UL><UL><LI>PCR mdnB, pur., dig., pur., Nadja, 6.9.11 (concentration: 65.7 ng/µl)
</LI></UL><P>NOTE: mdnABCDE from pARW071, Nad/Nic, 6.9.11 and the library were mixed up, so PCR and fillin reaction were repeated
</P><P><B> Further task:</B></P><UL><LI> digest of mdnABCDE+T7 from pARW089, pur., Nadja/Jessica, 6.9.11 and mdnABCDE from pARW089, pur., Nadja/Jessica, 6.9.11 for ligation with pSB1C3
</LI></UL><UL><LI>ligation of PCR mdnB, pur., dig., pur., Nadja, 6.9.11 with pSB1C3
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR:_Agarose_Gel:_pSBIC3_A.2C_C.2C_D.2C_E">PCR: Agarose Gel: pSBIC3_A, C, D, E </SPAN></H3><P><B>Time:</B> 2011-09-06</P><P><B>Investigators:</B>Nadja</P><P><B>Materials</B></P><UL><LI> digested pSBIC3_ACDE
</LI></UL><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> do you mean pSB1C3+mdnBC clone a to e? please use the same label names </TD></TR></TBODY></TABLE><P><B> Production of one 0,8 % agarose gel </B></P><UL><LI> 0,8 % gel: 0.4 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to the gel
</LI></UL><P><B>Loading gels and running</B></P><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix </TD><TD> 10µl (1:100)</TD></TR><TR><TD> 1 </TD><TD> pSB1C3_A</TD><TD> 30+ 6x loading dye </TD><TD> 2390, 243
</TD></TR><TR><TD> 2 </TD><TD> pSB1C3_C</TD><TD> 30+ 6x loading dye </TD><TD> 2390, 243
</TD></TR><TR><TD> 3 </TD><TD> pSB1C3_D</TD><TD> 30+ 6x loading dye </TD><TD> 2390, 243
</TD></TR><TR><TD> 4 </TD><TD> pSB1C3_E</TD><TD> 30+ 6x loading dye </TD><TD> 2390, 243
</TD></TR></TBODY></TABLE><P><B> Results:</B></P><P><A href="/File:UP_AG_digest_pSB1C3_A_C_D_E_NB_2011-09-06.jpg" title="File:UP AG digest pSB1C3 A C D E NB 2011-09-06.jpg">300px</A></P><P><B> Conclusions:</B></P><UL><LI> worked out
</LI></UL><P><B> Further task:</B></P><UL><LI> take pSB1C3_E for gel extraction and ligate it with mdnB
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Gel_Extraction_of_pSB1C3_E">Gel Extraction of pSB1C3_E </SPAN></H3><P><B>Time:</B> 2011-09-06</P><P><B>Investigators:</B> Jessica, Nadja</P><P><B>Materials</B></P><UL><LI> pSBIC3_E
</LI></UL><UL><LI> use of Macherey Nagel Nucleo Spin Extrac II kit</LI></UL><P><B> Protocol </B></P><UL><LI> 100mg gel – 200µl NT buffer ---- incubate for 5-10min at 50 C
</LI></UL><UL><LI> load sample on colum and spin for 1min at 11000 x g
</LI></UL><UL><LI> add 700µl NT3 and spin for 1min at 11000 x g
</LI></UL><UL><LI> spin again for 2 min at 11000 x g
</LI></UL><UL><LI>eluate DNA with 35µl NE by incubating 2 min at 50 C and spinning down for 1 min at 11000 x g
</LI></UL><P><B> Results:</B></P><UL><LI> measured concentration: 24,8 ng/ µl
</LI></UL><P><B> Further task:</B></P><UL><LI> ligation with mdnB
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_pSB1C3_E_and_mdnB">Ligation of pSB1C3_E and mdnB</SPAN></H3><P><B>Time:</B> 2011-09-06</P><P><B>Investigators:</B> Jessica, Nadja</P><P><B>Materials</B></P><UL><LI> pSBIC3_E
</LI></UL><UL><LI> mdnB
</LI></UL><UL><LI> use Ligation Calculator</LI></UL><P><B> Protocol</B></P><P><B> Components</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>Component</B></TD><TD><B>Molar Ratio</B></TD><TD><B>Concentration in ng/ µl</B></TD><TD><B>Length in bp</B></TD><TD><B>Max volume</B></TD></TR><TR><TD> Backbone </TD><TD> 1 </TD><TD> 24,8</TD><TD> 2390 </TD><TD> 7
</TD></TR><TR><TD> Insert </TD><TD> 3 </TD><TD> 1031 </TD><TD> 65,7 </TD><TD>7
</TD></TR></TBODY></TABLE><P><B> Ligation mix</B></P><UL><LI> 1µl Buffer
</LI></UL><UL><LI> 1µl Enzyme
</LI></UL><UL><LI> 2,6 µl Insert
</LI></UL><UL><LI> 5,4 µl Backbone
</LI></UL><UL><LI> 0 µl water
</LI></UL><UL><LI> total volume 10 µl
</LI></UL><UL><LI> at 14 C over night
</LI></UL><P><B> Further task:</B></P><UL><LI> Transformation
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="prepare_samples_for_sequencing_.28created_pARWIII.29"> prepare samples for sequencing (created pARWIII)</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine </P><P><B> Aim: </B> sequencing to control ligation of geneIII into pARW089(GATC)</P><P><B> Material/Method:</B></P><UL><LI> purified plasmids of clones 1, 2 and 4) (70 ng/µl, 20 µl total volume)
</LI></UL><UL><LI> primer mdnA_6 (10 pmol/µl, 30 µl total volume)
</LI></UL><P><B> Further tasks: </B> control sequence
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_pSB1C3">Digestion of pSB1C3</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B></P><UL><LI> cloning of biobricks mdnA, geneIII and mdna/geneIII (fusion gene) into pSB1C3
</LI></UL><P><B>Materials/Methods:</B></P><UL><LI> 8 µl pSB1C3 (ca 2 µg)
</LI></UL><UL><LI> 2 µl NEB 10x buffer 2
</LI></UL><UL><LI> 1 µl restriction enzyme XbaI
</LI></UL><UL><LI> 1 µ restriction enzyme PstI
</LI></UL><UL><LI> 0,2 µl BSA
</LI></UL><UL><LI> 7,8 µl water
</LI></UL><UL><LI> 3 h at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> gel electrophoresis for purification
</LI></UL><UL><LI> ligation of mdnA, geneIII and mdna/geneIII (fusion gene) into pSB1C3 
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="digestion_of_vector_pARWIII">digestion of vector pARWIII </SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B> deletion of kanamycin gene in pARWIII (pARW089 containing geneIII</P><P><B>Materials/Methods:</B></P><UL><LI> 2 µg sample (3 clones of pARWIII)
</LI></UL><UL><LI> 2 µl NEB 10x buffer 3
</LI></UL><UL><LI> 1 µl restriction enzyme NsiI
</LI></UL><UL><LI> add water: total volume 20 µl
</LI></UL><UL><LI> 3 h at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> gel electrophoresis for purification
</LI></UL><UL><LI> re-ligation
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_PCR_products_geneIII_and_mdnA_for_cloning_into_pSB1C3">Digestion of PCR products geneIII and mdnA for cloning into pSB1C3</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine</P><P><B>Aim:</B> cloning of mdnA, geneIII and mdnA/geneIII into pSB1C3</P><P><B>Material/Method:</B></P><UL><LI> 30 µl PCR product mdnA
</LI></UL><UL><LI> 1 µl restriction enzyme XbaI
</LI></UL><UL><LI> 1 µl restriction enzyme PstI
</LI></UL><UL><LI> 4 µl NEB 10x buffer 2
</LI></UL><UL><LI> 0,4 µl BSA
</LI></UL><UL><LI> 3,6 µl water
</LI></UL><UL><LI> 1,5 h, 37°C
</LI></UL><UL><LI> 20 µl PCR product geneIII
</LI></UL><UL><LI> 1 µl restriction enzyme XbaI
</LI></UL><UL><LI> 1 µl restriction enzyme PstI
</LI></UL><UL><LI> 3 µl NEB 10x buffer 2
</LI></UL><UL><LI> 4,7 µl water
</LI></UL><UL><LI> 0,3 µl BSA
</LI></UL><UL><LI> 1,5 h, 37°C
</LI></UL><UL><LI> 20 µl PCR product geneIII
</LI></UL><UL><LI> 1 µl restriction enzyme NgoMIV
</LI></UL><UL><LI> 1 µl restriction enzyme PstI
</LI></UL><UL><LI> 3 µl NEB 10x buffer 1
</LI></UL><UL><LI> 0,3 µl BSA
</LI></UL><UL><LI> 4,7 µl water
</LI></UL><UL><LI> 1,5 h, 37°C
</LI></UL><UL><LI> 20 µl PCR product mdnA
</LI></UL><UL><LI> 1 µl restriction enzyme XbaI
</LI></UL><UL><LI> 1 µl restriction enzyme AgeI
</LI></UL><UL><LI> 3 µl NEB 10x buffer 1
</LI></UL><UL><LI> 4,7 µl water
</LI></UL><UL><LI> 0,3 µl BSA
</LI></UL><UL><LI> 1,5 h, 37°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gel electrophoresis for purification
</LI></UL><UL><LI> ligation of mdnA and geneIII into pSB1C3
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_for_purification_of_digested_PCR_products_and_vectors">Agarose gel electrophoresis for purification of digested PCR products and vectors</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine</P><P><B>Aim:</B> control and purification of digested PCR products mdnA and geneIII</P><P><B>Material/Method:</B></P><UL><LI> digested PCR products (mdnA and geneIII)
</LI></UL><UL><LI> digested vectors (pSB1C3 and pARWIII)
</LI></UL><UL><LI> 1 % agarose gel
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> DNA Ladder Mix (1:10) (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas)
</LI></UL><UL><LI> Gel Red
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_geneIII_and_mdnA_into_pSB1C3">Ligation of geneIII and mdnA into pSB1C3</SPAN></H3><P><B>Investigator:</B> Sabine</P><P><B>Aim:</B> pSB1C3 containing mdnA, geneIII and mdnA/geneIII </P><P><B>Material/Method:</B></P><UL><LI> 4,5 µl insert mdnA (7 ng/µ)
</LI></UL><UL><LI> 10 µl pSB1C3 (10 ng/µl)
</LI></UL><UL><LI> 2 µl T4 ligase buffer (Fermentas)
</LI></UL><UL><LI> 1 µl T4 Ligase (Fermentas)
</LI></UL><UL><LI> 2,5 µl water
</LI></UL><UL><LI> 1 h at room temperature
</LI></UL><UL><LI> 6,3 µl insert geneIII (10 µg/µl)
</LI></UL><UL><LI> 10 µl pSB1C3 (10 ng/µl)
</LI></UL><UL><LI> 1 µl T4 ligase buffer (Fermentas)
</LI></UL><UL><LI> 2 µl T4 Ligase (Fermentas)
</LI></UL><UL><LI> 0,8 µl water
</LI></UL><UL><LI> 1 h at room temperature
</LI></UL><UL><LI> 3,2 µl insert mdnA (7,6 µ/µl)
</LI></UL><UL><LI> 6,5 µl insert geneIII (7 µg/µl)
</LI></UL><UL><LI> 7,3 µl pSB1C3 (10 ng/µl)
</LI></UL><UL><LI> 2 µl T4 ligase buffer (Fermentas)
</LI></UL><UL><LI> 1 µl T4 Ligase (Fermentas)
</LI></UL><UL><LI> 1 h at room temperature
</LI></UL><P><B>Further task:</B> transformation
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Re-ligation_of_NsiI-digested_pARWIII">Re-ligation of NsiI-digested pARWIII</SPAN></H3><P><B>Investigator:</B> Sabine</P><P><B>Aim:</B> create pARW089 containing geneIII without kanamycin resistence (for library) </P><P><B>Material/Method:</B></P><UL><LI> 17 µl NsiI-digested pARWIII (clones 1, 2 and 3)
</LI></UL><UL><LI> 1 µl T4 ligase buffer (Fermentas)
</LI></UL><UL><LI> 2 µl T4 Ligase (Fermentas)
</LI></UL><UL><LI> 1 h at room temperature
</LI></UL><P><B>Further task:</B> transformation
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_created_vectors_in_E._coli">Transformation of created vectors in <I>E. coli</I></SPAN></H3><P><B>Investigator:</B> Sabine</P><P><B>Aim:</B>amplification of vectors</P><P><B>Material:</B></P><UL><LI> pARWIII without kanamycin resistence (clones 1, 2 and 3)
</LI></UL><UL><LI> pSB1C3 containing mdnA, geneIII or mdnA/geneIII
</LI></UL><UL><LI> agar plates containing chloramphenicol (pSB1C3)
</LI></UL><UL><LI> agar plates containing kanamycin, ampicillin(pPDV)
</LI></UL><UL><LI> agar plates containing kanamycin (pARW089)
</LI></UL><P><B>Method:</B></P><UL><LI> addition of 4 µl ligation reaction to XL1-blue cells</LI></UL><UL><LI> incubation 25 min on ice,</LI></UL><UL><LI> heat shock 45 sec at 42°C,</LI></UL><UL><LI> incubation 2 min on ice,</LI></UL><UL><LI> addition of 750 µl LB medium,</LI></UL><UL><LI> incubation 60 min at 37 °C and 750 rpm</LI></UL><UL><LI> plating on agar plates containing 100 µg/ml tetracyclin and 100 µg/µl ampicillin</LI></UL><UL><LI> storage over night at 37°C</LI></UL><P><B>Further tasks:</B> control cell clones
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="PCR_of_TorA-bla_with_clevage_site_of_14_3C_and_TEV_for_BioBricks">PCR of TorA-bla with clevage site of 14_3C and TEV for BioBricks</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[3]</A>]</B></P><P><B>Investigators:</B> Sascha, Stefan, Sebastian</P><P>Aim: get PCR fragment of TOR bla sequence for BioBricks</P><P><B>Methode:</B></P><P>Primer TEV:
</P><P>(1) </P><P>(2) </P><P><B>Methode:</B></P><P>PCR</P><UL><LI>Template: 1 µL (pJC354 &lt;10ng)
</LI></UL><UL><LI>Nucleotides: 1 µL of 10 mM ready to use dNTP mix</LI></UL><UL><LI>5 µL 10 x Amplification buffer S</LI></UL><UL><LI>2 µL 25 mM MgCl2</LI></UL><UL><LI>2,5 µL primers = 25 pmol absolute (2,5 µL of each primer)</LI></UL><UL><LI>35,5 µL of pure water</LI></UL><UL><LI>0,5 µL TaqPol</LI></UL><P>Program:
</P><UL><LI>Denat: 4 min 94°C
</LI></UL><UL><LI>5x:
</LI></UL><P>Denat: 1 min 94°C
</P><P>Anneal: 1 min 51°C
</P><P>Extend: 1 min 72°C
</P><UL><LI>25x:
</LI></UL><P>Denat: 1 min 94°C
</P><P>Anneal: 1 min 61°C
</P><P>Extend: 1 min 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P>Further Tasks:
</P><P>PCR purification</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR:_mdnABCDE">PCR: mdnABCDE </SPAN></H3><P><B>Time:</B> 2011-09-06</P><P><B>Investigators:</B> Nicole, Nadja</P><P><B>Materials</B></P><UL><LI> vector: pARW089 (2*)
</LI></UL><UL><LI> Long Enzyme Mix (+MgCl2) Buffer Fermentas
</LI></UL><UL><LI> Long Enzyme Polymerase Fermentas
</LI></UL><UL><LI> dNTPs (NEB)
</LI></UL><UL><LI> water
</LI></UL><UL><LI> Primer:
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>#</B></TH><TH><B>Primer</B></TH></TR><TR><TD> x </TD><TD> pf_mdnABCDE089_NotI_SpeI_30.08.
</TD></TR><TR><TD>x </TD><TD> r_mdnABCDE_iGEM
</TD></TR></TBODY></TABLE><P><B>Protocol:</B></P><UL><LI> 2 µl pARW089 (diluted 1:100)
</LI></UL><UL><LI> 1 µl dNTPs (10 mM)
</LI></UL><UL><LI> 3 µl Primer forward (10 µM)
</LI></UL><UL><LI> 3 µl Primer backward (10 µM)
</LI></UL><UL><LI> 5 µl buffer
</LI></UL><UL><LI> 0.3 µl Ploymerase
</LI></UL><UL><LI> 35,7 µl water
</LI></UL><UL><LI> total volume: 50 µl
</LI></UL><UL><LI>mdnABCDE
</LI></UL><P>2. PCR programs
</P><UL><LI>IGLONG2</LI></UL><P><B>Result</B></P><UL><LI> PCR worked out
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Cultures_of_pARW071_and_pARW089_for_HPLC">Cultures of pARW071 and pARW089 for HPLC</SPAN></H3><P><B>Time:</B> 2011-09-06</P><P><B>Investigators:</B> Nicole, Nadine, Jessica</P><P><B>Materials</B></P><UL><LI> overnight cultures of pARW089 and pARW071
</LI></UL><UL><LI> 500mM Tris-HCl, pH 7.4
</LI></UL><P><B>Protocol:</B> (following protocol from Elke)
</P><UL><LI> inoculation of 400 ml LB medium with antibiotic
</LI></UL><UL><LI> at 37°C for about 7 h
</LI></UL><UL><LI> centrifuged 4x at 6000 g for 10 min in 50 ml falcons
</LI></UL><UL><LI> washed with Tris-HCl
</LI></UL><P><B>Output:</B></P><UL><LI> 4 falcons stored in -20°C (71 1, 71 2, 89 1, 89 2)
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Overnight_cultures_of_expression_backbones">Overnight cultures of expression backbones</SPAN></H3><P><B>Time:</B> 2011-09-06</P><P><B>Investigators:</B> Jessica</P><P><B>Aim:</B> Precultures for test of expression backbones</P><P><B>Materials</B></P><UL><LI> plate of pSB1A3_Ara_YFP (from 2011-09-03)
</LI></UL><UL><LI> test cultures of pSB1A3_Lac_YFP, pSB1K3_Ara_CFP, pSB1K3_Lac_CFP (from 2011-09-06)
</LI></UL><UL><LI> LB medium, Kan, Amp
</LI></UL><P><B>Method:</B></P><UL><LI> inoculation of 10 ml LB medium with antibiotic
</LI></UL><UL><LI> overnight at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> testing expression
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="88th_Labday_2011-09-07">88th Labday 2011-09-07</SPAN></H2><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_and_Ligation_of_TorA-bla_with_cleavage_site_of_14_3C_and_TEV_for_BioBricks_and_TEV_.2F_14_3C_biobricks">Digestion and Ligation of TorA-bla with cleavage site of 14_3C and TEV for BioBricks and TEV / 14_3C biobricks</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[4]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul</P><P>Aim: purification and digestion PCR fragment (produced: 06.09.2011) of TOR bla sequence for BioBricks</P><P><B>Methode:</B></P><P>PCR-Products were purificated using Nucleospin Extract II KIT.
</P><UL><LI>The torA-bla construct was digested with AgeI and XbaI
</LI></UL><UL><LI>The vector (BBa_K404304_pSB1C3) for torA-bla was digested with AgeI and XbaI
</LI></UL><UL><LI>14_3C and Tev proteases were digested with EcoRI and SpeI
</LI></UL><UL><LI>The vector (BBa_K404304_pSB1C3) for proteases was digested with EcoRI and SpeI
</LI></UL><P>The digestion was allowed to proceed for 2h at 37°C.
</P><UL><LI>Digested products were resolved on 1% agarose gel, the corresponding bands were excissed and extracted from the gel using Nucleospin gel extraction KIT
</LI></UL><UL><LI>Expected bands:
</LI></UL><UL><LI>Tev: 760 bp (bands: Tev=2,Tev2=3,Tev4=4, Tev7=5)
</LI></UL><UL><LI>14_3C: 570 bp (bands: 14_3C1=15, 14_3C2=16)
</LI></UL><UL><LI>TorA-bla: 1001 bp (bands: Cs_TevI=6, Cs_TevII=7, Cs_TevIII=8, Cs_14_3CI=10, Cs_14_3CII=11)
</LI></UL><UL><LI>Vector: ~2632 bp (bands: Vector for proteases=14, vector for Tor_bla=17)
</LI></UL><P><A href="/File:UP_AG_digest_BB_SG_20110907.JPG" title="File:UP AG digest BB SG 20110907.JPG">400px</A><A href="/File:UP_AG_digest_BBV_SG_20110907.JPG" title="File:UP AG digest BBV SG 20110907.JPG">400px</A></P><UL><LI>The excised fragments were ligated into corresponding digested and excised vectors: Tev2, Tev3, Tev4, Tev7 and 14_3C1 and 14_3C2 into vector for proteases; the torA-bla fragments were ligated into vector for TorA-bla)
</LI></UL><UL><LI>ligation was calculated with gibthon ligation calculator (<A href="http://www.gibthon.org/ligate.html" class="external text" rel="nofollow">Klick</A>) with a molar ration of 1:3 of plasmid to insert.
</LI></UL><P>Proteases:
</P><UL><LI> 1 µl T4-ligation (Fermentas)
</LI></UL><UL><LI> 1 µl Ligase Buffer
</LI></UL><P>1.4 µl vector - 6.6 µl Tev
</P><P>1.2 µl vector - 6.8 µl Tev2
</P><P>0.7 µl vector - 7.3 µl Tev4
</P><P>1.5 µl vector - 6.5 µl Tev7
</P><P>0.3 µl vector - 7.7 µl 14_3C1
</P><P>0.4 µl vector - 7.6 µl 14_3C2
</P><UL><LI>= 6x10 µl ligation batches
</LI></UL><P>TorA-bla
</P><UL><LI> 1 µl T4-ligation (Fermentas)
</LI></UL><UL><LI> 1 µl Ligase Buffer
</LI></UL><P>2.2 µl vector - 5.8 µl Cs_14_3CI
</P><P>2.4 µl vector - 2.6 µl Cs_14_3CII
</P><P>1.5 µl vector - 6.5 µl Cs_TevI
</P><P>1.7 µl vector - 6.3 µl Cs_TevII
</P><P>2.1 µl vector - 5.9 µl Cs_TevIII
</P><UL><LI>= 5x10 µl ligation batches
</LI></UL><UL><LI>ligation was allowed to proceed for 1h at room temperature.
</LI></UL><UL><LI>2µl of each ligation batch was transformed into competent Xl1-blue cells.
</LI></UL><UL><LI>Cells were plated in Cm containing agar plates
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Oligo-Fillin_for_mdnA-Library_.28repetition.292">Oligo-Fillin for mdnA-Library (repetition)2</SPAN></H3><P><B>Time: </B>2011-09-07, 7:20-8:40</P><P><B>Investigators: </B>Nadine, Nicole</P><P><B>Aim: </B> Production of a library carrying mutated mdnA ß-lactamase-screening (carrying AatII restriction site)</P><P><B>Material: </B></P><UL><LI> oligos (25 mM, HPLC purified, ordered by Sigma Aldrich):
</LI></UL><UL><LI> o_mdnA_library, 76
</LI></UL><UL><LI> o_focused_library_2, 74
</LI></UL><UL><LI> 10x Klenow buffer
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><UL><LI>H<SMALL>2</SMALL>O
</LI></UL><P><B>Method: </B></P><UL><LI> Reaction mix (total volume: 50 µl)
</LI></UL><UL><LI> 3 µl o_mdnA-library, 76
</LI></UL><UL><LI> 3 µl o_focused_library_2, 74
</LI></UL><UL><LI> 5 µl dNTPs
</LI></UL><UL><LI> 5 µl Klenow buffer
</LI></UL><UL><LI> 34 µl H<SMALL>2</SMALL>O
</LI></UL><UL><LI> Reaction conditions
</LI></UL><UL><LI> Program: ORIGAMI1
</LI></UL><UL><LI> after finishing program ORIGAMI1:
</LI></UL><UL><LI> addition of 1 µl Klenow-fragment
</LI></UL><UL><LI> incubation 1 hour, 37°C
</LI></UL><P><B>Results: </B></P><UL><LI> hybridized oligo for production of mdnA-library
</LI></UL><P><B>Output:</B></P><UL><LI> hybridization product, named foc 2
</LI></UL><UL><LI> stored in thermocycler
</LI></UL><P><B>Further tasks:</B></P><UL><LI> PCR purification
</LI></UL><UL><LI> restriction enzyme digestion using SfoI and AatII, also of pUP089
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparing_samples_mdnC.2C_mdnD.2C_mdnE_and_mdnABC_for_sequencing">Preparing samples mdnC, mdnD, mdnE and mdnABC for sequencing </SPAN></H3><P><B>Time: </B>2011-09-07, </P><P><B>Investigators: </B>Katharina, Nadine, Nicole</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Purification_of_Fillin_mdnA-Lib2">Purification of Fillin mdnA-Lib2</SPAN></H3><P><B>Time: </B>2011-9-7, </P><P><B>Investigators: </B>Nadine, Nicole</P><P><B>Aim:</B> Purification of oligos for focused library 2</P><P><B>Materials</B></P><UL><LI> product of filled-in reaction, 2011-09-07, Nadine, Nicole
</LI></UL><UL><LI> Machery-Nagel Nucleo Spin Extract II kit
</LI></UL><P><B>Method:</B></P><UL><LI> application based on manufacturer’s protocol for PCR product purification
</LI></UL><UL><LI> exceptions for elution procedure:
</LI></UL><UL><LI> H<SMALL>2</SMALL>O used for elution
</LI></UL><UL><LI> elution volume V=25 µl
</LI></UL><UL><LI> before centrifugation (for 1 min) incubation 5 min at 70°C
</LI></UL><UL><LI> DNA concentration measured by Nanotrop
</LI></UL><P><B>Results:</B></P><UL><LI>c<SMALL>DNA</SMALL>: 113.3 ng/ µl
</LI></UL><P><B>Output:</B></P><UL><LI> purified oligos (focused library 2)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> digestion using SfoI and AatII
</LI></UL><UL><LI> digestion of pUP089 using SfoI and AatII
</LI></UL><UL><LI> Purification of both
</LI></UL><UL><LI> Ligation of pUP089 and oligos (focused library 2)
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="digest_of_purified_Fillin_mdnA-Lib2_and_pUP089">digest of purified Fillin mdnA-Lib2 and pUP089</SPAN></H3><P><B>Time: </B>2011-9-7, </P><P><B>Investigators: </B>Nadine, Nicole</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="purification_of_digested_pSB1C3_and_pUP089">purification of digested pSB1C3 and pUP089</SPAN></H3><P><B>Time: </B>2011-09-07, </P><P><B>Investigators: </B>Niels, Jessica, Katharina</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_Fillin_mdnA-Lib2_and_pUP089">Ligation of Fillin mdnA-Lib2 and pUP089</SPAN></H3><P><B>Time: </B>2011-9-7, </P><P><B>Investigators: </B>Nadja</P><P><B>Materials</B></P><UL><LI> product of digested and purified Fillin mdnA Lib2 (2011-9-7, Nicole, Nadine) = 123,0 ng/ µl
</LI></UL><UL><LI> pUP089 = 14,9 ng/ µl
</LI></UL><UL><LI> Quick Ligase Buffer
</LI></UL><P><B>Protocol:total volume of 20µl</B>+
</P><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> backseat driver asks: where is the ligation control? we talked about this several times!!!!!!!!! now we have to repeat this -.- and time designation would be also nice </TD></TR></TBODY></TABLE><UL><LI> 10 µl Quick ligase buffer
</LI></UL><UL><LI> 1µl Enzyme
</LI></UL><UL><LI>1µl Insert
</LI></UL><UL><LI> 8µl backbone
</LI></UL><UL><LI> at 16 °C in PCR machine
</LI></UL><P><B>Further tasks:</B></P><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_mdnABCDE_in_pSB1C3">Ligation of mdnABCDE in pSB1C3</SPAN></H3><P><B>Time: </B>2011-9-7, </P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_purified_PCR_product_mdnABCDE_and_pSB1C3">Digest of purified PCR product mdnABCDE and pSB1C3</SPAN></H3><P><B>Time: </B>2011-9-7, 8:30-12:15</P><P><B>Investigators: </B>Nadine, Nicole</P><P><B>Aim: </B>generate a BioBrick of mdnABCDE, digest for ligation in pSB1C3</P><P><B>Materials:</B></P><UL><LI> PCR products:
</LI></UL><UL><LI> mdnABCDE from pARW089 from 2011-9-6, Nad/Nic
</LI></UL><UL><LI> mdnABCDE+T7 pARW089 from 2011-9-6, Nad/Nic
</LI></UL><UL><LI> mdnABCDE from pARW089 or pARW071 from 2011-9-6, Nadj
</LI></UL><UL><LI> BBa_K404304_pSB1C3 (#3), 284.6 ng/µl
</LI></UL><UL><LI> EcoRI, SpeI
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> BSA
</LI></UL><UL><LI> H<SUB>2</SUB>O
</LI></UL><P><B>Protocol:</B></P><UL><LI> PCR product:
</LI></UL><UL><LI> 30 µl PCR product
</LI></UL><UL><LI> 1 µl EcoRI
</LI></UL><UL><LI> 1 µl SpeI
</LI></UL><UL><LI> 5 µl Buffer 4
</LI></UL><UL><LI> 0.6 µl BSA
</LI></UL><UL><LI> 12.5 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> total volume: 50µl
</LI></UL><UL><LI> vector:
</LI></UL><UL><LI> 5 µl vector
</LI></UL><UL><LI> 1 µl EcoRI
</LI></UL><UL><LI> 1 µl SpeI
</LI></UL><UL><LI> 2 µl Buffer 4
</LI></UL><UL><LI> 0.2 µl BSA
</LI></UL><UL><LI> 10.8 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> total volume: 20µl
</LI></UL><UL><LI> incubation 3 hrs at 37°C (start: 9:15)
</LI></UL><P><B>Output:</B></P><UL><LI> mdnABCDE from pARW089, Eco, Spe, from 2011-9-7, Nad/Nic
</LI></UL><UL><LI> mdnABCDE+T7 pARW089, Eco, Spe from 2011-9-7, Nad/Nic
</LI></UL><UL><LI> mdnABCDE from pARW089, Eco, Spe or pARW071 from 2011-9-7, Nad/Nic
</LI></UL><UL><LI> BBa_K404304_pSB1C3, Eco, Spe, from 2011-9-7, Nad/Nic
</LI></UL><P><B>Further tasks:</B></P><UL><LI> agarose gel
</LI></UL><UL><LI> purification
</LI></UL><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="test_digest_of_mdnBC">test digest of mdnBC </SPAN></H3><P><B>Time:</B> 2011-9-7,10:30-</P><P><B>Investigators:</B> Nadine, Nicole </P><P><B> Aim: </B> prove of Insert (mdnBC)</P><P><B>Materials:</B></P><UL><LI> mdnBC klon a-e, Nie, 6.9.11
</LI></UL><UL><LI> HindIII, AvaI, (NEB)
</LI></UL><UL><LI> Buffer 2 (NEB, 10x)
</LI></UL><UL><LI> water
</LI></UL><P><B>Plan:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>insert</B></TD><TD><B>plasmid size in bp</B></TD><TD><B>enzymes</B></TD><TD><B>buffer</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> mdnBC </TD><TD> 4419 </TD><TD> HindIII, AvaI </TD><TD> 2 </TD><TD> 363, 836, 3220
</TD></TR></TBODY></TABLE><PRE><B> Digestion protocol </B></PRE><UL><LI>1 µl DNA
</LI></UL><UL><LI>0.5 µl HinIII
</LI></UL><UL><LI>0.5 µl AvaI
</LI></UL><UL><LI>2 µl 10x buffer
</LI></UL><UL><LI>16 µl pure water
</LI></UL><UL><LI>total: 20µl
</LI></UL><UL><LI>37°C for 2 hrs
</LI></UL><P><B>Conclusions: </B></P><P><B> Further tasks: </B></P><UL><LI> agarose gel
</LI></UL><UL><LI> sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="test_pSB1A3_Ara_YFP_and_pSB1A3_Lac_YFP_.28expression_backbones.29">test pSB1A3_Ara_YFP and pSB1A3_Lac_YFP (expression backbones) </SPAN></H3><P><B>Time:</B> 2011-9-7,12</P><P><B>Investigators:</B> Nadine, Nicole, Jessica </P><P><B>Method:</B></P><UL><LI> pSB1A3_Lac_YFP:
</LI></UL><UL><LI>ODs at 16:45: 0.095 (1) and 0.082 (2, control)
</LI></UL><UL><LI> no induction
</LI></UL><UL><LI> pSB1A3_Ara_YFP:
</LI></UL><UL><LI>ODs at 17:30 : 0.657 (1) and 0.713 (2, control)
</LI></UL><UL><LI> induction with arabinose, end concentration 5 mM
</LI></UL><P><B>Results:</B></P><UL><LI> fluorescence didn't increase, curve flattened
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR:_mdnABCDE_2">PCR: mdnABCDE </SPAN></H3><P><B>Time:</B> 2011-09-06</P><P><B>Investigators:</B> Nadja</P><P><B>Materials</B></P><UL><LI> vector: pARW071 (1)
</LI></UL><UL><LI> Long Enzyme Mix (+MgCl2) Buffer Fermentas
</LI></UL><UL><LI> Long Enzyme Mix Polymerase Fermentas
</LI></UL><UL><LI> dNTPs (NEB)
</LI></UL><UL><LI> water
</LI></UL><UL><LI> Primer:
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>#</B></TH><TH><B>Primer</B></TH></TR><TR><TD>84</TD><TD> pf_mdnABCDE089_NotI_SpeI_30.08.
</TD></TR><TR><TD>82</TD><TD> r_mdnABCDE_iGEM
</TD></TR></TBODY></TABLE><P><B>Protocol:</B></P><UL><LI> 2,6 µl pARW08 (diluted 1:100)
</LI></UL><UL><LI> 1 µl dNTPs (10 mM)
</LI></UL><UL><LI> 3 µl Primer forward (10 µM)
</LI></UL><UL><LI> 3 µl Primer backward (10 µM)
</LI></UL><UL><LI> 5 µl buffer
</LI></UL><UL><LI> 0.3 µl Ploymerase
</LI></UL><UL><LI> 35,4 µl water
</LI></UL><UL><LI> total volume: 50 µl
</LI></UL><UL><LI>mdnABCDE
</LI></UL><P>2. PCR programs
</P><UL><LI>IGLONG2</LI></UL><P><B>Further task</B></P><UL><LI> gel verification
</LI></UL><UL><LI> purification
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Overnight_cultures_of_expression_backbones_and_pUP089">Overnight cultures of expression backbones and pUP089</SPAN></H3><P><B>Time:</B> 2011-09-07, 18:00 </P><P><B>Investigators:</B> Niels</P><P><B>Aim:</B> Precultures for test of expression backbones and glycerol stocks</P><P><B>Materials</B></P><UL><LI> plates of pSB1A3_Ara_YFP, pSB1A3_Lac_YFP, pSB1K3_Ara_CFP, pSB1K3_Lac_CFP (from 2011-09-03)
</LI></UL><UL><LI> plates of pUP089 A and B
</LI></UL><UL><LI> LB medium, Kan, Amp
</LI></UL><P><B>Method:</B></P><UL><LI> inoculation of 10/5 ml LB medium with antibiotic
</LI></UL><UL><LI> overnight at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> testing expression
</LI></UL><UL><LI><BIG><B>glycerol stocks !!!!!!!</B></BIG></LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_test_digest_of_mdnBC">Repeated test digest of mdnBC </SPAN></H3><P><B>Time:</B> 2011-9-7, 20:00</P><P><B>Investigators:</B> Jessica, Niels, Nadja </P><P><B> Aim: </B> prove of Insert (mdnBC)</P><P><B>Materials:</B></P><UL><LI> mdnBC clone b, Nie, 6.9.11
</LI></UL><UL><LI> HindIII, AvaI, PstI-HF, XmaI (NEB)
</LI></UL><UL><LI> Buffer 2 and 4 (NEB, 10x)
</LI></UL><UL><LI> water
</LI></UL><P><B>Plan:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>insert</B></TD><TD><B>plasmid size in bp</B></TD><TD><B>enzymes</B></TD><TD><B>buffer</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> mdnBC </TD><TD> 4419 </TD><TD> HindIII, AvaI </TD><TD> 2 </TD><TD> 363, 836, 3220
</TD></TR><TR><TD> mdnBC </TD><TD> 4419 </TD><TD> PstI-HF, XmaI </TD><TD> 4 </TD><TD> 1559, 2868
</TD></TR></TBODY></TABLE><P><B> Digestion protocol </B></P><UL><LI>1 µl DNA
</LI></UL><UL><LI>0.5 µl per enzyme
</LI></UL><UL><LI>2 µl 10x buffer
</LI></UL><UL><LI>16 µl pure water
</LI></UL><UL><LI>total: 20µl
</LI></UL><UL><LI>37°C for overnight (on thermomixer in iGEM lab)
</LI></UL><P><B>Agarose gel:</B></P><P><B>Conclusions: </B></P><P><B> Further tasks: </B></P><UL><LI> sequencing
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Production_of_phages_containing_pPDV089_in_ER2738_cells">Production of phages containing pPDV089 in ER2738 cells</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina, Steffi</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI><B>first step</B>: amplification of cells containing pPDV089 (clone: 2S14):
</LI></UL><UL><LI> 50 ml DYT medium will be inoculated with the cells, so that OD600 = 0.1
</LI></UL><UL><LI> add antibiotics tetracyclin and ampicillin to the medium
</LI></UL><UL><LI> cells should be incubated 37 °C till OD600 = 0.3-0.5 (here: 0.356) is reached
</LI></UL><UL><LI><B>second step</B>: infection with helper phages
</LI></UL><UL><LI> add helper phages 10^11 phages/50 ml (3,5 µl)
</LI></UL><UL><LI> incubate for 10 min at 37°C (without shaking!)
</LI></UL><UL><LI> add 0,5 mM IPTG
</LI></UL><UL><LI> incubate 50 min at 28°C and rpm
</LI></UL><UL><LI> add 70 µg/ml kanamycin and incubate for 5 h at 28°C (...)
</LI></UL><UL><LI><B>third step</B>: phage purification
</LI></UL><UL><LI> centrifuge cell culture at 5000 x g/ 15 min
</LI></UL><UL><LI> fill supernatant in a new 50 ml falcon and centrifuge again (5000 g/15 min)
</LI></UL><UL><LI> 40 ml of the supernatant with 8 ml PEG-NaCl (20% (w/v) PEG-8000, 2,5 M NaCl)
</LI></UL><UL><LI> incubate over night at 4°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> go on with phage purification</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="SDS-_PAGE_for_Western_Blotting_and_coomassie_staining_of_purified_phages">SDS- PAGE for Western Blotting and coomassie staining of purified phages</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina, Sebastian</P><P><B>Aim:</B> control if geneIII-mdnA is expressed on the phages produced in XL1-blue cells</P><P><B>Method/Materials:</B></P><P><B>Further tasks:</B></P><UL><LI> Western Blot and coomassie staining </LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Western_Blot">Western Blot</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA is expressed on the phages produced in XL1-blue cells</P><P><B>Method/Materials:</B></P><UL><LI> membrane impregnated in methanol
</LI></UL><UL><LI> whatman paper
</LI></UL><UL><LI> blotting buffer
</LI></UL><UL><LI> blotting chamber
</LI></UL><UL><LI> 1 h at 200 mA
</LI></UL><UL><LI> blocking over night in TBS with 5% milk powder
</LI></UL><P><B>Further tasks:</B></P><UL><LI> antibody detection </LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Coomassie_staining">Coomassie staining </SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA is expressed on the phages produced in XL1-blue cells</P><P><B>Method/Materials:</B></P><UL><LI> 20 ml water
</LI></UL><UL><LI> 20 ml methanol
</LI></UL><UL><LI> 60 ml coomassie stock solution
</LI></UL><UL><LI> gel storage over night in coomassie solution shaking </LI></UL><P><B>Further tasks:</B></P><UL><LI> destaining of the gel </LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="overnight_culture_of_picked_E._coli_clones_transformed_with_pARWIII_.28pARW089_containing_geneIII.29_to_control_deletion_of_kanamycin_gene.2C_pSB1C3_containing_mdnA.2C_geneIII_or_mdnA.2FgeneIII">overnight culture of picked <I>E. coli</I> clones transformed with pARWIII (pARW089 containing geneIII) to control deletion of kanamycin gene, pSB1C3 containing mdnA, geneIII or mdnA/geneIII </SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B> control deletion of kanamycin gene in created pARWIII, control ligation of mdnA, geneIII and mdnA/geneIII into pSB1C3</P><P><B>Method/Materials:</B></P><UL><LI> 5 clones from pARW089 with geneIII (kanamycin)
</LI></UL><UL><LI> 5 clones from pSB1C3 with mdnA (chloramphenicol)
</LI></UL><UL><LI> 5 clones from pSB1C3 with geneIII (chloramphenicol)
</LI></UL><UL><LI> 5 clones from pSB1C3 with mdnA/geneIII (chloramphenicol)
</LI></UL><UL><LI> 5 ml LB medium per clone
</LI></UL><UL><LI> storage over night at 37°C and 800 rpm</LI></UL><P><B>Further tasks:</B></P><UL><LI> test digestion</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="89th_Labday_2011-09-08">89th Labday 2011-09-08</SPAN></H2><H3 style="background-color: rgb(254, 122, 122);"><SPAN class="mw-headline" id="Oligo-Fillin_for_mdnA-library_2_.28repetition.29_and_for_Phage-display-mdnA-library_.28carrying_AgeI_restriction_site.29">Oligo-Fillin for mdnA-library 2 (repetition) and for Phage-display-mdnA-library (carrying AgeI restriction site)</SPAN></H3><P><B>Investigators:</B> Nicole, Nadine
</P><P><B>Time:</B> 2011-09-08, 11:00-12:30</P><P><B>Aim: </B> Production of libraries carrying mutated mdnA, one for phage-display (carrying ageI restriction site) and one for ß-lactamase-screening (carrying AatII restriction site)</P><P><B>Material: </B></P><UL><LI> oligos (25 mM, HPLC purified, ordered by Sigma Aldrich):
</LI></UL><UL><LI> o_mdnA_library, 76 à for both libraries
</LI></UL><UL><LI> o_focused_library_2, 74 à for AatII carrying library
</LI></UL><UL><LI> o_focused_library_AgeI, à for AgeI carrying library
</LI></UL><UL><LI> 10x Klenow buffer
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><UL><LI>H<SMALL>2</SMALL>O
</LI></UL><P><B>Method: </B></P><UL><LI> Reaction mix (total volume: 50 µl)
</LI></UL><UL><LI> 3 µl fw-oligonucleotide (o_mdnA-library, 76)
</LI></UL><UL><LI> 3 µl rev-oligonucleotide (either o_focused_library_2, 74 or o_focused_library_AgeI, )
</LI></UL><UL><LI> 5 µl dNTPs
</LI></UL><UL><LI> 5 µl Klenow buffer
</LI></UL><UL><LI> 34 µl H<SMALL>2</SMALL>O
</LI></UL><UL><LI> Reaction conditions
</LI></UL><UL><LI> Program: ORIGAMI1
</LI></UL><UL><LI> after finishing program ORIGAMI1:
</LI></UL><UL><LI> addition of 1 µl Klenow-fragment
</LI></UL><UL><LI> incubation 1 hour, 37°C
</LI></UL><P><B>Results: </B></P><UL><LI> hybridized oligos for production of mdnA-library
</LI></UL><P><B>Output:</B></P><UL><LI> two hybridization products
</LI></UL><UL><LI> 1) AgeI-library, named AgeI
</LI></UL><UL><LI> 2) AatII-library, named foc 2
</LI></UL><UL><LI> stored in thermocycler
</LI></UL><P><B>Further tasks:</B></P><UL><LI> PCR purification
</LI></UL><UL><LI> restriction enzyme digestion using SfoI and either AatII or AgeI, also of either pUP089 or pARWIII
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_XL1_with_different_constructs">Transformation of XL1 with different constructs</SPAN></H3><P><B>Time:</B> 2011-09-08, 12:30-15:00</P><P><B>Investigators:</B> Nadine, Nicole </P><P><B>Materials:</B></P><UL><LI> XL1-Blue
</LI></UL><UL><LI> ligation products:
</LI></UL><UL><LI> pSB1C3+mdnABCDE+T7, Niels, 7.9.11, resistance: cm
</LI></UL><UL><LI> pSB1C3+mdnABCDE from pARW089, Niels, 7.9.11, resistance: cm
</LI></UL><UL><LI> pSB1C3+mdnB, Jes/Nadja, 6.9.11, resistance: cm
</LI></UL><UL><LI> pSB1C3+H20 (control), Jes/Nadja, 6.9.11, resistance: cm
</LI></UL><UL><LI> pUP089+lib2, Nadja, 7.9.11, resistance: kan
</LI></UL><UL><LI> plasmids from miniprep (Kat, 2011-8-20):
</LI></UL><UL><LI> pSB1A3_Ara_YFP clone A (#1), resistance: amp
</LI></UL><UL><LI> pSB1A3_Lac_YFP clone B (#4), resistance: amp
</LI></UL><UL><LI> pSB1K3_Ara_CFP clone A (#25), resistance: kan
</LI></UL><UL><LI> pSB1K3_Lac_CFP clone B (#21), resistance: kan
</LI></UL><UL><LI> pSB1K3_Lac_YFP clone C (#18), resistance: kan </LI></UL><UL><LI> antibiotics from stock:
</LI></UL><UL><LI> kanamycin
</LI></UL><UL><LI> ampiciline
</LI></UL><UL><LI> chloramphenicol
</LI></UL><UL><LI> LB-Agar
</LI></UL><P><B>Method:</B></P><UL><LI> ligation products:
</LI></UL><UL><LI> addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI> incubation 30 min on ice,
</LI></UL><UL><LI> heat shock 60 sec at 42°C,
</LI></UL><UL><LI> incubation 5 min on ice,
</LI></UL><UL><LI> addition of 750 µl LB medium,
</LI></UL><UL><LI> incubation at 37 °C shaking for 60 min,
</LI></UL><UL><LI> plating on LB medium with appropriate antibiotic (20 ml Agar and 20 µl antibiotic (see above) per plate)
</LI></UL><UL><LI> storage over night at 37°C (start: 15:00)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture (except for library 2)
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><UL><LI> count colonies of pUP089_lib2 and collect them w/ LB
</LI></UL><P><B>Output:</B></P><UL><LI>Plates w/ XL1 blue:
</LI></UL><UL><LI> pSB1C3+mdnABCDE+T7, Niels, 7.9.11, resistance: cm
</LI></UL><UL><LI> pSB1C3+mdnABCDE from pARW089, Niels, 7.9.11, resistance: cm
</LI></UL><UL><LI> pSB1C3+mdnB, Jes/Nadja, 6.9.11, resistance: cm
</LI></UL><UL><LI> pSB1C3+H20 (control), Jes/Nadja, 6.9.11, resistance: cm
</LI></UL><UL><LI> 10x pUP089+lib2, Nadja, 7.9.11, resistance: kan
</LI></UL><UL><LI> pSB1A3_Ara_YFP clone A (#1), resistance: amp
</LI></UL><UL><LI> pSB1A3_Lac_YFP clone B (#4), resistance: amp
</LI></UL><UL><LI> pSB1K3_Ara_CFP clone A (#25), resistance: kan
</LI></UL><UL><LI> pSB1K3_Lac_CFP clone B (#21), resistance: kan
</LI></UL><UL><LI> pSB1K3_Lac_YFP clone C (#18), resistance: kan
</LI></UL><UL><LI> Ligation products stored in freezer, green box, mdn-BioBricks
</LI></UL><UL><LI> pSB1C3+mdnABCDE+T7, pARW089, 7.9.11
</LI></UL><UL><LI> pSB1C3+mdnABCDE, pARW089, 7.9.11
</LI></UL><UL><LI> pSB1C3+mdnB, Nadja/Jes, 6.9.11
</LI></UL><UL><LI> pSB1C3+H20, Nadja/Jes, 6.9.11
</LI></UL><UL><LI>Expresion backbones stored in white/colorless box w/ label: Expression Backbones 2
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Purification_of_Fillin_mdnA-Library_2_.28carrying_either_AgeI_or_AatII_restriction_site.29_and_PCR_product_of_mdnABCDE_from_pARW071">Purification of Fillin mdnA-Library_2 (carrying either AgeI or AatII restriction site) and PCR product of mdnABCDE from pARW071</SPAN></H3><P><B>Time: </B>2011-9-8, </P><P><B>Investigators: </B>Nadine, Nicole</P><P><B>Aim:</B> Purification of oligos for focused library 2 for phage display and ß-lactamase screening and purification of PCR product mdnABCDE from pARW071 for further use</P><P><B>Materials</B></P><UL><LI> product of filled-in reaction, 2011-09-08, Nadine, Nicole
</LI></UL><UL><LI> Age (mdnA library for phage display)
</LI></UL><UL><LI> foc 2 (mdnA library for ß-lactamase screening)
</LI></UL><UL><LI> PCR product of mdnABCDE from pARW071 (named mdnA71), 2011-09-07, Nadja
</LI></UL><UL><LI> Machery-Nagel Nucleo Spin Extract II kit
</LI></UL><P><B>Method:</B></P><UL><LI> application based on manufacturer’s protocol for PCR product purification
</LI></UL><UL><LI> exceptions for elution procedure:
</LI></UL><UL><LI> H<SMALL>2</SMALL>O used for elution
</LI></UL><UL><LI> elution volume V=25 µl
</LI></UL><UL><LI> before centrifugation (for 1 min) incubation 5 min at 70°C
</LI></UL><UL><LI> DNA concentration measured by Nanotrop
</LI></UL><P><B>Output:</B></P><UL><LI> purified oligos
</LI></UL><UL><LI> mdnA focused library 2 (carrying AatII restriction site), named Age, stored in freezer
</LI></UL><UL><LI> mdnA focused library 2 (carrying AgeI restriction site), named foc 2, stored in freezer
</LI></UL><UL><LI> mdnABCDE from pARW071, named mdnA71, stored in freezer
</LI></UL><P><B>Further tasks:</B></P><UL><LI> digestion of foc 2 using SfoI and AatII
</LI></UL><UL><LI> digestion of pUP089 using SfoI and AatII
</LI></UL><UL><LI> digestion of Age using SfoI and AgeI
</LI></UL><UL><LI> digestion of pARWIII using SfoI and AgeI
</LI></UL><UL><LI> digestion of mdnA71 using EcoRI and SpeI
</LI></UL><UL><LI> digestion of pSB1C3 using EcoRI and SpeI
</LI></UL><H3 style="background-color: rgb(254, 122, 122);"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_mdnA_focused_library_2_carrying_AatII_restriction_site_and_pUP089">Restriction enzyme digestion of mdnA focused library 2 carrying AatII restriction site and pUP089</SPAN></H3><P><B>Investigators: Nicole, Nadine </B></P><P><B>Time: 2011-09-08,</B></P><P><B>Aim: </B> Digest of hybridized oligos for further production of mdnA focused library</P><P><B>Material: </B></P><UL><LI> clones
</LI></UL><UL><LI> purified mdnA oligos carrying AatII restriction site, named foc 2, 2011-09-08, Nicole/Nadine
</LI></UL><UL><LI> pUP089, clone A I, Nadine
</LI></UL><UL><LI> NEB buffer 4
</LI></UL><UL><LI> AatII
</LI></UL><UL><LI> SfoI
</LI></UL><P><B>Method: </B></P><UL><LI> Mix for digestion of foc 2
</LI></UL><UL><LI> 23 µl PCR purification product foc 2 (mdnA oligos)
</LI></UL><UL><LI> 2 µl AatII
</LI></UL><UL><LI> 2 µl SfoI
</LI></UL><UL><LI> 3 µl NEB buffer 4
</LI></UL><UL><LI> Mix for digestion of pUP089
</LI></UL><UL><LI> 4 µl DNA pUP089
</LI></UL><UL><LI> 2 µl AatII
</LI></UL><UL><LI> 2 µl SfoI
</LI></UL><UL><LI> 2 µl NEB buffer 4
</LI></UL><UL><LI> 10 µl H<SMALL>2</SMALL>O
</LI></UL><UL><LI> Incubation for 2 hours, 37 °C
</LI></UL><P><B>Results: </B></P><UL><LI> digested oligos for library
</LI></UL><UL><LI> digested pUP089
</LI></UL><P><B>Output:</B></P><UL><LI> digested oligos for mdnA library (carrying AatII restriction site), named foc 2, stored in ice box
</LI></UL><UL><LI> digested pUP089, named pUP089, stored in ice box
</LI></UL><P><B>Further tasks:</B></P><UL><LI> PCR purification (foc 2) resp. agrose gel and gel extraction (pUP089)
</LI></UL><UL><LI> ligation of both
</LI></UL><H3 style="background-color: rgb(254, 122, 122);"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_mdnABCDE71_and_pSB1C3">Restriction enzyme digestion of mdnABCDE71 and pSB1C3</SPAN></H3><P><B>Investigators: Nicole, Nadine </B></P><P><B>Time: 2011-09-08,</B></P><P><B>Aim: </B> Digest of PCR product mdnABCDE from pARW071 and pSB1C3 to prepare a biobrick</P><P><B>Material: </B></P><UL><LI> clones
</LI></UL><UL><LI> purified PCR product mdnABCDE, named mdnABCDE71, 2011-09-08, Nicole/Nadine
</LI></UL><UL><LI> pSB1C3,
</LI></UL><UL><LI> NEB buffer 4
</LI></UL><UL><LI> EcoRI-HF
</LI></UL><UL><LI> SpeI
</LI></UL><UL><LI> BSA
</LI></UL><P><B>Method: </B></P><UL><LI> Mix for digestion of mdnABCDE from pARW071 (total volume 50 µl)
</LI></UL><UL><LI> 23 µl PCR purification product foc 2 (mdnA oligos)
</LI></UL><UL><LI> 1 µl EcoRI-HF
</LI></UL><UL><LI> 1 µl SpeI
</LI></UL><UL><LI> 5 µl NEB buffer 4
</LI></UL><UL><LI> 0.6 µl BSA
</LI></UL><UL><LI> 19.5 H<SMALL>2</SMALL>O
</LI></UL><UL><LI> Mix for digestion of pSB1C3 (total volume 20 µl)
</LI></UL><UL><LI> 5.8 µl DNA pSB1C3
</LI></UL><UL><LI> 1 µl EcoRI-HF
</LI></UL><UL><LI>1 µl SpeI
</LI></UL><UL><LI> 2 µl NEB buffer 4
</LI></UL><UL><LI> 10 µl H<SMALL>2</SMALL>O
</LI></UL><UL><LI> 0.2 µl BSA
</LI></UL><UL><LI> Incubation for 2 hours, 37 °C
</LI></UL><P><B>Results: </B></P><UL><LI> digested oligos for library
</LI></UL><UL><LI> digested pARWIII
</LI></UL><P><B>Output:</B></P><UL><LI> digested mdnABCDE from pARW071, named digested mdnABCDE71, stored in ice box
</LI></UL><UL><LI> digested pSB1C3, named digested pSB1C3, stored in ice box
</LI></UL><P><B>Further tasks:</B></P><UL><LI> PCR purification (mdnABCDE71) resp. agrose gel and gel extraction (pSB1C3)
</LI></UL><UL><LI> ligation of both
</LI></UL><H3 style="background-color: rgb(254, 122, 122);"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_mdnA_focused_library_2_carrying_AgeI_restriction_site_and_pARWIII">Restriction enzyme digestion of mdnA focused library 2 carrying AgeI restriction site and pARWIII</SPAN></H3><P><B>Investigators: Nicole, Nadine </B></P><P><B>Time: 2011-09-08,</B></P><P><B>Aim: </B> Digest of hybridized oligos for further production of mdnA focused library for phage-display</P><P><B>Material: </B></P><UL><LI> clones
</LI></UL><UL><LI> purified mdnA oligos carrying AgeI restriction site, named Age, 2011-09-08, Nicole/Nadine
</LI></UL><UL><LI> pARWIII, clone 1_1, Sandrina, 2011-09-08
</LI></UL><UL><LI> NEB buffer 4
</LI></UL><UL><LI> AgeI
</LI></UL><UL><LI> SfoI
</LI></UL><P><B>Method: </B></P><UL><LI> Mix for digestion of Age
</LI></UL><UL><LI> 22.5 µl PCR purification product foc 2 (mdnA oligos)
</LI></UL><UL><LI> 2.5 µl AgeI
</LI></UL><UL><LI> 2 µl SfoI
</LI></UL><UL><LI> 3 µl NEB buffer 4
</LI></UL><UL><LI> Mix for digestion of pARWIII
</LI></UL><UL><LI> 6.5 µl DNA pARWIII
</LI></UL><UL><LI> 2.5 µl AgeI
</LI></UL><UL><LI> 2 µl SfoI
</LI></UL><UL><LI> 3 µl NEB buffer 4
</LI></UL><UL><LI> 6 µl H<SMALL>2</SMALL>O
</LI></UL><UL><LI> Incubation for 2 hours, 37 °C
</LI></UL><P><B>Results: </B></P><UL><LI> digested oligos for library
</LI></UL><UL><LI> digested pARWIII
</LI></UL><P><B>Output:</B></P><UL><LI> digested oligos for mdnA library (carrying AgeI restriction site), named Age, stored in ice box
</LI></UL><UL><LI> digested pARWIII, named digested-pARWIII, stored in ice box
</LI></UL><P><B>Further tasks:</B></P><UL><LI> PCR purification (Age) resp. agrose gel and gel extraction (pARWIII)
</LI></UL><UL><LI> ligation of both
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="PCR_of_TorA-bla_with_clevage_site_of_14_3C_and_TEV_for_BioBricks_2">PCR of TorA-bla with clevage site of 14_3C and TEV for BioBricks</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[5]</A>]</B></P><P><B>Investigators:</B> Sascha, Stefan, Sebastian</P><P>Aim: get PCR fragment of TOR bla sequence for BioBricks</P><P><B>Methode:</B></P><P>Primer TEV:
</P><P>(1) </P><P>(2) </P><P><B>Methode:</B></P><P>PCR</P><UL><LI>Template: 1 µL (pJC354 &lt;10ng)
</LI></UL><UL><LI>Nucleotides: 1 µL of 10 mM ready to use dNTP mix</LI></UL><UL><LI>10 µL 5 x Amplification buffer </LI></UL><UL><LI>2,5 µL primers = 25 pmol absolute (2,5 µL of each primer)</LI></UL><UL><LI>35 µL of pure water</LI></UL><UL><LI>0,5 µL Fusion</LI></UL><P>Program:
</P><UL><LI>Denat: 4 min 94°C
</LI></UL><UL><LI>30x:
</LI></UL><P>Denat: 30 sec 94°C
</P><P>Anneal: 30 sec 72°C
</P><P>Extend: 1 min 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P>Further Tasks:
</P><P>PCR purification</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Test_digest_of_pSB1C3.2BmdnDE">Test digest of pSB1C3+mdnDE </SPAN></H3><P><B>Time:</B> 2011-9-08, 17:30-20:30</P><P><B>Investigators:</B> Jessica </P><P><B> Aim: </B> prove of Insert (mdnDE)</P><P><B>Materials:</B></P><UL><LI> pSB1C3+mdnBC clones K1-K6, Kat, 8.9.11
</LI></UL><UL><LI> concentrations:
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>Sample</B></TD><TD><B>Concentration in ng/µl</B></TD></TR><TR><TD> clone K1 </TD><TD> 835.1
</TD></TR><TR><TD> clone K2 </TD><TD> 775.6
</TD></TR><TR><TD> clone K3 </TD><TD> 488.1
</TD></TR><TR><TD> clone K4 </TD><TD> 639.3
</TD></TR><TR><TD> clone K5 </TD><TD> 554.3
</TD></TR><TR><TD> clone K6 </TD><TD> 551.5
</TD></TR></TBODY></TABLE><UL><LI> HpaI (NEB)
</LI></UL><UL><LI> Buffer 4 (NEB, 10x)
</LI></UL><UL><LI> water
</LI></UL><P><B>Plan:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>insert</B></TD><TD><B>plasmid size in bp</B></TD><TD><B>enzymes</B></TD><TD><B>buffer</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> mdnDE </TD><TD> 5154 </TD><TD> HpaI </TD><TD> 4 </TD><TD> 1329, 3825
</TD></TR></TBODY></TABLE><P><B> Digestion protocol </B></P><UL><LI>1 µl DNA
</LI></UL><UL><LI>0.5 µl HpaI
</LI></UL><UL><LI>2 µl 10x buffer
</LI></UL><UL><LI>16 µl pure water
</LI></UL><UL><LI>total: 20µl
</LI></UL><UL><LI>37°C for </LI></UL><P><B>Agarose gel</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix (1:10) </TD><TD> 20µl
</TD></TR><TR><TD> 1 </TD><TD> - </TD></TR><TR><TD> 2 </TD><TD> clone K1 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 3 </TD><TD> clone K2 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 4 </TD><TD> clone K3 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 5 </TD><TD> clone K4 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 6 </TD><TD> clone K5 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 7 </TD><TD> clone K6 </TD><TD> 20 + 4x loading dye
</TD></TR></TBODY></TABLE><P><A href="/File:UP_AG_digest_C3mdnDE_2011-09-08_JE.jpg%3F" title="File:UP AG digest C3mdnDE 2011-09-08 JE.jpg?">250px</A></P><P><B>Conclusions: </B></P><UL><LI> clones K1, K2 and K5 are ok
</LI></UL><P><B> Further tasks: </B></P><UL><LI> sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Overnight_cultures_of_pSB1C3.2Bmyc">Overnight cultures of pSB1C3+myc</SPAN></H3><P><B>Time:</B> 2011-09-08, 18:30</P><P><B>Investigators:</B> Jessica</P><P><B>Materials</B></P><UL><LI> plates of pSB1C3+mycN and pSB1C3+mycC
</LI></UL><UL><LI> LB medium, Cm
</LI></UL><P><B>Method:</B></P><UL><LI> inoculation of 5 ml LB medium with Cm
</LI></UL><UL><LI> overnight at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> test digest
</LI></UL><UL><LI> sequencing
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Sidedirected_mutagenesis_of_TEV_protease_and_14_3C_protease">Sidedirected mutagenesis of TEV protease and 14_3C protease</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[6]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P>Aim: Sidedirected mutagenesis of TEV and 14_3C protease</P><P><B>Materials:</B></P><UL><LI> TEV protease vector (P9 by Gunther Stier)
</LI></UL><UL><LI> 14_3C protease vector (P10 by Gunther Stier)
</LI></UL><UL><LI> different Primers
</LI></UL><UL><LI> Phusion polymerase kit by NEB
</LI></UL><P><B>Methode:</B></P><P>PCR</P><UL><LI> Template: 1 µL (pJC354 &lt;10ng)
</LI></UL><UL><LI> Nucleotides: 1 µL of 10 mM ready to use dNTP mix</LI></UL><UL><LI> 10 µL 5 x Phusion HF Buffer</LI></UL><UL><LI> 2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)</LI></UL><UL><LI> 32,5 µL of pure water</LI></UL><UL><LI>0,5 µL Phusion HF polymerase</LI></UL><P>Program:
</P><UL><LI>Denat: 30 sec 98°C
</LI></UL><UL><LI>30x
</LI></UL><P>Denat: 10 sec 98°C
</P><P>Anneal: 15 sec 69°C (with gradient of +/- 4°C)
</P><P>Extend: 15 sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P>Used Primers:
</P><P>Fragment 1 of TEV protease (T1):
</P><UL><LI> f_TEV_AraFusion_NgoMIV
</LI></UL><UL><LI> r_TEV_ACCAGC
</LI></UL><P>Fragment 2 of TEV protease (T2):
</P><UL><LI> f_TEV_ACCAGC
</LI></UL><UL><LI> r_TEV_iGEM_BamHI
</LI></UL><P>Fragment 1 of 14_3C protease (P1):
</P><UL><LI> f_14_3C_AraFusion_NgoMIV
</LI></UL><UL><LI> r_14_3C_ACCAGC
</LI></UL><P>Fragment 2 of 14_3C protease (P2):
</P><UL><LI> f_14_3C_ACCAGC
</LI></UL><UL><LI> r_14_3C_tm_Xba208_A-T
</LI></UL><P>Fragment 3 of 14_3C protease (P3):
</P><UL><LI> f_14_3C_tm_Xba208_A-T
</LI></UL><UL><LI> r_14_3C_tm_Xba280_A-T
</LI></UL><P>Fragment 4 of 14_3C protease (P4):
</P><UL><LI> f_14_3C_tm_Xba280_A-T
</LI></UL><UL><LI> r_14_3C_iGEM_BamHI
</LI></UL><P><B>Further Tasks:</B></P><P>PCR purification, analytical Gel to check sizes of the amplified fragments, assembly PCR of Fragment </P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Analytical_gelelctrophoresis_and_PCR_clean_up_of_sidedirected_mutated_fragments_T1.2C_T2.2C_P1.2C_P2.2C_P3_and_P4_.5B.5B7.5D.5D">Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments T1, T2, P1, P2, P3 and P4 [<A href="http://syntbio.net/iGEM/wiki2011/index.php/Labjournal_september#Sidedirected_mutagenesis_of_TEV_protease_and_14_3C_protease" class="external autonumber" rel="nofollow">[7]</A>]</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[8]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P><B>Aim:</B> Check the size of amplified fragments and clean them up after PCR</P><P><B>Materials:</B></P><UL><LI> PCR fragments P1 to P4 and T1 and T2
</LI></UL><UL><LI> Fermentas 6x loading dye
</LI></UL><UL><LI> 100bp ladder (Gene Ruler by Fermentas)
</LI></UL><UL><LI> Macherey-Nagel Nucleo Spin II Kit for clean up
</LI></UL><P><B>Methode:</B></P><UL><LI> Clean up after manual for PCR purification of the used Kit
</LI></UL><UL><LI> gel electophoresis (2% agarose gel with analytical gel pockets): 30 min at 110 V
</LI></UL><P><B>Results:</B></P><P>The sizes of the fragments of the PCR products are matching the fragment sizes extracted with Geneious:
</P><UL><LI>T1
</LI></UL><UL><LI>sizes in Geneious: 411 bp
</LI></UL><UL><LI> estimated sizes in gel: 410 bp
</LI></UL><UL><LI>T2
</LI></UL><UL><LI>sizes in Geneious: 389 bp
</LI></UL><UL><LI> estimated sizes in gel: 400 bp
</LI></UL><UL><LI>P1
</LI></UL><UL><LI>sizes in Geneious: 167 bp
</LI></UL><UL><LI> estimated sizes in gel: 170 bp
</LI></UL><UL><LI>P2
</LI></UL><UL><LI>sizes in Geneious: 96 bp
</LI></UL><UL><LI> estimated sizes in gel: 100 bp
</LI></UL><UL><LI>P3
</LI></UL><UL><LI>sizes in Geneious: 99 bp
</LI></UL><UL><LI> estimated sizes in gel: 100 bp
</LI></UL><UL><LI>P4
</LI></UL><UL><LI>sizes in Geneious: 311 bp
</LI></UL><UL><LI> estimated sizes in gel 310 bp
</LI></UL><P><A href="/File:UP_AG_2011-09-08-0140_muta_TEV-Pre.JPG" title="File:UP AG 2011-09-08-0140 muta TEV-Pre.JPG">File:UP AG 2011-09-08-0140 muta TEV-Pre.JPG</A></P><P>DNA-Fragments after sidedirected mutagenesis:</P><UL><LI> TEV fragment 1 (T1) c=
</LI></UL><UL><LI> TEV fragment 2 (T2) c=
</LI></UL><UL><LI> 14_3C fragment 1 (P1) c=
</LI></UL><UL><LI> 14_3C fragment 2 (P2) c=
</LI></UL><UL><LI> 14_3C fragment 3 (P3) c=
</LI></UL><UL><LI> 14_3C fragment 4 (P4) c=
</LI></UL><P><B>Output:</B></P><P>Fragments T1 and T2, P1-P4 with expected sizes.
</P><P><B>Further Tasks:</B></P><P>Merge of PCR fragments P1 and P2 into P5, P3 and P4 into P6, T1 and T2 into TEV-protease.
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Assembly_PCR_of_mutated_TEV_protease_and_14_3C_protease_fragments">Assembly PCR of mutated TEV protease and 14_3C protease fragments</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[9]</A>]</B></P><P><B>Investigators:</B> Sascha, Stefan, Sebastian</P><P>Aim: Assembly of mutated TEV and 14_3C protease fragments</P><P><B>Materials:</B></P><UL><LI> TEV protease fragments T1 and T2 --&gt; TEV protease (total mutated)
</LI></UL><UL><LI> 14_3C protease fragments P1 and P2 --&gt; P5
</LI></UL><UL><LI> 14_3C protease fragments P3 and P4 --&gt; P6
</LI></UL><UL><LI> different Primers
</LI></UL><UL><LI> Phusion polymerase kit by NEB
</LI></UL><P><B>Methode:</B></P><P>PCR</P><UL><LI> Template: 2 µL (1µl of each fragment after sizes were checked and fragments got cleaned up with PCR clean up kit from Macherey-Nagel (Nucleo Spin Kit))
</LI></UL><UL><LI> Nucleotides: 1 µL of 10 mM ready to use dNTP mix</LI></UL><UL><LI> 10 µL 5 x Phusion HF Buffer</LI></UL><UL><LI> 2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)</LI></UL><UL><LI> 31,5 µL of pure water</LI></UL><UL><LI>0,5 µL Phusion HF polymerase</LI></UL><P>Program:
</P><UL><LI>Denat: 30 sec 98°C
</LI></UL><UL><LI>30x
</LI></UL><P>Denat: 10 sec 98°C
</P><P>Anneal: 15 sec 70°C (with gradient of +/- 2°C)
</P><P>Extend: 20 sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P>Used Primers:
</P><P>Assembly of fragment 1 and fragment 2 of TEV protease (T1+T2) --&gt; mutated TEV protease:
</P><UL><LI> f_TEV_AraFusion_NgoMIV
</LI></UL><UL><LI> r_TEV_iGEM_BamHI
</LI></UL><P>Assembly of fragment 1 and fragment 2 of 14_3C protease (P1+P2)--&gt; mutated 14_3C portease fragment 5 (P5):
</P><UL><LI> f_14_3C_AraFusion_NgoMIV
</LI></UL><UL><LI> r_14_3C_tm_Xba208_A-T
</LI></UL><P>Assembly of fragment 3 and fragment 4 of 14_3C protease (P3+P4)--&gt; mutated 14_3C portease fragment 6 (P6):
</P><UL><LI> f_14_3C_tm_Xba208_A-T
</LI></UL><UL><LI> r_14_3C_iGEM_BamHI
</LI></UL><P><B>Further Tasks:</B></P><P>PCR purification, analytical Gel to check sizes of the amplified fragments, assembly PCR of Fragment </P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Analytical_gelelctrophoresis_and_PCR_clean_up_of_sidedirected_mutated_fragments_TEV_protease.2C_P5_and_P6_.5B.5B10.5D.5D">Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments TEV protease, P5 and P6 [<A href="http://syntbio.net/iGEM/wiki2011/index.php/Labjournal_september#Assembly_PCR_of_mutated_TEV_protease_and_14_3C_protease_fragments" class="external autonumber" rel="nofollow">[10]</A>]</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[11]</A>]</B></P><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P><B>Aim:</B> Check the size of amplified fragments and clean them up after PCR</P><P><B>Materials:</B></P><UL><LI> PCR fragments P5, P6 and TEV protease
</LI></UL><UL><LI> Fermentas 6x loading dye
</LI></UL><UL><LI> 100bp ladder (Gene Ruler by Fermentas)
</LI></UL><UL><LI> Macherey-Nagel Nucleo Spin II Kit for clean up
</LI></UL><P><B>Methode:</B></P><UL><LI> Clean up after manual for PCR purification of the used Kit
</LI></UL><UL><LI> gel electophoresis (1.5 % agarose gel with analytical gel pockets): 30 min at 110 V
</LI></UL><P><B>Results:</B></P><P>The sizes of the fragments of the PCR products are matching the fragment sizes extracted with Geneious:
</P><UL><LI>TEV
</LI></UL><UL><LI>sizes in Geneious:
</LI></UL><UL><LI> estimated sizes in gel
</LI></UL><UL><LI>P5
</LI></UL><UL><LI>sizes in Geneious:
</LI></UL><UL><LI> estimated sizes in gel
</LI></UL><UL><LI>P6
</LI></UL><UL><LI>sizes in Geneious:
</LI></UL><UL><LI> estimated sizes in gel
</LI></UL><P><B>(Picture from GelDoc and sizes will be added as soon as possible)</B></P><P>DNA-Fragments after sidedirected mutagenesis:</P><UL><LI> TEV protease c=
</LI></UL><UL><LI> 14_3C fragment 5 (P5) c=
</LI></UL><UL><LI> 14_3C fragment 6 (P6) c=
</LI></UL><P>Further Tasks: Merge of PCR fragments P5 and P6 into 14_3C protease, digestion of TEV protease with NgoMIV and BamHI, Ligation of TEV protease with digested AraC induction system (NgoMIV and HindIII) and digested pJC354_ssTorA_XhoI_CS-TEV_NheI_BlaFL (HindIII and BamHI)
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparing_glycerol_stocks_of_pUP089">Preparing glycerol stocks of pUP089 </SPAN></H3><P><B>Time:</B> 2011-09-08 </P><P><B>Investigators:</B> Jessica </P><P><B>Materials:</B></P><UL><LI> overnight cultures of pUP089
</LI></UL><UL><LI> glycerol
</LI></UL><P><B>Method:</B></P><UL><LI> 700 µl culture
</LI></UL><UL><LI> 300 µl glycerol
</LI></UL><UL><LI> stored in -80°C blue iGEM box
</LI></UL><P><B> Output: </B></P><UL><LI> pUP089 Aa, Jes, 08.09.11
</LI></UL><UL><LI> pUP089 Ab, Jes, 08.09.11
</LI></UL><UL><LI> pUP089 Ba, Jes, 08.09.11
</LI></UL><UL><LI> pUP089 Bb, Jes, 08.09.11
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_mdnA-lib2_and_mdnA-Age">Ligation of mdnA-lib2 and mdnA-Age </SPAN></H3><P><B>Time:</B> 2011-09-08 </P><P><B>Investigators:</B> Nicole </P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Purification_of_phages_containing_pPDV089_2">Purification of phages containing pPDV089</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI> after over night incubation:
</LI></UL><UL><LI> centrifuge precipitaed phages: 5000 x g/ 45 min
</LI></UL><UL><LI> discard supernatant
</LI></UL><UL><LI> centrifuge again, 5000 x g/ 5 min
</LI></UL><UL><LI> remove supernatant carefully
</LI></UL><UL><LI> take pellet in 1 ml TBS
</LI></UL><UL><LI> move in 1.5 ml Eppi
</LI></UL><UL><LI> centrifuge again: 17000 x g/10 min
</LI></UL><UL><LI> supernatant in a new Eppi with 200 µl PEG-NaCl (mix!)
</LI></UL><UL><LI> incubate on ice for 60 min
</LI></UL><UL><LI> centrifuge precipitated phages: 17.000 x g/ 10 min (4°C)
</LI></UL><UL><LI> resuspend pellet in 300 µl TBS
</LI></UL><UL><LI> centrifuge:17 000 x g/ 10 min (4°C)
</LI></UL><UL><LI> supernatant in a fresh eppi= purified phages
</LI></UL><P><B>Further tasks:</B></P><UL><LI> measure concentration of phages</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Glycerol_stocks_of_phages_containing_pPDV089_2">Glycerol stocks of phages containing pPDV089</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI> 300 µl phages in TBS
</LI></UL><UL><LI> 750 µl 86 % glycerol
</LI></UL><UL><LI> store at -80°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> measure concentration of phages</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Concentration_of_phages_containing_pPDV089_measured_by_nanodrop_2">Concentration of phages containing pPDV089 measured by nanodrop</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B> control if geneIII-mdnA will be expressed on the phage</P><P><B>Results:</B></P><UL><LI> OD 269: 0,235
</LI></UL><UL><LI> OD 320: 0,010
</LI></UL><UL><LI> phages per ml:
</LI></UL><P>Ph/ml = (Abs 269- Abs 320) * 6*10^16/ bp (phagemid)
</P><UL><LI> bp (phagemid) = 10945
</LI></UL><P>Ph/ml= 1,23*10^12
</P><P><B>Further tasks:</B></P><UL><LI> measure concentration of phages by serial dilution</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="plasmid_pereperation_of_pARWIII_to_control_deletion_of_kanamycin_gene.2C_pSB1C3_containing_mdnA.2C_geneIII_or_mdnA.2FgeneIII">plasmid pereperation of pARWIII to control deletion of kanamycin gene, pSB1C3 containing mdnA, geneIII or mdnA/geneIII </SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine</P><P><B>Aim:</B> control deletion of kanamycin gene in created pARWIII, control ligation of mdnA, geneIII and mdnA/geneIII into pSB1C3</P><P><B>Method/Materials:</B></P><P>plasmid preperation:
</P><P>protocol 5.1 of the NucleoSpin Plasmid Kit
</P><P><B>Further tasks:</B></P><P>test digestion
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Antibody_detection_with_blotted_membranes">Antibody detection with blotted membranes</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B> control if geneIII-myc-mdnA is presented on the phages</P><P><B>Method/Materials:</B></P><UL><LI> incubate blocked membrane for 1 h with primary antibody (anti -c-myc- antibody) in TBS with 5% milk powder
</LI></UL><UL><LI> wash 3x 10 min with TBS buffer
</LI></UL><UL><LI> incubate for 1 h with secondary antibody
</LI></UL><UL><LI> wash 3x 10 min with TBS buffer
</LI></UL><UL><LI> develop membranes with ECL- Kit</LI></UL><P><B>Results:</B></P><UL><LI> signals were only seen at positive controls and one negative control
</LI></UL><P><B>Further tasks:</B></P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="90th_Labday_2011-09-09">90th Labday 2011-09-09</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparation_of_samples_for_HPLC_analysis">Preparation of samples for HPLC analysis</SPAN></H3><P><B>Time:</B> 2011-09-09</P><P><B>Investigators:</B> Jessica, Katharina</P><P><B>Material:</B></P><UL><LI> pellets of pARW089 and pARW071 samples (from 2011-09-06)
</LI></UL><UL><LI> 100% methanol
</LI></UL><UL><LI> 5&amp; methanol
</LI></UL><UL><LI> sterile water
</LI></UL><UL><LI> Sep-Pak Cartridges
</LI></UL><UL><LI> speed vac
</LI></UL><P><B>Method:</B></P><UL><LI> pellet was resuspended in 5 ml of sterile water
</LI></UL><UL><LI> the resuspended sample was sonicated for 5 min (3 sec on, 3 sec off)
</LI></UL><UL><LI> centrifugation for 10 min @ 11000 rpm
</LI></UL><UL><LI> supernatant was transfered to Sep-Pak Plus C18 Cartridges, that were equilibrated with 2 ml of 100% methanol
</LI></UL><UL><LI> cartridges were washed with 2 ml water
</LI></UL><UL><LI> samples were load
</LI></UL><UL><LI> cartridges were washed with 2 ml of 5% methanol
</LI></UL><UL><LI> samples were eluted with 2 ml of 100% methanol
</LI></UL><UL><LI> samples were put in a speedvac so that the methanol can evaporate
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Counting_of_colonies_on_plates_.28pUP089.2Blib2.29">Counting of colonies on plates (pUP089+lib2)</SPAN></H3><P><B>Time:</B> 2011-09-09,7:30 </P><P><B>Investigators:</B> Nadine, Nicole</P><P><B>Aim:</B> count number of colonies on plates</P><P><B>Material:</B></P><UL><LI> agar plate (kan): 10x pUP089+lib2, Nadja, 7.9.11, resistance: kan, from 2011-9-8, Nad/Nic</LI></UL><P><B>Results:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>plate</B></TD><TD><B># of colonies</B></TD><TD><B>picked colonies</B></TD></TR><TR><TD> 1 </TD><TD> 61 </TD><TD> I clone E and F
</TD></TR><TR><TD> 2 </TD><TD> 38 </TD><TD> II clone C and D
</TD></TR><TR><TD> 3 </TD><TD> 87 </TD><TD> III clone A and B
</TD></TR><TR><TD> 4 </TD><TD> 47 </TD><TD> IV clone G and H
</TD></TR><TR><TD> 5 </TD><TD> 33 </TD><TD> IV clone I and J
</TD></TR><TR><TD> 6 </TD><TD> 55</TD></TR><TR><TD> 7 </TD><TD> 37 </TD></TR><TR><TD> 8 </TD><TD> 70 </TD></TR><TR><TD> 9 </TD><TD> 28 </TD></TR><TR><TD> 10 </TD><TD> 36 </TD></TR><TR><TD><B>sum</B></TD><TD><B>492</B></TD></TR></TBODY></TABLE><UL><LI> example plates
</LI></UL><P><B>Conclusions:</B></P><UL><LI> 492 colonies
</LI></UL><UL><LI> some colonies will be picked for test sequencing (clone A-J)
</LI></UL><UL><LI> repeat fillin, ligation and transformation for more mutants
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Glycerol_stocks_of_pSB1C3.2BmdnDE_K1.2C_K2_and_K5">Glycerol stocks of pSB1C3+mdnDE K1, K2 and K5</SPAN></H3><P><B>Time:</B> 2011-09-09, 8:00</P><P><B>Investigators:</B> Nadine</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Sequencing_of_pSB1C3.2BmdnD.2FABC">Sequencing of pSB1C3+mdnD/ABC</SPAN></H3><P><B>Time:</B> 2011-09-09, 8:00</P><P><B>Investigators:</B> Nicole</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_XL1_with_different_constructs_2">Transformation of XL1 with different constructs</SPAN></H3><P><B>Time:</B> 2011-09-09, </P><P><B>Investigators:</B> Nadine, Nicole </P><P><B>Materials:</B></P><UL><LI> XL1-Blue
</LI></UL><UL><LI> ligation products (2011-9-8):
</LI></UL><UL><LI> mdnA-lib2
</LI></UL><UL><LI> mdnA-Age
</LI></UL><UL><LI> mdnABCDE from pARW071
</LI></UL><UL><LI> antibiotics from stock:
</LI></UL><UL><LI> chloramphenicol
</LI></UL><UL><LI> kanamycin
</LI></UL><UL><LI> ampiciline
</LI></UL><UL><LI> LB-Agar
</LI></UL><P><B>Method:</B></P><UL><LI> ligation products:
</LI></UL><UL><LI> addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI> incubation 30 min on ice,
</LI></UL><UL><LI> heat shock 60 sec at 42°C,
</LI></UL><UL><LI> incubation 5 min on ice,
</LI></UL><UL><LI> addition of 750 µl LB medium,
</LI></UL><UL><LI> incubation at 37 °C shaking for 60 min,
</LI></UL><UL><LI> plating on LB medium with appropriate antibiotic (20 ml Agar and 20 µl antibiotic (see above) per plate)
</LI></UL><UL><LI> storage over night at 37°C (start: 15:00)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture (except for library 2)
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><UL><LI> count colonies of pUP089_lib2 and collect them w/ LB
</LI></UL><P><B>Output:</B></P><UL><LI>Plates w/ XL1 blue:
</LI></UL><UL><LI> 10 x pUP089+mdnA-lib2 (on kan plates) + control
</LI></UL><UL><LI> 10 x pARWIII+mdnA-Age (on amp plates) + control
</LI></UL><UL><LI> pSB1C3+mdnABCDE from pARW071 (on cm plates) + control
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Survival-Test_of_one_14_3C_clone.28_SG9-P10.29_and_one_Tev-clone_.28SG13-T6.29">Survival-Test of one 14_3C clone( SG9-P10) and one Tev-clone (SG13-T6)</SPAN></H3><P><B> Investigators: </B> Paul, Stefan, Sascha
</P><P><B> Aim: </B> Test whether TorA-detection device functions
</P><UL><LI> SG13-T6 contains two silent mutations and one single amino-acid exchange (N--&gt;D)
</LI></UL><UL><LI> SG9-P10 contains different mutations:...
</LI></UL><P><B> Plates: </B></P><P>30x, 15x for each clone
</P><UL><LI>2x Cm (25 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM IPTG
</LI></UL><UL><LI>2x Cm + 1mM Ara
</LI></UL><UL><LI>2x Cm + Amp (50 µg/ml)
</LI></UL><UL><LI>2x Cm + Amp (200 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM Ara + Amp (50 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM Ara + Amp (100 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM Ara + Amp (200 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM Ara + Amp (400 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM Ara + Amp (800 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM Ara + 1mM IPTG + Amp (50 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM Ara + 1mM IPTG + Amp (100 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM Ara + 1mM IPTG + Amp (200 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM Ara + 1mM IPTG + Amp (400 µg/ml)
</LI></UL><UL><LI>2x Cm + 1mM Ara + 1mM IPTG + Amp (800 µg/ml)
</LI></UL><P><B> Method: </B></P><UL><LI> cells from glycerolstocks (-80°C) were inoculated in 5 ml LB-Medium (Cm = 25µg/ml). Then diluted and incubated until OD=0.002
</LI></UL><UL><LI> 100µl of cells with OD=0.002 were plated on prepared plates
</LI></UL><P><B> Results: </B></P><P>Survival screening have to be repeated, because there is no way to compare numbers of clones on plates (dual induction of protease via AraC and ssTorA_CS_blaFL construct via IPTG) with number of clones on the right controll plates (singel induction with IPTG to check the export via TAT-pathway of beta lactamase.</P><P>No resistance against ampicillin without induction with IPTG. Dual-induction with arabinose and IPTG does not lead to a dramatical reduced number of clones.
</P><P><B> Further Tasks:</B></P><UL><LI> Repeat the survival test with singel induction via IPTG as comparison for the cell death based on reduced export of beta lactamase into the periplasm of <I>E. coli</I></LI></UL><UL><LI> repeat survival test at 37 °C for both protease to check influence of creation of inclusion body of TEV protease at 37°C and influence of export capacity of beta lactamase via TAT-pathway
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Assembly_PCR_of_mutated_14_3C_protease">Assembly PCR of mutated 14_3C protease</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[12]</A>]</B></P><P><B>Investigators:</B> Sascha, Stefan, Sebastian</P><P>Aim: Assembly of mutated 14_3C</P><P><B>Materials:</B></P><UL><LI> 14_3C protease fragments P5 and P6 --&gt; 14_3C protease (total mutated)
</LI></UL><UL><LI> Primer 1: f_14_3C_AraFusion_NgoMIV
</LI></UL><UL><LI> Primer 2: r_14_3C_iGEM_BamHI
</LI></UL><UL><LI> Phusion polymerase kit by NEB
</LI></UL><P><B>Methode:</B></P><P>PCR</P><UL><LI> Template: 2 µL (1µl of each fragment after sizes were checked and fragments got cleaned up with PCR clean up kit from Macherey-Nagel (Nucleo Spin Kit))
</LI></UL><UL><LI> Nucleotides: 1 µL of 10 mM ready to use dNTP mix</LI></UL><UL><LI> 10 µL 5 x Phusion HF Buffer</LI></UL><UL><LI> 2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)</LI></UL><UL><LI> 31,5 µL of pure water</LI></UL><UL><LI>0,5 µL Phusion HF polymerase</LI></UL><P>Program:
</P><UL><LI>Denat: 30 sec 98°C
</LI></UL><UL><LI>30x
</LI></UL><P>Denat: 10 sec 98°C
</P><P>Anneal/extend: 30 sec 72°C
</P><UL><LI>Final Extend: 10min 72°C
</LI></UL><P><B>Further Tasks:</B></P><P>PCR purification, analytical Gel to check sizes of the amplified fragments. Digest and ligation.</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_NgoMIV_TEV-Protease_iGEM_BamHI.2C_NgoMIV_14_3C-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_fragments_and_pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5_and_pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5_vector_2">Digestion of NgoMIV_TEV-Protease_iGEM_BamHI, NgoMIV_14_3C-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV fragments and pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5 and pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5 vector</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[13]</A>]</B></P><P><B>Investigators:</B>Paul, Sascha, Stefan</P><P><B>Materials:</B></P><P>NgoMIV_TEV-Protease_iGEM_BamHI: 66,2 ng/µl
</P><P>NgoMIV_14_3C-Protease_iGEM_BamHI: 91,7ng/µl
</P><P>HindIII_iGEM_AraC_NgoMIV: 48,3 ng/µl
</P><P>pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (contains TEV cleavage site): 470 ng/µl
</P><P>pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5 vector (contains 14_3C cleavage site): 270,2 ng/µl
</P><P><B> Digestion protocol: </B></P><P>1: NgoMIV_TEV-Protease_iGEM_BamHI:
</P><UL><LI>10µl NgoMIV_iGEM_TEV-Protease_iGEM_BamHI
</LI></UL><UL><LI>1µl NgoMIV
</LI></UL><UL><LI>1µl BamHI-HF
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>32.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>2: NgoMIV_14_3C-Protease_iGEM_BamHI:
</P><UL><LI>10µl NgoMIV_14_3C-Protease_iGEM_BamHI
</LI></UL><UL><LI>1µl NgoMIV
</LI></UL><UL><LI>1µl BamHI-HF
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>32.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>3: HindIII_iGEM_AraC_NgoMIV:
</P><UL><LI>10µl HindIII_iGEM_AraC_NgoMIV
</LI></UL><UL><LI>1µl HindIII
</LI></UL><UL><LI>1µl NgoMIV
</LI></UL><UL><LI>5µl 10x buffer = NEB4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>32.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>4: pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5:
</P><UL><LI>4µl pUP189-pJC354-NheI-TEV-Xho_blaFL_GGH5
</LI></UL><UL><LI>1µl HindIII
</LI></UL><UL><LI>1µl BamHI-HF
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>39.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>5: pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5:
</P><UL><LI>8µl pUP189-pJC354-NheI-143C-Xho_blaFL_GGH5
</LI></UL><UL><LI>1µl HindIII
</LI></UL><UL><LI>1µl BamHI-HF
</LI></UL><UL><LI>5µl 10x buffer = NEB 4
</LI></UL><UL><LI>0.5µl 100x BSA
</LI></UL><UL><LI>37.5µl pure water
</LI></UL><UL><LI>=50µl
</LI></UL><P>--&gt;The reaction was allowed to proceed for 2h at 37°C!
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI_or_NgoMIV_iGEM_TEV-Protease_iGEM_BamHI.2C_HindIII_iGEM_AraC_NgoMIV_into_pJC354-NheI-143C-Xho_blaFL_GGH5_or_pJC354-NheI-TEV-Xho_blaFL_GGH5_vector_2"> Ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI or NgoMIV_iGEM_TEV-Protease_iGEM_BamHI, HindIII_iGEM_AraC_NgoMIV into pJC354-NheI-143C-Xho_blaFL_GGH5 or pJC354-NheI-TEV-Xho_blaFL_GGH5 vector</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[14]</A>]</B></P><P><B>Investigators:</B>Sebastian, Sascha, Stefan</P><P><B>Aim</B>:</P><UL><LI><B>1. </B>Triple-ligation of NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI (573bp), HindIII_iGEM_AraC_NgoMIV (1273bp) and pJC354-NheI-143C-Xho_blaFL_GGH5 vector (~4700bp) to create pUP_SG2_TorA_CS-14_3C_bla_AraC-14_3C</LI></UL><UL><LI><B>2.</B>Triple-ligation of NgoMIV_iGEM_TEV_iGEM_BamHI (760bp), HindIII_iGEM_AraC_NgoMIV (1273bp) and pJC354-NheI-TEV-Xho_blaFL_GGH5 vector (~4700bp)to create pUP_SG1_TorA_CS-TEV_bla_AraC-TEV</LI></UL><P>Calculation of volumes to be used with: <A href="http://www.gibthon.org/ligate.html" class="external text" rel="nofollow">ligation calculator</A> with 1:1 molar ratio
</P><P><B>Materials:</B></P><P>14_3C:
</P><P>1:
</P><UL><LI> 1 µL NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI fragment
</LI></UL><UL><LI> 1 µL AraC fragment
</LI></UL><UL><LI> 6 µL pJC354-NheI-143C-Xho_blaFL_GGH5 vector
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><UL><LI> 1 controls: As 1 but with water instead of fragment
</LI></UL><P>TEV:
</P><UL><LI> 1 µL NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment
</LI></UL><UL><LI> 1 µL AraC fragment
</LI></UL><UL><LI> 6 µL pJC354-NheI-TEV-Xho_blaFL_GGH5 vector
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><UL><LI> 1 control: As 4 but with water instead of fragment
</LI></UL><P>TEV backbone with 14_3C:
</P><UL><LI> 1 µL NgoMIV_iGEM_14_3C-Protease_iGEM_BamHI fragment
</LI></UL><UL><LI> 1 µL AraC fragment
</LI></UL><UL><LI> 6 µL pJC354-NheI-TEV-Xho_blaFL_GGH5 vector
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P>14_3C backbone with TEV:
</P><UL><LI> 1 µL NgoMIV_iGEM_TEV-Protease_iGEM_BamHI fragment
</LI></UL><UL><LI> 1 µL AraC fragment
</LI></UL><UL><LI> 6 µL pJC354-NheI-143C-Xho_blaFL_GGH5 vector
</LI></UL><UL><LI> 1 µL T4 liagtion buffer (Fermentas)
</LI></UL><UL><LI> 1 µL T4 ligase (Fermentas)
</LI></UL><UL><LI> =10µl
</LI></UL><P><B> Used method: </B></P><P>ligation at room temperatur for 1h
</P><P><B> Further task: Transformation of XL1blue cells with ligation products </B></P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_from_overnight_cultures_T_I.2FII.2C_14_3C_CS.2CPC.2C_TEV-CS-1.2C_P1">Miniprep from overnight cultures T I/II, 14_3C CS,PC, TEV-CS-1, P1 </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[15]</A>]</B></P><P><B>Investigators:</B> Niels </P><P><B>Aim:</B> isolate DNA &amp; sequencing </P><P><B>Samples for miniprep </B></P><UL><LI> T I
</LI></UL><UL><LI> T II
</LI></UL><UL><LI> 14_3C-CS
</LI></UL><UL><LI> TEV-CS
</LI></UL><UL><LI> P1
</LI></UL><UL><LI> PC </LI></UL><P><B>Materials </B></P><P>miniprep by using Kit :
</P><UL><LI> NucleoSpin Plasmid
</LI></UL><P><B>Nanodrop measurement</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>Sample</B></TD><TD><B>concentration [ng/µl]</B></TD><TD><B>260/280</B></TD><TD><B>260/230</B></TD></TR><TR><TD> T I </TD><TD> 164,6 </TD><TD> 1,87 </TD><TD> 2,04
</TD></TR><TR><TD> T II </TD><TD> 182,7 </TD><TD> 1,88 </TD><TD> 2,07
</TD></TR><TR><TD> 14_3C-cs </TD><TD> 263,1 </TD><TD> 1,86 </TD><TD> 2,13
</TD></TR><TR><TD> TEV-cs </TD><TD> 181,8 </TD><TD> 1,87 </TD><TD> 2,07
</TD></TR><TR><TD> PC </TD><TD> 150,5 </TD><TD> 1,85 </TD><TD> 2,05
</TD></TR><TR><TD> P1 </TD><TD> 357,6 </TD><TD> 1,86 </TD><TD> 2,19
</TD></TR></TBODY></TABLE><P><B>futher task </B>:
</P><UL><LI> sequencing
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Sequencing_of_miniprep_.2809-09-2011.29_-_T_I.2FII.2C_14_3C_CS.2CPC.2C_TEV-CS-1.2C_P1"> Sequencing of miniprep (09-09-2011) - T I/II, 14_3C CS,PC, TEV-CS-1, P1 </SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[16]</A>]</B></P><P><B>Investigators:</B> Niels, Stefan </P><P><B>Samples: miniprep from 09-09-11</B></P><UL><LI> T I
</LI></UL><UL><LI> T II
</LI></UL><UL><LI> TEV-cs
</LI></UL><UL><LI> 14_3C-cs
</LI></UL><UL><LI> P1
</LI></UL><UL><LI> PC
</LI></UL><UL><LI>each:
</LI></UL><UL><LI>1400 ng DNA
</LI></UL><UL><LI>ad 20 µl steril water
</LI></UL><P>sequenced by GATC
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_TEV_and_14_3C"> Transformation of TEV and 14_3C</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[17]</A>]</B></P><P><B>Investigators:</B>Sebastian, Sascha, Stefan</P><P><B>Aim</B>:</P><P><B>Materials:</B></P><P>Heat shock protocol
</P><P><B> Used method: </B></P><P><B> picking clones </B></P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Test_digest_of_pSB1C3.2BmycN.2FC">Test digest of pSB1C3+mycN/C </SPAN></H3><P><B>Time:</B> 2011-09-09 </P><P><B>Investigators:</B> Jessica </P><P><B> Aim: </B> prove of Insert (mdnAmycN/C)</P><P><B>Materials:</B></P><UL><LI> pSB1C3+mycN clones J1-J5, Niels, 9.9.11
</LI></UL><UL><LI> pSB1C3+mycC clones J1-J5, Niels, 9.9.11
</LI></UL><UL><LI> AvaII, PstI-HF (NEB)
</LI></UL><UL><LI> Buffer 4 (NEB, 10x)
</LI></UL><UL><LI> water
</LI></UL><P><B>Plan:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>insert</B></TD><TD><B>plasmid size in bp</B></TD><TD><B>enzymes</B></TD><TD><B>buffer</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> mdnAmycN </TD><TD> 2596 </TD><TD> AvaII, Pst-HF </TD><TD> 4 </TD><TD> 225, 942, 1439
</TD></TR><TR><TD> mdnAmycC </TD><TD> 2597</TD><TD> AvaII, Pst-HF</TD><TD> 4 </TD><TD> 225, 942, 1440
</TD></TR></TBODY></TABLE><P><B> Digestion protocol </B></P><UL><LI>2 µl DNA
</LI></UL><UL><LI>0.5 µl AvaII
</LI></UL><UL><LI>0.5 µl PstI-HF
</LI></UL><UL><LI>2 µl 10x buffer
</LI></UL><UL><LI>15 µl pure water
</LI></UL><UL><LI>total: 20µl
</LI></UL><UL><LI>37°C overnight</LI></UL><P><B>Agarose gel</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix (1:10) </TD><TD> 12
</TD></TR><TR><TD> 1 </TD><TD> mycN clone 1 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 2 </TD><TD> mycN clone 2 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 3 </TD><TD> mycN clone 3 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 4 </TD><TD> mycN clone 4 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 5 </TD><TD> mycN clone 5 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 6 </TD><TD> mycC clone 1 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 7 </TD><TD> mycC clone 2 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 8 </TD><TD> mycC clone 3 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 9 </TD><TD> mycC clone 4 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 10 </TD><TD> mycC clone 5 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix (1:10) </TD><TD> 12
</TD></TR></TBODY></TABLE><P><B>Conclusions: </B></P><UL><LI> digestion did not work out in a satisfying way
</LI></UL><P><B> Further tasks: </B></P><UL><LI>new restriction enzyme digestion
</LI></UL><UL><LI>sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_.2B_pSB1C3.2BmycN.2FmycC">Miniprep of overnight cultures + pSB1C3+mycN/mycC </SPAN></H3><P><B>Time:</B> 2011-09-09, 14:30</P><P><B>Investigators:</B> Niels</P><P><B>Samples for miniprep </B></P><UL><LI> pSB1C3+mycN J1
</LI></UL><UL><LI> pSB1C3+mycN J2
</LI></UL><UL><LI> pSB1C3+mycN J3
</LI></UL><UL><LI> pSB1C3+mycN J4
</LI></UL><UL><LI> pSB1C3+mycN J5
</LI></UL><UL><LI> pSB1C3+mycC J1
</LI></UL><UL><LI> pSB1C3+mycC J2
</LI></UL><UL><LI> pSB1C3+mycC J3
</LI></UL><UL><LI> pSB1C3+mycC J4
</LI></UL><UL><LI> pSB1C3+mycC J5</LI></UL><P><B>Materials </B></P><P>miniprep by using Kit :
</P><UL><LI> NucleoSpin Plasmid
</LI></UL><P><B>Nanodrop measurement</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>Sample</B></TD><TD><B>concentration [ng/µl]</B></TD><TD><B>260/280</B></TD><TD><B>260/230</B></TD></TR><TR><TD> mycN-1 </TD><TD> 189,7 </TD><TD> 1,87 </TD><TD> 2,46
</TD></TR><TR><TD> mycN-2 </TD><TD> 95,3 </TD><TD> 1,86 </TD><TD> 1,58
</TD></TR><TR><TD> mycN-3 </TD><TD> 132,0 </TD><TD> 1,85 </TD><TD> 2,57
</TD></TR><TR><TD> mycN-4 </TD><TD> 130,6 </TD><TD> 1,84 </TD><TD> 2,17
</TD></TR><TR><TD> mycN-5 </TD><TD> 152,8 </TD><TD> 1,86 </TD><TD> 2,46
</TD></TR><TR><TD> mycC-1 </TD><TD> 143,0 </TD><TD> 1,86 </TD><TD> 2,57
</TD></TR><TR><TD> mycC-2 </TD><TD> 92,5 </TD><TD> 1,88 </TD><TD> 2,11
</TD></TR><TR><TD> mycC-3 </TD><TD> 105,3 </TD><TD> 1,84 </TD><TD> 1,78
</TD></TR><TR><TD> mycC-4 </TD><TD> 109,5 </TD><TD> 1,84 </TD><TD> 1,98
</TD></TR><TR><TD> mycC-5 </TD><TD> 124,5 </TD><TD> 1,84 </TD><TD> 2,15
</TD></TR></TBODY></TABLE><P><B>futher task </B>:
</P><UL><LI> sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Sequencing_of_pSB1C3.2BmdnDE.2FmycN.2FmycC">Sequencing of pSB1C3+mdnDE/mycN/mycC </SPAN></H3><P><B>Time:</B> 2011-09-09, 15:30</P><P><B>Investigators:</B> Nicole, Niels, Jessica</P><P><B>Samples: miniprep from 09-09-11</B></P><UL><LI> pSB1C3+mdnDE K1 (primers: VR2, sf_mdne_1_n (# 9), sf_mdne_2_1 (# 10), sf_mdne_3_1 (# 11))
</LI></UL><UL><LI> pSB1C3+mdnDE K2
</LI></UL><UL><LI> pSB1C3+mdnDE K5
</LI></UL><UL><LI> pSB1C3+mycN J1
</LI></UL><UL><LI> pSB1C3+mycN J2
</LI></UL><UL><LI> pSB1C3+mycN J3
</LI></UL><UL><LI> pSB1C3+mycN J4
</LI></UL><UL><LI> pSB1C3+mycN J5
</LI></UL><UL><LI> pSB1C3+mycC J1
</LI></UL><UL><LI> pSB1C3+mycC J2
</LI></UL><UL><LI> pSB1C3+mycC J3
</LI></UL><UL><LI> pSB1C3+mycC J4
</LI></UL><UL><LI> pSB1C3+mycC J5</LI></UL><UL><LI>each:
</LI></UL><UL><LI>1400 ng DNA
</LI></UL><UL><LI>ad 20 µl steril water
</LI></UL><P>sequenced by GATC
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Picking_clones_for_overnight_cultures_of_Lib_2_for_confirmation">Picking clones for overnight cultures of Lib 2 for confirmation</SPAN></H3><P><B>Investigators:</B>Katharina</P><P><B>Time:</B> 2011-09-09</P><P><B>Materials:</B></P><UL><LI> agar plates with Lib2 clones
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> kanamycin
</LI></UL><P><B>Method:</B></P><UL><LI> picking 10 clones
</LI></UL><UL><LI> Lib2 clone A-J
</LI></UL><UL><LI> incubate over night in 37°C shaker (200 rpm)
</LI></UL><P><B>Output:</B></P><UL><LI> 10 liquid cultures Lib2
</LI></UL><P><B>Further tasks:</B></P><UL><LI> miniprep
</LI></UL><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Prepare_liquid_culture_for_mdnA_foc2">Prepare liquid culture for mdnA foc2</SPAN></H3><P><B>Investigators:</B>Katharina</P><P><B>Time:</B> 2011-09-09</P><P><B>Materials:</B></P><UL><LI> agar plates with Lib2 clones
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> kanamycin
</LI></UL><P><B>Method:</B></P><UL><LI> using 5 ml of LB media (without antibiotic) for sluicing down the colonies from the first plate
</LI></UL><UL><LI> transferring the media to the next plate
</LI></UL><UL><LI> procedure for all 10 plates
</LI></UL><UL><LI> incubate over night in 37°C shaker (200 rpm)
</LI></UL><P><B>Output:</B></P><UL><LI> 1 liquid cultures mdnA foc2
</LI></UL><P><B>Further tasks:</B></P><UL><LI> miniprep
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Picking_clones_for_overnight_cultures_of_pSB1C3.2BmdnABCDE_with_and_without_T7-promotor">Picking clones for overnight cultures of pSB1C3+mdnABCDE with and without T7-promotor</SPAN></H3><P><B>Investigators:</B>Katharina</P><P><B>Time:</B> 2011-09-09</P><P><B>Materials:</B></P><UL><LI> agar plates of pSB1C3+mndABCDE with T7 promotor and pSB1C3+mdnABCDE without T7-promotor
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> chloramphenicol
</LI></UL><P><B>Method:</B></P><UL><LI> picking 4 clones for pSB1C3+mdnABCDE without T7-promotor
</LI></UL><UL><LI> picking 2 clones for pSB1C3+mdnABCDE with T7-promotor
</LI></UL><UL><LI> incubate over night @ 37°C (200 rpm)
</LI></UL><P><B>Output:</B></P><UL><LI> 4 liquid cultures of pSB1C3+mdnABCDE without T7-promotor
</LI></UL><UL><LI> 2 liquid cultures of pSB1C3+mndABCDE with T7
</LI></UL><P><B>Further tasks:</B></P><UL><LI> miniprep
</LI></UL><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparing_overnight_cultures_of_expression_backbone-clones">Preparing overnight cultures of expression backbone-clones</SPAN></H3><P><B>Investigators:</B>Katharina</P><P><B>Time:</B> 2011-09-09</P><P><B>Materials:</B></P><UL><LI> agar plates of expression backbones
</LI></UL><UL><LI> pSB1A3_YFP_Ara
</LI></UL><UL><LI> pSB1A3_YFP_Lac
</LI></UL><UL><LI> pSB1K3_CFP_Ara
</LI></UL><UL><LI> pSB1K3_CFP_Lac
</LI></UL><UL><LI> pSB1K3_YFP_Lac
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> ampicillin, kanamycin
</LI></UL><P><B>Method:</B></P><UL><LI> picking 1 clone for each construct into 10 ml LB media+antibiotic
</LI></UL><UL><LI> incubation overnight @ 37°C (200 rpm)
</LI></UL><P><B>Output:</B></P><UL><LI> 5 liquid cultures of different expression backbones
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR:_mdnABCDE_3">PCR: mdnABCDE </SPAN></H3><P><B>Time:</B> 2011-09-09</P><P><B>Investigators:</B> Nadja</P><P><B>Materials</B></P><UL><LI> vector: pARW071 (1); pARW089 (2*)
</LI></UL><UL><LI> Long Enzyme Mix (+MgCl2) Buffer Fermentas
</LI></UL><UL><LI> Long Enzyme Mix Polymerase Fermentas
</LI></UL><UL><LI> dNTPs (NEB)
</LI></UL><UL><LI> water
</LI></UL><UL><LI> Primer:
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>#</B></TH><TH><B>Primer</B></TH></TR><TR><TD>84</TD><TD> pf_mdnABCDE089_NotI_SpeI_30.08.
</TD></TR><TR><TD>83</TD><TD>pf_mdnABCDE_T7_EcoRI_NotI
</TD></TR><TR><TD>82</TD><TD> r_mdnABCDE_iGEM
</TD></TR></TBODY></TABLE><P><B>Protocol:</B></P><P>1. General composition
</P><UL><LI> 2 µl vector: pARW071 (1); pARW089 (2*) (diluted 1:100)
</LI></UL><UL><LI> 1 µl dNTPs (10 mM)
</LI></UL><UL><LI> 3 µl Primer forward (10 µM)
</LI></UL><UL><LI> 3 µl Primer backward (10 µM)
</LI></UL><UL><LI> 5 µl buffer
</LI></UL><UL><LI> 0.3 µl Ploymerase
</LI></UL><UL><LI> 35,7 µl water
</LI></UL><UL><LI> total volume: 50 µl
</LI></UL><P>2. Composition
</P><UL><LI> pARW071 (1): 84, 82
</LI></UL><UL><LI> pARW089 (2*)-T7: 83, 82
</LI></UL><UL><LI> pARW089 (2*): 84. 82
</LI></UL><P>3. PCR programs
</P><UL><LI>IGLONG2</LI></UL><P><B>Further task</B></P><UL><LI> gel verification
</LI></UL><UL><LI> purification
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="test_digestion_to_control_if_pSB1C3_is_containing_mdnA.2C_geneIII_or_mdnA.2FgeneIII">test digestion to control if pSB1C3 is containing mdnA, geneIII or mdnA/geneIII </SPAN></H3><P><B>Investigators:</B> Sandrina, Steffi</P><P><B>Aim:</B>control ligation of mdnA, geneIII and mdnA/geneIII into pSB1C3</P><P><B>Method/Materials:</B></P><UL><LI> 5 clones from pSB1C3 with mdnA (chloramphenicol), 5 clones from pSB1C3 with geneIII (chloramphenicol), 5 clones from pSB1C3 with mdnA/geneIII (chloramphenicol):
</LI></UL><UL><LI>4 µl vector DNA (58- 120 ng/µl)
</LI></UL><UL><LI>2 µl buffer 3
</LI></UL><UL><LI>0,2 µl BSA
</LI></UL><UL><LI>0.5 µl XbaI
</LI></UL><UL><LI>0.5 µl PstI
</LI></UL><P>12.8 µl water
</P><P>incubate for 1 h at 37°C </P><P><B>Results:</B></P><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected insert size in bp</B></TD></TR><TR><TD> M </TD><TD> marker, DNA ladder mix Fermentas </TD></TR><TR><TD> 1 </TD><TD> free </TD></TR><TR><TD> 2 </TD><TD> mdnA-geneIII fusion gene in pSB1C3, clone 1 </TD><TD> 12</TD><TD> ca. 700
</TD></TR><TR><TD> 3 </TD><TD> mdnA-geneIII fusion gene in pSB1C3, clone 2 </TD><TD> 12</TD><TD> ca. 700
</TD></TR><TR><TD> 4 </TD><TD> mdnA-geneIII fusion gene in pSB1C3, clone 3 </TD><TD> 12</TD><TD> ca. 700
</TD></TR><TR><TD> 5 </TD><TD> mdnA-geneIII fusion gene in pSB1C3, clone 4 </TD><TD> 12</TD><TD> ca. 700
</TD></TR><TR><TD> 6 </TD><TD> mdnA-geneIII fusion gene in pSB1C3, clone 5 </TD><TD> 12</TD><TD> ca. 700
</TD></TR><TR><TD> 7 </TD><TD> geneIII in pSB1C3, clone 1 </TD><TD> 10 </TD><TD> ca. 500
</TD></TR><TR><TD> 8 </TD><TD> geneIII in pSB1C3, clone 2 </TD><TD> 10 </TD><TD> ca. 500
</TD></TR><TR><TD> 9 </TD><TD> geneIII in pSB1C3, clone 3 </TD><TD> 10 </TD><TD> ca. 500
</TD></TR><TR><TD> 10</TD><TD> geneIII in pSB1C3, clone 4 </TD><TD> 10 </TD><TD> ca. 500
</TD></TR><TR><TD> 11</TD><TD> geneIII in pSB1C3, clone 5 </TD><TD> 10 </TD><TD> ca. 500
</TD></TR><TR><TD> 12</TD><TD> mdnA in pSB1C3, clone 1 </TD><TD> 12</TD><TD> ca. 180
</TD></TR><TR><TD> 13</TD><TD> mdnA in pSB1C3, clone 2 </TD><TD> 12</TD><TD> ca. 180
</TD></TR><TR><TD> 14</TD><TD> mdnA in pSB1C3, clone 3 </TD><TD> 12</TD><TD> ca. 180
</TD></TR><TR><TD> 15</TD><TD> mdnA in pSB1C3, clone 4 </TD><TD> 12</TD><TD> ca. 180
</TD></TR><TR><TD> 16</TD><TD> mdnA in pSB1C3, clone 5 </TD><TD> 12</TD><TD> ca. 180
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><P>sequencing of clones
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Send_clones_2-6_.28geneIII-mdnA_in_pSB.29_for_sequencing">Send clones 2-6 (geneIII-mdnA in pSB) for sequencing</SPAN></H3><P><B>Investigators:</B> Sandrina, Steffi</P><P><B>Aim:</B>control ligation of mdnA/geneIII into pSB1C3</P><P><B>Method/Materials:</B></P><UL><LI> Primer: VR
</LI></UL><UL><LI> 20 µl with 70 ng/µl DNA
</LI></UL><P><B>Further tasks:</B></P><P>check alignment
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="test_digestion_to_control_if_kanamycin_resistance_gene_is_knocked_out_of_pARWIII">test digestion to control if kanamycin resistance gene is knocked out of pARWIII</SPAN></H3><P><B>Investigators:</B> Sandrina, Steffi</P><P><B>Aim:</B>control knock out of kanamycin on pARWIII</P><P><B>Method/Materials:</B></P><UL><LI> 5 clones:
</LI></UL><UL><LI>4 µl vector DNA (100- 300ng/µl)
</LI></UL><UL><LI>2 µl buffer 3
</LI></UL><UL><LI>0,2 µl BSA
</LI></UL><UL><LI>0.5 µl NsiI
</LI></UL><UL><LI>13.3 µl water
</LI></UL><P>incubate for 1 h at 37°C </P><P><B>Results:</B></P><P><B> Loading of gels</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected insert size in bp</B></TD></TR><TR><TD> M </TD><TD> marker, DNA ladder mix Fermentas </TD></TR><TR><TD> 1 </TD><TD> pARWIII, clone 1 </TD><TD> 10 </TD><TD> no insert band
</TD></TR><TR><TD> 2 </TD><TD> pARWIII, clone 2 </TD><TD> 10</TD><TD> no insert
</TD></TR><TR><TD> 3 </TD><TD> pARWIII, clone 3 </TD><TD> 10</TD><TD> no insert
</TD></TR><TR><TD> 4 </TD><TD> pARWIII, clone 4 </TD><TD> 10</TD><TD> no insert
</TD></TR><TR><TD> 5 </TD><TD> | 10</TD><TD> no insert
</TD><TD>-
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><P>sequencing of clones
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="91th_Labday_2011-09-10">91th Labday 2011-09-10</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="miniprep_of_Lib2_clones_for_confirmation">miniprep of Lib2 clones for confirmation</SPAN></H3><P><B>Time:</B> 2011-09-10,9:00 </P><P><B>Investigators:</B> Niels, Nadine, Katharina</P><P><B>Materials:</B></P><UL><LI> 10 overnight cultures (Lib2 clone A-J)
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> Lib2 clone A </TD><TD> 886.7
</TD></TR><TR><TD> Lib2 clone B </TD><TD> 979.7
</TD></TR><TR><TD> Lib2 clone C </TD><TD> 823.5
</TD></TR><TR><TD> Lib2 clone D </TD><TD> 815.5
</TD></TR><TR><TD> Lib2 clone E </TD><TD> 824.8
</TD></TR><TR><TD> Lib2 clone F </TD><TD> 987.7
</TD></TR><TR><TD> Lib2 clone G </TD><TD> 682.4
</TD></TR><TR><TD> Lib2 clone H </TD><TD> 683.4
</TD></TR><TR><TD> Lib2 clone I </TD><TD> 5.3
</TD></TR><TR><TD> Lib2 clone J </TD><TD> 767.5
</TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="miniprep_of_mdnA_foc2">miniprep of mdnA foc2</SPAN></H3><P><B>Time:</B> 2011-09-10,9:00 </P><P><B>Investigators:</B> Niels, Nadine, Katharina</P><P><B>Materials:</B></P><UL><LI> 1 overnight culture
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> mdnA foc2 </TD><TD> 610.0
</TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><UL><LI> give it to the screening group
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="miniprep_of_pSB1C3.2BmdnABCDE_with_and_without_T7_promotor">miniprep of pSB1C3+mdnABCDE with and without T7 promotor</SPAN></H3><P><B>Time:</B> 2011-09-10</P><P><B>Investigators:</B> Niels, Nadine, Katharina</P><P><B>Materials:</B></P><UL><LI> 6 overnight cultures
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> mdnABCDE with T7-promotor clone 1 </TD><TD> 6.1
</TD></TR><TR><TD> mdnABCDE with T7-promotor clone 2 </TD><TD> 2.7
</TD></TR><TR><TD> mdnABCDE without T7-promotor clone 1 </TD><TD> 12.1
</TD></TR><TR><TD> mdnABCDE without T7-promotor clone 2 </TD><TD> 6.7
</TD></TR><TR><TD> mdnABCDE without T7-promotor clone 3 </TD><TD> 4.9
</TD></TR><TR><TD> mdnABCDE without T7-promotor clone 4 </TD><TD> 2.9
</TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><UL><LI> doing new overnight culture and miniprep
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Counting_of_colonies_on_plates_.28pUP089.2Blib2.29_2">Counting of colonies on plates (pUP089+lib2)</SPAN></H3><P><B>Time:</B> 2011-09-09,10:00 </P><P><B>Investigators:</B> Nadine, Katharina, Niels</P><P><B>Aim:</B> count number of colonies on plates</P><P><B>Material:</B></P><UL><LI> agar plate (kan): 10x pUP089+lib2, Nic/Nad, 8.9.11, resistance: kan, from 2011-9-9, Nad/Kat</LI></UL><P><B>Results:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>plate</B></TD><TD><B># of colonies</B></TD></TR><TR><TD> 1 </TD><TD> 78
</TD></TR><TR><TD> 2 </TD><TD> 92
</TD></TR><TR><TD> 3 </TD><TD> 65
</TD></TR><TR><TD> 4 </TD><TD> 55
</TD></TR><TR><TD> 5 </TD><TD> 67
</TD></TR><TR><TD> 6 </TD><TD> 80
</TD></TR><TR><TD> 7 </TD><TD> 116
</TD></TR><TR><TD> 8 </TD><TD> 87
</TD></TR><TR><TD> 9 </TD><TD> 66
</TD></TR><TR><TD> 10 </TD><TD> 35
</TD></TR><TR><TD><B>sum</B></TD><TD><B>741</B></TD></TR><TR><TD> control </TD><TD> 11
</TD></TR></TBODY></TABLE><P><B>Conclusions:</B></P><UL><LI> 741 colonies
</LI></UL><UL><LI> wash colonies from plates
</LI></UL><UL><LI> o.n. cultures and miniprep
</LI></UL><UL><LI> combine w/ preps from 2011-9-10 (Kat)
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="New_test_digest_of_pSB1C3.2BmycN.2FC"> New test digest of pSB1C3+mycN/C </SPAN></H3><P><B>Time:</B> 2011-09-10 </P><P><B>Investigators:</B> Niels, Nadine, Katharina </P><P><B> Aim: </B> prove of Insert (mdnAmycN/C)</P><P><B>Materials:</B></P><UL><LI> pSB1C3+mycN clones J1-J5, Niels, 9.9.11
</LI></UL><UL><LI> pSB1C3+mycC clones J1-J5, Niels, 9.9.11
</LI></UL><UL><LI> ScaI, XbaI (NEB)
</LI></UL><UL><LI> Buffer 3 (NEB, 10x)
</LI></UL><UL><LI> BSA
</LI></UL><UL><LI> water
</LI></UL><P><B>Plan:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>insert</B></TD><TD><B>plasmid size in bp</B></TD><TD><B>enzymes</B></TD><TD><B>buffer</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> mdnAmycN </TD><TD> 2596 </TD><TD> ScaI, XbaI </TD><TD> 3 </TD><TD> 1024, 1570
</TD></TR><TR><TD> mdnAmycC </TD><TD> 2597 </TD><TD> ScaI, XbaI </TD><TD> 3 </TD><TD> 1024, 1570
</TD></TR></TBODY></TABLE><P><B> Digestion protocol </B></P><UL><LI>1 µl DNA
</LI></UL><UL><LI>0.5 µl ScaI
</LI></UL><UL><LI>0.5 µl XbaI
</LI></UL><UL><LI>2 µl 10x buffer
</LI></UL><UL><LI>0.3 µl BSA
</LI></UL><UL><LI>15.7 µl pure water
</LI></UL><UL><LI>total: 20µl
</LI></UL><UL><LI>37°C for 1h</LI></UL><P><B>Agarose gel</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix (1:10) </TD><TD> 12
</TD></TR><TR><TD> 1 </TD><TD> mycN clone 1 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 2 </TD><TD> mycN clone 2 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 3 </TD><TD> mycN clone 3 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 4 </TD><TD> mycN clone 4 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 5 </TD><TD> mycN clone 5 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 6 </TD><TD> mycC clone 1 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 7 </TD><TD> mycC clone 2 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 8 </TD><TD> mycC clone 3 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 9 </TD><TD> mycC clone 4 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> 10 </TD><TD> mycC clone 5 </TD><TD> 20 + 4x loading dye
</TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix (1:10) </TD><TD> 12
</TD></TR></TBODY></TABLE><P><B>Conclusions: </B></P><UL><LI> digestion did not work out in a satisfying way
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Test_Expression_Backbones:_pSB1K3_Ara.2FLac_CFP">Test Expression Backbones: pSB1K3_Ara/Lac_CFP</SPAN></H3><P><B>Time:</B> 2011-09-10,11:00 </P><P><B>Investigators:</B> Nadine, Katharina, Niels</P><P><B>Aim:</B></P><P>prove of promotors (Lac/Ara)
</P><P><B>Materials/Methods</B></P><UL><LI>start cell grow in fresh media
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>time</B></TD><TD><B>time of grow [min]</B></TD><TD><B>OD600 Ara_CFP</B></TD><TD><B>OD600 Ara_CFP control</B></TD><TD><B>OD600 Lac_CFP</B></TD><TD><B>OD600 Lac_CFP control</B></TD></TR><TR><TD> 11:00 </TD><TD> 0 </TD><TD> 0,147 </TD><TD> 0,147 </TD><TD> 0,121 </TD><TD> 0,130
</TD></TR><TR><TD> 11:30 </TD><TD> 30 </TD><TD> 0,156 </TD><TD> 0,160 </TD><TD> 0,132 </TD><TD> 0,139
</TD></TR><TR><TD> 12:00 </TD><TD> 60 </TD><TD> 0,182 </TD><TD> 0,183 </TD><TD> 0,149 </TD><TD> 0,152
</TD></TR><TR><TD> 12:30 </TD><TD> 90 </TD><TD> 0,212 </TD><TD> 0,203 </TD><TD> 0,172 </TD><TD> 0,174
</TD></TR><TR><TD> 13:00 </TD><TD> 120 </TD><TD> 0,260 </TD><TD> 0,257 </TD><TD> 0,210 </TD><TD> 0,207
</TD></TR><TR><TD> 13:30 </TD><TD> 150 </TD><TD> 0,323 </TD><TD> 0,331 </TD><TD> 0,259 </TD><TD> 0,260
</TD></TR><TR><TD> 14:00 </TD><TD> 180 </TD><TD> 0,406 </TD><TD> 0,416 </TD><TD> 0,326 </TD><TD> 0,370
</TD></TR><TR><TD> 14:30 </TD><TD> 210 </TD><TD> ind. </TD><TD> ind. </TD><TD> 0,370 </TD><TD> 0,403
</TD></TR><TR><TD> 15:00 </TD><TD> 150 </TD><TD> ind. </TD><TD> ind. </TD><TD> 0,446 </TD><TD> 0,466
</TD></TR></TBODY></TABLE><UL><LI> after inducing by IPTG/ARA - samples where measured for 2h every 15 min
</LI></UL><UL><LI> result negativ
</LI></UL><P><B>further task </B></P><UL><LI> repeat experiment
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Glycerol_stocks:_Expression_Backbones">Glycerol stocks: Expression Backbones </SPAN></H3><P><B>Time:</B> 2011-09-10,11:00 </P><P><B>Investigators:</B> Nadine, Katharina, Niels</P><P><B>Material:</B></P><UL><LI> 300 µl DNA of the following expression backbones
</LI></UL><UL><LI> pSB1A3_YFP_Ara
</LI></UL><UL><LI> pSB1A3_YFP_Lac
</LI></UL><UL><LI> pSB1K3_CFP_Ara
</LI></UL><UL><LI> pSB1K3_CFP_Lac
</LI></UL><UL><LI> pSB1K3_YFP_Lac
</LI></UL><UL><LI> mdnA Lib-foc2
</LI></UL><UL><LI> 700 µl 86 % glycerol
</LI></UL><UL><LI> store at -80°C
</LI></UL><H3 style="background-color: rgb(254, 122, 122);"><SPAN class="mw-headline" id="Oligo-Fillin_for_mdnA-library_2_.28repetition.29_and_for_Phage-display-mdnA-library_.28carrying_AgeI_restriction_site.29_2">Oligo-Fillin for mdnA-library 2 (repetition) and for Phage-display-mdnA-library (carrying AgeI restriction site)</SPAN></H3><P><B>Investigators:</B> Niels, Nadine, Katharina
</P><P><B>Time:</B> 2011-09-10</P><P><B>Aim: </B> Production of libraries carrying mutated mdnA, one for phage-display (carrying ageI restriction site)</P><P><B>Material: </B></P><UL><LI> oligos (25 mM, HPLC purified, ordered by Sigma Aldrich):
</LI></UL><UL><LI> o_mdnA_library, 76
</LI></UL><UL><LI> o_focused_library_AgeI, 85
</LI></UL><UL><LI> 10x Klenow buffer
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><UL><LI>H<SMALL>2</SMALL>O
</LI></UL><P><B>Method: </B></P><UL><LI> Reaction mix (total volume: 50 µl)
</LI></UL><UL><LI> 3 µl fw-oligonucleotide (o_mdnA-library, 76)
</LI></UL><UL><LI> 3 µl rev-oligonucleotide (o_focused_library_AgeI, 85)
</LI></UL><UL><LI> 5 µl dNTPs
</LI></UL><UL><LI> 5 µl Klenow buffer
</LI></UL><UL><LI> 34 µl H<SMALL>2</SMALL>O
</LI></UL><UL><LI> Reaction conditions
</LI></UL><UL><LI> Program: ORIGAMI1
</LI></UL><UL><LI> after finishing program ORIGAMI1:
</LI></UL><UL><LI> addition of 1 µl Klenow-fragment
</LI></UL><UL><LI> incubation 1 hour, 37°C
</LI></UL><P><B>Results: </B></P><UL><LI> hybridized oligos for production of mdnA-library
</LI></UL><P><B>Output:</B></P><UL><LI> AgeI-library, named AgeI
</LI></UL><P><B>Further tasks:</B></P><UL><LI> PCR purification
</LI></UL><UL><LI> restriction enzyme digestion using SfoI and AgeI, also of pARWIII
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_ParWIII_and_fill-in-reaction_of_Libary_2">Restriction enzyme digestion of ParWIII and fill-in-reaction of Libary 2</SPAN></H3><P><B>Investigators:</B> Katharina, Niels, Nadine, Steffi </P><P><B>Time:</B> 2011-09-10, 20:30</P><P><B>Material:</B></P><P><B> digest of Fill-in-Reaction of Libary 2 (2011-09-10) :</B></P><UL><LI>22,5 µl DNA (fill-in-product
</LI></UL><UL><LI>2,5 µl Age I
</LI></UL><UL><LI>2,0 µl Sfo I
</LI></UL><UL><LI>3µl Buffer 3
</LI></UL><UL><LI>37°C over night
</LI></UL><P>further task :
</P><UL><LI> purification by Kit
</LI></UL><P><B> digestion of ParWIII: </B></P><UL><LI> 6,5 µl template (pARWIII)
</LI></UL><UL><LI> 3 µl Buffer 4
</LI></UL><UL><LI> 2.5 µl Age I
</LI></UL><UL><LI> 2.0 µl Sfo I
</LI></UL><UL><LI> 16 µl water
</LI></UL><UL><LI> 37°C over night
</LI></UL><P>further task :
</P><UL><LI> purification by gel
</LI></UL><P><B> further tasks: </B></P><UL><LI>ligation
</LI></UL><UL><LI>transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_pSB1C3_for_ligation_with_mycN.2FC">Restriction enzyme digestion of pSB1C3 for ligation with mycN/C </SPAN></H3><P><B>Investigators:</B> Nadine, Katharina, Niels </P><P><B>Time:</B> 2011-09-10, 20:00</P><P><B>Material:</B></P><P><B>purified PCR products (2011-09-03) :</B></P><UL><LI> mycC: 4.2 ng/µl
</LI></UL><UL><LI> mycN: 15.2 ng/µl
</LI></UL><P><B>digestion of pSB1C3: </B></P><UL><LI> 6 µl DNA (235.0 ng/µl (2011-06-22))
</LI></UL><UL><LI> 3 µl Buffer 4
</LI></UL><UL><LI> 1.5 µl XbaI
</LI></UL><UL><LI> 2.0 µl PstI
</LI></UL><UL><LI> 0.5 µl BSA
</LI></UL><UL><LI> 11.5 µl water
</LI></UL><UL><LI> 37°C over night
</LI></UL><P><B> further tasks: </B></P><UL><LI>purification
</LI></UL><UL><LI>ligation
</LI></UL><UL><LI>transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_pSB1C3_for_control">Restriction enzyme digestion of pSB1C3 for control </SPAN></H3><P><B>Investigators:</B> Niels, Nadine, Katharina, Steffi </P><P><B>Time:</B> 2011-09-10, 20:30</P><P><B>Material:</B></P><P><B>pSB1C3(2011-06-22) :</B></P><UL><LI> concentration: 235,0 ng/µl
</LI></UL><P><B>digestion of pSB1C3 with Bgl I: </B></P><UL><LI> 2 µl DNA (235.0 ng/µl (2011-06-22))
</LI></UL><UL><LI> 1 µl Buffer 3
</LI></UL><UL><LI> 0.5 µl BglI
</LI></UL><UL><LI> 6.5 µl water
</LI></UL><UL><LI> total volume 10 µl
</LI></UL><UL><LI>37°C over night
</LI></UL><UL><LI> expected fragments[bp]:
</LI></UL><UL><LI> 1169
</LI></UL><UL><LI> 962
</LI></UL><UL><LI> 314
</LI></UL><UL><LI> 199
</LI></UL><P><B>digestion of pSB1C3 with SacI, TaqI: </B></P><UL><LI> 2 µl DNA (235.0 ng/µl (2011-06-22))
</LI></UL><UL><LI> 1 µl Buffer 4
</LI></UL><UL><LI> 0.5 µl SacI
</LI></UL><UL><LI> 0,5 µl TaqI
</LI></UL><UL><LI> 0,3 µl BSA
</LI></UL><UL><LI> 5,7 µl water
</LI></UL><UL><LI>total volume 10 µl
</LI></UL><UL><LI>37°C over night
</LI></UL><UL><LI>expected fragments [bg]:
</LI></UL><UL><LI>1446
</LI></UL><UL><LI>648
</LI></UL><UL><LI>548
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix (1:10) </TD><TD> 12
</TD></TR><TR><TD> 1 </TD><TD> pSB1C3 (BglI) </TD><TD> 10 + 2 loading dye
</TD></TR><TR><TD> 2 </TD><TD> pSB1C3 (SacI und TaqI) </TD><TD> 10 + 2 loading dye
</TD></TR></TBODY></TABLE><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Picking_clones_for_overnight_cultures_of_pSB1C3.2BmdnABCDE">Picking clones for overnight cultures of pSB1C3+mdnABCDE </SPAN></H3><P><B>Investigators:</B>Katharina, Steffi, Niels, Nadine</P><P><B>Time:</B> 2011-09-10 20:45</P><P><B>Materials:</B></P><UL><LI> agar plates of pSB1C3+mndABCDE (2011-09-09)
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> chloramphenicol
</LI></UL><P><B>Method:</B></P><UL><LI> picking 5 clones for pSB1C3+mdnABCDE (2011-09-09)
</LI></UL><UL><LI> incubate over night @ 37°C (200 rpm)
</LI></UL><P><B>Output:</B></P><UL><LI> 5 liquid cultures of pSB1C3+mdnABCDE without T7-promotor
</LI></UL><P><B>Further tasks:</B></P><UL><LI> miniprep
</LI></UL><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparing_overnight_cultures_of_expression_backbone-clones_2">Preparing overnight cultures of expression backbone-clones</SPAN></H3><P><B>Investigators:</B>Katharina, Niels, Nadine, Steffi</P><P><B>Time:</B> 2011-09-10 19:30</P><P><B>Materials/Method:</B></P><UL><LI> liquid culture :
</LI></UL><UL><LI> pSB1K3_CFP_Ara
</LI></UL><UL><LI> pSB1K3_CFP_Lac
</LI></UL><UL><LI> 10 ml LB medium
</LI></UL><UL><LI> 10 µl kanamycin
</LI></UL><P><B>Output:</B></P><UL><LI> 2 liquid cultures of expression CFP backbones with different promotors
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_mdnABCDE_.2B_pSB1C3">Transformation of mdnABCDE + pSB1C3</SPAN></H3><P><B>Time:</B> 2011-09-10 21:30</P><P><B>Investigators:</B>Niels, Nadine, Katharina, Steffi</P><P><B>Aim:</B> Transformation of <I>E. coli</I> cells with mdnABCDE + pSB1C3 (with and without T7) - ligation </P><P><B>Materials:</B></P><UL><LI> competent <I>E. coli</I> cells (XL1-Blue, 2011-08-29)
</LI></UL><UL><LI> ligation products: mdnABCDE + pSB1C3
</LI></UL><P><B>Method:</B></P><UL><LI>addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI>incubation 20 min on ice,
</LI></UL><UL><LI>heat shock 90 sec at 42°C,
</LI></UL><UL><LI>incubation 10 min on ice,
</LI></UL><UL><LI>addition of 750 µl LB medium,
</LI></UL><UL><LI>incubation at 37 °C shaking(750 rpm) for 60 min,
</LI></UL><UL><LI>centrifuge 30 min at 2000 x g
</LI></UL><UL><LI>discard the supernatant (till ~100µl media)
</LI></UL><UL><LI>plating on LB plate with appropriate antibiotic (Cm)
</LI></UL><UL><LI>storage over night at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><P><B>Output:</B></P><P>to be continued....
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Colony_PCR_of_plated_TEV_and_14_3C_containing_colonies"> Colony PCR of plated TEV and 14_3C containing colonies </SPAN></H3><P><B>Investigators:</B> Stefan, Paul, Sebastian
</P><P><B>Aim:</B></P><P>get dna for BioBricks</P><P><B>Used Primer:</B></P><P>Tev:
</P><P>f_TEV_iGEM, r_TEV_iGEM_BamHI
</P><P>14_3C:
</P><P>f_14_3C_iGEM, r_14_3C_iGem:BamHI
</P><P><B> Method: </B></P><UL><LI>2,5µl of each primer (10µM) = 5µl
</LI></UL><UL><LI>5µl 10y polymerase buffer
</LI></UL><UL><LI>2µl 25mM MgCl2
</LI></UL><UL><LI>1µl dNTP mix
</LI></UL><UL><LI>0,5µl Taq-Polymerase (GenAxxon)
</LI></UL><UL><LI>36,5 µl H2O
</LI></UL><P>=50µl
</P><P>Colonies were picked with a 20µl tip, dipped into PCR batch, and then plunged into 1ml LB to grew cells from picked colony (from these 1ml batches overnight cultures will be inoculated in case of positive clones).
</P><UL><LI> 10 colonies were picked for TEV and 10 colonies for 14_3C protease (see entry from 01.09.2011) = 20 PCR batches.
</LI></UL><P><B>PCR Program:</B></P><P>Initial denat = 3min 94°C
</P><P>25x
</P><P>denat: 2min 10sec 94°C
</P><P>anneal: 2min 10sec 70°C
</P><P>extend: 1min 72°C
</P><P>final extend: 10min 72°C
</P><UL><LI> The PCR products were resolved on a 1% analytical agarose gel
</LI></UL><P><B>Results: </B></P><P>no bands!
</P><P><B>Further Tasks: </B></P><P>repeat it!
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_TEV.2C_14_3C.2C_AraC_and_different_Vectors_.28pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL.2C_pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL_and_pSB1C3.29"> Digest of TEV, 14_3C, AraC and different Vectors (pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL, pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL and pSB1C3) </SPAN></H3><P><B>Investigator:</B> Nadine, Sebastian</P><P><B>Material:</B></P><P>Enzymes: (all pruchased from NEB)</P><UL><LI> HindIII
</LI></UL><UL><LI> BamHI
</LI></UL><UL><LI> NgoMIV
</LI></UL><UL><LI> XbaI
</LI></UL><UL><LI> PstI
</LI></UL><P>DNA-Fragments:</P><UL><LI> mutated TEV protease
</LI></UL><UL><LI> mutated 14_3C protease
</LI></UL><UL><LI> AraC
</LI></UL><UL><LI> pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL
</LI></UL><UL><LI>pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL
</LI></UL><UL><LI> pSB1C3
</LI></UL><P>NEBuffer 4, 3 and 1</P><P>BSA from NEB</P><P><B>Method:</B></P><P>All digestion batches got a total volume of 10µl and were incubated for 2 h at 37°C:
</P><UL><LI>1µl Buffer
</LI></UL><UL><LI>1µl each Enzyme
</LI></UL><UL><LI>0,1 µl BSA if required
</LI></UL><UL><LI>7 or 6,9 µl DNA
</LI></UL><P>Both protease (TEV and 14_3C) got digested with BamHI+NgoMIV or NgoMIV+ PstI, AraC was digested with NgoMIV+HindIII or NgoMIV+XbaI. Both vector which containing the cleavage site for the Protease got digested with HindIII+BamHI, the vector pSB1C3 was digested with XbaI and PstI.
</P><P><B>Output:</B></P><P>Digested fragements:
</P><UL><LI>NgoMIV_TEV_BamHI
</LI></UL><UL><LI>NgoMIV_TEV_PstI
</LI></UL><UL><LI>NgoMIV_14_3C_BamHI
</LI></UL><UL><LI>NgoMIV_14_3C_PstI
</LI></UL><UL><LI>HindIII_AraC_NgoMIV
</LI></UL><UL><LI>XbaI_AraC_NgoMIV
</LI></UL><UL><LI>BamHI_ pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL_HindIII
</LI></UL><UL><LI>BamHI_ pJS354_ssTorA_XhoI_CS-14_3C_NheI_blaFL_HindIII
</LI></UL><UL><LI>PstI_pSB1C3_XbaI
</LI></UL><P><B>Further Tasks:</B></P><P>Clean up of all fragments via agarose gel and extraction of digested fragment from the agarose gel, ligation of fragments and teransformation into <I>E. coli</I> XL1 blue.
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Gelextraction_of_digested_fragements.2C_ligation_and_transfomration_of_ligation_products_into_E._coli_XL1_blue"> Gelextraction of digested fragements, ligation and transfomration of ligation products into <I>E. coli</I> XL1 blue </SPAN></H3><P><B>Investigator:</B></P><P>Sebastian, Sascha, Stefan</P><P><B>Aim:</B></P><P>Getting digested fragments purified, to get lost of small DNA fragments, we don't need.</P><P><B>Materials:</B></P><P>Digested fragements:
</P><UL><LI>NgoMIV_TEV_BamHI
</LI></UL><UL><LI>NgoMIV_TEV_PstI
</LI></UL><UL><LI>NgoMIV_14_3C_BamHI
</LI></UL><UL><LI>NgoMIV_14_3C_PstI
</LI></UL><UL><LI>HindIII_AraC_NgoMIV
</LI></UL><UL><LI>XbaI_AraC_NgoMIV
</LI></UL><UL><LI>BamHI_ pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL_HindIII
</LI></UL><UL><LI>BamHI_ pJS354_ssTorA_XhoI_CS-14_3C_NheI_blaFL_HindIII
</LI></UL><UL><LI>PstI_pSB1C3_XbaI
</LI></UL><P><B>Methode:</B></P><P>Seperate digestion batches over an 1% agarose gel, cut of the bands with the right size and perform a gel purification with Nucleo spin II kit from Macherey-Nagel.
</P><P><B>Output.</B></P><P>Digested and purified fragments of:</P><UL><LI>NgoMIV_TEV_BamHI
</LI></UL><UL><LI>NgoMIV_TEV_PstI
</LI></UL><UL><LI>NgoMIV_14_3C_BamHI
</LI></UL><UL><LI>NgoMIV_14_3C_PstI
</LI></UL><UL><LI>HindIII_AraC_NgoMIV
</LI></UL><UL><LI>XbaI_AraC_NgoMIV
</LI></UL><UL><LI>BamHI_ pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL_HindIII
</LI></UL><UL><LI>BamHI_ pJS354_ssTorA_XhoI_CS-14_3C_NheI_blaFL_HindIII
</LI></UL><UL><LI>PstI_pSB1C3_XbaI
</LI></UL><P><B>Further tasks:</B></P><P>Ligation of fragments into digested vector and transformation of <I>E. coli</I> XL1 blue with ligation products. Picking clones, performing a colony PCR, overnight cultures from all positive clones and creation of an glycerol stock culture and miniprep of those positive plasmids for sequencing.
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_digested_fragments_of_TEV_protease.2C_14_3C_protease_and_different_vectors"> Ligation of digested fragments of TEV protease, 14_3C protease and different vectors </SPAN></H3><P><B>Investigators:</B> Stefan, Sascha, Paul
</P><P><B>Materials:</B></P><P>Fragments:</P><UL><LI>NgoMIV_TEV_BamHI
</LI></UL><UL><LI>NgoMIV_TEV_PstI
</LI></UL><UL><LI>NgoMIV_14_3C_BamHI
</LI></UL><UL><LI>NgoMIV_14_3C_PstI
</LI></UL><UL><LI>HindIII_AraC_NgoMIV
</LI></UL><UL><LI>XbaI_AraC_NgoMIV
</LI></UL><UL><LI>BamHI_ pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL_HindIII
</LI></UL><UL><LI>BamHI_ pJS354_ssTorA_XhoI_CS-14_3C_NheI_blaFL_HindIII
</LI></UL><UL><LI>PstI_pSB1C3_XbaI
</LI></UL><P>QuickLigase purchased by Fermentas</P><P>QuickLigase 10x Buffer purchased by Fermentas</P><P><B>Methodes:</B></P><P>Standard ligation protocol from Fermantas for QuickLigase Kit. Ratio between vector and both fragments (protease and AraC-induction system) where 3:1:1.</P><P><I>Reaction batches:</I></P><P>Every batch contains 1µl QuickLigase 10x Buffer, and 0,5 µl QuickLigase (both by Fermentas).
</P><UL><LI> Batch 1:
</LI></UL><UL><LI>NgoMIV_TEV_BamHI
</LI></UL><UL><LI>HindIII_AraC_NgoMIV
</LI></UL><UL><LI>BamHI_ pJS354_ssTorA_XhoI_CS-TEV_NheI_blaFL_HindIII
</LI></UL><UL><LI> Batch 2:
</LI></UL><UL><LI>NgoMIV_14_3C_BamHI
</LI></UL><UL><LI>HindIII_AraC_NgoMIV
</LI></UL><UL><LI>BamHI_ pJS354_ssTorA_XhoI_CS-14_3C_NheI_blaFL_HindIII
</LI></UL><UL><LI>Batch 3:
</LI></UL><UL><LI>NgoMIV_TEV_PstI
</LI></UL><UL><LI>XbaI_AraC_NgoMIV
</LI></UL><UL><LI>PstI_pSB1C3_XbaI
</LI></UL><UL><LI>Batch 4:
</LI></UL><UL><LI>NgoMIV_14_3C_PstI
</LI></UL><UL><LI>XbaI_AraC_NgoMIV
</LI></UL><UL><LI>PstI_pSB1C3_XbaI
</LI></UL><P>Every batch was incubated for 10 min at 37°C. After 10 min, all batches where put on ice.
</P><P><B>Output:</B></P><UL><LI> 4 different ligation batches
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Transformation of <I>E. coli</I> XL1 blue with ligation products, colony PCR to get to know, which clones are positive and for all positive clones: mini-prep of plasmid DNA, sequencing and preparation of glycerol stock clutures.
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Transfromation_of_E._coli_XL1_blue_with_ligation_products"> Transfromation of <I>E. coli</I> XL1 blue with ligation products </SPAN></H3><P><B>Investigators:</B> Sascha, Paul, Sebastian</P><P><B>Aim:</B></P><P>transformation of <I>E. coli</I> XL1 blue with ligation batches[<A href="http://www.syntbio.net/iGEM/wiki2011/index.php/Labjournal_september#Ligation_of_digested_fragments_of_TEV_protease.2C_14_3C_protease_and_different_vectors" class="external autonumber" rel="nofollow">[18]</A>]
</P><P><B>Materials:</B></P><UL><LI>4 different ligation batches [<A href="http://www.syntbio.net/iGEM/wiki2011/index.php/Labjournal_september#Ligation_of_digested_fragments_of_TEV_protease.2C_14_3C_protease_and_different_vectors" class="external autonumber" rel="nofollow">[19]</A>]
</LI></UL><UL><LI>competent <I>E. coli</I> XL1 blue cells
</LI></UL><P><B>Method:</B></P><P>Standard transformation protocol using heat shock.
</P><P><B>Output:</B></P><P>4 different <I>E. coli</I> cultures transformed with different ligation batches.
</P><P><B>Further tasks:</B></P><P>plating of the transformed <I>E. coli</I> cultures on LB agar plates containing chloramphenicol.
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Plating_of_transformed_E._coli_XL1_blue"> Plating of transformed <I>E. coli</I> XL1 blue </SPAN></H3><P><B>Investigators:</B> Stefan, Paul, Sebastian
</P><P><B>Aim:</B></P><P>getting single positive <I>E. coli</I> XL1 blue colonies transformed with the different ligation batches [<A href="http://www.syntbio.net/iGEM/wiki2011/index.php/Labjournal_september#Ligation_of_digested_fragments_of_TEV_protease.2C_14_3C_protease_and_different_vectors" class="external autonumber" rel="nofollow">[20]</A>]</P><P><B>Method:</B></P><P>100 µl of the transformed cultures where plated out on lB agar plates containing chloramphenicol, the rest of the transformed cultures where centrifuge at 4000 g for 3 min. The supernatant was discarded and the pellet re suspended in 100 µl LB media, which was also plated out on a new LB agar plate containing chloramphenicol. All agar plates where incubated over night at 37°C.
</P><P><B>Further tasks:</B></P><P>picking clones for colony PCR and mini prep of plasmid DNA, also establishing of glycerol stock cultures and preparing plasmids for sequencing.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/September_part_1">http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/September_part_1</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Labjournal/September_part_1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Labjournal/September_part_1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/September_part_1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/September_part_1&amp;oldid=156095" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></TBODY></TABLE></DIV></DIV></DIV></DIV></BODY></HTML>