<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Labjournal_May"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Labjournal/May">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Labjournal/May&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/May&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/May&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Labjournal/May" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Labjournal/May</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team">Team</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Members">Members</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Project</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Summary</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Highlights">Highlights</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Microviridin">     Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Phage_Display">     Phage Display</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#In_Vivo_Selection">     In Vivo Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Modeling">     Modeling</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Microviridin">Details Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">Details Phage</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">Details Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Modeling">Details Modeling</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">BioBricks</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/BioBricks">Details BioBricks</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Methods">Methods</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Meetings">Meetings</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics">Safety &amp; Ethics</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Assesment">Safety Assessment</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Questions">Safety Questions</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Ethics_Seminar">Ethics Seminar</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Survey">Survey</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Potsdam_Bioware_at_the_German_parliament">Parliament Visit</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Statement_of_the_German_government">     Statement</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Outreach">Outreach</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software">Software</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#BioLog_App">BioLog App</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#Download">Download</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Sponsors">Sponsors</A></DT></DL></LI></UL></DIV><DIV class="pageContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#1st_Labday_12.05.2011"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">1st Labday 12.05.2011</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparative_DNA_Electrophoresis.2C_Test_Experiment_629.2C_654_digest"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Preparative DNA Electrophoresis, Test Experiment 629, 654 digest</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_629_vec_frag.2C_654_insert"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Ligation 629 vec frag, 654 insert</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_ligation_629_vec_frag_654_insert"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Transformation of ligation 629_vec_frag 654_insert</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#2nd_Labday_2011-05-13"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">2nd Labday 2011-05-13</SPAN></A></LI><LI class="toclevel-2"><A href="#Design_of_primers_and_annotation_of_pARW071"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Design of primers and annotation of pARW071</SPAN></A></LI><LI class="toclevel-1"><A href="#3rd_Labday_2011-05-23"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">3rd Labday 2011-05-23</SPAN></A></LI><LI class="toclevel-2"><A href="#Overnight_cultures_of_pARW071_and_pARW089_carrying_cells"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Overnight cultures of pARW071 and pARW089 carrying cells</SPAN></A></LI><LI class="toclevel-1"><A href="#4th_Labday_2011-05-24"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">4th Labday 2011-05-24</SPAN></A></LI><LI class="toclevel-2"><A href="#Midiprep_of_pARW071_and_pARW089_carrying_cells"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Midiprep of pARW071 and pARW089 carrying cells</SPAN></A></LI><LI class="toclevel-1"><A href="#5th_Labday_2011-05-25"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">5th Labday 2011-05-25</SPAN></A></LI><LI class="toclevel-2"><A href="#Overnight_culture_of_E.coli_cells_carrying_MRC_and_N298_fosmids"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Overnight culture of E.coli cells carrying MRC and N298 fosmids</SPAN></A></LI><LI class="toclevel-1"><A href="#6th_Labday_2011-05-26"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">6th Labday 2011-05-26</SPAN></A></LI><LI class="toclevel-2"><A href="#Glycerol_stocks_of_E.coli_cells_carrying_MRC_and_N298_fosmids"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Glycerol stocks of E.coli cells carrying MRC and N298 fosmids</SPAN></A></LI><LI class="toclevel-1"><A href="#7th_Labday_2011-05-27"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">7th Labday 2011-05-27</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation_of_samples_for_sequencing"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Preparation of samples for sequencing</SPAN></A></LI><LI class="toclevel-1"><A href="#8th_Labday_2011-05-30"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">8th Labday 2011-05-30</SPAN></A></LI><LI class="toclevel-2"><A href="#Searching_for_promotor_region_and_primer_design"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Searching for promotor region and primer design</SPAN></A></LI><LI class="toclevel-1"><A href="#9th_Labday_2011-05-31"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">9th Labday 2011-05-31</SPAN></A></LI><LI class="toclevel-2"><A href="#Analysis_of_sequencing_results_and_Primer_design"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Analysis of sequencing results and Primer design</SPAN></A></LI></TD></TR></TBODY></TABLE><H2 style="background-color: rgb(50, 200, 0);"><SPAN class="mw-headline" id="1st_Labday_12.05.2011"> 1st Labday 12.05.2011</SPAN></H2><H3 style="background-color: rgb(51, 204, 255); font-weight: bold;"><SPAN class="mw-headline" id="Preparative_DNA_Electrophoresis.2C_Test_Experiment_629.2C_654_digest">Preparative DNA Electrophoresis, Test Experiment 629, 654 digest</SPAN></H3><P><B>Investigators:</B> Sascha, Jessica, Steffi, Sandrina, Nadine; <SMALL>introduction by Kristian, Tobias, Tim</SMALL></P><P><B>Aim:</B> get used to the lab</P><P><B>Time:</B> 13:40</P><P><B>Method:</B></P><P>Samples digested with AscI and NheI from Tim</P><P>vector: 629 source: Tobias; prep-type: ? ; prep-date: ; clone-no: ?</P><P>insert: 654 source: Tobias; prep-type: ? ; prep-date: ; clone-no: ?</P><P>Electrophoresis</P><P>sample preparation: 30 µl 629 (digested) + 6 µl 6x loading dye solution, 30 µl 654 (digested) + 6 µl 6x loading dye</P><P>0,5 g Agarose, 50 ml TAE (1%), 2 µl GelRed (3-6µl), at 100 Volt,running time: 30 minutes</P><P>marker: GeneRuler ladder mix (Fermentas)</P><TABLE border="1" class="wikitable"><TBODY><TR><TH>lane</TH><TH>Sample </TH><TH>Sample/µl] </TH><TH>Expected size (bp) </TH><TH>approx. size </TH></TR><TR><TD style="vertical-align: top;">1</TD><TD style="vertical-align: top;">marker</TD><TD style="vertical-align: top;">4 µl</TD></TR><TR><TD style="vertical-align: top;">3</TD><TD>629 </TD><TD>5 µl </TD><TD>4060, 140 </TD><TD> 6000, 3000, 150</TD></TR><TR><TD style="vertical-align: top;">4</TD><TD>629 </TD><TD>31 µl </TD><TD>4060, 140</TD><TD>6000, 3000, 150</TD></TR><TR><TD style="vertical-align: top;">6</TD><TD>654 </TD><TD>5 µl </TD><TD> 4140, 1268</TD><TD>6000, 4000, 1700</TD></TR><TR><TD style="vertical-align: top;">7</TD><TD>654 </TD><TD>31 µl </TD><TD>4140, 1268</TD><TD> 6000, 4000, 1700 (2 bands?)</TD></TR></TBODY></TABLE><P>            </P><P>Gel Extraction</P><P>DNA excision: lane 4: 6000 bp band and 3000 bp band; lane 7: 1700 bp band</P><P>DNA purification: with Qiagen Gel Extraction Kit according to Qiagen manual (incl. isopropanol step)</P><P><B>Result:</B> size estimation by eye;</P><P>lanes 3+4: plasmid was digested, small band has the expected size, two upper bands with unexpected size;</P><P>lanes 6+7: plasmid was digested; 6000 bp unexpected; 4000 bp band might correspond to expected 4140 bp band, 1700 bp band above expected 1268 bp</P><P><B>Conclusions:</B></P><UL><LI>exact vector fragment unknown, therefore 6000 bp band and 3000 bp

band were excised from lane 4, lane 7: 1700 bp band was excised; </LI><LI>because of unexpected sizes marker should be compared with

another marker, plasmids should be digested to yield several fragments

for size control, maybe heat inactivation might help to resolve higher bands</LI></UL><P><B>Output:</B> three plasmid fragments with 6000 bp (V1), 3000 bp (V2) and 1700 bp (I1) in 1,5 ml eppis,
</P><P>stored at -20 °C, box with fragments</P><H3 style="background-color: rgb(51, 204, 255); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_629_vec_frag.2C_654_insert">Ligation 629 vec frag, 654 insert</SPAN></H3><P><B>Investigators:</B> Sascha, Jessica, Steffi, Sandrina, Nadine</P><P><B>Aim:</B> create a new vector for Tim</P><P><B>Time:</B> 16:30</P><P><B>Method:</B></P><P>vector: 629 source: Tobias; prep-type: ; prep-date: 12.4.2011; clone-no: ?</P><P>insert: 654 source: Tobias; prep-type: ; prep-date: 12.4.2011; clone-no: ?</P><P>Ligation: 2 µl vector, 2 µl insert, 1 µl quick ligase, 5 µl 2x quick ligase NEB, 10 min</P><P><B>Output:</B> plasmid: pNoname_Tim_V1, pNoname_Tim_V2, 2 eppis labeled 'ligation', handed over to Tim</P><H3 style="background-color: rgb(51, 204, 255);"><SPAN class="mw-headline" id="Transformation_of_ligation_629_vec_frag_654_insert">Transformation of ligation 629_vec_frag 654_insert</SPAN></H3><P><B>Investigators:</B> Sascha, Jessica, Steffi, Sandrina, Nadine</P><P><B>Aim:</B>amplification of previous ligation</P><P><B>Time:</B> 16:40</P><P><B>Method:</B></P><P>ligation-samples from 12.5.11;</P><P>pNoname_Tim_V1 in BL21 (CaCl2 chemical competent, source: Tobias, date: unknown) and XL1 (CaCl2 chemical competent,
</P><P>source: Tobias, date: unknown)</P><P>pNoname_Tim_V2 in BL21 (CaCl2 chemical competent, source: Tobias, date: unknown) and XL1 (CaCl2 chemical competent, source: Tobias, date: unknown)</P><P>protocol:
</P><P>addition of 2 µl ligation reaction to cells in 1.5 ml Eppi,</P><P>incubation 30 min on ice,</P><P>heat shock 45 sec at 42°C,</P><P>addition of 750 µl SOC medium,</P><P>incubation at 37 °C for 45 min in Eppendorf thermomixer at 750 rpm,</P><P>centrifugation at 2000xg; for 3 min,</P><P>decandation of supernatant,</P><P>resuspension of pellet in approx. 150 µl,</P><P>plating on LB medium with 1,5 % agar, 100 µg/ml ampicillin,</P><P>storage over night at 37°C</P><P><B>Results and Conclusion (13.5.2011): </B></P><P>pNoname_Tim_V2 in BL21: 4 distinct colonies</P><P>test ligation is missing, not enough clones, pick one clone for test
</P><P>digest, repeat experiment</P><P><B>Output</B> four agar plates given to Tim (only one with clones)</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="2nd_Labday_2011-05-13">2nd Labday 2011-05-13</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Design_of_primers_and_annotation_of_pARW071">Design of primers and annotation of pARW071</SPAN></H3><P><B>Investigators:</B> Jessica, Katharina, Hanna, Nadja, Nicole </P><P><B>Aim:</B> Design of primers for sequencing pARW071, annotation of pARW071</P><P><B>Time:</B> 2011-05-12, 4:00 pm - 8:00 pm and 2011-05-13, 11:15 am - 12:30 am </P><P><B>Materials/ Software:</B></P><UL><LI> Geneious Pro 5.1.7 </LI></UL><UL><LI> Oligo Calc: Oligonucleotide Properties Calculator (<A href="http://www.basic.northwestern.edu/biotools/oligocalc.html" class="external free" rel="nofollow">http://www.basic.northwestern.edu/biotools/oligocalc.html</A>) </LI></UL><P><B>Method:</B></P><P>1. Annotation of pARW071 </P><UL><LI> Alignment of pARW071 and pDrive_Qiagen (PCR cloning vector)</LI></UL><UL><LI> Alignment of pARW071 and AM943877 (Microcystis aeruginosa NIES-298 microviridin biosynthetic gene cluster, strain NIES-298)</LI></UL><P>2. Primer design
</P><UL><LI> melting temperature based on 4+2 method, sum has to be 56°C </LI></UL><UL><LI> check specific binding using dotplot feature (parameters: low sensitivity: 10, tick at reverse complementary) of Genious Pro 5.7.1 </LI></UL><UL><LI> Check self-complementarity and melting temperature using Oligo Calc </LI></UL><P><B>Results:</B></P><P>Primer sequences
</P><UL><LI> sf_bb_1_1      ACAATTCACTGGCCGTCG </LI></UL><UL><LI> sf_bb_2_1      AAGATCCTTGAGAGTTTTCG </LI></UL><UL><LI> sf_bb_3_1_n   AAATTCCAACATGGATGCTG </LI></UL><UL><LI> sf_bb_4_1      TAATACGACTCACTATAGGG </LI></UL><UL><LI> sf_mdn_1_1    ACCAACAAATACAGAGCCAG </LI></UL><UL><LI> sf_mdna_1     GGTTATTGCAGGTGGTTAG </LI></UL><UL><LI> sf_mdnc_1     ATCGCTTCCCTACAGAGG </LI></UL><UL><LI> sf_mdnd_1     CTCCTAACCCCACC </LI></UL><UL><LI> sf_mdne_1_n  AAGGTTTTTGGGTCTTTATCG </LI></UL><UL><LI> sf_mdne_2_1  CGATAAATTCTCCACAATTACC </LI></UL><UL><LI> sf_mdne_3_1  AATTGAGAAAAAGCAAATAACC </LI></UL><UL><LI> sf_mndb_1     GTTATTACAACTTCGTTTGG </LI></UL><P>pARW071 with primers and annotations </P><P><A href="/File:UP_pARW071_annotation_2011-05-12.jpg" title="File:UP pARW071 annotation 2011-05-12.jpg">400px</A></P><P><B>Conclusion:</B></P><P>Primer ordering by Sigma-Aldrich </P><P><B>Further tasks</B></P><UL><LI> Midi Prep of <I>E. coli</I> carrying pARW071 and pARW089</LI></UL><UL><LI> Sequencing of pARW071
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="3rd_Labday_2011-05-23">3rd Labday 2011-05-23</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Overnight_cultures_of_pARW071_and_pARW089_carrying_cells">Overnight cultures of pARW071 and pARW089 carrying cells</SPAN></H3><P><B>Investigators:</B> Jessica, Nadja, Nicole </P><P><B>Time:</B> 2011-05-23, 5:30 pm - 7:30 pm </P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Ampicillin 100 µg/ ml</LI></UL><P>Clones: </P><P>1. <B>pARW071</B> pDrive-mdnA-E, N843(Top10 cells) - clone 1; <B>Source:</B> Elke Dittmann; <B>Date of plate:</B></P><P>2. <B>pARW089</B> pDrive-mdn cluster with Aat II/ Ehe I restriction sites before and afterwards the mdnA gene (XL1O gold cells) - clone 1; <B>Source:</B> Elke Dittmann; <B>Date of plate:</B></P><P><B>Method:</B></P><P>Inoculation of one clone each in 100 ml LB medium containing ampicillin</P><P>shaking over night at 37°C, 300 rpm, approx. 19 hours </P><P><B>Further tasks:</B></P><UL><LI> Midiprep </LI></UL><UL><LI> Sequencing of complete pARW071
</LI></UL><UL><LI> Sequencing of mdnA in pARW089
</LI></UL><P><B>Output:</B>2 over night cultures pARW071 in Top10 cells, pARW089 in XL10 cells
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="4th_Labday_2011-05-24">4th Labday 2011-05-24</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Midiprep_of_pARW071_and_pARW089_carrying_cells">Midiprep of pARW071 and pARW089 carrying cells</SPAN></H3><P><B>Investigators:</B> Nadja, Nicole </P><P><B>Aim:</B></P><UL><LI> Midiprep of pARW071 and pARW089 carrying cells </LI></UL><UL><LI> Glycerol stocks of <I>E. coli</I> cells carrying pARW071 and pARW089 </LI></UL><P><B>Time:</B> 2011-05-24, 3:00 pm - 7:00 pm </P><P><B>Materials:</B></P><UL><LI> Qiagen Plasmid Plus Midi Kit </LI></UL><UL><LI> Glycerol</LI></UL><UL><LI> over night cultures of pARW089 and pARW071 carrying cells
</LI></UL><P><B>Method:</B></P><P>1. Glycerol stocks </P><UL><LI> 700 µl of cell suspension </LI></UL><UL><LI> 300 µl glycerol </LI></UL><UL><LI> gently mixed and frozen by -80°C </LI></UL><P>2. Midiprep </P><UL><LI> using Qiagen Plasmid Plus Midi Kit Quick-Start Protocol, <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=UP_QIAGEN_Plasmid_Plus_Midi_Kit.pdf" class="new" title="UP QIAGEN Plasmid Plus Midi Kit.pdf">Media: UP_QIAGEN_Plasmid_Plus_Midi_Kit.pdf</A></LI></UL><P><B>Output:</B> DNA Eppi Tube labelled with &quot;1&quot; for pARW071 and &quot;2&quot; for pARW089, storage in pink DNA box
</P><P><B>Further tasks:</B></P><UL><LI> DNA concentration determination </LI></UL><UL><LI> Sequencing of complete pARW071
</LI></UL><UL><LI> Sequencing of mdnA in pARW089
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="5th_Labday_2011-05-25">5th Labday 2011-05-25</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Overnight_culture_of_E.coli_cells_carrying_MRC_and_N298_fosmids">Overnight culture of E.coli cells carrying MRC and N298 fosmids </SPAN></H3><P><B>Investigators:</B> Nadja, Nicole </P><P><B>Time:</B> 2011-05-25, 4:00 pm - 7:00 pm </P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> 12.5 µg/ml Chloramphenicol </LI></UL><P><B>Method:</B></P><P>One clone in approx. 2 ml LB medium </P><UL><LI> N298 fosmid (mdnB) Epi cells - plate/clone 1 </LI></UL><UL><LI> N298 fosmid (mdnB) Epi cells - plate/clone 2 </LI></UL><UL><LI> MRC fosmid (mdnJ variants) Epi cells - clone 1 </LI></UL><UL><LI> MRC fosmid (mdnJ variants) Epi cells - clone 1 </LI></UL><P>shaking over night at 37°C, 300 rpm
</P><P><B>Output:</B></P><UL><LI>over night cultures
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Glycerol stocks of each clone </LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="6th_Labday_2011-05-26">6th Labday 2011-05-26</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Glycerol_stocks_of_E.coli_cells_carrying_MRC_and_N298_fosmids">Glycerol stocks of E.coli cells carrying MRC and N298 fosmids </SPAN></H3><P><B>Investigators:</B> Nadja, Nicole </P><P><B>Aim:</B> Generating stocks for further research</P><P><B>Time:</B> 2011-05-26,17:45-18:15</P><P><B>Materials:</B></P><UL><LI> Glycerol </LI></UL><UL><LI> N298 fosmid (mdnB) Epi cells - plate/clone 1 </LI></UL><UL><LI> MRC fosmid (mdnJ variants) Epi cells - clone 1 </LI></UL><P><B>Method:</B></P><UL><LI> 700 µl of cell suspension </LI></UL><UL><LI> 300 µl glycerol </LI></UL><UL><LI> mixed gently and frozen by -80°C </LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="7th_Labday_2011-05-27">7th Labday 2011-05-27</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparation_of_samples_for_sequencing">Preparation of samples for sequencing</SPAN></H3><P><B>Investigators:</B> Jessica, Nicole, Nadja </P><P><B>Aim:</B> Prepare mixtures for sequencing</P><P><B>Time:</B> 2011-05-27,13:45-15:45</P><P><B>Materials:</B></P><UL><LI> Nanodrop </LI></UL><UL><LI> Primers ordered previously by Sigma Aldrich --&gt; see labday 2</LI></UL><UL><LI> Isolated DNA of pARW071 and pARW089 </LI></UL><P><B>Method:</B></P><P>1. Preparation of primer stocks </P><UL><LI> Primers arrived dry </LI></UL><UL><LI> Adding ddH<SMALL>2</SMALL>O (based on manufacture's sheet)</LI></UL><UL><LI> Reached concentration 100 µM </LI></UL><P>2. DNA concentration determination </P><UL><LI> Using nanodrop </LI></UL><P>3. Preparing sequencing mixtures </P><P>a. Dilution of DNA and primers </P><UL><LI><B>Primers:</B> needed concentration = 10 pmol/ µl, volume needed: 20 µl</LI></UL><UL><LI><B>DNA:</B> needed concentration = 60 ng/ µl, volume needed: 20 µl per 8 sequencing reactions </LI></UL><P>b. Preparation of sequencing tubes for sequencing by GATC </P><UL><LI> For each tube one barcode, green = DNA, yellow = primer </LI></UL><UL><LI> 2 tubes with DNA (pARW071 and pARW089) </LI></UL><UL><LI> 12 tubes with primers (sf_mdna_1 is used for pARW071 and pARW089)</LI></UL><P><B>Results:</B></P><P>1. Concentration of plasmid DNA </P><UL><LI> pARW071: 626.7 ng/ µl </LI></UL><UL><LI> pARW089: 857.3 ng/ µl </LI></UL><P>2. Sequencing ordered by GATC </P><P><B>Further tasks:</B></P><UL><LI> Analysis of sequencing of pARW071 and pARW089 (hopefully the results will arrive on monday) </LI></UL><UL><LI> Testing PCR of the complete pARW071 and pARW089, respectively </LI></UL><UL><LI> Inserting myc-tag </LI></UL><UL><LI> Cloning mdnA in phagemid </LI></UL><UL><LI> Cloning gene 3 in pARW089
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="8th_Labday_2011-05-30">8th Labday 2011-05-30</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Searching_for_promotor_region_and_primer_design">Searching for promotor region and primer design</SPAN></H3><P><B>Investigators:</B> Jessica, Nicole, Sabine, Sandrina, Steffi, Nadja </P><P><B>Aim:</B> Find the putative mdn promoters and design a primer (r_bb_1) which will be used for a PCR of the whole mdn</P><P><B>Time:</B> 2011-05-30,15:00-17:50</P><P><B>Materials:</B></P><UL><LI> Geneious Pro 5.1.7
</LI></UL><UL><LI> Oligo Calc: Oligonucleotide Properties Calculator (<A href="http://www.basic.northwestern.edu/biotools/oligocalc.html" class="external free" rel="nofollow">http://www.basic.northwestern.edu/biotools/oligocalc.html</A>)</LI></UL><UL><LI> Some online tools to search for the promotors:</LI></UL><P>1. <A href="http://linux1.softberry.com/berry.phtml?topic=bprom&amp;group=programs&amp;subgroup=gfindb" class="external free" rel="nofollow">http://linux1.softberry.com/berry.phtml?topic=bprom&amp;group=programs&amp;subgroup=gfindb</A></P><P>2. <A href="http://bioinformatics.biol.rug.nl/websoftware/ppp/ppp_start.php" class="external free" rel="nofollow">http://bioinformatics.biol.rug.nl/websoftware/ppp/ppp_start.php</A></P><P>3. <A href="http://genie.dartmouth.edu/scope/results.php?jobid=111290" class="external free" rel="nofollow">http://genie.dartmouth.edu/scope/results.php?jobid=111290</A></P><P><B>Method:</B></P><P>1. Primer design
</P><UL><LI> melting temperature based on 4+2 method, sum has to be 56°C
</LI></UL><UL><LI> check specific binding using dotplot feature (parameters: low sensitivity: 10, tick at reverse complementary) of Genious Pro 5.7.1
</LI></UL><UL><LI> Check self-complementarity and melting temperature using Oligo Calc
</LI></UL><P>2. Promoter search </P><UL><LI> Application of software tools
</LI></UL><UL><LI> Annotation of pARW071
</LI></UL><P><B>Results:</B></P><P>1. Primer sequence </P><UL><LI> r_bb_1    CGACGAATTCAGATTAGGAA
</LI></UL><P>2. Search for the promotors
</P><UL><LI> Search results see pdf file <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=UP_2011-05-30_PromoterSearchResults.pdf%3F" class="new" title="UP 2011-05-30 PromoterSearchResults.pdf?">Media: UP_2011-05-30_PromoterSearchResults.pdf?</A></LI></UL><P><B>Further tasks:</B></P><UL><LI> PCR of mdn
</LI></UL><UL><LI> Talk with Elke about the promoter
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="9th_Labday_2011-05-31">9th Labday 2011-05-31</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Analysis_of_sequencing_results_and_Primer_design">Analysis of sequencing results and Primer design</SPAN></H3><P><B>Investigators:</B> Jessica, Nicole, Sandrina, Nadja </P><P><B>Aim:</B> Confirm pARW071 sequence and order primer (r_mdna_1) for PCR of mdnA of pARW089</P><P><B>Time:</B> 2011-05-31,11:45-16:00</P><P><B>Materials:</B></P><UL><LI> Geneious Pro 5.1.7
</LI></UL><UL><LI> Oligo Calc: Oligonucleotide Properties Calculator (<A href="http://www.basic.northwestern.edu/biotools/oligocalc.html" class="external free" rel="nofollow">http://www.basic.northwestern.edu/biotools/oligocalc.html</A>)</LI></UL><P><B>Method:</B></P><P>1. Primer design
</P><UL><LI> melting temperature based on 4+2 method, sum has to be 56°C
</LI></UL><UL><LI> check specific binding using dotplot feature (parameters: low sensitivity: 10, tick at reverse complementary) of Genious Pro 5.7.1
</LI></UL><UL><LI> Check self-complementarity and melting temperature using Oligo Calc
</LI></UL><P>2. Sequencing analysis </P><UL><LI> Alignment of sequencing results with pARW071
</LI></UL><P><B>Results:</B></P><P>1. Primer sequence </P><UL><LI> r_mdna_1    CGGTGTAATCAAGAAAAGT
</LI></UL><P>2. Sequencing analysis
</P><UL><LI> pARW071
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Primer
</TH><TH> Mismatch
</TH><TH> Consensus location
</TH></TR><TR><TD> sf_bb_1 1
</TD><TD> -
</TD><TD> -
</TD></TR><TR><TD> sf_bb_2 1
</TD><TD> -
</TD><TD> -
</TD></TR><TR><TD> sf_bb_3 1n
</TD><TD> -
</TD><TD> -
</TD></TR><TR><TD> sf_bb_4 1
</TD><TD> 3
</TD><TD> 677, 777, 857
</TD></TR><TR><TD> sf_mdna_1
</TD><TD> -
</TD><TD> -
</TD></TR><TR><TD> sf_mdnb_1
</TD><TD> /
</TD><TD> /
</TD></TR><TR><TD> sf_mdnc_1
</TD><TD> 3
</TD><TD> 2678, 2860, 2960
</TD></TR><TR><TD> sf_mdn
</TD><TD> 2
</TD><TD> 3780, 4270
</TD></TR><TR><TD> sf_mdne_1n
</TD><TD> 3
</TD><TD> 4718, 5041, 5158
</TD></TR><TR><TD> sf_mdne_2 1
</TD><TD> 3
</TD><TD> 5480, 5800, 6050
</TD></TR><TR><TD> sf_mdne_3 1
</TD><TD> 1
</TD><TD> 6824
</TD></TR></TBODY></TABLE><UL><LI>pARW089
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Primer
</TH><TH> Mismatch
</TH><TH> Consensus location
</TH></TR><TR><TD> sf_mdna_1
</TD><TD> -
</TD><TD> -
</TD></TR></TBODY></TABLE><UL><LI><B>Legend:</B> - = no mismatches; / = sequencing did not work </LI></UL><P><B>Further tasks:</B></P><UL><LI> Check how serious the consequences of the found mismatches are
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/May">http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/May</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Labjournal/May" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Labjournal/May" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/May&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/May&amp;oldid=156859" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>