<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Labjournal_June"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Labjournal/June">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Labjournal/June&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/June&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/June&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Labjournal/June" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Labjournal/June</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team">Team</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Members">Members</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Project</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Summary</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Highlights">Highlights</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Microviridin">     Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Phage_Display">     Phage Display</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#In_Vivo_Selection">     In Vivo Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Modeling">     Modeling</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Microviridin">Details Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">Details Phage</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">Details Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Modeling">Details Modeling</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">BioBricks</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/BioBricks">Details BioBricks</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Methods">Methods</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Meetings">Meetings</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics">Safety &amp; Ethics</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Assesment">Safety Assessment</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Questions">Safety Questions</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Ethics_Seminar">Ethics Seminar</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Survey">Survey</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Potsdam_Bioware_at_the_German_parliament">Parliament Visit</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Statement_of_the_German_government">     Statement</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Outreach">Outreach</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software">Software</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#BioLog_App">BioLog App</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#Download">Download</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Sponsors">Sponsors</A></DT></DL></LI></UL></DIV><DIV class="pageContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#10th_Labday_2011-06-01"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">10th Labday 2011-06-01</SPAN></A></LI><LI class="toclevel-2"><A href="#Primer_design_for_Phage_Display"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Primer design for Phage Display</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#11th_Labday_2011-06-07"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">11th Labday 2011-06-07</SPAN></A></LI><LI class="toclevel-2"><A href="#Design_of_PCR_conditions_for_amplifying_the_complete_mdn_cluster"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Design of PCR conditions for amplifying the complete mdn cluster</SPAN></A></LI><LI class="toclevel-2"><A href="#Design_of_PCR_conditions_for_amplifying_mdnA"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Design of PCR conditions for amplifying mdnA</SPAN></A></LI><LI class="toclevel-1"><A href="#12th_Labday_2011-06-08"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">12th Labday 2011-06-08</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_of_the_complete_mdn_cluster"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">PCR of the complete mdn cluster</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_of_mdnA"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">PCR of mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Production_of_agarose_gel"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Production of agarose gel</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis_of_mdnA"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Agarose gel electrophoresis of mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Design_of_PCR_conditions_for_amplifying_mdnA_for_Phage_display"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Design of PCR conditions for amplifying mdnA for Phage display</SPAN></A></LI><LI class="toclevel-1"><A href="#13th_Labday_2011-06-09"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">13th Labday 2011-06-09</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis_of_mdn_cluster"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Agarose gel electrophoresis of mdn cluster</SPAN></A></LI><LI class="toclevel-1"><A href="#14th_Labday_2011-06-10"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">14th Labday 2011-06-10</SPAN></A></LI><LI class="toclevel-2"><A href="#Amplification_of_mdnA_for_phage_display"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Amplification of mdnA for phage display</SPAN></A></LI><LI class="toclevel-1"><A href="#15th_Labday_2011-06-14"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">15th Labday 2011-06-14</SPAN></A></LI><LI class="toclevel-2"><A href="#cultivation_of_cells_containing_pAK100_from_E._coli_glycerol_stocks"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">cultivation of cells containing pAK100 from E. coli glycerol stocks</SPAN></A></LI><LI class="toclevel-1"><A href="#16th_Labday_2011-06-15"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">16th Labday 2011-06-15</SPAN></A></LI><LI class="toclevel-2"><A href="#gel_electroporesis_PCR_product.28mdnA.2C_2011-06-10.29.2C_plasmid_preperation_und_digestion_with_SfiI_of_pAK100_bla_KDIR"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">gel electroporesis PCR product(mdnA, 2011-06-10), plasmid preperation und digestion with SfiI of pAK100 bla KDIR</SPAN></A></LI><LI class="toclevel-1"><A href="#17th_Labday_2011-06-16"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">17th Labday 2011-06-16</SPAN></A></LI><LI class="toclevel-2"><A href="#gel_electrophoresis_and_gel_extraction_of_digested_pAK100_bla_KDIR"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">gel electrophoresis and gel extraction of digested pAK100 bla KDIR</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_of_mdnA_for_phage_display_repeated_with_modified_conditions"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">PCR of mdnA for phage display repeated with modified conditions</SPAN></A></LI><LI class="toclevel-2"><A href="#gel_electrophoresis_of_PCR_product"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">gel electrophoresis of PCR product</SPAN></A></LI><LI class="toclevel-2"><A href="#Over_night_culture_of_E._coli_XL1-blue_and_RV308"><SPAN class="tocnumber">8.4</SPAN><SPAN class="toctext">Over night culture of E. coli XL1-blue and RV308</SPAN></A></LI><LI class="toclevel-1"><A href="#18th_Labday_2011-06-17"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">18th Labday 2011-06-17</SPAN></A></LI><LI class="toclevel-2"><A href="#competent_cells_-_E._coli_XL1_blue_.26_RV308"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">competent cells - E. coli XL1 blue &amp; RV308</SPAN></A></LI><LI class="toclevel-2"><A href="#primer_design_for_phage_display_2"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">primer design for phage display</SPAN></A></LI><LI class="toclevel-1"><A href="#19th_Labday_2011-06-20"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">19th Labday 2011-06-20</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation_of_LB-Agarplates"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Preparation of LB-Agarplates</SPAN></A></LI><LI class="toclevel-2"><A href="#Design_of_Primers_for_myc-tagging_mdnA"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">Design of Primers for myc-tagging mdnA</SPAN></A></LI><LI class="toclevel-1"><A href="#20st_Labday_2011-06-21"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">20st Labday 2011-06-21</SPAN></A></LI><LI class="toclevel-2"><A href="#Error_prone_PCR_.28varied_MnCl2_and_MgCl2_concentration.29_of_mdnA"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Error prone PCR (varied MnCl2 and MgCl2 concentration) of mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis_of_mutated_mdnA"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Agarose gel electrophoresis of mutated mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Over_night_culture_of_E._coli_BL21_containing_BBa_K404304_pSB1C3_.28biobrick_iGEM_Freiburg_2010.29_from_Sven"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">Over night culture of E. coli BL21 containing BBa_K404304_pSB1C3 (biobrick iGEM Freiburg 2010) from Sven</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_E._coli_XL_1_blu_and_RV_308_.2B_puK"><SPAN class="tocnumber">11.4</SPAN><SPAN class="toctext">Transformation of E. coli XL 1 blu and RV 308 + puK</SPAN></A></LI><LI class="toclevel-1"><A href="#21th_Labday_2011-06-22"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">21th Labday 2011-06-22</SPAN></A></LI><LI class="toclevel-2"><A href="#Gel_extraction_of_mutated_mdnA"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Gel extraction of mutated mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Midiprep_of_pSB1C3_carrying_cells"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">Midiprep of pSB1C3 carrying cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_and_agarose_gel_electrophoresis_of_mutated_mdnA"><SPAN class="tocnumber">12.3</SPAN><SPAN class="toctext">Restriction enzyme digestion and agarose gel electrophoresis of mutated mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Over_night_culture_of_E._coli_containing_pEX_respectively_pBAD_from_Tobias"><SPAN class="tocnumber">12.4</SPAN><SPAN class="toctext">Over night culture of E. coli containing pEX respectively pBAD from Tobias</SPAN></A></LI><LI class="toclevel-1"><A href="#22th_Labday_2011-06-23"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">22th Labday 2011-06-23</SPAN></A></LI><LI class="toclevel-2"><A href="#Midiprep_of_pEX_respectively_pBAD_carrying_cells"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">Midiprep of pEX respectively pBAD carrying cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Over_night_culture_of_E._coli_XL1-Blue_and_RV308_for_competent_cells"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">Over night culture of E. coli XL1-Blue and RV308 for competent cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Over_night_culture_of_E._coli_containing_vector_pBAD_resp._pEX_for_midiprep_resp._glycerol_stocks"><SPAN class="tocnumber">13.3</SPAN><SPAN class="toctext">Over night culture of E. coli containing vector pBAD resp. pEX for midiprep resp. glycerol stocks</SPAN></A></LI><LI class="toclevel-2"><A href="#Amplification_of_mdnA_and_GeneIII_for_phage_display"><SPAN class="tocnumber">13.4</SPAN><SPAN class="toctext">Amplification of mdnA and GeneIII for phage display</SPAN></A></LI><LI class="toclevel-2"><A href="#Design_of_primers_and_planing_of_further_experiments"><SPAN class="tocnumber">13.5</SPAN><SPAN class="toctext">Design of primers and planing of further experiments</SPAN></A></LI><LI class="toclevel-1"><A href="#23th_Labday_2011-06-24"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">23th Labday 2011-06-24</SPAN></A></LI><LI class="toclevel-2"><A href="#purification_of_PCR_products_2011-06-23"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">purification of PCR products 2011-06-23</SPAN></A></LI><LI class="toclevel-2"><A href="#preparation_of_competent_E._coli_XL1_and_RV_308"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">preparation of competent E. coli XL1 and RV 308</SPAN></A></LI><LI class="toclevel-2"><A href="#Midiprep_of_pEX_carrying_cells"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">Midiprep of pEX carrying cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Glycerol_stocks_of_E._coli_cells_carrying_pEX_and_pBAD_plasmids"><SPAN class="tocnumber">14.4</SPAN><SPAN class="toctext">Glycerol stocks of E. coli cells carrying pEX and pBAD plasmids</SPAN></A></LI><LI class="toclevel-1"><A href="#24th_Labday_2011-06-27"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">24th Labday 2011-06-27</SPAN></A></LI><LI class="toclevel-2"><A href="#Control_experiment_fpr_error_prone_PCR_and_enzyme_digestion_of_mdnA"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">Control experiment fpr error prone PCR and enzyme digestion of mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Midiprep_of_pJC354_WZA2_.28TorA-bla_vector.29_carrying_cells"><SPAN class="tocnumber">15.2</SPAN><SPAN class="toctext">Midiprep of pJC354_WZA2 (TorA-bla vector) carrying cells</SPAN></A></LI><LI class="toclevel-1"><A href="#25th_Labday_2011-06-28"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">25th Labday 2011-06-28</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis_of_mutated_mdnA_2"><SPAN class="tocnumber">16.1</SPAN><SPAN class="toctext">Agarose gel electrophoresis of mutated mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Gel_extraction_of_mutated_mdnA_2"><SPAN class="tocnumber">16.2</SPAN><SPAN class="toctext">Gel extraction of mutated mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_and_agarose_gel_electrophoresis_of_mutated_mdnA_2"><SPAN class="tocnumber">16.3</SPAN><SPAN class="toctext">Restriction enzyme digestion and agarose gel electrophoresis of mutated mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Error_prone_PCR_.28varied_MnCl2_concentration.29_of_mdnA"><SPAN class="tocnumber">16.4</SPAN><SPAN class="toctext">Error prone PCR (varied MnCl2 concentration) of mdnA</SPAN></A></LI><LI class="toclevel-1"><A href="#26th_Labday_2011-06-29"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">26th Labday 2011-06-29</SPAN></A></LI><LI class="toclevel-2"><A href="#5l_LB_Media_.28liquid.29"><SPAN class="tocnumber">17.1</SPAN><SPAN class="toctext">5l LB Media (liquid)</SPAN></A></LI><LI class="toclevel-2"><A href="#oligo_hybridization_for_TEV_and_14_3C_protease_cleavage_sites"><SPAN class="tocnumber">17.2</SPAN><SPAN class="toctext">oligo hybridization for TEV and 14_3C protease cleavage sites</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_of_pJC354_WZA2-vector"><SPAN class="tocnumber">17.3</SPAN><SPAN class="toctext">Restriction of pJC354_WZA2-vector</SPAN></A></LI><LI class="toclevel-2"><A href="#preparative_agarose_electrophoresis"><SPAN class="tocnumber">17.4</SPAN><SPAN class="toctext">preparative agarose electrophoresis</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_for_amplification_of_mdnA_with_C-terminal_myc-tag"><SPAN class="tocnumber">17.5</SPAN><SPAN class="toctext">PCR for amplification of mdnA with C-terminal myc-tag</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_of_mdnA_for_phage_display_.28strategy_1.29_repeated_with_new_ordered_forward_primer"><SPAN class="tocnumber">17.6</SPAN><SPAN class="toctext">PCR of mdnA for phage display (strategy 1) repeated with new ordered forward primer</SPAN></A></LI><LI class="toclevel-2"><A href="#Error_prone_PCR_.28varied_MnCl2_concentration.29_of_mdnA_2"><SPAN class="tocnumber">17.7</SPAN><SPAN class="toctext">Error prone PCR (varied MnCl2 concentration) of mdnA</SPAN></A></LI><LI class="toclevel-1"><A href="#27th_Labday_2011-06-30"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">27th Labday 2011-06-30</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis_of_myc-tagged_mdnA_and_mdnA_for_phage_display"><SPAN class="tocnumber">18.1</SPAN><SPAN class="toctext">Agarose gel electrophoresis of myc-tagged mdnA and mdnA for phage display</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis_of_error_prone_PCR_of_mdnA_from_2011-06-29_and_gel_extraction"><SPAN class="tocnumber">18.2</SPAN><SPAN class="toctext">Agarose gel electrophoresis of error prone PCR of mdnA from 2011-06-29 and gel extraction</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_PCR_of_mdnA_for_phage_display_.28strategy_1.29_repeated_with_new_ordered_forward_primer"><SPAN class="tocnumber">18.3</SPAN><SPAN class="toctext">Repeated PCR of mdnA for phage display (strategy 1) repeated with new ordered forward primer</SPAN></A></LI></TD></TR></TBODY></TABLE><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="10th_Labday_2011-06-01">10th Labday 2011-06-01</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Primer_design_for_Phage_Display">Primer design for Phage Display</SPAN></H3><P><B>Investigators:</B> Jessica, Sabine, Sandrina, Nadja </P><P><B>Aim:</B> Design primers for PCR of mdnA (from pARW089) to clone into pAK100 with <I>Sfi</I>I restriction sites</P><P><B>Time:</B> 2011-06-01,15:30-21:00</P><P><B>Materials:</B></P><UL><LI> Geneious Pro 5.1.7
</LI></UL><UL><LI> Oligo Calc: Oligonucleotide Properties Calculator (<A href="http://www.basic.northwestern.edu/biotools/oligocalc.html" class="external free" rel="nofollow">http://www.basic.northwestern.edu/biotools/oligocalc.html</A>)</LI></UL><P><B>Method:</B></P><P>Primer design
</P><UL><LI> melting temperature based on 4+2 method (for mdnA-overlapping part)
</LI></UL><UL><LI> adding sequences for restriction sites (considering reading frame)
</LI></UL><UL><LI> Check self-complementarity and melting temperature using Oligo Calc
</LI></UL><P><B>Results:</B></P><P>Primer sequences </P><UL><LI> pf_sfi_mdna_1 TCTAGATGGCGGCCCAGCCGGCCATGGCGATGGCATATCCCAAC
</LI></UL><UL><LI> pr_sfi_mdna_1 CCCTTCTGACTGGGAAGATTATGGCCTCGGGGGCCACTAATACTAGTAGCGGCCGCTGCAGGCT
</LI></UL><P><B>Further tasks:</B></P><UL><LI> do PCR of mdnA
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="11th_Labday_2011-06-07">11th Labday 2011-06-07</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Design_of_PCR_conditions_for_amplifying_the_complete_mdn_cluster">Design of PCR conditions for amplifying the complete mdn cluster</SPAN></H3><P><B>Investigators:</B> Jessica, Nicole, Nadja </P><P><B>Aim:</B> PCR amplification of the mdn cluster </P><P><B>Time:</B> 2011-06-07,18:00-19:30</P><P><B>Materials:</B></P><UL><LI> Sequence of vector pARW071, concentration 857.3 ng/ µl
</LI></UL><UL><LI> Manual: Fermentas Long PCR Enzyme Mix 
</LI></UL><UL><LI> Forward primer: sf_bb_4_1
</LI></UL><UL><LI> Reverse primer: r_bb_1
</LI></UL><UL><LI> Geneious Pro 5.1.7
</LI></UL><P><B>Method:</B></P><UL><LI> Design of a temperature profile based on the manual
</LI></UL><UL><LI> Definition of a mastermix based on the manual
</LI></UL><P><B>Results:</B></P><P>1. Temperature profil
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Temperature in °C </TH><TH> Time in s </TH></TR><TR><TD> 94 </TD><TD> Hold
</TD></TR><TR><TD> 94 </TD><TD> 180 </TD><TD> Initial denaturation
</TD></TR><TR><TD> 94 </TD><TD> 15 </TD><TD> Denaturation
</TD></TR><TR><TD> 46 </TD><TD> 30 </TD><TD> Annealing
</TD></TR><TR><TD> 68 </TD><TD> 300 </TD><TD> Elongation
</TD></TR><TR><TD> --&gt; 10 cycles
</TD></TR><TR><TD> 94 </TD><TD> 15 </TD><TD> Denaturation
</TD></TR><TR><TD> 46 </TD><TD> 30 </TD><TD> Annealing
</TD></TR><TR><TD> 68 </TD><TD> 300 + 2 per each cycle </TD><TD> Elongation
</TD></TR><TR><TD> --&gt; 20 cycles
</TD></TR><TR><TD> 68 </TD><TD> 600 </TD><TD> Final extension
</TD></TR></TBODY></TABLE><P>2. Mastermix
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component </TH><TH> Volume in µl </TH><TH> Final concentration
</TH></TR><TR><TD> Buffer Fermentas Long Enzyme Mix (+MgCl<SMALL>2</SMALL>) </TD><TD> 5 </TD></TR><TR><TD> dNTPs </TD><TD> 1 </TD><TD> 0.2 mM
</TD></TR><TR><TD> Primer (each) </TD><TD> 3 </TD><TD> 0.6 µM
</TD></TR><TR><TD> DNA (diluted 1:100)</TD><TD> 4.08 µl </TD><TD> 0.7 ng
</TD></TR><TR><TD> Polymerase </TD><TD> 0.3 µl </TD><TD> 1.5 U
</TD></TR><TR><TD> H<SMALL>2</SMALL>O </TD><TD> 33.62 µl </TD><TD> (up to 50 µl)
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> PCR of the complete mdn cluster
</LI></UL><UL><LI> verification using agarose gel electrophoresis
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Design_of_PCR_conditions_for_amplifying_mdnA">Design of PCR conditions for amplifying mdnA</SPAN></H3><P><B>Investigators:</B> Jessica, Nicole, Nadja </P><P><B>Aim:</B> PCR amplification of mdnA </P><P><B>Time:</B> 2011-06-07,18:00-19:30</P><P><B>Materials:</B></P><UL><LI> Sequence of vector pARW089, concentration 626.7 ng/ µl
</LI></UL><UL><LI> Manual: NEB OneTaq DNA Polymerase 
</LI></UL><UL><LI> Forward primer: sf_mdna_1
</LI></UL><UL><LI> Reverse primer: r_mdna_1
</LI></UL><UL><LI> Geneious Pro 5.1.7
</LI></UL><P><B>Method:</B></P><UL><LI> Design of a temperature profile based on the manual
</LI></UL><UL><LI> Definition of a mastermix based on the manual
</LI></UL><P><B>Results:</B></P><P>1. Temperature profil
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Temperature in °C </TH><TH> Time in s </TH></TR><TR><TD> 94 </TD><TD> Hold
</TD></TR><TR><TD> 94 </TD><TD> 30 </TD><TD> Initial denaturation
</TD></TR><TR><TD> 94 </TD><TD> 15 </TD><TD> Denaturation
</TD></TR><TR><TD> 45 </TD><TD> 35 </TD><TD> Annealing
</TD></TR><TR><TD> 68 </TD><TD> 20 </TD><TD> Elongation
</TD></TR><TR><TD> --&gt; 30 cycles
</TD></TR><TR><TD> 68 </TD><TD> 300 </TD><TD> Final extension
</TD></TR><TR><TD> 6 </TD><TD> Hold
</TD></TR></TBODY></TABLE><P>2. Mastermix
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component </TH><TH> Volume in µl </TH><TH> Final concentration
</TH></TR><TR><TD> Buffer NEB OneTaq </TD><TD> 10 </TD></TR><TR><TD> dNTPs </TD><TD> 1 </TD><TD> 0.2 mM
</TD></TR><TR><TD> Primer (each, diluted 1:10) </TD><TD> 1 </TD><TD> 0.2 µM
</TD></TR><TR><TD> DNA (diluted 1:100)</TD><TD> 5.58 </TD><TD> 0.7 ng
</TD></TR><TR><TD> Polymerase </TD><TD> 0.25 </TD></TR><TR><TD> H<SMALL>2</SMALL>O </TD><TD> 31.17 </TD><TD> (up to 50 µl)
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> PCR of mdnA
</LI></UL><UL><LI> verification using agarose gel electrophoresis
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="12th_Labday_2011-06-08">12th Labday 2011-06-08</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR_of_the_complete_mdn_cluster">PCR of the complete mdn cluster</SPAN></H3><P><B>Investigators:</B> Jessica, Nicole </P><P><B>Aim:</B> Testing the possibility of a PCR of the whole mdn cluster and the chosen conditions </P><P><B>Time:</B> 2011-06-08, 9:30-14:00</P><P><B>Materials:</B></P><UL><LI> DNA polymerases: Long PCR Enzyme Mix (Fermentas)
</LI></UL><UL><LI> Template DNA: pARW071 (vector)
</LI></UL><UL><LI> Forward primer: sf_bb_4_1
</LI></UL><UL><LI> Reverse Primer: r_bb_1
</LI></UL><P>--&gt; mastermix composition, see 11th labday 'Design of PCR conditions for amplifying the complete mdn cluster' </P><UL><LI> Eppendorf Mastercycler gradient
</LI></UL><UL><LI> used temperature profil
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH> Temperature in °C </TH><TH> Time in s </TH></TR><TR><TD> 94 </TD><TD> Hold
</TD></TR><TR><TD> 94 </TD><TD> 120 </TD><TD> Initial denaturation
</TD></TR><TR><TD> 94 </TD><TD> 20 </TD><TD> Denaturation
</TD></TR><TR><TD> 56 </TD><TD> 20 </TD><TD> Annealing
</TD></TR><TR><TD> 68 </TD><TD> 300 </TD><TD> Elongation
</TD></TR><TR><TD> --&gt; 10 cycles
</TD></TR><TR><TD> 94 </TD><TD> 20 </TD><TD> Denaturation
</TD></TR><TR><TD> 56 </TD><TD> 20 </TD><TD> Annealing
</TD></TR><TR><TD> 68 </TD><TD> 300 + 2 per each cycle </TD><TD> Elongation
</TD></TR><TR><TD> --&gt; 20 cycles
</TD></TR><TR><TD> 68 </TD><TD> 600 </TD><TD> Final elongation
</TD></TR><TR><TD> 6 </TD><TD> Hold
</TD></TR></TBODY></TABLE><P><B>Method:</B></P><UL><LI> pipetting the mastermix on ice
</LI></UL><UL><LI> preheating of the thermocycler: starting the program IGMDN1 (Eppendorf Mastercycler gradient)
</LI></UL><UL><LI> finally adding polymerase
</LI></UL><UL><LI> start the PCR program (press enter)
</LI></UL><P><B>Results:</B></P><UL><LI> PCR product
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Verification of the PCR product using agarose gel electrophoresis
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR_of_mdnA">PCR of mdnA</SPAN></H3><P><B>Investigators:</B> Jessica, Nicole </P><P><B>Aim:</B> Testing the possibility of amplifying mdnA using PCR and the chosen conditions </P><P><B>Time:</B> 2011-06-08,9:30-14:00</P><P><B>Materials:</B></P><UL><LI> DNA polymerases: NEB OneTaq DNA Polymerase
</LI></UL><UL><LI> Template DNA: pARW089 (vector)
</LI></UL><UL><LI> Forward primer: sf_mdna_1
</LI></UL><UL><LI> Reverse Primer: r_mdna_1
</LI></UL><P>--&gt; mastermix composition, see 11th labday 'Design of PCR conditions for amplifying mdnA' </P><UL><LI> Eppendorf Mastercycler personal
</LI><LI> used temperature profil
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH> Temperature in °C </TH><TH> Time in s </TH></TR><TR><TD> 94 </TD><TD> Hold
</TD></TR><TR><TD> 94 </TD><TD> 60 </TD><TD> Initial denaturation
</TD></TR><TR><TD> 94 </TD><TD> 20 </TD><TD> Denaturation
</TD></TR><TR><TD> 56 </TD><TD> 20 </TD><TD> Annealing
</TD></TR><TR><TD> 68 </TD><TD> 20 </TD><TD> Elongation
</TD></TR><TR><TD> --&gt; 30 cycles
</TD></TR><TR><TD> 68 </TD><TD> 300 </TD><TD> Final elongation
</TD></TR><TR><TD> 6 </TD><TD> Hold
</TD></TR></TBODY></TABLE><P><B>Method:</B></P><UL><LI> pipetting the mastermix on ice
</LI></UL><UL><LI> preheating of the thermocycler: starting the program IGMDNA1 (Eppendorf Mastercycler personal)
</LI></UL><UL><LI> finally adding polymerase
</LI></UL><UL><LI> start the PCR program (press enter)
</LI></UL><P><B>Results:</B></P><UL><LI> PCR product
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Verification of the PCR product using agarose gel electrophoresis
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Production_of_agarose_gel">Production of agarose gel</SPAN></H3><P><B>Investigators:</B> Jessica, Nicole </P><P><B>Aim:</B> Verification of PCR product, mdnA and whole mdn cluster, by agarose gel electrophoresis </P><P><B>Time:</B> 2011-06-08,9:30-14:00</P><P><B>Materials:</B></P><UL><LI> Agarose, type VII, low gelling temperature (Sigma)
</LI></UL><UL><LI> 1x TAE Buffer (produced by AG Biotechnology)
</LI></UL><UL><LI> GelRed
</LI></UL><P><B>Method:</B></P><P>1. mdn cluster (approx. 6712 bp) --&gt; 0.8 %
</P><UL><LI> 0.75 g agarose in 50 ml 1xTAE buffer
</LI></UL><UL><LI> dissolving/ boiling up using microwave
</LI></UL><UL><LI> after cooling, adding of 2 µl GelRed
</LI></UL><P>2. mdnA (approx. 276 bp) --&gt; 1.5 %
</P><UL><LI> 0.4 g agarose in 50 ml 1xTAE buffer
</LI></UL><UL><LI> dissolving/ boiling up using microwave
</LI></UL><UL><LI> after cooling, adding of 2 µl Gelred
</LI></UL><P><B>Results</B></P><UL><LI> 1.5 % agarose gel
</LI></UL><UL><LI> 0.8 % agarose gel
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Loading gel and performing electrophoresis
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_of_mdnA">Agarose gel electrophoresis of mdnA</SPAN></H3><P><B>Investigators:</B> Jessica, Nicole, Nadja </P><P><B>Aim:</B> Verification of PCR product, mdnA, by agarose gel electrophoresis</P><P><B>Time:</B> 2011-06-08,15:45-18:00</P><P><B>Materials:</B></P><UL><LI> 1.5 % agarose gel, produced previously
</LI></UL><UL><LI> PCR product of mdnA amplification
</LI></UL><UL><LI> DNA Ladder Gene Ruler, 100bp plus, DNA Ladder (Fermentas)</LI></UL><UL><LI> 6x Loading Dye (Fermentas)</LI></UL><P><B>Method:</B></P><P>1. Loading samples
</P><UL><LI> dilution of PCR product: 1 µl DNA and 4 µl water
</LI></UL><UL><LI> Adding 1 µl 6x loading dye to diluted PCR product
</LI></UL><P>Positions of samples/ marker
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Sample/[µl] </TH><TH> Expected size (bp) </TH><TH>approx. size
</TH></TR><TR><TD> 1 </TD><TD> marker </TD><TD> 2 </TD></TR><TR><TD> 2 </TD><TD> PCR product mdnA </TD><TD> 6 </TD><TD> 276 </TD><TD>280
</TD></TR><TR><TD> 4 </TD><TD> PCR product mdnA (dilution) </TD><TD> 6 </TD><TD> 276 </TD><TD>280
</TD></TR><TR><TD> 5 </TD><TD> marker (dilution 1:5) </TD><TD> 2 </TD></TR></TBODY></TABLE><P>2. Run
</P><UL><LI> 100 V
</LI></UL><UL><LI> time: 1 h
</LI></UL><P><B>Results:</B></P><P>approx. size: see methods
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Design_of_PCR_conditions_for_amplifying_mdnA_for_Phage_display">Design of PCR conditions for amplifying mdnA for Phage display</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine </P><P><B>Aim:</B> PCR reaction of mdnA with overlapping primers containing SfiI restriction sites for cloning into PAK100, which is needed for phage display </P><P><B>Time:</B> 2011-06-8,12:00-14:00 </P><P><B>Materials:</B></P><UL><LI> Sequence of vector pARW089
</LI></UL><UL><LI> Manual: OneTaq Polymerase (NEB)
</LI></UL><UL><LI> Forward primer: pf_mdnA_sfi_1
</LI></UL><UL><LI> Reverse primer: pr_mdnA_sfi_1
</LI></UL><UL><LI> Geneious
</LI></UL><P><B>Method:</B></P><UL><LI> Design of a temperature profile based on the manual
</LI></UL><P><B>Results:</B></P><P>1. Temperature profil
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Temperature in °C </TH><TH> Time in s </TH></TR><TR><TD> 94 </TD><TD> Hold
</TD></TR><TR><TD> 94 </TD><TD> 60 </TD><TD> Initial denaturation
</TD></TR><TR><TD> 94 </TD><TD> 20 </TD><TD> Denaturation
</TD></TR><TR><TD> 60 </TD><TD> 20 </TD><TD> Annealing
</TD></TR><TR><TD> 72 </TD><TD> 20 </TD><TD> Elongation
</TD></TR><TR><TD> --&gt; 15 cycles
</TD></TR><TR><TD> 94 </TD><TD> 60 </TD><TD> Denaturation
</TD></TR><TR><TD> 70 </TD><TD> 40 </TD><TD> Annealing
</TD></TR><TR><TD> 72 </TD><TD> 20 </TD><TD> Elongation
</TD></TR><TR><TD> --&gt; 15 cycles
</TD></TR><TR><TD> 70 </TD><TD> 300 </TD><TD> Final extension
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B> PCR
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="13th_Labday_2011-06-09">13th Labday 2011-06-09</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_of_mdn_cluster">Agarose gel electrophoresis of mdn cluster</SPAN></H3><P><B>Investigators:</B> Jessica, Steffi, Nicole </P><P><B>Aim:</B> Verification of amplification of the complete mdn cluster by PCR</P><P><B>Time:</B> 2011-06-09,10:00-17:30 </P><P><B>Materials:</B></P><UL><LI> 0.8 % agarose gel, produced previously
</LI></UL><UL><LI> PCR product of mdn amplification
</LI></UL><UL><LI> DNA Ladder Gene Ruler, 100bp plus, DNA Ladder (Fermentas)</LI></UL><UL><LI> 6x Loading Dye (Fermentas)</LI></UL><P><B>Method:</B></P><P>1. Loading samples
</P><UL><LI> dilution of PCR product:
</LI></UL><UL><LI> Adding 1 µl 6x loading dye to diluted PCR product
</LI></UL><P>Positions of samples/ marker
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Sample/[µl] </TH><TH> Expected size (bp) </TH><TH>approx. size
</TH></TR><TR><TD> 1 </TD><TD> marker </TD></TR><TR><TD> 2 </TD><TD> PCR product mdn cluster </TD></TR><TR><TD> 4 </TD><TD> PCR product mdn cluster (dilution) </TD></TR><TR><TD> 5 </TD><TD> PCR product mdn cluster (dilution) </TD></TR></TBODY></TABLE><P>2. Run
</P><UL><LI> 100 V
</LI></UL><UL><LI> time: 1 h
</LI></UL><P><B>Results:</B></P><P><A href="/File:UP_AG_PCR_mdn_cluster_2011-06-11_JE_001.jpg" title="File:UP AG PCR mdn cluster 2011-06-11 JE 001.jpg">250px</A></P><P>approx. size: see methods
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="14th_Labday_2011-06-10">14th Labday 2011-06-10</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Amplification_of_mdnA_for_phage_display">Amplification of mdnA for phage display</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine </P><P><B>Aim:</B> amplificate mdnA of pARW089 with primers pf_mdnA_sfi_1 and pr_mdnA_sfi_1</P><P><B>Time:</B> 2011-06-10,08:00-10:00, 14:00-20:00</P><P><B>Materials/Methods:</B></P><P>1. PCR (see 11th Labday 2011-06-08) </P><UL><LI> 0,75 ng Vector pARW089
</LI></UL><UL><LI> Thermo Hybraid Px2
</LI></UL><UL><LI> 0,25 µl OneTaq DNA Polymerase (NEB)
</LI></UL><UL><LI> 1 µl dNTPs
</LI></UL><UL><LI> 1 µl each forward and reverse primer
</LI></UL><UL><LI> 32,15 µl DNase free water
</LI></UL><UL><LI> 10 µl 5xOneTaq Standard Reaction Buffer
</LI></UL><P>2. PCR product purification
</P><UL><LI> QIAquick Gel Extraction Kit (250)
</LI></UL><P>3. Digestion with SfiI
</P><UL><LI> 50 µl sample
</LI></UL><UL><LI> NEB 10x buffer 4
</LI></UL><UL><LI> 100x BSA
</LI></UL><UL><LI> 3 µl restriction enzyme SfiI
</LI></UL><UL><LI> PCR product (mdnA with SfiI restriction sites)
</LI></UL><P><B>Further tasks:</B> gel electrophoresis and gel extraction
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="15th_Labday_2011-06-14">15th Labday 2011-06-14</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="cultivation_of_cells_containing_pAK100_from_E._coli_glycerol_stocks">cultivation of cells containing pAK100 from <I>E. coli</I> glycerol stocks</SPAN></H3><P><B>Investigators:</B> Sandrina </P><P><B>Aim:</B> cultivate cells conatining pAK100 (vector) to purify it and use it for phage display.
</P><P>1.amplificated mdnA should be cloned into pAK100
</P><P>2. genIII should be amplified from it to clone into pARW089 </P><P><B>Time:</B> 2011-06-14,13:45-14:45</P><P><B>Materials/Methods:</B></P><UL><LI>glycerol stock nr. 15, pAK100 bla KDIR, XL1-blu, 2003-07-31
</LI></UL><UL><LI>LB-medium with chloramphenicol (1:1000)
</LI></UL><P><B>Further tasks:</B> plasmid preperation und digestion with SfiI
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="16th_Labday_2011-06-15">16th Labday 2011-06-15</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="gel_electroporesis_PCR_product.28mdnA.2C_2011-06-10.29.2C_plasmid_preperation_und_digestion_with_SfiI_of_pAK100_bla_KDIR">gel electroporesis PCR product(mdnA, 2011-06-10), plasmid preperation und digestion with SfiI of pAK100 bla KDIR </SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine, Jessica </P><P><B>Aim:</B> purification of mdnA and pak100 to use it for phage display.
</P><P>1.amplificated mdnA should be cloned into pAK100
</P><P>2.genIII should be amplified from it to clone into pARW089 </P><P><B>Time:</B> 2011-06-15,12:00-18:00</P><P><B>Materials/Methods:</B></P><P>1. plasmid preparation
</P><P>GeneJET Plasmid Miniprep Kit (Fermentas, K0509)
</P><P>2. Digestion with SfiI
</P><UL><LI> 50 µl sample
</LI></UL><UL><LI> NEB 10x buffer 4
</LI></UL><UL><LI> 100x BSA
</LI></UL><UL><LI> 3 µl restriction enzyme SfiI
</LI></UL><UL><LI> PCR product (mdnA with SfiI restriction sites)
</LI></UL><P>3. gel electrophoresis
</P><P>sample preparation: 50 µl PCR product (digested) + 10 µl 6x loading dye solution</P><P>0,75 g Agarose, 50 ml TAE (1 x), 2 µl GELRED, at 100 Volt,running time: 45 minutes</P><P>marker: GeneRuler ladder 100 Bp plus ladder (Fermentas)</P><P><B>Results:</B></P><P>no results, PCR didn`t work
</P><P><B>Further tasks:</B> repeat PCR, gel electrophoresis of digested pak100
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="17th_Labday_2011-06-16">17th Labday 2011-06-16</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="gel_electrophoresis_and_gel_extraction_of_digested_pAK100_bla_KDIR">gel electrophoresis and gel extraction of digested pAK100 bla KDIR</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine, Anna </P><P><B>Time:</B> 2011-06-16,10:00-12:00 </P><P><B>Aim:</B> get a pAK100 plasmid digested with SfiI to clone mdnA into it (phage display, strategy 1) </P><P><B>Materials/Methods:</B></P><UL><LI>1.gel electrophoresis sample preparation: 50 µl plasmid product (digested) + 10 µl 6x loading dye solution 0,5 g Agarose, 50 ml TAE (1 x), 2 µl GELRED, at 100 Volt,running time: 45 minutes marker: 1 kb DNA ladder (NEB)
</LI></UL><UL><LI>2. gel extraction
</LI></UL><P>QIAquick Gel Extraction Kit (250)
</P><P><B>Results:</B></P><P>digestion worked, plasmids were extracted from the gel and stored at -20 °C
</P><P><B>Further tasks:</B></P><P>ligation of digested pAK100 with amplified mdnA containing SfiI restriction sites
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="PCR_of_mdnA_for_phage_display_repeated_with_modified_conditions">PCR of mdnA for phage display repeated with modified conditions</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine, Anna </P><P><B>Time:</B> 2011-06-16,10:00-12:00 </P><P><B>Aim:</B> amplification of mdnA with SfiI restriction sites (vector pARW089) </P><P><B>Materials/Methods:</B></P><P>see 2011-06-10
</P><P>changes:
</P><UL><LI> annealing temperature 56°C (stage 1)
</LI></UL><UL><LI> template 65 ng
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="gel_electrophoresis_of_PCR_product">gel electrophoresis of PCR product</SPAN></H3><P><B>Investigators:</B> Sandrina, Anna, Sabine </P><P><B>Time:</B> 2011-06-16,16:00-18:00 </P><P><B>Material and Methods:</B></P><P>see 2011-06-15
</P><P><B>Results:</B></P><P>no result </P><P>reason: reversed primer was ordered unreversed </P><P><B>Further tasks:</B></P><P>order a new primer
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Over_night_culture_of_E._coli_XL1-blue_and_RV308">Over night culture of <I>E. coli</I> XL1-blue and RV308</SPAN></H3><P><B>Investigators:</B> Sandrina, Anna, Sabine </P><P><B>Time:</B> 2011-06-16,18:00-19:00 </P><P><B>Aim:</B> cultering cells to produce competent cells </P><P><B>Material and Methods:</B></P><UL><LI> XL1-blue: 15 ml LB medium with 100 µg/mL tetracycline out of tetracycline stock solution 50 mg/mL
</LI></UL><UL><LI> RV308: 15 ml LB medium without streptomycin (not available)
</LI></UL><P><B>Further tasks:</B> produce competent cells
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="18th_Labday_2011-06-17">18th Labday 2011-06-17</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="competent_cells_-_E._coli_XL1_blue_.26_RV308">competent cells - <I>E. coli</I> XL1 blue &amp; RV308 </SPAN></H3><P><B>Investigator:</B> Niels </P><P><B>Aim:</B> produce competent cells
</P><P><B>Time:</B> 2011-06-15,09:00-16:30</P><P><B>Materials/Methods:</B></P><P>Work always sterile and cold and speedy!
</P><UL><LI> All volumes deal with the common cellline!
</LI></UL><UL><LI> Prepare 45 Eppis (1,5µl)
</LI></UL><UL><LI> get liquid nitrogen
</LI></UL><UL><LI> Use Milipore-filter for sterile CaCl2 , keep cool!
</LI></UL><UL><LI> prepare 15ml LB-Medium (or DYT) with the specific antibiotic (XL1-blue Tet, BL21 none!), inoculate and incubate over night
</LI></UL><UL><LI> prepare 100ml LB-Medium (or DYT) with the specific antibiotic, inoculate with 2ml of the over-night-culture. Nurture the culture until OD600 at 0,35 (0,2-0,5) (if the OD is too high, the cell won’t be competent)
</LI></UL><UL><LI> keep cell suspension in sterile falcons (50ml) 20 min on ice, then centrifuge for 20min; 4°C; 2500g
</LI></UL><UL><LI> discard supernatant, carefully resuspend on ice with 10ml cold CaCl2-solution (put a little of the 10ml solution in every falcon before!), pool every resuspended aliquot of one cell line and add 40ml CaCl2-solution (total volume 50ml). Keep 30 min on ice, then centrifuge for 20min; 4°C; 2500g
</LI></UL><UL><LI> discard supernatant, carefully resuspend pellet in 5,5ml CaCl2(80mM)/Glycerol (4:1), use the pippete boy and a sterile glas pipette to dissolve the pellet
</LI></UL><UL><LI> aliquot in Eppis á 150µl and store immediately at liquid nitrogen and afterwards at -80 °C</LI></UL><P><B>Results:</B></P><UL><LI> 22 tubes RV308 for expression(150µl competent cells at -80°C)
</LI></UL><UL><LI> 22 tubes XL1 blue for transformation(150µl competent cells at -80°C)
</LI></UL><P><B>Further tasks:</B> check by transformation
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="primer_design_for_phage_display_2">primer design for phage display </SPAN></H3><P><B>Investigators:</B> Jessica, Sabine, Sandrina, Anna </P><P><B>Aim:</B> Design primers for PCR of
</P><UL><LI> genIII (from pak100)
</LI></UL><UL><LI> mdnA (from pARW089)
</LI></UL><P>to fuse genIII and mdnA in pARW089
</P><P><B>Time:</B> 2011-06-17,14:00-18:00</P><P><B>Materials:</B></P><UL><LI> Geneious Pro 5.1.7
</LI></UL><UL><LI> Oligo Calc: Oligonucleotide Properties Calculator (<A href="http://www.basic.northwestern.edu/biotools/oligocalc.html" class="external free" rel="nofollow">http://www.basic.northwestern.edu/biotools/oligocalc.html</A>)</LI></UL><P><B>Method:</B></P><P>Primer design
</P><UL><LI> melting temperature based on 4+2 method (for mdnA-overlapping part)
</LI></UL><UL><LI> adding sequences for restriction sites (considering reading frame)
</LI></UL><UL><LI> Check self-complementarity and melting temperature using Oligo Calc
</LI></UL><P><B>Results:</B></P><P>Primer sequences (strategy 2) </P><UL><LI> pf_iGEM_GenIII_NgoMIV TAAGCTGCCGGCGAGCAGAAGCTGATCTCTGAGGAAGACCTGGGTGGTGGCTCTGGTTCC (2nt-NgoMIV-myc-GenIII)
</LI></UL><UL><LI> pr_GenIII_iGEM_AatII 5'-&gt;3' TGCTTAGACGTCTACTAGTATTAACCGGTAGACTCCTTATTACGCAGTA (GenIII-AgeI-SpeI-AatII-6nt)
</LI></UL><UL><LI> pf_mdnA-iGEM_EheI TTCCATGGCGCCAGAGGAATCTAGATGGCATATCCCAACGATC (6nt-EheI-RBS-XbaI-mdnA)
</LI></UL><UL><LI> pr_mdnA_iGEM_AAtII 5'-&gt;3' GACGTCCTGCAGCGGCCGCTACTAGTATTAACCGGTTATAATCTTCCCAGTCAGAAG (mdnA-AgeI-SpeI-NotI-PstI-AatII-nt)
</LI></UL><P>new ordered primer for amplification of mdnA from pARW089 with SfiI restriction sites (strategy 1)
</P><UL><LI> pr_sfi_mdna_1 5'-&gt;3' AGCCTGCAGCGGCCGCTACTAGTATTAGTGGCCCCCGAGGCCATAATCTTCCCAGTCAGAAGGG
</LI></UL><P><B>Further tasks:</B></P><P>PCR
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="19th_Labday_2011-06-20">19th Labday 2011-06-20</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparation_of_LB-Agarplates">Preparation of LB-Agarplates</SPAN></H3><P><B>Investigators:</B> Nadine, Katharina, Steffi, Nicole, Jessica </P><P><B>Time:</B> 2011-06-09,13:45-19:00 </P><P><B>Aim:</B> Preparation of LB-Agarplates </P><P><B>Recipe:</B></P><UL><LI> 10 g/l Yeast-Tryptone
</LI></UL><UL><LI> 10 g/l NaCl
</LI></UL><UL><LI> 5 g/l Yeast-Extract
</LI></UL><UL><LI> 15 g Agar-Agar
</LI></UL><UL><LI> fill up to 1000 ml with Millipore H2O and autoclave
</LI></UL><P><B>Further tasks:</B> add antibiotics (Ampicillin, Kanamycin) to LB-Medium and prepare plates
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Design_of_Primers_for_myc-tagging_mdnA">Design of Primers for myc-tagging mdnA </SPAN></H3><P><B>Investigators:</B> Nicole, Nadine, Katharina, Steffi </P><P><B>Aim:</B> Design primers for mdnA (from pARW089) in order to tag it w/ myc (+ Factor XA Protease restriction site) to clone into vector from Sven (iGEM vector from 2010)</P><P><B>Time:</B> 2011-06-01,16:00-19:00</P><P><B>Materials:</B></P><UL><LI> Geneious Pro 5.1.7
</LI></UL><UL><LI> Oligo Calc: Oligonucleotide Properties Calculator (<A href="http://www.basic.northwestern.edu/biotools/oligocalc.html" class="external free" rel="nofollow">http://www.basic.northwestern.edu/biotools/oligocalc.html</A>)</LI></UL><P><B>Method:</B></P><UL><LI> Primer design
</LI></UL><UL><LI> melting temperature based on 4+2 method (for mdnA-overlapping part)
</LI></UL><UL><LI> adding sequences for restriction sites (considering reading frame)
</LI></UL><UL><LI> Check self-complementarity and melting temperature using Oligo Calc
</LI></UL><UL><LI> Myc-tag
</LI></UL><UL><LI><A href="/wiki/images/e/ee/UP_sequence_myc_withoutamber.png" class="internal" title="UP sequence myc withoutamber.png">Media: UP_sequence_myc_withoutamber.png</A></LI></UL><UL><LI> why?: purification by affinity chromatography (using antybodies against myc-tag)
</LI></UL><UL><LI>Factor XA Protease
</LI></UL><UL><LI> why?: to be able to cleave myc tag after mdnA expression and purification
</LI></UL><UL><LI> restriction site: I E/D G R
</LI></UL><UL><LI> used sequence (forward): ATT GAA GGC CGT
</LI></UL><P><B>Results:</B></P><P>Primer sequences </P><P><B>Further tasks:</B></P><UL><LI> do PCR of mdnA
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="20st_Labday_2011-06-21">20st Labday 2011-06-21</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Error_prone_PCR_.28varied_MnCl2_and_MgCl2_concentration.29_of_mdnA">Error prone PCR (varied MnCl<SMALL>2</SMALL> and MgCl<SMALL>2</SMALL> concentration) of mdnA</SPAN></H3><P><B>Investigators:</B> Nadine, Steffi, Nicole </P><P><B>Time:</B> 2011-06-21,9:00-14:00 </P><P><B>Aim:</B> Random mutation of mdnA gene based on increased MnCl<SMALL>2</SMALL> and MgCl<SMALL>2</SMALL> concentration </P><P><B>Materials:</B></P><UL><LI> Marcus Schicklberger's (trainee, August 2005) protocol performing several error prone PCRs
</LI></UL><UL><LI> Manual Supplement of Cadwell and Joyce 2009 [[Media:
</LI></UL><UL><LI> Genaxxon Bioscience Taq S Polymerase, manufacturer's protocol <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=UP_Genaxxon_Taq_Polymerase_Protocol.pdf" class="new" title="UP Genaxxon Taq Polymerase Protocol.pdf">Media: UP_Genaxxon_Taq_Polymerase_Protocol.pdf</A></LI></UL><UL><LI> Genaxxon Bioscience taq Polymerase
</LI></UL><UL><LI> 10x Genaxxon Bioscience taq Polymerase buffer containing 15 mM MgCl<SMALL>2</SMALL></LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><UL><LI> forward primer: sf_mdnA_1 (10 µM)
</LI></UL><UL><LI> reverse primer: r_mdnA_1 (10 µM)
</LI></UL><UL><LI> template DNA pARW089 (857,3 ng/ µl measured by nanotrop a second time)
</LI></UL><UL><LI> Genaxxon Bioscience MgCl<SMALL>2</SMALL> stock solution (50 mM)
</LI></UL><UL><LI> Manganese(II) chloride (Merck)
</LI></UL><UL><LI> MnCl<SMALL>2</SMALL></LI></UL><UL><LI> sterile filter
</LI></UL><UL><LI> Eppendorf Mastercycler personal, program IGMDNA1
</LI></UL><P><B>Method:</B></P><P><B><I>Production of 50 mM Manganese(II) chloride stock solution</I></B></P><UL><LI> M<SMALL>MnCl<SMALL>2</SMALL></SMALL> = 161.81 g/ mol
</LI></UL><UL><LI> 50 mM in 1 ml needed --&gt; m<SMALL>MnCl<SMALL>2</SMALL></SMALL> = 8.1 mg
</LI></UL><UL><LI> sterile filtration: pore size of filter: 0.2 µm
</LI></UL><P><B><I>Error prone PCR of mdnA</I></B></P><UL><LI> increasing MnCl<SMALL>2</SMALL> concentration beginning with 0 mM in 0.05 mM steps, ending with 0.15 mM (--&gt; sample 1-4) </LI></UL><UL><LI> increasing MgCl<SMALL>2</SMALL> concentration beginning with 1.5 mM, in 5 mM steps, ending with 15 mM (--&gt; sample 5-8)
</LI></UL><P><B>1. Mix for error prone PCR with increasing MnCl<SMALL>2</SMALL> concentration</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component </TH><TH> Stock concentration </TH><TH> Volume in µl </TH><TH> Final concentration
</TH></TR><TR><TD> Genaxxon Bioscience Taq Polymerase buffer containing MgCl<SMALL>2</SMALL>) </TD><TD>15 mM (MgCl<SMALL>2</SMALL>) </TD><TD> 5 </TD><TD> 1.5 mM MgCL<SMALL>2</SMALL></TD></TR><TR><TD> MgCl<SMALL>2</SMALL></TD><TD> 50 mM </TD><TD>5.5 </TD><TD> 5.5 mM
</TD></TR><TR><TD> dNTPs </TD><TD> 10 mM </TD><TD> 2.0 </TD><TD> 0.4 mM
</TD></TR><TR><TD> Primer (each) </TD><TD> 10 µM </TD><TD> 2.5 </TD><TD> 0.5 µM
</TD></TR><TR><TD> DNA (diluted 1:100)</TD><TD> 8.57 ng/ µl </TD><TD> 5.58 </TD><TD> 0.7 ng
</TD></TR><TR><TD> Polymerase </TD><TD> 1 </TD><TD> 5 U
</TD></TR></TBODY></TABLE><P>in addition MnCl<SMALL>2</SMALL> and H<SMALL>2</SMALL>O as shown following
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Sample number </TH><TH> MnCl<SMALL>2</SMALL> in µl </TH><TH> Final concentration MnCl<SMALL>2</SMALL></TH><TH> H<SUB>2</SUB>O in µl
</TH></TR><TR><TD> 1 </TD><TD>0 </TD><TD> 0 </TD><TD> 25.52
</TD></TR><TR><TD> 2 </TD><TD> 0.5 </TD><TD>0.05 mM </TD><TD> 23.02
</TD></TR><TR><TD> 3 </TD><TD> 1.0 </TD><TD> 0.1 mM </TD><TD> 22.52
</TD></TR><TR><TD> 4 </TD><TD> 1.5 </TD><TD> 0.15 mM </TD><TD> 22.02
</TD></TR></TBODY></TABLE><P><B>2. Mix for error prone PCR with increasing MgCl<SMALL>2</SMALL> concentration</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component </TH><TH> Stock concentration </TH><TH> Volume in µl </TH><TH> Final concentration
</TH></TR><TR><TD> Genaxxon Bioscience Taq Polymerase buffer containing MgCl<SMALL>2</SMALL>) </TD><TD>15 mM (MgCl<SMALL>2</SMALL>) </TD><TD> 5 </TD><TD> 1.5 mM MgCL<SMALL>2</SMALL></TD></TR><TR><TD> dNTPs </TD><TD> 10 mM </TD><TD> 2.0 </TD><TD> 0.4 mM
</TD></TR><TR><TD> Primer (each) </TD><TD> 10 µM </TD><TD> 2.5 </TD><TD> 0.5 µM
</TD></TR><TR><TD> DNA (diluted 1:100)</TD><TD> 8.57 ng/ µl </TD><TD> 8.0 </TD><TD> 5 ng
</TD></TR><TR><TD> Polymerase </TD><TD> 1 </TD><TD> 5 U
</TD></TR></TBODY></TABLE><P>in addition MgCl<SMALL>2</SMALL> and H<SMALL>2</SMALL>O as shown following
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Sample number </TH><TH> MnCl<SMALL>2</SMALL> in µl </TH><TH> Final concentration MnCl<SMALL>2</SMALL></TH><TH> H<SUB>2</SUB>O in µl
</TH></TR><TR><TD> 5 </TD><TD> 0 </TD><TD> 1.5 mM </TD><TD> 29.0 µl
</TD></TR><TR><TD> 6 </TD><TD> 3.5 </TD><TD> 5 mM </TD><TD> 17.5 µl
</TD></TR><TR><TD> 7 </TD><TD> 7.0 </TD><TD> 10 mM </TD><TD> 14.0 µl
</TD></TR><TR><TD> 8 </TD><TD> 10.0 </TD><TD> 15 mM </TD><TD> 10.5 µl
</TD></TR></TBODY></TABLE><P><B>3. Complete pipet scheme</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component in µl </TH><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4 </TH><TH> 5 </TH><TH> 6 </TH><TH> 7 </TH><TH> 8
</TH></TR><TR><TD> Template DNA </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8
</TD></TR><TR><TD> dNTPs </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> 10x Genaxxon polymerase buffer (with MgCl<SMALL>2</SMALL>) </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5
</TD></TR><TR><TD> MgCl<SMALL>2</SMALL> (50 mM) </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 0 </TD><TD> 3.5 </TD><TD> 7.0 </TD><TD> 10.5
</TD></TR><TR><TD> MnCl<SMALL>2</SMALL> (5 mM) </TD><TD> 0 </TD><TD> 0.5 </TD><TD> 1 </TD><TD> 1.5 </TD><TD> 0 </TD><TD> 0 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> Forward primer (sf_mdnA_1) </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5
</TD></TR><TR><TD> Reverse primer (r_mdnA_1) </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5
</TD></TR><TR><TD> H<SMALL>2</SMALL>O </TD><TD> 23.5 </TD><TD> 23.0 </TD><TD> 22.5 </TD><TD> 22.0 </TD><TD> 29.0 </TD><TD> 25.5 </TD><TD> 22.0 </TD><TD> 18.5
</TD></TR></TBODY></TABLE><P><B>4. Temperature profile</B></P><UL><LI> see 12th labday
</LI></UL><UL><LI> Eppendorf Mastercycler personal, program IGMDNA1
</LI></UL><P><B>Results:</B></P><UL><LI> 8 PCR products containing mutated mdnA
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Verification of these PCR products by agarose gel electrophoresis
</LI></UL><UL><LI> Purification and cloning of positive samples
</LI></UL><UL><LI> Sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_of_mutated_mdnA">Agarose gel electrophoresis of mutated mdnA</SPAN></H3><P><B>Investigators:</B> Nadine, Jessica, Steffi, Nicole </P><P><B>Time:</B> 2011-06-21,14:00-17:00 </P><P><B>Aim:</B> Verification of random mutated mdnA (mutations are based on increased MnCl<SMALL>2</SMALL> and MgCl<SMALL>2</SMALL> concentration) </P><P><B>Materials:</B></P><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> 8 PCR products of mutated mdnA, see
</LI></UL><UL><LI> DNA Ladder Gene Ruler, 100bp plus, DNA Ladder (Fermentas)</LI></UL><UL><LI> 6x Loading Dye (Fermentas)</LI></UL><P><B>Method:</B></P><P><B>1. Production of two 1 % agarose gel</B></P><P>For each gel:</P><UL><LI> m<SMALL>Agarose</SMALL> = 0.5 g in 50 ml 1x TAE
</LI></UL><UL><LI> adding 2 µl gel red
</LI></UL><P><B>2. Loading samples</B></P><UL><LI> 30 µl PCR product and 6x loading dye
</LI></UL><UL><LI> 2 µl DNA ladder gene ruler
</LI></UL><P><B>Positions of marker and sample</B></P><P><I>Gel 1</I></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> 1 </TD><TD> marker </TD><TD> 2 </TD></TR><TR><TD> 2 </TD><TD> mdnA 1 </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 3 </TD><TD> mdnA 2 </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 4 </TD><TD> mdnA 3 </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 5 </TD><TD> mdnA 4 </TD><TD> 36 </TD><TD> 276
</TD></TR></TBODY></TABLE><P><I>Gel 2</I></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> 1 </TD><TD> marker </TD><TD> 2 </TD></TR><TR><TD> 2 </TD><TD> mdnA 5 </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 3 </TD><TD> mdnA 6 </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 4 </TD><TD> mdnA 7 </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 5 </TD><TD> mdnA 8 </TD><TD> 36 </TD><TD> 276
</TD></TR></TBODY></TABLE><P>3. Run
</P><UL><LI> 100 V
</LI></UL><UL><LI> time: 1 h
</LI></UL><P><B>Results:</B></P><UL><LI> Gel Extraction DNA excision: lane 1-4, 5,7: ~ 276 bp band
</LI></UL><UL><LI> stored at -20°C in orange box (iGEM 2011 UP DNA)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> DNA extraction
</LI></UL><UL><LI> Cloning
</LI></UL><UL><LI> Sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Over_night_culture_of_E._coli_BL21_containing_BBa_K404304_pSB1C3_.28biobrick_iGEM_Freiburg_2010.29_from_Sven">Over night culture of <I>E. coli</I> BL21 containing BBa_K404304_pSB1C3 (biobrick iGEM Freiburg 2010) from Sven</SPAN></H3><P><B>Investigators:</B> Jessica, Nadine, Stefan, Steffi </P><P><B>Aim:</B> Culturing <I>E. coli</I> containing BBa_K404304_pSB1C3 biobrick for midi-prep and using for cloning afterwards</P><P><B>Time:</B> 2011-06-21,16:50-17:00</P><P><B>Materials:</B></P><UL><LI> cells: <I>E. coli</I> BL21 containing BBa_K404304_pSB1C3 (biobrick iGEM Freiburg 2010) from Sven
</LI></UL><UL><LI> 50 ml LB media
</LI></UL><UL><LI> 50 µl Chloramphenicol (100 mg/ml)</LI></UL><P><B>Method:</B></P><UL><LI> add antibiotic to LB
</LI></UL><UL><LI> inoculation of cells
</LI></UL><UL><LI> 37 °C, 250 rpm, 17 hours
</LI></UL><P><B>Results:</B></P><P><B>Further tasks:</B></P><UL><LI> remove cells from shaker at 10:00 (Nici :D)
</LI></UL><UL><LI> do midi-prep
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_E._coli_XL_1_blu_and_RV_308_.2B_puK">Transformation of <I>E. coli</I> XL 1 blu and RV 308 + puK </SPAN></H3><P><B>Investigators:</B> Niels </P><P><B>Aim:</B> Check the quality of competent cells</P><P><B>Time:</B> 2011-06-21,10:00-12:30</P><P><B>Materials:</B></P><UL><LI> cells: <I>E. coli</I> XL 1 blu and RV 308
</LI></UL><UL><LI> 50 ml LB media
</LI></UL><UL><LI> 50 µl ampicillin (100 mg/ml)</LI></UL><P><B>Method:</B></P><P>protocol: addition of 1 µl plasmid (puK) to cells in 1.5 ml Eppi,
</P><UL><LI> incubation 30 min on ice,
</LI></UL><UL><LI> heat shock 90 sec at 42°C,
</LI></UL><UL><LI> addition of 750 µl LB medium,
</LI></UL><UL><LI> incubation at 37 °C for 60 min in Eppendorf thermomixer at 450 rpm,
</LI></UL><UL><LI> centrifugation at 2000xg; for 5 min,
</LI></UL><UL><LI> decandation of supernatant,
</LI></UL><UL><LI> resuspension of pellet in approx. 100 µl,
</LI></UL><UL><LI> plating on LB medium with 1,5 % agar and LB medium with 1,5 % agar, 100 µg/ml ampicillin,
</LI></UL><UL><LI> storage over night at 37°C
</LI></UL><P><B>Results:</B></P><UL><LI> negative
</LI></UL><P><B>Further tasks:</B></P><UL><LI> repeat
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="21th_Labday_2011-06-22">21th Labday 2011-06-22</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Gel_extraction_of_mutated_mdnA">Gel extraction of mutated mdnA </SPAN></H3><P><B>Investigators:</B> Katharina, Nicole </P><P><B>Aim:</B> Extraction of mdnA, which was amplified and mutated (using error prone PCR)</P><P><B>Time:</B> 2011-06-22,12:15-13:30</P><P><B>Materials:</B></P><UL><LI> Qiagen QIAquick Gel extraction Kit
</LI></UL><UL><LI> PCR products 1-5 and 7 containing mutated mdnA
</LI></UL><P><B>Method:</B></P><UL><LI> performing gel extraction based on manufacturer's protocol, <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=UP_Qiagen_QIAquick_Gel_Extraction_kit.pdf" class="new" title="UP Qiagen QIAquick Gel Extraction kit.pdf">Media: UP_Qiagen_QIAquick_Gel_Extraction_kit.pdf</A></LI></UL><UL><LI> elute in 30 µl buffer EB
</LI></UL><P><B>Results:</B></P><UL><LI> purified PCR products of mutated mdnA (1-5, 7)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> restriction enzyme digestion with purified mdnA and pARW089 using AatII/ NarI
</LI></UL><UL><LI> Verification by agarose gel electrophoresis
</LI></UL><UL><LI> Ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Midiprep_of_pSB1C3_carrying_cells">Midiprep of pSB1C3 carrying cells</SPAN></H3><P><B>Investigators:</B> Nadine, Vanessa </P><P><B>Aim:</B></P><UL><LI> Midiprep of pSB1C3</LI></UL><UL><LI> vector for biobricks (RFC25)
</LI></UL><P><B>Time:</B> 2011-06-22, 14:00-16:00 </P><P><B>Materials:</B></P><UL><LI> Qiagen Plasmid Plus Midi Kit </LI></UL><UL><LI> over night cultures: <I>E. coli</I> BL21 containing BBa_K404304_pSB1C3 (biobrick iGEM Freiburg 2010) from Sven
</LI></UL><UL><LI> NanoDrop
</LI></UL><P><B>Method:</B></P><P>Midiprep </P><UL><LI> using Qiagen Plasmid Plus Midi Kit Quick-Start Protocol, <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=UP_QIAGEN_Plasmid_Plus_Midi_Kit.pdf" class="new" title="UP QIAGEN Plasmid Plus Midi Kit.pdf">Media: UP_QIAGEN_Plasmid_Plus_Midi_Kit.pdf</A></LI></UL><P><B>Output:</B> 4 x DNA Eppi Tubes: pSB1C3, 22.06.2011, storage: orange box (iGEM DNA, -20°C)</P><P><B>Results:</B></P><UL><LI> concentration:
</LI></UL><UL><LI> tube 1: 284.6 ng/µl, 260/280 = 1.90, 260/230 = 2.32
</LI></UL><UL><LI> tube 2: 252.6 ng/µl, 260/280 = 1.91, 260/230 = 2.34
</LI></UL><UL><LI> tube 3: 235.0 ng/µl, 260/280 = 1.89, 260/230 = 2.01
</LI></UL><UL><LI> tube 4: 238.4 ng/µl, 260/280 = 1.9, 260/230 = 2.32
</LI></UL><P><B>Further tasks:</B></P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_and_agarose_gel_electrophoresis_of_mutated_mdnA">Restriction enzyme digestion and agarose gel electrophoresis of mutated mdnA</SPAN></H3><P><B>Investigators:</B> Katharina, Nicole </P><P><B>Time:</B> 2011-06-22,14:00-18:00 </P><P><B>Aim:</B> Restriction enzyme digestion of random mutated mdnA and pARW089 (vector) and verification using gel electrophoresis</P><P><B>Materials:</B></P><UL><LI> Restriction enzymes: AatII, NarI
</LI></UL><UL><LI> NEB Buffer 4
</LI></UL><UL><LI> PCR products of random mutated mdnA(1-5, 7), purified from gel
</LI></UL><UL><LI> vector pARW089
</LI></UL><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> DNA Ladder Gene Ruler, 100bp plus, DNA Ladder (Fermentas)</LI></UL><UL><LI> 6x Loading Dye (Fermentas)
</LI></UL><UL><LI> 1kb DNA ladder (NEB)</LI></UL><P><B>Method:</B></P><P><B>1. Restriction enzyme digestion</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD> components </TD><TD align="right">volume of PCR products /µl</TD><TD align="right">volume of pARW089 /µl
</TD></TR><TR><TD> DNA </TD><TD align="right">30 </TD><TD align="right"> 6
</TD></TR><TR><TD> NEB Buffer 4(10x)</TD><TD align="right"> 4</TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme AatII</TD><TD align="right"> 1</TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme NarI</TD><TD align="right"> 1</TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 4</TD><TD align="right"> 1
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 40</TD><TD align="right"> 10
</TD></TR></TBODY></TABLE><P><B>2. Production of two 1 % agarose gels</B></P><UL><LI> m<SMALL>Agarose</SMALL> = 1.0 g in 100 ml 1x TAE
</LI></UL><UL><LI> adding 4 µl gel red
</LI></UL><P><B>3. Loading samples</B></P><P>a. restriction enzyme digestion products of random mutated mdnA
</P><UL><LI> 30 µl PCR product and 6 µl 6x loading dye
</LI></UL><P>b. restriction enzyme digestion product of vector pARW089
</P><UL><LI> 10 µl PCR product and 6x loading dye
</LI></UL><UL><LI> 2 µl DNA ladder gene ruler, 100bp plus (Fermentas)
</LI></UL><UL><LI> 2 µl 1 kb DNA ladder (NEB)
</LI></UL><P><B>Positions of marker and sample</B></P><P><I>Gel 1</I></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> 1 </TD><TD> marker </TD><TD> 2 </TD></TR><TR><TD> 2 </TD><TD> mdnA 1 </TD><TD> 36 </TD></TR><TR><TD> 3 </TD><TD> mdnA 2 </TD><TD> 36 </TD></TR><TR><TD> 4 </TD><TD> mdnA 3 </TD><TD> 36 </TD></TR><TR><TD> 5 </TD><TD> mdnA 4 </TD><TD> 36 </TD></TR><TR><TD> 6 </TD><TD> mdnA 5 </TD><TD> 36 </TD></TR></TBODY></TABLE><P><I>Gel 2</I></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> 1 </TD><TD> marker </TD><TD> 2 </TD></TR><TR><TD> 2 </TD><TD> mdnA 7 </TD><TD> 36 </TD></TR><TR><TD> 5 </TD><TD> pARW089 </TD></TR><TR><TD> 6 </TD><TD> marker, 1kb DNA ladder (NEB) </TD><TD> 2 </TD></TR></TBODY></TABLE><P>4. Run
</P><UL><LI> 100 V
</LI></UL><UL><LI> time: 1 - 1.5 h
</LI></UL><P><B>Results:</B></P><P><A href="/File:UP_AG_EPPCR_mdna1-5_NA_2011-06-22.jpg" title="File:UP AG EPPCR mdna1-5 NA 2011-06-22.jpg">300px</A></P><P><A href="/File:UP_AG_EPPCR_mdna7_pARW089_KB_2011-06-22.jpg" title="File:UP AG EPPCR mdna7 pARW089 KB 2011-06-22.jpg">300px</A></P><UL><LI> stored in freezer
</LI></UL><P><B>Further tasks:</B></P><UL><LI> DNA extraction
</LI></UL><UL><LI> Cloning
</LI></UL><UL><LI> Sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Over_night_culture_of_E._coli_containing_pEX_respectively_pBAD_from_Tobias">Over night culture of <I>E. coli</I> containing pEX respectively pBAD from Tobias</SPAN></H3><P><B>Investigators:</B> Jessica, Nadine, Nicole </P><P><B>Aim:</B> Culturing <I>E. coli</I> containing PEX respectively pBAD for midi-prep</P><P><B>Time:</B> 2011-06-22,16:50-17:00</P><P><B>Materials:</B></P><UL><LI> cells: <I>E. coli</I> containing pEX respectively pBAD from Tobias
</LI></UL><UL><LI> 50 ml LB media
</LI></UL><UL><LI> 50 µl Ampicillin (100? mg/ml)</LI></UL><P><B>Method:</B></P><UL><LI> add antibiotic to LB
</LI></UL><UL><LI> inoculation of cells
</LI></UL><UL><LI> 37 °C, 250 rpm, 17 hours
</LI></UL><P><B>Results:</B></P><P><B>Further tasks:</B></P><UL><LI> remove cells from shaker at 10:00
</LI></UL><UL><LI> do midi-prep
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="22th_Labday_2011-06-23">22th Labday 2011-06-23</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Midiprep_of_pEX_respectively_pBAD_carrying_cells">Midiprep of pEX respectively pBAD carrying cells</SPAN></H3><P><B>Investigators:</B> Jessica </P><P><B>Aim:</B></P><UL><LI> Midiprep of pEX respectively pBAD</LI></UL><UL><LI> expression vectors
</LI></UL><P><B>Time:</B> 2011-06-23, 10:30-12:30 </P><P><B>Materials:</B></P><UL><LI> Qiagen Plasmid Plus Midi Kit </LI></UL><UL><LI> over night cultures: <I>E. coli</I> containing pEX respectively pBAD
</LI></UL><UL><LI> NanoDrop
</LI></UL><P><B>Method:</B></P><P>Midiprep </P><UL><LI> using Qiagen Plasmid Plus Midi Kit Quick-Start Protocol, <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=UP_QIAGEN_Plasmid_Plus_Midi_Kit.pdf" class="new" title="UP QIAGEN Plasmid Plus Midi Kit.pdf">Media: UP_QIAGEN_Plasmid_Plus_Midi_Kit.pdf</A></LI></UL><P><B>Results:</B></P><UL><LI> concentration:
</LI></UL><UL><LI> pEX tube 1: 36.4 ng/µl
</LI></UL><UL><LI> pEX tube 2: 62.4 ng/µl
</LI></UL><UL><LI> pBAD tube 1: 71.6 ng/µl
</LI></UL><UL><LI> pBAD tube 2: 94.6 ng/µl
</LI></UL><P><B>Output:</B></P><UL><LI> 2 x DNA Eppi Tubes: pBAD, 23.06.2011, storage: orange box (iGEM DNA, -20°C)
</LI></UL><UL><LI> concentration may be okay as pBAD is a low-copy plasmid
</LI></UL><UL><LI> pEX tubes thrown away because of low concentration
</LI></UL><P><B>Further tasks:</B></P><UL><LI> redo culturing and midiprep for pEX
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Over_night_culture_of_E._coli_XL1-Blue_and_RV308_for_competent_cells">Over night culture of <I>E. coli</I> XL1-Blue and RV308 for competent cells</SPAN></H3><P><B>Investigators:</B> Stefan </P><P><B>Aim:</B> Culturing cells to produce competent cells</P><P><B>Time:</B> 2011-06-23,15:30-16:00</P><P><B>Materials:</B></P><UL><LI> XL1-Blue: 15 ml LB medium with 100 µg/mL tetracycline out of tetracycline stock solution 50 mg/mL
</LI></UL><UL><LI> RV308: 15 ml LB medium without streptomycin (not available)
</LI></UL><P><B>Method:</B></P><UL><LI> add antibiotic to LB
</LI></UL><UL><LI> inoculation of cells
</LI></UL><UL><LI> 30 °C, 250 rpm, 18 hours
</LI></UL><UL><LI> 37 °C, 250 rpm, 5 hours
</LI></UL><UL><LI> inoculate 200 mL fresh culture with 2 mL from starter culture
</LI></UL><P><B>Results:</B></P><P><B>Further tasks:</B></P><UL><LI> produce competent cells
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Over_night_culture_of_E._coli_containing_vector_pBAD_resp._pEX_for_midiprep_resp._glycerol_stocks">Over night culture of <I>E. coli</I> containing vector pBAD resp. pEX for midiprep resp. glycerol stocks</SPAN></H3><P><B>Investigators:</B> Nicole </P><P><B>Aim:</B> Culturing cells to do midiprep and to produce glycerol stocks</P><P><B>Time:</B> 2011-06-23,17:30-18:00</P><P><B>Materials:</B></P><UL><LI><I>E. coli</I> containing expression vector pBAD: 5 ml LB medium with 5 µl ampicilin
</LI></UL><UL><LI><I>E. coli</I> containing expression vector pEX: 50 ml LB medium with 50 µl ampicilin</LI></UL><P><B>Method:</B></P><UL><LI> add antibiotic to LB
</LI></UL><UL><LI> inoculation of cells
</LI></UL><UL><LI> 37 °C, over night</LI></UL><P><B>Results:</B></P><P>Over night culture of</P><UL><LI><I>E. coli</I> containing expression vector pBAD
</LI></UL><UL><LI><I>E. coli</I> containing expression vector pEX</LI></UL><P><B>Further tasks:</B></P><UL><LI> Midiprep for <I>E. coli</I> containg vector pEX
</LI></UL><UL><LI> glycerol stocks of <I>E. coli</I> containg vector pEX as well as fpr <I>E. coli</I> containg vector pBAD</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Amplification_of_mdnA_and_GeneIII_for_phage_display">Amplification of mdnA and GeneIII for phage display</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine </P><P><B>Aim:</B></P><P>amplificate mdnA of pARW089 with primers pf_mdnA_sfi_1 and pr_mdnA_sfi_1 (phage display strategy 1)</P><P>amplificate mdnA of pARW089 with primers pf_mdnA_iGEM_EheI and pr_mdnA_iGEM_AatII (phage display strategy 2)</P><P>amplificate GeneIII of pak100 with primers pf_iGEM_GenIII_NgoMIV and pr_GenIII_iGEM_AatII (phage display strategy 2)</P><P><B>Time:</B> 2011-06-23, 10:00-12:00, 15:00-17:00</P><P><B>Materials/Methods:</B></P><P>1. PCR </P><UL><LI> 0,8 ng Vector pARW089
</LI></UL><UL><LI> Thermo Hybraid Px2 (conditions see <A href="#Design_of_PCR_conditions_for_amplifying_mdnA_for_Phage_display">2011-06-08</A>, changes: extension time 40 s)
</LI></UL><UL><LI> 0,25 µl OneTaq DNA Polymerase (NEB)
</LI></UL><UL><LI> 1 µl dNTPs
</LI></UL><UL><LI> 1 µl per primer
</LI></UL><UL><LI> 31,75 µl DNase free water
</LI></UL><UL><LI> 10 µl 5xOneTaq Standard Reaction Buffer
</LI></UL><P>2. gel electrophoresis (analytic)
</P><UL><LI> 5 µl PCR product, agarose gel: 1%)
</LI></UL><UL><LI> DNA Ladder Gene Ruler, 100bp plus, DNA Ladder (Fermentas)
</LI></UL><P><B>Positions of marker and sample</B></P><P><I>Gel </I></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> M </TD><TD> marker </TD><TD> 2 </TD></TR><TR><TD> 1 </TD><TD> mdnA with SfiI </TD><TD> 7 </TD><TD> 224
</TD></TR><TR><TD> 2 </TD><TD> mdnA iGEM </TD><TD> 7 </TD><TD> ca. 230
</TD></TR><TR><TD> 3 </TD><TD> geneIII iGEM</TD><TD> 7 </TD><TD> ca. 520
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><P><A href="/File:UP_PCR_sfi_mdna,mdna_rfc25,genIII.png" title="File:UP PCR sfi mdna,mdna rfc25,genIII.png">File:UP PCR sfi mdna,mdna rfc25,genIII.png</A></P><P>PCR for strategy 1 (lane number 1) probably did not work because the annealing temperature of the forward primer is located at 42°C (not 60 °C)
</P><P><B>Furter tasks:</B></P><UL><LI> purification of PCR products
</LI></UL><UL><LI> order new forward primer for strategy 1 with higher annealing temperature
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Design_of_primers_and_planing_of_further_experiments">Design of primers and planing of further experiments </SPAN></H3><P><B>Investigators:</B> Nadine, Nicole </P><P><B>Time:</B> 2011-06-23, 15:00-18:00 </P><P><B>Aim:</B> Design of primers and detailed planing of further experiment
</P><P><B>Materials:</B></P><P>1. Primer design
</P><UL><LI> Geneious Pro 5.1.7
</LI></UL><UL><LI> Oligo Calc: Oligonucleotide Properties Calculator (<A href="http://www.basic.northwestern.edu/biotools/oligocalc.html" class="external free" rel="nofollow">http://www.basic.northwestern.edu/biotools/oligocalc.html</A>)</LI></UL><P><B>Method:</B></P><P><B>1. Primer Design</B></P><UL><LI> add an start codon (ATG) to the previously designed primer pf_mdnA_mycN_xbal
</LI></UL><P><B>2. Detailed planing of further experiments</B></P><P><B>A. Control experiment of error prone PCR of mdnA and subsequent restriction enzyme digestion using NarI and AatII</B></P><P>Aim: Analysis of three lines after agarose gel electrophoresis after restriction enzyme digestion of mdnA using NarI and AatII; see
</P><P>Idea: DNA is not completly single stranded; would be avoid by heating mixture after restriction enzyme digestion for 10-20 min at 65°C</P><P><B>B. Random mutagenesis of mdnA and determination of mutation rate</B></P><P>Idea: using an increased MnCl<SMALL>2</SMALL> concentration to mutate mdnA and finally sequencing of mdnA to determine the mutation rate</P><P><B>C. mdnA as iGEM expression part in vector pEX or pBAD</B></P><P><B>D. mdnA as biobrick </B></P><P>Idea: Cloning of mdnA in iGEM cloning vector to submit as biobrick</P><P><B>E. Repetition of Elke's experiments as control for further experiments</B></P><P>Idea: Further, purify mdnA with the use of antibodies (anti myc-tag) and without HPLC. To compare which strategy is more efficient (myc-tag or HPLC) HPLC experiment as control is necessary.</P><P><B>F. mdnA with myc-tag (N- and C-terminal) in iGEM expression vector pEX or pBAD</B></P><P>Idea: myc-tagging of mdnA to purify it with antibodies (without use of HPLC)</P><P><B>G. Expression and purification of mdnA (with myc-tag)</B></P><P>Idea: Determine the efficiency of myc-tag/ antibody to purify mdnA.</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> To do </TH><TH> Experimental tasks
</TH></TR><TR><TD><B>A.</B></TD><TD><B>Control experiment for error prone PCR (increased MnCl<SMALL>2</SMALL> concentration) of mdnA (vector pARW089)</B></TD></TR><TR><TD> Preparative agarose gel electrophoresis
</TD></TR><TR><TD> DNA extraction and purification of (positive) samples
</TD></TR><TR><TD> Restriction enzyme digestion of mdnA using NarI and AatII; Splitting the samples fifty-fifty, heat one half of each sample 10-20 min at 65°C
</TD></TR><TR><TD> Analytic agarose gel electrophoresis of each sample (heated and non-heated)
</TD></TR><TR><TD><B>B.</B></TD><TD><B>Random mutagenesis of mdnA and determination of mutation rate</B></TD></TR><TR><TD> Error prone PCR of mdnA (in vector pARW089) using increased MnCl<SMALL>2</SMALL> concentration
</TD></TR><TR><TD> Preparative agarose gel electrophoresis
</TD></TR><TR><TD> DNA extraction and purification of the positive samples
</TD></TR><TR><TD> Restriction enzyme digestion of mdnA and pARW089 using NarI and AatII
</TD></TR><TR><TD> Preparative agarose gel electrophoresis of each sample
</TD></TR><TR><TD> DNA extraction and purification of the positive samples
</TD></TR><TR><TD> Ligation
</TD></TR><TR><TD> Transformation
</TD></TR><TR><TD> DNA isolation of one clone using midiprep/ miniprep
</TD></TR><TR><TD> Sequencing
</TD></TR><TR><TD><B>C.</B></TD><TD><B>mdnA as expression part</B></TD></TR><TR><TD> PCR of mdnA in pARW089 using pf_mdnA_iGEM_EheI and pr_mdnA_iGEM_AatII (primer designed by phage display group)
</TD></TR><TR><TD> Preparative agarose gel electrophoresis
</TD></TR><TR><TD> DNA extraction and purification of the positive sample
</TD></TR><TR><TD> Restriction enzyme digestion of mdnA and iGEM expression vector pEX or pBAD
</TD></TR><TR><TD> Preparative agarose gel electrophoresis
</TD></TR><TR><TD> DNA extraction and purification of the samples
</TD></TR><TR><TD> Ligation
</TD></TR><TR><TD> Transformation
</TD></TR><TR><TD> DNA isolation of one clone using midiprep/ miniprep
</TD></TR><TR><TD> Sequencing
</TD></TR><TR><TD><B>D.</B></TD><TD><B>mdnA in cloning vector</B></TD></TR><TR><TD> PCR of mdnA in pARW089 using pf_mdnA_iGEM_EheI and pr_mdnA_iGEM_AatII (primer designed by phage display group)
</TD></TR><TR><TD> Preparative agarose gel electrophoresis
</TD></TR><TR><TD> DNA extraction and purification of the positive sample
</TD></TR><TR><TD> Restriction enzyme digestion of mdnA and iGEM expression vector pEX or pBAD
</TD></TR><TR><TD> Preparative agarose gel electrophoresis
</TD></TR><TR><TD> DNA extraction and purification of the samples
</TD></TR><TR><TD> Ligation
</TD></TR><TR><TD> Transformation
</TD></TR><TR><TD> DNA isolation of one clone using midiprep/ miniprep
</TD></TR><TR><TD> Sequencing
</TD></TR><TR><TD><B>E.</B></TD><TD><B>Repetition of Elke's experiments as control for further experiments</B></TD></TR><TR><TD> Expression and purification (HPLC; without myc-tag) of mdnA x mdnA in pARW089x mdnA in iGEM expression vector pEX or pBAD
</TD></TR><TR><TD><B>F.</B></TD><TD><B>mdnA with myc-tag (N- and C-terminal) in iGEM expression vector pEX or pBAD</B></TD></TR><TR><TD> PCR of mdnA in pARW089 usingx C-terminal: pf_mdnA_mycC_Xbal (forward) and pr_mdnA_mycC_PstI_NotI_SpeI (reverse)x N-terminal: pf_mdnA_mycN_Xbal_1 (forward) and pr_mdnA_mycN_PstI_NotI_SpeI (reverse)
</TD></TR><TR><TD> Preparative agarose gel electrophoresis
</TD></TR><TR><TD> DNA extraction and purification of the positive samples
</TD></TR><TR><TD> Restriction enzyme digestion of mdnA and iGEM expression vector pEX or pBAD
</TD></TR><TR><TD> Preparative agarose gel electrophoresis
</TD></TR><TR><TD> DNA extraction and purification of the samples
</TD></TR><TR><TD> Ligation
</TD></TR><TR><TD><B>G.</B></TD><TD><B>Expression and purification of mdnA (with myc-tag)</B>x N-terminal myc-tagx C-terminal myc-tag
</TD></TR><TR><TD> Transformation of <I>E. coli</I> with pEX/ pBAD containing mdnA with myc-tag and with pARW089
</TD></TR></TBODY></TABLE><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="23th_Labday_2011-06-24">23th Labday 2011-06-24</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="purification_of_PCR_products_2011-06-23">purification of PCR products 2011-06-23</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B></P><UL><LI> purification of PCR products mdnA iGEM and geneIII iGEM (lanes number 2 + 3) for phage display (strategy 2)</LI></UL><P><B>Time:</B> 2011-06-24, 14:30-15:30 </P><P><B>Materials/Methods:</B></P><UL><LI> QIAquick Gel Extraction Kit (250)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> digestion of the two fragments with EheI and AgeI (mdnA iGem) and AatII and NgoMIV (geneIII iGEM)
</LI></UL><UL><LI> digestion of vector pARW089 with EheI and AatII
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="preparation_of_competent_E._coli_XL1_and_RV_308">preparation of competent <I>E. coli</I> XL1 and RV 308</SPAN></H3><P><B>Investigators:</B> Stefan</P><P><B>Aim:</B></P><UL><LI>get competent cells of <I>E. coli</I> XL 1 blue and RV 308</LI></UL><P><B>Time:</B> 2011-06-24, 14:00-19:30 </P><P><B>Materials:</B></P><UL><LI> over night cultures: <I>E. coli</I> XL 1 blu and RV 308</LI></UL><P><B>Method:</B></P><P>Work always sterile and cold and speedy!
</P><UL><LI> All volumes deal with the common cellline!
</LI></UL><UL><LI> The cooling-centrifuge is in the tool shed, cool down to 4°C early enough, close the lid correctly!
</LI></UL><UL><LI> Prepare early enough min. 100 Eppis (1,5µl) (per cellline) and cool down to -80°C before using
</LI></UL><UL><LI> Use Milipore-filter for sterile CaCl2 , keep cool!
</LI></UL><UL><LI>prepare 15ml LB-Medium (or DYT) with the specific antibiotic (XL1-blue Tet, BL21 none!), inoculate and incubate over night
</LI></UL><UL><LI> prepare 200ml LB-Medium (or DYT) with the specific antibiotic, inoculate with 2ml of the over-night-culture. Nurture the culture until OD600 at 0,35 (0,2-0,5) (if the OD is too high, the cell won’t be competent)
</LI></UL><UL><LI> keep cell suspension in sterile falcons (50ml) 20 min on ice, then centrifuge for 20min; 4°C; 2500g
</LI></UL><UL><LI>discard supernatant, carefully resuspend on ice with 10ml cold CaCl2-solution (put a little of the 10ml solution in every falcon before!), pool every resuspended aliquot of one cell line and add 40ml CaCl2-solution (total volume 50ml). Keep 30 min on ice, then centrifuge for 20min; 4°C; 2500g
</LI></UL><UL><LI> discard supernatant, carefully resuspend pellet in 5,5ml CaCl2(80mM)/Glycerol (4:1), use the pippete boy and a sterile glas pipette to dissolve the pellet
</LI></UL><UL><LI> aliquot in Eppis á 60µl and store immediately at - 80 °C</LI></UL><P><B>Results:</B></P><UL><LI> 91 aliquots a 60 µL of XL1 and RF
</LI></UL><P><B>Output:</B></P><P><B>Further tasks:</B></P><P>test the cells
</P><P>Eppis in - 80°C in Boxen geräumt, aber noch nicht umbeschriftet</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Midiprep_of_pEX_carrying_cells">Midiprep of pEX carrying cells</SPAN></H3><P><B>Investigators:</B> Nadine </P><P><B>Aim:</B></P><UL><LI> Midiprep of pEX</LI></UL><P><B>Time:</B> 2011-06-24, 10:00-12:00 </P><P><B>Materials:</B></P><UL><LI> Qiagen Plasmid Plus Midi Kit </LI></UL><UL><LI> over night cultures: 50 ml <I>E. coli</I> containing pEX from Tobi
</LI></UL><UL><LI> NanoDrop
</LI></UL><P><B>Method:</B></P><P>Midiprep </P><UL><LI> using Qiagen Plasmid Plus Midi Kit Quick-Start Protocol, <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=UP_QIAGEN_Plasmid_Plus_Midi_Kit.pdf" class="new" title="UP QIAGEN Plasmid Plus Midi Kit.pdf">Media: UP_QIAGEN_Plasmid_Plus_Midi_Kit.pdf</A></LI></UL><P><B>Output:</B> 1x DNA Eppi Tubes: <B>#5 pEX, 24.06.2011</B>, storage: orange box (iGEM DNA, -20°C)</P><P><B>Results:</B></P><UL><LI> concentration:
</LI></UL><UL><LI> tube 1: 248.4 ng/µl
</LI></UL><P><B>Further tasks:</B></P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Glycerol_stocks_of_E._coli_cells_carrying_pEX_and_pBAD_plasmids">Glycerol stocks of <I>E. coli</I> cells carrying pEX and pBAD plasmids </SPAN></H3><P><B>Investigators:</B> Nadine </P><P><B>Aim:</B> Generating stocks for further research</P><P><B>Time:</B> 2011-06-24,10:00-10:15</P><P><B>Materials:</B></P><UL><LI> Glycerol </LI></UL><UL><LI> 5 ml over-night culter of <I>E. coli</I> pBAD </LI></UL><UL><LI> 50 ml over-night culter of <I>E. coli</I> pEX (also for midi-prep)</LI></UL><P><B>Method:</B></P><UL><LI> 700 µl of cell suspension </LI></UL><UL><LI> 300 µl glycerol </LI></UL><UL><LI> mixed gently and frozen by -80°C </LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="24th_Labday_2011-06-27">24th Labday 2011-06-27</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Control_experiment_fpr_error_prone_PCR_and_enzyme_digestion_of_mdnA">Control experiment fpr error prone PCR and enzyme digestion of mdnA </SPAN></H3><P><B>Investigators:</B> Nadine, Steffi, Katharina, Nicole </P><P><B>Time:</B> 2011-06-27, 14:30-16:30 </P><P><B>Aim:</B> control experiment to verify error prone PCR and enzyme digestion of mdnA, which experiment was done previously
</P><P>--&gt; error prone PCR using increased MnCl<SMALL>2</SMALL> concentration
</P><P><B>Materials:</B></P><UL><LI> protocol of experiment, which was done previously; see<A href="#Restriction_enzyme_digestion_and_agarose_gel_electrophoresis_of_mutated_mdnA">2011-06-22</A></LI></UL><UL><LI> Genaxxon Bioscience taq Polymerase
</LI></UL><UL><LI> 10x Genaxxon Bioscience taq Polymerase buffer containing 15 mM MgCl<SMALL>2</SMALL></LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><UL><LI> forward primer: sf_mdnA_1 (10 µM)
</LI></UL><UL><LI> reverse primer: r_mdnA_1 (10 µM)
</LI></UL><UL><LI> template DNA pARW089 (857,3 ng/ µl)
</LI></UL><UL><LI> Genaxxon Bioscience MgCl<SMALL>2</SMALL> stock solution (50 mM)
</LI></UL><UL><LI> MnCl<SMALL>2</SMALL></LI></UL><UL><LI> MnCl<SMALL>2</SMALL> stock solution (50 mM)
</LI></UL><P><B>Method:</B></P><P><B>1. Mix for error prone PCR with increasing MnCl<SMALL>2</SMALL> concentration</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component </TH><TH> Stock concentration </TH><TH> Volume in µl </TH><TH> Final concentration
</TH></TR><TR><TD> Genaxxon Bioscience Taq Polymerase buffer containing MgCl<SMALL>2</SMALL>) </TD><TD>15 mM (MgCl<SMALL>2</SMALL>) </TD><TD> 5 </TD><TD> 1.5 mM MgCL<SMALL>2</SMALL></TD></TR><TR><TD> MgCl<SMALL>2</SMALL></TD><TD> 50 mM </TD><TD>5.5 </TD><TD> 5.5 mM
</TD></TR><TR><TD> dNTPs </TD><TD> 10 mM </TD><TD> 2.0 </TD><TD> 0.4 mM
</TD></TR><TR><TD> Primer (sf_mdnA_1) </TD><TD> 10 µM </TD><TD> 2.5 </TD><TD> 0.5 µM
</TD></TR><TR><TD> Primer (r_mdnA_1) </TD><TD> 10 µM </TD><TD> 2.5 </TD><TD> 0.5 µM
</TD></TR><TR><TD> DNA (diluted 1:100)</TD><TD> 8.57 ng/ µl </TD><TD> 8.0 </TD><TD> 0.7 ng
</TD></TR><TR><TD> Polymerase </TD><TD> 1 </TD><TD> 5 U
</TD></TR></TBODY></TABLE><P>in addition MnCl<SMALL>2</SMALL> and H<SMALL>2</SMALL>O as shown following
</P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Sample number </TH><TH> MnCl<SMALL>2</SMALL> in µl </TH><TH> Final concentration MnCl<SMALL>2</SMALL></TH><TH> H<SUB>2</SUB>O in µl
</TH></TR><TR><TD> 1 </TD><TD>0 </TD><TD> 0 </TD><TD> 23.5
</TD></TR><TR><TD> 2 </TD><TD> 0.5 </TD><TD>0.05 mM </TD><TD> 23.0
</TD></TR><TR><TD> 3 </TD><TD> 1.0 </TD><TD> 0.1 mM </TD><TD> 22.5
</TD></TR></TBODY></TABLE><P><B>2. Temperature profile</B></P><UL><LI> see 12th labday
</LI></UL><UL><LI> Eppendorf Mastercycler personal, program IGMDNA1
</LI></UL><P><B>Results:</B></P><UL><LI> 3 PCR products containing random mutated mdnA
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Verification of these PCR products by agarose gel electrophoresis
</LI></UL><UL><LI> Extraction and purification these (positive) samples
</LI></UL><UL><LI> Enzyme digestion using NarI and AatII
</LI></UL><UL><LI> Heating the half of each samples 10-20 min by 65°C and the other half not
</LI></UL><UL><LI> Verification using agrose gel electrophoresis
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Midiprep_of_pJC354_WZA2_.28TorA-bla_vector.29_carrying_cells">Midiprep of pJC354_WZA2 (TorA-bla vector) carrying cells</SPAN></H3><P><B>Investigators:</B> Sebastian, Paul </P><P><B>Aim:</B></P><UL><LI> Midiprep of pJC354_WZA2 carrying cells using Quiagen Plasmid Plus Midi Kit </LI></UL><P><B>Materials:</B></P><UL><LI> Qiagen Plasmid Plus Midi Kit </LI></UL><UL><LI> over night cultures ###
</LI></UL><P><B>Output:</B> DNA Eppi Tube label (-20°C), Concentration: ~1mg/ml
</P><P><B>Further tasks:</B></P><UL><LI> Annealing of oligos for cleavage sites
</LI></UL><UL><LI> Digest of vector with XhoI and NheI
</LI></UL><UL><LI> Ligation of annealed oligos into the vector
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="25th_Labday_2011-06-28">25th Labday 2011-06-28</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_of_mutated_mdnA_2">Agarose gel electrophoresis of mutated mdnA </SPAN></H3><P><B>Investigators:</B> Katharina </P><P><B>Aim:</B> Verification of random mutated mdnA</P><P><B>Time:</B> 2011-06-26,12:45-13:30</P><P><B>Materials:</B></P><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> 3 PCR products of mutated mdnA
</LI></UL><UL><LI> DNA Ladder Gene Ruler, 100bp plus, DNA Ladder (Fermentas)</LI></UL><UL><LI> 6x Loading Dye (Fermentas)</LI></UL><P><B>Method:</B></P><P><B>1. Production of a 1 % agarose gel</B></P><UL><LI> m<SMALL>Agarose</SMALL> = 0.5 g in 50 ml 1x TAE
</LI></UL><UL><LI> adding 2 µl gel red
</LI></UL><P><B>2. Loading samples</B></P><UL><LI> 30 µl PCR product and 6x loading dye
</LI></UL><UL><LI> 2 µl DNA ladder gene ruler
</LI></UL><P><B>3. Run</B></P><UL><LI> 100 V
</LI></UL><UL><LI> time: 1 h
</LI></UL><P><B>Results:</B></P><P><A href="/File:UP_Error_Prone_PCR_2010-06-28.png" title="File:UP Error Prone PCR 2010-06-28.png">300px</A></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> 1 </TD><TD> marker </TD><TD> 2 </TD></TR><TR><TD> 2 </TD><TD> PCR sample 1</TD><TD> 50 </TD></TR><TR><TD> 3 </TD><TD> PCR sample 2 </TD><TD> 50 </TD></TR><TR><TD> 4 </TD><TD> PCR sample 3</TD><TD> 50 </TD></TR></TBODY></TABLE><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Gel_extraction_of_mutated_mdnA_2">Gel extraction of mutated mdnA </SPAN></H3><P><B>Investigators:</B> Katharina </P><P><B>Aim:</B> Extraction of mdnA, which was amplified and mutated (using error prone PCR)</P><P><B>Time:</B> 2011-06-26,13:00-14:00</P><P><B>Materials:</B></P><UL><LI> NucleoSpin Extract II (Macherey-Nagel) extraction Kit
</LI></UL><UL><LI> PCR products 1 and 2 containing mutated mdnA
</LI></UL><P><B>Method:</B></P><UL><LI> performing gel extraction based on manufacturer's protocol
</LI></UL><UL><LI> elute in 30 µl buffer NE
</LI></UL><P><B>Results:</B></P><UL><LI> purified PCR products of mutated mdnA (1 and 2)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> restriction enzyme digestion with purified mdnA using AatII/ NarI
</LI></UL><UL><LI> Verification by agarose gel electrophoresis
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_and_agarose_gel_electrophoresis_of_mutated_mdnA_2">Restriction enzyme digestion and agarose gel electrophoresis of mutated mdnA</SPAN></H3><P><B>Investigators:</B> Katharina</P><P><B>Time:</B> 2011-06-28,14:00-17:00 </P><P><B>Aim:</B> Restriction enzyme digestion of random mutated mdnA and verification using gel electrophoresis</P><P><B>Materials:</B></P><UL><LI> Restriction enzymes: AatII, NarI
</LI></UL><UL><LI> NEB Buffer 4
</LI></UL><UL><LI> PCR products of random mutated mdnA(1 and 2), purified from gel
</LI></UL><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> DNA Ladder Gene Ruler, 100bp plus, DNA Ladder (Fermentas)</LI></UL><UL><LI> 6x Loading Dye (Fermentas)
</LI></UL><P><B>Method:</B></P><P><B>1. Restriction enzyme digestion</B></P><UL><LI> 5 µl of purified PCR product were left undigested</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TD> components </TD><TD align="right">volume of PCR products /µl|
</TD></TR><TR><TD> DNA </TD><TD align="right">25
</TD></TR><TR><TD> NEB Buffer 4(10x)</TD><TD align="right"> 4
</TD></TR><TR><TD>Enzyme AatII</TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme NarI</TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 9
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 40
</TD></TR></TBODY></TABLE><UL><LI> digestion for 1 h
</LI></UL><UL><LI> after digestion 17.5 µl of digested PCR product were taken for heat inactivation of the restriction enzyme (60°C for 20 min)
</LI></UL><P><B>2. Production of 1 % agarose gels</B></P><UL><LI> m<SMALL>Agarose</SMALL> = 0.5 g in 50 ml 1x TAE
</LI></UL><UL><LI> adding 2 µl gel red
</LI></UL><P><B>3. Loading samples</B></P><P>a. restriction enzyme digestion products of random mutated mdnA
</P><UL><LI> 35 µl PCR product and 5 µl 6x loading dye
</LI></UL><UL><LI> 2 µl DNA ladder gene ruler, 100bp plus (Fermentas)
</LI></UL><P>4. Run
</P><UL><LI> 120 V
</LI></UL><UL><LI> time: 45 min
</LI></UL><P><B>Results:</B></P><P><A href="/File:UP_Digestion_Error_Prone_2010-28-06.png" title="File:UP Digestion Error Prone 2010-28-06.png">File:UP Digestion Error Prone 2010-28-06.png</A></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> 1 </TD><TD> marker </TD><TD> 2 </TD></TR><TR><TD> 2 </TD><TD> undigested mutated mdnA sample 1</TD><TD> 5 </TD></TR><TR><TD> 3 </TD><TD> digested mutated mdnA sample 1 </TD><TD> 17.5 </TD></TR><TR><TD> 4 </TD><TD> digested and heat inactivated enzyme after digestion of mutaded mdnA sample 1</TD><TD> 17.5 </TD></TR><TR><TD> 5 </TD><TD> undigested mutated mdnA sample 2</TD><TD> 5 </TD></TR><TR><TD> 6 </TD><TD> digested mutated mdnA sample 2 </TD><TD> 17.5 </TD></TR><TR><TD> 7 </TD><TD> digested and heat inactivated enzyme after digestion of mutaded mdnA sample 2</TD><TD> 17.5 </TD></TR></TBODY></TABLE><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Error_prone_PCR_.28varied_MnCl2_concentration.29_of_mdnA">Error prone PCR (varied MnCl<SMALL>2</SMALL> concentration) of mdnA</SPAN></H3><P><B>Investigators:</B> Steffi </P><P><B>Time:</B> 2011-06-28,15:00-15:50 </P><P><B>Aim:</B> Random mutation of mdnA gene based on increased MnCl<SMALL>2</SMALL> concentration </P><P><B>Materials:</B></P><UL><LI> Genaxxon Bioscience Taq Polymerase
</LI></UL><UL><LI> 10x Genaxxon Bioscience Taq Polymerase buffer containing 15 mM MgCl<SMALL>2</SMALL></LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><UL><LI> forward primer: sf_mdnA_1 (10 µM)
</LI></UL><UL><LI> reverse primer: r_mdnA_1 (10 µM)
</LI></UL><UL><LI> template DNA pARW089 (857,3 ng/ µl measured by Nanodrop a second time)
</LI></UL><UL><LI> Genaxxon Bioscience MgCl<SMALL>2</SMALL> stock solution (50 mM)
</LI></UL><UL><LI> MnCl<SMALL>2</SMALL></LI></UL><UL><LI> Eppendorf Mastercycler personal, program IGMDNA1
</LI></UL><P><B><I>Error prone PCR of mdnA</I></B></P><UL><LI> increasing MnCl<SMALL>2</SMALL> concentration
</LI></UL><P><B>1. Mix for error prone PCR with increasing MnCl<SMALL>2</SMALL> concentration</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component </TH><TH> Stock concentration </TH><TH> Volume in µl </TH><TH> Final concentration
</TH></TR><TR><TD> Genaxxon Bioscience Taq Polymerase buffer containing MgCl<SMALL>2</SMALL>) </TD><TD>15 mM (MgCl<SMALL>2</SMALL>) </TD><TD> 5 </TD><TD> 1.5 mM MgCL<SMALL>2</SMALL></TD></TR><TR><TD> MgCl<SMALL>2</SMALL></TD><TD> 50 mM </TD><TD>5.5 </TD><TD> 5.5 mM
</TD></TR><TR><TD> dNTPs </TD><TD> 10 mM </TD><TD> 2.0 </TD><TD> 0.4 mM
</TD></TR><TR><TD> Primer (each) </TD><TD> 10 µM </TD><TD> 2.5 </TD><TD> 0.5 µM
</TD></TR><TR><TD> DNA (diluted 1:100)</TD><TD> 8.57 ng/ µl </TD><TD> 5.58 </TD><TD> 0.7 ng
</TD></TR><TR><TD> Polymerase </TD><TD> 1 </TD><TD> 5 U
</TD></TR></TBODY></TABLE><P><B>2. Complete pipet scheme</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component in µl </TH><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4 </TH><TH> 5 </TH><TH> 6
</TH></TR><TR><TD> Template DNA </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8
</TD></TR><TR><TD> dNTPs </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> 10x Genaxxon polymerase buffer (with MgCl<SMALL>2</SMALL>) </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5
</TD></TR><TR><TD> MgCl<SMALL>2</SMALL> (50 mM) </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 5.5
</TD></TR><TR><TD> MnCl<SMALL>2</SMALL> (5 mM) </TD><TD> 0 </TD><TD> 0.5 </TD><TD> 1 </TD><TD> 1.25 </TD><TD> 1.5 </TD><TD> 2
</TD></TR><TR><TD> Forward primer (sf_mdnA_1) </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5
</TD></TR><TR><TD> Reverse primer (r_mdnA_1) </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5
</TD></TR><TR><TD> H<SMALL>2</SMALL>O </TD><TD> 23.5 </TD><TD> 23.0 </TD><TD> 22.5 </TD><TD> 22.25 </TD><TD> 22.0 </TD><TD> 21.5
</TD></TR></TBODY></TABLE><P><B>3. Temperature profile</B></P><UL><LI> see 12th labday
</LI></UL><UL><LI> Eppendorf Mastercycler personal, program IGMDNA1
</LI></UL><P><B>Results:</B></P><UL><LI> 6 PCR products containing mutated mdnA
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Verification of these PCR products by agarose gel electrophoresis
</LI></UL><UL><LI> Purification and cloning of positive samples
</LI></UL><UL><LI> Sequencing
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="26th_Labday_2011-06-29">26th Labday 2011-06-29</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="5l_LB_Media_.28liquid.29"> 5l LB Media (liquid) </SPAN></H3><P><B>Investigators:</B> Sebastian, Niels </P><P><B>Aim:</B> Generating media for further research</P><P><B>Materials:</B></P><UL><LI> Yeast (5 g/l) </LI></UL><UL><LI> Tryptone (10 g/l) </LI></UL><UL><LI> NaCl (10 g/l)</LI></UL><UL><LI>autoclaved
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="oligo_hybridization_for_TEV_and_14_3C_protease_cleavage_sites">oligo hybridization for TEV and 14_3C protease cleavage sites</SPAN></H3><P><B>Investigators:</B> Sebastian, Paul</P><P><B>Materials:</B></P><P>2 reaction batches: of_NheI_CS-TEV_XhoI (Cleavage site for TEV) and of_NheI_143C-cleavage_XhoI (Cleavage site for 14_3C)
</P><UL><LI> 1 µl Oligo 1(of_NheI_CS-TEV_XhoI or of_NheI_143C-cleavage_XhoI forward)
</LI></UL><UL><LI> 1 µl Oligo 2(of_NheI_CS-TEV_XhoI or of_NheI_143C-cleavage_XhoI reverse)
</LI></UL><UL><LI> 4 µl 100mM TrisHCl pH 8
</LI></UL><UL><LI> 8 µl 5 mM MgCl2
</LI></UL><UL><LI> 26 µl H2O
</LI></UL><UL><LI> Eppendorf Mastercycler personal, program ORIGAMI:
</LI></UL><P>1 96°C 7 min
</P><P>2 96°C 1 min
</P><P>-1°C R=0,3°/s
</P><P>goto 2 rep 70
</P><P>hold 4°C
</P><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_of_pJC354_WZA2-vector">Restriction of pJC354_WZA2-vector</SPAN></H3><P><B>Investigators:</B> Sebastian, Paul, Sascha</P><P><B>Materials:</B></P><UL><LI> XhoI (NEB), NheI (NEB)
</LI></UL><UL><LI> 100x BSA (NEB), 10x buffer 4 (NEB)
</LI></UL><UL><LI> pJC354_WZA2-vector
</LI></UL><P><B>Protocol:</B></P><UL><LI> 1µl XhoI
</LI></UL><UL><LI> 1µl NheI
</LI></UL><UL><LI> 5µl 10x buffer 4
</LI></UL><UL><LI> 0,5 µl vector [~500ng]
</LI></UL><UL><LI> 0,5 µl 100x BSA
</LI></UL><UL><LI> 42 µl pure H2O
</LI></UL><UL><LI> Reaction for 2h at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> preparative agarose electrophoresis
</LI></UL><UL><LI> Ligation of digested vector with annealed oligos (over night, 18°C)
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="preparative_agarose_electrophoresis">preparative agarose electrophoresis</SPAN></H3><P><B>Investigators:</B> Sebastian, Paul, Sascha</P><P><B>Method:</B></P><P>Samples digested with NheI and XhoI</P><P>vector: digested pJC354_WZA2 source: Janina; prep-type: midi-prep ; prep-date:24.06.11 ; clone-no: 6</P><UL><LI>control undigested pJC354_WZA2 source: Janina; prep-type: midi-prep ; prep-date:24.06.11 ; clone-no: 6</LI></UL><P>Electrophoresis</P><P>sample preparation: 50 µl pJC354_WZA2 (digested) + 10 µl 6x loading dye solution, 1 µl pJC354_WZA2 (undigested) + 1 µl 6x loading dye + 4 µl dest. water</P><P>0,5 g Agarose, 50 ml TAE (1%), 2 µl GELRED, at 115 Volt,running time:_?___ minutes</P><P>marker: GeneRuler ladder mix (Fermentas)</P><TABLE border="1" class="wikitable"><TBODY><TR><TH>lane</TH><TH>Sample </TH><TH>Sample/µl </TH><TH>Expected size (bp) </TH><TH>approx. size </TH></TR><TR><TD style="vertical-align: top;">1</TD><TD style="vertical-align: top;">marker</TD><TD style="vertical-align: top;">6 µl</TD></TR><TR><TD style="vertical-align: top;">2</TD><TD>pJC354_WZA2 (digested)</TD><TD>~40 µl</TD><TD>130, 4700</TD><TD>130, 8000, 2400</TD></TR><TR><TD style="vertical-align: top;">3</TD><TD>pJC354_WZA2 (digested)</TD><TD>~20 µl </TD><TD>130, 4700</TD><TD>130, 8000, 2400</TD></TR><TR><TD style="vertical-align: top;">5</TD><TD>pJC354_WZA2 (undigested)</TD><TD>6 µl </TD><TD>4000 for supercoiled, 4700 for not supercoiled plasmid</TD><TD>2400,2400-3000, 7500 </TD></TR></TBODY></TABLE><P>            </P><P><A href="/File:UP_pJC354_WZA2_digest_XhoI_NheI.png" title="File:UP pJC354 WZA2 digest XhoI NheI.png">300px</A></P><P>Gel Extraction</P><P>DNA excision: lane 2 and 3: 8000 bp band;</P><P>DNA purification: with QIAquick Gel Extraction Kit according to Qiagen manual (incl. isopropanol step)(done by Tobias)</P><P><B>Result:</B> size estimation by eye;</P><P>lanes 2+3: plasmid was digested, small band has the expected size, two upper bands with unexpected size;</P><P>lanes 5: plasmid was undigested; 7500 bp unexpected; 3000 bp band might be undigested supercoiled plasmid</P><P><B>Conclusions:</B></P><UL><LI>exact vector fragment unknown, therefore 8000 bp band was</LI><LI>because of unexpected sizes marker should be compared with

another marker, plasmids should be digested to yield several fragments

for size control, maybe heat inactivation might help to resolve higher bands</LI></UL><P><B>Output:</B> one plasmid fragments with 8000 bp in 1,5 ml eppis,
</P><P>stored at -20 °C, box with fragments</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR_for_amplification_of_mdnA_with_C-terminal_myc-tag">PCR for amplification of mdnA with C-terminal myc-tag</SPAN></H3><P><B>Investigators:</B> Jessica </P><P><B>Aim:</B> amplificate mdnA of pARW089 and add C-terminal myc-tag with primers pf_mdnA_mycC_XbaI and pr_mdnA_mycC_PstI_NotI_SpeI</P><P><B>Time:</B> 2011-06-29, 14:00-18:00</P><P><B>Materials:</B></P><UL><LI> NEB OneTaq DNA Polymerase
</LI></UL><UL><LI> 5x Buffer NEB OneTaq
</LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><UL><LI> forward primer: pf_mdnA_mycC_XbaI (10 µM)
</LI></UL><UL><LI> reverse primer: pr_mdnA_mycC_PstI_NotI_SpeI (10 µM)
</LI></UL><UL><LI> template DNA pARW089
</LI></UL><UL><LI> Eppendorf Mastercycler gradient, program SABRINA
</LI></UL><P><B>50 µl reaction</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component </TH><TH> Stock concentration </TH><TH> Volume in µl </TH><TH> Final concentration
</TH></TR><TR><TD> 5x Buffer NEB OneTaq </TD><TD> 5x </TD><TD> 10 </TD><TD> 1x
</TD></TR><TR><TD> dNTPs </TD><TD> 10 mM </TD><TD> 1 </TD><TD> 0.2 mM
</TD></TR><TR><TD> Primer (each) </TD><TD> 10 µM </TD><TD> 1 </TD><TD> 0.2 µM
</TD></TR><TR><TD> DNA (diluted 1:100)</TD><TD> 8.57 ng/ µl </TD><TD> 5 </TD><TD> ~0.7 ng
</TD></TR><TR><TD> NEB OneTaq DNA Polymerase </TD><TD> 0.25 </TD><TD> 1.5 U
</TD></TR><TR><TD>H<SMALL>2</SMALL>O </TD><TD> 31.75 </TD></TR></TBODY></TABLE><P><B>Temperature profil</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Temperature in °C </TH><TH> Time in s </TH></TR><TR><TD> 94 </TD><TD> Hold
</TD></TR><TR><TD> 94 </TD><TD> 60 </TD><TD> Initial denaturation
</TD></TR><TR><TD> 94 </TD><TD> 20 </TD><TD> Denaturation
</TD></TR><TR><TD> 60 </TD><TD> 20 </TD><TD> Annealing
</TD></TR><TR><TD> 72 </TD><TD> 20 </TD><TD> Elongation
</TD></TR><TR><TD> --&gt; 15 cycles
</TD></TR><TR><TD> 94 </TD><TD> 60 </TD><TD> Denaturation
</TD></TR><TR><TD> 70 </TD><TD> 40 </TD><TD> Annealing
</TD></TR><TR><TD> 72 </TD><TD> 20 </TD><TD> Elongation
</TD></TR><TR><TD> --&gt; 15 cycles
</TD></TR><TR><TD> 70 </TD><TD> 300 </TD><TD> Final extension
</TD></TR><TR><TD>6 </TD><TD> Hold
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B> do gel electrophoresis
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="PCR_of_mdnA_for_phage_display_.28strategy_1.29_repeated_with_new_ordered_forward_primer">PCR of mdnA for phage display (strategy 1) repeated with new ordered forward primer</SPAN></H3><P><B>Investigators:</B> Sandrina </P><P><B>Time:</B> 2011-06-29,14:00-18:00 </P><P><B>Aim:</B> amplification of mdnA with SfiI restriction sites (vector pARW089) to clone it into pAk100 bla KDIR </P><P><B>Materials/Methods:</B></P><P>see 2011-06-10
</P><P>changes:
</P><UL><LI> elongation time 40 s
</LI></UL><UL><LI> template 5 µl
</LI></UL><UL><LI> program: Sabrina, eppendorf master cycler gradient
</LI></UL><P><B>further tasks:</B></P><P>gel electrophoresis with PCR product
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Error_prone_PCR_.28varied_MnCl2_concentration.29_of_mdnA_2">Error prone PCR (varied MnCl<SMALL>2</SMALL> concentration) of mdnA</SPAN></H3><P><B>Investigators:</B> Katharina </P><P><B>Time:</B> 2011-06-29,12:00-13:00 </P><P><B>Aim:</B> Random mutation of mdnA gene based on increased MnCl<SMALL>2</SMALL> concentration </P><P><B>Materials:</B></P><UL><LI> Genaxxon Bioscience Taq Polymerase
</LI></UL><UL><LI> 10x Genaxxon Bioscience Taq Polymerase buffer containing 15 mM MgCl<SMALL>2</SMALL></LI></UL><UL><LI> dNTPs (10 mM)
</LI></UL><UL><LI> forward primer: sf_mdnA_1 (10 µM)
</LI></UL><UL><LI> reverse primer: r_mdnA_1 (10 µM)
</LI></UL><UL><LI> template DNA pARW089 (857,3 ng/ µl measured by Nanodrop a second time)
</LI></UL><UL><LI> Genaxxon Bioscience MgCl<SMALL>2</SMALL> stock solution (50 mM)
</LI></UL><UL><LI> MnCl<SMALL>2</SMALL></LI></UL><UL><LI> Eppendorf Mastercycler personal, program IGMDNA1
</LI></UL><P><B><I>Error prone PCR of mdnA</I></B></P><UL><LI> increasing MnCl<SMALL>2</SMALL> concentration
</LI></UL><P><B>1. Mix for error prone PCR with increasing MnCl<SMALL>2</SMALL> concentration</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component </TH><TH> Stock concentration </TH><TH> Volume in µl </TH><TH> Final concentration
</TH></TR><TR><TD> Genaxxon Bioscience Taq Polymerase buffer containing MgCl<SMALL>2</SMALL>) </TD><TD>15 mM (MgCl<SMALL>2</SMALL>) </TD><TD> 5 </TD><TD> 1.5 mM MgCL<SMALL>2</SMALL></TD></TR><TR><TD> MgCl<SMALL>2</SMALL></TD><TD> 50 mM </TD><TD>5.5 </TD><TD> 5.5 mM
</TD></TR><TR><TD> dNTPs </TD><TD> 10 mM </TD><TD> 2.0 </TD><TD> 0.4 mM
</TD></TR><TR><TD> Primer (each) </TD><TD> 10 µM </TD><TD> 2.5 </TD><TD> 0.5 µM
</TD></TR><TR><TD> DNA (diluted 1:100)</TD><TD> 8.57 ng/ µl </TD><TD> 5.58 </TD><TD> 0.7 ng
</TD></TR><TR><TD> Polymerase </TD><TD> 1 </TD><TD> 5 U
</TD></TR></TBODY></TABLE><P><B>2. Complete pipet scheme</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> Component in µl </TH><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4 </TH><TH> 5 </TH><TH> 6
</TH></TR><TR><TD> Template DNA </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8 </TD><TD> 8
</TD></TR><TR><TD> dNTPs </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> 10x Genaxxon polymerase buffer (with MgCl<SMALL>2</SMALL>) </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5 </TD><TD> 5
</TD></TR><TR><TD> MgCl<SMALL>2</SMALL> (50 mM) </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 5.5 </TD><TD> 5.5
</TD></TR><TR><TD> MnCl<SMALL>2</SMALL> (5 mM) </TD><TD> 0 </TD><TD> 0.5 </TD><TD> 1 </TD><TD> 1.25 </TD><TD> 1.5 </TD><TD> 2
</TD></TR><TR><TD> Forward primer (sf_mdnA_1) </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5
</TD></TR><TR><TD> Reverse primer (r_mdnA_1) </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5 </TD><TD> 2.5
</TD></TR><TR><TD> H<SMALL>2</SMALL>O </TD><TD> 23.5 </TD><TD> 23.0 </TD><TD> 22.5 </TD><TD> 22.25 </TD><TD> 22.0 </TD><TD> 21.5
</TD></TR></TBODY></TABLE><P><B>3. Temperature profile</B></P><UL><LI> see 12th labday
</LI></UL><UL><LI> Eppendorf Mastercycler personal, program IGMDNA1
</LI></UL><P><B>Results:</B></P><UL><LI> 12 PCR products (each sample was prepared twice) containing mutated mdnA
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Verification of these PCR products by agarose gel electrophoresis
</LI></UL><UL><LI> Purification, digestion and cloning of positive samples
</LI></UL><UL><LI> Sequencing
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="27th_Labday_2011-06-30">27th Labday 2011-06-30</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_of_myc-tagged_mdnA_and_mdnA_for_phage_display">Agarose gel electrophoresis of myc-tagged mdnA and mdnA for phage display </SPAN></H3><P><B>Investigators:</B> Nadja, Jessica, Sandrina </P><P><B>Aim:</B> Verification of myc-tagged mdnA and mdnA for phage display (strategy 1)</P><P><B>Time:</B> 2011-06-30,10:00-12:00</P><P><B>Materials:</B></P><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> PCR products: myc-tagged mdnA, mdnA for phage display (strategy 1) from 2011-06-29
</LI></UL><UL><LI> DNA Ladder Gene Ruler, 100bp plus (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas) </LI></UL><P><B>Method:</B></P><P><B>1. Production of a 1 % agarose gel</B></P><UL><LI> m<SMALL>Agarose</SMALL> = 0.5 g in 50 ml 1x TAE
</LI></UL><UL><LI> adding 2 µl gel red </LI></UL><P><B>2. Loading samples</B></P><UL><LI> 5 µl PCR product and 6x loading dye
</LI></UL><UL><LI> 2 µl DNA ladder gene ruler (1:10) </LI></UL><P><B>3. Run</B></P><UL><LI> 115 V
</LI></UL><UL><LI> time: 1 h </LI></UL><P><B>Results:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> M </TD><TD> marker </TD><TD> 2 </TD></TR><TR><TD> 1 </TD><TD> myc-tagged mdnA </TD><TD> 6 </TD><TD> 233
</TD></TR><TR><TD> 2 </TD><TD> mdnA with SfiI </TD><TD> 6 </TD></TR></TBODY></TABLE><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_of_error_prone_PCR_of_mdnA_from_2011-06-29_and_gel_extraction">Agarose gel electrophoresis of error prone PCR of mdnA from 2011-06-29 and gel extraction </SPAN></H3><P><B>Investigators:</B> Nadja, Jessica</P><P><B>Aim:</B> Verification of random mutated mdnA</P><P><B>Time:</B> 2011-06-30, 12:00-17:00</P><P><B>Materials:</B></P><UL><LI> Agarose broad range (Roth)
</LI></UL><UL><LI> 1x TAE buffer
</LI></UL><UL><LI> Gel Red
</LI></UL><UL><LI> 12 PCR products: sample 1a to 6b from 2011-06-29
</LI></UL><UL><LI> DNA Ladder Gene Ruler, 100bp plus (Fermentas)
</LI></UL><UL><LI> 6x Loading Dye (Fermentas) </LI></UL><P><B>Method:</B></P><P><B>1. Production of a 1 % agarose gel</B></P><UL><LI> m<SMALL>Agarose</SMALL> = 0.5 g in 50 ml 1x TAE
</LI></UL><UL><LI> adding 2 µl gel red <B>2. Loading samples</B></LI></UL><UL><LI> 30 µl PCR product and 6x loading dye
</LI></UL><UL><LI> 2 µl DNA ladder gene ruler (1:10) </LI></UL><P><B>3. Run</B></P><UL><LI> 115 V
</LI></UL><UL><LI> time: 1 h </LI></UL><P><B>Results:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TH colspan="3"> gel 1 </TH><TH colspan="3"> gel 2
</TH></TR><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> marker </TD><TD> 2 </TD><TD> marker </TD><TD> 2 </TD></TR><TR><TD> 1 </TD><TD> PCR sample 1a </TD><TD> 36 </TD><TD> 276 </TD><TD> PCR sample 4a </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 2 </TD><TD> PCR sample 1b </TD><TD> 36 </TD><TD> 276 </TD><TD> PCR sample 4b </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 3 </TD><TD> PCR sample 2a </TD><TD> 36 </TD><TD> 276 </TD><TD> PCR sample 5a </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 4 </TD><TD> PCR sample 2b </TD><TD> 36 </TD><TD> 276 </TD><TD> PCR sample 5b </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 5 </TD><TD> PCR sample 3a </TD><TD> 36 </TD><TD> 276 </TD><TD> PCR sample 6a </TD><TD> 36 </TD><TD> 276
</TD></TR><TR><TD> 6 </TD><TD> PCR sample 3b </TD><TD> 36 </TD><TD> 276 </TD><TD> PCR sample 6b </TD><TD> 36 </TD><TD> 276
</TD></TR></TBODY></TABLE><P><B>Gel Extraction</B></P><UL><LI>DNA excision: PCR products 1a to 6b
</LI></UL><UL><LI>DNA purification: with Qiagen Gel Extraction Kit according to Qiagen manual (incl. isopropanol step, elute in 30 µl H<SMALL>2</SMALL>O)</LI></UL><P><B>Results:</B></P><UL><LI>DNA concentrations measured with Nanodrop were lower than 10 ng/µl
</LI></UL><P><B>Further tasks:</B></P><UL><LI>repeat with remaining PCR samples?
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_PCR_of_mdnA_for_phage_display_.28strategy_1.29_repeated_with_new_ordered_forward_primer">Repeated PCR of mdnA for phage display (strategy 1) repeated with new ordered forward primer</SPAN></H3><P><B>Investigators:</B> Sandrina, Sabine </P><P><B>Time:</B> 2011-06-30,15:00-18:00 </P><P><B>Aim:</B> amplification of mdnA with SfiI restriction sites (vector pARW089) to clone it into pAk100 bla KDIR </P><P><B>Materials/Methods:</B></P><P>see 2011-06-10
</P><P>changes:
</P><UL><LI> elongation time 40 s
</LI></UL><UL><LI> template 5 µl
</LI></UL><UL><LI> program: Sabrina, eppendorf master cycler gradient
</LI></UL><P><B>further tasks:</B></P><P>gel electrophoresis with PCR product
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/June">http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/June</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Labjournal/June" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Labjournal/June" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/June&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/June&amp;oldid=165783" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>