<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Project_Details_Selection"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Project/Details_Selection">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Project/Details_Selection&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Details_Selection&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Details_Selection&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Project/Details_Selection" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Project/Details Selection</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team">Team</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Members">Members</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Project</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Summary</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Highlights">Highlights</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Microviridin">     Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Phage_Display">     Phage Display</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#In_Vivo_Selection">     In Vivo Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Modeling">     Modeling</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Microviridin">Details Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">Details Phage</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">Details Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Modeling">Details Modeling</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">BioBricks</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/BioBricks">Details BioBricks</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Methods">Methods</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Meetings">Meetings</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics">Safety &amp; Ethics</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Assesment">Safety Assessment</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Questions">Safety Questions</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Ethics_Seminar">Ethics Seminar</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Survey">Survey</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Potsdam_Bioware_at_the_German_parliament">Parliament Visit</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Statement_of_the_German_government">     Statement</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Outreach">Outreach</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software">Software</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#BioLog_App">BioLog App</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#Download">Download</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Sponsors">Sponsors</A></DT></DL></LI></UL></DIV><DIV class="pageContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#In_Vivo_Selection"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">In Vivo Selection</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Site-directed_mutagenesis_of_Tobacco_Etch_Virus_and_14_3C_proteases_in_iGEM_standard"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Site-directed mutagenesis of Tobacco Etch Virus and 14_3C proteases in iGEM standard</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Construction_of_two_component_for_the_in_vivo_selection_assay"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Construction of two component for the in vivo selection assay</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Introducing_the_specific_protease_cleavage_site"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Introducing the specific protease cleavage site</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Adding_the_arabinose_inducible_induction_system_to_control_the_proteases.2C_TEV_and_14_3C"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Adding the arabinose inducible induction system to control the proteases, TEV and 14_3C</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-7"><A href="#Investigation_of_the_in_vivo_selection_assay"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Investigation of the in vivo selection assay</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Testing_the_export_efficiency_of_the_protease_activity_detector"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Testing the export efficiency of the protease activity detector</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Testing_the_.22Kill_Switch.22_by_activating_the_protease"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Testing the &quot;Kill Switch&quot; by activating the protease</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Transferring_and_testing_the_library"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Transferring and testing the library</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Background"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Twin_Arginine_Translocon_.28TAT.29"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Twin Arginine Translocon (TAT)</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Tobacco_Etch_Virus_.28TEV.29_protease"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">Tobacco Etch Virus (TEV) protease</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#14_3C-Protease"><SPAN class="tocnumber">1.6.3</SPAN><SPAN class="toctext">14_3C-Protease</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="In_Vivo_Selection"><I>In Vivo</I> Selection</SPAN></H2><H3><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H3><P>In addition to the phage display we developed a novel selection system. The design aimed for a inexpensive and time-saving alternative in contrast to an <I>in vitro</I> screen of protease inhibition. The assay allows us to select effective inhibitors for any protease, among the billions of randomly generated mutants of the Microviridin. For this purpose we designed a plasmid containing two devices, first a protease activity detector and second a protease generator.  For the protease activity detector we modified the TAT-hitchhiker system developed by DeLisa <I>et al.</I> 
We used the BioBrick &lt;partinfo&gt;I757010&lt;/partinfo&gt; (β-lactamase) as well as &lt;partinfo&gt;K208005&lt;/partinfo&gt; (ssTorA) and fused them together via a linker peptide. For the protease generator we cloned the arabinose induction system in front of a protease in iGEM standard.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:475px;"><DIV class="thumbcaption"><B>Figure 1:</B> Schema of our constructed selection vector containing protease generator and protease activity detector</DIV></DIV></DIV></DIV><P>The system works in a combined manner of the two devices. In order to confer β-lactam antibiotic resistance to cells the β-lactamase has to be exported in the periplasm. This transport is mediated by the TorA export sequence via the Twin-Arginine Translocation (TAT) system (DeLisa, 2009). The linker peptide between the TorA export sequence and the β-lactamase displays the corresponding protease cleavage site. In addition the linker peptide is chosen as short as possible to imitate folding because the TAT pathway only allows transport of correctly folded substrates. The protease as well as the linker peptide are designed as exchangeable parts.</P><P>If the protease device is functional, it will cleave the linker peptide between TorA and the β-lactamase construct which leaves the cell without any antibiotic resistance. The construct was tested with increasing ampicillin concentrations. The number of surviving colonies was depended on the export rate of the β-lactamase into the periplasm. A high cleavage rate of the linker peptide leads to a reduced ampicillin resistance. With expressed protease a dramatically drop of the number of surviving colonies could be observed.
The survival assay was carried out with and without expressed protease. By increasing of the ampicillin concentration we could detect a cutoff ampicillin concentration. The colony forming units (CFU) were counted and compared. To detected whether our constructed library is able to block the protease we co-transformed the library into <I>E. coli</I> carrying our pUP_SG1 plasmid.</P><H3><SPAN class="mw-headline" id="Site-directed_mutagenesis_of_Tobacco_Etch_Virus_and_14_3C_proteases_in_iGEM_standard">Site-directed mutagenesis of Tobacco Etch Virus and 14_3C proteases in iGEM standard</SPAN></H3><P>We chose two model proteases, the Tobacco Etch Virus protease, further along called TEV, and the 14_3C protease from the human rhinovirus, also known as PreScission. Both proteases contained iGEM restriction sites, one in case of the TEV and three for 14 3C, respectively. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 2:</B> Complete sequence alignment of the 14_3C protease</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 3:</B> Complete sequence alignment of the 14_3C protease</DIV></DIV></DIV></DIV><P>
Thus, site-directed mutagenesis was applied to mutate each restriction site in E. coli. For every mutation a forward and a reverse primer had to be designed. The following pictures show the forward primers for the side directed mutagenesis, the reverse primers are the reverse complement sequence of the forward primers.</P><P>For the 14_3C protease:
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 4:</B> Reverse primer to add the iGEM suffix on the HRV 14_3C protease</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 5:</B> Forward primer to mutate the SpeI restriction site in the gene sequence of the HRV 14_3C protease</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 6:</B> Forward primer to mutate the first XbaI restriction site in the gene sequence of the HRV 14_3C protease</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 7:</B> Forward primer to mutate the second XbaI restriction site in the gene sequence of the HRV 14_3C protease</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 8:</B> Forward primer to add the iGEM prefix to the HRV 14_3C protease</DIV></DIV></DIV><P>For the TEV protease:
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 9:</B> Reverse primer to add the iGEM suffix to the TEV protease</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 10:</B> Forward primer to mutate the SpeI restriction site in the gene sequence of the TEV protease</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 11:</B> Forward primer to add the iGEM prefix to the TEV protease</DIV></DIV></DIV><P>We used assembly PCR to gain the final product. In case of 14_3C two assembly PCRs had to be done. The expected size of the fragments are shown in the gel picture. In the following picture all fragments contain the correct sizes.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 12:</B> Picture of the, via PCR amplified and mutated Fragment of both protease</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Construction_of_two_component_for_the_in_vivo_selection_assay">Construction of two component for the <I>in vivo</I> selection assay</SPAN></H3><P>The <I>in vivo</I> assay is based on two devices. On the one hand the protease activity detector device, on the other a protease generator device. The detector device is based on the enzyme β-lactamase, which is only active inside the periplasm and provides the resistance against lactam antibiotics like ampicillin and penicillin. For the protease generator device we needed a second induction system on our plasmid, which is quite uncommon for any plasmid. So several problems had to solved to set up an effective <I>in vivo</I> selection assay:
</P><UL><LI>a second, very tight induction system has to be found
</LI><LI>a way to control the export rate of β-lactamase into the periplasm, which can be reduced by the protease
</LI></UL><P>We solved these problems by putting the protease detector device under the control of the lac-operon. The protease device is induced by the arabinose system of the pBAD_iGEMexpress vector. The idea to control the survival of the cells via the amount of exported protein is inspired by the Hitchhiker assay of DeLisa <I>et al</I>. Using the TAT pathway for the export of β-lactamase into the periplasm allows us to introduce a protease specific cleavage site in front of the signal sequence regulating the export.
</P><H4><SPAN class="mw-headline" id="Introducing_the_specific_protease_cleavage_site">Introducing the specific protease cleavage site</SPAN></H4><P>To adapt the protease activity detector device to any protease we want to screen, we had to modularize it. Our detector device was set up on a vector for the Hitchhiker selection. The plasmid offers two unique restriction sites on the plasmid, after the signal sequence of TorA one restriction site for <I>Xho</I>I and a <I>Nhe</I>I restriction site in front of the β-lactamase. Both were used to introduce the specific protease cleavage site in between the TorA signal sequence and the β-lactamase. By designing two complementary oligonucleotides containing the specific protease cleavage site flanked by two short linker regions. After the digest of the vector with <I>Xho</I>I and <I>Nhe</I>I the hybridized oligonucleotides, containing the overhangs for <I>Xho</I>I and <I>Nhe</I>I can be ligated into the vector.
</P><H4><SPAN class="mw-headline" id="Adding_the_arabinose_inducible_induction_system_to_control_the_proteases.2C_TEV_and_14_3C">Adding the arabinose inducible induction system to control the proteases, TEV and 14_3C</SPAN></H4><P>We needed a time independent induction of the protease device. Therefore we cloned the  arabionse induction system (<I>araC</I>) in front of the protease device out of the pBAD_iGEMexpress vector. The fusion of gene construct and induction system was always performed via an <I>NgoM</I>IV restriction site. This restriction site was introduced with our amplification primers.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:506px;"><DIV class="thumbcaption"><B>Figure 13:</B> Reverse primer for the amplification of the arabinose inducible induction system</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:410px;"><DIV class="thumbcaption"><B>Figure 14:</B> Forward primer for the amplification of the arabinose inducible induction system</DIV></DIV></DIV></DIV><P>We used the <A href="http://tools.ietf.org/html/rfc23" class="external mw-magiclink-rfc">RFC 23</A> cloning strategy to get a fusion protein of our desired protease and the induction system, which allows us to induce the protease. Both genes were amplified with PCR using two special primers generating a <I>NgoM</I>IV restrictioin site at the end of the induction system and in front of the protease. The amplified protease were digested with <I>BamH</I>I and <I>NgoM</I>IV, the arabinose inducible induction system with <I>NgoM</I>IV and <I>Hind</I>III. The vector, which contains the ssTorA_CS-Protease_β-lactamase device was digested with <I>Hind</I>III and <I>BamH</I>I. A triple ligation yielded the final construct with both devices: the protease detector and the protease generator. The two pictures below show the final vector constructs , which where used for the survival screening.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 15:</B> Final vector construct containing arabinose inducible TEV protease fusion and the protease detector device</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 16:</B> Final vector construct containing arabinose inducible HRV 14_3C protease fusion and the protease detector device</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Investigation_of_the_in_vivo_selection_assay">Investigation of the <I>in vivo</I> selection assay</SPAN></H3><P>

Two major points have to be demonstrated: First, the export of β-lactamase over the TAT pathway has to be efficient enough to provide a certain resistance against ampicillin and second, the activated protease is able to intimately reduce the resistance against ampicillin by cleavage of the TorA signal sequence from the β-lactamase.</P><H4><SPAN class="mw-headline" id="Testing_the_export_efficiency_of_the_protease_activity_detector">Testing the export efficiency of the protease activity detector</SPAN></H4><P>To test the export rate of the β-lactamse of the protease activity detector device we grew an <I>E. coli</I> culture up to an OD<SUB>600</SUB> of 0.6. The cells were then diluted to an OD of 0.002 and 100 µL were plated in triplicates with increasing ampicillin concentrations. The graph below shows the amount of surviving cells in percentage. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:483px;"><DIV class="thumbcaption"><B>Figure 17:</B> Amount of survived cells. The construct was induced with 1 mM IPTG. The rate of surviving cells without IPTG and without ampicilin was set to 100%.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 18:</B> Survival test at different ampicillin concentrations: A) 1000 cfu plated on agar plates containing chloramphenicol; B) Reduced amount of <I>E.coli</I> colonies, 1000 cfu were plated on agar plates containing chloramphenicol, 1mM IPTG to induce ampicillin resistance via BBa_K627012 and 400µg/mL ampicillin; C) no culture growth of <I>E.coli</I> colonies, 1000 cfu were plated on agar plates containing chloramphenicol, 1mM IPTG to induce ampicillin resistance via BBa_K627012 and 600µg/mL ampicillin</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 19:</B> Survival test at different ampicillin concentrations:  A) 1000 cfu plated on agar plates containing chloramphenicol;  B) Influence of resistance induction with 1 mM IPTG, 1000 cfu where plated on agar plates containing chloramphenicol and 1mM IPTG; C) Reduced amount of <I>E.coli</I> colonies, 1000 cfu were plated on agar plates containing chloramphenicol, 1 mM IPTG to induce ampicillin resistance via BBa_K627013 and 50 µg/mL ampicillin; D) Reduced amount of <I>E.coli</I> colonies, 1000 cfu were plated on agar plates containing chloramphenicol, 1 mM IPTG to induce ampicillin resistance via BBa_K627013 and 100 µg/mL ampicillin; E) Reduced amount of <I>E.coli</I> colonies, 1000 cfu were plated on agar plates containing chloramphenicol, 1 mM IPTG to induce ampicillin resistance via BBa_K627013 and 200 µg/mL ampicillin; F) Reduced amount of <I>E.coli</I> colonies, 1000 cfu were plated on agar plates containing chloramphenicol, 1 mM IPTG to induce ampicillin resistance via BBa_K627013 and 400 µg/mL ampicillin; G) No culture growth of <I>E.coli</I> colonies, 1000 cfu were plated on agar plates containing chloramphenicol, 1 mM IPTG to induce ampicillin resistance via BBa_K627013 and 800 µg/mL ampicillin</DIV></DIV></DIV></DIV><P>This assay was performed at two temperatures. At 37°C, the optimal growth temperature and at 30°C for best protein expression conditions. All assays show the same results. The cells were able to grow at ampicilin concentrations up to 400 µg/ml. Cells transformed with the final construct containing the detection device and the protease generator device were able to survive up to 800 µg/ml ampicilin (incubated at 30°C over 2 days).
</P><P>This leads to the conclusion, that the export of β-lactamase into the periplasm is very effective. It is also important to mention, that there is no cell survival when ampicilin is added to the media without the induction of the protease activity detection device by IPTG.
</P><H4><SPAN class="mw-headline" id="Testing_the_.22Kill_Switch.22_by_activating_the_protease">Testing the &quot;Kill Switch&quot; by activating the protease</SPAN></H4><P>The second major point which needs to be demonstrated is that we are able to kill the <I>E.coli</I> cells by activating the protease. This is referred as the &quot;Kill Switch&quot;. In the final assay we want to have the ability to induce the protease activity detector device and the protease generate independently from each other. But first of all we have to check if our system is working and if we are able to kill the cells by activating the protease.
To proof the principle we used the natural TEV protease and make a co-transformation with the protease activity detector device containing the cleavage site for the TEV protease. Both were induced by IPTG and so we can't control the expression rate neither the time of the induction of the protease generator device.
After the incubation at 37°C we were able to show the desired proof of principle: when we activate the protease the resistance against ampicillin; based on the export rate of β-lactamase into the periplasm via the TAT pathway, was intimately reduced. The cells with activated TEV protease were able to survive up to concentration of 50 µg/mL ampicillin.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 20:</B> Percentage of surviving cells on different ampicillin concentration. Blue - cells transformed with the single protease activity detector device for TEV protease; Red - cell transformed with both, the protease activity detector device for TEV protease and the TEV protease itself</DIV></DIV></DIV></DIV><P>Based on the ampicillin resistance of the plasmid of HRV14_3C protease, we were unable to show the proof of principle for this protease.
The next step was to use the complete test vector which contains the protease activity detector device and the protease generator device. We introduced the arabinose inducible induction system fused to the protease flanked by the restriction sites of <I>BamH</I>I and <I>Hind</I>III into the vector of the modified Hitchhiker selection system.
With these final vectors we had to repeat the survival screening to show that the system is also working when there is a dual induction based on only one plasmid. The cells were grown to an OD<SUB>600</SUB> of 0.6 and diluted to an OD<SUB>600</SUB> of 0.002. 100 µL of this dilution were plated on agar plates with different ampicillin concentrations and induced and induced protease. The results are shown in figure 20.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 21:</B> Cell growth on different ampicillin concentration. Blue - cells without induction of the TEV protease; Red - cells with induced TEV protease</DIV></DIV></DIV></DIV><P>
The survival screening assay for TEV protease was performed 30 °C and all plates were incubated for 2 days. The <I>E.coli</I> XL1 blue cells were able to survive up to ampicilin concentration of 800 µg/mL, which is an increase of 100% of the original resistance capacity. When 2% arabinose is added to the media, the cells showed an intimately reduce surviving rate. The cells were able to survive at ampicilin concentrations up 25 µg/mL.
This shows, that we got a high dynamic range from 50 up to 800 µg/mL ampicilin to screen for inhibition of the protease.
</P><H3><SPAN class="mw-headline" id="Transferring_and_testing_the_library">Transferring and testing the library</SPAN></H3><P>After the successful test of the protease activity we transferred the generated <I>mdnA</I> library into our cells. The following pictures show the amazing results. Cell growth could be detected up to 400 µg Ampicillin/mL.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption"><B>Figure 22:</B> Cell growth on Ampicillin without induced protease</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption"><B>Figure 23:</B> Cell growth on Ampicillin with induced protease</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Background">Background</SPAN></H3><H4><SPAN class="mw-headline" id="Twin_Arginine_Translocon_.28TAT.29">Twin Arginine Translocon (TAT)</SPAN></H4><P>The TAT pathway is responsible for the transport of folded proteins across energy-transducing membranes and it is able of discriminating of unfolded proteins. This transport pathway is common for bacteria, archea and plants and translocates circa 6% of <I>E.coli</I> produced secreted proteins. The translocated proteins must have a signal peptide for targeting of the TAT-transporter, such proteins are e.g. hydrogenases, dehydrogenases and reductases. The signal sequence itself is composed of an N-terminal positive charged domain, a hydrophobic domain and a C-terminal domain.
The translocon is composed of three parts: TatA, TatB and TatC.  The transport-pore is proposed to be formed during substrate binding by these three parts.  The TatA, TatB and TatC proteins may form complexes of different sizes, which on the other hand form pores matching the size of the folded substrate. The transport of proteins through the TAT pathway depends on the proton motive force. Calculations by Adler &amp; Theg showed that the transport of one folded substrate molecule requires the release of approximately 7.9x10<SUP>4</SUP> protons, which equals 10.000 ATP molecules. The signal sequence is cleaved off after translocation.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 24:</B> Schematic representation of TorA-signal sequence. Figure adopted from: Philip A. Lee, Danielle Tullman-Ercekand George Georgiou Annu. Rev. Microbiol. 2006. 60:373–95</DIV></DIV></DIV></DIV><P>
The used signal sequence originates from the TorA protein (Trimethylamin-N-Oxid-Reductase). It is the main respiratory enzyme which reduces TMAO under anaerobic conditions in the periplasm, where it is transported by the TAT pathway.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 25:</B> Schematic representation of TAT-dependent pathway. Figure adopted from: Philip A. Lee, Danielle Tullman-Ercekand George Georgiou Annu. Rev. Microbiol. 2006. 60:373–95</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Tobacco_Etch_Virus_.28TEV.29_protease">Tobacco Etch Virus (TEV) protease</SPAN></H4><P>TEV protease is the common name for the 27 kDa catalytic domain of the Nuclear Inclusion a endopeptidase (NIa) encoded by the tobacco etch virus. TEV protease is a useful reagent for cleaving fusion proteins. It recognizes a linear epitope of the general form E-Xaa-Xaa-Y -Xaa-Q-(G/S), with cleavage occurring between Q and G or Q and S. In TEV protease the serine nucleophile of the conventional Ser-Asp-His triad is a cysteine instead. This probably explains why TEV protease is resistant to many commonly used protease inhibitors.</P><H4><SPAN class="mw-headline" id="14_3C-Protease">14_3C-Protease</SPAN></H4><P>The 14_3C protease originates from the human rhinovirus. Rhinoviruses are the most frequent reason for infections of the upper and lower respiratory tract, also known as the cold. Because the 3C protease of human rhinovirus is necessary for the cleavage of the polyprotein translated from viral RNA it may serve as a potential target for development of antiviral targets. The recombinant type 14_3C protease from human rhinovirus (HRV 3C) recognizes the same cleavage site as the native enzyme: LeuGluValLeuPheGln↓GlyPro. The small, 22-kDa size of the protease got its optimal activity at 4°C but is still very active at 37 °C. It is commonly used for an easy tag removal after the purification of recombinant proteins carrying his-tag. The 14_3C works with a catalytic triade, containing the amino acid residues Ser-Asp-His at its active site.</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><UL><LI>Cabrita, L. D., Gilis, D., Robertson, A. L., Dehouck, Y., Rooman, M. and Bottomley, S.P. (2007) Enhancing the stability and solubility of TEV protease using in silico design. Protein Sci. 16: 2360-2367</LI><LI>Genest O., Ilbert M., Méjean V., Iobbi-Nivol C.,(2005) TorD, an Essential Chaperone for TorA Molybdoenzyme Maturation at High Temperature. J. Biol. Chem. 280: 15644-15648.
</LI><LI>Kapust R. B., Tözsér J., Fox J. D., Anderson D. E., Cherry S., Copeland T. D., Waugh D. S. (2001) Tobacco etch virus protease: mechanism of autolysis and rational design of stable mutants with wild-type catalytic proficiency. Protein Eng. 12:993-1000.</LI><LI>Lee P. A., Tullman-Ercekand D., Georgiou G., (2006) The bacterial twin-arginine translocation pathway. Annu. Rev. Microbiol. 60:373–95
</LI><LI>Lucast, L. J., Batey, R. T., and Doudna, J. A. (2001). Large-scale purification of a stable form of recombinant tobacco etch virus protease. Biotechniques 30:544-550.</LI><LI>Shih S. R., Chen S. J., Hakimelahi G. H., Liu H. J., Tseng C. T., Shia K. S., (2004) Selective human enterovirus and rhinovirus inhibitors: An overview of capsid-binding and protease-inhibiting molecules. Med Res Rev 24(4):449-74.</LI><LI>Wanga QM, Chen SH., (2007) Human rhinovirus 3C protease as a potential target for the development of antiviral agents.  Curr Protein Pept Sci. 8(1):19-27. </LI><LI>Waraho D.,  DeLisa M. P., (2009). Versatile selection technology for intracellular protein-protein interactions mediated by a unique bacterial hitchhiker transport mechanism. PNAS 106(10):3692-7
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">http://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Project/Details_Selection" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Project/Details_Selection" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Details_Selection&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Details_Selection&amp;oldid=262036" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>