<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Labjournal_September_part_2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Labjournal/September_part_2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Labjournal/September_part_2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/September_part_2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/September_part_2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Labjournal/September_part_2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Labjournal/September part 2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team">Team</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Members">Members</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Project</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Summary</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Highlights">Highlights</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Microviridin">     Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Phage_Display">     Phage Display</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#In_Vivo_Selection">     In Vivo Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Modeling">     Modeling</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Microviridin">Details Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">Details Phage</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">Details Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Modeling">Details Modeling</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">BioBricks</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/BioBricks">Details BioBricks</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Methods">Methods</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Meetings">Meetings</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics">Safety &amp; Ethics</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Assesment">Safety Assessment</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Questions">Safety Questions</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Ethics_Seminar">Ethics Seminar</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Survey">Survey</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Potsdam_Bioware_at_the_German_parliament">Parliament Visit</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Statement_of_the_German_government">     Statement</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Outreach">Outreach</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software">Software</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#BioLog_App">BioLog App</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#Download">Download</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Sponsors">Sponsors</A></DT></DL></LI></UL></DIV><DIV class="pageContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#92th_Labday_2011-09-11"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">92th Labday 2011-09-11</SPAN></A></LI><LI class="toclevel-2"><A href="#miniprep_of_pSB1C3.2BmdnABCDE_with_and_without_T7_promotor"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">miniprep of pSB1C3+mdnABCDE with and without T7 promotor</SPAN></A></LI><LI class="toclevel-2"><A href="#miniprep_of_mdnA_foc2_part_2"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">miniprep of mdnA foc2 part 2</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_mdnABCDE_with_Long_PCR_Enzyme_mix_from_Fermentas"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">PCR mdnABCDE with Long PCR Enzyme mix from Fermentas</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_of_mdnA_with_C-terminal_myc_tag_.28mycC.29"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">PCR of mdnA with C-terminal myc tag (mycC)</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_mycN_and_pSB1C3"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Restriction enzyme digestion of mycN and pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_Purification_of_mdnA.2BmycC.2C_Lib_2_for_Phage_Display_and_digested_mdnA.2BmycN"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">PCR Purification of mdnA+mycC, Lib 2 for Phage Display and digested mdnA+mycN</SPAN></A></LI><LI class="toclevel-2"><A href="#Gel_electrophoresis_of_pARWIII_and_pSB1C3"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Gel electrophoresis of pARWIII and pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_enzyme_digestion_of_pSB1C3_for_control"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Restriction enzyme digestion of pSB1C3 for control</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_pARWIII_.2B_Lib-2_for_Phage_Display_and_pSB1C3_.2B_mdnAmycN"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Ligation of pARWIII + Lib-2 for Phage Display and pSB1C3 + mdnAmycN</SPAN></A></LI><LI class="toclevel-2"><A href="#Testing_of_expression_backbones"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Testing of expression backbones</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparing_new_overnight_cultures_of_expression_backbones"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">Preparing new overnight cultures of expression backbones</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_pARWIII_.2B_Lib-2_for_Phage_Display"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">Transformation of pARWIII + Lib-2 for Phage Display</SPAN></A></LI><LI class="toclevel-2"><A href="#Colony_PCR_of_E._coli_XL1_blue_transformed_with_ligation_products_of_mutated_TEV_protease.2C_mutated_14_3C_protease.2C_AraC_form_pBAD_iGEM_express.2C_pJC354_ssTorA_XhoI_CS-14_3C_NheI_blaFL_and_pJC354_ssTorA_XhoI_CS-TEV_NheI_blaFL_and_analytical_AG_.281.5.25.29"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">Colony PCR of E. coli XL1 blue transformed with ligation products of mutated TEV protease, mutated 14_3C protease, AraC form pBAD_iGEM_express, pJC354_ssTorA_XhoI_CS-14_3C_NheI_blaFL and pJC354_ssTorA_XhoI_CS-TEV_NheI_blaFL and analytical AG (1.5%)</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#93th_Labday_2011-09-12"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">93th Labday 2011-09-12</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation_of_samples_for_HPLC_analysis_part_II"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Preparation of samples for HPLC analysis part II</SPAN></A></LI><LI class="toclevel-2"><A href="#competent_cells_-_E._coli_XL1_blue"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">competent cells - E. coli XL1 blue</SPAN></A></LI><LI class="toclevel-2"><A href="#check_resistance_of_competent_cells_-_E._coli_XL1_blue"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">check resistance of competent cells - E. coli XL1 blue</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_PCR_products_mycC.2C_mdnABCDE_71.2C_89.2C_T7_and_pSB1C3"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Digest of PCR products mycC, mdnABCDE 71, 89, T7 and pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_mycC.2C_digested_mdnABCDE_71-a.2Fb.2C_89-a.2Fb.2C_T7-a.2Fb_and_digested_pSB1C3_from_today"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Purification of mycC, digested mdnABCDE 71-a/b, 89-a/b, T7-a/b and digested pSB1C3 from today</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_mycC_and_digested_mdnABCDE_71-a.2Fb.2C_89-a.2Fb.2C_T7-a.2Fb_with_digested_pSB1C3_from_today"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Ligation of mycC and digested mdnABCDE 71-a/b, 89-a/b, T7-a/b with digested pSB1C3 from today</SPAN></A></LI><LI class="toclevel-2"><A href="#ELISA_with_purified_phages"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">ELISA with purified phages</SPAN></A></LI><LI class="toclevel-2"><A href="#Send_clones_13.2C_14_.28mdnA_in_pSB.29_and_7.2C8_.28geneIII_in_pSB.29_for_sequencing"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Send clones 13, 14 (mdnA in pSB) and 7,8 (geneIII in pSB) for sequencing</SPAN></A></LI><LI class="toclevel-1"><A href="#94th_Labday_2011-09-13"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">94th Labday 2011-09-13</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation_of_samples_for_HPLC_analysis"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Preparation of samples for HPLC analysis</SPAN></A></LI><LI class="toclevel-2"><A href="#HPLC_w.2F_samples_from_2011-9-12_.28repetition.29"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">HPLC w/ samples from 2011-9-12 (repetition)</SPAN></A></LI><LI class="toclevel-2"><A href="#Test_Expression_Backbones:_pSB1K3_Ara.2FLac_CFP"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Test Expression Backbones: pSB1K3_Ara/Lac_CFP</SPAN></A></LI><LI class="toclevel-2"><A href="#Expression_Backbones:_prepare_samples_for_fluorescence_microscope"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Expression Backbones: prepare samples for fluorescence microscope</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_pSB1C3_for_mdnABCDE"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Digest of pSB1C3 for mdnABCDE</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_XL1_with_mdnABC_and_a_control_with_H2O"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">Transformation of XL1 with mdnABC and a control with H<SUB>2</SUB>O</SPAN></A></LI><LI class="toclevel-2"><A href="#Overnight_cultures_of_pARW071_mdnB_and_pARW089_mdnB"><SPAN class="tocnumber">3.7</SPAN><SPAN class="toctext">Overnight cultures of pARW071_mdnB and pARW089_mdnB</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_set"><SPAN class="tocnumber">3.8</SPAN><SPAN class="toctext">Digestion set</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_set_verification_on_Agarose_Gel.2C_gel_extraction_and_ligation"><SPAN class="tocnumber">3.9</SPAN><SPAN class="toctext">Digestion set verification on Agarose Gel, gel extraction and ligation</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated:_Digestion_set"><SPAN class="tocnumber">3.10</SPAN><SPAN class="toctext">Repeated: Digestion set</SPAN></A></LI><LI class="toclevel-2"><A href="#Check_sequenced_vector_pSB1C3_conataining_mdnA-myc-geneIII"><SPAN class="tocnumber">3.11</SPAN><SPAN class="toctext">Check sequenced vector pSB1C3 conataining mdnA-myc-geneIII</SPAN></A></LI><LI class="toclevel-1"><A href="#95th_Labday_2011-09-14"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">95th Labday 2011-09-14</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_PCR_of_mdnA_for_cloning_it_into_pSB1C3"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Repeated PCR of mdnA for cloning it into pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Digestion_of_PCR_product_mdnA_for_cloning_into_pSB1C3"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Digestion of PCR product mdnA for cloning into pSB1C3</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_mdnA_with_pSB1C3_to_generate_mdnA_as_a_biobrick"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Ligation of mdnA with pSB1C3 to generate mdnA as a biobrick</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_pSB1C3_containing_mdnA_in_E._coli"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Transformation of pSB1C3 containing mdnA in E. coli</SPAN></A></LI><LI class="toclevel-2"><A href="#Phage_Display"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">Phage Display</SPAN></A></LI><LI class="toclevel-2"><A href="#Overnight_cultures_for_Western_Blot_.28mdnB.29"><SPAN class="tocnumber">4.6</SPAN><SPAN class="toctext">Overnight cultures for Western Blot (mdnB)</SPAN></A></LI><LI class="toclevel-1"><A href="#96th_Labday_2011-09-15"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">96th Labday 2011-09-15</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_pSB1C3_ABC.2BDE_and_pSB1C3_D.2BE_into_XL1_blue"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Transformation of pSB1C3_ABC+DE and pSB1C3_D+E into XL1 blue</SPAN></A></LI><LI class="toclevel-2"><A href="#Checking_fluorescence_of_expression_backbones_via_fluorescence_microscope"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Checking fluorescence of expression backbones via fluorescence microscope</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_Gel_electrophoresis_and_gel_extraction_of_mdnABC.2C_mdnD.2C_mdnDE.2C_mdnD"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Agarose Gel electrophoresis and gel extraction of mdnABC, mdnD, mdnDE, mdnD</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_several_overnight_cultures"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Miniprep of several overnight cultures</SPAN></A></LI><LI class="toclevel-2"><A href="#sequencing_of"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">sequencing of</SPAN></A></LI><LI class="toclevel-2"><A href="#Western_Blot_of_mdnB_.2B_pARW071.2FpARW089"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">Western Blot of mdnB + pARW071/pARW089</SPAN></A></LI><LI class="toclevel-2"><A href="#overnight_culture_of_picked_E._coli_clones_transformed_with_pSB1C3_containing_mdnA"><SPAN class="tocnumber">5.7</SPAN><SPAN class="toctext">overnight culture of picked E. coli clones transformed with pSB1C3 containing mdnA</SPAN></A></LI><LI class="toclevel-1"><A href="#97th_Labday_2011-09-16"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">97th Labday 2011-09-16</SPAN></A></LI><LI class="toclevel-2"><A href="#check_plates_-_resistance_of_competent_E._coli_XL1_blue_cells"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">check plates - resistance of competent E. coli XL1 blue cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Antibody_detection_of_mdnB_.2B_pARW071.2FpARW089"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Antibody detection of mdnB + pARW071/pARW089</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_pSB1C3.2BmdnB"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Miniprep of pSB1C3+mdnB</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_several_overnight_cultures_pSB1C3.2BmdnA"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">Miniprep of several overnight cultures pSB1C3+mdnA</SPAN></A></LI><LI class="toclevel-2"><A href="#Send_clones_4.2C5.2C8.2C10_.28mdnA_in_pSB1C3.29_for_sequencing_.28after_test_digestion.29"><SPAN class="tocnumber">6.5</SPAN><SPAN class="toctext">Send clones 4,5,8,10 (mdnA in pSB1C3) for sequencing (after test digestion)</SPAN></A></LI><LI class="toclevel-2"><A href="#Repeated_Phage_Display_with_another_trypsin_protease"><SPAN class="tocnumber">6.6</SPAN><SPAN class="toctext">Repeated Phage Display with another trypsin protease</SPAN></A></LI><LI class="toclevel-2"><A href="#Digest_of_pSB1A3_.2B_YFP_clone_III.2C_ligation_with_promoters_and_transformation"><SPAN class="tocnumber">6.7</SPAN><SPAN class="toctext">Digest of pSB1A3 + YFP clone III, ligation with promoters and transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#98th_Labday_2011-09-17"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">98th Labday 2011-09-17</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_pSB1C3.2Bmdn_ABC.2BDE_and_pSB1C3.2BmdnABC"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Miniprep of pSB1C3+mdn_ABC+DE and pSB1C3+mdnABC</SPAN></A></LI><LI class="toclevel-2"><A href="#Checking_plates_from_yesterday_.28Expression_Backones.29and_picking_clones"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Checking plates from yesterday (Expression Backones)and picking clones</SPAN></A></LI><LI class="toclevel-2"><A href="#Overnight_cultures_for_Western_Blot"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Overnight cultures for Western Blot</SPAN></A></LI><LI class="toclevel-2"><A href="#Control_of_repeated_Phage_Display_with_another_trypsin_protease"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">Control of repeated Phage Display with another trypsin protease</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_14_3C2.2C3.2C4_and_TEv_2.2C3.2C4"><SPAN class="tocnumber">7.5</SPAN><SPAN class="toctext">Miniprep of 14_3C2,3,4 and TEv 2,3,4</SPAN></A></LI><LI class="toclevel-1"><A href="#99th_Labday_2011-09-18"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">99th Labday 2011-09-18</SPAN></A></LI><LI class="toclevel-2"><A href="#Miniprep_of_pSB1A3_YFP_Ara_und_Lac_cultures"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Miniprep of pSB1A3_YFP Ara und Lac cultures</SPAN></A></LI><LI class="toclevel-2"><A href="#Western_Blot_of_mdnB_.2B_pARW071.2FpARW089_2"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Western Blot of mdnB + pARW071/pARW089</SPAN></A></LI><LI class="toclevel-2"><A href="#Antibody_detection_of_mdnB_.2B_pARW071.2FpARW089_2"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Antibody detection of mdnB + pARW071/pARW089</SPAN></A></LI><LI class="toclevel-1 tocsection-1"><A href="#99th_Labday_2011-09-20"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">99th Labday 2011-09-20</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Miniprep_of_pSB1A3_YFP_Ara-cultures"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Miniprep of pSB1A3_YFP Ara-cultures</SPAN></A></LI></TD></TR></TBODY></TABLE><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="92th_Labday_2011-09-11">92th Labday 2011-09-11</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="miniprep_of_pSB1C3.2BmdnABCDE_with_and_without_T7_promotor">miniprep of pSB1C3+mdnABCDE with and without T7 promotor</SPAN></H3><P><B>Time:</B> 2011-09-11</P><P><B>Investigators:</B> Niels, Jessica, Katharina</P><P><B>Materials:</B></P><UL><LI> 6 overnight cultures
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> mdnABCDE with T7-promotor clone 1 </TD><TD> 50.6
</TD></TR><TR><TD> mdnABCDE with T7-promotor clone 2 </TD><TD> 10.6
</TD></TR><TR><TD> mdnABCDE without T7-promotor clone 1 </TD><TD> 68.4
</TD></TR><TR><TD> mdnABCDE without T7-promotor clone 2 </TD><TD> 110.2
</TD></TR><TR><TD> mdnABCDE without T7-promotor clone 3 </TD><TD> 108.0
</TD></TR><TR><TD> mdnABCDE without T7-promotor clone 4 </TD><TD> 22.3
</TD></TR><TR><TD> mdnABCDE without T7-promotor clone 5(III) </TD><TD> 504.0
</TD></TR><TR><TD> mdnABCDE without T7-promotor clone 6 (IV) </TD><TD> 212.6
</TD></TR></TBODY></TABLE><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="miniprep_of_mdnA_foc2_part_2">miniprep of mdnA foc2 part 2</SPAN></H3><P><B>Time:</B> 2011-09-11, 9:00 </P><P><B>Investigators:</B> Jessica, Niels, Katharina</P><P><B>Materials:</B></P><UL><LI> 1 overnight culture
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> mdnA foc2 </TD><TD> 155.7
</TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><UL><LI> give it to the screening group
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR_mdnABCDE_with_Long_PCR_Enzyme_mix_from_Fermentas">PCR mdnABCDE with Long PCR Enzyme mix from Fermentas</SPAN></H3><P><B>Time:</B> 2011-09-11</P><P><B>Investigators:</B> Jessica, Katharina</P><P><B>Aim:</B> generate BioBrick of mdnABCDE, amplify mdnABCDE for ligation in pSB1C3</P><P><B>Materials</B></P><UL><LI> vector: pARW089 (8,6 ng/µl), pARW071 (6,3 ng/µl)
</LI></UL><UL><LI> DNA polymerases: Long PCR Enzyme Mix (Fermentas)
</LI></UL><UL><LI> Template DNA: pARW071 (vector)
</LI></UL><UL><LI> dNTPs
</LI></UL><UL><LI> water
</LI></UL><UL><LI> Primer:
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>#</B></TH><TH><B>Primer</B></TH></TR><TR><TD> 84</TD><TD> pf_mdnABCDE89_EcoRI_NotI_XbaI (froward for pARW071 and pARW089)
</TD></TR><TR><TD> 82</TD><TD> r_mdnABCDE_iGEM (reverse for all)
</TD></TR><TR><TD> 83</TD><TD> pf_mdnABCDE89+T7_EcoRI_NotI_XbaI (froward for pARW089, generating mdnABCDE+T7)
</TD></TR></TBODY></TABLE><P><B>Protocol:</B></P><UL><LI>mdnABCDE
</LI></UL><UL><LI> 2 µl vector
</LI></UL><UL><LI> 1 µl dNTPs (10 mM)
</LI></UL><UL><LI> 3 µl forward Primer
</LI></UL><UL><LI> 3 µl reverse Primer
</LI></UL><UL><LI> 0.3 µl Polymerase
</LI></UL><UL><LI> 5 µl Buffer Fermentas Long Enzyme Mix (+MgCl2)
</LI></UL><UL><LI> 35.3 µl water
</LI></UL><UL><LI> total volume: 50 µl
</LI></UL><P>2. PCR program
</P><UL><LI>IGLONG3
</LI></UL><UL><LI> first steps: 10x
</LI></UL><UL><LI> second steps: 25x
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>Step</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>Hot Start</TD><TD> 94°C </TD><TD> Hold
</TD></TR><TR><TD> Initial denaturation</TD><TD> 94°C </TD><TD> 180 sec
</TD></TR><TR><TD>Denaturation</TD><TD> 94°C</TD><TD> 15 s
</TD></TR><TR><TD>Annealing</TD><TD> 59 </TD><TD> 30 s
</TD></TR><TR><TD> Elongation</TD><TD> 68°C</TD><TD> 324 s
</TD></TR><TR><TD>DenaturationII </TD><TD> 94°C</TD><TD> 15 s
</TD></TR><TR><TD>AnnealingII </TD><TD> 68°C</TD><TD> 30 s
</TD></TR><TR><TD>ElongationII </TD><TD> 68°C</TD><TD> 324 s + 2 s per each cycle s
</TD></TR><TR><TD> Final Elongation </TD><TD> 68°C </TD><TD> 600 s
</TD></TR></TBODY></TABLE><P><B>Results:</B></P><P><B>Further tasks:</B></P><UL><LI> PCR purification
</LI></UL><UL><LI> digest
</LI></UL><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR_of_mdnA_with_C-terminal_myc_tag_.28mycC.29">PCR of mdnA with C-terminal myc tag (mycC) </SPAN></H3><P><B>Investigators:</B> Katharina, Jessica </P><P><B>Time:</B> 2011-09-11, 14:00</P><P><B>Material:</B></P><UL><LI> pARW089 (~8 ng/µl)
</LI></UL><UL><LI> dNTPs
</LI></UL><UL><LI> primer 23, 24
</LI></UL><UL><LI> HF Phusion Buffer 5x
</LI></UL><UL><LI> Phusion Polymerase
</LI></UL><P><B>Method:</B></P><UL><LI> 1 µl pARW089
</LI></UL><UL><LI> 1 µl dNTPs (10 mM each)
</LI></UL><UL><LI> 2.5 µl forward primer (10µM)
</LI></UL><UL><LI> 2.5 µl reverse primer (10µM)
</LI></UL><UL><LI> 10 µl HF Phusion Buffer 5x
</LI></UL><UL><LI> 0.5 µl Phusion Polymerase
</LI></UL><UL><LI> 32.5 µl water
</LI></UL><UL><LI>total 50 µl
</LI></UL><UL><LI> programm IGBIO2
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TH><B>Step</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>Hot Start</TD><TD> 98°C </TD><TD> Hold </TD></TR><TR><TD> Initial denaturation</TD><TD> 98°C </TD><TD> 30 sec</TD></TR><TR><TD> Denaturation</TD><TD> 98°C</TD><TD> 10 s </TD><TD rowspan="3"> 10x
</TD></TR><TR><TD> Annealing</TD><TD> 51°C</TD><TD> 20 s
</TD></TR><TR><TD>Extension</TD><TD> 72°C</TD><TD> 20 s
</TD></TR><TR><TD> Denaturation</TD><TD> 98°C</TD><TD> 10 s</TD><TD rowspan="3"> 20x
</TD></TR><TR><TD> Annealing</TD><TD> 72°C</TD><TD> 20 s
</TD></TR><TR><TD>Extension</TD><TD> 72°C</TD><TD> 20 s
</TD></TR><TR><TD> Final extension </TD><TD> 72°C </TD><TD> 10 min </TD></TR><TR><TD> 4°C </TD><TD> Hold</TD></TR></TBODY></TABLE><P><B>Result:</B></P><UL><LI> PCR product mycN (stored in ?)
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> restriction enzyme digestion
</LI></UL><UL><LI> ligation
</LI></UL><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_mycN_and_pSB1C3">Restriction enzyme digestion of mycN and pSB1C3</SPAN></H3><P><B>Investigators:</B> Jessica, Katharina </P><P><B>Time:</B> 2011-09-11</P><P><B>Material:</B></P><UL><LI> purified PCR products
</LI></UL><UL><LI> mycN (40.0 ng/µl)
</LI></UL><UL><LI> NEB Buffer 4
</LI></UL><UL><LI> XbaI
</LI></UL><UL><LI> PstI
</LI></UL><UL><LI> BSA
</LI></UL><UL><LI> digestion of mycN
</LI></UL><UL><LI> 20 µl DNA
</LI></UL><UL><LI> 3 µl Buffer 4
</LI></UL><UL><LI> 1.5 µl XbaI
</LI></UL><UL><LI> 2.0 µl PstI
</LI></UL><UL><LI> 0.3 µl BSA
</LI></UL><UL><LI> 3.2 µl water
</LI></UL><UL><LI> digestion of pSB1C3
</LI></UL><UL><LI> 6 µl DNA (235.0 ng/µl (2011-06-22))
</LI></UL><UL><LI> 3 µl Buffer 4
</LI></UL><UL><LI> 1.5 µl XbaI
</LI></UL><UL><LI> 2.0 µl PstI
</LI></UL><UL><LI> 0.3 µl BSA
</LI></UL><UL><LI> 11.5 µl water
</LI></UL><UL><LI> concentration after digestion
</LI></UL><UL><LI> mycN: 12.2
</LI></UL><UL><LI> pSB1C3: 12.1
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="PCR_Purification_of_mdnA.2BmycC.2C_Lib_2_for_Phage_Display_and_digested_mdnA.2BmycN">PCR Purification of mdnA+mycC, Lib 2 for Phage Display and digested mdnA+mycN</SPAN></H3><P><B>Time:</B> 2011-09-11</P><P><B>Investigators:</B> Niels, Jessica, Katharina</P><P><B>Materials</B></P><UL><LI> Macherey-Nagel Nucleo Spin Extract II
</LI></UL><UL><LI> elution in 50 µl
</LI></UL><UL><LI> concentration
</LI></UL><UL><LI> mdnA+mycC: 42.6 ng/µl
</LI></UL><UL><LI> Lib 2 for Phage Display: 27.9 ng/µl
</LI></UL><UL><LI> digested mdnA+mycN: 12.2 ng/µl
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Gel_electrophoresis_of_pARWIII_and_pSB1C3">Gel electrophoresis of pARWIII and pSB1C3</SPAN></H3><P><B>Investigators:</B> Niels, Jessica, Katharina </P><P><B>Time:</B> 2011-09-11</P><P><B>Material:</B></P><UL><LI> 1% agarose gel
</LI></UL><UL><LI> 2 µl Gel Red
</LI></UL><UL><LI> Fermentas DNA Ladder Mix
</LI></UL><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix (1:10) </TD><TD> 12
</TD></TR><TR><TD> 1 </TD><TD> pARWIII </TD><TD> 30 + 6 loading dye
</TD></TR><TR><TD> 2 </TD><TD> pSB1C3 (1) </TD><TD> 30 + 6 loading dye
</TD></TR><TR><TD> 3 </TD><TD> pSB1C3 (2) </TD><TD> 30 + 6 loading dye
</TD></TR></TBODY></TABLE><UL><LI> fragments were excised and purified using the Macherey-Nagel Nucleo Spin extract II Kit
</LI></UL><UL><LI> concentration after purification
</LI></UL><UL><LI> pARWII: 3.9 ng/µl
</LI></UL><UL><LI> pSB1C3 (1): 8.4 ng/µl
</LI></UL><UL><LI> pSB1C3 (2): 12.1 ng/µl
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Restriction_enzyme_digestion_of_pSB1C3_for_control">Restriction enzyme digestion of pSB1C3 for control </SPAN></H3><P><B>Investigators:</B> Katharina, Jessica, Niels, Steffi </P><P><B>Time:</B> 2011-09-11 </P><P><B>Material:</B></P><P><B>pSB1C3(2011-06-22) :</B></P><UL><LI> concentration: 235,0 ng/µl
</LI></UL><P><B>digestion of pSB1C3 with Pvu II: </B></P><UL><LI> 2 µl DNA (235.0 ng/µl (2011-06-22))
</LI></UL><UL><LI> 1 µl 10xBuffer
</LI></UL><UL><LI> 1 µl PvuII
</LI></UL><UL><LI> 6 µl water
</LI></UL><UL><LI> total volume 10 µl
</LI></UL><UL><LI>37°C over night
</LI></UL><P><B>further tast </B></P><P>electrophoretic separation
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_pARWIII_.2B_Lib-2_for_Phage_Display_and_pSB1C3_.2B_mdnAmycN">Ligation of pARWIII + Lib-2 for Phage Display and pSB1C3 + mdnAmycN</SPAN></H3><P><B>Time:</B> 2011-09-11</P><P><B>Investigators:</B> Steffi, Niels, Katharina, Jessica</P><P><B>Material</B></P><UL><LI> purified pARWIII (vector) (2011-09-11)
</LI></UL><UL><LI> purified Lib-2 for Phage Display (insert) (2011-09-11)
</LI></UL><UL><LI> purified pSB1C3 (2011-09-11)
</LI></UL><UL><LI> purified mdnA + mycN
</LI></UL><P><B>Method</B></P><UL><LI> ligation of pARWIII + Lib2 for Phage Display
</LI></UL><UL><LI> 1 µl 10x Buffer
</LI></UL><UL><LI> 1 µl T4 Ligase
</LI></UL><UL><LI> 7 µl vector
</LI></UL><UL><LI> 1 µl insert
</LI></UL><UL><LI>1h @ RT </LI></UL><UL><LI> ligation of pSB1C3 + mdnAmycC
</LI></UL><UL><LI> 1 µl 10x Buffer
</LI></UL><UL><LI> 1 µl T4 Ligase
</LI></UL><UL><LI> 7 µl vector
</LI></UL><UL><LI> 1 µl insert
</LI></UL><UL><LI>1h @ RT </LI></UL><UL><LI> control for each ligation: taking water instead of insert
</LI></UL><P><B>Further tasks:</B></P><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Testing_of_expression_backbones">Testing of expression backbones</SPAN></H3><P><B>Time:</B> 2011-09-11</P><P><B>Investigators:</B> Niels, Katharina, Jessica</P><P><B>Material</B></P><UL><LI> overnight cultures of pSB1K3_Ara_CFP and pSB1K3_Lac_CFP
</LI></UL><P><B>Results:</B></P><UL><LI> cells didn't grow properly
</LI></UL><P><B>Further tasks:</B></P><UL><LI> repeat with new overnight cultures
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparing_new_overnight_cultures_of_expression_backbones">Preparing new overnight cultures of expression backbones</SPAN></H3><P><B>Time:</B> 2011-09-11</P><P><B>Investigators:</B> Niels, Katharina, Jessica</P><P><B>Material</B></P><UL><LI> glycerol stocks of pSB1K3_Ara_CFP and pSB1K3_Lac_CFP
</LI></UL><UL><LI> stocks weren't prepared correctly, so
</LI></UL><UL><LI> plates of pSB1A3_YFP_Ara, pSB1A3_YFP_Lac, pSB1K3_CFP_Ara, pSB1K3_CFP_Lac, pSB1K3_YFP_Lac
</LI></UL><P><B>Further tasks:</B></P><UL><LI> expression test with pSB1K3_Ara_CFP and pSB1K3_Lac_CFP
</LI></UL><UL><LI> glycerol stocks of all 5 cultures
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_pARWIII_.2B_Lib-2_for_Phage_Display">Transformation of pARWIII + Lib-2 for Phage Display</SPAN></H3><P><B>Time:</B> 2011-09-11</P><P><B>Investigators:</B>Niels, Jessica, Katharina, Steffi</P><P><B>Aim:</B> Transformation of <I>E. coli</I> cells with ligation of pARWIII + Lib-2 for Phage Display</P><P><B>Materials:</B></P><UL><LI> competent <I>E. coli</I> cells (XL1-Blue, 2011-08-29)
</LI></UL><UL><LI> ligation products: pARWIII + Lib-2 for Phage Display
</LI></UL><P><B>Method:</B></P><UL><LI>addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI>incubation 20 min on ice,
</LI></UL><UL><LI>heat shock 90 sec at 42°C,
</LI></UL><UL><LI>incubation 10 min on ice,
</LI></UL><UL><LI>addition of 750 µl LB medium,
</LI></UL><UL><LI>incubation at 37 °C shaking(750 rpm) for 60 min,
</LI></UL><UL><LI>centrifuge 30 min at 2000 x g
</LI></UL><UL><LI>discard the supernatant (till ~100µl media)
</LI></UL><UL><LI>plating on LB plate with appropriate antibiotic (Amp)
</LI></UL><UL><LI>storage over night at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> counting colonies
</LI></UL><UL><LI> wash colonies from plates
</LI></UL><UL><LI>o.n. cultures and miniprep
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Colony_PCR_of_E._coli_XL1_blue_transformed_with_ligation_products_of_mutated_TEV_protease.2C_mutated_14_3C_protease.2C_AraC_form_pBAD_iGEM_express.2C_pJC354_ssTorA_XhoI_CS-14_3C_NheI_blaFL_and_pJC354_ssTorA_XhoI_CS-TEV_NheI_blaFL_and_analytical_AG_.281.5.25.29"> Colony PCR of <I>E. coli</I> XL1 blue transformed with ligation products of mutated TEV protease, mutated 14_3C protease, AraC form pBAD_iGEM_express, pJC354_ssTorA_XhoI_CS-14_3C_NheI_blaFL and pJC354_ssTorA_XhoI_CS-TEV_NheI_blaFL and analytical AG (1.5%)</SPAN></H3><P><B>Investigators:</B> Stefan, Sascha, Sebastian</P><P><B>Aim:</B> checking for positive clones for further tasks</P><P><B>Methode:</B></P><UL><LI> picked clones where incubated over night at 4°C in 1 ml LB media
</LI></UL><UL><LI> 2 µl were used as template for PCR</LI></UL><UL><LI> 5 µl Buffer S with 25 mM MgCl2 (purchased by Genaxxon)
</LI></UL><UL><LI> 2,5 µl each Primer (10 mM, see list below)
</LI></UL><UL><LI> 1 µl dNTP (each 10 mM)
</LI></UL><UL><LI> 2 µl Templates
</LI></UL><UL><LI> 1 µl Taq Polymerase (purchased by Genaxxon)
</LI></UL><UL><LI> 2 µl 25 mM MgCl2 (purchased by Genaxxon)
</LI></UL><UL><LI> 34 µl H2O
</LI></UL><UL><LI><B>Total volume:</B> 50 µl</LI></UL><P><B>PCR Program:</B></P><P>Initial denat = 5min 98°C
</P><P>25x
</P><P>denat: 1min 98°C
</P><P>anneal: 1min 45sec 55°C
</P><P>extend: 1min 45sec 72°C
</P><P>final extend: 10min 72°C
</P><P><B>Templates:</B></P><UL><LI> TEV1-TEV3 (clones with ligation product of TEV protease, AraC and pJC354_ssTorA_XhoI_CS-TEV_NheI_blaFL, approx. 1,6 kbp)
</LI></UL><UL><LI> TEVbb1+TEVbb2 (clones with ligation product of 14_3C protease, AraC and pJC354_ssTorA_XhoI_CS-TEV_NheI_blaFL, approx. 1,6 kbp)
</LI></UL><UL><LI> 14_3C1-14_3C5 (clones with ligation product of 14_3C protease, AraC and pJC354_ssTorA_XhoI_CS-14_3C_NheI_blaFL, approx. 1,5 kbp)
</LI></UL><UL><LI> 14_3Cbb1+14_3Cbb2 (clones with ligation product of TEV protease, AraC and pJC354_ssTorA_XhoI_CS-14_3C_NheI_blaFL, approx. 1,5 kbp)
</LI></UL><P><B>Primer:</B></P><P>Primer for all clones containing TEV protease:
</P><UL><LI>Primer 1 - f_AraC_iGEM_HindIII
</LI></UL><UL><LI>Primer 2 - r_TEV_ACCAGC
</LI></UL><P>Primer for all clones containing 14_3C protease:
</P><UL><LI>Primer 1 - f_AraC_iGEM_HindIII
</LI></UL><UL><LI>Primer 2 - r_14_3C_tm_XbaI280_A&gt;T
</LI></UL><P><B>Results</B>:
</P><P>Clones 14_3C5 (pUP_SG18_ssTorA_CS-14_3C_blaFL_AraC-14_3C5) and Tevbb1 (pUP_SG19_ssTorA_CS-TEV_blaFL_AraC-14_3C1) showed bands with the right size. All other clones showed no visible bands. Precultures are going to be cultivated over night at 37°C 750 rpm in thermo block. PCR will be repeated tomorrow.</P><P>100µl of the positive clones are diluted in 5 ml LB media (with chloramphenicol) and incubated over night at 37°C 250 rpm for mini prep of the plasmid DNA.</P><P><B> Picture of GelDoc will follow as soon as possible </B></P><P><B>Further tasks:</B></P><UL><LI> overnight culture for <B>plasmid preparation of pUP_SG18... and pUP_SG19...</B>(<B>sequencing and preparing biobricks</B>)
</LI></UL><UL><LI> creating <B>glycerol stock cultures</B> form all positive clones (XL1 blue transformed with pUP_SG18... and XL1 blue transformed with pUP_SG19)
</LI></UL><UL><LI><I>if pUP_SG19... doesnt show any unwanted mutations, this clones needs to be digested with XbaI and PstI and ligated with digested pUP_SG13... (also digested with XbaI and PstI)</I></LI></UL><UL><LI><B>survival screening</B> with pUP_SG18... and pUP_SG19... after sequencing without any unwanted mutations
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="93th_Labday_2011-09-12">93th Labday 2011-09-12</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparation_of_samples_for_HPLC_analysis_part_II">Preparation of samples for HPLC analysis part II</SPAN></H3><P><B>Time:</B> 2011-09-12</P><P><B>Investigators:</B> Jessica, Katharina</P><P><B>Material:</B></P><UL><LI> methanol
</LI></UL><UL><LI> PALL Syringe Filter
</LI></UL><UL><LI> HPLC vial
</LI></UL><P><B>Method:</B></P><UL><LI> the pellet obtained after speedvac was resuspended in 200µl of 80% methanol
</LI></UL><UL><LI> centrifugation for 3 min @ 13000 rpm
</LI></UL><UL><LI> samples were filtered using a small PALL Syringe Filter
</LI></UL><UL><LI> the filtered sampled was put into a HPLC vial
</LI></UL><UL><LI> performance of HPLC
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="competent_cells_-_E._coli_XL1_blue">competent cells - <I>E. coli</I> XL1 blue </SPAN></H3><P><B>Investigator:</B> Katharina, Niels, Steffi</P><P><B>Aim:</B> produce competent cells
</P><P><B>Materials/Methods:</B></P><TABLE class="wikitable" border="1"><TBODY><TR><TD><B>TFB I</B></TD><TD><B>1000ml</B></TD><TD><B>200ml</B></TD></TR><TR><TD> 100mM Rubidium Chloride </TD><TD> 12.1 </TD><TD> 2.42g
</TD></TR><TR><TD> 30mM Potassium Acetate</TD><TD> 2.944 </TD><TD> 0.59g
</TD></TR><TR><TD> 10mM Calcium Chloride </TD><TD> 1.47 </TD><TD> 0.29g
</TD></TR><TR><TD> 15% w/v Glycerol (87%) </TD><TD> 150 </TD><TD> 34.5g
</TD></TR></TBODY></TABLE><P>Adjust pH to 5.8 with acetic acid</P><P>Filter sterilize the solution</P><TABLE class="wikitable" border="1"><TBODY><TR><TD><B>TFB II</B></TD><TD><B>500ml</B></TD><TD><B>100ml</B></TD></TR><TR><TD> 50mM Rubidium Chloride </TD><TD> 0.6 </TD><TD> 0.121g
</TD></TR><TR><TD> 10mM MOPS</TD><TD> 1.05 </TD><TD> 0.210g
</TD></TR><TR><TD> 75mM Calcium Chloride </TD><TD> 5.51 </TD><TD> 1.100g
</TD></TR><TR><TD> 15% w/v Glycerol (87%) </TD><TD> 75 </TD><TD> 17.24g
</TD></TR></TBODY></TABLE><P>Adjust pH to 7.0 with KOH</P><P>Filter sterilize the solution</P><P>Work always sterile and cold and speedy!
</P><UL><LI> All volumes deal with the common cellline!
</LI></UL><UL><LI> Prepare 80 Eppis (1,5?l)
</LI></UL><UL><LI> get liquid nitrogen
</LI></UL><UL><LI> prepare 5 ml LB-Medium with the specific antibiotic (for XL1-blue: Tet), inoculate and incubate over night
</LI></UL><UL><LI> prepare 200 ml LB-Medium with the specific antibiotic, inoculate with 2 ml of the over-night-culture
</LI></UL><UL><LI> grow while shaking at 37°C, 190 rpm to an OD600 at 0,4-0,6
</LI></UL><UL><LI> keep cell suspension in sterile falcons (50 ml) 10 min on ice, then centrifuge for 5 min, 4°C, 4000 rpm
</LI></UL><UL><LI> discard supernatant, carefully resuspend on ice with 40 ml icecold TFB I and keep 10 min on ice
</LI></UL><UL><LI> centrifuge for 5 min, 4°C, 4000 rpm
</LI></UL><UL><LI> discard supernatant, carefully resuspend pellet in 8 ml TFB II
</LI></UL><UL><LI> aliquot in Eppis: 50?l per tube and store immediately at liquid nitrogen and afterwards at -80 °C
</LI></UL><P><B>Results:</B></P><UL><LI> 80 tubes XL1 blue for transformation (50 µl competent cells at -80°C)
</LI></UL><P><B>Further tasks:</B> check by transformation and check the resistance on agar plates with different antibiotics
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="check_resistance_of_competent_cells_-_E._coli_XL1_blue">check resistance of competent cells - <I>E. coli</I> XL1 blue</SPAN></H3><P><B>Investigator:</B> Katharina, Niels, Steffi</P><P><B>Aim:</B> check resistance of competent XL1 blue-cells on agar plates with different antibiotics
</P><P><B>Materials/Methods:</B></P><UL><LI>competent XL1 blue-cells from 2011-09-12 (Katharina, Niels, Steffi)
</LI></UL><UL><LI>LB-plates with Tetracycline, Chloramphenicol, Kanamycine, Ampicillin
</LI></UL><UL><LI>plating 50 µl on agar plates
</LI></UL><UL><LI>incubate at 37°C over night
</LI></UL><P><B>Further tasks:</B> control agar plates
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_PCR_products_mycC.2C_mdnABCDE_71.2C_89.2C_T7_and_pSB1C3">Digest of PCR products mycC, mdnABCDE 71, 89, T7 and pSB1C3 </SPAN></H3><P><B>Time:</B> 2011-09-12</P><P><B>Investigators:</B> Jessica, Nicole, Nadja</P><P><B>Materials</B></P><UL><LI> PCR products:
</LI></UL><UL><LI> mycC
</LI></UL><UL><LI>mdnABCDE 71
</LI></UL><UL><LI>mdnABCDE 89
</LI></UL><UL><LI>mdnABCDE T7
</LI></UL><UL><LI>pSB1C3
</LI></UL><UL><LI> EcoRI, SpeI, BSA, NEB Buffer 4</LI></UL><P><B>Reaction mix</B></P><UL><LI> for PCR products:
</LI></UL><UL><LI> 10 µl PCR product
</LI></UL><UL><LI>1.5 µl EcoRI
</LI></UL><UL><LI>1.5 µl SpeI
</LI></UL><UL><LI> 2 µl Buffer 4
</LI></UL><UL><LI>0.2 µl BSA
</LI></UL><UL><LI> 4.8 µl water
</LI></UL><UL><LI> for pSB1C3:
</LI></UL><UL><LI> 14.8 µl pSB1C3
</LI></UL><UL><LI>1.5 µl EcoRI
</LI></UL><UL><LI>1.5 µl SpeI
</LI></UL><UL><LI> 2 µl Buffer 4
</LI></UL><UL><LI>0.2 µl BSA
</LI></UL><P><B>Output</B></P><UL><LI>digests:
</LI></UL><UL><LI> mycC</LI></UL><UL><LI>mdnABCDE 71 a
</LI></UL><UL><LI>mdnABCDE 71 b
</LI></UL><UL><LI>mdnABCDE 89 a
</LI></UL><UL><LI>mdnABCDE 89 b
</LI></UL><UL><LI>mdnABCDE T7 a
</LI></UL><UL><LI>mdnABCDE T7 b
</LI></UL><UL><LI>pSB1C3 </LI></UL><P><B>Further tasks</B></P><UL><LI> gel electrophoresis
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Purification_of_mycC.2C_digested_mdnABCDE_71-a.2Fb.2C_89-a.2Fb.2C_T7-a.2Fb_and_digested_pSB1C3_from_today">Purification of mycC, digested mdnABCDE 71-a/b, 89-a/b, T7-a/b and digested pSB1C3 from today</SPAN></H3><P><B>Time:</B> 2011-09-12</P><P><B>Investigators:</B> Jessica, Nicole, Nadja</P><P><B>Materials</B></P><UL><LI> Macherey-Nagel Nucleo Spin Extract II
</LI></UL><UL><LI> elution in 35 µl H2O (of mycC and mdnABCDE 71-a/b, 89-a/b, T7-a/b)
</LI></UL><UL><LI> elution in 50 µl NE eluation buffer ( of pSB1C3)
</LI></UL><UL><LI> concentration
</LI></UL><UL><LI> mycC: 23,9 ng/µl
</LI></UL><UL><LI>mdnABCDE 71 a: 135,2 ng/µl
</LI></UL><UL><LI>mdnABCDE 71 b: 48,7 ng/µl
</LI></UL><UL><LI>mdnABCDE 89 a: 26,8 ng/µl
</LI></UL><UL><LI>mdnABCDE 89 b: 9,9 ng/µl
</LI></UL><UL><LI>mdnABCDE T7 a: 89,0 ng/µl
</LI></UL><UL><LI>mdnABCDE T7 b: 120,2 ng/µl
</LI></UL><UL><LI>pSB1C3: 32,5 ng/µl</LI></UL><P><B>Further tasks</B></P><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_mycC_and_digested_mdnABCDE_71-a.2Fb.2C_89-a.2Fb.2C_T7-a.2Fb_with_digested_pSB1C3_from_today">Ligation of mycC and digested mdnABCDE 71-a/b, 89-a/b, T7-a/b with digested pSB1C3 from today</SPAN></H3><P><B>Time:</B> 2011-09-12</P><P><B>Investigators:</B> Jessica, Nicole, Nadja</P><P><B>Materials</B></P><UL><LI> pSB1C3
</LI></UL><UL><LI> mycC and digested mdnABCDE 71-a/b, 89-a/b, T7-a/b</LI></UL><P><B> Protocol</B></P><P><B> Components</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>Component</B></TD><TD><B>Molar Ratio</B></TD><TD><B>Concentration in ng/ µl</B></TD><TD><B>Length in bp</B></TD><TD><B> volume</B></TD></TR><TR><TD> Backbone pSB1C3 </TD><TD> 1 </TD><TD> 32,5</TD><TD> 2390 </TD><TD> fill up to 8µl
</TD></TR><TR><TD> Insert: mdnABCDE 71-a </TD><TD> 3 </TD><TD> 135,2 </TD><TD> 6500 </TD><TD> 5
</TD></TR><TR><TD> Insert: mdnABCDE 71-b </TD><TD> 3 </TD><TD> 48,7 </TD><TD> 6500 </TD><TD> 7
</TD></TR><TR><TD> Insert: mdnABCDE 89-a </TD><TD> 3 </TD><TD> 28,8 </TD><TD> 6500 </TD><TD> 7
</TD></TR><TR><TD> Insert: mdnABCDE 89-b </TD><TD> 3 </TD><TD> 9,9 </TD><TD> 6500 </TD><TD> 7
</TD></TR><TR><TD> Insert: mdnABCDE T7-a </TD><TD> 3 </TD><TD> 89,0 </TD><TD> 6500 </TD><TD> 6
</TD></TR><TR><TD> Insert: mdnABCDE T7-b </TD><TD> 3 </TD><TD> 120,2 </TD><TD> 6500 </TD><TD> 6
</TD></TR><TR><TD> Insert: mycC </TD><TD> 3 </TD><TD> 23,9 </TD><TD> 220 </TD><TD> 1
</TD></TR></TBODY></TABLE><P><B> Ligation mix</B></P><UL><LI> * * for a:
</LI></UL><UL><LI> 1µl T4 ligase
</LI></UL><UL><LI> 1µl T4 buffer
</LI></UL><UL><LI> fill up with water
</LI></UL><UL><LI> total volume 10 µl
</LI></UL><UL><LI> at 14 C over night
</LI></UL><UL><LI> * * for b:
</LI></UL><UL><LI> 1µl quck ligase
</LI></UL><UL><LI> 10µl quick ligase buffer
</LI></UL><UL><LI> fill up with water
</LI></UL><UL><LI> total volume 20 µl
</LI></UL><UL><LI> at 14 C over night
</LI></UL><P><B> Further task:</B></P><UL><LI> Transformation
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="ELISA_with_purified_phages">ELISA with purified phages</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B> control if mdnA-myc-geneIII will be expressed on the phage</P><P><B>Method/Materials:</B></P><UL><LI> ELISA plate coated with 5 µg/ml anti-c-myc antibody in phosohate buffer (pH 7,4), volume 100 µl/ well
</LI></UL><UL><LI> incubate for 4 h at room temperature
</LI></UL><UL><LI> blocking in 2 % milk powder in TBS (300 µl/ well)
</LI></UL><UL><LI> incubate for 2 h at room temperature
</LI></UL><UL><LI> add 100 µl phages, produced in XL1-blu cells and in ER2738 cells diluted 1:5 in TBS-T (0,005%)
</LI></UL><UL><LI> incubate shaking for 60 min at room temperature
</LI></UL><UL><LI> wash 5 x with TBS-T (0,05%)
</LI></UL><UL><LI> add anti-gene-8-antibody (HRP coupled)
</LI></UL><UL><LI> incubate shaking for 60 min
</LI></UL><UL><LI> wash 5 x with TBS-T
</LI></UL><UL><LI> put substrate on the samples
</LI></UL><UL><LI> measure in plate photometer
</LI></UL><UL><LI> reference: helper phages
</LI></UL><P><B>Resluts:</B></P><UL><LI> x-axis: 1) absorption negative control (helper phages) 2) absorption of phages produced in XL1-blue cells
</LI></UL><UL><LI> conclusion: geneIII-myc-mdnA-fusion gene is expressed on the surface of phages produced in XL1-blue cells
</LI></UL><UL><LI> at wells incubated with phages produced in ER2738 cells no significant signals were observed
</LI></UL><P><B>Further tasks:</B></P><P>Phage Display
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Send_clones_13.2C_14_.28mdnA_in_pSB.29_and_7.2C8_.28geneIII_in_pSB.29_for_sequencing">Send clones 13, 14 (mdnA in pSB) and 7,8 (geneIII in pSB) for sequencing</SPAN></H3><P><B>Investigators:</B> Sandrina</P><P><B>Aim:</B>control ligation of mdnA and geneIII into pSB1C3 for generating biobricks</P><P><B>Method/Materials:</B></P><UL><LI> Primer: VR
</LI></UL><UL><LI> 20 µl with 70 ng/µl DNA
</LI></UL><P><B>Further tasks:</B></P><P>check alignment
</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="94th_Labday_2011-09-13">94th Labday 2011-09-13</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Preparation_of_samples_for_HPLC_analysis">Preparation of samples for HPLC analysis</SPAN></H3><P><B>Time:</B> 2011-09-13, 8:00</P><P><B>Investigators:</B> Nicole, Jessica, Nadine, Katharina</P><P><B>Aim:</B> Isolation of Microviridin from <I>E. coli</I></P><P><B>Material:</B></P><UL><LI> ON cultures (XL1-blue):
</LI></UL><UL><LI> pSB1C3+mdnABCDE klon 5 (cm resistance)
</LI></UL><UL><LI> pARW089 (kan resistance)
</LI></UL><UL><LI> pARW071 (kan resistance)
</LI></UL><UL><LI> LB medium
</LI></UL><UL><LI> kanamycin
</LI></UL><UL><LI> chloramphenicol
</LI></UL><UL><LI> induction solution (content???)
</LI></UL><UL><LI> 100% methanol
</LI></UL><UL><LI> 5&amp; methanol
</LI></UL><UL><LI> sterile water
</LI></UL><UL><LI> Sep-Pak Cartridges
</LI></UL><UL><LI> speed vac
</LI></UL><P><B>Method:</B></P><UL><LI> prepare main cultures:
</LI></UL><UL><LI> 150 ml LB + 150 µl appropriate antibiotics
</LI></UL><UL><LI> add 1.5 ml ON culture
</LI></UL><UL><LI> add 150 µl induction solution (time: 8:30)
</LI></UL><UL><LI> incubation at 30°C and 200 rpm to an OD of ??? </LI></UL><UL><LI> harvesting and lysis
</LI></UL><UL><LI> pellet was resuspended in 5 ml of sterile water
</LI></UL><UL><LI> the resuspended sample was sonicated for 5 min (3 sec on, 3 sec off)
</LI></UL><UL><LI> centrifugation for 10 min @ 11000 rpm
</LI></UL><UL><LI> microviridin isolation
</LI></UL><UL><LI> supernatant was transfered to Sep-Pak Plus C18 Cartridges, that were equilibrated with 2 ml of 100% methanol
</LI></UL><UL><LI> cartridges were washed with 2 ml water
</LI></UL><UL><LI> samples were load
</LI></UL><UL><LI> cartridges were washed with 2 ml of 5% methanol
</LI></UL><UL><LI> samples were eluted with 2 ml of 100% methanol
</LI></UL><UL><LI> samples were put in a speedvac so that the methanol can evaporate
</LI></UL><P><B>Note:</B></P><UL><LI> procedure was stopped because we had positive results from first HPLC
</LI></UL><P><B>Conclusions:</B></P><UL><LI> system has not to be induced (only fosmids have to be induced)
</LI></UL><UL><LI> HPLC will be repeated w/ remaining sample (2011-9-12), fractions will be collected and analyzed by Mass Spectrometry
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="HPLC_w.2F_samples_from_2011-9-12_.28repetition.29">HPLC w/ samples from 2011-9-12 (repetition)</SPAN></H3><P><B>Time:</B> 2011-09-13, 8:00</P><P><B>Investigators:</B> Nicole, Nadine, Katharina, Vanessa, Nadja</P><P><B>Aim:</B> this time: collect fraction during the run for MS</P><P><B>Material:</B></P><UL><LI> microviridin samples (from pARW089 and pARW071) from 2011-9-12
</LI></UL><UL><LI> HPLC (Shimadzu)
</LI></UL><P><B>HPLC program:</B></P><P><B>Results:</B></P><P><B>Conclusions:</B></P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Test_Expression_Backbones:_pSB1K3_Ara.2FLac_CFP">Test Expression Backbones: pSB1K3_Ara/Lac_CFP</SPAN></H3><P><B>Time:</B> 2011-09-13 </P><P><B>Investigators:</B> Niels
</P><P><B>Aim:</B></P><P>prove of promotors (Lac/Ara)
</P><P><B>Materials</B></P><UL><LI> pSB1K3_Lac_CFP culture
</LI></UL><UL><LI> psB1K3_Ara_CFP culture
</LI></UL><P><B>Methods</B></P><UL><LI> cells from overnight cultre (2011-09-12)were transfer in new media (50ml LB + 50µl Kan (50 mg/ml))
</LI></UL><UL><LI> start cell grow in fresh media
</LI></UL><UL><LI> inducing of Ara at 0,6 (OD600)
</LI></UL><UL><LI> after inducing by IPTG/ARA - samples where measured for 2h every 15 min
</LI></UL><UL><LI> results negativ
</LI></UL><P><B>further task </B></P><UL><LI> prove of fluorescence by microscope
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Expression_Backbones:_prepare_samples_for_fluorescence_microscope">Expression Backbones: prepare samples for fluorescence microscope</SPAN></H3><P><B>Time:</B> 2011-09-13 </P><P><B>Investigators:</B> Nicole,Vanessa, Jessica, Nadine, Nadja</P><P><B>Aim:</B></P><UL><LI> plate clones on induced and non-induced plates
</LI></UL><UL><LI> also: liquid culture
</LI></UL><P><B>Materials</B></P><UL><LI> pSB1K3_Lac_CFP culture
</LI></UL><UL><LI> psB1K3_Ara_CFP culture
</LI></UL><UL><LI> pSB1A3_Lac_YFP culture
</LI></UL><UL><LI> psB1A3_Ara_YFP culture
</LI></UL><UL><LI> LB-Agar
</LI></UL><UL><LI> kanamycin
</LI></UL><UL><LI> ampicilin
</LI></UL><UL><LI> arabinose
</LI></UL><UL><LI> IPTG
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_pSB1C3_for_mdnABCDE">Digest of pSB1C3 for mdnABCDE</SPAN></H3><P><B>Time: </B>2011-9-7, 11:00</P><P><B>Investigators: </B>Nadine, Nicole</P><P><B>Aim: </B>generate a BioBrick of mdnABCDE, digest for ligation in pSB1C3</P><P><B>Materials:</B></P><UL><LI> BBa_K404304_pSB1C3 (#3), 284.6 ng/µl
</LI></UL><UL><LI> EcoRI, SpeI, PstI
</LI></UL><UL><LI> Buffer 4
</LI></UL><UL><LI> BSA
</LI></UL><UL><LI> H<SUB>2</SUB>O
</LI></UL><P><B>Protocol:</B></P><UL><LI> vector (2 batches: batch 1 w/ SpeI and batch 2 w/pstI):
</LI></UL><UL><LI> 5 µl vector
</LI></UL><UL><LI> 1 µl EcoRI
</LI></UL><UL><LI> 1 µl SpeI/PstI
</LI></UL><UL><LI> 2 µl Buffer 4
</LI></UL><UL><LI> 0.2 µl BSA
</LI></UL><UL><LI> 10.8 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> total volume: 20µl
</LI></UL><UL><LI> incubation 3 hrs at 37°C (start: 9:15)
</LI></UL><P><B>Output:</B></P><P><B>Further tasks:</B></P><UL><LI> agarose gel
</LI></UL><UL><LI> purification
</LI></UL><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_XL1_with_mdnABC_and_a_control_with_H2O">Transformation of XL1 with mdnABC and a control with H<SUB>2</SUB>O</SPAN></H3><P><B>Time:</B> 2011-09-14; 23:00 </P><P><B>Investigators:</B> Nadine, Nicole, Jessica, Nadja </P><P><B>Materials:</B></P><UL><LI> XL1-Blue
</LI></UL><UL><LI> ligation product: pSB1C3+mdnABC, Jessica, Nadja, 28.08.2011 resistance: cm
</LI></UL><UL><LI> LB-Agar plates
</LI></UL><P><B>Method:</B></P><UL><LI> ligation products:
</LI></UL><UL><LI> addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI>incubation 30 min on ice,
</LI></UL><UL><LI> heat shock 60 sec at 42°C,
</LI></UL><UL><LI>incubation 5 min on ice,
</LI></UL><UL><LI> addition of 750 µl LB medium,
</LI></UL><UL><LI>incubation at 37 °C shaking for 60 min,
</LI></UL><UL><LI> plating on LB medium with appropriate antibiotic (20 ml Agar and 20 µl antibiotic (see above) per plate)
</LI></UL><UL><LI> storage over night at 37°C (start: 15:00)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Overnight_cultures_of_pARW071_mdnB_and_pARW089_mdnB">Overnight cultures of pARW071_mdnB and pARW089_mdnB</SPAN></H3><P><B>Time:</B> 2011-09-07, 00:00 </P><P><B>Investigators:</B> Nadine ,Nicole, Jessica; Nadja</P><P><B>Aim:</B> For characterization via Western blot</P><P><B>Materials</B></P><UL><LI> LB medium with Amp
</LI></UL><UL><LI> glycerol stocks of of pARW071_mdnB and pARW089_mdnB
</LI></UL><P><B>Method:</B></P><UL><LI>inoculation of 10 ml LB medium with antibiotic
</LI></UL><UL><LI> overnight at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> testing expression
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_set">Digestion set</SPAN></H3><P><B>Time:</B> 2011-09-15, 00:00 </P><P><B>Investigators:</B> Nadine ,Nicole, Jessica; Nadja</P><P><B>Aim:</B> To build the Biobrick pSB1C3_mdnABCDE</P><P><B>Method:</B></P><P><B> Digest compositions</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>sample</B></TD><TD><B>mdnE</B></TD><TD><B>mdnD</B></TD><TD><B>mdnDE</B></TD><TD><B>mdnABC</B></TD></TR><TR><TD><B>ingrediants</B></TD><TD><B> Volume in µl</B></TD><TD><B>Volume in µl</B></TD><TD><B> Volumein µl</B></TD><TD><B> Volume in µl</B></TD></TR><TR><TD> DNA </TD><TD> 5,3 </TD><TD> 5,0 </TD><TD> 2,0 </TD><TD> 1,5
</TD></TR><TR><TD> buffer 4 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> BSA </TD><TD> 0,3 </TD><TD> 0,3 </TD><TD> 0,3 </TD><TD> 0,3
</TD></TR><TR><TD> H2O </TD><TD> 18,7</TD><TD> 18,7 </TD><TD> 21,7 </TD><TD> 22,2
</TD></TR><TR><TD> PstI</TD><TD> 1,5 </TD><TD> 1,5 </TD><TD> 1,5 </TD><TD> 1,5
</TD></TR><TR><TD> XbaI </TD><TD> 1,5 </TD><TD> - </TD><TD> 1,5 </TD><TD> -
</TD></TR><TR><TD> SpeI </TD><TD> - </TD><TD> 1,5 </TD><TD> - </TD><TD> 1,5
</TD></TR></TBODY></TABLE><UL><LI> 1,5 h at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Agarose gel
</LI></UL><UL><LI> gel extraction
</LI></UL><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_set_verification_on_Agarose_Gel.2C_gel_extraction_and_ligation">Digestion set verification on Agarose Gel, gel extraction and ligation </SPAN></H3><P><B>Time:</B> 2011-09-15, 01:15-05:30 </P><P><B>Investigators:</B>Nadine, Nicole, Jessica, Nadja</P><P><B>Materials</B></P><UL><LI> digested mdn_ACD
</LI></UL><UL><LI> digested mdn_E
</LI></UL><UL><LI> digested mdn_D
</LI></UL><UL><LI> digested mdn_DE
</LI></UL><P><B> Production of one 0,8 % gel for mdnABC and mdnDE and one 1% for mdnE and mdnD</B></P><UL><LI> 0,8 % gel: 0.4 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> 1 % gel: 0.5 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to the gel
</LI></UL><P><B>Loading gels and running</B></P><P>1. 0,8 % gel
</P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix </TD><TD> 15µl (1:100)</TD></TR><TR><TD> 1 </TD><TD> mdn_ABC </TD><TD> 20µl + 4ml 6x loading dye </TD><TD> 4819
</TD></TR><TR><TD> 2 </TD><TD> mdn_DE </TD><TD> 20µl + 4ml 6x loading dye </TD><TD> 2759 (including mdnDE), 2377
</TD></TR></TBODY></TABLE><P>2. 1% gel
</P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix </TD><TD> 15µl (1:100)</TD></TR><TR><TD> 1 </TD><TD> mdn_D </TD><TD> 20µl + 4ml 6x loading dye </TD><TD> 2934
</TD></TR><TR><TD> 2 </TD><TD> mdn_E </TD><TD> 20µl + 4ml 6x loading dye </TD><TD> 2394, 2070 (including mdnE)
</TD></TR></TBODY></TABLE><UL><LI>100V
</LI></UL><UL><LI>Ca. 1h
</LI></UL><P><B> Gel extraction:</B></P><UL><LI> using Nucleospin Extract II
</LI></UL><UL><LI> NOTE: accidently elution in collection tube
</LI></UL><UL><LI> second elution with 35 µl NE Buffer
</LI></UL><P><B> Ligation:</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>Backbone</B></TD><TD><B>Concentration in ng/ µl</B></TD><TD><B>Length in bp</B></TD><TD><B> volume</B></TD><TD><B>Insert</B></TD><TD><B>Concentration in ng/ µl</B></TD><TD><B>Length in bp</B></TD><TD><B> volume</B></TD></TR><TR><TD> pSB1C3_ABC </TD><TD> 4.4 </TD><TD> 4819</TD><TD> 7.3 </TD><TD> ED </TD><TD> 5.7 </TD><TD> 2759</TD><TD> 9.7
</TD></TR><TR><TD> pSB1C3_D </TD><TD> 8.2 </TD><TD> 2934</TD><TD> 4.7 </TD><TD> E </TD><TD> 6.6 </TD><TD> 2070</TD><TD> 12.3
</TD></TR></TBODY></TABLE><P><B> Ligation mix</B></P><UL><LI> 1 µl T4 ligase
</LI></UL><UL><LI> 2 µl T4 buffer
</LI></UL><UL><LI> insert and vector according to table
</LI></UL><UL><LI> total volume 20 µl
</LI></UL><UL><LI> at RT for 45 min
</LI></UL><P><B> Further task:</B></P><UL><LI> transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Repeated:_Digestion_set">Repeated: Digestion set</SPAN></H3><P><B>Time:</B> 2011-09-15, 4:30 </P><P><B>Investigators:</B> Nadine ,Nicole, Jessica; Nadja</P><P><B>Aim:</B> To build the Biobrick pSB1C3_mdnABCDE</P><P><B>Method:</B></P><P><B> Digest compositions</B></P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>sample</B></TD><TD><B>mdnE</B></TD><TD><B>mdnD</B></TD><TD><B>mdnDE</B></TD><TD><B>mdnABC</B></TD></TR><TR><TD><B>ingrediants</B></TD><TD><B> Volume in µl</B></TD><TD><B>Volume in µl</B></TD><TD><B> Volumein µl</B></TD><TD><B> Volume in µl</B></TD></TR><TR><TD> DNA </TD><TD> 5,3 </TD><TD> 5,0 </TD><TD> 2,0 </TD><TD> 1,5
</TD></TR><TR><TD> buffer 4 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3 </TD><TD> 3
</TD></TR><TR><TD> BSA </TD><TD> 0,3 </TD><TD> 0,3 </TD><TD> 0,3 </TD><TD> 0,3
</TD></TR><TR><TD> H2O </TD><TD> 18,7</TD><TD> 18,7 </TD><TD> 21,7 </TD><TD> 22,2
</TD></TR><TR><TD> PstI</TD><TD> 1,5 </TD><TD> 1,5 </TD><TD> 1,5 </TD><TD> 1,5
</TD></TR><TR><TD> XbaI </TD><TD> 1,5 </TD><TD> - </TD><TD> 1,5 </TD><TD> -
</TD></TR><TR><TD> SpeI </TD><TD> - </TD><TD> 1,5 </TD><TD> - </TD><TD> 1,5
</TD></TR></TBODY></TABLE><UL><LI> ? h at 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Agarose gel
</LI></UL><UL><LI> gel extraction
</LI></UL><UL><LI> ligation
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Check_sequenced_vector_pSB1C3_conataining_mdnA-myc-geneIII">Check sequenced vector pSB1C3 conataining mdnA-myc-geneIII </SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B>control if biobrick is generated correctly</P><P><B>Method/Materials:</B></P><UL><LI> genious
</LI></UL><UL><LI> make alignment
</LI></UL><P><B>Results:</B></P><UL><LI> three positive clones (pSB1C3 containing mdnA-myc-geneIII) fusion gene
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="95th_Labday_2011-09-14">95th Labday 2011-09-14</SPAN></H2><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_PCR_of_mdnA_for_cloning_it_into_pSB1C3">Repeated PCR of mdnA for cloning it into pSB1C3</SPAN></H3><P><B>Investigator:</B> Sabine, Sandrina</P><P><B>Time:</B> 2011-09-14,10:30-12:30 </P><P><B>Aim:</B></P><UL><LI> amplification of mdnA with iGEM-restriction sites</LI></UL><P><B>Primer:</B></P><UL><LI> primer: pf_mdnA_iGEM_EheI and pr_mdnA_iGEM_AatII</LI></UL><P><B>Reaction Components:</B></P><UL><LI> 2 µl Vector pARW089 (16 ng)
</LI></UL><UL><LI> 0,25 µl Taq Polymerase S (BioScience)
</LI></UL><UL><LI> 1 µl dNTPs
</LI></UL><UL><LI> 1 µl per primer
</LI></UL><UL><LI> 5 µl 10x PCR Buffer S
</LI></UL><UL><LI> 39,75 µl water
</LI></UL><P><B>Further tasks:</B></P><UL><LI> purification
</LI></UL><UL><LI> digestion
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Digestion_of_PCR_product_mdnA_for_cloning_into_pSB1C3">Digestion of PCR product mdnA for cloning into pSB1C3</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine</P><P><B>Aim:</B> cloning of mdnA into pSB1C3</P><P><B>Time:</B> 12:30-14:00</P><P><B>Material/Method:</B></P><UL><LI> 30 µl PCR product mdnA
</LI></UL><UL><LI> 1 µl restriction enzyme XbaI
</LI></UL><UL><LI> 1 µl restriction enzyme PstI
</LI></UL><UL><LI> 4 µl NEB 10x buffer 2
</LI></UL><UL><LI> 0,4 µl BSA
</LI></UL><UL><LI> 3,6 µl water
</LI></UL><UL><LI> 1,5 h, 37°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gel electrophoresis for purification
</LI></UL><UL><LI> ligation of mdnA with pSB1C3
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Ligation_of_mdnA_with_pSB1C3_to_generate_mdnA_as_a_biobrick">Ligation of mdnA with pSB1C3 to generate mdnA as a biobrick</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine</P><P><B>Time:</B> 2011-09-14, 14:00-15:00</P><P><B>Material/Method:</B></P><UL><LI> 8,9 µl mdnA (4 ng/µl)
</LI></UL><UL><LI> 8,1 µl pSB1C3 (17 ng/µl)
</LI></UL><UL><LI> 2 µl 10x T4 ligase buffer 3
</LI></UL><UL><LI> 1 µl T4 ligase
</LI></UL><UL><LI> 1,4 µl water
</LI></UL><UL><LI> 1 h, room temperature
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> Transformation in XL1-blue cells
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_pSB1C3_containing_mdnA_in_E._coli">Transformation of pSB1C3 containing mdnA in <I>E. coli</I></SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine</P><P><B>Time:</B> 2011-09-14, 15:00-16:30</P><P><B>Aim:</B>amplification of vectors</P><P><B>Method:</B></P><UL><LI> addition of 4 µl ligation reaction to XL1-blue cells</LI></UL><UL><LI> incubation 20 min on ice,</LI></UL><UL><LI> heat shock 45 sec at 42°C,</LI></UL><UL><LI> incubation 2 min on ice,</LI></UL><UL><LI> addition of 750 µl LB medium,</LI></UL><UL><LI> incubation 60 min at 37 °C and 750 rpm</LI></UL><UL><LI> plating on agar plates containing 100 µg/ml chloramphenicol</LI></UL><UL><LI> storage over night at 37°C</LI></UL><P><B>Further tasks:</B> control cell clones
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Phage_Display">Phage Display</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine </P><P><B>Time:</B> 2011-09-14, 16:00-03:00</P><P><B>Aim:</B>check if unmodified mdnA on the surface of the phages binds to an immobilized protease. interactions of microviridin L with this protease were already detected</P><P><B>Material/Method:</B></P><UL><LI> in ELISA plate:
</LI></UL><UL><LI> coat wells with protease trypsin in NaCO3 with ph 9,4, for 4 h
</LI></UL><UL><LI> blocking in TBS with 3% BSA for 2 h
</LI></UL><UL><LI> wash 6x with TBS-T (0,005%)
</LI></UL><UL><LI> panning with phages 1:1 helper phages and phages of interest (presenting mdnA-geneIII fusion protein on their surface, 1 h)
</LI></UL><UL><LI> wash 6x with TBS-T
</LI></UL><UL><LI> elute bound phages with 0,2 M Glycin-HCl, 10 min
</LI></UL><UL><LI> neutralize with 1 M Tris (pH 7-8)
</LI></UL><UL><LI> mix eluted phages with preparatory culture of XL1-blue cells
</LI></UL><UL><LI> incubate 10 min at 37°C without shaking
</LI></UL><UL><LI> incubate 50 min at 37°C shaking
</LI></UL><UL><LI> plate on amp plates and on kan plates
</LI></UL><P><B>Further tasks:</B></P><P>check cell growing
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Overnight_cultures_for_Western_Blot_.28mdnB.29">Overnight cultures for Western Blot (mdnB)</SPAN></H3><P><B>Investigators:</B>???</P><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> Nadine: Who did this?</TD></TR></TBODY></TABLE><P><B>Time:</B>2011-9-14, ???</P><P><B>Aim:</B>Prepare cultures for Western Blot from pARW089, pARW071</P><P><B>Material:</B></P><UL><LI> LB
</LI></UL><UL><LI> Amp
</LI></UL><UL><LI> glycerol stocks:
</LI></UL><UL><LI> pARW089
</LI></UL><UL><LI> pARW071
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="96th_Labday_2011-09-15">96th Labday 2011-09-15</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Transformation_of_pSB1C3_ABC.2BDE_and_pSB1C3_D.2BE_into_XL1_blue">Transformation of pSB1C3_ABC+DE and pSB1C3_D+E into XL1 blue</SPAN></H3><P><B>Investigators:</B>Katharina</P><P><B>Time:</B>7:00-8:45</P><P><B>Aim:</B>Transformation of Ligation</P><P><B>Materials:</B></P><UL><LI> competent <I>E. coli</I> cells XL1-Blue
</LI></UL><UL><LI> ligation products:
</LI></UL><UL><LI> pSB1C3_ABC+De + ligation control
</LI></UL><UL><LI> pSB1C3_D+E + ligation control
</LI></UL><P><B>Method:</B></P><UL><LI>addition of 2 µl ligation reaction to cells (XL1-blue) in 1.5 ml Eppi,
</LI></UL><UL><LI>incubation 20 min on ice,
</LI></UL><UL><LI>heat shock 45 sec at 42°C,
</LI></UL><UL><LI>incubation 3 min on ice,
</LI></UL><UL><LI>addition of 750 µl LB medium,
</LI></UL><UL><LI>incubation at 37 °C shaking for 60 min,
</LI></UL><UL><LI>plating on LB medium with appropriate antibiotic (Cm)
</LI></UL><UL><LI>storage @ 37°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI> Picking clones for overnight culture
</LI></UL><UL><LI> Producing glycerol stocks
</LI></UL><UL><LI> miniprep
</LI></UL><UL><LI> sending for sequencing by latest Friday!!
</LI></UL><P><B>Output:</B></P><UL><LI> 4 plates with Cm:
</LI></UL><UL><LI> pSB1C3_ABC+DE
</LI></UL><UL><LI> control pSB1C3_ABC
</LI></UL><UL><LI> pSB1C3_D+E
</LI></UL><UL><LI> control pSB1C3_D
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Checking_fluorescence_of_expression_backbones_via_fluorescence_microscope">Checking fluorescence of expression backbones via fluorescence microscope</SPAN></H3><P><B>Time:</B> 2011-09-15; 7:00-10:00</P><P><B>Investigators:</B>Katharina</P><P><B>Materials</B></P><UL><LI> liquid cultures of the expression backbones
</LI></UL><UL><LI> pSB1A3_YFP_Ara
</LI></UL><UL><LI> pSB1A3_YFP_Lac
</LI></UL><UL><LI> pSB1K3_CFP_Ara
</LI></UL><UL><LI> pSB1K3_CFP_Lac
</LI></UL><UL><LI> pSB1K3_YFP_Lac
</LI></UL><P><B>Method</B></P><UL><LI> induction using IPTG and Arabinose, respectively
</LI></UL><UL><LI> keep shaking @ 37°C for 2h
</LI></UL><UL><LI> check fluorescence of induced and uninduced, control culture with fluorescence microscope (AG Walz)
</LI></UL><P><B>Results</B></P><UL><LI> no fluorescence was detected (just autofluorescence)
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Agarose_Gel_electrophoresis_and_gel_extraction_of_mdnABC.2C_mdnD.2C_mdnDE.2C_mdnD">Agarose Gel electrophoresis and gel extraction of mdnABC, mdnD, mdnDE, mdnD </SPAN></H3><P><B>Time:</B> 2011-09-15</P><P><B>Investigators:</B>Katharina</P><P><B>Materials</B></P><UL><LI> digested mdn_ACD
</LI></UL><UL><LI> digested mdn_E
</LI></UL><UL><LI> digested mdn_D
</LI></UL><UL><LI> digested mdn_DE
</LI></UL><P><B> Production of one 1% gel</B></P><UL><LI> 1 % gel: 0.5 g agarose in 50 ml 1x TAE buffer
</LI></UL><UL><LI> Adding 2 µl gel red to the gel
</LI></UL><P><B>Loading gels and running</B></P><P>1. 0,8 % gel
</P><TABLE border="1" class="wikitable"><TBODY><TR><TD><B>lane</B></TD><TD><B>Sample</B></TD><TD><B>Volume in µl</B></TD><TD><B>Expected size in bp</B></TD></TR><TR><TD> M </TD><TD> Gene Ruler DNA Ladder Mix </TD><TD> 15µl (1:100)</TD></TR><TR><TD> 1 </TD><TD> mdn_ABC </TD><TD> 30µl + 5µl 6x loading dye </TD></TR><TR><TD> 2 </TD><TD> mdn_D </TD><TD> 30µl + 5µl 6x loading dye </TD></TR><TR><TD> 3 </TD><TD> mdn_DE </TD><TD> 30µl + 5µl 6x loading dye </TD></TR><TR><TD> 4 </TD><TD> mdn_E </TD><TD> 30µl + 5µl 6x loading dye </TD></TR></TBODY></TABLE><UL><LI>100V
</LI></UL><UL><LI> Ca. 1h then 25V for 2h because of seminar
</LI></UL><UL><LI> samples were run out of the gel, but it was decided that we do not need these samples!
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_several_overnight_cultures">Miniprep of several overnight cultures</SPAN></H3><P><B>Investigators:</B> Steffi, Niels, Nadja</P><P><B> Aim: </B></P><P><B>1. Miniprep:</B></P><UL><LI> 2 overnight cultures
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl elution buffer
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl </TH><TH> 260/280 </TH><TH> 260/230
</TH></TR><TR><TD> N1 </TD><TD> 388,8 </TD><TD> 1,98 </TD><TD> 2,27
</TD></TR><TR><TD> N2 </TD><TD> 288,3 </TD><TD> 1,99 </TD><TD> 2,16
</TD></TR><TR><TD> N3 </TD><TD> 506,9 </TD><TD> 2,03 </TD><TD> 2,45
</TD></TR><TR><TD> N4 </TD><TD> 243,0 </TD><TD> 1,99 </TD><TD> 2,37
</TD></TR><TR><TD> N5 </TD><TD> 267,2 </TD><TD> 1,99 </TD><TD> 2,43
</TD></TR><TR><TD> N6 </TD><TD> 243,4 </TD><TD> 2,02 </TD><TD> 2,32
</TD></TR><TR><TD> N7 </TD><TD> 298,9 </TD><TD> 1,99 </TD><TD> 2,20
</TD></TR><TR><TD> N8 </TD><TD> 235,4 </TD><TD> 1,95 </TD><TD> 2,08
</TD></TR><TR><TD> N9 </TD><TD> 182,2 </TD><TD> 1,93 </TD><TD> 2,68
</TD></TR><TR><TD> N10 </TD><TD> 225,5 </TD><TD> 1,92 </TD><TD> 2,34
</TD></TR><TR><TD> N11 </TD><TD> 213,8 </TD><TD> 1,91 </TD><TD> 2,41
</TD></TR><TR><TD> N12 </TD><TD> 236,9 </TD><TD> 1,91 </TD><TD> 2,19
</TD></TR><TR><TD> N13 </TD><TD> 215,1 </TD><TD> 1,91 </TD><TD> 2,32
</TD></TR><TR><TD> N14 </TD><TD> 209,2 </TD><TD> 1,91 </TD><TD> 2,32
</TD></TR><TR><TD> N15 </TD><TD> 213,0 </TD><TD> 1,89 </TD><TD> 2,01
</TD></TR><TR><TD> N16 </TD><TD> 197,6 </TD><TD> 1,90 </TD><TD> 2,07
</TD></TR></TBODY></TABLE><UL><LI> stored in -20°C (red box, expression backbones)</LI></UL><P><B>2. Preparation of glycerol stocks:</B></P><UL><LI> adding 300 µl glycerol to 700 µl culture</LI></UL><P><B>further task</B></P><UL><LI>sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="sequencing_of">sequencing of </SPAN></H3><P><B>Investigators:</B> Niels, Vanessa</P><P><B> Aim: </B></P><P><B>1. sequencing:</B></P><UL><LI> 16 clones (miniprep 2011-09-15)
</LI></UL><UL><LI> 1500 ng in 20 µl:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> DNA [µl] </TH><TH> Water [µl]
</TH></TR><TR><TD> N1 </TD><TD> 3,5 </TD><TD> 16,5
</TD></TR><TR><TD> N2 </TD><TD> 5 </TD><TD> 15
</TD></TR><TR><TD> N3 </TD><TD> 3 </TD><TD> 17
</TD></TR><TR><TD> N4 </TD><TD> 5,5 </TD><TD> 14,5
</TD></TR><TR><TD> N5 </TD><TD> 5 </TD><TD> 15
</TD></TR><TR><TD> N6 </TD><TD> 5,5 </TD><TD> 14,5
</TD></TR><TR><TD> N7 </TD><TD> 4,5 </TD><TD> 15,5
</TD></TR><TR><TD> N8 </TD><TD> 6 </TD><TD> 14
</TD></TR><TR><TD> N9 </TD><TD> 8 </TD><TD> 12
</TD></TR><TR><TD> N10 </TD><TD> 6 </TD><TD> 14
</TD></TR><TR><TD> N11 </TD><TD> 6,5 </TD><TD> 13,5
</TD></TR><TR><TD> N12 </TD><TD> 6 </TD><TD> 14
</TD></TR><TR><TD> N13 </TD><TD> 6,5 </TD><TD> 13,5
</TD></TR><TR><TD> N14 </TD><TD> 6,5 </TD><TD> 14
</TD></TR><TR><TD> N15 </TD><TD> 6 </TD><TD> 14
</TD></TR><TR><TD> N16 </TD><TD> 7 </TD><TD> 13
</TD></TR></TBODY></TABLE><P><B>seqencing</B></P><UL><LI> by GATC
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Western_Blot_of_mdnB_.2B_pARW071.2FpARW089">Western Blot of mdnB + pARW071/pARW089 </SPAN></H3><P><B>Investigators:</B> Niels, </P><P><B> Aim: </B></P><P>detect mdnB by Immunoblot
</P><P><B>Method</B></P><UL><LI> overnight cultures spin at max speed
</LI></UL><UL><LI> supernatant discard
</LI></UL><UL><LI> cells were resuspended in 1 ml 0,5 M Tris-Hcl pH 7,4
</LI></UL><UL><LI> cells where broken by Branson Digital Sonifier
</LI></UL><UL><LI>1s on / 3s off
</LI></UL><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> Nadine: 40 min sonification?!? that is quite long for 1 ml cell suspension...</TD></TR></TBODY></TABLE><UL><LI>30%amplitude
</LI></UL><UL><LI>for 10 min
</LI></UL><UL><LI> spin for 30 min at 4°C by 3220xg
</LI></UL><UL><LI> 15 µl sample were mixed with 5µl 4x SDS-loading buffer
</LI></UL><UL><LI> incubated by 95°C for 5 min
</LI></UL><UL><LI> load on gradient gel
</LI></UL><UL><LI> 40V for 30min (max mA)
</LI></UL><UL><LI> 20mA until end (max V)
</LI></UL><UL><LI> gel and membrane were equilibrated for 10 minutes in Western blot transfer buffer
</LI></UL><UL><LI> blots at 200mA for 1 hour
</LI></UL><UL><LI> blocked over night in PBS-T (5% w/v milk powder) at 4°C
</LI></UL><P><B>Materials</B></P><P>Western blot transfer buffer
</P><UL><LI>15,6 mM Tris
</LI></UL><UL><LI>120 mM Glycine
</LI></UL><P>PBS-T (pH 7.4)
</P><UL><LI>140 mM NaCl
</LI></UL><UL><LI>2.7 mM KCL
</LI></UL><UL><LI>8 mM Na2HPO4
</LI></UL><UL><LI>18 mM KH2PO4
</LI></UL><UL><LI>0,01% Tween 20
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="overnight_culture_of_picked_E._coli_clones_transformed_with_pSB1C3_containing_mdnA">overnight culture of picked <I>E. coli</I> clones transformed with pSB1C3 containing mdnA </SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B> control ligation of mdnA into pSB1C3</P><P><B>Method/Materials:</B></P><UL><LI> 10 clones from pSB1C3 with mdnA (chloramphenicol)
</LI></UL><UL><LI> 5 ml LB medium per clone
</LI></UL><UL><LI> storage over night at 37°C and 800 rpm</LI></UL><P><B>Further tasks:</B></P><UL><LI> test digestion</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="97th_Labday_2011-09-16">97th Labday 2011-09-16</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="check_plates_-_resistance_of_competent_E._coli_XL1_blue_cells">check plates - resistance of competent <I>E. coli</I> XL1 blue cells</SPAN></H3><P><B>Investigators:</B> Sascha, Steffi</P><P><B>Aim:</B> check resistance of competent cells - <I>E. coli</I> XL1 blue (from 2011-09-12, Niels, Katharina, Steffi)</P><P><B>Materials:</B></P><UL><LI> agar plates from competent cells - <I>E. coli</I> XL1 blue from 2011-09-12 (Kat/Nie/Ste)</LI></UL><P><B>Results:</B></P><UL><LI> all competent cells - <I>E. coli</I> XL1 blue grow on agar plates with Tetracycline
</LI></UL><UL><LI> no <I>E. coli</I> XL1 clones on agar plates with Ampicillin, Kanamycine, Chloramphenicol
</LI></UL><P><A href="/File:UP_XL1_comp_cells.jpg" title="File:UP XL1 comp cells.jpg">File:UP XL1 comp cells.jpg</A></P><P><B>Conclusions:</B></P><UL><LI> competent cells - <I>E. coli</I> XL1 blue work and can be used for transformation
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Antibody_detection_of_mdnB_.2B_pARW071.2FpARW089">Antibody detection of mdnB + pARW071/pARW089 </SPAN></H3><P><B>Investigators:</B> Steffi</P><P><B>Aim:</B> control if mdnB + pARW071/pARW089 is presented on the membrane</P><P><B>Method:</B></P><UL><LI> incubate blocked membrane (Niels, 2011-09-15) for 1 h with primary antibody (anti-mdnB-antibody, 1:10000 in TBS-T)
</LI></UL><UL><LI> wash 3x 10 min with TBS-T buffer
</LI></UL><UL><LI> incubate for 1 h with secondary antibody (ZAMAK-POD, 1:5000 in TBS-T)
</LI></UL><UL><LI> wash 3x 10 min with TBS-T buffer
</LI></UL><UL><LI> develop membranes with ECL- Kit</LI></UL><P><B>Results:</B></P><UL><LI> no signals were seen
</LI></UL><P><B>Further tasks:</B></P><UL><LI> repeat SDS-PAGE for mdnB + pARW071/pARW089
</LI></UL><UL><LI> repeat Western Blot and Antibody-Detection
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_pSB1C3.2BmdnB">Miniprep of pSB1C3+mdnB</SPAN></H3><P><B>Investigator:</B> Katharina, Steffi</P><P><B>Materials:</B></P><UL><LI> liquid culture of pSB1C3+mdnB
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> pSB1C3+mdnB 1 </TD></TR><TR><TD> pSB1C3+mdnB 2 </TD></TR><TR><TD> pSB1C3+mdnB 3 </TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight:bold;"><SPAN class="mw-headline" id="Miniprep_of_several_overnight_cultures_pSB1C3.2BmdnA">Miniprep of several overnight cultures pSB1C3+mdnA </SPAN></H3><P><B>Investigator:</B> Katharina, Steffi</P><P><B>Materials:</B></P><UL><LI> liquid culture of pSB1C3+mdnA
</LI></UL><UL><LI> NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
</LI></UL><UL><LI> Protocol for high-copy plasmids
</LI></UL><UL><LI> elution with 50 µl H<SUB>2</SUB>O
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl
</TH></TR><TR><TD> pSB1C3+mdnA 1 </TD><TD> 323.7
</TD></TR><TR><TD> pSB1C3+mdna 2 </TD><TD> 309.3
</TD></TR><TR><TD> pSB1C3+mdna 3 </TD><TD> 342.8
</TD></TR><TR><TD> pSB1C3+mdnA 4 </TD><TD> 326.7
</TD></TR><TR><TD> pSB1C3+mdnA 5 </TD><TD> 248.8
</TD></TR><TR><TD> pSB1C3+mdnA 6 </TD><TD> 261.8
</TD></TR><TR><TD> pSB1C3+mdnA 7 </TD><TD> 266.6
</TD></TR><TR><TD> pSB1C3+mdnA 8 </TD><TD> 267.7
</TD></TR><TR><TD> pSB1C3+mdnA 9 </TD><TD> 229.7
</TD></TR><TR><TD> pSB1C3+mdnA 10 </TD><TD> 305.1
</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> test digestion for confirmation
</LI></UL><UL><LI> sending for sequencing
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Send_clones_4.2C5.2C8.2C10_.28mdnA_in_pSB1C3.29_for_sequencing_.28after_test_digestion.29">Send clones 4,5,8,10 (mdnA in pSB1C3) for sequencing (after test digestion)</SPAN></H3><P><B>Investigators:</B> Sabine, Sandrina</P><P><B>Aim:</B>control ligation of mdnA and geneIII into pSB1C3 for generating biobricks</P><P><B>Method/Materials:</B></P><UL><LI> Primer: VF2
</LI></UL><UL><LI> 20 µl with 70 ng/µl DNA
</LI></UL><P><B>Further tasks:</B></P><P>check alignment
</P><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Repeated_Phage_Display_with_another_trypsin_protease">Repeated Phage Display with another trypsin protease</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine </P><P><B>Time:</B> 2011-09-16, 9:00-17:00</P><P><B>Aim:</B>check if unmodified mdnA on the surface of the phages binds to an immobilized protease. interactions of microviridin L with this protease were already detected</P><P><B>Material/Method:</B></P><UL><LI> in ELISA plate:
</LI></UL><UL><LI> coat wells with protease trypsin in NaCO3 with ph 9,4, for 4 h
</LI></UL><UL><LI> blocking in TBS with 3% BSA for 2 h
</LI></UL><UL><LI> wash 6x with TBS-T (0,005%)
</LI></UL><UL><LI> panning with phages 1:1 helper phages and phages of interest (presenting mdnA-geneIII fusion protein on their surface, 1 h)
</LI></UL><UL><LI> wash 6x with TBS-T
</LI></UL><UL><LI> elute bound phages with 0,2 M Glycin-HCl, 10 min
</LI></UL><UL><LI> neutralize with 1 M Tris (pH 7-8)
</LI></UL><UL><LI> mix eluted phages with preparatory culture of XL1-blue cells
</LI></UL><UL><LI> store over night at 4°C
</LI></UL><P><B>Further tasks:</B></P><P>check binding of phages by plating them
</P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Digest_of_pSB1A3_.2B_YFP_clone_III.2C_ligation_with_promoters_and_transformation">Digest of pSB1A3 + YFP clone III, ligation with promoters and transformation</SPAN></H3><P><B>Investigators:</B> Nadine, Jessica, Niels</P><P><B>Time:</B> 2011-09-16, 16:00-22:00 </P><P><B>Material:</B></P><UL><LI> pSB1A3_YFP clone III, 15.08.11, Steffi
</LI></UL><UL><LI> digested promoters: Ara 1 (NW, 16.08.11), lac 1(NW, 16.08.11), Ara 2, lac 2
</LI></UL><P><B>Digest:</B></P><UL><LI> 30 µl reaction:
</LI></UL><UL><LI> 10 µl pSB1A3_YFP
</LI></UL><UL><LI> 3 µl NEB Buffer 4
</LI></UL><UL><LI> 0.3 µl BSA
</LI></UL><UL><LI> 1.5 µl EcoRI-HF
</LI></UL><UL><LI> 1.5 µl XbaI
</LI></UL><UL><LI> 13.7 µl water
</LI></UL><UL><LI> 37°C for 1.5 h
</LI></UL><P><B>Gel electrophoresis:</B></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> lane </TH><TH> Sample </TH><TH> Volume in µl </TH><TH> Expected size in bp
</TH></TR><TR><TD> M </TD><TD> GeneRuler™ DNA Ladder Mix (diluted 1:10) </TD><TD> 20 </TD></TR><TR><TD> 1 </TD><TD> - </TD><TD> - </TD></TR><TR><TD> 2 </TD><TD> digest of pSB1A3+YFP </TD><TD> 36 </TD></TR></TBODY></TABLE><P>[[File:|200px]]</P><P><B>Gel extraction:</B></P><UL><LI> using NucleoSpin Extact II
</LI></UL><UL><LI> elution with 50 µl NE buffer
</LI></UL><P><B>Ligation:</B></P><UL><LI> 1 µl T4 ligase buffer
</LI></UL><UL><LI> 1 µl T4 ligase
</LI></UL><UL><LI> 4 µl backbone
</LI></UL><UL><LI> 4 µl insert
</LI></UL><P><B>Transformation:</B></P><TABLE style="text-align: left; width: 300px; margin-left: auto; margin-right: 0px;" border="2" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; background-color: rgb(204, 204, 204);"> Nadine: please do not forget to write about the transformation protocol (I think it was Niels´ task?!?)</TD></TR></TBODY></TABLE><P><B>Output:</B></P><UL><LI> amp-LB agar plates
</LI></UL><UL><LI> pSB1A3_YFP_Ara1
</LI></UL><UL><LI> pSB1A3_YFP_Ara2
</LI></UL><UL><LI> pSB1A3_YFP_lac1
</LI></UL><UL><LI> pSB1A3_YFP_lac2
</LI></UL><UL><LI> pSB1A3_YFP control
</LI></UL><P><B>Further task:</B></P><UL><LI> picking clones for overnight plates
</LI></UL><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="98th_Labday_2011-09-17">98th Labday 2011-09-17</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_pSB1C3.2Bmdn_ABC.2BDE_and_pSB1C3.2BmdnABC">Miniprep of pSB1C3+mdn_ABC+DE and pSB1C3+mdnABC</SPAN></H3><P><B>Investigator:</B> Katharina</P><P><B>Materials:</B></P><UL><LI> liquid culture of pSB1C3+mdn_ABC+DE and pSB1C3+mdnABC
</LI></UL><UL><LI> Buffer A1, A2, A3 of the Macherey-Nagel Kit
</LI></UL><UL><LI> Isopropanol
</LI></UL><UL><LI> 70% EtOH
</LI></UL><P><B>Method:</B></P><UL><LI> pellet the cells
</LI></UL><UL><LI> resuspend in 150 µl Buffer A1
</LI></UL><UL><LI> add 150 µl Buffer A2 and invert the tube
</LI></UL><UL><LI> incubate @ room temperature for 5 min
</LI></UL><UL><LI> add 150 µl Buffer A3 and invert the tube
</LI></UL><UL><LI> centrifuge for 5 min @ 11000 rpm
</LI></UL><UL><LI> transfer supernatant to a new tube
</LI></UL><UL><LI> add 1000 µl isopropanol
</LI></UL><UL><LI> vortex and keep at room temperature for 5 min
</LI></UL><UL><LI> centrifuge for 5-10 min @ 11000 rpm
</LI></UL><UL><LI> discard the supernatant and wash the pellet in 70% ethanol
</LI></UL><UL><LI> centrifuge for 5-10 min @ 11000 rpm
</LI></UL><UL><LI> discard the supernatant
</LI></UL><UL><LI> dry the pellet (either @ room temperature or @ 37°C)
</LI></UL><UL><LI> resuspend the pellet in 30 µl sterile water
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl </TH><TH> 260/280 </TH><TH> 260/230
</TH></TR><TR><TD> mdnABC 1 </TD><TD> 4013.3 </TD><TD> 2.07 </TD><TD> 2.19
</TD></TR><TR><TD> mdnABC 2 </TD><TD> 4549.2 </TD><TD> 2.06 </TD><TD> 2.21
</TD></TR><TR><TD> mdn_ABC+DE 1 </TD><TD> 3691.1 </TD><TD> 2.03 </TD><TD> 2.20
</TD></TR><TR><TD> mdn_ABC+DE </TD><TD> 3012.9 </TD><TD> 1.97 </TD><TD> 2.07
</TD></TR></TBODY></TABLE><UL><LI> stored in green mdn-biobrick box (eppis have yellow labels on top)
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Checking_plates_from_yesterday_.28Expression_Backones.29and_picking_clones">Checking plates from yesterday (Expression Backones)and picking clones</SPAN></H3><P><B>Investigator:</B> Nadine</P><P><B>Time:</B> 2011-9-17, ~15:00</P><P><B>Materials:</B></P><UL><LI> amp-LB agar plates (2011-9-16, Niels)
</LI></UL><UL><LI> pSB1A3_YFP_Ara1
</LI></UL><UL><LI> pSB1A3_YFP_Ara2
</LI></UL><UL><LI> pSB1A3_YFP_lac1
</LI></UL><UL><LI> pSB1A3_YFP_lac2
</LI></UL><UL><LI> pSB1A3_YFP control
</LI></UL><UL><LI>LB
</LI></UL><UL><LI>ampicillin
</LI></UL><P><B>Results:</B></P><UL><LI>control plate: ~40 clones
</LI></UL><UL><LI>ligation plates: ~ 160 clones
</LI></UL><P><B>Protocol:</B></P><UL><LI> 5 ml LB
</LI></UL><UL><LI> add 5 µl Amp
</LI></UL><UL><LI> pick 3 colonies per plate and add to LB medium
</LI></UL><P><B>Output:</B></P><UL><LI> 12 overnight cultures:
</LI></UL><UL><LI> pSB1A3_YFP_Ara2, Klon 1, 2 and 3
</LI></UL><UL><LI> pSB1A3_YFP_Ara1, Klon 4, 5 and 6
</LI></UL><UL><LI> pSB1A3_YFP_lac2, Klon 7, 8 and 9
</LI></UL><UL><LI> pSB1A3_YFP_lac2, Klon 10, 11 and 12
</LI></UL><P><B>Further tasks:</B></P><UL><LI> miniprep tomorrow ( DO NOT FORGET THE GLYCEROL STOCKS)
</LI></UL><UL><LI> overnight cultures for monday for fluorescence tests
</LI></UL><UL><LI> sequencing on monday?!?!
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Overnight_cultures_for_Western_Blot">Overnight cultures for Western Blot </SPAN></H3><P><B>Investigator:</B> Nadine</P><P><B>Time:</B> 2011-9-17, ~15:00</P><P><B>Materials:</B></P><P>glycerol stocks:
</P><UL><LI> pARW089
</LI></UL><UL><LI> pARW071
</LI></UL><UL><LI> pUP089 (#G2)
</LI></UL><UL><LI> pSB1C3+mdnABCDE (#G61 and #G62)
</LI></UL><UL><LI>LB
</LI></UL><UL><LI>ampicillin
</LI></UL><UL><LI>chloramphenicol
</LI></UL><UL><LI>kanamycin
</LI></UL><P><B>Protocol:</B></P><UL><LI> 10 ml LB
</LI></UL><UL><LI> add 10 µl antiobiotics
</LI></UL><UL><LI> add cells
</LI></UL><UL><LI> incubate overnight at 37°C and 200 rpm
</LI></UL><H3 style="background-color: rgb(238, 221, 130); font-weight: bold;"><SPAN class="mw-headline" id="Control_of_repeated_Phage_Display_with_another_trypsin_protease">Control of repeated Phage Display with another trypsin protease</SPAN></H3><P><B>Investigator:</B> Sandrina, Sabine </P><P><B>Time:</B> 2011-09-17, 11:00-18:00</P><P><B>Aim:</B>check if unmodified mdnA on the surface of the phages binds to an immobilized protease. interactions of microviridin L with this protease were already detected</P><P><B>Material/Method:</B></P><UL><LI> over night culture of XL1-blue cells was infected with eluted phages
</LI></UL><UL><LI> 1000 cells were infected with 10.000 phages and plated on agar with kanamycin and on agar with ampicillin
</LI></UL><UL><LI> control: 1:1 mix of helperphages and produced phages of interest carrying mdnA-geneIII plated also on agar with kanamycin and on agar with ampicillin</LI></UL><P><B>Further tasks:</B></P><UL><LI> check cell growth
</LI></UL><H3 style="background-color: rgb(100, 150, 100); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_14_3C2.2C3.2C4_and_TEv_2.2C3.2C4">Miniprep of 14_3C2,3,4 and TEv 2,3,4</SPAN></H3><P><B>For better understanding of described experiment see also: [<A href="http://141.89.201.101/iGEM/wiki2011/images/c/c8/UP_SG_Klonierungsschema_Protease.pptx" class="external autonumber" rel="nofollow">[1]</A>]</B></P><P><B>Investigators:</B> Sascha, Stefan, Sebastian</P><P><B>Aim:</B> get plasmids</P><P><B>Methode:</B></P><UL><LI> pellet the cells
</LI></UL><UL><LI> resuspend in 150 µl Buffer A1
</LI></UL><UL><LI> add 150 µl Buffer A2 and invert the tube
</LI></UL><UL><LI> incubate @ room temperature for 5 min
</LI></UL><UL><LI> add 150 µl Buffer A3 and invert the tube
</LI></UL><UL><LI> centrifuge for 5 min @ 11000 rpm
</LI></UL><UL><LI> transfer supernatant to a new tube
</LI></UL><UL><LI> add 1000 µl isopropanol
</LI></UL><UL><LI> vortex and keep at room temperature for 5 min
</LI></UL><UL><LI> centrifuge for 5-10 min @ 11000 rpm
</LI></UL><UL><LI> discard the supernatant and wash the pellet in 70% ethanol
</LI></UL><UL><LI> centrifuge for 5-10 min @ 11000 rpm
</LI></UL><UL><LI> discard the supernatant
</LI></UL><UL><LI> dry the pellet (either @ room temperature or @ 37°C)
</LI></UL><UL><LI> resuspend the pellet in 30 µl sterile water
</LI></UL><P><B>Further Tasks:</B></P><P>send plasmids for sequenzing</P><H2 style="background-color: rgb(240, 20, 70);"><SPAN class="mw-headline" id="99th_Labday_2011-09-18">99th Labday 2011-09-18</SPAN></H2><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Miniprep_of_pSB1A3_YFP_Ara_und_Lac_cultures">Miniprep of pSB1A3_YFP Ara und Lac cultures </SPAN></H3><P><B>Time:</B> 2011-09-18 </P><P><B>Investigator:</B> Nadja</P><P><B>Materials:</B></P><UL><LI> liquid culture: * 12 overnight cultures:
</LI></UL><UL><LI> pSB1A3_YFP_Ara2, Klon 1, 2 and 3
</LI></UL><UL><LI> pSB1A3_YFP_Ara1, Klon 4, 5 and 6
</LI></UL><UL><LI> pSB1A3_YFP_lac2, Klon 7, 8 and 9
</LI></UL><UL><LI> pSB1A3_YFP_lac2, Klon 10, 11 and 12
</LI></UL><UL><LI> Buffer A1, A2, A3 of the Macherey-Nagel Kit
</LI></UL><UL><LI> Isopropanol
</LI></UL><UL><LI> 70% EtOH
</LI></UL><P><B>Method:</B></P><UL><LI> pellet the cells
</LI></UL><UL><LI> resuspend in 150 µl Buffer A1
</LI></UL><UL><LI> add 150 µl Buffer A2 and invert the tube
</LI></UL><UL><LI> incubate @ room temperature for 5 min
</LI></UL><UL><LI> add 150 µl Buffer A3 and invert the tube
</LI></UL><UL><LI> centrifuge for 5 min @ 11000 rpm
</LI></UL><UL><LI> transfer supernatant to a new tube
</LI></UL><UL><LI> add 1000 µl isopropanol
</LI></UL><UL><LI> vortex and keep at room temperature for 5 min
</LI></UL><UL><LI> centrifuge for 5-10 min @ 11000 rpm
</LI></UL><UL><LI> discard the supernatant and wash the pellet in 500µl 70% ethanol
</LI></UL><UL><LI> centrifuge for 5-10 min @ 11000 rpm
</LI></UL><UL><LI> discard the supernatant
</LI></UL><UL><LI> dry the pellet (either @ room temperature or @ 37°C)
</LI></UL><UL><LI> resuspend the pellet in 30 µl sterile water
</LI></UL><UL><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl </TH><TH> 260/280 </TH><TH> 260/230
</TH></TR><TR><TD> pSB1A3_YFP_Ara2 Clone1 </TD><TD> 718, 8 </TD><TD> 1,9 </TD><TD> 1,74
</TD></TR><TR><TD> pSB1A3_YFP_Ara2 Clone2 </TD><TD> 5360, 5 </TD><TD> 1,9 </TD><TD> 1,95
</TD></TR><TR><TD> pSB1A3_YFP_Ara2 Clone3 </TD><TD> 5882, 9 </TD><TD> 2,0 </TD><TD> 1,86
</TD></TR><TR><TD> pSB1A3_YFP_Ara1 Clone4 </TD><TD> 668 </TD><TD> 1, 2 </TD><TD> 1,14
</TD></TR><TR><TD> pSB1A3_YFP_Ara1 Clone5 </TD><TD> 5570, 3 </TD><TD> 1,9 </TD><TD> 1,58
</TD></TR><TR><TD> pSB1A3_YFP_Ara1 Clone6 </TD><TD> 7170,3 </TD><TD> 2,0 </TD><TD> 1,93
</TD></TR><TR><TD> pSB1A3_YFP_Lac2 Clone7</TD><TD> 3275,8 </TD><TD> 5,7 </TD><TD> -3.41
</TD></TR><TR><TD> pSB1A3_YFP_Lac2 Clone8 </TD><TD> 4471,6 </TD><TD> 2,0 </TD><TD> 1,61
</TD></TR><TR><TD> pSB1A3_YFP_Lac2 Clone9 </TD><TD> 4577,7 </TD><TD> 2,1 </TD><TD> 1,97
</TD></TR><TR><TD> pSB1A3_YFP_Lac1 Clone10 </TD><TD> 3535,8 </TD><TD> 1,9</TD><TD> 1,5
</TD></TR><TR><TD> pSB1A3_YFP_Lac1 Clone11 </TD><TD> 3904,6 </TD><TD> 2,1 </TD><TD> 2,22
</TD></TR><TR><TD> pSB1A3_YFP_Lac1 Clone12</TD><TD> 5395,4 </TD><TD> 2,0 </TD><TD> 1,91
</TD></TR></TBODY></TABLE><UL><LI> stored in a new green expression backbones box at -20°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI>overnight cultures for fluorescence measurement
</LI></UL><UL><LI>sending for sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Western_Blot_of_mdnB_.2B_pARW071.2FpARW089_2">Western Blot of mdnB + pARW071/pARW089 </SPAN></H3><P><B>Time:</B> 2011-09-18 </P><P><B>Investigator:</B> Sebastian, Nadja</P><P><B>Method:</B></P><UL><LI> overnight cultures spin at max speed
</LI></UL><UL><LI> supernatant discard
</LI></UL><UL><LI> cells were resuspended in 1 ml 0,5 M Tris-Hcl pH 7,4
</LI></UL><UL><LI> cells where broken by Branson Digital Sonifier
</LI></UL><UL><LI>2s on / 2s off
</LI></UL><UL><LI>30%amplitude
</LI></UL><UL><LI>for 2min
</LI></UL><UL><LI> spin for 10 min by 17.0 xg
</LI></UL><UL><LI> 15 µl sample were mixed with 5µl 4x SDS-loading buffer
</LI></UL><UL><LI> incubated by 95°C for 5 min
</LI></UL><UL><LI> load on gradient gel
</LI></UL><UL><LI> 50V for 30min (max mA)
</LI></UL><UL><LI> 40mA until end (max V)
</LI></UL><UL><LI> one gel and membrane were equilibrated for 10 minutes in Western blot transfer buffer
</LI></UL><UL><LI> membrane was immobilized with methanol
</LI></UL><UL><LI> blots at 200mA for 1 hour
</LI></UL><UL><LI> blocked over night in PBS-T (5% w/v milk powder) at 4°C
</LI></UL><UL><LI> second gel was stained and shaked over night in Coomassie
</LI></UL><P><B>Materials:</B></P><P>1. Western blot transfer buffer
</P><UL><LI>15,6 mM Tris
</LI></UL><UL><LI>120 mM Glycine
</LI></UL><P>2. PBS-T (pH 7.4)
</P><UL><LI>140 mM NaCl
</LI></UL><UL><LI>2.7 mM KCL
</LI></UL><UL><LI>8 mM Na2HPO4
</LI></UL><UL><LI>18 mM KH2PO4
</LI></UL><UL><LI>0,01% Tween 20
</LI></UL><P><B>Further tasks:</B></P><UL><LI>washing steps and treatment with first and second antibody for immunodetection of mdnB
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;"><SPAN class="mw-headline" id="Antibody_detection_of_mdnB_.2B_pARW071.2FpARW089_2">Antibody detection of mdnB + pARW071/pARW089 </SPAN></H3><P><B>Investigators:</B> Steffi</P><P><B>Aim:</B> control if mdnB + pARW071/pARW089 is presented on the membrane</P><P><B>Method:</B></P><UL><LI> incubate blocked membrane (Nadja, Sebastian, 2011-09-16) for 1 h with primary antibody (anti-mdnB-antibody, 1:10000 in TBS-T)
</LI></UL><UL><LI> wash 3x 10 min with TBS-T buffer
</LI></UL><UL><LI> incubate for 1 h with secondary antibody (ZAMAK-POD, 1:5000 in TBS-T)
</LI></UL><UL><LI> wash 3x 10 min with TBS-T buffer
</LI></UL><UL><LI> develop membranes with ECL- Kit</LI></UL><P><B>Results:</B></P><UL><LI> no signals were seen
</LI></UL><P><B>Further tasks:</B></P><UL><LI> repeat SDS-PAGE for mdnB + pARW071/pARW089
</LI></UL><UL><LI> repeat Western Blot and Antibody-Detection
</LI></UL><H2><SPAN class="mw-headline" id="99th_Labday_2011-09-20"><P style="background-color: rgb(240, 20, 70);">99th Labday 2011-09-20</P></SPAN></H2><H3><SPAN class="mw-headline" id="Miniprep_of_pSB1A3_YFP_Ara-cultures"><P style="background-color: rgb(254, 122, 122); font-weight: bold;">Miniprep of pSB1A3_YFP Ara-cultures </P></SPAN></H3><P><B>Investigator:</B> Steffi<B>Materials:</B></P><UL><LI>  2 overnight cultures
</LI><LI> pSB1A3_YFP Ara1 I
</LI><LI> pSB1A3_YFP Ara2 II
</LI></UL><LI> Buffer A1, A2, A3 of the Macherey-Nagel Kit
</LI><LI> Isopropanol
</LI><LI> 70% EtOH
</LI><P><B>Method:</B></P><UL><LI> pellet the cells
</LI><LI> resuspend in 250 µl Buffer A1
</LI><LI> add 250 µl Buffer A2 and invert the tube
</LI><LI> incubate @ room temperature for 5 min
</LI><LI> add 250 µl Buffer A3 and invert the tube immediately
</LI><LI> centrifuge for 10 min @ 11000 rpm
</LI><LI> transfer supernatant to a new tube
</LI><LI> add 0,7 volume of isopropanol
</LI><LI> invert the tupe and keep at room temperature for 5 min
</LI><LI> centrifuge for 10 min @ 11000 rpm
</LI><LI> discard the supernatant and wash the pellet in 500µl 70% ethanol
</LI><LI> centrifuge for 10 min @ 11000 rpm
</LI><LI> discard the supernatant
</LI><LI> dry the pellet (either @ room temperature or @ 37°C)
</LI><LI> resuspend the pellet in 30 µl sterile water
</LI><LI> measuring concentration with NanoDrop:
</LI></UL><TABLE class="wikitable" border="1"><TBODY><TR><TH> Sample </TH><TH> concentration in ng/µl </TH><TH> 260/280 </TH><TH> 260/230
</TH></TR><TR><TD> pSB1A3_YFP Ara1 I </TD><TD> 3441,1 </TD><TD> 2,0 </TD><TD> 2,26
</TD></TR><TR><TD> pSB1A3_YFP Ara2 II  </TD><TD> 4874,4 </TD><TD> 2,11 </TD><TD> 2,39
</TD></TR></TBODY></TABLE><UL><LI> stored in a new green expression backbones box at -20°C
</LI></UL><P><B>Further tasks:</B></P><UL><LI>overnight cultures for fluorescence measurement
</LI><LI>sending for sequencing
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;">Analysis of psB1A3_Ava_YFP and psB1A3_IPTG_YFP by fluorescence spectroscopy.
 </H3><P><B>Investigators:</B> Niels, Jessica</P><P><B>Materials:</B></P><UL><LI>psB1A3_Ava_YFP over night culture
</LI><LI>psB1A3_IPTG_YFP over night culture
</LI></UL><P><B>Methods:</B></P><UL><LI>1 ml culture were transfered in new media
</LI><LI>observed growing by measuring OD(600nm)
</LI><LI>inducing of promotor by 5 mM Ara or 1 mM IPTG at a OD(600nm) by 0.4-0.5)
</LI><LI>measuring of samples every 15 mintes by fluorescence spectroscopy
</LI></UL><P><B>Results:</B></P><P><B>further tasks:</B></P><UL><LI>analyze by fluorescence microscopy
</LI></UL><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;">Analysis of psB1A3_Ava_YFP and psB1A3_IPTG_YFP by fluorescence microscopy.
 </H3><P><B>Investigators:</B> Katharina</P><P><B>Materials:</B></P><UL><LI>psB1A3_Ava_YFP over night culture
</LI><LI>psB1A3_IPTG_YFP over night culture
</LI></UL><P><B>Methods:</B></P><UL><LI>1 ml culture were transfered in new media
</LI><LI>observed growing by measuring OD(600nm)
</LI><LI>inducing of promotor by 5 mM Ara or 1 mM IPTG at a OD(600nm) by 0.4-0.5)
</LI><LI>measuring of samples by fluorescence microscopy (1h after inducing)
</LI></UL><P><B>Results:</B></P><H3 style="background-color: rgb(254, 122, 122); font-weight: bold;">Sending of BioBricks to MIT
 </H3><P><B>Investigators:</B> Nicole, Nadine, Sebastian</P><P><B>BioBricks</B></P><TABLE class="wikitable" border="1"><TBODY><TR><TH> Label number </TH><TH> BioBrick Nickname </TH><TH> Tube label </TH><TH> c [ng/µl]
</TH></TR><TR><TD> BBa_K627000 </TD><TD> mdnED </TD><TD> 1 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627001 </TD><TD> mdnA </TD><TD> 2 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627002 </TD><TD> mdnB </TD><TD> 3 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627003 </TD><TD> mdnC </TD><TD> 4 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627004 </TD><TD> mdnD </TD><TD> 5 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627005 </TD><TD> mdnE </TD><TD> 6 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627006 </TD><TD> mdnA c-myc gene III </TD><TD> 7 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627007 </TD><TD> c-myc gene III </TD><TD> 8 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627008 </TD><TD> AraC TEV protease 1 </TD><TD> 9  </TD><TD> 25
</TD></TR><TR><TD> BBa_K627009 </TD><TD> AraC TEV protease 2 </TD><TD> 10 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627010 </TD><TD> AraC TEV protease 3 </TD><TD> 11 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627011 </TD><TD> AraC 14_3C protease </TD><TD> 12 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627012 </TD><TD> ssTorA CS-TEV BlaFL </TD><TD> 13 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627013 </TD><TD> ssTorA CS-14_3C BlaFL </TD><TD> 14 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627014 </TD><TD> A3_Ara_YFP clone 1 </TD><TD> 15 </TD><TD> 25
</TD></TR><TR><TD> BBa_K627015 </TD><TD> A3_lac_YFP clone 2 </TD><TD> 16 </TD><TD> 25
</TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/September_part_2">http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/September_part_2</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Labjournal/September_part_2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Labjournal/September_part_2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/September_part_2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Labjournal/September_part_2&amp;oldid=164242" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>