<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Project_Details_Phage"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Project/Details_Phage">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Project/Details_Phage&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Details_Phage&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Details_Phage&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Project/Details_Phage" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Project/Details Phage</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team">Team</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Members">Members</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Team#Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Project</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary">Summary</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Highlights">Highlights</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Microviridin">     Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Phage_Display">     Phage Display</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#In_Vivo_Selection">     In Vivo Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Summary#Modeling">     Modeling</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Microviridin">Details Microviridin</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">Details Phage</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Selection">Details Selection</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Modeling">Details Modeling</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">BioBricks</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Data_Page">Data Page</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/BioBricks">Details BioBricks</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Labjournal">Labjournal</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Methods">Methods</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Meetings">Meetings</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics">Safety &amp; Ethics</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Assesment">Safety Assessment</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Safety_Questions">Safety Questions</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Ethics_Seminar">Ethics Seminar</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Survey">Survey</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Potsdam_Bioware_at_the_German_parliament">Parliament Visit</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Statement_of_the_German_government">     Statement</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Safety_Ethics#Outreach">Outreach</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software">Software</A></DT><DD class="first"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#BioLog_App">BioLog App</A></DD><DD><A href="https://2011.igem.org/Team:Potsdam_Bioware/Software#Download">Download</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2011.igem.org/Team:Potsdam_Bioware/Sponsors">Sponsors</A></DT></DL></LI></UL></DIV><DIV class="pageContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Phage_Display"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Phage Display</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Cloning_strategy"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Cloning strategy</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Control_of_expression_of_mdnA-myc-geneIII_in_E._coli"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Control of expression of mdnA-myc-geneIII in E. coli</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Detection_of_phages_carrying_mdnA_on_their_surface_by_ELISA"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Detection of phages carrying mdnA on their surface by ELISA</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Testing_phage_display_with_unmodified_mdnA_to_examine_its_suitability_as_screening_method"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Testing phage display with unmodified mdnA to examine its suitability as screening method</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Testing_phage_display_with_unmodified_mdnA_against_further_proteases"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Testing phage display with unmodified mdnA against further proteases</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Phage_Display"> Phage Display </SPAN></H2><H3><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H3><P>Phage Display is an efficient tool for selecting proteins or peptides with specific binding properties from a large recombinant library. In phage display proteins are presented on the surface of bacteriophages. This enables the coupling of phenotype and stable packaged genotype, because the proteins which form the phage including the proteins of interest are coded in its genome. To test the suitability of phage display system as an appropriate screening method for recombinant <I>mdnA</I> libraries a vector containing a <I>mdnA-myc-geneIII</I>-fusion gene was generated. This vector contains a plasmid origin of replication, so it can be amplified like plasmids. Additionally it contains a f1 ori which enables the packaging of single strand DNA into phages. The vector also contains the whole <I>mdn</I>-cluster which is needed to produce the MdnA peptide. Between <I>mdnA</I> and <I>gene III</I> is located a <I>myc</I>-tag, which is used for an easy detection. The successful expression of the MdnA-myc-geneIII-fusion protein on the surface of the phage was determined by ELISA using anti-myc antibody 9E10 after transforming <I>E. coli</I> cells and purifying the produced phages. The next step was to perform a phage display on a known target of the MdnA. To test the fundamental suitability of this screening method, phages presenting MdnA on their surface and phages not presenting MdnA in a ratio of one to one were incubated with immobilized trypsin which is known as a target of MdnA. After the first panning round a notable concentrating of phages carrying MdnA was recognized.</P><H3><SPAN class="mw-headline" id="Cloning_strategy">Cloning strategy</SPAN></H3><P>The phage display vector pPDV089 was derived from the plasmid pARW089 which carries the whole <I>mdn</I>-cluster. This plasmid contains a plasmid origin of replication and additionally a f1 ori which enables the packaging of single strand DNA into phages. For selective amplification ampcillin and kanamycin resistance genes are included. To create the phagemid pPDV089 standard cloning procedure were performed. 
First <I>mdnA</I> was excised using the restriction enzymes  <I>Sfo</I>I and <I>Aat</I>II. The next step comprised the introduction of a <I>mdnA-geneIII</I>-fusion gene. Therefore  <I>gene III</I> was amplified from pak100blaKDIR and <I>mdnA</I> from pARW089 by PCR. The primers were designed to enable the introduction of iGEM and other restriction sites required for further cloning steps. The purified PCR product <I>geneIII</I> was digested by <I>NgoM</I>IV and <I>Aat</I>II whereas the PCR product <I>mdnA</I> was digested by <I>Sfo</I>I and <I>Age</I>I. Subsequent the three fragment ligation of <I>mdnA</I> and <I>geneIII</I> into the digested vector has been conducted. Thus a <I>mdnA-geneIII</I>-fusion part according to RFC25 was generated whereby <I>Age</I>I and <I>NgoM</I>IV overhangs are compatible and  placed in frame with the protein sequence. The ligation of <I>Age</I>I and <I>NgoM</I>IV overhangs resulted in a scar coding for the threonine and glycine. Because the introduction of restriction sites before <I>mdnA</I> led to a great distance between ribosome binding site (RBS) and <I>mdnA</I>. A second  RBS was inserted among <I>Sfo</I>I and <I>Xba</I>I recognition sites to ensure a sufficient expression rate of the <I>mdnA-geneIII</I>-fusion gene. The <I>myc</I> sequence located between <I>mdnA</I> and <I>gene III</I> allows the detection of the expression of the <I>mdnA-geneIII</I>-fusion protein on the surface of the phage using western blot or ELISA. In the last step the kanamycin resistance gene was disturbed because for phage display the selection of cells carriyng both helper phages and the phagemid is beneficial and the helper phages have a kanamycin resistance, too. Therefore a 300 bp fragment of the kanamycin resistance gene was deleted using the restriction enzyme <I>Nsi</I>I which had two recognition sites in the kanamycin gene.
</P><TABLE border="0" cellspacing="0" cellpadding="2"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 1: Cloning strategy for creating a plasmid which can be used for phage display with <I>mdnA</I>. </B><I>MdnA</I> was cut out of the vector and a <I>mdnA-myc-geneIII</I> fusion gene was created and ligated with the vector pARW089 containing the <I>mdn</I>-cluster without <I>mdnA</I>.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 2: Designed vector pPDV089 carrying the <I>mdnA-myc-gene III</I> fusion gene.</B> Therefore the <I>mdnA</I> sequence was cut out of the vector pARW089 and the ligated <I>mdnA-myc-geneIII</I> gene was inserted</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Control_of_expression_of_mdnA-myc-geneIII_in_E._coli">Control of expression of <I>mdnA</I>-<I>myc</I>-<I>geneIII</I> in <I>E. coli</I></SPAN></H3><P>The expression of the <I>mdnA</I>-<I>myc</I>-<I>geneIII</I> fusion gene was analyzed by western blotting. <I>E. coli</I> cells transformed with the phagemid pPDV089 were harvested and lysated. The proteins were electrophoretically separated and transferred to a membrane. The <I>mdnA</I>-<I>myc</I>-<I>geneIII</I>-fusion proteins were detected using specific anti-<I>myc</I>-antibodies and horseradish peroxidase (HRP)-linked antibodies as secondary antibodies. Enhanced chemiluminescence (ECL) was used to visualize the protein. ECL is based on the emission of light during the HRP-catalyzed oxidation of luminol, which was captured by a camera. The western blot analysis resulted in a band of a size just below the 30 kDa mark representing the <I>mdnA</I>-<I>myc</I>-<I>geneIII</I>-fusion protein (24 kDa).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 3: Control of expression of <I>mdnA</I>-<I>myc</I>-<I>geneIII</I> in <I>E. coli</I> by western blotting.</B> For detection anti-<I>myc</I>-antibodies and secondary HRP-linked antibodies were used. The resulting band represents the <I>mdnA</I>-<I>myc</I>-<I>geneIII</I>-fusion protein.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Detection_of_phages_carrying_mdnA_on_their_surface_by_ELISA">Detection of phages carrying <I>mdnA</I> on their surface by ELISA</SPAN></H3><P>The next step was the detection of the expression of the <I>mdnA</I>-<I>myc</I>-<I>geneIII</I>-fusion gene on the surface of the phage. So <I>E. coli</I> cells strain XL1-Blue were first transformed with the phagemid pPDV089 before they were infected with helper phages. <I>E. coli</I> cells containing both plasmids were selected. An ELISA test was performed to determine whether these cells are able to produce phage particles carrying the MdnA peptide on their surface. To perform this test anti-<I>c-myc</I>-antibodies were immobilized on ELISA plates and incubated with purified phages. For detection a second antibody coupled with horse radish peroxidase (HRP) was used which binds the gene VIII protein of the phages. The HRP substrate o-phenyldiamine (OPD) was added and in case of binding a color reaction was expected. The color shift from achromatic to yellow in wells incubated with phages produced in XL1-Blue cells showed the successful expression of <I>mdnA</I>-c-<I>myc</I>-<I>geneIII</I>-fusion protein on the phages.
For more precise results the absorption at 492 nm was measured. The data were presented in a bar plot. As a negative control helper phages were added instead of produced phages. Furthermore two wells were prepared were the secondary antibody was not added.
The graphic shows clearly the much higher absorption measured in wells, which were incubated with phage particles of interest produced in XL1-Blue cells. As it has already been pointed out this shows the succeeded expression of <I>mdnA</I>-c-<I>myc</I>-<I>geneIII</I>-fusion protein on the surface of the phages.
</P><DIV align="center"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 4:</B> Detection of phages carrying <I>mdnA</I> on their surface by ELISA. The bar plot shows the absorption at 492 nm. Anti-<I>myc</I>-antibodies were immobilized. For detection a second antibody coupled with horse radish peroxidase (HRP) was used which binds the gene VIII coat protein of the phages. The left bar shows the absorption of the wells containing helper phages (negative control), the right bar shows the absorption of wells containing <I>mdnA</I> carrying phages</DIV></DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Testing_phage_display_with_unmodified_mdnA_to_examine_its_suitability_as_screening_method">Testing phage display with unmodified <I>mdnA</I> to examine its suitability as screening method</SPAN></H3><P>To test the fundamental suitability of this screening method, phages representing unmodified <I>mdnA</I> on their surface and phages not representing <I>mdnA</I> (helper phages) in a ratio of one to one were incubated with immobilized trypsin which is known as a target of <I>mdnA</I>. The display was conducted in ELISA plates. The bound phages were eluted using a buffer with low pH value and neutralized afterwards. To check how many phages interacted with trypsin, <I>E. coli</I> cells XL1-Blue were re-infected with eluted phages and plated on agar with different antibiotics. Cells infected with phages carrying <I>mdnA</I> are able to grow on agar with ampicillin whereas cells infected with helper phages are able to grow on agar with kanamycin. To control the success of the panning round additionally <I>E. coli</I> cells were infected with phage mix before panning and plated on agar. Subsequent the number of clones grew on ampicillin and kanamycin before and after panning was compared. During the running of this step it was noticed that much more cells were infected with helper phages than with phages carrying <I>mdnA</I> despite of the engaged 1:1 ratio. This was surprising and indicated that <I>mdnA</I> on the surface of the phages may inhibit their infectivity. After controlling the plates an  infection ratio of phages carrying <I>mdnA</I> to helper phages of 1:400 was calculated. This fact should be analyzed in further experiments.
The results of the first phage display are plotted in the figure below. After one panning round an enrichment of phages carrying <I>mdnA</I> was expected. This is attributable to the fact that phage particles carrying <I>mdnA</I>-c-<I>myc'</I>-gene III-fusion protein on their surface are expected to bind specifically to the immobilized trypsin. Unfortunately this was not observed in this experiment. The ratio of cells growing on kanamycin agar before (4000) to cells growing on kanamycin agar (cells containing helper phages) after panning (750) was determined as 5:1. The ratio of cells growing on ampicillin agar before (12) to cells growing on ampicillin agar (cells containing <I>mdnA</I> carrying phages) after panning (2) was nearly equal. Thus no enrichment of <I>mdnA</I> carrying phages occurred in the first experiment. So it was decided to repeat this experiment under improved conditions. Therefor the number of washing steps during the described experimental procedure was increased. Here the ratio of cells growing on kanamycin agar before (3000) to cells growing on kanamycin agar (cells containing helper phages) after panning (29) was determined as 103:1. The ratio of cells growing on ampicillin agar before (26) to cells growing on ampicillin agar (cells containing <I>mdnA</I> carrying phages) after panning (2) was determined as 13:1. Thus an enrichment factor of eight was reached for the phages displaying <I>mdnA</I> on their surface. 
These results indicate that the unmodified <I>mdnA</I> expressed on the phages binds specifically to the immobilized trypsin. Therefore it can be deduced that <I>mdnA</I> is presented in a functional 3D structure. These findings suggest that phage display in general is an appropriate method for screening a recombinant <I>mdnA</I> library. Further experiments are required to optimize this system.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 5: Optimization of phage display.</B> After optimized conditions (right) a clear concentration of phages carrying <I>mdnA</I> after one panning round was noted. <I>E. coli</I> cells were infected with phage mix (helper phages and phages carrying <I>mdnA</I>) before and after panning and plated on agar containing kanamycin or ampicillin. The ratio of cells growing on ampicillin or kanamycin agar before panning to cells growing on ampicillin or kanamycin agar after was calculated. Helper phages which acted as negative control have a kanamycine resistance whereby phages carrying <I>mdnA</I> have an ampicillin resistance.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Testing_phage_display_with_unmodified_mdnA_against_further_proteases">Testing phage display with unmodified mdnA against further proteases</SPAN></H3><P>Furthermore the interaction of unmodified mdnA with other proteases was determined. From the literature (Ziemert, 2010) the high inhibitory activity of microviridin L, besides trypsin, against chymotrypsin and elastase is also known. So a phage display with these enzymes was performed. Additionally papain, proteinase K, mycolysin and pepsin were tested for which no interaction was shown yet. For all enzymes an equal amount of <I>E. coli</I> cells and phages were used. In agreement with data from the literature interaction of microviridin with chymotrypsin and elastase was confirmed. Additionally an interaction with papain was noted. All other proteases were not bound by microviridin. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 6: Phage display against different proteases.</B> After panning the number of clones was counted. In agreement with data from the literature interaction of microviridin with chymotrypsin and elastase was confirmed. Additionally an interaction with papain was noted.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><UL><LI>Krebber, A., Bornhauser, S., Burmester, J., Honegger, A., Willuda, J., Bosshard, H. R., Plückthun, A. (1997) Reliable cloning of functional antibody variable domains from hybridomas and spleen cell repertoires employing a reengineered phage display system. J. Immunol. Meth. 201(1):35-55 
</LI></UL><UL><LI>Rakonjac J., Feng J., Model P. (1999). Filamentous phage are released from the bacterial membrane by a two-step mechanism involving a short C-terminal fragment of pIII. J Mol Biol. 289(5):1253-65
</LI><LI>Smith, G.P. (1985). Filamentous fusion phage: Novel expression vectors that display cloned antigens on the virus surface. Science 228: 1315-17
</LI></UL><UL><LI>Ziemert, N., Ishida, K., Liaimer, A., Hertweck, C. &amp; Dittmann, E. (2008). Ribosomal synthesis of tricyclic depsipeptides in bloom-forming cyanobacteria. Angewandte Chemie (International ed. in English) 47, 7756-9
</LI></UL><UL><LI>Ziemert, N., Ishida, K., Weiz, A., Hertweck, C. &amp; Dittmann, E. (2010). Exploiting the natural diversity of microviridin gene clusters for discovery of novel tricyclic depsipeptides. Applied and environmental microbiology 76, 3568-74 
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage">http://2011.igem.org/Team:Potsdam_Bioware/Project/Details_Phage</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Project/Details_Phage" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Project/Details_Phage" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Details_Phage&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Details_Phage&amp;oldid=262007" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>