"
Page
Discussion
View source
History
teams
Log in
 
Team:Potsdam Bioware/Labjournal/October
From 2011.igem.org
Home
Team
Members
Photo Gallery
Contribution
Project
Data Page
Summary
Highlights
     Microviridin
     Phage Display
     In Vivo Selection
     Modeling
Details Microviridin
Details Phage
Details Selection
Details Modeling
BioBricks
Data Page
Details BioBricks
Labjournal
Labjournal
Methods
Meetings
Safety & Ethics
Safety Assessment
Safety Questions
Ethics Seminar
Survey
Parliament Visit
     Statement
Outreach
Software
BioLog App
Download
Sponsors
Contents
1 98th Labday 2011-09-24
1.1 Colony PCR of E.coli XL1 blue transformed UP_SG2 (1.5%)
1.2 Subcloning of 14 3C and AraC into SG2 (1.5%)
1.3 Production of phages containing pPDV089 in XL1-blue cells
2 99th Labday 2011-09-25
2.1 Agarose gel electrophoresis of maxiprimer I
2.2 Purification of phages containing pPDV089
2.3 Maxiprimer PCR I – Amplification of the end of the mdn cluster (Repetition, usage of two different polymerases)
3 100th Labday 2011-09-26
3.1 Repeated Phage Display with another trypsin protease
3.2 ELISA with purified phages
3.3 Control of Phage Display
4 101th Labday 2011-09-27
4.1 SDS- PAGE for coomassie staining with purified phages carrying mdnA on their surface
4.2 Coomassie staining
5 102th Labday 2011-10-06
5.1 Amplification of megaprimers for site-directed mutagenesis
5.2 Agarose gel electrophoresis and DNA extraction of megaprimers
5.3 Megaprimer PCR for site directed mutagenesis
6 103th Labday 2011-10-07
6.1 DpnI digestion and Transformation of mutated pARW089
6.2 Retransformation of XL1blue-cells with pPDV089
7 104th Labday 2011-10-11
7.1 Oligo-Fillin for Phage-display-mdnA-library (carrying AgeI restriction site)
7.2 Troubleshooting: Site-directed mutagenesis of pARW089 (for use as biobrick)
7.3 Megaprimer PCR for site directed mutagenesis
7.4 Digestion of ligation product geneIII into pARW089 (pARWIII) for ligation with generated mdnA library
7.5 DpnI digestion and Transformation of mutated pARW089
7.6 Over night culture of retransformed pPDV089
7.7 Digestion of geneIII for ligation with generated mdnA library
7.8 Transformation of mutated pARW089 with increased amount
7.9 Digestion of mdnA library (for phage display) using AgeI
7.10 Purification of digested mdnA library and digested gene III
7.11 Triple Ligation with pARWIII, mdnA-library and gene III
7.12 Transformation of triple ligated mdnA library, gene III and pARWIII
7.13 Sidedirected mutagenesis of HRV14 3C protease
7.14 Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments P1, P2,P3 and P4 of HRV14 3C
7.15 Assembly PCR of mutated HRV14 3C protease fragments
7.16 Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments P5 and P6 of HRV14 3C
7.17 Assembly PCR of mutated HRV14 3C protease fragments P5 and P6
8 105th Labday 2011-10-12
8.1 Production of phages containing pPDV089 in XL1-blue cells
8.2 Production of microviridin
8.3 Digest of HRV14 3C, AraC and Vector SG11
8.4 Ligation of digested fragments of HRV14_3C protease and vector SG11 and Transformation of XL1 blue with Ligation products
9 106th Labday 2011-10-13
9.1 Preparation of samples for HPLC analysis
9.2 Transformation of mutated pARW089 using super competent XL1 gold cells
9.3 Purification of phages containing pPDV089
9.4 Colony PCR of plated HRV14_3C containing colonies
10 107th Labday 2011-10-14
10.1 Digest of pARWIII and ligation with library
11 108th Labday 2011-10-15
11.1 Transformation with library for phage display
11.2 Miniprep, Glycerol stock cultures and Survival-Test of HRV14 3C clone(SG24-P10)
12 109th Labday 2011-10-16
12.1 Overnight culture of library
12.2 Survival-Test of one HRV14_3C clone( SG24-P10)
13 110th Labday 2011-10-17
13.1 Amplification of megaprimer 1 (site-directed mutagenesis)
13.2 Megaprimer PCR II (amplification of 2nd megaprimer (2nd part of mdn cluster, site-directed mutagenesis)
13.3 Sequencing of mdnA library (for phage display)
13.4 Over night culture with created mdnA library for phage display
14 111th Labday 2011-10-18
14.1 Production of phages containing microviridin library on their surface in XL1-blue cells
14.2 Sidedirected mutagenesis of TEV protease
14.3 PCR purification and analytical GE
14.4 Assembly PCR of TEV protease
14.5 PCR Purification, analytical GE, Digest and Ligation of TEV protease
15 112th Labday 2011-10-19
15.1 Amplification of megaprimer I, using Phusion (Finnzymes)
15.2 Agarose gelelectrophoresis and purification of megaprimer I
15.3 Purification of phages containing microviridin library
15.4 Colony PCR of TEV in vivo selcetion
15.5 Making TEV and HRV14_3C Biobricks
16 113th Labday 2011-10-20
16.1 ELISA with purified phages carrying unmodified mdnA on their surface
16.2 SDS- PAGE with purified phages carrying mdnA on their surface
16.3 Western Blot
16.4 Colony PCR,Mini-prep and Survival screening
16.5 Amplification of megaprimer II, using Phusion (Finnzymes)
16.6 Amplification of mdn-cluster
17 114th Labday 2011-10-21
17.1 Antibody detection with blotted membranes
17.2 Results of Survival screening at 37°C and mini prep of TEV and HRV14_3C BioBricks
17.3 Digest of mdn-cluster and pSB1C3 and ligation
18 115th Labday 2011-10-22
18.1 Results of survival screening at 30°C
18.2 Transformation with pSB1C3+mdn-cluster
19 116th Labday 2011-10-23
19.1 making competent cells and tranformation with Libary of SG26 carrying cells
19.2 Production of phages containing microviridin library on their surface in ER2738 cells
20 117th Labday 2011-10-24
20.1 Testing mdnA libary with SG26 carrying cells for TEV in vivo selection
20.2 Purification of phages containing microviridin library
21 118th Labday 2011-10-25
21.1 Control of sequencing results of sequenced clones from created library for phage display
21.2 Colony PCR of pSB1C3+mdn-cluster
22 119th Labday 2011-10-26
22.1 Phage Display with wildtype mdnA and different proteases
22.2 Digest of pARWIII and ligation with library
22.3 Transformation of triple ligated mdnA library, gene III and pARWIII
22.4 Miniprep of pSB1C3+mdn-cluster A clone 6 and sending for sequencing
22.5 SDS- PAGE with lysated cells which produced the mdnA-myc-geneIII protein
22.6 Western Blot
23 120th Labday 2011-10-27
23.1 Antibody detection with blotted membranes
23.2 control of performed phage display
98th Labday 2011-09-24
Colony PCR of E.coli XL1 blue transformed UP_SG2 (1.5%)
Investigators: Stefan, Paul, Sebastian
Aim: checking for positive clones for further tasks
Methode:
Primer: 42,67
picked clones where incubated over night at 4°C in 1 ml LB media
5 µl Buffer S with 25 mM MgCl2 (purchased by Genaxxon)
2,5 µl each Primer (10 mM, see list below)
1 µl dNTP (each 10 mM)
0.5 µl Taq Polymerase (purchased by Genaxxon)
2 µl 25 mM MgCl2 (purchased by Genaxxon)
36.5 µl H2O
Total volume: 50 µl
PCR Program:
Initial denat = 6min 94°C
30x
denat: 30sec 94°C
anneal: 30sec 55°C
extend: 30sec 72°C
final extend: 10min 72°C
Results:
no positive clone probably due to rubbishy digestion
Further tasks:
Subcloning of 14 3C and AraC into SG2 (1.5%)
Investigators: Stefan, Paul, Sascha
Aim: digestion of 14 3C protease
Methode:
14 µL H20
2 µL buffer
1 µL EcoRI
1 µL PstI
2 µL DNA
Total volume: 20 µl
digest over 2h or overnight depends how long Jessica is here
Results:
Production of phages containing pPDV089 in XL1-blue cells
Investigators: Sabine, Sandrina
Aim: control if geneIII-mdnA will be expressed on the phage and performing phage display
Method/Materials:
amplification of XL1-blue cells
50 ml over night culture will be inoculated with the cells, so that OD600 = 0.1
add tetracyclin to the medium
cells should be incubated 37 °C till OD600 = 0.3-0.5 (here: 0.356) is reached
add 30 µl phages carrying mdnA-myc-gene III on their surface
incubate for 10 min at 37 °C without shaking
incubate for 50 min shaking at 37°C
phage purification:
centrifuge cell culture at 5000 x g/ 15 min
fill supernatant in a new 50 ml falcon and centrifuge again (5000 g/15 min)
40 ml of the supernatant with 8 ml PEG-NaCl (20% (w/v) PEG-8000, 2,5 M NaCl)
incubate over night at 4°C
Further tasks:
go on with phage purification
99th Labday 2011-09-25
Agarose gel electrophoresis of maxiprimer I
Time: 2011-09-25, 20:00-23:00
Investigators: Nicole
Idea:Examination if PCR for Amplification of maxiprimer I works
Materials
Maxiprimer a and b, PCR done by Nadja, 2011-09-25
Broad range agarose
1x TAE buffer
Gel red
Gene Ruler 100 bp ladder (Fermentas)
Gene Ruler DNA ladder mix (Fermentas)
6x loading dye (Promega)
Methods:
2 % agarose gel (1 g agarose and 50 ml 1x TAE buffer)
addition of 2 µl gel red
Load 50 µl sample and 8 µl loading dye
3.5 µl of each DNA ladder
Results:
Label
Sample
Expected size in bases
1
100bp DNA ladder
2
A
129
3
A
129
4
B
129
5
B
129
6
DNA ladder mix
Media: UP_
Further tasks:
DNA extraction
Maxiprimer PCR I using the extracted DNA as forward primer
Gel electrophoresis and DNA extraction, expected size 431 bases
Maxiprimer PCR II using the extracted DNA as forward primer
Agarose gel electrophoresis and DNA extraction, expected size 6619 bases
Restriction enzyme digestion using EcoRI and SpeI
Ligation with pSB1C3
Transformation
Purification of phages containing pPDV089
Investigators: Sabine, Sandrina
Aim: control if geneIII-mdnA will be expressed on the phage, repeated phage display
Method/Materials:
after over night incubation:
centrifuge precipitaed phages: 5000 x g/ 45 min
discard supernatant
centrifuge again, 5000 x g/ 5 min
remove supernatant carefully
take pellet in 1 ml TBS
move in 1.5 ml Eppi
centrifuge again: 17000 x g/10 min
supernatant in a new Eppi with 200 µl PEG-NaCl (mix!)
incubate on ice for 60 min
centrifuge precipitated phages: 17.000 x g/ 10 min (4°C)
resuspend pellet in 300 µl TBS
centrifuge:17 000 x g/ 10 min (4°C)
supernatant in a fresh eppi= purified phages
Further tasks:
ELISA and phage display
Maxiprimer PCR I – Amplification of the end of the mdn cluster (Repetition, usage of two different polymerases)
Time: 2011-09-25, 20:00-23:00
Investigators: Nicole
Idea:Checking out two different polymerase to obtain amplification
Materials
purified Maxiprimer a and b, PCR done by Nadja, 2011-09-25
pARW089 clone 2, 85.7 ng/ µl
PfuUltra
PfuUltra HF Buffer
Primer reverse: r_mdnABCDE_iGEM (10 µM)
LongPCR Enzyme Mix (Fermentas)
Long PCR Enzyme Mix buffer (Fermentas)
dNTPs (10 mM)
Methods:
1. PfuUltra Polymerase
PCR mixture, maxiprimer A
Volume in µl
PfuUltra Buffer HF
5
dNTPs
1
Template DNA, pARW089
1.2
r_mdnABCDE_iGEM
1
Maxiprimer A
5.9
PfuUltra Polymerase
1
H
2
O
34.9
Labeled: P1
PCR mixture, maxiprimer B
Volume in µl
PfuUltra Buffer HF
5
dNTPs
1
Template DNA, pARW089
1.2
r_mdnABCDE_iGEM
1
Maxiprimer B
8.2
PfuUltra Polymerase
1
H
2
O
32.6
Labeled: P2
Thermal cycling program
Step
Temperature in °C
Time in seconds
Hot start
95
Hold
Initial Denaturation
95
120
Denaturation
95
30
Annealing
54
30
Extension
72
60
Cycle number
30
Final extension
72
600
2. Long PCR enzyme mix
PCR mixture, maxiprimer A
Volume in µl
Long PCR enzyme mix buffer
5
dNTPs
1
Template DNA, pARW089
1.2
r_mdnABCDE_iGEM
1
Maxiprimer A
24.1
Long PCR enzyme mix
0.3
H
2
O
17.4
Labeled: L1
PCR mixture, maxiprimer B
Volume in µl
Long PCR enzyme mix buffer
5
dNTPs
1
Template DNA, pARW089
1.2
r_mdnABCDE_iGEM
1
Maxiprimer B
21.8
Long PCR enzyme mix
0.3
H
2
O
19.7
Labeled: L2
Thermal cycling program
Step
Temperature in °C
Time in seconds
Hot start
94
Hold
Initial Denaturation
94
300
Denaturation
95
20
Annealing
54
30
Extension
68
22
Cycle number
10
Denaturation
94
20
Annealing
68
30
Extension
68
24
Cycle number
15
Final extension
68
600
Output:
2 samples (L1 and L2) in right Thermocycler
2 samples (P1 and P2) in left Thermocycler
Further tasks:
Gel electrophoresis and DNA extraction, expected size 431 bases
Maxiprimer PCR II using the extracted DNA as reverse primer
Gel electrophoresis and DNA extraction, expected size 6619 bases
Restriction enzyme digestion using EcoRI and SpeI
Ligation with pSB1C3
Transformation
100th Labday 2011-09-26
Repeated Phage Display with another trypsin protease
Investigator: Sandrina, Sabine
Time: 2011-09-16, 9:00-18:00
Aim:check if unmodified mdnA on the surface of the phages binds to an immobilized protease. interactions of microviridin L with this protease were already detected
Material/Method:
in ELISA plate:
coat wells with protease trypsin- EDTA in water for 2 h
blocking in 3% milk powder for 2 h
wash 6x with TBS-T (0,005%)
panning with phages 1:1 helper phages and phages of interest (presenting mdnA-geneIII fusion protein on their surface, 1 h)
wash 6x with TBS-T
elute bound phages with 0,2 M Glycin-HCl, 10 min
neutralize with 1 M Tris (pH 7-8)
mix eluted phages with preparatory culture of XL1-blue cells
store over night at 4°C
Further tasks:
check binding of phages by plating them
ELISA with purified phages
Investigators: Sabine, Sandrina
Time: 2011-09-16, 12:00-21:00
Aim: control if mdnA-myc-geneIII will be expressed on the phage
Method/Materials:
ELISA plate coated with 5 µg/ml anti-c-myc antibody in phosohate buffer (pH 7,4), volume 100 µl/ well
incubate for 4 h at room temperature
blocking in 2 % milk powder in TBS (300 µl/ well)
incubate for 2 h at room temperature
add 100 µl phages, produced in XL1-blu cells diluted 1:1 in TBS-T (0,005%)
incubate shaking for 60 min at room temperature
wash 5 x with TBS-T (0,05%)
add anti-gene-8-antibody (HRP coupled)
incubate shaking for 60 min
wash 5 x with TBS-T
put substrate on the samples
measure in plate photometer
reference: helper phages
Resluts: positive but weak weak signal at wells with mdnA phages because of the low concentration of phages
Control of Phage Display
Investigator: Sandrina, Sabine
Time: 2011-09-2
Aim:check if unmodified mdnA on the surface of the phages binds to an immobilized protease. interactions of microviridin L with this protease were already detected
Material/Method:
over night culture of XL1-blue cells was infected with eluted phages
1000 cells were infected with 10.000 phages and plated on agar with kanamycin and on agar with ampicillin
control: 1:1 mix of helperphages and produced phages of interest carrying mdnA-geneIII plated also on agar with kanamycin and on agar with ampicillin
Further tasks:
check cell growth
101th Labday 2011-09-27
SDS- PAGE for coomassie staining with purified phages carrying mdnA on their surface
Investigators: Sabine, Sandrina
Aim: preparing geneIII-myc-mdnA-protein for mass spectrometry
Method/Materials:
seperating gel:
acryl amide: 2,5 ml
seperating gel buffer 4x: 1,5 ml
water: 1,94
APS: 60 µl
stacking gel:
acryl amide: 0,28 ml
stacking gel buffer: 0,5 ml
water: 1,22 ml
APS: 10 µl
loading dye: RotiLoad1
incubate samples at 95°C for 20 min
run at 50 V for half an hour, then switch to 20 mA
Further tasks:
coomassie staining
Coomassie staining
Investigators: Sabine, Sandrina
Aim: control if geneIII-myc-mdnA could be enriched for mass spectrometry
Method/Materials:
20 ml water
20 ml methanol
60 ml coomassie stock solution
gel storage over night in coomassie solution shaking
Further tasks:
destaining of the gel
102th Labday 2011-10-06
Amplification of megaprimers for site-directed mutagenesis
Time: 2011-10-06, 9:00-12:00
Investigators: Nicole
Idea:Site-directed mutagenesis of XbaI restriction sites within the mdn cluster (use as biobrick)
Materials
Primer:
pf_muta_XbaI, 10 µM, 54°C (TATATTGAGGCATCAAGATCCAATTG)
pr_muta_XbaI, 10 µM, 55°C (GCTTTCATCTTCTTGATAAAAATGTTATCA)
pr_ohne_muta, 10 µM, 55°C (GCTTTCATCTTCTAGATAAAAATGTTATCA)
pARW089 clone 2, 8.57 ng/ µl
Genaxxopn Taq Polymerase S
10x Genaxxon Taq Polymerase Buffer (15 mM MgCl2)
25 mM MgCl2
dNTPs (10 mM)
Methods:
PCR mixture for A
A = mdn cluster without XbaI recognition sites at both sites
Volume in µl
dNTPs (10 mM)
1
10x Genaxxon Taq Polymerase Buffer
5
25 mM MgCl2
2
pf_muta_XbaI (10 µM)
1.5
pr_muta_XbaI
1.5
Template DNA, pARW089, 8.57 ng/ µl
1
Genaxxon Taq Polymerase S
0.5
H
2
O
37.5
Labeled: A1 and A2
PCR mixture for B
A = mdn cluster without XbaI recognition sites at one sites
Volume in µl
dNTPs (10 mM)
1
10x Genaxxon Taq Polymerase Buffer
5
25 mM MgCl2
2
pf_muta_XbaI (10 µM)
1.5
pr_ohne_muta
1.5
Template DNA, pARW089, 8.57 ng/ µl
1
Genaxxon Taq Polymerase S
0.5
H
2
O
37.5
Labeled: B1 and B2
Thermal cycling program
Program named MAXI1A
Step
Temperature in °C
Time in seconds
Hot start
94
Hold
Initial Denaturation
94
180
Denaturation
94
45
Annealing
51
45
Extension
72
45
Cycle number
25
Final extension
72
600
Output:
2 samples (A1, A2, B1, B2) in left thermocycler
Further tasks:
Gel electrophoresis and DNA extraction, expected size 127 bases
Megaprimer PCR using the extracted DNA as primer
DpnI digestion with the half of each PCR reaction
Transformation using XL1-Blue cells
Agarose gel electrophoresis and DNA extraction of megaprimers
Time: 2011-10-06, 12:00-14:00
Investigators: Nicole, Jessica
Idea:Purification of megaprimers (used for site-directed mutagenesis)
Materials
Agarose
Gel red
1x TAE Buffer
PCR products (A1, A2, B1, B2)
6x Loadig Dye (Promega)
100bp DNA ladder (Fermentas)
DNA ladder mix (Fermentas)
NucleoSpin Extract Kit II (Machery Nagel)
Methods:
Preparation of 2% Agarose gel
1 g Agarose in 50 ml TAE
2 µl Gel red
Loading of samples
8 µl 6x Loading Dye and 50 µl of each PCR product
5 µl 100 bp DNA ladder
8 µl DNA ladder mix
Running agarose gel
DNA extraction and purification
using NucleoSpin Extract II (Machery Nagel)
following maufacturer's protocol
A1 and A2 resp. B1 and B2 using one column
Elution buffer: H2O, volume: 35 µl, incubation 5 min at 70°C
Results:
Expected sizes and positions of samples
Position
Label
Expected size in bases
1
100bp DNA ladder
2
A1
127
3
A2
127
4
5
B1
127
6
B2
127
7
DNA ladder mix
Gel picture
File:UP
DNA concentrations
A: c = 18.9 ng/ µl
B: c = 23.0 ng/ µl
Output:
Megaprimer A (both mutations), stored PCR box
Megaprimer B (one mutation, stored PCR box
Further tasks:
PCR using these megaprimers
DpnI digestion
transformation
Megaprimer PCR for site directed mutagenesis
Time: 2011-10-06, 14:00-17:00
Investigators: Nicole, Jessica
Idea:Site-directed mutagenesis of XbaI restriction sites within the mdn cluster (use as biobrick)
Materials
Megaprimer (A and B), 2011-10-06, Jessica&Nicole
pARW089 clone 2, 85.7 ng/ µl
PfuUltra Polymerase (Agilent)
10x PfuUltra Polymerase Buffer
dNTPs (10 mM)
Methods:
PCR mixture
based on File:UP PfuUltra HF DNA Polymerase.pdf
A = mdn cluster without XbaI recognition sites at both sites
B = mdn cluster without XbaI recognition sites at one sites
dNTPs (10 mM)
1
10x PfuUltra Polymerase Buffer
5
Megaprimer
0.5 (approx. 3.91 ng needed)
25 mM MgCl2
2
Template DNA, pARW089, 85.7 ng/ µl
1.2
PfuUltra Polymerase
2
H
2
O
40.3
Thermal cycling program
Program named MAXI5
Step
Temperature in °C
Time in seconds
Hot start
94
Hold
Initial Denaturation
94
300
Denaturation
94
30
Annealing
60
60
Extension
68
20 min 50 sec
Cycle number
7
Denaturation
94
30
Annealing
60
60
Extension
68
20 min 50 sec + 10 sec per cycle
Cycle number
8
Output:
2 samples in left thermocycler, labeled 2A and 2B
Further tasks:
DpnI digestion with the half of each PCR reaction
Transformation using XL1-Blue cells
103th Labday 2011-10-07
DpnI digestion and Transformation of mutated pARW089
Time: 2011-10-07
Investigators: Nicole für Sebastian ;), done by Jessica
Aim:Site-directed mutagenesis of the mdn cluster (use as biobrick)
Materials:
DpnI (10 U/ µl)
PCR products, 2A and 2B, stored in freezer (or thermal cylcer)
LB medium
4 agar plates with ampicillin
Methods:
DpnI digestion
add 0.5 µl DpnI to 25 µl of the PCR product
0.5 µl DpnI
25 µl PCR product
2.8 µl NEB 4 buffer
mix gently and thoroughly by pipetting up and down or by vortexing
incubate 1 hour at 37°C
the other 25 µl of PCR product: undigested
Transformation
add 2 µl PCR product (one sample = DpnI digested, second sample = undigested) to competent cells
incubate 30 min one ice
heat pulse: 45 sec at 42°C
2 min on ice
add 750 µl LB medium to each sample
incubate 1 hour at 37°C and 850rpm
cenrifugation and resuspend in 500 µl LB medium
plate 250 µl on each of two plates with ampicillin or kanamycin antibiotic
incubate the transformation plates at 37°C for more than 16 hours
Output:
plates in incubator:
Sample a (cluster without any XbaI recognition site), DpnI digested
Sample a (cluster without any XbaI recognition site), undigested
Sample b (cluster with one XbaI recognition site), DpnI digested
Sample b (cluster with one XbaI recognition site), undigested
Further tasks:
preparation of overnight cultures of several clones, especially from DpnI digested samples
Miniprep
Retransformation of XL1blue-cells with pPDV089
Investigators: Sandrina
Aim: produce cells for phage display
Method/Materials:
addition of 0,3 µl ligated vector pPDV089 from clone 2S14 (after plasmid preperation) to competent XL1-blue cells
incubation 25 min on ice,
heat shock 45 sec at 42°C,
incubation 2 min on ice,
addition of 750 µl LB medium,
incubation 60 min at 37 °C and 750 rpm
plating on agar plates containing 100 µg/ml ampicillin
storage over night at 37°C
Further tasks:
over night culture, amplification of cells for phage display, test digestion
104th Labday 2011-10-11
Oligo-Fillin for Phage-display-mdnA-library (carrying AgeI restriction site)
Investigators: Nicole
Time: 2011-10-11, 9:00-14:00
Aim: Production of a library carrying mutated mdnA for phage-display (carrying ageI restriction site)
Material:
oligos (25 mM, HPLC purified, ordered by Sigma Aldrich):
o_mdnA_library, 76
o_focused_library_AgeI, 85
10x Klenow buffer
dNTPs (10 mM)
H
2
O
Machery Nagel NucleoSpin Extract II Kit
Method:
Reaction mix (total volume: 50 µl)
3 µl fw-oligonucleotide
3 µl rev-oligonucleotide
5 µl dNTPs
5 µl Klenow buffer
34 µl H
2
O
Reaction conditions
Program: ORIGAMI1
after finishing program ORIGAMI1:
addition of 1 µl Klenow-fragment
incubation 1 hour, 37°C
PCR purification using Machery Nagel NucleoSpin Extract II Kit, following manufacturer's protocol
Results:
purified hybridized oligos for production of mdnA-library
Output:
hybridization product: purified AgeI-library, named AgeI 2
stored in box: library, freezer
Further tasks:
restriction enzyme digestion and ligation
Troubleshooting: Site-directed mutagenesis of pARW089 (for use as biobrick)
Investigators: Nicole
Time: 2011-10-11, 11:00
Aim: Site-directed mutagenesis of XbaI recognition site within the mdn-cluster
Ideas:
Based on QuickChange II Site-Directed mutagenesis Kit, Instruction manual
2nd Transformation: increased DpnI treated DNA -> 4 µl
Agarose gel --> at least 80% of DNA supercoiled
Ethanol precipitation of DpnI digested PCR product, resuspension in decreased water volume and transformation
2nd PCR using LongPCR enzyme mix (Fermentas)
Further tasks:
see ideas
Megaprimer PCR for site directed mutagenesis
Time: 2011-10-06, 11:30-13:00
Investigators: Nicole, Jessica
Idea:Site-directed mutagenesis of XbaI restriction sites within the mdn cluster (use as biobrick)
Materials
Megaprimer (A and B), 2011-10-06, Jessica&Nicole
pARW089 clone 2, 85.7 ng/ µl
LongPCR enzyme mix (Fermentas)
10x Long PCR enzyme mix buffer (15 mM MgCl2)
dNTPs (10 mM)
Methods:
PCR mixture
based on [[File: ]]
A = mdn cluster without XbaI recognition sites at both sites
B = mdn cluster without XbaI recognition sites at one sites
dNTPs (10 mM)
1
10x LOng PCR enzyme mix (Fermentas)
5
Megaprimer
0.5 (approx. 50 pmol)
Template DNA, pARW089, 85.7 ng/ µl
1.2
PfuUltra Polymerase
0.3
H
2
O
42
Thermal cycling program
Program named MAXI5
Step
Temperature in °C
Time in seconds
Hot start
94
Hold
Initial Denaturation
94
300
Denaturation
94
20
Annealing
60
60
Extension
68
8 min 50 sec
Cycle number
7
Denaturation
94
20
Annealing
60
60
Extension
68
8 min 50 sec + 5 sec per cycle
Cycle number
8
Output:
2 samples in left thermocycler, labeled A and B
Further tasks:
DpnI digestion with the half of each PCR reaction
Transformation using XL1-Blue cells
Digestion of ligation product geneIII into pARW089 (pARWIII) for ligation with generated mdnA library
Investigator: Sandrina
Aim: get digested pARW089 (without mdnA) and geneIII for ligation with generated mdnA library
Material/Method:
30 µl/160 µg/ml pARWIII
1 µl restriction enzyme SfoI
1 µl restriction enzyme AatII
4 µl NEB 10x buffer 4
2 µl water
3 h, 37°C
Further Task:
purification
ligation
DpnI digestion and Transformation of mutated pARW089
Time: 2011-10-11
Investigators: Jessica, Nicole
Aim:Site-directed mutagenesis of the mdn cluster (use as biobrick)
Materials:
DpnI (10 U/ µl)
PCR products, A and B
LB medium
4 agar plates with ampicillin
Methods:
DpnI digestion
add 0.5 µl DpnI to 25 µl of the PCR product
0.5 µl DpnI
25 µl PCR product
mix gently and thoroughly by pipetting up and down or by vortexing
incubate 1 hour at 37°C
the other 25 µl of PCR product: undigested
Transformation
add 2 µl PCR product (one sample = DpnI digested, second sample = undigested) to competent cells
incubate 30 min one ice
heat pulse: 45 sec at 42°C
2 min on ice
add 750 µl LB medium to each sample
incubate 1 hour at 37°C and 850rpm
cenrifugation and resuspend in 500 µl LB medium
plate 250 µl on each of two plates with ampicillin or kanamycin antibiotic
incubate the transformation plates at 37°C for more than 16 hours
Output:
plates in incubator:
Sample a (cluster without any XbaI recognition site), DpnI digested
Sample a (cluster without any XbaI recognition site), undigested
Sample b (cluster with one XbaI recognition site), DpnI digested
Sample b (cluster with one XbaI recognition site), undigested
Further tasks:
preparation of overnight cultures of several clones, especially from DpnI digested samples
Miniprep
Over night culture of retransformed pPDV089
Investigator: Sandrina
Aim: production of phages carrying mdnA on their surface
Material/Method:
LB medium with ampicillin (1:1000)
incubate shaking over night at 37°C
Further Task:
production of phages
Digestion of geneIII for ligation with generated mdnA library
Investigator: Sandrina
Aim: get digested geneIII for ligation with generated mdnA library
Material/Method:
50 µl/10 µg/ml geneIII
1 µl restriction enzyme NgoMIV
6 µl NEB 10x buffer 1
3 µl water
1 h, 37°C
Further Task:
ligation
Transformation of mutated pARW089 with increased amount
Time: 2011-10-11, 17:00-20:00
Investigators: Jessica, Nicole
Aim:Site-directed mutagenesis of the mdn cluster (use as biobrick)
Materials:
PCR products, 2A and 2B
LB medium
2 agar plates with ampicillin
Methods:
add 2 µl PCR product (one sample = DpnI digested, second sample = undigested) to competent cells
incubate 30 min one ice
heat pulse: 45 sec at 42°C
2 min on ice
add 750 µl LB medium to each sample
incubate 1 hour at 37°C and 850rpm
cenrifugation and resuspend in 500 µl LB medium
plate 250 µl on each of two plates with ampicillin or kanamycin antibiotic
incubate the transformation plates at 37°C for more than 16 hours
Output:
plates in incubator:
Sample 2A (cluster without any XbaI recognition site), DpnI digested
Sample 2B (cluster with one XbaI recognition site), DpnI digested
Further tasks:
preparation of overnight cultures of several clones, especially from DpnI digested samples
Miniprep
Digestion of mdnA library (for phage display) using AgeI
Investigator: Jessica, Nicole
Time: 2011-10-11, 17:00-18:00
Aim: Production of pUPIII (carrying mdnA-library and gene III) for screening by phage display
Materials:
NEB Buffer 1
AgeI (NEB)
purified oligos, lib AgeI, Nic, 2011-10-11
Methods:
25 µl purified PCR product (lib AgeI)
3 µl NEB Buffer 1
2 µl AgeI
Result/ Output:
digested library for phage display, digested lib AgeI, stored freezer (box library)
Further Task:
purification of digested lib AgeI
tripple ligation with gene III and pARWIII
transformation
Purification of digested mdnA library and digested gene III
Investigator: Sandrina, Jessica, Nicole
Time: 2011-10-11, 18:00-19:00
Aim: Production of pUPIII (carrying mdnA-library and gene III) for screening by phage display
Materials:
Machery Nagel Nucleo Spin Extract II Kit
Methods:
following Machery Nagel Nucleo Spin Extract II Kit
Exceptions:
Elution buffer: H2O
Elution volume: 25 µl
Incubation 5 min, 70°C (before 1 min centrifugation)
Results:
concentrations:
mdnA library: c = 174 ng/ µl
gene III: c = 20.7 ng/ µl
Output:
purified and digested
gene III
mdnA library
Further Task:
tripple ligation with gene III and pARWIII
transformation
Triple Ligation with pARWIII, mdnA-library and gene III
Investigator: Sandrina, Jessica, Nicole
Time: 2011-10-11, 19:00-20:00
Aim: Production of pUPIII (carrying mdnA-library and gene III) for screening by phage display
Materials:
T4 DNA ligase buffer (Fermentas)
T4 DNA ligase
purified and digested gene III, 2011-10-11, Nic
purified and digested mdnA library, 2011-10-11, Nic
purified and digested pARWIII, 2011-10-11, San
Methods:
Concentrations and size of inserts and backbone
Concentration in ng/ µl
Size in bases
gene III
20.7
541
mdnA library
174
161
pARWIII
34.8
9989
Ligation mix
2 µl T4 DNA ligase buffer
1 µl T4 DNA ligase
13 µl pARWIII
3.5 µl gene III
1 µl mdnA library
for transformation control: instead of geneIII and mdnA library 4.5 µl H2O
Results/ OUtput:
ligation product, named geneIII mdnA lib pARWIII, 2011-10-11, stored in freezer (libary box)
transformation control, named H2O, 2011-10-11, stored in freezer (library box)
Further Task:
transformation
Transformation of triple ligated mdnA library, gene III and pARWIII
Time: 2011-10-11, 19:45-22:00
Investigators: Sandrina, Jessica, Nicole
Aim:Site-directed mutagenesis of the mdn cluster (use as biobrick)
Materials:
XL1-Blue chemocompetent cells
ligation product, named geneIII mdnA lib pARWIII, 2011-10-11
transformation control, named H2O, 2011-10-11
add 2 µl PCR product to competent cells (for ligation product 5 samples overall)
incubate 25 min one ice
heat pulse: 45 sec at 42°C
2 min on ice
add 750 µl LB medium to each sample
incubate 45 min at 37°C and 850rpm
cenrifugation and resuspend in 200 µl LB medium
plate 200 µl on plates with ampicillin
incubate the transformation plates at 37°C for approx. 16 hours
Output:
plates in incubator:
pUPIII 1-5, 2011-10-11, San&Nic
transformation control, 2011-10-11, San&Nic
Further tasks:
preparation of overnight cultures of several clones
Miniprep
Sidedirected mutagenesis of HRV14 3C protease
For better understanding of described experiment see also: [ [1] ]
Investigators: Sascha, Paul, Sebastian
Aim: Sidedirected mutagenesis of TEV and PreSciccion protease
Materials:
HRV14 3C protease vector (P10 by Gunther Stier)
different Primers
Phusion polymerase kit by NEB
Methode:
PCR
Template: 1 µL
Nucleotides: 1 µL of 10 mM ready to use dNTP mix
10 µL 5 x Phusion GC Buffer
2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)
32,5 µL of pure water
0,5 µL Phusion HF polymerase
Program:
Denat: 30 sec 98°C
30x
Denat: 10 sec 98°C
Annealing+Extend: 25 sec 72°C
Final Extend: 10min 72°C
Used Primers:
Fragment 1 of HRV14 3C protease (P1):
f_HRV14 3C_AraFusion_NgoMIV
r_HRV14 3C_ACCAGC
Fragment 2 of HRV14 3Cprotease (P2):
f_HRV14 3C_ACCAGC
r_HRV14 3C_tm_Xba208_A-T
Fragment 3 of HRV14 3Cprotease (P3):
f_HRV14 3C_tm_Xba208_A-T
r_HRV14 3C_tm_Xba280_A-T
Fragment 4 of HRV14 3Cprotease (P4):
f_HRV14 3C_tm_Xba280_A-T
r_HRV14 3C_iGEM_BamHI
Further Tasks:
PCR purification, analytical Gel to check sizes of the amplified fragments, assembly PCR of Fragment
Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments P1, P2,P3 and P4 of HRV14 3C
For better understanding of described experiment see also: [ [2] ]
Investigators: Sascha, Paul, Sebastian
Aim: Check the size of amplified fragments and clean them up after PCR
Materials:
PCR fragments P1-P4
Fermentas 6x loading dye
100bp ladder (Gene Ruler by Fermentas)
Macherey-Nagel Nucleo Spin II Kit for clean up
Methode:
Clean up after manual for PCR purification of the used Kit
gel electophoresis (1.5 % agarose gel with analytical gel pockets): 30 min at 110 V
Results:
The sizes of the fragments of the PCR products are matching the fragment sizes extracted with Geneious.
Further Tasks: Merge of PCR fragments P5 and P6 into PreSciccion protease, digestion of TEV protease with NgoMIV and BamHI, Ligation of TEV protease with digested AraC induction system (NgoMIV and HindIII) and digested pJC354_ssTorA_XhoI_CS-TEV_NheI_BlaFL (HindIII and BamHI)
Assembly PCR of mutated HRV14 3C protease fragments
For better understanding of described experiment see also: [ [3] ]
Investigators: Sascha, Stefan, Sebastian
Aim: Assembly of mutated TEV and PreSciccion protease fragments
Materials:
HRV14 3C protease fragments P1 and P2 --> P5
HRV14 3C protease fragments P3 and P4 --> P6
different Primers
Phusion polymerase kit by NEB
Methode:
PCR
Template: 2 µL (1µl of each fragment after sizes were checked and fragments got cleaned up with PCR clean up kit from Macherey-Nagel (Nucleo Spin Kit))
Nucleotides: 1 µL of 10 mM ready to use dNTP mix
10 µL 5 x Phusion HF Buffer
2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)
31,5 µL of pure water
0,5 µL Phusion HF polymerase
Program:
Denat: 30 sec 98°C
30x
Denat: 10 sec 98°C
Annealing+Extend: 35 sec 72°C
Final Extend: 10min 72°C
Used Primers:
Assembly of fragment 1 and fragment 2 of HRV14 3C protease (P1+P2)--> mutated HRV14 3C protease fragment 5 (P5):
f_HRV14 3C_AraFusion_NgoMIV
r_HRV14 3C_tm_Xba208_A-T
Assembly of fragment 3 and fragment 4 of HRV14 3C protease (P3+P4)--> mutated HRV14 3C protease fragment 6 (P6):
f_HRV14 3C_tm_Xba208_A-T
r_HRV14 3C_iGEM_BamHI
Further Tasks:
PCR purification, analytical Gel to check sizes of the amplified fragments, assembly PCR of Fragment
Analytical gelelctrophoresis and PCR clean up of sidedirected mutated fragments P5 and P6 of HRV14 3C
For better understanding of described experiment see also: [ [4] ]
Investigators: Sascha, Paul, Sebastian
Aim: Check the size of amplified fragments and clean them up after PCR
Materials:
PCR fragments P5 and P6
Fermentas 6x loading dye
100bp ladder (Gene Ruler by Fermentas)
Macherey-Nagel Nucleo Spin II Kit for clean up
Methode:
Clean up after manual for PCR purification of the used Kit
gel electophoresis (1.5 % agarose gel with analytical gel pockets): 30 min at 110 V
Results:
The sizes of the fragments of the PCR products are matching the fragment sizes extracted with Geneious.
Further Tasks: Merge of PCR fragments P5 and P6 into PreSciccion protease, digestion of TEV protease with NgoMIV and BamHI, Ligation of TEV protease with digested AraC induction system (NgoMIV and HindIII) and digested pJC354_ssTorA_XhoI_CS-TEV_NheI_BlaFL (HindIII and BamHI)
Assembly PCR of mutated HRV14 3C protease fragments P5 and P6
For better understanding of described experiment see also: [ [5] ]
Investigators: Sascha, Stefan, Sebastian
Aim: Assembly of mutated TEV and PreSciccion protease fragments
Materials:
PreSciccion protease fragment P5
PreSciccion protease fragment P6
different Primers
Phusion polymerase kit by NEB
Methode:
PCR
Template: 2 µL (1µl of each fragment after sizes were checked and fragments got cleaned up with PCR clean up kit from Macherey-Nagel (Nucleo Spin Kit))
Nucleotides: 1 µL of 10 mM ready to use dNTP mix
10 µL 5 x Phusion HF Buffer
2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)
31,5 µL of pure water
0,5 µL Phusion HF polymerase
Program:
Denat: 30 sec 98°C
30x
Denat: 10 sec 98°C
Anneal+Extend: 35 sec 72°C
Final Extend: 5 min 72°C
Used Primers:
Assembly of fragment 5 and fragment 6 of HRV14 3C protease (P3+P4)--> mutated PreSciccion portease fragment 6 (P6):
f_HRV14 3C_AraFusion_NgoMIV
r_HRV14 3C_iGEM_BamHI
Further Tasks:
PCR purification, analytical Gel to check sizes of the amplified fragments, digest and ligation with other fragments
105th Labday 2011-10-12
Production of phages containing pPDV089 in XL1-blue cells
Investigators:Sandrina
Aim: production of phages carrying unmodified mdnA on their surface
Method/Materials:
first step: amplification of cells containing pPDV089 (clone: 2S14):
500 ml LB medium will be inoculated with the cells, so that OD600 = 0.1
add antibiotics tetracyclin and ampicillin to the medium
cells should be incubated 37 °C till OD 600 = 0.3-0.5 (here: 0.356) is reached
second step: infection with helper phages
add helper phages 10^11 phages/50 ml (3,5 µl)
incubate for 10 min at 37°C (without shaking!)
add 0,5 mM IPTG
incubate 50 min at 28°C and rpm
add 70 µg/ml kanamycin and incubate for 5 h at 28°C (...)
third step: phage purification
centrifuge cell culture at 5000 x g/ 15 min
fill supernatant in a new 50 ml falcon and centrifuge again (5000 g/15 min)
40 ml of the supernatant with 8 ml PEG-NaCl (20% (w/v) PEG-8000, 2,5 M NaCl)
incubate over night at 4°C
Further tasks:
go on with phage purification
Production of microviridin
Time: 2011-10-12, 09:30-19:30
Investigators: Sandrina, Jessica, Nicole
Aim: Culturing cells to produce microviridin
Materials:
LB medium
overnight cultures of pARW089
500mM Tris-HCL
Method:
17 ml overnight culture + 374 ml LB medium
incubation for 7.5 h
one @ 30°C
one @ 37°C
centrifuged @ 4000g for 15 min
washed with 500mM Tris-HCL
centrifuged
supernatant discarded
pellet stored @ -20°C
Output:
four 50 ml falcons (2x 30°C, 2x 37°C)
Further tasks:
preparing samples for HPLC
Digest of HRV14 3C, AraC and Vector SG11
Investigator: Paul Stefan Sebastian
Material:
Enzymes: (all pruchased from NEB)
EcoRI-HF
PstI-HF
NgoMIV
DNA-Fragments:
mutated HRV14 3Cprotease
AraC
SG11
NEBuffer 4
Methode:
All digestion batches got a total volume of 10µl and were incubated for 2 h at 37°C:
1µl Buffer
1µl each Enzyme
0,1 µl BSA if required
7 or 6,9 µl DNA
HRV14 3C got digested with PstI-HF+NgoMIV, AraC was digested with NgoMIV+EcoRI-HF. Vector SG11 which containing the cleavage site for the HRV14 3C got digested with EcoRI-HF+PstI-HF.
Output:
Digested fragements:
NgoMIV_HRV14 3C_PstI
EcoRI_AraC_NgoMIV
PstI_SG11_EcoRI
Further Tasks:
Clean up of all fragments with PCR CleanUp Kit by M&N, ligation of fragments and transformation into E.coli XL1 blue.
Ligation of digested fragments of HRV14_3C protease and vector SG11 and Transformation of XL1 blue with Ligation products
Investigators: Stefan, Sascha, Paul
Materials:
Fragments:
NgoMIV_PreSciccion_PstI
EcoRI_AraC_NgoMIV
PstI_SG11_EcoRI
QuickLigase purchased by Fermentas
QuickLigase 10x Buffer purchased by Fermentas
Methodes:
Standard ligation protocol from Fermantas for QuickLigase Kit. Ratio between vector and both fragments (protease and AraC-induction system) where 3:3:1.
Batch was incubated for 10 min at 37°C. After 10 min, all batches where put on ice.
Output:
ligation batch
Next Step:
Transformation of XL1 blue E.coli with ligation batch using the standard protocol for heat shock transformation. Transformed cells were platted on agar plates containing cm as antibiotic.
Further tasks:
colony PCR to get to know, which clones are positive and for all positive clones: mini-prep of plasmid DNA, sequencing and preparation of glycerol stock clutures.
106th Labday 2011-10-13
Preparation of samples for HPLC analysis
Time: 2011-10-13
Investigators: Jessica, Nicole
Material:
pellets of pARW089 samples (from 2011-10-13)
100% methanol
5% methanol
sterile water
Sep-Pak Cartridges
speed vac
Method:
pellet was resuspended in 5 ml of sterile water
the resuspended sample was sonicated for 4 min (2 sec on, 2 sec off)
centrifugation for 15 min @ 4000 rpm in eppendorf Centrifuge 5810 R
supernatant was transfered to Sep-Pak Plus C18 Cartridges, that were equilibrated with 2 ml of 100% methanol
cartridges were washed with 2 ml water
samples were loaded
cartridges were washed with 2 ml of 5% methanol
samples were eluted with 2 ml of 100% methanol
samples were put in a speedvac so that the methanol can evaporate
Transformation of mutated pARW089 using super competent XL1 gold cells
Time: 2011-10-13, 16:30-19:00
Investigators: Jessica, Nicole
Aim: Site-directed mutagenesis of pARW089 (carrying XbaI recognition sites) for use as biobrick
Material:
super competent XL1 gold cells (tet and cm resistance), given by Site directed mutagenesis lightening kit (stratagene)
PCR product of site-directed mutagenesis, DpnI digested, Nic&Jes 2011-10-07
ß-Mercaptoethanol
SOC medium
Agarplate containing Amp
Method:
Followed by manufacturer's protocol File:UP
use 45 µl cells (thawn at ice)
add 2 µl ß-mercaptoethanol
incubate 2 min on ice
add 2 µl DpnI-treated sample (mixing)
incubate 30 min on ice
heat impulse: 42°C 30 sec
incubate 2 min on ice
add 500 µl SOC medium
incubate 1 hour at 37°C and 250 rpm
plate 500 µl on one agarplate (Amp)
incubate over night at 37°C
Results/ Output:
agar plate with mutated pARW089, named DpnI dig. pARW089 2011-10-13
stored in 37°C incubator
Further tasks:
Picking over night cultures
miniprep
Purification of phages containing pPDV089
Investigators: Sabine, Sandrina
Aim: phages carrying unmodified mdnA on their surface
Method/Materials:
after over night incubation:
centrifuge precipitaed phages: 5000 x g/ 45 min
discard supernatant
centrifuge again, 5000 x g/ 5 min
remove supernatant carefully
take pellet in 1 ml TBS
move in 1.5 ml Eppi
centrifuge again: 17000 x g/10 min
supernatant in a new Eppi with 200 µl PEG-NaCl (mix!)
incubate on ice for 60 min
centrifuge precipitated phages: 17.000 x g/ 10 min (4°C)
resuspend pellet in 300 µl TBS
centrifuge:17 000 x g/ 10 min (4°C)
supernatant in a fresh eppi= purified phages
Further tasks:
ELISA and phage display
Colony PCR of plated HRV14_3C containing colonies
Investigators: Stefan, Paul, Sebastian
Aim:
Checking for positive clones
Used Primer:
f_HRV14_3C_iGEM, r_HRV14_3C_iGem:BamHI
Method:
2,5µl of each primer (10µM) = 5µl
5µl 10y polymerase buffer
2µl 25mM MgCl2
1µl dNTP mix
0,5µl Taq-Polymerase (GenAxxon)
36,5 µl H2O
=50µl
Colonies were picked with a 20µl tip, dipped into PCR batch, and then plunged into 1ml LB to grew cells from picked colony (from these 1ml batches overnight cultures will be inoculated in case of positive clones).
20 colonies were picked HRV14 3C protease (see entry from 01.09.2011) = 20 PCR batches.
PCR Program:
Initial denat = 3min 94°C
25x
denat: 2min 10sec 94°C
anneal: 2min 10sec 70°C
extend: 1min 72°C
final extend: 10min 72°C
The PCR products were resolved on a 1% analytical agarose gel
Results:
all clones were positive, 2 were selected for further studies.
Further Tasks:
overnight culture, mini prep, test digest, sequencing of clones and survival screening
107th Labday 2011-10-14
Digest of pARWIII and ligation with library
Time: 2011-10-14
Investigators: Sandrina, Jessica, Nicole
Material:
Method:
Digest of pARWIII
Gel electrophoresis
Gel purification
Digest of geneIII
Purification
Ligation
2 µl T4 ligase buffer
1 µl T4 ligase
6 µl pARW089
10 µl geneIII
1 µl mdnA library
Output:
3 ligation products stored @ -20°C
Further task:
transformation of one ligation product and control
108th Labday 2011-10-15
Transformation with library for phage display
Time: 2011-10-15
Investigators: Sabine, Jessica
Material:
Method:
Output:
Further task:
Miniprep, Glycerol stock cultures and Survival-Test of HRV14 3C clone(SG24-P10)
Investigators: Paul, Stefan, Sascha
Aim: Getting Plasmid DNA, establish a glycerol stock culture and test whether TorA-detection device is working
Mateials::
Mini prep Kit purchased by M&N
steril Glycerol
different Agarplates
overnight culture of SG24
Plates:
1x Cm (25 µg/ml)
1x Cm + 1mM IPTG
1x Cm + 2% Ara
1x Cm + 2% Ara +1mM IPTG
1x Cm + Amp (100 µg/ml)
1x Cm + 1mM ITPG + Amp (25 µg/ml)
1x Cm + 1mM ITPG + Amp (50 µg/ml)
1x Cm + 1mM ITPG + Amp (75 µg/ml)
1x Cm + 1mM ITPG + Amp (100 µg/ml)
1x Cm + 1mM ITPG + Amp (150 µg/ml)
1x Cm + 1mM ITPG + Amp (200 µg/ml)
1x Cm + 1mM ITPG + Amp (300 µg/ml)
1x Cm + 1mM ITPG + Amp (400 µg/ml)
1x Cm + 1mM ITPG + Amp (600 µg/ml)
1x Cm + 1mM ITPG + Amp (800 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (25 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (50 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (75 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (100 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (150 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (200 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (300 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (400 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (600 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (800 µg/ml)
Method:
for glycerol stock, cells were mixed in a 1:1 ratio with steril glycerol
plasmid isolation based on the manual of M&N
for survival screening 50µl of cell dilution with OD=0.002 were plated on prepared plates
Results:
No Cell death visible, repeat with different Temperature!
Further Tasks:
repeat survival test at 30 °C
109th Labday 2011-10-16
Overnight culture of library
Time: 2011-10-16
Investigators: Jessica
Material:
Method:
Output:
Further task:
Survival-Test of one HRV14_3C clone( SG24-P10)
Investigators: Paul, Stefan, Sascha
Aim: Test whether TorA-detection device functions
Mateial
different agaplates, overnight culture of SG24
Plates:
1x Cm (25 µg/ml)
1x Cm + 1mM IPTG
1x Cm + 2% Ara
1x Cm + 2% Ara +1mM IPTG
1x Cm + Amp (100 µg/ml)
1x Cm + 1mM ITPG + Amp (25 µg/ml)
1x Cm + 1mM ITPG + Amp (50 µg/ml)
1x Cm + 1mM ITPG + Amp (75 µg/ml)
1x Cm + 1mM ITPG + Amp (100 µg/ml)
1x Cm + 1mM ITPG + Amp (150 µg/ml)
1x Cm + 1mM ITPG + Amp (200 µg/ml)
1x Cm + 1mM ITPG + Amp (300 µg/ml)
1x Cm + 1mM ITPG + Amp (400 µg/ml)
1x Cm + 1mM ITPG + Amp (600 µg/ml)
1x Cm + 1mM ITPG + Amp (800 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (25 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (50 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (75 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (100 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (150 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (200 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (300 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (400 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (600 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (800 µg/ml)
Method:
for survival screening 50µl of cell dilution with OD=0.002 were plated on prepared plates and incubated for 2 days at 30°c
Results:
No Cell death visible!
Further Tasks:
thinking in progress...
110th Labday 2011-10-17
Amplification of megaprimer 1 (site-directed mutagenesis)
Time: 2011-10-17
Investigators: Nadja, Jessica
Aim: Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used
Material:
primers 86, 87, 88
pARW089, 8.57 ng/ µl
Method:
Samples and used primers
a) pARW089 with both mutations: pf_muta_XbaI (86), pr_muta_XbaI (87)
b) pARW089 with one mutation: pf_muta_XbaI (86), pr_ohne_muta (88)
PCR mixture
Volume in µl
dNTPs (10 mM)
1
10x Genaxxon buffer S
5
25 mM MgCL2
2
Primer forward (10 µM)
1.5
Primer reverse (10 µM)
1.5
DNA pARW089 (8.57 ng/ul)
1
Genaxxon Taq Polymerase S
0.5
H2O
37.5
PCR conditions (Thermal cycler)
Programm IGMI1
Time
Temp.
Hold
94°C
Initial denaturation
3 min
94°C
Denaturation
45 sec
94°C
Annealing
45 sec
51°C
Extension
45 sec
72°C
Number of cycles
30 x
Final extension
10 min
72°C
Output:
PCR products, A and B
Further task:
Agarose gel ellectrophoresis and purification
Megaprimer PCR 2 (with this purified primers)
Megaprimer PCR II (amplification of 2nd megaprimer (2nd part of mdn cluster, site-directed mutagenesis)
Time: 2011-10-18
Investigators: Nadja, Jessica, Nicole
Aim: Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used
Material:
primers: megaprimer A and B, Jes&Nad 2011-10-17, r_mdn_iGEM
pARW089, 8.57 ng/ µl
LongPCR enzyme mix + buffer (fermentas)
dNTPs (10 mM)
Methods:
Samples and used primers
A) pARW089 with both mutations: Megaprimer 1 - A and pr_mdn_iGEM
B) pARW089 with one mutation: Megaprimer 1 - B and pr_mdn_iGEM
PCR mixture
Volume in µl
dNTPs (10 mM)
1
Long PCR enzyme mix
5
Primer forward, megaprimer (32 ng/ µl)(
20
Primer reverse (10 µM)
1
DNA pARW089 (8.57 ng/ul)
5
Long PCR enzyme mix
0.3
H2O
17.7
PCR conditions (Thermal cycler)
Programm IGMAXI3
Time
Temp.
Hold
94°C
Initial denaturation
5 min
94°C
Denaturation
30 sec
94°C
Annealing
30 sec
55°C
Extension
30 sec
68°C
Number of cycles
15 x
Final extension
10 min
68°C
Output:
PCR products, A and B
Further tasks:
Gel electrophoresis and purification
megaprimer PCR 3
Sequencing of mdnA library (for phage display)
Time: 2011-10-18
Investigators: Sandrina, Nicole
Aim: Producation of mdnA library (for phage display), confirmation of diversity
Material:
Library clone 1 and 5, San, 2011-10-18
sf_mdnA_1
Methods:
Concentration of DNA and used DNA amount
DNA concentration in ng/ µl
Volume DNA in µl
Volume H2O in µl
Library clone 1
60.8
20
0
Library clone 5
98.5
14
6
Primer used: sf_mdnA_1
Output:
samples sent for sequencing
Further tasks:
Analysis of sequencing
Over night culture with created mdnA library for phage display
Investigators: Sandrina
Aim:Phage display with microviridin library
Method/Materials:
LB medium
ampicillin and tetracyclin
Further tasks:
phage production and phage display
111th Labday 2011-10-18
Production of phages containing microviridin library on their surface in XL1-blue cells
Investigators:Sandrina, Sabine
Aim: production of phages carrying modified mdnA on their surfac for screening by phage display
Method/Materials:
first step: amplification of cells containing mutated mdnA:
500 ml LB medium will be inoculated with the cells, so that OD600 = 0.1
add antibiotics tetracyclin and ampicillin to the medium
cells should be incubated 37 °C till OD 600 = 0.3-0.5 (here: 0.356) is reached
second step: infection with helper phages
add helper phages 10^11 phages/50 ml (3,5 µl)
incubate for 10 min at 37°C (without shaking!)
add 0,5 mM IPTG
incubate 50 min at 28°C and rpm
add 70 µg/ml kanamycin and incubate for 5 h at 28°C (...)
third step: phage purification
centrifuge cell culture at 5000 x g/ 15 min
fill supernatant in a new 50 ml falcon and centrifuge again (5000 g/15 min)
40 ml of the supernatant with 8 ml PEG-NaCl (20% (w/v) PEG-8000, 2,5 M NaCl)
incubate over night at 4°C
Further tasks:
go on with phage purification
Sidedirected mutagenesis of TEV protease
For better understanding of described experiment see also: [ [6] ]
Investigators: Sascha, Paul, Sebastian
Aim: Sidedirected mutagenesis of TEV protease
Materials:
TEV protease vector (P9 by Gunther Stier)
different Primers
Phusion polymerase kit by NEB
Methode:
PCR
Template: 1 µL
Nucleotides: 1 µL of 10 mM ready to use dNTP mix
10 µL 5 x Phusion GC Buffer
2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)
32,5 µL of pure water
0,5 µL Phusion HF polymerase
Program:
Denat: 30 sec 98°C
30x
Denat: 10 sec 98°C
Anneal+Extend: 25 sec 72°C
Final Extend: 5min 72°C
Used Primers:
Fragment 1 of TEV protease (T1):
f_TEV_AraFusion_NgoMIV
r_TEV_ACCAGC
Fragment 2 of TEV protease (T2):
f_TEV_ACCAGC
r_TEV_iGEM_BamHI
Further Tasks:
PCR purification, analytical Gel to check sizes of the amplified fragments, assembly PCR of Fragment
PCR purification and analytical GE
For better understanding of described experiment see also: [ [7] ]
Investigators: Sascha, Paul, Sebastian
Aim: Clean up PCR Fragments T1 and T2 and check sizes of fragments
Materials:
M&N PCR puri Kit
Methode:
Kit was used after manual of M&N
Sizes of fragments were checked in 1% agarose gel, 5µl were mixed with 1µl 6xLoading Dye.
Output
Fragments with the right size and clean fractions of T1 and T2
Further Tasks:
Assembly PCR, Digest of Fragments, Ligation and Transformation
Assembly PCR of TEV protease
For better understanding of described experiment see also: [ [8] ]
Investigators: Sascha, Paul, Sebastian
Aim: Sidedirected mutagenesis of TEV protease
Materials:
Fragments T1 and T2
different Primers
Phusion polymerase kit by NEB
Methode:
PCR
Template: 2µl mix of T1 and T2 in a molar ratio of 1:1
Nucleotides: 1 µL of 10 mM ready to use dNTP mix
10 µL 5 x Phusion GC Buffer
2,5 µL primers = 25 pmol absolute (2,5 µL of each primer, see list below)
31,5 µL of pure water
0,5 µL Phusion HF polymerase
Program:
Denat: 30 sec 98°C
30x
Denat: 10 sec 98°C
Anneal+Extend: 35 sec 72°C
Final Extend: 5min 72°C
Used Primers:
f_TEV_AraFusion_NgoMIV
r_TEV_iGEM_BamHI
Further Tasks:
PCR purification, analytical Gel to check sizes of the amplified fragments, assembly PCR of Fragment
PCR Purification, analytical GE, Digest and Ligation of TEV protease
For better understanding of described experiment see also: [ [9] ]
Investigators: Sascha, Paul, Sebastian
Aim: Getting the TEV into its test vector
Materials:
mutated TEV protease
T4 Ligase, different Restriction Enzymes
Methode:
5µl of the PCR product of mutated TEV protease were checked with anayltical GE on a 1% agarosegel
PCR products got cleaned up by use of the M&N kit for PCR purification
Mutated TEV and Vector SG6 containing the cleavage site for TEV protease and AraC were digested with different enzymes and cleaned up again.
TEV with PstI and NgoMIV
AraC with EcoRI and NgoMIV
SG6 with PstI and EcoRI
the concentration of the fragments were estimated with NanoDrop
Ligation of all Fragment with T4 Ligase by Fermentas (see manual of T$ ligase by Fermentas)
Ligation Batch is used for Transformation of XL1 blue
transformed cells were platted on agarplates containing cm
Further Tasks:
colony PCR, plasmid prep survival screening and sequencing
112th Labday 2011-10-19
Amplification of megaprimer I, using Phusion (Finnzymes)
Time: 2011-10-19
Investigators: Jessica, Nicole
Aim: Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used
Material:
primers:
pf_muta_XbaI
pr_muta_XbaI
pr_ohne_muta
pARW089, 8.57 ng/ µl
Phusion HF Buffer (Finnzymes)
dNTPs (10 mM)
Methods:
Samples and used primers
A. pARW089 mutation: pf_muta_XbaI and pr_muta_XbaI
B. pARW089 without mutations: pf_muta_XbaI and pr_ohne_muta
PCR mixture
Volume in µl
dNTPs (10 mM)
1
Phusion HF Buffer
10
Primer forward, megaprimer (10 µM)
2.5
Primer reverse (10 µM)
2.5
DNA pARW089 (8.57 ng/ul)
1
Phusion polymerase
0.5
H2O
32.5
PCR conditions (Thermal cycler)
Time
Temp.
Hold
98°C
Initial denaturation
30 sec
98°C
Denaturation
10 sec
98°C
Annealing
20 sec
58°C
Extension
20 sec
72°C
Number of cycles
30 x
Final extension
7 min
72°C
Output:
PCR products, A1, A2, A3 and B1, B2, B3
Further tasks:
Gel electrophoresis and purification
megaprimer PCR 2
Agarose gelelectrophoresis and purification of megaprimer I
Time: 2011-10-19
Investigators: Jessica, Nicole
Aim: Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used
Material:
Agarose
6x Loading dye
DNA Ladder (NEB)
Machery Nagel Nucleo Spin Extract II
Methods:
Purification of phages containing microviridin library
Investigators: Sandrina
Aim: phages carrying modified mdnA on their surface
Method/Materials:
after over night incubation:
centrifuge precipitaed phages: 5000 x g/ 45 min
discard supernatant
centrifuge again, 5000 x g/ 5 min
remove supernatant carefully
take pellet in 1 ml TBS
move in 1.5 ml Eppi
centrifuge again: 17000 x g/10 min
supernatant in a new Eppi with 200 µl PEG-NaCl (mix!)
incubate on ice for 60 min
centrifuge precipitated phages: 17.000 x g/ 10 min (4°C)
resuspend pellet in 300 µl TBS
centrifuge:17 000 x g/ 10 min (4°C)
supernatant in a fresh eppi= purified phages
Further tasks:
repeat procedure with ER2738 cells, because no phage pellet could be obserevd after precipitation
phage display
Colony PCR of TEV in vivo selcetion
For better understanding of described experiment see also: [ [10] ]
Investigators: Sascha, Paul, Sebastian
Aim: Getting the positiv clones for TEV in vivo selction
Materials:
XL1 clonies containing TorA-BlaFl construct with TEV cleavage site and TEV protease
Phusion Polymerase set
different primers
Methode:
PCR: 3min intial Dneaturation 98°C, 25 cycles: 10 sec denaturation 98°C, 35 sec annealing+elongation at 72°C. Final elongation for 5min at 72°C.
used primers:
f_TEV_iGEM
r_TEV_iGEM_BamHI
Colonies were picked with a 10µl tip, some bacterias were used for PCR reaction, the rest were used as pre culture of colnes. After Colony PCR and check via GE in 1% agarose gel positive clones were used for overnight cultures.
Output:
2 positive clones SG25 and SG26
Further Tasks:
plasmid prep, survival screening and sequencing
Making TEV and HRV14_3C Biobricks
For better understanding of described experiment see also: [ [11] ]
Investigators: Sascha, Paul, Stefan
Aim: Getting the positiv clones for TEV in vivo selction
Materials:
Phusion Polymerase set
different primers
different restriction enzymes
pSB1C3 vector
Methode:
PCR: 3min intial Dneaturation 98°C, 25 cycles: 10 sec denaturation 98°C, 35 sec annealing+elongation at 72°C. Final elongation for 5min at 72°C.
used primers for TEV:
f_TEV_iGEM
r_TEV_iGEM_BamHI
used primers for HRV14_3C:
f_HRV14_3C_iGEM
r_HRV14_3C_iGEM_BamHI
Mutated TEV and HRV14_3C were amplified out of vector containing mutated proteased (SG25 for TEV, SG24 for HRV14_3C), cleaned up with PCR purification kit form M&N, digested with EcoRI-HF, PstI-HF. Also pSB1C3 vector was digested with EcoRI and PstI. Both, proteases and vector were cleaned up via GE on 1% agarose gel followed by PCR clean up with M&N kit. The different proteases were ligated with pSB1C3 using T4 ligase purchased by Fermentas and the ligation products were transformed into XL1 blue cells using standard heat shock protocol.
Further Tasks:
plasmid prep and sequencing
113th Labday 2011-10-20
ELISA with purified phages carrying unmodified mdnA on their surface
Investigators: Sabine, Sandrina
Time: 2011-09-16, 12:00-21:00
Aim: recontrol if mdnA-myc-geneIII will be expressed on the phage
Method/Materials:
ELISA plate coated with 5 µg/ml anti-c-myc antibody in phosohate buffer (pH 7,4), volume 100 µl/ well
incubate for 4 h at room temperature
blocking in 2 % milk powder in TBS (300 µl/ well)
incubate for 2 h at room temperature
add 100 µl phages, produced in XL1-blu cells diluted 1:1 in TBS-T (0,005%)
incubate shaking for 60 min at room temperature
wash 5 x with TBS-T (0,05%)
add anti-gene-8-antibody (HRP coupled)
incubate shaking for 60 min
wash 5 x with TBS-T
put substrate on the samples
measure in plate photometer
reference: helper phages
Resluts: positive signal at wells with mdnA phages
SDS- PAGE with purified phages carrying mdnA on their surface
Investigators: Sabine, Sandrina
Aim: preparing geneIII-myc-mdnA-protein for western blot
Method/Materials:
seperating gel:
acryl amide: 2,5 ml
seperating gel buffer 4x: 1,5 ml
water: 1,94
APS: 60 µl
stacking gel:
acryl amide: 0,28 ml
stacking gel buffer: 0,5 ml
water: 1,22 ml
APS: 10 µl
loading dye: RotiLoad1
incubate samples at 95°C for 20 min
run at 50 V for half an hour, then switch to 20 mA
Further tasks:
western blot
Western Blot
Investigators: Sabine, Sandrina
Aim: control if geneIII-mdnA is expressed on the phages produced in XL1-blue cells
Method/Materials:
membrane impregnated in methanol
whatman paper
blotting buffer
blotting chamber
1 h at 200 mA
blocking over night in TBS with 5% milk powder
Further tasks:
antibody detection
Colony PCR,Mini-prep and Survival screening
For better understanding of described experiment see also: [ [12] ]
Investigators: Sascha, Paul, Sebastian
Aim: Getting the TEV into its test vector
Materials:
clones containing TEV and TorA-bla construct with TEV protease
colonies of TEV and HRV14_3C protease
Phusion polymerase Kit
different agar plates
Methode:
standard colony PCR protocol were used for TEV and HRV14_3C with same used for ampilication, followed by analytical GE with 1% agarose gel. all positiv clones were used for over night culture for plasmid prep.
The two positive clones SG25 and SG26 were grown over night in LB media, diluted next day and grown again to an OD of 0.3 to get them into exponential growth phase and diluted again to an OD of 0.002. 100µl of the dilution of each clone gots plated on 2 of each different set of agar plates (see below).
used Plates:
4x Cm (25 µg/ml)
4x Cm + 1mM IPTG
4x Cm + 2% Ara
4x Cm + 2% Ara +1mM IPTG
4x Cm + Amp (100 µg/ml)
4x Cm + 1mM ITPG + Amp (25 µg/ml)
4x Cm + 1mM ITPG + Amp (50 µg/ml)
4x Cm + 1mM ITPG + Amp (75 µg/ml)
4x Cm + 1mM ITPG + Amp (100 µg/ml)
4x Cm + 1mM ITPG + Amp (150 µg/ml)
4x Cm + 1mM ITPG + Amp (200 µg/ml)
4x Cm + 1mM ITPG + Amp (300 µg/ml)
4x Cm + 1mM ITPG + Amp (400 µg/ml)
4x Cm + 1mM ITPG + Amp (600 µg/ml)
4x Cm + 1mM ITPG + Amp (800 µg/ml)
4x Cm + 2% Ara + 1mM IPTG + Amp (25 µg/ml)
4x Cm + 2% Ara + 1mM IPTG + Amp (50 µg/ml)
4x Cm + 2% Ara + 1mM IPTG + Amp (75 µg/ml)
4x Cm + 2% Ara + 1mM IPTG + Amp (100 µg/ml)
4x Cm + 2% Ara + 1mM IPTG + Amp (150 µg/ml)
4x Cm + 2% Ara + 1mM IPTG + Amp (200 µg/ml)
4x Cm + 2% Ara + 1mM IPTG + Amp (300 µg/ml)
4x Cm + 2% Ara + 1mM IPTG + Amp (400 µg/ml)
4x Cm + 2% Ara + 1mM IPTG + Amp (600 µg/ml)
4x Cm + 2% Ara + 1mM IPTG + Amp (800 µg/ml)
Each clone was incubated at 30 and 37°C.
Further Tasks:
See results of in vivo selection system, plasmid prep of TEV and HRV14_3C Biobricks and sequencing.
Amplification of megaprimer II, using Phusion (Finnzymes)
Time: 2011-10-20
Investigators: Jessica, Nicole
Aim: Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used
Material:
primers:
a) Megaprimer A and r_mdn_iGEM
b) Megaprimer B and r_mdn_iGEM
pARW089, 8.57 ng/ µl
Phusion HF Buffer (Finnzymes)
dNTPs (10 mM)
Methods:
PCR mixture
volume in µl
dNTPs (10 mM)
1
Phusion HF Buffer
10
Primer reverse r_mdnABCDE_iGEM (10 µM)
2.5
DNA pARW089 (8.57 ng/ul)
1
Phusion polymerase
0.5
varied volume of megaprimer I
A1a (0.2 µM)
A1b (0.5 µM)
B1a (0.2 µM)
B1a (0.2 µM)
A2a (0.2 µM)
A2b (0.5 µM)
B2a (0.2 µM)
B2b (0.5 µM)
Primer forward, Megaprimer I
11.6
29.0
9.0
22.0
7.2
18
7.4
18.5
H2O
23.4
6.0
26.0
13.0
27.8
17
27.6
16.5
PCR conditions (Thermal cycler)
Time
Temp.
Hold
98°C
Initial denaturation
30 sec
98°C
Denaturation
30 sec
98°C
Annealing
20 sec
72°C
Extension
20 sec
72°C
Number of cycles
30 x
Final extension
7 min
72°C
Gelelectrophoresis and Purification
Output:
megaprimer A2 and B2
Further tasks:
amplification of whole cluster
Amplification of mdn-cluster
Time: 2011-10-20
Investigators: Jessica, Nicole
Aim: Site-directed mutagenesis of pARW089 for use as biobrick, improved strategy was used
Material:
primers:
a) Megaprimer A2 and Pf_mdnABCDET7
b) Megaprimer B2 and Pf_mdnABCDET7
pARW089, 8.57 ng/ µl
Long Enzyme Mix (Fermentas)
dNTPs (10 mM)
Methods:
PCR mixture
volume in µl
dNTPs (10 mM)
1
Long enzyme mix (Fermentas) Buffer
5
Primer forward Pf_mdnABCDET7 (10 µM)
1
DNA pARW089 (8.57 ng/ul)
1.2
Long PCR Polymerase
0.3
varied volume of megaprimer II
50 pmol
125 ng
250 ng
Primer forward, Megaprimer II
25
0.5
1.0
H2O
16.5
41
40.5
PCR conditions (Thermal cycler)
Time
Temp.
Hold
98°C
Initial denaturation
3 min
94°C
Denaturation
20 sec
94°C
Annealing
30 sec
68°C
Extension
7 min
68°C
Number of cycles
7 x
Denaturation
20 sec
94°C
Annealing
30 sec
68°C
Extension
7 min + 3s/cycle
68°C
Number of cycles
10 x
Final extension
7 min
68°C
Gelelectrophoresis and Purification
Output:
mdn-cluster A and B
Further tasks:
Digest and ligation with pSB1C3
114th Labday 2011-10-21
Antibody detection with blotted membranes
Investigators: Sandrina, Sabine
Aim: control if geneIII-myc-mdnA is presented on the phages
Method/Materials:
incubate blocked membrane for 1 h with primary antibody (anti -c-myc- antibody) in TBS with 5% milk powder
wash 3x 10 min with TBS buffer
incubate for 1 h with secondary antibody
wash 3x 10 min with TBS buffer
develop membranes with ECL- Kit
Results:
signals were seen at positive controls and a very weak signal at mdnA-myc-geneIII lane
Further tasks:
Results of Survival screening at 37°C and mini prep of TEV and HRV14_3C BioBricks
For better understanding of described experiment see also: [ [13] ]
Investigators: Sascha, Paul, Sebastian
Aim: Getting the TEV and HRV14_3C Biobrick, see if invivo selection at 37°C works
Materials:
plasmid prep Kit of M&N
Methode:
plasmids were isolated as described in the manual of the Kit
looking for clones on plates
Output
plasmids for sequencing P1,P2,P4,P7+T1,T2,T6
no results for in vivo selection
Further Tasks:
sequencing
Digest of mdn-cluster and pSB1C3 and ligation
Time: 2011-10-21
Investigators: Jessica, Nicole
Material:
EcoRI-HF, PstI-HF
pSB1C3 (#3)
PCR products: mdn-cluster A and B
Methods:
Digest pSB1C3
4 µl NEB bufer 4
15 µl pSB1C3 (250 ng/µl)
2 µl EcoRI-HF
2 µl PstI-HF
17 µl water
Digest mdn-cluster
5 µl NEB buffer 4
33 µl PCR product
2 µl EcoRI-HF
2 µl PstI-HF
8 µl water
Gel purification of pSB1C3
PCR purification of mdn-cluster
Ligation
1 µl T4 ligase buffer
1 µl T4 ligase
1 µl pSB1C3
7 µl mdn-cluster
Output:
ligations: pSB1C3+mdn-cluster A, pSB1C3+mdn-cluster B
Further tasks:
transformation
115th Labday 2011-10-22
Results of survival screening at 30°C
For better understanding of described experiment see also: [ [14] ]
Investigators: Sascha, Paul, Sebastian
Aim: see if in vivo selection could work
Materials:
none
Method:
counting colonies on plates
Results
For Both; Sg25 and SG26, cells survival up to 800 µg/ml ampicillin without active protease. Only for SG26 we were able to see cell death, if protease is activated, cells die early --> no clones on plates with 50µg/ml ampicillin.
over night culture of SG26 carrying XL1 blue cells.
Further Tasks:
making comptent cell out of SG26
Transformation with pSB1C3+mdn-cluster
Investigators: Jessica
Time: 2011-10-22
Method:
addition of 4 µl ligation reaction to XL1-blue cells
incubation 25 min on ice,
heat shock 45 sec at 42°C,
incubation 2 min on ice,
addition of 750 µl LB medium,
incubation 60 min at 37 °C and 750 rpm
plating on agar plates containing 100 µg/ml tetracyclin and 100 µg/µl ampicillin
storage over night at 37°C
116th Labday 2011-10-23
making competent cells and tranformation with Libary of SG26 carrying cells
Investigators: Stefan, Paul, Sascha
Aim: transformig XL1 blue cells SG26 with mdnA libary,
Methode:
Standard protocol for chemical competent cells followed by standard heat shock protocol for transformation. Transformed cells were plated on 10 agar plates containing cm and kan as antibiotics.
Results:
Libary transformed into cells with the TEV in vivo selection system.
further Tasks:
Test cells for surviving at higher ampicillin concentration than 25 µg/ml.
Production of phages containing microviridin library on their surface in ER2738 cells
Investigators:Sandrina, Sabine
Aim: production of phages carrying modified mdnA on their surfac for screening by phage display
Method/Materials:
first step: amplification of cells containing mutated mdnA:
500 ml LB medium will be inoculated with the cells, so that OD600 = 0.1
add antibiotics tetracyclin and ampicillin to the medium
cells should be incubated 37 °C till OD 600 = 0.3-0.5 (here: 0.356) is reached
second step: infection with helper phages
add helper phages 10^11 phages/50 ml (3,5 µl)
incubate for 10 min at 37°C (without shaking!)
add 0,5 mM IPTG
incubate 50 min at 28°C and rpm
add 70 µg/ml kanamycin and incubate for 5 h at 28°C (...)
third step: phage purification
centrifuge cell culture at 5000 x g/ 15 min
fill supernatant in a new 50 ml falcon and centrifuge again (5000 g/15 min)
40 ml of the supernatant with 8 ml PEG-NaCl (20% (w/v) PEG-8000, 2,5 M NaCl)
incubate over night at 4°C
Further tasks:
go on with phage purification
117th Labday 2011-10-24
Testing mdnA libary with SG26 carrying cells for TEV in vivo selection
Investigators: Stefan, Paul, Sascha
Aim: see if there is any mdnA which leads to cell survival
Methode:
library was harvest from agar plates, put into a preculture containing cm and kan and grown for 3 h at 37°C, deluted to an OD of 0.02 and plated on different agar plates.
1x Cm (25 µg/ml)
1x Cm + 1mM IPTG
1x Cm + 2% Ara
1x Cm + 2% Ara +1mM IPTG
1x Cm + Amp (100 µg/ml)
1x Cm + 1mM ITPG + Amp (20 µg/ml)
1x Cm + 1mM ITPG + Amp (30 µg/ml)
1x Cm + 1mM ITPG + Amp (40 µg/ml)
1x Cm + 1mM ITPG + Amp (50 µg/ml)
1x Cm + 1mM ITPG + Amp (100 µg/ml)
1x Cm + 1mM ITPG + Amp (400 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (20 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (30 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (40 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (50 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (100 µg/ml)
1x Cm + 2% Ara + 1mM IPTG + Amp (400 µg/ml)
all plates were incubated at 30°c for two days
Results:
Libary transformed into cells with the TEV in vivo selection system.
further Tasks:
Test cells for surviving at higher ampicillin concentration than 25 µg/ml.
Purification of phages containing microviridin library
Investigators: Sandrina, Sabine
Aim: phages carrying modified mdnA on their surface
Method/Materials:
after over night incubation:
centrifuge precipitaed phages: 5000 x g/ 45 min
discard supernatant
centrifuge again, 5000 x g/ 5 min
remove supernatant carefully
take pellet in 1 ml TBS
move in 1.5 ml Eppi
centrifuge again: 17000 x g/10 min
supernatant in a new Eppi with 200 µl PEG-NaCl (mix!)
incubate on ice for 60 min
centrifuge precipitated phages: 17.000 x g/ 10 min (4°C)
resuspend pellet in 300 µl TBS
centrifuge:17 000 x g/ 10 min (4°C)
supernatant in a fresh eppi= purified phages
118th Labday 2011-10-25
Control of sequencing results of sequenced clones from created library for phage display
Investigators: Sandrina, Jessica
Aim:check if mdnA library, which could be screened by phage display was produced
Results:
all clones contained the wildtype mdnA-myc-geneIII-gene
Further tasks:
repetition of cloning the mdnA library
Colony PCR of pSB1C3+mdn-cluster
Investigators: Jessica
Time: 2011-10-25
Method:
PCR mixture
volume in µl
dNTPs (10 mM)
1
Buffer
4
Primer reverse (10 µM)
1
Primer forward (10 µM)
1
DNA pARW089 (8.57 ng/ul)
1
polymerase
0.2
water
7.8
Gel electrophoresis
Overnight culture of pSB1C3+mdn-cluster A clone 6
Further tasks:
miniprep and sequencing
119th Labday 2011-10-26
Phage Display with wildtype mdnA and different proteases
Investigator: Sandrina, Sabine, Jessica
Time: 2011-09-14, 9:00-22:00
Aim:check if unmodified mdnA on the surface of the phages binds to an immobilized protease. interactions of microviridin L with the proteases trypsin, chymotrypsin and elastase were already detected
Material/Method:
in ELISA plate:
coat wells with proteases trypsin, elastase, chymotrypsin, papain, proteinase K, pepsin and mycolysin in 0,2 M PBS with ph 5.0 , for 2 h
blocking in TBS with 3% BSA for 2 h
wash 6x with TBS-T (0,005%)
panning with phages 1:1 helper phages and phages of interest (presenting mdnA-geneIII fusion protein on their surface, 1 h)
wash 6x with TBS-T
elute bound phages with 0,2 M Glycin-HCl, 10 min
neutralize with 1 M Tris (pH 7-8)
mix eluted phages with preparatory culture of ER2738 cells
incubate 10 min at 37°C without shaking
incubate 50 min at 37°C shaking
plate on amp plates and on kan plates
Further tasks:
check cell growing
Digest of pARWIII and ligation with library
Time: 2011-10-26
Investigators: Sandrina, Jessica, Sabine
Material:
Method:
Digest of pARWIII
Gel electrophoresis
Gel purification
Digest of geneIII
Purification
Ligation
2 µl T4 ligase buffer
1 µl T4 ligase
6 µl pARW089
10 µl geneIII
1 µl mdnA library
Output:
3 ligation products stored @ -20°C
Further task:
transformation of one ligation product and control
Transformation of triple ligated mdnA library, gene III and pARWIII
Time: 2011-10-26, 19:45-22:00
Investigators: Sandrina, Jessica, Sabine, Nadja
Aim:Phage display with created mdnA library
Materials:
XL1-Blue chemocompetent cells
ligation product, named geneIII mdnA lib pARWIII, 2011-10-11
transformation control, named H2O, 2011-10-11
add 2 µl PCR product to competent cells (for ligation product 5 samples overall)
incubate 25 min one ice
heat pulse: 45 sec at 42°C
2 min on ice
add 750 µl LB medium to each sample
incubate 45 min at 37°C and 850rpm
cenrifugation and resuspend in 200 µl LB medium
plate 200 µl on plates with ampicillin
incubate the transformation plates at 37°C for approx. 16 hours
Further tasks:
preparation of overnight cultures of several clones
Miniprep
Miniprep of pSB1C3+mdn-cluster A clone 6 and sending for sequencing
Time: 2011-10-26
Investigators: Sandrina, Jessica, Sabine, Nadja
Materials/Method:
NucleoSpin® Plasmid (NoLid) (Macherey-Nagel)
Protocol for high-copy plasmids
elution with 50 µl H2O
Results: (from 2011-10-28)
only part of the cluster could be found in pSB1C3
SDS- PAGE with lysated cells which produced the mdnA-myc-geneIII protein
Investigators: Sabine, Sandrina
Aim: preparing the cells which expressed geneIII-myc-mdnA-protein for detecting the expression of this protein by western blot
Method/Materials:
seperating gel:
acryl amide: 2,5 ml
seperating gel buffer 4x: 1,5 ml
water: 1,94
APS: 60 µl
stacking gel:
acryl amide: 0,28 ml
stacking gel buffer: 0,5 ml
water: 1,22 ml
APS: 10 µl
loading dye: RotiLoad1
incubate samples at 95°C for 20 min
run at 50 V for half an hour, then switch to 20 mA
Further tasks:
western blot
Western Blot
Investigators: Sabine, Sandrina
Aim: control if geneIII-mdnA is expressed ER2738 cells
Method/Materials:
membrane impregnated in methanol
whatman paper
blotting buffer
blotting chamber
1 h at 200 mA
blocking over night in TBS with 5% milk powder
Further tasks:
antibody detection
120th Labday 2011-10-27
Antibody detection with blotted membranes
Investigators: Sandrina, Sabine
Aim: control if geneIII-myc-mdnA is expressed in the cells
Method/Materials:
incubate blocked membrane for 1 h with primary antibody (anti -c-myc- antibody) in TBS with 5% milk powder
wash 3x 10 min with TBS buffer
incubate for 1 h with secondary antibody
wash 3x 10 min with TBS buffer
develop membranes with ECL- Kit
Results:
signals were seen at lanes loaded with cell lysate --> it can be concluded that mdnA-myc-gneeIII protein is expressed in the cells
control of performed phage display
Investigators: Sandrina, Sabine
Aim: control if phage display in general is possible with mdnA-myc-geneIII
Method/Materials:
count growed clones
Results:
phages carriying mdnA on their surface could be enriched, when chymotrypsin and elastase acted as targets
Retrieved from " http://2011.igem.org/Team:Potsdam_Bioware/Labjournal/October "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
