<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_Lab_Book_Fingerprints skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/Lab Book/Fingerprints</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="modal fade" id="PCR-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>PCR Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the  section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY color="black"><TR><TD width="312" valign="top"><P align="center"><STRONG>Reactants</STRONG></P></TD><TD width="312" valign="top"><P align="center"><STRONG>Volume (µl)</STRONG></P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DNA
                </P></TD><TD width="312" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DMSO
                </P></TD><TD width="312" valign="top"><P align="center">
                    2.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   100 µM Forward Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   100 µM Reverse Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    5 x Herculase Buffer
               </P></TD><TD width="312" valign="top"><P align="center">
                    10
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   10 mM dNTPs
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    H<SUB>2</SUB>O
                </P></TD><TD width="312" valign="top"><P align="center">
                    34.5
                </P></TD></TR></TBODY></TABLE><P>
    Elongation time was altered according to the length of the sequences.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="restrictiondigest-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Restriction Digests Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P><FONT color="#000000">
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the    <STRONG>Lab Book</STRONG> section.
</FONT></P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="170" valign="top" style="color:#000000;"><P align="center"><STRONG>Post Plasmid Purification</STRONG></P></TD><TD width="151" valign="top"><P align="center"><STRONG>Post PCR</STRONG></P></TD><TD width="139" valign="top"><P align="center"><STRONG>gBlock</STRONG></P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>DNA</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1 mg
                </P></TD><TD width="151" valign="top"><P align="center">
                    50 µl
                </P></TD><TD width="139" valign="top"><P align="center">
                    100 ng
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Cutsmart Buffer (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    6
                </P></TD><TD width="151" valign="top"><P align="center">
                    6
                </P></TD><TD width="139" valign="top"><P align="center">
                    6
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Water (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    As required
                </P></TD><TD width="151" valign="top"><P align="center">
                    1
                </P></TD><TD width="139" valign="top"><P align="center">
                    As required
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Enzyme 1 (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="151" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="139" valign="top"><P align="center">
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Enzyme 2 (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="151" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="139" valign="top"><P align="center">
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG><U>Total Volume (µl)</U></STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    30
                </P></TD><TD width="151" valign="top"><P align="center">
                    60
                </P></TD><TD width="139" valign="top"><P align="center">
                    30
                </P></TD></TR></TBODY></TABLE><P>
    The digests were incubated at 37<SUP>o</SUP>C for 3 hours.
    
    Note that when performing plasmid digestions, 2.5 µl of alkaline phosphatase and 2.5 µl of Cutsmart Buffer was added at the 2 hour and 2.5 hour mark. Digested PCR products were separated from restriction enzymes and buffers using Strataclean resin (Aligent Technologies). Digested vectors were purified on a 1% agarose gel and then extracted. 
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ligation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Ligations</B></CENTER></DIV></DIV><DIV class="modal-body"><P><FONT color="black">
Ligation reactions were set up according to the table below.
</FONT></P><TABLE border="1" cellspacing="0" cellpadding="0" class="text"><TBODY><TR><TD width="111" valign="top"><P align="center"><STRONG>Vector (µl)</STRONG></P></TD><TD width="111" valign="top"><P align="center"><STRONG>2:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>3:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>4:1 (Vector: Insert) [µl]</STRONG></P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>T4 DNA Ligase </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Ligase Buffer </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Vector </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Insert</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    -
                </P></TD><TD width="111" valign="top"><P align="center">
                    2
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>H<SUB>2</SUB>0</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    7
                </P></TD><TD width="111" valign="top"><P align="center">
                    5
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD></TR></TBODY></TABLE><P>
    The reactions were incubated at room temperature for at least 3 hours.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Transformation Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>

    1. 100 µl of chemocompetent <I>E.coli</I> cells were defrosted. Thaw on ice.
</P><P>
    2. Agar plates were taken out of 4 <SUP>o</SUP>C to warm to room temperature.
</P><P>
    3. 10 µl of ligation reaction or 2 µl of plasmid purified DNA was added to cells. The cells and DNA were mixed by gently flicking the tube.
</P><P>
    4. Cell and DNA mixture was left on ice for 20 minutes.
</P><P>
    5. Mixture was placed in 42 <SUP>o</SUP>C water bath for 90 seconds.
</P><P>
    6. Mixture was put back on ice for 2 minutes.
</P><P>
    7. 1 ml of LB (without antibiotic) was added to cells.
</P><P>
    8. Cells were allowed to grow for 1 hour at 37 <SUP>o</SUP>C in a shaking incubator.
</P><P>
    9. Cells were then spun down and the supernatant discarded.
</P><P>
    10. The pellet was resuspended in the 100 µl of remaining media.
</P><P>
    11. Cells were plated on agar plates with appropriate antibiotic.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="miniprep-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Plasmid Purification</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. 1-5 ml bacterial overnight culture was pelleted by centrifugation at &gt;8000 rpm for 3 minutes at room temperature.
</P><P>
    2. Pelleted bacterial cells were resuspended in 250 µl of Buffer P1 and transferred to a microcentrifuge tube.
</P><P>
    3. 250 µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution becomes clear.
</P><P>
    4. 350 µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.
</P><P>
    5. The mixture was centrifuged for 10 minutes at 13000 rpm in a microcentrifuge.
</P><P>
    6. The supernatant was applied to a QIAprep spin column by pipetting.
</P><P>
    7. The QIAprep spin column was washed with 500 µl of Buffer PB by centrifugation for 1 minute and flow through discarded.
</P><P>
    8. 750µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute and flow through discarded.
</P><P>
    9. The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
</P><P>
    10. The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 50 µl of H<SUB>2</SUB>O to the QIAprep spin column and centrifuging for 1 minute.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="onc-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Overnight Cultures </B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>
    1. Four colonies were picked from the desired plate.
</P><P>
    2. For each colony, 5 ml of fresh LB plus the appropriate antibiotics was inoculated and grown overnight at 37 <SUP>o</SUP>C.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sds-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>SDS-PAGE and Western Blotting</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>
    1. 50 µl of an overnight culture was used to inoculate 5 ml of fresh LB containing the appropriate antibiotics and left to grow at 37<SUP>o</SUP>C in a
    shaking incubator.
</P><P>
    2. Once the OD<SUB>600</SUB>=0.6-0.8, 1 mM of IPTG was added to the culture and left to grow for a further 3 hours.
</P><P>
    3. 1 ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.
</P><P>
    4. Meanwhile, 950 µl of 2x Laemmli sample buffer was added to 50 µl of B-mercaptoethanol.
</P><P>
    5. The supernatant was discarded from the samples and the pellet resuspended in 200 µl of the above solution.
</P><P>
    6. The samples were then boiled at 90<SUP>o</SUP>C for 10 minutes.
</P><P>
    7. Afterwards, they were spun down for 1 minute in a microcentrifuge.
</P><P>
    8. At this stage, the samples could be stored at -20<SUP>o</SUP>C or ran on a gel immediately.
</P><P>
    9. Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the
    stacking gel at which point the voltage was increased to 200V for 20 minutes.
</P><P>
    10. One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.
</P><P>
    11. After the proteins had been transferred onto the membrane using a semi dry transfer ran for 40 minutes at 5V, the membrane was blocked in 5% Marvel milk overnight at 4<SUP>o</SUP>C.
</P><P>
    12. The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10 ml of 1xTBS buffer to which the primary anti-His antibody was
    added and left for one hour. The protein ladder was cut off before adding the primary antibody.
</P><P>
    13. The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10 ml of 1xTBS buffer, to which the secondary antibody was
    added and left for one hour.
</P><P>
    14. The membrane was then washed in 3x 10 ml 1xTBST buffer for 5 minutes.
</P><P>
    15. The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off
    at step 12.
</P><P>
    16. Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Growth Curve Assay Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>    1. 5ml overnight cultures of the appropriate strains were set up.
</P><P>
    2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
</P><P>
    3. Three were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced.
</P><P>
    4. 200 µl of each culture was loaded onto a 96 well microtitre plate with 4 repeats.
</P><P>
    5. The cultures were left to grow for 16 hours in the plate reader which took OD readings every 10 minutes.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Growth Curve Assay Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. 5ml overnight cultures of the appropriate strains were set up.
</P><P>
    2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
</P><P>
    3. 20 x 200 µl aliquots were loaded onto a 96 well microtitre plate.
</P><P>
    4. After the OD<SUB>600</SUB>=~0.6, three cultures were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced (with 4 repeats).
</P><P>
    5. The cultures were left to grow for 20 hours in the plate reader which took OD readings every 30 minutes.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="pp-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Protein Purification Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. A 150 ml overnight culture was set up for the desired protein.
</P><P>
    2. 3 x 1 L of fresh LB containing 1 ml of appropriate antibiotics was inoculated with each 50 ml overnight culture and left to grow until an OD<SUB>600</SUB>
    =0.6-1 in a shaking incubator at 37<SUP>o</SUP>C.
</P><P>
    3. The cultures were subsequently induced with 1 mM IPTG and left to grow for 6 hours in a shaking incubator at 20<SUP>o</SUP>C.
</P><P>
    4. They were then centrifuged for 20 minutes at 4200 rpm at 4<SUP>o</SUP>C.
</P><P>
    5. The pellets were then resuspended in 5 ml of ice cold 50 mM 7.5 Tris and transferred to a 50 ml falcon tube and then topped up to 40 ml with 50 mM 7.5 Tris.
</P><P>
    6. They were then spun down again at 4000 rpm for 15 minutes at 4<SUP>o</SUP>C, the supernatant discarded and pellets stored at -20<SUP>o</SUP>C.
</P><P>
    7. Once purification could proceed, the pellets were thawed on ice and resuspended in 40 ml of 50 mM Tris containing DNAase, 1 x protease inhibitor tablet
    and lysozyme. (Note: If purifying a membrane protein, 1% DDM and 5 M urea was also added).
</P><P>
    8. The cells were lysed using a cell disruptor.
</P><P>
    9. Afterwards the lysate was spun down in a J25.50 rotor at 15000 rpm at 4<SUP>o</SUP>C for 40 minutes.
</P><P>
    10. The supernatant was removed and transferred to a 50 ml falcon tube. The pellet was kept for testing. Both were kept on ice.
</P><P>
    11. Using FPLC, the supernatant was loaded onto a Ni<SUP>2+</SUP> column.
</P><P>
    12. The protein was eluted using an increasing concentration of imidazole. The two buffers used were Buffer A: 50 mM Tris, 200 mM NaCl, 20 mM imidazole and
    Buffer B: 50 mM Tris, 200 mM NaCl, 1 M Imidazole. (Note: If purifying a membrane protein 0.02% DDM was added to these buffers.)
</P><P>
    13. The fractions corresponding to any peaks were retained and concentrated using an appropriate filter size. A small amount of each fraction could be ran
    on an SDS gel at this stage.
</P><P>
    14. Using FPLC, the sample was then loaded onto a SEC column and any fractions corresponding to a peak were ran on an SDS gel to check for the presence of
    the desired protein.
</P><P>
    15. Fractions containing the desired protein were collated, to which 20% glycerol was added.
</P><P>
    16. The samples were then flash frozen and stored at -80<SUP>o</SUP>C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ap-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Assay Preparation</B></CENTER></DIV></DIV><DIV class="modal-body"><OL><LI> Take 3 independent colonies from each plate of your controls and samples and place each colony into 5 ml of LB with the appropriate antibiotics. </LI><LI> Leave to grow overnight at 30°C.</LI><LI> Take 50 μl of each overnight and inoculate 5 ml of fresh LB containing the appropriate antibiotics. Do this twice for every independent colony. </LI><LI> Allow these to grow at 30°C shaker until the O.D of each inoculate is between 0.3-0.5</LI><LI> Once the O.D is between 0.3-0.5 take 3 x 1 ml samples of each culture and place into separate Eppendorf tubes.</LI><LI> Take a further 1 ml and place into a cuvette and  subsequently measure the OD<SUB>600</SUB> marking down the OD for each replica. </LI><LI> Spin down the 1 ml cultures of each replica at 13,000 rpm for 3 minutes. </LI><LI> Remove the supernatant from each 1 ml culture and freeze cell pellet for later use in the β-galactosidase Assay. </LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bga-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>β-Galactosidase Assay</B></CENTER></DIV></DIV><DIV class="modal-body"><P><OL><LI> Resuspend cell pellet in 800 μl of buffer 2 (100 mM dibasic sodium phosphate (Na2HPO4); 20 mM KCl ; 2 mM MgSO4 ;0.8 mg/mL CTAB (hexadecyltrimethylammonium bromide) ; 0.4 mg/mL sodiumdeoxycholate) and subsequntly add 5.4 μL/mL beta-mercaptoethanol. </LI><LI> Add 50 μl of SDS (0.1%) to each sample. </LI><LI> Add 50 μl of Chloroform to each sample and vortex for 30 seconds. </LI><LI> Make substrate solution (60 mM Na2HPO4; 40 mM NaH2PO4; 1 mg/mL o-nitrophenyl-β-D-Galactoside (ONPG); 2.7 μL/mL β-mercaptoethanol)</LI><LI> Add 200 μl of substrate solution to each sample marking down the time it was added in relation to the first sample. </LI><LI> Wait for an observable color change and mark down the time that the color change occurred.</LI><LI> Add 600 μl of NaCO3(1M) once the color change has occurred.</LI><LI> Spin down the samples at 13,000 rpm for 3 minutes and place the supernatant into a cuvette. </LI><LI> Measure the OD<SUB>420</SUB>. </LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plate Reader Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>A 5 ml overnight culture of the desired strain was grown at 37 <SUP>o</SUP>C.</LI><LI>Equalise OD<SUB>600</SUB> if required. Dilute culture 1:1000.</LI><LI>Aliquot 198 µl into the required amount of wells in a 96-well plate.</LI><LI>Add additional substances and controls as required.</LI><LI>Preheat platereader to required temperature and start experiment.</LI><LI>Retrieve and analyse data.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="competent-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Competent Cells Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>A 5 ml overnight culture of the desired strain was grown at 37<SUP>o</SUP>C.</LI><LI>Subsample <SUP>1</SUP>/<SUB>10</SUB> of the volume of the subculture. I.e. 1 ml of the overnight culture into 10 ml fresh LB.</LI><LI>Grow subculture at at 37 <SUP>o</SUP>C for 2 h.</LI><LI>Spin down subculture at 4000 rpm at 4<SUP>o</SUP>C for 10 min.</LI><LI>Resuspend the pellet in ice cold transformation buffer. Use <SUP>1</SUP>/<SUB>10</SUB> of the initial volume of the subculture.</LI><LI>Use 100 µl per transformation.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="plating-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plating Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>Fresh agar plates were inoculated with cells on the toothpick.</LI><LI>Agar plates were incubated at 4<SUP>o</SUP>C over night.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="tf-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Tryptophan Fluorescence Protocol</DIV></DIV><DIV class="modal-body"><P>
    1. Intensity of tryptophan fluorescence of PotD was measured on a fluorimeter using an excitation of 280 nm and emission of 350 nm.
</P><P>
    2. Increasing concentrations of spermidine were added in 50 µM increments to give a final concentration of 250 µM spermidine to 5 µM of purified PotD.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sequencing-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Sequencing Protocol</DIV></DIV><DIV class="modal-body"><P>
    For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
      <UL><LI>Volume of sample equivalent to 550ng DNA</LI><LI>2 µl of 3 mM primer</LI><LI>Remaining volume of water for a total volume of 30 µl</LI></UL></P><P>
      The samples were submitted to <A href="https://www.dnaseq.co.uk/" target="_blank">DNA sequencing and services</A>.
    </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="overnightculture-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Overnight Culture Protocol</DIV></DIV><DIV class="modal-body"><P><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>5 ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.</LI><LI>Tubes were incubated at 37<SUP>o</SUP>C in a rotary incubator at 200rpm over night.</LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="gelextraction-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Gel Extraction Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.</LI><LI>The gel slice was transferred into a microcentrifuge tube. 700 µl of Buffer QG was added.</LI><LI>The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.</LI><LI>After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.</LI><LI>230 µl of isopropanol was added to the sample and mixed.</LI><LI>A QIAquick spin column was placed in a 2 ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>500 µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>750 µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>The QIAprep spin column was spun for an additional minute to remove residual wash buffer.</LI><LI>The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: <B>Amplification of <I>LbpA</I></B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: LbpA PCR. Lane 1: PCR Product, Lane 2: 1kb Ladder and Marker.</B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: LbpA PCR for Cloning into pQE80-L. Lane 1: LbpA PCR Product, Lane 2: 1kb Ladder and Marker</B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: LbpA Culture Samples. Lane 1: Sample from uninduced culture. Lane 2: Sample from culture induced with 1mM IPTG.</B> It seems that on comparison of the two cultures, that inducing expression of LbpA causes the cells to die given the significant reduction of protein levels visible on the gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: OD<SUB>600</SUB> Readings from Uninduced Culture and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG).</B> The table shows that after being induced with IPTG the cultures stop growing since their OD<SUB>600</SUB> readings are notably lower than that of the uninduced control. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, the induced cultures stop growing when LbpA expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, after reaching an OD<SUB>600</SUB> of 0.6,  the induced cultures stop growing when LbpA starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have dealt with the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: SDS Gel of Samples Taken from Uninduced Control and 1 mM IPTG Induced Culture 6 hours After Induction</B> For the induced culture, a faint band just above the 100 kDa marker is observable on the gel which corresponds to the expected size of LbpA.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 1: Gel of <I>SBP</I> PCR Product for Cloning into pQE80-L. Lane 1: <I>SBP</I> PCR Product 1 Lane 2: 1kb Ladder and Marker Lane 3: <I>SBP</I> PCR Product 2</B> Both bands observable on the gel correspond to the expected size of <I>SBP</I> which is ~600 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: SDS Gel of Uninduced and 1 mM IPTG Induced Cultures of PotD and SBP.</B> For PotD, it is clear from the gel that it is being successfully overexpressed since there is an intense band present at the 37 kDa marker which corresponds to the expected size of PotD in the induced culture sample. For SBP, there is no band present which suggests SBP is being overexpressed. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Characterisation of PotD.</B> Single colonies of MC1061 <I>E.coli</I> containing pQE80-L encoding <I>PotD</I> were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin. Once cultures reached an OD<SUB>600</SUB> of 0.6, one was induced with 1 mM IPTG and one was left uninduced as a control. They were left to grow for a further three hours, after which a 1ml sample was taken and the cells pelleted. The cells were resuspended in 100µl of Laemmli buffer, 20µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: SDS Gel of Uninduced and 0.5mM, 1mM and 2mM IPTG Induced Cultures of SBP.</B> It appears that upon induction of <I>SBP</I> with any concentration of IPTG the cells are dying given that there is a significant drop in protein levels observable on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, the induced cultures stop growing when SBP expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, after reaching an OD<SUB>600</SUB> of 0.6,  the induced cultures stop growing when <I>SBP</I> starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of PotD</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100 V. The two bands observable on the gel correspond to the expected size of PotD of 38 kDa. Western Blotting will confirm if these proteins are PotD. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: Characterisation of PotD following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure9-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 9: Characterisation of SBP following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure10-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 10: Characterisation of PotD/Spermidine Interaction.</B> The graph shows that the change in the tryptophan spectra increases as concentration of spermidine increases. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 1: pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> (respectively) Pre-Sequencing Digest</B> pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> minipreps were digested with EcoRI and PstI to check for presence of each insert. The bands observable on the gel correspond to the expected sizes for the pSB1C3 vector and both inserts.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: Amplification of <I>hHBA</I> and <I>hHBB</I></B><I>hHBA</I> and <I>hHBB</I> were amplified from the pSB1c3-<I>hHBA</I> and pSB1c3-<I>hHBB</I> minipreps obtained yesterday using primers to allow cloning of the genes into pT25 and pUT18 respectively. Although the band is fainter in the case of <I>hHBA</I>, the gel indicates the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Amplification of <I>hHBA</I></B><I>hHBA</I> was amplified from the pSB1C3-<I>hHBA</I> using primers to allow cloning of the genes into pT25. The gel indicates that the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: Amplification of <I>hHBA</I> and <I>hHBB</I></B><I>hHBA</I> and <I>hHBB</I> were amplified from the pSB1c3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> minipreps using primers to allow cloning of the genes into pQE80-L. The gel indicates that the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Pre-Sequence Digest of pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I></B> Both pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I> minipreps were digested with BamHI and KpnI to check for the presence of the inserts. The gel indicates that both inserts have been successfully cloned into pQE80-L. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Overexpression of <I>hHBB</I> and <I>hHBA</I>.</B> Single colonies of M15[pREP4] <I>E.coli</I> containing pQE80-L encoding <I>hHBB</I> and pQE80-L encoding <I>hHBA</I> were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD<SUB>600</SUB> of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of 2x Laemmli buffer, 20 µl of which was separated by SDS-PAGE. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: Characterisation of hHBB.</B> Single colonies of M15[pREP4] <I>E.coli</I> containing pQE80-L encoding <I>hHBB</I> were used to inoculate 5 ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD<SUB>600</SUB> of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: Amplification of <I>hHBN</I> for Cloning into pQE80-L </B> PCR product from amplification of <I>hHBN</I> from IDT plasmid using primers for cloninig into pQE80-L. The gel indicates <I>hHBN</I> has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure9-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 9: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of hHBB</B> 10 µl of each fraction was mixed with 10 µl of laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. The four bands observable on the gel correspond to the expected size of hHBB of 16 kDa. Western Blotting will confirm if these proteins are hHBB. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure10-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 10: Characterisation of hHBB following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure11-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 11: Amplification of <I>hHBN</I> for Cloning into pT25 </B> PCR product from amplification of <I>hHBN</I> from IDT plasmid using primers for cloninig into pT25. The gel indicates <I>hHBN</I> has been amplified successfully since the observed band corresponds to the expected size of 1200 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure14-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 14: Characterisation of hHBN following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure15-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 15: Amplification of <I>hHBN</I></B><I>hHBN</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of <I>hHBN</I> of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure16-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 15: Amplification of <I>hHBA</I></B><I>hHBA</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of <I>hHBA</I> of </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure17-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 16: Amplification of <I>hHBN</I></B><I>hHBN</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pUT18. The band observable on the gel corresponds to the expected size of <I>hHBN</I> of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 1: Amplification of <I>OBP2A</I> for insertion into pSB1C3 vector </B>. This is the PCR product from the amplification of <I>OBP2A</I> from the IDT plasmid using primers for cloning into pSB1C3. The gel indicates <I>OBP2A</I> has been amplified successfully since the observed band corresponds to the expected size of 500 bp. The concentration of the gel extracted <I>OBP2A</I> is 335.53 ng/ul. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 2: A subsequent gel was run of the amplified <I>OBP2A</I> after it was first excised from the gel in <I>Figure 1 </I></B> This was to determine if the gel extraction was successful. The gel indicates that <I>OBP2A</I> has been extracted from the gel successfully, since the observed band corresponds to the expected size of 500 bp. This can now be restricted in preparation for ligation into pSB1C3. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 3: A Gel Analysis of Restriction Digest from the 24/6.</B> This gel analysis was done to determine if the reason for the failed transformation was an improperly digested <I>OBP2A</I>. This gel indicates that the restricted product hasn't been lost as the <I>OBP2A</I> gene fragment can be seen as a distinct band at 500 bp. A re-ligation of <I>OBP2A</I> into the pSB1C3 backbone will now occur. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 4: Presequence Digest of pSB1C3-<I>OBP2A</I></B>. All seven plasmid purifications of the overnight cultures done on the 26/6 were digested to see if they contained the <I>OBP2A</I> insert. It is clear from the gel that all but sample 3 contain the insert for <I>OBP2A</I> as they all have faint bands at 500 bp's which corresponds to the size of <I>OBP2A</I>. It also appears that sample 5 contains the highest concentration, so that will be the sample that is sent for sequencing.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 5: Presequence digest of pUT18-<I>OBP2A</I></B> To determine if any samples contain the <I>OBP2A</I> insert. It can be seen from this gel that of the  5 samples that were digested only two are show to have the <I>OBP2A</I> insert, therefore sample 2 will be sent for sequencing.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 6: Amplification of <I>OBP2A</I> for insertion into pT25 vector </B>. This is the PCR product from the amplification of <I>OBP2A</I> for cloning into pT25. The gel indicates <I>OBP2A</I> has been amplified successfully since the observed band corresponds to the expected size of 100 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 7: Presequence Digest of pT25-<I>OBP2A</I></B> In order to determine if any of the samples contain the <I>OBP2A</I> insert. It can be seen from this gel that samples 5, 6 and 7 may have inserts in them as they each have two distinct bands, one of which corresponds to the 100 bp size for the <I>OBP2A</I> for pT25. As a result all three samples will be sent for sequencing.    </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 8: β-galactosidase Assay</B>  A graph of the results from the first attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>. As it can be seen that the positive result has failed(far left) not much can be inferred from the interaction of the two parts of <I>OBP2A</I>. However it looks like the two subunits of <I>OBP2A</I> may not be interacting. As this was a first attempt, subsequent assays will be performed in order to determine if in fact the two separated parts of OBP2A aren't interacting in vivo - like these results suggests.      </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure11-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 9: β-galactosidase Assay</B> A graph of the results from the second attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>.  It can also be inferred from this graph that the two parts of <I>OBP2A</I> don't seem to be interacting as it has a low millers activity when compared to the controls. Further assays will be performed in order to determine if the two separated parts of <I>OBP2A</I> aren't interacting in vivo like these results suggests.     </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure12-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 11: β-galactosidase Assay Under Anaerobic Conditions </B> A graph of the results from the anaerobic β-galactosidase assay aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>. The graph seems to suggest that the two subunits of <I>OBP2A</I> may not be interacting in vivo. This can be inferred from the fact that the millers activity of the <I>OBP2A</I> sample lies lower than the negative controls from within the experiment. As a result of this information, one more assay will be prepared in which the two subunits will be switched with regards to the bacterial two hybrid vectors to eliminate any possibility of this being a factor effecting the interaction between the two subunits.     </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: <B>Amplification of <I>LS</I></B><I>LS</I> was amplified using primers which would allow for subsequent cloning into the high expression vector pQE80-L. Lane 1 shows that the PCR has been successful since the band corresponds to the expected size of 2100 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: Amplification of <I>LS</I></B><I>LS</I>  was amplified from the IDT plasmid using primers to allow cloning of the gene into pSB1C3. The observed band corresponds to the expected size of LS, 2100 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Overexpression Assays of <I>LS</I></B> 5 ml cultures of M15pREP4 <I>E.coli</I> was grown to OD<SUB>600</SUB>=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37<SUP>o</SUP>C, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: Characterisation of <I>Lanosterol Synthase</I></B> 5 ml cultures of M15pREP4 <I>E.coli</I> was grown to OD<SUB>600</SUB>=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37<SUP>o</SUP>C, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. The proteins were transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Chromatogram following Affinity Chromatography of LS</B> Using FPLC, the protein sample was loaded onto a nickel column and eluted with an increasing concentration of imidazole (green line). The fractions corresponding to the peak were ran on an SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: SDS Gel of Peak Fractions </B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer an loaded onto a SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="BBa_K1058008-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: Alignment of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>. Sequences from top to bottom: sequencing result from forward primer; expected sequence; reverse complement of sequencing result from reverse primer; sequencing result from intermediate primer. Colour code: Green = Prefix/Suffix; Pink = Annealing site of intermediate primer; Yellow = Mismatch in one sequence; Red = In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem022-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 2: 1% agarose gel after PCR of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with primers specific to the promoter region, P<I>chr</I>, and the repressor region, <I>chrB</I>. Left: P<I>chr</I> (expected size 169bp); Middle: 1kb+ ladder, Right: <I>chrB</I> (expected size 939bp). Amplification of <I>chrB</I> was not successful. P<I>chr</I> was further processed by restriction digest with EcoRI and PstI.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem023-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 3: 1% agarose gel after PCR of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with primers specific to the repressor region, <I>chrB</I>. Left: 1.5 µl DMSO; Middle: 1kb+ ladder, Right: 2.5 µl DMSO. Expected size for each: 939bp. The annealing temperature for the PCR amplification was lowered to 50°C in order to increase the chances of accomodating a primer that is not 100% complementary.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem024-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 4: 1% agarose gel after PCR of <I>gfp</I> with corresponding primers. Left: <I>GFP</I> (expected size 717bp); Middle: 1kb+ ladder, Right: PCR product of <I>chrB</I>, using primers to add a BamHI restriction site in front of it and a 6-His-Tag and a HindIII restriction site at the end. (expected size of <I>chrB</I>: 939bp). Since amplification of <I>chrB</I> was unsuccessful, it will be repeated.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem026-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 5: 1% agarose gel after restriction digest of pSB1C3-P<I>chr</I>, pSB1C3-<I>chrB</I> (s), and pSB1C3-<I>chrB</I> (w) with EcoRI and PstI for confirmation of size. Ladder: 1kb+. Top: 1-4: P<I>chr</I> 1-4; <I>chrB</I> (s) 1-2. Bottom: <I>chrB</I> (s) 3-4; <I>chrB</I> (w) 1-4. Expected sizes: P<I>chr</I>: 169bp; <I>chrB</I>: 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-pChr1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 6: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green = Prefix/Suffix
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-ChrBw4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 7: Alignment of <I>chrB</I> (opt) sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green: Prefix and Suffix; Red: In frame stop codons; Blue: First 15 codons optimised for expression in E. coli.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem034-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 8: 1% agarose gel after restriction digest of pUniprom (Expected size 2454bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem035-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 11: 1% agarose gel after restriction digest of pSB1C3 with EcoRI and SpeI (Expected size 2070bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem032-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 9: 1% agarose gel after PCR of <I>chrB</I> (opt). (Expected size 939bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-first-transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 10: Plates after transformation of Top: pUniprom<I>chrB</I>, Middle: pUniprom<I>chrB</I> (opt) and Bottom: pSB1C3-P<I>chr-gfp</I> into JM110.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem036-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 12: 1% agarose gel after restriction digest of pUniprom<I>chrB</I>, pUniprom<I>chrB</I> (opt) and pSB1C3-P<I>chr-gfp</I> for confirmation of size. Ladder: 1kb+. Top: 1-4: pSB1C3-P<I>chr-gfp</I> 1-4; pUniprom<I>chrB</I> (opt) 1-2. Bottom: pUniprom<I>chrB</I> (opt) 3-4; pUniprom<I>chrB</I> 1-4. Expected sizes: P<I>chr-gfp</I>: 909bp; <I>chrB</I> and <I>chrB</I> (opt): 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem037-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 13: 1% agarose gel after colony PCR of pUniprom<I>chrB</I> (opt). Top: colonies 1-11; Bottom: colonies 12-22. Colonies 2, 4, 9, and 12 were selected for sequencing. Ladder: 1kb+ Expected size of <I>chrB</I> (opt): 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pSB1C3-pChr-GFP_reverse" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 14: Sequencing result of pSB1C3-P<I>chr-gfp</I> after double ligation. Even though the sequence is as expected, it is ligated in reverse. It was hence called pSB1C3-P<I>chr-gfp</I> (A) The Result for pSB1C3-P<I>chr-gfp</I> (colony 2) was not as expected, albeit in the correct orientation. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pChrGFPplates_revisited" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 15: Plates with pSB1C3-P<I>chr-gfp</I> in JM110 after double ligation under UV light. Numbered colonies were selected for initial size confirmation and sequencing. However, other colonies seemed to express GFP as well, hence a second attempt was made with different colonies from the same plate.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pChrGFP_revisited_seq" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 16: Alignment of sequencing results of pSB1C3-P<I>chr-gfp</I> - colonies 1, 5, 6, and 7 with sequences of <I>gfp</I> (mut2) and P<I>chr</I> sequence. Colony 1 remained consistent, the other tested colonies were without success. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem039" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 17: 1% Agarose gel after restriction digest of pUniprom with BamHI/PstI (expected size: 2454bp). Right: PCR product of Haptoglobin.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150723_pSB1C3-pChr" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 18: Plates after transformation of P<I>chr</I> into pSB1C3. Top: ratio insert:vector 2:1; Bottom: ratio insert:vector 3:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150727_pSB1C3-pChr_alignment" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 19: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green = Prefix/Suffix
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150728_pUniprom-ChrB" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 20: Plates after transformation of, Top: <I>chrB</I> into pUniprom; Bottom: <I>chrB</I> (opt) into pUniprom. Ratio insert:vector for ligations: left: 3:1; Right: 4:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem040" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 21: Gel after restriction digest of Left: pUniprom-<I>chrB</I>, and Right: pUniprom-<I>chrB</I> (opt) with BamHI/PstI for confirmation of insert size. Expected size 939bp. <I>chrB</I> colony 2 and <I>chrB</I> (opt) colony 3 were selected for sequencing.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150729_chrb-combined" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 22: Alignments of: Left: <I>chrB</I> and Right: <I>chrB</I> (opt). Colours: Pink = Restriction sites (BamHI/PstI); Blue = 6-His Tag; Red = In-frame stop codons; Cyan = optimised codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150731_pChrGFP-ligations" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 23: Plates after ligation of <I>gfp</I> into pSB1C3-P<I>chr</I>. Left: Ratio Insert:Vector 2:1; Right: Ratio Insert:Vector 3:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem043" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 24: Gel for size confirmation of pSB1C3-P<I>chr-gfp</I> (Top). Bottom: Left - Haptoglobin, Right - Lanosterol Synthase. pSB1C3-P<I>chr-gfp</I> (3) and pSB1C3-P<I>chr-gfp</I> (4) were submitted for sequencing.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150801_pchrgfp" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 25: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> and <I>gfp</I> (mut2) with the forward and reverse sequencing results of P<I>chr-gfp</I> this region. The sequence was as expected, it was hence called pSB1C3-P<I>chr-gfp</I> (B). Colour code: Green = Prefix/Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150809_blot" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 26: Top: anti-GFP: Single colonies of JM110 + pSB1C3-P<I>chr-gfp</I> (A), MC1061 + pSB1C3-P<I>chr-gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> chloramphenicol. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of each sample was mixed with 100 µl laemmli buffer, and boiled for 10min. 3 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and P<I>manA-gfp</I> as a positive control. 
        Bottom: anti-6-His - ChrB: Single colonies of JM110 pUniprom-<I>chrB</I> and JM110 pUniprom-<I>chrB</I> (opt) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 200 µl laemmli buffer, and the sample boiled for 10min. 10 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and <I>hydA - 6-His</I> as a positive control.
        Expected sizes: GFP ~ 27kDa, ChrB ~35kDa.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150811_transformations" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 27: Plates after transformation of single constituents of chromate sensor into <I>E. coli</I> MG1655. From top to bottom: pUniprom-<I>chrB</I> (from colony 2 - 29/07), pUniprom-<I>chrB</I> (opt) (from colony 3 - 29/07), pSB1C3-P<I>chr-gfp</I> (B), pSB1C3-P<I>chr-gfp</I> (A), <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150822_blot" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 28: Western blot against GFP: Single colonies of MG1655, containing all 4 combinations, pUniprom-<I>chrB</I> - pSB1C3-P<I>chr-gfp</I> (A), pUniprom-<I>chrB</I>- pSB1C3-P<I>chr-gfp</I> (B), pUniprom-<I>chrB</I> (opt) - pSB1C3-P<I>chr-gfp</I> (A), and pUniprom-<I>chrB</I> (opt) - pSB1C3-P<I>chr-gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> chloramphenicol and 100 <SUP>µg</SUP>/<SUB>ml</SUB> ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml PBS. 100 µl of these samples was mixed with 100 µl laemmli buffer, and the sample boiled for 10 min. The indicated volume of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3, and pUniprom were included as a negative control, and P<I>manA-gfp</I> as a positive control. The blot against 6-His – ChrB was without success and is not shown. No chromate of any kind was added to the medium previous to this blot, hence no expression of GFP was expected. 
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV>


K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB><DIV class="modal fade" id="dundee15_chr_150824_platereader_BBa_K1058008" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 29: F,OD-plot of results of plate reader experiment after exposing MG1655 + BBa_K1058008 to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="icecream" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 30: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="potato" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 31: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt) to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150824_platereader_pchrbgfp" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 32: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_chrbbielefeld" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 33: Plates after ligation of <I>chrB</I> (Bielefeld) into pT7KS. From top to bottom: Ratio insert:vector 2:1; Ratio Insert:Vector 3:1; pT7KS religation control.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_bl21" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 34: Plates after transformation of single constituents of chromate sensor into <I>BL21 (DE3)</I>. From top to bottom: pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt), pUniprom-<I>chrB</I>, and pUniprom-<I>chrB</I> (opt).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_igem047" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 35: 1% agarose gel after colony PCR of pT7KS-<I>chrB</I> (Bielefeld). Top: colonies 1-6; Bottom: colonies 7-12. Ladder: 1kb+ Expected size of <I>chrB</I> (Bielefeld): 939bp
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_platereaderinduced" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 36: F,OD-plot of results of plate reader experiment after inducing expression of ChrB from the T7 promoter of pUniprom <I>BL21 (DE3)</I> pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, and pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt). IPTG was added after 4 hours. One measurement was made every 10min across 22h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV><H1><HIGHLIGHT class="highlight">Fingerprint Ageing</HIGHLIGHT></H1><H3><HIGHLIGHT class="highlight">Lab Book</HIGHLIGHT></H3><SECTION id="about" class="row1"><DIV class="row"><DIV class="col-lg-4"><H3>FluID</H3><P class="about-content">FluID is a cell free, all in one spray which will be used to detect and dinstiguish between body fluids found at crime scenes.</P></DIV><DIV class="col-lg-4"><H3>Chromate Biosensor</H3><P class="about-content">Our chromate biosensor is a device that will be used to detect stainless steel residues left behind on bones in blunt or sharp force trauma cases.</P></DIV><DIV class="col-lg-4"><H3>Fingerprint Ageing</H3><P class="about-content">This device will search for the presence of a specifc component found in fingerprint deposits whose presence/absence will indicate approximate fingerprint age.</P></DIV></DIV></SECTION><DIV class="ribbon"><DIV class="rows"><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 24/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I>LS</I> into the high expression vector pQE80-L was started this week.</P></DIV><DIV id="collapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">26/8: Amplification of <I>LS</I> for Cloning into pQE80-L</SPAN><DIV id="collapsiblew1-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To amplify <I>LS</I> for subsequent cloning into pQE80-L.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#fp-figure1-modal" class="journal-protocol"> Figure 1</A></P><P><B>Next Steps:</B> The gel extracted PCR product will be digested and ligated into pQE80-L tomorrow.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/8: Restriction Digests and Ligations of <I>LS</I> with pQE80-L</SPAN><DIV id="collapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set digest <I>LS</I> with BamHI and KpnI and set up ligations with the high expression vector pQE80-L. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> Ligations will be left in the cold room over the weekend and transformed into M15pREP4 <I>E.coli</I> on Monday. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 31/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I>LS</I> into pQE80-L was completed this week and cloning of <I>LS</I> into pSB1C3 was begun.</P></DIV><DIV id="collapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">31/8: Transformations of pQE80-L-<I>LS</I> into M15pREP4 <I>E.coli</I></SPAN><DIV id="collapsiblew2-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up transformations of pQE80-L-<I>LS</I> into M15pREP4 <I>E.coli</I></P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformations</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> If the transformations are successful, colonies will be selected for overnight cultures tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/9: Overnight Cultures of M15pREP4 <I>E.coli</I> containing pQE80-L-<I>LS</I></SPAN><DIV id="blcollapsiblew2-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of M15pREP4 <I>E.coli</I> containing pQE80-L-<I>LS</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The overnight cultures will be miniprepped tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">2/9: Plasmid Purification of M15pREP4 <I>E.coli</I> Overnight Cultures containing pQE80-L-<I>LS</I></SPAN><DIV id="collapsiblew2-3" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the M15pREP4 <I>E.coli</I> Overnight Cultures containing pQE80-L-<I>LS</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The miniprep with highest concentration will be sent for sequencing. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/9: Amplification of <I>LS</I> for Cloning into pSB1C3</SPAN><DIV id="collapsiblew2-4" class="collapse box-content"><P><B>Aim of experiment:</B> To amplify <I>LS</I> for cloning into pSB1C3.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#fp-figure2-modal" class="journal-protocol"> Figure 2</A></P><P><B>Next Steps:</B> The gel extracted PCR product will be digested tomorrow and ligations with pSB1C3 will be set up.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">19/6: Restriction Digests and Ligations of <I>LS</I> with pSB1C3</SPAN><DIV id="collapsiblew2-5" class="collapse box-content"><P><B>Aim of experiment:</B> To digest the <I>LS</I> PCR product with EcoRI and PstI and then set up ligations with pSB1C3.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligations</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> Ligations will be left over the weekend and transformed into JM110 <I>E.coli</I> on Monday. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 7/9/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Cloning of <I>LS</I> into pSB1C3 was completed this week and sequencing confirmed it was successfully inserted into pQE80-L so overexpression assays were performed where LS was characterised via Western Blotting. </P></DIV><DIV id="collapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">7/9: Transformations of pSB1C3-<I>LS</I> into JM110<I>E.coli</I></SPAN><DIV id="collapsiblew3-1" class="collapse box-content"><P><B>Aim of experiment:</B> To transform pSB1C3-<I>LS</I> into JM110 <I>E.coli</I>.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A></P><P><B>Results:</B>N/A </P><P><B>Next Steps:</B>If transformations are successful, overnight cultures will be set up tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">8/9: Overnight Cultures of JM110 <I>E.coli</I> containing pSB1C3-<I>LS</I></SPAN><DIV id="collapsiblew3-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of JM110 <I>E.coli</I> containing pSB1C3-<I>LS</I> from the transformations carried out yesterday.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The overnight cultures will be miniprepped and sent for sequencing tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">9/9: Plasmid Purification of the JM110 <I>E.coli</I> Overnight Cultures containing pSB1C3-<I>LS</I></SPAN><DIV id="collapsiblew3-3" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the JM110 <I>E.coli</I> Overnight Cultures containing pSB1C3-<I>LS</I>.</P><P><B>Protocols Used: </B></P><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> The miniprep with the highest concentration will be sent for sequencing to confirm presence of insert. Sequencing for pQE80-L-<I>LS</I> came back correct so overexpression assays will be started tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/9: Overexpression Assays of <I>LS</I></SPAN><DIV id="collapsiblew3-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overexpression assays of <I>LS</I> using different concentrations of IPTG.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Overexpression Assays</A> Note: Cultures were induced with 0 mM, 0.5 mM, 1 mM and 2 mM IPTG. The membrane for the blot was left blocking overnight.</P><P><B>Results:</B><A data-toggle="modal" data-target="#fp-figure3-modal" class="journal-protocol"> Figure 3 </A></P><P><B>Next Steps:</B> The gel shows a band corresponding to the expected size of LS. However, Western Blotting will confirm if it is indeed LS. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/9: Characterisation of LS</SPAN><DIV id="collapsiblew3-5" class="collapse box-content"><P><B>Aim of experiment:</B> To complete characterisation of LS via Western Blotting.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Westen Blotting</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#fp-figure4-modal" class="journal-protocol"> Figure 4 </A></P><P><B>Next Steps:</B> The blot shows that LS has been successfully detected at the expected size of 80 kDa. There is also partial degradation of the protein. Nevertheless, purification of LS will be started next week. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 14/9/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Purification of lanosterol synthase was started this week.</P></DIV><DIV id="collapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/9: Overnight Cultures of M15pREP4 <I>E.coli</I> containing pQE80-L-<I>LS</I></SPAN><DIV id="collapsiblew4-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To set up overnight cultures of M15pREP4 <I>E.coli</I> containing pQE80-L-<I>LS</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures</A> Note: 150 ml cultures were set up.</P><P><B>Results: </B>N/A</P><P><B>Next Steps:</B> The overnight cultures will be used to innoculate a 3 L day culture tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">15/9: 3 L Day Cultures of M15pREP4 <I>E.coli</I> containing pQE80-L-<I>LS</I></SPAN><DIV id="collapsiblew4-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 3 L day cultures of M15pREP4 <I>E.coli</I> containing pQE80-L-<I>LS</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: The protocol was stopped at Step 6- protein purification will be continued tomorrow. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Affinity chromatography will be carried out tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">16/9: Purification of LS</SPAN><DIV id="collapsiblew4-3" class="collapse box-content"><P><B>Aim of experiment:</B> To continue purification of LS. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: Protocol was stopped at Step 13.</P><P><B>Results:</B><A data-toggle="modal" data-target="#fp-figure5-modal" class="journal-protocol"> Figure 5 </A><A data-toggle="modal" data-target="#fp-figure6-modal" class="journal-protocol"> Figure 6 </A></P><P><B>Next Steps:</B> The SDS gel suggests that the peak observed in the affinity chromatography step may due to a smaller sized protein which has shown up strongly on the gel. However, there is still a band observable around the 75 kDa marker which may be LS. The samples will be pooled together and concentrated. A small sample will be blotted and then size exclusion chromaography can be performed. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="sponsor"><DIV class="container"><DIV class="footer-btn"><A href="https://2015.igem.org/Team:Dundee/Introduction_Video" class="btn btn-info btn-lg" role="button">Watch Our Introduction Video</A></DIV></DIV></SECTION><FOOTER><DIV class=""><DIV class="container"><DIV class="col-md-3"><A href="http://www.dundee.ac.uk/" target="_blank"><H4>Dundee University</H4></A><P>Located on the east coast of Scotland, The University of Dundee is one of Scotland's leading research institutes.</P></DIV><DIV class="col-md-3"><H4>Social Media</H4><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="https://www.youtube.com/channel/UCJHBLvLhjzGqHqdjABjOY-w">youtube.com/DundeeiGEM2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.facebook.com/igemdundee2015">facebook.com/iGEMDundee2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.twitter.com/dundeeigemteam">twitter.com/DundeeiGEMTeam</A></DIV></DIV></DIV></DIV></DIV></FOOTER><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>