Team:Dundee/pmodal
×
PCR Protocol
PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the
section.
Reactants
Volume (µl)
DNA
1
DMSO
2.5
100 µM Forward Primer
0.5
100 µM Reverse Primer
0.5
5 x Herculase Buffer
10
10 mM dNTPs
0.5
Herculase
0.5
H2O
34.5
Elongation time was altered according to the length of the sequences.
Close
×
Restriction Digests Protocol
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the Lab Book section.
Post Plasmid Purification
Post PCR
gBlock
DNA
1 mg
50 µl
100 ng
Cutsmart Buffer (µl)
6
6
6
Water (µl)
As required
1
As required
Enzyme 1 (µl)
1.5
1.5
1.5
Enzyme 2 (µl)
1.5
1.5
1.5
Total Volume (µl)
30
60
30
The digests were incubated at 37oC for 3 hours.
Note that when performing plasmid digestions, 2.5 µl of alkaline phosphatase and 2.5 µl of Cutsmart Buffer was added at the 2 hour and 2.5 hour mark. Digested PCR products were separated from restriction enzymes and buffers using Strataclean resin (Aligent Technologies). Digested vectors were purified on a 1% agarose gel and then extracted.
Close
×
Ligations
Ligation reactions were set up according to the table below.
Vector (µl)
2:1 (Vector: Insert) [µl]
3:1 (Vector: Insert) [µl]
4:1 (Vector: Insert) [µl]
T4 DNA Ligase
1
1
1
1
Ligase Buffer
1
1
1
1
Vector
1
1
1
1
Insert
-
2
3
4
H20
7
5
4
3
The reactions were incubated at room temperature for at least 3 hours.
Close
×
Transformation Protocol
1. 100 µl of chemocompetent E.coli cells were defrosted. Thaw on ice.
2. Agar plates were taken out of 4 oC to warm to room temperature.
3. 10 µl of ligation reaction or 2 µl of plasmid purified DNA was added to cells. The cells and DNA were mixed by gently flicking the tube.
4. Cell and DNA mixture was left on ice for 20 minutes.
5. Mixture was placed in 42 oC water bath for 90 seconds.
6. Mixture was put back on ice for 2 minutes.
7. 1 ml of LB (without antibiotic) was added to cells.
8. Cells were allowed to grow for 1 hour at 37 oC in a shaking incubator.
9. Cells were then spun down and the supernatant discarded.
10. The pellet was resuspended in the 100 µl of remaining media.
11. Cells were plated on agar plates with appropriate antibiotic.
Close
×
Plasmid Purification
1. 1-5 ml bacterial overnight culture was pelleted by centrifugation at >8000 rpm for 3 minutes at room temperature.
2. Pelleted bacterial cells were resuspended in 250 µl of Buffer P1 and transferred to a microcentrifuge tube.
3. 250 µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution becomes clear.
4. 350 µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.
5. The mixture was centrifuged for 10 minutes at 13000 rpm in a microcentrifuge.
6. The supernatant was applied to a QIAprep spin column by pipetting.
7. The QIAprep spin column was washed with 500 µl of Buffer PB by centrifugation for 1 minute and flow through discarded.
8. 750µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute and flow through discarded.
9. The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
10. The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 50 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.
Close
×
Overnight Cultures
1. Four colonies were picked from the desired plate.
2. For each colony, 5 ml of fresh LB plus the appropriate antibiotics was inoculated and grown overnight at 37 oC.
Close
×
SDS-PAGE and Western Blotting
1. 50 µl of an overnight culture was used to inoculate 5 ml of fresh LB containing the appropriate antibiotics and left to grow at 37oC in a     shaking incubator.
2. Once the OD600=0.6-0.8, 1 mM of IPTG was added to the culture and left to grow for a further 3 hours.
3. 1 ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.
4. Meanwhile, 950 µl of 2x Laemmli sample buffer was added to 50 µl of B-mercaptoethanol.
5. The supernatant was discarded from the samples and the pellet resuspended in 200 µl of the above solution.
6. The samples were then boiled at 90oC for 10 minutes.
7. Afterwards, they were spun down for 1 minute in a microcentrifuge.
8. At this stage, the samples could be stored at -20oC or ran on a gel immediately.
9. Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the     stacking gel at which point the voltage was increased to 200V for 20 minutes.
10. One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.
11. After the proteins had been transferred onto the membrane using a semi dry transfer ran for 40 minutes at 5V, the membrane was blocked in 5% Marvel milk overnight at 4oC.
12. The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10 ml of 1xTBS buffer to which the primary anti-His antibody was     added and left for one hour. The protein ladder was cut off before adding the primary antibody.
13. The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10 ml of 1xTBS buffer, to which the secondary antibody was     added and left for one hour.
14. The membrane was then washed in 3x 10 ml 1xTBST buffer for 5 minutes.
15. The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off     at step 12.
16. Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.
Close
×
Growth Curve Assay Protocol
1. 5ml overnight cultures of the appropriate strains were set up.
2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
3. Three were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced.
4. 200 µl of each culture was loaded onto a 96 well microtitre plate with 4 repeats.
5. The cultures were left to grow for 16 hours in the plate reader which took OD readings every 10 minutes.
Close
×
Growth Curve Assay Protocol
1. 5ml overnight cultures of the appropriate strains were set up.
2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
3. 20 x 200 µl aliquots were loaded onto a 96 well microtitre plate.
4. After the OD600=~0.6, three cultures were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced (with 4 repeats).
5. The cultures were left to grow for 20 hours in the plate reader which took OD readings every 30 minutes.
Close
×
Protein Purification Protocol
1. A 150 ml overnight culture was set up for the desired protein.
2. 3 x 1 L of fresh LB containing 1 ml of appropriate antibiotics was inoculated with each 50 ml overnight culture and left to grow until an OD600 =0.6-1 in a shaking incubator at 37oC.
3. The cultures were subsequently induced with 1 mM IPTG and left to grow for 6 hours in a shaking incubator at 20oC.
4. They were then centrifuged for 20 minutes at 4200 rpm at 4oC.
5. The pellets were then resuspended in 5 ml of ice cold 50 mM 7.5 Tris and transferred to a 50 ml falcon tube and then topped up to 40 ml with 50 mM 7.5 Tris.
6. They were then spun down again at 4000 rpm for 15 minutes at 4oC, the supernatant discarded and pellets stored at -20oC.
7. Once purification could proceed, the pellets were thawed on ice and resuspended in 40 ml of 50 mM Tris containing DNAase, 1 x protease inhibitor tablet     and lysozyme. (Note: If purifying a membrane protein, 1% DDM and 5 M urea was also added).
8. The cells were lysed using a cell disruptor.
9. Afterwards the lysate was spun down in a J25.50 rotor at 15000 rpm at 4oC for 40 minutes.
10. The supernatant was removed and transferred to a 50 ml falcon tube. The pellet was kept for testing. Both were kept on ice.
11. Using FPLC, the supernatant was loaded onto a Ni2+ column.
12. The protein was eluted using an increasing concentration of imidazole. The two buffers used were Buffer A: 50 mM Tris, 200 mM NaCl, 20 mM imidazole and     Buffer B: 50 mM Tris, 200 mM NaCl, 1 M Imidazole. (Note: If purifying a membrane protein 0.02% DDM was added to these buffers.)
13. The fractions corresponding to any peaks were retained and concentrated using an appropriate filter size. A small amount of each fraction could be ran     on an SDS gel at this stage.
14. Using FPLC, the sample was then loaded onto a SEC column and any fractions corresponding to a peak were ran on an SDS gel to check for the presence of     the desired protein.
15. Fractions containing the desired protein were collated, to which 20% glycerol was added.
16. The samples were then flash frozen and stored at -80oC.
Close
×
Assay Preparation
Take 3 independent colonies from each plate of your controls and samples and place each colony into 5 ml of LB with the appropriate antibiotics.
Leave to grow overnight at 30°C.
Take 50 μl of each overnight and inoculate 5 ml of fresh LB containing the appropriate antibiotics. Do this twice for every independent colony.
Allow these to grow at 30°C shaker until the O.D of each inoculate is between 0.3-0.5
Once the O.D is between 0.3-0.5 take 3 x 1 ml samples of each culture and place into separate Eppendorf tubes.
Take a further 1 ml and place into a cuvette and  subsequently measure the OD600 marking down the OD for each replica.
Spin down the 1 ml cultures of each replica at 13,000 rpm for 3 minutes.
Remove the supernatant from each 1 ml culture and freeze cell pellet for later use in the β-galactosidase Assay.
Close
×
β-Galactosidase Assay
Resuspend cell pellet in 800 μl of buffer 2 (100 mM dibasic sodium phosphate (Na2HPO4); 20 mM KCl ; 2 mM MgSO4 ;0.8 mg/mL CTAB (hexadecyltrimethylammonium bromide) ; 0.4 mg/mL sodiumdeoxycholate) and subsequntly add 5.4 μL/mL beta-mercaptoethanol.
Add 50 μl of SDS (0.1%) to each sample.
Add 50 μl of Chloroform to each sample and vortex for 30 seconds.
Make substrate solution (60 mM Na2HPO4; 40 mM NaH2PO4; 1 mg/mL o-nitrophenyl-β-D-Galactoside (ONPG); 2.7 μL/mL β-mercaptoethanol)
Add 200 μl of substrate solution to each sample marking down the time it was added in relation to the first sample.
Wait for an observable color change and mark down the time that the color change occurred.
Add 600 μl of NaCO3(1M) once the color change has occurred.
Spin down the samples at 13,000 rpm for 3 minutes and place the supernatant into a cuvette.
Measure the OD420.
Close
×
Plate Reader Protocol
A 5 ml overnight culture of the desired strain was grown at 37 oC.
Equalise OD600 if required. Dilute culture 1:1000.
Aliquot 198 µl into the required amount of wells in a 96-well plate.
Add additional substances and controls as required.
Preheat platereader to required temperature and start experiment.
Retrieve and analyse data.
Close
×
Competent Cells Protocol
A 5 ml overnight culture of the desired strain was grown at 37oC.
Subsample 1/10 of the volume of the subculture. I.e. 1 ml of the overnight culture into 10 ml fresh LB.
Grow subculture at at 37 oC for 2 h.
Spin down subculture at 4000 rpm at 4oC for 10 min.
Resuspend the pellet in ice cold transformation buffer. Use 1/10 of the initial volume of the subculture.
Use 100 µl per transformation.
Close
×
Plating Protocol
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
Fresh agar plates were inoculated with cells on the toothpick.
Agar plates were incubated at 4oC over night.
Close
×
Tryptophan Fluorescence Protocol
1. Intensity of tryptophan fluorescence of PotD was measured on a fluorimeter using an excitation of 280 nm and emission of 350 nm.
2. Increasing concentrations of spermidine were added in 50 µM increments to give a final concentration of 250 µM spermidine to 5 µM of purified PotD.
Close
×
Sequencing Protocol
For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
Volume of sample equivalent to 550ng DNA
2 µl of 3 mM primer
Remaining volume of water for a total volume of 30 µl
The samples were submitted to DNA sequencing and services .
Close
×
Overnight Culture Protocol
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
5 ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.
Tubes were incubated at 37oC in a rotary incubator at 200rpm over night.
Close
×
Gel Extraction Protocol
The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.
The gel slice was transferred into a microcentrifuge tube. 700 µl of Buffer QG was added.
The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.
After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.
230 µl of isopropanol was added to the sample and mixed.
A QIAquick spin column was placed in a 2 ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
500 µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
750 µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.
Close
