Team:Dundee/Lab Book/Chromate Detector
×
PCR Protocol
PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the
section.
Reactants
Volume (µl)
DNA
1
DMSO
2.5
100 µM Forward Primer
0.5
100 µM Reverse Primer
0.5
5 x Herculase Buffer
10
10 mM dNTPs
0.5
Herculase
0.5
H2O
34.5
Elongation time was altered according to the length of the sequences.
Close
×
Restriction Digests Protocol
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the Lab Book section.
Post Plasmid Purification
Post PCR
gBlock
DNA
1 mg
50 µl
100 ng
Cutsmart Buffer (µl)
6
6
6
Water (µl)
As required
1
As required
Enzyme 1 (µl)
1.5
1.5
1.5
Enzyme 2 (µl)
1.5
1.5
1.5
Total Volume (µl)
30
60
30
The digests were incubated at 37oC for 3 hours.
Note that when performing plasmid digestions, 2.5 µl of alkaline phosphatase and 2.5 µl of Cutsmart Buffer was added at the 2 hour and 2.5 hour mark. Digested PCR products were separated from restriction enzymes and buffers using Strataclean resin (Aligent Technologies). Digested vectors were purified on a 1% agarose gel and then extracted.
Close
×
Ligations
Ligation reactions were set up according to the table below.
Vector (µl)
2:1 (Vector: Insert) [µl]
3:1 (Vector: Insert) [µl]
4:1 (Vector: Insert) [µl]
T4 DNA Ligase
1
1
1
1
Ligase Buffer
1
1
1
1
Vector
1
1
1
1
Insert
-
2
3
4
H20
7
5
4
3
The reactions were incubated at room temperature for at least 3 hours.
Close
×
Transformation Protocol
1. 100 µl of chemocompetent E.coli cells were defrosted. Thaw on ice.
2. Agar plates were taken out of 4 oC to warm to room temperature.
3. 10 µl of ligation reaction or 2 µl of plasmid purified DNA was added to cells. The cells and DNA were mixed by gently flicking the tube.
4. Cell and DNA mixture was left on ice for 20 minutes.
5. Mixture was placed in 42 oC water bath for 90 seconds.
6. Mixture was put back on ice for 2 minutes.
7. 1 ml of LB (without antibiotic) was added to cells.
8. Cells were allowed to grow for 1 hour at 37 oC in a shaking incubator.
9. Cells were then spun down and the supernatant discarded.
10. The pellet was resuspended in the 100 µl of remaining media.
11. Cells were plated on agar plates with appropriate antibiotic.
Close
×
Plasmid Purification
1. 1-5 ml bacterial overnight culture was pelleted by centrifugation at >8000 rpm for 3 minutes at room temperature.
2. Pelleted bacterial cells were resuspended in 250 µl of Buffer P1 and transferred to a microcentrifuge tube.
3. 250 µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution becomes clear.
4. 350 µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.
5. The mixture was centrifuged for 10 minutes at 13000 rpm in a microcentrifuge.
6. The supernatant was applied to a QIAprep spin column by pipetting.
7. The QIAprep spin column was washed with 500 µl of Buffer PB by centrifugation for 1 minute and flow through discarded.
8. 750µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute and flow through discarded.
9. The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
10. The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 50 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.
Close
×
Overnight Cultures
1. Four colonies were picked from the desired plate.
2. For each colony, 5 ml of fresh LB plus the appropriate antibiotics was inoculated and grown overnight at 37 oC.
Close
×
SDS-PAGE and Western Blotting
1. 50 µl of an overnight culture was used to inoculate 5 ml of fresh LB containing the appropriate antibiotics and left to grow at 37oC in a     shaking incubator.
2. Once the OD600=0.6-0.8, 1 mM of IPTG was added to the culture and left to grow for a further 3 hours.
3. 1 ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.
4. Meanwhile, 950 µl of 2x Laemmli sample buffer was added to 50 µl of B-mercaptoethanol.
5. The supernatant was discarded from the samples and the pellet resuspended in 200 µl of the above solution.
6. The samples were then boiled at 90oC for 10 minutes.
7. Afterwards, they were spun down for 1 minute in a microcentrifuge.
8. At this stage, the samples could be stored at -20oC or ran on a gel immediately.
9. Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the     stacking gel at which point the voltage was increased to 200V for 20 minutes.
10. One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.
11. After the proteins had been transferred onto the membrane using a semi dry transfer ran for 40 minutes at 5V, the membrane was blocked in 5% Marvel milk overnight at 4oC.
12. The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10 ml of 1xTBS buffer to which the primary anti-His antibody was     added and left for one hour. The protein ladder was cut off before adding the primary antibody.
13. The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10 ml of 1xTBS buffer, to which the secondary antibody was     added and left for one hour.
14. The membrane was then washed in 3x 10 ml 1xTBST buffer for 5 minutes.
15. The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off     at step 12.
16. Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.
Close
×
Growth Curve Assay Protocol
1. 5ml overnight cultures of the appropriate strains were set up.
2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
3. Three were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced.
4. 200 µl of each culture was loaded onto a 96 well microtitre plate with 4 repeats.
5. The cultures were left to grow for 16 hours in the plate reader which took OD readings every 10 minutes.
Close
×
Growth Curve Assay Protocol
1. 5ml overnight cultures of the appropriate strains were set up.
2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
3. 20 x 200 µl aliquots were loaded onto a 96 well microtitre plate.
4. After the OD600=~0.6, three cultures were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced (with 4 repeats).
5. The cultures were left to grow for 20 hours in the plate reader which took OD readings every 30 minutes.
Close
×
Protein Purification Protocol
1. A 150 ml overnight culture was set up for the desired protein.
2. 3 x 1 L of fresh LB containing 1 ml of appropriate antibiotics was inoculated with each 50 ml overnight culture and left to grow until an OD600 =0.6-1 in a shaking incubator at 37oC.
3. The cultures were subsequently induced with 1 mM IPTG and left to grow for 6 hours in a shaking incubator at 20oC.
4. They were then centrifuged for 20 minutes at 4200 rpm at 4oC.
5. The pellets were then resuspended in 5 ml of ice cold 50 mM 7.5 Tris and transferred to a 50 ml falcon tube and then topped up to 40 ml with 50 mM 7.5 Tris.
6. They were then spun down again at 4000 rpm for 15 minutes at 4oC, the supernatant discarded and pellets stored at -20oC.
7. Once purification could proceed, the pellets were thawed on ice and resuspended in 40 ml of 50 mM Tris containing DNAase, 1 x protease inhibitor tablet     and lysozyme. (Note: If purifying a membrane protein, 1% DDM and 5 M urea was also added).
8. The cells were lysed using a cell disruptor.
9. Afterwards the lysate was spun down in a J25.50 rotor at 15000 rpm at 4oC for 40 minutes.
10. The supernatant was removed and transferred to a 50 ml falcon tube. The pellet was kept for testing. Both were kept on ice.
11. Using FPLC, the supernatant was loaded onto a Ni2+ column.
12. The protein was eluted using an increasing concentration of imidazole. The two buffers used were Buffer A: 50 mM Tris, 200 mM NaCl, 20 mM imidazole and     Buffer B: 50 mM Tris, 200 mM NaCl, 1 M Imidazole. (Note: If purifying a membrane protein 0.02% DDM was added to these buffers.)
13. The fractions corresponding to any peaks were retained and concentrated using an appropriate filter size. A small amount of each fraction could be ran     on an SDS gel at this stage.
14. Using FPLC, the sample was then loaded onto a SEC column and any fractions corresponding to a peak were ran on an SDS gel to check for the presence of     the desired protein.
15. Fractions containing the desired protein were collated, to which 20% glycerol was added.
16. The samples were then flash frozen and stored at -80oC.
Close
×
Assay Preparation
Take 3 independent colonies from each plate of your controls and samples and place each colony into 5 ml of LB with the appropriate antibiotics.
Leave to grow overnight at 30°C.
Take 50 μl of each overnight and inoculate 5 ml of fresh LB containing the appropriate antibiotics. Do this twice for every independent colony.
Allow these to grow at 30°C shaker until the O.D of each inoculate is between 0.3-0.5
Once the O.D is between 0.3-0.5 take 3 x 1 ml samples of each culture and place into separate Eppendorf tubes.
Take a further 1 ml and place into a cuvette and  subsequently measure the OD600 marking down the OD for each replica.
Spin down the 1 ml cultures of each replica at 13,000 rpm for 3 minutes.
Remove the supernatant from each 1 ml culture and freeze cell pellet for later use in the β-galactosidase Assay.
Close
×
β-Galactosidase Assay
Resuspend cell pellet in 800 μl of buffer 2 (100 mM dibasic sodium phosphate (Na2HPO4); 20 mM KCl ; 2 mM MgSO4 ;0.8 mg/mL CTAB (hexadecyltrimethylammonium bromide) ; 0.4 mg/mL sodiumdeoxycholate) and subsequntly add 5.4 μL/mL beta-mercaptoethanol.
Add 50 μl of SDS (0.1%) to each sample.
Add 50 μl of Chloroform to each sample and vortex for 30 seconds.
Make substrate solution (60 mM Na2HPO4; 40 mM NaH2PO4; 1 mg/mL o-nitrophenyl-β-D-Galactoside (ONPG); 2.7 μL/mL β-mercaptoethanol)
Add 200 μl of substrate solution to each sample marking down the time it was added in relation to the first sample.
Wait for an observable color change and mark down the time that the color change occurred.
Add 600 μl of NaCO3(1M) once the color change has occurred.
Spin down the samples at 13,000 rpm for 3 minutes and place the supernatant into a cuvette.
Measure the OD420.
Close
×
Plate Reader Protocol
A 5 ml overnight culture of the desired strain was grown at 37 oC.
Equalise OD600 if required. Dilute culture 1:1000.
Aliquot 198 µl into the required amount of wells in a 96-well plate.
Add additional substances and controls as required.
Preheat platereader to required temperature and start experiment.
Retrieve and analyse data.
Close
×
Competent Cells Protocol
A 5 ml overnight culture of the desired strain was grown at 37oC.
Subsample 1/10 of the volume of the subculture. I.e. 1 ml of the overnight culture into 10 ml fresh LB.
Grow subculture at at 37 oC for 2 h.
Spin down subculture at 4000 rpm at 4oC for 10 min.
Resuspend the pellet in ice cold transformation buffer. Use 1/10 of the initial volume of the subculture.
Use 100 µl per transformation.
Close
×
Plating Protocol
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
Fresh agar plates were inoculated with cells on the toothpick.
Agar plates were incubated at 4oC over night.
Close
×
Tryptophan Fluorescence Protocol
1. Intensity of tryptophan fluorescence of PotD was measured on a fluorimeter using an excitation of 280 nm and emission of 350 nm.
2. Increasing concentrations of spermidine were added in 50 µM increments to give a final concentration of 250 µM spermidine to 5 µM of purified PotD.
Close
×
Sequencing Protocol
For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
Volume of sample equivalent to 550ng DNA
2 µl of 3 mM primer
Remaining volume of water for a total volume of 30 µl
The samples were submitted to DNA sequencing and services .
Close
×
Overnight Culture Protocol
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
5 ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.
Tubes were incubated at 37oC in a rotary incubator at 200rpm over night.
Close
×
Gel Extraction Protocol
The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.
The gel slice was transferred into a microcentrifuge tube. 700 µl of Buffer QG was added.
The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.
After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.
230 µl of isopropanol was added to the sample and mixed.
A QIAquick spin column was placed in a 2 ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
500 µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
750 µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.
Close
×
Figure 1: Amplification of LbpA The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 2: LbpA PCR. Lane 1: PCR Product, Lane 2: 1kb Ladder and Marker. The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 3: LbpA PCR for Cloning into pQE80-L. Lane 1: LbpA PCR Product, Lane 2: 1kb Ladder and Marker The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 4: LbpA Culture Samples. Lane 1: Sample from uninduced culture. Lane 2: Sample from culture induced with 1mM IPTG. It seems that on comparison of the two cultures, that inducing expression of LbpA causes the cells to die given the significant reduction of protein levels visible on the gel.
Close
×
Figure 5: OD600 Readings from Uninduced Culture and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG). The table shows that after being induced with IPTG the cultures stop growing since their OD600 readings are notably lower than that of the uninduced control.
Close
×
Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, the induced cultures stop growing when LbpA expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 7: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, after reaching an OD600 of 0.6,  the induced cultures stop growing when LbpA starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have dealt with the production of the foreign protein.
Close
×
Figure 8: SDS Gel of Samples Taken from Uninduced Control and 1 mM IPTG Induced Culture 6 hours After Induction For the induced culture, a faint band just above the 100 kDa marker is observable on the gel which corresponds to the expected size of LbpA.
Close
×
Figure 1: Gel of SBP PCR Product for Cloning into pQE80-L. Lane 1: SBP PCR Product 1 Lane 2: 1kb Ladder and Marker Lane 3: SBP PCR Product 2 Both bands observable on the gel correspond to the expected size of SBP which is ~600 bp.
Close
×
Figure 2: SDS Gel of Uninduced and 1 mM IPTG Induced Cultures of PotD and SBP. For PotD, it is clear from the gel that it is being successfully overexpressed since there is an intense band present at the 37 kDa marker which corresponds to the expected size of PotD in the induced culture sample. For SBP, there is no band present which suggests SBP is being overexpressed.
Close
×
Figure 3: Characterisation of PotD. Single colonies of MC1061 E.coli containing pQE80-L encoding PotD were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin. Once cultures reached an OD600 of 0.6, one was induced with 1 mM IPTG and one was left uninduced as a control. They were left to grow for a further three hours, after which a 1ml sample was taken and the cells pelleted. The cells were resuspended in 100µl of Laemmli buffer, 20µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 4: SDS Gel of Uninduced and 0.5mM, 1mM and 2mM IPTG Induced Cultures of SBP. It appears that upon induction of SBP with any concentration of IPTG the cells are dying given that there is a significant drop in protein levels observable on the gel.
Close
×
Figure 5: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, the induced cultures stop growing when SBP expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, after reaching an OD600 of 0.6,  the induced cultures stop growing when SBP starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 7: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of PotD 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100 V. The two bands observable on the gel correspond to the expected size of PotD of 38 kDa. Western Blotting will confirm if these proteins are PotD.
Close
×
Figure 8: Characterisation of PotD following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 9: Characterisation of SBP following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 10: Characterisation of PotD/Spermidine Interaction. The graph shows that the change in the tryptophan spectra increases as concentration of spermidine increases.
Close
×
Figure 1: pSB1C3-hHBA and pSB1C3-hHBB (respectively) Pre-Sequencing Digest pSB1C3-hHBA and pSB1C3-hHBB minipreps were digested with EcoRI and PstI to check for presence of each insert. The bands observable on the gel correspond to the expected sizes for the pSB1C3 vector and both inserts.
Close
×
Figure 2: Amplification of hHBA and hHBB hHBA and hHBB were amplified from the pSB1c3-hHBA and pSB1c3-hHBB minipreps obtained yesterday using primers to allow cloning of the genes into pT25 and pUT18 respectively. Although the band is fainter in the case of hHBA, the gel indicates the PCR has been successful.
Close
×
Figure 3: Amplification of hHBA hHBA was amplified from the pSB1C3-hHBA using primers to allow cloning of the genes into pT25. The gel indicates that the PCR has been successful.
Close
×
Figure 4: Amplification of hHBA and hHBB hHBA and hHBB were amplified from the pSB1c3-hHBA and pSB1C3-hHBB minipreps using primers to allow cloning of the genes into pQE80-L. The gel indicates that the PCR has been successful.
Close
×
Figure 5: Pre-Sequence Digest of pQE80-L-hHBA and pQE80-L-hHBB Both pQE80-L-hHBA and pQE80-L-hHBB minipreps were digested with BamHI and KpnI to check for the presence of the inserts. The gel indicates that both inserts have been successfully cloned into pQE80-L.
Close
×
Overexpression of hHBB and hHBA. Single colonies of M15[pREP4] E.coli containing pQE80-L encoding hHBB and pQE80-L encoding hHBA were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD600 of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of 2x Laemmli buffer, 20 µl of which was separated by SDS-PAGE.
Close
×
Figure 7: Characterisation of hHBB. Single colonies of M15[pREP4] E.coli containing pQE80-L encoding hHBB were used to inoculate 5 ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD600 of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 8: Amplification of hHBN for Cloning into pQE80-L PCR product from amplification of hHBN from IDT plasmid using primers for cloninig into pQE80-L. The gel indicates hHBN has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.
Close
×
Figure 9: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of hHBB 10 µl of each fraction was mixed with 10 µl of laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. The four bands observable on the gel correspond to the expected size of hHBB of 16 kDa. Western Blotting will confirm if these proteins are hHBB.
Close
×
Figure 10: Characterisation of hHBB following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 11: Amplification of hHBN for Cloning into pT25 PCR product from amplification of hHBN from IDT plasmid using primers for cloninig into pT25. The gel indicates hHBN has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.
Close
×
Figure 14: Characterisation of hHBN following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 15: Amplification of hHBN hHBN was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of hHBN of 1200 bp.
Close
×
Figure 15: Amplification of hHBA hHBA was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of hHBA of
Close
×
Figure 16: Amplification of hHBN hHBN was amplified using primers to allow for subsequent cloning into the two hybrid vector pUT18. The band observable on the gel corresponds to the expected size of hHBN of 1200 bp.
Close
×
Figure 1: Amplification of OBP2A for insertion into pSB1C3 vector . This is the PCR product from the amplification of OBP2A from the IDT plasmid using primers for cloning into pSB1C3. The gel indicates OBP2A has been amplified successfully since the observed band corresponds to the expected size of 500 bp. The concentration of the gel extracted OBP2A is 335.53 ng/ul.
Close
×
Figure 2: A subsequent gel was run of the amplified OBP2A after it was first excised from the gel in Figure 1 This was to determine if the gel extraction was successful. The gel indicates that OBP2A has been extracted from the gel successfully, since the observed band corresponds to the expected size of 500 bp. This can now be restricted in preparation for ligation into pSB1C3.
Close
×
Figure 3: A Gel Analysis of Restriction Digest from the 24/6. This gel analysis was done to determine if the reason for the failed transformation was an improperly digested OBP2A. This gel indicates that the restricted product hasn't been lost as the OBP2A gene fragment can be seen as a distinct band at 500 bp. A re-ligation of OBP2A into the pSB1C3 backbone will now occur.
Close
×
Figure 4: Presequence Digest of pSB1C3-OBP2A. All seven plasmid purifications of the overnight cultures done on the 26/6 were digested to see if they contained the OBP2A insert. It is clear from the gel that all but sample 3 contain the insert for OBP2A as they all have faint bands at 500 bp's which corresponds to the size of OBP2A. It also appears that sample 5 contains the highest concentration, so that will be the sample that is sent for sequencing.
Close
×
Figure 5: Presequence digest of pUT18-OBP2A To determine if any samples contain the OBP2A insert. It can be seen from this gel that of the  5 samples that were digested only two are show to have the OBP2A insert, therefore sample 2 will be sent for sequencing.
Close
×
Figure 6: Amplification of OBP2A for insertion into pT25 vector . This is the PCR product from the amplification of OBP2A for cloning into pT25. The gel indicates OBP2A has been amplified successfully since the observed band corresponds to the expected size of 100 bp.
Close
×
Figure 7: Presequence Digest of pT25-OBP2A In order to determine if any of the samples contain the OBP2A insert. It can be seen from this gel that samples 5, 6 and 7 may have inserts in them as they each have two distinct bands, one of which corresponds to the 100 bp size for the OBP2A for pT25. As a result all three samples will be sent for sequencing.
Close
×
Figure 8: β-galactosidase Assay A graph of the results from the first attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of OBP2A. As it can be seen that the positive result has failed(far left) not much can be inferred from the interaction of the two parts of OBP2A. However it looks like the two subunits of OBP2A may not be interacting. As this was a first attempt, subsequent assays will be performed in order to determine if in fact the two separated parts of OBP2A aren't interacting in vivo - like these results suggests.
Close
×
Figure 9: β-galactosidase Assay A graph of the results from the second attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of OBP2A.  It can also be inferred from this graph that the two parts of OBP2A don't seem to be interacting as it has a low millers activity when compared to the controls. Further assays will be performed in order to determine if the two separated parts of OBP2A aren't interacting in vivo like these results suggests.
Close
×
Figure 11: β-galactosidase Assay Under Anaerobic Conditions A graph of the results from the anaerobic β-galactosidase assay aimed to characterize the interaction between the two separated parts of OBP2A. The graph seems to suggest that the two subunits of OBP2A may not be interacting in vivo. This can be inferred from the fact that the millers activity of the OBP2A sample lies lower than the negative controls from within the experiment. As a result of this information, one more assay will be prepared in which the two subunits will be switched with regards to the bacterial two hybrid vectors to eliminate any possibility of this being a factor effecting the interaction between the two subunits.
Close
×
Figure 1: Amplification of LS LS was amplified using primers which would allow for subsequent cloning into the high expression vector pQE80-L. Lane 1 shows that the PCR has been successful since the band corresponds to the expected size of 2100 bp.
Close
×
Figure 2: Amplification of LS LS was amplified from the IDT plasmid using primers to allow cloning of the gene into pSB1C3. The observed band corresponds to the expected size of LS, 2100 bp.
Close
×
Figure 3: Overexpression Assays of LS 5 ml cultures of M15pREP4 E.coli was grown to OD600=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37oC, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel.
Close
×
Figure 4: Characterisation of Lanosterol Synthase 5 ml cultures of M15pREP4 E.coli was grown to OD600=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37oC, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. The proteins were transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 5: Chromatogram following Affinity Chromatography of LS Using FPLC, the protein sample was loaded onto a nickel column and eluted with an increasing concentration of imidazole (green line). The fractions corresponding to the peak were ran on an SDS gel.
Close
×
Figure 6: SDS Gel of Peak Fractions 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer an loaded onto a SDS gel.
Close
×
Figure 1: Alignment of BBa_K1058008 . Sequences from top to bottom: sequencing result from forward primer; expected sequence; reverse complement of sequencing result from reverse primer; sequencing result from intermediate primer. Colour code: Green = Prefix/Suffix; Pink = Annealing site of intermediate primer; Yellow = Mismatch in one sequence; Red = In frame stop codons.
'
Close
×
Figure 2: 1% agarose gel after PCR of BBa_K1058008 with primers specific to the promoter region, Pchr, and the repressor region, chrB. Left: Pchr (expected size 169bp); Middle: 1kb+ ladder, Right: chrB (expected size 939bp). Amplification of chrB was not successful. Pchr was further processed by restriction digest with EcoRI and PstI.
'
Close
×
Figure 3: 1% agarose gel after PCR of BBa_K1058008 with primers specific to the repressor region, chrB. Left: 1.5 µl DMSO; Middle: 1kb+ ladder, Right: 2.5 µl DMSO. Expected size for each: 939bp. The annealing temperature for the PCR amplification was lowered to 50°C in order to increase the chances of accomodating a primer that is not 100% complementary.
'
Close
×
Figure 4: 1% agarose gel after PCR of gfp with corresponding primers. Left: GFP (expected size 717bp); Middle: 1kb+ ladder, Right: PCR product of chrB, using primers to add a BamHI restriction site in front of it and a 6-His-Tag and a HindIII restriction site at the end. (expected size of chrB: 939bp). Since amplification of chrB was unsuccessful, it will be repeated.
'
Close
×
Figure 5: 1% agarose gel after restriction digest of pSB1C3-Pchr, pSB1C3-chrB (s), and pSB1C3-chrB (w) with EcoRI and PstI for confirmation of size. Ladder: 1kb+. Top: 1-4: Pchr 1-4; chrB (s) 1-2. Bottom: chrB (s) 3-4; chrB (w) 1-4. Expected sizes: Pchr: 169bp; chrB: 939bp.
'
Close
×
Figure 6: Alignment of Pchr sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green = Prefix/Suffix
'
Close
×
Figure 7: Alignment of chrB (opt) sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green: Prefix and Suffix; Red: In frame stop codons; Blue: First 15 codons optimised for expression in E. coli.
'
Close
×
Figure 8: 1% agarose gel after restriction digest of pUniprom (Expected size 2454bp).
'
Close
×
Figure 11: 1% agarose gel after restriction digest of pSB1C3 with EcoRI and SpeI (Expected size 2070bp).
'
Close
×
Figure 9: 1% agarose gel after PCR of chrB (opt). (Expected size 939bp).
'
Close
×
Figure 10: Plates after transformation of Top: pUnipromchrB, Middle: pUnipromchrB (opt) and Bottom: pSB1C3-Pchr-gfp into JM110.
'
Close
×
Figure 12: 1% agarose gel after restriction digest of pUnipromchrB, pUnipromchrB (opt) and pSB1C3-Pchr-gfp for confirmation of size. Ladder: 1kb+. Top: 1-4: pSB1C3-Pchr-gfp 1-4; pUnipromchrB (opt) 1-2. Bottom: pUnipromchrB (opt) 3-4; pUnipromchrB 1-4. Expected sizes: Pchr-gfp: 909bp; chrB and chrB (opt): 939bp.
'
Close
×
Figure 13: 1% agarose gel after colony PCR of pUnipromchrB (opt). Top: colonies 1-11; Bottom: colonies 12-22. Colonies 2, 4, 9, and 12 were selected for sequencing. Ladder: 1kb+ Expected size of chrB (opt): 939bp.
'
Close
×
Figure 14: Sequencing result of pSB1C3-Pchr-gfp after double ligation. Even though the sequence is as expected, it is ligated in reverse. It was hence called pSB1C3-Pchr-gfp (A) The Result for pSB1C3-Pchr-gfp (colony 2) was not as expected, albeit in the correct orientation. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 15: Plates with pSB1C3-Pchr-gfp in JM110 after double ligation under UV light. Numbered colonies were selected for initial size confirmation and sequencing. However, other colonies seemed to express GFP as well, hence a second attempt was made with different colonies from the same plate.
'
Close
×
Figure 16: Alignment of sequencing results of pSB1C3-Pchr-gfp - colonies 1, 5, 6, and 7 with sequences of gfp (mut2) and Pchr sequence. Colony 1 remained consistent, the other tested colonies were without success. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 17: 1% Agarose gel after restriction digest of pUniprom with BamHI/PstI (expected size: 2454bp). Right: PCR product of Haptoglobin.
'
Close
×
Figure 18: Plates after transformation of Pchr into pSB1C3. Top: ratio insert:vector 2:1; Bottom: ratio insert:vector 3:1.
'
Close
×
Figure 19: Alignment of Pchr sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green = Prefix/Suffix
'
Close
×
Figure 20: Plates after transformation of, Top: chrB into pUniprom; Bottom: chrB (opt) into pUniprom. Ratio insert:vector for ligations: left: 3:1; Right: 4:1.
'
Close
×
Figure 21: Gel after restriction digest of Left: pUniprom-chrB, and Right: pUniprom-chrB (opt) with BamHI/PstI for confirmation of insert size. Expected size 939bp. chrB colony 2 and chrB (opt) colony 3 were selected for sequencing.
'
Close
×
Figure 22: Alignments of: Left: chrB and Right: chrB (opt). Colours: Pink = Restriction sites (BamHI/PstI); Blue = 6-His Tag; Red = In-frame stop codons; Cyan = optimised codons.
'
Close
×
Figure 23: Plates after ligation of gfp into pSB1C3-Pchr. Left: Ratio Insert:Vector 2:1; Right: Ratio Insert:Vector 3:1.
'
Close
×
Figure 24: Gel for size confirmation of pSB1C3-Pchr-gfp (Top). Bottom: Left - Haptoglobin, Right - Lanosterol Synthase. pSB1C3-Pchr-gfp (3) and pSB1C3-Pchr-gfp (4) were submitted for sequencing.
'
Close
×
Figure 25: Alignment of Pchr sequence retrieved from BBa_K1058008 and gfp (mut2) with the forward and reverse sequencing results of Pchr-gfp this region. The sequence was as expected, it was hence called pSB1C3-Pchr-gfp (B). Colour code: Green = Prefix/Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 26: Top: anti-GFP: Single colonies of JM110 + pSB1C3-Pchr-gfp (A), MC1061 + pSB1C3-Pchr-gfp (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of each sample was mixed with 100 µl laemmli buffer, and boiled for 10min. 3 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and PmanA-gfp as a positive control.          Bottom: anti-6-His - ChrB: Single colonies of JM110 pUniprom-chrB and JM110 pUniprom-chrB (opt) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 200 µl laemmli buffer, and the sample boiled for 10min. 10 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and hydA - 6-His as a positive control.         Expected sizes: GFP ~ 27kDa, ChrB ~35kDa.
'
Close
×
Figure 27: Plates after transformation of single constituents of chromate sensor into E. coli MG1655. From top to bottom: pUniprom-chrB (from colony 2 - 29/07), pUniprom-chrB (opt) (from colony 3 - 29/07), pSB1C3-Pchr-gfp (B), pSB1C3-Pchr-gfp (A), BBa_K1058008 .
'
Close
×
Figure 28: Western blot against GFP: Single colonies of MG1655, containing all 4 combinations, pUniprom-chrB - pSB1C3-Pchr-gfp (A), pUniprom-chrB - pSB1C3-Pchr-gfp (B), pUniprom-chrB (opt) - pSB1C3-Pchr-gfp (A), and pUniprom-chrB (opt) - pSB1C3-Pchr-gfp (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol and 100 µg/ml ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml PBS. 100 µl of these samples was mixed with 100 µl laemmli buffer, and the sample boiled for 10 min. The indicated volume of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3, and pUniprom were included as a negative control, and PmanA-gfp as a positive control. The blot against 6-His – ChrB was without success and is not shown. No chromate of any kind was added to the medium previous to this blot, hence no expression of GFP was expected.
'
Close
K2CrO4, K2Cr2O7, CrCl3, and K2SO4
×
Figure 29: F,OD-plot of results of plate reader experiment after exposing MG1655 + BBa_K1058008 to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 30: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 31: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt) to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 32: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 33: Plates after ligation of chrB (Bielefeld) into pT7KS. From top to bottom: Ratio insert:vector 2:1; Ratio Insert:Vector 3:1; pT7KS religation control.
'
Close
×
Figure 34: Plates after transformation of single constituents of chromate sensor into BL21 (DE3). From top to bottom: pSB1C3-Pchr-gfp (B) + pUniprom-chrB, pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt), pUniprom-chrB, and pUniprom-chrB (opt).
'
Close
×
Figure 35: 1% agarose gel after colony PCR of pT7KS-chrB (Bielefeld). Top: colonies 1-6; Bottom: colonies 7-12. Ladder: 1kb+ Expected size of chrB (Bielefeld): 939bp
'
Close
×
Figure 36: F,OD-plot of results of plate reader experiment after inducing expression of ChrB from the T7 promoter of pUniprom BL21 (DE3) pSB1C3-Pchr-gfp (B) + pUniprom-chrB, and pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt). IPTG was added after 4 hours. One measurement was made every 10min across 22h.
'
Close
  University of Dundee
 Home
 About Us
Meet the Team
Attributions
 Project
Abstract
Project Overview
Devices
FluID
Fingerprint Ageing
Chromate Biosensor
Parts
Medals
Future Work
 Wet Lab
Lab Book
FluID
Fingerprints
Chromate Biosensor
Useful Information
Protocols
Sequences
Bacterial Two Hybrid
Safety
 Dry Lab
Introduction
FluID
Fingerprint Ageing
Chromate Biosensor
Appendix : Code
 Outreach
Practices
Ethics
Collaborations
Chromate Biosensor
Lab Book
FluID
FluID is a cell free, all in one spray which will be used to detect and dinstiguish between body fluids found at crime scenes.
Chromate Biosensor
Our chromate biosensor is a device that will be used to detect stainless steel residues left behind on bones in blunt or sharp force trauma cases.
Fingerprint Ageing
This device will search for the presence of a specifc component found in fingerprint deposits whose presence/absence will indicate approximate fingerprint age.
Week Beginning 22/06/2015
Summary
This week was all about getting started with the newly arrived biobrick BBa_K1058008.
Day 1 - 22/06: Plated BBa_K1058008 from agar stab.
Aim of experiment: To prepare BBa_K1058008 for sequence confirmation, storage, and further processing. The culture was plated from the agar stab and also a liquid culture set up.
Protocols Used: Plating, Overnight cultures
Results: N/A
Next Steps: Purification of the Plasmid, containing BBa_K1058008 and confirmation of size and sequence.
Day 2 - 23/06: Purified BBa_K1058008 from liquid culture.
Aim of experiment: To purify BBa_K1058008 for sequence confirmation.
Protocols Used: Miniprep plasmid purification, DNA sequencing
Results Sequencing: Figure 1
Next Steps: PCR of promoter region (pChr) and repressor region (ChrB) of BBa_K1058008 for storage in individual biobricks.
Week Beginning 29/06
Summary
During this week, BBa_K1058008 was disassembled into its constituent parts, pChr, and ChrB.
Day 1 - 29/06: PCR-amplification of pChr and ChrB from BBa_K1058008 .
Aim of experiment: To break down BBa_K1058008 into the promoter region pChr, and the repressor gene, ChrB, while adding standard prefix and suffix to each. Furthermore optimisation of first 15 codons of ChrB in order to increase the overall rate of transcription. Restriction digest of the respective ends and subsequent ligation of pChr and ChrB into pSB1C3 for submission to registry.
Protocols Used: PCR, Gel extraction, Restriction Digest
Results: Figure 2
Next Steps: Repeat of PCR of ChrB. Purification of the Plasmid, containing BBa_K1058008 and confirmation of size and sequence.
Week Beginning 06/07
Summary
During this week ligations of pChr and ChrB into pSB1C3 were finalised and the constituent parts of our newly designed chromate sensing system were prepared for ligation and transformation.
Day 1 - 06/07: Repeated amplification of ChrB. Ligation and transformation of pChr and ChrB.
Aim of experiment: To repeat PCR of ChrB for subsequent ligation into pSB1C3. Ligation of pChr and ChrB into pSB1C3. Transformation of each into a non-pathogenic lab strain of Escherichia coli.                   Overnight culture of E. coli DH5-alpha, containing GFPmut2, as a fluorescent reporter for our chromate sensing system
Protocols Used: PCR-amplification, Gel extraction, Restriction digest, Ligation, Overnight cultures
Results: Figure 3
2 bands were extracted from the gel for further characterisation, the strongest band, hence called ChrBs, and the band directly above, hence called ChrBw. pChr, ChrBs, and ChrBw were ligated into pSB1C3.                   pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw were transformed into E. coli MC1061.
Next Steps: Sequence and size confirmation of pChr-pSB1C3, ChrBs-pSB1C3, and ChrBw-pSB1C3. Ligation of pChr to GFP coding sequence for construction of a reporter plasmid.
Day 2 - 07/07: Purification of GFP and overnight cultures of pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw
Aim of experiment: Purification of GFPmut2 from overnight cultures. Overnight cultures of recombinant E. coli MC1061, containing pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw respectively for further size and sequence confirmation. Overnight culture of pUniprom for subsequent ligation of ChrB
Protocols Used: Miniprep, PCR-amplification, Gel extraction, Overnight cultures
Results: Figure 5
Next Steps: Repeat of amplification of ChrB. Sequence and size confirmation of pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw.
Day 3 - 08/07: Purification of plasmids pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw. Size and sequence confirmation.
Aim of experiment: To purify overnight cultures for size and sequence confirmation. Repeat of PCR amplification of pChr and ChrB. Restriction digestes of all parts in order to produce compatible sticky ends.
Protocols Used: Miniprep, PCR amplification, Restriction digest, Sequencing
Restriction digest of gel purified PCR-products of pChr, ChrB, and GFPmut2, using EcoRI/SpeI, BamHI/HindIII, and XbaI/PstI respecitvely.
Results:
Figure 6 - Size confirmation of pChr and ChrB.
Figure 7 - Sequence confirmation of pChr
Figure 8 - Sequence confirmation of ChrB.
Next Steps: Purification and digest of pUniprom for insertion of ChrB.
Day 4 - 09/07: Digested and cleaned up pUniprom for insertion of ChrB. Amplification of optimised ChrB with primers specific for the insertion into pUniprom.
Aim of experiment: To prepare pUniprom for insertion of ChrB by digesting it with BamHI and HindIII. To amplify ChrB version with first 15 codons optimised for insertion into pUniprom.
Protocols Used:
Restriction digest and Gel extraction of pUniprom
PCR and Gel extraction of optimised ChrB
Results:
Figure 9 Gel with pUniprom
Figure 10 Gel with ChrB
Next Steps: Ligation of ChrB and codon optimised ChrB into pUniprom.
Day 5 - 10/07: Ligations of ChrB and optimised ChrB into pUniprom, and of pChr and GFPmut2 into pSB1C3.
Aim of experiment: Ligation of ChrB and optimised ChrB into pUniprom via BamHI and HindIII restriction sites. Ligation of pChr and GFPmut2 to each other via SpeI/XbaI-link and into pSB1C3 via EcoRI and PstI restriction sites.
Protocols Used: Ligations
Results: N/A
Next Steps: Transformation of abovementioned ligations.
Week Beginning 13/07
Summary
During this week successful transformants were identified and propagated.
Day 1 - 13/07: Transformation of pUniprom-ChrB, pUniprom-ChrBopt, and pSB1C3-pChr-GFP.
Aim of experiment: Transformation of pUniprom-ChrB, pUniprom-ChrBopt, and pSB1C3-pChr-GFP into E. coli JM110 chassis. Setting up a backup restriction digest of pSB1C3 with EcoRI and SpeI in case the double ligation pSB1C3-pChr-GFP is unsuccessful.
Protocols Used: Transformations, Restriction digest
Results: Figure 11
Next Steps: Purification of recombinant vectors and confirmation of size and sequence. Store digested pSB1C3 at -20oC.
Day 2 - 14/07: Set up cultures for subsequent purification of recombinant vectors for size and sequence confirmation.
Aim of experiment: Purification of recombinant vectors, pUniprom-ChrB, pUniprom-ChrBopt, and pSB1C3-pChr-GFP, for size and sequence confirmation. Patch plating of the same colonies for storage and retrieval once size and sequence have been confirmed.
Protocols Used: Overnight cultures, Patch plating
Results: N/A
Next Steps: Purification of the recombinant plasmids for confirmation of size and sequence.
Day 3 - 15/07: Purified pUniprom-ChrB, pUniprom-ChrBopt, and pSB1C3-pChr-GFP from overnight cultures
Aim of experiment: Purification of pUniprom-ChrB, pUniprom-ChrBopt, and pSB1C3-pChr-GFP from overnight for confirmation of size and sequence. Restriction digest of pUnipromChrB and optimised pUniprom-ChrB optimised with BamHI and HindIII and of pSB1C3-pChr-GFP with EcoRI/PstI for size confirmation.
Protocols Used: Miniprep, Restriction digest, Colony PCR, Sequencing
Results:
Figure 12 Size confirmation of ChrB and ChrBopt.
Figure 13 Gel after colony PCR. Colonies 2, 4, 9, and 12 were selected for sequencing.
Figure 14 Sequencing result of pSB1C3-pChr-GFPmut2_1. The Result for pSB1C3-pChr-GFPmut2_2 was not as expected, albeit in the correct orientation.
Next Steps: Confirmation of sequence of ChrB-opt.
Day 4 - 16/07: Set up overnight cultures of colonies chosen after colony-PCR for subsequent purification and sequence confirmation.
Aim of experiment: Set up overnight cultures of ChrBopt colonies 2, 4, 9, and 12 for sequencing.
Protocols Used: Overnight cultures
Results: N/A
Next Steps: Purification of recombinant plasmids.
Day 5 - 17/07: Purified and sequenced chosen ChrBopt-colonies.
Aim of experiment: To purify recombinant plasmids for sequence confirmation.
Protocols Used: Miniprep, Sequencing
Results: ChrBopt showed HindIII restriction sites and was hence cleaved into partial inserts.
Next Steps: Use alternative cloning protocol for ChrB and ChrBopt.
Week Beginning 20/07
Summary
During this week an alternative cloning strategy for ChrB and ChrBopt was employed. Furthermore, the pSB1C3-pChr-GFP construct was cloned in a stepwise fashion.
Day 1 - 20/07: Went back to the original plate of pSB1C3-pChr-GFP for visualising colonies that do or do not express GFP.
Aim of experiment:To compare colonies on the patch plate of pSB1C3-pChr-GFP by differentially visualising colonies that do or do not express GFP under UV light. The background of this experiment was that if there is one colony that has both inserts in the correct orientation with respect to each other, then there is perhaps one that also has them with the correct orientation with respect to the plasmid. Colonies identified as expressing GFP were patch plated and grown in liquid cultures over night in order to confirm the sequence of the insert.
Protocols Used: Patch-plating, overnight cultures
Results: Figure 15 pSB1C3-pChr-GFP in E. coli JM110.
Next Steps: Sequence confirmation of newly selected colonies.
Day 2 - 21/07: Purified plasmids from new colonies and confirmed sequence. Restriction digest of pUniprom with BamHI and PstI.
Aim of experiment:Purification of newly selected colonies, 1, 5, 6, and 7, containing pSB1C3-pChr-GFP for subsequent sequence confirmation. Restriction digest of pUniprom with BamHI and PstI to employ an alternative cloning strategy of ChrB and ChrBopt.
Protocols Used:
Miniprep and Sequencing of pChrGFP insert.
Restriction digest of pUniprom.
Results:
Figure 16 Alignment of sequencing results.
Figure 17 Gel after restriction digest of pUniprom.
Next Steps:Stepwise ligation of pChr and GFP into pSB1C3 in order to clone the inserts in the correct orientation with respect to the plasmid. Insertion of ChrB and ChrBopt into pUniprom via BamHI and PstI links.
Day 3 - 22/07: Ligated pChr into pSB1C3 via EcoRI and SpeI sites and cloned the recombinant plasmid into E. coli JM110.
Aim of experiment: Ligation of pChr into pSB1C3 via EcoRI SpeI link. This is the first step of a 2-step cloning strategy with the aim of cloning pChr and GFP into pSB1C3 in the correct orientation with respect to the plasmid.
Protocols Used: Ligation, Transformation
Results: Figure 18 Plates after transformation of pChr into pSB1C3.
Next Steps: Sequence confirmation of pChr in pSB1C3.
Day 4 - 23/07: Set up overnight cultures of pSB1C3-pChr colonies.
Aim of experiment: Set up overnight cultures of pSB1C3-pChr colonies to prepare pSB1C3-pChr for sequence confirmation
Protocols Used: Overnight culture, Patch-plating
Results: N/A
Next Steps: Purification of the recombinant vector for size and sequence confirmation.
Day 5 - 24/07: Purified pSB1C3-pChr for sequence confirmation.
Aim of experiment: Purification of pSB1C3-pChr for sequence confirmation before ligating GFP into it.
Protocols Used:
Plasmid purification, Sequencing
Results: Figure 19 Alignment of sequenced pSB1C3-pChr vectors with pChr sequence.
Next Steps: Digest of pSB1C3-pChr with SpeI/PstI and ligation of GFPmut2.
Day 6 - 25/07: PCR-amplified ChrB and ChrBopt.
Aim of experiment: Amplification of ChrB and ChrBopt for insertion into pUniprom. Restriction digest of the PCR product with BamHI and PstI for producing compatible sticky ends
Protocols Used: PCR-amplification, Restriction digest
Results: N/A
Next Steps: Ligation and Transformation of ChrB and ChrBopt.
Day 7 - 26/07: Ligated ChrB and ChrBopt into pUniprom.
Aim of experiment: Insertion of ChrB and optimised ChrB into pUniprom vector.
Protocols Used: Ligation
Results: N/A
Next Steps: Transformation of ligated vectors.
Week Beginning 27/07
Summary
Final touches were made on the chromate system. All parts were ligated into their final vectors during this week. Preparations were made for characterisation of each of the parts.
Day 1 - 27/07: Transformed ChrB and ChrBopt into E. coli JM110
Aim of experiment:Transformation of the repressors, ChrB and ChrBopt, into JM110
Protocols Used: Transformation
Results: Figure 20 Plates after transformation.
Next Steps: Purification of pUniprom-ChrB and pUniprom-ChrBopt for sequence confirmation.
Day 2 - 28/07: Digested pSB1C3-pChr for insertion of GFP
Aim of experiment: Digesting sequence confirmed pSB1C3-pChr with SpeI and PstI for subsequent ligation with GFP. Overnight cultures of ChrB and ChrBopt for subsequent plasmid purification, size confirmation, and sequence confirmation.
Protocols Used:
Restriction digest, Ligation of GFP into pSB1C3-pChr.
Overnight culture of of ChrB and ChrBopt.
Next Steps: Transformation of recombinant vector for confirmation of size and sequence.
Results: N/A
Next Steps: Transformation of pSB1C3-pChr-GFP for size and sequence confirmation. Purification of ChrB and ChrBopt for sequence confirmation.
Day 3 - 29/07: Purified ChrB and ChrBopt for size and sequence confirmation.
Aim of experiment: Purification of ChrB and ChrBopt for confirmation of size and sequence.
Protocols Used: Plasmid purification, Restriction digest, Sequencing
Results:
Figure 21 Confirmation of insert size.
Figure 22 Confirmation of sequence.
Next Steps: Confirmation of expression of ChrB and ChrBopt by western blotting.
Day 4 - 30/07: Repeated transformation of pSB1C3-pChr-GFP into E.coli JM110
Aim of experiment: Digest of pSB1C3-pChr with SpeI and PstI for insertion of GFP. Repeat of unsuccessful transformation from 28/07.
Protocols Used: Restriction digest, Ligation, Transformation
Results: Figure 23 Plates after transformation
Next Steps: Overnight cultures for purification of the pSB1C3-pChr-GFP for confirmation of insert size and sequence.
Day 5 - 31/07: Set up overnight cultures for confirmation of pSB1C3-pChr-GFP.
Aim of experiment: Overnight cultures for subsequent plasmid purification of pSB1C3-pChr-GFP.
Protocols Used: Overnight culture
Results: N/A
Next Steps: Purification of the Plasmids for confirmation of insert size and sequence.
Day 6 - 01/08: Purified pSB1C3-pChr-GFP for size confirmation and sequencing.
Aim of experiment: Purification of pSB1C3-pChr-GFP for confirmation of insert size and sequence.
Protocols Used: Plasmid purification, Restriction digest, Sequencing
Results:
Figure 24 Gel for size confirmation.
Figure 25 Sequence confirmation.
Next Steps: Overnight cultures in preparation for western blotting for characterisation of expression of both, GFP, and ChrB and ChrBopt.
Day 7 - 02/08: Set up cultures in preparation for western blotting.
Aim of experiment: Overnight cultures of sequence confirmed ChrB, ChrBopt, pChrGFP1 (from 15/07), pChrGFP3 and pChrGFP4 (31/07) for subsequent western blotting.
Protocols Used: Overnight culture
Results: N/A
Next Steps: Western blotting for characterisation of expression of both, GFP, and ChrB and ChrBopt.
Week Beginning 03/08
Summary
This week was entirely used for characterising the expression of GFP from the promoter pChr, and the expression of ChrB from the tat-promoter in pUniprom.
Day 1 - Day 7 - 03/08 - 09/08: Determined expression of ChrB, ChrBopt, and GFP by western blotting.
Aim of experiment: To determine whether or not the required proteins, GFP, and ChrB, are expressed by blotting against GFP and against a 6His-tag respectively.
Protocols Used: SDS-PAGE and western blot
Results: This procedure was repeated several times during this week in order to find appropriate concentrations and volumes to load. Figure 26 was blotted after SDS-PAGE with the following concentrations:
GFP: Dilute pellet of 1ml overnight culture with 1ml TBS. Take 100µl of this solution and mix with 100µl Laemmli Buffer. Load 3µl on the SDS-gel.
HIS: Dilute pellet of 1ml overnight culture with 200µl Laemmli buffer. Load 10µl on the SDS-gel.
Figure 26
Next Steps:Transformation of both plasmids into MG1655 with the goal to characterise the interaction between ChrB and pChr. Furthermore the original biobrick, BBa_K1058008 will be tested under the same conditions for comparison.
Week Beginning 10/08
Summary
This week was spent with transforming all combinations of pSB1C3-pChr-GFP and pUniprom-ChrB into E. coli MG1655. Tests for their interaction were also prepared.
Day 1 - 10/08: Transformed both pSB1C3-pChr-GFP-plasmids, pUniprom-ChrB, and pUniprom-ChrBopt, as well as BBa_K1058008 into E. coli MG1655.
Aim of experiment: To transform the following combinations of pSB1C3-pChr-GFP and pUniprom-ChrB into MG1655:
pUniprom-ChrB2 (29/07) - pSB1C3-pChr-GFP (31/07)
pUniprom-ChrB2 (29/07) - pSB1C3-pChr-GFP (15/07)
pUniprom-ChrBopt3 (29/07) - pSB1C3-pChr-GFP (31/07)
pUniprom-ChrBopt3 (29/07) - pSB1C3-pChr-GFP (15/07)
in a stepwise fashion. Furthermore transform BBa_K1058008 into MG1655. Transformations were done in a stepwise fashion with only one of the plasmids at a time.
Protocols Used: Plasmid purification, Transformation
Results: Figure 27
Next Steps: Making newly transformed cells competent again in order to transform the second plasmid as well.
Day 2 - 11/08: Set up overnight cultures of all transformed cells.
Aim of experiment: To prepare overnight cultures of yesterdays transformaions in order to make them competent to transform the second plasmid as well.
Protocols Used: Overnight culture
Results: N/A
Next Steps: Make cells of overnight culture competent to transform in second plasmid.
Day 3 - 12/08: Made competent MG1655 with pUniprom-ChrB2, pUniprom-ChrBopt3, pSB1C3-pChr-GFP (15/07), pSB1C3-pChr-GFP (31/07).
Aim of experiment:To make transformed cells competent in order to transform the second plasmid. The same plasmids that were used for the initial transformations were also used for this set of transformations.
Protocols Used: Competent cells, Transformations
Results: N/A
Next Steps: Blot against GFP in order to confirm (or refute) that ChrB interacts with pChr and hence switches off expression of GFP.
Day 4 - 13/08: Transformed pUniprom-ChrB2 and pUniprom-ChrBopt3 into MG1655+pSB1C3-pChr-GFP(15/07) and MG1655+pChrGFP(31/07)
Aim of experiment: To build a strain of E. coli MG1655, containing both plasmids required for a chromate sensor.
Protocols Used: Transformation
Results: No successful transformants were found.
Next Steps: Repeat this experiment but transform pSB1C3-pChr-GFP (15/07) and pSB1C3-pChr-GFP (31/07) into competent MG1655+pUniprom-ChrB2 and MG1655+pUniprom-ChrBopt3 respectively.
Week Beginning 17/08
Summary
During this week E. coli MG1655 containing both plasmids for a. Western Blots were used to determine expression of ChrB, ChrBopt, and GFP.
Day 1 - 17/08: Transformations akin to 13/08 were done, but pSB1C3-pChr-GFP(15/07) and pSB1C3-pChr-GFP(31/07) were transformed into MG1655+pUniprom-ChrB2 and MG1655+pUniprom-ChrBopt3.
Aim of experiment: To build a strain of MG1655, containing both plasmids required for a chromate sensor.
Protocols Used: Transformation
Results: Successful transformants were found on all plates.
Next Steps: To determine expression of ChrB and ChrBopt (via 6His-tag) and expression of GFP.
Day 2 - 18/08: Preparation of samples for western blot.
Aim of experiment: To determine whether ChrB, ChrBopt, and GFP are expressed and have first estimates of differences in expression levels.
Protocols Used: Overnight culture
Results: N/A
Next Steps: Lyse cells and prepare samples for SDS-PAGE + Western-Blot.
Day 3-6 - 19/08 - 22/08: Repeated western blots with different concentrations to determine whether the expression of ChrB has an effect on the expression of GFP.
Aim of experiment: To determine whether expression of ChrB or ChrBopt has an effect on the expression of GFP.
Protocols Used: SDS-PAGE + Western Blot
Results: Figure 28
Next Steps: Check if different chromate salts have an effect on the respective expression levels.
Week Beginning 24/08
Summary
Plate reader experiments were carried out in order to determine the effect of K2CrO4, K2Cr2O7, CrCl3, and K2SO4 on the expression of GFP.
Day 1 - Day 7 - 24/08-30/08: Design of Plate reader experiments.
Aim of experiment: To design a series of plate reader experiments to measure OD600 and relative fluorescence of the chromate sensor as a response to different concentrations of K2CrO4, K2Cr2O7, CrCl3, and K2SO4.
Protocols Used: Plate Reader Experiments
Results: Figure 29 MG1655 + BBa_K1058008
Results: Figure 30 MG1655 + pSB1C3-pChr-GFP + pUniprom-ChrB
Results: Figure 31 MG1655 + pSB1C3-pChr-GFP + pUniprom-ChrBopt
Results: Figure 32 MG1655 + pSB1C3-pChr-GFP
Next Steps: Try the system with different expression levels of ChrB and ChrBopt. Induce overexpression by exploiting the T7-promoter in pUniprom after transforming the different combinations between pSB1C3-pChr-GFP and pUniprom-ChrB/pUniprom-ChrBopt into BL21(DE3) and adding IPTG.
Week Beginning 31/08
Summary
UK Synbio Conference and UK iGEM meetup at Westminster University.
Week Beginning 07/09
Summary
Parts from CeBiTec Bielefeld arrived and attempts at cloning were made. The chromate sensing system was transformed into BL21(DE3) and some of the plate reader experiments repeated.
Day 1 - 07/09: Transformation of pSB1K3 (Bielefeld) into MG1655.
Aim of experiment: The high-copy plasmid arrived at a concentration of 7 ng/µl. Hence it was transformed in order to produce more of it before purifying and digesting it.
Protocols Used: Transformation,
Overnight culture for new E. coli MG1655
Results: No transformants were found.
Next Steps: Retry transformation with new competent E. coli MG1655.
Day 2 - 08/09: Transformation of pSB1K3 into MG1655.
Aim of experiment: Test transformation of BBa_K1058008 into new MG1655 competent cells and re-transformation of pSB1K3 for subsequent miniprep.
Protocols Used: Transformation
Results: Successful test transformation, but no colonies on pSB1K3 plate.
Next Steps: Use an alternative compatible plasmid for ligation of codon optimised ChrB (Bielefeld).
Day 3 - 09/09: Transformation of pSB1K3 into MG1655.
Aim of experiment: Restriction digest of pT7KS with EcoRI and PstI as alternative to pSB1K3. Ligation of digested ChrB (Bielefeld) and transformation of pT7KS-ChrB. Also transformations of Chromate-sensor plasmids into BL21 (DE3) in the following combinations:
pSB1C3-pChr-GFP + pUniprom-ChrB
pSB1C3-pChr-GFP + pUniprom-ChrBopt
pUniprom-ChrB
pUniprom-ChrBopt
Protocols Used: Restriction Digest, Gel Extraction, Ligation
Results:Transformations of the plasmid with pT7KS-ChrB (Bielefeld) yielded 12 colonies, but also colonies on the vector control plate. Transformations of pSB1C3-pChr-GFP + pUniprom-ChrB, pSB1C3-pChr-GFP + pUniprom-ChrBopt, pUniprom-ChrB, and pUniprom-ChrBopt into BL21(DE3) were successful.
Results:
Figure 33 pT7KS-ChrB (Bielefeld) transformations
Figure 34 Transformations into BL21(DE3)
Next Steps: Confirmation of colonies with pT7KS-ChrB by colony PCR. Preparation of recombinant BL21(DE3) for plate reader experiments.
Day 4 - 10/09: Colony PCR of pT7KS-ChrB (Bielefeld).
Aim of experiment: Check if any of the colonies after ligation of ChrB (Bielefeld) into pT7KS were successful despite colonies on the control plate.
Protocols Used: Colony PCR
Results: Figure 34
Next Steps: Prepare transformed BL21(DE3) for plate reader experiments. Attempt new transformation of pT7KS-ChrB (Bielefeld).
Day 5 - 11/09: Preparation for Plate reader experiment with BL21(DE3)
Aim of experiment: Set up overnight cultures for subsequent plate reader experiments.
Protocols Used: Overnight cultures
Results: N/A
Next Steps: Set up plate reader, collect and analyse data. Assess the effect of induction of higher repressor-expression on the level of GFP.
Day 6 - 12/09: Set up plate reader.
Aim of experiment: To design a series of plate reader experiments to measure OD600 and relative fluorescence of the chromate sensor. At first no chromate was added to this plate in order to assess the effect of IPTG induction.
Protocols Used: Plate reader experiments
Results: Figure 35
Next Steps:Repeat experiments with added chromate substances.
Week Beginning 14/09
Summary
During this week the samples from the previous plate reader were retrieved and prepared for blotting against both GFP, and the 6His-Tag of ChrB and ChrBopt.
Day 1 - 14/09: Western blot of plate reader samples MG1655 + pSB1C3-pChr-GFP + pUniprom-ChrB, and MG1655 + pSB1C3-pChr-GFP + pUniprom-ChrBopt.
Aim of experiment: To identify whether or not there is any detectable difference in both, the level of GFP, and the level of ChrB or ChrBopt, before and after induction with IPTG.
Protocols Used: SDS-PAGE + Western blot
Results: Insert blot image here.
Next Steps: Repeat the assays of the previous weeks by subcloning ChrB, ChrBopt, and ChrB (Bielefeld) into vectors with different expression levels and see how it influences the level of GFP expression. Then add different chromate salts and also probe the system with other substances, e.g. other heavy-metal salts.
Watch Our Introduction Video
Dundee University
Located on the east coast of Scotland, The University of Dundee is one of Scotland's leading research institutes.
Social Media
youtube.com/DundeeiGEM2015
facebook.com/iGEMDundee2015
twitter.com/DundeeiGEMTeam
  University of Dundee
 Home
 About Us
Meet the Team
Attributions
 Project
Abstract
Project Overview
Devices
FluID
Fingerprint Ageing
Chromate Biosensor
Parts
Medals
Future Work
 Wet Lab
Lab Book
FluID
Fingerprints
Chromate Biosensor
Useful Information
Protocols
Sequences
Bacterial Two Hybrid
Safety
 Dry Lab
Introduction
FluID
Fingerprint Ageing
Chromate Biosensor
Appendix : Code
 Outreach
Practices
Ethics
Collaborations
