Team:Dundee/fmodal
×
Figure 1: Amplification of LbpA The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 2: LbpA PCR. Lane 1: PCR Product, Lane 2: 1kb Ladder and Marker. The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 3: LbpA PCR for Cloning into pQE80-L. Lane 1: LbpA PCR Product, Lane 2: 1kb Ladder and Marker The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 4: LbpA Culture Samples. Lane 1: Sample from uninduced culture. Lane 2: Sample from culture induced with 1mM IPTG. It seems that on comparison of the two cultures, that inducing expression of LbpA causes the cells to die given the significant reduction of protein levels visible on the gel.
Close
×
Figure 5: OD600 Readings from Uninduced Culture and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG). The table shows that after being induced with IPTG the cultures stop growing since their OD600 readings are notably lower than that of the uninduced control.
Close
×
Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, the induced cultures stop growing when LbpA expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 7: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, after reaching an OD600 of 0.6,  the induced cultures stop growing when LbpA starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have dealt with the production of the foreign protein.
Close
×
Figure 8: SDS Gel of Samples Taken from Uninduced Control and 1 mM IPTG Induced Culture 6 hours After Induction For the induced culture, a faint band just above the 100 kDa marker is observable on the gel which corresponds to the expected size of LbpA.
Close
×
Figure 1: Gel of SBP PCR Product for Cloning into pQE80-L. Lane 1: SBP PCR Product 1 Lane 2: 1kb Ladder and Marker Lane 3: SBP PCR Product 2 Both bands observable on the gel correspond to the expected size of SBP which is ~600 bp.
Close
×
Figure 2: SDS Gel of Uninduced and 1 mM IPTG Induced Cultures of PotD and SBP. For PotD, it is clear from the gel that it is being successfully overexpressed since there is an intense band present at the 37 kDa marker which corresponds to the expected size of PotD in the induced culture sample. For SBP, there is no band present which suggests SBP is being overexpressed.
Close
×
Figure 3: Characterisation of PotD. Single colonies of MC1061 E.coli containing pQE80-L encoding PotD were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin. Once cultures reached an OD600 of 0.6, one was induced with 1 mM IPTG and one was left uninduced as a control. They were left to grow for a further three hours, after which a 1ml sample was taken and the cells pelleted. The cells were resuspended in 100µl of Laemmli buffer, 20µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 4: SDS Gel of Uninduced and 0.5mM, 1mM and 2mM IPTG Induced Cultures of SBP. It appears that upon induction of SBP with any concentration of IPTG the cells are dying given that there is a significant drop in protein levels observable on the gel.
Close
×
Figure 5: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, the induced cultures stop growing when SBP expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, after reaching an OD600 of 0.6,  the induced cultures stop growing when SBP starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 7: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of PotD 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100 V. The two bands observable on the gel correspond to the expected size of PotD of 38 kDa. Western Blotting will confirm if these proteins are PotD.
Close
×
Figure 8: Characterisation of PotD following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 9: Characterisation of SBP following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 10: Characterisation of PotD/Spermidine Interaction. The graph shows that the change in the tryptophan spectra increases as concentration of spermidine increases.
Close
×
Figure 1: pSB1C3-hHBA and pSB1C3-hHBB (respectively) Pre-Sequencing Digest pSB1C3-hHBA and pSB1C3-hHBB minipreps were digested with EcoRI and PstI to check for presence of each insert. The bands observable on the gel correspond to the expected sizes for the pSB1C3 vector and both inserts.
Close
×
Figure 2: Amplification of hHBA and hHBB hHBA and hHBB were amplified from the pSB1c3-hHBA and pSB1c3-hHBB minipreps obtained yesterday using primers to allow cloning of the genes into pT25 and pUT18 respectively. Although the band is fainter in the case of hHBA, the gel indicates the PCR has been successful.
Close
×
Figure 3: Amplification of hHBA hHBA was amplified from the pSB1C3-hHBA using primers to allow cloning of the genes into pT25. The gel indicates that the PCR has been successful.
Close
×
Figure 4: Amplification of hHBA and hHBB hHBA and hHBB were amplified from the pSB1c3-hHBA and pSB1C3-hHBB minipreps using primers to allow cloning of the genes into pQE80-L. The gel indicates that the PCR has been successful.
Close
×
Figure 5: Pre-Sequence Digest of pQE80-L-hHBA and pQE80-L-hHBB Both pQE80-L-hHBA and pQE80-L-hHBB minipreps were digested with BamHI and KpnI to check for the presence of the inserts. The gel indicates that both inserts have been successfully cloned into pQE80-L.
Close
×
Overexpression of hHBB and hHBA. Single colonies of M15[pREP4] E.coli containing pQE80-L encoding hHBB and pQE80-L encoding hHBA were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD600 of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of 2x Laemmli buffer, 20 µl of which was separated by SDS-PAGE.
Close
×
Figure 7: Characterisation of hHBB. Single colonies of M15[pREP4] E.coli containing pQE80-L encoding hHBB were used to inoculate 5 ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD600 of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 8: Amplification of hHBN for Cloning into pQE80-L PCR product from amplification of hHBN from IDT plasmid using primers for cloninig into pQE80-L. The gel indicates hHBN has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.
Close
×
Figure 9: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of hHBB 10 µl of each fraction was mixed with 10 µl of laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. The four bands observable on the gel correspond to the expected size of hHBB of 16 kDa. Western Blotting will confirm if these proteins are hHBB.
Close
×
Figure 10: Characterisation of hHBB following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 11: Amplification of hHBN for Cloning into pT25 PCR product from amplification of hHBN from IDT plasmid using primers for cloninig into pT25. The gel indicates hHBN has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.
Close
×
Figure 14: Characterisation of hHBN following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 15: Amplification of hHBN hHBN was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of hHBN of 1200 bp.
Close
×
Figure 15: Amplification of hHBA hHBA was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of hHBA of
Close
×
Figure 16: Amplification of hHBN hHBN was amplified using primers to allow for subsequent cloning into the two hybrid vector pUT18. The band observable on the gel corresponds to the expected size of hHBN of 1200 bp.
Close
×
Figure 1: Amplification of OBP2A for insertion into pSB1C3 vector . This is the PCR product from the amplification of OBP2A from the IDT plasmid using primers for cloning into pSB1C3. The gel indicates OBP2A has been amplified successfully since the observed band corresponds to the expected size of 500 bp. The concentration of the gel extracted OBP2A is 335.53 ng/ul.
Close
×
Figure 2: A subsequent gel was run of the amplified OBP2A after it was first excised from the gel in Figure 1 This was to determine if the gel extraction was successful. The gel indicates that OBP2A has been extracted from the gel successfully, since the observed band corresponds to the expected size of 500 bp. This can now be restricted in preparation for ligation into pSB1C3.
Close
×
Figure 3: A Gel Analysis of Restriction Digest from the 24/6. This gel analysis was done to determine if the reason for the failed transformation was an improperly digested OBP2A. This gel indicates that the restricted product hasn't been lost as the OBP2A gene fragment can be seen as a distinct band at 500 bp. A re-ligation of OBP2A into the pSB1C3 backbone will now occur.
Close
×
Figure 4: Presequence Digest of pSB1C3-OBP2A. All seven plasmid purifications of the overnight cultures done on the 26/6 were digested to see if they contained the OBP2A insert. It is clear from the gel that all but sample 3 contain the insert for OBP2A as they all have faint bands at 500 bp's which corresponds to the size of OBP2A. It also appears that sample 5 contains the highest concentration, so that will be the sample that is sent for sequencing.
Close
×
Figure 5: Presequence digest of pUT18-OBP2A To determine if any samples contain the OBP2A insert. It can be seen from this gel that of the  5 samples that were digested only two are show to have the OBP2A insert, therefore sample 2 will be sent for sequencing.
Close
×
Figure 6: Amplification of OBP2A for insertion into pT25 vector . This is the PCR product from the amplification of OBP2A for cloning into pT25. The gel indicates OBP2A has been amplified successfully since the observed band corresponds to the expected size of 100 bp.
Close
×
Figure 7: Presequence Digest of pT25-OBP2A In order to determine if any of the samples contain the OBP2A insert. It can be seen from this gel that samples 5, 6 and 7 may have inserts in them as they each have two distinct bands, one of which corresponds to the 100 bp size for the OBP2A for pT25. As a result all three samples will be sent for sequencing.
Close
×
Figure 8: β-galactosidase Assay A graph of the results from the first attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of OBP2A. As it can be seen that the positive result has failed(far left) not much can be inferred from the interaction of the two parts of OBP2A. However it looks like the two subunits of OBP2A may not be interacting. As this was a first attempt, subsequent assays will be performed in order to determine if in fact the two separated parts of OBP2A aren't interacting in vivo - like these results suggests.
Close
×
Figure 9: β-galactosidase Assay A graph of the results from the second attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of OBP2A.  It can also be inferred from this graph that the two parts of OBP2A don't seem to be interacting as it has a low millers activity when compared to the controls. Further assays will be performed in order to determine if the two separated parts of OBP2A aren't interacting in vivo like these results suggests.
Close
×
Figure 11: β-galactosidase Assay Under Anaerobic Conditions A graph of the results from the anaerobic β-galactosidase assay aimed to characterize the interaction between the two separated parts of OBP2A. The graph seems to suggest that the two subunits of OBP2A may not be interacting in vivo. This can be inferred from the fact that the millers activity of the OBP2A sample lies lower than the negative controls from within the experiment. As a result of this information, one more assay will be prepared in which the two subunits will be switched with regards to the bacterial two hybrid vectors to eliminate any possibility of this being a factor effecting the interaction between the two subunits.
Close
×
Figure 1: Amplification of LS LS was amplified using primers which would allow for subsequent cloning into the high expression vector pQE80-L. Lane 1 shows that the PCR has been successful since the band corresponds to the expected size of 2100 bp.
Close
×
Figure 2: Amplification of LS LS was amplified from the IDT plasmid using primers to allow cloning of the gene into pSB1C3. The observed band corresponds to the expected size of LS, 2100 bp.
Close
×
Figure 3: Overexpression Assays of LS 5 ml cultures of M15pREP4 E.coli was grown to OD600=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37oC, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel.
Close
×
Figure 4: Characterisation of Lanosterol Synthase 5 ml cultures of M15pREP4 E.coli was grown to OD600=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37oC, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. The proteins were transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 5: Chromatogram following Affinity Chromatography of LS Using FPLC, the protein sample was loaded onto a nickel column and eluted with an increasing concentration of imidazole (green line). The fractions corresponding to the peak were ran on an SDS gel.
Close
×
Figure 6: SDS Gel of Peak Fractions 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer an loaded onto a SDS gel.
Close
×
Figure 1: Alignment of BBa_K1058008 . Sequences from top to bottom: sequencing result from forward primer; expected sequence; reverse complement of sequencing result from reverse primer; sequencing result from intermediate primer. Colour code: Green = Prefix/Suffix; Pink = Annealing site of intermediate primer; Yellow = Mismatch in one sequence; Red = In frame stop codons.
'
Close
×
Figure 2: 1% agarose gel after PCR of BBa_K1058008 with primers specific to the promoter region, Pchr, and the repressor region, chrB. Left: Pchr (expected size 169bp); Middle: 1kb+ ladder, Right: chrB (expected size 939bp). Amplification of chrB was not successful. Pchr was further processed by restriction digest with EcoRI and PstI.
'
Close
×
Figure 3: 1% agarose gel after PCR of BBa_K1058008 with primers specific to the repressor region, chrB. Left: 1.5 µl DMSO; Middle: 1kb+ ladder, Right: 2.5 µl DMSO. Expected size for each: 939bp. The annealing temperature for the PCR amplification was lowered to 50°C in order to increase the chances of accomodating a primer that is not 100% complementary.
'
Close
×
Figure 4: 1% agarose gel after PCR of gfp with corresponding primers. Left: GFP (expected size 717bp); Middle: 1kb+ ladder, Right: PCR product of chrB, using primers to add a BamHI restriction site in front of it and a 6-His-Tag and a HindIII restriction site at the end. (expected size of chrB: 939bp). Since amplification of chrB was unsuccessful, it will be repeated.
'
Close
×
Figure 5: 1% agarose gel after restriction digest of pSB1C3-Pchr, pSB1C3-chrB (s), and pSB1C3-chrB (w) with EcoRI and PstI for confirmation of size. Ladder: 1kb+. Top: 1-4: Pchr 1-4; chrB (s) 1-2. Bottom: chrB (s) 3-4; chrB (w) 1-4. Expected sizes: Pchr: 169bp; chrB: 939bp.
'
Close
×
Figure 6: Alignment of Pchr sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green = Prefix/Suffix
'
Close
×
Figure 7: Alignment of chrB (opt) sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green: Prefix and Suffix; Red: In frame stop codons; Blue: First 15 codons optimised for expression in E. coli.
'
Close
×
Figure 8: 1% agarose gel after restriction digest of pUniprom (Expected size 2454bp).
'
Close
×
Figure 11: 1% agarose gel after restriction digest of pSB1C3 with EcoRI and SpeI (Expected size 2070bp).
'
Close
×
Figure 9: 1% agarose gel after PCR of chrB (opt). (Expected size 939bp).
'
Close
×
Figure 10: Plates after transformation of Top: pUnipromchrB, Middle: pUnipromchrB (opt) and Bottom: pSB1C3-Pchr-gfp into JM110.
'
Close
×
Figure 12: 1% agarose gel after restriction digest of pUnipromchrB, pUnipromchrB (opt) and pSB1C3-Pchr-gfp for confirmation of size. Ladder: 1kb+. Top: 1-4: pSB1C3-Pchr-gfp 1-4; pUnipromchrB (opt) 1-2. Bottom: pUnipromchrB (opt) 3-4; pUnipromchrB 1-4. Expected sizes: Pchr-gfp: 909bp; chrB and chrB (opt): 939bp.
'
Close
×
Figure 13: 1% agarose gel after colony PCR of pUnipromchrB (opt). Top: colonies 1-11; Bottom: colonies 12-22. Colonies 2, 4, 9, and 12 were selected for sequencing. Ladder: 1kb+ Expected size of chrB (opt): 939bp.
'
Close
×
Figure 14: Sequencing result of pSB1C3-Pchr-gfp after double ligation. Even though the sequence is as expected, it is ligated in reverse. It was hence called pSB1C3-Pchr-gfp (A) The Result for pSB1C3-Pchr-gfp (colony 2) was not as expected, albeit in the correct orientation. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 15: Plates with pSB1C3-Pchr-gfp in JM110 after double ligation under UV light. Numbered colonies were selected for initial size confirmation and sequencing. However, other colonies seemed to express GFP as well, hence a second attempt was made with different colonies from the same plate.
'
Close
×
Figure 16: Alignment of sequencing results of pSB1C3-Pchr-gfp - colonies 1, 5, 6, and 7 with sequences of gfp (mut2) and Pchr sequence. Colony 1 remained consistent, the other tested colonies were without success. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 17: 1% Agarose gel after restriction digest of pUniprom with BamHI/PstI (expected size: 2454bp). Right: PCR product of Haptoglobin.
'
Close
×
Figure 18: Plates after transformation of Pchr into pSB1C3. Top: ratio insert:vector 2:1; Bottom: ratio insert:vector 3:1.
'
Close
×
Figure 19: Alignment of Pchr sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green = Prefix/Suffix
'
Close
×
Figure 20: Plates after transformation of, Top: chrB into pUniprom; Bottom: chrB (opt) into pUniprom. Ratio insert:vector for ligations: left: 3:1; Right: 4:1.
'
Close
×
Figure 21: Gel after restriction digest of Left: pUniprom-chrB, and Right: pUniprom-chrB (opt) with BamHI/PstI for confirmation of insert size. Expected size 939bp. chrB colony 2 and chrB (opt) colony 3 were selected for sequencing.
'
Close
×
Figure 22: Alignments of: Left: chrB and Right: chrB (opt). Colours: Pink = Restriction sites (BamHI/PstI); Blue = 6-His Tag; Red = In-frame stop codons; Cyan = optimised codons.
'
Close
×
Figure 23: Plates after ligation of gfp into pSB1C3-Pchr. Left: Ratio Insert:Vector 2:1; Right: Ratio Insert:Vector 3:1.
'
Close
×
Figure 24: Gel for size confirmation of pSB1C3-Pchr-gfp (Top). Bottom: Left - Haptoglobin, Right - Lanosterol Synthase. pSB1C3-Pchr-gfp (3) and pSB1C3-Pchr-gfp (4) were submitted for sequencing.
'
Close
×
Figure 25: Alignment of Pchr sequence retrieved from BBa_K1058008 and gfp (mut2) with the forward and reverse sequencing results of Pchr-gfp this region. The sequence was as expected, it was hence called pSB1C3-Pchr-gfp (B). Colour code: Green = Prefix/Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 26: Top: anti-GFP: Single colonies of JM110 + pSB1C3-Pchr-gfp (A), MC1061 + pSB1C3-Pchr-gfp (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of each sample was mixed with 100 µl laemmli buffer, and boiled for 10min. 3 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and PmanA-gfp as a positive control.          Bottom: anti-6-His - ChrB: Single colonies of JM110 pUniprom-chrB and JM110 pUniprom-chrB (opt) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 200 µl laemmli buffer, and the sample boiled for 10min. 10 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and hydA - 6-His as a positive control.         Expected sizes: GFP ~ 27kDa, ChrB ~35kDa.
'
Close
×
Figure 27: Plates after transformation of single constituents of chromate sensor into E. coli MG1655. From top to bottom: pUniprom-chrB (from colony 2 - 29/07), pUniprom-chrB (opt) (from colony 3 - 29/07), pSB1C3-Pchr-gfp (B), pSB1C3-Pchr-gfp (A), BBa_K1058008 .
'
Close
×
Figure 28: Western blot against GFP: Single colonies of MG1655, containing all 4 combinations, pUniprom-chrB - pSB1C3-Pchr-gfp (A), pUniprom-chrB - pSB1C3-Pchr-gfp (B), pUniprom-chrB (opt) - pSB1C3-Pchr-gfp (A), and pUniprom-chrB (opt) - pSB1C3-Pchr-gfp (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol and 100 µg/ml ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml PBS. 100 µl of these samples was mixed with 100 µl laemmli buffer, and the sample boiled for 10 min. The indicated volume of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3, and pUniprom were included as a negative control, and PmanA-gfp as a positive control. The blot against 6-His – ChrB was without success and is not shown. No chromate of any kind was added to the medium previous to this blot, hence no expression of GFP was expected.
'
Close
K2CrO4, K2Cr2O7, CrCl3, and K2SO4
×
Figure 29: F,OD-plot of results of plate reader experiment after exposing MG1655 + BBa_K1058008 to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 30: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 31: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt) to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 32: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 33: Plates after ligation of chrB (Bielefeld) into pT7KS. From top to bottom: Ratio insert:vector 2:1; Ratio Insert:Vector 3:1; pT7KS religation control.
'
Close
×
Figure 34: Plates after transformation of single constituents of chromate sensor into BL21 (DE3). From top to bottom: pSB1C3-Pchr-gfp (B) + pUniprom-chrB, pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt), pUniprom-chrB, and pUniprom-chrB (opt).
'
Close
×
Figure 35: 1% agarose gel after colony PCR of pT7KS-chrB (Bielefeld). Top: colonies 1-6; Bottom: colonies 7-12. Ladder: 1kb+ Expected size of chrB (Bielefeld): 939bp
'
Close
×
Figure 36: F,OD-plot of results of plate reader experiment after inducing expression of ChrB from the T7 promoter of pUniprom BL21 (DE3) pSB1C3-Pchr-gfp (B) + pUniprom-chrB, and pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt). IPTG was added after 4 hours. One measurement was made every 10min across 22h.
'
Close
