<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_labjournal_manu skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/labjournal manu</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Dundee iGEM 2015</TITLE></P><H1><HIGHLIGHT class="highlight">LABJOURNAL</HIGHLIGHT></H1><SECTION id="about" class="row1"><DIV class="row"><DIV class="col-lg-4"><H3>BioSpray</H3><P class="about-content">Our forensic toolkit aims to use synthetic biology approaches to improve on the current 
methods used by crime scene investigators.</P></DIV><DIV class="col-lg-4"><H3>Chromium Detector</H3><P class="about-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur a tincidunt elit. Aliquam porta nibh at enim luctus, auctor consequat dolor vehicula.</P></DIV><DIV class="col-lg-4"><H3>Fingerprint Aging</H3><P class="about-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur a tincidunt elit. Aliquam porta nibh at enim luctus, auctor consequat dolor vehicula.</P></DIV></DIV></SECTION><DIV class="ribbon"><DIV class="rows"><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 22/06/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">This week was all about getting started with the newly arrived biobrick <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A></P></DIV><DIV id="collapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/06: Plated <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> from agar stab.</SPAN><DIV id="collapsiblew1-1" class="collapse box-content"><P><B>Aim of experiment:</B> To prepare <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> for sequence confirmation, storage, and further processing. The culture was plated from the agar stab and also a liquid culture set up.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#plating-modal" class="journal-protocol"> Plating, </A><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of the Plasmid, containing <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> and confirmation of size and sequence.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/06: Purified <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> from liquid culture.</SPAN><DIV id="collapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To purify <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> for sequence confirmation.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep plasmid purification, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol"> DNA sequencing</A></P><P><B>Results Sequencing: </B><A data-toggle="modal" data-target="#BBa_K1058008-figure1-modal" class="journal-protocol"> Figure 1 </A></P><P><B>Next Steps:</B> PCR of promoter region (P<I>chr</I>) and repressor region (<I>chrB</I>) of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> for storage in individual biobricks.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 29/06/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> During this week, <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> was disassembled into its constituent parts, P<I>chr</I>, and <I>chrB</I>.</P></DIV><DIV id="collapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/06: PCR-amplification of <I>pChr</I> and <I>ChrB</I> from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>.</SPAN><DIV id="collapsiblew2-1" class="collapse box-content"><P><B>Aim of experiment: </B> To break down <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> into the promoter region <I>pChr</I>, and the repressor gene, <I>ChrB</I>, while adding standard prefix and suffix to each. Furthermore optimisation of first 15 codons of <I>ChrB</I> in order to increase the overall rate of transcription. Restriction digest of the respective ends and subsequent ligation of <I>pChr</I> and <I>ChrB</I> into pSB1C3 for submission to registry.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR, </A><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol"> Gel extraction, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction Digest</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15-chr-igem022-modal" class="journal-protocol"> Figure 2</A></P><P><B>Next Steps:</B> Repeat of PCR of <I>ChrB</I>. Purification of the Plasmid, containing <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> and confirmation of size and sequence.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 06/07/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">During this week ligations of <I>pChr</I> and <I>ChrB</I> into pSB1C3 were finalised and the constituent parts of our newly designed chromate sensing system were prepared for ligation and transformation.</P></DIV><DIV id="collapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">06/07: Repeated amplification of <I>ChrB</I>. Ligation and transformation of <I>pChr</I> and <I>ChrB</I>.</SPAN><DIV id="collapsiblew3-1" class="collapse box-content"><P><B>Aim of experiment:</B> To repeat PCR of ChrB for subsequent ligation into pSB1C3. Ligation of <I>pChr</I> and <I>ChrB</I> into pSB1C3. Transformation of each into a non-pathogenic lab strain of <I>Escherichia coli</I>.
                  Overnight culture of <I>E. coli</I> DH5-alpha, containing GFPmut2, as a fluorescent reporter for our chromate sensing system</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR-amplification, </A><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol"> Gel extraction, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction digest, </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligation, </A><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15-chr-igem023-modal" class="journal-protocol"> Figure 3 </A></P><P>2 bands were extracted from the gel for further characterisation, the strongest band, hence called <I>ChrBs</I>, and the band directly above, hence called <I>ChrBw</I>. <I>pChr</I>, <I>ChrBs</I>, and <I>ChrBw</I> were ligated into pSB1C3.
                  pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw were transformed into <I>E. coli</I> MC1061.</P><P><B>Next Steps:</B> Sequence and size confirmation of pChr-pSB1C3, ChrBs-pSB1C3, and ChrBw-pSB1C3. Ligation of <I>pChr</I> to <I>GFP coding sequence</I> for construction of a reporter plasmid.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">07/07: Purification of <I>gfp</I> and overnight cultures of pSB1C3-P<I>chr</I>, pSB1C3-<I>chrB</I> (s), and pSB1C3-<I>chrB</I> (w)</SPAN><DIV id="collapsiblew3-2" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of GFPmut2 from overnight cultures. Overnight cultures of recombinant <I>E. coli</I> MC1061, containing pSB1C3-P<I>chr</I>, pSB1C3-<I>chrB</I> (s), and pSB1C3-<I>chrB</I> (w) respectively for further size and sequence confirmation. Overnight culture of pUniprom for subsequent ligation of <I>chrB</I></P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep, </A><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR-amplification, </A><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol"> Gel extraction, </A><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15-chr-igem024-modal" class="journal-protocol"> Figure 4 </A></P><P><B>Next Steps:</B> Repeat of amplification of <I>ChrB</I>. Sequence and size confirmation of pSB1C3-P<I>chr</I>, pSB1C3-<I>chrB</I> (s), and pSB1C3-<I>chrB</I> (w).</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">08/07: Purification of plasmids pSB1C3-P<I>chr</I>, pSB1C3-<I>chrB</I> (s), and pSB1C3-<I>chrB</I> (w). Size and sequence confirmation.</SPAN><DIV id="collapsiblew3-3" class="collapse box-content"><P><B>Aim of experiment:</B> To purify overnight cultures for size and sequence confirmation. Repeat of PCR amplification of <I>pChr</I> and <I>ChrB</I>. Restriction digestes of all parts in order to produce compatible sticky ends.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep, </A><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR amplification, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing</A></P><P><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction digest</A> of gel purified PCR-products of P<I>chr</I>, <I>chrB</I>, and <I>gfp</I>, using EcoRI/SpeI, BamHI/HindIII, and XbaI/PstI respecitvely.</P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15-chr-igem026-modal" class="journal-protocol"> Figure 5 </A> - Size confirmation of <I>pChr</I> and <I>ChrB</I>.</P><P><A data-toggle="modal" data-target="#dundee15-chr-pChr1-modal" class="journal-protocol"> Figure 6 </A> - Sequence confirmation of P<I>chr</I></P><P><A data-toggle="modal" data-target="#dundee15-chr-ChrBw4-modal" class="journal-protocol"> Figure 7 </A> - Sequence confirmation of <I>chrB</I>.</P><P><B>Next Steps:</B> Purification and digest of pUniprom for insertion of <I>chrB</I>.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">09/07: Digested and cleaned up pUniprom for insertion of <I>chrB</I>. Amplification of optimised <I>chrB</I> with primers specific for the insertion into pUniprom.</SPAN><DIV id="collapsiblew3-4" class="collapse box-content"><P><B>Aim of experiment:</B> To prepare pUniprom for insertion of <I>chrB</I> by digesting it with BamHI and HindIII. To amplify <I>chrB</I> version with first 15 codons optimised for insertion into pUniprom.</P><P><B>Protocols Used:</B></P><P><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction digest</A> and
                    <A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol">Gel extraction</A> of pUniprom</P><P><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A> and
                    <A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol"> Gel extraction </A> of <I>chrB</I> (opt)</P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15-chr-igem034-modal" class="journal-protocol"> Figure 8 </A> Gel with pUniprom</P><P><A data-toggle="modal" data-target="#dundee15-chr-igem032-modal" class="journal-protocol"> Figure 9 </A>Gel with <I>chrB</I></P><P><B>Next Steps:</B> Ligation of <I>chrB</I> and <I>chrB</I> (opt) into pUniprom.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/07: Ligations of <I>chrB</I> and <I>chrB</I> (opt) into pUniprom, and of P<I>chr</I> and <I>gfp</I> into pSB1C3.</SPAN><DIV id="collapsiblew3-5" class="collapse box-content"><P><B>Aim of experiment:</B> Ligation of <I>chrB</I> and optimised <I>chrB</I> into pUniprom via BamHI and HindIII restriction sites. Ligation of P<I>chr</I> and <I>gfp</I> to each other via SpeI/XbaI-link and into pSB1C3 via EcoRI and PstI restriction sites.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Transformation of abovementioned ligations.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 13/07/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">During this week successful transformants were identified and propagated.</P></DIV><DIV id="collapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">13/07: Transformation of pUniprom-<I>chrB</I>, pUniprom-<I>chrB</I> (opt), and pSB1C3-P<I>chr-gfp</I>.</SPAN><DIV id="collapsiblew4-1" class="collapse box-content"><P><B>Aim of experiment:</B> Transformation of pUniprom-<I>chrB</I>, pUniprom-<I>chrB</I> (opt), and pSB1C3-P<I>chr-gfp</I> into <I>E. coli</I> JM110 chassis. Setting up a backup restriction digest of pSB1C3 with EcoRI and SpeI in case the double ligation pSB1C3-P<I>chr-gfp</I> is unsuccessful.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest</A></P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15-chr-first-transformation-modal" class="journal-protocol"> Figure 10 </A>Plates after transformation</P><P><A data-toggle="modal" data-target="#dundee15-chr-igem035-modal" class="journal-protocol"> Figure 11 </A> Gel after digest of pSB1C3</P><P><B>Next Steps:</B> Purification of recombinant vectors and confirmation of size and sequence. Store digested pSB1C3 at -20<SUP>o</SUP>C.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/07: Set up cultures for subsequent purification of recombinant vectors for size and sequence confirmation.</SPAN><DIV id="collapsiblew4-2" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of recombinant vectors, pUniprom-<I>chrB</I>, pUniprom-<I>chrB</I> (opt), and pSB1C3-P<I>chr-gfp</I>, for size and sequence confirmation. Patch plating of the same colonies for storage and retrieval once size and sequence have been confirmed.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures, </A><A data-toggle="modal" data-target="#plating-modal" class="journal-protocol"> Patch plating</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of the recombinant plasmids for confirmation of size and sequence.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">15/07: Purified pUniprom-<I>chrB</I>, pUniprom-<I>chrB</I> (opt), and pSB1C3-P<I>chr-gfp</I> from overnight cultures</SPAN><DIV id="collapsiblew4-3" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of pUniprom-<I>chrB</I>, pUniprom-<I>chrB</I> (opt), and pSB1C3-P<I>chr-gfp</I> from overnight for confirmation of size and sequence. Restriction digest of pUniprom-<I>chrB</I>, pUniprom-<I>chrB</I> (opt) with BamHI and HindIII and of pSB1C3-P<I>chr-gfp</I> with EcoRI/PstI for size confirmation. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#colony-modal" class="journal-protocol">Colony PCR, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing </A></P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15-chr-igem036-modal" class="journal-protocol"> Figure 12 </A> Size confirmation of <I>chrB</I> and <I>chrB</I> (opt).</P><A data-toggle="modal" data-target="#dundee15-chr-igem037-modal" class="journal-protocol"> Figure 13 </A> Gel after colony PCR of <I>chrB</I> (opt).<A data-toggle="modal" data-target="#dundee15_chr_pSB1C3-pChr-GFP_reverse" class="journal-protocol"> Figure 14 </A> Sequencing result of pSB1C3-P<I>chr-gfp</I> (colony 1).<P><B>Next Steps:</B> Confirmation of sequence of <I>chrB</I> (opt)</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">16/07: Set up overnight cultures of <I>chrB</I> (opt) - colonies chosen after colony-PCR for subsequent purification and sequence confirmation.</SPAN><DIV id="collapsiblew4-4" class="collapse box-content"><P><B>Aim of experiment:</B> Set up overnight cultures of <I>chrB</I> (opt) colonies 2, 4, 9, and 12 for sequencing.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of recombinant plasmids.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/07: Purified and sequenced chosen <I>chrB</I> (opt) - colonies.</SPAN><DIV id="collapsiblew4-5" class="collapse box-content"><P><B>Aim of experiment:</B> To purify recombinant plasmids for sequence confirmation.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing</A></P><P><B>Results:</B><I>chrB</I> (opt) showed HindIII restriction sites and was hence cleaved into partial inserts.</P><P><B>Next Steps:</B> Use alternative cloning protocol for <I>chrB</I> and <I>chrB</I> (opt).</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 20/07/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">During this week an alternative cloning strategy for <I>chrB</I> and <I>chrB</I> (opt) was employed. Furthermore, the pSB1C3-P<I>chr-gfp</I> - construct was cloned in a stepwise fashion.</P></DIV><DIV id="collapsiblew5" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">20/07: Went back to the original plate of pSB1C3-P<I>chr-gfp</I> for visualising colonies that do or do not express GFP. </SPAN><DIV id="collapsiblew5-1" class="collapse box-content"><P><B>Aim of experiment:</B> To compare colonies on the patch plate of pSB1C3-P<I>chr-gfp</I> by differentially visualising colonies that do or do not express GFP under UV light. The background of this experiment was that if there is one colony that has both inserts in the correct orientation with respect to each other, then there is perhaps one that also has them with the correct orientation with respect to the plasmid. Colonies identified as expressing GFP were patch plated and grown in liquid cultures over night in order to confirm the sequence of the insert.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#plating-modal" class="journal-protocol"> Patch-plating, </A><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> overnight cultures</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_pChrGFPplates_revisited" class="journal-protocol"> Figure 15</A> pSB1C3-P<I>chr-gfp</I> in <I>E. coli</I> JM110.</P><P><B>Next Steps:</B> Sequence confirmation of newly selected colonies.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">21/07: Purified plasmids from new colonies and confirmed sequence. Restriction digest of pUniprom with BamHI and PstI.</SPAN><DIV id="collapsiblew5-2" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of newly selected pSB1C3-P<I>chr-gfp</I> - colonies 1, 5, 6, and 7, for subsequent sequence confirmation. Restriction digest of pUniprom with BamHI and PstI to employ an alternative cloning strategy of <I>chrB</I> and <I>chrB</I> (opt).</P><P><B>Protocols Used:</B></P><P><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep </A> and
                    <A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing </A> of P<I>chr-gfp</I> - insert.</P><P><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest</A> of pUniprom.
                  </P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15_chr_pChrGFP_revisited_seq" class="journal-protocol"> Figure 16 </A> Alignment of sequencing results.</P><P><A data-toggle="modal" data-target="#dundee15_chr_igem039" class="journal-protocol"> Figure 17 </A> Gel after restriction digest of pUniprom.</P><P><B>Next steps:</B> Stepwise ligation of P<I>chr</I> and <I>gfp</I> into pSB1C3 in order to clone the inserts in the correct orientation with respect to the plasmid. Insertion of <I>chrB</I> and <I>chrB</I> (opt) into pUniprom via BamHI and PstI links.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/07: Ligated P<I>chr</I> into pSB1C3 via EcoRI and SpeI sites and cloned the recombinant plasmid into <I>E. coli</I> JM110.</SPAN><DIV id="collapsiblew5-3" class="collapse box-content"><P><B>Aim of experiment:</B> Ligation of P<I>chr</I> into pSB1C3 via EcoRI and SpeI sites. This is the first step of a 2-step cloning strategy with the aim of cloning P<I>chr</I> and <I>gfp</I> into pSB1C3 in the correct orientation with respect to the plasmid.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligation, </A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150723_pSB1C3-pChr" class="journal-protocol"> Figure 18 </A> Plates after transformation of P<I>chr</I> into pSB1C3.</P><P><B>Next Steps:</B> Sequence confirmation of P<I>chr</I> in pSB1C3.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/07: Set up overnight cultures of pSB1C3-P<I>chr</I> colonies.</SPAN><DIV id="collapsiblew1-4" class="collapse box-content"><P><B>Aim of experiment:</B> Set up overnight cultures of pSB1C3-P<I>chr</I> colonies to prepare pSB1C3-P<I>chr</I> for sequence confirmation</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight culture, </A><A data-toggle="modal" data-target="#plating-modal" class="journal-protocol">Patch-plating</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of the recombinant vector for size and sequence confirmation.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/07: Purified pSB1C3-P<I>chr</I> for sequence confirmation.</SPAN><DIV id="collapsiblew5-5" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of pSB1C3-P<I>chr</I> for sequence confirmation before ligating <I>gfp</I> into it.</P><P><B>Protocols Used:</B></P><P><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid purification, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150727_pSB1C3-pChr_alignment" class="journal-protocol"> Figure 19 </A> Alignment of sequenced pSB1C3-P<I>chr</I> vectors with P<I>chr</I> sequence.</P><P><B>Next Steps:</B> Digest of pSB1C3-P<I>chr</I> with SpeI/PstI and ligation of P<I>gfp</I></P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">25/07: PCR-amplified <I>chrB</I> and <I>chrB</I> (opt).</SPAN><DIV id="collapsiblew5-6" class="collapse box-content"><P><B>Aim of experiment:</B> Amplification of <I>chrB</I> and <I>chrB</I> (opt) for insertion into pUniprom. Restriction digest of the PCR product with BamHI and PstI for producing compatible sticky ends</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR-amplification, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction digest</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Ligation and Transformation of <I>chrB</I> and <I>chrB</I> (opt).</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">26/07: Ligated <I>chrB</I> and <I>chrB</I> (opt) into pUniprom.</SPAN><DIV id="collapsiblew5-7" class="collapse box-content"><P><B>Aim of experiment:</B> Insertion of <I>chrB</I> and <I>chrB</I> (opt) into pUniprom.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligation</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Transformation of ligated vectors.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 27/07/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Final touches were made on the chromate system. All parts were ligated into their final vectors during this week. Preparations were made for characterisation of each of the parts.</P></DIV><DIV id="collapsiblew6" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/07: Transformed <I>chrB</I> and <I>chrB</I> (opt) into <I>E. coli</I> JM110</SPAN><DIV id="collapsiblew6-1" class="collapse box-content"><P><B>Aim of experiment:</B> Transformation of the repressors, <I>chrB</I> and <I>chrB</I> (opt), into JM110</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformation</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150728_pUniprom-ChrB" class="journal-protocol"> Figure 20</A> Plates after transformation.</P><P><B>Next Steps:</B> Purification of pUniprom-<I>chrB</I> and pUniprom-<I>chrB</I> (opt) for sequence confirmation.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/07: Digested pSB1C3-P<I>chr</I> for insertion of <I>gfp</I></SPAN><DIV id="collapsiblew6-2" class="collapse box-content"><P><B>Aim of experiment:</B> Digesting sequence confirmed pSB1C3-P<I>chr</I> with SpeI and PstI for subsequent ligation with <I>gfp</I>. Overnight cultures of <I>chrB</I> and <I>chrB</I> (opt) for subsequent plasmid purification, size confirmation, and sequence confirmation.</P><P><B>Protocols Used:</B></P><P><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction digest, </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation</A> of <I>gfp</I> into pSB1C3-P<I>chr</I>.</P><P><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol">Overnight culture</A> of of <I>chrB</I> and <I>chrB</I> (opt).</P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Transformation of pSB1C3-P<I>chr-gfp</I> for size and sequence confirmation. Purification of <I>chrB</I> and <I>chrB</I> (opt) for sequence confirmation.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/07: Purified <I>chrB</I> and <I>chrB</I> (opt) for size and sequence confirmation.</SPAN><DIV id="collapsiblew6-3" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of <I>chrB</I> and <I>chrB</I> (opt) for confirmation of size and sequence.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid purification, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing</A></P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15_chr_igem040" class="journal-protocol">Figure 21 </A>Confirmation of insert size.</P><P><A data-toggle="modal" data-target="#dundee15_chr_150729_chrb-combined" class="journal-protocol"> Figure 22 </A>Confirmation of sequence.</P><P><B>Next Steps:</B> Confirmation of expression of <I>chrB</I> and <I>chrB</I> (opt) by western blotting.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/07: Repeated transformation of pSB1C3-P<I>chr-gfp</I> into <I>E.coli JM110</I></SPAN><DIV id="collapsiblew6-4" class="collapse box-content"><P><B>Aim of experiment:</B> Digest of pSB1C3-P<I>chr</I> with SpeI and PstI for insertion of GFP. Repeat of unsuccessful transformation from 28/07.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation, </A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150731_pChrGFP-ligations" class="journal-protocol"> Figure 23 </A>Plates after transformation</P><P><B>Next Steps:</B> Overnight cultures for purification of the pSB1C3-P<I>chr-gfp</I> for confirmation of insert size and sequence.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">31/07: Set up overnight cultures for confirmation of pSB1C3-P<I>chr-gfp</I>.</SPAN><DIV id="collapsiblew6-5" class="collapse box-content"><P><B>Aim of experiment:</B> Overnight cultures for subsequent plasmid purification of pSB1C3-P<I>chr-gfp</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight culture</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of the Plasmids for confirmation of insert size and sequence.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">01/08: Purified pSB1C3-P<I>chr-gfp</I> for size confirmation and sequencing.</SPAN><DIV id="collapsiblew6-6" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of pSB1C3-P<I>chr-gfp</I> for confirmation of insert size and sequence.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid purification, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing</A></P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15_chr_igem043" class="journal-protocol"> Figure 24 </A> Gel for size confirmation.</P><P><A data-toggle="modal" data-target="#dundee15_chr_150801_pchrgfp" class="journal-protocol"> Figure 25 </A> Sequence confirmation.</P><P><B>Next Steps:</B> Overnight cultures in preparation for western blotting for characterisation of expression of both, GFP, and ChrB and ChrBopt.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">02/08: Set up cultures of JM110 + pSB1C3-P<I>chr-gfp</I> (A), MC1061 + pSB1C3-P<I>chr-gfp</I> (B) and JM110 pUniprom-<I>chrB</I> and JM110 pUniprom-<I>chrB</I> (opt). in preparation for western blotting.</SPAN><DIV id="collapsiblew6-7" class="collapse box-content"><P><B>Aim of experiment:</B> Overnight cultures of <I>E. coli</I> strains with sequence confirmed pUniprom-<I>chrB</I>, pUniprom-<I>chrB</I> (opt), pSB1C3-P<I>chr-gfp</I> (A), pSB1C3-P<I>chr-gfp</I> (B) - colony 3 and pSB1C3-P<I>chr-gfp</I> (B) - colony 4 for subsequent western blotting.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight culture</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Western blotting for characterisation of expression of both, GFP, and ChrB and ChrB (opt).</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 03/08/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">This week was entirely used for characterising the expression of GFP from the promoter P<I>chr</I>, and the expression of ChrB from the <A href="http://parts.igem.org/Part:BBa_K895000" target="_blank">P<I>tat</I></A> in pUniprom.</P></DIV><DIV id="collapsiblew7" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">03/08 - 09/08: Determined expression of ChrB, ChrB (opt), and GFP by western blotting.</SPAN><DIV id="collapsiblew7-1" class="collapse box-content"><P><B>Aim of experiment:</B> To determine whether or not the required proteins, GFP, and ChrB, are expressed by blotting against GFP and against a 6-His Tag of ChrB and ChrB (opt) respectively.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> SDS-PAGE and western blot</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150809_blot" class="journal-protocol"> Figure 26 </A></P><P><B>Next Steps:</B> Transformation of both plasmids into MG1655 with the goal to characterise the interaction between ChrB and P<I>chr</I>. Furthermore the original biobrick, <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> will be tested under the same conditions for comparison.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/08/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">This week was spent with transforming all combinations of pSB1C3-P<I>chr-gfp</I> and pUniprom-<I>chrB</I> into <I>E. coli</I> MG1655. Tests for their interaction were also prepared.</P></DIV><DIV id="collapsiblew8" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/08: Transformed pSB1C3-P<I>chr-gfp</I> (A) and pSB1C3-P<I>chr-gfp</I> (B), and pUniprom-<I>chrB</I>, and pUniprom-<I>chrB</I> (opt), as well as <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> into <I>E. coli</I> MG1655.</SPAN><DIV id="collapsiblew8-1" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the following combinations of pSB1C3-pChr-GFP and pUniprom-ChrB into MG1655:
                    <UL><LI>pUniprom-<I>chrB</I> (from colony 2 - 29/07) - pSB1C3-P<I>chr-gfp</I> (A)</LI><LI>pUniprom-<I>chrB</I> (from colony 2 - 29/07) - pSB1C3-P<I>chr-gfp</I> (B)</LI><LI>pUniprom-<I>chrB</I> (opt) (from colony 3 - 29/07) - pSB1C3-P<I>chr-gfp</I> (A)</LI><LI>pUniprom-<I>chrB</I> (opt) (from colony 3 - 29/07) - pSB1C3-P<I>chr-gfp</I> (B)</LI></UL>
                  in a stepwise fashion. Furthermore transform <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> into MG1655. Transformations were done in a stepwise fashion with only one of the plasmids at a time.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid purification, </A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150811_transformations" class="journal-protocol"> Figure 27 </A>Plates after transformation.</P><P><B>Next Steps:</B> Making newly transformed cells competent again in order to transform the second plasmid as well.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/08: Set up overnight cultures of all transformed cells.</SPAN><DIV id="collapsiblew8-2" class="collapse box-content"><P><B>Aim of experiment:</B> To prepare overnight cultures of yesterdays transformations in order to make them competent to transform the second plasmid as well.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight culture</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Make cells of overnight culture competent to transform in second plasmid.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">12/08: Made competent MG1655 with pUniprom-ChrB2, pUniprom-ChrBopt3, pSB1C3-pChr-GFP (15/07), pSB1C3-pChr-GFP (31/07).</SPAN><DIV id="collapsiblew8-3" class="collapse box-content"><P><B>Aim of experiment:</B> To make transformed cells competent in order to transform the second plasmid. The same plasmids that were used for the initial transformations were also used for this set of transformations.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#competent-modal" class="journal-protocol"> Competent cells, </A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformations </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Blot against GFP in order to confirm (or refute) that ChrB interacts with P<I>chr</I> and hence switches off expression of GFP.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">13/08: Transformed pUniprom-<I>chrB</I> (from colony 2 - 29/07) and pUniprom-<I>chrB</I> (opt) (from colony 3 - 29/07) into MG1655 + pSB1C3-P<I>chr-gfp</I> (A) and pSB1C3-P<I>chr-gfp</I> (B)</SPAN><DIV id="collapsiblew8-4" class="collapse box-content"><P><B>Aim of experiment:</B> To build a strain of <I>E. coli</I> MG1655, containing both plasmids required for a chromate sensor.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformation</A></P><P><B>Results:</B> No successful transformants were found.</P><P><B>Next Steps:</B> Repeat this experiment but transform pSB1C3-P<I>chr-gfp</I> (A) and pSB1C3-P<I>chr-gfp</I> (B) into competent MG1655 + pUniprom-<I>chrB</I> and MG1655 + pUniprom-<I>chrB</I> (opt) respectively.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 17/08/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">During this week <I>E. coli</I> MG1655 containing both plasmids for a. Western Blots were used to determine expression of ChrB and GFP.</P></DIV><DIV id="collapsiblew9" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/08: Transformations akin to 13/08 were done, but pSB1C3-P<I>chr-gfp</I> (A) and pSB1C3-P<I>chr-gfp</I> (A) were transformed into MG1655 + pUniprom-<I>chrB</I> and MG1655 + pUniprom-<I>chrB</I> (opt).</SPAN><DIV id="collapsiblew9-1" class="collapse box-content"><P><B>Aim of experiment:</B> To build a strain of MG1655, containing both plasmids required for a chromate sensor.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformation </A></P><P><B>Results:</B> Successful transformants were found on all plates.</P><P><B>Next Steps:</B> To determine expression of ChrB and GFP.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/08: Preparation of samples for western blot.</SPAN><DIV id="collapsiblew9-2" class="collapse box-content"><P><B>Aim of experiment:</B> To determine whether ChrB and GFP are expressed and have first estimates of differences in expression levels.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight culture</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Lyse cells and prepare samples for SDS-PAGE + Western-Blot.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">19/08 - 22/08: Repeated western blots with different concentrations to determine whether the expression of ChrB has an effect on GFP levels.</SPAN><DIV id="collapsiblew9-3" class="collapse box-content"><P><B>Aim of experiment:</B> To determine whether expression of ChrB has an effect on the expression of GFP.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> SDS-PAGE + Western Blot</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150822_blot" class="journal-protocol"> Figure 28 </A></P><P><B>Next Steps:</B> Check if different chromate salts have an effect on the respective expression levels.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 24/08/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Plate reader experiments were carried out in order to determine the effect of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB> on the expression of GFP.</P></DIV><DIV id="collapsiblew10" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/08-30/08: Design of Plate reader experiments.</SPAN><DIV id="collapsiblew10-1" class="collapse box-content"><P><B>Aim of experiment:</B> To design a series of plate reader experiments to measure OD<SUB>600</SUB> and relative fluorescence of the chromate sensor as a response to different concentrations of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#platereader-modal" class="journal-protocol"> Plate Reader Experiments </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150824_platereader_BBa_K1058008" class="journal-protocol"> Figure 29 </A>MG1655 + <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#icecream" class="journal-protocol"> Figure 30 </A>MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I></P><P><B>Results:</B><A data-toggle="modal" data-target="#potato" class="journal-protocol"> Figure 31 </A>MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt)</P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150824_platereader_pchrbgfp" class="journal-protocol"> Figure 32 </A>MG1655 + pSB1C3-P<I>chr-gfp</I> (B)</P><P><B>Next Steps:</B> Try the system with different expression levels of ChrB. Induce overexpression by exploiting the T7-promoter in pUniprom after transforming the different combinations between pSB1C3-P<I>chr-gfp</I> and pUniprom<I>chrB</I> / pUniprom<I>chrB</I> (opt) into <I>BL21(DE3)</I> and adding IPTG.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 31/08/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">UK Synbio Conference and UK iGEM meetup at Westminster University.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 07/09/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Parts from CeBiTec Bielefeld arrived and attempts at cloning were made. The chromate sensing system was transformed into <I>BL21(DE3)</I> and some of the plate reader experiments repeated.</P></DIV><DIV id="collapsiblew12" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">07/09: Transformation of pSB1K3 (Bielefeld) into MG1655.</SPAN><DIV id="collapsiblew12-1" class="collapse box-content"><P><B>Aim of experiment:</B> The high-copy plasmid arrived at a concentration of 7 <SUP>ng</SUP>/<SUB>µl</SUB>. Hence it was transformed in order to produce more of it before purifying and digesting it.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformation, </A></P><P><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight culture</A> for new <I>E. coli</I> MG1655</P><P><B>Results:</B> No transformants were found.</P><P><B>Next Steps:</B> Retry transformation with new competent <I>E. coli</I> MG1655.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">08/09: Transformation of pSB1K3 into MG1655.</SPAN><DIV id="collapsiblew12-2" class="collapse box-content"><P><B>Aim of experiment:</B> Test transformation of BBa_K1058008 into new MG1655 competent cells and re-transformation of pSB1K3 for subsequent miniprep.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformation </A></P><P><B>Results:</B> Successful test transformation, but no colonies were found on the pSB1K3 plate.</P><P><B>Next Steps:</B> Use an alternative compatible plasmid for ligation of the codon optimised <I>chrB</I> (Bielefeld).</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">09/09: Transformation of pSB1K3 into MG1655.</SPAN><DIV id="collapsiblew12-3" class="collapse box-content"><P><B>Aim of experiment:</B> Restriction digest of pT7KS with EcoRI and PstI as alternative to pSB1K3. Ligation of digested <I>chrB</I> (Bielefeld) and transformation of pT7KS-<I>chrB</I>. Also transformations of Chromate-sensor plasmids into <I>BL21 (DE3)</I> in the following combinations: <UL><LI>pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I></LI><LI>pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt)</LI><LI>pUniprom-<I>chrB</I></LI><LI>pUniprom-<I>chrB</I> (opt)</LI></UL></P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction Digest, </A><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol"> Gel Extraction, </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligation </A></P><P><B>Results:</B> Transformations of the plasmid with pT7KS-<I>chrB</I> (Bielefeld) yielded 12 colonies, but also colonies on the vector control plate. Transformations of pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt), pUniprom-<I>chrB</I>, and pUniprom-<I>chrB</I> (opt) into <I>BL21(DE3)</I> were successful.</P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15_chr_150911_chrbbielefeld" class="journal-protocol"> Figure 33 </A> pT7KS-<I>chrB</I> (Bielefeld) transformations</P><P><A data-toggle="modal" data-target="#dundee15_chr_150911_bl21" class="journal-protocol"> Figure 34 </A> Transformations into <I>BL21(DE3)</I></P><P><B>Next Steps:</B> Confirmation of colonies with pT7KS-<I>chrB</I> (Bielefeld) by colony PCR. Preparation of recombinant <I>BL21(DE3)</I> for plate reader experiments.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/09: Colony PCR of pT7KS-<I>chrB</I> (Bielefeld).</SPAN><DIV id="collapsiblew12-4" class="collapse box-content"><P><B>Aim of experiment:</B> Check if any of the colonies after ligation of <I>chrB</I> (Bielefeld) into pT7KS were successful despite colonies on the control plate.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#colony-modal" class="journal-protocol"> Colony PCR </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150911_igem047" class="journal-protocol"> Figure 35 </A></P><P><B>Next Steps:</B> Prepare transformed <I>BL21(DE3)</I> for plate reader experiments. Attempt new transformation of pT7KS-<I>chrB</I> (Bielefeld).</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/09: Preparation for Plate reader experiment with <I>BL21(DE3)</I></SPAN><DIV id="collapsiblew12-5" class="collapse box-content"><P><B>Aim of experiment:</B> Set up overnight cultures for subsequent plate reader experiments.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Set up plate reader, collect and analyse data. Assess the effect of induction of higher repressor-expression on the level of GFP.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">12/09: Set up plate reader.</SPAN><DIV id="collapsiblew12-6" class="collapse box-content"><P><B>Aim of experiment:</B> To design a series of plate reader experiments to measure OD<SUB>600</SUB> and relative fluorescence of the chromate sensor. At first no chromate was added to this plate in order to assess the effect of IPTG induction.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#platereader-modal" class="journal-protocol"> Plate reader experiments</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_platereaderinduced" class="journal-protocol"> Figure 36 </A></P><P><B>Next Steps:</B> Repeat experiments with added chromate substances.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 14/09/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">During this week the samples from the previous plate reader were retrieved and prepared for blotting against both GFP, and the 6-His - Tag of ChrB.</P></DIV><DIV id="collapsiblew13" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/09: Western blot of plate reader samples MG1655 + pSB1C3-P<I>chr-gfp</I> + pUniprom-<I>chrB</I>, and MG1655 + pSB1C3-P<I>chr-gfp</I> + pUniprom-<I>chrB</I> (opt).</SPAN><DIV id="collapsiblew13-1" class="collapse box-content"><P><B>Aim of experiment:</B> To identify whether or not there is any detectable difference in both, the level of GFP, and the level of ChrB, before and after induction with IPTG.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> SDS-PAGE + Western blot </A></P><P><B>Results:</B> Even though the blot was very blurred, expression of GFP could be confirmed in all samples. There seemed to be no noteable difference between induced and uninduced.</P><P><B>Next Steps:</B> Repeat the assays of the previous weeks by subcloning <I>chrB</I>, <I>chrB</I> (opt), and <I>chrB</I> (Bielefeld) into vectors with different expression levels and see how it influences the level of GFP expression. Then add different chromate salts and also probe the system with other substances, e.g. other heavy-metal salts.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plate Reader Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>A 5 ml overnight culture of the desired strain was grown at 37 <SUP>o</SUP>C.</LI><LI>Equalise OD<SUB>600</SUB> if required. Dilute culture 1:1000.</LI><LI>Aliquot 198 µl into the required amount of wells in a 96-well plate.</LI><LI>Add additional substances and controls as required.</LI><LI>Preheat platereader to required temperature and start experiment.</LI><LI>Retrieve and analyse data.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="competent-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Competent Cells Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>A 5 ml overnight culture of the desired strain was grown at 37<SUP>o</SUP>C.</LI><LI>Subsample <SUP>1</SUP>/<SUB>10</SUB> of the volume of the subculture. I.e. 1 ml of the overnight culture into 10 ml fresh LB.</LI><LI>Grow subculture at at 37 <SUP>o</SUP>C for 2 h.</LI><LI>Spin down subculture at 4000 rpm at 4<SUP>o</SUP>C for 10 min.</LI><LI>Resuspend the pellet in ice cold transformation buffer. Use <SUP>1</SUP>/<SUB>10</SUB> of the initial volume of the subculture.</LI><LI>Use 100 µl per transformation.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="plating-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plating Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>Fresh agar plates were inoculated with cells on the toothpick.</LI><LI>Agar plates were incubated at 4<SUP>o</SUP>C over night.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sequencing-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Sequencing Protocol</DIV></DIV><DIV class="modal-body"><P>
    For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
      <UL><LI>Volume of sample equivalent to 550ng DNA</LI><LI>2 µl of 3 mM primer</LI><LI>Remaining volume of water for a total volume of 30 µl</LI></UL></P><P>
      The samples were submitted to <A href="https://www.dnaseq.co.uk/" target="_blank">DNA sequencing and services</A>.
    </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="overnightculture-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Overnight Culture Protocol</DIV></DIV><DIV class="modal-body"><P><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>5 ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.</LI><LI>Tubes were incubated at 37<SUP>o</SUP>C in a rotary incubator at 200rpm over night.</LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="PCR-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">PCR Protocol</DIV></DIV><DIV class="modal-body"><P>
    PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the <STRONG>Sequences</STRONG> section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY color="black"><TR><TD width="312" valign="top"><P align="center"><STRONG>Reactants</STRONG></P></TD><TD width="312" valign="top"><P align="center"><STRONG>Volume (µl)</STRONG></P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DNA
                </P></TD><TD width="312" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DMSO
                </P></TD><TD width="312" valign="top"><P align="center">
                    2.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Forward Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Reverse Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase Buffer
                </P></TD><TD width="312" valign="top"><P align="center">
                    10
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    dNTP
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    H<SUB>2</SUB>O
                </P></TD><TD width="312" valign="top"><P align="center">
                    34.5
                </P></TD></TR></TBODY></TABLE><P>
    **PCR Programme**
</P><P>
    Elongation time was altered according to the length of the sequences.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="colony-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Colony PCR Protocol</DIV></DIV><DIV class="modal-body"><P>
    The PCR reaction mixture were set up in multiples of the volumes in table below. The length of all sequences used is stated in the <STRONG>Sequences</STRONG> section.
</P><P>
    The DNA template is made in the following way:
    <OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>A numbered grid according to the desired number of PCR reactions is drawn on the back of the plate.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>Each colony is transferred to a 30 µl aliquot of sterile water.</LI><LI>With the same toothpick the grid with the respctive number on the plate was inoculated.</LI><LI>The plate was incubated at 37<SUP>o</SUP>C overnight.</LI><LI>The inoculated water-aliquots were boiled for 10min, then centrifuged at 13300rpm for 10min.</LI><LI>The supernatant of this preparation is the DNA fraction for the subsequent PCR reaction.</LI></OL></P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY color="black"><TR><TD width="312" valign="top"><P align="center"><STRONG>Reactants</STRONG></P></TD><TD width="312" valign="top"><P align="center"><STRONG>Volume (µl)</STRONG></P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DNA
                </P></TD><TD width="312" valign="top"><P align="center">
                    5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DMSO
                </P></TD><TD width="312" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Forward Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Reverse Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    +Mg Buffer
                </P></TD><TD width="312" valign="top"><P align="center">
                    2
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    dNTP
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.2
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Taq-polymerase
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.2
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    H<SUB>2</SUB>O
                </P></TD><TD width="312" valign="top"><P align="center">
                    10.6
                </P></TD></TR></TBODY></TABLE><P>15 µl of the reaction mixture was aliquoted into separate PCR tubes. 5 µl of DNA fraction was added to each respective tube in order to have a total reaction volume of 20 µl per tube.</P><P>
    **PCR Programme**
</P><P>
    Elongation time was altered according to the length of the sequences.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="miniprep-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Miniprep Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>5 ml bacterial overnight culture was pelleted by stepwise centrifugation in 1.5 ml microcentrifuge tubes at 13300rpm for 3 minutes at room temperature.</LI><LI>Pelleted bacterial cells were resuspended in 250 µl of Buffer P1.</LI><LI>250 µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution became clear.</LI><LI>350 µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.</LI><LI>The mixture was centrifuged for 10 minutes at 13300rpm in a microcentrifuge.</LI><LI>The supernatant was applied to a QIAprep spin column by pipetting.</LI><LI>500 µl of Buffer PB was added to the QIAprep spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>750 µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>The QIAprep spin column was spun for an additional minute to remove residual wash buffer.</LI><LI>The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H<SUB>2</SUB>O to the QIAprep spin column and centrifuging for 1 minute. </LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="gelextraction-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Gel Extraction Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.</LI><LI>The gel slice was transferred into a microcentrifuge tube. 700 µl of Buffer QG was added.</LI><LI>The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.</LI><LI>After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.</LI><LI>230 µl of isopropanol was added to the sample and mixed.</LI><LI>A QIAquick spin column was placed in a 2 ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>500 µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>750 µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>The QIAprep spin column was spun for an additional minute to remove residual wash buffer.</LI><LI>The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="restriction-digest-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Restriction Digest Protocol</DIV></DIV><DIV class="modal-body"><P>
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the    <STRONG>Journal</STRONG> section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="170" valign="top"><P><STRONG>Post Plasmid Purification</STRONG></P></TD><TD width="151" valign="top"><P><STRONG>Post PCR</STRONG></P></TD><TD width="139" valign="top"><P><STRONG>gBlock</STRONG></P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>DNA</STRONG></P></TD><TD width="170" valign="top"><P>
                    1mg
                </P></TD><TD width="151" valign="top"><P>
                    50 µl
                </P></TD><TD width="139" valign="top"><P>
                    100ng
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Cutsmart Buffer (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    6
                </P></TD><TD width="151" valign="top"><P>
                    6
                </P></TD><TD width="139" valign="top"><P>
                    6
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Water (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    As required
                </P></TD><TD width="151" valign="top"><P>
                    1
                </P></TD><TD width="139" valign="top"><P>
                    As required
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Enzyme 1 (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    1.5
                </P></TD><TD width="151" valign="top"><P>
                    1.5
                </P></TD><TD width="139" valign="top"><P>
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Enzyme 2 (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    1.5
                </P></TD><TD width="151" valign="top"><P>
                    1.5
                </P></TD><TD width="139" valign="top"><P>
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG><U>Total Volume (µl)</U></STRONG></P></TD><TD width="170" valign="top"><P>
                    30
                </P></TD><TD width="151" valign="top"><P>
                    60
                </P></TD><TD width="139" valign="top"><P>
                    30
                </P></TD></TR></TBODY></TABLE><P><UL><LI>The digests were incubated at 37<SUP>o</SUP>C for 3 hours.</LI><LI>Note that when performing plasmid digestions, 2.5 µl of alkaline phosphatase and 2.5 µl of its buffer were added at the 2 hour and 2.5 hour mark.</LI></UL></P></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ligation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Ligation Protocol</DIV></DIV><DIV class="modal-body"><P>
Ligation reactions were set up according to the table below. The vector to insert ratio is stated for each specific reaction in the    <STRONG>Journal</STRONG> section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="111" valign="top"><P align="center"><STRONG>Vector (µl)</STRONG></P></TD><TD width="111" valign="top"><P align="center"><STRONG>2:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>3:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>4:1 (Vector: Insert) [µl]</STRONG></P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>T4 DNA Ligase </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Ligase Buffer </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Vector </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Insert</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    -
                </P></TD><TD width="111" valign="top"><P align="center">
                    2
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>H<SUB>2</SUB>0</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    7
                </P></TD><TD width="111" valign="top"><P align="center">
                    5
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD></TR></TBODY></TABLE><P>
    The reactions were incubated at room temperature for at least 3 hours.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Transformation Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>100 µl of competent cells were defrosted. Thaw on ice.</LI><LI>Agar plates were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>1-5 µl of DNA was added to cells.</LI><LI>Cell and DNA mixture was left on ice for 20 minutes.</LI><LI>Mixture was placed in 42<SUP>o</SUP>C water bath for 90 seconds.</LI><LI>Mixture was put back on ice for 2 minutes.</LI><LI>1 ml of LB (without antibiotic) was added to cells.</LI><LI>Cells were allowed to grow for 1 hour at 37<SUP>o</SUP>C in a shaking incubator.</LI><LI>Cells were then spun down and the supernatant discarded.</LI><LI>The pellet was resuspended in the 100 µl.</LI><LI>Cells were plated on agar plates with appropriate antibiotic.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sds-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Protein Expression Optimization Protocol</DIV></DIV><DIV class="modal-body"><P><STRONG><U>SDS-PAGE and Western Blotting </U></STRONG></P><OL><LI>50 µl of an overnight culture was used to inoculate 5 ml of fresh LB containing the appropriate antibiotics and left to grow at 37<SUP>o</SUP>C in a shaking incubator.</LI><LI>Once the OD<SUB>600</SUB>=0.6-0.8, 1 mM of IPTG was added to the culture and left to grow for a further 3 hours.</LI><LI>1 ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.</LI><LI>Meanwhile, 950 µl of 2x Laemmli sample buffer was added to 50 µl of B-mercaptoethanol.</LI><LI>The supernatant was discarded from the samples and the pellet resuspended in 200 µl of the above solution.</LI><LI>The samples were then boiled at 90<SUP>o</SUP>C for 10 minutes.</LI><LI>Afterwards, they were spun down for 1 minute in a microcentrifuge.</LI><LI>At this stage, the samples could be stored at -20<SUP>o</SUP>C or ran on a gel immediately.</LI><LI>Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the stacking gel at which point the voltage was increased to 200V for 20 minutes.</LI><LI>One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.</LI><LI>After the proteins had been transferred onto the membrane, the membrane was blocked in 5% Marvel milk overnight at 4<SUP>o</SUP>C.</LI><LI>The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10 ml of 1xTBS buffer to which the primary anti-His antibody was added and left for one hour. The protein ladder was cut off before adding the primary antibody.</LI><LI>The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10 ml of 1xTBS buffer, to which the secondary antibody was added and left for one hour.</LI><LI>The membrane was then washed in 3x10 ml 1xTBST buffer for 5 minutes.</LI><LI>The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off at step 12.</LI><LI>Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="BBa_K1058008-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: Alignment of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>. Sequences from top to bottom: sequencing result from forward primer; expected sequence; reverse complement of sequencing result from reverse primer; sequencing result from intermediate primer. Colour code: Green = Prefix/Suffix; Pink = Annealing site of intermediate primer; Yellow = Mismatch in one sequence; Red = In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem022-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 2: 1% agarose gel after PCR of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with primers specific to the promoter region, P<I>chr</I>, and the repressor region, <I>chrB</I>. Left: P<I>chr</I> (expected size 169bp); Middle: 1kb+ ladder, Right: <I>chrB</I> (expected size 939bp). Amplification of <I>chrB</I> was not successful. P<I>chr</I> was further processed by restriction digest with EcoRI and PstI.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem023-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 3: 1% agarose gel after PCR of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with primers specific to the repressor region, <I>chrB</I>. Left: 1.5 µl DMSO; Middle: 1kb+ ladder, Right: 2.5 µl DMSO. Expected size for each: 939bp. The annealing temperature for the PCR amplification was lowered to 50°C in order to increase the chances of accomodating a primer that is not 100% complementary.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem024-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 4: 1% agarose gel after PCR of <I>gfp</I> with corresponding primers. Left: <I>GFP</I> (expected size 717bp); Middle: 1kb+ ladder, Right: PCR product of <I>chrB</I>, using primers to add a BamHI restriction site in front of it and a 6-His-Tag and a HindIII restriction site at the end. (expected size of <I>chrB</I>: 939bp). Since amplification of <I>chrB</I> was unsuccessful, it will be repeated.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem026-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 5: 1% agarose gel after restriction digest of pSB1C3-P<I>chr</I>, pSB1C3-<I>chrB</I> (s), and pSB1C3-<I>chrB</I> (w) with EcoRI and PstI for confirmation of size. Ladder: 1kb+. Top: 1-4: P<I>chr</I> 1-4; <I>chrB</I> (s) 1-2. Bottom: <I>chrB</I> (s) 3-4; <I>chrB</I> (w) 1-4. Expected sizes: P<I>chr</I>: 169bp; <I>chrB</I>: 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-pChr1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 6: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green = Prefix/Suffix
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-ChrBw4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 7: Alignment of <I>chrB</I> (opt) sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green: Prefix and Suffix; Red: In frame stop codons; Blue: First 15 codons optimised for expression in E. coli.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem034-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 8: 1% agarose gel after restriction digest of pUniprom (Expected size 2454bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem035-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 11: 1% agarose gel after restriction digest of pSB1C3 with EcoRI and SpeI (Expected size 2070bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem032-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 9: 1% agarose gel after PCR of <I>chrB</I> (opt). (Expected size 939bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-first-transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 10: Plates after transformation of Top: pUniprom<I>chrB</I>, Middle: pUniprom<I>chrB</I> (opt) and Bottom: pSB1C3-P<I>chr-gfp</I> into JM110.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem036-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 12: 1% agarose gel after restriction digest of pUniprom<I>chrB</I>, pUniprom<I>chrB</I> (opt) and pSB1C3-P<I>chr-gfp</I> for confirmation of size. Ladder: 1kb+. Top: 1-4: pSB1C3-P<I>chr-gfp</I> 1-4; pUniprom<I>chrB</I> (opt) 1-2. Bottom: pUniprom<I>chrB</I> (opt) 3-4; pUniprom<I>chrB</I> 1-4. Expected sizes: P<I>chr-gfp</I>: 909bp; <I>chrB</I> and <I>chrB</I> (opt): 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem037-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 13: 1% agarose gel after colony PCR of pUniprom<I>chrB</I> (opt). Top: colonies 1-11; Bottom: colonies 12-22. Colonies 2, 4, 9, and 12 were selected for sequencing. Ladder: 1kb+ Expected size of <I>chrB</I> (opt): 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pSB1C3-pChr-GFP_reverse" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 14: Sequencing result of pSB1C3-P<I>chr-gfp</I> after double ligation. Even though the sequence is as expected, it is ligated in reverse. It was hence called pSB1C3-P<I>chr-gfp</I> (A) The Result for pSB1C3-P<I>chr-gfp</I> (colony 2) was not as expected, albeit in the correct orientation. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pChrGFPplates_revisited" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 15: Plates with pSB1C3-P<I>chr-gfp</I> in JM110 after double ligation under UV light. Numbered colonies were selected for initial size confirmation and sequencing. However, other colonies seemed to express GFP as well, hence a second attempt was made with different colonies from the same plate.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pChrGFP_revisited_seq" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 16: Alignment of sequencing results of pSB1C3-P<I>chr-gfp</I> - colonies 1, 5, 6, and 7 with sequences of <I>gfp</I> (mut2) and P<I>chr</I> sequence. Colony 1 remained consistent, the other tested colonies were without success. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem039" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 17: 1% Agarose gel after restriction digest of pUniprom with BamHI/PstI (expected size: 2454bp). Right: PCR product of Haptoglobin.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150723_pSB1C3-pChr" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 18: Plates after transformation of P<I>chr</I> into pSB1C3. Top: ratio insert:vector 2:1; Bottom: ratio insert:vector 3:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150727_pSB1C3-pChr_alignment" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 19: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green = Prefix/Suffix
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150728_pUniprom-ChrB" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 20: Plates after transformation of, Top: <I>chrB</I> into pUniprom; Bottom: <I>chrB</I> (opt) into pUniprom. Ratio insert:vector for ligations: left: 3:1; Right: 4:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem040" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 21: Gel after restriction digest of Left: pUniprom-<I>chrB</I>, and Right: pUniprom-<I>chrB</I> (opt) with BamHI/PstI for confirmation of insert size. Expected size 939bp. <I>chrB</I> colony 2 and <I>chrB</I> (opt) colony 3 were selected for sequencing.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150729_chrb-combined" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 22: Alignments of: Left: <I>chrB</I> and Right: <I>chrB</I> (opt). Colours: Pink = Restriction sites (BamHI/PstI); Blue = 6-His Tag; Red = In-frame stop codons; Cyan = optimised codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150731_pChrGFP-ligations" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 23: Plates after ligation of <I>gfp</I> into pSB1C3-P<I>chr</I>. Left: Ratio Insert:Vector 2:1; Right: Ratio Insert:Vector 3:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem043" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 24: Gel for size confirmation of pSB1C3-P<I>chr-gfp</I> (Top). Bottom: Left - Haptoglobin, Right - Lanosterol Synthase. pSB1C3-P<I>chr-gfp</I> (3) and pSB1C3-P<I>chr-gfp</I> (4) were submitted for sequencing.
        </DIV></DIV><DIV class="modal-body">'</DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150801_pchrgfp" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 25: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> and <I>gfp</I> (mut2) with the forward and reverse sequencing results of P<I>chr-gfp</I> this region. The sequence was as expected, it was hence called pSB1C3-P<I>chr-gfp</I> (B). Colour code: Green = Prefix/Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150809_blot" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 26: Top: anti-GFP: Single colonies of JM110 + pSB1C3-P<I>chr-gfp</I> (A), MC1061 + pSB1C3-P<I>chr-gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> chloramphenicol. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of each sample was mixed with 100 µl laemmli buffer, and boiled for 10min. 3 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and P<I>manA-gfp</I> as a positive control. 
        Bottom: anti-6-His - ChrB: Single colonies of JM110 pUniprom-<I>chrB</I> and JM110 pUniprom-<I>chrB</I> (opt) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 200 µl laemmli buffer, and the sample boiled for 10min. 10 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and <I>hydA - 6-His</I> as a positive control.
        Expected sizes: GFP ~ 27kDa, ChrB ~35kDa.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150811_transformations" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 27: Plates after transformation of single constituents of chromate sensor into <I>E. coli</I> MG1655. From top to bottom: pUniprom-<I>chrB</I> (from colony 2 - 29/07), pUniprom-<I>chrB</I> (opt) (from colony 3 - 29/07), pSB1C3-P<I>chr-gfp</I> (B), pSB1C3-P<I>chr-gfp</I> (A), <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150822_blot" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 28: Western blot against GFP: Single colonies of MG1655, containing all 4 combinations, pUniprom-<I>chrB</I> - pSB1C3-P<I>chr-gfp</I> (A), pUniprom-<I>chrB</I> - pSB1C3-P<I>chr-gfp</I> (B), pUniprom-<I>chrB</I> (opt) - pSB1C3-P<I>chr-gfp</I> (A), and pUniprom-<I>chrB</I> (opt) - pSB1C3-P<I>chr-gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> chloramphenicol and 100 <SUP>µg</SUP>/<SUB>ml</SUB> ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml PBS. 100 µl of these samples was mixed with 100 µl laemmli buffer, and the sample boiled for 10 min. The indicated volume of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3, and pUniprom were included as a negative control, and P<I>manA-gfp</I> as a positive control. The blot against 6-His – ChrB was without success and is not shown. No chromate of any kind was added to the medium previous to this blot, hence no expression of GFP was expected. 
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV>


K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB><DIV class="modal fade" id="dundee15_chr_150824_platereader_BBa_K1058008" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 29: F,OD-plot of results of plate reader experiment after exposing MG1655 + BBa_K1058008 to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="icecream" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 30: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="potato" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 31: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt) to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150824_platereader_pchrbgfp" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 32: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_chrbbielefeld" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 33: Plates after ligation of <I>chrB</I> (Bielefeld) into pT7KS. From top to bottom: Ratio insert:vector 2:1; Ratio Insert:Vector 3:1; pT7KS religation control.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_bl21" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 34: Plates after transformation of single constituents of chromate sensor into <I>BL21 (DE3)</I>. From top to bottom: pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt), pUniprom-<I>chrB</I>, and pUniprom-<I>chrB</I> (opt).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_igem047" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 35: 1% agarose gel after colony PCR of pT7KS-<I>chrB</I> (Bielefeld). Top: colonies 1-6; Bottom: colonies 7-12. Ladder: 1kb+ Expected size of <I>chrB</I> (Bielefeld): 939bp
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_platereaderinduced" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 36: F,OD-plot of results of plate reader experiment after inducing expression of ChrB from the T7 promoter of pUniprom <I>BL21 (DE3)</I> pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, and pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt). IPTG was added after 4 hours. One measurement was made every 10min across 22h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>