<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_pmodal skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/pmodal</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="modal fade" id="PCR-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>PCR Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the  section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY color="black"><TR><TD width="312" valign="top"><P align="center"><STRONG>Reactants</STRONG></P></TD><TD width="312" valign="top"><P align="center"><STRONG>Volume (µl)</STRONG></P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DNA
                </P></TD><TD width="312" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DMSO
                </P></TD><TD width="312" valign="top"><P align="center">
                    2.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   100 µM Forward Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   100 µM Reverse Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    5 x Herculase Buffer
                </P></TD><TD width="312" valign="top"><P align="center">                10
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   10 mM dNTPs
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    H<SUB>2</SUB>O
                </P></TD><TD width="312" valign="top"><P align="center">
                    34.5
                </P></TD></TR></TBODY></TABLE><P>
    Elongation time was altered according to the length of the sequences.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="restrictiondigest-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Restriction Digests Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P><FONT color="#000000">
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the    <STRONG>Lab Book</STRONG> section.
</FONT></P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="170" valign="top" style="color:#000000;"><P align="center"><STRONG>Post Plasmid Purification</STRONG></P></TD><TD width="151" valign="top"><P align="center"><STRONG>Post PCR</STRONG></P></TD><TD width="139" valign="top"><P align="center"><STRONG>gBlock</STRONG></P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>DNA</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1 mg
                </P></TD><TD width="151" valign="top"><P align="center">
                    50 µl
                </P></TD><TD width="139" valign="top"><P align="center">
                    100 ng
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Cutsmart Buffer (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    6
                </P></TD><TD width="151" valign="top"><P align="center">
                    6
                </P></TD><TD width="139" valign="top"><P align="center">
                    6
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Water (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    As required
                </P></TD><TD width="151" valign="top"><P align="center">
                    1
                </P></TD><TD width="139" valign="top"><P align="center">
                    As required
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Enzyme 1 (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="151" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="139" valign="top"><P align="center">
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Enzyme 2 (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="151" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="139" valign="top"><P align="center">
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG><U>Total Volume (µl)</U></STRONG></P></TD><TD width="170" valign="top"><P align="center">          30
                </P></TD><TD width="151" valign="top"><P align="center">
                    60
                </P></TD><TD width="139" valign="top"><P align="center">
                    30
                </P></TD></TR></TBODY></TABLE><P>
    The digests were incubated at 37<SUP>o</SUP>C for 3 hours.
    
    Note that when performing plasmid digestions, 2.5 µl of alkaline phosphatase and 2.5 µl of Cutsmart Buffer was added at the 2 hour and 2.5 hour mark. Digested PCR products were separated from restriction enzymes and buffers using Strataclean resin (Aligent Technologies). Digested vectors were purified on a 1% agarose gel and then extracted. 
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ligation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Ligations</B></CENTER></DIV></DIV><DIV class="modal-body"><P><FONT color="black">
Ligation reactions were set up according to the table below.
</FONT></P><TABLE border="1" cellspacing="0" cellpadding="0" class="text"><TBODY><TR><TD width="111" valign="top"><P align="center"><STRONG>Vector (µl)</STRONG></P></TD><TD width="111" valign="top"><P align="center"><STRONG>2:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>3:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>4:1 (Vector: Insert) [µl]</STRONG></P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>T4 DNA Ligase </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Ligase Buffer </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Vector </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Insert</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    -
                </P></TD><TD width="111" valign="top"><P align="center">
                    2
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>H<SUB>2</SUB>0</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    7
                </P></TD><TD width="111" valign="top"><P align="center">
                    5
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD></TR></TBODY></TABLE><P>
    The reactions were incubated at room temperature for at least 3 hours.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Transformation Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>

    1. 100 µl of chemocompetent <I>E.coli</I> cells were defrosted. Thaw on ice.
</P><P>
    2. Agar plates were taken out of 4 <SUP>o</SUP>C to warm to room temperature.
</P><P>
    3. 10 µl of ligation reaction or 2 µl of plasmid purified DNA was added to cells. The cells and DNA were mixed by gently flicking the tube.
</P><P>
    4. Cell and DNA mixture was left on ice for 20 minutes.
</P><P>
    5. Mixture was placed in 42 <SUP>o</SUP>C water bath for 90 seconds.
</P><P>
    6. Mixture was put back on ice for 2 minutes.
</P><P>
    7. 1 ml of LB (without antibiotic) was added to cells.
</P><P>
    8. Cells were allowed to grow for 1 hour at 37 <SUP>o</SUP>C in a shaking incubator.
</P><P>
    9. Cells were then spun down and the supernatant discarded.
</P><P>
    10. The pellet was resuspended in the 100 µl of remaining media.
</P><P>
    11. Cells were plated on agar plates with appropriate antibiotic.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="miniprep-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Plasmid Purification</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. 1-5 ml bacterial overnight culture was pelleted by centrifugation at &gt;8000 rpm for 3 minutes at room temperature.
</P><P>
    2. Pelleted bacterial cells were resuspended in 250 µl of Buffer P1 and transferred to a microcentrifuge tube.
</P><P>
    3. 250 µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution becomes clear.
</P><P>
    4. 350 µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.
</P><P>
    5. The mixture was centrifuged for 10 minutes at 13000 rpm in a microcentrifuge.
</P><P>
    6. The supernatant was applied to a QIAprep spin column by pipetting.
</P><P>
    7. The QIAprep spin column was washed with 500 µl of Buffer PB by centrifugation for 1 minute and flow through discarded.
</P><P>
    8. 750µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute and flow through discarded.
</P><P>
    9. The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
</P><P>
    10. The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 50 µl of H<SUB>2</SUB>O to the QIAprep spin column and centrifuging for 1 minute.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="onc-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Overnight Cultures </B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>
    1. Four colonies were picked from the desired plate.
</P><P>
    2. For each colony, 5 ml of fresh LB plus the appropriate antibiotics was inoculated and grown overnight at 37 <SUP>o</SUP>C.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sds-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>SDS-PAGE and Western Blotting</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>
    1. 50 µl of an overnight culture was used to inoculate 5 ml of fresh LB containing the appropriate antibiotics and left to grow at 37<SUP>o</SUP>C in a
    shaking incubator.
</P><P>
    2. Once the OD<SUB>600</SUB>=0.6-0.8, 1 mM of IPTG was added to the culture and left to grow for a further 3 hours.
</P><P>
    3. 1 ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.
</P><P>
    4. Meanwhile, 950 µl of 2x Laemmli sample buffer was added to 50 µl of B-mercaptoethanol.
</P><P>
    5. The supernatant was discarded from the samples and the pellet resuspended in 200 µl of the above solution.
</P><P>
    6. The samples were then boiled at 90<SUP>o</SUP>C for 10 minutes.
</P><P>
    7. Afterwards, they were spun down for 1 minute in a microcentrifuge.
</P><P>
    8. At this stage, the samples could be stored at -20<SUP>o</SUP>C or ran on a gel immediately.
</P><P>
    9. Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the
    stacking gel at which point the voltage was increased to 200V for 20 minutes.
</P><P>
    10. One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.
</P><P>
    11. After the proteins had been transferred onto the membrane using a semi dry transfer ran for 40 minutes at 5V, the membrane was blocked in 5% Marvel milk overnight at 4<SUP>o</SUP>C.
</P><P>
    12. The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10 ml of 1xTBS buffer to which the primary anti-His antibody was
    added and left for one hour. The protein ladder was cut off before adding the primary antibody.
</P><P>
    13. The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10 ml of 1xTBS buffer, to which the secondary antibody was
    added and left for one hour.
</P><P>
    14. The membrane was then washed in 3x 10 ml 1xTBST buffer for 5 minutes.
</P><P>
    15. The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off
    at step 12.
</P><P>
    16. Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Growth Curve Assay Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>    1. 5ml overnight cultures of the appropriate strains were set up.
</P><P>
    2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
</P><P>
    3. Three were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced.
</P><P>
    4. 200 µl of each culture was loaded onto a 96 well microtitre plate with 4 repeats.
</P><P>
    5. The cultures were left to grow for 16 hours in the plate reader which took OD readings every 10 minutes.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Growth Curve Assay Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. 5ml overnight cultures of the appropriate strains were set up.
</P><P>
    2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
</P><P>
    3. 20 x 200 µl aliquots were loaded onto a 96 well microtitre plate.
</P><P>
    4. After the OD<SUB>600</SUB>=~0.6, three cultures were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced (with 4 repeats).
</P><P>
    5. The cultures were left to grow for 20 hours in the plate reader which took OD readings every 30 minutes.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="pp-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Protein Purification Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. A 150 ml overnight culture was set up for the desired protein.
</P><P>
    2. 3 x 1 L of fresh LB containing 1 ml of appropriate antibiotics was inoculated with each 50 ml overnight culture and left to grow until an OD<SUB>600</SUB>
    =0.6-1 in a shaking incubator at 37<SUP>o</SUP>C.
</P><P>
    3. The cultures were subsequently induced with 1 mM IPTG and left to grow for 6 hours in a shaking incubator at 20<SUP>o</SUP>C.
</P><P>
    4. They were then centrifuged for 20 minutes at 4200 rpm at 4<SUP>o</SUP>C.
</P><P>
    5. The pellets were then resuspended in 5 ml of ice cold 50 mM 7.5 Tris and transferred to a 50 ml falcon tube and then topped up to 40 ml with 50 mM 7.5 Tris.
</P><P>
    6. They were then spun down again at 4000 rpm for 15 minutes at 4<SUP>o</SUP>C, the supernatant discarded and pellets stored at -20<SUP>o</SUP>C.
</P><P>
    7. Once purification could proceed, the pellets were thawed on ice and resuspended in 40 ml of 50 mM Tris containing DNAase, 1 x protease inhibitor tablet
    and lysozyme. (Note: If purifying a membrane protein, 1% DDM and 5 M urea was also added).
</P><P>
    8. The cells were lysed using a cell disruptor.
</P><P>
    9. Afterwards the lysate was spun down in a J25.50 rotor at 15000 rpm at 4<SUP>o</SUP>C for 40 minutes.
</P><P>
    10. The supernatant was removed and transferred to a 50 ml falcon tube. The pellet was kept for testing. Both were kept on ice.
</P><P>
    11. Using FPLC, the supernatant was loaded onto a Ni<SUP>2+</SUP> column.
</P><P>
    12. The protein was eluted using an increasing concentration of imidazole. The two buffers used were Buffer A: 50 mM Tris, 200 mM NaCl, 20 mM imidazole and
    Buffer B: 50 mM Tris, 200 mM NaCl, 1 M Imidazole. (Note: If purifying a membrane protein 0.02% DDM was added to these buffers.)
</P><P>
    13. The fractions corresponding to any peaks were retained and concentrated using an appropriate filter size. A small amount of each fraction could be ran
    on an SDS gel at this stage.
</P><P>
    14. Using FPLC, the sample was then loaded onto a SEC column and any fractions corresponding to a peak were ran on an SDS gel to check for the presence of
    the desired protein.
</P><P>
    15. Fractions containing the desired protein were collated, to which 20% glycerol was added.
</P><P>
    16. The samples were then flash frozen and stored at -80<SUP>o</SUP>C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ap-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Assay Preparation</B></CENTER></DIV></DIV><DIV class="modal-body"><OL><LI> Take 3 independent colonies from each plate of your controls and samples and place each colony into 5 ml of LB with the appropriate antibiotics. </LI><LI> Leave to grow overnight at 30°C.</LI><LI> Take 50 μl of each overnight and inoculate 5 ml of fresh LB containing the appropriate antibiotics. Do this twice for every independent colony. </LI><LI> Allow these to grow at 30°C shaker until the O.D of each inoculate is between 0.3-0.5</LI><LI> Once the O.D is between 0.3-0.5 take 3 x 1 ml samples of each culture and place into separate Eppendorf tubes.</LI><LI> Take a further 1 ml and place into a cuvette and  subsequently measure the OD<SUB>600</SUB> marking down the OD for each replica. </LI><LI> Spin down the 1 ml cultures of each replica at 13,000 rpm for 3 minutes. </LI><LI> Remove the supernatant from each 1 ml culture and freeze cell pellet for later use in the β-galactosidase Assay. </LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bga-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>β-Galactosidase Assay</B></CENTER></DIV></DIV><DIV class="modal-body"><P><OL><LI> Resuspend cell pellet in 800 μl of buffer 2 (100 mM dibasic sodium phosphate (Na2HPO4); 20 mM KCl ; 2 mM MgSO4 ;0.8 mg/mL CTAB (hexadecyltrimethylammonium bromide) ; 0.4 mg/mL sodiumdeoxycholate) and subsequntly add 5.4 μL/mL beta-mercaptoethanol. </LI><LI> Add 50 μl of SDS (0.1%) to each sample. </LI><LI> Add 50 μl of Chloroform to each sample and vortex for 30 seconds. </LI><LI> Make substrate solution (60 mM Na2HPO4; 40 mM NaH2PO4; 1 mg/mL o-nitrophenyl-β-D-Galactoside (ONPG); 2.7 μL/mL β-mercaptoethanol)</LI><LI> Add 200 μl of substrate solution to each sample marking down the time it was added in relation to the first sample. </LI><LI> Wait for an observable color change and mark down the time that the color change occurred.</LI><LI> Add 600 μl of NaCO3(1M) once the color change has occurred.</LI><LI> Spin down the samples at 13,000 rpm for 3 minutes and place the supernatant into a cuvette. </LI><LI> Measure the OD<SUB>420</SUB>. </LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plate Reader Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>A 5 ml overnight culture of the desired strain was grown at 37 <SUP>o</SUP>C.</LI><LI>Equalise OD<SUB>600</SUB> if required. Dilute culture 1:1000.</LI><LI>Aliquot 198 µl into the required amount of wells in a 96-well plate.</LI><LI>Add additional substances and controls as required.</LI><LI>Preheat platereader to required temperature and start experiment.</LI><LI>Retrieve and analyse data.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="competent-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Competent Cells Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>A 5 ml overnight culture of the desired strain was grown at 37<SUP>o</SUP>C.</LI><LI>Subsample <SUP>1</SUP>/<SUB>10</SUB> of the volume of the subculture. I.e. 1 ml of the overnight culture into 10 ml fresh LB.</LI><LI>Grow subculture at at 37 <SUP>o</SUP>C for 2 h.</LI><LI>Spin down subculture at 4000 rpm at 4<SUP>o</SUP>C for 10 min.</LI><LI>Resuspend the pellet in ice cold transformation buffer. Use <SUP>1</SUP>/<SUB>10</SUB> of the initial volume of the subculture.</LI><LI>Use 100 µl per transformation.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="plating-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plating Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>Fresh agar plates were inoculated with cells on the toothpick.</LI><LI>Agar plates were incubated at 4<SUP>o</SUP>C over night.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="tf-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Tryptophan Fluorescence Protocol</DIV></DIV><DIV class="modal-body"><P>
    1. Intensity of tryptophan fluorescence of PotD was measured on a fluorimeter using an excitation of 280 nm and emission of 350 nm.
</P><P>
    2. Increasing concentrations of spermidine were added in 50 µM increments to give a final concentration of 250 µM spermidine to 5 µM of purified PotD.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sequencing-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Sequencing Protocol</DIV></DIV><DIV class="modal-body"><P>
    For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
      <UL><LI>Volume of sample equivalent to 550ng DNA</LI><LI>2 µl of 3 mM primer</LI><LI>Remaining volume of water for a total volume of 30 µl</LI></UL></P><P>
      The samples were submitted to <A href="https://www.dnaseq.co.uk/" target="_blank">DNA sequencing and services</A>.
    </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="overnightculture-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Overnight Culture Protocol</DIV></DIV><DIV class="modal-body"><P><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>5 ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.</LI><LI>Tubes were incubated at 37<SUP>o</SUP>C in a rotary incubator at 200rpm over night.</LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="gelextraction-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Gel Extraction Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.</LI><LI>The gel slice was transferred into a microcentrifuge tube. 700 µl of Buffer QG was added.</LI><LI>The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.</LI><LI>After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.</LI><LI>230 µl of isopropanol was added to the sample and mixed.</LI><LI>A QIAquick spin column was placed in a 2 ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>500 µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>750 µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>The QIAprep spin column was spun for an additional minute to remove residual wash buffer.</LI><LI>The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>