<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_Forensic_Toolkit_FluID skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/Forensic Toolkit/FluID</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><H1><HIGHLIGHT class="highlight">The FluID Detector</HIGHLIGHT></H1><H3><HIGHLIGHT class="highlight">Reports</HIGHLIGHT></H3><DIV class="container"><H1>Overview of the FluID detector</H1><P>One of the components of our forensic toolkit is the FluID- Body Fluid Detector - an all in one, cell free spray that will be used to detect four common body fluids namely blood, semen, nasal mucus and saliva. We wanted to exploit the fact that in each body fluid there are natural ligands that are either unique to each or highly abundant. The aim was to use specific binding proteins to target such components. We managed to overexpress and purify PotD and haptoglobin, the binding proteins that will target spermidine in semen and haemoglobin in blood, respectively. We are yet to purify the binding proteins for the two other body fluids although significant progress has been made towards achieving this goal. To fully characterise our binding proteins we also needed to overexpress the components that they are targeting, if it is a protein-protein interaction. For this, only haemoglobin B has been successfully purified but its interaction with haptoglobin still needs to be fully tested. However, characterization of the PotD/spermidine interaction has been started via tryptophan fluorescence with positive results. Please visit our project page to read about all the individual components of our FluID detector.</P></DIV><SECTION id="about" class="row1"><DIV class="row"><DIV class="col-lg-3"><H3>Blood</H3><P class="about-content">Click here to jump to our blood lab report where we purify the human proteins haptoglobin and haemoglobin B</P></DIV><DIV class="col-lg-3"><H3>Semen</H3><P class="about-content">Click here to jump to our lab report on trying to detect semen at crime scenes. We successfully purify and characterise the bacterial protein PotD</P></DIV><DIV class="col-lg-3"><H3>Saliva</H3><P class="about-content">Click here to jump to our lab report on trying to detect saliva and our methods involved in trying to get lactoferrin binding protein A over-expressed in <I>E. coli</I></P></DIV><DIV class="col-lg-3"><H3>Nasal Mucus</H3><P class="about-content">Click here to see our lab report on trying to detect nasal mucus. Here, we use a novel method aiming to displace wild-type OBP with our own mutated form of the protein.</P></DIV></DIV></SECTION><DIV class="container"><DIV class="col-md-12"><CENTER><H1>Blood</H1></CENTER><H2>Introduction</H2><P>There is a lot of potential information contained in a blood stain, which is why forensic investigators are so keen to seek it out. Most importantly, DNA can be extracted from blood samples and can be linked to a suspect or can eliminate a suspect from suspicion. The blood spatter pattern can reveal information about the type of weapon used and the position of the victim and culprit. Finally, blood typing can be used as an initial test to exclude some suspects however, blood type alone is not enough to identify a suspect because lots of people will also share the same blood type.</P><P>There are currently several different methods for detecting blood at a crime scene, each with its own advantages and disadvantages. The main methods used are Luminol or ALS (alternate light source) ‘crime-lite’. 
              These crime-lite’s are essentially a box of different coloured torches. Different biological fluids will absorb light at different wavelengths and are thus more visible to the naked eye using this technique. However, a positive reaction using this technique is not really confirmation for the presence of body fluid since other substrates may also fluoresce. It can also be a very slow process at the crime scene, with different crime lights, being used one at a time on all surfaces to try and identify any body fluid present.</P><P>Luminol (which you might have seen on TV programmes such as CSI) is a compound used to detect blood at a crime scene. The investigator prepares a solution of luminol and sprays it on the area under investigation. The lights are then turned off and any traces of blood show up as a bright blue glow. This is because the luminol reacts with iron in blood to produce chemiluminescence (which is the emission of light during a chemical reaction).

              The main advantage of Luminol is that it glows in the presence of very small quantities of iron so even trace amounts of blood can be detected. However, the disadvantages of this technique are that the chemiluminescence can also be triggered by a number of substances such as copper and bleach. This means that if a crime scene has been cleaned with bleach, the bleach residue will cause the entire crime scene to glow blue and this camouflages any traces of blood that might be present. In addition, the reaction tends to be very fast, only around 30 seconds before it stops glowing. Our spray will hopefully fluoresce for much longer, giving plenty of time for the scene to be analysed and photographed without the need for constant reapplication of the spray.</P><P>For the blood detection we have decided to exploit the interaction between haptoglobin and haemoglobin.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 1</B> - Structure of the human haptoglobin-haemoglobin complex. </P></FIGCAPTION></FIGURE><P>Haemoglobin is the tetrameric protein molecule in red blood cells that carries oxygen. It is composed of four polypeptide chains, which in adults consist of two alpha (a) globin chains and two beta (b) globin chains. In blood plasma, haptoglobin binds free haemoglobin released from red blood cells, inhibiting its oxidative activity. The haptoglobin-haemoglobin complex can then be removed by the reticuloendothelial system which is a part of the immune system. Haemoglobin is still found free in the blood plasma at a concentration of up to 0.1g/l and engineered haptoglobin therefore has the potential to bind to, and potentially allow detection of, any free haemoglobin found in the environment. 

</P><P>The aim was to design a cell-free system using highly pure haptoglobin. This requires overexpression of the synthetic gene and purification, via an engineered affinity tag, of the recombinant protein and attachment to a carboxylate-modified fluorescent microspheres. So in theory, when our FluID is applied to a blood sample the haptoglobin will bind haemoglobin and allow for easy visual detection of the fluorescence given off. </P><H2>Results</H2><P>Firstly, Human haptoglobin, haemoglobin A and haemoglobin B were optimized for expression in <I>E. coli </I> and modified to ensure they were compatible with BioBrick specifications and standards. These genes were then synthesized by IDT (Integrated DNA Technologies) and we initially cloned them into pSB1C3 to give the following BioBricks; <A href="http://parts.igem.org/Part:BBa_K1590000" target="_blank"><U>BBa_K1590000</U></A>, <A href="http://parts.igem.org/Part:BBa_K1590001" target="_blank"><U>BBa_K1590001</U></A>, and <A href="http://parts.igem.org/Part:BBa_K1590002" target="_blank"><U>BBa_K1590002</U></A>.
The synthetic genes were then sub-cloned into the pQE80-L overexpression vector that adds an N-terminal hexa-histidine tag onto the protein which allows for purification by immobilized metal affinity chromatography (IMAC).  



</P><P>To characterise the interactions between haptoglobin and haemoglobin we wanted to use both in vivo and in vitro techniques. To do this, the idea was to carry out crosslinking experiments with the purified synthetic genes as well as a bacterial two hybrid screen with haptoglobin, haemoglobin alpha and/ or haemoglobin beta. Crosslinking is the process of chemically joining two or more molecules by a covalent bond. This attachment between groups on two different proteins results in intermolecular crosslinks that allow a protein-protein interaction to be stabilised, captured, and analysed using a western immunoblot.</P><P>The bacterial two hybrid screen, based off of the original yeast two hybrid system uses a reconstituted transcription factor and enzymatic activities to detect physical interactions between proteins. You can read more about the theory behind the bacterial two hybrid system, linked below.</P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary btn-margin pull-right" href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid System</A></DIV><P>We managed to successfully overexpress Human haptoglobin and haemoglobin B in <I>E. coli </I>and purify both proteins. Unfortunately haemoglobin A subunit proved more difficult to overexpress. </P><H3>Overexpression and Purification of Haemoglobin B</H3><H4>Overexpression of haemoglobin B.</H4><P>Initial experiments were carried out to optimize expression of haemoglobin B within our <I>E.coli</I> chassis. We found the following conditions to be optimum: we subcultured 50µl of an overnight culture into fresh LB containing the appropriate antibiotics and grew the cells at 37°C until an OD<SUB>600</SUB> of ~0.6 was reached. The production of protein was then induced by adding 1mM (final concentration) of IPTG then the cells were grown for a further 3 hours at 37°C. Next, 1ml of this culture was then taken for analysis by western immunoblot which showed that haemoglobin B was successfully overexpressed. The results from this are illustrated below in Figure 2.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 1 - Overexpression of haemoglobin B.</B> Single colonies of M15[pREP4] <I>E. coli</I> containing pQE80-L encoding h<I>HBB</I> were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin and kanamycin. These overnight cultures were then subcultured and grown at 37°C. Once cultures reached an OD<SUB>600</SUB> of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours at 37°C, after which 1ml aliquots were pelleted, re-suspended in 100µl of Laemmli buffer and 20µl of these samples were separated by SDS-PAGE (12% acrylamide). This was transferred to a nitrocellulose membrane and probed with an anti-his antibody. Western analysis shows production of of Human Haemoglobin B in <I>E. coli</I> (expected size - 16kDa).</P></FIGCAPTION></FIGURE><H4>Purification of haemoglobin beta.</H4><P>This was then scaled up and 4 litres of <I>E. coli</I> containing pQE80-L h<I>HBB</I> was grown for purification of haemoglobin B by immobilized metal affinity chromatography (IMAC).  The optimum conditions for this were as follows: 4 x 1L of fresh LB growth medium containing ampicillin and kanamycin was inoculated with 50ml of haemoglobin beta overnight culture and left to grow until an OD<SUB>600</SUB> of between 0.6-1 at 37°C. The expression of protein was then induced by adding 1mM of IPTG then the cells were grown for a further 6 hours at 20°C. Cells were then pelleted and washed in a buffer of 50mM Tris-HCl pH7.5. Cells were then lysed and centrifuged to remove any cell debris. A crude extract was loaded onto a nickel affinity column and the protein was eluted with an imidazole gradient. The results from this can be seen in Figure 3.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 3 - Purification of Human Haemoglobin B by nickel IMAC</B>. A crude extract was loaded onto a nickel affinity column and the protein was eluted with an imidazole gradient (0-1000mM). The fractions corresponding to a peak were collected.  </P></FIGCAPTION></FIGURE><P>The fractions corresponding to the IMAC peak were retained and concentrated down to 500μl for further purification by size exclusion chromatography (SEC). Figure 4.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 4 - Characterisation of haemoglobin B following SEC (size exclusion chromatography). A)</B> The sample of concentrated fractions containing haemoglobin B from nickel affinity purification was loaded onto an SEC column and the protein was eluted. <B>B)</B> 10µl of each fraction corresponding to the two observed peaks was mixed with 10µl of laemmli buffer and loaded onto an SDS gel. The bands observable on the gel are in line with the expected size of haemoglobin beta - 16kDa. <B>C)</B> Western immunolot was then carried out against an anti-his antibody to confirm the presence of haemoglobin B. </P></FIGCAPTION></FIGURE><P>We have succesfully purified haemoglobin B and our next steps were to express and purify its binding partner haptoglobin. </P><H3>Overexpression and purification of Human Haptoglobin</H3><H4>Purification of haptoglobin</H4><P>Like haemoglobin B, human haptoglobin was overproduced and purified by nickel IMAC. The eluted fractions were analysed by SDS- PAGE and Western immunoblotting (Figure 5).</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 5 - Purification of human haptoglobin by nickel IMAC</B>. A crude extract was loaded onto a nickel affinity column and the protein was eluted with an imidazole gradient (0-1000mM). The fractions corresponding to a peak were collected. <B>A)</B> 10 μl of Laemmli buffer and loaded onto an SDS gel. The bold bands observable in fractions A9-A11 are in the line with the expected size of haptoglobin - ~45kDa. <B>B)</B> Samples separated by SDS-PAGE were transferred to a nitrocellulose membrane and probed with an anti-His antibody to confirm the presence of haptoglobin. 
</P></FIGCAPTION></FIGURE><P>The fractions obtained from nickel IMAC showed production of haptoglobin and so were concentrated down to 500 µl and loaded onto the Superdex 75 10/300 to carry out size exclusion chromatography. Results from this can be seen in Figure 6.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 6 - Further characterization of human haptoglobin following SEC. A) </B>Chromatogram showing the purification profile of the His-tagged human haptoglobin. The fractions corresponding to the two peaks observed on the chromatograph were further analysed by SDS page gel and western blotted. <B>B)</B> 10µl of the fractions A8-A9 and A9-A10 corresponding to peaks 1 + 2, respectively were mixed with 10µl of laemmli buffer and loaded onto a 12.5% SDS-PAGE gel. Band A observed on the gel corresponds to the expected size of haptoglobin – 45kDa. However, it is not clear what the bands present at ~37kDa may be. <B>C)</B> Samples separated by SDS-PAGE were transferred to a nitrocellulose membrane and probed with an anti-his antibody. This Western blot analysis confirmed the presence of human haptoglobin.</P></FIGCAPTION></FIGURE><P>To further identify the expressed protein at ~37kDa, the band was sent off for analysis by tryptic peptide mass spectrometry. This technique uses the proteolytic enzyme trypsin which cleaves at the carboxyl side of arginine and lysine residues. The sizes of the peptide fragments obtained after trypsin digestion, represent the peptide mass fingerprint and are characteristic of each protein. The peptide mass fingerprint spectrum of fragments derived through trypsin digest indicated that haptoglobin was indeed purified with good coverage of the amino acid sequence to the expected sequence (Figure 7). This shows that we have purified two forms of our protein, one containing the histidine tag and the other which has lost the tag. An attempt will be made to attach both of these to the fluorospheres.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 7 - Mass spectrometry analysis of haptoglobin.</B>The expected sequence of haptoglobin is shown here with the matched peptides from the mass spectrometry analysis shown in bold red.</P></FIGCAPTION></FIGURE><H2>Conclusion and Future Work</H2><P>Further work is needed to optimize the conditions for the expression of haemoglobin A. This has proved difficult and has been one of the biggest challenges in the lab for the blood detector aspect of FluID. To do this we could alter growth conditions such as temperature to decrease the rate of protein synthesis and hopefully lead to a more soluble protein being expressed. We could also alter the concentration of IPTG added to induce protein over expression or add sucrose to help stabilize the protein structure, this works by increasing the osmotic pressure within the <I>E. coli</I> cells leading to the expression of osmoprotectants which stabilise native proteins.</P><P>We would also hope to thoroughly test the interaction of haptoglobin with haemoglobin both in vivo and in vivo as well as both attached to and separate from a flourosphere to see whether fusion of haptoglobin to a fluorescent particle might alter the protein-protein interaction in any way. This would involve competing our crosslinking experiments of haptoglobin to haemoglobin alpha and beta using two different fixing agents - DSS (disuccinimidyl suberate) and formaldehyde.  

            </P><P>DSS reacts with primary amines which are generally found on lysine side chains and the N-terminal of the peptide chain to form amide bonds. While, formaldehyde, a classic fixative, also reacts with primary amines to form Schiff bases, and with amides to form hydroxymethyl compounds which can condense with another amide moiety to form methyl diamides.</P><P>To fully test the interaction in vivo our bacterial two hybrid screen would need to be completed. So far we have managed to test haemoglobin alpha with haemoglobin beta together and haptoglobin together with haemoglobin alpha. However, this has only been carried out in one configuration meaning we would need to repeat these experiments with our proteins fused to the opposite T18 or T25 fragment. Because the plasmids encoding these fragments have different copy numbers, the results from our assay could be affected by this. Steric hindrance may also occur in only one configuration so both need to be tested to avoid a false negative result due to this. We would also need to test haptoglobin against haemoglobin alpha to complete the bacterial two hybrid screen.</P><P>Finally, it would be great to be able to test our devices sensitivity. To do this our haptoglobin fused flourospheres would need to be let loose on real blood samples of different concentrations and the binding affinity measured.</P><DIV class="btn-margin pull-right"><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID" class="btn btn-primary btn-lg" role="button">Our Lab Records for FluID</A><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID#blood" class="btn btn-primary btn-lg" role="button">Our FluID Modelling</A><A href="#selection" class="btn btn-info btn-lg" role="button">↑</A></DIV></DIV></DIV><DIV class="container"><DIV class="col-md-12"><CENTER><H1>Semen</H1></CENTER><H2>Introduction</H2><P>For semen detection our main target ligand is the polyamine spermidine which is found in relatively high concentrations in seminal fluid (5-15 mM). Spermidine is made from another polyamine called putrescine and is the precursor of spermine. 
        Regulation of polyamine synthesis, degradation and transport is tightly controlled in bacteria. In <I>E. coli</I>, two of three identified transport systems are ABC transporters composed of a periplasmic binding protein, a pair of transmembrane proteins and a membrane protein possessing ATPase catalytic activity. 
        Out of these three components, we were interested in investigating the periplasmic binding protein PotD which specifically binds spermidine. The fact that this protein is responsible for transportation of spermidine and lacking in enzymatic activity meant that it was an ideal candidate for use in FluID for semen detection.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 1</B> - Crystal structure of PotD in complex with its ligand spermidine. PotD (blue) is a binding protein specific to spermidine (red) which is a polyamine found in semen.</P></FIGCAPTION></FIGURE><P>As with the other binding proteins, our aim was to overexpress this protein in <I>E. coli</I>, purify it and attach to it carboxylate-modified fluorescent microspheres.</P></DIV><DIV class="col-md-12"><H2>Results</H2><P>The gene encoding PotD was amplified from <I>E. coli</I> MG1655 and cloned into both the biobrick vector (<A href="http://parts.igem.org/Part:BBa_K1590009" target="_blank"><U>BBa_K1590009</U></A>) pSB1C3 and the high expression vector pQE80-L. The pQE80-L vector allows for inducible expression of our target proteins along with an N-terminal poly histidine tag.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 2. Characterisation of PotD. Single colonies of MC1061 <I>E. coli</I> containing pQE80-L encoding PotD were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin. Once cultures reached an OD<SUB>600</SUB> of 0.6, one was induced with 1mM IPTG and one was left uninduced as a control. They were left to grow for a further three hours, after which a 1ml sample was taken and the cells pelleted. The cells were resuspended in 100µl of Laemmli buffer, 20µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.</B> - </P></FIGCAPTION></FIGURE><P>Upon successful expression of PotD-His 3 L cultures were grown up for protein purification purposes, under the same conditions. However, instead of allowing a further 3 hours growth after induction, the cultures were left for 12 hours at 37°C. This was to ensure that plenty of protein was being produced and since PotD is native to <I>E. coli</I>, leaving them to grow this long did not have a significantly detrimental impact upon the cells. We used His tag affinity and size exclusion chromatography to purify PotD. The fractions taken from nickel IMAC were concentrated to a volume of 500 µl and further purified using SEC.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 3</B> - Characterization of PotD. A) Chromatogram showing the purification profile of the his-tagged bacterial protein, PotD. The fractions corresponding to the two peaks observed on the chromatograph were further analysed on SDS page gel. B) Coomassie Stain of the purified fractions (A12 + B12). 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V and then stained.  These were then Western immunoblotted to confirm the presence of our target protein PotD-His. </P></FIGCAPTION></FIGURE><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 4</B> - Samples from the SEC were ran on a 12.5% SDS-PAGE gel and then transferred to a nitrocellulose membrane and probed with an anti-His antibody. The band detected in the blot corresponds to the expected size of PotD of 37kDa.</P></FIGCAPTION></FIGURE><P>The next step was to characterize binding interactions between PotD and its ligand spermidine. This was carried out via tryptophan fluorescence; a technique which detects conformational changes of proteins by measuring changes in the emission spectra of tryptophan residues. Essentially, if there is an interaction occurring between PotD and spermidine, the spectra would shift indicating so.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 5. Characterisation of PotD/Spermidine Interaction. The graph shows that there is an increase in tryptophan quenching as concentration of spermidine increases. Increasing concentrations of spermidine were added in 50 µM increments to give a final concentration of 250 µM spermidine to 5 µM of purified PotDIntensity of tryptophan fluorescence of PotD was measured on a fluorimeter using an excitation of 280 nm and emission of 350 nm. The graph shows that there is an increase in tryptophan quenching as concentration of spermidine increases. 

</B> - </P></FIGCAPTION></FIGURE></DIV><DIV class="col-md-12"><H2>Conclusion and Future Work</H2><P>In terms of future work, the interaction between PotD and spermidine would need to be characterized more robustly perhaps using other methods such as ITC or SPR. This would help establish binding parameters which would be critical in informing how much protein we would need to put in our device to allow for reliable detection. Once this is achieved, attachment of the fluorescent microspheres would be performed. It would also be ensured that this modification would not interfere with the PotD/spermidine interaction.</P><DIV class="btn-margin pull-right"><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID" class="btn btn-primary btn-lg" role="button">Our Lab Records for FluID</A><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID#semen" class="btn btn-primary btn-lg" role="button">Our FluID Modelling</A><A href="#selection" class="btn btn-info btn-lg" role="button">↑</A></DIV></DIV></DIV><DIV class="container"><DIV class="col-md-12"><CENTER><H1>Saliva</H1></CENTER><H2>Introduction</H2><P>A major component which is detectable in saliva is the free iron sequestering compound known as lactoferrin, a protein involved in the innate immune system. Neisseria meningitidis is a gram-negative bacterium with an iron-binding outer membrane protein called lactoferrin binding protein A (LbpA). <I>N. meningitidis</I> uses this LbpA to extract iron from the host lactoferrin under pathogenic conditions to allow for the bacterium to perform essential cellular metabolism such as energy generation and DNA replication.</P><P>We predict that purified LbpA would be able to bind to the lacteroferrin present in saliva in vitro. Building on this, we hope it attach a fluorescent molecular probe to LbpA to allow for signal output of bound LbpA (with unbound LbpA washed away from saliva). Here, we aim to overexpress LbpA with a HIS6 tag in <I>E. coli</I> and purify by FPLC and SEC. From here we seek to characterise binding of LbpA to lactoferrin by tryptophan fluorescence and ultimately attach carboxylate fluorescent microspheres to LbpA.</P><H2>Results</H2><P>In order to do this chromosomal DNA from <I>Neisseria meningitidis</I> MC58 strain was kindly gifted to us by Dr James Moir from The University of York. This was used as a template for the amplification of the lbpA gene into the pSB1C3 plasmid (<A href="http://parts.igem.org/Part:BBa_K1590008" target="_blank"><U>BBa_K1590008</U></A>) and then subcloned into the high expression plasmid pQE80-L.</P><P>The pQE80-L – lbpA plasmid was then transformed into M15 pREP4 to allow for more tightly controlled protein expression. <I>E. coli</I> cultures harbouring pQE80-L encoding LbpA were subcultured and grown to an OD<SUB>600</SUB> = 0.6 before being induced with 1mM IPTG. Following 6 hours induction the cells appeared to die as shown in Figure 1.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 1 - LbpA Culture Samples.</B>Lane 1: Sample from uninduced culture. Lane 2: Sample from culture induced with 1 mM IPTG. On comparison of the two cultures, it seems that inducing expression of LbpA causes the cells to die given the significant reduction of protein levels visible on the gel. </P></FIGCAPTION></FIGURE><P>In order to investigate this further, we performed plate reader experiments where different concentrations of IPTG were used to induce LbpA expression. The cells were also given a longer period of time to grow. Although cells started to grow again ~7 hours after induction, this was determined to be due to the emergence of mutants.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 2 - Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B>The graph shows that on comparison with the uninduced control, the induced cultures stop growing when LbpA expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.</P></FIGCAPTION></FIGURE><P>Despite this, we did try to grow large cultures of M15 <I>E. coli</I> containing pQE80-L-LbpA¬, to see if we could detect any expression. We attempted this under a number of different growing conditions, however, none of these were successful. It is suspected that LbpA forms inclusion bodies and therefore insoluble which makes it difficult to purify.</P><H2>Future Work</H2><P>We would continue to try to find the the optimum conditions required to express LbpA and prevent them from forming inclusion bodies. However, we could also prepare and extract the inclusion bodies from <I>E. coli</I> and therefore purify LbpA.</P><P>If we were able to successfully purify LbpA we would, as in the case for the other binding proteins, attempt to attach the carboxylate modified fluorescent microspheres to LbpA and characterize its interaction with lactoferrin. </P><DIV class="btn-margin pull-right"><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID#saliva" class="btn btn-primary btn-lg" role="button">Our Lab Records for FluID</A><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID" class="btn btn-primary btn-lg" role="button">Our FluID Modelling</A><A href="#selection" class="btn btn-info btn-lg" role="button">↑</A></DIV></DIV></DIV><DIV class="container"><DIV class="col-md-12"><CENTER><H1>Nasal Mucus</H1></CENTER><H2>Introduction</H2><P>On the detection of nasal mucus our main protein of focus was the human odorant binding protein 2A (OBP2A). It is a 155 amino acid (excluding the signal peptide) lipocalin of relatively low molecular weight (19318 Daltons). Structurally it is an 8 sheet beta barrel flanked by a c-terminal alpha helix that that together forms an internal hydrophobic pore known as a calix. It is secreted by the olfactory epithelial cells of the nose where it lies in high abundance within nasal mucus. Its primary function in the human body is believed to be in the transport of hydrophobic odorant proteins across the otherwise impenetrable aqueous mucus layer to the olfactory receptors of the nose. Due to its high specificity and abundance within nasal mucus, OBP2A was selected as the protein for use in nasal mucus detection.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 1</B> - Illustrations of the proposed crystal structure of OBP2A at 2.60 Angstroms. Image: Left shows the 8 strand antiparallel beta barrel in blue. Alpha helix are show in green. Interacting cysteine residues are colored red. Image: Right shows the two different areas of OBP2A that were separated to better understand their interaction, the larger beta barrel is shown in blue and the small c-terminal tail show in red.</P></FIGCAPTION></FIGURE><P>In all other aspects of our FluID detector we have targeted a naturally occurring ligand interaction as the basis for subsequent nanobead attachment.  In the case of nasal mucus however we were unable to utilize such a pathway as our research suggested that there were no known interactions of high enough specificity for use in the strict detection of nasal mucus. What was our solution?  We aimed to separate the alpha helix c-terminal tail of OBP2A and use that as the substrate for fluorescent nanobead attachment. How would that detect the presence of nasal mucus? Through the use of PCR mutagenesis we aimed to improve the binding affinity that the separated c-terminal alpha helix would have for the rest of the OBP2A protein.  This in turn we hoped would displace the natively conformed c-terminal tail of OBP2A in exchange for our tighter binding synthetic tail. This synthetic tail could then be attached to a florescent nanoparticle and used in the detection of nasal mucus.</P><H2>Results</H2><P>The gene that encodes for OBP2A was first successfuly placed into the pSB1C3 biobrick vector and transformed into <I>E. coli</I> JM110. It is from there that OBP2A was separated into two parts. This was done so that we could first characterize the interaction of the two separated parts and prove that under normal conditions the two constituents would coalesce back into a completed OBP2A protein in vivo. This was done through the use of primers designed to amplify the different sections of the protein that were to be separated.  It was decided through observation of the amino acid sequence and through OBP2A crystal structure that a proline at position 118 in the amino acid sequence would be the position of separation for the two parts of OBP2A.</P><P>To characterize the interaction between the two separate parts of OBP2A the bacterial two hybrid system (BTH) was implemented and β-galactosidase assays were done.  This meant that these separated gene fragments had to be cloned into BTH vectors: pUT18 and pT25. These vectors contain gene fragments for the production of adenylate cyclase (an enzyme that is required for the production of active cyclic-AMP) that becomes active when the integrated gene fragment’s products interact with each other in vivo. This allows us to quantify the interaction through the use of the β-galactosidase assay. Once the gene fragments of OBP2A were successfully cloned into the BTH vectors, both vectors were transformed into MG1655 (Δcya) <I>E. coli</I> strain – a strain that lacks the necessary genes for the production of adenylate cyclase. It was from here that multiple β-galactosidase assays were performed and the miller’s activity( a measure of the level of interaction) was calculated for each control and sample.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 1</B> - Calculation used to determine the level of interaction between two proteins in vivo.</P></FIGCAPTION></FIGURE><P>As of yet these assay have indicated that the two subunits of OBP2A aren’t interacting in vivo. One suggestion as to why this may be the case is that the proteins when separated have exposed hydrophobic residues that cause an alteration of the protein structure of the two parts making them incompatible with each other. It is important to point out however that of the β-galactosidase assays that were done all of them have had controls that have failed to function, so it is impossible at this stage to definitively tell that the two separated parts of OBP2A aren’t interacting in vivo.</P><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure 1</B> - Above, illustrates the calculated miller’s activity of each control/sample and suggest that the OBP2A subunits aren’t interacting. As can be gauged from the sample on the far right of the graph. This suggests that OBP2A has a lower interaction than the negative and regulatory controls. </P></FIGCAPTION></FIGURE><H2>Conclusion and Future Work</H2><P>The next stage of experimentation we aim to do is to have repeated the beta-galactosidase assay, using more and better  controls to eliminate the current issues that the assay are  having with failed controls. Along with this we also aim to clone the two parts of OBP2A into the BTH vectors but in the opposite vectors to see if that has any effect on the interaction of the two parts of OBP2A. If this had shown considerable change in the level of interaction between the two parts of OBP2A then we would have to reconsider the use of bacterial two hybrid as the method for determining protein interaction.</P><DIV class="btn-margin pull-right"><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID#nasal" class="btn btn-primary btn-lg" role="button">Our Lab Records for FluID</A><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID" class="btn btn-primary btn-lg" role="button">Our FluID Modelling</A><A href="#selection" class="btn btn-info btn-lg" role="button">↑</A></DIV></DIV></DIV><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV><DIV class="container"><DIV class="footer-btn"><A href="https://2015.igem.org/Team:Dundee/Introduction_Video" class="btn btn-info btn-lg" role="button">Watch Our Introduction Video</A></DIV></DIV><FOOTER><DIV class=""><DIV class="container"><DIV class="col-md-3"><A href="http://www.dundee.ac.uk/" target="_blank"><H4>Dundee University</H4></A><P>Located on the east coast of Scotland, The University of Dundee is one of Scotland's leading research institutes.</P></DIV><DIV class="col-md-3"><H4>Social Media</H4><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="https://www.youtube.com/channel/UCJHBLvLhjzGqHqdjABjOY-w">youtube.com/DundeeiGEM2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.facebook.com/igemdundee2015">facebook.com/iGEMDundee2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.twitter.com/dundeeigemteam">twitter.com/DundeeiGEMTeam</A></DIV></DIV></DIV></DIV></DIV></FOOTER><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>