Team:Dundee/Lab Book/FluID
×
PCR Protocol
PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the
section.
Reactants
Volume (µl)
DNA
1
DMSO
2.5
100 µM Forward Primer
0.5
100 µM Reverse Primer
0.5
5 x Herculase Buffer
10
10 mM dNTPs
0.5
Herculase
0.5
H2O
34.5
Elongation time was altered according to the length of the sequences.
Close
×
Restriction Digests Protocol
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the Lab Book section.
Post Plasmid Purification
Post PCR
gBlock
DNA
1 mg
50 µl
100 ng
Cutsmart Buffer (µl)
6
6
6
Water (µl)
As required
1
As required
Enzyme 1 (µl)
1.5
1.5
1.5
Enzyme 2 (µl)
1.5
1.5
1.5
Total Volume (µl)
30
60
30
The digests were incubated at 37oC for 3 hours.
Note that when performing plasmid digestions, 2.5 µl of alkaline phosphatase and 2.5 µl of Cutsmart Buffer was added at the 2 hour and 2.5 hour mark. Digested PCR products were separated from restriction enzymes and buffers using Strataclean resin (Aligent Technologies). Digested vectors were purified on a 1% agarose gel and then extracted.
Close
×
Ligations
Ligation reactions were set up according to the table below.
Vector (µl)
2:1 (Vector: Insert) [µl]
3:1 (Vector: Insert) [µl]
4:1 (Vector: Insert) [µl]
T4 DNA Ligase
1
1
1
1
Ligase Buffer
1
1
1
1
Vector
1
1
1
1
Insert
-
2
3
4
H20
7
5
4
3
The reactions were incubated at room temperature for at least 3 hours.
Close
×
Transformation Protocol
1. 100 µl of chemocompetent E.coli cells were defrosted. Thaw on ice.
2. Agar plates were taken out of 4 oC to warm to room temperature.
3. 10 µl of ligation reaction or 2 µl of plasmid purified DNA was added to cells. The cells and DNA were mixed by gently flicking the tube.
4. Cell and DNA mixture was left on ice for 20 minutes.
5. Mixture was placed in 42 oC water bath for 90 seconds.
6. Mixture was put back on ice for 2 minutes.
7. 1 ml of LB (without antibiotic) was added to cells.
8. Cells were allowed to grow for 1 hour at 37 oC in a shaking incubator.
9. Cells were then spun down and the supernatant discarded.
10. The pellet was resuspended in the 100 µl of remaining media.
11. Cells were plated on agar plates with appropriate antibiotic.
Close
×
Plasmid Purification
1. 1-5 ml bacterial overnight culture was pelleted by centrifugation at >8000 rpm for 3 minutes at room temperature.
2. Pelleted bacterial cells were resuspended in 250 µl of Buffer P1 and transferred to a microcentrifuge tube.
3. 250 µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution becomes clear.
4. 350 µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.
5. The mixture was centrifuged for 10 minutes at 13000 rpm in a microcentrifuge.
6. The supernatant was applied to a QIAprep spin column by pipetting.
7. The QIAprep spin column was washed with 500 µl of Buffer PB by centrifugation for 1 minute and flow through discarded.
8. 750µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute and flow through discarded.
9. The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
10. The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 50 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.
Close
×
Overnight Cultures
1. Four colonies were picked from the desired plate.
2. For each colony, 5 ml of fresh LB plus the appropriate antibiotics was inoculated and grown overnight at 37 oC.
Close
×
SDS-PAGE and Western Blotting
1. 50 µl of an overnight culture was used to inoculate 5 ml of fresh LB containing the appropriate antibiotics and left to grow at 37oC in a     shaking incubator.
2. Once the OD600=0.6-0.8, 1 mM of IPTG was added to the culture and left to grow for a further 3 hours.
3. 1 ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.
4. Meanwhile, 950 µl of 2x Laemmli sample buffer was added to 50 µl of B-mercaptoethanol.
5. The supernatant was discarded from the samples and the pellet resuspended in 200 µl of the above solution.
6. The samples were then boiled at 90oC for 10 minutes.
7. Afterwards, they were spun down for 1 minute in a microcentrifuge.
8. At this stage, the samples could be stored at -20oC or ran on a gel immediately.
9. Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the     stacking gel at which point the voltage was increased to 200V for 20 minutes.
10. One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.
11. After the proteins had been transferred onto the membrane using a semi dry transfer ran for 40 minutes at 5V, the membrane was blocked in 5% Marvel milk overnight at 4oC.
12. The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10 ml of 1xTBS buffer to which the primary anti-His antibody was     added and left for one hour. The protein ladder was cut off before adding the primary antibody.
13. The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10 ml of 1xTBS buffer, to which the secondary antibody was     added and left for one hour.
14. The membrane was then washed in 3x 10 ml 1xTBST buffer for 5 minutes.
15. The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off     at step 12.
16. Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.
Close
×
Growth Curve Assay Protocol
1. 5ml overnight cultures of the appropriate strains were set up.
2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
3. Three were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced.
4. 200 µl of each culture was loaded onto a 96 well microtitre plate with 4 repeats.
5. The cultures were left to grow for 16 hours in the plate reader which took OD readings every 10 minutes.
Close
×
Growth Curve Assay Protocol
1. 5ml overnight cultures of the appropriate strains were set up.
2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
3. 20 x 200 µl aliquots were loaded onto a 96 well microtitre plate.
4. After the OD600=~0.6, three cultures were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced (with 4 repeats).
5. The cultures were left to grow for 20 hours in the plate reader which took OD readings every 30 minutes.
Close
×
Protein Purification Protocol
1. A 150 ml overnight culture was set up for the desired protein.
2. 3 x 1 L of fresh LB containing 1 ml of appropriate antibiotics was inoculated with each 50 ml overnight culture and left to grow until an OD600 =0.6-1 in a shaking incubator at 37oC.
3. The cultures were subsequently induced with 1 mM IPTG and left to grow for 6 hours in a shaking incubator at 20oC.
4. They were then centrifuged for 20 minutes at 4200 rpm at 4oC.
5. The pellets were then resuspended in 5 ml of ice cold 50 mM 7.5 Tris and transferred to a 50 ml falcon tube and then topped up to 40 ml with 50 mM 7.5 Tris.
6. They were then spun down again at 4000 rpm for 15 minutes at 4oC, the supernatant discarded and pellets stored at -20oC.
7. Once purification could proceed, the pellets were thawed on ice and resuspended in 40 ml of 50 mM Tris containing DNAase, 1 x protease inhibitor tablet     and lysozyme. (Note: If purifying a membrane protein, 1% DDM and 5 M urea was also added).
8. The cells were lysed using a cell disruptor.
9. Afterwards the lysate was spun down in a J25.50 rotor at 15000 rpm at 4oC for 40 minutes.
10. The supernatant was removed and transferred to a 50 ml falcon tube. The pellet was kept for testing. Both were kept on ice.
11. Using FPLC, the supernatant was loaded onto a Ni2+ column.
12. The protein was eluted using an increasing concentration of imidazole. The two buffers used were Buffer A: 50 mM Tris, 200 mM NaCl, 20 mM imidazole and     Buffer B: 50 mM Tris, 200 mM NaCl, 1 M Imidazole. (Note: If purifying a membrane protein 0.02% DDM was added to these buffers.)
13. The fractions corresponding to any peaks were retained and concentrated using an appropriate filter size. A small amount of each fraction could be ran     on an SDS gel at this stage.
14. Using FPLC, the sample was then loaded onto a SEC column and any fractions corresponding to a peak were ran on an SDS gel to check for the presence of     the desired protein.
15. Fractions containing the desired protein were collated, to which 20% glycerol was added.
16. The samples were then flash frozen and stored at -80oC.
Close
×
Assay Preparation
Take 3 independent colonies from each plate of your controls and samples and place each colony into 5 ml of LB with the appropriate antibiotics.
Leave to grow overnight at 30°C.
Take 50 μl of each overnight and inoculate 5 ml of fresh LB containing the appropriate antibiotics. Do this twice for every independent colony.
Allow these to grow at 30°C shaker until the O.D of each inoculate is between 0.3-0.5
Once the O.D is between 0.3-0.5 take 3 x 1 ml samples of each culture and place into separate Eppendorf tubes.
Take a further 1 ml and place into a cuvette and  subsequently measure the OD600 marking down the OD for each replica.
Spin down the 1 ml cultures of each replica at 13,000 rpm for 3 minutes.
Remove the supernatant from each 1 ml culture and freeze cell pellet for later use in the β-galactosidase Assay.
Close
×
β-Galactosidase Assay
Resuspend cell pellet in 800 μl of buffer 2 (100 mM dibasic sodium phosphate (Na2HPO4); 20 mM KCl ; 2 mM MgSO4 ;0.8 mg/mL CTAB (hexadecyltrimethylammonium bromide) ; 0.4 mg/mL sodiumdeoxycholate) and subsequntly add 5.4 μL/mL beta-mercaptoethanol.
Add 50 μl of SDS (0.1%) to each sample.
Add 50 μl of Chloroform to each sample and vortex for 30 seconds.
Make substrate solution (60 mM Na2HPO4; 40 mM NaH2PO4; 1 mg/mL o-nitrophenyl-β-D-Galactoside (ONPG); 2.7 μL/mL β-mercaptoethanol)
Add 200 μl of substrate solution to each sample marking down the time it was added in relation to the first sample.
Wait for an observable color change and mark down the time that the color change occurred.
Add 600 μl of NaCO3(1M) once the color change has occurred.
Spin down the samples at 13,000 rpm for 3 minutes and place the supernatant into a cuvette.
Measure the OD420.
Close
×
Plate Reader Protocol
A 5 ml overnight culture of the desired strain was grown at 37 oC.
Equalise OD600 if required. Dilute culture 1:1000.
Aliquot 198 µl into the required amount of wells in a 96-well plate.
Add additional substances and controls as required.
Preheat platereader to required temperature and start experiment.
Retrieve and analyse data.
Close
×
Competent Cells Protocol
A 5 ml overnight culture of the desired strain was grown at 37oC.
Subsample 1/10 of the volume of the subculture. I.e. 1 ml of the overnight culture into 10 ml fresh LB.
Grow subculture at at 37 oC for 2 h.
Spin down subculture at 4000 rpm at 4oC for 10 min.
Resuspend the pellet in ice cold transformation buffer. Use 1/10 of the initial volume of the subculture.
Use 100 µl per transformation.
Close
×
Plating Protocol
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
Fresh agar plates were inoculated with cells on the toothpick.
Agar plates were incubated at 4oC over night.
Close
×
Tryptophan Fluorescence Protocol
1. Intensity of tryptophan fluorescence of PotD was measured on a fluorimeter using an excitation of 280 nm and emission of 350 nm.
2. Increasing concentrations of spermidine were added in 50 µM increments to give a final concentration of 250 µM spermidine to 5 µM of purified PotD.
Close
×
Sequencing Protocol
For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
Volume of sample equivalent to 550ng DNA
2 µl of 3 mM primer
Remaining volume of water for a total volume of 30 µl
The samples were submitted to DNA sequencing and services .
Close
×
Overnight Culture Protocol
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
5 ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.
Tubes were incubated at 37oC in a rotary incubator at 200rpm over night.
Close
×
Gel Extraction Protocol
The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.
The gel slice was transferred into a microcentrifuge tube. 700 µl of Buffer QG was added.
The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.
After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.
230 µl of isopropanol was added to the sample and mixed.
A QIAquick spin column was placed in a 2 ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
500 µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
750 µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.
Close
×
Figure 1: Amplification of LbpA The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 2: LbpA PCR. Lane 1: PCR Product, Lane 2: 1kb Ladder and Marker. The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 3: LbpA PCR for Cloning into pQE80-L. Lane 1: LbpA PCR Product, Lane 2: 1kb Ladder and Marker The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 4: LbpA Culture Samples. Lane 1: Sample from uninduced culture. Lane 2: Sample from culture induced with 1mM IPTG. It seems that on comparison of the two cultures, that inducing expression of LbpA causes the cells to die given the significant reduction of protein levels visible on the gel.
Close
×
Figure 5: OD600 Readings from Uninduced Culture and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG). The table shows that after being induced with IPTG the cultures stop growing since their OD600 readings are notably lower than that of the uninduced control.
Close
×
Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, the induced cultures stop growing when LbpA expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 7: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, after reaching an OD600 of 0.6,  the induced cultures stop growing when LbpA starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have dealt with the production of the foreign protein.
Close
×
Figure 8: SDS Gel of Samples Taken from Uninduced Control and 1 mM IPTG Induced Culture 6 hours After Induction For the induced culture, a faint band just above the 100 kDa marker is observable on the gel which corresponds to the expected size of LbpA.
Close
×
Figure 1: Gel of SBP PCR Product for Cloning into pQE80-L. Lane 1: SBP PCR Product 1 Lane 2: 1kb Ladder and Marker Lane 3: SBP PCR Product 2 Both bands observable on the gel correspond to the expected size of SBP which is ~600 bp.
Close
×
Figure 2: SDS Gel of Uninduced and 1 mM IPTG Induced Cultures of PotD and SBP. For PotD, it is clear from the gel that it is being successfully overexpressed since there is an intense band present at the 37 kDa marker which corresponds to the expected size of PotD in the induced culture sample. For SBP, there is no band present which suggests SBP is being overexpressed.
Close
×
Figure 3: Characterisation of PotD. Single colonies of MC1061 E.coli containing pQE80-L encoding PotD were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin. Once cultures reached an OD600 of 0.6, one was induced with 1 mM IPTG and one was left uninduced as a control. They were left to grow for a further three hours, after which a 1ml sample was taken and the cells pelleted. The cells were resuspended in 100µl of Laemmli buffer, 20µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 4: SDS Gel of Uninduced and 0.5mM, 1mM and 2mM IPTG Induced Cultures of SBP. It appears that upon induction of SBP with any concentration of IPTG the cells are dying given that there is a significant drop in protein levels observable on the gel.
Close
×
Figure 5: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, the induced cultures stop growing when SBP expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, after reaching an OD600 of 0.6,  the induced cultures stop growing when SBP starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 7: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of PotD 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100 V. The two bands observable on the gel correspond to the expected size of PotD of 38 kDa. Western Blotting will confirm if these proteins are PotD.
Close
×
Figure 8: Characterisation of PotD following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 9: Characterisation of SBP following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 10: Characterisation of PotD/Spermidine Interaction. The graph shows that the change in the tryptophan spectra increases as concentration of spermidine increases.
Close
×
Figure 1: pSB1C3-hHBA and pSB1C3-hHBB (respectively) Pre-Sequencing Digest pSB1C3-hHBA and pSB1C3-hHBB minipreps were digested with EcoRI and PstI to check for presence of each insert. The bands observable on the gel correspond to the expected sizes for the pSB1C3 vector and both inserts.
Close
×
Figure 2: Amplification of hHBA and hHBB hHBA and hHBB were amplified from the pSB1c3-hHBA and pSB1c3-hHBB minipreps obtained yesterday using primers to allow cloning of the genes into pT25 and pUT18 respectively. Although the band is fainter in the case of hHBA, the gel indicates the PCR has been successful.
Close
×
Figure 3: Amplification of hHBA hHBA was amplified from the pSB1C3-hHBA using primers to allow cloning of the genes into pT25. The gel indicates that the PCR has been successful.
Close
×
Figure 4: Amplification of hHBA and hHBB hHBA and hHBB were amplified from the pSB1c3-hHBA and pSB1C3-hHBB minipreps using primers to allow cloning of the genes into pQE80-L. The gel indicates that the PCR has been successful.
Close
×
Figure 5: Pre-Sequence Digest of pQE80-L-hHBA and pQE80-L-hHBB Both pQE80-L-hHBA and pQE80-L-hHBB minipreps were digested with BamHI and KpnI to check for the presence of the inserts. The gel indicates that both inserts have been successfully cloned into pQE80-L.
Close
×
Overexpression of hHBB and hHBA. Single colonies of M15[pREP4] E.coli containing pQE80-L encoding hHBB and pQE80-L encoding hHBA were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD600 of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of 2x Laemmli buffer, 20 µl of which was separated by SDS-PAGE.
Close
×
Figure 7: Characterisation of hHBB. Single colonies of M15[pREP4] E.coli containing pQE80-L encoding hHBB were used to inoculate 5 ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD600 of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 8: Amplification of hHBN for Cloning into pQE80-L PCR product from amplification of hHBN from IDT plasmid using primers for cloninig into pQE80-L. The gel indicates hHBN has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.
Close
×
Figure 9: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of hHBB 10 µl of each fraction was mixed with 10 µl of laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. The four bands observable on the gel correspond to the expected size of hHBB of 16 kDa. Western Blotting will confirm if these proteins are hHBB.
Close
×
Figure 10: Characterisation of hHBB following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 11: Amplification of hHBN for Cloning into pT25 PCR product from amplification of hHBN from IDT plasmid using primers for cloninig into pT25. The gel indicates hHBN has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.
Close
×
Figure 14: Characterisation of hHBN following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 15: Amplification of hHBN hHBN was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of hHBN of 1200 bp.
Close
×
Figure 15: Amplification of hHBA hHBA was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of hHBA of
Close
×
Figure 16: Amplification of hHBN hHBN was amplified using primers to allow for subsequent cloning into the two hybrid vector pUT18. The band observable on the gel corresponds to the expected size of hHBN of 1200 bp.
Close
×
Figure 1: Amplification of OBP2A for insertion into pSB1C3 vector . This is the PCR product from the amplification of OBP2A from the IDT plasmid using primers for cloning into pSB1C3. The gel indicates OBP2A has been amplified successfully since the observed band corresponds to the expected size of 500 bp. The concentration of the gel extracted OBP2A is 335.53 ng/ul.
Close
×
Figure 2: A subsequent gel was run of the amplified OBP2A after it was first excised from the gel in Figure 1 This was to determine if the gel extraction was successful. The gel indicates that OBP2A has been extracted from the gel successfully, since the observed band corresponds to the expected size of 500 bp. This can now be restricted in preparation for ligation into pSB1C3.
Close
×
Figure 3: A Gel Analysis of Restriction Digest from the 24/6. This gel analysis was done to determine if the reason for the failed transformation was an improperly digested OBP2A. This gel indicates that the restricted product hasn't been lost as the OBP2A gene fragment can be seen as a distinct band at 500 bp. A re-ligation of OBP2A into the pSB1C3 backbone will now occur.
Close
×
Figure 4: Presequence Digest of pSB1C3-OBP2A. All seven plasmid purifications of the overnight cultures done on the 26/6 were digested to see if they contained the OBP2A insert. It is clear from the gel that all but sample 3 contain the insert for OBP2A as they all have faint bands at 500 bp's which corresponds to the size of OBP2A. It also appears that sample 5 contains the highest concentration, so that will be the sample that is sent for sequencing.
Close
×
Figure 5: Presequence digest of pUT18-OBP2A To determine if any samples contain the OBP2A insert. It can be seen from this gel that of the  5 samples that were digested only two are show to have the OBP2A insert, therefore sample 2 will be sent for sequencing.
Close
×
Figure 6: Amplification of OBP2A for insertion into pT25 vector . This is the PCR product from the amplification of OBP2A for cloning into pT25. The gel indicates OBP2A has been amplified successfully since the observed band corresponds to the expected size of 100 bp.
Close
×
Figure 7: Presequence Digest of pT25-OBP2A In order to determine if any of the samples contain the OBP2A insert. It can be seen from this gel that samples 5, 6 and 7 may have inserts in them as they each have two distinct bands, one of which corresponds to the 100 bp size for the OBP2A for pT25. As a result all three samples will be sent for sequencing.
Close
×
Figure 8: β-galactosidase Assay A graph of the results from the first attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of OBP2A. As it can be seen that the positive result has failed(far left) not much can be inferred from the interaction of the two parts of OBP2A. However it looks like the two subunits of OBP2A may not be interacting. As this was a first attempt, subsequent assays will be performed in order to determine if in fact the two separated parts of OBP2A aren't interacting in vivo - like these results suggests.
Close
×
Figure 9: β-galactosidase Assay A graph of the results from the second attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of OBP2A.  It can also be inferred from this graph that the two parts of OBP2A don't seem to be interacting as it has a low millers activity when compared to the controls. Further assays will be performed in order to determine if the two separated parts of OBP2A aren't interacting in vivo like these results suggests.
Close
×
Figure 11: β-galactosidase Assay Under Anaerobic Conditions A graph of the results from the anaerobic β-galactosidase assay aimed to characterize the interaction between the two separated parts of OBP2A. The graph seems to suggest that the two subunits of OBP2A may not be interacting in vivo. This can be inferred from the fact that the millers activity of the OBP2A sample lies lower than the negative controls from within the experiment. As a result of this information, one more assay will be prepared in which the two subunits will be switched with regards to the bacterial two hybrid vectors to eliminate any possibility of this being a factor effecting the interaction between the two subunits.
Close
×
Figure 1: Amplification of LS LS was amplified using primers which would allow for subsequent cloning into the high expression vector pQE80-L. Lane 1 shows that the PCR has been successful since the band corresponds to the expected size of 2100 bp.
Close
×
Figure 2: Amplification of LS LS was amplified from the IDT plasmid using primers to allow cloning of the gene into pSB1C3. The observed band corresponds to the expected size of LS, 2100 bp.
Close
×
Figure 3: Overexpression Assays of LS 5 ml cultures of M15pREP4 E.coli was grown to OD600=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37oC, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel.
Close
×
Figure 4: Characterisation of Lanosterol Synthase 5 ml cultures of M15pREP4 E.coli was grown to OD600=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37oC, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. The proteins were transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 5: Chromatogram following Affinity Chromatography of LS Using FPLC, the protein sample was loaded onto a nickel column and eluted with an increasing concentration of imidazole (green line). The fractions corresponding to the peak were ran on an SDS gel.
Close
×
Figure 6: SDS Gel of Peak Fractions 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer an loaded onto a SDS gel.
Close
×
Figure 1: Alignment of BBa_K1058008 . Sequences from top to bottom: sequencing result from forward primer; expected sequence; reverse complement of sequencing result from reverse primer; sequencing result from intermediate primer. Colour code: Green = Prefix/Suffix; Pink = Annealing site of intermediate primer; Yellow = Mismatch in one sequence; Red = In frame stop codons.
'
Close
×
Figure 2: 1% agarose gel after PCR of BBa_K1058008 with primers specific to the promoter region, Pchr, and the repressor region, chrB. Left: Pchr (expected size 169bp); Middle: 1kb+ ladder, Right: chrB (expected size 939bp). Amplification of chrB was not successful. Pchr was further processed by restriction digest with EcoRI and PstI.
'
Close
×
Figure 3: 1% agarose gel after PCR of BBa_K1058008 with primers specific to the repressor region, chrB. Left: 1.5 µl DMSO; Middle: 1kb+ ladder, Right: 2.5 µl DMSO. Expected size for each: 939bp. The annealing temperature for the PCR amplification was lowered to 50°C in order to increase the chances of accomodating a primer that is not 100% complementary.
'
Close
×
Figure 4: 1% agarose gel after PCR of gfp with corresponding primers. Left: GFP (expected size 717bp); Middle: 1kb+ ladder, Right: PCR product of chrB, using primers to add a BamHI restriction site in front of it and a 6-His-Tag and a HindIII restriction site at the end. (expected size of chrB: 939bp). Since amplification of chrB was unsuccessful, it will be repeated.
'
Close
×
Figure 5: 1% agarose gel after restriction digest of pSB1C3-Pchr, pSB1C3-chrB (s), and pSB1C3-chrB (w) with EcoRI and PstI for confirmation of size. Ladder: 1kb+. Top: 1-4: Pchr 1-4; chrB (s) 1-2. Bottom: chrB (s) 3-4; chrB (w) 1-4. Expected sizes: Pchr: 169bp; chrB: 939bp.
'
Close
×
Figure 6: Alignment of Pchr sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green = Prefix/Suffix
'
Close
×
Figure 7: Alignment of chrB (opt) sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green: Prefix and Suffix; Red: In frame stop codons; Blue: First 15 codons optimised for expression in E. coli.
'
Close
×
Figure 8: 1% agarose gel after restriction digest of pUniprom (Expected size 2454bp).
'
Close
×
Figure 11: 1% agarose gel after restriction digest of pSB1C3 with EcoRI and SpeI (Expected size 2070bp).
'
Close
×
Figure 9: 1% agarose gel after PCR of chrB (opt). (Expected size 939bp).
'
Close
×
Figure 10: Plates after transformation of Top: pUnipromchrB, Middle: pUnipromchrB (opt) and Bottom: pSB1C3-Pchr-gfp into JM110.
'
Close
×
Figure 12: 1% agarose gel after restriction digest of pUnipromchrB, pUnipromchrB (opt) and pSB1C3-Pchr-gfp for confirmation of size. Ladder: 1kb+. Top: 1-4: pSB1C3-Pchr-gfp 1-4; pUnipromchrB (opt) 1-2. Bottom: pUnipromchrB (opt) 3-4; pUnipromchrB 1-4. Expected sizes: Pchr-gfp: 909bp; chrB and chrB (opt): 939bp.
'
Close
×
Figure 13: 1% agarose gel after colony PCR of pUnipromchrB (opt). Top: colonies 1-11; Bottom: colonies 12-22. Colonies 2, 4, 9, and 12 were selected for sequencing. Ladder: 1kb+ Expected size of chrB (opt): 939bp.
'
Close
×
Figure 14: Sequencing result of pSB1C3-Pchr-gfp after double ligation. Even though the sequence is as expected, it is ligated in reverse. It was hence called pSB1C3-Pchr-gfp (A) The Result for pSB1C3-Pchr-gfp (colony 2) was not as expected, albeit in the correct orientation. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 15: Plates with pSB1C3-Pchr-gfp in JM110 after double ligation under UV light. Numbered colonies were selected for initial size confirmation and sequencing. However, other colonies seemed to express GFP as well, hence a second attempt was made with different colonies from the same plate.
'
Close
×
Figure 16: Alignment of sequencing results of pSB1C3-Pchr-gfp - colonies 1, 5, 6, and 7 with sequences of gfp (mut2) and Pchr sequence. Colony 1 remained consistent, the other tested colonies were without success. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 17: 1% Agarose gel after restriction digest of pUniprom with BamHI/PstI (expected size: 2454bp). Right: PCR product of Haptoglobin.
'
Close
×
Figure 18: Plates after transformation of Pchr into pSB1C3. Top: ratio insert:vector 2:1; Bottom: ratio insert:vector 3:1.
'
Close
×
Figure 19: Alignment of Pchr sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green = Prefix/Suffix
'
Close
×
Figure 20: Plates after transformation of, Top: chrB into pUniprom; Bottom: chrB (opt) into pUniprom. Ratio insert:vector for ligations: left: 3:1; Right: 4:1.
'
Close
×
Figure 21: Gel after restriction digest of Left: pUniprom-chrB, and Right: pUniprom-chrB (opt) with BamHI/PstI for confirmation of insert size. Expected size 939bp. chrB colony 2 and chrB (opt) colony 3 were selected for sequencing.
'
Close
×
Figure 22: Alignments of: Left: chrB and Right: chrB (opt). Colours: Pink = Restriction sites (BamHI/PstI); Blue = 6-His Tag; Red = In-frame stop codons; Cyan = optimised codons.
'
Close
×
Figure 23: Plates after ligation of gfp into pSB1C3-Pchr. Left: Ratio Insert:Vector 2:1; Right: Ratio Insert:Vector 3:1.
'
Close
×
Figure 24: Gel for size confirmation of pSB1C3-Pchr-gfp (Top). Bottom: Left - Haptoglobin, Right - Lanosterol Synthase. pSB1C3-Pchr-gfp (3) and pSB1C3-Pchr-gfp (4) were submitted for sequencing.
'
Close
×
Figure 25: Alignment of Pchr sequence retrieved from BBa_K1058008 and gfp (mut2) with the forward and reverse sequencing results of Pchr-gfp this region. The sequence was as expected, it was hence called pSB1C3-Pchr-gfp (B). Colour code: Green = Prefix/Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 26: Top: anti-GFP: Single colonies of JM110 + pSB1C3-Pchr-gfp (A), MC1061 + pSB1C3-Pchr-gfp (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of each sample was mixed with 100 µl laemmli buffer, and boiled for 10min. 3 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and PmanA-gfp as a positive control.          Bottom: anti-6-His - ChrB: Single colonies of JM110 pUniprom-chrB and JM110 pUniprom-chrB (opt) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 200 µl laemmli buffer, and the sample boiled for 10min. 10 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and hydA - 6-His as a positive control.         Expected sizes: GFP ~ 27kDa, ChrB ~35kDa.
'
Close
×
Figure 27: Plates after transformation of single constituents of chromate sensor into E. coli MG1655. From top to bottom: pUniprom-chrB (from colony 2 - 29/07), pUniprom-chrB (opt) (from colony 3 - 29/07), pSB1C3-Pchr-gfp (B), pSB1C3-Pchr-gfp (A), BBa_K1058008 .
'
Close
×
Figure 28: Western blot against GFP: Single colonies of MG1655, containing all 4 combinations, pUniprom-chrB - pSB1C3-Pchr-gfp (A), pUniprom-chrB - pSB1C3-Pchr-gfp (B), pUniprom-chrB (opt) - pSB1C3-Pchr-gfp (A), and pUniprom-chrB (opt) - pSB1C3-Pchr-gfp (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol and 100 µg/ml ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml PBS. 100 µl of these samples was mixed with 100 µl laemmli buffer, and the sample boiled for 10 min. The indicated volume of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3, and pUniprom were included as a negative control, and PmanA-gfp as a positive control. The blot against 6-His – ChrB was without success and is not shown. No chromate of any kind was added to the medium previous to this blot, hence no expression of GFP was expected.
'
Close
K2CrO4, K2Cr2O7, CrCl3, and K2SO4
×
Figure 29: F,OD-plot of results of plate reader experiment after exposing MG1655 + BBa_K1058008 to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 30: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 31: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt) to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 32: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 33: Plates after ligation of chrB (Bielefeld) into pT7KS. From top to bottom: Ratio insert:vector 2:1; Ratio Insert:Vector 3:1; pT7KS religation control.
'
Close
×
Figure 34: Plates after transformation of single constituents of chromate sensor into BL21 (DE3). From top to bottom: pSB1C3-Pchr-gfp (B) + pUniprom-chrB, pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt), pUniprom-chrB, and pUniprom-chrB (opt).
'
Close
×
Figure 35: 1% agarose gel after colony PCR of pT7KS-chrB (Bielefeld). Top: colonies 1-6; Bottom: colonies 7-12. Ladder: 1kb+ Expected size of chrB (Bielefeld): 939bp
'
Close
×
Figure 36: F,OD-plot of results of plate reader experiment after inducing expression of ChrB from the T7 promoter of pUniprom BL21 (DE3) pSB1C3-Pchr-gfp (B) + pUniprom-chrB, and pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt). IPTG was added after 4 hours. One measurement was made every 10min across 22h.
'
Close
  University of Dundee
 Home
 About Us
Meet the Team
Attributions
 Project
Abstract
Project Overview
Devices
FluID
Fingerprint Ageing
Chromate Biosensor
Parts
Medals
Future Work
 Wet Lab
Lab Book
FluID
Fingerprints
Chromate Biosensor
Useful Information
Protocols
Sequences
Bacterial Two Hybrid
Safety
 Dry Lab
Introduction
FluID
Fingerprint Ageing
Chromate Biosensor
Appendix : Code
 Outreach
Practices
Ethics
Collaborations
FluID Detector
Lab Book
FluID
FluID is a cell free, all in one spray which will be used to detect and dinstiguish between body fluids found at crime scenes.
Chromate Biosensor
Our chromate biosensor is a device that will be used to detect stainless steel residues left behind on bones in blunt or sharp force trauma cases.
Fingerprint Ageing
This device will search for the presence of a specifc component found in fingerprint deposits whose presence/absence will indicate approximate fingerprint age.
Saliva
Week Beginning 1/6/2015
Summary
Lactoferrin binding protein A (LbpA) was successfully cloned into the vector pSB1C3.
1/6: Amplification of LbpA for Cloning into pSB1C3
Aim of Experiment: Amplify LbpA from MC58 N.meningitidis template chromosome.
Protocols Used: PCR
Results: Figure 1
Next Steps: Since the LbpA gene was amplified successfully, it was ready to be digested for subsequent cloning into pSB1C3.
2/6: Restriction Digests and Ligations with pSB1C3
Aim of experiment: To digest LbpA with BamHI and EcoRI and then ligate it into pSB1C3.
Protocols Used: Restriction Digests Ligations
Results: N/A
Next Steps: Ligations of pSB1C3-LbpA were left overnight until the next day when they would be transformed into the JM110 E.coli.
3/6: Transformations of pSB1C3-LpbA into JM110 E.coli
Aim of experiment: To transform the pSB1C3-LbpA ligations into JM110 E.coli.
Protocols Used: Transformations
Results: N/A
Next Steps: Transformations will be checked tomorrow and if successful, overnight cultures will be set up.
4/6: Overnight Cultures of JM110 E.coli containing pSB1C3-LbpA
Aim of experiment: To set up overnight cultures of JM110 E.coli containing pSB1C3-LbpA.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: A miniprep will be performed tomorrow on the overnight cultures.
5/6: Plasmid Purification of JM110 E.coli containing pSB1C3-LbpA
Aim of experiment: To miniprep the JM110 E.coli overnight cultures containing pSB1C3-LbpA.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: The miniprep with the highest concentration will be sent for sequencing.
Week Beginning 8/6/2015
Summary
Sequencing confirmed that LbpA had been successfully cloned into pSB1C3 therefore this week cloning was attempted to put LbpA into the high expression vector pQE80-L.
8/6: Amplification of LbpA for Cloning into pQE80-L
Aim of experiment: To amplify LbpA for subsequent cloninng into the pQE80-L vector.
Protocols Used: PCR
Results: N/A
Next Steps: The PCR product will be digested and ligated into pQE80-L tomorrrow.
9/6: Restriction Digests and Ligations of LbpAinto pQE80-L
Aim of experiment: To restrict and ligate the LbpA gene into the pQE80-L plasmid.
Protocols Used: Restriction Digests Ligations
Results:: N/A
Next Steps: Transformations of pQE80-L-LbpA into JM110 E.coli will be carried out tomorrow.
10/6: Transformations of pQE80-L-LpbA into JM110 E.coli
Aim of experiment: To transform  pQE80-L-LpbA into JM110 E.coli.
Protocols Used: Transformations
Results: N/A
Next Steps: If the transformations are successful, colonies will be picked to set up overnight cultures.
Week Beginning 15/6/15
Summary
This week, cloning of LbpA into pQE80-L was continued due to last week's failed attempts.
15/6: PCR of LbpA for Cloning into pQE80-L
Aim of experiment: To re-amplify the LbpA gene for cloning into the pQE80-L vector.
Protocols Used: PCR
Results: Figure 3
Next Steps: The gel shows that LbpA has been successfully amplified so it will be digested and ligated into the pQE80-L vector tomorrow.
Week Beginning 22/6/15
Summary
LbpA was successfully cloned into the high expression vector pQE80-L.
23/6: Restriction Digest and Ligation of LbpA into pQE80-L
Aim of experiment: To digest the amplified LbpA gene with BamHI and KpnI and subsequently ligate it into the pQE80-L vector.
Protocols Used: Restriction Digests Ligations
Results: N/A
Next Steps: The ligations will be transformed into M15[pREP4] E.coli.
24/6: Transformations of pQE80-L-LbpA into M15[pREP4] E.coli
Aim of experiment: To transform pQE80-L-LbpA into M15[pREP4] E.coli.
Protocols Used: Transformations
Results: N/A
Next Steps: Overnight cultures will be set up tomorrow if the transformations are successful.
25/6: Overnight Cultures of M15[pREP4] E.coli containing pQE80-L-LbpA
Aim of experiment: To set up overnight cultures of M15[pREP4] E.coli containing pQE80-L-LbpA.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: Overnight cultures will be miniprepped and a pre-sequence restriction digest will be performed to check for the presence of the LbpA gene.
26/6: Plasmid Purification of the M15[pREP4] E.coli Overnight Cultures containing pQE80-L-LbpA
Aim of experiment: To miniprep the overnight cultures from yesterday and perform a pre-sequencing restriction digest using BamHI and KpnI to check for presence of LbpA gene.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: A sample of the miniprep was sent for sequencing to confirm the presence of LbpA in the pQE80-L vector.
Week Beginning 29/6/15
Summary
Protein expression optimization experiments were performed on LbpA to assay protein production levels. However, cell growth seems to cease when LbpA expression is induced.
1/7: M15[pREP4] E.coli Overnight Cultures containing pQE80-L-LbpA
Aim of experiment: To set up overnight cultures in preparation for protein expression experiments tomorrow.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The overnight cultures will be used to set up day cultures tomorrow which will be used to test expression of LbpA.
2/7: Optimization of LbpA Expression
Aim of experiment: To set up day cultures and induce expression of LbpA using different concentrations of IPTG and check levels of protein production via SDS-PAGE.
Protocols Used: Protein Expression Optimization
Results: N/A
Next Steps: The samples were stored in the -20 oC freezer overnight and will be ran on a gel tomorrow.
3/7: Continuation of Optimization of LbpA Expression
Aim of experiment: To test the samples obtained yesterday on an SDS gel.
Protocols Used: Protein Expression Optimization
Results: Figure 4
Next Steps: Further optimization experiments will be set up next week using different concentrations of IPTG.
Week Beginning 6/7/15
Summary
Further protein expression optimization experiments were set up, this time using a range of IPTG concentrations to induce expression of LbpA.
7/7: Further Optimization of LbpA Expression
Aim of experiment: To set up further tests to optimize expression of LbpA using different concentrations of IPTG.
Protocols Used: Protein Expression Optimization Note: An SDS gel was not ran, instead the OD600 readings were recorded for the uninduced and induced cultures at 20 minute intervals for 2 hours, 1 and half hours after the cultures were induced.
Results: Figure 5
Next Steps: A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after LbpA expression has been induced.
Week Beginning 13/7/15
Summary
Protein expression optimization experiments were continued this week. A growth curve assay was carried out to assess the effect of LbpA expression over a longer period of time.
14/7: Plate Reader Growth Curve Assay
Aim of experiment: To induce expression of LbpA using different concentration of IPTG and allow cells to grow for 16 hours in order to monitor cell growth.
Protocols Used: Growth Curve Assay
Results: Figure 6
Next Steps: A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after LbpA expression has been induced.
17/7: Sample Preparation for Western Blot
Aim of experiment: To check if the cells in the induced cultures ~7 hours after being induced are producing LbpA.
Protocols Used: Western Blot Sample Preparation Note: The day cultures were grown for 7 hours after they were induced and then the samples were frozen, to be blotted on Monday.
Results: N/A
Next Steps: The samples will be ran on an SDS gel on Monday and blotted.
Week Beginning 20/7/15
Summary
Samples from the growth curve assay were blotted to see if LbpA is being produced at the later stages of the growth curve. A further growth curve assay was also performed which monitored cell growth until a certain OD600 was reached, at which point the cultures were induced.
20/7: Western Blot LpbA Culture Samples
Aim of experiment: To western blot the LbpA culture samples from Friday.
Protocols Used: Western Blot
Results: Nothing was visible on the blot.
Next Steps: Another plate reader experiment will be set up on Thursday where the cultures will be monitored for three hours before induction with IPTG and then for a further 20 hours afterwards.
23/7: Growth Curve Assay
Aim of experiment: To check if the cells begin to die after induction with IPTG after having already grown to an OD600 of 0.5.
Protocols Used: Growth Curve Assay
Results: Figure 7
Next Steps: Samples of these cultures will be taken 6 hours after induction to check for the presence of LbpA.
Week Beginning 27/7/15
Summary
The presence of LbpA was confirmed in the culture 6 hours after induction with IPTG so protein purification was started this week.
27/7: Overnight Cultures of M15[pREP4] E.coli containing pQE80-L-LbpA
Aim of experiment: To set up a 150 ml overnight culture for sub-culturing tomorrow.
Protocols Used: Overnight Cultures Note: 150 ml of LB was used and 150 µl of each antibiotic.
Results: N/A
Next Steps: 150 ml of fresh LB will be inoculated tomorrow with 7.5 ml of the overnight cultures.
28/7: LbpA Expression Assay
Aim of experiment: To set up day cultures using the overnight cultures from yesterday and test for expression of LbpA 6 hours after induction.
Protocols Used: Protein Expression Test Note: 150 ml of fresh LB was inoculated with 7.5 ml of the overnight cultures. Only an uninduced control and an 1 mM IPTG induced culture was set up. The samples were not blotted, they were only ran on an SDS gel.
Results: Figure 8
Next Steps: A 3 L culture will be set up on Thursday under the same conditions as the overnight cultures in order to try and purify LbpA.
29/7: Overnight Cultures of M15[pREP4] E.coli containing pQE80-L-LbpA for 3L Culture
Aim of experiment: To set up overnight cultures for the 3L day cultures that will be grown tomorrow.
Protocols Used: Overnight Cultures Note: One 150 ml overnight culture was set up using 150 µl of each antibiotic.
Results: N/A
Next Steps: The 3 L culture will be set up tomorrow allowed to grow to OD600 of 0.6-1 and then induced with 1 mM IPTG and grown for 6 hours.
30/7: 3 L Culture of M15[pREP4] E.coli containing pQE80-L-LbpA for Purification of LbpA
Aim of experiment: To set up 3 L day cultures for purifying LbpA.
Protocols Used: Protein Purification Note: The procedure was stopped at Step 6 and the pellets were frozen at -20 oC.
Results: N/A
Next Steps: Protein purification will be continued on Monday.
Week Beginning 3/8/15
Summary
The first round of LbpA purification failed at the affinity chromatography stage so another 6 L culture was grown this week to attempt purification again.
3/8: Continuation of LbpA Purification
Aim of experiment: To continue with purification of LbpA.
Protocols Used: Protein Purification Note: The protocol was continued from Step 7
Results: N/A
Next Steps:Size exclusion chromatography was not carried out since an SDS gel showed LbpA was not present after performing affinity chromatography. Another 6L culture will be set up this week.
5/8: Overnight Cultures of M15[pREP4] E.coli containing pQE80-L-LbpA
Aim of experiment: To set up overnight cultures for the 6 L culture that will be set up tomorrow.
Protocols Used: Overnight Cultures Note: One 150 ml overnight culture was set up using 150 µl of each antibiotic.
Results: N/A
Next Steps:The overnight cultures will be used to set up the 6 L day culture tomorrow.
6/8: 6 L Day Cultures of M15[pREP4] E.coli containing pQE80-L-LbpA
Aim of experiment: To set up 6 L day cultures for purifying LbpA and begin the purification process.
Protocols Used: Protein Purification Note: The protocol was stopped at Step 6.
Results: N/A
Next Steps: Purification of LbpA will be continued on Monday.
Week Beginning 10/8/15
Summary
Purification of LbpA was continued but unfortunately was unsuccessful. If time permits, another round of purification may be reattempted.
10/8: Continuation of LbpA Purification
Aim of experiment: To continue with purification of LbpA.
Protocols Used: Protein Purification Note: The protocol was continued from Step 7.
Results: N/A
Next Steps: Purification of LbpA will be continued tomorrow.
11/8: Continuation of LbpA Purification
Aim of experiment: To purify LbpA using affinity chromatography.
Protocols Used: Protein Purification Note: The protocol was stopped at Step 13.
Results: N/A
Next Steps:Since no peaks were observed, purification of LbpA was abadoned at this stage. Purification may be retried.
Semen
Week Beginning 25/5/2015
Summary
Cloning of PotD into the biobrick vector, pSB1C3, was started this week.
28/5: PCR of PotD from MG1655 E.coli gDNA
Aim of Experiment: To amplify PotD from MG1655 E.coli gDNA.
Protocols Used: PCR
Results: N/A
Next Steps: The PCR product will be gel extracted, digested and ligated into the biobrick vector pSB1C3 tomorrow.
29/5: Gel Extraction, Restriction Digest and Ligation of SBP into pSB1C3
Aim of experiment: To gel extract the PotD PCR product and digest it with PstI and EcoRI. A ligation will then be set up with pSB1C3.
Protocols Used: Restriction Digest Ligation
Results: N/A
Next Steps: Ligations will be transformed into JM110 E.coli on Monday.
Week Beginning 1/6/2015
Summary
Cloning of PotD into the biobrick vector was completed this week.
1/6: Transformation of pSB1C3-PotD into JM110 E.coli
Aim of experiment: To transform pSB1C3-PotD into JM110 E.coli.
Protocols Used: Transformations
Results:N/A
Next Steps: If the transformations are successful, overnight cultures will be set up tomorrow.
2/6: Overnight Cultures of pSB1C3-PotD in JM110 E.coli
Aim of experiment: To set up overnight cultures of pSB1C3-PotD in JM110 E.coli
Protocols Used: Overnight Cultures
Results:: N/A
Next Steps: The overnight cultures will be miniprepped tomorrow.
3/6: Plasmid Purification of pSB1C3-PotD from Overnight Cultures
Aim of experiment: To miniprep the overnight cultures from yesterday.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: The miniprep with the highest concentration will be sent for sequencing to confirm PotD has been inserted into pSB1C3 successfully.
Week Beginning 15/6/15
Summary
This week, cloning of PotD into pQE80-L was started. The gene for Spermine Binding Protein (SBP) also arrived in a plasmid so it was transformed into MC1061 E.coli and plasmid purified.
15/6: PCR of PotD for Cloning into pQE80-L
Aim of experiment: To amplify the PotD gene for cloning into the pQE80-L vector.
Protocols Used: PCR
Results: Figure 3
Next Steps: The gel shows that PotD has been successfully amplified so it will be digested and ligated into the pQE80-L vector tomorrow.
17/6: Restriction Digest and Ligation of PotD into pQE80-L. Transformation of IDT Plasmid containing SBP into MC1061 E.coli
Aim of experiment: To digest the PotD PCR product with KpnI and BamHI and then ligate it into pQE80-L. To transform MC1061 E.coli with the IDT plasmid containing SBP.
Protocols Used:
PotD: Restriction Digests Ligations SBP: Transformations
Results: N/A
Next Steps: The pQE80-L-PotD ligations will be transformed into MC1061 E.coli. Overnight cultures of the transformations will be set up tomorrow.
18/6: Transformations of PotD + pQE80-L into MC1061 E.coli. Overnight Cultures of MC1061 E.coli with SBP + IDT Plasmid
Aim of experiment: To transform the pQE80-L-PotD ligations into MC1061 E.coli. To set up overnight cultures of the transformations of MC1061 E.coli with SBP + IDT Plasmid that were set up yesterday.
Protocols Used: PotD: Transformations SBP: Overnight Cultures
Results:N/A
Next Steps: If transformations are successful, overnight cultures will be set up tomorrow.
19/6: Repeat of Ligation of PotD into pQE80-L. Plasmid Purification of the IDT Plasmid Containing SBP
Aim of experiment: To repeat ligations of PotD into pQE80-L since transformations did not work. To miniprep the overnight cultures that were set up yesterday.
Protocols Used: PotD: Ligations SBP: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: Ligations will be left over the weekend and transformed on Monday.
Week Beginning 22/6/15
Summary
Cloning of PotD into the high expression vector pQE80-L was completed. Cloning of SBP into pSB1C3 and pQE80-L was started this week.
22/6: Transformation of pQE80-L-PotD Ligations. PCR and Gel Extraction of SBP for Cloning into pQE80-L and Restriction Digests and Ligation of SBP for Cloning into pSB1C3
Aim of Experiment: To transform pQE80-L-PotD into  MC1061E.coli. To set up a PCR of SBP using the purified plasmid from Friday and to gel extract the PCR product for cloning into pQE80-L. To digest SBP out of the same plasmid using EcoRI and PstI and subsequently ligate it into pSB1C3.
Protocols Used: PotD: Transformations SBP: PCR Restriction Digests Ligations
Results: SBP : Figure 1
Next Steps: Overnight cultures will be set up tomorrow for the PotD transformations. The SBP PCR product will be digested and ligated into pQE80-L tomorrow. The pSB1C3-SBP ligations will be transformed into MC1061 E.coli.
23/6: Overnight Cultures of MC1061 E.coli containing pSB1C3-PotD. Transformations of MC1061 E.coli with pSB1C3 + SBP. Restriction Digests and Ligations of SBP into pQE80-L and Transformations into MC1061 E.coli
Aim of experiment: To set up overnight cultures of MC1061 E.coli containing pSB1C3 + PotD. To transform MC1061 E.coli with pSB1C3 + SBP. To digest the SBP PCR product with BamHI and Kpn I, ligate it into pQE80-L and transform these ligations into MC1061 E.coli.
Protocols Used: PotD: Overnight Cultures SBP: Transformations Restriction Digests Ligations
Results: N/A
Next Steps: The MC1061 E.coli overnight cultures containing pQE80-L + PotD will be miniiprepped tomorrow. Overnight cultures will be set up of the MC1061 E.coli transformations with pSB1C3 + SBP and pQE80-L + SBP.
24/6: Plasmid Purification of the Overnight Cultures of MC1061 E.coli containing pSB1C3-PotD. Overnight Cultures of the Transformations of MC1061 E.coli with pSB1C3 + SBP and pQE80-L + SBP.
Aim of experiment: To miniprep the overnight cultures of MC1061 E.coli containing pSB1C3-PotD. To set up overnight cultures of the transformations of MC1061 E.coli with pSB1C3-SBP and pQE80-L-SBP
Protocols Used: PotD: Plasmid Purification (QIAprep® Spin Miniprep Kit) SBP: Overnight Cultures
Results: N/A
Next Steps: The miniprep with the highest concentration for pQE80-L + PotD was sent for sequencing. The SBP overnight cultures will be miniprepped tomorrow.
25/6: Plasmid Purification of the Overnight Cultures of MC1061 E.coli containing pSB1C3-SBP and pQE80-L-SBP
Aim of experiment: To miniprep the overnight cultures of MC1061 E.coli containing pSB1C3-SBP and pQE80-L + SBP.
Protocols Used: SBP: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: The miniprep with the highest concentration for both pSB1C3-SBP and pQE80-L-SBP were sent for sequencing.
Week Beginning 29/6/15
Summary
Sequencing showed SBP failed to insert successfully into pSB1C3 so cloning was repeated from the ligation stage this week. However, SBP and PotD were successfully inserted into the high expression vector pQE80-L, so overexpression assays were started this week for both.
29/6: Ligations of SBP into pSB1C3 and Transformations into MC1061 E.coli
Aim of Experiment: To ligate SBP into pSB1C3 and transform the ligations into MC1061 E.coli.
Protocols Used: SBP: Ligations Transformations
Results: SBP : Figure 1
Next Steps: Overnight cultures will be set up tomorrow if transformations are successful.
30/6: Overnight Cultures of MC1061 E.coli containing pSB1C3-SBP, pQE80-L-SBP and pQE80-L-PotD
Aim of experiment: To set up overnight cultures of MC1061 E.coli containing pSB1C3-SBP. To set up overnight cultures  of pQE80-L-SBP and pQE80-L + PotD in MC1061 E.coli for overexpression assays to be performed tomorrow.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The MC1061 E.coli overnight cultures containing pSB1C3- SBP will be miniiprepped tomorrow. The MC1061 E.coli overnight cultures containing pQE80-L- SBP and pQE80-L-PotD will be subcultured tomorrow in order to perform overexpression assays.
1/7: Plasmid Purification of the Overnight Cultures of MC1061 E.coli containing pSB1C3-SBP. PotD and SBP Expression Optimization
Aim of experiment: To miniprep the overnight cultures of MC1061 E.coli containing pSB1C3-SBP. To start protein expression optimization for PotD and SBP.
Protocols Used: SBP: Plasmid Purification (QIAprep® Spin Miniprep Kit) SBP and PotD: Protein Expression Optimization Note: Protocol was stopped at Step 8. Samples will be ran on an SDS gel tomorrow.
Results: N/A
Next Steps: The miniprep with the highest concentration for pSB1C3 + SBP was sent for sequencing. The PotD and SBP samples will be ran on an SDS gel tomorrow.
2/7: Continuation of PotD and SBP Expression Optimization
Aim of experiment: To run the samples of PotD and SBP on an SDS gel and begin Western Blotting.
Protocols Used: PotD and SBP: Protein Expression Optimization Note: Protocol was continued from Step 9.
Results: Figure 2
Next Steps: The membrane for the Western blot has been left at the blocking stage so will be continued tomorrow.
3/7: Western Blot of PotD and SBP Day Culture Samples
Aim of experiment: To continue the Western Blot of PotD and SBP.
Protocols Used: PotD and SBP: Western Blot
Results: Figure 3
Next Steps: PotD has been successfully characterized so purification of this protein will be started next week. Further expression optimization experiments for SBP will be set up next week using different concentrations of IPTG, since its characterization was unsuccessful.
Week Beginning 6/7/15
Summary
Further protein expression optimization experiments were set up, this time using a range of IPTG concentrations to induce expression of SBP.
6/7: Further Optimization of SBP Expression
Aim of experiment: To set up further tests to optimize expression of SBP using different concentrations of IPTG.
Protocols Used: Protein Expression Optimization Note: Cultures were induced with 0.5 mM, 1 mM and 2 mM IPTG. The Western Blot was left to block overnight.
Results: Figure 4
Next Steps: The Western Blot will be completed tomorrow.
7/7: Western Blot of SBP
Aim of experiment: To complete the Western Blot that was started yesterday.
Protocols Used: Western Blot Note: Protocol was continued from Step 12.
Results: Nothing was visible on the blot.
Next Steps: A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after SBP expression has been induced.
Week Beginning 13/7/15
Summary
Protein expression optimization experiments were continued this week. A growth curve assay was carried out to assess the effect of SBP expression over a longer period of time. Purification of PotD was also started this week.
14/7: Plate Reader Growth Curve Assay
Aim of experiment: To induce expression of SBP using different concentration of IPTG and allow cells to grow for 16 hours in order to monitor cell growth.
Protocols Used: Growth Curve Assay
Results: Figure 5
Next Steps: A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after SBP expression has been induced.
15/7: Overnight Cultures for 3 L Culture for PotD Purification
Aim of experiment: To set up overnight cultures that will be used tomorrow to set up 3 L cultures for PotD purification.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The 3 L day culture will be set up tomorrow.
16/7: Sample Preparation for Western Blot of SBP and 3 L Day Culture for PotD Purification
Aim of experiment: To check if the cells in the induced cultures ~7 hours after being induced are producing SBP. To set up 3 L Day Culture for PotD Purification
Protocols Used: SBP: Western Blot Sample Preparation Note: The day cultures were grown for 7 hours after they were induced and then the samples were frozen, to be blotted on Monday. PotD: Protein Purification Note: Cultures were left to grow overnight after being induced.
Results: N/A
Next Steps: The samples will be ran on an SDS gel on Monday and blotted. The PotD cultures will be harvested tomorrow.
17/7: Harvesting PotD Cultures
Aim of experiment: To harvest the day cultures grown yesterday for PotD purification.
Protocols Used: Protein Purification Note: After the cultures were spun down the pellets were frozen.
Results: N/A
Next Steps: Purification of PotD will continue on Monday.
Week Beginning 20/7/15
Summary
Samples from the growth curve assay were blotted to see if SBP is being produced at the later stages of the growth curve. A further growth curve assay was also performed which monitored cell growth until a certain OD600 was reached, at which point the cultures were induced. Purification of PotD was also continued this week.
20/7: Western Blot SBP Culture Samples
Aim of experiment: To western blot the SBP culture samples from Friday. To continue purfying PotD.
Protocols Used: SBP: Western Blot PotD: Protein Purification Note: Affinity chromatography was completed but no peaks were observable.
Results: Nothing was visible on the blot.
Next Steps: Another plate reader experiment will be set up on Thursday where the cultures will be monitored for three hours before induction with IPTG and then for a further 20 hours afterwards. Another 150 ml overnight culture was set up for PotD so 3 L cultures could be set up tomorrow.
21/7: 3 L Day Cultures for PotD Purification
Aim of experiment: To set up 3 L Day Culture for PotD Purification.
Protocols Used: Protein Purification Note: Cultures were left to grow overnight after being induced.
Results: N/A
Next Steps: Purification will be continued tomorrow.
22/7: Continuation of PotD Purification
Aim of experiment: To continue purification of PotD.
Protocols Used: Protein Purification Note: The protocol was continued from Step 4 and stopped at Step 13.
Results: N/A
Next Steps: Purification will be continued tomorrow.
23/7: Continuation of PotD Purification and Growth Curve Assay for SBP
Aim of experiment: To continue purification of PotD and to check if the cells begin to die after inducing expression of SBP with IPTG after having already grown to an OD600 of 0.5.
Protocols Used: PotD: Protein Purification Note: The protocol was continued from Step 13. SBP: Growth Curve Assay
Results: Figure 6
Next Steps: The samples from the size exclusion chromatography performed on PotD will be prepared tomorrow for blotting next week. For SBP further overexpression tests will be set up next week.
24/7: SDS-PAGE and Western Blot of PotD
Aim of experiment: To run SDS-PAGE of the elutions obtained on Friday after SEC.
Protocols Used: SDS-PAGE
Results: Figure 7
Next Steps: The same samples will be Western Blotted on Monday to confirm that the bands oberved are PotD.
Week Beginning 27/7/15
Summary
PotD was successfully purified and characterized this week. SBP was still unobservable in induced 150ml cultures so 3 L cultures containing pQE80-L-SBP were set up.
27/7: Overnight Cultures for SBP Expression Assays and SDS-PAGE and Western Blot of PotD
Aim of experiment: To set up a 150 ml overnight culture containing pQE80-L-SBP for sub-culturing tomorrow and to run an SDS-PAGE and Western Blot the PotD samples from Friday.
Protocols Used: SBP: Overnight Cultures Note: 150 ml of LB was used and 150 µl of each antibiotic. PotD: SDS-PAGE and Western Blotting Note: The membrane was left blocking overnight.
Results: N/A
Next Steps: 150 ml of fresh LB will be inoculated tomorrow with 7.5 ml of the overnight cultures containing pQE80-L-SBP. The PotD Western Blot will be completed tomorrow.
28/7: SBP Expression Assay and Completion of PotD Westen Blot
Aim of experiment: To set up day cultures using the overnight cultures from yesterday and test for expression of LbpA 6 hours after induction.
Protocols Used: Protein Expression Test Note: 150 ml of fresh LB was innoculated with 7.5ml of the overnight cultures. Only an uninduced control and an 1 mM IPTG induced culture was set up. The samples were not blotted, they were only ran on an SDS gel.
Results: SBP: Nothing was observable on the SDS gel. PotD: Figure 8
Next Steps: Since PotD has now been successful purified, the next step is to attach a fluorescent nanobead to the protein. Hopefully, this will be attempted in the coming weeks. Even though SBP was not visible on the SDS gel, 3 L cultures will be set up this week to try and harvest the protein since the culture size may be too small.
29/7: Overnight Cultures for 3L Culture
Aim of experiment: To set up overnight cultures containing pQE80-L-SBP for the 3L day cultures that will be grown tomorrow.
Protocols Used: Overnight Cultures Note: One 150ml overnight culture was set up using 150µl of each antibiotic.
Results: N/A
Next Steps: The 3L culture will be set up tomorrow allowed to grow to OD600 of 0.6-1 and then SBP expression induced with 1mM IPTG and grown for 6 hours.
30/7: 3 L Culture for Purification of SBP
Aim of experiment: To set up 3 L day cultures for purifying SBP.
Protocols Used: Protein Purification Note: The procedure was stopped at Step 6 and the pellets were frozen at -20oC.
Results: N/A
Next Steps: Protein purification will be continued on Monday.
Week Beginning 3/8/15
Summary
It seems that SBP has been successfully purified and characterized. However, there was still a lot of contamination in the samples after size eclusion chromatography was performed. .
3/8: Continuation of SBP Purification
Aim of experiment: To continue with purification of SBP.
Protocols Used: Protein Purification Note: The protocol was continued from Step 7
Results: N/A
Next Steps: Size exclusion chromatography will be carried out tomorrow.
4/8: Continuation of SBP Purification
Aim of experiment: To continue purification of SBP.
Protocols Used: Protein Purification Note: The protocol was continued from Step 14.
Results: N/A
Next Steps: The fractions corresponding to the peak observed during SEC, will be ran on an SDS gel and Western Blotted tomorrow.
6/8: Western Blot of SBP
Aim of experiment: To perform a Western Blot on the samples obtained from Size Exclusion Chromatography yesterday..
Protocols Used: Western Blot
Results: Figure 9
Next Steps: There was still some contamination in the gel filtrated samples so this will have to be removed before a nanobead can be attached to the protein.
Week Beginning 10/8/15
Summary
Interactions between PotD and spermidine were analysed via tryptophan fluorescence this week.
10/8: Tryptophan Fluorescence
Aim of experiment: To analyse the interaction of PotD and spermidine using tryptophan fluorescence.
Protocols Used: Tryptophan Fluorescence
Results: Figure 10
Next Steps: To continue characterization of the spermidine/ PotD interaction using tryptophan fluorescence and other methods.
Blood
Week Beginning 10/6/2015
Summary
Cloning of hHBA and hHBB into the biobrick vector, pSB1C3, was performed this week.
10/6: Restriction Digests of hHBA and hHBB from IDT gBlocks and Ligation into pSB1C3
Aim of Experiment: To digest hHBA and hHBB from IDT gBlocks using EcoRI and PstI for ligation into pSB1C3.
Protocols Used: Restriction Digests Ligations
Results: N/A
Next Steps: The pSB1C3-hHBA and pSB1C3-hHBB ligations will be transformed into JM110 E.coli.
11/6: Transformations of pSB1C3-hHBA and pSB1C3-hHBB Ligations into JM110 E.coli
Aim of experiment: To transform the pSB1C3-hHBA and pSB1C3-hHBB ligations into JM1110 E.coli.
Protocols Used: Transformations
Results: N/A
Next Steps: If transformations are successful, overnight cultures will be set up.
Week Beginning 15/6/2015
Summary
The sequence for pSB1C3-hHBB was confirmed this week so cloning of hHBB into the two hybrid system vector pUT18 was started. Sequencing of pSB1C3-hHBA was incorrect so another miniprep was sent for sequencing.
15/6: Overnight Cultures of JM110 E.coli containing pSB1C3-hHBA and pSB1C3-hHBB
Aim of experiment: To set up overnight cultures of JM110 E.coli containing pSB1C3-hHBA and pSB1C3-hHBB from the transformations performed on Tuesday.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: Overnight cultures will be miniprepped tomorrow and a test restriction digest will be performed to check for the presence of the inserts.
16/6: Plasmid Purification of the JM110 E.coli Overnight Cultures containing pSB1C3-hHBA and pSB1C3-hHBB
Aim of experiment: To miniprep the JM110 E.coli overnight cultures containing pSB1C3-hHBA and pSB1C3-hHBB and perform a pre-sequence digest check.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit) Restriction Digests
Results: Figure 1
Next Steps: The gel indicates that hHBA and hHBB have inserted successfully into pSB1C3 so samples of the minipreps will be sent for sequencing to confirm this.
17/6: Amplification of hHBA and hHBB for Cloning into the Two Hybrid System Vectors
Aim of experiment: To amplify hHBA and hHBB for Cloning into the Two Hybrid System Vectors pT25 and pUT18, respectively.
Protocols Used: PCR
Results: Figure 2
Next Steps: Sequencing of pSB1C3-hHBA and pSB1C3-hHBB showed that pSB1c3-hHBA had a 1 bp deletion so one of the other minipreps was sent for sequencing. pSB1C3-hHBB, however, was successful so the band for hHBB will be gel extracted and digested tomorrow.
18/6: Gel Extraction and Restriction Digests hHBB for Cloning into pUT18
Aim of experiment: To gel extract and digest hHBB with BamHI and EcoRI.
Protocols Used: Restriction Digests
Results: N/A
Next Steps: Ligations of hHBB will be set up tomorrow.
19/6: Ligations of hHBB into pUT18
Aim of experiment: To set up ligations of pUT18-hHBB.
Protocols Used: Ligations
Results: N/A
Next Steps: Ligations will be left over the weekend and transformed into BTH101 E.coli.
Week Beginning 22/6/15
Summary
Cloning of hHBA and hHBB into pT25 and pUT18, respectively, was continued this week.
22/6: Transformations of pUT18-hHBB into BTH101 E.coli
Aim of experiment: To transform pUT18-hHBB into BTH101 E.coli.
Protocols Used: Transformations
Results:N/A
Next Steps: If transformations are successful, overnight cultures will be set up tomorrow. Sequencing showed the sequence for pSB1C3-hHBA was incorrect again, a third miniprep will be sent for sequencing.
23/6: Overnight Cultures of BTH101 E.coli containing pUT18-hHBB
Aim of experiment: To set up overnight cultures of BTH101 E.coli containing pUT18-hHBB from the transformations carried out yesterday.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The overnight cultures will be miniprepped and sent for sequencing tomorrow.
24/6: Plasmid Purification of the BTH101 E.coli Overnight Cultures containing pUT18-hHBB
Aim of experiment: To miniprep the BTH101 E.coli Overnight Cultures containing pUT18-hHBB.
Protocols Used:
Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: The miniprep with the highest concentration will be sent for sequencing to confirm presence of insert. Sequencing for pSB1C3-hHBA came back correct so another PCR of hHBA will be set up tomorrow for cloning into pT25.
25/6: Amplification of hHBA for Cloning into pT25
Aim of experiment: To amplify hHBA for cloning into pT25
Protocols Used: PCR
Results: Figure 3
Next Steps: The PCR product digested and ligated into pT25 tomorrow.
26/6: Restriction Digests and Ligations of hHBA for Cloning into pT25
Aim of experiment: To digest the hHBA PCR product with BamHI and KpnI and set up ligations with pT25.
Protocols Used: Restriction Digests Ligations
Results: N/A
Next Steps: Ligations will be left over the weekend and transformed into BTH101 E.coli on Monday.
Week Beginning 29/6/15
Summary
Cloning of hHBA into pT25 was completed this week. Sequencing confirmed that hHBB has been successfully inserted into pUT18. Cloning of both hHBA and hHBB, individually, into the high expression vector pQE80-L was started this week.
29/6: Transformation of pT25-hHBA Ligations into BTH101 E.coli
Aim of Experiment: To transform pT25-hHBA into  BTH101 E.coli.
Protocols Used: Transformations
Results: N/A
Next Steps: Overnight cultures will be set up tomorrow for the pT25-hHBA transformations if successful. Sequencing confirmed hHBB has been successfully inserted into pUT18.
30/6: Overnight Cultures of MC1061 E.coli containing pUT18-hHBA and Amplification of hHBA and hHBB for Cloning into pQE80-L
Aim of experiment: To set up overnight cultures of MC1061 E.coli containing pT25-hHBA. To amplify hHBA.and hHBB for cloning into pQE80-L.
Protocols Used: Overnight Cultures Transformations Restriction Digests Ligations
Results: Figure 4
Next Steps: The BTH101 E.coli overnight cultures containing pT25-hHBB will be miniiprepped tomorrow. The PCR products will be digested and ligate dinto pQE80-L tomorrow.
1/7: Plasmid Purification of the Overnight Cultures of BTH101 E.coli containing pT25-hHBA and Restriction Digests and Ligations of hHBA and hHBB into pQE80-L
Aim of experiment: To miniprep the overnight cultures of BTH101 E.coli containing pT25-hHBA. To digest hHBA and hHBB with BamHI and KpnI and set up ligations with pQE80-L.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit) Restriction Digests Ligations
Results: N/A
Next Steps: The miniprep with the highest concentration for pT25-hHBA will be sent for sequencing. The pQE80-L-hHBA and pQE80-L-hHBB will be transformed into M15[pREP4] E.coli tomrorrow.
2/7: Transformations of pQE80-L-hHBA and pQE80-L-hHBB Ligations into M15[pREP4] E.coli
Aim of experiment: To transform the pQE80-L-hHBA and pQE80-L-hHBB ligations into M15[pREP4] E.coli.
Protocols Used: Transformations
Results: N/A
Next Steps: If transformations are successful, overnight cultures will be set up on Monday.
Week Beginning 6/7/15
Summary
Cloning of hHBA and hHBB into pQE80-L was completed this week. Cloning of Haptoglobin (hHBN) was also started this week.
6/7: Overnight Cultures of M15[pREP4] E.coli containing pQE80-L-hHBA and pQE80-L-hHBB
Aim of Experiment: To set up overnight cultures of BTH101 E.coli containing pQE80-L-hHBA and pQE80-L-hHBB.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps:The overnight cultures will be miniprepped tomorrow. Sequencing for pT25-hHBA was incorrect so cloning will be repeated from the restriction digest stage.
7/7: Plasmid Purification of M15[pREP4] E.coli Overnight Cultures containing pQE80-L-hHBA and pQE80-L-hHBB
Aim of experiment: To set up miniprep the BTH101 E.coli Overnight Cultures containing pQE80-L-hHBA and pQE80-L-hHBB.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: Figure 5
Next Steps: The pre-sequence digest indicates that both hHBA and hHBB have been inserted into pQE80-L successfully. The minipreps with the highest concentration will be sent for sequencing. The sequence for Haptoglobin (hHBN) arrived in an IDT plasmid, work will be be started on this tomorrow.
8/7: Transformation of IDT Plasmid-hHBN into MC1061 E.coli
Aim of experiment: To transform the IDT Plasmid-hHBN into MC1061 E.coli.
Protocols Used: Transformations
Results: N/A
Next Steps: Overnight cultures will be set up tomorrow.
9/7: Overnight Cultures of MC1061 E.coli containing IDT Plasmid-hHBN
Aim of experiment: To set up overnight cultures of MC1061 E.coli containing IDT Plasmid-hHBN.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The overnight cultures will be miniprepped tomorrow and hHBN will be amplified for cloning into pSB1C3.
10/7: Plasmid Purification of IDT Plasmid-hHBN and Digestion of hHBN for Cloning into pSB1C3
Aim of experiment: To miniprep the MC1061 E.coli containing IDT Plasmid-hHBN and to digest hHBN out of the plasmid using EcoRI and PstI.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit) Restriction Digests
Results: N/A
Next Steps: hHBN will be ligated into pSB1C3 on Monday. Sequencing also confirmed hHBA and hHBB have been successfully inserted into pQE80-L.
Week Beginning 13/7/15
Summary
hHBB was successfully characterized this week so protein purification was started. Cloning of hHBN into pSB1C3 was also completed.
13/7: Ligations of hHBN into pSB1C3 and Overnight Cultures of M15[pREP4] E.coli containing pQE80-L-hHBA and pQE80-L-hHBB
Aim of experiment: To ligate hHBN into pSB1C3.
Protocols Used: Ligations Overnight Cultures
Results:N/A
Next Steps: The pSB1C3-hHBN ligations will be transformed into MC1061 E.coli tomorrow. The overnight cultures will be subcultured tomorrow and induced with IPTG to test protein expression levels.
14/7: Transformation of pSB1C3-hHBN into MC1061 E.coli and Optmiization of hHBA and hHBB Expression
Aim of experiment: To transform pSB1C3-hHBN into MC1061 E.coli and set up expression optimization tests for hHBA and hHBB.
Protocols Used: Transformations Protein Expression Optimization
Results: Figure 6
Next Steps: If transformations are successful, overnight cultures will be set up tomorrow. Western Blotting will be continued tomorrow for hHBA and hHBB.
15/7: Overnight Cultures of MC1061 E.coli containing pSB1C3-hHBN and Western Blotting of hHBA and hHBB
Aim of experiment: To set up overnight culture of MC1061 E.coli containing pSB1C3-hHBN and finish Western Blotting of hHBA and hHBB.
Protocols Used: Overnight Cultures Western Blotting
Results: Figure 7
Next Steps: The blot shows that characterization of hHBB has been successful. Unfortunately, nothing was visible on the blot for hHBA. 3 L cultures for purification of hHBB will be set up this week. Characterization of hHBA may be reattempted. The overnight cultures will be miniprepped tomorrow.
16/7: Plasmid Purification of pSB1C3-hHBN and Overnight Cultures of M15[pREP4] E.coli containing pQE80-L-hHBB
Aim of experiment: To miniprep the MC1061 E.coli overnight cultures containing pSB1C3-hHBN and set up overnight cultures of M15[pREP4] E.coli containing pQE80-L-hHBB for 3L cultures.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit) Overnight Cultures
Results: N/A
Next Steps: The miniprep with the highest concentration will be sent for sequencing. The overnight cultures will be used to set up 3L day cultures tomorrow for purrification of hHBB.
17/7: 3 L Cultures of M15[pREP4] E.coli containing pQE80-L-hHBB
Aim of experiment: To set up 3 L day cultures of M15[pREP4] E.coli containing pQE80-L-hHBB for purification of hHBB.
Protocols Used: Protein Purification Note: The protocol was stopped at Step 6.
Results: N/A
Next Steps: Purification will be continued on Monday.
Week Beginning 20/7/15
Summary
hHBB was successfully purified this week and cloning of hHBN into pQE80-L was completed.
20/7: Amplification of hHBN for Cloning into pQE80-L and Purification of hHBB
Aim of experiment: To amplify hHBN for cloning into pQE80-L and continuation of hHBB purification.
Protocols Used: PCR Protein Purification Note: Protocol was continued from Step 6.
Results: Figure 8
Next Steps: The fractions from the affinity chromatography will be consolidated and concentrated tomorrow. Size exclusion chromatography will also be performed. The hHBN PCR product will be digested and ligated into pQE80-L tomorrow.
21/7: Size Exclusion Chromatography of hHBB and Restriction Digests and Ligations of hHBN into pQE80-L
Aim of experiment: To consolidate, concentrate and perform size exclusion chromatography on the hHBB samples. To digest the hHBN PCR product with BamHI and KpnI and subsequently ligate it into pQE80-L.
Protocols Used: Protein Purification SDS Note: An SDS gel was ran of the SEC fractions which corresponded to the observed peak. Western blotting was stopped at the blocking stage. Restriction Digests Ligations
Results: Figure 9
Next Steps: Western blotting will be completed tomorrow and will confirm if the bands observed on the SDS gel are indeed hHBB. The pQE80-LhHBN ligations will be transformed into M15[pREP4] E.coli tomorrrow.
22/7: Western Blot of hHBB and Transformations of M15[pREP4] E.coli with pQE80-LhHBN Ligations
Aim of experiment: To complete the Western blot of hHBB and transform pQE80-LhHBN into M15[pREP4] E.coli.
Protocols Used: Western Blotting Transformations
Results: Figure 10
Next Steps: The blot shows hHBB has been successfully purified. If the transformations are successful, overnight cultures will be set up tomorrow.
23/7: Overnight Cultures of M15[pREP4] E.coli containing pQE80-LhHBN
Aim of experiment: To set up overnight cultures of M15[pREP4] E.coli containing pQE80-LhHBN.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The overnight cultures will be miniprepped tomorrow.
24/7: Plasmid Purification of Overnight Cultures of M15[pREP4] E.coli containing pQE80-LhHBN
Aim of experiment: To miniprep M15[pREP4] E.coli overnight cultures containing pQE80-LhHBN.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: The miniprep with the highest concentration will be sent for sequencing.
Week Beginning 27/7/15
Summary
hHBN was successfully cloned into pQE80-L and cloning started for insertion into pT25.
27/7: Restriction Digests and Ligations of hHBN into pQE80-L
Aim of experiment: To digest hHBN again with BamHI and KpnI and set up ligations with pQE80-L.
Protocols Used: Resriction Digests Ligations
Results: N/A
Next Steps: Ligations will be transformed into M15[pREP4] E.coli tomorrow.
28/7: Transformation of M15[pREP4] E.coli with pQE80-L-hHBN
Aim of experiment: To transform M15[pREP4] E.coli with pQE80-L-hHBN.
Protocols Used: Transformations
Results: N/A
Next Steps: If transformations are successful, overnight cultures will be set up tomorrow.
29/7: Overnight Cultures of M15[pREP4] E.coli containing pQE80-L-hHBN
Aim of experiment: To set up overnight cultures of M15[pREP4] E.coli containing pQE80-L-hHBN
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The overnight cultures will be miniprepped tomorrow.
30/7: Plasmid Purification of M15[pREP4] E.coli Overnight Cultures containing pQE80-L-hHBN
Aim of experiment: To miniprep the M15[pREP4] E.coli Overnight Cultures containing pQE80-L-hHBN.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps:The miniprep with the highest concentration will be sent for sequencing.
31/7: Amplification of hHBN for Cloning into pT25
Aim of experiment: To amplfy hHBN for cloning into the two hybrid vector pT25.
Protocols Used: PCR
Results: Figure 11
Next Steps:The PCR product will be digested and ligated into pT25 on Monday.
Week Beginning 3/8/15
Summary
Cloning of hHBN into pT25 was completed this week but subsequent sequencing showed that it had not inserted successfully.
3/8: Restriction Digests and Ligations of hHBN for Cloning into pT25
Aim of experiment: To digest the hHBN PCR product with BamHI and KpnI and set up ligations with the pT25 vector.
Protocols Used: Restriction Digests Ligations
Results: N/A
Next Steps:The pT25-hHBN ligations will be transformed into MG1655 E.coli tomorrow.
4/8: Transformations of  pT25-hHBN into MG1655 E.coli
Aim of experiment: To set up transformations of pT25-hHBN into MG1655 E.coli.
Protocols Used: Transformations
Results: N/A
Next Steps: If transformations are successful, overnight cultures will be set up tomorrow.
5/8: Overnight Cultures of MG1655 E.coli containing pT25-hHBN
Aim of experiment: To set up overnight cultures of MG1655 E.coli containing pT25-hHBN.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The overnight cultures will be miniprepped tomorrow.
6/8: Plasmid Purification of MG1655 E.coli Overnight Cultures containing pT25-hHBN
Aim of experiment: To miniprep the  MG1655 E.coli overnight cultures containing pT25-hHBN.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: The miniprep with the highest concentration will be sent for sequencing.
Week Beginning 10/8/15
Summary
Overexpression assays of hHBN were set up this week and purification of the protein was begun.
10/8: Overnight Cultures for hHBN Overexpression Assays
Aim of experiment: To set up overnight cultures of M15[pREP4] E.coli containing pQE80-L-hHBN.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps:The overnight cultures will be used to set up day cultures that will be induced with different concentrations of IPTG to assay hHBN expression levels.
11/8: Overexpression Assay of pQE80-L-hHBN
Aim of experiment: To assay hHBN expression levels when induced with a range of IPTG concentrations.
Protocols Used: Overexpression Assay Note: Day cultures were grown to an OD600 of 0.6 and then induced with 0.25 mM, 0.5 mM, 0.75 mM and 1 mM IPTG and allowed to grow for 6 hours.
Results: SDS gel
Next Steps: The gel indicates that hHBN is not being overexpressed in any of the cultures. Further optimization experiments will be set up to try and overexpress hHBN.
12/8: Overnight Cultures of M15[pREP4] E.coli containing pQE80-L-hHBN
Aim of experiment: To set up overnight cultures of M15[pREP4] E.coli containing pQE80-L-hHBN which will be used to set up a 3 L day culture tomorrow.
Protocols Used: Overnight Cultures Note: A 150 ml overnight culture was set up.
Results: N/A
Next Steps: The overnight cultures will be used to set up 3 L day cultures tomorrow.
13/8: 3 L Day Cultures of M15[pREP4] E.coli containing pQE80-L-hHBN
Aim of experiment: To set up 3 L day cultures of M15[pREP4] E.coli containing pQE80-L-hHBN using the overnight culture set up yesterday.
Protocols Used: Protein Purification Note: The cultures were grown to OD600=0.9, then induced with 1 mM IPTG and allowed to grow for a further 8 hours. A 1 ml sample was taken before the cultures were spun down.
Results: N/A
Next Steps: Protein purification will be continued next week.
Week Beginning 17/8/15
Summary
Purification of haptoglobin was completed this week and confirmed via Western Blotting.
17/8: hHBN Purification
Aim of experiment: To continue purification of hHBN by performing affinity chromatography.
Protocols Used: Protein Purification Note: The protocol was stopped at Step 13.
Results: N/A
Next Steps: The samples corresponding to the peak observed in the chromatogram will be combined and purified using SEC tomorrow.
18/8: hHBN Purification
Aim of experiment: To continue purification of hHBN by performing size exclusion chromatography.
Protocols Used: Protein Purification
Results: N/A
Next Steps: The gel shows bands corresponding to 45 kDa, the expected size of hHBN. This will be checked via Western Blotting to ensure it is the correct protein.
19/8: Western Blotting of Purified hHBN
Aim of experiment: To perform Western Blotting in order to verify the protein that was purified is indeed hHBN.
Protocols Used: Western Blotting Note: The membrane was left blocking in milk overnight.
Results: N/A
Next Steps: The blot will be completed tomorrow.
20/8: Western Blotting of Purified hHBN
Aim of experiment: To complete Western Blotting of hHBN.
Protocols Used: Western Blotting The protocol was continued from Step 12.
Results: Figure 14
Next Steps: The blot indicates that hHBN has been successfully purified.
Week Beginning 24/8/2015
Summary
Cloning of hHBA into pT25 was carried out this week.
24/8: Amplification of hHBA for Cloning into pT25
Aim of experiment: To amplify hHBA for subsequent cloning into the two hybrid vector pT25.
Protocols Used: PCR
Results: Figure 15
Next Steps: The gel extracted PCR product will be digested and ligated into pT25 tomorrow.
25/8: Restriction Digests of hHBA and Ligations into pT25
Aim of experiment: To digest the hHBA with BamHI and KpnI and set up ligations with pT25.
Protocols Used: Restriction Digests Ligations
Results: N/A
Next Steps:The ligations will be left overnight and transformed into MG1065∆cya E.coli tomorrow.
26/8: Transformations of pT25-hHBA into MG1065∆cya E.coli
Aim of experiment: To transform the pT25-hHBA ligations into MG1065∆cya E.coli.
Protocols Used: Transformations
Results: N/A
Next Steps: If transformations are successful, colonies will be pick for setting up overnight cultures tomorrow.
27/8: Overnight Cultures of MG1065∆cya E.coli containing pT25-hHBA
Aim of experiment: To set up overnight cultures of MG1065∆cya E.coli containing pT25-hHBA.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The overnight cultures will be miniprepped tomorrow.
28/8: Plasmid Purification of MG1065∆cya E.coli containing pT25-hHBA Overnight Cultures
Aim of experiment: To miniprep the  MG1065∆cya E.coli overnight cultures containing pT25-hHBA.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: The miniprep with highest concentration was sent for sequencing to confirm if the insert has been cloned in successfully.
Week Beginning 7/9/2015
Summary
Cloning of hHBN into pUT18 was carried out this week.
7/9: Amplification of hHBN for Cloning into pUT18
Aim of experiment: To amplify hHBN for subsequent cloning into the two hybrid vector pUT18.
Protocols Used: PCR
Results: Figure 16
Next Steps: The gel extracted PCR product will be digested and ligated into pT25 tomorrow.
8/9: Restriction Digests of hHBN and Ligations into pUT18
Aim of experiment: To digest the hHBN with BamHI and KpnI and set up ligations with pUT18.
Protocols Used: Restriction Digests Ligations
Results: N/A
Next Steps:The ligations will be left overnight and transformed into MG1065∆cya E.coli tomorrow.
9/9: Transformations of pUT18-hHBN into MG1065∆cya E.coli
Aim of experiment: To transform the pUT18-hHBN ligations into MG1065∆cya E.coli.
Protocols Used: Transformations
Results: N/A
Next Steps: If transformations are successful, colonies will be picked for setting up overnight cultures tomorrow.
10/9: Overnight Cultures of MG1065∆cya E.coli containing pUT18-hHBN
Aim of experiment: To set up overnight cultures of MG1065∆cya E.coli containing pUT18-hHBN.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The overnight cultures will be miniprepped tomorrow.
11/9: Plasmid Purification of MG1065∆cya E.coli containing pUT18-hHBN Overnight Cultures
Aim of experiment: To miniprep the  MG1065∆cya E.coli overnight cultures containing pUT18-hHBN.
Protocols Used: Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: The miniprep with highest concentration was sent for sequencing to confirm if the insert has been cloned in successfully.
Nasal Mucus
Week Beginning 15/6/2015
Summary
Produced a purified plasmid of pIDT-OBP2A in order to use it as template in PCR.
17/6: Transformation of pIDT-OBP2A in MC1610 E.coli
Aim of Experiment: To transform pIDT-OBP2A into MC1061 e.coli strain.
Protocols Used: Transformation
Results: N/A
Next Steps: The result of this transformation can be seen tomorrow. If successful then the next step will be to produce an overnight culture in preparation of subsequent plasmid purification.
18/6: Overnight culture of pIDT-OBP2A plasmid in MC1061
Aim of experiment: To produce an overnight culture of pIDT-OBP2A from positive colonies of MC1061 from the transformation done on the 17/6.
Protocols Used: Overnight Culture
Results: N/A
Next Steps: If the overnight culture successfully grows then the next stage of experimentation will be to produce a plasmid purification of pIDT-OBP2A from that overnight culture.
19/6: Plasmid Purification of pIDT-OBP2A from the overnight culture
Aim of experiment: The overnight culture produced on the 18/6 will now undergo a plasmid purification in order to obtain  the pIDT-OBP2A plasmid.
Protocols Used: Plasmid Purification
Results: The plasmid purification yielded a concentration of 305.53 ng/ul.
Next Steps: Assuming the plasmid purification has been successful, then the next stage of experimentation will be to run a PCR using this purified pIDT-OBP2A plasmid as the template for subsequent amplification.
Week Beginning 22/6/2015
Summary
Removal of the signaling peptide from OBP2A and insertion of OBP2A into the biobrick vector pSB1C3.
22/6: PCR of pIDT-OBP2A using primers designed for the removal of the signal peptide
Aim of Experiment: To perform a PCR of the purified pIDT-OBP2A plasmid using a set of primers that have been designed to remove the signaling peptide at the start of the OBP2A gene, whilst leaving the rest of the gene intact.
Protocols Used: PCR
Results: N/A
Next Steps: If the PCR is successful, then the next step will be to gel extract OBP2A from a  the  gel corresponding to the size of the OBP2A gene fragment.
23/6: Gel extraction of OBP2A from the PCR performed on the 22/6
Aim of experiment: To perform a gel extraction of OBP2A from a PCR reaction that was performed on the 22/6. Once gel extraction has been performed a subsequent gel will be done to determine if the gel extraction had been successful.
Protocols Used: Gel Extraction
Results: Figure 1 Figure 2
Next Steps: To perform a restriction of OBP2A in preparation for ligation.
24/6:  Restriction, ligation and transformation of OBP2A into pSB1C3
Aim of experiment: To perform a restriction, ligation and subsequently a transformation of  of the gel extracted OBP2A into the JM110 E.coli strain.
Protocols Used: Restriction Digest : Ligation : Transformation
Results: N/A
Next Steps: If the transformation is successful and positive colonies form then these colonies will then be grown overnight in preparation for plasmid purification.
25/6: Re-ligation and transformation of OBP2A into pSB1C3
Aim of experiment: The transformation performed on the 24/6 failed and as a result OBP2A will now be re-ligated into the biobrick vector pSB1C3 and subsequently transformed into the JM110 E.coli strain.
Protocols Used: Ligation: Transformation
Results: Figure 3
Next Steps:If the transformation is successful and positive colonies form then these colonies will then be grown overnight in preparation for plasmid purification.
26/6: Overnight cultures of pSB1C3-OBP2A
Aim of experiment: To produce overnight cultures of pSB1C3-OBP2A from a positive colony from the transformation done on the 25/6.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: Produce plasmid purifications from the overnight cultures of pSB1C3-OBP2A in preparation for presequence digest.
27/6: Plasmid Purification and Presequence Digest of pSB1C3-OBP2A
Aim of experiment: To perform a plasmid purification of the overnight cultures of pSB1C3-OBP2A from the 26/6. Then to subsequently perform a presequence digest of those plasmid purification.
Protocols Used: Plasmid Purification Restriction Digests
Results: Figure 4
Next Steps: It was concluded from the results of these experiments that sample 5 of pSB1C3-OBP2A will be sent for sequencing.
Week Beginning 29/6/2015
Summary
Positive Sequencing of pSB1C3-OBP2A and subsequent plasmid purification of pSB1C3-OBP2A.
29/6: Plasmid purified pSB1C3-OBP2A is sent for sequencing
Aim of Experiment: Sample 5 of the plasmid purified pSB1C3-OBP2A that was done on the 24/6 is sent for sequencing to determine if OBP2A has successfully ligated into pSB1C3.
Protocols Used: N/A
Results: N/A
Next Steps: If sequencing comes back positive, then the next step will be to clone OBP2A into the two vectors of the bacterial two hybrid system.
30/6: Overnight culture of pSB1C3-OBP2A from  the sequenced colony
Aim of Experiment: Sequencing came back positive for pSB1C3-OBP2A. So an overnight culture of pSB1C3-OBP2A will be done in preparation for plasmid purification.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: Once the overnight culture has been given ample time to grow, the culture will then undergo a plasmid purification.
1/7: Plasmid Purification of pSB1C3-OBP2A
Aim of Experiment: To produce a plasmid purification of the sequenced pSB1C3-OBP2A from the overnight culture set up on the 30/6.
Protocols Used: Plasmid Purification
Results: The plasmid purified pSB1C3-OBP2A produced a concentration of 448.83 ng/ul.
Next Steps: The next stage of experimentation is to use the plasmid purification of pSB1C3-OBP2A as the template for PCR using primers that are designed to separate the gene for use in the two hybrid system.
Week Beginning 12/7/2015
Summary
Ran PCR of OBP2A using primers for insertion of OBP2A into pUT18 and pT25.
17/7: PCR of OBP2A for insertion into bacterial two hybrid vectors: pUT18 and pT25
Aim of Experiment: To perform a PCR of OBP2A for its insertion into the bacterial two hybrid vectors; pUT18 and pT25.
Protocols Used: PCR
Results: N/A
Next Steps:The next stage of experimentation will be to gel extract and then subsequently restrict the amplified gene fragments of OBP2A.
Week Beginning 20/7/2015
Summary
Attempted insertion of OBP2A into the vectors for the bacterial two hybrid system; pUT18 and pT25.
20/7: Gel Extraction and Restriction Digest of OBP2A from the PCR performed on 17/7
Aim of Experiment: To perform a gel extraction and then subsequently a restriction of OBP2A from the PCR produced on the 17/7 using primers designed for the insertion of OBP2A into the vectors for the bacterial two hybrid system, pUT18 and pT25.
Protocols Used: Restriction Digest
Results: N/A
Next Steps: If the gel extraction and subsequent restriction is successful, then the next step will be to ligate the restricted OBP2A into the bacterial two hybrid vector, pUT18 and pT25.
21/7: Ligation and Transformation of OBP2A into pUT18 and pT25.
Aim of Experiment: To perform a ligation and  then subsequently a transformation  of OBP2A into the vectors for the bacterial two hybrid system, pUT18 and pT25. Whilst using JM110 E.coli cells as a chassis for transformation.
Protocols Used: Ligation Transformation
Results:  N/A
Next Steps: The results from the transformation will be seen tomorrow. If successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight colonies in preparation for purification and sequencing.
22/7: Re-ligation and Transformation of OBP2A into pUT18 and PT25.
Aim of Experiment: To perform a re-ligation and transformation of OBP2A into pUT18 and pT25 - since the transformation performed on the 21/7 failed.
Protocols Used: Ligation Transformation
Results: N/A
Next Steps: If transformation is successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight colonies in preparation for purification and sequencing.
23/7: Production of overnight cultures of pUT18-OBP2A.
Aim of Experiment: To produce overnight cultures of pUT18-OBP2A. Unfortunately no positive colonies were produced from the transformation of OBP2A into pT25.
Protocols Used: Overnight Culture
Results: N/A
Next Steps: Once the cells have been given ample time to grow, they will undergo plasmid purification for subsequent sequencing.
24/7: Plasmid Purification and subsequent Presequence digest of pUT18-OBP2A
Aim of Experiment: To perform a plasmid purification of  the overnight cultures of pUT18-OBP2A and then a subsequent presequence digest.
Protocols Used: Plasmid Purification Restriction Digest
Results: Figure 5
Next Steps: If sequencing comes back positive for pUT18-OBP2A then this can now be implemented into the bacterial two hybrid system. Another attempt at cloning OBP2A into pT25 will be done next week.
Week Beginning 27/7/2015
Summary
Successful insertion of the two subunits of OBP2A into the vectors pUT18 and pT25.
27/7: PCR  and Gel extraction of OBP2A for Preparation of Insertion into  pT25.
Aim of Experiment: To perform a PCR and subsequent gel extraction of OBP2A for its insertion into pT25.
Protocols Used: PCR
Results: Figure 6
Next Steps: The next stage of experimentation will be to restrict OBP2A for subsequent ligation into pT25.
28/7: Restriction Digest of OBP2A for pT25 and Positive Sequencing of pUT18-OBP2A
Aim of Experiment: To perform a restriction digest of OBP2A for subsequent ligation into PT25. Also, sequencing came back positive for pUT18-OBP2A, so pUT18-OBP2A can now be implemented in the bacterial two hybrid system.
Protocols Used: Restriction Digests
Results: N/A
Next Steps: The next stage of experimentation will be to ligate OBP2A into the pT25 vector.
30/7: Ligation and Transformation of OBP2A into pT25.
Aim of Experiment: To perform a ligation and transformation of OBP2A into the bacterial two hybrid vector pT25, so that the interaction between the two constituents of OBP2A can be observed using the bacterial two hybrid system.
Protocols Used: Ligation Transformation
Results: N/A
Next Steps: If the transformation is successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight cultures of those colonies in preparation for purification and sequencing.
31/7: Overnight Cultures of pT25-OBP2A from Positive Colonies.
Aim of Experiment: To produce overnight cultures of pT25-OBP2A from the transformation done on the 30/7.
Protocols Used: Overnight Culture
Results: N/A
Next Steps: Once the cells have grown overnight they will undergo plasmid purification for subsequent sequencing.
1/8: Plasmid Purification and Subsequent Presequence Restriction Digest of pT25-OBP2A
Aim of Experiment: To perform a plasmid purification of pT25-OBP2A from the overnight cultures done on the 31/7 and then a subsequent presequence digest.
Protocols Used: Plasmid Purification Restriction Digest
Results: Figure 7
Next Steps: If sequencing comes back positive for pT25-OBP2A then this can now be implemented into the bacterial two hybrid system. The results from the sequencing should arrive early on the 3/8.
Week Beginning 3/8/2015
Summary
Positive sequencing of pT25-OBP2A, subsequent transformation into the bacterial two hybrid system and plating onto MacConkey agar.
3/8: Transformation of pUT18-OBP2A into BTH101 E.coli strain.
Aim of Experiment: To perform a transformation of pUT18-OBP2A into BTH101, so that a subsequent transformation of pT25-OBP2A into the same chassis can occur as both plasmids are required to reside in the same chassis so that their interaction can be observed.
Protocols Used: Transformation
Results: N/A
Next Steps: If the transformation is successful, then the next stage of experimentation will be to re-transform this culture of cells using the pT25-OBP2A plasmid.
4/8:Day culture and transformation of BTH101 containing pUT18-OBP2A with pT25OBP2A
Aim of Experiment: To perform a day culture and then a subsequent transformation of the BTH101 chassis containing  pUT18-OBP2A from the transformation performed on 3/8.
Protocols Used: Transformation
Results: N/A
Next Steps: If the transformation is successful then the next stage of experimentation will be to observe the  protein interaction of the two subunits of OBP2A using MacConkey agar plates.
5/8:Day culture and plating of positive colonies from transformation onto MacConkey agar.
Aim of Experiment: To perform a day culture and then subsequent plating of positive colonies from the transformation performed on the 4/8. Positive colonies from this transformation contain both vectors of the bacterial two hybrid system along with the two subunits of OBP2A and so now the interaction can be observed. The day culture will be plated onto MacConkey agar which once the cells have been given enough time to grow will give a visual demonstration of the  protein interaction through a color change.
Protocols Used: Transformation
Results: N/A
Next Steps: It has been decided that whilst the E.coli strain BTH101 that is being used in this experiment is useful for visualizing protein interaction, it is very unstable. A better E.coli strain called MG1655(Δcya) will be used instead as it is much more stable. This means that tomorrow overnights will be prepared of MG1655(Δcya)for transformation with pUT18-OBP2A and then subsequently pT25-OBP2A.
6/8: Overnight cultures of MG1655(Δcya)
Aim of Experiment: To perform an overnight culture of MG1655(Δcya)E.coli in preparation for transformation with pUT18-OBP2A and then pT25-OBP2A.
Protocols Used: Overnight Culture
Results: N/A
Next Steps: Once the overnight culture has been give sufficient time to grow these cells will then be transformed with pUT18-OBP2A and then subsequently re-transformed with pT25-OBP2A.
7/8: Results of MacConkey agar growth and transformation of  MG1655(Δcya)with pUT18-OBP2A
Aim of Experiment: To perform a transformation of MG1655(Δcya)with pUT18-OBP2A. Also the results of plating of our bacterial two hybrid interaction onto MacConkey agar has produced promising results as the colonies appear to be taking up a small amount of red dye - an indicator of protein interaction. This suggests that the two separated parts of OBP2A may in fact be interacting in vivo. This interaction will need to be quantified through the use of the β-galactosidase assay.
Protocols Used: Transformation
Results: N/A
Next Steps: If the transformation of MG1655(Δcya) with pUT18-OBP2A is successful then the next step of experimentation will be re-transform positive colonies with pT25-OBP2A.
8/8: Transformation of MG1655(Δcya) containing pUT18-OBP2A with pT25-OBP2A
Aim of Experiment: To perform a transformation of MG1655(Δcya) containing pUT18-OBP2A with pT25-OBP2A. Positive colonies were produced from the transformation done on the 7/7 and so as a result a day culture was produced and subsequently re-transformed with pT25-OBP2A.
Protocols Used: Transformation
Results: N/A
Next Steps: If the transformation of MG1655(Δcya) with pT25-OBP2A is successful then the next step of experimentation will be to undertake the β-galactosidase assay
Week Beginning 10/8/2015
Summary
First attempt at determination of the protein interaction of the OBP2A subunits using the β-galactosidase assay.
10/8: Plating of Controls in preparation of the β-galactosidase assay
Aim of Experiment: To plate control interactions that are used in the β-galactosidase assay to aid in the interpretation of the results form the assay. This first assay attempt will consist of one positive control, one negative control and one regulatory control.
Protocols Used: N/A
Results: N/A
Next Steps: The next stage of experimentation will be to perform overnight cultures from positive colonies in preparation for the β-galactosidase assay.
11/8: Overnight cultures of controls and OBP2A in preparation for the β-galactosidase assay.
Aim of Experiment: To prepare overnight cultures of positive colonies of MG1655(Δcya) containing pUT18-OBP2A and pT25-OBP2A, along with the controls in preparation for the  β-galactosidase assay.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: Once the cells have been given sufficient time to grow then the next stage of experimentation will be to inoculate those colonies into fresh liquid broth until they reach an O.D of 0.4 before being pellet and frozen for the assay.
12/8: Inoculation and preparation of cultures for the  β-galactosidase assay.
Aim of Experiment: To prepare cultures for use in the  β-galactosidase assay, this requires a small subsample of each overnight to be inoculated into fresh LB, two done for each colony. Then left to grow until the O.D of each culture is between 0.3-0.5. Once an O.D of 0.3-0.5 has been reached then 3 replicas of 1 ml samples are taken from each culture before being spun down, the supernatant then subsequently removed and the pellet frozen.
Protocols Used: Assay Preparation
Results: N/A
Next Steps: These samples are now ready for the β-galactosidase assay.
13/8: First attempt at the determination of the protein interaction of the two OBP2A subunits using the β-galactosidase assay
Aim of Experiment: To determine the millers activity of the two interacting subunits of OBP2A in vivo as a basis of the level of interaction between them. This was done using the β-galactosidase assay.
Protocols Used: β-galactosidase assay
Results: Figure 8
Next Steps: As it was the first attempt at a β-galactosidase assay, subsequent assays will be performed utilizing more controls.
14/8: Preparation of overnight cultures of controls and samples in preparation of performing a second β-galactosidase assay
Aim of Experiment: To perform overnight cultures of the controls and samples that will be necessary for doing this second β-galactosidase assay. This assay will include two positive controls which are known to have a high level of interaction - these are NarG-NarJ and Zip positive controls. There will also be two negative controls, an empty MG1655(Δcya) chassis and pUT18-OBP2A with NarJ. Finally there is one regulatory control: which contains two empty vectors of pUT18 and pT25.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The next stage of experimentation will be to prepare samples for use in the β-galactosidase assay.
Week Beginning 17/8/2015
Summary
Second attempt at determining the protein interactions of the two OBP2A subunits using the β-galactosidase assay and subsequent cloning of OBP2A into the bacterial two hybrid vectors in switched order.
17/8: Preparation of the controls and samples for use in the second attempt of the β-galactosidase assay
Aim of Experiment: To prepare cultures for use in the  β-galactosidase assay, this requires a small subsample of each overnight to be inoculated into fresh LB, two done for each colony. Then left to grow until the O.D of each culture is between 0.3-0.5. Once an O.D of 0.3-0.5 has been reached then 3 replicas of 1 ml samples are taken from each culture before being spun down, the supernatant then subsequently removed and the pellet frozen.
Protocols Used: β-galactosidase assay
Results: N/A
Next Steps: The next stage of experimentation will be to perform the second β-galactosidase assay on the interaction between the two subunits of OBP2A
18/8: Second attempt at determination of the protein interactions of the OBP2A subunits using the β-galactosidase assay
Aim of Experiment: To perform a second attempt at the β-galactosidase assay that will aid in the determination of the level of interaction that the two separated subunits of OBP2A are inflicting on one another.
Protocols Used: β-Galactosidase assay
Results: Figure 10
Next Steps: The next stage of experimentation will be to switch the positioning of the two subunits of OBP2A in pUT18 and pT25 by having the smaller subunit in pUT18 and the large subunit in pT25. Along with this another β-galactosidase assay will be performed however this time it will be done under anaerobic conditions to see if that has any appreciable effect on the interaction of the two subunits of OBP2A.
20/8: PCR of OBP2A in preparation of pUT18 and pT25 insertion
Aim of Experiment: To  Perform a PCR of OBP2A using primers for pU1T18 and pT25 insertion which have been designed to place the smaller OBP2Asubunit into pUT18 and the larger subunit into pT25.
Protocols Used: PCR
Results: N/A
Next Steps: If PCR is successful then the next stage of experimentation will be to perform a gel extraction and restriction in perpetration for ligation and transformation of OBP2A into pUT18 and pT25.
21/8: Second attempt at a PCR of OBP2A in preparation of pUT18 and pT25 insertion
Aim of Experiment: The PCR performed on the 20/8 failed as no bands were found to be present on the gel during gel viewing. As a result of this a subsequent PCR was done today.
Protocols Used: PCR
Results: N/A
Next Steps: If PCR is successful then the next stage in experimentation will be to perform a gel extraction and restriction in perpetration for ligation and transformation of OBP2A in pUT18 and pT25.
22/8: Second attempt at PCR of OBP2A for insertion into pUT18 and pT25 failed
Aim of Experiment: A second attempt at preparing OBP2A for insertion into the two vectors of the bacterial two hybrid system(pUT18 and pT25) has failed! As a result this will be left until next week, so that the rest of the week can be dedicated to preparing and undertaking  another β-galactosidase assay, that will be done under anaerobic conditions.
Protocols Used: N/A
Results: N/A
Next Steps: The next stage of experimentation is to prepare for a anaerobic β-galactosidase assay of OBP2A in the bacterial two hybrid vectors; pUT18 and pT25.
Week Beginning 24/8/2015
Summary
Performed an anaerobic β-galactosidase assay in order to determine the level of interaction between the two subunits of OBP2A.
24/8: Preparation of samples for the anaerobic β-galactosidase assay of OBP2A in pUT18 and pT25
Aim of Experiment: To prepare the  samples and controls of OBP2A for the anaerobic β-galactosidase assay, so they can be frozen overnight in preparation for the assay itself.
Protocols Used: Assay Preparation
Results:
Next Steps: The next stage of experimentation will be to perform the β-galactosidase assay itself.
25/8: Performed an anaerobic β-galactosidase assay in order to determine the level of interaction of OBP2A in the bacterial two hybrid vectors: pUT18 and pT25.
Aim of Experiment: Performed an anaerobic β-galactosidase assay of OBP2A in the bacterial two hybrid vectors: pUT18 and pT25. This is to determine the level of interaction that the smaller subunit of OBP2A has with its larger subunit in vivo. This particular assay has two positive controls, two negative controls and one regulatory control.
Protocols Used: β-galactosidase assay
Results: Figure 12
Next Steps: The results have shown that the two subunits of OBP2A in fact don't seem to be interacting in vivo - a possible reason for this might be aggregation of the two subunits preventing interaction. The  next stage of experimentation will be to test out the interaction when the subunit are switch in relation to heir positions on the vectors: pUT18 and pT25.
Watch Our Introduction Video
Dundee University
Located on the east coast of Scotland, The University of Dundee is one of Scotland's leading research institutes.
Social Media
youtube.com/DundeeiGEM2015
facebook.com/iGEMDundee2015
twitter.com/DundeeiGEMTeam
  University of Dundee
 Home
 About Us
Meet the Team
Attributions
 Project
Abstract
Project Overview
Devices
FluID
Fingerprint Ageing
Chromate Biosensor
Parts
Medals
Future Work
 Wet Lab
Lab Book
FluID
Fingerprints
Chromate Biosensor
Useful Information
Protocols
Sequences
Bacterial Two Hybrid
Safety
 Dry Lab
Introduction
FluID
Fingerprint Ageing
Chromate Biosensor
Appendix : Code
 Outreach
Practices
Ethics
Collaborations
