<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_labjournal_test skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/labjournal test</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><H1><HIGHLIGHT class="highlight">LABJOURNAL</HIGHLIGHT></H1><SECTION id="about" class="row1"><DIV class="row"><DIV class="col-lg-4"><H3>BioSpray</H3><P class="about-content">Our forensic toolkit aims to use synthetic biology approaches to improve on the current 
methods used by crime scene investigators.</P></DIV><DIV class="col-lg-4"><H3>Chromium Detector</H3><P class="about-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur a tincidunt elit. Aliquam porta nibh at enim luctus, auctor consequat dolor vehicula.</P></DIV><DIV class="col-lg-4"><H3>Fingerprint Aging</H3><P class="about-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur a tincidunt elit. Aliquam porta nibh at enim luctus, auctor consequat dolor vehicula.</P></DIV></DIV></SECTION><DIV class="ribbon"><DIV class="rows"><DIV class="row"><DIV class="border-week"><DIV class="box"><SPAN id="biospray-title">Saliva</SPAN><DIV id="sacollapsible" class="collapse biospray-content"><DIV class="row"><DIV class="box"><DIV class="labtitle">Week Beginning 1/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"><I>Lactoferrin binding protein A (LbpA)</I> was successfully cloned into the vector pSB1C3.</P></DIV><DIV id="sacollapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/6: Amplification of <I>LbpA</I></SPAN><DIV id="sacollapsiblew1-1" class="collapse box-content"><P><B>Aim of Experiment:</B> Amplify <I>LbpA</I> from <I>N.meningitidis</I> template chromosome.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> Figure 1 </A></P><P><B>Next Steps:</B> Since the <I>LbpA</I> gene was amplified successfully, it was ready to be digested for subsequent cloning into pSB1C3.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">2/6: Restriction Digests and Ligations</SPAN><DIV id="sacollapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To digest <I>LbpA</I> with BamHI and EcoRI and then ligate it into pSB1C3.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results:</B> N/A


                  </P><P><B>Next Steps:</B> Ligations of <I>LbpA</I> + pSB1C3 were left overnight until the next day when they would be transformed into the JM110 strain of <I>E.coli</I>.
                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/6: Transformations of <I>LpbA</I> + pSB1C3 into JM110 <I>E.coli</I> Strain </SPAN><DIV id="sacollapsiblew1-3" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the pSB1C3 + <I>LbpA</I> ligations into JM110 <I>E.coli</I> cells.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A></P><P><B>Results:</B> N/A
                  </P><P><B>Next Steps:</B> Transformations will be checked tomorrow and if successful, overnight cultures will be set up.
                  
                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">4/6: Overnight Cultures</SPAN><DIV id="sacollapsiblew1-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of the <I>LbpA</I> + pSB1C3 JM110 colonies.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures
          </A></P><P><B>Results:</B> N/A
                  </P><P><B>Next Steps:</B> A miniprep will be performed on the overnight cultures and a pre-sequence digest test performed tomorrow. 
                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">5/6: Plasmid Purification and Pre-Sequencing Digest Check</SPAN><DIV id="sacollapsiblew1-5" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the <I>LbpA</I> + pSB1C3 JM110 overnight cultures and perform a pre-sequence restriction digest to check for the presence of the insert.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprepÂ® Spin Miniprep Kit) </A><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests
          </A></P><P><B>Results:</B> N/A
                  </P><P><B>Next Steps:</B> Since the pre-sequencing digest worked, the miniprep will be sent for sequencing.                </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 8/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Sequencing confirmed that <I>LbpA</I> had been successfully cloned into pSB1C3 therefore this week cloning was attempted to put <I>LbpA</I> into the high expression vector pQE80-L.</P></DIV><DIV id="sacollapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">8/6: Amplification of <I>LbpA</I> into pQE80-L</SPAN><DIV id="sacollapsiblew2-1" class="collapse box-content"><P><B>Aim of experiment:</B> To amplify <I>LbpA</I> for subsequent cloninng into the pQE80-L vector.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure2-modal" class="journal-protocol"> Figure 2 </A></P><P><B> Next Steps:</B> The PCR product will be digested and ligated into pQE80-L tomorrrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">9/6: Restriction Digests and Ligations of <I>LbpA</I>into pQE80-L</SPAN><DIV id="sacollapsiblew2-2" class="collapse box-content"><P><B>Aim of experiment:</B>To restrict and ligate the <I>LbpA</I> gene into the pQE80-L plasmid.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations </A></P><P><B> Results:</B>: N/A </P><P><B> Next Steps:</B> Transformations of <I>LbpA</I> + pQE80-L into the JM110 strain will be carried out tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/6: Transformations of <I>LbpA</I> + pQE80-L into JM110</SPAN><DIV id="sacollapsiblew2-3" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the <I>LbpA</I> + pQE80-L ligations into the E.coli strain JM110.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A></P><P><B>Results:</B> N/A </P><P><B> Next Steps:</B> If the transformations are successful, colonies will be picked to set up overnight cultures. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/6: Colony PCR of <I>LbpA</I> + pQE80-L Transformation</SPAN><DIV id="sacollapsiblew2-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up colony PCR from the single colony obtained from the 2:1 ligation transformation. </P><P><B>Protocols Used: </B></P><P><B>Results: </B></P><P><B>Next Steps:</B> The gel indicates that the pQE80-L vector has sealed without the presence of the <I>LbpA</I> gene. <I>LbpA</I> will amplified again tomorrow. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 15/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">This week, cloning of <I>LbpA</I> into pQE80-L was continued due to last week's failed attempts.</P></DIV><DIV id="sacollapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">15/6: PCR of <I>LbpA</I> for Cloning into pQE80-L</SPAN><DIV id="sacollapsiblew3-1" class="collapse box-content"><P><B>Aim of experiment:</B> To re-amplify the <I>LbpA</I> gene for cloning into the pQE80-L vector.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure3-modal" class="journal-protocol">Figure 3</A></P><P><B>Next Steps:</B> The gel shows that <I>LbpA</I> has been successfully amplified so it will be digested and ligated into the pQE80-L vector tomorrow.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 22/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"><I>LbpA</I> was successfully cloned into the high expression vector pQE80-L.</P></DIV><DIV id="sacollapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/6: Restriction Digest and Ligation of <I>LbpA</I> into pQE80-L</SPAN><DIV id="sacollapsiblew4-1" class="collapse box-content"><P><B>Aim of experiment:</B> To digest the amplified <I>LbpA</I> gene with BamHI and KpnI and subsequently ligate it into the pQE80-L vector.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The ligations will be transformed into the M15pREP4 strain of <I>E.coli</I>. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/6: Transformations of <I>LbpA</I> + pQE80-L into E.coli</SPAN><DIV id="sacollapsiblew4-2" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the ligations of <I>LbpA</I>+ pQE80-L into the m15pREP4 strain of <I>E.coli.</I></P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Overnight cultures will be set up tomorrow if the transformations are successful. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">25/6: Overnight Cultures of <I>LbpA</I> + pQE80-L Transformations</SPAN><DIV id="sacollapsiblew4-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of the colonies obtained from the <I>LbpA</I> + pQE80-L transformations.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Overnight cultures will be miniprepped and a pre-sequence restriction digest will be performed to check for the presence of the <I>LbpA</I> gene. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">26/6: Plasmid Purification of the <I>LbpA</I> + pQE80-L Overnight Cultures</SPAN><DIV id="sacollapsiblew4-4" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures from yesterday and perform a pre-sequencing restriction digest using BamHI and KpnI to check for presence of <I>LbpA</I> gene.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification (QIAprepÂ® Spin Miniprep Kit)</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> A sample of the miniprep was sent for sequencing to confirm the presence of <I>LbpA</I> in the pQE80-L vector. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 29/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Protein expression optimization experiments were performed on LbpA to assay protein production levels. However, cell growth seems to cease when <I>LbpA</I> expression is induced.</P></DIV><DIV id="sacollapsiblew5" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/7: <I>LbpA</I> + pQE80-L Overnight Cultures</SPAN><DIV id="sacollapsiblew5-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures in preparation for protein expression experiments tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The overnight cultures will be used to set up day cultures tomorrow which will be used to test expression of LbpA. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">2/7: Optimization of <I>LbpA</I> Expression</SPAN><DIV id="sacollapsiblew5-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up day cultures and induce expression of <I>LbpA</I> using different concentrations of IPTG and check levels of protein production via SDS-PAGE.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> The samples were stored in the -20oC freezer overnight and will be ran on a gel tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/7: Continuation of Optimization of <I>LbpA</I> Expression</SPAN><DIV id="sacollapsiblew5-3" class="collapse box-content"><P><B>Aim of experiment:</B> To test the samples obtained yesterday on an SDS gel.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure4-modal" class="journal-protocol"> Figure 4 </A></P><P><B>Next Steps:</B> Further optimization experiments will be set up next week using different concentrations of IPTG. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 6/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Further protein expression optimization experiments were set up, this time using a range of IPTG concentrations to induce expression of <I>LbpA</I>.</P></DIV><DIV id="sacollapsiblew6" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">7/7: Further Optimization of <I>LbpA</I> Expression</SPAN><DIV id="sacollapsiblew6-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up further tests to optimize expression of LbpA using different concentrations of IPTG.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization </A> Note: An SDS gel was not ran, instead the OD<SUB>600</SUB> readings were recorded for the uninduced and induced cultures at 20 minute intervals for 2 hours, 1 and half hours after the cultures were induced. </P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure5-modal" class="journal-protocol"> Figure 5 </A></P><P><B>Next Steps:</B> A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after <I>LbpA</I> expression has been induced. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 13/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Protein expression optimization experiments were continued this week. A growth curve assay was carried out to assess the effect of <I>LbpA</I> expression over a longer period of time.  </P></DIV><DIV id="sacollapsiblew7" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/7: Plate Reader Growth Curve Assay</SPAN><DIV id="sacollapsiblew7-1" class="collapse box-content"><P><B>Aim of experiment:</B> To induce expression of <I>LbpA</I> using different concentration of IPTG and allow cells to grow for 16 hours in order to monitor cell growth.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#platereader-modal" class="journal-protocol"> Growth Curve Assay </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure6-modal" class="journal-protocol"> Figure 6 </A></P><P><B>Next Steps:</B> A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after <I>LbpA</I> expression has been induced. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/7: Sample Preparation for Western Blot</SPAN><DIV id="sacollapsiblew7-2" class="collapse box-content"><P><B>Aim of experiment:</B> To check if the cells in the induced cultures ~7 hours after being induced are producing LbpA. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot Sample Preparation </A> Note: The day cultures were grown for 7 hours after they were induced and then the samples were frozen, to be blotted on Monday. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The samples will be ran on an SDS gel on Monday and blotted. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 20/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Samples from the growth curve assay were blotted to see if LbpA is being produced at the later stages of the growth curve. A further growth curve assay was also performed which monitored cell growth until a certain OD<SUB>600</SUB> was reached, at which point the cultures were induced.</P></DIV><DIV id="sacollapsiblew8" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">20/7: Western Blot LpbA Culture Samples</SPAN><DIV id="sacollapsiblew8-1" class="collapse box-content"><P><B>Aim of experiment:</B> To western blot the LbpA culture samples from Friday.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot </A></P><P><B>Results:</B> Nothing was visible on the blot. </P><P><B>Next Steps:</B> Another plate reader experiment will be set up on Thursday where the cultures will be monitored for three hours before induction with IPTG and then for a further 20 hours afterwards. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/7: Growth Curve Assay</SPAN><DIV id="sacollapsiblew8-2" class="collapse box-content"><P><B>Aim of experiment:</B> To check if the cells begin to die after induction with IPTG after having already grown to an OD<SUB>600</SUB> of 0.5. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#platereader2-modal" class="journal-protocol"> Growth Curve Assay </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure7-modal" class="journal-protocol"> Figure 7 </A></P><P><B>Next Steps:</B> Samples of these cultures will be taken 6 hours after induction to check for the presence of LbpA. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 27/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">The presence of LbpA was confirmed in the culture 6 hours after induction with IPTG so protein purification was started this week. </P></DIV><DIV id="sacollapsiblew9" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/7: Overnight Cultures</SPAN><DIV id="sacollapsiblew9-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up a 150ml overnight culture for sub-culturing tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A> Note: 150ml of LB was used and 150Âµl of each antibiotic. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>150ml of fresh LB will be inoculated tomorrow with 7.5ml of the overnight cultures. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/7: <I>LbpA</I> Expression Assay</SPAN><DIV id="sacollapsiblew9-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up day cultures using the overnight cultures from yesterday and test for expression of LbpA 6 hours after induction.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Test </A> Note: 150ml of fresh LB was innoculated with 7.5ml of the overnight cultures. Only an uninduced control and an 1mM IPTG induced culture was set up. The samples were not blotted, they were only ran on an SDS gel. </P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure8-modal" class="journal-protocol"> Figure 8 </A></P><P><B>Next Steps:</B> A 3L culture will be set up on Thursday under the same conditions as the overnight cultures in order to try and purify LbpA.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/7: Overnight Cultures for 3L Culture</SPAN><DIV id="sacollapsiblew9-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures for the 3L day cultures that will be grown tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A> Note: One 150ml overnight culture was set up using 150Âµl of each antibiotic. </P><P><B>Results:</B> N/A   </P><P><B>Next Steps:</B> The 3L culture will be set up tomorrow allowed to grow to OD<SUB>600</SUB> of 0.6-1 and then induced with 1mM IPTG and grown for 6 hours.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/7: 3L Culture for Purification of LbpA</SPAN><DIV id="sacollapsiblew9-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 3L day cultures for purifying LbpA.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The procedure was stopped at Step 6 and the pellets were frozen at -20<SUP>o</SUP>C. </P><P><B>Results:</B> N/A   </P><P><B>Next Steps:</B> Protein purification will be continued on Monday.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 3/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">The first round of LbpA purification failed at the affinity chromatography stage so another 6L culture was grown this week to attempt purification again.</P></DIV><DIV id="sacollapsiblew10" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/8: Continuation of LbpA Purification</SPAN><DIV id="sacollapsiblew10-1" class="collapse box-content"><P><B>Aim of experiment:</B> To continue with purification of LbpA.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The protocol was continued from Step 7</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>Size exclusion chromatography was not carried out since an SDS gel showed LbpA was not present after performing affinity chromatography. Another 6L culture will be set up this week.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">5/8: Overnight Cultures for 6L Culture</SPAN><DIV id="sacollapsiblew10-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures for the 6L culture that will be set up tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A>Note: One 150ml overnight culture was set up using 150Âµl of each antibiotic.</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>The overnight cultures will be used to set up the 6L day culture tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">6/8: 6L Day Cultures for LbpA Purification</SPAN><DIV id="sacollapsiblew10-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 6L day cultures for purifying LbpA and begin the purification process.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: The protocol was stopped at Step 6.</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>Purification of LbpA will be continued on Monday. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">heh</P></DIV><DIV id="sacollapsiblew11" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/8: Continuation of LbpA Purification</SPAN><DIV id="sacollapsiblew11-1" class="collapse box-content"><P><B>Aim of experiment:</B> To continue with purification of LbpA.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The protocol was continued from Step 7</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><SPAN id="biospray-title">Semen</SPAN><DIV id="secollapsible" class="collapse biospray-content"><DIV class="row"><DIV class="box"><DIV class="labtitle">Week Beginning 25/5/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P></DIV><DIV id="secollapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/5: PCR of <I>PotD</I> from MG1655 <I>E.coli</I> gDNA</SPAN><DIV id="secollapsiblew1-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To amplify <I>PotD</I> from MG1655 <I>E.coli</I> gDNA.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results: </B>N/A</P><P><B>Next Steps:</B> The PCR product will be gel extracted, digested and ligated into the biobrick vector pSB1C3 tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/5: Gel Extraction, Restriction Digest and Ligation of <I>SBP</I> into pSB1C3</SPAN><DIV id="secollapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To gel extract the <I>PotD</I> PCR product and digest it with PstI and EcoRI. A ligation will then be set up with pSB1C3.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digest</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> Ligations will be transformed into JM110 <I>E.coli</I> on Monday.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 1/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Sequencing confirmed that <I>LbpA</I> had been successfully cloned into pSB1C3 therefore this week cloning was attempted to put <I>LbpA</I> into the high expression vector pQE80-L.</P></DIV><DIV id="secollapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/6: Transformation of pSB1C3 +<I>PotD</I> into JM110 <I>E.coli</I></SPAN><DIV id="secollapsiblew2-1" class="collapse box-content"><P><B>Aim of experiment:</B> To transform pSB1C3 + <I>PotD</I> into JM110 <I>E.coli</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A></P><P><B>Results:</B>N/A</P><P><B> Next Steps:</B> If the transformations are successful, overnight cultures will be set up tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">2/6: Overnight Cultures of pSB1C3 + <I>PotD</I> in JM110 <I>E.coli</I></SPAN><DIV id="secollapsiblew2-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of pSB1C3 + <I>PotD</I> in JM110 <I>E.coli</I></P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B> Results:</B>: N/A </P><P><B> Next Steps:</B> The overnight cultures will be miniprepped tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/6: Plasmid Purification of pSB1C3 + <I>PotD</I> from Overnight Cultures</SPAN><DIV id="secollapsiblew2-3" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures from yesterday.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprepÂ® Spin Miniprep Kit) </A></P><P><B>Results:</B> N/A </P><P><B> Next Steps:</B> The miniprep with the highest concentration will be sent for sequencing to confirm <I>PotD</I> has been inserted into pSB1C3 successfully.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 15/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">This week, cloning of <I>LbpA</I> into pQE80-L was continued due to last week's failed attempts.</P></DIV><DIV id="secollapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">15/6: PCR of <I>PotD</I> for Cloning into pQE80-L</SPAN><DIV id="secollapsiblew3-1" class="collapse box-content"><P><B>Aim of experiment:</B> To amplify the <I>PotD</I> gene for cloning into the pQE80-L vector.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure3-modal" class="journal-protocol">Figure 3</A></P><P><B>Next Steps:</B> The gel shows that <I>PotD</I> has been successfully amplified so it will be digested and ligated into the pQE80-L vector tomorrow.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">16/6: Gel Extraction of <I>PotD</I> PCR Product</SPAN><DIV id="secollapsiblew3-2" class="collapse box-content"><P><B>Aim of experiment:</B> To gel extract the <I>PotD</I> PCR product.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure3-modal" class="journal-protocol">Figure 3</A></P><P><B>Next Steps:</B> The gel shows that <I>PotD</I> has been successfully amplified so it will be digested and ligated into the pQE80-L vector tomorrow.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/6: Restriction Digest and Ligation of <I>PotD</I> into pQE80-L. Transformation of IDT Plasmid containing <I>SBP</I> into MC1061 <I>E.coli</I></SPAN><DIV id="secollapsiblew3-3" class="collapse box-content"><P><B>Aim of experiment:</B> To digest the <I>PotD</I> PCR product with KpnI and BamHI and then ligate it into pQE80-L. To transform MC1061 <I>E.coli</I> with the IDT plasmid containing <I>SBP</I>.</P><P><B>Protocols Used: </B></P><I>PotD</I>: <A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A><I>SBP</I>:<A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformations</A><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> The <I>PotD</I> + pQE80-L ligations will be transformed into MC1061 <I>E.coli</I>. Overnight cultures of the transformations will be set up tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/6: Transformations of <I>PotD</I> + pQE80-L into MC1061 <I>E.coli</I>. Overnight Cultures of MC1061 <I>E.coli</I> with SBP + IDT Plasmid</SPAN><DIV id="secollapsiblew3-4" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the <I>PotD</I> + pQE80-L ligations into MC1061 <I>E.coli</I>. To set up overnight cultures of the transformations of MC1061 <I>E.coli</I> with SBP + IDT Plasmid that were set up yesterday.</P><P><B>Protocols Used: </B><I>PotD</I>:<A data-toggle="modal" data-target="#transformations-modal" class="journal-protocol"> Transformations</A><I>SBP</I>: <A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure3-modal" class="journal-protocol">Figure 3</A></P><P><B>Next Steps:</B> If transformations are successful, overnight cultures will be set up tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">19/6: Repeat of Ligation of <I>PotD</I> into pQE80-L. Plasmid Purification of the IDT Plasmid Containing <I>SBP</I></SPAN><DIV id="secollapsiblew3-5" class="collapse box-content"><P><B>Aim of experiment:</B> To repeat ligations of <I>PotD</I> into pQE80-L since transformations did not work. To miniprep the overnight cultures that were set up yesterday.</P><P><B>Protocols Used: </B><I>PotD</I>: <A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A><I>SBP</I>: <A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprepÂ® Spin Miniprep Kit)</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Ligations will be left over the weekend and transformed on Monday. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 22/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P></DIV><DIV id="secollapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/6: Transformation of <I>PotD</I> + pQE80-L Ligations. PCR and Gel Extraction of <I>SBP</I> for Cloning into pQE80-L and Restriction Digests and Ligation of <I>SBP</I> for Cloning into pSB1C3</SPAN><DIV id="secollapsiblew4-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To transform <I>PotD</I> + pQE80-L into  MC1061<I>E.coli</I>. To set up a PCR of <I>SBP</I> using the purified plasmid from Friday and to gel extract the PCR product for cloning into pQE80-L. To digest <I>SBP</I> out of the same plasmid using EcoRI and PstI and subsequently ligate it into pSB1C3.</P><P><B>Protocols Used:</B><I>PotD</I>: <A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A><I>SBP</I>: <A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results: </B><I>SBP</I> :<A data-toggle="modal" data-target="#sbp-figure1-modal" class="journal-protocol"> Figure 1 </A></P><P><B>Next Steps:</B> Overnight cultures will be set up tomorrow for the <I>PotD</I> transformations. The <I>SBP</I> PCR product will be digested and ligated into pQE80-L tomorrow. The <I>SBP</I> + pSB1C3 ligations will be transformed into MC1061 <I>E.coli</I>. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/6: Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>PotD</I>. Transformations of MC1061 <I>E.coli</I> with pSB1C3 + <I>SBP</I>. Restriction Digests and Ligations of <I>SBP</I> into pQE80-L and Transformations into MC1061 <I>E.coli</I></SPAN><DIV id="secollapsiblew4-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>PotD</I>. To transform MC1061 <I>E.coli with pSB1C3 + <I>SBP</I>. To digest the <I>SBP</I> PCR product with BamHI and Kpn I, ligate it into pQE80-L and transform these ligations into MC1061 <I>E.coli</I>.</I></P><P><B>Protocols Used:</B><I>PotD</I>: <A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A><I>SBP</I>: <A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The MC1061 <I>E.coli</I> overnight cultures containing pQE80-L + <I> PotD</I> will be miniiprepped tomorrow. Overnight cultures will be set up of the MC1061 <I>E.coli</I> transformations with pSB1C3 + <I>SBP</I> and pQE80-L + <I>SBP</I>.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/6: Plasmid Purification of the Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>PotD</I>. Overnight Cultures of the Transformations of MC1061 <I>E.coli with pSB1C3 + <I>SBP</I> and pQE80-L + <I>SBP</I>.</I></SPAN><DIV id="secollapsiblew4-3" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>PotD</I>. To set up overnight cultures of the transformations of MC1061 <I>E.coli with pSB1C3 + <I>SBP</I> and pQE80-L + <I>SBP</I></I></P><P><B>Protocols Used:</B><I>PotD</I>: <A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprepÂ® Spin Miniprep Kit)</A><I>SBP</I>: <A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The miniprep with the highest concentration for pQE80-L + <I>PotD</I> was sent for sequencing. The <I>SBP</I> overnight cultures will be miniprepped tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">25/6: Plasmid Purification of the Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>SBP</I> and pQE80-L + <I>SBP</I></SPAN><DIV id="secollapsiblew4-4" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>SBP</I> and pQE80-L + <I>SBP</I>.</P><P><B>Protocols Used:</B><I>SBP</I>: <A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprepÂ® Spin Miniprep Kit) </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The miniprep with the highest concentration for both pSB1C3 + <I>SBP</I> and pQE80-L + <I>SBP</I> were sent for sequencing. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 29/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Sequencing showed <I> SBP</I> failed to insert successfully into pSB1C3 so cloning will be repeated from the ligation stage this week. However, SBP and PotD were successfully inserted into the high expression vector pQE80-L, so overexpression assays will be started this week for both.</P></DIV><DIV id="secollapsiblew5" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/6: Ligations of <I>SBP</I> into pSB1C3 and Transformations into MC1061 <I>E.coli</I></SPAN><DIV id="secollapsiblew5-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To ligate <I>SBP</I> into pSB1C3 and transform the ligations into MC1061 <I>E.coli</I>. </P><P><B>Protocols Used:</B><I>SBP</I>: <A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A></P><P><B>Results: </B><I>SBP</I> :<A data-toggle="modal" data-target="#sbp-figure1-modal" class="journal-protocol"> Figure 1 </A></P><P><B>Next Steps:</B> Overnight cultures will be set up tomorrow if transformations are successful. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/6: Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>SBP/i&gt;, pQE80-L + <I>SBP</I> and pQE80-L + <I>PotD</I>. Transformations of MC1061 <I>E.coli with pSB1C3 + <I>SBP</I>. Restriction Digests and Ligations of <I>SBP</I> into pQE80-L and Transformations into MC1061 <I>E.coli</I></I></I></SPAN><DIV id="secollapsiblew5-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>SBP</I>. To set up overnight cultures  of pQE80-L + <I>SBP</I> and pQE80-L + <I>PotD</I> in MC1061 <I>E.coli</I> for overexpression assays to be performed tomorrow. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The MC1061 <I>E.coli</I> overnight cultures containing pSB1C3 + <I> SBP</I> will be miniiprepped tomorrow. The MC1061 <I>E.coli</I> overnight cultures containing pQE80-L + <I> SBP</I> and pQE80-L + <I>PotD</I> will be subcultured tomorrow in order to perform overexpression assays.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/7: Plasmid Purification of the Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>SBP</I>. <I>PotD</I> and <I>SBP</I> Expression Optimization
              <DIV id="secollapsiblew5-3" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>SBP</I>. To start protein expression optimization for PotD and SBP.</P><P><B>Protocols Used:</B><I>SBP</I>: <A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprepÂ® Spin Miniprep Kit)</A><I>SBP</I> and <I>PotD</I>: <A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization</A> Note: Protocol was stopped at Step 8. Samples will be ran on an SDS gel tomorrow.</P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The miniprep with the highest concentration for pSB1C3 + <I>SBP</I> was sent for sequencing. The PotD and SBP samples will be ran on an SDS gel tomorrow. </P></DIV></SPAN></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">2/7: Continuation of <I>PotD</I> and <I>SBP</I> Expression Optimization</SPAN><DIV id="secollapsiblew5-4" class="collapse box-content"><P><B>Aim of experiment:</B> To run the samples of PotD and SBP on an SDS gel and begin Western Blotting.</P><P><B>Protocols Used:</B><I>PotD</I> and <I>SBP</I>: <A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization</A> Note: Protocol was continued from Step 9.</P><P><B>Results:</B><A data-toggle="modal" data-target="#sbp-figure2-modal" class="journal-protocol"> Figure 2 </A></P><P><B>Next Steps:</B> The membrane for the Western blot has been left at the blocking stage so will be continued tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/7: Western Blot of <I>PotD</I> and <I>SBP</I> Day Culture Samples</SPAN><DIV id="secollapsiblew5-5" class="collapse box-content"><P><B>Aim of experiment:</B> To continue the Western Blot of PotD and SBP.</P><P><B>Protocols Used:</B><I>PotD</I> and <I>SBP</I>: <A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#sbp-figure3-modal" class="journal-protocol"> Figure 3 </A></P><P><B>Next Steps:</B> PotD has been successfully characterized so purification of this protein will be started next week. Further expression optimization experiments for SBP will be set up next week using different concentrations of IPTG, since its characterization was unsuccessful.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 6/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Further protein expression optimization experiments were set up, this time using a range of IPTG concentrations to induce expression of <I>LbpA</I>.</P></DIV><DIV id="sacollapsiblew6" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">7/7: Further Optimization of <I>LbpA</I> Expression</SPAN><DIV id="sacollapsiblew6-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up further tests to optimize expression of LbpA using different concentrations of IPTG.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization </A> Note: An SDS gel was not ran, instead the OD<SUB>600</SUB> readings were recorded for the uninduced and induced cultures at 20 minute intervals for 2 hours, 1 and half hours after the cultures were induced. </P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure5-modal" class="journal-protocol"> Figure 5 </A></P><P><B>Next Steps:</B> A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after <I>LbpA</I> expression has been induced. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 13/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Protein expression optimization experiments were continued this week. A growth curve assay was carried out to assess the effect of <I>LbpA</I> expression over a longer period of time.  </P></DIV><DIV id="sacollapsiblew7" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/7: Plate Reader Growth Curve Assay</SPAN><DIV id="sacollapsiblew7-1" class="collapse box-content"><P><B>Aim of experiment:</B> To induce expression of <I>LbpA</I> using different concentration of IPTG and allow cells to grow for 16 hours in order to monitor cell growth.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#platereader-modal" class="journal-protocol"> Growth Curve Assay </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure6-modal" class="journal-protocol"> Figure 6 </A></P><P><B>Next Steps:</B> A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after <I>LbpA</I> expression has been induced. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/7: Sample Preparation for Western Blot</SPAN><DIV id="sacollapsiblew7-2" class="collapse box-content"><P><B>Aim of experiment:</B> To check if the cells in the induced cultures ~7 hours after being induced are producing LbpA. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot Sample Preparation </A> Note: The day cultures were grown for 7 hours after they were induced and then the samples were frozen, to be blotted on Monday. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The samples will be ran on an SDS gel on Monday and blotted. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 20/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Samples from the growth curve assay were blotted to see if LbpA is being produced at the later stages of the growth curve. A further growth curve assay was also performed which monitored cell growth until a certain OD<SUB>600</SUB> was reached, at which point the cultures were induced.</P></DIV><DIV id="sacollapsiblew8" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">20/7: Western Blot LpbA Culture Samples</SPAN><DIV id="sacollapsiblew8-1" class="collapse box-content"><P><B>Aim of experiment:</B> To western blot the LbpA culture samples from Friday.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot </A></P><P><B>Results:</B> Nothing was visible on the blot. </P><P><B>Next Steps:</B> Another plate reader experiment will be set up on Thursday where the cultures will be monitored for three hours before induction with IPTG and then for a further 20 hours afterwards. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/7: Growth Curve Assay</SPAN><DIV id="sacollapsiblew8-2" class="collapse box-content"><P><B>Aim of experiment:</B> To check if the cells begin to die after induction with IPTG after having already grown to an OD<SUB>600</SUB> of 0.5. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#platereader2-modal" class="journal-protocol"> Growth Curve Assay </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure7-modal" class="journal-protocol"> Figure 7 </A></P><P><B>Next Steps:</B> Samples of these cultures will be taken 6 hours after induction to check for the presence of LbpA. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 27/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">The presence of LbpA was confirmed in the culture 6 hours after induction with IPTG so protein purification was started this week. </P></DIV><DIV id="sacollapsiblew9" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/7: Overnight Cultures</SPAN><DIV id="sacollapsiblew9-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up a 150ml overnight culture for sub-culturing tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A> Note: 150ml of LB was used and 150Âµl of each antibiotic. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>150ml of fresh LB will be inoculated tomorrow with 7.5ml of the overnight cultures. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/7: <I>LbpA</I> Expression Assay</SPAN><DIV id="sacollapsiblew9-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up day cultures using the overnight cultures from yesterday and test for expression of LbpA 6 hours after induction.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Test </A> Note: 150ml of fresh LB was innoculated with 7.5ml of the overnight cultures. Only an uninduced control and an 1mM IPTG induced culture was set up. The samples were not blotted, they were only ran on an SDS gel. </P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure8-modal" class="journal-protocol"> Figure 8 </A></P><P><B>Next Steps:</B> A 3L culture will be set up on Thursday under the same conditions as the overnight cultures in order to try and purify LbpA.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/7: Overnight Cultures for 3L Culture</SPAN><DIV id="sacollapsiblew9-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures for the 3L day cultures that will be grown tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A> Note: One 150ml overnight culture was set up using 150Âµl of each antibiotic. </P><P><B>Results:</B> N/A   </P><P><B>Next Steps:</B> The 3L culture will be set up tomorrow allowed to grow to OD<SUB>600</SUB> of 0.6-1 and then induced with 1mM IPTG and grown for 6 hours.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/7: 3L Culture for Purification of LbpA</SPAN><DIV id="sacollapsiblew9-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 3L day cultures for purifying LbpA.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The procedure was stopped at Step 6 and the pellets were frozen at -20<SUP>o</SUP>C. </P><P><B>Results:</B> N/A   </P><P><B>Next Steps:</B> Protein purification will be continued on Monday.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 3/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">The first round of LbpA purification failed at the affinity chromatography stage so another 6L culture was grown this week to attempt purification again.</P></DIV><DIV id="sacollapsiblew10" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/8: Continuation of LbpA Purification</SPAN><DIV id="sacollapsiblew10-1" class="collapse box-content"><P><B>Aim of experiment:</B> To continue with purification of LbpA.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The protocol was continued from Step 7</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>Size exclusion chromatography was not carried out since an SDS gel showed LbpA was not present after performing affinity chromatography. Another 6L culture will be set up this week.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">5/8: Overnight Cultures for 6L Culture</SPAN><DIV id="sacollapsiblew10-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures for the 6L culture that will be set up tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A>Note: One 150ml overnight culture was set up using 150Âµl of each antibiotic.</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>The overnight cultures will be used to set up the 6L day culture tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">6/8: 6L Day Cultures for LbpA Purification</SPAN><DIV id="sacollapsiblew10-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 6L day cultures for purifying LbpA and begin the purification process.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: The protocol was stopped at Step 6.</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>Purification of LbpA will be continued on Monday. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">heh</P></DIV><DIV id="sacollapsiblew11" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/8: Continuation of LbpA Purification</SPAN><DIV id="sacollapsiblew11-1" class="collapse box-content"><P><B>Aim of experiment:</B> To continue with purification of LbpA.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The protocol was continued from Step 7</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="PCR-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>PCR Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the <STRONG>Sequences</STRONG> section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY color="black"><TR><TD width="312" valign="top"><P align="center"><STRONG>Reactants</STRONG></P></TD><TD width="312" valign="top"><P align="center"><STRONG>Volume (Âµl)</STRONG></P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DNA
                </P></TD><TD width="312" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DMSO
                </P></TD><TD width="312" valign="top"><P align="center">
                    2.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Forward Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Reverse Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase Buffer
                </P></TD><TD width="312" valign="top"><P align="center">
                    10
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    dNTP
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    H<SUB>2</SUB>O
                </P></TD><TD width="312" valign="top"><P align="center">
                    34.5
                </P></TD></TR></TBODY></TABLE><P>
    **PCR Programme**
</P><P>
    Elongation time was altered according to the length of the sequences.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="restrictiondigest-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Restriction Digests Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P><FONT color="#000000">
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the    <STRONG>Lab Book</STRONG> section.
</FONT></P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="170" valign="top" style="color:#000000;"><P align="center"><STRONG>Post Plasmid Purification</STRONG></P></TD><TD width="151" valign="top"><P align="center"><STRONG>Post PCR</STRONG></P></TD><TD width="139" valign="top"><P align="center"><STRONG>gBlock</STRONG></P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>DNA</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1mg
                </P></TD><TD width="151" valign="top"><P align="center">
                    50Âµl
                </P></TD><TD width="139" valign="top"><P align="center">
                    100ng
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Cutsmart Buffer (Âµl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    6
                </P></TD><TD width="151" valign="top"><P align="center">
                    6
                </P></TD><TD width="139" valign="top"><P align="center">
                    6
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Water (Âµl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    As required
                </P></TD><TD width="151" valign="top"><P align="center">
                    1
                </P></TD><TD width="139" valign="top"><P align="center">
                    As required
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Enzyme 1 (Âµl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="151" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="139" valign="top"><P align="center">
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Enzyme 2 (Âµl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="151" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="139" valign="top"><P align="center">
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG><U>Total Volume (Âµl)</U></STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    30
                </P></TD><TD width="151" valign="top"><P align="center">
                    60
                </P></TD><TD width="139" valign="top"><P align="center">
                    30
                </P></TD></TR></TBODY></TABLE><P>
    The digests were incubated at 37<SUP>o</SUP>C for 3 hours.
    
    Note that when performing plasmid digestions, 2.5Âµl of alkaline phosphatase and 2.5Âµl of Cutsmart Buffer was added at the 2 hour and 2.5 hour mark.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ligation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Ligation Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P><FONT color="black">
Ligation reactions were set up according to the table below. The vector to insert ratio is stated for each specific reaction in the    <STRONG>Lab Book</STRONG> section.
</FONT></P><TABLE border="1" cellspacing="0" cellpadding="0" class="text"><TBODY><TR><TD width="111" valign="top"><P align="center"><STRONG>Vector (Âµl)</STRONG></P></TD><TD width="111" valign="top"><P align="center"><STRONG>2:1 (Vector: Insert) [Âµl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>3:1 (Vector: Insert) [Âµl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>4:1 (Vector: Insert) [Âµl]</STRONG></P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>T4 DNA Ligase </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Ligase Buffer </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Vector </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Insert</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    -
                </P></TD><TD width="111" valign="top"><P align="center">
                    2
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>H<SUB>2</SUB>0</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    7
                </P></TD><TD width="111" valign="top"><P align="center">
                    5
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD></TR></TBODY></TABLE><P>
    The reactions were incubated at room temperature for at least 3 hours.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Transformation Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>

    1. 100Âµl of competent cells were defrosted. Thaw on ice.
</P><P>
    2. Agar plates were taken out of 4<SUP>o</SUP>C to warm to room temperature.
</P><P>
    3. 1-5Âµl of DNA was added to cells.
</P><P>
    4. Cell and DNA mixture was left on ice for 20 minutes.
</P><P>
    5. Mixture was placed in 42<SUP>o</SUP>C water bath for 90 seconds.
</P><P>
    6. Mixture was put back on ice for 2 minutes.
</P><P>
    7. 1ml of LB (without antibiotic) was added to cells.
</P><P>
    8. Cells were allowed to grow for 1 hour at 37<SUP>o</SUP>C in a shaking incubator.
</P><P>
    9. Cells were then spun down and the supernatant discarded.
</P><P>
    10. The pellet was resuspended in the 100Âµl.
</P><P>
    11. Cells were plated on agar plates with appropriate antibiotic.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="miniprep-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Miniprep Protocol</B></DIV></DIV><DIV class="modal-body"><P>
    1. 1-5ml bacterial overnight culture was pelleted by centrifugation at &gt;8000rpm for 3 minutes at room temperature.
</P><P>
    2. Pelleted bacterial cells were resuspended in 250Âµl of Buffer P1 and transferred to a microcentrifuge tube.
</P><P>
    3. 250Âµl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution becomes clear.
</P><P>
    4. 350Âµl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.
</P><P>
    5. The mixture was centrifuged for 10 minutes at 13000rpm in a microcentrifuge.
</P><P>
    6. The supernatant was applied to a QIAprep spin column by pipetting.
</P><P>
    7. The QIAprep spin column was washed with 500Âµl of Buffer PB by centrifugation for 1 minute and flow through discarded.
</P><P>
    8. 750Âµl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute and flow through discarded.
</P><P>
    9. The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
</P><P>
    10. The QIAprep spin column was placed in a clean 1.5ml microcentrifuge and the DNA eluted by adding 50Âµl of H<SUB>2</SUB>O to the QIAprep spin column and centrifuging for 1 minute.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="onc-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Overnight Culture Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>
    1. Four colonies were picked from the desired plate.
</P><P>
    2. For each colony, 5ml of fresh LB plus the appropriate antibiotics was inoculated and grown overnight at 37<SUP>o</SUP>C.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sds-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>SDS-PAGE and Western Blotting</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>
    1. 50Âµl of an overnight culture was used to inoculate 5ml of fresh LB containing the appropriate antibiotics and left to grow at 37<SUP>o</SUP>C in a
    shaking incubator.
</P><P>
    2. Once the OD<SUB>600</SUB>=0.6-0.8, 1mM of IPTG was added to the culture and left to grow for a further 3 hours.
</P><P>
    3. 1ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.
</P><P>
    4. Meanwhile, 950Âµl of 2x Laemmli sample buffer was added to 50Âµl of B-mercaptoethanol.
</P><P>
    5. The supernatant was discarded from the samples and the pellet resuspended in 200Âµl of the above solution.
</P><P>
    6. The samples were then boiled at 90<SUP>o</SUP>C for 10 minutes.
</P><P>
    7. Afterwards, they were spun down for 1 minute in a microcentrifuge.
</P><P>
    8. At this stage, the samples could be stored at -20<SUP>o</SUP>C or ran on a gel immediately.
</P><P>
    9. Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the
    stacking gel at which point the voltage was increased to 200V for 20 minutes.
</P><P>
    10. One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.
</P><P>
    11. After the proteins had been transferred onto the membrane using a semi dry transfer ran for 40 minutes at 5V, the membrane was blocked in 5% Marvel milk overnight at 4<SUP>o</SUP>C.
</P><P>
    12. The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10ml of 1xTBS buffer to which the primary anti-His antibody was
    added and left for one hour. The protein ladder was cut off before adding the primary antibody.
</P><P>
    13. The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10ml of 1xTBS buffer, to which the secondary antibody was
    added and left for one hour.
</P><P>
    14. The membrane was then washed in 3x10ml 1xTBST buffer for 5 minutes.
</P><P>
    15. The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off
    at step 12.
</P><P>
    16. Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Growth Curve Assay Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>    1. 5ml overnight cultures of the appropriate strains were set up.
</P><P>
    2. The next day, 4x 995Âµl of plain LB containing 1Âµl ampicillin and 1Âµl of chloramphenicol were inoculated with 5Âµl of the overnight culture.
</P><P>
    3. Three were induced with 0.5mM, 1mM and 2mM of IPTG and one was left uninduced.
</P><P>
    4. 200Âµl of each culture was loaded onto a 96 well microtitre plate with 4 repeats.
</P><P>
    5. The cultures were left to grow for 16 hours in the plate reader which took OD readings every 10 minutes.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Growth Curve Assay Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. 5ml overnight cultures of the appropriate strains were set up.
</P><P>
    2. The next day, 4x 995Âµl of plain LB containing 1Âµl ampicillin and 1Âµl of chloramphenicol were inoculated with 5Âµl of the overnight culture.
</P><P>
    3. 20 x 200Âµl aliquots were loaded onto a 96 well microtitre plate.
</P><P>
    4. After the OD<SUB>600</SUB>=~0.6, three cultures were induced with 0.5mM, 1mM and 2mM of IPTG and one was left uninduced (with 4 repeats).
</P><P>
    5. The cultures were left to grow for 20 hours in the plate reader which took OD readings every 30 minutes.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="pp-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Protein Purification Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. A 150ml overnight culture was set up for the desired protein.
</P><P>
    2. 3 x 1L of fresh LB containing 1ml of appropriate antibiotics was inoculated with each 50ml overnight culture and left to grow until an OD<SUB>600</SUB>
    =0.6-1 in a shaking incubator at 37<SUP>o</SUP>C.
</P><P>
    3. The cultures were subsequently induced with 1mM IPTG and left to grow for 6 hours in a shaking incubator at 20<SUP>o</SUP>C.
</P><P>
    4. They were then centrifuged for 20 minutes at 4200rpm at 4<SUP>o</SUP>C.
</P><P>
    5. The pellets were then resuspended in 5ml of ice cold 50mM 7.5 Tris and transferred to a 50ml falcon tube and then topped up to 40ml with 50mM 7.5 Tris.
</P><P>
    6. They were then spun down again at 4000rpm for 15 minutes at 4<SUP>o</SUP>C, the supernatant discarded and pellets stored at -20<SUP>o</SUP>C.
</P><P>
    7. Once purification could proceed, the pellets were thawed on ice and resuspended in 40ml of 50mM Tris containing DNAase, 1 x protease inhibitor tablet
    and lysozyme. (Note: If purifying a membrane protein, 1% DDM was also added).
</P><P>
    8. The cells were lysed using a cell disruptor.
</P><P>
    9. Afterwards the lysate was spun down in a J25.50 rotor at 15000rpm at 4<SUP>o</SUP>C for 40 minutes.
</P><P>
    10. The supernatant was removed and transferred to a 50ml falcon tube. The pellet was kept for testing. Both were kept on ice.
</P><P>
    11. Using FPLC, the supernatant was loaded onto a Ni<SUP>2+</SUP> column.
</P><P>
    12. The protein was eluted using an increasing concentration of imidazole. The two buffers used were Buffer A: 50mM Tris, 200mM NaCl, 20mM imidazole and
    Buffer B: 50mM Tris, 200mM NaCl, 1M Imidazole. (Note: If purifying a membrane protein 0.02% DDM was added to these buffers.)
</P><P>
    13. The fractions corresponding to any peaks were retained and concentrated using an appropriate filter size. A small amount of each fraction could be ran
    on an SDS gel at this stage.
</P><P>
    14. Using FPLC, the sample was then loaded onto a SEC column and any fractions corresponding to a peak were ran on an SDS gel to check for the presence of
    the desired protein.
</P><P>
    15. Fractions containing the desired protein were collated, to which 20% glycerol was added.
</P><P>
    16. The samples were then flash frozen and stored at -80<SUP>o</SUP>C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><I>Figure 1: <B>LbpA PCR. Lane 1: LbpA PCR Lane 2: 1kb Marker. </B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832bp which corresponds to the observed band on the gel.</I></DIV></DIV><DIV class="modal-body">'
           
      </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: LbpA PCR. Lane 1: PCR Product, Lane 2: 1kb Ladder.</B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-body">'
           
      </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: LbpA PCR for Cloning into pQE80-L. Lane 1: LbpA PCR Product, Lane 2: 1kb Ladder</B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-body">'
           
      </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: LbpA Culture Samples. Lane 1: Sample from uninduced culture. Lane 2: Sample from culture induced with 1mM IPTG.</B> It seems that on comparison of the two cultures, that inducing expression of LbpA causes the cells to die given the significant reduction of protein levels visible on the gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: OD<SUB>600</SUB> Readings from Uninduced Culture and Induced Cultures (0.5mM, 1mM and 2mM IPTG).</B> The table shows that after being induced with IPTG the cultures stop growing since their OD<SUB>600</SUB> readings are notably lower than that of the uninduced control. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5mM, 1mM and 2mM IPTG)</B> The graph shows that on comparison with the uninduced control, the induced cultures stop growing when LbpA expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have dealt with the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: Growth Curve of Uninduced and Induced Cultures (0.5mM, 1mM and 2mM IPTG)</B> The graph shows that on comparison with the uninduced control, after reaching an OD<SUB>600</SUB> of 0.6,  the induced cultures stop growing when LbpA starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have dealt with the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: SDS Gel of Samples Taken from Uninduced Control and 1mM IPTG Induced Culture 6 hours After Induction</B> For the induced culture, a faint band just above the 100kDa marker is observable on the gel which corresponds to the expected size of LbpA.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 1: Gel of <I>SBP</I> PCR Product for Cloning into pQE80-L. Lane 1: <I>SBP</I> PCR Product 1 Lane 2: 1kb Ladder Lane 3: <I>SBP</I> PCR Product 2</B> Both bands observable on the gel correspond to the expected size of <I>SBP</I> which is ~600bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">Ã</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: SDS Gel of Uninduced and 1mM IPTG Induced Cultures of PotD and SBP. For PotD, it is clear from the gel that it is being successfully overexpressed since there is an intense band present at the 37kDa marker which corresponds to the expected size of PotD in the induced culture sample. For SBP, there is no band present which suggests SBP is being overexpressed. </B></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">Â Â University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee">Â Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Â About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Â Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Â Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Â Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Â Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>