<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_Lab_Book_Chromate_Detector skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/Lab Book/Chromate Detector</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="modal fade" id="PCR-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>PCR Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the  section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY color="black"><TR><TD width="312" valign="top"><P align="center"><STRONG>Reactants</STRONG></P></TD><TD width="312" valign="top"><P align="center"><STRONG>Volume (µl)</STRONG></P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DNA
                </P></TD><TD width="312" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DMSO
                </P></TD><TD width="312" valign="top"><P align="center">
                    2.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   100 µM Forward Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   100 µM Reverse Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    5 x Herculase Buffer
                </P></TD><TD width="312" valign="top"><P align="center">
                    10
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   10 mM dNTPs
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    H<SUB>2</SUB>O
                </P></TD><TD width="312" valign="top"><P align="center">
                    34.5
                </P></TD></TR></TBODY></TABLE><P>
    Elongation time was altered according to the length of the sequences.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="restrictiondigest-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Restriction Digests Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P><FONT color="#000000">
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the    <STRONG>Lab Book</STRONG> section.
</FONT></P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="170" valign="top" style="color:#000000;"><P align="center"><STRONG>Post Plasmid Purification</STRONG></P></TD><TD width="151" valign="top"><P align="center"><STRONG>Post PCR</STRONG></P></TD><TD width="139" valign="top"><P align="center"><STRONG>gBlock</STRONG></P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>DNA</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1 mg
                </P></TD><TD width="151" valign="top"><P align="center">
                    50 µl
                </P></TD><TD width="139" valign="top"><P align="center">
                    100 ng
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Cutsmart Buffer (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    6
                </P></TD><TD width="151" valign="top"><P align="center">
                    6
                </P></TD><TD width="139" valign="top"><P align="center">     6
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Water (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    As required
                </P></TD><TD width="151" valign="top"><P align="center">
                    1
                </P></TD><TD width="139" valign="top"><P align="center">
                    As required
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Enzyme 1 (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="151" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="139" valign="top"><P align="center">
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Enzyme 2 (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="151" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="139" valign="top"><P align="center">
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG><U>Total Volume (µl)</U></STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    30
                </P></TD><TD width="151" valign="top"><P align="center">
                    60
                </P></TD><TD width="139" valign="top"><P align="center">
                    30
                </P></TD></TR></TBODY></TABLE><P>
    The digests were incubated at 37<SUP>o</SUP>C for 3 hours.
    
    Note that when performing plasmid digestions, 2.5 µl of alkaline phosphatase and 2.5 µl of Cutsmart Buffer was added at the 2 hour and 2.5 hour mark. Digested PCR products were separated from restriction enzymes and buffers using Strataclean resin (Aligent Technologies). Digested vectors were purified on a 1% agarose gel and then extracted. 
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ligation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Ligations</B></CENTER></DIV></DIV><DIV class="modal-body"><P><FONT color="black">
Ligation reactions were set up according to the table below.
</FONT></P><TABLE border="1" cellspacing="0" cellpadding="0" class="text"><TBODY><TR><TD width="111" valign="top"><P align="center"><STRONG>Vector (µl)</STRONG></P></TD><TD width="111" valign="top"><P align="center"><STRONG>2:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>3:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>4:1 (Vector: Insert) [µl]</STRONG></P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>T4 DNA Ligase </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Ligase Buffer </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Vector </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Insert</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    -
                </P></TD><TD width="111" valign="top"><P align="center">
                    2
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>H<SUB>2</SUB>0</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    7
                </P></TD><TD width="111" valign="top"><P align="center">
                    5
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD></TR></TBODY></TABLE><P>
    The reactions were incubated at room temperature for at least 3 hours.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Transformation Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>

    1. 100 µl of chemocompetent <I>E.coli</I> cells were defrosted. Thaw on ice.
</P><P>
    2. Agar plates were taken out of 4 <SUP>o</SUP>C to warm to room temperature.
</P><P>
    3. 10 µl of ligation reaction or 2 µl of plasmid purified DNA was added to cells. The cells and DNA were mixed by gently flicking the tube.
</P><P>
    4. Cell and DNA mixture was left on ice for 20 minutes.
</P><P>
    5. Mixture was placed in 42 <SUP>o</SUP>C water bath for 90 seconds.
</P><P>
    6. Mixture was put back on ice for 2 minutes.
</P><P>
    7. 1 ml of LB (without antibiotic) was added to cells.
</P><P>
    8. Cells were allowed to grow for 1 hour at 37 <SUP>o</SUP>C in a shaking incubator.
</P><P>
    9. Cells were then spun down and the supernatant discarded.
</P><P>
    10. The pellet was resuspended in the 100 µl of remaining media.
</P><P>
    11. Cells were plated on agar plates with appropriate antibiotic.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="miniprep-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Plasmid Purification</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. 1-5 ml bacterial overnight culture was pelleted by centrifugation at &gt;8000 rpm for 3 minutes at room temperature.
</P><P>
    2. Pelleted bacterial cells were resuspended in 250 µl of Buffer P1 and transferred to a microcentrifuge tube.
</P><P>
    3. 250 µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution becomes clear.
</P><P>
    4. 350 µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.
</P><P>
    5. The mixture was centrifuged for 10 minutes at 13000 rpm in a microcentrifuge.
</P><P>
    6. The supernatant was applied to a QIAprep spin column by pipetting.
</P><P>
    7. The QIAprep spin column was washed with 500 µl of Buffer PB by centrifugation for 1 minute and flow through discarded.
</P><P>
    8. 750µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute and flow through discarded.
</P><P>
    9. The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
</P><P>
    10. The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 50 µl of H<SUB>2</SUB>O to the QIAprep spin column and centrifuging for 1 minute.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="onc-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Overnight Cultures </B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>
    1. Four colonies were picked from the desired plate.
</P><P>
    2. For each colony, 5 ml of fresh LB plus the appropriate antibiotics was inoculated and grown overnight at 37 <SUP>o</SUP>C.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sds-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>SDS-PAGE and Western Blotting</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>
    1. 50 µl of an overnight culture was used to inoculate 5 ml of fresh LB containing the appropriate antibiotics and left to grow at 37<SUP>o</SUP>C in a
    shaking incubator.
</P><P>
    2. Once the OD<SUB>600</SUB>=0.6-0.8, 1 mM of IPTG was added to the culture and left to grow for a further 3 hours.
</P><P>
    3. 1 ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.
</P><P>
    4. Meanwhile, 950 µl of 2x Laemmli sample buffer was added to 50 µl of B-mercaptoethanol.
</P><P>
    5. The supernatant was discarded from the samples and the pellet resuspended in 200 µl of the above solution.
</P><P>
    6. The samples were then boiled at 90<SUP>o</SUP>C for 10 minutes.
</P><P>
    7. Afterwards, they were spun down for 1 minute in a microcentrifuge.
</P><P>
    8. At this stage, the samples could be stored at -20<SUP>o</SUP>C or ran on a gel immediately.
</P><P>
    9. Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the
    stacking gel at which point the voltage was increased to 200V for 20 minutes.
</P><P>
    10. One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.
</P><P>
    11. After the proteins had been transferred onto the membrane using a semi dry transfer ran for 40 minutes at 5V, the membrane was blocked in 5% Marvel milk overnight at 4<SUP>o</SUP>C.
</P><P>
    12. The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10 ml of 1xTBS buffer to which the primary anti-His antibody was
    added and left for one hour. The protein ladder was cut off before adding the primary antibody.
</P><P>
    13. The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10 ml of 1xTBS buffer, to which the secondary antibody was
    added and left for one hour.
</P><P>
    14. The membrane was then washed in 3x 10 ml 1xTBST buffer for 5 minutes.
</P><P>
    15. The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off
    at step 12.
</P><P>
    16. Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Growth Curve Assay Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>    1. 5ml overnight cultures of the appropriate strains were set up.
</P><P>
    2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
</P><P>
    3. Three were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced.
</P><P>
    4. 200 µl of each culture was loaded onto a 96 well microtitre plate with 4 repeats.
</P><P>
    5. The cultures were left to grow for 16 hours in the plate reader which took OD readings every 10 minutes.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Growth Curve Assay Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. 5ml overnight cultures of the appropriate strains were set up.
</P><P>
    2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
</P><P>
    3. 20 x 200 µl aliquots were loaded onto a 96 well microtitre plate.
</P><P>
    4. After the OD<SUB>600</SUB>=~0.6, three cultures were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced (with 4 repeats).
</P><P>
    5. The cultures were left to grow for 20 hours in the plate reader which took OD readings every 30 minutes.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="pp-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Protein Purification Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. A 150 ml overnight culture was set up for the desired protein.
</P><P>
    2. 3 x 1 L of fresh LB containing 1 ml of appropriate antibiotics was inoculated with each 50 ml overnight culture and left to grow until an OD<SUB>600</SUB>
    =0.6-1 in a shaking incubator at 37<SUP>o</SUP>C.
</P><P>
    3. The cultures were subsequently induced with 1 mM IPTG and left to grow for 6 hours in a shaking incubator at 20<SUP>o</SUP>C.
</P><P>
    4. They were then centrifuged for 20 minutes at 4200 rpm at 4<SUP>o</SUP>C.
</P><P>
    5. The pellets were then resuspended in 5 ml of ice cold 50 mM 7.5 Tris and transferred to a 50 ml falcon tube and then topped up to 40 ml with 50 mM 7.5 Tris.
</P><P>
    6. They were then spun down again at 4000 rpm for 15 minutes at 4<SUP>o</SUP>C, the supernatant discarded and pellets stored at -20<SUP>o</SUP>C.
</P><P>
    7. Once purification could proceed, the pellets were thawed on ice and resuspended in 40 ml of 50 mM Tris containing DNAase, 1 x protease inhibitor tablet
    and lysozyme. (Note: If purifying a membrane protein, 1% DDM and 5 M urea was also added).
</P><P>
    8. The cells were lysed using a cell disruptor.
</P><P>
    9. Afterwards the lysate was spun down in a J25.50 rotor at 15000 rpm at 4<SUP>o</SUP>C for 40 minutes.
</P><P>
    10. The supernatant was removed and transferred to a 50 ml falcon tube. The pellet was kept for testing. Both were kept on ice.
</P><P>
    11. Using FPLC, the supernatant was loaded onto a Ni<SUP>2+</SUP> column.
</P><P>
    12. The protein was eluted using an increasing concentration of imidazole. The two buffers used were Buffer A: 50 mM Tris, 200 mM NaCl, 20 mM imidazole and
    Buffer B: 50 mM Tris, 200 mM NaCl, 1 M Imidazole. (Note: If purifying a membrane protein 0.02% DDM was added to these buffers.)
</P><P>
    13. The fractions corresponding to any peaks were retained and concentrated using an appropriate filter size. A small amount of each fraction could be ran
    on an SDS gel at this stage.
</P><P>
    14. Using FPLC, the sample was then loaded onto a SEC column and any fractions corresponding to a peak were ran on an SDS gel to check for the presence of
    the desired protein.
</P><P>
    15. Fractions containing the desired protein were collated, to which 20% glycerol was added.
</P><P>
    16. The samples were then flash frozen and stored at -80<SUP>o</SUP>C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ap-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Assay Preparation</B></CENTER></DIV></DIV><DIV class="modal-body"><OL><LI> Take 3 independent colonies from each plate of your controls and samples and place each colony into 5 ml of LB with the appropriate antibiotics. </LI><LI> Leave to grow overnight at 30°C.</LI><LI> Take 50 μl of each overnight and inoculate 5 ml of fresh LB containing the appropriate antibiotics. Do this twice for every independent colony. </LI><LI> Allow these to grow at 30°C shaker until the O.D of each inoculate is between 0.3-0.5</LI><LI> Once the O.D is between 0.3-0.5 take 3 x 1 ml samples of each culture and place into separate Eppendorf tubes.</LI><LI> Take a further 1 ml and place into a cuvette and  subsequently measure the OD<SUB>600</SUB> marking down the OD for each replica. </LI><LI> Spin down the 1 ml cultures of each replica at 13,000 rpm for 3 minutes. </LI><LI> Remove the supernatant from each 1 ml culture and freeze cell pellet for later use in the β-galactosidase Assay. </LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bga-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>β-Galactosidase Assay</B></CENTER></DIV></DIV><DIV class="modal-body"><P><OL><LI> Resuspend cell pellet in 800 μl of buffer 2 (100 mM dibasic sodium phosphate (Na2HPO4); 20 mM KCl ; 2 mM MgSO4 ;0.8 mg/mL CTAB (hexadecyltrimethylammonium bromide) ; 0.4 mg/mL sodiumdeoxycholate) and subsequntly add 5.4 μL/mL beta-mercaptoethanol. </LI><LI> Add 50 μl of SDS (0.1%) to each sample. </LI><LI> Add 50 μl of Chloroform to each sample and vortex for 30 seconds. </LI><LI> Make substrate solution (60 mM Na2HPO4; 40 mM NaH2PO4; 1 mg/mL o-nitrophenyl-β-D-Galactoside (ONPG); 2.7 μL/mL β-mercaptoethanol)</LI><LI> Add 200 μl of substrate solution to each sample marking down the time it was added in relation to the first sample. </LI><LI> Wait for an observable color change and mark down the time that the color change occurred.</LI><LI> Add 600 μl of NaCO3(1M) once the color change has occurred.</LI><LI> Spin down the samples at 13,000 rpm for 3 minutes and place the supernatant into a cuvette. </LI><LI> Measure the OD<SUB>420</SUB>. </LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plate Reader Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>A 5 ml overnight culture of the desired strain was grown at 37 <SUP>o</SUP>C.</LI><LI>Equalise OD<SUB>600</SUB> if required. Dilute culture 1:1000.</LI><LI>Aliquot 198 µl into the required amount of wells in a 96-well plate.</LI><LI>Add additional substances and controls as required.</LI><LI>Preheat platereader to required temperature and start experiment.</LI><LI>Retrieve and analyse data.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="competent-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Competent Cells Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>A 5 ml overnight culture of the desired strain was grown at 37<SUP>o</SUP>C.</LI><LI>Subsample <SUP>1</SUP>/<SUB>10</SUB> of the volume of the subculture. I.e. 1 ml of the overnight culture into 10 ml fresh LB.</LI><LI>Grow subculture at at 37 <SUP>o</SUP>C for 2 h.</LI><LI>Spin down subculture at 4000 rpm at 4<SUP>o</SUP>C for 10 min.</LI><LI>Resuspend the pellet in ice cold transformation buffer. Use <SUP>1</SUP>/<SUB>10</SUB> of the initial volume of the subculture.</LI><LI>Use 100 µl per transformation.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="plating-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plating Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>Fresh agar plates were inoculated with cells on the toothpick.</LI><LI>Agar plates were incubated at 4<SUP>o</SUP>C over night.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="tf-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Tryptophan Fluorescence Protocol</DIV></DIV><DIV class="modal-body"><P>
    1. Intensity of tryptophan fluorescence of PotD was measured on a fluorimeter using an excitation of 280 nm and emission of 350 nm.
</P><P>
    2. Increasing concentrations of spermidine were added in 50 µM increments to give a final concentration of 250 µM spermidine to 5 µM of purified PotD.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sequencing-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Sequencing Protocol</DIV></DIV><DIV class="modal-body"><P>
    For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
      <UL><LI>Volume of sample equivalent to 550ng DNA</LI><LI>2 µl of 3 mM primer</LI><LI>Remaining volume of water for a total volume of 30 µl</LI></UL></P><P>
      The samples were submitted to <A href="https://www.dnaseq.co.uk/" target="_blank">DNA sequencing and services</A>.
    </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="overnightculture-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Overnight Culture Protocol</DIV></DIV><DIV class="modal-body"><P><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>5 ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.</LI><LI>Tubes were incubated at 37<SUP>o</SUP>C in a rotary incubator at 200rpm over night.</LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="gelextraction-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Gel Extraction Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.</LI><LI>The gel slice was transferred into a microcentrifuge tube. 700 µl of Buffer QG was added.</LI><LI>The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.</LI><LI>After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.</LI><LI>230 µl of isopropanol was added to the sample and mixed.</LI><LI>A QIAquick spin column was placed in a 2 ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>500 µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>750 µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>The QIAprep spin column was spun for an additional minute to remove residual wash buffer.</LI><LI>The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: <B>Amplification of <I>LbpA</I></B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: LbpA PCR. Lane 1: PCR Product, Lane 2: 1kb Ladder and Marker.</B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: LbpA PCR for Cloning into pQE80-L. Lane 1: LbpA PCR Product, Lane 2: 1kb Ladder and Marker</B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: LbpA Culture Samples. Lane 1: Sample from uninduced culture. Lane 2: Sample from culture induced with 1mM IPTG.</B> It seems that on comparison of the two cultures, that inducing expression of LbpA causes the cells to die given the significant reduction of protein levels visible on the gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: OD<SUB>600</SUB> Readings from Uninduced Culture and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG).</B> The table shows that after being induced with IPTG the cultures stop growing since their OD<SUB>600</SUB> readings are notably lower than that of the uninduced control. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, the induced cultures stop growing when LbpA expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, after reaching an OD<SUB>600</SUB> of 0.6,  the induced cultures stop growing when LbpA starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have dealt with the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: SDS Gel of Samples Taken from Uninduced Control and 1 mM IPTG Induced Culture 6 hours After Induction</B> For the induced culture, a faint band just above the 100 kDa marker is observable on the gel which corresponds to the expected size of LbpA.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 1: Gel of <I>SBP</I> PCR Product for Cloning into pQE80-L. Lane 1: <I>SBP</I> PCR Product 1 Lane 2: 1kb Ladder and Marker Lane 3: <I>SBP</I> PCR Product 2</B> Both bands observable on the gel correspond to the expected size of <I>SBP</I> which is ~600 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: SDS Gel of Uninduced and 1 mM IPTG Induced Cultures of PotD and SBP.</B> For PotD, it is clear from the gel that it is being successfully overexpressed since there is an intense band present at the 37 kDa marker which corresponds to the expected size of PotD in the induced culture sample. For SBP, there is no band present which suggests SBP is being overexpressed. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Characterisation of PotD.</B> Single colonies of MC1061 <I>E.coli</I> containing pQE80-L encoding <I>PotD</I> were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin. Once cultures reached an OD<SUB>600</SUB> of 0.6, one was induced with 1 mM IPTG and one was left uninduced as a control. They were left to grow for a further three hours, after which a 1ml sample was taken and the cells pelleted. The cells were resuspended in 100µl of Laemmli buffer, 20µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: SDS Gel of Uninduced and 0.5mM, 1mM and 2mM IPTG Induced Cultures of SBP.</B> It appears that upon induction of <I>SBP</I> with any concentration of IPTG the cells are dying given that there is a significant drop in protein levels observable on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, the induced cultures stop growing when SBP expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, after reaching an OD<SUB>600</SUB> of 0.6,  the induced cultures stop growing when <I>SBP</I> starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of PotD</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100 V. The two bands observable on the gel correspond to the expected size of PotD of 38 kDa. Western Blotting will confirm if these proteins are PotD. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: Characterisation of PotD following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure9-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 9: Characterisation of SBP following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure10-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 10: Characterisation of PotD/Spermidine Interaction.</B> The graph shows that the change in the tryptophan spectra increases as concentration of spermidine increases. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 1: pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> (respectively) Pre-Sequencing Digest</B> pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> minipreps were digested with EcoRI and PstI to check for presence of each insert. The bands observable on the gel correspond to the expected sizes for the pSB1C3 vector and both inserts.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: Amplification of <I>hHBA</I> and <I>hHBB</I></B><I>hHBA</I> and <I>hHBB</I> were amplified from the pSB1c3-<I>hHBA</I> and pSB1c3-<I>hHBB</I> minipreps obtained yesterday using primers to allow cloning of the genes into pT25 and pUT18 respectively. Although the band is fainter in the case of <I>hHBA</I>, the gel indicates the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Amplification of <I>hHBA</I></B><I>hHBA</I> was amplified from the pSB1C3-<I>hHBA</I> using primers to allow cloning of the genes into pT25. The gel indicates that the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: Amplification of <I>hHBA</I> and <I>hHBB</I></B><I>hHBA</I> and <I>hHBB</I> were amplified from the pSB1c3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> minipreps using primers to allow cloning of the genes into pQE80-L. The gel indicates that the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Pre-Sequence Digest of pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I></B> Both pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I> minipreps were digested with BamHI and KpnI to check for the presence of the inserts. The gel indicates that both inserts have been successfully cloned into pQE80-L. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Overexpression of <I>hHBB</I> and <I>hHBA</I>.</B> Single colonies of M15[pREP4] <I>E.coli</I> containing pQE80-L encoding <I>hHBB</I> and pQE80-L encoding <I>hHBA</I> were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD<SUB>600</SUB> of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of 2x Laemmli buffer, 20 µl of which was separated by SDS-PAGE. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: Characterisation of hHBB.</B> Single colonies of M15[pREP4] <I>E.coli</I> containing pQE80-L encoding <I>hHBB</I> were used to inoculate 5 ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD<SUB>600</SUB> of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: Amplification of <I>hHBN</I> for Cloning into pQE80-L </B> PCR product from amplification of <I>hHBN</I> from IDT plasmid using primers for cloninig into pQE80-L. The gel indicates <I>hHBN</I> has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure9-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 9: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of hHBB</B> 10 µl of each fraction was mixed with 10 µl of laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. The four bands observable on the gel correspond to the expected size of hHBB of 16 kDa. Western Blotting will confirm if these proteins are hHBB. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure10-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 10: Characterisation of hHBB following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure11-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 11: Amplification of <I>hHBN</I> for Cloning into pT25 </B> PCR product from amplification of <I>hHBN</I> from IDT plasmid using primers for cloninig into pT25. The gel indicates <I>hHBN</I> has been amplified successfully since the observed band corresponds to the expected size of 1200 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure14-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 14: Characterisation of hHBN following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure15-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 15: Amplification of <I>hHBN</I></B><I>hHBN</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of <I>hHBN</I> of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure16-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 15: Amplification of <I>hHBA</I></B><I>hHBA</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of <I>hHBA</I> of </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure17-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 16: Amplification of <I>hHBN</I></B><I>hHBN</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pUT18. The band observable on the gel corresponds to the expected size of <I>hHBN</I> of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 1: Amplification of <I>OBP2A</I> for insertion into pSB1C3 vector </B>. This is the PCR product from the amplification of <I>OBP2A</I> from the IDT plasmid using primers for cloning into pSB1C3. The gel indicates <I>OBP2A</I> has been amplified successfully since the observed band corresponds to the expected size of 500 bp. The concentration of the gel extracted <I>OBP2A</I> is 335.53 ng/ul. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 2: A subsequent gel was run of the amplified <I>OBP2A</I> after it was first excised from the gel in <I>Figure 1 </I></B> This was to determine if the gel extraction was successful. The gel indicates that <I>OBP2A</I> has been extracted from the gel successfully, since the observed band corresponds to the expected size of 500 bp. This can now be restricted in preparation for ligation into pSB1C3. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 3: A Gel Analysis of Restriction Digest from the 24/6.</B> This gel analysis was done to determine if the reason for the failed transformation was an improperly digested <I>OBP2A</I>. This gel indicates that the restricted product hasn't been lost as the <I>OBP2A</I> gene fragment can be seen as a distinct band at 500 bp. A re-ligation of <I>OBP2A</I> into the pSB1C3 backbone will now occur. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 4: Presequence Digest of pSB1C3-<I>OBP2A</I></B>. All seven plasmid purifications of the overnight cultures done on the 26/6 were digested to see if they contained the <I>OBP2A</I> insert. It is clear from the gel that all but sample 3 contain the insert for <I>OBP2A</I> as they all have faint bands at 500 bp's which corresponds to the size of <I>OBP2A</I>. It also appears that sample 5 contains the highest concentration, so that will be the sample that is sent for sequencing.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 5: Presequence digest of pUT18-<I>OBP2A</I></B> To determine if any samples contain the <I>OBP2A</I> insert. It can be seen from this gel that of the  5 samples that were digested only two are show to have the <I>OBP2A</I> insert, therefore sample 2 will be sent for sequencing.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 6: Amplification of <I>OBP2A</I> for insertion into pT25 vector </B>. This is the PCR product from the amplification of <I>OBP2A</I> for cloning into pT25. The gel indicates <I>OBP2A</I> has been amplified successfully since the observed band corresponds to the expected size of 100 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 7: Presequence Digest of pT25-<I>OBP2A</I></B> In order to determine if any of the samples contain the <I>OBP2A</I> insert. It can be seen from this gel that samples 5, 6 and 7 may have inserts in them as they each have two distinct bands, one of which corresponds to the 100 bp size for the <I>OBP2A</I> for pT25. As a result all three samples will be sent for sequencing.    </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 8: β-galactosidase Assay</B>  A graph of the results from the first attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>. As it can be seen that the positive result has failed(far left) not much can be inferred from the interaction of the two parts of <I>OBP2A</I>. However it looks like the two subunits of <I>OBP2A</I> may not be interacting. As this was a first attempt, subsequent assays will be performed in order to determine if in fact the two separated parts of OBP2A aren't interacting in vivo - like these results suggests.      </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure11-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 9: β-galactosidase Assay</B> A graph of the results from the second attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>.  It can also be inferred from this graph that the two parts of <I>OBP2A</I> don't seem to be interacting as it has a low millers activity when compared to the controls. Further assays will be performed in order to determine if the two separated parts of <I>OBP2A</I> aren't interacting in vivo like these results suggests.     </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure12-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 11: β-galactosidase Assay Under Anaerobic Conditions </B> A graph of the results from the anaerobic β-galactosidase assay aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>. The graph seems to suggest that the two subunits of <I>OBP2A</I> may not be interacting in vivo. This can be inferred from the fact that the millers activity of the <I>OBP2A</I> sample lies lower than the negative controls from within the experiment. As a result of this information, one more assay will be prepared in which the two subunits will be switched with regards to the bacterial two hybrid vectors to eliminate any possibility of this being a factor effecting the interaction between the two subunits.     </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: <B>Amplification of <I>LS</I></B><I>LS</I> was amplified using primers which would allow for subsequent cloning into the high expression vector pQE80-L. Lane 1 shows that the PCR has been successful since the band corresponds to the expected size of 2100 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: Amplification of <I>LS</I></B><I>LS</I>  was amplified from the IDT plasmid using primers to allow cloning of the gene into pSB1C3. The observed band corresponds to the expected size of LS, 2100 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Overexpression Assays of <I>LS</I></B> 5 ml cultures of M15pREP4 <I>E.coli</I> was grown to OD<SUB>600</SUB>=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37<SUP>o</SUP>C, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: Characterisation of <I>Lanosterol Synthase</I></B> 5 ml cultures of M15pREP4 <I>E.coli</I> was grown to OD<SUB>600</SUB>=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37<SUP>o</SUP>C, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. The proteins were transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Chromatogram following Affinity Chromatography of LS</B> Using FPLC, the protein sample was loaded onto a nickel column and eluted with an increasing concentration of imidazole (green line). The fractions corresponding to the peak were ran on an SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: SDS Gel of Peak Fractions </B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer an loaded onto a SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="BBa_K1058008-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: Alignment of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>. Sequences from top to bottom: sequencing result from forward primer; expected sequence; reverse complement of sequencing result from reverse primer; sequencing result from intermediate primer. Colour code: Green = Prefix/Suffix; Pink = Annealing site of intermediate primer; Yellow = Mismatch in one sequence; Red = In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem022-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 2: 1% agarose gel after PCR of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with primers specific to the promoter region, P<I>chr</I>, and the repressor region, <I>chrB</I>. Left: P<I>chr</I> (expected size 169bp); Middle: 1kb+ ladder, Right: <I>chrB</I> (expected size 939bp). Amplification of <I>chrB</I> was not successful. P<I>chr</I> was further processed by restriction digest with EcoRI and PstI.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem023-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 3: 1% agarose gel after PCR of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with primers specific to the repressor region, <I>chrB</I>. Left: 1.5 µl DMSO; Middle: 1kb+ ladder, Right: 2.5 µl DMSO. Expected size for each: 939bp. The annealing temperature for the PCR amplification was lowered to 50°C in order to increase the chances of accomodating a primer that is not 100% complementary.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem024-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 4: 1% agarose gel after PCR of <I>gfp</I> with corresponding primers. Left: <I>GFP</I> (expected size 717bp); Middle: 1kb+ ladder, Right: PCR product of <I>chrB</I>, using primers to add a BamHI restriction site in front of it and a 6-His-Tag and a HindIII restriction site at the end. (expected size of <I>chrB</I>: 939bp). Since amplification of <I>chrB</I> was unsuccessful, it will be repeated.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem026-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 5: 1% agarose gel after restriction digest of pSB1C3-P<I>chr</I>, pSB1C3-<I>chrB</I> (s), and pSB1C3-<I>chrB</I> (w) with EcoRI and PstI for confirmation of size. Ladder: 1kb+. Top: 1-4: P<I>chr</I> 1-4; <I>chrB</I> (s) 1-2. Bottom: <I>chrB</I> (s) 3-4; <I>chrB</I> (w) 1-4. Expected sizes: P<I>chr</I>: 169bp; <I>chrB</I>: 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-pChr1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 6: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green = Prefix/Suffix
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-ChrBw4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 7: Alignment of <I>chrB</I> (opt) sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green: Prefix and Suffix; Red: In frame stop codons; Blue: First 15 codons optimised for expression in E. coli.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem034-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 8: 1% agarose gel after restriction digest of pUniprom (Expected size 2454bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem035-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 11: 1% agarose gel after restriction digest of pSB1C3 with EcoRI and SpeI (Expected size 2070bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem032-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 9: 1% agarose gel after PCR of <I>chrB</I> (opt). (Expected size 939bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-first-transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 10: Plates after transformation of Top: pUniprom<I>chrB</I>, Middle: pUniprom<I>chrB</I> (opt) and Bottom: pSB1C3-P<I>chr-gfp</I> into JM110.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem036-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 12: 1% agarose gel after restriction digest of pUniprom<I>chrB</I>, pUniprom<I>chrB</I> (opt) and pSB1C3-P<I>chr-gfp</I> for confirmation of size. Ladder: 1kb+. Top: 1-4: pSB1C3-P<I>chr-gfp</I> 1-4; pUniprom<I>chrB</I> (opt) 1-2. Bottom: pUniprom<I>chrB</I> (opt) 3-4; pUniprom<I>chrB</I> 1-4. Expected sizes: P<I>chr-gfp</I>: 909bp; <I>chrB</I> and <I>chrB</I> (opt): 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem037-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 13: 1% agarose gel after colony PCR of pUniprom<I>chrB</I> (opt). Top: colonies 1-11; Bottom: colonies 12-22. Colonies 2, 4, 9, and 12 were selected for sequencing. Ladder: 1kb+ Expected size of <I>chrB</I> (opt): 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pSB1C3-pChr-GFP_reverse" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 14: Sequencing result of pSB1C3-P<I>chr-gfp</I> after double ligation. Even though the sequence is as expected, it is ligated in reverse. It was hence called pSB1C3-P<I>chr-gfp</I> (A) The Result for pSB1C3-P<I>chr-gfp</I> (colony 2) was not as expected, albeit in the correct orientation. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pChrGFPplates_revisited" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 15: Plates with pSB1C3-P<I>chr-gfp</I> in JM110 after double ligation under UV light. Numbered colonies were selected for initial size confirmation and sequencing. However, other colonies seemed to express GFP as well, hence a second attempt was made with different colonies from the same plate.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pChrGFP_revisited_seq" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 16: Alignment of sequencing results of pSB1C3-P<I>chr-gfp</I> - colonies 1, 5, 6, and 7 with sequences of <I>gfp</I> (mut2) and P<I>chr</I> sequence. Colony 1 remained consistent, the other tested colonies were without success. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem039" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 17: 1% Agarose gel after restriction digest of pUniprom with BamHI/PstI (expected size: 2454bp). Right: PCR product of Haptoglobin.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150723_pSB1C3-pChr" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 18: Plates after transformation of P<I>chr</I> into pSB1C3. Top: ratio insert:vector 2:1; Bottom: ratio insert:vector 3:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150727_pSB1C3-pChr_alignment" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 19: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green = Prefix/Suffix
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150728_pUniprom-ChrB" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 20: Plates after transformation of, Top: <I>chrB</I> into pUniprom; Bottom: <I>chrB</I> (opt) into pUniprom. Ratio insert:vector for ligations: left: 3:1; Right: 4:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem040" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 21: Gel after restriction digest of Left: pUniprom-<I>chrB</I>, and Right: pUniprom-<I>chrB</I> (opt) with BamHI/PstI for confirmation of insert size. Expected size 939bp. <I>chrB</I> colony 2 and <I>chrB</I> (opt) colony 3 were selected for sequencing.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150729_chrb-combined" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 22: Alignments of: Left: <I>chrB</I> and Right: <I>chrB</I> (opt). Colours: Pink = Restriction sites (BamHI/PstI); Blue = 6-His Tag; Red = In-frame stop codons; Cyan = optimised codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150731_pChrGFP-ligations" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 23: Plates after ligation of <I>gfp</I> into pSB1C3-P<I>chr</I>. Left: Ratio Insert:Vector 2:1; Right: Ratio Insert:Vector 3:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem043" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 24: Gel for size confirmation of pSB1C3-P<I>chr-gfp</I> (Top). Bottom: Left - Haptoglobin, Right - Lanosterol Synthase. pSB1C3-P<I>chr-gfp</I> (3) and pSB1C3-P<I>chr-gfp</I> (4) were submitted for sequencing.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150801_pchrgfp" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 25: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> and <I>gfp</I> (mut2) with the forward and reverse sequencing results of P<I>chr-gfp</I> this region. The sequence was as expected, it was hence called pSB1C3-P<I>chr-gfp</I> (B). Colour code: Green = Prefix/Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150809_blot" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 26: Top: anti-GFP: Single colonies of JM110 + pSB1C3-P<I>chr-gfp</I> (A), MC1061 + pSB1C3-P<I>chr-gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> chloramphenicol. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of each sample was mixed with 100 µl laemmli buffer, and boiled for 10min. 3 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and P<I>manA-gfp</I> as a positive control. 
        Bottom: anti-6-His - ChrB: Single colonies of JM110 pUniprom-<I>chrB</I> and JM110 pUniprom-<I>chrB</I> (opt) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 200 µl laemmli buffer, and the sample boiled for 10min. 10 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and <I>hydA - 6-His</I> as a positive control.
        Expected sizes: GFP ~ 27kDa, ChrB ~35kDa.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150811_transformations" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 27: Plates after transformation of single constituents of chromate sensor into <I>E. coli</I> MG1655. From top to bottom: pUniprom-<I>chrB</I> (from colony 2 - 29/07), pUniprom-<I>chrB</I> (opt) (from colony 3 - 29/07), pSB1C3-P<I>chr-gfp</I> (B), pSB1C3-P<I>chr-gfp</I> (A), <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150822_blot" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 28: Western blot against GFP: Single colonies of MG1655, containing all 4 combinations, pUniprom-<I>chrB</I> - pSB1C3-P<I>chr-gfp</I> (A), pUniprom-<I>chrB</I>- pSB1C3-P<I>chr-gfp</I> (B), pUniprom-<I>chrB</I> (opt) - pSB1C3-P<I>chr-gfp</I> (A), and pUniprom-<I>chrB</I> (opt) - pSB1C3-P<I>chr-gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> chloramphenicol and 100 <SUP>µg</SUP>/<SUB>ml</SUB> ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml PBS. 100 µl of these samples was mixed with 100 µl laemmli buffer, and the sample boiled for 10 min. The indicated volume of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3, and pUniprom were included as a negative control, and P<I>manA-gfp</I> as a positive control. The blot against 6-His – ChrB was without success and is not shown. No chromate of any kind was added to the medium previous to this blot, hence no expression of GFP was expected. 
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV>


K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB><DIV class="modal fade" id="dundee15_chr_150824_platereader_BBa_K1058008" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 29: F,OD-plot of results of plate reader experiment after exposing MG1655 + BBa_K1058008 to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="icecream" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 30: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="potato" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 31: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt) to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150824_platereader_pchrbgfp" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 32: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_chrbbielefeld" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 33: Plates after ligation of <I>chrB</I> (Bielefeld) into pT7KS. From top to bottom: Ratio insert:vector 2:1; Ratio Insert:Vector 3:1; pT7KS religation control.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_bl21" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 34: Plates after transformation of single constituents of chromate sensor into <I>BL21 (DE3)</I>. From top to bottom: pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt), pUniprom-<I>chrB</I>, and pUniprom-<I>chrB</I> (opt).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_igem047" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 35: 1% agarose gel after colony PCR of pT7KS-<I>chrB</I> (Bielefeld). Top: colonies 1-6; Bottom: colonies 7-12. Ladder: 1kb+ Expected size of <I>chrB</I> (Bielefeld): 939bp
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_platereaderinduced" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 36: F,OD-plot of results of plate reader experiment after inducing expression of ChrB from the T7 promoter of pUniprom <I>BL21 (DE3)</I> pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, and pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt). IPTG was added after 4 hours. One measurement was made every 10min across 22h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV><H1><HIGHLIGHT class="highlight">Chromate Biosensor</HIGHLIGHT></H1><H3><HIGHLIGHT class="highlight">Lab Book</HIGHLIGHT></H3><SECTION id="about" class="row1"><DIV class="row"><DIV class="col-lg-4"><H3>FluID</H3><P class="about-content">FluID is a cell free, all in one spray which will be used to detect and dinstiguish between body fluids found at crime scenes.</P></DIV><DIV class="col-lg-4"><H3>Chromate Biosensor</H3><P class="about-content">Our chromate biosensor is a device that will be used to detect stainless steel residues left behind on bones in blunt or sharp force trauma cases.</P></DIV><DIV class="col-lg-4"><H3>Fingerprint Ageing</H3><P class="about-content">This device will search for the presence of a specifc component found in fingerprint deposits whose presence/absence will indicate approximate fingerprint age.</P></DIV></DIV></SECTION><DIV class="ribbon"><DIV class="rows"><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 22/06/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">This week was all about getting started with the newly arrived biobrick <A href="http://parts.igem.org/Part:BBa_K1058008">BBa_K1058008.</A></P></DIV><DIV id="collapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - 22/06: Plated <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> from agar stab.</SPAN><DIV id="collapsiblew1-1" class="collapse box-content"><P><B>Aim of experiment:</B> To prepare <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> for sequence confirmation, storage, and further processing. The culture was plated from the agar stab and also a liquid culture set up.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#plating-modal" class="journal-protocol"> Plating, </A><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of the Plasmid, containing <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> and confirmation of size and sequence.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 2 - 23/06: Purified <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> from liquid culture.</SPAN><DIV id="collapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To purify <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> for sequence confirmation.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep plasmid purification, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol"> DNA sequencing</A></P><P><B>Results Sequencing: </B><A data-toggle="modal" data-target="#BBa_K1058008-figure1-modal" class="journal-protocol"> Figure 1 </A></P><P><B>Next Steps:</B> PCR of promoter region (<I>pChr</I>) and repressor region (<I>ChrB</I>) of <A href="http://parts.igem.org/Part:BBa_K1058008">BBa_K1058008</A> for storage in individual biobricks.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 29/06</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> During this week, <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> was disassembled into its constituent parts, <I>pChr</I>, and <I>ChrB</I>.</P></DIV><DIV id="collapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - 29/06: PCR-amplification of <I>pChr</I> and <I>ChrB</I> from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>.</SPAN><DIV id="collapsiblew2-1" class="collapse box-content"><P><B>Aim of experiment: </B> To break down <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> into the promoter region <I>pChr</I>, and the repressor gene, <I>ChrB</I>, while adding standard prefix and suffix to each. Furthermore optimisation of first 15 codons of <I>ChrB</I> in order to increase the overall rate of transcription. Restriction digest of the respective ends and subsequent ligation of <I>pChr</I> and <I>ChrB</I> into pSB1C3 for submission to registry.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR, </A><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol"> Gel extraction, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction Digest</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15-chr-igem022-modal" class="journal-protocol"> Figure 2</A></P><P><B>Next Steps:</B> Repeat of PCR of <I>ChrB</I>. Purification of the Plasmid, containing <A href="http://parts.igem.org/Part:BBa_K1058008">BBa_K1058008</A> and confirmation of size and sequence.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 06/07</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">During this week ligations of <I>pChr</I> and <I>ChrB</I> into pSB1C3 were finalised and the constituent parts of our newly designed chromate sensing system were prepared for ligation and transformation.</P></DIV><DIV id="collapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - 06/07: Repeated amplification of <I>ChrB</I>. Ligation and transformation of <I>pChr</I> and <I>ChrB</I>.</SPAN><DIV id="collapsiblew3-1" class="collapse box-content"><P><B>Aim of experiment:</B> To repeat PCR of ChrB for subsequent ligation into pSB1C3. Ligation of <I>pChr</I> and <I>ChrB</I> into pSB1C3. Transformation of each into a non-pathogenic lab strain of <I>Escherichia coli</I>.
                  Overnight culture of <I>E. coli DH5-alpha</I>, containing GFPmut2, as a fluorescent reporter for our chromate sensing system</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR-amplification, </A><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol"> Gel extraction, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction digest, </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligation, </A><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15-chr-igem023-modal" class="journal-protocol"> Figure 3 </A></P><P>2 bands were extracted from the gel for further characterisation, the strongest band, hence called <I>ChrBs</I>, and the band directly above, hence called <I>ChrBw</I>. <I>pChr</I>, <I>ChrBs</I>, and <I>ChrBw</I> were ligated into pSB1C3.
                  pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw were transformed into <I>E. coli MC1061.</I></P><P><B>Next Steps:</B> Sequence and size confirmation of pChr-pSB1C3, ChrBs-pSB1C3, and ChrBw-pSB1C3. Ligation of <I>pChr</I> to <I>GFP coding sequence</I> for construction of a reporter plasmid.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 2 - 07/07: Purification of <I>GFP</I> and overnight cultures of pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw</SPAN><DIV id="collapsiblew3-2" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of GFPmut2 from overnight cultures. Overnight cultures of recombinant <I>E. coli MC1061</I>, containing pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw respectively for further size and sequence confirmation. Overnight culture of pUniprom for subsequent ligation of ChrB</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep, </A><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR-amplification, </A><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol"> Gel extraction, </A><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15-chr-igem024-modal" class="journal-protocol"> Figure 5 </A></P><P><B>Next Steps:</B> Repeat of amplification of <I>ChrB</I>. Sequence and size confirmation of pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 3 - 08/07: Purification of plasmids pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw. Size and sequence confirmation.</SPAN><DIV id="collapsiblew3-3" class="collapse box-content"><P><B>Aim of experiment:</B> To purify overnight cultures for size and sequence confirmation. Repeat of PCR amplification of <I>pChr</I> and <I>ChrB</I>. Restriction digestes of all parts in order to produce compatible sticky ends.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep, </A><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR amplification, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing</A></P><P><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction digest</A> of gel purified PCR-products of pChr, ChrB, and GFPmut2, using EcoRI/SpeI, BamHI/HindIII, and XbaI/PstI respecitvely.</P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15-chr-igem026-modal" class="journal-protocol"> Figure 6 </A> - Size confirmation of <I>pChr</I> and <I>ChrB</I>.</P><P><A data-toggle="modal" data-target="#dundee15-chr-pChr1-modal" class="journal-protocol"> Figure 7 </A> - Sequence confirmation of <I>pChr</I></P><P><A data-toggle="modal" data-target="#dundee15-chr-ChrBw4-modal" class="journal-protocol"> Figure 8 </A> - Sequence confirmation of <I>ChrB</I>.</P><P><B>Next Steps:</B> Purification and digest of pUniprom for insertion of <I>ChrB</I>.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 4 - 09/07: Digested and cleaned up pUniprom for insertion of <I>ChrB</I>. Amplification of optimised <I>ChrB</I> with primers specific for the insertion into pUniprom.</SPAN><DIV id="collapsiblew3-4" class="collapse box-content"><P><B>Aim of experiment:</B> To prepare pUniprom for insertion of <I>ChrB</I> by digesting it with BamHI and HindIII. To amplify <I>ChrB</I> version with first 15 codons optimised for insertion into pUniprom.</P><P><B>Protocols Used:</B></P><P><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest</A> and
                    <A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol">Gel extraction</A> of pUniprom</P><P><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A> and
                    <A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol"> Gel extraction </A> of optimised <I>ChrB</I></P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15-chr-igem034-modal" class="journal-protocol"> Figure 9 </A> Gel with pUniprom</P><P><A data-toggle="modal" data-target="#dundee15-chr-igem032-modal" class="journal-protocol"> Figure 10 </A>Gel with ChrB</P><P><B>Next Steps:</B> Ligation of <I>ChrB</I> and codon optimised <I>ChrB</I> into pUniprom.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 5 - 10/07: Ligations of <I>ChrB</I> and optimised <I>ChrB</I> into pUniprom, and of <I>pChr</I> and <I>GFPmut2</I> into pSB1C3.</SPAN><DIV id="collapsiblew3-5" class="collapse box-content"><P><B>Aim of experiment:</B> Ligation of <I>ChrB</I> and optimised <I>ChrB</I> into pUniprom via BamHI and HindIII restriction sites. Ligation of <I>pChr</I> and <I>GFPmut2</I> to each other via SpeI/XbaI-link and into pSB1C3 via EcoRI and PstI restriction sites.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligations</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Transformation of abovementioned ligations.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 13/07</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">During this week successful transformants were identified and propagated.</P></DIV><DIV id="collapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - 13/07: Transformation of pUniprom-ChrB, pUniprom-ChrBopt, and pSB1C3-pChr-GFP.</SPAN><DIV id="collapsiblew4-1" class="collapse box-content"><P><B>Aim of experiment:</B> Transformation of pUniprom-ChrB, pUniprom-ChrBopt, and pSB1C3-pChr-GFP into <I>E. coli JM110</I> chassis. Setting up a backup restriction digest of pSB1C3 with EcoRI and SpeI in case the double ligation pSB1C3-pChr-GFP is unsuccessful.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformations, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15-chr-first-transformation-modal" class="journal-protocol"> Figure 11 </A></P><P><B>Next Steps:</B> Purification of recombinant vectors and confirmation of size and sequence. Store digested pSB1C3 at -20<SUP>o</SUP>C.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 2 - 14/07: Set up cultures for subsequent purification of recombinant vectors for size and sequence confirmation.</SPAN><DIV id="collapsiblew4-2" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of recombinant vectors, pUniprom-ChrB, pUniprom-ChrBopt, and pSB1C3-pChr-GFP, for size and sequence confirmation. Patch plating of the same colonies for storage and retrieval once size and sequence have been confirmed.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures, </A><A data-toggle="modal" data-target="#plating-modal" class="journal-protocol"> Patch plating</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of the recombinant plasmids for confirmation of size and sequence.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 3 - 15/07: Purified pUniprom-ChrB, pUniprom-ChrBopt, and pSB1C3-pChr-GFP from overnight cultures</SPAN><DIV id="collapsiblew4-3" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of pUniprom-ChrB, pUniprom-ChrBopt, and pSB1C3-pChr-GFP from overnight for confirmation of size and sequence. Restriction digest of pUnipromChrB and optimised pUniprom-ChrB optimised with BamHI and HindIII and of pSB1C3-pChr-GFP with EcoRI/PstI for size confirmation. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Miniprep, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#colony-modal" class="journal-protocol">Colony PCR, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing </A></P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15-chr-igem036-modal" class="journal-protocol"> Figure 12 </A> Size confirmation of <I>ChrB</I> and <I>ChrBopt</I>.</P><A data-toggle="modal" data-target="#dundee15-chr-igem037-modal" class="journal-protocol"> Figure 13 </A> Gel after colony PCR. Colonies 2, 4, 9, and 12 were selected for sequencing.<A data-toggle="modal" data-target="#dundee15_chr_pSB1C3-pChr-GFP_reverse" class="journal-protocol"> Figure 14 </A> Sequencing result of pSB1C3-pChr-GFPmut2_1. The Result for pSB1C3-pChr-GFPmut2_2 was not as expected, albeit in the correct orientation.<P><B>Next Steps:</B> Confirmation of sequence of <I>ChrB-opt.</I></P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 4 - 16/07: Set up overnight cultures of colonies chosen after colony-PCR for subsequent purification and sequence confirmation.</SPAN><DIV id="collapsiblew4-4" class="collapse box-content"><P><B>Aim of experiment:</B> Set up overnight cultures of <I>ChrBopt</I> colonies 2, 4, 9, and 12 for sequencing.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of recombinant plasmids.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 5 - 17/07: Purified and sequenced chosen <I>ChrBopt</I>-colonies.</SPAN><DIV id="collapsiblew4-5" class="collapse box-content"><P><B>Aim of experiment:</B> To purify recombinant plasmids for sequence confirmation.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing</A></P><P><B>Results:</B><I>ChrBopt</I> showed HindIII restriction sites and was hence cleaved into partial inserts.</P><P><B>Next Steps:</B> Use alternative cloning protocol for <I>ChrB</I> and <I>ChrBopt</I>.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 20/07</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">During this week an alternative cloning strategy for <I>ChrB</I> and <I>ChrBopt</I> was employed. Furthermore, the pSB1C3-pChr-GFP construct was cloned in a stepwise fashion.</P></DIV><DIV id="collapsiblew5" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - 20/07: Went back to the original plate of pSB1C3-pChr-GFP for visualising colonies that do or do not express GFP. </SPAN><DIV id="collapsiblew5-1" class="collapse box-content"><P><B>Aim of experiment:</B>To compare colonies on the patch plate of pSB1C3-pChr-GFP by differentially visualising colonies that do or do not express GFP under UV light. The background of this experiment was that if there is one colony that has both inserts in the correct orientation with respect to each other, then there is perhaps one that also has them with the correct orientation with respect to the plasmid. Colonies identified as expressing GFP were patch plated and grown in liquid cultures over night in order to confirm the sequence of the insert.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#plating-modal" class="journal-protocol"> Patch-plating, </A><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> overnight cultures</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_pChrGFPplates_revisited" class="journal-protocol"> Figure 15</A> pSB1C3-pChr-GFP in <I>E. coli JM110</I>.</P><P><B>Next Steps:</B> Sequence confirmation of newly selected colonies.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 2 - 21/07: Purified plasmids from new colonies and confirmed sequence. Restriction digest of pUniprom with BamHI and PstI.</SPAN><DIV id="collapsiblew5-2" class="collapse box-content"><P><B>Aim of experiment:</B>Purification of newly selected colonies, 1, 5, 6, and 7, containing pSB1C3-pChr-GFP for subsequent sequence confirmation. Restriction digest of pUniprom with BamHI and PstI to employ an alternative cloning strategy of <I>ChrB</I> and <I>ChrBopt</I>.</P><P><B>Protocols Used:</B></P><P><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep </A> and
                    <A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing </A> of pChrGFP insert.</P><P><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest</A> of pUniprom.
                  </P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15_chr_pChrGFP_revisited_seq" class="journal-protocol"> Figure 16 </A> Alignment of sequencing results.</P><P><A data-toggle="modal" data-target="#dundee15_chr_igem039" class="journal-protocol"> Figure 17 </A> Gel after restriction digest of pUniprom.</P><P><B>Next Steps:</B>Stepwise ligation of <I>pChr</I> and <I>GFP</I> into pSB1C3 in order to clone the inserts in the correct orientation with respect to the plasmid. Insertion of <I>ChrB</I> and <I>ChrBopt</I> into pUniprom via BamHI and PstI links.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 3 - 22/07: Ligated <I>pChr</I> into pSB1C3 via EcoRI and SpeI sites and cloned the recombinant plasmid into <I>E. coli JM110</I>.</SPAN><DIV id="collapsiblew5-3" class="collapse box-content"><P><B>Aim of experiment:</B> Ligation of <I>pChr</I> into pSB1C3 via EcoRI SpeI link. This is the first step of a 2-step cloning strategy with the aim of cloning <I>pChr</I> and <I>GFP</I> into pSB1C3 in the correct orientation with respect to the plasmid.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligation, </A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150723_pSB1C3-pChr" class="journal-protocol"> Figure 18 </A> Plates after transformation of pChr into pSB1C3.</P><P><B>Next Steps:</B> Sequence confirmation of pChr in pSB1C3.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 4 - 23/07: Set up overnight cultures of pSB1C3-pChr colonies.</SPAN><DIV id="collapsiblew1-4" class="collapse box-content"><P><B>Aim of experiment:</B> Set up overnight cultures of pSB1C3-pChr colonies to prepare pSB1C3-pChr for sequence confirmation</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol">Overnight culture, </A><A data-toggle="modal" data-target="#plating-modal" class="journal-protocol">Patch-plating</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of the recombinant vector for size and sequence confirmation.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 5 - 24/07: Purified pSB1C3-pChr for sequence confirmation.</SPAN><DIV id="collapsiblew5-5" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of pSB1C3-pChr for sequence confirmation before ligating <I>GFP</I> into it.</P><P><B>Protocols Used:</B></P><P><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid purification, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150727_pSB1C3-pChr_alignment" class="journal-protocol"> Figure 19 </A> Alignment of sequenced pSB1C3-pChr vectors with pChr sequence.</P><P><B>Next Steps:</B> Digest of pSB1C3-pChr with SpeI/PstI and ligation of GFPmut2.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 6 - 25/07: PCR-amplified <I>ChrB</I> and <I>ChrBopt</I>.</SPAN><DIV id="collapsiblew5-6" class="collapse box-content"><P><B>Aim of experiment:</B> Amplification of <I>ChrB</I> and <I>ChrBopt</I> for insertion into pUniprom. Restriction digest of the PCR product with BamHI and PstI for producing compatible sticky ends</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR-amplification, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction digest</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Ligation and Transformation of <I>ChrB</I> and <I>ChrBopt</I>.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 7 - 26/07: Ligated <I>ChrB</I> and <I>ChrBopt</I> into pUniprom.</SPAN><DIV id="collapsiblew5-7" class="collapse box-content"><P><B>Aim of experiment:</B> Insertion of ChrB and optimised ChrB into pUniprom vector.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligation</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Transformation of ligated vectors.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 27/07</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Final touches were made on the chromate system. All parts were ligated into their final vectors during this week. Preparations were made for characterisation of each of the parts.</P></DIV><DIV id="collapsiblew6" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - 27/07: Transformed <I>ChrB</I> and <I>ChrBopt</I> into <I>E. coli JM110</I></SPAN><DIV id="collapsiblew6-1" class="collapse box-content"><P><B>Aim of experiment:</B>Transformation of the repressors, <I>ChrB</I> and <I>ChrBopt</I>, into <I>JM110</I></P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150728_pUniprom-ChrB" class="journal-protocol"> Figure 20</A> Plates after transformation.</P><P><B>Next Steps:</B> Purification of pUniprom-ChrB and pUniprom-ChrBopt for sequence confirmation.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 2 - 28/07: Digested pSB1C3-pChr for insertion of <I>GFP</I></SPAN><DIV id="collapsiblew6-2" class="collapse box-content"><P><B>Aim of experiment:</B> Digesting sequence confirmed pSB1C3-pChr with SpeI and PstI for subsequent ligation with <I>GFP</I>. Overnight cultures of <I>ChrB</I> and <I>ChrBopt</I> for subsequent plasmid purification, size confirmation, and sequence confirmation.</P><P><B>Protocols Used:</B></P><P><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation</A> of <I>GFP</I> into pSB1C3-pChr.</P><P><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol">Overnight culture</A> of of <I>ChrB</I> and <I>ChrBopt</I>.</P><P><B>Next Steps:</B> Transformation of recombinant vector for confirmation of size and sequence.</P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Transformation of pSB1C3-pChr-GFP for size and sequence confirmation. Purification of <I>ChrB</I> and <I>ChrBopt</I> for sequence confirmation.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 3 - 29/07: Purified <I>ChrB</I> and <I>ChrBopt</I> for size and sequence confirmation.</SPAN><DIV id="collapsiblew6-3" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of <I>ChrB</I> and <I>ChrBopt</I> for confirmation of size and sequence.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid purification, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing</A></P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15_chr_igem040" class="journal-protocol">Figure 21 </A>Confirmation of insert size.</P><P><A data-toggle="modal" data-target="#dundee15_chr_150729_chrb-combined" class="journal-protocol"> Figure 22 </A>Confirmation of sequence.</P><P><B>Next Steps:</B> Confirmation of expression of <I>ChrB</I> and <I>ChrBopt</I> by western blotting.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 4 - 30/07: Repeated transformation of pSB1C3-pChr-GFP into <I>E.coli JM110</I></SPAN><DIV id="collapsiblew6-4" class="collapse box-content"><P><B>Aim of experiment:</B> Digest of pSB1C3-pChr with SpeI and PstI for insertion of GFP. Repeat of unsuccessful transformation from 28/07.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation, </A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150731_pChrGFP-ligations" class="journal-protocol"> Figure 23 </A>Plates after transformation</P><P><B>Next Steps:</B> Overnight cultures for purification of the pSB1C3-pChr-GFP for confirmation of insert size and sequence.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 5 - 31/07: Set up overnight cultures for confirmation of pSB1C3-pChr-GFP.</SPAN><DIV id="collapsiblew6-5" class="collapse box-content"><P><B>Aim of experiment:</B> Overnight cultures for subsequent plasmid purification of pSB1C3-pChr-GFP.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol">Overnight culture</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of the Plasmids for confirmation of insert size and sequence.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 6 - 01/08: Purified pSB1C3-pChr-GFP for size confirmation and sequencing.</SPAN><DIV id="collapsiblew6-6" class="collapse box-content"><P><B>Aim of experiment:</B> Purification of pSB1C3-pChr-GFP for confirmation of insert size and sequence.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid purification, </A><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol">Restriction digest, </A><A data-toggle="modal" data-target="#sequencing-modal" class="journal-protocol">Sequencing</A></P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15_chr_igem043" class="journal-protocol"> Figure 24 </A> Gel for size confirmation.</P><P><A data-toggle="modal" data-target="#dundee15_chr_150801_pchrgfp" class="journal-protocol"> Figure 25 </A> Sequence confirmation.</P><P><B>Next Steps:</B> Overnight cultures in preparation for western blotting for characterisation of expression of both, <I>GFP</I>, and <I>ChrB</I> and <I>ChrBopt</I>.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 7 - 02/08: Set up cultures in preparation for western blotting.</SPAN><DIV id="collapsiblew6-7" class="collapse box-content"><P><B>Aim of experiment:</B> Overnight cultures of sequence confirmed <I>ChrB</I>, <I>ChrBopt</I>, <I>pChrGFP1</I> (from 15/07), <I>pChrGFP3</I> and <I>pChrGFP4</I> (31/07) for subsequent western blotting.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight culture</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Western blotting for characterisation of expression of both, <I>GFP</I>, and <I>ChrB</I> and <I>ChrBopt</I>.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 03/08</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">This week was entirely used for characterising the expression of GFP from the promoter <I>pChr</I>, and the expression of <I>ChrB</I> from the <A href="http://parts.igem.org/Part:BBa_K895000" target="_blank"><I>tat-promoter</I></A> in pUniprom.</P></DIV><DIV id="collapsiblew7" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - Day 7 - 03/08 - 09/08: Determined expression of <I>ChrB</I>, <I>ChrBopt</I>, and GFP by western blotting.</SPAN><DIV id="collapsiblew7-1" class="collapse box-content"><P><B>Aim of experiment:</B> To determine whether or not the required proteins, <I>GFP</I>, and <I>ChrB</I>, are expressed by blotting against GFP and against a 6His-tag respectively.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol">SDS-PAGE and western blot</A></P><P><B>Results:</B> This procedure was repeated several times during this week in order to find appropriate concentrations and volumes to load. Figure 26 was blotted after SDS-PAGE with the following concentrations:
                    <UL><LI>GFP: Dilute pellet of 1ml overnight culture with 1ml TBS. Take 100µl of this solution and mix with 100µl Laemmli Buffer. Load 3µl on the SDS-gel.</LI><LI>HIS: Dilute pellet of 1ml overnight culture with 200µl Laemmli buffer. Load 10µl on the SDS-gel.</LI></UL></P><P><A data-toggle="modal" data-target="#dundee15_chr_150809_blot" class="journal-protocol"> Figure 26 </A></P><P><B>Next Steps:</B>Transformation of both plasmids into MG1655 with the goal to characterise the interaction between <I>ChrB</I> and <I>pChr</I>. Furthermore the original biobrick, <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> will be tested under the same conditions for comparison.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/08</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">This week was spent with transforming all combinations of pSB1C3-pChr-GFP and pUniprom-ChrB into <I>E. coli MG1655</I>. Tests for their interaction were also prepared.</P></DIV><DIV id="collapsiblew8" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - 10/08: Transformed both pSB1C3-pChr-GFP-plasmids, pUniprom-ChrB, and pUniprom-ChrBopt, as well as <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> into <I>E. coli MG1655</I>.</SPAN><DIV id="collapsiblew8-1" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the following combinations of pSB1C3-pChr-GFP and pUniprom-ChrB into <I>MG1655</I>:
                    <UL><LI>pUniprom-ChrB2 (29/07) - pSB1C3-pChr-GFP (31/07)</LI><LI>pUniprom-ChrB2 (29/07) - pSB1C3-pChr-GFP (15/07)</LI><LI>pUniprom-ChrBopt3 (29/07) - pSB1C3-pChr-GFP (31/07)</LI><LI>pUniprom-ChrBopt3 (29/07) - pSB1C3-pChr-GFP (15/07)</LI></UL>
                  in a stepwise fashion. Furthermore transform <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> into <I>MG1655</I>. Transformations were done in a stepwise fashion with only one of the plasmids at a time.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid purification, </A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150811_transformations" class="journal-protocol"> Figure 27 </A></P><P><B>Next Steps:</B> Making newly transformed cells competent again in order to transform the second plasmid as well.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 2 - 11/08: Set up overnight cultures of all transformed cells.</SPAN><DIV id="collapsiblew8-2" class="collapse box-content"><P><B>Aim of experiment:</B> To prepare overnight cultures of yesterdays transformaions in order to make them competent to transform the second plasmid as well.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol">Overnight culture</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Make cells of overnight culture competent to transform in second plasmid.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 3 - 12/08: Made competent <I>MG1655</I> with pUniprom-ChrB2, pUniprom-ChrBopt3, pSB1C3-pChr-GFP (15/07), pSB1C3-pChr-GFP (31/07).</SPAN><DIV id="collapsiblew8-3" class="collapse box-content"><P><B>Aim of experiment:</B>To make transformed cells competent in order to transform the second plasmid. The same plasmids that were used for the initial transformations were also used for this set of transformations.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#competent-modal" class="journal-protocol">Competent cells, </A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformations </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Blot against GFP in order to confirm (or refute) that <I>ChrB</I> interacts with <I>pChr</I> and hence switches off expression of GFP.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 4 - 13/08: Transformed pUniprom-ChrB2 and pUniprom-ChrBopt3 into <I>MG1655+pSB1C3-pChr-GFP</I>(15/07) and <I>MG1655+pChrGFP</I>(31/07)</SPAN><DIV id="collapsiblew8-4" class="collapse box-content"><P><B>Aim of experiment:</B> To build a strain of <I>E. coli MG1655</I>, containing both plasmids required for a chromate sensor.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformation</A></P><P><B>Results:</B> No successful transformants were found.</P><P><B>Next Steps:</B> Repeat this experiment but transform pSB1C3-pChr-GFP (15/07) and pSB1C3-pChr-GFP (31/07) into competent <I>MG1655+pUniprom-ChrB2</I> and <I>MG1655+pUniprom-ChrBopt3</I> respectively.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 17/08</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">During this week <I>E. coli MG1655</I> containing both plasmids for a. Western Blots were used to determine expression of <I>ChrB</I>, <I>ChrBopt</I>, and GFP.</P></DIV><DIV id="collapsiblew9" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - 17/08: Transformations akin to 13/08 were done, but pSB1C3-pChr-GFP(15/07) and pSB1C3-pChr-GFP(31/07) were transformed into <I>MG1655+pUniprom-ChrB2</I> and <I>MG1655+pUniprom-ChrBopt3</I>.</SPAN><DIV id="collapsiblew9-1" class="collapse box-content"><P><B>Aim of experiment:</B> To build a strain of MG1655, containing both plasmids required for a chromate sensor.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformation </A></P><P><B>Results:</B> Successful transformants were found on all plates.</P><P><B>Next Steps:</B> To determine expression of <I>ChrB</I> and <I>ChrBopt</I> (via 6His-tag) and expression of GFP.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 2 - 18/08: Preparation of samples for western blot.</SPAN><DIV id="collapsiblew9-2" class="collapse box-content"><P><B>Aim of experiment:</B> To determine whether <I>ChrB</I>, <I>ChrBopt</I>, and GFP are expressed and have first estimates of differences in expression levels.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight culture</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Lyse cells and prepare samples for SDS-PAGE + Western-Blot.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 3-6 - 19/08 - 22/08: Repeated western blots with different concentrations to determine whether the expression of <I>ChrB</I> has an effect on the expression of GFP.</SPAN><DIV id="collapsiblew9-3" class="collapse box-content"><P><B>Aim of experiment:</B> To determine whether expression of <I>ChrB</I> or <I>ChrBopt</I> has an effect on the expression of GFP.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> SDS-PAGE + Western Blot</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150822_blot" class="journal-protocol"> Figure 28 </A></P><P><B>Next Steps:</B> Check if different chromate salts have an effect on the respective expression levels.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 24/08</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Plate reader experiments were carried out in order to determine the effect of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB> on the expression of GFP.</P></DIV><DIV id="collapsiblew10" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - Day 7 - 24/08-30/08: Design of Plate reader experiments.</SPAN><DIV id="collapsiblew10-1" class="collapse box-content"><P><B>Aim of experiment:</B> To design a series of plate reader experiments to measure OD<SUB>600</SUB> and relative fluorescence of the chromate sensor as a response to different concentrations of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#platereader-modal" class="journal-protocol"> Plate Reader Experiments </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150824_platereader_BBa_K1058008" class="journal-protocol"> Figure 29 </A><I>MG1655 + <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A></I></P><P><B>Results:</B><A data-toggle="modal" data-target="#icecream" class="journal-protocol"> Figure 30 </A><I>MG1655 + pSB1C3-pChr-GFP + pUniprom-ChrB</I></P><P><B>Results:</B><A data-toggle="modal" data-target="#flake" class="journal-protocol"> Figure 31 </A><I>MG1655 + pSB1C3-pChr-GFP + pUniprom-ChrBopt</I></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150824_platereader_pchrbgfp" class="journal-protocol"> Figure 32 </A><I>MG1655 + pSB1C3-pChr-GFP</I></P><P><B>Next Steps:</B> Try the system with different expression levels of <I>ChrB</I> and <I>ChrBopt</I>. Induce overexpression by exploiting the T7-promoter in pUniprom after transforming the different combinations between pSB1C3-pChr-GFP and pUniprom-ChrB/pUniprom-ChrBopt into <I>BL21(DE3)</I> and adding IPTG.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 31/08</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">UK Synbio Conference and UK iGEM meetup at Westminster University.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 07/09</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Parts from CeBiTec Bielefeld arrived and attempts at cloning were made. The chromate sensing system was transformed into <I>BL21(DE3)</I> and some of the plate reader experiments repeated.</P></DIV><DIV id="collapsiblew12" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - 07/09: Transformation of pSB1K3 (Bielefeld) into <I>MG1655</I>.</SPAN><DIV id="collapsiblew12-1" class="collapse box-content"><P><B>Aim of experiment:</B> The high-copy plasmid arrived at a concentration of 7 <SUP>ng</SUP>/<SUB>µl</SUB>. Hence it was transformed in order to produce more of it before purifying and digesting it.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformation, </A></P><P><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight culture</A> for new <I>E. coli MG1655</I></P><P><B>Results:</B> No transformants were found.</P><P><B>Next Steps:</B> Retry transformation with new competent <I>E. coli MG1655</I>.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 2 - 08/09: Transformation of pSB1K3 into MG1655.</SPAN><DIV id="collapsiblew12-2" class="collapse box-content"><P><B>Aim of experiment:</B> Test transformation of BBa_K1058008 into new <I>MG1655</I> competent cells and re-transformation of pSB1K3 for subsequent miniprep.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformation </A></P><P><B>Results:</B> Successful test transformation, but no colonies on pSB1K3 plate.</P><P><B>Next Steps:</B> Use an alternative compatible plasmid for ligation of codon optimised <I>ChrB</I> (Bielefeld).</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 3 - 09/09: Transformation of pSB1K3 into MG1655.</SPAN><DIV id="collapsiblew12-3" class="collapse box-content"><P><B>Aim of experiment:</B> Restriction digest of pT7KS with EcoRI and PstI as alternative to pSB1K3. Ligation of digested <I>ChrB</I> (Bielefeld) and transformation of pT7KS-ChrB. Also transformations of Chromate-sensor plasmids into BL21 (DE3) in the following combinations: <UL><LI>pSB1C3-pChr-GFP + pUniprom-ChrB</LI><LI>pSB1C3-pChr-GFP + pUniprom-ChrBopt</LI><LI>pUniprom-ChrB</LI><LI>pUniprom-ChrBopt</LI></UL></P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restriction-digest-modal" class="journal-protocol"> Restriction Digest, </A><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol"> Gel Extraction, </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligation </A></P><P><B>Results:</B>Transformations of the plasmid with pT7KS-ChrB (Bielefeld) yielded 12 colonies, but also colonies on the vector control plate. Transformations of pSB1C3-pChr-GFP + pUniprom-ChrB, pSB1C3-pChr-GFP + pUniprom-ChrBopt, pUniprom-ChrB, and pUniprom-ChrBopt into <I>BL21(DE3)</I> were successful.</P><P><B>Results:</B></P><P><A data-toggle="modal" data-target="#dundee15_chr_150911_chrbbielefeld" class="journal-protocol"> Figure 33 </A> pT7KS-ChrB (Bielefeld) transformations</P><P><A data-toggle="modal" data-target="#dundee15_chr_150911_bl21" class="journal-protocol"> Figure 34 </A> Transformations into <I>BL21(DE3)</I></P><P><B>Next Steps:</B> Confirmation of colonies with pT7KS-ChrB by colony PCR. Preparation of recombinant <I>BL21(DE3)</I> for plate reader experiments.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 4 - 10/09: Colony PCR of pT7KS-ChrB (Bielefeld).</SPAN><DIV id="collapsiblew12-4" class="collapse box-content"><P><B>Aim of experiment:</B> Check if any of the colonies after ligation of ChrB (Bielefeld) into pT7KS were successful despite colonies on the control plate.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#colony-modal" class="journal-protocol"> Colony PCR </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_150911_igem050" class="journal-protocol"> Figure 34 </A></P><P><B>Next Steps:</B> Prepare transformed <I>BL21(DE3)</I> for plate reader experiments. Attempt new transformation of pT7KS-ChrB (Bielefeld).</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 5 - 11/09: Preparation for Plate reader experiment with <I>BL21(DE3)</I></SPAN><DIV id="collapsiblew12-5" class="collapse box-content"><P><B>Aim of experiment:</B> Set up overnight cultures for subsequent plate reader experiments.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#overnightculture-modal" class="journal-protocol"> Overnight cultures </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Set up plate reader, collect and analyse data. Assess the effect of induction of higher repressor-expression on the level of GFP.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 6 - 12/09: Set up plate reader.</SPAN><DIV id="collapsiblew12-6" class="collapse box-content"><P><B>Aim of experiment:</B> To design a series of plate reader experiments to measure OD<SUB>600</SUB> and relative fluorescence of the chromate sensor. At first no chromate was added to this plate in order to assess the effect of IPTG induction.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#platereader-modal" class="journal-protocol"> Plate reader experiments</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#dundee15_chr_platereaderinduced" class="journal-protocol"> Figure 35 </A></P><P><B>Next Steps:</B>Repeat experiments with added chromate substances.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 14/09</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">During this week the samples from the previous plate reader were retrieved and prepared for blotting against both GFP, and the 6His-Tag of <I>ChrB</I> and <I>ChrBopt</I>.</P></DIV><DIV id="collapsiblew13" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">Day 1 - 14/09: Western blot of plate reader samples <I>MG1655 + pSB1C3-pChr-GFP + pUniprom-ChrB</I>, and  <I>MG1655 + pSB1C3-pChr-GFP + pUniprom-ChrBopt</I>.</SPAN><DIV id="collapsiblew13-1" class="collapse box-content"><P><B>Aim of experiment:</B> To identify whether or not there is any detectable difference in both, the level of GFP, and the level of ChrB or ChrBopt, before and after induction with IPTG.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> SDS-PAGE + Western blot </A></P><P><B>Results:</B> Insert blot image here.</P><P><B>Next Steps:</B> Repeat the assays of the previous weeks by subcloning ChrB, ChrBopt, and ChrB (Bielefeld) into vectors with different expression levels and see how it <A href="https://2015.igem.org/Team:Dundee/soekrit">influences</A> the level of GFP expression. Then add different chromate salts and also probe the system with other substances, e.g. other heavy-metal salts.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="sponsor"><DIV class="container"><DIV class="footer-btn"><A href="https://2015.igem.org/Team:Dundee/Introduction_Video" class="btn btn-info btn-lg" role="button">Watch Our Introduction Video</A></DIV></DIV></SECTION><FOOTER><DIV class=""><DIV class="container"><DIV class="col-md-3"><A href="http://www.dundee.ac.uk/" target="_blank"><H4>Dundee University</H4></A><P>Located on the east coast of Scotland, The University of Dundee is one of Scotland's leading research institutes.</P></DIV><DIV class="col-md-3"><H4>Social Media</H4><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="https://www.youtube.com/channel/UCJHBLvLhjzGqHqdjABjOY-w">youtube.com/DundeeiGEM2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.facebook.com/igemdundee2015">facebook.com/iGEMDundee2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.twitter.com/dundeeigemteam">twitter.com/DundeeiGEMTeam</A></DIV></DIV></DIV></DIV></DIV></FOOTER><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>