<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_CGCraigon skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/CGCraigon</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><H1><HIGHLIGHT class="highlight">CSI: DUNDEE</HIGHLIGHT></H1><H3><HIGHLIGHT class="highlight">The Forensic Toolkit</HIGHLIGHT></H3><A href="https://2015.igem.org/Team:Dundee/intro_video" class="btn btn-primary btn-lg">Watch Our Introduction Video</A><DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><SPAN id="biospray-title">Nasal Mucus </SPAN><DIV id="nmcollapsible" class="collapse biospray-content"><DIV class="row"><DIV class="box"><DIV class="labtitle">Week Beginning 15/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Produced a purified plasmid of pIDT-<I>OBP2A</I> in order to use it as template in PCR. </P></DIV><DIV id="nmcollapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/6: Transformation of pIDT-<I>OBP2A</I> in MC1610 <I>E.coli</I> strain </SPAN><DIV id="nmcollapsiblew1-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To transform pIDT-<I>OBP2A</I> into MC1061 <I> e.coli</I> strain. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> The result of this transformation can be seen tomorrow. If successful then the next step will be to produce an overnight culture in preparation of subsequent plasmid purification. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/6: Overnight culture of pIDT-<I>OBP2A</I> plasmid in MC1061  </SPAN><DIV id="nmcollapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To produce an overnight culture of pIDT-<I>OBP2A</I> from positive colonies of MC1061 from the transformation done on the 17/6.  
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A></P><P><B>Results:</B> N/A
                  </P><P><B>Next Steps:</B> If the overnight culture successfully grows then the next stage of experimentation will be to produce a plasmid purification of pIDT-<I>OBP2A</I> from that overnight culture.                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">19/6: Plasmid Purification of pIDT-<I>OBP2A</I> from the overnight culture    </SPAN><DIV id="nmcollapsiblew1-3" class="collapse box-content"><P><B>Aim of experiment:</B> The overnight culture produced on the 18/6 will now undergo a plasmid purification in order to obtain  the pIDT-<I>OBP2A</I> plasmid.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification </A></P><P><B>Results:</B> The plasmid purification yielded a concentration of 305.53 ng/ul.
                  </P><P><B>Next Steps:</B> Assuming the plasmid purification has been successful, then the next stage of experimentation will be to run a PCR using this purified pIDT-<I>OBP2A</I> plasmid as the template for subsequent amplification.              </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="labtitle">Week Beginning 22/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Removal of the signaling peptide from <I>OBP2A</I> and insertion of  <I>OBP2A</I> into the biobrick vector pSB1C3. </P></DIV><DIV id="nmcollapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/6: PCR of pIDT-<I>OBP2A</I> using primers designed for the removal of the signal peptide  </SPAN><DIV id="nmcollapsiblew2-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a PCR of the purified pIDT-<I>OBP2A</I> plasmid using a set of primers that have been designed to remove the signaling peptide at the start of the <I>OBP2A</I> gene, whilst leaving the rest of the gene intact.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> If the PCR is successful, then the next step will be to gel extract <I>OBP2A</I> from a  the  gel corresponding to the size of the <I>OBP2A</I> gene fragment.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/6: Gel extraction of <I>OBP2A</I> from the PCR performed on the 22/6  </SPAN><DIV id="nmcollapsiblew2-2" class="collapse box-content"><P><B>Aim of experiment:</B> To perform a gel extraction of <I>OBP2A</I> from a PCR reaction that was performed on the 22/6. Once gel extraction has been performed a subsequent gel will be done to determine if the gel extraction had been successful.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol">Gel Extraction </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#nm-figure1-modal" class="journal-protocol"> Figure 1 </A><A data-toggle="modal" data-target="#nm-figure2-modal" class="journal-protocol"> Figure 2</A></P><P><B>Next Steps:</B> To perform a restriction of <I>OBP2A</I> in preparation for ligation.             
 </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/6:  Restriction, ligation and transformation of <I>OBP2A</I> into pSB1C3 </SPAN><DIV id="nmcollapsiblew2-3" class="collapse box-content"><P><B>Aim of experiment:</B>  To perform a restriction, ligation and subsequently a transformation of  of the gel extracted <I>OBP2A</I> into the JM110 <I>E.coli</I> strain.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digest  </A>:<A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation </A>:<A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation  </A></P><P><B>Results:</B> N/A


    </P><P><B>Next Steps:</B> If the transformation is successful and positive colonies form then these colonies will then be grown overnight in preparation for plasmid purification.                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">25/6: Re-ligation and transformation of <I>OBP2A</I> into pSB1C3 </SPAN><DIV id="nmcollapsiblew2-4" class="collapse box-content"><P><B>Aim of experiment:</B> The transformation performed on the 24/6 failed and as a result <I>OBP2A</I> will now be re-ligated into the biobrick vector pSB1C3 and subsequently transformed into the JM110 <I>E.coli</I> strain.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation:</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#nm-figure3-modal" class="journal-protocol"> Figure 3 </A></P><P><B>Next Steps:</B>If the transformation is successful and positive colonies form then these colonies will then be grown overnight in preparation for plasmid purification.                   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">26/6: Overnight cultures of pSB1C3-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew2-5" class="collapse box-content"><P><B>Aim of experiment:</B> To produce overnight cultures of pSB1C3-<I>OBP2A</I> from a positive colony from the transformation done on the 25/6.  
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B>  Produce plasmid purifications from the overnight cultures of pSB1C3-<I>OBP2A</I> in preparation for presequence digest.                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/6: Plasmid purification and presequence digest of pSB1C3-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew2-6" class="collapse box-content"><P><B>Aim of experiment:</B> To perform a plasmid purification of the overnight cultures of pSB1C3-<I>OBP2A</I> from the 26/6. Then to subsequently perform a presequence digest of those plasmid purification.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification </A><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Presequence digest</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure4-modal" class="journal-protocol"> Figure 4 </A><TABLE border="1" style="width:20%"><TBODY><TR><TH>Sample</TH><TH>Concentration(ng/ul)</TH></TR><TR><TD>1</TD><TD>308.74</TD></TR><TR><TD>2</TD><TD>342.3</TD></TR><TR><TD>3</TD><TD>369.90</TD></TR><TR><TD>4</TD><TD>321.97</TD></TR><TR><TD>5</TD><TD>460.78</TD></TR><TR><TD>6</TD><TD>261.61</TD></TR></TBODY></TABLE></P><P><B>Next Steps:</B> It was concluded from the results of these experiments that sample 5 of pSB1C3-<I>OBP2A</I> will be sent for sequencing.
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="box"><DIV class="labtitle">Week Beginning 29/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Positive Sequencing of pSB1C3-<I>OBP2A</I> and subsequent plasmid purification of pSB1C3-<I>OBP2A</I>. </P></DIV><DIV id="nmcollapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/6: Plasmid purified pSB1C3-<I>OBP2A</I> is sent for sequencing  </SPAN><DIV id="nmcollapsiblew3-1" class="collapse box-content"><P><B>Aim of Experiment:</B> Sample 5 of the plasmid purified pSB1C3-<I>OBP2A</I> that was done on the 24/6 is sent for sequencing to determine if <I>OBP2A</I> has successfully ligated into pSB1C3.   </P><P><B>Protocols Used:</B> N/A</P><P><B>Results: </B> N/A 

                </P><P><B>Next Steps:</B> If sequencing comes back positive, then the next step will be to clone <I>OBP2A</I> into the two vectors of the bacterial two hybrid system.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/6: Overnight culture of pSB1C3-<I>OBP2A</I> from  the sequenced colony  </SPAN><DIV id="nmcollapsiblew3-2" class="collapse box-content"><P><B>Aim of Experiment:</B> Sequencing came back positive for pSB1C3-<I>OBP2A</I>. So an overnight culture of pSB1C3-<I>OBP2A</I>  will be done in preparation for plasmid purification.     </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> Once the overnight culture has been given ample time to grow, the culture will then undergo a plasmid purification. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/7: Plasmid Purification of pSB1C3-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew3-3" class="collapse box-content"><P><B>Aim of Experiment:</B> To produce a plasmid purification of the sequenced pSB1C3-<I>OBP2A</I> from the overnight culture set up on the 30/6.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification </A></P><P><B>Results: </B> The plasmid purified pSB1C3-<I>OBP2A</I> produced a concentration of 448.83 ng/ul.  </P><P><B>Next Steps:</B> The next stage of experimentation is to use the plasmid purification of pSB1C3-<I>OBP2A</I> as the template for PCR using primers that are designed to separate the gene for use in the two hybrid system.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="labtitle">Week Beginning 12/7/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Ran PCR of <I>OBP2A</I> using primers for insertion of <I>OBP2A</I> into pUT18 and pT25. </P></DIV><DIV id="nmcollapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/7: PCR of <I>OBP2A</I> for insertion into bacterial two hybrid vectors: pUT18 and pT25 </SPAN><DIV id="nmcollapsiblew4-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a PCR of <I>OBP2A</I> for its insertion into the bacterial two hybrid vectors; pUT18 and pT25. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B>The next stage of experimentation will be to gel extract and then subsequently restrict the amplified gene fragments of <I>OBP2A</I>.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 20/7/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Attempted insertion of <I>OBP2A</I> into the vectors for the bacterial two hybrid system; pUT18 and pT25.  </P></DIV><DIV id="nmcollapsiblew5" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">20/7: Gel extraction and restriction of <I>OBP2A</I> from the PCR done on the 17/7  </SPAN><DIV id="nmcollapsiblew5-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a gel extraction and then subsequently a restriction of <I>OBP2A</I> from the PCR produced on the 17/7 using primers designed for the insertion of <I>OBP2A</I> into the vectors for the bacterial two hybrid system, pUT18 and pT25.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol">Gel Extraction</A><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digest </A></P><P><B>Results: </B>N/A</P><P><B>Next Steps:</B> If the gel extraction and subsequent restriction is successful, then the next step will be to ligate the restricted <I>OBP2A</I> into the bacterial two hybrid vector, pUT18 and pT25.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">21/7: Ligation and transformation of <I>OBP2A</I> into pUT18 and pT25.     </SPAN><DIV id="nmcollapsiblew5-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a ligation and  then subsequently a transformation  of <I>OBP2A</I> into the vectors for the bacterial two hybrid system, pUT18 and pT25. Whilst using JM110 <I>E.coli</I> cells as a chassis for transformation. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results:  N/A </B></P><P><B>Next Steps:</B> The results from the transformation will be seen tomorrow. If successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight colonies in preparation for purification and sequencing.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/7: Re-ligation and transformation of <I>OBP2A</I> into pUT18 and PT25.
  </SPAN><DIV id="nmcollapsiblew5-3" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a re-ligation and transformation of <I>OBP2A</I> into pUT18 and pT25 - since the transformation performed on the 21/7 failed. 
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">ligation</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results: N/A </B></P><P><B>Next Steps:</B> If transformation is successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight colonies in preparation for purification and sequencing.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/7: Production of overnight cultures of pUT18-OBP2A. </SPAN><DIV id="nmcollapsiblew5-4" class="collapse box-content"><P><B>Aim of Experiment:</B>  To produce overnight cultures of pUT18-<I>OBP2A</I>. Unfortunately no positive colonies were produced from the transformation of <I>OBP2A</I> into pT25.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> Once the cells have been given ample time to grow, they will undergo plasmid purification for subsequent sequencing.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/7: Plasmid purification and subsequent presequence digest of pUT18-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew5-5" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a plasmid purification of  the overnight cultures of pUT18-<I>OBP2A</I> and then a subsequent presequence digest.  
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification</A><A data-toggle="modal" data-target="#presqunce digest-modal" class="journal-protocol">Presequence digest</A></P><P><B>Results: <A data-toggle="modal" data-target="#nm-figure5-modal" class="journal-protocol"> Figure 5</A></B></P><P><B>Next Steps:</B> If sequencing comes back positive for pUT18-<I>OBP2A</I> then this can now be implemented into the bacterial two hybrid system. Another attempt at cloning <I>OBP2A</I> into pT25 will be done next week.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 27/7/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Successful insertion of the two subunits of <I>OBP2A</I> into the vectors pUT18 and pT25.   </P></DIV><DIV id="nmcollapsiblew6" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/7: PCR  and gel extraction of <I>OBP2A</I> for preparation of insertion into  pT25.   </SPAN><DIV id="nmcollapsiblew6-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a PCR and subsequent gel extraction of <I>OBP2A</I> for its insertion into pT25.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure6-modal" class="journal-protocol"> Figure 6 </A></P><P><B>Next Steps:</B> The next stage of experimentation will be to restrict <I>OBP2A</I> for subsequent ligation into pT25.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/7: Restriction of <I>OBP2A</I> for pT25 and positive sequencing of pUT18-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew6-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a restriction of <I>OBP2A</I> for subsequent ligation into PT25. Also, sequencing came back positive for pUT18-<I>OBP2A</I>, so pUT18-<I>OBP2A</I> can now be implemented in the bacterial two hybrid system.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digestion</A></P><P><B>Results: </B>N/A </P><P><B>Next Steps:</B> The next stage of experimentation will be to ligate <I>OBP2A</I> into the pT25 vector.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/7: Ligation and transformation of <I>OBP2A</I> into pT25.    </SPAN><DIV id="nmcollapsiblew6-3" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a ligation and transformation of <I>OBP2A</I> into the bacterial two hybrid vector pT25, so that the interaction between the two constituents of <I>OBP2A</I> can be observed using the bacterial two hybrid system.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">ligation</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If the transformation is successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight cultures of those colonies in preparation for purification and sequencing.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">31/7: Overnight cultures of pT25-<I>OBP2A</I> from positive colonies. </SPAN><DIV id="nmcollapsiblew6-4" class="collapse box-content"><P><B>Aim of Experiment:</B> To produce overnight cultures of pT25-<I>OBP2A</I> from the transformation done on the 30/7. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A></P><P><B>Results: </B><A> N/A </A></P><P><B>Next Steps:</B> Once the cells have grown overnight they will undergo plasmid purification for subsequent sequencing.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/8: Plasmid purification and subsequent presequence digest of pT25-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew6-5" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a plasmid purification of pT25-<I>OBP2A</I>  from the overnight cultures done on the 31/7 and then a subsequent presequence digest.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification</A><A data-toggle="modal" data-target="#pre-sequence digest-modal" class="journal-protocol">Presequence digest</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure7-modal" class="journal-protocol"> Figure 7
         </A><TABLE border="1" style="width:20%"><TBODY><TR><TH> Purified Plasmid Sample</TH><TH>Concentration(ng/ul)</TH></TR><TR><TD>1</TD><TD>50.37</TD></TR><TR><TD>2</TD><TD>42.36</TD></TR><TR><TD>3</TD><TD>37.13</TD></TR><TR><TD>4</TD><TD>14.87</TD></TR><TR><TD>5</TD><TD>80.02</TD></TR><TR><TD>6</TD><TD>44.34</TD></TR><TR><TD>7</TD><TD>22.83</TD></TR></TBODY></TABLE></P><P><B>Next Steps:</B> If sequencing comes back positive for pT25-<I>OBP2A</I> then this can now be implemented into the bacterial two hybrid system. The results from the sequencing should arrive early on the 3/8.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 3/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Positive sequencing of pT25-<I>OBP2A</I>, subsequent transformation into the bacterial two hybrid system and plating onto MacConkey agar.    </P></DIV><DIV id="nmcollapsiblew7" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/8: Transformation of pUT18-<I>OBP2A</I> into BTH101 <I>E.coli</I> strain.    </SPAN><DIV id="nmcollapsiblew7-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a transformation of pUT18-<I>OBP2A</I> into BTH101, so that a subsequent transformation of pT25-<I>OBP2A</I> into the same chassis can occur as both plasmids are required to reside in the same chassis so that their interaction can be observed.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> If the transformation is successful, then the next stage of experimentation will be to re-transform this culture of cells using the pT25-<I>OBP2A</I> plasmid.      </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">4/8:Day culture and transformation of BTH101 containing pUT18-<I>OBP2A</I> with pT25<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew7-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a day culture and then a subsequent transformation of the BTH101 chassis containing  pUT18-<I>OBP2A</I> from the transformation performed on 3/8.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> If the transformation is successful then the next stage of experimentation will be to observe the  protein interaction of the two subunits of <I>OBP2A</I> using MacConkey agar plates.     </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">5/8:Day culture and plating of positive colonies from transformation onto MacConkey agar.  </SPAN><DIV id="nmcollapsiblew7-3" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a day culture and then subsequent plating of positive colonies from the transformation performed on the 4/8. Positive colonies from this transformation contain both vectors of the bacterial two hybrid system along with the two subunits of <I>OBP2A</I> and so now the interaction can be observed. The day culture will be plated onto MacConkey agar which once the cells have been given enough time to grow will give a visual demonstration of the  protein interaction through a color change.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B>N/A </P><P><B>Next Steps:</B> It has been decided that whilst the <I>E.coli</I> strain BTH101 that is being used in this experiment is useful for visualizing protein interaction, it is very unstable. A better <I>E.coli</I> strain called MG1655(Δcya) will be used instead as it is much more stable. This means that tomorrow overnights will be prepared of MG1655(Δcya)for transformation with pUT18-<I>OBP2A</I> and then subsequently pT25-<I>OBP2A</I>.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">6/8: Overnight cultures of MG1655(Δcya)</SPAN><DIV id="nmcollapsiblew7-4" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform an overnight culture of MG1655(Δcya)<I>E.coli</I> in preparation for transformation with pUT18-<I>OBP2A</I> and then pT25-<I>OBP2A</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> Once the overnight culture has been give sufficient time to grow these cells will then be transformed with pUT18-<I>OBP2A</I> and then subsequently re-transformed with pT25-<I>OBP2A</I>.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">7/8: Results of MacConkey agar growth and transformation of  MG1655(Δcya)with pUT18-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew7-5" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a transformation of MG1655(Δcya)with pUT18-<I>OBP2A</I>. Also the results of plating of our bacterial two hybrid interaction onto MacConkey agar has produced promising results as the colonies appear to be taking up a small amount of red dye - an indicator of protein interaction. This suggests that the two separated parts of <I>OBP2A</I> may in fact be interacting in vivo. This interaction will need to be quantified through the use of the β-galactosidase assay. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> If the transformation of MG1655(Δcya) with pUT18-<I>OBP2A</I> is successful then the next step of experimentation will be re-transform positive colonies with pT25-<I>OBP2A</I>.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">8/8: Transformation of MG1655(Δcya) containing pUT18-<I>OBP2A</I> with pT25-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew7-6" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a transformation of MG1655(Δcya) containing pUT18-<I>OBP2A</I> with pT25-<I>OBP2A</I>. Positive colonies were produced from the transformation done on the 7/7 and so as a result a day culture was produced and subsequently re-transformed with pT25-<I>OBP2A</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> If the transformation of MG1655(Δcya) with pT25-<I>OBP2A</I> is successful then the next step of experimentation will be to undertake the β-galactosidase assay  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> First attempt at determination of the protein interaction of the <I>OBP2A</I> subunits using the β-galactosidase assay.   </P></DIV><DIV id="nmcollapsiblew8" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/8: Plating of Controls in preparation of the β-galactosidase assay   </SPAN><DIV id="nmcollapsiblew8-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To plate control interactions that are used in the β-galactosidase assay to aid in the interpretation of the results form the assay. This first assay attempt will consist of one positive control, one negative control and one regulatory control.       </P><P><B>Protocols Used: N/A </B></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> The next stage of experimentation will be to perform overnight cultures from positive colonies in preparation for the β-galactosidase assay.      </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/8: Overnight cultures of controls and <I>OBP2A</I> in preparation for the β-galactosidase assay.  </SPAN><DIV id="nmcollapsiblew8-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To prepare overnight cultures of positive colonies of MG1655(Δcya) containing pUT18-<I>OBP2A</I> and pT25-<I>OBP2A</I>, along with the controls in preparation for the  β-galactosidase assay. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> Once the cells have been given sufficient time to grow then the next stage of experimentation will be to inoculate those colonies into fresh liquid broth until they reach an O.D of 0.4 before being pellet and frozen for the assay.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title"> 12/8: Inoculation and preparation of cultures for the  β-galactosidase assay.    </SPAN><DIV id="nmcollapsiblew8-3" class="collapse box-content"><P><B>Aim of Experiment:</B>  To prepare cultures for use in the  β-galactosidase assay, this requires a small subsample of each overnight to be inoculated into fresh LB, two done for each colony. Then left to grow until the O.D of each culture is between 0.3-0.5. Once an O.D of 0.3-0.5 has been reached then 3 replicas of 1 ml samples are taken from each culture before being spun down, the supernatant then subsequently removed and the pellet frozen.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#assaypreparation-modal" class="journal-protocol"> Assay Preparation</A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> These samples are now ready for the β-galactosidase assay.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">13/8: First attempt at the determination of the protein interaction of the two <I>OBP2A</I> subunits using the β-galactosidase assay </SPAN><DIV id="nmcollapsiblew8-4" class="collapse box-content"><P><B>Aim of Experiment:</B> To determine the millers activity of the two interacting subunits of <I>OBP2A</I> in vivo as a basis of the level of interaction between them. This was done using the β-galactosidase assay. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#β-galactosidaseassay-modal" class="journal-protocol">β-galactosidase assay</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure8-modal" class="journal-protocol"> Figure 8 </A></P><P><B>Next Steps:</B> As it was the first attempt at a β-galactosidase assay, subsequent assays will be performed utilizing more controls. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/8: Preparation of overnight cultures of controls and samples in preparation of performing a second β-galactosidase assay    </SPAN><DIV id="nmcollapsiblew8-5" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform overnight cultures of the controls and samples that will be necessary for doing this second β-galactosidase assay. This assay will include two positive controls which are known to have a high level of interaction - these are NarG-NarJ and Zip positive controls. There will also be two negative controls, an empty MG1655(Δcya) chassis and pUT18-<I>OBP2A</I> with NarJ. Finally there is one regulatory control: which contains two empty vectors of pUT18 and pT25.     </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results: </B>N/A </P><P><B>Next Steps:</B> The next stage of experimentation will be to prepare samples for use in the β-galactosidase assay.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 17/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Second attempt at determining the protein interactions of the two <I>OBP2A</I> subunits using the β-galactosidase assay and subsequent cloning of <I>OBP2A</I> into the bacterial two hybrid vectors in switched order.  </P></DIV><DIV id="nmcollapsiblew9" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/8: Preparation of the controls and samples for use in the second attempt of the β-galactosidase assay  </SPAN><DIV id="nmcollapsiblew9-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To prepare cultures for use in the  β-galactosidase assay, this requires a small subsample of each overnight to be inoculated into fresh LB, two done for each colony. Then left to grow until the O.D of each culture is between 0.3-0.5. Once an O.D of 0.3-0.5 has been reached then 3 replicas of 1 ml samples are taken from each culture before being spun down, the supernatant then subsequently removed and the pellet frozen. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#assaypreparation-modal" class="journal-protocol">Assay Preparation</A></P><P><B>Results: </B>N/A </P><P><B>Next Steps:</B> The next stage of experimentation will be to perform the second β-galactosidase assay on the interaction between the two subunits of <I>OBP2A</I></P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/8: Second attempt at determination of the protein interactions of the <I>OBP2A</I> subunits using the β-galactosidase assay  </SPAN><DIV id="nmcollapsiblew9-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a second attempt at the β-galactosidase assay that will aid in the determination of the level of interaction that the two separated subunits of <I>OBP2A</I> are inflicting on one another. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#β-galactosidaseassay-modal" class="journal-protocol">β-galactosidase assay  </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure11-modal " class="journal-protocol"> Figure 10 </A></P><P><B>Next Steps:</B> The next stage of experimentation will be to switch the positioning of the two subunits of <I>OBP2A</I> in pUT18 and pT25 by having the smaller subunit in pUT18 and the large subunit in pT25. Along with this another β-galactosidase assay will be performed however this time it will be done under anaerobic conditions to see if that has any appreciable effect on the interaction of the two subunits of <I>OBP2A</I>.      </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title"> 20/8: PCR of <I>OBP2A</I> in preparation of pUT18 and pT25 insertion   </SPAN><DIV id="nmcollapsiblew9-3" class="collapse box-content"><P><B>Aim of Experiment:</B> To  Perform a PCR of <I>OBP2A</I> using primers for pU1T18 and pT25 insertion which have been designed to place the smaller <I>OBP2A</I>subunit into pUT18 and the larger subunit into pT25. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If PCR is successful then the next stage of experimentation will be to perform a gel extraction and restriction in perpetration for ligation and transformation of <I>OBP2A</I> into pUT18 and pT25.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">21/8: Second attempt at a PCR of <I>OBP2A</I> in preparation of pUT18 and pT25 insertion   </SPAN><DIV id="nmcollapsiblew9-4" class="collapse box-content"><P><B>Aim of Experiment:</B> The PCR performed on the 20/8 failed as no bands were found to be present on the gel during gel viewing. As a result of this a subsequent PCR was done today.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results: </B> N/A  </P><P><B>Next Steps:</B>  If PCR is successful then the next stage in experimentation will be to perform a gel extraction and restriction in perpetration for ligation and transformation of <I>OBP2A</I> in pUT18 and pT25.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/8: Second attempt at PCR of <I>OBP2A</I> for insertion into pUT18 and pT25 failed </SPAN><DIV id="nmcollapsiblew9-5" class="collapse box-content"><P><B>Aim of Experiment:</B> A second attempt at preparing <I>OBP2A</I> for insertion into the two vectors of the bacterial two hybrid system(pUT18 and pT25) has failed! As a result this will be left until next week, so that the rest of the week can be dedicated to preparing and undertaking  another β-galactosidase assay, that will be done under anaerobic conditions.  </P><P><B>Protocols Used:</B> N/A </P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> The next stage of experimentation is to prepare for a anaerobic β-galactosidase assay of <I>OBP2A</I> in the bacterial two hybrid vectors; pUT18 and pT25.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 24/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Performed an anaerobic β-galactosidase assay in order to determine the level of interaction between the two subunits of <I>OBP2A</I>.   </P></DIV><DIV id="nmcollapsiblew10" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/8: Preparation of samples for the anaerobic β-galactosidase assay of <I>OBP2A</I> in pUT18 and pT25  </SPAN><DIV id="nmcollapsiblew10-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To prepare the  samples and controls of <I>OBP2A</I> for the anaerobic β-galactosidase assay, so they can be frozen overnight in preparation for the assay itself.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#assaypreparation-modal" class="journal-protocol">Assay Preparation</A></P><P><B>Results: </B></P><P><B>Next Steps:</B> The next stage of experimentation will be to perform the β-galactosidase assay itself. 
        </P></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title"> 25/8: Performed an anaerobic β-galactosidase assay in order to determine the level of interaction of <I>OBP2A</I> in the bacterial two hybrid vectors: pUT18 and pT25.   </SPAN><DIV id="nmcollapsiblew10-2" class="collapse box-content"><P><B>Aim of Experiment:</B> Performed an anaerobic β-galactosidase assay of <I>OBP2A</I> in the bacterial two hybrid vectors: pUT18 and pT25. This is to determine the level of interaction that the smaller subunit of <I>OBP2A</I> has with its larger subunit in vivo. This particular assay has two positive controls, two negative controls and one regulatory control.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#β-galactosidaseassay-modal" class="journal-protocol"> β-galactosidase assay </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure12-modal" class="journal-protocol"> Figure 12 </A></P><P><B>Next Steps:</B> The results have shown that the two subunits of <I>OBP2A</I> in fact don't seem to be interacting in vivo - a possible reason for this might be aggregation of the two subunits preventing interaction. The  next stage of experimentation will be to test out the interaction when the subunit are switch in relation to heir positions on the vectors: pUT18 and pT25.    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 1: Amplification of <I>OBP2A</I> for insertion into pSB1C3 vector </B>. This is the PCR product from the amplification of <I>OBP2A</I> from the IDT plasmid using primers for cloning into pSB1C3. The gel indicates <I>OBP2A</I> has been amplified successfully since the observed band corresponds to the expected size of 500 bp. The concentration of the gel extracted <I>OBP2A</I> is 335.53 ng/ul. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 2: A subsequent gel was run of the amplified <I>OBP2A</I> after it was first excised from the gel in <I>figure 1 </I> - this was to determine if the gel extraction was successful. The gel indicates that <I>OBP2A</I> has been extracted from the gel successfully, since the observed band corresponds to the expected size of 500 bp. This can now be restricted in preparation for ligation into pSB1C3. </B></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 3: A gel analysis of my restriction digest from the 24/6. This gel analysis was done to determine if the reason for the failed transformation was an improperly digested <I>OBP2A</I>. This gel indicates that the restricted product hasn't been lost as the <I>OBP2A<I> gene fragment can be seen as a distinct band at 500 bp. A re-ligation of <I>OBP2A</I> into the pSB1C3 backbone will now occur. </I></I></B></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 4: Presequence Digest of pSB1C3-<I>OBP2A<I>. All seven plasmid purifications of the overnight cultures done on the 26/6 were digested to see if they contained the <I>OBP2A<I> insert. It is clear from the gel that all but sample 3 contain the insert for <I>OBP2A</I> as they all have faint bands at 500 bp's which corresponds to the size of <I>OBP2A<I>. It also appears that sample 5 contains the highest concentration, so that will be the sample that is sent for sequencing.  </I></I></I></I></I></I></B></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 5: Presequence digest of pUT18-<I>OBP2A<I> to determine if any samples contain the <I>OBP2A<I> insert. It can be seen from this gel that of the  5 samples that were digested only two are show to have the <I>OBP2A<I> insert, therefore sample 2 will be sent for sequencing.</I></I></I></I></I></I></B></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 6: Amplification of <I>OBP2A</I> for insertion into pT25 vector </B>. This is the PCR product from the amplification of <I>OBP2A</I> for cloning into pT25. The gel indicates <I>OBP2A</I> has been amplified successfully since the observed band corresponds to the expected size of 100 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 7: Presequence Digest of pT25-<I>OBP2A<I>, in order to determine if any of the samples contain the <I>OBP2A<I> insert. It can be seen from this gel that samples 5, 6 and 7 may have inserts in them as they each have two distinct bands, one of which corresponds to the 100 bp size for the <I>OBP2A<I> for pT25. As a result all three samples will be sent for sequencing.    </I></I></I></I></I></I></B></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 8: A graph of the results from the first attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of <I>OBP2A<I>. As it can be seen that the positive result has failed(far left) not much can be inferred from the interaction of the two parts of <I>OBP2A<I>. However it looks like the two subunits of <I>OBP2A</I> may not be interacting. As this was a first attempt, subsequent assays will be performed in order to determine if in fact the two separated parts of OBP2A aren't interacting in vivo - like these results suggests.      </I></I></I></I></B></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure11-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 9: A graph of the results from the second attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of <I>OBP2A<I>.  It can also be inferred from this graph that the two parts of <I>OBP2A<I> don't seem to be interacting as it has a low millers activity when compared to the controls. Further assays will be performed in order to determine if the two separated parts of <I>OBP2A</I> aren't interacting in vivo like these results suggests.     </I></I></I></I></B></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure12-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 11: A graph of the results from the anaerobic β-galactosidase assay aimed to characterize the interaction between the two separated parts of <I>OBP2A<I>. The graph seems to suggest that the two subunits of <I>OBP2A</I> may not be interacting in vivo. This can be inferred from the fact that the millers activity of the <I>OBP2A<I> sample lies lower than the negative controls from within the experiment. As a result of this information, one more assay will be prepared in which the two subunits will be switched with regards to the bacterial two hybrid vectors to eliminate any possibility of this being a factor effecting the interaction between the two subunits.     </I></I></I></I></B></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="onc-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Overnight Culture Protocol</DIV></DIV><DIV class="modal-body"><P><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>5ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.</LI><LI>Tubes were incubated at 37<SUP>o</SUP>C in a rotary incubator at 200rpm over night.</LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="PCR-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">PCR Protocol</DIV></DIV><DIV class="modal-body"><P>
    PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the <STRONG>Sequences</STRONG> section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY color="black"><TR><TD width="312" valign="top"><P align="center"><STRONG>Reactants</STRONG></P></TD><TD width="312" valign="top"><P align="center"><STRONG>Volume (µl)</STRONG></P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DNA
                </P></TD><TD width="312" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DMSO
                </P></TD><TD width="312" valign="top"><P align="center">
                    2.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Forward Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Reverse Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase Buffer
                </P></TD><TD width="312" valign="top"><P align="center">
                    10
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    dNTP
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    H<SUB>2</SUB>O
                </P></TD><TD width="312" valign="top"><P align="center">
                    34.5
                </P></TD></TR></TBODY></TABLE><P>
    **PCR Programme**
</P><P>
    Elongation time was altered according to the length of the sequences.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="miniprep-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Miniprep Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>5ml bacterial overnight culture was pelleted by stepwise centrifugation in 1.5ml microcentrifuge tubes at 13300rpm for 3 minutes at room temperature.</LI><LI>Pelleted bacterial cells were resuspended in 250µl of Buffer P1.</LI><LI>250µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution became clear.</LI><LI>350µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.</LI><LI>The mixture was centrifuged for 10 minutes at 13300rpm in a microcentrifuge.</LI><LI>The supernatant was applied to a QIAprep spin column by pipetting.</LI><LI>500µl of Buffer PB was added to the QIAprep spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>750µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>The QIAprep spin column was spun for an additional minute to remove residual wash buffer.</LI><LI>The QIAprep spin column was placed in a clean 1.5ml microcentrifuge and the DNA eluted by adding 30µl of H<SUB>2</SUB>O to the QIAprep spin column and centrifuging for 1 minute. </LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="gelextraction-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Gel Extraction Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.</LI><LI>The gel slice was transferred into a microcentrifuge tube. 700µl of Buffer QG was added.</LI><LI>The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.</LI><LI>After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.</LI><LI>230µl of isopropanol was added to the sample and mixed.</LI><LI>A QIAquick spin column was placed in a 2ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>500µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>750µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>The QIAprep spin column was spun for an additional minute to remove residual wash buffer.</LI><LI>The QIAprep spin column was placed in a clean 1.5ml microcentrifuge and the DNA eluted by adding 30µl of H2O to the QIAprep spin column and centrifuging for 1 minute.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="restrictiondigest-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Restriction Digest Protocol</DIV></DIV><DIV class="modal-body"><P>
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the    <STRONG>Journal</STRONG> section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="170" valign="top"><P><STRONG>Post Plasmid Purification</STRONG></P></TD><TD width="151" valign="top"><P><STRONG>Post PCR</STRONG></P></TD><TD width="139" valign="top"><P><STRONG>gBlock</STRONG></P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>DNA</STRONG></P></TD><TD width="170" valign="top"><P>
                    1mg
                </P></TD><TD width="151" valign="top"><P>                 50µl
                </P></TD><TD width="139" valign="top"><P>
                    100ng
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Cutsmart Buffer (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    6
                </P></TD><TD width="151" valign="top"><P>
                    6
                </P></TD><TD width="139" valign="top"><P>
                    6
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Water (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    As required
                </P></TD><TD width="151" valign="top"><P>
                    1
                </P></TD><TD width="139" valign="top"><P>
                    As required
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Enzyme 1 (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    1.5
                </P></TD><TD width="151" valign="top"><P>
                    1.5
                </P></TD><TD width="139" valign="top"><P>
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Enzyme 2 (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    1.5
                </P></TD><TD width="151" valign="top"><P>
                    1.5
                </P></TD><TD width="139" valign="top"><P>
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG><U>Total Volume (µl)</U></STRONG></P></TD><TD width="170" valign="top"><P>
                    30
                </P></TD><TD width="151" valign="top"><P>
                    60
                </P></TD><TD width="139" valign="top"><P>
                    30
                </P></TD></TR></TBODY></TABLE><P><UL><LI>The digests were incubated at 37<SUP>o</SUP>C for 3 hours.</LI><LI>Note that when performing plasmid digestions, 2.5µl of alkaline phosphatase and 2.5µl of its buffer were added at the 2 hour and 2.5 hour mark.</LI></UL></P></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ligation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Ligation Protocol</DIV></DIV><DIV class="modal-body"><P>
Ligation reactions were set up according to the table below. The vector to insert ratio is stated for each specific reaction in the    <STRONG>Journal</STRONG> section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="111" valign="top"><P align="center"><STRONG>Vector (µl)</STRONG></P></TD><TD width="111" valign="top"><P align="center"><STRONG>2:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>3:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>4:1 (Vector: Insert) [µl]</STRONG></P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>T4 DNA Ligase </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Ligase Buffer </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Vector </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Insert</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    -
                </P></TD><TD width="111" valign="top"><P align="center">
                    2
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>H<SUB>2</SUB>0</STRONG></P></TD><TD width="111" valign="top"><P align="center">                 7
                </P></TD><TD width="111" valign="top"><P align="center">
                    5
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD></TR></TBODY></TABLE><P>
    The reactions were incubated at room temperature for at least 3 hours.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Transformation Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>100µl of competent cells were defrosted. Thaw on ice.</LI><LI>Agar plates were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>1-5µl of DNA was added to cells.</LI><LI>Cell and DNA mixture was left on ice for 20 minutes.</LI><LI>Mixture was placed in 42<SUP>o</SUP>C water bath for 90 seconds.</LI><LI>Mixture was put back on ice for 2 minutes.</LI><LI>1ml of LB (without antibiotic) was added to cells.</LI><LI>Cells were allowed to grow for 1 hour at 37<SUP>o</SUP>C in a shaking incubator.</LI><LI>Cells were then spun down and the supernatant discarded.</LI><LI>The pellet was resuspended in the 100µl.</LI><LI>Cells were plated on agar plates with appropriate antibiotic.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="assaypreparation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Assay Preparation</DIV></DIV><DIV class="modal-body"><OL><LI>Take 3 independent colonies from each plate of your controls and samples and place each colony into 5 ml of LB with the appropriate antibiotics. </LI><LI>Leave to grow overnight at 30°C.</LI><LI>Take 50 μl of each overnight and inoculate 5 ml of fresh LB containing the appropriate antibiotics. Do this twice for every independent colony. </LI><LI>Allow these to grow at 30°C shaker until the O.D of each inoculate is between 0.3-0.5</LI><LI>Once the O.D is between 0.3-0.5 take 3 x 1 ml samples of each culture and place into separate Eppendorf tubes.</LI><LI>Take a further 1 ml and place into a cuvette and  subsequently measure the O.D at 600 nm marking down the O.D for each replica. </LI><LI>Spin down the 1 ml cultures of each replica at 13,000 rpm for 3 minutes. </LI><LI>Remove th supernatant from each 1 ml culture and freeze cell pellet for later use in the β-galactosidase Assay. </LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="β-galactosidaseassay-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"> β-galactosidase Assay    </DIV></DIV><DIV class="modal-body"><P><OL><LI> Resuspend cell pellet in 800ul of buffer 2 (100 mM dibasic sodium phosphate (Na2HPO4); 20 mM KCl ; 2 mM MgSO4 ;0.8 mg/mL CTAB (hexadecyltrimethylammonium bromide) ; 0.4 mg/mL sodiumdeoxycholate) and subsequntly add 5.4 μL/mL beta-mercaptoethanol. </LI><LI> Add 50 ul of SDS (0.1%) to each sample. </LI><LI> Add 50 ul of Chloroform to each sample and vortex for 30 seconds. </LI><LI> Make substrate solution (60 mM Na2HPO4; 40 mM NaH2PO4; 1 mg/mL o-nitrophenyl-β-D-Galactoside (ONPG); 2.7 μL/mL β-mercaptoethanol)</LI><LI> Add 200 ul of substrate solution to each sample marking down the time it was added in relation to the first sample. </LI><LI> Wait for an observable color change and mark down the time that the color change occurred.</LI><LI> Add 600 ul of NaCO3(1M) once the color change has occurred.</LI><LI> Spin down the samples at 13,000 rpm for 3 minutes and place the supernatant into a cuvette </LI><LI> Measure the O.D of each sample at 420 nm </LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV><DIV class="container"><DIV class="footer-btn"><A href="https://2015.igem.org/Team:Dundee/Introduction_Video" class="btn btn-info btn-lg" role="button">Watch Our Introduction Video</A></DIV></DIV><FOOTER><DIV class=""><DIV class="container"><DIV class="col-md-3"><A href="http://www.dundee.ac.uk/" target="_blank"><H4>Dundee University</H4></A><P>Located on the east coast of Scotland, The University of Dundee is one of Scotland's leading research institutes.</P></DIV><DIV class="col-md-3"><H4>Social Media</H4><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="https://www.youtube.com/channel/UCJHBLvLhjzGqHqdjABjOY-w">youtube.com/DundeeiGEM2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.facebook.com/igemdundee2015">facebook.com/iGEMDundee2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.twitter.com/dundeeigemteam">twitter.com/DundeeiGEMTeam</A></DIV></DIV></DIV></DIV></DIV></FOOTER><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>