<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_fmodal skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/fmodal</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="modal fade" id="lbpa-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: <B>Amplification of <I>LbpA</I></B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: LbpA PCR. Lane 1: PCR Product, Lane 2: 1kb Ladder and Marker.</B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: LbpA PCR for Cloning into pQE80-L. Lane 1: LbpA PCR Product, Lane 2: 1kb Ladder and Marker</B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: LbpA Culture Samples. Lane 1: Sample from uninduced culture. Lane 2: Sample from culture induced with 1mM IPTG.</B> It seems that on comparison of the two cultures, that inducing expression of LbpA causes the cells to die given the significant reduction of protein levels visible on the gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: OD<SUB>600</SUB> Readings from Uninduced Culture and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG).</B> The table shows that after being induced with IPTG the cultures stop growing since their OD<SUB>600</SUB> readings are notably lower than that of the uninduced control. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, the induced cultures stop growing when LbpA expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, after reaching an OD<SUB>600</SUB> of 0.6,  the induced cultures stop growing when LbpA starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have dealt with the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: SDS Gel of Samples Taken from Uninduced Control and 1 mM IPTG Induced Culture 6 hours After Induction</B> For the induced culture, a faint band just above the 100 kDa marker is observable on the gel which corresponds to the expected size of LbpA.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 1: Gel of <I>SBP</I> PCR Product for Cloning into pQE80-L. Lane 1: <I>SBP</I> PCR Product 1 Lane 2: 1kb Ladder and Marker Lane 3: <I>SBP</I> PCR Product 2</B> Both bands observable on the gel correspond to the expected size of <I>SBP</I> which is ~600 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: SDS Gel of Uninduced and 1 mM IPTG Induced Cultures of PotD and SBP.</B> For PotD, it is clear from the gel that it is being successfully overexpressed since there is an intense band present at the 37 kDa marker which corresponds to the expected size of PotD in the induced culture sample. For SBP, there is no band present which suggests SBP is being overexpressed. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Characterisation of PotD.</B> Single colonies of MC1061 <I>E.coli</I> containing pQE80-L encoding <I>PotD</I> were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin. Once cultures reached an OD<SUB>600</SUB> of 0.6, one was induced with 1 mM IPTG and one was left uninduced as a control. They were left to grow for a further three hours, after which a 1ml sample was taken and the cells pelleted. The cells were resuspended in 100µl of Laemmli buffer, 20µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: SDS Gel of Uninduced and 0.5mM, 1mM and 2mM IPTG Induced Cultures of SBP.</B> It appears that upon induction of <I>SBP</I> with any concentration of IPTG the cells are dying given that there is a significant drop in protein levels observable on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, the induced cultures stop growing when SBP expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, after reaching an OD<SUB>600</SUB> of 0.6,  the induced cultures stop growing when <I>SBP</I> starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of PotD</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100 V. The two bands observable on the gel correspond to the expected size of PotD of 38 kDa. Western Blotting will confirm if these proteins are PotD. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: Characterisation of PotD following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure9-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 9: Characterisation of SBP following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure10-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 10: Characterisation of PotD/Spermidine Interaction.</B> The graph shows that the change in the tryptophan spectra increases as concentration of spermidine increases. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 1: pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> (respectively) Pre-Sequencing Digest</B> pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> minipreps were digested with EcoRI and PstI to check for presence of each insert. The bands observable on the gel correspond to the expected sizes for the pSB1C3 vector and both inserts.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: Amplification of <I>hHBA</I> and <I>hHBB</I></B><I>hHBA</I> and <I>hHBB</I> were amplified from the pSB1c3-<I>hHBA</I> and pSB1c3-<I>hHBB</I> minipreps obtained yesterday using primers to allow cloning of the genes into pT25 and pUT18 respectively. Although the band is fainter in the case of <I>hHBA</I>, the gel indicates the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Amplification of <I>hHBA</I></B><I>hHBA</I> was amplified from the pSB1C3-<I>hHBA</I> using primers to allow cloning of the genes into pT25. The gel indicates that the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: Amplification of <I>hHBA</I> and <I>hHBB</I></B><I>hHBA</I> and <I>hHBB</I> were amplified from the pSB1c3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> minipreps using primers to allow cloning of the genes into pQE80-L. The gel indicates that the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Pre-Sequence Digest of pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I></B> Both pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I> minipreps were digested with BamHI and KpnI to check for the presence of the inserts. The gel indicates that both inserts have been successfully cloned into pQE80-L. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Overexpression of <I>hHBB</I> and <I>hHBA</I>.</B> Single colonies of M15[pREP4] <I>E.coli</I> containing pQE80-L encoding <I>hHBB</I> and pQE80-L encoding <I>hHBA</I> were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD<SUB>600</SUB> of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of 2x Laemmli buffer, 20 µl of which was separated by SDS-PAGE. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: Characterisation of hHBB.</B> Single colonies of M15[pREP4] <I>E.coli</I> containing pQE80-L encoding <I>hHBB</I> were used to inoculate 5 ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD<SUB>600</SUB> of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: Amplification of <I>hHBN</I> for Cloning into pQE80-L </B> PCR product from amplification of <I>hHBN</I> from IDT plasmid using primers for cloninig into pQE80-L. The gel indicates <I>hHBN</I> has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure9-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 9: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of hHBB</B> 10 µl of each fraction was mixed with 10 µl of laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. The four bands observable on the gel correspond to the expected size of hHBB of 16 kDa. Western Blotting will confirm if these proteins are hHBB. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure10-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 10: Characterisation of hHBB following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure11-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 11: Amplification of <I>hHBN</I> for Cloning into pT25 </B> PCR product from amplification of <I>hHBN</I> from IDT plasmid using primers for cloninig into pT25. The gel indicates <I>hHBN</I> has been amplified successfully since the observed band corresponds to the expected size of 1200 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure14-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 14: Characterisation of hHBN following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure15-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 15: Amplification of <I>hHBN</I></B><I>hHBN</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of <I>hHBN</I> of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure16-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 15: Amplification of <I>hHBA</I></B><I>hHBA</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of <I>hHBA</I> of </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure17-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 16: Amplification of <I>hHBN</I></B><I>hHBN</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pUT18. The band observable on the gel corresponds to the expected size of <I>hHBN</I> of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 1: Amplification of <I>OBP2A</I> for insertion into pSB1C3 vector </B>. This is the PCR product from the amplification of <I>OBP2A</I> from the IDT plasmid using primers for cloning into pSB1C3. The gel indicates <I>OBP2A</I> has been amplified successfully since the observed band corresponds to the expected size of 500 bp. The concentration of the gel extracted <I>OBP2A</I> is 335.53 ng/ul. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 2: A subsequent gel was run of the amplified <I>OBP2A</I> after it was first excised from the gel in <I>Figure 1 </I></B> This was to determine if the gel extraction was successful. The gel indicates that <I>OBP2A</I> has been extracted from the gel successfully, since the observed band corresponds to the expected size of 500 bp. This can now be restricted in preparation for ligation into pSB1C3. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 3: A Gel Analysis of Restriction Digest from the 24/6.</B> This gel analysis was done to determine if the reason for the failed transformation was an improperly digested <I>OBP2A</I>. This gel indicates that the restricted product hasn't been lost as the <I>OBP2A</I> gene fragment can be seen as a distinct band at 500 bp. A re-ligation of <I>OBP2A</I> into the pSB1C3 backbone will now occur. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 4: Presequence Digest of pSB1C3-<I>OBP2A</I></B>. All seven plasmid purifications of the overnight cultures done on the 26/6 were digested to see if they contained the <I>OBP2A</I> insert. It is clear from the gel that all but sample 3 contain the insert for <I>OBP2A</I> as they all have faint bands at 500 bp's which corresponds to the size of <I>OBP2A</I>. It also appears that sample 5 contains the highest concentration, so that will be the sample that is sent for sequencing.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 5: Presequence digest of pUT18-<I>OBP2A</I></B> To determine if any samples contain the <I>OBP2A</I> insert. It can be seen from this gel that of the  5 samples that were digested only two are show to have the <I>OBP2A</I> insert, therefore sample 2 will be sent for sequencing.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 6: Amplification of <I>OBP2A</I> for insertion into pT25 vector </B>. This is the PCR product from the amplification of <I>OBP2A</I> for cloning into pT25. The gel indicates <I>OBP2A</I> has been amplified successfully since the observed band corresponds to the expected size of 100 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 7: Presequence Digest of pT25-<I>OBP2A</I></B> In order to determine if any of the samples contain the <I>OBP2A</I> insert. It can be seen from this gel that samples 5, 6 and 7 may have inserts in them as they each have two distinct bands, one of which corresponds to the 100 bp size for the <I>OBP2A</I> for pT25. As a result all three samples will be sent for sequencing.    </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 8: β-galactosidase Assay</B>  A graph of the results from the first attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>. As it can be seen that the positive result has failed(far left) not much can be inferred from the interaction of the two parts of <I>OBP2A</I>. However it looks like the two subunits of <I>OBP2A</I> may not be interacting. As this was a first attempt, subsequent assays will be performed in order to determine if in fact the two separated parts of OBP2A aren't interacting in vivo - like these results suggests.      </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure11-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 9: β-galactosidase Assay</B> A graph of the results from the second attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>.  It can also be inferred from this graph that the two parts of <I>OBP2A</I> don't seem to be interacting as it has a low millers activity when compared to the controls. Further assays will be performed in order to determine if the two separated parts of <I>OBP2A</I> aren't interacting in vivo like these results suggests.     </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure12-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 11: β-galactosidase Assay Under Anaerobic Conditions </B> A graph of the results from the anaerobic β-galactosidase assay aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>. The graph seems to suggest that the two subunits of <I>OBP2A</I> may not be interacting in vivo. This can be inferred from the fact that the millers activity of the <I>OBP2A</I> sample lies lower than the negative controls from within the experiment. As a result of this information, one more assay will be prepared in which the two subunits will be switched with regards to the bacterial two hybrid vectors to eliminate any possibility of this being a factor effecting the interaction between the two subunits.     </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: <B>Amplification of <I>LS</I></B><I>LS</I> was amplified using primers which would allow for subsequent cloning into the high expression vector pQE80-L. Lane 1 shows that the PCR has been successful since the band corresponds to the expected size of 2100 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: Amplification of <I>LS</I></B><I>LS</I>  was amplified from the IDT plasmid using primers to allow cloning of the gene into pSB1C3. The observed band corresponds to the expected size of LS, 2100 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Overexpression Assays of <I>LS</I></B> 5 ml cultures of M15pREP4 <I>E.coli</I> was grown to OD<SUB>600</SUB>=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37<SUP>o</SUP>C, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: Characterisation of <I>Lanosterol Synthase</I></B> 5 ml cultures of M15pREP4 <I>E.coli</I> was grown to OD<SUB>600</SUB>=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37<SUP>o</SUP>C, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. The proteins were transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Chromatogram following Affinity Chromatography of LS</B> Using FPLC, the protein sample was loaded onto a nickel column and eluted with an increasing concentration of imidazole (green line). The fractions corresponding to the peak were ran on an SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: SDS Gel of Peak Fractions </B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer an loaded onto a SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="BBa_K1058008-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: Alignment of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>. Sequences from top to bottom: sequencing result from forward primer; expected sequence; reverse complement of sequencing result from reverse primer; sequencing result from intermediate primer. Colour code: Green = Prefix/Suffix; Pink = Annealing site of intermediate primer; Yellow = Mismatch in one sequence; Red = In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem022-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 2: 1% agarose gel after PCR of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with primers specific to the promoter region, P<I>chr</I>, and the repressor region, <I>chrB</I>. Left: P<I>chr</I> (expected size 169bp); Middle: 1kb+ ladder, Right: <I>chrB</I> (expected size 939bp). Amplification of <I>chrB</I> was not successful. P<I>chr</I> was further processed by restriction digest with EcoRI and PstI.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem023-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 3: 1% agarose gel after PCR of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with primers specific to the repressor region, <I>chrB</I>. Left: 1.5 µl DMSO; Middle: 1kb+ ladder, Right: 2.5 µl DMSO. Expected size for each: 939bp. The annealing temperature for the PCR amplification was lowered to 50°C in order to increase the chances of accomodating a primer that is not 100% complementary.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem024-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 4: 1% agarose gel after PCR of <I>gfp</I> with corresponding primers. Left: <I>GFP</I> (expected size 717bp); Middle: 1kb+ ladder, Right: PCR product of <I>chrB</I>, using primers to add a BamHI restriction site in front of it and a 6-His-Tag and a HindIII restriction site at the end. (expected size of <I>chrB</I>: 939bp). Since amplification of <I>chrB</I> was unsuccessful, it will be repeated.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem026-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 5: 1% agarose gel after restriction digest of pSB1C3-P<I>chr</I>, pSB1C3-<I>chrB</I> (s), and pSB1C3-<I>chrB</I> (w) with EcoRI and PstI for confirmation of size. Ladder: 1kb+. Top: 1-4: P<I>chr</I> 1-4; <I>chrB</I> (s) 1-2. Bottom: <I>chrB</I> (s) 3-4; <I>chrB</I> (w) 1-4. Expected sizes: P<I>chr</I>: 169bp; <I>chrB</I>: 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-pChr1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 6: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green = Prefix/Suffix
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-ChrBw4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 7: Alignment of <I>chrB</I> (opt) sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green: Prefix and Suffix; Red: In frame stop codons; Blue: First 15 codons optimised for expression in E. coli.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem034-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 8: 1% agarose gel after restriction digest of pUniprom (Expected size 2454bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem035-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 11: 1% agarose gel after restriction digest of pSB1C3 with EcoRI and SpeI (Expected size 2070bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem032-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 9: 1% agarose gel after PCR of <I>chrB</I> (opt). (Expected size 939bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-first-transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 10: Plates after transformation of Top: pUniprom<I>chrB</I>, Middle: pUniprom<I>chrB</I> (opt) and Bottom: pSB1C3-P<I>chr-gfp</I> into JM110.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem036-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 12: 1% agarose gel after restriction digest of pUniprom<I>chrB</I>, pUniprom<I>chrB</I> (opt) and pSB1C3-P<I>chr-gfp</I> for confirmation of size. Ladder: 1kb+. Top: 1-4: pSB1C3-P<I>chr-gfp</I> 1-4; pUniprom<I>chrB</I> (opt) 1-2. Bottom: pUniprom<I>chrB</I> (opt) 3-4; pUniprom<I>chrB</I> 1-4. Expected sizes: P<I>chr-gfp</I>: 909bp; <I>chrB</I> and <I>chrB</I> (opt): 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem037-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 13: 1% agarose gel after colony PCR of pUniprom<I>chrB</I> (opt). Top: colonies 1-11; Bottom: colonies 12-22. Colonies 2, 4, 9, and 12 were selected for sequencing. Ladder: 1kb+ Expected size of <I>chrB</I> (opt): 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pSB1C3-pChr-GFP_reverse" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 14: Sequencing result of pSB1C3-P<I>chr-gfp</I> after double ligation. Even though the sequence is as expected, it is ligated in reverse. It was hence called pSB1C3-P<I>chr-gfp</I> (A) The Result for pSB1C3-P<I>chr-gfp</I> (colony 2) was not as expected, albeit in the correct orientation. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pChrGFPplates_revisited" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 15: Plates with pSB1C3-P<I>chr-gfp</I> in JM110 after double ligation under UV light. Numbered colonies were selected for initial size confirmation and sequencing. However, other colonies seemed to express GFP as well, hence a second attempt was made with different colonies from the same plate.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pChrGFP_revisited_seq" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 16: Alignment of sequencing results of pSB1C3-P<I>chr-gfp</I> - colonies 1, 5, 6, and 7 with sequences of <I>gfp</I> (mut2) and P<I>chr</I> sequence. Colony 1 remained consistent, the other tested colonies were without success. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem039" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 17: 1% Agarose gel after restriction digest of pUniprom with BamHI/PstI (expected size: 2454bp). Right: PCR product of Haptoglobin.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150723_pSB1C3-pChr" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 18: Plates after transformation of P<I>chr</I> into pSB1C3. Top: ratio insert:vector 2:1; Bottom: ratio insert:vector 3:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150727_pSB1C3-pChr_alignment" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 19: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green = Prefix/Suffix
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150728_pUniprom-ChrB" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 20: Plates after transformation of, Top: <I>chrB</I> into pUniprom; Bottom: <I>chrB</I> (opt) into pUniprom. Ratio insert:vector for ligations: left: 3:1; Right: 4:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem040" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 21: Gel after restriction digest of Left: pUniprom-<I>chrB</I>, and Right: pUniprom-<I>chrB</I> (opt) with BamHI/PstI for confirmation of insert size. Expected size 939bp. <I>chrB</I> colony 2 and <I>chrB</I> (opt) colony 3 were selected for sequencing.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150729_chrb-combined" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 22: Alignments of: Left: <I>chrB</I> and Right: <I>chrB</I> (opt). Colours: Pink = Restriction sites (BamHI/PstI); Blue = 6-His Tag; Red = In-frame stop codons; Cyan = optimised codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150731_pChrGFP-ligations" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 23: Plates after ligation of <I>gfp</I> into pSB1C3-P<I>chr</I>. Left: Ratio Insert:Vector 2:1; Right: Ratio Insert:Vector 3:1.
        </DIV></DIV><DIV class="modal-body">' 

   </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem043" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 24: Gel for size confirmation of pSB1C3-P<I>chr-gfp</I> (Top). Bottom: Left - Haptoglobin, Right - Lanosterol Synthase. pSB1C3-P<I>chr-gfp</I> (3) and pSB1C3-P<I>chr-gfp</I> (4) were submitted for sequencing.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150801_pchrgfp" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 25: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> and <I>gfp</I> (mut2) with the forward and reverse sequencing results of P<I>chr-gfp</I> this region. The sequence was as expected, it was hence called pSB1C3-P<I>chr-gfp</I> (B). Colour code: Green = Prefix/Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150809_blot" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 26: Top: anti-GFP: Single colonies of JM110 + pSB1C3-P<I>chr-gfp</I> (A), MC1061 + pSB1C3-P<I>chr-gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> chloramphenicol. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of each sample was mixed with 100 µl laemmli buffer, and boiled for 10min. 3 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and P<I>manA-gfp</I> as a positive control. 
        Bottom: anti-6-His - ChrB: Single colonies of JM110 pUniprom-<I>chrB</I> and JM110 pUniprom-<I>chrB</I> (opt) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 200 µl laemmli buffer, and the sample boiled for 10min. 10 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and <I>hydA - 6-His</I> as a positive control.
        Expected sizes: GFP ~ 27kDa, ChrB ~35kDa.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150811_transformations" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 27: Plates after transformation of single constituents of chromate sensor into <I>E. coli</I> MG1655. From top to bottom: pUniprom-<I>chrB</I> (from colony 2 - 29/07), pUniprom-<I>chrB</I> (opt) (from colony 3 - 29/07), pSB1C3-P<I>chr-gfp</I> (B), pSB1C3-P<I>chr-gfp</I> (A), <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150822_blot" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 28: Western blot against GFP: Single colonies of MG1655, containing all 4 combinations, pUniprom-<I>chrB</I> - pSB1C3-P<I>chr-gfp</I> (A), pUniprom-<I>chrB</I> - pSB1C3-P<I>chr-gfp</I> (B), pUniprom-<I>chrB</I> (opt) - pSB1C3-P<I>chr-gfp</I> (A), and pUniprom-<I>chrB</I> (opt) - pSB1C3-P<I>chr-gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> chloramphenicol and 100 <SUP>µg</SUP>/<SUB>ml</SUB> ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml PBS. 100 µl of these samples was mixed with 100 µl laemmli buffer, and the sample boiled for 10 min. The indicated volume of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3, and pUniprom were included as a negative control, and P<I>manA-gfp</I> as a positive control. The blot against 6-His – ChrB was without success and is not shown. No chromate of any kind was added to the medium previous to this blot, hence no expression of GFP was expected. 
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV>


K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB><DIV class="modal fade" id="dundee15_chr_150824_platereader_BBa_K1058008" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 29: F,OD-plot of results of plate reader experiment after exposing MG1655 + BBa_K1058008 to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="icecream" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 30: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="potato" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 31: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt) to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150824_platereader_pchrbgfp" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 32: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_chrbbielefeld" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 33: Plates after ligation of <I>chrB</I> (Bielefeld) into pT7KS. From top to bottom: Ratio insert:vector 2:1; Ratio Insert:Vector 3:1; pT7KS religation control.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_bl21" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 34: Plates after transformation of single constituents of chromate sensor into <I>BL21 (DE3)</I>. From top to bottom: pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt), pUniprom-<I>chrB</I>, and pUniprom-<I>chrB</I> (opt).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_igem047" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 35: 1% agarose gel after colony PCR of pT7KS-<I>chrB</I> (Bielefeld). Top: colonies 1-6; Bottom: colonies 7-12. Ladder: 1kb+ Expected size of <I>chrB</I> (Bielefeld): 939bp
  </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_platereaderinduced" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 36: F,OD-plot of results of plate reader experiment after inducing expression of ChrB from the T7 promoter of pUniprom <I>BL21 (DE3)</I> pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, and pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt). IPTG was added after 4 hours. One measurement was made every 10min across 22h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>