Team:Dundee/Lab Book/Fingerprints
×
PCR Protocol
PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the
section.
Reactants
Volume (µl)
DNA
1
DMSO
2.5
100 µM Forward Primer
0.5
100 µM Reverse Primer
0.5
5 x Herculase Buffer
10
10 mM dNTPs
0.5
Herculase
0.5
H2O
34.5
Elongation time was altered according to the length of the sequences.
Close
×
Restriction Digests Protocol
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the Lab Book section.
Post Plasmid Purification
Post PCR
gBlock
DNA
1 mg
50 µl
100 ng
Cutsmart Buffer (µl)
6
6
6
Water (µl)
As required
1
As required
Enzyme 1 (µl)
1.5
1.5
1.5
Enzyme 2 (µl)
1.5
1.5
1.5
Total Volume (µl)
30
60
30
The digests were incubated at 37oC for 3 hours.
Note that when performing plasmid digestions, 2.5 µl of alkaline phosphatase and 2.5 µl of Cutsmart Buffer was added at the 2 hour and 2.5 hour mark. Digested PCR products were separated from restriction enzymes and buffers using Strataclean resin (Aligent Technologies). Digested vectors were purified on a 1% agarose gel and then extracted.
Close
×
Ligations
Ligation reactions were set up according to the table below.
Vector (µl)
2:1 (Vector: Insert) [µl]
3:1 (Vector: Insert) [µl]
4:1 (Vector: Insert) [µl]
T4 DNA Ligase
1
1
1
1
Ligase Buffer
1
1
1
1
Vector
1
1
1
1
Insert
-
2
3
4
H20
7
5
4
3
The reactions were incubated at room temperature for at least 3 hours.
Close
×
Transformation Protocol
1. 100 µl of chemocompetent E.coli cells were defrosted. Thaw on ice.
2. Agar plates were taken out of 4 oC to warm to room temperature.
3. 10 µl of ligation reaction or 2 µl of plasmid purified DNA was added to cells. The cells and DNA were mixed by gently flicking the tube.
4. Cell and DNA mixture was left on ice for 20 minutes.
5. Mixture was placed in 42 oC water bath for 90 seconds.
6. Mixture was put back on ice for 2 minutes.
7. 1 ml of LB (without antibiotic) was added to cells.
8. Cells were allowed to grow for 1 hour at 37 oC in a shaking incubator.
9. Cells were then spun down and the supernatant discarded.
10. The pellet was resuspended in the 100 µl of remaining media.
11. Cells were plated on agar plates with appropriate antibiotic.
Close
×
Plasmid Purification
1. 1-5 ml bacterial overnight culture was pelleted by centrifugation at >8000 rpm for 3 minutes at room temperature.
2. Pelleted bacterial cells were resuspended in 250 µl of Buffer P1 and transferred to a microcentrifuge tube.
3. 250 µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution becomes clear.
4. 350 µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.
5. The mixture was centrifuged for 10 minutes at 13000 rpm in a microcentrifuge.
6. The supernatant was applied to a QIAprep spin column by pipetting.
7. The QIAprep spin column was washed with 500 µl of Buffer PB by centrifugation for 1 minute and flow through discarded.
8. 750µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute and flow through discarded.
9. The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
10. The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 50 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.
Close
×
Overnight Cultures
1. Four colonies were picked from the desired plate.
2. For each colony, 5 ml of fresh LB plus the appropriate antibiotics was inoculated and grown overnight at 37 oC.
Close
×
SDS-PAGE and Western Blotting
1. 50 µl of an overnight culture was used to inoculate 5 ml of fresh LB containing the appropriate antibiotics and left to grow at 37oC in a     shaking incubator.
2. Once the OD600=0.6-0.8, 1 mM of IPTG was added to the culture and left to grow for a further 3 hours.
3. 1 ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.
4. Meanwhile, 950 µl of 2x Laemmli sample buffer was added to 50 µl of B-mercaptoethanol.
5. The supernatant was discarded from the samples and the pellet resuspended in 200 µl of the above solution.
6. The samples were then boiled at 90oC for 10 minutes.
7. Afterwards, they were spun down for 1 minute in a microcentrifuge.
8. At this stage, the samples could be stored at -20oC or ran on a gel immediately.
9. Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the     stacking gel at which point the voltage was increased to 200V for 20 minutes.
10. One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.
11. After the proteins had been transferred onto the membrane using a semi dry transfer ran for 40 minutes at 5V, the membrane was blocked in 5% Marvel milk overnight at 4oC.
12. The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10 ml of 1xTBS buffer to which the primary anti-His antibody was     added and left for one hour. The protein ladder was cut off before adding the primary antibody.
13. The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10 ml of 1xTBS buffer, to which the secondary antibody was     added and left for one hour.
14. The membrane was then washed in 3x 10 ml 1xTBST buffer for 5 minutes.
15. The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off     at step 12.
16. Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.
Close
×
Growth Curve Assay Protocol
1. 5ml overnight cultures of the appropriate strains were set up.
2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
3. Three were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced.
4. 200 µl of each culture was loaded onto a 96 well microtitre plate with 4 repeats.
5. The cultures were left to grow for 16 hours in the plate reader which took OD readings every 10 minutes.
Close
×
Growth Curve Assay Protocol
1. 5ml overnight cultures of the appropriate strains were set up.
2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
3. 20 x 200 µl aliquots were loaded onto a 96 well microtitre plate.
4. After the OD600=~0.6, three cultures were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced (with 4 repeats).
5. The cultures were left to grow for 20 hours in the plate reader which took OD readings every 30 minutes.
Close
×
Protein Purification Protocol
1. A 150 ml overnight culture was set up for the desired protein.
2. 3 x 1 L of fresh LB containing 1 ml of appropriate antibiotics was inoculated with each 50 ml overnight culture and left to grow until an OD600 =0.6-1 in a shaking incubator at 37oC.
3. The cultures were subsequently induced with 1 mM IPTG and left to grow for 6 hours in a shaking incubator at 20oC.
4. They were then centrifuged for 20 minutes at 4200 rpm at 4oC.
5. The pellets were then resuspended in 5 ml of ice cold 50 mM 7.5 Tris and transferred to a 50 ml falcon tube and then topped up to 40 ml with 50 mM 7.5 Tris.
6. They were then spun down again at 4000 rpm for 15 minutes at 4oC, the supernatant discarded and pellets stored at -20oC.
7. Once purification could proceed, the pellets were thawed on ice and resuspended in 40 ml of 50 mM Tris containing DNAase, 1 x protease inhibitor tablet     and lysozyme. (Note: If purifying a membrane protein, 1% DDM and 5 M urea was also added).
8. The cells were lysed using a cell disruptor.
9. Afterwards the lysate was spun down in a J25.50 rotor at 15000 rpm at 4oC for 40 minutes.
10. The supernatant was removed and transferred to a 50 ml falcon tube. The pellet was kept for testing. Both were kept on ice.
11. Using FPLC, the supernatant was loaded onto a Ni2+ column.
12. The protein was eluted using an increasing concentration of imidazole. The two buffers used were Buffer A: 50 mM Tris, 200 mM NaCl, 20 mM imidazole and     Buffer B: 50 mM Tris, 200 mM NaCl, 1 M Imidazole. (Note: If purifying a membrane protein 0.02% DDM was added to these buffers.)
13. The fractions corresponding to any peaks were retained and concentrated using an appropriate filter size. A small amount of each fraction could be ran     on an SDS gel at this stage.
14. Using FPLC, the sample was then loaded onto a SEC column and any fractions corresponding to a peak were ran on an SDS gel to check for the presence of     the desired protein.
15. Fractions containing the desired protein were collated, to which 20% glycerol was added.
16. The samples were then flash frozen and stored at -80oC.
Close
×
Assay Preparation
Take 3 independent colonies from each plate of your controls and samples and place each colony into 5 ml of LB with the appropriate antibiotics.
Leave to grow overnight at 30°C.
Take 50 μl of each overnight and inoculate 5 ml of fresh LB containing the appropriate antibiotics. Do this twice for every independent colony.
Allow these to grow at 30°C shaker until the O.D of each inoculate is between 0.3-0.5
Once the O.D is between 0.3-0.5 take 3 x 1 ml samples of each culture and place into separate Eppendorf tubes.
Take a further 1 ml and place into a cuvette and  subsequently measure the OD600 marking down the OD for each replica.
Spin down the 1 ml cultures of each replica at 13,000 rpm for 3 minutes.
Remove the supernatant from each 1 ml culture and freeze cell pellet for later use in the β-galactosidase Assay.
Close
×
β-Galactosidase Assay
Resuspend cell pellet in 800 μl of buffer 2 (100 mM dibasic sodium phosphate (Na2HPO4); 20 mM KCl ; 2 mM MgSO4 ;0.8 mg/mL CTAB (hexadecyltrimethylammonium bromide) ; 0.4 mg/mL sodiumdeoxycholate) and subsequntly add 5.4 μL/mL beta-mercaptoethanol.
Add 50 μl of SDS (0.1%) to each sample.
Add 50 μl of Chloroform to each sample and vortex for 30 seconds.
Make substrate solution (60 mM Na2HPO4; 40 mM NaH2PO4; 1 mg/mL o-nitrophenyl-β-D-Galactoside (ONPG); 2.7 μL/mL β-mercaptoethanol)
Add 200 μl of substrate solution to each sample marking down the time it was added in relation to the first sample.
Wait for an observable color change and mark down the time that the color change occurred.
Add 600 μl of NaCO3(1M) once the color change has occurred.
Spin down the samples at 13,000 rpm for 3 minutes and place the supernatant into a cuvette.
Measure the OD420.
Close
×
Plate Reader Protocol
A 5 ml overnight culture of the desired strain was grown at 37 oC.
Equalise OD600 if required. Dilute culture 1:1000.
Aliquot 198 µl into the required amount of wells in a 96-well plate.
Add additional substances and controls as required.
Preheat platereader to required temperature and start experiment.
Retrieve and analyse data.
Close
×
Competent Cells Protocol
A 5 ml overnight culture of the desired strain was grown at 37oC.
Subsample 1/10 of the volume of the subculture. I.e. 1 ml of the overnight culture into 10 ml fresh LB.
Grow subculture at at 37 oC for 2 h.
Spin down subculture at 4000 rpm at 4oC for 10 min.
Resuspend the pellet in ice cold transformation buffer. Use 1/10 of the initial volume of the subculture.
Use 100 µl per transformation.
Close
×
Plating Protocol
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
Fresh agar plates were inoculated with cells on the toothpick.
Agar plates were incubated at 4oC over night.
Close
×
Tryptophan Fluorescence Protocol
1. Intensity of tryptophan fluorescence of PotD was measured on a fluorimeter using an excitation of 280 nm and emission of 350 nm.
2. Increasing concentrations of spermidine were added in 50 µM increments to give a final concentration of 250 µM spermidine to 5 µM of purified PotD.
Close
×
Sequencing Protocol
For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
Volume of sample equivalent to 550ng DNA
2 µl of 3 mM primer
Remaining volume of water for a total volume of 30 µl
The samples were submitted to DNA sequencing and services .
Close
×
Overnight Culture Protocol
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
5 ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.
Tubes were incubated at 37oC in a rotary incubator at 200rpm over night.
Close
×
Gel Extraction Protocol
The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.
The gel slice was transferred into a microcentrifuge tube. 700 µl of Buffer QG was added.
The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.
After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.
230 µl of isopropanol was added to the sample and mixed.
A QIAquick spin column was placed in a 2 ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
500 µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
750 µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.
Close
×
Figure 1: Amplification of LbpA The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 2: LbpA PCR. Lane 1: PCR Product, Lane 2: 1kb Ladder and Marker. The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 3: LbpA PCR for Cloning into pQE80-L. Lane 1: LbpA PCR Product, Lane 2: 1kb Ladder and Marker The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.
Close
×
Figure 4: LbpA Culture Samples. Lane 1: Sample from uninduced culture. Lane 2: Sample from culture induced with 1mM IPTG. It seems that on comparison of the two cultures, that inducing expression of LbpA causes the cells to die given the significant reduction of protein levels visible on the gel.
Close
×
Figure 5: OD600 Readings from Uninduced Culture and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG). The table shows that after being induced with IPTG the cultures stop growing since their OD600 readings are notably lower than that of the uninduced control.
Close
×
Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, the induced cultures stop growing when LbpA expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 7: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, after reaching an OD600 of 0.6,  the induced cultures stop growing when LbpA starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have dealt with the production of the foreign protein.
Close
×
Figure 8: SDS Gel of Samples Taken from Uninduced Control and 1 mM IPTG Induced Culture 6 hours After Induction For the induced culture, a faint band just above the 100 kDa marker is observable on the gel which corresponds to the expected size of LbpA.
Close
×
Figure 1: Gel of SBP PCR Product for Cloning into pQE80-L. Lane 1: SBP PCR Product 1 Lane 2: 1kb Ladder and Marker Lane 3: SBP PCR Product 2 Both bands observable on the gel correspond to the expected size of SBP which is ~600 bp.
Close
×
Figure 2: SDS Gel of Uninduced and 1 mM IPTG Induced Cultures of PotD and SBP. For PotD, it is clear from the gel that it is being successfully overexpressed since there is an intense band present at the 37 kDa marker which corresponds to the expected size of PotD in the induced culture sample. For SBP, there is no band present which suggests SBP is being overexpressed.
Close
×
Figure 3: Characterisation of PotD. Single colonies of MC1061 E.coli containing pQE80-L encoding PotD were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin. Once cultures reached an OD600 of 0.6, one was induced with 1 mM IPTG and one was left uninduced as a control. They were left to grow for a further three hours, after which a 1ml sample was taken and the cells pelleted. The cells were resuspended in 100µl of Laemmli buffer, 20µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 4: SDS Gel of Uninduced and 0.5mM, 1mM and 2mM IPTG Induced Cultures of SBP. It appears that upon induction of SBP with any concentration of IPTG the cells are dying given that there is a significant drop in protein levels observable on the gel.
Close
×
Figure 5: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, the induced cultures stop growing when SBP expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG) The graph shows that on comparison with the uninduced control, after reaching an OD600 of 0.6,  the induced cultures stop growing when SBP starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.
Close
×
Figure 7: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of PotD 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100 V. The two bands observable on the gel correspond to the expected size of PotD of 38 kDa. Western Blotting will confirm if these proteins are PotD.
Close
×
Figure 8: Characterisation of PotD following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 9: Characterisation of SBP following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 10: Characterisation of PotD/Spermidine Interaction. The graph shows that the change in the tryptophan spectra increases as concentration of spermidine increases.
Close
×
Figure 1: pSB1C3-hHBA and pSB1C3-hHBB (respectively) Pre-Sequencing Digest pSB1C3-hHBA and pSB1C3-hHBB minipreps were digested with EcoRI and PstI to check for presence of each insert. The bands observable on the gel correspond to the expected sizes for the pSB1C3 vector and both inserts.
Close
×
Figure 2: Amplification of hHBA and hHBB hHBA and hHBB were amplified from the pSB1c3-hHBA and pSB1c3-hHBB minipreps obtained yesterday using primers to allow cloning of the genes into pT25 and pUT18 respectively. Although the band is fainter in the case of hHBA, the gel indicates the PCR has been successful.
Close
×
Figure 3: Amplification of hHBA hHBA was amplified from the pSB1C3-hHBA using primers to allow cloning of the genes into pT25. The gel indicates that the PCR has been successful.
Close
×
Figure 4: Amplification of hHBA and hHBB hHBA and hHBB were amplified from the pSB1c3-hHBA and pSB1C3-hHBB minipreps using primers to allow cloning of the genes into pQE80-L. The gel indicates that the PCR has been successful.
Close
×
Figure 5: Pre-Sequence Digest of pQE80-L-hHBA and pQE80-L-hHBB Both pQE80-L-hHBA and pQE80-L-hHBB minipreps were digested with BamHI and KpnI to check for the presence of the inserts. The gel indicates that both inserts have been successfully cloned into pQE80-L.
Close
×
Overexpression of hHBB and hHBA. Single colonies of M15[pREP4] E.coli containing pQE80-L encoding hHBB and pQE80-L encoding hHBA were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD600 of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of 2x Laemmli buffer, 20 µl of which was separated by SDS-PAGE.
Close
×
Figure 7: Characterisation of hHBB. Single colonies of M15[pREP4] E.coli containing pQE80-L encoding hHBB were used to inoculate 5 ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD600 of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 8: Amplification of hHBN for Cloning into pQE80-L PCR product from amplification of hHBN from IDT plasmid using primers for cloninig into pQE80-L. The gel indicates hHBN has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.
Close
×
Figure 9: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of hHBB 10 µl of each fraction was mixed with 10 µl of laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. The four bands observable on the gel correspond to the expected size of hHBB of 16 kDa. Western Blotting will confirm if these proteins are hHBB.
Close
×
Figure 10: Characterisation of hHBB following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 11: Amplification of hHBN for Cloning into pT25 PCR product from amplification of hHBN from IDT plasmid using primers for cloninig into pT25. The gel indicates hHBN has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.
Close
×
Figure 14: Characterisation of hHBN following Purification. 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 15: Amplification of hHBN hHBN was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of hHBN of 1200 bp.
Close
×
Figure 15: Amplification of hHBA hHBA was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of hHBA of
Close
×
Figure 16: Amplification of hHBN hHBN was amplified using primers to allow for subsequent cloning into the two hybrid vector pUT18. The band observable on the gel corresponds to the expected size of hHBN of 1200 bp.
Close
×
Figure 1: Amplification of OBP2A for insertion into pSB1C3 vector . This is the PCR product from the amplification of OBP2A from the IDT plasmid using primers for cloning into pSB1C3. The gel indicates OBP2A has been amplified successfully since the observed band corresponds to the expected size of 500 bp. The concentration of the gel extracted OBP2A is 335.53 ng/ul.
Close
×
Figure 2: A subsequent gel was run of the amplified OBP2A after it was first excised from the gel in Figure 1 This was to determine if the gel extraction was successful. The gel indicates that OBP2A has been extracted from the gel successfully, since the observed band corresponds to the expected size of 500 bp. This can now be restricted in preparation for ligation into pSB1C3.
Close
×
Figure 3: A Gel Analysis of Restriction Digest from the 24/6. This gel analysis was done to determine if the reason for the failed transformation was an improperly digested OBP2A. This gel indicates that the restricted product hasn't been lost as the OBP2A gene fragment can be seen as a distinct band at 500 bp. A re-ligation of OBP2A into the pSB1C3 backbone will now occur.
Close
×
Figure 4: Presequence Digest of pSB1C3-OBP2A. All seven plasmid purifications of the overnight cultures done on the 26/6 were digested to see if they contained the OBP2A insert. It is clear from the gel that all but sample 3 contain the insert for OBP2A as they all have faint bands at 500 bp's which corresponds to the size of OBP2A. It also appears that sample 5 contains the highest concentration, so that will be the sample that is sent for sequencing.
Close
×
Figure 5: Presequence digest of pUT18-OBP2A To determine if any samples contain the OBP2A insert. It can be seen from this gel that of the  5 samples that were digested only two are show to have the OBP2A insert, therefore sample 2 will be sent for sequencing.
Close
×
Figure 6: Amplification of OBP2A for insertion into pT25 vector . This is the PCR product from the amplification of OBP2A for cloning into pT25. The gel indicates OBP2A has been amplified successfully since the observed band corresponds to the expected size of 100 bp.
Close
×
Figure 7: Presequence Digest of pT25-OBP2A In order to determine if any of the samples contain the OBP2A insert. It can be seen from this gel that samples 5, 6 and 7 may have inserts in them as they each have two distinct bands, one of which corresponds to the 100 bp size for the OBP2A for pT25. As a result all three samples will be sent for sequencing.
Close
×
Figure 8: β-galactosidase Assay A graph of the results from the first attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of OBP2A. As it can be seen that the positive result has failed(far left) not much can be inferred from the interaction of the two parts of OBP2A. However it looks like the two subunits of OBP2A may not be interacting. As this was a first attempt, subsequent assays will be performed in order to determine if in fact the two separated parts of OBP2A aren't interacting in vivo - like these results suggests.
Close
×
Figure 9: β-galactosidase Assay A graph of the results from the second attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of OBP2A.  It can also be inferred from this graph that the two parts of OBP2A don't seem to be interacting as it has a low millers activity when compared to the controls. Further assays will be performed in order to determine if the two separated parts of OBP2A aren't interacting in vivo like these results suggests.
Close
×
Figure 11: β-galactosidase Assay Under Anaerobic Conditions A graph of the results from the anaerobic β-galactosidase assay aimed to characterize the interaction between the two separated parts of OBP2A. The graph seems to suggest that the two subunits of OBP2A may not be interacting in vivo. This can be inferred from the fact that the millers activity of the OBP2A sample lies lower than the negative controls from within the experiment. As a result of this information, one more assay will be prepared in which the two subunits will be switched with regards to the bacterial two hybrid vectors to eliminate any possibility of this being a factor effecting the interaction between the two subunits.
Close
×
Figure 1: Amplification of LS LS was amplified using primers which would allow for subsequent cloning into the high expression vector pQE80-L. Lane 1 shows that the PCR has been successful since the band corresponds to the expected size of 2100 bp.
Close
×
Figure 2: Amplification of LS LS was amplified from the IDT plasmid using primers to allow cloning of the gene into pSB1C3. The observed band corresponds to the expected size of LS, 2100 bp.
Close
×
Figure 3: Overexpression Assays of LS 5 ml cultures of M15pREP4 E.coli was grown to OD600=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37oC, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel.
Close
×
Figure 4: Characterisation of Lanosterol Synthase 5 ml cultures of M15pREP4 E.coli was grown to OD600=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37oC, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. The proteins were transferred to a nitrocellulose membrane and probed with an anti-His antibody.
Close
×
Figure 5: Chromatogram following Affinity Chromatography of LS Using FPLC, the protein sample was loaded onto a nickel column and eluted with an increasing concentration of imidazole (green line). The fractions corresponding to the peak were ran on an SDS gel.
Close
×
Figure 6: SDS Gel of Peak Fractions 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer an loaded onto a SDS gel.
Close
×
Figure 1: Alignment of BBa_K1058008 . Sequences from top to bottom: sequencing result from forward primer; expected sequence; reverse complement of sequencing result from reverse primer; sequencing result from intermediate primer. Colour code: Green = Prefix/Suffix; Pink = Annealing site of intermediate primer; Yellow = Mismatch in one sequence; Red = In frame stop codons.
'
Close
×
Figure 2: 1% agarose gel after PCR of BBa_K1058008 with primers specific to the promoter region, Pchr, and the repressor region, chrB. Left: Pchr (expected size 169bp); Middle: 1kb+ ladder, Right: chrB (expected size 939bp). Amplification of chrB was not successful. Pchr was further processed by restriction digest with EcoRI and PstI.
'
Close
×
Figure 3: 1% agarose gel after PCR of BBa_K1058008 with primers specific to the repressor region, chrB. Left: 1.5 µl DMSO; Middle: 1kb+ ladder, Right: 2.5 µl DMSO. Expected size for each: 939bp. The annealing temperature for the PCR amplification was lowered to 50°C in order to increase the chances of accomodating a primer that is not 100% complementary.
'
Close
×
Figure 4: 1% agarose gel after PCR of gfp with corresponding primers. Left: GFP (expected size 717bp); Middle: 1kb+ ladder, Right: PCR product of chrB, using primers to add a BamHI restriction site in front of it and a 6-His-Tag and a HindIII restriction site at the end. (expected size of chrB: 939bp). Since amplification of chrB was unsuccessful, it will be repeated.
'
Close
×
Figure 5: 1% agarose gel after restriction digest of pSB1C3-Pchr, pSB1C3-chrB (s), and pSB1C3-chrB (w) with EcoRI and PstI for confirmation of size. Ladder: 1kb+. Top: 1-4: Pchr 1-4; chrB (s) 1-2. Bottom: chrB (s) 3-4; chrB (w) 1-4. Expected sizes: Pchr: 169bp; chrB: 939bp.
'
Close
×
Figure 6: Alignment of Pchr sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green = Prefix/Suffix
'
Close
×
Figure 7: Alignment of chrB (opt) sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green: Prefix and Suffix; Red: In frame stop codons; Blue: First 15 codons optimised for expression in E. coli.
'
Close
×
Figure 8: 1% agarose gel after restriction digest of pUniprom (Expected size 2454bp).
'
Close
×
Figure 11: 1% agarose gel after restriction digest of pSB1C3 with EcoRI and SpeI (Expected size 2070bp).
'
Close
×
Figure 9: 1% agarose gel after PCR of chrB (opt). (Expected size 939bp).
'
Close
×
Figure 10: Plates after transformation of Top: pUnipromchrB, Middle: pUnipromchrB (opt) and Bottom: pSB1C3-Pchr-gfp into JM110.
'
Close
×
Figure 12: 1% agarose gel after restriction digest of pUnipromchrB, pUnipromchrB (opt) and pSB1C3-Pchr-gfp for confirmation of size. Ladder: 1kb+. Top: 1-4: pSB1C3-Pchr-gfp 1-4; pUnipromchrB (opt) 1-2. Bottom: pUnipromchrB (opt) 3-4; pUnipromchrB 1-4. Expected sizes: Pchr-gfp: 909bp; chrB and chrB (opt): 939bp.
'
Close
×
Figure 13: 1% agarose gel after colony PCR of pUnipromchrB (opt). Top: colonies 1-11; Bottom: colonies 12-22. Colonies 2, 4, 9, and 12 were selected for sequencing. Ladder: 1kb+ Expected size of chrB (opt): 939bp.
'
Close
×
Figure 14: Sequencing result of pSB1C3-Pchr-gfp after double ligation. Even though the sequence is as expected, it is ligated in reverse. It was hence called pSB1C3-Pchr-gfp (A) The Result for pSB1C3-Pchr-gfp (colony 2) was not as expected, albeit in the correct orientation. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 15: Plates with pSB1C3-Pchr-gfp in JM110 after double ligation under UV light. Numbered colonies were selected for initial size confirmation and sequencing. However, other colonies seemed to express GFP as well, hence a second attempt was made with different colonies from the same plate.
'
Close
×
Figure 16: Alignment of sequencing results of pSB1C3-Pchr-gfp - colonies 1, 5, 6, and 7 with sequences of gfp (mut2) and Pchr sequence. Colony 1 remained consistent, the other tested colonies were without success. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 17: 1% Agarose gel after restriction digest of pUniprom with BamHI/PstI (expected size: 2454bp). Right: PCR product of Haptoglobin.
'
Close
×
Figure 18: Plates after transformation of Pchr into pSB1C3. Top: ratio insert:vector 2:1; Bottom: ratio insert:vector 3:1.
'
Close
×
Figure 19: Alignment of Pchr sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green = Prefix/Suffix
'
Close
×
Figure 20: Plates after transformation of, Top: chrB into pUniprom; Bottom: chrB (opt) into pUniprom. Ratio insert:vector for ligations: left: 3:1; Right: 4:1.
'
Close
×
Figure 21: Gel after restriction digest of Left: pUniprom-chrB, and Right: pUniprom-chrB (opt) with BamHI/PstI for confirmation of insert size. Expected size 939bp. chrB colony 2 and chrB (opt) colony 3 were selected for sequencing.
'
Close
×
Figure 22: Alignments of: Left: chrB and Right: chrB (opt). Colours: Pink = Restriction sites (BamHI/PstI); Blue = 6-His Tag; Red = In-frame stop codons; Cyan = optimised codons.
'
Close
×
Figure 23: Plates after ligation of gfp into pSB1C3-Pchr. Left: Ratio Insert:Vector 2:1; Right: Ratio Insert:Vector 3:1.
'
Close
×
Figure 24: Gel for size confirmation of pSB1C3-Pchr-gfp (Top). Bottom: Left - Haptoglobin, Right - Lanosterol Synthase. pSB1C3-Pchr-gfp (3) and pSB1C3-Pchr-gfp (4) were submitted for sequencing.
'
Close
×
Figure 25: Alignment of Pchr sequence retrieved from BBa_K1058008 and gfp (mut2) with the forward and reverse sequencing results of Pchr-gfp this region. The sequence was as expected, it was hence called pSB1C3-Pchr-gfp (B). Colour code: Green = Prefix/Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 26: Top: anti-GFP: Single colonies of JM110 + pSB1C3-Pchr-gfp (A), MC1061 + pSB1C3-Pchr-gfp (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of each sample was mixed with 100 µl laemmli buffer, and boiled for 10min. 3 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and PmanA-gfp as a positive control.          Bottom: anti-6-His - ChrB: Single colonies of JM110 pUniprom-chrB and JM110 pUniprom-chrB (opt) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 200 µl laemmli buffer, and the sample boiled for 10min. 10 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and hydA - 6-His as a positive control.         Expected sizes: GFP ~ 27kDa, ChrB ~35kDa.
'
Close
×
Figure 27: Plates after transformation of single constituents of chromate sensor into E. coli MG1655. From top to bottom: pUniprom-chrB (from colony 2 - 29/07), pUniprom-chrB (opt) (from colony 3 - 29/07), pSB1C3-Pchr-gfp (B), pSB1C3-Pchr-gfp (A), BBa_K1058008 .
'
Close
×
Figure 28: Western blot against GFP: Single colonies of MG1655, containing all 4 combinations, pUniprom-chrB - pSB1C3-Pchr-gfp (A), pUniprom-chrB - pSB1C3-Pchr-gfp (B), pUniprom-chrB (opt) - pSB1C3-Pchr-gfp (A), and pUniprom-chrB (opt) - pSB1C3-Pchr-gfp (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol and 100 µg/ml ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml PBS. 100 µl of these samples was mixed with 100 µl laemmli buffer, and the sample boiled for 10 min. The indicated volume of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3, and pUniprom were included as a negative control, and PmanA-gfp as a positive control. The blot against 6-His – ChrB was without success and is not shown. No chromate of any kind was added to the medium previous to this blot, hence no expression of GFP was expected.
'
Close
K2CrO4, K2Cr2O7, CrCl3, and K2SO4
×
Figure 29: F,OD-plot of results of plate reader experiment after exposing MG1655 + BBa_K1058008 to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 30: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 31: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt) to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 32: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 33: Plates after ligation of chrB (Bielefeld) into pT7KS. From top to bottom: Ratio insert:vector 2:1; Ratio Insert:Vector 3:1; pT7KS religation control.
'
Close
×
Figure 34: Plates after transformation of single constituents of chromate sensor into BL21 (DE3). From top to bottom: pSB1C3-Pchr-gfp (B) + pUniprom-chrB, pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt), pUniprom-chrB, and pUniprom-chrB (opt).
'
Close
×
Figure 35: 1% agarose gel after colony PCR of pT7KS-chrB (Bielefeld). Top: colonies 1-6; Bottom: colonies 7-12. Ladder: 1kb+ Expected size of chrB (Bielefeld): 939bp
'
Close
×
Figure 36: F,OD-plot of results of plate reader experiment after inducing expression of ChrB from the T7 promoter of pUniprom BL21 (DE3) pSB1C3-Pchr-gfp (B) + pUniprom-chrB, and pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt). IPTG was added after 4 hours. One measurement was made every 10min across 22h.
'
Close
  University of Dundee
 Home
 About Us
Meet the Team
Attributions
 Project
Abstract
Project Overview
Devices
FluID
Fingerprint Ageing
Chromate Biosensor
Parts
Medals
Future Work
 Wet Lab
Lab Book
FluID
Fingerprints
Chromate Biosensor
Useful Information
Protocols
Sequences
Bacterial Two Hybrid
Safety
 Dry Lab
Introduction
FluID
Fingerprint Ageing
Chromate Biosensor
Appendix : Code
 Outreach
Practices
Ethics
Collaborations
Fingerprint Ageing
Lab Book
FluID
FluID is a cell free, all in one spray which will be used to detect and dinstiguish between body fluids found at crime scenes.
Chromate Biosensor
Our chromate biosensor is a device that will be used to detect stainless steel residues left behind on bones in blunt or sharp force trauma cases.
Fingerprint Ageing
This device will search for the presence of a specifc component found in fingerprint deposits whose presence/absence will indicate approximate fingerprint age.
Week Beginning 24/8/2015
Summary
Cloning of LS into the high expression vector pQE80-L was started this week.
26/8: Amplification of LS for Cloning into pQE80-L
Aim of Experiment: To amplify LS for subsequent cloning into pQE80-L.
Protocols Used: PCR
Results: Figure 1
Next Steps: The gel extracted PCR product will be digested and ligated into pQE80-L tomorrow.
27/8: Restriction Digests and Ligations of LS with pQE80-L
Aim of experiment: To set digest LS with BamHI and KpnI and set up ligations with the high expression vector pQE80-L.
Protocols Used: Restriction Digests Ligations
Results: N/A
Next Steps: Ligations will be left in the cold room over the weekend and transformed into M15pREP4 E.coli on Monday.
Week Beginning 31/8/2015
Summary
Cloning of LS into pQE80-L was completed this week and cloning of LS into pSB1C3 was begun.
31/8: Transformations of pQE80-L-LS into M15pREP4 E.coli
Aim of experiment: To set up transformations of pQE80-L-LS into M15pREP4 E.coli
Protocols Used: Transformations
Results: N/A
Next Steps: If the transformations are successful, colonies will be selected for overnight cultures tomorrow.
1/9: Overnight Cultures of M15pREP4 E.coli containing pQE80-L-LS
Aim of experiment: To set up overnight cultures of M15pREP4 E.coli containing pQE80-L-LS.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The overnight cultures will be miniprepped tomorrow.
2/9: Plasmid Purification of M15pREP4 E.coli Overnight Cultures containing pQE80-L-LS
Aim of experiment: To miniprep the M15pREP4 E.coli Overnight Cultures containing pQE80-L-LS.
Protocols Used: Plasmid Purification
Results: N/A
Next Steps: The miniprep with highest concentration will be sent for sequencing.
3/9: Amplification of LS for Cloning into pSB1C3
Aim of experiment: To amplify LS for cloning into pSB1C3.
Protocols Used: PCR
Results: Figure 2
Next Steps: The gel extracted PCR product will be digested tomorrow and ligations with pSB1C3 will be set up.
19/6: Restriction Digests and Ligations of LS with pSB1C3
Aim of experiment: To digest the LS PCR product with EcoRI and PstI and then set up ligations with pSB1C3.
Protocols Used: Restriction Digests Ligations
Results: N/A
Next Steps: Ligations will be left over the weekend and transformed into JM110 E.coli on Monday.
Week Beginning 7/9/15
Summary
Cloning of LS into pSB1C3 was completed this week and sequencing confirmed it was successfully inserted into pQE80-L so overexpression assays were performed where LS was characterised via Western Blotting.
7/9: Transformations of pSB1C3-LS into JM110 E.coli
Aim of experiment: To transform pSB1C3-LS into JM110 E.coli.
Protocols Used: Transformations
Results:N/A
Next Steps:If transformations are successful, overnight cultures will be set up tomorrow.
8/9: Overnight Cultures of JM110 E.coli containing pSB1C3-LS
Aim of experiment: To set up overnight cultures of JM110 E.coli containing pSB1C3-LS from the transformations carried out yesterday.
Protocols Used: Overnight Cultures
Results: N/A
Next Steps: The overnight cultures will be miniprepped and sent for sequencing tomorrow.
9/9: Plasmid Purification of the JM110 E.coli Overnight Cultures containing pSB1C3-LS
Aim of experiment: To miniprep the JM110 E.coli Overnight Cultures containing pSB1C3-LS.
Protocols Used:
Plasmid Purification (QIAprep® Spin Miniprep Kit)
Results: N/A
Next Steps: The miniprep with the highest concentration will be sent for sequencing to confirm presence of insert. Sequencing for pQE80-L-LS came back correct so overexpression assays will be started tomorrow.
10/9: Overexpression Assays of LS
Aim of experiment: To set up overexpression assays of LS using different concentrations of IPTG.
Protocols Used: Overexpression Assays Note: Cultures were induced with 0 mM, 0.5 mM, 1 mM and 2 mM IPTG. The membrane for the blot was left blocking overnight.
Results: Figure 3
Next Steps: The gel shows a band corresponding to the expected size of LS. However, Western Blotting will confirm if it is indeed LS.
11/9: Characterisation of LS
Aim of experiment: To complete characterisation of LS via Western Blotting.
Protocols Used: Westen Blotting
Results: Figure 4
Next Steps: The blot shows that LS has been successfully detected at the expected size of 80 kDa. There is also partial degradation of the protein. Nevertheless, purification of LS will be started next week.
Week Beginning 14/9/15
Summary
Purification of lanosterol synthase was started this week.
14/9: Overnight Cultures of M15pREP4 E.coli containing pQE80-L-LS
Aim of Experiment: To set up overnight cultures of M15pREP4 E.coli containing pQE80-L-LS.
Protocols Used: Overnight Cultures Note: 150 ml cultures were set up.
Results: N/A
Next Steps: The overnight cultures will be used to innoculate a 3 L day culture tomorrow.
15/9: 3 L Day Cultures of M15pREP4 E.coli containing pQE80-L-LS
Aim of experiment: To set up 3 L day cultures of M15pREP4 E.coli containing pQE80-L-LS.
Protocols Used: Protein Purification Note: The protocol was stopped at Step 6- protein purification will be continued tomorrow.
Results: N/A
Next Steps: Affinity chromatography will be carried out tomorrow.
16/9: Purification of LS
Aim of experiment: To continue purification of LS.
Protocols Used: Protein Purification Note: Protocol was stopped at Step 13.
Results: Figure 5 Figure 6
Next Steps: The SDS gel suggests that the peak observed in the affinity chromatography step may due to a smaller sized protein which has shown up strongly on the gel. However, there is still a band observable around the 75 kDa marker which may be LS. The samples will be pooled together and concentrated. A small sample will be blotted and then size exclusion chromaography can be performed.
Watch Our Introduction Video
Dundee University
Located on the east coast of Scotland, The University of Dundee is one of Scotland's leading research institutes.
Social Media
youtube.com/DundeeiGEM2015
facebook.com/iGEMDundee2015
twitter.com/DundeeiGEMTeam
  University of Dundee
 Home
 About Us
Meet the Team
Attributions
 Project
Abstract
Project Overview
Devices
FluID
Fingerprint Ageing
Chromate Biosensor
Parts
Medals
Future Work
 Wet Lab
Lab Book
FluID
Fingerprints
Chromate Biosensor
Useful Information
Protocols
Sequences
Bacterial Two Hybrid
Safety
 Dry Lab
Introduction
FluID
Fingerprint Ageing
Chromate Biosensor
Appendix : Code
 Outreach
Practices
Ethics
Collaborations
