Team:Dundee/labjournal manu
Dundee iGEM 2015
LABJOURNAL
BioSpray
Our forensic toolkit aims to use synthetic biology approaches to improve on the current  methods used by crime scene investigators.
Chromium Detector
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur a tincidunt elit. Aliquam porta nibh at enim luctus, auctor consequat dolor vehicula.
Fingerprint Aging
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur a tincidunt elit. Aliquam porta nibh at enim luctus, auctor consequat dolor vehicula.
Week Beginning 22/06/2015
Summary
This week was all about getting started with the newly arrived biobrick BBa_K1058008
22/06: Plated BBa_K1058008 from agar stab.
Aim of experiment: To prepare BBa_K1058008 for sequence confirmation, storage, and further processing. The culture was plated from the agar stab and also a liquid culture set up.
Protocols Used: Plating, Overnight cultures
Results: N/A
Next Steps: Purification of the Plasmid, containing BBa_K1058008 and confirmation of size and sequence.
23/06: Purified BBa_K1058008 from liquid culture.
Aim of experiment: To purify BBa_K1058008 for sequence confirmation.
Protocols Used: Miniprep plasmid purification, DNA sequencing
Results Sequencing: Figure 1
Next Steps: PCR of promoter region (Pchr) and repressor region (chrB) of BBa_K1058008 for storage in individual biobricks.
Week Beginning 29/06/2015
Summary
During this week, BBa_K1058008 was disassembled into its constituent parts, Pchr, and chrB.
29/06: PCR-amplification of pChr and ChrB from BBa_K1058008 .
Aim of experiment: To break down BBa_K1058008 into the promoter region pChr, and the repressor gene, ChrB, while adding standard prefix and suffix to each. Furthermore optimisation of first 15 codons of ChrB in order to increase the overall rate of transcription. Restriction digest of the respective ends and subsequent ligation of pChr and ChrB into pSB1C3 for submission to registry.
Protocols Used: PCR, Gel extraction, Restriction Digest
Results: Figure 2
Next Steps: Repeat of PCR of ChrB. Purification of the Plasmid, containing BBa_K1058008 and confirmation of size and sequence.
Week Beginning 06/07/2015
Summary
During this week ligations of pChr and ChrB into pSB1C3 were finalised and the constituent parts of our newly designed chromate sensing system were prepared for ligation and transformation.
06/07: Repeated amplification of ChrB. Ligation and transformation of pChr and ChrB.
Aim of experiment: To repeat PCR of ChrB for subsequent ligation into pSB1C3. Ligation of pChr and ChrB into pSB1C3. Transformation of each into a non-pathogenic lab strain of Escherichia coli.                   Overnight culture of E. coli DH5-alpha, containing GFPmut2, as a fluorescent reporter for our chromate sensing system
Protocols Used: PCR-amplification, Gel extraction, Restriction digest, Ligation, Overnight cultures
Results: Figure 3
2 bands were extracted from the gel for further characterisation, the strongest band, hence called ChrBs, and the band directly above, hence called ChrBw. pChr, ChrBs, and ChrBw were ligated into pSB1C3.                   pSB1C3-pChr, pSB1C3-ChrBs, and pSB1C3-ChrBw were transformed into E. coli MC1061.
Next Steps: Sequence and size confirmation of pChr-pSB1C3, ChrBs-pSB1C3, and ChrBw-pSB1C3. Ligation of pChr to GFP coding sequence for construction of a reporter plasmid.
07/07: Purification of gfp and overnight cultures of pSB1C3-Pchr, pSB1C3-chrB (s), and pSB1C3-chrB (w)
Aim of experiment: Purification of GFPmut2 from overnight cultures. Overnight cultures of recombinant E. coli MC1061, containing pSB1C3-Pchr, pSB1C3-chrB (s), and pSB1C3-chrB (w) respectively for further size and sequence confirmation. Overnight culture of pUniprom for subsequent ligation of chrB
Protocols Used: Miniprep, PCR-amplification, Gel extraction, Overnight cultures
Results: Figure 4
Next Steps: Repeat of amplification of ChrB. Sequence and size confirmation of pSB1C3-Pchr, pSB1C3-chrB (s), and pSB1C3-chrB (w).
08/07: Purification of plasmids pSB1C3-Pchr, pSB1C3-chrB (s), and pSB1C3-chrB (w). Size and sequence confirmation.
Aim of experiment: To purify overnight cultures for size and sequence confirmation. Repeat of PCR amplification of pChr and ChrB. Restriction digestes of all parts in order to produce compatible sticky ends.
Protocols Used: Miniprep, PCR amplification, Restriction digest, Sequencing
Restriction digest of gel purified PCR-products of Pchr, chrB, and gfp, using EcoRI/SpeI, BamHI/HindIII, and XbaI/PstI respecitvely.
Results:
Figure 5 - Size confirmation of pChr and ChrB.
Figure 6 - Sequence confirmation of Pchr
Figure 7 - Sequence confirmation of chrB.
Next Steps: Purification and digest of pUniprom for insertion of chrB.
09/07: Digested and cleaned up pUniprom for insertion of chrB. Amplification of optimised chrB with primers specific for the insertion into pUniprom.
Aim of experiment: To prepare pUniprom for insertion of chrB by digesting it with BamHI and HindIII. To amplify chrB version with first 15 codons optimised for insertion into pUniprom.
Protocols Used:
Restriction digest and Gel extraction of pUniprom
PCR and Gel extraction of chrB (opt)
Results:
Figure 8 Gel with pUniprom
Figure 9 Gel with chrB
Next Steps: Ligation of chrB and chrB (opt) into pUniprom.
10/07: Ligations of chrB and chrB (opt) into pUniprom, and of Pchr and gfp into pSB1C3.
Aim of experiment: Ligation of chrB and optimised chrB into pUniprom via BamHI and HindIII restriction sites. Ligation of Pchr and gfp to each other via SpeI/XbaI-link and into pSB1C3 via EcoRI and PstI restriction sites.
Protocols Used: Ligations
Results: N/A
Next Steps: Transformation of abovementioned ligations.
Week Beginning 13/07/2015
Summary
During this week successful transformants were identified and propagated.
13/07: Transformation of pUniprom-chrB, pUniprom-chrB (opt), and pSB1C3-Pchr-gfp.
Aim of experiment: Transformation of pUniprom-chrB, pUniprom-chrB (opt), and pSB1C3-Pchr-gfp into E. coli JM110 chassis. Setting up a backup restriction digest of pSB1C3 with EcoRI and SpeI in case the double ligation pSB1C3-Pchr-gfp is unsuccessful.
Protocols Used: Transformations, Restriction digest
Results:
Figure 10 Plates after transformation
Figure 11 Gel after digest of pSB1C3
Next Steps: Purification of recombinant vectors and confirmation of size and sequence. Store digested pSB1C3 at -20oC.
14/07: Set up cultures for subsequent purification of recombinant vectors for size and sequence confirmation.
Aim of experiment: Purification of recombinant vectors, pUniprom-chrB, pUniprom-chrB (opt), and pSB1C3-Pchr-gfp, for size and sequence confirmation. Patch plating of the same colonies for storage and retrieval once size and sequence have been confirmed.
Protocols Used: Overnight cultures, Patch plating
Results: N/A
Next Steps: Purification of the recombinant plasmids for confirmation of size and sequence.
15/07: Purified pUniprom-chrB, pUniprom-chrB (opt), and pSB1C3-Pchr-gfp from overnight cultures
Aim of experiment: Purification of pUniprom-chrB, pUniprom-chrB (opt), and pSB1C3-Pchr-gfp from overnight for confirmation of size and sequence. Restriction digest of pUniprom-chrB, pUniprom-chrB (opt) with BamHI and HindIII and of pSB1C3-Pchr-gfp with EcoRI/PstI for size confirmation.
Protocols Used: Miniprep, Restriction digest, Colony PCR, Sequencing
Results:
Figure 12 Size confirmation of chrB and chrB (opt).
Figure 13 Gel after colony PCR of chrB (opt).
Figure 14 Sequencing result of pSB1C3-Pchr-gfp (colony 1).
Next Steps: Confirmation of sequence of chrB (opt)
16/07: Set up overnight cultures of chrB (opt) - colonies chosen after colony-PCR for subsequent purification and sequence confirmation.
Aim of experiment: Set up overnight cultures of chrB (opt) colonies 2, 4, 9, and 12 for sequencing.
Protocols Used: Overnight cultures
Results: N/A
Next Steps: Purification of recombinant plasmids.
17/07: Purified and sequenced chosen chrB (opt) - colonies.
Aim of experiment: To purify recombinant plasmids for sequence confirmation.
Protocols Used: Miniprep, Sequencing
Results: chrB (opt) showed HindIII restriction sites and was hence cleaved into partial inserts.
Next Steps: Use alternative cloning protocol for chrB and chrB (opt).
Week Beginning 20/07/2015
Summary
During this week an alternative cloning strategy for chrB and chrB (opt) was employed. Furthermore, the pSB1C3-Pchr-gfp - construct was cloned in a stepwise fashion.
20/07: Went back to the original plate of pSB1C3-Pchr-gfp for visualising colonies that do or do not express GFP.
Aim of experiment: To compare colonies on the patch plate of pSB1C3-Pchr-gfp by differentially visualising colonies that do or do not express GFP under UV light. The background of this experiment was that if there is one colony that has both inserts in the correct orientation with respect to each other, then there is perhaps one that also has them with the correct orientation with respect to the plasmid. Colonies identified as expressing GFP were patch plated and grown in liquid cultures over night in order to confirm the sequence of the insert.
Protocols Used: Patch-plating, overnight cultures
Results: Figure 15 pSB1C3-Pchr-gfp in E. coli JM110.
Next Steps: Sequence confirmation of newly selected colonies.
21/07: Purified plasmids from new colonies and confirmed sequence. Restriction digest of pUniprom with BamHI and PstI.
Aim of experiment: Purification of newly selected pSB1C3-Pchr-gfp - colonies 1, 5, 6, and 7, for subsequent sequence confirmation. Restriction digest of pUniprom with BamHI and PstI to employ an alternative cloning strategy of chrB and chrB (opt).
Protocols Used:
Miniprep and Sequencing of Pchr-gfp - insert.
Restriction digest of pUniprom.
Results:
Figure 16 Alignment of sequencing results.
Figure 17 Gel after restriction digest of pUniprom.
Next steps: Stepwise ligation of Pchr and gfp into pSB1C3 in order to clone the inserts in the correct orientation with respect to the plasmid. Insertion of chrB and chrB (opt) into pUniprom via BamHI and PstI links.
22/07: Ligated Pchr into pSB1C3 via EcoRI and SpeI sites and cloned the recombinant plasmid into E. coli JM110.
Aim of experiment: Ligation of Pchr into pSB1C3 via EcoRI and SpeI sites. This is the first step of a 2-step cloning strategy with the aim of cloning Pchr and gfp into pSB1C3 in the correct orientation with respect to the plasmid.
Protocols Used: Ligation, Transformation
Results: Figure 18 Plates after transformation of Pchr into pSB1C3.
Next Steps: Sequence confirmation of Pchr in pSB1C3.
23/07: Set up overnight cultures of pSB1C3-Pchr colonies.
Aim of experiment: Set up overnight cultures of pSB1C3-Pchr colonies to prepare pSB1C3-Pchr for sequence confirmation
Protocols Used: Overnight culture, Patch-plating
Results: N/A
Next Steps: Purification of the recombinant vector for size and sequence confirmation.
24/07: Purified pSB1C3-Pchr for sequence confirmation.
Aim of experiment: Purification of pSB1C3-Pchr for sequence confirmation before ligating gfp into it.
Protocols Used:
Plasmid purification, Sequencing
Results: Figure 19 Alignment of sequenced pSB1C3-Pchr vectors with Pchr sequence.
Next Steps: Digest of pSB1C3-Pchr with SpeI/PstI and ligation of Pgfp
25/07: PCR-amplified chrB and chrB (opt).
Aim of experiment: Amplification of chrB and chrB (opt) for insertion into pUniprom. Restriction digest of the PCR product with BamHI and PstI for producing compatible sticky ends
Protocols Used: PCR-amplification, Restriction digest
Results: N/A
Next Steps: Ligation and Transformation of chrB and chrB (opt).
26/07: Ligated chrB and chrB (opt) into pUniprom.
Aim of experiment: Insertion of chrB and chrB (opt) into pUniprom.
Protocols Used: Ligation
Results: N/A
Next Steps: Transformation of ligated vectors.
Week Beginning 27/07/2015
Summary
Final touches were made on the chromate system. All parts were ligated into their final vectors during this week. Preparations were made for characterisation of each of the parts.
27/07: Transformed chrB and chrB (opt) into E. coli JM110
Aim of experiment: Transformation of the repressors, chrB and chrB (opt), into JM110
Protocols Used: Transformation
Results: Figure 20 Plates after transformation.
Next Steps: Purification of pUniprom-chrB and pUniprom-chrB (opt) for sequence confirmation.
28/07: Digested pSB1C3-Pchr for insertion of gfp
Aim of experiment: Digesting sequence confirmed pSB1C3-Pchr with SpeI and PstI for subsequent ligation with gfp. Overnight cultures of chrB and chrB (opt) for subsequent plasmid purification, size confirmation, and sequence confirmation.
Protocols Used:
Restriction digest, Ligation of gfp into pSB1C3-Pchr.
Overnight culture of of chrB and chrB (opt).
Results: N/A
Next Steps: Transformation of pSB1C3-Pchr-gfp for size and sequence confirmation. Purification of chrB and chrB (opt) for sequence confirmation.
29/07: Purified chrB and chrB (opt) for size and sequence confirmation.
Aim of experiment: Purification of chrB and chrB (opt) for confirmation of size and sequence.
Protocols Used: Plasmid purification, Restriction digest, Sequencing
Results:
Figure 21 Confirmation of insert size.
Figure 22 Confirmation of sequence.
Next Steps: Confirmation of expression of chrB and chrB (opt) by western blotting.
30/07: Repeated transformation of pSB1C3-Pchr-gfp into E.coli JM110
Aim of experiment: Digest of pSB1C3-Pchr with SpeI and PstI for insertion of GFP. Repeat of unsuccessful transformation from 28/07.
Protocols Used: Restriction digest, Ligation, Transformation
Results: Figure 23 Plates after transformation
Next Steps: Overnight cultures for purification of the pSB1C3-Pchr-gfp for confirmation of insert size and sequence.
31/07: Set up overnight cultures for confirmation of pSB1C3-Pchr-gfp.
Aim of experiment: Overnight cultures for subsequent plasmid purification of pSB1C3-Pchr-gfp.
Protocols Used: Overnight culture
Results: N/A
Next Steps: Purification of the Plasmids for confirmation of insert size and sequence.
01/08: Purified pSB1C3-Pchr-gfp for size confirmation and sequencing.
Aim of experiment: Purification of pSB1C3-Pchr-gfp for confirmation of insert size and sequence.
Protocols Used: Plasmid purification, Restriction digest, Sequencing
Results:
Figure 24 Gel for size confirmation.
Figure 25 Sequence confirmation.
Next Steps: Overnight cultures in preparation for western blotting for characterisation of expression of both, GFP, and ChrB and ChrBopt.
02/08: Set up cultures of JM110 + pSB1C3-Pchr-gfp (A), MC1061 + pSB1C3-Pchr-gfp (B) and JM110 pUniprom-chrB and JM110 pUniprom-chrB (opt). in preparation for western blotting.
Aim of experiment: Overnight cultures of E. coli strains with sequence confirmed pUniprom-chrB, pUniprom-chrB (opt), pSB1C3-Pchr-gfp (A), pSB1C3-Pchr-gfp (B) - colony 3 and pSB1C3-Pchr-gfp (B) - colony 4 for subsequent western blotting.
Protocols Used: Overnight culture
Results: N/A
Next Steps: Western blotting for characterisation of expression of both, GFP, and ChrB and ChrB (opt).
Week Beginning 03/08/2015
Summary
This week was entirely used for characterising the expression of GFP from the promoter Pchr, and the expression of ChrB from the Ptat in pUniprom.
03/08 - 09/08: Determined expression of ChrB, ChrB (opt), and GFP by western blotting.
Aim of experiment: To determine whether or not the required proteins, GFP, and ChrB, are expressed by blotting against GFP and against a 6-His Tag of ChrB and ChrB (opt) respectively.
Protocols Used: SDS-PAGE and western blot
Results: Figure 26
Next Steps: Transformation of both plasmids into MG1655 with the goal to characterise the interaction between ChrB and Pchr. Furthermore the original biobrick, BBa_K1058008 will be tested under the same conditions for comparison.
Week Beginning 10/08/2015
Summary
This week was spent with transforming all combinations of pSB1C3-Pchr-gfp and pUniprom-chrB into E. coli MG1655. Tests for their interaction were also prepared.
10/08: Transformed pSB1C3-Pchr-gfp (A) and pSB1C3-Pchr-gfp (B), and pUniprom-chrB, and pUniprom-chrB (opt), as well as BBa_K1058008 into E. coli MG1655.
Aim of experiment: To transform the following combinations of pSB1C3-pChr-GFP and pUniprom-ChrB into MG1655:
pUniprom-chrB (from colony 2 - 29/07) - pSB1C3-Pchr-gfp (A)
pUniprom-chrB (from colony 2 - 29/07) - pSB1C3-Pchr-gfp (B)
pUniprom-chrB (opt) (from colony 3 - 29/07) - pSB1C3-Pchr-gfp (A)
pUniprom-chrB (opt) (from colony 3 - 29/07) - pSB1C3-Pchr-gfp (B)
in a stepwise fashion. Furthermore transform BBa_K1058008 into MG1655. Transformations were done in a stepwise fashion with only one of the plasmids at a time.
Protocols Used: Plasmid purification, Transformation
Results: Figure 27 Plates after transformation.
Next Steps: Making newly transformed cells competent again in order to transform the second plasmid as well.
11/08: Set up overnight cultures of all transformed cells.
Aim of experiment: To prepare overnight cultures of yesterdays transformations in order to make them competent to transform the second plasmid as well.
Protocols Used: Overnight culture
Results: N/A
Next Steps: Make cells of overnight culture competent to transform in second plasmid.
12/08: Made competent MG1655 with pUniprom-ChrB2, pUniprom-ChrBopt3, pSB1C3-pChr-GFP (15/07), pSB1C3-pChr-GFP (31/07).
Aim of experiment: To make transformed cells competent in order to transform the second plasmid. The same plasmids that were used for the initial transformations were also used for this set of transformations.
Protocols Used: Competent cells, Transformations
Results: N/A
Next Steps: Blot against GFP in order to confirm (or refute) that ChrB interacts with Pchr and hence switches off expression of GFP.
13/08: Transformed pUniprom-chrB (from colony 2 - 29/07) and pUniprom-chrB (opt) (from colony 3 - 29/07) into MG1655 + pSB1C3-Pchr-gfp (A) and pSB1C3-Pchr-gfp (B)
Aim of experiment: To build a strain of E. coli MG1655, containing both plasmids required for a chromate sensor.
Protocols Used: Transformation
Results: No successful transformants were found.
Next Steps: Repeat this experiment but transform pSB1C3-Pchr-gfp (A) and pSB1C3-Pchr-gfp (B) into competent MG1655 + pUniprom-chrB and MG1655 + pUniprom-chrB (opt) respectively.
Week Beginning 17/08/2015
Summary
During this week E. coli MG1655 containing both plasmids for a. Western Blots were used to determine expression of ChrB and GFP.
17/08: Transformations akin to 13/08 were done, but pSB1C3-Pchr-gfp (A) and pSB1C3-Pchr-gfp (A) were transformed into MG1655 + pUniprom-chrB and MG1655 + pUniprom-chrB (opt).
Aim of experiment: To build a strain of MG1655, containing both plasmids required for a chromate sensor.
Protocols Used: Transformation
Results: Successful transformants were found on all plates.
Next Steps: To determine expression of ChrB and GFP.
18/08: Preparation of samples for western blot.
Aim of experiment: To determine whether ChrB and GFP are expressed and have first estimates of differences in expression levels.
Protocols Used: Overnight culture
Results: N/A
Next Steps: Lyse cells and prepare samples for SDS-PAGE + Western-Blot.
19/08 - 22/08: Repeated western blots with different concentrations to determine whether the expression of ChrB has an effect on GFP levels.
Aim of experiment: To determine whether expression of ChrB has an effect on the expression of GFP.
Protocols Used: SDS-PAGE + Western Blot
Results: Figure 28
Next Steps: Check if different chromate salts have an effect on the respective expression levels.
Week Beginning 24/08/2015
Summary
Plate reader experiments were carried out in order to determine the effect of K2CrO4, K2Cr2O7, CrCl3, and K2SO4 on the expression of GFP.
24/08-30/08: Design of Plate reader experiments.
Aim of experiment: To design a series of plate reader experiments to measure OD600 and relative fluorescence of the chromate sensor as a response to different concentrations of K2CrO4, K2Cr2O7, CrCl3, and K2SO4.
Protocols Used: Plate Reader Experiments
Results: Figure 29 MG1655 + BBa_K1058008
Results: Figure 30 MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB
Results: Figure 31 MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt)
Results: Figure 32 MG1655 + pSB1C3-Pchr-gfp (B)
Next Steps: Try the system with different expression levels of ChrB. Induce overexpression by exploiting the T7-promoter in pUniprom after transforming the different combinations between pSB1C3-Pchr-gfp and pUnipromchrB / pUnipromchrB (opt) into BL21(DE3) and adding IPTG.
Week Beginning 31/08/2015
Summary
UK Synbio Conference and UK iGEM meetup at Westminster University.
Week Beginning 07/09/2015
Summary
Parts from CeBiTec Bielefeld arrived and attempts at cloning were made. The chromate sensing system was transformed into BL21(DE3) and some of the plate reader experiments repeated.
07/09: Transformation of pSB1K3 (Bielefeld) into MG1655.
Aim of experiment: The high-copy plasmid arrived at a concentration of 7 ng/µl. Hence it was transformed in order to produce more of it before purifying and digesting it.
Protocols Used: Transformation,
Overnight culture for new E. coli MG1655
Results: No transformants were found.
Next Steps: Retry transformation with new competent E. coli MG1655.
08/09: Transformation of pSB1K3 into MG1655.
Aim of experiment: Test transformation of BBa_K1058008 into new MG1655 competent cells and re-transformation of pSB1K3 for subsequent miniprep.
Protocols Used: Transformation
Results: Successful test transformation, but no colonies were found on the pSB1K3 plate.
Next Steps: Use an alternative compatible plasmid for ligation of the codon optimised chrB (Bielefeld).
09/09: Transformation of pSB1K3 into MG1655.
Aim of experiment: Restriction digest of pT7KS with EcoRI and PstI as alternative to pSB1K3. Ligation of digested chrB (Bielefeld) and transformation of pT7KS-chrB. Also transformations of Chromate-sensor plasmids into BL21 (DE3) in the following combinations:
pSB1C3-Pchr-gfp (B) + pUniprom-chrB
pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt)
pUniprom-chrB
pUniprom-chrB (opt)
Protocols Used: Restriction Digest, Gel Extraction, Ligation
Results: Transformations of the plasmid with pT7KS-chrB (Bielefeld) yielded 12 colonies, but also colonies on the vector control plate. Transformations of pSB1C3-Pchr-gfp (B) + pUniprom-chrB, pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt), pUniprom-chrB, and pUniprom-chrB (opt) into BL21(DE3) were successful.
Results:
Figure 33 pT7KS-chrB (Bielefeld) transformations
Figure 34 Transformations into BL21(DE3)
Next Steps: Confirmation of colonies with pT7KS-chrB (Bielefeld) by colony PCR. Preparation of recombinant BL21(DE3) for plate reader experiments.
10/09: Colony PCR of pT7KS-chrB (Bielefeld).
Aim of experiment: Check if any of the colonies after ligation of chrB (Bielefeld) into pT7KS were successful despite colonies on the control plate.
Protocols Used: Colony PCR
Results: Figure 35
Next Steps: Prepare transformed BL21(DE3) for plate reader experiments. Attempt new transformation of pT7KS-chrB (Bielefeld).
11/09: Preparation for Plate reader experiment with BL21(DE3)
Aim of experiment: Set up overnight cultures for subsequent plate reader experiments.
Protocols Used: Overnight cultures
Results: N/A
Next Steps: Set up plate reader, collect and analyse data. Assess the effect of induction of higher repressor-expression on the level of GFP.
12/09: Set up plate reader.
Aim of experiment: To design a series of plate reader experiments to measure OD600 and relative fluorescence of the chromate sensor. At first no chromate was added to this plate in order to assess the effect of IPTG induction.
Protocols Used: Plate reader experiments
Results: Figure 36
Next Steps: Repeat experiments with added chromate substances.
Week Beginning 14/09/2015
Summary
During this week the samples from the previous plate reader were retrieved and prepared for blotting against both GFP, and the 6-His - Tag of ChrB.
14/09: Western blot of plate reader samples MG1655 + pSB1C3-Pchr-gfp + pUniprom-chrB, and MG1655 + pSB1C3-Pchr-gfp + pUniprom-chrB (opt).
Aim of experiment: To identify whether or not there is any detectable difference in both, the level of GFP, and the level of ChrB, before and after induction with IPTG.
Protocols Used: SDS-PAGE + Western blot
Results: Even though the blot was very blurred, expression of GFP could be confirmed in all samples. There seemed to be no noteable difference between induced and uninduced.
Next Steps: Repeat the assays of the previous weeks by subcloning chrB, chrB (opt), and chrB (Bielefeld) into vectors with different expression levels and see how it influences the level of GFP expression. Then add different chromate salts and also probe the system with other substances, e.g. other heavy-metal salts.
×
Plate Reader Protocol
A 5 ml overnight culture of the desired strain was grown at 37 oC.
Equalise OD600 if required. Dilute culture 1:1000.
Aliquot 198 µl into the required amount of wells in a 96-well plate.
Add additional substances and controls as required.
Preheat platereader to required temperature and start experiment.
Retrieve and analyse data.
Close
×
Competent Cells Protocol
A 5 ml overnight culture of the desired strain was grown at 37oC.
Subsample 1/10 of the volume of the subculture. I.e. 1 ml of the overnight culture into 10 ml fresh LB.
Grow subculture at at 37 oC for 2 h.
Spin down subculture at 4000 rpm at 4oC for 10 min.
Resuspend the pellet in ice cold transformation buffer. Use 1/10 of the initial volume of the subculture.
Use 100 µl per transformation.
Close
×
Plating Protocol
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
Fresh agar plates were inoculated with cells on the toothpick.
Agar plates were incubated at 4oC over night.
Close
×
Sequencing Protocol
For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
Volume of sample equivalent to 550ng DNA
2 µl of 3 mM primer
Remaining volume of water for a total volume of 30 µl
The samples were submitted to DNA sequencing and services .
Close
×
Overnight Culture Protocol
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
5 ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.
Tubes were incubated at 37oC in a rotary incubator at 200rpm over night.
Close
×
PCR Protocol
PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the Sequences section.
Reactants
Volume (µl)
DNA
1
DMSO
2.5
Forward Primer
0.5
Reverse Primer
0.5
Herculase Buffer
10
dNTP
0.5
Herculase
0.5
H2O
34.5
**PCR Programme**
Elongation time was altered according to the length of the sequences.
Close
×
Colony PCR Protocol
The PCR reaction mixture were set up in multiples of the volumes in table below. The length of all sequences used is stated in the Sequences section.
The DNA template is made in the following way:
Agar plates with required antibiotic were taken out of 4oC to warm to room temperature.
A numbered grid according to the desired number of PCR reactions is drawn on the back of the plate.
Cells or colonies were picked up from the source with a toothpick under sterile conditions.
Each colony is transferred to a 30 µl aliquot of sterile water.
With the same toothpick the grid with the respctive number on the plate was inoculated.
The plate was incubated at 37oC overnight.
The inoculated water-aliquots were boiled for 10min, then centrifuged at 13300rpm for 10min.
The supernatant of this preparation is the DNA fraction for the subsequent PCR reaction.
Reactants
Volume (µl)
DNA
5
DMSO
1
Forward Primer
0.5
Reverse Primer
0.5
+Mg Buffer
2
dNTP
0.2
Taq-polymerase
0.2
H2O
10.6
15 µl of the reaction mixture was aliquoted into separate PCR tubes. 5 µl of DNA fraction was added to each respective tube in order to have a total reaction volume of 20 µl per tube.
**PCR Programme**
Elongation time was altered according to the length of the sequences.
Close
×
Miniprep Protocol
5 ml bacterial overnight culture was pelleted by stepwise centrifugation in 1.5 ml microcentrifuge tubes at 13300rpm for 3 minutes at room temperature.
Pelleted bacterial cells were resuspended in 250 µl of Buffer P1.
250 µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution became clear.
350 µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.
The mixture was centrifuged for 10 minutes at 13300rpm in a microcentrifuge.
The supernatant was applied to a QIAprep spin column by pipetting.
500 µl of Buffer PB was added to the QIAprep spin column and centrifuged for 1 minute. The flow-through was discarded.
750 µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute. The flow-through was discarded.
The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.
Close
×
Gel Extraction Protocol
The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.
The gel slice was transferred into a microcentrifuge tube. 700 µl of Buffer QG was added.
The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.
After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.
230 µl of isopropanol was added to the sample and mixed.
A QIAquick spin column was placed in a 2 ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
500 µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
750 µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.
The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.
Close
×
Restriction Digest Protocol
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the Journal section.
Post Plasmid Purification
Post PCR
gBlock
DNA
1mg
50 µl
100ng
Cutsmart Buffer (µl)
6
6
6
Water (µl)
As required
1
As required
Enzyme 1 (µl)
1.5
1.5
1.5
Enzyme 2 (µl)
1.5
1.5
1.5
Total Volume (µl)
30
60
30
The digests were incubated at 37oC for 3 hours.
Note that when performing plasmid digestions, 2.5 µl of alkaline phosphatase and 2.5 µl of its buffer were added at the 2 hour and 2.5 hour mark.
Close
×
Ligation Protocol
Ligation reactions were set up according to the table below. The vector to insert ratio is stated for each specific reaction in the Journal section.
Vector (µl)
2:1 (Vector: Insert) [µl]
3:1 (Vector: Insert) [µl]
4:1 (Vector: Insert) [µl]
T4 DNA Ligase
1
1
1
1
Ligase Buffer
1
1
1
1
Vector
1
1
1
1
Insert
-
2
3
4
H20
7
5
4
3
The reactions were incubated at room temperature for at least 3 hours.
Close
×
Transformation Protocol
100 µl of competent cells were defrosted. Thaw on ice.
Agar plates were taken out of 4oC to warm to room temperature.
1-5 µl of DNA was added to cells.
Cell and DNA mixture was left on ice for 20 minutes.
Mixture was placed in 42oC water bath for 90 seconds.
Mixture was put back on ice for 2 minutes.
1 ml of LB (without antibiotic) was added to cells.
Cells were allowed to grow for 1 hour at 37oC in a shaking incubator.
Cells were then spun down and the supernatant discarded.
The pellet was resuspended in the 100 µl.
Cells were plated on agar plates with appropriate antibiotic.
Close
×
Protein Expression Optimization Protocol
SDS-PAGE and Western Blotting
50 µl of an overnight culture was used to inoculate 5 ml of fresh LB containing the appropriate antibiotics and left to grow at 37oC in a shaking incubator.
Once the OD600=0.6-0.8, 1 mM of IPTG was added to the culture and left to grow for a further 3 hours.
1 ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.
Meanwhile, 950 µl of 2x Laemmli sample buffer was added to 50 µl of B-mercaptoethanol.
The supernatant was discarded from the samples and the pellet resuspended in 200 µl of the above solution.
The samples were then boiled at 90oC for 10 minutes.
Afterwards, they were spun down for 1 minute in a microcentrifuge.
At this stage, the samples could be stored at -20oC or ran on a gel immediately.
Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the stacking gel at which point the voltage was increased to 200V for 20 minutes.
One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.
After the proteins had been transferred onto the membrane, the membrane was blocked in 5% Marvel milk overnight at 4oC.
The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10 ml of 1xTBS buffer to which the primary anti-His antibody was added and left for one hour. The protein ladder was cut off before adding the primary antibody.
The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10 ml of 1xTBS buffer, to which the secondary antibody was added and left for one hour.
The membrane was then washed in 3x10 ml 1xTBST buffer for 5 minutes.
The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off at step 12.
Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.
Close
×
Figure 1: Alignment of BBa_K1058008 . Sequences from top to bottom: sequencing result from forward primer; expected sequence; reverse complement of sequencing result from reverse primer; sequencing result from intermediate primer. Colour code: Green = Prefix/Suffix; Pink = Annealing site of intermediate primer; Yellow = Mismatch in one sequence; Red = In frame stop codons.
'
Close
×
Figure 2: 1% agarose gel after PCR of BBa_K1058008 with primers specific to the promoter region, Pchr, and the repressor region, chrB. Left: Pchr (expected size 169bp); Middle: 1kb+ ladder, Right: chrB (expected size 939bp). Amplification of chrB was not successful. Pchr was further processed by restriction digest with EcoRI and PstI.
'
Close
×
Figure 3: 1% agarose gel after PCR of BBa_K1058008 with primers specific to the repressor region, chrB. Left: 1.5 µl DMSO; Middle: 1kb+ ladder, Right: 2.5 µl DMSO. Expected size for each: 939bp. The annealing temperature for the PCR amplification was lowered to 50°C in order to increase the chances of accomodating a primer that is not 100% complementary.
'
Close
×
Figure 4: 1% agarose gel after PCR of gfp with corresponding primers. Left: GFP (expected size 717bp); Middle: 1kb+ ladder, Right: PCR product of chrB, using primers to add a BamHI restriction site in front of it and a 6-His-Tag and a HindIII restriction site at the end. (expected size of chrB: 939bp). Since amplification of chrB was unsuccessful, it will be repeated.
'
Close
×
Figure 5: 1% agarose gel after restriction digest of pSB1C3-Pchr, pSB1C3-chrB (s), and pSB1C3-chrB (w) with EcoRI and PstI for confirmation of size. Ladder: 1kb+. Top: 1-4: Pchr 1-4; chrB (s) 1-2. Bottom: chrB (s) 3-4; chrB (w) 1-4. Expected sizes: Pchr: 169bp; chrB: 939bp.
'
Close
×
Figure 6: Alignment of Pchr sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green = Prefix/Suffix
'
Close
×
Figure 7: Alignment of chrB (opt) sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green: Prefix and Suffix; Red: In frame stop codons; Blue: First 15 codons optimised for expression in E. coli.
'
Close
×
Figure 8: 1% agarose gel after restriction digest of pUniprom (Expected size 2454bp).
'
Close
×
Figure 11: 1% agarose gel after restriction digest of pSB1C3 with EcoRI and SpeI (Expected size 2070bp).
'
Close
×
Figure 9: 1% agarose gel after PCR of chrB (opt). (Expected size 939bp).
'
Close
×
Figure 10: Plates after transformation of Top: pUnipromchrB, Middle: pUnipromchrB (opt) and Bottom: pSB1C3-Pchr-gfp into JM110.
'
Close
×
Figure 12: 1% agarose gel after restriction digest of pUnipromchrB, pUnipromchrB (opt) and pSB1C3-Pchr-gfp for confirmation of size. Ladder: 1kb+. Top: 1-4: pSB1C3-Pchr-gfp 1-4; pUnipromchrB (opt) 1-2. Bottom: pUnipromchrB (opt) 3-4; pUnipromchrB 1-4. Expected sizes: Pchr-gfp: 909bp; chrB and chrB (opt): 939bp.
'
Close
×
Figure 13: 1% agarose gel after colony PCR of pUnipromchrB (opt). Top: colonies 1-11; Bottom: colonies 12-22. Colonies 2, 4, 9, and 12 were selected for sequencing. Ladder: 1kb+ Expected size of chrB (opt): 939bp.
'
Close
×
Figure 14: Sequencing result of pSB1C3-Pchr-gfp after double ligation. Even though the sequence is as expected, it is ligated in reverse. It was hence called pSB1C3-Pchr-gfp (A) The Result for pSB1C3-Pchr-gfp (colony 2) was not as expected, albeit in the correct orientation. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 15: Plates with pSB1C3-Pchr-gfp in JM110 after double ligation under UV light. Numbered colonies were selected for initial size confirmation and sequencing. However, other colonies seemed to express GFP as well, hence a second attempt was made with different colonies from the same plate.
'
Close
×
Figure 16: Alignment of sequencing results of pSB1C3-Pchr-gfp - colonies 1, 5, 6, and 7 with sequences of gfp (mut2) and Pchr sequence. Colony 1 remained consistent, the other tested colonies were without success. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 17: 1% Agarose gel after restriction digest of pUniprom with BamHI/PstI (expected size: 2454bp). Right: PCR product of Haptoglobin.
'
Close
×
Figure 18: Plates after transformation of Pchr into pSB1C3. Top: ratio insert:vector 2:1; Bottom: ratio insert:vector 3:1.
'
Close
×
Figure 19: Alignment of Pchr sequence retrieved from BBa_K1058008 with the sequencing result of this region. Colour code: Green = Prefix/Suffix
'
Close
×
Figure 20: Plates after transformation of, Top: chrB into pUniprom; Bottom: chrB (opt) into pUniprom. Ratio insert:vector for ligations: left: 3:1; Right: 4:1.
'
Close
×
Figure 21: Gel after restriction digest of Left: pUniprom-chrB, and Right: pUniprom-chrB (opt) with BamHI/PstI for confirmation of insert size. Expected size 939bp. chrB colony 2 and chrB (opt) colony 3 were selected for sequencing.
'
Close
×
Figure 22: Alignments of: Left: chrB and Right: chrB (opt). Colours: Pink = Restriction sites (BamHI/PstI); Blue = 6-His Tag; Red = In-frame stop codons; Cyan = optimised codons.
'
Close
×
Figure 23: Plates after ligation of gfp into pSB1C3-Pchr. Left: Ratio Insert:Vector 2:1; Right: Ratio Insert:Vector 3:1.
'
Close
×
Figure 24: Gel for size confirmation of pSB1C3-Pchr-gfp (Top). Bottom: Left - Haptoglobin, Right - Lanosterol Synthase. pSB1C3-Pchr-gfp (3) and pSB1C3-Pchr-gfp (4) were submitted for sequencing.
'
Close
×
Figure 25: Alignment of Pchr sequence retrieved from BBa_K1058008 and gfp (mut2) with the forward and reverse sequencing results of Pchr-gfp this region. The sequence was as expected, it was hence called pSB1C3-Pchr-gfp (B). Colour code: Green = Prefix/Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
'
Close
×
Figure 26: Top: anti-GFP: Single colonies of JM110 + pSB1C3-Pchr-gfp (A), MC1061 + pSB1C3-Pchr-gfp (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of each sample was mixed with 100 µl laemmli buffer, and boiled for 10min. 3 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and PmanA-gfp as a positive control.          Bottom: anti-6-His - ChrB: Single colonies of JM110 pUniprom-chrB and JM110 pUniprom-chrB (opt) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 200 µl laemmli buffer, and the sample boiled for 10min. 10 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and hydA - 6-His as a positive control.         Expected sizes: GFP ~ 27kDa, ChrB ~35kDa.
'
Close
×
Figure 27: Plates after transformation of single constituents of chromate sensor into E. coli MG1655. From top to bottom: pUniprom-chrB (from colony 2 - 29/07), pUniprom-chrB (opt) (from colony 3 - 29/07), pSB1C3-Pchr-gfp (B), pSB1C3-Pchr-gfp (A), BBa_K1058008 .
'
Close
×
Figure 28: Western blot against GFP: Single colonies of MG1655, containing all 4 combinations, pUniprom-chrB - pSB1C3-Pchr-gfp (A), pUniprom-chrB - pSB1C3-Pchr-gfp (B), pUniprom-chrB (opt) - pSB1C3-Pchr-gfp (A), and pUniprom-chrB (opt) - pSB1C3-Pchr-gfp (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol and 100 µg/ml ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml PBS. 100 µl of these samples was mixed with 100 µl laemmli buffer, and the sample boiled for 10 min. The indicated volume of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3, and pUniprom were included as a negative control, and PmanA-gfp as a positive control. The blot against 6-His – ChrB was without success and is not shown. No chromate of any kind was added to the medium previous to this blot, hence no expression of GFP was expected.
'
Close
K2CrO4, K2Cr2O7, CrCl3, and K2SO4
×
Figure 29: F,OD-plot of results of plate reader experiment after exposing MG1655 + BBa_K1058008 to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 30: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 31: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt) to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 32: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-Pchr-gfp (B) to a range of concentrations between 10 nM and 100 µM of K2CrO4, K2Cr2O7, CrCl3, and K2SO4. One measurement was made every 10min across 16h.
'
Close
×
Figure 33: Plates after ligation of chrB (Bielefeld) into pT7KS. From top to bottom: Ratio insert:vector 2:1; Ratio Insert:Vector 3:1; pT7KS religation control.
'
Close
×
Figure 34: Plates after transformation of single constituents of chromate sensor into BL21 (DE3). From top to bottom: pSB1C3-Pchr-gfp (B) + pUniprom-chrB, pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt), pUniprom-chrB, and pUniprom-chrB (opt).
'
Close
×
Figure 35: 1% agarose gel after colony PCR of pT7KS-chrB (Bielefeld). Top: colonies 1-6; Bottom: colonies 7-12. Ladder: 1kb+ Expected size of chrB (Bielefeld): 939bp
'
Close
×
Figure 36: F,OD-plot of results of plate reader experiment after inducing expression of ChrB from the T7 promoter of pUniprom BL21 (DE3) pSB1C3-Pchr-gfp (B) + pUniprom-chrB, and pSB1C3-Pchr-gfp (B) + pUniprom-chrB (opt). IPTG was added after 4 hours. One measurement was made every 10min across 22h.
'
Close
  University of Dundee
 Home
 About Us
Meet the Team
Attributions
 Project
Abstract
Project Overview
Devices
FluID
Fingerprint Ageing
Chromate Biosensor
Parts
Medals
Future Work
 Wet Lab
Lab Book
FluID
Fingerprints
Chromate Biosensor
Useful Information
Protocols
Sequences
Bacterial Two Hybrid
Safety
 Dry Lab
Introduction
FluID
Fingerprint Ageing
Chromate Biosensor
Appendix : Code
 Outreach
Practices
Ethics
Collaborations
