<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_cgcraigon skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/cgcraigon</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><H1><HIGHLIGHT class="highlight">CSI: DUNDEE</HIGHLIGHT></H1><H3><HIGHLIGHT class="highlight">The Forensic Toolkit</HIGHLIGHT></H3><A href="https://2015.igem.org/Team:Dundee/intro_video" class="btn btn-primary btn-lg">Watch Our Introduction Video</A><DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><SPAN id="biospray-title">Nasal Mucus </SPAN><DIV id="nmcollapsible" class="collapse biospray-content"><DIV class="row"><DIV class="box"><DIV class="labtitle">Week Beginning 15/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Produced a plasmid preparation of pIDT-<I>OBP2A</I> in order to use it as template in PCR. </P></DIV><DIV id="nmcollapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/6: Transformation of pIDT-<I>OBP2A</I> in MC1610 <I>E.coli</I> strain. </SPAN><DIV id="nmcollapsiblew1-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To transform pIDT-<I>OBP2A</I> into MC1061 <I> e.coli</I> strain </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> The result of this transformation can be seen tomorrow. If successful then the next step will be to produce an overnight culture in preparation of subsequent plasmid purification. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/6: Overnight culture of pIDT-<I>OBP2A</I> plasmid in MC1061.  </SPAN><DIV id="nmcollapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To produce an overnight culture of pIDT-<I>OBP2A</I> from positive colonies of MC1061 <I>E.coli</I> strain from the transformation done on the 17/6.  
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results:</B> N/A
                  </P><P><B>Next Steps:</B> If the overnight culture successfully grows then the next stage of experimentation will be to produce a plasmid preparation of pIDT-<I>OBP2A</I> from this overnight culture.                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">19/6: Produce a plasmid preparation of pIDT-<I>OBP2A</I> from the overnight culture.   </SPAN><DIV id="nmcollapsiblew1-3" class="collapse box-content"><P><B>Aim of experiment:</B> The overnight culture produced yesterday will now undergo a plasmid preparation in order to obtain  the pIDT-<I>OBP2A</I> plasmid.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Miniprep </A></P><P><B>Results:</B> The plasmid preperation yeilded a concentration of 305.53 ng/ul.
                  </P><P><B>Next Steps:</B> To run a PCR using the purified pIDT-<I>OBP2A</I> plasmid as a template for amplification.              </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 22/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Removal of the signaling peptide from <I>OBP2A</I> and insertion of  <I>OBP2A</I> into the biobrick vector pSB1C3. </P></DIV><DIV id="nmcollapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/6: PCR of pIDT-<I>OBP2A</I> using primers designed for the removal of the signaling peptide.  </SPAN><DIV id="nmcollapsiblew2-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a PCR on the plasmid preparation of pIDT-<I>OBP2A</I> produced last week, using primers that have been designed in a way as to remove the signalling peptide at the start of the <I>OBP2A</I> gene.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> The next step will be to run the PCR product on a gel and perform a gel extraction on the part of the gel corresponding to the size of the <I>OBP2A</I> gene fragment.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/6: Gel extraction of <I>OBP2A</I> from the PCR performed on the 22/6.  </SPAN><DIV id="nmcollapsiblew2-2" class="collapse box-content"><P><B>Aim of experiment:</B> To perform a gel extraction of OBP2A from a PCR reaction mixture that was performed on the 22/6. Once gel extraction has been performed a subsequent gel will be done to determine if the gel extraction has been successful.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#gel extraction -modal" class="journal-protocol">Gel extraction </A></P><P><B>Results:</B> As it can be seen from the gel image, a band is present corresponding to the size of the <I>OBP2A</I> gene fragment.

 </P><P><B>Next Steps:</B> To perform a restriction of <I>OBP2A</I> in preparation for ligation.              
 </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/6:  Restriction, ligation and transformation of <I>OBP2A</I> into pSB1C3.  </SPAN><DIV id="nmcollapsiblew2-3" class="collapse box-content"><P><B>Aim of experiment:</B> To firstly restrict the gel extracted <I>OBP2A</I>. Secondly, to then ligate the restricted <I>OBP2A</I> into the biobrick vector pSB1C3. Third and finally to transform this plasmid into the JM110 <I>E.coli</I> strain.    
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#digest-modal" class="journal-protocol">Restriction Digest  </A>:<A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Ligation </A>:<A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Transformation  </A></P><P><B>Results:</B> N/A


                  </P><P><B>Next Steps:</B>If the transformation is successful and positive colonies form on the agar plate then these colonies will then be grown overnight in preparation for Miniprep.                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">25/6: Religation and transformation of <I>OBP2A</I> into pSB1C3. </SPAN><DIV id="nmcollapsiblew2-4" class="collapse box-content"><P><B>Aim of experiment:</B> The transformation performed on the 24/6 failed and as a result <I>OBP2A</I> will again ligated into the biobrick vector pSB1C3 and then transformed into JM110 <I>E.coli</I> strain.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Ligation:</A><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Transformation</A></P><P><B>Results:</B> N/A


                  </P><P><B>Next Steps:</B>If the transformation is successful and positive colonies form on the agar plate then these colonies will then be grown overnight in preparation for Miniprep.                   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">26/6: Preparation of overnight cultures of pSB1C3-<I>OBP2A</I>. </SPAN><DIV id="nmcollapsiblew2-5" class="collapse box-content"><P><B>Aim of experiment:</B> To produce overnight cultures of pSB1C3-<I>OBP2A</I> from positive colonies of JM110 <I>E.coli</I> strain from the transformation done on the 25/6.  
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results:</B> N/A


                  </P><P><B>Next Steps:</B>  Produce a plasmid preparation of pSB1C3-<I>OBP2A</I> in preparation for pre-sequence digest.                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/6: Plasmid preparation and pre-sequence digest of pSB1C3-<I>OBP2A</I>.   </SPAN><DIV id="nmcollapsiblew2-6" class="collapse box-content"><P><B>Aim of experiment:</B> To perform a plasmid preparation of overnight cultures of pSB1C3-<I>OBP2A</I> from positive colonies of JM110 <I>E.coli</I> strain from the transformation done on the 25/6. Then to subsequently perform a pre-sequence digest of the plasmid preparation.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results:</B> Insert table values!


                  </P><P><B>Next Steps:</B> Looking at the pre-sequence digest it has been decided that sample 5 will be sent for sequencing. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 29/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Correct insertion of <I>OBP2A</I> into pSB1C3 determined through sequencing.   </P></DIV><DIV id="nmcollapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/6: Plasmid preparation of pSB1C3-<I>OBP2A</I> is sent for sequencing.  </SPAN><DIV id="nmcollapsiblew3-1" class="collapse box-content"><P><B>Aim of Experiment:</B> Sent sample 5 of the plasmid preparation that was done on the 24/6 for sequencing to determine if <I>OBP2A</I> has successfully ligated into pSB1C3.   </P><P><B>Protocols Used:</B></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If sequencing comes back positive, then the next step will be to clone this gene insert into the two constituents of the two hybrid system.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/6: Production of overnight cultures of pSB1C3-<I>OBP2A</I> from sequenced colony.  </SPAN><DIV id="nmcollapsiblew3-2" class="collapse box-content"><P><B>Aim of Experiment:</B> Sequencing came back positive for pSB1C3-<I>OBP2A</I>. Soan overnight culture of pSB1C3-<I>OBP2A</I> from the sequenced colony will be done overnight in preparation for plasmid preparation.     </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B>Once the overnight culture has been given ample time to grow, the culture will then undergo a plasmid preparation. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/7: Plasmid Preparation of pSB1C3-<I>OBP2A</I>.  </SPAN><DIV id="nmcollapsiblew3-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To produce a plasmid preparation of the sequenced pSB1C3-<I>OBP2A</I> from the overnight culture set up on the 30/6.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Miniprep </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P> Insert plasmid concentration. 
                  <P><B>Next Steps:</B> the next step of experimentation is to use the plasmid preparation as a template for PCR using primer designed to separate the gene for use in the two hybrid system.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="box"><DIV class="labtitle">Week Beginning 12/7/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Ran PCR of <I>OBP2A</I> using primers for insertion of <I>OBP2A</I> into PUT18 and PT25. </P></DIV><DIV id="nmcollapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/6: PCR of <I>OBP2A</I> for insertion into bacterial two hybrid. </SPAN><DIV id="nmcollapsiblew4-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To PCR <I>OBP2A</I> for its insertion into the bacterial two hybris vectors; pUT18 and pT25. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B>Tthe next stage of experimentation will be to gel extract and restrict the PCR gene fragments.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 20/7/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Attempted insertion of <I>OBP2A</I> into the vectors for the bacterial two hybrid system.  </P></DIV><DIV id="nmcollapsiblew5" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">20/7: Gel extraction and restriction of the PCR of <I>OBP2A</I> from the 17/7  </SPAN><DIV id="nmcollapsiblew5-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a gel extraction and then subsequently a restriction of <I>OBP2A</I> from the PCR  produced on the 17/7 that used  primers designed for the insertion of<I>OBP2A</I> into the vectors for the bacterial two hybrid system, pUT18 and pT25.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#GelExtraction-modal" class="journal-protocol">Gel extraction</A><A data-toggle="modal" data-target="#Restriction-modal" class="journal-protocol">Restriction </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If the gel extraction and restriction is successful, then the next step will be to ligate these restrictions into the bacterial two hybrid vectors, pUT18 and pT25.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">21/7: Ligation &amp; transformation of <I>OBP2A</I> into pUT18 and pT25.     </SPAN><DIV id="nmcollapsiblew5-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a ligation and  then subsequently a transformation  of <I>OBP2A</I> into the vectors for the bacterial two hybrid system, pUT18 and pT25. Whilst using JM110 <I>E.coli</I> cells as a chassis for transformation. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#Liagtion-modal" class="journal-protocol">Ligation</A><A data-toggle="modal" data-target="#Transformation-modal" class="journal-protocol">Transfromation </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If transformation is successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight colonies in preparation for purification and sequencing.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/7: re-ligation and transformation of <I>OBP2A</I> into pUT18 and PT25.
  </SPAN><DIV id="nmcollapsiblew5-3" class="collapse box-content"><P><B>Aim of Experiment:</B> As the transformation from the 21/7 failed - no positive colonies grew on the antibiotic plates. A re-ligation and transformation of <I>OBP2A</I> into pUT18 and pT25 was done.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">ligation</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If transformation is successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight colonies in preparation for purification and sequencing.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/7: Production of overnight cultures of pUT18-OBP2A. </SPAN><DIV id="nmcollapsiblew5-4" class="collapse box-content"><P><B>Aim of Experiment:</B> Although the transformation of <I>OBP2A</I> into pT25 was unsuccessful, the transformation of <I>OBP2A</I> into pUT18 did produce positive colonies. These colonies were grown overnight in preparation for Miniprep.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> Once the cells have grown overnight they will undergo plasmid preparation for subsequent sequencing.  </P></DIV></DIV></DIV></DIV><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/7: Plasmid preparation and pre-sequencing digest of pUT18-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew5-5" class="collapse box-content"><P><B>Aim of Experiment:</B> The overnight cultures of pUT18-<I>OBP2A</I>underwent plasmid preparation and then a subsequent pre-sequence digest before being sent for sequencing.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Miniprep  </A><A data-toggle="modal" data-target="#pre-sequence digest-modal" class="journal-protocol">Pre-sequence digest</A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If sequencing comes back positive for pUT18-<I>OBP2A</I> then this can now be implemented into the bacterial two hybrid. Second attempt at cloning of <I>OBP2A</I> into pT25 will be done next week.                </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 27/7/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Successful insertion of <I>OBP2A</I> into both bacterial two hybrid vectors pUT18 and pT25.   </P></DIV><DIV id="nmcollapsiblew6" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/7: PCR  and gel extraction of <I>OBP2A</I> for preparation of insertion into  pT25.   </SPAN><DIV id="nmcollapsiblew6-1" class="collapse box-content"><P><B>Aim of Experiment:</B> As he first attempt of <I>OBP2A</I> insertion into pT25 was unsuccessful, a further PCR and gel extraction was done.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> The next stage of experimentation will be to restrict <I>OBP2A</I> for subsequent ligation into pT25.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/7: Restriction of <I>OBP2A</I> for pT25 and positive sequencing of pUT18-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew6-2" class="collapse box-content"><P><B>Aim of Experiment:</B> A restriction of <I>OBP2A</I> for PT25 was performed. Sequencing came back positive for pUT18-<I>OBP2A</I>, so pUT18-<I>OBP2A</I> can now be implemented in the bacterial two hybrid system.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#Restriction-modal" class="journal-protocol">Restriction</A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> The next stage of experimentation will be to ligate <I>OBP2A</I> into the pT25 vector.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/7: Ligation and transformation of <I>OBP2A</I> into pT25.    </SPAN><DIV id="nmcollapsiblew6-3" class="collapse box-content"><P><B>Aim of Experiment:</B> As the restriction was done yesterday a ligation and transformation of <I>OBP2A</I> into the bacterial two hybrid vector pT25 was done today.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">ligation</A><A data-toggle="modal" data-target="#Transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If transformation is successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight cultures of those colonies in preparation for purification and sequencing.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">31/7: Produced overnight culture from positive colonies of pT25-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew6-4" class="collapse box-content"><P><B>Aim of Experiment:</B> The transformation of <I>OBP2A</I> into pT25 produced positive colonies, so overnight cultures of these colonies were done today.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> Once the cells have grown overnight they will undergo plasmid preparation for subsequent sequencing.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/8: Plasmid preparation and pre-sequencing digest of pT25-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew6-5" class="collapse box-content"><P><B>Aim of Experiment:</B> The overnight cultures of pT25-<I>OBP2A</I>underwent plasmid preparation and then a subsequent pre-sequence digest before being sent for sequencing.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Miniprep  </A><A data-toggle="modal" data-target="#pre-sequence digest-modal" class="journal-protocol">Pre-sequence digest</A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If sequencing comes back positive for pT25-<I>OBP2A</I> then this can now be implemented into the bacterial two hybrid system. The results from the sequencing should arrive early on the 3/8.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 3/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P></DIV><DIV id="nmcollapsiblew7" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/8: Transformation of pUT18-<I>OBP2A</I> into BTH101 <I>E.coli</I> strain.    </SPAN><DIV id="nmcollapsiblew7-1" class="collapse box-content"><P><B>Aim of Experiment:</B> In order to gauge the interaction between the two separate parts of my protein both pT25-<I>OBP2A</I> and pUT18-<I>OBP2A</I> must be transformed into the same cell. In this case the transformation is being done with the BTH101 <I>E.coli</I> strain.This is the first of two transformations.     </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Trasformation</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If the transformation is successful, then the next stage of experimentation will be to retransform this culture of cells using the pT25-<I>OBP2A</I> plasmid.      </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">4/8:Day culture and transformation of BTH101 <I>E.coli</I> containing pUT18-<I>OBP2A</I> with pT25<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew7-2" class="collapse box-content"><P><B>Aim of Experiment:</B> The transformation performed yesterday has produced positive colonies. These positive colonies were then grown throughout the day before being made competent and used in the transformation of pT25-<I>OBP2A</I></P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If the transformation is successful then the next stage of experimentation will be to observe  the interaction of the two subunits in vivo using MacConkey agar plates.     </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">5/8:Day culture and plating of positive colonies from transformation onto MacConkey agar.  </SPAN><DIV id="nmcollapsiblew7-3" class="collapse box-content"><P><B>Aim of Experiment:</B> Positive colonies have formed on the agar plate of yesterday's transformation. Since this means that those positive colonies contain both bacterial two hybrid vectors then it is now possible to plate them on MacConkey agar as this will give us a visual demonstration of protein interaction.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#Transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> It has been decided that whilst the <I>E.coli</I> strain BTH101 that is being used in this experiment is useful for protein interaction. A better <I>E.coli</I> strain called MG1655(-cya) will be used instead as it is far more stable. This means that tomorrow overnights will be prepared of MG1655(-cya)for transformation with pUT18 and pT25 containing the different subunits of <I>OBP2A</I>.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">6/8: Overnight of MG1655(-cya)</SPAN><DIV id="nmcollapsiblew7-4" class="collapse box-content"><P><B>Aim of Experiment:</B> Performed overnight of MG1655(-cya)<I>E.coli</I> in preparation for transformation with pUT18-<I>OBP2A</I> and pT25-<I>OBP2A</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> Once the overnight culture has been give the correct amount of time to grow these cells will be made competent and transformed with pUT18-<I>OBP2A</I> and pT25-<I>OBP2A</I>.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">7/8: Results of MacConkey agar growth and transformation of  MG1655(-cya)with pUT18-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew7-5" class="collapse box-content"><P><B>Aim of Experiment:</B> Colonies of BTH101 that have formed on the MacConkey agar have grown to sufficient size that it is now possible to see that the colonies are uptaking a small amount of the red dye - which possibly suggests that the two separated parts of <I>OBP2A</I> are interacting. This will be shown through the use of the betagalactosidae assay. Cells of MG1655(-cya) from yesterdays overnight were firstly made competent then subsequently transformed with pUT18-<I>OBP2A</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Miniprep  </A><A data-toggle="modal" data-target="#pre-sequence digest-modal" class="journal-protocol">Pre-sequence digest</A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If the transformation of MG1655(-cya) with pUT18-<I>OBP2A</I> is successful then the next step of experimentation will be retransform positive colonies with pT25-<I>OBP2A</I>.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P></DIV><DIV id="nmcollapsiblew8" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/8: Plating of Controls in preparation of betaglactosidase assay   </SPAN><DIV id="nmcollapsiblew8-1" class="collapse box-content"><P><B>Aim of Experiment:</B> In order to perform the betagalactosidase assay several controls are implemented to allow for a concise interpretation of the results. This first assay attempt will consist of one positive control, one negative control and one intermediate control.       </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Trasformation</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> The next step will be to perform overnights in preparation of the betagalctosidase assay.      </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/8: Overnights cultures in preparation of betagalctosidase assay   </SPAN><DIV id="nmcollapsiblew8-2" class="collapse box-content"><P><B>Aim of Experiment:</B> Prepared overnights of positive colonies of MG1655(-cya) containing pUT18-<I>OBP2A</I> and pT25-<I>OBP2A</I>, along with the control colonies in preparation for the betagalactosidase assay. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> Once the cells have been given sufficient time to grow then, the next stage of experimentation will be to inoculate those colonies into free liquid broth until they reach an O.D of 0.4 before being pellet and frozen for the assay. This will be done tomorrow.     </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">12/8:Innoculation and preparation of cultures for betagalctosidase assay tomorrow.   </SPAN><DIV id="nmcollapsiblew8-3" class="collapse box-content"><P><B>Aim of Experiment:</B> A small subsample of each overnight was inoculated into fresh LB, there were two done for each colony. These were then left to grow until the O.D of each culture was between 0.3-0.5. Once there 3 replicas of 1ml were taken form each culture before being spun down, the supernatant removed and frozen.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#Transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> The samples are now ready for the betagalcatsidase assay which will be done tomorrow.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">13/8: First attempt at determination of the protein interaction of the <I>OBP2A</I> subunits using the betagalctosidase assay. </SPAN><DIV id="nmcollapsiblew8-4" class="collapse box-content"><P><B>Aim of Experiment:</B> To determine the millers activity of the subunits of <I>OBP2A</I> when interacting in vivo as a basis of the level of interaction between them. This was done using the betagalactosidase assay. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> Since it isn't clear from the data that the subunits of <I>OBP2A</I> aren't interaction in vivo a further assay will be prepared using more controls. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/8: Plating of Controls in preparation of betaglactosidase assay    </SPAN><DIV id="nmcollapsiblew8-5" class="collapse box-content"><P><B>Aim of Experiment:</B> Colonies of BTH101 that have formed on the MacConkey agar have grown to sufficient size that it is now possible to see that the colonies are uptaking a small amount of the red dye - which possibly suggests that the two separated parts of <I>OBP2A</I> are interacting. This will be shown through the use of the betagalactosidae assay. Cells of MG1655(-cya) from yesterdays overnight were firstly made competent then subsequently transformed with pUT18-<I>OBP2A</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Miniprep  </A><A data-toggle="modal" data-target="#pre-sequence digest-modal" class="journal-protocol">Pre-sequence digest</A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If the transformation of MG1655(-cya) with pUT18-<I>OBP2A</I> is successful then the next step of experimentation will be retransform positive colonies with pT25-<I>OBP2A</I>.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P></DIV><DIV id="nmcollapsiblew9" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/8: Plating of Controls in preparation of betaglactosidase assay  </SPAN><DIV id="nmcollapsiblew9-1" class="collapse box-content"><P><B>Aim of Experiment:</B> In order to perform the betagalactosidase assay several controls are implemented to allow for a concise interpretation of the results. This first assay attempt will consist of one positive control, one negative control and one intermediate control.       </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Trasformation</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> The next step will be to perform overnights in preparation of the betagalctosidase assay.      </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/8: Overnights cultures in preparation of betagalctosidase assay   </SPAN><DIV id="nmcollapsiblew9-2" class="collapse box-content"><P><B>Aim of Experiment:</B> Prepared overnights of positive colonies of MG1655(-cya) containing pUT18-<I>OBP2A</I> and pT25-<I>OBP2A</I>, along with the control colonies in preparation for the betagalactosidase assay. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> Once the cells have been given sufficient time to grow then, the next stage of experimentation will be to inoculate those colonies into free liquid broth until they reach an O.D of 0.4 before being pellet and frozen for the assay. This will be done tomorrow.     </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">12/8:Innoculation and preparation of cultures for betagalctosidase assay tomorrow.   </SPAN><DIV id="nmcollapsiblew9-3" class="collapse box-content"><P><B>Aim of Experiment:</B> A small subsample of each overnight was inoculated into fresh LB, there were two done for each colony. These were then left to grow until the O.D of each culture was between 0.3-0.5. Once there 3 replicas of 1ml were taken form each culture before being spun down, the supernatant removed and frozen.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#Transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> The samples are now ready for the betagalcatsidase assay which will be done tomorrow.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">13/8: First attempt at determination of the protein interaction of the <I>OBP2A</I> subunits using the betagalctosidase assay. </SPAN><DIV id="nmcollapsiblew9-4" class="collapse box-content"><P><B>Aim of Experiment:</B> To determine the millers activity of the subunits of <I>OBP2A</I> when interacting in vivo as a basis of the level of interaction between them. This was done using the betagalactosidase assay. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> Since it isn't clear from the data that the subunits of <I>OBP2A</I> aren't interaction in vivo a further assay will be prepared using more controls. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/8: Plating of Controls in preparation of betaglactosidase assay    </SPAN><DIV id="nmcollapsiblew9-5" class="collapse box-content"><P><B>Aim of Experiment:</B> Colonies of BTH101 that have formed on the MacConkey agar have grown to sufficient size that it is now possible to see that the colonies are uptaking a small amount of the red dye - which possibly suggests that the two separated parts of <I>OBP2A</I> are interacting. This will be shown through the use of the betagalactosidae assay. Cells of MG1655(-cya) from yesterdays overnight were firstly made competent then subsequently transformed with pUT18-<I>OBP2A</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Miniprep  </A><A data-toggle="modal" data-target="#pre-sequence digest-modal" class="journal-protocol">Pre-sequence digest</A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If the transformation of MG1655(-cya) with pUT18-<I>OBP2A</I> is successful then the next step of experimentation will be retransform positive colonies with pT25-<I>OBP2A</I>.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 17/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P></DIV><DIV id="nmcollapsiblew10" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/8: Second attempt at determination of the protein interactions of the <I>OBP2A</I> subunits using the betagalctosidase assay.   </SPAN><DIV id="nmcollapsiblew10-1" class="collapse box-content"><P><B>Aim of Experiment:</B> to test the hypotehesis that if a protein is seperated and the parts are reintrduced in vivo will they recombine, we have prepared a second attempt at performing the betagalactosidase assay in order to test the interaction of the two protein parts.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Betagalctosidase assay</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> The data suggest that the two parts of <I>OBP2A</I> do not nteract in vivo, however further assays will be donte to confirm that this is in fact the case. </A></P><P><B>Next Steps:</B> The next stage of experimentaton will be to switch the positioning of the two subunits of <I>OBP2A</I> in pUT18 and pT25 by having the smaller subunit in pUT18 and the large subunit in pT25. Along with this another betagalctosiase assay will be performed however this time it will be done under anerobic conditions.       </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title"> 20/8: PCR of <I>OBP2A</I> in preparation of pUT18 and pT25 insertion   </SPAN><DIV id="nmcollapsiblew10-2" class="collapse box-content"><P><B>Aim of Experiment:</B> Performed a PCR of <I>OBP2A</I> using primers for pU1T18 and pT25 insertion which have been designed to place the smaller <I>OBP2A</I> into pUT18 and the larger subunit into pT25. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If PCR is successful then the next stage in experimentation will be to perform a gel extraction and restriction in perpetration for ligation and transformation of <I>OBP2A</I> into pUT18 and pT25.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">21/8: Second attempt at a PCR of <I>OBP2A</I> in preparation of pUT18 and pT25 insertion   </SPAN><DIV id="nmcollapsiblew10-3" class="collapse box-content"><P><B>Aim of Experiment:</B> The PCR performed yesterday failed as no bands were found to be present on the gel during gel viewing. As a result of this a subsequent PCR was done today.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B>  If PCR is successful then the next stage in experimentation will be to perform a gel extraction and restriction in perpetration for ligation and transformation of <I>OBP2A</I>in pUT18 and pT25.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/8: Second attempt at PCR of <I>OBP2A</I> for insertion into pUT18 and pT25 failed </SPAN><DIV id="nmcollapsiblew10-4" class="collapse box-content"><P><B>Aim of Experiment:</B> Asecond attempt at preparing <I>OBP2A</I> for insertion into the two vectors of te bacterial two hybrid system(pUT18 and pT25) has failed! As a result this will be left until next week, so that the rest of the week can be dedicated to preparing  and undertaking  another betagalcatosidae assay, that will be done under anaerobic conditions.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">N/A </A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> The next stage of experimentation is to prepare for a anerobic betagalctosidase assay of my <I>OBP2A</I> in pT25 and pUT18 in the orignally designed conformation of large subunit in pUT18 and the smalled subunit in pT25.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 24/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P></DIV><DIV id="nmcollapsiblew11" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/8: Preperation of samples for the anerobic betagalctosidase assay of <I>OBP2A</I> in pUT18 and pT25  </SPAN><DIV id="nmcollapsiblew11-1" class="collapse box-content"><P><B>Aim of Experiment:</B> Prepared the samples of <I>OBP2A</I> for the anerobic betagalctosidase assay, so they can be frozen overnight in preperation for the assay itself that will be done tomorrow.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Assay Preperation</A></P><P><B>Results: </B></P><P><B>Next Steps:</B> The next stage of experimentation will be to perform the betagalctosidase assay itself, this will most likely be done tomorrow.         </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title"> 25/8: Perfored an anaerobic betagalcosidase assay in order to determine the level of interaction of <I>OBP2A</I> in the bacterial two hybrid vectors: pUT18 and pT25.   </SPAN><DIV id="nmcollapsiblew11-2" class="collapse box-content"><P><B>Aim of Experiment:</B> Performed an anaerobic betagalctosidase assay of <I>OBP2A</I> in the bacterial two hybrid vectors: pUT18 and pT25. This is to determine the level of interaction that the smaller subunit of <I>OBP2A</I> has with its larger subunit in vivo. This particular assay has two positive controls, two negative controls and one regulatory control.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> Betagalctosidae assay </A> p&gt;
         </P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> Must add in results.  </A></P><P><B>Next Steps:</B> As the results have shown that the two subunits of <I>OBP2A</I> in fact don't seem to be interacting in vivo - a possible reason might be aggregation of the two subunits preventing interaction. Then the next stage of experimentation will be to test out the interaction when the subunit are switch with regards to the vectors pUT18 and pT25. This will be the goal of the following weeks once the team is back from London.     </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV><DIV class="container"><DIV class="footer-btn"><A href="https://2015.igem.org/Team:Dundee/Introduction_Video" class="btn btn-info btn-lg" role="button">Watch Our Introduction Video</A></DIV></DIV><FOOTER><DIV class=""><DIV class="container"><DIV class="col-md-3"><A href="http://www.dundee.ac.uk/" target="_blank"><H4>Dundee University</H4></A><P>Located on the east coast of Scotland, The University of Dundee is one of Scotland's leading research institutes.</P></DIV><DIV class="col-md-3"><H4>Social Media</H4><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="https://www.youtube.com/channel/UCJHBLvLhjzGqHqdjABjOY-w">youtube.com/DundeeiGEM2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.facebook.com/igemdundee2015">facebook.com/iGEMDundee2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.twitter.com/dundeeigemteam">twitter.com/DundeeiGEMTeam</A></DIV></DIV></DIV></DIV></DIV></FOOTER><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>