<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_Lab_Book_FluID skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/Lab Book/FluID</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="modal fade" id="PCR-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>PCR Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the  section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY color="black"><TR><TD width="312" valign="top"><P align="center"><STRONG>Reactants</STRONG></P></TD><TD width="312" valign="top"><P align="center"><STRONG>Volume (µl)</STRONG></P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DNA
                </P></TD><TD width="312" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DMSO
                </P></TD><TD width="312" valign="top"><P align="center">
                    2.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   100 µM Forward Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   100 µM Reverse Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    5 x Herculase Buffer
                </P></TD><TD width="312" valign="top"><P align="center">
                    10
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                   10 mM dNTPs
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    H<SUB>2</SUB>O
                </P></TD><TD width="312" valign="top"><P align="center">
                    34.5
                </P></TD></TR></TBODY></TABLE><P>
    Elongation time was altered according to the length of the sequences.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="restrictiondigest-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Restriction Digests Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P><FONT color="#000000">
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the    <STRONG>Lab Book</STRONG> section.
</FONT></P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="170" valign="top" style="color:#000000;"><P align="center"><STRONG>Post Plasmid Purification</STRONG></P></TD><TD width="151" valign="top"><P align="center"><STRONG>Post PCR</STRONG></P></TD><TD width="139" valign="top"><P align="center"><STRONG>gBlock</STRONG></P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>DNA</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1 mg
                </P></TD><TD width="151" valign="top"><P align="center">
                    50 µl
                </P></TD><TD width="139" valign="top"><P align="center">
                    100 ng
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Cutsmart Buffer (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    6
                </P></TD><TD width="151" valign="top"><P align="center">
                    6
                </P></TD><TD width="139" valign="top"><P align="center">
                    6
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Water (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    As required
                </P></TD><TD width="151" valign="top"><P align="center">
                    1
                </P></TD><TD width="139" valign="top"><P align="center">
                    As required
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Enzyme 1 (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="151" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="139" valign="top"><P align="center">
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG>Enzyme 2 (µl)</STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="151" valign="top"><P align="center">
                    1.5
                </P></TD><TD width="139" valign="top"><P align="center">
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P align="center"><STRONG><U>Total Volume (µl)</U></STRONG></P></TD><TD width="170" valign="top"><P align="center">
                    30
                </P></TD><TD width="151" valign="top"><P align="center">
                    60
                </P></TD><TD width="139" valign="top"><P align="center">
                    30
                </P></TD></TR></TBODY></TABLE><P>
    The digests were incubated at 37<SUP>o</SUP>C for 3 hours.
    
    Note that when performing plasmid digestions, 2.5 µl of alkaline phosphatase and 2.5 µl of Cutsmart Buffer was added at the 2 hour and 2.5 hour mark. Digested PCR products were separated from restriction enzymes and buffers using Strataclean resin (Aligent Technologies). Digested vectors were purified on a 1% agarose gel and then extracted. 
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ligation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Ligations</B></CENTER></DIV></DIV><DIV class="modal-body"><P><FONT color="black">
Ligation reactions were set up according to the table below.
</FONT></P><TABLE border="1" cellspacing="0" cellpadding="0" class="text"><TBODY><TR><TD width="111" valign="top"><P align="center"><STRONG>Vector (µl)</STRONG></P></TD><TD width="111" valign="top"><P align="center"><STRONG>2:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>3:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>4:1 (Vector: Insert) [µl]</STRONG></P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>T4 DNA Ligase </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Ligase Buffer </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Vector </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Insert</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    -
                </P></TD><TD width="111" valign="top"><P align="center">
                    2
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>H<SUB>2</SUB>0</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    7
                </P></TD><TD width="111" valign="top"><P align="center">
                    5
                </P></TD><TD width="129" valign="top"><P align="center">                4
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD></TR></TBODY></TABLE><P>
    The reactions were incubated at room temperature for at least 3 hours.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Transformation Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>

    1. 100 µl of chemocompetent <I>E.coli</I> cells were defrosted. Thaw on ice.
</P><P>
    2. Agar plates were taken out of 4 <SUP>o</SUP>C to warm to room temperature.
</P><P>
    3. 10 µl of ligation reaction or 2 µl of plasmid purified DNA was added to cells. The cells and DNA were mixed by gently flicking the tube.
</P><P>
    4. Cell and DNA mixture was left on ice for 20 minutes.
</P><P>
    5. Mixture was placed in 42 <SUP>o</SUP>C water bath for 90 seconds.
</P><P>
    6. Mixture was put back on ice for 2 minutes.
</P><P>
    7. 1 ml of LB (without antibiotic) was added to cells.
</P><P>
    8. Cells were allowed to grow for 1 hour at 37 <SUP>o</SUP>C in a shaking incubator.
</P><P>
    9. Cells were then spun down and the supernatant discarded.
</P><P>
    10. The pellet was resuspended in the 100 µl of remaining media.
</P><P>
    11. Cells were plated on agar plates with appropriate antibiotic.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="miniprep-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Plasmid Purification</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. 1-5 ml bacterial overnight culture was pelleted by centrifugation at &gt;8000 rpm for 3 minutes at room temperature.
</P><P>
    2. Pelleted bacterial cells were resuspended in 250 µl of Buffer P1 and transferred to a microcentrifuge tube.
</P><P>
    3. 250 µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution becomes clear.
</P><P>
    4. 350 µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.
</P><P>
    5. The mixture was centrifuged for 10 minutes at 13000 rpm in a microcentrifuge.
</P><P>
    6. The supernatant was applied to a QIAprep spin column by pipetting.
</P><P>
    7. The QIAprep spin column was washed with 500 µl of Buffer PB by centrifugation for 1 minute and flow through discarded.
</P><P>
    8. 750µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute and flow through discarded.
</P><P>
    9. The QIAprep spin column was spun for an additional minute to remove residual wash buffer.
</P><P>
    10. The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 50 µl of H<SUB>2</SUB>O to the QIAprep spin column and centrifuging for 1 minute.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="onc-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Overnight Cultures </B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>
    1. Four colonies were picked from the desired plate.
</P><P>
    2. For each colony, 5 ml of fresh LB plus the appropriate antibiotics was inoculated and grown overnight at 37 <SUP>o</SUP>C.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sds-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>SDS-PAGE and Western Blotting</B></CENTER></DIV></DIV><DIV class="modal-body"><FONT color="black"><P>
    1. 50 µl of an overnight culture was used to inoculate 5 ml of fresh LB containing the appropriate antibiotics and left to grow at 37<SUP>o</SUP>C in a
    shaking incubator.
</P><P>
    2. Once the OD<SUB>600</SUB>=0.6-0.8, 1 mM of IPTG was added to the culture and left to grow for a further 3 hours.
</P><P>
    3. 1 ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.
</P><P>
    4. Meanwhile, 950 µl of 2x Laemmli sample buffer was added to 50 µl of B-mercaptoethanol.
</P><P>
    5. The supernatant was discarded from the samples and the pellet resuspended in 200 µl of the above solution.
</P><P>
    6. The samples were then boiled at 90<SUP>o</SUP>C for 10 minutes.
</P><P>
    7. Afterwards, they were spun down for 1 minute in a microcentrifuge.
</P><P>
    8. At this stage, the samples could be stored at -20<SUP>o</SUP>C or ran on a gel immediately.
</P><P>
    9. Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the
    stacking gel at which point the voltage was increased to 200V for 20 minutes.
</P><P>
    10. One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.
</P><P>
    11. After the proteins had been transferred onto the membrane using a semi dry transfer ran for 40 minutes at 5V, the membrane was blocked in 5% Marvel milk overnight at 4<SUP>o</SUP>C.
</P><P>
    12. The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10 ml of 1xTBS buffer to which the primary anti-His antibody was
    added and left for one hour. The protein ladder was cut off before adding the primary antibody.
</P><P>
    13. The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10 ml of 1xTBS buffer, to which the secondary antibody was
    added and left for one hour.
</P><P>
    14. The membrane was then washed in 3x 10 ml 1xTBST buffer for 5 minutes.
</P><P>
    15. The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off
    at step 12.
</P><P>
    16. Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.
</P></FONT></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Growth Curve Assay Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>    1. 5ml overnight cultures of the appropriate strains were set up.
</P><P>
    2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
</P><P>
    3. Three were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced.
</P><P>
    4. 200 µl of each culture was loaded onto a 96 well microtitre plate with 4 repeats.
</P><P>
    5. The cultures were left to grow for 16 hours in the plate reader which took OD readings every 10 minutes.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Growth Curve Assay Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. 5ml overnight cultures of the appropriate strains were set up.
</P><P>
    2. The next day, 4x 995 µl of plain LB containing 1 µl ampicillin and 1 µl of chloramphenicol were inoculated with 5 µl of the overnight culture.
</P><P>
    3. 20 x 200 µl aliquots were loaded onto a 96 well microtitre plate.
</P><P>
    4. After the OD<SUB>600</SUB>=~0.6, three cultures were induced with 0.5 mM, 1 mM and 2 mM of IPTG and one was left uninduced (with 4 repeats).
</P><P>
    5. The cultures were left to grow for 20 hours in the plate reader which took OD readings every 30 minutes.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="pp-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Protein Purification Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><P>
    1. A 150 ml overnight culture was set up for the desired protein.
</P><P>
    2. 3 x 1 L of fresh LB containing 1 ml of appropriate antibiotics was inoculated with each 50 ml overnight culture and left to grow until an OD<SUB>600</SUB>
    =0.6-1 in a shaking incubator at 37<SUP>o</SUP>C.
</P><P>
    3. The cultures were subsequently induced with 1 mM IPTG and left to grow for 6 hours in a shaking incubator at 20<SUP>o</SUP>C.
</P><P>
    4. They were then centrifuged for 20 minutes at 4200 rpm at 4<SUP>o</SUP>C.
</P><P>
    5. The pellets were then resuspended in 5 ml of ice cold 50 mM 7.5 Tris and transferred to a 50 ml falcon tube and then topped up to 40 ml with 50 mM 7.5 Tris.
</P><P>
    6. They were then spun down again at 4000 rpm for 15 minutes at 4<SUP>o</SUP>C, the supernatant discarded and pellets stored at -20<SUP>o</SUP>C.
</P><P>
    7. Once purification could proceed, the pellets were thawed on ice and resuspended in 40 ml of 50 mM Tris containing DNAase, 1 x protease inhibitor tablet
    and lysozyme. (Note: If purifying a membrane protein, 1% DDM and 5 M urea was also added).
</P><P>
    8. The cells were lysed using a cell disruptor.
</P><P>
    9. Afterwards the lysate was spun down in a J25.50 rotor at 15000 rpm at 4<SUP>o</SUP>C for 40 minutes.
</P><P>
    10. The supernatant was removed and transferred to a 50 ml falcon tube. The pellet was kept for testing. Both were kept on ice.
</P><P>
    11. Using FPLC, the supernatant was loaded onto a Ni<SUP>2+</SUP> column.
</P><P>
    12. The protein was eluted using an increasing concentration of imidazole. The two buffers used were Buffer A: 50 mM Tris, 200 mM NaCl, 20 mM imidazole and
    Buffer B: 50 mM Tris, 200 mM NaCl, 1 M Imidazole. (Note: If purifying a membrane protein 0.02% DDM was added to these buffers.)
</P><P>
    13. The fractions corresponding to any peaks were retained and concentrated using an appropriate filter size. A small amount of each fraction could be ran
    on an SDS gel at this stage.
</P><P>
    14. Using FPLC, the sample was then loaded onto a SEC column and any fractions corresponding to a peak were ran on an SDS gel to check for the presence of
    the desired protein.
</P><P>
    15. Fractions containing the desired protein were collated, to which 20% glycerol was added.
</P><P>
    16. The samples were then flash frozen and stored at -80<SUP>o</SUP>C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ap-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Assay Preparation</B></CENTER></DIV></DIV><DIV class="modal-body"><OL><LI> Take 3 independent colonies from each plate of your controls and samples and place each colony into 5 ml of LB with the appropriate antibiotics. </LI><LI> Leave to grow overnight at 30°C.</LI><LI> Take 50 μl of each overnight and inoculate 5 ml of fresh LB containing the appropriate antibiotics. Do this twice for every independent colony. </LI><LI> Allow these to grow at 30°C shaker until the O.D of each inoculate is between 0.3-0.5</LI><LI> Once the O.D is between 0.3-0.5 take 3 x 1 ml samples of each culture and place into separate Eppendorf tubes.</LI><LI> Take a further 1 ml and place into a cuvette and  subsequently measure the OD<SUB>600</SUB> marking down the OD for each replica. </LI><LI> Spin down the 1 ml cultures of each replica at 13,000 rpm for 3 minutes. </LI><LI> Remove the supernatant from each 1 ml culture and freeze cell pellet for later use in the β-galactosidase Assay. </LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bga-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>β-Galactosidase Assay</B></CENTER></DIV></DIV><DIV class="modal-body"><P><OL><LI> Resuspend cell pellet in 800 μl of buffer 2 (100 mM dibasic sodium phosphate (Na2HPO4); 20 mM KCl ; 2 mM MgSO4 ;0.8 mg/mL CTAB (hexadecyltrimethylammonium bromide) ; 0.4 mg/mL sodiumdeoxycholate) and subsequntly add 5.4 μL/mL beta-mercaptoethanol. </LI><LI> Add 50 μl of SDS (0.1%) to each sample. </LI><LI> Add 50 μl of Chloroform to each sample and vortex for 30 seconds. </LI><LI> Make substrate solution (60 mM Na2HPO4; 40 mM NaH2PO4; 1 mg/mL o-nitrophenyl-β-D-Galactoside (ONPG); 2.7 μL/mL β-mercaptoethanol)</LI><LI> Add 200 μl of substrate solution to each sample marking down the time it was added in relation to the first sample. </LI><LI> Wait for an observable color change and mark down the time that the color change occurred.</LI><LI> Add 600 μl of NaCO3(1M) once the color change has occurred.</LI><LI> Spin down the samples at 13,000 rpm for 3 minutes and place the supernatant into a cuvette. </LI><LI> Measure the OD<SUB>420</SUB>. </LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="platereader-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plate Reader Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>A 5 ml overnight culture of the desired strain was grown at 37 <SUP>o</SUP>C.</LI><LI>Equalise OD<SUB>600</SUB> if required. Dilute culture 1:1000.</LI><LI>Aliquot 198 µl into the required amount of wells in a 96-well plate.</LI><LI>Add additional substances and controls as required.</LI><LI>Preheat platereader to required temperature and start experiment.</LI><LI>Retrieve and analyse data.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="competent-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Competent Cells Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>A 5 ml overnight culture of the desired strain was grown at 37<SUP>o</SUP>C.</LI><LI>Subsample <SUP>1</SUP>/<SUB>10</SUB> of the volume of the subculture. I.e. 1 ml of the overnight culture into 10 ml fresh LB.</LI><LI>Grow subculture at at 37 <SUP>o</SUP>C for 2 h.</LI><LI>Spin down subculture at 4000 rpm at 4<SUP>o</SUP>C for 10 min.</LI><LI>Resuspend the pellet in ice cold transformation buffer. Use <SUP>1</SUP>/<SUB>10</SUB> of the initial volume of the subculture.</LI><LI>Use 100 µl per transformation.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="plating-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plating Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>Fresh agar plates were inoculated with cells on the toothpick.</LI><LI>Agar plates were incubated at 4<SUP>o</SUP>C over night.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="tf-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Tryptophan Fluorescence Protocol</DIV></DIV><DIV class="modal-body"><P>
    1. Intensity of tryptophan fluorescence of PotD was measured on a fluorimeter using an excitation of 280 nm and emission of 350 nm.
</P><P>
    2. Increasing concentrations of spermidine were added in 50 µM increments to give a final concentration of 250 µM spermidine to 5 µM of purified PotD.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sequencing-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Sequencing Protocol</DIV></DIV><DIV class="modal-body"><P>
    For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
      <UL><LI>Volume of sample equivalent to 550ng DNA</LI><LI>2 µl of 3 mM primer</LI><LI>Remaining volume of water for a total volume of 30 µl</LI></UL></P><P>
      The samples were submitted to <A href="https://www.dnaseq.co.uk/" target="_blank">DNA sequencing and services</A>.
    </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="overnightculture-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Overnight Culture Protocol</DIV></DIV><DIV class="modal-body"><P><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>5 ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.</LI><LI>Tubes were incubated at 37<SUP>o</SUP>C in a rotary incubator at 200rpm over night.</LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="gelextraction-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Gel Extraction Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.</LI><LI>The gel slice was transferred into a microcentrifuge tube. 700 µl of Buffer QG was added.</LI><LI>The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.</LI><LI>After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.</LI><LI>230 µl of isopropanol was added to the sample and mixed.</LI><LI>A QIAquick spin column was placed in a 2 ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>500 µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>750 µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>The QIAprep spin column was spun for an additional minute to remove residual wash buffer.</LI><LI>The QIAprep spin column was placed in a clean 1.5 ml microcentrifuge and the DNA eluted by adding 30 µl of H2O to the QIAprep spin column and centrifuging for 1 minute.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: <B>Amplification of <I>LbpA</I></B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: LbpA PCR. Lane 1: PCR Product, Lane 2: 1kb Ladder and Marker.</B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: LbpA PCR for Cloning into pQE80-L. Lane 1: LbpA PCR Product, Lane 2: 1kb Ladder and Marker</B> The image shows the gel ran of the LbpA PCR product. The expected size of the fragment was 2832 bp which corresponds to the observed band on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: LbpA Culture Samples. Lane 1: Sample from uninduced culture. Lane 2: Sample from culture induced with 1mM IPTG.</B> It seems that on comparison of the two cultures, that inducing expression of LbpA causes the cells to die given the significant reduction of protein levels visible on the gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: OD<SUB>600</SUB> Readings from Uninduced Culture and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG).</B> The table shows that after being induced with IPTG the cultures stop growing since their OD<SUB>600</SUB> readings are notably lower than that of the uninduced control. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, the induced cultures stop growing when LbpA expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, after reaching an OD<SUB>600</SUB> of 0.6,  the induced cultures stop growing when LbpA starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have dealt with the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="lbpa-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: SDS Gel of Samples Taken from Uninduced Control and 1 mM IPTG Induced Culture 6 hours After Induction</B> For the induced culture, a faint band just above the 100 kDa marker is observable on the gel which corresponds to the expected size of LbpA.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 1: Gel of <I>SBP</I> PCR Product for Cloning into pQE80-L. Lane 1: <I>SBP</I> PCR Product 1 Lane 2: 1kb Ladder and Marker Lane 3: <I>SBP</I> PCR Product 2</B> Both bands observable on the gel correspond to the expected size of <I>SBP</I> which is ~600 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: SDS Gel of Uninduced and 1 mM IPTG Induced Cultures of PotD and SBP.</B> For PotD, it is clear from the gel that it is being successfully overexpressed since there is an intense band present at the 37 kDa marker which corresponds to the expected size of PotD in the induced culture sample. For SBP, there is no band present which suggests SBP is being overexpressed. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Characterisation of PotD.</B> Single colonies of MC1061 <I>E.coli</I> containing pQE80-L encoding <I>PotD</I> were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin. Once cultures reached an OD<SUB>600</SUB> of 0.6, one was induced with 1 mM IPTG and one was left uninduced as a control. They were left to grow for a further three hours, after which a 1ml sample was taken and the cells pelleted. The cells were resuspended in 100µl of Laemmli buffer, 20µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: SDS Gel of Uninduced and 0.5mM, 1mM and 2mM IPTG Induced Cultures of SBP.</B> It appears that upon induction of <I>SBP</I> with any concentration of IPTG the cells are dying given that there is a significant drop in protein levels observable on the gel.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, the induced cultures stop growing when SBP expression is induced. After ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: Growth Curve of Uninduced and Induced Cultures (0.5 mM, 1 mM and 2 mM IPTG)</B> The graph shows that on comparison with the uninduced control, after reaching an OD<SUB>600</SUB> of 0.6,  the induced cultures stop growing when <I>SBP</I> starts to be expressed. Similar to the previous growth curve assay, after ~7 hours, it is possible that the emergence of mutants causes the the OD to rise again or the cells have adapted to the production of the foreign protein.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of PotD</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100 V. The two bands observable on the gel correspond to the expected size of PotD of 38 kDa. Western Blotting will confirm if these proteins are PotD. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: Characterisation of PotD following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure9-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 9: Characterisation of SBP following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sbp-figure10-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 10: Characterisation of PotD/Spermidine Interaction.</B> The graph shows that the change in the tryptophan spectra increases as concentration of spermidine increases. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 1: pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> (respectively) Pre-Sequencing Digest</B> pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> minipreps were digested with EcoRI and PstI to check for presence of each insert. The bands observable on the gel correspond to the expected sizes for the pSB1C3 vector and both inserts.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: Amplification of <I>hHBA</I> and <I>hHBB</I></B><I>hHBA</I> and <I>hHBB</I> were amplified from the pSB1c3-<I>hHBA</I> and pSB1c3-<I>hHBB</I> minipreps obtained yesterday using primers to allow cloning of the genes into pT25 and pUT18 respectively. Although the band is fainter in the case of <I>hHBA</I>, the gel indicates the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Amplification of <I>hHBA</I></B><I>hHBA</I> was amplified from the pSB1C3-<I>hHBA</I> using primers to allow cloning of the genes into pT25. The gel indicates that the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: Amplification of <I>hHBA</I> and <I>hHBB</I></B><I>hHBA</I> and <I>hHBB</I> were amplified from the pSB1c3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> minipreps using primers to allow cloning of the genes into pQE80-L. The gel indicates that the PCR has been successful. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Pre-Sequence Digest of pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I></B> Both pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I> minipreps were digested with BamHI and KpnI to check for the presence of the inserts. The gel indicates that both inserts have been successfully cloned into pQE80-L. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Overexpression of <I>hHBB</I> and <I>hHBA</I>.</B> Single colonies of M15[pREP4] <I>E.coli</I> containing pQE80-L encoding <I>hHBB</I> and pQE80-L encoding <I>hHBA</I> were used to inoculate 5ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD<SUB>600</SUB> of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of 2x Laemmli buffer, 20 µl of which was separated by SDS-PAGE. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 7: Characterisation of hHBB.</B> Single colonies of M15[pREP4] <I>E.coli</I> containing pQE80-L encoding <I>hHBB</I> were used to inoculate 5 ml of fresh LB growth medium supplemented with ampicillin and kanamycin. Once cultures reached an OD<SUB>600</SUB> of 0.6, they were induced with a range of IPTG concentrations. They were left to grow for a further three hours, after which a 1 ml sample was taken and the cells pelleted. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE. The cells were resuspended in 100 µl of Laemmli buffer, 20 µl of which was separated by SDS-PAGE and transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 8: Amplification of <I>hHBN</I> for Cloning into pQE80-L </B> PCR product from amplification of <I>hHBN</I> from IDT plasmid using primers for cloninig into pQE80-L. The gel indicates <I>hHBN</I> has been amplified successfully since the observed band corresponds to the expected size of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure9-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 9: SDS-PAGE of Fractions Obtained After Size Exclusion Chromatography of hHBB</B> 10 µl of each fraction was mixed with 10 µl of laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. The four bands observable on the gel correspond to the expected size of hHBB of 16 kDa. Western Blotting will confirm if these proteins are hHBB. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure10-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 10: Characterisation of hHBB following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure11-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 11: Amplification of <I>hHBN</I> for Cloning into pT25 </B> PCR product from amplification of <I>hHBN</I> from IDT plasmid using primers for cloninig into pT25. The gel indicates <I>hHBN</I> has been amplified successfully since the observed band corresponds to the expected size of 1200 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure14-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 14: Characterisation of hHBN following Purification.</B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer and loaded onto an SDS gel which was ran for 1 hour at 100V. It was then transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure15-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 15: Amplification of <I>hHBN</I></B><I>hHBN</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of <I>hHBN</I> of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure16-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 15: Amplification of <I>hHBA</I></B><I>hHBA</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pT25. The band observable on the gel corresponds to the expected size of <I>hHBA</I> of </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="bl-figure17-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 16: Amplification of <I>hHBN</I></B><I>hHBN</I> was amplified using primers to allow for subsequent cloning into the two hybrid vector pUT18. The band observable on the gel corresponds to the expected size of <I>hHBN</I> of 1200 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 1: Amplification of <I>OBP2A</I> for insertion into pSB1C3 vector </B>. This is the PCR product from the amplification of <I>OBP2A</I> from the IDT plasmid using primers for cloning into pSB1C3. The gel indicates <I>OBP2A</I> has been amplified successfully since the observed band corresponds to the expected size of 500 bp. The concentration of the gel extracted <I>OBP2A</I> is 335.53 ng/ul. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 2: A subsequent gel was run of the amplified <I>OBP2A</I> after it was first excised from the gel in <I>Figure 1 </I></B> This was to determine if the gel extraction was successful. The gel indicates that <I>OBP2A</I> has been extracted from the gel successfully, since the observed band corresponds to the expected size of 500 bp. This can now be restricted in preparation for ligation into pSB1C3. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 3: A Gel Analysis of Restriction Digest from the 24/6.</B> This gel analysis was done to determine if the reason for the failed transformation was an improperly digested <I>OBP2A</I>. This gel indicates that the restricted product hasn't been lost as the <I>OBP2A</I> gene fragment can be seen as a distinct band at 500 bp. A re-ligation of <I>OBP2A</I> into the pSB1C3 backbone will now occur. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 4: Presequence Digest of pSB1C3-<I>OBP2A</I></B>. All seven plasmid purifications of the overnight cultures done on the 26/6 were digested to see if they contained the <I>OBP2A</I> insert. It is clear from the gel that all but sample 3 contain the insert for <I>OBP2A</I> as they all have faint bands at 500 bp's which corresponds to the size of <I>OBP2A</I>. It also appears that sample 5 contains the highest concentration, so that will be the sample that is sent for sequencing.  </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 5: Presequence digest of pUT18-<I>OBP2A</I></B> To determine if any samples contain the <I>OBP2A</I> insert. It can be seen from this gel that of the  5 samples that were digested only two are show to have the <I>OBP2A</I> insert, therefore sample 2 will be sent for sequencing.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 6: Amplification of <I>OBP2A</I> for insertion into pT25 vector </B>. This is the PCR product from the amplification of <I>OBP2A</I> for cloning into pT25. The gel indicates <I>OBP2A</I> has been amplified successfully since the observed band corresponds to the expected size of 100 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure7-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 7: Presequence Digest of pT25-<I>OBP2A</I></B> In order to determine if any of the samples contain the <I>OBP2A</I> insert. It can be seen from this gel that samples 5, 6 and 7 may have inserts in them as they each have two distinct bands, one of which corresponds to the 100 bp size for the <I>OBP2A</I> for pT25. As a result all three samples will be sent for sequencing.    </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure8-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 8: β-galactosidase Assay</B>  A graph of the results from the first attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>. As it can be seen that the positive result has failed(far left) not much can be inferred from the interaction of the two parts of <I>OBP2A</I>. However it looks like the two subunits of <I>OBP2A</I> may not be interacting. As this was a first attempt, subsequent assays will be performed in order to determine if in fact the two separated parts of OBP2A aren't interacting in vivo - like these results suggests.      </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure11-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 9: β-galactosidase Assay</B> A graph of the results from the second attempt at a β-galactosidase assay that aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>.  It can also be inferred from this graph that the two parts of <I>OBP2A</I> don't seem to be interacting as it has a low millers activity when compared to the controls. Further assays will be performed in order to determine if the two separated parts of <I>OBP2A</I> aren't interacting in vivo like these results suggests.     </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="nm-figure12-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B> Figure 11: β-galactosidase Assay Under Anaerobic Conditions </B> A graph of the results from the anaerobic β-galactosidase assay aimed to characterize the interaction between the two separated parts of <I>OBP2A</I>. The graph seems to suggest that the two subunits of <I>OBP2A</I> may not be interacting in vivo. This can be inferred from the fact that the millers activity of the <I>OBP2A</I> sample lies lower than the negative controls from within the experiment. As a result of this information, one more assay will be prepared in which the two subunits will be switched with regards to the bacterial two hybrid vectors to eliminate any possibility of this being a factor effecting the interaction between the two subunits.     </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: <B>Amplification of <I>LS</I></B><I>LS</I> was amplified using primers which would allow for subsequent cloning into the high expression vector pQE80-L. Lane 1 shows that the PCR has been successful since the band corresponds to the expected size of 2100 bp.</DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure2-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 2: Amplification of <I>LS</I></B><I>LS</I>  was amplified from the IDT plasmid using primers to allow cloning of the gene into pSB1C3. The observed band corresponds to the expected size of LS, 2100 bp. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure3-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 3: Overexpression Assays of <I>LS</I></B> 5 ml cultures of M15pREP4 <I>E.coli</I> was grown to OD<SUB>600</SUB>=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37<SUP>o</SUP>C, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 4: Characterisation of <I>Lanosterol Synthase</I></B> 5 ml cultures of M15pREP4 <I>E.coli</I> was grown to OD<SUB>600</SUB>=0.6 then induced with a range of IPTG concentrations. Samples were left to grow for a further 6 hours at 37<SUP>o</SUP>C, at which point they were spun down and the pellet resuspended in 2x Laemmli buffer. The samples were boiled for 10 minutes and then loaded on to an SDS gel. The proteins were transferred to a nitrocellulose membrane and probed with an anti-His antibody. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure5-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 5: Chromatogram following Affinity Chromatography of LS</B> Using FPLC, the protein sample was loaded onto a nickel column and eluted with an increasing concentration of imidazole (green line). The fractions corresponding to the peak were ran on an SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="fp-figure6-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><B>Figure 6: SDS Gel of Peak Fractions </B> 10 µl of each fraction was mixed with 10 µl of 2x Laemmli buffer an loaded onto a SDS gel. </DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="BBa_K1058008-figure1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 1: Alignment of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>. Sequences from top to bottom: sequencing result from forward primer; expected sequence; reverse complement of sequencing result from reverse primer; sequencing result from intermediate primer. Colour code: Green = Prefix/Suffix; Pink = Annealing site of intermediate primer; Yellow = Mismatch in one sequence; Red = In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem022-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 2: 1% agarose gel after PCR of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with primers specific to the promoter region, P<I>chr</I>, and the repressor region, <I>chrB</I>. Left: P<I>chr</I> (expected size 169bp); Middle: 1kb+ ladder, Right: <I>chrB</I> (expected size 939bp). Amplification of <I>chrB</I> was not successful. P<I>chr</I> was further processed by restriction digest with EcoRI and PstI.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem023-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 3: 1% agarose gel after PCR of <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with primers specific to the repressor region, <I>chrB</I>. Left: 1.5 µl DMSO; Middle: 1kb+ ladder, Right: 2.5 µl DMSO. Expected size for each: 939bp. The annealing temperature for the PCR amplification was lowered to 50°C in order to increase the chances of accomodating a primer that is not 100% complementary.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem024-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 4: 1% agarose gel after PCR of <I>gfp</I> with corresponding primers. Left: <I>GFP</I> (expected size 717bp); Middle: 1kb+ ladder, Right: PCR product of <I>chrB</I>, using primers to add a BamHI restriction site in front of it and a 6-His-Tag and a HindIII restriction site at the end. (expected size of <I>chrB</I>: 939bp). Since amplification of <I>chrB</I> was unsuccessful, it will be repeated.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem026-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 5: 1% agarose gel after restriction digest of pSB1C3-P<I>chr</I>, pSB1C3-<I>chrB</I> (s), and pSB1C3-<I>chrB</I> (w) with EcoRI and PstI for confirmation of size. Ladder: 1kb+. Top: 1-4: P<I>chr</I> 1-4; <I>chrB</I> (s) 1-2. Bottom: <I>chrB</I> (s) 3-4; <I>chrB</I> (w) 1-4. Expected sizes: P<I>chr</I>: 169bp; <I>chrB</I>: 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-pChr1-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 6: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green = Prefix/Suffix
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-ChrBw4-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 7: Alignment of <I>chrB</I> (opt) sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green: Prefix and Suffix; Red: In frame stop codons; Blue: First 15 codons optimised for expression in E. coli.
        </DIV></DIV><DIV class="modal-body">' 

 </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem034-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 8: 1% agarose gel after restriction digest of pUniprom (Expected size 2454bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem035-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 11: 1% agarose gel after restriction digest of pSB1C3 with EcoRI and SpeI (Expected size 2070bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem032-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 9: 1% agarose gel after PCR of <I>chrB</I> (opt). (Expected size 939bp).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-first-transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 10: Plates after transformation of Top: pUniprom<I>chrB</I>, Middle: pUniprom<I>chrB</I> (opt) and Bottom: pSB1C3-P<I>chr-gfp</I> into JM110.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem036-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 12: 1% agarose gel after restriction digest of pUniprom<I>chrB</I>, pUniprom<I>chrB</I> (opt) and pSB1C3-P<I>chr-gfp</I> for confirmation of size. Ladder: 1kb+. Top: 1-4: pSB1C3-P<I>chr-gfp</I> 1-4; pUniprom<I>chrB</I> (opt) 1-2. Bottom: pUniprom<I>chrB</I> (opt) 3-4; pUniprom<I>chrB</I> 1-4. Expected sizes: P<I>chr-gfp</I>: 909bp; <I>chrB</I> and <I>chrB</I> (opt): 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15-chr-igem037-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 13: 1% agarose gel after colony PCR of pUniprom<I>chrB</I> (opt). Top: colonies 1-11; Bottom: colonies 12-22. Colonies 2, 4, 9, and 12 were selected for sequencing. Ladder: 1kb+ Expected size of <I>chrB</I> (opt): 939bp.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pSB1C3-pChr-GFP_reverse" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 14: Sequencing result of pSB1C3-P<I>chr-gfp</I> after double ligation. Even though the sequence is as expected, it is ligated in reverse. It was hence called pSB1C3-P<I>chr-gfp</I> (A) The Result for pSB1C3-P<I>chr-gfp</I> (colony 2) was not as expected, albeit in the correct orientation. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pChrGFPplates_revisited" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 15: Plates with pSB1C3-P<I>chr-gfp</I> in JM110 after double ligation under UV light. Numbered colonies were selected for initial size confirmation and sequencing. However, other colonies seemed to express GFP as well, hence a second attempt was made with different colonies from the same plate.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_pChrGFP_revisited_seq" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 16: Alignment of sequencing results of pSB1C3-P<I>chr-gfp</I> - colonies 1, 5, 6, and 7 with sequences of <I>gfp</I> (mut2) and P<I>chr</I> sequence. Colony 1 remained consistent, the other tested colonies were without success. Colours: Green: Prefix and partial Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem039" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 17: 1% Agarose gel after restriction digest of pUniprom with BamHI/PstI (expected size: 2454bp). Right: PCR product of Haptoglobin.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150723_pSB1C3-pChr" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 18: Plates after transformation of P<I>chr</I> into pSB1C3. Top: ratio insert:vector 2:1; Bottom: ratio insert:vector 3:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150727_pSB1C3-pChr_alignment" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 19: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> with the sequencing result of this region. Colour code: Green = Prefix/Suffix
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150728_pUniprom-ChrB" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 20: Plates after transformation of, Top: <I>chrB</I> into pUniprom; Bottom: <I>chrB</I> (opt) into pUniprom. Ratio insert:vector for ligations: left: 3:1; Right: 4:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem040" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 21: Gel after restriction digest of Left: pUniprom-<I>chrB</I>, and Right: pUniprom-<I>chrB</I> (opt) with BamHI/PstI for confirmation of insert size. Expected size 939bp. <I>chrB</I> colony 2 and <I>chrB</I> (opt) colony 3 were selected for sequencing.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150729_chrb-combined" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 22: Alignments of: Left: <I>chrB</I> and Right: <I>chrB</I> (opt). Colours: Pink = Restriction sites (BamHI/PstI); Blue = 6-His Tag; Red = In-frame stop codons; Cyan = optimised codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150731_pChrGFP-ligations" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 23: Plates after ligation of <I>gfp</I> into pSB1C3-P<I>chr</I>. Left: Ratio Insert:Vector 2:1; Right: Ratio Insert:Vector 3:1.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_igem043" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 24: Gel for size confirmation of pSB1C3-P<I>chr-gfp</I> (Top). Bottom: Left - Haptoglobin, Right - Lanosterol Synthase. pSB1C3-P<I>chr-gfp</I> (3) and pSB1C3-P<I>chr-gfp</I> (4) were submitted for sequencing.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150801_pchrgfp" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 25: Alignment of P<I>chr</I> sequence retrieved from <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A> and <I>gfp</I> (mut2) with the forward and reverse sequencing results of P<I>chr-gfp</I> this region. The sequence was as expected, it was hence called pSB1C3-P<I>chr-gfp</I> (B). Colour code: Green = Prefix/Suffix; Blue: SpeI/XbaI-scar; Cyan: Ribosomal binding site, Red: In frame stop codons.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150809_blot" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 26: Top: anti-GFP: Single colonies of JM110 + pSB1C3-P<I>chr-gfp</I> (A), MC1061 + pSB1C3-P<I>chr-gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> chloramphenicol. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of each sample was mixed with 100 µl laemmli buffer, and boiled for 10min. 3 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and P<I>manA-gfp</I> as a positive control. 
        Bottom: anti-6-His - ChrB: Single colonies of JM110 pUniprom-<I>chrB</I> and JM110 pUniprom-<I>chrB</I> (opt) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 200 µl laemmli buffer, and the sample boiled for 10min. 10 µl of each sample was loaded on a  SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and <I>hydA - 6-His</I> as a positive control.
        Expected sizes: GFP ~ 27kDa, ChrB ~35kDa.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150811_transformations" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 27: Plates after transformation of single constituents of chromate sensor into <I>E. coli</I> MG1655. From top to bottom: pUniprom-<I>chrB</I> (from colony 2 - 29/07), pUniprom-<I>chrB</I> (opt) (from colony 3 - 29/07), pSB1C3-P<I>chr-gfp</I> (B), pSB1C3-P<I>chr-gfp</I> (A), <A href="http://parts.igem.org/Part:BBa_K1058008" target="_blank">BBa_K1058008</A>.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150822_blot" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 28: Western blot against GFP: Single colonies of MG1655, containing all 4 combinations, pUniprom-<I>chrB</I> - pSB1C3-P<I>chr-gfp</I> (A), pUniprom-<I>chrB</I> - pSB1C3-P<I>chr-gfp</I> (B), pUniprom-<I>chrB</I> (opt) - pSB1C3-P<I>chr-gfp</I> (A), and pUniprom-<I>chrB</I> (opt) - pSB1C3-P<I>chr-gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 <SUP>µg</SUP>/<SUB>ml</SUB> chloramphenicol and 100 <SUP>µg</SUP>/<SUB>ml</SUB> ampicillin. After 16h of incubation at 37°C with agitation at 200rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml PBS. 100 µl of these samples was mixed with 100 µl laemmli buffer, and the sample boiled for 10 min. The indicated volume of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3, and pUniprom were included as a negative control, and P<I>manA-gfp</I> as a positive control. The blot against 6-His – ChrB was without success and is not shown. No chromate of any kind was added to the medium previous to this blot, hence no expression of GFP was expected. 
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV>


K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB><DIV class="modal fade" id="dundee15_chr_150824_platereader_BBa_K1058008" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 29: F,OD-plot of results of plate reader experiment after exposing MG1655 + BBa_K1058008 to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="icecream" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 30: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="potato" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 31: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt) to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150824_platereader_pchrbgfp" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 32: F,OD-plot of results of plate reader experiment after exposing MG1655 + pSB1C3-P<I>chr-gfp</I> (B) to a range of concentrations between 10 nM and 100 µM of K<SUB>2</SUB>CrO<SUB>4</SUB>, K<SUB>2</SUB>Cr<SUB>2</SUB>O<SUB>7</SUB>, CrCl<SUB>3</SUB>, and K<SUB>2</SUB>SO<SUB>4</SUB>. One measurement was made every 10min across 16h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_chrbbielefeld" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 33: Plates after ligation of <I>chrB</I> (Bielefeld) into pT7KS. From top to bottom: Ratio insert:vector 2:1; Ratio Insert:Vector 3:1; pT7KS religation control.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_bl21" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 34: Plates after transformation of single constituents of chromate sensor into <I>BL21 (DE3)</I>. From top to bottom: pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt), pUniprom-<I>chrB</I>, and pUniprom-<I>chrB</I> (opt).
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_150911_igem047" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 35: 1% agarose gel after colony PCR of pT7KS-<I>chrB</I> (Bielefeld). Top: colonies 1-6; Bottom: colonies 7-12. Ladder: 1kb+ Expected size of <I>chrB</I> (Bielefeld): 939bp
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="dundee15_chr_platereaderinduced" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Figure 36: F,OD-plot of results of plate reader experiment after inducing expression of ChrB from the T7 promoter of pUniprom <I>BL21 (DE3)</I> pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I>, and pSB1C3-P<I>chr-gfp</I> (B) + pUniprom-<I>chrB</I> (opt). IPTG was added after 4 hours. One measurement was made every 10min across 22h.
        </DIV></DIV><DIV class="modal-body">' 

       </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV><H1><HIGHLIGHT class="highlight">FluID Detector</HIGHLIGHT></H1><H3><HIGHLIGHT class="highlight">Lab Book</HIGHLIGHT></H3><SECTION id="about" class="row1"><DIV class="row"><DIV class="col-lg-4"><H3>FluID</H3><P class="about-content">FluID is a cell free, all in one spray which will be used to detect and dinstiguish between body fluids found at crime scenes.</P></DIV><DIV class="col-lg-4"><H3>Chromate Biosensor</H3><P class="about-content">Our chromate biosensor is a device that will be used to detect stainless steel residues left behind on bones in blunt or sharp force trauma cases.</P></DIV><DIV class="col-lg-4"><H3>Fingerprint Ageing</H3><P class="about-content">This device will search for the presence of a specifc component found in fingerprint deposits whose presence/absence will indicate approximate fingerprint age.</P></DIV></DIV></SECTION><DIV class="ribbon"><DIV class="rows"><DIV class="row"><DIV class="border-week"><DIV class="box"><SPAN id="biospray-title">Saliva</SPAN><DIV id="sacollapsible" class="collapse biospray-content"><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 1/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"><I>Lactoferrin binding protein A (LbpA)</I> was successfully cloned into the vector pSB1C3.</P></DIV><DIV id="sacollapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/6: Amplification of <I>LbpA</I> for Cloning into pSB1C3</SPAN><DIV id="sacollapsiblew1-1" class="collapse box-content"><P><B>Aim of Experiment:</B> Amplify <I>LbpA</I> from MC58 <I>N.meningitidis</I> template chromosome.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> Figure 1 </A></P><P><B>Next Steps:</B> Since the <I>LbpA</I> gene was amplified successfully, it was ready to be digested for subsequent cloning into pSB1C3.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">2/6: Restriction Digests and Ligations with pSB1C3</SPAN><DIV id="sacollapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To digest <I>LbpA</I> with BamHI and EcoRI and then ligate it into pSB1C3.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results:</B> N/A


                  </P><P><B>Next Steps:</B> Ligations of pSB1C3-<I>LbpA</I> were left overnight until the next day when they would be transformed into the JM110 <I>E.coli</I>.
                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/6: Transformations of pSB1C3-<I>LpbA</I> into JM110 <I>E.coli</I></SPAN><DIV id="sacollapsiblew1-3" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the pSB1C3-<I>LbpA</I> ligations into JM110 <I>E.coli</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A></P><P><B>Results:</B> N/A
                  </P><P><B>Next Steps:</B> Transformations will be checked tomorrow and if successful, overnight cultures will be set up.
                  
                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">4/6: Overnight Cultures of JM110 <I>E.coli</I> containing pSB1C3-<I>LbpA</I></SPAN><DIV id="sacollapsiblew1-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of JM110 <I>E.coli</I> containing pSB1C3-<I>LbpA</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures
          </A></P><P><B>Results:</B> N/A
                  </P><P><B>Next Steps:</B> A miniprep will be performed tomorrow on the overnight cultures. 
                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">5/6: Plasmid Purification of JM110 <I>E.coli</I> containing pSB1C3-<I>LbpA</I></SPAN><DIV id="sacollapsiblew1-5" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the JM110 <I>E.coli</I> overnight cultures containing pSB1C3-<I>LbpA</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit) </A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The miniprep with the highest concentration will be sent for sequencing.                </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 8/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Sequencing confirmed that <I>LbpA</I> had been successfully cloned into pSB1C3 therefore this week cloning was attempted to put <I>LbpA</I> into the high expression vector pQE80-L.</P></DIV><DIV id="sacollapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">8/6: Amplification of <I>LbpA</I> for Cloning into pQE80-L</SPAN><DIV id="sacollapsiblew2-1" class="collapse box-content"><P><B>Aim of experiment:</B> To amplify <I>LbpA</I> for subsequent cloninng into the pQE80-L vector.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR </A></P><P><B>Results: </B>N/A</P><P><B> Next Steps:</B> The PCR product will be digested and ligated into pQE80-L tomorrrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">9/6: Restriction Digests and Ligations of <I>LbpA</I>into pQE80-L</SPAN><DIV id="sacollapsiblew2-2" class="collapse box-content"><P><B>Aim of experiment:</B> To restrict and ligate the <I>LbpA</I> gene into the pQE80-L plasmid.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations </A></P><P><B> Results:</B>: N/A </P><P><B> Next Steps:</B> Transformations of pQE80-L-<I>LbpA</I> into JM110 <I>E.coli</I> will be carried out tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/6: Transformations of pQE80-L-<I>LpbA</I> into JM110 <I>E.coli</I></SPAN><DIV id="sacollapsiblew2-3" class="collapse box-content"><P><B>Aim of experiment:</B> To transform  pQE80-L-<I>LpbA</I> into JM110 <I>E.coli</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A></P><P><B>Results:</B> N/A </P><P><B> Next Steps:</B> If the transformations are successful, colonies will be picked to set up overnight cultures. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 15/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">This week, cloning of <I>LbpA</I> into pQE80-L was continued due to last week's failed attempts.</P></DIV><DIV id="sacollapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">15/6: PCR of <I>LbpA</I> for Cloning into pQE80-L</SPAN><DIV id="sacollapsiblew3-1" class="collapse box-content"><P><B>Aim of experiment:</B> To re-amplify the <I>LbpA</I> gene for cloning into the pQE80-L vector.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure3-modal" class="journal-protocol">Figure 3</A></P><P><B>Next Steps:</B> The gel shows that <I>LbpA</I> has been successfully amplified so it will be digested and ligated into the pQE80-L vector tomorrow.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 22/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"><I>LbpA</I> was successfully cloned into the high expression vector pQE80-L.</P></DIV><DIV id="sacollapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/6: Restriction Digest and Ligation of <I>LbpA</I> into pQE80-L</SPAN><DIV id="sacollapsiblew4-1" class="collapse box-content"><P><B>Aim of experiment:</B> To digest the amplified <I>LbpA</I> gene with BamHI and KpnI and subsequently ligate it into the pQE80-L vector.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The ligations will be transformed into M15[<I>pREP4</I>] <I>E.coli</I>. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/6: Transformations of pQE80-L-<I>LbpA</I> into M15[<I>pREP4</I>] <I>E.coli</I></SPAN><DIV id="sacollapsiblew4-2" class="collapse box-content"><P><B>Aim of experiment:</B> To transform pQE80-L-<I>LbpA</I> into M15[<I>pREP4</I>] <I>E.coli</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Overnight cultures will be set up tomorrow if the transformations are successful. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">25/6: Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>LbpA</I></SPAN><DIV id="sacollapsiblew4-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>LbpA</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Overnight cultures will be miniprepped and a pre-sequence restriction digest will be performed to check for the presence of the <I>LbpA</I> gene. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">26/6: Plasmid Purification of the M15[<I>pREP4</I>] <I>E.coli</I> Overnight Cultures containing pQE80-L-<I>LbpA</I></SPAN><DIV id="sacollapsiblew4-4" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures from yesterday and perform a pre-sequencing restriction digest using BamHI and KpnI to check for presence of <I>LbpA</I> gene.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification (QIAprep® Spin Miniprep Kit)</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> A sample of the miniprep was sent for sequencing to confirm the presence of <I>LbpA</I> in the pQE80-L vector. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 29/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Protein expression optimization experiments were performed on LbpA to assay protein production levels. However, cell growth seems to cease when <I>LbpA</I> expression is induced.</P></DIV><DIV id="sacollapsiblew5" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/7: M15[<I>pREP4</I>] <I>E.coli</I> Overnight Cultures containing pQE80-L-<I>LbpA</I></SPAN><DIV id="sacollapsiblew5-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures in preparation for protein expression experiments tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The overnight cultures will be used to set up day cultures tomorrow which will be used to test expression of <I>LbpA</I>. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">2/7: Optimization of <I>LbpA</I> Expression</SPAN><DIV id="sacollapsiblew5-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up day cultures and induce expression of <I>LbpA</I> using different concentrations of IPTG and check levels of protein production via SDS-PAGE.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> The samples were stored in the -20 <SUP>o</SUP>C freezer overnight and will be ran on a gel tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/7: Continuation of Optimization of <I>LbpA</I> Expression</SPAN><DIV id="sacollapsiblew5-3" class="collapse box-content"><P><B>Aim of experiment:</B> To test the samples obtained yesterday on an SDS gel.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol">Protein Expression Optimization </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure4-modal" class="journal-protocol"> Figure 4 </A></P><P><B>Next Steps:</B> Further optimization experiments will be set up next week using different concentrations of IPTG. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 6/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Further protein expression optimization experiments were set up, this time using a range of IPTG concentrations to induce expression of <I>LbpA</I>.</P></DIV><DIV id="sacollapsiblew6" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">7/7: Further Optimization of <I>LbpA</I> Expression</SPAN><DIV id="sacollapsiblew6-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up further tests to optimize expression of <I>LbpA</I> using different concentrations of IPTG.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization </A> Note: An SDS gel was not ran, instead the OD<SUB>600</SUB> readings were recorded for the uninduced and induced cultures at 20 minute intervals for 2 hours, 1 and half hours after the cultures were induced. </P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure5-modal" class="journal-protocol"> Figure 5 </A></P><P><B>Next Steps:</B> A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after <I>LbpA</I> expression has been induced. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 13/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Protein expression optimization experiments were continued this week. A growth curve assay was carried out to assess the effect of <I>LbpA</I> expression over a longer period of time.  </P></DIV><DIV id="sacollapsiblew7" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/7: Plate Reader Growth Curve Assay</SPAN><DIV id="sacollapsiblew7-1" class="collapse box-content"><P><B>Aim of experiment:</B> To induce expression of <I>LbpA</I> using different concentration of IPTG and allow cells to grow for 16 hours in order to monitor cell growth.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#platereader-modal" class="journal-protocol"> Growth Curve Assay </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure6-modal" class="journal-protocol"> Figure 6 </A></P><P><B>Next Steps:</B> A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after <I>LbpA</I> expression has been induced. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/7: Sample Preparation for Western Blot</SPAN><DIV id="sacollapsiblew7-2" class="collapse box-content"><P><B>Aim of experiment:</B> To check if the cells in the induced cultures ~7 hours after being induced are producing LbpA. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot Sample Preparation </A> Note: The day cultures were grown for 7 hours after they were induced and then the samples were frozen, to be blotted on Monday. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The samples will be ran on an SDS gel on Monday and blotted. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 20/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Samples from the growth curve assay were blotted to see if LbpA is being produced at the later stages of the growth curve. A further growth curve assay was also performed which monitored cell growth until a certain OD<SUB>600</SUB> was reached, at which point the cultures were induced.</P></DIV><DIV id="sacollapsiblew8" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">20/7: Western Blot LpbA Culture Samples</SPAN><DIV id="sacollapsiblew8-1" class="collapse box-content"><P><B>Aim of experiment:</B> To western blot the LbpA culture samples from Friday.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot </A></P><P><B>Results:</B> Nothing was visible on the blot. </P><P><B>Next Steps:</B> Another plate reader experiment will be set up on Thursday where the cultures will be monitored for three hours before induction with IPTG and then for a further 20 hours afterwards. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/7: Growth Curve Assay</SPAN><DIV id="sacollapsiblew8-2" class="collapse box-content"><P><B>Aim of experiment:</B> To check if the cells begin to die after induction with IPTG after having already grown to an OD<SUB>600</SUB> of 0.5. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#platereader2-modal" class="journal-protocol"> Growth Curve Assay </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure7-modal" class="journal-protocol"> Figure 7 </A></P><P><B>Next Steps:</B> Samples of these cultures will be taken 6 hours after induction to check for the presence of LbpA. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 27/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">The presence of LbpA was confirmed in the culture 6 hours after induction with IPTG so protein purification was started this week.</P></DIV><DIV id="sacollapsiblew9" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/7: Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>LbpA</I></SPAN><DIV id="sacollapsiblew9-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up a 150 ml overnight culture for sub-culturing tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A> Note: 150 ml of LB was used and 150 µl of each antibiotic. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> 150 ml of fresh LB will be inoculated tomorrow with 7.5 ml of the overnight cultures. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/7: <I>LbpA</I> Expression Assay</SPAN><DIV id="sacollapsiblew9-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up day cultures using the overnight cultures from yesterday and test for expression of LbpA 6 hours after induction.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Test </A> Note: 150 ml of fresh LB was inoculated with 7.5 ml of the overnight cultures. Only an uninduced control and an 1 mM IPTG induced culture was set up. The samples were not blotted, they were only ran on an SDS gel. </P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure8-modal" class="journal-protocol"> Figure 8 </A></P><P><B>Next Steps:</B> A 3 L culture will be set up on Thursday under the same conditions as the overnight cultures in order to try and purify LbpA.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/7: Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>LbpA</I> for 3L Culture</SPAN><DIV id="sacollapsiblew9-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures for the 3L day cultures that will be grown tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A> Note: One 150 ml overnight culture was set up using 150 µl of each antibiotic. </P><P><B>Results:</B> N/A   </P><P><B>Next Steps:</B> The 3 L culture will be set up tomorrow allowed to grow to OD<SUB>600</SUB> of 0.6-1 and then induced with 1 mM IPTG and grown for 6 hours.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/7: 3 L Culture of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>LbpA</I> for Purification of LbpA</SPAN><DIV id="sacollapsiblew9-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 3 L day cultures for purifying LbpA.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The procedure was stopped at Step 6 and the pellets were frozen at -20 <SUP>o</SUP>C. </P><P><B>Results:</B> N/A   </P><P><B>Next Steps:</B> Protein purification will be continued on Monday.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 3/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">The first round of LbpA purification failed at the affinity chromatography stage so another 6 L culture was grown this week to attempt purification again.</P></DIV><DIV id="sacollapsiblew10" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/8: Continuation of LbpA Purification</SPAN><DIV id="sacollapsiblew10-1" class="collapse box-content"><P><B>Aim of experiment:</B> To continue with purification of LbpA.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The protocol was continued from Step 7</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>Size exclusion chromatography was not carried out since an SDS gel showed LbpA was not present after performing affinity chromatography. Another 6L culture will be set up this week.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">5/8: Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>LbpA</I></SPAN><DIV id="sacollapsiblew10-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures for the 6 L culture that will be set up tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A>Note: One 150 ml overnight culture was set up using 150 µl of each antibiotic.</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>The overnight cultures will be used to set up the 6 L day culture tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">6/8: 6 L Day Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>LbpA</I></SPAN><DIV id="sacollapsiblew10-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 6 L day cultures for purifying LbpA and begin the purification process.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: The protocol was stopped at Step 6.</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Purification of LbpA will be continued on Monday. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Purification of LbpA was continued but unfortunately was unsuccessful. If time permits, another round of purification may be reattempted. </P></DIV><DIV id="sacollapsiblew11" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/8: Continuation of LbpA Purification</SPAN><DIV id="sacollapsiblew11-1" class="collapse box-content"><P><B>Aim of experiment:</B> To continue with purification of LbpA.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The protocol was continued from Step 7.</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Purification of LbpA will be continued tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/8: Continuation of LbpA Purification</SPAN><DIV id="sacollapsiblew11-2" class="collapse box-content"><P><B>Aim of experiment:</B> To purify LbpA using affinity chromatography.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The protocol was stopped at Step 13. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>Since no peaks were observed, purification of LbpA was abadoned at this stage. Purification may be retried. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><SPAN id="biospray-title">Semen</SPAN><DIV id="secollapsible" class="collapse biospray-content"><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 25/5/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I> PotD</I> into the biobrick vector, pSB1C3, was started this week.</P></DIV><DIV id="secollapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/5: PCR of <I>PotD</I> from MG1655 <I>E.coli</I> gDNA</SPAN><DIV id="secollapsiblew1-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To amplify <I>PotD</I> from MG1655 <I>E.coli</I> gDNA.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results: </B>N/A</P><P><B>Next Steps:</B> The PCR product will be gel extracted, digested and ligated into the biobrick vector pSB1C3 tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/5: Gel Extraction, Restriction Digest and Ligation of <I>SBP</I> into pSB1C3</SPAN><DIV id="secollapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To gel extract the <I>PotD</I> PCR product and digest it with PstI and EcoRI. A ligation will then be set up with pSB1C3.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digest</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> Ligations will be transformed into JM110 <I>E.coli</I> on Monday.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 1/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I>PotD</I> into the biobrick vector was completed this week. </P></DIV><DIV id="secollapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/6: Transformation of pSB1C3-<I>PotD</I> into JM110 <I>E.coli</I></SPAN><DIV id="secollapsiblew2-1" class="collapse box-content"><P><B>Aim of experiment:</B> To transform pSB1C3-<I>PotD</I> into JM110 <I>E.coli</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A></P><P><B>Results:</B>N/A</P><P><B> Next Steps:</B> If the transformations are successful, overnight cultures will be set up tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">2/6: Overnight Cultures of pSB1C3-<I>PotD</I> in JM110 <I>E.coli</I></SPAN><DIV id="secollapsiblew2-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of pSB1C3-<I>PotD</I> in JM110 <I>E.coli</I></P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B> Results:</B>: N/A </P><P><B> Next Steps:</B> The overnight cultures will be miniprepped tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/6: Plasmid Purification of pSB1C3-<I>PotD</I> from Overnight Cultures</SPAN><DIV id="secollapsiblew2-3" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures from yesterday.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit) </A></P><P><B>Results:</B> N/A </P><P><B> Next Steps:</B> The miniprep with the highest concentration will be sent for sequencing to confirm <I>PotD</I> has been inserted into pSB1C3 successfully.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 15/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">This week, cloning of <I>PotD</I> into pQE80-L was started. The gene for <I>Spermine Binding Protein (SBP) </I> also arrived in a plasmid so it was transformed into MC1061 <I>E.coli</I> and plasmid purified.</P></DIV><DIV id="secollapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">15/6: PCR of <I>PotD</I> for Cloning into pQE80-L</SPAN><DIV id="secollapsiblew3-1" class="collapse box-content"><P><B>Aim of experiment:</B> To amplify the <I>PotD</I> gene for cloning into the pQE80-L vector.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#lbpa-figure3-modal" class="journal-protocol">Figure 3</A></P><P><B>Next Steps:</B> The gel shows that <I>PotD</I> has been successfully amplified so it will be digested and ligated into the pQE80-L vector tomorrow.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/6: Restriction Digest and Ligation of <I>PotD</I> into pQE80-L. Transformation of IDT Plasmid containing <I>SBP</I> into MC1061 <I>E.coli</I></SPAN><DIV id="secollapsiblew3-3" class="collapse box-content"><P><B>Aim of experiment:</B> To digest the <I>PotD</I> PCR product with KpnI and BamHI and then ligate it into pQE80-L. To transform MC1061 <I>E.coli</I> with the IDT plasmid containing <I>SBP</I>.</P><P><B>Protocols Used: </B></P><I>PotD</I>: <A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A><I>SBP</I>:<A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformations</A><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> The pQE80-L-<I>PotD</I> ligations will be transformed into MC1061 <I>E.coli</I>. Overnight cultures of the transformations will be set up tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/6: Transformations of <I>PotD</I> + pQE80-L into MC1061 <I>E.coli</I>. Overnight Cultures of MC1061 <I>E.coli</I> with SBP + IDT Plasmid</SPAN><DIV id="secollapsiblew3-4" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the pQE80-L-<I>PotD</I> ligations into MC1061 <I>E.coli</I>. To set up overnight cultures of the transformations of MC1061 <I>E.coli</I> with <I>SBP</I> + IDT Plasmid that were set up yesterday.</P><P><B>Protocols Used: </B><I>PotD</I>:<A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A><I>SBP</I>: <A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures</A></P><P><B>Results:</B>N/A </P><P><B>Next Steps:</B> If transformations are successful, overnight cultures will be set up tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">19/6: Repeat of Ligation of <I>PotD</I> into pQE80-L. Plasmid Purification of the IDT Plasmid Containing <I>SBP</I></SPAN><DIV id="secollapsiblew3-5" class="collapse box-content"><P><B>Aim of experiment:</B> To repeat ligations of <I>PotD</I> into pQE80-L since transformations did not work. To miniprep the overnight cultures that were set up yesterday.</P><P><B>Protocols Used: </B><I>PotD</I>: <A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A><I>SBP</I>: <A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Ligations will be left over the weekend and transformed on Monday. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 22/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I>PotD</I> into the high expression vector pQE80-L was completed. Cloning of <I>SBP</I> into pSB1C3 and pQE80-L was started this week.</P></DIV><DIV id="secollapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/6: Transformation of pQE80-L-<I>PotD</I> Ligations. PCR and Gel Extraction of <I>SBP</I> for Cloning into pQE80-L and Restriction Digests and Ligation of <I>SBP</I> for Cloning into pSB1C3</SPAN><DIV id="secollapsiblew4-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To transform pQE80-L-<I>PotD</I> into  MC1061<I>E.coli</I>. To set up a PCR of <I>SBP</I> using the purified plasmid from Friday and to gel extract the PCR product for cloning into pQE80-L. To digest <I>SBP</I> out of the same plasmid using EcoRI and PstI and subsequently ligate it into pSB1C3.</P><P><B>Protocols Used:</B><I>PotD</I>: <A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A><I>SBP</I>: <A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results: </B><I>SBP</I> :<A data-toggle="modal" data-target="#sbp-figure1-modal" class="journal-protocol"> Figure 1 </A></P><P><B>Next Steps:</B> Overnight cultures will be set up tomorrow for the <I>PotD</I> transformations. The <I>SBP</I> PCR product will be digested and ligated into pQE80-L tomorrow. The pSB1C3-<I>SBP</I> ligations will be transformed into MC1061 <I>E.coli</I>. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/6: Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3-<I>PotD</I>. Transformations of MC1061 <I>E.coli</I> with pSB1C3 + <I>SBP</I>. Restriction Digests and Ligations of <I>SBP</I> into pQE80-L and Transformations into MC1061 <I>E.coli</I></SPAN><DIV id="secollapsiblew4-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of MC1061 <I>E.coli</I> containing pSB1C3 + <I>PotD</I>. To transform MC1061 <I>E.coli</I> with pSB1C3 + <I>SBP</I>. To digest the <I>SBP</I> PCR product with BamHI and Kpn I, ligate it into pQE80-L and transform these ligations into MC1061 <I>E.coli</I>.</P><P><B>Protocols Used:</B><I>PotD</I>: <A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A><I>SBP</I>: <A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The MC1061 <I>E.coli</I> overnight cultures containing pQE80-L + <I> PotD</I> will be miniiprepped tomorrow. Overnight cultures will be set up of the MC1061 <I>E.coli</I> transformations with pSB1C3 + <I>SBP</I> and pQE80-L + <I>SBP</I>.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/6: Plasmid Purification of the Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3-<I>PotD</I>. Overnight Cultures of the Transformations of MC1061 <I>E.coli</I> with pSB1C3 + <I>SBP</I> and pQE80-L + <I>SBP</I>.</SPAN><DIV id="secollapsiblew4-3" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures of MC1061 <I>E.coli</I> containing pSB1C3-<I>PotD</I>. To set up overnight cultures of the transformations of MC1061 <I>E.coli</I> with pSB1C3-<I>SBP</I> and pQE80-L-<I>SBP</I></P><P><B>Protocols Used:</B><I>PotD</I>: <A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A><I>SBP</I>: <A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The miniprep with the highest concentration for pQE80-L + <I>PotD</I> was sent for sequencing. The <I>SBP</I> overnight cultures will be miniprepped tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">25/6: Plasmid Purification of the Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3-<I>SBP</I> and pQE80-L-<I>SBP</I></SPAN><DIV id="secollapsiblew4-4" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures of MC1061 <I>E.coli</I> containing pSB1C3-<I>SBP</I> and pQE80-L + <I>SBP</I>.</P><P><B>Protocols Used:</B><I>SBP</I>: <A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit) </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The miniprep with the highest concentration for both pSB1C3-<I>SBP</I> and pQE80-L-<I>SBP</I> were sent for sequencing. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 29/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Sequencing showed <I> SBP</I> failed to insert successfully into pSB1C3 so cloning was repeated from the ligation stage this week. However, SBP and PotD were successfully inserted into the high expression vector pQE80-L, so overexpression assays were started this week for both.</P></DIV><DIV id="secollapsiblew5" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/6: Ligations of <I>SBP</I> into pSB1C3 and Transformations into MC1061 <I>E.coli</I></SPAN><DIV id="secollapsiblew5-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To ligate <I>SBP</I> into pSB1C3 and transform the ligations into MC1061 <I>E.coli</I>. </P><P><B>Protocols Used:</B><I>SBP</I>: <A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A></P><P><B>Results: </B><I>SBP</I> :<A data-toggle="modal" data-target="#sbp-figure1-modal" class="journal-protocol"> Figure 1 </A></P><P><B>Next Steps:</B> Overnight cultures will be set up tomorrow if transformations are successful. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/6: Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3-<I>SBP</I>, pQE80-L-<I>SBP</I> and pQE80-L-<I>PotD</I></SPAN><DIV id="secollapsiblew5-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of MC1061 <I>E.coli</I> containing pSB1C3-<I>SBP</I>. To set up overnight cultures  of pQE80-L-<I>SBP</I> and pQE80-L + <I>PotD</I> in MC1061 <I>E.coli</I> for overexpression assays to be performed tomorrow. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The MC1061 <I>E.coli</I> overnight cultures containing pSB1C3-<I> SBP</I> will be miniiprepped tomorrow. The MC1061 <I>E.coli</I> overnight cultures containing pQE80-L-<I> SBP</I> and pQE80-L-<I>PotD</I> will be subcultured tomorrow in order to perform overexpression assays.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/7: Plasmid Purification of the Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3-<I>SBP</I>. <I>PotD</I> and <I>SBP</I> Expression Optimization</SPAN><DIV id="secollapsiblew5-3" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures of MC1061 <I>E.coli</I> containing pSB1C3-<I>SBP</I>. To start protein expression optimization for PotD and SBP.</P><P><B>Protocols Used:</B><I>SBP</I>: <A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A><I>SBP</I> and <I>PotD</I>: <A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization</A> Note: Protocol was stopped at Step 8. Samples will be ran on an SDS gel tomorrow.</P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The miniprep with the highest concentration for pSB1C3 + <I>SBP</I> was sent for sequencing. The PotD and SBP samples will be ran on an SDS gel tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">2/7: Continuation of <I>PotD</I> and <I>SBP</I> Expression Optimization</SPAN><DIV id="secollapsiblew5-4" class="collapse box-content"><P><B>Aim of experiment:</B> To run the samples of PotD and SBP on an SDS gel and begin Western Blotting.</P><P><B>Protocols Used:</B><I>PotD</I> and <I>SBP</I>: <A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization</A> Note: Protocol was continued from Step 9.</P><P><B>Results:</B><A data-toggle="modal" data-target="#sbp-figure2-modal" class="journal-protocol"> Figure 2 </A></P><P><B>Next Steps:</B> The membrane for the Western blot has been left at the blocking stage so will be continued tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/7: Western Blot of <I>PotD</I> and <I>SBP</I> Day Culture Samples</SPAN><DIV id="secollapsiblew5-5" class="collapse box-content"><P><B>Aim of experiment:</B> To continue the Western Blot of PotD and SBP.</P><P><B>Protocols Used:</B><I>PotD</I> and <I>SBP</I>: <A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#sbp-figure3-modal" class="journal-protocol"> Figure 3 </A></P><P><B>Next Steps:</B> PotD has been successfully characterized so purification of this protein will be started next week. Further expression optimization experiments for SBP will be set up next week using different concentrations of IPTG, since its characterization was unsuccessful.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 6/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Further protein expression optimization experiments were set up, this time using a range of IPTG concentrations to induce expression of <I>SBP</I>.</P></DIV><DIV id="secollapsiblew6" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">6/7: Further Optimization of <I>SBP</I> Expression</SPAN><DIV id="secollapsiblew6-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up further tests to optimize expression of <I>SBP</I> using different concentrations of IPTG.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization </A> Note: Cultures were induced with 0.5 mM, 1 mM and 2 mM IPTG. The Western Blot was left to block overnight. </P><P><B>Results:</B><A data-toggle="modal" data-target="#sbp-figure4-modal" class="journal-protocol"> Figure 4 </A></P><P><B>Next Steps:</B> The Western Blot will be completed tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">7/7: Western Blot of <I>SBP</I></SPAN><DIV id="secollapsiblew6-2" class="collapse box-content"><P><B>Aim of experiment:</B> To complete the Western Blot that was started yesterday. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot </A> Note: Protocol was continued from Step 12.</P><P><B>Results:</B> Nothing was visible on the blot. </P><P><B>Next Steps:</B> A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after <I>SBP</I> expression has been induced.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 13/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Protein expression optimization experiments were continued this week. A growth curve assay was carried out to assess the effect of <I>SBP</I> expression over a longer period of time. Purification of <I>PotD</I> was also started this week. </P></DIV><DIV id="secollapsiblew7" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/7: Plate Reader Growth Curve Assay</SPAN><DIV id="secollapsiblew7-1" class="collapse box-content"><P><B>Aim of experiment:</B> To induce expression of <I>SBP</I> using different concentration of IPTG and allow cells to grow for 16 hours in order to monitor cell growth.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#platereader-modal" class="journal-protocol"> Growth Curve Assay </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#sbp-figure5-modal" class="journal-protocol"> Figure 5 </A></P><P><B>Next Steps:</B> A plate reader experiment will be set up next week to monitor the growth rate of the the cells over a longer period of time after <I>SBP</I> expression has been induced. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">15/7: Overnight Cultures for 3 L Culture for PotD Purification</SPAN><DIV id="secollapsiblew7-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures that will be used tomorrow to set up 3 L cultures for PotD purification. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The 3 L day culture will be set up tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">16/7: Sample Preparation for Western Blot of SBP and 3 L Day Culture for PotD Purification</SPAN><DIV id="secollapsiblew7-3" class="collapse box-content"><P><B>Aim of experiment:</B> To check if the cells in the induced cultures ~7 hours after being induced are producing SBP. To set up 3 L Day Culture for <I> PotD</I> Purification </P><P><B>Protocols Used: </B><I>SBP</I>: <A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot Sample Preparation </A> Note: The day cultures were grown for 7 hours after they were induced and then the samples were frozen, to be blotted on Monday. <I>PotD</I>:  <A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A>Note: Cultures were left to grow overnight after being induced. </P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> The samples will be ran on an SDS gel on Monday and blotted. The <I>PotD</I> cultures will be harvested tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/7: Harvesting <I>PotD</I> Cultures</SPAN><DIV id="secollapsiblew7-4" class="collapse box-content"><P><B>Aim of experiment:</B> To harvest the day cultures grown yesterday for <I>PotD</I> purification. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: After the cultures were spun down the pellets were frozen. </P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification of <I>PotD</I> will continue on Monday.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 20/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Samples from the growth curve assay were blotted to see if SBP is being produced at the later stages of the growth curve. A further growth curve assay was also performed which monitored cell growth until a certain OD<SUB>600</SUB> was reached, at which point the cultures were induced. Purification of PotD was also continued this week.</P></DIV><DIV id="secollapsiblew8" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">20/7: Western Blot <I>SBP</I> Culture Samples</SPAN><DIV id="secollapsiblew8-1" class="collapse box-content"><P><B>Aim of experiment:</B> To western blot the SBP culture samples from Friday. To continue purfying PotD.</P><P><B>Protocols Used: </B><I>SBP</I>: <A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot </A><I>PotD</I>: <A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: Affinity chromatography was completed but no peaks were observable. </P><P><B>Results:</B> Nothing was visible on the blot. </P><P><B>Next Steps:</B> Another plate reader experiment will be set up on Thursday where the cultures will be monitored for three hours before induction with IPTG and then for a further 20 hours afterwards. Another 150 ml overnight culture was set up for <I>PotD</I> so 3 L cultures could be set up tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">21/7: 3 L Day Cultures for PotD Purification</SPAN><DIV id="secollapsiblew8-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 3 L Day Culture for PotD Purification. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: Cultures were left to grow overnight after being induced. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Purification will be continued tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/7: Continuation of PotD Purification </SPAN><DIV id="secollapsiblew8-3" class="collapse box-content"><P><B>Aim of experiment:</B> To continue purification of PotD. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: The protocol was continued from Step 4 and stopped at Step 13.</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Purification will be continued tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/7: Continuation of PotD Purification and Growth Curve Assay for <I>SBP</I></SPAN><DIV id="secollapsiblew8-4" class="collapse box-content"><P><B>Aim of experiment:</B> To continue purification of PotD and to check if the cells begin to die after inducing expression of <I>SBP</I> with IPTG after having already grown to an OD<SUB>600</SUB> of 0.5. </P><P><B>Protocols Used: </B><I>PotD</I>: <A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: The protocol was continued from Step 13. <I>SBP</I>: <A data-toggle="modal" data-target="#platereader2-modal" class="journal-protocol"> Growth Curve Assay </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#sbp-figure6-modal" class="journal-protocol"> Figure 6 </A></P><P><B>Next Steps:</B> The samples from the size exclusion chromatography performed on PotD will be prepared tomorrow for blotting next week. For <I> SBP</I> further overexpression tests will be set up next week.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/7: SDS-PAGE and Western Blot of PotD  </SPAN><DIV id="secollapsiblew8-5" class="collapse box-content"><P><B>Aim of experiment:</B> To run SDS-PAGE of the elutions obtained on Friday after SEC.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> SDS-PAGE</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#sbp-figure7-modal" class="journal-protocol"> Figure 7 </A></P><P><B>Next Steps:</B> The same samples will be Western Blotted on Monday to confirm that the bands oberved are PotD.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 27/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">PotD was successfully purified and characterized this week. SBP was still unobservable in induced 150ml cultures so 3 L cultures containing pQE80-L-<I>SBP</I> were set up. </P></DIV><DIV id="secollapsiblew9" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/7: Overnight Cultures for <I>SBP</I> Expression Assays and SDS-PAGE and Western Blot of PotD</SPAN><DIV id="secollapsiblew9-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up a 150 ml overnight culture containing pQE80-L-<I>SBP</I> for sub-culturing tomorrow and to run an SDS-PAGE and Western Blot the PotD samples from Friday.</P><P><B>Protocols Used: </B><I>SBP</I>: <A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A> Note: 150 ml of LB was used and 150 µl of each antibiotic. <I>PotD</I>: <A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> SDS-PAGE and Western Blotting</A> Note: The membrane was left blocking overnight.</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> 150 ml of fresh LB will be inoculated tomorrow with 7.5 ml of the overnight cultures containing pQE80-L-<I>SBP</I>. The PotD Western Blot will be completed tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/7: <I>SBP</I> Expression Assay and Completion of PotD Westen Blot</SPAN><DIV id="secollapsiblew9-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up day cultures using the overnight cultures from yesterday and test for expression of LbpA 6 hours after induction.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Test </A> Note: 150 ml of fresh LB was innoculated with 7.5ml of the overnight cultures. Only an uninduced control and an 1 mM IPTG induced culture was set up. The samples were not blotted, they were only ran on an SDS gel. </P><P><B>Results:</B><I>SBP</I>: Nothing was observable on the SDS gel. <I>PotD</I>: <A data-toggle="modal" data-target="#sbp-figure8-modal" class="journal-protocol"> Figure 8 </A></P><P><B>Next Steps:</B> Since PotD has now been successful purified, the next step is to attach a fluorescent nanobead to the protein. Hopefully, this will be attempted in the coming weeks. Even though SBP was not visible on the SDS gel, 3 L cultures will be set up this week to try and harvest the protein since the culture size may be too small. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/7: Overnight Cultures for 3L Culture</SPAN><DIV id="secollapsiblew9-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures containing pQE80-L-<I>SBP</I> for the 3L day cultures that will be grown tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A> Note: One 150ml overnight culture was set up using 150µl of each antibiotic. </P><P><B>Results:</B> N/A   </P><P><B>Next Steps:</B> The 3L culture will be set up tomorrow allowed to grow to OD<SUB>600</SUB> of 0.6-1 and then <I>SBP</I> expression induced with 1mM IPTG and grown for 6 hours.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/7: 3 L Culture for Purification of SBP</SPAN><DIV id="secollapsiblew9-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 3 L day cultures for purifying SBP.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The procedure was stopped at Step 6 and the pellets were frozen at -20<SUP>o</SUP>C. </P><P><B>Results:</B> N/A   </P><P><B>Next Steps:</B> Protein purification will be continued on Monday.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 3/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">It seems that SBP has been successfully purified and characterized. However, there was still a lot of contamination in the samples after size eclusion chromatography was performed. .</P></DIV><DIV id="secollapsiblew10" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/8: Continuation of SBP Purification</SPAN><DIV id="secollapsiblew10-1" class="collapse box-content"><P><B>Aim of experiment:</B> To continue with purification of SBP.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The protocol was continued from Step 7</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Size exclusion chromatography will be carried out tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">4/8: Continuation of SBP Purification</SPAN><DIV id="secollapsiblew10-2" class="collapse box-content"><P><B>Aim of experiment:</B> To continue purification of SBP.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A>Note: The protocol was continued from Step 14. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The fractions corresponding to the peak observed during SEC, will be ran on an SDS gel and Western Blotted tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">6/8: Western Blot of SBP</SPAN><DIV id="secollapsiblew10-3" class="collapse box-content"><P><B>Aim of experiment:</B> To perform a Western Blot on the samples obtained from Size Exclusion Chromatography yesterday..</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blot </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#sbp-figure9-modal" class="journal-protocol"> Figure 9 </A></P><P><B>Next Steps:</B> There was still some contamination in the gel filtrated samples so this will have to be removed before a nanobead can be attached to the protein. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Interactions between PotD and spermidine were analysed via tryptophan fluorescence this week.</P></DIV><DIV id="secollapsiblew11" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/8: Tryptophan Fluorescence</SPAN><DIV id="secollapsiblew11-1" class="collapse box-content"><P><B>Aim of experiment:</B> To analyse the interaction of PotD and spermidine using tryptophan fluorescence. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#tf-modal" class="journal-protocol"> Tryptophan Fluorescence</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#sbp-figure10-modal" class="journal-protocol"> Figure 10 </A></P><P><B>Next Steps:</B> To continue characterization of the spermidine/ PotD interaction using tryptophan fluorescence and other methods. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><SPAN id="biospray-title">Blood</SPAN><DIV id="blcollapsible" class="collapse biospray-content"><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I>hHBA</I> and <I>hHBB</I> into the biobrick vector, pSB1C3, was performed this week.</P></DIV><DIV id="blcollapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/6: Restriction Digests of <I>hHBA</I> and <I>hHBB</I> from IDT gBlocks and Ligation into pSB1C3</SPAN><DIV id="blcollapsiblew1-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To digest <I>hHBA</I> and <I>hHBB</I> from IDT gBlocks using EcoRI and PstI for ligation into pSB1C3.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results: </B>N/A</P><P><B>Next Steps:</B> The pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> ligations will be transformed into JM110 <I>E.coli</I>. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/6: Transformations of pSB1C3-hHBA and pSB1C3-hHBB Ligations into JM110 <I>E.coli</I></SPAN><DIV id="blcollapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> ligations into JM1110 <I>E.coli</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformations</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> If transformations are successful, overnight cultures will be set up.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 15/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">The sequence for pSB1C3-<I>hHBB</I> was confirmed this week so cloning of <I>hHBB</I> into the two hybrid system vector pUT18 was started. Sequencing of pSB1C3-<I>hHBA</I> was incorrect so another miniprep was sent for sequencing.</P></DIV><DIV id="blcollapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">15/6: Overnight Cultures of JM110 <I>E.coli</I> containing pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I></SPAN><DIV id="blcollapsiblew2-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of JM110 <I>E.coli</I> containing pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> from the transformations performed on Tuesday.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> Overnight cultures will be miniprepped tomorrow and a test restriction digest will be performed to check for the presence of the inserts.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">16/6: Plasmid Purification of the JM110 <I>E.coli</I> Overnight Cultures containing pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I></SPAN><DIV id="blcollapsiblew2-2" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the JM110 <I>E.coli</I> overnight cultures containing pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> and perform a pre-sequence digest check.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification (QIAprep® Spin Miniprep Kit) </A><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure1-modal" class="journal-protocol"> Figure 1 </A></P><P><B>Next Steps:</B> The gel indicates that <I>hHBA</I> and <I>hHBB</I> have inserted successfully into pSB1C3 so samples of the minipreps will be sent for sequencing to confirm this.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/6: Amplification of <I>hHBA</I> and <I>hHBB</I> for Cloning into the Two Hybrid System Vectors </SPAN><DIV id="blcollapsiblew2-3" class="collapse box-content"><P><B>Aim of experiment:</B> To amplify <I>hHBA</I> and <I>hHBB</I> for Cloning into the Two Hybrid System Vectors pT25 and pUT18, respectively. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure2-modal" class="journal-protocol"> Figure 2 </A></P><P><B>Next Steps:</B> Sequencing of pSB1C3-<I>hHBA</I> and pSB1C3-<I>hHBB</I> showed that pSB1c3-<I>hHBA</I> had a 1 bp deletion so one of the other minipreps was sent for sequencing. pSB1C3-<I>hHBB</I>, however, was successful so the band for <I>hHBB</I> will be gel extracted and digested tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/6: Gel Extraction and Restriction Digests <I>hHBB</I> for Cloning into pUT18 </SPAN><DIV id="blcollapsiblew2-4" class="collapse box-content"><P><B>Aim of experiment:</B> To gel extract and digest <I>hHBB</I> with BamHI and EcoRI.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> Ligations of <I>hHBB</I> will be set up tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">19/6: Ligations of <I>hHBB</I> into pUT18</SPAN><DIV id="blcollapsiblew2-5" class="collapse box-content"><P><B>Aim of experiment:</B> To set up ligations of pUT18-<I>hHBB</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> Ligations will be left over the weekend and transformed into BTH101 <I>E.coli</I>. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 22/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Cloning of <I>hHBA</I> and <I>hHBB</I> into pT25 and pUT18, respectively, was continued this week.</P></DIV><DIV id="blcollapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/6: Transformations of pUT18-<I>hHBB</I> into BTH101 <I>E.coli</I></SPAN><DIV id="blcollapsiblew3-1" class="collapse box-content"><P><B> Aim of experiment:</B> To transform pUT18-<I>hHBB</I> into BTH101 <I>E.coli</I>.</P><P><B> Protocols Used: </B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A></P><P><B> Results:</B>N/A </P><P><B>Next Steps:</B> If transformations are successful, overnight cultures will be set up tomorrow. Sequencing showed the sequence for pSB1C3-<I>hHBA</I> was incorrect again, a third miniprep will be sent for sequencing.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/6: Overnight Cultures of BTH101 <I>E.coli</I> containing pUT18-<I>hHBB</I></SPAN><DIV id="blcollapsiblew3-2" class="collapse box-content"><P><B> Aim of experiment:</B> To set up overnight cultures of BTH101 <I>E.coli</I> containing pUT18-<I>hHBB</I> from the transformations carried out yesterday.</P><P><B> Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures</A></P><P><B> Results:</B> N/A </P><P><B>Next Steps:</B> The overnight cultures will be miniprepped and sent for sequencing tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/6: Plasmid Purification of the BTH101 <I>E.coli</I> Overnight Cultures containing pUT18-<I>hHBB</I></SPAN><DIV id="blcollapsiblew3-3" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the BTH101 <I>E.coli</I> Overnight Cultures containing pUT18-<I>hHBB</I>.</P><P><B>Protocols Used: </B></P><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> The miniprep with the highest concentration will be sent for sequencing to confirm presence of insert. Sequencing for pSB1C3-<I>hHBA</I> came back correct so another PCR of <I>hHBA</I> will be set up tomorrow for cloning into pT25. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">25/6: Amplification of <I>hHBA</I> for Cloning into pT25</SPAN><DIV id="blcollapsiblew3-4" class="collapse box-content"><P><B>Aim of experiment:</B> To amplify <I>hHBA</I> for cloning into pT25</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure3-modal" class="journal-protocol"> Figure 3 </A></P><P><B>Next Steps:</B> The PCR product digested and ligated into pT25 tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">26/6: Restriction Digests and Ligations of <I>hHBA</I> for Cloning into pT25</SPAN><DIV id="blcollapsiblew3-5" class="collapse box-content"><P><B>Aim of experiment:</B> To digest the <I>hHBA</I> PCR product with BamHI and KpnI and set up ligations with pT25.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests</A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Ligations will be left over the weekend and transformed into BTH101 <I>E.coli</I> on Monday. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 29/6/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I>hHBA</I> into pT25 was completed this week. Sequencing confirmed that <I>hHBB</I> has been successfully inserted into pUT18. Cloning of both <I>hHBA</I> and <I>hHBB</I>, individually, into the high expression vector pQE80-L was started this week.</P></DIV><DIV id="blcollapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/6: Transformation of pT25-<I>hHBA</I> Ligations into BTH101 <I>E.coli</I></SPAN><DIV id="blcollapsiblew4-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To transform pT25-<I>hHBA</I> into  BTH101 <I>E.coli</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A></P><P><B>Results: </B>N/A</P><P><B>Next Steps:</B> Overnight cultures will be set up tomorrow for the pT25-<I>hHBA</I> transformations if successful. Sequencing confirmed <I>hHBB</I> has been successfully inserted into pUT18. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/6: Overnight Cultures of MC1061 <I>E.coli</I> containing pUT18-<I>hHBA</I> and Amplification of <I>hHBA</I> and <I>hHBB</I> for Cloning into pQE80-L</SPAN><DIV id="blcollapsiblew4-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of MC1061 <I>E.coli</I> containing pT25-<I>hHBA</I>. To amplify <I>hHBA</I>.and <I>hHBB</I> for cloning into pQE80-L.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure4-modal" class="journal-protocol"> Figure 4 </A></P><P><B>Next Steps:</B> The BTH101 <I>E.coli</I> overnight cultures containing pT25-<I>hHBB</I> will be miniiprepped tomorrow. The PCR products will be digested and ligate dinto pQE80-L tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/7: Plasmid Purification of the Overnight Cultures of BTH101 <I>E.coli</I> containing pT25-<I>hHBA</I> and Restriction Digests and Ligations of <I>hHBA</I> and <I>hHBB</I> into pQE80-L</SPAN><DIV id="blcollapsiblew4-3" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the overnight cultures of BTH101 <I>E.coli</I> containing pT25-<I>hHBA</I>. To digest <I>hHBA</I> and <I>hHBB</I> with BamHI and KpnI and set up ligations with pQE80-L.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The miniprep with the highest concentration for pT25-<I>hHBA</I> will be sent for sequencing. The pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I> will be transformed into M15[<I>pREP4</I>] <I>E.coli</I> tomrorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">2/7: Transformations of pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I> Ligations into M15[<I>pREP4</I>] <I>E.coli</I></SPAN><DIV id="blcollapsiblew4-4" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I> ligations into M15[<I>pREP4</I>] <I>E.coli</I>.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> If transformations are successful, overnight cultures will be set up on Monday. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 6/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I>hHBA</I> and <I>hHBB</I> into pQE80-L was completed this week. Cloning of <I>Haptoglobin (hHBN) </I> was also started this week.</P></DIV><DIV id="blcollapsiblew5" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">6/7: Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I></SPAN><DIV id="blcollapsiblew5-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To set up overnight cultures of BTH101 <I>E.coli</I> containing pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B>The overnight cultures will be miniprepped tomorrow. Sequencing for pT25-<I>hHBA</I> was incorrect so cloning will be repeated from the restriction digest stage.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">7/7: Plasmid Purification of M15[<I>pREP4</I>] <I>E.coli</I> Overnight Cultures containing pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I></SPAN><DIV id="blcollapsiblew5-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up miniprep the BTH101 <I>E.coli</I> Overnight Cultures containing pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">  Plasmid Purification (QIAprep® Spin Miniprep Kit) </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure5-modal" class="journal-protocol"> Figure 5</A></P><P><B>Next Steps:</B> The pre-sequence digest indicates that both <I>hHBA</I> and <I>hHBB</I> have been inserted into pQE80-L successfully. The minipreps with the highest concentration will be sent for sequencing. The sequence for <I>Haptoglobin (hHBN)</I> arrived in an IDT plasmid, work will be be started on this tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">8/7: Transformation of IDT Plasmid-<I>hHBN</I> into MC1061 <I>E.coli</I></SPAN><DIV id="blcollapsiblew5-3" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the IDT Plasmid-<I>hHBN</I> into MC1061 <I>E.coli</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> Overnight cultures will be set up tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">9/7: Overnight Cultures of MC1061 <I>E.coli</I> containing IDT Plasmid-<I>hHBN</I></SPAN><DIV id="blcollapsiblew5-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of MC1061 <I>E.coli</I> containing IDT Plasmid-<I>hHBN</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> The overnight cultures will be miniprepped tomorrow and <I>hHBN</I> will be amplified for cloning into pSB1C3. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/7: Plasmid Purification of IDT Plasmid-<I>hHBN</I> and Digestion of <I>hHBN</I> for Cloning into pSB1C3</SPAN><DIV id="blcollapsiblew5-5" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the MC1061 <I>E.coli</I> containing IDT Plasmid-<I>hHBN</I> and to digest <I>hHBN</I> out of the plasmid using EcoRI and PstI.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B><I>hHBN</I> will be ligated into pSB1C3 on Monday. Sequencing also confirmed <I>hHBA</I> and <I>hHBB</I> have been successfully inserted into pQE80-L.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 13/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">hHBB was successfully characterized this week so protein purification was started. Cloning of <I>hHBN</I> into pSB1C3 was also completed.</P></DIV><DIV id="blcollapsiblew6" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">13/7: Ligations of <I>hHBN</I> into pSB1C3 and Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBA</I> and pQE80-L-<I>hHBB</I></SPAN><DIV id="blcollapsiblew6-1" class="collapse box-content"><P><B>Aim of experiment:</B> To ligate <I>hHBN</I> into pSB1C3. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures</A></P><P><B>Results:</B>N/A</P><P><B>Next Steps:</B> The pSB1C3-<I>hHBN</I> ligations will be transformed into MC1061 <I>E.coli</I> tomorrow. The overnight cultures will be subcultured tomorrow and induced with IPTG to test protein expression levels. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/7: Transformation of pSB1C3-<I>hHBN</I> into MC1061 <I>E.coli</I> and Optmiization of <I>hHBA</I> and <I>hHBB</I> Expression </SPAN><DIV id="blcollapsiblew6-2" class="collapse box-content"><P><B>Aim of experiment:</B> To transform pSB1C3-<I>hHBN</I> into MC1061 <I>E.coli</I> and set up expression optimization tests for <I>hHBA</I> and <I>hHBB</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Protein Expression Optimization </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure6-modal" class="journal-protocol"> Figure 6 </A></P><P><B>Next Steps:</B> If transformations are successful, overnight cultures will be set up tomorrow. Western Blotting will be continued tomorrow for <I>hHBA</I> and <I>hHBB</I>. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">15/7: Overnight Cultures of MC1061 <I>E.coli</I> containing pSB1C3-<I>hHBN</I> and Western Blotting of <I>hHBA</I> and <I>hHBB</I></SPAN><DIV id="blcollapsiblew6-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight culture of MC1061 <I>E.coli</I> containing pSB1C3-<I>hHBN</I> and finish Western Blotting of <I>hHBA</I> and <I>hHBB</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blotting</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure7-modal" class="journal-protocol"> Figure 7 </A></P><P><B>Next Steps:</B> The blot shows that characterization of hHBB has been successful. Unfortunately, nothing was visible on the blot for hHBA. 3 L cultures for purification of hHBB will be set up this week. Characterization of hHBA may be reattempted. The overnight cultures will be miniprepped tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">16/7: Plasmid Purification of pSB1C3-<I>hHBN</I> and Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBB</I></SPAN><DIV id="blcollapsiblew6-4" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the MC1061 <I>E.coli</I> overnight cultures containing pSB1C3-<I>hHBN</I> and set up overnight cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBB</I> for 3L cultures. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The miniprep with the highest concentration will be sent for sequencing. The overnight cultures will be used to set up 3L day cultures tomorrow for purrification of hHBB. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/7: 3 L Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBB</I></SPAN><DIV id="blcollapsiblew6-5" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 3 L day cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBB</I> for purification of hHBB. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: The protocol was stopped at Step 6. </P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Purification will be continued on Monday.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 20/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">hHBB was successfully purified this week and cloning of <I>hHBN</I> into pQE80-L was completed. </P></DIV><DIV id="blcollapsiblew7" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">20/7: Amplification of <I>hHBN</I> for Cloning into pQE80-L and Purification of hHBB</SPAN><DIV id="blcollapsiblew7-1" class="collapse box-content"><P><B>Aim of experiment:</B> To amplify <I>hHBN</I> for cloning into pQE80-L and continuation of hHBB purification.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR </A><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: Protocol was continued from Step 6. </P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure8-modal" class="journal-protocol"> Figure 8</A></P><P><B>Next Steps:</B> The fractions from the affinity chromatography will be consolidated and concentrated tomorrow. Size exclusion chromatography will also be performed. The <I>hHBN</I> PCR product will be digested and ligated into pQE80-L tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">21/7: Size Exclusion Chromatography of hHBB and Restriction Digests and Ligations of <I>hHBN</I> into pQE80-L</SPAN><DIV id="blcollapsiblew7-2" class="collapse box-content"><P><B>Aim of experiment:</B> To consolidate, concentrate and perform size exclusion chromatography on the hHBB samples. To digest the <I>hHBN</I> PCR product with BamHI and KpnI and subsequently ligate it into pQE80-L. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> SDS </A> Note: An SDS gel was ran of the SEC fractions which corresponded to the observed peak. Western blotting was stopped at the blocking stage. <A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure9-modal" class="journal-protocol"> Figure 9 </A></P><P><B>Next Steps:</B> Western blotting will be completed tomorrow and will confirm if the bands observed on the SDS gel are indeed hHBB. The pQE80-L<I>hHBN</I> ligations will be transformed into M15[<I>pREP4</I>] <I>E.coli</I> tomorrrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/7: Western Blot of hHBB and Transformations of M15[<I>pREP4</I>] <I>E.coli</I> with pQE80-L<I>hHBN</I> Ligations</SPAN><DIV id="blcollapsiblew7-3" class="collapse box-content"><P><B>Aim of experiment:</B> To complete the Western blot of hHBB and transform pQE80-L<I>hHBN</I> into M15[<I>pREP4</I>] <I>E.coli</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blotting </A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure10-modal" class="journal-protocol"> Figure 10 </A></P><P><B>Next Steps:</B> The blot shows hHBB has been successfully purified. If the transformations are successful, overnight cultures will be set up tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/7: Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L<I>hHBN</I></SPAN><DIV id="blcollapsiblew7-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L<I>hHBN</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> The overnight cultures will be miniprepped tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/7: Plasmid Purification of Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L<I>hHBN</I></SPAN><DIV id="blcollapsiblew7-5" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep M15[<I>pREP4</I>] <I>E.coli</I> overnight cultures containing pQE80-L<I>hHBN</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> The miniprep with the highest concentration will be sent for sequencing. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 27/7/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"><I>hHBN</I> was successfully cloned into pQE80-L and cloning started for insertion into pT25.</P></DIV><DIV id="blcollapsiblew8" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/7: Restriction Digests and Ligations of <I>hHBN</I> into pQE80-L</SPAN><DIV id="blcollapsiblew8-1" class="collapse box-content"><P><B>Aim of experiment:</B> To digest <I>hHBN</I> again with BamHI and KpnI and set up ligations with pQE80-L.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Resriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> Ligations will be transformed into M15[<I>pREP4</I>] <I>E.coli</I> tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/7: Transformation of M15[<I>pREP4</I>] <I>E.coli</I> with pQE80-L-<I>hHBN</I></SPAN><DIV id="blcollapsiblew8-2" class="collapse box-content"><P><B>Aim of experiment:</B> To transform M15[<I>pREP4</I>] <I>E.coli</I> with pQE80-L-<I>hHBN</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> If transformations are successful, overnight cultures will be set up tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/7: Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBN</I></SPAN><DIV id="blcollapsiblew8-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBN</I></P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The overnight cultures will be miniprepped tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/7: Plasmid Purification of M15[<I>pREP4</I>] <I>E.coli</I> Overnight Cultures containing pQE80-L-<I>hHBN</I></SPAN><DIV id="blcollapsiblew8-4" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the M15[<I>pREP4</I>] <I>E.coli</I> Overnight Cultures containing pQE80-L-<I>hHBN</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>The miniprep with the highest concentration will be sent for sequencing. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">31/7: Amplification of <I>hHBN</I> for Cloning into pT25</SPAN><DIV id="blcollapsiblew8-5" class="collapse box-content"><P><B>Aim of experiment:</B> To amplfy <I>hHBN</I> for cloning into the two hybrid vector pT25.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol"> PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure15-modal" class="journal-protocol"> Figure 11 </A></P><P><B>Next Steps:</B>The PCR product will be digested and ligated into pT25 on Monday.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 3/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I>hHBN</I> into pT25 was completed this week but subsequent sequencing showed that it had not inserted successfully. </P></DIV><DIV id="blcollapsiblew9" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/8: Restriction Digests and Ligations of <I>hHBN</I> for Cloning into pT25</SPAN><DIV id="blcollapsiblew9-1" class="collapse box-content"><P><B>Aim of experiment:</B> To digest the <I>hHBN</I> PCR product with BamHI and KpnI and set up ligations with the pT25 vector.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol"> Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligations</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>The pT25-<I>hHBN</I> ligations will be transformed into MG1655 <I>E.coli</I> tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">4/8: Transformations of  pT25-<I>hHBN</I> into MG1655 <I>E.coli</I></SPAN><DIV id="blcollapsiblew9-2" class="collapse box-content"><P><B>Aim of experiment:</B> To set up transformations of pT25-<I>hHBN</I> into MG1655 <I>E.coli</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol"> Transformations </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B> If transformations are successful, overnight cultures will be set up tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">5/8: Overnight Cultures of MG1655 <I>E.coli</I> containing pT25-<I>hHBN</I></SPAN><DIV id="blcollapsiblew9-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of MG1655 <I>E.coli</I> containing pT25-<I>hHBN</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A   </P><P><B>Next Steps:</B> The overnight cultures will be miniprepped tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">6/8: Plasmid Purification of MG1655 <I>E.coli</I> Overnight Cultures containing pT25-<I>hHBN</I></SPAN><DIV id="blcollapsiblew9-4" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the  MG1655 <I>E.coli</I> overnight cultures containing pT25-<I>hHBN</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification (QIAprep® Spin Miniprep Kit)</A></P><P><B>Results:</B> N/A   </P><P><B>Next Steps:</B> The miniprep with the highest concentration will be sent for sequencing. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Overexpression assays of <I>hHBN</I> were set up this week and purification of the protein was begun. </P></DIV><DIV id="blcollapsiblew10" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/8: Overnight Cultures for <I>hHBN</I> Overexpression Assays</SPAN><DIV id="blcollapsiblew10-1" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBN</I>. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures </A></P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B>The overnight cultures will be used to set up day cultures that will be induced with different concentrations of IPTG to assay <I>hHBN</I> expression levels.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/8: Overexpression Assay of pQE80-L-<I>hHBN</I></SPAN><DIV id="blcollapsiblew10-2" class="collapse box-content"><P><B>Aim of experiment:</B> To assay <I>hHBN</I> expression levels when induced with a range of IPTG concentrations. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Overexpression Assay </A>Note: Day cultures were grown to an OD<SUB>600</SUB> of 0.6 and then induced with 0.25 mM, 0.5 mM, 0.75 mM and 1 mM IPTG and allowed to grow for 6 hours. </P><P><B>Results:</B> SDS gel </P><P><B>Next Steps:</B> The gel indicates that <I>hHBN</I> is not being overexpressed in any of the cultures. Further optimization experiments will be set up to try and overexpress <I>hHBN</I>. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">12/8: Overnight Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBN</I></SPAN><DIV id="blcollapsiblew10-3" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBN</I> which will be used to set up a 3 L day culture tomorrow.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol"> Overnight Cultures</A> Note: A 150 ml overnight culture was set up. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The overnight cultures will be used to set up 3 L day cultures tomorrow.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">13/8: 3 L Day Cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBN</I></SPAN><DIV id="blcollapsiblew10-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up 3 L day cultures of M15[<I>pREP4</I>] <I>E.coli</I> containing pQE80-L-<I>hHBN</I> using the overnight culture set up yesterday.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A> Note: The cultures were grown to OD<SUB>600</SUB>=0.9, then induced with 1 mM IPTG and allowed to grow for a further 8 hours. A 1 ml sample was taken before the cultures were spun down. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> Protein purification will be continued next week. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 17/8/15</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Purification of haptoglobin was completed this week and confirmed via Western Blotting. </P></DIV><DIV id="blcollapsiblew11" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/8: hHBN Purification</SPAN><DIV id="blcollapsiblew11-1" class="collapse box-content"><P><B>Aim of experiment:</B> To continue purification of hHBN by performing affinity chromatography. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification </A> Note: The protocol was stopped at Step 13.</P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The samples corresponding to the peak observed in the chromatogram will be combined and purified using SEC tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/8: hHBN Purification </SPAN><DIV id="blcollapsiblew11-2" class="collapse box-content"><P><B>Aim of experiment:</B> To continue purification of hHBN by performing size exclusion chromatography. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#pp-modal" class="journal-protocol"> Protein Purification</A></P><P><B>Results:</B> N/A </P><P><B>Next Steps</B>: The gel shows bands corresponding to 45 kDa, the expected size of hHBN. This will be checked via Western Blotting to ensure it is the correct protein. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">19/8: Western Blotting of Purified hHBN</SPAN><DIV id="blcollapsiblew11-3" class="collapse box-content"><P><B>Aim of experiment:</B> To perform Western Blotting in order to verify the protein that was purified is indeed hHBN. </P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blotting </A> Note: The membrane was left blocking in milk overnight. </P><P><B>Results:</B> N/A </P><P><B>Next Steps:</B> The blot will be completed tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">20/8: Western Blotting of Purified hHBN</SPAN><DIV id="blcollapsiblew11-4" class="collapse box-content"><P><B>Aim of experiment:</B> To complete Western Blotting of hHBN.</P><P><B>Protocols Used: </B><A data-toggle="modal" data-target="#sds-modal" class="journal-protocol"> Western Blotting</A> The protocol was continued from Step 12.</P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure14-modal" class="journal-protocol"> Figure 14</A></P><P><B>Next Steps:</B> The blot indicates that hHBN has been successfully purified.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 24/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I>hHBA</I> into pT25 was carried out this week. </P></DIV><DIV id="blcollapsiblew12" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/8: Amplification of <I>hHBA</I> for Cloning into pT25</SPAN><DIV id="blcollapsiblew12-1" class="collapse box-content"><P><B>Aim of experiment:</B> To amplify <I>hHBA</I> for subsequent cloning into the two hybrid vector pT25.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure16-modal" class="journal-protocol"> Figure 15</A></P><P><B>Next Steps:</B> The gel extracted PCR product will be digested and ligated into pT25 tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">25/8: Restriction Digests of <I>hHBA</I> and Ligations into pT25 </SPAN><DIV id="blcollapsiblew12-2" class="collapse box-content"><P><B>Aim of experiment:</B> To digest the <I>hHBA</I> with BamHI and KpnI and set up ligations with pT25. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligations</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B>The ligations will be left overnight and transformed into MG1065∆cya <I>E.coli</I> tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">26/8: Transformations of pT25-<I>hHBA</I> into MG1065∆cya <I>E.coli</I></SPAN><DIV id="blcollapsiblew12-3" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the pT25-<I>hHBA</I> ligations into MG1065∆cya <I>E.coli</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformations </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> If transformations are successful, colonies will be pick for setting up overnight cultures tomorrow.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/8: Overnight Cultures of MG1065∆cya <I>E.coli</I> containing pT25-<I>hHBA</I></SPAN><DIV id="blcollapsiblew12-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of MG1065∆cya <I>E.coli</I> containing pT25-<I>hHBA</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The overnight cultures will be miniprepped tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/8: Plasmid Purification of MG1065∆cya <I>E.coli</I> containing pT25-<I>hHBA</I> Overnight Cultures</SPAN><DIV id="blcollapsiblew12-5" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the  MG1065∆cya <I>E.coli</I> overnight cultures containing pT25-<I>hHBA</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification (QIAprep® Spin Miniprep Kit)</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The miniprep with highest concentration was sent for sequencing to confirm if the insert has been cloned in successfully. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 7/9/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Cloning of <I>hHBN</I> into pUT18 was carried out this week. </P></DIV><DIV id="blcollapsiblew13" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">7/9: Amplification of <I>hHBN</I> for Cloning into pUT18</SPAN><DIV id="blcollapsiblew13-1" class="collapse box-content"><P><B>Aim of experiment:</B> To amplify <I>hHBN</I> for subsequent cloning into the two hybrid vector pUT18.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#bl-figure17-modal" class="journal-protocol"> Figure 16</A></P><P><B>Next Steps:</B> The gel extracted PCR product will be digested and ligated into pT25 tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">8/9: Restriction Digests of <I>hHBN</I> and Ligations into pUT18 </SPAN><DIV id="blcollapsiblew13-2" class="collapse box-content"><P><B>Aim of experiment:</B> To digest the <I>hHBN</I> with BamHI and KpnI and set up ligations with pUT18. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests </A><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligations</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B>The ligations will be left overnight and transformed into MG1065∆cya <I>E.coli</I> tomorrow. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">9/9: Transformations of pUT18-<I>hHBN</I> into MG1065∆cya <I>E.coli</I></SPAN><DIV id="blcollapsiblew13-3" class="collapse box-content"><P><B>Aim of experiment:</B> To transform the pUT18-<I>hHBN</I> ligations into MG1065∆cya <I>E.coli</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformations </A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> If transformations are successful, colonies will be picked for setting up overnight cultures tomorrow.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/9: Overnight Cultures of MG1065∆cya <I>E.coli</I> containing pUT18-<I>hHBN</I></SPAN><DIV id="blcollapsiblew13-4" class="collapse box-content"><P><B>Aim of experiment:</B> To set up overnight cultures of MG1065∆cya <I>E.coli</I> containing pUT18-<I>hHBN</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The overnight cultures will be miniprepped tomorrow.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/9: Plasmid Purification of MG1065∆cya <I>E.coli</I> containing pUT18-<I>hHBN</I> Overnight Cultures</SPAN><DIV id="blcollapsiblew13-5" class="collapse box-content"><P><B>Aim of experiment:</B> To miniprep the  MG1065∆cya <I>E.coli</I> overnight cultures containing pUT18-<I>hHBN</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification (QIAprep® Spin Miniprep Kit)</A></P><P><B>Results:</B> N/A
       </P><P><B>Next Steps:</B> The miniprep with highest concentration was sent for sequencing to confirm if the insert has been cloned in successfully. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><SPAN id="biospray-title">Nasal Mucus </SPAN><DIV id="nmcollapsible" class="collapse biospray-content"><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 15/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Produced a purified plasmid of pIDT-<I>OBP2A</I> in order to use it as template in PCR. </P></DIV><DIV id="nmcollapsiblew1" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/6: Transformation of pIDT-<I>OBP2A</I> in MC1610 <I>E.coli</I></SPAN><DIV id="nmcollapsiblew1-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To transform pIDT-<I>OBP2A</I> into MC1061 <I> e.coli</I> strain. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> The result of this transformation can be seen tomorrow. If successful then the next step will be to produce an overnight culture in preparation of subsequent plasmid purification. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/6: Overnight culture of pIDT-<I>OBP2A</I> plasmid in MC1061  </SPAN><DIV id="nmcollapsiblew1-2" class="collapse box-content"><P><B>Aim of experiment:</B> To produce an overnight culture of pIDT-<I>OBP2A</I> from positive colonies of MC1061 from the transformation done on the 17/6.  
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A></P><P><B>Results:</B> N/A
                  </P><P><B>Next Steps:</B> If the overnight culture successfully grows then the next stage of experimentation will be to produce a plasmid purification of pIDT-<I>OBP2A</I> from that overnight culture.                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">19/6: Plasmid Purification of pIDT-<I>OBP2A</I> from the overnight culture    </SPAN><DIV id="nmcollapsiblew1-3" class="collapse box-content"><P><B>Aim of experiment:</B> The overnight culture produced on the 18/6 will now undergo a plasmid purification in order to obtain  the pIDT-<I>OBP2A</I> plasmid.</P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol"> Plasmid Purification </A></P><P><B>Results:</B> The plasmid purification yielded a concentration of 305.53 ng/ul.
                  </P><P><B>Next Steps:</B> Assuming the plasmid purification has been successful, then the next stage of experimentation will be to run a PCR using this purified pIDT-<I>OBP2A</I> plasmid as the template for subsequent amplification.              </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="labtitle">Week Beginning 22/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Removal of the signaling peptide from <I>OBP2A</I> and insertion of  <I>OBP2A</I> into the biobrick vector pSB1C3. </P></DIV><DIV id="nmcollapsiblew2" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/6: PCR of pIDT-<I>OBP2A</I> using primers designed for the removal of the signal peptide  </SPAN><DIV id="nmcollapsiblew2-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a PCR of the purified pIDT-<I>OBP2A</I> plasmid using a set of primers that have been designed to remove the signaling peptide at the start of the <I>OBP2A</I> gene, whilst leaving the rest of the gene intact.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> If the PCR is successful, then the next step will be to gel extract <I>OBP2A</I> from a  the  gel corresponding to the size of the <I>OBP2A</I> gene fragment.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/6: Gel extraction of <I>OBP2A</I> from the PCR performed on the 22/6  </SPAN><DIV id="nmcollapsiblew2-2" class="collapse box-content"><P><B>Aim of experiment:</B> To perform a gel extraction of <I>OBP2A</I> from a PCR reaction that was performed on the 22/6. Once gel extraction has been performed a subsequent gel will be done to determine if the gel extraction had been successful.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#gelextraction-modal" class="journal-protocol">Gel Extraction </A></P><P><B>Results:</B><A data-toggle="modal" data-target="#nm-figure1-modal" class="journal-protocol"> Figure 1 </A><A data-toggle="modal" data-target="#nm-figure2-modal" class="journal-protocol"> Figure 2</A></P><P><B>Next Steps:</B> To perform a restriction of <I>OBP2A</I> in preparation for ligation.             
 </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/6:  Restriction, ligation and transformation of <I>OBP2A</I> into pSB1C3 </SPAN><DIV id="nmcollapsiblew2-3" class="collapse box-content"><P><B>Aim of experiment:</B>  To perform a restriction, ligation and subsequently a transformation of  of the gel extracted <I>OBP2A</I> into the JM110 <I>E.coli</I> strain.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digest  </A>:<A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation </A>:<A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation  </A></P><P><B>Results:</B> N/A


                  </P><P><B>Next Steps:</B> If the transformation is successful and positive colonies form then these colonies will then be grown overnight in preparation for plasmid purification.                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">25/6: Re-ligation and transformation of <I>OBP2A</I> into pSB1C3 </SPAN><DIV id="nmcollapsiblew2-4" class="collapse box-content"><P><B>Aim of experiment:</B> The transformation performed on the 24/6 failed and as a result <I>OBP2A</I> will now be re-ligated into the biobrick vector pSB1C3 and subsequently transformed into the JM110 <I>E.coli</I> strain.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation:</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#nm-figure3-modal" class="journal-protocol"> Figure 3 </A></P><P><B>Next Steps:</B>If the transformation is successful and positive colonies form then these colonies will then be grown overnight in preparation for plasmid purification.                   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">26/6: Overnight cultures of pSB1C3-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew2-5" class="collapse box-content"><P><B>Aim of experiment:</B> To produce overnight cultures of pSB1C3-<I>OBP2A</I> from a positive colony from the transformation done on the 25/6.  
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results:</B> N/A</P><P><B>Next Steps:</B>  Produce plasmid purifications from the overnight cultures of pSB1C3-<I>OBP2A</I> in preparation for presequence digest.                </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/6: Plasmid Purification and Presequence Digest of pSB1C3-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew2-6" class="collapse box-content"><P><B>Aim of experiment:</B> To perform a plasmid purification of the overnight cultures of pSB1C3-<I>OBP2A</I> from the 26/6. Then to subsequently perform a presequence digest of those plasmid purification.   
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification </A><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Restriction Digests</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure4-modal" class="journal-protocol"> Figure 4 </A></P><P><B>Next Steps:</B> It was concluded from the results of these experiments that sample 5 of pSB1C3-<I>OBP2A</I> will be sent for sequencing.
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="box"><DIV class="labtitle">Week Beginning 29/6/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Positive Sequencing of pSB1C3-<I>OBP2A</I> and subsequent plasmid purification of pSB1C3-<I>OBP2A</I>. </P></DIV><DIV id="nmcollapsiblew3" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">29/6: Plasmid purified pSB1C3-<I>OBP2A</I> is sent for sequencing  </SPAN><DIV id="nmcollapsiblew3-1" class="collapse box-content"><P><B>Aim of Experiment:</B> Sample 5 of the plasmid purified pSB1C3-<I>OBP2A</I> that was done on the 24/6 is sent for sequencing to determine if <I>OBP2A</I> has successfully ligated into pSB1C3.   </P><P><B>Protocols Used:</B> N/A</P><P><B>Results: </B> N/A 

                </P><P><B>Next Steps:</B> If sequencing comes back positive, then the next step will be to clone <I>OBP2A</I> into the two vectors of the bacterial two hybrid system.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/6: Overnight culture of pSB1C3-<I>OBP2A</I> from  the sequenced colony  </SPAN><DIV id="nmcollapsiblew3-2" class="collapse box-content"><P><B>Aim of Experiment:</B> Sequencing came back positive for pSB1C3-<I>OBP2A</I>. So an overnight culture of pSB1C3-<I>OBP2A</I>  will be done in preparation for plasmid purification.     </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> Once the overnight culture has been given ample time to grow, the culture will then undergo a plasmid purification. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/7: Plasmid Purification of pSB1C3-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew3-3" class="collapse box-content"><P><B>Aim of Experiment:</B> To produce a plasmid purification of the sequenced pSB1C3-<I>OBP2A</I> from the overnight culture set up on the 30/6.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification </A></P><P><B>Results: </B> The plasmid purified pSB1C3-<I>OBP2A</I> produced a concentration of 448.83 ng/ul.  </P><P><B>Next Steps:</B> The next stage of experimentation is to use the plasmid purification of pSB1C3-<I>OBP2A</I> as the template for PCR using primers that are designed to separate the gene for use in the two hybrid system.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="labtitle">Week Beginning 12/7/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Ran PCR of <I>OBP2A</I> using primers for insertion of <I>OBP2A</I> into pUT18 and pT25. </P></DIV><DIV id="nmcollapsiblew4" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/7: PCR of <I>OBP2A</I> for insertion into bacterial two hybrid vectors: pUT18 and pT25 </SPAN><DIV id="nmcollapsiblew4-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a PCR of <I>OBP2A</I> for its insertion into the bacterial two hybrid vectors; pUT18 and pT25. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B>The next stage of experimentation will be to gel extract and then subsequently restrict the amplified gene fragments of <I>OBP2A</I>.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 20/7/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Attempted insertion of <I>OBP2A</I> into the vectors for the bacterial two hybrid system; pUT18 and pT25.  </P></DIV><DIV id="nmcollapsiblew5" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">20/7: Gel Extraction and Restriction Digest of <I>OBP2A</I> from the PCR performed on 17/7  </SPAN><DIV id="nmcollapsiblew5-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a gel extraction and then subsequently a restriction of <I>OBP2A</I> from the PCR produced on the 17/7 using primers designed for the insertion of <I>OBP2A</I> into the vectors for the bacterial two hybrid system, pUT18 and pT25.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digest </A></P><P><B>Results: </B>N/A</P><P><B>Next Steps:</B> If the gel extraction and subsequent restriction is successful, then the next step will be to ligate the restricted <I>OBP2A</I> into the bacterial two hybrid vector, pUT18 and pT25.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">21/7: Ligation and Transformation of <I>OBP2A</I> into pUT18 and pT25.     </SPAN><DIV id="nmcollapsiblew5-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a ligation and  then subsequently a transformation  of <I>OBP2A</I> into the vectors for the bacterial two hybrid system, pUT18 and pT25. Whilst using JM110 <I>E.coli</I> cells as a chassis for transformation. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol">Ligation</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results:  N/A </B></P><P><B>Next Steps:</B> The results from the transformation will be seen tomorrow. If successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight colonies in preparation for purification and sequencing.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/7: Re-ligation and Transformation of <I>OBP2A</I> into pUT18 and PT25.
  </SPAN><DIV id="nmcollapsiblew5-3" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a re-ligation and transformation of <I>OBP2A</I> into pUT18 and pT25 - since the transformation performed on the 21/7 failed. 
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligation</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results: N/A </B></P><P><B>Next Steps:</B> If transformation is successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight colonies in preparation for purification and sequencing.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">23/7: Production of overnight cultures of pUT18-OBP2A. </SPAN><DIV id="nmcollapsiblew5-4" class="collapse box-content"><P><B>Aim of Experiment:</B>  To produce overnight cultures of pUT18-<I>OBP2A</I>. Unfortunately no positive colonies were produced from the transformation of <I>OBP2A</I> into pT25.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> Once the cells have been given ample time to grow, they will undergo plasmid purification for subsequent sequencing.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/7: Plasmid Purification and subsequent Presequence digest of pUT18-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew5-5" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a plasmid purification of  the overnight cultures of pUT18-<I>OBP2A</I> and then a subsequent presequence digest.  
                  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification</A><A data-toggle="modal" data-target="#presqunce digest-modal" class="journal-protocol">Restriction Digest</A></P><P><B>Results:</B><A data-toggle="modal" data-target="#nm-figure5-modal" class="journal-protocol"> Figure 5 </A></P><P><B>Next Steps:</B> If sequencing comes back positive for pUT18-<I>OBP2A</I> then this can now be implemented into the bacterial two hybrid system. Another attempt at cloning <I>OBP2A</I> into pT25 will be done next week.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 27/7/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Successful insertion of the two subunits of <I>OBP2A</I> into the vectors pUT18 and pT25.   </P></DIV><DIV id="nmcollapsiblew6" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">27/7: PCR  and Gel extraction of <I>OBP2A</I> for Preparation of Insertion into  pT25.   </SPAN><DIV id="nmcollapsiblew6-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a PCR and subsequent gel extraction of <I>OBP2A</I> for its insertion into pT25.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure6-modal" class="journal-protocol"> Figure 6 </A></P><P><B>Next Steps:</B> The next stage of experimentation will be to restrict <I>OBP2A</I> for subsequent ligation into pT25.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">28/7: Restriction Digest of <I>OBP2A</I> for pT25 and Positive Sequencing of pUT18-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew6-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a restriction digest of <I>OBP2A</I> for subsequent ligation into PT25. Also, sequencing came back positive for pUT18-<I>OBP2A</I>, so pUT18-<I>OBP2A</I> can now be implemented in the bacterial two hybrid system.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#restrictiondigest-modal" class="journal-protocol">Restriction Digests</A></P><P><B>Results: </B>N/A </P><P><B>Next Steps:</B> The next stage of experimentation will be to ligate <I>OBP2A</I> into the pT25 vector.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">30/7: Ligation and Transformation of <I>OBP2A</I> into pT25.    </SPAN><DIV id="nmcollapsiblew6-3" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a ligation and transformation of <I>OBP2A</I> into the bacterial two hybrid vector pT25, so that the interaction between the two constituents of <I>OBP2A</I> can be observed using the bacterial two hybrid system.   </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ligation-modal" class="journal-protocol"> Ligation</A><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: N/A</B></P><P><B>Next Steps:</B> If the transformation is successful and positive colonies form on the antibiotic plates, then the next step will be to grow overnight cultures of those colonies in preparation for purification and sequencing.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">31/7: Overnight Cultures of pT25-<I>OBP2A</I> from Positive Colonies. </SPAN><DIV id="nmcollapsiblew6-4" class="collapse box-content"><P><B>Aim of Experiment:</B> To produce overnight cultures of pT25-<I>OBP2A</I> from the transformation done on the 30/7. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> Once the cells have grown overnight they will undergo plasmid purification for subsequent sequencing.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">1/8: Plasmid Purification and Subsequent Presequence Restriction Digest of pT25-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew6-5" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a plasmid purification of pT25-<I>OBP2A</I>  from the overnight cultures done on the 31/7 and then a subsequent presequence digest.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#miniprep-modal" class="journal-protocol">Plasmid Purification</A><A data-toggle="modal" data-target="#pre-sequence digest-modal" class="journal-protocol"> Restriction Digest</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure7-modal" class="journal-protocol"> Figure 7 </A></P><P><B>Next Steps:</B> If sequencing comes back positive for pT25-<I>OBP2A</I> then this can now be implemented into the bacterial two hybrid system. The results from the sequencing should arrive early on the 3/8.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 3/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content">Positive sequencing of pT25-<I>OBP2A</I>, subsequent transformation into the bacterial two hybrid system and plating onto MacConkey agar.    </P></DIV><DIV id="nmcollapsiblew7" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">3/8: Transformation of pUT18-<I>OBP2A</I> into BTH101 <I>E.coli</I> strain.    </SPAN><DIV id="nmcollapsiblew7-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a transformation of pUT18-<I>OBP2A</I> into BTH101, so that a subsequent transformation of pT25-<I>OBP2A</I> into the same chassis can occur as both plasmids are required to reside in the same chassis so that their interaction can be observed.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> If the transformation is successful, then the next stage of experimentation will be to re-transform this culture of cells using the pT25-<I>OBP2A</I> plasmid.      </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">4/8:Day culture and transformation of BTH101 containing pUT18-<I>OBP2A</I> with pT25<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew7-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a day culture and then a subsequent transformation of the BTH101 chassis containing  pUT18-<I>OBP2A</I> from the transformation performed on 3/8.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> If the transformation is successful then the next stage of experimentation will be to observe the  protein interaction of the two subunits of <I>OBP2A</I> using MacConkey agar plates.     </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">5/8:Day culture and plating of positive colonies from transformation onto MacConkey agar.  </SPAN><DIV id="nmcollapsiblew7-3" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a day culture and then subsequent plating of positive colonies from the transformation performed on the 4/8. Positive colonies from this transformation contain both vectors of the bacterial two hybrid system along with the two subunits of <I>OBP2A</I> and so now the interaction can be observed. The day culture will be plated onto MacConkey agar which once the cells have been given enough time to grow will give a visual demonstration of the  protein interaction through a color change.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B>N/A </P><P><B>Next Steps:</B> It has been decided that whilst the <I>E.coli</I> strain BTH101 that is being used in this experiment is useful for visualizing protein interaction, it is very unstable. A better <I>E.coli</I> strain called MG1655(Δcya) will be used instead as it is much more stable. This means that tomorrow overnights will be prepared of MG1655(Δcya)for transformation with pUT18-<I>OBP2A</I> and then subsequently pT25-<I>OBP2A</I>.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">6/8: Overnight cultures of MG1655(Δcya)</SPAN><DIV id="nmcollapsiblew7-4" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform an overnight culture of MG1655(Δcya)<I>E.coli</I> in preparation for transformation with pUT18-<I>OBP2A</I> and then pT25-<I>OBP2A</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Culture </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> Once the overnight culture has been give sufficient time to grow these cells will then be transformed with pUT18-<I>OBP2A</I> and then subsequently re-transformed with pT25-<I>OBP2A</I>.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">7/8: Results of MacConkey agar growth and transformation of  MG1655(Δcya)with pUT18-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew7-5" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a transformation of MG1655(Δcya)with pUT18-<I>OBP2A</I>. Also the results of plating of our bacterial two hybrid interaction onto MacConkey agar has produced promising results as the colonies appear to be taking up a small amount of red dye - an indicator of protein interaction. This suggests that the two separated parts of <I>OBP2A</I> may in fact be interacting in vivo. This interaction will need to be quantified through the use of the β-galactosidase assay. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> If the transformation of MG1655(Δcya) with pUT18-<I>OBP2A</I> is successful then the next step of experimentation will be re-transform positive colonies with pT25-<I>OBP2A</I>.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">8/8: Transformation of MG1655(Δcya) containing pUT18-<I>OBP2A</I> with pT25-<I>OBP2A</I></SPAN><DIV id="nmcollapsiblew7-6" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a transformation of MG1655(Δcya) containing pUT18-<I>OBP2A</I> with pT25-<I>OBP2A</I>. Positive colonies were produced from the transformation done on the 7/7 and so as a result a day culture was produced and subsequently re-transformed with pT25-<I>OBP2A</I>. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#transformation-modal" class="journal-protocol">Transformation</A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> If the transformation of MG1655(Δcya) with pT25-<I>OBP2A</I> is successful then the next step of experimentation will be to undertake the β-galactosidase assay  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 10/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> First attempt at determination of the protein interaction of the <I>OBP2A</I> subunits using the β-galactosidase assay.   </P></DIV><DIV id="nmcollapsiblew8" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">10/8: Plating of Controls in preparation of the β-galactosidase assay   </SPAN><DIV id="nmcollapsiblew8-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To plate control interactions that are used in the β-galactosidase assay to aid in the interpretation of the results form the assay. This first assay attempt will consist of one positive control, one negative control and one regulatory control.       </P><P><B>Protocols Used: N/A </B></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> The next stage of experimentation will be to perform overnight cultures from positive colonies in preparation for the β-galactosidase assay.      </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">11/8: Overnight cultures of controls and <I>OBP2A</I> in preparation for the β-galactosidase assay.  </SPAN><DIV id="nmcollapsiblew8-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To prepare overnight cultures of positive colonies of MG1655(Δcya) containing pUT18-<I>OBP2A</I> and pT25-<I>OBP2A</I>, along with the controls in preparation for the  β-galactosidase assay. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> Once the cells have been given sufficient time to grow then the next stage of experimentation will be to inoculate those colonies into fresh liquid broth until they reach an O.D of 0.4 before being pellet and frozen for the assay.    </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title"> 12/8: Inoculation and preparation of cultures for the  β-galactosidase assay.    </SPAN><DIV id="nmcollapsiblew8-3" class="collapse box-content"><P><B>Aim of Experiment:</B>  To prepare cultures for use in the  β-galactosidase assay, this requires a small subsample of each overnight to be inoculated into fresh LB, two done for each colony. Then left to grow until the O.D of each culture is between 0.3-0.5. Once an O.D of 0.3-0.5 has been reached then 3 replicas of 1 ml samples are taken from each culture before being spun down, the supernatant then subsequently removed and the pellet frozen.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ap-modal" class="journal-protocol"> Assay Preparation</A></P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> These samples are now ready for the β-galactosidase assay.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">13/8: First attempt at the determination of the protein interaction of the two <I>OBP2A</I> subunits using the β-galactosidase assay </SPAN><DIV id="nmcollapsiblew8-4" class="collapse box-content"><P><B>Aim of Experiment:</B> To determine the millers activity of the two interacting subunits of <I>OBP2A</I> in vivo as a basis of the level of interaction between them. This was done using the β-galactosidase assay. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#bga-modal" class="journal-protocol">β-galactosidase assay</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure8-modal" class="journal-protocol"> Figure 8 </A></P><P><B>Next Steps:</B> As it was the first attempt at a β-galactosidase assay, subsequent assays will be performed utilizing more controls. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">14/8: Preparation of overnight cultures of controls and samples in preparation of performing a second β-galactosidase assay    </SPAN><DIV id="nmcollapsiblew8-5" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform overnight cultures of the controls and samples that will be necessary for doing this second β-galactosidase assay. This assay will include two positive controls which are known to have a high level of interaction - these are NarG-NarJ and Zip positive controls. There will also be two negative controls, an empty MG1655(Δcya) chassis and pUT18-<I>OBP2A</I> with NarJ. Finally there is one regulatory control: which contains two empty vectors of pUT18 and pT25.     </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#onc-modal" class="journal-protocol">Overnight Cultures </A></P><P><B>Results: </B>N/A </P><P><B>Next Steps:</B> The next stage of experimentation will be to prepare samples for use in the β-galactosidase assay.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 17/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Second attempt at determining the protein interactions of the two <I>OBP2A</I> subunits using the β-galactosidase assay and subsequent cloning of <I>OBP2A</I> into the bacterial two hybrid vectors in switched order.  </P></DIV><DIV id="nmcollapsiblew9" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">17/8: Preparation of the controls and samples for use in the second attempt of the β-galactosidase assay  </SPAN><DIV id="nmcollapsiblew9-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To prepare cultures for use in the  β-galactosidase assay, this requires a small subsample of each overnight to be inoculated into fresh LB, two done for each colony. Then left to grow until the O.D of each culture is between 0.3-0.5. Once an O.D of 0.3-0.5 has been reached then 3 replicas of 1 ml samples are taken from each culture before being spun down, the supernatant then subsequently removed and the pellet frozen. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#bga-modal" class="journal-protocol">β-galactosidase assay  </A></P><P><B>Results: </B>N/A </P><P><B>Next Steps:</B> The next stage of experimentation will be to perform the second β-galactosidase assay on the interaction between the two subunits of <I>OBP2A</I></P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">18/8: Second attempt at determination of the protein interactions of the <I>OBP2A</I> subunits using the β-galactosidase assay  </SPAN><DIV id="nmcollapsiblew9-2" class="collapse box-content"><P><B>Aim of Experiment:</B> To perform a second attempt at the β-galactosidase assay that will aid in the determination of the level of interaction that the two separated subunits of <I>OBP2A</I> are inflicting on one another. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#bga-modal" class="journal-protocol">β-Galactosidase assay  </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure11-modal " class="journal-protocol"> Figure 10 </A></P><P><B>Next Steps:</B> The next stage of experimentation will be to switch the positioning of the two subunits of <I>OBP2A</I> in pUT18 and pT25 by having the smaller subunit in pUT18 and the large subunit in pT25. Along with this another β-galactosidase assay will be performed however this time it will be done under anaerobic conditions to see if that has any appreciable effect on the interaction of the two subunits of <I>OBP2A</I>.      </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title"> 20/8: PCR of <I>OBP2A</I> in preparation of pUT18 and pT25 insertion   </SPAN><DIV id="nmcollapsiblew9-3" class="collapse box-content"><P><B>Aim of Experiment:</B> To  Perform a PCR of <I>OBP2A</I> using primers for pU1T18 and pT25 insertion which have been designed to place the smaller <I>OBP2A</I>subunit into pUT18 and the larger subunit into pT25. </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results: </B><A data-toggle="modal" data-target="#lbpa-figure1-modal" class="journal-protocol"> N/A </A></P><P><B>Next Steps:</B> If PCR is successful then the next stage of experimentation will be to perform a gel extraction and restriction in perpetration for ligation and transformation of <I>OBP2A</I> into pUT18 and pT25.   </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">21/8: Second attempt at a PCR of <I>OBP2A</I> in preparation of pUT18 and pT25 insertion   </SPAN><DIV id="nmcollapsiblew9-4" class="collapse box-content"><P><B>Aim of Experiment:</B> The PCR performed on the 20/8 failed as no bands were found to be present on the gel during gel viewing. As a result of this a subsequent PCR was done today.      </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#PCR-modal" class="journal-protocol">PCR</A></P><P><B>Results: </B> N/A  </P><P><B>Next Steps:</B>  If PCR is successful then the next stage in experimentation will be to perform a gel extraction and restriction in perpetration for ligation and transformation of <I>OBP2A</I> in pUT18 and pT25.  </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">22/8: Second attempt at PCR of <I>OBP2A</I> for insertion into pUT18 and pT25 failed </SPAN><DIV id="nmcollapsiblew9-5" class="collapse box-content"><P><B>Aim of Experiment:</B> A second attempt at preparing <I>OBP2A</I> for insertion into the two vectors of the bacterial two hybrid system(pUT18 and pT25) has failed! As a result this will be left until next week, so that the rest of the week can be dedicated to preparing and undertaking  another β-galactosidase assay, that will be done under anaerobic conditions.  </P><P><B>Protocols Used:</B> N/A </P><P><B>Results: </B> N/A </P><P><B>Next Steps:</B> The next stage of experimentation is to prepare for a anaerobic β-galactosidase assay of <I>OBP2A</I> in the bacterial two hybrid vectors; pUT18 and pT25.  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-week"><DIV class="box"><DIV class="labtitle">Week Beginning 24/8/2015</DIV><DIV class="box-content"><P class="journal-summary-heading">Summary</P><P class="journal-content"> Performed an anaerobic β-galactosidase assay in order to determine the level of interaction between the two subunits of <I>OBP2A</I>.   </P></DIV><DIV id="nmcollapsiblew10" class="collapse week-content"><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title">24/8: Preparation of samples for the anaerobic β-galactosidase assay of <I>OBP2A</I> in pUT18 and pT25  </SPAN><DIV id="nmcollapsiblew10-1" class="collapse box-content"><P><B>Aim of Experiment:</B> To prepare the  samples and controls of <I>OBP2A</I> for the anaerobic β-galactosidase assay, so they can be frozen overnight in preparation for the assay itself.    </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#ap-modal" class="journal-protocol">Assay Preparation</A></P><P><B>Results: </B></P><P><B>Next Steps:</B> The next stage of experimentation will be to perform the β-galactosidase assay itself. 
               </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="border-day"><DIV class="box"><SPAN class="box-title"> 25/8: Performed an anaerobic β-galactosidase assay in order to determine the level of interaction of <I>OBP2A</I> in the bacterial two hybrid vectors: pUT18 and pT25.   </SPAN><DIV id="nmcollapsiblew10-2" class="collapse box-content"><P><B>Aim of Experiment:</B> Performed an anaerobic β-galactosidase assay of <I>OBP2A</I> in the bacterial two hybrid vectors: pUT18 and pT25. This is to determine the level of interaction that the smaller subunit of <I>OBP2A</I> has with its larger subunit in vivo. This particular assay has two positive controls, two negative controls and one regulatory control.  </P><P><B>Protocols Used:</B><A data-toggle="modal" data-target="#bga-modal" class="journal-protocol"> β-galactosidase assay </A></P><P><B>Results: </B><A data-toggle="modal" data-target="#nm-figure12-modal" class="journal-protocol"> Figure 12 </A></P><P><B>Next Steps:</B> The results have shown that the two subunits of <I>OBP2A</I> in fact don't seem to be interacting in vivo - a possible reason for this might be aggregation of the two subunits preventing interaction. The  next stage of experimentation will be to test out the interaction when the subunit are switch in relation to heir positions on the vectors: pUT18 and pT25.    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="sponsor"><DIV class="container"><DIV class="footer-btn"><A href="https://2015.igem.org/Team:Dundee/Introduction_Video" class="btn btn-info btn-lg" role="button">Watch Our Introduction Video</A></DIV></DIV></SECTION><FOOTER><DIV class=""><DIV class="container"><DIV class="col-md-3"><A href="http://www.dundee.ac.uk/" target="_blank"><H4>Dundee University</H4></A><P>Located on the east coast of Scotland, The University of Dundee is one of Scotland's leading research institutes.</P></DIV><DIV class="col-md-3"><H4>Social Media</H4><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="https://www.youtube.com/channel/UCJHBLvLhjzGqHqdjABjOY-w">youtube.com/DundeeiGEM2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.facebook.com/igemdundee2015">facebook.com/iGEMDundee2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.twitter.com/dundeeigemteam">twitter.com/DundeeiGEMTeam</A></DIV></DIV></DIV></DIV></DIV></FOOTER><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>