<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_Part_Collection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/Part Collection</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><H1><HIGHLIGHT class="highlight">The Building Blocks of Our Project</HIGHLIGHT></H1><H3><HIGHLIGHT class="highlight">Part Collection</HIGHLIGHT></H3><DIV class="ribbon"><DIV class="container"><DIV class="col-md-6"><H1>Part: BBa_K1590000  (<I>hHBA</I>) </H1><H2>FluID - Blood Detection</H2><P>A synthetic coding sequence for Human haemoglobin A. The protein forms part of a tetramer consisting of two alpha-chains and two beta-chains (see BBa_K1590001 for haemoglobin B). The sequence was codon optimized for expression in an <I>Escherichia coli</I> chassis.</P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary" href="http://parts.igem.org/Part:BBa_K1590000" target="_blank">Registry page for this part</A></DIV></DIV></DIV><DIV class="ribbon"><DIV class="container"><DIV class="col-md-6"><H1>Part: BBa_K1590001  (<I>hHBB</I>) </H1><H2>FluID - Blood Detection</H2><P> A synthetic coding sequence for Human haemoglobin B. The protein forms part of a tetramer consisting of two alpha-chains and two beta-chains (see BBa_K1590000 for haemoglobin A). The sequence was codon optimized for expression in an <I>Escherichia coli</I> chassis.</P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary" href="http://parts.igem.org/Part:BBa_K1590001" target="_blank">Registry page for this part</A></DIV></DIV><DIV class="col-md-6"><FIGURE align="center"><FIGCAPTION class="report-img"><P> Characterisation of haemoglobin B. A) Concentrated protein fractions eluted during the nickel affinity purification step were then further purified by size exclusion chromatography (SEC). B) 10 µl of each fraction corresponding to the two observed peaks were mixed with 10 µl of laemmli buffer and loaded onto a SDS gel (12.5% acrylamide) and stained with Coomassie Blue and also C) transferred to nitrocellulose membrane and probed with an anti-His antibody. The bands observable on both the stained gel and western blot were similar to the expected size of haemoglobin B - 16 kDa. </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="ribbon"><DIV class="container"><DIV class="col-md-6"><H1>Part: BBa_K1590002  (<I>hHBN</I>) </H1><H2>FluID - Blood Detection</H2><P> A synthetic coding sequence for Human haptoglobin. Haptoglobin is a human protein with high affinity for haemoglobin. This biobrick is a synthetic gene optimized for expression in <I>E. coli</I>. In blood plasma, haptoglobin binds free haemoglobin released from red blood cells.</P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary" href="http://parts.igem.org/Part:BBa_K1590002" target="_blank">Registry page for this part</A></DIV></DIV><DIV class="col-md-6"><FIGURE align="center"><FIGCAPTION class="report-img"><P>Further characterization of human haptoglobin following SEC. A) Chromatogram showing the elution profile of the His-tagged human haptoglobin. The fractions corresponding to the two peaks observed on the chromatograph were further analysed by SDS page gel and western blotted. B) 10 µl of the fractions A7-A8 and A8-A9 corresponding to peaks 1 + 2, respectively were mixed with 10 µl of Laemmli buffer and loaded onto a 12.5% SDS-PAGE gel. Band A observed on the gel corresponds to the expected size of haptoglobin – 45 kDa. However, it is not clear what the bands present at ~37 kDa may be, possibly haptoglobin that has lost its His-tag. C) Samples separated by SDS-PAGE were transferred to a nitrocellulose membrane and probed with an anti-his antibody. This Western blot analysis confirmed the presence of human haptoglobin. p&gt;
                    </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="ribbon"><DIV class="container"><DIV class="col-md-6"><H1>Part: BBa_K1590003  (P<I><SUB>chr</SUB></I>)</H1><H2> Chromate Detection </H2><P> This promoter is found upstream of the <I>chrBACF</I> - operon in <I>Ochrobactrum tritici</I> 5bvl1, located in the transposable element Tn<I>OtChr</I> of 7189 bp length. P<I>chr</I> is suspected to be inducible by chromate via the chromate-responsive repressor ChrB. </P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary" href="http://parts.igem.org/Part:BBa_K1590003" target="_blank">Registry page for this part</A></DIV></DIV><DIV class="col-md-6"><FIGURE align="center"><FIGCAPTION class="report-img"><P>Western analysis of GFP production driven by the chr promoter: Single colonies of JM110 + pSB1C3-P<I>chr</I>-<I>gfp</I> (A) and MC1061 + pSB1C3-P<I>chr</I>-<I>gfp</I> (B) were used to inoculate 5 ml of LB broth supplemented with 100 µg/ml chloramphenicol. After 16 h of incubation at 37°C with agitation at 200 rpm, each sample was subcultured into 5 ml of fresh, equally supplemented LB and cells were grown for 2 hours more. 1 ml of the subculture was then retrieved and pelleted. The pellet was resuspended in 1 ml TBS. 100 µl of the sample was mixed with 100 µl laemmli buffer, and boiled for 10 min. 3 µl of each sample was loaded on a SDS gel (12% acrylamide). pSB1C3 was included as a negative control, and P<I>manA</I>-<I>gfp</I> as a positive control. </P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="ribbon"><DIV class="container"><DIV class="col-md-6"><H1>Part: BBa_K1590004  (<I>ChrB</I>)</H1><H2>Chromate Detection</H2><P> The protein encoded by this sequence is a putative chromate-responsive repressor of P<I>chr</I> (BBa_K1590003). This sequence is found downstream of P<I>chr</I> in the <I>chrBACF</I> - operon in <I>Ochrobactrum tritici</I> 5bvl1, located in the transposable element Tn<I>OtChr</I>. ChrB is suspected to inhibit the otherwise constitutive promoter P<I>chr</I> in the absence of Cr(VI) by binding to an imperfect inverted repeat sequence located upstream of the initial ATG codon. Cr(VI) was expected to lift this repression, leading to the expression of the genes downstream of P<I>chr</I>. </P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary" href="http://parts.igem.org/Part:BBa_K1590004" target="_blank">Registry page for this part</A></DIV></DIV><DIV class="col-md-6"><FIGURE align="center"><FIGCAPTION class="report-img"><P>Comparison of presence western blot analysis of GFP in pSB1C3-P<I>chr</I>-<I>gfp</I> (A) + pUniprom-<I>chrB</I>, and pSB1C3-P<I>chr</I>-<I>gfp</I> (A) + pUniprom-<I>chrB</I> (opt).  It was found that GFP was produced in the absence of chromate for both systems. The reason for these unexpected results could not be discerned, and further experiments are required to understand those. At this stage of the project the results indicate that ChrB might not be a repressor.</P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="ribbon"><DIV class="container"><DIV class="col-md-6"><H1>Part: BBa_K1590006  (<I>LSS</I>)</H1><H2> Fingerprint Aging </H2><P> A synthetic coding sequence for Human Lanosterol Synthase. Lanosterol synthase (LSS) an oxidosqualene cyclase (OSC) enzyme that specifically binds to squalene epoxide (2,3- oxidosqualene), which is present in fingerprints. Our modelling showed us that squalene epoxide is the compound with the most distinct degradation pattern in fingerprints, and it was hence selected as an appropriate target for approximating the age of a fingerprint.</P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary" href="http://parts.igem.org/Part:BBa_K1590006" target="_blank">Registry page for this part</A></DIV></DIV><DIV class="col-md-6"><FIGURE align="center"><FIGCAPTION class="report-img"><P>Detection of His-tagged LSS in whole cells of <I>E. coli</I>. Single colonies of <I>E. coli</I> strain M15 pREP4 harbouring LSS. Cells were used to inoculate 5 ml of LB growth medium supplemented with 100 µg/ml ampicillin and 50 µg/ml Kanamycin. Once the OD<SUB>600</SUB> reached 0.7 the cells were then induced with IPTG, as indicated. Cells were then grown for a further 4 hours at 37°C, 1 ml aliquots were pelleted and cells reuspended in 100 µl laemmli buffer and 20 µl of samples were separated by SDS-PAGE (12% acrylamide) and transferred to nitrocellulose membrane and probed with anti-His antibody.</P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="ribbon"><DIV class="container"><DIV class="col-md-6"><H1>Part: BBa_K1590007  (<I>Obp2A</I>) </H1><H2>FluID - Nasal Mucus Detection</H2><P> A synthetic coding sequence for Human Odourant Binding Protein 2A. Human Odorant Binding Protein 2A is a 155 amino acid (excluding the signal peptide) lipocalin of relatively low molecular weight (19318 Daltons). Structurally it forms an 8 sheet beta barrel flanked by a C-terminal alpha helix that together forms an internal hydrophobic pore known as a calix. It is secreted by the olfactory epithelial cells of the nose where it lies in high abundance within nasal mucus. Its primary function in the human body is believed to be in the transport of hydrophobic odorant proteins across the otherwise impenetrable aqueous mucus layer to the olfactory receptors of the nose. Due to its high specificity and abundance within nasal mucus, OBP2A was selected as the protein for use in nasal mucus detection.</P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary" href="http://parts.igem.org/Part:BBa_K1590007" target="_blank">Registry page for this part</A></DIV></DIV><DIV class="col-md-6"><FIGURE align="center"><FIGCAPTION class="report-img"><P><B>Figure</B> This figure  illustrates the calculated miller’s activity of each control/sample and suggest that the OBP2A subunits aren’t interacting. This  can be gauged from the sample on the far right of each graph. They suggest that OBP2A has a lower interaction than the negative/regulatory controls.</P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="ribbon"><DIV class="container"><DIV class="col-md-6"><H1>Part: BBa_K1590008  (<I>LbpA</I>)</H1><H2>FluID - Saliva Detection</H2><P> Coding sequence for Lactoferrin Binding Protein A of <I>Neisseria Meningitidis</I>. This protein sequesters Iron for the host from Lactoferrin. A major component found in saliva is a free iron sequestering compound known as lactoferrin, a protein involved in the innate immune system. <I>Neisseria meningitidis</I> is a gram-negative bacterium with an iron-binding outer membrane protein called lactoferrin binding protein A (LbpA). <I>N. meningitidis</I> uses this LbpA to extract iron from the host lactoferrin under pathogenic conditions to allow for the bacterium to perform essential cellular metabolism such as energy generation and DNA replication. </P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary" href="http://parts.igem.org/Part:BBa_K1590008" target="_blank">Registry page for this part</A></DIV></DIV></DIV><DIV class="ribbon"><DIV class="container"><DIV class="col-md-6"><H1>Part: BBa_K1590009  (<I>PotD</I>)</H1><H2>FluID - Semen Detection</H2><P><I>Escherichia coli</I>PotD sequence, encoding Spermidine/putrescine-binding periplasmic protein. For semen detection our main target ligand is the polyamine spermidine which is found in relatively high concentrations in seminal fluid (5-15 mM). Spermidine is made from another polyamine called putrescine and is the precursor of spermine. Regulation of polyamine synthesis, degradation and transport is tightly controlled in bacteria. In <I>E. coli</I>, two of three identified transport systems are ABC transporters composed of a periplasmic binding protein, a pair of transmembrane proteins and a membrane protein possessing ATPase catalytic activity. Out of these three components, we were interested in investigating the periplasmic binding protein PotD which specifically binds spermidine. The fact that this protein is responsible for transportation of spermidine and lacking in enzymatic activity meant that it was an ideal candidate for use in FluID for semen detection </P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary" href="http://parts.igem.org/Part:BBa_K1590009" target="_blank">Registry page for this part</A></DIV></DIV><DIV class="col-md-6"><FIGURE align="center"><FIGCAPTION class="report-img"><P> Further characterization of PotD following SEC. A) Chromatogram showing the purification profile of His-tagged PotD. The fractions corresponding to the two peaks observed on the chromatograph were further analysed on SDS page gel and western blotted. B) 10 µl of the fractions A12 + B12 were mixed with with 10 µl of Laemmli buffer and loaded onto a 12.5% SDS-PAGE gel, stained with Coomassie Blue and also C) transferred to nitrocellulose membrane and probed with an anti-His antibody. The bands observable on both the stained gel and western blot were similar to the expected size of PotD - 37 kDa. 

</P></FIGCAPTION></FIGURE></DIV></DIV><DIV class="ribbon"><DIV class="container"><DIV class="col-md-6"><H1>Part: BBa_K1590010  (<I>Sbp</I>) </H1><H2>FluID - Semen Detection</H2><P> Synthetic coding sequence for Murine Spermine Binding Protein
For semen detection our main target ligand is the polyamine spermidine which is found in relatively high concentrations in seminal fluid (5-15 mM). Spermidine is made from another polyamine called putrescine and is the precursor of spermine. Regulation of polyamine synthesis, degradation and transport is tightly controlled in bacteria. In <I>E. coli</I>, two of three identified transport systems are ABC transporters composed of a periplasmic binding protein, a pair of transmembrane proteins and a membrane protein possessing ATPase catalytic activity. Out of these three components, we were interested in investigating the periplasmic binding protein PotD which specifically binds spermidine. However, we made initial attempts in characterising SBP as well, and submitted the biobrick accordingly. </P><DIV class="button-center"><A role="button" class="btn btn-lg btn-primary" href="http://parts.igem.org/Part:BBa_K1590010" target="_blank">Registry page for this part</A></DIV></DIV></DIV></DIV></DIV><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV><DIV class="container"><DIV class="footer-btn"><A href="https://2015.igem.org/Team:Dundee/Introduction_Video" class="btn btn-info btn-lg" role="button">Watch Our Introduction Video</A></DIV></DIV><FOOTER><DIV class=""><DIV class="container"><DIV class="col-md-3"><A href="http://www.dundee.ac.uk/" target="_blank"><H4>Dundee University</H4></A><P>Located on the east coast of Scotland, The University of Dundee is one of Scotland's leading research institutes.</P></DIV><DIV class="col-md-3"><H4>Social Media</H4><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="https://www.youtube.com/channel/UCJHBLvLhjzGqHqdjABjOY-w">youtube.com/DundeeiGEM2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.facebook.com/igemdundee2015">facebook.com/iGEMDundee2015</A></DIV></DIV><DIV class="row"><DIV class="col-md-10"><A target="_blank" href="http://www.twitter.com/dundeeigemteam">twitter.com/DundeeiGEMTeam</A></DIV></DIV></DIV></DIV></DIV></FOOTER><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>