<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dundee_protocolmodal skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dundee/protocolmodal</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="modal fade" id="plating-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Plating Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>Fresh agar plates were inoculated with cells on the toothpick.</LI><LI>Agar plates were incubated at 4<SUP>o</SUP>C over night.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sequencing-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Sequencing Protocol</DIV></DIV><DIV class="modal-body"><P>
    For sequencing a sample, the following ingredients were added to a microcentrifuge tube:
      <UL><LI>Volume of sample equivalent to 550ng DNA</LI><LI>2µl of 3mM primer</LI><LI>Remaining volume of water for a total volume of 30µl</LI></UL></P><P>
      The samples were submitted to <A href="https://www.dnaseq.co.uk/" target="_blank">DNA sequencing and services</A>.
    </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="overnightculture-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Overnight Culture Protocol</DIV></DIV><DIV class="modal-body"><P><OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>5ml of LB medium, supplemented with the required antibiotic, were inoculated with the toothpick.</LI><LI>Tubes were incubated at 37<SUP>o</SUP>C in a rotary incubator at 200rpm over night.</LI></OL></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="PCR-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">PCR Protocol</DIV></DIV><DIV class="modal-body"><P>
    PCR reaction mixtures were set up according to the table below. The length of all sequences used is stated in the <STRONG>Sequences</STRONG> section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY color="black"><TR><TD width="312" valign="top"><P align="center"><STRONG>Reactants</STRONG></P></TD><TD width="312" valign="top"><P align="center"><STRONG>Volume (µl)</STRONG></P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DNA
                </P></TD><TD width="312" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DMSO
                </P></TD><TD width="312" valign="top"><P align="center">
                    2.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Forward Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Reverse Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase Buffer
                </P></TD><TD width="312" valign="top"><P align="center">
                    10
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    dNTP
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Herculase
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    H<SUB>2</SUB>O
                </P></TD><TD width="312" valign="top"><P align="center">
                    34.5
                </P></TD></TR></TBODY></TABLE><P>
    **PCR Programme**
</P><P>
    Elongation time was altered according to the length of the sequences.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="colony-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Colony PCR Protocol</DIV></DIV><DIV class="modal-body"><P>
    The PCR reaction mixture were set up in multiples of the volumes in table below. The length of all sequences used is stated in the <STRONG>Sequences</STRONG> section.
</P><P>
    The DNA template is made in the following way:
    <OL><LI>Agar plates with required antibiotic were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>A numbered grid according to the desired number of PCR reactions is drawn on the back of the plate.</LI><LI>Cells or colonies were picked up from the source with a toothpick under sterile conditions.</LI><LI>Each colony is transferred to a 30µl aliquot of sterile water.</LI><LI>With the same toothpick the grid with the respctive number on the plate was inoculated.</LI><LI>The plate was incubated at 37<SUP>o</SUP>C overnight.</LI><LI>The inoculated water-aliquots were boiled for 10min, then centrifuged at 13300rpm for 10min.</LI><LI>The supernatant of this preparation is the DNA fraction for the subsequent PCR reaction.</LI></OL></P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY color="black"><TR><TD width="312" valign="top"><P align="center"><STRONG>Reactants</STRONG></P></TD><TD width="312" valign="top"><P align="center"><STRONG>Volume (µl)</STRONG></P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DNA
                </P></TD><TD width="312" valign="top"><P align="center">
                    5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    DMSO
                </P></TD><TD width="312" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Forward Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Reverse Primer
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.5
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    +Mg Buffer
                </P></TD><TD width="312" valign="top"><P align="center">
                    2
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    dNTP
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.2
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    Taq-polymerase
                </P></TD><TD width="312" valign="top"><P align="center">
                    0.2
                </P></TD></TR><TR><TD width="312" valign="top"><P align="center">
                    H<SUB>2</SUB>O
                </P></TD><TD width="312" valign="top"><P align="center">
                    10.6
                </P></TD></TR></TBODY></TABLE><P>15µl of the reaction mixture was aliquoted into separate PCR tubes. 5µl of DNA fraction was added to each respective tube in order to have a total reaction volume of 20µl per tube.</P><P>
    **PCR Programme**
</P><P>
    Elongation time was altered according to the length of the sequences.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="miniprep-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Miniprep Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>5ml bacterial overnight culture was pelleted by stepwise centrifugation in 1.5ml microcentrifuge tubes at 13300rpm for 3 minutes at room temperature.</LI><LI>Pelleted bacterial cells were resuspended in 250µl of Buffer P1.</LI><LI>250µl of Buffer P2 was added and mixed thoroughly by inverting the tube 4-6 times until solution became clear.</LI><LI>350µl of Buffer N3 was added and immediately and thoroughly mixed by inverting the tube 4-6 times.</LI><LI>The mixture was centrifuged for 10 minutes at 13300rpm in a microcentrifuge.</LI><LI>The supernatant was applied to a QIAprep spin column by pipetting.</LI><LI>500µl of Buffer PB was added to the QIAprep spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>750µl of Buffer PE was added to the QIAprep spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>The QIAprep spin column was spun for an additional minute to remove residual wash buffer.</LI><LI>The QIAprep spin column was placed in a clean 1.5ml microcentrifuge and the DNA eluted by adding 30µl of H<SUB>2</SUB>O to the QIAprep spin column and centrifuging for 1 minute. 

</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="gelextraction-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Gel Extraction Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>The desired DNA fragment was excised from the agarose gel with a clean, sharp scalpel.</LI><LI>The gel slice was transferred into a microcentrifuge tube. 700µl of Buffer QG was added.</LI><LI>The gel slice was incubated in a water bath at 50°C for 10 min (or until the gel slice has completely dissolved). The tube was vortexed every 2–3 min to help dissolve gel.</LI><LI>After the gel slice had been dissolved completely, the color of the mixture was checked that it was yellow (similar to Buffer QG without dissolved agarose). If the color of the 

mixture was orange or violet, 10 μl 3 M sodium acetate, pH 5.0 would be added, and mixed. The color of the mixture would turn yellow.</LI><LI>230µl of isopropanol was added to the sample and mixed.</LI><LI>A QIAquick spin column was placed in a 2ml collection tube. The sample was applied to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>500µl of QG buffer was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>750µl of Buffer PE was added to the QIAquick spin column and centrifuged for 1 minute. The flow-through was discarded.</LI><LI>The QIAprep spin column was spun for an additional minute to remove residual wash buffer.</LI><LI>The QIAprep spin column was placed in a clean 1.5ml microcentrifuge and the DNA eluted by adding 30µl of H2O to the QIAprep spin column and centrifuging for 1 minute.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="restriction-digest-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Restriction Digest Protocol</DIV></DIV><DIV class="modal-body"><P>
Restriction digests were set up according to the table below. The appropriate reaction and restriction enzymes that were used is stated in the    <STRONG>Journal</STRONG> section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="170" valign="top"><P><STRONG>Post Plasmid Purification</STRONG></P></TD><TD width="151" valign="top"><P><STRONG>Post PCR</STRONG></P></TD><TD width="139" valign="top"><P><STRONG>gBlock</STRONG></P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>DNA</STRONG></P></TD><TD width="170" valign="top"><P>
                    1mg
                </P></TD><TD width="151" valign="top"><P>
                    50µl
                </P></TD><TD width="139" valign="top"><P>       100ng
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Cutsmart Buffer (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    6
                </P></TD><TD width="151" valign="top"><P>
                    6
                </P></TD><TD width="139" valign="top"><P>
                    6
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Water (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    As required
                </P></TD><TD width="151" valign="top"><P>
                    1
                </P></TD><TD width="139" valign="top"><P>
                    As required
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Enzyme 1 (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    1.5
                </P></TD><TD width="151" valign="top"><P>
                    1.5
                </P></TD><TD width="139" valign="top"><P>
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG>Enzyme 2 (µl)</STRONG></P></TD><TD width="170" valign="top"><P>
                    1.5
                </P></TD><TD width="151" valign="top"><P>
                    1.5
                </P></TD><TD width="139" valign="top"><P>
                    1.5
                </P></TD></TR><TR><TD width="164" valign="top"><P><STRONG><U>Total Volume (µl)</U></STRONG></P></TD><TD width="170" valign="top"><P>
                    30
                </P></TD><TD width="151" valign="top"><P>
                    60
                </P></TD><TD width="139" valign="top"><P>
                    30
                </P></TD></TR></TBODY></TABLE><P><UL><LI>The digests were incubated at 37<SUP>o</SUP>C for 3 hours.</LI><LI>Note that when performing plasmid digestions, 2.5µl of alkaline phosphatase and 2.5µl of its buffer were added at the 2 hour and 2.5 hour mark.</LI></UL></P></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="ligation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Ligation Protocol</DIV></DIV><DIV class="modal-body"><P>
Ligation reactions were set up according to the table below. The vector to insert ratio is stated for each specific reaction in the    <STRONG>Journal</STRONG> section.
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="111" valign="top"><P align="center"><STRONG>Vector (µl)</STRONG></P></TD><TD width="111" valign="top"><P align="center"><STRONG>2:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>3:1 (Vector: Insert) [µl]</STRONG></P></TD><TD width="129" valign="top"><P align="center"><STRONG>4:1 (Vector: Insert) [µl]</STRONG></P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>T4 DNA Ligase </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Ligase Buffer </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Vector </STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="111" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD><TD width="129" valign="top"><P align="center">
                    1
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>Insert</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    -
                </P></TD><TD width="111" valign="top"><P align="center">
                    2
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD></TR><TR><TD width="140" valign="top"><P><STRONG>H<SUB>2</SUB>0</STRONG></P></TD><TD width="111" valign="top"><P align="center">
                    7
                </P></TD><TD width="111" valign="top"><P align="center">
                    5
                </P></TD><TD width="129" valign="top"><P align="center">
                    4
                </P></TD><TD width="129" valign="top"><P align="center">
                    3
                </P></TD></TR></TBODY></TABLE><P>
    The reactions were incubated at room temperature for at least 3 hours.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="transformation-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Transformation Protocol</DIV></DIV><DIV class="modal-body"><OL><LI>100µl of competent cells were defrosted. Thaw on ice.</LI><LI>Agar plates were taken out of 4<SUP>o</SUP>C to warm to room temperature.</LI><LI>1-5µl of DNA was added to cells.</LI><LI>Cell and DNA mixture was left on ice for 20 minutes.</LI><LI>Mixture was placed in 42<SUP>o</SUP>C water bath for 90 seconds.</LI><LI>Mixture was put back on ice for 2 minutes.</LI><LI>1ml of LB (without antibiotic) was added to cells.</LI><LI>Cells were allowed to grow for 1 hour at 37<SUP>o</SUP>C in a shaking incubator.</LI><LI>Cells were then spun down and the supernatant discarded.</LI><LI>The pellet was resuspended in the 100µl.</LI><LI>Cells were plated on agar plates with appropriate antibiotic.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="sds-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel">Protein Expression Optimization Protocol</DIV></DIV><DIV class="modal-body"><P><STRONG><U>SDS-PAGE and Western Blotting </U></STRONG></P><OL><LI>50µl of an overnight culture was used to inoculate 5ml of fresh LB containing the appropriate antibiotics and left to grow at 37<SUP>o</SUP>C in a shaking incubator.</LI><LI>Once the OD<SUB>600</SUB>=0.6-0.8, 1mM of IPTG was added to the culture and left to grow for a further 3 hours.</LI><LI>1ml of each culture was taken and spun down in a microcentrifuge for 3 minutes.</LI><LI>Meanwhile, 950µl of 2x Laemmli sample buffer was added to 50µl of B-mercaptoethanol.</LI><LI>The supernatant was discarded from the samples and the pellet resuspended in 200µl of the above solution.</LI><LI>The samples were then boiled at 90<SUP>o</SUP>C for 10 minutes.</LI><LI>Afterwards, they were spun down for 1 minute in a microcentrifuge.</LI><LI>At this stage, the samples could be stored at -20<SUP>o</SUP>C or ran on a gel immediately.</LI><LI>Two gradient gels consisting of a 12% separating gel and a stacking gel were prepared and the samples ran at 100V until proteins reach end of the stacking gel at which point the 

voltage was increased to 200V for 20 minutes.</LI><LI>One gel was stained using Instant Blue and the other was used to transfer the proteins onto a membrane.</LI><LI>After the proteins had been transferred onto the membrane, the membrane was blocked in 5% Marvel milk overnight at 4<SUP>o</SUP>C.</LI><LI>The next day, the membrane was washed 3 times in 1xTBST buffer and then suspended in 10ml of 1xTBS buffer to which the primary anti-His antibody was added and left for one hour. 

The protein ladder was cut off before adding the primary antibody.</LI><LI>The membrane was then washed 3 times in 1xTBST buffer for 5 minutes and then suspended in 10ml of 1xTBS buffer, to which the secondary antibody was added and left for one 

hour.</LI><LI>The membrane was then washed in 3x10ml 1xTBST buffer for 5 minutes.</LI><LI>The blot was then soaked in developing solution for 5 minutes before being wrapped in a plastic cover aligned with the protein ladder that was cut off at step 12.</LI><LI>Once in the developing room, the film was exposed to the blot for 30 seconds, 2 minutes and 5 minutes and then placed in the film developer.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV class="modal fade" id="pp-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"><DIV class="modal-dialog" role="document"><DIV class="modal-content"><DIV class="modal-header"><BUTTON type="button" class="close" data-dismiss="modal" aria-label="close" aria-hidden="true" role="dialog" tabindex="-1"><SPAN aria-hidden="true">×</SPAN></BUTTON><DIV class="modal-title" id="myModalLabel"><CENTER><B>Protein Purification Protocol</B></CENTER></DIV></DIV><DIV class="modal-body"><OL><LI>A 150ml overnight culture was set up for the desired protein.</LI><LI> 3 x 1L of fresh LB containing 1ml of appropriate antibiotics was inoculated with each 50ml overnight culture and left to grow until an OD<SUB>600</SUB>
      =0.6-1 in a shaking incubator at 37<SUP>o</SUP>C.</LI><LI>The cultures were subsequently induced with 1mM IPTG and left to grow for 6 hours in a shaking incubator at 20<SUP>o</SUP>C.</LI><LI>They were then centrifuged for 20 minutes at 4200rpm at 4<SUP>o</SUP>C.</LI><LI>The pellets were then resuspended in 5ml of ice cold 50mM 7.5 Tris and transferred to a 50ml falcon tube and then topped up to 40ml with 50mM 7.5 Tris.</LI><LI>They were then spun down again at 4000rpm for 15 minutes at 4<SUP>o</SUP>C, the supernatant discarded and pellets stored at -20<SUP>o</SUP>C.</LI><LI>7. Once purification could proceed, the pellets were thawed on ice and resuspended in 40ml of 50mM Tris containing DNAase, 1 x protease inhibitor tablet
      and lysozyme. (Note: If purifying a membrane protein, 1% DDM was also added).</LI><LI>The cells were lysed using a cell disruptor.</LI><LI>Afterwards the lysate was spun down in a J25.50 rotor at 15000rpm at 4<SUP>o</SUP>C for 40 minutes.</LI><LI>The supernatant was removed and transferred to a 50ml falcon tube. The pellet was kept for testing. Both were kept on ice.</LI><LI>Using FPLC, the supernatant was loaded onto a Ni<SUP>2+</SUP> column.</LI><LI>The protein was eluted using an increasing concentration of imidazole. The two buffers used were Buffer A: 50mM Tris, 200mM NaCl, 20mM imidazole and
      Buffer B: 50mM Tris, 200mM NaCl, 1M Imidazole. (Note: If purifying a membrane protein 0.02% DDM was added to these buffers.)</LI><LI>The fractions corresponding to any peaks were retained and concentrated using an appropriate filter size. A small amount of each fraction could be ran
      on an SDS gel at this stage.</LI><LI>Using FPLC, the sample was then loaded onto a SEC column and any fractions corresponding to a peak were ran on an SDS gel to check for the presence of
      the desired protein.</LI><LI>Fractions containing the desired protein were collated, to which 20% glycerol was added.</LI><LI>The samples were then flash frozen and stored at -80<SUP>o</SUP>C.</LI></OL></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-primary" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><DIV id="nav"><NAV class="navbar navbar-default navbar-fixed-top navbar-collapse" role="navigation"><DIV class="container"><DIV class="navbar-header"><A href="http://www.dundee.ac.uk/" target="_blank"><SPAN class="navbar-brand">  University of Dundee</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:Dundee"> Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> About Us</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Project</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit">Project Overview</A></LI><LI class="dropdown-header">Devices</LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Forensic_Toolkit/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Part_Collection">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Medals">Medals</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Future">Future Work</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Wet Lab</A></LI><LI class="dropdown-header">Lab Book</LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Fingerprints">Fingerprints</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Lab_Book/Chromate_Detector">Chromate Biosensor</A></LI><LI class="dropdown-header">Useful Information</LI><LI><A href="https://2015.igem.org/Team:Dundee/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Sequences">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Bacterial_Two_Hybrid">Bacterial Two Hybrid</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/FluID">FluID</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Fingerprints">Fingerprint Ageing</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Chromate">Chromate Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Modeling/Appendix1">Appendix : Code</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown"> Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Dundee/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></BODY></HTML>