<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Beijing_Normal_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Beijing_Normal/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Beijing_Normal/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Beijing_Normal/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Beijing_Normal/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Beijing_Normal/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Beijing Normal/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="color:#ffffff;background-color:#6699cc;" cellpadding="3" cellspacing="1" border="0" align="center" width="80%"><TBODY><TR><TH align="center"><A href="/Team:Beijing_Normal" title="Team:Beijing Normal">Home</A></TH><TH align="center"><A href="/Team:Beijing_Normal/Team" title="Team:Beijing Normal/Team">The Team</A></TH><TH align="center"><STRONG class="selflink">The Project</STRONG></TH><TH align="center"><A href="/Team:Beijing_Normal/Parts" title="Team:Beijing Normal/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:Beijing_Normal/Notebook" title="Team:Beijing Normal/Notebook">Notebook</A></TH></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overall"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overall</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Project_Details"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Details</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#background_information"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">background information</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Design_Abstract"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Design Abstract</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#The_Experiments"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">The Experiments</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Special_protocol"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Special protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Simplified_Hot_Start_PCR"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Simplified Hot Start PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Explanation"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">Explanation</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Effective_Additives"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Effective Additives</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Explanation_2"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Explanation</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#References_for_special_protocol:"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">References for special protocol:</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Design_Details"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Design Details</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#1._Switch_on_the_degradation_pathway"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">1. Switch on the degradation pathway</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#2._Solve_the_bottleneck_in_PCBs_degradation_pathway"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">2. Solve the bottleneck in PCBs degradation pathway</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#3._Control_on_the_amount_of_PCBs_molecular_that_enter_the_cell."><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">3. Control on the amount of PCBs molecular that enter the cell.</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#4._System_guarantee"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">4. System guarantee</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#References_for_Design_Details:"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">References for Design Details:</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Results"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#1._Protocols"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">1. Protocols</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#2._Plasmids_constructed"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">2. Plasmids constructed</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#3._A_functional_part"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">3. A functional part</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Acknowledgments"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Acknowledgments</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overall"><B>Overall</B></SPAN></H2><P>We are aiming to create some magic intelligent bacteria to track and ‘eat’ pollutants PCBs  (Polychlorinated Biphenyl) and dioxins efficiently, based on the methods of synthetic biology.
</P><P>Polychlorinated biphenyls (PCBs) are a family of compounds produced commercially by the direct chlorination of biphenyl using ferric chloride and/or iodine as the ctalyst.The total amount of PCBs produced in the world is estimated 1.2 million tons.Because PCBs have been released into the environment in many countries over decades, these compounds have become serious and global environmental contaminants.PCBs tend to accumulate in biota owing to their lypophilic property.
</P><P>The biphenyl molecule is made up of two connected rings of six carbon atoms each, and a PCB is any molecule having multiple chlorines attached to the biphenyl nucleus.
</P><P>Two distinct classes of bacteria have now been identified that biodegrade PCBs by different mechanisms, including aerobic bacteria which live in oxygenated environments and anaerobic bacteria which live in oxygen free environments such as aquatic sediments. The aerobes attack PCBs oxidatively , breaking open the carbon ring and destroying the compounds. Anaerobes, on the other hand, leave the biphenyl rings intact while removing the chlorines.
</P><P>Polychlorinated dibenzo-p-dioxins (PCDD) and polychlorinated dibenzofurans (PCDF) were introduced into the biosphere on a large scale as by-products from the manufacture of chlorinated phenols and the incineration of wastes. Due to their high toxicity they have
been the subject of great public and scientific scrutiny.
</P><P>The evidence in the literature suggests that PCDD/F compounds are subject to biodegradation in the environment as part of the natural chlorine cycle. Lower chlorinated dioxins can be degraded by aerobic bacteria from the genera of Sphingomonas, Pseudomonas and Burkholderia.However, higher chlorinated dioxins requires anaerobic degradation process.
</P><P>Organic pollutants such as PCB and dioxins, produced in human beings activities in the last century, are toxic and carcinogenic which are able to promulgate widely and accumulate to a high level of concentration by food chain. Due to their inherent thermal and chemical stability, it is commonly considered as indestructible under normal incineration or burial.
</P><P>Nonetheless, by endowing some bacteria ability of utilizing such molecules as carbon source, cooperative evolution makes all possible! Enzymes assembled from related degradation pathways into our host strain serve as the function part. We introduce popular components involved in chemotaxis, quorum-sensing to regulatory parts, sense the environment signal, respond to move, accelerate growing and produce related degradation enzymes. After the cleaning work being finished, bacteria will return to the normal state.
</P><P>Taking the condition of our lab into account, we decide just deal with the aerobic degradation path way. And do some work on increasing the degradation effeciency.
</P><H2><SPAN class="mw-headline" id="Project_Details"> Project Details</SPAN></H2><H3><SPAN class="mw-headline" id="background_information"> background information </SPAN></H3><UL><LI>Although many environmental pollutants are efficiently degraded by microorganisms, others such as PCBs and dioxins persist and constitute a severe health hazard. In some instances, persistence is a consequence of the inadequate catabolic potential of the available microorganisms. Gene technology, combined with a solid knowledge of catabolic pathways and microbial physiology, enables the experimental evolution of new or improved catabolic activities for such pollutants.
</LI></UL><UL><LI>Catabolic pathway for degradation of biphenyl and organization of bph gene cluster is as follows:
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Source: Kensuke Furukawa and Hidehiko Fujihara, <B>Microbial Degradation of Polychlorinated Biphenyls: Biochemical and Molecular Features</B>, <I>Journal of bioscience and bioengineering</I>, 105:433–449 (2008), permitted to use by Kensuke Furukawa</DIV></DIV></DIV></DIV><P><B>Enzymes responsible for oxidative degradation of PCBs</B></P><P>1 BphA: biphenyl dioxygenase
</P><P>Biphenyl dioxygenase is a Riesketype three-component enzyme, comprising a terminal dioxygenase that is composed of a large subunit (encoded by bphA1) and a small subunit (encoded by bphA2), a ferredoxin (encoded by bphA3) and a ferredoxin reductase (encodedby bphA4)
</P><P>It catalyzed the conversion of biphenyl to dihydrodiol compound in step 1.
</P><P>2 BphB: dihydriol dehydrogenase
</P><P>In the second step of the upper pathway BphB enzymes catalyzes the conversion of dihidrodiol to dihydroxy compound.
</P><P>3 BphC: 2,3-dihydroxybiphnyl dioxygenase
</P><P>The third enzyme in upper pathway is a 2,3-dihyroxybiphnyl dioxygenase involved in the ring meta-cleavage at the 1,2 position.
</P><P>4 BphD: hydrolase 
</P><P>In the forth step BphD, a hydrolase, hydrolyzes the ring meta-cleavage yellow compound(HOPDA) to chlorobenzoic acid and 2-hydroxypenta-2,4-dienoate.
</P><H3><SPAN class="mw-headline" id="Design_Abstract"> Design Abstract </SPAN></H3><P>Polychlorinated biphenyls (PCBs) are a group of organic pollutants that are persistent when released into the environment. Our task is to design an effective as well as intelligent PCBs degrader. Thus our work could be divided into two parts. One is to develop a sensing system for the detection of PCBs and activation of the downstream degradation pathway; the other is to enhance the degradation efficiency.
</P><P>DHBD(2,3-dihydroxybiphenyl 1,2-dioxygenase), the third enzyme of the bph pathway, is of particular significance because it is incapable o f catalyzing the ring cleavage step and is subject to various types of inhabitation, as well as suicide inactivation. According to the recent research, ortho-chlorinated PCB metabolites (DHBs) are potent and physiologically significant inhibitors of DHBD, so we design a feedback activation pathway to increase the BphC transcription and expression under a 2, 3-DHBP and 4-CB inducible promoter Ppcb. As 2, 3-DHBP is the second metabolite in the pathway, 2, 3-DHBP at a concentration more than 0.1mM will induce the activate Ppcb and thus increase gene expression downstream. 
</P><P>As dihydrodiols and dihydroxybiphenyls are very toxic to bacterial even after short incubation time, we design a feedback repression pathway use sRNA components— sodB and rhyB. Part of sodB was fusion with bphA and biosurfactant pathway and will be repressed by rhyB which is under the control the promoter Ppcb. As only when rhyB molecules is much more than sodB fusion mRNA, it will play a role of repression, only the high concentration of DHBP will arise this reaction. In this case, the amount of these two metabolites will be reduced in two ways.
</P><P>As to the sensor part, dihydroxylated PCBs are substrate of the clcA-encoded chlorocatechol dioxygenase and thus induce the clcR and related promoter, so we use this as the sensing system. And T7 amplification system is add downstream to amplify the signal.
</P><H3><SPAN class="mw-headline" id="The_Experiments"> The Experiments </SPAN></H3><UL><LI>1.Get the parts from biobrick
</LI></UL><P>We largely follow the instructions provided by the webpage, however, more TE buffer is added(10ul). It seems that this will increase the amount of the plasmids dissolved and improve transformation effeciency.
</P><UL><LI>2 PCR
</LI><LI> 2.1 The pfu/Taq complex system
</LI></UL><PRE>Reagent	           Concentration/Activity	       50ul in total	
  taq buffer     	    10x	                         5      
  pfu/taq complex		                        0.8~1.0	
  dNTPmix	            10mM each	                 4	  
  Primer 1 	            10uM	                 1.5	  
  Primer 2	            10um	                 1.5	  
  Template DNA	         changeable	                 --	  
  ddH2O	                     ---                     add to 50ul	                
</PRE><UL><LI> 2.2 The program under pfu/Tag complex system
</LI></UL><P>Progress	     Program I	                 
</P><PRE>  Predenaturing     95℃        5 min	         
  Denaturing	    95℃        30sec	 
  Annealing	    (Tm-5) ℃   30sec	         
  Extension	    72℃        theoretically 1min/1kb
  Last extention    72℃        5min
  Hold               4℃
</PRE><UL><LI>3 restriction enzyme digestion
</LI></UL><P>select suitable enzymes and buffer. 
Analyse the system using NEB cutter in case of double digestion. 
<A href="http://www.neb.com/nebecomm/DoubleDigestCalculator.asp" class="external text" rel="nofollow">NEB cutter finder</A>.
37℃ water bath for 2~3h, 4h is preferable
</P><UL><LI>4 ligation
</LI></UL><P>The key to a successful ligation according to our experience is avoiding high temperature.
After a gel extraction with 32ul elution buffer, a approximately 20ul product is obtained. Then mixed with 20ul Buffer 1(Takara Ligation Kit). Reaction at 16℃  water bath is widely recommended, and time for ligation we use is 2h or more. 
</P><UL><LI>5 transformation
</LI></UL><P>Add 10ul ligation product into 100ul competence cells(Top10 or others) and keep it in 4℃ refrigeratory for 30min. After a hotshock of 45sec(for chemical competent) or 90sec(for CaCl2 competent), the cells are placed in the 4℃ refrigeratory again for 3-5min. Then add 600~800ul SOC to hotshock cells and incubate in 37℃ shaking table for 1h(160rpm-180rpm). At last the cells are palced on petri dishes with relative antibiotics at appropriate concentration. Then place in 37℃(temperature accords to the properties of different hosts and plasmids),incubate for 12h or more.
</P><H2><SPAN class="mw-headline" id="Special_protocol"> Special protocol </SPAN></H2><P>To obtain some genes, we often have to use bacteria chromosome or large size plasmid as template in which high GC% content and complex secondary structures are seriously hampering PCR and thus leading to complete failure. To solve this problem, we have developed a special protocol-- hot start method combined with additive(sole or mixed)-- which is most helpful . 
</P><H3><SPAN class="mw-headline" id="Simplified_Hot_Start_PCR"> Simplified Hot Start PCR </SPAN></H3><P>Hot start method is to prevent primer dimer and low specified product yield. In our experiment, We use common taq polymerase to replace commercial high-priced hot-start polymerase.  
</P><P>After a 5min 95℃ pre-heat step, DNA polymerase is  added to each PCR tube before the PCR cycling begin. After that, the regular PCR cycling could begin.
</P><H4><SPAN class="mw-headline" id="Explanation"> Explanation </SPAN></H4><P>When reaction components are mixed at room temperature, reaction set up below the optimal primer annealing temperature, which permits nonspecific primer annealing and extention.Undesired, non-specific primer extetion products formed this way may be amplified in the PCR, resulting in misprimed products and primer ologomers.In hot start PCR, DNA polymerase is withheld from the mixture until the system has reached a temperature that favours specific primer annealing. As a result hot start PCR can greatly improve specificity, sensitivity and yield in a PCR.
</P><H3><SPAN class="mw-headline" id="Effective_Additives"> Effective Additives </SPAN></H3><P>The most simple additive in our PCR is DMSO(&gt;=99.9% purity) at a concentration of 5%(V/V). It works well in most 'problematic' PCR, however, when DMSO fails in some case, we have to turn to a magic mixed additive (2.7 M betaine, 6.7 mM DTT, 6.7% DMSO, and 55 ug/ml BSA) for help. This excellent additive has solved several PCR where even 5% DMSO fails. 
</P><P>In certain PCR there is no yield without this additives, for example the ones by which we amplify bphA1A2A3A4 and bphBC.
</P><H4><SPAN class="mw-headline" id="Explanation_2"> Explanation </SPAN></H4><P>One major factor limiting the output of PCR routines is that a number of DNA sequences are poorly or not amplifiable under standard reaction conditions, either because of their high GC-content or/and their instrict propertoties to form secondary structures. This protocol is intended for GC-rich DNA sequences. This is a concentration dependent combination of betaine, dithiothreitol, and dimethyl suloxide. According to the references the concentration ratio is: a mixture containing 2.7 M betaine, 6.7 mM DTT, 6.7% DMSO, and 55 ug/ml BSA. It is stable at -20℃ for at least 3 months.
</P><H3><SPAN class="mw-headline" id="References_for_special_protocol:"> References for special protocol: </SPAN></H3><P>1. David E. Birch et al., <B>Simplified hot start PCR</B>, <I>Nature</I> 381:445-446 (1996)
</P><P>2. S.Kaijalainen et al., <B>An alternative hot start technique for PCR in small volumes using beads of wax-embedded reaction components dried in trehalose</B>, <I>Nucleic Acids Research</I> 21:2959-2960 (1993)
</P><P>3. Yukihiro Kitade et al., <B>Effect of DMSO on PCR of Porphyra yezoensis(Rhodophyta) gene</B>, <I>Journal of Applied Phycology</I> 15:555-557 (2003)
</P><P>4. Markus Ralser et al., <B>An efficient and economic enhancer mix for PCR</B>, <I>Biochemical and Biophysical Research Communications</I> 347:747–751 (2006)
</P><H2><SPAN class="mw-headline" id="Design_Details"> Design Details </SPAN></H2><P>Enhance the degradation efficiency &amp; detect PCBs
</P><H3><SPAN class="mw-headline" id="1._Switch_on_the_degradation_pathway">1. Switch on the degradation pathway</SPAN></H3><P>The system is awakened by PCBs.
</P><P>
We add bphR1 upstream the pathway and bphR2 downstream. bphR2 could sense the presence of PCBs and bphR1 could sense HOPDA, the third step product. These two regulators could switch on the degradation pathway.   
</P><P>In the absence of PCBs, small amounts of BphR2 protein bind to the bphR2 operator to re press bphR2 transcription (auto-repression). Under this condition, the levels of transcription of bph genes are very low. In the presence of PCBs, the BphR2 protein binds to the operators of bphR1 and bphA1A2A3A4BC and activates their transcription; thus high levels of HOPDA (the ring meta-cleavage compound) are produced from PCBs. In the presence of HOPDA, the transcription of bphR1 is further promoted, and its product BphR1 activates the transcription of bphR1 itself and downstream pathway [1,2].
</P><H3><SPAN class="mw-headline" id="2._Solve_the_bottleneck_in_PCBs_degradation_pathway">2. Solve the bottleneck in PCBs degradation pathway</SPAN></H3><P>Our idea: Add a spur track to solve the bottleneck of the pump.
</P><P>BphC (2,3-dihydroxybiphenyl 1,2-dioxygenase), the third enzyme of the bph pathway, is of particular significance because it is incapable o f catalyzing the ring cleavage step and is subject to various types of inhabitation, as well as suicide inactivation. According to the recent research, ortho-chlorinated PCB metabolites (DHBs) are potent and physiologically significant inhibitors of BphC, so we design a feedback activation pathway using small RNA to increase the BphC transcription and expression under a 2, 3-DHBP and 4-CB inducible promoter Ppcb [3]. As 2, 3-DHBP is the second metabolite in the pathway, 2, 3-DHBP at a concentration more than 0.1mM will induce the activate PpcbC and thus increase gene expression downstream. The activated threshold of PpcbC is relatively high, so this pathway will be activated when main PCBs degradation above has began. 
</P><H3><SPAN class="mw-headline" id="3._Control_on_the_amount_of_PCBs_molecular_that_enter_the_cell.">3. Control on the amount of PCBs molecular that enter the cell.</SPAN></H3><P>As dihydrodiols (first step product) and dihydroxybiphenyls (DHBP, second step product) are very toxic to bacterial even after short incubation time, we design a feedback repression pathway use sRNA components— sodB and rhyB. Part of sodB was fusion with bphA and biosurfactant pathway and will be repressed by rhyB[4] which is under the control the promoter Ppcb. As only when rhyB molecules is much more than sodB fusion mRNA, it will play a role of repression, only the high concentration of DHBP will arise this reaction. This design will control the amount of PCBs molecular that enter through the membrane. 
</P><H3><SPAN class="mw-headline" id="4._System_guarantee">4. System guarantee</SPAN></H3><P>In our design T7 amplification system is used to amplify the signal. T7 polymerase is added downstream the main degradation pathway. When T7 polymerase is translated, it will activate the T7 promoter and downstream reporter gene, YFP. Then the signal could be detected. This system could be used to test the function of the degradation pathway.
</P><H3><SPAN class="mw-headline" id="References_for_Design_Details:">References for Design Details:</SPAN></H3><P>[1] Kensuke Furukawa, et al., <B>Microbial Degradation of Polychlorinated Biphenyls: Biochemical and Molecular Features</B>, <I>Journal of Bioscience and Bioengineering</I> 105:433–449 (2008)
</P><P>[2] Stephen Y. K. Seah, et al., <B>Comparative specificities of two evolutionarily divergent hydrolases involved in microbial degradation of polychlorinated biphenyls</B>, <I>Journal of Bacteriology</I> 183:1511–1516 (2001)
</P><P>[3] Sang-ho Park, et al., <B>Construction of transformant reporters carrying fused genes using pcbC promoter of Pseudomonas sp DJ-12 for detection of aromatic pollutants</B>, <I>Environmental Monitoring and Assessment</I> 92:241–251 (2004)
</P><P>[4] Qinhong Wang, et al., <B>Engineering Bacteria for Production of Rhamnolipid as an Agent for Enhanced Oil Recovery</B>, <I>Biotechnology and Bioengineering</I>, 98:842-853 (2007)
</P><P>[5] Jessika Feliciano, et al., <B>ClcR-based biosensing system in the detection of cis-dihydroxylated (chloro-)biphenyls</B>, <I>Analytical and Bioanalytical Chemistry</I>, 385: 807–813 (2006)
</P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><H3><SPAN class="mw-headline" id="1._Protocols">1. Protocols</SPAN></H3><P>We have made great efforts on refining experiments protocols and shared several protocols that proved to be effecient. Our results show that you will have successful experiments if these protocols were used.
</P><P>We have shared experience with several other teams, and comunicated a lot.
</P><H3><SPAN class="mw-headline" id="2._Plasmids_constructed">2. Plasmids constructed</SPAN></H3><P>We have submitted 6 standard parts including dxnA, dbfB, redA2,dxnB, bphB, bphC, bphD. These are genes coding critical enzymens in the degradation pathway of dioxin and PCBs.
</P><H3><SPAN class="mw-headline" id="3._A_functional_part">3. A functional part </SPAN></H3><P>Promoter PpcbC and FYP was fused into a plasmids using pZE12 as the vector.
</P><H2><SPAN class="mw-headline" id="Acknowledgments"> Acknowledgments </SPAN></H2><UL><LI>Professor Rolf-Michael wittich and Professor Kenneth N. Timmis,                                                    Division of Microbiology, GBF-National Reasearch Centre for Biotechnology, D-38124 Braunschweig, Germany
</LI></UL><P>For the bacteria Sphingomonas sp. Strain RW1 they provided.
</P><UL><LI> Professor David N. Dowling,  Department of Microbiology, University College, Cork, Ireland
</LI></UL><P>For the bacteria E. coil SMl0 provided.
</P><UL><LI> Professor Junfeng Niu, School of Enviroment, Beijing Normal University 
</LI></UL><P>For his instructions and advices
</P><UL><LI> Dr Yingwu Huang,  Bioinformatics Institute, Tsinghua University
</LI></UL><P>For his idea and instructions 
</P><UL><LI><A href="/Team:Tsinghua" title="Team:Tsinghua">Tsinghua Team</A></LI></UL><P>For the experience we shared and the indispensable help.
</P><UL><LI><A href="/Team:Chiba" title="Team:Chiba">Chiba Team</A></LI></UL><P>For all the information we shared and all the joy we had, especially Mai and Yoshimi.
</P><UL><LI><A href="/Team:Tokyo_Tech" title="Team:Tokyo Tech">Tokyo Tech Team</A></LI></UL><P>For all the information and advice and encouragement.
</P><UL><LI><A href="/Team:USTC" title="Team:USTC">USTC Team</A></LI></UL><P>For useful suggestions on our experiments and delicious food in Hefei, especially Bo and Jian.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Beijing_Normal/Project">http://2008.igem.org/Team:Beijing_Normal/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Beijing_Normal/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Beijing_Normal/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Beijing_Normal/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Beijing_Normal/Project&amp;oldid=104739" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>