<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_RHIT_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:RHIT/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:RHIT"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:RHIT/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:RHIT/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:RHIT/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:RHIT/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:RHIT/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:RHIT/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:RHIT/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:RHIT/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A></DIV><A href="https://2017.igem.org/Team:RHIT/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:RHIT/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:RHIT/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:RHIT/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Public Engagement
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=RHIT"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="igem_2017_content_wrapper"><DIV class="column full_size"><H2>Goal</H2><P>The primary goal of this project was to develop and characterize a small riboswitch library. All the riboswitches selected repressed translation when bound to either Vitamin B9 or Vitamin B12. These riboswitches were placed in PSB1C3 to be submitted to the registry. However, while a submission of the basic parts would be most useful to those who wish to use them to regulate future projects, it had to be ensured that the parts were effective. Additionally, the parts were to be characterized such that a general expectation could be formed of how the parts behave, not only in the specific system tested, but in other systems.</P><H2>Why parts were selected</H2><P>After searching the literature for riboswitches, five were selected. Three switches that were regulated by vitamin B9 and two regulated by vitamin B12. These riboswitches were selected because the entire sequence was available and because the spacing between the promoter and desired gene or translated region was known.</P><P><B><I>L. casei</I> B9 Responsive Riboswitch</B><B>Part Number:</B> BBa_ K230000</P><P style="text-indent: 5em"><B>Paper: </B>
Leigh, J. Rational Design, Re-engineering and Characterisation of Tetrahydrofolate Riboswitches in Bacteria (2015) (Doctoral Dissertation). University of Manchester eScholar.
<A href="https://www.escholar.manchester.ac.uk/api/datastream?publicationPid=uk-ac-man-scw:281096&amp;datastreamId=FULL-TEXT.PDF">https://www.escholar.manchester.ac.uk/api/datastream?publicationPid=uk-ac-man-scw:281096&amp;datastreamId=FULL-TEXT.PDF</A></P><P>This part is originally from the<I> Lactobacillus casei</I> genome. According to literature, this tetrahydrofolate (THF) riboswitch could be used with certain chemical (has to be synthesized) to increase the magnitude of response. Although having a lower basal expression level, the response of this riboswitch to ligands could be made much greater than the one in the Chimeral B9 responsive riboswitch</P><P><B>Chimeral B9 Responsive Riboswitch</B><B>Part Number:</B> BBa_ K230001</P><P style="text-indent: 5em"><B>Paper: </B>
Trausch, J.J., Ceres, P., Reyes, F.E. &amp; Batey, R.T. (2011). The structure of a tetrahydrofolate-sensing riboswitch reveals two ligand binding sites in a single aptamer. Structure, 19, 1413-1423. <A href="https://doi.org/10.1016/j.str.2011.06.019">https://doi.org/10.1016/j.str.2011.06.019</A></P><P>This part originated from the <I>Streptococcus mutans</I> genome This is a THF riboswitch from <I>S. mutans</I> with expression platform from <I>Bacillus subtilis</I> on both sides. The expression platforms were used in the original paper to carry out experiments that measure the regulation strength of vitamin B9. It is very well characterized as the binding strength and level of regulatory response with different molecules have been determined through experimentation. The two locations of binding are also very well documented. This switch was chosen due to its potential as a standard for the characterization of other THF riboswitches. </P><P><B><I>S. mutans</I> B9 Responsive Riboswitch</B><B>Part Number:</B> BBa_ K230002</P><P style="text-indent: 5em"><B>Paper:</B>
Leigh, J. Rational Design, Re-engineering and Characterisation of Tetrahydrofolate Riboswitches in Bacteria (2015) (Doctoral Dissertation). University of Manchester eScholar.
<A href="https://www.escholar.manchester.ac.uk/api/datastream?publicationPid=uk-ac-man-scw:281096&amp;datastreamId=FULL-TEXT.PDF">https://www.escholar.manchester.ac.uk/api/datastream?publicationPid=uk-ac-man-scw:281096&amp;datastreamId=FULL-TEXT.PDF</A></P><P>This part has the same riboswitch as BBa_K230001, but without the chimeric metE expression platform.</P><P><B><I>E. coli</I> B12 Responsive Riboswitch</B><B>Part Number:</B> BBa_ K230003</P><P style="text-indent: 5em"><B>Paper:</B>
 Johnson, J.E., Reyes, F.E., Polaski, J.T. &amp; Batey, R.T (2012). B12 Cofactors Directly Stablize an mRNA Regulatory Switch. Nature, 492, 133-137. <A href="https://www.doi.org/10.1038/nature11607">https://www.doi.org/10.1038/nature11607</A></P><P>This part originated from <I>Escherichia coli</I>. It is a commonly used adenosylcobalamin riboswitch and is known for its direct relationship with the btuB gene, one of the genes controlling vitamin B12 uptake in E. coli. It was chosen due to information availability.</P><P><B><I>T. tengcongensis</I> B12 Responsive Riboswitch</B><B>Part Number:</B> BBa_ K230004</P><P style="text-indent: 5em"><B>Paper:</B> 
Johnson, J.E., Reyes, F.E., Polaski, J.T. &amp; Batey, R.T (2012). B12 Cofactors Directly Stablize an mRNA Regulatory Switch. Nature, 492, 133-137. <A href="https://www.doi.org/10.1038/nature11607">https://www.doi.org/10.1038/nature11607</A></P><P>This part originated from <I>Thermoanaerobacter tengcongensis</I>. NCBI Blast showed that this riboswitch might exist in the bacteria right before the btuR gene, a gene related to vitamin B12 uptake. </P><H2>Design of Parts</H2><P>All the parts were synthesized by Integrated DNA Technologies. The synthesized DNA was the sequence of the riboswitch in between the biobrick prefix and suffix. This part was then assembled into the linearized vector pSB1C3 using Gibson assembly. Protocols for the assembly are included in the Lab Notebook section.</P><H2>Construct</H2><P>The construct was designed so that the parts could be analyzed relative to other systems. Generally, the construct consisted of a constitutive promoter, the riboswitch - which includes a ribosomal binding site, a reporter, and a double terminator placed within the biobrick prefix and suffix.</P><P>The promoter used was from the Anderson library of promoters. The relative strength of the promoter -measured by fluorescence- of BBa_J23106 is known relative to the other promoters in the library. The anderson promoter that all are relative to, BBa_J23100 was set as a basis of one. The promoter used in our construct was known to have approximately 47% the strength of BBa_J23100. The GFP variant used- BBa_E0050 was that used and interlab, and the methods of measurement are fairly standard. In between the riboswitch and GFP were small sequences of the 5’ untranslated region that were found in the literature. These sequences had worked in between the region that functions as the riboswitch and the translated region. The constructs were assembled in the expression vector BBa_J61002.</P><H2>How they were Created</H2><P>All the construct were created through Gibson Assembly. The parts that were not already linearized beforehand were amplified using PCR and primers constructed to yield the proper overlap for assembly. Plasmid pSB1C3 was cut with Spe1 and EcoR1 before assembly to yield the full prefix and suffix, not the truncated form that can be found in the linearized pSB1C3. 
A transformation using the assembly products was performed into NEB DH5α competent<I> E. coli </I>before the verification of the constructs. Verification was done by running colony PCR and gels on the transformed <I>E. coli</I>. Once verified, the constructs were mini-prepped and transformed into the testing chassis- a strain of <I>E. coli</I> that would better withstand the testing conditions- MG1655 (DE3). </P><H2>Experimental Design</H2><P>Two primary sets of experiments were run on the testing constructs, one utilizing fluorescence readings and the other with isothermal titration calorimetry.

The fluorescence readings were divided into segments. Both had measurements taken from aliquots of liquid cultures on plate reader at zero, two, four, eight, and twenty hours. Five milliliters of M9 minimal media were grown up for one set of fluorescence readings, the constructs were tested with 2, 1, and 0.5 mM of the respective ligand. Fluorescence measurements were normalized to the fluorescence of a negative control in media, BBa_R0040 per unit of OD630 of the assay relative to that of the the media. This set of readings were meant to test the sensitivity of the riboswitch regulation to the concentration of the vitamin.

The other set of fluorescence readings was where the constructs were tested with 0 and 0.03 mM of the added vitamin. These readings were meant to establish a clear difference between the riboswitch in the presence of the aptamer and without.These readings were normalized to the blank media.

Isothermal titration calorimetry was done on the riboswitch constructs after <I>in vitro </I>RNA synthesis. The binding constant of the RNA to the ligands was measured. These experiments are ongoing.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>