<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNIK_Copenhagen_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNIK_Copenhagen/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNIK_Copenhagen/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIK_Copenhagen/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIK_Copenhagen/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNIK_Copenhagen/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNIK Copenhagen/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="wrapper"><DIV id="nav"><UL><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen">Home</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Project">Overview</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/ExperimentalApproach">Experimental Approach</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Project/Applications">Applications</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Parts">Submitted Parts</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Attributions">Attributions</A></LI></UL><LI><A href="#">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1094">Official Team Profile</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Team">Meet the team</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Danish_collaboration">The Danish Collaboration</A></LI><LI><A href="#">Safety and Ethics</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/OpenSource">Ethics</A></LI><LI><A href="#">Bricks of Knowledge</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/BricksOfKnowledge">Invitation</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/BricksOfKnowledgeMovieBricks">Bricks of Knowledge</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/TheCphStrain">The CPH Strain</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Press">Press</A></LI><LI><A href="https://2013.igem.org/Team:UNIK_Copenhagen/Sponsors">Sponsors</A></LI><DIV id="social_media"><A href="https://twitter.com/iGEMCopenhagen" class="twitter-follow-button" data-show-screen-name="false" data-show-count="false" data-lang="en">Follow @twitterapi</A><A href="https://twitter.com/iGEMCopenhagen" class="twitter-share-button" data-count="none" data-lang="en">Tweet</A></DIV></DIV><DIV id="indent_content"><H1> Notebook </H1><H4>Get our Nootebook in a  <A href="JavaScript:newPopup('https://2013.igem.org/wiki/index.php?title=Team:UNIK_Copenhagen/Notebook/Printable&amp;printable=yes');"><U>printable version here</U></A>.</H4><P>If nothing happens when you click, check if you have disabled popup-windows. For more detailed descriptions of the methods used, please see our <A href="https://static.igem.org/mediawiki/2013/9/99/UNIK_Copenhagen_Protocols.pdf" target="_blank">protocols</A>.</P></DIV><DIV id="all_content"><DIV id="tabs"><DIV id="items"><DIV id="content1" class="content"><DIV class="left_page"><H1>Week 1 - July 8<SUP>th</SUP>-14<SUP>th</SUP></H1><P><U>Culturing of magnetotactic bacteria (MTBs)</U>
Making base medium for growing magnetotactic bacteria. This medium was also used to make agar plates. 
MS-1 inoculated in liquid base medium. Falcon tubes were flushed with the nitrogen before use and incubated at 28 degrees. 
Plates were kept in a closed container with an Oxoid AnaeroGen pad.<U>Assembly of constructs</U>
Glycerol stock of <I>E. coli</I> with pBBR1MCS-2 was thawed to grow on these plates.
Primers for MamC (MS-1 strain) was ordered. 
</P></DIV><DIV class="right_page"><P>Team photo in the park at Frederiksberg Campus </P></DIV></DIV><DIV id="content2" class="content"><DIV class="left_page"><H1>Week 2 - July 15<SUP>th</SUP>-21<SUP>st</SUP></H1><P><U>Assembly of constructs</U>
Overnight liquid cultures of pBBR1MCS-2 and eGFP plasmid (from Adam Takos) 
Touchdown PCR amplification of MamC from MS-1 gDNA (both Phusion and Hotmaster) and of eGFP (only Hotmaster). 
</P></DIV><DIV class="right_page"><P>Gel purification of bands from PCR amplification and subsequent nanodrop. Due to poor yield from this extraction gradient PCR was performed (56-68 degrees). 

Traditional cloning of eGFP fragments into pDRIVE. 
Transformation of eFbFP-pEX-A (synthesized gene, Eurofins) and eGFP-pDRIVE. </P></DIV></DIV><DIV id="content3" class="content"><DIV class="left_page"><H1>Week 3 - July 22<SUP>nd</SUP>-28<SUP>th</SUP></H1><P><U>Culturing of MTBs</U>
making charcoal media and plates. MS-1 and MSR-1 was inoculated. Liquid cultures were setup in Hungate tubes 200ul, 400ul and 600ul of each strain (MS-1 and MSR-1). Charcoal plates were inoculated as well with both strains.
<U>Assembly of constructs</U>
Colony PCRs of eFbFP-pEX-A and eGFP-pDRIVE. Only eFbFP was successful. 

Overnight cultures were made of colonies 1-4 (eFbFP-pEX-A). Miniprep of eFbFP-pEX-A
</P></DIV><DIV class="right_page"><P>


cultures. Samples sent for sequencing. 
Overnight cultures were made of colonies 1-4 (eFbFP-pEX-A). 
Miniprep of eFbFP-pEX-A cultures. Samples sent for sequencing. 
 
 
Colony PCR of eGFP-pDRIVE was repeated with the same outcome.
Cloning and transformation was also repeated.  CloneJET PCR cloning kit was used to create eGFP-pJET (pJET was chosen instead of pDRIVE). Not successful. 

Gradient PCR of eGFP premade vector (56-65 degrees). Gel electrophoresis was carried out and the bands were purified from the gel. 

An attempt to clone eGFP into pDRIVE was not successful.
The cloning of eFbFP into expression vector pBBR1MCS-2 was successful instead.  

Colony PCR of eGFP-pDRIVE and eFbFP-pBBR1MCS-2. Gel electrophoresis revealed only one colony to be successful (eFbFP-pBBR1MCS-2 no. 17). 
</P></DIV></DIV><DIV id="content4" class="content"><DIV class="left_page"><H1>Week 4 - July 29<SUP>th</SUP>- August 4<SUP>th</SUP></H1><P><U>Assembly of constructs</U>
Colony PCR of eGFP-pDRIVE was repeated with increased elongation time (1 min.) 




Cloning looks successful and colonies 1-4 were set up as liquid cultures overnight. 

Colony PCR of eFbFP-pBBR1MCS-2 because only one colony was positive in week 3. Increased elongation time. 



Overnight liquid cultures were setup for colonies 2, 4, 5 and 11. 
Miniprep was performed on these cultures alongside eGFP-pDRIVE colonies 1-4. This gave good yields and the plasmids were sent for sequencing. This showed eGFP-pDRIVE no. 4 and eFbFP-pBBR1MCS-2 no. 17 was correct! 
</P></DIV><DIV class="right_page"><P>
Cloning and transformation of eGFP-pBBR1MCS-2 using plasmid no. 4 (mentioned just above). The transformation only gave rise to 2 colonies, thus suggesting the cloning to be unsuccessful. The cloning was repeated and transformation was carried out again. Colony PCR was performed alongside MamC-pJET (see gel below).

Liquid cultures were setup from eFbFP-pBBR1MCS-2 no. 2, 4, 5, 11, 17 for 2 hours in order to  measure fluorescence. First, OD(600nm) was measured to ensure equal density of cells. The measurements did not show fluorescence (ex. 450 nm, em. 495 nm) from the cultures.

Cloning and transformation of MamC-pDRIVE. Colony PCR showed the cloning to be of cryptic success. Colony PCR was redone without success. 
Cloning was repeated using the CloneJET PCR cloning kit. MamC-pJET was transformed and plated. Colony PCR was carried out alongside eGFP-pBBR1MCS-2. 

The gel electrophoresis indicated MamC-pJET to be successfully cloned. However, only a few of the eGFP-pBBR1MCS-2 colonies appeared to have contain the insert.  </P></DIV></DIV><DIV id="content5" class="content"><DIV class="left_page"><H1>Week 5 - August 5<SUP>th</SUP>-11<SUP>th</SUP></H1><P><U>Culturing of MTBs</U>
Colony PCR using MamC primer was done with MS-1 and MSR-1 colonies. MamC was amplified from the genomic DNA. This did not give rise to any bands via gel electrophoresis. 
MamC was, however, successfully amplified from MSR-1 later in the week (see gel below). 
</P></DIV><DIV class="right_page"><P><U>Assembly of constructs</U>
Colonies 1, 2, 3 (MamC-pJET) and II, IV, VIII (eGFP-pBBR1MCS-2) were setup in liquid cultures (see colony PCR from previous week). Miniprep was carried out and plasmids were sent for sequencing. All MamC sequences were perfect while all eGFP sequences were wrong. The cloning of eGFP into pDRIVE was successfully redone (documentation lost). Fluorescence was also measured of eFbFP-pBBR1MCS-2 strains. No fluorescence detected as the sequencing explains. Inducing with IPTG (final conc. 1 mM) had no effect.
<U>The CPH strain</U>
Enrichment of magnetotactic bacteria was done according to the protocol. It was unclear whether the bacteria observed in the microscope were actually magnetotactic. They were compared to MS-1 and MSR-1 samples. 




</P></DIV></DIV><DIV id="content6" class="content"><DIV class="left_page"><H1>Week 6 - August 12<SUP>th</SUP>-18<SUP>th</SUP></H1><P><U>Assembly of constructs</U>
Since we couldn't detect any fluorescence from eFbFP-pBBR1MCS-2 we needed troubleshoot this. We found a frameshift in the vector and thus we repeated the cloning of eFbFP into pBBR1MCS-2. This was done alongside cloning eGFP into pBBR1MCS-2. Transformation and colony PCR were carried out. All colonies appeared to be positive on the gel electrophoresis (see picture below).

MamC-pJET did not contain the restriction sites we needed. Therefore, the whole procedure for obtaining MamC-pJET was repeated using other primers (with different overhangs).


</P></DIV><DIV class="right_page"><P> 
Six of each genotype were set up for liquid cultures overnight (colonies 1, 2, 6, 9, 13, 16 for eFbFP and 18, 19, 23, 25, 27, 31 for eGFP). Miniprep was done and samples were sent for sequencing. None of the eFbFP gave rise to good quality sequencing. Colonies 18, 19, 23 and 27 of eGFP contained the correct sequence. 

Empty pBBR1MCS-2 vector was transformed into E. cloni in order to obtain more stock vector. 
<U>Culturing of MTBs</U>
MSR-1 base medium culture was inoculated in new medium and put in 28 degrees shaker over the weekend. This was done to check whether the strain would grow aerobically as well. 

</P></DIV></DIV><DIV id="content7" class="content"><DIV class="left_page"><H1>Week 7 - August 19<SUP>th</SUP>-25<SUP>th</SUP></H1><P><U>Assembly of constructs</U> 
Plasmids from last week eFbFP/eGFP-pBBR1MCS-2 were digested alongside the newly assembled MamC-pJET in order to assess whether the restriction sites were present or not (see gel no. 1).

All lanes with eGFP or MamC were positive (see gel above) and the bands were purified from the gel. From this purification very poor concentration was obtained. From the lanes with eFbFP, no. 6 stands out from the negative control. Therefore we decided we continue working with this plasmid. A re-ligation of eFbFP-pBBR1MCS-2 was performed. 

Cloning of MamC into linearized eGFP-pBBR1MCS-2 and subsequently transforming the plasmid was also done. Colony PCR was carried out on MamC-eGFP-pBBR1MCS-2 and eFbFP-pBBR1MCS-2. We decided to use the Gateway (GTW) system to express our constructs (see gel no. 2). 
To save time the colony PCR was performed using gateway primers (MamC F and eGFP R, eFbFP F and R). The PCR product of MamC-eGFP-pBBR1MCS-2 colonies 1, 3, 4, 5, 6 and eFbFP-pBBR1MCS-2 3, 7, 10, 13, 15 were PCR purified.



</P><DIV id="under"><DIV id="under_page"><P> Gel no. 1: Digestion of eGFP-pBBR1MCS-2, eFbFP-pBBR1MCS-2 and MamC-pJET </P></DIV><DIV id="under_page"><P> Gel no. 2: Colony PCR of MamC-eGFP-pBBR1MCS-2 and eFbFP-pBBR1MCS-2 using Gateway primers </P></DIV><DIV id="under_page"><P> Gel no. 3: Colony PCR of eFbFP-pDONR207 and MamC-eGFP-pDONR207 </P></DIV><DIV id="under_page2"><P> Gel no. 4: 16S PCR amplification. Negative control - <I>H. C. Ørsteds Parken</I> sample -  <I>Christiania</I> sample </P></DIV></DIV></DIV><DIV class="right_page"><P>
Gateway BP reaction was performed to clone MamC-eGFP into pDONR207 using MamC-eGFP (GTW PCR product) no. 3 and 4 was used and eFbFP (GTW PCR product) no. 3 and 7. Colony PCR was carried out (see gel no. 3).

Colonies 2, 3, 5, 7, 9, 11, 13, 14 were selected to make liquid culture. Miniprep was done and samples were sent for sequencing.
<U>BioBrick submission</U>
PCR amplification of MamC15 (week 6), eFbFP-pEX-A, eGFP18 (week 6) and eGFP19 (week 6). Products were run on a gel and the bands were purified. Ligation was set up to clone the fragments into pSB1C3 and subsequently transformation was performed as well. 

<U>Culturing of MTBs</U>
Microscopy of MSR-1 in order to check morphology is corresponding to what is reported in the literature (Jogler &amp; Schüler, Annu. Rev. Microbiol. 2009. 63:501-21). We did not see any signs of bacterial growth in the liquid cultures set up in week 6. 
<U>The CPH strain</U>
PCR amplification of 16S from our lake sample (enriched) and run on gel (see gel no. 4).
The gel shows some amount of contamination in the negative control (no template). 


</P></DIV></DIV><DIV id="content8" class="content"><DIV class="left_page"><H1>Week 8 - August 26<SUP>th</SUP>-September 1<SUP>st</SUP></H1><P><U>Assembly of constructs</U>
Sequencing of eFbFP-pDONR207 and MamC-eGFP-pDONR207. Gateway LR reaction for cloning MamC-eGFP (no 11 and no 13) into pJAM1786 and transformation into <I>E. coli</I>. 
Colony PCR was performed on MamC-eGFP-pJAM1786 and eFbFP-pDONR207 (see gel below)

Colonies 1, 2, 6, 10, 11, 13, 14, 16 were inoculated in liquid culture and grown overnight. Miniprep was performed and LR reaction was done using eFbFP-pDONR207 no 11. Enhanced FbFP-pJAM1786 was transformed and grown overnight. 

Expressing eFbFP in the gateway system was repeated. PCR amplification using Gateway primers  
</P></DIV><DIV class="right_page"><P> 
on eFbFP-pEX-A was carried out. Gateway BP reaction was done to move the fragment into pDONR207. Afterwards the plasmid was transformed into <I>E. coli</I>. 
<U>The CPH strain</U>

Cloning of 16S PCR products into pJET and subsequent transformation into <I>E. coli</I>. Liquid cultures were grown inoculating cells from 24 different colonies (12 from <I>H.C. Ørsted Parken</I> sample and 12 from <I>Christiania</I> sample). Miniprep was carried out and samples were sent for sequencing. 

<U>BioBrick submission</U>
Colony PCR was performed.



Colonies 1, 5, 12 and 15 were inoculated to make liquid cultures, that were grown overnight. 
</P></DIV></DIV><DIV id="content9" class="content"><DIV class="left_page"><H1>Week 9 - September 2<SUP>nd</SUP>-8<SUP>th</SUP></H1><P><U>Assembly of constructs</U>
Cultures of eFbFP-pJAM1786 was minipreped and samples were sent for sequencing. Microscopy revealed MamC-eGFP-pJAM1786 and eFbFP-pJAM1786 emitted fluorescence. To measure fluorescence quantitatively, we set up 8 liquid cultures of MamC-eGFP-pJAM1786 and 3 of MamC-pSB1C3 as a non-fluorescent control. Fluorescence was measured for dilutions 1X, 2X, 5X, 10X, 50X (Ex. 485 nm em. 535 nm.) using an ELISA reader (black plate) and gave rise characterization for BioBrick BBa_K1094401. 
OD(600nm) was also measured to ensure approx. equal cell density.  
<U>BioBrick submission</U>
Miniprep was performed and samples sent for sequencing. The sequencing results revealed that sequences of  MamC-pSB1C3, eGFP(18)-pSB1C3


</P></DIV><DIV class="right_page"><P> 
and eGFP(19)-pSB1C3 were correct while eFbFP-pSB1C3 held the wrong insert. We uncovered that this was due two &quot;illegal&quot; restriction sites in the FbFP sequence. We, therefore, ordered another synthetic gene from IDT with the restriction sites removed. 
To submit MamC-eGFP as a BioBrick we PCR amplified (1:20 min elongation time) the whole construct (MamC F primer and eGFP R primer). To do this we used 3 different templates. 
Afterwards, digestion and ligation into pSB1C3 was performed and the plasmid was transformed into E. coli. 
Colony PCR was performed.
Colonies 5 and 6 were inoculated into liquid culture and grown overnight. Miniprep was performed and samples sent for sequencing. 



</P></DIV></DIV><DIV id="content10" class="content"><DIV class="left_page"><H1>Week 10 - September 9<SUP>th</SUP>-15<SUP>th</SUP></H1><P><U>Assembly of constructs</U>
The newly synthesized eFbFP came in. Traditional cloning was used to put it into pSB1C3 and the plasmid was transformed. Colony PCR was performed.

Colonies 2, 4, 5, 6, and 8 were inoculated in liquid culture. Miniprep was carried out and samples sent for sequencing. 
<U>BioBrick submission</U>
We sent 4 BioBricks for the registry. Enhanced FbFP-pSB1C3 was sent in 5 copies because we did not have the sequencing results yet. 

</P></DIV><DIV class="right_page"><P><U>Culturing of MTBs</U>
New base media was made according to Schüler et al. 1991 (see protocols). A variant was made with 0,2% agar as well. Both MS-1 and MSR-1 were inoculated and grown in 28 degrees in both media. 3 days after the tubes showed signs of bacteria growing. 

<U>Confocal imaging</U>
Cultures set up for confocal imaging. Strains MamC-eGFP-pJAM1786 and eGFP-pBBR1MCS-2 were inoculated. 
</P></DIV></DIV><DIV id="content11" class="content"><DIV class="left_page"><H1>Week 11 - September 16<SUP>th</SUP>-22<SUP>nd</SUP></H1><P><U>Assembly of constructs</U>
To express and characterize eFbFP and eGFP (without MamC) we needed to move the sequences to the Gateway system. Therefore, PCR using Gateway primers was performed and a BP reaction was set up in order to clone eFbFP (new) and eGFP into pDONR207. Transformation and colony PCR was performed. 


Colonies 1, 2, 4, 6, 9, 10, 13, 14 were put into liquid culture and grown overnight. Cultures were centrifuged after 1 day, medium was removed and the cells were stored at -20 degrees. 

</P></DIV><DIV class="right_page"><P><U>Confocal imaging</U>
Confocal images were taken of MamC-eGFP-pJAM1786 and eGFP-pBBR1MCS-2 alongside MS-1 and MSR-1 cultures. 
Hover the mouse over the picture to see!</P><P> First photo: eGFP,   Second photo: MamC-eGFP </P></DIV></DIV><DIV id="content12" class="content"><DIV class="left_page"><H1>Week 12 - September 23<SUP>rd</SUP>-29<SUP>th</SUP></H1><P><U>Assembly of constructs</U>
Miniprep was carried out on the frozen pellets from last week. LR reaction was setup to yield eGFP-pJAM1786 and eFbFP-pJAM1786. Colony PCR was performed. 

Looking at the bands from the Colony PCR, one can tell that there might have been problems with the gel. We decided to consider the odd looking bands in the left to be due to the gel and therefore we inoculated the restreaked cultures into liquid LB. 

Using an opaque ELISA reader plate we measured fluorescence of 7 eGFP-pJAM1786 and 6 eFbFP-pJAM1786 cultures
</P></DIV><DIV class="right_page"><P> 
(3 control cultures, containing pBBR1MCS-2 without insert, were used as well).
The measurements failed to show  eFbFP-pJAM1786 to be significantly different from the control cultures. This was in contrast to eGFP-pJAM1786 that was shown be fluorescent. 

Western blot was performed on the 3 strains above (eFbFP, eGFP and control), MamC-eGFP-pJAM1786 and a GFP control. Strangly, the MamC-eGFP-pJAM1786 cultures did not show fluorescence. This was highly unexpected since the cultures that was used for this glycerol stock earlier had been shown to be fluorescent. Therefore, we did not expect any bands in the MamC-eGFP lanes. 

The western blot somewhat failed since only one of the standard protein markers showed in the membrane. (See a visualization <A href="https://static.igem.org/mediawiki/2013/8/83/UNIK_Copenhagen_Notebook_ref_Western.jpg
" target="_blank">here</A>). The GFP standard (band at the right side) seem to be highly overloaded. The lanes to left is eGFP-pJAM in different amounts. 



</P></DIV></DIV><DIV id="content13" class="content"><DIV class="left_page"><H1>Week 12 - Sept. 30<SUP>th</SUP>-Oct. 6<SUP>th</SUP></H1><P><U> Wrapping it up - and preparing for Lyon </U>
This last week before the jamboree we've spent finishing up our parts characterisation and getting the wiki shining before the freeze.

The presentation crew have been working on their performance for Lyon. The rest of the team and supervisors got to see a rehersal and feed them inputs and comments. We're all looking forward to put on a show in Lyon.


Team Magneto is ready and looking forward to a nice time in Lyon!


</P></DIV></DIV></DIV><DIV id="items"><UL id="tabs"><LI><A href="javascript:ThisTab('tab1', 'content1');" id="tab1" class="active">1</A></LI><LI><A href="javascript:ThisTab('tab2', 'content2');" id="tab2">2</A></LI><LI><A href="javascript:ThisTab('tab3', 'content3');" id="tab3">3</A></LI><LI><A href="javascript:ThisTab('tab4', 'content4');" id="tab4">4</A></LI><LI><A href="javascript:ThisTab('tab5', 'content5');" id="tab5">5</A></LI><LI><A href="javascript:ThisTab('tab6', 'content6');" id="tab6">6</A></LI><LI><A href="javascript:ThisTab('tab7', 'content7');" id="tab7">7</A></LI><LI><A href="javascript:ThisTab('tab8', 'content8');" id="tab8">8</A></LI><LI><A href="javascript:ThisTab('tab9', 'content9');" id="tab9">9</A></LI><LI><A href="javascript:ThisTab('tab10', 'content10');" id="tab10">10</A></LI><LI><A href="javascript:ThisTab('tab11', 'content11');" id="tab11">11</A></LI><LI><A href="javascript:ThisTab('tab12', 'content12');" id="tab12">12</A></LI><LI><A href="javascript:ThisTab('tab13', 'content13');" id="tab13">13</A></LI></UL></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UNIK_Copenhagen/Notebook">http://2013.igem.org/Team:UNIK_Copenhagen/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNIK_Copenhagen/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNIK_Copenhagen/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNIK_Copenhagen/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNIK_Copenhagen/Notebook&amp;oldid=284987" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>