<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CO_Mines_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CO Mines/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="w3-top"><DIV class="w3-bar" id="myNavbar"><LI id="list" class="dropdown1"><A href="https://2018.igem.org/Team:CO_Mines" class="w3-bar-item w3-button" style="color:black;"><B>Home</B></A></LI><LI id="list" class="dropdown1"><A href="javascript:void(0);" class="dropbtn1" style="color:black;"><B>Team</B></A><DIV class="dropdown-content1"><A href="https://2018.igem.org/Team:CO_Mines/Team">Team Members</A><A href="https://2018.igem.org/Team:CO_Mines/Collaborations">Collaborations</A></DIV></LI><LI id="list" class="dropdown1"><A href="javascript:void(0);" class="dropbtn1" style="color:black;"><B>Project</B></A><DIV class="dropdown-content1"><A href="https://2018.igem.org/Team:CO_Mines/Description">Description</A><A href="https://2018.igem.org/Team:CO_Mines/Design">Design</A><A href="https://2018.igem.org/Team:CO_Mines/Experiments">Experiments</A><A href="https://2018.igem.org/Team:CO_Mines/Notebook">Notebook</A><A href="https://2018.igem.org/Team:CO_Mines/InterLab">InterLab</A><A href="https://2018.igem.org/Team:CO_Mines/Results">Results</A><A href="https://2018.igem.org/Team:CO_Mines/Attributions">Attributions</A></DIV></LI><LI id="list" class="dropdown1"><A href="https://2018.igem.org/Team:CO_Mines/Safety" class="dropbtn1" style="color:black;"><B>Safety</B></A></LI><LI id="list" class="dropdown1"><A href="https://igem.org/2018_Judging_Form?team=CO_Mines" class="dropbtn1" style="color:black;"><B>Judging Form</B></A></LI></DIV></DIV><P>Our project is broken down into two major components, the binding and sensing of cadmium in solution. Each component contained several mechanisms and processes that affected the various iterations of our design. For the binding portion this includes the intermediary transfer plasmids used to move the binding sequence from a DNA fragment into a functioning <I>E. coli</I> culture. For the sensing portion this includes the sensitivity of the sensor and how we managed to test the actual activity of the promoter versus the presence of cadmium.</P><H1>Binding:</H1><P>The binding sequence was a pseudo metallothionein which has been shown to bind to heavy metals such as cadmium, lead, and mercury. The sequence is very short, so we ordered a sequence from IDT that included the peptide sequence along with several methods of cloning which are described below. pMAL was chosen as the vector as the binding peptide would be fused to maltose binding protein(MBP). This fusion stabilizes the peptide and expresses it in the periplasm of the cell. This expression would allow for cadmium and other heavy metals to attach to the cell. If these cells could be used in a biofilm or bioreactor, heavy metals could be taken out of mines tailing streams.</P><P><B> Fig. 1: </B> Design of the binding peptide sequence </P><H3> Restriction Endonucleases (BamHI and HindIII) </H3><P>  BamHI and HindIII are used to allow us to clone our gene into the pMAL vector behind the MBP. A sequence with specific restriction enzymes sites, in our case BamHI and HindIII, allows for accessible and targeted interaction between the pMAL vector and the binding sequence. </P><P><B>Fig. 2:</B> Digestion Protocol for inserting the peptide into the vector (Adapted from an image on Addgene [1]) </P><H3> HiFi Assembly (pUC19 homology) </H3><P>  Another method of cloning designed on the sequence was to use the pUC19 homology to allow for HiFi Assembly. Using the HiFi Assembly protocol, our binding sequence would be added to the pUC19 vector. This would allow for the creation of stock colonies containing the binding peptide insert, ensuring a source of the insert. The sequence could then be moved from pUC19 into the pMAl vector. </P><H3> PCR primers (M13) </H3><P> The binding peptide sequence is flanked by both the forward and reverse M13 PCR primer sequences. By using M13 primers the binding sequence could amplified outside of a vector, or cloned via TOPO TA cloning. This both allows for a highly efficient cloning mechanism and a quick amplification method on the IDT product. </P><H1>Sensing:</H1><P>The sensing portion of the project consists of a cadC promoter from <I>Staphylococcus aureus</I> paired with a green fluorescent protein (GFP), specifically GFPmut3a. This specific promoter was chosen because a previous study determined the part to be highly selective to cadmium and able to measure quantities as low as 5 µg/L. The selectivity and ability to measure low quantities are crucial for cadmium sensing as the EPA cadmium limit is 5 µg/L [2]. The final design of the synthesized cadC sensor can be seen below. 
</P><P>The design of the sensor had several key ideas in mind. The first was to allow an easy transfer into the iGEM backbone. This is solved by including the Biobrick Prefix and Suffix into the synthesized part. To allow for an easy combination of the cadC Promoter and the binding part, EcoRI and BamHI were placed around the promoter. This would allow for an easy transfer into pMAL. The Bio-Brick terminator, BBa_B0015, was added at the end of the GFP sequence to stop transcription. This terminator was chosen as it showed strong termination in the testing. To ensure that that testing could be conducted immediately, the fragment was placed in pUCIDT with ampicillin resistance by IDT. This would also allow for stocks to be created immediately so a source of the fragment could be maintained.  </P><P>The testing was done using a 96 well plate reader to allow for continuous growth and fluorescence scans. The goal was to first create a series of graphs to analyse the sensor’s range, and then to determine the effect of cadmium concentrations on the <I> E. coli</I> growth. From this point the plan was to then use Pb and Zn salts to test the selectivity of the sensor. The data would then be analysed to look for any possible areas of improvement. </P><H5>References:</H5></DIV><DIV><P><B>[1]</B> “Plasmid Modification by Annealed Oligo Cloning.” <I>Addgene: The Nonprofit Plasmid Repository,</I> 2018, <A href="https://www.addgene.org/protocols/annealed-oligo-cloning/">www.addgene.org/protocols/annealed-oligo-cloning/.</A></P><P><B>[2]</B> “National Primary Drinking Water Regulations,” <I>EPA </I>, 22-Mar-2018. [Online]. https://www.epa.gov/ground-water-and-drinking-water/national-primary-drinking-water-regulations. [Accessed: 17-Oct-2018].
</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>