<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CO_Mines_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CO Mines/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="w3-top"><DIV class="w3-bar" id="myNavbar"><LI id="list" class="dropdown1"><A href="https://2018.igem.org/Team:CO_Mines" class="w3-bar-item w3-button" style="color:black;"><B>Home</B></A></LI><LI id="list" class="dropdown1"><A href="javascript:void(0);" class="dropbtn1" style="color:black;"><B>Team</B></A><DIV class="dropdown-content1"><A href="https://2018.igem.org/Team:CO_Mines/Team">Team Members</A><A href="https://2018.igem.org/Team:CO_Mines/Collaborations">Collaborations</A></DIV></LI><LI id="list" class="dropdown1"><A href="javascript:void(0);" class="dropbtn1" style="color:black;"><B>Project</B></A><DIV class="dropdown-content1"><A href="https://2018.igem.org/Team:CO_Mines/Description">Description</A><A href="https://2018.igem.org/Team:CO_Mines/Design">Design</A><A href="https://2018.igem.org/Team:CO_Mines/Experiments">Experiments</A><A href="https://2018.igem.org/Team:CO_Mines/Notebook">Notebook</A><A href="https://2018.igem.org/Team:CO_Mines/InterLab">InterLab</A><A href="https://2018.igem.org/Team:CO_Mines/Results">Results</A><A href="https://2018.igem.org/Team:CO_Mines/Attributions">Attributions</A></DIV></LI><LI id="list" class="dropdown1"><A href="https://2018.igem.org/Team:CO_Mines/Safety" class="dropbtn1" style="color:black;"><B>Safety</B></A></LI><LI id="list" class="dropdown1"><A href="https://igem.org/2018_Judging_Form?team=CO_Mines" class="dropbtn1" style="color:black;"><B>Judging Form</B></A></LI></DIV></DIV><I>The following table includes details on daily progress in the cadmium binding and cadmium sensing components of our project, from June to October 2018.</I><TABLE><TBODY><TR><TH colspan="2">June 12, 2018</TH></TR><TR><TD>BINDING</TD><TD>Organized schedules and set up notebooks on Canvas</TD></TR><TR><TD>SENSING</TD><TD>Determined objectives until July 13</TD></TR><TR><TH colspan="2">June 13, 2018</TH></TR><TR><TD rowspan="2">SENSING</TD><TD>Put needed materials in iGEM2018 Ordering Requests</TD></TR><TR><TD>Working to find pI258 from S. aureus (NCTC 50581) that has the cadC gene</TD></TR><TR><TH colspan="2">June 14, 2018</TH></TR><TR><TD rowspan="2">SENSING</TD><TD>Checked existing stock of BamHI, HindIII (for binding) and EcoRI</TD></TR><TR><TD>Started research in mining conditions - 2-8 acidic pH, Zn and Pb typically present with Cd</TD></TR><TR><TH colspan="2">June 15, 2018</TH></TR><TR><TD rowspan="3">SENSING</TD><TD>Went through materials to order, acquired gel electroporation machine from genetics lab</TD></TR><TR><TD>Discussed methods for measuring Cd and fluorescence</TD></TR><TR><TD>Registered for iGEM and IDT</TD></TR><TR><TH colspan="2">June 18, 2018</TH></TR><TR><TD rowspan="3">BINDING</TD><TD>Inoculated E.coli with the pMAL-p4x plasmid in LB/amp solution (4 for mini-prep, 1 for freezer stock)</TD></TR><TR><TD>Running questions include &quot;How to order oligos: premixed or all separate?</TD></TR><TR><TD>Running concerns include &quot;Possible problems associated with low GC content in sequence&quot;</TD></TR><TR><TD rowspan="2">SENSING</TD><TD>Determined objectives until July 13</TD></TR><TR><TD>Researched IDT gBlocks protocols</TD></TR><TR><TH colspan="2">June 19, 2018</TH></TR><TR><TD rowspan="2">BINDING</TD><TD>Performed a mini-prep of pMAL-p4x plasmids, and gel electrophoresis of mini-prep [<B>Figure 1</B>]</TD></TR><TR><TD>Created 9 freezer stocks of pMAL-p4x plasmid</TD></TR><TR><TD rowspan="2">SENSING</TD><TD>Discuss IDT and order sequences (using custom gene synthesis - put promoter through GFP on sequence and don't order pAD123), use oligos as 1 sequence instead of 4</TD></TR><TR><TD>Began Safety Forms</TD></TR><TR><TH colspan="2">June 20, 2018</TH></TR><TR><TD rowspan="3">SENSING</TD><TD>Overlapped binding sequence using gBlocks, add homology and/or a PCR primer</TD></TR><TR><TD>Received iGEM distribution kit</TD></TR><TR><TD>Worked on cadC promoter and gene sequence - Put sacI binding site at end of GFP protein, need to change BamHI and remove XbaI site</TD></TR><TR><TH colspan="2">June 21, 2018</TH></TR><TR><TD rowspan="2">SENSING</TD><TD>Added puc19 homology and m13 primers to binding sequence, ordered binding sequence from IDT as a gBlock</TD></TR><TR><TD>Worked further on Safety Forms</TD></TR><TR><TH colspan="2">June 22, 2018</TH></TR><TR><TD rowspan="2">SENSING</TD><TD>Worked on IDT sequence, ideas include removing PCR parts and GFP primers, adding base pairs after GFP gene and use restriction enzymes in suffix</TD></TR><TR><TD>Intend to use cad promoter in distribution kit to begin sensing tests</TD></TR><TR><TH colspan="2">June 25, 2018</TH></TR><TR><TD rowspan="3">SENSING</TD><TD>Worked on Project Description, finished Safety Sheet, researched electrocomponent cell protocol and EHS (hazards) presentation</TD></TR><TR><TD>Added terminator to promoter and GFP sequence, need to order from IDT</TD></TR><TR><TD>Discussed funding and marketing options</TD></TR><TR><TH colspan="2">June 26, 2018</TH></TR><TR><TD rowspan="2">SENSING</TD><TD>Project Description completed, Safety Form completed, discussed basic outline for binding protein cloning</TD></TR><TR><TD>Filled out Check-In Form for the sequence from S. aureus</TD></TR><TR><TH colspan="2">June 27, 2018</TH></TR><TR><TD>BINDING</TD><TD>Performed a mini-prep of pUC19 plasmids</TD></TR><TR><TD rowspan="2">SENSING</TD><TD>Began gel set-up for mini-prep</TD></TR><TR><TD>Prepared a materials spreadsheet to be updated regularly</TD></TR><TR><TH colspan="2">June 28, 2018</TH></TR><TR><TD rowspan="3">BINDING</TD><TD>Performed gel electrophoresis of pUC19 mini-prep [<B>Figure 2</B>]</TD></TR><TR><TD>Found concentrations of mini-prepped plasmids (nanodrop)</TD></TR><TR><TD>Digested pMAL-p4x with BamHI and HindIII -- let sit overnight in water bath</TD></TR><TR><TH colspan="2">June 29, 2018</TH></TR><TR><TD rowspan="2">BINDING</TD><TD>Gel results of pMAL-p4x digest from the day before [<B>Figure 3</B>]</TD></TR><TR><TD>Cut out linearized plasmid from gel</TD></TR><TR><TD>SENSING</TD><TD>Researched antibody for protein detection of malE protein: included independent antibodies in mice, secondary antibodies in goats and rabbits</TD></TR><TR><TH colspan="2">July 2, 2018</TH></TR><TR><TD rowspan="2">BINDING</TD><TD>Gel purification of pMAL digestion, added ethanol to wash buffer</TD></TR><TR><TD>Attempted to make 500mM EDTA</TD></TR><TR><TD>SENSING</TD><TD>Researched recipes for typical buffers/reagents</TD></TR><TR><TH colspan="2">July 3, 2018</TH></TR><TR><TD rowspan="2">BINDING</TD><TD>Digested pUC19 with HincII - let sit in water bath overnight</TD></TR><TR><TD>Nanodropped the pMAL digestion</TD></TR><TR><TH colspan="2">July 5, 2018</TH></TR><TR><TD rowspan="3">BINDING</TD><TD>Gel electrophoresis results of digested pUC19 [<B>Figure 4</B>]</TD></TR><TR><TD>Cut out digested parts from the gel</TD></TR><TR><TD>Made and autoclaved SOB stock</TD></TR><TR><TD>SENSING</TD><TD>Acquired Tris, HCl stocks; mixed EDTA solution</TD></TR><TR><TH colspan="2">July 9, 2018</TH></TR><TR><TD>BINDING</TD><TD>Purified pUC19 gel, higher band</TD></TR><TR><TH colspan="2">July 10, 2018</TH></TR><TR><TD rowspan="2">BINDING</TD><TD>Suspended binding DNA sequence</TD></TR><TR><TD>Twice performed a HiFi assembly of pUC19 and binding sequence</TD></TR><TR><TD>SENSING</TD><TD>Acquired TE stock, cleaned nanodrop to hinder broken column error</TD></TR><TR><TH colspan="2">July 11, 2018</TH></TR><TR><TD rowspan="4">BINDING</TD><TD>Transformed pUC19 with insert into E. coli - plated on LB/amp plate with inoculation tubes</TD></TR><TR><TD>Made LB and LB/Cm plates</TD></TR><TR><TD>Made stocks: Cm, 10% glycerol, 1M glucose (respectively)</TD></TR><TR><TD>Autoclaved glycerol, glucose, and EDTA stocks, and glass beads</TD></TR><TR><TD rowspan="2">SENSING</TD><TD>Designed M13 primers</TD></TR><TR><TD>Began preparing InterLab materials</TD></TR><TR><TH colspan="2">July 12, 2018</TH></TR><TR><TD rowspan="3">BINDING</TD><TD>Created 9 freezer stocks from the pUC19/binding E. coli, and left overnight</TD></TR><TR><TD>Made LB/Cm and LB/Amp plates</TD></TR><TR><TD>Made and autoclaved SOC</TD></TR><TR><TD>SENSING</TD><TD>Trained the 96-plate reader, protocol to come</TD></TR><TR><TH colspan="2">July 13, 2018</TH></TR><TR><TD rowspan="3">BINDING</TD><TD>Miniprepped some of the 9 stocks, accidentally discarded product of stocks 1-3</TD></TR><TR><TD>Resuspended parts for InterLab (On plate 7 in the delivery kit, wells 2B, 2D, 2F, 2H, 2J, 2L, 2N, 2P); transformed parts for InterLab into E. coli</TD></TR><TR><TD>Prepared and labeled LB/Cm plates</TD></TR><TR><TD rowspan="2">SENSING</TD><TD>Received sensing sequence</TD></TR><TR><TD>Plated E. coli on LB plate for electrocompetent cells</TD></TR><TR><TH colspan="2">July 16, 2018</TH></TR><TR><TD rowspan="5">BINDING</TD><TD>Suspended sensing sequence and plasmid [concentration of 5ng/uL (added 800uL of TE)]</TD></TR><TR><TD>Transformed sensing plasmid [LB/Amp plates - inoculation tubes; no neg control]</TD></TR><TR><TD>Cut pUC19-binding plasmid DNA from Fri miniprep [HindIII, BamHI]</TD></TR><TR><TD>Replated transformed iGEM parts for InterLab because we were unsure that they were labeled correctly [LB/Cm plates --&gt; inoculation tubes]</TD></TR><TR><TD>Calibration 1 procedure for InterLab</TD></TR><TR><TD rowspan="3">SENSING</TD><TD>Resuspended and transformed sensing sequence</TD></TR><TR><TD>Took screenshots of 96-well plate process to assemble a protocol</TD></TR><TR><TD>Autoclaved glassware and beads</TD></TR><TR><TH colspan="2">July 17, 2018</TH></TR><TR><TD rowspan="3">BINDING</TD><TD>Created liquid cultures (all in shaking incubator to grow overnight)</TD></TR><TR><TD>4 sensing plasmid cultures, 2 cultures for each iGEM part, 2 cultures for electrocompetent cell</TD></TR><TR><TD>Gel cut of pUC19 vector [<B>Figures 5-6</B>]</TD></TR><TR><TD rowspan="3">SENSING</TD><TD>96-well plates and M13 primers ordered</TD></TR><TR><TD>Planning for CU meet-up (next Wednesday)</TD></TR><TR><TD>Started electrocompetent cells</TD></TR><TR><TH colspan="2">July 18, 2018</TH></TR><TR><TD rowspan="3">BINDING</TD><TD>Made SOB</TD></TR><TR><TD>PCR, first of the pUC19 with the binding insert, then with the binding insert by self, using M13 primers [<B>Figures 7-8</B>]</TD></TR><TR><TD>PCR labels 1-2, 9: IDT insert (not in pUC19); PCR labels 3-8: pUC19 cultures</TD></TR><TR><TH colspan="2">July 19, 2018</TH></TR><TR><TD rowspan="2">BINDING</TD><TD>Interlab Calibration 2</TD></TR><TR><TD>Made LB in autoclave</TD></TR><TR><TH colspan="2">July 20, 2018</TH></TR><TR><TD>BINDING</TD><TD>Made PBS</TD></TR><TR><TH colspan="3">July 23, 2018</TH></TR><TR><TD rowspan="3">BINDING</TD><TD>Interlab Calibration 3</TD></TR><TR><TD>Made LB/Cm plates, cultures</TD></TR><TR><TD>Cut PCR of Cadmium Binding peptide (CBP) with BamHI and HindIII for ligation into pMAL</TD></TR><TR><TH colspan="3">July 24, 2018</TH></TR><TR><TD rowspan="3">BINDING</TD><TD>Ligation of binding peptide into pMAL</TD></TR><TR><TD>Cell measurement procedure for Interlab finished</TD></TR><TR><TD>Diluted and plated for CFU Interlab</TD></TR><TR><TH colspan="2">July 25, 2018</TH></TR><TR><TD rowspan="2">CU MEET-UP</TD><TD>See &quot;Team &gt;&gt; Collaborations&quot; for more details</TD></TR><TR><TD>Counted colonies on plates for CFU in Interlab</TD></TR><TR><TH colspan="2">July 26, 2018</TH></TR><TR><TD>BINDING</TD><TD>Compiled all Interlab material</TD></TR><TR><TH colspan="2">July 27, 2018</TH></TR><TR><TD rowspan="2">BINDING</TD><TD>Transformed ligation of pMAL and CBP</TD></TR><TR><TD>Plate on LB/Amp plates (2), negative control (left to grow over the weekend)</TD></TR><TR><TH colspan="2">July 30, 2018</TH></TR><TR><TD colspan="2">Moved labs</TD></TR><TR><TD rowspan="2">BINDING</TD><TD>Made 10 liquid cultures from transformation performed 7/27</TD></TR><TR><TD>Made and autoclaved LB, autoclaved beads</TD></TR><TR><TH colspan="2">July 31, 2018</TH></TR><TR><TD rowspan="3">BINDING</TD><TD>Cultures did not grow (There is a chance that there is Cm in the LB used)</TD></TR><TR><TD>Cd stock made (1 g/L)</TD></TR><TR><TD>Made more liquid cultures (using other LB): 10 of pMAL with CBP, 5 from sensing</TD></TR><TR><TH colspan="2">August 1, 2018</TH></TR><TR><TD>BINDING</TD><TD>Miniprep pMAL/CBP cultures</TD></TR><TR><TD>SENSING</TD><TD>Run the initial test of sensing (50 ug/L of Cd)</TD></TR><TR><TH colspan="2">August 7, 2018</TH></TR><TR><TD>BINDING</TD><TD>Cut pMAL/CBP plasmid with BtsaI (overnight)</TD></TR><TR><TH colspan="2">August 8, 2018</TH></TR><TR><TD>BINDING</TD><TD>Take out restriction enzyme reaction</TD></TR><TR><TH colspan="2">August 9, 2018</TH></TR><TR><TD rowspan="2">BINDING</TD><TD>Running Gel for BtsaI Digested pMAL with binding peptide [<B>Figures 9-10</B>]</TD></TR><TR><TD>Digested normal pMAL with BtsaI</TD></TR><TR><TH colspan="2">August 10, 2018</TH></TR><TR><TD rowspan="2">BINDING</TD><TD>Running Gel for Digested pMAL with and without CBP insert [<B>Figures 9-10</B>]</TD></TR><TR><TD>(Digested with Btsal)</TD></TR><TR><TH colspan="2">August 15, 2018</TH></TR><TR><TD>BINDING</TD><TD>Second PCR of the binding peptide conducted, with a total of 6 PCR reactions performed</TD></TR><TR><TH colspan="2">August 16, 2018</TH></TR><TR><TD>BINDING</TD><TD>Running gel for second PCR of binding peptide [<B>Figure 11</B>]</TD></TR><TR><TH colspan="2">August 17, 2018</TH></TR><TR><TD>BINDING</TD><TD>Digestion performed on all of the PCR's using BamHI and HindHI</TD></TR><TR><TH colspan="2">August 23, 2018</TH></TR><TR><TD>Weekly meeting to discuss future plans</TD></TR><TR><TH colspan="2">August 24, 2018</TH></TR><TR><TD>iGem track selected</TD></TR><TR><TH colspan="2">August 27, 2018</TH></TR><TR><TD>Weekly goals planned out</TD></TR><TR><TD>SENSING</TD><TD>First kinetic scan planned</TD></TR><TR><TH colspan="2">August 28, 2018</TH></TR><TR><TD>SENSING</TD><TD>2000 ug/mL Cd<SUP>+2</SUP> stocks prepared for both LB AMP and IB CM</TD></TR><TR><TH colspan="2">August 29, 2018</TH></TR><TR><TD>SENSING</TD><TD>Single Colonies were selected from the sensing transformation plates, the 1.1.1 plate from the last portion of the iGEM interlab, and the pMAL transformation plate</TD></TR><TR><TH colspan="2">August 30, 2018</TH></TR><TR><TD>SENSING</TD><TD>Kinetic test performed. See <B>Figure 12</B> for procedural setup. Cd<SUP>+2</SUP> concentrations were achieved via a serial dilution. The procedure on the 96 well plate can be found under<B> Project &gt;&gt; Experiments</B>. A weekly meeting was also held.</TD></TR><TR><TH colspan="2">August 31, 2018</TH></TR><TR><TD>SENSING</TD><TD>Data from the kinetics scan collected from the 96 well plate reader</TD></TR><TR><TH colspan="2">September 3, 2018</TH></TR><TR><TD>Weekly goals planned out</TD></TR><TR><TH colspan="2">September 4, 2018</TH></TR><TR><TD>SENSING</TD><TD>PST1 ordered from NEB</TD></TR><TR><TH colspan="2">September 6, 2018</TH></TR><TR><TD>SENSING</TD><TD>Weekly meeting held. Data analysis done for the first Kinect Scan. The results can be found under <B>Project &gt;&gt; Results</B>.</TD></TR><TR><TH colspan="2">September 7, 2018</TH></TR><TR><TD>Next week planned out to ligate the CadC into the iGEM vector and to ligate the CBP into pMAL</TD></TR><TR><TH colspan="2">September 10, 2018</TH></TR><TR><TD>BINDING</TD><TD>pMAL refrigerator colonies were grown overnight to be mini prepped.</TD></TR><TR><TD>SENSING</TD><TD>A digest was performed on the iGEM linearized plasmid and the CadC sequence using EcoRI and BamHI. The digests were kept overnight at 42°C.</TD></TR><TR><TH colspan="2">September 11, 2018</TH></TR><TR><TD>BINDING</TD><TD>pMAL overnights spun down and then mini prepped. The pMAL was digested with BamHI and HindI.</TD></TR><TR><TH colspan="2">September 12, 2018</TH></TR><TR><TD>BINDING</TD><TD>Gel electroporation performed on the digests from 9/10 and 9/11. See <B>Figure 13</B>. Gel purifications were performed on all of the gelled digests.</TD></TR><TR><TD>SENSING</TD><TD>Ligation performed between the CadC sequence and the iGEM backbone. The ligation was transformed and left on  plate overnight. </TD></TR><TR><TH colspan="2">September 13, 2018</TH></TR><TR><TD>Plate from the ligation on 9/12 was viewed, and showed no colony growth. A weekly meeting was held.</TD></TR><TR><TH colspan="2">September 14, 2018</TH></TR><TR><TD>iGEM abstract completed and submitted</TD></TR><TR><TH colspan="2">September 17, 2018</TH></TR><TR><TD>Week planned out for another ligation attempt</TD></TR><TR><TH colspan="2">September 19, 2018</TH></TR><TR><TD>SENSING</TD><TD>Overnight digest of the iGEM linearized fragments performed with EcoRI and BamHI. The same was done for the IDT CadC product. Another ligation was attempted between the igem plasmid and the sensing digest.</TD></TR><TR><TH colspan="2">September 20, 2018</TH></TR><TR><TD>BINDING</TD><TD>Separate gel ran for the digest CBP PCR. Gel purification was performed on all of the digested DNA portions.</TD></TR><TR><TD>SENSING</TD><TD>PCR primers designed to amplify the sensing part. Another gel electrophoresis was performed on the sensing ligation, pMAL digest, Sensing Digest, and iGEM digest. The amount of DNA used in this gel was higher. See <B>Figure 14</B>.</TD></TR><TR><TH colspan="2">September 21, 2018</TH></TR><TR><TD rowspan="2">SENSING</TD><TD>iGem PCR and sensing PCR primers ordered. A ligation was attempted using the digested PCR and digested pMAL.</TD></TR><TR><TD>A gel of the ligation, pMAL digest, pMAL, and Sensing Digest was run. Gelled binding PCR digest. Gel fragments collected. This gel used higher amounts of dna by adding a greater amount of DNA. See <B>Figures 15 and 16</B>.</TD></TR><TR><TH colspan="2">September 24, 2018</TH></TR><TR><TD>SENSING</TD><TD>Due to an inability to get into the lab the ligation was left for 3 days. The ligation was heat treated and then transformed. A PCR using the sensing primers was performed.</TD></TR><TR><TH colspan="2">September 25, 2018</TH></TR><TR><TD>SENSING</TD><TD>	The transformation resulted in no growth. A PCR of the sensing part was run. 	</TD></TR><TR><TH colspan="2">	September 26, 2018	</TH></TR><TR><TD>	Lab dishes done, and lab cleaned. Overnight colonies were made for the iGem positive control and the sensing plasmid. 	</TD></TR><TR><TH colspan="2">	September 27, 2018	</TH></TR><TR><TD>	SENSING	</TD><TD>	The overnight colonies were spun down and mini-prepped. A gel with the miniprep and the PCRs from Monday were run. A PCR was performed on the iGem and sensing plasmids. The sensing PCR had a modification done to the annealing temperature. A gel was ran of the PCR reactions. See <B>Figures 17 and 18</B>.	</TD></TR><TR><TH colspan="2">	September 28, 2018	</TH></TR><TR><TD>	SENSING	</TD><TD>	Another PCR was run for both the sensing and iGem vector PCR with lower primer concentrations	</TD></TR><TR><TH colspan="2">	October 1, 2018	</TH></TR><TR><TD>	SENSING	</TD><TD>	Gel run for the PCRs from 9/28/18	</TD></TR><TR><TH colspan="2">	October 3, 2018	</TH></TR><TR><TD>	SENSING	</TD><TD>	Weekly meeting held and next few weeks planned out. Another PCR of the sensing was run with m13 primers, plus a PCR of the iGem. Overnight colonies were prepared for a second sensing kinetic test. Filtered LB medium was prepared and used to grow up the sensing colonies for the test. A gel was run for the 2 PCRs. See <B>Figure 19</B>.	</TD></TR><TR><TH colspan="2">	October 4, 2018	</TH></TR><TR><TD>	SENSING	</TD><TD>	None of the sensing colonies grew. No OD600 was done due to lack of cuvettes. The positive and negative control did grow. A PCR was set up to attempt the sensing PCR again, the PCR was then gelled. The successful iGem PCR from 10/3 and the PCR from sensing 8/4 were digested with the respective enzymes and left overnight. See <B>Figure 20</B>.	</TD></TR><TR><TH colspan="2">	October 5, 2018	</TH></TR><TR><TD>	SENSING	</TD><TD>	A PCR was run to get more sensing and iGem DNA. The PCR product was run in a gel. The products where amplification was seen were then cut with EcoRI, PstI, and DpnI (for the iGem PCRs only). See <B>Figure 21</B>.	</TD></TR><TR><TH colspan="2">	October 6, 2018	</TH></TR><TR><TD>	SENSING	</TD><TD>	We ran a gel and gel purified the overnight digestion of the iGEM and sensing PCRs from 10/5. Then ligated the cut PCR products from 10/3. Then, the ligation product was transformed. At the same time as the transformation, the ligation product was run in a gel. See <B>Figure 22</B>.	</TD></TR><TR><TH colspan="2">	October 10, 2018	</TH></TR><TR><TD>	SENSING	</TD><TD>	Overnight colonies were prepared for a kinetic scan 	</TD></TR><TR><TH colspan="2">	October 11, 2018	</TH></TR><TR><TD>	SENSING	</TD><TD>	Kinetic Scan 2.2 was performed with the test set up in the well plate reader described in <B>Figure 23</B>.	</TD></TR></TBODY></TABLE><P style="“font-size:15px;text-align:center&quot;"><B>Figure 1:</B> pMAL Miniprep results - 5% agarose, 1 kb ladder - Conducted on June 19, 2018</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 2:</B> pUC19 Miniprep results - 5% agarose, 1 kb ladder - Conducted on June 28, 2018</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 3:</B> pMAL Miniprep results - 5% agarose, 1 kb ladder - Conducted on June 29, 2018</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 4:</B> Digested pUC19 results - 5% agarose, 1 kb ladder - Conducted on July 5, 2018</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 5:</B> pUC19 binding digestion Miniprep results - 5% agarose, 1 kb ladder - Conducted on July 17, 2018</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 6:</B> pUC19 binding digestion Miniprep results - 5% agarose, 1 kb ladder - Conducted on July 17, 2018</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 7:</B> pUC19 with binding PCR results - 5% agarose, 1 kb ladder - Conducted on July 19, 2018</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 8:</B> pUC19 with binding PCR results - 5% agarose, 1 kb ladder - Conducted on July 19, 2018</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 9:</B> pMAL with cadmium binding peptide (CBP) insert results - 5% agarose, 1 kb ladder - Conducted August 9-10, 2018 - Colonies are a product of the transformation of the pMAL-p4x and CBP ligation - All columns are digested colonies (digested with Bts1 from NEB) except the first column to the right of the ladder.</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 10:</B> pMAL with cadmium binding peptide (CBP) insert results - 5% agarose, 1 kb ladder - Conducted August 9-10, 2018 - Colonies are a product of the transformation of the pMAL-p4x and CBP ligation - All columns are digested colonies (digested with Bts1 from NEB) except the first column to the right of the ladder.</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 11:</B> Ligation between cadC and iGEM backbone - 5% agarose, 1 kb ladder - Conducted August 9-10, 2018 - Colonies are a product of the transformation of the pMAL-p4x and CBP ligation - All columns are digested colonies (digested with Bts1 from NEB) except the first column to the right of the ladder.</P><P style="“font-size:15px;text-align:center&quot;"><B>Figure 12:</B> cadmium binding peptide (CBP) PCR 1 - 5% agarose, 1 kb ladder - Conducted August 9-10, 2018 - Colonies are a product of the transformation of the pMAL-p4x and CBP ligation - All columns are digested colonies (digested with Bts1 from NEB) except the first column to the right of the ladder.</P><P><B> Figure 13: </B> PCR product of CBP 1 - 1% agarose </P><P><B> Figure 19: </B></P><P><B> Figure 20: </B> PCR product of iGEM and cadC - 1% agarose </P><P><B> Figure 21: </B> gel of restriction digest of the PCR products - 1% agarsoe </P><P><B> Figure 22 </B> Ligation of the PCR of iGEM backbone and cadC - 1% agarose </P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>