<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Fudan_D_Background"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Fudan_D/Background">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Fudan_D/Background&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Fudan_D/Background&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Fudan_D/Background&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Fudan_D/Background" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Fudan D/Background</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Background</TITLE></P><DIV id="header"><DIV class="wrapper-header"><UL id="fudan_nav" class="sf-menu"><LI><A href="https://2012.igem.org/Team:Fudan_D" class="fudan_fix">NAVIGATION</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Home" class="fudan_fix">HOME</A></LI><LI class="current_page_item"><A href="https://2012.igem.org/Team:Fudan_D/Background" class="fudan_fix">PROJECT</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Ribozyme">Ribozyme</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Intron">Intron</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Modeling_Result">Modeling &amp; 

Result</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Submitted_Parts.html">Submitted 

Parts</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Safety">Safety</A></LI></UL><LI><A href="https://2012.igem.org/Team:Fudan_D/Page1" class="fudan_fix">TEAM</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Page1">Members</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Fudan_D">Official Team Profile</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/notebook.html" class="fudan_fix">NOTEBOOK</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Human_Practice" class="fudan_fix">HUMAN PRACTICE</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Contact" class="fudan_fix">CONTACT</A></LI><LI><A href="https://2012.igem.org/Team:Fudan_D/Attributions" class="fudan_fix">ATTRIBUTIONS</A></LI></DIV></DIV><DIV id="main"><DIV class="page-title-holder"><DIV class="page-title">BACKGROUND</DIV></DIV><DIV class="wrapper"><DIV class="page-content"><DIV class="one-half"><H4>Telomere</H4><H5 class="accordion-trigger custom"><A href="#">Overview</A></H5><DIV class="accordion-container"><DIV class="block"><P>The telomere refers to the DNA sequence in the chromosome end. Telomeric 

DNA in most eukaryotes consists of short tandem repeats. In humans the telomeric repeat sequence is 5’- TTAGGG-3’, while in 

budding yeast telomeric DNA contains degenerate repeat sequences with the consensus 5’-(TG)0–6TGGGTGTG(G)-3’. The length 

of the duplex telomeric tract ranges from &lt;30 bp in some ciliates, to 200–300 bp in budding yeast, to 5–15 kb in humans, 

and up to ~50 kb in mice.(Lingner 2006)</P></DIV></DIV><H5 class="accordion-trigger custom"><A href="#">Function</A></H5><DIV class="accordion-container"><DIV class="block"><P>Telomeres are critical at the cellular level for genome stability. The following 

roles of telomeres are well established:</P><P>First, telomeres can regulate the lifespan of cells. Telomere shortening of telomere that accumulate 

following an excessive number of cell division cycles induce cellular senescence.</P><P>Second, telomeres protect natural chromosome ends from unwanted DNA repair activities.</P><P>Third, telomeres establish a heterochromatic state at chromosome ends. Abnormal telomeric chromatin 

states have been linked to severe stochastic telomere sequence loss phenotypes, suggesting crucial roles of this structure 

during telomere replication.(Lingner 2009)</P></DIV></DIV><H5 class="accordion-trigger custom"><A href="#">The DNA end replication 

problem</A></H5><DIV class="accordion-container"><DIV class="block"><P>“All eukaryotes, and a few prokaryotes, keep their genomes in the form of 

linear DNA molecules. This structure requires special mechanisms to fully replicate DNA ends due to the following reasons: 

First, DNA replication is semiconservative; DNA polymerases use a parental template strand to synthesize a complementary 

daughter molecule. Eukaryotic telomeres end with 3’ protrusions at both chromosomal ends. Thus, the 5’ parental strand is 

resected and it cannot provide a template for the synthesis of a 3’ overhang. Semiconservative replication of the 5’-end-

containing strand during leading strand synthesis should lead to loss of the 3’ overhang. However, the presumed blunt end 

intermediate is not detected and the templating strand is probably quickly resected to regenerate the 3’ overhang. 

Therefore, nucleolytic processing of telomere ends after semiconservative DNA replication contributes to telomere shortening.

”</P></DIV></DIV><H5 class="accordion-trigger custom"><A href="#">Telomere counting mechanism</A></H5><DIV class="accordion-container"><DIV class="block"><P>The telomeres tend to maintain a constant length in a certain cell. When the 

telomere is shorter than that of wild type cells, it will undergo a progressive elongation during each cell division; If 

there exists a over-elongated telomere (longer than normal), there will be a progressive degradation of several base pairs 

each cell division, a mechanism called Telomere counting mechanism. The counting mechanism is due to the interplay of 

telomerase activity and the DNA end replication problem.</P><P>At wild-type length in yeast (300 nucleotides), the frequency of telomere elongation was only around 6

–8%, while it increased steadily to nearly 50% upon shortening of telomeres to 100 nucleotides, which does not correlate 

well with telomere length. Thus, the extent of telomere elongation is not regulated in a length-dependent manner.</P><P>When telomerase is present but repressed at its site of action due to an over-elongated telomere, the 

telomere initially shortens at a constant pace with the successive generations. As it is approaching the vicinity of the 

regulated length, its pace of degradation progressively decreases and tends towards zero at equilibrium. In summary, when the 

telomere is over-elongated and the activity of telomerase maintain the same level, the degradation rate appears to be 

constant, independent of length.(Marcand, Brevet et al. 1999)</P></DIV></DIV><H5 class="accordion-trigger custom"><A href="#">Reference</A></H5><DIV class="accordion-container"><DIV class="block"><P>[1] Ancelin K, B. M., Bauwens S, Koering CE, Brun C, Ricoul M, Pommier JP, 

Sabatier L, Gilson E (2002). &quot;Targeting assay to study the cis functions of human telomeric proteins: evidence for inhibition 

of telomerase by TRF1 and for activation of telomere degradation by TRF2.&quot; Mol Cell Biol 22: 3474–3487.</P><P>[2] Friedland, A. E., T. K. Lu, et al. (2009). &quot;Synthetic gene networks that count.&quot; Science 

Signalling 324(5931): 1199.</P><P>[3] Dana, G. V., T. Kuiken, et al. (2012). &quot;Synthetic biology: Four steps to avoid a synthetic-

biology disaster.&quot; Nature 483(7387): 29-29.</P><P>[4] Blackburn, E. H. and C. W. Greider (1985). &quot;Identification of a specific telomere terminal 

transferase activity in Tetrahymena extracts.&quot; Cell 43: 405-413.</P><P>[5] Hug, N. and J. Lingner (2006). &quot;Telomere length homeostasis.&quot; Chromosoma 115(6): 413-425.</P><P>[6] Li, B. and A. J. Lustig (1996). &quot;A novel mechanism for telomere size control in Saccharomyces 

cerevisiae.&quot; Genes &amp; development 10(11): 1310-1326.</P><P>[7] Marcand, S., V. Brevet, et al. (1999). &quot;Progressive cis-inhibition of telomerase upon telomere 

elongation.&quot; The EMBO Journal 18(12): 3509-3519.</P><P>[8] Burrill, D. R. and P. A. Silver (2010). &quot;Making cellular memories.&quot; Cell 140(1): 13.	</P><P>[9] Ulaner, G. A. and L. C. Giudice (1997). &quot;Developmental regulation of telomerase activity in 

human fetal tissues during gestation.&quot; Molecular human reproduction 3(9): 769-773.	</P><P>[10] Tham, W. H. and V. A. Zakian (2002). &quot;Transcriptional silencing at Saccharomyces telomeres: 

implications for other organisms.&quot; Oncogene 21(4): 512.	</P></DIV></DIV></DIV><DIV class="one-half last"><H4>APPLICATIONS</H4><H5 class="toggle-trigger custom"><A href="#">Biosafety</A></H5><DIV class="toggle-container"><DIV class="block"><P>“Picture vast expanses of land, perhaps in the desert or near the 

sea, where giant ponds of synthetic microalgae pump out biofuels. Imagine seeds coated with engineered Escherichia coli 

bacteria that induce lateral root growth to reduce soil erosion, the goal of the Project Auxin team in the iGEM competition 

(see https://2011.igem.org).</P><P>These may sound like amazing innovations that have no downside, but what if some of the 

modified organisms persist and spread, despite efforts to control them? Could they disrupt the normal functions of ecosystems 

by transferring their altered DNA to other microbes? Might they, for instance, increase competition for resources, or disrupt 

crucial ecological functions?” (Genya V. Dana 2012)</P><P>While synthetic biology provides us with a promising future with endless possibilities, the 

biosafety remains The Sword of Damocles for all of us. These questions have been raised before and are always considered 

today but with no certain solution ahead. Unlike transgenic crops, synthetic microbes will be altered in more sophisticated 

and fundamental ways making them potentially more difficult to regulate, manage and monitor.</P><P>There are four areas of risk raised by Genya V. Dana et al.(Genya V. Dana 2012):</P><P>
	First, differences in the physiology of natural and synthetic organisms will affect how they interact with the 

surround ing environment.</P><P>
	Second, escaped  microorganisms have the potential to survive in receiving environments (for years if in a dormant 

state) and to compete successfully with non-modified counterparts. </P><P>
	Third, synthetic organisms might evolve and adapt quickly, perhaps filling new ecological niches. </P><P>
	Fourth, about gene transfer. Microorganisms are known for their ability to exchange genetic material with other 

organisms or to take up free DNA from the environment. 
</P></DIV></DIV><H5 class="toggle-trigger custom"><A href="#">Cell memory</A></H5><DIV class="toggle-container"><DIV class="block"><P>Biological memory can be defined as a sustained cellular response 

to a transient stimulus. There are several ways to accomplish this task. One way is through transcriptional states, which 

involve populations of molecules regulating gene expression. If the transcriptional response is bi-stable, a chemical state 

becomes defined as on or off and, given certain parameters, this state can be inherited through DNA replication and cell 

division. Another way is to utilize the DNA-based memory unit, which employs recombinase to modify DNA sequence in a site-

specific manner.(Ari E. Friedland 2009; Silver 2009)</P><P>Several kinds of counters have been constructed nowadays; we will introduce two kinds of them. 

One is termed the riboregulated transcriptional cascade (RTC) counter, which is based on a transcriptional cascade with 

additional translational regulation. The other was built by chaining together modular DNA-based counting units, using 

recombinases, such as Cre and flpe, which can invert DNA between two oppositely oriented cognate recognition sites. Both the 

constructs were built to count up to three events, and said that they both should be extensible with the use of other unique 

polymerases or recombinases, of which many are known. (Ari E. Friedland 2009)</P><P>But in mathematic model, when the number of linear memory unit becomes larger, the noise will 

be severer and impede the efficiency of memorization.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><DIV class="wrapper-footer"><UL class="footer-cols"><LI class="col"><H6>LINKS</H6></LI><LI><A href="https://igem.org/Main_Page">iGEM</A></LI><LI><A href="http://www.fudan.edu.cn/">Fudan University</A></LI><LI><A href="http://life.fudan.edu.cn/">School of Life Sciences, Fudan University</A></LI></UL><LI class="col"><H6>SPONSORS</H6></LI><LI><A href="http://www.cicc.com.cn">China International Capital Corporation (CICC)</A></LI><LI><A href="http://www.goldmansachs.com/">Goldman Sachs</A></LI><LI><A href="http://www.citigroup.com">Citigroup Inc</A></LI><LI><A href="http://www.pfizer.com">Pfizer</A></LI><LI class="col"><H6>Our Slogan</H6>   <H8>Inspire a Generation!</H8></LI><DIV class="footer-bottom"><P>Edit by Fudan-Destiny Team</P><P>Powered by luiszuno.com</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Fudan_D/Background">http://2012.igem.org/Team:Fudan_D/Background</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Fudan_D/Background" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Fudan_D/Background" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Fudan_D/Background&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Fudan_D/Background&amp;oldid=235356" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>