<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Hong_Kong_HKUST_riboregulator_characterization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Hong_Kong_HKUST/riboregulator/characterization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Hong_Kong_HKUST/riboregulator/characterization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Hong_Kong_HKUST/riboregulator/characterization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Hong_Kong_HKUST/riboregulator/characterization&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Hong_Kong_HKUST/riboregulator/characterization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Hong Kong HKUST/riboregulator/characterization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><NAV role="navigation"><UL class="access-menu"><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST">Home</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor">Pneumosensor</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor/modules">Modules description</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor/parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor/characterization">Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor/future_work">Future Work</A></LI></UL><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator">Riboregulator</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/characterization">Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/CR_TA_Feature_Page">Feature Page</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/future_work">Future Work</A></LI><LI class="indent_list"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/RNA_devices_catalog">Catalog Page</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/data">Data</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice">Human Practice</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/start-up_kit">Start-up Kit</A></LI><LI class="indent_list"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/start-up_kit/handbook">Handbook</A></LI><LI class="indent_list"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/start-up_kit/report">Report</A></LI><LI class="indent_list"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/start-up_kit/search_engine">Search Engine</A></LI><LI class="indent_list"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/start-up_kit/interview">Interview</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/safety_and_ethics">Safety and Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team/members">Members</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team/advisors">Advisors</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team/instructors">Instructors</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team/attribution">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team/acknowledgement">Acknowledgement</A></LI><LI><A href="https://igem.org/Team.cgi">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/wetlab">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/wetlab/notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/wetlab/protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/wetlab/safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/Achievements/medal_requirement">Medal Requirements</A></LI></NAV></P><DIV id="content_container"><DIV id="description_area"><H2>Riboregulator Characterization</H2></DIV><DIV class="content_1"><H3>Introduction</H3><TABLE class="content_table" align="center" valign="top"><TBODY><TR class="content_row"><TD class="content_cell"><DIV class="content_area_one_row"><P>Riboregulator is a type regulatory RNA that can regulate translation. One component of the riboregulator system, 
					cis-repressing RNA, crRNA, contains a cis-repressing sequence which is located at the 5’ of the RBS and the gene of interest.
					When the transcript is formed, the cis-repressing sequence can form a loop to form complementary base pairs with the RBS and blocking the ribosome's entry 
					to RBS. crRNA is commonly called “lock” because it “locks” the translation of proteins. When there is a lock, we need a “key”.  The taRNA is the component of the 
					system that
					act as a key. It can interact (in trans) with the cis-repressing sequence to unlock the RBS and therefore activate translation. The HKUST iGEM 2014 
					team characterized 4 riboregulator already available in the Part Registry and 1 riboregulator introduced by our team. 
					
					<B>Table 1 List of riboregulator pairs characterized by HKUST iGEM 2014 team:</B></P></DIV></TD></TR><TR><TH scope="col">Name and registry code</TH><TH scope="col">Group</TH><TH scope="col">Cognate pair</TH></TR><TBODY><TR><TD scope="row">Lock 1 <A href="http://parts.igem.org/Part:BBa_J01010">BBa_J01010 </A> Key 1<A href="http://parts.igem.org/Part:BBa_J01008">BBa_J01008 </A></TD><TD>iGEM 2005_Berkeley (Golden Bear)</TD><TD>Yes</TD></TR><TR><TD scope="row">Lock 3 <A href="http://parts.igem.org/Part:BBa_J01080">BBa_J01080 </A> Key3 <A href="http://parts.igem.org/Part:BBa_J01086">BBa_J01086 </A></TD><TD>iGEM 2005_Berkeley (Golden Bear)</TD><TD>Yes</TD></TR><TR><TD scope="row">Medium lock <A href="http://parts.igem.org/Part:BBa_K175031">BBa_K175031</A> Key for medium lock <A href="http://parts.igem.org/Part:BBa_K175032">BBa_K175032 </A></TD><TD>iGEM09_TUDelft</TD><TD>Yes</TD></TR><TR><TD scope="row">Lock 3c <A href="http://parts.igem.org/Part:BBa_J23031">BBa_J23031 </A> Key 3c<A href="http://parts.igem.org/Part:BBa_J23008">BBa_J23008 </A></TD><TD>iGEM 2006_Berkeley</TD><TD>No</TD></TR></TBODY></TBODY></TABLE><P>Riboregulators have cognate pairs. For certain crRNA, there is a corresponding taRNA that can activate and “unlock” the repression by crRNA. We originally thought that 
		Lock 3c and Key 3c (Table 1.) were cognate pairs, but they turned out to be that the iGEM 2006_Berkeley simply made variants of Lock 3 and Key 3. They put an 
		alphabet at the end of the name every time they produced different variants of lock 3 and key 3. The lock 3 and key 3 variants were created independently from each other so 
		the letters at the end of name does not mean correspondence. Other teams should take note of this when they consider using riboregulator variants from iGEM 2006_Berkeley.</P></DIV></DIV><DIV class="content_1"><H3>Riboregulator Results</H3><TABLE class="content_table" align="center" valign="top"><TBODY><TR class="content_row"><TD class="content_cell"><DIV class="content_area_one_row"><H5 style="font-size: 13px">Figure 1. Fluorescence (F)/OD600 measurements of riboregulator pairs after arabinose induction and their corresponding controls. </H5><H6 style="font-size: 13px"> All samples were inoculated in M9 minimal salt medium overnight in no or various arabinose concentrations (%w/v). The samples were diluted around 10 fold the next day. Measurements were made when the samples reached around the mid-log phase (OD600 = 0.3 to 0.5). 
						Graphs depict the triplicate mean + standard deviation. (A) Schematic diagram of the genetic context of the experiment. Note that the diagram generalized the CR and TA sequences. (B) Measurement for Lock 1 (<A href="http://parts.igem.org/Part:BBa_J01010">BBa_J01010</A>) and Key 1 (<A href="http://parts.igem.org/Part:BBa_J01008">BBa_J01008</A>) cognate pair. (C) Measurement for Lock 3(<A href="http://parts.igem.org/Part:BBa_J01080">BBa_J01080</A>) and Key 3 (<A href="http://parts.igem.org/Part:BBa_J01086">BBa_J01086</A>) cognate pair. (D) Measurement for Medium lock (<A href="http://parts.igem.org/Part:BBa_K175031">BBa_K175031</A>) and Key for medium lock (<A href="http://parts.igem.org/Part:BBa_K175032">BBa_K175032</A>) cognate pair. (E) Measurement for Lock 3c (<A href="http://parts.igem.org/Part:BBa_J23031">BBa_J23031</A>) and Key 3c (<A href="http://parts.igem.org/Part:BBa_J23008">BBa_J23008</A>). </H6><P>
To characterize the different riboregulator pairs, we kept the genetic context identical except for the various cr-repressing sequences, trans-activating sequences and the RBS. The 
RBS sequence also had to be different for some of the riboregulator systems because the cr-repressing sequence depends on the RBS sequence. In order to repress translation, the 
cis-repressing sequence need to interact with the RBS, and so the interaction depends on the sequences. Since different teams used different RBS to design their cis-repressing 
sequences, we also had to use corresponding RBS for characterization. We had a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_J23102">BBa_J23102 </A>) to drive the 
expression of the cis-repressed GFP translation unit. For the expression taRNA, we wanted to control the expression and therefore we decided to use the arabinose inducible 
P<SUB>BAD</SUB> promoter (<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500 </A>). The promoter was chosen because the 3’ end after the transcription start site of the 
promoter is short. Longer 3’ end can affect the function of the taRNA (Isaacs et al., 2004) (Figure 1. A). 
		
</P><P>For the riboregulator system to work, the repression of GFP synthesis needs to be first observed when the cis-repressing sequence is added 5’ of the RBS of the system. 
		Significant repression can be seen in Lock 1-Key1, Lock 3-Key 3, and Medium lock (Lock m)-Key for medium lock (Key m) cognate pairs (Figure 1. B, C, D respectively). Almost 
		full repression was observed for the three cognate pairs. For Lock 3c-Key 3c pair, we do not see repression when cis-repressing sequence is introduced to the system. Instead,
		converse can be observed. When we don’t have cis-repressing sequence, we see significant drop in the fluorescence (Figure 1.E). One possible reason could be that the RBS 
		sequence that we used for the controls of Lock 3c Key 3c was incorrect. For Lock3c the target RBS sequence was not mentioned. It seemed like a variation of 
		<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034 </A> with shorter 3’ end. In order to build the construct type 2 and 3 (Figure 1. A), the RBS sequence had to be 
		deduced from the Lock 3c sequence. From the Lock 3c sequence, we have used a part of sequence that resembled the RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034
		</A>). The RBS sequence used may have been too short to be functional. Therefore, no fluorescence is observed when cis-repressing sequence is not present. On the other hand, 
		fluorescence can be observed when cis-repressing sequence is present because firstly, the RBS is sequence is correct, and secondly because the cis-repressing sequence failed 
		to repress the translation. </P><P>
After repression, the system needs to be activated when taRNA is expressed. After the addition of arabinose, taRNA is expressed. Out of the three cognate riboregulator pairs that 
were repressed, only two showed significant increase after arabinose induction. Lock 1- Key 1 cognate pair showed around 13-fold increase for both 1% and 2.5% (%w/v) arabinose 
induction. Lock 3- Key 3 cognate pair showed around 1.5 and 3 fold increase for 1% and 2.5% of arabinose induction respectively. Lock 1-Key 1 and Lock 3- Key 3 behaved differently 
for different concentrations of arabinose induction. Full induction was observed at 1% arabinose for Lock 1- Key 1 cognate pairs while full induction for Lock 3- Key 3 was observed at
 2.5% arabinose. Statistically, no significant fold increase could be observed for Lock m- Key m cognate pair. </P></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="content_1"><H3>Discussion</H3><TABLE class="content_table" align="center" valign="top"><TBODY><TR class="content_row"><TD class="content_cell"><DIV class="content_area_one_row"><P><I>The fold increase for Lock 1- Key 1 is lower than that of riboregulator pair mentioned in the Isaacs et al.’s paper.   </I>

iGEM 2005_Berkely, when they first introduced the riboregulator system to the iGEM community, they  mentioned in the Part Registry page that Lock 1 and Key 1 are “Biobricked 
version of Isaacs’ riboregulator” crR12 and taR12 respectively. We can therefore expect the Lock 1 and Key 1 fold increase after induction to be similar to what Isaacs et al. have 
observed and mentioned in the paper. Isaacs et al. mention that for crR12 and taR12 cognate pair, they have observed 19-fold increase (Isaacs et al., 2004), our results showed only 
showed around 13 fold increase. One possibility for the deviation could be because the Lock 1 and Key 1 sequences are not 100% match with crR12 and taR12. Because scars are 
introduced in 5’ and 3’ end of a Biobricked parts, iGEM 2005_Berekeley had to shorten the original 5’ and 3’ end of the crR12 and taR12. This change actually changes the crRNA and 
taRNA sequence and therefore could result to the deviated results. Another possibility is simply because the genetic context that we did our characterization was different from what 
Isaac et al. used to characterize their riboregulator system. If this is the case, we can at least see that the system is not very modular: changing the genetic context can change 
the fold increase.  <I>The fluorescence after induction is still low compared to that of unrepressed controls.   </I>

Although we saw significant fold increase for two riboregulator systems that we have characterize, compared to the fluorescence of unrepressed controls, the fluorescence is very low.
 For Lock 1-Key 1 riboeregulator system, the fluorescence after induction only correspond to around 0.4% of the that of the unrepressed control. For the Lock 3-Key3 system, 
 the value was around 0.3%. The lower expression partly is because of the lower mRNA levels. After the introduction of the cis-repressing sequence the mRNA level was 40% of 
 that of the controls (Isaacs et al., 2004). Another reason for low fluorescence after induction could have resulted because of our genetic context. We have used a relatively 
 strong constitutive promoter (<A href="http://parts.igem.org/Part:BBa_J23102">BBa_J23102</A>) to express the crRNA and a relatively weak arabinose inducible promoter 
 (<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500 </A>). This could have caused imbalance of crRNA and taRNA levels. We could have had lower taRNA level and therefore failed
 to fully activate the riboregulator system. Further investigation is required. Simply changing the arabinose inducible promoter to a strong promoter can tell us whether this is the 
 case. <I>Different Lock-Key cognate pairs behaved differently to different arabinose concentration.</I>

Lock 1- Key 1 riboregulator cognate pair was fully induced and leveled off at 1% arabinose concentration. For Lock 3- Key 3 pair, the full induction was observed at 2.5% arabinose
 concentration. We did not conduct further investigation to understand the difference in the response. Further investigation on how changing the riboregulator sequence can change
 the sensitivity of the system could be an interesting study.
</P></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="content_1"><H3>Methods</H3><TABLE class="content_table" align="center" valign="top"><TBODY><TR class="content_row"><TD class="content_cell"><DIV class="content_area_one_row"><P><I>Fluorometry </I>

Triplicate of each sample were inoculated overnight in a deep-well 96 well plate. M9 minimal salt solution was used for the inoculation because it gives low background fluorescence.
  1% and 2.5% (w/v) arabinose concentration was used for overnight induction. Samples were diluted around 10-fold the next day and regrown to mid-log phase (OD600=0.3 ~ 0.6). 200µl 
  of sample were drawn out from the deep-well plate and plated on a clear round bottom plate for measurement. For each sample, OD<SUB>595</SUB> and Fluorescence was measured using 
  EnVision Multilabel Reader. The excitation wavelength was 485/14nm and the emission wavelength was 535/25nm. Conversion factor for OD<SUB>595</SUB> to OD<SUB>600</SUB> was obtained
  by calculating the slope of the OD<SUB>600</SUB> v.s. OD<SUB>595</SUB> graph. The conversion factor, 1.24 was multiplied to OD<SUB>595</SUB> reading to covert the measurements to
  OD<SUB>600</SUB>. Background fluorescence was subtracted for each fluorescence measurement by making another standard Fluorescence v.s. OD<SUB>600</SUB> graph. DH10B with 
  pSB3K3-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A> was used to produce the standard curve. The corrected fluorescence was then divided by the corresponding 
  OD<SUB>600</SUB>. Finally the average and standard deviation were calculated.    

</P></DIV></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV id="content_container"><DIV id="description_area"><H2>P<SUB>BAD</SUB> Characterization<B>(Please note that the following data has been shown to be problematic because our new data do not match our old data. We will update the following information as soon as possible - HKUST iGEM 2015.23Jan2015)</B></H2></DIV><DIV class="content_1"><H3>Introduction</H3><TABLE class="content_table" align="center" valign="top"><TBODY><TR class="content_row"><TD class="content_cell"><DIV class="content_area_one_row"><P>P<SUB>BAD</SUB> promoter is an arabinose inducible promoter. In nature, the promoter exist in the arabinose operon to regulate the transcription of araB, araA, and araD. 
		The arabinose operon or the ara operon encodes enzymes needed or the catabolism of arabinose to xylulose 5- phosphate which is an intermediate of the pentose phosphate pathway
		. The Pc promoter which is adjacent to the P<SUB>BAD</SUB> promoter transcribes the araC gene in the opposite direction. AraC protein is responsible to repress the activity of
		the P<SUB>BAD</SUB> promoter when arabinose is absent. Once arabinose is present,the AraC protein binds to the arabinose and dimerize. The dimerize form of AraC-arabinose can
		activate the P<SUB>BAD</SUB> promoter (Schleif, 2010).</P></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="content_1"><H3>P<SUB>BAD</SUB> promoter <A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500 </A> in Part Registry</H3><TABLE class="content_table" align="center" valign="top"><TBODY><TR class="content_row"><TD class="content_cell"><DIV class="content_area_one_row"><P> 

There are several P<SUB>BAD</SUB> promoters in the Part Registry. The promoter that we were interested in was <A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A> because of 
two reasons. First, <A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>, along with P<SUB>BAD</SUB>, has araC gene is regulated by the Pc promoter. Without the AraC, the 
repression and induction of P<SUB>BAD</SUB> can only work on strains that are AraC+. By coupling the araC gene with the P<SUB>BAD</SUB> promoter, we can be free from such restraints.
 Second, <A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A> needed debugging. <A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>, although it is useful, it is not
 requestable because of inconsistency in sequencing. Also, in the experience page, two teams, Groningen 2011 and Cambridge 2011 had some discrepancy on how the promoter responded
 to the arabinose induction. In brief, Groningen results show that the induction of the promoter by arabinose was gradual while Cambridge results show that it was an “on-or-off” 
 response. We wanted to analyze these problems so that the part could be more reliable for other users. Cambridge cited a paper that mentioned that variation in response could have 
 resulted from cell strain variation. (Khelbnikov, 2001)</P></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="content_1"><H3>Results</H3><TABLE class="content_table" align="center" valign="top"><TBODY><TR class="content_row"><TD class="content_cell"><DIV class="content_area_one_row"><P><U>All-or-none response was observed for individual cells</U>.Flow cytometry can measure the fluorescence of individual cells. The individual measurement of cell 
		fluorescence can be plotted in a forward scatter (FSC) intensity value versus fluorescence graph. An arbitrary vertical line divided the region of the graph with low 
		fluorescence (Q3) and high fluorescence (Q4). 

For the negative control, DH10B transformed with pSB3K3-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>, we see cells distributed in Q3 while absent in Q4. 
The reverse was true for the positive control, DH10B transformed with pSB3K3-<A href="http://parts.igem.org/Part:BBa_I20260">BBa_I20260</A> (GFP generator regulated by 
<A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>).  We see most cells distributed in Q4. For our experimental sample, 
(pSB3K3-<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>) similar trend could observed for three strains. 
At no arabinose induction, we see most of the cells in Q3. The distribution of cells shifted to Q4 when arabinose was added to the medium. We also see that the cells remained in Q3 
even after arabinose induction. The bimodal distribution indicate that some cells got induced while other cells were in repressed state. 
</P><H5 style="font-size: 13px">Figure 2.  Forward scatter intensity (FSC) versus GFP graphs for samples with P<SUB>BAD</SUB> promoter regulating GFP generator. </H5><H6 style="font-size: 13px"> All samples were inoculated in M9 minimal salt medium overnight in various arabinose concentrations (%w/v). The samples were 
						diluted around 10 fold the next day. Sample were fixed and the fluorescence was measured using flow cytometer. The graphs were plotted for the control 
						constructs, pSB3K3-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A> (-) and pSB3K3-<A href="http://parts.igem.org/Part:BBa_I20260">BBa_I20260</A> 
						(<A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>) in the absence of arabinose. FSC versus GFP graphs for pSB3K3-
						<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>
						(<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>) in 0, 0.2 and 1.0% arabinose concentration were plotted.  
						Each set of graphs were obtained for three different cell strains, DH10B, DH5α and BW25113. </H6><P>The distribution can also give us some idea of P<SUB>BAD</SUB> promoter leakage in different cells strain. If the data points are on or near the boundary between Q3 and Q4, 
we know that the promoter is leaky. The results indicate that the promoter is relatively more leaky in DH10B compared to other cell strains. 

The percentages of cells in each Q3 and Q4 are highlighted in Figure 2. After addition of arabinose, we clearly see more percentage in Q4. Around 80% of the cells for DH10B 
and BW25113 are in Q4. Relatively lower percentage of cells in Q4 is observed for DH5α. Only around 60 to 70% of cells were in Q4 after arabinose induction. The shift in the 
distribution from Q3 to Q4 also corresponded to the increase of RPU across different arabinose concentration. At 0% arabinose concentration, for all three strain, higher percentage 
of cells are in Q3 and therefore the RPU is low. After arabinose induction, more cells are in Q4 and RPU is high. The percentage of cells in Q4 levels off for arabinose concentration
 that we have tested. The trend was observed for RPU measurement. 

We can also conclude that even after maximum period of induction (overnight induction) of arabinose, we still see some cells uninduced (in Q3). </P><H5 style="font-size: 13px">Figure 3. The percentage of cells in induced and uninduced state, and RPU across different arabinose concentration.  </H5><H6 style="font-size: 13px"> Q3 and Q4 represent the 3<SUP>rd</SUP> and 4<SUP>th</SUP> quadrants of the forward scatter versus GFP curve mentioned in 
						Figure 2. The experimental condition was the same as the procedure mentioned in the caption of the Figure 1. The left y-axis is for the percent of cells in
						Q3 and Q4 while the right y-axis is for RPU. Graphs depict the triplicate mean ± standard deviation. (A) Graph for pSB3K3-
						<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A> in DH10B. (B) Graph for 
						pSBK3K3-<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A> in DH5α. (C) Graph for 
						pSB3K3-<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A> in BW25113.  </H6><P><B><U>RPU of P<SUB>BAD</SUB> in different cell strains</U></B>

In order to solve the discrepancy between Groningen 2011 and Cambridge 2011, we have calculated the Relative Promoter Unit of the P<SUB>BAD</SUB> promoter in three different cell 
strains across increasing arabinose concentration. The three strains that were chosen was: DH10B, BW25113, and DH5α. Groningen 2011 used cell strain DH5α to obtain the 3-D graph 
while Cambridge 2011 used BW27783. Unfortunately, we did not had access to the strain, so we had to use another strain, BW25113, which is commonly used by synthetic biologists. 
Only DH5alpha has araC in its genome. All other strains have deletion of araC (and other genes in the L-arabinose operon). </P><H5 style="font-size: 13px">Figure 4. RPU of P<SUB>BAD</SUB> promoter in three different cell strains across different arabinose concentration. </H5><H6 style="font-size: 13px"> Relative Promoter Unit of P<SUB>BAD</SUB> promoter was calculated in three strains: DH10B, BW25113 and DH5alpha. Gradient arabinose concentration (% w/v) from 0% to 1.0% with 0.2% increments was used to test the variation of promoter strength (RPU) in different concentration of arabinose. Each strain of cells inoculated overnight in various arabinose concentration above. The cells were diluted around 10 fold and grown until they reached mid-log phase (OD<SUB>600</SUB> 0.3-0.5). Cells were fixed and fluorescence was measured using flow cytometer. The graph represent triplicate mean ±SD.  </H6><P>
We thought that Relative Promoter Unit (RPU) defined by Endy et al. would be a better a measure of promoter strength at different arabinose concentration than simply comparing the 
fluorescence measurement of different strains. This is because simply measuring and comparing fluorescence as an output can also be affected by other experimental factors such as the
genetic context. Because RPU is a ratio to fluorescence measurement, the effects caused by these factors can be minimized (See Methods for RPU calculation).   

We observed clearly a different promoter response among the three cell strains. The all-or-none response was observed for the strains, but the levels of RPU were different. 
For DH10B, RPU leveled off around 0.45. For BW25113, RPU leveled off around 0.3. For DH10B and BW25113 we see a very clear “all-or-none” response. The RPU reaches a plateau at 
0.2% of arabinose. For DH5α, it is less obvious, because the initial increase of RPU at 0.2% arabinose is around 0.1, lower than the other two strains. With the given graph, 
statically, we cannot say that the RPU level gradually increases as % arabinose increases.  

The lower RPU of DH5α was expected because DH5α has araC gene, gene for the repressor, in its genome. This could have caused the lower promoter strength of 
P<SUB>BAD</SUB> in DH5α. All in all, all three strains produced all-or-none response. 

For DH10B and BW25113 we see a very clear “all-or-none” response. The RPU reaches a plateau at 0.2% of arabinose. For DH5α, it is less obvious, because the initial increase 
of RPU at 0.2% arabinose is around 0.1, lower than the other two strains. With the given graph, statically, we cannot say that the RPU level gradually increases as % arabinose 
increases. All in all, all three strains produced all-or-none response. </P><P><B><U>P<SUB>BAD</SUB> Leakage</U></B>
Leakage was observed for P<SUB>BAD</SUB> promoter in DH10B. Although the RPU was relatively higher compared to that of other two strains, DH10B showed leakage at no arabinose 
induction. The RPU was around 0.16 for DH10B which is quite significant if we want to have a non-leaky system. For BW25113 and DH5α the leakages were below RPU of 0.1. 
The lowest leakage was observed for DH5α. The low leakage in DH5α, once again, could be explained by the additional copy of araC in the genome. The low leakage BW25113, 
however, is harder to explain. Nonetheless, if we want a low-leakage system that can reach relatively high RPU upon arabinose induction, out of the three strain we have used, 
BW25113 would be the best option. </P><P><B><U>3-D graphs for DH10B and DH5α</U></B>
We also tried to produce a 3-D graph. We, however, had some changes from the genetic context that Groningen 2011 used. At 3’ of the P<SUB>BAD</SUB> promoter, instead of using 
<A href="http://parts.igem.org/Part:BBa_E0840">BBa_E0840</A> (GFP generator), we used <A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A> (GFP generator). We also used a 
low copy pSB3K3 plasmid as the backbone instead of the high copy pSB1C3. We also could not take measurements every 15 minutes interval because we had to measure the fluorescence at 
each time point manually. Fluorescence was instead measured every two hours. Lastly, OD<SUB>600</SUB> was also monitored and graphed (Figure 4. A and B) </P><H5 style="font-size: 13px">Figure 5. Fluorescence and OD<SUB>600</SUB> measurements of DH10B and DH5α induced in different arabinose concentrations.  </H5><H6 style="font-size: 13px"> Triplicate of DH10B and DH5α samples were inoculated in deep well 96 well plate overnight in M9 minimal salt medium.
						Arabinose was added to match the final working concentration from 0 to 1.0 % (w/v) with 0.2% increments. Fluorescence and OD<SUB>600</SUB> was measured every
						two hours for ten hours. (A) Increase of OD<SUB>600</SUB> measurement for DH10B strain in different arabinose concentration. The graph represents triplicate 
						mean ± SD (B) Increase of OD<SUB>600</SUB> measurement for DH5α strain in different arabinose concentration. (C) Fluorescence VS arabinose concentration
						VS Time 3-D graph for DH10B. Each point represent triplicate mean. (D) Fluorescence VS arabinose concentration VS Time 3-D graph for DH5α. Each point 
						represent triplicate mean. </H6><P>
The OD<SUB>600</SUB> value reflects cell concentration. OD<SUB>600</SUB> for both strain in different arabinose concentration increased exponentially. Growth rate for cells without 
arabinose induction (0%) was greatest for both DH10B and DH5α. For other concentration of arabinose induction the growth rates were similar for both of the cell strains (Figure
 4. A and B). The similar growth rate across the 10 hour period indicates that the cell concentration for samples in different arabinose concentration increased similarly and 
 therefore the cell concentration at each point of measurement was similar across different arabinose concentration. We can therefore assume that differences in the cell 
 concentrations have minimal effect on the different fluorescence levels at different arabinose concentration. 

Unlike what Groningen 2011 observed in their experiment, (<A href="http://parts.igem.org/Part:BBa_I0500:Experience">Experience page.</A>) we observed all-or-none response for both 
of the cell strains. For DH10B and DH5α the fluorescence levelled off at 0.2% arabinose concentration. In the process of subtracting autofluorescence, we got some negative 
corrected fluorescence. These region correspond to the blue areas of the 3-D graphs.   


</P></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="content_1"><H3>Discussion</H3><TABLE class="content_table" align="center" valign="top"><TBODY><TR class="content_row"><TD class="content_cell"><DIV class="content_area_one_row"><P><I>Groningen 2011 results may not truly represent the gradual induction of P<SUB>BAD</SUB> promoter </I>
We believe that it is actually difficult to analyze the promoter’s gradual or all-or-none response looking the 3-dimensional graph that Groningen 2011 team presented. The 3 
dimensional graph has three parameters: time, various arabinose concentrations and fluorescence. The graph does not consider the OD<SUB>600</SUB> value which can represent the 
concentration of cells. If the growth rate of the cells are different in different arabinose concentration, the final concentration of cells at given point of time can vary. 
Groningen 2011 team did not provide with OD<SUB>600</SUB> versus time graphs. Therefore it is hard to tell well there was variation in the cell growth under the context of Groningen
 2011’s experiment. Because the fluorimetry measurements, the method that Groningen 2011 used, measures the fluorescence of the entire population of cells, the fluorescence can be 
 affected by the concentration of cells and hence show a response that is not all-or-none. Looking at Groningen’s results, it would be more appropriate to say that the population of
 cells that is transformed with plasmid containing <A href="http://parts.igem.org/Part:BBa_K607036">BBa_K607036</A>(<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500 
 </A>-<A href="http://parts.igem.org/Part:BBa_E0840">BBa_E0840 </A>) showed gradual response to increasing arabinose concentration. <I>Different genetic context could be responsible for the different response of the P<SUB>BAD</SUB> promoter. </I>

Due to lack of manpower, unfortunately, we could not produce 3-D graphs for cells with pSB1C3 backbone as controls. Assuming that Groningen 2011 result was valid, we can speculate 
that difference of the P<SUB>BAD</SUB> promoter response could have resulted from the different backbone that we have used to characterize the promoter. In fact, Cambridge 2011, 
who also observed all-or-none response (<A href="http://parts.igem.org/Part:BBa_I0500:Experience">Experience page.</A>) used pSB3K3 as backbone for their constructs. Without proper 
controls, it is hard to make a conclusion.
</P></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="content_1"><H3>Methods</H3><TABLE class="content_table" align="center" valign="top"><TBODY><TR class="content_row"><TD class="content_cell"><DIV class="content_area_one_row"><P><U><B>Construction</B></U>
1. Construct pSB3K3-<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>

2. Transforming pSB3K3-<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A> to DH10B, DH5α, and BW25113.


3. Transforming pSB3K3-<A href="http://parts.igem.org/Part:BBa_I20260">BBa_I20260</A> (Standard Constitutive Promoter/Reference Promoter) from the 2014 Distribution Kit to DH10B,
 DH5α, and BW25113.

4. Transforming pSB3K3<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A> (GFP generator) from the 2014 Distribution Kit to DH10B, DH5α, and BW25113.</P><P><U><B>Measurement</B></U>
1. Preparing supplemented M9 medium (M9 Minimal salt medium protocols could be seen on the <A href="https://2014.igem.org/Team:Hong_Kong_HKUST/wetlab/protocols">Protocols</A>
 page, or download the <A href="https://static.igem.org/mediawiki/2014/8/8b/M9_Minimal_medium_protocol.pdf">PDF</A> file) 

2. Culturing <I>E. coli</I> DH10B strain carrying the whole construct listed on the construction part. Grow cell culture overnight (Incubate 37°C and shake for 15 hours) with M9 
minimal medium (we used Corning® 96 well storage system storage block, 2 mL, V-bottom, sterile to culture the cells, and Corning® microplate sealing tape white Rayon (with acrylic), sterile, suitable for cell/tissue culture applications, breathable sterile membrane.)

3. Take out 20-30μl of overnight cell culture (we used Multichannel Pipetman) and mix it with M9 medium and arabinose with specific concentration (0%, 0.2%, 0.4%, 0.6%, 0.8%, 1%) 
in the 96 Deep Well plate. 

4. Incubate in 37°C and shake for 3 - 4 hours.

5. Take out 200ul of cells from the 96 deep well plates, and put it on a micro test plate 96 well flat bottom. (we used Micro test plate 96 well flat bottom, made by SARSTEDT.)


6. Measuring the GFP intensity and OD<SUB>595</SUB> values (we used <B>Envision Multilabel Reader</B>) every 30 minutes after the above mentioned <I>E. coli</I> strains are 
cultured to mid-log phase (OD<SUB>600</SUB> = 0.3 - 0.5)

Filter used on Envision Multilabel Reader: 
- Absorbance :Photometric 595nm,
- Excitation :485nm FITC, 
- Emission :535nm FITC, 
- Mirror module : FITC (403) at bottom. 

- In between measurements, keep incubating the cells in 37°C while shaking. <B>OR</B>

- When cells were in the mid-log phase, cells were fixed and fluorescence was measured using <B>FACS</B>. 

7. Calculating the Relative Promoter Units (RPU) using the obtained data; </P><P><U><B>Data Processing for data from Envision Multilabel Reader</B></U>
1. After <I>E. coli</I> carrying the right construct was grown to mid-log phase, GFP intensity and OD<SUB>595</SUB> were measured every 30 minutes (up to 120min); 

2. GFP intensity are subtracted with the background fluorescence which is the fluorescence of pSB3K3-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>. Curve reflecting 
GFP expression change was plotted (from 4 measurements from time=0 to time=120); OD<SUB>595</SUB> was converted to OD<SUB>600</SUB>, and average values were taken; 

3. GFP synthesis rate was then obtained by calculating the slope of the above mentioned curve; 

4. Absolute promoter activity of P<SUB>BAD</SUB> and <A href="http://parts.igem.org/Part:BBa_I20260">BBa_I20260</A> were calculated by dividing the GFP synthesis rate with the 
average OD<SUB>600</SUB> value; 

5. Averaged absolute promoter activity was then obtained by averaging the respective 3 sets of absolute promoter activity values; 


6. Finally, R.P.U was calculated by dividing the averaged P<SUB>BAD</SUB> and absolute promoter activity over the averaged <A href="http://parts.igem.org/Part:BBa_J23101">
BBa_J23101</A> absolute promoter activity. R.P.U value of P<SUB>BAD</SUB> in different concentration of arabinose is shown. Leakage could be analyzed according to the R.P.U value 
that shows the GFP expression of P<SUB>BAD</SUB> promoter in the absence of arabinose (0%).
Equation of the RPU calculation is shown below: 

Based on this equation, the GFP synthesis rate (dF/dt) for the experiment (<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>) divided by the ABS (which means the absorbance or OD<SUB>600</SUB>) of the cells containing the experiment construct. GFP synthesis rate (dF/dt) for the reference promoter (<A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>) divided by the ABS (which means the absorbance or OD<SUB>600</SUB>) of the cells containing the reference promoter construct.

Then, RPU is obtained by dividing the GFP synthesis rate/ABS of experiment with the GFP synthesis rate/ABS of the reference promoter.


</P><P><U><B>Data Processing for data from FACS</B></U>
1. RPU was calculated by first subtracting the autofluorescence (fluorescence of cells with pSB3K3-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240 </A>)
2. Dividing the fluorescence of cells containing pSB3K3-<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500 </A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240 </A> 
with cells containing pSB3K3-<A href="http://parts.igem.org/Part:BBa_I20260">BBa_I20260 </A>.
Equation is shown below:


Based on this equation, the amount of fluorescence [F] for the experiment (<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>) multiplied by the growth rate (μ) of the cells containing the experiment construct. Amount of fluorescence [F] for the reference promoter (<A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>) multiplied by the growth rate (μ) of the cells containing the reference promoter construct.

Then, RPU is obtained by dividing the fluorescence*growth rate of experiment with the fluorescence*growth rate of the reference promoter.

3. For simplicity, we assumed the growth rate of the cells transformed with pSB3K3-<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500 </A>-<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240 </A> and cells transformed with pSB3K3-<A href="http://parts.igem.org/Part:BBa_I20260">BBa_I20260 </A> had same similar growth rate. 



				</P></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="content_1"><TABLE class="content_table" align="center"><TBODY><TR class="content_row"><TD class="content_cell" colspan="2"><DIV class="content_area_two_row"><P><U><B>References</B></U></P><P>

Schleif R. <I>AraC protein, regulation of the L-arabinose operon in Escherichia coli, and the light switch mechanism of AraC action.</I> FEMS Microbiol Rev (2010) 1–18.

Khelbnikov, A., Datsenko, K., Skaug, T., Wanner, B., &amp; Keasling, J. (2001).<I> Homogeneous expression of the P(BAD) promoter in Escherichia coli by constitutive expression of the low-affinity high-capacity AraE transporter. </I> Microbiology, 147(12), 3241-3247.

J. R. Kelly, A. J. Rubin, J. H. Davis, J. Cumbers, M. J. Czar, ..., D. Endy. (2009). Measuring the activity of BioBrick promoters using an in vivo reference standard. <I>Journal of Biological Engineering</I>, 3, 4. doi: 10.1186/1754-1611-3-4

Isaacs, F., Dwyer, D., Ding, C., Pervouchine, D., Cantor, C., &amp; Collins, J. (2004).<I> Engineered riboregulators enable post-transcriptional control of gene expression. </I> Nature Biotechnology, 841-847.
</P></DIV></TD></TR></TBODY></TABLE></DIV></DIV><FOOTER><A href="#" class="scrollToTop"><DIV class="scrollToTop">
		Back to top
		</DIV></A></FOOTER><H4><A href="https://2014.igem.org/Team:Hong_Kong_HKUST">Home</A></H4><UL class="site_list"><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor">Pneumosensor</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator">Riboregulator</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/data">Data</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice">Human Practice</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team">Team</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/wetlab">WetLab</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/Achievements">Achievements</A></LI></UL><H4><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor">Pneumosensor</A></H4><UL class="site_list"><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor/modules">Modules description</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor/parts">Parts</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor/characterization">Characterization</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor/results">Results</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/pneumosensor/future_work">Future Work</A></LI></UL><H4><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator">Riboregulator</A></H4><UL class="site_list"><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/parts">Parts</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/characterization">Characterization</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/CR_TA_Feature_Page">Feature Page</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/future_work">Future Work</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/RNA_devices_catalog">Catalog Page</A></LI></UL><H4><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice">Human Practice</A></H4><UL class="site_list"><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/start-up_kit">Start-up Kit</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/start-up_kit/handbook">Handbook</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/start-up_kit/report">Report</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/start-up_kit/search_engine">Search Engine</A></LI><LI><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/start-up_kit/interview">Interview</A></LI></UL><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/outreach">Outreach</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/human_practice/safety_and_ethics">Safety and Ethics</A></LI><H4><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team">Team</A></H4><UL class="site_list"><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team/members">Members</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team/advisors">Advisors</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team/instructors">Instructors</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team/attribution">Attributions</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/team/acknowledgement">Acknowledgement</A></LI><LI class="site_link"><A href="https://igem.org/Team.cgi?year=2014">Official Team Profile</A></LI></UL><H4><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/wetlab">WetLab</A></H4><UL class="site_list"><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/wetlab/notebook">Notebook</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/wetlab/protocols">Protocols</A></LI><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/wetlab/safety">Safety</A></LI></UL><H4><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/Achievements">Achievement</A></H4><UL class="site_list"><LI class="site_link"><A href="https://2014.igem.org/Team:Hong_Kong_HKUST/Achievements/medal_requirement">Medal Requirements</A></LI><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/characterization">http://2014.igem.org/Team:Hong_Kong_HKUST/riboregulator/characterization</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Hong_Kong_HKUST/riboregulator/characterization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Hong_Kong_HKUST/riboregulator/characterization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Hong_Kong_HKUST/riboregulator/characterization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Hong_Kong_HKUST/riboregulator/characterization&amp;oldid=405138" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></BODY></HTML>