<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Groningen_Expression"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Groningen/Expression">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Groningen/Expression&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Expression&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Expression&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Groningen/Expression" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Groningen/Expression</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><H2><SPAN class="mw-headline" id="Expression_of_chaplins">Expression of chaplins</SPAN></H2><P><B>Summary</B></P><P>The goal of our project is to let <I>Bacillus subtilis</I> make a hydrophobic coating by forming a <A href="https://2010.igem.org/Team:Groningen#/biofilm" class="external text" rel="nofollow">biofilm</A> and then expressing and secreting <A href="https://2010.igem.org/Team:Groningen/Hydrophobins#Chaplins" class="external text" rel="nofollow">chaplins</A>. However, first we needed to test whether <I>B. subtilis</I> was capable of expressing chaplins, since they could impair the cellgrowth due to their hydrophobic and self assembling properties. We succesfully expressed chaplins C, E and H in <I>B. subtilis</I> using a tightly regulated subtilin inducable system called &quot;SURE&quot;. Furthermore we tested the SURE system for optimal subtilin concentration with GFP. We want <I>B. subtilis</I> to auto-induce the expression of the chaplins after biofilmformation. Therefore we looked into two operons in <I>B. subtilis</I>; one that gets triggered in late exponential growth (<I>srfA</I> operon) and one that is involved in the formation of biofilm (<I>yqxM-sipW-tasA</I> operon). Using the <I>srfA</I> promoter (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K305007" class="external text" rel="nofollow">BBa_K305007</A>), we succesfully expressed GFP demonstrating that this promoter could be used to auto-induce the expression chaplins.
</P><H3><SPAN class="mw-headline" id="Subtilin_induced_expression_of_chaplins">Subtilin induced expression of chaplins</SPAN></H3><P>The biofilm forming capacity of <I>Bacillus subtilis</I> makes it a good host for our application. In addition, <I>B. subtilis</I> is known for its ability to produce and secrete large amounts of protein at high cell densities. However, despite its track record as an efficient production organism and the fact that both <I>B. subtilis</I> and <I>Streptomyces coelicolor</I> are gram-positive bacteria, it is not certain wether chaplins can be heterologously expressed in <I>B. subtilis</I>. Improper folding, unsuccessful export, or even the very nature of the chaplins, could still lead to hampered expression.
We took several steps to ensure optimal expression. The coding sequences of the chaplins were codon optimized for <I>B. subtilis</I> and synthesized. We placed a ribosome binding site in front of the coding sequences that is known to work well in <I>B. subtilis</I>, and flanked these constructs with the biobrick prefix and suffix. 
</P><P><B>SURE expression system</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Subtilin induction of GFP by the SURE system (Bongers <I>et al</I>, 2005)</DIV></DIV></DIV><P>Because it is uncertain how chaplin expression will affect <I>B. subtilis</I>, the initial expression attempts were performed with the stringently controlled, subtilin-regulated gene expression (SURE) system (Bongers <I>et al</I>, 2005). This system uses the subtilin sensing machinery present in a strain of <I>B. subtilis</I> that autoinduces the production of more of the <A href="http://en.wikipedia.org/wiki/Lantibiotics" class="external text" rel="nofollow">lantibiotic</A> subtilin. The subtilin sensor histidine kinase SpaK phosphorylates the response regulator SpaR, which can then bind to so-called <I>spa</I> boxes in the promoter regions of genes involved in subtilin biosynthesis (Kleerebezem <I>et al</I>, 2004). In the SURE system, a <I>B. subtilis</I> strain naturally lacking the subtilin biosynthesis and regulation genes has the <I>spaRK</I> genes introduced into its genome. A plasmid carrying a <I>spa</I> box promoter that is transformed to this strain can then drive the expression of proteins upon subtilin induction of SpaRK signalling. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Subtilin induced GFP expression in Bacillus subtilis using the <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K305011" class="external text" rel="nofollow">BBa_K305011</A> backbone</DIV></DIV></DIV><P>We have adapted this system to make it BioBrick compatible for easy expression of our chaplins, combinations of chaplins, or any other biobrick part that is composed of an RBS followed by a protein coding sequence. We introduced the BioBrick prefix and suffix into the expression plasmid, downstream of the mutated <I>spaS</I> promoter, producing our subtilin inducible expression backbone part, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K305011" class="external text" rel="nofollow">BBa_K305011</A>. To test the expression and find a suitable subtilin concentration for induction of the chaplins we made use of GFP fluorescence measurements. We inserted the part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_E0240" class="external text" rel="nofollow">BBa_E0240</A> into the BioBrick site and induced liquid cultures of <I>B. subtilis</I> carrying this plasmid (and the <I>spaRK</I> genes) with different volumes of subtilin-containing culture supernatant of a subtilin producing strain of <I>B. subtilis</I>. These results demonstrate that addition of 0.5 to 1%(vol/vol) of subtilin to the culture is sufficient to reach optimal induction.
</P><P><B>Chaplin detection</B>
In our first expression tests we focused on detecting the chaplins in either the medium in which our chaplin producing population grows 

Using the same methods that were used to dissolve and monomerize chaplins from Streptomyces, we treated cell pellets and <A href="http://en.wikipedia.org/wiki/Trichloricacetic_acid" class="external text" rel="nofollow">TCA</A> precipitated supernatant with <A href="http://en.wikipedia.org/wiki/Trifluoroacetic_acid" class="external text" rel="nofollow">TFA</A> to purify chaplin proteins. <A href="http://en.wikipedia.org/wiki/Trifluoroacetic_acid" class="external text" rel="nofollow">TFA</A> treatment with 99% pure TFA breaks down most proteins and monomerises assembled chaplin fibers, this enables us to detect the chaplins on SDS gel. 
</P><P>
Since our early expression experiments didn't yield conclusive results regarding the detection of our chaplins, we tried staining our samples with an amyloid specific stain called <A href="http://en.wikipedia.org/wiki/Thioflavin" class="external text" rel="nofollow">Thioflavin T</A>. Initial testing with the supernatant and washed pellet gave [intriguing resultsGr] yet not clear.  Our emission graphs showed some irregularities with the subtilin induced samples, but seemed to be distorted by background noise caused by other materials in the sample. To further purify our samples we decided to [extractioncellwallsGR disrupt] our liquid culture and boil it two times in 2% SDS, before treating the freeze dried sample with 99%. TFA Using such a harsh  method, we hope to denaturate most proteins to prevent their interference in chaplin detection and highten the relative concentration of chaplin proteins in tested samples, this turned out to be a more successful method.
</P><P>Using this method we succesfully detected chaplins C, E and H in purified cell walls from induced B. subtilis cultures, confirmation of these results was provided by malditov mass spectrometry.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Purified chaplins from S. coelicolor exitate at 482 when stained with THT. Different dilutions were used to set up a ladder.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Purified E and H chaplins from B subtilis exitate at 482 when stained with THT</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Purified C chaplins from B subtilis exitate at 482 when stained with THT</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Purified E chaplins from B subtilis exitate at 482 when stained with THT</DIV></DIV></DIV></TD></TR></TBODY></TABLE><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Mass spectometry results confirm the production of Chaplin H (5128) and E (5144) in Bacillus (green). Peaks is Streptomyces (blue) corrispond, although the peak of chaplin E (5144) indicates that somewhere during translation or in post-translational processes a 21kb threonine group is lost. The non induced B. subtilis strain serves as a control (red). Chaplin C has not been detected this way, but that was not to be expected (Claessen et al. (2009).</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Timed_expression_of_chaplins_in_a_biofilm">Timed expression of chaplins in a biofilm </SPAN></H3><P>An important question is which promoter we should use to control the chaplin expression. We assume that an ideal promoter would not be active until the biofilm has formed because the expression of hydrophobic proteins might influence the formation of it. Two promoters where found that are active in biofilms but not during normal growth. 
</P><P><I><B>srfA</B></I></P><P>The <A href="http://dbtbs.hgc.jp/COG/prom/srfAA-srfAB-comS-srfAC-srfAD.html" class="external text" rel="nofollow"><I>srfA</I> operon</A> has been reported to be important for natural competence and sporulation in <I>Bacillus subtilis</I>. All these activities occur in biofilms, the promoter is not active until the end of exponential growth. It is controlled by the <A href="https://2010.igem.org/Team:Groningen/Expression_model#ComXPA_quorum_sensing_system" class="external text" rel="nofollow">ComXPA quorum sensing system</A> and hence active in states of high cell densities. Therefore the <I>srfA</I> promoter would be suitable for chaplin expression. Two different lengths of the <I>srfA</I> promoter where chosen due to uncertainties concerning the region between the response element and the transcription start side of the SrfAA protein. In the original promoter this region is unusually long, by shortening it 190bpâs we hope to achieve a higher transcription efficiency. So we came up with two different promoters, the <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K305008" class="external text" rel="nofollow">original</A> one and the <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K305007" class="external text" rel="nofollow">shortened</A> one. We followed the promoter activity by placing a RBS+GFP BioBrick downstream of these promoters. We succesfully demonstrated that the short <I>srfA</I>-promoter variant drives the expression of GFP in a cell density dependent way. However, as can be seen in the image below, the fluorescence starts increasing during early exponential growth, rather than late exponential growth. This might be because this multi-copy plasmid system is more sensitive to srfA promoter activators. It could also be caused by our removal of the sequences downstream of the regulator binding sites and upstream of the srfA coding sequence. These sequences may have a regulatory role, preventing transcription/translation at earlier growth stages.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">srfA promotor activity during cell growth (Nakano MM. 1991)</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Promoter activity of our short srfaA promoter determined by measuring GFP reporter fluorescence, with overlaid growth curve.</DIV></DIV></DIV><P><I><B>yqxM</B></I></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">yqxM promotor activity during cell growth of different mutants (Axel G. 1999)</DIV></DIV></DIV><P>The <A href="http://dbtbs.hgc.jp/COG/prom/yqxM-sipW-tasA.html" class="external text" rel="nofollow"><I>yqxM-sipW-tasA</I></A> operon is controlled by the <I>yqxM</I> promoter. It is needed for biofilm formation because TasA is a key protein of the extracellular matrix. The promotor gets activated via a cascade of other regulatory elements, including SrfA, in response to quorum sensing. Since the chaplins should work in a similar way to TasA we think the <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K305006" class="external text" rel="nofollow"><I>yqxM</I></A> promoter would be very suitable for chaplin expression during the stationary phase. We fused the yqxM promoter with GFP but could not observe any expression, since the GFP worked with the srfA promoter we conclude that the yqxM promoter does not work.
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P><SMALL>Bongers RS, Veening JW, Van Wieringen M, Kuipers OP, and Kleerebezem M. Development and characterization of a subtilin-regulated expression system in Bacillus subtilis: strict control of gene expression by addition of subtilin. <A href="http://aem.asm.org/cgi/content/short/71/12/8818Appl" class="external text" rel="nofollow">Environ Microbiol 2005 Dec; 71(12) 8818-24. pmid:16332878</A></SMALL></P><P>Kleerebezem, M., R. Bongers, G. Rutten, W. M. de Vos, and O. P. Kuipers.
2004. Autoregulation of subtilin biosynthesis in Bacillus subtilis: the role of
the spa-box in subtilin-responsive promoters. <A href="http://gbb.eldoc.ub.rug.nl/FILES/root/2004/PeptidesKleerebezem/2004PeptidesKleerebezem.pdf" class="external text" rel="nofollow">Peptides 25:1415â1424</A></P><P>StÃ¶ver AG, Driks A. Regulation of synthesis of the <I>Bacillus subtilis</I> transition-phase, spore-associated antibacterial protein TasA. <A href="http://jb.asm.org/cgi/content/short/181/17/5476" class="external text" rel="nofollow">J. Bacteriol. Sept. 1999, p. 5476-5481, Vol. 181, No. 17</A></P><P>Nakano MM, Xia LA, Zuber P. Transcription initiation region of the srfA operon, which is controlled by the comP-comA signal transduction system in <I>Bacillus subtilis</I>. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC208261/" class="external text" rel="nofollow">PMC208261</A></P><P>Frances Chu, Daniel B. Kearns, Anna McLoon, Yunrong Chai, Roberto Kolter and Richard Losicka, A Novel Regulatory Protein Governing Biofilm Formation in <I>Bacillus subtilis</I><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2430766/" class="external text" rel="nofollow">PMC2430766</A></P><P>Hayashi K, Ohsawa T, Kobayashi K, Ogasawara N, Ogura M. The H2O2 stress-responsive regulator PerR positively regulates srfA expression in <I>Bacillus subtilis</I>. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1251593/" class="external text" rel="nofollow">PMC1251593</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Groningen/Expression">http://2010.igem.org/Team:Groningen/Expression</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Groningen/Expression" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Groningen/Expression" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Groningen/Expression&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Groningen/Expression&amp;oldid=208505" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>