<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Northwestern_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Northwestern/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Northwestern/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Northwestern/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Northwestern/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Northwestern/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Northwestern/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Dropdown menu from bootstrap</TITLE></P><DIV class="container"><DIV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2014.igem.org/Team:Northwestern/Home">Home</A></DIV><DIV class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="dropdown"><A href="https://2014.igem.org/Team:Northwestern/Project" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2014.igem.org/Team:Northwestern/Project">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Northwestern/Growth">Cultures</A></LI><LI><A href="https://2014.igem.org/Team:Northwestern/CFPS">CFPS</A></LI><LI><A href="https://2014.igem.org/Team:Northwestern/parts">Part submitted</A></LI></UL><LI><A href="https://2014.igem.org/Team:Northwestern/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Northwestern/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Northwestern/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Northwestern/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Northwestern/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Northwestern/Human_Practice">Outreach Projects</A></LI></DIV></DIV></DIV></DIV><TITLE>Notebook</TITLE><DIV class="animated fadeIn"><DIV class="jumbotron"><DIV class="container"><H1>Notebook</H1></DIV></DIV></DIV><DIV class="container"><UL class="nav nav-tabs" role="tablist"><LI class="active"><A href="#june" role="tab" data-toggle="tab"><H5>June</H5></A></LI><LI><A href="#july" role="tab" data-toggle="tab"><H5>July</H5></A></LI><LI><A href="#august" role="tab" data-toggle="tab"><H5>August</H5></A></LI><LI><A href="#september" role="tab" data-toggle="tab"><H5>September</H5></A></LI><LI><A href="#october" role="tab" data-toggle="tab"><H5>October</H5></A></LI></UL><DIV class="tab-content"><DIV class="tab-pane fade in active" id="june"><DIV class="container"><DIV class="row"><DIV class="col-lg-8"><H3>Week 1</H3><P><SPAN class="label label-info">6/17</SPAN> We started our iGEM bootcamp by shadowing Will Bothfeld (our primary graduate student TA) during ligating, transforming and working with DNA software. In addition, we learned the fundamentals of primer design, colony PCR and gel electrophoresis. We ended by continuing research and discussing all of our projects of interest.</P><P><SPAN class="label label-info">6/18</SPAN> To start our day, we narrowed down on project ideas to CRISPR-Cas system, quick quench GFP, and non-model organisms; we then discussed their individual pros and cons. During our meeting with Professors and graduate students, we had an in depth discussion on this topic as well as laboratory logistics such as lab equipment and our summer budget. With this information and guidance, we were able add to our initial thoughts on each of our potential projects by starting corresponding flowcharts of events. </P><P><SPAN class="label label-info">6/19</SPAN> All team members finished necessary safety training modules online to partake in wet lab work. We then cleaned the lab and organized materials/equipment. Will proposed a problem to us regarding primer design that we worked on first individually and then collaboratively to compiled an answer to for later review.</P><P><SPAN class="label label-info">6/20</SPAN> After learning the basics of and observing plate reading, we compiled a series of protocols that we felt would be important to have before getting starting on DNA cloning first-hand. We also updated our lab shelves and freezer inventory thoroughly to get a sense of what we should be ordering and asking in our sponsorship emails.</P></DIV><DIV class="col-lg-8"><H3>Week 2</H3><P><SPAN class="label label-info">6/23</SPAN> In the morning, we had a meeting with a TA, Karthik Sekar, to discuss the iGEM registry and parts along with how to use them. Following that, we had a meeting our professors and grad students in which we learned more about the project of non-model organisms. After our meeting, we made LB Mix and discussed which project to pick- non-model organisms or quenching GFP. In the end, we came to agreement to do our project on non-model organisms by having E. coli and yeast as our controls and other strains of bacteria and fungi as our non-model organisms. We would test out how promoters would differ in their activity across all these strains by the use of GFP.</P><P><SPAN class="label label-info">6/24</SPAN> Throughout the day, we sent out emails to sponsors for lab supplies. We also worked on a problem that Karthik gave us to do in which we had to construct a plasmid with an origin of replication and a PhoP promoter controlling PFP expression as the insert using iGEM registry parts. We also started cloning with Will.</P><P><SPAN class="label label-info">6/25</SPAN> We took an inventory of the chemicals we had from before in our lab and of the contents in our freezers. We met with Karthik again to discuss the solution we came up to the problem he gave us to do. In addition, we brainstormed project titles and created a tentative timeline for the summer. Lastly, we read a paper by Paul Freemont of the Imperial College of London that talked in detail about cell free systems being accurate representations of E. coli cells. </P><P><SPAN class="label label-info">6/26</SPAN> This day had relatively more wet lab work. We started by running PCR, and then running a gel for 1000 base pairs to confirm if the PCR worked. After analyzing our results, we researched promoters that we could potentially use to test for our project. We also searched for professors and labs around campus from which we could get strains of non model organisms. The final task we did was shadow Dr. Jian Li, a member of Jewett lab who showed us how to collect cells for lysate preparation and observation.</P><P><SPAN class="label label-info">6/27</SPAN> We started our day at the Jewett lab where a grad student, Rey Martin, showed us how to lyse E. coli cells and perform extract. We made another gel and tested for ladders. Meanwhile, some members of the team made M9 media and then autoclaved it. Two of our members met with a representative of IDT to talk about buying their products. Lastly, we continued researching promoters and reaching out to labs asking for non-model organisms.</P></DIV></DIV></DIV></DIV><DIV class="tab-pane fade" id="july"><DIV class="container"><DIV class="row"><DIV class="col-lg-8"><H3>Week 3</H3><P><SPAN class="label label-info">6/30</SPAN> We met with the professors to further discuss our project and our overall timeline.  We gained information about cell-free systems from the meeting. We also decided not to use fungus as a non-model organism as eukaryotes would make the project overly complex. After, we had a meeting with Joe to talk about mathematical modeling. In our discussion, we also tried to refine and narrow our project idea.</P><P><SPAN class="label label-info">7/1</SPAN> We started the morning in the Jewett Lab shadowing a grad student who was carrying out a cell-free reaction. We decided to split the team in two.  One team focused on harvesting three strains of E. coli cells and the other dealt with the initial steps of cloning.  All of us continued researching promoters since the cell-free system uses T7 bacteriophage and the promoters we had initially selected from the Chris Anderson list were incompatible.  For cloning we used the 5 promoters which we originally planned to use. The T7 compatible promoters from the iGEM registry were not present in the distribution kit, so we need to order from IDT or some other similar company.  We also obtained a sample of Pseudomonas cells (one of the non-model strains) from the Wang lab which was incubated overnight.</P><P><SPAN class="label label-info">7/2</SPAN> Mitch and Sharon teamed up to harvest cells. We calibrated the spec by making serial dilutions and making a linear curve. We finished growing the starter culture and inoculated appropriate concentrations in 2x YTPG media. We grew these for a few hours and collected the cells by cycles of centrifugation. We used S30 buffer for extraction. We also had a meeting with Anthony, a grad student, who agreed to help us out by providing the non-model organism, streptomyces. </P><P><SPAN class="label label-info">7/3</SPAN> We transformed plasmids into competent cells and then plated them.  We designed primers for the inserts that we obtained the day before. We put the Pseudomonas cells in a frozen stock, while Karthik gave us an article about spinach aptamer to read.</P><H3> Week 4 </H3><P><SPAN class="label label-info">7/7</SPAN> We continued the lysate making process by resuspending and sonicating cell pellets. We made frozen stock of successful colonies from previous plating, but started another round of cloning using the same 5 plasmids and DH5alpha cells as before in the hopes to get better cell colonies. We also made a RAM chart to keep tasks and priorities organized. </P><P><SPAN class="label label-info">7/8</SPAN> Tiffany and Adam had a meeting with Joe to talk more about mathematical modelling. All of us also had a meeting with the professors in which we talked in depth about outreach, T7 promoters, and judging criteria for the various medals. We continued to do cloning and cell free protein synthesis, and we had a review of primers with Will. </P><P><SPAN class="label label-info">7/10 </SPAN> We started the day by autoclaving beakers and flasks. David, Adam and Tiffany had a meeting with Joe to further discuss modelling. We grew two cultures of Pseudomonas cells in parallel from a frozen stock to establish a growth curve by checking OD every half an hour. We also had a discussion about how we will measure our results and we didn’t come to a clear conclusion, but some ideas that were thrown out were to measure strength by how much protein or mRNA is produced and how quickly protein is produced. We plan to test different RBS’s as well. </P><P><SPAN class="label label-info">7/11</SPAN> Our team worked together to create a powerpoint to present to our advisors and update them on our progress. We then re-stocked our shelves with autoclaved materials and more media. We made our final edits to our g-block design and added it to our powerpoint. Afterwards, we headed to Chicago to meet with the UChicago iGEM team to discuss our respective projects and potential collaboration. The meeting proved very helpful- once we heard about their project, we found that their promoter could be tested in our lysates. We hope to get this promoter in plasmid form the next time we meet.</P><H3> Week 5 </H3><P><SPAN class="label label-info">7/14</SPAN> Abdullah and Kristi started working on a presentation on synthetic biology for the Center for Talent Development (CTD) high school program. We met with our advisors and found points of our initial testing plan that needed revision. We also got Streptomyces fluorescens from the Kelleher lab and prepared a survey for outreach. </P><P><SPAN class="label label-info">7/15</SPAN> In the morning, we made LB and then autoclaved it. We also procured some glucose from Will and autoclaved that too. From Professor Yun Wang, we got a BSL1 strain of Pseudomonas in the afternoon. Abdullah, Kristi, and Adam gave a presentation of synthetic biology to a biomedicine class for highschoolers. In addition, we continued to grow strep as it takes two days while searching for different RBS sequences for our non model organisms. </P><P><SPAN class="label label-info">7/16</SPAN> Abdullah and David went to Roycemore School to meet a CTD teacher to talk about having a synthetic biology presentation for his honors biology class. Adam, Sharon and Kristi went to a BioExcel event where they advised students on their application based projects, for example, how to prevent HIV. We also got N. Europea from Freddy from the Wells lab. After getting ISP ingredients from Jian, we made ISP and plates. We also analyzed the surveys that we handed out to CTD students the day before. </P><P><SPAN class="label label-info">7/17</SPAN> We reached out to UChicago informing them that we can provide them with some E. coli strains. We did extensive research on RBS of the non model organisms we are experimenting out. We also ordered a gBlock but it had some issues so we decided to fix it the following day. </P><P><SPAN class="label label-info">7/18</SPAN> We started to make a growth curve of strep, which was proving to be complicated as the OD kept on oscillating. We also revised our gBlock and finally ordered it to IDT. For the meeting with our professors on Monday, we prepared a powerpoint of what we’ve done this past week and questions we have. Lastly, we continued researching on RBS’s. </P><H3> Week 6 </H3><P><SPAN class="label label-info">7/16</SPAN> In the morning, we attended a seminar about biotechnology and liver metabolism by Dr. Jamey Young. We ordered the RBSs and promoters and we autoclaved flasks, solutions, etc. We finished and submitted the safety form that was due and we made a growth curve for Strep and Pseudomonas. </P><P><SPAN class="label label-info">7/17</SPAN> Adam grew/plated backbone for miniprep, while David and Tiffany made a growth curve for Pseudomonas. Kristi and Abdullah grew E. coli strains for lysates and Mitch and Sharon worked on a growth curve for Strep. David and Kristi also made YTPG. We also made additional LB and had a meeting with the professors, who gave us tips on how to go about making a survey and test promoters. </P><H3> Week 7 </H3><P><SPAN class="label label-info">7/23</SPAN> Adam grew a specific colony for miniprep, whereas Abdullah and Mitch miniprepped plasmid. Sharon did a cell-free protein synthesis and met with Jian to talk more about strep and what to keep in mind when experimenting with it. Sharon and Mitch checked OD of strep by vortexing at different temperatures, with the advice of Will. Kristi, Tiffany, and David harvested E. coli cells. We also responded to UChicago and WashU’s iGEM teams who were interested in collaborating with us. Lastly, Mitch and Abdullah got some water from a pond in the hope that we could culture a few environmental strains for sequencing in the future. </P><P><SPAN class="label label-info">7/24</SPAN> Adam miniprepped the backbone and worked with Kristi to compile a list of oligos in a spreadsheet for Will. Abdullah and Mitch harvested Pseudomonas while David and Kristi did PCR practice on old dNTPs. David and Tiffany lysed the E. coli and Sharon measured the biomass of strep throughout the day. Lastly, Adam and Sharon researched on GC-rich RFP that would be compatible with strep. </P><P><SPAN class="label label-info">7/25</SPAN> Abdullah and Mitch continued working with Pseudomonas by lysing them. David and Tiffany continued working with E. coli along with Kristi to perform a Bradford assay with the lysed cells. David and Tiffany also aliquoted the new E. coli lysates whereas Sharon aliquoted the old E. coli lysates. Adam, meanwhile, finalized the track selection and worked with Kristi to make LB plates without Amp for the environmental strains. David and Kristi did the PCR test digest and David was also involved with Sharon with cell free protein synthesis. At the end of the day, we all worked on making a powerpoint for the meeting on Monday with the professors. </P><H3> Week 8 </H3><P><SPAN class="label label-info">7/28</SPAN> We obtained new environmental strains from the Lakefill in the morning to grow in a LB media throughout the day. David and Tiffany reran the test PCR with diluted DNA to check the quality of our dNTPs and Taq polymerases. After running gel electrophoresis, we gathered clearer bands than the first run through. Adam transformed and plated the backbone plasmid for our g-block into e.coli DH5a cells. We then made LB/agar and LB/agar/cam plates to plate our environmental sample and plasmid backbone, respectively. In our meeting with the professors, we were advised on how to better our results for cell-free protein synthesis. </P><P><SPAN class="label label-info">7/29</SPAN> Mitch and Sharon early in the day harvested strep while Adam and David made NE buffer. Abdullah and David made environmental plates with water from the thermo-insulated pond whereas Sharon and Kristi made strep lysates. Abdullah and Tiffany measured the pH of N Europea and David along with Sharon PCR’ed the g-block. Joe stopped by to talk about codon optimization with Abdullah, Sharon, and Tiffany. </P><P><SPAN class="label label-info">7/30</SPAN> Abdullah and David cut the PCR g-block, ran a gel and extracted it which failed. Adam grew backbone from colony in a tube and miniprepped. Mitch and Tiffany cut the backbone, split 2 aliquots, dephosphorylated, ran gel and extracted it. Kristi and Sharon ligated g-block into backbone and transformed into DH5 alpha while also carrying out cell free protein synthesis in strep. </P><P><SPAN class="label label-info">7/31</SPAN> With not much wet lab work to do, we cleaned up the lab and restocked whatever was needed. We made ISP and then ISP plates. We replaced all the full biohazard bags and removed any unnecessary culture from our fridges. We also made LB plates for environmental strains. </P></DIV></DIV></DIV></DIV><DIV class="tab-pane fade" id="august"><DIV class="container"><DIV class="row"><DIV class="col-lg-8"><H3> Week 9 </H3><P><SPAN class="label label-info">8/4</SPAN> Kristi and Mitch cut the PCR g-block, ran the gel and extracted it. Abdullah, David and Sharon ligated the g-block into backbone and transformed it into DH5 alpha. David and Kristi made competent cells. However, the results of our gel showed that our PCR g-block wasn’t PCR’ed correctly, so we decided to do everything again the next day. </P><P><SPAN class="label label-info">8/5</SPAN> We did the same thing as said for 8/04, and the gel indicated that the PCR worked. We also presented to a CTD biomedicine class about synthetic biology. </P><P><SPAN class="label label-info">8/6</SPAN> David and Sharon checked for colonies on the plates, and restreaked 10 colonies onto new plates. Abdullah and Kristi did colony PCR on the restreaked colonies. Mitch and Sharon restreaked the environmental samples. We were supposed to run a gel for colony PCR, but decided to do it the next day. </P><P><SPAN class="label label-info">8/7</SPAN> We performed colony PCR which proved to be unsuccessful. </P><P><SPAN class="label label-info">8/8</SPAN> Abdullah and Mitch retested the PCR while Sharon and David did PCR purification. Adam ligated, transformed and plated the gBlock. Kristi and Sharon, meanwhile PCR’ed each clone, one with a forward primer and one with a reverse primer. </P><H3> Week 10 </H3><P><SPAN class="label label-info">8/11</SPAN> Adam ligated, transformed and plated the gBlock and backbone. Kristi and Mitch did colony PCR for the old clones while Abdullah and David made YTPG. We also prepared a presentation for the weekly meeting we had the next day. </P><P><SPAN class="label label-info">8/12</SPAN> Adam selected 5 colonies from the 10 we made and miniprepped them. Abdullah and Kristi did colony PCR on them. David and Mitch kept track of the growth of Pseudomonas in LB and before leaving transferred some to YTPG. We also ran a gel/test digest of the 3 colonies out of the 5 which actually had DNA. </P><P><SPAN class="label label-info">8/13</SPAN> We made pseudomonas lysates again and also did a Bradford to test the for presence of protein. </P><P><SPAN class="label label-info">8/15</SPAN> Cell-Free Protein Synthesis! </P><H3> Week 11 </H3><P><SPAN class="label label-info">8/18</SPAN> Analyzed CFPS data from weekend experiment and found our lysates were largely ineffective (positive control worked well). Began growing up Pseudomonas, 3 E. coli (K12, DH5a, BL21), and Streptomyces for harvesting the next day. </P><P><SPAN class="label label-info">8/19</SPAN> Harvested the 5 strains of bacteria and pelleted the cells, flash freezing them and storing in the -80 for lysing the next day. Re-ligated gBlock into backbone, transformed and plated to grow overnight. </P><P><SPAN class="label label-info">8/20</SPAN> Re-suspended the pelleted cells and created new lysates using sonication. Bradford Assay performed to test protein levels in all of the lysates. Mini-prepped  and purified the promising colonies and submitted them for sequencing. </P><P><SPAN class="label label-info">8/21</SPAN> Ran Cell-Free Protein Synthesis on new lysates. </P><P><SPAN class="label label-info">8/22</SPAN> Analyzed results of Cell-Free Protein Synthesis. Received sequencing results for gBlock, began ligations again to successfully link gBlock and backbone.  </P><H3> Week 12 </H3><P><SPAN class="label label-info">8/26</SPAN> Colonies 1A-7 and 1A-8 were ligated.  Gels were also run to test if the gBlock was present in the plasmid. </P><P><SPAN class="label label-info">8/27</SPAN> Colonies 1A-7 and 1A-8 were transformed and plated.  More competent cells were grown. Research was done on the 16S RNA which we plan to use to sequence our environmental strains which would then be used to make lysates. </P><P><SPAN class="label label-info">8/28</SPAN> Colony PCR run on the environmental strains as well as 1A-7.  1A-8 showed no growth.  Controls made of BL21 and DH5-alpha.  Nitrosomonas was harvested, but since the harvesting protocol is optimized for E. coli, this proved tricky. </P><P><SPAN class="label label-info">8/29</SPAN> We did Colony PCR of 1A-7 which was one of the clones that worked.  Ran a gel which showed 900 bp band for all trials of 1A-7.  We made more CAM plates and checked the growth of 1A-7, 1A-8, (the other things that are in the incubator that Kristi did).  We then inoculated of 1A-7-1 through 1A7-16. </P><P><SPAN class="label label-info">8/30</SPAN> Adam and Tiffany made 35 mini-preps of 1A-7-11 and measured and recorded the protein concentration using the Nano-drop and the ones that Kristi did.  Sharon and Kristi did PCR and made a gel of RBSs to extract the correct bands. </P></DIV></DIV></DIV></DIV><DIV class="tab-pane fade" id="september"><DIV class="container"><DIV class="row"><DIV class="col-lg-8"><H3> Week 13 </H3><P><SPAN class="label label-info">8/31</SPAN> Sharon, Tiffany, and Mitch cut, dephosphoralated, ligated, and plated the 20 RBSs using DH5-alpha competent cells. </P><P><SPAN class="label label-info">9/2</SPAN> We checked on the growth of the RBS which looked promising.  Sharon ran a colony PCR on all of the ligated product from Sunday and then ran a diagnostic gel on the PCR product. Mitch and Abdullah dephosphoralated, ligated, and transformed six promoters using DH5-alpha.

</P><P><SPAN class="label label-info">9/4</SPAN> We ran 6 gels today to see if the RBS’s showed up in the cell/ligated with the cell. Only 1 band from the 6 gels confirmed that since it was in the 300 bp range. Adam and Abdullah meanwhile miniprepped RBS’s with plasmids so there would be genomic interference with them. After miniprepping, Sharon prepared them for PCR. </P><P><SPAN class="label label-info">9/5</SPAN> We made Cam plates.  David plated a plasmid from the iGEM kit with a standard biobrick backbone. Ran a gel of the RBSs.

</P><H3>Week 14</H3><P><SPAN class="label label-info">9/10</SPAN> We all inoculated colonies from the RBS ligated plates into LB, picking 5 colonies from each plate. Since there were 16 plates of various RBS’s, we had 80 tubes of inoculated colonies. </P><P><SPAN class="label label-info">9/11</SPAN> We all miniprepped RBS ligated cells and prepared it for sequencing which we expect to finish the next day. Sharon and Abdullah also ran two gels with RBS ligated cells to see if a 300 band shows up for them, and 5/16 RBS’s did show a band there. </P><P><SPAN class="label label-info">9/12</SPAN> Adam and Mitch completed the sequencing reactions for submission.  6 tubes of BBBB A15 were inoculated by Tiffany and Sharon.  Sharon ran all of the lysates (three strains of E. coli, S. rimosus, P. fluorescens, and two environmental samples) in a cell-free protein synthesis reaction with RBS X5.  Mitch mini-prepped BBBB-A15-1 through BBBB-A15-6.</P><P><SPAN class="label label-info">9/13</SPAN> Adam cut the mini-prepped backbone and gel extracted the product.  Sharon ligated in 16 of the RBS’s and Tiffany transformed.</P><H3>Week 15</H3><P><SPAN class="label label-info">9/15</SPAN> Nothing grew on any of the plates from the weekend.  We started to grow more cells to make E. coli lysates of the three different strains.  We also made more media, YTPG and LB.  Adam recut the mini-prepped backbone and ran it through a gel because of the previous unsuccessful ligation.</P></DIV></DIV></DIV></DIV><DIV class="tab-pane fade" id="october"><DIV class="container"><DIV class="row"><DIV class="col-lg-8"><H3>Week 16</H3><P><SPAN class="label label-info">10/6</SPAN> We redid the ligation for the gBlock into BBBB and then transformed and plated it.  The colony from plate R3 from 10/4/14 was miniprepped and sent in for sequencing.  We gel extracted BBBB and the Spinach Aptamer and prepped for a ligation. </P><P><SPAN class="label label-info">10/7</SPAN> Adam checked the plate from Monday for colonies and since there were some present, they were miniprepped and sent in for sequencing.  Mitch and Tiffany prepared for another round of ligation and transformation. </P><P><SPAN class="label label-info">10/8</SPAN> We ligated the RBS into the Spinach Aptamer. </P><P><SPAN class="label label-info">10/9</SPAN> We sent in the Spinach Aptamer as a part to the iGEM registry. </P><H3> Week 17</H3><P><SPAN class="label label-info">10/12</SPAN> We inoculated the RBS and gBlock. </P><P><SPAN class="label label-info">10/13</SPAN> The inoculated cultures from the previous day showed no growth.  In the morning, the RBS+gblock construct was miniprepped using the Omega solution 1 and Epoch for all other solutions.  The sequencing reactions for the biobrick construct were prepared.  The concentration of these were around 20ng/ul.  In the afternoon the fluorophore was resuspended in DMSO and diluted with water to the desired concentration. </P><P><SPAN class="label label-info">10/14</SPAN> Cell free protein synthesis was done with the three strains of E. coli (BL21, DH5a, and K12), along with S. rimosus, P. fluorescens, and two environmental strains, enterobacteria and comamonas.  </P><P><SPAN class="label label-info">10/15</SPAN>  The results for CFPS returned, unfortunately there were only kinetic results from spaces A-C and 1-8 which were BL21 and DH5a and two of the 3 K12 samples.  Graphs are in the CFPS tab under “Projects”. </P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Northwestern/Notebook">http://2014.igem.org/Team:Northwestern/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Northwestern/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Northwestern/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Northwestern/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Northwestern/Notebook&amp;oldid=362934" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>