"
Page
Discussion
View source
History
teams
Log in
 
Team:Waterloo
From 2011.igem.org
Edit History Move Discussion
Home
In Vivo Protein Fusion Assembly Using Self Excising Ribozyme
ABSTRACT
Introns, self-excising ribozymes, can become a useful tool to create in vivo protein fusions of BioBrick parts. To make this possible, intron sequences are used to flank non-protein parts embedded in coding sequences. An intron sequence with an embedded recombination site is capable of in vivo insertion of a compatible protein fusion part. As an example, a GFP-fusion was created with an intervening lox site that is removed from the final protein using the intron to form a fully functional GFP protein. In vivo protein fusions can be applied to a larger number of modular systems to make complicated expression systems, such as synthetic antibodies or plants capable of Cry-toxin domain shuffling.
WE WOULD LIKE TO THANK OUR GENEROUS SPONSORS.
Project
Modeling
Outreach
Human
Practices
Project
The goal of Waterloo's 2011 iGEM project is to implement self-excising ribozymes (introns) as biobricks. But first, what are self-excising ribozymes? Ribozymes are ribonucleic acid (RNA) enzymes and enzymes are reaction catalysts. So ribozymes are just RNA sequences that catalyze a (trans-esterification) reaction to remove itself from the rest of the RNA sequence. Essentially these are considered introns, which are intragenic regions spliced from mRNA to produce mature RNA with a continuous exon (coding region) sequence. Self-excising introns/ribozymes consist of type I and II introns. They are considered self-splicing because they do not require proteins to intitialize the reaction. Therefore, by understanding the sequences and structure of these self-excising introns and making them useable, we can use them as tools to make other experiments easier.
1.0 INTRODUCTION
This design provides a reasonable basis to implement in vivo applications involving RNA level regulatory sequences. The fusion proteins produced surpass strictly what is coded in the DNA. As a result of incorporating ribozyme segments in between two halves of the protein coded in the DNA construct, a regulatory sequence (such as a recombination site) could be included. Since recombination sites can interrupt the functional production of a protein if translated fully (resulting in excess amino acids in the polypeptide), the incorporated ribozyme portions remove them before the translation phase of gene expression so that a functional protein is produced. For example, Cry proteins, which account for the insecticidal activity (toxicity) of Bacillus thuringiensis, could be the fusion protein produced for a particular insecticide. Using our experimental design, the sequence containing the code for the Cry protein (at the DNA stage) is split by ribozyme segments containing a recombination site. In this case, the recombination site is the regulatory sequence that will be removed once transcribed into RNA. At the DNA level, recombination (shuffling) will occur, exchanging DNA strand segments. Therefore, when the shuffled DNA sequence is transcribed into RNA, the recombination site is spliced out of the sequence with the ribozymes, and the resulting RNA code is different than that of the un-shuffled code. Consequently, the translated Cry protein is different. This system would oppose pesticide resistance among the target organism.
1.1 A Little Bit About Group 1 Introns
All group I introns in bacteria have presently been shown to self-splice (with few exceptions) and maintain a conserved secondary structure comprised of a paired element which uses a guanosine (GMP, GDP or GTP) cofactor. Conversely, only a small portion of group II introns have been verified as ribozymes (they are not related to group I introns) and generally have too many regulators to easily work with. It is mainly the structural similarity of these introns that designates them to group II. We will mainly be working with group I introns, such as the Staphylococcus phage twort.ORF143.
Group I introns contain a conserved core region consisting of two helical domains (P4–P6 and P3–P7). Recent studies have demonstrated that the elements required for catalysis are mostly in the P3 to P7 domain. They are ribozymes that consecutively catalyze two trans-esterification reactions that remove themselves from the precursor RNA and ligate the flanking exons. They consist of a universally conserved core region and subgroup-specific peripheral regions, which are not essential for catalysis but are known to cause a reduction in catalytic efficiency if removed. To compensate for this, a high concentration of magnesium ions, spermidine or other chemicals that stabilize RNA structures can be added. Thus, the peripheral regions likely stabilize the structure of the conserved core region, which is essential for catalysis.
1.2 Trans-Esterification Reactions
The secondary structures, such as P6, formed by group I introns facilitates base pairing between the 5' end of the intron and the 3' end of the exon, as well as generates an internal guide sequence. Additionally, there is a pocket produced to encourage binding of the Guanosine cofactor. The Guanine nucleotide is placed on the first nucleotide of the intron. The 3'OH of Guanosine group nucleophilically attacks and cleaves the bond between the last nucleotide of the first exon and the first nucleotide of the 5' end of the intron; concurrently, trans-esterification occurs between the 3'OH and the 5'phosphorous from the 5' end of the intron. Subsequent conformational rearrangements ensure that the 3'OH of the first exon is placed in proximity of the 3' splice site. In this way, further trans-esterification reactions and splicing occurs.
Retreived June 21, 2011 from Self-Splicing RNAs [1] This diagram shows the trans-esterification reaction and splicing of group I introns from a sequence.
1.3 Fusion Proteins
Fusion proteins are combined forms of smaller protein subunits and are normally constructed at the DNA level by ligating portions of coding regions. A simple construction of traditional fusion protein involves inserting the target gene into a region of the cloned host gene. However, the subsequent project design, in its simple construction, interrupts the cloned protein with ribozyme sequences flanking a stop codon. The method proposed deals with excision and ligation at the RNA level, therefore, the unaltered DNA sequence does not code for a functional protein. The ligation of protein coding sequences can create functional fusion proteins for many applications including antibody or pesticide production; however, this method of production is limited to producing the same fusion protein each time since the sequence is not modified in between the transcription and translation phases of gene expression. One disadvantage of this is the resultant resistance of a pathogen to antibodies or a target organism to pesticides. For example, a specific pesticide (Cry toxin) may eventually not be effective to its target plant if subsequent plant generations inhibit its uptake, overproduce the sensitive antigen protein so that normal cellular function persists, reduce the ability of this protein to bind to the pesticide or metabolically inactivates the herbicide. Similar mechanisms contribute to antibiotic resistance. Any resistant organisms will inevitably prevail in subsequent generations. Recombination sites could potentially be incorporated into the subsequent project design to circumvent some of the difficulties with traditional fusion proteins as a result of host resistance. However, recombination sites may interrupt the functional fusion protein from forming. Ribozyme segments at the RNA regulation level can potentially remove disrupting sequence after such shuffling occurs. Therefore, the intervening sequence maintains its DNA level functionality but is removed when no longer needed at the RNA level. Fusion protein design focused on the DNA level does not have this dynamic regulation.
1.4 The Cre-Lox System
In bacteriophage P1 exists the cre enzyme and recognition sites called lox P sites. This viral recombination system functions to excise a particular DNA sequence by flanking lox P sites and introduce the cre enzyme when the target is to be excised. The cre enzyme both cuts at the lox P site and ligates the remaining sequences together. The excised DNA is then degraded. This is similar to our project design; however, instead of requiring the addition of an enzyme at the desired excision time, the self-excising nature of ribozymes automatically functions during the normal process of gene expression (RNA level).
2.0 PROJECT IN DETAILS
2.1 EXPERIMENTAL DESIGN
Our protocol will involves the insertion of a functional protein, split by the self-removing elements, between CUCUUAGU and AAUAAGAG in the P6 region of twort.ORF143. GFP (green fluorescent protein) is split into two parts, which will be referred to as GFP1 and GFP2. With a constitutive promoter, GFP1 and GFP2 will be separated by a class 1 A2 intron split into two (for now, IN1 and IN2) sequences that flank another sequence inserted into the P6 loop, which was chosen because anything attached to this region will remain outside the protein. Note that this experimental design also contains an in frame stop codon, which is expected to be spliced out of the sequence with IN1 and IN2 and will utilize the RFC53 convention. Following GFP2 is a transcriptional terminator (TT). The method of making this construct is detailed in RFC53. Below is Figure 1 through Figure 3. They illustrate the order of parts in the design and the trans-esterification reaction that results in a function GFP:
Figure 2 shows the experimental design of the sequence immediately following transcription. It contains a constituent promoter, RBS Ribosome Binding Site), GFP1, IN1, in-frame stop codon, IN2, GFP2 and TT. The dotted lines and scissors indicate that the introns will be spliced out of the sequence at these points, however, the introns are self-excising.
Figure 3 is a representative view of the sequence folding in order to catalyze the trans-esterification reaction, however, there are many hairpin loops actually formed. This is the process of post-transcriptional modification. Specifically, Group I intron splicing events utilize a guanosine nucleotide to bind another sequence and dislodge the 5' site, then the cleavage initializes another splicing event with the remaining hydroxyl end to dislodge the rest of the RNA sequence and ligate the remaining exons. The remaining fusion protein code is different than that of the primary transcript.
Figure 4 shows a non-disruptive ligation scar and active GFP after the self-excision of IN1 and IN2. This is the modified RNA transcript prior to translation..
2.2 CONSTRUCTION MAPs AND RFC 53
As per RFC 53 convention, enzyme digestions are followed in the particular order outlined below. The standard procedure makes this technique reproducible, therefore, more easily extrapolated to other applications. Compared to other protein fusion methods, this design facilitates additional regulation within necessary guidelines. However, the embedded post-transcriptional modification in this design is a complication to consider in simpler designs where regulation at this level is not necessary. As such, unnecessary bulk in plasmid vectors is known to add to metabolic load and decrease replication rate compared to non-plasmid carriers.
2.2.1 General Construction Map
Figure 5 graphically shows the laboratory procedure for the experimental design in the form of an enzyme map:
K576005 contains the first component of GFP (GFP1)
K576003 contains the first part of the intron sequence (IN1
J61046 contains the lox site
K576006 contains the second component of GFP (GFP2)
K576004 contains the second part of the intron sequence (IN2)
K576007 contains GFP1 and IN1
K576009 contains GFP1, IN1 and lox1
K576011 contains the promoter (P), ribosomal binding sit (RBS), GFP1, IN1, lox site, IN2, GFP2 and transcriptional terminator (TT). This is the final construct (experimental design)
2.1.2 Controls' Construction Map
Controls are necessary to prove that the design of this experimental investigation is functional and more practically for comparison of fluorescence in the laboratory. In the positive control, GFP1 and GFP2 flank either RFC25 or RFC53, which will not disrupt translation regardless of the linker. Therefore, fluorescence is expected. The experimental run will ideally show fluorescence resulting from the self-excision of IN1 and IN2.
Figure 6 represents the positive control. For consistency, GFP1 and GFP2 are separated by RFC53 (RFC25 could have also be used), which should not disrupt expression of a functional GFP.
In the negative control (using the same constitutive promoter), GFP1 and GFP2, followed by a transcriptional terminator, flank RFC10 (Request For Comments) resulting in a stop-codon-containing scar. No fluorescence is expected for this component (background) because translation is interrupted. This is meant to control for the possibility of a non functional fusion protein. The expectation is that this fusion of GFP1 and GFP2 will not fluoresce, which is a consequence of some fusion protein techniques. Figure 7 shown below details the negative control design:
Figure 7 is an illustration of the negative control containing the interrupted GFP with an in-frame stop codon, ensuring that it is not expressed. Since this sequence contains no excision sites (Group I introns), we do not expect any modification to this area of the RNA.
The figure below shows the construction map for the controls.
K576005 contains the first component of GFP (GFP1)
K576006 contains the second component of GFP (GFP2)
K576013 contains the promoter (P), ribosomal binding site (RBS), GFP and transcriptional terminator (TT). This is the positive control.
K576005 contains the first component of GFP (GFP1))
J61046 contains the lox site
K576006 contains the second component of GFP (GFP2)
contains the promoter (P), ribosomal binding site (RBS), GFP1, lox site, GFP2 and transcriptional terminator (TT). This is the negative control.
2.3 MAKING THE CONSTRUCT WITH RFC 53
Figure 9: Making the Construct: Parts were sequenced into PUC57, digested and ligated into PSB1C3.
Figure 9 is a flow chart of the general work flow involved in the construction of our experimental plasmid, as per RFC53 conventions.
1) The insert is isolated through a series of enzyme digestions. One intron (in blue) is shown here as a representation. The insert is isolated for subsequent ligation.
2) Similarly, the pSB1C3 vector is isolated through enzyme digestion. Note that "N" indicates that this is the vector portion. The vector is also isolated for the ligation step. It must also be noted that pSB1C3 vector contains a cut site of SacI, an enzyme that is used in RFC 53. Relocating the part in BBa_K371053 resolves this issue.
3) The two components (insert and vector) are ligated together to produce the final construct.
4) According to the experimental design, the final construct will contain self-excising ribozymes, which in the last step result in a non-disruptive ligation scar and, therefore, the expression of GFP.
2.3 Preliminary Testing
Although completion of a preliminary version of the final construct was achieved, lack of GFP fluorescence proved suspicion of questionable band placements during second and third stage electrophoresis. Final diagnostic digestion reaction confirmed abnormalities from designed constructs. Testing via digestion was completed for every intermediate, control and final constructs. Consequently, BBa_K576003, K576004, K576005, and K576006 were the only parts able to be confirmed. All the other intermediates and constructs have questionable band location which disrupted final construct fluorescence.
Figure 10: shows the diagnostic digestion from September 16, 2011: All subclones shown to be correct, however, positive control not correct. Relative placement of GFPs and INTS required further verification and showed incorrect placement. This accounts for lack of fluorescence in final stage.
The above electrophoresis picture describes the resultant bands from the diagnostic digestion. Although bands 5, 6 and 7 (sub clones) have been confirmed, the adjacent positive control (band 8) and all GFP and intron digestions are not consistent with the expected patterns. The GFP-INT and GFP-INT-lox constructions (bands 9, 10 and 11) have been verified as inaccurate. The questionable placements of these bands indicate that the cut sites, thus the fragment length and containing sequence, do not match the planned construction. Therefore, it is not likely that they contain functional GFP, introns or lox, which would result in a lack of fluorescence in the final stage of construction. Further testing to reconstruct the contaminated clones is necessary for the functional final product; however, lab work has stopped due to time constraint. A diagnostic digestion at each step is recommended to circumvent any similar issues upon the continuation of this project.
3.0 PRACTICAL APPLICATION
The biggest advantage of the ribozyme project is the ability to create in vivo protein fusions. These can then be applied to a larger number of modular systems that can be used to make complicated expression systems. One such system is the creation synthetic antibodies. If protein sequences are flanked by intron sequences and then set up along the same stretch of DNA, different combinations of fusion proteins will be created based on how the intron excision occurs. Another system where the ribozyme project can be applied is DNA shuffling experiments. The Cry toxin is used as an effective biopesticide, however for now it has a very small range of insects that it effects. The ways to increase its range would be to change the structure of one of the vital domains so that it is able to recognize a wider spectrum of receptors in the host mid gut cell. To create different variations of this domains an in vivo DNA shuffling experiments using the ribozymes could be carried out.
4.0 REFERENCES
Belfort,M., Cech, T., Celander, D., Chandry, P., Heuer, T. (1991). Folding of group I introns from bacteriophage T4 involves internalization of the catalytic core. Howard Hughes Medical Institute, Department of Chemistry and Biochemistry, University of Colorado. 88(24): 11105–11109.
Belfort, M., Chu, F., Maley, F., Maley, G. and West, D. (1986). Characterization of the lntron in the Phage T4 Thymidylate Synthase Gene and Evidence for Its Self-Excision from the Primary Transcript. Wadsworth Center for Laboratories and Research. Vol. 45, X7-166.
Bernstein, K.E., Bunting, M., Capecchi, M.R., Greer, J.M., Thomas, K.R. (1999). Targeting genes for self-excision in the germ line.
Cassin, P., Gambier, R., Scheppler, J. (2000). Biotechnology Explorations: Applying the Fundamentals. Washington, DC: ASM Press.
Cech, T. (1990). Self-Splicing of Group I Introns. Biochemistry 59:543-8.
Clancy, S. (2008) RNA splicing: introns, exons and spliceosome. Nature Education 1(1). Genetics Primer, Fanconi Anemia Genetics. Last updated 08 February 2004. (http://members.cox.net/amgough/Fanconi-genetics-genetics-primer.htm).
Glick, B., Pasternak, J., Pattern, C. (2010). Molecular Biotechnology Principles and Applications of Recombinant DNA Fourth Edition. Washington, DC: ASM Press.
Goldberg, M., Hartwell, L., Hood, L., Reynolds, A., Silver, L., Veres, R. (2008). Genetics From Genes to Genomes Third Edition. New York: McGraw Hill Companies.
Group 1 Intron Sequence Structure and Database (http://www.rna.whu.edu.cn/gissd/alignment.html). Howard Hughes Medical Institute, Department of Chemistry and Biochemistry, University of Colorado. 88(24): 11105–11109.
Ikawa, Y., Inoue, T., Ohuchi, S., Shiraishi, H. (2002). Modular engineering of Group I introns ribozyme. Graduate School of Biostudies, Kyoto University. 30(15): 3473-3480.
Landthaler, M. and Shub, D. (1999). Unexpected abundance of self-splicing introns in the genome of bacteriophage Twort: Introns in multiple genes, a single gene with three introns, and exon skipping by group I ribozymes. Microbiology Vol. 96, pp.7005–7010.
Minnick, M.F., Raghavan, R. (2009). Group I Introns and Inteins: Disparate Origins but Convergent Parasitic Strategies. Journal of Bacteriology. 191 (20), 6193-6202.
Peters Ph.D., Pamela (N/A). Restriction Enzymes Background Paper An Excellence Classic Collection. (http://www.accessexcellence.org/AE/AEC/CC/restriction.php).
Self-Splicing RNAs (http://mol-biol4masters.masters.grkraj.org/html/RNA_Processing3C-Self_Splicing_RNAs.htm). http://www.bio.davidson.edu/courses/genomics/method/CreLoxP.html
Motivation and Goals
This year’s modelling project focused on extending the work done by the modelling team in 2010.
Waterloo’s 2010 iGEM project, "Staphiscope", utilized amplifier parts developed by Cambridge in 2009 to detect low levels of Staph Aureus. These amplifier parts were characterized by the Cambridge team, but only under control of AraC/pBAD promoter, which differed from the promoter used in our 2010 Staphiscope project.
In order to characterize the amplifiers, a parameter scan was undertaken to find promoter-independent Hill parameters of each amplifier, consistent with data of full system. However, empirical verification of our results was lacking. This year, we sought to obtain this data, which (in conjunction with Cambridge data and model), would allow us to find Hill parameters for each amplifier.
Model
To allow for comparison of data, we used the same model as Cambridge in 2009 .
In this model, araC represses the pBAD promoter in the absence of the inducer, arabinose. When arabinose is present, it binds to araC, preventing repression of the promoter and allowing transcription of reporter (GFP). This situation is modelled by a Hill function; we seek the Hill parameters of this function.
Thus, when AraC/pBAD system is induced with arabinose, we expect to see a steady increase of fluorescence from a low level, followed by a plateau of fluorescence at steady state.
Method
To measure fluorescence, we closely followed the assay described in the paper "Measuring the activity of BioBrick promoters using an in vivo reference standard" , in the section "Assay of Promoter Collections".
Three cultures were grown overnight at 37 degrees Celsius with spinning at 200 rpm: untransformed BW27783, BW27783 containing BBa_I0500, and BW27783 containing BBa_I20260. These were then diluted 1:100 and regrown for roughly 4 hours under the same conditions. They were then diluted to an OD between 0.05 and 0.09, and regrown for 1 hour, again under the same conditions.
After this, the cultures were diluted into a 96-well plate at 8 different concentrations of inducer (arabinose), ranging from 0 to 6.4 uM. The plate was then incubated in a Wallac Victor3 multi-well fluorimeter at 37 degrees Celsius, and repeating measurements of absorbance and fluorescence were taken at 10 minute intervals, with shaking after each measurement. Untransformed BW27783, at each concentration of arabinose, was used to measure background fluorescence, and wells containing only broth were included to measure background absorbance. The machine settings used were identical to those described in the paper referenced above.
With this data, we aimed to calculate the steady-state per-cell GFP concentration during log-phase growth, for both BBa_I0500 and BBa_I20260 (measurement kit for the standard promoter, J23101). The ratio of these values would then characterize the strength of the AraC/pBAD promoter in units of RPU. The justification for this approach can be found in the supplemental material of the paper referenced above.
Results
The results of the experiment were anomalous, and considered too unreliable to be conclusive. There was no clear relationship between cell fluorescence and inducer concentration.
The fluorescence curve did not qualitatively match the predictions of the model; across all concentrations, and for each of the 3 cultures, we observed a high initial fluorescence, with a rapid drop to a lower steady state value. For each culture, this drop in fluorescence aligned well with the growth curve.
In addition, the untransformed BW27783 cells exhibited consistently higher fluorescence than cells containing BBa_I0500, which was highly anomalous. Because of this, we could not reliably use these cells to measure background fluorescence.
Below, a sample graph of Total Fluorescence is shown for each of the 3 cultures. These are curves of the total fluorescence for each culture, averaged over 3 replicates for each culture.
Discussion
It is believed that an error in our strain of BW27783 is most likely responsible for the anomalous qualitative features of our data. This is because for each concentration of inducer, the untransformed BW27783 cells exhibit a fluorescence curve highly similar to that of BW27783 containing BBa_I0500, and yet the untransformed cells should not be expressing GFP.
Prior to the measurement assay, BW27783 cells transformed with BBa_I0500 were plated and examined for fluorescence, both with and without the presence of inducer. The uninduced cells were not found to fluoresce, while the induced cells did fluoresce. The fluorescing cultures were used to make the frozen stock of BBa_I0500 which was used in the measurement assay. This indicates that our untransformed BW27783 should not fluoresce without the presence of inducer. Furthermore, the untransformed BW27783 cells used in the measurement assay were at no point prior to the assay exposed to arabinose.
To explain the fluorescence of the untransformed BW27783 in the measurement assay, it is speculated that our strain of BW27783 exhibits a rapid production of GFP in response to even low concentrations of inducer. Experimental error is also a likely source of inaccuracy in the data, although the qualitative features described were consistent across 3 trials of the experiment. Research into these results is still ongoing.
University of Waterloo: iGEM OUTREACH
The purpose of UW iGEM Outreach has always been and will continue to be the connection between our community and us. To help build a better understanding of synthetic biology, how it has affected the world around us and to create a basic, fundamental knowledge of the subject that can be incorporated into the way we see things. That perspective can be positive or negative, but we aim to provide the baseline knowledge required that will allow our community members to form a fact-based opinion.
This year, UW iGEM: Outreach focused on designing and running workshops targeted at schoolchildren. We hoped to share our love and passion for biology with tomorrow's future scientists and engineers. We plan to continue building on what we currently have and to eventually develop a complete syllabus for all grade levels. These workshops will be available for download for other educators and enthusiasts interested in their own outreach.
Workshop Materials
We have submitted two community bricks! One for our Grade 12 workshop and the other for our Engineering Science Quest activity for Grades 3-4. The downloadable material is the same as what you can find here on our wiki page.
-Synthetic Biology and You: Interactive Workshop for Grades 11-12
-All About Bacteria: How Clean Are Your Hands?
Grades 3-4: All About Bacteria - Outline | Handout (Duration: 2-day workshop, 1.5 hours total)
Grades 5-6: All About DNA - Outline (Duration: 1 hour)
Grade 12: Synthetic Biology and You - Materials | Ppt (part 1) | Ppt (part 2) (Duration: 2-3 hours)
Are you interested in doing any of these activities with kids around this age? Feel free to use any of our materials above and/or contact us at uwigem.outreach.hp@gmail.com. It is an inexpensive, interactive and fun way to have kids involved in genetics, microbiology and synthetic biology at a very early age. And definitely a lot of fun!
Events
We were also fortunate enough this year to have been given the opportunity to run our workshops at two different outreach events, both of which were on a grand  scale. Now, we'd like to share our experiences with you.
Grade 12 Outreach Workshop: March 25th, 2011
The first was an organized Grade 12 workshop aimed at biology students to gain a better understanding of synthetic biology, the industries it has been affecting, career prospects as well as two hands-on activities. Over the planning span of 3 months, this event was organized in close accordance with the Kitchener-Waterloo school board and with the Marketing and Recruitment Co-ordinator for Science at the University of Waterloo. Through the creation of a brochure and meeting with individuals from the school board we had sent out an invitation all across the district for students to come in for our event. Eventually we had gotten more than 85 students to attend, which was great as it was the first time we had implemented this idea.
The next step had then been to recruit interested volunteers for the event so we could have our own students give a helping hand and who shared the same passion of sharing knowledge as we did. In order to facilitate this we had sent out emails, gone to various lectures and talked to students all over campus to get them involved. After recruiting 12 volunteers the brainstorming process had begun. We had wanted to have an interactive workshop where students were not just listening to us talk, but were actually involved in a stimulating activity that they could be excited about. The first part of our presentation looked into what was synthetic biology, what were Genetically Modified Organisms (GMOs) and what were the positives and negatives of them. Once that was through we started the first event.   The first event we had was to talk about our very own Canadian genetically enhanced Yorkshire pig called the EnviroPig™. It has the capability to digest plant phosphate more efficiently than traditional Yorkshire pigs, which do not contain the enzyme to break it down, phytase. This gene can be found in E.coli which had been inserted into a pig embryo to allow it to produce phytase in its salivary glands. This is a real technology in Canada and is currently a very hot topic of debate among many citizens, so we thought it would be ideal to introduce students to the world of biotechnology right in their very own backyards. Once students had been exposed to the information, we had given them a package which we had compiled giving the positives and negatives of societal, technological, ethical, environmental and economical issues.
Of course, it did not end there! Students were then put into groups of 5 and as mentioned above, one of the key goals of UW iGEM Outreach is to allow members of our community to make informed decisions based on facts. Supplied with markers and paper students were to give a 1-2 minute presentation on why they did/did not believe that the EnviroPig™ should continue receiving funding from the government of Canada. The best and most convincing presentation won- we had great discussions from the negative aspects to positive aspects to even a compiled rap song about the EnviroPig™!
After the first event we continued on with our presentation. Here we discussed current industries that had been affected by synthetic biology. This incorporated the pharmaceutical and biofuels industries specifically. We discussed how pharmaceuticals had incorporated an aspect known as biosynthesis where they could customize and fine-tune certain pathways using standardized parts in order to provide efficient and precise drug delivery systems. In biofuels we talked about first and second generation biofuels and how the use of synthetic biology has the ability to create a ‘superbug’ to look for corresponding metabolic pathways for yielding ideal results. After this we did the second activity.
The second event that had been implemented was to incorporate what we do in our labs, outside of the lab. Essentially, we wanted to introduce the idea of synthetic biology to students and how it was an extension of what we knew as genetic engineering and consisted of students not just from science but from math, engineering and computer science. The activity called, ‘Design Your Own Pathway’ gave a series of scenarios we had given students with a library of BioBricks to create a certain pathway. Progressively each scenario was harder, more complicated and required the use of multiple BioBricks. The BioBricks that we had used were from the library and were real parts. Essentially this was to enable students to have a feel of how we have a ‘mix and match’ concept when it comes to synthetic biology. The activity had been set up as a relay race, where students in the same teams as the previous activity had to race each other to finish all scenarios. The activity and concept had been such a success that after the workshop teachers had asked to use our activity in their own classrooms.
Finally, we completed the presentation with a discussion of career paths that students could pursue that did not necessarily involve working in a lab. We wanted students to understand that the world of synthetic biology and biotechnology encompasses the involvement of individuals from multiple backgrounds and can penetrate into many different industries.
There was such an interest in our workshop that our prospects for the next one are aimed at more than 150 students. One thing’s for sure, mission accomplished and there’s definitely more to come!  Would you also like to have your own workshop at your high school or university? Please feel free to view the downloadable materials for the presentation and two activities or contact us at uwigem.outreach.hp@gmail.com.
Engineering Science Quest: July-August 2011
Founded in 1990, the Engineering Science Quest or ESQ is a not-for-profit program that operates with the goal of exposing children in the Kitchener-Waterloo region to the world of engineering, science and technology through engaging them in a variety of hands-on activities. Promotion is primarily done through workshops in-school but also have satellite programs which reach out to rural and Native communities as well.
This is not the first time that UW iGEM has been involved in ESQ and we are proud to say that our continued involvement has allowed us to develop a standard set of activities which we are pleased to present to kids ranging from Grades 3-6 with more than 100 students. Currently we are also developing ideas for older kids that are similar to our activities from the Outreach workshop we had in March for Grades 10-12 and for even younger kids from Grades 1-2. Through involvement with managers specifically for ESQ this year we were able to have continual workshops every week from July 11th- August 12th, 2011. This was done with the recruitment of volunteers who again shared the same passion as we did in connecting with our community to facilitate that baseline knowledge; to get students introduced or even extend their knowledge on the world of synthetic biology and biotechnology.
The first activity for Grades 3-4 was called, “All About Bacteria: Do You Really Need to Wash Your Hands?” In this activity we introduce kids to the idea of biology, bacteria, synthetic biology, and iGEM. We also introduce them to basic ideas of sanitary techniques and tools used in a standard lab such as petri plates, agar, swabs etc. and how to layout an experiment; what is your hypothesis, results and conclusions? Once we discuss these basic concepts we allow the kids to take a swab of their hand and plate it on half of a petri dish. They then clean their hands with sanitizer and swab the other half of the petri dish. They then receive another petri dish where they can swab other places to find other ‘neat’ bacteria that may be lying around on the floor, counters, door knobs or wherever else they want (except up their nose, in their eyes, ears or mouth!). At this point and throughout the activity, interaction with the kids is key, as they always have stories or thoughts and experiences that are enlightening to share- even to us university students.
Once we explain what incubation is, we allow kids to come in the next day and see all the different types of bacteria that may have grown on their petri plates and come up with conclusions from the results they have seen. We provide them with a record sheet where they are able to write down what they saw such as; morphology, colour, its surface and even elevation. To be able to interact with kids at such a young age and to introduce them to concepts which students are normally exposed to in higher grades, allows them to explore a part of their world that they never knew existed. Building curiosity at a basal level helps us connect with our community and bridge the gap towards making more informed decisions.
The second activity for Grades 5-6 was called, “DNA Extraction from Your Cheeks”. This activity centers around the idea of DNA, where it is found, what it looks like and how every living organism contains very similar genomes, proteins and enzymes. There were four steps to this process, first to collect cheek cells, second to burst cells open to release DNA, third to separate DNA from proteins and debris and finally isolate the concentrated DNA. Kids obtain a cup of Gatorade containing a saline solution and swish the drink in their mouth for about a minute while gently chewing on their cheek cells. Then detergent is added to the test tube and meat tenderizer is added and the tube is inverted gently a couple of times. Cold rubbing alcohol is then added with a pipette which should allow the DNA to be visible. Then kids   transfer the DNA into a PCR tube where they can hang it on a string to make a really neat necklace.
Kids were keen and interested in these activities, especially learning about various different concepts that they had not yet been exposed to. “Who knew that DNA could come out of your own cheeks?” was the idea we wanted to pass along- that every living organism contains DNA. From bacteria to animals.
Are you interested in doing these activities with kids around this age? Feel free to contact us at uwigem.outreach.hp@gmail.com. It is an inexpensive, interactive and fun way to have kids involved in genetics, microbiology and synthetic biology at a very early age.
Introduction
Synthetic Biology has been perceived as a field of science that has introduced tools, concepts and technologies which are sometimes sociologically and ethically insupportable. The non-scientific community has developed a stigma against the application of various synthetic biology technologies. The problem does not lie within the technology, but by an institution’s poorly executed strategy in educating the prospective end user about the relative benefit of its product. At times, the rate of technological advancements in the bioengineering industry surpasses the end-user’s ability to understand the necessity of the technology.
The science behind these synthetic biology tools has been the current driver in instilling a sense of appeal and intrigue within the scientific community. However, it is critical that the institutions that develop these technologies educate the non-scientific community of the importance and benefit of these tools to help relieve these evolving stigmas.
This year, the University of Waterloo’s Human Practices team’s focus was to develop a marketing strategy for their 2010 design project, the Staphiscope. The Staphiscope is an enhancement to the existing conventional plating methods that are used to detect Methicillin Resistant Staphylococus Aureus.
The scope of the analysis includes:
.
What is the selling point of the product? What differentiates the Staphiscope from what already exists and is accepted by the non-scientific community?
Does the concept of competition exist within synthetic biology? If so, how does an institution market the Staphiscope to optimize their opportunities for growth?
How can the marketing strategy approach be differentiated to ensure that both the scientific and non-scientific community are exposed to the relative benefits of the Staphiscope?
How do we channels the knowledge and learnings acquired within the synthetic biology industry and channel it to other stakeholders that will prospectively be impacted by the technology?
The purpose of the analysis is to ensure that the scientific industry understands the importance of bridging the scientific theory of a product to its economic feasibility. Moreover, it encourages institutions who participate in developing synthetic biology technologies to align their perception of a revolutionary design to what will be socially acknowledged by a non-scientific community.
What is the Staphiscope?
Methicilin resistant Staphylococcus aureus (MRSA) are bacteria whose presence has been quite problematic in terms of human pathogenic infections. Since its discovery in the 1880s, Staphylococcus aureus has been the known cause for several kinds of minor skin, and major post-surgical infections. Prior to 1940, mortality rates in relation to this pathogenic organism reached 80% (Deurenberg and Stobberingh, 2008.) The first wave of resistant S. aureus came two years after the introduction of penicillin for medicinal practice in 1940. In less than twenty years, most of the S. aureus strains known to man were unaffected by the antibiotic. In the late 1950s, a penicillinase-resistant penicillin was brought forth to the medicinal market; this was methicilin (Deurenberg and Stobberingh, 2008.) The bacterium's subsequent resistance to methicilin came two years after its introduction in 1959. The mecA gene responsible for methicillin resistance is also to blame for the organisms’ desensitization to several classes of antibiotics (Deurenberg and Stobberingh, 2008). This has led to the creation of the term: MRSA, now commonly used worldwide.
While an MRSA infection is much like a S. aureus infection, the difference comes in the lack of sensitivity of the MRSA to several classes of antibiotics. This makes MRSA infections a serious threat in both health and economical aspects.
The infection could be easily transferred directly (through regular skin-to-skin contact), as well as indirectly (through contamination of surfaces). The ease with which infection could occur, as well as the high mortality rate involved with the infection, make MRSA a serious threat to the health of patients worldwide (Durai et al., 2010.) There are only a few methods, which currently exit for MRSA screening. These are plating, liquid-broth inoculation, and PCR (polymerase chain reaction) assays. While these will be discussed in further detail later on in the report, it is important to note that these methods could be costly and time consuming, and could sometimes present incorrect results (Durai et al., 2010.)
In 2010, the International Genetically Engineered team at the university of Waterloo had been working on a method which can be used in supplement to the already-existing ones for the detection of MRSA infections. It is a proof of concept for diagnosing the presence of Staphylococcus aureus in an infection site before it has had the chance to create an infection. This synthetic biology–based diagnostic method will take advantage of the quorum sensing mechanism of Staphylococcus aureus, and utilize Escherichia coli as the sensor and reporter.  For more information with regards to the Design of the Staphiscope please see the University of Waterloo’s 2010 iGEM Page: https://2010.igem.org/Team:Waterloo
Who are the “Competitors”?
Treatment of an S. aureus infection is of extreme importance, as outcomes can range from simple pustules to death. Early detection and treatment of S. aureus carriers can reduce healthcare costs and greatly improve the health of the patient. There are only a few methods currently available to hospitals for the diagnosis of S. aureus infection. These methods are time consuming, expensive and, in some cases, not very reliable. The purpose of the Staphiscope is to provide the option of accelerating the method of detecting S. aureus, without compromising reliability. It will be a particularly useful tool in speeding up the conventional methods of serial plating.
The following section analyzes the current products on the market and compares them to the Staphiscope.
BD GeneOhm StaphSR
Traditional culture methods for the detection of S. aureus are usually 2-3 days. The new BD GeneOhm™ StaphSR delivers a diagnosis in less than 2 hours, using rapid real-time PCR. Also, it can simultaneously detect and differentiate Methicillin-Resistant S. aureus and S. aureus from a positive result (blood culture). Currently under development are additional specimen claims – nasal and wound samples. Nasal claim will be able to determine colonization status and the wound claim will add to the rapid identification of positively infected patients. Obtaining positive results in a timely fashion aids clinicians in the prevention and control of infections.
Method
Specimen is prepared by transferring an aliquot of positive blood culture into a buffer team and then vortexed at high speed.
Cell suspension is lysed (via vortex and centrifuge) and heated. Then the lysis is cooled. Molecular reagents are reconstituted and the sample undergoes PCR. Results are obtained in about 1 hour.
References
"BD - GeneOhm - Products." BD: Medical Supplies, Devices and Technology; Laboratory Products; Antibodies. BD, 2011. Web. 23 July 2011.
.
GenoType MRSA
Genotype MRSA uses DNA Strip Technology to identify both S. aureus and S. epidermidis cultures. Therefore, coagulase-positive and coagulase-negative staphylococci are differentiated. Simultaenously, the product can also detect Methicillin-Resistant S. aureus and S. aureus , which is incredibly important for therapeutic purposes. The test is performed from an overnight culture and guarantees results in only 4 hours.
Method
Culture is isolated from a sample or a primary culture is used.
DNA is extracted via methods producing amplifiable DNA from bacteria (for example     QIAamp DNA Mini Kit from Qiagen).
Nucleic acids are selectively replicated in an amplification reaction. In the next step, the amplicons are chemically denatured, since detection on the DNA•STRIP® is done using single-stranded DNA. During the conjugate reaction, the specifically bound amplicon is marked with the enzyme alkaline phosphatase and is then made visible in a colorimetric detection reaction. In this way, a specific banding pattern develops on the DNA•STRIP®. Using a test-specific evaluation template, the test result can be read out quickly and clearly.
References
"GenoType® MRSA | Identification of MRSA from Cultured Material." Hain Lifescience | Ihr Partner in Der Modernen Labordiagnostik. Hain Lifescience, 2011. Web. 23 July 2011. .
"DNA•STRIP® Technology." Hain Lifescience | Ihr Partner in Der Modernen Labordiagnostik. Hain Lifescience, 2011. Web. 23 July 2011.
.
Brilliance Agar
Brilliance Agar is a chromogenic screening plate used for the diagnosis of Methicillin-Resistant S. aureus. It uses a novel chromogen to show a blue colour when Methicillin-Resistant S. aureus phosphatase activity. For sensitivity, it contains antibacterial compounds which inhibit growth of various competitor bacteria. This product has high sensitivity and specificity, which minimizes healthcare costs. However, results are presumptive and would need to be confirmed. Organisms with atypical resistance can give false negative results.
Method
Patient is inoculated using a screening swab or an isolated colony or liquid suspension is used. Sample is plated on Brilliance Agar. Plates are incubated at 37 ºC. Results are obtained in 18 hours. Methicillin-Resistant S. aureus grows as denim blue colonies.
References
"Brilliance MRSA AGAR." Oxoid - Worldclass Manufacturer of Dehydrated Culture Media and Diagnostics Products. Oxoid, 2010. Web. 23 July 2011.
.
Competitive Analysis Matrix
Rating Scale
Methodology for Rating Scale
The Competitive Analysis Matrix uses a format that divides the product’s market profile into several categories, providing a framework that ensures all issues are considered.  Each category is scored in the course of the ratings process (using a numerical scoring system). Products are scored 1 through 27; those facing greater competitive threats would wind up with an overall low market profile score. Please note that ratings represent, in the end, an opinion. Please note, thorough assessment of each product’s market profile requires a broader framework, involving a comprehensive review of the product’s competitive position.
Conclusions
Based on a comparison and competitive analysis of the Staphiscope and the three major detection products on the market, the Staphiscope was found to have the highest market profile score. It uniquely scored highest in reliability and ease of use. Although all products on the market signify great innovation in the field, Staphiscope has great potential to gain market share.
Porter’s Five Forces analysis
Marketing Strategy
Pitching the Staphiscope to a Research and Development Company When pitching the Staphiscope to the R&D industry, it is of paramount importance to establish its competitive advantage over traditional methods for S. aureus detection. To have a competitive advantage, a product or company must excel in at least one of the three following categories while remaining equal to its competitors in the others (Szarka, 2010):
Market access
Infrastructure
Technology
Market access refers to how easily the target demographic can get the product. Even a revolutionary technology can be hampered by a lack of market access. Infrastructure is related to whether or not a supply chain exists, as well as facilities for production and distribution. The third point, technology, is where the Staphiscope excels; its superior technology is what will separate it from competitors. Other selling points of the Staphiscope include its speed, efficiency, and room for expansion (could this technique be applied to other cases?). Conversely, there are also barriers that must be overcome when pitching to the R&D industry; namely, one must avoid falling prey to “not invented here” syndrome (Szarka, 2010). This term applies to the mindset some companies have of only trusting in-house innovation, and can be difficult to alter. Other barriers include the lacking of Staphiscope use on a commercial scale, and public perception regarding E. coli. While the latter issue can be combated via a public awareness campaign, the former is simply a fact of any innovative technology.
Companies such as Amyris Biotechnologies and Codon Devices have attempted to commercialize in the field of synthetic biology. Their experiences provide valuable lessons that can applied to the Staphiscope (Wilan, 2005). The most relevant of these is that patent protection should be a serious concern when developing a novel technology, especially in such a burgeoning and competitive field. Consolidating IP ownership and control early is equally important, as doing so will prevent a lack of unified vision from stalling the Staphiscope’s progress. Of course, there are a plethora of patent and IP issues surrounding the field of synthetic biology that must be dealt with before or during the commercialization process. Chief amongst these is the scope of protection that can be provided to new technologies (Chugh  Sakushyma, 2009). Additionally, looking for a pharmaceutical partner can help ease the transition into commercialization.
It is important to realize that commercialization is a multi-year effort. As such, it can be beneficial to develop a commercialization plan (Office of Technology Transfer). This plan gives a general outline of what information the R&D industry will be looking for when it comes time to make a pitch. Details about market size and growth rates, predicted sales figures for the first five years, licensing agreements and more can all prove invaluable when attempting to bring a product to market.  Preparedness in regards to this information can make the difference between a successful pitch and an unsuccessful one.
In summary, pitching the Staphiscope to the R&D industry is a complex process that must be handled correctly to ensure its successful commercialization.
References
Office of Technology Transfer. Argonne National Laboratory. “Commercialization Plan Worksheet”. http://www.anl.gov/techtransfer/pdf/LicensingQuestionaire1-0.pdf
Wilan, K.H. Nature Publishing Group. 2005. “Commercializing synthetic biology”. http://www.nature.com/bioent/startup/072005/full/bioent870.html
Chugh, A., Saukshyma, T. 2009. “Commercializing synthetic biology: Socio-ethical concerns and challenges under intellectual property regime”. http://stopogm.net/webfm_send/315
Szarka, M. “Adventures in Commercialization”. 2010. http://webcast.utm.utoronto.ca/1/watch/515.aspx
How Do We Achieve Public Acceptance?
For years, it has been believed that the prevalence of MRSA infections has resided within the walls of either a hospital or health care facility. Due to understated hygienic practices, a modified and relatively more virulent strain of Staphylococcus Aureus has also threatened several community based settings. In addition to Hospital Acquired MRSA, general health practitioners must also take into account the occurrence of community acquired MRSA.  It is important to educate the public about the applications and relative benefit of the Staphiscope. Thus, there needs to be a channel of communication that allows for a transfer of knowledge between the scientific and non-scientific community.
Who Do We Want To Educate?
Given that MRSA may not only be spread within a hospital but also within the community, there are quite a few demographics that would gain value in understanding the benefit of the Staphiscope.
Hospital Setting
At least 2% of patients in hospitals or health-care facilities are likely to carry a strain of Staphylococcus Aureus that is resistant to antibiotics such as Methicillin. The following individuals are susceptible in contracting an MRSA infection:
Individuals who have weak immune systems
People receiving kidney dialysis or cancer treatments
Individuals who have been hospitalized or have had surgery over the past year
MRSA within the Community
Patients who are discharged may have an inactive form of MRSA that remains colonized in their system. Thus, those who live within the community are still at risk of acquiring and suffering from antibiotic resistant Staph infections that are relatively more virulent. MRSA has been detected and acquired by healthy people in both institutionalized and individual based settings. The following individuals are susceptible in contracting MRSA in the community:
Day-care facilities or facilities in which equipment of personal items are shared
Military bases
Individuals who get tattoos
The Strategy
In the field of synthetic biology, there is an unrecognized paradox that hinders the level of interdisciplinary communication between the scientific and non-scientific community. The stigmas built outside the realms of a scientific community grow at a greater rate than a scientist’s ability to answer the plethora of questions surrounding the regulation, safety and potentiality of these synthetic biology technologies.  Another problem lies in a scientific institution’s inclination to address the ethical concerns surrounding their synthetic biology technology or process. Generally an individual who has not be previously exposed to the field, will construct an ethical judgement proceeding the phase where a company is about to commercialize their product. At this critical stage of a company’s product plan, stakeholders are reluctant in addressing these ethical concerns as it may discourage their ability to advance and innovate their existing technology.  Regardless of the threats that these ethical judgments have previously imposed, there is a necessity in developing an approach in which the end result is an internationally renowned sense of public acceptance and support.  “Without public support and understanding of research into synthetic biology, both funding and regulation are unlikely to support significant scientific advances” (Gaisser, et. al, 2009). To address this issue, an expert committee in Europe supported by the New and Emerging Science and Technology programme set out a roadmap that outlines the four interconnected attributes that will instil a sense of public acceptance within the non-scientific community. These four attributes are:
Scientific Milestones
Knowledge Transfer
Funding
Regulation
This roadmap outlines that without the implementation of a knowledge transfer strategy, there is limited confidence attaining in achievements under the other three key areas of the model. Scientific milestones would not be achievable without a sustained level of funding provided through other groups impacted by the prospective commercialization of these synthetic biology tools. Moreover, regulation of synthetic biology tools is influenced by the ethically themed perceptions created by both the scientific and non-scientific community.
The knowledge transfer strategy would be an interdisciplinary model that would impact the following stakeholders:
Natural and Social Scientists
Engineers
Industry Representatives
Non-governmental organizations
Non-scientific community (ie. hospital staff, hospital patients and members within institutionalized settings)
Decision makers (ie. potential investors)
Funding Agencies
Knowledge Transfer Strategy
In this context, knowledge transfer is the exchange of research knowledge between the institution producing the synthetic biology technology and their target end-user. The purpose of the knowledge transfer strategy is to develop a communication platform between all stakeholders to mitigate the unsourced perceptions, bias’ and stigmas against the field of synthetic biology. Moreover, it is better to develop an interdisciplinary team at the early stages of the technology development phases versus segregating the two communities until the technology is prospectively in its commercialization phase.
Method
The following are examples of methods that can be used by the synthetic biology industry in promoting tools such as the Staphiscope to its potential end-users (ie. individuals within hospitals and community facilities)
Development of an interdisciplinary network that consists of synthetic biologists, engineers, potential investors, and the general public
Sustainable dialogue between all stakeholders within the interdisciplinary network. This can be achieved through: Workshops that aim in exchanging information to individuals that will benefit from the application of the Staphiscope in either a hospital or community based setting
Educational materials that are specifically tailored to the different parties that could be impacted by the commercialization of the StaphiScope (ie. Hospital Microbiologists, Patients, and the General Public)
Incorporating the General Public and other stakeholders at the research phase of the technology development process. This results in a mutual benefit to both the institution as well as the end-user in that they are able to collaboratively discuss the ethical feasibility of the technology.
Use of knowledge broker- A knowledge broker is essentially an intermediary resource that assists in translating research knowledge to information that is comprehensible to a member that is not from the field of synthetic biology. An individual that has attained perceptions, knowledge and experiences from the scientific, business and social field is best suited to convey the importance of the Staphiscope to the stakeholders that have been listed above.
Partnerships between Synthetic Biology institutions and hospitals or other community based facilities.
Synthetic biologists must consider the importance of implementing an approach to help relieve the prospective stigmas that could potentially impact the relative success of their technology. The general public plays an influential role in the innovation capabilities of these synthetic biology technologies. Thus it is critical that these synthetic biology institutions involve the public and create awareness of their technologies at all stages of the research and development stage. Integrating a strategy that incorporates the exchange of knowledge  is a fundamental leverage that would help create an aura of public acceptance and support all dimensions of the synthetic biology roadmap.
Human Practices Commercialization Toolkit
Following the Regional Jamboree, the Human Practices team began to develop the tools, templates and models that would assist a synthetic biology community and its associate stakeholders to understand the commercialization process. More importantly, our aim is to develop informational material that would achieve the integration of a knowledge transfer strategy in the field of synthetic biology. By streamlining the knowledge transfer strategy throughout each stage of the commercialization process, we are able to achieve a paralleled level of communication between the scientific and non-scientific community. Below is a sample of the toolkit our team has developed.
The Commercialization Road Map
The University of Waterloo Human Practices Team has devised a commercialization road map specific to the field of synthetic biology. The purpose of this road map is to outline the different stages of the technology development process. More importantly, it indicates what initiatives must be completed at each stage, who needs to be involved, and the strategies that can be utilized to attain the intended end-goal; A Scientific Milestone. To view the road map please click the link below.
Synthetic Biology Commercialization Roadmap
The Knowledge Transfer Strategy Action Plan
The University of Waterloo Human Practices Team has devised a knowledge transfer strategy to help parallel the level of communication between the scientific and non-scientific community. This document outlines the strategies that are currently being used or proposed at the University of Waterloo but can be easily used on a wider scale by the entire iGEM community. To view the knowledge transfer strategy plan, please click the link below.
Knowledge Transfer Strategy
Conclusion
The purpose of this analysis was to understand the relative feasibility of a synthetic biology tool from the perceptions of a scientific and non-scientific community. When devising a marketing strategy, most companies define a technology’s commercial success by its appeal towards investors or its ability to enter a market heavily saturated by other established technologies. Given that most synthetic biology institutions are still at their preliminary phases in terms of commercialization, it is critical that their product is widely accepted and acknowledged by all disciplines that are impacted or will be potentially exposed to the technology.
In the future we hope to take our research and methodologies and present them to the stakeholders discussed within the analysis.  Our next steps are to identify institutions and facilities that would benefit from the use of the knowledge transfer strategy and develop sample activities or educational materials that can be used by these facilities to promote the Staphiscope.
References
A.D.A.M Health Solutions. "Methicillin-resistant Staphylococcus Aureus; Community-acquired MRSA (CA-MRSA); Hospital-acquired MRSA (HA-MRSA)." National Center for Biotechnology Information. 9 June 2011. Web.
.
"Ethical Issues in Synthetic Biology." Synthetic Biology Project. Web.
.
Gaisser, Sibylle, Thomas Reiss, Astrid Lunkes, Kristian Muller, and Hubert Bernauer. "Making the Most of Synthetic Biology : Article : EMBO Reports." Nature Publishing Group : Science Journals, Jobs, and Information. Nature Publishing Group, 2009. Web.
.
Herrera, Stephen. "Preparing the World for Synthetic Biology." Technology Review. Jan. 2005. Web.
.
The Lancet. "Community-acquired MRSA." 14 Oct. 2006. Web.
.
Mitton, Craig, Carol Adair, Emily McKenzie, Scott Patten, and Brenda Waye Perry. "Knowledge Transfer and Exchange: Review and Synthesis of the Literature." University of British Columbia Okanagan and Child and Family Research Institute of BC; University of Calgary; Alberta Mental Health Board. Web.
.
Pombo, David. "Community Acquired MRSA." 29 Sept. 2006. Web.
.
“Safety and Ethics of Synthetic Life." Organisation for International Dialogue and Conflict Management (Austria). Web.
.
Members
Advisors
About UW
OUR TEAM!
OUR UNDERGRADUATES!
Dan Barlow
Ekta Bibra
Angela Biskupovic
Diljot Chhina
Amanda Clarke
Arpita Desai
Jon Eubank
Matt Gingerich
Peter Hong
Kasia Karpinska-Leydier
Jordan Lapointe
Chris Milousis
James Scott
Michel Simard
Brandon Wang
Hilary Yeung
George Zarubin
OUR GRADUATES!
John Heil
Eddie Ma
Andre Masella
Danielle Nash
OUR ADVISORS!
Dr. Brian Ingalls
Dr. Trevor Charles
Dr. Barb Moffatt
Dr. Marc Aucoin
UNIVERSITY OF WATERLOO
University of Waterloo was founded in 1957 and has grown to accommodate 30,000 undergraduate and graduate students, and has become Canada’s leading university in comprehensive learning. Also, the university has consistently been voted as the most innovative, most likely to produce the leaders of tomorrow, and best overall University in Canada for over 18 years (according to Maclean’s Magazine). Waterloo’s reputation is however based on its excellent and pioneering co-op program which offers students a balance of work and school on a per term basis, making it a unique learning experience. The city of Waterloo has recognized University of Waterloo and its students, by meeting its demands in terms of funding and involvement. The University has also opened up two new campuses; the pharmacy building, and the joint McMaster medical building in Kitchener, as well as the architecture building in Cambridge, contributing to not only the city of waterloo but the whole Grand River area.
WATERLOO - KITCHENER COMMUNITY
City of Waterloo mainly revolves around the two universities: University of Waterloo and Laurier University. Waterloo is surrounded by Kitchener and thus, the two cities are known as the twin cities, also referred to as Kitchener - Waterloo. The population of the city of Waterloo is always fluctuating due to temporary residents at Waterloo’s two universities. Total population in 2009 was recorded to be 121, 700; approximately 20,000 of which were temporary post-secondary students. Due to its small size, people in the past have tried to merge the two cities together but have been unsuccessful. As of today, both cities have their own identity and their own separate city governments.
Parts List
UW's parts for 2011.
BBa_K576003 -    RNA       - Left part of self-excising ribozyme
BBa_K576004 -    RNA         - Right part of self-excising ribozyme
BBa_K576005 -    Reporter     - Left part of GFP (GFP 1) with promoter (J23101) and RBS (B0034)
BBa_K576006 -    Reporter     - Right part of GFP (GFP 2) with transcription terminator
BBa_K576007 -    Intermediate     - Left part of GFP with left part of self-excising ribozyme attached using RFC 53 construction.
BBa_K576008 -    Intermediate     - Right part of the self-excising ribozyme attached to the right part of GFP using RFC 53 construction
BBa_K576009 -    Intermediate - Lox attached on to BBa_K576005 on the right of the part. Standard assembly (RFC 10) was used for this construction.
BBa_K576010 -    Intermediate - Lox attached on to BBa_K576008 on the left of the part. BBa_K576009 or BBa_K576010 can be used depending on your convenience
BBa_K576011 -   Reporter - Final construction of the 2011 project. The self-excising ribozyme should be cut out of from the rest of the sequence and thus expressing the full GFP.
BBa_K576012 -    Reporter - Negative control of the experiment. The lox recombination site interrupts the GFP expression
BBa_K576013 -    Reporter - Positive control of the experiment. Everything in between has been cut out by the self-excising intron and the GFP is fully expressed.
Lab
Notebook
Safety
Lab Notebook 2011
The following entries pertain to the Quantification Project
Tuesday, May 31, 2011
Transformation of BBa_I20260 from iGem Kit Plate 2, Well 17F.
Wednesday, June 1, 20111
Created a frozen stock of I20260
Inoculated BBa_I0500 and BBa_E0240 from frozen stock already made from last year
Thursday, June 2, 20111
Miniprepped and nanodropped I0500, and E0240.
Inoculated I20260 from the frozen stock created the previous day
Friday, June 3, 20111
Digestion Reaction. Digested I0500 with EcoRI and SpeI. Digested E0240 (first sample) with EcoRI and PstI and digested E0240 (second sample) with EcoRI and XbaI.
Miniprepped and nanodropped I20260
Monday, June 6, 20111
Inoculated I20260, I0500 and E0240
Tuesday, June 7, 20111
Miniprepped and nandodropped I20260, and I0500 and E0240 for extra sample
Inoculated E0240
Wednesday June 8, 20111
Digestion reaction. Digested I0500 with EcoRI and SpeI. Digested E0240 (first sample) with EcoRI and PstI and digested E0240 (second sample) with EcoRI and XbaI. Digested I20260 with EcoRI and PstI.
Miniprepped and nanodropped E0240 for back up
Thursday June 9, 20111
Gel extraction of E0240 (EcoRI+XbaI), I0500 (EcoRI+SpeI), E0240 (EcoRI+PstI) and I20260 (EcoRI+PstI). Two samples of each parts were gel extracted.
Nanodropped the samples from Gel extraction. Concentration of I0500 and I20260 were too low.
Inoculated I0500 and I20260
Friday June 10, 20111
Miniprepped I0500 and I20260 and ran digestion reaction. Same enzymes were used as the ones listed above.
Tuesday, June 14, 20111
Gel extraction of I0500, I20260 and the back-up samples of E0240.
Thursday, June 16, 20111
Ligation reaction: Ligated I0500 on to E0240. Ligated I20260 on to pSB1A2 (from E0240)
Monday, June 20, 20111
Transformation of ligated parts (I0500+E0240, and I20260 on pSB1A2). Plated the transformant and incubated overnight
Tuesday, June 21, 20111
Created plates containing 1% arabinose.
Replicated plates containing I20260 colonies on to a different plate with Ampicilin antibiotic. Incubated overnight.
Wednesday, June 22, 2011
Replicated I0500+E0240 to plates containing 1% arabinose. Incubated overnight
Inoculated a single colony from I20260 plate
Thursday, June 23, 2011
Created frozen stock of I20260
Inoculated a single colony from I0500+E0240 plate
Friday, June 24, 2011
Created frozen stock of I0500+E0240
Monday, July 4, 2011
Inoculated the I20260, and I0500+E0240
Created 1M Arabinose solution
Tuesday, July 5, 2011
Made serial dilutions of arabinose solution (from 1M to 1uM).
Diluted the cultures to 0.50 Absorbance at OD600.
Set up experiment for quantification experiment. Refer to the diagram below for how the experiment was setup.
Monday, July 11, 2011
Inoculated the I20260, and I0500+E0240
Created 1M Arabinose solution
Tuesday, July 12, 2011
Another quantification experiment run. Machine failure.
Wednesday, July 13, 2011
Inoculated the I20260, and I0500+E0240
Created 1M Arabinose solution
Thursday, July 14, 2011
Last quantification experiment run.
The following entries pertain to the Ribozyme Project
Wednesday July 6, 2011
Received 3 of 4 sequences the previous week (IN1, IN2 and GFP2).
A quick spin down UWAT014-3/UWAT014-2 using centrifuge.
Resuspended DNA in 40ul of MQ water (Concentration: 2ug/40ul=1ug/20ul)
Transformed into DH5-alpha (sequences in PUC57).
grown overnight on ampicillin plates.
Resuspension of PSB1C3 in the Spring 2011 distribution kit (Plate 1 well 3A). Contains BBa_J04450.
Resuspension of PSB1C3 in 10ul of MQ water (aspirated), wait approximately 5 minutes.
1ul of resuspension was transformed into DH5-alpha. Grown overnight on plate.
Thursday July 7, 2011
N1 (amp), IN2 (amp), GFP2 (amp) and PSB1C3 (cm) broth cultures innoculated (3 each)..
Friday July 8, 2011
Frozen stock of IN1, IN2 and GFP2 in PUC57 and PSB1C3 backbone made..
Miniprep for IN1, IN2, GFP2 and PUC57 completed:.
Sequences
In1
In2
GFP 2
pSB1C3
260/280
1.85
1.80
1.88
1.86
ng/ul
229.8
236.1
198.6
166.2
GFP1 Sequence(588nt)in PUC57 received from Bio Basic Canada INC..
Tuesday July 12, 2011
Liquid cultures of GFP1 (x2), IN1, IN2, GFP2 and PSB1C3 were innoculated with the appropriate antibiotic in the broth..
Wednesday July 13, 2011
GFP1, IN1, IN2, GFP2 and PSB1C3 were minipreped to isolate plasmid DNA..
Frozen stock of GFP1 made..
Thursday July 14, 2011
GFP1, IN1, IN2, GFP2, PSB1C3 digested with EcoRI and PstI. GFP2 also digested with ndeI..
Innoculation of liquid culture (GFP1, IN1, IN2, GFP2, PSB1C3)..
Friday July 15, 2011
Gel extraction of GFP1, IN1, IN2, GFP2 and PSB1C3. However, the results were not as anticipated..
Miniprep of cultures innoculated yesterday.
Monday July 18, 2011
Cultures were miniprepped, however, GFP1 did not have a sufficient concentration to undergo digestion..
Proceeded with digestion for GFP2 (ndeI), IN1, IN2 and PSBIC3 with EcoRI and PstI..
Innoculation of GFP1 (x4).
Tuesday July 19, 2011
Miniprep and digestion of GFP1..
Gel extraction of each digestion (PSB1C3(x2), GFP1, IN1, IN2, GFP2)..
Wednesday July 20, 2011
Lox resuspended and digested..
Ligation of GFP2, IN1, IN2 and Lox into PSB1C3..
Thursday July 21, 2011
Transformation of GFP2, IN1, IN2 and Lox. Each was plated on cm containing media and grown overnight..
Friday July 22, 2011
All negative plates did not produce colonies..
Growth was good on all positive plates except for IN1, which only produced two main colonies..
Monday July 25, 2011
Gel extraction IN1, IN2 and GFP1, however, GFP1 failed..
Ligation of IN1 and IN2 into PSB1C3..
Tuesday July 26, 2011.
Innoculation of GFP2 into cm containing LB broth tube and lox into amp containing LB broth tube..
Gel extraction did not work (likely a problem with digestion or transformation).
Innoculation x5 of GFP1..
Wednesday July 27, 2011
Frozen stock of GFP2 and lox made.
Miniprep of GFP1 x4 replicates.
Gel extraction of IN1 and IN2 resulted in improved concentrations.
Transformation.
July 30, 2011
Miniprepped GFP 1 and GFP 2.
August 2, 2011
Miniprepped Intron 1 and intron 2..
August 3, 2011
Digestion of GFP 1 and Intron 2 with SacI and EarI.
Digestion of GFP2 and Intron 1 with SacI and SapI.
August 4, 2011
Nothing.
Figuring out unexpected SacI cute site in the middle of pSB1C3 vector.
August 5, 2011
Nothing.
August 6, 2011
Digestion of GFP 1 with Bgl II and Ear I.
Digestion of Intron 1 with Bgl II and Ear I.
Digestion of GFP 1 with Ear I and Pst I.
Digestion of GFP 2 with EarI and Pst I..
Gel extraction of all the samples listed above.
August 7, 2011
Ligation of GFP 1 with Intrton 1 and GFP 1 with GFP 2.
Transformation of the two ligation mixtures listed above.
August 8, 2011
Minipreping Int2, lox and GFP2.
Digestion.
Gel extraction.
August 9, 2011
Miniprepping BBa_K576007.
Digestion.
gel extraction.
August 10, 2011
Ligation and transformation of BBa_K576007 and J61046 to create K576009.
August 11, 2011
Ligation and transformation.
August 12, 2011
Streak plating GFP1-GFP2 (BBa_K576013).
Backbone for RFC arrived and was streaked on cm plate.
August 15th, 2011
Inoculation of a single colony of K371053 into an LB broth
August 16th, 2011
Made frozen stock of K371053
Inoculated K371053 (X3), K576004, and K576006.
August 17th, 2011
Miniprepped K371053(vector), K576004 (insert),  and K5756006 (insert).
Standard assembly of the two parts and a vector listed above. Ran gel extraction experiment after.
August 18th, 2011
Ligation and transformation reaction for transferring K576004 and K576006 from pSB1C3 to K371053.
August 19th, 2011
Inoculation of single colonies from the transformed samples from the previous days.
Inoculated K576009
August 22nd, 2011
Created frozen stock and inoculated K576004 and K576006 on K371053
Miniprepped, digested (SpeI and PstI) and gel extracted K576009.
August 23rd, 2011
Miniprepped, digested and gel extracted K576004 (SpeI and PstI) and K576006 (XbaI and PstI) on K371053.
August 24th, 2011
Ligation and transformation of samples prepared on the 23rd.
August 25th, 2011
Single colony for K576008 was picked and inoculated
August 26th, 2011
Created frozen stock of K576008
August 28th, 2011
Inoculated K576008 and K576009 in LB broth containing chloramphenicol
August 29th,2011
miniprepped and digested K576008 and K576009.
August 30th, 2011
Gel extraction of K576008 (insert) and K576009 (vector).
August 31st, 2011
Ligation and transformation of final construction (BBa_K576011)
September 1st-2nd, 2011
No lab work was done.
September 4th, 2011
inoculated K576011 and K576008 in LB broth containing Chloromphenicol antibiotic
September 5th, 2011
Miniprepped, digested (EcoRI and PstI) and gel electrophoresed part K576008 to move the part into pSB1C3 for submission. Questionable band placement; Did not go further with the experiment.
Miniprepped all the intermediate steps, controls and final construction for submission to partsregistry.
September 6th, 2011
Miniprepped, digested and gel extracted part K576008. Confirmed band placement via aPe program. Failed at the final step of the gel extraction process; Wash Solution without ethanol (miniprep solution provided from Biobasic) was added
September 7th, 2011
Miniprepped, digested and gel extracted part K576008.
Septermber 8th, 2011
Ligation and transformation reaction for K576008. Moved K576008 part from the BBa_K371053 to pSB1C3 for submission purposes.
Septermber 9th – 15th, 2011
no lab work was done
September 16th, 2011
Final diagnostic digestion reaction. Digested every intermediate, control and final constructs. BBa_K576003, K576004, K576005, and K576006 were the only parts able to be confirmed. All the other intermediates and constructs have questionable band location. Lab work stopped due to time constraint.
SAFETY
Laboratory Safety
The Ribozyme Project is not expected to raise any research, public or environmental safety concerns other than those normally associated with Biosafety Level 2 organisms, such as Escherichia coli (DH5-alpha), which is classified as very low to moderate. The use of this project is primarily reserved for research and laboratory use, therefore, should not purposefully be exposed to the public or environment except after further testing in its specific applications (such as with particular fusion proteins). Furthermore, the basis of our project is to establish a self-excising sequence (ribozymes), which should limit the expression of any intervening sequences to the RNA level. If the intervening sequence were something of environmental or public relevance (such as antibiotic resistance), the experimental design indicates that the sequence will be removed and, thus, not expressed. This is a relevant contribution of the design in limiting expression to the RNA level, which eases environmental hazard concern upon the accidental release of a GMO containing this biobrick. Therefore, the new biobrick parts submitted should not raise any safety issues.
The necessary facility, equipment and handling procedures associated with Level 2 Biosafety concerns were met:
1.Pipetting aids
2.Biosafety cabinets where applicable
3.Laboratory separated from other activities
4.Biohazard sign
5.Proper safety and disposal equipment, including autoclave
6.Personal protective equipment, worn only in the laboratory
7.Screw-capped tubes and bottles
8.Plastic disposable pasteur pipettes, when necessary
All precautions with respect to recombinant DNA were observed:
1.All waste was autoclaved before being thrown away.
2.Researchers practiced aseptic technique and personal hygiene and safety precautions
3.Procedures likely to generate aerosols are performed in a biosafety cabinet
4.Bench surfaces were disinfected with ethanol.
4.Potentially contaminated waste is separated from general waste
Safety Questions
1. Would the materials used in your project and/or your final product pose: The materials used in the lab are non toxic to health of individuals as well as to the environment. One of the major reagents that is used is GelRed which is used as a substitute for Ethidium Bromide. Gel Red is unable to penetrate into cells and so is a non-mutagenic agent. As well it has the same spectral characteristics as Ethidium Bromide and so has the same effectiveness of use. The project itself is safe even if released into the environment by design or accident since the part being expressed is the Green Fluorescent Protein (GFP). Unless the sequences are mutated, the project poses no risk.
Please explain your responses (whether yes or no) to these questions.
Specifically, are any parts or devices in your project associated with (or known to cause):
- pathogenicity, infectivity, or toxicity? No
- threats to environmental quality? No
- security concerns? No
The parts that are associated with the project this year are at the same level of risk as the any of the regular parts that already exist. All parts are constructed in an antibiotic containing backbone so that accidental release of will pose minimal risk to contaminating other bacterial populations.
2.Under what biosafety provisions will / do you operate?
a.Does your institution have its own biosafety rules and if so what are they? The University of Waterloo had a Bio-Safety plan in place to ensure the proper use to bio-hazardous materials in teaching and research at the university. A more detailed overview of their plans is outlined at the Bio-Safety Website
b. Does your institution have an Institutional Biosafety Committee or equivalent group? If yes, have you discussed your project with them? The laboratories operating at the University of Waterloo have obtained permits from the Bio-Safety Committee in order to perform intended research. Since the Waterloo iGEM team performs all laboratory work in a parent lab under the guidance of the Masters and PhD students of that lab, the projects carried out in the lab are covered by the permits obtained by the parent lab.
c. Will / did you receive any biosafety and/or lab training before beginning your project? If so, describe this training. All lab volunteers are required to take an online training to familiarize themselves with the Biosafety practices of the University of Waterloo. The training is followed up by a quiz ensuring proper understanding of the material. Upon completion of the training and quiz a hands- on lab training is provided under supervision of the parent lab’s PhD student. The hands-on training involves instruction of use of the appropriate equipment that is used in the lab, as well as how to maintain and discard materials in a safe manner.
d. Does your country have national biosafety regulations or guidelines? If so, provide a link to them online if possible. Canada operates under the guidelines set up by the Public Health Agency of Canada. The Agency is the national authority on matters concerning biosafety and biosecurity. Risks to the public are reduced by standardizing controls over activities that involve human pathogenic agents, domestic or imported. While these guidelines are in place the current iGEM project does not involve work with any agents or materials that may pose a risk to humans. The link to the Public Health Agency of Canada is provided below: Public Health Agency of Canada
Upload file Disclaimers Privacy policy My preferences Special pages
Retrieved from " http://2011.igem.org/Team:Waterloo "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
