<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCLA_Notebook_Biobrick"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCLA/Notebook/Biobrick">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCLA/Notebook/Biobrick&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCLA/Notebook/Biobrick&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCLA/Notebook/Biobrick&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCLA/Notebook/Biobrick" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCLA/Notebook/Biobrick</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><UL id="nav"><LI style="margin-left: 20px;"><A href="https://2013.igem.org/Team:UCLA">HOME</A></LI><LI><A href="/Team:UCLA/Project">PROJECT</A></LI><LI><A href="/Team:UCLA/Team">TEAM</A></LI><LI><A href="/Team:UCLA/Parts">PARTS</A></LI><LI><A href="/Team:UCLA/Modeling">MODELING</A></LI><LI><A href="/Team:UCLA/HumanPractices">HUMAN PRACTICES</A></LI><LI><DIV id="spec"><A href="/Team:UCLA/Notebook/Biobrick"><FONT color="black">NOTEBOOK</FONT></A></DIV></LI><LI><A href="/Team:UCLA/Safety">SAFETY</A></LI><LI><A href="/Team:UCLA/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://igem.org/2013_Judging_Form?id=1176">JUDGING</A></LI></UL></DIV><UL id="subnav"><LI style="margin-left: 12px;" id="current"><A href="https://2013.igem.org/Team:UCLA/Notebook/Biobrick">Mtd Biobrick Construction</A></LI><LI><A href="/Team:UCLA/Notebook/Library">mRNA Display Library Generation</A></LI><LI><A href="/Team:UCLA/Notebook/mRNAdisplay">mRNA Display</A></LI></UL><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Making_the_Mtd_Biobrick"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Making the Mtd Biobrick</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_to_generate_fragments_of_mtd"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">PCR to generate fragments of mtd</SPAN></A></LI><LI class="toclevel-2"><A href="#Splicing_Overlap-Extension_.28SOE.29_PCR_to_connect_fragments_3_and_4"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Splicing Overlap-Extension (SOE) PCR to connect fragments 3 and 4</SPAN></A></LI><LI class="toclevel-2"><A href="#Gibson_Assembly_of_Fragments_1.2C2.2C_and_3-4"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Gibson Assembly of Fragments 1,2, and 3-4</SPAN></A></LI><LI class="toclevel-2"><A href="#PCR_amplification_of_mtd"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">PCR amplification of mtd</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Making_the_Mtd_Biobrick">Making the Mtd Biobrick</SPAN></H1><P>To create this biobrick, we utilized a combination of splicing overlap extension PCR and Gibson Assembly to extract a sequence from the BPP-1 phage's genomic DNA that would be free of the standard biobrick restriction enzyme sites. The protocols for those methods are listed below: </P><H3><SPAN class="mw-headline" id="PCR_to_generate_fragments_of_mtd">PCR to generate fragments of <I>mtd</I></SPAN></H3><P>In this step, four separate fragments of the <I>mtd</I> gene are created via PCR. This step is necessary to eliminate illegal internal biobrick sites (EcoRI, PstI, NotI) and to add necessary biobrick sites to the 5' and 3' ends.</P><P><TABLE id="tfhover" class="tftable" border="1"><TBODY><TR><TH>Fragment</TH><TH>Forward Primer</TH><TH>Reverse Primer</TH></TR><TR><TD>1</TD><TD>CCTTGAATTCGCGGCCGCATCTAGAATGAGTACCGCAGTCCAGTTCCG</TD><TD>GCCTGCGCTGCCGCGTTGCTTCC</TD></TR><TR><TD>2</TD><TD>GGAAGCAACGCGGCAGCGCAGGC</TD><TD>CCAGCGCCTGGAACTCGTTGTAGTTGGGCAGG</TD></TR><TR><TD>3</TD><TD>CCTGCCCAACTACAACGAGTTCCAGGCGCTGG</TD><TD>CCGATCCGCGGCCTCCAGTGTTGG
</TD></TR><TR><TD>4</TD><TD>CCAACACTGGAGGCCGCGGATCGG
</TD><TD>AAGGCTGCAGCGGCCGCTACTAGTCTCAAGAATCAGG</TD></TR></TBODY></TABLE></P><P>The 20 µL PCR mix for each fragment is as follows: </P><P><TABLE id="tfhover" class="tftable" border="1"><TBODY><TR><TH>Reagent</TH><TH>Volume</TH></TR><TR><TD><I>mtd</I> genomic template</TD><TD>1.0 µL (4.5 ng total)</TD></TR><TR><TD>10 µM forward primer</TD><TD>1.0 µL</TD></TR><TR><TD>10 µM reverse primer</TD><TD>1.0 µL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.4 µL</TD></TR><TR><TD>Buffer HF</TD><TD>4.0 µL</TD></TR><TR><TD>Phusion Polymerase</TD><TD>0.2 µL</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>12.4 µL</TD></TR></TBODY></TABLE></P><P>The thermocycler program for each reaction is as follows: </P><P><TABLE id="tfhover" class="tftable" border="1"><TBODY><TR><TH># Cycles</TH><TH>Temperature (°C)</TH><TH>Time</TH></TR><TR><TD>1</TD><TD>98</TD><TD>0:30</TD></TR><TR><TD>30</TD><TD>98Variable (70 for fragment 1, 69 for fragment 2, 69 for fragment 3, 72 for fragment 3)72</TD><TD>0:100:200:15</TD></TR><TR><TD>1</TD><TD>72</TD><TD>5:00</TD></TR><TR><TD>1</TD><TD>4</TD><TD>--</TD></TR></TBODY></TABLE></P><P>The PCR product is then resolved on a 1% agarose gel and excised and column-extracted.</P><H3><SPAN class="mw-headline" id="Splicing_Overlap-Extension_.28SOE.29_PCR_to_connect_fragments_3_and_4">Splicing Overlap-Extension (SOE) PCR to connect fragments 3 and 4</SPAN></H3><P>The 20 µL reaction mix is as follows: </P><P><TABLE id="tfhover" class="tftable" border="1"><TBODY><TR><TH>Reagent</TH><TH>Volume or Mass</TH></TR><TR><TD>Fragment 3</TD><TD>20 ng total</TD></TR><TR><TD>Fragment 4</TD><TD>20 ng total</TD></TR><TR><TD>10 µM forward primer</TD><TD>1.0 µL</TD></TR><TR><TD>10 µM reverse primer</TD><TD>1.0 µL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.4 µL</TD></TR><TR><TD>Buffer HF</TD><TD>4.0 µL</TD></TR><TR><TD>Phusion Polymerase</TD><TD>0.2 µL</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>fill to 20 µL</TD></TR></TBODY></TABLE></P><P>The thermocycler program for this step is as follows: </P><P><TABLE id="tfhover" class="tftable" border="1"><TBODY><TR><TH># Cycles</TH><TH>Temperature (°C)</TH><TH>Time</TH></TR><TR><TD>1</TD><TD>98</TD><TD>0:30</TD></TR><TR><TD>30</TD><TD>987272</TD><TD>0:100:200:15</TD></TR><TR><TD>1</TD><TD>72</TD><TD>5:00</TD></TR><TR><TD>1</TD><TD>4</TD><TD>--</TD></TR></TBODY></TABLE></P><H3><SPAN class="mw-headline" id="Gibson_Assembly_of_Fragments_1.2C2.2C_and_3-4">Gibson Assembly of Fragments 1,2, and 3-4 </SPAN></H3><P>The mix for the Gibson Assembly is as follows: </P><P><TABLE id="tfhover" class="tftable" border="1"><TBODY><TR><TH>Reagent</TH><TH>Concentration or Volume</TH></TR><TR><TD>Fragment 1</TD><TD>0.2 pM</TD></TR><TR><TD>Fragment 2</TD><TD>0.2 pM</TD></TR><TR><TD>Fragment 3-4</TD><TD>0.2 pM</TD></TR><TR><TD>Assembly Master Mix (from New England Biolabs)</TD><TD>fill to 5.0 µL</TD></TR></TBODY></TABLE></P><P>The mix is then incubated at 50 °C for 1 hour. </P><H3><SPAN class="mw-headline" id="PCR_amplification_of_mtd">PCR amplification of <I>mtd</I></SPAN></H3><P>The mix for the amplification is as follows:</P><P><TABLE id="tfhover" class="tftable" border="1"><TBODY><TR><TH>Reagent</TH><TH>Volume or Mass</TH></TR><TR><TD><I>mtd</I></TD><TD>20 ng</TD></TR><TR><TD>10 µM forward primer (CCTTGAATTCGCGGCCGCATCTAGAATGAGTACCGCAGTCCAGTTCCG
)</TD><TD>0.6 µL</TD></TR><TR><TD>10 µM reverse primer (AAGGCTGCAGCGGCCGCTACTAGTCTCAAGAATCAGG
)</TD><TD>0.6 µL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.4 µL</TD></TR><TR><TD>2x xtreme buffer</TD><TD>10 µL</TD></TR><TR><TD>KOD xtreme hot start polymerase</TD><TD>0.4 µL</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>fill to 20 µL</TD></TR></TBODY></TABLE></P><P>The thermocycler program for the amplification is as follows:</P><P><TABLE id="tfhover" class="tftable" border="1"><TBODY><TR><TH># Cycles</TH><TH>Temperature (°C)</TH><TH>Time</TH></TR><TR><TD>1</TD><TD>94</TD><TD>2:00</TD></TR><TR><TD>35</TD><TD>987068</TD><TD>0:100:301:20</TD></TR><TR><TD>1</TD><TD>10</TD><TD>--</TD></TR></TBODY></TABLE></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UCLA/Notebook/Biobrick">http://2013.igem.org/Team:UCLA/Notebook/Biobrick</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCLA/Notebook/Biobrick" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCLA/Notebook/Biobrick" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCLA/Notebook/Biobrick&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCLA/Notebook/Biobrick&amp;oldid=227248" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>