<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCLA_Project_mrnaDisplay"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCLA/Project/mrnaDisplay">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCLA/Project/mrnaDisplay&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCLA/Project/mrnaDisplay&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCLA/Project/mrnaDisplay&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCLA/Project/mrnaDisplay" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCLA/Project/mrnaDisplay</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><UL id="nav"><LI style="margin-left: 20px;"><A href="https://2013.igem.org/Team:UCLA">HOME</A></LI><LI><DIV id="spec"><A href="/Team:UCLA/Project"><FONT color="black">PROJECT</FONT></A></DIV></LI><LI><A href="/Team:UCLA/Team">TEAM</A></LI><LI><A href="/Team:UCLA/Parts">PARTS</A></LI><LI><A href="/Team:UCLA/Modeling">MODELING</A></LI><LI><A href="/Team:UCLA/HumanPractices">HUMAN PRACTICES</A></LI><LI><A href="/Team:UCLA/Notebook/Biobrick">NOTEBOOK</A></LI><LI><A href="/Team:UCLA/Safety">SAFETY</A></LI><LI><A href="/Team:UCLA/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://igem.org/2013_Judging_Form?id=1176">JUDGING</A></LI></UL><CENTER><UL id="subnav"><LI style="margin-left: 12px;"><A href="https://2013.igem.org/Team:UCLA/Project">Overview</A></LI><LI><A href="https://2013.igem.org/Team:UCLA/Project/NaturalSystem">Natural System</A></LI><LI id="current"><A href="/Team:UCLA/Project/mrnaDisplay">mRNA Display</A></LI><LI><A href="/Team:UCLA/Project/ProjectDesign">Project Design</A></LI></UL></CENTER><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Historical_background"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Historical background</SPAN></A></LI><LI class="toclevel-1"><A href="#What_is_it.3F"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">What is it?</SPAN></A></LI><LI class="toclevel-1"><A href="#How_does_it_work.3F"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">How does it work?</SPAN></A></LI><LI class="toclevel-1"><A href="#What_does_the_Mtd_have_to_do_with_mRNA_display.3F"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">What does the Mtd have to do with mRNA display?</SPAN></A></LI><LI class="toclevel-1"><A href="#Other_approaches_for_directed_protein_targeting.3F"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Other approaches for directed protein targeting?</SPAN></A></LI><LI class="toclevel-1"><A href="#What_are_the_advantages_of_this_route_.2F_why_did_we_choose_this_route_for_our_project.3F"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">What are the advantages of this route / why did we choose this route for our project?</SPAN></A></LI><LI class="toclevel-1"><A href="#Issues_with_using_this_route.3F"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Issues with using this route?</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Historical_background">Historical background</SPAN></H4><P>Richard Roberts and Jack Szostak from Harvard developed this technique in 1997. They were able to generate and identify a myc epitope starting from a random mRNA library using this technique (Roberts).
</P><H4><SPAN class="mw-headline" id="What_is_it.3F">What is it?</SPAN></H4><P>mRNA display is a technique used to select for proteins that bind to specific targets. It allows for the identification of these selected proteins because they are covalently attached to the DNA that codes for them (Roberts).
</P><DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Overview of mRNA display (Liu)</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="How_does_it_work.3F">How does it work?</SPAN></H4><OL><LI>A library of DNA sequence variants is generated using error prone PCR or PCR with variable primers.</LI><LI>The DNA library is transcribed <I>in vitro</I> to an RNA library.</LI><LI>Using a DNA splint, RNA strands are ligated to single stranded DNA oligonucleotides with puromycin at the 3' end</LI><LI>The RNA-puromycin fusions are then translated <I>in vitro</I></LI><LI>The Puromycin mimics a tRNA and is incorporated into the A site of the ribosome. However 	the puromycin has a non hydrolizable amide bond, therefore terminating translation and releasing 	the polypeptide, which is still bound to the RNA template through the puromycin. This results in a library of proteins covalently bonded to the RNA that coded for them.</LI><LI>In order to create a more stable structure, the RNA is converted to a DNA-RNA duplex through 	<I>in vitro</I> reverse transcription.</LI><LI>The library is screened for the protein that best binds to the target molecule.</LI><LI>Identification of the protein will be simple, as the corresponding DNA strand is still attached to it. PCR of the DNA followed by sequencing yields the exact identity of the protein at the DNA level. </LI></OL></DIV><H4><SPAN class="mw-headline" id="What_does_the_Mtd_have_to_do_with_mRNA_display.3F">What does the Mtd have to do with mRNA display?</SPAN></H4><P>The Mtd protein is the scaffold for our mRNA display experiment. Most proteins are not capable of undergoing the large number of mutations necessary to create a large library while still maintaining their structure and function.  The Mtd is unique in that it has a structure that allows it to be mutagenized in certain sites and still remain stable.  The Mtd has evolved in the natural system of <I>Bordetella</I> bacteriophage to have precisely this property.  Therefore, it acts as an effective scaffold because the majority of the protein is able to remain constant and stable while a small localized region of each protein gets mutagenized (Yuan).
</P><H4><SPAN class="mw-headline" id="Other_approaches_for_directed_protein_targeting.3F">Other approaches for directed protein targeting?</SPAN></H4><UL><LI>Phage display:
		An <I>in vivo</I> technique similar in principle to mRNA display in which bacteriophage are allowed to  replicate to create a diverse library with different binding affinities.  The DNA from the bacteriophage that corresponds to the protein that binds to the target can be isolated and then sequenced. </LI><LI>Ribosome display:
	Another <I>in vitro</I> technique that fuses the RNA transcript to the protein that it codes for.  Instead of a puromycin linker between the transcript and the protein, it is the ribosome that links the protein and the template that codes for it (Smith). </LI></UL><H4><SPAN class="mw-headline" id="What_are_the_advantages_of_this_route_.2F_why_did_we_choose_this_route_for_our_project.3F">What are the advantages of this route / why did we choose this route for our project?</SPAN></H4><P>One of the major advantages of the mRNA display approach is that it takes place completely <I>in vitro</I>.  This eliminates the limitations and complications that come with working <I>in vivo</I> and gives us more control over the entire process.  Also, using an <I>in vitro</I> method allows us to create a more diverse library.  Generating the library <I>in vitro</I> can potentially yield several orders of magnitude more protein variants than <I>in vivo</I> systems such as phage display.
</P><P>With respect to ribosome display, which is also an <I>in vitro</I> method, mRNA display has an advantage when it comes to screening for protein binding.  The disadvantage of ribosome display is that the linker between the protein and the template DNA is the actual ribosome.  This is an issue because the ribosome is a very large protein and it has the potential to interact with the target protein, thereby throwing off the selection process. In contrast, the puromycin link in mRNA display is much smaller and is not likely to interfere in any way with the target protein (Lipovsek).
</P><H4><SPAN class="mw-headline" id="Issues_with_using_this_route.3F">Issues with using this route?</SPAN></H4><P>What we are essentially doing with this project is taking the Mtd protein which comes from an <I>in vivo</I> system (the BPP-1 phage) and using it as the scaffold in our <I>in vitro</I> protein selection system.  We know that in that native system of the BPP-1 phage, a large library of phages are generated with unique variants of the Mtd protein on their tail fibers.  Out of this large library, some phage are able to effectively bind to a certain target using the Mtd protein.
</P><P>A major concern is that the Mtd protein may not behave the same way <I>in vitro</I> as it does in the natural BPP-1 system.  Although we are theoretically able to generate more variants of Mtd than in the natural phage system, it is possible that isolating it from the phage could seriously affect its binding capabilities.  For example, the native phage has several tail fibers, each with a copy of the Mtd that could potentially bind to a target.  It may be that one isolated Mtd has a very low binding affinity and that it is necessary to have several in order to create a strong enough binding interaction with the target (Dai).
</P><DIV id="reference"><B>REFERENCES</B><P>Dai, W., et al. <A href="http://www.ncbi.nlm.nih.gov/pubmed/20160083">&quot;Three-dimensional structure of tropism-
switching <I>Bordetella</I> bacteriophage.&quot;</A><I>PNAS</I>. 107.9 (2010):4347-52. Print.</P><P>Lipovsek. D., A. Pluckthun. <A href="http://www.bioc.uzh.ch/plueckthun/pdf/APpub0250.pdf">&quot;In vitro protein evolution by ribosome display and mRNA display.&quot;</A><I>Journal of Immunological Methods</I>. 290 (2004):51â67. Print.
</P><P>Liu, R., et al. <A href="http://www.ncbi.nlm.nih.gov/pubmed/10889994">&quot;Optimized synthesis of RNA-protein fusions for in vitro protein selection&quot;.</A><I>Meth Enzymol</I>. 318 (2000):268â93. Print.</P><P>Roberts, R.W., J.W. Szostak. <A href="http://www.ncbi.nlm.nih.gov/pubmed/9356443">&quot;RNA-peptide fusions for the in vitro selection of peptides and proteins.&quot;</A><I>Proc. Natl. Acad. Sci. USA</I>. 94.23 (1997):12297-302. Print</P><P>Smith, G., V. Petrenko. <A href="http://www.biosci.missouri.edu/SmithGP/PhageDisplayWebsite/PetrenkoSmithChemReviews.PDF">&quot;Phage Display.&quot;</A><I>Chem. Rev.</I> 97 (1997):391-410. Print. </P><P>Yuan, T.Z., et al. <A href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0055617#references">&quot;Protein Engineering with Biosynthesized Libraries from Bordetella bronchiseptica Bacteriophage.&quot;</A><I>PLoS ONE</I>. 8.2 (2013). Web.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UCLA/Project/mrnaDisplay">http://2013.igem.org/Team:UCLA/Project/mrnaDisplay</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCLA/Project/mrnaDisplay" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCLA/Project/mrnaDisplay" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCLA/Project/mrnaDisplay&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCLA/Project/mrnaDisplay&amp;oldid=240657" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>