<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCL_Project_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCL/Project/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCL/Project/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Project/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL/Project/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCL/Project/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCL/Project/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="all_content"><DIV id="container"><DIV class="full_page"><P class="body_text"><B>In the wet-lab we followed standard protocols with some of our own revisions. The details of our procedure are shown below. For an overview of what these procedures were used for, pleases see <A href="https://2013.igem.org/Team:UCL/Project/Experiments" target="_blank">experiments</A>.</B></P></DIV><P class="major_title">Bacterial Lab Protocols</P><DIV class="row_small"><DIV class="protocol"><P class="minor_title">1.4% Agar</P><P class="body_text">
In 500mL Duran bottle insert:
</P><TABLE><TBODY><TR><TH><B>Reagent</B></TH><TH>Required Quantity</TH></TR><TR><TD>Agar</TD><TD>7g</TD></TR><TR><TD>RO H2O</TD><TD>500mL</TD></TR></TBODY></TABLE></DIV><DIV class="description"><P class="minor_title">5X M9 Salts</P><P class="body_text">
In 500mL Duran bottle insert:
</P><TABLE><TBODY><TR><TH><B>Reagent</B></TH><TH>Required Quantity</TH></TR><TR><TD>Na2HPO4</TD><TD>32g</TD></TR><TR><TD>KH2PO4</TD><TD>7.5g</TD></TR><TR><TD>NaCl</TD><TD>1.25g</TD></TR><TR><TD>NH4Cl</TD><TD>2.5g</TD></TR><TR><TD>RO HCL</TD><TD>500ml</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="row_small"><DIV class="protocol"><P class="minor_title"> 0.1M CaCl2/15% glycerol</P><P class="body_text">
In a 50mL Falcon insert:
</P><TABLE><TBODY><TR><TH><B>Reagent</B></TH><TH>Required Quantity (mL)</TH></TR><TR><TD>1M CaCl2</TD><TD>5</TD></TR><TR><TD>100% Glycerol</TD><TD>7.5</TD></TR><TR><TD>RO H2O</TD><TD>37.5</TD></TR></TBODY></TABLE></DIV><DIV class="description"><P class="minor_title">Minimal Agar</P><P class="body_text">
Mix:
</P><TABLE><TBODY><TR><TH><B>Reagent</B></TH><TH>Required Quantity</TH></TR><TR><TD>5x M9 Salts</TD><TD>10mL</TD></TR><TR><TD>2 mg/ml Thiamine</TD><TD>50µl</TD></TR><TR><TD>20% D Glucose</TD><TD>1 mL</TD></TR><TR><TD>1M CaCl2</TD><TD>5µl</TD></TR><TR><TD>1M MgSO4</TD><TD>100µl</TD></TR><TR><TD>1.4% Agar</TD><TD>39 mL</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="row_small" style="height:400px;"><DIV class="protocol"><P class="minor_title">LB Media</P><P class="body_text">
In 500mL Duran bottle insert:
<TABLE><TBODY><TR><TH><B>Reagent</B></TH><TH>Required Quantity</TH></TR><TR><TD>LB Broth</TD><TD>10g</TD></TR><TR><TD>RO H2O</TD><TD>500 mL</TD></TR></TBODY></TABLE></P></DIV><DIV class="description"><P class="minor_title">Generating Competence Cells</P><P class="body_text">
Locate a glycerol stock of untransformed E. coli, streak cells onto minimal agar plates and incubate at 37C for 16 hours.
</P><P class="body_text">
Once complete, pick a colony from the plate and place into a 50mL Falcon, containing 5mL LB &amp; 100 uL 1M MgSO4 for 16 hours.
</P><P class="body_text">
After this, inoculate a 100mL shake flask with 1mL of culture from the Falcon tube. Take absorbance readings every 30 minutes until the absorbance reading is above 0.3. Once this is achieved, transfer the contents into two 50mL Falcon tubes and place on ice for 10 minutes. Perform centrifugation (~6,000 RPM) for 5 minutes and then resuspend in 10mL Calcium Chloride. Aliquot into eppendorf tubes (~500 uL per tube) and then store at very low temperatures (&lt;-50C).
</P></DIV></DIV><DIV class="row_small"><DIV class="description"><P class="minor_title">Streaking Plates</P><P class="body_text">
Obtain agar plates (as many as required), streaking loops and cells to be streaked. Dip a streaking loop in the cell culture, and gently (so there is no damage to the agar) streak the loop onto the plate as described in the diagram below. Once finished, incubate at 37C overnight. 
</P></DIV></DIV><DIV class="row_small"><DIV class="protocol"><P class="minor_title">Plate generation (AMP, CMP &amp; NoDrug)</P><P class="body_text">
Heat up 50 mL of agar until molten (usually ~300 seconds using a 800W microwave). Douse in cold water to lower temperature. When still warm, but able to handle, it is possible to add an antibiotic drug for selection purposes (~50 uL). Once this complete, pour ~10mL into a petri dish and ensure that the whole surface is covered. Leave lid off for 30 minutes. Place lid on dish and then use, or store at ~5C.
</P></DIV><DIV class="description"><P class="minor_title">Glycerol Stock Generation</P><TABLE><TBODY><TR><TH><B>Reagent</B></TH><TH>Required Quantity</TH></TR><TR><TD>LB Media</TD><TD>3mL</TD></TR><TR><TD>Culture</TD><TD>3 µl</TD></TR><TR><TD>Relevant Anti-bioitc</TD><TD>3 µl</TD></TR></TBODY></TABLE><P class="body_text">
12-16 hour 37C incubation. Insert into 1.5mL microcentrifuge tubes. Note absorbance. Add below, then store (-20C).
<TABLE><TBODY><TR><TH><B>Reagent</B></TH><TH>Required Quantity (µl)</TH></TR><TR><TD>Culture</TD><TD>500</TD></TR><TR><TD>Glycerol Stock</TD><TD>166</TD></TR></TBODY></TABLE></P></DIV></DIV><DIV class="row_small"><DIV class="protocol"><P class="minor_title">50X to 1X Dilution</P><P class="body_text">
To a 1L Duran bottle, insert:
<TABLE><TBODY><TR><TH><B>Reagent</B></TH><TH>Required Quantity (mL)</TH></TR><TR><TD>50X TAE Buffer</TD><TD>20</TD></TR><TR><TD>RO H2O</TD><TD>980</TD></TR></TBODY></TABLE></P></DIV><DIV class="description"><P class="minor_title">Transformation</P><P class="body_text"><B>1)</B> Remove one of your aliquots of competent cells from the -80C freezer and place onto ice.
<B>2)</B> Add ~2uL of DNA to the competent cells. Leave in ice for ~45 minutes.
<B>3)</B> Place tubes into 37C water bath for 10 minutes (heat shock).
<B>4)</B> Place tubes into ice for 2 minutes.
<B>5)</B> Add 1.3mL of Lb to the tubes and transfer all of the contents to new tubes. Incubate for 1 hour at 37C.
<B>6)</B> Centrifuge at high RPM for 2 minutes. Discard the supernatant.
<B>7)</B> Resuspend cell pellet into 100uL of LB.
<B>8)</B> Spread contents onto petri dishes containing LB agar (may also contain antibiotic resistance for better selectivity.
<B>9)</B> Incubate for 16 hours at 37C and then pick colonies if growth has occurred.
</P></DIV></DIV><DIV class="row_small" style="height:400px;"><DIV class="protocol"><P class="minor_title">Maxi/mini Preparation</P><P class="body_text">
See <A href="http://www.google.co.uk/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=1&amp;ved=0CDsQFjAA&amp;url=http%3A%2F%2Fwww.qiagen.com%2Fresources%2FDownload.aspx%3Fid%3D%257B46205595-0440-459E-9D93-50EB02E5707E%257D%26lang%3Den%26ver%3D2&amp;ei=-YpFUpvcF-yd0wWIw4HACg&amp;usg=AFQjCNFGR5hl0QYv64lnVZDZWaw26BKA0A&amp;sig2=JjaWz8EP2dxWJAMCpnLxCA&amp;bvm=bv.53217764,d.d2k
" target="_blank">protocol</A> for mini/maxi prep from Qiagen:
</P></DIV><DIV class="description"><P class="minor_title">Analytical Digest</P><P class="body_text">
Add the following items to a 1.5mL microcentrifuge tube and briefly (&lt;10s) centrifuge to ensure all contents are mixed:
<TABLE><TBODY><TR><TH><B>Reagent</B></TH><TH>Required Quantity (µL)</TH></TR><TR><TD>DNA</TD><TD>1.5g</TD></TR><TR><TD>Enzyme 1</TD><TD>150mL</TD></TR><TR><TD>Enzyme 2</TD><TD>3mL</TD></TR></TBODY></TABLE></P><P class="body_text">
Heat up the solution in a conical flask, until agarose has dissolved completely and the solution becomes clear. To the clear solution, add 2 ul of Ethidium Bromide and shake. Pour solution onto the the gel plate with the comb. Wait until gel has solidified after 20 minutes, the gel may now be ready for digest. 
</P></DIV></DIV><DIV class="row_small" style="height:400px;"><DIV class="protocol"><P class="minor_title">Gel Electrophoreisis</P><P class="body_text">
Add loading buffer to all samples (including laddder), then remove the comb from the solidified agarose gel, place the solidified agarose gel onto gel box and cover the gel box with 1x TAE buffer. Carefully load samples into the gel wells. Then cover the gel box with the lid. Run the gel on 120 Volts, 60 minutes condition. 
</P></DIV><DIV class="description"><P class="minor_title">Nanodrop</P><P class="body_text">
Before starting the software module, clean the sample surfaces with DI water to remove any dried sample that might be present. Open the Nanodrop program and the appropriate module (e.g., DNA). Wipe off the top and bottom sensors of the instrument with a Kimwipe. Pipette 1 μL of RO water onto the sensor. Bring down the lever arm. Follow the onscreen prompts to calibrate. Wipe the sensors and pipette on 2 μL of the corresponding blank (Buffer EB or whatever solution your prep is in). Bring down the lever arm. Follow the onscreen prompts to blank. Wipe the sensors and pipette on 2 μL of your sample. Bring down the lever arm. Click Measure and record the concentration measured. For DNA, the peak should be at 260 nm, and as a general rule, the 260/280 ratio should be between 1.8 and 2.0. To test multiple samples, just wipe the sensor in between measurements with a Kimwipe. Recalibration or re-blanking is not necessary.  Clean the sample surfaces once more after you are finished.
</P></DIV></DIV><DIV class="row_small" style="height:500px;"><DIV class="protocol"><P class="minor_title">PCR</P><P class="body_text">
in a small 100uL reaction tube, add the following reagents:
</P><TABLE><TBODY><TR><TH><B>Component</B></TH><TH>50ul Reaction</TH><TH>Final Concentration </TH></TR><TR><TD>H2O</TD><TD>to 50uL</TD></TR><TR><TD>5X NED Phusion buffer</TD><TD>10uL</TD><TD>1X</TD></TR><TR><TD>10mM dNTP</TD><TD>1ul</TD><TD>200uM</TD></TR><TR><TD>10mM Forward Primer</TD><TD>2.5uL</TD><TD>0.5uM</TD></TR><TR><TD>10mM Reverse Primer</TD><TD>2.5uL</TD><TD>0.5uM</TD></TR><TR><TD>Template DNA</TD><TD>2uL</TD><TD>&lt;250ng</TD></TR><TR><TD>DMSO (optional)</TD><TD>1.5uL</TD><TD>3%</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5uL</TD><TD>1.0 units</TD></TR></TBODY></TABLE><P class="body_text">
Thermocycling conditions:
</P><TABLE><TBODY><TR><TH><B>Step</B></TH><TH>Temperature</TH><TH>Time </TH></TR><TR><TD>Initial Denaturation</TD><TD>98C</TD><TD>30s</TD></TR><TR><TD>25-35 cycles</TD><TD>98C

45-72C

72C</TD><TD>5-10s

10-30s

15-30s</TD></TR><TR><TD>Final Extension</TD><TD>72C</TD><TD>5-10min</TD></TR><TR><TD>Hold</TD><TD>4-10C</TD></TR></TBODY></TABLE></DIV><DIV class="description"><P class="minor_title">Ligation</P><P class="body_text">
In a 1.5mL eppendorf tube, add the following reagents, adds up to 10uL reaction volume
</P><TABLE><TBODY><TR><TH><B>Reagent</B></TH><TH>Volume</TH></TR><TR><TD>T4 Ligase</TD><TD>1uL</TD></TR><TR><TD>T4 ligase buffer</TD><TD>1uL</TD></TR><TR><TD>Insert</TD><TD>2ul</TD></TR><TR><TD>Backbone</TD><TD>2ul</TD></TR><TR><TD>H2O</TD><TD>4uL</TD></TR></TBODY></TABLE></DIV></DIV><P class="major_title">Mammalian Lab Protocols</P><DIV class="row_small"><DIV class="description"><P class="minor_title">Passaging Adherent Cells</P><P class="body_text">
In order to keep cells healthy or increase stock, they must be sub-cultured - moving some cells from a previous culture into a new container with fresh growth medium. Here, we assume a 100mm dish. All solutions/equipment that come in contact with the cells must be sterile and work must be done in a laminar flow hood.  
</P><P class="body_text"><B>1)</B> Pipette spent medium and discard to waste.
</P><P class="body_text"><B>2)</B> Gently wash cells with PBS (5-10mL), then remove PBS to waste. Be careful not to disturb the cellular monolayer. This removes serum residue with trypsin inhibitors.
</P><P class="body_text"><B>3)</B> Add trypsin (2-5mL) to suspend cells. Ensure monolayer is covered. Incubate for 3-5 minutes at 37C. 
</P></DIV><DIV class="full_row"><P class="body_text"><B>NOTE:</B> Care should be taken to
avoid leaving cells exposed to the trypsin
longerthan necessary. Care should also be
taken that the cells not be forced to detach
prematurely, as this may result in clumping.
</P><P class="body_text"><B>4)</B>Add serum-containing medium(10mL) and pipette the cells up and
down until the cells are dispersed into
a single cells suspension. 
</P><P class="body_text"><B>5)</B> Add the appropriate volume of cell
suspension (dependent on confluence/cell count - generally for 100% confluence split 1:4) to a new ﬂask/dish containing medium (end volume 10mL).
</P><P class="body_text"><B>7)</B> Place dish(es) in incubator at 37C. Leave for 3-4 days before next passage.  
</P></DIV></DIV><DIV class="row_small"><DIV class="description"><P class="minor_title">Stable Transfection Of Adherent Cells</P><P class="body_text">
For the stable transfection of eukaryotic adherent cell types in a single well of a 6-well plate. When transfecting multiple wells, make a 'master mix' with 110% of all solutions.
</P><P class="body_text"><B>1)</B> The day before transfection, seed 0.9-4x10^5 cell per well of the six well plate with 2ml of appropriate growth medium. This should produce a confluence of 40-80% for the next day's transfection.
</P><P class="body_text"><B>2)</B> Incubate cells in their normal growth conditions (37^0 C and 5% CO2) for 24 hours.
</P></DIV><DIV class="full_row"><P class="body_text"><B>3)</B> Dilute 2µg of DNA dissolved in TE buffer (min conc. 0.1µg/µl) with serum, protein and antibiotic free medium (to avoid macromolecular interference with complex formation) to a total of 100µl. Mix.
</P><P class="body_text"><B>4)</B> Add 10µl of superfect (SF) reagent to the solution. Vortex for 10 seconds.
</P><P class="body_text"><B>5)</B> Incubate at room temperature for 5-10mins to allow for complex formation.
</P><P class="body_text"><B>6)</B> Meanwhile, gently aspirate growth medium from dish and wash cells with 3ml. 
</P><P class="body_text"><B>7)</B> Add 600µl of cell growth medium (with serum and antibiotics)to reaction tube. Mix up and down with pipette and immediately transfer total volume to well.    
</P><P class="body_text"><B>8)</B> Change medium and wash with PBS.  
</P><P class="body_text"><B>9)</B> Incubate for 24-48 hours. 
</P><P class="body_text"><B>10)</B> Assay for gene expression. 
</P></DIV></DIV><DIV class="row_small"><DIV class="description"><P class="minor_title">Amyloid Degradation Assay</P><P class="body_text">
Dissolve Aβ in dimethyl sulfoxide (Me2SO, Sigma) to a concentration of 5mM. Dilute in MQ water to a final concentration of 25 μm immediately prior to use.
</P><P class="body_text">
To prepare Aβ fibrils (fAβ), dilute 5 mm Aβ1-42 or Aβ1-40 in Me2SO in 10 mm HCl to 100 μm (for Aβ1-42) or 200 μm (for Aβ1-40), vortex for 30 s, and incubate at 37 °C for 5 days.
</P><P class="body_text">
Dissolve Aβ in dimethyl sulfoxide (Me2SO, Sigma) to a concentration of 5mM. Dilute in MQ water to a final concentration of 25 μm immediately prior to use.
</P></DIV><DIV class="full_row"><P class="body_text">
To prepare Aβ fibrils (fAβ), dilute 5 mm Aβ1-42 or Aβ1-40 in Me2SO in 10 mm HCl to 100 μm (for Aβ1-42) or 200 μm (for Aβ1-40), vortex for 30 s, and incubate at 37 °C for 5 days.
</P><P class="body_text">
a.  Activate pro-MMP-9 with 1 mm p-aminophenylmercuric acetate at 37 °C for 24 h prior to use. This step is not necessary with <A href="http://parts.igem.org/Part:BBa_K1018001" target="_blank">(BBa_K1018000)</A>, as it contains the active form. 
</P><P class="body_text">
b. For fAβ digestions, 200 nm protease was added to 10 μl of fAβ in reaction buffer and incubated at 37 °C for 4 h to 5 days.
</P><P class="body_text">
c. After digestion, analyse the reaction by Congo red assay.
</P></DIV></DIV><DIV class="row_small"><DIV class="description"><P class="minor_title">Congo Red Spectrophotometric assay </P><P class="body_text"><B>1)</B> Make up a 7 mg/mL solution of Congo Red in a buffer solution of 5mM potassium phosphate, 150mM NaCl (pH7.4). Filter through a 0.2µm syringe immediately before using.
</P><P class="body_text"><B>2)</B> At room temperature zero a UV–Vis spectrophotometer between 400 and700 nm with a disposable cuvette containing 1mL phosphate buffer.
</P><P class="body_text"><B>3)</B> To the the phosphate buffer, add 5µL of the Congo Red solution. Scan between 400 and 700 nm and take a record of the spectrum.
</P></DIV><DIV class="full_row"><P class="body_text"><B>4)</B> Add 5–10µL of protein solution (or transfected HeLa/microglia lysate mixed with degraded amyloid - remember to also include a control) to the cuvette. Incubate for 30 min at room temperature. A red precipitate may become visible. Pipette the solution up and down to mix the contents. Take a record of the spectrum between 400 and 700 nm.
</P><P class="body_text"><B>5)</B> Subtract mathematically the Congo Red spectrum from the protein/lysate-Congo Red spectrum. A maximal spectral difference at 540nm is indicative of amyloid fibrils.
</P><P class="body_text"><B>6)</B> For a microscopic analysis, transfer the protein/lysate-Congo Red solution to a centrifuge tube. Centrifuge (12,000–14,000 rpm) to pellet the fibrils. Wash the fibrils with water, resuspend the fibrils in a small amount of water, and place on a microscope slide. Let the sample dry in air and analyse under polarized light. If transfection with MMP-9 has been successful, the assay should not be strongly indicative of fibrils.
</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UCL/Project/Protocols">http://2013.igem.org/Team:UCL/Project/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCL/Project/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCL/Project/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCL/Project/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCL/Project/Protocols&amp;oldid=317350" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>