"
Page
Discussion
View source
History
teams
Log in
 
Team:SDU-Denmark/Tour23
From 2014.igem.org
Edible coli
Welcome
Interactive wiki-tour
Menu
Introduction
The team
Gallery
Social networking
Attributions
Edible coli
Facts and statistics
How can we help?
System design
Modelling
Process
Helping others
Notebook
SOPs
Safety
Results
Expressions
OneProt
Fatty acids
Flavor
Submitted parts
Judging criteria
Policy and practices
An expert opinion
Ethics
Business plan
Sharing the message
The video adventure
Future
Next steps
Reality
The end
Introduction
The team
Gallery
Social networking
Attributions
Edible coli
Facts and statistics
How can we help?
System design
Modelling
Process
Helping others
Notebook
SOPs
Safety
Results
Expressions
OneProt
Fatty acids
Flavor
Submitted parts
Judging criteria
Policy and practices
An expert opinion
Ethics
Business plan
Sharing the message
The video adventure
Future
Next steps
Reality
The end
System design
"You have to think anyway, so why not think big?" - Donald Trump
Original thought
The original design of the system making up the Edible coli contains the four elements listed below, in a K12 MG1655 strain of Escherichia coli:
Excreting cellulases for the degradation of cellulose to glucose:
Cellulose (C6H10O5)n consists of β-1,4 linked D-glucose units. For the Edible coli to gain nutrients from cellulose in the form of glucose units, the β-glucosidic bonds in-between must be broken by  hydrolysis. For this degradation three enzymatic activities are needed by the enzymes, collectively  known as cellulases: Endoglucanase, exoglucanase and β-glucosidase. Endoglucanase hydrolyses  internal β-1,4 glucosidic bonds in the cellulose fiber, while exoglucanase hydrolyses the external  bonds, releasing cellobiose disaccharides. The cellobiose disaccharides are then cleaved by β- glucosidase into two glucose molecules each. Source: Lynd, L.R., et al.: Microbial Cellulose Utilization: Fundamentals and Biotechnology. Microbiology and Molecular Biology Reviews, 2002. Vol. 66:3, p. 506-577. (Link)
Reaction and biobricks needed for the reaction to run (BioBricks produced by the Edinburgh iGEM team 2008):
Figure 1: Endoglucanase: cenA ( BBa_K118023 ), exoglucanase: cex ( BBa_K118022 ), and β-glucosidase: bgIX ( BBa_K118028 ).
Producing a nutrional, self-designed protein – the OneProt:
Among others, using the glucose from cellulose degradation as a nutrition source, the Edible coli will be able to produce a high quantity of essential amino acids, incorporated into a self-designed  protein.
One protein to rule them all!
The OneProt design:
Figure 2: The optimal rates of essential amino acids.
Figure 3: The composition of essential amino acids.
Figure 4: The composition of non-essential amino acids.
The nutritional protein consists of the right amount of essential and non-essential amino acids including the right ratio of the essential amino acids, needed in the daily diet, as recommended by the WHO/ FAO/UNU Expert Consultation. Source: WHO/FAO/UNU Expert Consultation: Protein and Amino Acid Requirements in Human Nutrition. United Nations University, 2002. No. 935, p. 164. (Link) The optimal rates of essential amino acids are shown in figure 2.
Figure 5: Predicted structure of the OneProt. The protein sequence encodes 480 amino acids in total. The amount of essential amino acids is based on the recommended ratio between essential and non-essential amino acids, which  should be 27.7%. Source: WHO/FAO/UNU Expert Consultation: Protein and Amino Acid Requirements in Human Nutrition. United Nations University, 2002. No. 935, p. 150. (Link) The ratios of given essential amino acids are based on the recommendations in figure 2, and the ratios of given non-essential amino acids are based on how large an amount is used  and processed in the body, and whether they can be formed from essential amino acids or not.
In all, the protein comprises 136 essential (28.3%) and 344 non-essential (71.6%) amino acids,  with the distributions shown in figure 3 and figure 4. To make sure the design would not contain any  harmful protein structures or misfold, the sequence has been shuffled, codon optimized for E. coli, K12 MG1655, checked for restriction sites and its structure has been predicted, using Phyre2.
The expression of this coding sequence (basic part: BBa_K1475001 ), and thus the synthesis of nutritional protein will be regulated by the  repressible promoter, pTet ( BBa_R0040 ):
Figure 6: Expression of OneProt regulated by repressible Tet promoter.
The activity of pTet regulated by the repressor protein, TetR ( BBa_C0040 ), should be tested with and without LVA-tag by regulating the expression of GFP:
Figure 7: Expression of GFP regulated by repressible Tet promoter: TetR(+LVA) vs. TetR(no LVA).
GFP controlled by TetR(+LVA)-pTet ( BBa_K1475006 )
GFP controlled by TetR(no LVA)-pTet ( BBa_K1475005 )
Synthesizing ω3- and ω6 fatty acids:
Figure 8: Conventional biosynthetic pathway for the production of ω3- and ω6 fatty acids. Source: Ruiz-López, N., et al.: Metabolic engineering of the omega-3 long chain polyunsaturated fatty acid biosynthetic  pathway into transgenic plants. Journal of Experimental Botany, 2011. Vol. 63:7, p. 2397-2410. Stearic acid (18:0) is part of the metabolism of E. coli. A Δ9 desaturase can convert stearic acid  into oleic acid (18:1Δcis:9), a Δ12 desaturase can convert oleic acid into linoleic acid (18:2Δcis:9,12), which is an  essential ω6 fatty acid, and a Δ15 desaturase can convert linoleic acid into α-linoleic acid (18:3Δcis:9,12,15),  which is an essential ω3 fatty acid, ALA.
A conventional biosynthetic pathway from linoleic and α-linoleic acid precursors to other ω3- and ω6 fatty acid products is shown in figure 1. Source: Ruiz-López, N., et al.: Metabolic engineering of the omega-3 long chain polyunsaturated fatty acid biosynthetic  pathway into transgenic plants. Journal of Experimental Botany, 2011. Vol. 63:7, p. 2397-2410. (Link) ALA can be converted into both EPA and DHA in the human body, all important in cellular functions. Source: Nelson, D.L. and Cox, M.M.:Lehninger – Principles of Biochemistry, fifth edition. W.H. Freeman and Company, 2008. p. 345.
Synechocystis sp. strain PCC6803, a cyanobacterium, has been shown to contain Δ9, Δ12 and Δ15 desaturases, which has been cloned into E. coli. Source: Sakamoto, T., et al.: Δ9 Acyl-Lipid Desaturases of Cyanobacteria. The Journal of Boilogical Chemistry, 1994. Vol. 269:14, p. 25576-25580. (Link)   Source: Wada, H., et al.: The desA gene of the cyanobacterium Synechocystis sp. strain PCC6803 is the structural gene for delta 12 desaturase. Journal of Bacteriology, 1993. Vol. 175:18, p. 6056-6058. (Link) These three biobricks are needed for the conversion of stearic acid into α-linoleic acid (Manchester, 2013) - The expression of these three enzymes will be regulated by the repressible promoter, pLac ( BBa_R0010 ):
Figure 9: Expression of Δ9 desaturase ( BBa_K1027001 ), Δ12 desaturase ( BBa_K1027002 ) and Δ15 desaturase, all regulated by the repressible Lac promoter.
Δ12 desaturase is also found in Caenorhabditis elegans ( BBa_K1475002 ).
The expression of nutritional protein, OneProt, and desaturases for the biosynthesis of essential  fatty acids should be regulated by two different regulatory promoters, pTet and pLac relatively,  to investigate if the activity of these promoters can be fine-tuned – and if so, to estimate the optimal  activity of both for a maximum production of both the protein and the given fatty acids in the  same organism, Edible coli.
Tasting good:
For the Edible coli to be a bacterium considered in the context of food, it should taste good. Several flavors are a possibility, but we chose a touch of lemon:
Limonene synthase catalyzes synthesis of the terpenoid (+)-limonene from the precursor geranyl  diphosphate, which is mainly responsible for the lemon taste in Citrus Limon. Source: Lükcker, J., et al.: Monoterpene biosynthesis in lemon (Citrus Limon). European Journal of Biochemistry, 2002. Vol.  269:13, p. 3160-3171. (Link) The biobrick, dsx+LIMS1+appY ( BBa_K118024 ) (Edinburgh, 2008), generates 1-deoxyxylulose-5-phosphate synthase, encoded by dsx, which catalyzes the first step in the biosynthesis of terpenoids, and a transcriptional regulator related  to anaerobic energy metabolism, encoded by appY. Overexpression of dxs and appY has been  reported to increase yields of terpenoids. Source: Kang, M.J. et al.: Identification of Genes Affecting Lycopene Accumulation in Escherichia coli Using a Shot-Gun Method. Biotechnology and  Bioengineering, 2005. Vol. 91, p. 636-642. (Link) LIMS1 encodes limonene synthase from Citrus Limon.
Figure 10: Expression of dxs, LIMS1 and appY genes for generation of limonene.
With the lemon flavor incorporated, the system should be operated in a knock out strain of K12  MG1655: The odor-free YYC912 ( BBa_J45999 ).
Original overview drawing of the Edible coli project.
Next chapter
Next chapter
Dig deeper
End of tour
Want more? Go to iGEM.org!
If you want to learn more about Synthetic Biology you should visit iGEM's homepage, which contains loads   on the subject. You can even visit other teams' wikis and take a look at what they accomplished. Or maybe   join a team yourself?
A tour for everyone
Click here to begin the interactive wiki-tour
Take a quick glance at our project or dig deeper to get a complete walkthrough of   the wiki. Or do something in-between, the choice is yours.
Retrieved from " http://2014.igem.org/Team:SDU-Denmark/Tour23 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
