"
Page
Discussion
View source
History
teams
Log in
 
Team:Gothenburg/Parts/Constructs
From 2014.igem.org
TemplateUp
Home
Team
Official Team profile
Unofficial Team profile
Parts
Constructs
Part Submission
Results
Notebook
Calendar
Methods
Modeling
BTB
Human Practice
Ethics
Safety
Attributions
Apply!
Project Description
Nowadays the determination of replicative age of yeast cells is done by counting the budding scars of each cell under a microscope, a time consuming and not efficient process. Our team goal within the iGEM competition is to construct a yeast generation counter. The idea is that each time the cell divides a different fluorescent protein [1, 2] is beeing expressed. Therefore, by examining the cell under a microscope or in a flow cytometer, one can determine how many times the cell has been divided. This would be achieved by constructing a logical AND gate in the cell [3] where the input signals consist of a cyclin activated dCas9-VP64 and a guide RNA (gRNA) [4] signal from the previous cell cycle. The output response consist of a different fluorescent protein and a new gRNA molecule, see Figure 1.
Figure 1. Schematic representation of the logical AND gate with the input      and output signals.
dCas9-VP64 is an engineered inducible transcription factor only active when dimerized with an interchangeable gRNA molecule. The gRNA also determines the specificity of the transcription factor which enables the dCas9-VP64 to activate e different genes depending on the sequence of the gRNA molecule and the promoter. gRNA consists of two parts, a scaffold and a 20 bp Specificity Determining Sequence (SDS) on the 5' end. The scaffold constitutes the majority of the gRNA molecule and gives it its structure whereas the SDS binds to the target site in the gene promoter. Cyclins are proteins that are involved in the progression of the cell cycle, therefore activated at specific times of the cycle [5]. To mimic the specific production pattern of cyclins the dCas9-VP64 gene is placed under the control of the yeast HO promoter (pHO), which triggers the expression in the G1 phase. When the dCas9-VP64 and gRNA dimerize and the transcription factor is activated, it in turn activates the transcription of a new fluorescent protein and a new gRNA molecule to act as a memory for the next cycle.
In order for this generation counter to achieve its full potential, i.e. becoming an efficient and versatile tool for yeast age determination, it has to reset itself every time a new daughter cell is born. In this regard, we devised a construct called the Daughter-resetter; it contains the DSE4 promoter (pDSE4) which is an endogenous promoter from Saccharomyces cerevisiae. pDSE4 is a daughter specific promoter which initiates transcription of the DSE4 gene only in daughter cells and  more specifically in the early G1 phase [6, 7].  The specific function is unknown but it is speculated that it can be responsible in the degradation of the cell wall and separate daughter cells from mother cells [7].  This promoter allows for the counter to be reset for daughter cells, it controls the transcription of the first fluorescent protein, i.e. YFP, and the first gRNA signal.
In our approach, YFP and gRNA are transcribed as a single mRNA, which causes  specific challenges. The YFP-mRNA is desired to be translated into a polypeptide, but the gRNA should remain as RNA in the cell until the subsequent cell division. To solve the problem of the joint transcription a ribozyme sequence is added in between the two genes.  Ribozymes,  when transcribed  into RNA,  form  secondary structures  which  spontaneously  cause them to  be cleaved of from the surrounding  mRNA [8,9]. Therefore the YFP and gRNA will become separate mRNA strands.  But a problem occurs in the preprocessing of the mRNA [10]. The 5’ 7-methylguanosine cap and the 3’-  poly-adenosine  tail  which  act  both  as  protection  of  the  mRNA and as transportation signals will form on either ends of the pre-mRNA thus only a cap on the YFP and a polyA-tail on the gRNA. The aim is to have both these additions on the YFP-mRNA to signal for translation but not any on the gRNA. Therefore the YFP ORF is located first after the promoter to receive the 3’-cap and a synthetic polyA-tail is added to the 3’-end [9]. To remove the formed polyA-tail on the gRNA 3’-end an additional ribozyme sequence is added which will cleave off itself together with the polyA-tail after transcription and pre-processing.
The gRNA is flanked by a 28-nucletide sequence on each side to protect it from degradation and make the activity inducible by Csy4. Csy4 is an endonuclease from Pseudomonas aeruginosa  which  activates  the  gRNA  by  cleaving  off  the  flanking  sequences  after transcription. The expression of Csy4 is triggered by the CLN1 promoter (pCLN1), which in its natural function controls the expression of the early G1-phase cyclin Cln1.  Therefore Csy4 also is expressed only in early G1.
The unprocessed gRNA will presumably stay in the plasma until the next cell cycle, where Csy4 is expressed to activate the gRNA and the mother specific promoter HO will activate the production of dCAS9-VP64. The dCAS9-VP64 will then bind to the gRNA to form the active transcription factor, thus both requirements of the AND gate are fulfilled.  The activated dCAS9-VP64 will bind to the CYC1m1 promoter and the transcription of the new fluorescent protein, i.e. CFP, and the second gRNA are induced.
All produced protein (YFP, CFP, Csy4, dCAS9-VP64) have a degradation-tag added, either a truncated Sic1 or Clb3 domain. These have been added to prevent signal bleed between cell cycles. The Sic1 domain is added to the N-terminus of Csy4 and dCas9-VP64 with a four glycine linker sequence in between. Sic1 is a cyclin-dependent kinase inhibitor, which controls the G1/S-phase transition in yeast and is degraded before entry into the S-phase. The degradation of Sic1 is initiated by phosphorylation by Cdc28 which in turn is a signal for the ubiquitin-mediated proteolysis pathway [11]. The fusion of Sic1 to Csy4 and dCas9-VP64 is performed to initiate the degradation of both proteins to remove them for the beginning of the next cell cycle. Clb3 is a cyclin that activates Cdc28 and thereby triggers the transition from G2 to M phase in the cell cycle. It contains a nine amino acid long destruction box that signals for the degradation by ubiquitination at the end of the M-phase [12]. The Clb3 degradation domain was employed to trigger degradation of the fluorescent proteins.
In summary, once a daughter cell is formed, pDSE4 will be activated. As a consequence, only the first fluorescent protein (YFP) will be expressed. Together with YFP, the gRNA signal for the next cell cycle will be expressed. When the second cell cycle begins, Csy4 and dCas9 will be expressed; the former will activate the gRNA, cleaving off the 28-nt at both sides, while the latter will bind to the active gRNA. The gRNA-dCas9-VP64 complex will activate only the promoter containing the sequence complementary to the given specificity determinant sequence. This will allow only the second fluorescent protein (CFP) and the next gRNA signal to be expressed.  Given the combinatorial freedom of engineering the SDS, we expect our design to be extensible to an arbitrary long count. Once this age counter is successfully implemented in yeast cells, it would be possible to sort cells according to their replicative age automatically and with high-throughput.
Figure 2. Schematic concept of the planned workflow of the four constructs.
All parts were amplified from a genome of S. cerevisiae using specific primers, with expection of the PolyA-Ribozyme-28nt-SDS-gRNA-28nt-Ribozyme-fragments, the Csy4 and the dCas9, which were ordered.
The four constructs were planned to be transformed into four seperate low copy plasmids, pTEF413-16, with selective amino acid markers for uracil, histidine, leucine and tryptophane.
References
1.                   Hackett,  E.A.,  et  al.,  A  family  of  destabilized  cyan  fluorescent  proteins  as transcriptional reporters in S. cerevisiae. Yeast, 2006. 23(5): p. 333-349.
2.                   Andersen,  J.B., et al., New unstable  variants  of green fluorescent  protein  for studies of transient  gene expression  in bacteria. Applied and environmental  microbiology, 1998. 64(6): p. 2240-2246.
3.                   Moon, T.S., et al., Genetic programs  constructed  from layered logic gates in single cells. Nature, 2012. 491(7423): p. 249-253.
4. Farzadfard, F., S.D. Perli, and T.K. Lu, Tunable and multifunctional eukaryotic transcription factors based on CRISPR/Cas. ACS synthetic biology, 2013. 2(10): p. 604-613.
5.                   Nasmyth, K., At the heart of the budding yeast cell cycle. Trends in Genetics, 1996. 12(10): p. 405-412
6. Afonso, B. Silverm P.A, and Ajo-Franklin, C,M. A synthetic circuit for selectively arresting daughter cells to create aging populations. Nucleic acids research, 2010. 38(8): p2727-35
7. Colman-Lerner A., Chin TE., and Brent R. Yeast Cbk1 and Mob2 activate daughter-specific genetic programs to induce asymmetric cell fates. Cell, 2001.107(6): p. 739-750.
8. Huang, Y., Carmichael, G.G.  Role of Polyadenylation in Nucleocytoplasmic Transport of mRNA. Molecular and Cellular Biology 1996. 16(4): p. 1534-1542.
9. Dower K., et  al.,  A synthetic  A tail  rescues  yeast  nuclear  accumulation  of  a  ribozyme- terminated transcript. RNA, 2004. 10(12): p. 1888-1899.
10. Howe, K. J., RNA polymerase II conducts a symphony of pre-mRNA processing activities. Biochimica et Biophysica Acta - Gene Structure and Expression, 2002. 1577(2): p. 308-324.
11. Sheaff, R.J., and Roberts J.M., End of the line: proteolytic degradation of cyclin-dependent kinase inhibitors. Chemistry and biology, 1996. 3(11) :p. 869-873.
12. Nasmyth K., At the heart of the budding yeast cell cycle. Trend in genetics, 1996. 12(10): p. 405-412.
Upload file
Special pages
My preferences
Privacy policy
Disclaimers
Retrieved from " http://2014.igem.org/Team:Gothenburg/Parts/Constructs "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
