<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Gothenburg_Calendar"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Gothenburg/Calendar">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Gothenburg/Calendar&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Gothenburg/Calendar&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Gothenburg/Calendar&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Gothenburg/Calendar" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Gothenburg/Calendar</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>TemplateUp</TITLE></P><DIV class="stuff"><NAV><UL style="padding-left:105px"><LI><A class="red" href="https://2014.igem.org/Team:Gothenburg">Home</A></LI><LI><A class="orange" href="https://2014.igem.org/Team:Gothenburg/Team">Team</A></LI><LI><A class="orange" href="https://igem.org/Team.cgi?id=1503" target="_blank">Official Team profile</A></LI><LI><A class="orange" href="https://2014.igem.org/wiki/index.php?title=Team:Gothenburg/Unofficial_Team_Page">Unofficial Team profile</A></LI></UL><LI><A class="yellow" href="https://2014.igem.org/Team:Gothenburg/Parts">Parts</A></LI><LI><A class="yellow" href="https://2014.igem.org/Team:Gothenburg/Parts/Constructs">Constructs</A></LI><LI><A class="yellow" href="https://2014.igem.org/Team:Gothenburg/Parts/Part_Submission">Part Submission</A></LI><LI><A class="yellow" href="https://2014.igem.org/Team:Gothenburg/Parts/Results">Results</A></LI><LI><A class="green" href="https://2014.igem.org/Team:Gothenburg/Notebook">Notebook</A></LI><LI><A class="green" href="https://2014.igem.org/Team:Gothenburg/Calendar">Calendar</A></LI><LI><A class="green" href="https://2014.igem.org/Team:Gothenburg/Methods">Methods</A></LI><LI><A class="teal" href="https://2014.igem.org/Team:Gothenburg/Modeling">Modeling</A></LI><LI><A class="cyan" href="https://2014.igem.org/Team:Gothenburg/BTB">BTB</A></LI><LI><A class="cyan" href="https://2014.igem.org/Team:Gothenburg/BTB/Human_Practice">Human Practice</A></LI><LI><A class="cyan" href="https://2014.igem.org/Team:Gothenburg/BTB/Ethics">Ethics</A></LI><LI><A class="blue" href="https://2014.igem.org/Team:Gothenburg/Safety">Safety</A></LI><LI><A class="violet" href="https://2014.igem.org/Team:Gothenburg/Attributions">Attributions</A></LI><LI><A class="pink" href="https://2014.igem.org/Team:Gothenburg/Apply">Apply!</A></LI></NAV></DIV><ARTICLE id="contentDialog" title="Content">
        Click a date with posts to show the content, or click outside this box to close it!
    </ARTICLE><SECTION id="calendar"><TABLE id="june"><H1>June</H1><TBODY><TR class="week"><TD id="2014-05-26" class="date prev-month"><H1>26</H1></TD><TD id="2014-05-27" class="date prev-month"><H1>27</H1></TD><TD id="2014-05-28" class="date prev-month"><H1>28</H1></TD><TD id="2014-05-29" class="date prev-month"><H1>29</H1></TD><TD id="2014-05-30" class="date prev-month"><H1>30</H1></TD><TD id="2014-05-31" class="date prev-month"><H1>31</H1></TD><TD id="2014-06-01" class="date"><H1>1</H1></TD></TR><TR class="week"><TD id="2014-06-02" class="date"><H1>2</H1></TD><TD id="2014-06-03" class="date"><H1>3</H1></TD><TD id="2014-06-04" class="date"><H1>4</H1></TD><TD id="2014-06-05" class="date"><H1>5</H1></TD><TD id="2014-06-06" class="date"><H1>6</H1></TD><TD id="2014-06-07" class="date"><H1>7</H1></TD><TD id="2014-06-08" class="date"><H1>8</H1></TD></TR><TR class="week"><TD id="2014-06-09" class="date"><H1>9</H1></TD><TD id="2014-06-10" class="date"><H1>10</H1><UL class="titles"><LI>Safety Instructions</LI></UL><SPAN class="hidden"><P> The team received mandatory safety training and general lab routine information.
			</P><P> A really cool video about Chalmers' chemical safety routine (recorded at our very own lab!) can be seen in http://www.chalmers.se/insidan/EN/about-chalmers/environmental-work/policies/chemical-safety-routine</P></SPAN></TD><TD id="2014-06-11" class="date"><H1>11</H1></TD><TD id="2014-06-12" class="date"><H1>12</H1></TD><TD id="2014-06-13" class="date"><H1>13</H1></TD><TD id="2014-06-14" class="date"><H1>14</H1></TD><TD id="2014-06-15" class="date"><H1>15</H1></TD></TR><TR class="week"><TD id="2014-06-16" class="date"><H1>16</H1></TD><TD id="2014-06-17" class="date"><H1>17</H1><UL class="titles"><LI>Introduction to centrifuges and autoclaves</LI></UL><SPAN class="hidden"><P> As a part of the mandatory training required by the SysBio group, the team received instructions on how to operate the centrifuges 
					and autoclaves of the labs.
					</P><P>We also worked on our risk declaration; a mandatory document containing our experiments descriptions and information about the chemicals we are going to use, their handling, storage and associated risk, as well as waste handling.</P></SPAN></TD><TD id="2014-06-18" class="date"><H1>18</H1></TD><TD id="2014-06-19" class="date"><H1>19</H1></TD><TD id="2014-06-20" class="date"><H1>20</H1><UL class="titles"><LI>Finished mandatory training</LI></UL><SPAN class="hidden"><P>Our Risk Declaration was approved by the lab manager and our supervisors. We also sign a safety certificate stating that we are aware 
					of the safety instructions and with that, completed the mandatory &quot;check-in list for new incomers&quot; of the lab.</P></SPAN></TD><TD id="2014-06-21" class="date"><H1>21</H1></TD><TD id="2014-06-22" class="date"><H1>22</H1></TD></TR><TR class="week"><TD id="2014-06-23" class="date"><H1>23</H1><UL class="titles"><LI>Media Preparation</LI></UL><SPAN class="hidden"><P>On the first day of laboration, we prepared LB, YPD and SC-Ura/His/Trp-Leu and YPD media. 
					
                </P></SPAN></TD><TD id="2014-06-24" class="date"><H1>24</H1></TD><TD id="2014-06-25" class="date"><H1>25</H1><UL class="titles"><LI>Amplification and Purification of non-synthetic parts</LI></UL><SPAN class="hidden"><P> Succesfully PCR amplified five out of the seven parts from yeast genome via a Phusion High-Fidelity DNA Polymerase. 
						</P><P>The PCR program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC </LI><LI>Gradient from 52 to 61ºC for 30s </LI><LI>1 min at 72ºC </LI><LI>Go to step 2 (repeat 39 times) </LI><LI>10 min at 72ºC. </LI></OL></P><P>A diagnostic gel was performed and the purification of the succesfully amplified parts was done using a PCR Purification Kit from GenJet following the instructions 
						of the manufacture's manual but with no addition of isopropanol. </P></SPAN></TD><TD id="2014-06-26" class="date"><H1>26</H1><UL class="titles"><LI>Plasmid Purification</LI><LI>Amplification and purification of parts</LI></UL><SPAN class="hidden"><H2>Plasmid Purification</H2><P> Performed plasmid purification on E. coli cells containing plasmids p413, p414, p415, p416, p2055 and a plasmid containig our Cas9 sequence 
						(pTPG1-dCas9-UPGP)via a GenJet Plasmid Purification Kit. The manufacter's instructions were followed, except the elution buffer was subtituted by 
						water on the last step. </P><H2>Amplification and purification of parts</H2><P>Performed a PCR amplification of the non-synthetic parts that failed on the day before and on the dCas9 sequence and the three fluorescent proteins
						-Yellow, Cyano and Red (YFP, CFP and RFP, respectively). </P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase (see protocol here) and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC </LI><LI>Gradient from 50 to 55ºC for 30s </LI><LI>30s at 60ºC </LI><LI>2min 15s at 72ºC </LI><LI>Go to step 2 (repeat 39 times) </LI><LI>10 min at 72ºC. </LI></OL></P><P>A diagnostic gel was performed and the purification of the succesfully amplified parts was done using a PCR Purification Kit from GenJet following the instructions 
						of the manufacture's manual but with no addition of isopropanol. </P></SPAN></TD><TD id="2014-06-27" class="date"><H1>27</H1><UL class="titles"><LI>Gel extraction</LI><LI>PCR amplification and merging</LI></UL><SPAN class="hidden"><H2>Gel extraction</H2><P>Extracted the fluorecent proteins amplified on the day before with a GeneJet gel extraction kit.The manufacter's 
						instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>PCR amplification and merging</H2><P>Performed a PCR to fuse 	the pDSE4, pHO and dCas9 parts together.</P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>Gradient from 50 to 55ºC for 30s</LI><LI>30s at 60ºC</LI><LI>2min 15s at 72ºC</LI><LI>Go to step 2 (repeat 39 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P>A diagnostic gel was performed. The bands did not seem correct, so this PCR was repeated on the next day.</P></SPAN></TD><TD id="2014-06-28" class="date"><H1>28</H1><UL class="titles"><LI>PCR amplification and merging</LI><LI>Plasmid restriction</LI></UL><SPAN class="hidden"><H2>PCR amplification and merging</H2><P>Repeated the PCR to fuse the pDSE4, pHO and dCas9 parts together.</P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase (see protocol here) and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>50ºC for 30s</LI><LI>58ºC for 30s</LI><LI>2min 15s at 72ºC</LI><LI>Go to step 2 (repeat 39 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P>A diagnostic gel was performed and this did not seemed to work either.</P><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF, p415TEF and p416TEF by using the enzymes SacI and XbaI. </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P></SPAN></TD><TD id="2014-06-29" class="date"><H1>29</H1></TD></TR><TR class="week"><TD id="2014-06-30" class="date"><H1>30</H1><UL class="titles"><LI>PCR amplification and merging</LI></UL><SPAN class="hidden"><H2>PCR amplification and merging</H2>
						Repeated the PCR to fuse the pDSE4, pHO and dCas9 parts together.<P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>50ºC for 30s</LI><LI>58ºC for 30s</LI><LI>2min 30s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P> A diagnostic gel was performed, however the sizes of the resulting fragments did not match the expected.</P><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF, p415TEF and p416TEF by using the enzymes SacI and XbaI. </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><P>Samples were loaded into a gelGreen, that showed p413, p415 and p416 approximately in the 5000bp size band, while p414 presented two bands, one around 4000bp and another at 2000bp.</P><P> this indicates that the 5000bp band corresponds to the uncut plasmid, while the 4000bp and 2000bp are probably contaminations.</P></SPAN></TD><TD id="2014-07-01" class="date next-month"><H1>1</H1><UL class="titles"><LI>Plasmid restriction</LI><LI>PCR amplification</LI></UL><SPAN class="hidden"><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF, p415TEF and p416TEF by using the enzymes SacI and XbaI. </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><H2>PCR amplification</H2><P>Amplified the parts pDSE4, cas9 and pHO. The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>50ºC for 30s</LI><LI>58ºC for 30s</LI><LI>2min 15s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P>	A diagnostic gel was performed, however the sizes of the resulting fragments did not match the expected.
					
					</P></SPAN></TD><TD id="2014-07-02" class="date next-month"><H1>2</H1><UL class="titles"><LI>PCR amplification</LI><LI>Gel extraction</LI><LI>Concentration determination</LI></UL><SPAN class="hidden"><H2>PCR amplification</H2><P>Repeated the amplification of the parts pDSE4, cas9 and pHO. The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>50ºC for 30s</LI><LI>58ºC for 30s</LI><LI>2min 15s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P>	A diagnostic gel was performed and the sizes of the fragments matched for the pDSE4 and pHO, but not for the Cas9.</P><H2>Gel extraction</H2><P>Extracted the DNA fragments previously amplified with a GeneJet gel extraction kit.The manufacter's 
						instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Concentration determination</H2><P>The concentration of all the fragments obtained was measured via a NanoDrop. The samples were measured in duplicates. The results ranged from 8.5 ng of DNA per UL to 42ng/UL.</P></SPAN></TD><TD id="2014-07-03" class="date next-month"><H1>3</H1></TD><TD id="2014-07-04" class="date next-month"><H1>4</H1><UL class="titles"><LI>Plasmid Purification</LI><LI>Plasmid restriction</LI><LI>Gel extraction</LI></UL><SPAN class="hidden"><H2>Plasmid purification</H2><P>Performed plasmid purification on E. coli cells containing plasmids p413, p414, p415, p416 and p2055 via a GenJet Plasmid Purification Kit. The manufacter's instructions were followed, except the elution buffer was subtituted by 
						water on the last step. Each plasmid was purified in duplicates.
					</P><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF, p415TEF and p416TEF by using the enzymes SacI and XbaI. </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water. Control tubes containing just one of the enzymes each were also prepared.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><P> A diagnostic gelGreen was performed and showed that the cleavage of plasmids p413, p415 and p416 was successful.</P><H2>Gel extraction</H2><P>Extracted the cleaved plasmids with a GeneJet gel extraction kit.The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					
					
					</P></SPAN></TD><TD id="2014-07-05" class="date next-month"><H1>5</H1><UL class="titles"><LI>Concentration determination</LI></UL><SPAN class="hidden"><H2>Concentration determination</H2><P>The concentration of the circular and linear plasmids obtained was measured via a NanoDrop. The samples were measured in duplicates. The results ranged from 6.55 ng of DNA per UL to 543.4ng/UL.</P></SPAN></TD><TD id="2014-07-06" class="date next-month"><H1>6</H1></TD></TR></TBODY></TABLE><TABLE id="july"><H1>July</H1><TBODY><TR class="week"><TD id="2014-06-30" class="date prev-month"><LI>PCR amplification and merging</LI><SPAN class="hidden"><H2>PCR amplification and merging</H2>
						Repeated the PCR to fuse the pDSE4, pHO and dCas9 parts together.<P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>50ºC for 30s</LI><LI>58ºC for 30s</LI><LI>2min 30s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P> A diagnostic gel was performed, however the sizes of the resulting fragments did not match the expected.
					</P><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF, p415TEF and p416TEF by using the enzymes SacI and XbaI. </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><P>Samples were loaded into a gelGreen, that showed p413, p415 and p416 approximately in the 5000bp size band, while p414 presented two bands, one around 4000bp and another at 2000bp.</P><P> this indicates that the 5000bp band corresponds to the uncut plasmid, while the 4000bp and 2000bp are probably contaminations.</P></SPAN></TD><TD id="2014-07-01" class="date"><H1>1</H1><UL class="titles"><LI>Plasmid restriction</LI><LI>PCR amplification</LI></UL><SPAN class="hidden"><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF, p415TEF and p416TEF by using the enzymes SacI and XbaI. </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><H2>PCR amplification</H2><P>Amplified the parts pDSE4, cas9 and pHO. The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>50ºC for 30s</LI><LI>58ºC for 30s</LI><LI>2min 15s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P>	A diagnostic gel was performed, however the sizes of the resulting fragments did not match the expected.
					
					</P></SPAN></TD><TD id="2014-07-02" class="date"><H1>2</H1><UL class="titles"><LI>PCR amplification</LI><LI>Gel extraction</LI><LI>Concentration determination</LI></UL><SPAN class="hidden"><H2>PCR amplification</H2><P>Repeated the amplification of the parts pDSE4, cas9 and pHO. The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>50ºC for 30s</LI><LI>58ºC for 30s</LI><LI>2min 15s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P>	A diagnostic gel was performed and the sizes of the fragments matched for the pDSE4 and pHO, but not for the Cas9.</P><H2>Gel extraction</H2><P>Extracted the DNA fragments previously amplified with a GeneJet gel extraction kit.The manufacter's 
						instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Concentration determination</H2><P>The concentration of all the fragments obtained was measured via a NanoDrop. The samples were measured in duplicates. The results ranged from 8.5 ng of DNA per UL to 42ng/UL.</P></SPAN></TD><TD id="2014-07-03" class="date"><H1>3</H1></TD><TD id="2014-07-04" class="date"><H1>4</H1><UL class="titles"><LI>Plasmid Purification</LI><LI>Plasmid restriction</LI><LI>Gel extraction</LI></UL><SPAN class="hidden"><H2>Plasmid purification</H2><P>Performed plasmid purification on E. coli cells containing plasmids p413, p414, p415, p416 and p2055 via a GenJet Plasmid Purification Kit. The manufacter's instructions were followed, except the elution buffer was subtituted by 
						water on the last step. Each plasmid was purified in duplicates.
					</P><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF, p415TEF and p416TEF by using the enzymes SacI and XbaI. </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water. Control tubes containing just one of the enzymes each were also prepared.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><P> A diagnostic gelGreen was performed and showed that the cleavage of plasmids p413, p415 and p416 was successful.</P><H2>Gel extraction</H2><P>Extracted the cleaved plasmids with a GeneJet gel extraction kit.The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					
					</P></SPAN></TD><TD id="2014-07-05" class="date"><H1>5</H1><UL class="titles"><LI>Concentration determination</LI></UL><SPAN class="hidden"><H2>Concentration determination</H2><P>The concentration of the circular and linear plasmids obtained was measured via a NanoDrop. The samples were measured in duplicates. The results ranged from 6.55 ng of DNA per UL to 543.4ng/UL.</P></SPAN></TD><TD id="2014-07-06" class="date"><H1>6</H1></TD></TR><TR class="week"><TD id="2014-07-07" class="date"><H1>7</H1><UL class="titles"><LI>Plasmid restriction</LI></UL><SPAN class="hidden"><H2>Plasmid restriction</H2><P>Cleaved the plasmids p414TEF with the enzymes SacI and XbaI. </P><P>Approximately 500ng of the plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water. Control tubes containing just one of the enzymes each were also prepared.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><P> A diagnostic gelGreen was performed and showed multiple size bands for the samples treated with XbaI. Consultation to the literature showed that this enzyme has not an unique restriction site for p414. </P><P>Therefore the restriction was repeated with the enzyme SpeI, that has an unique cleavage site in p414 located 6bp away from the XbaI's desirable cleavage site. The same procedure described above was used, substituting XbaI for SpeI.</P><P> The disgnostic gel performed on this second cleavage showed correct band sizes, but with too low lines. This indicates low concentration of the plasmid, therefore it was not extracted.</P></SPAN></TD><TD id="2014-07-08" class="date"><H1>8</H1><UL class="titles"><LI>Plasmid restriction</LI></UL><SPAN class="hidden"><H2>Plasmid restriction</H2><P>Cleaved the plasmids p414TEF with the enzymes SacI and SpeI. </P><P>Approximately 500ng of the plasmid was mixed with o.5UL of SacI, 0.5UL of SpeI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water. Control tubes containing just one of the enzymes each were also prepared.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><P> A diagnostic gelGreen was performed and showed again too light bands or no bands at all. </P></SPAN></TD><TD id="2014-07-09" class="date"><H1>9</H1><UL class="titles"><LI>Plasmid restriction</LI></UL><SPAN class="hidden"><H2>Plasmid restriction</H2><P>Repeated the cleavage the plasmids p414TEF with the enzymes SacI and SpeI. </P><P>Approximately 500ng of the plasmid was mixed with o.5UL of SacI, 0.5UL of SpeI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water. Control tubes containing just one of the enzymes each were also prepared.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><P> A diagnostic gelGreen was performed and SacI seemed to be leaving an excessive amount of uncut plasmid while SpeI apparently degrades the sample. </P></SPAN></TD><TD id="2014-07-10" class="date"><H1>10</H1><UL class="titles"><LI>Plasmid restriction</LI><LI>Gel extraction</LI></UL><SPAN class="hidden"><H2>Plasmid restriction</H2><P>Repeated the cleavage the plasmids p414TEF with the enzymes SacI with SpeI or BamHI. </P><P>Approximately 500ng of the plasmid was mixed with o.5UL of SacI, 0.5UL of SpeI or BamHI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water. Control tubes containing just one of the enzymes each and without phosphatase were also prepared.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 20 minutes and then enzymatic inactivation was performed for 5min in a heating block at 75ºC for samples containing BamHI and at 65ºC for samples containing SpeI.</P><P> A diagnostic gelGreen was performed and SacI again seemed to be leaving an excessive amount of uncut plasmid. However, samples treated with SacI and SpeI with and without phosphatase and one of the samples treated with SacI, BamHI and phosphatase seemed to work and were extracted. </P><H2>Gel extraction</H2><P>Extracted the succesfully cleaved plasmids with a GeneJet gel extraction kit.The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					
					</P></SPAN></TD><TD id="2014-07-11" class="date"><H1>11</H1></TD><TD id="2014-07-12" class="date"><H1>12</H1></TD><TD id="2014-07-13" class="date"><H1>13</H1></TD></TR><TR class="week"><TD id="2014-07-14" class="date"><H1>14</H1></TD><TD id="2014-07-15" class="date"><H1>15</H1></TD><TD id="2014-07-16" class="date"><H1>16</H1></TD><TD id="2014-07-17" class="date"><H1>17</H1><UL class="titles"><LI>PCR amplification and merging</LI><LI>Gel extraction</LI></UL><SPAN class="hidden"><H2>PCR amplification and merging</H2><P>Performed a PCR to fuse the following fragments:	<LI>pHO and Sic1 fragments for the Cas9 construct </LI><LI>D-tag and YFP for the Daughter Resetter construct</LI><LI>pCLN1 and Sic1 for the Csy4 construct</LI><LI>D-tag and CFP for the Counter 2 construct </LI></P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>50ºC for 30s</LI><LI>30s at 60ºC</LI><LI>2min 15s at 72ºC</LI><LI>Go to step 2 (repeat 39 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P>A diagnostic gelRed was performed and revealed the expected band sizes for the merging of D-tag with CFP and pHO with Sic1.</P><H2>Gel extraction</H2><P>Extracted the successfully merged fragments with a GeneJet gel extraction kit.The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					
					</P></SPAN></TD><TD id="2014-07-18" class="date"><H1>18</H1><UL class="titles"><LI>PCR amplification and merging</LI><LI>Gel extraction</LI><LI>Dissolution of synthetic parts</LI></UL><SPAN class="hidden"><H2>PCR amplification and merging</H2><P>Performed a PCR to fuse the following fragments:	<LI>D-tag and YFP for the Daughter Resetter construct</LI><LI>pCLN1 and Sic1 for the Csy4 construct</LI></P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>Gradient from 47 to 55ºC for 30s</LI><LI>30s at 60ºC</LI><LI>2min 15s at 72ºC</LI><LI>Go to step 2 (repeat 39 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P>A diagnostic gelRed was performed and revealed the expected band sizes for the merging of D-tag with YFP.</P><H2>Gel extraction</H2><P>Extracted the successfully merged fragments with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Dissolution of synthetic parts</H2><P>The received synthetic parts (g-blocks) were centrifuged at 3000g for 3min and then dissolved with 10UL of sterile water. The solution was transferred to 10UL of TE buffer, resulting in a final concentration of 10ng/UL.</P></SPAN></TD><TD id="2014-07-19" class="date"><H1>19</H1></TD><TD id="2014-07-20" class="date"><H1>20</H1></TD></TR><TR class="week"><TD id="2014-07-21" class="date"><H1>21</H1></TD><TD id="2014-07-22" class="date"><H1>22</H1><UL class="titles"><LI>Amplification and purification of synthetic parts</LI><LI>Gel extraction</LI><LI>Preparation of competent cells</LI></UL><SPAN class="hidden"><H2>Amplification and purification of synthetic parts</H2><P>Performed a PCR amplification of the synthetic parts previously diluted. Namely
						<LI>pCYC1 m1</LI><LI>Csy4</LI><LI>Counter 1</LI><LI>Counter 2</LI></P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC </LI><LI>54ºC for 30s </LI><LI>30s at 60ºC </LI><LI>20s at 72ºC </LI><LI>Go to step 2 (repeat 39 times) </LI><LI>10 min at 72ºC. </LI></OL></P><P>A diagnostic gel was performed and showed the expected band sizes for all the fragments except pCYC1.</P><H2>Gel extraction</H2><P>Extracted the successfully merged parts with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Preparation of competent cells</H2><P>Following the instructions of the protocol &quot;Frozen-EZ Yeast Transformation II&quot; by ZymoResearch, Saccaromyces cerevisiae competent cells were prepared and stored at -80ºC.</P></SPAN></TD><TD id="2014-07-23" class="date"><H1>23</H1><UL class="titles"><LI>Amplification and purification of synthetic parts</LI><LI>Gel extraction</LI><LI>Yeast transformation</LI></UL><SPAN class="hidden"><H2>Amplification and purification of synthetic parts</H2><P>Repeated the PCR amplification of the synthetic part pCYC1 m1.</P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC </LI><LI>54ºC for 30s </LI><LI>30s at 60ºC </LI><LI>20s at 72ºC </LI><LI>Go to step 2 (repeat 39 times) </LI><LI>10 min at 72ºC. </LI></OL></P><P>A diagnostic gel was performed and showed the expected band sizes.</P><H2>Gel extraction</H2><P>Extracted the successfully merged part with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Yeast Transformation</H2><P> Performed transformation on the competent cells previously prepared according to the protocol &quot;Yeast Transformation&quot;.</P><P> Equimollar quantities of the fragments were added to plasmids with different markers. More specifically:
						<LI>Cas9 construct: pHO, Sic1 and Cas9 fragments into the plasmid p413 (Histidine marker)</LI><LI>Csy4 construct: Sic1, pCLN1 and Csy4 fragments into p415 plasmid (Leucine marker)</LI></P></SPAN></TD><TD id="2014-07-24" class="date"><H1>24</H1><UL class="titles"><LI>Plasmid restriction</LI></UL><SPAN class="hidden"><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF and p416TEF with the enzymes SacI and XbaI. </P><P>Approximately 500ng of the plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water. Control tubes containing just one of the enzymes each were also prepared.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 30 minutes and then enzymatic inactivation was performed for 5min in a heating block at 65ºC.</P><P> A diagnostic gelGreen was performed and showed fragments with the correct sizes. </P><H2>Gel extraction</H2><P>Extracted the successfully restricted plasmids with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
						
					</P></SPAN></TD><TD id="2014-07-25" class="date"><H1>25</H1><UL class="titles"><LI>Dissolution of synthetic parts</LI><LI>Amplification of synthetic parts</LI><LI>PCR purification</LI><LI>Concentration determination</LI></UL><SPAN class="hidden"><H2>Dissolution of synthetic parts</H2><P>The received synthetic part pCYC1 m2 was centrifuged at 3000g for 3min and then dissolved with 10UL of sterile water. The solution was transferred to 10UL of TE buffer, resulting in a final concentration of 10ng/UL.</P><H2>Amplification of synthetic parts</H2><P>Performed PCR amplification of the synthetic part pCYC1 m2.</P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC </LI><LI>54ºC for 30s </LI><LI>30s at 60ºC </LI><LI>20s at 72ºC </LI><LI>Go to step 2 (repeat 39 times) </LI><LI>10 min at 72ºC. </LI></OL></P><P>A diagnostic gel was performed and showed the expected band sizes.</P><H2>PCR purification</H2><P>The purification of the succesfully amplified part was done using a PCR Purification Kit from GenJet following the instructions 
						of the manufacture's manual but with no addition of isopropanol and elution was done with 30UL of sterile water. </P><H2>Concentration determination</H2><P>The concentration of the pCYC1 m2 samples obtained was measured via a NanoDrop. The samples were measured in duplicates. The results ranged from 15.1 ng of DNA per UL to 34.2ng/UL.</P></SPAN></TD><TD id="2014-07-26" class="date"><H1>26</H1></TD><TD id="2014-07-27" class="date"><H1>27</H1></TD></TR><TR class="week"><TD id="2014-07-28" class="date"><H1>28</H1><UL class="titles"><LI>Yeast Transformation</LI></UL><SPAN class="hidden"><H2>Yeast Transformation</H2><P> Performed transformation on the competent cells previously prepared according to the protocol &quot;Yeast Transformation&quot;.</P><P> Equimollar quantities of the fragments were added to plasmids with different markers. More specifically:
						<LI>Cas9 construct: pHO, Sic1 and Cas9 fragments into the plasmid p413 (Histidine marker)</LI><LI>Daughter Resetter construct: D-tag, YFP, pDSE4 and Counter 1 synthetic part fragments into p414 plasmid</LI><LI>Counter 2 construct: D-tag, CFP, pCYC1 and Counter 2 synthetic part fragments into p416 plasmid</LI></P></SPAN></TD><TD id="2014-07-29" class="date"><H1>29</H1><SPAN class="hidden">/span&gt;
                </SPAN></TD><TD id="2014-07-30" class="date"><H1>30</H1><UL class="titles"><LI>Observation of the previously transformed cells</LI><LI>Plasmid restriction</LI><LI>Preparation of Competent cells</LI></UL><SPAN class="hidden"><H2>Observation of the previously transformed cells</H2><P>One of the plates with cells transformed on the 28th showed growth, but with contamination by filamentous fungi, while the other presented no growth at all.</P><P>A single yeast colony from the contaminated plate was replated on a YPD-agar plate.
					</P><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF and p416TEF with the enzymes SacI and XbaI. </P><P>Approximately 500ng of the plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water. Control tubes containing just one of the enzymes each were also prepared.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 30 minutes and then enzymatic inactivation was performed for 5min in a heating block at 65ºC.</P><P> A diagnostic gelGreen was performed and showed fragments with the correct sizes. </P><H2>Gel extraction</H2><P>Extracted the successfully restricted plasmids with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Preparation of Competent cells</H2><P> Competent E.coli cells were prepared according to the protocol described by Hanahan(1983) and stored at -80ºC.
					</P></SPAN></TD><TD id="2014-07-31" class="date"><H1>31</H1><UL class="titles"><LI>Inoculation of cells</LI></UL><SPAN class="hidden"><H2>Inoculation of cells</H2><P>The single colony replated on the day before was inoculated in 4mL YPD medium as an overnight culture for transformation.
					</P></SPAN></TD><TD id="2014-08-01" class="date next-month"><H1>1</H1><UL class="titles"><LI>Yeast transformation</LI><LI>Inoculation of cells</LI></UL><SPAN class="hidden"><H2>Yeast Transformation</H2><P> Performed transformation on the competent cells previously prepared according to the protocol &quot;Yeast Transformation&quot;.</P><P> Equimollar quantities of the fragments were added to plasmids with different markers. More specifically:
						<LI>Cas9 construct: pHO, Sic1 and Cas9 fragments into the plasmid p413</LI><LI>Daughter Resetter construct: D-tag, YFP, pDSE4 and Counter 1 synthetic part fragments into p414 plasmid</LI><LI>Counter 2 construct: D-tag, CFP, pCYC1 and Counter 2 synthetic part fragments into p416 plasmid</LI></P><H2>Inoculation of cells</H2><P>Cells transformed on the 28th that showed growth after 3 days were plated on new adequate selective media.
					</P></SPAN></TD><TD id="2014-08-02" class="date next-month"><H1>2</H1></TD><TD id="2014-08-03" class="date next-month"><H1>3</H1></TD></TR></TBODY></TABLE><TABLE id="august"><H1>Augusti</H1><TBODY><TR class="week"><TD id="2014-07-28" class="date prev-month"><H1>28</H1><UL class="titles"><LI>Yeast Transformation</LI></UL><SPAN class="hidden"><H2>Yeast Transformation</H2><P> Performed transformation on the competent cells previously prepared according to the protocol &quot;Yeast Transformation&quot;.</P><P> Equimollar quantities of the fragments were added to plasmids with different markers. More specifically:
						<LI>Cas9 construct: pHO, Sic1 and Cas9 fragments into the plasmid p413 (Histidine marker)</LI><LI>Daughter Resetter construct: D-tag, YFP, pDSE4 and Counter 1 synthetic part fragments into p414 plasmid</LI><LI>Counter 2 construct: D-tag, CFP, pCYC1 and Counter 2 synthetic part fragments into p416 plasmid</LI></P></SPAN></TD><TD id="2014-07-29" class="date prev-month"><H1>29</H1></TD><TD id="2014-07-30" class="date prev-month"><H1>30</H1><UL class="titles"><LI>Observation of the previously transformed cells</LI><LI>Plasmid restriction</LI><LI>Preparation of Competent cells</LI></UL><SPAN class="hidden"><H2>Observation of the previously transformed cells</H2><P>One of the plates with cells transformed on the 28th showed growth, but with contamination by filamentous fungi, while the other presented no growth at all.</P><P>A single yeast colony from the contaminated plate was replated on a YPD-agar plate.
					</P><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF and p416TEF with the enzymes SacI and XbaI. </P><P>Approximately 500ng of the plasmid was mixed with o.5UL of SacI, 0.5UL of XbaI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water. Control tubes containing just one of the enzymes each were also prepared.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 30 minutes and then enzymatic inactivation was performed for 5min in a heating block at 65ºC.</P><P> A diagnostic gelGreen was performed and showed fragments with the correct sizes. </P><H2>Gel extraction</H2><P>Extracted the successfully restricted plasmids with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Preparation of Competent cells</H2><P> Competent E.coli cells were prepared according to the protocol described by Hanahan(1983) and stored at -80ºC.
					</P></SPAN></TD><TD id="2014-07-31" class="date prev-month"><H1>31</H1><UL class="titles"><LI>Inoculation of cells</LI></UL><SPAN class="hidden"><H2>Inoculation of cells</H2><P>The single colony replated on the day before was inoculated in 4mL YPD medium as an overnight culture for transformation.
					</P></SPAN></TD><TD id="2014-08-01" class="date"><H1>1</H1><UL class="titles"><LI>Yeast transformation</LI><LI>Inoculation of cells</LI></UL><SPAN class="hidden"><H2>Yeast Transformation</H2><P> Performed transformation on the competent cells previously prepared according to the protocol &quot;Yeast Transformation&quot;.</P><P> Equimollar quantities of the fragments were added to plasmids with different markers. More specifically:
						<LI>Cas9 construct: pHO, Sic1 and Cas9 fragments into the plasmid p413</LI><LI>Daughter Resetter construct: D-tag, YFP, pDSE4 and Counter 1 synthetic part fragments into p414 plasmid</LI><LI>Counter 2 construct: D-tag, CFP, pCYC1 and Counter 2 synthetic part fragments into p416 plasmid</LI></P><H2>Inoculation of cells</H2><P>Cells transformed on the 28th that showed growth after 3 days were plated on new adequate selective media.
						</P></SPAN></TD><TD id="2014-08-02" class="date"><H1>2</H1></TD><TD id="2014-08-03" class="date"><H1>3</H1></TD></TR><TR class="week"><TD id="2014-08-04" class="date"><H1>4</H1><UL class="titles"><LI>PCR amplification and merging</LI><LI>Gel extraction</LI><LI>Plasmid restriction</LI></UL><SPAN class="hidden"><H2>PCR amplification and merging</H2><P>Performed PCR to fuse the following fragments:<LI>D-tag and CFP </LI><LI>pHO and Sic1 </LI><LI>pDSE4 and YFP-D-tag(previously merged) </LI></P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>30s at 98ºC</LI><LI>Gradient from 43 to 53ºC for 30s</LI><LI>1min 30s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P> A diagnostic gelGreen was performed and the sizes of all the resulting fragments match the expected.</P><H2>Gel extraction</H2><P>Extracted the successfully merged parts with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Plasmid restriction</H2><P>While reviewing primer design documents, the team realized we were using the wrong enzyme to cleave the plasmids: it should be SacI with XhoI instead of XbaI. Therefore, our plasmids did not have the correct overlap with the constructs.</P><P>Cleaved the plasmids p413TEF, p414TEF, p415TEF and p416TEF by using the enzymes SacI and XhoI. </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of SacI, 0.5UL of XhoI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><P>A diagnostic gelGreen was performed and the cleavage showed to be unsuccessful.
						</P></SPAN></TD><TD id="2014-08-05" class="date"><H1>5</H1><UL class="titles"><LI>Plasmid restriction</LI><LI>Gel extraction</LI><LI>Concentration determination</LI><LI>PCR amplification and merging</LI></UL><SPAN class="hidden"><H2>Plasmid restriction</H2><P>Repeated the cleavage of the plasmids p413TEF, p414TEF, p415TEF and p416TEF by using the enzymes SacI and XhoI. </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of SacI, 0.5UL of XhoI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><P>A diagnostic gelGreen was performed and the expected band sizes were observed. </P><H2>Gel extraction</H2><P>Extracted the successfully cleaved plasmids with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Concentration determination</H2><P>Used a NanoDrop spectrometer to determinate the concentration of the previously extracted plasmids. The concentrations ranged from 5.95 to 7.75ng/UL.</P><H2>PCR amplification and merging</H2><P>Performed PCR to fuse the following fragments:<LI>D-tag-CFP (previously merged) with pCYC1 </LI><LI>pHO and Sic1 (previously merged)with Cas9 </LI><LI>pDSE4-YFP-D-tag(previously merged) and Counter 1 (synthetic part)</LI></P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>30s at 98ºC</LI><LI>Gradient from 43 to 60ºC for 30s</LI><LI>2min 30s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P> A diagnostic gelGreen was performed and the sizes of all the resulting fragments did not match the expected.</P></SPAN></TD><TD id="2014-08-06" class="date"><H1>6</H1><UL class="titles"><LI>PCR amplification and merging</LI><LI>Gibson Assembly</LI></UL><SPAN class="hidden"><H2>PCR amplification and merging</H2><P>Repeated the PCR to fuse the following fragments:<LI>D-tag-CFP (previously merged) with pCYC1 </LI><LI>pHO and Sic1 (previously merged)with Cas9 </LI><LI>pDSE4-YFP-D-tag(previously merged) and Counter 1 (synthetic part)</LI></P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>Gradient from 45 to 62ºC for 30s</LI><LI>2min 30s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P> A diagnostic gelGreen was performed and the sizes of all the resulting fragments were smaller than expected and the bands were difuse.</P><H2>Gibson Assembly</H2><P> Attempted to transform the Daughter Resetter construct (pDSE4,YFP,D-tag and Counter 1) into E. coli following the Gibson Assembly protocol from New England Biolab. 
							</P><P> The cells were inoculated in LB-Amp medium and let to grow overnight.
					</P></SPAN></TD><TD id="2014-08-07" class="date"><H1>7</H1><UL class="titles"><LI>Plasmid Purification</LI><LI>Gel extraction</LI><LI>Gibson Assembly</LI></UL><SPAN class="hidden"><H2>Plasmid Purification</H2><P> Performed plasmid purification on E. coli cells containing plasmids p413, p414, p415, p416, p2055 and a plasmid containig our Cas9 sequence 
						(pTPG1-dCas9-UPGP)via a GenJet Plasmid Purification Kit. The manufacter's instructions were followed, except the elution buffer was subtituted by 
						water on the last step. </P><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF, p415TEF and p416TEF by using the enzymes SacI and XhoI. </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of SacI, 0.5UL of XhoI, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 10 minutes and then enzymatic inactivation was performed for 5min in a heating block at 60ºC.</P><P>A diagnostic gelGreen was performed and the expected band sizes were observed. </P><H2>Gel extraction</H2><P>Extracted the successfully cleaved plasmids with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Gibson Assembly</H2><P>Continued with the Gibson Assembly protocol (New England Biolab) and used p414 in the cells with the assembly poducts. A positive control with cells and the circular plasmid was also prepared.</P><P>All the inoculated cultures were placed at 37ºC.
				</P></SPAN></TD><TD id="2014-08-08" class="date"><H1>8</H1><UL class="titles"><LI>Observed results from the Gibson Assembly</LI><LI>Yeast Transformation</LI></UL><SPAN class="hidden"><H2>Observed results from the Gibson Assembly</H2><P>The positive control showed several monoclonal colonies.</P><P>However, the plate with the actual construct showed only one colony which is unlikely to contain the construct.</P><H2>Yeast Transformation</H2><P> Performed transformation on the competent cells previously prepared according to the protocol &quot;Frozen-EZ Yeast Transformation II&quot;.</P><P> Equimollar quantities of the fragments were added to plasmids with different markers. More specifically:
						<LI>Cas9 construct: pHO, Sic1 and Cas9 fragments into the plasmid p413</LI><LI>Daughter Resetter construct: D-tag, YFP, pDSE4 and Counter 1 synthetic part fragments into p414 plasmid</LI><LI>Counter 2 construct: D-tag, CFP, pCYC1 and Counter 2 synthetic part fragments into p416 plasmid</LI></P></SPAN></TD><TD id="2014-08-09" class="date"><H1>9</H1></TD><TD id="2014-08-10" class="date"><H1>10</H1><SPAN class="hidden">!</SPAN></TD></TR><TR class="week"><TD id="2014-08-11" class="date"><H1>11</H1><UL class="titles"><LI>PCR amplification and merging</LI><LI>E. coli transformation</LI></UL><SPAN class="hidden"><H2>PCR amplification and merging</H2><P>Performed PCR to fuse the following fragments:<LI>D-tag and CFP </LI><LI>pHO and Sic1 </LI><LI>pDSE4 and D-tag and YFP</LI><LI>dCas9</LI><LI>YFP and Counter 1</LI><LI>D-tag and pCYC1</LI></P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>30s at 98ºC</LI><LI>53ºC for 30s</LI><LI>2min 30s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P> The samples were stored under refrigeration and analysed on the next day.</P><H2>E. coli transformation</H2><P>Added 10UL of the registry Cas9 plasmid (after a 1:10 dilution) to competent E.coli cells. A control with no DNA addition was also prepared.</P><P>The cells were kept on ice for 30min and then heat shocked at 42ºC for 60s and then kept on ice again for 3min.</P><P>250mL of LB medium were then added and cells were incubated at 37ºC for 90min, under 200rpm shaking.</P><P>After incubation, the cells were centrifuged at 9000rpm for 1min and the supernatant was discarded. The pellet was plated in LB+chlorophenicol plates and let to grow overnight at 37ºC.</P></SPAN></TD><TD id="2014-08-12" class="date"><H1>12</H1><UL class="titles"><LI>Diagnostic gel and gel extraction</LI><LI>E. coli transformation</LI></UL><SPAN class="hidden"><H2>Diagnostic gel and gel extraction</H2><P>A gelRed was run to evaluate the PCR products from the day before. The fragments of dCas9, YFP-Counter 1, D-tag-pCYC1 and D-tag-CFP presented the expected band sizes.</P><P>The successfully merged and amplified were extracted fragments with a GeneJet gel extraction kit.The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>E. coli transformation</H2><P>The succesfully transformed colonies from the day before were replated on new LB medium. </P><P>The transormation procedure was repeated for dCas9 and p413TEF.</P><P>Added 10UL of the registry Cas9 and p413TEF plasmids (after a 1:10 dilution) to competent E.coli cells. A control with no DNA addition was also prepared.</P><P>The cells were kept on ice for 30min and then heat shocked at 42ºC for 60s and then kept on ice again for 3min.</P><P>250mL of LB medium were then added and cells were incubated at 37ºC for 90min, under 200rpm shaking.</P><P>After incubation, the cells were centrifuged at 9000rpm for 1min and the supernatant was discarded. The pellet was plated in LB+chlorophenicol plates and let to grow overnight at 37ºC.</P></SPAN></TD><TD id="2014-08-13" class="date"><H1>13</H1><UL class="titles"><LI>Concentration determination</LI><LI>Yeast Electroporation</LI></UL><SPAN class="hidden"><H2>Concentration determination</H2><P>The concentration of the fragments obtained on the day before was measured via a NanoDrop. The samples were measured in duplicates. The results ranged from 15.3 ng of DNA per UL to 101.6ng/UL.</P><H2>Yeast Electroporation</H2><P>Prepared an overnight culture according to the &quot;Yeast electroporation&quot; protocol.
					</P></SPAN></TD><TD id="2014-08-14" class="date"><H1>14</H1><UL class="titles"><LI>Yeast electroporation</LI></UL><SPAN class="hidden"><H2>Yeast Electroporation</H2><P>Following the &quot;Yeast Electroporation&quot; protocol, Saccaromyces cerevisiae were transformed with the Daughter Resetter construct (D-tag, YFP, pDSE4 and Counter 1 synthetic part fragments into p414 plasmid),
						the Cas9 construct (pHO, Sic1 and Cas9 fragments into the plasmid p413) and the Counter 2 construct (D-tag, CFP, pCYC1 and Counter 2 synthetic part fragments into p416 plasmid).</P><P>Cells were plated in the adequate restrictive media.</P></SPAN></TD><TD id="2014-08-15" class="date"><H1>15</H1><UL class="titles"><LI>Plasmid Purification</LI><LI>PCR amplification and merging</LI></UL><SPAN class="hidden"><H2>Plasmid Purification</H2><P> Performed plasmid purification on the overnight culture of E. coli cells containing the registry Cas9 via a GenJet Plasmid Purification Kit. The manufacter's instructions were followed, except the elution buffer was subtituted by 
						water on the last step. </P><H2>PCR amplification and merging</H2><P>Performed PCR to fuse and amplify the following fragments:<LI>Registry Cas9 plasmid </LI><LI>p413TEF YFP </LI><LI>Registry Cas9</LI><LI>pDSE4</LI></P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>58ºC for 30s</LI><LI>2min 30s at 72ºC</LI><LI>Go to step 2 (repeat 40 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P> The samples were stored under refrigeration and analysed on August 18th.</P></SPAN></TD><TD id="2014-08-16" class="date"><H1>16</H1></TD><TD id="2014-08-17" class="date"><H1>17</H1></TD></TR><TR class="week"><TD id="2014-08-18" class="date"><H1>18</H1><UL class="titles"><LI>Diagnostic gel</LI><LI>PCR purification</LI><LI>Plasmid restriction</LI><LI>Gel extraction</LI></UL><SPAN class="hidden"><H2>Diagnostic gel</H2><P>A gelGreen was run to evaluate the PCR products from the 15th. The fragments showed the expected band sizes.</P><H2>PCR purification</H2><P>The purification of the successfully amplified part was done using a PCR Purification Kit from GenJet following the instructions 
						of the manufacture's manual but with no addition of isopropanol and elution was done with 30UL of sterile water. </P><H2>Plasmid restriction</H2><P>Cleaved the plasmids p413TEF, p414TEF, p416TEF and the registry cas9 plasmid by using the enzymes SpeI and XbaI (for the resgistry plasmid) and SacI and XhoI (for the others). </P><P>Approximately 500ng of each plasmid was mixed with o.5UL of each enzyme, 2UL of FastDigest buffer, 1UL of FastAP 
						(alkaline phosphatase) and up to 20 UL of deionized sterile water.</P><P>This reaction mixture was incubated at in a water bath at 37ºC for 30 minutes and then enzymatic inactivation was performed for 5min in a heating block at 65ºC.</P><P>	A diagnostic gelGreen was performed and the sizes of all the resulting fragments match the expected except p415TEF.</P><H2>Gel extraction</H2><P>Extracted the successfully restricted plasmids with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					
					</P></SPAN></TD><TD id="2014-08-19" class="date"><H1>19</H1><UL class="titles"><LI>Gibson Assembly</LI><LI>Yeast transformation</LI></UL><SPAN class="hidden"><H2>Gibson Assembly</H2><P> Attempted to transform the following constructs into E. coli following the Gibson Assembly protocol from New England Biolab:
								<LI>Daughter Resetter construct (pDSE4,YFP,D-tag and Counter 1 synthetic part) </LI><LI>Cas9 construct (pHO, Sic1 and dCas9)</LI><LI>Counter 2 construct (D-tag, CFP, pCYC1 and Counter 2 synthetic part)</LI></P><P> The cells were inoculated in LB-Amp medium and let to grow overnight.</P><H2>Yeast Transformation</H2><P> Performed transformation on the competent cells previously prepared according to the protocol &quot;Yeast Transformation&quot;.</P><P> Equimollar quantities of the fragments were added to plasmids with different markers. More specifically:
						<LI>Cas9 construct: pHO, Sic1 and Cas9 fragments into the plasmid p413</LI><LI>Csy4 construct: Sic1, pCLN1 and Csy4 fragments into p415 plasmid</LI></P><P> The cells were inoculated in LB-Amp medium and let to grow overnight at 30ºC.</P><H2>E. coli transformation</H2><P>Added 10UL of the registry Cas9 plasmid (after a 1:10 dilution) to competent E.coli cells. A control with no DNA addition was also prepared.</P><P>The cells were kept on ice for 30min and then heat shocked at 42ºC for 60s and then kept on ice again for 3min.</P><P>250mL of LB medium were then added and cells were incubated at 37ºC for 90min, under 200rpm shaking.</P><P>After incubation, the cells were centrifuged at 9000rpm for 1min and the supernatant was discarded. The pellet was plated in LB+chlorophenicol plates and let to grow overnight at 37ºC.</P></SPAN></TD><TD id="2014-08-20" class="date"><H1>20</H1><UL class="titles"><LI>Colony PCR</LI></UL><SPAN class="hidden"><H2>Colony PCR</H2><P>Ran colony PCR on the colonies obtained so far (from different transformation attempts and both in E.coli and yeast). A single colony from each plate was suspended in 100mL LiAc, 1% SD solution and incubated at 70ºC for 5min.
						Then 300mL of 96% ethanol were added and the solution was centrifuged at 15000g for 3min.</P><P>The pellet obtained was washed with 70% ethanol and dissolved in 100mL of water and then centrifuged at 15000g for 15s. 1mL of the obtained supernatant was used for the PCR.&gt;
						</P><P>The PCR program used was: <OL><LI>10 min at 98ºC </LI><LI>10s at 98ºC </LI><LI>Gradient from 50 to 59ºC for 30s </LI><LI>2 min at 72ºC </LI><LI>Go to step 2 (repeat 39 times) </LI><LI>10 min at 72ºC. </LI></OL></P><P>A diagnostic gel was performed and correct band sizes were observed for counter 1 and counter 2 transformations done in 18/8.</P></SPAN></TD><TD id="2014-08-21" class="date"><H1>21</H1></TD><TD id="2014-08-22" class="date"><H1>22</H1><UL class="titles"><LI>Plasmid Purification</LI><LI>Title two</LI></UL><SPAN class="hidden"><H2>Plasmid Purification</H2><P> Performed plasmid purification on E. coli cells containing the plasmid containing the Cas9 part improvement (from 19/8) via a GenJet Plasmid Purification Kit. The manufacter's instructions were followed, except the elution buffer was subtituted by 
						water on the last step. 
						instructions were followed, except the elution buffer was substituted by water on the last step.
					</P><H2>Concentration determination</H2><P>The concentration of the obtained plasmids was measured via a NanoDrop. The samples were measured in four replicates. The results ranged from 35.4 ng of DNA per UL to 289ng/UL.</P></SPAN></TD><TD id="2014-08-23" class="date"><H1>23</H1><UL class="titles"><LI>Fluorescence microscopy</LI><LI>PCR amplification</LI><LI>Gel extraction</LI></UL><SPAN class="hidden"><H2>Fluorescence microscopy</H2><P> Checked cells transformed with the Daughter Resetter construct under a fluorescent microscope and fluorescent yellow cells were observed, as well as dead cells under the green and yellow filter.</P><H2>PCR amplification</H2><P>Performed a PCR to amplify Csy4 cconstruct, dCas9, pHO, Sic1 and pCLN1.</P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>Gradient from 50 to 55ºC for 30s</LI><LI>30s at 60ºC</LI><LI>2min 15s at 72ºC</LI><LI>Go to step 2 (repeat 39 times)</LI><LI>10 min at 72ºC.</LI></OL></P><P>A diagnostic gel was performed. The bands match the expected size for all samples except one of the replicates of dCas9.</P><H2>Gel extraction</H2><P>Extracted the successfully amplified constructs with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					
					</P></SPAN></TD><TD id="2014-08-24" class="date"><H1>24</H1><UL class="titles"><LI>PCR amplification</LI><LI>Concentration determination</LI><LI>Yeast plasmid extraction</LI></UL><SPAN class="hidden"><H2>PCR amplification</H2><P>Performed a PCR to amplify the D-tag of the intended part submission.</P><P>The PCR was done using the Phusion High-Fidelity DNA Polymerase and the program used was: <OL><LI>3 min at 98ºC </LI><LI>10s at 98ºC</LI><LI>Gradient from 50 to 60ºC for 10s</LI><LI>7s at 72ºC</LI><LI>Go to step 2 (repeat 39 times)</LI><LI>5 min at 72ºC.</LI></OL></P><P>A diagnostic gel was performed and the purification of the succesfully amplified parts was done using a PCR Purification Kit from GenJet following the instructions 
						of the manufacture's manual but with no addition of isopropanol. </P><H2>Yeast plasmid extraction</H2><H2><P>Extracted plasmids from yeast cells transformed with the Daughter Resetter and Counter 2 constructs via a Zymoprep Yeast plasmid Miniprep II, following the instructions on the kit. </P></H2><H2>Concentration determination</H2><P>The concentration of the D-tag and plasmid samples was measured via a NanoDrop. The samples were measured in duplicates. The results ranged from 17.1 ng of DNA per UL to 54.6ng/UL.</P></SPAN></TD></TR><TR class="week"><TD id="2014-08-25" class="date"><H1>25</H1><UL class="titles"><LI>Restriction digestion</LI></UL><SPAN class="hidden"><H2>Restriction digestion</H2><P>Using a FastDigest kit with enzymes XbaI and SpeI, attempted to cleave p413, p415, D-tag and D-tag-CFP and linearized psB1C3 backbone. </P><P> A diagnostic gelGreen was performed and the obtained fragments did not match the expected sizes.</P></SPAN></TD><TD id="2014-08-26" class="date"><H1>26</H1><UL class="titles"><LI>Restriction digestion</LI><LI>Gel extraction</LI><LI>Concentration determination</LI></UL><SPAN class="hidden"><H2>Restriction digestion</H2><P>Repeated the cleavage of p413, p415, D-tag and D-tag-CFP and linearized psB1C3 backbone, using a FastDigest kit with enzymes EcoRI and PstI.</P><P> A diagnostic gelGreen was performed and the obtained fragments did not match the expected sizes.</P><P> A diagnostic gelGreen was performed and the obtained fragments match the expected sizes.</P><H2>Gel extraction</H2><P>Extracted the successfully restricted constructs with a GeneJet gel extraction kit. The manufacter's instructions were followed, except the elution buffer was subtituted by water on the last step.
					</P><H2>Concentration determination</H2><P>The concentration of the restricted constructs was measured via a NanoDrop. The samples were measured in duplicates. The results ranged from 1.9 ng of DNA per UL to 39.15ng/UL.</P></SPAN></TD><TD id="2014-08-27" class="date"><H1>27</H1><UL class="titles"><LI>Yeast Transformation</LI><LI>E. coli transformation</LI></UL><SPAN class="hidden"><H2>Yeast Transformation</H2><P> Performed transformation on the competent cells previously prepared according to the protocol &quot;Yeast Transformation&quot;.</P><P> Equimollar quantities of the fragments were added to plasmids with different markers. More specifically:
						<LI>Cas9 construct: pHO, Sic1 and Cas9 fragments into the plasmid p413 (Histidine marker)</LI><LI>Csy4 construct: Sic1, pCLN1 and Csy4 fragments into p415 plasmid (Leucine marker)</LI></P><H2>E. coli transformation</H2><P>Added 3UL of the linearize biobrick backbone and 0.5UL of D-tag to competent E.coli cells. A control with only the linearized backbone was also prepared.</P><P>The cells were kept on ice for 30min and then heat shocked at 42ºC for 60s and then kept on ice again for 3min.</P><P>250mL of LB medium were then added and cells were incubated at 37ºC for 90min, under 200rpm shaking.</P><P>After incubation, the cells were centrifuged at 9000rpm for 1min and the supernatant was discarded. The pellet was plated in LB+chlorophenicol plates and let to grow overnight at 37ºC.</P></SPAN></TD><TD id="2014-08-28" class="date"><H1>28</H1><UL class="titles"><LI>Inoculation of plates</LI></UL><SPAN class="hidden"><H2>Inoculation of plates</H2><P>Replated transformed yeast cells that showed growth (constructs Csy4 and dCas9 from August  8th and 14th, respectively), as well as E.coli cells transformed with the D-tag part submission.</P></SPAN></TD><TD id="2014-08-29" class="date"><H1>29</H1><UL class="titles"><LI>Colony PCR</LI></UL><SPAN class="hidden"><H2>Colony PCR</H2><P>Ran colony PCR on the replated plates from the day before. A single colony from each plate was suspended in 100mL LiAc, 1% SD solution and incubated at 70ºC for 5min.
						Then 300mL of 96% ethanol were added and the solution was centrifuged at 15000g for 3min.</P><P>The pellet obtained was washed with 70% ethanol and dissolved in 100mL of water and then centrifuged at 15000g for 15s. 1mL of the obtained supernatant was used for the PCR.&gt;
						</P><P>The PCR program used was: <OL><LI>10 min at 98ºC </LI><LI>10s at 98ºC </LI><LI>Gradient from 50 to 59ºC for 30s </LI><LI>2 min at 72ºC </LI><LI>Go to step 2 (repeat 39 times) </LI><LI>10 min at 72ºC. </LI></OL></P><P>A diagnostic gel was performed and correct band sizes were not observed for Csy4 and dCas9. Degradadion and difused bands were observed for the D-tag.</P></SPAN></TD><TD id="2014-08-30" class="date"><H1>30</H1></TD><TD id="2014-08-31" class="date"><H1>31</H1></TD></TR></TBODY></TABLE><TABLE id="september"><H1>September</H1><TBODY><TR class="week"><TD id="2014-09-01" class="date"><H1>1</H1><UL class="titles"><LI>Restriction digestion</LI></UL><SPAN class="hidden"><H2>Restriction digestion</H2><P>Restriction of possible fully constructed Csy4 construct from the 22nd of August using a FastDigest kit with enzymes XhoI and SpeI.</P><P> A diagnostic gelGreen was performed and the obtained fragments did not match the expected sizes.</P><P>A diagnostic gelGreen was performed and the obtained fragments did not match the expected sizes.</P></SPAN></TD><TD id="2014-09-02" class="date"><H1>2</H1></TD><TD id="2014-09-03" class="date"><H1>3</H1></TD><TD id="2014-09-04" class="date"><H1>4</H1></TD><TD id="2014-09-05" class="date"><H1>5</H1></TD><TD id="2014-09-06" class="date"><H1>6</H1></TD><TD id="2014-09-07" class="date"><H1>7</H1></TD></TR><TR class="week"><TD id="2014-09-08" class="date"><H1>8</H1></TD><TD id="2014-09-09" class="date"><H1>9</H1></TD><TD id="2014-09-10" class="date"><H1>10</H1></TD><TD id="2014-09-11" class="date"><H1>11</H1></TD><TD id="2014-09-12" class="date"><H1>12</H1></TD><TD id="2014-09-13" class="date"><H1>13</H1></TD><TD id="2014-09-14" class="date"><H1>14</H1></TD></TR><TR class="week"><TD id="2014-09-15" class="date"><H1>15</H1></TD><TD id="2014-09-16" class="date"><H1>16</H1></TD><TD id="2014-09-17" class="date"><H1>17</H1></TD><TD id="2014-09-18" class="date"><H1>18</H1></TD><TD id="2014-09-19" class="date"><H1>19</H1></TD><TD id="2014-09-20" class="date"><H1>20</H1></TD><TD id="2014-09-21" class="date"><H1>21</H1></TD></TR><TR class="week"><TD id="2014-09-22" class="date"><H1>22</H1></TD><TD id="2014-09-23" class="date"><H1>23</H1></TD><TD id="2014-09-24" class="date"><H1>24</H1></TD><TD id="2014-09-25" class="date"><H1>25</H1></TD><TD id="2014-09-26" class="date"><H1>26</H1></TD><TD id="2014-09-27" class="date"><H1>27</H1></TD><TD id="2014-09-28" class="date"><H1>28</H1></TD></TR><TR class="week"><TD id="2014-09-29" class="date"><H1>29</H1></TD><TD id="2014-09-30" class="date"><H1>30</H1></TD><TD id="2014-10-01" class="date next-month"><H1>1</H1></TD><TD id="2014-10-02" class="date next-month"><H1>2</H1></TD><TD id="2014-10-03" class="date next-month"><H1>3</H1></TD><TD id="2014-10-04" class="date next-month"><H1>4</H1></TD><TD id="2014-10-05" class="date next-month"><H1>5</H1></TD></TR></TBODY></TABLE></SECTION><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Gothenburg/Calendar">http://2014.igem.org/Team:Gothenburg/Calendar</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Gothenburg/Calendar" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Gothenburg/Calendar" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Gothenburg/Calendar&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Gothenburg/Calendar&amp;oldid=221639" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>