<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Fudan_Lux_biowave"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Fudan_Lux/biowave">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Fudan_Lux/biowave&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Fudan_Lux/biowave&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Fudan_Lux/biowave&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Fudan_Lux/biowave" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Fudan Lux/biowave</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>NOVA</TITLE></P><DIV style="position:fixed;width:70px;height:20px;right:50px;bottom:50px"><A href="#">back to top</A></DIV><DIV id="menu" style="background:#A1DAF7;"><DIV id="menu-holder"><DIV class="wrapper"><UL id="nav" class="sf-menu sf-js-enabled sf-shadow"><LI><A href="https://2012.igem.org/Team:Fudan_Lux">Home<SPAN class="subheader">Welcome</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Fudan_Lux/Team">Team<SPAN class="subheader">Awesome</SPAN></A></LI><LI class="current-menu-item"><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/project_introduction">Project<SPAN class="subheader">Cool</SPAN></A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/project_introduction">Overview</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/biowave">Project Biowave</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/nanotube">Project Bacto-Trafficking</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/result">Result</A></LI></UL><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/parts">Parts <SPAN class="subheader">Showcase</SPAN></A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/Modeling">Modeling<SPAN class="subheader">In Math</SPAN></A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/Notebook">Notebook<SPAN class="subheader">Get in touch</SPAN></A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/practice">Practice<SPAN class="subheader">Get in touch</SPAN></A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/labcloud">Project Labcloud</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/Associations">Associations</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/tianmu">Tianmu mountain</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/other practise">Other Practise</A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/safety">Safety<SPAN class="subheader">Get in touch</SPAN></A></LI><LI><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/Gallery">Gallery<SPAN class="subheader">Featured work</SPAN></A></LI></DIV></DIV></DIV><DIV id="main"><DIV class="wrapper"><DIV id="content"><DIV id="page-title"><SPAN class="title">Project Biowave</SPAN><SPAN class="subtitle">You have never seen something like this.</SPAN></DIV><DIV id="posts" class="single"><DIV class="post"><H1><A name="Background">Background</A></H1><DIV class="thumb-shadow"><DIV class="post-thumbnail"><UL class="meta"><LI><P>A group of rough-toothed dolphins surfacing synchronously</P><A href="http://www.cascadiaresearch.org/hawaii/july2010.htm"><P>Origin Link</P></A></LI></UL></DIV></DIV><DIV><P class="dropcap">One of the most interesting and complex types of group behaviors in animals is that several organisms act simultaneously and repetitively at regular intervals of time.<SUP>1</SUP> Ordinarily, this kind of behavior is called “<STRONG>synchronous</STRONG>”. It has been observed in Thailand that male Pteroptyx malaccae fireflies, congregated in trees, flash in rhythmic synchrony with a period of about 560 ± 6 msec (at 28°C). Males of a special Chinese species of firefly called <I><B>Qiongyuying</B></I> flash synchronously during mating season. Besides, we also observed that bees crawling in the same small area display a synchrony of wings fluttering in order to scare off intruders，and we filmed a video clip to demonstrate this phenomenon.</P><P>Moreover, synchrony has been described in a range of mammal groups, including <I><B>odontocete cetaceans</B></I> (a kind of toothed whale).<SUP>2</SUP> The <I><B>odontocete cetaceans</B></I> behave synchronously when they breathe on the surface of water.</P></DIV></DIV><DIV class="post"><DIV class="thumb-shadow"><DIV class="post-thumbnail"><UL class="meta"><LI><P>Synchronous fireflies (<I>Pteroptyx Tener</I>) flashing on the mangrove tree (<I>Sonneratia Caseolaris</I>) at Kuala Selangor, Malaysia (2008). Courtesy of Photo Researchers, Inc.</P><A href="http://www.learner.org/courses/mathilluminated/units/12/textbook/05.php"><P>Origin Link</P></A></LI></UL></DIV></DIV><P>In the microcosm, a synthetic gene oscillator whose oscillatory cycle can be tuned by altering inducer levels, temperature and the media source has been designed by Jesse Stricker and his cooperators to accomplish the synchronous oscillation. It uses both E.coli and yeast as component cells. In this oscillator system, chemicals (<EM>IPTG, arabinos</EM>e) are used as a series of signals to initiate synchronous oscillations within single cells of the colony.<SUP>3</SUP></P></DIV><DIV class="post"><DIV class="thumb-shadow"><DIV class="post-thumbnail"><UL class="meta"><LI><P>Oscillations in the dual-feedback circuit.<STRONG>a</STRONG>, Network diagram of the dual-feedback oscillator.<STRONG>b</STRONG>, Single-cell fluorescence trajectories induced with 0.7% arabinose and 2mM IPTG.
</P><A href="http://www.nature.com/nature/journal/v456/n7221/full/nature07389.html"><P>Origin Link</P></A></LI></UL></DIV></DIV><P>Compared with chemical molecules, optical signal is more efficient for both the macrocosm and the microcosm. We wonder whether we can change the induced signal into light, which is visible and easy to control. It propagates much faster and has countless channels, as each wavelength is a different channel. </P></DIV><DIV class="post"><H1><A name="Introduction">Introduction</A></H1><P class="dropcap">Recently, there has been increasing interest in visible light controlled system. Optogenetics, a seven-year-old field branching out from molecular biology and neuroscience, is leading the trend of creating novel synthesized proteins as “tools” </P><P>helping people understand the brain mechanism. These “opsins” with light-activated domain borrowed from microbes or plants, were transferred into model animal organisms. These controllers are acting in the central part of the light system. Unlike the drug treatment, light can be delivered by lasers with extremely high spatial precision; therefore, one can manipulate only target cells.  </P><P>Our Biowave project is more than performing <SPAN class="pullquote-left"> “Optogenetics can solve controversies that have been going on for many, many years.”</SPAN>light control. By establishing the light depending feedback system within the cell we have realized a light communication system. This is the very first time that putting the intracellular and intercellular light communication together in the biological system. Using the fundamental gene circuit and brand new parts we created, each cell becomes a communication node. </P><P>The first Biowave circuit we have created is the Negative Feedback Biowave. The circuit comprising light sensor and light generator established the negative feedback between them. The properties of time delay, caused by gene expression or the accumulation of the element’s concentration, and negative feedback could form an oscillation of light output within a single cell or the neighbouring cells. By placing the bacteria on a Petri dish and letting them evenly distributed, each cell could sense the light of others equally. With the certain condition and observation analysis, the whole dish could form a detectable synchronized oscillation. In a macroscopic view, this kind of synchronization could be distracted by the attenuation of light. So we could expect a wave like pattern on this level.</P><P>The Positive Feedback Biowave is another attempt for the intracellular and intercellular light communication. Thus the positive feedback under this situation makes the system a bistable circuit; each cell can remain in either a light state or a dark state. But the light irradiation across the colony makes things different. The bacteria would form a static strip or other bistable system caused patterns within or beyond our expected.</P><TABLE border="0" cellpadding="0" cellspacing="0" width="651"><TBODY><TR height="14" style="height:14.0pt"><TD class="xl67" width="208" style="width:208pt">LIGHT</TD><TD class="xl67" width="184" style="width:184pt">CHEMICAL</TD></TR><TR height="14" style="height:14.0pt"><TD height="14" class="xl67" width="100" style="height:14.0pt;width:100pt">manipulate </TD><TD class="xl67" width="208" style="width:208pt">controllable</TD><TD class="xl67" width="184" style="width:184pt">Hard to control</TD></TR><TR height="14" style="height:14.0pt"><TD height="14" class="xl67" width="100" style="height:14.0pt;width:100pt">elimination</TD><TD class="xl68" width="208" style="width:208pt">light induction could be changed
  or erased<SPAN style="mso-spacerun:yes"> </SPAN></TD><TD class="xl67" width="184" style="width:184pt"><SPAN style="mso-spacerun:yes"> </SPAN>hard to wipe out</TD></TR><TR height="28" style="height:28.0pt"><TD height="28" class="xl67" width="100" style="height:28.0pt;width:100pt">transfer
  speed</TD><TD class="xl68" width="208" style="width:208pt">Signal travels fast(as a speed
  of light)
  Non-time-delay</TD><TD class="xl67" width="184" style="width:184pt">Speed is limited by chemical
  diffusion System has a minimum delay</TD></TR><TR height="28" style="height:28.0pt"><TD height="28" class="xl67" width="100" style="height:28.0pt;width:100pt">crosstalk</TD><TD class="xl68" width="208" style="width:208pt">never has the crosstalk with
  constitutive pathway</TD><TD class="xl67" width="184" style="width:184pt">may cross the constitutive
  pathway</TD></TR><TR height="28" style="height:28.0pt"><TD height="28" class="xl67" width="100" style="height:28.0pt;width:100pt">measurement</TD><TD class="xl68" width="208" style="width:208pt">Visible and detectable
  easy to measure</TD><TD class="xl67" width="184" style="width:184pt">hard to detect or measure</TD></TR><TR height="42" style="height:42.0pt"><TD height="42" class="xl67" width="100" style="height:42.0pt;width:100pt">noise
  immunity<SPAN style="mso-spacerun:yes"> </SPAN></TD><TD class="xl67" width="208" style="width:208pt">sensitive 
  could be easily disturbed by the nature light</TD><TD class="xl67" width="184" style="width:184pt">only the same molecule or the
  analogue could disturb the signal pathway</TD></TR><TR height="28" style="height:28.0pt"><TD height="28" class="xl67" width="100" style="height:28.0pt;width:100pt">channel</TD><TD class="xl67" width="208" style="width:208pt">has limited channels(only 390nm
  to 700nm)</TD><TD class="xl67" width="184" style="width:184pt">countless channels(each kind of
  chemical analogues group is a channel)</TD></TR></TBODY></TABLE></DIV><DIV class="post"><DIV><H1><A name="model">Design Process</A></H1><H6>Establish feedback system</H6>
Feedback exists between two parts when each affects the others. In our system, these two parts are the light sensor and the concentration of the light generator protein. By involving the light sensor in the loop, we can establish the feedback.<H6>Light generator</H6><STRONG>Approach</STRONG>Next issue we placed our focus on the properties of the candidate parts. At the beginning we laid eyes on the luciferin illumination system. But the engineering bacteria we were using do not have the luciferin producing enzyme system or the luciferin cycle. Fortunately the 2010iGEM Cambridge has successfully deliver this cycling system into the E.coli. But the wave length does not fit the light sensor which we will discuss later. So we picked another well-known bio illuminator, the Lux operon. It is also presented by 2010 Cambridge. <P><STRONG>Lux operon</STRONG>
           The Lux operon is a set of genes active in bacterial luminescence. The genes encoding the luciferase and a fatty acid reductase complex which synthesizes the substrate are contained in a single operon. When cloned into a recipient host, the lux operon facilitates the generation of a blue-green light with a maximum intensity at 490 nm without the requirement of adding exogenous substrate, which was the prime reason why we chose it to be our light source in this study.</P><P>In our project, we using PCR extracted the DNA sequence from the BBa_K325909, and cloned it under the controllable promoter. The coding device relieved from the original promoter is also available as the BBa_K785003. The growth curve of the Lux transformed dH5α strain is measured. See the summery at the result page.</P><H1>Light sensor</H1><STRONG> ycgF-ycgE-ycgZ endogenous light sensing system</STRONG><P>  Light sensor that directly regulates the expression at the transcription level is rarely seen. Bacterial opsins usually appeal as light activated kinases or phosphodiesterases. 
BBa_K238013, an exact promoter of the blue light receptor, is a portion of an E.coli constitutive light sensing system. This system consists of a receptor, ycgF, which is responsive to blue light. When blue light strikes, the When blue light strikes, the receptor changes conformation and dimerizes driven by the BLUF-domain. This allows it to bind to the ycgE repressor through its EAL-domain, releasing the repressor from the promoter region. In summary, the irradiance of blue light will cause a positive induction, expression downstream  
Light sensing feedback using ycgF-YcgE-YcgZ sensing system.</P><P><STRONG>MG1655 strain</STRONG>The PCR process results that dH5α,the strain we used as normal competent cell, does not contain such endogenous system. The MG1655 strain as a wild type of E.coli K12 was used for the construction of our light sensing design. <A href="http://openwetware.org/wiki/E._coli_genotypes#MG1655"> See MG1655 genotype</A>. </P></DIV><DIV><H1><A name="lightsensor">Sensor protein design</A></H1><P><STRONG>Issue</STRONG>By using the YcgZ light inducing system, we have basically completed our gene circuits. But the performance issue is a critical topic to all the endogenous system.
</P><P>With the help of the previous iGEMer, we could have a comprehensive knowledge of the ycgZ performance. Two charts placed here show the promote level of YcgZ and the leaky expression. </P><P>Weak promotion and high level leaky expression may reduce the ratio of signal to noise. More than that, negative feedback requires a repressive domain with in the sensation system.
To solve the issue rise above, we have launched the DESIGN (Design of Enhanced Sensor Instead General Negative part) subproject aiming at providing a high sensitivity, low leaky expression and plasmid based light sensation negative regulation system.

</P><P><STRONG>tetR repressor</STRONG></P>TetR act as a repressor is operating by binding to the specific DNA sequence, which is a palindromic sequence. The whole binding structure is classified as the HTH motif containing a 
<P>helix-turn-helix protein sequence which is the α2 and α3 helixes in the structure image. 
Using tetR as a repression has several advantages. The tetR has no leaky expression, and the repression can be adjusted by the tetracycline. </P><P><STRONG>LOV domain</STRONG>LOV (light, oxygen or voltage) domains are protein photo sensors that are conserved in bacteria, archaea, plants and fungi, and detect blue light via a flavin cofactor (FMN or FAD).</P><P>Figure 2this cartoon presents just a few examples of the hundreds of LOV domain- containing proteins. The attached domain include: GAF cyclic GMP-specific phosphodiesterases; HTH helix-turn-helix; HK histidine kinase; MASE1 membrane-associated sensor1; PAS PER-ARNT-SIM; REC CheY-like phosphoacceptor (receiver); SpollE sporulation stage II protein E; STAS sulphate transporter anti-σ antagonist; ZnF zinc-finger.</P><P>In bacteria, LOV proteins generally follow the domain organization that is described in other bacterial signaling proteins. The activation domain could be multifunctional; the chart shows the diversity in LOV-signaling output. Such complex domain arrangements can allow the integrationof multiple environmental signals and can provide mechanisms for amplification or attenuation of the light signal.
Clearly, LOV domains are versatile photoswitches. But how do LOV domains regulate such a structurally disparate group of proteins? The mechanisms could be simply described by the following chart. The LOV domain transfer the signal by the Jα helix or the dimerization of the Ncap.
The properties of LOV domain show us the possibility of designing light switch. We have laid eyes on such two LOV opsins: EL222 and AsLov2.
</P><P><STRONG>LOV-HTH</STRONG>
The LOV-HTH protein design is inspire by the natural protein EL222, a 222amino acid protein isolated from the marine bacterium Erythrobacter litoralis HTCC2594. In addition to an N-terminal LOV
domain, EL222 also contains a C-terminal helix-turn-helix (HTH) DNA-binding domain representative of LuxR-type DNA-binding proteins.
The EL222 has the naturally light switch and DNA binding activity, but the regulation sequence should be modified into a repressive operon. The tetR operating sequence shows the same HTH motif. So the HTH swap could be processed. The swapped sequence is shown in the image.
After the HTH swap, Lov-HTH is transformed into a light sensing domain with the PtetO repressive switch function.</P><P><STRONG>LOV-tetR</STRONG>
Beside the unfolding and the dimerization, the Jα helix lies directly behind the normal Lov domain could also make the effective signal output by conducting the allosteric signal. This kind of physical mechanical interaction is delivered by the lever arm like helix. This kind of methodology is successful achieved by Devin Strickland and his colleagues. 
Under our condition, constructing a lever arm between the As Lov2 domain and the tetR is quite crucial. Our strategy is forming a helix to helix connection. The C-terminal of Lov2 and the N-terminal of tetR is directly linked together by delete the non-helix-forming amino acid sequence at the linker part. The linker itself construct a Jα lever arm.
At the dark situation, the Jα delivered allosteric effect make the lov domain become a hindrance of the tetR binding to the DNA. When the protein exposed to a 450nm blue light, Jα can release the LOV domain, DNA binding activity is restored. That trigger the repression switch, expression under the PtetO will be repressed. </P></DIV><DIV><H1><A name="Spectrum Analysis">Spectrum Analysis</A></H1><P>Thus the synchronized oscillation is spatiotemporal distributed; the observation and analysis should be space-time continuum. In order to realize the demonstration of synchronization, we have developed an analysis methodology combining the time-lapse photography and the frequency spectrum analysis.</P><P><STRONG>Time-lapse photography</STRONG></P><P>TOne crucial requirement for the synchronization is the evenly distribution of the bacteria. That means our synchronized oscillation should be performed on a Petri dish, the observation and analysis should adjust to the dish incubation. And there should be absolutely no light disturbance from the environment.</P><P>We have built a wooden camera obscura where the bacteria are cultivated. Digital SLR on the top take photos of luminescence every five minutes. Such a process is considered as the Time-lapse photography. This method allows us to analysis the synchronization spatiotemporally and demonstrates it vividly. </P><P><STRONG>Frequency spectrum analysis</STRONG></P><P>Time-lapse photography provides a series of images which listed according to the time order. To ensure the synchronized oscillation is detectable and reasonable, we choose more than 2000 sampling points on the dish image, and extracted time serials data of each sampling points, and then shown the distribution of frequency of all sampling points by histogram.
</P><P>By this processing, we could easily tell which biowave system is actually preforming a synchronized oscillation. If the histogram shows a concentrating distribution, then we could say the system has make the oscillation synchronized; if the distribution disperse, we could tell that the oscillation of each cell is independent, there is no communication between cells.
</P></DIV><DIV><H1><A name="Application">Application</A></H1><P>Since every cell has its own oscillation, tumor cells must be no exception. However, the spectra of different cells distributes distinctively. We are concerned with the utilization of this oscillator in cancer therapy. Taking energy transformation into consideration, we believe that the formation of the oscillator can initiate changes that should not be ignored to the surroundings. We are curious about it, whether the parameter of the oscillator can be adjusted to a certain number, under which the Escherichia coli or other attenuated strains can inhibit the growth of tumor cells without doing harm to normal cells.</P><P>The oscillator can be used as a tool for the exploration of the evolution of Optogenetics. 
We propose to culture the E. coli strain with the transformed oscillator respectively on each of the three culture media: full nutrition, basic nutrition and lack of different nutrients. By comparing the ratio of plasmid loss and other indices of strain under natural evolution and strains under stressful evolution, we might have chances to find out the reasons that cells seldom use light, which has many advantages over chemicals in our view, as a signal.
</P></DIV><DIV><H1><A name="References">References:</A></H1><P>Julien Herrou* and Sean Crosson Function, structure and mechanism of bacterial photosensory LOV proteinsNature Reviews Microbiology 9, 713-723 (October 2011) </P><P>Devin Strickland, Keith Moffat, and Tobin R. Sosnick* Light-activated DNA binding in a designed
allosteric protein PNAS August 5, 2008 vol. 105 no. 31 10709-10714</P><P>Juan L. Ramos, Manuel Martínez-Bueno The TetR Family of Transcriptional Repressors Microbiol Mol Biol Rev. 2005 June; 69(2): 326–356.</P><P>Abigail I. Nasha,1, Reginald McNultyb,1, Mary Elizabeth Shillitob, Trevor E. Swartzc,2, Roberto A. Bogomolnic, Hartmut Lueckeb,3, and Kevin H. Gardnera,3 Structural basis of photosensitivity in a bacterial light-oxygen-voltage/helix-turn-helix (LOV-HTH) DNA-binding protein PNAS June 7, 2011 vol. 108 no. 23 9449-9454</P></DIV></DIV></DIV><UL id="sidebar"><LI><H6>Biowave</H6></LI><LI class="cat-item"><A href="#Background" title="View all posts">Background</A></LI><LI class="cat-item"><A href="#Introduction" title="View all posts">Introduction</A></LI><LI class="cat-item"><A href="#model" title="View all posts">Design Process</A></LI><LI class="cat-item"><A href="#lightsensor" title="View all posts">Sensor Protein Design</A></LI><LI class="cat-item"><A href="#Spectrum Analysis" title="View all posts">Spectrum Analysis</A></LI><LI class="cat-item"><A href="https://2012.igem.org/wiki/index.php?title=Team:Fudan_Lux/result#11" title="View all posts">Results</A></LI><LI class="cat-item"><A href="#Application" title="View all posts">Application</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="footer"><DIV class="wrapper"><UL id="footer-cols"><LI class="col"><H6>About the Team</H6>
							He he he he!
						</LI></UL></DIV></DIV><DIV id="bottom"><DIV class="wrapper"><DIV id="bottom-text">All rights reserved. Tempalate Edit by Fudan-lux team <A href="http://www.luiszuno.com">Powered By Luiszuno.com</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Fudan_Lux/biowave">http://2012.igem.org/Team:Fudan_Lux/biowave</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Fudan_Lux/biowave" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Fudan_Lux/biowave" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Fudan_Lux/biowave&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Fudan_Lux/biowave&amp;oldid=237302" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>