<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Chassis"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Chassis">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Chassis&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Chassis&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Chassis&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Chassis" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Chassis</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="title"><DIV id="belink"><A href="http://www3.imperial.ac.uk/bioengineering">Department of Bioengineering</A></DIV><DIV id="mblink"><A href="http://www3.imperial.ac.uk/molecularbiosciences">Division of Molecular Biosciences</A></DIV></DIV><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="plan">Plan</A></LI><LI><A href="#" class="listhead" id="results">Results</A></LI><LI><A href="#" class="listhead" id="extra">Extras</A></LI></UL></DIV><DIV id="headpic"><DIV id="headover"><P id="headtit">Parasight<SPAN class="dark"> <SPAN class="highlight">|</SPAN>  Parasite detection with a rapid response</SPAN></P><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London" class="listmain" id="results">Home</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Tour/Page_One" class="listmain" id="results">Tour</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modules" class="listmain" id="results">Modules</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Chassis" class="listmain" id="results">Chassis</A></LI><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Human_Practices" class="listmain" id="results">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Schistosoma" class="listmain" id="results">Schistosoma</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Research" class="listmain" id="results">Research</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Diary/Week_One" class="listmain" id="project">Diary</A></LI></UL></UL><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modelling" class="listmain" id="results">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Strategy" class="listmain" id="results">Assembly</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Lab_Diaries" class="listmain" id="results">Lab Diaries</A></LI><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Protocol" class="listmain" id="results">Lab Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Safety" class="listmain" id="results">Safety</A></LI></UL></UL><UL class="results"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Results" class="listmain" id="results">Experimental Results</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Parts/Favourites" class="listmain" id="results">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/The_Team" class="listmain" id="results">The Team</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Media/Videos" class="listmain" id="results">Media</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/School_Workshops" class="listmain" id="results">School Workshops</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="listmain" id="results">Software Tool</A></LI><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Acknowledgments" class="listmain" id="results">Acknowledgments</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Brainstorming" class="listmain" id="results">Brainstorming</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Glossary" class="listmain" id="results">Glossary</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Sitemap" class="listmain" id="results">Sitemap</A></LI></UL></UL></DIV></DIV></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:25px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Chassis &amp; Vectors
</TD></TR><TR><TD><B>This page is an overview of our chassis <I>B. subtilis</I>, why we chose it, and its key points. We also discuss the vectors used.</B></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Chassis
</TD></TR><TR><TD><P>
Here we list the reasons why <I>B. subtilis</I> serves as a more appropriate chassis option with regards to our project design and main project specifications. 
To re-iterate the pertinent project specifications:
</P><UL><LI>Safe
</LI><LI>Portable
</LI><LI>Cheap
</LI><LI>Fast
</LI><LI>Easy to use
</LI></UL><P>We selected <I>B. subtilis</I> because it is a well characterized gram positive bacterium that is non pathogenic. It was convenient but also a coincidence that within our chosen chassis were features with which we could manipulate to use in our detection module.
</P><P>Below is a brief table summary of how <I>B. subtilis</I> meets our system requirements and also some of the issues we encountered throughout the implementation process and how we overcame them.
</P></TD></TR><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"><I>Bacillus subtilis</I> Breakdown
</TD></TR><TR><TD style="background-color:#FFCC00;text-align:center"><B>Required Components</B></TD><TD style="background-color:#FFCC00;text-align:center"><B>Features of B.subtilis</B></TD></TR><TR><TD style="background-color:#FFFFFF;height:100px;width:400px;text-align:top;"><UL>
   For our detection module we need:
   <LI>A peptide display system combined with
   </LI><LI>A quorum sensing system activated by a linear AIP.
   </LI><LI>A portable chassis </LI></UL></TD><TD style="background-color:#FFFFFF;height:100px;width:400px;text-align:top;"><UL><LI>Utilizes a quorum sensing system based on using small linear  peptides as autoinducers
  </LI><LI>LytC peptide delivery and cell wall anchoring system 
  </LI><LI>Bacillus contains a single cell membrane which alleviates additional complexity transporting our re-engineered surface peptide between compartments
  </LI><LI>Spore forming under stressful conditions
  </LI><LI>High motility for faster detection of the parasite!</LI></UL></TD></TR></TBODY></TABLE><B>Notes:</B><P>Bacillus uniquely utilizes a quorum sensing system based on using small peptides as autoinducers. Many other bacteria including gram negative bacteria use a QSS based on circular or post-translationally modified peptide display systems; this introduced problems with our desire to re-engineer the small peptide fragment. <I>Bacillus</I> however presents linear peptides on its cell wall. Small autoinducing peptides of a downstream signaling system could be released by the cleavage of these membrane-anchored cell surface proteins. This involves including a protease cleavage site between the membrane anchored domain and the autoinducing peptide. Proteases released from the Cercaria parasite would target this site and hence we can detect them as our system becomes activated. 
Protease detector peptides (small linear quorum sensing peptide attached to protease recognition sequence) may also get trapped between the cell wall and cell membrane. To avoid this we looked into mechanisms of transport and attachment of these peptides to the outside of cell wall. An alternative was to secrete detector peptides into the medium as <I>B. subtilis</I> is very good at secreting proteins. The LytC cell wall delivery and anchoring system was chosen to satisfy our system of peptide presentation. Specifically, our protease cleavage site linker peptide plus autoinducing peptide were attached to the cell wall binding domain of an endogenous (to <I>Bacillus</I>) LytC surface protein.
</P><P>N.B. An additional option considered was to attach the small protease detector peptides on to pili (fimbriae) that are normally presented on the bacterial cell. This has been done routinely on <I>E.coli</I> fimbriae at non-conserved regions and outer membrane proteins to present various antigens <A href="/wiki/index.php?title=1&amp;action=edit&amp;redlink=1" class="new" title="1 (page does not exist)">1</A>. There is also a good review of a variety of export patways that have been used <A href="/wiki/index.php?title=2&amp;action=edit&amp;redlink=1" class="new" title="2 (page does not exist)">2</A> in gram negative bacteria. There is a lot work done on gram positive bacteria, although there is a detailed and very accessible review <A href="/wiki/index.php?title=3&amp;action=edit&amp;redlink=1" class="new" title="3 (page does not exist)">3</A> of current understanding of G+ve cell surface.
</P><TABLE width="860px" border="0"><TBODY><TR><TD style="background-color:#FF6666;text-align:center"><B>Issues</B></TD><TD style="background-color:#FF6666;text-align:center"><B>Solution</B></TD></TR><TR><TD style="background-color:#FFFFFF;height:100px;width:400px;text-align:top;"><UL><LI>Extracellular proteases naturally produced and secreted by gram positive bacteria.
These could interfere with the system and could result in false positives or degrade the protease detection peptide. 
</LI></UL></TD><TD style="background-color:#FFFFFF;height:100px;width:400px;text-align:top;"><UL><LI> Protease deficient strains are available</LI></UL></TD></TR></TBODY></TABLE>Safety
<TABLE width="860px" border="0"><TBODY><TR><TD style="background-color:#FFFF99;text-align:center"><B>Required Components</B></TD><TD style="background-color:#FFFF99;text-align:center"><B>Features of B.subtilis</B></TD></TR><TR><TD style="background-color:#FFFFFF;height:100px;width:400px;text-align:top;"><UL><LI>Our system is to be utilized in the field by non experts and to used in the environment,
So we need to ensure the bacterium is SAFE.</LI></UL></TD><TD style="background-color:#FFFFFF;height:100px;width:400px;text-align:top;"><UL><LI>Non-pathogenic</LI></UL></TD></TR></TBODY></TABLE><B>Notes:</B><P><I>Bacillus</I> is a naturally occurring soil bacterium that presents no pathogenicity to humans. We aim to introduce our new DNA into existing essential genes. The disruption of these genes renders the bacteria unlikely to exist outside of the lab or eventual product environment.
</P>Design Considerations
<P><TABLE width="860px" border="0"><TBODY><TR><TD style="background-color:#FFFF99;text-align:center"><B>Required Components</B></TD><TD style="background-color:#FFFF99;text-align:center"><B>Features of ''B.subtilis''</B></TD></TR><TR><TD style="background-color:#FFFFFF;height:100px;width:400px;text-align:top;"><UL><LI>Integration of DNA into the host genome</LI></UL></TD><TD style="background-color:#FFFFFF;height:100px;width:400px;text-align:top;"><UL><LI>Natural Competency and Integration </LI><LI>Very well characterized Gram +ve bacterium</LI></UL></TD></TR></TBODY></TABLE></P><B>Notes:</B><P>Fortunately, <I>B. subtilis</I> was the chassis of choice of the IC iGEM 2008 team. This means that there are now <I>Bacillus</I>-friendly parts in the registry that we can use and also further characterize and contribute towards.
</P><P><TABLE width="860px" border="0"><TBODY><TR><TD style="background-color:#FF6666;text-align:center"><B>Issues</B></TD><TD style="background-color:#FF6666;text-align:center"><B>Solution</B></TD></TR><TR><TD style="background-color:#FFFFFF;height:100px;width:400px;text-align:top;"><UL><LI>Recognition of vectors as foreign DNA and subsequent rejection </LI><LI>Vector incompatibility with those used by ''E.coli''</LI></UL></TD><TD style="background-color:#FFFFFF;height:100px;width:400px;text-align:top;"><UL><LI> Chromosome integration methods, ultimately ensures our introduced DNA is more stable and retained</LI></UL></TD></TR></TBODY></TABLE></P><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Vectors
</TD></TR><TR><TD>Working with <I>B. subtilis</I> we were able to implement three additional  <A href="/Team:Imperial_College_London/Safety" title="Team:Imperial College London/Safety">  safety</A> mechanism in our design by creating three novel transformation vectors: The &lt;bbpart&gt;BBa_K316027&lt;/bbpart&gt; and &lt;bbpart&gt;BBa_K316022&lt;/bbpart&gt; target the AmyE and &lt;bbpart&gt;BBa_K316020&lt;/bbpart&gt; the PyrD genomic loci, thereby disrupting vitally important genes for starch catabolism and uracil synthesis respectively. The cells will thus depend on appropriately supplemented medium and will be unable to survive outside of a controlled environment. Additionally in all the vectors the antibiotic-resistance markers are flanked by Dif-sites which allow excision of the marker sites after successful usage. The &lt;bbpart&gt;BBa_K316027&lt;/bbpart&gt; vector targeting the AmyE locus also contains LacI which allows for Hyper-spank controlled testing constructs to be characterized:
</TD></TR><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">AmyE vector
</TD></TR><TR><TD><P>&lt;bbpart&gt;BBa_K316022&lt;/bbpart&gt;</P><P><B>AmyE locus</B>
This vector has been designed using the amyE 5' &lt;bbpart&gt;BBa_K143008&lt;/bbpart&gt; and 3' &lt;bbpart&gt;BBa_K143009&lt;/bbpart&gt; integration sequences for integration into <I>B. subtilis</I> genome. Insertion into the amyE locus provides a selection marker as the bacterium will no longer be able to breakdown starch. An iodine assay can be used to confirm integration. This phenotype makes the transformed bacterium considerably less likely to survive in natural environments.
</P><P><B>Chloramphenicol Resistance</B>
This vector also contains a positive selection marker, flanked by two dif sites. Chloramphenicol acetyltransferase &lt;bbpart&gt;BBa_J31005&lt;/bbpart&gt; provides resistance to chloramphenicol antibiotic. Dif &lt;bbpart&gt;BBa_K316002&lt;/bbpart&gt; sites allow excision of the antibiotic marker after integration, thus allowing the same marker to be used again or as a precaution against horizontal gene transfer.
</P><P><B>Blunt end cloning site</B>
PmeI restriction site &lt;bbpart&gt;BBa_K316013&lt;/bbpart&gt; is designed as a cloning site. Due to the 8bp recognition sequence it is a rare site that can be used to cut the vector only once.
</P><P><B>Features</B></P></TD></TR><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">AmyE-LacI vector
</TD></TR><TR><TD><P>&lt;bbpart&gt;BBa_K316027&lt;/bbpart&gt;</P><P><B>AmyE locus</B>
This vector has been designed using the amyE 5' &lt;bbpart&gt;BBa_K143008&lt;/bbpart&gt; and 3 &lt;bbpart&gt;BBa_K143009&lt;/bbpart&gt;' integration sequences for integration into <I>B. subtilis</I> genome. Insertion into the amyE locus provides a selection marker as the bacterium will no longer be able to breakdown starch. An iodine assay can be used to confirm integration. This phenotype makes the transformed bacterium considerably less likely to survive in natural environments.
</P><P><B>LacI</B>
With constitutively expressed LacI &lt;bbpart&gt;BBa_K143033&lt;/bbpart&gt; the vector is suitable for <I>B. subtilis</I> transformations with hyper-spank &lt;bbpart&gt;BBa_K143015&lt;/bbpart&gt; controlled gene-circuits. 
</P><P><B>Chloramphenicol Resistance</B>
This vector also contains a positive selection marker, flanked by two dif sites. Chloramphenicol acetyltransferase &lt;bbpart&gt;BBa_J31005&lt;/bbpart&gt; provides resistance to chloramphenicol antibiotic.
</P><P><B>Blunt end cloning site</B>
PmeI restriction site &lt;bbpart&gt;BBa_K316013&lt;/bbpart&gt; is designed as a cloning site. Due to the 8bp recognition sequence it is a rare site that can be used to cut the vector only once.
</P><P><B>Features</B></P></TD></TR><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">PyrD Vector
</TD></TR><TR><TD><P>&lt;bbpart&gt;BBa_K316020&lt;/bbpart&gt;</P><P><B>PyrD locus</B>
This vector has been designed using the Pyrd 5' &lt;bbpart&gt;BBa_K143001&lt;/bbpart&gt; and 3 &lt;bbpart&gt;BBa_K143002&lt;/bbpart&gt;' integration sequences for integration into <I>B. subtilis</I> genome. Insertion into the pyrD locus provides a negative selection marker as the bacterium will no longer be able to synthesise uracil. Thus medium supplement of 40ug/ml is required for growth. This phenotype makes the transformed bacterium considerably less likely to survive in natural environments.
</P><P><B>Spectinomycin Resistance</B>
This vector also contains a positive selection marker, flanked by two dif sites. Aad9 &lt;bbpart&gt;BBa_K143065&lt;/bbpart&gt; provides resistance to spectinomycin antibiotic. Dif  sites &lt;bbpart&gt;BBa_K316002&lt;/bbpart&gt; allow excision of the antibiotic marker after integration, thus allowing the same marker to be used again or as a precaution against horizontal gene transfer.
</P><P><B>Blunt end cloning site</B>
PmeI restriction site &lt;bbpart&gt;BBa_K316013&lt;/bbpart&gt; is designed as a cloning site. Due to the 8bp recognition sequence it is a rare site that can be used to cut the vector only once.
</P><P>Features
</P></TD></TR><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"><I>Dif-site</I> excision in <I>B. subtilis</I></TD></TR><TR><TD><P><I>B. subtilis</I><I>dif</I>: sequence-specific recombinase recognition site
</P><P>In cells with circular chromosomes, recombinatorial repair and homologous recombination can generate multimeric chromosomes <CITE>1</CITE>. <I>Dif</I> sites are part of a system to ensure that multimeric chromosomes can be separated to monomers, which is required for proper sharing of genetic material between daughter cells.
In <I>B. subtilis</I> the tyrosine family recombinases such as RipX and CodV mediate the separation at 28-bp sequence Bs <I>dif</I><CITE>1</CITE>. The site-specific recombinases are able to recognize two directly repeated <I>dif</I> sites and excise the fragment flanked by the two sites <CITE>2</CITE>.
</P><P>Removal of a specific gene from a genome integrated construct:
</P><P>The site-specific recombinases, endogenous to <I>B. subtilis</I> strains are able to recognise <I>dif</I> sites and recombine out the strand of DNA directly flanked by the two sites. Recombination leaves a single <I>dif</I> site.
</P><P>The construct was previously engineered to homologously recobine into the genome of <I>B. subtilis</I>. Integration sequences such as amyE &lt;bbpart&gt;BBa_K143001&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K143002&lt;/bbpart&gt; can be used to achieve this. 
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">References
</TD></TR><TR><TD><P>1) Sciochetti SA, Piggot PJ, and Blakely GW. <I>Identification and characterization of the dif Site from Bacillus subtilis</I>. J Bacteriol 2001 Feb; 183(3) 1058-68. doi:10.1128/JB.183.3.1058-1068.2001 pmid:11208805. PubMed HubMed
</P><P>2) Bloor AE and Cranenburgh RM. <I>An efficient method of selectable marker gene excision by Xer recombination for gene replacement in bacterial chromosomes</I>. Appl Environ Microbiol 2006 Apr; 72(4) 2520-5. doi:10.1128/AEM.72.4.2520-2525.2006 pmid:16597952.
</P><P><I>E.coli</I> was considered as a possible option. Despite the gram negative outer membrane, there are strains that have been made more permeable through knockout of Lipid A biosynthesis in lipopolysaccharides [4] There are also proteins that can disrupt membranes when they are inserted in them, thus making them more permeable. These changes in permeability are likely due to transient ruptures of outer membrane and so unlikely to make a very responsive or robust detecton organism [4].
</P><P>Surplus information on <I>B. Subtilis</I> chassis <A href="http://www.subtiwiki.uni-goettingen.de/wiki/index.php/Main_Page" class="external text" rel="nofollow">Subti-Wiki</A></P><P>1) <A href="http://mic.sgmjournals.org/cgi/content/full/146/12/3025" class="external text" rel="nofollow"><I>Fimbrial surface display systems in bacteria: from vaccines to random libraries</I>. Klemm1 &amp; Schembri, 2000</A></P><P>2) <A href="http://www.microbialcellfactories.com/content/5/1/22" class="external text" rel="nofollow"><I>Surface display of proteins by Gram-negative bacterial autotransporters</I>. Rutherford &amp; Mourez, 2006</A></P><P>3) <A href="http://www3.interscience.wiley.com/cgi-bin/fulltext/118603263/PDFSTART" class="external text" rel="nofollow"><I>Protein cell surface display inGram-positive bacteria: fromsingle protein tomacromolecular protein structure</I>. Desvaux <I>et al</I>, 2006</A></P><P>4) <A href="http://aac.asm.org/cgi/content/full/43/6/1459?view=long&amp;pmid=10348770" class="external text" rel="nofollow">Outer Membrane Permeability Barrier in Escherichia coli Mutants That Are Defective in the Late Acyltransferases of Lipid A Biosynthesis.<I> Vaara1 &amp; Nurminen, 1999</I></A></P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Imperial_College_London/Chassis">http://2010.igem.org/Team:Imperial_College_London/Chassis</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Chassis" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Chassis" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Chassis&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Chassis&amp;oldid=208491" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>