<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Protocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Protocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Protocol&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Protocol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="title"><DIV id="belink"><A href="http://www3.imperial.ac.uk/bioengineering">Department of Bioengineering</A></DIV><DIV id="mblink"><A href="http://www3.imperial.ac.uk/molecularbiosciences">Division of Molecular Biosciences</A></DIV></DIV><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="plan">Plan</A></LI><LI><A href="#" class="listhead" id="results">Results</A></LI><LI><A href="#" class="listhead" id="extra">Extras</A></LI></UL></DIV><DIV id="headpic"><DIV id="headover"><P id="headtit">Parasight<SPAN class="dark"> <SPAN class="highlight">|</SPAN>  Parasite detection with a rapid response</SPAN></P><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London" class="listmain" id="results">Home</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Tour/Page_One" class="listmain" id="results">Tour</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modules" class="listmain" id="results">Modules</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Chassis" class="listmain" id="results">Chassis</A></LI><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Human_Practices" class="listmain" id="results">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Schistosoma" class="listmain" id="results">Schistosoma</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Research" class="listmain" id="results">Research</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Diary/Week_One" class="listmain" id="project">Diary</A></LI></UL></UL><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modelling" class="listmain" id="results">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Strategy" class="listmain" id="results">Assembly</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Lab_Diaries" class="listmain" id="results">Lab Diaries</A></LI><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Protocol" class="listmain" id="results">Lab Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Safety" class="listmain" id="results">Safety</A></LI></UL></UL><UL class="results"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Results" class="listmain" id="results">Experimental Results</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Parts/Favourites" class="listmain" id="results">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/The_Team" class="listmain" id="results">The Team</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Media/Videos" class="listmain" id="results">Media</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/School_Workshops" class="listmain" id="results">School Workshops</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="listmain" id="results">Software Tool</A></LI><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Acknowledgments" class="listmain" id="results">Acknowledgments</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Brainstorming" class="listmain" id="results">Brainstorming</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Glossary" class="listmain" id="results">Glossary</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Sitemap" class="listmain" id="results">Sitemap</A></LI></UL></UL></DIV></DIV></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:25px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Lab protocols
</TD></TR><TR><TD><B>Listed below are the protocols we used for the project. We hope you find them useful!</B></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Restriction Digests
</TD></TR><TR><TD><B>Method:</B><OL><LI>Determine the concentration of the DNA sample by running both the vector and insert on a 1% agarose gel and comparing the bands intensity with the ladder (concentration known).
</LI><LI>Calculate how much solution is needed to obtain desired total amount of DNA for digestion.
</LI><LI>The volume of DNA solution can be no more than 70% of the total solution. Therefore calculate the total volume of digestion (probably around 20µl or 30µl).
</LI><LI>Transfer the DNA, BSA, the appropriate buffer and ddH2O into a microcentrifuge tube. Finally, add the enzymes to the solution. N.B. The enzymes should be kept on ice before being added to the digestion.
</LI><LI>Incubate for 60-90min at 37°C. Put in the freezer or on ice immediately after to stop further digestion. Especially important for EcoRI and other enzymes with star activity.
</LI><LI>Use gel electrophoresis to confirm correct digestion.
</LI><LI>Gel purification can be used to obtain the desired digestion product from the gel.
</LI></OL><P><B>Reaction mixtures:</B></P></TD></TR><TR><TD style="background-color:#FFCC66;height:30px;width:150;text-align:center"><B>Required components</B></TD><TD style="background-color:#FFCC66;height:30px;width:150;text-align:center"><B>Example</B></TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">1/20 Enzyme 1</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">1.5µl EcoRI</TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">1/20 Enzyme 2</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">1.5µl PstI</TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">1/10 BSA</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">3µl BSA</TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">1/10 Buffer* (x 10)</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">3µl Buffer 4 (x 10)</TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">X/10 DNA solution</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">20µl DNA (pSB1C3)</TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">7-X/10 ddH2O</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">1µl ddH2O</TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">Total: 10/10</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">Total: 30µl</TD></TR></TBODY></TABLE><P>Or you can use this as a guide:
</P><OL><LI> 20µl reaction volume unless digesting large amounts of DNA (use 30µl)
</LI><LI> 4µl DNA (if from Midi-preps, use 8µl Mini-Prep DNA)
</LI><LI> 2µl Buffer * (1 in 10µl total volume)
</LI><LI> 2µl 10xBSA (1 in 10µl total volume)
</LI><LI> 1µl Enzyme 1 (use 1.5µl for 30µl digests)
</LI><LI> 1µl Enzyme 2 (most Bio-Brick REF assembly protocols require a second enzyme) (use 1.5µl for 30µl digests)
</LI></OL><P>The buffer depends on the restriction enzymes used.
</P><P><B>Prefix Insertion:</B></P><P><TABLE width="450px" border="0"><TBODY><TR><TD style="background-color:#FFCC66;height:30px;width:150;text-align:center"><B>Enzymes used:</B></TD><TD style="background-color:#FFCC66;height:30px;width:150;text-align:center"><B>Required buffer:</B></TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">Prefix (insert)</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">EcoRI &amp; SpeI</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">Buffer 2</TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">Suffix (vector)</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">EcoRI &amp; XbaI</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">EcoRI buffer</TD></TR></TBODY></TABLE></P><P><B>Suffix insertion:</B><TABLE width="450px" border="0"><TBODY><TR><TD style="background-color:#FFCC66;height:30px;width:150;text-align:center"><B>Enzymes used:</B></TD><TD style="background-color:#FFCC66;height:30px;width:150;text-align:center"><B>Required buffer:</B></TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">Prefix (vector)</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">SpeI &amp; PstI</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">Buffer 2</TD></TR><TR><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">Suffix (insert)</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">XbaI &amp; PstI</TD><TD style="background-color:#eeeeee;height:30px;width:150 px;text-align:center;">Buffer 3</TD></TR></TBODY></TABLE></P><P>SpeI doesn’t cut particularly at the end of PCR products particularly well as there are few flanking bases. Can leave overnight and add the second enzyme as a second 90 minute cutting step.
</P><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Ligations
</TD></TR><TR><TD>A typical ligation reaction mixture is around 10 μl and contains
<OL><LI> 1 μl DNA T4 ligase
</LI><LI> 1 μl DNA T4 ligase buffer (check to ensure it contains ATP) (10x)
</LI><LI> Purified, linearised vector*
</LI><LI> Purified, linearised insert*
</LI><LI> ddH2O
</LI></OL><UL><LI>There should be a ratio of 6:1 for moles of insert to vector. This can be calculated using the following equation:
</LI></UL><P>Insert mass (ng) = 6 x (Insert length (bp)/vector length (bp) x Vector mass (ng)
Once the solution is made up, the tubes are vortexed and then spun down for around 10 seconds in a microcentrifuge. The ligation is done at 14°C in a water bath in the cold cabinet, and is left overnight.
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"><I>E. coli</I> Transformations
</TD></TR><TR><TD><UL><LI>One 15ml tube for each sample, in addition to one for a negative control, is put on ice.
</LI><LI>Tubes containing 1ml LB were incubated in a water bath is set to 42°C .
</LI><LI>Between 25µl and 40µl of competent cells is transferred to each tube. 
</LI><LI>The cells are left on ice for 10min. 
</LI><LI>5µl of the DNA sample are transferred into each tube, but ddH2O is added to the control tube(s). The liquids are added directly into the cell culture.
</LI></UL><P>N.B. During pipetting the sides of the tube should not be touched to avoid contamination. Bubbles should be avoided because they can cause the cells stress. 
</P><UL><LI>The tubes are transferred into the 42°C water bath for exactly 45 seconds and then put on ice for 2 minutes. Timing must be exact.
</LI><LI>The tubes are put on a rack and 1ml LB is added to all of them. This levels the temperature of the solution at about 37°C.
</LI><LI>The tubes are then put into a shaking incubator at 37°C for 1 hour.
</LI><LI>The solution from the 15ml tubes is then transferred to a microcentrifuge tube and spun at 13500 rpm for a few seconds.
</LI><LI>The supernatant is discarded and the remaining LB is mixed with the pelleted cells. This increases the concentration of the cells in the LB.
</LI><LI>50 – 100μl of this solution is then pipetted onto chloraphenicol plates and left overnight at 37°C. The next day colony PCR can be used to examine if the transformation was successful.
</LI></UL></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">PCR
</TD></TR><TR><TD><B>PCR Reaction Mix</B><OL><LI> 25µl Total Reaction Volume
</LI><LI> 18.75µl ddH2O 
</LI><LI> 2.5µl Buffer (Barns for any enzyme or  Taq, Pfu buffer depending on enzyme used)
</LI><LI> 1µl Forward primer
</LI><LI> 1µl Reverse primer
</LI><LI> 1µl Template
</LI><LI> 0.5µl dNTPs
</LI><LI> 0.25µl Enzyme (Taq, Pfu etc)
</LI></OL><P><B>PCR programme</B></P><OL><LI> Heated lid - 110°C
</LI><LI> 35 cycles
</LI><LI> 95°C for 1.5 mins - Denature the template 
</LI><LI> --°C anneal primers
</LI><LI> t°C optimal / time optimal to extend (depends on enzyme used)
</LI></OL><P>(-t°C optimal: 72°C for Taq // time optimal: 2-3kb/60 sec) (-t°C optimal:68°C for Pfu // time optimal: 1kb/15 sec)
</P><OL><LI> Final step 68/72°C for 10/5mins – to allow full extension of any oligonucleotides
</LI></OL><P><B>Single Colony PCR</B>
A master mix is generally used for SCP, as well as Taq polymerase because the high error rate is not an issue here as it is purely confirmatory. 
Cells from an individual colony are first spread onto a replica plate, and the same loop is then used to inoculate a microcentrifuge tube containing 100μ ddH20 which will later be heated to 95°C for 5 minutes to be used in the SCP (the same loop is finally used to inoculate LB for the overnight cultures).
The protocol for the first SCP was as follows:
</P><UL><LI>19.75μl ddH20
</LI><LI>2.5μl Barnes buffer
</LI><LI>1μl template (this comes from the tube that contains 100μ ddH20 and was inoculated with cells.
</LI><LI>0.5μl dNTPs
</LI><LI>0.5μl forward primer
</LI><LI>0.5μl reverse primer
</LI><LI>0.25μl Taq polymerase.
</LI></UL><P>We also used a positive control (other DNA to which the primers will definitely anneal) and a negative control (ddH20).
</P><P>The temperature cycle was as follows:
</P><OL><LI> 95°C for 30 seconds
</LI><LI> 30 cycles of: 95°C for 30 seconds, 62°C for 90 seconds, 68 °C for 30 seconds
</LI><LI> 68°C for 10 minutes
</LI><LI> Hold at 4°C
</LI></OL></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Overnight Cultures
</TD></TR><TR><TD>Tubes containing 5ml of LB medium are inoculated with cells from one colony and then 5μl of antibiotic (for example chloramphenicol) is added. They are then left at 37°C overnight.
</TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">SDS-PAGE
</TD></TR><TR><TD><I>Short for:</I> Sodium Dodecyl Sulfate Polyacrylamide Gel Electrophoresis
<OL><LI><B>Prepare gel</B> : Two glass plates are cleaned with ethanol and are fitted into a holder. The separation layer of the gel is prepared first, following the recipe but before and after addition of the last two substances the solution should be inverted. The mixture, that now starts to polymerize, in now pipetted between the glass plates until it reaches the green bar. Around 700µl of ethanol are than added on top of the gel, which is left to solidify. The separating gel contains 10% acrylamide <B>(toxic!)</B> that has been polymerized by TEMED. Stacking gel contains less acrylamide for wide pores. After the gel has solidified take out the comb. Once solidified the stacking gel can be prepared using a different recipe but same method. Once the ethanol has been removes the solution is poured onto of the gel and the comb inserted. The gel is now left to solidify.
</LI><LI><B>Load samples</B> : The proteins, which have been denatured by SDS, are loaded into the wells.
</LI><LI><B>Run gels</B> : The gel is run at 100V for around 2-3 hours until dye front has reached the bottom of  the gel.
</LI><LI><B>Analysis of results</B> : The gel can be analyzed by staining with Coomassie blue or Western Blot.
</LI></OL></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Catechol Assay
</TD></TR><TR><TD><UL><LI> Catechol assay is performed in the plate reader on a 96 well plate
</LI><LI> Each well must be filled with 100um of solution
</LI><LI> Usually use 90ul of cell culture and 10um of catechol solution
</LI><LI> Catechol stock solution is at 100mM concentration. And when added to the well we have a 10fold dilution. For example if an aliquot concentration of 1mM catechol is made, which would be used for assay, when the well is added catechol drops to a concentration of 0.1mM.
</LI><LI> Always dilute catechol with H2O.
</LI><LI><B>Always</B> have a <B>blank</B> of 90ul medium (the one which you grew the cells overnight) with 10ul catechol solution
</LI><LI><B>Always</B> have a <B>negative</B> of 90ul growing cells and 10ul of H2O.
</LI></UL></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Other Useful Information
</TD></TR><TR><TD><B>PCR purification</B><P>Used to purify DNA to remove primers, salts and enzymes. It can also be used to purify away small fragments from restriction digests, for example when cutting a vector open. We used the E.Z.N.A.® Cycle Pure Kit and protocol (Omega bio-tek) (ddH2O instead of Elusion Buffer used in last step).
</P><P><B>Gel purification</B></P><P>We used the QIAquick® Gel Extraction Kit (250) and protocol (ddH2O instead of Elusion Buffer used in last step).
</P><OL><LI> Excise DNA from the gel and put into BF falcon clip-top tube (Blue Box) (Sybr Safe performs better under blue light)
</LI><LI> Check excision of the right band and weigh the slice ( gel can be frozen at -20°C to be extracted at a later date)
</LI><LI> Add 3xvolume of buffer QG
</LI><LI> Incubate at 50°C for 10 minutes or until completely dissolved, vortex every 2-3 minutes
</LI><LI> Check colour – consult kit protocol if not orange
</LI><LI> Add 1xvolume Isopropanol (crucial step to ensure that the DNA binds the column
</LI><LI> Put 800µl into the QIA quick spin column with 2ml collection tube
</LI><LI> Centrifuge for 1 minute and discard the flow through, repeat if necessary.
</LI><LI> Add 500µl buffer QG to the quick spin column and centrifuge for 1 min
</LI><LI> Add 750µl buffer PE to the quick spin column and centrifuge for 1 min
</LI><LI> Dry the column by centrifuging for 1 minute
</LI><LI> Place QIA quick spin column into 1.5ml collection tube (eppendorf tube)
</LI><LI> Elute the DNA with 35µl of ddH2O
</LI></OL><P><B>Minipreps</B></P><P>The <A href="http://www.omegabiotek.com/product_detail.php?ID=21#" class="external text" rel="nofollow">E.Z.N.A.® Kit</A> and protocol was used.
</P><P><B>Midipreps</B></P><P>The <A href="http://www.qiagen.com/products/plasmid/qiagenplasmidpurificationsystem/hispeedplasmidmaxikit.aspx#Tabs=t0" class="external text" rel="nofollow">QIAGEN HiSpeed Plasmid Midi Kit</A> and protocol was used.
</P><P><B>Agarose gels</B></P><OL><LI> 1% Agarose gel (for DNA 1g Agarose for each 100ml 1xTAE buffer)
</LI><LI> Marker – dilute invitrogen 1kb plus DNA Ladder (1 in 10 Loading Buffer (LB))
</LI></OL><P><B>Diluting Primers</B></P><OL><LI> Add a volume of H2O equivalent to the yield of primer on the information sheet 1µg=1µl ddH2O
</LI><LI> Leave to stand for 20 minutes
</LI><LI> Mix thoroughly (pipette up and down)
</LI><LI> Store at -20°C
</LI><LI> Dilute 1 in 10 before use.
</LI></OL><P><B>Oligo annealing</B></P><OL><LI> 20µ total reaction volume
</LI><LI> 2µl of each single stranded primer
</LI><LI> 16µl ddH2O
</LI><LI> 2µl Buffer 2 (1 in 10 total volume)
</LI></OL><P><B>Sequencing reaction mix</B></P><OL><LI> 15µl total reaction volume
</LI><LI> 3µl of sequencing primer (amplifies the relevant DNA fragment)
</LI><LI> 50-100ηg of DNA
</LI><LI> Label with sequencing labels and make sure that the codes from these are entered on MWG website.
</LI></OL><P><B>DpnI Digests</B></P><P>DpnI digests methylated DNA, such as template DNA extracted from cells (colony PCR/Mini/Midi-prep), while leaving non-methylated PCR products uncut.
</P><OL><LI> Can use 1µl of DpnI straight in the 25µl PCR product (PCR buffer is sufficient)
</LI><LI> Alternatively use Buffer 4 – does not require BSA.
</LI><LI> Incubate for 1 hr at 37°C
</LI></OL><P><B>Rapid Alkaline Phosphatase</B></P><P>Dephosphorylation useful to prevent vector self re-ligation.
</P><OL><LI> 10µl Total Reaction Volume
</LI><LI> --µl DNA vector ( concentration depends on relative concentrations of the parts to be ligated)
</LI><LI> 1µl Phosphatase buffer
</LI><LI> 1µl Alkaline Phosphatase
</LI><LI> -µl make up 10 µl with ddH2O
</LI></OL></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Imperial_College_London/Protocol">http://2010.igem.org/Team:Imperial_College_London/Protocol</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Protocol&amp;oldid=200811" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>