<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Research"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Research">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Research&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Research&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Research&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Research" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Research</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="title"><DIV id="belink"><A href="http://www3.imperial.ac.uk/bioengineering">Department of Bioengineering</A></DIV><DIV id="mblink"><A href="http://www3.imperial.ac.uk/molecularbiosciences">Division of Molecular Biosciences</A></DIV></DIV><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="plan">Plan</A></LI><LI><A href="#" class="listhead" id="results">Results</A></LI><LI><A href="#" class="listhead" id="extra">Extras</A></LI></UL></DIV><DIV id="headpic"><DIV id="headover"><P id="headtit">Parasight<SPAN class="dark"> <SPAN class="highlight">|</SPAN>  Parasite detection with a rapid response</SPAN></P><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London" class="listmain" id="results">Home</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Tour/Page_One" class="listmain" id="results">Tour</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modules" class="listmain" id="results">Modules</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Chassis" class="listmain" id="results">Chassis</A></LI><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Human_Practices" class="listmain" id="results">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Schistosoma" class="listmain" id="results">Schistosoma</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Research" class="listmain" id="results">Research</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Diary/Week_One" class="listmain" id="project">Diary</A></LI></UL></UL><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modelling" class="listmain" id="results">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Strategy" class="listmain" id="results">Assembly</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Lab_Diaries" class="listmain" id="results">Lab Diaries</A></LI><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Protocol" class="listmain" id="results">Lab Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Safety" class="listmain" id="results">Safety</A></LI></UL></UL><UL class="results"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Results" class="listmain" id="results">Experimental Results</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Parts/Favourites" class="listmain" id="results">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/The_Team" class="listmain" id="results">The Team</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Media/Videos" class="listmain" id="results">Media</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/School_Workshops" class="listmain" id="results">School Workshops</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="listmain" id="results">Software Tool</A></LI><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Acknowledgments" class="listmain" id="results">Acknowledgments</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Brainstorming" class="listmain" id="results">Brainstorming</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Glossary" class="listmain" id="results">Glossary</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Sitemap" class="listmain" id="results">Sitemap</A></LI></UL></UL></DIV></DIV></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:25px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Research
</TD></TR><TR><TD><B>A large amount of background reading went into designing our project. Here we each present some of the research papers that we found the most interesting, and what they are all about.</B></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Maddie
</TD></TR><TR><TD><P><B><I>Intersubunit interaction and catalytic activity of catechol 2,3-dioxygenases</I>. Akiko Okuta <I>et al</I>, 2003</B></P><P>This paper was crucial in understanding and developing a means by which we could re-engineer GFP onto our key enzyme C230 in order to render it unable to tetramerize. We had to determine at which terminal to add GFP without disrupting overall function when the monomers are released. This paper tells of such interactions as they truncate and modify the C230 monomer in different ways and then assess its function. From it we were able to deduce the N-terminal to be the best option.   
</P><P><B><I>Chromogenic identification of genetic regulatory signals in Bacillus subtilis based on expression of a cloned Pseudomonas gene</I>. Zukowski <I>et al</I>, 1983 </B></P><P>This paper was provided by Jeremy Bartosiak-Jentys as were most of our key XylE papers referenced. This paper provided the vital information that C2,30 had been successfully expressed in Bacillus subtilis and so gave us confidence that our system would work given our chosen chassis. 
</P><P><B><I>An Autoinhibited Coiled-Coil Design Strategy for Split-Protein Protease Sensors</I>. Ghosh <I>et al</I>, 2009</B></P><P>This was a key paper I came across during brainstorming weeks. It described coiled coil systems that we could use to produce a pre-made but inactive form of our output enzyme, or similarly of the protease that activates the output enzyme. I corresponded with Neel an author of the paper over the summer and he was very useful in providing me information on whether the constructs he'd help create would work in our system. It turned out eventually that the output enzyme we had in mind at the time (GFP) could not be split without spontaneous reformation. This unavoidable high background would mean our system would signal very poorly upon activation. Despite investing lots of time into the idea of split proteins we abandoned the idea after coming across C2,30 which proved a much less problematic alternative and which output could be visualized by eye. The coiled constructs were even sent to us but unfortunately we weren't able to delegate any time to testing with these. Thank you anyway Neel!!! We took the linker sequence that was used in this paper for our fusion protein.
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"> Florian
</TD></TR><TR><TD><P><B><I>Accumulation of a Recombinant Aspergillus oryzae Lipase Artificially Localized on the Bacillus subtilis Cell Surface.</I>  Kobayashi <I>et al</I>, 2000</B></P><P>cutL cDNA encoding a extracellular lipase, L1, from Aspergillus orzyae, was ligated to the cell wall binding domain (CWB) of the major autolysin LytC of B. subtilis. Expression and functionality of the recombinant protein CWB-Cut-L was tested using SDS-PAGE and zymography, showing that catalytic activity of L1 was maintained.
</P><P>This paper was of fundamental importance for our detection module as it laid out nicely the possibility of anchoring peptides on the bacterial cell surface whilst maintaining the functionality. This proved to be precisely what we needed, allowing also for easy testing, as the major autolysin LytC used in this paper, is bind to the cell wall non-covalently and can thus be washed of the cell wall easily using high salt concentrations.
</P><P><B><I>Human schistosomiasis</I>. Gryseels <I>et al</I>, 2006</B></P><P>Schistosomiasis is a tropical disease caused by the water-borne schistosoma parasitehas infected over 200 milion people world wide. The disease is a cause of morbidity and can give rise to a number of complications in liver, intestines and urinary system. These are the result of complex immunological interactions with the parasite eggs in the human tissues. Furthermore diagnosis, treatment and prophylaxis are discussed.
</P><P>This excellent review was most helpful in getting a broad overview of the parasitic life cycle, the disease schsitosomiasis as well as the conditions it can provoke in our body, without lacking detail on these parts. The article rounded up this summary with a great introduction into the treatment, diagnosis and approaches to the development of a vaccination, making it essential in for our wiki page on the parasite and schistosomiasis.
</P><P><B><I>Surface Proteins of Gram-Positive Bacteria and Mechanisms of Their Targeting to the Cell Wall Envelope</I>. Navarre &amp; Schneewind, 1999</B></P><P>This review was a very comprehensive guide to the all different methods used by bacteria such as B. subtilis to secrete proteins and anchor them to the cell wall. It included homologous sequences from many different bacteria, as well as illustrating the systems and difference between these systems in simple manner without lacking detail.
</P><P>This knowledge was indispensable in the design of our cell wall bound detection module, as it gave us an overview of the systems the could potentially be exploited, allowing for further research being much more directed towards specific parts of protein excretion and cell wall binding pathways.
</P><P><B><I>Design of the linkers which effectively separate domains of a bifunctional fusion protein</I>. Arai, <I>et al</I>, 2001</B></P><P>Aim of the paper was to create a series of linker sequences to connect the domains of bifunctional proteins with each other. This allows regulation of the distance and as a result the interference between the domains and was demonstrated with green fluorescent proteins.
</P><P>We made use of 2 of the presented linker sequences, one helical and one flexible linker, in our detection module to connect LytC, the surface protein anchor, with the AIP. We only added a short sequence at both ends of the sequence to allow for efficient functioning of the anchor (beginning) and cleavage (end).
</P><P><B><I>Solution Structure of the Peptidoglycan Binding Domain of Bacillus subtilis Cell Wall Lytic Enzyme CwlC: Characterization of the Sporulation-Related Repeats by NMR</I>. Mishima <I>et al</I>, 2005</B></P><P>This paper examines the structure of the cell wall binding domain of CwlC, which is closely related to LytC, using NMR. It shows that the cell wall binding domain is made up of several repeats and interaction of these with the peptidoglycan residues.
</P><P>Not only did this paper provide new insight into the functional mechanism of non-covalent cell wall anchoring in Gram positive bacteria, but it also suggested an easy method of isolating such proteins: they can be washed of the cells in a solution with high salt concentration. This method was key to the testing strategy we came up with later.
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"> Harriet
</TD></TR><TR><TD><B><I>The neglected tropical diseases:  The ancient afflictions of stigma and poverty and the prospects for their integrated control and elimination.</I>. Hotez <I>et al</I>, 2006</B><P>This paper explains why NTDs are so in need of research and the problems they create in developing countries. Reading this made us want to raise awareness of NTDs and promote research into their diagnosis, treatment and environmental control. The authors also state that reducing the NTD burden would impact on seven of the eight Millennium Development Goals (MDGs) set by the UN. I found the section on Integrated Control particularly interesting, as this is an area in which our detection kit could be applied.
</P><P><B><I>Diffusion of synthetic biology: a challenge to biosafety Systems and Synthetic Biology</I>. Schmidt, 2008</B></P><P>This is a great read if you're interested in the regulation of synthetic biology. I also found it really inspiring as it looks towards the future of the field, but still keeps a sense of realism in terms of biosaftey measures and the accessibility of synthetic biology.
</P><P><B><I>An unmodified heptadecapeptide pheromone induces competence for genetic transformation in Streptococcus pneumoniae</I>. Havarstein, Coomaraswamy &amp; Morrison, 1995</B></P><P>We were really concerned that the system might not be sensitive enough. However, we learnt about threshold levels of the AIP from reading this paper and it reassured us that only a very small concentration (10ng/ml) of CSP-1 is needed to produce a response. Phew!
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Ben
</TD></TR><TR><TD><B><I>Accumulation of an arti¢cial cell wall-binding lipase by Bacillus subtilis wprA and/or sigD mutants</I>. Kobayashi <I>et al</I>, 2000.</B><P>This paper describes how the LytC cell wall binding domain efficiently binds the cell wall of <I>B. subtilis</I> and describes how fusion proteins can be containing cell wall binding domains. It convinced us that we would be able to use this as a method of presenting the protease that we want to detect with a cleavable protein bound to the cell surface.
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Kyasha
</TD></TR><TR><TD><B><I>Mechanism of Interaction of Salmonella and Schitosoma Species</I>. Melhem &amp; LoVerde, 1984</B><P>The interaction of these pathogens can often cause problems with treatment of salmonella infection. In the paper it is tested, by use of pilus positive and pilus negative strains of bacteria, if pili are used for this bacteria-parasite interaction, which is demonstrated conclusively. Furthermore this interaction is shown to be specific as mannose-like receptors and probably glycolipids of the parasite are targeted by the bacteria.
</P><P>Discovering these findings was an important step for us, on the way to finding a feasible approach to parasite detection. It was the first example of specific bacteria-parasite interactions we came across and originally we hoped to exploit the bacterial receptor to detect schistosoma. However no suitable downstream mechanism was found to link receptor with output so ultimately we did not use this system at all but chose a much more elegant approach to parasite detection.
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Kirill
</TD></TR><TR><TD><B><I>An efficient method of selectable marker gene excision by Xer recombination for gene replacement in bacterial chromosomes</I>. Bloor &amp; Cranenburgh, 2006</B><P>In this article, they use dif sites in a smart way to cut out bits of dna we only want in the cell for a short period of time using endogenous recombinases that the cell (<I>E. coli</I> and <I>B. subtilis</I> have already).
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Wolf
</TD></TR><TR><TD><B><I>Cercarial Elastase Is Encoded by a Functionally Conserved Gene Family across Multiple Species of Schistosomes</I>. Salter <I>et al</I>, 2002</B><P>The importance of cercarial elastase across several schistosome species that infect humans is examined. Elastase is found to be the single most important factor for successful skin penetration. Most importantly the paper determines the cleavage specificity of elastase to its substrate.
</P><P>This paper was of crucial importance for two reasons: We had know about the elastase before but this paper first focused on this element of schistosoma and its role in the parasite life cycle. Additionally this paper provided invaluable information about the elastase cleavage specificity and has helped in the design of our cleavable linker.
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Anita
</TD></TR><TR><TD><B><I>Comparative study of parameter sensitivity analyses of the TCR-activated Erk-MAPK signalling pathway</I>. Rundell &amp; Zheng, 2006</B><P>This paper explains parameter sensitivity analysis of a mathematical model, using the example of the MAPK signalling pathway. It also shows different methods of sensitivity analysis. Very useful for modelling!
</P><P><B><I>Dynamics and Control of Synthetic Bio-molecular Networks</I> Sontag &amp; Vecchio, 2007</B></P><P>This is a really useful introduction to synthetic and systems biology, with a very good bit on modelling (shown by famous examples such as the toggle switch!). If you have time, also look at Eduardo Sontag's homepage, where there is loads of good stuff on modelling!
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Piotr
</TD></TR><TR><TD><B><I>Modular cell biology: retroactivity and insulation</I> Vecchio &amp; Ninfa, 2007</B><P>This paper approaches the issue of retroactivity in biological system keeping the engineering perspective. Trying to explain the topic authors go into great effort of making analogies to electrical or control engineering. It has some information on amplifying on the gene level rather than protein level (what is part of out project)!
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"> Nick
</TD></TR><TR><TD><B><I>Structure/function relationships in OmpR and other winged-helix transcription factors</I>. Kenny, L. J., 2002</B><P>When designing the fast response model, I wanted to find something that mediated a really fast output. I spent a lot of time researching OmpR, which is a response regulator in <I>E. coli</I>. However, we had to use <I>B. subtilis</I> as our chassis, which has a different membrane structure which meant that OmpR was no longer a candidate for this module.
</P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Imperial_College_London/Research">http://2010.igem.org/Team:Imperial_College_London/Research</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Research" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Research" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Research&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Research&amp;oldid=205176" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>