<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Parts_Favourites"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Parts/Favourites">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Parts/Favourites&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Parts/Favourites&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Parts/Favourites&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Parts/Favourites" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Parts/Favourites</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="title"><DIV id="belink"><A href="http://www3.imperial.ac.uk/bioengineering">Department of Bioengineering</A></DIV><DIV id="mblink"><A href="http://www3.imperial.ac.uk/molecularbiosciences">Division of Molecular Biosciences</A></DIV></DIV><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="plan">Plan</A></LI><LI><A href="#" class="listhead" id="results">Results</A></LI><LI><A href="#" class="listhead" id="extra">Extras</A></LI></UL></DIV><DIV id="headpic"><DIV id="headover"><P id="headtit">Parasight<SPAN class="dark"> <SPAN class="highlight">|</SPAN>  Parasite detection with a rapid response</SPAN></P><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London" class="listmain" id="results">Home</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Tour/Page_One" class="listmain" id="results">Tour</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modules" class="listmain" id="results">Modules</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Chassis" class="listmain" id="results">Chassis</A></LI><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Human_Practices" class="listmain" id="results">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Schistosoma" class="listmain" id="results">Schistosoma</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Research" class="listmain" id="results">Research</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Diary/Week_One" class="listmain" id="project">Diary</A></LI></UL></UL><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modelling" class="listmain" id="results">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Strategy" class="listmain" id="results">Assembly</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Lab_Diaries" class="listmain" id="results">Lab Diaries</A></LI><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Protocol" class="listmain" id="results">Lab Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Safety" class="listmain" id="results">Safety</A></LI></UL></UL><UL class="results"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Results" class="listmain" id="results">Experimental Results</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Parts/Favourites" class="listmain" id="results">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/The_Team" class="listmain" id="results">The Team</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Media/Videos" class="listmain" id="results">Media</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/School_Workshops" class="listmain" id="results">School Workshops</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="listmain" id="results">Software Tool</A></LI><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Acknowledgments" class="listmain" id="results">Acknowledgments</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Brainstorming" class="listmain" id="results">Brainstorming</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Glossary" class="listmain" id="results">Glossary</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Sitemap" class="listmain" id="results">Sitemap</A></LI></UL></UL></DIV></DIV></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:25px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Parts
</TD><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;" align="right"><STRONG class="selflink"> Favourites</STRONG> | <A href="/Team:Imperial_College_London/Parts" title="Team:Imperial College London/Parts"> Full List</A></TD></TR><TR><TD colspan="2"><B>We have contributed 23 parts to the Registry of Standard Biological Parts and we really hope that other people will find them useful in the future. Here you can find detailed information on our favourite parts, or the registry information on the complete parts list.</B></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:left;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">XylE - catechol 2,3-dioxygenase from P.putida with terminator
</TD></TR><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"><A href="http://partsregistry.org/Part:BBa_K316003" class="external text" rel="nofollow"><B>BBa_K316003</B></A></TD></TR><TR><TD>Catechol or catechol 2,3-dioxygenases (C2,3O) + O(2) is converted by a ring cleavage into 2-hydroxymuconate semialdehyde which is the cytotoxic and bright yellow-coloured product<CITE>1</CITE>. This is a key enzyme in many (soil) bacterial species used for the degradation of aromatic compounds. Catechol 2,3-dioxygenase<CITE>2</CITE> was originally isolated from Pseudomonas putida  and is active only as a homotetramer. Tetramerization, mediated via beta-sheets in the N-terminal regions of the monomers, allows formation of the active site which covalently binds a ferrous ion. The reaction itself takes place within seconds after the addition by Pasteur pipette or spraying of catechol at a 100mM stock solution diluted with DDH20 (used by our lab.) The toxic breakdown product is thought to interfere with cell wall integrity and cellular machinery such that exposed cells gradually die.
<P><B>Safety</B></P><P>Catechol is classed as irritant in the EU but as toxic in the USA, as well as being a possible carcinogen. It should therefore be handled with care and proper safety equipment. More information is available on the Material Safety Data Sheet<A href="http://www.sciencelab.com/msds.php?msdsId=9927131" class="external autonumber" rel="nofollow">[1]</A>.
</P><P><SPAN class="h3bb"><BIG><B>Sequence and Features</B></BIG></SPAN></P><P>&lt;partinfo&gt;BBa_K316003 SequenceAndFeatures&lt;/partinfo&gt;</P><P><SPAN class="h3bb"><BIG><B>Part Characterisation</B></BIG></SPAN></P></TD></TR><TR><TD><P><SPAN class="h3bb"><BIG><B>Optimum absorbtion wavelength for catechol assays</B></BIG></SPAN></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Peak absorbance of catechol breakdown product (2-hydroxymuconic semialdehyde</DIV></DIV></DIV><P>The enzymatic reaction catalysed by XylE can also serve as a powerful reporter. The substrate - catechol - is colourless. However within seconds of its addition, the colonies or suspended cultures of XylE-expressing cells become clearly yellow<CITE>3</CITE> to the naked eye. This reaction allows direct measurement of XylE activity by measuring product concentrations, which absorbs light in the visible spectrum, at 380nm. In contrast to other common reporters like GFP, LacZ or Luciferase which do easily express correctly in thermophile environments, a XylE assay has also been shown to be functional in here. The spectrophotometric assay compared the spectra of two cultures of E.coli (one XylE gene transformed and the other not) were compared on addition of 0.1mM Catechol substrate.
</P><P>A spectrophotometric assay of two cultures of E.coli (Blue: contains &lt;bbpart&gt;BBa_K316004&lt;/bbpart&gt;, Red: not expressing XylE ) were compared on addition of 0.1mM Catechol substrate. The spectra show that in XylE transformed cells, a broad peak appears at about 380nm. The absorbance at this particular wavelength is due to the yellow  product of the reaction (2-hydroxymuconic semialdehyde (HMS)).
</P></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><P><SPAN class="h3bb"><BIG><B><I>In Vitro</I> Assay</B></BIG></SPAN></P><P>Due to technical limitations, to measure kinetic parameters of XylE is to lyse cells and . In this experiment cell lysate was assayed with increasing catechol concentrations. The rate at which the yellow product appears is directly proportional to the velocity of the reaction. The rate reaction was monitored by measuring color output of the reaction in the plate reader.
</P><P>Cell lysate was tested for dioxygenase activity to determine appropriate dilutions for the assay. The cell lysate was obtained from a 100ml overnight culture and diluted by a factor of 20 to obtain a suitable concentration of total enzyme for the plate reader assay. The concentrations of catechol used were 1, 2, 5, 10, 25, 50 mM.
</P><P>Data collected was used to construct the Michaelis-Menten curve for the in vitro kinetics of XylE in cell lysate. 
Michaelis-Menten curve was drawn using velocity values calculated from the slope at the initial stages of the reaction, as this is the only time when substrate concentration values are accurate. The plot was delineated by non-linear regression analysis using GraFit software tool<A href="http://www.erithacus.com/grafit/" class="external autonumber" rel="nofollow">[2]</A>. The calculated Km is 0.71mM catechol (with a Vmax of 3.37 in O.D. arbitrary units for this dilution of cell lysate).
</P><P>For more detailed information, please check our Results section <A href="/Team:Imperial_College_London/Results/Exp6" title="Team:Imperial College London/Results/Exp6">Team:Imperial_College_London/Results/Exp6</A></P></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><P><SPAN class="h3bb"><BIG><B>Cytotoxicity of Catechol</B></BIG></SPAN></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">O.D. at 600 over 3h for XylE-transformed Top10 cells in presence of different catechol concentrations, growing in M9 medium.</DIV></DIV></DIV><P>The addition of catechol had distinctive deleterious effects on XylE expressing cells. While at 0% catechol growth-behavior did not show a significant change (dark blue), even the lowest concentration of 0.25% catechol appeared to drastically reduce cell-survival (red). In contrast, CMR-control cells did not change their growing behavior in the presence of catechol. From this we conclude that the breakdown product of catechol, 2-hydroxymuconic semialdehyde, and not Catechol itself, has strong cytotoxic effects
</P></TD></TR></TBODY></TABLE><P>Characterisation data was obtained using GFP-XylE constructs &lt;bbpart&gt;BBa_K316007&lt;/bbpart&gt; and XylE under two different promoters: <I>B. subtilis</I> derived Pveg &lt;bbpart&gt;BBa_K316005&lt;/bbpart&gt; and J23101 &lt;bbpart&gt;BBa_K316004&lt;/bbpart&gt; from <I>E. coli</I>. These are described on our wiki<A href="https://2010.igem.org/Team:Imperial_College_London/Results" class="external autonumber" rel="nofollow">[3]</A> and the aforementioned parts pages.
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/10368270" class="external text" rel="nofollow">Kita <I>et al.</I> 1999</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/12519074" class="external text" rel="nofollow">Okuta <I>et al.</I> 2003</A></P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/6405380" class="external text" rel="nofollow">Zukowski <I>et al.</I> 1983</A></P><TABLE style="width:900px;background:#f5f5f5;text-align:left;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">B. subtilis transformation vector with LacI, targets amyE locus
</TD></TR><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"><A href="http://partsregistry.org/Part:BBBa" class="external text" rel="nofollow">K316027 <B>BBBa K316027</B></A></TD></TR><TR><TD>This vector has been designed using the amyE 5' and 3' integration sequences for integration into <I>B.subtilis</I> genome
<P><B>AmyE locus</B>
This vector has been designed using the amyE 5' &lt;bbpart&gt;BBa_K143008&lt;/bbpart&gt; and 3 &lt;bbpart&gt;BBa_K143009&lt;/bbpart&gt;' integration sequences for integration into <I>B. subtilis</I> genome. Insertion into the amyE locus provides a selection marker as the bacterium will no longer be able to breakdown starch. An iodine assay can be used to confirm integration. This phenotype makes the transformed bacterium considerably less likely to survive in natural environments.
</P><P><B>LacI</B>
With constitutivly expressed LacI &lt;bbpart&gt;BBa_K143033&lt;/bbpart&gt; the vector is suitable for <I>B. subtilis</I> transformations with hyper-spank &lt;bbpart&gt;BBa_K143015&lt;/bbpart&gt; controlled gene-circuits. 
</P><P><B>Chloramphenicol Resistance</B>
This vector also contains a positive selection marker, flanked by two dif sites. Chloramphenicol acetyltransferase &lt;bbpart&gt;BBa_J31005&lt;/bbpart&gt; provides resistance to chloramphenicol antibiotic.
</P><P><B>Blunt end cloning site</B>
PmeI restriction site &lt;bbpart&gt;BBa_K316013&lt;/bbpart&gt; is designed as a cloning site. Due to the 8bp recognition sequence it is a rare site that can be used to cut the vector only once.
</P><P><B>Figure I.</B> Complete amyE vector for <I>B. subtilis</I> genome integration with LacI expression.
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:left;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Pveg-spoVG-LytC-Helical Linker-Elastase Cleavage Site-AIP-His Tag
</TD></TR><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"><A href="http://partsregistry.org/Part:BBa_K316037" class="external text" rel="nofollow"><B>BBa_K316037</B></A></TD></TR><TR><TD><B>Introduction:</B> This part, our detection module, was designed to anchor a signal peptide (AIP) to the cell wall of B. subtilis which is released upon cleave of the part by a protease, in this case the cercarial elastase <A href="http://www.uniprot.org/uniprot/P12546" class="external text" rel="nofollow"><B>(UniProt,</B></A><A href="http://merops.sanger.ac.uk/cgi-bin/pepsum?id=S01.144" class="external text" rel="nofollow"><B>MEROPS)</B></A>, to activate our output. The part is fully assembled and ready to be expressed in a prokaryotic cell as it contains a promoter we characterised, an optimised RBS and a double stop codon. The protein product includes a cell wall binding domain (CWB) of LytC, <A href="http://partsregistry.org/Part:BBa_K316030" class="external text" rel="nofollow">BBa_K316030</A> a linker that has been designed to carry a protease cleave site and the AIP <A href="http://partsregistry.org/Part:BBa_K316047" class="external text" rel="nofollow">BBa_K316047</A>. This part is essential for the great modularity of our system, as a change of just 4 amino acids can be used to confer specificity to another protease see our <A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="external text" rel="nofollow"><B>Software Tool</B></A>.
<P><B>Pveg-spoVG:</B> This was build in 2008 by the Imperial College iGEM team <A href="http://partsregistry.org/Part:BBa_K143053" class="external text" rel="nofollow">BBa_K143053</A>.
</P><P><B>LytC:</B> The CWB is on the 5’ end and anchors the detection module to the cell wall of B. subtilis. The CWB was separately submitted as biobrick <A href="http://partsregistry.org/Part:BBa_K316030" class="external text" rel="nofollow">BBa_K316030</A>.
</P><P><B> Helical Linker:</B> The Linker separates the CWB and the AIP and creates space for the protease to access the cleavage site; it consists of two main sections. The first six amino acids (SRGSRA) were suggested to be used specifically with LytC <A href="http://www.ncbi.nlm.nih.gov/pubmed/14594841" class="external text" rel="nofollow">Yamamoto <I>et al.</I> 2003</A>. The second section consists of a helical amino acid sequence that is stiff and prevents interaction of the AIP with its receptor and thus false positive activation of the output system. It was specifically designed to separate protein domains <A href="http://www.ncbi.nlm.nih.gov/pubmed/11579220" class="external text" rel="nofollow">Arai <I>et al.</I> 2001</A>.
</P><P><B> Elastase Cleavage Site:</B> This sequence forms the 3’ end of the linker and is directly attached to the 5’ end of the AIP. It is four amino acids (SWPL) long and was designed to be efficiently cleaved by the schistosoma cercarial elastase <A href="http://www.jbc.org/content/277/27/24618.abstract" class="external text" rel="nofollow">Salter <I>et al.</I> 2002</A>.
</P><P><B>AIP:</B> The linear auto-inducing peptide (AIP) from the Streptococcus pneumonia ComCDE system, called Competence-Stimulating Peptide-1 (CSP-1 or ComC) . Upon cleavage it is free to diffuse and bind it's receptor ComD <A href="http://partsregistry.org/Part:BBa_K316015" class="external text" rel="nofollow">BBa_K316015</A>. Signalling cascade from ComD is then able to activate transcription of genes under specific promoter sequences.
</P><P><B>His-Tag:</B> To be able to purify the protein for testing, we attached a His-Tag on our linker-AIP peptide. As it would probably interfere with recognition of the AIP by the receptor it has to be removed from the final construct. 
</P><P><B>Stop Codon:</B> In order to end translation a double stop codon was put in place.
</P><P><B>Diagramatic representation of surface protein.</B></P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Imperial_College_London/Parts/Favourites">http://2010.igem.org/Team:Imperial_College_London/Parts/Favourites</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Parts/Favourites" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Parts/Favourites" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Parts/Favourites&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Parts/Favourites&amp;oldid=203971" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>