<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Modules_Detection"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Modules/Detection">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Modules/Detection&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Modules/Detection&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Modules/Detection&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Modules/Detection" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Modules/Detection</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="title"><DIV id="belink"><A href="http://www3.imperial.ac.uk/bioengineering">Department of Bioengineering</A></DIV><DIV id="mblink"><A href="http://www3.imperial.ac.uk/molecularbiosciences">Division of Molecular Biosciences</A></DIV></DIV><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="plan">Plan</A></LI><LI><A href="#" class="listhead" id="results">Results</A></LI><LI><A href="#" class="listhead" id="extra">Extras</A></LI></UL></DIV><DIV id="headpic"><DIV id="headover"><P id="headtit">Parasight<SPAN class="dark"> <SPAN class="highlight">|</SPAN>  Parasite detection with a rapid response</SPAN></P><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London" class="listmain" id="results">Home</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Tour/Page_One" class="listmain" id="results">Tour</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modules" class="listmain" id="results">Modules</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Chassis" class="listmain" id="results">Chassis</A></LI><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Human_Practices" class="listmain" id="results">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Schistosoma" class="listmain" id="results">Schistosoma</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Research" class="listmain" id="results">Research</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Diary/Week_One" class="listmain" id="project">Diary</A></LI></UL></UL><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modelling" class="listmain" id="results">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Strategy" class="listmain" id="results">Assembly</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Lab_Diaries" class="listmain" id="results">Lab Diaries</A></LI><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Protocol" class="listmain" id="results">Lab Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Safety" class="listmain" id="results">Safety</A></LI></UL></UL><UL class="results"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Results" class="listmain" id="results">Experimental Results</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Parts/Favourites" class="listmain" id="results">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/The_Team" class="listmain" id="results">The Team</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Media/Videos" class="listmain" id="results">Media</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/School_Workshops" class="listmain" id="results">School Workshops</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="listmain" id="results">Software Tool</A></LI><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Acknowledgments" class="listmain" id="results">Acknowledgments</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Brainstorming" class="listmain" id="results">Brainstorming</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Glossary" class="listmain" id="results">Glossary</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Sitemap" class="listmain" id="results">Sitemap</A></LI></UL></UL></DIV></DIV></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:25px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Modules
</TD><TD style="font-family: helvetica, arial, sans-serif;font-size:1.5em;color:#ea8828;" align="right"><A href="/Team:Imperial_College_London/Modules" title="Team:Imperial College London/Modules">Overview</A> | <STRONG class="selflink">Detection</STRONG> | <A href="/Team:Imperial_College_London/Modules/Signaling" title="Team:Imperial College London/Modules/Signaling">Signaling</A> | <A href="/Team:Imperial_College_London/Modules/Fast_Response" title="Team:Imperial College London/Modules/Fast Response">Fast Response</A></TD></TR><TR><TD colspan="2"><B>Our design consists of three modules; Detection, Signaling and a Fast Response, each of which can be exchanged with other systems. We used a combination of modelling and human practices to define our specifications. Take a look at the overview page to get a feel for the outline, then head to the full module pages to find out how we did it.</B></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Detection Module
</TD></TR><TR><TD> Having discussed with experts if <I>Schistosoma</I> cercaria would be the most useful lifecycle stage of <I>Schistosoma</I> to detect, we developed a surface protein that allows detection of <I>Schistosoma</I> in water. The key motivation for targeting the parasite larvae, called cercaria, is prevention of the disease altogether. The cercaria are often found in water, but their location as well as time of their appearance is unpredictable, putting the population in endemic areas at risk of infection. By providing a fast and reliable test, non-profit organisations  would be able to regularly check the water for the presence of cercaria and warn people of the danger. Furthermore we are exploiting an essential aspect of the cercarias’ biochemistry without which they are unable to progress in their life cycle. To learn more about the parasite life cycle follow this link to our section on <A href="https://2010.igem.org/Team:Imperial_College_London/Schistosoma" class="external text" rel="nofollow"><B><I>Schistosoma</I></B></A>.
</TD></TR><TR><TD>Overview of the detection module.
</TD></TR></TBODY></TABLE></DIV><P><B>Design</B></P><P>We make use of a cercarial protease, called elastase <A href="http://www.uniprot.org/uniprot/P12546" class="external text" rel="nofollow"><B>(UniProt,</B></A><A href="http://merops.sanger.ac.uk/cgi-bin/pepsum?id=S01.144" class="external text" rel="nofollow"><B>MEROPS)</B></A>, which is necessary for cercaria to penetrate human. The surface our <I>B. subtilis</I> is coated with a surface protein which is design to release a sequestered quorum sensing signal upon cleavage by the elastase.
</P><P><B>The AIP activates signal transduction</B></P><P>We used the autoinducing peptide (AIP) from the ComCDE system of <I>S. Pneumoniae</I> because it is linear and lacks posttranslational modifications. Using a system foreign to <I>B. subtilis</I> also reduces false activation and the noise our system has to deal with, making it more robust. Because of the nature of the AIP receptor ComD, in order to activate signal transduction the AIP must be in the cleaved form, i.e. not attached to the surface protein anymore. This adds an additional level of robustness to the system.
</P><DIV align="CENTER"><TABLE style="width:354px;background:#e7e7e7;text-align:center;font-family: helvetica, arial, sans-serif;color:#555555;margin- top:5px;padding: 2px;"><TBODY><TR><TD>The detection module consists of three section: Cell Wall anchor, Linker and AIP
</TD></TR></TBODY></TABLE></DIV><P><B>The Linker is a substrate for the cercarial elastase</B></P><P>The AIP is connected to a cell wall anchor via a linker. It is this linker that confers specificity to our surface protein which is to be cleaved by the elastase. The elastase itself recognises a four amino acid sequence, which we included in our linkers. In order to test the optimal design for the linker, we created 6 different linkers which vary in length and flexibility. Furthermore we created a do-it-yourself <A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="external text" rel="nofollow"><B>Software Tool</B></A> which enables the custom design of the surface protein sequence to include a protease cleavage site of choice. This allows the detection of any protease by incorporating such new surface proteins into our system.
</P><P><B>Detection of Parasite proteases</B></P><P>In order to detect <I>Schistosoma</I> in water they have to release their elastase. This does not usually occur in the absence of a stimulus however it is easy to trigger this behaviour <I>in vivo</I>. Upon detection of human (or mouse) skin lipids and temperatures close to 37°C, the invasive behaviour is triggered. Several proteases are released from pre- and postacetabular glands or the cercaria at the leading edge of the invading parasite. While multiple enzymes are released, only one protease activity was found to be necessary for invasion in <I>S. mansoni</I>, and <I>S. haematobium</I><A href="http://books.google.co.uk/books?id=JL7qs-O3z1MC&amp;pg=PA299&amp;lpg=PA299&amp;dq=advances+in+parasitology+london&amp;source=bl&amp;ots=b8mvkz6zyC&amp;sig=t7Pwy0egEDQ9dW-GIZEK7Pb6xxs&amp;hl=en&amp;ei=n05ATKLnJsqNjAePyOAG&amp;sa=X&amp;oi=book_result&amp;ct=result&amp;resnum=3&amp;ved=0CCEQ6AEwAg#v=onepage&amp;q=advances%20in%20parasitology%20london&amp;f=false" class="external text" rel="nofollow">(Kasny <I>et al.</I> 2009)</A>: Schistosoma elastase 2a and b (SmCE-1a/b) <A href="http://www.jbc.org/content/277/27/24618.abstract" class="external text" rel="nofollow">(Salter <I>et al.</I> 2002)</A>. SmCE are trypsin family serine proteases the specificity of which has been investigated by various studies. It appears that the following cleavage sites are preferred: 
</P><UL><LI> P4 - S,T 
</LI><LI> P3 - S,W,Y 
</LI><LI> P2 - P 
</LI><LI> P1 - L 
</LI></UL><P>Subtle differences exist between SmCE-2a and b as far as their P4 and P3 sites are concerned. Cleavage kinetics were determined for four different sites, P4-P1: SWPL, TWPL, RWPL, RRPL with R previously determined as unfavourable at P4 and P3. For P4 an 11-fold difference in activity was determined between favourable S/T and R, while a 3-fold difference was determined for P3 W to R <A href="http://www.jbc.org/content/277/27/24618.abstract" class="external text" rel="nofollow">(Salter <I>et al.</I> 2002)</A>. The most favourable sequence would therefore be SWPL.
</P><P><TABLE width="300px" border="0" align="center"><TBODY><TR><TD style="background-color:#FFCC66;height:50px;width:50;text-align:center"><B>Uniprot accession</B></TD><TD style="background-color:#FFCC66;height:50px;width:50;text-align:center"><B>MEROPS ID</B></TD><TD style="background-color:#FFCC66;height:50px;width:50;text-align:center"><B>Clan</B></TD><TD style="background-color:#FFCC66;height:50px;width:50;text-align:center"><B>Family</B></TD><TD style="background-color:#FFCC66;height:50px;width:50;text-align:center"><B>pH optimum</B></TD><TD style="background-color:#FFCC66;height:50px;width:50;text-align:center"><B>Molecular weight (practical/theoretical)</B></TD></TR><TR><TD style="background-color:#eeeeee;height:50px;width:50 px;text-align:center;">P12546</TD><TD style="background-color:#eeeeee;height:50px;width:50 px;text-align:center;">S01.144</TD><TD style="background-color:#eeeeee;height:50px;width:50 px;text-align:center;">PA(S)</TD><TD style="background-color:#eeeeee;height:50px;width:50 px;text-align:center;">S1</TD><TD style="background-color:#eeeeee;height:50px;width:50 px;text-align:center;">4-10.5</TD><TD style="background-color:#eeeeee;height:50px;width:50 px;text-align:center;">25/29 kDA</TD></TR></TBODY></TABLE></P><B>The cell wall binding domain anchors our module to the cell surface</B><P>The AIP-linker peptide is  anchored to the cell wall by LytC, a protein native to <I>B. subtilis'. The process of anchoring the recombinant protein to the cell wall helps to increase the efficiency of detection. It uses electrostatic interactions of several cell wall binding domain repeats to non-covalently bind to the cell wall, and expose our detection apparatus to the elastase. We chose it over other options, such as covalently bound proteins that use the Gram positive sortase system, membrane bound proteins or other non-covalently bound cell wall proteins for several reasons. Firstly, we wanted to minimise the chance of false activation of signal transduction, so we eliminated cell membrane bound proteins, as these would bring the AIP in much greater proximity to the receptor ComD. Whilst the sortase system is well understood for some bacteria such as </I>Staphylococcus aureus<I>, it has not been extensively studies in </I>B. subtilis<I> and differences in the peptidoglycan structure meant it would have been much more error prone to use this system in our chassis. Thus the most viable option were non-covalently bound surface proteins, and LytC the best candidate within this group as a result of several studies, most notably by Kobayashi et al. (2003), that had previously used it to anchor catalytic domains, as well as short peptides, to the cell wall of </I>B. subtilis<I>.</I></P><P><A href="http://www3.interscience.wiley.com/journal/119078114/abstract?CRETRY=1&amp;SRETRY=0" class="external text" rel="nofollow">Tsuchiya et al. 1999</A></P><P><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6VSD-426YS50-D&amp;_user=217827&amp;_coverDate=12%2F31%2F2000&amp;_rdoc=1&amp;_fmt=high&amp;_orig=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_acct=C000011279&amp;_version=1&amp;_urlVersion=0&amp;_userid=217827&amp;md5=4764a7bdabc3649925f082f989e835f5" class="external text" rel="nofollow">Kobayashi et al. 2000</A></P><P><A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6VSD-46G3T2V-4&amp;_user=217827&amp;_coverDate=01%2F31%2F2002&amp;_rdoc=1&amp;_fmt=high&amp;_orig=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_acct=C000011279&amp;_version=1&amp;_urlVersion=0&amp;_userid=217827&amp;md5=c807c9de3e3579777b08fdd77cd32294" class="external text" rel="nofollow">Kobayashi et al. 2002</A></P><P><A href="http://jb.asm.org/cgi/reprint/185/22/6666" class="external text" rel="nofollow">Yamamoto et al. 2003</A></P><DIV align="CENTER"><TABLE style="width:254px;background:#e7e7e7;text-align:center;font-family: helvetica, arial, sans-serif;color:#555555;margin- top:5px;padding: 2px;"><TBODY><TR><TD>Crystal structure the protein LytC used to anchor our peptide to the cell wall. (EBI, 2010)
</TD></TR></TBODY></TABLE></DIV><P>Additionally, since we do not intend to knock out the original gene from the <I>B. subtilis</I> strain we will not interfere with its normal expression, avoiding expression at the wrong time or over expression. Furthermore since we aim to remove the catalytic domain from LytC or CwlC, our fusion proteins should lose their cell wall turn-over functions and only act as cell wall anchors.
</P><P><B>Summary and Testing</B></P><P>Before finalising our system we modelled the requirements for our detection module to analyse its functionality. For more information on this follow this link to our <A href="https://2010.igem.org/Team:Imperial_College_London/Modelling/Protein_Display/Objectives" class="external text" rel="nofollow"><B>Modelling</B></A>. It showed that we could reach the threshold concentration of AIP needed to activate our system, 10ng/ml, quite easily, with the amount of surface protein we expect to express, as well as estimating the time needed to set off our system at a given concentration of elastase in the water. 
The assembly of the protein was done in several steps. We first amplified LytC out of the <I>B. subtilis</I> genome by PCR, simultaneously using primer extension to add optimised ribosome binding sites to it, and insert the PCR product into pSB1C3. We later ligated the protein with the promoter Pveg. Parallel to this, we ligated the 6 linker sequences ordered from Eurofins|mwg with a double terminator BOO14. We then made use of a natural ACCI site in the LytC gene, to ligate our promoter-LytC sequence with our linker-terminator sequence to create 6 different surface protein constructs, for testing in <I>B. subtilis</I>.
</P><DIV align="CENTER"><TABLE style="width:354px;background:#e7e7e7;text-align:center;font-family: helvetica, arial, sans-serif;color:#555555;margin- top:5px;padding: 2px;"><TBODY><TR><TD>The detection module is non-covalently bound to the cell wall that overlies the phospholipid bilayer
</TD></TR></TBODY></TABLE></DIV><P>In order to be able to detect out protein we will use His-tags. However these are only been added to test constructs and not the final module, as the tag would most likely interfere with the module’s function. The his-tags have been added to the C-terminus of the AIP, probably inhibiting correct detection of the AIP by the two component system comD/E. However using a his-tag allows us to test for three different, very important things: We can lyse a bacterial culture to isolate our his-tagged protein from the lysate. Detection in this step would demonstrate successful expression of our fusion protein. Secondly we can place our bacteria in a solution with high osmolarity. As mentioned this eludes the non-covalently bound proteins from the cell wall by interfering with the electrostatic interactions of the cell wall binding domain with the peptidoglycans, which should also work for our fusion proteins. Detection of our protein in this step would demonstrate expression as well as correct localisation in the cell wall. Last of all we could place the cells in medium containing the elastase or a protease with the same or very similarly specificity. If we are able to pull down the his-tagged peptide then it would demonstrate expression, localisation and that the correct cleavage site is accessible to the protease and the AIP can be successfully cleaved off. For a complete list of the BioBrick created for this module see our page on <A href="https://2010.igem.org/Team:Imperial_College_London/Parts" class="external text" rel="nofollow"><B>Parts</B></A> submitted to the registry.
</P><P>
Here's a picture of the final construct:
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Imperial_College_London/Modules/Detection">http://2010.igem.org/Team:Imperial_College_London/Modules/Detection</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Modules/Detection" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Modules/Detection" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Modules/Detection&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Modules/Detection&amp;oldid=203504" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>