<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Brainstorming"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Brainstorming">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Brainstorming&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Brainstorming&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Brainstorming&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Brainstorming" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Brainstorming</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="title"><DIV id="belink"><A href="http://www3.imperial.ac.uk/bioengineering">Department of Bioengineering</A></DIV><DIV id="mblink"><A href="http://www3.imperial.ac.uk/molecularbiosciences">Division of Molecular Biosciences</A></DIV></DIV><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="plan">Plan</A></LI><LI><A href="#" class="listhead" id="results">Results</A></LI><LI><A href="#" class="listhead" id="extra">Extras</A></LI></UL></DIV><DIV id="headpic"><DIV id="headover"><P id="headtit">Parasight<SPAN class="dark"> <SPAN class="highlight">|</SPAN>  Parasite detection with a rapid response</SPAN></P><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London" class="listmain" id="results">Home</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Tour/Page_One" class="listmain" id="results">Tour</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modules" class="listmain" id="results">Modules</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Chassis" class="listmain" id="results">Chassis</A></LI><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Human_Practices" class="listmain" id="results">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Schistosoma" class="listmain" id="results">Schistosoma</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Research" class="listmain" id="results">Research</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Diary/Week_One" class="listmain" id="project">Diary</A></LI></UL></UL><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modelling" class="listmain" id="results">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Strategy" class="listmain" id="results">Assembly</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Lab_Diaries" class="listmain" id="results">Lab Diaries</A></LI><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Protocol" class="listmain" id="results">Lab Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Safety" class="listmain" id="results">Safety</A></LI></UL></UL><UL class="results"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Results" class="listmain" id="results">Experimental Results</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Parts/Favourites" class="listmain" id="results">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/The_Team" class="listmain" id="results">The Team</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Media/Videos" class="listmain" id="results">Media</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/School_Workshops" class="listmain" id="results">School Workshops</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="listmain" id="results">Software Tool</A></LI><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Acknowledgments" class="listmain" id="results">Acknowledgments</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Brainstorming" class="listmain" id="results">Brainstorming</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Glossary" class="listmain" id="results">Glossary</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Sitemap" class="listmain" id="results">Sitemap</A></LI></UL></UL></DIV></DIV></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:25px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Brainstorming
</TD></TR><TR><TD><B>On this page is an overview of ideas that our team collected and did research on before we decided on the finalised project. You can see how some of our ideas were dropped altogether, whereas others set the theme for the Parasight project: speed of response, water sanitation, and detection of parasites.</B></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">The ideas
</TD></TR><TR><TD><B>Water Sanitation</B><P>Quite early in the project the idea of water sanitation was put forward. We initially collected ideas about different substances, such as heavy metals, and pathogens we might be able to detect using relatively simple biosensors. In this list different parasites and bacteria were included.
</P><P><B>Pathogen detection via quorum sensing</B></P><P>We later considered detecting the quorum sensing molecule N-acyl homoserine lactone produced by V. cholera for example. This molecule could than activate a simple colour output in our bacteria and thus exploit pathogen functions for detection. We later dropped this idea because, although it has a powerful application, it was too similar in design to biosensors that had been designed previously in iGEM projects. There were also a number of additional problems with this idea that made us not pursue it further. 
</P><P><B>Detection via receptors for bacteria-parasite interactions</B></P><P>A step further towards our final project was the idea to detect water borne parasites. Initially we did much research on parasite-bacteria interactions in order to find a sound mechanism we could implement in our system and link to an output. Even though we found some bacteria that interact with parasites, such as Salmonella with Schistosoma in the human gut, these we more often than not non-specific interactions and thus could give rise to false positive activations of our system. 
</P><P><B>Population control</B></P><P>This concept aimed to keep the total number of bacteria in equilibrium below the number they would naturally achieve, making the population number oscillate around this arbitrary threshold. This might be done by some clever use of quorum sensing systems acting on survival, growth or resistance genes to influence the performance of bacteria depending on the population density. However this construct would be evolutionarily unstable, as bacteria that lost the ability to self regulate growth would inevitably proliferate and out compete the functional bacteria. 
</P><P><B>Safe biofilms</B></P><P>We came up with similar approaches to making biofilms safer, for example by making the bacteria interdependent. This might be done by promoting cell survival via quorum sensing systems, so bacteria that become detached from the biofilm would die shortly after. This might make them safer for the environment, limiting their spread, as well as for use in medicine, preventing tissue invasion.
</P><P><B>Cell state reporters</B></P><P>This idea was to monitor cell activity which might be useful for scientists in many situations. We proposed the use of cell state reporters to quickly give visual information of the activities or phase the cells are undergoing. This might help to maintain a healthy population and to determine whether population grows anaerobically or aerobically. Furthermore it could be used as a warning signal when cells undergo undesired/stressful phases and to examine if the cells grow in synchrony. Many different regulon systems could potentially exploited for this purpose including without limitation: 
</P><UL><LI> Ntr and Nif system that detect ammonia limitation
</LI><LI> Arc and Fnr system that detect anaerobic conditions
</LI><LI> The SOS and response to UV light
</LI><LI> Sporulation in response to starvation and the heat shock response
</LI></UL><P>Use of system specific promoter, enhancer or repressor elements could be used to create a system that gives feed back in the form of different fluorescent proteins to the degree to which the system is activated and by extension about the environment the cell is in. 
</P><P><B>Biofuels</B></P><P>We spent some time thinking about new ways in which bacteria could be used to produce biofuels, as well as trying to come up with more substances that could be used as biofuels, rather than relying on petrol, ethanol and simple sugars too much. Initial ideas included without limitation the use of octopine or nopaline from the T-DNA of Argobacterium. However ultimately we did not find this an exciting enough project to pursue.
</P><P><B>TV screen</B></P><P>We thought about enhancing the iGEM project of the Valencia team 2008 who built a black and white bioscreen using yeast cells with voltage gated ion channels to produce a colour change. We considered implementing colour in this system to create a simple bio-TV. However the project would have been too similar to the original project so we dropped it quite early through our project development process.
</P><P><B>Rhyzobium bacteria</B></P><P>We considered engineering Rhyzobium bacteria to fix nitrogen in larger quantities so they could be used more easily as fertilizer. Furthermore, but almost more importantly, we wanted to manipulate them to interact with a greater number of plant species, as to enhance the range of crops that could benefit from our project. However this would have been a very complex process that would probably not have fit into the time and labour constraints of an iGEM project.
</P><P><B>Logic gates</B></P><P>Extending the logic gate concept developed in previous iGEM competitions, we considered whether a cell could have its logical state altered by an external input, for example frequency or intensity of light. This would in effect create a biological multiplexer with every input being a different logic gate, and the multiplexer stage being defined by a third input.
</P><P><B>Rapid response system</B></P><P>Another idea that continued to keep our attention was the concept of a fast response. Most products of synthetic biology require a long time for gene expression until an output such as a pigment can be seen. We considered many different approaches, basically all involving an input to stimulate a two component system. The signal transducer was to activate a previously inactive enzyme and act on a substrate. The substrate was either to be colourless and only become visible after the enzyme acted on it, or be concentrated, such as a tight chain of pigments, that are than cleaved apart to colour the whole cell.  We later settled for a more simplified, yet still highly efficient variant of this system to make our sensor more robust.
</P><P><B>Fv-fragment guided input module</B></P><P>In order to detect molecules for which no natural receptor exists, we considered using antibody based receptors, either fused to the signal transduction portions of bacterial receptors, or by introducing the whole Fc-binding receptor of the mammalian immune system into bacteria and combining it with whole antibodies. However antibodies are complex, glycosylated proteins that cannot be expressed by bacteria easily and supplying antibodies to each detection kit would be very expensive. Furthermore introducing genes essential to the human immune system into bacteria, such as E. coli was considered a very risky idea and the human practices side of our project would be weakened.
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">The Story behind the fast response module..
</TD></TR><TR><TD><P>An initial idea during the brainstorming sessions was to engineer a system which responded to an [unidentified] input and resulted in an instantaneous colour response. This was hoped to be in the form in pigment production. We envisioned explosions of pigments similar to fireworks and that we could combine differently coloured pigment explosions to manipulate the overall colour output. Needless to say this idea was inspired by 2009 iGEM champions Cambridge and their rainbow of pigment production Biobricks. Our spin on the idea was to produce a lightning fast response- as speed of pigment production generally isn’t so immediate. Adapting this idea we became fixated with the scenario of capturing a colourless intermediate (by tethering it to a protein scaffold) which upon activation of an enzyme was process in one step to a coloured product. It became quickly apparent that in order to achieve such a fast response as we desired we were going to need to bypass the traditional steps of transcription and translation. These were believe to be the most time consuming steps in our system; synthesis of the enzyme. We went through a spell of trying to think of a way around these steps focusing on utilizing protein engineering.
</P><P>A major breakthrough was when we stumbled across the idea of split proteases . In this scenario we would have a pool of pre-made enzyme present in the chassis which would be activated by the transcription and translation of a very small protease (TEV). This would act as an amplification step and maximise our output. We followed up on this idea and contacted some of the main names behind the papers, we were fortunate to build up a correspondence with Prof Neel Ghosh in particular who assured us that the kinetics of his split coiled coil protease system were adequate for our system. Whilst the mechanism of fast activation of an enzyme seemed to be taking off we needed to define the source of the coloured output. 
</P><P>We toyed with all sorts of possibilities; phycobilins, fluorescent proteins, but kept encountering the same problems; either the pigment was in such a form that it couldn’t easily be modified (by us e.g. tethering it to a scaffold or rendering it colourless by mutation) or the glow response was too slow. When we discovered GFP had been split we were quite confident this would be a suitable output. One main criteria for our system was a stark colourless to coloured change. It was gradually deduced that using split GFP would in fact generate too much background fluorescence such that the change would be hard to detect. This is due to intrinsic and inherent affinity that the split GFP segments have for each other- they would be in an equilibrium of intact and separate parts, which means we couldn’t use them in our system as they would fluoresce even without input stimulation. It should be noted that we considered using Beta galactosidase also. This is naturally transcribed and translated in two halves (genes present on two different chromosomes) which means it’s easy to tag each half, but again the  halves have such a high affinity for each other that false positives  are a big problem; an equilibrium between monomer and dimer states always exists.
</P><P>A host of other possibilities were investigated; Camgaroos and even smell molecules but either the input stimulus wasn’t desirable or the substrate molecules were hard to work with. It was by luck that we happened across our final gene. A supervisor had been to a presentation of SynBio masters students work in the last year and recalled one students work in particular. Jeremy Bartosiak-Jentys’s work revolved around XylE-catechol assaying. The key points include a response that occurs in seconds to minutes with a colourless to coloured (bright yellow) pigment production. We researched into this enzyme and even visited the mentioned masters student who gave use some cells and we watched the assay ourselves. The response was in seconds and you couldn’t miss it; it was such a bright yellow. Suddenly our fast response output problem was solved, we were going to use XylE, and even better; it was in the registry.
The next challenge was how to couple activation of C230 with our chosen input. After much research and aided by Jeremy it was known that C230 acted as a tetramer in its functional form. We paid visits to protein engineers and alongside structural research into the monomer decided to modify a monomer by N-terminally linking (GGGSGGGS-TEVcleavage site) it to GFP. This re-engineered monomer was predicted to be incapable of tetramerization and thus inactive. Only when TEV cleaves off GFP would it be able to regain functionality. Thus our system for fast response had been founded. TEV was one of the many considered proteases chosen on a basis of size and specificity; we didn’t want the protease to chop up our key enzyme or for the synthesis step to hinder our timescale of response. TEV is very small and its synthesis timescale is negligible therefore despite not being able to entirely bypass the transcription translation step (easily) we have found a way around it and still meet our system requirements.
</P><H2><SPAN class="mw-headline" id="Key_Papers">Key Papers</SPAN></H2><UL><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/17685552" class="external text" rel="nofollow">Split beta-lactamase sensor for the sequence-specific detection of DNA methylation</A></LI><LI><A href="http://www.hindawi.com/journals/jbb/2009/591923.html" class="external text" rel="nofollow">Speedy TEV protease</A></LI><LI><A href="http://pubs.acs.org/doi/pdf/10.1021/ja9050857" class="external text" rel="nofollow">Coiled coil constructs!</A></LI></UL></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Imperial_College_London/Brainstorming">http://2010.igem.org/Team:Imperial_College_London/Brainstorming</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Brainstorming" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Brainstorming" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Brainstorming&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Brainstorming&amp;oldid=170055" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>