"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/Protocol
From 2010.igem.org
Department of Bioengineering
Division of Molecular Biosciences
Project
Plan
Results
Extras
Parasight  |  Parasite detection with a rapid response
Home
Tour
Modules
Chassis
Human Practices
Schistosoma
Research
Diary
Modelling
Assembly
Lab Diaries
Lab Protocols
Safety
Experimental Results
Parts
Achievements
The Team
Media
School Workshops
Software Tool
Acknowledgments
Brainstorming
Glossary
Sitemap
Lab protocols
Listed below are the protocols we used for the project. We hope you find them useful!
Restriction Digests
Method:
Determine the concentration of the DNA sample by running both the vector and insert on a 1% agarose gel and comparing the bands intensity with the ladder (concentration known).
Calculate how much solution is needed to obtain desired total amount of DNA for digestion.
The volume of DNA solution can be no more than 70% of the total solution. Therefore calculate the total volume of digestion (probably around 20µl or 30µl).
Transfer the DNA, BSA, the appropriate buffer and ddH2O into a microcentrifuge tube. Finally, add the enzymes to the solution. N.B. The enzymes should be kept on ice before being added to the digestion.
Incubate for 60-90min at 37°C. Put in the freezer or on ice immediately after to stop further digestion. Especially important for EcoRI and other enzymes with star activity.
Use gel electrophoresis to confirm correct digestion.
Gel purification can be used to obtain the desired digestion product from the gel.
Reaction mixtures:
Required components
Example
1/20 Enzyme 1
1.5µl EcoRI
1/20 Enzyme 2
1.5µl PstI
1/10 BSA
3µl BSA
1/10 Buffer* (x 10)
3µl Buffer 4 (x 10)
X/10 DNA solution
20µl DNA (pSB1C3)
7-X/10 ddH2O
1µl ddH2O
Total: 10/10
Total: 30µl
Or you can use this as a guide:
20µl reaction volume unless digesting large amounts of DNA (use 30µl)
4µl DNA (if from Midi-preps, use 8µl Mini-Prep DNA)
2µl Buffer * (1 in 10µl total volume)
2µl 10xBSA (1 in 10µl total volume)
1µl Enzyme 1 (use 1.5µl for 30µl digests)
1µl Enzyme 2 (most Bio-Brick REF assembly protocols require a second enzyme) (use 1.5µl for 30µl digests)
The buffer depends on the restriction enzymes used.
Prefix Insertion:
Enzymes used:
Required buffer:
Prefix (insert)
EcoRI & SpeI
Buffer 2
Suffix (vector)
EcoRI & XbaI
EcoRI buffer
Suffix insertion:
Enzymes used:
Required buffer:
Prefix (vector)
SpeI & PstI
Buffer 2
Suffix (insert)
XbaI & PstI
Buffer 3
SpeI doesn’t cut particularly at the end of PCR products particularly well as there are few flanking bases. Can leave overnight and add the second enzyme as a second 90 minute cutting step.
Ligations
A typical ligation reaction mixture is around 10 μl and contains
1 μl DNA T4 ligase
1 μl DNA T4 ligase buffer (check to ensure it contains ATP) (10x)
Purified, linearised vector*
Purified, linearised insert*
ddH2O
There should be a ratio of 6:1 for moles of insert to vector. This can be calculated using the following equation:
Insert mass (ng) = 6 x (Insert length (bp)/vector length (bp) x Vector mass (ng) Once the solution is made up, the tubes are vortexed and then spun down for around 10 seconds in a microcentrifuge. The ligation is done at 14°C in a water bath in the cold cabinet, and is left overnight.
E. coli Transformations
One 15ml tube for each sample, in addition to one for a negative control, is put on ice.
Tubes containing 1ml LB were incubated in a water bath is set to 42°C .
Between 25µl and 40µl of competent cells is transferred to each tube.
The cells are left on ice for 10min.
5µl of the DNA sample are transferred into each tube, but ddH2O is added to the control tube(s). The liquids are added directly into the cell culture.
N.B. During pipetting the sides of the tube should not be touched to avoid contamination. Bubbles should be avoided because they can cause the cells stress.
The tubes are transferred into the 42°C water bath for exactly 45 seconds and then put on ice for 2 minutes. Timing must be exact.
The tubes are put on a rack and 1ml LB is added to all of them. This levels the temperature of the solution at about 37°C.
The tubes are then put into a shaking incubator at 37°C for 1 hour.
The solution from the 15ml tubes is then transferred to a microcentrifuge tube and spun at 13500 rpm for a few seconds.
The supernatant is discarded and the remaining LB is mixed with the pelleted cells. This increases the concentration of the cells in the LB.
50 – 100μl of this solution is then pipetted onto chloraphenicol plates and left overnight at 37°C. The next day colony PCR can be used to examine if the transformation was successful.
PCR
PCR Reaction Mix
25µl Total Reaction Volume
18.75µl ddH2O
2.5µl Buffer (Barns for any enzyme or  Taq, Pfu buffer depending on enzyme used)
1µl Forward primer
1µl Reverse primer
1µl Template
0.5µl dNTPs
0.25µl Enzyme (Taq, Pfu etc)
PCR programme
Heated lid - 110°C
35 cycles
95°C for 1.5 mins - Denature the template
--°C anneal primers
t°C optimal / time optimal to extend (depends on enzyme used)
(-t°C optimal: 72°C for Taq // time optimal: 2-3kb/60 sec) (-t°C optimal:68°C for Pfu // time optimal: 1kb/15 sec)
Final step 68/72°C for 10/5mins – to allow full extension of any oligonucleotides
Single Colony PCR A master mix is generally used for SCP, as well as Taq polymerase because the high error rate is not an issue here as it is purely confirmatory.  Cells from an individual colony are first spread onto a replica plate, and the same loop is then used to inoculate a microcentrifuge tube containing 100μ ddH20 which will later be heated to 95°C for 5 minutes to be used in the SCP (the same loop is finally used to inoculate LB for the overnight cultures). The protocol for the first SCP was as follows:
19.75μl ddH20
2.5μl Barnes buffer
1μl template (this comes from the tube that contains 100μ ddH20 and was inoculated with cells.
0.5μl dNTPs
0.5μl forward primer
0.5μl reverse primer
0.25μl Taq polymerase.
We also used a positive control (other DNA to which the primers will definitely anneal) and a negative control (ddH20).
The temperature cycle was as follows:
95°C for 30 seconds
30 cycles of: 95°C for 30 seconds, 62°C for 90 seconds, 68 °C for 30 seconds
68°C for 10 minutes
Hold at 4°C
Overnight Cultures
Tubes containing 5ml of LB medium are inoculated with cells from one colony and then 5μl of antibiotic (for example chloramphenicol) is added. They are then left at 37°C overnight.
SDS-PAGE
Short for: Sodium Dodecyl Sulfate Polyacrylamide Gel Electrophoresis
Prepare gel : Two glass plates are cleaned with ethanol and are fitted into a holder. The separation layer of the gel is prepared first, following the recipe but before and after addition of the last two substances the solution should be inverted. The mixture, that now starts to polymerize, in now pipetted between the glass plates until it reaches the green bar. Around 700µl of ethanol are than added on top of the gel, which is left to solidify. The separating gel contains 10% acrylamide (toxic!) that has been polymerized by TEMED. Stacking gel contains less acrylamide for wide pores. After the gel has solidified take out the comb. Once solidified the stacking gel can be prepared using a different recipe but same method. Once the ethanol has been removes the solution is poured onto of the gel and the comb inserted. The gel is now left to solidify.
Load samples : The proteins, which have been denatured by SDS, are loaded into the wells.
Run gels : The gel is run at 100V for around 2-3 hours until dye front has reached the bottom of  the gel.
Analysis of results : The gel can be analyzed by staining with Coomassie blue or Western Blot.
Catechol Assay
Catechol assay is performed in the plate reader on a 96 well plate
Each well must be filled with 100um of solution
Usually use 90ul of cell culture and 10um of catechol solution
Catechol stock solution is at 100mM concentration. And when added to the well we have a 10fold dilution. For example if an aliquot concentration of 1mM catechol is made, which would be used for assay, when the well is added catechol drops to a concentration of 0.1mM.
Always dilute catechol with H2O.
Always have a blank of 90ul medium (the one which you grew the cells overnight) with 10ul catechol solution
Always have a negative of 90ul growing cells and 10ul of H2O.
Other Useful Information
PCR purification
Used to purify DNA to remove primers, salts and enzymes. It can also be used to purify away small fragments from restriction digests, for example when cutting a vector open. We used the E.Z.N.A.® Cycle Pure Kit and protocol (Omega bio-tek) (ddH2O instead of Elusion Buffer used in last step).
Gel purification
We used the QIAquick® Gel Extraction Kit (250) and protocol (ddH2O instead of Elusion Buffer used in last step).
Excise DNA from the gel and put into BF falcon clip-top tube (Blue Box) (Sybr Safe performs better under blue light)
Check excision of the right band and weigh the slice ( gel can be frozen at -20°C to be extracted at a later date)
Add 3xvolume of buffer QG
Incubate at 50°C for 10 minutes or until completely dissolved, vortex every 2-3 minutes
Check colour – consult kit protocol if not orange
Add 1xvolume Isopropanol (crucial step to ensure that the DNA binds the column
Put 800µl into the QIA quick spin column with 2ml collection tube
Centrifuge for 1 minute and discard the flow through, repeat if necessary.
Add 500µl buffer QG to the quick spin column and centrifuge for 1 min
Add 750µl buffer PE to the quick spin column and centrifuge for 1 min
Dry the column by centrifuging for 1 minute
Place QIA quick spin column into 1.5ml collection tube (eppendorf tube)
Elute the DNA with 35µl of ddH2O
Minipreps
The E.Z.N.A.® Kit and protocol was used.
Midipreps
The QIAGEN HiSpeed Plasmid Midi Kit and protocol was used.
Agarose gels
1% Agarose gel (for DNA 1g Agarose for each 100ml 1xTAE buffer)
Marker – dilute invitrogen 1kb plus DNA Ladder (1 in 10 Loading Buffer (LB))
Diluting Primers
Add a volume of H2O equivalent to the yield of primer on the information sheet 1µg=1µl ddH2O
Leave to stand for 20 minutes
Mix thoroughly (pipette up and down)
Store at -20°C
Dilute 1 in 10 before use.
Oligo annealing
20µ total reaction volume
2µl of each single stranded primer
16µl ddH2O
2µl Buffer 2 (1 in 10 total volume)
Sequencing reaction mix
15µl total reaction volume
3µl of sequencing primer (amplifies the relevant DNA fragment)
50-100ηg of DNA
Label with sequencing labels and make sure that the codes from these are entered on MWG website.
DpnI Digests
DpnI digests methylated DNA, such as template DNA extracted from cells (colony PCR/Mini/Midi-prep), while leaving non-methylated PCR products uncut.
Can use 1µl of DpnI straight in the 25µl PCR product (PCR buffer is sufficient)
Alternatively use Buffer 4 – does not require BSA.
Incubate for 1 hr at 37°C
Rapid Alkaline Phosphatase
Dephosphorylation useful to prevent vector self re-ligation.
10µl Total Reaction Volume
--µl DNA vector ( concentration depends on relative concentrations of the parts to be ligated)
1µl Phosphatase buffer
1µl Alkaline Phosphatase
-µl make up 10 µl with ddH2O
Retrieved from " http://2010.igem.org/Team:Imperial_College_London/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
