<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Modelling_Protein_Display_Detailed_Description"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Modelling/Protein Display/Detailed Description</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="title"><DIV id="belink"><A href="http://www3.imperial.ac.uk/bioengineering">Department of Bioengineering</A></DIV><DIV id="mblink"><A href="http://www3.imperial.ac.uk/molecularbiosciences">Division of Molecular Biosciences</A></DIV></DIV><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="plan">Plan</A></LI><LI><A href="#" class="listhead" id="results">Results</A></LI><LI><A href="#" class="listhead" id="extra">Extras</A></LI></UL></DIV><DIV id="headpic"><DIV id="headover"><P id="headtit">Parasight<SPAN class="dark"> <SPAN class="highlight">|</SPAN>  Parasite detection with a rapid response</SPAN></P><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London" class="listmain" id="results">Home</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Tour/Page_One" class="listmain" id="results">Tour</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modules" class="listmain" id="results">Modules</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Chassis" class="listmain" id="results">Chassis</A></LI><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Human_Practices" class="listmain" id="results">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Schistosoma" class="listmain" id="results">Schistosoma</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Research" class="listmain" id="results">Research</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Diary/Week_One" class="listmain" id="project">Diary</A></LI></UL></UL><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modelling" class="listmain" id="results">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Strategy" class="listmain" id="results">Assembly</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Lab_Diaries" class="listmain" id="results">Lab Diaries</A></LI><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Protocol" class="listmain" id="results">Lab Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Safety" class="listmain" id="results">Safety</A></LI></UL></UL><UL class="results"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Results" class="listmain" id="results">Experimental Results</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Parts/Favourites" class="listmain" id="results">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/The_Team" class="listmain" id="results">The Team</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Media/Videos" class="listmain" id="results">Media</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/School_Workshops" class="listmain" id="results">School Workshops</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="listmain" id="results">Software Tool</A></LI><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Acknowledgments" class="listmain" id="results">Acknowledgments</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Brainstorming" class="listmain" id="results">Brainstorming</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Glossary" class="listmain" id="results">Glossary</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Sitemap" class="listmain" id="results">Sitemap</A></LI></UL></UL></DIV></DIV></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:25px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;" valign="top">Modelling
</TD><TD style="font-family: helvetica, arial, sans-serif;font-size:1.3em;color:#ea8828;" align="right"><A href="/Team:Imperial_College_London/Modelling" title="Team:Imperial College London/Modelling"> Overview</A> | <B>Detection Model</B> | <A href="/Team:Imperial_College_London/Modelling/Signalling/Objectives" title="Team:Imperial College London/Modelling/Signalling/Objectives">Signaling Model</A> | <A href="/Team:Imperial_College_London/Modelling/Output/Objectives" title="Team:Imperial College London/Modelling/Output/Objectives">Fast Response Model</A> |<A href="/Team:Imperial_College_London/Modelling/Experiments" title="Team:Imperial College London/Modelling/Experiments"> Interactions</A></TD></TR><TR><TD colspan="2"><B>A major part of the project consisted of modelling each module. This enabled us to decide which ideas we should implement. Look at the Fast Response page for a great example of how modelling has made a major impact on our design!</B></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:1.2em;color:#ea8828;" align="right"><A href="/Team:Imperial_College_London/Modelling/Protein_Display/Objectives" title="Team:Imperial College London/Modelling/Protein Display/Objectives">Objectives</A> | <STRONG class="selflink">Description</STRONG> | <A href="/Team:Imperial_College_London/Modelling/Protein_Display/Results_and_Conclusion" title="Team:Imperial College London/Modelling/Protein Display/Results and Conclusion">Results</A> | <A href="/Team:Imperial_College_London/Modelling/Protein_Display/Parameters_and_Constants" title="Team:Imperial College London/Modelling/Protein Display/Parameters and Constants">Constants</A> | <A href="/Team:Imperial_College_London/Modelling/Protein_Display/Download_MatLab_Files" title="Team:Imperial College London/Modelling/Protein Display/Download MatLab Files">MATLAB Code</A></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Detailed Description
</TD></TR><TR><TD>
This model consists of eight parts that had to be developed:
<OL><LI>Identification of all active and relevant elements of the isolated part of the system.</LI><LI>Identification of interactions between the identified elements.</LI><LI>Determining the threshold concentration of Auto Inducing Peptide (AIP) needed for activation of the receptor.</LI><LI>Determining the volume of the cell wall.</LI><LI>Defining a control volume around the bacterial cell (after cleavage, the surface protein will float around in the extracellular environment).</LI><LI>Determining the importance of localised concentrations in a control volume.</LI><LI>Determining the surface protein production to estimate the maximum abundance on the bacterial surface.</LI><LI>Final version of equations</LI></OL></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">1. Elements of the system
</TD></TR><TR><TD><OL><LI>The surface protein consists of a cell wall binding domain, a linker and AIP (Auto Inducing Peptide). It is expressed by constitutive gene expression. It was assumed that the bacteria would be fully grown before carrying out the detection so the cell wall would be covered with as many surface proteins as the cell can maintain.</LI><LI>Schistosoma elastase (enzyme released by the parasite) cleaves the AIP from the cell wall binding domain at the linker site. In the laboratory, we used TEV protease as we could not get hold of Schistosoma elastase, so the model was adjusted appropriately (TEV enzyme kinetic parameters were used).</LI><LI>The ComD receptor is activated by a sufficiently high AIP concentration. Once activated, ComD signals into the cell to activate the colour response.</LI></OL></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">2. Interactions between elements
</TD></TR><TR><TD><P>
Apart from the proteins being expressed from genes, there was only one more chemical reaction identified in this part of the system. This is the cleavage of proteins, which is an enzymatic reaction: 
<UL><LI>Substrate (S) = Protein</LI><LI>Enzyme (E) = TEV (Protease)</LI><LI>Product (P) = Peptide</LI></UL>
This enzymatic reaction can be rewritten as a set of ordinary differential equations (ODEs), which is of similar form to the 1-step amplification model.



This model is quite peculiar as we realised that its behaviour is not only dependent on time, but space as well. Namely, the cleavage reaction happens on the cell wall of the bacteria. However, AIP that has already been cleaved is allowed to diffuse in any direction. Since we were not sure how to model this scenario, we decided to determine the importance of localised concentrations in a control volume.

</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">3. Threshold concentration of AIP
</TD></TR><TR><TD><P>
The optimal peptide concentration required to activate ComD is 10 ng/ml <A href="http://ukpmc.ac.uk/backend/ptpmcrender.cgi?accid=PMC40587&amp;blobtype=pdf">[1]</A>. This is the threshold value for ComD activation. However, the minimum concentration of peptide to give a detectable activation is 0.5ng/ml. 

The threshold for the minimal activation of the receptor is c<SUB>th</SUB>=4.4658×10<SUP>-9</SUP> mol/L. Click on the button below to uncover the conversion from ng/ml to mol/L.

</P><DIV id="wrapper"><DIV class="accordionButton">Converting 10 ng/ml to 4.4658×10<SUP>-9</SUP> mol/L</DIV><DIV class="accordionContent"><UL><LI>The mass of a peptide is 2.24kDa = 3.7184×10<SUP>-21</SUP>g.</LI><LI>The number of molecules in one ml is 10ng/3.7184×10<SUP>-21</SUP>g = 2.6893×10<SUP>12</SUP>. In a litre, the number of molecules is 2.6893×10<SUP>15</SUP>molecules/L.</LI><LI>Dividing this value by Avogadro's constant gives the threshold concentration of c<SUB>th</SUB>=4.4658×10<SUP>-9</SUP> mol/L.</LI><LI>The threshold for minimal activation of a receptor is 2.2329×10<SUP>-10</SUP> mol/L.</LI></UL></DIV></DIV></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">4. Cell Wall Volume
</TD></TR><TR><TD>
It was necessary to calculate the volume of the cell wall as we needed it for the calculation of concentrations in the enzymatic reaction.
Volume of <I>B. subtilis</I> is 2.79μm<SUP>3</SUP> and the thickness of the cell wall is 35nm <A href="http://jb.asm.org/cgi/reprint/176/5/1413?ijkey=27dafbac7e23dee50390d3fe67d9d1bab0c6f48c">[5]</A>. In order to approximate the cell wall volume, assume that <I>B. subtilis</I> is a sphere - not a rod. Calculate the outer radius from the total volume: 0.874μm. Now subtract the thickness of the cell wall from the outer radius to determine the inner radius of the sphere: 0.839μm. The volume of the cell wall is equal to the difference between outer volume and the inner volume (calculated from the inner radius): <B>cell wall volume=0.32×10<SUP>-15</SUP>m<SUP>3</SUP></B></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">5. Control volume selection
</TD></TR><TR><TD>
Note that the product of the enzymatic reaction, AIP, is allowed to diffuse outside the cell. Hence, it is important to determine the boundaries of the system. It is worth considering whether diffusion or fluid movements will play a significant role.
<P>Initially, we defined a control volume assuming that bacteria would grow in close colonies on the plate. We realized that our initial choice of control volume was not accurate, since our bacteria are meant to be used in suspension so we had to reconsider this issue. If you wish to have a look at our initial working, click on the button below.
</P><DIV id="wrapper"><DIV class="accordionButton">Initial Choice of Control Volume</DIV><DIV class="accordionContent">
This control volume is considered to be wrong, but the details were kept for reference.<B>Control volume initial choice</B>

The control volume:
The inner boundary is determined by the bacterial cell (proteins after being displayed and cleaved cannot diffuse back into bacterium). The outer boundary is more time scale dependent. We have assumed that after mass cleavage of the display-proteins by TEV, many of these AIPs will bind to the receptors quite quickly (eg. 8 seconds). Our volume is determined by the distance that AIPs could travel outwards by diffusion within that short time. In this way, we are sure that the concentration of AIPs outside our control volume after a given time is approximately 0.
This approach is not very accurate and can lead us to false negative conclusions (as in reality there will be a concentration gradient, with the highest concentration on the cell wall).
</DIV></DIV></TD></TR><TR><TD>Control volume (Volume of <I>B. subtilis</I> to be excluded. x indicates the distance travelled by AIPs from the surface in time t).
</TD></TR></TBODY></TABLE></DIV><P>
Difussion distance was calculated using Fick's 1<SUP>st</SUP> Law: x=<VAR>√</VAR>2Dt, where: x - diffusion distance, D - diffusion constant, t - time of diffusion

D<SUB>average</SUB> = 10.7×10<SUP>-11</SUP> m<SUP>2</SUP>s<SUP>-1</SUP> for a protein in agarose gel for pH=5.6 <A href="http://www.sciencedirect.com/science?_ob=MImg&amp;_imagekey=B6V5N-4B3MXDC-2-K&amp;_cdi=5791&amp;_user=217827&amp;_pii=S1369703X03002377&amp;_origin=search&amp;_coverDate=07%2F01%2F2004&amp;_sk=999809998&amp;view=c&amp;wchp=dGLzVtb-zSkzS&amp;md5=c17d0e7320f03931006f9b1a10a438b9&amp;ie=/sdarticle.pdf">[3]</A>
t = 8s (arbitrarily chosen)

This gives: x = 4.14×10<SUP>-5</SUP>m

The control volume can be calculated by adding 2x to the length and the diamter of the cell. 

This gives a control volume (CV) = 4.81×10<SUP>-7</SUP>m<SUP>3</SUP></P></DIV></DIV><B>Using CFU to estimate the spacing between cells</B>
CFU stands for Colony-forming unit. It is a measure of bacterial numbers. For liquids, CFU is measured per ml.
We already have data of CFU/ml from the Imperial iGEM 2008 team, so we could use this data to estimate the number of cells in a given volume using a spectrometer at 600nm wavelength. 
The graph below is taken from the Imperial iGEM 2008 Wiki page <A hred="https://2008.igem.org/Imperial_College/18_September_2008">[4]</A>.
<DIV align="CENTER"><TABLE style="background:#e7e7e7;text-align:center;font-family: helvetica, arial, sans-serif;color:#555555;margin- top:5px;padding: 2px;"><TBODY><TR><TD>Calibration curve relating cell concentration to optical density at 600nm (OD600).
</TD></TR></TBODY></TABLE></DIV><P>
The graph shows values of CFU/ml for different optical densities. The range of CFU/ml is therefore between 0.5×10<SUP>8</SUP> - 5×10<SUP>8</SUP>. 

In this calculation, we assumed that only one cell will grow and become one colony (i.e. no more than one cell will form no more than one colony). Therefore, the maximum number of cells in 1ml of solution is 5×10<SUP>8</SUP>. Taking the volume of 1 ml = 10<SUP>-3</SUP> dm<SUP>3</SUP> and dividing by the (maximum) number of cells in 1ml gives the average control volume (CV) around each cell: 2×10<SUP>-12</SUP> dm<SUP>3</SUP>/cell. For simplicity, we choose the control volume to be cubic. Taking the third root of this value gives the length of each side of the control volume.
</P><P>Side length of cubic Control volume is y = 1.26×10<SUP>-4</SUP> dm = 1.26×10<SUP>-5</SUP> m.</P><B>Choice of Control Volume allows simplifications</B>
Firstly, assume that the cells will be placed in the centre of the CV. Hence, after cleavage the protein will have an average distance of y/2 to travel in order to cross the boundary of the CV. This is calculated to happen within 0.2s. Even if the bacterium was not placed in the centre of the CV, the protein will travel from one end of the cube to the other in less than one second (~0.8s). Hence, it will take almost a second for the concentration of AIPs around the cell to be uniform. We noticed that this time value is not very small (at a cellular level), so we decided that the control volume is too big to give us a good approximation of what is happening.
<TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">6. Localised concentrations
</TD></TR><TR><TD>
Since we failed to determine a control volume across which the concentration of AIP could be assumed to be uniform, it was deduced that localised concentrations will play an important role in this model. Hence, we tried to come up with some kind of measurement of localised concentrations. Since the whole reaction happens at the cell wall and just the final product (AIP) floats around freely, we decided just to scale the AIP concentration by a factor after completion of reaction to simulate the loss of AIPs that diffuse away from the cell surface. 

It was arbitrarily chosen that 20% to 50% of AIPs will bind to receptors rather than diffuse away. There are several arguments that would suggest this kind of percentage:
<UL><LI> AIPs being very close to the cell surface will have an equal probability of heading back towards the bacterium as diffusing away from it (since the cell is a lot bigger than the AIPs).</LI><LI>It is likely that there will be some chemical interactions between AIPs and the bacterium which could be attracting the AIPs to the host bacterium (e.g. electrostatic attraction could be possible).</LI></UL>
The localised concentration coefficient scales the AIP concentrations at the very end - after the ODE equations have been solved.
</TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">7. Protein production
</TD></TR><TR><TD><UL><LI>This paper mentions that each cell displays 2.4×10<SUP>5</SUP> peptides. <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2000.tb09188.x/pdf">[2]</A></LI><LI>2.4×10<SUP>5</SUP> molecules = 2.4×10<SUP>5</SUP>/6.02×10<SUP>23</SUP> mol = 0.398671×10<SUP>-18</SUP> mol</LI><LI>Volume of ''B.subtilis'' cell wall: 0.32×10<SUP>-15</SUP>m<SUP>3</SUP></LI><LI>Concentration = [mol/L]</LI><LI>c = 1.24×10<SUP>-3</SUP> mol/L. This is the concentration of protein that will be displayed on a single cell of B.sub. 
</LI></UL>

Hence, we can deduce that the final concentration that the protein expression will tend to is: c = 1.24×10<SUP>-3</SUP> mol/dm<SUP>3</SUP> = c<SUB>final</SUB>. This value will be the initial concentration of protein that is cleaved by TEV. The simple production of protein by transcription and translation was incorporated into our model using the following equation:

In the equation above: [p] = protein concentration, s = protein production rate, d = protein degradation rate
Therefore, we can model the protein production by transcription and translation and adjust the production constants so that the concentration will tend towards c<SUB>final</SUB> according to the ODE below. The degradation rate was kept constant (same as the one used in the output amplification module). This allowed us to generate the following data:
</TD></TR><TR><TD>Production of protein by transcription and translation.
</TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">8. Final version of ODE equations
</TD></TR><TR><TD>
We used linear properties of ODEs presented in section 2 and 7 to combine them in the following way: 

Since the cleaved surface protein abundant in the cell wall, we decided to add the intermediate complex concentration as contributing to the degradation term. 

</TD></TR><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;" align="right"><A href="/Team:Imperial_College_London/Modelling/Protein_Display/Results_and_Conclusion" title="Team:Imperial College London/Modelling/Protein Display/Results and Conclusion"> Click here for the results of this model...</A></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">References
</TD></TR><TR><TD><OL><LI>Havarstein, L., Coomaraswamy, G. &amp; Morrison, D. (1995) An unmodified heptadecapeptide pheromone induces competence for genetic transformation in Streptococcus pneumoniae. Proc. Natl. [Online] 92, 11140-11144. Available from: http://ukpmc.ac.uk/backend/ptpmcrender.cgi?accid=PMC40587&amp;blobtype=pdf [Accessed 27th August 2010]</LI><LI>Kobayashi, G. et al (2000) Accumulation of an artificial cell wall-binding lipase by Bacillus subtilis wprA and/or sigD mutants. FEMS Microbiology Letters. [Online] 188(2000), 165-169. Available from: http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2000.tb09188.x/pdf [Accessed 27th August 2010]</LI><LI>Gutenwik, J., Nilsson, B. &amp; Axelsson, A. (2003) Determination of protein diffusion coefficients in agarose gel with a diffusion cell. Biochemical Engineering Journal. [Online] 19(2004), 1-7. Available from: http://www.sciencedirect.com/science?_ob=MImg&amp;_imagekey=B6V5N-4B3MXDC-2-K&amp;_cdi=5791&amp;_user=217827&amp;_pii=S1369703X03002377&amp;_origin=search&amp;_coverDate=07%2F01%2F2004&amp;_sk=999809998&amp;view=c&amp;wchp=dGLzVtb-zSkzS&amp;md5=c17d0e7320f03931006f9b1a10a438b9&amp;ie=/sdarticle.pdf [Accessed August 20th 2010]</LI><LI>Imperial College London (2008) Biofabricator Subtilis - Designer Genes. [Online] Available from: https://2008.igem.org/Imperial_College/18_September_2008 [Accessed 1st September 2010]</LI><LI>Graham L. L. &amp; Beverisge T. J. (1993) Structural Differentiation of the Bacillus subtilis 168 Cell Wall. Journal of Bacteriolofy. [Online] 5(1994), 1413-1420. Available from: http://jb.asm.org/cgi/reprint/176/5/1413?ijkey=27dafbac7e23dee50390d3fe67d9d1bab0c6f48c [Accessed October 26th 2010]</LI></OL></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description">http://2010.igem.org/Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Modelling/Protein_Display/Detailed_Description&amp;oldid=207047" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>