<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Lab_Diaries_XylE_team"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Lab_Diaries/XylE_team">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Lab_Diaries/XylE_team&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Lab_Diaries/XylE_team&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Lab_Diaries/XylE_team&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Lab_Diaries/XylE_team" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Lab Diaries/XylE team</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="title"><DIV id="belink"><A href="http://www3.imperial.ac.uk/bioengineering">Department of Bioengineering</A></DIV><DIV id="mblink"><A href="http://www3.imperial.ac.uk/molecularbiosciences">Division of Molecular Biosciences</A></DIV></DIV><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="plan">Plan</A></LI><LI><A href="#" class="listhead" id="results">Results</A></LI><LI><A href="#" class="listhead" id="extra">Extras</A></LI></UL></DIV><DIV id="headpic"><DIV id="headover"><P id="headtit">Parasight<SPAN class="dark"> <SPAN class="highlight">|</SPAN>  Parasite detection with a rapid response</SPAN></P><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London" class="listmain" id="results">Home</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Tour/Page_One" class="listmain" id="results">Tour</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modules" class="listmain" id="results">Modules</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Chassis" class="listmain" id="results">Chassis</A></LI><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Human_Practices" class="listmain" id="results">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Schistosoma" class="listmain" id="results">Schistosoma</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Research" class="listmain" id="results">Research</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Diary/Week_One" class="listmain" id="project">Diary</A></LI></UL></UL><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modelling" class="listmain" id="results">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Strategy" class="listmain" id="results">Assembly</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Lab_Diaries" class="listmain" id="results">Lab Diaries</A></LI><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Protocol" class="listmain" id="results">Lab Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Safety" class="listmain" id="results">Safety</A></LI></UL></UL><UL class="results"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Results" class="listmain" id="results">Experimental Results</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Parts/Favourites" class="listmain" id="results">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/The_Team" class="listmain" id="results">The Team</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Media/Videos" class="listmain" id="results">Media</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/School_Workshops" class="listmain" id="results">School Workshops</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="listmain" id="results">Software Tool</A></LI><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Acknowledgments" class="listmain" id="results">Acknowledgments</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Brainstorming" class="listmain" id="results">Brainstorming</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Glossary" class="listmain" id="results">Glossary</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Sitemap" class="listmain" id="results">Sitemap</A></LI></UL></UL></DIV></DIV></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:25px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;" valign="top">Lab Diaries
</TD><TD style="font-family: helvetica, arial, sans-serif;font-size:1.5em;color:#ea8828;" align="right"><A href="/Team:Imperial_College_London/Lab_Diaries" title="Team:Imperial College London/Lab Diaries"> Overview</A> | <A href="/Team:Imperial_College_London/Lab_Diaries/Surface_protein_team" title="Team:Imperial College London/Lab Diaries/Surface protein team">Surface Protein Team</A> | <STRONG class="selflink">XylE Team</STRONG> | <A href="/Team:Imperial_College_London/Lab_Diaries/Vectors_team" title="Team:Imperial College London/Lab Diaries/Vectors team">Vectors Team</A> | <A href="/Team:Imperial_College_London/Lab_Diaries/Modelling_team" title="Team:Imperial College London/Lab Diaries/Modelling team">Modelling Team</A></TD></TR><TR><TD colspan="2"><B>Here are the technical diaries for our project. We've split them up into three lab teams and the modelling team. We think it's really important that absolutely anyone can find out what we've been doing. For a really detailed look at what we did, and when, you've come to the right place!</B></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"> XylE Team
</TD></TR><TR><TD><P><B>Objectives:</B></P><UL><LI>construction of XylE fusion protein
</LI><LI>Testing expression of XylE in E.coli and characterization under the control of a constitutive promoter
</LI><LI>Construction of the -ComE promoter/XylE fussion protein- expression system
</LI><LI>Construction of the -LacI promoter/XylE fusion construct- expression system
</LI></UL><P><B>Here's a picture of the final construct:</B></P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Typical XylE assay
</TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"> Week 6
</TD></TR><TR><TD style="background-color:#FFFF66;text-align:center;color:#555555;"><B>Week 6</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Monday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Tuesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Wednesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Thursday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Friday</B></TD></TR><TR><TD style="background-color:#FFCC66;width:50px;text-align:center;color:#555555;"><B>Morning</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>mini-prep kit of XylE-transformed ''E.coli'' (already overnight grown)</LI></UL></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:center;color:#555555;"><B>Afternoon</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Starting of “Testing expression of XylE in ''E.coli''” objective</LI><LI>1)Annealing EcoRI and speI oligos to J23101 promoter which will be annealed later in front of the RBS-XylE registry gene (overnight) </LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>gel analysis of mini-prep derived XylE plasmid.(requires first digestion of the vector with restriction enzymes)</LI></UL></TD></TR></TBODY></TABLE><P><B>Thursday, 12-Aug-2010</B></P><UL><LI> Annealing DNA strands of J23101 promoter in a water bath
</LI></UL><P>we constructed the standard <I>E.coli</I> promoter J23101 with sticky ends. These ends are complementary to restriction sites made by EcoRI and SpeI enzyme. This promoter will be later used in 3A assembly to construct a promoter-RBS-XylE design in a psB1C3 vector. <I>E.coli</I> will be transformed with this final construct plasmid to assess XylE activity and characterization. It will also be one of the submitted BioBricks.
</P><UL><LI> Prepared two overnight cultures of XylE transformed <I>E.coli</I> (one 50microliters and one of 450μl)
</LI></UL><P>these cultures are going to be used tomorrow for mini prepping. Mini prep will allow us to isolate <I>E.coli</I> 's plasmid DNA(which contains the XylE gene).
</P><P><B>Friday, 13-Aug-2010</B></P><UL><LI> Mini-prep of XylE transformed <I>E.coli</I></LI></UL><P>Mini prep is usually used to confirm that our gene of interest has not been changed in any way, as the isolated plasmid id sent for sequencing. However, since XylE was taken from the registry, we assume that it is fine and no sequencing is required. The mini prep will later be used for the midi-prep (that gives out higher yields of DNA needed for cloning).
</P><UL><LI> Gel analysis of plasmid DNA retrieved from mini prep of XylE transformed <I>E.coli</I>, cut with restriction enzymes. From light to the left, 50μg digested DNA : 50 undigested DNA : 450 digested DNA : 450 undigested DNA. In lanes 1 and 3 the smaller band has a size of about 1kB which corresponds to RBS-XylE gene. The bigger bands are the cut vectors. In lanes 2 and 4 is the uncut BioBrick from the registry. It appears smaller on the gel than it actually is as circular DNA travels faster through the pores of agarose gel rather than linearised DNA.
</LI></UL><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"> Week 7
</TD></TR><TR><TD style="background-color:#FFFF66;text-align:center;color:#555555;"><B>Week 7</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Monday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Tuesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Wednesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Thursday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Friday</B></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:center;color:#555555;"><B>Morning</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>midi prep XylE ''E.coli'' (2hrs)</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>gel extraction kit on XylE gene trapped in agarose </LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>3A assemply: make replica plates (overnight)</LI><LI>Catechol assay of ''E.coli'' </LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Mini prep XylE ''E.coli''</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Midi prep XylE ''E.coli''</LI></UL></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:center;color:#555555;"><B>Afternoon</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>restriction digestion of XylE for 3A assemply</LI><LI>gel purification of XylE from restriction digestion</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>3A assembly of vector, XylE and J23101 promoter </LI><LI>above construct transformed in ''E.coli''</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>primers arrived!</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>PCR extension of XylE and GFP, round 1</LI></UL></TD></TR></TBODY></TABLE><P><B>Monday, 16-Aug-2010</B></P><UL><LI> Midi prepped the XylE-transformed <I>E.coli</I>. The DNA yield from the midi prep was 134μg as determined by spectrophotometry. This is the XylE that is going to be used for all further experiments.
</LI><LI> Restriction digestion of midi prepped XylE by Xbal and PstI to prepare it for 3A assembly. (with J23101 promoter and PSB1C3 vector)
</LI><LI> Gel analysis of the restriction digestion mixture to isolate XylE gene
</LI></UL><P>
Midi prep XylE digestion with xbaI and PstI The smaller size bands at lanes 2 &amp; 3 are the ones that are going to be cut out and used in gel purification to extract the XylE gene (with sticky ends for XbaL and PstI).
</P><UL><LI> I made an overnight culture of <I>Bacillus</I></LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Midi prep XylE digestion with xbaI and PstI The smaller size bands at lanes 2 &amp; 3 are the ones that are going to be cut out and used in gel purification to extract the XylE gene (with sticky ends for XbaL and PstI).</DIV></DIV></DIV></DIV><P><B>Tuesday, 17-Aug-2010</B></P><UL><LI> Using the Gel Extraction Kit, we isolated the restriction enzyme cut XylE gene from the agarose gel band.
</LI><LI> Gel analysis of XylE, J23101 promoter and pSB1C3 vector samples to determine their ratios for 3A assembly ligation
</LI><LI> 3A assemply of XylE, J23101 promoter and pSB1C3 vector.
</LI><LI> Transformation of XL-Blue competent <I>E.coli</I> with the above construct.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:346px;"><DIV class="thumbcaption">gel analysis of XylE, J23101 promoter and pSB1C3 vector samples to determine the volume ratios of samples to be used for 3A assemply ligation</DIV></DIV></DIV></DIV><P>gel analysis of XylE, J23101 promoter and pSB1C3 vector samples to determine the volume ratios of samples to be used for 3A assemply ligation
</P><UL><LI> I followed Chris's <I>Bacillus</I> transformation protocol to transform <I>Bacillus</I> with constitutive GFP and RFP DNA as well as a control without DNA
</LI></UL><TABLE style="width:800px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Successful Bacillus transformation!
</TD></TR></TBODY></TABLE><P><B>Thursday, 19-Aug-2010</B></P><UL><LI> We run the XylE-GFP1 PCR reaction to construct the His-GFP-Flag and Linker-XylE-Spe construct.
</LI></UL><P><B>Friday, 20-Aug-2010</B></P><P>The J23101 gene in a BioBrick vector containg RFP gene
</P><UL><LI> We run a gel on XylE-GFP1 PCR reaction. Results: GFP was extended successfully, XylE extension FAILED (too much non-specific annealing)
</LI><LI> Catechol assay on 2hrs bench ligation of promoter, XylE and vector failed.
</LI><LI> Two replica plates of overnight ligated J23101, XylE and pSB1C3 transformed <I>E.coli</I> (one for catechol assay)
</LI><LI> The His-GFP-Flag DNA was gel purified
</LI><LI> Transformation of XL-1Blue cells with J23101 in J62001 vector from the registry. One more attemp to construct a successful promoter-XylE ligation, since we believe that the strand annealed promoter was of bad quality
</LI></UL><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Week 8
</TD></TR><TR><TD style="background-color:#FFFF66;text-align:center;color:#555555;"><B>Week 8</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Monday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Tuesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Wednesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Thursday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Friday</B></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:center;color:#555555;"><B>Morning</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>PCR extention of His-GFP-flag round 2</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>midi prep registry obtained J23101</LI><LI>gel purification of XbaI-His-GFP-flag-TEVs</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>gel analysis of overnight ligation and gel separation</LI><LI>gel analysis of linker-XylE-SpeI PCR reaction  </LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>make replica plate and catechol assay plate </LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>catechol assay</LI></UL></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:center;color:#555555;"><B>Afternoon</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Gel purification of linker-XylE-Spe PCR construct</LI><LI>Gel analysis of His-GFP-flag round 2 PCR rection and then gel separation of DNA </LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>restriction digestion, gel analysis and gel purification of registry obtained J23101 </LI><LI>PCR extension of linker-XylE-SpeI</LI><LI>ligation of pSB1C3, J23101 and XylE (overnight)</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>gel purification of overnight ligation</LI><LI>transformation of ''E.coli'' with overnight ligation product and selection by plating</LI><LI>gel purification of TEVs-linker-XylE-SpeI construct </LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>PCR construction of fusion protein</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Gel analysis and gel purification of fusion XylE protein</LI></UL></TD></TR></TBODY></TABLE><P><B>Monday, 23-Aug</B></P><UL><LI> Set up overnight cultures for midi prep
</LI><LI> Gel separation of linker-XylE-Spe DNA <FONT color="red"><B>FAILED</B></FONT></LI><LI> PCR reaction for extension of His-GFP-Flag
</LI><LI> Catechol assay on <I>E.coli</I> transformed with overnight ligated J23101, XylE and pSB1C3. <FONT color="red"><B>FAILED</B></FONT></LI></UL><P><B>Tuesday, 24th-Aug</B></P><UL><LI> Midi prepped registry obtained J23101. A yield of 130ng/μl of promoter was obtained. The promoter is in a BioBrick vector called J62001. The promoter is upstream of RFP gene.
</LI><LI>the vector carrying the promoter was digested with SpeI and PstI, while XylE gene was digested with XbaI and PstI.
</LI><LI> The promoter and the XylE gene were gel purified.
</LI><LI> A reaction between the promoter(still on vector) and XylE was set on for overnight ligation
</LI><LI> PCR purification of GFP2 -&gt; GFP construct ready for full fusion protein. 
</LI><LI> Gel purification of XylE lost DNA along the way. Thus PCR XylE1 with gradient for temperature scanning (taq): PCR round 1 included 62°C and only rev primer to create pool of successful extensions with with rev primer (60-62°C). PCR round 2 with Fwd primer and temperature scale (68-68°C, 72-74°C).
</LI></UL><P><B>Wednesday, 25th-Aug</B></P><P>Performed gel analysis on the purified XylE and J23101 to obtain ratios for ligation. First gel was scrapped as it produced appalling (explanation for Nick: really bad) results, 2nd gel run was successful. 
</P><UL><LI> Performed a ligation reaction between the vector containing J23101, and XylE (one on bench and one overnight one). 
</LI><LI> Transformation of the new plasmid into competent <I>E.coli</I>. Successfully transformed colonies can be selected for by loss of RFP expression.
</LI><LI> XylE-1 PCR with temperature cascade. Gel analysis and purification. 
</LI></UL><P><B>Thursday, 26th-Aug</B></P><UL><LI> White colonies from the promoter-XylE transformed <I>E.coli</I> were picked and transferred to new amp plates. One is the replica plate and the other is the catechol assay plate.
</LI><LI> XylE-1, two rounds of PCR/purification were run to obtain a sufficiently clear band. An additional PCR run for XylE-1 was discarded afterwards.
</LI></UL><P><B>Friday, 27th-Aug</B></P><UL><LI> Catechol assay performed on promoter-XylE transformed <I>E.coli</I>. <FONT color="green"><B>SUCCESSFULL</B></FONT></LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plate before adding catechol assay</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">After addition of catechol colonies turn yellow-orange in seconds!!</DIV></DIV></DIV></DIV><UL><LI> XylE-2 PCR and gel-purification cycles (2x) to obtain clear band. XylE-2 is now ready for assembly of the GFP-XylE fusion protein.
</LI></UL><P><B>Saturday, 28th-Aug</B></P><UL><LI> Preparing the annealing step between the GFP-2 and XylE-2 constructs, we discovered sequence dissimilarities in the TEV-cleavable regions which we planned to use for the annealing step. Nevertheless a PCR was run with appropriate conditions (allowing for a minimal amount of unspecific annealing). 
</LI></UL><P><B>Sunday, 29th-Aug</B></P><UL><LI> Gel analysis of the attempted annealing reaction of GFP-2 XylE-2 showed unsufficiently clear bands for gel-purification. A new reaction is being prepared: 10 rounds of annealing PCR, followed by addition of primers (5' primer for GFP-2 and 3' primer for XylE-2) in order to introduce an amplification step in the reaction. --- Following Kirstin's advice, we are discarding this reaction and wait for the arrival of new primers for XylE-2 (5' + TEV) and GFP-2 (3' +TEV).
</LI></UL><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Week 9
</TD></TR><TR><TD style="background-color:#FFFF66;text-align:center;color:#555555;"><B>Week 9</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Monday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Tuesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Wednesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Thursday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Friday</B></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:center;color:#555555;"><B>Morning</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>mini prep of promoter-XylE transformed ''E.coli''</LI><LI>design of reverse primer for GFP to add the corrected TEV sequence to the construct.</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>midi prep of promoter-XylE transformed ''E.coli''</LI><LI>restriction digestion of pSB1C3+terminator vector DNA</LI><LI>restriction digestion of promoter-XylE DNA  </LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Gel purification kit of cut promoter-XylE DNA</LI><LI>Preparation of M9 and LB medium for assay </LI><LI>Gel analysis of of insert and vector DNA</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>find optimun wavelength for catechol assays</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>find optimun cell density and catechol concentration for assay</LI><LI>Reverse primer for GFP with modified TEV arrived - primer dilution</LI></UL></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:center;color:#555555;"><B>Afternoon</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>gel analysis of mini prepped promoter-XylE</LI><LI>cross-check for primer design (GFP-TEV-2) and ordering</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>PCR purification of cut pSB1C3+terminator vector DNA </LI><LI>Gel analysis of cut promoter-XylE DNA</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>dephosphorylation of the vector+terminator</LI><LI>ligation reaction between promoter-XylE insert and pSB1C3+terminator vector</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>GFP 2 PCR with new reverse primer (adopted TEV sequence)</LI></UL></TD></TR></TBODY></TABLE><P><B>Monday, 30th-Aug</B></P><UL><LI> Mini prep of 4 x J23101-XylE taken from 4 different colonies (8 14 24 27). 
</LI><LI> The gel analysis showed successful vector uptake. 
</LI><LI> Set up an overnight culture for midi using colony 24 (gel analysis showed similar results colony 24 was picked randomly.)
</LI><LI> A new primer was designed in order to add a corrected TEV-protease-cleavable sequence to the His-GFP-Flag construct. This was controlled and ordered.
</LI></UL><P><B>Tuesday, 31st-Aug</B></P><UL><LI> Midi prep of colony 24 for XylE-J23101 the final concentration was ~100ng/μl which wasnt so great but Chris says the protocol produces very poor yields. 
</LI><LI> We are performing the next building step of our vector. PSB1C3 containing terminator B0014 was cut with EcoRI and XbaI. The insert was cut with EcoRI and SpeI and both were incubated for 1.5hrs. Wolf is now running a gel to purify out the insert via gel purification and perform a PCR purification on the vector.
</LI><LI>Advisors have decided it's best not to use Jeremy's tagged XylE due to the 93% difference. Kirsten will be tagging the registry XylE and we shall purify and assay with that instead. 
</LI><LI>We shall see purification expert Kieran tomorrow and talk through the process. Chris will also talk us through our characterization of XylE experiments- we will use the robot after it's been programmed but until then we can use the plate-reader.
</LI></UL><P><B>Thursday, 2nd-Sept</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Spectra of XylE transformed E.coli after addition of catechol assay. The <B>broad peak around 380nm wavelength</B> arises is due to the presence of the product of the enzymatic reaction involving pyrocatechol and XylE enzyme. This peak if absent if a culture of XylE transformed cells are measured without the addition of catechol</DIV></DIV></DIV></DIV><UL><LI>Spectrophotometry experiments with XylE transformed <I>E.coli</I> in LB medium (M9 culture was contaminated) reveiled the followings: On catechol assay of the trasformed cells, the <B>positive yellow output can be quantitively measured by a broad peak at 380nm.</B></LI><LI>Transformation of competent <I>E.coli</I> cells with promoter-XylE-terminator pSB1C3vector.
</LI></UL><P><B>Friday, 3rd-Sept</B></P><UL><LI>experiment to determine concentrations of catechol and cell density for assays
</LI><LI>The new GFP + TEV primer arrived, was diluted and used to set up the appropriate PCR.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Catechol assay on XylE-trasformed cells in a 96-well plate (A to H decreasing cell concentration, 1-10 decreasing catechol concentration, column 11 and 12 negative and control)</DIV></DIV></DIV></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Week 10
</TD></TR><TR><TD style="background-color:#FFFF66;text-align:center;color:#555555;"><B>Week 10</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Monday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Tuesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Wednesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Thursday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Friday</B></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:center;color:#555555;"><B>Morning</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>PCR extension of PVeg promoter </LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Gel purification of extended PVeg promoter</LI><LI>Annealing PCR for GFP-XylE fusion protein</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>vector with insert tranformed into ''E.coli'' and plate to select for successful transformation</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>prepare overnight cultures of pVeg-RBS transformed ''E.coli'' </LI><LI>Gel-analysis of amplification-PCR for XylE-2</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>midi prep of PVeg-RBS vector</LI></UL></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:top;color:#555555;"><B>Afternoon</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Gel analysis and purification of the GFP-TEV construct</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>restriction digestion and gel purification of thr extended pVeg promoter</LI><LI>overnight ligation of the cut pVeg promoter in a vector </LI><LI>Gel analysis of GFP-XylE annealing PCR - unsuccessful reaction</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Amplification PCR for XylE-2</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Gel-purification of amplification-PCR for XylE-2</LI></UL></TD></TR></TBODY></TABLE><P><B>Monday, 6th-Sept</B></P><UL><LI> PCR extension of pVeg promoter: EcoRI---pVeg---RBS-SpeI
</LI></UL><UL><LI> I performed a catechol assay on the picked transformed colonies to deduce which ones were successfully transformed with the insert plus vector. 1-5 7 and 10 failed to turn yellow (1-5 were background controls)leaving 8 yellow colonies. 
</LI><LI> I had to perform a colony PCR on two selected colonies 8 and 14 to check the correct insert+terminator was present. 
</LI><LI> Colony 8 when run on a gel analysis showed the correct size 924 XylE + 97 J23101 + 35 B0014. 
</LI><LI> This was set up as an overnight culture.
</LI><LI> Gel analysis of the GFP-TEV construct showed satisfactory bands.
</LI></UL><P><B>Tuesday 7th Sept</B></P><UL><LI> Mini prep of the overnight Colony 8 culture (by Wolf)
</LI><LI> Midi prep of overnight culture
</LI><LI> Gel purification of PCR product EcoRI--pVeg--RBS--SpeI
</LI><LI> Overnight restriction digestion of EcoRI--pVeg--RBS--SpeI with SpeI
</LI><LI> Started data analysis of plate assay
</LI><LI> Annealing PCR reaction included 2 samples without and one with additional primers (XbaI-His-GFP Fwd, SpeI-XylE Rev). While the primer-including reaction did not show any clearly identifiable bands, the others showed clear, if very weak bands at 800 and 1000bp, which represent the GFP and XylE constructs respectively. No band was identifiable at the 1.7 kb range, which would have indicated a successful annealing reaction. However, problems with the lense of the gel-analyser were only discovered later to have severely reduced the band brightness. Potentially a PCR-reaction with appropriate primers to amplify an annealed product(XbaI-His-GFP Fwd, SpeI-XylE Rev), could have been successful. However the reaction mixture was disposed of before this became clear.
</LI></UL><P><B>Wednesday, 8th Sep</B></P><UL><LI> A second PCR extension of pVeg promoter to introduce the RBS and cut sites. It was also gel purified and stored in gel lumbs in the freezer. (maybe needed later so keep in mind).
</LI><LI> Overnight restriction digestion completed. Then, it was run on the gel to check if it worked and then gel purified again.
</LI><LI> Vector PSBI-C3 was digested to remove the terminator and make it sticky for the insert (pVeg-RBS). Then it was run on the gel to check if restriction has worked, but the gel didn't run far enough to determine easily between undigested and digested vector.
</LI><LI> For further annealing reactions for GFP-XylE constructs additional XylE(2) template was required -&gt; amplification PCR for XylE(2).
</LI></UL><P><B>Thursday, 9th Sep</B></P><UL><LI> Gel with digested and undigested vector PSBI-C3 was run and then the digested vector was purified.
</LI><LI> Gel was run to determine the DNA concentration ratio for the ligation of PSBI-C3 and pVeg-RBS.
</LI><LI> Vector PSBI-C3 was dephosphorylated.
</LI><LI> Ligation of PSBI-C3 and pVeg-RBS has been set up overnight.
</LI><LI> Gel-analysis and gel-purification of the XylE(2) amplification PCR product.
</LI><LI> The transformation of <I>E.Coli</I> with pVeg-RBS in PSBI-C3 and PSBI-C3 by itself (to check see how successful dephosphoryaltion of PSBI-C3 was and estimate the percentage of bacteria that contain the insert) was completed
</LI></UL><UL><LI> Concentration of the midi prep of J23101-XylE-B0014 was determined to be ~600ng/μl (using new protocol)
</LI><LI> Kyasha kindly digested my midi with EcoRI and SpeI and performed a gel analysis. The results show a potential additional plasmid contaminating my midi however the concentration of DNA was extremely high. NB Chris said that it could be sheared DNA from a midi prep step. 
</LI></UL><UL><LI> The midi prepped plasmid was transformed into testing <I>E.coli</I> strain TOP10.
</LI></UL><P><B>Friday, 10 sep</B></P><P>The transformation was a SUCCESS. 2x replica plates were made plate 1# 1-6 plate #2 6-11; colony 6 and colony 9 of plates 1 and 2 respectively were transfered into a 5ml liquid culture + 5μl CmR. These will later be turned into glycerol stocks. After the replica plates have grown up mini preps on a number of colonies shall be performed - this hopefully will eliminate the contaminating plasmid DNA. This will be followed by a midi prep.
</P><P><B>Saturday, 11 Sep</B></P><P>The transformation of <I>E.Coli</I> with PSB1-C3 with insert did not work :(
</P><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Week 11
</TD></TR><TR><TD style="background-color:#FFFF66;text-align:center;color:#555555;"><B>Week 11</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Monday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Tuesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Wednesday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Thursday</B></TD><TD style="background-color:#FFFF99;text-align:center;color:#555555;"><B>Friday</B></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:center;color:#555555;"><B>Morning</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Gel-analysis of amplification-PCR for XylE-2</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>midi prep of pVeg-RBS vector</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>In preparation of ''catechol cell-death assay'' tranformation of Top10 cells with CMR plasmid (psb1C3)</LI></UL></TD></TR><TR><TD style="background-color:#FFCC66;width:100px;text-align:center;color:#555555;"><B>Afternoon</B></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>Gel-purification of amplification-PCR for XylE-2</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>plating and overnight incubation of transformed top10 CMR</LI></UL></TD><TD style="background-color:#eeeeee;height:100px;width:200px;text-align:left;color:#555555;"><UL><LI>overnight cultures for catechol cell death assay: Top10 cells XylE/CMR </LI></UL></TD></TR></TBODY></TABLE><P><B>Monday 13th september</B>
Overnights weren't set up on sunday so they were made up alongside some assay cultures. J23101-XylE-B0014 colonies 8 and 10 of the replica plate #2 were picked. Chris also provided a replica plate containing 3k3 vector colonies. This was over a year old and he was unsure whether it was the correct plasmid or if the cells would grow up. I picked all available colonies 102 150 151 and 260 of kanamycin resistance. Lastly for assays 2x LB 2xM9 cultures were made 5ml +5ul antibiotic.
</P><P><B>Tues 14th September</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">test confirming that yellow product of XylE enzmymatic reaction is leaks back from the cytosol into the solution.1C and 2C is the XylE producing cells after centrifuging and redissolving of the pellet and 1S and 2S is the supernatant after centrifuging</DIV></DIV></DIV><UL><LI>assay on plate reader of:
</LI><LI>0-0.75 initial O.D. of transformed <I>E.coli</I></LI><LI>0-1mM initial catechol concentration
</LI></UL><P>The assay was carried out with <I>E.coli</I>, top ten spcies, transformed with J23101-XylE-B0014 in pSB1C3 vector.The overnight culture wastransfered in new medium this morning for 4 hrs before assaying. LB medium was used for dilutions and blank. Catechol was diluted in ddH2O. 
<A href="/File:Catechol_assay_test_2.xls" title="File:Catechol assay test 2.xls">Data analysis of the assay</A></P><UL><LI>Mini preps of XylE 8,10 colonies and 3K3;102 151 260 colonies. Analytical digests were performed using E+S on XylE, E+S AccI and HindIII+E for 3K3 vector respectively. The band patterns correctly identified the vector to be 3k3. Colony 10 of XylE and Colony 260 of the 3k3 vector were set up direct into 100ml LB and so will require til 12pm for midi prepping on weds. Nick performed a second plate reader assay to determine the minimal concentration of Catechol to use for assays. I took 990ul M9 culture containing colony 10 and 10ul catechol was added. This was incubated for 10 mins and then spun down. The supernatant was removed into a cuvette and the cells resuspended in M9 salts. The ODs were then read on the spectrophotometer at 380 and 600nm. It was found that control cuvette of M9 salts had a miniscule reading. M9 + cells ~0.007. Supernatant + cells were ~ 2.5 therefore we could deduce that the coloured catechol breakdown product is exported out of the cells.
</LI></UL><P>Piotr:
</P><UL><LI> Mini prep of 1 sample (6 samples lost due to mistake) of GFP-Xyle fusion protein and its digestion with Spe and Xba (gel to be run on the next day)
</LI><LI> Preparing <I>E.Coli</I> colonies for the next day for mini prep to be redone 
</LI><LI> PCR of 6 samples of GFP-Xyle fusion
</LI></UL><P><B>Weds 15th September</B></P><UL><LI> Midi preps of overnights.
</LI><LI> In preparation for assays determining the effect of catechol and/or breakdown product on cell viability we prepared Top10 cells, one strain containing pVeg-XylE-terminator and the other containing a CMR plasmid of similar size. As the XylE cells have already been prepared, only Top10 transformation with CMR-plasmid had to be carried out.
</LI></UL><P><B>Thurs 16th September</B></P><UL><LI>Top10 CMR transformation was successful. Four overnight cultures of each Top10 XylE and Top10 CMR were set off with either M9 or LB medium respectively.
</LI><LI>Piotr: Did mini preps from bacteria with blunt ended Xyle-GFP fusion and did diagnostic using both:
</LI></UL><P>- digest with XbaI and SpeI
</P><P>- PCR reactions with the following primers added: HIS-GFP, XylE-Xba and the other sample with HIS-GFP and GFP-flag
</P><P>The PCR reactions were not very conclusive on the gels but digests allowed to determine that colonies 1-&gt;5 seem to have the right sizes of DNA in them and on Friday they will be prepared to be sent off for sequencing
</P><UL><LI> Continuation of the midi from overnight step. A gel analysis was run which showed the correct bands were present. A gel purification of the protein was then performed and the bands excised.
</LI></UL><P><B>Friday 17th September</B></P><UL><LI>Catechol assay of transformed <I>E.coli</I> Top ten(J23101-XylE-terminator) in M9 medium + data analysis
</LI><LI><A href="/File:M9_catechol_concentrations_assay_alternative.xls" title="File:M9 catechol concentrations assay alternative.xls">Data analysis</A></LI><LI><A href="/File:Catechol_assay_in_M9.xls" title="File:Catechol assay in M9.xls">alternative</A></LI></UL><P>The gel purifications of 3k3 vector and XylE were used for ligation.
3K3 was dephosphorylated and then ligated in a ratio 5:1 with the insert. This was left overnight and Chris tried to transform with it on sunday.. ligation and transformation failed :-(
</P><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Week 12
</TD></TR><TR><TD><P><B>Monday 20th Sept</B>
Performed ligation again. Errors in the sequencing we received back regarding J23101-XylE-B0014.
One was a log error (in XylE) phew. The other is in PSB1C3 out of the scar site and should be OK.
Chris is currently assembling the pVeg promoter+ RBS in a vector. Once he has finished this, I will combine XylE-GFP and XylE with this promoter for comparison characterization with J23101 (in <I>E.coli</I> and <I>Bacillus</I>)
</P><P><B>Tues 21st Sept</B></P><UL><LI> Ligation to be transformed in a bit. (Issue with the vector not being cut efficiently=failed ligations?) 
</LI><LI> I am cutting the original XylE midi as well as PSB1C3 containing B0014 and ligating them today (hopefully) the vector is purified, waiting on the gel run and then extraction of XylE insert. A ligation ratio must be determined from a gel analysis then dephosphorylation of the vector and then ligation. This will produce a promoter-less XylE+terminator vector which will be readily available to switch in the desired promoter (E+S cut)
</LI></UL><UL><LI> Florian is performing a mini prep on pVeg promoter which if it is correctly identified in the gel i can use to ligate to my XylE + PSB1C3-B0014. 
</LI><LI> Piotr is PCRing the reverse version of XylE that will be under the control of inducible promoter LacI (not delivered from synthesis yet) that will become a testing construct.
</LI></UL><P><B>Wed 22nd Sept</B></P><UL><LI> Piotr's PCR has been successful for 3 samples (1. recommended annealing temp; 2. recommended + 2 degrees and 3. recommended + 4 degrees). Sample no1. has been run on the gel and gel purified. It has been cut overnight with SpeI and tommorrow in the morning XbaI will be added.
</LI><LI> I (Maddie!) performed steps towards the ligation (XylE-3K3) again from scratch (inefficient cutting steps??) including digest, gel analysis and extraction. 
</LI><LI> I ligated PSBC3-B0014 and XylE
</LI><LI> I picked successfully transformed colonies for a replica plate and set up overnight mini cultures of 1,2,3 ComC DE promoter FWD and 4,5,6 ComC promoter Rev
</LI></UL><P><B>Thurs 23rd Sept First day of AUTUMN!!</B></P><UL><LI>I determined the ligation ratio for XylE and 3k3 (1:3) performed dephosphorylation of vector then ligation, to be transformed along with the other ligation tomorrow hopefully. 
</LI><LI>I did mini preps of the Comc DE promoters, performed a ES test digest and am currently waiting for gel results. I shall then set up overnight midis.
</LI><LI>Miniprep results were fabulous, Ive chosen colony 1 and colony 4 which are ComCDE promoters FWD and REV respectively. They are made up into overnight midi cultures for tomorrow.
</LI></UL><P><B>Fri 24th Sept</B></P><UL><LI>Midi preps of ComC DE promoters FWD and REV (colonies 1,4) 
</LI><LI>to be followed by a digest. XbaI Pst for FWD (not known for the reverse currently) to insert XylE after it. 
</LI><LI>Ligation with XylE and PSB1C3-B0014 possibly.
</LI><LI>Transformation of XylE-PSB1C3-B0014 and of XylEe-3k3 into <I>E.coli</I> later
</LI><LI> Hi Nick, hope revision is going well. Love from Team XylE xox
</LI></UL></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Week 13
</TD></TR><TR><TD><P><B>Monday 27th Sept</B></P><UL><LI> Successfull transformations: now have XylE-PSB1C3-B0014 (no promoter) and 3K3-J23101-XylE-B0014. No background plate colonies..many colonies for 3K3-XylE and a few for PSB1C3-XylE. Colonies have been replica plated (plus catechol assay plate for 3K3-XylE) and set up for mini prepping tomorrow.  
</LI><LI> Sumo tagged XylE will be given to us by Kirsten at some point this week and we will have to purify it. This can then be used for in vitro testing and comparisons made with the linker-XylE protein. 
</LI><LI> pVeg and ComC DE promoter FWD can be added into the promoter-less construct after it has been midi prepped and digested accordingly. ComC DE promoter REV can be added onto the reverse XylE construct (PCR'd by Piotr) AFTER it has been put into PSB1C3 and is more manageable. Kirsten is taking care of the GFP-XylE fusion for now, including putting the pVeg promoter plus RBS infront of it (proving difficult.) We still need to receive LacI promoter to create the inducible test version of the fusion XylE-GFP protein.
</LI></UL><P><B>Tues 28th Sept</B></P><UL><LI> Performed mini preps and then E+S / E+S AND AseI digests of colonies 1-7 3k3-XylE and 1-3 C3-B0014-XylE respectively. Gel runs showed them to be the correct thing :)))))) so, 3k3-XylE final vector is now left as 2-7 minis (catechol assay test showed colony 1 was background so DO NOT USE) in my DNA box (orange tubes). Colony 1 of the C3-B0014-XylE was selected for midi prepping and will be performed tomorrow. 
</LI></UL><P><B>Weds 29th Sept</B></P><UL><LI> Midi prep of XylE-GFP fusion protein along with C3-B0014-XylE. The C3-B0014-XylE midi will then be used in 2x cloning digests (E+S) to insert pVeg and ComC DE FWD promoters. These will be obtained and also digested E+S to get out of their current vectors a gel analysis and purification will be run of all 3 in parallel, followed by a ligation ratio gel analysis, dephosphorylation of vector then ligation.
</LI><LI> Primers for J23101-XylE arrive today so they can be used to PCR out J23101-XylE (From my J23101-Xyl-E midi) with blunt ends and this will be ligated into the final Spec vector so that we can get some data with Bacillus.
</LI><LI> J23101-XylE-B0014 XylE-3K3 and PSB1C3-B0014-XylE were all sent off for sequencing
</LI></UL><P><B>Thurs 30th Sept</B></P><UL><LI> I gel purified ComCDE FWD promoter, C3-XylE-B0014, pVeg vector and XylE-B0014. Did a gel analysis to get ligation ratio. 
</LI><LI> Set up midi culture of J23101-XylE-B0014 colony 8
</LI><LI> Set up midi culture to repeat GFP-XylE midi prep
</LI><LI> Set up 6x minis (plus replica plate) of final Spec colonies 1-6
</LI><LI> PCR reaction using newly arrived primers to get blunt ended J23101-XylE.
</LI></UL><P><B>Friday 1st Oct</B></P><UL><LI>morning maddie, from earl's court! :)
</LI></UL><P>ahahahaha awesome. hey you :)
</P><UL><LI> Today I peformed ligations for ComCDE FWD promoter and PSB1C3-XylE-B0014 and also PSB1C3-pVeg and XylE-B0014 they will be left overnight and transformed by chris tmro thaaanks.
</LI><LI> I ran the PCR Kirill performed yday to get out blunt ended J23101-XylE on a gel; there was no template DNA! so I've just set up the reaction again... hopefully i'll purify it later for ligation into the final Spec vector that kyasha has maaaaade :))) then we can test XylE in <I>Bacillus</I>.
</LI><LI> I'm currently performing Midi's of my J23101-XylE-B0014 (running low) and of GFP-XylE (3rd time!)
</LI></UL></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD><P>have a look :)
it's from John Hoppkins wiki 2008. There are one or two good ones!
</P><P>Love: Before I heard the doctors tell The dangers of a kiss;
</P><P>I had considered kissing you. The nearest thing to bliss.
</P><P>But now I know biology and sit and sigh and moan;
</P><P>six million mad bacteria and I thought we were alone!
</P><P>
Biology is the science of moving tiny droplets of invisible liquids from one tube to another.
</P><P>
The human body was designed by a civil engineer. Who else would run a toxic waste pipeline through a recreational area?
</P><P>
I have a hunch that the unknown sequences of DNA will decode into copyright notices and patent protections.
</P><P>
It has recently been discovered that research causes cancer in rats.
</P><P>
A mouse is an animal that, if killed in sufficiently many and creative ways, will generate a PhD.
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Week 14
</TD></TR><TR><TD><P><B>Monday 4th Oct</B>
Did a replica plate and catechol assay plate/ minis 2 6 7 9 12 16. (Successful pVeg-XylE transformation)
</P><P><B>Tues 5th Oct</B>
4 16 colonies were background the rest turned yellow. Mini prepped 2 6 7 9 12 16. Made overnight assay cultures of 3k3-XylE and C3-xylE
</P><P><B>Weds 6th Oct</B></P><UL><LI> Digested mini preps 2 6 7 9 12 16 with E+S will run on gel to determine if it's the correct plasmid, then will select a colony to midi prep tomorrow.
</LI><LI> Will perform the same assays already performed by Nick with the 3k3-XylE constructs to compare results.
</LI><LI> Just realised Im working with background colony 16!! disregard this after the gel run!
</LI><LI> I'm currently keeping my undigested pVeg-XylE minis in Florian's DNA box (orange lids and labels)
</LI></UL><P><B>Thursday 7th Oct</B></P><UL><LI> Labpartner, goodmorning and from greece!! :p
</LI></UL><UL><LI> Niiiiiiiiiiiiiiick!!! you left the country! you were too ugly. good luck babe!
</LI></UL><UL><LI> Midi prepped pVeg-XylE-B0014 in PSB1C3; will digest out the insert (E+S) and put it into 3K3(E+X). Then both the midi (in PSB1C3) and the ligation 3k3-pVeg-XylE-B0014 can be transformed into TOP10. J23101-XylE-B0014 in 3K3 Vector also needs to be put into TOP10 currently only the PSB1C3 version is in TOP10. We need the rest for comparison testing as TOP10 is the more widely used testing strain.
</LI><LI> Florian kindly digested pVeg-XylE-B0014-C3 with E+S
</LI><LI> Ran PCR to get out blunt pVeg-XylE-B0014 using primers. Did 3 reactions at variations around 60 degrees. 
</LI></UL><P><B>Friday 8th Oct</B></P><UL><LI> Ran gel analysis of my PCRs 1, 2 and 3* appear to have worked. Samples 1 and 3* show stronger bands and so should be used in the next ligation step into the SPEC vector (which kyasha will be doing) then transformation into <I>Bacillus</I> (meeee.)
</LI><LI> Chris is purifying digested pVeg-XylE-B0014-C3 to get the insert out and this will then be ran on a gel alongside cut (E+X) 3k3 to determine a ratio for ligation.
</LI><LI> Meeting today at 4pm, no one is here!! it'll be me Florian, Ben and Piotr attending..yikes
</LI></UL><P><B>Sunday 10th of the 10th of the 2010!!!!! and im in LAB </B></P><UL><LI> I did ligations of pVeg-XylE into 3k3 and of Reverse-XylE into the digested C-Tev LacI vector. 
</LI><LI> I also set up assay cultures for monday 2x pVegXE 2x XE-C3 2x XE-3K3 and 2x GFP-XylE!!!
</LI></UL></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Week 15
</TD></TR><TR><TD><P><B>Monday 11th Oct</B></P><UL><LI>No Nicolas Kylilis, what a douche!
</LI><LI>Diluted my overnights 250μl into 5ml to grow up to around 0.5 OD
</LI><LI>Gonna perform assays with them this afternoon after some growing time :)))))))
</LI><LI>Doing data analysis with my last assay results comparing high copy (1C3) vs low copy (3K3)and the effects on XylE activity..boring
</LI><LI>Finalising the polo-shirt logo design, it's looking fiiiine, we're gonna look goooood ;)
</LI></UL><P><B>Tues 12th Oct</B></P><UL><LI>Transformations worked! we now have pVeg-XylE in 3k3 and reverse-XylE with lacI promoter (C-TEV vector)i did replica plates, catechol assay plates and overnights for minis. Will mini prep tmro and transform them into TOP10 and we should be good to go with all constructs. 
</LI><LI> Assays messed up, my fault :( 3k3 didnt respond to catechol, so i think that I picked a background colony, i wont pick it again. The results are a bit ridiculous and will probably be chucked.
</LI></UL><P><B>Weds 13th Oct</B>
mini prepped 3k3-pVeg-XylE-B0014 and went to the school workshop! set off overnight midi of culture 9
</P><P><B>Thurs 14th Oct</B>
Midi prepped 3k3-pVeg
</P><P><B>Fri 15th</B>
Transformed all my constructs into T0P10 3k3-J23101/pveg and PSB1C3-J23101/pVeg
</P></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"> Output Photo Gallery
</TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">First Successful catechol assay :)))</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">First successful catechol assay</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Experiment which determined the yellow product to be exported out of the cells into the media and not retained by the cells</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Cell vs supernatant assay</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Cuvette assay set up</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">:-)</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">final batch of transformed XylE constructs into TOP10s</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fusion GFP-XylE vs XylE cuvette response assay</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">View from above</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Plate after assay!</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Imperial_College_London/Lab_Diaries/XylE_team">http://2010.igem.org/Team:Imperial_College_London/Lab_Diaries/XylE_team</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Lab_Diaries/XylE_team" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Lab_Diaries/XylE_team" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Lab_Diaries/XylE_team&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Lab_Diaries/XylE_team&amp;oldid=207329" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>