<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_London_Modules_Fast_Response"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College_London/Modules/Fast_Response">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College_London/Modules/Fast_Response&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Modules/Fast_Response&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College_London/Modules/Fast_Response&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College_London/Modules/Fast_Response" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College London/Modules/Fast Response</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="title"><DIV id="belink"><A href="http://www3.imperial.ac.uk/bioengineering">Department of Bioengineering</A></DIV><DIV id="mblink"><A href="http://www3.imperial.ac.uk/molecularbiosciences">Division of Molecular Biosciences</A></DIV></DIV><DIV id="menu"><DIV id="menuback"><UL id="menulist"><LI><A href="#" class="listhead" id="project">Project</A></LI><LI><A href="#" class="listhead" id="plan">Plan</A></LI><LI><A href="#" class="listhead" id="results">Results</A></LI><LI><A href="#" class="listhead" id="extra">Extras</A></LI></UL></DIV><DIV id="headpic"><DIV id="headover"><P id="headtit">Parasight<SPAN class="dark"> <SPAN class="highlight">|</SPAN>  Parasite detection with a rapid response</SPAN></P><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London" class="listmain" id="results">Home</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Tour/Page_One" class="listmain" id="results">Tour</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modules" class="listmain" id="results">Modules</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Chassis" class="listmain" id="results">Chassis</A></LI><UL class="project"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Human_Practices" class="listmain" id="results">Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Schistosoma" class="listmain" id="results">Schistosoma</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Research" class="listmain" id="results">Research</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Diary/Week_One" class="listmain" id="project">Diary</A></LI></UL></UL><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Modelling" class="listmain" id="results">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Strategy" class="listmain" id="results">Assembly</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Lab_Diaries" class="listmain" id="results">Lab Diaries</A></LI><UL class="plan"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Protocol" class="listmain" id="results">Lab Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Safety" class="listmain" id="results">Safety</A></LI></UL></UL><UL class="results"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Results" class="listmain" id="results">Experimental Results</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Parts/Favourites" class="listmain" id="results">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/The_Team" class="listmain" id="results">The Team</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Media/Videos" class="listmain" id="results">Media</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/School_Workshops" class="listmain" id="results">School Workshops</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Software_Tool" class="listmain" id="results">Software Tool</A></LI><UL class="extra"><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Acknowledgments" class="listmain" id="results">Acknowledgments</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Brainstorming" class="listmain" id="results">Brainstorming</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Glossary" class="listmain" id="results">Glossary</A></LI><LI><A href="https://2010.igem.org/Team:Imperial_College_London/Sitemap" class="listmain" id="results">Sitemap</A></LI></UL></UL></DIV></DIV></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:25px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Modules
</TD><TD style="font-family: helvetica, arial, sans-serif;font-size:1.5em;color:#ea8828;" align="right"><A href="/Team:Imperial_College_London/Modules" title="Team:Imperial College London/Modules">Overview</A> | <A href="/Team:Imperial_College_London/Modules/Detection" title="Team:Imperial College London/Modules/Detection">Detection</A> | <A href="/Team:Imperial_College_London/Modules/Signaling" title="Team:Imperial College London/Modules/Signaling">Signaling</A> | <STRONG class="selflink">Fast Response</STRONG></TD></TR><TR><TD colspan="2"><B>Our design consists of three modules; Detection, Signaling and a Fast Response, each of which can be exchanged with other systems. We used a combination of modelling and human practices to define our specifications. Take a look at the overview page to get a feel for the outline, then head to the full module pages to find out how we did it.</B></TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Fast Response Module Output Overview
</TD></TR><TR><TD>The third component of our bacterial detector is a fast response reporter module that upon detection of an input signal gives out a yellow signal output visible by the naked eye. Reporters in biochemical research traditionally rely on transcription-translation of a specific reporter molecule (e.g. green fluorescent protein) which can be time consuming and/or might involve sophisticated equipment for its detection. In this module we introduce a novel reporter system. This system involves the expression of a low molecular weight protease which acts on an pre-existing pool of substrate and facilitates a double amplification step. This should result in an output signal several orders of magnitude faster than traditional reporters.
<P>An overview of our system can be found below followed by a more in-depth analysis of the system.  
</P></TD></TR></TBODY></TABLE></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">Video of a C230 assay!
</TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;" colspan="2"> C230 and C230-GFP fusion
</TD></TR><TR><TD>This figure illustrates the point that Catechol-2,3-dioxygenase enzyme is active only when it tetramerises
</TD></TR></TBODY></TABLE></DIV><P>Catechol (2,3) dioxygenase (C23O) is the protein product of the XylE gene. It originates from <I>Pseudomonas putida</I> and the active protein is made up of a homotetramer of C2,3O monomers. The enzyme catalyses the conversion of a colourless substrate (catechol or substituted catechols) into a bright yellow product (2-hydroxymuconic semialdehyde) within seconds of substrate addition.
</P><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD>In our project, we take advantage of the fact that C23O is only active as a tetramer as the active site forms as a result of intersubunit and interface interactions after oligomerisation. We have constructed a gene product that encodes a fusion protein. This fusion protein is made of GFP fused to the N-terminus of C23O. The GFP does not allow C23O monomers to associate due to a steric hindrance effect, so that the active tetramer cannot form. Instead the fusion protein remains as a monomer in the cytosol of the cell.
</TD></TR><TR><TD>GFP is N-terminally added to Catechol-2,3-dioxygenase enzyme so that it does not allows formation of the tetramer due to a steric hindrance effect. This means that the enzyme is inactive as it must remain in its monomer form
</TD></TR></TBODY></TABLE></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD>Detection of the parasite results in a signal in the form of the phosphorylated transcription factor ComE. This induces expression of TEV protease, which cleaves fusion protein to activate the system
</TD></TR></TBODY></TABLE><P>A very important feature of the fusion protein is its potential for induction of the system. A closer look at the fusion gene construct reveals the secret of the inducibility of our system. The GFP gene is fused to the XylE gene through a protein sequence that is recognised and cleaved by TEV protease. This amino acid sequence is recognised by the active site of the TEV protease so that when TEV is present in the cell, GFP is cleaved off C23O monomers. The free monomers are no longer sterically hindered by the GFP and are now able to associate with each other. Tetramerisation of the monomers produces the active C23O enzyme, which in the presence of catechol substrate produces a yellow colour.
</P><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD>Our GFP-XylE gene construct also has some other addition features, which make it easier to work with. It contains an N-terminal His-tag which would assist in purification and for <I>in vitro</I> characterisation. The same applies to the flag tag sequence which can be recognised by monoclonal antibodies for easier detection and purification. The linker sequence is there to ensure that the TEV recognition sequence is readily accessible to the protease for cleavage.
</TD></TR><TR><TD>The GFP-XylE gene construct
</TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"> TEV protease
</TD></TR><TR><TD>Protein Structure Model of TEV protease
</TD></TR></TBODY></TABLE><P>TEV protease is a natural polyprotein cutting viral protease. In our system ComE transcription factors bind to their native ComCDE promoter which was engineered as the promoter for TEV protease. This means that TEV protease is <B>only</B> transcribed and translated upon detection of the <I>Schistosoma</I> parasite.  
</P><P><B>Why TEV protease?</B> 
This enzyme has been used extensively in protein engineering as it has a number of key qualities including a high specificity (ENYLFQG), and importantly for our system it has a relatively low molecular weight (242 aa) compared to other protease candidates available. This means that the synthesis of the protease will be relatively quick. 
For more information about TEV, please see the PDB file with the accession number <B>1Q31</B>.
TEV protease expression in <I>E.Coli</I> (and consequently <I>B .subtilis</I>) is made efficient by codon optimisation and inducing expression of additional tRNAs.
</P><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;" colspan="2">Fast &lt; faster &lt;<I>Imperial College Turbo FAST</I></TD></TR><TR><TD><P>So what are the key features that make our system a novel example of a fast response mechanism?
</P></TD><TD style="font-family: helvetica, arial, sans-serif;font-size:1.5em;color:#ea8828;"><A href="/Team:Imperial_College_London/Modelling" title="Team:Imperial College London/Modelling"> Modelling</A></TD></TR><TR><TD><UL><LI>Due to the small size of the TEV gene, it is likely to be transcribed and translated very quickly and therefore will not introduce limiting kinetic factors within our fast response system. 
</LI></UL></TD></TR></TBODY></TABLE><UL><LI>TEV protease would then act upon a pre-existing pool of inactive substrate already present in the cell. This substrate (GFP-C23O fusion protein) is an inactive form of the chosen output enzyme C23O. Since TEV protease is a relatively fast enzyme, it has a high turnover rate of substrate molecules per unit of time. This is <B>AMPLIFICATION STEP #1</B>.
</LI></UL><UL><LI>The now activated C23O is itself an enzyme itself with a high turnover rate. As a consequence, a vast number of catechol molecules would be converted to yellow coloured product per unit of time. <B>This is AMPLIFICATION STEP #2</B>.
</LI></UL><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">C23O Additional Material
</TD></TR><TR><TD>For our fast response module we have taken advantage of the tetramerization feature which produces the functioning C230 enzyme; the active site forms as a result of intersubunit and interface interactions after oligomerization;
</TD></TR></TBODY></TABLE>After observing C23O in Pymol, and taking into account additional information obtained (Kita <I>et al</I>),  which shows that the projecting loop is needed for dimerisation and that this is very near to the N-terminus, we chose to engineer (by straightforward DNA synthesis) GFP plus a linker (GGGSGGGS ENYLFQG) onto the N-termini of our C230 monomers.
<P>Below is the reaction performed by our system’s enzyme:
</P><DIV align="CENTER"><TABLE style="width:504px;background:#e7e7e7;text-align:center;font-family: helvetica, arial, sans-serif;color:#555555;margin- top:5px;padding: 2px;"><TBODY><TR><TD>Image taken from <A href="ftp://ftp.genome.jp/pub/kegg" class="external free" rel="nofollow">ftp://ftp.genome.jp/pub/kegg</A> ; Where catechol is the colourless substrate converted by ring cleavage into 2 hydroxymuconate semialdehyde.
</TD></TR></TBODY></TABLE></DIV><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;">And one more assay....
</TD></TR></TBODY></TABLE><TABLE style="width:900px;background:#f5f5f5;text-align:justify;font-family: helvetica, arial, sans-serif;color:#555555;margin-top:5px;" cellspacing="20"><TBODY><TR><TD style="font-family: helvetica, arial, sans-serif;font-size:2em;color:#ea8828;"> References
</TD></TR><TR><TD><P>[1] 'An archetypical extradiol-cleaving catecholic dioxygenase: the crystal structure of catechol 2,3-dioxygenase (metapyrocatechase) from Pseudomonas putida mt-2' 1999 Akiko Kita, Shin-ichi Kita, Ikuhide Fujisawa, Koji Inaka, Tetsuo Ishida, Kihachiro Horiike, Mitsuhiro Nozaki and Kunio Miki
</P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Imperial_College_London/Modules/Fast_Response">http://2010.igem.org/Team:Imperial_College_London/Modules/Fast_Response</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College_London/Modules/Fast_Response" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College_London/Modules/Fast_Response" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College_London/Modules/Fast_Response&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College_London/Modules/Fast_Response&amp;oldid=204413" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>