"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College London/Lab Diaries/Surface protein team
From 2010.igem.org
Department of Bioengineering
Division of Molecular Biosciences
Project
Plan
Results
Extras
Parasight  |  Parasite detection with a rapid response
Home
Tour
Modules
Chassis
Human Practices
Schistosoma
Research
Diary
Modelling
Assembly
Lab Diaries
Lab Protocols
Safety
Experimental Results
Parts
Achievements
The Team
Media
School Workshops
Software Tool
Acknowledgments
Brainstorming
Glossary
Sitemap
Lab Diaries
Overview | Surface Protein Team | XylE Team | Vectors Team | Modelling Team
Here are the technical diaries for our project. We've split them up into three lab teams and the modelling team. We think it's really important that absolutely anyone can find out what we've been doing. For a really detailed look at what we did, and when, you've come to the right place!
Surface Protein Team
Our aim was to assemble the surface protein construct, consisting of the LytC cell wall binding domain, CWBD), the linker containing a protease cleavage site and the autoinducing peptide (AIP). This protein would be under the control of the pVeg promoter.
Here's a picture of the final construct:
Week 6
Day
Monday
Tuesday
Wednesday
Thursday
Friday
Morning
Gel electrophoresis of pSB1C3 PCR product between EcoRI site and PstI
PCR Purification of pSB1C3
Afternoon
Restriction digest of pSB1C3 and pSB1AK3 with EcoRI and PstI.
Friday, 13th-Aug-2010
Plan:
Perform gel electrophoresis to confirm that the pSB1C3 PCR between the EcoRI and the PstI site worked.
If PCR worked pSB1C3 will be PCR purified
Restriction digest of purified pSB1C3 and pSB1AK3 with PstI and EcoRI
Gel electrophoresis and gel purification of excised double terminator BOO14
If possible, set up ligation over the weekend
Report:
Analyzing the PCR product with gel electrophoresis, we observed -as expected- a 2kb fragment on the gel. This confirms that the PCR of pSB1C3 was successful as the vector is approximately 2kb long.
Having confirmed that the PCR was successful we purified the PCR product from the solution using the E.Z.N.A. Cycle Pure Kit (Omega bio-tek) but used 25µl of ddH2O instead of elusion buffer in the last step.
We then did a restriction digestion for:
The pSB1C3 PCR product using the enzymes PstI and EcoRI in buffer 4
pSB1A3 with B0014 using the enzymes PstI and EcoRI also in buffer 4
Our last step was to use gel electrophoresis to separate the product of the restriction digest to confirm successful digestion and purification. Unfortunately we lost the excised BOO14 during electrophoresis so digestion of pSB1AK3 has to be repeated.
Saturday, 14th-Aug-2010
Plan:
Repeat digestion of pSB1AK3 with PstI and EcoRI
Gel electrophoresis of the digestion products to separate BOO14 and the vector to isolate BOO14
Report:
Digestion of pSB1AK3 with PstI and EcoRI was performed to excise the terminator BOO14.
Gel electrophoresis was successfully used to separate the digestion products pSB1Ak3 and BOO14, although yield of the later appears to be relatively low probably due to the short length of the terminator. The band of BOO14 was cut out of the gel.
Week 7
Day
Monday
Tuesday
Wednesday
Thursday
Friday
Morning
Gel purification of BOO14
Determine concentrations of BOO14 and pSB1C3
Restriction digest of pSB1C3 with EcoRI and PstI
Work on Wiki
Check plates with transformants for colonies
Perform colony PCR to confirm if ligation and transformation were successful
Prepare a replica plate
Set up of pSB1C3-BOO14 cultures for mini preps
Set up PCR reaction with potentially contaminated agents
Gel electrophoresis of PCR products to determine which component is contaminated with DNA
Mini prep to isolate pSB1C3-BOO14
Repeat gel electrophoresis of pSB1C3-BOO14 restriction digest
Perform PCR to amlify CWB out of the ''B. subtilis'' genome
Afternoon
Dephosphorylation of pSB1C3
Set up ligation of pSB1C3 and BOO14 over night
Transformation of ''E. coli'' with ligation product pSB1C3-BOO14
Analysis of colony PCR using gel electrophoresis
Meeting with the supervisors
Repeat colony PCR with uncontaminated reagents
Analysis of PCR product with gel electrophoresis
Restriction digest with EcoRI and SpeI to confirm correct insert (BOO14)
Gel electrophoresis to analyse restriction fragments
Gel electrophoresis of the PCR product to confirm correct amplification
Set up midi prep cultures of pSB1C3-BOO14
Monday, 17th-Aug-2010
We successfully purified BOO14 from the gel using the QIAquickÆ Gel Extraction Kit (250) but used 50µl of ddH2O instead of elusion buffer in the last step.
Then we determined the relative concentrations of BOO14 and pSB1C3 by gel electrophoresis and comparing the intensity of the bands with the 1kb ladder.
Having determined the rough concentration of our DNA we set up over night ligation of pSB1C3 with BOO14. We used two different ratios of pSB1C3 to BOO14: 0.5µl:3.5µl and 1µl:3µl. As the final concentration of the ligation product is very low, we will transform E. coli to be able to analyse bigger quantities of the vector at a later point.
Tuesday, 18th-Aug-2010
E. coli was transformed with the pSB1C3-BOO14 ligation using chemical competence and heat shock. We prepared plates with our transformants to be incubated overnight.
We reorganized and updated our Lab-Page on the Wiki, including new tables, upload of pictures and result as well as user-interface optimisation.
Wednesday, 19th-Aug-2010
The plates with pSB1C3-BOO14 transformants had colonies growing on it.
Some of these colonies were tested by colony PCR to confirm that BOO14 was in the vector.
Gel electrophoresis of the PCR product indicated that our ligation and transformation were successful, however due to contamination, as demonstrated by our negative control, we will have to repeat the colony PCR on Thursday. We set up an additional PCR to test the individual components of our PCR reaction to find out which one is contaminated with DNA.
We also set up cultures of pSB1C3-BOO14 for mini preps tomorrow.
Thursday, 20th-Aug-2010
Analysis of the PCR components indicated that the Barns buffer was contaminated with DNA.
We prepared mini preps of pSB1C3-BOO14 from the cultures set up yesterday.
We then repeated the colony PCR and analysed the product with gel electrophoresis. We observed a band at 100bp, which indicates that the ligation was successful, and that BOO14 is in pSB1C3.
We tried to confirm this with a restriction digest of the mini prepped plasmid with EcoRI and SpeI, however, analysis with gel electrophoresis showed there was only a very faint band on the gel at 100bp maybe because the gel had been run too long.
Friday, 21st-Aug-2010
We repeated the gel electrophoresis but reduced the time it ran for to 10 minutes as BOO14 is very short. However we still only observed a very faint band at 100bp, so we can't be completely sure that the ligation was successful.
Therefore we carried out another restriction digest with AseI (which cuts within B0014) and NcoI (which cuts within pSB1C3).
We also performed a PCR to amplify LytC cell wall binding domain (CWB) from the Bacillus genome.
Analysis of the PCR product with gel electrophoresis showed that lytC had not been amlified properly. There we will do a series of different PCR reaction tomorrow to determine the optimal temperatures.
Sunday, 22nd Aug 2010
Analysis of the restriction fragments from yesterdays restriction digest with gel electrophoresis showed that:
The ligation definitely worked! The correct fragment sizes were observed on the gel.
The primers in the PCR may have annealed nonspecifically, because the PCR product did not resolve properly in the gel.
To prepare the ligated plasmid pSB1C3-BOO14 for midi prep tomorrow, overnight cultures were set up using 100ml LB (containing chloramphenicol), which was innoculated with cells from the original replica plates.
A second PCR was set up with a gradient of increasing annealing temperatures, which would hopefully ensure specific binding of primers. (We used taq in this case, just to see which temperature gave us the best result)
The gel of the PCR products showed that all 3 PCRs were successful. So we can now use Pfu in the next PCR to obtain the LytC cell wall binding domain.
Week 8
Day
Monday
Tuesday
Wednesday
Thursday
Friday
Morning
Midi prep of pSB1C3-BOO14
PCR of LytC using Pfu polymerase
DpnI restriction digest of PCR product
Restriction digest of LytC with XbaI
Midi prep of pSB1C3-BOO14
Gel electrophoresis to confirm correct midi prep product
Restriction digest of pSB1C3-BOO14 with XbaI and SpeI
Gel electrophoresis to measurement of relative concentrations of pSB1C3 and LytC after restriction digest
Dephosphorylation of pSB1C3
Ligation of pSB1C3 with LytC
PCR of promoter pVeg out of vector using taq
Screen plates for successful transformation
Gel electrophoresis of pVeg PCR sample 2 and PCR purified sample 1
Afternoon
Gel electrophoresis of LytC
Gel purification of LytC
Midi prep of pSB1C3-BOO14
Overnight digestion of LytC with SpeI
Set up new pSC1C3-BOO14 culture for midi prep
Restriction digest with XbaI and SpeI to confirm BOO14 in pSB1C3
Gel purification of restriction product
Gel electrophoresis to analyse restriction product
Ligation for transformation
PCR of LytC using Pfu for high fidelity
Restriction digest of pVeg with EcoRI and XbaI
Monday, 23rd Aug 2010
We successsfully performed PCR using Pfu polymerase to amplify the LytC CWB domain.
The PCR product was then digested with DpnI to remove template DNA and then gel purified.
pSB1C3-BOO14 was midi prepped, and the final DNA concentration was measured to be 120ng/µl.
Overnight digestion of LytC with SpeI was set up because restriction will be inefficient on the PCR product. XbaI will be added to the mixture in the morning making the final volume of the digestion 30µl.
Tuesday, 24th-Aug-2010
XbaI was added to the restriction digest of LytC. PCR purification was performed to isolate the CWB.
Due to problems with our midi-prep of pSB1C3, we only digested the two components but then had to set up another culture of B. subtilis for another round of midi prep.
Wednesday, 25th-Aug-2010
We performed another midi prep using the culture of pSB1C3-BOO14 set up yesterday.
After a restriction digest of the plasmid with XbaI and SpeI and consequent gel purification the product was analysed with gel electrophoresis and the plasmid was found to contain BOO14 so that this midi prep can be used for the following steps.
Thursday, 26th-Aug-2010
We performed a restriction digest of pSB1C3-BOO14 with XbaI and SpeI.
We then determined the relative concentrations of LytC and pSB1C3 to prepare ligation. We found that pSB1C3 had a about 4 time higher concentration than LytC.
We set up a 10µl dephosphorylation reaction with 2µl of pSB1C3 and incubated for 10min followed by heat deactivation of the T4 alkaline phosphatase.
2.5µl of the dephosphorylation reaction were used together with 2µl of LytC for a 10µl ligation reaction. This was split into a 5µl overnight ligation and a 5µl bench ligation reaction.
After 2 hours incubation at room temperature we transformed E. coli with the ligated vector and later plated it.
We set up a test PCR using taq polymerase to determine the best protocol to get the promoter pVeg out the current vector pSB1AK3.
Gel electrophoresis of the PCR products suggested that the following protocol was most suitable:
5 cycles at 60∞C
25 cycles at 66∞C
We set up another PCR reaction using Pfu for to obtain two identical samples of pVeg with high fidelity
Sample 1 was later PCR purified by Kirill and Kyascha (thanks guys!)
Friday, 27th-Aug-2010
Both samples of pVeg were successfully puified and can both be used in the following cloning steps
Our transformation was unsuccessful and no background was observed either. We thus used our overnight ligation to transform E. coli again.
We decided to digest the promoter with EcoRI and SpeI rather than EcoRI and XbaI as originally planned. Digestion with SpeI was set up overnight but the digestion volume was accidentially made up to a total of 30µl rather than 28.5µl allowing for EcoRI to be added the next day.
Saturday, 28th-Aug-2010
Transformation of E. coli using our overnight digestion was unsuccessful which means we have to restart construction of our LytC vector.
EcoRI was added to the pVeg digestion and both samples were incubated for 1 hour
5µl of sample 1, which is to be PCR purified later, were loaded on a gel and 30µl (all) of sample 2, which is to be gel purified. The electrophoresis confirmed that pVeg had been digested and the bands of smaple 2 (two lanes in total) were cut out of the gel for gel purification (0.44g).
Week 9
Day
Monday
Tuesday
Wednesday
Thursday
Friday
Morning
Gel Purify pVeg (digested E+S) sample 2
Digestion of pSB1C3 EcoRI + SpeI
Gel electrophoresis of digestion product
Preparation of LB agar
Measuring DNA concentration in 5 midi-preps
Update Wiki
Update Wiki
Measure midi prep concentrations in the Biochem building
Check for successful transformations
If we have transformants: # Make a replica plate # Perform a colony PCR
Afternoon
Gel purification of pSB1C3
Gel electrophoresis to determine relative concentrations of insert and vector
Set up of overnight ligation
Meeting with supervisors
Try to measure midi prep concentration
Run gel electrophoresis to analyze colony PCR results
Monday, 30th-Aug-2010
Both sample of pVeg which had been digested with EcoRI and SpeI were purified: Sample 1 was PCR purified (by another team) and sample 2 gel purified. Two methods of purification were used because pVeg is so small that either method posed the risk of loosing our product.
Gel electrophoresis later confirmed that both samples were purified successfully.
We also set up a PCR of LytC for blunt ended ligation as an alternative to our previous cloning strategy.
Tuesday, 31st-Aug-2010
Plan:
Restriction digest of pSB1C3 EcoRI and SpeI
Gel electrophoresis of digestion product
Gel purification of pSB1C3
Gel electrophoresis to determine concentrations of insert and vector
Dephosphorylation of pSB1C3
Set up of overnight ligation
Report:
Restriction digest of pSB1C3-BOO14 was performed with EcoRI and SpeI to remove the terminator and later allow ligation with pVeg.
Gel electrophoresis was used to confirmed that digestion of pSB1C3-BOO14 with EcoRI and SpeI was successful.
The digested vector was then gel purified and the relative concentrations of the digested pSB1C3 and pVeg were determined. Our samples contained about 30 times as much vector as promoter and the vector is about 20 times larger than the insert.
therefore we decided to ligate overnight in a ration of 1.5 volumes of pVeg to 1 volume of dephosphorylated vector.
LytC was digested overnight with SpeI.
Wednesday, 1st-Sep-2010
We performed another restriction digest of pSB1C3-BOO14 using XbaI and SpeI.
XbaI was added to the restriction digestion of LytC.
Correct digestion of LytC and pSB1C3-BOO14 was confirmed using gel electrophoresis and pSb1C3 as well as LytC were then gel purified. All planned steps worked and overnight ligation of pSB1C3 and LytC were set up.
We also prepared some LB agar today and had a meeting with the supervisors.
Thursday, 2nd-Sep-2010
We tried to measure the concentration of 5 midi preps Chris had made of linkers from synthesis. However the spectrophotometer caused great problem as it continuously, and even when operated by different people, produced wrong and inconsistent results as well as failing to remain callibrated.
We therefore used gel electrophoresis to determine the sample concetrations (2 lanes each, 1µl and 5µl).
However gel electrophoresis also yielded bad results so we will measure concentrations tomorrow using a different method.
We spend some time updating the Wiki today.
Friday, 3rd-Sep-2010
We finally measured the concentration of DNA in the midi preps (of plasmids containing synthesis products). The concentrations varied between 78ng/µl and 230ng/µl and one will have to be repeated as the yield was too low.
The transformation of the LytC-pSB1C3 was had produced many colonies. 20 colony PCRs were set up and a replica plate made.
Unfortunately gel electrophoresis suggested that the ligation was not successful, so another round of gel electrophoresis will be performed on monday to confirm these results.
Week 10
Day
Monday
Tuesday
Wednesday
Thursday
Friday
Morning
Repeat gel electrophoresis of 7th colony PCR
20 further colony PCRs from the LytC transformants.
Mini preps from overnight cultures.
Amplification of pVeg using PCR from pSB1AK3
Gel electrophoresis to confirm correct amplification of pVeg
Excision and gel purification of pVeg
XbaI was added to overnight digestion of pVeg with SpeI
Gel electrophoresis to analyse pVeg restrcition digest
Because pVeg was lost during the last steps PCR amlification of pVeg from PSB1AK3 was repeated.
PCR purification of pVeg
Mini prep using overnight cultures of pSB1C3-LytC
Restriction digest with EcoRI and SpeI as well as XbaI and SpeI for all 23 mini preps of pSB1C3-LytC
Culture of K5 for midi prep failed: New culture was set up
Afternoon
Set up different mini prep cultures
Gel electrophoresis to analyse colony PCR
Restriction digest of pSB1C3-LytC mini preps with XbaI and SpeI
Gel electrophoresis to analyse restriction fragments
Restriction digest of pVeg with XbaI and SpeI
Gel electrophoresis of restriction fragments for analysis
Diagnostic digest of pVeg was repeated with SpeI overnight
Restriction digest of LytC in blunt-ended ligation vector
Set up of midi prep cultur from K5
Set up 23 mini prep cultures of pSB1C3-LytC
Test culture was set up to test new chloramphenicol aliquot
Repeat of digest of pVeg with SpeI overnight (EcoRI will be added tomorrow)
Monday, 6th-Sep-2010
We repeated the gel electrophoresis for the 7th colony PCR, because it showed a faint band around 1kb, indicating that LytC was present in pSB1C3.
Simultaneously we set up another 20 colony PCRs for colonies from the pSB1C3-LytC ligation/transformation and made a replica plate.
We also set up mini prep cultures from (a) the blunt-ended ligation of LytC and a vector and (b) numbers 5, 7, 18 and 19 of pSB1C3-lytC from the original colony PCR.
The second colony PCR products were analysed using gel electrophoresis, but we think the primers may not be annealing because the bands at 1kb were not observed.
Tuesday, 7th-Sep-2010
We used the overnight cultures to make mini preps of both blunt-ended ligation of LytC and pSB1C3-LytC.
We then performed a restriction digest of the pSB1C3-LytC mini preps with XbaI and SpeI to confirm that:
The LytC cell wall binding domain is in the plasmid
The CWB is in the correct orientation
The uncut and cut plasmids were analysed with gel electrophoresis however the result suggested that ligation had been unsuccessful or that the restriction digest had failed.
Wednesday, 8th-Sep-2010
We used PCR to amplify pVeg  from pSB1AK3 again.
Gel electrophoresis confirmed that PCR of pVeg was successful although a big, strong band appeared far above the expected length of pVeg as well.
pVeg was cut out of the gel and purified.
A restriction digest of pVeg with XbaI and SpeI was performed.
When analysing the restriction fragments using gel electrophoresis it appeared to have failed (it turned out later that one of the settings of the UV-box had been changed so not enough light was detected)
pVEG was then digested with SpeI ovenight
Thursday, 9th-Sep-2010
XbaI was added to overnigh digestion of pVeg with SpeI.
The XbaI and SpeI restriction digestion of pVeg was analysed using gel electrophoresis but it pVeg had been lost.
Therefore the PCR amlification from pSB1AK3 was repeated.
This was followed by PCR purification which was then used for an overnight digestion with SpeI. Unlike last time, the gel purification was not used because it decreased our yield too much.
A restriction digest of LytC in the blunt-ended vector was performed.
Subsequent analysis by gel electrophoresis showed a possible candidate from Kirsten's blunt ended ligations (K5).
A midi prep culture for K5 was set up.
23 mini prep cultures from our ligation plate of pSB1C3-LytC were set up.
A test culture was set up to check if a new aliquot of chloramphenicol works.
Friday, 10th-Sep-2010
pVeg was set up in a restriction digest with SpeI
The overnight culture of K5 failed because the wrong antibiotic was used. A new culture was set up with the correct antibiotic.
23 Mini preps of LytC-pSB1C3 ligation were made.
A restriction digest of LytC-pSB1C3 was performed with EcoRI and SpeI as well as XbaI and SpeI for the next day.
Another restriction digest of pVeg was set up using EcoRI and SpeI this time (EcoRI will be added on the next morning).
Saturday, 11th-Sep-2010
EcoRI was added to the pVeg restriction digest.
Gel electrophoresis of the restriction digest of the mini preps indicated that restriction did not work as expected, maybe because one of the enzymes did not cut as a result of a wrong insert. Another possibility is that LytC is in pSB1C3 in the wrong direction, thus disrupting the restriction sites.
Gel electrophoresis was used to analyse the restriction fragments of pVeg, which indicated that pVeg had been cut correctly.
pVeg was cut out of the gel to be gel purified for ligation with pSB1C3.
The K5 midi prep was started and paused after step 13 of the protocol.
Sunday, 12th-Sep-2010
Digested pVeg was gel purified and ligated with digested, dephosphorylated pSB1C3.
Restriction digest of pSB1C3-LytC with EcoRI and SpeI as well as EcoRI only was performed.
Gel electrophoresis showed that SpeI did not cut the vector, probably as a result of incorrect ligation.
The midi prep of K5 was completed
when measuring the concentration of the midi prepit turend out to be close to 0ng/ul.
3 mini prep cultures for K5 were set up.
Week 11
Day
Monday
Tuesday
Wednesday
Thursday
Friday
Morning
3 Mini-Preps of K5
Restriction digest of K5 and pSB1C3-BOO14 with XbaI and SpeI
Meeting with Prof Fenwick and Dr Harrison from the SCI
Screen plates for transformed colonies
Set up mini prep cultures of pSB1C3-LytC and pSB1C3-pVeg
Wiki meeting
Mini preps of pSB1C3-LytC (7) were made but the pSB1C3-pVeg (4) mini preps failed
New cultures for mini preps of pSB1C3-pVeg were set up
Mini preps of pSB1C3-pVeg were successfully made
Afternoon
Gel electrophoresis of restriction fragments
Gel purification of digested K5 and pSB1C3
Gel electrophoresis to determine concentrations of purified fragments
Dephosphorylation of pSB1C3
Ligation of pSB1C3 with LytC overnight
Transform ''E. coli'' with pVeg ligations (has to be repeated)
Transform ''E. coli'' with the ligated pSB1C3-LytC (from K5 culture)
Transform ''E. coli'' with the ligated pSB1C3-pVeg
Repeat PCR amplification of pVeg from pSB1AK3 with the shorter extension time
Gel electrophoresis to analyse PCR
Restriction digest of pSB1C3-LytC with 1) XbaI and SpeI 2) AccI
Gel electrophoresis of digest
Restriction digest of pSB1C3-pVeg, pSB1C3-LytC and linker sequences in pSB1C3 with EcoRI and SpeI.
Gel electrophoresis of digests
Monday, 13th-Sep-2010
The 3 minipreps of K5 were made.
Two mini preps were analysed using a restriction digest with XbaI and SpeI.
pSB1C3-B0014 was also digested with XbaI and SpeI to remove B0014.
Gel electrophoresis was used to confirm correct restriction and the bands were then gel purified.
After gel purification the relative concentration of vector to insert were determined by gel electrophoresis.
After dephosphorylation of pSB1C3, overnight ligation was set up.
E. coli were transformed with pSB1C3-pVeg but could not be plated so transformation will have to repeated.
Tuesday, 14th-Sep-2010
The meeting with Prof Fenwick and Dr Harrison from the Schistosoma Control Initiative (SCI) was very rewarding. Feedback can be found on the Human Practices page.
We performed two transformations of E. coli with:
the ligated pSB1C3-LytC (from K5 culture)
the ligated pSB1C3-pVeg
We also repeated the PCR amplification of pVeg from pSB1AK3 with the shorter extension time, which produced a much higher concentration of pveg so it will be easier to ligate it with pSB1C3.
Wednesday, 15th-Sep-2010
Transformations of E. coli with pSB1C3-pVeg and pSB1C3-LytC was successful.
Mini prep cultures were set up for both pSB1C3-LytC (7) and pSB1C3-pVeg (5).
The team had a meeting to discuss the wiki.
Thursday, 16th-Sep-2010
The mini preps of pSB1C3-LytC were successful, however the mini preps of pSB1C3-pveg failed.
Restriction digest of pSB1C3-lytC with XbaI and SpeI for the double digest, as well as AccI for the single digest were performed.
Cultures for new pSB1C3-pVeg mini preps were set up.
Unfortunately gel electrophoresis of the digestion of pSb1C3-LytC were not conclusive because Either XbaI or SpeI did not cut, nor did AccI. Therefore the digest will be repeated tomorrow with EcoRI rather than XbaI.
Friday, 17th-Sep-2010
Mini preps of pVeg were successfully made
Restriction digest of pSB1C3-pVeg, pSB1C3-LytC and several linker sequences also in pSB1C3 were set up using EcoRI and SpeI.
Gel electrophoresis was used to analyse the restriction digest. It showed that pSB1C3-pVeg as well as the linkers had been ligated and transformed into E. coli successfully. However we most likely did not succeed in construction pSB1C3-LytC yet.
Week 12
Day
Monday
Tuesday
Wednesday
Thursday
Friday
Morning
Prepare K5M3 (LytC in blunt ended ligation vector) as well as H2 and H3 for sequencing
Analysed digestion of H2 and H3 with AseI
Mini prep of Gly-X Com CDE 1-2
Midi prep cultures set up yesterday were used to make successful midi preps of pSB1C3-pVeg
Mini prep of ComD from the cultures set up yesterday
Start midi prep of the linkers from the cultures set up yesterday
Afternoon
Meeting with supervisors for track selection and abstract
Mini prep cultures and replica plate set up for Gly-XCom AB
Restriction digest of the linkers with EcoRI and SpeI as well as AccI
Gel electrophoresis
Set up midi prep cultures for pSB1C3-pVeg (H2,H3)
Restriction digest of 3 mini preps of Flex Com CDE using EcoRI and SpeI for double and AccI for single digest.
Gel electrophoresis of Flex Com CDE restriction fragements
Determine concentration of DNA in pSB1C3-pVeg midi prep
Set up midi prep cultures of the linkers that were sucessfully tested
Make replica plate and set up mini prep cultures of pSB1C3-ComD
Restriction digest of ComD mini preps with EcoRI and SpeI
Finish midi preps of linkers
Measure midi prep DNA concentrations
Monday, 20th-Sep-2010
Samples of LytC in the blunt-ended vector (K5M3) as well as pVeg (H2, H3) were send off for sequencing. No primer had to be added to K5M3 because it is avaliable at the company.
We analysed the restriction digest from yesterday using gel electrophoresis. AseI did not cut either sample of pVeg which is a positive result because AseI does not cut pVeg but BOO14, allowing us to say with some confidence that we have got pVeg. Sequencing will give us certainty soon.
Replica plate and mini prep cultures for the linker Gly-X Com AB were set up
Tuesday, 21st-Sep-2010
The cultures of Gly-X Com CDE 1-2 were used to prepare mini preps.
The previously made mini preps of linkers as well as today's were analysed with a restriction digest using EcoRI and SpeI for a double and AccI for a single digest.
Gel electrophoresis of the restriction fragments showed that many, but not all of the samples, appear to have worked well. HEL-TEV 1-3 have worked, Gly-X Com CDE 2-3 have worked, Flex Com CDE 1-3 all failed so far, but more mini prep samples will be tested tomorrow, Flex TEV 2 worked, and both Gly-X Com AB 1-2 were successful too.
Midi prep culture for pSB1C3-pVeg (H2,H3) were set up.
Wednesday, 21st-Sep-2010
Overnight cultures (H2,H3) of pSB1c3-pveg were used to make midi preps.
Another 3 mini preps of the linker Flex Com CDE were anaysed with a restriction digest using EcoRI and SpeI for the double and AccI for the single digest.
Gel electrophoresis of the restriction fragements showed that the ligation/transformation has not been successful, so it will be repeated.
Sequencing results confirmed, that pSB1C3-pVeg has the correct sequence. Midi preps concentration will be measured tomorrow.
Thursday, 22nd-Sep-2010
Determine concentration of DNA in midi prep of pSB1C3-pVeg:
Concentration of DNA in H2 was ~950ng/µl
Concentration of DNA in H3 was ~430ng/µl
We set up midi prep cultures of the linkers that have been successfully mini preped and tested by a diagnositc digest.
We set up mini prep cultures as well as a replica plate of the pSB1C3-ComD.
Friday, 23rd-Sep-2010
A mini prep of pSB1C3-ComD was made successfully.
Subsequent analysis by restriction digest with EcoRI and SpeI confirmed that ComD is in the vector in the correct orientation.
The midi preps of the linkers Hel-TEV, Flex TEV, Gly-X ComCDE and Gly-X Com AB were made and the concentration determined.
Retrieved from " http://2010.igem.org/Team:Imperial_College_London/Lab_Diaries/Surface_protein_team "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
