<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MichiganState_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MichiganState/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="verytop"><NAV class="navbar navbar-default navbar-fixed-top visible-md visible-lg gradient bmc-bg" style=" padding-top: 20px; padding-left: 8vw; padding-right: 8vw"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 10px;"><LI class="dropdown dropdown8"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState">Project</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState">Project Home</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/References">References</A></LI></UL><LI class="dropdown dropdown1"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState/Model">Modeling</A></LI><LI class="dropdown dropdown5"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Composite_Part">Composite Parts</A></LI><LI class="dropdown dropdown13"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/HP_Site_Visit">Site Visit</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/HP_Expert_Interviews">Expert Interviews</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/HP_Survey">Survey</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Education">Education</A></LI><LI class="dropdown dropdown5"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Attributions">Attributions</A></LI><LI class="dropdown dropdown10"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/InterLab">InterLab Study</A></LI><LI class="dropdown dropdown8"><A class="dropdown-toggle active dropdowntextcenter" href="https://igem.org/2018_Judging_Form?id=2633">Judging Form</A></LI></DIV></DIV></NAV></DIV><DIV class="col-md-12" style="padding-top: 50px"><DIV class="plan" style="padding: 0px 16vw; border: 3px solid transparent; background-color: #FFF"><H1><FONT color="#5bc7d8">Results</FONT></H1><P><UL><LI><A href="#Iso_char">Isolation and Characterization of Endophytic Bacteria</A></LI><LI><A href="#Transformation">Transformation of Endophytic Bacteria</A></LI><LI><A href="#Plant_persist">Plant Persistence Tests</A></LI><LI><A href="#Next_steps">Next Steps</A></LI><LI><A href="#Proj_achieve">Project Achievements</A></LI></UL></P><H3><FONT color="#72C489">Isolation and Characterization of Endophytic Bacteria</FONT></H3><P>Our team isolated and characterized 39 strains of endophytic bacteria harbored in the roots of the wild grass <I>Arrhenatherum elatius</I> and the cultivated bioenergy crop switchgrass (<I>Panicum virgatum</I>). These strains were evaluated for resistance to chloramphenicol and streptomycin using the Kirby-Bauer test (Susceptible, Intermediate, or Resistant). Additionally, strains were tested for ACC utilization by plating on DF-ACC agar plates. We measured strain growth rates, with OD<SUB>600</SUB> 10X dilution after 18 hrs incubation at 30°C. Risk group was determined based on classification by 16S amplification and sequencing.</P><P><I><B>Table 1</B>: Characteristics of evaluated strains.</I></P><H5>KB Antibiotic Key: R = resistant; S = susceptible; I = intermediately susceptible/resistant</H5><H5>ACC Utilizer Key: Y = yes (does utilize ACC); N = no (does not utilize ACC)</H5><H5>Risk Groups: classified according to <A href="http://culturecollection.vtt.fi" target="blank">this resource.</A></H5><TABLE><TBODY><TR><TH>ID</TH><TH>KB Chlor</TH><TH>KB Strep</TH><TH>ACC utilizer</TH><TH>Synonym</TH><TH>OD600 after 18 hrs at 30°C, 10X dilution</TH><TH>Risk group</TH></TR><TR><TD>1A1T-03</TD><TD><I>Pseudomonas corrugata/brassciacearum</I></TD><TD>N</TD><TD>0.227</TD><TD>1</TD></TR><TR><TD>1A1T-04</TD><TD><I>Pseudomonas fluorescens/mandelii</I></TD><TD>N</TD><TD>0.051</TD><TD>1</TD></TR><TR><TD>1C20-02</TD><TD><I>Pseudomonas fluorescens/mandelii</I></TD><TD>N</TD><TD>0.03</TD><TD>1</TD></TR><TR><TD>UAP1-01</TD><TD><I>Pseudomonas baetica/moraviensis</I></TD><TD>N</TD><TD>0.226</TD><TD>1</TD></TR><TR><TD>UBP2-01</TD><TD><I>Pseudomonas baetica/moraviensis</I></TD><TD>N</TD><TD>0.181</TD><TD>1</TD></TR><TR><TD>UBP2-02</TD><TD><I>Pseudomonas putida/koreensis</I></TD><TD>N</TD><TD>0.152</TD><TD>1</TD></TR><TR><TD>FP0-02</TD><TD><I>Pseudomonas</I> sp.</TD><TD>N</TD><TD>0.178</TD></TR><TR><TD>FBP2-01</TD><TD>N</TD><TD>FT0-01</TD><TD>0.16</TD></TR><TR><TD>FBP3-01</TD><TD>N</TD><TD>0.338</TD></TR><TR><TD>UCP1-01</TD><TD><I>Pseudomonas mohnii</I></TD><TD>N</TD><TD>0.102</TD><TD>1</TD></TR><TR><TD>UCP1-02</TD><TD><I>Pseudomonas baetica/moraviensis</I></TD><TD>N</TD><TD>0.161</TD><TD>1</TD></TR><TR><TD>UCP1-03</TD><TD><I>Enterobacter asburiae</I></TD><TD>N</TD><TD>0.336</TD><TD>1</TD></TR><TR><TD>UCP2-01</TD><TD><I>Pseudomonas moraviensis/baetica</I></TD><TD>N</TD><TD>0.191</TD><TD>1</TD></TR><TR><TD>UP0-02</TD><TD><I>Burkholderia stabilis</I></TD><TD>Y</TD><TD>0.086</TD><TD>2</TD></TR><TR><TD>FT0-01</TD><TD>S</TD><TD>S</TD><TD>N</TD><TD>FBP2-01</TD><TD>0.227</TD></TR><TR><TD>UT0-01</TD><TD><I>Pseudomonas baetica</I></TD><TD>N</TD><TD>0.139</TD><TD>1</TD></TR><TR><TD>UCP2-02</TD><TD><I>Pseudomonas putida/koreensis</I></TD><TD>I</TD><TD>S</TD><TD>N</TD><TD>0.301</TD><TD>1</TD></TR><TR><TD>1C2P-04</TD><TD><I>Pseudomonas rhodesiae</I></TD><TD>I</TD><TD>I</TD><TD>Y</TD><TD>1</TD></TR><TR><TD>FCP2-01</TD><TD><I>Enterobacter ludwigii</I></TD><TD>S</TD><TD>S</TD><TD>N</TD></TR><TR><TD>FCP1-02</TD><TD><I>Acinetobacter calcoaceticus</I></TD><TD>R</TD><TD>S</TD><TD>N</TD><TD>2</TD></TR><TR><TD>FCP1-01</TD><TD><I>Pseudomonas fluorescens</I></TD><TD>N</TD><TD>1</TD></TR><TR><TD>UAP1-02</TD><TD><I>Enterobacter/Klebsiella</I></TD><TD>N</TD></TR><TR><TD>UAP2-01</TD><TD><I>Pseudomonas vancouverensis/mohnii/ fluorescens</I></TD><TD>R</TD><TD>I</TD><TD>N</TD><TD>1</TD></TR><TR><TD>FP0-01</TD><TD><I>Acinetobacter calcoaceticus/rhizosphaerae</I></TD><TD>N</TD><TD>2</TD></TR><TR><TD>UP0-01</TD><TD><I>Lysinibacillus</I> sp.</TD><TD>R</TD><TD>R</TD><TD>N</TD></TR><TR><TD>1A1T-02</TD><TD><I>Pseudomonas</I> sp.</TD><TD>I</TD><TD>S</TD><TD>Y</TD></TR><TR><TD>UP0-04</TD><TD><I>Burkholderia stabilis</I></TD><TD>Y</TD><TD>2</TD></TR><TR><TD>UP0-03</TD><TD><I>Enterobacter/Klebsiella</I></TD><TD>N</TD></TR><TR><TD>1C2P-01</TD><TD><I>Pseudomonas</I> sp.</TD><TD>R</TD><TD>S</TD><TD>N</TD></TR><TR><TD>1A1T-01</TD><TD><I>Pseudomonas</I> sp.</TD><TD>Y</TD></TR><TR><TD>2FAP1-02</TD><TD><I>Pseudomonas</I> sp.</TD><TD>S</TD><TD>Y</TD></TR><TR><TD>2UAP1-02</TD><TD><I>Pseudomonas</I> sp.</TD><TD>S</TD><TD>Y</TD></TR><TR><TD>2FP0-01</TD><TD><I>Bacillus</I> sp.</TD><TD>S</TD><TD>N</TD></TR><TR><TD>2UCP1-01</TD><TD>S</TD><TD>N</TD></TR><TR><TD>2UBP1-01</TD><TD><I>Bacillus mycoides</I></TD><TD>S</TD><TD>N</TD></TR><TR><TD>2FP0-02</TD><TD><I>Pseudomonas</I> sp.</TD><TD>S</TD><TD>N</TD></TR><TR><TD>2UT0-01</TD><TD><I>Pseudomonas</I> sp.</TD><TD>S</TD><TD>Y</TD></TR><TR><TD>2FCP1-01</TD><TD><I>Bacillus mycoides</I></TD><TD>S</TD><TD>N</TD></TR><TR><TD>2FAP1-01</TD><TD><I>Pseudomonas</I> sp.</TD><TD>S</TD><TD>N</TD></TR><TR><TD>2UAP1-01</TD><TD><I>Pseudomonas</I> sp.</TD><TD>S</TD><TD>N</TD></TR><TR><TD>2FT0-01</TD><TD><I>Pseudomonas</I> sp.</TD><TD>S</TD><TD>N</TD></TR></TBODY></TABLE><H5>Figure 1. ACC utilization assay. UPO-02 (upper-right quadrant on left plate) is positive for ACC utilization.</H5><H3><FONT color="#72C489">Transformation of endophytic bacteria</FONT></H3><P>After a series of transformation attempts, a transformation protocol was developed which could be consistently used. About 25-50% of colonies on later plates are transformants.</P><H5><A href="https://static.igem.org/mediawiki/parts/d/db/T--MichiganState--transformation_results_small.png" target="blank">Figure 2</A>.  A) FCP2-01 transformed with GFP (left) vs wild-type (right) under blue light with orange filter. B) Transformed colonies on an agar plate.</H5><H3><FONT color="#72C489">Plant persistence tests</FONT></H3><P>A key test of the effectiveness of an endophyte to impact its host if the endophyte can persist in and on the plant. FCP2-01 was isolated from the roots, and therefore we examined ability for the endophyte to inhabit the roots of <I>Brachypodium distachyon</I>. Seeds were inoculated by submerging in bacterial culture, then grown axenically in glass culture tubes in a growth chamber. Confocal micrographs were acquired at intervals of 1 week for 1-5 weeks post-inoculation. Bacteria were observed on roots at weeks 3 and 4 most clearly, but overwatering may have been responsible for no visible colonies in week 5.

</P><H5><A href="https://static.igem.org/mediawiki/parts/thumb/1/13/T--MichiganState--root_colonization_3wk_GA_results_full.png/900px-T--MichiganState--root_colonization_3wk_GA_results_full.png">Figure 3</A>. 3 weeks post inoculation on plant roots. A) FCP2-01 forming large colonies about 2 mm long in B. distachyon roots B) Detail of panel A. C) FCP2-01 growing between plant cells on the root surface. D) Detail of colonies on root epidermis.</H5><H5><A href="https://static.igem.org/mediawiki/parts/thumb/c/c4/T--MichiganState--root_colonization_4wk_GA_results_full.png/900px-T--MichiganState--root_colonization_4wk_GA_results_full.png">Figure 4</A>. 4 weeks post inoculation on plant roots. A) FCP2-01 on B. distachyon roots. B) FCP2-0 colonizing root hairs. C) FCP2-01 growing on epidermis of the root. D) Enlarged image of panel C.</H5><H3><FONT color="#72C489">Next Steps</FONT></H3><UL><LI>Demonstrate expression and activity of <I>acdS</I> in FCP2-01</LI><LI>Develop a regulation system based on abscisic acid to turn on only in drought</LI><LI>Inoculate <I>B. distachyon</I> and measure effects of FCP2-01 expressing <I>acdS</I> on plant growth and yield under osmotic stresses</LI><LI>Test transformation protocol on additional endophytic isolates of grasses</LI><LI>Evaluate ability for endophytic bacteria to enter roots, and how endophytic fungi may affect this process</LI><LI>Test species specificity of FCP2-01 and ability to horizontally transmit between a plant and a neighbor</LI><LI>Pathogenicity screens of FCP2-01 and more complete evaluation of antibiotic resistance</LI><LI>Review genomes of <I>Enterobacter ludwigii</I> for insights into plasmid stability and other factors affecting biotechnology potential</LI><LI>Field trials of FCP2-01 (if the strain is cleared as safe) to test persistence and physiological effects in real-world applications</LI></UL><H3><FONT color="#72C489">Project Achievements</FONT></H3><H5>Successful Results</H5><UL><LI>Isolation and screening of endophytic bacterial isolates provided a diversity of strains and for synthetic biology experimentation.</LI><LI>A transformation protocol was developed for a endophytic bacteria, which provides a chassis for affecting plant physiology without engineering the plant</LI><LI>Reinfection and persistence were observed in a distantly related grass 51.6 ± 7.9 (SD) million years since divergence (Vicentini et al. 2008). FCP2-01 is capable of stably expressing a fusion protein of <I>acdS</I> and GFP for at least 4 weeks in the plant.</LI><LI><A href="https://2018.igem.org/Team:MichiganState/Model">Modeling</A></LI></UL><H5>Unsuccessful results</H5><UL><LI>Most transformations did not work. We were unable to transform and express any of our vectors in <I>Pseudomonas</I> spp. Attempts at transformation via conjugation were also unsuccessful.</LI><LI>Due to timeline disruptions by unviable seeds, we were unable to evaluate the effect of engineered FCP2-01 on plant growth</LI><LI>None of our engineered strain could grow with ACC as the only available nitrogen source. It is possible that the attachment of a GFP tag (used for identifying transformed colonies and visualizing bacteria in and on roots) disrupted the quaternary structure of the ACC deaminase tetramer.</LI></UL><H4>References</H4><UL><LI>Vicentini A, Barber JC, Aliscioni SS, Giussani LM, Kellogg EA. The age of the grasses and clusters of origins of C4 photosynthesis. Global Change Biology. 2008 Dec;14(12):2963-77.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>