<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MichiganState_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MichiganState/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="verytop"><NAV class="navbar navbar-default navbar-fixed-top visible-md visible-lg gradient bmc-bg" style=" padding-top: 20px; padding-left: 8vw; padding-right: 8vw"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 10px;"><LI class="dropdown dropdown8"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState">Project</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState">Project Home</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/References">References</A></LI></UL><LI class="dropdown dropdown1"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState/Model">Modeling</A></LI><LI class="dropdown dropdown5"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Composite_Part">Composite Parts</A></LI><LI class="dropdown dropdown13"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/HP_Site_Visit">Site Visit</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/HP_Expert_Interviews">Expert Interviews</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/HP_Survey">Survey</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Education">Education</A></LI><LI class="dropdown dropdown5"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Attributions">Attributions</A></LI><LI class="dropdown dropdown10"><A class="dropdown-toggle active dropdowntextcenter" href="https://2018.igem.org/Team:MichiganState/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:MichiganState/InterLab">InterLab Study</A></LI><LI class="dropdown dropdown8"><A class="dropdown-toggle active dropdowntextcenter" href="https://igem.org/2018_Judging_Form?id=2633">Judging Form</A></LI></DIV></DIV></NAV></DIV><DIV class="col-md-12" style="padding-top: 50px"><DIV class="plan" style="padding: 0px 16vw"><FONT color="“#5bc7d8”"><H2>Modeling Interactions Between 1-aminocyclopropane-1-carboxylate deaminase (ACCD) and 1-aminocyclopropane-1-carboxylate (ACC)</H2></FONT><FONT size="4"><UL><LI><A href="#Introd">Introduction</A></LI><LI><A href="#Kin_model">Kinetic Modeling</A></LI><LI><A href="#1TYZ_Struct">1TYZ Strutural Modeling Using PYMOL</A></LI><LI><A href="#Homologous_align">Homologous Alignment Modeling</A></LI></UL></FONT><FONT color="#72C489"><H3>Introduction</H3></FONT><P>
1-aminocyclopropane-1-carboxylate deaminase is a pyridoxal 5’ phosphate (PLP) dependent enzyme that catalyzes the chemical conversion of  1-aminocyclopropane-1-carboxylate,  the immediate precursor of ethylene synthesis in plants, and water to ɑ-ketobutyrate + ammonia. Thus, the two substrates of this enzyme are 1-aminocyclopropane-1-carboxylate  and water, and its two products are ketobutyric acid and ammonia <A href="https://www.academia.edu/23449809/Structural_Analysis_of_Pseudomonas_1-Aminocyclopropane-1-carboxylate_Deaminase_Complexes_Insight_into_the_Mechanism_of_a_Unique_Pyridoxal-5_-phosphate_Dependent_Cyclopropane_Ring-Opening_Reaction_" target="_blank">(1).</A> This enzyme belongs to the family of hydrolyses, those acting on carbon-nitrogen bonds other than peptide bonds, specifically in compounds that have not been otherwise categorized within EC number 3.5. 
</P><P><I><B>Figure 1</B>: A visual representation of the ACC deaminase catalyzed reaction.</I></P><P>
ACC is emitted from seeds or plant roots and then metabolized by bacteria expressing ACC deaminase activity, which stimulates plant ACC outflow, decreasing the root ACC concentration and root ethylene evolution and increasing root growth. The widespread of ACC deaminase activity has been noted in several <I>Burkholderia</I> species. The common association of these species with plants suggests that this genus could be a major contributor to plant growth under natural conditions. From this perspective, the approach of reducing the production of ethylene levels in plants by increasing the activity of ACC deaminase may be an effective strategy to improve plant growth under a variety of stressful conditions, such as flooding, high saline conditions, and drought. In this work we investigated the kinetic properties of ACC deaminase, structural properties including ways to mutate residues surrounding the cofactor Pyridoxal Phosphate binding site, and ultimately used modeling to determine which homolog of ACC deaminase would most closely resemble our PDB model, 1TYZ. 
</P><P><I><B>Figure 3</B>: Crystal structure of 1-aminocyclopropane-1-carboyxlate deaminase complexed with ACC</I></P><FONT color="#72C489"><H3>Kinetic Modeling</H3></FONT><P><B>Enzyme kinetics</B> is the study of the chemical reactions catalyzed by enzymes. The reaction rate is measured under different environmental conditions to determine the overall functionality of the protein. By studying an enzyme’s kinetics, one can determine the catalytic mechanism of the enzyme, its role in metabolism, how its activity is controlled, and how a drug or agonist might inhibit the function of the enzyme. Our team looked at the kinetic modeling to help determine if the activity of our model protein of ACC deaminase, PDB 1TYZ, was efficient. </P><P><I><B>Figure 4</B>: Structure of cofactor pyridoxal phosphate (PLP) on left; structure of 1-aminocyclopropane-1-carboxylic acid (ACC) on right</I></P><P><I><B>Figure 5</B>: K dissociation constants for each selected step in the ACC pathway.</I></P><P>Following the Michaelis-Menten hypothesis and plotting the experimental values for dissociation constants shown below in Table 1, a graphical representation, Figure 6, of the ACC deaminase reaction was created.</P><P><I><B>Table 1</B>: Summary of pH-dependence of the steady state kinetic parameters for wild-type ACCD and its mutants. MSR is the mean square residual of the non-linear fit. The equations used are listed in the <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059330/" target="_blank">original paper</A> under the Materials and Methods section.</I></P><P><I><B>Figure 6</B>: The graph above shows the relationship between substrate/enzyme concentration and time it takes for the reaction to reach completion. This information is important to determine efficiency and activity.</I></P><FONT color="#72C489"><H3>1TYZ Structural Modeling Using PYMOL</H3></FONT><P>
To probe the possibilities of mutating some of the residues surrounding the active site, specifically Tyr294, several observations of this protein structure were noted by manipulating the 3D model of ACC deaminase, 1TYZ. A gem-diamine intermediate is trapped in the enzyme complex with ACC, and Tyr294 is in close proximity to the pro-S methylene carbon of ACC in the gem-diamine complexes. This indicates a direct role of this residue in the ring-opening reaction and also suggesting that Tyr294 may also be responsible for the abstraction of the alpha-proton from D-amino acids, a precursor to the subsequent deamination reaction. These structural data provide evidence favoring a mechanism in which the ring cleavage is begun by a nucleophilic attack at the pro-S â-methylene carbon of ACC, with Tyr294 as the nucleophile. However, these observations are also consistent with an alternative possibility in which the ring opening is acid-catalyzed and may be facilitated by charge relay through PLP, where Tyr294 functions as a general acid. For most PLP-dependent enzymes, the reaction starts with the conversion of an internal aldimine between PLP and an active site lysine residue to an external aldimine between PLP and the amino group of the substrate which then followed by the removal of the R-proton of the substrate, which triggers subsequent transformations such as transamination and racemization.
</P><P><I><B>Figure 7</B>: The structure of the gem-diamine intermediate in Pseudomonas ACC deaminase complexed with ACC. The bonds are indicated with dotted lines. The interaction between the Tyr294 hydroxyl group and the pro-S carbon of ACC is indicated by a thin red line.</I></P><P>
Figure 7 shows that the bound PLP cofactor in the <I>Pseudomonas</I> enzyme forms an internal aldimine with Lys51 through its side chain N_ amino group. The N atom of the PLP pyridinium ring forms a hydrogen bond with the carboxylate group of Glu295 (2.6 Å), whereas the O2 oxygen interacts with the side chain amino group of Asn79 (3.0 Å) and a water molecule (2.6 Å). The phosphate of PLP is anchored near the N-terminus of an alpha-helix formed by residues 202- 213, and interacts with the main chain amide groups of Val198, Thr199, Gly200, and Thr202, and with the side chains of Lys54, Thr199, and Thr202. Several protein residues also contact PLP through van der Waals interactions. These include Leu322, Asn50, and notably, Tyr294, the latter of which stacks against the plane of the pyridinium ring. These interactions firmly position the cofactor in the enzyme active site. As above there exists a bound sulfate molecule in the active site of the native enzyme which interacts with the main chain amides of Asn79 and Gln80 and with the side chains of Ser78 and Tyr294.
</P><P>
The fact that Tyr294 is located near the cyclopropane ring in the active site of ACC deaminase strongly suggested a direct catalytic role for this residue in the ring-opening reaction. The reactivity of Tyr294 may be enhanced by hydrogen bonding to Tyr268, which is only 2.8 Å away. Previous studies shows that these two tyrosine residues are strictly conserved among the known ACC deaminase homologues. The conserved presence of the two-tyrosine system in ACC deaminase homologues suggests a significant function for this tyrosine pair. To probe the possible involvement of the two tyrosine residues in the reaction, previous studies where the Y294F and Y268F mutants were prepared, suggested that both exhibited chromatographic behavior and spectral properties identical with those of the wild-type enzyme. The PLP content in the Y268F and Y294F mutant proteins was also the same as that observed for the wild-type enzyme. These mutant proteins were capable of binding ACC in the active site. However, no ACC deaminase activity of the Y294F mutant enzyme was detected using the coupled
lactate dehydrogenase/NADH assay, and no keto product was generated as assessed by 1H NMR spectroscopy after a 24-h incubation period with the substrate. The activity of the Y268F mutant  also dramatically diminished. 
</P><P>
After a detailed analyses of the structure and kinetics of ACC deaminase 1TYZ, we came to conclude that the crystal structure of 1-aminocyclopropane-1-carboyxlate deaminase complexed with ACC is in optimal state, and therefore any mutation done to residues such as Try 294, Glu295 that are in in close proximity to the cofactor Pyridoxal Phosphate <U>will completely destroy the structure of the enzyme rather than promoting its activity</U>. It was also concluded from our homology modeling that the gene of ACC deaminase from <I>Pseudomonas sp.</I> which we extracted from a root endophyte, was the closest match to the 1TYZ ACC deaminase used for structural modeling. 
</P><FONT color="#72C489"><H3>Homologous Alignment Modeling</H3></FONT><P>After analyzing the structure of 1TYZ using the protein program PYMOL, we used Phyre2 to produce 3D protein structures of three different ACC deaminase genes from one bacterium and two fungi. After producing a pdb file for each of the genes, Phyre2 also produced a homology comparison to each of the ACC deaminase structures submitted to the protein database. Below are the results from all three homolog alignments.</P><P><I><B>Figure 8</B>: Below are the homolog alignment results for the Fusarium oxysporum gene. As you can see, the highest match is ACC deaminase protein 1TYZ with 57% match.</I></P><P><I><B>Figure 9</B>: Below are the homolog alignment results for the Trichoderma asperellum gene. The highest match is ACC deaminase protein 1TYZ with 60% match. </I></P><P><I><B>Figure 10</B>: Below are the homolog alignment results for the Pseudomonas sp. gene. The highest match is ACC deaminase protein 1TYZ with 83% match.</I></P><P>From these homolog matches we can conclude that the closest to our model ACC deaminase gene, 1TYZ, is the gene extracted from Pseudomonas sp. This makes sense since 1TYZ was produced in Pseudomonas. Because of this, we chose to use the Pseudomonas gene to transform our endophyte with. This way, we would have the most information regarding the efficiency and functionality of the protein our endophyte was producing. There were a couple amino acid differences between the Pseudomonas gene and the other two. The most important one which should be mentioned, is directly involved with catalytic pocket function. </P><P>This point is at position 80 in the amino acid sequence. Both 1TYZ and our Pseudomonas strain have Q or glutamine instead of arginine in position 80. The image below shows this distinction. This is very interesting, since arginine is a very large molecule and is a positively charged molecule. Glutamine, on the other hand is a smaller, polar molecule. </P><P><I><B>Figure 11</B>: The ring is PLP in the active site of this monomer, and the molecule in purple is arginine and the molecule in blue is glutamine.</I></P><P>Overall, the three genes we had available to us aligned almost perfectly to 1TYZ. Below is a 360° rotation and zoom into the active site of the enzyme where PLP is seen. Each color is a different one of the homologs overlapping with 1TYZ. The color distinctions are given in the figure legend. </P><VIDEO width="100%" controls=""><P><I><B>Media 1</B>: Movie made using PYMOL that shows the different alignment and active site of one monomer; shown in green is 1TYZ, blue is Pseudomonas, magenta is Trichoderma, and Fusarium in yellow. </I></P></VIDEO></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>