<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NYMU-Taipei_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NYMU-Taipei/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:NYMU-Taipei"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NYMU-Taipei/Overview"><DIV class="submenu_button" id="Overview_page">
					Overview
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Pigments"><DIV class="submenu_button" id="Pigments_page">
					Pigments
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Nitrogen_starvation"><DIV class="submenu_button" id="Nitrogen_starvation_page">
					Nitrogen starvation
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Model"><DIV class="submenu_button" id="Model_page">
					Modeling
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Improve"><DIV class="submenu_button" id="Improve_page">
					Improvement
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstration
				</DIV></A></DIV><DIV class="menu_button"> LABWORK
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NYMU-Taipei/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Safety"><DIV class="submenu_button" id="Safety_page">
					Safety
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A></DIV><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NYMU-Taipei/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Gold &amp; Integrated HP
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> ABOUT US
		</DIV><DIV class="submenu_wrapper" id="About_Us_submenu"><A href="https://2017.igem.org/Team:NYMU-Taipei/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NYMU-Taipei/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Model"><DIV class="submenu_button" id="Model_page">
					Modeling
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=NYMU-Taipei"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="igem_2017_footer_wrapper "><DIV class="footer_half_size"><P class="footer_title">CONTACT US</P><P class="footer_text">Email us: 2017igem.nymutaipei@gmail.com
				Call us: 886-2-28267316
				Facebook: <A href="https://www.facebook.com/NYMU-iGEM-Team-244897845537138/?fref=ts" target="_blank">NYMU iGEM Team</A></P></DIV><DIV class="footer_half_size"><P class="footer_title">AFFILIATIONS &amp; ACKNOWLEDGMENT</P><P class="footer_text"><A href="https://2017.igem.org/Main_Page" target="_blank">IGEM 2017</A><A href="http://nymu-e.web.ym.edu.tw/bin/home.php" target="_blank">National Yang-Ming University</A><A href="https://2017.igem.org/Team:NYMU-Taipei/Attributions">Special Thanks</A></P></DIV></DIV><BUTTON onclick="topFunction()" id="myBtn" title="Go to top">Top</BUTTON><DIV id="Timeline"><H1>Timeline</H1><DIV class="panel"><DIV id="s1" class="expandable" style="height: 24px;padding-top:5px;"><A href="#!" onclick="toggleHeight1(this, 270); return false" style="font-family: 'Acme', sans-serif;font-size:28px;color: #479fcc;height: 20px;">
					June, 2017
				</A><P><B>2017-06-19</B>   prepare stock1 and stock5<B>2017-06-20</B>   (1) reprepare stock5
					　　　　　(2) prepare BG11 medium
					　　　　　(3) unfreeze <I>Synechocystis</I> sp. PCC6803 and start to cultivate in BG11<B>2017-06-29</B>   (1) prepare Pseudomonas aeruginosa growing medium
					　　　　　(2) activate Pseudomonas aeruginosa and start to cultivate in Petri dish
					　　　　　(3) subculture <I>Synechocystis</I> sp. PCC6803
				</P><CENTER><A href="#!" onclick="toggleHeight1(this, 270);" style="font-size:20px;color:#2c498c">
					click to close
				</A></CENTER></DIV></DIV><DIV class="panel"><DIV id="s2" class="expandable" style="height: 24px;padding-top:5px;"><A href="#!" onclick="toggleHeight2(this, 1440); return false" style="font-family: 'Acme', sans-serif;font-size:28px;color: #479fcc;height: 20px;">
					July, 2017
				</A><P><B>2017-07-02</B>   extract <I>Pseudomonas aeruginosa</I> plasmid and freeze into refrigerator<B>2017-07-03</B>   proliferate <I>Pseudomonas aeruginosa</I> and freeze into refrigerator<B>2017-07-04</B>   (1) prepare BG11 medium
					　　　　　(2) proliferate <I>Chlorella vulgaris</I> in Petri dish and centrifuge tube<B>2017-07-05 to 07-10</B>   cultivate <I>Synechocystis</I> sp. PCC6803 and <I>Chlorella vulgaris</I> in incubator<B>2017-07-11</B>     (1) extract <I>Synechocystis</I> sp. PCC6803 plasmid
					　　　　　(2) prepare LB containing ampicillin and chloramphenicol and prepare Petri dish<B>2017-07-12</B>   transform BBa_K112806, BBa_K737007, BBa_K112000, BBa_J04450, BBa_K1356003, BBa_K1356004, and BBa_K356005 into competent cell<B>2017-07-13</B>    (1) proliferate 07-12 transformed E.coli
					　　　　　(2) proliferate <I>Chlorella vulgaris</I>
					　　　　　(3) proliferate <I>Synechocystis</I> sp. PCC6803
					　　　　　(4) activate <I>Synechococcus elongatus</I> PCC7942<B>2017-07-14</B>    (1)extract 07-13 proliferated E.coli plasmid
					　　　　　(2) digestion: BBa_K112806, BBa_K737007, BBa_K112000, BBa_J04450, BBa_K1356003, BBa_K1356004, and BBa_K1356005
					　　　　　　(check the correction of transformed E.coli)<B>2017-07-17</B>    (1) digestion again: BBa_K1356005, BBa_K112806, and BBa_K112000
					　　　　　(2) retransform BBa_K112806 and BBa_K112000<B>2017-07-18</B>    (1) proliferate BBa_K112806 transformed E.coli (BBa_K112000 failed)
					　　　　　(2) subculture <I>Synechococcus elongatus</I> PCC7942
					　　　　　(3) cultivate <I>Synechocystis</I> sp. PCC6803 in BG11<B>2017-07-19</B>    (1) extract 07-18 BBa_K112806 transformed E.coli plasmid
					　　　　　(2) digestion check
					　　　　　(3) save BBa_K112806 transformed E.coli
					　　　　　(4) save 07-18 <I>Synechocystis</I> sp. PCC6803<B>2017-07-20</B>   (1) activate Rhizobium etli in PY medium
					　　　　　(2) dilute primer 001~008
					　　　　　(3) run NrtA and terminator PCR (failed)<B>2017-07-21 to 07-23</B>   cultivate <I>Chlorella vulgaris</I> again for measuring growth curve<B>2017-07-24</B>   (1) prepare <I>Rhizobium etli</I> liquid culture
					　　　　　(2) activate <I>Synechococcus elongatus</I> PCC7942<B>2017-07-25</B>   (1) extract <I>Synechocystis</I> sp. PCC6803 genomic DNA
					　　　　　(2) run NrtA PCR<B>2017-07-26</B>   (1) extract <I>Rhizobium etli</I> genomic DNA
					　　　　　(2) run NrtA and terminator PCR
					　　　　　(3) run NrtA+ terminator fusion PCR (failed)<B>2017-07-27</B>   (1) run NrtA+ terminator fusion PCR again
					　　　　　(2) purify the successful fusion product 
					　　　　　(3) digestion check<B>2017-07-28</B>   run the whole NrtA cloning procedure again, using KOD polymerase
					　　　　　(1) run NrtA and terminator PCR
					　　　　　(2) digestion check NrtA and terminator and purify the PCR product
					　　　　　(3) run NrtA and terminator fusion PCR
					　　　　　(4) digestion check and purify the fusion PCR product (failed)<B>2017-07-29</B>   (1) run NrtA and terminator fusion PCR again
					　　　　　(2) digestion check and purify the fusion PCR product
					　　　　　(3) ligation: backbone(psb1C3) and promoter, RBS, NrtA, and terminator
					　　　　　(4) transformation in to E.coli<B>2017-07-30</B>   cultivate the transformed E.coli in liquid culture<B>2017-07-31</B>    (1) extract 07-30 transformed E.coli plasmid
					　　　　　(2) restriction enzyme check (failed)</P><CENTER><A href="#!" onclick="toggleHeight2(this,1440);" style="font-size:20px;color:#2c498c">
					click to close
				</A></CENTER></DIV></DIV><DIV class="panel"><DIV id="s3" class="expandable" style="height: 24px;padding-top:5px;"><A href="#!" onclick="toggleHeight3(this, 2200); return false" style="font-family: 'Acme', sans-serif;font-size:28px;color: #479fcc;height: 20px;">
					August, 2017
				</A><P><B>2017-08-04</B>   Digest check: BBa_K112806<B>2017-08-05</B>  <I>Chlorella Vulgaris</I> growth curve measuring<B>2017-08-06</B>   PCR(Dream Taq): Sucide mechanism-BBa_K112806, K737007, J04450, K112000<B>2017-08-07</B>  (1) PCR(KOD plus): Sucide mechanism-BBa_K112806, K737007, J04450, K112000 
					　　　　　(2) NrtA- annealing, PNK, ligation, transformation<B>2017-08-08</B>   Suicide mechanism-Annealing, cloning, ligation, transformation<B>2017-08-09</B>   Extraction of <I>Synechoccocus</I> PCC7942 genomic DNA<B>2017-08-10</B>   (1) <I>Chlorella Vulgaris</I> growth curve measuring
					　　　　　(2) Fusion PCR: NrtA + terminator
					　　　　　(3) Digestion check: NrtA cut with BamH1
					　　　　　(4) Subculture <I>S.elongatus</I> PCC7942<B>2017-08-11</B>    (1) Ligation: NrtA + terminator + promoter + RBS
					　　　　　(2) Ligation: BBa_K737007+K112806, J04450+K112000 (Endolysin and Holin)
					　　　　　(3) Transformation of Endolysin and Holin
					　　　　　(4) Measure nitrate and nitrite concentration of <I>Chlorella Vulgaris</I>　　　　　(5) OD value measurement of <I>Chlorella Vulgaris</I> and <I>S.elongatus</I> PCC7942<B>2017-08-12</B>   (1) Measure nitrate and nitrite concentration of <I>Chlorella Vulgaris</I>　　　　　(2) OD value measurement of <I>Chlorella Vulgaris</I> and <I>S.elongatus</I> PCC7942<B>2017-08-13</B>   (1) Transformation: BBa_K274002 and K1152008 into E. coli
					　　　　　(2) Measure nitrate and nitrite concentration of <I>Chlorella Vulgaris</I>　　　　　(3) OD value measurement of <I>Chlorella Vulgaris</I> and <I>S.elongatus</I> PCC7942<B>2017-08-14</B>   (1) Colony PCR: K1152008
					　　　　　(2) Suicide mechanism- PCR, annealing, cloning, ligation, transformation 
					　　　　　(3) NrtA-PCR, annealing, cloning, ligation, transformation
					　　　　　(4) Liquid culture-K1152008<B>2017-08-15</B>   (1) Measure nitrate and nitrite concentration of <I>Chlorella Vulgaris</I>　　　　　(2) OD value measurement of <I>Chlorella Vulgaris</I> and <I>S.elongatus</I> PCC7942
					　　　　　(3) Suicide mechanism- final annealing, cloning, ligation, transformation
					　　　　　(4) Plasmid extraction and RE check: k1152008<B>2017-08-16</B>   (1) Measure nitrate and nitrite concentration of <I>Chlorella Vulgaris</I>　　　　　(2) OD value measurement of Chlorella Vulgaris and <I>S.elongatus</I> PCC7942
					　　　　　(3) PCR(Dream Taq): PrbCL and indC<B>2017-08-18</B>   (1) Suicide mechanism- PCR, annealing, cloning, ligation, transformation
					　　　　　(2) PCR(KOD plus): PrbCL and indC<B>2017-08-19</B>   (1) Suicide mechanism- PCR, annealing, cloning, ligation, transformation
					　　　　　(2) Inoculate E. coli containing pBAD18
					　　　　　(3) Fusion PCR: PrbCL+indC<B>2017-08-22</B>   (1) PCR: Lycopene and MelA
					　　　　　(2) Fusion PCR: PrbCL+indC<B>2017-08-24</B>   (1) Liquid culture: BBa_K274002
					　　　　　(2) Extract PCC7942 genomic DNA
					　　　　　(3) Inoculate E. coli containing PBR322
					　　　　　(4) Transformation: BBa_B0015 into E. coli<B>2017-08-25</B>   (1) Endolysin- PCR, annealing, cloning, ligation, transformation
					　　　　　(2) Liquid culture: PBR322
					　　　　　(3) Liquid culture: B0015<B>2017-08-26</B>   (1) Plasmid extraction: endolysin
					　　　　　(2) Plasmid extraction: PBR322
					　　　　　(3) Plasmid extraction: B0015<B>2017-08-27</B>   (1) PCR: Lycopene and MelA
					　　　　　(2) Digestion check: PBR322
					　　　　　(3) Digestion check: B0015
					　　　　　(4) pigment backbone primer design
					　　　　　(5) PCR (Dream Taq): CrtZ
					　　　　　(6) Plasmid extraction: endolysin
					　　　　　(7) Digestion check: endolysin<B>2017-08-28</B>   (1) Colony PCR: NrtA
					　　　　　(2) Liquid culture: NrtA
					　　　　　(3) Sequencing: endolysin
					　　　　　(4) Fusion PCR: PrbcL+IndC
					　　　　　(5) PCR(Dream taq): Holin(kit)
					　　　　　(6) Check pBR322 concentration and transform pBR322<B>2017-08-29</B>   (1) Nile red oil measuring of <I>Chlorella Vulgaris</I> (~0901)
					　　　　　(2) Plasmid extraction: NrtA
					　　　　　(3) Holin construction
					　　　　　(4) sequencing: NrtA<B>2017-08-30</B>   (1) PCR (Dream Taq): Ampicillin resistance, 5NSII, 3NSII, B0015, and PBR322 ori
					　　　　　(2) Endolysin construct finish
					　　　　　(3) Plasmid extraction: pBR322 
					　　　　　(4) Restriction enzyme check: pBR322
					　　　　　(5) Backbone construct-3NSII, 5NSII,AMP, B0015, and ORI (PCR: dream taq)<B>2017-08-31</B>   (1) PCR (KOD plus): Ampicillin, 5NSII, 3NSII, B0015, and PBR322 ori
					　　　　　(2) PCR (Dream Taq): PBR322 ori and B0015
					　　　　　(3) pSB1C3-NrtA construct finish
					　　　　　(4) Backbone construct-3NSII, 5NSII, AMP, B0015, and ORI (PCR: KOD)
					　　　　　(5) Transform: BBa_K112000
					　　　　　(6) LBA: glucose + CM (0.3%)
				</P><CENTER><A href="#!" onclick="toggleHeight3(this, 2200);" style="font-size:20px;color:#2c498c">
					click to close
				</A></CENTER></DIV></DIV><DIV class="panel"><DIV id="s4" class="expandable" style="height: 24px;padding-top:5px;"><A href="#!" onclick="toggleHeight4(this, 1900); return false" style="font-family: 'Acme', sans-serif;font-size:28px;color: #479fcc;height: 20px;">
					September, 2017
				</A><P><B>2017-09-01</B>   (1)Fusion PCR: Ampicillin resistance, B0015
					　　　　　(2) Three pieces PCR: 5NSII, 3NSII, and PBR322 ori<B>2017-09-02</B>  (1) Ligation: Ampicillin resistance, B0015+ 5NSII, 3NSII, and PBR322 ori
					　　　　　(2) Transformation: pPIGBACK (pigment plasmid backbone)
					　　　　　(3) PCR (Dream Taq): CrtZ and PrbCL<B>2017-09-03</B>  (1) Colony PCR: pPIGBACK
					　　　　　(2) Liquid culture: pPIGBACK
					　　　　　(3) PCR (KOD): CrtZ and PrbCL<B>2017-09-04</B>  (1) Plasmid extraction: pPIGBACK
					　　　　　(2) Digestion check: pPIGBACK
					　　　　　(3) Fusion PCR: CrtZ+ PrbCL<B>2017-09-05</B>  (1) pPIGBACK construct finish
					　　　　　(2) Fusion PCR: PrbCL+IndC
					　　　　　(3) Ligation: CrtZ+ PrbCL with pPIGBACK 
					　　　　　(4) Transformation: CrtZ construct<B>2017-09-06</B>  (1) Colony pcr: CrtZ construct
					　　　　　(2) Liquid culture: CrtZ
					　　　　　(3) Lycopene primers design 
					　　　　　(4) IndC primers redesign<B>2017-09-07</B>  (1) plasmid extraction: CrtZ-Ppigback
					　　　　　(2) restriction enzyme check: CrtZ-pPIGBACK<B>2017-09-08</B>  (1) transform CrtZ-Ppigback into PCC7942
					　　　　　(2) nitrate and nitrite adsorption assay of Chlorella vulgaris (co-coculture with NrtA)
					　　　　　(3) transform suicide mechanism construct (LBA+glucose 0.3%)<B>2017-09-09</B>  (1) nitrate and nitrite adsorption assay of Chlorella vulgaris (co-coculture with NrtA)
					　　　　　(2) fusion PCR: PrbcL+indC<B>2017-09-10</B>   indC construct (~transformation)<B>2017-09-19</B>   (1) Endolysin-Holin construct
					　　　　　(2) Site-directed primers design<B>2017-09-20</B>  (1) Liquid culture: BBa_J04450
					　　　　　(2) Liquid culture: CrtZ-PCC7942 clones<B>2017-09-21</B>   Plasmid extraction: J04450, CrtZ-PCC7942<B>2017-09-22</B>   Site-directed PCR: 3NSII<B>2017-09-23</B>   Site-directed-3NSII construct finish<B>2017-09-24</B>  (1) plasmid extraction: Site-directed-3NSII
					　　　　　(2) Site-directed PCR: 5NSII-1
					　　　　　(3) Psb1c3-PrbcL construct finish<B>2017-09-25</B>  (1) Psb1c3-NrtA construct finish
					　　　　　(2) colony PCR: Psb1c3-PrbcL
					　　　　　(3) Liquid culture: Psb1c3-PrbcL 
					　　　　　(4) IndC construct<B>2017-09-26</B>  (1) Plasmid extraction and restriction enzyme check: Site-directed-5NSII-1
					　　　　　(2) Liquid culture: Psb1c3-NrtA
					　　　　　(3) Colony PCR: IndC 
					　　　　　(4) Psb1c3-3NSII construct finish 
					　　　　　(5) Site-directed PCR: 5NSII-2<B>2017-09-27</B>  (1) Psb1c3-NrtA construct finish
					　　　　　(2) Plasmid extraction and restriction enzyme check: IndC
					　　　　　(3) Plasmid extraction and restriction enzyme check: IndC 
					　　　　　(4) Psb1c3-3NSII construct finish Plasmid extraction and RE check: Site-directed 5NSII-2 
					　　　　　(5) Site-directed PCR: AmpR<B>2017-09-28</B>  (1) Plasmid extraction and RE check: Site-directed-AmpR
					　　　　　(2) Plasmid extraction and RE check: Psb1c3-5NSII
					　　　　　(3) Psb1c3-3NSII(SD) finish
					　　　　　(4) Liquid culture: Psb1c3-PrbcL<B>2017-09-29</B>  (1) Psb1c3-5NSII construct finish
					　　　　　(2) Psb1c3-5NSII-ORI-3NSII construct finish
					　　　　　(3) Colony PCR: Psb1c3-PrbcL- IndC 
					　　　　　(4) Colony PCR: Psb1c3-PrbcL- IndC 
					　　　　　(5) Psb1c3-PrbcL(SD) construct finish 
					　　　　　(6) Psb1c3-PrbcL-IndC construct finish 
					　　　　　(7) Site-directed: amp finish 
					　　　　　(8) transform CrtZ-pPIGBACK into PCC7942<B>2017-09-30</B>  (1) Psb1c3-PrbcL(SD) finish
					　　　　　(2) Liquid culture: CrtZ-pPIGBACK

				</P><CENTER><A href="#!" onclick="toggleHeight4(this, 1900);" style="font-size:20px;color:#2c498c">			click to close
				</A></CENTER></DIV></DIV><DIV class="panel"><DIV id="s5" class="expandable" style="height: 24px;padding-top:5px;"><A href="#!" onclick="toggleHeight5(this, 2240); return false" style="font-family: 'Acme', sans-serif;font-size:28px;color: #479fcc;height: 20px;">
					October, 2017
				</A><P><B>2017-10-01</B>   (1) Colony PCR: 5NSII-Psb1c3
					　　　　　(2) Liquid culture: 5NSII-Psb1c3
					　　　　　(3) Colony PCR: 5NSII-ORI-3NSII-Psb1c3
					　　　　　(4) Liquid culture: Site-directed mutated clones
					　　　　　(5) Construct: Psb1c3-PrbcL-IndC
					　　　　　(6) Liquid culture: NrtA-C33

					<B>2017-10-02</B>   (1) Finish all aite-directed mutagenesis
					　　　　　(2) plasmid extraction: Psb1c3-5NSII, 5NSII-ORI-3NSII-Psb1c3
					　　　　　(3) Digestion check: Psb1c3-5NSII, 5NSII-ORI-3NSII-Psb1c3
					　　　　　(4) Colony PCR: Psb1c3-PrbcL-IndC
					　　　　　(5) Liquid culture: Psb1c3-PrbcL-IndC
					　　　　　(6) Liquid culture (French Press): NrtA-C33, competent cell

					<B>2017-10-03</B>   (1) French Press and dialysis
					　　　　　(2) Sequencing: MelA
					　　　　　(3) Construct Psb1c3-NrtA-Endolysin

					<B>2017-10-04</B>   (1) Construct pPIGBACK-J04450
					　　　　　(2) Construct Psb1c3-B0015-AMP
					　　　　　(3) Construct Psb1c3-PrbcL-CrtZ
					　　　　　(4) check nitrate concentration
					　　　　　(5) Colony PCR: NrtA-endolysin
					　　　　　(6) Liquid culture: NrtA-endolysin

					<B>2017-10-05</B>   (1) Colony PCR: pPIGBACK-J04450
					　　　　　(2) Liquid culture: pPIGBACK-J04450
					　　　　　(3) Digestion check: NrtA-endolysin

					<B>2017-10-06</B>   (1) DNA extraction and digestion-pPIGBACK-CrtZ
					　　　　　(2) Finish construct pPIGBACK-J04450
					　　　　　(3) Colony PCR: Psb1c3-B0015-AMP
					　　　　　(4) Liquid culture: Psb1c3-B0015-AMP

					<B>2017-10-07</B>   (1) Plasmid extraction: Psb1c3-B0015-AMP
					　　(2) Digest check: Psb1c3-B0015-AMP
					　　　　　(3) Construct: Psb1c3-PrbcL-CrtZ

					<B>2017-10-08</B>   (1) Colony PCR: Psb1c3-PrbcL-CrtZ

					<B>2017-10-09</B>   (1) Liquid culture: competent cell for electroporation 
					　　　　　(2) Frech Press
					　　　　　(3) Nitrate concentration assay

					<B>2017-10-10</B>   (1) Electroporation: competent cell

					<B>2017-10-11</B>   (1) Test electroporation: Psb1c3-J04450

					<B>2017-10-13</B>   (1) Construct: Psb1c3-IndC (electroporation)
					　　　　　(2) Construct: pPIGBACK-IndC (electroporation)
					　　　　　(3) Construct: pPIGBACK-PrbcL-CrtZ
					　　　　　(4) Construct: Psb1c3-B0015-AMP
					　　　　　(5) Starch functional test: pPIGBACK-PrbcL-CrtZ
					　　　　　(6) Suicide mechanism functional test

					<B>2017-10-14</B>   (1) Colony PCR: pPIGBACK-IndC
					　　　　　(2) Liquid culture: pPIGBACK-IndC
					　　　　　(3) Colony PCR: pPIGBACK-PrbcL-CrtZ
					　　　　　(4) <I>Chlorella Vulgaris</I> oil functional test: NrtA
					　　　　　(5) Starch functional test: pPIGBACK-PrbcL-CrtZ
					　　　　　(6) Suicide mechanism functional test

					<B>2017-10-15</B>   (1) Finish construct pPIGBACK-PrbcL-IndC, Psb1c3-PrbcL-IndC
					　　　　　(2) Liquid culture: All parts
					　　　　　(3) Starch functional test: pPIGBACK-PrbcL-CrtZ

					<B>2017-10-16</B>   (1) <I>Chlorella Vulgaris</I> oil functional test: NrtA
					　　　　　(2) Plasmid extraction: all parts
					　　　　　(3) Digestion check: all parts
					　　　　　(4) Starch functional test: pPIGBACK-PrbcL-CrtZ

					<B>2017-10-17</B>   (1) NrtA add to device again
					　　　　　(2) Starch functional test: pPIGBACK-PrbcL-CrtZ

					<B>2017-10-18</B>   (1) NrtA add to device again
					　　　　　(2) Starch functional test: pPIGBACK-PrbcL-CrtZ

					<B>2017-10-19</B>   (1) NrtA add to device again
					　　　　　(2) Starch functional test: pPIGBACK-PrbcL-CrtZ

					<B>2017-10-20</B>   (1) NrtA add to device again
					　　　　　(2) Starch functional test: pPIGBACK-PrbcL-CrtZ

					<B>2017-10-21</B>   (1) NrtA add to device again
					　　　　　(2) Starch functional test: pPIGBACK-PrbcL-CrtZ

					<B>2017-10-22</B>   (1) PCR: MelA, IndC
					　　　　　(2) Ligation: Psb1c3-MelA, pPIGBACK-MelA, Psb1c3-IndC, pPIGBACK-IndC
					　　　　　(3) Transformation: Psb1c3-MelA, pPIGBACK-MelA, Psb1c3-IndC, pPIGBACK-IndC

					<B>2017-10-23</B>   (1) PCR: MelA, IndC, Psb1c3-B0015-AMP
					　　　　　(2) Ligation: Psb1c3-MelA, pPIGBACK-MelA, Psb1c3-IndC, pPIGBACK-IndC, Psb1c3-B0015-AMP
					　　　　　(3) Liquid culture: Psb1c3-MelA, pPIGBACK-MelA, Psb1c3-IndC, pPIGBACK-IndC

					<B>2017-10-24</B>   (1) PCR: PrbcL-MelA, Psb1c3-IndC, pPIGBACK-IndC
					　　　　　(2) Ligation: Psb1c3-PrbcL-MelA, pPIGBACK-PrbcL-MelA
					　　　　　(3) Transformation: Psb1c3-PrbcL-MelA, pPIGBACK-PrbcL-MelA, Psb1c3-IndC, pPIGBACK-IndC, Psb1c3-B0015-AMP
					　　　　　(4) Liquid culture: Psb1c3-B0015-AMP

					<B>2017-10-25</B>   (1) Digestion check all parts
					　　　　　(2) Plasmid extraction: Psb1c3-B0015-AMP
					　　　　　(3) Digestion check: Psb1c3-B0015-AMP

				</P><CENTER><A href="#!" onclick="toggleHeight5(this, 2240);" style="font-size:20px;color:#2c498c">
					click to close
				</A></CENTER></DIV></DIV></DIV><DIV id="Labnote"><H1>Labnote</H1><A href="https://static.igem.org/mediawiki/2017/0/04/T--NYMU-Taipei--labnote.pdf" target="_blank">Download the file</A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>