<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NYMU-Taipei_Pigments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NYMU-Taipei/Pigments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:NYMU-Taipei"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NYMU-Taipei/Overview"><DIV class="submenu_button" id="Overview_page">
					Overview
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Pigments"><DIV class="submenu_button" id="Pigments_page">
					Pigments
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Nitrogen_starvation"><DIV class="submenu_button" id="Nitrogen_starvation_page">
					Nitrogen starvation
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Model"><DIV class="submenu_button" id="Model_page">
					Modeling
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Improve"><DIV class="submenu_button" id="Improve_page">
					Improvement
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstration
				</DIV></A></DIV><DIV class="menu_button"> LABWORK
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NYMU-Taipei/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Safety"><DIV class="submenu_button" id="Safety_page">
					Safety
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A></DIV><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NYMU-Taipei/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Gold &amp; Integrated HP
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> ABOUT US
		</DIV><DIV class="submenu_wrapper" id="About_Us_submenu"><A href="https://2017.igem.org/Team:NYMU-Taipei/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NYMU-Taipei/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Model"><DIV class="submenu_button" id="Model_page">
					Modeling
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A><A href="https://2017.igem.org/Team:NYMU-Taipei/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=NYMU-Taipei"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="igem_2017_footer_wrapper "><DIV class="footer_half_size"><P class="footer_title">CONTACT US</P><P class="footer_text">Email us: 2017igem.nymutaipei@gmail.com
				Call us: 886-2-28267316
				Facebook: <A href="https://www.facebook.com/NYMU-iGEM-Team-244897845537138/?fref=ts" target="_blank">NYMU iGEM Team</A></P></DIV><DIV class="footer_half_size"><P class="footer_title">AFFILIATIONS &amp; ACKNOWLEDGMENT</P><P class="footer_text"><A href="https://2017.igem.org/Main_Page" target="_blank">IGEM 2017</A><A href="http://nymu-e.web.ym.edu.tw/bin/home.php" target="_blank">National Yang-Ming University</A><A href="https://2017.igem.org/Team:NYMU-Taipei/Attributions">Special Thanks</A></P></DIV></DIV><BUTTON onclick="topFunction()" id="myBtn" title="Go to top">Top</BUTTON><DIV class="Nitrogen_Starvation"><CENTER><H1>Pigments</H1></CENTER><DIV class="pigments_wrapper"><P>　　In our project, we transfer five types of <FONT class="mark_description">pigment-related gene sequences</FONT> (Indigoidine, Zeaxanthin, Melanin, Astaxanthin and Lycopene) into our cyanobacterial cells. We expect to get five different colors of microalgae, so we could see whether adding other pigment colors to the original color of microalgae would add wavelength absorbance and have <FONT class="mark_description">better photosynthetic efficiencies</FONT>. Due to better photosynthetic efficiencies, we could elevate oil accumulation in microalgae, which would have great benefit in both industry and scientific usage.
				</P></DIV><DIV class="panel"><DIV id="s1" class="expandable" style="height: 30px;padding-top:15px;"><A href="#!" onclick="toggleHeight1(this, 1300); return false" style="font-family:'Acme', sans-serif;font-size:30px;color:#205e1a;height: 30px;">
					Backbone Design (pPIGBACK)
				</A><DIV style="color:#205e1a;"><P>　　To genetically engineer cyanobacteria, we chose <FONT class="mark_backbone"><I>Synechococcus elongatus PCC 7942</I></FONT> as our engineering host. 
					Our main strategy is to embark on <FONT class="mark_backbone"><B>gene double-crossover homologous recombination</B></FONT> in <I>S. elongatus PCC 7942</I> genome, which is the first cyanobacterial strain to be transformed by exogenous DNAs and is reliably transformable through natural uptake of extracellular DNAs.
					</P><P>　　First, we constructed a vector which is able to finish double-crossover homologous gene recombination in <I>S. elongatus PCC 7942</I>. 
					The vector (pPIGBACK) contains <FONT class="mark_backbone">5’- and 3’-ends of the neutral site II (<B>NSII</B>)</FONT> and an <FONT class="mark_backbone">ampicillin resistance gene (<B>AmpR</B>)</FONT> for antibiotic selection. 
					Then we fused AmpR with double terminator, <FONT class="mark_backbone"><B>BBa_B0015</B></FONT>, which is proved to be functional in cyanobacteria. 
					Additionally, to easily manipulate DNAs for gene cloning and plasmid preparation in <I>E. coli DH5α</I>, the <FONT class="mark_backbone">replication origin (<B>ORI</B>)</FONT> of pBR322 was also introduced to make the plasmid vector replicable in <I>E. coli</I>.
					Then, in order to overexpress foreign genes in the cyanobacteria, the <FONT class="mark_backbone">intrinsic promoter of Rubisco large subunit (<B>PrbcL</B>)</FONT> was chosen as the target for vector construction. 
					PrbcL regulates the expression of the most abundant proteins in photosynthetic species and has been proven to have a high activity to express foreign genes, so we chose PrbcL as the promoter of our pigment gene.<SUP>1</SUP></P><P>　　The strategy we chose to construct the vector is to fuse B0015 and AmpR together first. 
					Secondly, we fused 5’- and 3’-ends of the neutral site II (NSII) with PBR322 replication origin (ORI) together. 
					At last, we ligated two parts together. 
					The vector (pPIGBACK) is used to transform into PCC7942 with the inserted pigment gene in our experiments.
					After mass reproduction in <I>E. coli DH5α</I>, <I>PCC7942</I> were transformed through the uptake of plasmid DNAs extracted from <I>E. coli DH5α</I>. 
					The transformed strains (transformants) were usually successfully obtained after 2 to 3 weeks and survived the ampicillin treatment.
					</P><CENTER><A href="#!" onclick="toggleHeight1(this, 1300);" style="color:#56914a">
					click to close
					</A></CENTER></DIV></DIV></DIV><DIV class="panel" style="background-color:rgba(222, 236, 247, 0.9)"><DIV id="s2" class="expandable" style="height: 30px;padding-top:15px;"><A href="#!" onclick="toggleHeight2(this, 900); return false" style="font-family:'Acme', sans-serif;font-size:30px;color:#002384;height: 30px;">
					Indigoidine (IndC)
				</A><DIV style="color:#0e1f28;"><P>　　Indigoidine is cyan pigment produced by natural-existing bacteria. Due to the resemblance of Indigoidine and industrial dye indigo, we believe that Indigoidine will find uses in both industry and biology fields. In nowadays studies, an <FONT class="mark_blue">Indigoidine synthetase Sc-IndC</FONT> and an associated helper protein <FONT class="mark_blue">Sc-IndB</FONT> were identified from <I>Streptomyces chromofuscus ATCC 49982</I> and successfully expressed in <I>Escherichia coli BAP1</I> to produce the blue pigment<SUP>2</SUP>. The IndB gene codes for a putative phosphatase and the IndC gene codes for Indigoidine synthase. Together, these enzymes <FONT class="mark_blue">convert L-glutamine into Indigoidine</FONT>. Recently, it has been shown that <FONT class="mark_blue">IndC alone can produce Indogoidine, and the inclusion of IndB expression in the system will increase yields significantly</FONT><SUP>3</SUP>.</P><P>　　As we know, L-Glutamine is the direct biosynthetic precursor of Indigoidine, and it is a key amino acid in primary metabolism and thus naturally exists in <I>S. elongatus PCC7942</I>. Because glutamine related products are already existed in <I>S. elongatus PCC7942</I>, we only need to <FONT class="mark_blue">express the gene Sc-IndC in <I>S. elongatus PCC7942</I> which leads to the production of Indigoidine</FONT>. However, due to the access difficulties of <I>Streptomyces chromofuscus ATCC 49982</I>, we decided to use the previous part for IndC, which has been submitted to the iGEM Parts Registry (BBa_K1152008)<SUP>4</SUP>. According to the part design, our Indigoidine gene comes from <I>Photorhabdus luminescens laumondii TT01 (DSM15139)</I>.</P><CENTER><A href="#!" onclick="toggleHeight2(this, 900);" style="color:#2c498c">
					click to close
					</A></CENTER></DIV></DIV></DIV><DIV class="panel" style="background-color:#rgba(246, 255, 178, 0.9)"><DIV id="s6" class="expandable" style="height: 30px;padding-top:15px;"><A href="#!" onclick="toggleHeight6(this, 780); return false" style="font-family:'Acme', sans-serif;font-size:30px;color:#f6ff00;height: 30px;text-shadow: 1px 1px 1px #393a1f;">
					Zeaxanthin (CrtZ)
				</A><DIV style="color:#393a1f;"><P>　　Zeaxanthin belongs to carotenoid family and is widely found in the nature. It is also a natural color making corns, carrots and marigolds yellow. Moreover, zeaxanthin is an essential nutrient substance to human’s eyes, and some healthy supplements are made of it. Most of green plants produce zeaxanthin as an intermediate in carotenoid pathway. However, some photosynthetic bacteria such as cyanobacteria lack of zeaxanthin. Therefore, we try to <FONT class="mark_yellow">transform zeaxanthin-related genes to cyanobacteria to make them yellow</FONT><SUP>6</SUP>.
					</P><P>　　After literature search, we did not find research articles on this subject. Under our instructor’s help, we develop a way to make cyanobacteria yellow. We compare the complete carotenoid pathway with <I>Synechococcus elongatus</I> PCC 7942 whole genomic DNA on KEGG and we find every zeaxanthin-related gene is included in PCC 7942 genomic DNA except <FONT class="mark_yellow">beta-carotene hydroxylase (crtZ) </FONT>. And we find crtZ coding sequence with ribosome-binding site is an igem released part (BBa_I742158), which is from a plant pathogen, <I>Pantoea ananatis</I><SUP>5</SUP>.
					</P><P>　　We had successfully transformed CrtZ to <I>Escherichia coli</I> to reproduce massively, and then transformed CrtZ with pPIGBACK to <I>Synechococcus elongatus</I> PCC 7942. After a week, the transformed <I>Synechococcus elongatus</I> PCC 7942 <FONT class="mark_yellow">expressed more yellow than the control group</FONT>. To test whether the photosynthetic efficiency is better, we used iodine to measure starch concentration and compare it with wild type.<SUP>7, 8</SUP></P><CENTER><A href="#!" onclick="toggleHeight6(this, 780);" style="color:#666838;">
					click to close
					</A></CENTER></DIV></DIV></DIV><DIV class="panel" style="background-color:rgba(186, 186, 186, 0.9)"><DIV id="s3" class="expandable" style="height: 30px;padding-top:15px;"><A href="#!" onclick="toggleHeight3(this, 800); return false" style="font-family:'Acme', sans-serif;font-size:30px;color:#303030;height: 30px;">
					Melanin (MelA)
				</A><DIV style="color:#000000;"><P>　　Melanin, a biological pigment widely found in terrestrial flora and fauna, is a complex oxidation product of amino acid tyrosine. Melanin plays diverse roles in a myriad of organisms. As far as the human is concerned, melanin is the primary determinant of human skin color and pupils or irises of the eyes. It is also an important signal molecule in the human neural system. For microorganisms, melanin would protect them by against ultraviolet radiation effect from sunlight, which is detrimental to most of the organisms.</P><P>　　Due to the fact that the dark pigment derived from <FONT class="mark_black">MelA gene has extensive wavelength absorbance</FONT>, we decided to transform MelA gene combined with particular constitute promoter into <I>Synechococcus elongatus</I> PCC 7942 to measure the growth curve and photosynthetic efficiency of it. Based on the well-elaborated procedure provide by IGEM Tokyo Tech 2009<SUP>9</SUP>, we had intended to replicate their experiment to produce melanin massively first in <I>E. coli</I>, but failed to succeed due to inconsistence of DNA sequence. Therefore, we had no alternative but to turn to DNA synthesis and directly transform MelA gene ligated with our backbone into microalgae.</P><CENTER><A href="#!" onclick="toggleHeight3(this, 800);" style="color:#333333">
					click to close
					</A></CENTER></DIV></DIV></DIV><DIV class="panel" style="background-color:rgba(255, 234, 234, 0.9)"><DIV id="s4" class="expandable" style="height: 30px;padding-top:15px;"><A href="#!" onclick="toggleHeight4(this, 900); return false" style="font-family:'Acme', sans-serif;font-size:30px;color:#af0e0e;height: 30px;">
					Astaxanthin (CrtW &amp; CrtZ)
				</A><DIV style="color:#722525;"><P>　　Astaxanthin is a high value and natural pink pigment which can be found in microalgae, yeast and some sea creatures. It is special due to its antioxidant activity and has been suggested being beneficial in cardiovascular, immune, inflammatory and neurodegenerative diseases and skin health.  Although it has lots of benefits, astaxanthin is still a product result in a minority amount in the carotenoid synthesis pathway compare with other carotenoid families and yet, the artificial chemical synthesis cost high and result in the least production.
					</P><P>　　Astaxanthin synthesis does not naturally exist in the <I>S. elongatus</I> PCC7942. But fortunately, after literature review, we found out that <I>S. elongatus</I> PCC7942 has a similar pathway with other microalgae which can synthesize astaxanthin, and the only difference is,  <I>S. elongatus</I> PCC7942 lack of two necessary gene: <FONT class="mark_red">beta-carotene ketolase (crtW)</FONT> and <FONT class="mark_red">beta-carotene hydroxylase (crtZ) </FONT> to undergo this pathway.<SUP>10, 11</SUP> Thus, we used IDT service to synthesize these two genes and construct it on pPIGBACK, a vector which can express the carrying genes in <I>S. elongatus</I> PCC7942.
					</P><CENTER><A href="#!" onclick="toggleHeight4(this, 900);" style="color:#994040">
					click to close
					</A></CENTER></DIV></DIV></DIV><DIV class="panel" style="background-color:rgba(255, 234, 234, 0.9)"><DIV id="s8" class="expandable" style="height: 30px;padding-top:15px;"><A href="#!" onclick="toggleHeight8(this, 2100); return false" style="font-family:'Acme', sans-serif;font-size:30px;color:#af0e0e;height: 30px;">
					Lycopene (CrtL knockout)
				</A><DIV style="color:#722525;"><P>　　Lycopene is a kind of carotenoid pigment with a bright red color. It can be found in tomatoes and other red fruits and vegetables. In photosynthesis, Lycopene plays a role in absorbing sunlight (from the wavelength about 460nm to 500nm)<SUP>12</SUP> and transferring the energies to Chlorophyll through electrons. It can also protect Chlorophyll from the damage of sunlight. The structure of Lycopene makes it a potent antioxidant among various common carotenoids. In Ames test, referring to testing whether a given chemical would cause mutations in the DNA, Lycopene shows its great ability to trap singlet oxygen and reduce mutagenesis.<SUP>13</SUP> Numerous studies have also shown that Lycopene has a significant contribution to prevent cardiovascular disease, diabetes, osteoporosis and cancers (especially prostate cancer).  We expect that producing Lycopene in cyanobacteria will <FONT class="mark_red">enhance the efficiency of photosynthesis</FONT>, therefore, leading to the increase of biofuel. Moreover, Lycopene can <FONT class="mark_red">provide additional values in human health</FONT> as we mentioned above.</P><P>　　In order to retain Lycopene in cyanobacteria <I>Synechococcus</I> sp. PCC7942, our chassis organism, we used the strategy－“<FONT class="mark_red">Gene Knock Out</FONT>”. Through our studies, we found that Lycopene is an intermediate in the biosynthesis pathway of beta-carotenoids in cyanobacteria <I>Synechococcus</I> sp. PCC7942<SUP>14</SUP> There is an enzyme, called Lycopene cyclase3, converts acyclic hydrocarbon Lycopene into the bicyclic Beta-carotene. Once we can <FONT class="mark_red">knock out the Lycopene cyclase gene (CrtL)</FONT>, the Lycopene cyclase won’t exist. If the lycopene cyclase doesn’t exist, the Lycopene will remain and not be transformed into other compounds. With the accumulation of Lycopene, meeting the goal of our project is anticipated.</P><P>　　We construct a plasmid to knock out the Lycopene cyclase gene (CrtL). The plasmid contains the “upstream of CrtL” (we call it <FONT class="mark_red"> “Lycopene-US”</FONT>), the downstream of CrtL(we call it <FONT class="mark_red"> “Lycopene-DS”</FONT>) and <FONT class="mark_red">Ampicillin resistant gene(AmpR)</FONT>. First, we used PCC7942 as our template to do the polymerase chain reaction (PCR) to gain the sequence of Lycopene-US and Lycopene-DS. And we used the backbone pPIGBACK we had made as the PCR template to gain AmpR sequence. Second, with the primers we designed, we conducted a <FONT class="mark_red">three pieces fusion PCR</FONT> to connect these three sequences as the following order：<FONT class="mark_red">Lycopene-US + AmpR + Lycopene-DS</FONT>.</P><P>　　Third, the constructed plasmid containing” Lycopene-US + AmpR + Lycopene-DS” is transformed into PCC7942. There is a probability for Homologous Recombination<SUP>11</SUP> to occur in the genome of PCC7942 through this transferred plasmid. The Lycopene cyclase gene (CrtL) will be replaced by AmpR in Homologous Recombination. Finally, the “Gene Knock Out” is completed.</P><P>　　To conform to the iGEM part registry, we use IDT service to synthesize the DNA sequences of Lycopene-US and Lycopene-DS. Compared with two sequences from PCC7942 wild type, the IDT sequence synthesized sequences will not contain the Restriction Enzyme sites (RE site) of PstI and XbaI because we have changed codons in each RE site according to the codon usage table of PCC7942. The submission parts to the iGEM Parts Registry are 
					<FONT class="mark_red">　　BBa_K2350007(<I>Lycopene-US</I>) 
					　　BBa_K2350008 (<I>Lycopene-DS</I>).
					　　BBa_K2350014 (<I>Lycopene-US + AmpR + Lycopene-DS</I>).</FONT></P><CENTER><A href="#!" onclick="toggleHeight8(this, 2100);" style="color:#994040">
					click to close
					</A></CENTER></DIV></DIV></DIV><DIV class="panel"><DIV id="s9" class="expandable" style="height: 30px;padding-top:15px;"><A href="#!" onclick="toggleHeight9(this, 3900); return false" style="font-family:'Acme', sans-serif;font-size:30px;color:#205e1a;height: 30px;">
					Functional Test
				</A><DIV style="color:#205e1a;"><P>　　We expect the transformed PCC7942 with CrtZ gene could produce zeaxanthin and become more yellow. Just like our prediction, the results show that <FONT class="mark_backbone">the transformed PCC7942 is more yellow than wild type apparently</FONT>.</P><P>　　Moreover, we expect the transformed PCC7942 which produces zeaxanthin would change wavelength absorbance and have better photosynthetic efficiency.</P><H4>Wavelength Absorbance</H4><P>　　To test this hypothesis, we used spectrophotometer to measure the absorbance of CrtZ and wild type at 400 to 700 nm. The outcome is that <FONT class="mark_backbone">the OD value of CrtZ transformant at 400 to 500 nm is higher than wild type</FONT>. The result verified our first prediction, a change of wavelength absorbance. Not only this, the change of wavelength absorbance is at 400 to 500 nm, which is blue light. This indicated CrtZ transformant can absorb blue light and reflects yellow light, so CrtZ transformant is more yellow than wild type. The measurement matches what we saw intuitively.</P><H4>Photosynthetic Efficiency</H4><P>　　To test whether the photosynthetic efficiency of CrtZ transformant is better than wild type, we used <FONT class="mark_backbone">iodine to measure starch concentration</FONT>. First, the initial concentration of microalgae of CrtZ transformants and wild type should be the same, so that the measurement could be fair. We measured the OD value at 730 nm, which represented the concentration of microalgae. Then we calculated how much microalgae and BG11 (the medium) we should added to make the same amount of microalgae in each plate. Second, we started to measure starch concentration. We measured the OD value of each plate at 730 nm, which represented the cell number. Then we added 50 μl iodine into each cuvette, waited for five minutes, and measured the OD value at 620 nm, which represented the starch content. We repeated this step consequently for seven days. Here are our results.</P><P>　　Then we divided the starch content by cell number, and we knew there were how much starch in every unit cell.</P><P>　　Then we calculated the variation of starch content per cell per day.</P><P>　　The first day, wild type (WT) had more starch than CrtZ transformants. However, the increase of starch per day of CrtZ transformants is more than wild type. The results imply that <FONT class="mark_backbone">CrtZ transformants can produce more starch per day than the wild type</FONT>. Moreover, the results correspond to our hypothesis -- CrtZ transformants actually have better photosynthetic efficiency!</P><H4>Survival Ability</H4><P>　　We also are considering about the survival ability of CrtZ transformants. Therefore, we conducted a competitive experiment of CrtZ transformants to wild type. We co-cultured CrtZ transformant and wild type together, and then observed the color after seven days. Similarly, the initial concentration of microalgae of CrtZ transformant and wild type should be the same, and we used the same method we used in photosynthetic efficiency experiments.</P><P>　　After seven days, <FONT class="mark_backbone">CrtZ transformant and wild type both survived</FONT>. The result indicates that CrtZ transformant is strong and had great survival ability comparable to wild type!</P><CENTER><A href="#!" onclick="toggleHeight9(this, 3900);" style="color:#56914a">
					click to close
					</A></CENTER></DIV></DIV></DIV><DIV class="panel"><DIV id="s5" class="expandable" style="height: 30px;padding-top:15px;"><A href="#!" onclick="toggleHeight5(this, 860); return false" style="font-family:'Acme', sans-serif;font-size:30px;color:#205e1a;height: 30px;">
					Conclusion
				</A><DIV style="color:#205e1a;"><P>　　pPIGBACK-CrtZ was transformed into <I>Synechoccocus elongatus</I> PCC7942 after 20 days cultivation, the electrophoresis result based on PCR CrtZ gene from extracted genomic DNA of transformants is showed below. <FONT class="mark_backbone">Transformation efficiency</FONT> of pPIGBACK-CrtZ is 11.4 transformants per μg DNA, and correctness is 52% (10/19), which is relatively high compared to low successful rate of gene recombination.</P><P>　　Therefore, we can conclude that pPIGBACK is quite a reliable vector which could accomplish gene double-crossover homologous recombination in <I>S. elongatus</I> PCC 7942 genome, because the successful rate of gene double-crossingover homologous recombination is low in cyanobacteria. Moreover, when comparing pPIGBACK-CrtZ transformants with wild type, we can assure that pPIGBACK could be express in <I>S. elongatus</I> PCC 7942, which is such a milestone in our project. With <FONT class="mark_backbone">the high correctness of pPIGBACK-CrtZ transformants</FONT>, we have the confidence that multiple colors of cyanobacteria is possible and could be functional in the near future.</P><DIV style="width:70%;margin:auto;"><P style="font-size:18px;">This figure is pPIGBACK-CrtZ transformants electrophoresis result of CrtZ gene amplified from extracted transformants genomic DNA. C1~C20 represents the pPIGBACK-CrtZ transformant clone 1 to clone 20, and M represents 1 Kb marker.</P></DIV><CENTER><A href="#!" onclick="toggleHeight5(this, 860);" style="color:#56914a">
					click to close
					</A></CENTER></DIV></DIV></DIV><DIV class="panel"><DIV id="s7" class="expandable" style="height: 30px;padding-top:15px;"><A href="#!" onclick="toggleHeight7(this, 960); return false" style="font-family:'Acme', sans-serif;font-size:30px;color:#205e1a;height: 30px;">
					Reference
				</A><DIV style="color:#205e1a;font-size:16px;"><OL><P><LI>Pei-Hong Chen, Hsien-Lin Liu, Yin-Ju Chen, Yi-Hsiang Cheng, Wei-Ling Lin, Chien-Hung Yeh and Chuan-Hsiung Chang. (2012). Enhancing CO2 bio-mitigation by genetic engineering of cyanobacteria. <I>Energy Environ. Sci., 2012,5,</I> 8318-8327.
					</LI></P><P><LI>Alexander O. Brachmann, Ferdinand Kirchner, Carsten Kegler, Sebastian C. Kinski, Imke Schmitt, and Helge B. Bode. (2011). Triggering the production of the cryptic blue pigment indigoidine from <I>Photorhabdus luminescens</I>. <I>Journal of Biotechnology, 157,</I> 96-99.
					</LI></P><P><LI>Xu, F., Gage, D., and Zhan, J. (2015). Efficient production of indigoidine in <I>Escherichia coli</I>. <I>Journal of Industrial Microbiology &amp; Biotechnology, 42,</I> 1149–1155.		
					</LI></P><P><LI>2013 Team: Heidelberg <A href="https://2013.igem.org/Team:Heidelberg/Project/Indigoidine">(https://2013.igem.org/Team:Heidelberg/Project/Indigoidine)</A></LI></P><P><LI>Copeland,A. et al.(2014). Carotenoid pathway of <I>Synechococcus elongatus</I> PCC 7942, complete genomegatus PCC 7942. Unpublished.
					</LI></P><P><LI>Shinichi Takaichi(2011). Carotenoids in Algae: Distributions, Biosyntheses and Functions. <I>Mar Drugs. 2011; 9(6):</I> 1101–1118.
					</LI></P><P><LI>Chuan-Hsiung Chang(2012). Enhancing CO2 bio-mitigation by genetic engineering of cyanobacteria. <I>Energy Environ. Sci., 2012, 5,</I> 8318.
					</LI></P><P><LI>NCBI <A href="https://www.ncbi.nlm.nih.gov/nuccore/CP000100">(https://www.ncbi.nlm.nih.gov/nuccore/CP000100)</A></LI></P><P><LI>2009 Team: Tokyo Tech <A href="https://2009.igem.org/Team:Tokyo_Tech">(https://2009.igem.org/Team:Tokyo_Tech)</A></LI></P><P><LI>Nadja A. Henke, Sabine A. E. Heider, Petra Peters-Wendisch and Volker F. Wendisch. (2016). Production of the Marine Carotenoid Astaxanthin by Metabolically Engineered <I>Corynebacterium glutamicum</I>. <I>Marine drug, 14(7): 124.</I></LI></P><P><LI>Chengwei Liang, Fangqing Zhao, Wei Wei, Zhangxiao Wen, Song Qin. (2006). Carotenoid Biosynthesis in Cyanobacteria: Structural and Evolutionary Scenarios Based on Comparative Genomics. <I>Int. J. Biol. Sci., 2006, 2.</I></LI></P><P><LI>Miller, E. S., Mackinney, G., &amp; Zscheile, F. P. (1935). Absorption Spectra of Alpha and Beta Carotenes And Lycopene. <I>Plant Physiology, 10</I>(2), 375–381.
					</LI></P><P><LI>David Heber, Qing-Yi Lu (2002). Overview of Mechanisms of Action of Lycopene. <I>Experimental Biology and Medicine, 227(10)</I>, 920 – 923.
					</LI></P><P><LI>Cunningham, F. X., Sun, Z., Chamovitz, D., Hirschberg, J., &amp; Gantt, E. (1994). Molecular structure and enzymatic function of lycopene cyclase from the cyanobacterium <I>Synechococcus</I> sp strain PCC7942. <I>The Plant Cell, 6</I>(8), 1107–1121.
					</LI></P><P><LI>Jacobus, A. P., &amp; Gross, J. (2015). Optimal Cloning of PCR Fragments by Homologous Recombination in <I>Escherichia coli</I>. <I>PLoS ONE, 10</I>(3), e0119221.
					</LI></P></OL><CENTER><A href="#!" onclick="toggleHeight7(this, 960);" style="color:#56914a">
					click to close
					</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>