Team:HZAU-China/Results
Home
Project
Description
Design
Results
Demonstrate
Wetlab
Experiments
Improve
InterLab
Drylab
Model
Software
Hardware
HP
Silver HP
Integrated and Gold
Team
Members
Collaborations
Attributions
Safety
Parts
Results The function of dCas9 and gRNA We first construct the gRNA targeting at OriC and test if it could work as we expected. Through OD measurement we             observed an inhibition of growth when dCas9 and gRNA are all expressed.
Figure 1. In this figure, several controls are conducted to demonstrate that it is the function of CRISPR that affect             the growth of culture not the function of gOriC nor the toxicity of dCas9. The purple line represents the experimental             group while the others represent different controls. For further description of this synchronization method, we quantifying the number of oriC, the origin of chromosome replication, versus terC, the terimination of chromosome replication, by using qPCR. The results confirm that the initiation             of chromosome replication is inhibited.
Figure 2. In this figure, the blue column represents the ratio of oriC to terC of cells growing in common condition. The red column represents that of cells suppressed by dCas9-oriC             for 1 hour and the green column for 2 hours. The purple column shows the ratio of oriC to terC of cells treated             with Rifampicin, a widly used chemical agent to block chromosome replication initiation. The results shows that             the efficiency of dCas9-oriC system is relatively comparable to this widly used agent. Flow cytometry is also used to conduct the run-out experiment to verify the synchronization.
Figure 3. In this figure, different peaks represent different oriC numbers. The untreated sample have 4 peaks while             after treated with dCas9-oriC system the copy number of the same sample has reduced to 2 peaks, which shows that             the oriC number of each cell is reduced. According to our results, we can conclude that the dCas9 and gRNA(oriC) can be used to synchronize the cell cycles             of E. coli. The light induced CRISPR/dCas9 system at transcription level An optimized CcaS-CcaR system is constructed under the design of our advisor, the performance of optimized system             is as following.
We next insert the sequence of gRNA(OirC) after the Pcpcg2, and measure the OD performance under the green light             or red light separately.
The result shows a difference in growth rate under green light and red light. But simply repression cannot achieve             a computer-controlled system. We further tested if this regulation is reversible. For more information please             click DEMONSTRATION. The light induced CRISPR/dCas9 system at protein level After getting DNA from IDT, we first constructed and tested the function of pMag and nMag through Luciferase assay.
The result verifies the function of pMag and nMag. We next tested the expression of split dCas9 to check if it is hydrolysis by SDS-PAGE.
After inserting it into pET plasmid, the results turnout that the protein is not hydrolysis. The last reason we doubt is that if it is because the split method that makes the split dCas9 canâ€™t complement. We replaced the pMag and nMag with a more reliable chemical induced dimerization protein, FKBP & FRB. The result             shows that no inhibition of growth is observed after adding rapamycin, which induces the dimerization of FKBP             and FRB (data not shown). In summary, in this project we achieved controlling cell in the level of DNA replication, proved the feasibility             of using CRISPR to regulate cell cycle and further regulate it with light. A reverse regulation of our system             is demonstrated by OD measurement and qPCR. At the same time, corresponding software and hardware are developed             to achieve the computer-controlled cell cycle. The software is used to predict the internal replication procession             and control the hardware to regulate cell by light. For more information, please view Demonstrate. Notes
When using the CcaS-CcaR system please note that the atc concentration is different from the former concentration                 200ng/ml. After transformation of two plasmids related to CcaS-CcaR system into bacteria, it becomes more                 sensitive to aTc. A final concentration of 200ng/ml will be toxic to the bacteria, and a working concentration                 as 0.1~1ng/ml is recommended.
In a light-control system, the existence of light sensing molecule is very important, but it is also very easy to be neglected.
The aTc may lose its inducing function if added into a relatively high concentration culture (OD~0.6) when you                 use CcaS-CcaR system.
Future plan The future plan includes two aspects. One is further developing the system. First we plan to replace the inducible             promoter with a constitutive promoter J23115. We plan to developing CcaS-CcaR CRISPRi system by knocking it into             genome to make it more significant. At the same time, we hope that we can make the pMag-nMag system work because             this system is smaller and less-affecting on bacterial metabolism. The other aspect is using this system for several             certain applications, like characterizing the stability of genome gene expression after synchronization, and so on.             We plan to knock the sfGFP into the genome and measure the fluorescence per cell to determine the noise between             each cell and characterize whether synchronization can make the gene on genome more stable.
