<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HZAU-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HZAU-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav_body"><UL id="nav_organ"><LI class="menu_item"><A class="nav_tissue" href="https://2017.igem.org/Team:HZAU-China">Home</A></LI><LI class="menu_item"><A class="nav_tissue">Project</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Demonstrate">Demonstrate</A></LI></UL><LI class="menu_item"><A class="nav_tissue">Wetlab</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/InterLab">InterLab</A></LI><LI class="menu_item"><A class="nav_tissue">Drylab</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Software">Software</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Hardware">Hardware</A></LI><LI class="menu_item"><A class="nav_tissue">HP</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/HP/Gold_Integrated">Integrated and Gold</A></LI><LI class="menu_item"><A class="nav_tissue">Team</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Attributions">Attributions</A></LI><LI class="menu_item"><A class="nav_tissue" href="https://2017.igem.org/Team:HZAU-China/Safety">Safety</A></LI><LI class="menu_item"><A class="nav_tissue" href="https://2017.igem.org/Team:HZAU-China/Parts">Parts</A></LI></DIV><DIV class="HZAU_div_main"><A class="HZAU_title">Results</A><A class="fubiaoti">The function of dCas9 and gRNA</A><A class="zhengwen">We first construct the gRNA targeting at OriC and test if it could work as we expected. Through OD measurement we
            observed an inhibition of growth when dCas9 and gRNA are all expressed.</A><A class="tuzhu_wenzi"> Figure 1. In this figure, several controls are conducted to demonstrate that it is the function of CRISPR that affect
            the growth of culture not the function of gOriC nor the toxicity of dCas9. The purple line represents the experimental
            group while the others represent different controls.</A><A class="zhengwen">For further description of this synchronization method, we quantifying the number of
            <I>oriC</I>, the origin of chromosome replication, versus
            <I>terC</I>, the terimination of chromosome replication, by using qPCR. The results confirm that the initiation
            of chromosome replication is inhibited.</A><A class="tuzhu_wenzi"> Figure 2. In this figure, the blue column represents the ratio of
            <I>oriC</I> to
            <I>terC</I> of cells growing in common condition. The red column represents that of cells suppressed by dCas9-oriC
            for 1 hour and the green column for 2 hours. The purple column shows the ratio of oriC to terC of cells treated
            with Rifampicin, a widly used chemical agent to block chromosome replication initiation. The results shows that
            the efficiency of dCas9-oriC system is relatively comparable to this widly used agent. </A><A class="zhengwen">Flow cytometry is also used to conduct the run-out experiment to verify the synchronization.</A><A class="tuzhu_wenzi"> Figure 3. In this figure, different peaks represent different oriC numbers. The untreated sample have 4 peaks while
            after treated with dCas9-oriC system the copy number of the same sample has reduced to 2 peaks, which shows that
            the oriC number of each cell is reduced.</A><A class="zhengwen">According to our results, we can conclude that the dCas9 and gRNA(oriC) can be used to synchronize the cell cycles
            of
            <I>E. coli</I>.</A><A class="fubiaoti">The light induced CRISPR/dCas9 system at transcription level</A><A class="zhengwen">An optimized CcaS-CcaR system is constructed under the design of our advisor, the performance of optimized system
            is as following.</A><A class="zhengwen">We next insert the sequence of gRNA(OirC) after the Pcpcg2, and measure the OD performance under the green light
            or red light separately.</A><A class="zhengwen_disblock">The result shows a difference in growth rate under green light and red light. But simply repression cannot achieve
            a computer-controlled system. We further tested if this regulation is reversible. For more information please
            click </A><A class="zhengwen_disblock" href="https://2017.igem.org/Team:HZAU-China/Demonstrate">DEMONSTRATION.</A><A class="fubiaoti">The light induced CRISPR/dCas9 system at protein level</A><A class="zhengwen">After getting DNA from IDT, we first constructed and tested the function of pMag and nMag through Luciferase assay.</A><A class="zhengwen">The result verifies the function of pMag and nMag. We next tested the expression of split dCas9 to check if it is hydrolysis by SDS-PAGE.</A><A class="zhengwen">After inserting it into pET plasmid, the results turnout that the protein is not hydrolysis.</A><A class="zhengwen">The last reason we doubt is that if it is because the split method that makes the split dCas9 canâ€™t complement.</A><A class="zhengwen">We replaced the pMag and nMag with a more reliable chemical induced dimerization protein, FKBP &amp; FRB. The result
            shows that no inhibition of growth is observed after adding rapamycin, which induces the dimerization of FKBP
            and FRB (data not shown).</A><A class="zhengwen_disblock">In summary, in this project we achieved controlling cell in the level of DNA replication, proved the feasibility
            of using CRISPR to regulate cell cycle and further regulate it with light. A reverse regulation of our system
            is demonstrated by OD measurement and qPCR. At the same time, corresponding software and hardware are developed
            to achieve the computer-controlled cell cycle. The software is used to predict the internal replication procession
            and control the hardware to regulate cell by light. For more information, please view</A><A class="zhengwen_disblock" href="https://2017.igem.org/Team:HZAU-China/Demonstrate">Demonstrate.</A><A class="biaoti"> Notes</A><LI><A class="zhengwen">When using the CcaS-CcaR system please note that the atc concentration is different from the former concentration
                200ng/ml. After transformation of two plasmids related to CcaS-CcaR system into bacteria, it becomes more
                sensitive to aTc. A final concentration of 200ng/ml will be toxic to the bacteria, and a working concentration
                as 0.1~1ng/ml is recommended.</A></LI><LI>In a light-control system, the existence of light sensing molecule is very important, but it is also very easy to be neglected.</LI><LI>The aTc may lose its inducing function if added into a relatively high concentration culture (OD~0.6) when you
                use CcaS-CcaR system.</LI><A class="biaoti"> Future plan</A><A class="zhengwen">The future plan includes two aspects. One is further developing the system. First we plan to replace the inducible
            promoter with a constitutive promoter J23115. We plan to developing CcaS-CcaR CRISPRi system by knocking it into
            genome to make it more significant. At the same time, we hope that we can make the pMag-nMag system work because
            this system is smaller and less-affecting on bacterial metabolism. The other aspect is using this system for several
            certain applications, like characterizing the stability of genome gene expression after synchronization, and so on.
            We plan to knock the sfGFP into the genome and measure the fluorescence per cell to determine the noise between
            each cell and characterize whether synchronization can make the gene on genome more stable.
        </A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>