<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HZAU-China_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HZAU-China/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav_body"><UL id="nav_organ"><LI class="menu_item"><A class="nav_tissue" href="https://2017.igem.org/Team:HZAU-China">Home</A></LI><LI class="menu_item"><A class="nav_tissue">Project</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Demonstrate">Demonstrate</A></LI></UL><LI class="menu_item"><A class="nav_tissue">Wetlab</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/InterLab">InterLab</A></LI><LI class="menu_item"><A class="nav_tissue">Drylab</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Software">Software</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Hardware">Hardware</A></LI><LI class="menu_item"><A class="nav_tissue">HP</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/HP/Gold_Integrated">Integrated and Gold</A></LI><LI class="menu_item"><A class="nav_tissue">Team</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:HZAU-China/Attributions">Attributions</A></LI><LI class="menu_item"><A class="nav_tissue" href="https://2017.igem.org/Team:HZAU-China/Safety">Safety</A></LI><LI class="menu_item"><A class="nav_tissue" href="https://2017.igem.org/Team:HZAU-China/Parts">Parts</A></LI></DIV><DIV class="HZAU_div_main"><A class="HZAU_title">Experiment</A><A class="biaoti">Materials and methods</A><A class="fubiaoti">Plasmid construction</A><A class="zhengwen">The fragment was PCR amplified with pfu(company) or PrimeSTAR(Takara) according to product length. Product is recycled with gel extraction kit from qiagene after electrophoresis. Golden gate is conducted with total 10 liter containing 100 ng template, 5 liter buffer, 5U BsaI, and 10 U T4 ligase. With 50 cycles with 37 5min and 16 10min finished with 55 5min and deactivate at 98 for 5 min. Then transformed into trans5a using commercial component cell (xxx). homologous recombination to conducted using vyzyme xxxxx kit and operate following the guidebook.</A><A class="fubiaoti">The OD measurement</A><A class="zhengwen">Cell are cultured overnight in LB broth containing corresponding antibiotics, and diluted into 1% fresh LB broth. When OD is reaching 0.6 add aTc of final concentration 200 ng/ ml to induce expression of dCas9 and inject it into 24 cell culture plate. Culture the plate in 37 and 150 rpm. Every hour put it into a plate reader to measure its OD.</A><A class="fubiaoti">Quantitative PCR</A><A class="zhengwen">The qPCR experiment is conducted using qPCR kit from takara(Code No.RR820). The sample is collected every 30min and freeze at liquid nitrogen after centrifugal. The genome is for qPCR is extracted using Code No.9763(takara).</A><A class="fubiaoti">Flow cytometry </A><A class="zhengwen">Bacteria shall be cultured in LB until OD is between 0.6 and 1.0 (37℃, 200r/min). Dilute the culture medium with LB to approximately 108 cells mL-1. Add rifampicin (rif) and cephalexin (ceph) to the final concentration is 200μg/mL and 12μg/mL. Culture in the previous condition for 4 hours and the bacteria can be then used for flow cytometry.
All culture samples were centrifuged at 16000g for 1 min and resuspended in ice-cold ethanol to a final concentration of 70%. Samples in 70% ethanol can be stored at 4℃ until they were prepared for flow cytometry. Cells were pelleted by centrifugation at 16000g for 1min at 4℃. Wash in 10mM Tris (pH=8)/ 10mM MgCl2 (buffer A) as before and resuspend in buffer A. Take 0.5μL of the sample. Add it into 10μL DAPI(1μg/mL) diluted with buffer A and dye for 5min in dark. Dilute the sample to 1mL with buffer A and it can be used for flow cytometry analysis (Becton-Dickinson).</A><A class="fubiaoti">The Agarose Pad Assay</A><A class="zhengwen">To acquire the growth rate of bacteria for establishing the growth function, an agarose pad assay is conducted. Mix the 0.15g agarose and 10ml LB to reach the concentration of 1.5%,then heat it and add needed matter after cooling slightly.The melted agarose is supposed to be pipetted onto a cover glass(The volume is about 2.5ml.).Then put another clean cover glass on it to get a smooth flat surface after about 3 minutes. The next step is to pipette about 2.5 μL E.coli solution of 0.02 OD and get a new cover glass on it till the solution dries.It’s worth noting that maintain the favorable environment is vital.We used parafilm to avoid the evaporation and tinfoil to keep out the light when observed dCas9 4-1 which was induced by aTc. Observe the agarose pad under microscope to find the optimum bacteria, for instance, which are under the fission.The final thing you do to take and save picture 10 minutes per in 8 to 10 hours.</A><A class="fubiaoti">Microfluidics</A><A class="zhengwen">To better observe the growth of E.coli, we adopted 3D-Microfluidics tech to take real-time observation on monolayer bacteria, which is more visualized and convenient to understand the division of a single cell.
     Our steps of experiment are below：
1.	Growing spdCas9/gRNA OriC3 E.coli cultures in LB(100ng/ml ampicillin and 70ng/ml Chloramphenicol) overnight(200rpm, 37℃).
2.	Take 1ml bacteria culture and add it into 100ml Erlenmeyer flask which contains 100ml LB（100ng/ml ampicillin and 70ng/ml Chloramphenicol）and cultivate for 2~3h（200rpm, 37℃). When OD get near to 0.6, concentrate 100ml bacteria culture to 5ml.
3.	Use 2.5ml sterile syringe to collect 2.5ml concentrated, 5ml sterile syringe for LB(ampicillin and  Chloramphenicol) and 5ml LB(ampicillin and Chloramphenicol) containing 200ng/ml aTc.
4.	Injecting the bacteria culture through the tiny tube into the microfluidic chip. E.coli flow into the chamber and grow in there.
5.	Firstly, inject LB without aTc at the speed of 200ul/h, observe by inverted microscope for 10h. Then inject LB with 200ng/ml aTc at the speed of 200ul/h, observe by inverted microscope for 10h.
</A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>