"
Page
Discussion
View source
History
teams
Log in
 
Team:Purdue/Notebook
From 2012.igem.org
Home
Team
Members
Advisors
Sponsors
Attributions
Project
Overview
Design
Protocol
Results
Parts
Modeling
Human Practices
Biomaker Bench
Girl Scouts Biotechnology Badge
Notebook
Safety
May 2012
S
M
T
W
T
F
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
 
 
June 2012
S
M
T
W
T
F
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July 2012
S
M
T
W
T
F
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
 
 
 
 
August 2012
S
M
T
W
T
F
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
 
September 2012
S
M
T
W
T
F
S
 
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
Monday, May 15
List of useful contacts on the google doc, including DowAgro
Begin to list the different devices/constructs that will be used in our project
Attachment (adhesion)
Filtration
Modularize the sequence so we can test individually (e.g. put a fluorescent protein coding sequence at the end of each segment – construct silica binding protein first with constitutive promoter/repressible promoter to produce fluoresent protein and make sure it does what you think it should)
Investigate multiple silica binding protein (SBP);must choose several top candidates for each element.
Hierarchy
Perfecting the FFL
And (low affinity, not dimer), Or (high affinity)
Schematic Diagram
Modeling and Experimental
Communication in terms of data (e.g.  kinetic parameters)
Review Characterization Data Sheets
Strong integration of modeling translates to a strong performance in the competition.
List of things we need
Competent Cells (Laris Avramova (core molecular biologist, 222), Tarun (electron microscopy)may have the needed cells)
Antibiotics (AMP, tetracycline)
Enzymes (Pst1, Xba1, EcoR1, Spe1, Ligase, polymerase/PCR reagents, T5exonuclease )
Parts (available in the registry)
Constitutive promoter (orthogonal t7 promoter)
Signaling Promoters (investigate the precedent for construction FFL)
RBS –(B0034)
Thermodynamic models for designing RBSs, etc (Voights model)
Terminators
Proteins Transcription Factors
Monday, May 21
We're all looking forward to an exciting iGEM summer! Our SURF students have just arrived and are gradually being introduced to synthetic biology and iGEM.
Sean gave a crash course on synthetic biology to Mrudula, Rachel, Amanda and August. The powerpoint is available here , compiled by our wonderful graduate mentor, Janie.
Tuesday, May 22
First Journal Club Meeting - we identified the reducible elements of our system, detailed an outline of the project to the SURF students (e.g. What is the advantage of using this entire process? Is not it kind of roundabout?)
For the NEXT MEETING Tuesday 29th May :
Identify, in these teams:
a. What Adhesion system we want to use (Amanda, Peter, Mrudula)
b. Which silica-binding system we want to use (Rachel, August)
c. Control Elements (Max, Mrudula, Rachel, Sean)
d. Find strain auxotrophic for RSC (gene which breaks down arabinose) (Jim)
Announcements:
Be ready to explain your assigned element of the project (starting generally and moving more specifically)
Read the introductory/background elements of the thesis that Dr. Rickus will put in the dropbox
General Announcements
Be ready to work the BioBuilder HighSchool Teacher iGEM workshop during the week of June 4th – June 8th (more details to come)
Everyone is welcome to visit Drs. Rickus and Clase’s lab group meeting on Thursday at 12PM
Thursday May 24th
Update on the human practice component available through the Powerpoint
Tuesday, May 29
Overview of small group presentations
Adhesion Proteins – Amanda, Mrudula, and Peter
Ag43 : [University of Queensland 2009] PROS: makes chains instead of aggregates, works well in flow, in the registry, abiotic adherence CONS: not concomitant
TibA: PROS: modular, concomitant, auto-transport CONS: not in the registry
AIDA-1: PROS: binds Ag43 and self, in registry, higher shear tolerance CONS: only expressed in certain cells, blocked by Fimbriae (due to length)
FimA-H: [Michigan 2010] PROS: forms chains, compatible w/ E. coli, shear resistant, grows in constant flow, binds well to glycoproteins CONS: inhibits function of other proteins
Curli: [Lyons 2011] PROS: well characterized in registry, amyloid fibers CONS: inhibits Ag43 and AIDA-1
Decision:
primary: AIDA-1 [order from registry and improve bad sequencing or re-synthesize]
secondary (if AIDA-1 proves unfeasible): TibA [with goal of characterizing part, must synthesize]
Silica Binding Proteins – Rachel and August
INP – Silicatein [MN 2011] – PROS: CONS: very large, no data on viability, not biobrick compatible.
OmpA-Silicatein alpha fusion – PROS: shorter than INP-Silicatein, active in neutral pH, no illegal sites, known to work CONS: must construct fusion peptide vector NOTES: optimum at low temperatures [OmpA - K103006]
R5 peptide: PROS: active at neutral pH, small, has been used in E. coli CONS: part of a larger protein (no silaffin post-translational modifications), contains EcoRI site
Modeling the Network Motif - Max, Mrudula, Rachel, and Sean
Modeled the simplified system
Matlab and MathCad Model
Need concrete entry parameters for more robust models
For Next week:
Construct your parts in DNA 2.0 and anything in registry start detailing
Wednesday, May 30
In Lab:
Cleaned and organized the laboratory space
Ordered laboratory suppleies
Made LB agar plates and LB liquid media with ampicillin
Created the adhesions and SBP devices in silico using Gene Designer by DNA 2.0
Thursday, May 31
Attended Rickus and Porterfield laboratory group meeting
Each person introduced themselves and their research
Decided on the use of future meetings
Researched primer design and began to design primers. These primers will be used for PCR to perform the Gibson assembly method
Friday, June 1
Performed Cell Transformations of the following parts:
Lac promotor and CFP generator (BBa_I13601)
PoPs receiver (BBa_F2620)
Tet repressor (BBa_C0040)
Tet promotor with green fluorescent protein (BBa_I13522)
The transformation protocol can be found here
Completed Primer Sequences for surface expression protein
Monday, June 4
Welcomed Bio Builders workshop participants
Janie gave a powerpoint on the basics of synthetic biology to the workshop participants
Amanda, August, Max, Mrudula, and Rachel helped the participants with an experiment based on MIT's 2006 iGEM project, a banana scent generator
Mrudula and Amanda completed minipreps for the previously transformed parts. The results of the minipreps are shown below.
Tet repressor: 79.8ng/µL
PoPs receiver: 149.6ng/µL
Lac promotor and CFP generator: 46.2ng/µL
Tet promotor wih GFP: 22.9ng/µL
These concentrations are too low to use in any assembly method; therfore, we will need to attempt transformations and minipreps again before doing any assembly method.
Tuesday, June 5
Amanda, August, Mrudula, Rachel, and Soo spent the morning watching presentations on abstraction and devices with the Bio Builder workshop participants  Soo gave a presentation to the participants on how to use TinkerCell  The iGEM team helped  the Bio Builder participants to transform green and purple fluorescent protein into E. coli
Created a timeline for the rest of the summer as seen below
Week 3:
Design and order primers for Gibson assembly
Order reagents
Complete a miniprep for the previously transformed parts
Select plasmid backbones
Week 4:
PCR parts for Gibson assembly
Complete Gibson assembly
Decide on which parts will be synthetic
What devices can be synthesized or should we synthesize individual parts?
Create competent cells
Week 5:
Complete plasmid midi and mega preps
Transform parts into expression vectors
Week 6:
Complete functional analysis of protein expression
Construct the full system
Week 7:
Build the flow system for biofilm establishment
Complete functional analysis of biofilm thickness
Week 8-10:
Iterate and improve on flow setup
Wednesday, June 6
Sean gave a presentation to the Bio Builder participants on 3A and Gibson assembly  Sean lead a 3A assembly lab with the participants  Worked on designing primers
Thursday, June 7
Continued to design primers
Monday, June 11
We attempted to optimize our primers using online guidlines and software.
Here is a website that details primer design.
We used a calculator and an Oligo Analyzer which can calculate the liklihood of occurances such as hairpins and dimers.
Upon analysis, we found that the primers that would be required to use Gibson assembly were not ideal. In some cases, the forward and the reverse primers did not have Tm values that were close enough to each other. We also had difficulty with making our primers a reasonable length. Furthermore, many of the primers had a high probability (Gibson free energy value lower than zero) of froming dimers or hairpins. Finally, we deicided that it would most likely be more successful to use 3A assembly instead of Gibson.  In order to save time and ensure the accuracy of the sequence, we decided to synthesize instead of assemble the whole silica making device in a high copy plasmid.  Welcomed Rubeena to the iGEM team. We are looking forward to working with her!
Wednesday, June 13
August, Mrudula, and Rachel were trained on how to use the flow cytometer  Made competent cells (DH5a). The protocol for making competent cells is found here
Thursday, June 14
Rachel had a demonstration on how to make silica. The procedure is shown here . We welcomed Chris to the iGEM team. We know he will be a great asset to the team! We had an overview meeting for Rubeena and Chris to help them catch up on the project.
Friday, June 15
We transformed all of the parts needed for the adhesion device. These included the following
AIDA-1 (BBa_K257018)
Plasmid (pSB1AK3)
pTet (BBa_R0040)
RBS (BBa_B0034)
Terminator (BBa_B0015)
Plasmid (pSB1AC3)
CFP (BBa_E0020)
Here is a powerpoint summarizing our team meeting.
Saturday, June 16
Removed the following parts from the incubator and placed in the refrigerator with parafilm wax
BBa_E00032 (GFP device from plates)
Bba_
Sunday, June 17
We reevaluated our previous decision to synthesize the silica making device. We have now decided to use Gibson assembly to create the OmpA-silicatein fusion sequence and 3A assembly for the rest of the device. We believe that this will save money and, considering the time taken to order and deliver the sequence, will not take any more time than synthesizing.
Monday, June 18
Completed minipreps on recently transformed parts. The results are shown below.
GFP: 17.5ng/µL and 7.6 ng/µL
AIDA-1: 8.7ng/µL and 6ng/µL
pSB1AC3: 5.2ng/µL snd 4.2ng/µL
CFP: 13.6ng/µL and 12.7ng/µL
RBS (BBa_B0032): 24.9ng/µL and 24.2ng/µL
PoPs: 107.8ng/µL and 131.4ng/µL
pSB1AK3: 7.0ng/µL
Terminator: 11.5ng/µL
RBS (BBa_B0034): 21.0ng/µL and 21.5ng/µL
pTet: 5.2ng/µL
These concentrations are too low to proceed with 3A assembly. Therefore, we are going to grow more of the transformed bacteria, and we are going to perform more minipreps.
Tuesday, June 19
August completed a second round of minipreps . The results are listed below.
RBS (BBa_B0034): 14.8ng/µL
RBS (BBa_C0040): 5.0ng/µL
CFP: 9.0ng/µL
PoPs: 14.9ng/µL
AIDA-1: 7.0ng/µL
Here is a powerpoint summarizing our team meeting.
Wednesday, June 20
Due to the low yields from our previous tranformations and minipreps, we have decided to attempt our transformations with a new protocol. The new protocol can be found here
Today, we transformed pLac, TetR, terminator (BBa_B0015), RBS (BBa_B0034), pSB1A3, and pSB1C3.
The minipreps gave the following results
RBS: 14.8ng/µL
TetR: 5.0ng/µL
CFP: 9.0ng/µL
PoPs: 14.5ng/µL
AIDA-1: 7.0ng/µL
Thursday, June 21
Due to the low yield of our transformations and minipreps, we have now decided to troubleshoot our transformation procedures. To do this, we will be doing a transformation with Tarun, a graduate mentor to the team. We will be using Tarun's procedures and supplies. We will also have a control plate that will contain cells that had no DNA inserted. If these transformations succeed, then we will have narrowed the possibilities of what went wrong with our procedures. Tarun's transformation procedure can be found on the here . We performed these test transformations with BBa_I13602 (CFP).
Upon closer inspection of the AIDA-1 biobrick (BBa_K257018), we have found the following problems
the sequencing is bad
there are large gaps from the sequencing as compared to what was documented. Specifically, the sequenced biobrick was missing approximately 500bp
the sequence was supposed to be 1326bp long. We have decided that it would cost too much to have the biobrick sequenced ourselves
the gel is bad
For the above reasons, we have decided to not use AIDA-1. After reevaluating the other adhesion options, we have decided to use Curli (BBa_342003). Due to its inconsistant sequencing, we have decided to have the part found in the kit sequenced ourselves.
Friday, June 22
Due to our increasing size, we have decided to divide into committees. These committees will help us divide the workload fairly and ensure that all tasks are being completed in a timely manner. The committees and their members are listed below. An asteric indicates the committee head.
Wetlab
OmpA-silicatein
*August
Haefa
Curli
*Rubeena
Purchasing
*Peter
Human Practices
*Peter
*Max
Experimental Design/Characterization
*Amanda
*Mrudula
Chris
Arthi
Rachel
Jim
Project Planning
*Namita
Wiki
*Rachel
Sean (background)
We checked our test transformations from yesterday. None of the plates showed growth.
Here is a powerpoint summarizing our team meeting.
Monday, June 25
The foci of this week are to troubleshoot our transformations and research ways to characterize our constructs once they are assembled. August and Mrudula will be in lab troubleshooting. Amanda and Mrudula will be resarching ways to characterize a biofilm. Rachel will be researching ways to characterize a silica coat.
Tuesday, June 26
Here is a powerpoint summarizing our team meeting.
Friday, June 29
We went to Dow Agro, one of our sponsors, to give them an overview of our project. The powerpoint presentation we gave can be found Powerpoint . After we gave our presentation, we asked the Dow representative, Ryan, if he had any guidance with concern to Gibson assembly.
Saturday, June 30
Part of the team volunteered at the high school Jamboree. We saw many amazing teams with remarkable ideas! Congratulations to all the teams for their hard work over the past semester, and good luck as future iGEMers!
Sunday, July 1
We attempted another test transformation , this time using bought Excel 10ß competent cells. We had both a negative and positive control. We also plated different concentrations of the final solution to see if we were over or under plating. We also used agar plates with ampicillin that Tarun made. We used store bought SOC. The part used was BBa_J04450.
Monday, July 2
The tranformations performed yesterday were successful. Both the positive control and the plaes with our part showed growth. The transformation efficiency was extremely high. We got single colonies on the plates with 50µL and overgrowth on the plates that had 150µL. We will only plate at 50µL from now on.
Due to time constraints, we have decided to synthesize both of our constructs. In parallel to characterization of the synthesized constructs, we are going to assemble both constructs using Gibson assembly. The assembles constructs will differ from the synthesized constructs in that the promoter will be a PoPs device and there will be a fluorescent protein on each. In this way, we can find if the addition of these two characterization parts will affect the functionality of our constructs.
The synthesized constructs were ordered from DNA 2.0 . Pictures of the Gene Designer (DNA 2.0 software) renditions of the three constructs are given below.
Silicatein construct
Curli
We transformed Curli so that we can have it sequenced soon
List of What Needs to get Done this Week:
Research protocols on setting up and running experiments using biofilms
Research flow and static protocols for biofilms
Research ways to characterize silica matrix and acquire quantifiable data
Create timelines for each assay researched
Order reagents and get training on any equipment needed for assays
Tuesday, July 3
We are now awaiting the delivery of our synthesized constructs. In preparation, we are researching the varying types of assays that we may be able to use. You can find information about the different assays we researched can be found here .
Silicatein-Alpha Characterization
We have decided to use scanning electron microscopy (SEM), x-ray diffraction (XRD), and a live dead assay (specifically, the Baclight Bacterial Viability Kit from Invitrogen).   Further assays will be done once the two constructs are used within the same bacteria.  Researched stages of biofilm formation:
1. Initial Attachment  -  attachment occurs via van der waals forces
2. Irreversible Attachment  -  this is where the EPS forms
3. Maturation I  -  matrix formed with adhesion proteins
4. Maturation II  -  the full biofilm forms
5. Dispersion  -  bacteria released
Also researched shear stress modeling, chassis development, microsensors, and confocal microscopy assays.
Furthermore, research was done on different ways to characterize the silica coat. Three extremely helpful articles are listed below.
Cell-Mediated Deposition of Porous Silica on Bacterial Biofilms
Sol-gel synthsis and structure of silica hybrid materials
Thin-film silica sol-gel coatings for neural microelectrodes
A powerpoint highlighting possible characterization methods is given here
After discussion with the team and our faculty advisor, Dr. Rickus, we have decided to use the scanning eletron microscopy (SEM), x-ray diffraction (XRD), and a live/dead assay (specifically, the Baclight Bacterial Viability Kit ).
Thursday, July 5
Researched possible competitors in the field of bioreactor filtration.
Hydro Engineering - produces "Hydrokleen", which is a bioreactor filtration system that would generally be used in industrial plants and acts as a water recycler.
Seven Trent Services - produces Tetra Amphidrone fixed sequencing Batch Biological Filter, a waste water treatment process that uses a biological filter to remove various pollutants
Kinetics -  this company makes a range of water treatment processes and machines that can be used in industry as well as consumer markets
ABEC - produces bioreactors, but could not find more detail into what kind
Friday, July 6
Here is a powerpoint summarizing our team meeting.
Sunday, July 8
Research was done on growing a static biofilm. "Growing and Analyzing Static Biofilms" was a very helpful article.
Created a cell culture to have competent cells for a biofilm culture.
Also did research into developing a static biofilm assay to test how well our bacteria makes biofilm before and after our transformation. "Growing and Analyzing Static Biofilms" by Judisth H. Mernitt, Daniel E. Kadouri, and George A. O'Toole was helpful in determining how we would do this assay.
Monday, July 9
Performed the Growth Rate Assay on NEB Beta and DH5 Alpha E.coli strains to determine the time frame we will use to grow our static biofilms.A typical bacterial growth curve is shown below. This is what we expect our curves to look like.
Below is the results of the growth rate assay for NEB Beta and DH5 alpha. Overall, the plots show the expected shape with a lag, log, and stationary phase.
Also worked out the lab plan for the rest of the week.
Tuesday, July 10
We did the same Growth Rate Assay as yesterday, but with the XL-10 Gold E.coli strain because we think it will work better for the biofilm purposes.
Below is the results of the growth rate assay for XL-10 Gold. The plots show the expected shape with a lag, log, and stationary phase.
Here is the excel file containing all of the data from the three growth rate assays.
Also researched and developed the static biofilm assay that we can do to determine the biofilm rates of our non-transformed E.coli.
Wednesday, July 11
Chris T created primers for a test Gibson reaction. The test construct will be a the plasmid pGA3K3, PoPs device (BBa_F2620), an RFP with RBS(BBa_K093005), and a double terminator (BBa_B0014). The primers can be viewed in this document . We ordered the primers from Integrated DNA Technologies .
Friday, July 13
Performed the Static Biofilm Assay .
Had a team meeting, discussed goals for the following week:
Write out protocols and email them to the entire team for revisions
Plan a meeting to go through and troubleshoot the protocols together
Find a way to determine the optimized concentration of curli
Purchase all materials needed for future protocols
Monday, July 16
Researched how Membrane Aerated Bioreactor work and how we will use one for testing. There are multiple types of Membrane Aerated Biofilms: MABR (the type we will be using in our experiments), Membrane biofilter, and extractive membrane bioreactor (EMB).  These reactors can be in shell and tube or plate and frame configurations.  We will be using the shell and tube configuration. The two limiting substrates are introduced at opposite ends of the bioreactor.  The limiting substrates are C-substrate (carbon substrate) and/or oxygen. The biofilm thickness must be less than the oxygen penetration (50-200 micrometers) but stay less than 150 micrometers for best performance.
Thursday, July 19
We attempted to do PCR for the parts for which Chris made primers . To prove that the PCR was successful, we ran aliquotes of the reaction products through a gel. An image of the gel can be seen here . No bands can be seen for the aliquotes; therfore, we concluded that the PCR did not work. We believe that there was a problem with the cycler; therefore, we will be trying the procedure again with a different machine.
Monday, July 23
We attempted PCR again with a different machine. Unfortunately, we used too high of a concentration of primers. Therefore, when we ran a gel, two bands appeared for each sample. The excess primer would greatly lower the efficiency of a Gibson reaction. Furthermore, the bands on the gel all had similar size even though the parts are not of similar size. Therefore, we believe that nonspecific amplification took place. An image of the gel can be seen here .
There were some modifications to the Static Biofilm Assay Protocol.  Once the plates had grown for a long enough time period (every hour for 1-8 hours and then 24 hours and 48 hours) they were transferred to the 4 degree freezer.  The plates were then removed, the excess liquid was dumped out, and the plates were then sprayed off with water and dumped out again.  The plates were than stained with crystal violet with the following steps:
add 125 microliters of crystal violet to each well plate
let sit for 15 minutes
wash off (Note: at this point the plates are stable for several weeks)
Analyzing Plates:
500 microliters of 80% ethanol | 20% acetone were added to each well
300 microliters (or enough until the well were full) were taken from the 24 well plate to the 96 well plates
this was then analyzed using the spectrophotometer to determine the OD of each sample
Tuesday, July 24
We attempted PCR for a third time. This time, the gel shows single bands of varying size. An image of the gel can be seen here . We are now going to proceed to the Gibson reaction.
Upload
Thursday, July 26
Purpose: To make a solution to grow XL-10 gold strain of E.coli so that it can be integrated into a membrane aerated biofilm reactor (MABR)  Materials:
SOB - 250 mL
10 mM MgCl2 (.238 g)
20 mM glucose (.900 g)
250 microliters of cultured XL-10 gold culture
Procedure: Add all the ingredients together in a flask and swirl.  Place in 37 degree room and shake at 220 rpm. Check OD.
Feeding Solution for current biofilm:
200 mL nutrient (glucose and nitrogen)
1mL solution 2 and 3
100 microliters solution 1
Friday, July 27
Purpose: Ruoxi Wu i helping set up the membrane aerated biofilm reactor (MABR)
Now that we have the culture growing, we have to determine what media will be the best that can be used to sustain the cells once they are in the reactor.  The nutrient solutions that Wen, Jane, and Ruoxi are using are specific to Pseudomanas aeruginosa. This research is to determine the protocol for the bioreactor specifically the media.  Research for media: Design and performance study of a novel immobolized hollow fiber membrane bioreactor by Ping Wang (1st author)
nutrients are supplied to the forming biofilm through diffusion (very poor efficiency)
only the cells in a thin layer close to the surface of oxygen and media are active
The bacterial strain used in this paper is E.coli DM5d
The growth limiting substate is M9 medium
Glucose concentration in medium is equal to 5 g/L
Protocol from Paper to culture bacterial cells:
Agar plates with single columns Fl streaking DH5d
new agar plates made every 14 days
pick single colony into 100 mL media
culture overnight at 37 degrees Celsius and 120 rpm
check OD 600
Researched preparation of M9 minimal media recipe .
Monday, August 6
Purpose: PBS is used to clean out the waste from the bioreactor   Materials:
NaCl
KCl
Na2HPO4
KH2PO4
Distilled H20
HCl
Procedure:
Combine 80 g NaCl, 2 g KCl, 14.4 g Na2HPO4 (dibasic anhydrous), 18.1 g Na2HPO4 * 2H2O (dibasic dihydrate), 27.2 g Na2HPO4 * 7H20 (dibasic heptahydrate), 2.4 g KH2PO4 (monobasic anhydrous) and 800 mL H20
Adjust pH to 2.4 with HCl
Add H20 to 1 L
Autoclave
Friday, August 17
Purpose: To measure the biofilm's detachment and viability after growing for 15 days   Materials:
ISOTON
PBS
Small Biocontainer
Tweezers and scissors
Beaker
Procedure:
Turn off the pump and take out the feed loop fl the beaker
Lay the bioreactor on its side and let enough of the media solution drain out so that one biofilm is revealed (collect media in a beaker)
Remove the biofilm using tweezers
Cut into small 1.5" - 2" pieces
Place in containers and pour some of the media into the dishes to keep the biofilms submerged
This biofilm is now ready to be tested   Purpose: The purpose of this experiement is to determine how much of the biofilm comes off (detaches) from the tube when placed under different water flows   Materials:
Biofilm holders
ISOTON
Procedure:
Once the biofilm has been extracted from the bioreactor and it is in small pieces, this procedure can be started
Places the small pieces in the holder that Wen zang constructed
Set up a pump to pump water into the tube and back out into a waste container
Make sure the biofilm is completely submerged the entire time
Collect the PBS that flows over the biofilm in a beaker
Measure the amount of run off and the time to determine the flow rate
Save the sample
To determine the amount of cells in the sample a Multisizer 4 Coulter Counter was used - this tells the concentration of particles of different sizes found in the sample
Different flow rates were used to analyze the biofilm
Retrieved from " http://2012.igem.org/Team:Purdue/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
