<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Purdue_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Purdue/Notebook">Page               </A></LI><LI><A href="/Talk:Team:Purdue/Notebook">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Purdue/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Purdue/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Purdue/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Purdue/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="cssmenu" z-index:5000=""><UL><LI><A href="https://2012.igem.org/Team:Purdue"><SPAN>Home</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Team"><SPAN>Members</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Advisors"><SPAN>Advisors</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Sponsors"><SPAN>Sponsors</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Attributions"><SPAN>Attributions</SPAN></A></LI></UL><LI><A href="https://2012.igem.org/Team:Purdue/Project"><SPAN>Project</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Project"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Design"><SPAN>Design</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Protocol"><SPAN>Protocol</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Results"><SPAN>Results</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Parts"><SPAN>Parts</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Modeling"><SPAN>Modeling</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Human_Practices"><SPAN>Human Practices</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Biomaker_Bench"><SPAN>Biomaker Bench</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Biomaker_Badge"><SPAN>Girl Scouts Biotechnology Badge</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Purdue/Safety"><SPAN>Safety</SPAN></A></LI></DIV><TABLE style="display: inline-block; border: 1px solid; float: left; " id="calendar" cellspacing="0" cellpadding="0" summary="May"><CAPTION> May 2012 </CAPTION><TBODY><TR><TH scope="col" abbr="Sunday" title="Sunday">S</TH><TH scope="col" abbr="Monday" title="Monday">M</TH><TH scope="col" abbr="Tuesday" title="Tuesday">T</TH><TH scope="col" abbr="Wednesday" title="Wednesday">W</TH><TH scope="col" abbr="Thursday" title="Thursday">T</TH><TH scope="col" abbr="Friday" title="Friday">F</TH><TH scope="col" abbr="Saturday" title="Saturday">S</TH></TR><TR><TD> </TD><TD> </TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD></TR><TR><TD>6</TD><TD>7</TD><TD>8</TD><TD>9</TD><TD>10</TD><TD>11</TD><TD>12</TD></TR><TR><TD>13</TD><TD>14</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#5/15">15</A></TD><TD>16</TD><TD>17</TD><TD>18</TD><TD>19</TD></TR><TR><TD>20</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#5/21">21</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#5/22">22</A></TD><TD>23</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#5/24">24</A></TD><TD>25</TD><TD>26</TD></TR><TR><TD>27</TD><TD>28</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#5/29">29</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#5/30">30</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#5/31">31</A></TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><TABLE style="display: inline-block; border: 1px solid; " id="calendar" cellspacing="0" cellpadding="0" summary="June"><CAPTION> June 2012 </CAPTION><TBODY><TR><TH scope="col" abbr="Sunday" title="Sunday">S</TH><TH scope="col" abbr="Monday" title="Monday">M</TH><TH scope="col" abbr="Tuesday" title="Tuesday">T</TH><TH scope="col" abbr="Wednesday" title="Wednesday">W</TH><TH scope="col" abbr="Thursday" title="Thursday">T</TH><TH scope="col" abbr="Friday" title="Friday">F</TH><TH scope="col" abbr="Saturday" title="Saturday">S</TH></TR><TR><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/1">1</A></TD><TD>2</TD></TR><TR><TD>3</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/4">4</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/5">5</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/6">6</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/7">7</A></TD><TD>8</TD><TD>9</TD></TR><TR><TD>10</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/11">11</A></TD><TD>12</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/13">13</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/14">14</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/15">15</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/16">16</A></TD></TR><TR><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/17">17</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/18">18</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/19">19</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/20">20</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/21">21</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/22">22</A></TD><TD>23</TD></TR><TR><TD>24</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/25">25</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/26">26</A></TD><TD>27</TD><TD>28</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/29">29</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#6/30">30</A></TD></TR></TBODY></TABLE><TABLE style=" display: inline-block; border: 1px solid; " id="calendar" cellspacing="0" cellpadding="0" summary="July"><CAPTION> July 2012 </CAPTION><TBODY><TR><TH scope="col" abbr="Sunday" title="Sunday">S</TH><TH scope="col" abbr="Monday" title="Monday">M</TH><TH scope="col" abbr="Tuesday" title="Tuesday">T</TH><TH scope="col" abbr="Wednesday" title="Wednesday">W</TH><TH scope="col" abbr="Thursday" title="Thursday">T</TH><TH scope="col" abbr="Friday" title="Friday">F</TH><TH scope="col" abbr="Saturday" title="Saturday">S</TH></TR><TR><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/1">1</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/2">2</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/3">3</A></TD><TD>4</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/5">5</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/6">6</A></TD><TD>7</TD></TR><TR><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/8">8</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/9">9</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/10">10</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/11">11</A></TD><TD>12</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/13">13</A></TD><TD>14</TD></TR><TR><TD>15</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/16">16</A></TD><TD>17</TD><TD>18</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/19">19</A></TD><TD>20</TD><TD>21</TD></TR><TR><TD>22</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/23">23</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/24">24</A></TD><TD>25</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/26">26</A></TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#7/27">27</A></TD><TD>28</TD></TR><TR><TD>29</TD><TD>30</TD><TD>31</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><TABLE style="display: inline-block; border: 1px solid; " id="calendar" cellspacing="0" cellpadding="0" summary="August"><CAPTION> August 2012 </CAPTION><TBODY><TR><TH scope="col" abbr="Sunday" title="Sunday">S</TH><TH scope="col" abbr="Monday" title="Monday">M</TH><TH scope="col" abbr="Tuesday" title="Tuesday">T</TH><TH scope="col" abbr="Wednesday" title="Wednesday">W</TH><TH scope="col" abbr="Thursday" title="Thursday">T</TH><TH scope="col" abbr="Friday" title="Friday">F</TH><TH scope="col" abbr="Saturday" title="Saturday">S</TH></TR><TR><TD> </TD><TD> </TD><TD> </TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD></TR><TR><TD>5</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#8/6">6</A></TD><TD>7</TD><TD>8</TD><TD>9</TD><TD>10</TD><TD>11</TD></TR><TR><TD>12</TD><TD>13</TD><TD>14</TD><TD>15</TD><TD>16</TD><TD><A href="https://2012.igem.org/Team:Purdue/Notebook#8/17">17</A></TD><TD>18</TD></TR><TR><TD>19</TD><TD>20</TD><TD>21</TD><TD>22</TD><TD>23</TD><TD>24</TD><TD>25</TD></TR><TR><TD>26</TD><TD>27</TD><TD>28</TD><TD>29</TD><TD>30</TD><TD>31</TD><TD> </TD></TR></TBODY></TABLE><TABLE style="display: inline-block; border: 1px solid; " id="calendar" cellspacing="0" cellpadding="0" summary="September"><CAPTION> September 2012 </CAPTION><TBODY><TR><TH scope="col" abbr="Sunday" title="Sunday">S</TH><TH scope="col" abbr="Monday" title="Monday">M</TH><TH scope="col" abbr="Tuesday" title="Tuesday">T</TH><TH scope="col" abbr="Wednesday" title="Wednesday">W</TH><TH scope="col" abbr="Thursday" title="Thursday">T</TH><TH scope="col" abbr="Friday" title="Friday">F</TH><TH scope="col" abbr="Saturday" title="Saturday">S</TH></TR><TR><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD>1</TD></TR><TR><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD><TD>8</TD></TR><TR><TD>9</TD><TD>10</TD><TD>11</TD><TD>12</TD><TD>13</TD><TD>14</TD><TD>15</TD></TR><TR><TD>16</TD><TD>17</TD><TD>18</TD><TD>19</TD><TD>20</TD><TD>21</TD><TD>22</TD></TR><TR><TD>23</TD><TD>24</TD><TD>25</TD><TD>26</TD><TD>27</TD><TD>28</TD><TD>29</TD></TR></TBODY></TABLE><H2 id="5/15">
Monday, May 15
</H2><H5> List of useful contacts on the google doc, including DowAgro </H5><H5> Begin to list the different devices/constructs that will be used in our project </H5><H5> Attachment (adhesion)</H5><H5> Filtration </H5><UL><LI>Modularize the sequence so we can test individually (e.g. put a fluorescent protein coding sequence at the end of each segment – construct silica binding protein first with constitutive promoter/repressible promoter to produce fluoresent protein and make sure it does what you think it should)</LI><LI>Investigate multiple silica binding protein (SBP);must choose several top candidates for each element.</LI></UL><H5> Hierarchy </H5><H5> Perfecting the FFL </H5><UL><LI> And (low affinity, not dimer), Or (high affinity)</LI><LI><A href="https://static.igem.org/mediawiki/2012/d/d5/Schematic.jpg">Schematic Diagram</A></LI></UL><H5> Modeling and Experimental </H5><UL><LI> Communication in terms of data (e.g.  kinetic parameters)</LI><LI> Review Characterization Data Sheets </LI><LI> Strong integration of modeling translates to a strong performance in the competition. </LI></UL><H5> List of things we need </H5><UL><LI> Competent Cells (Laris Avramova (core molecular biologist, 222), Tarun (electron microscopy)may have the needed cells)</LI><LI> Antibiotics (AMP, tetracycline)
</LI><LI> Enzymes (Pst1, Xba1, EcoR1, Spe1, Ligase, polymerase/PCR reagents, T5exonuclease )
</LI><LI> Parts (available in the registry)
</LI><LI> Constitutive promoter (orthogonal t7 promoter)</LI><LI> Signaling Promoters (investigate the precedent for construction FFL)</LI><LI> RBS –(B0034)</LI><UL><LI> Thermodynamic models for designing RBSs, etc (Voights model) </LI></UL><LI> Terminators </LI><LI> Proteins Transcription Factors </LI></UL><H2> Monday, May 21 </H2><UL><H5> We're all looking forward to an exciting iGEM summer! Our SURF students have just arrived and are gradually being introduced to synthetic biology and iGEM. </H5><H5> Sean gave a crash course on synthetic biology to Mrudula, Rachel, Amanda and August. The powerpoint is available <A href="https://static.igem.org/mediawiki/2012/2/24/SynBioIntroduction.ppt">here</A>, compiled by our wonderful graduate mentor, Janie.</H5></UL><H2> Tuesday, May 22 </H2><H5>
 First Journal Club Meeting - we identified the reducible elements of our system, detailed an outline of the project to the SURF students (e.g. What is the advantage of using this entire process? Is not it kind of roundabout?)</H5><H5> For the NEXT MEETING Tuesday 29th May : </H5><UL><H5>	Identify, in these teams: </H5><LI> a.	What Adhesion system we want to use (Amanda, Peter, Mrudula) </LI><LI> b.	Which silica-binding system we want to use (Rachel, August) </LI><LI> c.	Control Elements (Max, Mrudula, Rachel, Sean) </LI><LI> d.	Find strain auxotrophic for RSC (gene which breaks down arabinose) (Jim) </LI></UL><H5> Announcements: </H5><UL><LI>	Be ready to explain your assigned element of the project (starting generally and moving more specifically) </LI><LI>	Read the introductory/background elements of the thesis that Dr. Rickus will put in the dropbox </LI></UL><H5> General Announcements </H5><UL><LI>	Be ready to work the BioBuilder HighSchool Teacher iGEM workshop during the week of June 4th – June 8th (more details to come) </LI><LI>	Everyone is welcome to visit Drs. Rickus and Clase’s lab group meeting on Thursday at 12PM </LI></UL><H2> Thursday May 24th </H2><H5>Update on the human practice component available through the <A href="https://static.igem.org/mediawiki/2012/1/1b/CommunitylabsNewTemplate.ppt">Powerpoint</A></H5><H2> Tuesday, May 29 </H2><H5> Overview of small group presentations </H5><H5> Adhesion Proteins – Amanda, Mrudula, and Peter </H5><UL><LI> Ag43 : [University of Queensland 2009] PROS: makes chains instead of aggregates, works well in flow, in the registry, abiotic adherence CONS: not concomitant </LI><LI> TibA: PROS: modular, concomitant, auto-transport CONS: not in the registry </LI><LI> AIDA-1: PROS: binds Ag43 and self, in registry, higher shear tolerance CONS: only expressed in certain cells, blocked by Fimbriae (due to length)</LI><LI> FimA-H: [Michigan 2010] PROS: forms chains, compatible w/ E. coli, shear resistant, grows in constant flow, binds well to glycoproteins CONS: inhibits function of other proteins </LI><LI> Curli: [Lyons 2011] PROS: well characterized in registry, amyloid fibers CONS: inhibits Ag43 and AIDA-1 </LI></UL><H5> Decision: </H5><UL><LI> primary: AIDA-1 [order from registry and improve bad sequencing or re-synthesize] </LI><LI> secondary (if AIDA-1 proves unfeasible): TibA [with goal of characterizing part, must synthesize] </LI></UL><H5> Silica Binding Proteins – Rachel and August </H5><UL><LI> INP – Silicatein [MN 2011] – PROS: CONS: very large, no data on viability, not biobrick compatible. </LI><LI> OmpA-Silicatein alpha fusion – PROS: shorter than INP-Silicatein, active in neutral pH, no illegal sites, known to work CONS: must construct fusion peptide vector NOTES: optimum at low temperatures [OmpA - K103006]</LI><LI> R5 peptide: PROS: active at neutral pH, small, has been used in E. coli CONS: part of a larger protein (no silaffin post-translational modifications), contains EcoRI site </LI></UL><H5> Modeling the Network Motif - Max, Mrudula, Rachel, and Sean </H5><UL><LI> Modeled the simplified system </LI><LI> Matlab and MathCad Model </LI><LI> Need concrete entry parameters for more robust models </LI></UL><H5> For Next week: </H5><UL><LI> Construct your parts in DNA 2.0 and anything in registry start detailing </LI></UL><H2> Wednesday, May 30 </H2><H5> In Lab: </H5><UL><LI> Cleaned and organized the laboratory space </LI><LI> Ordered laboratory suppleies </LI><LI> Made <A href="http://www.bd.com/ds/productCenter/244520.asp">LB agar</A> plates and <A href="https://2012.igem.org/Team:Purdue/Protocol#Antibiotics">LB liquid media with ampicillin</A></LI><LI> Created the adhesions and SBP devices in silico using Gene Designer by DNA 2.0 </LI></UL><H2> Thursday, May 31 </H2><H5> Attended Rickus and Porterfield laboratory group meeting </H5><UL><LI> Each person introduced themselves and their research </LI><LI> Decided on the use of future meetings </LI></UL><H5> Researched primer design and began to design primers. These primers will be used for PCR to perform the Gibson assembly method </H5><H2> Friday, June 1 </H2><H5> Performed Cell Transformations of the following parts: </H5><UL><LI> Lac promotor and CFP generator (BBa_I13601) </LI><LI> PoPs receiver (BBa_F2620)</LI><LI> Tet repressor (BBa_C0040)</LI><LI> Tet promotor with green fluorescent protein (BBa_I13522)</LI></UL><H5> The transformation protocol can be found <A href="http://openwetware.org/wiki/Transforming_chemically_competent_cells">here</A></H5><H5> Completed Primer Sequences for surface expression protein </H5><H2> Monday, June 4 </H2><H5> Welcomed Bio Builders workshop participants </H5><H5> Janie gave a powerpoint on the basics of synthetic biology to the workshop participants </H5><H5> Amanda, August, Max, Mrudula, and Rachel helped the participants with an experiment based on MIT's 2006 iGEM project, a banana scent generator </H5><H5>Mrudula and Amanda completed <A href="https://2012.igem.org/Team:Purdue/Protocol#Miniprep">minipreps</A> for the previously transformed parts. The results of the minipreps are shown below. </H5><UL><LI> Tet repressor: 79.8ng/µL </LI><LI> PoPs receiver: 149.6ng/µL </LI><LI> Lac promotor and CFP generator: 46.2ng/µL </LI><LI> Tet promotor wih GFP: 22.9ng/µL </LI></UL><H5> These concentrations are too low to use in any assembly method; therfore, we will need to attempt transformations and minipreps again before doing any assembly method. </H5><H2> Tuesday, June 5 </H2><H5> Amanda, August, Mrudula, Rachel, and Soo spent the morning watching presentations on abstraction and devices with the Bio Builder workshop participants
 Soo gave a presentation to the participants on how to use TinkerCell
 The iGEM team helped  the Bio Builder participants to transform green and purple fluorescent protein into E. coli
</H5><UL><H5> Created a timeline for the rest of the summer as seen below </H5><LI> Week 3: </LI><UL><LI>Design and order primers for Gibson assembly </LI><LI> Order reagents </LI><LI> Complete a miniprep for the previously transformed parts </LI><LI> Select plasmid backbones </LI></UL><LI> Week 4: </LI><UL><LI> PCR parts for Gibson assembly </LI><LI> Complete Gibson assembly </LI><LI> Decide on which parts will be synthetic </LI><LI> What devices can be synthesized or should we synthesize individual parts? </LI><LI> Create competent cells </LI></UL><LI>Week 5:</LI><UL><LI>Complete plasmid midi and mega preps </LI><LI> Transform parts into expression vectors </LI></UL><LI> Week 6: </LI><UL><LI> Complete functional analysis of protein expression </LI><LI> Construct the full system </LI></UL><LI> Week 7: </LI><UL><LI> Build the flow system for biofilm establishment </LI><LI> Complete functional analysis of biofilm thickness </LI></UL><LI> Week 8-10: </LI><UL><LI> Iterate and improve on flow setup </LI></UL></UL><H2> Wednesday, June 6 </H2><H5>
 Sean gave a presentation to the Bio Builder participants on 3A and Gibson assembly
 Sean lead a 3A assembly lab with the participants
 Worked on designing primers </H5><H2> Thursday, June 7 </H2><H5> Continued to design primers </H5><H2> Monday, June 11 </H2><H5> We attempted to optimize our primers using online guidlines and software. </H5><UL><LI><A href="http://bioweb.uwlax.edu/genweb/molecular/seq_anal/primer_design/primer_design.htm">Here</A> is a website that details primer design. </LI><LI>We used a <A href="http://www.neb.com/nebecomm/tech_reference/TmCalc/Default.asp">calculator</A> and an <A href="http://www.idtdna.com/analyzer/Applications/OligoAnalyzer/">Oligo Analyzer</A> which can calculate the liklihood of occurances such as hairpins and dimers. </LI></UL><H5>
 Upon analysis, we found that the primers that would be required to use Gibson assembly were not ideal. In some cases, the forward and the reverse primers did not have Tm values that were close enough to each other. We also had difficulty with making our primers a reasonable length. Furthermore, many of the primers had a high probability (Gibson free energy value lower than zero) of froming dimers or hairpins. Finally, we deicided that it would most likely be more successful to use 3A assembly instead of Gibson.
 In order to save time and ensure the accuracy of the sequence, we decided to synthesize instead of assemble the whole silica making device in a high copy plasmid.
 Welcomed Rubeena to the iGEM team. We are looking forward to working with her! </H5><H2> Wednesday, June 13 </H2><H5> August, Mrudula, and Rachel were trained on how to use the flow cytometer
 Made competent cells (DH5a). The protocol for making competent cells is found <A href="http://partsregistry.org/Help:Protocols/Competent_Cells">here</A></H5><H2> Thursday, June 14 </H2><H5>Rachel had a demonstration on how to make silica. The procedure is shown <A href="https://2012.igem.org/Team:Purdue/Protocol#Silica">here</A>.
We welcomed Chris to the iGEM team. We know he will be a great asset to the team!
We had an overview meeting for Rubeena and Chris to help them catch up on the project. </H5><H2> Friday, June 15 </H2><H5> We transformed all of the parts needed for the adhesion device. These included the following </H5><UL><LI> AIDA-1 (BBa_K257018) </LI><LI> Plasmid (pSB1AK3) </LI><LI> pTet (BBa_R0040) </LI><LI> RBS (BBa_B0034) </LI><LI> Terminator (BBa_B0015) </LI><LI> Plasmid (pSB1AC3) </LI><LI> CFP (BBa_E0020) </LI></UL><H5><A href="https://static.igem.org/mediawiki/2012/0/0d/June_15.ppt">Here</A> is a powerpoint summarizing our team meeting. </H5><H2> Saturday, June 16 </H2><H5> Removed the following parts from the incubator and placed in the refrigerator with parafilm wax </H5><UL><LI> BBa_E00032 (GFP device from plates) </LI><LI> Bba_ </LI></UL><H2> Sunday, June 17 </H2><H5> We reevaluated our previous decision to synthesize the silica making device. We have now decided to use Gibson assembly to create the OmpA-silicatein fusion sequence and 3A assembly for the rest of the device. We believe that this will save money and, considering the time taken to order and deliver the sequence, will not take any more time than synthesizing. </H5><H2> Monday, June 18 </H2><H5> Completed <A href="https://2012.igem.org/Team:Purdue/Protocol#Miniprep">minipreps</A> on recently transformed parts. The results are shown below. </H5><UL><LI> GFP: 17.5ng/µL and 7.6 ng/µL </LI><LI> AIDA-1: 8.7ng/µL and 6ng/µL </LI><LI> pSB1AC3: 5.2ng/µL snd 4.2ng/µL </LI><LI> CFP: 13.6ng/µL and 12.7ng/µL </LI><LI> RBS (BBa_B0032): 24.9ng/µL and 24.2ng/µL </LI><LI> PoPs: 107.8ng/µL and 131.4ng/µL </LI><LI> pSB1AK3: 7.0ng/µL </LI><LI> Terminator: 11.5ng/µL </LI><LI> RBS (BBa_B0034): 21.0ng/µL and 21.5ng/µL </LI><LI> pTet: 5.2ng/µL </LI></UL><H5> These concentrations are too low to proceed with 3A assembly. Therefore, we are going to grow more of the transformed bacteria, and we are going to perform more minipreps. </H5><H2> Tuesday, June 19 </H2><H5> August completed a second round of <A href="https://2012.igem.org/Team:Purdue/Protocol#Miniprep">minipreps</A>. The results are listed below. </H5><UL><LI> RBS (BBa_B0034): 14.8ng/µL </LI><LI> RBS (BBa_C0040): 5.0ng/µL </LI><LI> CFP: 9.0ng/µL </LI><LI> PoPs: 14.9ng/µL </LI><LI> AIDA-1: 7.0ng/µL </LI></UL><H5><A href="https://static.igem.org/mediawiki/2012/6/6d/June_19.ppt">Here</A> is a powerpoint summarizing our team meeting. </H5><H2> Wednesday, June 20 </H2><H5> Due to the low yields from our previous tranformations and minipreps, we have decided to attempt our transformations with a new protocol. The new protocol can be found <A href="https://2012.igem.org/Team:Purdue/Protocol#UpdatedTransformation">here</A></H5><H5> Today, we transformed pLac, TetR, terminator (BBa_B0015), RBS (BBa_B0034), pSB1A3, and pSB1C3. </H5><H5> The minipreps gave the following results </H5><OL><LI> RBS: 14.8ng/µL </LI><LI> TetR: 5.0ng/µL </LI><LI> CFP: 9.0ng/µL </LI><LI> PoPs: 14.5ng/µL </LI><LI> AIDA-1: 7.0ng/µL </LI></OL><H2> Thursday, June 21 </H2><H5> Due to the low yield of our transformations and minipreps, we have now decided to troubleshoot our transformation procedures. To do this, we will be doing a transformation with Tarun, a graduate mentor to the team. We will be using Tarun's procedures and supplies. We will also have a control plate that will contain cells that had no DNA inserted. If these transformations succeed, then we will have narrowed the possibilities of what went wrong with our procedures. Tarun's transformation procedure can be found on the <A href="https://2012.igem.org/Team:Purdue/Protocol#Tarun">here</A>. We performed these test transformations with BBa_I13602 (CFP). </H5>

Upon closer inspection of the AIDA-1 biobrick (BBa_K257018), we have found the following problems
<UL><LI> the sequencing is bad </LI><UL><LI> there are large gaps from the sequencing as compared to what was documented. Specifically, the sequenced biobrick was missing approximately 500bp </LI><LI> the sequence was supposed to be 1326bp long. We have decided that it would cost too much to have the biobrick sequenced ourselves </LI></UL><LI> the gel is bad </LI></UL>
For the above reasons, we have decided to not use AIDA-1. After reevaluating the other adhesion options, we have decided to use Curli (BBa_342003). Due to its inconsistant sequencing, we have decided to have the part found in the kit sequenced ourselves. 

<H2> Friday, June 22 </H2><H5> Due to our increasing size, we have decided to divide into committees. These committees will help us divide the workload fairly and ensure that all tasks are being completed in a timely manner. The committees and their members are listed below. An asteric indicates the committee head. </H5><UL><LI><B> Wetlab </B></LI><UL><LI> OmpA-silicatein </LI><UL>
*AugustHaefa
</UL><LI> Curli </LI><UL> 
*Rubeena
</UL></UL><LI><B> Purchasing </B></LI><UL>
*Peter
</UL><LI><B> Human Practices </B></LI><UL>
*Peter*Max
</UL><LI><B> Experimental Design/Characterization </B></LI><UL>
*Amanda*MrudulaChrisArthiRachelJim
</UL><LI><B> Project Planning </B></LI><UL>
*Namita
</UL><LI><B> Wiki </B></LI><UL> 
*RachelSean (background)
</UL></UL><H5> We checked our test transformations from yesterday. None of the plates showed growth.</H5><H5><A href="https://static.igem.org/mediawiki/2012/1/1d/June_22.ppt">Here</A> is a powerpoint summarizing our team meeting. </H5><H2> Monday, June 25 </H2><H5> The foci of this week are to troubleshoot our transformations and research ways to characterize our constructs once they are assembled. August and Mrudula will be in lab troubleshooting. Amanda and Mrudula will be resarching ways to characterize a biofilm. Rachel will be researching ways to characterize a silica coat. </H5><H2> Tuesday, June 26 </H2><H5><A href="https://static.igem.org/mediawiki/2012/8/85/June_26.ppt">Here</A> is a powerpoint summarizing our team meeting. </H5><H2> Friday, June 29 </H2><H5> We went to Dow Agro, one of our sponsors, to give them an overview of our project. The powerpoint presentation we gave can be found  <A href="https://2012.igem.org/wiki/index.php?title=Special:Upload&amp;wpDestFile=DowPresentation.pptx">Powerpoint</A>. After we gave our presentation, we asked the Dow representative, Ryan, if he had any guidance with concern to Gibson assembly. </H5><H2> Saturday, June 30 </H2><H5> Part of the team volunteered at the high school Jamboree. We saw many amazing teams with remarkable ideas! Congratulations to all the teams for their hard work over the past semester, and good luck as future iGEMers! </H5><H2> Sunday, July 1 </H2><H5> We attempted another test <A href="https://2012.igem.org/Team:Purdue/Protocol#Tarun">transformation</A>, this time using bought Excel 10ß competent cells. We had both a negative and positive control. We also plated different concentrations of the final solution to see if we were over or under plating. We also used agar plates with ampicillin that Tarun made. We used store bought SOC. The part used was BBa_J04450. </H5><H2> Monday, July 2 </H2><H5> The tranformations performed yesterday were successful. Both the positive control and the plaes with our part showed growth. The transformation efficiency was extremely high. We got single colonies on the plates with 50µL and overgrowth on the plates that had 150µL. We will only plate at 50µL from now on. </H5><H5> Due to time constraints, we have decided to synthesize both of our constructs. In parallel to characterization of the synthesized constructs, we are going to assemble both constructs using Gibson assembly. The assembles constructs will differ from the synthesized constructs in that the promoter will be a PoPs device and there will be a fluorescent protein on each. In this way, we can find if the addition of these two characterization parts will affect the functionality of our constructs. </H5><H5>The synthesized constructs were ordered from <A href="https://www.dna20.com/">DNA 2.0</A>. Pictures of the Gene Designer (DNA 2.0 software) renditions of the three constructs are given below.</H5><UL><LI><A href="https://2012.igem.org/File:OmpA-Silicatein.jpeg">Silicatein construct</A></LI><LI><A href="https://2012.igem.org/File:Curli_without_GFP.jpg">Curli</A></LI></UL><H5> We transformed Curli so that we can have it sequenced soon </H5>

List of What Needs to get Done this Week:
<UL><LI> Research protocols on setting up and running experiments using biofilms </LI><LI> Research flow and static protocols for biofilms </LI><LI> Research ways to characterize silica matrix and acquire quantifiable data </LI><LI> Create timelines for each assay researched </LI><LI> Order reagents and get training on any equipment needed for assays </LI></UL><H2> Tuesday, July 3 </H2><H5> We are now awaiting the delivery of our synthesized constructs. In preparation, we are researching the varying types of assays that we may be able to use. You can find information about the different assays we researched can be found <A href="https://2012.igem.org/Team:Purdue/Characterization">here</A>. </H5><H5> Silicatein-Alpha Characterization </H5>
We have decided to use scanning electron microscopy (SEM), x-ray diffraction (XRD), and a live dead assay (specifically, the <A href="http://products.invitrogen.com/ivgn/product/L7012?ICID=search-product">Baclight Bacterial Viability Kit</A> from Invitrogen). 

Further assays will be done once the two constructs are used within the same bacteria.

Researched stages of biofilm formation:

<P>  1. Initial Attachment  -  attachment occurs via van der waals forces</P><P>  2. Irreversible Attachment  -  this is where the EPS forms</P><P>  3. Maturation I  -  matrix formed with adhesion proteins</P><P>  4. Maturation II  -  the full biofilm forms</P><P>  5. Dispersion  -  bacteria released</P>

Also researched shear stress modeling, chassis development, microsensors, and confocal microscopy assays.


Furthermore, research was done on different ways to characterize the silica coat. Three extremely helpful articles are listed below.
<UL><LI><A href="http://onlinelibrary.wiley.com/doi/10.1002/bit.23195/full">Cell-Mediated Deposition of Porous Silica on Bacterial Biofilms</A></LI><LI><A href="http://www.springerlink.com/content/9352q2105531m466/fulltext.pdf">Sol-gel synthsis and structure of silica hybrid materials</A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S0165027009001071">Thin-film silica sol-gel coatings for neural microelectrodes</A></LI></UL>
A powerpoint highlighting possible characterization methods is given <A href="https://static.igem.org/mediawiki/2012/8/8e/Experiments_for_Silica_Characterization.ppt">here</A>
After discussion with the team and our faculty advisor, Dr. Rickus, we have decided to use the scanning eletron microscopy (SEM), x-ray diffraction (XRD), and a live/dead assay (specifically, the <A href="http://products.invitrogen.com/ivgn/product/L7012?ICID=search-product">Baclight Bacterial Viability Kit</A>).


<H2> Thursday, July 5 </H2>
Researched possible competitors in the field of bioreactor filtration.

<UL><LI> Hydro Engineering - produces &quot;Hydrokleen&quot;, which is a bioreactor filtration system that would generally be used in industrial plants and acts as a water recycler. </LI><LI> Seven Trent Services - produces Tetra Amphidrone fixed sequencing Batch Biological Filter, a waste water treatment process that uses a biological filter to remove various pollutants </LI><LI> Kinetics -  this company makes a range of water treatment processes and machines that can be used in industry as well as consumer markets </LI><LI> ABEC - produces bioreactors, but could not find more detail into what kind </LI></UL><H2> Friday, July 6 </H2><H5><A href="https://static.igem.org/mediawiki/2012/7/72/July_6.ppt">Here</A> is a powerpoint summarizing our team meeting. </H5><H2> Sunday, July 8 </H2><H5> Research was done on growing a static biofilm. 
<A href="http://media.wiley.com/CurrentProtocols/0471729256/0471729256-sampleUnit.pdf">&quot;Growing and Analyzing Static Biofilms&quot;</A> was a very helpful article. </H5>Created a cell culture to have competent cells for a biofilm culture.Also did research into developing a static biofilm assay to test how well our bacteria makes biofilm before and after our transformation. <A href="http://media.wiley.com/CurrentProtocols/0471729256/0471729256-sampleUnit.pdf">&quot;Growing and Analyzing Static Biofilms&quot;</A> by Judisth H. Mernitt, Daniel E. Kadouri, and George A. O'Toole was helpful in determining how we would do this assay.<H2> Monday, July 9 </H2>Performed the <A href="https://2012.igem.org/Team:Purdue/Protocol#Stationary Phase"> Growth Rate Assay</A> on NEB Beta and DH5 Alpha E.coli strains to determine the time frame we will use to grow our static biofilms.A typical bacterial growth curve is shown below. This is what we expect our curves to look like. 

Below is the results of the growth rate assay for NEB Beta and DH5 alpha. Overall, the plots show the expected shape with a lag, log, and stationary phase. 
Also worked out the lab plan for the rest of the week.<H2> Tuesday, July 10 </H2>

We did the same <A href="https://2012.igem.org/Team:Purdue/Protocol#Stationary Phase"> Growth Rate Assay</A> as yesterday, but with the XL-10 Gold E.coli strain because we think it will work better for the biofilm purposes.
Below is the results of the growth rate assay for XL-10 Gold. The plots show the expected shape with a lag, log, and stationary phase. 
<A href="https://static.igem.org/mediawiki/2012/4/41/Bacteria_Growth_Rate.xls">Here </A> is the excel file containing all of the data from the three growth rate assays. 


Also researched and developed the static biofilm assay that we can do to determine the biofilm rates of our non-transformed E.coli.<H2> Wednesday, July 11 </H2><H5> Chris T created primers for a test Gibson reaction. The test construct will be a the plasmid pGA3K3, PoPs device (BBa_F2620), an RFP with RBS(BBa_K093005), and a double terminator (BBa_B0014). The primers can be viewed in this <A href="https://static.igem.org/mediawiki/2012/d/d7/PoPS_Characterization_Circuit_Primers_Color.pdf">document </A>. We ordered the primers from <A href="http://www.idtdna.com/site">Integrated DNA Technologies</A>. </H5><H2> Friday, July 13 </H2>Performed the <A href="https://2012.igem.org/Team:Purdue/Protocol#Static Biofilm"> Static Biofilm Assay</A>.Had a team meeting, discussed goals for the following week:<UL><LI> Write out protocols and email them to the entire team for revisions </LI><LI> Plan a meeting to go through and troubleshoot the protocols together </LI><LI> Find a way to determine the optimized concentration of curli</LI><LI> Purchase all materials needed for future protocols </LI></UL><H2> Monday, July 16 </H2>Researched how Membrane Aerated Bioreactor work and how we will use one for testing.
There are multiple types of Membrane Aerated Biofilms: MABR (the type we will be using in our experiments), Membrane biofilter, and extractive membrane bioreactor (EMB).  These reactors can be in shell and tube or plate and frame configurations.  We will be using the shell and tube configuration. The two limiting substrates are introduced at opposite ends of the bioreactor.  The limiting substrates are C-substrate (carbon substrate) and/or oxygen. The biofilm thickness must be less than the oxygen penetration (50-200 micrometers) but stay less than 150 micrometers for best performance. 
<H2> Thursday, July 19 </H2><H5> We attempted to do <A href="https://2012.igem.org/Team:Purdue/Protocol#PCR">PCR</A> for the parts for which Chris made <A href="https://static.igem.org/mediawiki/2012/d/d7/PoPS_Characterization_Circuit_Primers_Color.pdf">primers</A>. To prove that the PCR was successful, we ran aliquotes of the reaction products through a gel. An image of the gel can be seen <A href="https://static.igem.org/mediawiki/2012/a/ac/Rachel_PCR.jpg">here</A>. No bands can be seen for the aliquotes; therfore, we concluded that the PCR did not work. We believe that there was a problem with the cycler; therefore, we will be trying the procedure again with a different machine. 

</H5><H2> Monday, July 23 </H2><H5> We attempted <A href="https://2012.igem.org/Team:Purdue/Protocol#PCR">PCR</A> again with a different machine. Unfortunately, we used too high of a concentration of primers. Therefore, when we ran a gel, two bands appeared for each sample. The excess primer would greatly lower the efficiency of a Gibson reaction. Furthermore, the bands on the gel all had similar size even though the parts are not of similar size. Therefore, we believe that nonspecific amplification took place. An image of the gel can be seen <A href="https://static.igem.org/mediawiki/2012/5/5e/Max_and_Mrudula_PCR.jpg">here</A>. </H5>

There were some modifications to the <A href="https://2012.igem.org/Team:Purdue/Protocol#Static Biofilm">Static Biofilm Assay</A> Protocol.  Once the plates had grown for a long enough time period (every hour for 1-8 hours and then 24 hours and 48 hours) they were transferred to the 4 degree freezer.  The plates were then removed, the excess liquid was dumped out, and the plates were then sprayed off with water and dumped out again.  The plates were than stained with crystal violet with the following steps:
<UL><LI> add 125 microliters of crystal violet to each well plate </LI><LI> let sit for 15 minutes </LI><LI> wash off (Note: at this point the plates are stable for several weeks) </LI></UL>
Analyzing Plates:
<UL><LI> 500 microliters of 80% ethanol | 20% acetone were added to each well </LI><LI> 300 microliters (or enough until the well were full) were taken from the 24 well plate to the 96 well plates </LI><LI> this was then analyzed using the spectrophotometer to determine the OD of each sample </LI></UL><H2> Tuesday, July 24 </H2><H5> We attempted <A href="https://2012.igem.org/Team:Purdue/Protocol#PCR">PCR</A> for a third time. This time, the gel shows single bands of varying size. An image of the gel can be seen <A href="https://static.igem.org/mediawiki/2012/9/93/PCR_3.jpg">here</A>. We are now going to proceed to the Gibson reaction. </H5><A href="https://2012.igem.org/wiki/index.php?title=Special:Upload&amp;wpDestFile=">Upload</A><H2> Thursday, July 26 </H2>

Purpose: To make a solution to grow XL-10 gold strain of <I>E.coli</I> so that it can be integrated into a membrane aerated biofilm reactor (MABR)

Materials:
<UL><LI> SOB - 250 mL </LI><LI> 10 mM MgCl2 (.238 g) </LI><LI> 20 mM glucose (.900 g) </LI><LI> 250 microliters of cultured XL-10 gold culture </LI></UL>
Procedure: Add all the ingredients together in a flask and swirl.  Place in 37 degree room and shake at 220 rpm. Check OD.


Feeding Solution for current biofilm:
<UL><LI> 200 mL nutrient (glucose and nitrogen)</LI><LI> 1mL solution 2 and 3 </LI><LI> 100 microliters solution 1 </LI></UL><H2> Friday, July 27 </H2>

Purpose: Ruoxi Wu i helping set up the membrane aerated biofilm reactor (MABR)


Now that we have the culture growing, we have to determine what media will be the best that can be used to sustain the cells once they are in the reactor. 
The nutrient solutions that Wen, Jane, and Ruoxi are using are specific to <I>Pseudomanas aeruginosa</I>. This research is to determine the protocol for the bioreactor specifically the media.

Research for media:
Design and performance study of a novel immobolized hollow fiber membrane bioreactor by Ping Wang (1st author)
<UL><LI> nutrients are supplied to the forming biofilm through diffusion (very poor efficiency) </LI><LI> only the cells in a thin layer close to the surface of oxygen and media are active </LI><LI> The bacterial strain used in this paper is <I>E.coli</I> DM5d </LI><LI> The growth limiting substate is M9 medium </LI><LI> Glucose concentration in medium is equal to 5 g/L </LI><LI> Protocol from Paper to culture bacterial cells: </LI><UL><LI> Agar plates with single columns Fl streaking DH5d </LI><LI> new agar plates made every 14 days </LI><LI> pick single colony into 100 mL media </LI> culture overnight at 37 degrees Celsius and 120 rpm <LI> check OD 600 </LI></UL></UL>
Researched preparation of <A href="https://2012.igem.org/Team:Purdue/Protocol#M9">M9 minimal media recipe</A>.

<H2> Monday, August 6 </H2>
Purpose: PBS is used to clean out the waste from the bioreactor 

Materials: 
<UL><LI> NaCl </LI><LI> KCl </LI><LI> Na2HPO4 </LI><LI> KH2PO4 </LI><LI> Distilled H20 </LI><LI> HCl </LI></UL>
Procedure:
<UL><LI> Combine 80 g NaCl, 2 g KCl, 14.4 g Na2HPO4 (dibasic anhydrous), 18.1 g Na2HPO4 * 2H2O (dibasic dihydrate), 27.2 g Na2HPO4 * 7H20 (dibasic heptahydrate), 2.4 g KH2PO4 (monobasic anhydrous) and 800 mL H20 </LI><LI> Adjust pH to 2.4 with HCl </LI><LI> Add H20 to 1 L </LI><LI> Autoclave </LI><H2> Friday, August 17 </H2>
Purpose: To measure the biofilm's detachment and viability after growing for 15 days 

Materials: 
<UL><LI> ISOTON </LI><LI> PBS </LI><LI> Small Biocontainer </LI><LI> Tweezers and scissors </LI><LI> Beaker </LI></UL>

Procedure:
<UL><LI> Turn off the pump and take out the feed loop fl the beaker </LI><LI> Lay the bioreactor on its side and let enough of the media solution drain out so that one biofilm is revealed (collect media in a beaker) </LI><LI> Remove the biofilm using tweezers </LI><LI> Cut into small 1.5&quot; - 2&quot; pieces </LI><LI> Place in containers and pour some of the media into the dishes to keep the biofilms submerged </LI></UL>
This biofilm is now ready to be tested 

Purpose: The purpose of this experiement is to determine how much of the biofilm comes off (detaches) from the tube when placed under different water flows 

Materials:
<UL><LI> Biofilm holders </LI><LI> ISOTON </LI></UL>

Procedure:
<UL><LI> Once the biofilm has been extracted from the bioreactor and it is in small pieces, this procedure can be started </LI><LI> Places the small pieces in the holder that Wen zang constructed </LI><LI> Set up a pump to pump water into the tube and back out into a waste container </LI><LI> Make sure the biofilm is completely submerged the entire time </LI><LI> Collect the PBS that flows over the biofilm in a beaker </LI><LI> Measure the amount of run off and the time to determine the flow rate </LI><LI> Save the sample </LI><LI> To determine the amount of cells in the sample a Multisizer 4 Coulter Counter was used - this tells the concentration of particles of different sizes found in the sample </LI><LI> Different flow rates were used to analyze the biofilm </LI><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Purdue/Notebook">http://2012.igem.org/Team:Purdue/Notebook</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Purdue/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Purdue/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Purdue/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Purdue/Notebook&amp;oldid=280270" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></UL></DIV></DIV></DIV></BODY></HTML>