"
Page
Discussion
View source
History
teams
Log in
 
Team:Purdue/Modeling
From 2012.igem.org
Home
Team
Members
Advisors
Sponsors
Attributions
Project
Overview
Design
Protocol
Results
Parts
Modeling
Human Practices
Biomaker Bench
Girl Scouts Biotechnology Badge
Notebook
Safety
Modeling
Levels of Investigation
How biofilm responds to shear, flow, temp, surface attachment, etc
How Silica traps the particle during flow
How Curli adheres and how OmpA-Silicatein Alpha polymerizes Silica/How the silica crystalizes (introduction of Salicylic Acid)
How protien expression responds to external and metabolic variation (e.g. introduction of IPTG to system)
How constructs work with each other/controls systems
How RBS/ Promoter effects protein expression
Distribution of Modeling
Matlab
Protein Production
Feed Forward Loop Control Structure
TinkerCell
Quorum sensing
Feed Forward Loop
JMP
Experimental Design and Characterization Experiments
Comsol/Other
Waterflow and shear force on final system
Silica formation
Design
Desired Outcomes of Models
Levels of Abstractions
Biofilter response to Environmental conditions
Shear
Flow
Temperature
Abiotic Surface (Adhesion)
Formation of Silica Matrix
BioFilm Development
Model of Bacterial Growth, Death, Breaking Off
Expression of Proteins
Response to addition of IPTG
Optimal Production rate/expression of Curli and OmpA-Silicatein Alpha protiens
Control Systems
Fine Tuning of Protein Expression with RBS/Promoter combination variants
Platforms
Considerations and Assumption
Parameters
Equations
The growth of the Biofilm can be modeled with modified continuous growth models. Traditional models of population growth make use of basic differential equations to measure the change in population over unit time. At the most fundamental level
Population Change Rate =
dN
= bN-dN
dt
Where N is the population of individuals, b is the rate of births or immigration, and d is the rate of deaths or emigration.
To understand these dynamics of population in the context of biofilm growth, we treat the bacterial attachment and detachment as a reversible reaction of the form
N Bacteria Attached <
k1
>  N Bacteria Detached
k2
Where k1 is the rate constant of detachment and k2 is the rate constant of attachment. Because we have chosen to simplfily the cell attachment and detachment as a reaction, we are able to employ reaction kinects to predict the rate of binding that will dictate the constants to population growth.
Such assumptions allow us to make use of basic principles of reaction kinetics such as the Law of Mass Action , which dictates that a rate of reaction is always proportional to the concentration of its reactions. Therefore, using the attached and detached cells as the reactants in the equation the rate of attachment becomes
Rate of Growth of Attached Bacteria  =
dNatt
= k2Ndet + βNdet - k1Natt - αNatt
dt
introducing the constants α, which is degradation and dilution, and β, which is reproduction rate.
To predict the our system and allow easier fine tuning of our feed-back loop, a model of our genetic circuits was constructed in MathCad through three iterations and using the system of a single cell within in the biofilm:
Interation One
In the first iteration, certain simplifying assumptions were made. We assume that the LacI protein has reached a steady state level in the cell. Upon the introduction of lactose into the media, LacI is inactivated immediately, resulting in activation of repressed genes. All interactions are modeled with Michaelis-Menten binding kinetics, running the expression of each protein as a differential equation and plots it against the time the circuit runs, with addition of Lactose at the 10 unit mark. It is evident at this point that promoter regulating expression of the ompR234 protein (which is directly proportional to the variable 'XCurli' in the model) is switched off, leading to gradual degradation of the protein in the system.  Each term, X_Curli, X_Sila, and X_tetR, are designed to have a primary expression term (the first term in each differential equation) followed by a term which accounts for the degradation and dilution of the protein within the cell.  The parameters entered are system-independent estimates, stable over a range of behaviors. Sensitivity analysis of the parameters in the system shows that time-evolution is robust to the initial parameter estimates.
Interation Two
The second iteration addresses interactions of the OmpR234 protein with the activators EnvZ, and EnvZ's interaction with upregulation of the Curli expression gene. The visualization of this model reveals the sensitivity of the feedback loop.
Interation Three
The final iteration of the model creates a mathematical model that more correctly assumes that OmpR234 interacts at the protein level rather than the transcriptional level. The model provides evidence that OmpR234 provides too much a time delay for optimal deactiviation of the Curli adhesion and protein, providing an avenue for future project improvement.
Retrieved from " http://2012.igem.org/Team:Purdue/Modeling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
