<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Exeter_Regulatory_Sequences"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Exeter/Regulatory_Sequences">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Exeter/Regulatory_Sequences&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Exeter/Regulatory_Sequences&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Exeter/Regulatory_Sequences&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Exeter/Regulatory_Sequences" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Exeter/Regulatory Sequences</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Exeter | ERASE</TITLE></P><DIV id="wrapper"><DIV id="header"><UL id="header_list"><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter">Home</A></DIV></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/TheProblem">The Problem</A></DIV></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/Project">Our Project</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/Modelling">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/DegradationConstructs">The Enzymes</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/EnzymeValidation">Enzyme Validation</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/invivoactivity"><I>in vivo</I>: Raman </A></LI><LI><A href="https://2014.igem.org/Team:Exeter/invivo"><I>in vivo</I>: Observations </A></LI><LI><A href="https://2014.igem.org/Team:Exeter/enzyme-kinetics"> HPLC </A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Kill_Switches">Kill Switches</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/EColiStressTesting"> Xenobiotic Tolerance</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Detection">Detection of Xenobiotics</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/iLOVCharacterisation">iLOV Characterisation</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Parts">Our Parts</A></LI></UL></DIV><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/MeetTheTeam">The Team</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/MeetTheTeam">The Undergraduate Team</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Supervisors">Supervisors</A></LI><LI><A href="https://igem.org/Team.cgi?id=1398">Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Sponsors">Sponsors</A></LI></DIV><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/MedalRequirements">Medal Requirements</A></DIV></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/Notebook">Notebook</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/LabBook">Lab Book</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Protocols">Protocols</A></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/Outreach">Outreach</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/BeyondTheBench">Beyond The Bench</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Stakeholders">Stakeholders</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Presentations">Presentations</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Game">  ERASE-The Game!</A></LI></DIV></DIV><DIV id="description"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#TheNemRrecognitionpromoter(BBa_K1398008)"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">The NemR recognition promoter (BBa_K1398008)</SPAN></A></LI><LI class="toclevel-1"><A href="#TheNemRpromoterconstruct(BBa_K1398007)"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The NemR promoter construct (BBa_K1398007)</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="TheNemRrecognitionpromoter(BBa_K1398008)">The NemR recognition promoter (<A href="http://parts.igem.org/Part:BBa_K1398008">BBa_K1398008</A>)</SPAN></H2><P> One of the problems faced by the 2009 Edinburgh iGEM team was the issue of how TNT was to be detected in order to generate a gene expression response.</P><A href="https://2009.igem.org/Team:Edinburgh/biology%28results%29">https://2009.igem.org/Team:Edinburgh/biology%28results%29</A><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28biobricks%29">https://2009.igem.org/Team:Edinburgh/biology%28biobricks%29</A><P>While their design and parts demonstrated a staggering amount of work and understanding, we felt that their proposed TNT-sensing construct was too complex and could be simplified. As a system of parts we felt that it relied too heavily on several freshly designed and uncharacterised parts working together in unison to be easily engineered for success. </P><P align="center"><I><STRONG>Figure 7: </STRONG>TNT sensing pathway developed by the Edinburgh 2009 iGEM team. 
<A href="https://2009.igem.org/Team:Edinburgh/biology%28tntsensing%29">https://2009.igem.org/Team:Edinburgh/biology%28tntsensing%29</A>
TNT binds to TNT.R1 in the periplasm. The TNT-TNT.R1 complex induces a conformational change in the Trg-EnvZ (Trz) fusion protein. Trg-EnvZ autophosphorylates and subsequently phosphorylates ompR. Phosphorylated ompR activates transcription. </I></P><P>As a result, we decided to see if we could discover and develop a simpler system that would be easier to engineer and conceptually extrapolate to other chemicals. After the discovery that the NemA enzyme could already be expressed at a very low level in <I>E. coli</I>, it was hypothesised that <I>E. coli</I> should have a gene regulatory molecule which controlled the nemA operon enabling its expression only in environments where TNT was present. In <I>E. coli</I>, nemA is located downstream of a gene encoding a transcription factor known as NemR, previously entitled YdhM29.  Disruption to the gene encoding NemR results in a decrease in nemA expression, both in the basal and induced states, which indicated that the nemA gene and NemR form a single, connected operon. Umezawa et al. (2008) proposed that the function of NemR is as a redox-operated transcriptional repressor of the nemA gene. 
NemR is proposed to remain bound to the operon and, when it binds TNT in the environment, undergo a conformational change to expose the promoter. The region of the promoter sequence to which NemR binds (the ‘NemR recognition box) has also been identified<SUP>30</SUP> (Figure 8). </P><P align="center"><I><STRONG>Figure 8: </STRONG> Diagram depicting the sequence of the ‘NemR box’. Adapted from Umezawa et al.<SUP>31</SUP></I></P><P>On the assumption that TNT was able to naturally diffuse into the <I>E. coli</I> cell, we hypothesised that if we designed a promoter containing this binding site gene sequence, the promoter would therefore become inducible by the addition of TNT through the use of the NemR protein which is naturally found in <I>E. coli</I> and would be a vastly simpler TNT-detecting gene expression system as it would rely on the success of considerably fewer engineered parts.
It has been shown that NemR also responds to cysteine-modifying electrophiles, including NEM, showdomycin, and, more weakly, iodoacetamide <SUP>33</SUP>. The repressor function of NemR was inactivated through the addition of Cysteine modifying reagents. Furthermore, based upon a predicted 3D structure (Figure 9), it has been proposed that NemR utilises reversible oxidation of a conserved Cys-106  residue as the signal for confirmation change and dissociation from the DNA strand thus activating gene expression<SUP>34</SUP>.</P><P align="center"><I><STRONG>Figure 9: </STRONG>3D  molecular structure of the E. coli NemR monomer. Cysteine residues are in orange, conserved Cys-106 in red, and DNA binding helices in green<SUP>33</SUP>.</I></P><P>Finally, it has been indicated that the NemR protein was responsive to Hypochlorous acid (HOCl), the active component of household bleach. Addition of bleach resulted in the expression of two detoxifying enzymes for bleach: glyoxalase I and NemA<SUP>35</SUP>.</P><P>Thus, it is likely that our  vastly simplified promoter, while designed and refined by us to respond to TNT, would actually have a more ubiquitous role in responding to a range of important electrophiles and thus is ripe for characterisation not only by us but also for future iGEM teams.</P><H2><SPAN class="mw-headline" id="TheNemRpromoterconstruct(BBa_K1398007)">The NemR promoter construct (<A href="http://parts.igem.org/Part:BBa_K1398007">BBa_K1398007</A>)</SPAN></H2><P>The development of the NemR promoter was originally as part of the design of a kill switch we intended to implement. However, it was clear that impressive kill switches have already been widely developed and used. We decided instead to focus on a NemR switch that could be used to regulate them such that the bacterium would die in the absence of TNT. When we designed the NemR promoter, we debated where the ‘NemR box’ would be inserted into the construct sequence to enable successful gene repression and expression. We eventually opted for the NemR box to be placed in-between the -12 and -33 region of the promoter as this region appeared reasonably conserved across the Anderson promoter group available to us on the registry:  <A href="http://parts.igem.org/Promoters/Catalog/Anderson"> http://parts.igem.org/Promoters/Catalog/Anderson</A>.
Thus we designed the NemR construct, <A href="http://parts.igem.org/Part:BBa_K1398007">BBa_K1398007</A> as follows:</P><P align="center"><I><STRONG>Figure 10: </STRONG> The BBa NemR construct we designed </I></P><P>The construct begins with the synthetic promoter NemR, which combines a high-expression promoter with the NemR recognition box (<A href="http://parts.igem.org/Part:BBa_K1398008">BBa_K1398008</A>). It is followed by a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>), the fluorescent reporter iLOV (<A href="http://parts.igem.org/Part:BBa_K660004">BBa_K660004</A>), a double STOP codon and a double terminator made up of <A href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A> and <A href="http://parts.igem.org/Part:BBa_B0012">BBa_B0012</A>.</P><P>The addition of the iLOV reporter gene was also to help further characterise part <A href="http://parts.igem.org/Part:BBa_K660004">BBa_K660004</A>  submitted into the iGEM database in 2011 by the team from Glasgow: <A href="https://2011.igem.org/Team:Glasgow/LOV2"> https://2011.igem.org/Team:Glasgow/LOV2</A>. Given that this promoter would respond to TNT, this means we are also generating a biosensor for TNT.</P><P>While hybrid promoters have been designed before upon similar ideas, this is usually done to develop a switch with additional functionality to respond to the insertion of an additional engineered system (eg. For example, it is common that we utilise dual Lac and IPTG inducible promoters in the design of gene circuits).  Our promoter construct’s incredibly simple design compared to previous teams proposes the idea that by identifying the binding site of a repressor protein, anyone could theoretically make a promoter of any level of expression strength that is responsive to any chemical stimulus found in nature, and likely many that are not. </P><P>We aim to demonstrate this future possibility through the creation of our TNT-responsive NemR-sensitive promoter. Any strong output could therefore be inhibited or, alternatively, a weakly expressed gene could have its rate of expression enhanced by identifying the correct gene sequence. Alternatively, any construct could theoretically could be made to respond to a very specific chemical signal, such as TNT. If we are able to prove that our TNT-specific promoter is successful, it will provide the proof of concept to apply this theory to millions of other chemical signals which gene circuits could be designed to be receptive to. </P>


----
<H2><SPAN class="mw-headline" id="References">References:</SPAN></H2></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/9013822/">http://www.ncbi.nlm.nih.gov/pubmed/9013822/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/figure/f2/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/figure/f2/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/</A></LI><LI><A href="http://www.jbc.org/content/288/19/13789/F3.expansion.html">http://www.jbc.org/content/288/19/13789/F3.expansion.html</A></LI><LI><A href="http://www.jbc.org/content/288/19/13789.full">http://www.jbc.org/content/288/19/13789.full</A></LI><LI><A href="http://www.jbc.org/content/288/19/13789.full">http://www.jbc.org/content/288/19/13789.full</A></LI><H2> Navigation </H2><P><A href="https://2014.igem.org/Team:Exeter/DegradationConstructs">Previous: The Enzymes </A></P><P><A href="https://2014.igem.org/Team:Exeter/EnzymeValidation">Next: Enzyme Validation </A></P></DIV><P><TITLE>Exeter | ERASE</TITLE></P><DIV id="footer"><A id="page_top" href="#">Top</A><P><A href="https://twitter.com/ExeteriGEM2014" class="twitter-follow-button" data-show-count="false">Follow @ExeteriGEM2014</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Exeter/Regulatory_Sequences">http://2014.igem.org/Team:Exeter/Regulatory_Sequences</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Exeter/Regulatory_Sequences" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Exeter/Regulatory_Sequences" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Exeter/Regulatory_Sequences&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Exeter/Regulatory_Sequences&amp;oldid=351812" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>