<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Exeter_DegradationConstructs"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Exeter/DegradationConstructs">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Exeter/DegradationConstructs&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Exeter/DegradationConstructs&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Exeter/DegradationConstructs&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Exeter/DegradationConstructs" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Exeter/DegradationConstructs</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Exeter | ERASE</TITLE></P><DIV id="wrapper"><DIV id="header"><UL id="header_list"><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter">Home</A></DIV></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/TheProblem">The Problem</A></DIV></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/Project">Our Project</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/Modelling">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/DegradationConstructs">The Enzymes</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/EnzymeValidation">Enzyme Validation</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/invivoactivity"><I>in vivo</I>: Raman </A></LI><LI><A href="https://2014.igem.org/Team:Exeter/invivo"><I>in vivo</I>: Observations </A></LI><LI><A href="https://2014.igem.org/Team:Exeter/enzyme-kinetics"> HPLC </A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Kill_Switches">Kill Switches</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/EColiStressTesting"> Xenobiotic Tolerance</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Detection">Detection of Xenobiotics</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/iLOVCharacterisation">iLOV Characterisation</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Parts">Our Parts</A></LI></UL></DIV><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/MeetTheTeam">The Team</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/MeetTheTeam">The Undergraduate Team</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Supervisors">Supervisors</A></LI><LI><A href="https://igem.org/Team.cgi?id=1398">Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Sponsors">Sponsors</A></LI></DIV><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/MedalRequirements">Medal Requirements</A></DIV></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/Notebook">Notebook</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/LabBook">Lab Book</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Protocols">Protocols</A></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/Outreach">Outreach</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/BeyondTheBench">Beyond The Bench</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Stakeholders">Stakeholders</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Presentations">Presentations</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Game">  ERASE-The Game!</A></LI></DIV></DIV><DIV id="description"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#TheSolution:OldYellowEnzymes"><SPAN class="tocnumber">1.</SPAN><SPAN class="toctext">Old Yellow Enzymes</SPAN></A></LI><LI class="toclevel-1"><A href="#XenB(BBa_K1398001)"><SPAN class="tocnumber">2.</SPAN><SPAN class="toctext">XenB (BBa_K1398001)</SPAN></A></LI><UL><LI class="toclevel-2"><A href="#DegradationofTNT"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Degradation of TNT</SPAN></A></LI><LI class="toclevel-2"><A href="#DegradationofNitroglycerin"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Degradation of Nitroglycerin</SPAN></A></LI><LI class="toclevel-2"><A href="#TheXenBConstruct:(BBa_K1398001)"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">The XenB Construct: (BBa_K1398001)</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#NemA(BBa_K1398002)"><SPAN class="tocnumber">3.</SPAN><SPAN class="toctext">NemA (BBa_K1398002)</SPAN></A></LI><UL><LI class="toclevel-2"><A href="#TheNemAConstruct:(BBa_K1398002)"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">The NemA Construct: (BBa_K1398002)</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">4.</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="TheSolution:OldYellowEnzymes">Old Yellow Enzymes</SPAN></H1><P> Several microbial enzymes have the ability to catalyse the breakdown of nitro- and nitroaromatic compounds such as TNT and NG. These enzymes fall into two main families: </P><OL><LI><STRONG>Oxygen-Insensitive Nitroreductases:</STRONG> These enzymes sequentially perform two-electron reductions of nitro groups. They typically contain Flavin mononucleotides FMN which they use, along with NADPH as a cofactor and electron donor. Examples include the nitroreductases NfsA and NfsB from <I>Escherichia coli,</I><SUP>1</SUP> PnrA and PnrB from <I>Pseudomonas putida,</I><SUP>2</SUP> and NitA and NitB from <I>Clostridium acetobutylicum</I>.<SUP>3</SUP></LI><LI><STRONG>Old Yellow Enzymes (OYE):</STRONG> The physiological function of this family of NADPH dehydrogenases is not yet well established; however, they are often associated with nitroaromatic compound reduction. Within the OYE family, two types of enzymes have been described:</LI><UL><UL><P>Type I hydride transferases, which, like the oxygen-insensitive nitroreductases above, reduce the nitroaromatic compounds to hydroxylamine derivatives.</P><P>Type II hydride transferases, which catalyse a nucleophilic attack on the aromatic ring of TNT.<SUP>4,5</SUP></P></UL></UL><P>Of bacterial OYE family members, those that are the best characterised are XenA - XenF of <I>P. putida</I> KT24406 XenB from <I>Pseudomonas fluorescens,</I><SUP>7</SUP> PETN reductase from <I>Enterobacter cloacae</I> PB28, NemA reductase from <I>E. coli</I><SUP>9</SUP> and YqjM from <I>Bacillus subtilis</I>.<SUP>10</SUP></P></OL><P>In 2009 the <A href="https://2009.igem.org/Team:Edinburgh">Edinburgh iGEM team</A> developed the concept of a Nitrate/Nitrite biosensor which could be used to detect TNT. They generated a biobrick (<A href="http://parts.igem.org/Part:BBa_K216006">BBa_K216006</A>) for the gene <I>onr</I> (organic nitrate reductase) that encodes pentaerythritol tetranitrate (PETN) reductase to function as a TNT degrader. However, neither were completely characterised. </P><P>We sought to find new, mostly uncharacterised, enzymes to function as additional solutions to offer alternative mechanisms which may be more suited to particular problems. Given that the PETN reductase was a member of the Old Yellow Enzyme family, we decided that we would target additional members of the OYE group to help expand the both the range of enzymes that can be utilised to degrade explosives and the range of unexploded ordnance chemicals that could be degraded to those that contain nitroglycerin.</P><P>Our initial shortlist contained the following the proteins: XenA, XenB, NemA and YqjM. From these we selected two to examine over the summer. These were <STRONG>XenB</STRONG> and <STRONG>NemA</STRONG> which have been submitted to the Registry of Standard Biological Parts as <A href="http://parts.igem.org/Part:BBa_K1398001">BBa_K1398001</A> and <A href="http://parts.igem.org/Part:BBa_K1398002">BBa_K1398002</A>respectively.</P><H1><SPAN class="mw-headline" id="XenB(BBa_K1398001)">XenB (<A href="http://parts.igem.org/Part:BBa_K1398001">BBa_K1398001</A>)</SPAN></H1><P> XenB is an NADH-dependent flavoprotein (Xenobiotic Reductase B) from the soil bacterium <I>Pseudomonas fluorescens</I>. We chose XenB because:</P><OL><LI>XenB may serve a dual purpose as a Nitroglycerin and TNT degrading enzyme (see below).</LI><LI>The phylogeny of OYE members<SUP>11</SUP> suggests that XenB is similar in sequence to the previously submitted PETN reductase and is comparatively well understood.</LI><LI>Heterologous expression of XenB from <I>P. fluorescens</I> has been previously demonstrated in <I>E. coli</I> DH5α<SUP>12</SUP> enhancing our chances of successful expression. </LI></OL><H2><SPAN class="mw-headline" id="DegradationofTNT">Degradation of TNT</SPAN></H2><P> Pak et al. (2000)<SUP>14</SUP> reported the purification of the NADH-dependent flavoprotein oxidoreductase xenobiotic reductase B (XenB) from <I>Pseudomonas fluorescens</I>. </P><P> These authors provide a guide to some of the degradative reactions of XenB from <I>P. fluorescens</I>. This indicates that XenB may catalysed the reduction of TNT, either by adding a hydride to the aromatic ring and forming a dihydride Meisenheimer complex<SUP>15</SUP> or by catalysing the reduction of nitro groups directly (Figure 1). 
However several unidentified products and proposed intermediates indicate that this process is yet to be fully elucidated.</P><P align="center"><I><STRONG>Figure 1:</STRONG> Diagram depicting the potential reductive steps catalysed by XenB. (Adapted from Pak et al.)<SUP>14</SUP></I></P><P> </P><P> XenB from <I>Pseudomonas fluorescens</I> is not the only XenB enzyme that exists. A more commonly studied XenB enzyme is found in <I>Pseudomonas putida</I>. This XenB shares 88% identity to our chosen XenB from <I>P. fluorescens</I>.<SUP>16</SUP></P><P> XenB from <I>P. putida</I> (as well as other xenobiotic reductases) demonstrate type II hydride transferase activity against TNT. Research into these xenobiotic reductases provided a greater understanding of the potential products and intermediates generated in the reactions with TNT (Figure 2).  Using figure 1 as a model,  in the mechanisms suggested by Pak et al. (2000) (above) it seems likely that the unknown m/z 196 compound was 4ADNT and that the proposed bridge product (m/z= 376) is instead linked by an amine group bridge (Figure 2). </P><P align="center"><I><STRONG>Figure 2:</STRONG> Diagram depicting the known reductive steps catalysed by XenB. (Adapted from van Dillewijn et al.)<SUP>17</SUP></I></P><P> </P><P> We were unable to find more recent attempts to characterise the mechanism of XenB from <I>P. fluorescens</I>. However, given that these mechanisms for <I>P. putida</I> Xenobiotic reductase proteins are similar to the mechanisms of the type II hydride transferase family<SUP>18</SUP> it suggests we are more likely to find the amine-bridged products in Figure 2. </P><H2><SPAN class="mw-headline" id="DegradationofNitroglycerin">Degradation of Nitroglycerin</SPAN></H2><P> XenB also displays an apparent capacity to denitrify Nitroglycerin (NG).  It has been demonstrated that XenB catalyses the NADH-dependent cleavage of nitro groups from NG, releasing nitrite (Figure 3). 
<I>P. fluorescens</I> XenB also exhibits five-fold regioselectivity for removal of the central nitro group from NG compared to <I>P. putida</I> XenB. </P><P align="center"><I><STRONG>Figure 3:</STRONG> Simplistic diagram depicting the chemistry involved in Glycerol Trinitrate transformation by XenB.<SUP>19</SUP></I></P><P> </P><P>As nitrate is released from the degradation of Nitroglycerin, there is a suggestion that construct could be built upon to work simultaneously as an explosive degrader and soil enricher, thus taking an environmental toxicant and converting it into an environmental benefit. </P><H2><SPAN class="mw-headline" id="TheXenBConstruct:(BBa_K1398001)">The XenB Construct: (<A href="http://parts.igem.org/Part:BBa_K1398001">BBa_K1398001</A>)</SPAN></H2><P>The coding sequence for <I>P. fluorescens</I> XenB has been deposited within GenBank (accession no. <A href="http://www.ncbi.nlm.nih.gov/nuccore/AF154062">AF154062</A>)<SUP>20</SUP> and this forms the functional unit of our construct (<A href="http://parts.igem.org/Part:BBa_K1398001">BBa_K1398001</A>) (Figure 4):  </P><P align="center"><I><STRONG> Figure 4: </STRONG>The XenB construct depicting the constituent parts of the construct.</I></P><P> </P><P>The construct contains the coding sequence for XenB (<A href="http://parts.igem.org/Part:BBa_K1398000">BBa_K1398000</A>). Expression of XenB is driven by a Lactose-inducible promoter (<A href="http://parts.igem.org/Part:BBa_R0010">BBa_R0010</A>) coupled with a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>). The construct is terminated using a double terminator made up of 

<A href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A> and 
<A href="http://parts.igem.org/Part:BBa_B0012">BBa_B0012</A>.</P><H1><SPAN class="mw-headline" id="NemA(BBa_K1398002)">NemA (<A href="http://parts.igem.org/Part:BBa_K1398002">BBa_K1398002</A>)</SPAN></H1><P> The second of our enzymes is the detoxification enzyme N-Ethylmaleimide (NEM) reductase from <I>Escherichia coli</I> encoded by the gene nemA).<SUP>21</SUP> Often referred to simply as NemA, the flavin-dependent NEM Reductase was chosen for several reasons: </P><OL><LI>	Firstly, like XenB, NemA has a dual capacity to degrade both TNT and Nitroglycerin.
</LI><LI>	NemA has a high degree of homology (87% identical) to pentaerythritol tetranitrate (PETN) reductase (<A href="http://parts.igem.org/Part:BBa_K216006">BBa_K216006</A>) from <I>Enterobacter cloacae</I>.<SUP>22</SUP> The <I>E. cloacae</I> PETN reductases and <I>E. coli</I> NEM reductase showed broadly similar activity profiles, with high activity against nitrate esters.<SUP>23</SUP> However, NemA was shown to have a higher substrate preference for PETN and Nitroglycerin than PETN reductase, the uncharacterised gene of which was submitted by the iGEM team from Edinburgh in 2009 William’s et al. (2004).<SUP>24</SUP></LI><LI>	Finally, NemA was found to already exist in <I>E. coli</I>, albeit at a low expression level. This suggested to us a capability to be expressed successfully. However, we also realised this meant that we would more likely to face natural control mechanisms that could silence the activity of the enzyme if it is overexpressed.  It also meant that we would need to assess the natural tolerance of our <I>E. coli</I> DH5α strain to the explosives to ensure our engineered enzyme provided enhanced tolerance to TNT and NG compared to the wild-type.  
</LI></OL><P> Through a comparative study of PETN, NemA, Morphinone reductase and the yeast OYE, the TNT mechanism for the similar PETN reductase was detailed.<SUP>25</SUP> Due to the sequence similarity, we are using this as a model for the pathways involved in anaerobic degradation of TNT by NemA (Figure 5). </P><P align="center"><I><STRONG>Figure 5: </STRONG>Diagram depicting the potential mechanistic steps in the transformation of TNT by PETN reductase. (Adapted from Rylott et al. <SUP>25</SUP> and Williams et al.)<SUP>26</SUP></I></P><P> </P><P> In addition to degradation of TNT and NG, NemA may also function as an efficient chromate reductase to remediate hexavalent chromium, a serious and widespread environmental pollutant.<SUP>27</SUP></P><H2><SPAN class="mw-headline" id="TheNemAconstruct(BBa_K1398003)">The NemA construct (<A href="http://parts.igem.org/Part:BBa_K1398003">BBa_K1398003</A>)</SPAN></H2><P> With the nemA DNA sequence identified<SUP>28</SUP> we were able to design our construct to evaluate NemA as follows (Figure 6): </P><P align="center"><I><STRONG>Figure 6: </STRONG> The NemA construct depicting the constituent parts of the construct. </I></P><P> The construct contains the coding sequence for NemA (<A href="http://parts.igem.org/Part:BBa_K1398002">BBa_K1398002</A>), an enzyme involved in the degradation of toxic compounds for their reuse in nitrogen metabolism. The construct also contains a Lactose-inducible promoter (<A href="http://parts.igem.org/Part:BBa_R0010">BBa_R0010</A>), a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>) and a double terminator made up of <A href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A> and <A href="http://parts.igem.org/Part:BBa_B0012">BBa_B0012</A>). The protein has been codon-optimised for expression in <I>E. coli</I>. </P>

----

<H1><SPAN class="mw-headline" id="References">References:</SPAN></H1><OL><LI>Zenno et al., 1996 NfsA</LI><LI>Caballero et al., 2005 PnrA/B</LI><LI>Kutty and Bennett, 2005 NitA/B</LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764</A></LI><LI>Miura et al.1997 NemA</LI><LI>YqjM from Bacillus subtilis (Fitzpatrick et al.,2003).</LI><LI><A http:="">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC103757/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC103757/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374/</A></LI><LI><A href="http://pubchem.ncbi.nlm.nih.gov/summary/summary.cgi?sid=14709034">http://pubchem.ncbi.nlm.nih.gov/summary/summary.cgi?sid=14709034</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576690/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576690/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/10515912">http://www.ncbi.nlm.nih.gov/pubmed/10515912</A></LI><LI><A href="https://2009.igem.org/Team:Edinburgh/biology%28tntsensing%29">http://www.ncbi.nlm.nih.gov/pubmed/10515912 </A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/9013822/">http://www.ncbi.nlm.nih.gov/pubmed/9013822/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764/</A></LI><LI><A href="http://www.sciencedirect.com/science/article/pii/S0958166910002028">http://www.sciencedirect.com/science/article/pii/S0958166910002028</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596305/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596305/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/9013822/">http://www.ncbi.nlm.nih.gov/pubmed/9013822/</A></LI></OL><H2> Navigation </H2><P><A href="https://2014.igem.org/Team:Exeter/Modelling">Previous: Modelling </A></P><P><A href="https://2014.igem.org/Team:Exeter/EnzymeValidation">Next: Enzyme Validation </A></P></DIV><P><TITLE>Exeter | ERASE</TITLE></P><DIV id="footer"><A id="page_top" href="#">Top</A><P><A href="https://twitter.com/ExeteriGEM2014" class="twitter-follow-button" data-show-count="false">Follow @ExeteriGEM2014</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Exeter/DegradationConstructs">http://2014.igem.org/Team:Exeter/DegradationConstructs</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Exeter/DegradationConstructs" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Exeter/DegradationConstructs" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Exeter/DegradationConstructs&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Exeter/DegradationConstructs&amp;oldid=398952" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>