"
Page
Discussion
View source
History
teams
Log in
 
Team:Exeter/DegradationConstructs
From 2014.igem.org
Exeter | ERASE
Home
The Problem
Our Project
Modelling
The Enzymes
Enzyme Validation
in vivo: Raman
in vivo: Observations
HPLC
Kill Switches
Xenobiotic Tolerance
Detection of Xenobiotics
iLOV Characterisation
Our Parts
The Team
The Undergraduate Team
Supervisors
Official Profile
Attributions
Sponsors
Medal Requirements
Notebook
Lab Book
Protocols
Outreach
Beyond The Bench
Stakeholders
Presentations
ERASE-The Game!
Contents
1. Old Yellow Enzymes
2. XenB (BBa_K1398001)
2.1 Degradation of TNT
2.2 Degradation of Nitroglycerin
2.3 The XenB Construct: (BBa_K1398001)
3. NemA (BBa_K1398002)
3.1 The NemA Construct: (BBa_K1398002)
4. References
Old Yellow Enzymes
Several microbial enzymes have the ability to catalyse the breakdown of nitro- and nitroaromatic compounds such as TNT and NG. These enzymes fall into two main families:
Oxygen-Insensitive Nitroreductases: These enzymes sequentially perform two-electron reductions of nitro groups. They typically contain Flavin mononucleotides FMN which they use, along with NADPH as a cofactor and electron donor. Examples include the nitroreductases NfsA and NfsB from Escherichia coli,1 PnrA and PnrB from Pseudomonas putida,2 and NitA and NitB from Clostridium acetobutylicum.3
Old Yellow Enzymes (OYE): The physiological function of this family of NADPH dehydrogenases is not yet well established; however, they are often associated with nitroaromatic compound reduction. Within the OYE family, two types of enzymes have been described:
Type I hydride transferases, which, like the oxygen-insensitive nitroreductases above, reduce the nitroaromatic compounds to hydroxylamine derivatives.
Type II hydride transferases, which catalyse a nucleophilic attack on the aromatic ring of TNT.4,5
Of bacterial OYE family members, those that are the best characterised are XenA - XenF of P. putida KT24406 XenB from Pseudomonas fluorescens,7 PETN reductase from Enterobacter cloacae PB28, NemA reductase from E. coli9 and YqjM from Bacillus subtilis.10
In 2009 the Edinburgh iGEM team developed the concept of a Nitrate/Nitrite biosensor which could be used to detect TNT. They generated a biobrick ( BBa_K216006 ) for the gene onr (organic nitrate reductase) that encodes pentaerythritol tetranitrate (PETN) reductase to function as a TNT degrader. However, neither were completely characterised.
We sought to find new, mostly uncharacterised, enzymes to function as additional solutions to offer alternative mechanisms which may be more suited to particular problems. Given that the PETN reductase was a member of the Old Yellow Enzyme family, we decided that we would target additional members of the OYE group to help expand the both the range of enzymes that can be utilised to degrade explosives and the range of unexploded ordnance chemicals that could be degraded to those that contain nitroglycerin.
Our initial shortlist contained the following the proteins: XenA, XenB, NemA and YqjM. From these we selected two to examine over the summer. These were XenB and NemA which have been submitted to the Registry of Standard Biological Parts as BBa_K1398001 and BBa_K1398002 respectively.
XenB ( BBa_K1398001 )
XenB is an NADH-dependent flavoprotein (Xenobiotic Reductase B) from the soil bacterium Pseudomonas fluorescens. We chose XenB because:
XenB may serve a dual purpose as a Nitroglycerin and TNT degrading enzyme (see below).
The phylogeny of OYE members11 suggests that XenB is similar in sequence to the previously submitted PETN reductase and is comparatively well understood.
Heterologous expression of XenB from P. fluorescens has been previously demonstrated in E. coli DH5α12 enhancing our chances of successful expression.
Degradation of TNT
Pak et al. (2000)14 reported the purification of the NADH-dependent flavoprotein oxidoreductase xenobiotic reductase B (XenB) from Pseudomonas fluorescens.
These authors provide a guide to some of the degradative reactions of XenB from P. fluorescens. This indicates that XenB may catalysed the reduction of TNT, either by adding a hydride to the aromatic ring and forming a dihydride Meisenheimer complex15 or by catalysing the reduction of nitro groups directly (Figure 1).  However several unidentified products and proposed intermediates indicate that this process is yet to be fully elucidated.
Figure 1: Diagram depicting the potential reductive steps catalysed by XenB. (Adapted from Pak et al.)14
 
XenB from Pseudomonas fluorescens is not the only XenB enzyme that exists. A more commonly studied XenB enzyme is found in Pseudomonas putida. This XenB shares 88% identity to our chosen XenB from P. fluorescens.16
XenB from P. putida (as well as other xenobiotic reductases) demonstrate type II hydride transferase activity against TNT. Research into these xenobiotic reductases provided a greater understanding of the potential products and intermediates generated in the reactions with TNT (Figure 2).  Using figure 1 as a model,  in the mechanisms suggested by Pak et al. (2000) (above) it seems likely that the unknown m/z 196 compound was 4ADNT and that the proposed bridge product (m/z= 376) is instead linked by an amine group bridge (Figure 2).
Figure 2: Diagram depicting the known reductive steps catalysed by XenB. (Adapted from van Dillewijn et al.)17
 
We were unable to find more recent attempts to characterise the mechanism of XenB from P. fluorescens. However, given that these mechanisms for P. putida Xenobiotic reductase proteins are similar to the mechanisms of the type II hydride transferase family18 it suggests we are more likely to find the amine-bridged products in Figure 2.
Degradation of Nitroglycerin
XenB also displays an apparent capacity to denitrify Nitroglycerin (NG).  It has been demonstrated that XenB catalyses the NADH-dependent cleavage of nitro groups from NG, releasing nitrite (Figure 3). P. fluorescens XenB also exhibits five-fold regioselectivity for removal of the central nitro group from NG compared to P. putida XenB.
Figure 3: Simplistic diagram depicting the chemistry involved in Glycerol Trinitrate transformation by XenB.19
 
As nitrate is released from the degradation of Nitroglycerin, there is a suggestion that construct could be built upon to work simultaneously as an explosive degrader and soil enricher, thus taking an environmental toxicant and converting it into an environmental benefit.
The XenB Construct: ( BBa_K1398001 )
The coding sequence for P. fluorescens XenB has been deposited within GenBank (accession no. AF154062 )20 and this forms the functional unit of our construct ( BBa_K1398001 ) (Figure 4):
Figure 4: The XenB construct depicting the constituent parts of the construct.
 
The construct contains the coding sequence for XenB ( BBa_K1398000 ). Expression of XenB is driven by a Lactose-inducible promoter ( BBa_R0010 ) coupled with a strong RBS ( BBa_B0034 ). The construct is terminated using a double terminator made up of BBa_B0010 and BBa_B0012 .
NemA ( BBa_K1398002 )
The second of our enzymes is the detoxification enzyme N-Ethylmaleimide (NEM) reductase from Escherichia coli encoded by the gene nemA).21 Often referred to simply as NemA, the flavin-dependent NEM Reductase was chosen for several reasons:
Firstly, like XenB, NemA has a dual capacity to degrade both TNT and Nitroglycerin.
NemA has a high degree of homology (87% identical) to pentaerythritol tetranitrate (PETN) reductase ( BBa_K216006 ) from Enterobacter cloacae.22 The E. cloacae PETN reductases and E. coli NEM reductase showed broadly similar activity profiles, with high activity against nitrate esters.23 However, NemA was shown to have a higher substrate preference for PETN and Nitroglycerin than PETN reductase, the uncharacterised gene of which was submitted by the iGEM team from Edinburgh in 2009 William’s et al. (2004).24
Finally, NemA was found to already exist in E. coli, albeit at a low expression level. This suggested to us a capability to be expressed successfully. However, we also realised this meant that we would more likely to face natural control mechanisms that could silence the activity of the enzyme if it is overexpressed.  It also meant that we would need to assess the natural tolerance of our E. coli DH5α strain to the explosives to ensure our engineered enzyme provided enhanced tolerance to TNT and NG compared to the wild-type.
Through a comparative study of PETN, NemA, Morphinone reductase and the yeast OYE, the TNT mechanism for the similar PETN reductase was detailed.25 Due to the sequence similarity, we are using this as a model for the pathways involved in anaerobic degradation of TNT by NemA (Figure 5).
Figure 5: Diagram depicting the potential mechanistic steps in the transformation of TNT by PETN reductase. (Adapted from Rylott et al. 25 and Williams et al.)26
 
In addition to degradation of TNT and NG, NemA may also function as an efficient chromate reductase to remediate hexavalent chromium, a serious and widespread environmental pollutant.27
The NemA construct ( BBa_K1398003 )
With the nemA DNA sequence identified28 we were able to design our construct to evaluate NemA as follows (Figure 6):
Figure 6: The NemA construct depicting the constituent parts of the construct.
The construct contains the coding sequence for NemA ( BBa_K1398002 ), an enzyme involved in the degradation of toxic compounds for their reuse in nitrogen metabolism. The construct also contains a Lactose-inducible promoter ( BBa_R0010 ), a strong RBS ( BBa_B0034 ) and a double terminator made up of BBa_B0010 and BBa_B0012 ). The protein has been codon-optimised for expression in E. coli.
----
References:
Zenno et al., 1996 NfsA
Caballero et al., 2005 PnrA/B
Kutty and Bennett, 2005 NitA/B
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764
Miura et al.1997 NemA
YqjM from Bacillus subtilis (Fitzpatrick et al.,2003).
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC103757/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC92374/
http://pubchem.ncbi.nlm.nih.gov/summary/summary.cgi?sid=14709034
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576699/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2576690/
http://www.ncbi.nlm.nih.gov/pubmed/10515912
http://www.ncbi.nlm.nih.gov/pubmed/10515912
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764
http://www.ncbi.nlm.nih.gov/pubmed/9013822/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764/
http://www.sciencedirect.com/science/article/pii/S0958166910002028
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC427764/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596305/
http://www.ncbi.nlm.nih.gov/pubmed/9013822/
Navigation
Previous: Modelling
Next: Enzyme Validation
Exeter | ERASE
Top
Follow @ExeteriGEM2014
Retrieved from " http://2014.igem.org/Team:Exeter/DegradationConstructs "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
