<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Exeter_Detection"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Exeter/Detection">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Exeter/Detection&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Exeter/Detection&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Exeter/Detection&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Exeter/Detection" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Exeter/Detection</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Exeter | ERASE</TITLE></P><DIV id="wrapper"><DIV id="header"><UL id="header_list"><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter">Home</A></DIV></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/TheProblem">The Problem</A></DIV></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/Project">Our Project</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/Modelling">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/DegradationConstructs">The Enzymes</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/EnzymeValidation">Enzyme Validation</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/invivoactivity"><I>in vivo</I>: Raman </A></LI><LI><A href="https://2014.igem.org/Team:Exeter/invivo"><I>in vivo</I>: Observations </A></LI><LI><A href="https://2014.igem.org/Team:Exeter/enzyme-kinetics"> HPLC </A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Kill_Switches">Kill Switches</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/EColiStressTesting"> Xenobiotic Tolerance</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Detection">Detection of Xenobiotics</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/iLOVCharacterisation">iLOV Characterisation</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Parts">Our Parts</A></LI></UL></DIV><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/MeetTheTeam">The Team</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/MeetTheTeam">The Undergraduate Team</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Supervisors">Supervisors</A></LI><LI><A href="https://igem.org/Team.cgi?id=1398">Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Sponsors">Sponsors</A></LI></DIV><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/MedalRequirements">Medal Requirements</A></DIV></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/Notebook">Notebook</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/LabBook">Lab Book</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Protocols">Protocols</A></LI><LI id="header_list_element"><DIV class="header_link_div"><A class="header_link" href="https://2014.igem.org/Team:Exeter/Outreach">Outreach</A></DIV></LI><LI><A href="https://2014.igem.org/Team:Exeter/BeyondTheBench">Beyond The Bench</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Stakeholders">Stakeholders</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Presentations">Presentations</A></LI><LI><A href="https://2014.igem.org/Team:Exeter/Game">  ERASE-The Game!</A></LI></DIV></DIV><DIV id="description"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#TheProjectTestingthesensitivityofourpromoterstoTNTNG"><SPAN class="tocnumber">1.</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1"><A href="#ResultsOverview"><SPAN class="tocnumber">2.</SPAN><SPAN class="toctext">Results &amp; Experimental Overview</SPAN></A></LI><LI class="toclevel-1"><A href="#Results"><SPAN class="tocnumber">3.</SPAN><SPAN class="toctext">In-Depth Results</SPAN></A></LI><LI class="toclevel-1"><A href="#Conclusion"><SPAN class="tocnumber">4.</SPAN><SPAN class="toctext">Summary</SPAN></A></LI><LI class="toclevel-1"><A href="#Materials"><SPAN class="tocnumber">5.</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-1"><A href="#Method"><SPAN class="tocnumber">6.</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">7.</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="TheNemRrecognitionpromoter(BBa_K1398007)">The NemR recognition promoter</SPAN></H2><P> As part of our design we wanted cells that would self destruct when the target molecules (TNT or NG) were depleted. This would prevent cells persisting in the soil that was being decontaminated after their job was done. One of the problems faced by the 2009 Edinburgh iGEM team was the issue of how TNT was to be detected in order to generate a gene expression response. See <A href="https://2009.igem.org/Team:Edinburgh/biology%28results%29"> Edinburgh '09 Results</A> and <A href="https://2009.igem.org/Team:Edinburgh/biology%28biobricks%29">Edinburgh '09 Biobricks</A></P><P>They developed the following system: </P><P align="center"><I><STRONG>Figure 1: </STRONG>TNT sensing pathway developed by the Edinburgh 2009 iGEM team. 
<A href="https://2009.igem.org/Team:Edinburgh/biology%28tntsensing%29">https://2009.igem.org/Team:Edinburgh/biology%28tntsensing%29</A>
TNT binds to TNT.R1 in the periplasm. The TNT-TNT.R1 complex induces a conformational change in the Trg-EnvZ (Trz) fusion protein. Trg-EnvZ autophosphorylates and subsequently phosphorylates ompR. Phosphorylated ompR activates transcription. </I></P><P>There are a large number of parts within this system that all have to work. We decided instead to develop a simpler system that would be easier to engineer . After the discovery that the NemA enzyme could already be expressed at in <I>E. coli</I>, it was hypothesised that <I>E. coli</I> should have a gene regulatory molecule which controlled the nemA operon. This would enable its expression in environments where TNT was present. In <I>E. coli</I>, nemA is located downstream of a gene encoding a transcription factor known as NemR, previously entitled YdhM<SUP>1</SUP>.  Disruption to the gene encoding NemR results in a decrease in nemA expression, both in the basal and induced states. This indicates that the nemA gene and NemR form a single, connected operon. Umezawa et al. (2008) proposed that the function of NemR is as a redox-operated transcriptional repressor of the nemA gene. 
NemR is proposed to remain bound to the operon and, when it binds TNT in the environment, undergo a conformational change to expose the promoter, resulting in increased protein expression. The region of the promoter sequence to which NemR binds (the ‘NemR recognition box) has also been identified<SUP>2</SUP> (Figure 2). </P><P align="center"><I><B>Figure 2: </B> Diagram depicting the sequence of the ‘NemR box’. Adapted from Umezawa et al.<SUP>3</SUP></I></P><P>On the assumption that TNT was able to naturally diffuse into the <I>E. coli</I> cell, we hypothesised that if we designed a promoter containing this binding site DNA sequence, the promoter would  become inducible through the addition of TNT. The NemR transcription factor which is naturally found in <I>E. coli</I> and we confirmed ts presence in the genome of our laboratory strain. This would be a simpler TNT-detecting gene expression system as it would rely on the success of  fewer engineered parts.
It has been shown that NemR also responds to cysteine-modifying electrophiles, including NEM, showdomycin, and, more weakly, iodoacetamide <SUP>4</SUP>. The repressor function of NemR was inactivated through the addition of Cysteine modifying reagents. Furthermore, based upon a predicted 3D structure (Figure 3), it has been proposed that NemR utilises reversible oxidation of a conserved Cys-106  residue as the signal for confirmation change and dissociation from the DNA strand thus activating gene expression<SUP>5</SUP>.</P><P align="center"><I><STRONG>Figure 3: </STRONG>3D  molecular structure of the E. coli NemR monomer. Cysteine residues are in orange, conserved Cys-106 in red, and DNA binding helices in green<SUP>6</SUP>.</I></P><P>Finally, it has also been indicated that the NemR protein was responsive to Hypochlorous acid (HOCl), the active component of household bleach. Addition of bleach resulted in the expression of two detoxifying enzymes for bleach: glyoxalase I and NemA<SUP>7</SUP>.</P><P>We have therefore designed and tested two promoters based on the NemR response described above. These are parts (<A href="http://parts.igem.org/Part:BBa_K1398004">BBa_K1398004</A>) and (<A href="http://parts.igem.org/Part:BBa_K1398007">BBa_K1398007</A>). <A href="http://parts.igem.org/Part:BBa_K1398004">BBa_K1398004</A> includes the entire region upstream of the nemR coding sequence, placed in front of our reporter gene, iLOV.<A href="http://parts.igem.org/Part:BBa_K1398007">BBa_K1398007</A> instead uses the specific sequence known to be the NemR binding site, and places it between the -35 and -10 sections of a high-level constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A> developed as part of the <A href="http://parts.igem.org/Promoters/Catalog/Anderson">Anderson Collection</A> of promoters. See <A href="https://2014.igem.org/Team:Exeter/Parts#3">Composite Parts</A> for a deconstruction of each part. </P><P>  We observed the change in production of the fluorescent protein iLOV at varying concentrations of TNT and NG. We did this by measuring the growth and fluorescence of 200ul cultures on a 96-well plate. We found that <A href="http://parts.igem.org/Part:BBa_K1398004">BBa_K1398004</A> appears to express protein constitutively relative to growth, while <A href="http://parts.igem.org/Part:BBa_K1398007">BBa_K1398007</A> had increased expression in response to higher levels of TNT.</P><H2><SPAN class="mw-headline" id="TheProjectTestingthesensitivityofourpromoterstoTNTNG">The Project: Testing the sensitivity of our promoters to TNT/NG</SPAN></H2><H2><SPAN class="mw-headline" id="ResultsOverview">Experimental Overview</SPAN></H2><P>We have tested how well our promoters responded to non-lethal doses of TNT and nitroglycerin by observing the change in fluorescence over time in a fresh culture after a set volume of the chemical had been added. A set volume of overnight culture was added to MYE media in a 96 well plate, as well as a volume of TNT/NG. By using a 96-well plate we were able to massively increase the range of concentrations of compounds we were able to test. The information used to find out lethal and non-lethal levels of TNT came from our studies into <A href="https://2014.igem.org/Team:Exeter/EColiStressTesting">The Toxicity of TNT and Nitroglycerin to <I>E. coli</I></A>.</P><P>Throughout this experiment several strains of <I>E. coli</I> were used. 004- and 007-Top10 were being tested for their fluorescence in response to TNT/NG. WT-Top10 cells were used as a negative control, to ensure we weren’t recording any natural fluorescence of Top10. GFP-Top10 was used a positive control to ensure we recorded the fluorescence of the cultures correctly. As MYE media was used to grow the cultures it was also used a negative control, to ensure we weren’t measuring any fluorescence from the media.</P><P><B>Experiment One</B>: Is iLOV fluorescent when expressed in <I>E. coli</I>?</P><P><B>Experiment Two</B>: Are our promoters functional with and/or without TNT?</P><H2><SPAN class="mw-headline" id="Results">In-Depth Results</SPAN></H2><P><B>Experiment One</B>: Is iLOV fluorescent when expressed in <I>E. coli</I>?</P><FIGURE><FIGCAPTION><I><B>Figure 4</B></I></FIGCAPTION></FIGURE><P>The fluorescence of WT-cultures, as well as cultures expressing GFP and iLOV. The fluorescence of iLOV (at excitation = 440 nm, emission = 520 nm) reaches a 4400 at 20h, while WT-Top10 cultures at 20h have a fluorescence of 880.</P><P>Figure 4 shows the fluorescence of iLOV in <I>E. coli</I> can clearly be seen in this experiment. There is clearly a huge increase in fluorescence compared to WT-Top10. It can also be seen that under regular conditions maximal iLOV fluorescence is reached between 20-25 hours. This information was used when selecting a time point for the comparison of fluorescence to optical density. The relative fluorescence of GFP and iLOV are not comparable in this experiment as they are under the control of different promoters, although it is suspected that the fluorescence of GFP is greater.</P><P><B>Experiment Two</B>: Are our promoters functional with and/or without TNT?</P><FIGURE><FIGCAPTION><I><B>Figure 5</B></I></FIGCAPTION></FIGURE><P>Figure 5 shows the fluorescence of each cell culture in comparison to its optical density. As the concentration of TNT increases the Flu:OD of 004-Top10 stays fairly consistent, varying between 900 and 1120, while the Flu:OD of 007-Top10 increases as TNT concentration increases, from 380 at 0ul to 1020 at 10ul. The values for fluorescence and optical density were taken at 24 h.</P><FIGURE><FIGCAPTION><I><B>Figure 6</B></I></FIGCAPTION></FIGURE><P>Figure 6 shows the Flu:OD of each culture at each level of TNT in comparison to the Flu:OD at TNT = 0ul. This figure makes it clear that 007 increases in relative fluorescence as TNT concentration increases; eventually reaching 2.7x its original value. The values for fluorescence and optical density were taken at 24 h.</P><P>As shown in &quot;<I>E. coli</I>’s response to Xenobiotic Compounds” cell growth decreases in response to increasing concentrations of TNT. As cell growth decreases protein expression also decreases, proportionally. Therefore it is difficult to associate an increased promoter response with an increased level of TNT. The easiest way to show this data is to present fluorescence relative to the optical density of the culture, which is roughly proportional to growth.</P><P>The expression of 004 remains fairly constant relative to growth, while 007 increases in relative activity as the concentration of TNT increases, indicating that it is responding to the increased levels. </P><P>004 appears to be a general constitutive promoter, but 007 appears to have a definite response to TNT over other promoters. Although expression appears to slightly increase between 0, 2.5, 5 and 7.5 ul TNT, there is a steep difference between the expression shown in 7.5ul TNT and 10ul. This may indicate a certain cut-off point at which the promoter activates.</P><H2><SPAN class="mw-headline" id="Conclusion">Summary</SPAN></H2><P>iLOV presents a viable option for use in <I>E. coli</I>, as its fluorescence is much greater than that of <I>E. coli</I>. Of our two promoters, 004 appears to have no specific response to TNT, while 007 did appear to have a relative increase in expression as TNT concentration increased. This suggests to us that with some modification it could form the basis of biosensor for TNT, or some part of a regulatory system.</P><H2><SPAN class="mw-headline" id="Materials">Materials</SPAN></H2><DL><DT>TECAM 200 PRO microplate reader</DT><DT>Grenier 96 well black plates</DT><DT>Top10 <I>E. coli</I></DT><DD>We used One Shot® TOP10 Chemically Competent <I>E. coli</I> from Invitrogen as a chassis for our constructs.</DD><DT>Top10 (+BBa_K1398004) <I>E. coli</I></DT><DT>Top10 (+BBa_K1398007) <I>E. coli</I></DT><DT>Top10 (+Constitutive iLOV) <I>E. coli</I></DT><DT>Top10 (+Constitutive GFP) <I>E. coli</I></DT><DT>1000 ug ml<SUP>-1</SUP> Trinitrotoluene, or 4.4uM</DT><DD>Supplied by AccuStandard. Dissolved in MeOH:AcCN.</DD><DT><A href="https://2014.igem.org/Team:Exeter/Protocols#7">LB Media</A></DT><DD>Used to grow overnight cultures of <I>E. coli</I>.</DD><DT><A href="https://2014.igem.org/Team:Exeter/Protocols#8">MYE Media</A></DT><DD>MYE is a modified minimal media used to grow bacteria in the 96-well plates. MYE was used here as LB has a natural fluorescence that would interfere with our readings. It was created by Howard et al. (2013) and was modified from Schirmer et al. (2010).
</DD></DL><H2>Method</H2><P>These experiments were carried out on 96 well plates, using a TECAM 200 PRO microplate reader. The Top10 strains were grown on LB media overnight and scanned while on MYE media. Each strain was grown overnight in 10 ml of LB media in a shaking incubator at 37 oC. To create the culture for each well, 200ul of MYE media was mixed with 3 ul of the required strain, as well as a volume of TNT or NG specific to each well. When MYE of LB media was used as a control 200 ul was used.</P><P>The wells were scanned in the TECAM machine, a process which took around 5 minutes. The cultures were kept at 37oC while this occurred. The plates were then transferred to a shaking incubator (800 rpm), usually for 55 minutes. In cases where plates where run overnight they were left in the TECAM machine and the shaking function was used.</P><P>Each plate had a different arrangement of cell cultures. The layout of the plate and location of the cultures used in each experiment is listed below.</P><P><B>Experiment One</B></P><P>This test was carried out on Plate 1. The only recorded use of iLOV within iGEM is by the Glasgow 2011 Team, and it’s functionality within <I>E. coli</I> was not shown. As we are using it as a reporter protein for our promoters we wanted to make sure that it definitely functioned within the organism.</P><P><I>E. coli</I> was grown in MYE media for 27 hours while its fluorescence and optical density was measured. We carried out this test with WT-Top10, iLOV-Top10, GFP-Top10 and pure MYE media.</P><P><B>Experiment Two</B></P><P>This test was carried out on Plate 2. <I>E. coli</I> was grown for 32 hours with the addition of a set, non-lethal volume of TNT (0-10 ul, with steps of 2.5 ul) at t =0. Over this time the optical density and the fluorescence of the cultures were measured. We did this test to examine how our promoters responded to TNT.</P><P><B>Plate 1</B>:</P><TABLE><TBODY><TR><TD>A1-12    </TD><TD>Top10</TD></TR><TR><TD>B1-12</TD><TD>iLOV</TD></TR><TR><TD>C1-12</TD><TD>GFP</TD></TR><TR><TD>D1-12</TD><TD>MYE Media</TD></TR></TBODY></TABLE><P><B>Plate 2</B>:</P><TABLE><TBODY><TR><TD>A7-9</TD><TD>004, with 10ul TNT</TD></TR><TR><TD>A10-12</TD><TD>007, with 10ul TNT</TD></TR><TR><TD>B7-9</TD><TD>004, with 7.5ul TNT</TD></TR><TR><TD>B10-12</TD><TD>007, with 7.5ul TNT</TD></TR><TR><TD>C7-9</TD><TD>004, with 5ul TNT</TD></TR><TR><TD>C10-12</TD><TD>007, with 5ul TNT</TD></TR><TR><TD>D7-9</TD><TD>004, with 2.5ul TNT</TD></TR><TR><TD>D10-12</TD><TD>007, with 5ul TNT</TD></TR><TR><TD>E10-12</TD><TD>Constitutive iLOV, with 0ul TNT</TD></TR><TR><TD>F10-12</TD><TD>Top10, with 0ul TNT</TD></TR><TR><TD>G7-9</TD><TD>004, with 0ul TNT</TD></TR><TR><TD>G10-12</TD><TD>LB, with 0ul TNT</TD></TR><TR><TD>H7-9</TD><TD>007, with 0ul TNT</TD></TR><TR><TD>H10-12    </TD><TD>MYE, with 0ul TNT</TD></TR></TBODY></TABLE>

----
<H1><SPAN class="mw-headline" id="References"> References:</SPAN></H1><OL><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/9013822/">http://www.ncbi.nlm.nih.gov/pubmed/9013822/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/figure/f2/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/figure/f2/</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519536/</A></LI><LI><A href="http://www.jbc.org/content/288/19/13789/F3.expansion.html">http://www.jbc.org/content/288/19/13789/F3.expansion.html</A></LI><LI><A href="http://www.jbc.org/content/288/19/13789.full">http://www.jbc.org/content/288/19/13789.full</A></LI></OL><H2> Navigation </H2><P><A href="https://2014.igem.org/Team:Exeter/EColiStressTesting">Previous: <I>E. coli.</I> Xenobiotic Tolerance </A></P><P><A href="https://2014.igem.org/Team:Exeter/iLOVCharacterisation">Next: iLOV Characterisation  </A></P></DIV><P><TITLE>Exeter | ERASE</TITLE></P><DIV id="footer"><A id="page_top" href="#">Top</A><P><A href="https://twitter.com/ExeteriGEM2014" class="twitter-follow-button" data-show-count="false">Follow @ExeteriGEM2014</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Exeter/Detection">http://2014.igem.org/Team:Exeter/Detection</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Exeter/Detection" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Exeter/Detection" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Exeter/Detection&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Exeter/Detection&amp;oldid=394016" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>