"
Page
Discussion
View source
History
teams
Log in
 
Team:Brown/Notebook Meetings/7-6-09
From 2009.igem.org
Home
Team
About Us
Brown University
Project
Abstract
The Allergic Response
Histamine Sensor
Histamine Binding Protein
S.epidermidis
System Schematic
Human Practices
Parts
Notebook
Weekly Logs
Protocols
Recipes
Team Meetings
Links
Acknowledgements
Sponsors
iGEM Meeting Minutes- 7/6/09
9:05 am
Update from Team 1
Failed digests and PCR- could it be poor technique, or insert not in pBluescript?
Drop off fedex at Biomed fr¬ont desk for sequencing; get package out to Yale right away
Hope to isolate EV131 insert at least by the end of the week, if not earlier
Suggestions:
Run lower percentage gel to get resolution between 3000 and 3500, usually run 1% gel
Run a couple of individual digests with individual enzymes
PCR
Amplify any other template alongside as a control
Update from Team 2
Helped team 3 with project a bit
Looked at quorum sequences
Two genes with a DNA sequence in between both; both genes have been annotated, but not the promoters that lie in the region in between
Team 2 wants to amplify the region from one of the promoters, including genes downstream
Entire gene strand is approx. 3 kb
Once amplified, they want to ligate death gene in front
Plan on using electroporation protocol
Suggestion: be careful with ribosome binding site, make sure that you do have a good ribosome binding site
Amplify everything up to the ATG codon at 5’ end
Make biobrick ends before P1, after P2, and individual ones in between genes downstream of promoter
Adrian question: do you foresee any problems of the quorum promoter being on constitutively? Is there any negative feedback? If there’s 50-100 copies, then the constitutive low level becomes a high level of conscription
A Plan B?: secretion tag DNA stretch that signals secretion through the sec pathway, 30 amino acids on 5’ end
Update from Team 3
Paper on computational design of histadine receptors
Will contacted author of paper, who has done lots of research on altering cell receptors
Response: wants a formal project proposal before he will actually help the team; which proteins we’re working with, what we want to do
We were looking at TrigEZ protein→ will active the promotion of whatever genes we will put in, it binds to sugar binding protein which binds to sugars, followed by a phosporylation cascade
Modeled with ribose, but ribose is too different from histamine→ author suggested to look for amino acid receptor
Found “Tar” receptor→ aspartate binding
If it works, then we’ll perform directed mutagenesis
Hope to combine Tar and ENVZ
Transmembrane domain is with the Tar
Two component system
Hope to change it to a histamine binding site
Do fusion and mutagenesis in parallel?
Do not do mutagenesis on fused proteins; do it only on the binding domain
Wessel: You’ll need to design a selection method so that cells will die unless in presence of aspartate; way to test aspartate binding
1. System for screening positive binding: Fuse Tar and ENVZ, design some system that enables cell survival; could easily be lactamase or some ampicillin resistance. Looking for GFP is fair, but then you have to screen through visually (could be a second choice to above).
2. Get screening to work; take aspartate binding module Tar and mutagise. Re-insert into plasmid that contains signaling gene. Screen- survival of cells.
3. Insert downstream elements that allow the histamine to activate.
Binding interaction sites lie between 1-6 and 38-188; need to sit down and look at sequence and identify exactly which sites
How to make sure that you can mutate DNA?
Author suggested using Histadine protein, and then worrying about signaling later; seems like a good strategy
Chose this family of proteins in particular because they are highly conserved and stable, assuming that mutated variants will also be stable
Retrieved from " http://2009.igem.org/Team:Brown/Notebook_Meetings/7-6-09 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
