"
Page
Discussion
View source
History
teams
Log in
 
Team:Brown/Project HBP
From 2009.igem.org
Home
Team
About Us
Brown University
Project
Abstract
The Allergic Response
Histamine Sensor
Histamine Binding Protein
S.epidermidis
System Schematic
Human Practices
Parts
Notebook
Weekly Logs
Protocols
Recipes
Team Meetings
Links
Acknowledgements
Sponsors
EV131 computer-modeled structure (Source: ncbi.nlm.nih.gov)
rEV131 effectively sequesters histamine without disrupting histamine receptors, hence this treatment may prove to be a viable alternative to current antihistamines in combating allergic symptoms without causing negative side effects. rEV131 is a high affinity histamine binding protein secreted by the tick Rhipicephalus appendiculatus. This protein allows the tick to overcome the host’s inflammatory response by sequestering histamine at the site of feeding, outcompeting the host's histamine receptors for the ligand, thereby effectively combating the allergic response. rEV131 has the highest affinity (lowest dissociation constant) among all histamine binding proteins secreted by R. appendiculatus (Paesen et al, 1999).
In our genetic construct, rEV131 will be placed under the OmpC promoter so that histamine molecule binding to our engineered receptor triggers the transcription of rEV131.
Though the protein has been purified, cloned, expressed and well characterized (Paesen et al, 1999) (Couillin et al, 2004) our future plans include further characterization of the protein by conducting binding assays in which we will vary levels of free histamine, above and below allergic threshold, to mimic the extracellular fluid in the nasal cavity. We will also demonstrate protein-ligand specificity by exposing the protein to molecules similar to histamine, such as histidine and imidazole. Further research goals include investigating modifications to the EV131 protein that could enhance histamine-binding affinity, including modifying or introducing novel binding pockets and honing protein residues
Cloning the EV131 Sequence into an Expression Plasmid, Transformation of E. coli, and Cell Selection
Cloning of EV131 into pNOTAT expression plasmid
Gel visualization of pGEMT-EV131 digest and undigested, digested pNO-TAT controls
The insert of interest coding for EV131 is 572 base pairs (bp) in length. After synthesizing primers with the restriction sites EcoRI and BamHI engineered onto the ends, we performed a polymerase chain reaction (PCR) in order to amplify the insert from the pBluescript SK II plasmid in which it was sent. In order to express the recombinant protein, the coding DNA sequence was cloned into the pGEM easy-T expression vector for more efficient excision before cloning into the pNOTAT expression vector.
The pNOTAT expression vector is important because it adds a 6-His tag to the protein which allows for selective detection (SDS page) and purification (nickel column chromatography) of our particular protein EV131. Once the protein’s coding sequence was cloned into a pNOTAT expression vector, the vector was introduced into competent BL21 E. coli cells by way of heat shock transformation. The BL21 competent cell strain allows for high-efficiency protein expression of any gene that is under the control of a T7 promoter and has a ribosome binding site. BL21 cells contain the T7 bacteriophage gene I, encoding T7 RNA polymerase under the control of the lac UV5 promoter, which is induced by isopropyl-β-D-thiogalactoside (IPTG). After transformation, an aliquot of the transformed cells was spread onto an agar plate to allow for isolation of E. coli colonies containing the expression plasmid ligated to EV131. pNOTAT expression vectors carry a gene for ampicillin resistance, which allows E. coli containing the expression plasmid to be selected for on agar plates containing ampicillin.
Expression of rEV131 Protein in BL21 E.coli
Efficient and controlled expression of protein in E. coli cells is regulated by the presence of the lac repressor protein. During normal cell growth, this protein binds to the operator sequences in the plasmid and prevents recombinant protein expression. Expression of recombinant proteins encoded by the pNOTAT expression vector is induced by the addition of IPTG, which binds to the lac repressor protein and inactivates it. Once the lac repressor is inactivated, the host cell’s RNA polymerase can transcribe the sequences downstream from the promoter. The transcripts produced are then translated into the recombinant protein. Inducing protein expression with IPTG means that the cellular metabolism concentrates almost exclusively on the production of recombinant protein under tight control. 5-mL liquid cultures of cell colonies isolated from ampicillin plates were grown up and then induced with IPTG. The amount of time allowed for protein expression was varied in a range from two to five hours for different cultures. Thereafter, we began sample preparations for SDS-PAGE and Nickel Column Chromatography.
Protein Purification Under Native or Denaturing Conditions
Protein purification at this stage of our project is still in process. In order to retain the biological activity of the protein, we wish to purify 6-His-tagged proteins under native conditions. For this, the 6-His-tagged protein must be soluble. In this case, however, there is greater potential for nonspecific proteins to interact with the Nickel-NTA resin. Also, since there is a chance of the 6-His tag being hidden by the tertiary structure of the native protein, the soluble proteins require denaturation before they can be purified on the Ni- NTA column. As a control, a parallel purification under denaturing conditions will be carried out. If purification is only possible under denaturing conditions, the tag can be made generally accessible by moving it to the opposite terminus of the protein. The QIAgen QIAexpress® Ni-NTA Fast Start Kit is an appropriate kit for the purification and detection of recombinant 6-His-tagged proteins in high yield.
Histamine and Competing Molecule Binding Assays
Test 1: Testing for Histamine Binding Affinity
As a model of the free histamine-EV131 interaction, we will be testing the binding affinity strength of a solution of EV131 being poured into a column that contains histamine beads. The threshold level for eliciting a significant allergic response is 12 ng/mL (Proud et al, 1992). The binding capacity of EV131 is two molecules of histamine for one molecule of protein (Paesen et al, 1999) (Couillin et al, 2004). We will test the amount of protein necessary to sequester the threshold level of histamine by running protein solutions of varying concentrations along the histamine bead column. We will measure the initial amount of protein added to the column and then measure the remaining solution eluted from the column. This will determine the amount of protein needed to sequester histamine at threshold levels.
Another component of the binding affinity test includes the running of free histamine along the histamine-protein-bound column. We will be able to determine the rate of dissociation of the EV131 based upon the interaction of which histamine the EV131 will competitively bind to: the stationary histamine column or the poured free histamine. We will observe the time the protein takes to dissociate from the column and confirm the dissociation constant for the protein as described in the literature.
Test 2: Testing EV131 with Histamine Competitor Molecules
After having investigated the histamine-EV131 interaction, we will test the binding affinity that molecules similar in structure to histamine will have with EV131. Such molecules include histadine, imidazole, and aspartate. The procedure will be similar to that of testing free histamine versus stationary histamine binding with the recombinant EV131 protein. We will pour solutions of free histadine, imidazole, or aspartate into a stationary histamine bead column with EV131 protein bound to it. Thereafter, we will observe how much histamine, imidazole, or aspartate elutes with the protein bound to it. Ideally, the results of this experiment will suggest that EV131 has a much higher binding affinity for histamine than do other molecules of a similar structure.
Insights and Further Improvements Upon EV131 Recombinant Protein
EV131 is a novel protein because it has a high affinity for histamine and it outcompetes histamine receptors located throughout the body for histamine. For these reasons, EV131 has tremendous potential as a therapeutic drug for allergies and other inflammatory diseases. We hope to improve the existing functions of EV131 and to broaden the scope of protein functions by modifying the protein. We could potentially improve the binding affinity of the protein for histamine by increasing the number of binding pockets available, which would optimize the protein to histamine ratio. We could also look at the possibility of constructing a fusion protein that includes cell receptor signals, such as ApoE, that may signal faster uptake and degradation of saturated histamine binding proteins or histamine sensors. This can be accomplished by structurally changing the protein, which will involve single amino acid modifications via site directed mutagenesis. Forthcoming research into these methods will help determine their feasibility and practicality.
References
Paesen, G.C., Adams, P.L., Harlos, K., Nuttall, P.A., and Stuart, D.I. "Tick Histamine-Binding Proteins: Isolation,
Cloning, and Three-Dimensional Structure," Molecular Cell, 1999 , 3, 661-671.
Couillin, I., Maillet, I., Vargaftig, B.B., Jacobs, M., Paesen, G.C., Nuttall, P.A., Lefort, J., Moser, R., Weston-Davies, W., and Bernhard, R. "Arthropod-Derived Histamine-Binding Protein Prevents Murine Allergic Asthma," Journal of Immunology, 2004, 173 , 3281-3286.
Proud, D., Bailey, G.S., Naclerio, R.M., Reynolds, C.J., Cruz A.A., Eggleston, P.A., Lichtenstein, L.M., and Togias, A.G. "Tryptase and histamine as markers to evaluate mast cell activation during the responses to nasal challenge with allergen, cold, dry air, and hyperosmolar solutions," Journal of Allergy and Clinical Immunology, 1992 , 89 , 1098-1110.
Retrieved from " http://2009.igem.org/Team:Brown/Project_HBP "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
