<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Brown_Project_Histamine_Sensor"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Brown/Project_Histamine_Sensor">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Brown/Project_Histamine_Sensor&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown/Project_Histamine_Sensor&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown/Project_Histamine_Sensor&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Brown/Project_Histamine_Sensor" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Brown/Project Histamine Sensor</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu"><UL><LI id="mainHome"><A href="https://2009.igem.org/Team:Brown">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Brown/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Team">About Us</A></LI><LI><A href="http://www.brown.edu">Brown University</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Brown/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Introduction">The Allergic Response</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Histamine_Sensor">Histamine Sensor</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_HBP">Histamine Binding Protein</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_S. epidermidis">S.epidermidis</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_All_Together">System Schematic</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Implications">Human Practices</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Brown/Parts">Parts</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Brown/Notebook_Weekly_Logs">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Weekly_Logs">Weekly Logs </A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Protocols">Protocols</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Recipes">Recipes</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_meetings">Team Meetings</A></LI><LI id="mainLinks"><A class="aLinks" href="https://2009.igem.org/Team:Brown/Links_Acknowledgements">Links</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Links_Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Links_Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><P>During the allergic response, the concentration of histamine in the extracellular fluid of the nasal cavity increases. To initiate a response; therefore, a histamine sensor is necessary.
</P><P>Because natural histamine receptors exist only in eukaryotic cells as G-coupled protein receptors, they are unusable for our prokaryotic system. Therefore we have set out to engineer our own receptor. This novel receptor will sense extracellular concentrations of histamine and initiate an intracellular cascade, signaling cells to respond appropriately to the increase in histamine concentration. 
</P><P>To engineer a novel histamine receptor, we are mutating two existing prokaryotic chemoreceptors so that they bind histamine rather than their wild type ligands.
</P><H1><SPAN class="mw-headline" id="Re-engineering_Chemoreceptor_.231:_Ribose_Binding_Protein"> Re-engineering Chemoreceptor #1: Ribose Binding Protein </SPAN></H1>
We modified ribose binding protein (RBP), which normally binds ribose in the periplasmic space of Escherichia coli, to bind histamine. Our computational approach to accomplish this task was modeled after that taken in Looger, et al., &quot;Computational design of receptor and  sensor proteins with novel functions&quot;, Nature (2003). Using Rosetta macromolecular modeling software, we modified the program's existing Enzyme Design Function (such as that used in Röthlisberger, et al &quot;Kemp elimination catalysts by computational enzyme design&quot;, Nature (2008)) to enable the re-design of RBP, a non-enzymatic protein. The successful modification of RBP would result in its ability to bind histamine. <DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The native E. coli ribose-binding protein</DIV></DIV></DIV><P><B><BIG>Protein design:</BIG></B></P><P>1) Took the PDB file for the crystal structure of RBP cocrystallized with ribose (2DRI). Removed water molecules and added missing hydrogens. 
</P><P>2) Used UCSF Chimera to geometrically search for all van der Waals interactions between ribose and RBP in the crystal structure. Identified the amino acids responsible for these interactions as those most likely present in the ligand binding pocket of RBP. 
</P>
3) Used UCSF Chimera to mutate all the identified residues in RBP to alanine (which has neutral chemical properties, and almost no side chain), thereby effectively creating a &quot;blank&quot; version of RBP that has no specific binding pocket for any ligand (removing its binding affinity for ribose)--called the polyala RBP. <DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">The polyala RBP.</DIV></DIV></DIV><P>4) Replace ribose in the alanine-mutated RBP (polyala RBP) PDB file with a 3D structure of histamine in a low energy conformation. 
</P><P>5) Used Rosetta's Ligand Docking mode on 100 of Brown's Center for Computational Molecular Biology (<A href="http://www.brown.edu/Research/CCMB/" class="external free" rel="nofollow">http://www.brown.edu/Research/CCMB/</A>) clustered servers to simulate the docking of histamine into the polyala RBP. To dock histamine into polyala RBP, Rosetta Ligand uses Monte Carlo minimization to find the relative orientations of the ligand and receptor that minimize steric contacts between the two while still keeping histamine roughly within the original ligand binding pocket. This produces 10000 PDB files of histamine docked to the polyala RBP.
</P><P>6) Next, we sorted the 10000 docked PDB files by their interface energies between ligand and protein. We selected the top 2500. 
</P><P>7) Next, we took the top 2500 and input them into the Rosetta Enzyme Design mode, run on the same cluster. We specified the residues mutated to alanine as those to re-design by mutation. The redesign process attempts to stabilize the protein-ligand pair by mutating residues at certain specified locations, those we originally mutated to alanine. This redesign process used Rosetta Design’s probabilistic simulated annealing algorithm to find the particular residues that minimize total energy between protein and ligand (minimized energy indicates a stable state, favoring binding). Because this process is probabilistic, final designs are not guaranteed to yield the lowest possible energy conformations. However, by doing thousands of designs in parallel, we increased the likelihood of isolating mutations that result in histamine binding. 
</P><P>8) Finally, we sorted through the output designs using several criteria: predicted interface energies between the protein and histamine, amount of hydrogen bonding between the protein and histamine (H-bonds are very good for ligand binding), and predicted folding ability of protein. We filtered the final designs to only those with a total score better than or equal to that of the scaffold RBP. We then set cutoffs for several of the relevant criteria, found the sets of the best designs in each relevant category, and found the designs that were in the intersection of those sets. The final design (in bold) selected had the best overall scores in each category. We had the DNA for the top design synthesized by GeneArt AG. We are in the process of testing the protein’s ability to bind histamine and its stability/folding ability.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:712px;"><DIV class="thumbcaption">The final selection list of mRBP.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:729px;"><DIV class="thumbcaption">The score of the native RBP</DIV></DIV></DIV></DIV><TABLE><TBODY><TR><TH>total_score
</TH><TH>tot_pstat_pm
</TH><TH>tot_nlpstat_pm
</TH><TH>tot_burunsat_pm
</TH><TH>tot_hbond
</TH><TH>SR_1_interf_E_1_2
</TH><TH>SR_1_dasa_1_2
</TH></TR><TR><TD>Weighted sum of the other terms. (Lower better)
</TD><TD>How well packed the protein is. (0-1)
</TD><TD>How well packed the protein is without the ligand. (0-1)
</TD><TD>Number buried unsatisfied polars. (Lower better)
</TD><TD>Number of H-bonds. (Higher better)
</TD><TD>How well the ligand binds to protein. (Lower better)
</TD><TD>How exposed the ligand is. (0-1)
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">The mutated Ribose Binding Protein (mRBP)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Histamine in the mRBP ligand binding pocket</DIV></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The TrgEnvZ-OmpR pathway. From Fig. 3 of Looger, et al., &quot;Computational design of receptor and  sensor proteins with novel functions&quot;, Nature (2003).</DIV></DIV></DIV><P>In normal E. coli cells, ribose binding to RBP forms a ribose-RBP complex that interacts with the periplasmic domain of Trg, which in turn activates an intracellular cascade that induces chemotaxis. 
To link activation of Trg to induction of gene expression, we created a fusion between Trg periplasmic receptor domain and the EnvZ intracellular kinase domain by stitching the two intracellular domains at a shared NdeI restriction site. (as done in Baumgartner JW, et al. “Transmembrane signalling by a hybrid protein: communication from the domain of chemoreceptor Trg that recognizes sugar-binding proteins to the kinase/phosphatase domain of osmosensor EnvZ” (1994)). EnvZ phosphorylates the transcription factor OmpR, which causes transcription of DNA regulated by OmpC promoter. 
</P><H1><SPAN class="mw-headline" id="Re-engineering_Chemoreceptor_.232:_Tar_Receptor"> Re-engineering Chemoreceptor #2: Tar Receptor </SPAN></H1><P>In parallel, we modified the receptor Tar, which normally binds aspartate in the periplasmic space of E. coli, to bind histamine. 
</P><P>Dr. Loren Looger at the Howard Hughes Medical Institute's Janelia Farm campus used his protein design software Chameleon to calculate mutations that would transform Tar’s aspartate binding pocket to a histamine binding pocket. His algorithm gave us the top 16 receptor designs and we are currently in the process of creating this library of mutants. We have designed primers for each of these designs and are introducing these mutations by both the “Round-the-Horn Site-Directed Mutagenesis” protocol on OpenWetWare and Strategene’s Quikchange Mutagenesis II Kit. 
</P><P>In normal E. coli cells, aspartate binds to the Tar receptor. The Tar-EnvZ chimera protein (created by Utsumi et. al in &quot;Activation of bacterial porin gene expression by a chimeric signal transducer in response to aspartate&quot; 1989) allowed the Taz protein to be linked to the EnvZ cascade in the same manner as Trg-EnvZ. Just as in that system, ligand binding to its receptor leads to an intracellular signaling cascade promoting gene transcription.
</P><H1><SPAN class="mw-headline" id="Initiating_an_Appropriate_Response:_Linking_Histamine_Sensation_to_Intracellular_Transcription"> Initiating an Appropriate Response: Linking Histamine Sensation to Intracellular Transcription </SPAN></H1><P>The creation of a novel histamine receptor to detect elevated histamine is only the first step to providing an appropriate cell response to allergens. Once a functional histamine sensor is inserted into the cell membrane, its activation must be linked to an intracellular cascade responsible for triggering <A href="https://2009.igem.org/Team:Brown/Project_HBP" class="external text" rel="nofollow">gene transcription</A>. The successful mutation of the binding pockets on both RBP and Tar was therefore followed by the complete construction of each receptor as it would function in the cell membrane. For both RBP and Tar, this meant replicating the work done by Masayori Inouye et. al to create the chimeric proteins Trg-EnvZ (Trz) and Tar-EnvZ (Taz) (respectively). Chimeric proteins were created by fusing the intracellular domains of Tar and Trg (Trg is the membrane receptor associated with ligand-bound RBP) to the kinase domain of EnvZ. Activation of the functional receptor domains by histamine binding is thus able to initiate an intracellular cascade that phosphorylates the transcription factor ompR thereby activating gene transcription under the ompC promoter. By replacing the gene normally present under this promoter with our gene of interest, we successfully manipulated the cascade to produce an appropriate cellular response when allergens are present.
</P><P><B><BIG>Assay to Test Signaling Cascade Functionality</BIG></B></P><P>Our assay to test these receptors’ affinities for histamine is based on fluorescence. Because the intracellular cascades of both Trg-EnvZ and Tar-EnvZ induce transcription under the OmpC promoter, we have constructed a cassette that places the OmpC promoter over the RFP gene for red fluorescence. Testing for functionality of the cascade then simply involved observing whether RFP expression occurred under ompC, a simple fluorescence assay conducted on an epifluorescence microscope. Qualitative visualization of red-fluorescing colonies transformed with both the receptor and cascade components indicated a functional intracellular signaling system. 
</P><P>We have tested this signaling cascade by performing the following series of transformations. The E. coli strain RU1012 was used, because it is an EnvZ knockout strain. As efforts to construct a novel histamine receptor were being conducted in parallel to these assays, testing of the cascade was conducted with the original chimeric chemoreceptor Tar-EnvZ. The binding of the wild-type ligand aspartate to Tar and the intracellular transcription of ompC-RFP it initiated was thus used as a proof-of-concept of our future histamine-initiated system. 
</P><P>1. RU1012 with no plasmid
</P><P>2. RU1012 with OmpC-RFP 
</P><P>3. RU1012 with Tar-EnvZ
</P><P>4. RU1012 with Trg-EnvZ
</P><P>5. RU1012 with OmpC-RFP + Tar-EnvZ
</P><P>6. RU1012 with OmpC-RFP + Trg-EnvZ
</P><P><B>Results: Testing the Cascade</B></P><P>Our fluorescence assays of these transformed colonies indicate that signal transduction is indeed effective. 
</P><P>All photographs were taken on an epi-fluorescent microscope: Olympus SZX16, excitation source X-cite Series 120. The first image in each series is under bright light, the second under a fluorescent filter for RFP.  
</P><P>1. RU1012 with no plasmid
</P><P>
2. RU1012 with OmpC-RFP 
</P><P>
3. RU1012 with Tar-EnvZ
</P><P>
4. RU1012 with Trg-EnvZ
</P><P>
5. RU1012 with OmpC-RFP + Tar-EnvZ
</P><P>
6. RU1012 with OmpC-RFP + Trg-EnvZ
</P><P><I><BIG>References</BIG></I></P><UL><LI> Baumgartner, James W., Changhoon Kim, Renée E. Brisette, Masoyuri Inouye, Changkyu Park, and Gerald L. Hazelbauer. &quot;Transmembrane signalling by a hybrid protein: communication from the domain of chemoreceptor Trg that recognizes sugar-binding proteins to the kinase/phosphatase domain of osmosensor EnvZ.&quot; Journal of Bacteriology 176: 1157-163. Journal of Bacteriology. Web. 25 July 2009. &lt;<A href="http://www.ncbi.nlm.nih.gov:80/pmc/articles/PMC205168/" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov:80/pmc/articles/PMC205168/</A>&gt;.
</LI></UL><UL><LI> Looger, Loren L., Mary A. Dwyer, James J. Smith, and Homme H. Hellinga. &quot;Computational design of receptor and sensor proteins with novel functions.&quot; Nature 423 (2003): 185-90. PubMed. Web. 15 July 2009. &lt;<A href="http://www.ncbi.nlm.nih.gov/pubmed/12736688" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/12736688</A>&gt;.
</LI></UL><UL><LI> Meiler, Jens, and David Baker. &quot;ROSETTALIGAND: protein-small molecule docking with full side-chain flexibility.&quot; Protein 65.3 (2006): 538-48. PubMed. Web. 12 Aug. 2009. &lt;<A href="http://www.ncbi.nlm.nih.gov/pubmed/16972285" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/16972285</A>&gt;.
</LI></UL><UL><LI> Rhiju, Das, and David Baker. &quot;Macromolecular modeling with rosetta.&quot; Annual Review of Biochemistry 77: 368-82. PubMed. Web. 12 July 2009. &lt;<A href="http://www.ncbi.nlm.nih.gov/pubmed/18410248" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/18410248</A>&gt;.
</LI></UL><UL><LI> Rothlisberger, Daniela, Olga Kersonsky, Andrew M. Wollacott, Lin Jiang, Jason Dechancie, Jamie Betger, Jasmine L. Gallaher, Eric A. Althoff, Alexandre Zanghellini, Orly Dym, Shira Albeck, Kendall N. Houk, Dan S. Tawfik, and David Baker. &quot;Kemp elimination catalysts by computational enzyme design.&quot; Nature 453 (2008): 190-95. Nature. 19 Mar. 2008. Web. 5 July 2009. &lt;<A href="http://www.nature.com/nature/journal/v453/n7192/abs/nature06879.html" class="external free" rel="nofollow">http://www.nature.com/nature/journal/v453/n7192/abs/nature06879.html</A>&gt;.
</LI></UL><UL><LI> Utsumi, R., R. E. Brisette, A. Rampersaud, S. A. Forst, K. Oosawa, and M. Inouye. &quot;Activation of bacterial porin gene expression by a chimeric signal transducer in response to aspartate.&quot; Science 245.4923 (1989): 1246-249. Science. Web. 27 June 2009. &lt;<A href="http://www.sciencemag.org/cgi/content/abstract/245/4923/1246" class="external free" rel="nofollow">http://www.sciencemag.org/cgi/content/abstract/245/4923/1246</A>&gt;.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Brown/Project_Histamine_Sensor">http://2009.igem.org/Team:Brown/Project_Histamine_Sensor</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Brown/Project_Histamine_Sensor" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Brown/Project_Histamine_Sensor" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Brown/Project_Histamine_Sensor&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Brown/Project_Histamine_Sensor&amp;oldid=168921" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>