"
Page
Discussion
View source
History
teams
Log in
 
Team:Brown/Notebook Meetings/6-29-09
From 2009.igem.org
Home
Team
About Us
Brown University
Project
Abstract
The Allergic Response
Histamine Sensor
Histamine Binding Protein
S.epidermidis
System Schematic
Human Practices
Parts
Notebook
Weekly Logs
Protocols
Recipes
Team Meetings
Links
Acknowledgements
Sponsors
Home
Team
About Us
Brown University
Project
Abstract
The Allergic Response
Histamine Sensor
Histamine Binding Protein
S.epidermidis
System Schematic
Human Practices
Parts
Notebook
Weekly Logs
Protocols
Recipes
Team Meetings
Links
Acknowledgements
Sponsors
iGEM Advising Meeting
With Gary, Diana, Adrian
SFH 218
June 29, 2009
9:02: Team 3 explains the situation. We are keeping the toggle switch as a backup but we will also be trying to make a histamine receptor. We explain the Hellinga paper.
9:12: Gary likes the idea because it opens a new area for the team, adds more to the project, and our scientific approach is good. Do we have a plan B? We need to plan another backup.
9:17: Adrian’s concerns: How are we going to put the sequence from the computer into the cell? How are we going to assay the binding affinity of these mutants?
9:19: Ashley expresses our concern that we have no idea how to start. Gary says we can get the ribose sensor and associate that with the reporting pathway. For the computing part, ask experts. We can ask Washington State to send us the chimeric protein.
9:24: Diana suggests mutagenesis v. synthesizing. But lets just wait to see what the computer spits out. She is concerned downstream as to how we are going to test this. Mutagenesis technology? Gary says its just a small hurdle.
9:28: Team 2 reports. We got our S. epidermidis. Hopefully primers will arrive today to PCR the quorum sensing genes.
9:29: How do you find the promoter sequences on NCBI? Gary suggests looking at the original paper for a sequence that it minimally required for the promoter. “It may just be the nature of the beast” – Gary. Adrian has an answer: Ecocyc.org is a giant database for promoters.
9:31: Team 1 update. They’ve been working on getting EV131 out of pBlueScript. But when they ran the gels they didn’t see 2 bands. On Friday they ran a bunch of controls, will be running gels today.
9:36: Gary suggests to test the viability of the frozen cells. Also, we should test the primers to make sure the DNA sequence is in there. Also issues with the ladder….? The band might be there but not as bright because it’s less DNA. Or we could cut something to make the desired piece of DNA bigger. The 2 restriction enzymes are compatible.
9:42: Plan for the week… Let’s say the digest works. Extract the band, put it into pGEM. Start PCR. pGEM is only for PCRing. Then if that works successfully place into pNoTat expression vector. Still looking into binding assays for EV131.
9:45: Gary: Look at the reading frame! Look at the pNoTat sequence – look at where restriction site is relative to EV131 codon.
Retrieved from " http://2009.igem.org/Team:Brown/Notebook_Meetings/6-29-09 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
