<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Brown_Notebook_Protocols_PCR"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Brown/Notebook_Protocols/PCR">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Brown/Notebook_Protocols/PCR&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown/Notebook_Protocols/PCR&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown/Notebook_Protocols/PCR&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Brown/Notebook_Protocols/PCR" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Brown/Notebook Protocols/PCR</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu"><UL><LI id="mainHome"><A href="https://2009.igem.org/Team:Brown">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Brown/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Team">About Us</A></LI><LI><A href="http://www.brown.edu">Brown University</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Brown/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Introduction">The Allergic Response</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Histamine_Sensor">Histamine Sensor</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_HBP">Histamine Binding Protein</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_S. epidermidis">S.epidermidis</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_All_Together">System Schematic</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Implications">Human Practices</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Brown/Parts">Parts</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Brown/Notebook_Weekly_Logs">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Weekly_Logs">Weekly Logs </A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Protocols">Protocols</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Recipes">Recipes</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_meetings">Team Meetings</A></LI><LI id="mainLinks"><A class="aLinks" href="https://2009.igem.org/Team:Brown/Links_Acknowledgements">Links</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Links_Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Links_Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><P><B><BIG>Protocol: Polymerase Chain Reaction</BIG></B></P><P><B>Background/Purpose</B></P><P>Polymerase Chain Reaction (PCR) is used to amplify DNA segments by way of template strands, primers, and DNA polymerase. PCR is initiated by a short sequence of primers (one forward and one reverse). Primers are short, complementary sequences of roughly 18-28 nucleotides that anneal to the template strands to begin the synthesis of the new DNA strands. Following the primers annealing to the separate template strands respectively, the DNA polymerase begins to add complementary nucleotides to the template strands thereafter to complete the polymerase chain reaction. 
</P><P>DNA is synthesized in the 5’→3’ direction. Since DNA is antiparallel, the DNA polymerases add nucleotides in opposite direction.  That is, each primer anneals at opposite ends of the desired DNA of interest, prompting the DNA polymerases to build towards each other. The region in between the two primers is replicated. See Figure 1 for the PCR Mechanism.
</P><P><B>Methods</B></P><P>Primer Design
</P><P>1.	Identify gene of interest
</P><P>2.	Go to NCBI and navigate to the “All Databases” drop-down menu in the upper left part of the webpage and select the “Protein” tab.
</P><P>3.	Now type in the text box “gene of interest organism”
Example: Search “Protein” for “Groucho sea urchin”
</P><P>4.	Click on “Go”
</P><P>5.	Click on one of the relevant results
</P><P>6.	Scroll down to “Origin.” Copy and paste this one-letter code amino acid sequence into a 
Word document. Format this by deleting the numbers dispersed in the overall sequence. Spaces in between the sequence will suffice.
Example: 
Pristionchus pacificus
</P><PRE>      mdkragsrgg ggggnpflda leklkddynh mqaqlssqra eldkmnaeke qlqrhymmyy
      emscglnmem qkqsevakrm tallqsmlqy aphdaqasti qamerakqis lqelqqltaa
      sqaqqmlgmt gpmaamgglg gmagalggpg glnmaaiaaa mgaglrppap pgggggddrp
      apsssrqsss rqrsgspagg ekkpkleted gdddeidvqn ddpagpaang ktggrdsvhs
      gisssgastp aaaaaknfaa qlgqqrlpla qldpatrmmm qgmmapngka pysyrvdstg
      nlaptmfppd altdpgvpks vkavhdlphg evvcavaisk daqrvftggk gcvkiwdlaa
      ntsaararle clednyirsc klfaegthlv vggeasnill fdietqkeva kldttaqacy
      alalnqeskl lyaccadgav vifdlasmqe varlpghtdg ascvdlsgdg lrlwtggldh
      tlrswdirer relsnidfas qifslgcspt edwvavgldt nqievvntap gvkeryqlhr
      hdscvlslrf ahsgkwfctt gkdnllnvwr spygalsvra sesssvlscd ishddsvivt
      gsgekkatvy qvqyesss
</PRE><P>7.	Identify another relevant organism with the same gene of interest and repeat the aforementioned process.
</P><P>8.	Alignment: Go to Clustal W. Paste amino acid sequences in the following format (the title “&gt;speciesname1” should not contain any spaces):
</P><P>&gt;speciesname1
</P><PRE>      mdkragsrgg ggggnpflda leklkddynh mqaqlssqra eldkmnaeke qlqrhymmyy
      emscglnmem qkqsevakrm tallqsmlqy aphdaqasti qamerakqis lqelqqltaa
      sqaqqmlgmt gpmaamgglg gmagalggpg glnmaaiaaa mgaglrppap pgggggddrp
      apsssrqsss rqrsgspagg ekkpkleted gdddeidvqn ddpagpaang ktggrdsvhs
      gisssgastp aaaaaknfaa qlgqqrlpla qldpatrmmm qgmmapngka pysyrvdstg
      nlaptmfppd altdpgvpks vkavhdlphg evvcavaisk daqrvftggk gcvkiwdlaa
      ntsaararle clednyirsc klfaegthlv vggeasnill fdietqkeva kldttaqacy
      alalnqeskl lyaccadgav vifdlasmqe varlpghtdg ascvdlsgdg lrlwtggldh
      tlrswdirer relsnidfas qifslgcspt edwvavgldt nqievvntap gvkeryqlhr
      hdscvlslrf ahsgkwfctt gkdnllnvwr spygalsvra sesssvlscd ishddsvivt
gsgekkatvy qvqyesss
</PRE><P>&gt;speciesname2
</P><PRE>      mypspvrhpa aggpppqgpi kftiadtler ikeefnflqa qyhsiklece klsnektemq
      rhyvmyyems yglnvemhkq teiakrlntl inqllpflqa dhqqqvlqav erakqvtmqe
      lnliigqqih aqqvpggppq pmgalnpfga lgatmglphg pqgllnkppe hhrpdikptg
      legpaaaeer lrnsvspadr ekyrtrspld iendskrrkd eklqedegek sdqdlvvdva
      nemeshsprp ngehvsmevr dreslngerl ekpsssgikq erppsrsgss ssrstpslkt
      kdmekpgtpg akartptpna aapapgvnpk qmmpqgpppa gypgapyqrp adpyqrppsd
      paygrpppmp ydphahvrtn giphpsaltg gkpaysfhmn gegslqpvpf ppdalvgvgi
      prharqintl shgevvcavt isnptkyvyt ggkgcvkvwd isqpgnknpv sqldclqrdn
      yirsvkllpd grtlivggea snlsiwdlas ptprikaelt saapacyala ispdskvcfs
      ccsdgniavw dlhneilvrq fqghtdgasc idispdgsrl wtggldntvr swdlregrql
      qqhdfssqif slgycptgdw lavgmenshv evlhaskpdk yqlhlhescv lslrfaacgk
      wfvstgkdnl lnawrtpyga sifqsketss vlscdistdd kyivtgsgdk katvyeviy
</PRE><P>9.	Click on “ClustalW” under “Multiple Alignment”
</P><P>10.	Copy and paste the aligned sequence into a Word document. If the sequences do not align, change the font to Courier and/or minimize the font.
</P><P>The sequences are presented in the direction of 5’→3’.The stars designate conserved regions between the species. Take note of the conserved regions as they will be crucial for primer design.
</P><P>11.	Two primers must be designed, one forward and one reverse. 
</P><P>Selection of Primers:
Select a conserved region of ~7 one-letter amino acid codes. Explicitly write out the nucleotide sequences that correspond to the one-letter amino acid code. The U must be replaced by a T in the three-letter codon.
</P><P>
Example:
</P><P>Forward Primer
5’ MMFECKW 3’ (amino acid sequence)
</P><P>5' ATG	ATG TTc/t GAa/g	TGc/t AAa/g TGG 3' (nucleotide sequence)
</P><P>In this case, the template nucleotide sequence is also the forward primer sequence. As indicated in Figure 1, the forward primer is replicating DNA in the 5’→ 3’ direction, meaning that the primer is annealing to the bottom 3’→ 5’ template strand, which is just the complementary sequence of the nucleotide sequence obtained from ClustalW. The forward primer’s sequence is exactly the same as the parental strand sequence.
</P><P><B>FINAL FORWARD PRIMER SEQUENCE:</B></P><P>5’ ATG-ATG-TTc/t-GAa/g-TGc/t-AAa/g-TGG 3’
</P><P>
Reverse Primer:
</P><P>5’ PCYTHNMC 3’ (amino acid sequence)
</P><P>5' CCa/c/g/t TGc/t TAc/t TGG CAc/t AAc/t ATG TGc/t 3' (nucleotide sequence)
</P><P>Because this is the reverse primer, the nucleotide sequence is the reverse complement of the DNA strand.
</P><P>5’ GGa/g/c/t ACg/a ATg/a ACC GTg/a TTg/a TAC ACg/a 3'
</P><P>Ideally, it is best to avoid a wobble base at either end of the primer. This problem can be solved by simply eliminating the last nucleotide if need be. In this case, the g/a nucleotide can be eliminated on the 5’ end.
</P><P><B>FINAL REVERSE PRIMER SEQUENCE:</B></P><P>5’ CA-CAT-g/aTT-a/gTG-CCA-a/gTA-a/gCA-a/g/c/tGG 3’
</P><P>
12.	Some amino acids have degeneracy, the condition in which more than one codon corresponds to one amino acid. During primer design, it is best to minimize degeneracy.
</P><P>13.	Other factors to consider during Primer Design:
</P><UL><LI> Each primer should be ~18-28 nucleotides long (~7 amino acids).
</LI><LI> The 3’ end of primers should end with a G or C or GC, but not all three should be G/C. If the 3’ end does not end with G/C, eliminate the final nucleotide. 
</LI><LI> No wobble base should be located on either end of the primer.
</LI><LI> Distance between the two primers should be ~ 500-1000 nucleotides, which is ~ 160-330 one-letter amino acid codes.
</LI><LI> Melting temperature should be between 55-70* Celsius. Calculate this at Oligo Calc.
</LI><LI> GC content should be roughly 50-60%. Calculated this at Oligo Calc.
</LI></UL><P>14.	After having determined the best possible primer sequences that satisfy all of the aforementioned criteria, order the primers.
Polymerase Chain Reaction
</P><P>	PCR employs the method of thermal cycling:
Denaturation:  At 94°C, hydrogen bonds between complementary bases of the DNA strands are disrupted, separating the two template strands.
Annealing:  At 58 °C, the primers anneal to their respective single-stranded DNA templates. Extension/Elongation: At 72 °C, DNA polymerase synthesizes the new DNA strands by adding dNTPs that are complementary to the template in 5' to 3' direction.
</P><P>	The key ingredients are listed as follows:
</P><P>•	dH2O  (Adjusted to the total 50 µL volume. In this case, ~28.5 µL)
</P><P>•	Buffer .10 µL
</P><P>•	Primers .4 µL/ primer
</P><P>•	dNTPs (building blocks from which the DNA polymerases synthesizes a new DNA strand) .2 µL
</P><P>•	template DNA (DNA region to be amplified) .1 µL
</P><P>•	Taq Polymerase (enzyme originally isolated from the bacterium Thermus aquaticus; heat-stable) 0.5µL
</P><P>MgCl2 is necessary for the activation of active sites. It may or may not be included in the buffer. Adjust the volumes accordingly.
</P><P>Notes: When working with the 200 µL PCR tubes, be careful not to touch the cap since human DNA will contaminate the sample. Place the PCR tubes in the PCR machine. If necessary, program the machine for the 35 cycles. Press “Start.” Collect the PCR product after 2-3 hours.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Brown/Notebook_Protocols/PCR">http://2009.igem.org/Team:Brown/Notebook_Protocols/PCR</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Brown/Notebook_Protocols/PCR" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Brown/Notebook_Protocols/PCR" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Brown/Notebook_Protocols/PCR&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Brown/Notebook_Protocols/PCR&amp;oldid=160499" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>