"
Page
Discussion
View source
History
teams
Log in
 
Team:Brown/Notebook Protocols/PCR
From 2009.igem.org
Home
Team
About Us
Brown University
Project
Abstract
The Allergic Response
Histamine Sensor
Histamine Binding Protein
S.epidermidis
System Schematic
Human Practices
Parts
Notebook
Weekly Logs
Protocols
Recipes
Team Meetings
Links
Acknowledgements
Sponsors
Protocol: Polymerase Chain Reaction
Background/Purpose
Polymerase Chain Reaction (PCR) is used to amplify DNA segments by way of template strands, primers, and DNA polymerase. PCR is initiated by a short sequence of primers (one forward and one reverse). Primers are short, complementary sequences of roughly 18-28 nucleotides that anneal to the template strands to begin the synthesis of the new DNA strands. Following the primers annealing to the separate template strands respectively, the DNA polymerase begins to add complementary nucleotides to the template strands thereafter to complete the polymerase chain reaction.
DNA is synthesized in the 5’→3’ direction. Since DNA is antiparallel, the DNA polymerases add nucleotides in opposite direction.  That is, each primer anneals at opposite ends of the desired DNA of interest, prompting the DNA polymerases to build towards each other. The region in between the two primers is replicated. See Figure 1 for the PCR Mechanism.
Methods
Primer Design
1. Identify gene of interest
2. Go to NCBI and navigate to the “All Databases” drop-down menu in the upper left part of the webpage and select the “Protein” tab.
3. Now type in the text box “gene of interest organism” Example: Search “Protein” for “Groucho sea urchin”
4. Click on “Go”
5. Click on one of the relevant results
6. Scroll down to “Origin.” Copy and paste this one-letter code amino acid sequence into a  Word document. Format this by deleting the numbers dispersed in the overall sequence. Spaces in between the sequence will suffice. Example:  Pristionchus pacificus
mdkragsrgg ggggnpflda leklkddynh mqaqlssqra eldkmnaeke qlqrhymmyy       emscglnmem qkqsevakrm tallqsmlqy aphdaqasti qamerakqis lqelqqltaa       sqaqqmlgmt gpmaamgglg gmagalggpg glnmaaiaaa mgaglrppap pgggggddrp       apsssrqsss rqrsgspagg ekkpkleted gdddeidvqn ddpagpaang ktggrdsvhs       gisssgastp aaaaaknfaa qlgqqrlpla qldpatrmmm qgmmapngka pysyrvdstg       nlaptmfppd altdpgvpks vkavhdlphg evvcavaisk daqrvftggk gcvkiwdlaa       ntsaararle clednyirsc klfaegthlv vggeasnill fdietqkeva kldttaqacy       alalnqeskl lyaccadgav vifdlasmqe varlpghtdg ascvdlsgdg lrlwtggldh       tlrswdirer relsnidfas qifslgcspt edwvavgldt nqievvntap gvkeryqlhr       hdscvlslrf ahsgkwfctt gkdnllnvwr spygalsvra sesssvlscd ishddsvivt       gsgekkatvy qvqyesss
7. Identify another relevant organism with the same gene of interest and repeat the aforementioned process.
8. Alignment: Go to Clustal W. Paste amino acid sequences in the following format (the title “>speciesname1” should not contain any spaces):
>speciesname1
mdkragsrgg ggggnpflda leklkddynh mqaqlssqra eldkmnaeke qlqrhymmyy       emscglnmem qkqsevakrm tallqsmlqy aphdaqasti qamerakqis lqelqqltaa       sqaqqmlgmt gpmaamgglg gmagalggpg glnmaaiaaa mgaglrppap pgggggddrp       apsssrqsss rqrsgspagg ekkpkleted gdddeidvqn ddpagpaang ktggrdsvhs       gisssgastp aaaaaknfaa qlgqqrlpla qldpatrmmm qgmmapngka pysyrvdstg       nlaptmfppd altdpgvpks vkavhdlphg evvcavaisk daqrvftggk gcvkiwdlaa       ntsaararle clednyirsc klfaegthlv vggeasnill fdietqkeva kldttaqacy       alalnqeskl lyaccadgav vifdlasmqe varlpghtdg ascvdlsgdg lrlwtggldh       tlrswdirer relsnidfas qifslgcspt edwvavgldt nqievvntap gvkeryqlhr       hdscvlslrf ahsgkwfctt gkdnllnvwr spygalsvra sesssvlscd ishddsvivt gsgekkatvy qvqyesss
>speciesname2
mypspvrhpa aggpppqgpi kftiadtler ikeefnflqa qyhsiklece klsnektemq       rhyvmyyems yglnvemhkq teiakrlntl inqllpflqa dhqqqvlqav erakqvtmqe       lnliigqqih aqqvpggppq pmgalnpfga lgatmglphg pqgllnkppe hhrpdikptg       legpaaaeer lrnsvspadr ekyrtrspld iendskrrkd eklqedegek sdqdlvvdva       nemeshsprp ngehvsmevr dreslngerl ekpsssgikq erppsrsgss ssrstpslkt       kdmekpgtpg akartptpna aapapgvnpk qmmpqgpppa gypgapyqrp adpyqrppsd       paygrpppmp ydphahvrtn giphpsaltg gkpaysfhmn gegslqpvpf ppdalvgvgi       prharqintl shgevvcavt isnptkyvyt ggkgcvkvwd isqpgnknpv sqldclqrdn       yirsvkllpd grtlivggea snlsiwdlas ptprikaelt saapacyala ispdskvcfs       ccsdgniavw dlhneilvrq fqghtdgasc idispdgsrl wtggldntvr swdlregrql       qqhdfssqif slgycptgdw lavgmenshv evlhaskpdk yqlhlhescv lslrfaacgk       wfvstgkdnl lnawrtpyga sifqsketss vlscdistdd kyivtgsgdk katvyeviy
9. Click on “ClustalW” under “Multiple Alignment”
10. Copy and paste the aligned sequence into a Word document. If the sequences do not align, change the font to Courier and/or minimize the font.
The sequences are presented in the direction of 5’→3’.The stars designate conserved regions between the species. Take note of the conserved regions as they will be crucial for primer design.
11. Two primers must be designed, one forward and one reverse.
Selection of Primers: Select a conserved region of ~7 one-letter amino acid codes. Explicitly write out the nucleotide sequences that correspond to the one-letter amino acid code. The U must be replaced by a T in the three-letter codon.
Example:
Forward Primer 5’ MMFECKW 3’ (amino acid sequence)
5' ATG ATG TTc/t GAa/g TGc/t AAa/g TGG 3' (nucleotide sequence)
In this case, the template nucleotide sequence is also the forward primer sequence. As indicated in Figure 1, the forward primer is replicating DNA in the 5’→ 3’ direction, meaning that the primer is annealing to the bottom 3’→ 5’ template strand, which is just the complementary sequence of the nucleotide sequence obtained from ClustalW. The forward primer’s sequence is exactly the same as the parental strand sequence.
FINAL FORWARD PRIMER SEQUENCE:
5’ ATG-ATG-TTc/t-GAa/g-TGc/t-AAa/g-TGG 3’
Reverse Primer:
5’ PCYTHNMC 3’ (amino acid sequence)
5' CCa/c/g/t TGc/t TAc/t TGG CAc/t AAc/t ATG TGc/t 3' (nucleotide sequence)
Because this is the reverse primer, the nucleotide sequence is the reverse complement of the DNA strand.
5’ GGa/g/c/t ACg/a ATg/a ACC GTg/a TTg/a TAC ACg/a 3'
Ideally, it is best to avoid a wobble base at either end of the primer. This problem can be solved by simply eliminating the last nucleotide if need be. In this case, the g/a nucleotide can be eliminated on the 5’ end.
FINAL REVERSE PRIMER SEQUENCE:
5’ CA-CAT-g/aTT-a/gTG-CCA-a/gTA-a/gCA-a/g/c/tGG 3’
12. Some amino acids have degeneracy, the condition in which more than one codon corresponds to one amino acid. During primer design, it is best to minimize degeneracy.
13. Other factors to consider during Primer Design:
Each primer should be ~18-28 nucleotides long (~7 amino acids).
The 3’ end of primers should end with a G or C or GC, but not all three should be G/C. If the 3’ end does not end with G/C, eliminate the final nucleotide.
No wobble base should be located on either end of the primer.
Distance between the two primers should be ~ 500-1000 nucleotides, which is ~ 160-330 one-letter amino acid codes.
Melting temperature should be between 55-70* Celsius. Calculate this at Oligo Calc.
GC content should be roughly 50-60%. Calculated this at Oligo Calc.
14. After having determined the best possible primer sequences that satisfy all of the aforementioned criteria, order the primers. Polymerase Chain Reaction
PCR employs the method of thermal cycling: Denaturation:  At 94°C, hydrogen bonds between complementary bases of the DNA strands are disrupted, separating the two template strands. Annealing:  At 58 °C, the primers anneal to their respective single-stranded DNA templates. Extension/Elongation: At 72 °C, DNA polymerase synthesizes the new DNA strands by adding dNTPs that are complementary to the template in 5' to 3' direction.
The key ingredients are listed as follows:
• dH2O  (Adjusted to the total 50 µL volume. In this case, ~28.5 µL)
• Buffer .10 µL
• Primers .4 µL/ primer
• dNTPs (building blocks from which the DNA polymerases synthesizes a new DNA strand) .2 µL
• template DNA (DNA region to be amplified) .1 µL
• Taq Polymerase (enzyme originally isolated from the bacterium Thermus aquaticus; heat-stable) 0.5µL
MgCl2 is necessary for the activation of active sites. It may or may not be included in the buffer. Adjust the volumes accordingly.
Notes: When working with the 200 µL PCR tubes, be careful not to touch the cap since human DNA will contaminate the sample. Place the PCR tubes in the PCR machine. If necessary, program the machine for the 35 cycles. Press “Start.” Collect the PCR product after 2-3 hours.
Retrieved from " http://2009.igem.org/Team:Brown/Notebook_Protocols/PCR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
