<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Brown_Notebook_weekly_Logs_Weekly_Team3_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Brown/Notebook weekly Logs/Weekly Team3 Notebook</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu"><UL><LI id="mainHome"><A href="https://2009.igem.org/Team:Brown">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Brown/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Team">About Us</A></LI><LI><A href="http://www.brown.edu">Brown University</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Brown/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Introduction">The Allergic Response</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Histamine_Sensor">Histamine Sensor</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_HBP">Histamine Binding Protein</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_S. epidermidis">S.epidermidis</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_All_Together">System Schematic</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Implications">Human Practices</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Brown/Parts">Parts</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Brown/Notebook_Weekly_Logs">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Weekly_Logs">Weekly Logs </A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Protocols">Protocols</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Recipes">Recipes</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_meetings">Team Meetings</A></LI><LI id="mainLinks"><A class="aLinks" href="https://2009.igem.org/Team:Brown/Links_Acknowledgements">Links</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Links_Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Links_Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="S.epidermidis_and_Secretion_Notebook">S.epidermidis and Secretion Notebook</SPAN></H1><P>07/16/09 - 07/23/09
</P><P>Date: 07/16/09
</P><UL><LI>We are going to use pSB1A3 instead of PSB1C3 (Ampicillin resistance instead chloramphenicol resistance). We are transforming cells with them.
</LI></UL><P>
Date: 07/17/09
</P><UL><LI>We miniprep the DNA from the overnight liquid cultures of pSB1A3-RFP. Now, we are digesting pSB1A3 with EcoRI and SpeI according to the same protocol from 07/15/09). We are using 11ul of SB1A3 plasmid DNA (500ng at 6.2ng/ul) to make 20ul of double digests.  We are adding 3ul of dH2O to the digest. However, the gel extraction fails because too much gel (1.2g) was cut out.
</LI></UL><P>
Date: 07/19/09
</P><UL><LI>We are ligating Signal Peptide insert into pSB1A3 according to the following:
</LI><LI>Ligation with a total volume of 10ul:
</LI><LI>Linearized DNA and semi vol vector = 2 – 4 ul
</LI><LI>1x Ligase Buffer
</LI><LI>T4 DNA Ligase = 200ul
</LI></UL><UL><LI>The gel extraction’s concentration turns out to be too low so we are redoing the digest of pSB1A3 with EcoRI+ SpeI using Tom Knights Protocol:
</LI></UL><P>50ul Reaction
</P><UL><LI>NEB2 Buffer/ Multicore = 5ul
</LI><LI>pSB1A3 DNA (700ng) = 15ul
</LI><LI>100x BSA = 0.5ul
</LI><LI>EcoRI = 1ul
</LI><LI>SpeI = 1ul
</LI><LI>dH2O = 27.4ul
</LI></UL><UL><LI>We run the digest program on Thermal Cycler. We are incubating the digest at 37oC for 10 hr, then at 80 oC for 20min, and at 4 oC forever or until we need it.
</LI></UL><P>Date: 07/20/09- Notes from Adrian
</P><P>Flowchart for Digestion, Ligation, and Transformation
</P><P>&lt;PHOTO HERE&gt;</P><P>
Date: 07/20/09- Mathing It Out, Take 1!  
</P><UL><LI>Ligation of Signal Peptide: BioBrick Into BioBrick Vector
</LI><LI>10:1 ratio
</LI><LI>4.4ul vector DNA of 11.4ng/ul
</LI><LI>0.4ul SigPep insert
</LI><LI>3 x [insert length (bp) / vector length (bp)] x Vector mass (ng)
</LI><LI>3 x [131bp / 2157bp] x 50ng = 9.1ng insert
</LI><LI>1ul 10x Ligase Buffer
</LI><LI>9.1ul H2O
</LI><LI>0.5ul Ty DNA Ligase
</LI></UL><UL><LI>5:1 ratio
</LI><LI>4.4ul vector DNA of 11.4ng/ul
</LI><LI>1ul SigPep insert
</LI><LI>3 x [insert length (bp) / vector length (bp)] x Vector mass (ng)
</LI><LI>3 x [131bp / 2157bp] x 50ng = 9.1ng insert
</LI></UL><LI>1ul 10x Ligase Buffer
</LI><LI>8.5ul H2O
</LI><LI>0.5ul Ty DNA Ligase
</LI><UL><LI>We have desgined the primers for Trg/Envz:
</LI><LI>Note: They must be digested (can be either PCRed by Tag or PfuTurbo).
</LI><LI>Name: GC, Time, Length
</LI><LI>TrgF: 50, 65.3, 32
</LI><LI>TrgR: 37.5, 61.3, 32
</LI><LI>EnvzF: 48.1, 63.9, 27
</LI><LI>EnvzR: 59.3, 71.0, 27
</LI></UL><P>
Date: 07/21/09
</P><UL><LI>Colonies were picked from E.coli that were transformed with ligations of pSB1A3 + SP, and we started miniprep culture; insert: vector ratio of 3:1 worked best for transformation (the result for that ratio was about 5x as many colonies as 1:1 and 6:1).
</LI></UL><P>Date: 07/22/09- The Plan and the Execution! 
</P><UL><LI>Here are results of the miniprepped pSB1A3-SP DNA:
</LI><LI>Insert: Vector Ratio	DNA Concentration (ng/ul)
</LI><LI>3:1	69.0
</LI><LI>6:1	94.5
</LI></UL><UL><LI>We have 2 digestions running:
</LI><LI>1) pSB1A3-SP with SpeI and PstI
</LI><LI>2) SB1A2-GFP with XbaI and PstI
</LI></UL><UL><LI>SpeI and XbaI are compatible restriction sites.
</LI></UL><UL><LI>Here is our ideal plan: 
</LI><LI>SP 
</LI><LI>GFP
</LI></UL><UL><LI>We have 3 tubes that, due to a labeling mishap, might have the GFP/protein in it. 
</LI><LI>We will digest and run all 3 out on a gel with the digest SB1A3-SP, and try to identify GFP by size.
</LI></UL><UL><LI>Double Digest using the protocol on 7/18/09 [Total volume = 50ul]
</LI><LI>Mystery 1: 40ul DNA (at 16ng/ul) with 2.5ul H2O- 640ng DNA
</LI><LI>Mystery 2: 30ul DNA(at ~30ng/ul) with 12.5ul H2O- 900ng DNA
</LI><LI>Mystery 3: 15ul DNA (at 65.5ng/ul) with 27.5ul H2O 	         982.5ng DNA
</LI><LI>pSB1A3-SP: 10ul DNA(at 94.5ng/ul) with 32.5ul H2O                 1ng DNA
</LI></UL><UL><LI>The digests fail, but we are able to identified Mystery 1 and 2 as GFP (possibly because the band was about the same size as pSB1A3-SP).  Now, we are growing overnight liquid culture of GFP cells. Also, we are growing plates of GFP, RFP, and LacX . 
</LI></UL><UL><LI>PfuTurbo PCR of full Agr  Operon:
</LI><LI>First, resuspend Rev_Agr_EagI (19.43nm) in 194.3ul H2O to 100uM.
</LI><LI>Dilute to 0.4uM
</LI><LI>100x =0.4 (10)
</LI><LI>x = 0.4ul primer with 8.96ul H2O.
</LI></UL><LI>Per Reaction:
</LI><LI>dH2O- 41ul
</LI><LI>10x cloned Pfu reaction buffer- 5ul
</LI><LI>dNTPs at 10uM-  1ul
</LI><LI>DNA template at 60ng/ul / 42.3ng/ul- 1ul
</LI><LI>Primer 1- 1ul
</LI><LI>Primer 2-1ul
</LI><LI>PfuTurbo- 1ul
</LI><UL><LI>Transformed:
</LI><LI>RFP (Kan)
</LI><LI>GFP (Amp)
</LI><LI>LacZ -   (Kan)
</LI></UL><UL><LI>From the plate, we pick a colony containing SigPep – PSB1A3 ligation, and we prepare a glycerol stock for it and it is growing at 37oC overnight (12-16hr). Tomorrow, this glycerol stock will be freeze at -80 oC. 
</LI></UL><P>Date: 07/23/09- Mathing It Out, Take 2! 
</P><UL><LI>The glycerol stock for SigPep – PSB1A3 ligation that we made yesterday has been put into the -80 oC freezer. We have plated the glycerol stock solution of pSB1A3 – RA on the LB+Amp plate to test whether or not the glycerol stock works. We found out that the glycerol stock volume is less than we expected so we decide to make another glycerol stock for PSB1A3 – RA.
</LI></UL><UL><LI>Calculation for making SMMP [ Total volume = 100ml]
</LI></UL><P>1) 5.5 parts (55%) SMM Buffer = 55ml:
a. 1M Sucrose (Molar mass = 342.29648g/mol)
</P><UL><LI>1mol /L = X/ 0.055L
</LI><LI>X12 = 18.8263g
</LI></UL><P>b. 0.04M Maleic Acid (Molar mass = 116.1g/mol)
</P><UL><LI>0.04mol/L = X/0.055L
</LI><LI>X = 0.2554g
</LI></UL><P>c. 0.04M MgCl2 (Molar mass = 95.211g/mol)
</P><UL><LI>0.04mol/L = X/0.055L
</LI><LI>X3 =0.0022mol
</LI></UL><UL><LI>To get 0.0022mol of MgCl2 from Magnesium Chloride Hexahydrate (Molar mass = 203.31g/mol)
</LI><LI>X4 = 0.4473g
</LI><LI>Put 18.8263g
</LI></UL><P>2) 4 parts (40%) 7% Panassy Broth = 40ml:
3) 0.5 parts (5%) 10% BSA = 5ml:
</P><UL><LI>Summary of SMMP
</LI></UL><P>1) 5.5parts SMM Buffer
a. Sucrose- 18.8263g
b. Maleic Acid- 0.2554g
c. MgCl2 x 6H2O- 0.4473g
d. ddH2O- Bring up to 55ml
</P><P>2) 4 parts 7% Panassy Broth
a. 100% panassy Broth- 2.8ml
b. ddH2O- Bring up to 40ml
</P><P>3) 0.5 parts 10%BSA
a. 10x BSA- 50ul
b. ddH2O- Bring up to 4.95ml
</P><UL><LI>Redid glycerol stock for PSB1A3 – RA.
</LI></UL><P>Date: 07/24/09- Essentials basics I …
</P><UL><LI>Liquid cultures; terminator sequence and ribosome binding sequences
</LI><LI>Take restriction digests and ligate them together, then transform cells with it.
</LI></UL><P>1) Run digests out on the gel; should have bands around 3000bp and 700bp 
</P><UL><LI>Gel extract just GFP.
</LI></UL><P>2) Ligation
3) Transformation
</P><UL><LI>Ligate the GFP and SigPep overnight at 4oC.
</LI></UL><UL><LI>Ligation Calculation:
</LI><LI>1.5ul vector (sp)
</LI><LI>Insert mass (ng) 6 x (720bp insert / 2500bp) x 50ng
</LI></UL><UL><LI>2  reactions running:
</LI></UL><P>1) 6:1 →0.3 ul H2O
2) 3:1 →4.9 ul H2O
</P><UL><LI>1ul 10x Ligase Buffer 
</LI><LI>0.5ul T4 DNA Ligase 
</LI></UL><UL><LI>Have miniprepped pBlueScript and grew liquid culture of RBS and TERM.
</LI></UL><P>
Date: 07/25/09- Essentials basics II …
</P><UL><LI>Have minipreped RBS and TERM cultures.
</LI><LI>GFP and SigPep Ligation have been transformed into the cells. 
</LI></UL><P>
Date: 07/26/09- Essentials basics III …
</P><UL><LI>The transformation was successful. Now we are growing 8ml liquid cultures at 1:30pm, which needs to be taken out early morning tomorrow to be miniprepped. 
</LI></UL><P>
Date: 07/27/09- Primers
</P><UL><LI>TSB plates and Tet stock solutions have been made.
</LI><LI>2 overnight cultures growing:
</LI></UL><P>1. Tet + TSB (Control)
2. Tet + TSB + S. epi.
</P><UL><LI>Have ordered primers:
</LI><LI>BioBrick Sequencing VF2
</LI><LI>BB Seq V12
</LI><LI>BB clone forward/rev
</LI><LI>BB KpnI Fwd
</LI><LI>BB Eag I Rev
</LI><LI>TrgEnvz BB Fwd/rev
</LI></UL><UL><LI>Have to design primers for Agr promoter region that reverses the strands direction. 
</LI></UL><P>
Date: 07/28/09- Redo
</P><UL><LI>Liquid S. epi cultures didn’t grow, which might be due to too much tetracycline in the medium.
</LI><LI>I have started new cultures from glycerol stock and plate. 
</LI></UL><P>Date: 07/28/09- Assembly
</P><UL><LI>The overnight liquid cultures didn’t grow again. It might be a good idea to order new S. epi.
</LI><LI>The 3 possible reasons for the cells not to grow:
</LI></UL><P>1) Glycerol stock?
2) TSB?
3) Tet’s concentration?
</P><UL><LI>To analyze the cause(s), I will have 7 reactions running:
</LI><LI>In liquid cultures: 
</LI></UL><P>1) Control = TSB +Tet
2) S. epi from new glycerol stock  + TSB (No Tet)
3) S. ep from another new glycerol stock + TSB + Tet
</P><UL><LI>On plates: 
</LI></UL><P>4) Control = TSB + Tet
5) S. epi from the old glycerol stock + TSB + Tet
6) S. epi (old glycerol that hasn’t been working + TSB + Tet)
7) New S. epi + TSB + Tet
</P><UL><LI>Going to start the assembly of OmpC + RBS + GFP + TS so that it will be ready when we make Trg-Envz. Please look at the sketches of the assembly on pg 63 -64 of the lab notebook. 
</LI><LI>Doing 4restriction digests: (1ug DNA per digest → 50ul digest)
</LI></UL><P>1. OmpC: E + S → 10ul DNA, 32.5ul H2O
2. RBS: E + S → 20ul DNA, 22.5ul H2O
3. RFP: E + S → 10ul DNA, 32.5ul H2O
4. TS: E + S → 10ul DNA, 32.5ul H2O
</P><P>
Date: 07/31/09- Rxns I …
</P><UL><LI>Primer dilution:
</LI><LI>Kpn1 – Trg – Fwd add 195.6ul H2O
</LI><LI>NdeI – Trg – Rev add 162.6ul H2O 
</LI></UL><UL><LI>For Tag PCR – dilute to 10uM
</LI><LI>100x = 10 (10)
</LI><LI>X =1ul primer each + 9ul dH2O
</LI></UL><UL><LI>PCR Rxn:
</LI><LI>Green Master Mix- 25ul
</LI><LI>Fwd Primer (10uM)- 2ul
</LI><LI>Rev Primer (10uM)- 2ul
</LI><LI>DNA- 1ul
</LI><LI>dH2O- 20ul
</LI></UL><UL><LI>Running reactions:
</LI><LI>2 reactions of Trg-genomics
</LI><LI>5 reactions of EnvZ: 3 miniprep, 2 genomic
</LI><LI>They are labeled as Trg-gen1, Trg-gen 2, Envz - mini 1, Envz – mini 2, Envz – mini 3, Envz – gen1, Envz – gen2. 
</LI></UL><UL><LI>Trg – PCR – ligate into pGEMTEasy overnight.
</LI><LI>Envz – PCR – Cut pBlueScript at XbaI and Pst I
</LI></UL><UL><LI>Cut PCR, Ligate  OR Also ligate into pGEMTEasy and cut with sites in that.
</LI></UL><UL><LI>I am trying the same PCR again because the temp for the annealing was too low when it was done the first time.  This time, the annealing time is set at 47OC.
</LI></UL><P>Date: 08/03/9- Rxns II …
</P><UL><LI>Primers received today:
</LI><LI>TrgEnvz_BB_Fwd
</LI><LI>TrgEnvz_BB_Rev 
</LI><LI>Envz_NdeI_Fwd	(Envz half of fusion)
</LI><LI>Envz_SacI_rev		(Envz half of fusion)
</LI><LI>BB_KpnI_Fwd		(pLM6 compatibility primers) 
</LI><LI>BB_EagI_Rev 		(pLM6 compatibility primers) 
</LI></UL><UL><LI>PCR SigPep +GFP with Kpn and EagI compatibility primers
</LI></UL><UL><LI>PCR (2 rxns):
</LI><LI>Green mater mix- 25ul
</LI><LI>Fwd primer (10uM)- 2ul
</LI><LI>Rev primer (10uM)- 2ul
</LI><LI>DNA- 1ul
</LI><LI>dH2O- 20ul
</LI></UL><UL><LI>We are digesting Trg, Envz, and pBluescript to make TrgEnvZ. Trying with both purified and non-purified PCR product (Purified has much lower concentration) → using Tom Knight’s protocol.
</LI></UL><UL><LI>DNA, Enzymes, Buffer, Amount of DNA, H2O
</LI><LI>Trg KpnI + Nde I: NE Buffer 1, Purified 30ul (~660ng), 12.5ul
</LI><LI>EnvZ NdeI + SpecI: Purified 30ul (~800ng), 11.5ul
</LI><LI>pBluescript	KpnI + SpecI: Multicore, 10ul, 22.5ul
</LI></UL><UL><LI>8 Ligations
</LI><LI>KpnI/Trg(~780bp )/NdeI + NdeI/EnvZ(~680bp )/SpecI 
</LI><LI>Trg – pur gel pure pBlue Scrip (3:1)  ~3kb Linearized
</LI><LI>Trg – nonpure PCR gel pure pBlueScript (6:1)  ~3kb Linearized
</LI><LI>EnvZ – pure gel pure pBlue Scrip (3:1)  ~3kb Linearized
</LI><LI>EnvZ – nonpure gel pure pBlue Scrip (6:1)  ~3kb Linearized
</LI></UL><UL><LI>Mass of pure Trg insert = 3 x [insert length (bp) /vector length (bp)] x vector mass ng (= 50ng) = 39ng ←→ 1.95ul (3:1) = 78ng ←→ 3.9ul (6:1)
</LI><LI>Mass of EnvZ insert = 40ng ←→(3:1) = 68ng ←→ 6.8ul (6:1)
</LI><LI>Trg insert (nonpure) = 0.5ul (3:1) = 1ul (6:1)
</LI></UL><UL><LI>Trg pure = 20ng/ul
</LI><LI>Trg PCR not pure = 78ng/ul
</LI><LI>EnvZ pure = 10ng/ul
</LI><LI>Redo EnvZ nonpure = 50ng/ul
</LI><LI>EnvZ nonpure = 0.8ul (3:1)
</LI><LI>EnvZ nonpure =6.25ul Vector 1.36ul (6:1)
</LI></UL><UL><LI>3:1		6:1
</LI><LI>Reaction #: Trg, EnvZ, H2O, Reaction#: Trg, EnvZ, H2O
</LI><LI>1: 1.95ul, 0.8ul, 0ul, 1: 3.9ul, 1.36ul, 0ul</LI><LI>2:	1.95ul, 0.8ul, 0.5ul, 2: 3.9ul, 1.36ul, 0ul
</LI><LI>3:	0.5ul, 4ul, 0ul, 3: 1ul, 6.8ul, 0ul
</LI><LI>4: 0.5ul, 0.8ul, 0.8ul, 4: 1ul, 1.36ul, 0.89ul
</LI></UL><P>Date: 08/04/09: Rxns III …
</P><UL><LI>Plates didn’t grow so we have to re-do Trg and EnvZ PCRS and ligate to pGEMT Easy.
</LI><LI>Doing a sequential double digest of pLM6 and SigPep +GFP with KpnI then EagI. (KpnI requires no salt in the buffer {NE Buffer I} and EagI requires salt {NE Buffer III}). To make *EagI work, I will add NaCl and EagI to the digest after one hour. Protocol on Open Wetware recommends 1-5ul of SM NaCl solution → NE Buffer III is 100uM. Otherwise, I am using Tom Knight’s protocol:
</LI></UL><UL><LI>NE Buffer I-5ul
</LI><LI>KpnI- 1ul
</LI><LI>BSA- 0.5ul
</LI><LI>500– 100ng DNA, 3ul (SP + GFP), 1.5ul (pLM6)
</LI><LI>H2O- 40.5ul, 42ul
</LI></UL><UL><LI>Finished PCR (redo of 7/31/09)
</LI><LI>Purified PCR.
</LI><LI>Ran PCR on a gel; looks good except a longer band above the main band for EnvZ.
</LI></UL><UL><LI>Now: pGEMTEasy Ligation
</LI><LI>3:1 → insert: vector
</LI><LI>Trg: {50ng vector x 0.82kb insert x 3} / 3kb vector = 41ng insert needed
</LI><LI>EnvZ: {50ng vector x 0.68kb insert x 3} / 3kb vector = 34ng insert needed
</LI></UL><UL><LI>H2O
</LI><LI>Trg gen1 = 0.4ul	6.6ul
</LI><LI>Trg gen1 = 0.4ul	6.6ul
</LI><LI>EnvZ gen1 = 0.4ul	6.6ul
</LI><LI>EnvZ gen2 = 0.3ul	6.7ul
</LI><LI>EnvZ mini1 = 0.3ul	6.7ul
</LI><LI>EnvZ mini2 = 0.3ul	6.7ul
</LI><LI>EnvZ mini3 = 0.3ul	6.7ul
</LI><LI>SigPep = 2.3ul	4.7ul
</LI><LI>1ul 10x Buffer, 1ul vector, and 1ul ligase.
</LI></UL><P>
Date: 08/05/09
</P><UL><LI>Colony PCR of 20 ul ligation
</LI><LI>9ul PCR mastermix
</LI><LI>0.25ul 40uM Fwd Primer
</LI><LI>0.25ul 40uM Rev Primer
</LI><LI>0.5ul colony template
</LI></UL><P>Date: 08/06/09
</P><UL><LI>Electroporated S. Epi again following MicroPulser S. aureus program. Let them grow 48hrs on both TSB-Tet and TSB-amp plates.
</LI></UL><UL><LI>Picked colonies of pGEM Trg and Envz ligations.
</LI><LI>PCR’d Pagr: anneal temp at 52oC. 
</LI><LI>All of S. epi gen DNA is of 58.ng.
</LI><LI>Reaction#: DNA, H2O
</LI><LI>1: 1ul, 20 ul
</LI><LI>2: 1.4 ul, 19.6 ul
</LI><LI>3: 2.9 ul, 17.1 ul
</LI><LI>4: 3.5 ul, 16.5 ul
</LI></UL><UL><LI>Realized I can put X-gel and IPTG directly onto LB plates. Using protocol 1ul Molecular Cloning (1.124: Ch.1: Protocol 27)
</LI><LI>Pipette 40ul of 2% X-gel solution and 7ul of 20% IPTG solution onto LB plate before plating cells.
</LI><LI>To make 20% IPT: put 15ul of 1M IPTG in 985ul H2O.
</LI></UL><LI>Spread with a sterilized spreader, and then incubate until all liquid has disappeared.
</LI><UL><LI>Doing digest of pSB1A2 – RFP with EcoRI and Xba in: 
</LI><LI>Digest pGEM – Trg with Kpn I and NdeI in NE Buffer1 + BSA
</LI><LI>Digest pGEM – EnvZ with NdeI and PstI in Buffer D + BSA
</LI><LI>Digest pBS - KS with Kpn I and PstI in NE Buffer1 with extra TstI with multicore.
</LI><LI>Digest pAgr with EcoRI + SpecI in Buffer E + BSA (Which will be ligated to BioBrick vector)
</LI></UL><UL><LI>PCR1: DNA 	14.8ul 	+ H2O	27.7ul
</LI><LI>PCR2: DNA 	10.1ul 	+ H2O	32.4ul
</LI><LI>PCR3: DNA 	12.2ul	+ H2O	30.3ul
</LI></UL><P>
Date: 08/08/09
</P><UL><LI>Please look at the lab notebook for list of digests that were run.
</LI></UL><P>Date: 08/09/09
</P><UL><LI>Many gels were run. Please see the lab notebook. 
</LI></UL><UL><LI>For Ligations:
</LI><LI>Double ligation of Trg and EnvZ into pBS –Ks and pAgr into pSB1A2.
</LI><LI>A1	A2			B1	B2	
</LI><LI>3:1 (ul)	6:1 (ul)	pBS – KS		3:1 (ul)	6:1(ul)	pSB1A2
</LI><LI>9.1	9.1	50ng vector		16.1	16.1	50ng vector
</LI><LI>10.5	21.1	Insert1 (Trg)		0.43	0.87	pAgr
</LI><LI>7.4	14.9	EnvZ		- 	-	-
</LI><LI>3	5	10x Ligase Buffer		2	2	1ul of 10x Ligase Buffer
</LI><LI>0	0	H2O		0.5	0	H2O
</LI><LI>1.5	2.5	0.5ul T4 DNA Ligase		1	1	0.5ul T4 DNA Ligase
</LI></UL><UL><LI>Transformations:
</LI><LI>+ Control: pBS –KS on IPTG X-gal
</LI><LI>pBSKS → Trg + EnvZ 3:1 on IPTG X-gal (A1)
</LI><LI>pBSKS → Trg + EnvZ 6:1 on IPTG X-gal (A2)
</LI></UL><UL><LI>PSB1A2 + pAgr 3:1 on LB –Amp (B1)
</LI><LI>PSB1A2 + pAgr 6:1 on LB –Amp (B2)
</LI></UL><UL><LI>Made stock solutions of:
</LI><LI>Tet = 0.5ug/ml
</LI><LI>Chloramphenicol = 10ug/ml
</LI></UL><P>
Date: 08/09/09
</P><UL><LI>Transformation of pLM6
</LI><LI>30ul of E.Coli competent cells
</LI><LI>2ul of pLM6 (add and stir gently)
</LI><LI>Incubate on ice for 30min
</LI><LI>Heat shock at 42oC for 30sec.
</LI><LI>Incubate on ice for 2min.
</LI><LI>Add 200ul of LB (substituted for S.O.C medium).
</LI><LI>Incubate at 37 oC for 1hr with constant shaking.
</LI><LI>Plate using glass beads:
</LI><LI>Just Amp + LB plate (control)
</LI><LI>50ul of cells + Amp + LB plate
</LI><LI>250ul of cells + Amp + LB plate
</LI></UL><LI>Grow overnight at 37 oC (12-16hr)
</LI><UL><LI>REALIZED pLM6 is a shuttle vector between E.Coli and Staph. with both AmpR and CHR.
</LI><LI>Doing pGEM-T EZ ligation with pAgr PCR product:
</LI><LI>Protocol from Open Wetware:
</LI><LI>1ul T4 DNA ligase
</LI><LI>X vector 
</LI><LI>Y insert
</LI><LI>(9.5 – X – Y) ul H2O
</LI><LI>0.5ul Ligase Buffer 
</LI></UL><P>1. To make 10ng of pGEM (Vector) = o.1ul from 100ng/ul of pGEM stock.
2. Get 6ng of insert
3. Tube 1: pGEM-T EZ with [49.6ng/ul] PCR pAgar.
</P><UL><LI>dH2O- 8.3ul
</LI><LI>10x ligation buffer- 1ul
</LI><LI>pAgr (insert)- 0.1ul
</LI><LI>100ng/ul pGEM-T EZ (vector)- 0.1ul
</LI><LI>T4 DNA ligase- 0.5ul
</LI><LI>Total- 10ul
</LI></UL><P>4. Tube 2: pGEM-T EZ with [41ng/ul] PCR pAgar.
</P><UL><LI>dH2O- 8.2ul
</LI><LI>10x ligation buffer- 1ul
</LI><LI>pAgr (insert)- 0.2ul
</LI><LI>100ng/ul pGEM-T EZ (vector)- 0.1ul
</LI><LI>T4 DNA ligase- 0.5ul
</LI><LI>Total- 10ul
</LI></UL><P>5. Tube 3: pGEM-T EZ with [33.7ng/ul] PCR pAgar.
</P><UL><LI>dH2O- 8.2ul
</LI><LI>10x ligation buffer- 1ul
</LI><LI>pAgr (insert)- 0.2ul
</LI><LI>100ng/ul pGEM-T EZ (vector)- 0.1ul
</LI><LI>T4 DNA ligase- 0.5ul
</LI><LI>Total- 10ul
</LI></UL><UL><LI>Ligation; the tubes are sitting at room temp for 3-6hr. 
</LI><LI>Afterward, the pGEM-pAgr (3 plates) and pBS-TrgEnvZ (3 plates) are grown overnight. 
</LI><LI>Redoing ligations of Trg – EnvZ from gel extractions; following OWW consensus protocol and using 10ng vector instead of 50ng.
</LI><LI>Tube #	pBS (ul)	Trg at 2.7ng/ul (ul), Concentration of EnvZ (ng/ul), EnvZ (ul), H2O (ul)
</LI></UL><P>1- 2.7	2.5	2	3.4	0
2- 2.7	2.5	2.5	2.7	0.5
3- 2.7	2.5	2.1	3.2	0
4- 2.7	2.5	4.4	1.5	1.7
</P><P>
Date: 08/11/09
</P><UL><LI>Grew up E.Coli with pLM6 and pGEM – pAgr in liquid culture and miniprepped. 
</LI></UL><P>
Date: 08/12/09
</P><UL><LI>Digested pSB1A2 – RFP with Eco + Xba from pGEM – pAgr with Eco +Spec.
</LI><LI>Gel extracting them.
</LI><LI>The yield of the gel extraction is too low so the leftover digest will be run again on the gel:
</LI><LI>Results: 
</LI><LI>R = 6.1ng/ul
</LI><LI>A4 = 2.8ng/ul
</LI><LI>A5 = 2.5ng/ul
</LI><LI>A6 = 1.4ng/ul
</LI></UL><P>Date: 08/13/09
</P><UL><LI>Gel Extractions were pretty low.
</LI><LI>Re-digest: 4ug
</LI><LI>A: Trg 1 +2 		w/ Kpn1 and Nde 		+ NEB1 	+ BSA
</LI><LI>B: EnvZ M1B 		w/ NdeI and SpecI 		+ NEB4 	+ BSA
</LI><LI>C: pBSKS		 w/ Kpn1 and SpecI 		+ NEB1/Multicore 	+ BSA
</LI><LI>D: SigPep + GFP	w/ Kpn1 and Eag1-HF 	+ NEB4 	+ BSA
</LI></UL><UL><LI>DNA (ul), H2O (ul)
</LI><LI>A: Trg 1: 17.9, 24.0
</LI><LI>A: Trg:	10.9, 31.6
</LI><LI>B: EnvZ M1B: 19.4, 23.1
</LI><LI>C: pBKS: 19.2, 23.3
</LI><LI>D: SigPep +GFP: 20.3, 22.2
</LI></UL><P>
Date: 08/15/09
</P><UL><LI>Gel extractions:
</LI><LI>Gel Mass (g), Buffer QG (ul)
</LI><LI>A1: 0.0490, 147
</LI><LI>A2: 0.0438, 131
</LI><LI>B: 0.0891, 267
</LI><LI>C1: 0.1260, 378
</LI><LI>C2: 0.1493, 447
</LI><LI>D: 0.0564, 168
</LI></UL><P>
Date: 08/16/09
</P><UL><LI>Ligations:
</LI><LI>1 = Trg + EnvZ into pBlueScript
</LI><LI>2 = pAgr into pSB1A2
</LI></UL><UL><LI>1	A	B		
</LI><LI>Trg (ul), H2O (ul), Trg 2 (ul), H2O (ul), EnvZ (ul), pBSKS - lin (ul)
</LI><LI>3:1, 3.3, 2.7, 3.7, 2.3, 1.7, 1.8
</LI><LI>6:1, 6.6, 5.5, 7.4, 4.7, 3.3, 3.6 – 22ul 
</LI></UL><UL><LI>2	C	
</LI><LI>pAgr (ul), pSB1A2 –lin (ul)
</LI><LI>3:1	, 0.43, 9.8
</LI><LI>6:1	, 0.87, 9.8
</LI><LI>1ul of 10x Ligase Buffer and 0.5 ligase.
</LI><LI>Digest pLM6 overnight with Kpn1 and EagI.
</LI></UL><P>
Date: 08/19/09
</P><UL><LI>Early morning – digest pLM6 with Kpn1 and EagI.
</LI><LI>NEB4, 4.5hrs: Redo:
</LI><LI><DL><DD> Concentration (ng/ul), Digest 4ng – pLM6 (ul), H2O, 1hr
</DD></DL></LI><LI>1: 218, 18.3, 24.2, A1
</LI><LI>2: 182, 22, 20.5, A2
</LI><LI>3: 253, 15.8, 26.7, A3
</LI><LI>4: 226, 17.7, 24.8, A4
</LI></UL><UL><LI>Digest TrgEnvZ with Kpn1 and SpeI to ensure proper assembly.
</LI><LI>NEB Buffer1 OR Multicore.
</LI><LI>	[Large] = 2.5ng	NEB Buffer (ul)	H2O (ul)		#	[Small] =0.5ng	Multicore (ul)	H2O (ul)
</LI><LI>3	389 ng/ul	6.4	36.1		1	44 ng/ul	11.4	31.1
</LI><LI>5	480 ng/ul	5.2	37.3		2	39.2 ng/ul	12.8	29.7    4	    42 ng/ul	11.9	30.6
</LI></UL><UL><LI>TrgEnvZ send for sequencing
</LI></UL><UL><LI>Digest 1.6ug 
</LI><LI><DL><DD> DNA (ul), H2O (ul)
</DD></DL></LI><UL><LI>A1: 7.3, 35.2
</LI><LI>A2: 8.8, 33.7
</LI><LI>A3: 6.3, 36.2
</LI><LI>A4: 7.1, 35.4
</LI></UL></UL><UL><LI>pAgr Digest 1ug
</LI><LI>DNA (45.6ng/ul) = 21.9ul	
</LI><LI>H2O = 20.6ul
</LI><LI>EcoRI and Spec I compatible with Buffer E
</LI></UL><UL><LI>pSB1A2 + RFP digest 1.5ng
</LI><LI>DNA (101ug/ul) = 14.9ul
</LI><LI>H2O = 27.6 ul
</LI><LI>EcoRI and Xba I compatible with Buffer H
</LI></UL><UL><LI>Strange; pLM6 band is much smaller than expected. 
</LI><LI>Also, pSB1A2 is too large.
</LI><LI>pAgr worked!
</LI></UL><UL><LI>Now, digesting miniprepped GFP in pSB1A2 to see its size.
</LI><LI>If that works, we can clone in pAGr directly because this is just for assembly.
</LI><LI>We are retransforming pSB1A2 + GFP, pSB1A2 + RFP, pLM6, and both Trg EnvZs.
</LI></UL><UL><LI>Back to digestion:
</LI><LI>GFP in pSB1A2 → with EcoRI and XbaI in 5ul Buffer H.
</LI><LI>Take 750ng from [72ng/ul ] 
</LI></UL><UL><LI>10.4ul DNA 
</LI><LI>32.1ul H2O
</LI><LI>0.5ul BSA
</LI></UL><UL><LI>This produces the correct band!
</LI><LI>Now, ligate the gel extract pSB1A2 lin overnight with pAgr digest at 4oC. 
</LI><LI>Ratio, DNA (ul), H2O (ul)	
</LI><LI>3:1, 0.65, 2.3 - All rxns 6.6 pSB1A2, 1ul of 10x Ligase Buffer, 0.5ul T4 Ligase
</LI><LI>6:1, 1.29, 1.6	
</LI></UL><UL><LI>Plating:
</LI><LI>Name: DNA (ul), Plating amount (ul) in LB + Amp
</LI><LI>Trg EnvZ: 480- 0.5, 50
</LI><LI>Trg EnvZ: 390- 1, 50
</LI><LI>pAgr – pSB1A2: (3:1), 2.5, 100
</LI><LI>pAgr – pSB1A2: (6:1), 2.5, 100
</LI><LI>pLM6: 0.5, 50
</LI></UL><P>Date: 08/21/09
</P><UL><LI>The transformation worked! The 25ul ones were better because it has less bacteria growing so easier to pick out single colonies.
</LI><LI>Making overnight cultures from the plates with transformed cells:
</LI></UL><UL><LI>5ml LB + 5ul Amp + Single colony from:
</LI><LI>Tube 1: Trg EnvZ 390 Transformation with 25ul
</LI><LI>Tube 2: Trg EnvZ 480 Transformation with 25ul
</LI><LI>Tube 3: pLM6 Transformation with 25ul
</LI><LI>Tube 4: pAgr – pSB1A2 (3:1) Transformation with 25ul
</LI><LI>Tube 5: pAgr – pSB1A2 (6:1) Transformation with 25ul
</LI><LI>Tube 6: Trg EnvZ 390 Transformation with 50 ul
</LI><LI>Tube 7: Trg EnvZ 480 Transformation with 50 ul
</LI><LI>Tube 8: pLM6 Transformation with 50 ul
</LI><LI>Tube 9: pAgr – pSB1A2 (3:1) Transformation with 50 ul
</LI><LI>Tube 10: pAgr – pSB1A2 (6:1) Transformation with 50 ul
</LI></UL><UL><LI>pLM6 miniprepped.
</LI><LI>Digest pLM6 with KpnI and EagI – HF/ EagI for ligation with compatible SigPep + GFP.
</LI><LI>pLM6 #1(3ug DNA digested): 24ul DNA with 18.5ul H2O
</LI><LI>pLM6 #2(3ug DNA digested): 20.3ul DNA with 22.2ul H2O + 5ul NEB 4, 0.5ul BSA, 1ul of each enzyme.
</LI></UL><UL><LI>Also, digest original pLM6 with KpnI and EagI – HF
</LI><LI>5.5ul DNA (2.5ug digested) with- 37 ul H2O
</LI></UL><P>Date: 08/24/09
</P><UL><LI>Debugging pLM6
</LI></UL><P>1. Digest pLM6 (retransformed) with addt’s enzymes – EcoRI, Pst, 500bp insert.
</P><UL><LI>pLM6 #1: 5.1ul DNA	, 37.4 ul H2O
</LI><LI>pLM6 #2: 6 ul DNA, 36.5 ul H2O + 5ul NEB 4 (with 0.5ul BSA), 0.5ul BSA, 1ul of each enzyme.
</LI></UL><P>2. Sequence pLM6 (retransformed) – look for sequencing promoter.
3. Retransform original pLM6 with DH5 alpha cells selected in Chloramphenicol.
4. Run retransformed miniprepped pLM6 on a gel with digested ones.
</P><UL><LI>Glycerol stocks with Amp were made for Trg EnvZ 390 and pAgr - pSB1A2 (3:1). They are in 37oC incubator and is being grown overnight with constant shaking.
</LI></UL><P>Date: 09/12/09
</P><UL><LI>We got B. subtilis BD170 with B. subtilis – S. epi shuttle vector pC194 from AICC. Vector has the ChR and a hind II site. B. subtilis arrived in freeze dried form; we inoculated LB *medium with 10ug/ul Ch with freeze dried B. subtilis and grew for 10 hrs. Then it was miniprepped to get pC194 out. The miniprep has low concentrations (~ 10ug/ul). Plates 1ul each of B. subtilis liquid cultures onto Ch plates.
</LI></UL><P>Date: 09/14/09
</P><UL><LI>I grew up more B. Subtilis and miniprepped them. I did minipreps of 3ml of 20hr B. subtilis liquid culture and got high plasmid DNA yield (50 – 80ng/ul).
</LI></UL><P>Attempting to electorporate S. epi with miniprepped pC194 using BioRad electroporator protocol. We are growing control cultures from each of the 2 attempts at making competent cells (3ml TSB + 3ul Tet). Electroporating 3ul of [80ng/ul] pC194 miniprep into 50ul of first electro-competent batch of S. epi. Electroporating 2 of 50ul aliquots from the second in an attempt to make electro-competent S.epi. They are plated on to Ch plates for 36 – 48hrs. Electroporating using standard S. aureus program in BioRad machine.
</P><P>
Date: 09/18/09
</P><UL><LI>Remaking S.epi electrocompetent cells for new protocol.
</LI><LI>TSB + Tet + Electro S.epi
</LI><LI>Stock – stated growth at 11am.
</LI><LI>Check absorbance at 12pm; we want OD578 = 0.5 – 0.65.
</LI><LI>Received vectors from Bruckuer.
</LI><LI>1)	Transform to E. coli + miniprep
</LI><LI>2)	Digest for constructs
</LI></UL><UL><LI>pAgr + GFP – 473 culture with EcoRI + Pst I
</LI><LI>SigPep + GFP – 474 culture with EcoRI, Pst I
</LI></UL><P>Date: 09/27/09
</P><UL><LI>Each Rxn (all with Buffer H)
</LI><LI>5ul Buffer H, Xul DNA, 0.5ul 100x BSA, 1ul each enzyme, 42.5 – x ul Milli-Q H2O.
</LI><LI>, Label, Concentration [ng/ul], Amount needed for 1ug of DNA (ul), H2O (ul)
</LI><UL><LI>1, pRB474, 140.9, 7.1, 35.4
</LI><LI>2, SigPep + GFP, 229, 4.4, 38.1
</LI><LI>3, pRB473, 143, 7, 35.5
</LI><LI>4, pAgr + GFP, 223, 223, 4.5, 38
</LI></UL></UL><UL><LI>4hr incubation at 37oC (in hermal cycler) + 20min at 80oC to inactivate enzyme.
</LI></UL><UL><LI>Next, the SigPep + GFP (474) and pAgr + GFP (473) need to be cut out and it is best if they are linearized. 
</LI></UL><P>Date: 09/29/09
</P><UL><LI>Electroporation didn’t work again. Could it be that we didn’t grow post electroporated cells in a subinhibitory concentration of antibiotic.
</LI><LI>So, today, electroporate, grow each in TSB+Tet and TSB + Amp but before plating, grow for an hour in SMMP + Low Amp Concentration (with) for Amp plates and (without) for *Tet plates.
</LI><LI>Label	Concentration [ng/ul]	ul needed to attain 352ng DNA
</LI><LI>pRB474	140.9	2.5
</LI><LI>pRB473	143	2.5
</LI><LI>Making new electrocompetent cells for the last time.
</LI><LI>10/09/09 Electoporation Debugging
</LI><LI>Plasmids: pRB473pRB474 (Amp resistant), pC194 (Ch resistant)
</LI><LI>(Amp resistant)	473 		2mm		Old SMMP		New SMMP
</LI><LI>(Amp resistant)	473		1mm							
</LI><LI>Electro cells on Tet + TSB plates
</LI><LI>(Ch resistant)		pC194 		2mm		 Old SMMP		New SMMP
</LI><LI>(Ch resistant)		pC194 		1mm
</LI><LI>(Amp resistant)	474		2mm
</LI></UL><P>1) Electro cells + new SMMP OR old SMMP, 2mm and 1mm plated on TSB – Tet
2) pC194 + New SMMP OR old SMMP- 2mm and 1mm plated on TSB – Ch
3) pRB473 Old SMMP- TSB – Amp
4) pRB474 Old SMMP- 2mmm gap
</P><UL><LI>So in total, 10 rxns were done.
</LI><LI>1 – E 	SO 1
</LI><LI>2 – E 	SO 2	
</LI><LI>3 – E 	SN 1
</LI><LI>4 – E 	SN 2
</LI><LI>5 – pC194 	SO 1
</LI><LI>6 – pC194 	SO 2
</LI><LI>7 – pC194 	SN 1
</LI><LI>8 – pC194 	SN 2
</LI><LI>9 – pRB473	 250
</LI><LI>10 – pRB474	 250
</LI></UL><UL><LI>Plan for 10/9/09 to 10/12/09 and sketches of assemblies are in the lab notebook (pg 109 to 111).
</LI></UL><P>Date: 10/10/09
</P><UL><LI>Miniprepped pGA –mRBP (modified RBP in the geneart plasmid).
</LI><LI>Started liquid culture of RBS (BBa _B0030) and “CAT” (BBa_J3100S).
</LI><LI>No growth on S. epi plates yet.
</LI></UL><P>
Date: 10/11/09
</P><UL><LI>There is growth in all of the liquid cultures: 
</LI><LI>pRB473 + pAgr +GF
</LI><LI>BBa_J23119
</LI><LI>pRB474 + SigPep + GFP
</LI></UL><UL><LI>However, cannot do miniprep because there is no tube for centrifuging.
</LI><LI>Need to start new cultures again for miniprep!
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook">http://2009.igem.org/Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Brown/Notebook_weekly_Logs/Weekly_Team3_Notebook&amp;oldid=153639" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>