<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Brown_Notebook_weekly_Logs_Weekly_Team1_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Brown/Notebook weekly Logs/Weekly Team1 Notebook</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="main_wrapper"><DIV id="header"><DIV id="navigation"><DIV id="menu"><UL><LI id="mainHome"><A href="https://2009.igem.org/Team:Brown">Home</A></LI><LI id="mainTeam"><A class="aTeam" href="https://2009.igem.org/Team:Brown/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Team">About Us</A></LI><LI><A href="http://www.brown.edu">Brown University</A></LI></UL><LI id="mainProject"><A class="aProject" href="https://2009.igem.org/Team:Brown/Project">Project</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project">Abstract</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Introduction">The Allergic Response</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Histamine_Sensor">Histamine Sensor</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_HBP">Histamine Binding Protein</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_S. epidermidis">S.epidermidis</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_All_Together">System Schematic</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Project_Implications">Human Practices</A></LI><LI id="mainParts"><A class="aParts" href="https://2009.igem.org/Team:Brown/Parts">Parts</A></LI><LI id="mainNotebook"><A class="aNotebook" href="https://2009.igem.org/Team:Brown/Notebook_Weekly_Logs">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Weekly_Logs">Weekly Logs </A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Protocols">Protocols</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_Recipes">Recipes</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Notebook_meetings">Team Meetings</A></LI><LI id="mainLinks"><A class="aLinks" href="https://2009.igem.org/Team:Brown/Links_Acknowledgements">Links</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Links_Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2009.igem.org/Team:Brown/Links_Sponsors">Sponsors</A></LI></DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Histamine_Binding_Protein_Weekly_Lab_Log">Histamine Binding Protein Weekly Lab Log</SPAN></H1><P>Week 3: June 29 – July 3, 2009 
</P><P>Monday Lab Meeting:
</P><UL><LI>Team 1 is working on digesting insert out of pBluescript using ECOR1 and BAMH1
</LI><LI>pVU2 Site→ use to analyze digest of plasmid
</LI><LI>When cloning inserts into vectors: make sure insert sequence is in frame
</LI><LI>Keep in mind the reading frame when cloning into pNO-TAT→ check for additional sequences on both ends of inserts
</LI></UL><P>Adrian’s Restriction Digest Suggestions:
</P><UL><LI>1 unit of enzyme cuts 1µg DNA ~1 hour
</LI><LI>Use narrow wells with 0.5 µg DNA, wide wells with 4.5 µg DNA
</LI><LI>Digest mastermix
</LI><LI>Can cut with enzyme again if not sure whether plasmid
</LI></UL><P>Week Summary:
</P><P>6/29
</P><UL><LI>Ran gel of last Friday’s troubleshooting digests performed on various plasmids to confirm enzyme efficiency
</LI><LI>Recorded DNA concentrations for last Thursday’s minipreps→  medium-high concentrations
</LI><LI>Re-ran digest since troubleshooting digest was successful→  unsuccessful digest
</LI><LI>Plated out glycerol stocks→ Michael
</LI><LI>Prepared minipreped pBluescript w/ insert samples for sequencing
</LI><LI>Learnt how to properly use the nanodrop
</LI><LI>PCR primers with restriction sites engineered arrived via FedEx
</LI></UL><P>6/30
</P><UL><LI>PCR rxn of pBluescript w/ insert using primers that came in on 6/29 performed
</LI><LI>Re-ran digest from 6/29 with great care; mastermix used
</LI><LI>Sequences of pBluescript and pNOTAT examined to be in frame
</LI><LI>Learnt how to re-suspend primers
</LI></UL><P>7/1
</P><UL><LI>Ran improved PCR with adjusted temperatures and using Green Mastermix with Taq
</LI><LI>New troubleshooting restriction digest performed: digested 5 different plasmids
</LI><LI>Read binding affinity papers sent by Steph, and also read through pGEM vector kit protocol
</LI></UL><P>Week 4: July 6 – July 10 2009 
</P><P>Monday Lab Meeting:
</P><UL><LI>Failed digests and PCR- could it be poor technique, or insert not in pBluescript?
</LI><LI>Drop off fedex at Biomed front desk for sequencing; get package out to Yale right away
</LI><LI>Hope to isolate EV131 insert at least by the end of the week, if not earlier
</LI></UL><P>Suggestions:
</P><UL><LI>Run lower percentage gel to get resolution between 3000 and 3500, usually run 1% gel
</LI><LI>Run a couple of individual digests with individual enzymes
</LI><LI>PCR: Amplify any other template alongside as a control
</LI></UL><P>Week Summary
</P><P>7/6
</P><UL><LI>Dropped of samples for sequencing	
</LI><LI>Ran digests of EV131 + individual enzymes (ECOR1, BAMH1, ECO&amp;BAM, and “GUS” control plasmid + ECOR1)
</LI><LI>Ran digest on a low resolution gel (1%-0.8%) to see difference between 3&amp;3.5 kb (linearization of DNA)
</LI><LI>Ran digests on other plasmids (“GUS” plasmid) to test enzymes
</LI><LI>Ran PCR of pBluescript w/ EV131 insert with new primers (w/ restriction sites)
</LI><LI>Ran control PCR of “GUS” plasmid alongside
</LI><LI>Arian gave suggestions for improving PCR
</LI></UL><P>7/7
</P><UL><LI>PCR run again; however, no EV131 expected product seen on gel
</LI><LI>Liquid culture of cells containing pBluescript plasmid→ miniprep them tomorrow
</LI></UL><P>7/8
</P><UL><LI>Notes on making sure insert if actually in the plasmid: either insert fail or primer fail
</LI><LI>Will run these together:
</LI></UL><P>1) T7 w/ T3:   
a) 50 bp→ no insert present
b) 500 bp→ an insert is present, but wrong insert selected
c) 800 bp→ correct insert selected
</P><P>2) M13 Forward w/ Reverse universal primers→ 570 kB
3) GUS DNA w/ T3 &amp; T7
4) GUS DNA w/ specific primers
</P><UL><LI>Miniprep of overnight liquid cultures performed; DNA concentrations recorded
</LI></UL><P>7/9
</P><UL><LI>PCR using T7/T3 &amp; M13(F) &amp; R universal primers on GUS control 
</LI><LI>169 bp is expected size of the band will be if there is no insert, 741 bp is expected size if insert is present 
</LI><LI>Expected band observed!
</LI></UL><P>
Week 5: July 13 – July 17 2009 
</P><P>
Monday Lab Meeting:
</P><P>Suggestions:
</P><UL><LI>Order new primers with buffering nucleotides on 5’ ends (no need for reverse  complement of restriction site sequence)
</LI><LI>Look at sequence chromatograph for single base pair deletions/changes
</LI><LI>Run new PCR with new primers along side GUS control
</LI><LI>Order pGEM vector kit
</LI><LI>Make sure pNOTAT vector works/is available
</LI></UL><P>Week Summary:
</P><P>7/13
</P><UL><LI>New primers designed and ordered with different combinations of buffer nucleotides, BAMH1 and ECOR1 sites
</LI></UL><P>7/15
</P><UL><LI>Biobrick primers ordered for EV131
</LI></UL><UL><LI>Mostly waiting this week for primers to arrive
</LI><LI>Some more digests attempted, but unsuccessful
</LI></UL><P>
Week 6: July 20-July 26, 2009
</P><P>7/20
</P><UL><LI>Conducted 4 PCRs of EV131. Successful!
</LI><LI>Three Ligations overnight of pGEM Easy-T Vector and EV131
</LI></UL><P>7/21
</P><UL><LI>Tranformations of overnight ligations onto 3 plates of DH-5 Alpha competent cells
</LI><LI>Conducted more successful PCR of EV131 in order to amplify more of EV131
</LI></UL><P>7/22
</P><UL><LI>Conducted another ligation of pGEM Easy T-Vector and EV131
</LI></UL><P>7/23
</P><UL><LI>Miniprepped cells transformed from 7/21 ligations
</LI><LI>Ran a gel of this to confirm band size of approximately 3600 base pair
</LI><LI>Tranformations of 7/22 ligations into DH-5 Alpha competent cells
</LI><LI>Conducted PCR with BioBrick primers
</LI></UL><P>7/24
</P><UL><LI>Sent miniprepped samples of 7/21 ligations for sequencing
</LI><LI>Re-do of Transformations of 7/22 ligations into DH-5 Alpha cells because T1 did not grow. Miniprepped MP1, MP2, MP3
</LI></UL><P>Week 7: July 27-August 1, 2009
</P><P>
7/27
</P><UL><LI>Digestion of miniprepped ligation products to confirm successful ligation. Confirmed.
</LI><LI>Ligation of pGEM T Easy Vector and EV131
</LI></UL><P>7/28
</P><UL><LI>Tranformations of ligations into DH-5 Alpha cells. Healthy growth of cells on plate.
</LI></UL><P>7/29
</P><UL><LI>Inoculation of cell colony into a liquid culture
</LI><LI>Miniprepped the cells in the liquid culture
</LI><LI>Digestion of minipreps
</LI></UL><P>7/30
</P><UL><LI>Sequencing results confirm that EV131 was successfully ligated into pGEM vector.
</LI><LI>Double digested the pGEM-EV131 ligation in order to isolate EV131 in preparation for ligation of pNOTAT-EV131
</LI><LI>Digested pNOTAT with BamH1 and EcoR1
</LI><LI>Gel Extracted EV131 insert and pNOTAT vector
</LI><LI>Ligated EV131 and pNOTAT
</LI></UL><P>7/31
</P><UL><LI>Tranformation of ligated products into DH5-Alpha cells. 
</LI><LI>Liquid cultures started late night.
</LI></UL><P>8/1
</P><UL><LI>Miniprepped pNOTAT-EV131 ligated colonies from liquid cultures
</LI><LI>Made three BL21 glycerol stocks
</LI></UL><P>
Week 8: August 3-August 7, 2009
</P><P>8/3
</P><UL><LI>Digested pNOTAT-EV131 miniprepped ligations from 7/30
</LI><LI>Received correct EV131 biobrick primers
</LI><LI>Conducted PCR of EV131 biobrick
</LI><LI>Ran gel to qualitatively confirm a successful pNOTAT-EV131 ligation. Results incomplete.
</LI></UL><P>8/4
</P><UL><LI>Digest and gel extraction of biobrick backbone vector
</LI><LI>Submitted sequencing for pNOTAT-EV131 ligation
</LI></UL><P>8/5
</P><UL><LI>Various ligations of biobrick backbone vector and PCRed EV131 biobrick insert
</LI><LI>Began various liquid cultures of BL21 cells from glycerol stock solution with tetracycline antibiotic application due to Tet plates
</LI><LI>Transformations of biobricked EV131-vector 
</LI></UL><P>8/6
</P><UL><LI>Began liquid cultures of biobricked EV131-vector
</LI><LI>UTRA symposium
</LI></UL><P>8/7
</P><UL><LI>Miniprepped biobricked EV131-vector
</LI><LI>Sequencing of biobricked EV131-vector submitted
</LI><LI>Made competent BL21 cells
</LI></UL><P>
Week 9: August 10 – August 14, 2009
</P><P>Monday Lab Meeting:
</P><UL><LI>0.2% IPTG to 5 mL BL21 liquid culture (5-10 µL) to induce protein expression
</LI><LI>0.2 g for 100 mL
</LI></UL><P>Week Summary:
</P><P>8/10
</P><UL><LI>Liquid cultures of BL21 transformed cells started on 8/7→ cell density too high
</LI><LI>Therefore, 500µL of that 5 mL culture was removed and resuspended in 5 mL  of fresh LB media + 0.5 µL-1 µL Tetracycline→ these cultures left to grow at 37ºC for 2 hours
</LI><LI>Following incubation, 10 µL of 1 mM IPTG added to 5 mL cultures to protein expression (10µL/5mL = 0.002 = 0.2%)
</LI><LI>Left induced cells (6 liquid cultures induced with IPTG, 6 blank cultures as controls) for 2-5 hours. Tubes were removed after 1 hour intervals (ex. tube 1 removed after 2 hours, tube 2 removed after 3 hours)
</LI><LI>pNOTAT-EV131 Ligation sequencing results received→ entire sequence struck through
</LI><LI>We will redo the pNOTAT-EV131 ligation and resend for sequencing
</LI></UL><P>→ Double digest of MP1 and MP2 samples (minipreps of EV131-pGEM) 7/29, using ECOR1 and BAMH1, put in incubator for 2 hours at 12 pm
</P><P>	          RD 1	RD 2
Buffer E	2 µL	         2 µL
H2O	6.8 µL	10.8 µL
BSA	        0.2 µL	0.2 µL
DNA	10 µL	6 µL
ECOR1	0.5 µL	0.5 µL
BAMH1	0.5 µL	0.5 µL
</P><UL><LI>Gel run of double digests of pGEM-EV131 ligation→ EV131 insert gel extracted
</LI><LI>Insert will be ligated with pNOTAT vector
</LI><LI>Gel extraction concentrations:
</LI></UL><P>	    260/280	Concentration (ng/µL)
GE 1	1.87	10.5
GE 2	3.19	4.2
</P><P>
8/11
</P><UL><LI>12 samples  (6 IPTG induced, 6 controls) prepared for SDS PAGE
</LI><LI>Sample preparation included spinning down, pouring supernatant, re-suspending pellet in lysis buffer, spinning down, and loading supernatant
</LI><LI>Undenatured samples added→ streaks visualized on gel, not very clean
</LI><LI> Adrian: running native (undenatured) samples means you’re running a globular protein which will not allow you to visualize actual protein size because it will folded and not linearized. You run native proteins only if you need to retain biological function (ie. enzymes)
</LI><LI>Denaturing SDS-PAGE kit ordered→ will try this tomorrow
</LI><LI>Remaining 5 mL cell culture that was frozen was thawed and used to prepared new samples for tomorrow’s SDS PAGE
</LI></UL><P>8/12
</P><UL><LI>SDS-PAGE of induced and un-induced BL21 cells run for forty minutes approximately (8/11 AM)
</LI><LI>Gel coomasie stained for approximately 4 hours (8/11 PM)
</LI><LI>Gel de-stained overnight (8/11- picture in notebook)
</LI><LI>Troubleshooting/Adrian’s Suggestions: Make sure there isn’t too much buffer in chamber, and that buffer level is just coating gel top
</LI><LI>Run gel for longer maybe?
</LI><LI>Check sample preparation
</LI></UL><UL><LI>Ligation of pNOTAT (from RD on 7/30) and EV131 (8/10) performed
</LI><LI>17.5 ng/µL pNOTAT, 10.5 ng/µL EV131
</LI><LI>Insert mass = (3-10) x insert length/vector length x vector mass (50 g) = 6  x  (572/3000) x 50 ng = 57.2 ng
</LI><LI>50 ng/ 17.5 ng/mL = 2.9 µL pNOTAT
</LI><LI>57.2 ng/10.5 ng/µL = 5.4 µL EV131
</LI></UL><P>	H2O	10x Ligation  Buffer	pNOTAT	EV131	T4 DNA ligase
L1	1.2 µL	1 µL	2.9 µL	5.4 µL	0.5 µL
L2	1.2 µL	1 µL	2.9 µL	5.4 µL	0.5 µL
L3	1.2 µL	1 µL	2.9 µL	5.4 µL	0.5 µL
</P><UL><LI>Transformation into BL21 competent cells→ plated onto 3 plates  (50 µL, 100 µL, 200 µL)
</LI></UL><P>
8/13
</P><UL><LI>5 mL liquid cultures of pNOTAT/EV131 transformed BL21 cells- 6 tubes (1 µL Tetracycline, 0.5 µL Tetracycline)
</LI><LI>Will miniprep tonight cells tonight, linearize with restriction digest, check for successful ligation on gel
</LI><LI>Preparation of SDS-Page samples:
</LI></UL><P>→Aliquot 100 µL of each sample into eppendorf
→Spin for 30 seconds at 13,000 rpm
→Discard supernatant
→Resuspend cell pellet in 20 µL H2O
</P><UL><LI>2 SDS-PAGEs run of 12 samples using new denaturing protein kit. Good resolution   (see picture in notebook)
</LI></UL><P>
8/14
</P><UL><LI>SDS-PAGE re-run of 10µL samples remaining from yesterday
</LI><LI>Received sequencing for pNOTAT-EV131→ positive ligation! Confirmed that it was in-frame using expacy.com
</LI></UL><UL><LI>Meeting with Gary Wessel:
</LI></UL><P>→ Confirmed sequencing for EV131-pNOTAT ligation
→ Tutorial  on using expacy.com to make sequence is in frame with vector
→ EV131 protein actually expected approximately 3 kDa b/c of addition of 6-His
→ Gary suggested we blot this gel with 6-His antibody instead of staining
→ Also discussed how to increase protein yield: incubate BL21 at room temperature and 37ºC cultures simultaneously→ see if it will increase yield
</P><UL><LI>Today’s SDS-PAGE will not be stained→ we will transfer to Nitrocellulose membrane for Western blot→ blotting for 6-His protein 
</LI><LI>How to make SDS buffer: 50 mL of MES, 950 mL of deionized H2O
</LI><LI>Western Blot: Part 1- Gel transfer to Nitocellulose:
</LI></UL><P>→uses same chamber as SDS PAGE
→ Electrode set up (“Run to RED”)
→ Black e-→sponge→sponge→wattman paper→GEL→Nitorcellulose→wattman paper→sponge→sponge→Red e-
→ Set up above “sandwich” in a tray filled with buffer
→ Avoid air bubbles, do not touch center of gel
→Move electrode + membrane sandwich to electrophoresis chamber
→ Pour SDS-Tris +MeOH buffer in center only, ensure no leaks
→Run at 30 V for 75 mintues to allow transfer to membrane
→ Blot immediately, or store nitrocellulose between wattman paper at RT
→ Prepare solutions (Blotto, antibodies to 1:5000 dilution, etc.) for blotting
</P><UL><LI>We will blot gel on Monday
</LI></UL><P>
Week 10: August 17 – 21, 2009
</P><P>
Week summary:
</P><P>8/17 
</P><UL><LI>Ran immunoblot on nitrocellulose paper that gel was transferred to on Friday. 
</LI><LI>Results: Blank immunoblot except for the initially transferred ladder
</LI></UL><P>Blotting protocol:
</P><P>Reagents:
1) Blotto (50 ml 1M Tris [pH 7.5], 10.8g NaCl, 0.5 ml Tween20, 30 g non-fat dry milk)
2) Primary antibody: Anti-mouse monoclonal 6-his, 1:5000 dilution, Secondary antibody: Anti-mouse (conjugated) fluorescence, 1:5000 dilution
3) Alkaline phosphotase buffer (per liter, 50 mM Tris [pH 9.6], NaCl 10.8g, 5mM MgCl2)
4) Staining solution NBT/BCIP
</P><P>Procedure:
1)	Place the nitrocellulose transfer in excess Blotto. Incubate at room temp for at least 30 mins
2)	React with primary antibody (rapidly thawed and diluted to 1:5000 in Blotto). Incubate at room temp, with shaking, for at least 30 mins
3)	React with alkaline-phosphotase-conjugated secondary antibody (diluated to 1:5000 in Blotto). Incubate at room temp, with shaking, for at least 30 mins
4)	Wash blots 2X in Blotto, for 5 minutes each and then 1X with alkaline phosphotase buffer for 5 mins
5)	Pour off wash alkaline phosphotase buffer and wash 1X with 5 ml staining solution (5ml alkaline phosphotase buffer, 33ul NBT, 17ul BCIP). Agitate and monitor for several minutes.  
6)	To record, photograph immediately, or store dry (though it will lose some color intensity)
</P><P>8/18 
</P><UL><LI>Re-ran SDS page gel with IPTG induced and non-induced BL21 (with pNOTAT) samples (from 8/10)
</LI><LI>Issues with transfer process: After 15 mins, the power source was turned off due to loud beeping and restarted after 45 mins. Gel transfer was run at 22V (to avoid beeping sound) for 75 mins and clean transfer of the ladder was observed.
</LI></UL><P>8/19 
</P><UL><LI>Immunoblot conducted on 8/18 gel transfer with following changes to procedure:
</LI><LI>~ 30 ml blotto used for blocking
</LI><LI>New technique for secondary antibody application: Blotto and nitrocellulose paper placed in plastic sealed pouch and then 2° antibody injected into pouch
</LI><LI>To make 1:5000 dilution of 1° antibody, need 6ul antibody 
</LI><LI>2° antibody needed at 1:200 dilution and NOT 1:5000
</LI></UL><UL><LI>Made alkaline phosphotase buffer as per protocol above.
</LI></UL><UL><LI>Results: Nitrocellulose transfer shows slightly smeared band at bottom but no clear indication of EV131 protein express
</LI></UL><UL><LI>Started new overnight liquid cultures of 8/7 BL21-pNOTAT transformations
</LI></UL><P>8/20 
</P><P>Summary of troubleshooting meeting with Gary:
</P><UL><LI>Discussed problems with western gel and pinpointed incorrect use of antibiotic resistance for growing transformations and liquid cultures.
</LI><LI>BL21 cells are TET resistant, but pNO-TAT expression vectors are AMP resistant
</LI><LI>To grow BL21 cells when making them competent have to add TET, but when transforming pNO-TAT into BL21 cells must add AMP to select only for BL21 cells with pNO-TAT plasmid
</LI><LI>Possible to grow cells in AMP-TET or AMP plates
</LI></UL><UL><LI>Transformed 8/1 pNOTAT minipreps into competent BL21 cells (using protocol from 7/31) on AMP-TET plates.
</LI><LI>Plates showed no growth when checked at midnight. Plan to use only AMP plates for next set of transformations.
</LI></UL><P>8/21
</P><UL><LI>Plated transformations of pNO-TAT with EV131 into BL21 cells (from yesterday) onto AMP only resistant plates. Showed no growth again.
</LI><LI>Repeated transformations using only MP1 (that showed positive sequencing result) and sterile loop method instead of glass beads. Showed no growth once again.
</LI></UL><P>
Week 11: August 24 – 28, 2009
</P><P>
Week summary:
</P><P>8/24 
</P><UL><LI>Due to failed transformations of BL21 + ligated pNO-TAT plasmid decided to make new stocks of BL21 cells to redo transformation. 
</LI><LI>Specific changes: Ideal optical density = 0.3, MP DNA in transformation should be 20-30 ng
</LI><LI>BL21 liquid culture started
</LI></UL><P>8/25 
</P><UL><LI>BL21 liquid cultures tested for optical density
</LI><LI>10am - #2: 0.83, #3: 0.97
</LI><LI>12pm - #2: 0.34, #3: 0.54
</LI><LI>Transformed pNOTAT-ev131 into new BL21 competent cells (12 pm sample) on AMP plates
</LI></UL><P>8/26 
</P><UL><LI>No growth on AMP plates, only control TET plate showed growth
</LI><LI>Problem: Very little 8/4 MP ligated product
</LI></UL><UL><LI>Transformed MP 1-6 (8/13) and MP of pGEM ligation (7/23) as a control for AMP plate. Possible contamination on plate T5 prior to plating transformation sample.
</LI></UL><P>8/27 
</P><UL><LI>Only plate T3 showed 5-6 single colonies to be grown up in LB overnight. Remaining plates showed no growth, incubated for 4h more, discarded when still no signs of growth. 
</LI></UL><UL><LI>3 liquid cultures from plate T3 showed growth, placed in new media to be miniprepped.
</LI></UL><P>8/28 
</P><UL><LI>Prepared 2 minipreps from T3 liquid cultures to send for sequencing to confirm transformation of pNO-TAT ligated to EV131 in BL21 cells.
</LI><LI>Nanodrop concentrations (260/280):
</LI><LI>MP1 25.9ul (2.04)
</LI><LI>MP2 7.7ul (2.14)
</LI><LI>Concentrations are low but samples prepared for sequencing anyway.
</LI></UL><P>- BREAK BEFORE NEW SCHOOL YEAR COMMENCED -
</P><P>Week 12: September 14 – 18, 2009
</P><P>
9/14  
</P><UL><LI>Sequencing results from 8/28 were futile
</LI><LI>Miniprepped 8/13 ligations to send for sequencing
</LI><LI>Nanodrop concentrations:
</LI><LI>MP1 19.9 (1.93)
</LI><LI>MP2 23.2 (1.89)
</LI><LI>MP3 200.6 (1.22)
</LI><LI>Sample MP3 is useless due to bad quality DNA
</LI></UL><UL><LI>T3 plate (8/27) liquid cultures started. 
</LI></UL><P>9/15 
</P><UL><LI>2 liquid cultures from T3 plate showed growth. Induced with IPTG.
</LI><LI>Streaked glycerol stocks of EV131-pBLue (6/25) cells on AMP plates to grow overnight
</LI><LI>IPTG induced samples taken out after 2 hrs and 4 hrs.
</LI></UL><P>9/16 
</P><UL><LI>Good growth on EV131-pBlue Amp plates (Growing up liquid cultures to be induced with IPTG put on hold because we don’t have an antibody to detect protein produced by pBluescript [does not have his tag])
</LI></UL><UL><LI>IPTG induced samples prepared for Western Blot. Discarded because samples were too viscous to be loaded)
</LI><LI>Colonies picked from plates 5 &amp; 6 that were streaked with Gary’s BL21 cells. (Plates 1,2,3,4 showed no growth and are back in the incubator to see if growth occurs)
</LI><LI>6 liquid cultures started for each plate.
</LI></UL><P>9/17
</P><UL><LI>BL-21-pNOTAT liquid cultures induced with 10 uL of 1 mM IPTG and allowed to grow for 2, 3, and 4 hour intervals in the 37*C incubator
</LI><LI>Liquid cultures did not show positive cell growth. 
</LI></UL><UL><LI>Inoculation of fresh colonies from BL21 plate into liquid cultures.
</LI></UL><P>9/18
</P><UL><LI>No cell growth from liquid cultures that were started yesterday.
</LI><LI>Prepared the 9/16 IPTG-induced cell cultures into SDS Page samples.
</LI><LI>Began 10 new BL21-pNOTAT liquid cultures.
</LI></UL><P>
Week 13: September 21-25, 2009
</P><P>
9/22
</P><UL><LI>Out of 10 liquid cultures, 7 show healthy growth. 
</LI><LI>Miniprepped these samples, with relatively low concentrations, in the range of 10-20 ng/uL
</LI></UL><P>9/23
</P><UL><LI>Ran an SDS page of IPTG induced samples from 9/16.
</LI></UL><P>9/24
</P><UL><LI>Conducted Western Blotting, with application of primary antibody anti-His mouse Antibody (1:5000) and secondary antibody rabbit and mouse HRP (1:200).
</LI><LI>Result: Completely blank nitrocellulose paper except for the ladder.
</LI></UL><P>9/25
</P><UL><LI>Troubleshooting of incorrect secondary antibody, switch to Alkaline Phosphatase antibody
</LI><LI>Began to regrow pNOTAT BL21 cells
</LI><LI>Began new pNOTAT/eV131 ligations
</LI></UL><P>Week 14: September 28- October 2, 2009
</P><P>
9/28
</P><UL><LI>Conducted a restriction digest of pGEM-eV131 and pNOTAT plasmid in preparation for pNOTAT-eV131 ligation
</LI><LI>pGEM-eV131 restriction digests unsuccessful. pNOTAT plasmid restriction digest successful
</LI></UL><P>9/29
</P><UL><LI>Reconducted restriction digests from yesterday
</LI><LI>Visualization of gel and gel extraction concentrations suggest successful restriction digest results
</LI></UL><P>9/30
</P><UL><LI>Conducted ligations of eV131 and pNOTAT while varying the ligation ratios in different samples. 
</LI></UL><P>10/2
</P><UL><LI>Received BL21 (DE3) pLysS Supercompetent Cells
</LI><LI>Conducted 6 transformations on amp plates using the 9/30 ligations, and 1 control transformation using control DNA
</LI></UL><P>10/3
</P><UL><LI>Only growth of 10-15 colonies on 1 of the 6 ligation plates and growth on control plate as well
</LI></UL><P>Week 15 &amp; 16: October 5-16, 2009
</P><P>
10/5
</P><UL><LI>Reconducting transformations on freshly made amp plates
</LI><LI>Plated on 10 fresh plates. Lack of growth on plates.
</LI></UL><P>10/6
</P><UL><LI>Asked for the extended guidance and oversight of Gary to see if he can troubleshoot further.
</LI></UL><P>10/14
</P><UL><LI>Conducted transformations of biobrick ligations onto 2 plates utilizing the supercompetent cells
</LI></UL><P>10/15
</P><UL><LI>Yielded successful growth on the plates
</LI><LI>Began 6 liquid cultures
</LI></UL><P>10/16
</P><UL><LI>Miniprepped the biobrick liquid cultures. Successful concentrations in the range of 35-55 ng/uL.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook">http://2009.igem.org/Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Brown/Notebook_weekly_Logs/Weekly_Team1_Notebook&amp;oldid=157948" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>