<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Imperial_College_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Imperial College/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><DIV class="menu"><UL class="nav-list"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                      PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Project">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Mechanisms">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Applied_Design">APPLICATIONS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Future">FUTURE WORK</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Brainstorming">BRAINSTORM</A></LI></UL><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                    DRY LAB
                    </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Dryover">OVERVIEW</A></LI><LI><A href=" https://2018.igem.org/Team:Imperial_College/Model">MODELLING</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Software">SOFTWARE</A></LI><LI><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                        WET LAB
                        </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Wetover">OVERVIEW</A></LI><LI><A href=" https://2018.igem.org/Team:Imperial_College/Demonstrate">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Experiments">METHODS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Protocols">REFERENCES</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Measurement">BASIC</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Supplementary">SUPPLEMENTARY</A></LI><LI><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                        HUMAN PRACTICES
                        </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Communication">COMMUNICATION</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/IHP">INTEGRATED HP</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Public_Engagement">OUTREACH</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Collaborations">COLLABORATIONS</A></LI><LI><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                       PEOPLE
                        </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Sponsors">SPONSORS</A></LI><LI><A id="navbarDropdown" href="https://2018.igem.org/Team:Imperial_College/Judges">FOR JUDGES</A></LI><LI><A id="navbarDropdown" href="https://2018.igem.org/Team:Imperial_College/Journal">JOURNAL</A></LI></DIV></NAV></DIV><DIV class="container"><DIV class="content"><DIV class="titleimg"><H1>Results</H1></DIV><H3>Building and Testing Pixcell Constructs</H3><H4 class="underbold"> Assembling the Pixcell Constructs</H4><DIV onclick="expandAll('expt1p');" class="expandButton"><P class="expand expt1p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P> The PixCell Construct consists of a repurposed version of the soxRS regulon from E. coli, consisting of SoxR and GFP being expressed from either side of the pSoxR/pSoxS bidrirectional promoter. pSoxR provides constitutive expression of SoxR. When oxidised, either directly by redox-cycling molecule or by oxidative stress, SoxR binds and activates transcription of GFP downstream of pSoxS.</P></DIV><DIV type="button" class="accordion">Relevance</DIV><DIV class="panel expt1p"><P> The Pixcell construct was designed to test whether we could control the expression of GFP by controlling the oxidation state of SoxR through our electrochemical set up.</P></DIV><DIV type="button" class="accordion">Results</DIV><DIV class="panel expt1p" style="transition:max-height 1s ease-in-out;"><P> Using the Golden Gate assembly method, with two inserts; the sensing portion of the soxRS regulon and GFP, which were originally amplified out from the E. coli genome and a storage vector respectively, the construct was assembled. Gel electrophoresis showed that a construct of approximately the right length was produced and the construct was then sequence verified. </P><DIV class="caption"><P><B>Figure 1.</B> PCR results from the amplification of the entire construct</P></DIV><P>In order to lower the basal level of GFP expression a novel degradation tag was added. Therefore, any increase in GFP expression induced by the oxidation of SoxR would be more predominant.</P><DIV class="caption"><P><B>Figure 2.</B> Illustration of the PixCell Construct</P></DIV><DIV class="caption"><P><B>Figure 3.</B> Illustration of the PixCell Construct DegTag</P></DIV></DIV><DIV type="button" class="accordion">Summary</DIV><DIV class="panel expt1p"><P> The PixCell construct enables us to test whether we can control gene expression by controlling the oxidation state of SoxR. The PixCell construct DegTag enables the system to be more dynamic in that seeing the system switch from on to off is quicker. </P></DIV><DIV id="expt2"><H4 class="underbold">Characterizing Pixcell Constructs</H4><DIV onclick="expandAll('expt2p');" class="expandButton"><P class="expand expt2p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>PixCell, the 2018 Imperial College iGEM team, characterised this device in a series of steps in order to make it respond to an electrical stimulus. The device functions by oxidising pyocyanin and ferrocyanide with an electrode. Oxidised pyocyanin in turn oxidises SoxR to activate the device. The potential also oxidises ferrocyanide to ferricyanide, allowing electrons to be drawn away from pyocyanin via the quinone pool to amplify this response. Full growth curves and GFP expression profiles with standard error bars for each construct at each redox-molecule concentration are available in the “Supplementary Data” section.
</P></DIV><DIV type="button" class="accordion">Relevance</DIV><DIV class="panel expt2p"><P>These results demonstrate that the assembled PixCell circuit can be switched on by oxidative signals and switched off by reductive signals. These experimental data were also handed over the dry lab, where information about fold induction and degradation rates supported and further implemented the theoretical models.</P></DIV><DIV type="button" class="accordion">Results</DIV><DIV class="panel expt2p" style="transition:max-height 1s ease-in-out;"><P>The construct was first tested with a range of pyocyanin concentrations (0-100μM) to measure the response of the system to the redox-cycling drug. High-concentrations of pyocyanin exert significant stress upon the cell leading to cell death. A working concentration of 2.5μM of pyocyanin is therefore recommended when using it as an inducer or within an electrogenetic device, as it provides an optimal trade-off between fold induction and cell viability.
</P><DIV class="caption"><P><B>Figure 1.</B> Growth of cells containing the construct in a range of pyocyanin concentrations, with untransformed DJ901 as a negative control and DJ901 transformed with a GFP expression cassette as a positive control. Data obtained from three biological replicas. Above 5 uM of pyocyanin show reduced viability. Cells with the PixCell constructs survive more, as they have the SoxR adaptive response. </P></DIV><DIV class="caption"><P><B>Figure 2.</B>GFP fluorescence normalised with respect to OD600 at a range of pyocyanin concentrations, with untransformed DJ901 as a negative control and DJ901 transformed with a GFP expression cassette as a positive control. Data obtained from three biological replicas. At low pyocyanin concentration a significant increase in GFP is observed in cells containing the PixCell construct. GFP folds at high pyocyanin concentration due to reduced cell's viability.</P></DIV><P>A significant sixfold induction in fluorescence was seen in the device when comparing values at 0μM and 2.5μM of pyocyanin, as shown in Fig. 3. 
</P><DIV class="caption"><P><B>Figure 3.</B>GFP fluorescence as a function of pyocyanin concentration, taken at steady state (900 mins), normalised with respect to OD600, and averaged over three biological replicas, for circuits with and without deg tag and with untransformed DJ901 as a negative control and DJ901 transformed with a GFP expression cassette as a positive control. Data obtained from three biological replicas.</P></DIV><P>Oxidisation of pyocyanin, without intentional induction, was observed in the aerobic environment, as evidenced by the response of the cells. To prevent this behaviour, sodium sulfite (an oxygen scavenger) was added to the medium to maintain an OFF state in the absence of deliberate induction. Without this innovation, the device could only work in anaerobic conditions, limiting the potential of electronic induction of gene expression. A final sodium sulfite concentration of 0.02% was selected for the electrogenetic system as it prevented GFP expression from an agar plate in aerobic conditions with 2.5μM pyocyanin and 2.5mM ferrocyanide. 
</P><DIV class="caption"><P><B>Figure 4.</B>Plate reader data showing absorbance at 390 nm (absorption peak of oxidised pyocyanin) as a function of sodium sulfite concentration. Addition of sodium sulfite reduces a solution of 125 uM pyocyanin dissolved in autoclaved water.  </P></DIV><P>We observed that the oxygen scavenger sodium sulfite reduces pyocyanyn in a linear relationship until a concentration of 2.5% sulfite added. At these concentrations, we observed electrochemical interference when perfoming square-wave voltammetry experiments. We hypothesised that sulfites could react with pyocyanin, producing pyocyanin sulfonate, which has an absorption peak at 400 nm
</P><DIV class="caption"><P><B>Figure 5.</B>OD390 (absorption peak of pyo-ox) and OD400 (absorption peak of pyocyanin sulfonate) as a function of sulfite concentration. </P></DIV><P>The results shown in figure 5 supported the hypothesis that pyocyanin sulfonate is produced at high sulfite concentration. Therefore, we performed the next experiments at sulfite concentrations between 0 and 0.03% (figure 6),  which appeared to fall in the linear region of the graph.
</P><DIV class="caption"><P><B>Figure 6.</B> Agar plates of construct-bearing cells with varying sodium sulfite conditions, imaged under UV transluminescence, showing that 0.02% is sufficient to suppress unintentional induction. All the plates contain 2.5 uM pyocyanin and 2.5 mM ferrocyanide.</P></DIV><P>Next the concentration of ferrocyanide was optimised, as shown in figure 7.  A range of ferrocyanide and ferricyanide concentrations (0-100mM) were tested using 2.5μM pyocyanin and 0.02% sodium sulfite. This was done in order to find a single concentration at which ferrocyanide would provide no GFP expression whereas ferricyanide would allow for a large induction of GFP expression. The final condition of 10mM was selected for an optimal fold change; these concentrations also did not significantly impact cell growth. A 10mM concentration of ferricyanide provided at least an order of magnitudefold induction of GFP expression compared to ferrocyanide. Therefore if bulk oxidation of ferrocyanide to ferricyanide could be achieved then gene expression could be electronically induced in aerobic conditions.
</P><DIV class="caption"><P><B>Figure 7.</B>GFP fluorescence as a function of ferricyanide and ferrocyanide concentration, taken at steady state (700 mins), normalised with respect to OD600, and averaged over three biological replicas, for circuits with and without deg tag and the same controls as Fig. 1. Experiments performed at a sodium sulfite concentration of 0.02% and with 2.5μM pyocyanin.</P></DIV><P>In figure 8, we demonstrate that the chosen conditions are sufficient to produce high GFP expression when oxidised ferricyanide is added into the plate and low GFP when the same concentration of reduced ferrocyanide is added.</P><DIV class="caption"><P><B>Figure 8.</B>LB+ Agar plates containing 2.5 uM pyocyanin, 0.02 % sodium sulfite and 10 mM ferrocyanide (row above) and 10 mM ferricyanide (row below) containing the PixCell cnstructs and positive and negative controls.</P></DIV></DIV><DIV type="button" class="accordion">Summary</DIV><DIV class="panel expt2p"><P>These experiments allowed us to demonstrate that our constructs are responsive to oxidising input and enabled us to identify the concentrations of redox-molecule. This allowed us to select a suitable chemical composition of 2.5uM Pyo, 0.02% sodium sulfite and 10mM ferrocyanide, which works both in liquid and solid cultures.
</P></DIV><H4 class="underbold"> Electrochemistry of Redox Modulators</H4><DIV onclick="expandAll('expt3p');" class="expandButton"><P class="expand expt3p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>To demonstrate the ability of our electrode set-up to selectively oxidise and reduce our redox modulators in solution, we investigated the redox status of the our redox molecules with square wave voltammetry (SWV) and by measuring the absorbance at 390 nm with a plate reader. SWV data showed that a voltage of +0.5 at the working electrode enables bulk oxidation, whereas a voltage of -0.4 allows bulk reduction. However, the presence of oxygen in aerobic environments interferes with the oxidation signal carried by the electrodes. Measuing optical absorbance at 390 nm with a plate reader, we then investigated the redox kinetics of pyocyanin in the presence of the reducing agent sodium sulfite. 
</P></DIV><DIV type="button" class="accordion">Relevance</DIV><DIV class="panel expt3p"><P>These results show that our set-up is able to selectively and reversibly oxidise and reduce our redox modulators. Furthermore , by performing these electrochemistry experiments, we proved that addition of the reducing agent sodium sulfite enables complete chemical reduction of the system, even in aerobic environment.
</P></DIV><DIV type="button" class="accordion">Results</DIV><DIV class="panel expt3p" style="transition:max-height 1s ease-in-out;"><DIV class="center"><DIV class="caption"><P><B>Figure 1.</B> Square-wave voltammetry setup</P></DIV><P>
The above setup was used to perform square-wave voltammetry to investigate bulk oxidation and reduction of our system. The setup was controlled using a multichannel potentiostat connected to a computer. 
</P><DIV class="caption"><P><B>Figure 2. Square-wave voltammetry of redox modulator</B></P></DIV><P>The graphs above summarise the results showing that our system was consistently bulk reduced at -0.3V and bulk oxidised at +0.5V. The addition of Sulfite and cell containing the pixCell construct did not significantly alter the reduction and oxidation profile. These results led us to chose +0.5V as our oxidising potential in the following experiments testing the spatial electronic control of gene induction.</P><DIV class="caption"><P class="center"><B>Figure 3.</B> Amperometry of the final plate conditions </P></DIV><P>
Amperometry was performed on plates containing 2.5uM Pyocyanin, 10mM FCN(R), Ampicillin, Kanamycin, 0.02% sulfite and E. coli containing PixCell construct Deg Tag. The results show that the current decays exponentially until a steady state is reached indicating bulk oxidation of redox modulators. 
</P></DIV></DIV><DIV type="button" class="accordion">Summary</DIV><DIV class="panel expt3p"><P>These results describe the first ever aerobic electrogenetic control system, that works both in liquid and solid E. coli cultures. </P></DIV><H4 class="underbold">Spatial Electronic Control of Gene Induction</H4><DIV onclick="expandAll('expt4p');" class="expandButton"><P class="expand expt4p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>Here we demonstrated the first spatial activation of gene expression using electronic control. 
</P></DIV><DIV type="button" class="accordion">Relevance</DIV><DIV class="panel expt4p"><P>The results below serve as proof-of-concept for our electrode set-up </P></DIV><DIV type="button" class="accordion">Results</DIV><DIV class="panel expt4p" style="transition:max-height 1s ease-in-out;"><P>An electrode rig was set up to apply the potentials identified above to cells grown on an agar plate containing the final reaction condition of 2.5μM pyocyanin, 0.02% sodium sulfite and 10mM ferrocyanide. Redox reactions only occur at the electrode surface during an electrochemistry experiment, meaning that oxidised pyocyanin and ferrocyanide were only produced in close proximity to the working electrode upon the application of a +0.5V pulse. 

Fluorescence images of agar plates clearly show localised expression of GFP around the working  electrode (W), as observed in fig. 4 and 5. This not only shows that the device allows for electronic control of gene expression, but also demonstrates high spatial control meaning it can be used for programmable spatial patterning of cell populations. 
</P><DIV class="center"><DIV class="caption"><P><B>Figure 1.</B> Setup of the electrode rig for the electronic control of spatial gene induction</P></DIV><DIV class="caption"><P><B>Figure 2.</B> Image of the electrode rig placed in the incubator. </P></DIV><P>Figures 1 and Image 2 illustrate the setup of the electrode rig. The working, counter and reference electrodes are penetrating the plate through holes. Cells were plated onto an agar plate containing the final working concentrations: 10mM FCN(R), 2.5 uM pyocyanin, 0.03% sulfite, ampicillin and kanamycin. Agar is fully covering the electrodes which are controlled by a potentiostat. An oxidising potential of +0.5% was applied. </P><DIV class="caption"><P><B>Figure 3.</B> Expected outcome for the electronic control of gene expression on an agar plate</P></DIV><P>Figure 3 illustrates the expected increased fluorescence around the working electrode. The oxidising potential results in bulk oxidation and activation of our system. </P><DIV class="caption"><P><B>Figure 4.</B> GFP expression after electric stimulation</P></DIV><P>We demonstrated elevated GFP expression around the working electrode in figure 4. </P><DIV class="caption"><P><B>Figure 5.</B> Comparison of PixCell Construct, PixCell Construct Deg Tag, positive control and negative control after application of an oxidising potential. </P></DIV><P>We next investigated GFP expression of PixCell Construct and PixCell Construct Deg Tag relative to the negative control (DJ901) and the positive control which constitutively expresses GFP. 

Fluorescence image analysis shows increased fluorescence around the working electrodes demonstrating electronic control of spatial gene expression. The addition of a degradation tag to GFP substantially reduces background fluorescence. </P><DIV class="caption"><P><B>Figure 6.</B> 3D surface plot showing relative fluorescence of PixCell Construct, PixCell Construct Deg Tag, positive control and negative control after electric stimulation. Clear peaks are evident at the working electrode sites for cells containing the PixCell Constructs.</P></DIV><P>To better visualise the results 3D surface plots were made showing GFP expression in the z axis. GFP expression level from the positive and negative control was used to set a relative scale of GFP expression. Peaks around the working electrodes clear indicate increased GFP expression. PixCell construct Deg Tag showed lower background fluorescence compared to PixCell construct. </P><DIV class="caption"><P><B>Figure 7.</B> Comparison of PixCell Construct, PixCell Construct Deg Tag, positive control and negative control after application of a reducing potential</P></DIV><P>Next the 4 cell types were exposed to a reducing potential of -0.3V (figure 7). Working electrodes do not show increased fluorescence. Higher background fluorescence in the plate with cells containing PixCell Construct compared to the the previous fluorescence images exposed to an oxidising potential is due to variation in spreading. </P><DIV class="caption"><P><B>Figure 8. </B>GFP expression on a plate with final conditions in the absence of pyocyanin</P></DIV></DIV><P>After successfully inducing spatially controlled activation of gene expression the system was tested in the absence of the redox modulator pyocyanin. This suggests that there is an electrode-induced  mechanism for triggering the pSoxS pathway in the absence of pyocyanin. The plate contains 10mM FCN(R), 0.02% sulfite, Ampicillin, Kanamycin. Results show that increased expression levels of GFP can be achieved around the working electrode in the absence of pyocyanin. Further fine tuning of the system will be required to achieve the level of precision obtained on a plate containing our final working conditions. </P></DIV><DIV type="button" class="accordion">Summary</DIV><DIV class="panel expt4p"><P>These results demonstrate precise electronic control of spatial gene expression, in solid cultures grown in aerobic condition.</P></DIV><H3>Building and Characterizing the PixCell Library</H3><H4 class="underbold">Creating the Sox Library</H4><DIV onclick="expandAll('expt5p');" class="expandButton"><P class="expand expt5p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>The PixCell part library consists of 5 SoxR transcription factors and 8 pSoxS promoters which were created for construction of electrogenetic circuits with variable activity. This allows for control of this system on both the genetic and the electrochemical layer. SoxR transcription factors were obtained from a series of homologues from non-pathogenic species. pSoxS was completely rengineered as a promoter. The promoter was made unidirectional by knocking out the upstream constitutively active pSoxS portion, a terminator was placed upstream to allow for efficient tandem assemblies and a downstream ribozyme was added to remove context dependency. Promoters were either sourced from homologues in non-pathogenic bacteria or were produced by conservative mutations to the consensus sequences and SoxR binding site. The library includes a mutant promoter which provides transcriptional repression rather than activation. 
</P></DIV><DIV type="button" class="accordion">Relevance</DIV><DIV class="panel expt5p"><P>These results expand the electrogenetic synthetic biology toolkit.</P></DIV><DIV type="button" class="accordion">Results</DIV><DIV class="panel expt6p" style="transition:max-height 1s ease-in-out;"><DIV class="caption"><P><B>Figure 1.</B>Parts were synthesised with prefix and suffixes specific to the BASIC assembly method. A modified submission vector is provided with the library which allows for assembly of BASIC parts into the standardised BioBrick pSB1C3 vector, allowing for the stored parts to be compatible with both assembly standards. All library parts were assembled into this vector and growth curves of cells containing these storage plasmids exhibited no cytotoxicity.</P></DIV><DIV class="caption"><P><B>Figure 2.</B>The BASIC assembly method was used for high-throughput assembly of every promoter with every transcription factor to test all possible activities achievable with the library. BASIC uses standardised linkers that ligate onto the sticky ends of parts that have been digested with BsaI. These linkers stitch the parts together to assemble constructs. All constructs were assembled in low-medium copy number p15A vectors (in order to reduce any cytotoxicity) with SoxR being expressed from an improved version of the BBa_J23101 promoter, with superfolder-GFP being used as a reporter. SoxR and GFP were expressed from strong RBSs contained within the linkers used to assemble the constructs. </P></DIV><DIV class="caption"><P><B>Figure 3.</B>Diagram of BASIC assembly workflow for library characterization </P></DIV></DIV><H4 class="underbold">Characterizing the Sox Library</H4><DIV onclick="expandAll('expt6p');" class="expandButton"><P class="expand expt5p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>All assembled constructs were transformed into DH5-alpha with single colonies being grown for 15 hours with cells being grown in LB with either 0μM and 2.5μM pyocyanin. The normalised fluorescence from each cell was then calculated and compared. Firstly all fluorescence values were divided by the fluorescence of the library seed (BBa_K2862006 + BBa_K2862014) which is made frdom our versions of the native E. coli SoxR and pSoxS. This gave an RPU scale giving a range of promoter activities between 0-2.17 RPUs, exhibiting tunability of our system. 
</P></DIV><DIV type="button" class="accordion">Relevance</DIV><DIV class="panel expt6p"><P>These results allowed us to evaluate whether we had created parts that could be used in a modular fashion</P></DIV><DIV type="button" class="accordion">Results</DIV><DIV class="panel expt6p" style="transition:max-height 1s ease-in-out;"><DIV class="caption"><P><B>Figure 1.</B> On the left, the difference between fluorescence at 2.5μM and 0μM of pyocyanin was also calculated. This not only showed strong inductions from some library members, but also showed that for certain transcription factors the repressor pSoxS mutant was functional. This allows for selection of a SoxR-pSoxS pairings with a specific induction strength. 
 </P></DIV><DIV class="caption"><P><B>Figure 2.</B>OD600 values for library seed, positive and negative controls at increasing pyocyanin concentration</P></DIV><P>Taking the steady state fluorescence values we can show GFP expression (and therefore construct activity) as a function of pyocyanin activity. This shows a standard sigmoidal behaviour as predicted by our model and exhibited by PixCell  Construct. 
</P></DIV></DIV><H3>Testing Alternative Non-toxic Redox Modulators</H3><H4 class="underbold">Testing Phenazine Methosulfate</H4><DIV onclick="expandAll('expt7p');" class="expandButton"><P class="expand expt7p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>Our electrogenetic control system relies on the electron carrying action of the redox-cycling drug pyocyanin, which is a toxic metabolite synthesised by Pseudomonas aeruginosa. This poses limitations to our system, with regards to safety and possible applications. Therefore, we searched in the literature for alternative SoxR inducers and and tested potential non-toxic alternatives. Shown below are the characterization data for the best candidate alternative: phenazine, methosulfate.
</P></DIV><DIV type="button" class="accordion">Relevance</DIV><DIV class="panel expt7p"><P>These results demonstrate the first ever use of PMS as a non-toxic electrogenetic inducer.</P></DIV><DIV type="button" class="accordion">Results</DIV><DIV class="panel expt7p" style="transition:max-height 1s ease-in-out;"><P>The above setup was used to perform Sqaure-wave voltammetry confirming that ferrocyanide was bulk reduced at -0.3V and bulk oxidised at +0.5V as this was the voltage at which current peaked. </P><DIV class="caption"><P><B>Figure 1.</B> Phenazine as as Cheaper, Non-toxic Redox Modulator</P></DIV><DIV class="caption"><P><B>Figure 2.</B> OD600 as a function of time and PMS concentration</P></DIV><DIV class="caption"><P><B>Figure 5.</B> Phenazine induces GFP expression in PixCell Patterning Circuits</P></DIV><DIV class="caption"><P><B>Figure 6.</B> Phenazine induces GFP expression in PixCell Patterning Circuits</P></DIV></DIV><DIV type="button" class="accordion">Summary</DIV><DIV class="panel expt7p"><P>These results demonstrates the SoxR/pSoxS system that we devised, can be used a non-toxic and effective tool for chemical, as well as electronic, gene induction.  PMS is also incredibly cheap, being 40,000x cheaper than pyocyanin, ~190x cheaper than arabinose and ~29x cheaper than aTc per reaction.  
</P></DIV><H3>Applications</H3><H4 class="underbold">Biocontainment- Gp2 Growth Inhibition </H4><DIV onclick="expandAll('expt8p');" class="expandButton"><P class="expand expt8p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>In order to test our PixCell electrogenetic part library we constructed a biocontainment device to prototype one of our proposed applications. This was constructed using an existing part in the iGEM registry: the growth inhibitor Gp2 (BBa_K1893019). By placing this downstream of pSoxS (BBa_K2862006) and expressing SoxR (BBa_K2862014) from a constitutive promoter (BBa_K2862019) we could inhibit cell growth in DH5a in the oxidising conditions we produced from our electrochemical set up. We envisioned this being used with our array in a biocontainment device in which any cells that spread to the edges of the device have their growth seized by an oxidising potential. This device could be used to prevent the accidental release of GMOs, much like how electric fences are used to restrain animals.

</P></DIV><DIV type="button" class="accordion">Relevance</DIV><DIV class="panel expt8p"><P>These results serve as proof of concepts for the biocontainment application, as described in the “Applications” sections
</P></DIV><DIV type="button" class="accordion">Results</DIV><DIV class="panel expt8p" style="transition:max-height 1s ease-in-out;"><DIV class="caption"><P><B>Figure 1.</B>Gp2 growth curves (OD600 as a function of time in oxidised and reduced condition. Negative control)</P></DIV><DIV class="caption"><P><B>Figure 2.</B>Gp2 negative control (DH5a with same construct but no Gp2) growth curves</P></DIV><DIV class="caption"><P><B>Figure 3.</B>Effect of Gp2 on growth after 1000 min in oxidising vs reducing environment</P></DIV><P>When looking at the steady state OD600 values of these constructs a clear significant ~2-fold reduction is seen in the application construct in oxidising conditions. Furthermore there is no difference between the growth of the reducing condition and the controls. This not only provides a proof-of concept for out application, but also proves the system is not significantly leaky. This means it is feasible to use a higher copy number plasmid or more toxic gene (such as MazF) in this device to provide an even more effective biocontainment device. .</P></DIV><DIV type="button" class="accordion">Summary</DIV><DIV class="panel expt8p"><P>With some improvement, and when used in combination with the PixCell electrode array, this biocontainment device could be used for biocontainment of GMOs in contained-use devices. 
</P></DIV><DIV class="footer"><DIV class="socialicons"><H4> official.imperial.igem@gmail.com</H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>