Team:Imperial College/Experiments
PROJECT
DESCRIPTION
DESIGN
APPLICATIONS
ACHIEVEMENTS
FUTURE WORK
BRAINSTORM
DRY LAB
OVERVIEW
MODELLING
HARDWARE
SOFTWARE
WET LAB
OVERVIEW
RESULTS
METHODS
REFERENCES
PARTS
BASIC
SUPPLEMENTARY
HUMAN PRACTICES
OVERVIEW
COMMUNICATION
INTEGRATED HP
OUTREACH
COLLABORATIONS
PEOPLE
TEAM
ATTRIBUTIONS
SPONSORS
FOR JUDGES
JOURNAL
Methods
Contents
Assembling the Pixcell Constructs
Characterizing Pixcell Constructs
Electrochemistry of Redox Modulators
Spatial Electronic Control of Gene Induction
Creating the Sox Library
Characterising the Sox Library
Biocontainment- Gp2 Growth Inhibition
Experiments
Assembling the Pixcell Constructs
Access All
Description
This protocol describes how to create the Pixcell construct and Pixcell construct with a degradation tag using Golden Gate assembly. Creating each of the constructs takes approximately two days
Materials
pBR322
E.coli MG1655 genomic DNA
GFP Storage Vector
10µM of Primers
Q5 polymerase (New England Biolabs, #M0491)
1% Agarose Gel
TAE Buffer
Monarch® DNA Gel Extraction Kit (NEB, #T1020S)
Golden Gate Assembly Mix (NEB, E1600S)
DJ901 and Turbo Cells
KCM
Phusion polymerase (NEB, #M0530S)
T4 DNA ligase (NEB, #M0202)
Notes
Part
Description
pBR322
A medium copy vector encoding ampicillin resistance with a pMB1,origin of replication and a mcherry cassette with BsaI sites on either end.
SoxRS
The gene encoding the transcription factor Sox R and the bidirectional promoter pSoxS/pSoxR which it regulates
GFP
Superfolded GFP which is used as a reporter gene to be placed under the control of inducible side of the promoter, pSoxS
Procedure
Name
Template
Direction
Function
Tm(°C)
Sequence
PPC1
MG655 Genome
Forward
Amplification
of SoxRS regulon
67.1
CTAGTGGGTCTCCCTCGAGAGAAAGACAAAGACCGG
PPC2
MG1655 Genome
Reverse
Amplification of SoxRS regulon
64
GTCGATGGTCTCCCATAAATCTGCCTCTTTTCAGTG
PPC3
GFP Storage Vector
Forward
Amplification of GFP
65
GTCGATGGTCTCGTATGCGTAAAGGCGAAGAA
PPC4
GFP Storage Vector
Reverse
Amplification of GFP
68
CTAGTGGGTCTCGGGACAGTAGCGAAAAAACCCCG
PPC5
PixCell Construct
Forward
Addition of a degradation tag
65
GCAAACGACGAAACTACGCTTTAGTAATGATACTAGAGCGCAAAAAACCCC
PPC6
PixCell Construct
Reverse
Addition of a degradation tag
65
CTAAAGCGTAGTTTCGTCGTTTGCCTTATACAGCTCGTCCATACCGTGG
PCR Reactions
SoxRS regulon Amplification
The PCR was carried out using the primers PPC1 and PPC2 and Q5 polymerase (New England Biolabs) following a protocol of 20 seconds of initial denaturation at 98 ºC, 30 cycles of 98 °C for 10 seconds, 59.5 °C for 30 seconds, and 72 °C for 30 seconds, and a final extension at 72 °C for 5 minutes.
GFP Amplification.
The PCR was carried out using the primers PPC3 and PPC4 and Q5 polymerase (New England Biolabs) following a protocol of 20 seconds of initial denaturation at 98 ºC, 30 cycles of 98 °C for 10 seconds, 61.5 °C for 30 seconds, and 72 °C for 30 seconds, and a final extension at 72 °C for 5 minutes.
Gel Electrophoresis
Both PCR product were run on a 1% agarose gel and extracted using the New England Biolabs gel extraction kit using the recommended protocol.
Golden Gate Assembly
The two amplicons were cloned into the pBR322 vector by adding 75ng of pBR322 and 30.1ng of SoxRS regulon and 23.4ng of GFP amplicon to the golden gate assembly mix (NEB, E1600S). Following the NEB protocol the mixture was incubated at 37°C for 1 hr and at 55°C, 5 mins.
Transformation
The PixCell construct was transformed into DJ901 and Turbo cells using the KCM heat shock protocol.
Addition of the Degradation Tag
Turbo cells containing the PixCell construct were grown overnight and miniprepped following the NEB protocol in order to obtain template DNA for the following PCR reaction. The PCR was carried out using the primers PPC5 and PPC6 and Phusion polymerase (New England Biolabs) with 3% DMSO. Following a protocol of 30 seconds of initial denaturation at 98 ºC, 30 cycles of 98 °C for 10 seconds, 65 °C for 30 seconds, and 72 °C for 3 minutes, and a final extension at 72 °C for 10 minutes. These primers produce complementary overhangs which when ligated together add a degradation tag we designed to the 3’ end of the coding region of GFP.
Ligation
The ends of the PCR amplicon produced from the primers PPC5 and PPC6 were ligated using T4 DNA ligase (NEB) following the recommended NEB protocol.
Transformation
The PixCell construct with deg tag was transformed into DJ901 cells using the KCM heat shock protocol.
Characterizing Pixcell Constructs
Access All
Description
This protocol describes how to measure E. coli growth and GFP expression over time with a plate reader, using the optical density at 600 nm as signal for cell density and excitation and emission wavelengths of 475. All characterization was performed using the FLUOstar BMG Labtech in Greiner Bio-One F-bottom dark plates with clear bottom, to decrease fluorescence background. Each cycle of this experiment takes two days and has to be repeated two days, in order to have 2 biological and 2 technical replicates, for a total of 4 days and 4 96-well plates used.
Materials
Autoclaved LB media
Autoclaved 10x concentated LB media
500 mM pyocyanin stock solution (from Sigma-Aldrich, CAS: No. 86-66-5)
125 mM potassium ferricyanide stock solution (from Sigma-Aldrich, CAS: No. 13746-66-2)
500 mM ferrocyanide stock solution (from Sigma-Aldrich, CAS 14459-95-1 )
20% Sodium Sulfite Solution
Agar plates with colonies of E. coli DJ901 strains with respective constructs: PixCell Patterning Circuit 1 (BBa_K2862021). PixCell Patterning Circuit 2 (BBa_K2862022), negative control (DJ901 WT) and positive control (constitutively expressed GFP).
37°C shaking incubator
BMG Labtech FLUOstar plate reader
4x Greiner BioOne 96-F 96-well microplate (black with clear bottom)
14 mL culture tubes
autoclaved eppendorfs
Notes
Work in very sterile conditions. To prevent contaminations in the blanks, use triple antibiotics. Store pyocyanin and ferrocyanide and ferricyanide solutions in the freezer until use. We used 10x concentrated LB in order not to dilute nutrients at higher pyocyanin concentration; this was back diluted with autoclaved water. Mastemixes have double the working concentrations, as 100 uL of solution is then diluted with 100 uL of liquid cultures.
Procedure
Grow starter liquid cultures of cells  (Day 0, 10-15 min preparation +  overnight)
Working in very sterile conditions, take 5 14mL culture tubes and prepare them according to table below:
Label
E. coli strain
Construct
Antibiotic  (Ab)
V Ab (uL)
V LB (mL)
PC1
DJ901
BBa_K2862021
Kan + Amp
5 + 5
5
PC2
DJ901
BBa_K2862022
Kan + Amp
5 + 5
5
-
DJ901
none
Kan
5
5
+
DJ901
GFP_Boo
Kan + Chl
5 + 5
5
Blanks
None
None
Kan + Amp + Chl
5 + 5
5
With a P20 pipette tip we picked the desired colonies from an agar plate and released the contaminated tip in the culture tip (for PC1, PC2 and Pos the picked colonies appeared fluorescence under a UV transilluminator).  We then place inoculated culture tubes in a 37° C in a shaking incubator overnight. For faster growth the angle between the vertical axis of the tube and the shaking plane of the incubator was set at 45°)
Prepare mastermixes (30 min)
Mastermixes for each redox molecule concentration  prepared in 1.5 mL eppendorfs. The mastermixes are enough to fill 2 96-well microplates with 100 uL of solutions in each well, in order to have two replicate of the same experiment. Take 8 autoclaved 1.5 mL eppendorfs and prepare them as described in the table below.
Mastermizes for Day 1 and Day 2 :
Condition
V Pyo (uL)
Kan (uL)
LB 10x uL
Autoclaved Water (uL)
Tot Vol (uL)
A = 0 mM Pyo
0
2.44
122
1098
1220
B = 0.01 mM Pyo
0.0976
2.44
122
1097.9024
1220
C = 0.025 mM Pyo
0.244
2.44
122
1097.756
1220
D = 0.05 uM Pyo
0.488
2.44
122
1097.512
1220
E = 0.1 mM Pyo
0.976
2.44
122
1097.024
1220
F = 0.25 mM Pyo
2.44
2.44
122
1095.56
1220
G = 0.5 mM Pyo
4.88
2.44
122
1093.12
1220
H = 1 mM Pyo
9.76
2.44
122
1088.24
1220
Mastermixes for Day 3 and Day 4:
Condition
V Pyo (uL)
Kan (uL)
LB 10x uL
Autoclaved Water (uL)
Tot Vol (uL)
A = 2.5 mM Pyo
24.4
2.44
122
1073.6
1220
B = 5 mM Pyo
48.8
2.44
122
1049.2
1220
C = 10 mM Pyo
97.6
2.44
122
1000.4
1220
D = 25 mM Pyo
244
2.44
122
854
1220
E = 50 mM Pyo
488
2.44
122
610
1220
F = 100 mM Pyo
976
2.44
122
122
1220
We transferred master mixes into clean eppendorf tubes, with volumes specified below, to add the right antibiotics:
Condition
Volume of A-H Mastermix (uL)
Ampicillin (uL)
Chloramphenicol (uL)
PC1 + PC2
406
2.44
0
+
203
0
2.44
-
203
0
0
Blanks
406
2.44
2.44
This gave a total of 32 eppendorf tubes for Day 1 and 24 on Day 2, ready to be transferred into 96-well plates.
Fill 96-well plate with media solutions
Add 100 uL of media in the desired wells, according following the label on mastermixes and table below:
1
2
3
4
5
6
7
8
9
10
11
12
A
PC1
PC1
BA
PC2
PC2
BA
-
-
BA
+
+
BA
B
PC1
PC1
BB
PC2
PC2
BB
-
-
BB
+
+
BB
C
PC1
PC1
BC
PC2
PC2
BC
-
-
BC
+
+
BC
D
PC1
PC1
BD
PC2
PC2
BD
-
-
BD
+
+
BD
E
PC1
PC1
BE
PC2
PC2
BE
-
-
BE
+
+
BE
F
PC1
PC1
BF
PC2
PC2
BF
-
-
BF
+
+
BF
G
PC1
PC1
BG
PC2
PC2
BG
-
-
BG
+
+
BG
H
PC1
PC1
BH
PC2
PC2
BH
-
-
BH
+
+
BH
OD600 matching
After overnight growth, we diluted the all the cultures to an OD of 0.1. We diluted using a FLUOstar plate reader measuring single endpoint absorbance at 600 nm. We added 100 uL of LB + antibiotic into a blank well. We filled the the microplane with the cell culture solutions and took endpoint measurements of the filled wells and blanked them with the blank well. We note down the OD600 value of the wells and dilute them to tan OD600 of 0.1 according to the cell dilution calculator formula described below: (a value of 0.1 is suggested for a volume of 100 uL
Cell dilution calculator: Volume of LB+Antibiotic to add in culture tube = [( Vcells * ODcell)/desired OD] - Vcells E.g. to dilute 5mL of cells at an OD600 of 0.3 to OD of 0.1 = [ (5mL*0.3)/0.1 - 5]  =10 mL
Add cells to 96-well plate
We pipetted 100 uL of liquid cultures ot OD600= 0.1 into the corresponding wells,following 96-well microplate layout and culture tubes label.
Set up the script in the plate reader and start experiment
For Pyocyanin experiments:
We set up a script on the BMG Labtech Omega control software to perform endpoint  absorbance measurements at 600 nm and fluorescent intensity measurements every 10 minutes for 24 hours (288 cycles for 600 seconds cycles time). Both absorbance and fluorescence measurements were recorded from the bottom of the plate. The excitation/emission wavelengths for GFP detection was set up at 475-510 nm. The plate reader was set up to shake at a double orbital mode at an RPM of 200.
Data analysis
We exported the raw data excel spreadsheet and imported into MATLAB software. We calculated average OD600, standard deviation and standard errors (STDEV/SQRT(4), where 4 was our number of replicates. We performed the same calculations for fluorescence values. We divided fluorescence values at each time point by the corresponding OD600 values, to normalise GFP expression by the number of cells. We calculated standard deviations and standard errors analogously for OD600. We then plotted OD600 as a function of time, GFP/OD600. From these plots we identified the time at which cell growth and GFP expression reached a steady state and called this time T_survive. We then took the GFP/OD600 values at T_survive for each construct and plot them against pyocyanin concentration to plot concentration curves, as shown in the “Results” section. Similar analysis was performed for ferrocyanide and ferricyanide and PMS experiments
Electrochemistry of Redox Modulators
Access All
Description
Protocol for cyclic voltammetry and amperommetry experiments
Notes
Procedure
Square-wave voltammetry was obtained with the setup descripted in the results section.
Solutions in the flask containing the reference and working electrode were identical to the solutions in the respective flasks containing the counter electrode.
Square-wave voltammetry was performed with the PalmSens 4-channel MultiEmStat3 between -0.3 and +0.5 Volt.
The counter electrodes contained 60% Carbon with a diameter of 5mm. Working electrodes contained 90% Carbon with a 2mm diameter. Salt bridges were consists of 3% agar containing 1 molar potassium chloride inside plastic tubing. The reference electrode was produced by placing a silver wire within 3% agar containing 3 molar potassium chloride inside plastic tubing.
Square wave voltammetry was performed with the following treatment settings: 10s equilibrium time, voltage step of 0.005V, amplitude 0.05V, frequency 25 Hz, the voltage beginning at -0.3V and ending at 0.5V.
Amperometry was performed in  a plate containing the final conditions: 10mM FCN(R), 2.5 uM Pyocyanin, 0.02% Sulfite, Ampicillin, Kanamycin and cells containing the PixCell construct with the following treatment settings: 10sec equilibrium time, time interval 0.05s run time up to 65000 seconds, V=+0.5.
Spatial Electronic Control of Gene Induction
Access All
Description
Given that we had determined the voltage by which pyocynanin becomes oxidised, we replicated that oxidation within a plate. Thereby turning on GFP expression by oxidising SoxR indirectly.
Procedure
Three 5mm holes were drilled into 9cm agar plates. Working, counter reference electrodes were prepared as described in the results section.
Electrodes were inserted into the plate penetrating 2mm and glued to the plate.
Solutions containing 10mM FCN(R), 2.5 mM Pyocyanin, 0.3% Sulfite, Ampicillin and Kanamycin were pitpetted into a 50ml falcon tube.
50ml of molten 1% agar was added to the falcon tube. Shaking ensured that the solution was well mixed. After mixing 22ml of the solution was pipetted onto the agar plate.
It was ensured that agar covered the electrodes fully.
After the agar was set, 100uL of cells containing the PixCell construct Deg Tag at an OD of 0.3. were plated onto the agar.
Afterwards the plate was turned upside down, closed and sealed using tape. The setup was then placed into a 37°C incubator.
Potentials were then applied overnight for 13h as described here.
To investigate fluorescence the agar was transferred onto a fresh plate without electrodes penetrating from the bottom. Fluorescence was measured using the Fujifilm FLA-5000 Fluorescent Image Analyser on the following settings: Laser wavelength: 250nm, filter: FITC, gradation: 65536 (16bit), resolution: 50uM. Images where then imported into imageJ for image analysis. Minimum brightness was set to 13867 in all images to discard small fluorescence values which correspond to background fluorescence.
Creating the Sox Library
Access All
Description
This protocol describes how to create a library of constructs varying two parts using BASIC assembly method. Following it we constructed a 48 constructs library, using 3 repeated parts and a variable promoter (pSoxS) with 8 variants, and a variable transcription factor (SoxR) with 6 variants (one of them unsubmitted to registry).
Notes
All the experiments for this part of the project were performed on E. coli DH5α, origin strain for DJ901, as this last one could not be made super-competent (109 cfu / μg pUC19  DNA) enough to transform BASIC assembly products. Individual parts were integrated in chloramphenicol resistant high copy plasmid pSB1C3_BASIC vectors and assemblies were performed in ampicillin resistant mid copy plasmid p15A vectors. All individual parts were synthesised and ordered as gblocks with BASIC prefix and suffix through IDT. The linkers used were provided by Biolegio and the adaptors were courtesy of Dr. Baldwin’s lab at Imperial College London.
Construct name
Promoter (pSoxS)
Transcription factor (SoxR)
Plasmid
Constitutive promoter
GFP
PC_A1
BBa_K2862006
BBa_K2862014
P15A-Amp
BBa_K2862019
Yes
PC_A2
BBa_K2862006
BBa_K2862015
P15A-Amp
BBa_K2862019
Yes
PC_A3
BBa_K2862006
BBa_K2862016
P15A-Amp
BBa_K2862019
Yes
PC_A4
BBa_K2862006
BBa_K2862017
P15A-Amp
BBa_K2862019
Yes
PC_A5
BBa_K2862006
BBa_K2862018
P15A-Amp
BBa_K2862019
Yes
PC_A6
BBa_K2862006
SoxR-8 (not submitted)*
P15A-Amp
BBa_K2862019
Yes
PC_B1
BBa_K2862007
BBa_K2862014
P15A-Amp
BBa_K2862019
Yes
PC_B2
BBa_K2862007
BBa_K2862015
P15A-Amp
BBa_K2862019
Yes
PC_B3
BBa_K2862007
BBa_K2862016
P15A-Amp
BBa_K2862019
Yes
PC_B4
BBa_K2862007
BBa_K2862017
P15A-Amp
BBa_K2862019
Yes
PC_B5
BBa_K2862007
BBa_K2862018
P15A-Amp
BBa_K2862019
Yes
PC_B6
BBa_K2862007
SoxR-8 (not submitted)*
P15A-Amp
BBa_K2862019
Yes
PC_C1
BBa_K2862008
BBa_K2862014
P15A-Amp
BBa_K2862019
Yes
PC_C2
BBa_K2862008
BBa_K2862015
P15A-Amp
BBa_K2862019
Yes
PC_C3
BBa_K2862008
BBa_K2862016
P15A-Amp
BBa_K2862019
Yes
PC_C4
BBa_K2862008
BBa_K2862017
P15A-Amp
BBa_K2862019
Yes
PC_C5
BBa_K2862008
BBa_K2862018
P15A-Amp
BBa_K2862019
Yes
PC_C6
BBa_K2862008
SoxR-8 (not submitted)*
P15A-Amp
BBa_K2862019
Yes
PC_D1
BBa_K2862009
BBa_K2862014
P15A-Amp
BBa_K2862019
Yes
PC_D2
BBa_K2862009
BBa_K2862015
P15A-Amp
BBa_K2862019
Yes
PC_D3
BBa_K2862009
BBa_K2862016
P15A-Amp
BBa_K2862019
Yes
PC_D4
BBa_K2862009
BBa_K2862017
P15A-Amp
BBa_K2862019
Yes
PC_D5
BBa_K2862009
BBa_K2862018
P15A-Amp
BBa_K2862019
Yes
PC_D6
BBa_K2862009
SoxR-8 (not submitted)*
P15A-Amp
BBa_K2862019
Yes
PC_E1
BBa_K2862010
BBa_K2862014
P15A-Amp
BBa_K2862019
Yes
PC_E2
BBa_K2862010
BBa_K2862015
P15A-Amp
BBa_K2862019
Yes
PC_E3
BBa_K2862010
BBa_K2862016
P15A-Amp
BBa_K2862019
Yes
PC_E4
BBa_K2862010
BBa_K2862017
P15A-Amp
BBa_K2862019
Yes
PC_E5
BBa_K2862010
BBa_K2862018
P15A-Amp
BBa_K2862019
Yes
PC_E6
BBa_K2862010
SoxR-8 (not submitted)*
P15A-Amp
BBa_K2862019
Yes
PC_F1
BBa_K2862011
BBa_K2862014
P15A-Amp
BBa_K2862019
Yes
PC_F2
BBa_K2862011
BBa_K2862015
P15A-Amp
BBa_K2862019
Yes
PC_F3
BBa_K2862011
BBa_K2862016
P15A-Amp
BBa_K2862019
Yes
PC_F4
BBa_K2862011
BBa_K2862017
P15A-Amp
BBa_K2862019
Yes
PC_F5
BBa_K2862011
BBa_K2862018
P15A-Amp
BBa_K2862019
Yes
PC_F6
BBa_K2862011
SoxR-8 (not submitted)*
P15A-Amp
BBa_K2862019
Yes
PC_G1
BBa_K2862012
BBa_K2862014
P15A-Amp
BBa_K2862019
Yes
PC_G2
BBa_K2862012
BBa_K2862015
P15A-Amp
BBa_K2862019
Yes
PC_G3
BBa_K2862012
BBa_K2862016
P15A-Amp
BBa_K2862019
Yes
PC_G4
BBa_K2862012
BBa_K2862017
P15A-Amp
BBa_K2862019
Yes
PC_G5
BBa_K2862012
BBa_K2862018
P15A-Amp
BBa_K2862019
Yes
PC_G6
BBa_K2862012
SoxR-8 (not submitted)*
P15A-Amp
BBa_K2862019
Yes
PC_H1
BBa_K2862013
BBa_K2862014
P15A-Amp
BBa_K2862019
Yes
PC_H2
BBa_K2862013
BBa_K2862015
P15A-Amp
BBa_K2862019
Yes
PC_H3
BBa_K2862013
BBa_K2862016
P15A-Amp
BBa_K2862019
Yes
PC_H4
BBa_K2862013
BBa_K2862017
P15A-Amp
BBa_K2862019
Yes
PC_H5
BBa_K2862013
BBa_K2862018
P15A-Amp
BBa_K2862019
Yes
PC_H6
BBa_K2862013
SoxR-8 (not submitted)*
P15A-Amp
BBa_K2862019
Yes
Figure 1. Table showing assembly parts combinations. RBS for GFP and SoxR were in the linker oligo, being RBS3 the one used for both GFP and SoxR.
*As SoxR-8 was original from C.amalonaticus a predicted human pathogen, we did not submit that part to the registry, however we worked with it because of its interesting properties.
Procedure
Parts assembly for library generation was performed as described in Storch, Casini et al., 2015. Briefly, linkers were prepared adding 49 µl of annealing buffer ( 10mM TRIS-HCl buffer pH7.9, 100mM NaCl, 10mM MgCl2 ) to 0.5 µl (1 µM) of linker and 0.5 µl (1 µM) of adapter oligos, and incubating at 95ºC for 1 min. Once linkers were ready they were mixed with the parts for the digestion/ligation reaction and incubated at 37ºC for 1 hour, 20ºC for 20 min, 65ºC for 20 min. Each 30 µl reaction had 3µl of ATP, 3µl of NEB4, 3µl BSA, 5µl of prefix linker (1µM), 5µl of suffix linker (1 µM), 1µl of BsaI (20 U/µl), 0.5 µl of T4 ligase (400 U/µl), 1µl DNA part (200 ng/µl) and water. Parts and linkers were mixed according to the following table.
Part
Promoter (for all 8 pSoxS)
Transcription factor (for all 6 SoxR)
Plasmid (p15A)
Constitutive promoter (Pfam2-J23101)
GFP
Linker prefix
LMP
UTR2-3
LMS
L5
UTR1-3
Linker suffix
UTR1-3
LMS
LMP
UTR2-3
L5
Figure 2. Linkers added to each part for assembly (linker notation is based on Stoch et al. 2015 analyzed linker data)
Digested parts were purified using Magbead purification in a AMPure XP magnetic purification kit according to manufacturer’s recommendations (Beckman Coulter). These were then assembled in 48 different combinations, to form our 48 different library constructs. Each 10vl assembly reaction contained 1µl of each purified DNA part with linkers, 1µl of NEB4 and 1µl of BSA. Assembly was performed incubating the mix at 50ºC for 45min.
Characterising the Sox Library
Access All
Description
The 48 parts library was characterised using the same procedures described for Pixcell construct characterisation (see previous protocol). The only difference is that instead of analysing each construct with 14 different inducer conscentrations we only analysed the base construct PC_A1. Then we characterised the other 47 constructs with only 3 inducer concentrations and fitted this data onto the sigmoidal curve created by PC_A1.
Notes
We determined that PC_A1 is our base construct as it is Pixcell construct in library format. This means that it has the same transcription factor (SoxR-0) and the same promoter (pSoxS-0) only that this one in Pfam1 promoter architecture. Therefore, this construct should give us the most similar results to Pixcell construct and show the effect of library architecture.
Procedure
Promoter induction assay
The best colonies from each succesful construct were characterised in an induction assay with pyocyanin. Assays were performed with different concentrations of pyocyanin (0-100 µM) and 100 µl of OD 0.1 cells in BMG Clariostat plate readers at 30º C.
Part growth assay
As some constructs did not grow any succesful colony we performed toxicity assays with the individual parts. Each part in pSB1C3-BASIC was transformed into E.coli DH5α strains using NEB high efficiency transformation protocol (C2987H/C2987I). Then succesful transformants were grown at 37ºC for 24 hours in a shaking BMG Clariostat plate reader and OD600 measurements were taken every 10min (see previous protocol in Pixcell construct characterization).
Biocontainment- Gp2 Growth Inhibition
Access All
Description
A biocontainment device to prototype one of our proposed applications. By placing this downstream of pSoxS we could inhibit cell growth in the oxidising conditions we can produce from our electrochemical set up. We envision this being used with our array in a biocontainment device in which any cells that spread to the edges of the device have their growth seized by an oxidising potential. This device could be used to prevent the accidental release of GMOs, much like how electric fences are used to restrain animals.
Procedure
Constructs were assembled by BASIC assembly into a low-copy number pSC101 backbone. SoxR was placed under the control of a medium strength RBS whereas Gp2 was constructed with an RBS library. Both RBSs were located within the BASIC linker sequences. Following assembly transformed cells were plated on LB-agar containing 10mM ferrocyanide and 0.02% sodium sulfite. Five biological replicates seeded from a single colony for each construct were grown out in LB containing 10mM ferrocyanide and 0.02% sodium sulfite for 6 hours. These cultures were then diluted to a final OD of 0.1. 5μl of each culture was added to 195μl LB solutions in a Greiner BioOne F-bottom 96-well plates to a final concentration of 2.5μl pyocyanin, 10mM ferricyanide and 0.02% sulfite for the OFF condition and to 2.5μl pyocyanin, 10mM ferrocyanide and 0.02% sulfite for the ON condition. Plates were incubated for 24 hours in a BMG Labtech FLUOstar at 37oC with absorbance readings at 600nm every 5 minutes.
official.imperial.igem@gmail.com
