<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Imperial_College_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Imperial College/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><DIV class="menu"><UL class="nav-list"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                      PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Project">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Mechanisms">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Applied_Design">APPLICATIONS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Future">FUTURE WORK</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Brainstorming">BRAINSTORM</A></LI></UL><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                    DRY LAB
                    </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Dryover">OVERVIEW</A></LI><LI><A href=" https://2018.igem.org/Team:Imperial_College/Model">MODELLING</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Software">SOFTWARE</A></LI><LI><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                        WET LAB
                        </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Wetover">OVERVIEW</A></LI><LI><A href=" https://2018.igem.org/Team:Imperial_College/Demonstrate">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Experiments">METHODS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Protocols">REFERENCES</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Measurement">BASIC</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Supplementary">SUPPLEMENTARY</A></LI><LI><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                        HUMAN PRACTICES
                        </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Communication">COMMUNICATION</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/IHP">INTEGRATED HP</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Public_Engagement">OUTREACH</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Collaborations">COLLABORATIONS</A></LI><LI><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                       PEOPLE
                        </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Sponsors">SPONSORS</A></LI><LI><A id="navbarDropdown" href="https://2018.igem.org/Team:Imperial_College/Judges">FOR JUDGES</A></LI><LI><A id="navbarDropdown" href="https://2018.igem.org/Team:Imperial_College/Journal">JOURNAL</A></LI></DIV></NAV></DIV><DIV class="container"><DIV class="content"><DIV class="titleimg"><H1>Modelling</H1></DIV><H3> Diffusion modelling</H3><P2>We wanted to know whether or not it was feasible to use electrodes on top of an agar plate to create a localised, well-defined oxidising or reducing environment. Modelling the diffusion of an arbitrary chemical species away from an electrode would give us a measure of both; the spatial resolution we could expect from the induced concentration profile and also whether or not we could feasibly create enough of a difference in the concentrations of oxidised and reduced species within that region that the cells would perform differently to cells far away from the electrode.

         </P2><H4 class="underbold"> Analytical Evaluation</H4><DIV onclick="expandAll('expt1p');" class="expandButton"><P class="expand expt1p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion">Purpose</DIV><DIV class="description"><P> We wanted to perform the initial evaluation for the concentration profile by hand to ensure we knew every assumption made in its entirety, and to maximise our understanding of the system. We did not want to simply use numerical modelling software without first understanding the key concepts and limitations within the system to be able to identify aberrant results and unjustified assumptions.</P></DIV><DIV type="button" class="accordion">Method</DIV><DIV class="panel expt1p"><P> The analytical model is a reworking of a system solved by Bell, Seelanan and O'Hare (2017). We initially modelled the diffusion of the chemical species away from the electrode by treating the electrode as a disc source with a constant flux of the redox molecule across its interface with the medium. We were able to use a pure diffusion model (without systematic drift) because the high ionic concentration of the medium shields any electric fields around the electrodes to a lengthscale of 0.8nm.  We then assumed that the agar medium itself was not generating additional molecules, and therefore that the only source of said molecules was the disc electrode. This assumption, combined with the conservation of species, allows us to describe the concentration of the species within the agar with the partial differential equation known as the Laplace equation. The Laplace equation describes diffusion processes in the limit of no time-dependence (the steady state reached at infinite time). </P><P>
The radially symmetric nature of the problem (the problem looks the same from any angle, as the electrode is a disc, not a rectangle or any other shape) allows us to use the 2D axisymmetric form of the Laplace equation, rather than the full 3D form, which makes solving the problem easier.

</P></DIV><DIV type="button" class="accordion">Assumptions</DIV><DIV class="panel expt1p" style="transition:max-height 1s ease-in-out;"><P> Due to the slow speed of diffusion, and the square root proportionality between time and diffusion distance, we assumed that the agar plate was arbitrarily deep, and that the concentration of the species tended to 0 as depth (z) increased. This can also be conceptualised as the diffusion “front” not reaching the bottom of the plate within the timescales that we are interested in. This assumption was also applied in the radial direction (ie the diffusion front does not reach the edge of the agar plate within the timescales that we are measuring within). This assumption was later confirmed through the use of Comsol modelling. We also constrained the problem to positive values of z, meaning that no molecules can leave the agar through its top surface. We also assumed that the chemical species analysed would not adsorb onto, or otherwise interact with, the agar medium. 
</P></DIV><DIV type="button" class="accordion">Mathematical Solution</DIV><DIV class="panel expt1p"><P> The full mathematical solution for the problem, as well as the justification for the assumption that the molecules are only transported by diffusion, is given in the <A class="btn btn-primary btn-lg" href="https://static.igem.org/mediawiki/2018/c/cf/T--Imperial_College--analytical_modelling_guide.pdf" role="button">appendix </A></P><P>However, we shall say here that the problem was formulated in cylindrical coordinates (again due to the radially symmetric nature of the problem), and non-dimensionalised both to simplify working and to generalise the problem to any desired scales (scales here meaning length scale of the problem and scale of diffusion, normalised for diffusion constant and flux through the electrode). The scalings are given below. </P><P><B>Formula 1.</B> Scalings used to non-dimensionalise the analytical model</P><P>
The problem was evaluated analytically and by hand up until the last step. That last step involved the evaluation of a complicated infinite integral, which was non-trivial. However, because the integrand tends to 0 as the variable tends to infinity, use of numerical integration in this case was justified to give the solution for the dimensionless concentration profile.</P><DIV class="caption"><P><B>Figure 2.</B> Visualisations of the integrand, or the distribution in Hankel space, for the analytic model</P></DIV><P>The exact magnitude of concentration for a given species at a specific flux can be found through simply reinserting the diffusion coefficient for that species. Interestingly, the diffusion coefficients for the species of interest are all reasonably similar, and so give reasonably similar results. Despite this, a wider range of diffusion coefficients was tested (between 5e-10m^2s^-1 and 1e-9m^2s^-1) to account for any differences from literature value due to interactions with the agar medium, or differences due to molecular interactions within the solution.
 </P></DIV><DIV type="button" class="accordion">Results and conclusions</DIV><DIV class="panel expt1p"><P> Observing the scalings for the non-dimensionalised problem, it can be seen that the lengthscale over which the concentration decays towards its bulk value is directly proportional to the electrode radius, and therefore scales linearly with it. The concentration at any given point is directly proportional to the flux through the electrode, and inversely proportional to the Diffusion coefficient. The half-maximal width of the concentration profile is as wide as 1.145 electrode diameters, which confirms the feasibility of generating spatially resolved regions of oxidising or reducing environment, with the resolution limited only by the electrode radius, which can be considered an engineering parameter. For a 2mm radius electrode, it can be seen that the concentration drops to below a quarter of its peak magnitude by 6mm from the centre, and below a sixth of its peak magnitude by 8mm. These miniscule concentrations would be easily negated by the effect of an electrode of opposite polarity centred 8mm from the centre of the first, whilst still bringing the two electrode’s regions of influence close enough together for image formation. Thus, 2mm radius electrodes, with centes spaced 8mm from each other, were selected for use in the PCB. It should be noted that, for a patterning application, electrode spacing at a given electrode size is a tradeoff between pixel distinction and the percentage of a surface under direct influence of an electrode. The engineer should attempt to maximise coverage in order to minimise the number of cells that are not driven hard into either ‘off’ or ‘on’.</P><DIV class="center"><P><B>Figure 3.</B> Analytic concentration profile for 2mm radius electrode</P><P><B>Figure 4.</B> Profiles for different diffusion coefficients, showing its behaviour as a scaling factor in steady state</P></DIV><DIV class="caption"><P><B>Figure 5.</B> Concentration distributions throughout the volume of interest</P></DIV></DIV><H4 class="underbold"> Numerical studies with Comsol Multiphysics</H4><DIV onclick="expandAll('expt2p');" class="expandButton"><P class="expand expt2p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion">Purpose</DIV><DIV class="description"><P> Comsol is a very high-level Computer-aided design (CAD) tool, which supports the analysis of many types of problems with wide-ranging physics, and the coupling of different physical interfaces together to compute intricate and complicated problems. It has built-in geometry and meshing tools, which allow the user to create model geometry in-package, without the need of external CAD software, and was therefore perfect for performing our simulations.
</P><P>
The manual calculation of the diffusion profile, although useful, is confined to steady state, and cannot provide the same level of tunability as a model developed in a dedicated simulation software package such as Comsol. Building a model in Comsol allowed us to check the results of the analytical calculations, to observe the evolution of the concentration profile over time, and to quickly and easily analyse different geometries and test the assumptions made when formulating the analytical problem. 
</P></DIV><DIV type="button" class="accordion">Basic 2D axisymmetric geometry model</DIV><DIV class="panel expt2p" style="transition:max-height 1s ease-in-out;"><P> This model was constructed in order to test assumptions used in the analytical solution, specifically the assumptions that the diffusion “front” would not reach the edges of the plate within the relevant timescale. To do this, we used a time-dependent study with the 2D axisymmetric geometry pictured below, with the disc positioned flush to the surface.
</P><DIV class="caption"><P><B>Figure 6.</B> Basic 2D axisymmetric geometry, where the disc electrode (small circle) is flush to the surface. The &quot;wedge&quot; cut out of the image allows a visualisation of the concentration profile as a function of depth.</P></DIV>Results and Conclusions

As can be seen from the figure below, after 20 minutes, the concentration front was nowhere near the bottom of the plate, let alone the sides of the plate. Note that the edge of the domain shown is not the edge of the plate; the domain shown has a radius of 1cm, and the true edge of the plate would be much further from the electrode than this.
The model corroborates the analytical solution, showing a very steep concentration gradient at the edge of the electrode, again making a good argument for the achievement of spatial resolution in a patterning application such as ours. In addition,  the uniform density of flux lines emanating from the electrode, and the manner in which all of them, even the central ones, move outwards radially, shows efficient use of the electrode area for distribution of the molecule.
<DIV class="center"><P><B>Figure 7.</B> Volumetric concentration distribution after 20 minutes</P></DIV><DIV class="caption"><P><B>Figure 8.</B> Flux lines showing the paths of diffusion from the electrode surface</P></DIV><P>From this model, we can conclude that the assumptions in the analytical solution regarding the diffusion “front” are valid, and that it is valid to treat the dish in the analytical solution as both arbitrarily deep and wide. Additionally, it is reasonable to think that localised oxidised or reduced volumes can be induced in the agar plate in a period of 20 minutes by the use of electrodes, which was confirmed experimentally.

</P></DIV><DIV type="button" class="accordion">Recessed 2D axisymmetric geometry model</DIV><DIV class="panel expt2p"><P> When a manufacturer produces a PCB with pads on the underside, such as the one we ordered, the pads are plated, and then the underside of the PCB lacquered. This means that the actual surface of the pad is slightly recessed from the surface of the rest of the PCB. Therefore, we wanted to test how a small recession such as this would affect the concentration profile around the electrode. We again used 2D axisymmetric geometry, shown below, where the electrode disc is the top surface of the recessed cylinder.

</P><DIV class="caption"><P><B>Figure 9.</B> 2D axisymmetric geometry where the disc electrode (small circle) is recessed from the surface. The &quot;wedge&quot; cut out of the image allows a visualisation of the concentration profile as a function of depth.</P></DIV>Results and Conclusions


The simulation shows that even very small recessions drastically reduce the diffusive flux away from the surface. This is because when the surface is recessed, even by a few micrometres, all flux of the species must be perpendicular to the surface. This constraint drastically limits the space into which the molecules can diffuse, as there is a concentration gradient only in the z-direction, as opposed to in both the r- and z-directions in the flush case. This in turn limits the diffusive flux into the surrounding environment, and results in an increased concentration at the electrode surface due to limited diffusion out of the recession. 

<DIV class="caption"><P><B>Figure 10.</B> Animation showing the behaviour of the concentration profile for different recession depths, between 1μm and 0.1mm</P></DIV><P>The flux lines emanating from the electrode in this model show how the recession, and hence the constriction of flux, adversely affect the efficiency of molecular transport. Comparing the flux line plots to those of the non-recessed model, it can be seen how they are significantly less uniform across the electrode, and cluster around its edge, as this is where the majority of the molecular flux occurs. This can also be seen in the lack of divergence of the central flux lines; molecules diffusing from the centre of the electrode can only diffuse downwards.</P><P><B>Figure 11.</B> Flux lines showing diffusion paths from a recessed electrode</P>It should be noted that the decreased diffusive flux into the wider environment is not necessarily a bad thing, as although it limits the volume a particular electrode can affect, it increases the sharpness of the concentration profile (and hence gives a sharper on/off transition for induced cells at the boundary), and increases the concentration directly underneath the electrode, resulting in greater fold induction of any electro-inducible organisms in this region. Indeed, the magnitude of recession of electrodes within their mounting could be approached as an engineering specification, with a smaller recession resulting in wider but less well-defined fields of influence, and a larger recession resulting in smaller, sharper and more highly induced fields of influence.

<P><B>Figure 12.</B> Animation showing the concentration profiles across both the basic and recessed geometries, specifically the significant difference between the basic geometry and 1μm recession</P></DIV><H4 class="underbold"> Transcription-Translation Modelling</H4><DIV onclick="expandAll('expt3p');" class="expandButton"><P class="expand expt3p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion">Purpose</DIV><DIV class="description"><P> In order to better understand the system we are engineering, we constructed a deterministic transcription-translation model of the gene circuit using ordinary differential equations in Matlab. This model allowed us to explore the dynamics of the system, to better understand the parameters that the system would be sensitive to, to use curve fitting tools to estimate unknown parameters, and to check whether the constructs were behaving as expected. 
</P></DIV><DIV type="button" class="accordion">Method</DIV><DIV class="panel expt3p"><P> Using parameters from literature, we constructed a model consisting of 4 coupled ODEs: one for transcription of soxR, one for translation of soxR, one for transcription of GFP, and one for translation of GFP. SoxR production was modelled as constitutive, as there was insufficient evidence in literature to support feedback control of the transcription of soxR. 

The binding of soxR to its promoter was modelled using the phenomenological transfer function given below. This transfer function has the effect of combining both the repressive and activatory modes of Hill functions into one, to account for the binding of both oxidised (activatory) and reduced (repressive) soxR molecules. There was no clear data to be found regarding the cooperativity of the soxR transcription factor; however, it is known that soxR forms dimers, so the maximum cooperativity is constrained at 2. Therefore, we decided to estimate the cooperativity as that of the dimeric tetR transcriptional regulator.  

TABLE
Parameter		Value			Source
</P><DIV class="caption"><P><B>Formula 2.</B> Transfer function describing SoxR promoter activity as a function oxidised and reduced SoxR</P></DIV>


The reaction between Pyocyanin and SoxR was modelled as a fast, reversible reaction with the following equation, where K represents the reciprocal of the concentration at half maximal GFP, and m is a measure of the gradient of the resulting sigmoid. Thus, the full set of governing equations for the system is below.

<DIV class="caption"><P><B>Formula 3.</B> Full governing system for the Pixcell construct</P></DIV></DIV><DIV type="button" class="accordion">Assumptions</DIV><DIV class="panel expt3p" style="transition:max-height 1s ease-in-out;"><P> Because Pyocyanin oxidises SoxR directly, we assumed that the oxidation of SoxR by Pyocyanin was fast compared to the timescales of the biological processes in the model, which are limited by the diffusion of large biological complexes. We therefore modelled the quantities of oxidised and reduced SoxR as instantaneously created from the total available pool of soxR, splitting the pool in the following way. 

</P><P><B>Formula 4.</B> Proportions of oxidised and reduced SoxR as a function of Pyocyanin concentration</P>

We also assumed that the cells had been left to grow to steady state before the simulation. Thus, as the model does not contain any feedback or otherwise complex dynamic features with respect to SoxR transcription and translation, the ODEs describing SoxR do not affect the rest of the system. Thus, they can be set to their steady state value before running the model. We know from literature that the steady state expression of our SoxR gene in its parent strain is 75 molecules per cell, so we can multiply this by our copy number, and set the total SoxR in our model to this value. This removes the need for parameters for the transcription and translation of SoxR, and the need for the governing equations themselves, but these were left in for the sake of consistency and in case future work requires analysis of soxR dynamics.

<P><B>Table 1.</B> Known model parameters and their sources</P><TABLE><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR><TR><TD>Nplasmid</TD><TD>pMB1 ORI</TD><TD>15</TD><TD>1</TD><TD>(Morgan, 2018)</TD></TR><TR><TD>PpsoxS</TD><TD>Promoter strength of psoxS        </TD><TD>5.8 x 10^-3</TD><TD>1</TD><TD>Calculated from average mRNA molecules per cell in E.coli, and the average half-life</TD></TR><TR><TD>Kox</TD><TD>Dissociation constant of Oxidised SoxR</TD><TD>4.5 x 10 ^-10</TD><TD>M</TD><TD> (Chen et al., 2018)</TD></TR><TR><TD>Kred</TD><TD>Dissociation constantof Reduced SoxR</TD><TD>1.1 x 10 ^-10</TD><TD>M</TD><TD> (Chen et al., 2018)</TD></TR><TR><TD>dmRNA-GFP</TD><TD>Average degradation rateof mRNA in E.Coli based on half-life</TD><TD>2.5x10^-3</TD><TD>1/s</TD><TD>(Chen et al., 2018)</TD></TR><TR><TD>rbsGFP</TD><TD>RBS strength of GFP</TD><TD>0.183</TD><TD>1/s</TD><TD>RBS strength calculator</TD></TR><TR><TD>Dgfp</TD><TD>degedration rate of GFP</TD><TD>6.9X10^(-5)</TD><TD>(Halter et al., 2018)</TD></TR><TR><TD>SoxR @SS</TD><TD>Steady-state soxR    in wild-type species</TD><TD>75</TD><TD>1</TD><TD> (Pomposiello and Demple, 2018)</TD></TR><TR><TD>n</TD><TD>Hill coefficient for TetR</TD><TD>1.6</TD><TD>1</TD><TD>(Ahn et al., 2018)</TD></TR></TBODY></TABLE></DIV><H4 class="underbold"> Curve Fitting</H4><DIV onclick="expandAll('expt5p');" class="expandButton"><P class="expand expt4p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion">Description</DIV><DIV class="description"><P>For every model we wished to fit to data, we developed an objective function to minimise the square difference between the model and the experimental data by varying its parameters. We then used optimisation algorithms to find the optimum parameter set.

</P></DIV><DIV type="button" class="accordion">Construct 1 Fitting</DIV><DIV class="panel expt5p"><P> Most notably, this method of curve fitting was used to estimate parameters for the reaction between Pyocyanin and SoxR, which were unknown, as well as the basal expression, the GFP:RFU scale of the measurements, and the degradation rate of GFP, given the transcription/translation rates sourced from literature.
</P><P>
In order to utilise the entire available dataset, rather than merely the steady state GFP values for each Pyocyanin concentration, we constructed an objective function that could generate ODE plots. The reasoning behind using this more complicated strategy is as follows.
</P><P>
The steady state GFP reading can be computed using Formula 5 below. This is done by assuming that the system has been left for a long time and has settled, so that no quantity within the system is varying with time. Thus, all derivative terms in the model are zero, and so can be rearranged and combined together into one equation. Observing this steady state equation, it can be seen that there is orthogonality in all of the parameters in Table 2, besides the Scale and the degradation rate of GFP; a twofold increase in the Scale parameter can be compensated for by a twofold increase in the GFP degradation rate. However, the same is not true in the time-dependent model. It can be seen from Figures 12 and 13 that a twofold increase or decrease in these parameters leads to a differing transient response, and the same steady state response. Thus, an objective function that utilises transient data can fit both the GFP degradation rate and the GFP:RFU scale simultaneously.
</P><P><B>Formula 5.</B> Steady State GFP as a function of Pyocyanin</P><P><B>Table 2.</B> Parameters to be fitted, with their descriptions</P><TABLE><TBODY><TR><TH>Parameter</TH><TH>Description</TH></TR><TR><TD>Scale</TD><TD>Scaling to normalise RFU to number of GFP molecules</TD></TR><TR><TD>K</TD><TD>Concentration of oxidised Pyocyanin that leads to half of the soxR molecules being oxidised and half reduced</TD></TR><TR><TD>m</TD><TD>Stoichiometry of the reaction between Pyocyanin and soxR</TD></TR><TR><TD>Basal</TD><TD>Basal (leaky) expression of GFP</TD></TR><TR><TD>dGFP</TD><TD>Untagged degradation rate of GFP</TD></TR><TR><TD>dGFP_tagged</TD><TD>Degradation rate of degradation tagged GFP</TD></TR><TR><TD>Delay</TD><TD>Delay in cellular response to oxidising conditions</TD></TR></TBODY></TABLE><P><B>Figure 12.</B> Time-dependent curves show the non-equivalence of parameters</P><P><B>Figure 13.</B> Differential transient behaviour highlighted by the removal of basal expression</P>

It should be noted that, due to the steady state nature of the soxR dynamics in the fitted data, we can remove the ODEs describing soxR transcription and translation in order to decrease the significant computational cost associated with running many ODE solvers for each optimisation iteration. The governing equations to which the optimiser is fitting are then given below.

<P><B>Formula 6.</B> Reduced set of equations to which are used by the optimisation algorithms to fit experimental data</P></DIV><DIV type="button" class="accordion">Method</DIV><DIV class="panel expt5p" style="transition:max-height 1s ease-in-out;"><P> Convergence of optimisation algorithms is of vital importance, and plotting the norm of the residuals (ie the dataset-wide error of the model) can be qualitatively very informative. An optimisation problem can be said to have converged if the norm of the residuals tends towards a certain value, at which the parameters allow the model to best fit the data. It should be noted that “best fit” is a misnomer; in practice, it can be very difficult to find a global minimum for the objective function, and so the computer can only find the best possible parameters close to the region it starts within. It can be seen from Figure 14 that convergence is obtained, and from Figure 15 that a reasonable fit of the data is achieved. 

</P><P><B>Figure 14.</B> Convergence of model utilising the entire available dataset</P><P><B>Figure 15.</B> Model function with fitted parameters compared to experimental data</P>



However, upon inspection of the residual plot, below, it can be seen that there is noticeable patterning; there are regions which are consistently above or below zero. For a good model fit, the residuals should be randomly distributed about zero. 

<P><B>Figure 16.</B> Dataset-wide plot of the residuals of the fitted model, showing undesirable clustering in their distribution</P>


One noticeable feature of the residual plot is the overestimation of GFP expression at high Pyocyanin concentrations. Upon observation of the OD600 data (LINK TO OD600), it can be seen that, for Pyocyanin concentrations for 10μM and above, cell growth is significantly hindered. Although the model normalises for cell number, this lower OD600 is a sign of struggling cells, which activate stress pathways and downregulate non-essential genes, such as the synthetic construct. Therefore, the fit, and therefore the accuracy of extracted parameters, can be improved by omitting this data. Observing the convergence of the fitting algorithm for data up to 5μM, it can be observed that the norm of the residuals is drastically reduced. This signifies a better fit to the data. Note that this does not necessarily mean a better model, but is a useful measure nonetheless.

<P><B>Figure 17.</B> Convergence of the model for healthy cell data, noting the improvement of fit</P><P><B>Figure 18.</B> Model function with parameters fitted to healthy cell data, compared to experimental data</P><P><B>Figure 19.</B> Healthy cell model residual plot, noting reduced clustering</P>

Observing the plot of the residuals for the reduced dataset, additional patterning can be seen in low time points.In order to account for this bias, data could be normalised and then fitted. For example, in the plots below, the average of the first six data points are subtracted uniformly from the data. Although this provides a small reduction in the value of the norm of the residuals upon model convergence, the fit is not markedly improved, and the residuals show higher clustering than before the normalisation. Thus, we concluded that the model cannot be easily normalised without significantly altering the data to improve the fit, which would invalidate the model and destroy any predictive power it might have. We therefore conclude that our best estimates of the parameters are found by fitting the model to the unaltered, healthy cell data.


<DIV class="center"><P><B>Figure 20.</B> Convergence for healthy cells, normalised to account for correlation between initial fluorescence and Pyocyanin concentration</P><P><B>Figure 21.</B> Model fit for healthy cells, normalised for initial variable bias</P></DIV><DIV class="caption"><P><B>Figure 22.</B> Plot of residuals for normalised healthy cell data, noting higher clustering than the non-fitted group</P></DIV></DIV><DIV type="button" class="accordion">Results</DIV><DIV class="panel expt5p" style="transition:max-height 1s ease-in-out;"><P>The fitted parameters returned by the curve fitting regime are given in the table below.

<TABLE><TBODY><TR><TH>Parameter</TH><TH>Description</TH></TR><TR><TD>Scale</TD><TD>Scaling to normalise RFU to number of GFP molecules</TD></TR><TR><TD>K</TD><TD>Reciprocal of the concentration of oxidised Pyocyanin that leads to half of the soxR molecules being oxidised and half reduced</TD></TR><TR><TD>m</TD><TD>Stoichiometry of the reaction between Pyocyanin and soxR</TD></TR><TR><TD>Basal</TD><TD>Basal (leaky) expression of GFP</TD></TR><TR><TD>dGFP</TD><TD>Untagged degradation rate of GFP</TD></TR><TR><TD>dGFP_tagged</TD><TD>Degradation rate of degradation tagged GFP</TD></TR><TR><TD>Delay</TD><TD>Delay in cellular response to oxidising conditions</TD></TR></TBODY></TABLE></P><P> Several interesting observations can be made from these estimated parameters. The basal expression fitted, although high, is not exactly unexpected due to the leakiness of the system, and could be a reasonable approximation for the real value. The untagged degradation rate for GFP is within 5% of the literature value, which, although interesting, cannot be taken as a direct measure of quality of the model due to the noise in the data, and the inherent tolerances and context dependencies of any metabolic parameter. The degradation rate of the tagged GFP was significantly slower than expected. To check this, the steady-state models are plotted against the final timepoint from the data. This model suggests that the degradation tag only increases the degradation rate by approximately  a factor of 1.2, not by the expected factor of 8 (Tschirhart et al. 2017). On the basis of this unexpected result, the construct was sent for sequencing, and a mutation was confirmed within the degradation tag. It was decided, however, to keep using the mutated sequence, as it provided a good balance between reducing background and still retaining a measurable fluorescence when induced.

</P><P><B>Figure 23.</B> Plot of model fits for tagged and untagged constructs at steady state</P></DIV><H3 class="center">Analytical Modelling Guide</H3><H5>References</H5><UL><LI>Ahn, S., Tahlan, K., Yu, Z. and Nodwell, J. (2018). Investigation of Transcription Repression and Small-Molecule Responsiveness by TetR-Like Transcription Factors Using a Heterologous Escherichia coli-Based Assay.</LI><LI>Bell, C., Seelanan, P. and O'Hare, D. (2017). Microelectrode generator–collector systems for electrolytic titration: theoretical and practical considerations. The Analyst, 142(21), pp.4048-4057.</LI><LI>Chen, H., Shiroguchi, K., Ge, H. and Xie, X. (2018). Genome-wide study of mRNA degradation and transcript elongation in Escherichia coli.</LI><LI>Halter, M., Tona, A., Bhadriraju, K., Plant, A. and Elliott, J. (2018). Automated live cell imaging of green fluorescent protein degradation in individual fibroblasts.</LI><LI>Kobayashi, K., Fujikawa, M. and Kozawa, T. (2018). Binding of Promoter DNA to SoxR Protein Decreases the Reduction Potential of the [2Fe–2S] Cluster.</LI><LI>Morgan, K. (2018). Plasmids 101: Origin of Replication. [online] Blog.addgene.org. Available at: https://blog.addgene.org/plasmid-101-origin-of-replication [Accessed 18 Oct. 2018].</LI><LI>Pomposiello, P. and Demple, B. (2018). Redox-operated genetic switches: the SoxR and OxyR transcription factors.</LI><DIV class="footer"><DIV class="socialicons"><H4> official.imperial.igem@gmail.com</H4></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>