<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Imperial_College_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Imperial College/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-container"><NAV><DIV class="menu"><UL class="nav-list"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                      PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Project">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Mechanisms">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Applied_Design">APPLICATIONS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Future">FUTURE WORK</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Brainstorming">BRAINSTORM</A></LI></UL><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                    DRY LAB
                    </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Dryover">OVERVIEW</A></LI><LI><A href=" https://2018.igem.org/Team:Imperial_College/Model">MODELLING</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Software">SOFTWARE</A></LI><LI><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                        WET LAB
                        </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Wetover">OVERVIEW</A></LI><LI><A href=" https://2018.igem.org/Team:Imperial_College/Demonstrate">RESULTS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Experiments">METHODS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Protocols">REFERENCES</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Measurement">BASIC</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Supplementary">SUPPLEMENTARY</A></LI><LI><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                        HUMAN PRACTICES
                        </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Communication">COMMUNICATION</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/IHP">INTEGRATED HP</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Public_Engagement">OUTREACH</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Collaborations">COLLABORATIONS</A></LI><LI><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                       PEOPLE
                        </A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:Imperial_College/Sponsors">SPONSORS</A></LI><LI><A id="navbarDropdown" href="https://2018.igem.org/Team:Imperial_College/Judges">FOR JUDGES</A></LI><LI><A id="navbarDropdown" href="https://2018.igem.org/Team:Imperial_College/Journal">JOURNAL</A></LI></DIV></NAV></DIV><DIV class="container"><DIV class="content"><DIV class="titleimg"><H1>Methods</H1></DIV><H3>Contents</H3><DIV class="contents"><UL><LI><A href="#expt1">Assembling the Pixcell Constructs</A></LI><LI><A href="#expt2">Characterizing Pixcell Constructs</A></LI><LI><A href="#expt3">Electrochemistry of Redox Modulators</A></LI><LI><A href="#expt4">Spatial Electronic Control of Gene Induction</A></LI><LI><A href="#expt5">Creating the Sox Library</A></LI><LI><A href="#expt6">Characterising the Sox Library</A></LI><LI><A href="#expt7">Biocontainment- Gp2 Growth Inhibition</A></LI></UL></DIV><H3 class="marginbottom">Experiments</H3><H4 class="underbold"> Assembling the Pixcell Constructs</H4><DIV onclick="expandAll('expt1p');" class="expandButton"><P class="expand expt1p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>This protocol describes how to create the Pixcell construct and Pixcell construct with a degradation tag using Golden Gate assembly. Creating each of the constructs takes approximately two days </P></DIV><DIV type="button" class="accordion">Materials</DIV><DIV class="panel expt1p"><P><LI>pBR322</LI><LI>E.coli MG1655 genomic DNA</LI><LI>GFP Storage Vector</LI><LI>10µM of Primers</LI><LI>Q5 polymerase (New England Biolabs, #M0491) </LI><LI>1% Agarose Gel</LI><LI>TAE Buffer</LI><LI>Monarch® DNA Gel Extraction Kit (NEB, #T1020S)</LI><LI>Golden Gate Assembly Mix (NEB, E1600S)</LI><LI>DJ901 and Turbo Cells</LI><LI>KCM</LI><LI>Phusion polymerase (NEB, #M0530S) </LI><LI>T4 DNA ligase (NEB, #M0202) </LI></P></DIV><DIV type="button" class="accordion">Notes</DIV><DIV class="panel expt1p"><TABLE class="tg"><TBODY><TR><TH class="tg-7q3x">Part </TH><TH class="tg-0lax">Description</TH></TR><TR><TD class="tg-m9r4">pBR322</TD><TD class="tg-m9r4">A medium copy vector encoding ampicillin resistance with a pMB1,origin of replication and a mcherry cassette with BsaI sites on either end.</TD></TR><TR><TD class="tg-m9r4">SoxRS</TD><TD class="tg-m9r4">The gene encoding the transcription factor Sox R and the bidirectional promoter pSoxS/pSoxR which it regulates</TD></TR><TR><TD class="tg-m9r4">GFP</TD><TD class="tg-m9r4">Superfolded GFP which is used as a reporter gene to be placed under the control of inducible side of the promoter, pSoxS</TD></TR></TBODY></TABLE></DIV><DIV type="button" class="accordion">Procedure</DIV><DIV class="panel expt1p"><TABLE class="tg"><TBODY><TR><TH class="tg-958c">Name</TH><TH class="tg-958c">Template</TH><TH class="tg-958c">Direction</TH><TH class="tg-958c">Function</TH><TH class="tg-958c">Tm(°C)</TH><TH class="tg-958c">Sequence</TH></TR><TR><TD class="tg-mfhl">PPC1</TD><TD class="tg-mfhl">MG655 Genome</TD><TD class="tg-mfhl">Forward</TD><TD class="tg-mfhl">Amplificationof SoxRS regulon</TD><TD class="tg-mfhl">67.1</TD><TD class="tg-mfhl">CTAGTGGGTCTCCCTCGAGAGAAAGACAAAGACCGG</TD></TR><TR><TD class="tg-mfhl">PPC2</TD><TD class="tg-mfhl">MG1655 Genome</TD><TD class="tg-mfhl">Reverse</TD><TD class="tg-mfhl">Amplification of SoxRS regulon</TD><TD class="tg-mfhl">64</TD><TD class="tg-mfhl">GTCGATGGTCTCCCATAAATCTGCCTCTTTTCAGTG</TD></TR><TR><TD class="tg-mfhl">PPC3</TD><TD class="tg-mfhl">GFP Storage Vector</TD><TD class="tg-mfhl">Forward</TD><TD class="tg-mfhl">Amplification of GFP</TD><TD class="tg-mfhl">65</TD><TD class="tg-mfhl">GTCGATGGTCTCGTATGCGTAAAGGCGAAGAA</TD></TR><TR><TD class="tg-mfhl">PPC4</TD><TD class="tg-mfhl">GFP Storage Vector</TD><TD class="tg-mfhl">Reverse</TD><TD class="tg-mfhl">Amplification of GFP</TD><TD class="tg-mfhl">68</TD><TD class="tg-mfhl">CTAGTGGGTCTCGGGACAGTAGCGAAAAAACCCCG</TD></TR><TR><TD class="tg-mfhl">PPC5</TD><TD class="tg-mfhl">PixCell Construct</TD><TD class="tg-mfhl">Forward</TD><TD class="tg-mfhl">Addition of a degradation tag</TD><TD class="tg-mfhl">65</TD><TD class="tg-mfhl">GCAAACGACGAAACTACGCTTTAGTAATGATACTAGAGCGCAAAAAACCCC</TD></TR><TR><TD class="tg-mfhl">PPC6</TD><TD class="tg-mfhl">PixCell Construct</TD><TD class="tg-mfhl">Reverse</TD><TD class="tg-mfhl">Addition of a degradation tag</TD><TD class="tg-mfhl">65</TD><TD class="tg-mfhl">CTAAAGCGTAGTTTCGTCGTTTGCCTTATACAGCTCGTCCATACCGTGG</TD></TR></TBODY></TABLE><OL><LI>PCR Reactions</LI><DIV class="underbold">SoxRS regulon Amplification</DIV><P>The PCR was carried out using the primers PPC1 and PPC2 and Q5 polymerase (New England Biolabs) following a protocol of 20 seconds of initial denaturation at 98 ºC, 30 cycles of 98 °C for 10 seconds, 59.5 °C for 30 seconds, and 72 °C for 30 seconds, and a final extension at 72 °C for 5 minutes.
<LI>GFP Amplification.</LI></P><P>The PCR was carried out using the primers PPC3 and PPC4 and Q5 polymerase (New England Biolabs) following a protocol of 20 seconds of initial denaturation at 98 ºC, 30 cycles of 98 °C for 10 seconds, 61.5 °C for 30 seconds, and 72 °C for 30 seconds, and a final extension at 72 °C for 5 minutes.</P><LI>Gel Electrophoresis</LI><P>Both PCR product were run on a 1% agarose gel and extracted using the New England Biolabs gel extraction kit using the recommended protocol.</P><LI>Golden Gate Assembly</LI><P>The two amplicons were cloned into the pBR322 vector by adding 75ng of pBR322 and 30.1ng of SoxRS regulon and 23.4ng of GFP amplicon to the golden gate assembly mix (NEB, E1600S). Following the NEB protocol the mixture was incubated at 37°C for 1 hr and at 55°C, 5 mins.</P><LI>Transformation</LI><P>The PixCell construct was transformed into DJ901 and Turbo cells using the KCM heat shock protocol.</P><LI>Addition of the Degradation Tag</LI><P>Turbo cells containing the PixCell construct were grown overnight and miniprepped following the NEB protocol in order to obtain template DNA for the following PCR reaction.
The PCR was carried out using the primers PPC5 and PPC6 and Phusion polymerase (New England Biolabs) with 3% DMSO. Following a protocol of 30 seconds of initial denaturation at 98 ºC, 30 cycles of 98 °C for 10 seconds, 65 °C for 30 seconds, and 72 °C for 3 minutes, and a final extension at 72 °C for 10 minutes. These primers produce complementary overhangs which when ligated together add a degradation tag we designed to the 3’ end of the coding region of GFP.</P><LI>Ligation</LI><P>The ends of the PCR amplicon produced from the primers PPC5 and PPC6 were ligated using T4 DNA ligase (NEB) following the recommended NEB protocol.</P><LI>Transformation</LI><LI><P>The PixCell construct with deg tag was transformed into DJ901 cells using the KCM heat shock protocol.</P><DIV id="expt2"><H4 class="underbold"> Characterizing Pixcell Constructs</H4><DIV onclick="expandAll('expt2p');" class="expandButton"><P class="expand expt2p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>This protocol describes how to measure E. coli growth and GFP expression over time with a plate reader, using the optical density at 600 nm as signal for cell density and excitation and emission wavelengths of 475.
All characterization was performed using the FLUOstar BMG Labtech in Greiner Bio-One F-bottom dark plates with clear bottom, to decrease fluorescence background.
Each cycle of this experiment takes two days and has to be repeated two days, in order to have 2 biological and 2 technical replicates, for a total of 4 days and 4 96-well plates used.
</P></DIV><DIV type="button" class="accordion">Materials</DIV></LI><LI>Autoclaved LB media</LI><LI>Autoclaved 10x concentated LB media</LI><LI>500 mM pyocyanin stock solution (from Sigma-Aldrich, CAS: No. 86-66-5)</LI><LI>125 mM potassium ferricyanide stock solution (from Sigma-Aldrich, CAS: No. 13746-66-2)</LI><LI>500 mM ferrocyanide stock solution (from Sigma-Aldrich, CAS 14459-95-1 )</LI><LI>20% Sodium Sulfite Solution</LI><LI>
Agar plates with colonies of E. coli DJ901 strains with respective constructs: PixCell Patterning Circuit 1 (BBa_K2862021). PixCell Patterning Circuit 2 (BBa_K2862022), negative control (DJ901 WT) and positive control (constitutively expressed GFP).</LI><LI>37°C shaking incubator</LI><LI>BMG Labtech FLUOstar plate reader</LI><LI>4x Greiner BioOne 96-F 96-well microplate (black with clear bottom)</LI><LI>14 mL culture tubes</LI><LI>autoclaved eppendorfs</LI><DIV type="button" class="accordion">Notes</DIV><DIV class="panel expt2p"><P>Work in very sterile conditions. To prevent contaminations in the blanks, use triple antibiotics.

Store pyocyanin and ferrocyanide and ferricyanide solutions in the freezer until use.

We used 10x concentrated LB in order not to dilute nutrients at higher pyocyanin concentration; this was back diluted with autoclaved water. Mastemixes have double the working concentrations, as 100 uL of solution is then diluted with 100 uL of liquid cultures.

                        </P></DIV><DIV type="button" class="accordion">Procedure</DIV><DIV class="panel expt2p"><OL><LI>Grow starter liquid cultures of cells  (Day 0, 10-15 min preparation +  overnight)</LI><P>Working in very sterile conditions, take 5 14mL culture tubes and prepare them according to table below: </P><TABLE class="tg"><TBODY><TR><TH class="tg-1wig">Label</TH><TH class="tg-1wig"><I>E. coli</I> strain</TH><TH class="tg-1wig">Construct</TH><TH class="tg-1wig">Antibiotic  (Ab)</TH><TH class="tg-1wig">V Ab (uL)</TH><TH class="tg-1wig">V LB (mL)</TH></TR><TR><TD class="tg-0lax">PC1</TD><TD class="tg-0lax">DJ901</TD><TD class="tg-0lax">BBa_K2862021</TD><TD class="tg-096r">Kan + Amp</TD><TD class="tg-0lax">5 + 5</TD><TD class="tg-0lax">5</TD></TR><TR><TD class="tg-0lax">PC2</TD><TD class="tg-0lax">DJ901</TD><TD class="tg-0lax">BBa_K2862022</TD><TD class="tg-0lax">Kan + Amp</TD><TD class="tg-0lax">5 + 5</TD><TD class="tg-0lax">5</TD></TR><TR><TD class="tg-0lax">-</TD><TD class="tg-0lax">DJ901</TD><TD class="tg-0lax">none</TD><TD class="tg-0lax">Kan</TD><TD class="tg-0lax">5 </TD><TD class="tg-0lax">5</TD></TR><TR><TD class="tg-0lax">+</TD><TD class="tg-0lax">DJ901</TD><TD class="tg-0lax">GFP_Boo</TD><TD class="tg-0lax">Kan + Chl</TD><TD class="tg-0lax">5 + 5</TD><TD class="tg-0lax">5</TD></TR><TR><TD class="tg-0lax">Blanks</TD><TD class="tg-0lax">None</TD><TD class="tg-0lax">None</TD><TD class="tg-0lax">Kan + Amp + Chl</TD><TD class="tg-0lax">5 + 5</TD><TD class="tg-0lax">5</TD></TR></TBODY></TABLE><P>With a P20 pipette tip we picked the desired colonies from an agar plate and released the contaminated tip in the culture tip (for PC1, PC2 and Pos the picked colonies appeared fluorescence under a UV transilluminator).  We then place inoculated culture tubes in a 37° C in a shaking incubator overnight. For faster growth the angle between the vertical axis of the tube and the shaking plane of the incubator was set at 45°)</P><LI>Prepare mastermixes (30 min)</LI><P> Mastermixes for each redox molecule concentration  prepared in 1.5 mL eppendorfs. The mastermixes are enough to fill 2 96-well microplates with 100 uL of solutions in each well, in order to have two replicate of the same experiment. Take 8 autoclaved 1.5 mL eppendorfs and prepare them as described in the table below.</P><P>Mastermizes for Day 1 and Day 2 :</P><TABLE class="tg"><TBODY><TR><TH class="tg-s268">Condition</TH><TH class="tg-s268">V Pyo (uL)</TH><TH class="tg-s268">Kan (uL)</TH><TH class="tg-0lax">LB 10x uL</TH><TH class="tg-0lax">Autoclaved Water (uL)</TH><TH class="tg-0lax">Tot Vol (uL)</TH></TR><TR><TD class="tg-s268">A = 0 mM Pyo</TD><TD class="tg-s268">0</TD><TD class="tg-s268">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1098</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-s268">B = 0.01 mM Pyo</TD><TD class="tg-s268">0.0976</TD><TD class="tg-s268">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1097.9024</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-0lax">C = 0.025 mM Pyo</TD><TD class="tg-0lax">0.244</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1097.756</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-0lax">D = 0.05 uM Pyo</TD><TD class="tg-0lax">0.488</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1097.512</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-0lax">E = 0.1 mM Pyo</TD><TD class="tg-0lax">0.976</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1097.024</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-0lax">F = 0.25 mM Pyo</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1095.56</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-0lax">G = 0.5 mM Pyo</TD><TD class="tg-0lax">4.88</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1093.12</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-0lax">H = 1 mM Pyo</TD><TD class="tg-0lax">9.76</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1088.24</TD><TD class="tg-0lax">1220</TD></TR></TBODY></TABLE><P>Mastermixes for Day 3 and Day 4: </P><TABLE class="tg"><TBODY><TR><TH class="tg-s268">Condition</TH><TH class="tg-s268">V Pyo (uL)</TH><TH class="tg-s268">Kan (uL)</TH><TH class="tg-0lax">LB 10x uL</TH><TH class="tg-0lax">Autoclaved Water (uL)</TH><TH class="tg-0lax">Tot Vol (uL)</TH></TR><TR><TD class="tg-s268">A = 2.5 mM Pyo</TD><TD class="tg-s268">24.4</TD><TD class="tg-s268">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1073.6</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-s268">B = 5 mM Pyo</TD><TD class="tg-s268">48.8</TD><TD class="tg-s268">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1049.2</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-0lax">C = 10 mM Pyo</TD><TD class="tg-0lax">97.6</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1000.4</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-0lax">D = 25 mM Pyo</TD><TD class="tg-0lax">244</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">854</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-0lax">E = 50 mM Pyo</TD><TD class="tg-0lax">488</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">610</TD><TD class="tg-0lax">1220</TD></TR><TR><TD class="tg-0lax">F = 100 mM Pyo</TD><TD class="tg-0lax">976</TD><TD class="tg-0lax">2.44</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">122</TD><TD class="tg-0lax">1220</TD></TR></TBODY></TABLE><P> We transferred master mixes into clean eppendorf tubes, with volumes specified below, to add the right antibiotics:</P><TABLE class="tg"><TBODY><TR><TH class="tg-xgv8">Condition</TH><TH class="tg-xgv8">Volume of A-H Mastermix (uL) </TH><TH class="tg-xgv8">Ampicillin (uL)</TH><TH class="tg-oyd9">Chloramphenicol (uL)</TH></TR><TR><TD class="tg-cify">PC1 + PC2</TD><TD class="tg-cify">406</TD><TD class="tg-cify">2.44</TD><TD class="tg-ksfr">0</TD></TR><TR><TD class="tg-cify">+</TD><TD class="tg-cify">203</TD><TD class="tg-cify">0</TD><TD class="tg-ksfr">2.44</TD></TR><TR><TD class="tg-ksfr">-</TD><TD class="tg-ksfr">203</TD><TD class="tg-ksfr">0</TD><TD class="tg-ksfr">0</TD></TR><TR><TD class="tg-ksfr">Blanks</TD><TD class="tg-ksfr">406</TD><TD class="tg-ksfr">2.44</TD><TD class="tg-ksfr">2.44</TD></TR></TBODY></TABLE>

This gave a total of 32 eppendorf tubes for Day 1 and 24 on Day 2, ready to be transferred into 96-well plates.

<LI>Fill 96-well plate with media solutions</LI><P>Add 100 uL of media in the desired wells, according following the label on mastermixes and table below: </P><TABLE class="tg"><TBODY><TR><TH class="tg-fymr">1</TH><TH class="tg-fymr">2</TH><TH class="tg-fymr">3</TH><TH class="tg-fymr">4</TH><TH class="tg-fymr">5</TH><TH class="tg-fymr">6</TH><TH class="tg-fymr">7</TH><TH class="tg-fymr">8</TH><TH class="tg-fymr">9</TH><TH class="tg-fymr">10</TH><TH class="tg-fymr">11</TH><TH class="tg-fymr">12</TH></TR><TR><TD class="tg-fymr">A</TD><TD class="tg-cai3">PC1</TD><TD class="tg-cai3">PC1</TD><TD class="tg-0pky">BA</TD><TD class="tg-f14n">PC2</TD><TD class="tg-f14n">PC2</TD><TD class="tg-0pky">BA</TD><TD class="tg-xuin">-</TD><TD class="tg-xuin">-</TD><TD class="tg-0pky">BA</TD><TD class="tg-r3a5">+</TD><TD class="tg-r3a5">+</TD><TD class="tg-0pky">BA</TD></TR><TR><TD class="tg-fymr">B</TD><TD class="tg-cai3">PC1</TD><TD class="tg-cai3">PC1</TD><TD class="tg-0pky">BB</TD><TD class="tg-f14n">PC2</TD><TD class="tg-f14n">PC2</TD><TD class="tg-0pky">BB</TD><TD class="tg-xuin">-</TD><TD class="tg-xuin">-</TD><TD class="tg-0pky">BB</TD><TD class="tg-r3a5">+</TD><TD class="tg-r3a5">+</TD><TD class="tg-0pky">BB</TD></TR><TR><TD class="tg-fymr">C</TD><TD class="tg-cai3">PC1</TD><TD class="tg-cai3">PC1</TD><TD class="tg-0pky">BC</TD><TD class="tg-f14n">PC2</TD><TD class="tg-f14n">PC2</TD><TD class="tg-0pky">BC</TD><TD class="tg-xuin">-</TD><TD class="tg-xuin">-</TD><TD class="tg-0pky">BC</TD><TD class="tg-r3a5">+</TD><TD class="tg-r3a5">+</TD><TD class="tg-0pky">BC</TD></TR><TR><TD class="tg-fymr">D</TD><TD class="tg-cai3">PC1</TD><TD class="tg-cai3">PC1</TD><TD class="tg-0pky">BD</TD><TD class="tg-f14n">PC2</TD><TD class="tg-f14n">PC2</TD><TD class="tg-0pky">BD</TD><TD class="tg-xuin">-</TD><TD class="tg-xuin">-</TD><TD class="tg-0pky">BD</TD><TD class="tg-r3a5">+</TD><TD class="tg-r3a5">+</TD><TD class="tg-0pky">BD</TD></TR><TR><TD class="tg-fymr">E</TD><TD class="tg-cai3">PC1</TD><TD class="tg-cai3">PC1</TD><TD class="tg-0pky">BE</TD><TD class="tg-f14n">PC2</TD><TD class="tg-f14n">PC2</TD><TD class="tg-0pky">BE</TD><TD class="tg-xuin">-</TD><TD class="tg-xuin">-</TD><TD class="tg-0pky">BE</TD><TD class="tg-r3a5">+</TD><TD class="tg-r3a5">+</TD><TD class="tg-0pky">BE</TD></TR><TR><TD class="tg-fymr">F</TD><TD class="tg-cai3">PC1</TD><TD class="tg-cai3">PC1</TD><TD class="tg-0pky">BF</TD><TD class="tg-f14n">PC2</TD><TD class="tg-f14n">PC2</TD><TD class="tg-0pky">BF</TD><TD class="tg-xuin">-</TD><TD class="tg-xuin">-</TD><TD class="tg-0pky">BF</TD><TD class="tg-r3a5">+</TD><TD class="tg-r3a5">+</TD><TD class="tg-0pky">BF</TD></TR><TR><TD class="tg-fymr">G</TD><TD class="tg-cai3">PC1</TD><TD class="tg-cai3">PC1</TD><TD class="tg-0pky">BG</TD><TD class="tg-f14n">PC2</TD><TD class="tg-f14n">PC2</TD><TD class="tg-0pky">BG</TD><TD class="tg-xuin">-</TD><TD class="tg-xuin">-</TD><TD class="tg-0pky">BG</TD><TD class="tg-r3a5">+</TD><TD class="tg-r3a5">+</TD><TD class="tg-0pky">BG</TD></TR><TR><TD class="tg-fymr">H</TD><TD class="tg-cai3">PC1</TD><TD class="tg-cai3">PC1</TD><TD class="tg-0pky">BH</TD><TD class="tg-f14n">PC2</TD><TD class="tg-f14n">PC2</TD><TD class="tg-0pky">BH</TD><TD class="tg-xuin">-</TD><TD class="tg-xuin">-</TD><TD class="tg-0pky">BH</TD><TD class="tg-r3a5">+</TD><TD class="tg-r3a5">+</TD><TD class="tg-0pky">BH</TD></TR></TBODY></TABLE><LI>OD600 matching</LI><P>After overnight growth, we diluted the all the cultures to an OD of 0.1. We diluted using a FLUOstar plate reader measuring single endpoint absorbance at 600 nm. We added 100 uL of LB + antibiotic into a blank well. We filled the the microplane with the cell culture solutions and took endpoint measurements of the filled wells and blanked them with the blank well. We note down the OD600 value of the wells and dilute them to tan OD600 of 0.1 according to the cell dilution calculator formula described below: (a value of 0.1 is suggested for a volume of 100 uL</P><P>Cell dilution calculator:
Volume of LB+Antibiotic to add in culture tube = [( Vcells * ODcell)/desired OD] - Vcells
E.g. to dilute 5mL of cells at an OD600 of 0.3 to OD of 0.1 = [ (5mL*0.3)/0.1 - 5]  =10 mL
</P><LI>Add cells to 96-well plate</LI><P>We pipetted 100 uL of liquid cultures ot OD600= 0.1 into the corresponding wells,following 96-well microplate layout and culture tubes label.</P><LI>Set up the script in the plate reader and start experiment</LI><P>For Pyocyanin experiments:</P><P>We set up a script on the BMG Labtech Omega control software to perform endpoint  absorbance measurements at 600 nm and fluorescent intensity measurements every 10 minutes for 24 hours (288 cycles for 600 seconds cycles time). Both absorbance and fluorescence measurements were recorded from the bottom of the plate. The excitation/emission wavelengths for GFP detection was set up at 475-510 nm. The plate reader was set up to shake at a double orbital mode at an RPM of 200.  </P><LI>Data analysis</LI><P>We exported the raw data excel spreadsheet and imported into MATLAB software. We calculated average OD600, standard deviation and standard errors (STDEV/SQRT(4), where 4 was our number of replicates. We performed the same calculations for fluorescence values. We divided fluorescence values at each time point by the corresponding OD600 values, to normalise GFP expression by the number of cells. We calculated standard deviations and standard errors analogously for OD600. We then plotted OD600 as a function of time, GFP/OD600. From these plots we identified the time at which cell growth and GFP expression reached a steady state and called this time T_survive. We then took the GFP/OD600 values at T_survive for each construct and plot them against pyocyanin concentration to plot concentration curves, as shown in the “Results” section. Similar analysis was performed for ferrocyanide and ferricyanide and PMS experiments  </P></OL></DIV><H4 class="underbold"> Electrochemistry of Redox Modulators</H4><DIV onclick="expandAll('expt3p');" class="expandButton"><P class="expand expt3p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>Protocol for cyclic voltammetry and amperommetry experiments
</P></DIV><DIV type="button" class="accordion">Notes</DIV><DIV type="button" class="accordion">Procedure</DIV><DIV class="panel expt3p"><OL><LI>Square-wave voltammetry was obtained with the setup descripted in the results section.</LI><LI>Solutions in the flask containing the reference and working electrode were identical to the solutions in the respective flasks containing the counter electrode.</LI><LI>Square-wave voltammetry was performed with the PalmSens 4-channel MultiEmStat3 between -0.3 and +0.5 Volt.</LI><LI> The counter electrodes contained 60% Carbon with a diameter of 5mm. Working electrodes contained 90% Carbon with a 2mm diameter. Salt bridges were consists of 3% agar containing 1 molar potassium chloride inside plastic tubing. The reference electrode was produced by placing a silver wire within 3% agar containing 3 molar potassium chloride inside plastic tubing. </LI><LI>Square wave voltammetry was performed with the following treatment settings: 10s equilibrium time, voltage step of 0.005V, amplitude 0.05V, frequency 25 Hz, the voltage beginning at -0.3V and ending at 0.5V.</LI><LI>Amperometry was performed in  a plate containing the final conditions: 10mM FCN(R), 2.5 uM Pyocyanin, 0.02% Sulfite, Ampicillin, Kanamycin and cells containing the PixCell construct with the following treatment settings: 10sec equilibrium time, time interval 0.05s run time up to 65000 seconds, V=+0.5. </LI><H4 class="underbold"> Spatial Electronic Control of Gene Induction</H4><DIV onclick="expandAll('expt4p');" class="expandButton"><P class="expand expt4p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>Given that we had determined the voltage by which pyocynanin becomes oxidised, we replicated that oxidation within a plate. Thereby turning on GFP expression by oxidising SoxR indirectly.</P></DIV><DIV type="button" class="accordion">Procedure</DIV><DIV class="panel expt4p"><OL><LI>Three 5mm holes were drilled into 9cm agar plates. Working, counter reference electrodes were prepared as described in the results section.</LI><LI> Electrodes were inserted into the plate penetrating 2mm and glued to the plate.</LI><LI> Solutions containing 10mM FCN(R), 2.5 mM Pyocyanin, 0.3% Sulfite, Ampicillin and Kanamycin were pitpetted into a 50ml falcon tube. </LI><LI> 50ml of molten 1% agar was added to the falcon tube. Shaking ensured that the solution was well mixed. After mixing 22ml of the solution was pipetted onto the agar plate. </LI><LI> It was ensured that agar covered the electrodes fully. </LI><LI> After the agar was set, 100uL of cells containing the PixCell construct Deg Tag at an OD of 0.3. were plated onto the agar. </LI><LI>Afterwards the plate was turned upside down, closed and sealed using tape. The setup was then placed into a 37°C incubator.</LI><LI> Potentials were then applied overnight for 13h as described here. </LI><LI>To investigate fluorescence the agar was transferred onto a fresh plate without electrodes penetrating from the bottom. Fluorescence was measured using the Fujifilm FLA-5000 Fluorescent Image Analyser on the following settings: Laser wavelength: 250nm, filter: FITC, gradation: 65536 (16bit), resolution: 50uM. Images where then imported into imageJ for image analysis. Minimum brightness was set to 13867 in all images to discard small fluorescence values which correspond to background fluorescence. </LI></OL></DIV><H4 class="underbold">Creating the Sox Library</H4><DIV onclick="expandAll('expt5p');" class="expandButton"><P class="expand expt5p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P> This protocol describes how to create a library of constructs varying two parts using BASIC assembly method. Following it we constructed a 48 constructs library, using 3 repeated parts and a variable promoter (pSoxS) with 8 variants, and a variable transcription factor (SoxR) with 6 variants (one of them unsubmitted to registry). </P></DIV><DIV type="button" class="accordion">Notes</DIV><DIV class="panel expt5p"><P>
All the experiments for this part of the project were performed on E. coli DH5α, origin strain for DJ901, as this last one could not be made super-competent (109 cfu / μg pUC19  DNA) enough to transform BASIC assembly products. Individual parts were integrated in chloramphenicol resistant high copy plasmid pSB1C3_BASIC vectors and assemblies were performed in ampicillin resistant mid copy plasmid p15A vectors. All individual parts were synthesised and ordered as gblocks with BASIC prefix and suffix through IDT. The linkers used were provided by Biolegio and the adaptors were courtesy of Dr. Baldwin’s lab at Imperial College London. 

</P><TABLE class="tg"><TBODY><TR><TH class="tg-u4ur">Construct name</TH><TH class="tg-u4ur">Promoter (pSoxS)</TH><TH class="tg-u4ur">Transcription factor (SoxR)</TH><TH class="tg-u4ur">Plasmid</TH><TH class="tg-u4ur">Constitutive promoter</TH><TH class="tg-u4ur">GFP</TH></TR><TR><TD class="tg-m9r4">PC_A1</TD><TD class="tg-m9r4">BBa_K2862006</TD><TD class="tg-m9r4">BBa_K2862014</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_A2</TD><TD class="tg-m9r4">BBa_K2862006</TD><TD class="tg-m9r4">BBa_K2862015</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_A3</TD><TD class="tg-m9r4">BBa_K2862006</TD><TD class="tg-m9r4">BBa_K2862016</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_A4</TD><TD class="tg-m9r4">BBa_K2862006</TD><TD class="tg-m9r4">BBa_K2862017</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_A5</TD><TD class="tg-m9r4">BBa_K2862006</TD><TD class="tg-m9r4">BBa_K2862018</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_A6</TD><TD class="tg-m9r4">BBa_K2862006</TD><TD class="tg-m9r4">SoxR-8 (not submitted)*</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_B1</TD><TD class="tg-m9r4">BBa_K2862007</TD><TD class="tg-m9r4">BBa_K2862014</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_B2</TD><TD class="tg-m9r4">BBa_K2862007</TD><TD class="tg-m9r4">BBa_K2862015</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_B3</TD><TD class="tg-m9r4">BBa_K2862007</TD><TD class="tg-m9r4">BBa_K2862016</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_B4</TD><TD class="tg-m9r4">BBa_K2862007</TD><TD class="tg-m9r4">BBa_K2862017</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_B5</TD><TD class="tg-m9r4">BBa_K2862007</TD><TD class="tg-m9r4">BBa_K2862018</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_B6</TD><TD class="tg-m9r4">BBa_K2862007</TD><TD class="tg-m9r4">SoxR-8 (not submitted)*</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_C1</TD><TD class="tg-m9r4">BBa_K2862008</TD><TD class="tg-m9r4">BBa_K2862014</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_C2</TD><TD class="tg-m9r4">BBa_K2862008</TD><TD class="tg-m9r4">BBa_K2862015</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_C3</TD><TD class="tg-m9r4">BBa_K2862008</TD><TD class="tg-m9r4">BBa_K2862016</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_C4</TD><TD class="tg-m9r4">BBa_K2862008</TD><TD class="tg-m9r4">BBa_K2862017</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_C5</TD><TD class="tg-m9r4">BBa_K2862008</TD><TD class="tg-m9r4">BBa_K2862018</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_C6</TD><TD class="tg-m9r4">BBa_K2862008</TD><TD class="tg-m9r4">SoxR-8 (not submitted)*</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_D1</TD><TD class="tg-m9r4">BBa_K2862009</TD><TD class="tg-m9r4">BBa_K2862014</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_D2</TD><TD class="tg-m9r4">BBa_K2862009</TD><TD class="tg-m9r4">BBa_K2862015</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_D3</TD><TD class="tg-m9r4">BBa_K2862009</TD><TD class="tg-m9r4">BBa_K2862016</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_D4</TD><TD class="tg-m9r4">BBa_K2862009</TD><TD class="tg-m9r4">BBa_K2862017</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_D5</TD><TD class="tg-m9r4">BBa_K2862009</TD><TD class="tg-m9r4">BBa_K2862018</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_D6</TD><TD class="tg-m9r4">BBa_K2862009</TD><TD class="tg-m9r4">SoxR-8 (not submitted)*</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_E1</TD><TD class="tg-m9r4">BBa_K2862010</TD><TD class="tg-m9r4">BBa_K2862014</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_E2</TD><TD class="tg-m9r4">BBa_K2862010</TD><TD class="tg-m9r4">BBa_K2862015</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_E3</TD><TD class="tg-m9r4">BBa_K2862010</TD><TD class="tg-m9r4">BBa_K2862016</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_E4</TD><TD class="tg-m9r4">BBa_K2862010</TD><TD class="tg-m9r4">BBa_K2862017</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_E5</TD><TD class="tg-m9r4">BBa_K2862010</TD><TD class="tg-m9r4">BBa_K2862018</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_E6</TD><TD class="tg-m9r4">BBa_K2862010</TD><TD class="tg-m9r4">SoxR-8 (not submitted)*</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_F1</TD><TD class="tg-m9r4">BBa_K2862011</TD><TD class="tg-m9r4">BBa_K2862014</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_F2</TD><TD class="tg-m9r4">BBa_K2862011</TD><TD class="tg-m9r4">BBa_K2862015</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_F3</TD><TD class="tg-m9r4">BBa_K2862011</TD><TD class="tg-m9r4">BBa_K2862016</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_F4</TD><TD class="tg-m9r4">BBa_K2862011</TD><TD class="tg-m9r4">BBa_K2862017</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_F5</TD><TD class="tg-m9r4">BBa_K2862011</TD><TD class="tg-m9r4">BBa_K2862018</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_F6</TD><TD class="tg-m9r4">BBa_K2862011</TD><TD class="tg-m9r4">SoxR-8 (not submitted)*</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_G1</TD><TD class="tg-m9r4">BBa_K2862012</TD><TD class="tg-m9r4">BBa_K2862014</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_G2</TD><TD class="tg-m9r4">BBa_K2862012</TD><TD class="tg-m9r4">BBa_K2862015</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_G3</TD><TD class="tg-m9r4">BBa_K2862012</TD><TD class="tg-m9r4">BBa_K2862016</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_G4</TD><TD class="tg-m9r4">BBa_K2862012</TD><TD class="tg-m9r4">BBa_K2862017</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_G5</TD><TD class="tg-m9r4">BBa_K2862012</TD><TD class="tg-m9r4">BBa_K2862018</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_G6</TD><TD class="tg-m9r4">BBa_K2862012</TD><TD class="tg-m9r4">SoxR-8 (not submitted)*</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_H1</TD><TD class="tg-m9r4">BBa_K2862013</TD><TD class="tg-m9r4">BBa_K2862014</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_H2</TD><TD class="tg-m9r4">BBa_K2862013</TD><TD class="tg-m9r4">BBa_K2862015</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_H3</TD><TD class="tg-m9r4">BBa_K2862013</TD><TD class="tg-m9r4">BBa_K2862016</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_H4</TD><TD class="tg-m9r4">BBa_K2862013</TD><TD class="tg-m9r4">BBa_K2862017</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_H5</TD><TD class="tg-m9r4">BBa_K2862013</TD><TD class="tg-m9r4">BBa_K2862018</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR><TR><TD class="tg-m9r4">PC_H6</TD><TD class="tg-m9r4">BBa_K2862013</TD><TD class="tg-m9r4">SoxR-8 (not submitted)*</TD><TD class="tg-m9r4">P15A-Amp</TD><TD class="tg-m9r4">BBa_K2862019</TD><TD class="tg-m9r4">Yes</TD></TR></TBODY></TABLE><DIV class="caption"><P><B>Figure 1.</B> Table showing assembly parts combinations. RBS for GFP and SoxR were in the linker oligo, being RBS3 the one used for both GFP and SoxR. </P><P>*As SoxR-8 was original from C.amalonaticus a predicted human pathogen, we did not
submit that part to the registry, however we worked with it because of its interesting
properties.</P></DIV></DIV><DIV type="button" class="accordion">Procedure</DIV><DIV class="panel expt5p"><P>Parts assembly for library generation was performed as described in Storch, Casini et al.,
2015. Briefly, linkers were prepared adding 49 µl of annealing buffer (
10mM TRIS-HCl buffer pH7.9, 100mM NaCl, 10mM MgCl2 ) to 0.5 µl (1 µM) of linker and
0.5 µl (1 µM) of adapter oligos, and incubating at 95ºC for 1 min. Once linkers were ready
they were mixed with the parts for the digestion/ligation reaction and incubated at 37ºC for
1 hour, 20ºC for 20 min, 65ºC for 20 min. Each 30 µl reaction had 3µl of ATP, 3µl of NEB4,
3µl BSA, 5µl of prefix linker (1µM), 5µl of suffix linker (1 µM), 1µl of BsaI (20 U/µl), 0.5 µl of
T4 ligase (400 U/µl), 1µl DNA part (200 ng/µl) and water. Parts and linkers
were mixed according to the following table.</P><TABLE class="tg"><TBODY><TR><TD class="tg-kftd">Part</TD><TD class="tg-m9r4">Promoter (for all 8 pSoxS)</TD><TD class="tg-m9r4">Transcription factor (for all 6 SoxR)</TD><TD class="tg-m9r4">Plasmid (p15A)</TD><TD class="tg-m9r4">Constitutive promoter (Pfam2-J23101)</TD><TD class="tg-m9r4">GFP</TD></TR><TR><TD class="tg-kftd">Linker prefix</TD><TD class="tg-m9r4">LMP</TD><TD class="tg-m9r4">UTR2-3</TD><TD class="tg-m9r4">LMS</TD><TD class="tg-m9r4">L5</TD><TD class="tg-m9r4">UTR1-3</TD></TR><TR><TD class="tg-kftd">Linker suffix</TD><TD class="tg-m9r4">UTR1-3</TD><TD class="tg-m9r4">LMS</TD><TD class="tg-m9r4">LMP</TD><TD class="tg-m9r4">UTR2-3</TD><TD class="tg-m9r4">L5</TD></TR></TBODY></TABLE><DIV class="caption"><P><B>Figure 2.</B> Linkers added to each part for assembly (linker notation is based on Stoch et al. 2015 analyzed linker data)</P><P>Digested parts were purified using Magbead purification in a AMPure XP magnetic
purification kit according to manufacturer’s recommendations (Beckman Coulter). These
were then assembled in 48 different combinations, to form our 48 different library
constructs. Each 10vl assembly reaction contained 1µl of each purified DNA part with
linkers, 1µl of NEB4 and 1µl of BSA. Assembly was performed incubating the mix at 50ºC for
45min.</P></DIV></DIV><H4 class="underbold"> Characterising the Sox Library</H4><DIV onclick="expandAll('expt6p');" class="expandButton"><P class="expand expt5p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>The 48 parts library was characterised using the same procedures described for Pixcell construct characterisation (see previous protocol). The only difference is that instead of analysing each construct with 14 different inducer conscentrations we only analysed the base construct PC_A1. Then we characterised the other 47 constructs with only 3 inducer concentrations and fitted this data onto the sigmoidal curve created by PC_A1.</P></DIV><DIV type="button" class="accordion">Notes</DIV><DIV class="panel expt6p"><P>We determined that PC_A1 is our base construct as it is Pixcell construct in library format. This means that it has the same transcription factor (SoxR-0) and the same promoter (pSoxS-0) only that this one in Pfam1 promoter architecture. Therefore, this construct should give us the most similar results to Pixcell construct and show the effect of library architecture. </P></DIV><DIV type="button" class="accordion">Procedure</DIV><DIV class="panel expt6p"><P><B>Promoter induction assay</B></P><P>The best colonies from each succesful construct were characterised in an induction assay
with pyocyanin. Assays were performed with different concentrations of pyocyanin (0-100
µM) and 100 µl of OD 0.1 cells in BMG Clariostat plate readers at 30º C.

</P><P><B>Part growth assay</B></P>

As some constructs did not grow any succesful colony we performed toxicity assays with the
individual parts. Each part in pSB1C3-BASIC was transformed into E.coli DH5α strains using
NEB high efficiency transformation protocol (C2987H/C2987I). Then succesful transformants
were grown at 37ºC for 24 hours in a shaking BMG Clariostat plate reader and OD600
measurements were taken every 10min (see previous protocol in Pixcell construct characterization).</DIV><H4 class="underbold"> Biocontainment- Gp2 Growth Inhibition</H4><DIV onclick="expandAll('expt7p');" class="expandButton"><P class="expand expt7p colorChange" style="font-size:25px;padding-top:1px;">Access All</P></DIV><DIV type="button" class="accordion"> Description </DIV><DIV class="description"><P>A biocontainment device to prototype one of our proposed applications. By placing this downstream of pSoxS we could inhibit cell growth in the oxidising conditions we can produce from our electrochemical set up. We envision this being used with our array in a biocontainment device in which any cells that spread to the edges of the device have their growth seized by an oxidising potential. This device could be used to prevent the accidental release of GMOs, much like how electric fences are used to restrain animals.</P></DIV><DIV type="button" class="accordion">Procedure</DIV><DIV class="panel expt7p"><P>Constructs were assembled by BASIC assembly into a low-copy number pSC101 backbone. SoxR was placed under the control of a medium strength RBS whereas Gp2 was constructed with an RBS library. Both RBSs were located within the BASIC linker sequences. Following assembly transformed cells were plated on LB-agar containing 10mM ferrocyanide and 0.02% sodium sulfite. Five biological replicates seeded from a single colony for each construct were grown out in LB containing 10mM ferrocyanide and 0.02% sodium sulfite for 6 hours. These cultures were then diluted to a final OD of 0.1. 5μl of each culture was added to 195μl LB solutions in a Greiner BioOne F-bottom 96-well plates to a final concentration of 2.5μl pyocyanin, 10mM ferricyanide and 0.02% sulfite for the OFF condition and to 2.5μl pyocyanin, 10mM ferrocyanide and 0.02% sulfite for the ON condition. Plates were incubated for 24 hours in a BMG Labtech FLUOstar at 37oC with absorbance readings at 600nm every 5 minutes.</P></DIV><DIV class="footer"><DIV class="socialicons"><H4> official.imperial.igem@gmail.com</H4></DIV></DIV></OL></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>