Team:ShanghaiTechChina B/Project
Toggle navigation
Home
Project
Intestinal Disease
EGF Expression
Therapeutic Biofilm
NO Sensor
Quorum Sensing
Kill Switch
Warship
Model
Lab Work
Parts
Safety
Protocols
Notebook
Human Practice
Introduction
Interviews
Survey
Social Media
HUST-Cheering
iDPI
Poster
Collaborations
SYSU-MEDICINE
FAFU-CHINA
About Us
Team Roster
Acknowledgement
Attributions
Judging
Attributions
Collaborations
HP Silver
HP Gold
Description
Proof
Demonstrate
Integrated Practices
Model
Hardware
Intestinal Disease
The intestines, are crucial parts of the gastrointestinal tract. Their dysfunction often leads to problems in digestion and absorption, and can result in serious chronic diseases if not being treated properly. Inflammatory bowel disease (IBD) is one of inflammatory conditions that affect the colon and small intestine.
IBD, including Crohn's disease and ulcerative colitis, affect the bowel linings. Crohn's disease may affect any part of the gastrointestinal tract from mouth to anus. Ulcerative colitis (UC) is a long-term condition that results in inflammation and ulcers of the colon and rectum. Signs and symptoms in IBD often include abdominal pain, diarrhea (which may be bloody if inflammation is severe), fever, and weight loss. It can be caused by many factors, including genetic factors, environmental factors, microbes and autoimmune conditions. It has a high incidence in developed countries, about 3.2 per 1,000 people in Europe and North America. It is less common in Asia and Africa, but rises rapidly along with economic growth.
Inflammatory bowel diseases are refractory diseases. It is often hard to diagnose at the earlier stage and it deteriorates with time and becomes chronic, therefore it reduces the quality of life and raises significant economic burden to the family. Unfortunately, there is no cure for this disease. Current management of Crohn's disease includes surgery, medications and fecal microbiota transplant. These treatments have limited effects on IBD.
Surgery
Ulcerative colitis can in most cases be cured by proctocolectomy, although this may not eliminate extra-intestinal symptoms. This procedure is necessary in the event of: exsanguinating hemorrhage, frank perforation, or documented or strongly suspected carcinoma.
Surgery cannot cure Crohn's disease but may be needed to treat complications such as abscesses, strictures or fistulae. Crohn's disease can recur in the healthy parts of the intestine, usually at the resection site.[1] Postsurgical recurrence of Crohn's disease is relatively common. Also, surgeries possess their own risk such as infection.
Drugs
Chemical drugs for IBD are tailored for each patient. So doctors must consider various factors to determine drugs and doses.
Generally, depending on the level of severity, IBD may require immunosuppression to control the symptoms. Available drugs include antibiotics, corticosteroids, immunosuppression agents (TNF inhibitors infliximab, adalimumab, and golimumab), Nicotine and Iron supplementation. Immunosuppression are usually beneficial to patients. However, it may make the body becomes more vulnerable to infections and malignancy.
Fecal microbiota transplant
FMT is the process of transplantation of fecal bacteria from a healthy individual into a recipient. It is a relatively new treatment option for IBD which has attracted many attention since 2010 and many related standards has not been completely established yet.  This therapy needs strict tests for donors, and the curative effect is not significant enough so far.
Our team iGEM2016 ShanghaiTechChina_B is striving to provide new angles in treatment. We chose IBD as an entry point and designed a complete set of therapeutic methods to treat it by utilizing our engineered E.coli. Bacteria treatment has many advantages. It targets precisely, cures conveniently and costs less. Actually we are hoping our approaches can also be applied to treat other intestinal diseases in the future.
Reference
https://en.wikipedia.org/wiki/Inflammatory_bowel_disease#Treatment
[1] Karimuddin, Ahmer; Gilles, Gary. "Surgery for Abdominal/Intestinal Crohn's Disease". Trusted Therapies. Trusted Therapies. Retrieved 19 May 2015.
[2] Colman RJ, Rubin DT (2014). "Fecal microbiota transplantation as therapy for inflammatory bowel disease: a systematic review and meta-analysis". J Crohns Colitis. 8 (12): 1569–81. doi:10.1016/j.crohns.2014.08.006. PMC 4296742free to read. PMID 25223604.
EGF Expression
Introduction
At the beginning of our project design, we thought of many possible treatment but during the process of reasoning we encountered some difficulties. In the spring of 2016, at National Center for Protein Science Shanghai we interviewed professor Lu.P who was a researcher at the Paterson Institute for Cancer Research in the University of Manchester. After he understood that we wanted to fight IBD, he pointed out to us that EGF, which is an important factor of his ongoing research, is a molecule that has the ability to promote regeneration of cells in the inflammatory region and therefore can be a possible treatment to IBD.
Figure 1. 3D structure of EGF protein
Human epidermal growth factor (hEGF) is a 6045-Da protein with 53 amino acid residues.
It can stimulate cell growth, proliferation and differentiation by binding to its receptor EGFR both in vivo and vitro. Some people utilize it as cosmetic product because of the potent ability of recovering epithelium.
EGF expression
We found inspiration in the interview with Lu, P. and designed a bacterial strain that can secrete hEGF in patients’ gut. E.coli. has been chosen as expression strain because it is aborigines in human’s gut and easy to be programing. A signal peptide phoA is added at the N-terminus to facilitate secretion. We engineered two peptide tags, a phoA signal peptide at the N-terminus and a FLAG tag at the C-terminus of hEGF protein. PhoA is a fragment of signal peptide that guides the protein to go across cytomembrane and do not decrease activity of the protein. FLAG-tag is a polypeptide protein tag that can be added to a protein using recombinant DNA technology, having the sequence motif DYKDDDDK.
Figure 2. Western Blot of EGF with his-tag. The red rectangle marked area is our expressed EGF protein. The track on the left is the control group without IPTG induction.
Physiological activity of recombinant EGF
Figure 3. Technological process of recombinant EGF purification.(note: this picture is from the website of Qiagen)
In order to test the physiological activity of recombinant EGF, we substituted FLAG tag for 6*HIS tag and purified it by Ni-NTA Agarose.
The purified recombinant protein was applied to mouse models of intestinal inflammation by enemas (performing by SYSU-MEDICINE, click here to get more information). The IBD mice showed many symptoms, including weight loss and colon length reduction (Fig. 6 and 7). We sacrificed mice after 5-day treatment with hEGF or saline, and measured the weight changes over time, as well as the colon length. We found hEGF, but not saline enema rescued the weight loss phenotype (Fig. 6). In addition, the colon length of hEGF treated group was longer than that of saline (Fig. 7). In aggregate, with the joint efforts from both two iGEM teams, we consolidated that recombinant hEGF protein produced from engineered bacteria can effectively relieve several IBD symptoms in an in vivo model, suggesting engineered bacteria capable of releasing therapeutic agents (such as hEGF) could be an attractive approach to fight against IBD.
Figure 4. Weight changes of IBD mice over time.
Figure 5. The length of colons of IBD mice.
The result shows that recombinant EGF enemas are an effective treatment for intestinal inflammation in mouse and therefore can be potent effector of our project. We use EGF as functional module of all the experiments and designs in the section below.
Parts for this section:
BBa_K2152001
BBa_K2152000
Reference
[1] Baumgart D C, Sandborn W J. Inflammatory bowel disease: clinical aspects and established and evolving therapies[J]. The Lancet, 2007, 369(9573): 1641-1657.
[2] Wirtz S, Neufert C, Weigmann B, et al. Chemically induced mouse models of intestinal inflammation[J]. Natureprotocols, 2007, 2(3): 541-546.
NO Sensor
Introduction
As the first discovered cellular signaling molecule, Nitric Oxide (NO) plays a vital role in many physiological and pathological processes. There is a significant increase of NO concentration within the guts of IBD patients[1-3]. Thus, NO can be used as targeting molecule for therapeutic agents. In our project, we utilized NO as a signal “marker” to recruit our engineered E.coli to inflammatory positions and as a trigger to initiate expression of a potential therapeutic peptide.
Nitric Oxide (NO) production increase in IBD
Figure 1. Biologically synthesized nitric oxide (NO) is produced through Nitric oxide synthases (NOSs) in human. The metastable free radical nitric oxide (NO) is then released into plasma, where NO has a half-life of only a few seconds, and further transferred into nitrate (NOx).
Previous research showed that nitric oxide synthase (NOS) activity is enhanced in IBD (ulcerative colitis and Crohn's disease), so is the rectal nitric oxide (NO) concentrations[1-3]. Rectal nitric oxide (NO) has a short gradient distance but a sharp concentration peak at the diseased regions of the gut. It has been reported that, the average rectal nitric oxide (NO) concentration in normal people is ~60 ppb (60 nM), whereas in IBD patients is ~5,500 ppb (5.5 μM)[3]. These features make NO assessments to be a practicable method for determining the presence of an on-going inflammatory reaction within the gut.
Sodium nitroprusside (SNP) can release Nitric Oxide (NO) at a pathological concentration
There are several reagents that can release NO in aqueous solution. One of them is called Sodium nitroprusside (SNP), a member of the NO-releasing drugs, which is originally used as a peripheral vasodilator.
In order to understand the kinetics of SNP releasing NO in LB, or say the environment where E.coli is cultured, we detected the concentration of NO in SNP-LB solution by Griess test (Figure 2,3)[4]. And plotted the Dynamic curves of different concentrations of SNP releasing NO (Figure 4).
Figure 2. The Griess test. When Griess reagent I (N-alpha-naphthyl-ethylenediamine) is added, the nitrites (NO2−) form a diazonium salt. After Griess reagent II (sulphanilic acid) is added, a pink color develops, and have a maximum absorbance peak at 540 nm.[4]
Figure 3. The concentration of NO released by SNP in LB medium were determined by Griess test. Add the Griess reagents to samples within 96-well plate and read the absorbance at 540 nm by the microplate photometer.
Figure 4. The Dynamic curves of different concentrations of SNP releasing NO. The SNP concentrations are 10 μM, 100 μM, 1 mM, 10 mM and 0 μM (control) in each group. Data are means(SD) (n=3 samples per group). The dot line indicates the average rectal NO concentration in IBD patients.
The data reveals the temporal changes of the released nitric oxide (NO) concentration in SNP-LB solution. From the dynamic curves, we could see that even low concentration of SNP could continuously release NO for 5~6 hours or even more.
Previous research has measured the rectal NO concentration of IBD. The average rectal nitric oxide (NO) concentration ~60 ppb (60 nM) in normal people and ~5,500 ppb (5.5 μM) in IBD patients which is dot lined in Figure 4[3]. The pathological NO concentration is within the range of the SNP-released NO concentration.
Thus, we could use SNP solution to mimic the rectal NO concentration in IBD, and keep it for hours. So we can put our engineered E.coli under pathological condition in vitro, to initiate expression of the therapeutic peptide.
NO (Nitric Oxide) Sensing Module
Microbiome have evolved several biological NO (nitric oxide) sensors to detect the existence of nitric oxide in the environment, including SoxR protein, FNR family, NorR protein and so on [4].
NorR protein is σ54-dependent, AAA transcriptional activator, with a N-terminal regulatory domain, a central AAA domain which is regulated by NO molecule, and a C-terminal DNA binding domain, which will bind to promoter pNorV to initiate transcription when activated.
Figure 5. The mechanism of NO (Nitric Oxide) sensor by NorR protein. NorR protein is produced by E.coli. Once binds to NO molecule, it is activated. Then NorR will further initiate the transcription at the promoter pNorV.
We integrated this system into E.coli, which would be able to detect NO existence and then express the therapeutic molecules, i.e. EGF peptide, to treat IBD (Figure 6).
Figure 6. The utility of NO (Nitric Oxide) Sensor in treating IBD. The rectal NO concentration is abnormally high in the gut of IBD patients. So, NorR proteins produced by E.coli will bind NO molecule, and then initiate the transcription and expression of EGF peptides. The expressed EGF peptides will contribute to the recovery of IBD.
Here, GFP expression is taken as an example to demonstrate this approach.
We constructed NO Sensing Module to detect the existence of Nitric Oxide (Figure 7). The nitric oxide released by SNP can mimic the pathologically high rectal NO concentration in the inflammatory regions in IBD.
Figure 7. The constructed NO (Nitric Oxide) Sensing Module. The vector is pETduet-1, on which we replaced one of two T7 promoters with the promoter pNorV, and the downstream is GFP. The NorR protein production is under IPTG induction, so we could control the module more accurately.
So, the GFP expression can be induced in the presence of nitric oxide (released by SNP) and IPTG (to control the production of NorR protein as shown in Figure 7). We tested this NO Sensing Module with fluorescence microscope and got a positive result (Figure 8).
Figure 8. The NO Sensing Module pictures of fluorescence microscope. The SNP was added to the final concentration of 100 μM (a), 10 mM (b) and (c), together with IPTG. Neither SNP nor IPTG was not added to the control group (d).
Most E.coli (50.1%) expressed GFP in 100 μM SNP-LB culture (Figure 8a), in which the nitric oxide concentration was close to the rectal nitric oxide concentration (5.5mM) in IBD (Figure 4), 8 hours after SNP is added. In contrast, only 2.5% E.coli expressed GFP without SNP (Figure 8c), which may be leaky IPTG induced expression. We noticed that the growth of E.coli and GFP expression were suppressed in 10 mM SNP-LB culture (Figure 8b), in which SNP showed a dose-dependent cytotoxic effect on the cell and only 5.8% E.coli expressed GFP[6]. In short, GFP expressing NO Sensing Module works.
Our experiment proved the practicability of NO Sensing Module in vitro, where GFP is a demo for EGF expression. This approach can be applied to many diseases like IBD, to detect the inflammatory regions and then express therapeutic peptides (i.e. EGF peptides) that can promote the recovery of the disease.
Parts for this section:
BBa_K256004
BBa_K256000
Reference
[1] Rachmilewitz D, Stamler J S, Bachwich D, et al. Enhanced colonic nitric oxide generation and nitric oxide synthase activity in ulcerative colitis and Crohn's disease[J]. Gut, 1995, 36(5): 718-723.
[2] Kimura H, Miura S, Shigematsu T, et al. Increased nitric oxide production and inducible nitric oxide synthase activity in colonic mucosa of patients with active ulcerative colitis and Crohn's disease[J]. Digestive diseases and sciences, 1997, 42(5): 1047-1054.
[3] Ljung T, Herulf M, Beijer E, et al. Rectal nitric oxide assessment in children with Crohn disease and ulcerative colitis. Indicator of ileocaecal and colorectal affection[J]. Scandinavian journal of gastroenterology, 2001, 36(10): 1073-1076.
[4] Nagano T. Practical methods for detection of nitric oxide[J]. Luminescence, 1999, 14(6): 283-290.
[5] Spiro S. Regulators of bacterial responses to nitric oxide[J]. FEMS microbiology reviews, 2007, 31(2): 193-211.
[6] Sumitani K, Kamijo R, Nagumo M. Cytotoxic effect of sodium nitroprusside on cancer cells: involvement of apoptosis and suppression of c-myc and c-myb proto-oncogene expression[J]. Anticancer research, 1996, 17(2A): 865-871.
Therapeutic Biofilm
Introduction
The IBD can often become a chronic disease and is difficult to treat. Therefore, ‘cocktail’ therapy by combining different strategies may very helpful to patients. Similar approaches have been used successfully in treating AIDS and many types of cancer. Other than aforementioned secreted EGF, we are also interested in applying therapeutic biofilms to treat gut diseases, since it is stable and programmable.
Biofilm
The biofilms are organized communities of cells ensconced in a network of extracellular matrix (ECM) composed of polysaccharides, proteins, nucleic acids and other biomolecular components[1]. Although the researches of biofilm have focused on preventing their formation and promoting their disruption because of their significant roles in pathogenicity during a long period of time, several novel applications of biofilm have been proposed in the last decade, because their potential to be a synthetic biological platform for self-assembling functional materials has been realized and developed[3].
So, biofilms can be used to load therapeutic drugs or functional peptides, and then releasing them extracellularly, where functional peptide domains can fuse into an effectively therapeutic factor. Here, we take the EGF loaded Curli biofilm as an example to illustrate this approach.
Example: Peptide-loaded Curli biofilm
The Curli biofilms secreted dominantly by E.coli are highly robust functional amyloid nanoﬁbres with a diameter of approximately 4–7nm that exist as extended tangled networks encapsulating the cells [2].
The most important point is, Curli fibers are self-assembled by CsgA amyloid proteins, which can be engineered to fuse other functional peptide domains. Thus, we incorporate therapeutic enzymes or peptides (i.e. EGF peptide) into Curli biofilm.
Figure 1. The illustration of EGF-loaded Curli biofilm. Curli biofilms are formed from the extracellular self-assembly of CsgA amyloid proteins, which can be engineered to fuse other functional peptide domains. For instance, if the sequence of EGF peptide is cloned next to CsgA secretion sequence connected by a GSGGSG (G-Glycine, S-Serine) linker, the EGF peptides will be fused into CsgA proteins and be loaded on Curli biofilms by self-assembly of CsgA-EGF monomers, whereas EGF peptide domains are still functional, presumably.
As Figure 1 states, the linker between CsgA protein domain and EGF peptide domain can be designed as the targets of hydrolytic enzymes, which are up-regulated at sites of inflammation [3]. Thus releasing of EGF peptides would be precisely and locally concentrated, rather than entire-intestinal concentrated. Furthermore, Curli biofilm has a long-term stability, which enables sustained EGF peptide release in this way over hours or even days.
We confirmed the formation of EGF-loaded Curli biofilm through Congo Red dye (Figure 2) and Transmission electron microscopy (TEM) (Figure 3).
Figure 2. Curli biofilms can be stained by Congo Red dye, which is a practical observing method for them [4]. The samples in two tubes are parallel.
Figure 3. The Transmission electron microscopy (TEM) pictures of Curli fibers. (a) The EGF-loaded Curli biofilm encapsulates the entire cell, and (b) the fibers of EGF-loaded Curli biofilm are extremely robust that can withstand vigorous washing process. (c) The negative control that no Curli biofilm is present.
In conclusion, we consider EGF-loaded Curli biofilm as a model to demonstrate the utility of the Biofilm-Based strategy to treating gut diseases like IBD. Our approach could potentially be applied to many other diseases where controlled therapeutic peptide release at a stable and continuous level.
Parts for this section:
BBa_K2152002
Reference
[1] Flemming, H. C. & Wingender, J. The bioﬁlm matrix. Nat. Rev. Microbiol. 8, 623–633 (2010).
[2] Nguyen, P. Q. et al. Programmable bioﬁlm-based materials from engineered curli nanoﬁbres. Nat. Commun. 5:4945 doi: 10.1038/ncomms5945 (2014).
[3] Zhang S, Ermann J, Succi M D, et al. An inflammation-targeting hydrogel for local drug delivery in inflammatory bowel disease[J]. Science translational medicine, 2015, 7(300): 300ra128-300ra128.
[4] Marcus, A., Sadimin, E., Richardson, M., Goodell, L. & Fyfe, B. Fluorescence microscopy is superior to polarized microscopy for detecting amyloid deposits in Congo red-stained trephine bone marrow biopsy specimens. Am. J. Clin. Pathol. 138, 590–593 (2012).
Quorum Sensing
During the design of the project, it came to our mind that NO is a highly unstable molecule. So NO signal will decay over increased ditance. But how to amplify this signal of disease and further regulate it? We thought of Quorum Sensing system in microbes. When a gutrio detects NO molecule, it will release a more stable signal for other gutrio to respond to the disease along with it.
Fig. 1 depicts our quorum sensing system.
Figure 1. Quorum sensing system for gram-negative bacteria.
We constructed plasmids as Fig. 2.
Figure 2. Quorum sensing plasmid construction.
LuxR protein combined with signal molecules can activate pLuxR promoter and start expression of GFP(for testing) or EGF(for functioning). A few sender bacteria can send signal molecules to many receiver bacteria thereby achieve signal amplification.
We didn’t have enough time to build the quorum sensing system. We strived to build plasmids in the picture above using multiple method of gene cloning, we got struck in cloning of pLuxR—an indispensable promoter. Although we have verified the expression of LuxR which can sense signal molecules released by senders.
The following picture is the SDS-PAGE of LuxR protein expressed in E.coli promoted by T7 promoter.
Figure 3. SDS-PAGE of LuxR protein expressed in E.coli. the area between two red lines is our expressed LuxR protein.
We believe that this system has the potential to be further modified for greater use. In 2009, iGem team from Chiba University used quorum sensing system for timing. Huazhong University of Science and Technology will use this system for filtering. These examples are telling us that if we can introduce this multifunction port, much more amazing functions may be achieved, periodical drug delivery for instance.
Parts for this section:
BBa_C0062
BBa_R0061
BBa_R0062
Kill Switch
In the applications of synthetic biology, people always question about the safety of genetic engineered organism. There is a potential risk of horizontal gene transfer to other non-lab strains, posting a threat to the natural ecosystem. Therefore, we need to take some measures to guarantee the safety of our bacterial materials. Moreover, for potential medical use of engineered organism, we have to take extra step to ensure the biosafety.
In our project, we designed bacteria to treat gut inflammation. It is critical to control the fate of genetically modified bacteria when needed. We did so by engineering a kill switch in bacteria. The kill switch should allow us to terminate bacteria after therapy, so as to avoid interrupting the normal gut ecosystem and the risk of horizontal gene transfer. We decided to use two type of killer genes, killer red and phi 174 gene E, as the kill switches.
Background
Killer red
Killer red is a red fluorescent protein that could produce ROS (Reactive oxygen species) by light illumination. As shown in Fig.1, green light (585 nm) stimulation of the QYG chromophore of killer red produces ROS from water to kill cell[1][2][3]
Figure 1. The sketch map of killer red mechanism
The killer red is very specific and would not produce ROS without light illumination. In human gut, the killer red should not induce any tissue damage since there is no access to light. Deliver light to guts will terminate engineered bacteria when necessary.
Phi 174 Gene E
Gene E is a cell lysis protein from phage, which kills bacteria by inhibiting cell wall synthesis. We decided to test this gene under different promoters so that we could induce cell death when needed. Compared to the killer red, gene E is more efficient in killing and its expression can be controlled by chemicals (such as IPTG), thereby more convenient in killing.
Results
The killer red limits bacteral growth
Figure 2. Confocal images of killer red expression in E.coli.
Then, We tried to cultivate the induced killer red under the light of oven, and the growth curve did not show significant difference between the experimental group and control group, illustrating that when under the illumination of white light, it would not produce ROS that could do harm to cells, confirming it’s safety in gut.
Finally, to test the efficiency of killer red, we put it under the illumination of the specific light of fluorescent microscope for 4 hours, and then cultivate them in 37℃ drying oven overnight, and we could see it clearly that the culture medium which both under induction and illumination is significantly have less single clones, and the others(one illuminated by light and not induced, one induced without illumination of light, one neither induced nor illuminated) surface are rough, indicating the large amount of E.coli.
Figure 3. A: the experiment group which was induced and illuminated under the green light resource of fluorescent for 4 hours; B: the control group which was induced but without illumination; C: the group which was not induced but with illumination; D: the group without inducer and illumination
Gene E expression lysis bacteria
Firstly, we used the temperature sensitive operon to control the expression of gene E. We inserted gene E into the multiple clone site of pBV220, and used OD 600 to measure the growth of E.coli. We made the subtraction and used the slope to indicate the efficiency of gene E.
Figure 4. The growth curve of gene E which is in pBV220, which induced in 42℃ when OD 600 nm= 1
Figure 5. The difference of OD 600 between which was cultivated in 42℃ and 28℃, showing the efficiency of gene E
To confirmed that we could use gene E in people's gut, we test is' efficiency when it is cultivated under 37℃
However, we found it surprising that it would be much more efficient when in 37℃ than in 42℃.
Figure 6. The growth curve of gene E which is under temperature sensitive operon induced in OD = 1 in 3 different temperatures
Figure 7. The difference of OD 600 between 28 ℃ and 37 ℃ and the difference of OD 600 between 28℃ and 42℃, the different slope shows different efficiency
Then, we test the expression of gene E under the control of some other conditions such as the induction of different small molecules. If we want to terminate the therapy, we could feed our patient with small chemistry molecules, which could terminate bacteria’s life.
So, we construct the gene circuit, putting gene E downstream T7 promoter, and induce the expression of gene E under small chemistry molecular, IPTG. These diagram below illustrate the efficiency of gene E which is under T7 promoter. Under the induction of the IPTG, the gene E can strongly inhibit the growth of E.coli.
Figure 8. The growth curve of gene E under T7 promoter which was induced in the time 0, showing the effecient of gene E
Figure 9. The difference of growth curve of gene E under T7 promoter which indicate the efficient of gene E
On the other hand, we found that, even when we use the BL21(DEe)pLysS, which could express toxin protein, the results showed some leaking expression, which make it’s growth curve below that of control group (which contains the same vector without gene E). However, the effect of gene E in experiment group can be seen clearly after the addition of IPTG.
In our future plan, we design to use pBAD (part of arabinose operon), whose inducer is not toxin to human body as IPTG, to construct our gene circuit. pBAD has similar mechanism with pLac and gene E is expected to work under pBAD as well. With the limit of time and money, we did not conduct this experiment, but we expect to finish this work with the help and funds of school.
Other parts about kill switch
During our experiment, we independently proposed some creative ideas and tested them.
Firstly, when we measure the OD 600 of bacteria fluid, we often use 1ml fluid each time. But to keep the total volume, which makes it no effect on the subsequent measurement, we need to make a lot of LB medium and cultivate a large amount of bacteria for one time, which is not economical.
Accidentally, we used nanodrop to measure OD600 in iHuman institution where has no common spectrophotometer. Nanodrop is a kind of instrument which only needs very little liquid to measure OD, which could largely saves LB liquid medium and simplifies the procedure of measurement. However, for that the optical path is not the same for nanodrop and common spectrophotometer, it is necessary to make the standard curve which could ensure the corresponding OD of nanodrop for each bacteria’s growth stage. We used the same sample to be measured by nanodrop and spectrophotometer, and then made this curve from these data, getting the coefficient for the translation of OD600 of common spectrophotometer to nanodrop.
Figure 10. The standard curve which showed the coefficient of the translation from value of OD 600 of spectrophotometer to that of nanodrop
Parts for this section:
BBa_K2152004
BBa_K2152006
Reference
[1] Sergei Pletnev, Nadya G. Gurskaya, Nadya V. Pletneva, Konstantin A. Lukyanov, Dmitri M. Chudakov, Vladimir I. Martynov, Vladimir O. Popov, Mikhail V. Kovalchuk, Alexander Wlodawer, Zbigniew Dauter, and Vladimir Pletnev. Structural Basis for Phototoxicity of the Genetically Encoded Photosensitizer KillerRed Journal of Biological Chemistry 2009 284: 32028-32039. First Published on September 8, 2009, doi:10.1074/jbc.M109.054973
[2] Molecular graphics and analyses were performed with the UCSF Chimera package. Chimera is developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco, funded by grants from the National Institutes of Health National Center for Research Resources (2P41RR001081) and National Institute of General Medical Sciences (9P41GM103311).
[3] Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, Ferrin TE. UCSF Chimera--a visualization system for exploratory research and analysis. Journal of Computational Chemistry. 2004 Oct;25(13):1605-12.
Warship
"Simplicity involves digging through the depth of the complexity" (by Steve Jobs). We aimed to build a simple device (we termed ‘warship’) to increase the competitive power of introduced bacteria against native microbiota and biosafety levels of using engineered bacteria. This was inspired by the feedback from our presentation in a workshop (HUST_Cheering) in July. Other iGEMers were curious about the 'fate' of our engineered bacteria in human's gut. Our Gutrio has to compete with other living microorganisms in a such complicated gut environment. Intestinal environment differs from individuals to individuals and the agents cannot survive if they are not well-prepared. Moreover, it is quite dangerous for armed Gutrios to expand their territory without limitation.
We achieved this by sandwiching alien bacteria with pieces of semi-permeable membrane, thereby detaining concentrated aliens and yet still allowing exchange of chemicals between aliens and gut. We extensively elucidated that aliens would not leak out and surrounding bacteria could not invade aliens. Importantly, we revealed small molecules can freely diffuse across the semi-membrane. Therefore, in principle, therapeutic agents produced by aliens can diffuse into gut environment. Thus, our simple device allowed us to target two birds (alien competitive power and biosafety) with one stone.
Figure 1. Warship bacteria constraint device. Size may be reduced by 3D printing in the future.
Warship bacteria constraint device. Size may be reduced by 3D printing in the future.
The biggest diameter of particles that are allowed through the semi-permeable membrane is 0.22 um. We immersed our device in intestinal-pH solution, concentrated hydrochloric acid, sodium bicarbonate solution and common organic solvent, used mechanical compression to test its strength and it was intact after all of the tests.
First we tested if our device allows small molecules like IPTG pass through the membrane (Figure 2). We put E.coli with a IPTG-induced green fluorescence protein inside the device.
Figure 2. We add IPTG to the solution outside the membrane. It promoters the expression of gfp inside the Warship successfully.
We also tested if the bacteria inside the device can get out (Figure 3).
Figure 3. We put microbes with different antibiotic-resistance outside and inside Warship respectively. They cannot pass through the membrane.
This device can ensure that the engineered bacteria will be alive in the intestines and is able to exchange matter with the outside of the device.
Model
This summer, we were trying to tracing abnormally high NO concentration in people's guts.  We want to simulate NO distribution in our bowels through modeling. We turn to the discussion of time-dependent diffusion processes, where we are interested in the spreading and distributing of NO with time.
Diffusion Equation
The equation for the rate of change of the concentration of particles in an inhomogeneous region, diffusion equation, also called 'Fick's second law of diffusion', which in this case relates the rate of change of NO concentration at a point to the spatial variation of the NO concentration at that point:$$\frac{\partial c}{\partial t}=D\frac{\partial ^{2}c}{\partial x^{2}}$$
Figure 1. The net flux in a region is the difference between the flux entering fromthe region of high concentration (on the left) and the flux leaving to the region of low concentration (on the right). (Reference: P W Atkins. Atkins’ Physical chemistry, 7th Ed., Oxford: Oxford University Press, 2002, 776)
Justification
Consider a thin slab of cross-sectional area A that extends from x to x+l.
Let the NO concentration at x be c at the time t. The amount (number of moles) of particles that enter the slab in the infinitesimal interval dt is JAdt, so the rate of increase in molar NO concentration inside the slab (which has volume Al) on account of the flux from the left is$$\frac{\partial c}{\partial t}=\frac{JA\mathrm{d}t}{Al\mathrm{d}t}=\frac{J}{l}$$
There is also an outflow through the right-hand window. The flux through that window is J′, and the rate of change of NO concentration that results is$$\frac{\partial c}{\partial t}=-\frac{J'A\mathrm{d}t}{Al\mathrm{d}t}=-\frac{J'}{l}$$
The net rate of change of NO concentration is therefore$$\frac{\partial c}{\partial t}=\frac{J-J'}{l}$$
Each flux is proportional to the NO concentration gradient at the window. So, by using Fick's first law, we can write$$J-J'=-D\frac{\partial c}{\partial x}+D\frac{\partial c'}{\partial x}=-D\frac{\partial c}{\partial x}+D\frac{\partial}{\partial x}\left[c+\frac{\partial c}{\partial x}l \right]=Dl\frac{{\partial}^{2}c}{\partial {x}^{2}}$$
The viscosity of water is 0.891 cP (or 8.91 × 10-4kg·m-1·-1), and the effective radius of NO approaches 0.36 nm in this case.
Diffusion Coefficient
With the constant D,which is called the diffusion coefficient, to be obtained, the formula above cannot be solved.
The following picture shows the flux of particles down a concentration gradient. Fick’s first law states that the flux of matter (the number of particles passing through an imaginary window in a given interval divided by the area of the window and the duration of the interval) is proportional to the density gradient at that point. $$J(\mathrm{matter})=-D\frac{\mathrm{d}\mathcal{N}}{\mathrm{d}z}$$
Figure 2. The flux of particles down a concentration gradient. Fick’s first law states that the flux of matter (the number of particles passing through an imaginary window in a given interval divided by the area of the window and the duration of the interval) is proportional to the density gradient at that point. (Reference: P W Atkins. Atkins’ Physical chemistry, 7th Ed., Oxford: Oxford University Press, 2002, 757)
If we divide both sides of this equation by Avogadro’s constant, thereby converting numbers into amounts (numbers of moles), then Fick’s law becomes$$J=-D\frac{\mathrm{d}c}{\mathrm{d}x}$$
In this expression, D is the diffusion coefficient and dc/dx is the slope of the molar concentration. We can obtain NO's diffusion coefficient in water either by experiments or by calculations. Finally we choose to do the calculations.
For dilute non-electrolyte A, its diffusion coefficient in solvent B (also called the infinite-dilute diffusion coefficient) can be estimated by Wilke-Chang equation:$${D}_{AB}=7.4\times {10}^{-15}\frac{{(\Phi {M}_{B})}^{T}T}{\mu {{V}_{A}}^{0.6}}\mathrm{{m}^{2}/s}$$
In the equation,
DAB — The inter-diffusion coefficient in an infinitely-dilute solution, m2/s.
T — The temperature of the solution, K. In our case, it is 0.5.
μ — The viscosity of B, Pa·s.
MB — The molar mass of B, kg/mol.
Φ — The parameter of association of solvent. (Introduction of the association parameter into the formula is brought about by the fact that associated molecules behave like large-size molecules and diffuse at a lower rate; the degree of association varying with mixture composition and with molecule types. Therefore, Wilke and Chang presented the values for most widespread solvents: for water φ=2.6; methanol, 1.9; ethanol, 1.5; benzene, ester, heptane and non-associated solvents, 1.)
VA — The molar volume of solute A at a boiling point under normal conditions, cm3. It can be estimated by Tyn-Calus Method: V=0.285V1.048, while Vc is the critical volume of a substance and measures 57.7 for NO.
Results
Figure 3. NO concentration distribution with time around the wound.
The six pictures demonstrate the NO concentration distribution with time (with the 1 s interval) around the wound caused by IBD. The concentration is shown along the Z-axis (unit: D). The intestinal lies along the X-axis (unit: 5 mm, Y-axis the same) with intestinal walls surrounding it.  To simplify the situation, we assume that the dissipative NO molecules gathered altogether as a dot at time 0 which is located on the lower part of the intestinal walls, and the walls consume all the NO molecules within its range.
For loads of assumptions and approximations are used, this model still needs refined conditions to approach the fact. However, it has simulated the characteristic of NO diffusion in the intestinal and provides us with some information on detection of NO concentration from the calculation point of view.
Future Plan
Due to the deadline of iGEM competition, we have not implemented every single aspect of our big picture. We are sure to carry on after the Giant Jamboree. Here is our future plan.
SECTION
FUTURE PLAN
NO sensor
Determine the threshold of the sensor in simulated 'gut' tube.
Quorum sensing
Amplify NO signal with the quorum sensing system.
EGF expression
Increase secretion by modulation of signal peptides.
Therapeutic biofilm
Link more therapeutic molecules to extracellular matrix.
Kill switch
Use a better promoter (pBAD) to switch on killer genes.
Warship
Utilize 3D printing technique to reduce the size of Warship.
Intestinal Disease
EGF Expression
NO Sensor
Therapeutic Biofilm
Quorum Sensing
Kill Switch
Warship
Model
Future Plan
Back to top
iGEM2016
Copyright © 2016 ShanghaiTechChina_B iGEM Team
