<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ShanghaiTechChina_B_Lab_Work skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ShanghaiTechChina B/Lab Work</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#Intestinal_Disease">Intestinal Disease</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#EGF_Expression">EGF Expression</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#Therapeutic_Biofilm">Therapeutic Biofilm</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#NO_Sensor">NO Sensor</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#Quorum_Sensing">Quorum Sensing</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#Kill_Switch">Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#Warship">Warship</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#Model">Model</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="https://2016.igem.org/Team:ShanghaiTechChina_B/Lab_Work">Lab Work</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Lab_Work#Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Lab_Work#Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Lab_Work#Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Lab_Work#Notebook">Notebook</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2016.igem.org/Team:ShanghaiTechChina_B/Human_Practice">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Human_Practice#Introduction">Introduction</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Human_Practice#Interviews">Interviews</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Human_Practice#Survey">Survey</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Human_Practice#Social Media">Social Media</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Human_Practice#HUST_Cheering">HUST-Cheering</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Human_Practice#iDPI">iDPI</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Human_Practice#Poster">Poster</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2016.igem.org/Team:ShanghaiTechChina_B/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Collaborations#SYSU_MEDICINE">SYSU-MEDICINE</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Collaborations#FAFU_CHINA">FAFU-CHINA</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2016.igem.org/Team:ShanghaiTechChina_B/About_Us">About Us</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/About_Us#Team Roster">Team Roster</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/About_Us#Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/About_Us#Attributions">Attributions</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#" data-toggle="dropdown">Judging</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/HP/Silver">HP Silver</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/HP/Gold">HP Gold</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Hardware">Hardware</A></LI></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-9"><DIV class="bs-docs-section"><DIV class="h1" id="Parts">Parts</DIV><DIV class="p">Here are Parts we used in our project this summer.</DIV><TABLE class="table"><TBODY><TR><TD><A href="http://parts.igem.org/Part:BBa_K2152000">BBa_K2152000</A></TD><TD>EGF_HIS</TD><TD>Synthesized EGF Peptide, with his tag</TD><TD style="color:red">New Parts</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2152001">BBa_K2152001</A></TD><TD>EGF_FLAG</TD><TD>Synthesized EGF Peptide, with flag tag</TD><TD style="color:red">New Parts</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2152002">BBa_K2152002</A></TD><TD>CsgA_EGF</TD><TD>EGF Peptide incorporated into Amyloid Protein CsgA</TD><TD style="color:red">New Parts</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2152004">BBa_K2152004</A></TD><TD>T7+RBS+Gene E</TD><TD>Bacteriophage Phi X 174 lysis gene E, with T7 promoter (IPTG inducible) and RBS</TD><TD style="color:red">New Parts</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2152006">BBa_K2152006</A></TD><TD>T7+RBS+Killler Red</TD><TD>Phototoxic Red Fluorescent Protein, with T7 promoter (IPTG inducible) and RBS</TD><TD style="color:red">New Parts</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K256004">BBa_K256004</A></TD><TD>NorR</TD><TD>Nitric Oxide (NO) regulated protein NorR</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K256000">BBa_K256000</A></TD><TD>pNorV</TD><TD>Promoter of NorR system, transcription will be initiated by NorR Protein if Nitric Oxide (NO) is present</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_C0062">BBa_C0062</A></TD><TD>LuxR</TD><TD>Vibrio fischeri Quorum Sensing System LuxR Protein</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_R0061">BBa_R0061</A></TD><TD>LuxI</TD><TD>Vibrio fischeri Quorum Sensing System LuxI Protein</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_R0062">BBa_R0062</A></TD><TD>pLuxR</TD><TD>Promoter of LuxR system, transcription will be initiated by LuxR protein if AHL (produed by LuxI protein) is present</TD></TR></TBODY></TABLE><DIV class="h1" id="Safety">Safety</DIV><DIV class="p">Over the past few months, we attempted to utilize our engineered bacteria to help treating IBD. We attached great significance to safety issues--not only to our lab work, but also to the project design and products.</DIV><DIV class="h3">Lab work safety</DIV><DIV class="p">To meet our school requirement, all students have received systematic safety training from a supervisor before doing any lab work. In addition, we have read a series of courses called “The Experiment Safety” to ensure our own safety. We wore proper protective equipment (gloves, lab coats, and eyewear if necessary) at all times in the lab. As shown in Fig.1, we wore proper lab coat and gloves when performing the Nitric Oxide releasing experiment inside a biosafety hood.</DIV><DIV class="p">To meet our school requirement, all students have received systematic safety training from a supervisor before doing any lab work. In addition, we have read a series of courses called “The Experiment Safety” to ensure our own safety. We wore proper protective equipment (gloves, lab coats, and eyewear if necessary) at all times in the lab. As shown in Fig.1, we wore proper lab coat and gloves when performing the Nitric Oxide releasing experiment inside a biosafety hood.</DIV><DIV class="row"><DIV style="font-size:16px;text-align:center;width:80%;display:block;margin:10%;margin-bottom:1em;"><B>Figure 1.</B> Performing Nitric Oxide releasing experiment inside a hood.
                                 </DIV></DIV><DIV class="h3">Project safety</DIV><DIV class="p">Our team mainly worked with E.coli, which belongs to Level 1 risk group. We therefore stuck to biosafety level 1 requirement. Most notably, we have blueprinted that our engineered bacteria can applied for potential therapeutic use in human patients. We attached great significance to the safety issue of the bacteria we have modified, which might be a threat to host patients as well as the environment if without proper control. We thereby designed two kill switchesto limit their spreading at the end of our therapeutic schedule.</DIV><DIV class="p">Our first <A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#Kill_Switch">'kill switch'</A> is the killer red, a toxic fluorescent protein that kills bacteria with reactive oxygen species after light illumination. We have validated its function in stopping bacteria growth. Next, we also applied a cell lysis protein gene E to kill bacteria, and found its induced expression would certainly curtain bacteria growth. Thus, our proof-of-principle experiment demonstrated the kill switches we engineered inside of bacteria can be effective to stop bacteria spreading.</DIV><DIV class="p">In addition, we sought a physical approach by designing a <A href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#Warship">'warship'</A> to lock bacteria inside of a confined device, but still allowed exchanging chemicals with the outsides (so that bacteria can still receive nutrient and secret useful molecules). As expected, we proved that this small warship served its function in limiting bacteria spreading</DIV><DIV class="h1" id="Protocols">Protocols</DIV><DIV class="h3">Cell Cultures</DIV><OL><LI>Add 5 mL of LB fresh medium to a 10mL sterile tube or test tube.</LI><LI>Add the antibiotic to give the appropriate concentration.</LI><LI>Scoop one colony from the plate with a micropipette tip.</LI><LI>Put the colony into the sterile tube (or test tube) containing the medium and antibiotic.</LI><LI>Incubate for at least 12 h at 37ºC, with shaking at 200 rpm.</LI></OL><DIV class="h3">Cryostock of E. coli strain</DIV><OL><LI>Mix 500 µL of overnight E.coli culture with 500 µL of autoclaved 50 % glycerol</LI><LI>Store at -80 °C.</LI></OL><DIV class="h3">Competent cells</DIV><OL><LI>Take one E.coli DH5α colony and start a 5 mL overnight culture at 37°C, with shaking.</LI><LI>Dilute the overnight culture 1:100 into 50 mL LB medium, and continue the incubation for 2~3h until the OD600= 0.4-0.6</LI><LI>Let the culture sit on ice for 10 min.</LI><LI>Centrifuge at 4000 rpm for 10 min at 4°C. </LI><LI>Remove as much as possible of the supernatant without disturbing the pellet.</LI><LI>Resuspend the pellet in 10mL of ice-cold 100mM CaCl<SUB>2</SUB> solution.</LI><LI>Incubate on ice for 15 min</LI><LI>Centrifuge at 4000 rpm for 10 min at 4°C.</LI><LI>Again, remove as much as possible of the supernatant without disturbing the pellet, and resuspend the pellet in 10ml of ice-cold 100mM CaCl<SUB>2</SUB>/20% glycerol solution.</LI><LI>Incubate on ice for 10 min.</LI><LI>Aliquot in 40 µL amounts to chilled 1.5 ml tubes.</LI><LI>Store immediately at -80ºC</LI></OL><DIV class="h3">Transformation</DIV><OL><LI>Take the competent cells from the storage at -80ºC and thaw them on ice for 10 min.</LI><LI>Add 5 µL of plasmid solution, or 10ul of ligation reaction mixture, to the 50 µL of competent cells.</LI><LI>Incubate for 30 min on ice.</LI><LI>Heat shock the cells at 42ºC for 90s</LI><LI>Incubate for 5 min on ice. </LI><LI>Add 1000 µL of LB medium. </LI><LI>Incubate for 60 min at 37ºC, with shaking.</LI><LI>Spin cells down at 6000 rpm for 1 min.</LI><LI>Discard all but ~100 µL of the supernatant and resuspend the pellet in the remaining solution.</LI><LI>Plate the remaining suspension on agar plate.</LI></OL><DIV class="h3">Plasmid DNA Purification (from TIANGEN <SUP>®</SUP> Kit)</DIV><OL><LI>Add 1-5 mL of E. coli culture in LB medium in to a 1.5-2 mL tube. Centrifuge at 12000 rpm for 1 min at room temperature. Remove the supernatant, and add 600 µL of MQ water to the pellet</LI><LI>Add 250 µL of Resuspension Solution P1 to the pelleted cells (Ensure RNase A has been added to the Resuspension Solution P1), and resuspend the pellet completely.</LI><LI>Add 250 µL of the Lysis Solution P2, and mix thoroughly by inverting the tube 6-8 times until the solution becomes viscous and slightly clear.</LI><LI>Add 350 µL of the Neutralization Solution P3 and mix immediately and thoroughly by inverting the tube 6-8 times. The neutralized bacterial lysate should become cloudy. </LI><LI>Centrifugate at 12000 rpm for 10 min.</LI><LI>Transfer the supernatant to the supplied spin column CP3. Avoid disturbing or transferring the white precipitate. </LI><LI>Centrifugate at 12000 rpm for 1 min, and discard the flow-through and place the spin column back into the same collection tube.</LI><LI>Optional, for EndA+ strains only, not for DH5α, TOP10: Add 500 µL of Wash Solution PD, and centrifuge for 1 min. Discard the flow-through and place the spin column back into the same collection tube.</LI><LI>Add 600 µL of the Wash Solution PW (Ensure ethanol has been added to the Wash Solution PW), and centrifuge for 1 min. Discard the flow-through and place the spin column back into the same collection tube.</LI><LI>Repeat the wash procedure (step 9) using 600 µL of the Wash Solution PW</LI><LI>Centrifuge for an additional 2 min to remove residual Wash Solution PW. This step is essential to avoid residual ethanol in plasmid preps.</LI><LI>Transfer the spin column into a fresh 1.5 mL tube. Add 50-100 µL of the Elution Buffer EB to the center of spin column membrane to elute the plasmid DNA. Incubate for 2 min at room temperature and centrifuge at 12000 rpm for 2 min.</LI><LI>Store the purified plasmid DNA at -20°C.</LI></OL><DIV class="h3">PCR Clean-up (from TIANGEN ® Kit)</DIV><OL><LI>Estimate the volume of PCR mixture or restriction reaction solution. Add Purification Buffer PB, with 5 times of the volume of PCR mixture solution, and mix thoroughly. i.e. Add 250 µL of Purification Buffer PB to 50 µL of PCR mixture solution.</LI><LI>Transfer the solution to a spin column CB2. Incubate for 2 min at room temperature and centrifuge at 12000 rpm for 1 min. Discard the flow-through and place the spin column back into the same collection tube.</LI><LI>Add 600 µL of the Wash Solution PW (Ensure ethanol has been added to the Wash Solution PW), and centrifuge for 1 min. Discard the flow-through and place the spin column back into the same collection tube.</LI><LI>Repeat the wash procedure (step 4) using 600 µL of the Wash Solution PW</LI><LI>Centrifuge for an additional 2 min to remove residual Wash Solution PW. This step is essential to avoid residual ethanol in plasmid preps.</LI><LI>Transfer the spin column into a fresh 1.5 mL tube. Add 30-50 µL of the Elution Buffer EB to the center of spin column membrane to elute the plasmid DNA. Incubate for 2 min at room temperature and centrifuge at 12000 rpm for 2 min.</LI><LI>Store the purified DNA at -20°C.</LI></OL><DIV class="h3">Restriction (NEB enzymes)</DIV><OL><LI>Calculate the volume of the DNA sample containing 1000 ng and the volume of ddH2O to make up to 50 µl.</LI><LI>Mix the components below.</LI><TABLE class="table"><TBODY><TR><TD>ddH2O</TD><TD>μL</TD></TR><TR><TD>DNA sample</TD><TD>1mg</TD></TR><TR><TD>NEB buffer (Cutsmart/3.1)</TD><TD>5 μL</TD></TR><TR><TD>NEB restriction enzyme 1</TD><TD>1 μL</TD></TR><TR><TD>NEB restriction enzyme 2</TD><TD>1 μL</TD></TR><TR><TD>Total</TD><TD>50 μL</TD></TR></TBODY></TABLE><LI>Incubate at 37°C for 3 h to overnight (despite the recommendation of only 15 min by NEB).</LI><LI>Use the PCR purification kit or gel isolation kit to purify the restriction product.</LI></OL><DIV class="h3">DNA Electrophoresis</DIV><DIV class="h4">Casting a 30 ml gel</DIV><OL><LI>For a 1% 30ml agarose gel, weight 0.3 g of agarose in a conical flask.</LI><LI>Add 30 ml 1X TAE buffer.</LI><LI>Microwave for 1~3 min, and then let the agarose solution cool down.</LI><LI>Add 4 µL of Gel-Green DNA stain to the solution.</LI><LI>Pour the gel solution into the gel tray. Remove any air bubbles with a micropipette tip and put in comb.</LI><LI>Let the gel solidify.</LI></OL><DIV class="h4">Running the gel</DIV><OL><LI>Remove the comb, and place the gel tray into the TAE buffer chamber.</LI><LI>Mix the DNA sample (~2 µl) with loading dye (1 µl). This can be done either in PCR tubes, or on a piece of parafilm.</LI><LI>Load the samples into the wells. After that, load 2µl of DNA marker into the middle well.</LI><LI>Run at 90-130V for 20-30 min. (depends on the length of DNA sample)</LI></OL><DIV class="h3">Gel Extraction (from MACHEREY-NAGEL<SUP>®</SUP> Kit)</DIV><OL><LI>Visualize the DNA in the agarose gel using a long-wavelength UV lamp. Minimize UV exposure time to avoid damaging the DNA. </LI><LI>Take a clean scalpel to excise the DNA fragment from an agarose gel. Remove all excess aga.rose. Weigh the gel slice and transfer it to a clean 1.5 mL tube.</LI><LI>For each 100 mg of agarose gel add 200 μL Buffer NTI (Membrane Binding Solution).</LI><LI>Incubate mixture solution for 5–10 min at 50 °C. Vortex the mixture solution every 2–3 min until the gel slice is completely dissolved.</LI><LI>Place a “NucleoSpin<SUP>®</SUP> Gel and PCR Clean-up” column into a 2 mL collection tube, and load up to 700 μL sample. </LI><LI>Centrifuge at 12,000rpm for 30 s. Discard flow-through and place the column back into the collection tube. Load remaining sample if necessary and repeat the centrifugation step.</LI><LI>Add 700 μL Buffer NT3 (Membrane Wash Solution) to the column. Centrifuge at 12000rpm for 30 s. Discard flow-through and place the column back into the collection tube.</LI><LI>Centrifuge at 12000rpm for 1 min to remove Buffer NT3 completely. </LI><LI>Place the column into a fresh 1.5 mL tube. Add 15–30 μL of the Buffer NE and incubate at room temperature for 1 min. Centrifuge at 12000 rpm for 1 min.</LI><LI>Store the purified DNA at -20°C.</LI></OL><DIV class="h3">Ligation</DIV><OL><LI>Calculate the volume of the Vector DNA &amp; Fragment DNA, and the volume of ddH2O to make up to 20 µl. The mole ratio of Vector DNA/Fragment DNA should be 1:10-1:100.</LI><LI>Mix the components below.</LI><TABLE class="table"><TBODY><TR><TD>ddH2O</TD><TD>μL</TD></TR><TR><TD>Fragment DNA</TD><TD>μL</TD></TR><TR><TD>Vector DNA</TD><TD>μL</TD></TR><TR><TD>T4 ligase 10X buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 ligase</TD><TD>1 μL</TD></TR><TR><TD>Total</TD><TD>20 μL</TD></TR></TBODY></TABLE><LI>Incubate for 3h at 37ºC(NEB), or overnight at 16ºC. (TAKARA)</LI><LI>Transform competent cells. Or use the PCR purification kit or gel isolation kit to purify the ligation product and store at -20 ºC.</LI></OL><DIV class="h3">Sonication Cell Lysis</DIV><OL><LI>Pellet 300 mL of overnight IPTG induced E.coli BL21 culture by centrifugation twice at 6000rpm for 5 min. Remove the supernatants.</LI><LI>Resuspend the cell pellet in 30 mL of PBS.</LI><LI>Use the ultrasonic cell crusher to lysis cells for 1 h. Set the apparatus at 50% of maximum power and apply the bursts for every 5 seconds.</LI><LI>Centrifuge at 20000 rpm for 5 min. Check the color of the pellet as it may tell the effectiveness of the cell lysis.</LI><LI>Take the supernatant, and store at -20 ºC.</LI></OL><DIV class="h3">IPTG Inducted Expression</DIV><OL><LI>Grow a BL21 strain at 37°C overnight.</LI><LI>Dilute the overnight culture with 50X and grow the culture at 37°C for 1.5h.</LI><LI>Measure the OD600. If the OD600 is between 0.35 and 0.6, the induction can be made.</LI><LI>Induce the cells with IPTG with a final concentration of 0·5mM IPTG. The induction can be performed at 37°C.</LI><LI>After 4-8 h of induction, collect the BL21 culture sample.</LI><LI>Lysis the cell (Protocol Sonication Cell Lysis), or store at -20 ºC.</LI></OL><DIV class="h3">Make the separating gel for SDS-PAGE</DIV><OL><LI>Prepare the gel solution with appropriate concentration of Acylamide.</LI><LI>Add appropriate amount of separating gel solution into the gap between the glass plates.</LI><LI>Fill in water into the gap between the glass plates until an overflow, to ensure the top of the separating gel be horizontal.</LI><LI>Let the gel solidify.</LI></OL><DIV class="h3">Make the stacking gel for SDS-PAGE</DIV><OL><LI>Discard the water.</LI><LI>Add in stacking gel until an overflow.</LI><LI>Put in the comb without trapping air under the teeth, and let the gel solidify.</LI><LI>Move out the comb. Take the glass plates out of the casting frame and set them in the cell buffer dam. Pour the running buffer into the inner chamber and keep pouring after overflow until the buffer surface reaches the required level in the outer chamber.</LI></OL><DIV class="h3">Run the SDS-PAGE</DIV><OL><LI>Mix the samples with loading buffer, and then heat them in boiling water for 5-10 min.</LI><LI>Load prepared samples into wells.</LI><LI>Load the protein marker into the first lane.</LI><LI>Run the gel for 1–2 h at 100 V.</LI></OL><DIV class="h3">Western Blotting</DIV><OL><LI>Inducing the BL21 strain to express the EGF-Flag for 4 h. (Protocol Inducted Expression)</LI><LI>Collect the BL21 culture and lysis it (Protocol Sonication Cell Lysis), and prepare for SDS-PAGE gel.</LI><LI>Run SDS-PAGE gel with cell lysis samples and protein marker. (Protocol SDS-PAGE)</LI><LI>Transfer protein from gel to membrane:</LI><LI>activate membrane in Methanol for 2min. Soak membrane, filter paper and gel in transfer buffer for 2 min.</LI><LI>Make sandwich: filter bottom paper, membrane, gel (upside down), filter top paper. Soak the sandwich. Move out bubbles with roller wheel.</LI><LI>Transfer at 25V for 10 min, using Bio-Rad Trans-Blot® Transfer system.</LI></OL><DIV class="h3">Development blot with antibody:</DIV><OL><LI>Block the membrane for 1 h at room temperature using blocking buffer.</LI><LI>Incubate the membrane in proper dilution of primary antibody overnight at 4°C with shaking.</LI><LI>Wash the membrane in three washes of TBST for 10 min each.</LI><LI>Incubate the membrane in proper dilution of second antibody (with HRP) for 2 h with shaking.</LI><LI>Again, wash the membrane in three washes of TBST for 10 min each.</LI><LI>Detect the blot with ECL western blotting reagents.</LI></OL><DIV class="h3">Detect the Nitric Oxide with Griess Reagents (from Beyotime ® Kit)</DIV><OL><LI>Warm the Griess Reagent I and II to room temperature.</LI><LI>Dilute the tested sample with standard 1M NaNO2 sample, to make the standard NO gradient samples.</LI><LI>Add 50 μL of every tested sample and standard NO gradient sample to each well in the 96 well plate.</LI><LI>Add 50 μL of Griess Reagent I to each well.</LI><LI>Add 50 μL of Griess Reagent II to each well.</LI><LI>Measure the absorbance(Abs) at 540 nm, using the microplate reader.</LI><LI>Plot the standard curve, and calculate the concentration of Nitric Oxide in the tested sample.</LI></OL><DIV class="h1" id="Notebook">Notebook</DIV><DIV class="h3">Nitric Oxide Sensor</DIV><TABLE class="table"><TBODY><TR><TD style="width:15%;">Week 1 (Jul.3)</TD><TD>PCR cloning pNorV from PUC57, NorR from PCCI-4K</TD></TR><TR><TD style="width:15%;">Week 2-6</TD><TD>Insert NorR into pETDuet-1</TD></TR><TR><TD style="width:15%;">Week 7</TD><TD>Insert GFP into pETDuet-1</TD></TR><TR><TD style="width:15%;">Week 8</TD><TD>Measure the release of nitric oxide by SNP</TD></TR><TR><TD style="width:15%;">Week 9-12</TD><TD>Insert pNorV into pETDuet-1</TD></TR><TR><TD style="width:15%;">Week 14</TD><TD>Circuit verification: transform plasmids into BL21(DE3) and test Nitric oxide induced GFP expression</TD></TR></TBODY></TABLE><DIV class="h3">EGF</DIV><TABLE class="table"><TBODY><TR><TD style="width:15%;">Week 1 (Jul.3)</TD><TD>PCR of EGF-His/EGF-Flag from PCCI-4K</TD></TR><TR><TD style="width:15%;">Week 2-4</TD><TD>Insert EGF into pET 28-a</TD></TR><TR><TD style="width:15%;">Week 5-6</TD><TD>Insert EGF into pETDuet-1</TD></TR><TR><TD style="width:15%;">Week 7</TD><TD>MeExpression of EGF</TD></TR><TR><TD style="width:15%;">Week 9</TD><TD>InPurify EGF-His protein and send it to SYSU-MEDICINE</TD></TR></TBODY></TABLE><DIV class="h3">Biofilm</DIV><TABLE class="table"><TBODY><TR><TD style="width:15%;">Week 1 (Aug.7)</TD><TD>PCR of CsgA and EGF fragments</TD></TR><TR><TD style="width:15%;">Week 2</TD><TD>Gibson cloning to link CsgA and EGF</TD></TR><TR><TD style="width:15%;">Week 3-4</TD><TD>Construct of CsgA-EGF into the plasmid from Allen Chen</TD></TR><TR><TD style="width:15%;">Week 5</TD><TD>Biofilm expression: Congo red staining of biofilm</TD></TR><TR><TD style="width:15%;">Week 6</TD><TD>Biofilm expression: Biofilm observation under transmission Electro microscope</TD></TR><TR><TD style="width:15%;">Week 7-8</TD><TD>Construct CsgA-EGF standard part</TD></TR></TBODY></TABLE><DIV class="h3">Quorum Sensing</DIV><TABLE class="table"><TBODY><TR><TD style="width:15%;">Week 1 (Jul.3)</TD><TD>PCR of luxR, and luxI from pUC57; synthesize pluxR cDNA</TD></TR><TR><TD style="width:15%;">Week 2-4</TD><TD>Insert luxI into pET28-a</TD></TR><TR><TD style="width:15%;">Week 5-7</TD><TD>Insert luxR into pETDuet-1</TD></TR><TR><TD style="width:15%;">Week 8-9</TD><TD>Insert GFP into pETDuet-1</TD></TR><TR><TD style="width:15%;">Week 9-13</TD><TD>Insert pluxR into pETDuet-1 (failed)</TD></TR></TBODY></TABLE><DIV class="h3">Kill Switch</DIV><TABLE class="table"><TBODY><TR><TD style="width:15%;">Aug.12</TD><TD>Sequence killer red plasmid from the kit</TD></TR><TR><TD style="width:15%;">Aug.15-20</TD><TD>PCR of killer red sequences from pSB1C3</TD></TR><TR><TD style="width:15%;">Sep.2-6</TD><TD>Insert killer red into pET-28a</TD></TR><TR><TD style="width:15%;">Sep.7</TD><TD>Sequence killer red by T7 primers</TD></TR><TR><TD style="width:15%;">Sep.8</TD><TD>Transform killer red into BL21(DE3) PLysS</TD></TR><TR><TD style="width:15%;">Sep.9-15</TD><TD>Induce expression of killer red and test whether green light will kill baceteria</TD></TR><TR><TD style="width:15%;">Sep.16-20</TD><TD>Take pictures of killer red bacteria by confocal fluorescence microscopy</TD></TR><TR><TD style="width:15%;">Sep.21-Oct.10</TD><TD>Prove the cell killing function of killer red and repeat experiment</TD></TR></TBODY></TABLE><DIV class="h3">Kill switch—gene E</DIV><TABLE class="table"><TBODY><TR><TD style="width:15%;">Aug.31</TD><TD>Receive gene E in pBV220 (under a heat inducible promoter) from a lab in Shanghai Advanced Research Institute, Chinese Academy of Sciences</TD></TR><TR><TD style="width:15%;">Sep.13</TD><TD>Test the efficiency of gene E mediated bacteria lysis under two different temperature (28℃ and 42℃)</TD></TR><TR><TD style="width:15%;">Sep.29</TD><TD>Add another experimental group (induction in 37℃)</TD></TR><TR><TD style="width:15%;">Oct.5</TD><TD>Insert gene E into the multiple clone site of pet28a and induce the expression of gene E with IPTG</TD></TR><TR><TD style="width:15%;">Oct.7</TD><TD>Test cell lysis by IPTG induction in pet28a expressing gene E</TD></TR><TR><TD style="width:15%;">Oct.10</TD><TD>Insert T7 promoter system and gene E into the standard plasmid pSB1C3</TD></TR></TBODY></TABLE><DIV class="h3">Warship</DIV><TABLE class="table"><TBODY><TR><TD style="width:15%;">Week 1 (Aug.6)</TD><TD>Design the structure and test the material</TD></TR><TR><TD style="width:15%;">Week 2-4</TD><TD>Test its leakiness to bacteria and permeability to small molecules</TD></TR></TBODY></TABLE><DIV class="h3">Collaboration with FAFU-CHINA</DIV><TABLE class="table"><TBODY><TR><TD style="width:15%;">Week 1 (Oct.2)</TD><TD>Express Cyt-1</TD></TR><TR><TD style="width:15%;">Week 2</TD><TD>Purify Cyt-1 and send back to FAFU-CHINA</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-md-3"><DIV class="bs-docs-sidebar hidden-print hidden-xs hidden-sm"><UL class="nav bs-docs-sidenav"><LI><A href="#Parts">Parts</A></LI><LI><A href="#Safety">Safety</A></LI><LI><A href="#Protocols">Protocols</A></LI><LI><A href="#Notebook">Notebook</A></LI><LI><A class="back_to_top" href="#top">Back to top</A></LI></UL></DIV></DIV></DIV></DIV><FOOTER class="bs-docs-footer"><DIV class="container"><UL class="bs-docs-footer-links"><LI>iGEM2016</LI><LI>Copyright © 2016 ShanghaiTechChina_B iGEM Team</LI></UL></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>