<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cooper_Union_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cooper Union/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Cooper Union 2015 iGEM</TITLE></P><DIV class="parent-container"><DIV class="menu-parent"><UL class="menu"><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union"> Home </A></LI><LI class="menu-project"> Project
		</LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/DeNovoSynthesis">De Novo Synthesis </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Loomino_Description">Loomino Design</A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Experiments">Experiments and Protocols </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Results">Results </A></LI></UL><LI class="menu-team"> Team
		</LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Team">Members </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Attributions"> Attributions </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Collaborations"> Collaborations </A></LI><LI class="menu-parts"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Parts">Biobrick Parts</A></LI><LI class="menu-notebook"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Notebook"> Notebook </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Practices"> Human Practices </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Safety">Safety </A></LI></DIV><DIV class="text"><H2> Project Results</H2><H3>TdT Variants</H3><P>Sequences for the three different forms of TdT were designed and ordered.  We successfully cloned each of these sequences into the biobrick vector pSB1C3 and the expression vector pET28.  Samples were sent out for sequencing to confirm that the proper sequences were present in the plasmids.  The biobrick plasmids were sent to igem while the pET28 vectors were transformed into the Rosetta cell line.  Unfortunately, this is as far as we got with this portion of the project.  At this point, we only need to purify the proteins from the Rosetta cells and test them on oligonucleotides to make sure they function properly.  From there, we would compare their efficiency with commercially available TdT using next gen sequencing technology.</P><H3>Glass Slides</H3><P>To test whether or not we had DNA on our glass slides we treated them with ethidium bromide (EtBr).  Since EtBr is known to attach to DNA strands, any glass slides with DNA on them should also have EtBr.  Exposing these slides to ultraviolet light would make the EtBr fluoresce allowing us to determine which slides had DNA present.  Figure 1 shows that EtBr localized in the regions where DNA was added to the slides.  On each slide, the number of oranges circles corresponds to the number and location of the regions DNA was added to.  This is strong evidence that DNA was present on these slides.  Figure 2 shows 4 different glass slides that underwent a wash step similar to the ones that would be done by the loomino device.  The top two slides were negative controls and demonstrate that no residual EtBr was present in the absence of DNA bonded to the glass.  The bottom two slides show slight orange fluorescence highlighted by the black boxes superimposed on the image.  This indicates that there is at least some DNA present on these slides after the wash.  It is unclear whether or not the DNA washed off or just the EtBr.</P><CENTER><H4>Figure 1</H4></CENTER><CENTER><H4>Figure 2</H4></CENTER><H3>RNA Ligation</H3><P>To recap, the RNA Ligation is an important step our system because it provides mean for PCR amplification of the newly assembled oligonucleotide. After De Novo Synthess via Terminal Deoxynucleotidyl Transferase, a single stranded oligonucleotide with a known primer binding site is ligated to the end of the TdT amplified oligonucleotide. Then, appropriate primers and PCR elements are added to the reaction chamber to amplify this newly formed sequence of DNA. The following gel shows the results from our RNA ligation reaction. In general, two ligations were set up in this reactions. The first kind involves a ligation between a 52mer(meaning 52 bp long) and a 23mer with a 5' phosphate. As a control, the second type of reaction is between a 52mer and a 43mer without any extra phosphate(This was done to also show that an extra phosphate was necessary for the ligation of both oligos). In addition left over un-ligated components, lanes 5, 6, 8, and 9 exhibit the same band above the 52mer. It is possible to state that this is the the newly ligated oligonucleotide, a combination of both the 52mer and the 23mer. It is still unclear why the RNA Ladder in the first well is having difficulty running properly; it could be because of some sort of degradation of the product.</P><CENTER><H4>Figure 2</H4></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>