<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cooper_Union_DeNovoSynthesis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cooper Union/DeNovoSynthesis</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Cooper Union 2015 iGEM</TITLE></P><DIV class="parent-container"><DIV class="menu-parent"><UL class="menu"><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union"> Home </A></LI><LI class="menu-project"> Project
		</LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/DeNovoSynthesis">De Novo Synthesis </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Loomino_Description">Loomino Design</A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Experiments">Experiments and Protocols </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Results">Results </A></LI></UL><LI class="menu-team"> Team
		</LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Team">Members </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Attributions"> Attributions </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Collaborations"> Collaborations </A></LI><LI class="menu-parts"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Parts">Biobrick Parts</A></LI><LI class="menu-notebook"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Notebook"> Notebook </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Practices"> Human Practices </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Safety">Safety </A></LI></DIV><DIV class="text"><H2> Background on Last Year's Project </H2><P>The following diagram summarizes the process used to synthesize a DNA sequence using the Terminal deoxynucleotidyltransferase (TdT).  TdT is an enzyme that adds nucleotides to the three prime end of a DNA molecule.  The idea of creating a DNA synthesizer using this enzyme was originally conceived by Ud-Dean et al. in the paper a theoretical model for template-free synthesis
of long DNA sequence[1].  Our project builds off of and improves this idea in a number of ways.  One such improvement was the idea of using heat-labile nucleotides.  Heat-labile nucleotides are nucleotides with a blocking group on their three prime end that prevents polymerases like TdT from adding to them.  Conveniently, this blocking group falls off of the nucleotide when it is heated, thus the name heat-labile.  By using heat-labile nucleotides, TdT can add a single base pair at a time.  By cycling the heating, washing, and addition of different nucleotides in the proper order any sequence can be created.</P><H2> Project Description for Biological Components </H2><P>The biology component of our project focused on two main concepts crucial to the functioning of the complete device.  The first of these was to test the idea of bonding DNA to a glass microscope slide as a means of preventing it from being lost in wash steps.  We also sought to characterize different variants of Terminal deoxynucleotidyltransferase (TdT) to find one best suited for the synthesizer.  A third but less pursued topic of study was the possibility of making a heat stable TdT variant using an intein system to circularize the enzyme.</P>
	We considered a number of chemical means by which one can bind DNA to glass [2].  The system we chose involved bonding DNA with a five prime thiol modified group to a silanized glass slide.  This method was chosen because the silanization process could be carried out in-house and adding a modified thiol group to the five prime end of DNA is a relatively inexpensive modification.  The methodology was based on a paper by Rogers et al. at Johns Hopkins [3].  After the reaction is complete, the DNA should be bonded to the glass by a disulfide bond.  To test that our glass slides had DNA on them, we would use ethidium bromide since it is known to attach to DNA strands.
	In parallel with the glass slides, our team designed three mutated TdT sequences based on a paper by Repasky et al.[4]   In order to characterize these forms of TdT, we would have to ligate them into a high copy expression vector and then transform them into a cell line optimized for expression.  Protein purification of these cells would then follow.  Finally, we would have to test this protein by treating oligonucleotides with the TdT and running them on a polyacrylamide gel.  This would reveal whether or not the TdT can add base pairs to the free three prime end of a DNA molecule.
	The intein based TdT system was the last biological component of our project.  It is a useful addition if the synthesizer utilizes heat-labile nucleotides.  Ideally, the final system will utilize U.V.-labile nucleotides which would make this system superfluous.  Despite this, we still worked on this angle of the project since U.V.-labile nucleotides can be very expensive.  In order to circularize TdT, we would have to design a linker sequence between the N and C termini of the protein.  We decided to use the software developed by the 2014 Heidelberg team to design an optimized sequence of amino acids to link the N-terminus of TdT to its C-terminus.  Their program yielded an amino acid sequence that would circularize the TdT protein.  Using this sequence and Heidelberg’s intein system, we designed g-block DNA sequences that would allow for the synthesis and purification of various forms of heat stable TdT.  <H3>References</H3><P>1. Ud-Dean et al. A theoretical model for template-free synthesis of long DNA sequence. Syst Synth Biol 2:67–73 (2008).</P>
2. Strategies for Attaching Oligonucleotides to Solid Supports. Integrated DNA Technologies (2014).
3. Rogers et al. Immobilization of Oligonucleotides onto a Glass Support via Disulfide Bonds: A Method for Preparation of DNA Microarrays. Analytical Biochemistry 266, 23–30 (1999).
4. Repasky et al. Mutational Analysis of Terminal Deoxynucleotidyltransferase Mediated N-Nucleotide Addition in V(D)J Recombination. Journal of Immunology 172(9):5478-88 (May 2004).</DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>