<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cooper_Union_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cooper Union/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Cooper Union 2015 iGEM</TITLE></P><DIV class="parent-container"><DIV class="menu-parent"><UL class="menu"><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union"> Home </A></LI><LI class="menu-project"> Project
		</LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/DeNovoSynthesis">De Novo Synthesis </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Loomino_Description">Loomino</A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Experiments">Experiments and Protocols </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Results">Results </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Design">Design </A></LI></UL><LI class="menu-team"> Team
		</LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Team">Members </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Attributions"> Attributions </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Collaborations"> Collaborations </A></LI><LI class="menu-parts"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Parts">Biobrick Parts</A></LI><LI class="menu-notebook"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Notebook"> Notebook </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Practices"> Human Practices </A></LI><LI class="menu"><A class="menu" href="https://2015.igem.org/Team:Cooper_Union/Safety">Safety </A></LI></DIV><DIV class="text"><DL><DT><A href="#bmedia">1 - Bacteria Media w/ agar (solid media)</A></DT><DT><A href="#dyna">2 - Dynabead Prep</A></DT><DT><A href="#btrans">3 - Bacterial Transformation</A></DT><DT><A href="#cpcrpsb">4 - Colony PCR of pSB1C3 with TdT Mutant Inserts</A></DT><DT><A href="#dynaclean">5 - Dynabead Cleanamp</A></DT><DT><A href="#dd">6 - Double Digest of PET, PSB, TdT variants</A></DT><DT><A href="#slides">7 - Immobilizing Disulfide Modified Oligos to Mercaptosilane Treated Glass Slides</A></DT><DT><A href="#rnalig">8 - RNA Ligase Reaction</A></DT><DT><A href="#gelex">9 - Gel Extraction</A></DT><DT><A href="#mini">10 - Miniprep</A></DT><DT><A href="#pure">11 - PCR Purification</A></DT><DT><A href="#nucadd">12 - TdT Nucleotide Addition</A></DT><DT><A href="#phosphor">13 - Non-Radioactive phosphorlyation with T4 PNK</A></DT><H2>Experiments &amp; Protocols</H2><H3>1- Bacteria Media w/ agar (solid media)</H3><H5>Materials</H5><LI>500mL bottle</LI><LI>Deionized Water</LI><LI>Agar</LI><LI>LB Broth</LI><LI>Chloramphenicol</LI><LI>Yeast Extract</LI><LI>Bacto Peptone</LI><H5>Procedure</H5><OL type="”1”"><LI>Add 250mL of deionized water to the 500mL bottle. </LI><LI>Weigh out 3.625g of agar using the weighing boat and add it to the 250mL of water.</LI><LI>Weigh out 5.0g of LB Broth using the weighing boat and add it to the 250mL of water.</LI><LI>Screw lid on bottle loosely.</LI><LI>Place autoclave tape on the lid of the bottle.</LI><LI>Place tray with approximately 1cm of water in height into the autoclave. </LI><LI>Place 500mL bottle into autoclave with &quot;liquid&quot; selected and press &quot;start&quot;.</LI><LI>After an hour, remove bottle from autoclave. Let bottle sit for approximately 30 minutes.</LI><LI>Add 370uL of chloramphenicol to the 250mL solution.</LI><LI>Pour solution into petri dishes, covering the bottom of the dish.</LI><LI>Place two stripes on each dish.</LI><LI>Let dishes sit overnight.</LI></OL><A href="#top">Top of Page</A><H3>2 - Dynabead Prep</H3><H5>Materials</H5><LI>B&amp;W wash Buffer (2X and 1X)</LI><LI>Dynabeads</LI><LI>DNA solution</LI><LI>Magnets</LI><LI>PCR Tubes</LI><H5>Procedure</H5><OL type="”1”"><LI>Pipette 50 microL of dynabeads in PCR tubes (3 of them)</LI><LI>Wash beads by pipetting in 200 microL of 1X B&amp;W buffer</LI><LI>Vortex PCR tubes for about 30 seconds</LI><LI>Place a magnet at the bottom of the tube until a solid pellet of dynabeads forms at the bottom</LI><LI>Pipette out the supernatant leaving the pellet intact at the bottom</LI><LI>Repeat steps 2-5 two times (performed a total 3 times)</LI><LI>To each tube add 100 microL 2X buffer, 5 microL DNA (100 microM), and 95 microL of water</LI><LI>Incubate on stirring apparatus/agitator at room temperature for 15 minutes</LI><LI>Wash the beads again by repeating steps 2-5 three times</LI><LI>Add 105 microL 1X buffer to the tubes to resuspend the beads</LI><LI>Freeze PCR tubes at -20 degrees Celsius until they are needed </LI></OL><A href="#top">Top of Page</A><H3>3 - Bacterial Transformation</H3><LI>For C2987H: Thaw a tube of NEB 5-alpha Competent E. coli cells on ice for 10 minutes.</LI><LI>For C2987I: Thaw a tube of NEB 5-alpha Competent E. coli cells on ice until the last ice crystals disappear. Mix gently and carefully pipette 50 µl of cells into a transformation tube on ice.</LI><LI>Add 1-5 µl containing 1 pg-100 ng of plasmid DNA to the cell mixture. Carefully flick the tube 4-5 times to mix cells and DNA. Do not vortex.</LI><LI>Place the mixture on ice for 30 minutes. Do not mix.</LI><LI>Heat shock at exactly 42°C for exactly 30 seconds. Do not mix.</LI><LI>Place on ice for 5 minutes. Do not mix.</LI><LI>Pipette 950 µl of room temperature SOC into the mixture.</LI><LI>Place at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.</LI><LI>Warm selection plates to 37°C.</LI><LI>Mix the cells thoroughly by flicking the tube and inverting, then perform several 10-fold serial dilutions in SOC.</LI><LI>Spread 50-100 µl of each dilution onto a selection plate and incubate overnight at 37°C. </LI><LI>Alternatively, incubate at 30°C for 24-36 hours or 25°C for 48 hours.</LI><A href="#top">Top of Page</A><H3>4 - Colony PCR of pSB1C3 with TdT Mutant Inserts</H3><H5>Materials</H5><LI>VR Primer</LI><LI>VF2 Primer</LI><LI>pSB1C3 colonies</LI><LI>ddH20</LI><LI>10 mm dNTPs</LI><LI>Taq Polymerase</LI><LI>10X Standard Taq Polymerase Buffer</LI><LI>6X Loading Dye</LI><LI>DNA Ladder</LI><LI>1% Agarose Gel</LI><H5>Procedure</H5><OL type="”1”"><LI>Use a pipette tip to scrape off a colony and place the colony in 200 microL of ddH20</LI><LI>Take 21 microL of bacteria water and add it to a PCR tube</LI><LI>Add half a microliter of primer VR and another half microliter of primer VF2 to PCR tube</LI><LI>Add half a microliter of 10 mm dNTPs, 0.125 microliters of Taq Polymerase, and the appropriate amount of Standard Taq Polymerase Buffer (Usually 2.5 for a 25 microliter reaction)</LI><LI>Run in PCR machine with desired thermal cycling</LI><LI>Prepare a 1% agarose gel if one is not available</LI><LI>Prepare sample for gel electrophoresis using a proper proportion of 6X loading dye to sample </LI><LI>Load DNA ladder and sample into gel and run gel</LI><LI>Analyze gel using U.V. light and determine the approximate length of the amplified sequence</LI></OL><A href="#top">Top of Page</A><H3>5 - Dynabead Cleanamp</H3><H5>Materials</H5><LI>10x TdT buffer</LI><LI>2.5 mM CoCl2</LI><LI>Oligo</LI><LI>50 mM CleanAmp dATP</LI><LI>100 mM dATP</LI><LI>TdT</LI><LI>dH2O</LI><H5>Procedure</H5><OL type="”1”"><LI>Preheat water baths to 37 celsius and 95 celsius.
</LI><LI>Mix:
- 5 µl 10x TdT buffer,
- 5 µl 2.5 mM CoCl2,
- 1.3 µl of 43 mer oligo ( 1 µg at 813 ng/µl), 
- 0.4 µl 50 mM CleanAmp dATP,
- 1 µl TdT ( 20 units at 20000 units/mL),
- dH2O to final volume of 50 mL.
</LI><LI>Split into two separate equal volume aliquots.
</LI><LI>Incubate at 37 celsius for 30 minutes.
</LI><LI>Incubate 1 aliquot at 95 celsius for 15 minutes.
</LI><LI>For the non-CleanAmp dATP, replace CleanAmp dATP with 0.2 µl 100 mM dATP.
</LI><LI>For the negative control, remove the volume of TdT.
</LI><LI>Store at -20 celsius.
</LI></OL><A href="#top">Top of Page</A><H3>6 - Double Digest of PET, PSB, TdT variants</H3><H5>Materials</H5><LI>PET28</LI><LI>PSB1C3</LI><LI>Nuclease-free water</LI><LI>EcoR1-HF</LI><LI>Pst</LI><LI>XbaI</LI><LI>BamhI</LI><LI>TdT (TdT del1-27, del26-143, &amp; GIP subAAA 213-215)</LI><LI>NEBuffer 3.1</LI><LI>NEBuffer 2.1</LI><H5>Procedure</H5><OL type="”1”"><LI>Digest of PET28 and PSB1C3</LI><OL type="”i”"><LI>Mix together these materials in this order for PET28 tube:</LI><OL type="”a”"><LI>3.1 buffer---5uL</LI><LI>Water---32.2uL</LI><LI>XbaI---1uL</LI><LI>BamhI---1uL</LI><LI>PET28---10.8uL </LI></OL><LI>Mix together these materials in this order for PSB1C3 tube:</LI><OL type="”a”">
​		<LI>2.1 buffer---5uL</LI><LI>Water---31.2uL </LI><LI>EcoRI---1uL</LI><LI>PstI---1uL</LI><LI>PSB1C3---11.8uL</LI></OL><LI>Incubate both tubes at 37˚C for 1 hour</LI></OL><LI>Digestion of TdT sequences</LI><OL type="”i”"><LI>Take three tubes and mix together these ingredients for each:</LI><OL type="”a”"><LI>3.1 buffer---1.5uL</LI><LI>Water---2.5uL</LI><LI>XbaI---0.5uL</LI><LI>BamhI---0.5uL</LI></OL><LI>Place each TdT variant in each tube </LI><LI>Take three more tubes and mix together these ingredients for each:</LI><OL type="”a”"><LI>2.1 buffer---1.5uL</LI><LI>Water---2.5uL</LI><LI>EcoRI---0.5uL</LI>PstI---0.5uL</OL><LI>Place each TdT variant in each tube</LI><LI>Incubate both tubes at 37˚C for 1 hour</LI><LI>Purify TdT DNA</LI><OL type="”a”"><LI>tubes containing EcoRI and PstI are purified by incubating at 80˚C for 20 minutes</LI><LI>tubes containing XbaI and PstI are purified using life technologies &quot;Procedure for Purifying PCR Products&quot; (in orange folder)</LI></OL></OL></OL><A href="#top">Top of Page</A><H3>7 - Immobilizing Disulfide Modified Oligos to Mercaptosilane Treated Glass Slides</H3><DT><H5>Materials</H5></DT><LI>Untreated Glass Slides or Glass Coverslips</LI><LI>Disulfide Modified Oligonucleotides (Ordered from IDT with the C6 S-S 5’ Thiol Modifier)</LI><LI>500 mM Sodium Bicarbonate Buffer (pH 9.0)</LI><LI>Anhydrous Ethanol</LI><LI>25% Ammonium Hydroxide</LI><LI>1% 3-mercaptopropyl trimethoxysilane, 95% Ethanol, 16 mM acetic acid (pH 4.5)</LI><LI>95% Ethanol, 16 mM acetic acid (pH 4.5)</LI><LI>10 mM Tris-HCl (pH 7.5), 150 mM NaCl, 0.05% Tween 20</LI><LI>Running RO water or ddH20</LI><LI>Vacuum Oven</LI><LI>Vacuum Pump</LI><LI>Fume Hood</LI><LI>Petri dishes</LI><LI>Parafilm</LI><OL type="”1”"><LI>Place untreated glass slides in a clean Petri dish</LI><LI>Submerge glass slides in 25% Ammonium Hydroxide overnight</LI><LI>Rinse the same glass slides under running RO water for 10 minutes</LI><LI>Rinse the slides with Anhydrous Ethanol </LI><LI>Immerse the slides in 1% 3-mercaptopropyl trimethoxysilane, 95% Ethanol, 16 mM acetic acid (pH 4.5) for 30 - 45 minutes</LI><LI>Rinse the same slides with 95% Ethanol, 16 mM acetic acid (pH 4.5)</LI><LI>Cure the glass slides in a vacuum oven at 150C for 2 hours</LI><LI>While slides are curing, suspend 1 - 40 uM of disulfide modified DNA in 150 uL of of 500 mM Sodium Bicarbonate Buffer (pH 9.0)</LI><LI>Remove cured slides from vacuum oven, and array disulfide modified DNA suspended in 500 mM Sodium Bicarbonate Buffer (pH 9.0) onto slides. Various arrays, ranging from 5 x 5 lattices using 2 uL dots to 1 large 50 uL dot were used </LI><LI>Place the slides in a moist environment, such as on a rack in a 30C water bath, overnight</LI><LI>Clean the slides using three washes of 10 mM Tris-HCl (pH 7.5), 150 mM NaCl, 0.05% Tween 20</LI><LI>Store the slides in 4C temperature</LI></OL><A href="#top">Top of Page</A><H3>8 - RNA Ligase Reaction</H3><H5>Materials</H5><LI>1 ug Single Stranded DNA with 5’ Phosphate (X mer) [Retrieved as an IDT Custom Oligo]</LI><LI>1 ug Single Stranded DNA  (Y mer) [Retrieved as an IDT Custom Oligo or can be any primer]</LI><LI>T4 RNA Ligase 1</LI><LI>T4 RNA Ligase 1 Reaction Buffer</LI><LI>PEG 8000</LI><LI>ATP</LI><LI>10 mM Tris-HCl pH 8.0, 2.5 mM EDTA</LI><H5>Procedure</H5><OL type="”1”"><LI>Make a 20 uL reaction that contains the following:</LI><UL><LI>1 ug of ssDNA with 5’ Phosphate</LI><LI>1 ug of ssDNA (Any primer or oligo)</LI><LI>25% PEG 8000</LI><LI>1 mM ATP</LI><LI>1X T4 RNA Ligase 1 Reaction Buffer</LI><LI> uL of T4 RNA Ligase 1</LI></UL><LI>Incubate the mixture overnight for 16-18 hours at room temperature</LI><LI>Terminate the reaction by adding 40 uL of 10 mM Tris-HCl pH 8.0, 2.5 mM EDTA</LI></OL><A href="#top">Top of Page</A><H3>9 - Gel Extraction</H3><H5>Materials</H5><LI>DNA sample ran through agarose gel</LI><LI>Sharp edge, like scalpel or gel cutter</LI><LI>Qiagen Buffers QG, PE, EB</LI><LI>Isopropanol</LI><H5>Procedure</H5><OL type="”1”"><LI>Prewarm waterbath to 50°C</LI><LI>Excise the DNA fragment from the agarose gel with a clean, sharp scalpel.</LI><LI>Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 µl)</LI><LI>Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation.</LI><LI>After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose).</LI><LI>Add 1 gel volume of isopropanol to the sample and mix.</LI><LI>Place a QIAquick spin column in a provided 2 ml collection tube.</LI><LI>To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.</LI><LI>Discard flow-through and place QIAquick column back in the same collection tube.
(Optional): Add 0.5 ml of Buffer QG to QIAquick column and centrifuge for 1 min.</LI><LI>To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at ≥10,000 x g (~13,000 rpm).</LI><LI>Place QIAquick column into a clean 1.5 ml microcentrifuge tube.</LI><LI>To elute DNA, add 50 µl of Buffer EB (10 mM Tris·Cl, pH 8.5) or H2O to the center of the QIAquick membrane and centrifuge the column for 1 min at maximum speed.</LI></OL><A href="#top">Top of Page</A><H3>10 - Miniprep</H3><H5>Materials</H5><LI>5mL of overnight bacteria culture</LI><LI>Qiagen Buffers P1, P2, N3, PE, EB</LI><H5>Procedure</H5><OL type="”1”"><LI>Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a microcentrifuge tube.</LI><LI>Add 250 µl Buffer P2 and gently invert the tube 4–6 times to mix</LI><LI>Add 350 µl Buffer N3 and invert the tube immediately but gently 4–6 times.</LI><LI>Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.</LI><LI>Apply the supernatants from step 4 to the QIAprep spin column by decanting or pipetting.</LI><LI>Centrifuge for 30–60 s. Discard the flow-through.
(Optional): Wash the QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30–60 s. Discard the flow-through.</LI><LI>Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 30–60 s.</LI><LI>Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer.</LI><LI>Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 µl </LI><LI>Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.</LI></OL><A href="#top">Top of Page</A><H3>11 - PCR Purification</H3><H5>Materials</H5><LI>PCR Reaction to be purified</LI><LI>Qiagen Buffers PB, PE, EB</LI><H5>Procedure</H5><OL type="”1”"><LI>Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.</LI><LI>If pH indicator has been added to Buffer PB, check that the color of the mixture is yellow.</LI><LI>Place a QIAquick spin column in a provided 2 ml collection tube.</LI><LI>To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.</LI><LI>Discard flow-through. Place the QIAquick column back into the same tube.</LI><LI>To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.</LI><LI>Discard flow-through and place the QIAquick column back in the same tube.</LI><LI>Centrifuge the column for an additional 1 min.</LI><LI>Place QIAquick column in a clean 1.5 ml microcentrifuge tube.</LI><LI>To elute DNA, add 50 µl Buffer EB (10 mM Tris·Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min.</LI></OL><A href="#top">Top of Page</A><H3>12 - TdT Nucleotide Addition</H3><H5>Materials</H5><LI>DNA source - commercially made oligo's</LI><LI><A href="”https://www.neb.com/products/m0315-terminal-transferase”"> Terminal Transferase</A></LI><LI><A href="”http://www.trilinkbiotech.com/cart/scripts/prodView.asp?idproduct=4665”" target="_blank">CleanAmp dNTP's</A></LI><H5>Procedure</H5><LI>Repeat for each nucleotide</LI><OL type="”1”"><LI>Pre-heat water baths to 37°C and 95°C</LI><LI>Mix:</LI></OL><OL type="”i”"><LI>5.0 μL 10X TdT Buffer</LI><LI>5.0 μL 2.5 mM CoCl2 solution</LI><LI>1μg 43 mer oligo (1.3μL of 813 ng/μL)</LI><LI>0.4 μL 50mM CleanAmp dATP</LI><LI>20 Units TdT (1 μL of 20,000 U/μL)</LI><LI>dH2O to final volume of 50 μL</LI><LI>Incubate at 37°C for 30 minutes.</LI><LI>Prepare 2 equal volume aliquots of the mixture (25 μL each).</LI><LI>Incubate 1 aliquot at 95°C for 15 minutes.</LI><LI>Add additional 10 U TdT (0.5 μL of 20,000 U/μL) to both aliqouts.</LI><LI>Cool 95°C water bath to 70°C.</LI><LI>Incubate at 37°C for 60 minutes.</LI><LI>Heat inactivate at 70°C for 10 minutes.</LI><LI>Store at -20°C.</LI></OL><LI>Controls</LI><OL type="”1”"><LI>Negative (No TdT)</LI><OL type="”i”"><LI>Pre-heat water baths to 37°C and 70°C</LI><LI>Mix:</LI></OL><OL type="”a”"><LI>5.0 μL 10X TdT Buffer</LI><LI>5.0 μL 2.5 mM CoCl2 solution</LI><LI>500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)</LI><LI>1.0 μL 10mM dATP</LI><LI>dH2O to final volume of 50 μL</LI><LI>Incubate at 37°C for 60 minutes</LI><LI>Heat inactivate at 70°C for 10 minutes.</LI><LI>Store at -20°C.</LI></OL><LI>Five Minute Incubation Time</LI><OL type="”i”"><LI>Pre-heat water baths to 37°C and 70°C</LI><LI>Mix:</LI></OL><OL type="”a”"><LI>5.0 μL 10X TdT Buffer</LI><LI>5.0 μL 2.5 mM CoCl2 solution</LI><LI>500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)</LI><LI>1.0 μL 10mM dATP</LI><LI>10 Units TdT (0.5 μL of 20,000 U/μL)</LI><LI>dH2O to final volume of 50 μL</LI><LI>Incubate at 37°C for 5 minutes</LI><LI>Heat inactivate at 70°C for 10 minutes.</LI><LI>Store at -20°C.</LI></OL>Sixty Minute Incubation Time<OL type="”i”"><LI>Pre-heat water baths to 37°C and 70°C</LI><LI>Mix:</LI></OL><OL type="”a”"><LI>5.0 μL 10X TdT Buffer</LI><LI>5.0 μL 2.5 mM CoCl2 solution</LI><LI>500 ng 24-mer oligo (0.9 μL of 568.8 ng/μL)</LI><LI>1.0 μL 10mM dATP</LI><LI>10 Units TdT (0.5 μL of 20,000 U/μL)</LI><LI>dH2O to final volume of 50 μL</LI><LI>Incubate at 37°C for 60 minutes</LI><LI>Heat inactivate at 70°C for 10 minutes.</LI><LI>Store at -20°C.</LI></OL></OL><A href="#top">Top of Page</A><H3>13- Non-Radioactive phosphorlyation with T4 PNK</H3><H5>Procedure</H5><OL type="”1”"><LI>Mix the following:</LI></OL><UL><LI>up to 300 pmol of single stranded oligo (43mer)</LI><LI>5 microliter of T410X PNK Buffer</LI><LI>5 microliter of 10 mM ATP</LI><LI>1 microliter of PNK</LI><LI>ddH20 upto a total volume of 50 uL</LI></UL><A href="#top">Top of Page</A><B><I><U>References</U></I></B><P>
3 : Bacterial Transformation - <I>https://www.neb.com/protocols/1/01/01/high-efficiency-transformation-protocol-c2987</I>
4 : Colony PCR of pSB1C3 with TdT Mutant Inserts - <I>https://www.neb.com/protocols/1/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273</I>
7 : Immobilizing Disulfide Modified Oligos to Mercaptosilane Treated Glass Slides - <I>http://xobi.net/Lib/AnalBio99v266p23.pdf</I>
8 : RNA Ligase Reaction - <I>https://www.neb.com/protocols/1/01/01/ligation-of-an-oligo-to-single-stranded-dna-cdna-using-t4-rna-ligase-1</I>
9 : Gel Extraction - <I>http://www.aun.edu.eg/molecular_biology/gene&amp;protein/Gel%20extraction-Qiagen.pdf</I>
10 : Miniprep - <I>http://stanxterm.aecom.yu.edu/wiki/data/Product_manuals_attach/qiagenmini.pdf</I>
11 : PCR Purification - <I>https://static.igem.org/mediawiki/2012/a/a3/QIAquick_PCR-purification.pdf</I>
12 : TdT Nucleotide Addition - <I>http://www.neb.com/protocols/1/01/01/non-radioactive-phosphorylation-with-t4-pnk-or-pnk3-phosphatase-minus</I></P></DL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>