<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SDU-Denmark_labwork_Constructs"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SDU-Denmark/labwork/Constructs">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SDU-Denmark/labwork/Constructs&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SDU-Denmark/labwork/Constructs&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SDU-Denmark/labwork/Constructs&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SDU-Denmark/labwork/Constructs" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SDU-Denmark/labwork/Constructs</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>iGEM TEAM ::: SDU-DENMARK</TITLE></P><DIV class="container"><DIV id="sidenav"><IFRAME src="https://2012.igem.org/Team:SDU-Denmark/menu" frameborder="0" height="1000" width="200" scrolling="no" allowtransparency="true"><P>Your browser does not support iframes.</P></IFRAME></DIV><DIV id="footer">
            The iGEM Team of University of Southern Denmark 2012
<FONT color="FF6600"><B>igem.sdu.2012@gmail.com</B></FONT></DIV><DIV id="contenta"><H1>Data page / Constructs</H1><H2>Data For Our Favorite New Parts</H2><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899011" target="_new"><B>Part:BBa_K899011</B></A><P>
Our submitted FFT gene in pSB1C3 vector, was characterized in pJET blunt vector. We produced growth curve and inulin staining. From our results we can conclude, that FFT might be able to produce inulin, and is not able to produce any inulin without the presence of 1-kestose sugars produced by the SST enzyme. 
</P><H2>Data For Our Favorite non-characterized composite parts</H2><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899005" target="_new"><B>Part:BBa_K899005</B></A><P>
This is our theoretical composite bio-brick system of SST producing the 1-kestose trisaccharide as mentioned. This  brick also includes the toxin-antitoxin system which is comprised of ccdB as a toxin and MazE as antitoxin.
</P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899006" target="_new"><B>Part:BBa_K899006</B></A><P>
The complementary part to K899005 producing FFT and mazF as a toxin and ccdA as antitoxin
</P><H2>Data For Registry entries</H2><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899000" target="_new"><B>Part:BBa_K899000</B></A><P>
This part is the coding sequence for Sucrose:sucrose 1-fructosyltransferase isolated from jerusalem artichoke.
</P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899001" target="_new"><B>Part:BBa_K899001</B></A><P>
This part is the coding sequence for Fructan:Fructan 1-Fructosyltransferase isolated from jerusalem artichoke
</P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899002" target="_new"><B>Part:BBa_K899002</B></A><P>
Composite part: BBa_K899000(SST) + Reporter system containing a LacL promotor, GFP and two terminators.
</P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899003" target="_new"><B>Part:BBa_K899003</B></A><P>
Composite part: BBa_K899001(FFT) + Reporter system containing a LacL promotor, GFP and two terminators.
</P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899004" target="_new"><B>Part:BBa_K899004</B></A><P>
Composite part of Reporter system to test for transcription of both SST and FFT. Contains LacL promotor, RBS, SST, RBS, FFT RBS, GFP and two terminators
</P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899007" target="_new"><B>Part:BBa_K899007</B></A><P>
ccdB is a toxin that fits to the ccdA antitoxin
</P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899008" target="_new"><B>Part:BBa_K899008</B></A><P>
antitoxin ccdA working by inhibitting the toxin ccdB (BBa_K899007)
</P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899009" target="_new"><B>Part:BBa_K899009</B></A><P>
Double ribosomal binding site, containing kozak and shine-dalgarno sequence
</P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899010" target="_new"><B>Part:BBa_K899010</B></A><P>
Shine-dalgarno sequence (AGGAGGT) and a kozak sequence (CACC).
</P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K899012" target="_new"><B>Part:BBa_K899012</B></A><P>
SST (BBa_K899001) with double RBS (BBa_K899010)
</P><H2>The System</H2><P>
The above picture illustrates our idealized system the way we wanted it to be . It consists of two plasmids with FFT and SST respectively. Both plasmids have a toxin-antitoxin system introduced which works as a form of safety to prevent horizontal gene transfer (read more about it further down the page). The two enzymes, FFT and SST, produce together the non-digestible inulin from sucrose. Along with the two plasmids there is also a sigma-E gene that functions as a regulator gene in our kill-switch system. 
The sigma-E is an alternative sigma factor that controls the extracytoplasmic stress response in E. coli. The idea is to have an L-Rhamnose sensitive promoter that expresses this essential gene. 
The L-Rhamnose would be in the yoghurt with our bacteria and function as a limiting lifetime for the consumed bacteria. L-Rhamnose is a non-digestible sugar which passes through the system and thereby inactivates the sigma-E gene. The consumed bacteria from the yoghurt would inevitably die after a while.  
</P><H2>Killswitch</H2><P>
During the construction of a killswitch, we stumbled upon a very concerning fact. A normal killswitch inducible by a promoter can be rendered useless in the case of frameshift mutations (and any other mutation, that affects the gene-product). Therefore we chose to construct killswitches with a less likely chance of being frameshifted out of order. 

In order to make the most effective killswitch we have take several aspects in consideration. As you might observe on the figure above, it is not just a promoter and a death gene, since this would be rendered useless by frameshift mutations or loss of promoter effectivity etc.. The design of our killswitch aims to produce a reliable killswitch that decreases the risk of misfunction. This is very important because the introduction of GMO in humans requires utter carefulness and security, should something go wrong.

Our killswitch consists of a constitutive promoter, 3 structural genes in a polycistronic system whereas 2 of them are translationally coupled with 1 bp frameshift. The frameshift-coupled genes are mazF and yafO which is both toxic to the bacteria. MazE is located after another shine-dalgarno sequence and is therefore not affected by frameshifts in the mazF-yafO region.
The point of the frameshifted translational coupling between mazF and yafO is, that the yafO not will be translated if the frameshift does not occur. Should a frameshift occur in the mazF toxin the yafO will be expressed, killing the cell. Does a mutation in mazE occur the cell will die because mazE is the antitoxin of mazF. Should the genes be untranscribed at any point, the mazE should be degraded faster than the mazF killing the cell.
The sequences have been codon optimized, in order to remove any stop codons potentially terminating translation before the ribosome reaches the yafO toxin.

After developing this idea we found, that ribosomes can jump up to 10 bp back and forth from a stop-codon, therefore this frameshift system will not work in practice. Though it is subject for further development, since moving the toxin’s an appropriate distance from the stop codon would solve that problem. We have not been able to find attempts on this in the literature so it would be an interesting subject to investigate further. Since we don’t have the time to investigate this idea, we instead propose another idea for a safety mechanism that will help controlling the bacteria of our culture.<B>Safety mechanism targeting sigma-E:</B>

This construct will contain an L-rhamnose promoter that is activated by L-rhamnose. This promoter will then transcribe the sigma-E factor which should be deleted from the genome of our chassi bacteria. This factor is a life-essential part of the bacteria, hence killing it, if it should remain untranslated. Also it would be necessary to delete the toxin-antitoxin of hicAB since it is shown that hicA is capable of keeping the cell alive upon lac of sigma-E [<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2631989/" target="_new">jørgensen et al</A>]. The reason for putting the sigma-E under control of the rhamnose promoter is, that by adding rhamnose to our product (fx yoghurt), the bacteria will only be alive as long as the rhamnose is present, and when the rhamnose i removed from the gut the bacteria will die.<B>Optimizing the sequence</B>
The RNA code optimized<SPAN style="background-color:red">Red markings indicates silent mutations</SPAN><B>Translational overview</B><SPAN style="background-color:green">start</SPAN><SPAN style="background-color:red">stop</SPAN><SPAN style="background-color:brown">Inserted AA (gly)</SPAN>

Should a frameshift occur, the sequence would look like this (notice the F in the beginning is the insertion of 1 bp). The weakness of the killswitch is, that if the frameshift occurs in the start codon the ribosome will terminate translation before it gets to the coding sequence of yafO.

deletion of 1 bp /2 bp insertion</P><H2>Construct flowchart</H2><P>
This flowchart shows our proposed workflow for constructing our desired final construct. See partsregistry for further details regarding the individual parts. (see further up on this page for the relevant links.)   
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:SDU-Denmark/labwork/Constructs">http://2012.igem.org/Team:SDU-Denmark/labwork/Constructs</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SDU-Denmark/labwork/Constructs" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SDU-Denmark/labwork/Constructs" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SDU-Denmark/labwork/Constructs&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SDU-Denmark/labwork/Constructs&amp;oldid=237352" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>