<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCUT-China_A_Project_Lysis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCUT-China A/Project Lysis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>菜单列表</TITLE></P><DIV class="left_nav" id="left_nav"><DL class="list_dl"><DIV class="list_title"><H1>SCUT-China_
                    <SPAN>A</SPAN></H1></DIV><DT class="list_dt"><P><A href="https://2017.igem.org/Team:SCUT-China_A">Home</A></P></DT><DT class="list_dt"><P>Achievement</P></DT><DD class="list_dd"><UL><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Achievement_Result">Result</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Achievement_Parts">Parts</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Achievement_Judging">Judging</A></P></LI></UL></DD><DT class="list_dt" id="project"><P>Project</P></DT><DD class="list_dd"><UL><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Project_Overview">Overview</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Project_Design">Design</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Project_Lysis" style="font-weight: bold;">Lysis</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Project_Chromogenic-Reaction">Chromogenic Reaction</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Project_Colourimetric-App">Colourimetric App</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Project_Demonstrate">Demonstrate</A></P></LI></UL></DD><DT class="list_dt"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Modelling">Modelling</A></P></DT><DT class="list_dt"><P>Practice</P></DT><DD class="list_dd"><UL><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Practice_Overview">Overview</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Practice_Silver">Sliver</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Practice_Gold">Gold</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Practice_Engagement">Engagement</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Practice_Collabaration">Collabaration</A></P></LI></UL></DD><DT class="list_dt"><P><A href="https://2017.igem.org/Team:SCUT-China_A/Safety">Safety</A></P></DT><DT class="list_dt"><P><A href="https://2017.igem.org/Team:SCUT-China_A/InterLab">Interlab</A></P></DT><DT class="list_dt"><P>About Us</P></DT><DD class="list_dd"><UL><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/About-us_Team">Team</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/About-us_Attribution">Attribution</A></P></LI><LI class="list_li"><P><A href="https://2017.igem.org/Team:SCUT-China_A/About-us_Notebook">Notebook</A></P></LI></UL></DD></DL></DIV><DIV class="main_body"><DIV class="main_body2"><DIV class="nine columns"><H1 class="article_title">LYSIS</H1><DIV class="texttitle">Background</DIV><DIV class="textsubtitle">Metal ions detecting module</DIV><P>        We chose five metal ions to construct our metal detecting module, and three of them
                    have outstanding response effect. It is remarkable that they all belong to the MerR family. So here we
                    are going to explain how this family work.
                            The MerR family is a group of transcriptional regulators with similar N-terminal
                    helix-turn-helix DNA binding regions and C-terminal effector binding regions that are specific to the
                    effector recognized. The few MerR-like regulators that have been studied experimentally have been shown
                    to activate suboptimal σ<SBP>70-dependent promoters, in which the spacing between the -35 and -10 elements
                    recognized by the σ factor is greater than the optimal 17±1 bp.
                             So, how does it respond to heavy metal ions? Take MerR-pMerR as an example,
                    it has several processes.
                             1. In the absence of Hg<SUP>2+</SUP> and MerR, RNA polymerase preferentially transcribes
                    from the merR promoter, increasing the amount of MerR present in the cell.
                            2. Once MerR binds to merOP, transcription of the MerR promoter is repressed
                    and the DNA becomes bent and unwound at the operator sequence. RNA polymerase is recruited to the mer
                    promoter, forming a ternary complex of DNA, MerR and RNA polymerase.
                            3. Binding of Hg<SUP>2+</SUP> to one of two binding sites on the MerR make DNA distortion
                    at the centre of the operator which cause the reorientation of the -35 and -10 sequences make them interact
                    with the RNA polymerase σ70 subunit to form an open transcriptional complex and transcription is initiated.
                    <SUP>[1]</SUP>         Our team chose several regulators (as the form below) in this family as our
                    detecting module. And we also chose other regulators belonging to other family such as ChrB
                    <SUP>[4]</SUP>, RcnR
                    <SUP>[5]</SUP>.
                     </SBP></P><DIV class="textsubtitle">Lysis module </DIV><P>        We chose
                    <SPAN> SRRz lysis gene</SPAN> to create our lysis module. The SRRz gene is from phage. It consists of three
                    parts S, R, Rz.
                             The product of
                    <SPAN>S gene</SPAN> would cause lesions on the cytoplasmic membrane through which the product coded by the
                    <SPAN>R gene</SPAN> escapes to the periplasm and causes murein-degrading, while the
                    <SPAN>Rz gene</SPAN>’s product may be an endopeptidase that can cleave the oligopeptide crosslinks in the peptidoglycan
                    and/or between peptidoglycan and the outer membrane.
                    <SUP>[6]</SUP></P><DIV class="texttitle">Method</DIV><DIV class="textsubtitle">Lysis experiment</DIV><P>1. Grow the cells on 5 mL LB medium+Chloramphenicol overnight at 37℃ and 220 rpm.
                     2. Take 4 mL cultures into 200 mL LB medium+Chloramphenicol in a conical flask.
                     3. Incubate the cultures at 37℃ and 220 rpm for about 1.5-2 hours.
                     4. Set the instrument to read OD600, measure OD600 of the cultures.
                     5. When the value of OD600 is about 0.6-0.8, divide the 200mL liquid into 12 cuvettes. Each cuvette
                    contains 10 mL cultures.
                     6. Then add metal ion solution with these following concentration, each concentration has 2 replicates.
                     </P> 7. Incubate the cultures at 37℃ and 220 rpm.
                     8. Measure its OD600 at 0.5h ,1h, 1.5h, 2h, 3h, 4h, 6h.
        <DIV class="texttitle">Result</DIV><DIV class="textsubtitle">Lysis gene expression</DIV><P>         We have developed the lysis module (<A href="http://parts.igem.org/Part:BBa_K2360000">BBa_K2360000</A>) and five metal detecting module for E. coli BL21.
                    Combining the lysis module and detecting module, we have constructed five devices which can respond to
                    different metal ions.
                             Because the lysis gene will be expressed when the detecting module makes a
                    response to the metal ions, which will make bacterial cleavage, resulting in the decrease of OD600, we
                    measured the growth curve of our engineering bacteria in different concentration of metal ions to prove
                    its function.
                  </P>         As shown in the growth curves of E. coli with cadmium detecting device (figure
                    1), the growth rate of E. coli BL21 is decline obviously after adding the concentration of Cd<SUP>2+</SUP> from
                    10<SUP>-8</SUP>M to 10<SUP>-5</SUP>M, indicating that this device can detect the cadmium ions.
                           As shown in the growth curves of E. coli with cadmium detecting device (figure
                    2), the growth rate of E. coli BL21 is decline obviously after adding the concentration of Hg<SUP>2+</SUP> from
                    10<SUP>-7</SUP>M to 10<SUP>-5</SUP>M, indicating that this device can detect the mercury ions.
                           As shown in the growth curves of E. coli with lead detecting device (figure
                    3), the growth rate of E. coli BL21 is decline obviously after adding the concentration of Pb<SUP>2+</SUP> from
                    10<SUP>-7</SUP>M to 10<SUP>-5</SUP>M, indicating that this device can detect the lead ions.
                             It was a pity that we found the OD600 of E. coli with did not decline after
                    we added Cr<SUP>6+</SUP> and Ni<SUP>2+</SUP>, which showed these two devices did not work as we expected. (figure 4 and figure
                    5)
                             The result not only showed us its effect to respond the metal ions of each
                    device, but also showed us something inspiring. Compared with traditional reporters such as GFP, which
                    needs about 3.5 hours for mature <SUP>[7]</SUP>, our engineered bacteria can work and produce a clear phenomenon
                    in about 30 minutes (lead detecting device), which showed its high efficiency.
                             According to the result, we chose the three of them, mercury detecting device(<A href="http://parts.igem.org/Part:BBa_K2360008">BBa_K2360008</A>),
                    cadmium detecting device(<A href="http://parts.igem.org/Part:BBa_K2360010">BBa_K2360010</A>) and lead detecting device(<A href="http://parts.igem.org/Part:BBa_K2360011">BBa_K2360011</A>), which have obvious response
                    to enter the next stage of the project.
                <DIV class="textsubtitle">Specificity</DIV><P>        Although we have chosen specifically transcriptional regulators, we want to
                    verify that they can perform the same specifically in our system. So we verified the detecting devices
                    specificity of mercury, cadmium and lead by adding Cd<SUP>2+</SUP>, Hg<SUP>2+</SUP>, Pb<SUP>2+</SUP>, Cr<SUP>6+</SUP>, Ni<SUP>2+</SUP> to their culture mediums.
                </P><P>         It was obvious that the E. coli with lead detecting device lysed after we added
                    Pb<SUP>2+</SUP>, while other E. coli grew normally after adding Cd<SUP>2+</SUP>, Hg<SUP>2+</SUP>, Cr<SUP>6+</SUP> and Ni<SUP>2+</SUP>. The result indicated
                    that the lead detecting device has good specificity for specific metal ions. (Figure 6.)
                </P><P>         This result showed us that E. coli with mercury detecting device can merely
                    lyse after we added Hg<SUP>2+</SUP>, and did not cause the same effect. Consequently, this device’s specificity
                    is good. (figure 7.)
                </P><P>         The result of cadmium detecting device’s specificity experiment was that this
                    device’s ion response was not obvious in the lysis, which may be caused by misoperation.
                </P><DIV class="texttitle">Conclusion</DIV><P>        Combining the lysis module and detecting module, we have constructed five devices corresponding to different
                    metal ions, which have been proved by the lysis experiment that three of them, mercury detecting device(<A href="http://parts.igem.org/Part:BBa_K2360008">BBa_K2360008</A>),
                    cadmium detecting device(<A href="http://parts.igem.org/Part:BBa_K2360010">BBa_K2360010</A>) and lead detecting device(<A href="http://parts.igem.org/Part:BBa_K2360011">BBa_K2360011</A>), had the ability to make
                    a response to its corresponding metal ion. What’s more the mercury detecting and the lead detecting devices
                    performed an obvious specificity. And we found out the response speed of SRRz gene is faster than GFP.
                    So we have created three efficient devices for metal ion detecting.
                 </P><DIV class="texttitle">References</DIV><P>
                        [1]. Nigel L. Brown, Jivko V. Stoyanov, Stephen P. Kidd, Jon L. Hobman. The MerR family of transcriptional regulators [J]. FEMS Microbiology Reviews 27 (2003) 145^163.
                        [2]. Lara Bereza-Malcolm, Sanja Aracic and Ashley E. Franks. Development and Application of a Synthetically-Derived Lead Biosensor Construct for Use in Gram-Negative Bacteria [J]. Sensors (Basel). 2016 Dec 18;16(12).
                        [3]. Brocklehurst K R, Megit S J, Morby A P. Characterisation of CadR from Pseudomonas aeruginosa: a Cd(II)-responsive MerR homologue[J]. Biochemical &amp; Biophysical Research Communications, 2003, 308(2):234-9.
                        [4]. Branco R, Morais P V. Identification and characterization of the transcriptional regulator ChrB in the chromate resistance determinant of Ochrobactrum tritici 5bvl1 [J]. Plos One, 2013, 8(11):e77987-e77987. 
                        [5]. Iwig J, Rowe J, Chivers P. Nickel homeostasis in Escherichia coli - the rcnR-rcnA efflux pathway and its linkage to NikR function[J]. Molecular Microbiology, 2006, 62(1):252.
                        [6]. Kloos D U, Strätz M, Güttler A, et al. Inducible cell lysis system for the study of natural transformation and environmental fate of DNA released by cell death.[J]. 1994, 176(23):7352-7361.
                        [7]. Leveau J H, Lindow S E. Predictive and interpretive simulation of green fluorescent protein expression in reporter bacteria[J]. Journal of Bacteriology, 2001, 183(23):6752.
                        
                </P></DIV></DIV></DIV><DIV class="right_nav" id="right_nav"><DIV id="sidebar" style="color:#000000"><H4><A href="javascript:void(0);" onclick="naver('Background')">Background</A></H4><H4><A href="javascript:void(0);" onclick="naver('Method')">Method</A></H4><H4><A href="javascript:void(0);" onclick="naver('Result')">Result</A></H4><H4><A href="javascript:void(0);" onclick="naver('Conclusion')">Conclusion</A></H4><H4><A href="javascript:void(0);" onclick="naver('References')">References</A></H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>