Team:SCUT-China A/Project Lysis
菜单列表
SCUT-China_ A
Home
Achievement
Result
Parts
Judging
Project
Overview
Design
Lysis
Chromogenic Reaction
Colourimetric App
Demonstrate
Modelling
Practice
Overview
Sliver
Gold
Engagement
Collabaration
Safety
Interlab
About Us
Team
Attribution
Notebook
LYSIS
Background
Metal ions detecting module
        We chose five metal ions to construct our metal detecting module, and three of them                     have outstanding response effect. It is remarkable that they all belong to the MerR family. So here we                     are going to explain how this family work.
        The MerR family is a group of transcriptional regulators with similar N-terminal                     helix-turn-helix DNA binding regions and C-terminal effector binding regions that are specific to the                     effector recognized. The few MerR-like regulators that have been studied experimentally have been shown                     to activate suboptimal σ
70-dependent promoters, in which the spacing between the -35 and -10 elements                     recognized by the σ factor is greater than the optimal 17±1 bp.
         So, how does it respond to heavy metal ions? Take MerR-pMerR as an example,                     it has several processes.
         1. In the absence of Hg2+ and MerR, RNA polymerase preferentially transcribes                     from the merR promoter, increasing the amount of MerR present in the cell.
        2. Once MerR binds to merOP, transcription of the MerR promoter is repressed                     and the DNA becomes bent and unwound at the operator sequence. RNA polymerase is recruited to the mer                     promoter, forming a ternary complex of DNA, MerR and RNA polymerase.
        3. Binding of Hg2+ to one of two binding sites on the MerR make DNA distortion                     at the centre of the operator which cause the reorientation of the -35 and -10 sequences make them interact                     with the RNA polymerase σ70 subunit to form an open transcriptional complex and transcription is initiated. [1]
         Our team chose several regulators (as the form below) in this family as our                     detecting module. And we also chose other regulators belonging to other family such as ChrB [4], RcnR [5].
Lysis module
        We chose SRRz lysis gene to create our lysis module. The SRRz gene is from phage. It consists of three                     parts S, R, Rz.
         The product of S gene would cause lesions on the cytoplasmic membrane through which the product coded by the R gene escapes to the periplasm and causes murein-degrading, while the Rz gene’s product may be an endopeptidase that can cleave the oligopeptide crosslinks in the peptidoglycan                     and/or between peptidoglycan and the outer membrane. [6]
Method
Lysis experiment
1. Grow the cells on 5 mL LB medium+Chloramphenicol overnight at 37℃ and 220 rpm.
2. Take 4 mL cultures into 200 mL LB medium+Chloramphenicol in a conical flask.
3. Incubate the cultures at 37℃ and 220 rpm for about 1.5-2 hours.
4. Set the instrument to read OD600, measure OD600 of the cultures.
5. When the value of OD600 is about 0.6-0.8, divide the 200mL liquid into 12 cuvettes. Each cuvette                     contains 10 mL cultures.
6. Then add metal ion solution with these following concentration, each concentration has 2 replicates.
7. Incubate the cultures at 37℃ and 220 rpm.
8. Measure its OD600 at 0.5h ,1h, 1.5h, 2h, 3h, 4h, 6h.
Result
Lysis gene expression
         We have developed the lysis module ( BBa_K2360000 ) and five metal detecting module for E. coli BL21.                     Combining the lysis module and detecting module, we have constructed five devices which can respond to                     different metal ions.
         Because the lysis gene will be expressed when the detecting module makes a                     response to the metal ions, which will make bacterial cleavage, resulting in the decrease of OD600, we                     measured the growth curve of our engineering bacteria in different concentration of metal ions to prove                     its function.
         As shown in the growth curves of E. coli with cadmium detecting device (figure                     1), the growth rate of E. coli BL21 is decline obviously after adding the concentration of Cd2+ from                     10-8M to 10-5M, indicating that this device can detect the cadmium ions.
         As shown in the growth curves of E. coli with cadmium detecting device (figure                     2), the growth rate of E. coli BL21 is decline obviously after adding the concentration of Hg2+ from                     10-7M to 10-5M, indicating that this device can detect the mercury ions.
         As shown in the growth curves of E. coli with lead detecting device (figure                     3), the growth rate of E. coli BL21 is decline obviously after adding the concentration of Pb2+ from                     10-7M to 10-5M, indicating that this device can detect the lead ions.
         It was a pity that we found the OD600 of E. coli with did not decline after                     we added Cr6+ and Ni2+, which showed these two devices did not work as we expected. (figure 4 and figure                     5)
         The result not only showed us its effect to respond the metal ions of each                     device, but also showed us something inspiring. Compared with traditional reporters such as GFP, which                     needs about 3.5 hours for mature [7], our engineered bacteria can work and produce a clear phenomenon                     in about 30 minutes (lead detecting device), which showed its high efficiency.
         According to the result, we chose the three of them, mercury detecting device( BBa_K2360008 ),                     cadmium detecting device( BBa_K2360010 ) and lead detecting device( BBa_K2360011 ), which have obvious response                     to enter the next stage of the project.
Specificity
        Although we have chosen specifically transcriptional regulators, we want to                     verify that they can perform the same specifically in our system. So we verified the detecting devices                     specificity of mercury, cadmium and lead by adding Cd2+, Hg2+, Pb2+, Cr6+, Ni2+ to their culture mediums.
         It was obvious that the E. coli with lead detecting device lysed after we added                     Pb2+, while other E. coli grew normally after adding Cd2+, Hg2+, Cr6+ and Ni2+. The result indicated                     that the lead detecting device has good specificity for specific metal ions. (Figure 6.)
         This result showed us that E. coli with mercury detecting device can merely                     lyse after we added Hg2+, and did not cause the same effect. Consequently, this device’s specificity                     is good. (figure 7.)
         The result of cadmium detecting device’s specificity experiment was that this                     device’s ion response was not obvious in the lysis, which may be caused by misoperation.
Conclusion
        Combining the lysis module and detecting module, we have constructed five devices corresponding to different                     metal ions, which have been proved by the lysis experiment that three of them, mercury detecting device( BBa_K2360008 ),                     cadmium detecting device( BBa_K2360010 ) and lead detecting device( BBa_K2360011 ), had the ability to make                     a response to its corresponding metal ion. What’s more the mercury detecting and the lead detecting devices                     performed an obvious specificity. And we found out the response speed of SRRz gene is faster than GFP.                     So we have created three efficient devices for metal ion detecting.
References
[1]. Nigel L. Brown, Jivko V. Stoyanov, Stephen P. Kidd, Jon L. Hobman. The MerR family of transcriptional regulators [J]. FEMS Microbiology Reviews 27 (2003) 145^163.
[2]. Lara Bereza-Malcolm, Sanja Aracic and Ashley E. Franks. Development and Application of a Synthetically-Derived Lead Biosensor Construct for Use in Gram-Negative Bacteria [J]. Sensors (Basel). 2016 Dec 18;16(12).
[3]. Brocklehurst K R, Megit S J, Morby A P. Characterisation of CadR from Pseudomonas aeruginosa: a Cd(II)-responsive MerR homologue[J]. Biochemical & Biophysical Research Communications, 2003, 308(2):234-9.
[4]. Branco R, Morais P V. Identification and characterization of the transcriptional regulator ChrB in the chromate resistance determinant of Ochrobactrum tritici 5bvl1 [J]. Plos One, 2013, 8(11):e77987-e77987.
[5]. Iwig J, Rowe J, Chivers P. Nickel homeostasis in Escherichia coli - the rcnR-rcnA efflux pathway and its linkage to NikR function[J]. Molecular Microbiology, 2006, 62(1):252.
[6]. Kloos D U, Strätz M, Güttler A, et al. Inducible cell lysis system for the study of natural transformation and environmental fate of DNA released by cell death.[J]. 1994, 176(23):7352-7361.
[7]. Leveau J H, Lindow S E. Predictive and interpretive simulation of green fluorescent protein expression in reporter bacteria[J]. Journal of Bacteriology, 2001, 183(23):6752.
Background
Method
Result
Conclusion
References
