<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Davis_Protein_Engineering_Design"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UC_Davis/Protein_Engineering_Design">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Davis/Protein_Engineering_Design&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Davis/Protein_Engineering_Design&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Davis/Protein_Engineering_Design&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Davis/Protein_Engineering_Design" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Davis/Protein Engineering Design</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE> UC Davis iGEM 2014 </TITLE><UL class="nav"><LI class="dropdown leftbookend"><A href="/Team:UC_Davis" style="font-size:13px;">Home</A></LI><LI class="dropdown leftbookend rightbookend"><A href="https://2014.igem.org/Team:UC_Davis/Project_Overview" style="font-size:13px;">Project</A></LI><LI><A href="https://2014.igem.org/Team:UC_Davis/Project_Overview">Overview</A></LI><LI><A href="/Team:UC_Davis/Protein_Engineering">Protein Engineering</A></LI><LI><A href="/Team:UC_Davis/Potentiostat_Design">Potentiostat Design</A></LI><LI><A href="/Team:UC_Davis/Electrochemistry">Electrochemistry</A></LI><LI><A href="/Team:UC_Davis/Signal_Processing">Signal Processing</A></LI><LI><A href="/Team:UC_Davis/Criteria">Achievements</A></LI><LI><A href="/Team:UC_Davis/Parts">Parts</A></LI></UL><LI class="dropdown rightbookend"><A href="/Team:UC_Davis/Policy_Practices_Overview" style="font-size:13px;">Policy &amp; Practices</A></LI><LI><A href="/Team:UC_Davis/Policy_Practices_Overview">Overview</A></LI><LI><A href="/Team:UC_Davis/Olive_Mills">Olive Mill Visits</A></LI><LI><A href="/Team:UC_Davis/Public_Perception">Public Perception</A></LI><LI><A href="/Team:UC_Davis/Legislation">Legislation</A></LI><LI class="dropdown leftbookend rightbookend"><A href="https://2014.igem.org/Team:UC_Davis/Team_Overview" style="font-size:13px;">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:UC_Davis/Team_Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:UC_Davis/Bios">Bios</A></LI><LI><A href="/Team:UC_Davis/Contact">Contact Us</A></LI><LI><A href="https://igem.org/Team.cgi?id=1324">Official Team Profile</A></LI><LI class="dropdown leftbookend rightbookend"><A href="/Team:UC_Davis/Safety" style="font-size:13px;">Safety</A></LI><LI class="dropdown leftbookend rightbookend"><A href="https://2014.igem.org/Main_Page" style="font-size:13px;">iGEM</A></LI></P><DIV><DIV class="switchbox1"><H2>Design</H2><A href="https://2014.igem.org/Team:UC_Davis/Protein_Engineering_Design"><SPAN><H2>Design</H2></SPAN></A></DIV><DIV class="switchbox1"><H2>Build</H2><A href="https://2014.igem.org/Team:UC_Davis/Protein_Engineering_Build"><SPAN><H2>Build</H2></SPAN></A></DIV><DIV class="switchbox1"><H2>Test</H2><A href="https://2014.igem.org/Team:UC_Davis/Protein_Engineering_Test"><SPAN><H2>Test</H2></SPAN></A></DIV></DIV><DIV class="floatboxwide" style="background: rgba(168, 182, 120, .95)"><H3 style="color:#212f20;margin-bottom: 0;">Why Aldehyde Dehydrogenases?</H3></DIV><DIV class="floatboxPotentiostat"><P>The aldehyde dehydrogenase family of enzymes (EC: 1.2.1.3, 1.2.1.5) was selected for use with our electrochemical biosensor. This family of enzymes catalyze the reaction of aliphatic, straight chain aldehydes and the oxidized form of beta-nicotinamide adenine dinucleotide (NAD<SUP>+</SUP>) to produce the corresponding carboxylic acid and the reduced form of beta-nicotinamide adenine dinucleotide (NADH). 

The aldehyde dehydrogenase enzyme family is perfect for our engineering and electrochemical applications:
</P><P>
This enzyme uses NAD<SUP>+</SUP> as a coenzyme and produces NADH in a 1:1 molar ratio with the amount of aldehyde catalyzed. The concentration of NADH can be readily measured with a spectrophotometer reading absorbance at 340nm, allowing us to easily measure the rate of the reaction catalyzed by an aldehyde dehydrogenase enzyme.
</P><P>
The active site of aldehyde dehydrogenase is in the center of a long tunnel, where NAD<SUP>+</SUP> enters from one side and the aldehyde substrate enters from the other side. This tunnel (highlighted in orange) gives us a large amount of flexibility in engineering amino acid residues which will alter the catalytic efficiency of this enzyme toward certain aldehyde species.
</P><P>
We identified several commercial electrodes which oxidize NADH back to NAD+ and produce a current. This will allow us to measure the amount of NADH produced over time using an electrochemical approach.
</P><P>
Preferred: Crystal structure available on the Protein Data Bank (PDB) 
</P><P>With aldehyde dehydrogenases in mind, we used two approaches to identify enzymes with the desired specificities we would use in our biosensor: bioprospecting and engineering. </P></DIV><DIV class="floatboxwide" style="background: rgba(168, 182, 120, .95)"><H3 style="color:#212f20;margin-bottom: 0;">Approach 1: Bioprospecting</H3></DIV><DIV class="floatboxPotentiostat"><P>
We used several online databases to search for aldehyde dehydrogenases with unique specificity profiles. Keeping the context of the aldehyde dehydrogenase within our electrochemical sensor in mind, we focused exclusively on aldehyde dehydrogenases which used NAD(H) as a coenzyme (EC: 1.2.1.3) as opposed to aldehyde dehydrogenases which used NADP(H) (EC: 1.2.1.5). However, we did identify several aldehyde dehydrogenases which could use either coenzyme. BRENDA was our main resource for identifying aldehyde dehydrogenases which catalyzed the oxidation reaction on specific substrates. The specificity data provided within the literature cited for each entry on BRENDA helped guide our decisions for the set of enzymes we chose to characterize.

Selection Criteria:</P><P>
DNA sequence readily available on Genbank or UniProtKB
</P><P>
Aldehyde dehydrogenase possesses unique specificity for one or more of the aldehyde species known to occur in olive oil
</P><P>
Aldehyde dehydrogenase uses NAD<SUP>+</SUP> as a coenzyme for the dehydrogenase reaction
</P><P>
Aldehyde dehydrogenase is documented to express in Escherichia coli and is easily purified in high yield (&gt;1 mg/mL)
</P></DIV><TABLE id="t01"><TBODY><TR><TH>Source</TH><TH>UniProtKB ID</TH><TH>EC Number</TH><TH>Crystal Structure Available?</TH></TR><TR><TD>Escherichia coli (Strain K12)</TD><TD>P23883</TD><TD>1.2.1.5</TD><TD>No</TD></TR><TR><TD>Rhodococcus erythropolis</TD><TD>Q4F895</TD><TD>1.2.1.3</TD><TD>No</TD></TR><TR><TD>Rattus norvegicus (Rat)</TD><TD>P11883</TD><TD>1.2.1.5</TD><TD>1AD3</TD></TR><TR><TD>Human (ALDH3A1)</TD><TD>P30838</TD><TD>1.2.1.5</TD><TD>3SZA</TD></TR><TR><TD>Human (ALDH2)</TD><TD>P05091</TD><TD>1.2.1.3</TD><TD>1O01</TD></TR></TBODY></TABLE><DIV class="floatboxwide" style="background: rgba(168, 182, 120, .95)"><H3 style="color:#212f20;margin-bottom: 0;">Approach 2: Engineering</H3></DIV><DIV class="mainContainer"><P>
We chose the aldehyde dehydrogenase from Escherichia coli as our template for engineering additional specificity profiles outside of the ones we identified in the literature. While there was not a crystal structure available for E. coli aldehyde dehydrogenase, we decided it would be the best template for engineering since it expressed the best (&gt;3 mg protein per 25mL culture) out of all the aldehyde dehydrogenase we investigated. Thus, we generated a homology model of E. coli aldehyde dehydrogenase and used the interactive protein folding design software Foldit to design mutants. Investigation of the homology model and crystal structures of other aldehyde dehydrogenases revealed that substrates must travel through a tunnel composed of hydrophobic residues to reach the catalytic center of the enzyme. Taking this into consideration, our design strategy was to alter the residues in the substrate tunnel in an attempt to change the enzyme's catalytic efficiency towards certain substrates.

We used four strategies for designing our mutants:</P><P>
Introducing hydrophilic residues in the substrate tunnel to discourage catalysis of more hydrophobic aldehydes
</P><P>
Increasing solvent exposure to the catalytic center by introducing smaller residues in the tunnel
</P><P>
Increasing packing around the catalytic center with larger hydrophobic residues
</P><P>
Introducing positive/negative charges in the substrate tunnel
</P><P>We designed a total of 20 mutants:<TABLE id="t02"><TBODY><TR><TH>Mutations</TH><TH>Description</TH></TR><TR><TD>D119I;F169A;Y296A</TD><TD>Increase solvent exposure to active site, open enterance to tunnel</TD></TR><TR><TD>D119L;Y296F;N455A</TD><TD>Increase hydrophobicity at far end of tunnel</TD></TR><TR><TD>D456L;D458L</TD><TD>Change far pocket polar residues to posess more nonpolar characteristics</TD></TR><TR><TD>F169A</TD><TD>Open up active site adjacent to catalytic residues w/ hydrophobic</TD></TR><TR><TD>F169D</TD><TD>Introduce negative charge adjactent to active site</TD></TR><TR><TD>F169H;F463H</TD><TD>Introduce mild positive charge adjactent to active site</TD></TR><TR><TD>F169L;W176L;F463L</TD><TD>Open up active site adjacent to catalytic residues w/ hydrophobic</TD></TR><TR><TD>F169N;L173N;W176Q;I303Q;F463S</TD><TD>Super solvent exposure to active site</TD></TR><TR><TD>F463W</TD><TD>Better packing around catalytic residues</TD></TR><TR><TD>G457E</TD><TD>G457E occurs in PDB: 1O01, may be natural source of substrate specificity</TD></TR><TR><TD>I303M</TD><TD>Better packing around catalytic residues</TD></TR><TR><TD>I303Q</TD><TD>Introduce hydrophilic character to active site adjactent to catalytic residues</TD></TR><TR><TD>I303S;F463S</TD><TD>Introduce hydrophilic character to active site adjactent to catalytic residues</TD></TR><TR><TD>K475M</TD><TD>Remove negative charge from tunnel</TD></TR><TR><TD>L172Y;Y296W;W176V</TD><TD>Decrease size of hydrophic tunnel region</TD></TR><TR><TD>V301D</TD><TD>Introduce negative charge adjactent to active site</TD></TR><TR><TD>V301H</TD><TD>Introduce mild positive charge adjactent to active site</TD></TR><TR><TD>W176Q</TD><TD>Introduce hydrophilic character to active site adjactent to catalytic residues</TD></TR><TR><TD>W176V</TD><TD>Increase size of hydrophobic pocket at the alkyl end of 2-butenal</TD></TR><TR><TD>Y296L</TD><TD>Created salt bridge at entrance of tunnel</TD></TR></TBODY></TABLE></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UC_Davis/Protein_Engineering_Design">http://2014.igem.org/Team:UC_Davis/Protein_Engineering_Design</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Davis/Protein_Engineering_Design" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Davis/Protein_Engineering_Design" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Davis/Protein_Engineering_Design&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Davis/Protein_Engineering_Design&amp;oldid=393443" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>