"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/Protein Engineering Build
From 2014.igem.org
UC Davis iGEM 2014
Home
Project
Overview
Protein Engineering
Potentiostat Design
Electrochemistry
Signal Processing
Achievements
Parts
Policy & Practices
Overview
Olive Mill Visits
Public Perception
Legislation
Acknowledgements
Overview
Bios
Contact Us
Official Team Profile
Safety
iGEM
Design
Design
Build
Build
Test
Test
Ordered DNA and Gibson Assembly of Plasmids
Gibson Assembly
The sequences for the aldehyde dehydrogenases we characterize were pulled from UniProtKB. The following nucleotide sequences were added to the 3’ and 5’ ends of the aldehyde dehydrogenase genes:
5’ Addition to Insert: 5' - GAAATAATTTTGTTTAACTTTAAGAAGGAGATATACATATG
3’ Addition to Insert: 5' - CTCGAGCACCACCACCACCACCACTGA
The 3’+5’ additions made to the gene insert are identical to with the terminal ends of the linearized pET29b-(+) plasmid. The DNA encoding the aldehyde dehydrogenase gene and 3’+5’ additions were obtained from Life Technologies as DNA Strings. Linearized pET29b-(+) plasmid cut at the NdeI and XhoI restriction sites was also obtained.
Gibson assembly was used to insert the aldehyde dehydrogenase gene into our expression plasmids. Gibson assembly works in three basic steps. First, a 5’ exonuclease chews back the the ends of the insert and the linearized backbone, revealing complimentary 3’ sticky ends which may now anneal to each other. After two DNA segments anneal, a DNA polymerase writes in any additional DNA that the 5’ exonuclease removed. Finally, a DNA ligase repairs any nicks, creating a fully assembled plasmid.
See our protocol for Gibson Assembly
Transforming into BLR strain Escherichia coli
Fully assembled plasmids were transformed into BLR strain Escherichia coli cells for expression. Individual colonies were grown in 3mL of TB and 50ug/mL kanamycin for 18 hours. One milliliter of culture was combined with 50% glycerol and flash frozen for storage.
Kunkel Mutagenesis
We collaborated with Transcriptic to introduce our desired mutations into the Escherichia coli aldehyde dehydrogenase gene using Kunkel mutagenesis. First, single-stranded plasmid DNA was produced using CJ23 strain Escherichia coli with the help of the M13K07 helper phage. The CJ23 strain of E. coli lacks important DNA repair machinery and incorporates a large amount of uracil into its DNA in place of thymine. The helper phage uses the F1 promoter site on the pET29b-(+) plasmid to transcribe a single stranded (+)-strand copy of the uracil-containing plasmid. Next, 33nt oligos were designed to anneal to the single-stranded plasmid where we wanted to introduce a mutation. The oligos consist of a three-nucleotide codon encoding the mutation we wanted to introduce flanked by 15nt on either side complementary to the single-stranded plasmid. DNA polymerase and DNA ligase extend the oligos and ligated any nicks before the newly created double-stranded plasmid was transformed into BLR Strain E. coli. The BLR strain E. coli DNA repair machinery removes the original uracil-containing DNA and uses the newly synthesized mutagenic stand as a template. The result is a complete plasmid containing our desired mutations.
See our protocol for Kunkel Mutagenesis
Expression and Purification
To express and purify our aldehyde dehydrogenases, an ice scrape of previously flash frozen culture was introduced to 25mL of TB containing 50ug/mL kanamycin in a 50mL falcon tube. The falcon tube was covered with a gas-permeable seal and the culture was grown for 24 hours in a 300rpm shaker at 37 degrees celsius. Cultures were pelleted at 4700rpm for 20 minutes and resuspended in 25mL of TB containing 50ug/mL kanamycin and 1mM isopropyl beta-D-1 thiogalactoside (IPTG) to induce expression. Cultures were grown for 30 hours in a 300rpm shaker at 18 degrees celsius and then pelleted at 4700rpm.  Pelleted cells were resuspended in 500uL 1x PBS, pH 7.4 (137mM NaCl, 2.7mM KCl, 10.M Na2HPO4, 2mM KH2PO4). To lyse, 1mL of cells was added to 500uL lysis buffer containing 2% triton, 2mg/mL lysozyme, 0.2mg/mL DNase, 1mM TCEP (tris(2-carboxyethyl)phosphine), and 1mM PMSF. Cells were left to lyse on a rocker for 30 minutes and were then centrifuged at 15,000 rpm for 30 minutes. Supernatant was passed through a column containing HisPur cobalt resin to bind our His-tag containing enzymes. The column was washed with eight one-milliter portions of 1x PBS containing 1mM TCEP and one one-millilieter portion of 1x PBS. Our enzymes were eluted from the column with 1x PBS containing 100mM Imidizole, 5% glycerol, and 1mM DTT.  Enzymes were flash frozen in liquid nitrogen and stored at -80 degrees celsius for long-term storage.
Retrieved from " http://2014.igem.org/Team:UC_Davis/Protein_Engineering_Build "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
