"
Page
Discussion
View source
History
teams
Log in
Â 
Team:UC Davis/Protein Engineering Design
From 2014.igem.org
UC Davis iGEM 2014
Home
Project
Overview
Protein Engineering
Potentiostat Design
Electrochemistry
Signal Processing
Achievements
Parts
Policy & Practices
Overview
Olive Mill Visits
Public Perception
Legislation
Acknowledgements
Overview
Bios
Contact Us
Official Team Profile
Safety
iGEM
Design
Design
Build
Build
Test
Test
Why Aldehyde Dehydrogenases?
The aldehyde dehydrogenase family of enzymes (EC: 1.2.1.3, 1.2.1.5) was selected for use with our electrochemical biosensor. This family of enzymes catalyze the reaction of aliphatic, straight chain aldehydes and the oxidized form of beta-nicotinamide adenine dinucleotide (NAD+) to produce the corresponding carboxylic acid and the reduced form of beta-nicotinamide adenine dinucleotide (NADH).
The aldehyde dehydrogenase enzyme family is perfect for our engineering and electrochemical applications:
This enzyme uses NAD+ as a coenzyme and produces NADH in a 1:1 molar ratio with the amount of aldehyde catalyzed. The concentration of NADH can be readily measured with a spectrophotometer reading absorbance at 340nm, allowing us to easily measure the rate of the reaction catalyzed by an aldehyde dehydrogenase enzyme.
The active site of aldehyde dehydrogenase is in the center of a long tunnel, where NAD+ enters from one side and the aldehyde substrate enters from the other side. This tunnel (highlighted in orange) gives us a large amount of flexibility in engineering amino acid residues which will alter the catalytic efficiency of this enzyme toward certain aldehyde species.
We identified several commercial electrodes which oxidize NADH back to NAD+ and produce a current. This will allow us to measure the amount of NADH produced over time using an electrochemical approach.
Preferred: Crystal structure available on the Protein Data Bank (PDB)
With aldehyde dehydrogenases in mind, we used two approaches to identify enzymes with the desired specificities we would use in our biosensor: bioprospecting and engineering.
Approach 1: Bioprospecting
We used several online databases to search for aldehyde dehydrogenases with unique specificity profiles. Keeping the context of the aldehyde dehydrogenase within our electrochemical sensor in mind, we focused exclusively on aldehyde dehydrogenases which used NAD(H) as a coenzyme (EC: 1.2.1.3) as opposed to aldehyde dehydrogenases which used NADP(H) (EC: 1.2.1.5). However, we did identify several aldehyde dehydrogenases which could use either coenzyme. BRENDA was our main resource for identifying aldehyde dehydrogenases which catalyzed the oxidation reaction on specific substrates. The specificity data provided within the literature cited for each entry on BRENDA helped guide our decisions for the set of enzymes we chose to characterize.
Selection Criteria:
DNA sequence readily available on Genbank or UniProtKB
Aldehyde dehydrogenase possesses unique specificity for one or more of the aldehyde species known to occur in olive oil
Aldehyde dehydrogenase uses NAD+ as a coenzyme for the dehydrogenase reaction
Aldehyde dehydrogenase is documented to express in Escherichia coli and is easily purified in high yield (>1 mg/mL)
Source
UniProtKB ID
EC Number
Crystal Structure Available?
Escherichia coli (Strain K12)
P23883
1.2.1.5
No
Rhodococcus erythropolis
Q4F895
1.2.1.3
No
Rattus norvegicus (Rat)
P11883
1.2.1.5
1AD3
Human (ALDH3A1)
P30838
1.2.1.5
3SZA
Human (ALDH2)
P05091
1.2.1.3
1O01
Approach 2: Engineering
We chose the aldehyde dehydrogenase from Escherichia coli as our template for engineering additional specificity profiles outside of the ones we identified in the literature. While there was not a crystal structure available for E. coli aldehyde dehydrogenase, we decided it would be the best template for engineering since it expressed the best (>3 mg protein per 25mL culture) out of all the aldehyde dehydrogenase we investigated. Thus, we generated a homology model of E. coli aldehyde dehydrogenase and used the interactive protein folding design software Foldit to design mutants. Investigation of the homology model and crystal structures of other aldehyde dehydrogenases revealed that substrates must travel through a tunnel composed of hydrophobic residues to reach the catalytic center of the enzyme. Taking this into consideration, our design strategy was to alter the residues in the substrate tunnel in an attempt to change the enzyme's catalytic efficiency towards certain substrates.
We used four strategies for designing our mutants:
Introducing hydrophilic residues in the substrate tunnel to discourage catalysis of more hydrophobic aldehydes
Increasing solvent exposure to the catalytic center by introducing smaller residues in the tunnel
Increasing packing around the catalytic center with larger hydrophobic residues
Introducing positive/negative charges in the substrate tunnel
We designed a total of 20 mutants:
Mutations
Description
D119I;F169A;Y296A
Increase solvent exposure to active site, open enterance to tunnel
D119L;Y296F;N455A
Increase hydrophobicity at far end of tunnel
D456L;D458L
Change far pocket polar residues to posess more nonpolar characteristics
F169A
Open up active site adjacent to catalytic residues w/ hydrophobic
F169D
Introduce negative charge adjactent to active site
F169H;F463H
Introduce mild positive charge adjactent to active site
F169L;W176L;F463L
Open up active site adjacent to catalytic residues w/ hydrophobic
F169N;L173N;W176Q;I303Q;F463S
Super solvent exposure to active site
F463W
Better packing around catalytic residues
G457E
G457E occurs in PDB: 1O01, may be natural source of substrate specificity
I303M
Better packing around catalytic residues
I303Q
Introduce hydrophilic character to active site adjactent to catalytic residues
I303S;F463S
Introduce hydrophilic character to active site adjactent to catalytic residues
K475M
Remove negative charge from tunnel
L172Y;Y296W;W176V
Decrease size of hydrophic tunnel region
V301D
Introduce negative charge adjactent to active site
V301H
Introduce mild positive charge adjactent to active site
W176Q
Introduce hydrophilic character to active site adjactent to catalytic residues
W176V
Increase size of hydrophobic pocket at the alkyl end of 2-butenal
Y296L
Created salt bridge at entrance of tunnel
Retrieved from " http://2014.igem.org/Team:UC_Davis/Protein_Engineering_Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
