<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:Aix-Marseille"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Team"><DIV class="menu_item direct_link">
				TEAM
			</DIV></A><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Aix-Marseille/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Protocols"><DIV class="submenu_item">
					Protocols				
                               </DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Perspectives"><DIV class="submenu_item">
					Perspectives 
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Safety"><DIV class="submenu_item">
					Safety
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Aix-Marseille/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Composite_Part"><DIV class="submenu_item">
					Composite Parts
				</DIV></A></DIV><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Aix-Marseille/Human_Practices"><DIV class="submenu_item">
					Integrated HP
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A></DIV><DIV class="menu_item"> 
			ACKNOWLEDGEMENTS  
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Aix-Marseille/Sponsors"><DIV class="submenu_item">
					Sponsors
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Attributions"><DIV class="submenu_item">
					Attributions
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Donors"><DIV class="submenu_item">
					Crowdfunding
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=Aix-Marseille"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><H1 class="main_title">Design</H1></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><P>The project goal is to lure the bed bugs into a trap loaded with a specific mix of pheromones. These pheromones will be produced using engineered <I>E. coli</I> bacteria.
In the trap, the bed bugs will be infected with spores of the fungal entomopathogen <I>Beauveria Bassiana</I>. 
They will then return to their colony, or nest, and infect all the other bed bugs, and their eggs, with our fungus so killing the nest. 
This epidemic infection will be ensured by fungal sporulation. 
The spores should also provide lasting protection in the nest site against reinfection.
</P><H2><SPAN class="mw-headline" id="Trap_design">Trap design</SPAN></H2><P>A prototype was easily manufactured with simple materials as cardboard, glue, insect neutral strip and coton. There are two entrances for bed bugs.
</P><P>To test the attractive efficiency of pheromones, we elaborated a trap prototype with a cocktail of pheromones. We used DMDS and DMTS (commercial chemicals). Researchers evaluated the attractive power of these two pheromones <A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/pubmed/25529634">[1]</A>. These two compounds are particularly volatile with an exceptional power to lure bed bugs. To optimize the cocktail, we added  (E)-2-hexenal, (E)-2-octenal, 2-hexanone, acetophenone, octenal, hexanone at 1.7 mg/mL mixed in mineral oil.
</P><P>To have significant results, we have elaborated a strategy. We created three groups of  :  traps with pheromones, traps without pheromones (negative control), and traps with the  pheromones cocktail surrounded by <I>Beauveria bassiana</I> on a cotton support.
</P><P>A testing protocol was inspired by : <A rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/25529634">Bed bug aggregation pheromone finally identified</A>. The traps were seperated by 30 cm each, and they were installed in three different locations in the infested room.
</P><H2><SPAN class="mw-headline" id="Pheromones">Pheromones</SPAN></H2><P>The first part of our trap is based on a pheromone cocktail to attract bed bugs into our trap. 
Bed bugs use a complex collection of pheromones and we aimed in our project to produce 4: DMDS, DMTS, Benzaldehyde and Benzyl alcohol.
</P><H3><SPAN class="mw-headline" id="DMDS_and_DMTS">DMDS and DMTS</SPAN></H3><P>Some studies have shown that DMDS (dimethyl disulfide) and DMTS (dimethyl trisulfide) are the most attractive pheromones for bed bugs <A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/pubmed/25529634">[2]</A>.
</P><P>These two volatile molecules are present in bed bugs dejections, and they allow the bed bugs to find their way back to the colonies when they go out at night. 
In order to make the bed bugs come to our trap, we want to create an engineered <I>E. coli</I>, that will produce these 2 pheromones.
</P><P>The DMDS and DMTS biosynthesis is a result of the degradation of sulfur-containing amino acids to thiol, ammonia and alpha keto-acids by using methionine gamma-lyase (MGL). The methyl sulfide produced is then naturally oxidized to DMDS and DMTS as shown below:
</P><P>A similar design was already used by the <A rel="nofollow" class="external text" href="https://2014.igem.org/Team:Wageningen_UR/overview">Wageningen UR iGEM team in 2014</A>. 
In order to test our part, to ensure that there was a production of the protein and that the produced protein was active. 
We decided to modify the part they used in their project <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1493300">BBa_K1493300</A> in two ways.
First, by adding a histidine tag, so making part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2718005">BBa_K2718005</A>, this is to allow us to purify the protein, and also to test production using an antibody.
This part was then coupled to an inducible promoter <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:R0011">BBa_R0011</A> to make the composite part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2718006">BBa_K2718006</A>, allowing the inducible production in <I>E. coli</I>.
</P><A rel="nofollow" class="external autonumber" href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition">[3]</A><P><I>E.coli</I> bacteria is a good model organism to do such a construction, as it produces L-methionine by itself <A rel="nofollow" class="external autonumber" href="https://biocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&amp;object=HOMOSER-METSYN-PWY">[4]</A>.
</P><P>In our project, we used the methionine gamma-lyase enzyme from <I>Brevibacterium linens</I>. 
This MGL protein is an enzyme that belongs to the trans-sulfuration enzyme family. 
It is made up of 425 amino acids and catalyzes the chemical carbon-sulfur lyase reaction <A rel="nofollow" class="external autonumber" href="https://www.uniprot.org/uniprot/Q5MQ31">[5]</A><A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/nuccore/AY622198.1?report=fasta&amp;from=94&amp;to=1371">[6]</A><A rel="nofollow" class="external autonumber" href="https://www.ebi.ac.uk/QuickGO/term/GO:0018826">[7]</A>. 
The part we have created with the MGL gene can be found in our <A rel="nofollow" class="external text" href="https://2018.igem.org/Team:Aix-Marseille/Parts">Parts</A> tab as 
<A rel="nofollow" class="external text" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2718006">BBa_K2718006</A>.
</P><H3><SPAN class="mw-headline" id="Benzaldehyde_and_Benzyl_alcohol">Benzaldehyde and Benzyl alcohol</SPAN></H3><P>We decided to create another engineered <I>E.coli</I> strain, that could produce benzaldehyde and benzyl alcohol.
Benzyl alcohol is also a pheromone that can attract bed bugs<A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/pubmed/25529634">[8]</A>.
</P><P>To do so, we’ve decided to engineer a new metabolic pathway into <I>E. coli</I>, 
this pathway is derived from another natural pathway, found in <I>Pseudomonas fluorescens</I>. 
In this pathway, phenylpyruvate (a substrate in phenylalanine biosynthesis) is used as the starting material for benzyl alcohol and benzaldehyde biosynthesis. 
Engineering this pathway requires the introduction of 3 different enzymes: 
Mandelate synthase, 
(S)-mandelate dehydrogenase and benzoyl formate decarboxylase, 
which are respectively encoded by the <I>hmaS</I>, <I>mdlB</I> and <I>mdlC</I> genes.
</P><A rel="nofollow" class="external autonumber" href="https://www.researchgate.net/publication/278742142_Engineering_Escherichia_coli_for_renewable_benzyl_alcohol_production">[9]</A><P>We worked on those 3 differents genes, coming from different organism : <I>Pseudomonas fluorescens</I> (<I>mdlB</I> and <I>mdlC</I>) and <I>Amycolatopsis orientalis</I> (<I>hmaS</I>), because of there similarity with <I>E.coli</I> and previous experiments in the litterature <A rel="nofollow" class="external autonumber" href="https://www.sciencedirect.com/science/article/pii/S2214030115300018">[10]</A>.
</P><P>In our design, the 3 genes will be put in a single operon with an RBS and strong promoter construction, 
and the operon will be inserted into a plasmid to produce an engineered <I>E.coli</I> strain.
</P><P>But the initial tests were done on 3 differents <I>E.coli</I> strains, each expressing a separate gene from the pathway (<I>hmaS</I>, <I>mdlB</I> or <I>mdlC</I>). 
The 3 sequences were optimized to be correctly expressed in the host organism and ordered with an RBS sequence upstream, as well as a biobrick prefix and biobrick suffix flanking the gene sequence.
</P><H4><SPAN class="mw-headline" id="hmaS"><I>hmaS</I></SPAN></H4><P>Mandelate synthase, or 4-hydroxyphenylpyruvate dioxygenase 2 is an enzyme that catalyzes the chemical reaction below, by acting as a decarboxylase and a hydroxylase: 
</P><DIV class="center" style="width:auto; margin-left:auto; margin-right:auto;"><B> 4-hydroxyphenylpyruvic acid + O2  &lt;=&gt;  mandelate + CO2</B></DIV><P>It belongs to oxidoreductases family of enzymes and is also involved in the vancomycin biosynthesis pathway. 
This enzyme is encoded by hmaS gene, that is present in <I>Amycolatopsis orientalis</I>. 
It is the first step in the biosynthesis of benzyl alcohol<A rel="nofollow" class="external autonumber" href="https://www.uniprot.org/uniprot/G4V4S7">[11]</A><A rel="nofollow" class="external autonumber" href="https://www.ebi.ac.uk/ena/data/view/CCD33152">[12]</A><A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/protein/O52791.1?report=genbank&amp;log$=protalign&amp;blast_rank=1&amp;RID=S7YPXGAC014">[13]</A>.
The parts we have created with the <I>hmaS</I> gene are <A rel="nofollow" class="external text" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2718010">BBa_K2718010</A> with the coding sequence and a ribosome binding site; and <A rel="nofollow" class="external text" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2718011">BBa_K2718011</A> with the promoter <A rel="nofollow" class="external text" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_R0011">BBa_R0011</A>.
</P><H4><SPAN class="mw-headline" id="mdlB"><I>mdlB</I></SPAN></H4><P>(S)-mandelate dehydrogenase is the second enzyme in the benzaldehyde and benzyl alcohol biosynthetic pathway. 
It catalyzes the (S)-mandelate oxidation reaction using NAD+ (or NADP+) as the oxidant. 
The products of the reaction are phenylglyoxylate and the reduced nicotinamide nucleotide:
</P><DIV class="center" style="width:auto; margin-left:auto; margin-right:auto;"><B>(S)-mandelate + NAD(P)+  &lt;=&gt;  phenylglyoxylate + NAD(P)H + H+</B></DIV><P>This enzyme is encoded by the <I>mdlB</I> gene and contains 393 amino acids. 
We worked on the <I>mdlB</I> gene from <I>Pseudomonas putida</I><A rel="nofollow" class="external autonumber" href="https://www.uniprot.org/uniprot/P20932">[14]</A><A rel="nofollow" class="external autonumber" href="https://www.ebi.ac.uk/ena/data/view/AAC15503">[15]</A>.
Unfortunately, we were unable to show a functional gene so we did not submit this part.
</P><H3><SPAN class="mw-headline" id="mdlC"><I>mdlC</I></SPAN></H3><P>Expression of the <I>mdlC</I> gene produces the benzoyl formate decarboxylase enzyme, a protein with  528 amino acids. 
It catalyses the decarboxylation of phenyglyoxylate to produce benzaldehyde (a bed bug pheromone) with the reaction:
</P><DIV class="center" style="width:auto; margin-left:auto; margin-right:auto;"><B> phenylglyoxylate  &lt;=&gt;  benzaldehyde + CO2</B></DIV><P>The benzaldehyde can then be converted benzyl alcohol by an <I>E.coli</I> alcohol dehydrogenase using NAD+ as the oxidant<A rel="nofollow" class="external autonumber" href="https://www.uniprot.org/uniprot/P20906">[16]</A><A rel="nofollow" class="external autonumber" href="https://www.ebi.ac.uk/ena/data/view/AAC15502">[17]</A>.
Benzyl alcohol, like benzaldehyde, is a bed bug pheromone.
Unfortunately, as for <I>mdlC</I>, we were unable to show a working and functional part based on this gene so it was not submitted to the registry.
</P><H2><SPAN class="mw-headline" id="Beauveria_bassiana"><I>Beauveria bassiana</I></SPAN></H2><P>Entomopathogenic fungi are natural biopesticides acting through contact. 
<I><B>Beauveria bassiana</B></I> was first used commercially to fight insects pests in agriculture in Canada, and since then its use has spread all over the world. 
<I><B>Beauveria bassiana</B></I> is capable of infecting a broad range of insect hosts including bed bugs.
The fungus penetrates bed bug's exoskeleton: the cuticle. 
Once inside the insect's hemolymph, the fungus feeds and grows eventually causing the death of the insect<A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/pubmed/22555012">[18]</A>. 
</P><P><A rel="nofollow" class="external autonumber" href="https://www.lemonde.fr/sciences/article/2015/08/31/les-fourmis-malades-et-medecins_4741717_1650684.html">[19]</A></P><P>Entomopathogenic fungi are common and widespread in almost all classes of insects,
and <I><B>Beauveria bassiana</B></I> is one of the few being developed for commercial use. Indeed, in Canada, <B>Silvia Todorova</B> created bio-insecticide based on a Quebec strain of <I>Beauveria bassiana</I>.  <A rel="nofollow" class="external autonumber" href="http://anatisbioprotection.com/blog/bioceres-beauveria-bassiana.html">[20]</A></P><P>
One interest of entomopathogenic fungi is their ability to attack all stages in the insect life cycle as they are effective against eggs, larvae, intermediate stages and adults.
</P><P><I><B>Beauveria bassiana</B></I> has been known for a long time causing white muscardine disease.
It is commonly known as white muscardine fungus, and was originally classified as <I>Tritirachium shiotae</I>, 
it grows naturally in soils throughout the world and acts as a pathogen on various insect species<A rel="nofollow" class="external autonumber" href="https://pdfs.semanticscholar.org/1463/3f328f5c541399802579e20e153012223f25.pdf">[21]</A><A rel="nofollow" class="external autonumber" href="https://rspublication.com/ijst/2014/dec14/35.pdf">[22]</A>.
</P><P>Our goal is to use this fungus to fight bed bugs. 
Our original objective was to <B>enhance the fungus</B> to make it virulent for bed bugs. 
We wanted to <B>reduce the lethal doses and break the bugs faster</B>. 
However, in view of the biosafety considerations and public acceptability of hyper-virulent GMO pathogens,
from our <A rel="nofollow" class="external text" href="https://2018.igem.org/Team:Aix-Marseille/Human_Practices">Human Practices work</A>, we opted to break bugs faster by adding enzymes to our traps.
</P><P>We did some <A rel="nofollow" class="external text" href="https://2018.igem.org/Team:Aix-Marseille/Experiments#bb">initial tests</A> with our <I>Beauveria bassiana</I> sample on garden bugs. 
Then, we tested the ability of this fungus to grow on different culture media to prepare fungus for our traps.
At the end of our project, we tested <I>Beauveria bassiana</I> in real conditions on bed bugs.
</P><H2><SPAN class="mw-headline" id="Chitinase_and_lipase">Chitinase and lipase</SPAN></H2><P>Chitinase was an essential biobrick to design to enhance the virulence of the entomopathogenic fungus <I><B>Beauveria bassiana</B></I>.
After bibliographic researches to design chitinase biobrick, chit1 gene from <I><B>Beauveria bassiana</B></I> was selected because  a paper demonstrated that an overexpression of this gene has been a success for increased fungus virulence <A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC544255/">[23]</A>.
</P><P>For the design of our chitinase brick <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2718020">BBa_K2718020</A>, 
a research of the secretion signal peptide sequence localization and the linker region between it and the catalytic domain was realized.
We studied in detail the sequence to locate these different domains. 
The plan was to separate the chit1 signal sequence from catalytic domain to conserve only the catalytic domain, and optionally be able to add its peptide signal sequence paired to a promoter-RBS. 
This initial plan was to create a new biobrick with a functional signal peptide in RFC25. 
This standardization is one of the principles of engineering supported by the biobrick concept. 
This promoter-RBS-secretion peptide signal biobrick would be useful to others in future projects.
Unfortunately, this part of the project did not get completed. 
The INTERPRO plateform was used to identify the catalytic domain of chit1 sequence. 
The final chitinase biobrick contained the catalytic domain of chit1. 
This sequence was optimized to be the most compatible with <I>E.coli</I> codon expression. 
</P><P>The RFC25 sufix and prefix were used for design chitinase biobrick to be compatible with adding a targeting signal peptide, for example with the promoter-RBS-chit1 signal peptide biobrick.
</P><P>The decision to use <I>E.coli</I> for chitinase production was to produce chitinase easily in order to test the functionality of the protein by activity tests. 
<I>E.coli</I> is an easy chassis to use for quick results, and we anticipated with the short time in the laboratory this would be a good choice.
We also planned to test the potential of the enzyme to increase the virulence of <I>Beauveria bassiana</I> in tests on bed bugs. 
</P><P>For chitinase expression by <I>E.coli</I>, we decided to used an inducible promoter to avoid toxicity problems from bacteria, a constitutive expression are often problematical for bacteria. 
We planned use pLac promoter to can easily induce expression thanks to IPTG induction.
</P><P>As lipase is also important to allow the fungus to cross the first layer of bed bug's cuticle. 
We also wished to express the <I><B>Beauveria bassiana</B></I> that is important for killing bed bugs.
We had planned to use the same strategy for a lipase biobrick but unfortunately, IDT wasn't able to, the synthesize our sequence. 
As a second attempt, we tried to order the lipase sequence in two parts to increase chances of synthesis by IDT, unfortunately again this failed. 
As a final strategy, we decided to try to amplify the gene for the lipase gene directly from genomic DNA. For this, we designed designing primers, ordered them, and attempted the PCR but again the experiments failed. 
So we decided to drop this part of the project.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>