<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:Aix-Marseille"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Team"><DIV class="menu_item direct_link">
				TEAM
			</DIV></A><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Aix-Marseille/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Protocols"><DIV class="submenu_item">
					Protocols				
                               </DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Perspectives"><DIV class="submenu_item">
					Perspectives 
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Safety"><DIV class="submenu_item">
					Safety
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Aix-Marseille/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Composite_Part"><DIV class="submenu_item">
					Composite Parts
				</DIV></A></DIV><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Aix-Marseille/Human_Practices"><DIV class="submenu_item">
					Integrated HP
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A></DIV><DIV class="menu_item"> 
			ACKNOWLEDGEMENTS  
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Aix-Marseille/Sponsors"><DIV class="submenu_item">
					Sponsors
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Attributions"><DIV class="submenu_item">
					Attributions
				</DIV></A><A href="https://2018.igem.org/Team:Aix-Marseille/Donors"><DIV class="submenu_item">
					Crowdfunding
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=Aix-Marseille"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><H1 class="main_title">Notebook</H1></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><P><DETAILS><SUMMARY>WEEK 1</SUMMARY></DETAILS></P><H2><SPAN class="mw-headline" id="Training">Training</SPAN></H2><P><I>During the first week, we had several training sessions, under the supervision of Gauthier DANGLA-PELISSIER (Instructor), to get acquainted with the lab work, master everyday protocols (PCR/Colony-PCR, competent cells preparation, transformation, cloning, minipreps, PCR clean-ups...), and establish a classic workflow.</I></P><H4><SPAN class="mw-headline" id="Day_1:_Lab_setup">Day 1: Lab setup</SPAN></H4><P>We recovered equipment from our home university, cleaned-up the lab, and had a first meeting to establish the project's workflow.
</P><H4><SPAN class="mw-headline" id="Day_2:_Preparing_DH5.CE.B1_competent_cells_stock_and_transformation">Day 2: Preparing <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A> stock and <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A></SPAN></H4><P>We prepared a stock of <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>. Afterward, we prepared Petri dishes and we <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformed the cells</A> to test their efficiency using a plasmid provided by our instructor.
</P><H4><SPAN class="mw-headline" id="Day_3:_Colony-PCR_and_agarose_gel_electrophoresis">Day 3: Colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> and agarose gel electrophoresis</SPAN></H4><P>The next day we had colonies on our Petri dishes and wanted to check if they acquired the transformed plasmid with the right inserted gene. We picked out some supposedly positive colonies and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A>. We then migrated the PCR products on an agarose gel.
</P><H4><SPAN class="mw-headline" id="Day_4:_DNA_purification_miniprep">Day 4: <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A></SPAN></H4><P>We purified the plasmid with the right inserted gene from the confirmed positive colonies using a DNA purification kit.
In a further step, we digested the purified plasmid with respective restriction enzymes to double-check the presence of the inserted gene.
</P><H4><SPAN class="mw-headline" id="Day_5:_Western-blot">Day 5: <A href="/Team:Aix-Marseille/Protocols#Western_blot" title="Team:Aix-Marseille/Protocols">Western-blot</A></SPAN></H4><P>The last day, we checked the induced production rate of the relative protein to the inserted gene (Produced in a preliminary step by our instructor).
<DETAILS><SUMMARY>WEEK 2</SUMMARY></DETAILS></P><P><I>For the whole week, we ran into a lot of problems making <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>, so we spent a lot of time optimizing the protocol.</I></P><H4><SPAN class="mw-headline" id="Day_1">Day 1</SPAN></H4><P><B>Biobricks design</B>: We designed Hmas, MdlB, MdlC and Hyd1 biobricks.<B>Megaprimers design</B>: We designed megaprimers to add the Tag-his to our biobriks sequences.<B>Chitinase biobrick</B>: We received the chitinase gene sequence from IDT and ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> to amplify it. <B>Methionine-γ-lyase biobrick</B>: We recovered the biobrick from the iGEM 2018 kit.</P><H4><SPAN class="mw-headline" id="Day_2">Day 2</SPAN></H4><P><B>pLacI promoter biobrick</B>: We recovered the biobrick from the iGEM 2018 kit to assemble it with other biobricks.<B>Chitinase biobrick</B>: We took the PCR product and verified the amplification on an agarose gel. We ran into hybridization problems so we had to determine the optimal hybridization temperature using a gradient-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A>.</P><H4><SPAN class="mw-headline" id="Day_3">Day 3</SPAN></H4><P><B>Chitinase biobrick</B>: We cloned the PCR product in a pSB1C3 plasmid.<B>Methionine-γ-lyase biobrick</B>: We transformed the biobrick (From the iGEM 2018 kit) into BL21 competent strains because we ran into problems making competent DH5α strains .</P><H4><SPAN class="mw-headline" id="Day_4">Day 4</SPAN></H4><P><B>Chitinase biobrick</B>: We ran another cloning process on the previous <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> product and transformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>Methionine-γ-lyase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on selected colonies. Afterward, we launched starters on the confirmed positive colonies.<B>Interlab</B>: We had a training on the measurement devices and created the respective programs.<B>Biobrick design</B>: We designed a new biobrick (OmpT-AIDAC) a cleavage sequence.<B>RFP biobrick</B>: We recovered the biobrick (From the iGEM 2018 kit) that will be used for white-red screenings.</P><H4><SPAN class="mw-headline" id="Day_5">Day 5</SPAN></H4><P><B>pLacI promoter biobrick</B>: We transformed the plasmid into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>Chitinase biobrick</B>: We transformed DH5α strains with the plasmid containing the chitinase sequence.<B>Interlab</B>: We recovered the 8 biobricks (From the iGEM 2018 kit) and transformed them into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>Methionine-γ-lyase biobrick</B>: We purified the DNA using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and stocked it.<B>MdlB biobrick</B>: We received the biobrick and ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on it.<B>RFP biobrick</B>: We transformed the biobrick into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<DETAILS><SUMMARY>WEEK 3</SUMMARY></DETAILS></P><H4><SPAN class="mw-headline" id="Day_1_2">Day 1</SPAN></H4><P><B>pLacI promoter biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies.<B>RFP biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and launched overnight starters to purify the plasmid later on (No colony-PCR because the colonies are red: RFP expression).<B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies.
We amplified the chitinase IDT gene sequence to recover the stock.<B>Methionine-γ-lyase biobrick</B>: We recovered the methionine-gamma-lyase biobrick (from the iGEM 2018 kit) and transformed it into [Team:Aix-Marseille/Protocols#DH5α_competent_cells_preparation|DH5α competent cells]].<B>InterLab</B>: We recovered the 8 <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformations</A> and launched 8 starters in duplicates.</P><H4><SPAN class="mw-headline" id="Day_2_2">Day 2</SPAN></H4><P><B>pLacI promoter biobrick</B>: We purified the plasmid with the inserted RFP gene using a DNA purification kits, and restarted because the final DNA concentration was too low (13 ng/µL). <B>RFP biobrick</B>: We purified the inserted RFP gene using DNA purification kits. <B>Chitinase biobrick</B>: We ran a gradient-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> to find the optimal primers hybridization temperature. We then extracted the DNA using a gel extraction kit, digested the gene with respective enzymes, and ran an overnight ligation in a pSB1C3 plasmid at 16°C.<B><A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A></B>: We prepared a new <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A> stock and verified their efficiency using an RFP plasmid.<B>Methionine-γ-lyase biobrick</B>: We ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the positive colonies and verified it on an agarose gel.<B>InterLab</B>: We launched the 3 calibration measurements and ran the first cell measurements.</P><H4><SPAN class="mw-headline" id="Day_3_2">Day 3</SPAN></H4><P><B>pLacI promoter biobrick</B>: We purified the DNA using a different <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit.<B>Chitinase biobrick</B>: We ran a ligation in pSB1C3 at room temperature.<B><A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A></B>: We launched DH5α starters to renew the competent cells stock.<B>Methionine-γ-lyase biobrick</B>: We purified the DNA using the Promega <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit. Afterward, we launched an overnight 
<A href="/Team:Aix-Marseille/Protocols#Exponential_megapriming_PCR_.28EMP.29" title="Team:Aix-Marseille/Protocols">megapriming</A> using specific primers to add a Tag-his to our gene sequence so we can purify the resulting protein later on.<B>InterLab</B>: We analyzed the collected data and launched new starters to run a second measurement test.<B>MdlB biobrick</B>: We ran a ligation in pSB1C3 at room temperature.</P><H4><SPAN class="mw-headline" id="Day_4_2">Day 4</SPAN></H4><P><B>Methionine-γ-lyase biobrick</B>: We digested the magapriming product by DpnI (To cut the methylated GATC sites), and transformed it into DH5α competent strains.<B><A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A></B>: we renewed our stock and verified their efficiency using the RFP plasmid.<B>InterLab</B>: We ran a second measurement test.<B>MdlB biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies.<B>Extracurricular</B>: We made our breaking bugs logo using <I>E. coli</I> strains that express the GFP.</P><H4><SPAN class="mw-headline" id="Day_5_2">Day 5</SPAN></H4><P><B><A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A></B>: We double-checked the competent cells efficiency by transforming the RFP plasmid.<B>MdlB biobrick</B>: We recovered the overnight starters, and purified the DNA using minpreps kits, measured the DNA concentration on the NanoDrop and ran a digestion test.<DETAILS><SUMMARY>WEEK 4</SUMMARY></DETAILS></P><P><I>We had the same problems making <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A> (contaminations...). We had the help of our instructor who pointed out a lot of flaws with what we were doing in the process.</I></P><H4><SPAN class="mw-headline" id="Day_1_3">Day 1</SPAN></H4><P><B>Chitinase biobrick</B>: We took stocked strains and launched starters to purify the DNA the next day.<B>MdlB biobrick</B>: We tried cloning the gene sequence into pSB1C3 and transformed the prouct into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>. Furthermore, we ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the IDT sequence to renew our stock.<B>Interlab</B>: We ran measurments for the third time and launched the CFUs protocol.<B>Methionine-γ-lyase biobrick</B>: We sent the magrapriming product to sequencing to check if we managed to add the Tag-his.</P><H4><SPAN class="mw-headline" id="Day_2_3">Day 2</SPAN></H4><P><B>Chitinase biobrick</B>: We purified the DNA using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and had 225,87 ng/uL n concentration. Afterward, we sent the product for sequencing, and simultaneously, launched a <A href="/Team:Aix-Marseille/Protocols#Exponential_megapriming_PCR_.28EMP.29" title="Team:Aix-Marseille/Protocols">megapriminging</A>  to add the Tag-his for <A href="/Team:Aix-Marseille/Protocols#Protein_purification" title="Team:Aix-Marseille/Protocols">purification</A> later-on.<B>RFP biobrick</B>: We amplified it to renew our stock for contamination purposes.<B>Interlab</B>: We had a training on how to do dilution series.<B>Methionine-γ-lyase biobrick</B>: The sequencing results were positve: we managed to add the Tag-his to our sequence.<B>MdlB biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies.<B>Hyd1 biobrick</B>: We received the gene sequence from IDT, ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on it and stocked it. </P><H4><SPAN class="mw-headline" id="Day_3_3">Day 3</SPAN></H4><P><B>Chitinase biobrick</B>: We digested the <A href="/Team:Aix-Marseille/Protocols#Exponential_megapriming_PCR_.28EMP.29" title="Team:Aix-Marseille/Protocols">megapriminging</A> product by DpnI and transformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>MdlB biobrick</B>: We tried cloning the gene sequence, for a second time, into pSB1C3 and transformed the product into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>. Furthermore, we ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the IDT sequence to renew our stock.<B>Interlab</B>: We launched starters for a second CFUs test.<B>Hyd1 biobrick</B>: We migrated the <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> product on an agarose gel, and did a second <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> after a first failure. After a second failure, we ran a gradient-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> to determine the correct hybridization temperature.
</P><H4><SPAN class="mw-headline" id="Day_4_3">Day 4</SPAN></H4><P><B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates. No colonies were observable whereas the sequencing results showed positive results (The chitinase sequence was inserted into the pSB1C3 plasmid. We suspected the DH5α cells: turns out they were <I>Bacillus Subtilis</I> strains (Smell check). We had to run another <A href="/Team:Aix-Marseille/Protocols#Exponential_megapriming_PCR_.28EMP.29" title="Team:Aix-Marseille/Protocols">megapriminging</A>.<B>MdlB biobrick</B>: We used the same strains that we used for the chitinase.<B>Interlab</B>: We ran the CFUs test.<B>Hyd1 biobrick</B>: We performed a <A href="/Team:Aix-Marseille/Protocols#Monarch_PCR_and_DNA_cleanup_kit" title="Team:Aix-Marseille/Protocols">PCR clean-up</A> and migrated the PCR product.
</P><H4><SPAN class="mw-headline" id="Day_5_3">Day 5</SPAN></H4><P><B>Chitinase biobrick</B>: We digested the megapriming product by DpnI and transformed it into new <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>Hyd1 biobrick</B>: We cloned the gene sequence in pSB1C3 and trasnformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<DETAILS><SUMMARY>WEEK 5</SUMMARY></DETAILS></P><H4><SPAN class="mw-headline" id="Day_1_4">Day 1</SPAN></H4><P><B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. We launched starters from the confirmed positive colonies for the next day.<B>Interlab</B>: We counted the boxes and calculated the CFU/mL. Afterward, we prepared new Petri dishes to run another CFU test.<B>Hyd1 biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. We had 8 confirmed positive colonies and we launched two starters from two of them for the next day.<B>MdlC biobrick</B>: We received the gene sequence from IDT, then ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> and a gradient-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on it, but we failed.<B>Methionine-γ-lyase biobrick</B>: We ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the pLacI and methionine-gamma-lyase fragments.<B>Hmas biobrick</B>: We received the gene sequence from IDT and ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on it, but failed.</P><H4><SPAN class="mw-headline" id="Day_2_4">Day 2</SPAN></H4><P><B>Chitinase biobrick</B>: We purified the DNA using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and sent the product for sequencing.<B>Interlab</B>: We made the dilution series and incubated the plates overnight.<B>Hyd1 biobrick</B>: We purified the DNA using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and digested it with specific restriction enzymes. We didn't have conclusive results, so we launched new different starters.<B>pLacI promoter biobrick</B>: We launched a <A href="/Team:Aix-Marseille/Protocols#Exponential_megapriming_PCR_.28EMP.29" title="Team:Aix-Marseille/Protocols">megapriminging</A> to add the RBS to the promoter.<B>Methionine-γ-lyase biobrick</B>: We assembled the sequence with the pLacI promoter, tried to clone it in pSB1C3, and transformed the product into  <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.</P><H4><SPAN class="mw-headline" id="Day_3_4">Day 3</SPAN></H4><P><B>MdlB biobrick</B>: We tried cloning the gene sequence into pSB1C3 and transformed the product into DH5α competent cells.<B>Interlab</B>: We counted the boxes and calculated the CFU/mL. We then elaborated a workflow for the flow cytometry measurements.<B>Hyd1 biobrick</B>: We purified the DNA from a confirmed positive colony using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit.<B>pLacI promoter biobrick</B>: We digested the magapriming product by DpnI (To cut the methylated GATC sites), and transformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>Methionine-γ-lyase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we launched starters on the confirmed positive colonies.</P><H4><SPAN class="mw-headline" id="Day_4_4">Day 4</SPAN></H4><P><B>MdlB biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. We then launched starters from the confirmed positive colonies.<B>Hyd1 biobrick</B>: We sent the miniprep product for sequencing.<B>pLacI promoter biobrick</B>: We launched starters on the confirmed positive colonies for the next day.<B>Methionine-γ-lyase biobrick</B>: We purified the DNA using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and sent the product for sequencing.</P><H4><SPAN class="mw-headline" id="Day_5_4">Day 5</SPAN></H4><P><B>Chitinase biobrick</B>: We recovered the sequencing results which were negative.<B>pLacI promoter biobrick</B>: We purified the DNA from a confirmed positive colony using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and sent the product for sequencing.</P><P><DETAILS><SUMMARY>WEEK 6</SUMMARY></DETAILS></P><H4><SPAN class="mw-headline" id="Day_1_5">Day 1</SPAN></H4><P><B>Methionine-γ-lyase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and launched starters of the supposedly positive colonies with the good gene sequence (Assembled with pLacI).<B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. We prepared starters from the confirmed positive colonies.<B>Interlab</B>: We prepared 16 starters from our GFP expressing strains for our flow cytometry measurements.</P><H4><SPAN class="mw-headline" id="Day_2_5">Day 2</SPAN></H4><P><B>MdlB biobrick</B>: We cloned the gene in pSB1C3 and trasnformed it into DH5α  competent cells.<B>Methionine-γ-lyase biobrick</B>: We recovered the sequencing results that were negative (Sequence without pLacI). We to try assembling the pLacI biobrick to are gene sequence a second time.<B>Chitinase biobrick</B>: We purified the DNA using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and sent for sequencing.<B>Hyd1 biobrick</B>: We recovered the sequencing results that were positive.<B>Interlab</B>: We recovered our starters and took the fluorescence measurements on the TECAN and BIO-RAD S3e Cell Sorter flow cytometer.<I><B>Beauveria bassiana</B></I><B>growth</B>: We prepared a specific <A href="/Team:Aix-Marseille/Protocols#Sabouraud_medium_preperation" title="Team:Aix-Marseille/Protocols">medium</A> (Sabouraud) for the fungus so can start the growth process. We had liquid sabouraud stock and sabouraud, agar, and chloramphenicol Petri dishes. </P><H4><SPAN class="mw-headline" id="Day_3_5">Day 3</SPAN></H4><P><B>MdlB biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. We had non-conclusive results.<B>Hmas biobrick</B>: We cloned the gene sequence in pSB1C3 and transformed the product in DH5α  competent cells.<B>Chitinase biobrick</B>: We recovered the sequencing results that were negative (No Tag-his). We had to make another <A href="/Team:Aix-Marseille/Protocols#Exponential_megapriming_PCR_.28EMP.29" title="Team:Aix-Marseille/Protocols">megapriminging</A> to try to add the tag-his on our gene sequencing with minor changes to the protocol.<B>Interlab</B>: We retrieved and analyzed our measurements data.<I><B>Beauveria bassiana</B></I><B>growth</B>: We took aliquots and measured the OD600 to check-out the growth rate. Furthermore, we counted the cells on a Malassez cell.<B>Hmas biobrick</B>: We cloned the gene sequence in pSB1C3 and transformed into DH5α  competent cells.</P><H4><SPAN class="mw-headline" id="Day_4_5">Day 4</SPAN></H4><P><B>Hmas biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we prepare starters from the confirmed positive colonies.<B>Chitinase biobrick</B>: We digested the megapriming product with DpnI and transformed it into DH5α  competent cells.<B>Hmas biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. We had no positive colonies.<I><B>Beauveria bassiana</B></I><B>growth</B>: We took aliquots and measured the OD600 to check-out the growth rate. Furthermore, we counted the cells on a Malassez cell.</P><H4><SPAN class="mw-headline" id="Day_5_5">Day 5</SPAN></H4><P><B>Hmas biobrick</B>: We purified the DNA using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and ran a digestion test to double-check if we had the gene inserted.<B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates where we had no colonies. Another <A href="/Team:Aix-Marseille/Protocols#Exponential_megapriming_PCR_.28EMP.29" title="Team:Aix-Marseille/Protocols">megapriminging</A> attent was launched.<I><B>Beauveria bassiana</B></I><B>growth</B>: We took aliquots and measured the OD600 to check-out the growth rate. Furthermore, we counted the cells on a Malassez cell.<DETAILS><SUMMARY>WEEK 7</SUMMARY></DETAILS></P><H4><SPAN class="mw-headline" id="Day_1_6">Day 1</SPAN></H4><P><B>MdlB biobrick</B>: We tried optimizing the <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> protocol using different mixes because we were unable to amplify the gene properly. Nonetheless, we failed.<B>Lipase biobrick</B>: Because IDT couldn't provide us with the gene sequence, we tried amplifying it from wild-type <I>Beauveria bassiana</I> strains.<B>Methionie-γ-lyase biobrick</B>: We recovered the sequencing results. They were negative, the pLacI assembly to the gene sequence failed.<B>Chitinase biobrick</B>: We digested the megapriming product with DnpI and transformed it in <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>pLacI promoter biobrick</B>: We launched a <A href="/Team:Aix-Marseille/Protocols#Exponential_megapriming_PCR_.28EMP.29" title="Team:Aix-Marseille/Protocols">megapriminging</A> to add the RBS to the sequence.<I><B>Beauveria bassiana</B></I><B>growth</B>: We made dilution series on previously launched starters (At different temperatures 25°C, 30°C, and 37°C)  to check out the growth rate of the fungus and calculate the CFU/mL.
</P><H4><SPAN class="mw-headline" id="Day_2_6">Day 2</SPAN></H4><P><B>MdlC biobrick</B>: We ran a gradient-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> using the Phusion-HF DNA polymerase. We had non-conclusive positive results.<B>Methionie-γ-lyase biobrick</B>: We tried assembling the to genes while cloning them (We digested pLacI with SpeI and PstI, and digested The tagged methionine-gamma-lyase sequence with XbaI and PstI.<B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates but had no colonies. Afterward, we tried transforming the megapriming product for the second time and added an RFP control. <B>pLacI promoter biobrick</B>: We digested the megapriming product with Dpn1 and transformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>Hmas biobrick</B>: We purified the DNA from an overnight culture, ran a digestion test and sent for sequencing.<I><B>Beauveria bassiana</B></I><B>growth</B>: We counted the cells using a Malassez cell.</P><H4><SPAN class="mw-headline" id="Day_3_6">Day 3</SPAN></H4><P><B>MdlC biobrick</B>: We ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the gene sequence and had positive results. Afterward, we purified the DNA using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and had 70 ng/uL in concentration.<B>MdlB biobrick</B>: We ran another gradient-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> using the Phusion-HF DNA polymerase, and we increased the cycles' number. We had negative results. <B>Methionie-γ-lyase biobrick</B>: We ran the ligation process and transformed the product in <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we launched starters on the confirmed positive colonies.<B>pLacI promoter biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we launched starters on the confirmed positive colonies.<B>Hmas biobrick</B>: We recovered the sequencing results that were negative.<I><B>Beauveria bassiana</B></I><B>growth</B>: After 2 days of growth, we found out that the dilution series product are yeast.</P><H4><SPAN class="mw-headline" id="Day_4_6">Day 4</SPAN></H4><P><B>MdlC biobrick</B>: We digested the sequence with EcoRI and PstI to clone it in pSB1C3 the next day.<B>Methionie-γ-lyase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we launched starters on the confirmed positive colonies.<B>Chitinase biobrick</B>: We purified the DNA using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and sent for sequencing.<B>pLacI promoter biobrick</B>: We purified the DNA using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and sent for sequencing.<I><B>Beauveria bassiana</B></I><B>growth</B>: We spent time debbugging the problem and tried to find a solution to eliminate the yeats.</P><H4><SPAN class="mw-headline" id="Day_5_6">Day 5</SPAN></H4><P><B>MdlC biobrick</B>: We cloned the gene sequence in pSB1C3 and transformed it into competent cells.<B>MdlB biobrick</B>: We ran another <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the gene sequence, but increased the matrix volume in the <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> mix. We had conclusive and positive results.<B>Methionie-γ-lyase biobrick</B>: We purified the plasmid using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and sent it for sequencing. We then digested the sequence with EcoRI and PstI to clone it in pSB1C3.<I><B>Beauveria bassiana</B></I><B>growth</B>: We recovered the plates growing at 25°C, where we observed preliminary filamentous colonies.</P><P><DETAILS><SUMMARY>WEEK 8</SUMMARY></DETAILS></P><H4><SPAN class="mw-headline" id="Day_1_7">Day 1</SPAN></H4><P><B>MdlC biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates where we had no colonies. We ran another digestion test on the MdlC gene sequence for cloning.<B>HmaS biobrick</B>: We digested the gene sequence with EcoRI and PstI for cloning in pSB1C3. We had the right band at the right size when migrated on an agarose gel.<B>Chitinase biobrick</B>: We ran a gradient-<A href="/Team:Aix-Marseille/Protocols#Exponential_megapriming_PCR_.28EMP.29" title="Team:Aix-Marseille/Protocols">megapriminging</A> test using AS-25 chitinase's primers.<B>Methionine-γ-lyase biobrick</B>: We recovered the sequencing positive results. We managed to add the pLacI promoter to the gene sequence. We then digested our DNA with EcoRI an PstI to clone it in the pSB1C3 plasmid.</P><H4><SPAN class="mw-headline" id="Day_2_7">Day 2</SPAN></H4><P><B>MdlC biobrick</B>: We cloned the digested gene sequence in a pSB1C3 plasmid and transformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.
We then ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> to renew the DNA stock.<B>MdlB biobrick</B>: We ran gradient-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> using a Phusion DNA polymerase to determine the optimal primers hybridization temperature.<B>HmaS biobrick</B>: We cloned the digested gene sequence in pSB1C3 and transformed into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>. Afterward, we ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> to renew our DNA stocks but failed.<B>Chitinase biobrick</B>: We digested the megrapriming product by DpnI and transformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>Methionine-γ-lyase biobrick</B>: We cloned our digested gene sequence in pSB1C3 and transformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.</P><H4><SPAN class="mw-headline" id="Day_3_7">Day 3</SPAN></H4><P><B>MdlC biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. We then prepared starters from the confirmed positive colonies for plasmid purification.<B>MdlB biobrick</B>: We <A href="/Team:Aix-Marseille/Protocols#Monarch_PCR_and_DNA_cleanup_kit" title="Team:Aix-Marseille/Protocols">cleaned-up</A> the gradient-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> product and digested the target plasmid. We had negative results when migrated on an agarose gel.<B>HmaS biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we prepared starters on the confirmed positive colonies for plasmid purification.<B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. We then prepared starters from the confirmed positive colonies for plasmid purification.<B>Methionine-γ-lyase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we prepared starters from the confirmed positive colonies for plasmid purification.<B>Constructs for</B><I><B>Beauveria bassiana</B></I>: We searched for plasmids to transform our fungus. We found pEG202 and pJG4-5 (with prokaryotic and eukaryotic replication origins). Afterward, we trasnformed them into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.
We designed the <I>bar</I> phosphinothricin resistance gene, a biobrick for fungus' growth selection.</P><H4><SPAN class="mw-headline" id="Day_4_7">Day 4</SPAN></H4><P><B>MdlC biobrick</B>: We purified the plasmid using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit an sent samples for sequencing.<B>HmaS biobrick</B>: We purified the plasmid and sent samples for sequencing.<B>Methionine-γ-lyase biobrick</B>: We purified the DNA, transformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>, and sent samples for sequencing.<B>Constructs for</B><I><B>Beauveria bassiana</B></I>: We recovered the trandformation plates and prepared starters for plasmid purification.
We recovered three biobricks from the iGEM 2018 kit: GPD eukaryotic promoter, eukaryotic IRES RBS, eukaryotic terminator. Afterward, we transformed them into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.</P><H4><SPAN class="mw-headline" id="Day_5_7">Day 5</SPAN></H4><P><B>Chitinase biobrick</B>: We purified the plasmid using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit an sent samples for sequencing.<B>Methionine-γ-lyase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. <B>Constructs for</B><I><B>Beauveria bassiana</B></I>: We purified the plasmids using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit. To double-check the plasmids we digested pEG202 with NgoMIV and XbaI and pJG4-5 with AgeI and SpeI.
We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> for double-checking.</P><P><DETAILS><SUMMARY>WEEK 9</SUMMARY></DETAILS></P><H4><SPAN class="mw-headline" id="Day_1_8">Day 1</SPAN></H4><P><B>MdlC biobrick</B>: We faced a lot of problems with the amplification from the IDT stock. We spent the week debugging and optimizing our protocol.<B>HmaS biobrick</B>: We recovered the sequencing results. We were able to clone the gene. Afterward, we cloned it in another plasmid to add the pLacI promoter. We transformed the product into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<B>Chitinase biobrick</B>: We ran another an exponential <A href="/Team:Aix-Marseille/Protocols#Exponential_megapriming_PCR_.28EMP.29" title="Team:Aix-Marseille/Protocols">megapriminging</A> PCR to add a tag-his to our gene sequence. Afterward, we digested the <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> product we DpnI and transformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.<I><B>Beauveria bassiana</B></I><B>growth</B>: We prepared <A href="/Team:Aix-Marseille/Protocols#Beauveria_bassiana_competent_blastospores" title="Team:Aix-Marseille/Protocols">competent blastospores</A> for future use, and stocked aliquotes at -80°C.</P><H4><SPAN class="mw-headline" id="Day_2_8">Day 2</SPAN></H4><P><B>HmaS biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we prepared starters from the confirmed positive colonies.<B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we prepared starters from the confirmed positive colonies.</P><H4><SPAN class="mw-headline" id="Day_3_8">Day 3</SPAN></H4><P><B>HmaS biobrick</B>: We purified the DNA plasmid using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit and sent samples for sequencing.<B>Chitinase biobrick</B>: We purified the DNA plasmid using a <A href="/Team:Aix-Marseille/Protocols#Monarch_miniprep_kit" title="Team:Aix-Marseille/Protocols">DNA purification miniprep</A> kit. Firstly, we digested the plasmid with XbaI and PstI to clone the gene in a plasmid with pLacI and RBS. Secondly, we digested the plasmid with NgoI and PstI to clone the gene in another plasmid with pLacI, RBS, and a signal peptide.</P><H4><SPAN class="mw-headline" id="Day_4_8">Day 4</SPAN></H4><P><B>HmaS biobrick</B>: We recovered the sequencing results. We were able to clone the gene. Afterward, we cloned it in a pSB1C3 plasmid.<B>Chitinase biobrick</B>: We <A href="/Team:Aix-Marseille/Protocols#Monarch_PCR_and_DNA_cleanup_kit" title="Team:Aix-Marseille/Protocols">cleaned-up</A> the digestion products using the Machinery-Nagel kit. Afterward, we cloned the digested genes in respective plasmids and trasnformed the products into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.</P><H4><SPAN class="mw-headline" id="Day_5_8">Day 5</SPAN></H4><P><B>MdlB biobrick</B>: We received the sequence from IDT with an added tag-his. Then, we ran a <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on it to amplify the gene.
<B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we prepared starters from the confirmed positive colonies.</P><P><DETAILS><SUMMARY>WEEK 10</SUMMARY></DETAILS></P><H4><SPAN class="mw-headline" id="Day_1_9">Day 1</SPAN></H4><P><B>MdlC biobrick</B> and <B>MdlB biobrick</B>: For the whole week, we spent a lot of time optimizing the <A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> protocol (Double checking primer annealing, modfying the PCR program...).<B>HmaS biobrick</B>: We transformed the pSB1C3 plasmid in BL21 strains to start the protein production using the following protocol.<B>Chitinase biobrick</B>: We cloned the gene sequence in pLacI pSB1C3 plasmid and transformed into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.</P><H4><SPAN class="mw-headline" id="Day_2_9">Day 2</SPAN></H4><P><B>MdlC biobrick</B>: After multiple clonings and <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformations</A>, we were finally able to send purified plasmid samples for sequencing.<B>MdlB biobrick</B>: We kept optimizing portocols.<B>HmaS biobrick</B>: We prepared starters according to the protocol. After the induced production of the protein, we ran a several western blots to check the production rates.<B>Chitinase biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates an ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we prepared starters from the confirmed positive colonies.</P><H4><SPAN class="mw-headline" id="Day_3_9">Day 3</SPAN></H4><P><B>Chitinase biobrick</B>: We purified the plasmid and sent samples for sequencing.</P><H4><SPAN class="mw-headline" id="Day_4_9">Day 4</SPAN></H4><P><B>Chitinase biobrick</B>: Werecovered the sequecning results. We have a full sequence.</P><P><DETAILS><SUMMARY>WEEK 11</SUMMARY></DETAILS></P><H4><SPAN class="mw-headline" id="Day_1_10">Day 1</SPAN></H4><P><B>Chitinase biobrick</B>: We started the protein production using the following protocol. (It took 3 days).<B>Methionine-γ-lyase biobrick</B>: We purified the produced protein.<B>HmaS biobrick</B>: We calibrated the <A href="/Team:Aix-Marseille/Protocols#HPLC_.28High_Performance_Liquid_Chromatography.29_test_on_..." title="Team:Aix-Marseille/Protocols">HPLC</A> system in preparations to measurements to test the protein functionality.<B>MdlB biobrick</B>: We recovered isolated colonies and ran an optimized colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A>. Afterward, we prepared starters from the positive colonies.<B>MdlC biobrick</B>: We recovered the sequencing results. We had negatives ones, so we started over trying to optimize the protocols.
</P><H4><SPAN class="mw-headline" id="Day_2_10">Day 2</SPAN></H4><P><B>Methionine-γ-lyase biobrick</B>: We stored the purified samples at -20°C. Afterward, we created a standard curve to the test the functionality of the protein.<B>HmaS biobrick</B>: We prepared starters to start the HPLC measurements.
</P><P>
You can see how these results have been exploited in the <A rel="nofollow" class="external text" href="https://2018.igem.org/Team:Aix-Marseille/Experiments">Experiment</A> tab.
</P><P><B>MdlB biobrick</B>: We purified the DNA and sent samples for sequencing.<B>MdlC biobrick</B>: We digested the gene sequence with respective enzymes, cloned it in a pSB1C3 plasmid, and transformed it into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>.</P><H4><SPAN class="mw-headline" id="Day_3_10">Day 3</SPAN></H4><P><B>MdlC biobrick</B>: We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates and ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies. Afterward, we prepared starters from the confirmed positive-colonies.
</P><H4><SPAN class="mw-headline" id="Day_4_10">Day 4</SPAN></H4><P><B>Chitinase biobrick</B>: After testing out different production conditions in DH5α strains, we stuck with the most optimal one to get to purify large quantities of the wanted protein.<B>MdlC biobrick</B>: We purified the DNA and sent samples for sequencing.</P><H4><SPAN class="mw-headline" id="Day_5_9">Day 5</SPAN></H4><P><B>Chitinase biobrick</B>: After testing out different production conditions in BL21 strains, we stuck with the most optimal one to get to purify large quantities of the wanted protein.<DETAILS><SUMMARY>WEEK 12</SUMMARY></DETAILS></P><H4><SPAN class="mw-headline" id="Day_1_11">Day 1</SPAN></H4><P><B>Methionine-γ-lyase biobrick :</B> We <A href="/Team:Aix-Marseille/Protocols#Methionine-y-lyase_activity_test_using_DTNB_protocol" title="Team:Aix-Marseille/Protocols">tested</A> with DTNB (5,5′-Dithiobis2-nitrobenzoic acid) on purified protein and absorbance measurements on the TECAN were performed. 
<B>MdlC biobrick : </B>We digested the gene sequence with the respective enzymes, cloned it into a pSB1C3 plasmid and extracted it from the gel.  
<B>Chitinase biobrick:</B> We purified and identified large quantities of the wanted protein.</P><H4><SPAN class="mw-headline" id="Day_2_11">Day 2</SPAN></H4><P><B>Methionine-γ-lyase biobrick :</B> We breaked cells with emulsiflex and made activity test on purified protein in triplicate with absorbance measurement on the TECAN.
<B>MdlC biobrick : </B>We transformed MdlC in pSB1C3 plasmid. 
<B>MdlB biobrick: </B>We cloned it in a pSB1C3 plasmid, and transformed it in <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>. 
<B>Chitinase biobrick: </B>We prepared activity test solutions (réaction buffer) and discolored gels of the day before. 
<B>Promotor-RBS-peptide signal sequence ompA biobrick : </B>We cloned RBS promotor and signal sequence in pSB1C3 plasmid and transformed them into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>. 
<B>Bar biobrick :</B> We cloned RBS promotor and signal sequence in PSB1C3 plasmid and transformed them into <A href="/Team:Aix-Marseille/Protocols#DH5.CE.B1_competent_cells_preparation" title="Team:Aix-Marseille/Protocols">DH5α competent cells</A>. 
</P><H4><SPAN class="mw-headline" id="Day_3_11">Day 3</SPAN></H4><P><B>Methionine-γ-lyase biobrick : </B>We made again activity test in triplicate with absorbance measurement on the TECAN. We start the analyse results. 
<B>MdlC biobrick :</B> We ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> with Slic1 and 4 on the supposedly positive colonies. Afterward, we prepared starters from the confirmed positive colonies.
<B>MdlB biobrick: </B>We ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> with Slic 1 and 4 but we had negatives ones, so we started over trying to optimize the protocols.
<B>Chitinase biobrick :</B> We prepared all activity test solutions. 
<B>Promotor-RBS-peptide signal sequence ompA biobrick :</B> We ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A>  but we had negatives ones, so we started over trying to optimize the protocols. 
<B>Bar biobrick : </B>We ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A>  but we had negatives ones, so we started over trying to optimize the protocols.
</P><H4><SPAN class="mw-headline" id="Day_4_11">Day 4</SPAN></H4><P><B>Methionine-γ-lyase biobrick : </B>On cell culture  of Methionine-γ-lyase needed for the <A href="/Team:Aix-Marseille/Protocols#Methionine-y-lyase_activity_test_using_DTNB_protocol" title="Team:Aix-Marseille/Protocols">activity test</A> (DTNB test and MBTH test), we induced at OD 0,7 with <A href="/Team:Aix-Marseille/Protocols#IPTG_induced_protein_production" title="Team:Aix-Marseille/Protocols">IPTG</A> and incubateur during 5h at 30°C. Measurement of OD was made after incubation and we freezed pellets at 20°C. We made then activity test with MTBH (3-Methyl-2-benzothiazolinone hydrazone hydrochloride hydrate) on purified protein and absorbance measurement was made on the TECAN but results were not good as expected. 
<B>MdlC biobrick : </B>We recovered the <A href="/Team:Aix-Marseille/Protocols#Transformation" title="Team:Aix-Marseille/Protocols">transformation</A> plates an ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> on the supposedly positive colonies of last week. Afterward, we prepared starters from the confirmed positive colonies.
<B>MdlB biobrick: </B>We ran a colony-<A href="/Team:Aix-Marseille/Protocols#Polymerase_Chain_Reaction_.28PCR.29" title="Team:Aix-Marseille/Protocols">PCR</A> of last week but we had negatives ones, so we started over to optimize the protocols. 
<B>Chitinase biobrick :</B> We made an etalon range to realised activity test at 30°C during one hour. 4466,58 ug/mL of chitinase (and contaminants) was obtained. 
</P><H4><SPAN class="mw-headline" id="Day_5_10">Day 5</SPAN></H4><P><B>Methionine-γ-lyase biobrick :</B> We made same activity test with MTBH and absorbance measurement was made on the TECAN but results were not good. 
<B>MdlC biobrick : </B>We purified the plasmid and sent samples for sequencing.
<B>Chitinase biobrick :</B> We made activity tests thanks to <A href="/Team:Aix-Marseille/Protocols#Schales.27_test" title="Team:Aix-Marseille/Protocols">Schales'procedure</A>, a colorimetric method for detecting reduced sugars released by chitin hydrolisis by chitinase.
</P><P><DETAILS><SUMMARY>WEEK 13</SUMMARY></DETAILS></P><P>During the last week in the laboratory, we gathered all the results and analyzed them, and prepared our constructs for shipment to the iGEM headquarters. Afterward, we packed away our equipment and cleaned-up our 3 months workspace.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>