<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UPO-Sevilla_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UPO-Sevilla/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header-row"><DIV class="container" style="height:75px"><DIV class="row" style="height:75px"><DIV class="span9"><DIV class="navbar  pull-right" style="font-size:80%"><DIV class="navbar-inner"><DIV class="nav-collapse collapse navbar-responsive-collapse"><UL class="nav"><LI><A href="https://2016.igem.org/Team:UPO-Sevilla">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Sevilla">Our city</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Notebook">Notebook</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Part_Collection">Part Collection</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Safety">Safety</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Dry Lab</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/HP/Silver">HP Silver</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/HP/Gold">HP Gold</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Engagement">Engagement</A></LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="span12"><DIV class="page-header"><H1 style="text-align:center;margin-left:100px">
				EXPERIMENTS
			</H1></DIV></DIV></DIV><H2 style="text-align:center"><B>Glycerol Module</B></H2><H3><B>Bacterial growth measurement</B></H3><P style="font-size:15px;margin-bottom:10px">This physiological assay was performed as described in Protocols <A href="https://2016.igem.org/Team:UPO-Sevilla/Protocols#17">(Bacterial growth measurement)</A>. By measuring bacterial growth in a medium with glycerol as the only carbon source, we can explain the glycerol consumption of the different strains.</P><P style="font-size:15px;margin-bottom:10px">Growth of bacteria expressing <I>glpF</I> (with the <B>BBa_K1973027</B> part inserted in the genome) was compared to that of the control, bacteria with an “empty” mini-Tn7, a Tn7 with no added information, in different conditions. It was expected that bacteria expressing <I>glpF</I> in presence of an activator would reach higher growth rates than the control.</P><P style="font-size:15px;margin-bottom:10px">The growth media prepared for this assay were minimal media with different carbon sources and different concentrations: 25 mM succinate, 25 mM glycerol, 5 mM glycerol, 1 mM glycerol and 0,2 mM glycerol. Each medium was prepared twice, and we added 20 μl of salycilate 1M (a final concentration of 2 mM) to one of each kind. Salycilate acts as an activator of the expression system. Minimal medium components are the following (for a 100 ml stock): 10 ml NaCl-P 10X, 1 ml NH<SUB>4</SUB>Cl 100 g/L, 1 ml vitamins solution 100X, 0,2 ml microelements solution 500X, the carbon source and water until reaching the final volume.</P><P style="font-size:15px;margin-bottom:10px">By performing this assay for the first time, we saw that both bacteria, that expressing glpF and the control, grew in a similar manner when using succinate as a sole carbon source. However, in no case exponential phase was reached when using glycerol. Then, we put the same microplate with the same bacteria in the fluorimeter to keep growing for another 23 hours. After that time, we saw bacteria had started growing in media with glycerol as a sole carbon source (Fig. 1).</P><P style="font-size:15px;margin-bottom:10px">We can stand up that modified bacteria reach a higher absorbance than control in a minimal media with glycerol 5 mM (data not shown). However, growth curves were not reliable as there was not a correlation between glycerol concentration and growth. Because of this, and because of the prolonged lag phase, we repeated the experiment.</P><P style="font-size:15px;margin-bottom:10px">The delay is due to a system developed in <I>P. putida</I> KT2442 that implies a prolonged lag phase when metabolizing glycerol. According to Escapa et al. (2013), this can be avoided by the addition of growth precursors, such as octanoic acid 1 mM. This acid, apart from improving the growth, would increase the useful molecule production.</P><P style="font-size:15px;margin-bottom:10px">We then repeated the experiment adding octanoic acid 1 mM as a growth precursor. In this case, we observed lag phase was substantially reduced, and bacteria reached exponential phase when growing in glycerol during the 23 hours the experiment lasts (Fig. 1). Results show bacteria expressing glpF reach a higher absorbance than control in media with glycerol (Fig. 2).</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 1. Growth graphs of experiments 1 and 2. Here we show growth curves in minimal media with salycilate and carbon source indicated in the legend. We did not add octanoic acid in experiment 1, where lag phase lasts between 35 and 36 hours. We did add octanoate in experiment 2, where lag phase is reduced to less than 10 hours.</P><P style="font-size:15px;margin-bottom:10px">Experiment 2 results show genetically modified bacteria expressing <I>glpF</I> grow much more than wild type in media with glycerol. This behavior is observed in all media with glycerol at different concentrations of this substrate (Fig. 2). This way, we conclude that the expression of the gene encoding for the glycerol transporter of the inner membrane is enough to increase consumption of this molecule, as modeling studies predicted.</P><P style="font-size:15px;margin-bottom:10px">In both experiments 2 and 3 (data not shown), we observe that wild type growth decreases as glycerol concentration increases. However, bacteria expressing <I>glpF</I> grow more as glycerol concentration increases (Fig. 2). GlpF seems to avoid the “damaging” effect of glycerol in bacterial growth. This means an advantage on an industrial scale, as modified bacteria would consume a higher concentration of glycerol than wild type, that is, more glycerol in less time.</P><P style="font-size:15px;margin-bottom:10px">Moreover, these results show growth in media with salycilate don’t differ significantly from that in media without it. This means basal expression of <I>glpF</I> is sufficient to increase growth in presence of glycerol.</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 2. Graphs showing bacterial growth in media with glycerol as a sole carbon source. Genetically modified bacteria reach a higher absorbance than wild type in both media with and without salycilate. Salycilate effect seems to not be significant for the increase in glycerol consumption.</P><H3><B>Biofilm curves in glycerol</B></H3><P style="font-size:15px;margin-bottom:10px">We also performed a second physiological assay involving the formation of biofilms of <I>P. putida</I> KT2442 expressing glpF. We wanted to study the effects of expressing this gene on the formation of a biofilm in a medium with glycerol as a sole carbon source. To that end we made a dilution series-based growth curves as indicated on Protocols <A href="https://2016.igem.org/Team:UPO-Sevilla/Protocols#12">(Dilution series-based growth curves)</A>, comparing our strain containing the BBa_K1973027 part and the wild type.</P><P style="font-size:15px;margin-bottom:10px">By measuring absorbance, we studied planktonic growth and biofilm of both strains in minimal media with glycerol 25 mM as a sole carbon source (composition indicated in Bacterial growth measurement) with and without salycilate.</P><P style="font-size:15px;margin-bottom:10px">We represented all data in graphs and studied the behavior of these bacteria. We can see that bacteria expressing <I>glpF</I> under the <I>nahR</I>-P<I>sal</I> expression system reach higher growth rates than wild type, as we saw in the previous assay (Bacterial growth measurement) (Fig. 3). This way, we have demonstrated this behavior through two different assays. Regarding biofilm formation, we observe wild type bacteria seem to reach higher absorbance levels at some points and at the end of the assay. However, both strains show a similar conduct (Fig. 3).</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 3. Graph showing planktonic growth and biofilm of modified bacteria expressing <I>glpF</I> and wild type in minimal media with glycerol as sole carbon source and salycilate. We can observe planktonic growth of modified bacteria reach higher absorbance levels than wild type.</P><P style="font-size:15px;margin-bottom:10px">Same results are obtained in media without salycilate (data not shown). As indicated in the previous assay, salycilate addition is not necessary to improve glycerol assimilation.</P><P style="font-size:15px;margin-bottom:10px">This assay shows us biofilm formation is similar in modified bacteria and wild type when growing in minimal media with glycerol. While planktonic growth is improved in modified bacteria, biofilm formation does not show significant differences.</P><H3><B>Tables</B></H3><P style="font-size:12px;margin-bottom:20px;text-align:center">Table 1. Oligonucleotides used in the mutagenic PCR of <I>glpF</I>. Here we represent the name of the oligonucleotides, their sequences and the number of nucleotides of each one. The nucleotides that are written in small letter do not hybridize with the gene sequence (prefix, suffix and mutations).</P><P style="font-size:12px;margin-bottom:20px;margin-top:20px;text-align:center">Table 2. All plasmids generated and needed for this module.</P><P style="font-size:12px;margin-bottom:20px;margin-top:20px;text-align:center">Table 3. Oligonucleotides used for sequencing the gene in pSB1K3</P><P style="font-size:12px;margin-bottom:20px;margin-top:20px;text-align:center">Table 4. Oligonucleotides used for verifying the insertion of the constructions in the genome of <I>P. putida</I> KT2442</P><H2 style="text-align:center"><B>Biofilm Module (LapG and expression system <I>xylS2</I>/P<I>m</I>)</B></H2><H3><B>Analyze of the biofilm of the strains with the <I>nahR</I>-P<I>sal</I> expression system</B></H3><P style="font-size:15px;margin-bottom:10px">The strains used in these experiments were generated by electroporation and transposition, as described in Protocols <A href="https://2016.igem.org/Team:UPO-Sevilla/Protocols#7">(Electroporation and transposition)</A>, and they are KT2442-TpMRB128 (<I>nahR</I>-P<I>sal-lapG</I>), KT2442-TpMRB133 (<I>nahR</I>-P<I>sal-nasF-lapG</I>), KT2442 lapG<SUP>-</SUP>-TpMRB128 and KT2442 lapG<SUP>-</SUP>-TpMRB133. We did serial dilution-based growth curves of these strains to test the activity of <I>lapG</I> under the <I>nahR</I>-P<I>sal</I> and <I>nahR</I>-P<I>sal-nasF</I> expression system in LB and LB+salicylate 2 mM, as described in Protocols <A href="https://2016.igem.org/Team:UPO-Sevilla/Protocols#12">(Dilution series-based growth curves)</A>. TpMRB128 is the part <B>BBa_K1973006</B> and TpMRB133 is the part <B>BBa_K1973007</B>.</P><P style="font-size:15px;margin-bottom:10px">In the experimental design we have made 1-3  replays for each condition. We consider a condition as each combination of the strains, the presence of salicylate and the kind of expression system. We have taken 10 measurements corresponding to different dilutions  in each condition. And we made 8 lectures of each dilution per biological replica. The data are  represented like the average ± standard error. We have made a t-test check for the statistic analyze and we consider significant the values of p&lt;0.05. It has been represented with: * for p&lt;0.05, ** for p&lt;0.01 and *** for p&lt;0.001.</P><P style="font-size:15px;margin-bottom:10px">The growth curves test of the strains KT2442-TpMRB128 and KT2442 lapG<SUP>-</SUP>-TpMRB128 will reveal if the <I>nahR</I>-P<I>sal</I> expression system express correctly the <I>lapG</I> gene at the induction conditions with and without salicylate (Figure 4). After the experiments, we observed that both strains have a similar planktonic growth curves to the control and between them in LB and LB+salicylate. As regards the biofilm growth, KT2442 <I>lapG</I><SUP>-</SUP> produce 2-3 times more biofilm than the wild type and do not disperse the biofilm due to the fact that KT2442 <I>lapG</I><SUP>-</SUP> is a biofilm superproducer. The complementation of <I>lapG</I> in the <I>lapG</I> mutant under the induced expression system causes a delay of the biofilm formation, a reduction of the maximum amount of biofilm and the biofilm disperses. The expression of LapG in the wild type does not cause a big change in the behavior of the biofilm development, it only produces a little reduction of the amount of biofilm. </P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 4. Serial dilution-based growth curves of the expression system <I>nahR</I>-P<I>sal-lapG</I> (20 hours). The image represents the growth curves of the strains wild type in LB (A), wild type in LB+salicylate 2 mM (B), <I>lapG</I><SUP>-</SUP> in LB (C) and <I>lapG</I><SUP>-</SUP> in LB+salicylate 2mM (D). We assay the strains wild type Tn7 Ø and <I>lapG</I><SUP>-</SUP> Tn7 Ø concurrently to compare the induction effect of the expression system. Each blot represent the average of 24 data.</P><P style="font-size:15px;margin-bottom:10px">We observe a possible biofilm dispersal when we induce the expression system in lapG mutant, but we can't be sure of that. So, we will repeat the experiment with a little modification, we will incubate the plates for 30 hours (Figure 5) instead of 20 hours  to observe if the biofilm disperses at all or keeps high like the <I>lapG</I> mutant. We only did one replica of this experiment, so the results of the curves are not clear enough. But the biofilm growth tendency of the strain, when is induced by salicylate, shows that the expression of LapG protein promotes the biofilm dispersal like the wild type. The strains without inducing have grown in a different rate, although the tendency of both curves are similar. We suppose that their behavior is similar and the induction by salicylate does not affect to the biofilm development.</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 5. Serial dilution-based growth curves of the expression system <I>nahR</I>-P<I>sal-nasF-lapG</I> (30 hours). The image represents the growth curves of the strains <I>lapG</I><SUP>-</SUP> in LB (A) and <I>lapG</I><SUP>-</SUP> in LB+salicylate 2 mM (B). We assay the strain <I>lapG</I><SUP>-</SUP> Tn7 Ø concurrently to compare the induction effect of the expression system. Each blot represent the average of 8 data.</P><P style="font-size:15px;margin-bottom:10px">At the same way, we test the <I>nahR</I>-P<I>sal-nasF</I> expression system in the strains KT2442-TpMRB133 and KT2442 <I>lapG</I><SUP>-</SUP>-TpMRB133 (Figure 6). In this case, the planktonic growth of the both strains keep a similar pattern to the control and between them. As same as before, the strain KT2442 generate 2-3 less time than the <I>lapG</I> mutant. As well as, the biofilm curve of induced KT2442 <I>lapG</I><SUP>-</SUP>-TpMRB133 keeps the same tendency that the control strain, so this strain have the same biofilm generation in LB and LB+salicylate.</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 6. Serial dilution-based growth curves of the expression system <I>nahR</I>-P<I>sal-nasF-lapG</I> (20 hours). The image represents the growth curves of the strains wild type in LB (A), wild type in LB+salicylate 2mM (B), <I>lapG</I><SUP>-</SUP> in LB (C) and <I>lapG</I><SUP>-</SUP> in LB+salicylate 2 mM (D). We assay the strains wild type Tn7 Ø and <I>lapG</I><SUP>-</SUP> Tn7 Ø concurrently to compare the induction effect of the expression system. Each blot represent the average of 24 data.</P><H3><B>Study of the activity of the mutant P<I>m</I> promoters by a beta-galactosidase activity assay</B></H3><P style="font-size:15px;margin-bottom:10px">First of all, we introduce the three variants of P<I>m</I> (original P<I>m</I>, P<I>m</I>1, P<I>m</I>2 and P<I>m</I>3) with the fusion protein <I>lacZ-gfp</I>mut3 by triparental mating, as described in Protocols <A href="https://2016.igem.org/Team:UPO-Sevilla/Protocols#8">(Triparental mating)</A>, in the strains KT2442-TpMRB170 (<I>nahR</I>-P<I>sal-xylS2</I>) and KT2442-TpMRB159 (<I>nahR</I>-P<I>sal-nasF-xylS2</I>) that were generated by electroporation and transposition, as described in Protocols <A href="https://2016.igem.org/Team:UPO-Sevilla/Protocols#7">(Electroporation and transposition)</A>. We test the P<I>m</I> promoters in LB and LB+salicylate 2 mM and our control is the plasmid with the fusion protein <I>lacZ-gfp</I>mut3 without promoter (pMRB1). We create this promoter by mutagenic PCR,as described in Protocols <A href="https://2016.igem.org/Team:UPO-Sevilla/Protocols#11">(Site-directed mutagenesis using overlap extension PCR)</A>, so we do not know the effect of this mutation in a promoter and it could convert in a constitutive or inducible by other molecule promoter, rise or reduce its basal or induce expression...  TpMRB170 is the part <B>BBa_K1973024</B>, TpMRB159 is the part <B>BBa_K1973021</B>, P<I>m</I>1 is the part <B>BBa_K1973013</B>, P<I>m</I>2 is the part <B>BBa_K1973014</B> and P<I>m</I>3 is the part <B>BBa_K1973015</B>.</P><P style="font-size:15px;margin-bottom:10px">We test the 8 constructions using the beta-galactosidase activity assay in 2 different conditions (LB and LB+salicylate 2 mM) (Figure 7). We did 4 replicates of the strains with the transposon <I>nahR</I>-P<I>sal-xylS2</I> (BBa_K1973024) and 2 replicates of the strains with the transposon <I>nahR</I>-P<I>sal-nasF-xylS2</I> (BBa_K1973021). There is a significant difference between the expression of the promoters when the regulation module (expression of <I>xylS2</I>) is induced and when it is not induced. As well as, the strains with <I>nahR</I>-P<I>sal-xylS2</I> have more expression when it is induced and when it is not induced in comparison with the strains with the attenuator.</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 7. Beta-galactosidase assay of P<I>m</I> promoters. The image represents the beta-galactosidase activity of the different P<I>m</I> promoters in the strains expressing <I>xylS2</I> without <I>nasF</I> (A) and with the attenuator (B). We assay the strains KT2442-TpMRB170/pMRB1  and KT2442-TpMRB159/pMRB1 concurrently to compare the induction effect of the expression promoter.</P><P style="font-size:15px;margin-bottom:10px">The three variants of the P<I>m</I> promoter work satisfactory and the P<I>m</I>1 promoter (BBa_K1973013) is the variant that has the mayor expression in the system without the attenuator when it is induced. On the other hand,  the P<I>m</I>3 promoter (BBa_K1973915) is the variant that has the mayor expression in the system with the attenuator when it is induced. We have had problems with the expression of the original P<I>m</I> promoter and it seems like this promoter does not work. So, we could not have made a comparison between the activity of the original P<I>m</I> and its variants, but our mutant promoters  work perfectly. </P><H2 style="text-align:center"><B>Biofilm Module (YhjH and PleD*)</B></H2><H3><B>Construction of Superproducer Strains of PleD* and YhjH</B></H3><P style="font-size:15px;margin-bottom:10px">The enzyme PleD* is a derivative of the diguanylate cyclase PleD from bacteria <I>Caulobacter crescentus</I>. The original protein presents some domains that inhibit the diguanylate cyclase activity when phosphorylated, allowing to regulate this activity depending on the intracellular concentration of c-diGMP. In the case of the mutant version - PleD* - these phosphorylation domains have been modified to prevent this inhibition and therefore, the physiological regulation of the enzyme by the bacteria, allowing a high synthesis of c-diGMP (Romero-Jiménez et al., 2015). On the other hand, YhjH is a phosphodiesterase that originally cames from <I>Escherichia coli</I> (Christensen et al., 2013). The usage of this enzyme to diminish the c-diGMP levels has been proved previously on <I>Pseudomonas putida</I> (Jimenez-Fernandez et al., 2015).</P><P style="font-size:15px;margin-bottom:10px">Having as an objective the implementation of a system that allows us to control externally the synthesis and turn-down of c-diGMP, we had cloned both genes <I>pleD*</I> and <I>yhjH</I> separately under the control of inducible expression system <I>nahR</I>-P<I>sal</I>, which is regulated by salicylate (parts <B>BBa_K1973011</B> and <B>BBa_K1973004</B>, respectively). This allows us to have high levels of expression under inducing conditions and low ones in non-inducing conditions. To obtain lower expression levels, we used a variant of our constructs that contained the nasF attenuator, cloned between the promoter and the gene of interest (parts <B>BBa_K1973018</B> and <B>BBa_K1973008</B>, respectively). This sequence limits the transcription, diminishing the levels of both basal and inducible expression, limiting toxicity in the case the product of our gene of interest is deleterious (Cebolla et al., 2001). These constructs were cloned in miniTn7BB-Gm device, allowing its stable integration in attTn7 site in <I>P. putida</I> and other Gram-negatives genome (parts <B>BBa_K1973023</B>, <B>BBa_K1973022</B>, <B>BBa_K1973012</B> and <B>BBa_K1973010</B>, respectively). For this, we made use the transposition functions of the miniTn7 device using the helping vector pTNS2. We therefore had four new strains that contained the new functions provided by these genetic pieces (figure 8).</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 8. Integrated constructions by means of the miniTn7BB-Gm device. Example of how the constructs would be inserted in the attTn7 site in <I>P.putida</I> genome making use of the miniTn7 transposon. The other constructs followed the same scheme, both with and without <I>nasF</I> attenuator, for both <I>pleD*</I> and <I>yhjH</I> genes.</P><H3><B>Characterization of the effect of the production of PleD* and YhjH on c-diGMP levels</B></H3><P style="font-size:15px;margin-bottom:10px">It has been demonstrated that in vivo synthesis of high levels of diguanylate cyclases (DGC) or phosphodiesterases (PDE) has as a result an increment or diminishment of the intracellular concentrations for c-diGMP, respectively (Christensen et al., 2015; Romero-Jiménez et al., 2015). In order to determine whether the production of both enzymes has any impact on the intracellular levels of c-diGMP, the plasmid pCdrA::<I>gfp<SUP>C</SUP></I> was introduced in all the strains containing DGC or PDE constructs. This plasmid counts with a GFP fusion to the P<I>cdrA</I> promoter from <I>Pseudomonas aeruginosa</I>, which is inducible by c-diGMP. The GFP used for this fusion is <I>gfp</I>mut3, containing a stable derivative of this protein (Rybtke et al., 2012). The strains transformed with pCdrA::<I>gfp<SUP>C</SUP></I> plasmid were grown in 10% diluted LB plates at 30ºC and shaking, both in presence and absence of salicylate, in an absorbance/fluorescence lector, allowing to measure 600 nm absorbance and fluorescence alongside the curve (Figure 9). From the experimental data, a GFP differential accumulation rate was calculated during growth exponential phase as it is described in the protocols. This allowed to compare the data objectively between the different strains assayed (Figure 10).</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 9. Absorbance and fluorimetry data. Graphic representation of the bacterial growth (dots) and accumulated fluorescence (squares) produced by P<I>cdrA</I> promoter fusion with stable GFP protein in all the assayed strains in both induction (blue) and non-induction (orange) conditions. Fluorescence levels are similar to those of the wild type in all situations except for the strain producing PleD* without attenuator in induction conditions. There were growing differences between the wild type and strains producing both PleD* and YhjH without the attenuator under induction conditions.</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 10. Differential accumulation rate. Graph in which are shown the data relating the differential accumulation rate with their error bars. The producing strains that had the nasF attenuator did present similar fluorescence levels as the wild type. The YhjH producing strain has a diminishment in the levels of fluorescence up to three times in induction conditions. The PleD* producing strain has an increase of up to 15 times in fluorescence over the wild type in induction conditions.</P><P style="font-size:15px;margin-bottom:10px">The growth and fluorescence levels of all strains was similar in the absence of salicylate, with the only exception of a slight decrease (2 times) in the fluorescence levels of Yhjh-producing strain without the <I>nasF</I> attenuator. These results point out that the <I>nahR</I>-P<I>sal</I> expression system does not allow the production of physiologically significant levels either PleD* or YhjH in non-inducing conditions, with independence of the presence of nasF attenuator. The addition of salicylate did not affect in a significant way to the growth of any strain, with the exception of the PleD* producing strain without the attenuating system. The growth rate of this strain was severely diminished. The addition of salicylate did not affect in a significant way the expression of P<I>cdrA-gfp</I>mut3 fusion in the wild type or any of the strains containing the attenuator, whereas the levels of fluorescence were significantly higher in the strain expressing <I>pleD*</I> without attenuator. Furthermore, the fluorescence levels were significantly lower in the strain expressing <I>yhjH</I> without attenuator. As a summary, our results point out that the induction with salicylate of the synthesis of PleD* increases the intracellular concentration of c-diGMP, whereas the synthesis of YhjH decreases the intracellular concentration of this second messenger.</P><H3><B>Characterization of EPS production effect when altering c-diGMP levels</B></H3><P style="font-size:15px;margin-bottom:10px">One of the most affected processes by the changes in the intracellular concentration of c-diGMP is the production of cellulose and other exopolysaccharides during biofilm formation (Monds $ O’Toole, 2009). The increasing of c-diGMP levels provokes an increase in the production of these EPS, which contribute to the maturation of the biofilm’s macrocolonies (Pérez-Mendoza et al., 2014; Romero-Jiménez et al., 2015). In order to study the changes performed in the production of EPS that were induced by the levels of c-diGMP, we made use of a Congo Red adsorption assay (a dying that binds cellulose and other types of EPS). For this, the strains to study were inoculated in solid-medium plates supplemented with this dying (Fig. 11), adding sodium salicylate whenever necessary in order to induce the expression of the constructs within the strains.</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 11. Congo Red assay. Congo Red assay in which the build Pseudomonas putida strains were tested, as well as the wild type strain and a wild type strain containing the empty miniTn7 device, kept as negative controls both in inducing (left) and non-inducing (right) conditions. The negative control strains kept a flat-colony, low-Congo-Red adsorption phenotype that was reproduced in all strains both in inducing and non-inducing conditions, except for the PleD* producing strain, which kept a rugose-colony, high-Congo-Red adsorption phenotype, with remarkable crests and high red coloration.</P><P style="font-size:15px;margin-bottom:10px">Both the wild type and the empty miniTn7 device strains produced flat colonies with no remarkable dying adsorption, what suggests that they do not produce high quantities of Congo-Red-adsorbing polysaccharides. This phenotype did replicate in all strains in non-inducing conditions. In presence of salicylate, all colonies had again the same phenotype, except for the colony corresponding to the <I>nahR</I>-P<I>sal-pleD*</I> construction, which had a rugose-colony phenotype with high crests and an intense red dying. These data suggest that the PleD* producing strain without the <I>nasF</I> attenuator produces high levels of EPS capable of adsorbing Congo Red, therefore the synthesis of them is regulated by c-diGMP in <I>P. putida</I>.</P><H3><B>Characterization of flagellar motility when altering c-diGMP levels</B></H3><P style="font-size:15px;margin-bottom:10px">Flagellar motility is another of the aspects in bacterial physiology that becomes affected by the alteration in c-diGMP levels, high c-diGMP levels meaning an inhibition of the motility by the repression of the genes in charge of both the synthesis and functioning of flagella (Pérez-Mendoza et al., 2015; Romero-Jiménez et al., 2015). In order to check whether an alteration of c-diGMP levels had an effect in motility in the studied strains, a motility assay in soft-agar plates, or swimming, was performed. For this, the strains were inoculated in semi-solid plates both in presence and absence of salicylate, afterwards evaluating their capacity to move from the inoculation point to form a motility halo (Fig. 12).</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 12. Swimming assay. Assay in which we can observe the swimming halos produced by the bacteria when swimming, both in inducing (right) and non-inducing (left) conditions. The wild type strain produces a halo that is similar to the ones produced by the rest of strains both in non-inducing and inducing conditions, except for the PleD* producing strain without the <I>nasF</I> attenuator, which shows a non-motile phenotype in inducing conditions.</P><P style="font-size:15px;margin-bottom:10px">All the strains assayed did produce similar halos to that produced by the wild type, both in inducing and non-inducing conditions. The only exception was the PleD* producing strain without the <I>nasF</I> attenuator, which produced a halo similar to that of the wild type in the non-induced plate but did not show any halo at all in the induced plate. This strongly suggests the levels of c-diGMP produced by PleD* synthesis from the construction inhibit the synthesis or the functioning of flagella.</P><H3><B>Characterization of the formation and dispersal of the biofilm when altering c-diGMP levels</B></H3><P style="font-size:15px;margin-bottom:10px">c-diGMP is involved in the necessary signalling for both formation and dispersal of biofilm, promoting its formation whenever the levels are high enough and provoking its dispersal whenever the intracellular concentrations are low (Monds &amp; O’Toole, 2009).</P><P style="font-size:15px;margin-bottom:10px">In order to monitor both planktonic and biofilm growth of the studied strains, dilution-based growth curves were performed. This method uses a series of dilutions in a multi-well plate to recapitulate the temporal evolution of both planktonic and biofilm populations (López-Sánchez et al., 2013) (Fig. 13). </P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 13. Dilution-based growth and biofilm curves. Graphical representation of both growth and biofilm formation in the different studied strains, always compared with the wild type strain carrying the miniTn7 device with no genetic construction in it. Dashed lines correspond to growth whereas solid lines refer to the biofilm development. Orange dots refer to wild type strain, whereas blue dots refer to the different producing strains. Strains hosting the attenuator did not have differences towards the wild type. YhjH producing strain shows a delay in biofilm formation in inducing conditions. PleD* producing strain shows an increase in biofilm formation and its non-dispersal.</P><P style="font-size:15px;margin-bottom:10px">The pattern both in planktonic growth and biofilm formation from the wild type was essentially as described in (López-Sánchez et al., 2013). This strain showed an increase in biofilm mass coincident with the exponential phase of planktonic growth, and after getting a maximal in late exponential phase, biofilm dispersal was observed during stationary phase. Strains hosting attenuator <I>nasF</I> showed the same pattern as the wild type, both in inducing and non-inducing conditions, either in growth or biofilm development curves. Strains producing YhjH and PleD* without the attenuator showed this same pattern in non-inducing conditions. However, in presence of salicylate, the PleD* producing strain without the attenuator system showed a slower growth accompanied by 7 times higher levels of biofilm than those of the wild type strain. In the other hand, Yhjh producing strain without the attenuator system showed a normal planktonic growth, however showing a great delay on biofilm formation. These experiments suggest that the expression of a diguanylate cyclase, which produces high c-diGMP levels, provokes a biofilm super formation and its non-dispersal, as well as a slower growth, whereas low levels of c-diGMP provoked by the expression of a phosphodiesterase provoke a delay in biofilm formation.</P><P style="font-size:15px;margin-bottom:10px">Biofilm-producing bacteria are not only capable of preforming it over solid surfaces, but also can they produce it in the liquid-gas interphase of liquid cultures. This type of biofilm is named as pellicle. In order to characterize the effect on this phenotype of PleD* and YhjH super production, a pellicle assay was performed to all the studied strains both in presence and absence of salicylate (Fig. 14).</P><P style="font-size:15px;margin-bottom:10px">As previously described (Jimenez-Fernandez et al., 2015), wild type strain showed a low-pellicle formation phenotype independently from the inductor presence. Similarly, there was no pellicle formation observed in neither strains carrying the nasF attenuator, nor was it in the YhjH producing strain without the attenuator. In contrast, the PleD* producing strain without the attenuator culture that had salicylate showed a high biomass quantity forming a pellicle. In addition, the culture was much less turbid than the others, and a sediment of cell aggregate was observed at the bottom of the tube. These results suggest that the c-diGMP levels obtained with the construction hosted by the PleD* producing strain without nasF promote pellicle formation and cell-cell interaction for aggregate formation.</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Figure 14. Pellicle formation assay. Pellicle assay with no inductor (above) and with inductor (down) of the labelled strains. All strains show a low-pellicle formation phenotype except for PleD* producing strain without nasF in inducing conditions, which shows both a wide pellicle formation and cell aggregates at the bottom of the tube. Black arrows point the formation of pellicle over the walls of the tube and decanting of cell aggregates at the bottom of it.</P><P style="font-size:15px;margin-bottom:10px"><I>P. putida</I> is able to adhere to a variety of surfaces permanently on a short-time scale – a few minutes – and proliferate on them to produce microcolonies. In order to determine the effect on this capacity by the production of PleD* and YhjH, adhesion assays were performed to the producing strains by means of phase contrast microscopy in a multi-well plate, both in presence and absence of salicylate. Results are shown in figure 15.</P><P style="font-size:12px;margin-bottom:20px;text-align:center">Fig. 15. Adhesion assay. Microscopy images of YhjH and PleD* strains without transcription attenuator <I>nasF</I> with and without inductor. Wild type shows a microcolonies phenotype that fills the whole field. PleD* producer strain shows bacterial aggregates that are not uniformly distributed in induction conditions with salicylate. YhjH producer strain shows much less adhered cells, finding a lot of them in suspension.</P><P style="font-size:15px;margin-bottom:10px">In absence of salicylate all strains showed an adhesion and microcolonies formation phenotype similar to that of wild type. Producer strains carrying the attenuator showed an identical phenotype to that of wild type in presence of inductor (data not shown). However, in induction conditions we observe important effects in PleD* and YhjH producer strains phenotypes without the attenuator <I>nasF</I>. PleD* producer strain cells were mainly forming aggregates, and planktonic cells were not seen. Nevertheless, YhjH producer strain showed a lower number of bacteria adhered to the surface forming microcolonies in comparison with the wild type, and a lot of planktonic cells. These results indicate that, in absence of nasF, PleD* synthesis induction causes the formation of aggregates that adhere steadily to the surface whereas YhjH synthesis induction inhibits stable adhesion and promotes transition to a planktonic state.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>