<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UPO-Sevilla_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UPO-Sevilla/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header-row"><DIV class="container" style="height:75px"><DIV class="row" style="height:75px"><DIV class="span9"><DIV class="navbar  pull-right" style="font-size:80%"><DIV class="navbar-inner"><DIV class="nav-collapse collapse navbar-responsive-collapse"><UL class="nav"><LI><A href="https://2016.igem.org/Team:UPO-Sevilla">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Sevilla">Our city</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Notebook">Notebook</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Part_Collection">Part Collection</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Safety">Safety</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Dry Lab</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/HP/Silver">HP Silver</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/HP/Gold">HP Gold</A></LI><LI><A href="https://2016.igem.org/Team:UPO-Sevilla/Engagement">Engagement</A></LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="span12"><DIV class="page-header"><H1 style="text-align:center;margin-left:100px">
				DEMONSTRATE
			</H1></DIV></DIV></DIV><H2 style="text-align:center"><B>Glycerol Module</B></H2><H3><B>DISCUSSION</B></H3><P style="font-size:15px;margin-bottom:10px">Genetic modification performed in this module consists on the introduction of the glpF gene under an adjustable expression system. This gene codifies for an inner membrane porin that acts as a facilitator of glycerol transport to cytoplasm. This gene was obtained from the genome of <I>P. aeruginosa</I> PAO1, as this bacterium codifies for a protein with a higher maximum transport speed than that of the host bacterium. The genetic modification was performed with the help of the mini-Tn7 system, which was developed in a BioBrick format by members of the previous iGEM Team of our University in the year 2011<SUP>1</SUP>. Mini-Tn7 system has a few advantages compared to other insertion systems. This system can adapt to a great variety of host bacteria, has a high efficiency and it permits insertion in a specific locus of the genome with a specific orientation, avoiding affecting other regions of the bacterial chromosome. Moreover, mini-Tn7 does not need to be continuously selected with antibiotics, as insertion is maintained for at least 100 generations in absence of antibiotics<SUP>2</SUP>.</P><P style="font-size:15px;margin-bottom:10px">Functional assays show that bacteria have a prolonged lag phase when growing in presence of glycerol as sole carbon source in normal conditions. This phase can be reduced by adding a supplement, as can be octanoic acid. It constitutes a highly complex regulation system and role of octanoate is not still known. However, it could act by activating the global energetic system of the bacterium or by generating specific metabolites or cofactors for glycerol assimilation<SUP>3</SUP>.</P><P style="font-size:15px;margin-bottom:10px"><I>P. putida</I> populations growing in glycerol show a phenotypic variation due to a phenomenon called persistence. Because of this, a fraction of the population does not grow. Persistence protects against external factors, such as antibiotics, that affect growing bacteria, and it also permits the search for alternative carbon sources different from glycerol. This regulation is controlled by <I>glpR</I> gene, which codifies for a repressor of the <I>glpF, glpK</I> and <I>glpD</I> genes, involved in glycerol assimilation. The prolonged lag phase is due to the action of this repressor, as growth in glycerol is only possible when genes previously mentioned are expressed. This way, a mutant that does not express this repressor shows a reduction in the lag phase and a homogeneous growth of all population<SUP>3,4</SUP>. GlpR repressor is controlled by glycerol-3-P (G3P) levels in such a way that a genetically modified bacterium that produces higher levels of this metabolite has a similar phenotype to Δ<I>glpR</I> mutant<SUP>4</SUP>. Therefore both suppression of the repressor or overexpression of G3P can help by increasing even more glycerol consumption.</P><P style="font-size:15px;margin-bottom:10px">As modeling studies predicted, glycerol transport from periplasm to cytoplasm supposes a bottleneck that has been avoided by <I>glpF</I> expression. We have demonstrated that the inner membrane porin synthesis permits a higher growth of the microorganism in presence of glycerol as a sole carbon source than that of the wild type.</P><P style="font-size:15px;margin-bottom:10px">Functional assays results also indicate that wild type growth is reduced as glycerol concentration increases. It is still not known the cause for this phenomenon, but it is thought to be due to a reduction of the water potential of the surroundings. This causes an osmotic stress that reduces cellular growth because it increases breathing rates and utilizes cell energy to keep turgidity<SUP>5</SUP>. Nevertheless, this “injurious” effect of glycerol is not observed in the case of bacteria expressing GlpF protein, which show a higher growth as glycerol concentration increases. This indicates expression of <I>glpF</I> somehow avoids this effect, probably by favoring its introduction and assimilation and helping the reduction of the water potential of the surroundings.</P><P style="font-size:15px;margin-bottom:10px">Regarding biofilm formation, we cannot see meaningful differences between modified bacteria and wild type when growing in minimal media with glycerol. This indicates <I>glpF</I> expression does not affect biofilm formation, but does improve bacterial growth.</P><H4><B>References</B></H4><OL style="list-style:decimal inside"><LI style="font-size:15px;margin-bottom:10px">http://parts.igem.org/Genome_Integration</LI><LI style="font-size:15px;margin-bottom:10px">Choi, K.-H., &amp; Schweizer, H. P. (2006). mini-Tn7 insertion in bacteria with single attTn7 sites: example <I>Pseudomonas aeruginosa</I>. Nature Protocols, 1(1), 153–161. http://doi.org/10.1038/nprot.2006.24</LI><LI style="font-size:15px;margin-bottom:10px">Escapa, I. F., del Cerro, C., García, J. L., &amp; Prieto, M. A. (2013). The role of GlpR repressor in <I>Pseudomonas putida</I> KT2440 growth and PHA production from glycerol. Environmental Microbiology, 15(1), 93–110. http://doi.org/10.1111/j.1462-2920.2012.02790.x</LI><LI style="font-size:15px;margin-bottom:10px">Schweizer, H. P., &amp; Po, C. (1996). Regulation of glycerol metabolism in <I>Pseudomonas aeruginosa</I>: Characterization of the glpR repressor gene. Journal of Bacteriology, 178(17), 5215–5221.</LI><LI style="font-size:15px;margin-bottom:10px">Lambertsen, L., Sternberg, C., &amp; Molin, S. (2004). Mini-Tn7 transposons for sitespecific tagging of bacteria with fluorescent proteins. Environmental Microbiology, 6(7), 726–732. http://doi.org/10.1111/j.1462-2920.2004.00605.x</LI></OL><H3><B>THEY DIDN’T GO WELL…</B></H3><P style="font-size:15px;margin-bottom:10px">We started trying to clone oprB gene, which codifies for a transporter of the outer membrane. It was hard to complete the mutagenic PCR (really hard!!!) and we did a first attempt to clone it into pSB1K3 plasmid. Sadly, it didn’t go well. However, Metabolic pathway modeling software analysis told us the bottleneck in the glycerol assimilation is <I>glpF</I>, and its expression is enough to improve it in high rates. As it was no necessary to express <I>oprB</I>, we focused on cloning, integrating and expressing <I>glpF</I> and we didn’t keep on trying to clone <I>oprB</I>.</P><P style="font-size:15px;margin-bottom:10px">Also, we did amplified <I>glpK</I> gene by PCR (which codes for a enzyme involved in glycerol catabolism) and tried to clone it into pSB1K3. This first attempt didn’t go well, but, as in the first case, we didn’t tried it anymore as we saw it is not necessary.</P><H2 style="text-align:center"><B>Biofilm Module (LapG and expression system <I>xylS2</I>/P<I>m</I>)</B></H2><H3><B>DISCUSSION</B></H3><P style="font-size:15px;margin-bottom:10px">The protease LapG cleaves the adhesin LapA (main protein of the biofilm adhesion) what reduce the numbers of the adhesion interactions and then the amount of biofilm is reduced, too. The expression of LapG under the expression system <I>nahR</I>-P<I>sal</I> (BBa_K1973006) in the presence of salicylate influence lightly in the biofilm development because it is only reduced the maximum amount of biofilm.  The activity of the added LapG in KT2442 is not counteracted by the inner activity of LapD, that degrades LapG, that causes a lightly reduction of the biofilm. However,  the increase of the LapG concentration in KT2442 <I>lapG</I><SUP>-</SUP> produces a modification in the biofilm formation and dispersal what cause a delay of the biofilm formation due to the degradation of LapA by LapG, and a reduction of the maximum and minimum amount of biofilm. The biofilm dispersal of <I>lapG</I> mutant does not produce, but when we complemented the mutation with the expression of LapG, the strain disperse like the wild type. Even so, we could not complement the LapG deficiency in the lapG mutant. This might be because the expression under <I>nahR</I>-P<I>sal</I> is not regulated at the same way as the wild type, the expression level of our system is constant during all the biofilm development, but the level of LapG in the wild type vary during this process. In both strains the basal expression of the expression system is not enough to cause a change in the biofilm development. To sum up this section we have achieved a strain that is able to disperse the biofilm only when we induce our expression system.</P><P style="font-size:15px;margin-bottom:10px">The expression system <I>nahR</I>-P<I>sal-nasF-lapG</I> (BBa_K1973007) has demonstrated that it works correctly. The planktonic and biofilm curves of KT2442 <I>lapG</I><SUP>-</SUP>-TpMRB133 have the same values when we test them in both conditions: adding or not salicylate. That means that the <I>lapG</I> gen does not express, so the <I>nasF</I> attenuator achieve its function, blocking the expression of any gen downstream the P<I>sal</I> promoter. Expressing the transcription antiterminator protein NasR under the P<I>sal</I> promoter would be the next experiment to finish the study of this system expression behavior . After doing this experiment, we will get a expression system of <I>lapG</I> with minimum basal expression levels and when we induce the promoter the expression will raise hugely. </P><P style="font-size:15px;margin-bottom:10px">The beta-galactosidase assay permits us to probe the functionality of the mutant P<I>m</I> promoters and XylS2 protein. The <I>xylS2</I> gene is under the P<I>sal</I> or P<I>sal-nasF</I> promoter, so when we induce the expression system, it express this gene and XylS2 protein actives the P<I>m</I> promoter. The functionality of XylS2 is checked when the assay works because we express a GFP under the P<I>m</I> promoter. P<I>m</I>1 promoter (BBa_K1973013) is the one whose expression is higher in the strain with <I>nahR</I>-P<I>sal-xylS2</I> (BBa_K1973024) in the genome. Its expression have a huge difference at induce and no induce conditions, getting an activity of 6000 Miller units at no induce conditions and 47000 Miller units. The expression system with the attenuator <I>nasF</I> (BBa_K1973021) also works correctly because the expression of the fusion protein LacZ-GFPmut3 decreases with this system. The mutations in the promoter P<I>m</I> have not eliminated its expression, but the expression of the original promoter P<I>m</I> is similar to the control in the assay. This should be a fail in the introduction of the original promoter Pm in the plasmid because its natural activity is very high. To sum up, we have achieved creating the expression system xylS2/Pm in a Biobrick format and it works successfully. </P><H3><B>THEY DIDN’T GO WELL…</B></H3><P style="font-size:15px;margin-bottom:10px">We did not get testing the activity of the original promoter P<I>m</I> by the beta-galactosidase assay, so we do not know the specific effect of the mutation in this promoter. It is not a real problem to finish the project, but that would give an idea to us about how good or bad it is our mutant promoters.</P><H2 style="text-align:center"><B>Biofilm Module (YhjH and PleD*)</B></H2><H3><B>DISCUSSION</B></H3><P style="font-size:15px;margin-bottom:10px">The objective of this part of the project was to implement a genetic system that allowed to form and disperse the biofilm bacterial structure at will, making use of external signalling for this. To get this aim, we used an expression system inducible by salicylate, <I>nahR</I>-P<I>sal</I> (Cebolla et al., 2001), with the further option of using or not the <I>nasF</I> attenuator system to diminish the expression of each of the two assayed genes, <I>yhjH</I> and <I>pleD*</I>. The first one codifies for a phosphodiesterase and the later one for a diguanylate cyclase, which efficiency had already been proved (Christensen et al., 2013; Pérez-Mendoza et al., 2014). This translates into the possibility of modulating c-diGMP levels, which are responsible for the formation and dispersal of the biofilm (Monds &amp; O’Toole, 2009; Wood et al., 2011) by means of the presence of an external signal (in this case, salicylate) added to the culture media.</P><P style="font-size:15px;margin-bottom:10px">Bacterial biofilms are complex structures that need a great quantity of genes both for its formation and dispersal, as in both cases genes responsible for the production or degradation f adhesins play a role. Quiet a big part of these genes are somewhat controlled by the altering in c-diGMP intracellular levels, which favour motility loss and formation of biofilm when rising, however favouring regain of motility and biofilm dispersal when decreasing (Christensen et al., 2013; Monds &amp; O’Toole, 2009; Pérez-Mendoza et al., 2014). Due to this global regulation of many of the processes involved in biofilm structure, we decided to use a system that modified the intracellular concentrations of c-diGMP in order to manipulate biofilm formation and dispersal. With this simple way we get a fine and robust control of Pseudomonas putida’s bacterial biofilm dynamics.</P><P style="font-size:15px;margin-bottom:10px">We must highlight the fact that this behaviour towards the alteration of c-diGMP levels is not exclusive of <I>P. putida</I>, but it is a generalised model regarding the control of biofilm in most biofilm-building bacteria (Monds &amp; O’Toole, 2009). This means that our system can be extrapolated to many other bacterial strains with this same purpose. In addition, both the expression system inducible by salicylate <I>nahR</I>-P<I>sal</I> with the attenuator system <I>nasF</I> and the enzymes used, YhjH and PleD*, have their efficacy demonstrated in other Gram-negative bacterial systems and, in the case of the expression system, in some Gram-positive bacterial systems (Benedetti et al., 2016; Cebolla et al., 2001; Christensen et al., 2013; Pérez-Mendoza et al., 2014), what ensures that our system is universal. We must bear in mind in the case of PleD* that the mutations that it carries out allow it to produce higher quantities of c-diGMP, as it does not have the same inhibition mechanisms that its non-mutant variant, based on some phosphorylation dominium that does not exist in the mutant variant (Romero-Jiménez et al., 2015).</P><P style="font-size:15px;margin-bottom:10px">The inducible-by-salicylate expression system is a powerful tool to work with, not only by its universal profile and the great knowledge there exists about it (Cebolla et al., 2001) as we said before, but also for the economic saving it stands, as salicylate is a much cheaper molecule than other inductors; also, by its modular character, which allows us to combine it with other structures that can amplify even more the genetic expression, producing even greater induction windows. These structures are <I>xyls2</I> gene, which is a variant from xylS gene that allows it to answer to salicylate as well as to 3-methylbenzoate, and Pm promoter, that activates in the presence of XylS/XylS2 protein and salicylate (Cebolla et al., 2001).</P><P style="font-size:15px;margin-bottom:10px">In order to ensure the stability of the constructions build in our bacteria, we integrated them in their genome by means of the mini transposon miniTn7BB-Gm. This transposon has already been used for the expression from the genome of the genes we are using (Benedetti et al., 2016; Romero-Jiménez et al., 2015), so their its efficacy is proven. It must be highlighted that many bacteria have in their genomes the recognition site attTn7 (K.-H. Choi et al., 2005), where the transposon integrates uniquely and causing no damage in the bacterial development, as it does not interrupt any gene or regulation genetic structure (Lambertsen, 2001). There is a gentamycine resistance in its sequence that can be easily removed once it is integrated in the genome, so we find ourselves with a highly powerful genetic tool that ensures the construction of stable and safe bacterial strains.</P><P style="font-size:15px;margin-bottom:10px">Regarding the strains built in this part of the project, we can observe that those which possess the <I>nasF</I> attenuator system show a c-diGMP levels very similar to those of the wild type in non-inducing conditions. In inducing conditions, there is a slight increase of them in the case of the DGC and a slight decrease in the case of the PDE. These results are consistent with what we can observe in the literature about this attenuator in combination with the inducible-by-salicylate expression system <I>nahR</I>-P<I>sal</I> (Royo et al., 2005). This attenuator allows the basal expression levels of our genes to reduce till the levels of the wild type strain, what means that the system can be turned down until the proper time, avoiding any deleterious effect in case the protein was harmful. In this sense, we will in the future count with the antiterminator protein NasR, which associates with the attenuator and allows transcription. This gene has already been cloned, though we could not build devices with it that helped us see if it really worked in our case. As the literature results show (Royo et al., 2005), we could expect that the induced levels of these constructs were similar to those that have not the attenuator system, but allowing a lower basal levels than the constructs that do not have <I>nasF</I>.</P><P style="font-size:15px;margin-bottom:10px">Regarding the YhjH producing strain without the transcription attenuator, we could observe that the non-induced basal levels produced a two-times decrease in the c-diGMP levels in comparison with the wild type, whereas this decrease gets up to three times in inducing conditions in comparison with the wild type strain. These basal transcriptional levels that produce changes in the c-diGMP intracellular concentrations are consistent with what is expected from this system without the attenuator (Cebolla et al., 2001; Royo et al., 2005). Furthermore, the usage of YhjH in other expression systems has produced similar results in the reduction of between two and three times in the c-diGMP intracellular levels (Benedetti et al., 2016: Christensen et al., 2013). The fact that the c-diGMP levels have not been able to reduce more could be due to two different explanations: in one hand, there are several PDE that when possessing the EAL domain, they also possess a product inhibition (pGpG in this case) resulting on the degradation of c-diGMP (Lacey et al., 2010), so it would be impossible to reduce any farther the c-diGMP levels without any mutagenic change to the enzyme; in the other hand, biofilm-forming bacteria such as <I>Pseudomonas putida</I> do usually stand a huge collection of phosphodiesterases and diguanylate cyclases that answer to different types of extracellular signals. Therefore, it would not be strange that bacterium, facing a sharp decrease in c-diGMP levels, activated another path to prevent and even more sharp decrease in those levels.</P><P style="font-size:15px;margin-bottom:10px">On the contrary than in other studies, in this project several assays were performed that allowed a more deeply characterization of the YhjH-producing strain’s phenotype. The most relevant assay in this case was the adhesion one, where we could observe that not only there was a reduction in the number of microcolonies and cells attached to the surface, but also that planktonic life was favoured, what is consistent with the statements about the effects of low c-diGMP levels in the induction of flagella genes (Hengge, 2009; Monds &amp; O’Toole, 2009).</P><P style="font-size:15px;margin-bottom:10px">Regarding the PleD*-producing strain without the transcription attenuator, we can again observe that the basal transcription levels of c-diGMP production without induction are slightly higher than those of the wild type strain, what is once more consistent with the data observed about this expression system in the literature (Cebolla et al., 2001; Royo et al., 2005). In addition, we obtained induced levels highly above from the standards (almost 15 times), which is unmistakable due to the lack of product inhibition held by the mutant PleD* used. These same assays showed us a diminishment in bacterial growth that was not observed in any other case when expressing this same enzyme from alternative expression systems (Pérez-Mendoza et al., 2014), probably due to the fact that the expression system assayed was more powerful than the one used in the literature.</P><P style="font-size:15px;margin-bottom:10px">When performing other assays regarding the production of EPS, we could observe that this strain held a high-EPS-production phenotype, showing rugose colonies with many crests and an intense red dying due to the absorption of Congo Red. This phenotype was more intense than those showed in the literature by this same enzyme under different expression systems (Pérez-Mendoza et al., 2014) or by other DGC under other expression systems (Benedetti et al., 2016).</P><P style="font-size:15px;margin-bottom:10px">Furthermore, the biofilm formation assay and its later dying with Cristal Violet shows data that point out an increase of seven times in the formation of biofilm, what stands out when comparing this with other milder results obtained from other DGC with different expression systems (Benedetti et al., 2016; Pérez-Mendoza et al., 2014). In the pellicle assay performed, we could observe the biofilm formed in the liquid-air interphase, observing a great pellicle in the case of our strain, much more abundant than those observed when employing this same enzyme under another expression system (Pérez-Mendoza et al., 2014) or using mutants with biofilm super production phenotypes (Jimenez-Fernandez et al., 2015).</P><P style="font-size:15px;margin-bottom:10px">In summary, we can say that we have been able to implement a highly robust super producing biofilm system under an inducible-by-salicylate expression system.</P><P style="font-size:15px;margin-bottom:10px">Due to the metabolic advantages that the biofilm has in comparison with planktonic bacteria, the build of an at-our-will inducible system that allows us to maintain the biofilm formed and that does not allow its dispersal is a great contribution in the field of industrial bacterial production. Furthermore, the modularity that the Biobricks give to the project allows us to include whatever genes we needed for the molecule of interest’s production, having therefore developed a production platform that feeds in glycerol.</P><P style="font-size:15px;margin-bottom:10px">In addition, there have been developed a biofilm dispersal system that, in combination with the first one above, would allow a higher manipulation of this system, altogether being a great contribution and a significant advance in this field. Therefore, we should now design a system that allowed us to put both structures together, performing experiments that permitted to demonstrate the efficacy of the phosphodiesterase in the dispersal of the biofilm once it is formed. Last but not least, another interesting modification of the system would be the addition of the antiterminator protein NasR, in order to see whether the induction window actually reaches the same point with the attenuator than without it, giving us another level of control to the formation and dispersal of the biofilm.</P><H2 style="text-align:center"><B>Achievements and conclusions</B></H2><DIV style="margin-top:120px;margin-left:180px"><P style="font-size:15px;margin-bottom:10px;text-align:center">Stable modification of our chassis bacterium by means of miniTn7.</P><P style="font-size:15px;margin-bottom:10px;text-align:center">Powerful ON/OFF molecular switch.</P><P style="font-size:15px;margin-bottom:10px;text-align:center">Increased glycerol uptake and metabolization.</P><P style="font-size:15px;margin-bottom:10px;text-align:center">Strong biofilm induction/ dispersion system.</P><P style="font-size:15px;margin-bottom:10px;text-align:center">Modeling plots and equations for both metabolism and biofilm modules.</P></DIV><H2 style="text-align:center"><B>Future goals</B></H2><P style="font-size:15px;margin-bottom:10px;text-align:center">The genes for the propionate synthesis must be cloned and tested.</P><P style="font-size:15px;margin-bottom:10px;text-align:center">The complete system is to be tested at the industrial level (bioreactor).</P><P style="font-size:15px;margin-bottom:10px;text-align:center">A <I>glpR</I> mutant strain could be use to improve the expression system for the glycerol intake and consumption.</P><P style="font-size:15px;margin-bottom:10px;text-align:center">A bistable system with further regulation levels can be developed.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>