"
Page
Discussion
View source
History
teams
Log in
 
Team:UESTC-Software/Installation.html
From 2014.igem.org
UESTC-Software
Android
Desktop
Server
How To Install Crispr-X For Android
Step1: Setting unknown source
(If you have allowed installation of apps from unknown sources,please skip step1)
Setting -> Security -> Unknown sources
Setp2: Scan QR code and download the app
Step3: Install Crisp-X.apk
Step4: Enjoy it!
How To Install Crispr-X For Desktop OS
Step1: Setup Java Run Environment
(If you have setup Java Run Environment,please skip step1)
You can get a newest JRE from www.java.com .
Step2: Download & Setup Crispr-X Desktop App
Click here to download a zip file.
Step3: Just unzip crispr-X.zip, and run crispr-X/crispr-X.jar.
Step4: Enjoy it!
CRISPR-X Server and Database Manual
1.Dependence
Name
Version
Notes
Linux
Fedora, Ubuntu, Debian or related system
Here, related system means that the Linux distribution supports '.rpm' or '.deb' packages.
PHP
5.2.9 or higher version
Python
2.5.2 or higher 2.X version
MySQL
5.0.88
MySQL Connector
6.1
Connector/C (libmysqlclient).  It will be downloaded on installation automatically.
RNAfold
2.1.8
This software is free, and should be installed globally.
ImageMagick
6.4.0
This software is free, and should be installed globally. And CLI supports.
2.Deployment
Firstly, you need to execute the shell script, 'install.sh' under 'server' folder, which helps you to install dependencies automatically and compile our auxiliary programs. Then goes to database and server configure.
For database, CasDB, please fellow this instruction step by step:
1.On your server, unpack the source code, redirect into the folder 'CasDB' under 'server' directory. Please pay attention that the compressed file 'CasDB.tar.gz' is for our automatic test on Travis CI. It’s useless for CasDB deployment!
2.Make a new folder named 'database' under 'CasDB'. Next, download model organism genome data into this folder. We use data from NCBI database. Temporally support these fellow model organisms
Model Organism
Data Source
E.coli-k12-MG1655
ftp://ftp.ncbi.nih.gov/genomes/Bacteria/Escherichia_coli_K_12_substr__MG1655_uid57779/
Saccharomyces_cerevisiae
ftp://ftp.ncbi.nih.gov/genomes/Fungi/Saccharomyces_cerevisiae_uid128/
Arabidopsis_thaliana
ftp://ftp.ncbi.nih.gov/genomes/Arabidopsis_thaliana/
Drosophila_melanogaster
ftp://ftp.ncbi.nih.gov/genomes/Drosophila_melanogaster/
The whole directory structure should look like this:
-server     -CasDB         -getGeneInfo.py         - sgRNAimport.py         -database             -[Species Name]                 -[Chromosome Name]                     -[Data files, we need ffn, fna, ptt files]
3.Then create a new database, rename it as you like. Using this database, execute 'CasDB.sql' and then execute 'view.sql'.
4.Configure the python script, 'getGeneInfo.py'. Just modify one line according your environment and setup:
db = MySQLdb.connect("YOUR-DB-HOST","DB-USERNAME","DB-PASSWD","DB-NAME" )
Then run this python script by typing 'python getGeneInfo.py' in command line interface. This script will work automatically to recognize the species name and chromosomes. Besides this, more gene information will be calculated and inserted into CasDB. But it may take some time.
5.Configure the python script, 'sgRNAimport.py'. Just as above modify one line according your environment and setup:
db = MySQLdb.connect("YOUR-DB-HOST","DB-USERNAME","DB-PASSWD","DB-NAME" )
Then run this python script by typing 'python sgRNAimport.py' in command line interface. This script will work automatically to recognize the species name and chromosomes. Besides this, sgRNA will be found and inserted into CasDB. But it may take some time.
For server, you must configure PHP scripts manually. It’s very easy. Just modify 'server/config.php' according to guides in the source file.
Overview
Modeling
Demo
Validation
Download
Safety
Journal
Human Practice
Requirements
Medal Fulfillment
API Documentation
Testing Documentation
Installation
User Guide
Retrieved from " http://2014.igem.org/Team:UESTC-Software/Installation.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
