<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UESTC-Software_Modeling_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UESTC-Software/Modeling.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UESTC-Software/Modeling.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UESTC-Software/Modeling.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UESTC-Software/Modeling.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UESTC-Software/Modeling.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UESTC-Software/Modeling.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>UESTC-Software</TITLE></P><DIV id="body" style="overflow:auto;"><DIV class="midBoard" style="padding-bottom:80px;"><H1 style="padding-top: 30px">Models and Algorithms</H1><DIV class="parts" style="padding: 20px 50px 20px 100px;"><DIV class="question" id="p1">1.Overview</DIV><P>Modeling is a powerful tool in synthetic biology and engineering. In our project, we aim to design a bioinformatics tool “CRISPR-X”, which is a software developed for design of CRISPR sgRNA with minimized off-target effects and high cutting rate.</P><P>The CRISPR-associated (Cas)9 can be programmed with a single guide RNA (sgRNA) to generate site-specific DNA breaks, but there are few known rules governing on-target efficacy of this system[1,2]. Related reports suggest gRNAs are most effective with a GC-content between 40 and 80%. [1] In addition, a guanine at position 20 in the target site, which appears to improve cutting rate. [1] Therefore, we use efficacy score to characterize the activity of the sgRNA.</P><P>For sgRNA sequences can be 17-20 nt in length to achieve similar levels of on-target gene editing，and up to 10,000 fold improvement in target specificity when truncated (17 or 18 base pair) sgRNA is used. [3] We design the length of sgRNA sequences vary from 17nt to 20nt.</P><P>First of all, we find the protospacer-adjacent motif (PAM) based on user-specified gene region. Then, we find sgRNA corresponding to the PAM. Next, we find that whether there is a potential off-target binding site for the sgRNA over the entire gene region, and evaluate the specificity and efficacy of the sgRNA. Finally, we provide a secondary structure and the restriction enzyme cutting sites for the sgRNA.</P></DIV><DIV class="parts" style="padding: 20px 50px 20px 100px;"><DIV class="question" id="p2">2.Parameters</DIV><TABLE border="1px" cellspacing="0px" style="border-collapse:collapse;word-break:break-word;border-color: #c7d3af;
color: #999;line-height: 1.1em;"><TBODY><TR><TD class="pc1">Parameters</TD><TD class="pc2">Description</TD><TD class="pc3">Range</TD><TD class="pc4">Unit</TD><TD class="pc5">Remark</TD></TR><TR><TD class="pc1"><SPAN class="serif">d0</SPAN></TD><TD class="pc2">Average distance for all the mismatch nucleotides to the PAM of any off-target site</TD><TD class="pc3">0-19</TD><TD class="pc4">nt</TD></TR><TR><TD class="pc1"><SPAN class="serif">i</SPAN></TD><TD class="pc2">Continuous variables for the number of mismatch nucleotide</TD><TD class="pc3">1-<SPAN class="serif">Nmm</SPAN></TD><TD class="pc4">1</TD></TR><TR><TD class="pc1"><SPAN class="serif">j</SPAN></TD><TD class="pc2">Continuous variables for the total number of off-target sites exclude the perfect-hit off-target sites</TD><TD class="pc3">1-(<SPAN class="serif">Nfg</SPAN>-<SPAN class="serif">Nph</SPAN>)</TD><TD class="pc4">1</TD></TR><TR><TD class="pc1"><SPAN class="serif">M</SPAN></TD><TD class="pc2">Weight matrix</TD><TD class="pc3">[0,0,0.014,0,0,0.395,0.317,0,0.389,0.079,0.445,0.508,0.613,0.851,0.732,0.828,0.615,0.804,0.685,0.583]</TD><TD class="pc5">Reference: DNA targeting specificity of RNA-guided Cas9 nucleases, Hsu et al, 2013</TD></TR><TR><TD class="pc1"><SPAN class="serif">n</SPAN></TD><TD class="pc2">Mismatch position</TD><TD class="pc3">1-20</TD></TR><TR><TD class="pc1"><SPAN class="serif">Nfg</SPAN></TD><TD class="pc2">The total number of off-target sites</TD><TD class="pc3">≥0</TD><TD class="pc4">1</TD></TR><TR><TD class="pc1"><SPAN class="serif">Nmm</SPAN></TD><TD class="pc2">The number of mismatch nucleotide for the not perfect-hit off-target sites</TD><TD class="pc3">1-4</TD><TD class="pc4">1</TD></TR><TR><TD class="pc1"><SPAN class="serif">Nph</SPAN></TD><TD class="pc2">The number of perfect-hit off-target sites</TD><TD class="pc3">≥0</TD><TD class="pc4">1</TD><TD class="pc5">In our scoring algorithm, we allow the maximum value of <SPAN class="serif">Nph</SPAN> is 4, when <SPAN class="serif">Nph</SPAN>≥4, <SPAN class="serif">Sguide</SPAN>=0</TD></TR><TR><TD class="pc1"><SPAN class="serif">r1</SPAN></TD><TD class="pc2">The proportion of specificity score in the total score</TD><TD class="pc3">0-1</TD><TD class="pc4">1</TD><TD class="pc5">In our scoring algorithm, it’s default value is 0.65</TD></TR><TR><TD class="pc1"><SPAN class="serif"><SPAN class="serif">r2</SPAN></SPAN></TD><TD class="pc2">The proportion of efficacy score in the total score</TD><TD class="pc3">0-1</TD><TD class="pc4">1</TD><TD class="pc5">In our scoring algorithm, it’s default value is 0.35</TD></TR><TR><TD class="pc1"><SPAN class="serif">S1</SPAN></TD><TD class="pc2">The score of the first step</TD><TD class="pc3">≥0</TD><TD class="pc4">1</TD></TR><TR><TD class="pc1"><SPAN class="serif">S20</SPAN></TD><TD class="pc2">The subtracted score for the 20th nucleotide is not a guanine</TD><TD class="pc3">=35</TD><TD class="pc4">1</TD></TR><TR><TD class="pc1"><SPAN class="serif">Seff</SPAN></TD><TD class="pc2">The efficacy score</TD><TD class="pc3">0-100</TD><TD class="pc4">1</TD><TD class="pc5">Represent the level of efficacy for the sgRNA</TD></TR><TR><TD class="pc1"><SPAN class="serif">Sgc</SPAN></TD><TD class="pc2">The subtracted score for different  GC ratio</TD><TD class="pc3">0,35,65</TD><TD class="pc4">1</TD></TR><TR><TD class="pc1"><SPAN class="serif">Sguide</SPAN></TD><TD class="pc2">The total score of the sgRNA</TD><TD class="pc3">0-100</TD><TD class="pc4">1</TD><TD class="pc5">Composed of Seff and <SPAN class="serif">Sspe</SPAN>, marking the overall properties(specificity and efficacy) of the sgRNA</TD></TR><TR><TD class="pc1"><SPAN class="serif">Smm</SPAN></TD><TD class="pc2">The subtracted score of the mismatch nucleotide for the not perfect-hit off-target sites</TD><TD class="pc3">≥0</TD><TD class="pc4">1</TD></TR><TR><TD class="pc1"><SPAN class="serif">Sph</SPAN></TD><TD class="pc2">The subtracted score of the perfect-hit off-target sites</TD><TD class="pc3">≥0</TD><TD class="pc4">1</TD></TR><TR><TD class="pc1"><SPAN class="serif">Sspe</SPAN></TD><TD class="pc2">The specificity score</TD><TD class="pc3">0-100</TD><TD class="pc4">1</TD><TD class="pc5">Represent the level of specificity for the sgRNA</TD></TR></TBODY></TABLE></DIV><DIV class="parts" style="padding: 20px 50px 20px 100px;"><DIV class="question" id="p3">3.Scoring algorithm</DIV><P><UL style="width: 100%;
word-break: break-word;">Judged conditions:
<LI><PRE style="padding: 0;background: none;border: none;color: #777;line-height: inherit;">①Bad GC ratio (&lt; 40% or &gt; 80%) : <SPAN class="serif">Sgc</SPAN> = 65;
Not so good GC ratio (40% - 50% or 70% - 80%): <SPAN class="serif">Sgc</SPAN> = 35;
Good GC ratio (51%-69%): <SPAN class="serif">Sgc</SPAN> = 0.[1]
</PRE></LI><LI>②The 20th nucleotide is not G: <SPAN class="serif">S20</SPAN> = 35;[1]</LI><LI>③If the sgRNA designed perfectly hit another sites, the penalty <SPAN class="serif">Sph</SPAN> = 25;if perfectly hit more than or equal to 4 loci, the total score <SPAN class="serif">Sguide</SPAN> is 0.
</LI></UL></P><P><UL style="width:100%;">Steps:
<LI>(1) Firstly, find out the number of off – target sequence <SPAN class="serif">Nfg</SPAN>, if <SPAN class="serif">Nfg</SPAN> = 0, output <SPAN class="serif">Sspe</SPAN>= <SPAN class="serif">r1</SPAN>*100;
Otherwise, detect the third condition. If there is a sgRNA designed perfectly hit another site, regard the number of it as <SPAN class="serif">Nph</SPAN>, and then the score of the first step: <SPAN class="serif">S1</SPAN> = <SPAN class="serif">Sph</SPAN> * <SPAN class="serif">Nph</SPAN> (<SPAN class="serif">Nph</SPAN> is 4 or less). When <SPAN class="serif">S1</SPAN> is equal to or less than 75, perform step (2), otherwise the output <SPAN class="serif">Sguide</SPAN> = 0;
If there is no sgRAN designed perfectly hit other sites, the score of the first step <SPAN class="serif">S1</SPAN> = 0. This illustrates that there is no nucleotide which are matched between sgRNA and the place missed. Then perform step(2).
</LI><LI>(2) When performing step (2), remove the <SPAN class="serif">Nph</SPAN> which is perfectly hit first.
For <SPAN class="serif">Nfg</SPAN>-<SPAN class="serif">Nph</SPAN> which does not perfectly hit, please combine the weight ratio which obtained in the literature:
<SPAN class="serif">M</SPAN>=[ 0, 0, 0.014, 0, 0, 0.395, 0.317, 0, 0.389, 0.079, 0.445, 0.508, 0.613, 0.851, 0.732, 0.828, 0.615, 0.804, 0.685, 0.583];[4]
Using the formula:</LI></UL></P><P>Assuming that specific score: efficacy score = <SPAN class="serif">r1</SPAN>: <SPAN class="serif">r2</SPAN> (the default is <SPAN class="serif">r1</SPAN>:<SPAN class="serif">r2</SPAN> = 0.65:0.35), and then use formula specificity scores <SPAN class="serif">Sspe</SPAN> =(when 100-, <SPAN class="serif">Sspe</SPAN>=0), efficacy score <SPAN class="serif">Seff</SPAN> = r2 * (100 - (<SPAN class="serif">Sgc</SPAN> + <SPAN class="serif">S20</SPAN>)), the total score: ; 
Finally according to <SPAN class="serif">Sguide</SPAN> score, arranging the sgRNA from high to low, outputting sgRNA, total score <SPAN class="serif">Sguide</SPAN>, specificity scores <SPAN class="serif">Sspe</SPAN>, efficacy score <SPAN class="serif">Seff</SPAN>, the chromosome and its site connected to sgRNA, the GC ratio.
</P></DIV><DIV class="parts" style="padding: 20px 50px 20px 100px;"><DIV class="question" id="p4">4.Algorithm illustration</DIV><P>In literature [4], The algorithm used to score single off-targets is:</P><P>This algorithm is adopted by CRISPR-P, the inadequacies of this algorithm are: (a) Despite the presence of off-target sites, but sometimes it's subtracted score will still be 0 (which seems unreasonable under certain circumstances, and it will confuse the scoring of those sgRNA that don’t exist off-target sites). (b) Using W function, which cannot be expressed by elementary functions, it will take some additional time in calculation.</P><P>However, our algorithm can avoid these two shortcomings. Our algorithm is:</P><P>First, we use the summation of exponential replaced W function; Secondly, when there exist off-target sites, our running results will be with the subtracted score, and we use the rounding to further ensure this situation. The following table can show the score contrast.</P><TABLE border="1px" cellspacing="0px" style="border-collapse:collapse;word-break:break-word;border-color: #c7d3af;
color: #999;"><TBODY><TR><TD>Off-target sequence</TD><TD>Mismatches</TD><TD>CRISPR-P Score</TD><TD>OUR Software Score</TD></TR><TR><TD>GTT<SPAN class="red">TC</SPAN>TCC<SPAN class="red">GT</SPAN>AATCGCGTCA</TD><TD>4</TD><TD>0.8</TD><TD>0.989</TD></TR><TR><TD>GTTCTTCCA<SPAN class="red">C</SPAN>AAT<SPAN class="red">TC</SPAN>CGT<SPAN class="red">T</SPAN>A</TD><TD>4</TD><TD>0</TD><TD>0.391</TD></TR><TR><TD><SPAN class="red">T</SPAN>TTCTTCCAGAATCG<SPAN class="red">T</SPAN>G<SPAN class="red">A</SPAN>C<SPAN class="red">T</SPAN></TD><TD>4</TD><TD>0</TD><TD>0.426</TD></TR><TR><TD>GAA<SPAN class="red">A</SPAN>AA<SPAN class="red">T</SPAN>TCCTCTTATTTCA</TD><TD>2</TD><TD>3.9</TD><TD>2.177</TD></TR><TR><TD>GAA<SPAN class="red">C</SPAN>AACTCCTCTTATT<SPAN class="red">A</SPAN>CA</TD><TD>2</TD><TD>2.4</TD><TD>1.187</TD></TR><TR><TD>GAAGAACT<SPAN class="red">A</SPAN>C<SPAN class="red">G</SPAN>CTTAT<SPAN class="red">GA</SPAN>CA</TD><TD>4</TD><TD>0</TD><TD>0.402</TD></TR></TBODY></TABLE><P><UL><B>Reference:</B><LI>[1] Wang, T., Wei, J. J., Sabatini, D. M., &amp; Lander, E. S. (2014). Genetic screens in human cells using the CRISPR-Cas9 system. Science, 343(6166), 80-84.</LI><LI>[2] Gagnon, J. A., Valen, E., Thyme, S. B., Huang, P., Ahkmetova, L., Pauli, A., ... &amp; Schier, A. F. (2014). Efficient mutagenesis by Cas9 protein-mediated oligonucleotide insertion and large-scale assessment of single-guide RNAs.PloS one, 9(5), e98186.</LI><LI>[3] Fu, Y., Sander, J. D., Reyon, D., Cascio, V. M., &amp; Joung, J. K. (2014). Improving CRISPR-Cas nuclease specificity using truncated guide RNAs.Nature biotechnology, 32(3), 279-284.</LI><LI>[4] Hsu, P. D., Scott, D. A., Weinstein, J. A., Ran, F. A., Konermann, S., Agarwala, V., ... &amp; Zhang, F. (2013). DNA targeting specificity of RNA-guided Cas9 nucleases. Nature biotechnology, 31(9), 827-832.</LI></UL></P></DIV></DIV><DIV class="leftMenu"><DIV id="go1" class="go" style="width: 120px;text-align: left;">1.Overview</DIV><DIV id="go2" class="go" style="width: 120px;text-align: left;">2.Parameters</DIV><DIV id="go3" class="go" style="width: 120px;text-align: left;">3.Scoring algorithm</DIV><DIV id="go4" class="go" style="width: 120px;text-align: left;">4.Algorithm illustration</DIV></DIV><DIV class="navbar"><UL class="leftUl UL"><LI><A href="Overview.html">Overview</A></LI><LI><A href="Modeling.html">Modeling</A></LI><LI><A href="Demo.html">Demo</A></LI><LI><A href="Validation.html">Validation</A></LI><LI><A href="Download.html">Download</A></LI><LI><A href="safety.html">Safety</A></LI></UL><UL class="rightUl UL"><LI><A href="journal.html">Journal</A></LI><LI><A href="HumanPractice.html">Human Practice</A></LI></UL><LI><A href="Requirement.html">Requirements</A></LI><LI><A href="Medal.html">Medal Fulfillment</A></LI><LI><A href="API.html">API Documentation</A></LI><LI><A href="Testing.html">Testing Documentation</A></LI><LI><A href="Installation.html">Installation</A></LI><LI><A href="UserGuide.html">User Guide</A></LI></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UESTC-Software/Modeling.html">http://2014.igem.org/Team:UESTC-Software/Modeling.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UESTC-Software/Modeling.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UESTC-Software/Modeling.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UESTC-Software/Modeling.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UESTC-Software/Modeling.html&amp;oldid=392055" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>