<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UESTC-Software_Overview_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UESTC-Software/Overview.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UESTC-Software/Overview.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UESTC-Software/Overview.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UESTC-Software/Overview.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UESTC-Software/Overview.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UESTC-Software/Overview.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>UESTC-Software</TITLE></P><DIV id="body" style="overflow:auto;"><DIV class="midBoard" style="padding-bottom:80px;"><H1 style="padding-top: 30px">Overview</H1><DIV class="parts" style="padding: 20px 50px 20px 100px;overflow:hidden;"><P>Just like “Plants vs. Zombies&quot; has become one of the most popular games, CRISPR/Cas system learned, borrowed and modified from the natural game “Bacteria vs. Phages&quot; has been the hottest technology for genome editing. The CRISPR craze wept across scientific community last year. Synthetic Biology and iGEM were no exception. Twelve 2013 iGEM teams worked on CRISPR in their projects, with chassis varying from E. Coli, yeast, to plant and Mammalian. More than a dozen parts related to CRISPR/Cas were submitted last years. Some of them are in the DNA Kit of Parts for 2014 iGEM, shipped worldwide from the iGEM Headquarters. The sgRNA design tools are an important part for CRISPR/Cas technology and have attracted bioinformatician all over the world. </P><P>However, all available tools neglect the purpose of a given experiment, pay no attention to BioBricks standards, and do not support standard for synthetic biology data exchange (SBOL). Thus, a sgRNA design tool for genome editing in synthetic biology is desirable. In this project, we present CRISPR-X, a sgRNA design tool fully supporting SBOL and BioBricks standards with dynamic algorithm based on intent function, chassis, and newest experimental data. It adopts a C/S and B/S framework. The program on the server side is implemented with C and front-ends are implemented with Python (Browser ends) or JAVA ( Android ends, Desktop ends ), making it across all platforms.</P></DIV><DIV class="parts" style="padding: 20px 50px 20px 100px;overflow:hidden;"><DIV class="question" id="p1">The Brief Introduction And Functioning Of The CRISPR-Cas Systems</DIV><P>Development of tools for targeted genome editing and regulation of gene expression has significantly expanded our ability to elucidate the mechanisms of interesting biological phenomena, and to engineer desirable biological systems. The clustered, regularly interspaced short palindromic repeats (CRISPR) in combination with a CRISPR-associated nuclease 9 (Cas9) were recently demonstrated to be versatile tools for genome engineering [1]. Figure 1 shows  the functioning of the type II CRISPR-Cas systems in bacteria. Phase 1: in the immunization phase, the CRISPR system stores the molecular signature of a previous infection by integrating fragments of invading phage or plasmid DNA into the CRISPR locus as ‘spacers’. Phase 2: in the immunity phase, the bacterium uses this stored information to defend against invading pathogens by transcribing the locus and processing the resulting transcript to produce CRISPR RNAs (crRNAs) that guide effector nucleases to locate and cleave nucleic acids complementary to the spacer. First, tracrRNAs(trans-activating crRNA) hybridize to repeat regions of the pre-crRNA. Second, endogenous RNase III cleaves the hybridized crRNA-tracrRNA, and a second event removes the 5′ end of the spacer, yielding mature crRNAs that remain associated with the tracrRNA and Cas9. The complex cleaves complementary ‘protospacer’ sequences only if a PAM sequence is present [2].</P><P style="font-size: 14px;text-align: center;font-weight: 900;margin: 0 auto 20px;">Figure 1. Functioning of the type II CRISPR-Cas systems in bacteria.[2]</P></DIV><DIV class="parts" style="padding: 20px 50px 20px 100px;overflow:hidden;"><DIV class="question" id="p2">Potential applications for CRISPR-Cas9</DIV><P>In bacteria and archaea, CRISPR-Cas systems provide immunity by incorporating fragments of invading phage and plasmid DNA into CRISPR loci and using the corresponding CRISPR RNAs (crRNAs) to guide the degradation of homologous sequences [3]. Figure 2 shows  the potential applications for CRISPR-Cas9.The diverse potential applications of Cas9 range from targeted genome editing (via simplex and multiplex double-strand breaks and nicks) to targeted genome regulation (via tethering of epigenetic effector domains to either the Cas9 or sgRNA, and via competition with endogenous DNA binding factors) and possibly programmable genome reorganization and visualization. Cas9 might also be engineered to function as an RNA-guided recombinase, and via RNA tethers could serve as a scaffold for the assembly of multiprotein and nucleic acid complexes[2].</P><P style="font-size: 14px;text-align: center;font-weight: 900;margin: 0 auto 20px;">Figure 2 .The potential applications for CRISPR-Cas9.[2]</P></DIV><DIV class="parts" style="padding: 20px 50px 20px 100px;overflow:hidden;"><DIV class="question" id="p3">Background and significance for CRISPR-X</DIV><P>Although the CRISPR/Cas9/sgRNA system efficiently cleaves intracellular DNA at desired target sites, major concerns remain on potential “off-target&quot; cleavage that may occur throughout the whole genome. In order to improve CRISPR-Cas9 specificity for targeted genome editing and transcriptional control, we describe a bioinformatics tool “CRISPR-X&quot;, which is a software developed for design of CRISPR sgRNA with high cutting rate and minimized off-target effects. This software consists of programs to perform a search for CRISPR target sites (protospacers) with user-defined parameters, predict and evaluate genome-wide Cas9 potential off-target cleavage sites.</P></DIV><DIV class="parts" style="padding: 20px 50px 20px 100px;overflow:hidden;"><DIV class="question" id="p4">Comparison With Other CRISPR sgRNA Design Tools</DIV><P>Many online or stand-alone tools have been developed to design CRISPR target sites or predict off-target sites. Online tools “Cas9 Design&quot; (<A href="http://cas9.cbi.pku.edu.cn/index.jsp">http://cas9.cbi.pku.edu.cn/index.jsp</A>)[4] and “CRISPR/Cas9 gRNA finder&quot; (<A href="http://spot.colorado.edu/~slin/cas9.html">http://spot.colorado.edu/~slin/cas9.html</A>) can be used to design single or paired sgRNAs, but does not find off-targets. Cas-OFFinder (<A href="http://www.rgenome.net/cas-offinder/portable">http://www.rgenome.net/cas-offinder/portable</A>) is a web and stand-alone tool, which very rapidly finds off-targets for individual CRISPR sgRNA, but does not find candidate sgRNAs[5]. Another stand-alone tool is CasOT. It can be used to find candidate sites from input sequence and print out potential off-target sites as well. It attempts to 'score' the effect of the off-target by notifying if it is placed inside a coding exon[6]. ZiFiT ZiFiT (<A href="http://zifit.partners.org/ZiFiT/ChoiceMenu.aspx">http://zifit.partners.org/ZiFiT/ChoiceMenu.aspx</A>)[7] is only available for genomes of 9 spieces. Other online tools, such as&quot;CRISPR-P&quot; (<A href="http://cbi.hzau.edu.cn/cgi-bin/CRISPR">http://cbi.hzau.edu.cn/cgi-bin/CRISPR</A>) , “Optimized CRISPR Design&quot; (<A href="http://crispr.mit.edu/">http://crispr.mit.edu/</A>) and E-CRISP (<A href="http://www.e-crisp.org/E-CRISP/">http://www.e-crisp.org/E-CRISP/</A>)[8]can identify all off-target sequences (preceding either NAG or NGG PAMs) across the genome. These tools can automatically rank each possible sgRNA according to its total predicted off-target cleavage; the top-ranked sgRNAs may represent those that are likely to have the greatest on-target and the least off-target cleavage. Although these online tools are powerful and easy to use, the length of protospacer( not including &quot;NGG&quot;) is only 20nt, not containing 17nt or 18nt (which achieve up to 10,000 fold improvement in target specificity[9]) ,which limits their application. Detailed information about comparison of different CRISRP/Cas9 design tools is listed in the Table 1.</P><TABLE border="1px" cellspacing="0px" style="border-collapse:collapse;word-break:break-word;border-color: #c7d3af;
color: #999;"><TBODY><TR><TD class="oc1">CRISRP/Cas9 design tools</TD><TD class="oc2">Search CRISPR target sites?</TD><TD class="oc3">Length of protospacer(nt), not including &quot;NGG&quot;</TD><TD class="oc4">Length of input sequence (bp)?</TD><TD class="oc5">Kinds of PAM(NGG, NRG, NGMTT, NAGAAW)</TD><TD class="oc6">Evaluate off-targets</TD><TD class="oc7">Maximum number ofmismatches</TD><TD class="oc8">Check off-target site whether located in gene</TD><TD class="oc9">Number of species’genome</TD><TD class="oc10">Scoring consider sgRNAs’ efficacy and cutting rate</TD><TD class="oc11">Websites</TD></TR><TR class="red"><TD class="oc1">CRISPR-X</TD><TD class="oc2">Yes</TD><TD class="oc3">17-20</TD><TD class="oc4">Not limit</TD><TD class="oc5">All 4</TD><TD class="oc6">Yes</TD><TD class="oc7">4</TD><TD class="oc8">Yes</TD><TD class="oc9">≥5</TD><TD class="oc10">Yes</TD><TD class="oc11"><A href="https://2014.igem.org/Team:UESTC-Software/" target="_blank">https://2014.igem.org/Team:UESTC-Software/<SPAN id="_editor_bookmark_start_0" style="display: none; line-height: 0px;">‍</SPAN></A></TD></TR><TR><TD class="oc1">ZiFiT</TD><TD class="oc2">Yes</TD><TD class="oc3">Not limit</TD><TD class="oc4">1000</TD><TD class="oc5">Only NGG</TD><TD class="oc6">Yes</TD><TD class="oc7">3</TD><TD class="oc8">No</TD><TD class="oc9">9</TD><TD class="oc10">No</TD><TD class="oc11"><A href="http://zifit.partners.org/ZiFiT/ChoiceMenu.aspx" target="_blank">http://zifit.partners.org/ZiFiT/ChoiceMenu.aspx</A></TD></TR><TR><TD class="oc1">Optimized CRISPR Design</TD><TD class="oc2">Yes</TD><TD class="oc3">20</TD><TD class="oc4">23-500</TD><TD class="oc5">Only NAG and NGG</TD><TD class="oc6">Yes</TD><TD class="oc7">4</TD><TD class="oc8">Yes</TD><TD class="oc9">15</TD><TD class="oc10">No</TD><TD class="oc11"><A href="http://crispr.mit.edu/" target="_blank">http://crispr.mit.edu/</A></TD></TR><TR><TD class="oc1">E-CRISP</TD><TD class="oc2">Yes</TD><TD class="oc3">20</TD><TD class="oc4">Not limit</TD><TD class="oc5">Only NAG and NGG</TD><TD class="oc6">Yes</TD><TD class="oc7">?</TD><TD class="oc8">Yes</TD><TD class="oc9">20</TD><TD class="oc10">Yes</TD><TD class="oc11"><A href="http://www.e-crisp.org/E-CRISP/" target="_blank">http://www.e-crisp.org/E-CRISP/</A></TD></TR><TR><TD class="oc1">Cas-OFFinder</TD><TD class="oc2">No</TD><TD class="oc3">20,18,24</TD><TD class="oc4">20,18,24</TD><TD class="oc5">All 4</TD><TD class="oc6">Yes</TD><TD class="oc7">10</TD><TD class="oc8">No</TD><TD class="oc9">16</TD><TD class="oc10">No</TD><TD class="oc11"><A href="http://www.rgenome.net/cas-offinder/portable" target="_blank">http://www.rgenome.net/cas-offinder/portable</A></TD></TR><TR><TD class="oc1">CRISPR-P</TD><TD class="oc2">Yes</TD><TD class="oc3">20</TD><TD class="oc4">23-5000</TD><TD class="oc5">Only NAG and NGG</TD><TD class="oc6">Yes</TD><TD class="oc7">4</TD><TD class="oc8">Yes</TD><TD class="oc9">28</TD><TD class="oc10">No</TD><TD class="oc11"><A href="http://cbi.hzau.edu.cn/cgi-bin/CRISPR" target="_blank">http://cbi.hzau.edu.cn/cgi-bin/CRISPR</A></TD></TR><TR><TD class="oc1">CasOT</TD><TD class="oc2">Yes</TD><TD class="oc3">18-30</TD><TD class="oc4">&lt;1000</TD><TD class="oc5">Only NAG and NGG</TD><TD class="oc6">Yes</TD><TD class="oc7">6</TD><TD class="oc8">Yes</TD><TD class="oc9">Any</TD><TD class="oc10">No</TD><TD class="oc11"><A href="http://eendb.zfgenetics.org/casot" target="_blank">http://eendb.zfgenetics.org/casot</A></TD></TR><TR><TD class="oc1">Cas9 Design</TD><TD class="oc2">Yes</TD><TD class="oc3">20</TD><TD class="oc4">Not limit</TD><TD class="oc5">Only NGG</TD><TD class="oc6">No</TD><TD class="oc7">No</TD><TD class="oc8">No</TD><TD class="oc9">No</TD><TD class="oc10">No</TD><TD class="oc11"><A href="http://cas9.cbi.pku.edu.cn/index.jsp" target="_blank">http://cas9.cbi.pku.edu.cn/index.jsp</A></TD></TR><TR><TD class="oc1">CRISPR/Cas9 gRNA finder</TD><TD class="oc2">Yes</TD><TD class="oc3">13-32</TD><TD class="oc4">Not limit</TD><TD class="oc5">Only NGG</TD><TD class="oc6">No</TD><TD class="oc7">No</TD><TD class="oc8">No</TD><TD class="oc9">No</TD><TD class="oc10">No</TD><TD class="oc11"><A href="http://spot.colorado.edu/%7Eslin/cas9.html" target="_blank">http://spot.colorado.edu/~slin/cas9.html<SPAN id="_editor_bookmark_start_0" style="display: none; line-height: 0px;">‍</SPAN></A></TD></TR></TBODY></TABLE></DIV><DIV class="parts" style="padding: 20px 50px 20px 100px;overflow:hidden;"><DIV class="question" id="p5">Relationship With Synthetic Biology And iGEM Standard Parts</DIV><P>Genome editing is an important part of synthetic biology. It has important application value in studying the function of genes, as well as gene correction and cell replacement therapy. CRISPR / Cas technology is a rising genome editing technology, which greatly improves the ability to modify and edit the genome sequence for scientists. Our software designs CRISPR sgRNA with minimized off-target effects and high cutting rate. A part is compatible with an assembly standard, as long as its sequence has no sites of relevant restriction enzymes. Our software designs a RFC filter option that contains RFC[10],RFC[12], RFC[21], RFC[23] and RFC[25]. This ensures that each sgRNA meets the requirements of the chosen assembly standard.</P></DIV><DIV class="parts" style="padding: 20px 50px 20px 100px;overflow:hidden;"><DIV class="question" id="p6">Best Practices </DIV><P>We do follow best practice in our project development. From the start of our project, we host and manage our project on github. So we have a clear and detailed develop history on github. Besides that, we use more modern software continuous integration (CI) tools. In details,<A href="https://travis-ci.org/">Travis CI</A> performs our automatic build and test tasks of CRISPR-X server core. To review our build history, please visit <A href="https://travis-ci.org/igemsoftware/UESTC-Software_2014/builds">https://travis-ci.org/igemsoftware/UESTC-Software_2014/builds</A>. Most importantly, we use <A href="https://coveralls.io/">Coverall.io</A>to evaluate our test coverage automatically. It’s significant for us to design better test cases. And our test coverage log can be found on this page, <A href="https://coveralls.io/r/uestc-igem-2014/CRISPR-X">https://coveralls.io/r/uestc-igem-2014/CRISPR-X</A>. To make our software more robust, we also did many test. Test report can be found here, <A href="https://2014.igem.org/Team:UESTC-Software/Testing.html">https://2014.igem.org/Team:UESTC-Software/Testing.html</A></P></DIV><DIV class="parts" style="padding: 20px 50px 20px 100px;overflow:hidden;"><DIV class="question" id="p7">Future work </DIV><P><UL><LI>1. More model organisms will be supported.</LI><LI>2. We would help users share results with their collaborators through email. </LI><LI>3. For further long term, we would integrate with functional design, helping users in whole CRISPR experiment. </LI><LI>4. We would make CRISPR-X support plugins, so other iGEMers can extend it easily. </LI></UL></P><P><UL><B>Reference:</B><LI>[1] Xie, S., Shen, B., Zhang, C., Huang, X., &amp; Zhang, Y. (2014). sgRNAcas9: A software package for designing CRISPR sgRNA and evaluating potential off-target cleavage sites. PloS one, 9(6), e100448.</LI><LI>[2] Mali, P., Esvelt, K. M., &amp; Church, G. M. (2013). Cas9 as a versatile tool for engineering biology. Nature methods, 10(10), 957-963.</LI><LI>[3] Terns, M. P., &amp; Terns, R. M. (2011). CRISPR-based adaptive immune systems. Current opinion in microbiology, 14(3), 321-327.</LI><LI>[4] Ma, M., Ye, A. Y., Zheng, W., &amp; Kong, L. (2013). A guide RNA sequence design platform for the CRISPR/Cas9 system for model organism genomes.BioMed research international, 2013.</LI><LI>[5] Bae S, Park J, Kim JS (2014) Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases. Bioinformatics [Epub ahead of print].</LI><LI>[6] Xiao A, Cheng Z, Kong L, Zhu Z, Lin S, et al.. (2014) CasOT: a genome-wide Cas9/gRNA off-target searching tool. Bioinformatics [Epub ahead of print].</LI><LI>[7] Hsu, P. D., Scott, D. A., Weinstein, J. A., Ran, F. A., Konermann, S., Agarwala, V., ... &amp; Zhang, F. (2013). DNA targeting specificity of RNA-guided Cas9 nucleases. Nature biotechnology, 31(9), 827-832.</LI><LI>[8] Heigwer, F., Kerr, G., &amp; Boutros, M. (2014). E-CRISP: fast CRISPR target site identification. Nature methods, 11(2), 122-123.</LI><LI>[9] Fu, Y., Sander, J. D., Reyon, D., Cascio, V. M., &amp; Joung, J. K. (2014). Improving CRISPR-Cas nuclease specificity using truncated guide RNAs.Nature biotechnology, 32(3), 279-284.</LI></UL></P></DIV><DIV class="navbar"><UL class="leftUl UL"><LI><A href="Overview.html">Overview</A></LI><LI><A href="Modeling.html">Modeling</A></LI><LI><A href="Demo.html">Demo</A></LI><LI><A href="Validation.html">Validation</A></LI><LI><A href="Download.html">Download</A></LI><LI><A href="safety.html">Safety</A></LI></UL><UL class="rightUl UL"><LI><A href="journal.html">Journal</A></LI><LI><A href="HumanPractice.html">Human Practice</A></LI></UL><LI><A href="Requirement.html">Requirements</A></LI><LI><A href="Medal.html">Medal Fulfillment</A></LI><LI><A href="API.html">API Documentation</A></LI><LI><A href="Testing.html">Testing Documentation</A></LI><LI><A href="Installation.html">Installation</A></LI><LI><A href="UserGuide.html">User Guide</A></LI></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UESTC-Software/Overview.html">http://2014.igem.org/Team:UESTC-Software/Overview.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UESTC-Software/Overview.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UESTC-Software/Overview.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UESTC-Software/Overview.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UESTC-Software/Overview.html&amp;oldid=388378" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>