Team:Hamburg/Notebook
HOME
TEAM
PROJECT
Overview
microRNA 2911
Killgene PezT
Experiments & Protocols
Results
PARTS
Team Parts
Basic Parts
Composite Parts
NOTEBOOK
ATTRIBUTIONS
COLLABORATIONS
HUMAN PRACTICES
Seminar
Symposium
Risk Assessment
SAFETY
MODELING
RNA
Networks
Notebook
Monday, 22nd June 2015
Henning, Nikolas
Notion: linearized plasmid backbones have been stored at room temperature for eight weeks. Maybe need to resupply.
P4 mg/ml preparation of Chloramphenicol Stock (3 in EtOH)
dissolve 34 mg of Chloramphenicol in 1 ml 100% ethanol
filter through a 0.22 μm filter to sterilize
use at 1:1000 dilution in LB or LB-Agar
comment: maybe prepare bigger stock next time (10 ml). It’s not very exact working with 34 mg CAmp
marked as CAmp (top) and CAmp / 22.06.15 / 34 mg/ml (side) in Antibiotics
stored at -20 °C
( http://www.cytographica.com/lab/solutions/Chlor.htm )
Preparation of LB-Agar plates with Chloramphenicol (CAmp)
heat 250 ml LB-Agar, cool down to ca. 50 °C and add 250 μl Chloramphenicol
Preparation of 20 plates under sterile conditions
stored at 4 °C, marked as CAmp Plates iGEM, 22.06.2015
Tuesday, 23rd June 2015
Dana, Nikolas
‘Competent Cell Test Kit’
spin down DNA (0.5, 5, 10, 20 & 50 pg/µL RFP Construct BBa_J04450, pSB1C3) from Competent Cell Test Kit ( 30 sec, 10.000 rpm)
thaw competent cells XL1 blue Tetracyclin resistent) on ice, label one 2 ml tube for each concentration ( 6 (five concentrations plus 1 control) + 1 tube for control of the backbone ‘’) and pre-chill
1 µl of DNA in each tube, respectively
add 50 µl of competent cells to each tube, incubate for 30 min on ice
pre-heat water bath (42 °C), heat shock for 1 min
incubate for 5 min on ice
add 200 µl LB media, incubate for 1 h at 37 °C
pipet 20 µl on each plate (triplets, control only 1, 2 for backbone), respectively (add 50 µl LB media for easier distributing)
incubate overnight at 37 °C
The 50 pg/µL sample was forgotten on ice and the trafo will be repeated on wednesday
Wednesday, 24th June 2015
Heiko, Jannis, Julian, Nikolas
The trafo failed: there are no colonies on all plates -> repetition necessary
‘Competent Cell Test Kit’
repetition of the trafo using the 50 pg/µL solution. the 3 plates are stored at 37 °C in room 1.069
Use of DH5α instead of XL1 Blue in the Transformation, following the protocol used on tuesday
Preparation of 10 mL Annealing buffer (10mM Tris, pH7.5-8.0, 50mM NaCL, 1mM EDTA)
Preparation of 26 LB-Agar-CAmp plates (34µg/mL CAmp)
to do on Wednesday (left over of protocol)
Count the number of colonies on a light field or a dark background, such as a lab bench. Use the following equation to calculate your competent cell efficiency. If you've done triplicates of each sample, use the average cell colony count in             the calculation.
(colonies on plate) / ng of DNA plated x 1000 ng/µg
Note: The measurement "ng of DNA plated" refers to how much DNA was plated onto each agar plate, not the total amount of DNA used per transformation. You can calculate this number using the following equation:
1 µL x concentration of DNA (refer to vial) x (volume plated / total reaction volume)
Thursday, 25th June 2015
Heiko, Maurice, Vincent
Counting the colonies in the trafos from Wednesday and calculating the transformation efficiency
Max. efficiency: ~ 3e+7 cfu / μg DNA
around one order of magnitude lower than iGEM standard
sealed the plates and put into 4 °C fridge
Picked one colony from each plate and inoculated each into 10 mL of liquid LB-CAmp-medium
remaining LB medium in the 4 °C fridge
tubes in the 37 °C incubation room
Annealing of the miRNA2911 oligos
got the amount of DNA from the delivery note
diluted both oligos to 100 μM
took 500 μL each and annealed in water bath at 94 °C for 10 min
cooling to room temperature in the water over 60 min
stored the solution at 4 °C in the fridge
Friday, 26th June 2015
Henning, Nikolas, Sonja
Mini-Prep of 10 ml cultures from 25.06. → Trafo of Backbones (Trafo efficiancy-kit)
→ Protocol used: High speed plasmid medi kit (geneaid)
1C3-50A: 0.133 µg/µl, V = 48 µl
1C3-50A: 0.133 µg/µl, V = 49 µl
1C3-BBc: 0.225 µg/µl, V = 49 µl
1C3-BBc: 0.198 µg/µl, V = 49 µl
stored in freezer
Annealing of GroEL oligos (old and new oligos)
total of 4 samples
note: the old GroEL-oligos were diluted with a (very) wrong amount of buffer and shouldn’t be used anymore
https://www.addgene.org/plasmid-protocols/annealed-oligo-cloning/
used 2 g each in 50µl Buffer total
Heatblock at 95 °C for 3 min and cool down slowly for approx 1 h
Stored as “GroEL Insert” in freezer
Preparation of Agar-Plates with Kanamycin
16 Plates
Kan (50 µg/ml)
Stored in Fridge (“26.06 LB Agar Kan”)
Tuesday, 30th June 2015
Artur, Dana, Henning, Kira, Nikolas
miRNA
Control of annealing → c=19.55 ng/µL
Agarplates
without resistance for DH5ɑ
DH5ɑ
distribution on agarplates without antibiotic → 37 °C
GroEL
first annealing, no DS-DNA → repeated with waterbath (95°C)
0.092 µg/µL → 4.6 µg (after 50x)
Wednesday, 1st July 2015
Artur, Henning, Julian, Nikolas
control of miRNA2911 and old GroEL-Annealing in Agarose-gel
both positive
New annealing of GroEl-Promotor with full volume (300 µl each)
Prepare start culture of DH5α for main culture on thursday
stored in 37°C room
three tubes stored in 37°C room
Preparation of BW29655 Coli on agar plates for start culture on thursday
Restriction of vector backbone (1C3) with EcoRI and PstI
Ligation of Vector (1C3) with GroEL-Insert (first annealing from last week)
protocols are in the hardcopy labbook
Thursday, 2nd July 2015
Artur, Dana, Henning, Kira, Nikolas
Casting the agarose gel
Repetition of annealing control in gel
miRNA2911 positive, GroEL negative (false sample)
Ligation of GroEL in 1C3-Backbone
Repetition of annealing control in gel
Preparation of main culture DH5α
storage of BW29655 plates in fridge for miniprep on monday
Friday, 3rd July 2015
Heiko, Maurice, Vincent
measured the OD600 of the competent DH5α cells => 0,44
prepared new Annealing Buffer
0.5 M EDTA (Shini)
5 M NaCl (Shini)
1 M Tris (self-made:made in 10 mL ddH2O, pH 8.04)
Annealing Buffer:
EDTA 20 µL
Tris 100 µL
NaCl 100 µL
+ H2O
→ 10 mL
diluted new FW miRNA 573 µL (73 µL still in the freezer, orange box)
diluted new REV miRNA 769 µL (269 µL still in the freezer, orange box)
Annealing (94°C) (water quench, 10 min, then cooling at RT)
→ product in freezer
OD600 = 0.5
gone on working with cells from 02/07/15
protocol “Kompetente Zellen” (Shini)
end: 13:50h, cells at -80°C
Monday, 6th July 2015
Dana, Heiko, Henning, Sonja
measured the concentration of the miRNA-Annealing from friday → 2.2 µg/µl
Ligation of the miRNA-Insert in pSB-1C3 (cut with EcoRI & PstI)
forgot the ligation control again -.-
efficiency kit used for trafo-control
competent DH5α + 0.5 - 50 ng RFP construct
Trafo of GroEL (2x) and miRNA in DH5α
Preparation of start culture with BW29655
4x 5 ml without any AB!
Stored at 37°C in shaker
Tuesday, 7th July 2015
Artur, Heiko, Vincent
Preparation of SOC-medium (instruction “Roche, Lab FAQ”, by Dagmar)
Bacto-Tryptone 2.000 g
Bacto-Hefe-Extract 0.500 g
NaCl 0.050 g
KCl 0.019 g
MgCl2 * (H2O) 6 0.203 g (without (H2O)6 -> 0.095 g)
Glucose 0.036 g
pH 7.0 with pH-Meter
10 mL (fridge)
Transformation with Competent Cell Test Kit (5 diff. conc.: 1.5, 5, 10, 25, 50 pg/µL) in 5 aliquots of DH5α
each with 50 µL DH5α + 1 µL DNA
Incubation on ice (30 min)
Heat shock (water, 42°C)
5 min on ice (rather 15 min)
200 µL SOC in tubes
Incubation for 2 h at 37°C with rotation
Plating (2 plates each: 20 µL + 200 µL) → 10 plates
Incubation 37°C over night
2 Glycerol stocks from BW29655 liquid cultures
500 µL (50% Glycerol, 50% H2O)
500 µL BW29655
→ - 80°C
Pooling of the liquid cultures of BW29655 in 200 mL LB medium (sterile)
→ 37°C
Wednesday, 8th July 2015
Heiko, Henning, Nikolas, Sonja
Preparation of Agar plates with Chloramphenicol (CAMP, 8.7.2015)
Transformation of miRNA-Ligation and GroEL-Ligation product as well as PezT-Plasmid (Kan resistance!!!) in DH5α
each with 50 µL DH5α + 1 µL DNA
Incubation on ice (30 min)
Heat shock (water, 42°C)
5 min on ice (rather 15 min)
200 µL SOC in tubes
Incubation for 2 h at 37°C with rotation
Plating (2 plates each: 20 µL + 200 µL) → 10 plates (using kan resistance for PezT-Plasmid)
Incubation 37 °C over night
Preparation of new start cultures of BW29655
3x 5 ml “iGEM a,b, c BW”
stored at 37°C over night
Thursday, 9th July 2015
Artur
Sealing of the plates with the transformation and storing in the fridge
Storing the BW29655 liquid cultures in the fridge
Tuesday, 14th July 2015
Maurice, Vincent
Picked liquid cultures of miRNA and pezT
Evaluation of the transformation from July 8th
colony counts in the physical labjournal
Preparation of ca. 170 mL LB+Kan medium (30 µg/mL Kan)
stored in the fridge
Picked one miRNA colony and 3 pezT colonies
transferred into 5 mL of LB (+CAmp/Kan) each
incubation at 37 °C over night
Transfer of 500 µL BW29655 culture into 50 mL LB for competent cell protocol
incubation at RT and 200 rpm over night
Wednesday, 15th July 2015
Dana, Julian, Kira
OD of BW29655 culture was too high (4), stored in the freezer ; again 500 µL BW29655 culture were transferred into 50 mL LB for competent cell protocol
over night culture of miRNA and 3 pezT colonies were prepped, stored in freezer in the second box, blue label on top and blue label on each eppi
First cell cultures checked again at 5 pm OD600: 5
We stopped the growth of the new cells via freezing (in the Freezer, with the date of today),
please restart growing step in the 37°C room.
Thursday, 16th July 2015
Henning, Nikolas, Heiko
digestion control of prepped miRNA and pezT from wednesday. All controls were positive.
Maybe we pipetted enzyme to all the controls
digested with EcoRV and XhoI
Repetition of the digestion controls
digested with EcoRV and XhoI
digest of vector plasmid (prepped from trafo in first week)
digested with EcoRI and SpeI
again the control was positive (3 fragments)
isolated and prepped the supposed fragment
control of the prepped fragment in gel together with repetition of the control
used two controls, one with alternative buffers and loading dye, one as before
Friday, 17th July 2015
Henning, Nikolas, Heiko
Glycerol-Stock (50%)
sterile filtration
pezT-Backstock
for each (A,B,C), in -80°C
New Pipes
Monday, 20th July 2015
Nikolas, Sonja
TB PIPES Buffer
für 100 mL:
1.865 g KCl
0,22 g CaCl x 2 H20
2 mL 0.5 M PIPES
Auf pH 6.7 bringen
0,889 g MnCl2 x 2H2O
ddH2O ad 100 ml
Restriktion des 1C3 BB:
4 µL pSB-1C3 Vector
1 µL EcoRI
1 µL PstI
2 µL FD buffer
2 µL ddH20
1 h 37° C stored in fridge
Tuesday, 21st July 2015
Nikolas
Ligation GroEl in 1C3:
2 µL T4 Ligase buffer
1 µL T4 Ligase
9 µL 1C3 BB
2 µL “i2” insert
6 µL ddH20
DNA concentration in GroEL tubes
GroEL 1:         2 ng/µL
i:             2.5 ng/µL
GroEL Insert:  171 ng/µL
Kompetente Zell Kultur anlegen
OD600= 0.066 (14:30)
Wednesday, 22nd July 2015
Nikolas, Henning, Sonja
New GroEl ligation (see Tuesday)
Thursday, 23rd July 2015
Dana, Henning
Make BW29655 competent
OD measurement of  BW29655 → OD600 = 0.644 → broke centrifuge :(
Cut pSB_1C3 with EcoRI & PstI
20 µL pSB-1C3 Vector
1 µL EcoRI
1 µL PstI
3 µL FD buffer
5 µL ddH20
20 minutes at 37 °C, clean with Kit → measure concentration : 23 ng/µL (460 ng)
Ligation
1      µL pSB-1C3 (23 ng)
1/ 1.5 µL GroEl-Insert (35 ng) / miRNA (33 ng)
2      µL T4 Buffer
1      µL T4 Ligase
5      µL ddH2O
15 minutes at room temperature, then 10 min at 65 °C
Transformation
1 µL Ligation Product
50 µL DH5α competent cells
30 minutes on ice, 1 min heat shock (42 °C), 5 min on ice
add 200 µL SOC media, 2 h at 37 °C
plate out on C-amp-plates
Friday, 24th July 2015
Heiko, Vincent
Freezing the competent cells in liquid nitrogen
Trafo results:
2 measurements OD600 ( 11:00, 12:30) = 0.00
use of the competent cells from 17/07 “BW 2. Versuch”OD600 = 0.546
Protocol “Kompetente Zellen”
use of the centrifuge in the 3rd floor (Rm. 3.004)
N2 (l) (80°C) at 15:00
Monday, 27th July 2015
Nikolas, Sonja
miRNA-mini preparation
3 tubes (source: 200 µL plate)
37 °C
GroEl ligation (NEW)
1) cut pSB1C3 with EcoRI and PstI
1 µL    EcoRI
1 µL    PstI
2 µL    pSB1C3
1 µL    Buffer
5 µL    a. bidest.
→ 37 °C for 15 minutes
→ 65 °C for 10 minutes heatshock
2) Backbone dephosphorylation
10 µL    pSB1C3 (cut)
1 µL    FastAP
4 µL    FastAP Buffer
→ 37 °C for 30 minutes
→ purify with PCR clean up kit
3) Ligation
20 µL    pSB1C3 (dephosphorylated, cut)
10 µL    Insert
8  µL    T4 Buffer
4  µL    T4 ligase
Transformation Efficiency Kit für BW_29965 (see iGEM protocol)
Tuesday, 28th July 2015
Dana
miRNA preparation:
OD was too low for preparation (Probes were not on the shaker)
→ Julian will pick new colonies on wednesday
→ Preparation on Thursday
GroEl-Ligation → Trafo
10 µL Ligation + 50 µL DH5α
30 minutes on ice, 1 min heat shock (42 °C), 5 min on ice
add 200 µL SOC media
incubate for 2 h at 37 °C
plate out on C-amp-plates
Transformation Efficiency- BW29655
→ all but one were not turned upside down for incubation at 37 °C over night
Explanation for DNA per Plate:
(0.5;5;10;20;50) x (20 or 200 µL-volume plated/ 250 µL -total reaction volume (50 µL cells + 200 µL SOC)) x 1 ng/1000 pg
Efficiency:
(colonies on plate) / ng of DNA plated x 1000ng/µg
for 20 µL-Probes
10 pg/µL → 1/0.0008 cfu/ng x 1000 ng/µg → 1.25 e+3 cfu/µg
20 pg/µL → 1/0.0016 cfu/ng x 1000 ng/µg → 6.25 e+5 cfu/µg
50 pg/µL → 3/0.004 cfu/ng x 1000 ng/µg → 7.5 e+2cfu/µg
for 200 µL-Probes
5  pg/µL → 10/0.0004 cfu/ng x 1000 ng/µg → 2.5 e+7cfu/µg
10 pg/µL → 12/0.008 cfu/ng x 1000 ng/µg → 1.5 e+7 cfu/µg
20 pg/µL → 125/0.016 cfu/ng x 1000 ng/µg → 1.6 e+7 cfu/µg
50 pg/µL → 70/0.04 cfu/ng x 1000 ng/µg → 1.75 e+7 cfu/µg
Wednesday, 29th July 2015
Julian
Picked the colonies, named them and put them into 3 glasses each filled with 5mL LB+CAmp with a colony inside.
Counted the colonies on the prepared agar plates: All were white.
Control:             no colonies
20µL :               no colonies
200µL, 1st pick:     23-26 colonies, and about 10 really little ones
200µL, 2nd pick:     27-30 colonies, and about 10 really little ones
Thursday, 30th July 2015
Nikolas, Sonja
miRNA plasmid isolation
NanoDrop
c(04)= 0.035 µg/µL
c(05)= 0.038 µg/µL
c(06)= 0.068 µg/µL
Friday, 31st July 2015
Kira, Vincent
Preparation of the GroEL-minis
stored in freezer
concentration: (1)= 118 ng/µL, (2)= 178 ng/µL, (3)= 233 ng/µL
Monday, 3rd August 2015
Heiko, Julian
GroEL (picked colonies)
instruction “Presto Mini Plasmid Kit”
at step 7 method 2 used for higher DNA concentration
each time 3 min absorption time
last elution: centrifugation for 4 min at 20°C
Results from the picked miRNA (29.07.15)
Resistance with BW29655 in 5µL each:
→ CAmp, Kan, Without
→ 37°C over night
Preparation of new agar plates with CAmp resistance
Tuesday, 4th August 2015
Henning
Redistribution of some samples from 37°C to freezer and the other way round
Wednesday, 5th August 2015
Henning, Heiko, Vincent
Picking the colonies
Plating of pezT (14.07.15) (l) A, B, C on plates (200 µL each)
→ 37 °C
Picking from plates: The 5 BBas, miRNA (23.07.15), GroEL
pipette-tips in 5 mL LB+CAmp each
two preparations each
→ 37 °C
Mutagenesis-PCR
“PCR-Phusion protocol” (NEB)
primers diluted to 10 µM
charging the PCR (middle PCR device block 2, program 42 in directory “Katja”)
supporting information:
dNTPs from Katja, primers solved in Annealing buffer because lyophilized
concentration with NanoDrop (volume: 4 µL) : 41 ng/µL
Thursday, 6th August 2015
Dana
PCR-Mutagenesis (1&2), check on gel, two different dilutions
10 µL: 9 µL sample + 1 µL loading dye
10 µL: 2 µL sample + 7 µL water + 1 µL loading dye
new PCR (3)
Nanodrop:     PCR 1: 148 ng/µL
PCR 2: 175 ng/µL
PCR 3:   68 ng/µL
PCR-fragment 3 not detectable
light-inducable promotor:
1 : BBa_K801042
2 : BBa_K801041
3 : BBa_K801043
4 : BBa_K1017301
5 : BBa_K1017726
prepped samples (ng/µL)
1A : 375    1B : 328
2A : 275    2B : 275
3A : 252    3B : 265
4A : 178    4B : 157
5A : 108    5B : 68
Friday, 7th August 2015
Dana
pezT-Mutagenesis
PCR-Fragments 1 + 2, gel clean up, cut out of gel
Gel extraction with kit, probes are labelled as 1 + 2 in the second box in the freezer
Monday, 10th August 2015
Heiko, Julian
Programming the PCR
new pezT plates plated out (How to do IPTG induction?)
PCR (5x buffer, Phusion Polymerase, DMSO from Siegfried)
DNA concentration from purified PCR 1, 2
1    230 ng/µL
2    100 ng/µL
Biometra Thermocycler (DNA under 1 kB)                ??
At “Ulrike” number 15, IGEM
Again PCR 1 + 2
gel: PCR 3, 2, 1
9 µL product + 1 µL loading dye
Tuesday, 11th August 2015
Henning
Extraction of Dna from gels isolated yesterday. Concentrations are too low.
Next time prepare full volume of PCR-product in Gel for the Extraction.
Wednesday, 12th August 2015
Heiko, Julian, Dana
pezT
repeated thinning streak to get single colony
Mutagenesis-PCR
repeated PCR 1 & 2
load result onto gel
gelextraction
concentration PCR 1: 8 ng/µL    PCR 2: 10 ng/µL
Lightinduceable promotor:
trafo of BBa_…(1A,2A,3A,4A,5A) into BW29655
10 µL of BBa each
Thursday, 13th August 2015
Heiko, Vincent
Mutagenesis-PCR
gel extraction
PCR 3 with cleaned PCR fragments
PCR 4 (iGEM MUT4)
Lightinducable promotors
picked single colonies from trafo into 5 mL LB (+Camp)
pezT
picked single colonies into 5 mL LB (+Kan) (1,2,3)
Friday, 14th August 2015
Dana
Lightinducable Promotor:
Glycerolstocks (stored at -80°C)
pezT-Mutagenesis
Restriction:
20 µL PCR4
1 µL EcoRI
1 µL PstI
1 µL FD Buffer
5 µL ddH20
→ 20 minutes at 37 °C
Clean up with PCR-Clean Up Kit
Concentration after clean up: 38 ng/µL
Ligation
1 µL Vektor (pSB-1C3, 23 ng)
1.2 µL pezT-Insert (38 ng)
2 µL T4 Buffer
1 µL T4 Ligase
5 µL ddH20 (6 µL for control)
→ 20 minutes at RT, 10 min 65°C
Tuesday, 18th August 2015
Dana
Constitutive promoter (BBa_J23100, 35 bp) (for 3A-Assembly with miRNA)
Amp-Plates
Biobrick on Plate 4 of distribution kit;
Well 17D, in Plasmid Backbone BBa_61002, Amp-Resistance
punch hole into foil of cover, add 10 µL of dH2O, pipette up and down,
let sit for 5 minutes
transfer into tube with competent cells (BWs), two seperate samples,
5 µL into each, + one control, trafo
Wednesday, 19th August 2015
Henning
Constitutive promoter (BBa_J23100, 35 bp) (for 3A-Assembly with miRNA)
All plates were positive including control. Maybe the BW are resistant against Amp?
Inoculation of new Amp plates with competent BW (20 µl) for control
other samples are not usable either (too much bacteria)
PezT
control gel of ligation and ligation control from monday.
2 µl sample + 7 µl water + 1 µl LD
Ligation seems not to be successful due to two equal signals in sample and control.
Repetition is necessary.
Friday, 21st August 2015
Henning, Maurice, Vincent, Artur
Loading the samples into the gel
Supplies
made approx. 20 new Amp-plates
stored in fridge
Transformations
pezT ligation product into DH5α (two batches -> Ligation A and B)
BBa_J23102 into DH5α (two batches -> Promoter A and B)
BBa_K592009 into DH5α (two batches -> Pigment A and B)
plated all and incubated at 37 °C
PezT amplification PCR
PCR of the mutagenesis PCR product with the standard Fusion protocol
control gel of PezT amplification PCR
measured the concentration of the PCR product
control gel of PezT amplification PCR
measured the concentration of the PCR product
control gel with 1% agarose
discard the sample due to strong impurity
Preparation of PezT containing 5 ml cultures for IPTG induction
made new liquid culture from glycerol stock in LB + Kan medium
incubated at 37 °C
preparation of 9 different dillutions of 3 samples to get good OD on saturday
1:5, 1:10 and 1:20 dilutions of samples with OD 0.8
Saturday, 22nd August 2015
Nikolas, Sonja
Picking Minis (Duplicates):
Pigment A    (in CAMP LB)
Pigment B    (in CAMP LB)
Ligation A    (in CAMP LB)
Ligation B    (in CAMP LB)
Promoter A    (in Amp LB)
Promoter B    (in Amp LB)
Incubation at 37° C
IPTG Induktion von pezT Colis:
OD measurement of prepared cultures
selecting culture with an Od of ~0,4 (OD=0,354)
Dilute selected culture in 20 ml LB
and split those in 2 Erlenmeyerflasks (~12 mL each, control+sample)
IPTG induction by adding 12 µl 1 M IPTG
OD measurement every 20 minutes
We noticed a constant decline in the OD
and decided to abort the experiment after 1 h
Sunday, 23rd August 2015
Henning
Glyercolstock of trafo samples from saturday
added 300 µl Glycerol (70%) to 700 µl bacterial culture
stored at -70 °C (one night at -20 °C)
Preparation of mini-cultures from saturday
preparation of all mini cultures from saturday
Ligation, pigment and promoter
appropriate to the kit’s protocol
measured the concentration
stored at -20 °C
Control digestion of the PezT ligations
100 µg sample DNA
1 µl EcoRI
1 µl PstI
1 µl FD Buffer
x µl ddH2O
20 minutes at 37°C
loaded on 1% agarose gel
Dilution of pezT cultures for IPTG induction
prepared two culture dilutions
approx. 125 ml LB with Kan each
added half of the control culture from Saturday
stored one in the fridge and the other down to the bench at RT
Monday, 24th August 2015
Artur, Vincent
Repetition of the control digestion
beside ligation sample volumes:
1µL EcoRI
1µL PstI
1µL FD Green Buffer
ad 10 µL H2O
checked with control gel
suggestion: repeat again with positive control of unmutated PezT
creation of miRNA probe
four single stranded PCRs with VF or VR primers and miRNA insert over night/ 100 cycles
suggestion for further steps: tag with 32P via 32P-ATP and PNK
(stored at -20° C, Tuesday)
IPTG induction of PezT culture
first try: three samples with
LB-Kan
PezT cells + IPTG (1mM)
PezT cells - IPTG (1mM)
second try:
PezT culture growth over night
Tuesday, 25th August 2015
Artur, Maurice
Continuing IPTG induction of PezT culture
measured ODs
no recognition of sufficient OD decrease in both tries
suggestion: make new IPTG stock
3A-Assembly
of GroEL biobrick with pigment biobrick and constitutive promoter with miRNA
following the iGEM protocol (restriction and ligation), both in pSB1K3 backbone (Kan resistance)
protocol changes/used materials: fast digest restriction enzymes and buffer,
no BSA, our samples (pigment A1 from 23.08., promoter A1 from 23.08, GroEL from 31.07 (233 ng/µL),
miRNA (elution) from 30.07 (38 ng/µL)), 1:10 dilutions of samples (except miRNA)
obtained two samples (1 GroEL + pigm 3A Assembly, 2 prom + miRNA 3A Assem) for transformation,
stored at -20° C
Thursday, 27th August 2015
Dana, Vincent
pezT IPTG induction
plated liquid cultures from the 26th on Kan-plates (20 µL each)
incubated at 37 °C
OD600 of liquid cultures with and without IPTG was the same
plates with and without IPTG are similarly covered with colonies
GroEL+Pigment & Promoter+miRNA
picked colonies into liquid cultures
incubated at 37 °C
Colony PCR
picked colonies from miRNA, GroEL, GroEL+Pigment and Promoter+miRNA
8 colonies each
made reciprocal plate for picked colonies
incubated at 37 °C
used DreamTaq-Mix
made control gel
Friday, 28th August 2015
Nikolas, Sonja
GroEL+Pigment & Promoter+miRNA
Miniprep, DNA stored at -20 °C
Glycerol Stock; stored at -80 °C
pezT Colony PCR
Preparation of 2 plates from liquid culture pezT A
Incubation at 37 °C
Saturday, 29th August 2015
Julian
pezT Colony PCR
Put the cultures in the fridge
Monday, 31st August 2015
Artur, Nikolas
pezT Colony PCR
Picked 2 colonies from pezT plates (stored in the fridge)
used DreamTaq MM
Made control gel
GroEL+Pigment
Diluted cultures A and B to an OD of 0,45
incubated 20 min at 37 °C
Heatshock37 in water bath 1 min 42 ° C
incubated at 37 °C
Decided to repeat the heatshock for culture B to ensure sufficient stress level
Tuesday, 1st September 2015
Heiko, Julian, Vincent
GroEL+Pigment
cultures from 31st August did not express the pigment
diluted cultures in liquid medium and incubated at 42 °C over night
pezT Colony PCR
repetition of colony PCR with DreamTaq-Mix
Meinhart plasmid, the PezT preparations A, B and C,
a colony from the dilution plating, negative control
control gel in 1 % agarose
two fragments in the Meinhart plasmid and the PezT preparations A and B at ca. 3000 nt and 2500 nt
Wednesday, 2nd September 2015
Nikolas, Julian, Maurice
pezT Colony PCR
Colony + pezT A, B, C
Important: Strip tubes/lids are leaky -> The water in a few samples evaporated just like yesterday -> no PCR
Light Promoter 3A Assembly
3A assembly of BBa_K1017726 and BBa_101301 according to the iGEM protocol
Ligation stored in freezer
Important: The BBs might be in bacteria, we inoculated a mini culture to test this possibility
Thursday, 3rd September 2015
Heiko, Maurice
analysis of ‘BBs in bacteria’ suspicion
no growth in Kan mini cultures  and C mini cultures (BBa_K1017726 and negative control),
but growth of C mini culture with BBa_101301
nevertheless continuing the transformation protocol, could be false positive
transformation of Light Promoter 3A Assembly ligation product
four samples: E/P restricted pSB1K3 BB (Kan plates), 3A assembly ligations product (Kan Plates),
negative control with DH5-alpha only (Kan plate) and positive control of pSB1C3-RFP standard biobrick (brown eppi) (C plates)
growth over night in 37° room
repetition of pezT colony PCR
scheme like yesterday
gel with PCR products
Friday, 4th September 2015
Nikolas, Sonja
Annealing miRNA and GroEl
10 µL miRNA fw + 10 µL miRNA rev
10 µL GroEl fw + 10 µL GroEl rev
10 min at 94 °C
Cool down to RT slowly
pSB1C3 restriction
10 µL pSB1C3 RFP
1 µL EcoR1
1 µL Pst1
2 µL FD buffer
6 µL ddH2O
30 min at 37 °C
Heat inactivation 80°C for 20 min
pSB1C3 dephosphorylation
10 µL pSB1C3 (cut)
1 µL FastAP
2 µL FastAP buffer
7 µL ddH2O
30 min at 37 °C
Cleaning of pSB1C3 with PCR Clean Up Kit
NanoDrop
pSB1C3 (dephosphorylated)    8 ng/µL
Ligation
With miRNA old and new, with GroEl old and new
3 µL pSB1C3
1 µL insert
2 µL T4 buffer
1 µL T4 ligase
3 µL ddH2O
Transformation
GroEl old and new, miRNA old and new in DH5 alpha
DH5α 30 min on ice
1 min heat shock 42.2 °C
15 min on ice
ad 200 µL SOC-medium
1 h and 10 min, 37 °C plate rotator
Samples plated on cAmp plates:
200 µL
20 µL    each
1 neg. control a 200 µL
Stored at 37 °C overnight.
New SOC-medium
200 µL aliquots in freezer
GroEl+ pigment A, B mini
5 mL overnight culture in LB Kan medium
Stored at 37 °C.
Saturday, 5th September 2015
Nikolas, Sonja
No colonies after transformation on Friday.
NanoDrop
(For annealings)
GroEl new    4.26 µg/µL        + 50 µL annealing buffer
GroEl old    0.0  µg/µL
miRNA new  2.313 µg/µL      + 50 µL annealing buffer
miRNA old    0.0 µg/µL
Maxi for GroEl+ pigment A, B
300 mL culture
Overnight 37 °C
(were not prepped)
GroEl promotor test
1.5 µL per sample
10, 20, 30 min heat shock (42 °C)
samples transferred to 5 mL overnight cultures after heat shock, stored at 37 °C.
Backbone restriction
200 ng plasmid of each sample:
pSB1C3: 8 µL
BB 16.7.: 14.3 µL
BBC 26.6.: 1.2 µL
For each sample:
3 µL FD buffer
1 µL EcoR1
1 µL Pst1
final sample volume 20 µL
Backbone dephosphorylation
20 µL restriction product
1.5 µL FastAP
2.4 µL Fast AP buffer
30 min 37 °C
Stored in freezer at -20 °C.
Sunday, 6th September 2015
Julian, Henning
Preparation of new LB-Agar-Camp plates
Preparation of 36 new Chloramphenicol containing LB-Agar plates
PCR-Clean-Up of dephosphorylated Ligations from Sep. 5th
using PCR-Clean-Up Kit
concentrations of eluted samples are noted in the hardcopy labbook
Annealing of miRNA oligos
mixed 100 µl of miRNA and GroEL oligos (100 mM each)
heated up to 95°C (heatblock)
slowly cool down to RT (approx. 2h)
measured concentration (hardcopy labbook)
Digestion of pSB-1C3 backbones
used different samples of backbone
linearized backbone, provided by iGEM (50 µg total)
backbone from stock preparation (very old)
backbone from RFP-construct (very low concentration (pmol)??)
digested with EcoRI and PstI for 20 min (FD)
heat inactivation of enzymes at 65°C for 20 min
Dephosphorylation of pSB-1C3 backbones
dephosphorylated with fast AP for 10 min at 37°C
PCR-Clean-Up of all samples
Not necessary, fast AP ist heat sensitive!
concentrations are noted in the hardcopy labbook
Ligation of miRNA and GroEL inserts with pSB-1C3 backbones
used insert samples from different steps of the hole experimental phase
and from the annealing from the same day
Prepared different combinations of Insert and backbone samples
Used different concentrations of insert (3:1 & 30:1)
→ 19 samples
Ligated with T4 DNA ligase for 20 min at RT
Heat inactivation at 65°C for 10 min
chilled on ice
Transformation of ligation constructs
used a total of 24 different samples (19 new + 5 old ligation constructs)
added 12 µl of ligation to 50µl of competent DH5a
incubation on ice for 30 min
Heatshock at 42°C for 30 sec
chilled on ice until use (approx. 5 min)
Added 950 µl LB medium (RT)
Incubation at 37°C for 30 min while shaking
spreaded 200 µl of cells and ligation mixture to LB-Agar plates (RT, Camp, very thin!)
Incubated over night at 37°C
Monday, 7th September 2015
Dana
Trafos from yesterday (sunday 6th)
no growth on any plate, put back into 37 °C→ on the table on the left
New trafos
all ligations from sunday 6th were transformed into DH5a
2 hours incubation with SOC-medium
labelled 2-18, on the table in 37°C room on the right
pSB1C3-RFP-positive control
picked 3 colonies, minis in 37 °C
Tuesday, 8th September 2015
Nikolas
Mini preparation
picked 3, 6, 17 and 22 (duplicates for each plate)
5 mL overnight cultures
stored at 37 °C
Wednesday, 9th September 2015
Dana
iGEM Plasmids
Kit Plate 1    1A    BBa_K314110
Kit Plate 1    3A    BBa_K873002
Kit Plate 1    12L    BBa_K517003
→ Trafo into DH5a from the group of Ignatova
Minis from yesterday:
6+ 17: media were red, discarded them
3+22: mini plasmid prepp with kit, taken them to get sequenced
Concentrations:
3A: 165 ng/µL
3B: 208 ng/µL
22A: 230 ng/µL
22B: 180 ng/µL
Thursday, 10th September 2015
Julian
Picking minis:
Visual colonies had been picked.
Due to the fact that only some colonies had been on the plates
i have returned them to the 37°C storage room.
Efficiency calculation:
Couldnt be calculated because i didn't find a control plate with 20µL.
Friday, 11th September 2015
Dana
Started with restriction of pSB1C3 backbone
restriction with PstI, 1h, 37 °C
PCR clean up
restriction with EcoRI, 1h 37 °C, stopped here, because data from sequencing came back
Overview Sequencing: 48DD03    GroEl    118 ng/µL
48DD06    GroEl    178 ng/µL
48DD09    GroEl    233 ng/µL
48DD02    miRNA     35 ng/µL
48DD05    miRNA     38 ng/µL
48DD08    miRNA     68 ng/µL
48DD24    3A       165 ng/µL    miRNA Ligation from 06.09.15
48DD23    3B       208 ng/µL    miRNA Ligation from 06.09.15
48DD20    22A      230 ng/µL    miRNA Ligation from 23.07.15
48DD21    22B      180 ng/µL    miRNA Ligation from 23.07.15
As miRNA (48DD05) and 22a+b are postive, miRNA 05 and 22A were transformed into DH5a
Sunday, 13th September 2015
Julian
Picking of the minis
Monday, 14th September 2015
Julian
Prepping of the minis (plasmid kit)
Tuesday, 15th September 2015
Julian
Annealing of the oligos
10 min at 94°C (Heatblock)
Temperature reduction to room temperature in one hour
Restriction of the linear backbone
60µL pSB-1C3 (25ng/µL)
3µL EcoR1 (fast digest)
3µL Pst1 (slow digest)
9µL FD-Buffer
⇒ 1 hour at 37°C ⇒ PCR clean up kit
90µL restricted Backbone
450µL DF-Buffer
rest as said in the protocol
⇒ concentration BB1 = 23,5 ng/µL
Dephosphorilation of the restricted Backbone
30µL pSB-1C3 (cut)
3µL Fast AP
tip from Chao, not to use the Fast AP Buffer
PCR Clean up kit
⇒ Concetration restricted and dephosphorilated BB2= 5 ng/µL
Ligation
Dilution of the annealed oligos up to 10 ng/µL
ratio of 1:3 vector to insert
Groel
10µL     rest. cut BB (5ng/µL ⇒ 50ng)
2,2µL     alligatet Groel oligos (10ng/µL ⇒ 22ng)
2µL     T4- DNA Ligase Buffer
1µL     T4 -DNA Ligase
5µL ddH2O
miRNA
10µL     rest. cut BB (5ng/µL ⇒ 50ng)
1,5µL     alligatet Groel oligos (10ng/µL ⇒ 15ng)
2µL     T4- DNA Ligase Buffer
1µL     T4 -DNA Ligase
5µL     ddH2O
control
10µL     rest. cut BB (5ng/µL ⇒ 50ng)
2µL     T4- DNA Ligase Buffer
1µL     T4 -DNA Ligase
7µL     ddH2O
⇒ Overnight Ligation with 16°C
Wednesday, 16th September 2015
Julian, Maurice
Transformation of the ligated Oligos (as told as in the protocol written by Henning and Dana)
Restriction of the miRNA with constitutive promoter to change the backbone
⇒ Ligation with new backbone (pSB-1C3)  ⇒ Transformation with new backbone (37°C room)
Thursday, 17th September 2015
Sonja, Julian, Maurice
Restriction of new IDT-Products ( const. Prom. + miRNA)
Ligation (into the vector)
Preparation of new liquid and solid bacterial cultures
Friday, 18th September 2015
Henning, Julian
Prepping, drying and colony PCR of the minis (all made by official and already established protocols)
Unspecific validation of the heat induction promotor via  an agarose gel
Finishing the Wiki-Page in particular by Dana, Vincent and Maurice, the  rest of the team participated too
We thank our sponsors:
