<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hamburg_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hamburg/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenu"><DIV id="navmenubox"><UL id="nav" class="col-md-10"><LI><A href="https://2015.igem.org/Team:Hamburg">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Description">PROJECT</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Description">Overview</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Project/miRNA">microRNA 2911</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Project/PezT">Killgene PezT</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Experiments">Experiments &amp; Protocols</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Results">Results</A></LI></UL><LI><A href="https://2015.igem.org/Team:Hamburg/Parts">PARTS</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Parts">Team Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Basic_Part">Basic Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Practices">HUMAN PRACTICES</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Practices/Seminar">Seminar</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Practices/Symposium">Symposium</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Practices/Risks">Risk Assessment</A></LI></DIV><LI><A href="https://2015.igem.org/Team:Hamburg/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Modeling">MODELING</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Modeling/RNA">RNA</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Modeling/Networks">Networks</A></LI></DIV></DIV></DIV></DIV><DIV class="container" id="main"><H2>Experiments &amp; Protocols</H2><H3>Stocks and working supplies</H3><DIV class="protocol"><P><B>Chloramphenicol Stock:</B></P><P>P4 mg/ml preparation of Chloramphenicol Stock (3 in EtOH)</P><OL><LI>dissolve 34 mg of Chloramphenicol in 1 ml 100% ethanol</LI><LI>filter through a 0.22 μm filter to sterilize</LI><LI>use at 1:1000 dilution in LB or LB-Agar</LI><LI><I>comment: prepare bigger stock next time (10 ml). It’s not very exact working with 34 mg CAmp</I></LI><LI>mark as <I>CAmp</I> (top) and <I>CAmp / dd.mm.yy / 34 mg/ml</I> (side) in <I>Antibiotics</I></LI><LI>store at -20 °C</LI></OL></DIV><DIV class="protocol"><P><B>LB-Agar Plates (+ Chloramphenicol/ + Kanamycin):</B></P><OL><LI>Heat 250 ml LB-Agar </LI><LI>Cool down to ca. 50 °C </LI><LI>Add 250 μl Chloramphenicol/ 12.5 μg Kanamycin (50 µg/ml) / 25 µL Ampicillin (80 µg/mL)</LI><LI>Preparation of 20 plates under sterile conditions</LI><LI>Store at 4 °C, mark as LB <I>CAmp/Kan/Amp Plates iGEM, dd.mm.yyyy</I></LI></OL></DIV><DIV class="protocol"><P><B>Annealing Buffer:</B></P><UL><LI>0.5 M EDTA (provided by Dr. Shinichiro Komaki)</LI><LI>5 M NaCl (provided by Dr. Shinichiro Komaki)</LI><LI>1 M Tris (self-made:made in 10 mL ddH2O, pH 8.04)</LI></UL><P>for 10 mL:</P><UL><LI>EDTA 20 µL</LI><LI>Tris 100 µL</LI><LI>NaCl 100 µL</LI><LI>+ H2O → 10 mL</LI></UL></DIV><DIV class="protocol"><P><B>SOC medium: </B></P><P>(instruction “Roche, Lab FAQ”, by Dagmar Stang)</P><UL><LI>Bacto-Tryptone 2.000 g</LI><LI>Bacto-Hefe-Extract 0.500 g</LI><LI>NaCl 0.050 g</LI><LI>KCl 0.019 g</LI><LI>MgCl2 * (H2O)6 0.203 g (without (H2O)6 → 0.095 g)</LI><LI>Glucose 0.036 g</LI></UL><P>pH 7.0 with pH-Meter</P><P>→ 10 mL (fridge)</P></DIV><DIV class="protocol"><P><B>TB PIPES Buffer:</B></P><P>for 100 mL:</P><UL><LI>KCl 1.865 g </LI><LI>CaCl x 2 H20 0.22 g</LI><LI>0.5 M PIPES 2 mL </LI><LI>pH 6.7</LI><LI>MgCl2 x 2H2O 0.889 g</LI><LI>ddH2O → 100 ml</LI></UL></DIV><H3>Working instructions</H3><DIV class="protocol"><P><B>Competent Cells:</B></P><P>Reagents:</P><UL><LI>Cells </LI><LI>LB plate</LI><LI>LB medium</LI><LI>TB-PIPES buffer</LI><LI>Liquid nitrogen</LI></UL><P>Day 1:</P><OL><LI>Flame the metal inoculating loop </LI><LI>Scrape off a portion from the top of the frozen (DH5-alpha) cells </LI><LI>Streak it onto the LB plate</LI><LI>Put the stock back to -80 °C immediately</LI><LI>Leave the plates for 5 minutes and place them upside down in the 37 °C incubator overnight</LI></OL><P>Day 2:</P><OL><LI>Pick a single colony into 3 mL of LB medium</LI><LI>Inoculate the culture over night at 37 °C with shaking at 200 rpm</LI><LI>Transfer 0.5 mL of the overnight preculture into 50 mL LB medium (1-2 L Erlenmeyer flask) </LI><LI>Let the culture grow to OD600 = 0.6 at 18 °C with shaking at 200 rpm</LI><LI>Transfer the culture into a 50 mL Falcon tube and leave for 10 min on ice</LI><LI>Spin down cells at 3500 rpm for 8 min at 4 °C</LI><LI>Resuspend in 1/3 volume TB-PIPES buffer (17 mL for 50 mL original culture)</LI><LI>Leave for 10 min on ice</LI><LI>Spin down cells at 3500 rpm for 8 min at 4 °C</LI><LI>Resuspend in 1/12.5 volume ( 4 mL for 50 mL original culture)</LI><LI>Add 7% DMSO (sterile)</LI><LI>Leave for 10 min on ice</LI><LI>Aliquot 50 µL in pre-chilled Eppendorf tubes and freeze by immersion in liquid nitrogen</LI><LI>Store at −80 ° C</LI></OL></DIV><DIV class="protocol"><P><B>Transformation Efficiency ‘Competent Cell Test Kit’:</B></P><P><U><A href="http://parts.igem.org/Help:Transformation_Efficiency_Kit" rel="nofollow">(iGEM Help: Transformation Efficiency Kit)</A></U></P><OL><LI>spin down DNA (0.5, 5, 10, 20 &amp; 50 pg/µL RFP Construct BBa_J04450, pSB1C3) from Competent Cell Test Kit ( 30 sec, 10.000 rpm)</LI><LI>thaw competent cells on ice, label one 2 ml tube for each concentration + pre-chill</LI><LI>1 µl of DNA in each tube, respectively</LI><LI>add 50 µl of competent cells to each tube, incubate for 30 min on ice</LI><LI>pre-heat (42 °C), heat shock for 1 min</LI><LI>incubate for 5 min on ice</LI><LI>add 200 µl SOC media, incubate for 2 h at 37 °C</LI><LI>pipet 20 µl on each plate (triplets), respectively </LI><LI>incubate overnight at 37 °C</LI><LI>count colonies, calculate transformation efficiency</LI></OL></DIV><DIV class="protocol"><P><B>Annealing:</B></P><OL><LI>get the amount of DNA from the delivery note</LI><LI>dilute both oligos to 100 μM</LI><LI>anneal in water bath at 94 °C for 10 min</LI><LI>cool to room temperature in water bath over 60 min</LI><LI>store solution at 4 °C in fridge</LI></OL></DIV><DIV class="protocol"><P><B>Control of Annealing:</B></P><UL><LI>in agarose gel (1%)</LI><LI>1 µL loading dye</LI><LI>9 µL sample (or appropriate amount &amp; add H20 up to 9 µL)</LI></UL></DIV><DIV class="protocol"><P><B>Make agarose gel:</B></P><UL><LI>0.5 g Agarose</LI><LI>50 mL TAE-buffer (gel room)</LI></UL><OL><LI>Heat in microwave until agarose is dissolved</LI><LI>Transfer into gel slide, tip pipet into EtBr, then into gel</LI><LI>Cool for 30 minutes</LI></OL></DIV><DIV class="protocol"><P><B>Restriction:</B></P><P>20 µL pSB-1C3 (500 ng)</P><P>1 µL EcoRI</P><P>1 µL PstI</P><P>3 µL FD Buffer</P><P>5 µL ddH2O</P><P>20 minutes at 37 °C, following clean up with kit</P></DIV><DIV class="protocol"><P><B>Backbone dephosphorylation:</B></P><UL><LI>10 µL pSB1C3 (cut)</LI><LI>1 µL FastAP</LI><LI>4 µL FastAP Buffer</LI></UL><P>→ 37 °C for 30 minutes</P><P>→ purify with PCR clean up kit</P></DIV><DIV class="protocol"><P><B>Ligation:</B></P><P>1 µL pSB-1C3 vector (23 ng)</P><P>1 µL /1.5 µL GroEl-Insert (3.5 ng) / miRNA-Insert (3.3 ng)</P><P>2 µL T4 Buffer</P><P>1 µL T4 Ligase</P><P>5 µL ddH20 ( 6 µL for control)</P><P>15 min at room temperature, then 10 min at 65 °C</P></DIV><DIV class="protocol"><P><B>Transformation:</B></P><UL><LI>10 µL ligation product</LI><LI>50 µL competent cells</LI></UL><OL><LI>30 min on ice</LI><LI>1 min Heatshock at 42 °C</LI><LI>15 min on ice</LI><LI>add 200 µL SOC-Medium</LI><LI>2 h at 37 °C on rotator </LI><LI>plate out on appropriate plates (C-Amp/ Kan)</LI><LI>incubate over night at 37 °C</LI></OL></DIV><DIV class="protocol"><P><B>Pick colony:</B></P><UL><LI>Inoculate each picked colony into 5 mL of liquid LB-(CAmp/ respective antibioticum)-medium, respectively</LI><LI>Tubes in the 37 °C incubation room, on rotator!!</LI></UL></DIV><DIV class="protocol"><P><B>Minipreparation:</B></P><UL><LI>use Geneaid Mini Plasmid Preparation Kit</LI><LI>store samples in freezer</LI></UL></DIV><DIV class="protocol"><P><B>Glycerol stocks:</B></P><P>from liquid culture</P><UL><LI>500 µL (50% Glycerol, 50% H2O)</LI><LI>500 µL BW29655 or DH5a</LI><LI>→ -80°C</LI></UL></DIV><DIV class="protocol"><P><B>Preparation for sequencing:</B></P><UL><LI>Primer concentration: 100 µM→ Concentration needed: 5 µM (dilute at 1:20)</LI><LI>take 50 nm of sample mix with 5 µL of ( 5 µM primer, only forward)</LI></UL></DIV></DIV><DIV class="container"><P>We thank our sponsors:</P></DIV></DIV></BODY></HTML>