Team:Hamburg/Results
HOME
TEAM
PROJECT
Overview
microRNA 2911
Killgene PezT
Experiments & Protocols
Results
PARTS
Team Parts
Basic Parts
Composite Parts
NOTEBOOK
ATTRIBUTIONS
COLLABORATIONS
HUMAN PRACTICES
Seminar
Symposium
Risk Assessment
SAFETY
MODELING
RNA
Networks
Project Results
miRNA2911 ( BBa_K1860700 )
Aim
Our goal was to use synthetic biology to produce miRNA2911, which is endogenously expressed in Lonicera japonica.
Results
We were able to construct a Biobrick (BBa_K1860700) that contains the anti-viral miRNA2911. The Biobrick was validated by sequencing.
Design of Oligos
We started our project by designing oligos of the miRNA2911. It was really important to consider that the desired sequence had to be in line with the iGEM standard, meaning that the resulting Biobrick can be used for 3A-Assemblies later on. Hence the miRNA2911 is flanked by the appropriate restriction sites (EcoRI, XbaI, SpeI, PstI). Depicted in Figure 1 are the designed sequences.
Annealing of Oligos
In the next step the oligos needed to be annealed. We used the standard ‘Annealing Protocol’ and the figure depicts the results of this annealing. The appropriate bands of the resulted miRNA2911 insert are at 61 bp.
Ligation
After confirmation of successful annealing we proceeded with the ligation of the insert (miRNA2911) into the standard backbone pSB1C3. For this purpose the linearized backbone (pSB1C3) was cut with EcoRI and PstI and the insert was ligated into the vector. The insert does not need to be cut, as it was designed to form overhangs after annealing.
Shown in the following figure is the verification of ligation. The ligation was verified positively, as the ligated vector runs at 2050 bp and the control sample, only containing pSB1C3, runs at around 2000 bp, as seen in the agarose gel.
Following, the ligated product was transformed into the E.coli strain DH5α. Single colonies were picked, liquid cultures were grown overnight and plasmids were prepped.
Verification
Colony PCR
To be able to verify, if our E.coli actually carry our desired plasmid, a colony PCR was performed. Therefore an aliquot of the liquid culture was plated out again and grown overnight. Sixteen different colonies were tested using the standard sequencing primers. The results of six of these PCRs are shown in the following figure. The gel verifies that the E.coli are carrying our desired vector containing the miRNA2911.
Sequencing
Finally we verified our new Biobrick by sequencing.
constitutive promoter and miRNA (BBa_K1860702)
We combined the generated BioBrick (BBa_K1860700), containing the miRNA2911, with a constitutive promoter (BBa_J23102), that was provided with the Standard Distribution, by 3A-Assembly.  This is necessary to further on be able to test, if miRNA2911 is:
actually expressed
stable in E.coli and
how it is actually folded
Therefore we used the standard 3A-Assembly protocol. The constitutive promoter (BBa_J23102) was provided in pSB1A3, our constructed Biobrick (BBa_K1860700) is carried in pSB1C3, hence the assembly was performed into pSB1K3.
GroEL - heat-inducible promoter (BBa_K1860701)
Design of Oligos
GroEl is a heat-inducible promoter and we started by designing oligos of GroEL. The sequence was kindly provided by Prof. Ignatova. Just like for the design of the miRNA2911 here we also needed to reconsider that the desired sequence containing GroEL had to be in line with the iGEM standard. Hence also the GroEL promoter is flanked by the appropriate restriction sites (EcoRI, XbaI, SpeI, PstI).
Annealing of Oligos
In the next step, the oligos needed to be annealed. We used the standard ‘Annealing Protocol’ and the following figure depicts the results of this annealing. The appropriate bands of the resulted GroEL insert are at 91 bp.
Ligation
After confirmation of successful annealing we proceeded with the ligation of the insert (GroEL) into the standard backbone pSB1C3. For this purpose the linearized backbone (pSB1C3) was cut with EcoRI and PstI and the insert was ligated into the vector. The insert did not need to be cut, as it was designed to form overhangs after annealing.
Due to the submission deadline, the BioBrick plasmid was sent to the registry without sequencing.
pezT
Aim
Our aim was to generate a BioBrick containing the PezT gene for autolysis as shown in “project → kill gene”. We wanted to amplify the gene from a plasmid provided by Prof. Meinhart (Robert Koch Institute, Heidelberg, Germany) via PCR. To get rid of illegal restriction sites, such as PstI, we planned a mutagenesis PCR. Finally the PezT gene should get ligated into the pSB1C3 backbone.
Results
We designed primers for mutagenesis and started first PCR steps but had to stop due to the submission deadline.
We thank our sponsors:
