<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hamburg_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hamburg/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navmenu"><DIV id="navmenubox"><UL id="nav" class="col-md-10"><LI><A href="https://2015.igem.org/Team:Hamburg">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Description">PROJECT</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Description">Overview</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Project/miRNA">microRNA 2911</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Project/PezT">Killgene PezT</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Experiments">Experiments &amp; Protocols</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Results">Results</A></LI></UL><LI><A href="https://2015.igem.org/Team:Hamburg/Parts">PARTS</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Parts">Team Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Basic_Part">Basic Parts</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Practices">HUMAN PRACTICES</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Practices/Seminar">Seminar</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Practices/Symposium">Symposium</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Practices/Risks">Risk Assessment</A></LI></DIV><LI><A href="https://2015.igem.org/Team:Hamburg/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Hamburg/Modeling">MODELING</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Modeling/RNA">RNA</A></LI><LI><A class="liste" href="https://2015.igem.org/Team:Hamburg/Modeling/Networks">Networks</A></LI></DIV></DIV></DIV></DIV><DIV class="container" id="main"><H2>Notebook</H2><DIV class="entry"><H3 class="date">Monday, 22nd June 2015</H3><P class="names">Henning, Nikolas</P><UL><LI>Notion: linearized plasmid backbones have been stored at room temperature for eight weeks. Maybe need to resupply.</LI><LI>P4 mg/ml preparation of Chloramphenicol Stock (3 in EtOH)
        </LI><LI>dissolve 34 mg of Chloramphenicol in 1 ml 100% ethanol</LI><LI>filter through a 0.22 μm filter to sterilize</LI><LI>use at 1:1000 dilution in LB or LB-Agar</LI><LI>comment: maybe prepare bigger stock next time (10 ml). It’s not very exact working with 34 mg CAmp</LI><LI>marked as CAmp (top) and CAmp / 22.06.15 / 34 mg/ml (side) in Antibiotics</LI><LI>stored at -20 °C</LI><LI>(<A href="http://www.cytographica.com/lab/solutions/Chlor.htm">http://www.cytographica.com/lab/solutions/Chlor.htm</A>)</LI></UL><LI>Preparation of LB-Agar plates with Chloramphenicol (CAmp)
        </LI><LI>heat 250 ml LB-Agar, cool down to ca. 50 °C and add 250 μl Chloramphenicol</LI><LI>Preparation of 20 plates under sterile conditions</LI><LI>stored at 4 °C, marked as CAmp Plates iGEM, 22.06.2015</LI></DIV><DIV class="entry"><H3 class="date">Tuesday, 23rd June 2015</H3><P class="names">Dana, Nikolas</P><UL><LI>‘Competent Cell Test Kit’
        </LI><LI>spin down DNA (0.5, 5, 10, 20 &amp; 50 pg/µL RFP Construct BBa_J04450, pSB1C3) from Competent Cell Test Kit ( 30 sec, 10.000 rpm)</LI><LI>thaw competent cells XL1 blue Tetracyclin resistent) on ice, label one 2 ml tube for each concentration ( 6 (five concentrations plus 1 control) + 1 tube for control of the backbone ‘’) and pre-chill
          </LI><LI>1 µl of DNA in each tube, respectively
          </LI><LI>add 50 µl of competent cells to each tube, incubate for 30 min on ice</LI><LI>pre-heat water bath (42 °C), heat shock for 1 min</LI><LI>incubate for 5 min on ice
          </LI><LI>add 200 µl LB media, incubate for 1 h at 37 °C
          </LI><LI>pipet 20 µl on each plate (triplets, control only 1, 2 for backbone), respectively (add 50 µl LB media for easier distributing)</LI><LI>incubate overnight at 37 °C
          </LI></UL>The 50 pg/µL sample was forgotten on ice and the trafo will be repeated on wednesday

      </DIV><DIV class="entry"><H3 class="date">Wednesday, 24th June 2015</H3><P class="names">Heiko, Jannis, Julian, Nikolas</P><UL><LI>The trafo failed: there are no colonies on all plates -&gt; repetition necessary
      </LI><LI>‘Competent Cell Test Kit’
        </LI><LI>repetition of the trafo using the 50 pg/µL solution. the 3 plates are stored at 37 °C in room 1.069</LI><LI>Use of DH5α instead of XL1 Blue in the Transformation, following the protocol used on tuesday</LI></UL><LI>Preparation of 10 mL Annealing buffer (10mM Tris, pH7.5-8.0, 50mM NaCL, 1mM EDTA)</LI><LI>Preparation of 26 LB-Agar-CAmp plates (34µg/mL CAmp)</LI><LI>to do on Wednesday (left over of protocol)
        </LI><LI>Count the number of colonies on a light field or a dark background, such as a lab bench. Use the following equation to calculate your competent cell efficiency. If you've done triplicates of each sample, use the average cell colony count in
            the calculation.
            </LI><LI>(colonies on plate) / ng of DNA plated x 1000 ng/µg</LI><LI>Note: The measurement &quot;ng of DNA plated&quot; refers to how much DNA was plated onto each agar plate, not the total amount of DNA used per transformation. You can calculate this number using the following equation:</LI><LI>1 µL x concentration of DNA (refer to vial) x (volume plated / total reaction volume)
              </LI></DIV><DIV class="entry"><H3 class="date">Thursday, 25th June 2015</H3><P class="names">Heiko, Maurice, Vincent</P><UL><LI>Counting the colonies in the trafos from Wednesday and calculating the transformation efficiency
        </LI><LI>Max. efficiency: ~ 3e+7 cfu / μg DNA </LI><LI>around one order of magnitude lower than iGEM standard</LI><LI>sealed the plates and put into 4 °C fridge</LI></UL><LI>Picked one colony from each plate and inoculated each into 10 mL of liquid LB-CAmp-medium
        </LI><LI>remaining LB medium in the 4 °C fridge</LI><LI>tubes in the 37 °C incubation room</LI><LI>Annealing of the miRNA2911 oligos
        </LI><LI>got the amount of DNA from the delivery note</LI><LI>diluted both oligos to 100 μM</LI><LI>took 500 μL each and annealed in water bath at 94 °C for 10 min</LI><LI>cooling to room temperature in the water over 60 min</LI><LI>stored the solution at 4 °C in the fridge</LI></DIV><DIV class="entry"><H3 class="date">Friday, 26th June 2015</H3><P class="names">Henning, Nikolas, Sonja</P><UL><LI>Mini-Prep of 10 ml cultures from 25.06. → Trafo of Backbones (Trafo efficiancy-kit)
        </LI><LI>→ Protocol used: High speed plasmid medi kit (geneaid)</LI><LI>1C3-50A: 0.133 µg/µl, V = 48 µl</LI><LI>1C3-50A: 0.133 µg/µl, V = 49 µl</LI><LI>1C3-BBc: 0.225 µg/µl, V = 49 µl</LI><LI>1C3-BBc: 0.198 µg/µl, V = 49 µl</LI><LI>stored in freezer</LI></UL><LI>Annealing of GroEL oligos (old and new oligos)
        </LI><LI>total of 4 samples
            </LI><LI>note: the old GroEL-oligos were diluted with a (very) wrong amount of buffer and shouldn’t be used anymore</LI><LI><A href="https://www.addgene.org/plasmid-protocols/annealed-oligo-cloning/">https://www.addgene.org/plasmid-protocols/annealed-oligo-cloning/</A></LI><LI>used 2 g each in 50µl Buffer total</LI><LI>Heatblock at 95 °C for 3 min and cool down slowly for approx 1 h</LI><LI>Stored as “GroEL Insert” in freezer</LI><LI>Preparation of Agar-Plates with Kanamycin
        </LI><LI>16 Plates</LI><LI>Kan (50 µg/ml)</LI><LI>Stored in Fridge (“26.06 LB Agar Kan”)</LI></DIV><DIV class="entry"><H3 class="date">Tuesday, 30th June 2015</H3><P class="names">Artur, Dana, Henning, Kira, Nikolas</P><UL><LI>miRNA
        </LI><LI>Control of annealing → c=19.55 ng/µL</LI></UL><LI>Agarplates
        </LI><LI>without resistance for DH5ɑ</LI><LI>DH5ɑ
        </LI><LI>distribution on agarplates without antibiotic → 37 °C</LI><LI>GroEL
        </LI><LI>first annealing, no DS-DNA → repeated with waterbath (95°C)</LI><LI>0.092 µg/µL → 4.6 µg (after 50x)</LI></DIV><DIV class="entry"><H3 class="date">Wednesday, 1st July 2015</H3><P class="names">Artur, Henning, Julian, Nikolas</P><UL><LI>control of miRNA2911 and old GroEL-Annealing in Agarose-gel
        </LI><LI>both positive</LI></UL><LI>New annealing of GroEl-Promotor with full volume (300 µl each)</LI><LI>Prepare start culture of DH5α for main culture on thursday
        </LI><LI>stored in 37°C room</LI><LI>three tubes stored in 37°C room</LI><LI>Preparation of BW29655 Coli on agar plates for start culture on thursday</LI><LI>Restriction of vector backbone (1C3) with EcoRI and PstI</LI><LI>Ligation of Vector (1C3) with GroEL-Insert (first annealing from last week)</LI><LI>protocols are in the hardcopy labbook</LI></DIV><DIV class="entry"><H3 class="date">Thursday, 2nd July 2015</H3><P class="names">Artur, Dana, Henning, Kira, Nikolas</P><FIGURE class="right"><FIGCAPTION>Casting the agarose gel</FIGCAPTION></FIGURE><UL><LI>Repetition of annealing control in gel
        </LI><LI>miRNA2911 positive, GroEL negative (false sample)</LI></UL><LI>Ligation of GroEL in 1C3-Backbone</LI><LI>Repetition of annealing control in gel</LI><LI>Preparation of main culture DH5α</LI><LI>storage of BW29655 plates in fridge for miniprep on monday</LI></DIV><DIV class="entry"><H3 class="date">Friday, 3rd July 2015</H3><P class="names">Heiko, Maurice, Vincent</P><UL><LI>measured the OD600 of the competent DH5α cells =&gt; 0,44</LI><LI>prepared new Annealing Buffer
        </LI><LI>0.5 M EDTA (Shini)</LI><LI>5 M NaCl (Shini)</LI><LI>1 M Tris (self-made:made in 10 mL ddH2O, pH 8.04)</LI></UL><LI>Annealing Buffer:
        </LI><LI>EDTA 20 µL</LI><LI>Tris 100 µL</LI><LI>NaCl 100 µL</LI><LI>+ H2O </LI><LI>→ 10 mL</LI><LI>diluted new FW miRNA 573 µL (73 µL still in the freezer, orange box)</LI><LI>diluted new REV miRNA 769 µL (269 µL still in the freezer, orange box)</LI><LI>Annealing (94°C) (water quench, 10 min, then cooling at RT)
        </LI><LI>→ product in freezer</LI><LI>OD600 = 0.5</LI><LI>gone on working with cells from 02/07/15
        </LI><LI>protocol “Kompetente Zellen” (Shini)</LI><LI>end: 13:50h, cells at -80°C</LI></DIV><DIV class="entry"><H3 class="date">Monday, 6th July 2015</H3><P class="names">Dana, Heiko, Henning, Sonja</P><UL><LI>measured the concentration of the miRNA-Annealing from friday → 2.2 µg/µl</LI><LI>Ligation of the miRNA-Insert in pSB-1C3 (cut with EcoRI &amp; PstI)</LI><LI>forgot the ligation control again -.-</LI><LI>efficiency kit used for trafo-control
        </LI><LI>competent DH5α + 0.5 - 50 ng RFP construct</LI></UL><LI>Trafo of GroEL (2x) and miRNA in DH5α</LI><LI>Preparation of start culture with BW29655
        </LI><LI>4x 5 ml without any AB!</LI><LI>Stored at 37°C in shaker</LI></DIV><DIV class="entry"><H3 class="date">Tuesday, 7th July 2015</H3><P class="names">Artur, Heiko, Vincent</P><UL><LI>Preparation of SOC-medium (instruction “Roche, Lab FAQ”, by Dagmar)
        </LI><LI>Bacto-Tryptone 2.000 g</LI><LI>Bacto-Hefe-Extract 0.500 g</LI><LI>NaCl 0.050 g</LI><LI>KCl 0.019 g</LI><LI>MgCl2 * (H2O) 6 0.203 g (without (H2O)6 -&gt; 0.095 g)</LI><LI>Glucose 0.036 g</LI><LI>pH 7.0 with pH-Meter</LI><LI>10 mL (fridge)</LI></UL><LI>Transformation with Competent Cell Test Kit (5 diff. conc.: 1.5, 5, 10, 25, 50 pg/µL) in 5 aliquots of DH5α
        </LI><LI>each with 50 µL DH5α + 1 µL DNA</LI><LI>Incubation on ice (30 min)</LI><LI>Heat shock (water, 42°C)</LI><LI>5 min on ice (rather 15 min) </LI><LI>200 µL SOC in tubes</LI><LI>Incubation for 2 h at 37°C with rotation</LI><LI>Plating (2 plates each: 20 µL + 200 µL) → 10 plates</LI><LI>Incubation 37°C over night </LI><LI>2 Glycerol stocks from BW29655 liquid cultures
        </LI><LI>500 µL (50% Glycerol, 50% H2O)</LI><LI>500 µL BW29655</LI><LI>→ - 80°C</LI><LI>Pooling of the liquid cultures of BW29655 in 200 mL LB medium (sterile)
        </LI><LI>→ 37°C</LI></DIV><DIV class="entry"><H3 class="date">Wednesday, 8th July 2015</H3><P class="names">Heiko, Henning, Nikolas, Sonja</P><UL><LI>Preparation of Agar plates with Chloramphenicol (CAMP, 8.7.2015)</LI><LI>Transformation of miRNA-Ligation and GroEL-Ligation product as well as PezT-Plasmid (Kan resistance!!!) in DH5α
        </LI><LI>each with 50 µL DH5α + 1 µL DNA</LI><LI>Incubation on ice (30 min)</LI><LI>Heat shock (water, 42°C)</LI><LI>5 min on ice (rather 15 min) </LI><LI>200 µL SOC in tubes</LI><LI>Incubation for 2 h at 37°C with rotation</LI><LI>Plating (2 plates each: 20 µL + 200 µL) → 10 plates (using kan resistance for PezT-Plasmid)</LI><LI>Incubation 37 °C over night </LI></UL><LI>Preparation of new start cultures of BW29655
        </LI><LI>3x 5 ml “iGEM a,b, c BW”</LI><LI>stored at 37°C over night</LI></DIV><DIV class="entry"><H3 class="date">Thursday, 9th July 2015</H3><P class="names">Artur</P><UL><LI>Sealing of the plates with the transformation and storing in the fridge</LI><LI>Storing the BW29655 liquid cultures in the fridge</LI></UL></DIV><DIV class="entry"><H3 class="date">Tuesday, 14th July 2015</H3><P class="names">Maurice, Vincent</P><FIGURE class="right"><FIGCAPTION>Picked liquid cultures of miRNA and pezT</FIGCAPTION></FIGURE><UL><LI>Evaluation of the transformation from July 8th
        </LI><LI>colony counts in the physical labjournal</LI></UL>Preparation of ca. 170 mL LB+Kan medium (30 µg/mL Kan)
      <LI>stored in the fridge</LI><LI>Picked one miRNA colony and 3 pezT colonies
        </LI><LI>transferred into 5 mL of LB (+CAmp/Kan) each</LI><LI>incubation at 37 °C over night</LI><LI>Transfer of 500 µL BW29655 culture into 50 mL LB for competent cell protocol
        </LI><LI>incubation at RT and 200 rpm over night</LI></DIV><DIV class="entry"><H3 class="date">Wednesday, 15th July 2015</H3><P class="names">Dana, Julian, Kira</P><UL><LI>OD of BW29655 culture was too high (4), stored in the freezer ; again 500 µL BW29655 culture were transferred into 50 mL LB for competent cell protocol</LI><LI>over night culture of miRNA and 3 pezT colonies were prepped, stored in freezer in the second box, blue label on top and blue label on each eppi
    </LI></UL><LI>First cell cultures checked again at 5 pm OD600: 5</LI><LI>We stopped the growth of the new cells via freezing (in the Freezer, with the date of today),</LI><LI>please restart growing step in the 37°C room.</LI></DIV><DIV class="entry"><H3 class="date">Thursday, 16th July 2015</H3><P class="names">Henning, Nikolas, Heiko</P><UL><LI>digestion control of prepped miRNA and pezT from wednesday. All controls were positive.</LI><LI>Maybe we pipetted enzyme to all the controls</LI><UL><LI>digested with EcoRV and XhoI</LI></UL><LI>Repetition of the digestion controls</LI><UL><LI>digested with EcoRV and XhoI</LI></UL><LI>digest of vector plasmid (prepped from trafo in first week)</LI><UL><LI>digested with EcoRI and SpeI</LI><LI>again the control was positive (3 fragments)</LI><LI>isolated and prepped the supposed fragment</LI><LI>control of the prepped fragment in gel together with repetition of the control</LI><LI>used two controls, one with alternative buffers and loading dye, one as before</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Friday, 17th July 2015</H3><P class="names">Henning, Nikolas, Heiko</P><UL><LI>Glycerol-Stock (50%)</LI><UL><LI>sterile filtration</LI></UL><LI>pezT-Backstock</LI><UL><LI>for each (A,B,C), in -80°C</LI></UL><LI>New Pipes</LI></UL></DIV><DIV class="entry"><H3 class="date">Monday, 20th July 2015</H3><P class="names">Nikolas, Sonja</P><UL><LI>TB PIPES Buffer</LI><LI>für 100 mL:</LI><UL><LI>1.865 g KCl</LI><LI>0,22 g CaCl x 2 H20</LI><LI>2 mL 0.5 M PIPES</LI></UL><LI>Auf pH 6.7 bringen</LI><UL><LI>0,889 g MnCl2 x 2H2O</LI><LI>ddH2O ad 100 ml</LI></UL></UL><UL><LI>Restriktion des 1C3 BB:</LI><UL><LI>4 µL pSB-1C3 Vector</LI><LI>1 µL EcoRI</LI><LI>1 µL PstI</LI><LI>2 µL FD buffer</LI><LI>2 µL ddH20</LI></UL><LI>1 h 37° C stored in fridge</LI></UL></DIV><DIV class="entry"><H3 class="date">Tuesday, 21st July 2015</H3><P class="names">Nikolas</P><UL><LI>Ligation GroEl in 1C3:</LI><UL><LI>2 µL T4 Ligase buffer</LI><LI>1 µL T4 Ligase</LI><LI>9 µL 1C3 BB</LI><LI>2 µL “i2” insert</LI><LI>6 µL ddH20</LI></UL><LI>DNA concentration in GroEL tubes</LI><UL><LI>GroEL 1:         2 ng/µL</LI><LI>i:             2.5 ng/µL</LI><LI>GroEL Insert:  171 ng/µL</LI></UL><LI>Kompetente Zell Kultur anlegen</LI><UL><LI>OD600= 0.066 (14:30)</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Wednesday, 22nd July 2015</H3><P class="names">Nikolas, Henning, Sonja</P><UL><LI>New GroEl ligation (see Tuesday)
</LI></UL></DIV><DIV class="entry"><H3 class="date">Thursday, 23rd July 2015</H3><P class="names">Dana, Henning</P><UL><LI>Make BW29655 competent</LI><UL><LI>OD measurement of  BW29655 → OD600 = 0.644 → broke centrifuge :(</LI></UL><LI>Cut pSB_1C3 with EcoRI &amp; PstI</LI><UL><LI>20 µL pSB-1C3 Vector</LI><LI>1 µL EcoRI</LI><LI>1 µL PstI</LI><LI>3 µL FD buffer</LI><LI>5 µL ddH20</LI></UL><LI>20 minutes at 37 °C, clean with Kit → measure concentration : 23 ng/µL (460 ng)</LI><LI>Ligation</LI><UL><LI>1      µL pSB-1C3 (23 ng)</LI><LI>1/ 1.5 µL GroEl-Insert (35 ng) / miRNA (33 ng)</LI><LI>2      µL T4 Buffer</LI><LI>1      µL T4 Ligase</LI><LI>5      µL ddH2O</LI></UL><LI>15 minutes at room temperature, then 10 min at 65 °C</LI><LI>Transformation</LI><UL><LI>1 µL Ligation Product</LI><LI>50 µL DH5α competent cells</LI><LI>30 minutes on ice, 1 min heat shock (42 °C), 5 min on ice</LI><LI>add 200 µL SOC media, 2 h at 37 °C</LI><LI>plate out on C-amp-plates</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Friday, 24th July 2015</H3><P class="names">Heiko, Vincent</P><FIGURE class="right"><FIGCAPTION>Freezing the competent cells in liquid nitrogen</FIGCAPTION></FIGURE><UL><LI>Trafo results:</LI><LI>2 measurements OD600 ( 11:00, 12:30) = 0.00</LI><LI>use of the competent cells from 17/07 “BW 2. Versuch”OD600 = 0.546</LI><LI>Protocol “Kompetente Zellen”</LI><LI>use of the centrifuge in the 3rd floor (Rm. 3.004)</LI><LI>N2 (l) (80°C) at 15:00</LI></UL></DIV><DIV class="entry"><H3 class="date">Monday, 27th July 2015</H3><P class="names">Nikolas, Sonja</P><UL><LI>miRNA-mini preparation</LI><UL><LI>3 tubes (source: 200 µL plate)</LI><LI>37 °C</LI></UL><LI>GroEl ligation (NEW)</LI><UL><LI><B>1) cut pSB1C3 with EcoRI and PstI</B></LI><LI>1 µL    EcoRI</LI><LI>1 µL    PstI</LI><LI>2 µL    pSB1C3</LI><LI>1 µL    Buffer</LI><LI>5 µL    a. bidest.</LI></UL><LI>→ 37 °C for 15 minutes</LI><LI>→ 65 °C for 10 minutes heatshock</LI><LI><B>2) Backbone dephosphorylation</B></LI><UL><LI>10 µL    pSB1C3 (cut)</LI><LI>1 µL    FastAP</LI><LI>4 µL    FastAP Buffer</LI></UL><LI>→ 37 °C for 30 minutes</LI><LI>→ purify with PCR clean up kit</LI><LI><B>3) Ligation</B></LI><UL><LI>20 µL    pSB1C3 (dephosphorylated, cut)</LI><LI>10 µL    Insert</LI><LI>8  µL    T4 Buffer</LI><LI>4  µL    T4 ligase</LI></UL><LI>Transformation Efficiency Kit für BW_29965 (see iGEM protocol)</LI></UL></DIV><DIV class="entry"><H3 class="date">Tuesday, 28th July 2015</H3><P class="names">Dana</P><UL><LI>miRNA preparation:</LI><UL><LI>OD was too low for preparation (Probes were not on the shaker)</LI><LI>→ Julian will pick new colonies on wednesday</LI><LI>→ Preparation on Thursday</LI></UL><LI>GroEl-Ligation → Trafo</LI></UL><LI>10 µL Ligation + 50 µL DH5α</LI><LI>30 minutes on ice, 1 min heat shock (42 °C), 5 min on ice</LI><LI>add 200 µL SOC media</LI><LI>incubate for 2 h at 37 °C</LI><LI>plate out on C-amp-plates</LI><LI>Transformation Efficiency- BW29655</LI><UL><LI>→ all but one were not turned upside down for incubation at 37 °C over night</LI><LI>Explanation for DNA per Plate:</LI><LI>(0.5;5;10;20;50) x (20 or 200 µL-volume plated/ 250 µL -total reaction volume (50 µL cells + 200 µL SOC)) x 1 ng/1000 pg</LI></UL><LI>Efficiency:</LI><UL><LI>(colonies on plate) / ng of DNA plated x 1000ng/µg</LI><LI>for 20 µL-Probes</LI><LI>10 pg/µL → 1/0.0008 cfu/ng x 1000 ng/µg → 1.25 e+3 cfu/µg</LI><LI>20 pg/µL → 1/0.0016 cfu/ng x 1000 ng/µg → 6.25 e+5 cfu/µg</LI><LI>50 pg/µL → 3/0.004 cfu/ng x 1000 ng/µg → 7.5 e+2cfu/µg</LI><LI>for 200 µL-Probes</LI><LI>5  pg/µL → 10/0.0004 cfu/ng x 1000 ng/µg → 2.5 e+7cfu/µg</LI><LI>10 pg/µL → 12/0.008 cfu/ng x 1000 ng/µg → 1.5 e+7 cfu/µg</LI><LI>20 pg/µL → 125/0.016 cfu/ng x 1000 ng/µg → 1.6 e+7 cfu/µg</LI><LI>50 pg/µL → 70/0.04 cfu/ng x 1000 ng/µg → 1.75 e+7 cfu/µg</LI></UL></DIV><DIV class="entry"><H3 class="date">Wednesday, 29th July 2015</H3><P class="names">Julian</P><UL><LI>Picked the colonies, named them and put them into 3 glasses each filled with 5mL LB+CAmp with a colony inside.</LI><LI>Counted the colonies on the prepared agar plates: All were white.</LI><UL><LI>Control:             no colonies</LI><LI>20µL :               no colonies</LI><LI>200µL, 1st pick:     23-26 colonies, and about 10 really little ones</LI><LI>200µL, 2nd pick:     27-30 colonies, and about 10 really little ones</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Thursday, 30th July 2015</H3><P class="names">Nikolas, Sonja</P><UL><LI>miRNA plasmid isolation</LI><LI>NanoDrop</LI><UL><LI>c(04)= 0.035 µg/µL</LI><LI>c(05)= 0.038 µg/µL</LI><LI>c(06)= 0.068 µg/µL</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Friday, 31st July 2015</H3><P class="names">Kira, Vincent</P><UL><LI>Preparation of the GroEL-minis</LI><UL><LI>stored in freezer</LI><LI>concentration: (1)= 118 ng/µL, (2)= 178 ng/µL, (3)= 233 ng/µL</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Monday, 3rd August 2015</H3><P class="names">Heiko, Julian</P><UL><LI>GroEL (picked colonies)</LI><UL><LI>instruction “Presto Mini Plasmid Kit”</LI><UL><LI>at step 7 method 2 used for higher DNA concentration</LI><LI>each time 3 min absorption time</LI><LI>last elution: centrifugation for 4 min at 20°C</LI></UL></UL><LI>Results from the picked miRNA (29.07.15)</LI><UL><LI>Resistance with BW29655 in 5µL each:</LI><UL>
    → CAmp, Kan, Without
    → 37°C over night
	</UL></UL><LI>Preparation of new agar plates with CAmp resistance</LI></UL></DIV><DIV class="entry"><H3 class="date">Tuesday, 4th August 2015</H3><P class="names">Henning</P><UL><LI>Redistribution of some samples from 37°C to freezer and the other way round</LI></UL></DIV><DIV class="entry"><H3 class="date">Wednesday, 5th August 2015</H3><P class="names">Henning, Heiko, Vincent</P><FIGURE class="right"><FIGCAPTION>Picking the colonies</FIGCAPTION></FIGURE><UL><LI>Plating of pezT (14.07.15) (l) A, B, C on plates (200 µL each)
	<UL>
→ 37 °C
</UL></LI><LI>Picking from plates: The 5 BBas, miRNA (23.07.15), GroEL
</LI><LI>pipette-tips in 5 mL LB+CAmp each
</LI><LI>two preparations each
<UL>
    → 37 °C
	</UL></LI></UL><LI>Mutagenesis-PCR
</LI><LI>“PCR-Phusion protocol” (NEB)
</LI><LI>primers diluted to 10 µM
</LI><LI>charging the PCR (middle PCR device block 2, program 42 in directory “Katja”)
</LI><LI>supporting information:
</LI><LI>dNTPs from Katja, primers solved in Annealing buffer because lyophilized
</LI><LI>concentration with NanoDrop (volume: 4 µL) : 41 ng/µL
</LI></DIV><DIV class="entry"><H3 class="date">Thursday, 6th August 2015</H3><P class="names">Dana</P><UL><LI>PCR-Mutagenesis (1&amp;2), check on gel, two different dilutions</LI><LI>10 µL: 9 µL sample + 1 µL loading dye</LI><LI>10 µL: 2 µL sample + 7 µL water + 1 µL loading dye</LI><LI>new PCR (3)</LI><LI>Nanodrop:     PCR 1: 148 ng/µL
    PCR 2: 175 ng/µL
    PCR 3:   68 ng/µL</LI><LI>PCR-fragment 3 not detectable</LI><LI>light-inducable promotor:</LI><UL><LI>1 : BBa_K801042</LI><LI>2 : BBa_K801041</LI><LI>3 : BBa_K801043</LI><LI>4 : BBa_K1017301</LI><LI>5 : BBa_K1017726</LI></UL><LI>prepped samples (ng/µL)</LI><UL><LI>1A : 375    1B : 328</LI><LI>2A : 275    2B : 275</LI><LI>3A : 252    3B : 265</LI><LI>4A : 178    4B : 157</LI><LI>5A : 108    5B : 68</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Friday, 7th August 2015</H3><P class="names">Dana</P><UL><LI>pezT-Mutagenesis
	</LI><LI>PCR-Fragments 1 + 2, gel clean up, cut out of gel
</LI><LI>Gel extraction with kit, probes are labelled as 1 + 2 in the second box in the freezer
</LI></UL></DIV><DIV class="entry"><H3 class="date">Monday, 10th August 2015</H3><P class="names">Heiko, Julian</P><FIGURE class="right"><FIGCAPTION>Programming the PCR</FIGCAPTION></FIGURE><UL><LI>new pezT plates plated out (How to do IPTG induction?)
</LI><LI>PCR (5x buffer, Phusion Polymerase, DMSO from Siegfried)
</LI><LI>DNA concentration from purified PCR 1, 2
</LI><LI>1    230 ng/µL
</LI><LI>2    100 ng/µL
</LI></UL><LI>Biometra Thermocycler (DNA under 1 kB)                ??
</LI><LI>At “Ulrike” number 15, IGEM
</LI><LI>Again PCR 1 + 2 
</LI><LI>gel: PCR 3, 2, 1 
</LI><LI>9 µL product + 1 µL loading dye
</LI></DIV><DIV class="entry"><H3 class="date">Tuesday, 11th August 2015</H3><P class="names">Henning</P><UL>
	Extraction of Dna from gels isolated yesterday. Concentrations are too low. 
Next time prepare full volume of PCR-product in Gel for the Extraction.
</UL></DIV><DIV class="entry"><H3 class="date">Wednesday, 12th August 2015</H3><P class="names">Heiko, Julian, Dana</P><UL><LI>pezT</LI><UL><LI>repeated thinning streak to get single colony</LI></UL><LI>Mutagenesis-PCR</LI><UL><LI>repeated PCR 1 &amp; 2</LI><LI>load result onto gel</LI><LI>gelextraction</LI><LI>concentration PCR 1: 8 ng/µL    PCR 2: 10 ng/µL</LI></UL><LI>Lightinduceable promotor:</LI><UL><LI>trafo of BBa_…(1A,2A,3A,4A,5A) into BW29655</LI><LI>10 µL of BBa each</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Thursday, 13th August 2015</H3><P class="names">Heiko, Vincent</P><UL><LI>Mutagenesis-PCR
	</LI><LI>gel extraction
</LI><LI>PCR 3 with cleaned PCR fragments
</LI><LI>PCR 4 (iGEM MUT4)
</LI></UL><LI>Lightinducable promotors
</LI><LI>picked single colonies from trafo into 5 mL LB (+Camp)
</LI><LI>pezT
</LI><LI>picked single colonies into 5 mL LB (+Kan) (1,2,3)
</LI></DIV><DIV class="entry"><H3 class="date">Friday, 14th August 2015</H3><P class="names">Dana</P><UL><LI>Lightinducable Promotor:</LI><UL><LI>Glycerolstocks (stored at -80°C)</LI></UL><LI>pezT-Mutagenesis</LI><UL><LI>Restriction:</LI><UL><LI>20 µL PCR4</LI><LI>1 µL EcoRI</LI><LI>1 µL PstI</LI><LI>1 µL FD Buffer</LI><LI>5 µL ddH20</LI>                
→ 20 minutes at 37 °C</UL><LI>Clean up with PCR-Clean Up Kit</LI><LI>Concentration after clean up: 38 ng/µL</LI><LI>Ligation</LI><UL><LI>1 µL Vektor (pSB-1C3, 23 ng)</LI><LI>1.2 µL pezT-Insert (38 ng)</LI><LI>2 µL T4 Buffer</LI><LI>1 µL T4 Ligase</LI><LI>5 µL ddH20 (6 µL for control)</LI>     
→ 20 minutes at RT, 10 min 65°C
</UL></UL></UL></DIV><DIV class="entry"><H3 class="date">Tuesday, 18th August 2015</H3><P class="names">Dana</P><UL><LI>Constitutive promoter (BBa_J23100, 35 bp) (for 3A-Assembly with miRNA)</LI><UL><LI>Amp-Plates</LI><LI>Biobrick on Plate 4 of distribution kit; Well 17D, in Plasmid Backbone BBa_61002, Amp-Resistance
</LI><LI>punch hole into foil of cover, add 10 µL of dH2O, pipette up and down, let sit for 5 minutes
</LI><LI>transfer into tube with competent cells (BWs), two seperate samples, 5 µL into each, + one control, trafo
</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Wednesday, 19th August 2015</H3><P class="names">Henning</P><UL><LI>Constitutive promoter (BBa_J23100, 35 bp) (for 3A-Assembly with miRNA)
	</LI><LI>All plates were positive including control. Maybe the BW are resistant against Amp?
</LI><LI>Inoculation of new Amp plates with competent BW (20 µl) for control
</LI><LI>other samples are not usable either (too much bacteria)
</LI></UL><LI>PezT
 </LI><LI>control gel of ligation and ligation control from monday. 
</LI><LI>2 µl sample + 7 µl water + 1 µl LD
</LI><LI>Ligation seems not to be successful due to two equal signals in sample and control.
</LI><LI>Repetition is necessary.
</LI></DIV><DIV class="entry"><H3 class="date">Friday, 21st August 2015</H3><P class="names">Henning, Maurice, Vincent, Artur</P><FIGURE class="right"><FIGCAPTION>Loading the samples into the gel</FIGCAPTION></FIGURE><UL><LI>Supplies</LI><UL><LI>made approx. 20 new Amp-plates</LI><LI>stored in fridge</LI></UL><LI>Transformations</LI><UL><LI>pezT ligation product into DH5α (two batches -&gt; Ligation A and B)</LI><LI>BBa_J23102 into DH5α (two batches -&gt; Promoter A and B)</LI><LI>BBa_K592009 into DH5α (two batches -&gt; Pigment A and B)</LI><UL><LI>plated all and incubated at 37 °C</LI></UL></UL><LI>PezT amplification PCR</LI><UL><LI>PCR of the mutagenesis PCR product with the standard Fusion protocol</LI></UL><LI>control gel of PezT amplification PCR</LI><UL><LI>measured the concentration of the PCR product</LI></UL><LI>control gel of PezT amplification PCR</LI><UL><LI>measured the concentration of the PCR product</LI><LI>control gel with 1% agarose</LI><LI>discard the sample due to strong impurity</LI></UL><LI>Preparation of PezT containing 5 ml cultures for IPTG induction</LI><UL><LI>made new liquid culture from glycerol stock in LB + Kan medium</LI><UL><LI>incubated at 37 °C</LI></UL><LI>preparation of 9 different dillutions of 3 samples to get good OD on saturday</LI><LI>1:5, 1:10 and 1:20 dilutions of samples with OD 0.8</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Saturday, 22nd August 2015</H3><P class="names">Nikolas, Sonja</P><UL><LI>Picking Minis (Duplicates):</LI><UL><LI>Pigment A    (in CAMP LB)</LI><LI>Pigment B    (in CAMP LB)</LI><LI>Ligation A    (in CAMP LB)</LI><LI>Ligation B    (in CAMP LB)</LI><LI>Promoter A    (in Amp LB)</LI><LI>Promoter B    (in Amp LB)</LI></UL><LI>Incubation at 37° C</LI><LI>IPTG Induktion von pezT Colis:</LI><UL><LI>OD measurement of prepared cultures</LI><LI>selecting culture with an Od of ~0,4 (OD=0,354)</LI><LI>Dilute selected culture in 20 ml LB and split those in 2 Erlenmeyerflasks (~12 mL each, control+sample)
</LI><LI>IPTG induction by adding 12 µl 1 M IPTG</LI><LI>OD measurement every 20 minutes</LI></UL><LI>We noticed a constant decline in the OD and decided to abort the experiment after 1 h
</LI></UL></DIV><DIV class="entry"><H3 class="date">Sunday, 23rd August 2015</H3><P class="names">Henning</P><UL><LI>Glyercolstock of trafo samples from saturday</LI><UL><LI>added 300 µl Glycerol (70%) to 700 µl bacterial culture</LI><LI>stored at -70 °C (one night at -20 °C)</LI></UL><LI>Preparation of mini-cultures from saturday</LI><UL><LI>preparation of all mini cultures from saturday</LI><LI>Ligation, pigment and promoter</LI><LI>appropriate to the kit’s protocol</LI><LI>measured the concentration</LI><LI>stored at -20 °C</LI></UL><LI>Control digestion of the PezT ligations</LI><UL><LI>100 µg sample DNA</LI><LI>1 µl EcoRI</LI><LI>1 µl PstI</LI><LI>1 µl FD Buffer</LI><LI>x µl ddH2O</LI><LI>20 minutes at 37°C</LI><LI>loaded on 1% agarose gel</LI></UL><LI>Dilution of pezT cultures for IPTG induction</LI><UL><LI>prepared two culture dilutions</LI><LI>approx. 125 ml LB with Kan each</LI><LI>added half of the control culture from Saturday</LI><LI>stored one in the fridge and the other down to the bench at RT</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Monday, 24th August 2015</H3><P class="names">Artur, Vincent</P><UL><LI>Repetition of the control digestion</LI><UL><LI>beside ligation sample volumes:</LI><UL><LI>1µL EcoRI</LI><LI>1µL PstI</LI><LI>1µL FD Green Buffer</LI><LI>ad 10 µL H2O</LI></UL><LI>checked with control gel</LI><LI>suggestion: repeat again with positive control of unmutated PezT</LI></UL><LI>creation of miRNA probe</LI><UL><LI>four single stranded PCRs with VF or VR primers and miRNA insert over night/ 100 cycles</LI><LI>suggestion for further steps: tag with 32P via 32P-ATP and PNK</LI><LI>(stored at -20° C, Tuesday)</LI></UL><LI>IPTG induction of PezT culture</LI><UL><LI>first try: three samples with</LI><UL><LI>LB-Kan</LI><LI>PezT cells + IPTG (1mM)</LI><LI>PezT cells - IPTG (1mM)</LI></UL><LI>second try:</LI><UL><LI>PezT culture growth over night</LI></UL></UL></UL></DIV><DIV class="entry"><H3 class="date">Tuesday, 25th August 2015</H3><P class="names">Artur, Maurice</P><UL><LI>Continuing IPTG induction of PezT culture</LI><UL><LI>measured ODs</LI><LI>no recognition of sufficient OD decrease in both tries</LI><LI>suggestion: make new IPTG stock</LI></UL><LI>3A-Assembly</LI><UL><LI>of GroEL biobrick with pigment biobrick and constitutive promoter with miRNA following the iGEM protocol (restriction and ligation), both in pSB1K3 backbone (Kan resistance)
</LI><LI>protocol changes/used materials: fast digest restriction enzymes and buffer, no BSA, our samples (pigment A1 from 23.08., promoter A1 from 23.08, GroEL from 31.07 (233 ng/µL), miRNA (elution) from 30.07 (38 ng/µL)), 1:10 dilutions of samples (except miRNA)
</LI><LI>obtained two samples (1 GroEL + pigm 3A Assembly, 2 prom + miRNA 3A Assem) for transformation, stored at -20° C
</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Thursday, 27th August 2015</H3><P class="names">Dana, Vincent</P><UL><LI>pezT IPTG induction</LI><UL><LI>plated liquid cultures from the 26th on Kan-plates (20 µL each)</LI><UL><LI>incubated at 37 °C</LI></UL><LI>OD600 of liquid cultures with and without IPTG was the same</LI><LI>plates with and without IPTG are similarly covered with colonies</LI></UL><LI>GroEL+Pigment &amp; Promoter+miRNA</LI><UL><LI>picked colonies into liquid cultures</LI><UL><LI>incubated at 37 °C</LI></UL></UL><LI>Colony PCR</LI><UL><LI>picked colonies from miRNA, GroEL, GroEL+Pigment and Promoter+miRNA</LI><UL><LI>8 colonies each</LI></UL><LI>made reciprocal plate for picked colonies</LI><UL><LI>incubated at 37 °C</LI></UL><LI>used DreamTaq-Mix</LI><LI>made control gel</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Friday, 28th August 2015</H3><P class="names">Nikolas, Sonja</P><UL><LI>GroEL+Pigment &amp; Promoter+miRNA</LI><UL><LI>Miniprep, DNA stored at -20 °C</LI><LI>Glycerol Stock; stored at -80 °C</LI></UL><LI>pezT Colony PCR</LI><UL><LI>Preparation of 2 plates from liquid culture pezT A</LI></UL><LI>Incubation at 37 °C</LI></UL></DIV><DIV class="entry"><H3 class="date">Saturday, 29th August 2015</H3><P class="names">Julian</P><UL><LI>pezT Colony PCR</LI><UL><LI>Put the cultures in the fridge</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Monday, 31st August 2015</H3><P class="names">Artur, Nikolas</P><UL><LI>pezT Colony PCR</LI><UL><LI>Picked 2 colonies from pezT plates (stored in the fridge)</LI><LI>used DreamTaq MM</LI><LI>Made control gel</LI></UL><LI>GroEL+Pigment</LI><UL><LI>Diluted cultures A and B to an OD of 0,45</LI><LI>incubated 20 min at 37 °C</LI><LI>Heatshock37 in water bath 1 min 42 ° C</LI><LI>incubated at 37 °C</LI><LI>Decided to repeat the heatshock for culture B to ensure sufficient stress level</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Tuesday, 1st September 2015</H3><P class="names">Heiko, Julian, Vincent</P><UL><LI>GroEL+Pigment</LI><UL><LI>cultures from 31st August did not express the pigment</LI><LI>diluted cultures in liquid medium and incubated at 42 °C over night</LI></UL><LI>pezT Colony PCR</LI><UL><LI>repetition of colony PCR with DreamTaq-Mix</LI><UL><LI>Meinhart plasmid, the PezT preparations A, B and C, a colony from the dilution plating, negative control
</LI></UL><LI>control gel in 1 % agarose
</LI><LI>two fragments in the Meinhart plasmid and the PezT preparations A and B at ca. 3000 nt and 2500 nt</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Wednesday, 2nd September 2015</H3><P class="names">Nikolas, Julian, Maurice</P><UL><LI>pezT Colony PCR</LI><UL><LI>Colony + pezT A, B, C</LI><LI>Important: Strip tubes/lids are leaky -&gt; The water in a few samples evaporated just like yesterday -&gt; no PCR</LI></UL><LI>Light Promoter 3A Assembly</LI><UL><LI>3A assembly of BBa_K1017726 and BBa_101301 according to the iGEM protocol</LI><LI>Ligation stored in freezer</LI><LI>Important: The BBs might be in bacteria, we inoculated a mini culture to test this possibility</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Thursday, 3rd September 2015</H3><P class="names">Heiko, Maurice</P><UL><LI>analysis of ‘BBs in bacteria’ suspicion</LI><UL><LI>no growth in Kan mini cultures  and C mini cultures (BBa_K1017726 and negative control), but growth of C mini culture with BBa_101301
</LI><LI>nevertheless continuing the transformation protocol, could be false positive</LI></UL><LI>transformation of Light Promoter 3A Assembly ligation product</LI><UL><LI>four samples: E/P restricted pSB1K3 BB (Kan plates), 3A assembly ligations product (Kan Plates), negative control with DH5-alpha only (Kan plate) and positive control of pSB1C3-RFP standard biobrick (brown eppi) (C plates)
</LI><LI>growth over night in 37° room</LI></UL><LI>repetition of pezT colony PCR</LI><UL><LI>scheme like yesterday</LI><LI>gel with PCR products</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Friday, 4th September 2015</H3><P class="names">Nikolas, Sonja</P><UL><LI>Annealing miRNA and GroEl</LI><UL><LI>10 µL miRNA fw + 10 µL miRNA rev</LI><LI>10 µL GroEl fw + 10 µL GroEl rev</LI></UL>
10 min at 94 °C 
Cool down to RT slowly 

<LI>pSB1C3 restriction</LI><UL><LI>10 µL pSB1C3 RFP</LI><LI>1 µL EcoR1</LI><LI>1 µL Pst1</LI><LI>2 µL FD buffer</LI><LI>6 µL ddH2O</LI></UL>  

30 min at 37 °C 
Heat inactivation 80°C for 20 min 

<LI>pSB1C3 dephosphorylation</LI><UL><LI>10 µL pSB1C3 (cut)</LI><LI>1 µL FastAP</LI><LI>2 µL FastAP buffer</LI><LI>7 µL ddH2O</LI></UL> 

30 min at 37 °C
Cleaning of pSB1C3 with PCR Clean Up Kit

<LI>NanoDrop</LI><UL><LI>pSB1C3 (dephosphorylated)    8 ng/µL</LI></UL><LI>Ligation</LI><UL> 
With miRNA old and new, with GroEl old and new
<LI>3 µL pSB1C3</LI><LI>1 µL insert</LI><LI>2 µL T4 buffer</LI><LI>1 µL T4 ligase</LI><LI>3 µL ddH2O</LI></UL><LI>Transformation</LI><UL>
GroEl old and new, miRNA old and new in DH5 alpha
<LI>DH5α 30 min on ice</LI><LI>1 min heat shock 42.2 °C</LI><LI>15 min on ice</LI><LI>ad 200 µL SOC-medium</LI> 

1 h and 10 min, 37 °C plate rotator

Samples plated on cAmp plates:
<UL><LI>200 µL</LI><LI>20 µL    each</LI><LI>1 neg. control a 200 µL</LI></UL>
Stored at 37 °C overnight.
</UL><LI>New SOC-medium</LI><UL><LI>200 µL aliquots in freezer</LI></UL><LI>GroEl+ pigment A, B mini</LI><UL><LI>5 mL overnight culture in LB Kan medium</LI> 
Stored at 37 °C.
</UL></UL></DIV><DIV class="entry"><H3 class="date">Saturday, 5th September 2015</H3><P class="names">Nikolas, Sonja</P><UL>
No colonies after transformation on Friday.

<LI>NanoDrop</LI><UL> 
(For annealings) 

<LI>GroEl new    4.26 µg/µL        + 50 µL annealing buffer</LI><LI>GroEl old    0.0  µg/µL</LI><LI>miRNA new  2.313 µg/µL      + 50 µL annealing buffer</LI><LI>miRNA old    0.0 µg/µL</LI></UL><LI>Maxi for GroEl+ pigment A, B</LI><UL><LI>300 mL culture</LI> 
Overnight 37 °C
(were not prepped)
</UL><LI>GroEl promotor test</LI><UL><LI>1.5 µL per sample</LI><LI>10, 20, 30 min heat shock (42 °C)</LI> 
samples transferred to 5 mL overnight cultures after heat shock, stored at 37 °C.
</UL><LI>Backbone restriction</LI><UL>
200 ng plasmid of each sample:
<LI>pSB1C3: 8 µL</LI><LI>BB 16.7.: 14.3 µL</LI><LI>BBC 26.6.: 1.2 µL</LI> 

For each sample:
<UL><LI>3 µL FD buffer</LI><LI>1 µL EcoR1</LI><LI>1 µL Pst1</LI>
final sample volume 20 µL
</UL></UL><LI>Backbone dephosphorylation</LI><UL><LI>20 µL restriction product</LI><LI>1.5 µL FastAP</LI><LI>2.4 µL Fast AP buffer</LI> 

30 min 37 °C 
Stored in freezer at -20 °C.
</UL></UL></DIV><DIV class="entry"><H3 class="date">Sunday, 6th September 2015</H3><P class="names">Julian, Henning</P><UL><LI>Preparation of new LB-Agar-Camp plates</LI><UL><LI>Preparation of 36 new Chloramphenicol containing LB-Agar plates</LI></UL><LI>PCR-Clean-Up of dephosphorylated Ligations from Sep. 5th</LI><UL><LI>using PCR-Clean-Up Kit</LI><LI>concentrations of eluted samples are noted in the hardcopy labbook</LI></UL><LI>Annealing of miRNA oligos</LI><UL><LI>mixed 100 µl of miRNA and GroEL oligos (100 mM each)</LI><LI>heated up to 95°C (heatblock)</LI><LI>slowly cool down to RT (approx. 2h)</LI><LI>measured concentration (hardcopy labbook)</LI></UL><LI>Digestion of pSB-1C3 backbones</LI><UL><LI>used different samples of backbone</LI><UL><LI>linearized backbone, provided by iGEM (50 µg total)</LI><LI>backbone from stock preparation (very old)</LI><LI>backbone from RFP-construct (very low concentration (pmol)??)</LI></UL><LI>digested with EcoRI and PstI for 20 min (FD)</LI><LI>heat inactivation of enzymes at 65°C for 20 min</LI></UL><LI>Dephosphorylation of pSB-1C3 backbones</LI><UL><LI>dephosphorylated with fast AP for 10 min at 37°C</LI><LI>PCR-Clean-Up of all samples</LI><UL><LI>Not necessary, fast AP ist heat sensitive!</LI></UL><LI>concentrations are noted in the hardcopy labbook</LI></UL><LI>Ligation of miRNA and GroEL inserts with pSB-1C3 backbones</LI><UL><LI>used insert samples from different steps of the hole experimental phase and from the annealing from the same day
</LI><LI>Prepared different combinations of Insert and backbone samples</LI><LI>Used different concentrations of insert (3:1 &amp; 30:1)</LI> 
→ 19 samples
<LI>Ligated with T4 DNA ligase for 20 min at RT</LI><LI>Heat inactivation at 65°C for 10 min</LI><LI>chilled on ice</LI></UL><LI>Transformation of ligation constructs
</LI><LI>used a total of 24 different samples (19 new + 5 old ligation constructs)</LI><LI>added 12 µl of ligation to 50µl of competent DH5a</LI><LI>incubation on ice for 30 min</LI><LI>Heatshock at 42°C for 30 sec</LI><LI>chilled on ice until use (approx. 5 min)</LI><LI>Added 950 µl LB medium (RT)</LI><LI>Incubation at 37°C for 30 min while shaking</LI><LI>spreaded 200 µl of cells and ligation mixture to LB-Agar plates (RT, Camp, very thin!)</LI><LI>Incubated over night at 37°C</LI></UL></DIV><DIV class="entry"><H3 class="date">Monday, 7th September 2015</H3><P class="names">Dana</P><UL><LI>Trafos from yesterday (sunday 6th)</LI><UL><LI>no growth on any plate, put back into 37 °C→ on the table on the left</LI></UL><LI>New trafos</LI><UL><LI>all ligations from sunday 6th were transformed into DH5a</LI><LI>2 hours incubation with SOC-medium</LI><LI>labelled 2-18, on the table in 37°C room on the right</LI></UL><LI>pSB1C3-RFP-positive control</LI><UL><LI>picked 3 colonies, minis in 37 °C</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Tuesday, 8th September 2015</H3><P class="names">Nikolas</P><UL><LI>Mini preparation</LI><UL><LI>picked 3, 6, 17 and 22 (duplicates for each plate)</LI><LI>5 mL overnight cultures</LI> 
stored at 37 °C
</UL></UL></DIV><DIV class="entry"><H3 class="date">Wednesday, 9th September 2015</H3><P class="names">Dana</P><UL><LI>iGEM Plasmids</LI><UL><LI>Kit Plate 1    1A    BBa_K314110</LI><LI>Kit Plate 1    3A    BBa_K873002</LI><LI>Kit Plate 1    12L    BBa_K517003</LI></UL>
→ Trafo into DH5a from the group of Ignatova

<LI>Minis from yesterday:</LI><UL><LI>6+ 17: media were red, discarded them</LI><LI>3+22: mini plasmid prepp with kit, taken them to get sequenced</LI><UL>
Concentrations: 
<LI>3A: 165 ng/µL</LI><LI>3B: 208 ng/µL</LI><LI>22A: 230 ng/µL</LI><LI>22B: 180 ng/µL</LI></UL></UL></UL></DIV><DIV class="entry"><H3 class="date">Thursday, 10th September 2015</H3><P class="names">Julian</P><UL><LI>Picking minis:</LI><UL><LI>Visual colonies had been picked. Due to the fact that only some colonies had been on the plates i have returned them to the 37°C storage room. 
</LI></UL><LI>Efficiency calculation:</LI><UL><LI>Couldnt be calculated because i didn't find a control plate with 20µL.</LI></UL></UL></DIV><DIV class="entry"><H3 class="date">Friday, 11th September 2015</H3><P class="names">Dana</P><UL><LI>Started with restriction of pSB1C3 backbone</LI><UL><LI>restriction with PstI, 1h, 37 °C</LI><LI>PCR clean up</LI><LI>restriction with EcoRI, 1h 37 °C, stopped here, because data from sequencing came back</LI></UL><LI>Overview Sequencing:
48DD03    GroEl    118 ng/µL
48DD06    GroEl    178 ng/µL 
48DD09    GroEl    233 ng/µL
48DD02    miRNA     35 ng/µL 
48DD05    miRNA     38 ng/µL 
48DD08    miRNA     68 ng/µL
48DD24    3A       165 ng/µL    miRNA Ligation from 06.09.15 
48DD23    3B       208 ng/µL    miRNA Ligation from 06.09.15
48DD20    22A      230 ng/µL    miRNA Ligation from 23.07.15 
48DD21    22B      180 ng/µL    miRNA Ligation from 23.07.15 
</LI><LI>As miRNA (48DD05) and 22a+b are postive, miRNA 05 and 22A were transformed into DH5a</LI></UL></DIV><DIV class="entry"><H3 class="date">Sunday, 13th September 2015</H3><P class="names">Julian</P><UL><LI>Picking of the minis</LI></UL></DIV><DIV class="entry"><H3 class="date">Monday, 14th September 2015</H3><P class="names">Julian</P><UL><LI>Prepping of the minis (plasmid kit)</LI></UL></DIV><DIV class="entry"><H3 class="date">Tuesday, 15th September 2015</H3><P class="names">Julian</P><UL><LI>Annealing of the oligos</LI><UL><LI>10 min at 94°C (Heatblock)</LI><LI>Temperature reduction to room temperature in one hour</LI></UL><LI>Restriction of the linear backbone</LI><UL><LI>60µL pSB-1C3 (25ng/µL)</LI><LI>3µL EcoR1 (fast digest)</LI><LI>3µL Pst1 (slow digest)</LI><LI>9µL FD-Buffer</LI> 
⇒ 1 hour at 37°C
⇒ PCR clean up kit
<UL><LI>90µL restricted Backbone</LI><LI>450µL DF-Buffer</LI><LI>rest as said in the protocol</LI> 
⇒ concentration BB1 = 23,5 ng/µL
</UL></UL><LI>Dephosphorilation of the restricted Backbone</LI><UL><LI>30µL pSB-1C3 (cut)</LI><LI>3µL Fast AP</LI><LI>tip from Chao, not to use the Fast AP Buffer</LI><LI>PCR Clean up kit</LI> 
⇒ Concetration restricted and dephosphorilated BB2= 5 ng/µL
</UL><LI>Ligation</LI><UL><LI>Dilution of the annealed oligos up to 10 ng/µL</LI><LI>ratio of 1:3 vector to insert</LI><LI>Groel</LI><UL><LI>10µL     rest. cut BB (5ng/µL ⇒ 50ng)</LI><LI>2,2µL     alligatet Groel oligos (10ng/µL ⇒ 22ng)</LI><LI>2µL     T4- DNA Ligase Buffer</LI><LI>1µL     T4 -DNA Ligase</LI><LI>5µL     ddH2O</LI></UL><LI>miRNA</LI><UL><LI>10µL     rest. cut BB (5ng/µL ⇒ 50ng)</LI><LI>1,5µL     alligatet Groel oligos (10ng/µL ⇒ 15ng)</LI><LI>2µL     T4- DNA Ligase Buffer</LI><LI>1µL     T4 -DNA Ligase</LI><LI>5µL     ddH2O</LI></UL><LI>control</LI><UL><LI>10µL     rest. cut BB (5ng/µL ⇒ 50ng)</LI><LI>2µL     T4- DNA Ligase Buffer</LI><LI>1µL     T4 -DNA Ligase</LI><LI>7µL     ddH2O</LI></UL>
⇒ Overnight Ligation with 16°C
</UL></UL></DIV><DIV class="entry"><H3 class="date">Wednesday, 16th September 2015</H3><P class="names">Julian, Maurice</P><UL><LI>Transformation of the ligated Oligos (as told as in the protocol written by Henning and Dana)</LI><LI>Restriction of the miRNA with constitutive promoter to change the backbone</LI> 
⇒ Ligation with new backbone (pSB-1C3) 
⇒ Transformation with new backbone (37°C room)
</UL></DIV><DIV class="entry"><H3 class="date">Thursday, 17th September 2015</H3><P class="names">Sonja, Julian, Maurice</P><UL><LI> Restriction of new IDT-Products ( const. Prom. + miRNA)
        <IL> Ligation (into the vector)
        <IL> Preparation of new liquid and solid bacterial cultures
</IL></IL></LI></UL></DIV><DIV class="entry"><H3 class="date">Friday, 18th September 2015</H3><P class="names">Henning, Julian</P><UL><LI> Prepping, drying and colony PCR of the minis (all made by official and already established protocols)
        </LI><LI> Unspecific validation of the heat induction promotor via  an agarose gel
        </LI><LI> Finishing the Wiki-Page in particular by Dana, Vincent and Maurice, the  rest of the team participated too
</LI></UL></DIV></DIV><DIV class="container"><P>We thank our sponsors:</P></DIV></DIV></BODY></HTML>