<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_British_Columbia_Model1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:British_Columbia/Model1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:British_Columbia/Model1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/Model1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:British_Columbia/Model1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:British_Columbia/Model1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:British Columbia/Model1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE> Team: British Columbia - 2011.igem.org </TITLE></P><DIV id="wrap"><DIV id="left"><UL id="menu"><LI><A href="https://2011.igem.org/Team:British_Columbia">Home</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Members">Team Members</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Advisors">Advisors</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/University">Where We Live</A></LI><LI><A href="https://igem.org/Team.cgi?id=517">Official Profile</A></LI></UL><LI><A>Project</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Abstract">Abstract</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Story">The Story</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Objectives">Objectives</A></LI><LI><A>Achievements</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Accomplishments">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Parts">Parts</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Americas">Americas Jamboree</A></LI><LI><A>Notebook</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Protocol">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Modeling">Modeling</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Model2">Beetle Epidemic</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Model1">Monoterpene Production</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Model3">Synthase Structure</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/HP">Human practices</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Mentorship">Mentorship</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Wild">Synthetic Biology Gone Wild</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Perceptions">Public Perceptions</A></LI><LI><A href="https://2011.igem.org/Team:British_Columbia/Sponsors">Acknowledgements</A></LI><DIV id="bod"><CENTER><H3>Secretion and Production of Monoterpenes in Yeast</H3></CENTER><B>OBJECTIVE:</B> To create a model to assay the production of monoterpenes in our genetically engineered yeast cells given that they contain the metabolic genes IDI1, K6R-hmg2, erg20-2 and a monoterpene synthase.<B>Model Creator:</B> Gurpal Bisra<B>Model Advisors:</B> Dr. Eric Lagally, Dr. James Piret, Dr. Steve Withers, Dr. Joanne Fox, Shing Zhan, Alina Chan, Rafael Saer, Eric Finlay<P>I, Gurpal Bisra, simplified the mevalonate pathway <B>[1]</B> to only enzymes associated with our project – IDI1, HMG2 and ERG20. I created first order Michaelis-Menten differential equations for the simplified chemical reactions. For this, Dr. Eric Lagally referred me to 2 sources: “Writing Differential Equations” <B>[2]</B> and “Chapter 6- Transport and Kinetics” <B>[3]</B>. However, many constants and concentrations must be known to solve these differential equations. An enzyme database “BRENDA” contains some kcat and km values <B>[4]</B>. However, more research must be done to determine the currently unknown constants. These differential equations can be solved in MATLAB to plot monoterpene concentration as a function of substrate or time.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:597px;"><DIV class="thumbcaption">Figure 1: The Mevalonate Pathway as found in yeast <B>(Ignea et al. 2011)</B>.</DIV></DIV></DIV></DIV><P>Our team wants to mass produce monoterpenes so we...
</P><P>1) Replaced HMG2 with the mutant variant K6R-hmg2, which is stabilized against protein degradation <B>[5]</B>.
</P><P>2) Replaced ERG20 with the mutant variant K197E erg20-2, which produces &quot;25% FPP and 75% GPP instead of 75% FPP and 25% GPP&quot; as compared to the wildtype ERG20 gene <B>[6]</B>. We want more GPP than FPP because our goal is to produce monoterpenes and not diterpenoids <B>[7]</B>.
</P><P>3) Introduced a gene which encodes for a synthase to use GPP to form monoterpenes. We chose 5 different synthases that each produces different monoterpenes in different proportions <B>[8]</B>:
</P><P>Synthase PgTPS-Pin2 catalyzes GPP into producing 70.5% β-pinene and 29% α-pinene + other monoterpenes in negligible amounts.
</P><P>Synthase PsTPS-Car1 catalyzes GPP into producing 66.4% 3-carene and 2.7% α-pinene + other monoterpenes in negligible amounts.
</P><P>Synthase PgxeTPS-Cin catalyzes GPP into producing 65.6% 1,8 cineole, 3.0% α-pinene and 2.6% β-pinene + other monoterpenes in negligible amounts.
</P><P>Synthase PgTPS-Cin catalyzes GPP into producing 81.1% 1,8 cineole, 1.9% α-pinene and 1.9% β-pinene + other monoterpenes in negligible amounts.
</P><P>Synthase PsTPS-Pin catalyzes GPP into producing 83.4% α-pinene and 12.6% β-pinene + other monoterpenes in negligible amounts.
</P><P>Based on these changes, I created a modified model:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:572px;"><DIV class="thumbcaption">Figure 2: Simplified model of the mevalonate pathway with the K6R-hmg2 and erg20-2 genes.</DIV></DIV></DIV></DIV><P><CENTER>
Block diagrams of biochemical pathway for <A href="JavaScript:newPopup('https://static.igem.org/mediawiki/2011/8/86/MODEL_1_-_PsTPS-Pin_Final.jpg');">PsTPS-Pin synthase</A>, <A href="JavaScript:newPopup('https://static.igem.org/mediawiki/2011/8/86/MODEL_1_-_PgTPS-Pin2_Final.jpg');">PgTPS-Pin2 synthase</A>, <A href="JavaScript:newPopup('https://static.igem.org/mediawiki/2011/3/37/MODEL_1_-_PsTPS-Car1_Final.jpg');">PsTPS-Car1 synthase</A>, <A href="JavaScript:newPopup('https://static.igem.org/mediawiki/2011/a/ab/MODEL_1_-_PgxeTPS-Cin_Final.jpg');">PgxeTPS-Cin synthase</A>, <A href="JavaScript:newPopup('https://static.igem.org/mediawiki/2011/b/ba/MODEL_1_-_PgTPS-Cin_Final.jpg');">PgTPS-Cin synthase</A>.</CENTER></P><P>Please note:
</P><P><B>1) The enzymes ERG12, ERG8 and MVD1 are NOT RATE LIMITING; therefore, we chose not to include them in our simplified model. However, these enzymes are accounted for in our simulations.</B></P><P><B>2) The RATE LIMITING STEP is at <A href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=hmg2">HMG2</A>.</B></P><P>Now, I needed to model these chemical equations as a series of differential equations. As an engineer, I don't have a biochemistry background so this problem was new for me. This <A href="http://www.boomer.org/c/p3/c02/c0207.html">website</A> provided the basics to get started. 2 things to take into account are:
</P><P>1) The direction of the arrow:
-if the arrow goes into a component, the equation segment is positive
-if the arrow leaves a component, the equation segment is negative
</P><P>2) Types of Rate Processes:
-if the rate process is zeroth order, just enter the rate constant
-if the rate process i first order, multiply the rate constant by the initial substrate concentration
-for a Michaelis-Menten process, include the amount or concentration of the component at the tail of the arrow (call it X1 for example) and you will get: d(X1)/dt = (-Vm*X1)/(Km+X1)
</P><P>Most of the equations fall under 2 types of Michaelis–Menten equations:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:345px;"><DIV class="thumbcaption">Figure 3: The two main types of chemical reactions.</DIV></DIV></DIV></DIV><P>For the first type of reaction, I had to make 2 assumptions:
</P><P><B>1) At steady state, [ES] does not change with time, therefore it's derivative with time is = 0. Therefore, k1[E][S] = (k2+k3)[ES]</B></P><P><B>2) S &gt;&gt; Eo</B></P><P>The second type of Michaelis-Menten kinetic equation applies to cases where I have 2 substrates combine into 1 product via a condensation reaction; DMA-PP and I-PP are condensed to generate geranyl diphosphate (GPP, C10), which is further converted with IPP into ferensyl diphosphate (FPP, C15), with prenyl transferase such as FPP synthase <B>(Misawa, 2011)</B>.
</P><P>To model these types of reactions, I found an expression for the reaction velocity to be:
v = d(X)/dt = (Vmax)/( (kms1/[S1]) + (kms2/[S2]) + 1 )
where [S1] = concentration of substrate 1, [S2] = concentration of substrate 2. They have an associated kinetic value. Vmax = kcat*[E] where [E] = the concentration of enzyme.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:831px;"><DIV class="thumbcaption">Figure 4: Differential equations common in all products.</DIV></DIV></DIV></DIV><P>The final thing I needed to consider before writing out the differential equations was how to use data indicating what percentage of monoterpenes were produced. These final steps also took the form of the Michaelis-Menten equations but I modified them by multiplying each by the fraction of product produced. For example, please look at equations 10 and 12 in the differential equations below to see fraction of products are produced for the PgTPS-Pin2 synthase.
</P><P>

The differential equations specific to the <A href="JavaScript:newPopup('https://static.igem.org/mediawiki/2011/2/24/Synthase_PsTPS-Pin_DE1.jpg');">PsTPS-Pin synthase</A>, <A href="JavaScript:newPopup('https://static.igem.org/mediawiki/2011/2/2c/Synthase_PgTPS-Pin2_DE.jpg');">PgTPS-Pin2 synthase</A>, <A href="JavaScript:newPopup('https://static.igem.org/mediawiki/2011/c/c9/Synthase_PsTPS-Car1_DE.jpg');">PsTPS-Car1 synthase</A>, <A href="JavaScript:newPopup('https://static.igem.org/mediawiki/2011/5/59/Synthase_PgxeTPS-Cin_DE1.jpg');">PgxeTPS-Cin synthase</A>, <A href="JavaScript:newPopup('https://static.igem.org/mediawiki/2011/d/da/Synthase_PgTPS-Cin_DE1.jpg');">PgTPS-Cin synthase</A>.</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#PARAMETERS_-_KINETIC_CONSTANTS"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">PARAMETERS - KINETIC CONSTANTS</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#SIMULATIONS"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">SIMULATIONS</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#RESULTS"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">RESULTS</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#SENSITIVITY_ANALYSIS_-_FOR_MANUFACTURING"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">SENSITIVITY ANALYSIS - FOR MANUFACTURING</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#REFERENCES"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">REFERENCES</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="PARAMETERS_-_KINETIC_CONSTANTS">PARAMETERS - KINETIC CONSTANTS</SPAN></H3><P>In order to model each enzyme, at least 2 values must be known: Km and Kcat. These values must be determined when using Michaelis-Menten kinetics to model enzymes. 
</P><P>I found most of my values using the BRENDA database. Not all constants have actually been determined yet for Saccharomyces cerevisiae. After all, our project is based on research that was conducted earlier this year. 
The values shown in black below were found using the BRENDA database <B>[4]</B>. The values are red had to be inferred. For example, the enzyme HMG2 and K6R-hmg2 were only different in the fact that K6R-hmg2 acted as a stabilizing against protein degration;therefore, the kinetic values for each should be the same  
<B>[5]</B>.
Also, the values were erg20-2 were determined by deciding that the ratio of kcat/km should be 3 times higher that the same ratio for ERG20. This was based on the fact that erg20-2 has been show to produce &quot;25% FPP and 75% GPP instead of 75% FPP and 25% GPP&quot; as compared to the wildtype ERG20 gene <B>[9]</B>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:922px;"><DIV class="thumbcaption">Figure 5: Kcat values for enzymes found in our simplified biochemical pathway.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:922px;"><DIV class="thumbcaption">Figure 6: Km values for enzymes found in our simplified biochemical pathway.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="SIMULATIONS">SIMULATIONS</SPAN></H3><P>Below is a MATLAB code that illustrates how the concentration of β-pinene can be found with one of our engineered yeast cells containing the mutant metabolic genes K6R-hmg2, erg20-2 and the PgTPS-Pin2 synthase gene.
</P><P>Save the following .m file into MATLAB as named &quot;PgTPSPin2.m&quot;. This program runs under the assumption that time runs from 0 seconds to 1,000,000 seconds and that the initial concentration of substrate = 5 where as the initial concentration of each product = 0. The output is a concentration of all compounds found in the biochemical pathway (M) vs. time (s).
</P><P>Then, input the following commands into MATLAB while being in the appropriate directory:
<B>&gt;&gt; xspan =[0 1000000];</B><B>&gt;&gt; ynot = [5 0 0 0 0 0 0 0 0];</B><B>&gt;&gt; [X,Y] = ode45(@PgTPSPin2,xspan,ynot);</B><B>&gt;&gt; plot(X,Y)</B></P><P>These are the most basic commands. One can also use the diff(x) function to determine time derivative of concentrations as well.
</P><DIV id="box">function dy = PgTPSPin2(t,y);
 % Contains K6R-hmg2 and erg20-2
 % Define Constants:
 % y(1) = [HMG-CoA]
 V_A = 5.833*10^(-11);
 km_A = 0.175*10^(-3);
 % y(2) = [MVA]
 V_sigma = 1.283*10^(-8);
 km_sigma = 7.4*10^(-3);
 % y(3) = [MVA-5-phosphate]
 V_pi = 1.00*10^(-9);
 km_pi = 0.041*10^(-3);
 % y(4) = [MVA-pyrophosphate]
 V_delta = 4.90*10^(-6);
 km_delta = 0.123*10^(-3);
 % y(5) = [I-PP]
 V_C = 1.817*10^(-9);
 km_C = 0.036*10^(-3);
 V_D = 1.817*10^(-9);
 km_D = 0.035*10^(-3);
 Vmax = 3.883*10^(-8);
 km_E = 0.027*10^(-3);
 km_F = 0.459*10^(-3);
 km_G = 17.176*10^(-3);
 % y(6) = [DMA-PP]
 % y(7) = [FPP]
 % y(8) = [GPP]
 V_H1 = 2.84*10^(-8);
 km_H1 = 0.006*10^(-3);
 % y(9) = [B-pinene]
 % y(10) = [a-pinene]
 % MATLAB Code:
 dy = zeros(9,1);
 dy(1) = -((V_A*y(1))/(km_A+y(1)));
 dy(2) = ((V_A*y(1))/(km_A+y(1)))-((V_sigma*y(2))/(km_sigma+y(2)));
 dy(3) = ((V_sigma*y(2))/(km_sigma+y(2)))-((V_pi*y(3))/(km_pi+y(3)));
 dy(4) = ((V_pi*y(3))/(km_pi+y(3)))-((V_delta*y(4))/(km_delta+y(4)));
 dy(5) = ((V_delta*y(4))/(km_delta+y(4)))-((V_C*y(5))/(km_C+y(5)))-((V_D*y(6))/(km_D+y(6)))-
 ((Vmax)/((km_F/y(5))+(km_E/y(6))+(1)))-((V_D*y(6))/(km_D+y(6)))-((Vmax)/((km_F/y(5))+(km_G/y(8))+(1)));
 dy(6) = ((V_C*y(5))/(km_C+y(5)))-((V_D*y(6))/(km_D+y(6)))-((Vmax)/((km_F/y(5))+(km_E/y(6))+(1)));
 dy(7) = ((Vmax)/((km_F/y(5))+(km_G/y(8))+(1)));
 dy(8) = -((Vmax)/((km_F/y(5))+(km_G/y(8))+(1)))+((Vmax)/((km_F/y(5))+(km_E/y(6))+(1)))-((V_H1*y(8))/(km_H1+y(8)))*(0.705);
 dy(9) = ((V_H1*y(8))/(km_H1+y(8)))*(0.705);
 % dy(9) = ((V_H1*y(8))/(km_H1+y(8)))*(0.295); </DIV><H3><SPAN class="mw-headline" id="RESULTS">RESULTS</SPAN></H3><P>One of my objectives was to determine how much more GPP is produced using the erg20-2 gene instead of just the ERG20 gene. Below is a graph which illustrates the concentration of GPP produced using the pinene synthases, PgTPS-Pin2 and PsTPS-Pin, found in the simplified biochemical pathway as a function of time with the ERG20 or with the erg20-2 gene. Relatively speaking, <B>my simulation showed that 1.3127 times more GPP is produced with the addition of the erg20-2 gene</B>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:784px;"><DIV class="thumbcaption">Figure 7: The concentration of GPP produced using the pinene synthases, PgTPS-Pin2 and PsTPS-Pin, found in the simplified biochemical pathway as a function of time with the ERG20 or with the erg20-2 gene.</DIV></DIV></DIV></DIV><P>Next, I wanted to determine what effect K6R-hmg2 and erg20-2 would have on the concentration of monoterpenes produced. I ran 2 separate simulations and merged their outputs on the plot below. <B>My simulation showed that 17.10% more beta-pinene is produced and 17.00 % more alpha-pinene is produced when K6R-hmg2 and erg20-2 are used instead of HMG2 and ERG20 alone</B>. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:758px;"><DIV class="thumbcaption">Figure 8: The concentration of monoterpenes produced using the pinene synthases, PgTPS-Pin2 and PsTPS-Pin, found in the simplified biochemical pathway as a function of time.</DIV></DIV></DIV></DIV><P>
I went one step further and also took the time derivatives of the monoterpene output. I plotted them as a function of time as shown below. My simulation showed that the maximum velocity of:
</P><P><B>beta-pinene with K6R-hmg2 and erg20-2 is 5.674*10^(-10) M/s</B><B>alpha-pinene with K6R-hmg2 and erg20-2 is 2.372*10^(-10) M/s</B><B>beta-pinene with HMG2 and ERG20 is 4.638*10^(-10) M/s</B><B>alpha-pinene with HMG2 and ERG20 is 1.941*10^(-10) M/s</B></P><P><B>Therefore, the K6R-hmg2 and the erg20-2 genes increase the rate of beta-pinene production by 22.34% and alpha-pinene by 22.21%.</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:798px;"><DIV class="thumbcaption">Figure 9: The time derivatives of concentration of monoterpenes produced using the pinene synthases, PgTPS-Pin2 and PsTPS-Pin, found in the simplified biochemical pathway as a function of time.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="SENSITIVITY_ANALYSIS_-_FOR_MANUFACTURING">SENSITIVITY ANALYSIS - FOR MANUFACTURING</SPAN></H3><P>After proving that the addition of the K6R-HMG2 and erg20-2 would increase the production of monoterpenes, I performed sensitivity analysis to determine which enzymes would most contribute to the increase in production of GPP substrate, monoterpenes and the rate at which monoterpenes are produced.
</P><P>In order to do this, I tripled each of the following enzymes and ran the simulations separately:
 [K6R-HMG2]
 [IDI1] for the production of DMA-PP
 [IDI1] for the production of I-PP
 [erg20-2]
 [PgTPS-Pin2]
</P><P>I compared each case to the above simulations which were based on the fact that each enzyme was at the same relative concentrations. This was the case for both endogenous cells which contained the ERG20 and HMG2 genes as well as for the mutant cells containing the erg20-2 and K6R-HMG2 genes instead. All measurements were compared at steady state.
</P><P>The first result is the concentration of GPP increased using the pinene synthases (PgTPS-Pin2 and PsTPS-Pin), with selected enzymes' concentrations being tripled, for the simplified biochemical pathway. The simulations suggest the following:
 tripling the [K6R-HMG2] increases the [GPP] by 8.7000 times
 not tripling any enzymes, for the mutant cell, yields an increase of [GPP] of 1.6929 times
 tripling the [IDI1], for the production of DMA-PP, increases the [GPP] by 1.4484 times
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:733px;"><DIV class="thumbcaption">Figure 10: The concentration of GPP increased using the pinene synthases (PgTPS-Pin2 and PsTPS-Pin), with selected enzymes' concentrations being tripled, for the simplified biochemical pathway. Mutant cells contain the erg20-2 and K6R-HMG2 genes where as endogenous cells have the HMG2 and ERG20 genes instead.</DIV></DIV></DIV></DIV><P>The second result is the concentration of GPP decreased using the pinene synthases (PgTPS-Pin2 and PsTPS-Pin), with selected enzymes' concentrations being tripled, for the simplified biochemical pathway. The simulations suggest the following:
 tripling the [IDI1], for the production of I-PP, decreases the [GPP] by 37.63%
 tripling the [PgTPS-Pin2] decreases the [GPP] by 41.45%
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:773px;"><DIV class="thumbcaption">Figure 11: The concentration of GPP decreased using the pinene synthases (PgTPS-Pin2 and PsTPS-Pin), with selected enzymes' concentrations being tripled, for the simplified biochemical pathway. Mutant cells contain the erg20-2 and K6R-HMG2 genes where as endogenous cells have the HMG2 and ERG20 genes instead.</DIV></DIV></DIV></DIV><P>The third result was for the concentration of B-pinene increased using the pinene synthases (PgTPS-Pin2 and PsTPS-Pin), with selected enzymes' concentrations being tripled, for the simplified biochemical pathway. The simulations suggest the following:
 tripling the [K6R-HMG2] increases the [B-pinene] by 7.3166 times
 tripling the [IDI1], for the production of DMA-PP, increases the [B-pinene] by 1.3052 times
 not tripling any enzymes, for the mutant cell, yields an increase of [B-pinene] of 1.1702 times
 tripling the [PgTPS-Pin2] increases the [B-pinene] by 1.1718 times
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:794px;"><DIV class="thumbcaption">Figure 12: The concentration of B-pinene increased using the pinene synthases (PgTPS-Pin2 and PsTPS-Pin), with selected enzymes' concentrations being tripled, for the simplified biochemical pathway. Mutant cells contain the erg20-2 and K6R-HMG2 genes where as endogenous cells have the HMG2 and ERG20 genes instead.</DIV></DIV></DIV></DIV><P>The fourth result was for the concentration of B-pinene decreased using the pinene synthases (PgTPS-Pin2 and PsTPS-Pin), with selected enzymes' concentrations being tripled, for the simplified biochemical pathway. The simulations suggest the following:
 tripling the [IDI1], for the production of I-PP, decreases the [B-pinene] by 35.65%
</P><P>In particular, the model breaks down when the concentration of [IDI1], for the production of I-PP, is tripled because negative values are noted. Therefore, one should not attempt to introduce a higher concentration of that gene into a cell for manufacturing purposes.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:783px;"><DIV class="thumbcaption">Figure 13: The concentration of B-pinene decreased using the pinene synthases (PgTPS-Pin2 and PsTPS-Pin), with selected enzymes' concentrations being tripled, for the simplified biochemical pathway. Mutant cells contain the erg20-2 and K6R-HMG2 genes where as endogenous cells have the HMG2 and ERG20 genes instead.</DIV></DIV></DIV></DIV><P>The final result was for the derivative of the [B-pinene] increased using the pinene synthases (PgTPS-Pin2 and PsTPS-Pin), with selected enzymes' concentrations being tripled, for the simplified biochemical pathway.The simulations suggest the following:
 tripling the [K6R-HMG2] increases the rate of [GPP] production by 16.4536 times
 tripling the [PgTPS-Pin2] increases the rate of [GPP] production by 3.67743 times
 not tripling any enzymes, for the mutant cell, yields an increase of rate of [GPP] production by 2.4292 times
 tripling the [IDI1], for the production of DMA-PP, increases the rate of [GPP] production by 2.0348 times
 tripling the [IDI1], for the production of I-PP, decreases the rate of [GPP] production by 1.5222 times
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:790px;"><DIV class="thumbcaption">Figure 14: The derivative of the [B-pinene] increased using the pinene synthases (PgTPS-Pin2 and PsTPS-Pin), with selected enzymes' concentrations being tripled, for the simplified biochemical pathway. Mutant cells contain the erg20-2 and K6R-HMG2 genes where as endogenous cells have the HMG2 and ERG20 genes instead.</DIV></DIV></DIV></DIV><P>To summarize, the engineered mevalonate pathway should be modified to increase the [K6R-HMG2] and [IDI1] for DMA-PP production for mass producing monoterpenes for manufacturing purposes. The tripling the [K6R-HMG2] increases the [GPP] by 8.7000 times where as tripling the [IDI1] for the production of I-PP and [PgTPS-Pin2] decreases the [GPP] by 37.63% and 41.45% respectively. The tripling the [K6R-HMG2] and [IDI1] for the production of DMA-PP increases the [B-pinene] by 7.3166 times and 1.3052 times respectively. In contrast, tripling the [IDI1] for the production of I-PP decreases the [B-pinene] by 35.65%. The simulations suggested that the increase of any enzyme would help increase the rate of monoterpene production. For manufacturing purposes, the engineered mevalonate pathway should be modified to increase the [K6R-HMG2] and [IDI1] for DMA-PP production for mass producing monoterpenes.
</P><H3><SPAN class="mw-headline" id="REFERENCES">REFERENCES</SPAN></H3><P><B>[1]</B>	Ignea, C; Cvetkovic, I; Loupassaki, S; Kefalas, P; Johnson, CB; Kampranis, SC; Makris, AM. Improving yeast strains using recyclable integration cassettes, for the production of plant terpenoids. Micob Cell Factories. 2011, 10:4.  
<B>[2]</B>	Bourne, David W. Writing Differential Equations. 2001. <A href="http://www.boomer.org/c/p3/c02/c0207.html" class="external free" rel="nofollow">http://www.boomer.org/c/p3/c02/c0207.html</A>.
<B>[3]</B>	Dr. Jakubowski. Chapter 6- Transport and Kinetics. 2011. <A href="http://employees.csbsju.edu/hjakubowski/classes/ch331/transkinetics/olssrapeqenzkineq.html" class="external free" rel="nofollow">http://employees.csbsju.edu/hjakubowski/classes/ch331/transkinetics/olssrapeqenzkineq.html</A><B>[4]</B>	Scheer, Maurice. BRENDA. 2011. <A href="http://www.brenda-enzymes.org/" class="external free" rel="nofollow">http://www.brenda-enzymes.org/</A><B>[5]</B>	 Gardner RG, Hampton RY. A 'distributed degron' allows regulated entry into the ER degradation pathway. Embo J. 1999, 18: 5994-6004.
<B>[6]</B>	Oswald, M; Fischer, M; Dirninger, N; Karst, F. Monoterpenoid biosynthesis in Saccharomyces cerevisiae. FEMS Yeast Res. 2007. 413-421.
<B>[7]</B>	Misawa, Norihiko. Pathway engineering for functional isoprenoids. Current Opinion in Biotechnology. 2011. 22:1-7.
<B>[8]</B>	Keeling, Christopher I; Weisshaar, Sabrina; Ralph, Steven G; Jancsik, Sharon; Hamberger, Britta; Dullat, Harpreet K; Bohlmann, Jӧrg. Transcriptome mining, functional characterization, and phylogeny of a large terpene synthase gene family in spruce. BMC Plant Biology. 2011. 11:43.
<B>[9]</B>	Fischer, Marc JC; Meyer, Sophie; Claudel, Patricia; Bergdoll, Marc; Karst, Francis. Metabolic engineering of monoterpene synthesis in yeast. Biotechnology and Bioengineering. 2011. 108:1883–1892.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:British_Columbia/Model1">http://2011.igem.org/Team:British_Columbia/Model1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:British_Columbia/Model1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:British_Columbia/Model1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:British_Columbia/Model1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:British_Columbia/Model1&amp;oldid=249752" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>