"
Page
Discussion
View source
History
teams
Log in
 
Team:British Columbia/Notebook/Week 10
From 2011.igem.org
Team: British Columbia - 2011.igem.org
Home
Team
Team Members
Advisors
Where We Live
Official Profile
Project
Abstract
The Story
Objectives
Achievements
Achievements
Parts
Data
Americas Jamboree
Notebook
Notebook
Protocols
Safety
Modeling
Beetle Epidemic
Monoterpene Production
Synthase Structure
Human practices
Mentorship
Synthetic Biology Gone Wild
Public Perceptions
Acknowledgements
Brainstorming
Week 1: Jun 5-11
Week 2: Jun 12-18
Week 3: Jun 19-25
Week 4: Jun 26-Jul 2
Week 5: Jul 3-9
Week 6: Jul 10-16
Week 7: Jul 17-23
Week 8: Jul 24-30
Week 9: Jul 31-Aug 6
Week 10: Aug 7-13
Week 11: Aug 14-20
Week 12: Aug 21-27
Week 13: Aug 28-Sep 3
Week 14: Sep 4-10
Week 15: Sep 11-17
Week 16: Sep 18-28
Americas Jamboree!!!
Week 19: Oct 9-15
Leading up to the Finals!
Week 10: August 7-13
Contents
1Lab Meeting - August 8
2UBC iGEM Logo
31,8-Cineole
4beta-Pinene
5IDI1, HMG2 metabolic genes
6Characterization: Limonene Synthase - Part BBa_K118025
7Characterization: Limonene Synthase (LIMS1) - Part: BBa_I742110
8iGEM Social at Jacob's!
Lab Meeting - August 8
Today, we mostly discussed fundraising. We came up with a list of potential sponsors; each member is responsible for contacting their assigned sponsor. We need money!
Laura gave a very detailed update on her ideas for human practices, which includes videos, and a presentation at Science World.
UBC iGEM Logo
Joe is waiting for his primers to come. In the mean time, he is working on this year's iGEM logo on Adobe Illustrator.  Living in the most beautiful place on Earth, with trees, mountains and serenity, it is hard not to be inspired by the scenic beauty of British Columbia, Canada. At the same, Joe wants to illustrate the effects of the mountain pine beetle epidemic on our forests, which is fueled by recent global climate changes. Even a beautiful place like this can't escape these changes, showing how our Earth's ecosystems are ultimately linked.   Here is what he has so far...
basic iGEM logo
now with trees...
more trees...
with UBC 2011
with mountains...
Image A
Image B
1,8-Cineole
pSB1C3 overnight culture
pSB1C3 in microcentrifuge tubes - miniprep in progress!
Jacob, now armed with what he believes are two versions of fully SDM'd 1,8-cineole synthase (still no sequencing results), decided to come in on the weekend and use PCR to add restriction sites to the ends of his genes, ligate his genes onto suitable plasmids, and grow these constructs in E. coli. If everything worked perfectly, he could present some satisfying results at the Monday meeting, but nothing ever seems to work perfectly. There were several setbacks.
The PCRs for preparing both synthase genes (pg-tps-cin and pgxe-tps-cin) for biobricks and 415 PAG GPD and GAL (both his-tagged and not) worked well, with the exception of the pg-tps-1,8-cineole for the biobrick. As a side note, Jacob ran duplicates of each reaction, one with 0.5 uL added magnesium per 50 uL reaction, and one without. Only the ones with magnesium showed any product in the resulting gel.
Next came the ligations.  Vicki prepared new pSB1C3 due to our previous stocks' odd behaviour (which included empty tubes that seemed to produce ligations and tubes that ran on gels in two pieces without being cut), and cut it with ECORI and PSTI to visualize it on a gel. The gel showed bands at the correct lengths (~1000bp) (Besides, the overnight culture turned pink/red - what else could it be if not pSB1C3 with RFP in it!). Jacob used some of this cut product for the biobrick ligation, along with ECORI and PSTI cut pgxe-tps-cin.
Preparing the ligation products for the yeast shuttle plasmid was a bit harder. The restriction sites added to the end of the synthases were SMAI and NOTI, the former of which only works in NEB buffer 4 at 25°, while the latter only had a 25% effectiveness in NEB buffer 4, and worked best at 37°. Jacob felt there was not enough product for a sequential digest, so he used NEB buffer 4 and let it sit at 25° for an hour then 37° for an hour.
The biobrick was prepared by ligating the pSB1C3 with the synthases using T4 ligase, then transformed. The ligations for the yeast parts would have taken too long, so Jacob waited until the next day for the shuttle vector ligations.
The first biobrick ligation transformations were a failure.
The second biobrick ligation transformations were a success! That is to say, there were colonies, and they were not expressing the RFP protein. Overnight cultures of the three present colonies showed that two had the RFP gene, but one did not. The one that did not express the RFP gene was shown to very likely be the proper construct by cPCR of the original colony and incomplete sequencing. The pg-tps-cin PCR to attach the suffix and prefix was unsuccessful, and will be repeated next week.
The eventual transformations of the synthases on 415 PAG GPD and GAL grew colonies, but they are of dubious quality. The plates used supported colonies on the negative control plate, which had only competent cells. This raised the question of whether there was a problem with the plates or the cells themselves, and Jacob is running tests to check.
CPCRs of the 415 GPD and GAL ligation transformants showed that some of the colonies had the synthases, and some did not.
The results of the minipreps from overnight cultures of the colonies that had the synthases were then used to transform yeast. Jacob transformed 9 strains of yeast in total, (pg-gpd,pgxe-gpd, and pgxe-gal into wildtype, erg-20, and erg20-2 mutants), and grew them on leucine deficient media.
beta-Pinene
Vicki holding a bottle of contaminated LB broth
Vicki's plates from last week (2 days ago) had transformants on every plate!! (She should not have had colonies on her negative control). Something is wrong.  Just in case, she runs a cPCR on a few colonies for each of her treatment plates. Gel verification shows bands at the incorrect length (~2000bp when she expects ~1000bp); there is even a lane in the water control! She decides to throw out the water and make a new aliquot.  It appears that there is something wrong with the competent cells (as Jacob mentions above) - some of the cells are ampicillin resistant (but not all as we do not see a lawn of cells on the plates). Oh no...
Not yet feeling defeated, Vicki re-dos her ligations and transformations for assembling of the yeast plasmid with original and SDM synthases with and without his-tag on the GPD promoter. The plate has 2 transformants! Excitedly, Vicki runs a cPCR on the colonies; unfortunately, gel verification shows 2 unexpected bands (below 500bp and at 1000bp). In case the cPCR failed, she decides to repeat the experiment while varying the annealing temperatures. Gel verification shows bands at 1kb (expects ~500bp). Sigh..
After a talk with our graduate advisor, Alina, Vicki decides to prioritize due to lack of time and money. Hence, she will only assemble the yeast construct that contains the original+no-his-tag pinene synthase.
Because Marianne has her SDM+no-his-tag synthase on both a GPD (constitutive) and GAL (galactose-induced) promoter, Vicki decides that constructing a yeast plasmid with the original+no-his-tag synthase on both a GPD and GAL promoter will make the characterization of beta-pinene synthase more complete (can compare the GPD and GAL for original synthase and SDMed synthase). In preparation for assembling the yeast construct containing the GAL promoter, she transforms and minipreps the yeast plasmid (pAD415GAL) as we are running low. Vicki digests pAD415GAL at SpeI and XhoI to verify the identity of the plasmid, and for the most part, it looks okay (there seems to be some incomplete digestion despite digesting for 2 hours).
To prepare for another run of GC-MS with beta-pinene, Vicki starts an overnight culture in glucose media for 6 different samples:
SDM.no-his-tag.GAL in wildtype = SNL3 W
SDM.no-his-tag.GAL in pBS (contains = SNL3 B
SDM.no-his-tag.GAL in pKS (mutated: erg20-2) = SNL3 K
SDM.no-his-tag.GPD in wildtype = SND1 W
SND1 B
SND1 K
Marianne attempted to build her beta-pinene biobrick once again (digestion, ligation, transformation). She really hoped this would be the last time she has to do this. Marianne checked the plates the next day and found empty plates. Marianne really hates beta-pinene. Marianne has been trying to conserve the PCR purified beta-pinene synthase genes because they are running really low and she doesn't like the idea of PCR-ing it out again. But Marianne decided to use up everything left in the tubes and try to assemble the biobrick one last time before she quits trying. When she is doing her digest, she finds out that all of the competent cells we have are amp-resistant. As she is about to accept the fact that she is not meant to assemble this biobrick, Alina offers a few of her competent cells for transformation, so Marianne pushes on. The next day, Marianne took out her plates and the had white colonies! Marianne cPCRed and confirmed that the the genes are of expected lengths. Happily, she set up overnight cultures for a few of the colonies for miniprepping the next day.
Marianne and Vicki had a very productive week. On top of all of the above things, they conducted yeast extractions to prepare samples for GC-MS and managed to hand them in at the GC-MS lab. Everyone is excited for our team's first GC-MS data.
IDI1, HMG2 metabolic genes
Sam performed a ligation of the IDI1 and HMG2 genes to a yeast TRP gateway vector.  He tried to verify a complete vector with colony PCR again and again and again, but to no avail.  The gel results had bands for IDI1 on some trials, but never IDI1 and HMG2 together.  He discontinued this wetlab track to focus on modelling.
Preparing samples for GCMS
Characterization: Limonene Synthase - Part BBa_K118025
Daisy has finally gotten the transformation of the limonene synthase to work! This is the 5th or 6th transformation she has done to try and get the limonene synthase into bacteria.  Let us examine the different transformations and why they did not work. Try not to giggle:
Transformation 1: Double transformation into BL21 with pRARE2 and Limonene synthase plasmid
Transformation 2: Transformation of Limonene synthase plasmid into C41 DE3
Transformation 3: Transformation of limonene synthase plasmid into DH5 alpha
Transformation 4: Transformation of limonene synthase plasmid into DH5 alpha (2nd batch)
Transformation 5: Transformation of limonene synthase plasmid into DH5 alpha (plates and cells from Hieter lab - thanks Alina)
Why so many transformations to get one plasmid into bacteria? Here's why:
Transformation 1: Double transformation has low efficiency.
Transformation 2: Did not let the cells incubate long enough.
Transformation 3: Plated the cells on Chlor and amp plates. The plasmid only has Amp resistance. No cells grew.
Transformation 4: Plated cells on amp plates. Saw colonies. Did a control. The cells without the plasmid also grew on the plates. It's likely the competent cells are amp resistant. The plates are okay as Daisy made a new amp stock just to make sure.
Transformation 5: The DH5 alpha cells were from Hieter lab and so were the plates.
Daisy is picking a colony to grow to miniprep more plasmid.
Characterization: Limonene Synthase (LIMS1) - Part: BBa_I742110
Vicki also miniprepped one of the colonies to obtain more plasmid (more BBa_K118025). She then proceeded to PCR the synthase out using her custom primers. Gel verification shows success: bright, distinct bands at around 1.6kb (expected band length). This should be BBa_I742110 aka LIMS1.
iGEM Social at Jacob's!
The birthday boys
Pipetting party!
To celebrate Jacob and Gurpal's birthdays, the team decided to have a party at Jacob's! (Thanks Jacob for offering your place!). Everyone had many great laughs as we ate pizza, drank, and played Mafia. Cheers!
Back to the Notebook
Retrieved from " http://2011.igem.org/Team:British_Columbia/Notebook/Week_10 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
