<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UiOslo_Norway_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UiOslo Norway/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row nav"><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway">HOME</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Safety">SAFETY</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Description">PROJECT</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Experiments">LAB</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Parts">PARTS</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Human_Practices">PRACTICES</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Team">TEAM</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Entrepreneurship">AWARDS</A></DIV><DIV class="col-md-2 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Attributions">ATTRIBUTIONS</A></DIV></DIV><DIV class="row subsite"><DIV class="col-md-12"><DIV id=""><H2 class="boxheader middle">Results</H2></DIV><P class="boxnotes leftext">
  
Here follows a selection of our results, with the physiologically relevant results highlighted. For the total collection of our results, go to the “Total Results” section. </P><P class="boxnotes leftext">
All of the following graphs are results from adding 20 ul of 500 mg/mL nitrocefin to a total volume of 1 mL of urine, bacteria and purified protein. </P><P class="boxnotes leftext">
In our first experiment we measured both intact and lysed bacterias respons to Nitrocefin. The absorbance was measured at 486nm as recommended from the Mura et. al. 2015 paper. https://www.clinicalkey.com/#!/content/journal/1-s2.0-S0732889315002011
As we wanted our test to be a quick method we measured time intervals from 0 - 40min. </P><P class="boxnotes leftext">
Results from the initial experiments</P><P class="boxnotes leftext">
These measurements was done with synthetic urine ordered online. The red line in the graphs above signifies a clinically relevant bacterial number, along with a detectable change in A486 as the result of hydrolyzation of nitrocefin. </P><P class="boxnotes leftext">
The graphs below displays the same type of experiments as above but in real urine as we wanted to confirm that the test would work under physiological conditions.</P><P class="boxnotes leftext">
AmpR Bacteria in Real Urine: </P><P class="boxnotes leftext">It is clear that real urine somewhat disturbs the signal. However, the signals shown here are of a much higher intensity than necessary to be detectable in PhoneLab, here being visible even to the human eye. </P><P class="boxnotes leftext">
This indicates that even with a lower bacterial count, we could still have gotten measurable readouts. </P><P class="boxnotes leftext">

This graph displays measurements done with the purified protein from BBa_K1189031. This protein function as our positive control to our test. </P><P class="boxnotes leftext">
As one can observe from the graph the hydrolisation of the beta lactam ring is clearly inhibited by the class A inhibitor. The solution without the inhibitor have a clear color change and a strong absorbance after only 5 min. </P><P class="boxnotes leftext">
We also did one experiment with different dilutions of purified protein to investigate the detection limit. Below is the results where we observed that the cut - off value should be set to 1000 - fold dilution of protein.</P><P class="boxnotes leftext">These results represents the foundation of our diagnostic test. Here we have shown that Nitrocefin will react with different amounts of bacteria present, both intact and lysed as well as purified β - lactamase. </P><P class="boxnotes leftext"><B>Overview over total results (in detail)</B></P><P class="boxnotes leftext">Here follows the total results from our experiments. All graphs shown below represent results from experiments performed with E. coli and protein in combination with an added 20ul of 500 mg/mL nitrocefin to a total volume of 1 mL. </P><P class="boxnotes leftext">AmpR E.coli in synthetic urine: </P><P class="boxnotes leftext"> These graphs shows different dilutions of bacteria in synthetic urine and the following changes in A486 as a response to the hydrolyzation of nitrocefin. The graphs clearly show that lysing the bacteria yields a lower detection limit than that obtained by using intact bacteria. </P><P class="boxnotes leftext">AmpR E.coli in real urine:  </P><P class="boxnotes leftext">All of the above graphs show a measurable increase in A486 over time following the hydrolyzation of nitrocefin with varying bacteria numbers. These experiments were performed with a higher bacteria count than the ones with synthetic urine. </P><P class="boxnotes leftext">The protein used here is the expressed and purified <A href="http://parts.igem.org/Part:BBa_K1189031" target="_blank">BBa_K1189031</A> produced by the Calgary team of 2013. It is clear that adding of CVA, an inhibitor of class A beta-lactamases effectively stops the enzymatic reaction even at very high protein concentrations. </P><P class="boxnotes leftext">The above graph shows the essence of our proof of concept. Following transformation of chemically competent Top10 E.coli and IPTG-induced expression, we lysed the bacteria and added nitrocefin along with different combinations of inhibitors. The graph clearly shows that class A inhibitors (CVA) effectively stops the reaction in the cuvettes of both the purified protein (positive control) and that with lysed bacteria. Class B and C inhibitors (DPA and APB, respectively) on the other hand, does not affect the reaction, and a visible color change was seen (shown by the blue and orange lines on the graph). The negative control here was lysed bl21 E.coli without AmpR. The experiment was performed in synthetic urine. </P><P class="boxnotes leftext">The UiOslo team used both purified and bacterial form of part <A href="http://parts.igem.org/Part:BBa_K1189031" target="_blank">BBa_K1189031</A> (referred to in the graph as class A biobrick and class A protein, respectivly) and contributed to further characterization of this part. These results display our contribution to the iGEM registry by showing BBa_K1189031 reaction to Nitrocefin and inhibitor for class A beta - lactamases.For more detailed results and explanations check out part BBa_K1189031 on the iGEM registry main page. Thus, UiOslo team confirms the function of BBa_K1189031 and presents new information and characterization of this part. </P><P class="boxnotes leftext"><B>Determine buffer capacity of urine:</B>
    This was a pilot experiment done in the first weeks of lab start – up. 
</P><P class="boxnotes leftext">We wanted to investigate the possibility of measuring small pH changes in urine, as the cleavage of Nitrocefin and other compounds that contain a beta lactam ring will result in release of H+.</P><P class="boxnotes leftext">In this experiment we added small amounts of HCL to a urine sample and measured the pH. We had two urine samples from two different individuals. These results were obtained from our initial experiments, proving that pH measurements was not a viable option for us.</P><P class="boxnotes leftext imgcap"><I>
Graph 1: pH measurements from urine sample 1, 2 and a mixed sample. As we can see from the 
graph the pH does not change significantly when adding HCL. By adding 150uL 
 of HCL 1M the pH has only changed from 5.25 to 5. This is a direct effect of 
 the buffer capacity of urine. Same results was obtained from urine sample 2.
</I></P><P class="boxnotes leftext"><B>Conclusion:</B> Even though pH is a logarithmic scale, the small changes in pH observed are not enough for our purposes. We added a quite strong acid, and a beta lactam ring hydrolysis will not release enough H+ to be detected this way.  Other error sources are pH in urine is different for each individual, and is influenced by food consumption and how much and what you drink. </P><P class="boxnotes leftext"><B>pH change in urine in presence of bacteria:</B>
Before we decided to move away from the pH measurements we wanted to investigate one last thing. Would bacteria in the urine sample affect pH?
To investigate this we used urine samples collected from two team members and added E. Coli BL21 to the samples. 
</P><P class="boxnotes leftext">The E.Coli strain had been put on overnight culture the day before.
     OD~1</P><P class="boxnotes leftext imgcap"><I>
 Graph 1: Displays the pH measurements when E. Coli is added to a 
urine sample. As one can observe there are no significant change in 
pH when adding increasing amounts of E. Coli. The small changes that 
do occur will be set as background noise. </I></P><P class="boxnotes leftext"><B>Conclusion:</B> The pH does not change significantly when bacteria is added to the sample.  To conclude the overall experiment with pH measurements in urine, we determined not to pursue this direction. The buffer capacity of urine is too strong to measure any small amounts of H+ released.</P><P class="boxnotes leftext"><B>Generation of biobrick BBa_K1927000</B>
This biobrick’s sequence is collected from a clinical isolate obtained from The National Expertise Center for Antibiotic Resistance in Tromsø. These isolates are collected from different health institutions from all over Norway.
This particular gene encodes the enzyme called blaNDM – 1. </P><P class="boxnotes leftext">
Bacteria containing these genes convey resistance to a broad range of β – lactam antibiotics.
The sequence was designed with specific flanking regions that would make it suitable for Gibson Assembly into the pSB1C3 shipping vector. We decided we would make this biobrick without any promotor because of the safety concerns that follows a multi - resistant gene. </P><P class="boxnotes leftext">
With some help the flanking regions was designed so it could directly be cloned into the shipping vector without any PCR. We performed PCR on our pSB1C3 part that we retrieved from one of the biobricks in the distribution kit, also with specific designed primers.</P><P class="boxnotes leftext"><B>Flanking region of gene:</B></P><P class="boxnotes leftext">
GCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGATG-INSERT-TACTAGTAGCGGCCGCTGCAGTCCGGCAAAAAAGGGCAAG</P><P class="boxnotes leftext"><B>Primers:</B>
V1: tactagtagcggccgctgcagtc 23/64oC/61% 
V2: catctagaagcggccgcgaattc 23/62oC/57%
We performed Gibson Assembly (see protocol for details) and transformed the reaction into TOP10 chemically competent cells. The cells were then plated on LB plates containing chloramphenicol.</P><P class="boxnotes leftext imgcap"><I>Figure 1: A few colonies managed to grow on the plates. Confirming the presence of pSB1C3 
and also that the assembly was successful. Religation of vector does not happen as frequently 
in Gibson Assembly as it does in regular ligation.</I></P><P class="boxnotes leftext">
Even though colonies had grown on the plate we wanted to confirm the presence of our insert. We picked two colonies and performed colony PCR to confirm our insert. We used the primers recommended from iGEM and more details about PCR program you may find under protocols.</P><P class="boxnotes leftext imgcap"><I>Figure 2: We used biobrick BBa_K1189031 as a positive control and empty vector as negative control.  
The positive control seemed to be too big in size (bp) for the annealing time used in the program. 
Thus it did not give a clear band. 
Our biobrick however gives a clear band at around 1000bp which corresponds to the sequence  length,
thus the presence of insert is confirmed.</I>
Lane 1: Ladder	
Lane 2: Positive control, BBa_K1189031
Lane 3: BBa_K1927002
Lane 4 and 5: J04500 (part only)
Lane 6:empty vector
</P><P class="boxnotes leftext imgcap"><I>Figure 3: Displays our biobrick cut with different enzymes. 
NotI did not cut that efficient and the gel displays incomplete cutting.
Lane two is our construct cut once, there is a clear band just above 2000bp which 
indicates that our construct is successfully linearized and show corresponding base pairs.</I>
 Lane 1: 1kb ladder gene ruler
Lane 2: BBa_K1927000 cut once w/XbaI	
Lane 3:BBa_K1927000 cut w/ NotI
Lane 4: uncut plasmid. 
We did an additional restriction digest with the newly made biobrick BBa_K1927001:
</P><P class="boxnotes leftext imgcap"><I>Figure 4: Displays another confirmation that the gene of interest 
is within the shipping vector pSB1C3.</I>
Lane 1: 1kb ladder gene ruler
Lane 2: BBa_K1927000 cut w/XbaI and SpeI
Lane 3:BBa_K1927001 cut w/XbaI and SpeI</P><P class="boxnotes leftext"><B>Generation of biobrick BBa_K1927002 and BBa_K1927003</B>
This biobricks sequence is collected from a clinical isolate and it’s called blaCMY – 6 plasmid – mediated amp. 
This represents a class C enzyme from the broad family of β – lactamases.
</P><P class="boxnotes leftext">The gene was designed in the same way as BBa_K1927000 
(see generation of BBa_K1927000) and is designed with specific flanking regions and without promoter for safety reasons.</P><P class="boxnotes leftext imgcap"><I>Figure 1: Result from Gibson Assembly. Transformed G.A product 
into TOP10 chemically competent cells. Colonies grown on 
chloramphenicol plates.</I></P><P class="boxnotes leftext imgcap">To confirm that the insert is present we had the plasmid digested with XbaI and PstI along with BBa_K1927000 as a positive control.</P><P class="boxnotes leftext imgcap">Lane 1: 1kb ladder gene ruler
Lane 2: BBa_K1927000 cut w/ XbaI and PstI
Lane 3:BBa_K1927001 cut W/XbaI and PstI</P><P class="boxnotes leftext imgcap">The cut insert shows band at 1000bp which confirms the generation of the desired construct.
</P><P class="boxnotes leftext"><B>Generation of biobrick BBa_K1927002 and BBa_K1927003</B>
We wanted to calibrate our detection test with a biobrick of our own, in addition to the positive control (E. coli amp), negative control (E. coli without antibiotic resistance) and the purified protein from biobrick BBa_K1189031 as an additional positive control.  </P><P class="boxnotes leftext">
We retrieved the gene sequence for the Amp-gene from the standard E. coli vector pUC19 (http://www.snapgene.com/resources/plasmid_files/basic_cloning_vectors/pUC19/). The AmpR gene product is a β-lactamase class A which confers resistance to ampicillin, carbenicillin, and related antibiotics. The gene was designed to include prefix, suffix and start of vector sequence to make it ideal for Gibson assembly later on. Gene and primers was synthetized by IDT.  We performed a Gibson assembly with the shipping vector pSB1C3 and our gene to generate biobrick BBa_K1927002. </P><P class="boxnotes leftext imgcap"><I>
Figure 1: LB agar plate with chloramphenicol for growth of transformed 
bacteria(TOP10) with BBa_K1927002. The insert was inserted into the shipping 
vector pSB1C3 by Gibson assembly. </I></P><P class="boxnotes leftext imgcap"><I>
Figure 2: Gel analysis of colony PCR with BBa_K1927002. 
The insert was inserted into the shipping vector pSB1C3 by using Gibson assembly.  
  Lane 10 are DNA ladder (GeneRuler 1 kb DNA ladder, Thermo Fisher Scientific). 
Lane 11 are positive control (validated part BBa_K1927000). 
 Lane 18-19 are different colonies from Gibson assembly. They show the appropriate 
band at approx. 1000bp.</I></P><P class="boxnotes leftext">
But since the pSB1C3 vector is not an expression vector, we could not use this for calibrating our test. We therefore looked into the iGEM library and found BBa_J04500 which has a ribosomal binding site and IPTG inducible promotor. By combining AmpR gene downstream for the BBa_J04500 we could theoretically express the gene product which is a β-lactamase class A. This was done by restriction digestion of part BBa_J04500 with Spel and Pstl and the AmpR-insert with Xbal and PstI. Restriction digest was followed by ligation and transformation of TOP10 cells and grown on LB agar plate with chloramphenicol. Colony PCR was performed to verify successful ligation with our insert. Restriction digestion of miniprepped DNA was performed to confirm prior to sequencing.  </P><P class="boxnotes leftext imgcap"><I>Figure 3: Gel analysis of colony PCR with BBa_K1927003.
 The insert was inserted into the shipping vector pSB1C3 by using 2 different approaches; 
3A assembly and a single restriction digest followed by ligation. 
Lane 1 and 10 are DNA ladder (GeneRuler 1 kb DNA ladder, Thermo Fisher Scientific). 
 Lane 2 and 11 are positive control (validated part BBa_K1927000). 
Lane 3-7 are different colonies from 3A assembly.  Lane 8-9 and 12-14 are restriction digest 
and ligation approach. Lane 4, 6-8, 12-13 shows the appropriate band at approx. 1000bp.</I></P><P class="boxnotes leftext imgcap"><I>
Figure 4: Restriction digest of miniprepped DNA with restriction enzymes Pstl and Xbal. 
Lane 1 shows DNA ladder (GeneRuler 1 kb DNA ladder, Thermo Fisher Scientific),  
while lane 1-7 shows the same digest with miniprepped DNA derived from different 
bacteria colonies (transformed competent TOP10 cells with BBa_K1927003). 
Lane 1, 6 and 7 shows the appropriate bands with the insert at approx.  
1000bp and vector at approx. 2000bp.</I></P><P class="boxnotes leftext imgcap"><I>
Figure 5: LB agar plate with chloramphenicol for growth of transformed bacteria (TOP10)
BBa_K1927003. The insert was inserted into the shipping vector pSB1C3 by 
using 2 different approaches; 3A assembly (1) and a single restriction digest followed 
by ligation (2).</I></P><P class="boxnotes leftext">
The bacteria were then grown on a LB agar plate with ampicillin and IPTG for selection and expression for colonies with BBa_K1927003. These bacteria with our biobrick was further experimentally validated by performing our nitrocefin experiment setup. </P><P class="boxnotes leftext imgcap"><I>
Figure 6: LB agar plate with ampicillin and IPTG for 
growth of transformed bacteria (TOP10) with BBa_K1927003. </I></P><P class="boxnotes leftext"><B>
Functional validation of biobrick BBa_K1927003</B>
The biobrick consist of an ampicillin resistant gene called ampR, and its sequence is collected from the pUC19 vector online. To functionally validating one of our brick we made the shipping vector pSB1C3 into an expression vector. We did a lot of research and decided to try the inducible promotor part J04500 http://parts.igem.org/Part:BBa_J04500  that came with the distribution kit. With this promotor upstream placed upstream of our gene of interest we could induce the transcription of our ampicillin resistant gene. Se drawing below:</P><P class="boxnotes leftext imgcap"><I>
Figure 7: Illustration of the plan to combine the two parts  
BBa_J04500 and BBa_K1927002 for expression of gene product 
β-lactamase class A.</I></P><P class="boxnotes leftext">
To generate our biobrick we used the 3A assembly protocol recommended by igem. http://parts.igem.org/Help:Protocols/3A_Assembly, where J04500 is part A and our ampR gene is part B. See protocol for more information.</P><P class="boxnotes leftext">
To functional validate our brick we made overnight cultures of the hopeful colonies and plated them on agar plates containing IPTG (final volume of 1mM) and ampicillin. The plate was incubated overnight at 37 degrees.  Colonies managed to grow on these plates, which confirm the biobricks beta lactamase activity.</P><P class="boxnotes leftext"><B>The diagnostic tool</B>
With an OD of ~1 we made a 16 and 64 times dilutions of the bacteria. This was to reduce the amount of bacteria so the sample is biologically relevant. </P><P class="boxnotes leftext imgcap"><I>
Graph 1: As observed for both the graph and the tables above the 
absorbance at 486nm increases gradually with time. 
The cleavage of Nitrocefin generates a red color in the sample, 
which is visible by eye. The color will also depend on the amount of 
bacteria in the sample as shown in graph 1. The 64 times diluted bacteria have a lower 
absorbance in most of the time points.  </I></P><P class="boxnotes leftext lastp imgcap"><I>
Figure 8: Figure 1 displays the inside of our 3D model. 
Cuvette to the left is synthetic urine and bacteria, but without Nitrocefin.  
The cuvette to the right is same sample only with Nitrocefin.  
This picture is taken after 20 minutes and it’s clear that the color change is visible by eye.</I></P></DIV></DIV><DIV class="row footer"><DIV class="col-md-4 footerelement"><H2 class="boxheader footerheader">Contact</H2><P class="boxnotes footercontact"> Blindernveien 31, 0371 Oslo</P><P class="boxnotes footercontact"> uioslonorway@gmail.com</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>