<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UiOslo_Norway_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UiOslo Norway/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row nav"><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway">HOME</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Safety">SAFETY</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Description">PROJECT</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Experiments">LAB</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Parts">PARTS</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Human_Practices">PRACTICES</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Team">TEAM</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Entrepreneurship">AWARDS</A></DIV><DIV class="col-md-2 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Attributions">ATTRIBUTIONS</A></DIV></DIV><DIV class="row subsite"><DIV class="col-md-12"><DIV class="row"><DIV class="col-md-6"><DIV id=""><H2 class="boxheader left">Notebook</H2></DIV><P class="boxnotes leftext">
 Below is our lab notebook where you can read about our experiences in the laboratory during the iGEM season in both good days and bad. Enjoy!</P><P class="boxnotes leftext"><B>Week 18</B>
First week of working in the lab. The team used some time to get to know the lab properly. 
We also performed a pilot experiment to investigate if we were able to measure small pH changes in urine. Cleavage of a beta lactam ring will result in release of H+, and if it could be detected, we could detect activity of beta lactamases. We also investigated pH changes in urine with different amounts of bacteria present. If bacteria influenced pH in any way, it could be difficult to detect pH changes in urinary tract infections. 
The buffer capacity of urine seemed to be too strong to detect small pH changes and we started to look into other ideas for detection.</P><P class="boxnotes leftext"><B>Week 19</B>
Pilot experiment with a beta lactam derivative called Nitrocefin. We used a constant amount of urine and added different amounts of bacteria to simulate a urinary tract infection. Solution with Escherichia Coli containing plasmid for amp resistance in presence of Nitrocefin turned red. The color change was visible within minutes.</P><P class="boxnotes leftext"><B>Week 20</B>
We investigated the kinetics of the beta lactam derivative Nitrocefin by testing different amounts of synthetic urine and bacteria and look for color change. We also run one experiment where we mimicked a biological urinary tract infections with ~10^5 bacteria. The absorbance was considerable lower, but high enough to get a proper readout.</P><P class="boxnotes leftext"><B>Week 21</B>
No work was done in the wet- lab. We did more research on urinary tract infections and planned experiments ahead.</P><P class="boxnotes leftext"><B>Week 22</B>
The last couple of experiments had been successful, and we wanted to investigate the cut – off value of Nitrocefin. We were wondering how much bacteria could be present to get a proper readout. Thus, the experiment was called titration of bacteria with Nitrocefin. We used triplicates so we could calculate standard deviation, which would give the results more weight.</P><P class="boxnotes leftext"><B>Week 23</B>
We did the same experiment as last week but with lysed bacteria to investigate if we could lower the detection limit. The results showed that the detection limit was lowered with lysed bacteria.</P><P class="boxnotes leftext"><B>Week 24</B>
No work in the wet lab this week. We had come a long way with developing the test and the team had to sit down with the supervisor and discuss further development.</P><P class="boxnotes leftext"><B>Week 25</B>
By this time, we understood that we needed standardize these type of experiments. By running triplicates, we could easily standardize our experiments. Synthetic urine was bought online. 
We repeated the previous experimental setup with live bacteria and with synthetic urine to investigate if it would influence our Nitrocefin read-outs. The experiment was successful and from now on we started working only with synthetic urine.</P><P class="boxnotes leftext"><B>Week 26</B>
This week we did an experiment to investigate how much Nitrocefin was needed to give a proper readout and measurable absorbance. Until now, we had been working with the stock concentration that came from the manufacturer. </P><P class="boxnotes leftext"><B>Week 27</B>
We repeated the last experiment with synthetic urine and Nitrocefin absorbance measurements. This time we tested the synthetic urine with lysed bacteria to investigate if the absorbance still was measurable as it was with real urine.</P><P class="boxnotes leftext"><B>Week 28</B>
No experiments in the lab. We searched the iGEM registry for usable biobricks to our project, we found one β-lactamase from Calgary team 2013 and had it ordered online.</P><P class="boxnotes leftext"><B>Week 29 and 30</B>
We used these two weeks to work with the β-lactamase from the Calgary 2013 iGEM team. We received a glycerol stock and transformed the bacteria into cells more suitable for protein purification. 
We managed to purify the β-lactamase protein successfully by using the protocol from Calgary 2013 wiki webpage. We used Bradford assay to determine the protein concentration.</P><P class="boxnotes leftext"><B>Week 31</B>
By now it was time to think Biobricks! We needed to generate biobricks of our own to send to the iGEM registry. We had ordered an ampR gene from IDT and planned to clone it into the iGEM shipping vector pSB1C3.
We did restriction cutting of both ampR and pSB1C3.
SAP treatment of vector.
The digest was run on gel electrophoresis and gel purified.
The gel purification was not successful and we lost almost all of our DNA and could not continue with the rest of the cloning.</P><P class="boxnotes leftext"><B>Week 32</B>
Because of last week failed experiment we started planning other ways to generate biobricks. We started looking into Gibson Assembly, which was supposed to be easier and quicker than the classical cloning. We designed primers appropriate for G.A.</P><P class="boxnotes leftext lastp"><B>Week 33</B>
When the primers for G.A had arrived, we performed phusion PCR to prepare for Gibson Assembly. The designed primers for our ampR gene worked perfectly fine. However, the primers for the vector did not seemed to work. We tried different phusion buffers and we performed gradient PCR to check if different annealing temperatures was needed. None of them worked.</P></DIV><DIV class="col-md-6"><P class="boxnotes leftext"><B>Week 34</B>
We had two other genes ordered from IDT. These sequences were collected from ESBL clinical isolates, and we designed the genes with special flanking regions that made the suitable for Gibson Assembly. We also ordered new primers for the new Gibson Assembly setup. </P><P class="boxnotes leftext"><B>Week 35</B>
The different inhibitor compounds that we wanted to use in our diagnostic test had arrived. The compounds are inhibitors of different classes of β-lactamase. We prepared buffers for the different compounds and planned the following experiments with the inhibitors. We wanted to use the Calgary biobrick, the purified β-lactamase and ampicillin resistant E. Coli that we had available in our own lab.</P><P class="boxnotes leftext"><B>Week 36</B>
Pilot experiment with the different three inhibitors of class A, B and C β-lactamase. As we only had class A available we only tested this one in the first run. The experiment was a success and the inhibitor for class A worked perfectly!
We also performed Gibson Assembly of genes with class B and C that we ordered online from IDT.
Transformed the G. A product into chemically competent cells and plated out on agar plates with appropriate antibiotics. The next day we checked for colonies and no colonies had grown. </P><P class="boxnotes leftext"><B>Week 37</B>
We repeated the Gibson Assembly with no success. 
We also performed an experiment with the purified protein from Calgary 2013. We investigated the detection limit for the purified protein and did several dilutions with Nitrocefin. We also tested the class A inhibitor with the protein, and again the inhibitor worked perfectly and inhibited the hydrolization of Nitrocefin. </P><P class="boxnotes leftext"><B>Week 38</B>
After several attempts of Gibson Assembly we discussed the protocol with Athanasios (he also helped us design primers for G.A). With his help, we manage to perform Gibson Assembly successfully with our class B gene and the next day, colonies had grown on the plates. 
We took two cultures from this and did an overnight culture.
The next day we did miniprep to collect the DNA, and measured concentration on NanoDrop. We performed restriction digest one time with only one enzyme and one with NotI that would cut in both prefix and suffix. When we checked the gel NotI had not cut properly and we ran another restriction digest with XbaI and PstI. This digest was successful and there was two appropriate bands on the gel.</P><P class="boxnotes leftext"><B>Week 39</B>
This week we performed Gibson Assembly with our class C gene using the same protocol as last time. The product was transformed into chemically competent cells. Colonies grew on the plates the next day.
We did overnight cultures of a couple of colonies and did miniprep the next day. 
In addition to this, to check that our biobrick was the correct one we performed colony PCR. This was not successful and our positive control did not work either. We repeated the colony PCR with minor changes and by running the products on gel, electrophoresis the colony PCR was successful, as the appropriate bands was present on the gel. 
Both our biobricks were sent for sequencing and confirmed.</P><P class="boxnotes leftext"><B>Week 40</B>
This week we planned to perform 3A assembly with the protocol provided by iGEM. We wanted to make a biobrick with our ampR gene couplet to a promotor.
We transformed the biobrick J04500 and did colony PCR on 5 of the colonies that had grown on the plates.  We did overnight cultures and minipreped the colonies that had the right insert. This part was then restriction digested with the appropriate restriction enzyme according to the 3A protocol.
We did restriction digestion on our ampR gene and the pSB1C3 plasmid backbone according to the 3A protocol. The vector was then SAP treated. Usually after any restriction digest the products either are purified on gel or spin column. The 3A protocol from iGEM did not have any purification step. We continued with ligation of only the gene of interest and pSB1C3. The ligation product was transformed and plated on appropriate plates. No colonies had grown the next day. </P><P class="boxnotes leftext">
We repeated the restriction digest only with more DNA so we could afford to lose some DNA in the purification. The restriction products was purified and placed in the fridge over the weekend.
This week we did some experiments for the NTNU-iGEM team as well.</P><P class="boxnotes leftext"><B>Week 41</B>
We ligated the three products from the restriction digestion last week according to the 3A, protocol only we did two reactions with slightly different concentrations of insert. 
Products was transformed and plated on appropriate antibiotic plates. 
We also performed restriction digest of the minipreped J04500 and planned to ligate our ampR gene directly into this part as this already had a promotor. This seemed like an easier method than the 3A as this protocol include ligation of three parts together. We transformed ligation product as well.
We performed Gibson Assembly of our ampR gene, as we wanted to make a biobrick of this as well, without promotor. We transformed the product and put on agar plates with appropriate antibiotics.</P><P class="boxnotes leftext">
All our ligations and G.A had seemed to work and colonies had grown the next day.
We then did colony PCR on 5 colonies from each plate, did overnight cultures, minipreped two each of the successful colony PCR and run restriction digest of the minipreped DNA to check for our insert.
The restriction digest was also successful.
</P><P class="boxnotes leftext">
This week we also performed our inhibitor experiment with clinical isolates. We did overnight cultures of the isolates and lysed them the next day. We had difficulties dissolving the different inhibitors solutions and we could not get a proper absorbance when bacteria, synthetic urine and Nitrocefin was added to the solution. </P><P class="boxnotes leftext lastp"><B>Week 42</B>
No lab was done this week, as it’s the week of wiki freeze. We felt confident that we had enough results for our project.
   </P></DIV></DIV></DIV></DIV><DIV class="row footer"><DIV class="col-md-4 footerelement"><H2 class="boxheader footerheader">Contact</H2><P class="boxnotes footercontact"> Blindernveien 31, 0371 Oslo</P><P class="boxnotes footercontact"> uioslonorway@gmail.com</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>