Team:UiOslo Norway/HP/Gold
HOME
SAFETY
PROJECT
LAB
PARTS
PRACTICES
TEAM
AWARDS
ATTRIBUTIONS
Gold
While we are attempting to fight antibiotic resistance with our novel diagnostic test, we acknowledge the fact that our test only works at a protein level. As the central dogma of molecular biology dictates, protein is the last instance in the information flow in a cell. The problem of resistance genes could of course also reside on a genetic level, upstream of actual enzymes. This is important, as a bacteria may very well have resistance genes without expressing them at a certain time. The expression could even be turned on by the presence of antibiotics, which makes it important to not distribute antibiotics when such genes are present. All of these things points to a need for a diagnostic tool that not only detects enzymatic activity, but also the genes leading to said enzymatic activity. Such a tool would enable us to test for resistant microbes before they actually express their resistance. Further, earlier detection would lower the probability of spreading of the resistant bacteria, potentially protecting a lot of people from painful, dangerous infections.
This reasoning led us to CRISPR/Cas9. Cas9 is useful in this matter because of its ability to recognize specific nucleotide sequences. By now, it is known that by redesigning gRNA/tracrRNA/crRNA, one can design Cas9-systems that can recognize any sequence. Usually, this is used in combination with the native endonuclease activity of Cas9, enabling the researcher to cut DNA at specific locations and manipulate the target sequence. This differs from our approach. By taking a slightly unorthodox, but innovative approach to CRISPR/Cas9 usage, we want to interface it with our PhoneLab.
In order to combine CRISPR/Cas9 technology with our diagnostic tool, we imagine to preserve the “searching” capability of CRISPR/Cas9 systems, and replacing the “cutting” part with a detectable readout similar to what we have with the nitrocefin-based detection. This means that instead of cleaving the resistance genes, we want to couple the pairing of nucleotide targets and Cas9 to a colorimetric signal. By doing so, detection of genes could relatively easily be interfaced with the detection of ESBL activity in our PhoneLab, thus creating a more complete diagnostic tool for ESBL infections.
This could be achieved in several ways, but as a contribution to our project, we propose the following design of a CRISPR/Cas9 system to be used in PhoneLab.
First, we would need to design tracr/crRNA sequences compatible with known ESBL genes. These RNAs would be linked to a null-nuclease Cas9 (dCas9) - a Cas9 enzyme modified so that it binds tracr/crRNA and is guided to its targets,, but does not cleave the target genes. For each target gene, we propose to have two sets of tracr/crRNA, complementary to different, but adjacent parts of the same gene. Each of these RNA molecules would have a dCas9 partner, which again would be fused to half of a detectable protein partner in a split-enzyme assay. When the tracr/crRNA finds its goal, dCas9 would bring the two parts of the split-enzyme together, thus creating a detectable signal. So, in this system, two independent RNA/DNA-binding events would be required to take place, which lowers the probability of false positives.
The detectable protein partner of dCas9 is currently a matter of research for us, but one possibility is to use the subunits of galactosidase. LacZ alpha and -omega are two separate peptides that form the active form of galactosidase when in contact with each other. By fusing these two peptides to two separate dCas9 constructs recognizing adjacent parts of the same gene, the presence of said gene would be coupled to the forming of a functioning galactosidase. Thus, by adding for example liquid compounds similar to X-gal, a visible color change could be achieved, similar to the one seen in our project with nitrocefin.
By designing different tracr/crRNA sequences, this design could potentially be altered to recognize any gene present in high enough numbers in a liquid sample.
Figure 1: Conceptual sketch of the proposed dCas9-system
To investigate how this would work with different classes of ESBL, we aligned the sequences found in clinical isolates of ESBL infections in Norway (information provided by Ørjan Samuelsen in Tromsø).
Figure 2: Generated using the TCOFFEE algorithm in Jalview. Sequences provided
by Ørjan Samuelsen at the National Expertise Center of Antibiotic Resistance in Tromsø.
Although it includes relatively few sequences, this phylogeny suggests that the classes A, B C and D of ESBL are so different on a genetical level that the design we propose could probably be used to pin-point the class of ESBLs present in a sample, simply by designing crRNA/tracrRNA compatible to sequences unique to each class.
In summary: to expand our project, we want to create a potentially universal gene detection tool by utilizing parts from the iGEM registry, as there are several dCas9-constructs and LacZ peptides there. The set-up would include both bioinformatics, when designing the right RNA sequences, regular cloning, to fuse the LacZ peptides to dCas9 and protein expression to form the final constructs. If realized, this would enable us to efficiently test for both ESBL proteins and genes within minutes using our PhoneLab. By adding more slots for cuvettes in our hardware, this test could have been integrated into the set-up and enabled us to test for both ESBL proteins and genes at the same time.  Thus, we imagine to use CRISPR/Cas9 methods in a relatively new and creative way, and have a concrete plan as to how to improve and expand our project.
This outline is a result of conferring with supervisors Dirk Linke, Eric de Muinck and Paul Grini along with correspondence with Kevin M Esvelt from the Wyss Institute for Biologically Inspired Engineering, Harvard University, Cambridge, Massachusetts, USA (corresponding author of the article “Cas9 as a versatile tool for engineering biology”, PubMed ID: 24076990).
Positive controls:
In our hardware, additional room was made for cuvettes containing purified protein controls. The software takes advantage of this by addressing that cuvette, labeling it as “positive” (red), and compares the other cuvettes, containing urine samples and the nitrocefin and inhibitor compounds, to that. By adjusting the protein concentration, the amplitude of the signal created by the purified protein can mimic that created by ESBL-producing bacteria. As a negative control, an additional slot is set out for urine, nitrocefin and inhibitors without any resistant bacteria or protein. In this way, we have created a harmless, but effective way of telling PhoneLab what a positive result looks like, and integrated it into our design.
Future aspects:
If we were to further advance our project, the imminent steps to be taken would be to realize our CRISPR/Cas9 system, optimize the set-up with clinically isolated ESBL-samples and purifying class B and C ESBL-proteins.
Contact
Blindernveien 31, 0371 Oslo
uioslonorway@gmail.com
