<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UiOslo_Norway_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UiOslo Norway/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row nav"><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway">HOME</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Safety">SAFETY</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Description">PROJECT</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Experiments">LAB</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Parts">PARTS</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Human_Practices">PRACTICES</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Team">TEAM</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Entrepreneurship">AWARDS</A></DIV><DIV class="col-md-2 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Attributions">ATTRIBUTIONS</A></DIV></DIV><DIV class="row secondmenu"><DIV class="col-md-2 secondmenuelement"><A class="secondmenuheader" href="#protocols"> PROTOCOLS </A></DIV><DIV class="col-md-2 secondmenuelement"><A class="secondmenuheader" href="#proofofConcept"> PROOF OF CONCEPT </A></DIV><DIV class="col-md-2 secondmenuelement"><A class="secondmenuheader" href="#demonstrate">  DEMONSTRATE </A></DIV><DIV class="col-md-2 secondmenuelement"><A class="secondmenuheader" href="#results"> RESULTS </A></DIV><DIV class="col-md-2 secondmenuelement"><A class="secondmenuheader" href="#notebook"> NOTEBOOK </A></DIV></DIV><DIV class="row subsite"><DIV class="col-md-12"><DIV id="protocols"><H2 class="boxheader middle">Protocols</H2></DIV><DIV class="row"><DIV class="col-md-6"><DIV id="accordion" class="proto" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
          Nitrocefin titration protocol
        </A></H4></DIV><DIV id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><P class="boxnotes leftext"><B>EQUIPMENT, MATERIALS AND SOLUTIONS</B></P><P class="boxnotes leftext">
     Pipettes
      Pipet tips, assorted sizes
      50 ml falcon tubes
      Urine
      MQ water
      Spectrophotometer
      Nitrocefin(41906-86- 9  Oxoid) and Nitrocefin reconstitution fluid (67-68- 5 Oxoid 5 Reconstitution fluid: Dimethyl sulphoxide (DMSO) 2mL)
  Overnight E. Coli culture with amp resistance, (Bl21 plasmid) 
    Waste container for contaminated liquids
     </P><P class="boxnotes leftext"><B>PROCEDURE</B></P><P class="boxnotes leftext">We always wear lab coat when working in the lab.</P><P class="boxnotes leftext"><I>Overnight culture.</I></P><P class="boxnotes leftext">1. Label all containers to be inoculated, either plates or flasks/tubes with liquid
medium. Flasks/tubes should be labelled by writing on autoclave tape to signify
live bacteria and as a control of waste treatment to signify when the culture has
been inactivated after use.</P><P class="boxnotes leftext">2. Transfer bacteria from culture to new media using applicable equipment,
bacterial loops for bacteria in frozen sample or on plate, or sterile pipette for
liquid culture.</P><P class="boxnotes leftext">3. Add also the appropriate antibiotic.
</P><P class="boxnotes leftext">4. Close all containers to avoid unintentional exposure or contamination
immediately after transfer of bacteria.</P><P class="boxnotes leftext">5. Mix or streak out depending on media type.</P><P class="boxnotes leftext">6. Incubate cultures at 37˚C in designated incubators for 12 – 18h.</P><P class="boxnotes leftext">7. Discard all contaminated loops and tips as bacterial waste and clean work area with 70% ethanol.</P><P class="boxnotes leftext"><I>Preparing the Nitrocefin solution</I></P><P class="boxnotes leftext">1. Add 2mL of reconstitution fluid to Nitrocefin (powder) and shake well.
Working concentration 500mg/ml.</P><P class="boxnotes leftext">2. Nitrocefin is light sensitive and should be kept in the dark.</P><P class="boxnotes leftext">3. Prepare the urine sample, dilute bacterial suspension with urine or other
suitable buffers to obtain appropriate concentrations.
Spectrophotometer measurements</P><P class="boxnotes leftext">4. Add suitable amount of the different dilutions of Nitrocefin to the urine and
bacterial samples.</P><P class="boxnotes leftext">5. Transfer the sample into a kuvette with a pipette, often 1mL is enough.</P><P class="boxnotes leftext">6. Measure the absorbance in spectrophotometer at 486nm.</P><P class="boxnotes leftext">7. Wash the kuvettes for each new sample with dH2O.</P><P class="boxnotes leftext lastp">8. Discard all contaminated consumables and liquids as risk waste and clean work area with 70% ethanol.</P></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingThree"><H4 class="panel-title"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
          Transformation protocol
        </A></H4></DIV><DIV id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree"><P class="boxnotes leftext"><B>Transformation into chemically competent cells:</B></P><P class="boxnotes leftext">1. Thaw 100ul aliquot competent cells on ice.Flick gently once to mix. OBS! Do not vortex, as this can shear the cells. </P><P class="boxnotes leftext">2. Add 0.5-1 ul of plasmid DNA that is at 100 ng/ul concentration. </P><P class="boxnotes leftext">3. Flick gently to mix the solution.</P><P class="boxnotes leftext">4. Incubate on ice for 30 min.</P><P class="boxnotes leftext">5. Heat shock at 42C for 30s.</P><P class="boxnotes leftext">6. Incubate on ice for 2-3 min</P><P class="boxnotes leftext">7. Add 900ul SOC medium, grow for 1 hour at 37C <A href="http://bitesizebio.com/10246/fast-track-your-ampicillin-plasmid-transformations/" target="_blank">This step can be omitted if plasmid encodes for ampicillin resistance.</A></P><P class="boxnotes leftext">8. Spin down gently (5 min at 3000g)</P><P class="boxnotes leftext">9. Remove 900 ul supernatant</P><P class="boxnotes leftext">10. Resuspend cells into 100 ul of media</P><P class="boxnotes leftext">11.  Plate out onto plates with appropriate antibiotic for selection. </P><P class="boxnotes leftext lastp">12. Incubate overnight at 37C and store at 4C afterwards. </P></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="false" aria-controls="collapseTwo">
          Restriction digestion protocol
        </A></H4></DIV><DIV id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><P class="boxnotes leftext">This is a general restriction digest protocol that the Uioslo Norway team used frequently. Depending on how much DNA to digest, the amount of buffer and water will vary. If digesting 100ng DNA 0.5uL of restriction enzymes is recommended and the total volume should be 10 or 20uL. If digesting more DNA, increase also the total volume.  </P><P class="boxnotes leftext">This protocol was used for the restriction enzymes XbaI, EcoRI, SpeI and PstI.</P><P class="boxnotes leftext"><B>Restriction Enzyme:</B> Usually 1uL <B>DNA:</B> 100ng-1000ng<B>NEB buffer 2:</B> 5uL – 10uL<B>BSA:</B> 0.5uL<B>Nuclease free water:</B> Up to 20uL – 50uL depending on how much DNA added.</P><P class="boxnotes leftext">Incubate the reaction on 37°C (check optimal conditions of enzymes used) for 1h.
Heat kill the reaction on 80°C for 20min.</P><P class="boxnotes leftext"><B>SAP treatment:</B>
Shrimp alkaline phosphatase will dephosphorylate the 5’ end of DNA, and prevent the vector from religation. 
We used SAP after every digestion reaction with vector.
1uL of SAP was added to each restriction reaction.
Incubate at 37°C for 30min and heat kill at 65°C for 5min.
</P><P class="boxnotes leftext lastp"><B>Purification:</B>
The restriction digest products should always be purified before proceeding to ligation.
Usually a spin column or gel extraction is performed.
UiOslo Norway team used Quiagen PCR clean up kit spin column, which worked nicely without having to lose too much DNA during the purification.
When using spin column, follow the protocol provided by the manufacturer.
Measure the DNA concentration before proceeding to ligation.</P></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseTwo">
          PCR protocol
        </A></H4></DIV><DIV id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><P class="boxnotes leftext">1. Prepare a “Phusion mix” with the total volume of 40 µl:</P><P class="boxnotes leftext">5X GC Buffer: 8 µl 
10mM dNTPs: 0,8 µl
Phusion Polymerase: 0,5 µl
Forward primer: 2 µl
Reverse primer: 2 µl
Template 20ng/µl: 0,5 µl
H2O: 26,2 µl
</P><P class="boxnotes leftext">2.  Prepare the program on the PCR machine:</P><P class="boxnotes leftext">Comment: Remember to adjust step 3 to match the specific primers used in the reaction. The annealing temperature depends on length and G-C content. If the annealing temperature is unknown, a gradient PCR should be performed. </P><P class="boxnotes leftext lastp">3. When the PCR reaction is finished:
 1. Add 1 µl DpnI into the “Phusion mix” and incubate at 50®C for 1 hour.
 2. Do a PCR cleanup kit and follow the accompanied protocol. 
 3. Run the PCR product on a gel for verification. </P></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseFive" aria-expanded="false" aria-controls="collapseTwo">
          Miniprep Protocol
        </A></H4></DIV><DIV id="collapseFive" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><P class="boxnotes leftext"><B>EQUIPMENT, MATERIALS AND SOLUTIONS</B></P><P class="boxnotes leftext">Pipettes
Pipette tips 
Quiagen miniprep kit 
Eppendorf tubes 
Centrifuge for microcentrifuge tubes
Vortex mixer</P><P class="boxnotes leftext"><B>EQUIPMENT, MATERIALS AND SOLUTIONS</B></P><P class="boxnotes leftext"><I>We always wear gloves and lab coats when working in the lab.</I></P><P class="boxnotes leftext">Plasmid isolation is carried out according to the protocol from the manufacture. The only deviation from the protocol is a reduction in the miniprep elution volume from 100 to 50 µl, to obtain a higher plasmid concentration.</P><P class="boxnotes leftext">Before the first use of the Miniprep kit, add RNase to the Buffer A1, and store at 4 °C. Add the indicated volume of ethanol to Buffer A4 and Buffer AQ.</P><P class="boxnotes leftext">In advance cultivate <I>E.coli</I> cells; P-SOP# Growing bacterial cultures.</P><P class="boxnotes leftext lastp">Clean the table with 70% ethanol.
1. Cultivate and harvest bacterial cells. Pellet 1- 5 mL of bacterial cells.
2. Cell lysis.
3. Clarification of the lysate. 
4. Bind DNA, add binding buffer. 
5. Wash silica membrane.
6. Dry silica membrane, let the column stay on the bench for 1min.
7. Elute DNA, use smaller volumes to obtain higher concentrations of DNA.</P></DIV></DIV></DIV></DIV><DIV class="col-md-6"><DIV id="accordion" class="proto" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseSix" aria-expanded="false" aria-controls="collapseTwo">
        Lysis buffer recipe
        </A></H4></DIV><DIV id="collapseSix" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><P class="boxnotes leftext"><B>100 mL lysis buffer with:</B>
- 1% Triton X-100
- 0,1 M NaCl 
- 10 mM Tris (pH 8,0) 
- 1mM EDTA </P><P class="boxnotes leftext"><B>How to make lysis buffer:</B>
- Add 91 mL MQ H2O to a 200ml bottle 
- Add 8 mL of Tris-HCl (pH 8) buffer (see recipe further down)
- Add 0,037g of EDTA 
- Add 0,58g of NaCl 
- Add 1 mL of Triton </P><P class="boxnotes leftext"><B>How to make Tris-HCl(pH 8) buffer:</B>
- Add 80 ml MQ H2O 
- 1,2114 g Tris. 
- Adjust the pH down to 8 by dropwise adding HCl (1M). </P><P class="boxnotes leftext"><B>Numbers used in calculation:</B>
- MW NaCl: 58,4428 g/mol  
- MW Tris: 121,05 g/mol 
- MW EDTA: 372,24 g/mol  </P><P class="boxnotes leftext lastp"> Reference: Mura et al. </P></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseSeven" aria-expanded="false" aria-controls="collapseTwo">
          Making of inhibitor solutions
        </A></H4></DIV><DIV id="collapseSeven" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><P class="boxnotes leftext">To make solution containing 50 mM DPA, 5mM CVA and 90 mM APB, mix the following compounds: </P><P class="boxnotes leftext">To perform experiments, mix the following solutions: </P><P class="boxnotes leftext">The bacterial lysate can be replaced with intact bacteria or purified protein as one desires. </P><P class="boxnotes leftext">Interpretation of readouts: 
- Increase in A486 after adding solution A indicates class A beta lactamase 
- Increase in A486 after adding solution B indicates class B beta lactamase 
- Increase in A486 after adding solution C indicates class C beta lactamase 
- Increase in A486 after adding solution R indicates beta lactamase of a different class than A, B and C. </P><P class="boxnotes leftext lastp">If precipitation occurs, the volumes of the solvents when mixing the A, B,C and R can be changed to the following: 2 mL HEPES, 2 mL phosphate buffer and 6 mL DMSO. In addition, the solutions can be warmed up to 37*C, vortexed and sonicated. This will in our experience resolve precipitation issues and not affect the readouts of the experiment. 
</P></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseEight" aria-expanded="false" aria-controls="collapseTwo">
          Ligation Protocol
        </A></H4></DIV><DIV id="collapseEight" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><P class="boxnotes leftext">Both vector and insert should be purified in advance before performing ligation.</P><P class="boxnotes leftext"><B>T4 DNA ligase buffer:</B> 5uL<B>T4 DNA ligase:</B> 1uL<B>Vector DNA (2kb):</B> 50ng<B>Insert DNA (1kb):</B> Usually 3 times more than vector<B>Nuclease free water:</B> Up to 20uL</P><P class="boxnotes leftext">A molar ratio between 1:1 and 1:3 of vector and insert is recommended.
For ligation with only one insert, we used a molar ratio of 1:3. 
For ligation with 2 inserts we used a molar ratio of 1:1. (See 3A assembly on iGEM webpage)</P><P class="boxnotes leftext lastp">Incubate 10min or longer in roomtemperature and heat inactivation at 65°C for 10min.
Transform ~5uL of the ligation reaction into 100uL competent cells.</P></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseNine" aria-expanded="false" aria-controls="collapseTwo">
          Gibson assembly protocol
        </A></H4></DIV><DIV id="collapseNine" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><P class="boxnotes leftext">Gibson assembly was performed to generate some of our biobricks
Gene insert was ordered from IDT and was designed so prefix and suffix flanks each side of the gene.</P><P class="boxnotes leftext">Prior to G.A:</P><P class="boxnotes leftext">In order to have enough vector we put on an overnight culture of an already established biobrick in our lab that contains the plasmid backbone pSB1C3. (See protocol for overnight culture).</P><P class="boxnotes leftext"><B>Miniprep</B>
(See miniprep protocol)
The minipreps was compined and DNA concentration was measured  with NanoDrop.
</P><P class="boxnotes leftext"><B>PCR:</B>
Polymerase chain reaction was performed on the purified DNA to amplify the amount so it would be sufficient to perform Gibson assembly. 
</P><P class="boxnotes leftext">The PCR reaction:</P><P class="boxnotes leftext"><B>Dpn1 digestion:</B>
Most of the PCR product was then digested with Dpn1 before run on gel. This is to ensure that the original template is degraded.
 - Add 1 unit of enzyme per 1ng DNA.
 - 1h incubation at 37C, then 20min at 80C to terminate.</P><P class="boxnotes leftext"><B>Gel electrophoresis:</B>
By using specifically designed primers only the vector will be amplified, and not the insert from the PCR. Samples was run on gel electrophoresis to check which bands we have. PCR product should be 2070bp.
</P><P class="boxnotes leftext">Samples run on gel:
1. PCR product (Uncut plasmid) Total plasmid is 5644bp
2. PCR product cut once
3. PCR product with Dpn1 digestion.
4. Ladder
</P><P class="boxnotes leftext"><B>Gel purification:</B>
PCR product with Dpn1 digestion was purified from gel using a Gel Purification kit.</P><P class="boxnotes leftext">DNA concentration was measured with NanoDrop.</P><P class="boxnotes leftext">To perform Gibson assembly, the amount of insert needs to be 2-3 fold in excess compared to amount of vector.</P><P class="boxnotes leftext">Usually the smaller the volume of the reaction the better.</P><P class="boxnotes leftext">Incubate at 37°C for 1h. 15min is also ok.</P><P class="boxnotes leftext lastp">Transform ~5uL of product into 100uL chemically competent cells.</P></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTen" aria-expanded="false" aria-controls="collapseTwo">
           Colony PCR protocol
        </A></H4></DIV><DIV id="collapseTen" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><P class="boxnotes leftext">Colony PCR is performed to check if you have successfully cloned your insert into your vector.</P><P class="boxnotes leftext"><B>EQUIPMENT, MATERIALS AND SOLUTIONS</B>
Pipettes
Pipette tips
PCR tubes (0.2 mL)
Colonies grown on agar plates
Nuclease-free water
Primers
Hot star master mix (5x)</P><P class="boxnotes leftext"><B>PROCEDURE:</B>
        1. Mark the colonies (usually 5) on the plate that you want to run colony PCR on.</P><P class="boxnotes leftext">2. Make a mastermix of the PCR reaction: 
For one reaction you need: 
10uL Hot Star master mix (5x) 
1uL template 
5uL Primer F (10uM) 
5uL Primer R (10uM) <U>3ul Nuclease – free water</U>
        24uL in total </P><P class="boxnotes leftext"> 3. Use a pipette tip to scrape a little bit of the culture and transfer it to the PCR tube, rub the pipette tip along the side of the tube. Then transfer the tip to a prepared tube with LB medium. This can be set to grown overnight.</P><P class="boxnotes leftext">4. Use a pipette tip to scrape a little bit of the culture and transfer it to the PCR tube, rub the pipette tip along the side of the tube. Then transfer the tip to a prepared tube with LB medium. This can be set to grown overnight. </P><P class="boxnotes leftext lastp">5. Run the PCR on a thermocycler. Different temperature must be set according to primers used.
If using the iGEM VF2 and VR the following program may be used: 
1. 94°C 3min
2. 94°C 30sek
3. 58°C 30sek
4. 72°C 1min30sek 
5. Repeat step 2-4 35 times 
6. 72°C 10min 
7. 4°C pause </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row subsite"><DIV class="col-md-12"><DIV class="row"><DIV class="col-md-6"><DIV id="proofofConcept"><H2 class="boxheader right">Proof Of Concept</H2></DIV><P class="boxnotes rightext">
 
As a proof of concept, we have successfully demonstrated all the central aspects of our projects with functioning set-ups.</P><P class="boxnotes rightext"><B>
Lab: </B>
- Our colorimetric assay with nitrocefin is tested and validated, and our detection limits are within a clinically relevant range </P><P class="boxnotes rightext">
- Using BBa_K1189031, a beta lactamase encoding biobrick designed by the Calgary team of 2013, we successfully showed that by using different combinations of inhibitors, we can pin-point the enzymatic class of the beta lactamase present in the samples.
</P><P class="boxnotes rightext">
- By purifying the BBa_K1189031 protein, testing it with nitrocefin and inhibitors and creating and validating our own biobricks for class A, B and C beta lactamases, we have demonstrated safe positive controls to test for these three classes of ESBLs. </P><P class="boxnotes rightext"><B>Hardware: </B>
Our 3D-printed case successfully interfaces urine samples to the phone, illuminates the sample,places and holds the samples on a optimal distance for camera focus. </P><P class="boxnotes rightext"><B>Software: </B></P><P class="boxnotes rightext">
We have successfully created an app with an user interface that accesses the camera, reads and analyze the color change induced by the cleavage of nitrocefin from a picture taken in our software. The app also includes an information page where users can read about guidelines regarding UTIs. </P><P class="boxnotes rightext">
Thus, we present a functioning chassis for a mobile diagnostic platform that easily, and within minutes can identify the presence of ESBL producing bacteria in urine samples, without ever having to expose any bacteria to antibiotics. 


   </P></DIV></DIV></DIV></DIV><DIV class="row subsite"><DIV class="col-md-12"><DIV id="demonstrate"><H2 class="boxheader middle">Demonstrate</H2></DIV><VIDEO class="labvid" autoplay=""><P class="boxnotes leftext imgcap"><I>Demonstrating our diagnostic tool in real-time.</I></P></VIDEO></DIV></DIV><DIV class="row subsite"><DIV class="col-md-12"><DIV id="results"><H2 class="boxheader middle">Results</H2></DIV><P class="boxnotes leftext">
  
Here follows a selection of our results, with the physiologically relevant results highlighted. For the total collection of our results, go to the “Total Results” section. </P><P class="boxnotes leftext">
All of the following graphs are results from adding 20 ul of 500 mg/mL nitrocefin to a total volume of 1 mL of urine, bacteria and purified protein. </P><P class="boxnotes leftext">
In our first experiment we measured both intact and lysed bacterias respons to Nitrocefin. The absorbance was measured at 486nm as recommended from the Mura et. al. 2015 paper. https://www.clinicalkey.com/#!/content/journal/1-s2.0-S0732889315002011
As we wanted our test to be a quick method we measured time intervals from 0 - 40min. </P><P class="boxnotes leftext">
Results from the initial experiments</P><P class="boxnotes leftext">
These measurements was done with synthetic urine ordered online. The red line in the graphs above signifies a clinically relevant bacterial number, along with a detectable change in A486 as the result of hydrolyzation of nitrocefin. </P><P class="boxnotes leftext">
The graphs below displays the same type of experiments as above but in real urine as we wanted to confirm that the test would work under physiological conditions.</P><P class="boxnotes leftext">
AmpR Bacteria in Real Urine: </P><P class="boxnotes leftext">It is clear that real urine somewhat disturbs the signal. However, the signals shown here are of a much higher intensity than necessary to be detectable in PhoneLab, here being visible even to the human eye. </P><P class="boxnotes leftext">
This indicates that even with a lower bacterial count, we could still have gotten measurable readouts. </P><P class="boxnotes leftext">

This graph displays measurements done with the purified protein from BBa_K1189031. This protein function as our positive control to our test. </P><P class="boxnotes leftext">
As one can observe from the graph the hydrolisation of the beta lactam ring is clearly inhibited by the class A inhibitor. The solution without the inhibitor have a clear color change and a strong absorbance after only 5 min. </P><P class="boxnotes leftext">
We also did one experiment with different dilutions of purified protein to investigate the detection limit. Below is the results where we observed that the cut - off value should be set to 1000 - fold dilution of protein.</P><P class="boxnotes leftext">These results represents the foundation of our diagnostic test. Here we have shown that Nitrocefin will react with different amounts of bacteria present, both intact and lysed as well as purified β - lactamase. </P><P class="boxnotes leftext"><B>Overview over total results (in detail)</B></P><P class="boxnotes leftext">Here follows the total results from our experiments. All graphs shown below represent results from experiments performed with E. coli and protein in combination with an added 20ul of 500 mg/mL nitrocefin to a total volume of 1 mL. </P><P class="boxnotes leftext">AmpR E.coli in synthetic urine: </P><P class="boxnotes leftext"> These graphs shows different dilutions of bacteria in synthetic urine and the following changes in A486 as a response to the hydrolyzation of nitrocefin. The graphs clearly show that lysing the bacteria yields a lower detection limit than that obtained by using intact bacteria. </P><P class="boxnotes leftext">AmpR E.coli in real urine:  </P><P class="boxnotes leftext">All of the above graphs show a measurable increase in A486 over time following the hydrolyzation of nitrocefin with varying bacteria numbers. These experiments were performed with a higher bacteria count than the ones with synthetic urine. </P><P class="boxnotes leftext">The protein used here is the expressed and purified <A href="http://parts.igem.org/Part:BBa_K1189031" target="_blank">BBa_K1189031</A> produced by the Calgary team of 2013. It is clear that adding of CVA, an inhibitor of class A beta-lactamases effectively stops the enzymatic reaction even at very high protein concentrations. </P><P class="boxnotes leftext">The above graph shows the essence of our proof of concept. Following transformation of chemically competent Top10 E.coli and IPTG-induced expression, we lysed the bacteria and added nitrocefin along with different combinations of inhibitors. The graph clearly shows that class A inhibitors (CVA) effectively stops the reaction in the cuvettes of both the purified protein (positive control) and that with lysed bacteria. Class B and C inhibitors (DPA and APB, respectively) on the other hand, does not affect the reaction, and a visible color change was seen (shown by the blue and orange lines on the graph). The negative control here was lysed bl21 E.coli without AmpR. The experiment was performed in synthetic urine. </P><P class="boxnotes leftext">The UiOslo team used both purified and bacterial form of part <A href="http://parts.igem.org/Part:BBa_K1189031" target="_blank">BBa_K1189031</A> (referred to in the graph as class A biobrick and class A protein, respectivly) and contributed to further characterization of this part. These results display our contribution to the iGEM registry by showing BBa_K1189031 reaction to Nitrocefin and inhibitor for class A beta - lactamases.For more detailed results and explanations check out part BBa_K1189031 on the iGEM registry main page. Thus, UiOslo team confirms the function of BBa_K1189031 and presents new information and characterization of this part. </P><P class="boxnotes leftext"><B>Determine buffer capacity of urine:</B>
    This was a pilot experiment done in the first weeks of lab start – up. 
</P><P class="boxnotes leftext">We wanted to investigate the possibility of measuring small pH changes in urine, as the cleavage of Nitrocefin and other compounds that contain a beta lactam ring will result in release of H+.</P><P class="boxnotes leftext">In this experiment we added small amounts of HCL to a urine sample and measured the pH. We had two urine samples from two different individuals. These results were obtained from our initial experiments, proving that pH measurements was not a viable option for us.</P><P class="boxnotes leftext imgcap"><I>
Graph 1: pH measurements from urine sample 1, 2 and a mixed sample. As we can see from the 
graph the pH does not change significantly when adding HCL. By adding 150uL 
 of HCL 1M the pH has only changed from 5.25 to 5. This is a direct effect of 
 the buffer capacity of urine. Same results was obtained from urine sample 2.
</I></P><P class="boxnotes leftext"><B>Conclusion:</B> Even though pH is a logarithmic scale, the small changes in pH observed are not enough for our purposes. We added a quite strong acid, and a beta lactam ring hydrolysis will not release enough H+ to be detected this way.  Other error sources are pH in urine is different for each individual, and is influenced by food consumption and how much and what you drink. </P><P class="boxnotes leftext"><B>pH change in urine in presence of bacteria:</B>
Before we decided to move away from the pH measurements we wanted to investigate one last thing. Would bacteria in the urine sample affect pH?
To investigate this we used urine samples collected from two team members and added E. Coli BL21 to the samples. 
</P><P class="boxnotes leftext">The E.Coli strain had been put on overnight culture the day before.
     OD~1</P><P class="boxnotes leftext imgcap"><I>
 Graph 1: Displays the pH measurements when E. Coli is added to a 
urine sample. As one can observe there are no significant change in 
pH when adding increasing amounts of E. Coli. The small changes that 
do occur will be set as background noise. 
</I></P><P class="boxnotes leftext"><B>Conclusion:</B> The pH does not change significantly when bacteria is added to the sample.  To conclude the overall experiment with pH measurements in urine, we determined not to pursue this direction. The buffer capacity of urine is too strong to measure any small amounts of H+ released.</P><P class="boxnotes leftext"><B>Generation of biobrick BBa_K1927000</B>
This biobrick’s sequence is collected from a clinical isolate obtained from The National Expertise Center for Antibiotic Resistance in Tromsø. These isolates are collected from different health institutions from all over Norway.
This particular gene encodes the enzyme called blaNDM – 1. </P><P class="boxnotes leftext">
Bacteria containing these genes convey resistance to a broad range of β – lactam antibiotics.
The sequence was designed with specific flanking regions that would make it suitable for Gibson Assembly into the pSB1C3 shipping vector. We decided we would make this biobrick without any promotor because of the safety concerns that follows a multi - resistant gene. </P><P class="boxnotes leftext">
With some help the flanking regions was designed so it could directly be cloned into the shipping vector without any PCR. We performed PCR on our pSB1C3 part that we retrieved from one of the biobricks in the distribution kit, also with specific designed primers.</P><P class="boxnotes leftext"><B>Flanking region of gene:</B></P><P class="boxnotes leftext">
GCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGATG-INSERT-TACTAGTAGCGGCCGCTGCAGTCCGGCAAAAAAGGGCAAG</P><P class="boxnotes leftext"><B>Primers:</B>
V1: tactagtagcggccgctgcagtc 23/64oC/61% 
V2: catctagaagcggccgcgaattc 23/62oC/57%
We performed Gibson Assembly (see protocol for details) and transformed the reaction into TOP10 chemically competent cells. The cells were then plated on LB plates containing chloramphenicol.</P><P class="boxnotes leftext imgcap"><I>Figure 1: A few colonies managed to grow on the plates. Confirming the presence of pSB1C3 
and also that the assembly was successful. Religation of vector does not happen as frequently 
in Gibson Assembly as it does in regular ligation.</I></P><P class="boxnotes leftext">
Even though colonies had grown on the plate we wanted to confirm the presence of our insert. We picked two colonies and performed colony PCR to confirm our insert. We used the primers recommended from iGEM and more details about PCR program you may find under protocols.</P><P class="boxnotes leftext imgcap"><I>Figure 2: We used biobrick BBa_K1189031 as a positive control and empty vector as negative control.  
The positive control seemed to be too big in size (bp) for the annealing time used in the program. 
Thus it did not give a clear band. 
Our biobrick however gives a clear band at around 1000bp which corresponds to the sequence  length,
thus the presence of insert is confirmed.</I>
Lane 1: Ladder	
Lane 2: Positive control, BBa_K1189031
Lane 3: BBa_K1927002
Lane 4 and 5: J04500 (part only)
Lane 6:empty vector
</P><P class="boxnotes leftext imgcap"><I>Figure 3: Displays our biobrick cut with different enzymes. 
NotI did not cut that efficient and the gel displays incomplete cutting.
Lane two is our construct cut once, there is a clear band just above 2000bp which 
indicates that our construct is successfully linearized and show corresponding base pairs.</I>
 Lane 1: 1kb ladder gene ruler
Lane 2: BBa_K1927000 cut once w/XbaI	
Lane 3:BBa_K1927000 cut w/ NotI
Lane 4: uncut plasmid. 
We did an additional restriction digest with the newly made biobrick BBa_K1927001:
</P><P class="boxnotes leftext imgcap"><I>Figure 4: Displays another confirmation that the gene of interest 
is within the shipping vector pSB1C3.</I>
Lane 1: 1kb ladder gene ruler
Lane 2: BBa_K1927000 cut w/XbaI and SpeI
Lane 3:BBa_K1927001 cut w/XbaI and SpeI</P><P class="boxnotes leftext"><B>Generation of biobrick BBa_K1927002 and BBa_K1927003</B>
This biobricks sequence is collected from a clinical isolate and it’s called blaCMY – 6 plasmid – mediated amp. 
This represents a class C enzyme from the broad family of β – lactamases.
</P><P class="boxnotes leftext">The gene was designed in the same way as BBa_K1927000 
(see generation of BBa_K1927000) and is designed with specific flanking regions and without promoter for safety reasons.</P><P class="boxnotes leftext imgcap"><I>Figure 1: Result from Gibson Assembly. Transformed G.A product 
into TOP10 chemically competent cells. Colonies grown on 
chloramphenicol plates.</I></P><P class="boxnotes leftext imgcap">To confirm that the insert is present we had the plasmid digested with XbaI and PstI along with BBa_K1927000 as a positive control.</P><P class="boxnotes leftext imgcap">Lane 1: 1kb ladder gene ruler
Lane 2: BBa_K1927000 cut w/ XbaI and PstI
Lane 3:BBa_K1927001 cut W/XbaI and PstI</P><P class="boxnotes leftext imgcap">The cut insert shows band at 1000bp which confirms the generation of the desired construct.
</P><P class="boxnotes leftext"><B>Generation of biobrick BBa_K1927002 and BBa_K1927003</B>
We wanted to calibrate our detection test with a biobrick of our own, in addition to the positive control (E. coli amp), negative control (E. coli without antibiotic resistance) and the purified protein from biobrick BBa_K1189031 as an additional positive control.  </P><P class="boxnotes leftext">
We retrieved the gene sequence for the Amp-gene from the standard E. coli vector pUC19 (http://www.snapgene.com/resources/plasmid_files/basic_cloning_vectors/pUC19/). The AmpR gene product is a β-lactamase class A which confers resistance to ampicillin, carbenicillin, and related antibiotics. The gene was designed to include prefix, suffix and start of vector sequence to make it ideal for Gibson assembly later on. Gene and primers was synthetized by IDT.  We performed a Gibson assembly with the shipping vector pSB1C3 and our gene to generate biobrick BBa_K1927002. </P><P class="boxnotes leftext imgcap"><I>
Figure 1: LB agar plate with chloramphenicol for growth of transformed 
bacteria(TOP10) with BBa_K1927002. The insert was inserted into the shipping 
vector pSB1C3 by Gibson assembly. </I></P><P class="boxnotes leftext imgcap"><I>
Figure 2: Gel analysis of colony PCR with BBa_K1927002. 
The insert was inserted into the shipping vector pSB1C3 by using Gibson assembly.  
  Lane 10 are DNA ladder (GeneRuler 1 kb DNA ladder, Thermo Fisher Scientific). 
Lane 11 are positive control (validated part BBa_K1927000). 
 Lane 18-19 are different colonies from Gibson assembly. They show the appropriate 
band at approx. 1000bp.</I></P><P class="boxnotes leftext">
But since the pSB1C3 vector is not an expression vector, we could not use this for calibrating our test. We therefore looked into the iGEM library and found BBa_J04500 which has a ribosomal binding site and IPTG inducible promotor. By combining AmpR gene downstream for the BBa_J04500 we could theoretically express the gene product which is a β-lactamase class A. This was done by restriction digestion of part BBa_J04500 with Spel and Pstl and the AmpR-insert with Xbal and PstI. Restriction digest was followed by ligation and transformation of TOP10 cells and grown on LB agar plate with chloramphenicol. Colony PCR was performed to verify successful ligation with our insert. Restriction digestion of miniprepped DNA was performed to confirm prior to sequencing.  </P><P class="boxnotes leftext imgcap"><I>Figure 3: Gel analysis of colony PCR with BBa_K1927003.
 The insert was inserted into the shipping vector pSB1C3 by using 2 different approaches; 
3A assembly and a single restriction digest followed by ligation. 
Lane 1 and 10 are DNA ladder (GeneRuler 1 kb DNA ladder, Thermo Fisher Scientific). 
 Lane 2 and 11 are positive control (validated part BBa_K1927000). 
Lane 3-7 are different colonies from 3A assembly.  Lane 8-9 and 12-14 are restriction digest 
and ligation approach. Lane 4, 6-8, 12-13 shows the appropriate band at approx. 1000bp.</I></P><P class="boxnotes leftext imgcap"><I>
Figure 4: Restriction digest of miniprepped DNA with restriction enzymes Pstl and Xbal. 
Lane 1 shows DNA ladder (GeneRuler 1 kb DNA ladder, Thermo Fisher Scientific),  
while lane 1-7 shows the same digest with miniprepped DNA derived from different 
bacteria colonies (transformed competent TOP10 cells with BBa_K1927003). 
Lane 1, 6 and 7 shows the appropriate bands with the insert at approx.  
1000bp and vector at approx. 2000bp.</I></P><P class="boxnotes leftext imgcap"><I>
Figure 5: LB agar plate with chloramphenicol for growth of transformed bacteria (TOP10)
BBa_K1927003. The insert was inserted into the shipping vector pSB1C3 by 
using 2 different approaches; 3A assembly (1) and a single restriction digest followed 
by ligation (2).</I></P><P class="boxnotes leftext">
The bacteria were then grown on a LB agar plate with ampicillin and IPTG for selection and expression for colonies with BBa_K1927003. These bacteria with our biobrick was further experimentally validated by performing our nitrocefin experiment setup. </P><P class="boxnotes leftext imgcap"><I>
Figure 6: LB agar plate with ampicillin and IPTG for 
growth of transformed bacteria (TOP10) with BBa_K1927003. </I></P><P class="boxnotes leftext"><B>
Functional validation of biobrick BBa_K1927003</B>
The biobrick consist of an ampicillin resistant gene called ampR, and its sequence is collected from the pUC19 vector online. To functionally validating one of our brick we made the shipping vector pSB1C3 into an expression vector. We did a lot of research and decided to try the inducible promotor part J04500 http://parts.igem.org/Part:BBa_J04500  that came with the distribution kit. With this promotor upstream placed upstream of our gene of interest we could induce the transcription of our ampicillin resistant gene. Se drawing below:</P><P class="boxnotes leftext imgcap"><I>
Figure 7: Illustration of the plan to combine the two parts  
BBa_J04500 and BBa_K1927002 for expression of gene product 
β-lactamase class A.</I></P><P class="boxnotes leftext">
To generate our biobrick we used the 3A assembly protocol recommended by igem. http://parts.igem.org/Help:Protocols/3A_Assembly, where J04500 is part A and our ampR gene is part B. See protocol for more information.</P><P class="boxnotes leftext">
To functional validate our brick we made overnight cultures of the hopeful colonies and plated them on agar plates containing IPTG (final volume of 1mM) and ampicillin. The plate was incubated overnight at 37 degrees.  Colonies managed to grow on these plates, which confirm the biobricks beta lactamase activity.</P><P class="boxnotes leftext"><B>The diagnostic tool</B>
With an OD of ~1 we made a 16 and 64 times dilutions of the bacteria. This was to reduce the amount of bacteria so the sample is biologically relevant. </P><P class="boxnotes leftext imgcap"><I>
Graph 1: As observed for both the graph and the tables above the 
absorbance at 486nm increases gradually with time. 
The cleavage of Nitrocefin generates a red color in the sample, 
which is visible by eye. The color will also depend on the amount of 
bacteria in the sample as shown in graph 1. The 64 times diluted bacteria have a lower 
absorbance in most of the time points.  </I></P><P class="boxnotes leftext imgcap"><I>
Figure 8: Figure 1 displays the inside of our 3D model. 
Cuvette to the left is synthetic urine and bacteria, but without Nitrocefin.  
The cuvette to the right is same sample only with Nitrocefin.  
This picture is taken after 20 minutes and it’s clear that the color change is visible by eye.</I></P></DIV></DIV><DIV class="row subsite"><DIV class="col-md-12"><DIV class="row"><DIV class="col-md-6"><DIV id="notebook"><H2 class="boxheader left">Notebook</H2></DIV><P class="boxnotes leftext">
 Below is our lab notebook where you can read about our experiences in the laboratory during the iGEM season in both good days and bad. Enjoy!</P><P class="boxnotes leftext"><B>Week 18</B>
First week of working in the lab. The team used some time to get to know the lab properly. 
We also performed a pilot experiment to investigate if we were able to measure small pH changes in urine. Cleavage of a beta lactam ring will result in release of H+, and if it could be detected, we could detect activity of beta lactamases. We also investigated pH changes in urine with different amounts of bacteria present. If bacteria influenced pH in any way, it could be difficult to detect pH changes in urinary tract infections. 
The buffer capacity of urine seemed to be too strong to detect small pH changes and we started to look into other ideas for detection.</P><P class="boxnotes leftext"><B>Week 19</B>
Pilot experiment with a beta lactam derivative called Nitrocefin. We used a constant amount of urine and added different amounts of bacteria to simulate a urinary tract infection. Solution with Escherichia Coli containing plasmid for amp resistance in presence of Nitrocefin turned red. The color change was visible within minutes.</P><P class="boxnotes leftext"><B>Week 20</B>
We investigated the kinetics of the beta lactam derivative Nitrocefin by testing different amounts of synthetic urine and bacteria and look for color change. We also run one experiment where we mimicked a biological urinary tract infections with ~10^5 bacteria. The absorbance was considerable lower, but high enough to get a proper readout.</P><P class="boxnotes leftext"><B>Week 21</B>
No work was done in the wet- lab. We did more research on urinary tract infections and planned experiments ahead.</P><P class="boxnotes leftext"><B>Week 22</B>
The last couple of experiments had been successful, and we wanted to investigate the cut – off value of Nitrocefin. We were wondering how much bacteria could be present to get a proper readout. Thus, the experiment was called titration of bacteria with Nitrocefin. We used triplicates so we could calculate standard deviation, which would give the results more weight.</P><P class="boxnotes leftext"><B>Week 23</B>
We did the same experiment as last week but with lysed bacteria to investigate if we could lower the detection limit. The results showed that the detection limit was lowered with lysed bacteria.</P><P class="boxnotes leftext"><B>Week 24</B>
No work in the wet lab this week. We had come a long way with developing the test and the team had to sit down with the supervisor and discuss further development.</P><P class="boxnotes leftext"><B>Week 25</B>
By this time, we understood that we needed standardize these type of experiments. By running triplicates, we could easily standardize our experiments. Synthetic urine was bought online. 
We repeated the previous experimental setup with live bacteria and with synthetic urine to investigate if it would influence our Nitrocefin read-outs. The experiment was successful and from now on we started working only with synthetic urine.</P><P class="boxnotes leftext"><B>Week 26</B>
This week we did an experiment to investigate how much Nitrocefin was needed to give a proper readout and measurable absorbance. Until now, we had been working with the stock concentration that came from the manufacturer. </P><P class="boxnotes leftext"><B>Week 27</B>
We repeated the last experiment with synthetic urine and Nitrocefin absorbance measurements. This time we tested the synthetic urine with lysed bacteria to investigate if the absorbance still was measurable as it was with real urine.</P><P class="boxnotes leftext"><B>Week 28</B>
No experiments in the lab. We searched the iGEM registry for usable biobricks to our project, we found one β-lactamase from Calgary team 2013 and had it ordered online.</P><P class="boxnotes leftext"><B>Week 29 and 30</B>
We used these two weeks to work with the β-lactamase from the Calgary 2013 iGEM team. We received a glycerol stock and transformed the bacteria into cells more suitable for protein purification. 
We managed to purify the β-lactamase protein successfully by using the protocol from Calgary 2013 wiki webpage. We used Bradford assay to determine the protein concentration.</P><P class="boxnotes leftext"><B>Week 31</B>
By now it was time to think Biobricks! We needed to generate biobricks of our own to send to the iGEM registry. We had ordered an ampR gene from IDT and planned to clone it into the iGEM shipping vector pSB1C3.
We did restriction cutting of both ampR and pSB1C3.
SAP treatment of vector.
The digest was run on gel electrophoresis and gel purified.
The gel purification was not successful and we lost almost all of our DNA and could not continue with the rest of the cloning.</P><P class="boxnotes leftext"><B>Week 32</B>
Because of last week failed experiment we started planning other ways to generate biobricks. We started looking into Gibson Assembly, which was supposed to be easier and quicker than the classical cloning. We designed primers appropriate for G.A.</P><P class="boxnotes leftext lastp"><B>Week 33</B>
When the primers for G.A had arrived, we performed phusion PCR to prepare for Gibson Assembly. The designed primers for our ampR gene worked perfectly fine. However, the primers for the vector did not seemed to work. We tried different phusion buffers and we performed gradient PCR to check if different annealing temperatures was needed. None of them worked.</P></DIV><DIV class="col-md-6"><P class="boxnotes leftext"><B>Week 34</B>
We had two other genes ordered from IDT. These sequences were collected from ESBL clinical isolates, and we designed the genes with special flanking regions that made the suitable for Gibson Assembly. We also ordered new primers for the new Gibson Assembly setup. </P><P class="boxnotes leftext"><B>Week 35</B>
The different inhibitor compounds that we wanted to use in our diagnostic test had arrived. The compounds are inhibitors of different classes of β-lactamase. We prepared buffers for the different compounds and planned the following experiments with the inhibitors. We wanted to use the Calgary biobrick, the purified β-lactamase and ampicillin resistant E. Coli that we had available in our own lab.</P><P class="boxnotes leftext"><B>Week 36</B>
Pilot experiment with the different three inhibitors of class A, B and C β-lactamase. As we only had class A available we only tested this one in the first run. The experiment was a success and the inhibitor for class A worked perfectly!
We also performed Gibson Assembly of genes with class B and C that we ordered online from IDT.
Transformed the G. A product into chemically competent cells and plated out on agar plates with appropriate antibiotics. The next day we checked for colonies and no colonies had grown. </P><P class="boxnotes leftext"><B>Week 37</B>
We repeated the Gibson Assembly with no success. 
We also performed an experiment with the purified protein from Calgary 2013. We investigated the detection limit for the purified protein and did several dilutions with Nitrocefin. We also tested the class A inhibitor with the protein, and again the inhibitor worked perfectly and inhibited the hydrolization of Nitrocefin. </P><P class="boxnotes leftext"><B>Week 38</B>
After several attempts of Gibson Assembly we discussed the protocol with Athanasios (he also helped us design primers for G.A). With his help, we manage to perform Gibson Assembly successfully with our class B gene and the next day, colonies had grown on the plates. 
We took two cultures from this and did an overnight culture.
The next day we did miniprep to collect the DNA, and measured concentration on NanoDrop. We performed restriction digest one time with only one enzyme and one with NotI that would cut in both prefix and suffix. When we checked the gel NotI had not cut properly and we ran another restriction digest with XbaI and PstI. This digest was successful and there was two appropriate bands on the gel.</P><P class="boxnotes leftext"><B>Week 39</B>
This week we performed Gibson Assembly with our class C gene using the same protocol as last time. The product was transformed into chemically competent cells. Colonies grew on the plates the next day.
We did overnight cultures of a couple of colonies and did miniprep the next day. 
In addition to this, to check that our biobrick was the correct one we performed colony PCR. This was not successful and our positive control did not work either. We repeated the colony PCR with minor changes and by running the products on gel, electrophoresis the colony PCR was successful, as the appropriate bands was present on the gel. 
Both our biobricks were sent for sequencing and confirmed.</P><P class="boxnotes leftext"><B>Week 40</B>
This week we planned to perform 3A assembly with the protocol provided by iGEM. We wanted to make a biobrick with our ampR gene couplet to a promotor.
We transformed the biobrick J04500 and did colony PCR on 5 of the colonies that had grown on the plates.  We did overnight cultures and minipreped the colonies that had the right insert. This part was then restriction digested with the appropriate restriction enzyme according to the 3A protocol.
We did restriction digestion on our ampR gene and the pSB1C3 plasmid backbone according to the 3A protocol. The vector was then SAP treated. Usually after any restriction digest the products either are purified on gel or spin column. The 3A protocol from iGEM did not have any purification step. We continued with ligation of only the gene of interest and pSB1C3. The ligation product was transformed and plated on appropriate plates. No colonies had grown the next day. </P><P class="boxnotes leftext">
We repeated the restriction digest only with more DNA so we could afford to lose some DNA in the purification. The restriction products was purified and placed in the fridge over the weekend.
This week we did some experiments for the NTNU-iGEM team as well.</P><P class="boxnotes leftext"><B>Week 41</B>
We ligated the three products from the restriction digestion last week according to the 3A, protocol only we did two reactions with slightly different concentrations of insert. 
Products was transformed and plated on appropriate antibiotic plates. 
We also performed restriction digest of the minipreped J04500 and planned to ligate our ampR gene directly into this part as this already had a promotor. This seemed like an easier method than the 3A as this protocol include ligation of three parts together. We transformed ligation product as well.
We performed Gibson Assembly of our ampR gene, as we wanted to make a biobrick of this as well, without promotor. We transformed the product and put on agar plates with appropriate antibiotics.</P><P class="boxnotes leftext">
All our ligations and G.A had seemed to work and colonies had grown the next day.
We then did colony PCR on 5 colonies from each plate, did overnight cultures, minipreped two each of the successful colony PCR and run restriction digest of the minipreped DNA to check for our insert.
The restriction digest was also successful.
</P><P class="boxnotes leftext">
This week we also performed our inhibitor experiment with clinical isolates. We did overnight cultures of the isolates and lysed them the next day. We had difficulties dissolving the different inhibitors solutions and we could not get a proper absorbance when bacteria, synthetic urine and Nitrocefin was added to the solution. </P><P class="boxnotes leftext lastp"><B>Week 42</B>
No lab was done this week, as it’s the week of wiki freeze. We felt confident that we had enough results for our project.
   </P></DIV></DIV></DIV></DIV><DIV class="row footer"><DIV class="col-md-4 footerelement"><H2 class="boxheader footerheader">Contact</H2><P class="boxnotes footercontact"> Blindernveien 31, 0371 Oslo</P><P class="boxnotes footercontact"> uioslonorway@gmail.com</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>