<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UiOslo_Norway_Human_Practices skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UiOslo Norway/Human Practices</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row nav"><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway">HOME</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Safety">SAFETY</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Description">PROJECT</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Experiments">LAB</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Parts">PARTS</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Human_Practices">PRACTICES</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Team">TEAM</A></DIV><DIV class="col-md-1 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Entrepreneurship">AWARDS</A></DIV><DIV class="col-md-2 navelement"><A class="navlink" href="https://2016.igem.org/Team:UiOslo_Norway/Attributions">ATTRIBUTIONS</A></DIV></DIV><DIV class="row secondmenu"><DIV class="col-md-1 secondmenuelement"><A class="secondmenuheader" href="#humanprac">HUMAN PRACTICES</A></DIV><DIV class="col-md-1 secondmenuelement"><A class="secondmenuheader" href="#public">PUBLIC ENGAGEMENT</A></DIV><DIV class="col-md-1 secondmenuelement"><A class="secondmenuheader" href="#integrated">INTEGRATED PRACTICES</A></DIV><DIV class="col-md-1 secondmenuelement"><A class="secondmenuheader" href="#gold">GOLD</A></DIV><DIV class="col-md-1 secondmenuelement"><A class="secondmenuheader" href="#silver">SILVER</A></DIV></DIV><DIV class="row subsite"><DIV class="col-md-12"><DIV class="row"><DIV class="col-md-6"><DIV id="humanprac"><H2 class="boxheader left">Human Practices</H2></DIV><P class="boxnotes leftext">We consider Human practices to be one of the most important and rewarding things about iGEM. 
During our project we gain a bigger perspective on synthetic biology, its importance and its impact on society. Our goal was to spread the knowledge not only about synthetic biology, but especially about our project antibiotic resistance in general. We can't fight this battle alone.</P><P class="boxnotes leftext">
Antibiotic resistance involves people all over the globe, and it is impossible to decrease the trend without as many as people being involved.
We have been in touch with several experts in the field and we have held many lectures about our project to other students, from secondary school pupils to master students. We have appeared several newspapers, attended a science festival, got interviewed for a TV-show, been in magazines and even on the radio. </P><P class="boxnotes leftext">You can read more about our work under Education, Public engagement and Integrated practices. We have learned so much while doing this and we have met so many interesting people on the way. Most important of all; it has been so much fun!</P><DIV id="HER"><H2 class="boxheader left">Education</H2></DIV><P class="boxnotes leftext"><B>Increasing knowledge for primary and secondary school pupils</B>
The 27th and the 28th of September &quot;Ungforsk&quot; was arranged at the University. &quot;Ungforsk&quot; roughly translated means young science and was arranged for the 20th time. Secondary school pupils get to visit the University were they can visit labs and learn more about science. We were so lucky that we got to hold a total of six presentations for those who came to visit. Ragnhild, Paul, Marthe and Torleif held those seminars, and they did a really good job.
Torleif also handed out our comic book to Dokka VGS, the high school in the municipality of Nordre Land. Let's hope they'll get more excited about synthetic biology.<B>Increasing knowledge for new students at the university</B>
The 30th of August and the 6th of September we held lectures for new biology students at the University. In addition to spreading the word about iGEM, we presented our project and told the new students of the problem of antibiotic resistance. We held three lectures each day and six in total. The students seemed very interested about iGEM, so we hope next years team will have loads of interested students. Marthe, Hans, Paul and Camilla held those lectures.
Roza and Marthe also held a lecture at Sundvolden the 22nd of August.<B>Increasing knowledge for master students</B>
Ragnhild held a lecture the 16th of August for master students. It was held at Kristine Bonnevies house at the University and there were also pizza afterwards.</P><P class="boxnotes leftext"><B>Learning material for high school students</B>
The UiOslo team have used the iGEm season to teach synthetic biology to kids and high school students. 
Because of this we have generated a 3D file for a bacteria. The idea behind this was to generate a bacteria that could be taken apart
so one could have a look inside the bacteria, of what it contains and how it is build up. 
The 3D-model is made in the online 3D-modelling software at https://www.tinkercad.com. The 3D- 
model represent a E. coli bacterium and shows its DNA, plasmid, ribosomes, pili and flagellum.
Unfortunately because of time limitations and technical difficulties, the 3D file was not printed. 
However the 3D-model is available online, and we have given the file to the “Biologylab” at Department of Biosciences.
Hopefully, they can use another printer and use it for educational purposes. The 3D-model represent
a E. coli bacterium and shows different parts which can be helpful during lectures about e.g. protein
synthesis and gene transfer.

In addition to this, we contributed to generate electron microscopy pictures of our clinical isolates.
These pictures will be used to educate high school students and contribute to increase knowledge 
  about biology among high school students. The bacteria was fixed and sent down to the electron microscopy department
at UiO. These amazing pictures is made by Antje Hofgaard who works as an senior engineer at the department.
  
    </P><DIV id="public"><H2 class="boxheader right">Public engagement</H2></DIV><P class="boxnotes rightext">

We have done a lot to get public attention about iGEM, but also about us and antibiotic resistance. We have used social media, we have a blog, radio, magazines, different newspapers and even published a comic book to hand out. We have also gotten help from the headmaster to write about us in his blog, and we have also made it on different webpages. <B>Facebook and Instagram</B>We are very active on social media. On Facebook we have over 560 followers and we are updating with professional articles and other things that are relevant. On Instagram we have about 300 followers. We share fun stuff as well as relevant articles and keeping people updated about our project.<B>Blog</B>We also have a blog which we try to update regularly. We share things about our project, but also conferences we've attendend, as well as other fun stuff to keep people interested. Visit our webpage <A href="https://uioslonorway.wordpress.com" target="_blank">here.</A><B>Radio</B> We have also been on the radio, on Østlandssending on NRK. NRK is Norway's national channel and everyone have free access to listen. You can listen to it <A href="https://radio.nrk.no/serie/distriktsprogram-oestlandssendingen/DKOA02014516/01-08-2016#t=2h25m15s" target="_blank">here.</A><B>Magazines</B>We have been in the University magazine Apollon. You can read the article <A href="http://www.apollon.uio.no/artikler/2016/3_bakterier_syntetiske.html" target="_blank">here.</A>(It's in Norwegian though).<B>Newspapers</B>
We have also been visible in different newspapers. Both on the internet and on paper. You can read about us <A href="http://www.oblad.no/nyhet/universitetet-i-oslo/resistente-bakterier/oppegard-student-utvikler-viktig-og-billig-bakterietest/s/5-68-191615" target="_blank">here</A> (Østlandets Arbeiderblad), 
<A href="http://rbnett.alda.no/bestillpluss?1&amp;artRefId=13087222&amp;aviskode=ROM&amp;targetUrl=http%253A%252F%252Fwww.rbnett.no%252F%253Fservice%253DpaywallRedirect%2526articleUrl%253Dhttp%253A%252F%252Fwww.rbnett.no%252Fpluss%252Farticle13087222.ece" target="_blank">here</A>(Romsdals Budstikke), and

<A href="http://www.pd.no/lokale-nyheter/porsgrunn/hoyere-utdanning/kjemper-mot-hele-verden/s/5-40-114346" target="_blank">here</A> (Porsgrunn Dagblad). You have to pay a fee to get to read the article.<B>Comic book</B>
We made our own comic book about syntethic biology! You can see some pictures of it on the right side, or you can visit our stand and get one yourself. We handed the comic book to primary and secondary school pupils, but also students at the University!

</P><DIV id="integrated"><H2 class="boxheader left">Integrated Practices</H2></DIV><P class="boxnotes rightext">
Here is a list over activities, seminars we've held, articles about us, a radio interview and other things we have done to bring attention to our project.
Biotorsdag
24th of November
The Giant Jamboree, Boston, USA
26th of October – 29th of October
Cutting Edge Festival, Science Park
18th of October
Presentation and stand. This event is the largest, most prestigeous innovation event in Norway, where a lot of internationally renown companies and scientists present. We also aided Waag Society from Amsterdam in their &quot;DIY Antibiotics&quot; workshop. <A href="http://www.uio.no/forskning/satsinger/livsvitenskap/aktuelt/aktuelle-saker/2016/teikneserie-om-syntetisk-biologi.html" target="_blank">Comic book</A>
September
We released our own comic book to raise awareness about synthethic biology!
Ungforsk
28th of Septemeber
Three presentations.
Ungforsk
27th of September
Three presentations.
Bjørnelab, Science Library
6th of September
Three presentations for new biologists students.
Bjørnelab, Science Library
30th of August
Three presentation for new biologists students.
Program seminar, Sundvolden
22nd of August
Presentation for new students.
Workshop, Youngstorget
22nd of August<A href="http://www.apollon.uio.no/artikler/2016/3_bakterier_syntetiske.html" target="_blank">Apollon Magazine</A>
Bacteria edition
Article in the magazine.
Apollon
18th of August
Presentation
Presentation for master students
16th of August
Presentation.<A href="https://radio.nrk.no/serie/distriktsprogram-oestlandssendingen/DKOA02014516/01-08-2016#t=2h25m15s" target="_blank">Østlandssendingen, NRK Radio</A>
1st of August
Talked about our project on the radio.<A href="http://www.pd.no/lokale-nyheter/porsgrunn/hoyere-utdanning/kjemper-mot-hele-verden/s/5-40-114346" target="_blank">Porsgrunn Dagblad</A>
28th of July
Article in the local newspapers.<A href="http://www.oblad.no/nyhet/universitetet-i-oslo/resistente-bakterier/oppegard-student-utvikler-viktig-og-billig-bakterietest/s/5-68-191615" target="_blank">Østlandets Blad</A>
25th of July
Article in the local newspapers.<A href="http://rbnett.alda.no/bestillpluss?1&amp;artRefId=13087222&amp;aviskode=ROM&amp;targetUrl=http%253A%252F%252Fwww.rbnett.no%252F%253Fservice%253DpaywallRedirect%2526articleUrl%253Dhttp%253A%252F%252Fwww.rbnett.no%252Fpluss%252Farticle13087222.ece" target="_blank">Romsdals Budstikke</A>
25th of July
Article in the local newspapers.<A href="http://redir.opoint.com/?key=CtApjBjJ2wECazdSqEIq" target="_blank">Oppland Arbeiderblad</A>
July
Article in the local newspapers.<A href="https://uioslonorway.wordpress.com" target="_blank">The Nordic iGEM conference, Stockholm, Sweden</A>
17th of June – 19th of June
Mini Jamboree, workshops and fun.<A href="http://www.uio.no/om/aktuelt/rektors-blogg/2016/28000-potensielle-gr%C3%BCndere-ved-universitetet-i-osl.html" target="_blank">Headmaster's blog</A>
Our headmasters wrote about us and our project on his blog.<A href="https://www.biokjemisk.no/nbs-nytt/se-nbs-nytt-online/" target="_blank">NBS-nytt</A>
You can read an article about our trip to Boston and our project. It will be released after the trip.

GENialt
They made an article about us, which will be released soon.
</P></DIV></DIV></DIV></DIV><DIV class="row subsite"><DIV class="col-md-12"><DIV id="gold"><H2 class="boxheader middle">Gold</H2></DIV><P class="boxnotes leftext">

While we are attempting to fight antibiotic resistance with our novel diagnostic test, we acknowledge the fact that our test only works at a protein level. As the central dogma of molecular biology dictates, protein is the last instance in the information flow in a cell. The problem of resistance genes could of course also reside on a genetic level, upstream of actual enzymes. This is important, as a bacteria may very well have resistance genes without expressing them at a certain time. The expression could even be turned on by the presence of antibiotics, which makes it important to not distribute antibiotics when such genes are present. All of these things points to a need for a diagnostic tool that not only detects enzymatic activity, but also the genes leading to said enzymatic activity. Such a tool would enable us to test for resistant microbes before they actually express their resistance. Further, earlier detection would lower the probability of spreading of the resistant bacteria, potentially protecting a lot of people from painful, dangerous infections. </P><P class="boxnotes leftext">
This reasoning led us to CRISPR/Cas9. Cas9 is useful in this matter because of its ability to recognize specific nucleotide sequences. By now, it is known that by redesigning gRNA/tracrRNA/crRNA, one can design Cas9-systems that can recognize any sequence. Usually, this is used in combination with the native endonuclease activity of Cas9, enabling the researcher to cut DNA at specific locations and manipulate the target sequence. This differs from our approach. By taking a slightly unorthodox, but innovative approach to CRISPR/Cas9 usage, we want to interface it with our PhoneLab. </P><P class="boxnotes leftext">
In order to combine CRISPR/Cas9 technology with our diagnostic tool, we imagine to preserve the “searching” capability of CRISPR/Cas9 systems, and replacing the “cutting” part with a detectable readout similar to what we have with the nitrocefin-based detection. This means that instead of cleaving the resistance genes, we want to couple the pairing of nucleotide targets and Cas9 to a colorimetric signal. By doing so, detection of genes could relatively easily be interfaced with the detection of ESBL activity in our PhoneLab, thus creating a more complete diagnostic tool for ESBL infections. </P><P class="boxnotes leftext">
This could be achieved in several ways, but as a contribution to our project, we propose the following design of a CRISPR/Cas9 system to be used in PhoneLab. </P><P class="boxnotes leftext">
First, we would need to design tracr/crRNA sequences compatible with known ESBL genes. These RNAs would be linked to a null-nuclease Cas9 (dCas9) - a Cas9 enzyme modified so that it binds tracr/crRNA and is guided to its targets,, but does not cleave the target genes. For each target gene, we propose to have two sets of tracr/crRNA, complementary to different, but adjacent parts of the same gene. Each of these RNA molecules would have a dCas9 partner, which again would be fused to half of a detectable protein partner in a split-enzyme assay. When the tracr/crRNA finds its goal, dCas9 would bring the two parts of the split-enzyme together, thus creating a detectable signal. So, in this system, two independent RNA/DNA-binding events would be required to take place, which lowers the probability of false positives. </P><P class="boxnotes leftext">
The detectable protein partner of dCas9 is currently a matter of research for us, but one possibility is to use the subunits of galactosidase. LacZ alpha and -omega are two separate peptides that form the active form of galactosidase when in contact with each other. By fusing these two peptides to two separate dCas9 constructs recognizing adjacent parts of the same gene, the presence of said gene would be coupled to the forming of a functioning galactosidase. Thus, by adding for example liquid compounds similar to X-gal, a visible color change could be achieved, similar to the one seen in our project with nitrocefin.</P><P class="boxnotes leftext">
By designing different tracr/crRNA sequences, this design could potentially be altered to recognize any gene present in high enough numbers in a liquid sample. </P><P class="boxnotes leftext imgcap"><I>Figure 1: Conceptual sketch of the proposed dCas9-system</I></P><P class="boxnotes leftext">
To investigate how this would work with different classes of ESBL, we aligned the sequences found in clinical isolates of ESBL infections in Norway (information provided by Ørjan Samuelsen in Tromsø). 
</P><P class="boxnotes leftext imgcap"><I>Figure 2: Generated using the TCOFFEE algorithm in Jalview. Sequences provided 
by Ørjan Samuelsen at the National Expertise Center of Antibiotic Resistance in Tromsø.</I></P><P class="boxnotes leftext">
Although it includes relatively few sequences, this phylogeny suggests that the classes A, B C and D of ESBL are so different on a genetical level that the design we propose could probably be used to pin-point the class of ESBLs present in a sample, simply by designing crRNA/tracrRNA compatible to sequences unique to each class. </P><P class="boxnotes leftext">
In summary: to expand our project, we want to create a potentially universal gene detection tool by utilizing parts from the iGEM registry, as there are several dCas9-constructs and LacZ peptides there. The set-up would include both bioinformatics, when designing the right RNA sequences, regular cloning, to fuse the LacZ peptides to dCas9 and protein expression to form the final constructs. If realized, this would enable us to efficiently test for both ESBL proteins and genes within minutes using our PhoneLab. By adding more slots for cuvettes in our hardware, this test could have been integrated into the set-up and enabled us to test for both ESBL proteins and genes at the same time.  Thus, we imagine to use CRISPR/Cas9 methods in a relatively new and creative way, and have a concrete plan as to how to improve and expand our project.</P><P class="boxnotes leftext">
This outline is a result of conferring with supervisors Dirk Linke, Eric de Muinck and Paul Grini along with correspondence with Kevin M Esvelt from the Wyss Institute for Biologically Inspired Engineering, Harvard University, Cambridge, Massachusetts, USA (corresponding author of the article “Cas9 as a versatile tool for engineering biology”, PubMed ID: 24076990). </P><P class="boxnotes leftext"><B>Positive controls: </B>
In our hardware, additional room was made for cuvettes containing purified protein controls. The software takes advantage of this by addressing that cuvette, labeling it as “positive” (red), and compares the other cuvettes, containing urine samples and the nitrocefin and inhibitor compounds, to that. By adjusting the protein concentration, the amplitude of the signal created by the purified protein can mimic that created by ESBL-producing bacteria. As a negative control, an additional slot is set out for urine, nitrocefin and inhibitors without any resistant bacteria or protein. In this way, we have created a harmless, but effective way of telling PhoneLab what a positive result looks like, and integrated it into our design. </P><P class="boxnotes leftext"><B>Future aspects: </B>
If we were to further advance our project, the imminent steps to be taken would be to realize our CRISPR/Cas9 system, optimize the set-up with clinically isolated ESBL-samples and purifying class B and C ESBL-proteins. </P></DIV></DIV><DIV class="row subsite"><DIV class="col-md-12"><DIV id="silver"><H2 class="boxheader middle">Silver</H2></DIV><P class="boxnotes leftext">
Our project has had several issues that we have had to consider, some of which were:
- Patient confidentiality and availability of our product to the general public
- Optimization of our assay to achieve a clinically relevant detection range
- Proving that our set-up safely tests for ESBL-producing bacteria
- Creating and testing positive controls for PhoneLab that do not involve actual dangerous pathogens 
- Addressing antibiotic resistance potential on a genetic level in addition to phenotypic resistance </P><P class="boxnotes leftext">
All of the above problems have been points of thought and investigation to us. Even though there was not enough time to come up with working solutions for all of them, several of them have been addressed, some of them even borderline solved and integrated into our final design. For the investigation part of said issues, see below. For integrative design of solved problems, see “Gold”. </P><P class="boxnotes leftext"><B>Patient confidentiality:</B>
Along with the birth of our idea, to create a product that would have access to peoples personal devices, and inform them as to whether resistant bacteria are present in them or not, came the realization that the eventual generation and storage of data would have to operate on a high level of security. To investigate this, we brainstormed around the issue, and reached out to Gissle Hannemyr, an expert in software security at the University of Oslo. This is something that in general has been of great importance to us; to utilize the potential and the potent expertise that resides in the institution that we are lucky enough to be a part of.</P><P class="boxnotes leftext">
However, he did not respond immediately, and this ended up being something that we kept in mind, and would have moved forward with if we were to continue with the project. </P><P class="boxnotes leftext"><B>Detection limits:</B> 
The first thing we did in respect to clinically relevant detection limits, was to contact experts to find out the bacterial number one can expect in urinary tract infections. The numbers that returned (averaging around 10^5/10^6 bacteria/mL) did not correlate with the detection limits we saw in our initial experiments with nitrocefin. </P><P class="boxnotes leftext">
Thus, we proceeded to better our set-up. To do so, we investigated different factors that could improve the detection limit. In the end, by using a high concentration of nitrocefin and lysing the bacteria, we achieved a detection limit of clinical significance. </P><P class="boxnotes leftext"><B>Testing our set-up with ESBL-producing bacteria: </B>
This was important to us; as a proof of concept, we formed our set-up using “regular” AmpR bacteria. But, eventually, we wanted to move on to the big bads; clinical isolates of ESBL-producing bacteria. To do this, several things had to be done. First of all, we had to get our hands on both the bacteria themselves and information about them. This was achieved largely through corresponding with Ørjan Samuelsen at the Norwegian Center of Competence on Antibiotic Resistance in Tromsø. Second, ESBL-producing bacteria classify as bio-safety level 2, meaning that we had to thoroughly think through and plan whatever we wanted to do with them. This was done by conferring with the experienced people working in our lab, walking them through our set-up and having them supervise us when we were handling the bacteria. </P><P class="boxnotes leftext">
We did in the end attempt a run of our set-up with clinical isolates. This was not entirely successful, but we learned a lot from the experience, and we feel certain that we would have achieved clear results from this if we had time to do it again. For more information about the strains that were used, the set-up and the results, see under the “Lab”-section. </P><P class="boxnotes leftext"><B>Positive controls:</B>
The first positive control that came to mind when designing our project, was to use bacteria with known resistance. Naturally, this was not something we desired to do - we do not want our test to rely on distribution of resistant bacteria. To work around this, we decided to purify a representative protein, a β-lactamase. A purified enzyme could, in theory, serve as a very sensitive positive control, and would be safe to carry around. </P><P class="boxnotes leftext">
This was achieved; we successfully purified a class A β-lactamase using a biobrick from the Calgary team of 2013 (see the “biobrick” section). We also moved on to prove that even very low concentrations of the protein produced a reliable signal with nitrocefin, that it was inhibited by clavulenic acid (CVA, the inhibitor for class A enzymes in the Penta Well test), and integrated this into our hardware and software. </P><P class="boxnotes leftext">
We also want to stress the issue that we created the chassis, in form of biobricks, to also express class B and C ESBLs, but did not do this due to bio-safety concerns. Expressing those proteins would mean creating dangerous, resistant bacteria in our lab, which we did not want to do unnecessarily. Therefore, we settled on expressing and testing for class A, and could easily also have done so for the other classes.</P><P class="boxnotes leftext lastp"><B>Addressing resistance on a genetical level: </B>
Upon the realization that the Penta Well test deals only with the enzymatic activity of antibiotic resistance, we decided to attempt detecting also the genes causing said enzymatic activity. This led to the CRISPR/Cas9 design described further under “Gold”. 

</P></DIV></DIV><DIV class="row footer"><DIV class="col-md-4 footerelement"><H2 class="boxheader footerheader">Contact</H2><P class="boxnotes footercontact"> Blindernveien 31, 0371 Oslo</P><P class="boxnotes footercontact"> uioslonorway@gmail.com</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>