"
Page
Discussion
View source
History
teams
Log in
 
Team:UCSF UCB/sabrina
From 2014.igem.org
Sabrina's Lab Notebook
06/09/14
PCR for Constitutive Promoters - pTEF1 (M3)
Materials
1x reaction
4.5x Master Mix
5x Phusion HF Buffer
10 µl
45 µl
dNTP's (10 mM)
1 µl
4.5 µl
Forward Primer (10µm)
2.5 µl
11.25 µl
Reverse Primer (10µm)
2.5 µl
11.25 µl
*Template DNA
0.3 µl
1.35 µl
Phusion Polymerase
0.5 µl
2.25 µl
Water
33.2 µl
149.4 µl
Total
50 µl
225 µl
(Note: Add enzymes last)
PCR Reaction Procedures:
Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom.
Pipette 50ul from the Master Mix into four labled PCR tubes
Put in the thermocycler for the following cycles:
Initial Denaturation   | 98° C | 30s    35 Cycles of:        Denaturation | 98° C | 10s       Annealing    | 55° C | 20s       Extension    | 72° C | 30s Final Extension    | 72° C | 5m Hold               | 4°  C | Forever
Keep samples for gel extraction on the following day.
06/10/14
Loading the Gel
*Add 5ul of Loading Dye (10X Blue Juice) to samples.
Gel Map:
1  2  3  4  5  6  7  8  9  10  11  12  13  14  15    [----M6---][----M7----] [-----M3-----]  1  2  3  4  5  6  7  8  9  10  11  12  13  14  15    [----TEF1----]   [-M10]
Both lane #1 are 100 bp ladder
*George had less to PCR due to a spill accident when spinning tubes
Gel Extraction Procedure:
Cut Gel
Weigh it in a colorless tube
Add 3 volumes Buffer QG to 1 volime Gel (100mg ~ 100µl)
Incubate @ 50° C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)
Add 1 gel volume isopropanal to the sample and mix
Place a QlAquick soin column in a provided 2ml collection tube
Place sample in column & spin for 1 min -> discard flow through
To wash add 0.75 ml Buffer PE to column & centrifue for 1 min, then dry spin
Place column in 1.5 ml tube
Add 30µl H2O & centrifuge for 1 min.
Restriction Enzyme Digest with ApaI (Backbone Digestion):
40ul of Gel Extraction DNA
5ul of Cutsmart Buffer
0.5ul of ApaI
*Digest at room temperature overnight
06/11/14
PCR Purification and Ligation
Digest with XhoI
Add 0.5ul of XhoI Enzyme (in Enzyme Box) to ApaI Digest from 06/10/14
Incubate in the 37°C shaker for ~2 hours.
PCR Purification
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
To bind DNA, apply sample to column and spin for 1 min
Discard flow through
To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Ligation (pSV606 and pTEF1-M3)
1:3 Backbone to Insert
Concentration of M3 Digest = 127.1 ng/ul
Materials:
Reagents
1x
10x ligase buffer
1.0µl
DNA Backbone (PSV606)
0.2µl
Insert
0.2µl
T4 DNA Ligase
0.5µl
H2O
8.1µl
Total
10 µl
Keep at room temperature for at least 2 hours.
Transformation
Materials
1x
Ligation (pSV606 and M3)
10µl
E. Coli Competent Cells
50ul
Mix both together and keep on ice for 30 minutes
42°C Heat shock for 45 seconds
Put immediately on ice for at least 2 minutes
Add 250 ul of SOC media, shake for 1 hr at 37°C.
Plate on selective media and store at 37°C.
06/12/14
Re-Ligation and Tranformation of M3
Note: Most of our plates did not form colonies, with the exceptions of pTEF1-M10 and pTEF1-M3.
Due to this, we will all have to re-ligate using higher concentration of the pSV606 backbone.
Kara belives that the failure in our transformation was due to mediocre competent cells. We will be using better competent cells.
Forgot to make a Negative control. Make sure to make one next time.
Ligation #2:
Materials:
Reagents
1x
10x ligase buffer
1.0µl
DNA Backbone (PSV606)
0.4µl
Insert
0.2µl
T4 DNA Ligase
0.5µl
H2O
7.9µl
Total
10 µl
-Added 0.2ul more Backbone.
Keep at room temperature for at least 2 hours.
Transformation
Materials
1x
Ligation (pSV606 and M3)
10µl
E. Coli Competent Cells
50ul
Mix both together and keep on ice for 30 minutes
42°C Heat shock for 45 seconds
Put immediately on ice for at least 2 minutes
Add 250 ul of SOC media, shake for 1 hr at 37°C.
Plate on selective media and store at 37°C.
06/13/14
Yeast Transformation (CB008 + Inducible Promoters)
Assisted Ianto and others in Transforming Inducible Promoters into Yeast.
| Reagents                  | | ------------------------- | | YPD                       | | 1 M LiOAc                 | | 10X TE pH 7.5             | | 1X TE pH 7.5, 0.1 M LiOAc | | 50% PEG 3350              | | DMSO                      | | Salmon Sperm DNA (ssDNA)  |
PEG is viscous, so pipette slowly to prevent air bubbles from forming.
boil ssDNA aliquots for 10 min, then ice down for 10 min before use.
Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C
Set up digest to linearize DNA
Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C
Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)
Harvest cells in centrifuge - 3000 rpm, 2-5 min
Wash with 1 ml 0.1 M LiOAin TE
Pellet cells - 3000 rpm , 2-5 min
Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation
to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA
Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc) Optional: Add 75 µl DMSO definitely did
Vortex
Incubate 42° C for 30m & begin drying plates
Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)
Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD
Plate on selective media
Incubate 1-3 days
06/16/14
Colony PCR for Screening E.Coli Procedure:
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 25ul of water.Do this for 5 colonies. Use 5ul for PCR reaction below.
Set up PCR Reaction below:
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
6 µl
10 µM Reverse primer
1 µl
6 µl
Water
3 µl
18 µl
Bacterial cells (template)
5 µl
-----
Total
90 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation   | 95° C | 5 min    35 Cycles of:        Denaturation | 95° C | 45s       Annealing    | 55° C | 30s       Extension    | 72° C | 1min per kb Final Extension    | 72° C | 10m Hold               |  4°C  | Forever
Load PCR products on a gel. (5ul) (GoTaq already has loading dye)
For all positive bands on the gel, add rest of bacterial cells from PCR tubes to 5ml LB and grow overnight at 37°C for miniprep on the following day.
Result of E.Coli Colony PCR (Constitutive Promoters): All lanes worked.
Grew 2 out of 5 Successful Colony PCR E.Coli Colonies in liquid culture.
Unfortunately, the rack which contained our cultures was not secured and caused our cultures to fall down and possibly be contaminated.
We re-inocculated the colonies due to possible contamination.
Colony PCR for Yeast (Inducible Promoters CB008 and CB008DB) Procedure:
Boil Yeast at 95°C for 1 hour.
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 15ul of NaOH. Take 3 colonies from each plate. Use 5ul for PCR reaction below.
Set up PCR reaction below:
Reagents
1X
67X
2X GoTaq Green PCR Master Mix
10 µl
670 µl
10 µM Forward primer
1 µl
67 µl
10 µM Reverse primer
1 µl
67 µl
Water
3 µl
201 µl
cells (template)
5 µl
-------
Total
1005 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation    | 95°C | 5 min    35 Cycles of:        Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C | Forever
Load PCR products on a gel. (5ul) (GoTaq already has loading dye)
Results: only 4 positive lanes
Possible Explanation: Not enough DNA in PCR tubes.
06/17/14
Miniprep of Constitutive Promoters
Miniprep Precedure:
Obtain E.Coli Cultures from 06/16/14 in 37°C  Incubation Room.
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Sent miniprep into sequencing. Results came out positive. Constitutive promoters successfully transformed into plasmids.
06/18/14
Cultures Set up for Yeast Tranformation
Made cultures of CB008 and CB008DB for each of our constitutive promoters.
06/20/14
Linearization of Plasmids with Constitutive Promoters - PSV606
Linearization Materials:
10 µl DNA (miniprep in my box in -20°C Fridge) 5 µl Cutsmart Buffer 1 µl PME1 34 µl H2O
Incubate at 37 ° C for 2 hours
Transformation of Constitutive Promoters into Yeast (CB008 and CB008DB)
| Reagents                  | | ------------------------- | | YPD                       | | 1 M LiOAc                 | | 10X TE pH 7.5             | | 1X TE pH 7.5, 0.1 M LiOAc | | 50% PEG 3350              | | DMSO                      | | Salmon Sperm DNA (ssDNA)  |
PEG is viscous, so pipette slowly to prevent air bubbles from forming.
boil ssDNA aliquots for 10 min, then ice down for 10 min before use.
Make sure to add reagents in order!
Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C
Set up digest to linearize DNA (see procedure above)
Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C
Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)
Harvest cells in centrifuge - 3000 rpm, 2-5 min
Wash with 1 ml 0.1 M LiOAin TE
Pellet cells - 3000 rpm , 2-5 min
Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation
to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA
Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc) Optional: Add 75 µl DMSO definitely did
Vortex
Incubate 42° C for 30m & begin drying plates
Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)
Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD
Plate on selective media
Incubate 1-3 days
06/23/14
Yeast Colony PCR of Constitutive Promoters
Boil Yeast at 95°C for 1 hour.
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 15ul of NaOH. Take 3 colonies from each plate. Use 5ul for PCR reaction below.
Set up PCR reaction below:
Reagents
1X
67X
2X GoTaq Green PCR Master Mix
10 µl
670 µl
10 µM Forward primer
1 µl
67 µl
10 µM Reverse primer
1 µl
67 µl
Water
3 µl
201 µl
cells (template)
5 µl
-------
Total
1005 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation    | 95°C | 5 min    35 Cycles of:        Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C | Forever
Load 5ul of PCR products on a gel.  (GoTaq already has loading dye)
Results: All Constitutive Promoters were successfully Transformed.
Made cultures to be diluted for glycerol stocks. Cultures made of the yeast transformations of the constitutive promoters.
06/24/14
Flow Cytometry Protocol:
Day Before:
Start overnight cultures of your strains to be tested in 5-10ml SD Complete media. (Do not use YPD. YPD has fluorescence background which would cause problems for FACS testing.)
Day Of:
Flow Procedures:
Dilute overnight cultures 1:100 in the 96 well shaker plate. (Shaker plates can be found in the Lim Lab in the back glass cabinet where large glass bottles can also be found.)
Allow cells to enter growth stage by putting on plate shaker at 30°C going 1000rpm (found in Flow room or the 30°C room in the other hall)for 2-3 hours after dilution.
Induce with alpha factor. The stock alpha factor in the freezer is 3mM and we use it at the final concentrations of: 0, 1nM, 10nM, 100nM, and 1uM.
Allow introduction to proceed on plate shaker for 90 min to 2 hrs, no longer.
Transfer 250ul of each culture to the 96 well flow cytometry v-bottom plate. (found in flow room on shelf) Arrest cells with 10ul of cycloheximide. (stops translations of proteins; stops growth)
Run on flow cytometer.
Analyze Data.
Things to check for Flow:
Check that machine is on standby and not run. (Should be on standby when not in use)
Make sure that a clean plate has been ran by the person before you. Otherwise, run a clean plate. (A1-A4 bleach, B1-B4 water)
Check fluid buffer box and storage tanks. Discard waste fluid in sink 2 doors down (remember to add bleach to tank when done) and replace buffer box if needed.
Make sure you have a positive and negative control so that you can set the parameters.
Open up FACSDiva. Sign into iGEM account.
Create new FACS experiment. Highlight needed wells and use the blue button to create wells for use. Create specimens among the wells and rename them.
Open up inspector to check parameters.
Run well. (Do not click 'Run Plate'.)
Once finished, export data onto a USB.
Run a clean plate.
* Note: NEVER TURN THE WIFI ON! THE COMPUTER WILL CRASH!
Voltage Parameters:
FSC:  250  SSC:  280  FITC: 550
Sample Loading Parameters:
Sample Volume: 200 Mixing Volume: 150 Flow Speed: 0.5ul/sec when adjusting, otherwise 1ul/sec Mixing quantity: 3
Using Flowjo and Matlab to Analyze Data
Flowjo:
Export Data from USB and transfer to iGEM2014 folder.
Open Flowjo and drag data over to box.
Set an appropriate gate.
Go to data, select all data and press 'Σ'. Select median, mean, and count.
Press the refresh button. Additional numerical data should be present.
Matlab:
Open up a new script. and comment the title of the script. (Use '%' for comments.)
Enter the alpha factor info that will serve as the x-axis. (Put in log form.)
Enter the yGEM data. This will serve as the y-axis.
Type your figure info and press the "go".
Example of Matlab script to create graphs based on Flowjo Data:
%14-6-19 Alpha-Factor Promoter ASG7
clear % this will get rid of all previous graphs
Clear all % same as above but for ALL graphs
alpha = [0 1 10 100 1000] % nanomolar nM, spaces show a new number
alpha = [0.01 1 10 100 1000] % 0.01 is really 0. Flowjo cannot comput 0s
yGEM = [558 715 1040 2230 4731] %mean GFP C1-C5 ASG7
figure(1)
semilogx(alpha,yGEM4,'--c'), %('--c') means dotted line, cyan colored line, and star for points.
hold on
xlabel' alpha ') % [] denotes concentration, () denotes "in"
ylabel('mean GFP (au)') % au means arbitrary unit
title('Alpha-factor vs mean GDP yGEM4')
legend('yGEM4')
axis([0.01 1500 200 4000]); % axis([xmin xmax ymin ymax])
Use Adobe Illustrator to beautify graphs. Newer Matlab versions may have the option to beautify graph without the use of Illustrator.
Code available in dropbox.
06/24/14
Plate Map for Flow Cytometry of Inducible Promoters:
PCR of rtTA Promoter
Became part of Jessica's rtTA portion of the project.
Inducible Promoters + rtTA into CB008 and CB008DB
PCR Purification of remaining pHY4 + inducible promoters + GFP (Digestion done by Derrick) (Backbone)
PCR of inducible promoter primer + rtTA with new rtTA RV primer
Materials
1x reaction
12x Master Mix
5x Phusion HF Buffer
10 µl
120 µl
dNTP's (10 mM)
1 µl
12  µl
Forward Primer (10µm)
2.5 µl
30  µl
Reverse Primer (10µm)
2.5 µl
30  µl
Template DNA: rtTA
0.5 µl
6   µl
Phusion Polymerase
0.5 µl
6   µl
Water
33 µl
396 µl
Total
50 µl
600 µl
(Note: Add enzymes last)
PCR Reaction Procedures:
Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom.
Pipette 50ul from the Master Mix into four labled PCR tubes
Put in the thermocycler for the following cycles:
Initial Denaturation  | 95°C | 5 min    30 Cycles of:        Denaturation | 95°C | 30s       Annealing    | 50°C | 20s       Extension    | 72°C | 2 min Final Extension    | 72°C | 5 min Hold               | 4°C  | Forever
Keep samples for gel extraction on the following day.
Promoter
Concentration
PRM2
0.451
YDR124W
15.46
SAG1
12.10
PRM6
21.74
PRM1
4.186
PCL2
6.348
E. Coli Transformation for Increase of Plasmids:
Plasmids: HY3E (rtTA), HY130E (TRP), HY12E1 (URA3), HY34E2 (HIS2), HY111E2 (LEU2), HY4 (stock plasmid)
06/25/14
Glycerol Stocks of Constitutive Promoters in Yeast (CB008 and CB008 DB)
Diluted overnight cultures of constitutive promoters in yeast (CB008 and CB008DB)
added 250ul culture to 5ml YPD
put in 30°C incubator for 2 hours
Glycerol Stocks Procedure:
Obtain Glycerol Stock tubes from drawer near autoclave box. (Tubes have orange caps)
Add 350ul of 60% Glycerol or 420ul of 50% Glycerol to 350ul of cells.
Vortex well.
Label tubes and put into -70°C Freezer. (3rd row, 1st drawer)
Graphs of Flow for yGEM4:
Gel Map of pTEF1 Strains Yeast Homology:
Graph of Flow for All Inducible Promoters:
|  yGEM#  | Promoter|                        |---------|---------| |yGEM4    |   ASG7  | |yGEM5    |   PCL2  | |yGEM6    |   SAG1  | |yGEM10   |   PRM2  | |yGEM11   |   CLG1  | |yGEM12   | YDR124W | |yGEM13   |   PRM6  | |yGEM14   |   PRM2  | |yGEM15   |   ECM18 | |yGEM16   |   PRM3  |  |  yGEM#  |          Strain            | |---------|----------------------------| |yGEM23   |CB008 pTEF1-GFP-URA3        | |yGEM24   |CB008 pTEF1(M3)-GFP-URA3    | |yGEM25   |CB008 pTEF1(M6)-GFP-URA3    | |yGEM26   |CB008 pTEF1(M7)-GFP-URA3    | |yGEM27   |CB008 pTEF1(M10)-GFP-URA3   | |yGEM28   |CB008DB pTEF1-GFP-URA3      | |yGEM29   |CB008DB pTEF1(M3)-GFP-URA3  |  |yGEM30   |CB008DB pTEF1(M6)-GFP-URA3  | |yGEM31   |CB008DB pTEF1(M7)-GFP-URA3  | |yGEM32   |CB008DB pTEF1(M10)-GFP-URA3 |  | Promoter|  Owner  |                    |---------|---------| |PRM2     | George  | |ASG7     | Sabrina | |PCL2     | Eric    | |CLG1     | Derrick | |YDR124W  | Ianto   | |PRM6     | Ianto   | |PRM1     | Robert  | |ECM18    | Robert  | |PRM3     | Eleanor | |SAG1     | Jessica |
Diluted Overnight Cultures - Constitutive Promoters
Added 250ul of culture to 5ml YPD - put in incubator 30°C. Finished ~ 9:30am
06/26/14
Miniprep of Plasmids for rtTA
Miniprep Precedure:
Obtain E.Coli Cultures from 06/25/14 in 37°C  Incubation Room.
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Nanodrop Concentrations:
PRM2 = 107.0 ng/ul
ASG7 = 111.2 ng/ul
PCL2 = 96.49 ng/ul
CLG1 = 114.8 ng/ul
YDR124W = 90.53 ng/ul
HYM1 = 77.29 ng/ul
PRM6 = 99.83 ng/ul
PRM1 = 75.35 ng/ul
ECM18 = 18.81 ng/ul (to be remade)
PRM3 = 130.8 ng/ul
SAG1 = 104.1 ng/ul
pTEF1 = 182.3 ng/ul
PCR of rtTA Promoter
PCR done on 6/24/14 failed.
Added DMSO this time.
Became part of Jessica's rtTA portion of the project.
Inducible Promoters + rtTA into CB008 and CB008DB
PCR Purification of remaining pHY4 + inducible promoters + GFP (Digestion done by Derrick) (Backbone)
PCR of inducible promoter primer + rtTA with new rtTA RV primer
Materials
1x reaction
12x Master Mix
5x Phusion HF Buffer
10 µl
120 µl
dNTP's (10 mM)
1 µl
12  µl
Forward Primer (10µm)
2.5 µl
30  µl
Reverse Primer (10µm)
2.5 µl
30  µl
Template DNA: rtTA
0.5 µl
6   µl
Phusion Polymerase
0.5 µl
6   µl
Water
31.5 µl
396 µl
DMSO
1.5ul
30  ul
---------------------
-------------
----------------
Total
50 µl
600 µl
(Note: Add enzymes last)
PCR Reaction Procedures:
Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom.
Pipette 50ul from the Master Mix into four labled PCR tubes
Put in the thermocycler for the following cycles:
Initial Denaturation  | 95°C | 5 min    30 Cycles of:        Denaturation | 95°C | 30s       Annealing    | 50°C | 20s       Extension    | 72°C | 2 min Final Extension    | 72°C | 5 min Hold               | 4°C  | Forever
Keep samples for gel extraction on the following day.
06/27/14
rtTA PCR Purification
PCR Purification Prodedure:
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
To bind DNA, apply sample to column and spin for 1 min
Discard flow through
To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Streak Plate of Constitutive Promoters
As instructed by Kara, I made a streak plate with all 5 Constitutive Promoters on it.
Accidentally made the plate by stamping it instead of actually streaking it. It looks kinda funny.
Along with the streak plate, I also made cultures of of CB008 SAG1 and CB008 ECM18, along with two negative cultures of CB008. Cultures of the Contitutive Promoters were made also.
All cultures made with SD-Complete. (for Flow)
Gibson Assembly and Transformation
Gibson Assembly Procedures:
|        Materials       |  Amount  |    |------------------------|----------| |Gibson Master Mix (2x)  |   5ul    | |Backbone (pHY4+AFRP+GFP)|   3ul    | |Insert (rtTA + primer)  | Varies   | |ddH2O                   | Varies   |      Promoter|Concentration of Insert| Amount of Insert Needed  |H2O Needed|      --------| --------------------- | ------------------------ | -------- |     PRM2    | 107.0 ng/ul           | 1 ul                     |    1ul   |     ASG7    | 111.2 ng/ul           | 1 µl                     |    1ul   |     PCL2    | 96.49 ng/ul           | 1 µl                     |    1ul   |     CLG1    | 114.8 ng/ul           | 1 µl                     |    1ul   |     YDR124W | 90.53 ng/ul           | 1 µl                     |    1ul   |     HYM1    | 72.29 ng/ul           | 1 µl                     |    1ul   |     PRM6    | 99.83 ng/ul           | 1 µl                     |    1ul   |     PRM1    | 75.35 ng/ul           | 1 µl                     |    1ul   |     ECM18   | 18.87 ng/ul           | 3 µl                     |    0ul   |     PRM3    | 130.8 ng/ul           | 1 µl                     |    1ul   |     SAG1    | 104/1 ng/ul           | 1 µl                     |    1ul   |     Pos Ctrl| Gibson Positive Ctrl  | 2 µl Backbone, 2ul Insert|    1ul   |     Neg Ctrl| ASG7 Backbone         | 3 ul Backbone, 0ul Insert| 2ul   |
PRM2 did not have a backbone and thus could not be transformed yet.
Obtain Mach1 Cells from -70°C Freezer
25ul for 10ul Gibson Mix.
Mix Materials.
Incubate at 50°C for 1 hour.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 1 hr. (Warm plates also)
Plate on LB-CARB and keep in drawer over the weekend.
06/30/14
Results of Transformation:
| Promoter | Colonies|                   |----------|---------| | PRM2     |   N/A   | | ASG7     |    7    | | PCL2     |    6    | | CLG1     |   15    | | YDR124W  |    2    | | HYMI     |    0    | | PRM6     |    9    | | PRM1     |    8    | | ECM18    |    8    | | PRM3     |    5    | | SAG1     |    7    | | Pos Ctrl |  TMTC   | | Neg Ctrl |   10    |
Colony PCR of rtTA Transformation:
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 25ul of water. Do this for 3-5 colonies. Use 5ul for PCR reaction below.
Set up PCR Reaction below:
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
6 µl
10 µM Reverse primer
1 µl
6 µl
Water
3 µl
18 µl
Bacterial cells (template)
5 µl
-----
Total
90 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation   | 95°C | 5 min    30 Cycles of:        Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C  | Forever
Load PCR products on a gel. (5ul) (GoTaq already has loading dye)
For all positive bands on the gel, add rest of bacterial cells from PCR tubes to 5ml LB and grow overnight at 37°C for miniprep on the following day.
Gel of rtTA Tranformation Colony PCR:
Original Gel failed due to the bottom part of the gel disappearing. (possibly due to the bottom part not solidifying enough)
Jessica made PRM2 Backbone
ECM18 PCR'ed
Both were Gibson'ed and transformed.
Cultures were made of pHY4 AFRP's + rtTA.
5ml YPD + Transformed Colonies
Put in 37°C Incubator Room Shaker overnight.
07/01/14
Miniprep of Inducible Promoters + rtTA
Note: Try not to do more than one miniprep per promoter.
Miniprep Precedure: (did in triplicate for some reason)
Obtain E.Coli Cultures from 06/30/14 in 37°C  Incubation Room.
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Nanodrop Concentrations (ng/ul):
ASG7 1 = 203.0 ng/ul
ASG7 2 = 282.9 ng/ul
ASG7 3 = 244.9 ng/ul
PCL2 1 = 205.5 ng/ul
PCL2 2 = 205.3 ng/ul
PCL2 3 = 305.3 ng/ul
CLG1 1 = 155.5 ng/ul
CLG1 2 = 187.6 ng/ul
CLG1 3 = 230.0 ng/ul
YDR124W 1 = 89.44 ng/ul
PRM6 1 = 246.4ng/ul
PRM6 2 = 17.32 ng/ul
PRM6 3 = 154.0 ng/ul
PRM3 1 = 244.9 ng/ul
PRM3 3 = 180.1 ng/ul
SAG1 1 = 146.5 ng/ul
SAG1 2 = 243.8 ng/ul
SAG1 3 = 137.8 ng/ul
*Did not use PRM 1 #1, 2, and 3 or PRM3 #2 because the cultures did not grow.
Sent into sequencing.
Colony PCR for Screening Yeast - Constitutive Promoters + GFP
Used Streak plate in 4°C Fridge.
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
-----
10 µM Reverse primer
1 µl
6 µl
Water
5 µl
30 µl
Bacterial cells (template)
3 µl
-----
Total
96 ul
add 15ul of master mix to each tube.
Initial Denaturation   | 95°C | 5 min    30 Cycles of:        Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C  | Forever
Load 5ul of PCR products on a gel. (GoTaq already has loading dye)
10uM RV Primer = GFP RV Primer - kept in Primer Working Stock (2014 Primers)
10uM FW Primer = Individual pTEF1/mutant Primers kept in the -20°C freezer boxes of the respective owners. (pTEF1=Jeffrey, M3=Sabrina, M6=Eleanor, M7=Robert, M10=George)
M7 used M10 primer due to the primers being so similar. M7 ran out of its own primer.
Gel of Yeast Colony PCR (Constitutive Promoters)
Gel mostly failed - to be redone the following day.
Sent into sequencing.
Sequencing for pTEF promoters cancelled due to the absence of primers that were supposed to be sent in or selected on the promoter catalog.
07/02/14
Re-Colony PCR for Yeast - Constitutive Promoters + GFP
Colony PCR for Screening Yeast - Constitutive Promoters + GFP
Used Streak plate in 4°C Fridge.
picked 3 different colonies.
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
-----
10 µM Reverse primer
1 µl
6 µl
Water
5 µl
30 µl
Bacterial cells (template)
3 µl
-----
Total
96 ul
add 15ul of master mix to each tube.
Initial Denaturation   | 95°C | 5 min    30 Cycles of:       Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C  | Forever
Load 5ul of PCR product on a gel. (GoTaq already has loading dye)
10uM RV Primer = GFP RV Primer - kept in Primer Working Stock (2014 Primers)
10uM FW Primer = Individual pTEF1/mutant Primers kept in the -20°C freezer boxes of the respective owners. (pTEF1=Jeffrey, M3=Sabrina, M6=Eleanor, M7=Robert, M10=George)
M7 used M10 primer due to the primers being so similar. M7 ran out of its own primer.
Gel of Yeast Colony PCR (Const. + GFP):
Yeast PCR Purification
To extract Yeast DNA for Sequencing
PCR Purification Prodedure:
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
To bind DNA, apply sample to column and spin for 1 min
Discard flow through
To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Nanodrop Concentrations:
pTEF1       = 12.30 ng/ul
pTEF1 (M3)  = 6.242 ng/ul
pTEF1 (M6)  = 33.68 ng/ul
pTEF1 (M7)  = 39.72 ng/ul
pTEF1 (M10) = 19.43 ng/ul
Did not send into sequencing due to the conclusion that the constitutive promoters were successfully transformed based on past sequencing and flow cytometry.
I restreaked the streak plate from the patch plate with the colonies marked (✓) on them. I did not trust the other streak plate. Incubated in 30°C incubator.
Re-Transformation of pHY4 Inducuble Promoters + rtTA
Gibson Assembly Procedures:
|        Materials       |  Amount  |    |------------------------|----------| |Gibson Master Mix (2x)  |   5ul    | |Backbone (pHY4+AFRP+GFP)|   4ul    | |Insert (rtTA + primer)  |   1ul    | |ddH2O                   |   0ul    | |Total                   |   10ul   |
Obtain NEB5α (C2987) Cells instead of Mach1 cells from -70°C Freezer
25ul for 10ul Gibson Mix.
Mix Materials.
Incubate at 50°C for 1 hour.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 30 min rather than 1 hr. (Warm plates also)
Plate on LB-CARB and incubate in 37°C Incubator Room.
Re-Transformed Promoters: (never been transformed, not enough colonies, etc)
PCL2
YDR124W
HYM1
PRM6
PRM1
ECM18
PRM3
PRM2
Negative Control (heated at 55°C for 40 min due to being made late)
ALWAYS MAKE A NEGATIVE CONTROL!
Jessica did a Colony PCR for rtTA Promoters and a Miniprep for the 1st Transformation.
07/03/14
Results of Re-Transformtation of Inducible Promoters + GFP
Few to no colonies grew onthe transformed plates, which leads me to think that the Gibson Assembly failed because only PRM2 worked and was Gibson'ed prior to the other transformed promoters.
PRM2 grew many tiny colonies.
Promoter
Colonies
PRM2
TMTC
PCL2
2
PRM6
3
PRM3
2
YDR124W
~ 40
PRM1
0
ECM18
0
HYM1
0
Neg Ctrl
0
Possible Explanations for Failure:
Gibson Assembly failed due to bad Gibson Master Mix
Backbone concentrations were not high enough, which would make sense because PRM2 had a new backbone which had a higher concentration than the rest of the promoters.
PRM2 Backbone Concentration: 208.7 ng/ul
Backbone Digestion
Jessica remade the backbones, but the concentrations are still very low.
Used the minipreps from my box in the -20°C fridge.
Colony PCR of Miniprep (Inducible Promoter + GFP)
|Reagents                      | 1X   | 20X   | |----------------------------- |------| ----- | |2X GoTaq Green PCR Master Mix | 10µl | 200µl | |10 µM Forward primer          | 1µl  | ----- | |10 µM Reverse primer          | 1µl  | 20µl  | |Water                         | 5µl  | 100µl | |Miniprep                      | 1µl  | ----- |  |Total                         | ---- | 320ul |      Initial Denaturation   | 95°C | 5 min        30 Cycles of:           Denaturation | 95°C | 45s           Annealing    | 55°C | 30s           Extension    | 72°C | 1min per kb     Final Extension | 72°C | 10m     Hold               |  4°C  | Forever
10uM FW Primer = individual inducible promoter primers - kept in Jessica's box.
10uM RV Primer = rtTA primer - kept in 2014 iGEM primer working stock box.
Sent into sequencing.
Gel of Miniprep Colony PCR (Inducible Promoter + GFP):
07/07/14
Confusion with Minipreps
Due to a confusion on 7/3/14, all minipreps from before 7/7/14 are to be thrown away from my box. Only the needed minipreps are to be kept.
Inserts were remade for no reason. (old inserts were fine)
Backbones may not have been digested properly. (minimum of 2 hrs needed)
07/08/14
PCR Amplification of new rtTA
Amplification of new rtTA with homology to AFRP's.
extract rtTA from pTS47 (~100ng/ul)
Materials
1x reaction
12x Master Mix
5x Phusion HF Buffer
10 µl
120 µl
dNTP's (10 mM)
1 µl
12 µl
Forward Primer (10µm)
2.5 µl
individually added
Reverse Primer (10µm)
2.5 µl
30 µl
Template DNA:rtTA
1 µl
12 µl
Phusion Polymerase
0.5 µl
6 µl
Water
31 µl
372 µl
DMSO
1.5 µl
18 µl
Total
50 ul
570 ul
(Note: Add enzymes last)
PCR Reaction Procedures:
Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom.
Pipette 50ul from the Master Mix into four labled PCR tubes
Put in the thermocycler for the following cycles:
Initial Denaturation   | 95°C | 5 min    30 Cycles of:        Denaturation | 95°C | 30 sec       Annealing    | 55°C | 30 sec       Extension    | 72°C | 2 min Final Extension    | 72°C | 5 min Hold               | 4° C | Forever
Keep samples for gel extraction on the following day.
Nanodrop Concentration:
PRM1 = 115.9 ng/ul
PRM2 = 176.6 ng/ul
PRM3 = 169.2 ng/ul
PRM6 = 154.9 ng/ul
ECM18 = 122.5 ng/ul
SAG1 = 161.9 ng/ul
YDR124W = 99.70 ng/ul
CLG1 = 153.5 ng/ul
ASG7 = 139.9 ng/ul
HYM1 = 123.5 ng/ul
PCL2 = 1-2.3 ng/ul
07/09/14
Gibson Assembly of rtTA Inducible Promoters (Transformation #3)
Gibson Assembly Procedures:
|        Materials       |  Amount  |    |------------------------|----------| |Gibson Master Mix (2x)  |   5ul    | |Backbone (pHY4+AFRP+GFP)|   3ul    | |Insert (rtTA + primer)  |   1ul    | |ddH2O                   |   1ul    | |Total                   |   10ul   |
Mix Materials.
Incubate at 50°C for 1 hour.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 1 hr. (Warm plates also)
Plate on LB-CARB and incubate in 37°C Incubator Room overnight.
07/10/14
Results of Transformation from 7/9/14
Most of the plates grew well except:
PRM1
PRM2
PRM6
HYM1
Positive control worked.
Negative control also worked.
maybe the backbones were not digested enough?
Going to redo backones for PRM1 and PRM2 (to be redigested)
Backbone Digest for PRM1 and PRM2
Materials
PRM1
PRM2
NotI
0.5 ul
0.5 µl
XhoI
0.5 µl
0.5 µl
CutSmart
5 µl
5 ul
pGEM
0 µl
4 µl
Original Digest
28 µl
40 µl
H2O
16 µl
0 µl
Total
50 ul
50 ul
Incubate at 37°C for at least 2 hours.
PCR Purify.
PCR Purification Procedure:
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
To bind DNA, apply sample to column and spin for 1 min
Discard flow through
To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Concentrations of Backbone Digestions of PRM1 and PRM2 were too low; not usable.
Colony PCR Gel of Transformation #3:
07/11/14
Miniprep of Transformation #3
Made cultures of the the transformations.
Sent Transformation #3 into sequencing.
07/14/14
Results of Sequencing and Colony PCR
PCR showed that rtTA and GFP were both present.
Sequencing showed that GFP was in the backbone, not rtTA
rtTA was probably not in the backbone and was floating near the colonies instead, so when we picked up the colonies for Colony PCR, we also picked up the rtTA with them.
Will use Backbone set #2 in Jessica's box along with Derrick's PRM1 and PRM2 backbones from 6/09/14.
Will use the new inserts from 7/11/14 in my box.
Gibson Assembly and Transformation #4
Gibson Assembly Procedures:
|        Materials       |  Amount  |    |------------------------|----------| |Gibson Master Mix (2x)  |   5ul    | |Backbone (pHY4+AFRP+GFP)|  Varies  | |Insert (rtTA + primer)  |   1ul    | |ddH2O                   |  Varies  | |Total                   |   10ul   |  Promoter |Concentration of Backbone| Amount of Insert Needed  |H2O Needed|  |------- | ----------------------- | ------------------------ | -------- | PRM1     | ~50.0 ng/ul             | 1 ul                     |   3 ul   | PRM2     | ~50.0 ng/ul             | 1 µl                     |   3 ul   | PRM3     | 15.20 ng/ul             | 1 µl                     |   1 ul   | PRM6     | 21.74 ng/ul             | 1 µl                     |   1 ul   | ECM18    | 21.39 ng/ul             | 1 µl                     |   1 ul   | SAG1     | 12.10 ng/ul             | 1 µl                     |   1 ul   | YDR124W  | 15.46 ng/ul             | 1 µl                     |   1 ul   | CLG1     | 16.59 ng/ul             | 1 µl                     |   1 ul   | ASG7     | 23.77 ng/ul             | 3 µl                     |   1 ul   | HYM1     | 12.96 ng/ul             | 1 µl                     |   1 ul   | PCL2     | 6.348 ng/ul             | 1 µl                     |   0 ul   | Pos Ctrl | Gibson Positive Ctrl    | 3 µl Backbone 1 ul Insert|   0 ul   | Neg Ctrl | PRM1 Backbone           | 1 ul Backbone 0 ul Insert|   4 ul   |
Concentrations of PRM1 and PRM2 Backbones were around 200 ng/ul. I diluted them 1:4 down to 50 ng/ul.
Mix Materials.
Incubate at 50°C for 1 hour.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 1 hr. (Warm plates also)
Plate on LB-CARB and keep in drawer over the weekend.
07/15/14
Glycerol Stocks for pTET + GFP + rtTA
Made cultures of pTET + GFP + Const. Promoter + rtTA.
CB008: pTEF1, M6, M7, and M10
CB008DB: M6, and M7
Made a streak plate with the strains on them.
Did a PCR of the rtTA Backbone and Inducible Promoter Insert.
Used pGEM20 rtTA backbone (~100 ng/ul dilution)
Gel Picture of PCR:
Transformation of HY130E into DH5Fα Cells
All of DNA (~2ul) + 50ul DH5α.
Put on ice for 10 min.
Heatshock for 45 sec at 42°C.
Put on ice for at least 2 min.
Add 250ul SOC.
Incubate for 30 min in 37°C Room Shaker.
Plate 100ul on LB-CARB plate.
Plate 200ul on another LB-CARB plate.
07/16/14
rtTA Insert
Used pGEM16 Template to make rtTA insert.
Gel Extraction of rtTA PCR
Gel Extraction Procedure:
Cut Gel
Weigh it in a colorless tube
Add 3 volumes Buffer QG to 1 volime Gel (100mg ~ 100µl)
Incubate @ 50°C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)
Add 1 gel volume isopropanal to the sample and mix
Place a QlAquick soin column in a provided 2ml collection tube
Place sample in column & spin for 1 min -> discard flow through
To wash add 0.75 ml Buffer PE to column & centrifue for 1 min, then dry spin
Place column in 1.5 ml tube
Add 30µl H2O & centrifuge for 1 min.
Gel #1:
Gel #2:
Fragment Weights: (try to cut smaller pieces)
PRM1 = 0.54 g = 540ul
PRM2 = 0.56 g = 560ul
PRM3 = 1.15 g = 1150ul
PRM6 = 1.21 g = 1210ul
ECM18 = 0.61 g = 610ul
SAG1 = 0.68 g = 680ul
YDR124W = 0.45 g = 450ul
CLG1 = 0.4 g = 400ul
ASG7 = 0.31 g = 310ul
HYM1 = 0.50 g = 500ul
PCL2 = 0.42 g = 420ul
M3 = 0.69 g = 690ul
Initial Concentrations of Gel Extraction were too low. Re-PCRed and obtained new concentrations. See Jeffrey's notebook for initial concentrations.
Assisted Jeffrey in doing gel extraction for M3.
New Concentrations:
PRM1 = 102.2 ng/ul
PRM2 = 110.6 ng/ul
PRM3 = 135.5 ng/ul
PRM6 = 130.7 ng/ul
ECM18 = 136.8 ng/ul
SAG1 = 152.4 ng/ul
YDR124W = 78.89 ng/ul
CLG1 = 133.7 ng/ul
ASG7 = 68.15 ng/ul
HYM1 = 91.08 ng/ul
PCL2 = 86.63 ng/ul
M3 = 47.19 ng/ul
Stored in Jessica's box in -20°C fridge.
07/17/14
Gibson and Transformation (Inducible Promoter + rtTA) #5
Gibson Assembly Procedure:
|        Materials       |  Amount  |    |------------------------|----------| |Gibson Master Mix (2x)  |   5ul    | |Backbone (pHY4+AFRP+GFP)|  Varies  | |Insert (rtTA + primer)  |  Varies  | |ddH2O                   |  Varies  | |Total                   |   10ul   |  Promoter |Amount of Backbone Needed| Amount of Insert Needed  |H2O Needed|  |--------| ----------------------- | ------------------------ | -------- | PRM1     | 1ul                     | 1 ul                     |   3 ul   | PRM2     | 3ul                     | 1 µl                     |   3 ul   | PRM3     | 3ul                     | 1 µl                     |   1 ul   | PRM6     | 3ul                     | 1 µl                     |   1 ul   | ECM18    | 3ul                     | 1 µl                     |   1 ul   | SAG1     | 3ul                     | 1 µl                     |   1 ul   | YDR124W  | 3ul                     | 1 µl                     |   1 ul   | CLG1     | 3ul                     | 1 µl                     |   1 ul   | ASG7     | 3ul                     | 2 µl                     |   0 ul   | HYM1     | 3ul                     | 1 µl                     |   1 ul   | PCL2     | 4ul                     | 1 µl                     |   0 ul   | Pos Ctrl | 3ul                     | 2 ul                     |   0 ul   | Neg Ctrl | 3ul                     | 1 ul                     |   1 ul   |
Used YDR124W Backbone for Negative Control
Mix Materials.
Incubate at 50°C for 1 hour.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 1 hr. (Warm plates also)
Plate on LB-CARB and keep in drawer over the weekend.
07/18/14
Results of Transformation #5
| Promoter | Colonies|                   |----------|---------| | PRM1     |    0    | | PRM2     |    4    | | PRM3     |    0    | | PRM6     |  TMTC   | | ECM18    |    0    | | SAG1     |  ~25    | | YDR124W  |    0    | | CLG1     |    0    | | ASG7     |    0    | | HYM1     |    0    | | PCL2     |    0    | | Pos Ctrl |    0    | | Neg Ctrl |    0    |
Used only 100ul for transformation.
Going to try again with the remaining 150ul.
Stored in drawer at room temperature for transformation over the weekend.
Since Positive Control did not work, I suspect that either the positive control was bad or that the Gibson was bad.
07/21/14
Results of Transformation #6
| Promoter | Colonies|                   |----------|---------| | PRM1     |   30    | | PRM2     |   30    | | PRM3     |   40    | | PRM6     |  TMTC   | | ECM18    |   30    | | SAG1     |   50    | | YDR124W  |   20    | | CLG1     |   20    | | ASG7     |   20    | | HYM1     |   30    | | PCL2     |   30    | | Pos Ctrl |   20    | | Neg Ctrl |   20    |
Since Positive Control did grow very much and the Negative Control worked, I suspect that either the positive control was bad or that the Gibson was bad.
Incubated for 4 more hours in the 37°C Incubator Room so that the colonies could grow slightly bigger. (All the colonies are very tiny)
Made cultures for PRM2, PRM6, and SAG1 for Colony PCR/Miniprep.
Sent Colony PCR/Miniprep of PRM2, PRM6, and SAG1 into sequencing.
Since most of the transformations did not actually seem to work, we will be trying Seamless to see if the transformations are any better.
Backbone Redigestion
Decided to redigest backbones for higher concentrations
Backbones: PRM3, SAG1, YDR124W, HYM1, PCL2
pGEM #: pGEM3, pGEM6, pGEM7, pGEM10, pGEM11
Materials
PRM3
SAG1
YDR124W
HYM1
PCL2
NotI
0.5 ul
0.5 µl
0.5 ul
0.5 ul
0.5 ul
XhoI
0.5 µl
0.5 µl
0.5 ul
0.5 ul
0.5 ul
CutSmart
5 µl
5 ul
5 ul
5 ul
5 ul
pGEM
5 µl
4 µl
4 ul
7 ul
7 ul
H2O
34 µl
40 µl
40 ul
37 ul
37 ul
Total
50 ul
50 ul
50 ul
50 ul
50 ul
Incubate at 37°C for at least 2 hours.
Heat inactivate at 65°C for 20 min.
Add 10ul of 6x loading dye to each sample.
Gel extract.
Gel Extraction Procedure:
Cut Gel
Weigh it in a colorless tube
Add 3 volumes Buffer QG to 1 volime Gel (100mg ~ 100µl)
Incubate @ 50° C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)
Add 1 gel volume isopropanal to the sample and mix
Place a QlAquick soin column in a provided 2ml collection tube
Place sample in column & spin for 1 min -> discard flow through
To wash add 0.75 ml Buffer PE to column & centrifue for 1 min, then dry spin
Place column in 1.5 ml tube
Add 30µl H2O & centrifuge for 1 min.
New Concentrations of Backbones:
Promoter
Concentration
PRM3
13.48 ng/ul
SAG1
44.26 ng/ul
YDR124W
82.00 ng/ul
HYM1
39.66 ng/ul
PCL2
82.97 ng/ul
Seamless Cloning and Assembly Reaction
Seamless Procedure:
Promoter |Amount of Backbone Needed| Amount of Insert Needed  | Seamless |  |--------| ----------------------- | ------------------------ | -------- | PRM1     | 1ul                     | 2 ul                     |   3 ul   | PRM2     | 3ul                     | 2 µl                     |   3 ul   | PRM3     | 3ul                     | 1.5 µl                   |   1 ul   | PRM6     | 3ul                     | 1.5 ul                   |   1 ul   | ECM18    | 3ul                     | 1.5 µl                   |   1 ul   | SAG1     | 3ul                     | 1.3 µl                   |   1 ul   | YDR124W  | 3ul                     | 2.5 µl                   |   1 ul   | CLG1     | 3ul | 1.5 µl                   |   1 ul   | ASG7     | 3ul                     | 3 µl                     |   0 ul   | HYM1     | 3ul                     | 2.2 µl                   |   1 ul   | PCL2     | 4ul                     | 2.3 µl                   |   0 ul   |  |        Materials                |  Amount  | |---------------------------------|----------| |Inserts (200ng each)             |   x ul   | |Linear Cloning Vector (50ng each)|   y ul   | |GeneArt 2X Enzyme Mix            | x+y ul   |
In a microcentrifuge tube, set up the seamless cloning and assembly reaction. It is CRUTIAL that you add the GeneArt 2x Enzyme Mix as the LAST COMPONENT.
Quickly thaw the GeneArt 2x Enzyme Mix on ice and pipette up and down to mix thoroughly. Add (x+y) ul of thawed enzyme. Mix to each reaction. Immediately return GeneArt 2x Enzyme Mix to -80°C Freezer.
Mix the reaction components completely by pipetting up and down SLOWLY 3 times and then gently tap the side of the tubed 3-5 times to mix.
If needed, briefly centrifge (< 500rpm for < 5sec) to collect reaction components to the bottom of the microcentrifuge tube.
Incubate the reaction mix at room temperature for 15-30 min. Do not incubate longer than 30 min!
After incubation is complete, plate the reaction mix on ice for 2-5 min before proceeding to transformation step. Do not let the samples stay on ice for more than 5 min before transformation.
Transformation:
Use 3ul of the seamless cloning and assembly reaction to transform NEB5α cells. Do not use electrocompetent cells.
Follow normal Transformation procedures.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 1 hr. (Warm plates also)
Plate on LB-CARB and keep in drawer over the weekend.
Miniprep of Transformations (PRM2, PRM6, SAG1)
Used cultures made on 7/18/14 (put in incubator on 7/20/14)
Miniprep Precedure:
Obtain E.Coli Cultures from 06/16/14 in 37°C  Incubation Room.
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Promoter
Concentration
PRM2
225.0 ng/ul
PRM6
491.5 ng/ul
SAG1
439.0 ng/ul
Stored in my box in -20°C fridge.
Sent into sequencing.
Anasuya came in and talked to us about some useful articles to read: mine is the one about "Binding of the B cell Activation..."
07/22/14
Results of Transformation
Seamless Transformation from 7/21/14
| Promoter | Colonies|                   |----------|---------| | PRM1     |  some   | | PRM2     |  some   | | PRM3     |    0    | | PRM6     |    0    | | ECM18    |    0    | | SAG1     |    0    | | YDR124W  |    0    | | CLG1     |    0    | | ASG7     |    0    | | HYM1     |    0    | | PCL2     |    0    | | Pos Ctrl |  some   | | Neg Ctrl |  some   |
Possible Explanations:
Used wrong inserts
Seamless went wrong
Mutations of template
Backbones not digested properly (not likely)
PRM1 and PRM2 backbones have GFP
Concentrations of both insert and backbone too low
Final Concensus: We will use another method to transform the cells:
Instead of using Gibson or Seamless, we will PCR with primers for all 7kb.
Currently ordering primers.
Meanwhile, transformations from 7/18/14 worked - colonies grew for all plates
will attempt colony PCR to determine the presence of rtTA
plates to be PCR'ed: PRM3, ECM18, YDR124W, CLG1, ASG7, HYM1, and PCL2
plates to be excluded: PRM1 and PRM2 (both backbones have GFP), PRM6 (has both mutation and rtTA, determined by previous sequencing), SAG1 (backbone has GFP but has recently been redigested), '+' control, '-' control
Culturing pGEM #1-11 (except pGEM7) to make pGEM for backbone digestion tomorrow.
07/23/14
pGEM Miniprep and Overnight Backbone Digestion
Collected cultures from 37°C incubator room - shaker was off for some reason - I turned it back on
Cells grew ok despite shaker incident
pGEM 4 and pGEM6 did not grow very well
Did a miniprep and gel extraction for backbone digestion
07/24/14
Digested Backbones
Did a gel extract.
Concentrations of Backbones:
PRM1 = 60.10 ng/ul
PRM2 = 61.79 ng/ul
PRM3 = 41.36 ng/ul
PRM6 = 55.34 ng/ul
ECM18 = 55.11 ng.ul
SAG1 = 89.55 ng/ul
YDR124W = 35.55 ng/ul
CLG1 = 49.42 ng/ul
ASG7 = 40.27 ng/ul
HYM1 = 33.18 ng/ul
PCL2 = 29.47 ng/ul
Backbones stored in my bix in -20°C fridge.
07/25/14
Transformation
Collected overnight digests of rtTA and Msn2 from 37°C incubator room.
PCR Purification
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
To bind DNA, apply sample to column and spin for 1 min
Discard flow through
To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Concentrations of digests:
rtTA = 28.98 ng/ul
Msn2 = 34.86 ng/ul
Ligation
1:3 Backbone to Insert
Materials:
Reagents
1x
10x ligase buffer
1 µl
Backbone
2 µl
Msn2
1 µl
rtTA
1 ul
T4 DNA Ligase
1 µl
H2O
4 µl
Total
10 µl
Keep at room temperature for at least 2 hours.
Transformation
Materials
1x
Ligation
10µl
E. Coli Competent Cells
25ul
Mix both together and keep on ice for 30 minutes.
42°C Heat shock for 45 seconds.
Put immediately on ice for at least 2 minutes.
Add 250 ul of SOC media, shake for 1 hr at 37°C.
Plate on selective media and store at room temperature over the weekend.
Had extra cells, so I added 50ul to PCL2.
07/29/14
Colony PCR of AFRP + rtTA
Use rtTA FW primer instead of individual promoter primers
Cultures to use for Colony PCR: PRM2-1, PRM3-2, PRM6-1, ECM18-1, SAG1-3, ASG7-2, HYM1-3
use plates for the rest of the AFRPs
Use primers 40 and 41, rtTA FW and RV
Colony PCR Procedure:
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 25ul of water.Do this for 5 colonies. Use 5ul for PCR reaction below.
Set up PCR Reaction below:
Reagents
1X
34X
2X GoTaq Green PCR Master Mix
10 µl
340 µl
10 µM Forward primer
1 µl
34 µl
10 µM Reverse primer
1 µl
34 µl
Water
1.5µl
51 µl
Bacterial cells (template)
5 µl
------
DMSO
1.5 µl
51 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation   | 95° C | 5 min    35 Cycles of:        Denaturation | 95° C | 45s       Annealing    | 55° C | 30s       Extension    | 72° C | 1min per kb Final Extension    | 72° C | 10m Hold               |  4°C  | Forever
Load PCR products on a gel. (5ul) (GoTaq already has loading dye)
For all positive bands on the gel, add rest of bacterial cells from PCR tubes to 5ml LB and grow overnight at 37°C for miniprep on the following day.
Gel Map of Colony PCR:
07/30/14
Re-Colony PCR of SAG1
Redid Colony PCR for SAG1
Miniprep of Colony PCR
Cultures to be miniprep-ed:
PRM1.6
PRM2.1
PRM3.2
PRM6.4
ECM18.1
YDR124W.6
CLG1.5
ASG7.4
HYM1.3
PCL2.6
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
07/31/14
Sequencing of AFRP + rtTA
Sequencing of 7/30/14 Minipreps
Sent in: PRM1, PRM2, PRM3, PRM6, ECM18, YDR124W, CLG1, ASG7, HYM1, PCL2
SAG1 on hold - PRM6 and PCL2 need to be re-Colony PCRed.
Did a Colony PCR for PRM6, SAG1, and PCL2
Made cultures with 5ml LB+CARB and 20ul cells
PRM6 = 7, 8, 9
SAG1 = 10, 11, 12
PCL2 = 7, 8, 9 (plate may have been contaminated, used white colonies)
Colony PCR failed - no rtTA - decided to re-transform
Ligation
see procedure above
stored at 16°C overnight (room temperature)
08/01/14
Transformation of Ligation into NEB5α (C2987) cells
see procedure above - used 25ul cells - stored in 37°C incubator room -Transformed PCL2, PRM6, SAG1, and negative control
Yeast Transformation
linearized DNA
boiled salmon sperm at 100°C for 10 min
Yeast Transformation Procedure:
Reagents
YPD
1 M LiOAc
10X TE pH 7.5
1X TE pH 7.5, 0.1 M LiOAc
50% PEG 3350
DMSO
Salmon Sperm DNA (ssDNA)
PEG is viscous, so pipette slowly to prevent air bubbles from forming.
boil ssDNA aliquots for 10 min, then ice down for 10 min before use.
Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C
Set up digest to linearize DNA
Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C
Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)
Harvest cells in centrifuge - 3000 rpm, 2-5 min
Wash with 1 ml 0.1 M LiOAin TE
Pellet cells - 3000 rpm , 2-5 min
Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation
to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA
Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc) Optional: Add 75 µl DMSO definitely do
Vortex
Incubate 42° C for 30m & begin drying plates
Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)
Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD
Plate on selective media
Incubate 1-3 days at 30°C
08/04/14
Results of E.Coli and Yeast Transformations
E.Coli plates:
PRM6 = 30
SG1 = 20
PCL2 = 20
Negative Control = 20
Yeast plates:
CB008: all plates grew at least 30+ colonies
CB008DB: all plates grew at least 30+ colonies
YPD became contaminates - plates may be compromised
Did a Colony PCR on successful E.Coli plates.
FW primer = rtTA FW #40
RV primer = rtTA RV #41
cultures = 5ml LB+CARB + 20ul cells
SD, YPD, and LB were contaminated, as a result, some of our yeast plates became contaminated.
Jessica did a Colony PCR for the successful Yeast Plates
Used the uncontaminated colonies for Colony PCR
See Colony PCR procedures for both E.Coli and Yeast above.
08/05/14
Miniprep of E.Coli Cultures
Cultures = PRM6, SAG1, PCL2
AFRP+rtTA
Miniprep Precedure:
Obtain E.Coli Cultures from 8/4/14 in 37°C  Incubation Room.
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Miniprep Concentrations:
PRM6 = 611.2 ng/ul
SAG1 = 478.1 ng/ul
PCL2 = 384.8 ng/ul
Sent into sequencing
Linearization of E.Coli Minipreps
DIgestion Materials:
Reaction
Amount
DNA
~2000ng
Pme1
0.5ul
CutSmart
2.5ul
H2O
x ul
Total
25ul
Concentration Calculation:
Promoter
Concentration of Miniprep
Amount of Miniprep Needed
H2O
PRM1
376.7 ng/ul
6 ul
16 ul
PRM2
356.8 ng/ul
6 µl
16 ul
PRM3
364.5 ng/ul
6 µl
16 ul
PRM6
611.2 ng/ul
4 ul
18 ul
ECM18
371.7 ng/ul
6 µl
16 ul
SAG1
478.1 ng/ul
5 µl
17 ul
YDR124W
328.4 ng/ul
6 µl
16 ul
CLG1
543.1 ng/ul
4 µl
18 ul
ASG7
348.8 ng/ul
6 µl
16 ul
HYM1
449.6 ng/ul
5 µl
17 ul
PCL2
384.8 ng/ul
6 µl
16 ul
Add reagents and incubate for 2 hours at 37°C  Incubation Room.
Also assisted Jeffrey in lineaerizing 11E3128 (423.0 ng/ul) and pGEM22 (384.8 ng/ul).
Transformed linearizations into yeast
CB008: PRM6, ECM18, SAG1, PCL2
CB008DB: all 11 promoters
Forgot to make negative control
Yeast Transformation Procedure:
Reagents
YPD
1 M LiOAc
10X TE pH 7.5
1X TE pH 7.5, 0.1 M LiOAc
50% PEG 3350
DMSO
Salmon Sperm DNA (ssDNA)
PEG is viscous, so pipette slowly to prevent air bubbles from forming.
boil ssDNA aliquots for 10 min, then ice down for 10 min before use.
Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C
Set up digest to linearize DNA
Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C
Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)
Harvest cells in centrifuge - 3000 rpm, 2-5 min
Wash with 1 ml 0.1 M LiOAin TE
Pellet cells - 3000 rpm , 2-5 min
Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation
to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA
Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc) Optional: Add 75 µl DMSO definitely do
Vortex
Incubate 42° C for 30m & begin drying plates
Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)
Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD
Plate on selective media
Incubate 1-3 days
08/08/14
Results of Transformation
CB008 = all plates grew 20-30 colonies except for PRM6, which did not grow.
CB008DB = all plates grew 20-30 colonies
Both plates had small colonies
Did a Colony PCR on the Transformations
Flow Testing
CB008 Const+rtTA+GFP+mfa+pAGA mCherry
Plate map for all time points (4 plates):
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 m6-1]----]       [CB008 m10-3]----] B [CB008 m6-2]----]       [CB008DB m7-1]---] C [CB008 m6-3]----]       [CB008DB m7-2]---] D [CB008 m7-1]----]       [CB008DB m7-3]---] E [CB008 m7-2]----]  F [CB008 m7-3]----]            G [CB008 m10-1]---]  H [CB008 m10-2]---]
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12
See notes on flow in Eric's notebook.
08/12/14
Colony PCR of Yeast for rtTA and mfa
Boil Yeast at 95°C for 20 minutes.
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 15ul of NaOH. Take 3 colonies from each plate. Use 5ul for PCR reaction below.
Set up PCR reaction below:
Reagents (rtTA)
1X
43X
2X GoTaq Green PCR Master Mix
10 µl
430 µl
10 µM Forward primer
1 µl
43 µl
10 µM Reverse primer
1 µl
43 µl
Water
1.5µl
64.5 µl
cells (template)
5 µl
-------
DMSO
1.5ul
64.5 ul
FW primer = pSV606
Reagents (rtTA)
1X
7X
2X GoTaq Green PCR Master Mix
10 µl
70 µl
10 µM Forward primer
1 µl
7 µl
10 µM Reverse primer
1 µl
7 µl
Water
1.5µl
10.5 µl
cells (template)
5 µl
-------
DMSO
1.5ul
10.5 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation    | 95°C | 5 min    35 Cycles of:        Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C | Forever
Load PCR products on a gel. (5ul) (GoTaq already has loading dye)
Results of Colony PCR:
08/13/14
Doxycyline Dilutions (Dilutions by Ianto)
Stock in Siberia Fridge (Middle Shelf, Middle Rack, Top Drawer, 3rd Box) 100 mg/ml
x
Concentration
Take from x
Amount of x Needed
H2O Needed
A
60
Stock
60 ul
940 ul
B
30
Stock
30 ul
970 ul
C
9
A
150 ul
850 ul
D
6
A
50 ul
900 ul
E
3
A
50 ul
950 ul
F
0.9
C
100 ul
900 ul
G
0.6
D
100 ul
900 ul
H
0.3
E
100 ul
900 ul
I
0.09
F
100 ul
900 ul
J
0.06
G
100 ul
900 ul
K
0.03
H
100 ul
900 ul
Vortex while like making and before adding to each dilution.
Assisted Derrick in running plates (time points: 0, 1.5, 3, 5)
See Derrick's lab norebook for reference.
08/17/14
Due to contamination of the plates from the flow run of 8/13/14, another run will be needed of the same experiment.
Plate had large clumps of cells due to not being cleaned out well.
Make sure to put finished shaker plate in bleach bin (near Hyun's desk, under the table in a blue bin)
prepared cultures of pTEF1, M6, M7, and M10 to be diluted on the following day for flow.
CB008 const+rtTA+GFP+mfa+pAGA1+mCherry
Plate Map
1
2
3
4
5
6
7
8
9
10
11
12
A
1
1
1
1
1
1
7
7
7
7
7
7
B
1
1
1
1
1
1
10
10
10
10
10
10
C
1
1
1
1
1
1
10
10
10
10
10
10
D
6
6
6
6
6
6
10
10
10
10
10
10
E
6
6
6
6
6
6
F
6
6
6
6
6
6
G
7
7
7
7
7
7
H
7
7
7
7
7
7
1 = PTEF1 6 = M6 7 = M7 10 = M10
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6       [----------6 µg/ml----------] 7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12      [----------6 µg/ml----------]
08/18/14
CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2
Ran plate with 5 time points (0, 1.5, 3, 5, 8)
added 15ul of dox to induce
Added 10ul of cycloheximide to each well to stop growth.
Finished running the first two plates (0hr and 1.5hr)
Slow run may be due to the cells being overdilute. (source: Kara)
Plate run to be continued and finished on 8/19/14
CB008DB Const+rtTA+GFP+mfa+pAGA+mCherry
Started cultures for this experiment.
To be ran on 8/20/14
All 5 Const. Promoters available.
1
2
3
4
5
6
7
8
9
10
11
12
A
1
1
1
1
1
1
6
6
6
6
6
6
B
1
1
1
1
1
1
7
7
7
7
7
7
C
1
1
1
1
1
1
7
7
7
7
7
7
D
3
3
3
3
3
3
7
7
7
7
7
7
E
3
3
3
3
3
3
10
10
10
10
10
10
F
3
3
3
3
3
3
10
10
10
10
10
10
G
6
6
6
6
6
6
10
10
10
10
10
10
H
6
6
6
6
6
6
1 = PTEF1 3 = M3 6 = M6 7 = M7 10 = M10
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6       [----------6 µg/ml----------] 7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12      [----------6 µg/ml----------]
08/20/14
CB008DB Const+rtTA+GFP+pAGA1+mCherry Run
Ran plate with 5 time points (0, 1.5, 3, 5, 8)
Ran first plate and noticed that specimen 5 only had 10ul loaded due to the addition of a new speciment (parameters were reset)
Readjusted the parameters and reran the well that only had 10ul loaded
Make sure to check parameters for all wells before running plate.
Stopped plate run at 3hr plate - D7
08/21/14
CB008DB Const+rtTA+GFP+pAGA1+mCherry Run (Cont.)
Continued running 3hr plate from 8/20/14
Finished 3 and 5 hr plates but could not do 8 hr plate because the plate was dropped.
Hyun said not to run DB plates
08/25/14
CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Preparation
Plate Map
1
2
3
4
5
6
7
8
9
10
11
12
A
1
1
1
1
1
1
7
7
7
7
7
7
B
1
1
1
1
1
1
10
10
10
10
10
10
C
1
1
1
1
1
1
10
10
10
10
10
10
D
6
6
6
6
6
6
10
10
10
10
10
10
E
6
6
6
6
6
6
F
6
6
6
6
6
6
G
7
7
7
7
7
7
H
7
7
7
7
7
7
1 = PTEF1 6 = M6 7 = M7 10 = M10
Grew cultures and put into 30°C incubator.
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6       [----------6 µg/ml----------] 7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12      [----------6 µg/ml----------]
08/26/14
CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Run
Was only able to finish running 0 hr plate and 5 hr plate up to C2.
Wells D4 and G3 on plates should not have sample due to error during dilutions.
All of M10 seems to not have grown compared to the other strains.
08/27/14
CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Run (Cont.)
Accidentally did not run the last specimen (M10) on the 5hr plate due to Eric's advice on not wasting time because M10 did not grow.
Re-streaking of old plates
Started restreaking old streak plates onto organized plates. (organized by pGEM #s)
08/28/14
Data Analysis of CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Run
After using FlowJo and Matlab to analyze data, we have come to the conclusion that CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry does not have RFP.
Compared pPCL2 RFP to pAGA1 RFP runs. pAGA1 has much RFP levels.
Prepared cultures of CB008DB Const+rtTA+GFP+mfa+pPCL2+mCherry M3, M6, M7, and M10 to be ran on the following day to confirm the presence or lack of presence of RFP.
pTEF1 missing, may need to grow a glycerol stock streak plate of it.
09/02/14
CB008DB Const+rtTA+GFP+mfa+pPCL2+mCherry Run
Shaker plate not completely clean, I attemped to clean it out with water. Saw chunks of dried cells on the bottom of some of the wells. MAKE SURE YOU PUT THE SHAKER PLATE IN THE BLEACH BIN AFTER USE!!!
M10 may be contaminated due to the shaker plate.
M6 culture was not labeled correctly in the incubator shaker, M6 may not actually be the right specimen.
Ran 2 time points: 0 hr and 5 hrs.
Data to be analyzed in the near future after asking Ianto how to run only two time points.
09/04/14
Co-culture of CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry (pTEF1, M7, M10)
Plate Map:
A1-A6 = pTEF1+M7
B1-B6 = pTEF1+M7
C1-C6 = pTEF1+M7
D1-D6 = M7+M10
E1-E6 = M7+M10
F1-F6 = M7+M10
G1-G6 = M10+pTEF1
H1-H6 = M10+pTEF1
A7-A12 = M10+pTEF1
Dox Map:
Lane 1 = 0
Lane 2 = 0.03
Lane 3 = 0.06
Lane 4 = 0.09
Lane 5 = 0.6
Lane 6 = 6
Lane 7 = 0
Lane 8 = 0.03
Lane 9 = 0.06
Lane 10 = 0.09
Lane 11 = 0.6
Lane 12 = 6
Grew cultures of CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry
pTEF1
M7
M10
stored in 30°C incubator shaker.
09/05/14
Co-culture of CB008 Const+rtTA+GFP+mfa+pAGA+mCherry Run
5 Timepoints: 0, 1.5, 3, 5, 8.
Successfully ran all the plates.
09/10/14
Data Analysis of Co-culture of CB008 Const+rtTA+GFP+mfa+pAGA+mCherry Run
After running the data on FlowJo and Matlab, we compared the data to the original pAga1+mCherry pTEF1, M7, and M10 data.
The comparison showed slight variation and unexpected results.
Maybe due to the larger cells being too stressed from producing so many proteins, and the overall population producing less as a result of this.
After comparing the data, it was decided that the best option would be to rerun this experiment but with pTEF1, M7, and M10 separately put on the plate as a comparison.
Started growing cultures of pTEF1, M7, and M10 in the incubator.
09/11/14
Co-Culture of CB009 Const+rtTA+GFP+mfa+pAGA+mCherry Round 2
Plate Map:
A1-A6 = pTEF1
B1-B6 = pTEF1
C1-C6 = pTEF1
D1-D6 = M7
E1-E6 = M7
F1-F6 = M7
G1-G6 = M10
H1-H6 = M10
A7-A12 = M10
B7-B12 = M7+pTEF1
C7-C12 = M7+pTEF1
D7-D12 = M7+pTEF1
E7-E12 = M7+M10
F7-F12 = M7+M10
G7-G12 = M7+M10
Dox Map:
Lane 1 = 0
Lane 2 = 0.03
Lane 3 = 0.06
Lane 4 = 0.09
Lane 5 = 0.6
Lane 6 = 6
Lane 7 = 0
Lane 8 = 0.03
Lane 9 = 0.06
Lane 10 = 0.09
Lane 11 = 0.6
Lane 12 = 6
Did not make a co-culture of pTEF1+M10 because the two promoters are too close in activity to make much of a difference.
09/12/14
Attempt of Co-Culture CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2 Run
Cancelled due to wrong culture being grown and due to the absence of induction.
Make sure that Eleanor and George pay attention when growing cultures and when inducing with doxycycline.
pPCL2 was grown instead of pAGA1. pPCL2 cannot be used in a co-culture due to its lack of RFP.
Experiment to be continued at another time.
09/14/14
Co-Culture CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2 Preparation
Plate Map:
A1-A6 = pTEF1
B1-B6 = pTEF1
C1-C6 = pTEF1
D1-D6 = M7
E1-E6 = M7
F1-F6 = M7
G1-G6 = M10
H1-H6 = M10
A7-A12 = M10
B7-B12 = M7+pTEF1
C7-C12 = M7+pTEF1
D7-D12 = M7+pTEF1
E7-E12 = M7+M10
F7-F12 = M7+M10
G7-G12 = M7+M10
Dox Map:
Lane 1 = 0
Lane 2 = 0.03
Lane 3 = 0.06
Lane 4 = 0.09
Lane 5 = 0.6
Lane 6 = 6
Lane 7 = 0
Lane 8 = 0.03
Lane 9 = 0.06
Lane 10 = 0.09
Lane 11 = 0.6
Lane 12 = 6
Did not make a co-culture of pTEF1+M10 because the two promoters are too close in activity to make much of a difference.
Made sure to use pAGA1 this time.
Cultures growing in yeast incubator.
09/15/14
Co-Culture CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2 Run
Time Points: 0, 1.5, 3, 5, 8.
Made sure to induce with doxycycline this time.
Also made sure to create the right file name this time.
Successfully finished running the plate
Analyzed the data and made scatter plots
Made cultures of AFRP+rtTA+GFP+mfa+pAGA+mCherry
PRM1
PRM2
PRM3
YDR124W
CLG1
AFRP+rtTA+GFP+mfa+pAGA+mCherry Preparation
Plate Map:
A1-A6 = PRM1
B1-B6 = PRM1
C1-C6 = PRM1
D1-D6 = PRM2
E1-E6 = PRM2
F1-F6 = PRM2
G1-G6 = PRM3
H1-H6 = PRM3
A7-A12 = PRM3
B7-B12 = YDR124W
C7-C12 = YDR124W
D7-D12 = YDR124W
E7-E12 = CLG1
F7-F12 = CLG1
G7-G12 = CLG1
Dox Map:
Lane 1 = 0
Lane 2 = 0.03
Lane 3 = 0.06
Lane 4 = 0.09
Lane 5 = 0.6
Lane 6 = 6
Lane 7 = 0
Lane 8 = 0.03
Lane 9 = 0.06
Lane 10 = 0.09
Lane 11 = 0.6
Lane 12 = 6
09/16/14
AFRP+rtTA+GFP+mfa+pAGA+mCherry Run
Time Points = 0, 1.5, 3, 5, 8.
Successfully ran all plates.
09/17/14
Absent due to doctor's appointment.
09/18/14
Data Analysis of AFRP+rtTA+GFP+mfa+pAGA+mCherry Run
Ran the data on FlowJo and Matlab.
Got interesting results like bimodal populations and large amounts of noise.
Eleanor's Transformations
Assisted Eleanor in her transformations of CB008 M3 and M6 with pTET+mfa, pAGA1+mCherry, and pPCL2+mCherry.
None of her transformations worked other than M3+mfa which had 2 colonies. I put the plates back into the incubator in case they worked.
Made cultures of M3+rtTA and M6+mfa to re-transform the following day.
09/19/14
Results of Eleanor's Transformations
Plates that grew colonies:
CB008 M3 pAGA1+mCherry = 1 colony
CB008 M3 mfa = 2 colonies
CB008 M6 mfa+pPCL2+mCherry = 4 colonies
Will do a Colony PCR to check plates - unlikely that plates worked
Linearized plasmids: mfa and pAGA1+mCherry
Kara advises us to not used previously linearized plasmid.
will attempt to use Jeffret's digested pPCL2+mCherry for M6.
Re-transformed and put into incubator:
CB008 M3+mfa
CB008 M3+pAGA1+mCherry
CB008 M3+mfa+pAGA1+mCherry
CB008 M6+mfa+pAGA1+mCherry
CB008 M6+mfa+pPCL2+mCherry
9/22/14
Colony PCR of Transformations
Ran colony PCR's of the Transformations from 9/19 and 9/15 of the CB008 Constitutive Promoters.
Checked for mfa and pPCL2/pAGA1+mCherry separately.
Colony PCR Gel: (Eleanor said that M6 already has mfa even if there is no band)
09/25/14
PCR Amplification of Bar1, mfa, and pGEM1-16 (except 7 and 10)
Due to the primers not covering all of the needed insert parts, new primers with new overhangs were ordered. I was put in charge of PCR amplifying the plasmids with these new primers.
Used primers #103-110.
mfa Template: Shuaixin's PRM2 + mfa (7.606) Plasmid
Bar1 Template: Shuaixin's PCL2 + Bar1 (6.62) Plasmid
George did the PCR for Ste2.
Used Seamless to transform all the plasmids. George made a pGEM10 PCR insert, which I used for transformation.
Used pSB1C3 as backbone. Used cleaned up undigested one.
09/26/14
Results of Transformation
All the pGEM plates transformed very well.
Bar1, Ste2, and mfa did not transform very well.
Negative control grew for some reason also, hopefully because of a contamination of PCR product.
Colony PCR of Transformation
After a few more hours of growing in the incubator, Bar1, Ste2, and mfa grew more cells.
Did a colony PCR on all the plates (3 colonies from each plate) (a total of 62 Colony PCRs including 5 colonies of Eleanor's Bar1 Transformations)
Jeffrey ran the gel for my Colony PCR.
Colony PCR Gel:
09/28/14
I moved to UC Davis and will be taking a break from lab work to concentrate on classes.
Kara made cultures from the plates for the successful Colony PCR's to miniprep on Monday.
09/29/14
Miniprep of Cultures
Eleanor did the miniprep of the cultures from 9/28/14 and sent them into sequencing.
09/30/14
Results of Sequencing
Sequencing showed that the miniprep was wrong. May have to redo everything or pick new colonies off the plates? Need to discuss with Kara on this matter.
Retrieved from " http://2014.igem.org/Team:UCSF_UCB/sabrina "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
