"
Page
Discussion
View source
History
teams
Log in
 
Team:UCSF UCB/eric
From 2014.igem.org
Thurs 14/05/29
1-13 Promoter Number Key: 1) PRM2 2) ASG7 3) FUS2 4) PCL2 5) FUS3 6) CLG1 7) YDR124W 8) HYM1 9) PRM6 10) PRM1 11) ECM18 12) PRM3 13) SAG1
PCR of 13 Alpha-factor inducible promoters
Reagents
1x reaction
4.5x MM
1) 5x HF Phusion Buffer
10ul
45ul
2) 10mM dNTPs
1ul
4.5ul
3) 10mM FW primers
2.5ul
11.25ul
4) 10mM Rev primers
2.5ul
11.25ul
5) Phusion Polymerase
0.5ul
2.25ul
6) Template DNA
0.5ul
2.25ul
7) ddH2O
33ul
148.5ul
~1kb: YDR124W, SAG1, PCL2, HYM1, FUS3
~700bp: CLG1, PRM2
~500bp: PRM6, PRM3, PRM1, FUS2, ECM18, ASG7
PCR Cycle (30) 98ºC    30secs 98ºC    10secs 55ºC    20secs 72ºC    30secs 72ºC    5mins 4ºC     hold  PCR products of #3,10,12 did not work. We Purified and nanodropped for concentrations:                 ng/ul 1) PRM2         13.49 2) ASG7         12.50 3) FUS2 4) PCL2         9.489 5) FUS3         11.77 6) CLG1         16.30 7) YDR124W      35.75 8) HYM1         23.61 9) PRM6         79.03 10) PRM1 11) ECM18       14.17 12) PRM3 13) SAG1        26.07
Fri 14/05/30
Miniprepped overnight cultures of phy4 (4)
Conentrations:
55.15 ng/ul
65.40 ng/ul <-- pipette error, less than 43ul so we added 5ul H2O
52.89 ng/ul
31.65 ng/ul
Used #1,2 for digest with ApaI and XhoI
Digestion (Yesterdays Promoter PCR products, 10)
29ul of product 33ul of 10x Cutsmart buffer  1) Add 0.5ul of ApaI. Mix gently, spin down, incubate at room temp. for 2 hours 2) Add 0.5ul of XhoI. Mix gently, spin down, incubate at 37ºC for 1 hour 3) PCR cleanup (purple tubes) for PCR reactions (Promoters)  Concentrations after PCR cleanup: 1) 14.54 2) 4.326 redo, 5.489 4) -0.05256 redo, 1.730 5) 1.886 redo, 2.642 6) 41.05 7) 3.187 8) 19.09 9) 9.677 11) 1.868 13) 23.78
Digestion (Backbone pHY4) Cutting out present promoter to replate with ours
43ul of DNA 5ul of Cutsmart  1) Add 1ul ApaI. Mix gently, spin down, incubate at room temp for 2 hours 2) Add 1ul of XhoI, Mix gently, spin down, incubate at 37ºC for 1 hour 3) Gel extract backbone, don't want the promoter cutout
Mon 14/06/02
Ligated #1,2,5,6,7,8,9,13 with the 6kb pHY4 backbone
We only had enough backbone for 8 reactions so we chose the 8 promoters with the highest concentration.
Used Gibson ligation calculator to calculate mLs of reagents of our reaction.
Genes
Concentrations
Backbone(ul)
Insert(ul)
Water(ul)
PRM2
14.54
3
1.1
12.9
ASG7
5.489
3
2
12
FUS3
1.73
2.3
10
4.7
CLG1
41.05
3
0.4
13.6
YDR124W
3.187
3
7.1
6.9
HYM1
19.09
3
1.2
12.8
PRM6
9.677
3
1.2
12.8
SAG1
23.78
3
0.9
13.1
Transformed α-inducible Promoters in pHY4 into DH5α competent cells. 50uls for each transformation instead of 25uls.
9 Plates for 8 promoters and 1 negative control.
Redo-PCR of AFRPs that didn't work or had low yield. Redoing: 3,4,7,10,11,12
After PCR with the same primers, we ran it through a 1% gel
Tues 14/06/03
Transformation of pHY4 + AFRP
#7  ⎫ #9  | Grew at least a few colonies so they did okay #13 ⎭
The rest, 1,2,5,6,8 didn't grow likely because of our low yield template promoter DNA.  We need to restart and PCR promoters again for higher yield DNA. Then ligate and transform
Yesterdays AFRP PCR yields: #4   84.35 ng/ul #7   134.3 ng/ul #10  84.14 ng/ul #11  91.99 ng/ul #12  64.64 ng/ul
Miniprepped cultures of Mach1 with pHY4 to get more pHY4 template.
*The PE buffer we used previously did not contain ethanol which was why we were having low yields for minipreps and other purifications.
Concentraions of pHY4 1) 360.2 ng/ul 2) 398.8 ng/ul 3) 322.2 ng/ul 4) 240.2 ng/ul
Digestion of AFRP PCR products (4,7,10,11,12)
50ul PCR product 5ul of 10x Cutsmart buffer
1) add 1ul of ApaI, incubate at room temp for 2 hours
2) add 1ul of Xho1, incubate at 37ºC for 1 hour
Digestion of pHY4 backbone (2 tubes)
43ul of DNA 5ul Cutsmart buffer  * 1 unit of enzyme = 1ug of DNA = 1000ng     * 300ng/ul *43ul = 12,900 = Need 13units     * Unit count depends on the RE. Need to check the product.     * Always want excess RE  1) add 1ul ApaI, incubate at room temp for 2 hours 2) add 1ul of Xho1, incubate at 37ºC for 1 hour
Ran the backbone digest through a gel.
The streaks of pHY4 suggests that its been cut up by other nucleases so we can't purify this as our backbone.  So we created overnight cultures of Mach1+pHY4 and are sequencing leftover digested phy4 to see if we have the backbone we want.
Created overnight cultures of the transformations that seemed to work: 7,9,13
Tomorrow we will miniprep and purify the construct for promoter testing.
Weds 14/06/04
Miniprepped yesterdays cultures of YDR124W, SAG1, and PRM6, pHY4
Concentrations of minipreps YDR124W.1     539.4 ng/ul YDR124W.2     555.4 ng/ul YDR124W.3   594.3 ng/ul YDR124W.4   652.5 ng/ul YDR124W.5   481.5 ng/ul YDR124W.6   377.7 ng/ul  PRM6.1      545.3 ng/ul PRM6.2      569.1 ng/ul PRM6.3        656.8 ng/ul PRM6.4      570.8 ng/ul PRM6.5      558.6 ng/ul  SAG1.1      538.1 ng/ul SAG1.2      532.1 ng/ul SAG1.3      505.8 ng/ul  pHY4.1      179.8 ng/ul pHY4.2      155.6 ng/ul pHY4.3      231.7 ng/ul
Ianto and Jessica digested and PCR purified 1,2,6,8,9,10,13
AFRP PCR Concentrations: 1. PRM2   58.91 ng/ul 2. ASG7   68.53 ng/ul 6. CLG1   15.13 ng/ul 8. HYM1   78.56 ng/ul 9. PRM6   132.6 ng/ul 10. PRM1  41.72 ng/ul 13. SAG1  91.24 ng/ul
Made a positive control digest of Hyun's pHY4 backbone.
Tomorrow we will run it through a gel and gel extracted it.
Thurs 14/06/05
Yesterday we made solutions for the yeast transformations we will do today.
Check chemical container information to confirm molar mass, sometimes they may be different.  50mL of LiOAc: Molar mass = 60.99g/mol / 20 = 3.3 grams in 50mL of H2O ~pH8  50mL Trizma: Molar mass = 157.60g/mol / 10 / 20 = 0.788 grams  50mL EDTA Molar mass = 292.24g/mol.  Used 0.146 grams in 50mL of H2O ~pH7.5  50% PEG.  Used 25grams of PEG + H2O until 50mL.  Heat in 50ºC water bath to melt all PEG.  Store tube in foil to protect from light at 4ºC
Fri 14/06/06
Out for the day. What I missed:
1-13 Promoter Number Key:     1) PRM2     2) ASG7     3) FUS2     4) PCL2     5) FUS3     6) CLG1     7) YDR124W     8) HYM1     9) PRM6     10) PRM1     11) ECM18     12) PRM3     13) SAG1
Transformed 1,2,4,6,7,10,11,12, and negative control.  The negative control grew colonies so the backbone may have liated back by itself so we are using phosphitase to remove phosphates so that it cannot ligate together.
They then colony PCRed and sequenced.
AFRP
1
2
3
4
5
1
✓
X
X
✓
✓
2
✓
✓
X
✓
✓
4
✓
✓
✓
✓
✓
6
✓
✓
✓
✓
✓
10
X
✓
✓
X
X
11
X
✓
X
✓
✓
12
✓
✓
✓
✓
✓
Mon 14/06/09
Today we are miniprepping 2 of each successful colonies from the colony PCR
YDR124W ⎫ PRM6    | All grew 100+ colonies SAG1    ⎭
We are doing colony PCR on 5 colonies of each transformaiton. All 5 colonies are also patched as a stock and are circled on the plate when colony PCR confirms that they have the gene knock in.
Anneling temp at 50ºC for 30secs Polymerase extension at 72ºC for 1min Length of PCR product should be 940bp
Tues 14/06/10
Ianto and Jessica are digesting pHY4 + AFRP, all 11, and are PCRing out the bakbone w/o the GFP.  Derrick has PCRed the rtTA with ends that have homolog to the digest ends NotI and XhoI of the plasmid being digested. 9 of 11 PCRs have worked for Derrick.  Digest overnight at 37ºC
Finished product should be pHY4+AFRP+rtTA. Then they will be digested with PmeI ad transformed into yeast.
Weds 14/06/11
Redoing PCR of 2 and 12 of rtTA with homology ends to the 2, 12 promoters and the backbone.
PCR of gibson homology of rtTA to ASG7 & PRM3
5x HF buffer            10ul dnTPs                   1ul FW primer               2.5ul Rev primer          2.5ul Phusion Polymerase  0.5ul Template                0.5ul ddH2O                   33ul  PCR cycle (30) 98ºC   30secs 98ºC   10secs 55ºC   20secs 72ºC   30secs 72ºC   5mins 4ºC        hold
Concentrations of the 9 successful so far of PCR of gibson homology
1.PRM2       190.9 ng/ul 4.PCL2   149.1 ng/ul 6.CLG1       412.1 ng/ul 7.YDR124W  90.37 ng/ul 8.HYM1       248.9 ng/ul 9.PRM6      145.7 ng/ul 10.PRM1    71.88 ng/ul 11.ECM18   199.7 ng/ul 13.SAG1    65.55 ng/ul
We gibsoned inserts of rtTA with homology to the PCR backbone w/o GFP.  The gibson reaction was in 10ul with 2 controls (backbone w/o insert)-1)CLG1, 2)PCL2
Incubated samples at 50ºC for 1hour
Thurs 14/06/12
The gibson reaction with backbone w/o GFP and insert of rtTA did not work
Today we will do a seamless reaction which is similar to gibson but is better with inserts that have shorter homology sequences.
Mon 14/06/16
Transformations of rtTA+AFRP in E.coli are going to be colony PCRed.  Of the 11, SAG1 & PRM1 did not grow.  We are PCRing rtTA, seeing if it has successfully transformed.
Protocol: Inoculate colony in 25ul of H2O, use 5ul for PCR template and the rest can be use for making 5mL cultures
Insert E.coli colony PCR
PCR mastermix
Reagents
1x
4x
Template DNA
5ul
n/a
Gotaq mix (2x)
12.5ul
50ul
FW primer
1.25ul
5ul
Rev Primer
1.25ul
5ul
H2O
5ul
20ul
PCR cycle (30x) 98ºC   2min 98ºC   30secs 55ºC   30secs 72ºC   1.5mins 72ºC   5mins 4ºC        hold
Colony PCR of the 22 transformations.
Insert yeast colony PCR
PCR mastermix
1x
70x
2x GoTaq
10ul
700ul
10uM FW
1ul
70ul
10uM Rev
1ul
70ul
H2O
5ul
350ul
boiled yeast template
3ul
n/a
PCR cycle (30x) 95ºC   5mins 95ºC   45secs 50ºC   30secs 72ºC   1min/kb 72ºC   10min 4ºC        hold
Every PCR is being run on 1% agarose gel
Number of colony PCRs: 66 colonies from yeast, 27 from 9 rtTA+AFRP, 25 from constitutive promoters
Tues 14/06/17
Since colony PCR of the 22 transformations didn't all work, we are redoing it.  Only 5 strains of the 22 strains were successful (11 CB008, 11 CB008DB).  Checking URA site integration, expecting 940bp bands.
Colony PCRs to redo:
CB008DB
CB008
PRM1
PRM1
YDR124W
YDR124W
PRM3
PRM3
PRM6
PRM6
PCL2
ECM18
SAG1
PRM2
HYM1
CLG1
ASG7
Mastermix
1x
80x
2x GoTaq
10ul
800ul
10uM FW
1ul
80ul
10uM Rev
1ul
80ul
H2O
5ul
400ul
Boiled Yeast template
3ul
n/a
FW/REV primer: "606V2"
Yesterday we made culturesof the AFRP+rtTA (all successful), 2 of each.  Today we miniprepped them to prepare to transform into yeast.  Transform into yeast after linearization/digest with Pme1.
Concentrations of AFRP+rtTA constructs:
ECM18+rtTA #1    308.3 ng/ul ECM18+rtTA #2    231.6 ng/ul PCL2+rtTA #1     162.7 ng/ul PCL2+rtTA #2     247.3 ng/ul PRM6+rtTA #1     248.1 ng/ul PRM6+rtTA #2     327.9 ng/ul ASG7+rtTA #1     351.7 ng/ul ASG7+rtTA #2     593.5 ng/ul HYM1+rtTA #1     221.7 ng/ul HYM1+rtTA #2     413.0 ng/ul PRM2+rtTA #1     324.3 ng/ul PRM2+rtTA #2     339.6 ng/ul PRM3+rtTA #1     351.5 ng/ul PRM3+rtTA #2     316.7 ng/ul CLG1+rtTA #1     289.0 ng/ul CLG1+rtTA #2     276.8 ng/ul YDR124W+rtTA #1  366.0 ng/ul YDR124W+rtTA #2  239.3 ng/ul
Weds 14/06/18
Transforming 9 AFRP+rtTA plasmids into yeast.  Ianto linearized the plasmids #1 and some of #2 depending on the sequences that came back (most won't work)
Preparing for the first flow experiment tomorrow.  Everyone gets a promoter.
AFRP+rtTA owners PRM2 - George ASG7 - Sabrina PCL2 - Eric CLG1 - Derrick YDR124W - Ianto PRM6 - Ianto PRM1 - Robert ECM18 - Robert PRM3 - Eleanor SAG1 -Jessica
Also transformed the 2 AFRP+GFP that didnt work last time HYM1 (08/DB) & PRM3 (DB)
Made overnight cultures of the strain CB008+AFRP to test in on the flow after induction with 5 alpha-factor concentrations (0nM, 10nM, 1000nM, 10,000nM, 100,000nM)
Thurs 14/06/19
Overnight cultures of CB008+AFRP+GFP diluted ~100x (to a final concentration of OD600 0.5-0.1, Saturated overnight cultures should be OD600 of ~7) in SD complete media, and grown for 3 hours, 1000rpm shaker, 30ºC. Growing in 2mL well plates
Induce with Alpha-factor. Stock is in 3mM. Final concentrations are 0, 1nM, 10nM, 100nM, 1000nM.  Alpha-factor cannot be refrozen, so throw leftover away.
Induce for 90mins, but no longer than 120mins
Transfer 250u of each well into a V-bottom 96-well plate containing 10ul of the fixing chemical Cyclohexamide. (thats 4 ul for every 100ul of culture)  Cyclohexamide stops protein production by inhibiting ribosomes.
Run on the flow cytometer.
Flow Cytometer things to remember:
check the sheath fluid box to see if its empty
check the waster container.  If full, dispose and add new bleach
take care not to leave the machine on run. Save sheath fluid by keeping on standby when not in use
find parameters using negative controls. (FSC, SSC, Fluorescense marker)
DO NOT RUN PLATE. RUN WELLS instead. (Run the first well by itself first b/c it needs time to create file folders.) this causes problems.
Run plate only saves data of the first well.
Can rerun through the first well again which would suck up air.  Air no good for the machine.
highlight wells and change settings on the right in the aquisition dashboard screen
mixing speed 180ul/sec
mix 3 times rather than 2
run at 1ul/sec for samples and 0.5ul/sec for negative controls when finding parameters.
make sre uL doesn't exceed samle to prevent sucking up air. (Rule of thumb is to run 50ul less than allotted sample sizes for wells.)
Voltage settings were gotten by running negative controls and adjusting to readouts seen in histogram form of FSC, SSC, FITC
Parameters of todays flow
FSC: 250 SSC: 280 FITC(GFP): 550 B(RFP): 650  Flow rate: 1µL/sec Sample Volume: 200µL Mixing Volume: 100µL Mixing speed: 180µL/sec
Plate map
1   2   3   4   5   6   7   8   9   10  11  12 A   [------NEG------]       [------NEG-------] B   [------PRM2-----]       [------PRM1------] C   [------ASG7-----]       [------EMC18-----] D   [------PCL2-----]       [------PRM3------] E   [------NEG------]       [------SAG1------] F   [------CLG1-----] G   [------YDR124W--] H   [------PRM6-----]
Analyze data.
Fri 14/06/20
Continuation of Weds 14/06/18:
6 of the 9 transformations in yeast failed for the AFRP+rtTA
Those that failed are: ASG7 CLG1 YDR124W HYM1 ECM18 PRM3
This means we need to go back to our E.Coli plates and miniprep colonies for sequencing again, then linearize and transform into yeast.  Yesterday cultures were made of these 6, 5 colonies each except YDR124W which only had 3.  Today we are miniprepping them and sending them for sequencing.
Nanodrop Concentrations: YDR124W.3   241.7 ng/ul YDR124W.4   386.5 ng/ul YDR124W.5   439.6 ng/ul ASG7.3      207.7 ng/ul ASG7.4      319.8 ng/ul ASG7.5      287.2 ng/ul ASG7.6      300.2 ng/ul ASG7.7      338.5 ng/ul CLG1.3      408.0 ng/ul CLG1.4      306.0 ng/ul CLG1.5      277.6 ng/ul CLG1.6      249.8 ng/ul CLG1.7      277.1 ng/ul HYM1.3      440.1 ng/ul HYM1.4      342.4 ng/ul HYM1.5      388.7 ng/ul HYM1.6      226.7 ng/ul HYM1.7      351.2 ng/ul ECM18.3     377.0 ng/ul ECM18.4     426.8 ng/ul ECM18.5     368.8 ng/ul ECM18.6     459.7 ng/ul ECM18.7     372.9 ng/ul PRM3.3      430.4 ng/ul PRM3.4      441.8 ng/ul PRM3.5      361.7 ng/ul PRM3.6      375.6 ng/ul PRM3.7      333.1 ng/ul
Mon 14/06/23
Found out that we were missing the activation domain of rtTA, the rtTA we've been cloning for the past 2-3 weeks has been wrong.
New primers for the full rtTA has been made
Digested more backbone with AFRP+GFP to prepare for correct rtTA insert.
Tues 14/06/24
Today we did Flowjo data analysis with Hyun and Kara with data from our last flow experiment, Thur 14/06/19
Data from usb transfer to save file folder in nanodrop room computer.
Drag all well data into flojo
Set up a gate for all data to select for appropriate size cells. Excludes large cells which are clumped together and small debree or bacteria, which are small.
Find the mean fluorescense and use that data to make a graph in matlab.
Code for my PCL2 graph:
alpha = [0.01 1 10 100 1000] CB008 = [446 438 532 592 590] yGEM5 = [2568 2759 3487 5829 7935]  figure(1) semilogx(alpha,CB008,'--bo') hold all semilogx(alpha,yGEM5,'--rs') xlabel('[Alpha-Factor][nM]') ylabel('MeanGFP[Au]') title('yGEM5 GFP vs [Alpha-Factor]') legend('CB008','yGEM5')
Primers for the correct rtTA with the activation domain came.  Jessica and Sabrina are doing the PCR for rtTA.
Gel photo of backbone digest +promoter+GFP
Already cut plasmid   49ul Fresh plasmid         5ul Cutsmart              6ul Not1                  0.5ul Xho1                  1ul  incubate at 37ºC for 2 hours
After digestion, Ianto & Jessica cut out and gel extracted the ~8kb backbone
Concentrations: (Ianto & Jessica) pHY4+ASG7+GFP   23.77 ng/ul pHY4+CLG1+GFP   16.59 ng/ul pHY4+HMY1+GFP   12.96 ng/ul pHY4+ECM18+GFP  21.39 ng/ul pHY4+PRM3+GFP   15.20 ng/ul  Concentrations: (Derrick) pHY4+YDR124W+GFP   15.46 ng/ul pHY4+SAG1+GFP      12.10 ng/ul pHY4+PRM6+GFP      21.74 ng/ul pHY4+PRM1+GFP      4.186 ng/ul pHY4+PCL2+GFP      6.348 ng/ul pHY4+PRM2+GFP      0.451 ng/ul
I was asked to transform 6 things in DH5α: 1. HY3E (has rtTA) 2. pHY4 (for stock) 3. HY130E (TRP) 4. HY121E1 (URA3) 5. HYS4E2 (HIS3) 6. HY111E2 (LEU2)
Weds 14/06/25
11:30am, made cultures my transformations done yesterday for minipreps.
DH5α Transformations
Colony Count
1. HY3E (has rtTA)
8
2. pHY4 (for stock)
100+
3. HY130E (TRP)
100+
4. HY121E1 (URA3)
100+
5. HYS4E2 (HIS3)
100+
6. HY111E2 (LEU2)
100+
Derrick and Ianto made yeast cultures to test promoters in flow tomorrow.
The yeast was taken from patch plates.
They created streak plates to get individual colonies as well.  We need individual colonies for biological replicates.
Thurs 14/06/26
Yesterday I made cultures of 6 transformations for minipreps. 1. HY3E (has rtTA) 2. pHY4 (for stock) 3. HY130E (TRP) 4. HY121E1 (URA3) 5. HYS4E2 (HIS3) 6. HY111E2 (LEU2) They grew for 22 hours so their concentrations are very high.
Miniprep concentrations (minipreps done by Jessica):
HY3E #1      628.1 ng/ul HY3E #2      875.2 ng/ul HY130E #1    549.3 ng/ul HY130E #2    696.8 ng/ul HY121E1 #1   152.9 ng/ul HY121E1 #2   539.7 ng/ul HYS4E2 #1    424.4 ng/ul HYS4E2 #2    672.7 ng/ul HY111E2 #1   555.3 ng/ul HY111E2 #2   679.0 ng/ul pHY4 #1      487.0 ng/ul pHY4 #2      554.2 ng/ul
Flow cytometry, testing (triplicate) PRM2, ASG7, PLC2, CLG1
Alpha-factor concentrations: (Induce for 90mins)
(0nm, 0.5nm, 1nm, 10nm, 100nm, 1000nm, 3000nm)
Starting concentration is 3mM or 3,000,000nM Make 100x stocks and add 10ul to the 1mL cultures: 0nM 50nM 100nM 1000nM 10,000nM 100,000nM 300,000nM
Plate 1 & 2 Alpha Factor Concentration Map:
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H     Plate 1:     H   G   F   E   D   C   B   A    1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------PRM2-1---------] 5       [--------PRM2-2---------] 6       [--------PRM2-3---------] 7       [--------ASG7-1---------] 8       [--------ASG7-2---------] 9       [--------ASG7-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]  Plate 2:     H    G   F   E   D   C   B   A    1       [--------CLG1-1---------] 2       [--------CLG1-2---------] 3       [--------CLG1-3---------] 4 5 6 7 8 9 10 11 12
Fri 14/06/27
Continuation of CB008+AFRP+GFP FACs.
PRM6+GFP did not grow up yesterday.
ROUND 3 PLATE 1 PROMOTER MAP
H   G   F   E   D   C   B   A    1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------YDR124W-1------] 5       [--------YDR124W-2------] 6       [--------YDR124W-3------] 7       [--------PRM3-1---------] 8       [--------PRM3-2---------] 9       [--------PRM3-3---------] 10      [--------PRM1-1---------] 11      [--------PRM1-2---------] 12      [--------PRM1-3---------]
ROUND 2 PLATE 2 PROMOTER MAP [DID NOT COMPLETE]
H    G   F   E   D   C   B   A    1       [--------ECM18-1--------] 2       [--------ECM18-2--------] 3       [--------ECM18-3--------] 4       [--------SAG1-1---------] 5       [--------SAG1-2---------] 6       [--------SAG1-3---------] 7 8 9 10 11 12
Instructions for starting flow:
Turn on and wait 30mins for the machine to warm up
Re-initialize the HTS, them prime it three times.  You'll get a bubble in the lines if you don't
Run CST!!! Follow the sheet in front of the monitor for more instructions for that part. (CST uses beads to calibrate the laser detection.) add one drop into 250ul of sheath fluid in A1.
Bead LOT ID: use the one that is most current
Load A1-A4 with bleach and B1-B4 with water [flip for the opposite corner] -Cytometer>CST.  Make sure cytometer performance results passed
Run Clean Plate: 1. Click on experiment. Experiment>Open experiment 2. Open clean plate "Daily Clean" - 96 well U-bottom" 3. Be here to see if the cleaning is going correctly, low events (less than 100events/sec for bleach and less than 10events/sec for water) 4. If over the events, run another clean plate 5. View events in aquisition dashboard in view>Acquisition dashboard 6. Can also make a clean plate using HTS>Clean
Mon 14/06/30
Continuation of CB008+AFRP+GFP FACS.  Also testing Constitutive pTEF1 promoters +GFP today
ROUND 4 MAP
H    G   F   E   D   C   B   A    1       [--------ECM18-1--------] 2       [--------ECM18-2--------] 3       [--------ECM18-3--------] 4       [--------SAG1-1---------] 5       [--------SAG1-2---------] 6       [--------SAG1-3---------] 7       [--------CB008-1--------] 8       [--------CB008-2--------] 9       [--------CB008-3--------] 10 11 12
Constitutive Promoter MAP
1   2   3   4   5   6   7   8   9   10  11  12 A   [-CB008-]        B   [-pTEF1-]        C   [-m3----]        D   [-m6----]        E   [-m7----]        F   [-m10---] G    H
Tues 14/07/08
Meeting with Anusuya:
Site directed mutagenesis of ribosome binding sites to fine tune expression of proteins.  Tune down the effect of the promoters
Two promoters back to back to increase expression?
Find out how to differentiate yeast strains when they are co-cultured.
Maybe work more with constitutive rather than the inducible because there might be variable behaviors and its hard to model.
Email ben and ask for code from last year.
Start modeling progress, promoters and outputs. Co-opt code that was written last year
Flow with Bar1 deleted yeast.
Find examples of Tcells that start out with slightly different inputs that strongly converge over time.  Chaotic phenomenon.
look up Chaotic phenomenon in immune cells.
Weds 14/07/09
1) Transforming pTEF1 + rtTA (and mutants m6, m7, m10) into CB008+pTET+GFP and CB008DB+pTET+GFP.  8 Transformations. 2) Creating cultures of 5 CB008DB+AFRP+GFP strains for flow analysis on Thursday.
[Flow with CB008DB AFRP+GFP] https://www.dropbox.com/s/ymmlrcgm3zkoxr0/Flow%20with%20CB008DB%20AFRP%2BGFP.xlsx [pTEF+rtTA progress] https://www.dropbox.com/s/wilm904hbvosox6/pTEF%2BrtTA%20progress.xlsx
Thurs 14/07/10
Flow of CB008DB strains +AFRP+GFP
plate 1
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4       [--------CLG1-1---------] 5       [--------CLG1-2---------] 6       [--------CLG1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]
Plate 2
H   G   F   E   D   C   B   A    1       [--------SAG1-1---------] 2       [--------SAG1-2---------] 3       [--------SAG1-3---------] 4       [--------AGA1-1---------] 5       [--------AGA1-2---------] 6       [--------AGA1-3---------] 7        8        9        10       11       12
Dilutions: 3 hour incubation with 100x dilutions in 2mL well plates. 9:35am -> 12:35pm
Added Alpha-factor.  There are several possible mistakes when inducing. On 2-3 of the wells of the 100nm and 1000nm concentration, alpha factor was on the walls of the plate and may not have been added to the culture fully.  Also, when using the multi-dispenser, the initial 10ul doesnt all fall off the tip, leaving residual liquid hanging. 12:50 -> 2:20pm
Alpha-Factor Concentrations
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H
Fri 14/07/11
CB008DB AFRP+GFP Round 2 Plate 1of1
Glycerol stocks of YDR124W, PRM6, & ASG7 did not grow in SD complete media overnight. The cells may have settled before freezing.
3 Hour incubation plate
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4        5       [--------PRM1-1---------] 6       [--------PRM1-2---------] 7       [--------PRM1-3---------] 8       [--------ECM18-1--------] 9       [--------ECM18-2--------] 10      [--------ECM18-3--------] 11       12
Incubated for an extra 25mins by accident. 9:30am -> 12:55pm
Added alpha-factor 1:05 -> 2:35PM
Flow Plate
H   G   F   E   D   C   B   A    1       [-------CB008DB-1-------] 2       [-------CB008DB-2-------] 3       [-------CB008DB-3-------] 4       [---------PRM1-1--------] 5       [---------PRM1-2--------]    6       [---------PRM1-3--------] 7       [--------ECM18-1--------] 8       [--------ECM18-2--------] 9       [--------ECM18-3--------]    10       11       12
Concentrations of Alpha-Factor are the same.
Mon 14/07/14
Group Meeting at 1pm Until then, everyone was working on their powerpoints for the presentation.
~3:00pm Colony PCR of CB008/CB008DB (+Constitutive Promoters +rtTA) (+pTET +GFP) 3 colonies of each were patched and PCRed onto URA knockout plates.
PCR Reaction: 10ul 2x GoTaq Mix 3ul ddH2O 1ul FW primer 606V2 (primers check for chromosomal integration at the URA2 site) 1ul RV primer 606V2 5ul boiled Template in 20mM NaOH  PCR Protocol: 95ºC 5mins 95ºC 30secs  | 50ºC 30secs  | x30 72ºC 1min    | 4ºC  hold
Running colony PCR on a gel tomorrow.
Tues 14/07/15
1). Running 24 colony PCRs. Expected band size is 940bp.
Gel Photo:
2). 6 Strains worked, the two that didn't are CB008DB+pTEF1+rtTA and CB008DB+m10+rtTA
3). We are redoing colony PCR of those 2 strains that failed, each with 5 colonies each this time. Update: Second colony PCR did not work so we will be doing a third colony PCR
Gel Photo:
4). Redoing Colony PCR for a third time, 5 colonies each again.  This time we included 2 positive controls to check if the PCRs are at fault.  If this doesn't work we will have to redo transformations for these two and we will most likely not run the DB strains on the flow tomorrow. Update: Colony PCR did not work. Gel Photo:
Flow Cytometry with CB008DB +AFRP+GFP 100x dilutions, 3 hour growth: 9:40am -> 12:40pm, alpha-factor induction 12:50 -> 2:20pm
plate 1
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4       [--------ASG7-1---------] 5       [--------ASG7-2---------] 6       [--------ASG7-3---------] 7       [--------YDR124W-1------] 8       [--------YDR124W-2------] 9       [--------YDR124W-3------] 10      [--------PRM6-1---------] 11      [--------PRM6-2---------] 12      [--------PRM6-3---------]
NOTE E3 & F3 had some weird clumping on the bottom of the well and somehow can't be resuspended, even with pipetting.  A2 skipped, human error.
Wed 14/07/16
1). None of 13 Colonies chosen from m10 worked so today George did another 10 colony PCRs.  Again, did not work so we need to retransform m10+rtTA into CB008DB+pTET+GFP. Gel Photo:
2). Flow plate maps today:  Attenuating GFP with 12 concentrations of doxycycline Both strains with Constitutive Promoter+rtTA +pTET+GFP
Ianto made 100x diluted cultures in the morning Incubation 7:25am -> 10:25am Induction 11:17am -> 5:17pm
[Doxycycline]: 100 mg/ml stock or 100,000 ug/ml (in 100ul Aliquots)  Induction time: 6hrs Concentrations needed for 100x:
0 ug/ml 3 ug/ml 6 ug/ml 9 ug/ml 30 ug/ml 60 ug/ml 90 ug/ml 300 ug/ml 600 ug/ml 900 ug/ml 3000 ug/ml 6000 ug/ml
Dox Dilutions:
Stock: 100mg/mL  **1x**           **100x**          Prep 60µg/mL       A: 6mg/mL            (6:100) 60µL of Stock in 940µL of water 30µg/mL       B: 3mg/mL            30µL of Stock in 970µL of water 9µg/mL        C: 900µg/mL          150µL of A in 850µL of water 6µg/mL        D: 600µg/mL          100µL of A in 900µL of water 3µg/mL        E: 300µg/mL          50µL of A in 950µL of water      Can also do 100ul of B in 900ul of water 0.9µg/mL      F: 90µg/mL           100µL of C in 900µL of water 0.6µg/mL      G: 60µg/mL           100µL of D in 900µL of water 0.3µg/mL      H: 30µg/mL           100µL of E in 900µL of water 0.09µg/mL     I: 9µg/mL            100µL of F in 900µL of water 0.06µg/mL     J: 6µg/mL            100µL of G in 900µL of water 0.03µg/mL     K: 3µg/mL            100µL of H in 900µL of water
Doxycycline concentrations plate map:
H   G   F   E   D   C   B   A    1                   [----0 µg/ml----] 2                   [---0.03 µg/ml--] 3                   [---0.06 µg/ml--] 4                   [---0.09 µg/ml--] 5                   [---0.3 µg/ml---] 6                   [---0.6 µg/ml---] 7                   [---0.9 µg/ml---] 8                   [---3.0 µg/ml---] 9                   [---6.0 µg/ml---] 10                  [---9.0 µg/ml---] 11                  [---30 µg/ml----] 12                  [---60 µg/ml----]
pTEF1 promoters
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008 Control]--------------]   B [---------------[CB008 pTET_GFP Control]-----]   C [---------------[CB008 pTET_GFP pTEF1_rtTA]--]   D [---------------[CB008 pTET_GFP pTEF1 m6]----]      E [---------------[CB008 pTET_GFP pTEF1 m7]----]     F [---------------[CB008 pTET_GFP pTEF1 m10]---]   G H    1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB Control]------------]   B [---------------[CB008DB pTET_GFP Control]---]   C [---------------[CB008DB pTET_GFP pTEF1 1of2 ]  D [---------------[CB008DB pTET_GFP pTEF1 2of2 ] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G  H
All wells ran its 10k events threshhold within seconds due to the amount of time they've been allowed to grow, 9 hrs.
3). Making cultures of CB008DB+pTET+GFP for tranformations of m10+rtTA tomorrow.
4). Yesterday's flow plate was contaminated and our data is very weird. (refer to Ianto's lab notebook on the same date) We will be redoing that experiment so today we should make more cultures for flow tomorrow.
Thu 14/07/17
Anusuya meeting: We should have everyone create a labnotebook/blog of what they do everyday so that we can keep track of everything. Background of T-cell signalling. Need good examples of what naturally happens.  We need a specific thing we want to focus on and model for our project.
Read:  1). interaction between T-reg cells and T-helper cells.  How T-reg with macrophages, their interaction represent some negative feedback.  Macrophage somehow suppresses T-regs. 2). T-reg, dentridic cells.  relates to neural.  Stanford 2010, interaction btwn T-reg, T-helper, Th1-th2. 3). Topic of Neural information? microglia interaction? Might play a role in recruiting and regulating T-cells when there is injury to the brain.
1). Read some wiki and review papers, dont need data papers be able to understand whats going on with these interactions so that we have many examples where interactions play important roles.
2). Look from a non-immune viewpoint.  There are interactions beyond the immune system so look into those and come up with examples. Perhaps we can build on the work from UCSF iGEM 2010.
3). COME UP WITH EXAMPLES, need an elavator pitch.  Whats still unknown about these cell types? and how does this model we are making with yeast help us to understand whats going on.  Specific termonology.  Have some slides with these examples set up for the next meeting.  Layout a rational.
4). Look up alpha factor diffusion rates.  Delegate tasks for all of these points.
5). Have a point person for the medal requirements.  Need someone to keep track of everything.
6). Maybe dedicate a 10min session to brief everyone about what has been done for the day and the end of every day.  Have it written day and keep it in a text file for people to access.
7). Folders for all our files should be put into folders for the wiki.  Update them!  Intellecutal material, protocols, etc.
8). Create a library of motifs we've constructed and explain how its valuable.  Find examples of them where they exist in the immune system.
Type 2 restriction sites. AAR1  BSA1
Modeling: Write code for both strains and their find out their output of alpha-factor.  Then have both sets of code and add them together to see RFP output when those two cells are together.
factors to account for the model: diffusion constants, degredation rate, cell density (later)
Overnight cultures of CB008DB+pTET+GFP grew well. Continuing to tranform m10+rtTA. Diluted 20x at 9:45am -> 11:45am
Fri 14/07/18
Transferred labnotebook to this textfile.
Looked up examples related to our project
Prepared for monday's group meeting
Helped Robert with what he should be doing in the lab
Mon 14/07/21
100+ Colonies grew for our m10+rtTA transformations in CB008DB+pTET+GFP.  We are now colony PCRing 10 colonies.
2-4pm group meeting
4 out of the 10 colony PCRs worked! m10+rtTA is now available for flow testing.
Everyone is assigned an article to read for an example related to our iGEM project. They were told to make slides of the signalling pathway if they could.
Tues 14/07/22
Made a streak plate of CB008DB+m10+rtTA+pTET+GFP from the patch plate.
100x dilutions done at 9:20am 9:20am -> 12:30pm Alpha-factor induction 12:47pm -> 2:17pm
Plate 1:
H      G   F   E   D   C   B   A    1  pTEF1-1 [--------CB008DB-1------] 2  pTEF1-2 [--------CB008DB-2------] 3  pTEF1-3 [--------CB008DB-3------] 4  M7-1    [--------ASG7-1---------] 5  M7-2    [--------ASG7-2---------] 6  M7-3    [--------ASG7-3---------] 7          [--------YDR124W-1------] 8          [--------YDR124W-2------] 9          [--------YDR124W-3------] 10         [--------PRM6-1---------] 11         [--------PRM6-2---------] 12         [--------PRM6-3---------]
Plate 2:
H   G   F   E   D   C   B   A    1       [--------AGA1-1---------] 2       [--------AGA1-2---------] 3       [--------AGA1-3---------] 4       [--------CLG1-1---------] did not grow as much as the others 5       [--------CLG1-2---------] 20ul instead of 10ul for CLG1 6       [--------CLG1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]
Plate 3:
H    G   F   E   D   C   B   A    1        [--------ECM18-1--------] 2        [--------ECM18-2--------] 3        [--------ECM18-3--------] 4        [--------SAG1-1---------] 5        [--------SAG1-2---------] 6        [--------SAG1-3---------] 7         8         there is no PRM3 made. They're in glycerol stock. 9         10       [--------PRM1-1---------] 11       [--------PRM1-2---------] 12       [--------PRM1-3---------]
Weds 14/07/23
Came up with more cards for Cards against humanity_UCSF iGEM
Working on the CRISPR paper, discussion.  Research for what has been done with CRISPR delivery and how our work is valuable.
Thurs 14/07/24
Working on the CRISPR paper and the Explo Powerpoint.
Made cultures for glycerol stocks of 8 things:
1. CB008 +pTEF1+rtTA +pTET+GFP +pTET+mfalpha 2. CB008 +m6+rtTA +pTET+GFP +pTET+mfalpha 3. CB008 +m7+rtTA +pTET+GFP +pTET+mfalpha 4. CB008 +m10+rtTA +pTET+GFP +pTET+mfalpha 5. CB008DB +m6+rtTA +pTET+GFP +pTET+mfalpha 6. CB008DB +m7+rtTA +pTET+GFP +pTET+mfalpha 7. CB008DB +pTEF1+rtTA +pTET+GFP 8. CB008DB +m10+rtTA +pTET+GFP
7 & 8 need +pTET+mfalpha
In the coming days we will also transfrom everything with pAGA+mCherry.
Fri 14/07/25
Stuff to do other than lab:
CRISPR paper. Read, Write
Powerpoint for Explo due today
Research for examples relating to the project
Cards Against Humanity: UCSF_UCB iGEM 2014 (illustrator/InDesign)
Make glycerol stocks of:
1:50 dilutions of each and grow for 3 hours in 5ml cultures to .4 ~ .5 OD600.  Then follow glycerol stock protocol.
9am -> 12pm growth to log phase
1. CB008 +pTEF1+rtTA +pTET+GFP +pTET+mfalpha 2. CB008 +m6+rtTA +pTET+GFP +pTET+mfalpha 3. CB008 +m7+rtTA +pTET+GFP +pTET+mfalpha 4. CB008 +m10+rtTA +pTET+GFP +pTET+mfalpha 5. CB008DB +m6+rtTA +pTET+GFP +pTET+mfalpha 6. CB008DB +m7+rtTA +pTET+GFP +pTET+mfalpha 7. CB008DB +pTEF1+rtTA +pTET+GFP 8. CB008DB +m10+rtTA +pTET+GFP
1:20 dilutions for transformations after 3 hours growth in 5ml cultures.  Transforming pTET+mfalpha into #7,8 and pAGA+mCherry into #1-6.
9am -> 12pm growth to log phase
1. CB008 +pTEF1+rtTA +pTET+GFP +pTET+mfalpha 2. CB008 +m6+rtTA +pTET+GFP +pTET+mfalpha 3. CB008 +m7+rtTA +pTET+GFP +pTET+mfalpha 4. CB008 +m10+rtTA +pTET+GFP +pTET+mfalpha 5. CB008DB +m6+rtTA +pTET+GFP +pTET+mfalpha 6. CB008DB +m7+rtTA +pTET+GFP +pTET+mfalpha 7. CB008DB +pTEF1+rtTA +pTET+GFP 8. CB008DB +m10+rtTA +pTET+GFP
~23 primers are coming in to clone AFRP upstream of BAR1 and Ste2 separately.
Note:  1. PCL2+mCherry in HY130 backbone is being constructed by Robert.  We will probably make additional strains with this in place of pAGA+mCherry. 2. Jeffrey is working on getting m3+rtTA into CB008DB +pTET+GFP.
Mon 14/07/28
Last friday, Eleanor transformed mCherry into 6 strains with Constitutive Promoter, GFP, Alpha factor.  Today I am colony PCRing them along with 3 other strains that have other things being integrated: 1) +rtTA for the m3s 2) +mfalpha for DB pTEF1 & m10.
Next steps: 1. Need to retransform DB m10 and CB008 m7 because they have no colonies. 2. Need to make glycerol stocks of the ones that worked for Colony PCR 3. For tomorrow, make cultures for tomorrow's flow (9 strains + control)
Auxotrophic marker Colony PCR primer names: His: FW RA145(#87) & REV RA146(#89) Trp: FW RA145(#87) & REV
Today I also worked on the Cards Against iGEM inDesign layout.
Tues 14/07/29
Flow scheduled for 5-8pm. Doing Consti+rtTA+pTET+GFP Dox run. No alpha, no RFP. Runnin all 5 DB strains today (and 4-5 CB008 strains tomorrow pending successful colony PCR of CB008 m3.)
Dilutions start 7:16am -> 10:16am Induction 10:45 -> 4:45pm
Dox concentrations are the same as last time.
All plates identical assortment:
Plate 1:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------] C [---------------[CB008DB pTET_GFP pTEF1]-----] D [---------------[CB008DB pTET_GFP pTEF1 m3]--] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G [---------------[CB008DB pTET_GFP pTEF1 m10]-] H
Plate 2:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------]   C [---------------[CB008DB pTET_GFP pTEF1]-----]  D [---------------[CB008DB pTET_GFP pTEF1 m3]--] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G [---------------[CB008DB pTET_GFP pTEF1 m10]-] H
Plate 3:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------]   C [---------------[CB008DB pTET_GFP pTEF1]-----] entire lane 10 is induced D [---------------[CB008DB pTET_GFP pTEF1 m3]--] with wrong concentration. E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G [---------------[CB008DB pTET_GFP pTEF1 m10]-] H
time course flow allows us to create a stocastic model of our products: rtTA, GFP, alpha-factor, RFP.
Colony PCR2 redo from yesterday because only 1 worked
Constitutive Promoter Constructs progress:
Today George and I are retransforming:
CB008DB+pTET+GFP+m3+rtTA[+pTET+mfalpha]
CB008+pTET+GFP+m7+rtTA+pTET+mfalpha[+pAGA+mCherry]
CB008+pTET+GFP+m10+rtTA+pTET+mfalpha[+pAGA+mCherry]
The salmon sperm boiled down to about 5ul for all 3 transformations. It also seemed a little viscous.
CB008+pTET+GFP+m10+rtTA+pTET+mfalpha[+pAGA+mCherry] already exists so our transformation of this (#3) was pointless.  What we meant to do was CB008DB+m10+rtTA+pTET+GFP[+pTET+mfalpha]
After all our colony PCRs for the day, this is the update of the excell progress sheet:
Made cultures of all 10 things on the Consti list. 9 for transformations and 1 for glycerol stock.
Made cultures for tomorrows flow.
Weds 14/07/30
None of the cultures for today's flow grew. Only the controls grew. Looking back at the URA knock out patch plates that we inoculated from, they didn't grow very well. I think they moreso had a hard time growing than it not being correctly transformed because they came back positive in the colony PCRs (14/7/15). I transferred them onto a YPD patch plate to see if they would grow.
Derrick and I plated on YPD CB008/DB strains with pTET+GFP+consti+rtTA because the current patch plate on URA of CB008s does not have good growth.  Cultures made from this patch plate did not grow either so holding out flow back.
Jeffrey and George are retransforming the ones that failed.
Thurs 14/07/31
Our steak plates of CB008/DB +pTET+GFP+Consti+rtTA over grew only on CB008DB.  All the CB008s did not grow and have no sign of growth.  This may mean that we will have to retransform consti+rtTA into our CB008 strains.  So far, our glycerol stocks have not been working well for us; we don't know what the problem is.
Worked on the Explo powerpoint today.
Fri 14/08/01
Made streak plates with Derrick of everything we have so that we have fresh/organized cultures.
Worked on Cards Against iGEM InDesign. Printed and cut out our first deck for testing.
Mon - Weds 14/08/04-06
Out on vacation.
Thurs 14/08/07
Tomorrow we are running flow. Testing the final circuit with 5 strains. triplicate, 5 Dox concentrations (Shaved off 7 of the top concentrations of Dox), 4 time points.
Concentrations:  0 ug/ml 0.03 ug/ml 0.06 ug/ml 0.09 ug/ml 0.6 ug/ml
Exploratorium After Dark event 6pm-10pm
Fri 14/08/08
OD600 of .1
Flow today with CB008 pTEF1+rtTA pTET+GFP pTET+MFalpha pAGA1+mCherry. Signed up from 4:30pm -> 9pm
Diluted 1:200 at 7:45am and grown for 2 hours
Time points with [Dox] induction: 0, 1.5hr, 3hr, 5hr
Plate map for all time points (4 plates):
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 m6-1]----]       [CB008 m10-3]----] B [CB008 m6-2]----]       [CB008DB m7-1]---] C [CB008 m6-3]----]       [CB008DB m7-2]---] D [CB008 m7-1]----]       [CB008DB m7-3]---] E [CB008 m7-2]----]  F [CB008 m7-3]----]            G [CB008 m10-1]---]  H [CB008 m10-2]---]
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12
Flow notes: The first plate for time 0 contains the concentrations of doxycycline so we saw a bimodal RFP response.  An explanation for this can be that the addition of doxycline had driven expression of mfalpha -> pAGA+mcherry before the cyclohexamide fixed the cells.
Finished flow at 9:40pm
Mon 14/08/11
Group Meeting at 10am -> 12pm
Flow with AFRP+rtTA +pTET+GFP 7 strains (Mon-Tues Flow), 7 alpha concentrations. 7 AFRP promoters+rtTA sucessfully transformed into CB008.
The AFRP strains that have successfully been transformed (7):
CB008 +CLG1+rtTA +pTET+GFP CB008 +YDR124W+rtTA +pTET+GFP CB008 +ASG7+rtTA +pTET+GFP CB008 +HYM1+rtTA +pTET+GFP CB008 +PRM1+rtTA +pTET+GFP CB008 +PRM2+rtTA +pTET+GFP CB008 +PRM3+rtTA +pTET+GFP
HYM1 plate, 0.6ug/ml & 0.06ug/ml concentrations were switched.
1:200 dilution grown for 2hrs 8:00am -> 10:00am induced at 10:30am Fix at 3:30pm Flow signed up from 4:30pm -> 9:00pm Finished flow at 8:40pm
Doxycycline concentrations plate map (12):
H   G   F   E   D   C   B   A    1          [--------0 µg/ml----------] 2          [--------0.03 µg/ml-------] 3          [--------0.06 µg/ml-------] 4          [--------0.09 µg/ml-------] 5 [--------0.3 µg/ml--------] 6          [--------0.6 µg/ml--------] 7          [--------0.9 µg/ml--------] 8          [--------3.0 µg/ml--------] 9          [--------6.0 µg/ml--------] 10         [--------9.0 µg/ml--------] 11         [--------30 µg/ml---------] 12         [--------60 µg/ml---------]
Alpha-Factor Concentrations (7):
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}------------] B   [-------------------{0.5 nM-----------] C   [-------------------{1 nM}------------] D   [-------------------{10 nM}-----------] E   [-------------------{100 nM}----------] F   [-------------------{1000 nM}---------] G   [-------------------{3000 nM}---------] H
Plates:
Plate #1 (ASG7) Plate #2 (CLG1) Plate #3 (HYM1) Plate #4 (YDR124W)
Play tested the printed Cards Against iGEM. Eliminated about 21 black cards. We have 24 good black cards, Goal 40-50.µ
Tues 14/08/12
Flow with the other 3 CB008 +AFRP+rtTA +pTET+GFP (PRM1, PRM2, PRM3) constructs. Signed up from 4pm -> 8pm.
1:200 dilution at 8:11am -> 10:11am
Same Dox and alpha-factor concentrations as yesterday.
Plate#1 (PRM1) Plate#2 (PRM2) Plate#3 (PRM3)
Helped Jessica with her progress with AFRP+rtTA constructs. Streaked out the 5 successful transformations of pTET_mfalpha for the CB008 (12-Aug) and made cultures of those for tranformations (+pAGA+mcherry/+pPCL2+mcherry/+pPCL2+BFP).
AFRP+rtTA progress chart:
Weds 14/08/13
1:200 dilutions, 2hr growth 7:50am -> 9:50am
Time points with [Dox] induction: 0, 1.5hr, 3hr, 5hr
Making Dox dilutions, follow this:
Stock: 100mg/mL  **1x**           **100x**          Prep 60µg/mL       A: 6mg/mL            (6:100) 60µL of Stock in 940µL of water 30µg/mL       B: 3mg/mL            30µL of Stock in 970µL of water 9µg/mL        C: 900µg/mL          150µL of A in 850µL of water 6µg/mL        D: 600µg/mL          100µL of A in 900µL of water 3µg/mL        E: 300µg/mL          50µL of A in 950µL of water      Can also do 100ul of B in 900ul of water 0.9µg/mL      F: 90µg/mL           100µL of C in 900µL of water 0.6µg/mL      G: 60µg/mL           100µL of D in 900µL of water 0.3µg/mL      H: 30µg/mL           100µL of E in 900µL of water 0.09µg/mL     I: 9µg/mL            100µL of F in 900µL of water 0.06µg/mL     J: 6µg/mL            100µL of G in 900µL of water 0.03µg/mL     K: 3µg/mL            100µL of H in 900µL of water
Plate map for all time points (4 plates):
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 pTEF1-1]-]       [CB008DB m3-3]---]  B [CB008 pTEF1-2]-]       [CB008DB m6-1]---] C [CB008 pTEF1-3]-]       [CB008DB m6-2]---] D [CB008DB pTEF1-1]       [CB008DB m6-3]---] E [CB008DB pTEF1-2]   F [CB008DB pTEF1-3]            G [CB008DB m3-1]--]  H [CB008DB m3-2]--]
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12
Deleted and exported files on the flow to prevent data crowding. (All files exported onto iGEM2 usb) Note: 140627 Alpha Factor Responsive Promoter 2 of 2 gives an error when exporting. "There are no tubes with data to export". Data may have been deleted? idk
Thurs 14/08/14
1:200 dilutions at 7:50 -> 9:50am
Time points with [Dox] induction: 0, 1.5hr, 3hr, 5hr, 8hr
Plate map for all time points (5 plates):
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 pTEF1-1]-]       [CB008 m7-3]-----] B [CB008 pTEF1-2]-]       [CB008 m10-1]----] C [CB008 pTEF1-3]-]       [CB008 m10-2]----] D [CB008 m6-1]----]       [CB008 m10-3]----] E [CB008 m6-2]----]  F [CB008 m6-3]----]            G [CB008 m7-1]----]  H [CB008 m7-2]----]
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7                       [--0 µg/ml--] 8                       [-0.03 µg/ml] 9                       [-0.06 µg/ml] 10                      [-0.09 µg/ml] 11                      [-0.6 µg/ml-] 12
In the 2ml culture plate, we noticed what seems to look like debri or contamination on the bottom of the wells from rows A,B,C.  We tried to make the time point plates by pipetting everything singly.
After taking a look at the 0 time point plate through the flow, we can see that the size of molecules running through are not yeast but bacterial contamination.  We need to totally redo this experiment again because this contamination gives us inaccurate data.
Yesterday, the 0 time point plate was not run all the way (only 10ul was run and there were not enough events).  Derrick saved the plate and I will run 100uls of each of those wells. The wells are G,H (1-5), and A,B,C,D (7-11).
2pm -> 4pm Practiced presenting poster with the group and Anusuya.
Fri 14/08/15
Last official day of iGEM
All day event.  Driving down to Santa Cruz for the Symposium held by UC Santa Cruz.
Sun 14/08/24
Made cultures of Contsti+full contruct + pPCL2+mCherry for Monday's flow (4).
pTEF1, m6, m7, m10
Retrieved from " http://2014.igem.org/Team:UCSF_UCB/eric "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
