<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCSF_UCB_sabrina"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCSF_UCB/sabrina">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCSF_UCB/sabrina&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF_UCB/sabrina&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF_UCB/sabrina&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCSF_UCB/sabrina" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCSF UCB/sabrina</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1>Sabrina's Lab Notebook</H1><H2>06/09/14</H2><H3>PCR for Constitutive Promoters - pTEF1 (M3)</H3><TABLE><THEAD><TR><TH>Materials</TH><TH>1x reaction</TH><TH>4.5x Master Mix</TH></TR></THEAD><TBODY><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD><TD>45 µl</TD></TR><TR><TD>dNTP's (10 mM)</TD><TD>1 µl</TD><TD>4.5 µl</TD></TR><TR><TD>Forward Primer (10µm)</TD><TD>2.5 µl</TD><TD>11.25 µl</TD></TR><TR><TD>Reverse Primer (10µm)</TD><TD>2.5 µl</TD><TD>11.25 µl</TD></TR><TR><TD>*Template DNA</TD><TD>0.3 µl</TD><TD>1.35 µl</TD></TR><TR><TD>Phusion Polymerase</TD><TD>0.5 µl</TD><TD>2.25 µl</TD></TR><TR><TD>Water</TD><TD>33.2 µl</TD><TD>149.4 µl</TD></TR><TR><TD>Total</TD><TD>50 µl</TD><TD>225 µl</TD></TR></TBODY></TABLE><P>(Note: Add enzymes last)</P><H5>PCR Reaction Procedures:</H5><OL><LI>Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom. </LI><LI>Pipette 50ul from the Master Mix into four labled PCR tubes</LI><LI><P>Put in the thermocycler for the following cycles:</P><PRE><CODE>Initial Denaturation   | 98° C | 30s 
  35 Cycles of: 
      Denaturation | 98° C | 10s
      Annealing    | 55° C | 20s
      Extension    | 72° C | 30s
Final Extension    | 72° C | 5m
Hold               | 4°  C | Forever
</CODE></PRE></LI><LI><P>Keep samples for gel extraction on the following day.</P></LI></OL><H2>06/10/14</H2><H3>Loading the Gel</H3><P>*Add 5ul of Loading Dye (10X Blue Juice) to samples.</P><P>Gel Map:</P><PRE><CODE>1  2  3  4  5  6  7  8  9  10  11  12  13  14  15
   [----M6---][----M7----] [-----M3-----]

1  2  3  4  5  6  7  8  9  10  11  12  13  14  15
   [----TEF1----]   [-M10]
</CODE></PRE><P>Both lane #1 are 100 bp ladder</P><P>*George had less to PCR due to a spill accident when spinning tubes</P><H4>Gel Extraction Procedure:</H4><OL><LI>Cut Gel</LI><LI>Weigh it in a colorless tube</LI><LI>Add 3 volumes Buffer QG to 1 volime Gel (100mg ~ 100µl)</LI><LI>Incubate @ 50° C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)</LI><LI>Add 1 gel volume isopropanal to the sample and mix</LI><LI>Place a QlAquick soin column in a provided 2ml collection tube</LI><LI>Place sample in column &amp; spin for 1 min -&gt; discard flow through</LI><LI>To wash add 0.75 ml Buffer PE to column &amp; centrifue for 1 min, then dry spin</LI><LI>Place column in 1.5 ml tube</LI><LI>Add 30µl H2O &amp; centrifuge for 1 min.</LI></OL><H4>Restriction Enzyme Digest with ApaI (Backbone Digestion):</H4><OL><LI>40ul of Gel Extraction DNA</LI><LI>5ul of Cutsmart Buffer</LI><LI>0.5ul of ApaI</LI></OL><P>*Digest at room temperature overnight</P><H2>06/11/14</H2><H3>PCR Purification and Ligation</H3><H4>Digest with XhoI</H4><OL><LI>Add 0.5ul of XhoI Enzyme (in Enzyme Box) to ApaI Digest from 06/10/14</LI><LI>Incubate in the 37°C shaker for ~2 hours.</LI></OL><H4>PCR Purification</H4><OL><LI>5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample</LI><LI>To bind DNA, apply sample to column and spin for 1 min</LI><LI>Discard flow through </LI><LI>To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min</LI><LI>Discard flow through and place column back in the same tube</LI><LI>Cetrifuge for an addtitional minute (Dry Spin)</LI><LI>Place column in clean 1.5 ml tube</LI><LI>Add 30µl water to column, let stand for 1 min and centrifuge for 1 min </LI></OL><H4>Ligation (pSV606 and pTEF1-M3)</H4><P>1:3 Backbone to Insert</P><H5>Concentration of M3 Digest = 127.1 ng/ul</H5><P>Materials:</P><TABLE><THEAD><TR><TH>Reagents</TH><TH>1x</TH></TR></THEAD><TBODY><TR><TD>10x ligase buffer</TD><TD>1.0µl</TD></TR><TR><TD>DNA Backbone (PSV606)</TD><TD>0.2µl</TD></TR><TR><TD>Insert</TD><TD>0.2µl</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>0.5µl</TD></TR><TR><TD>H2O</TD><TD>8.1µl</TD></TR><TR><TD>Total</TD><TD>10 µl</TD></TR></TBODY></TABLE><P>Keep at room temperature for at least 2 hours.</P><H4>Transformation</H4><TABLE><THEAD><TR><TH>Materials</TH><TH>1x</TH></TR></THEAD><TBODY><TR><TD>Ligation (pSV606 and M3)</TD><TD>10µl</TD></TR><TR><TD>E. Coli Competent Cells</TD><TD>50ul</TD></TR></TBODY></TABLE><OL><LI>Mix both together and keep on ice for 30 minutes</LI><LI>42°C Heat shock for 45 seconds</LI><LI>Put immediately on ice for at least 2 minutes</LI><LI>Add 250 ul of SOC media, shake for 1 hr at 37°C.</LI><LI>Plate on selective media and store at 37°C. </LI></OL><H2>06/12/14</H2><H3>Re-Ligation and Tranformation of M3</H3><H5>Note: Most of our plates did not form colonies, with the exceptions of pTEF1-M10 and pTEF1-M3.</H5><UL><LI>Due to this, we will all have to re-ligate using higher concentration of the pSV606 backbone.</LI><LI>Kara belives that the failure in our transformation was due to mediocre competent cells. We will be using better competent cells.</LI><LI>Forgot to make a Negative control. Make sure to make one next time.</LI></UL><H3>Ligation #2:</H3><P>Materials:</P><TABLE><THEAD><TR><TH>Reagents</TH><TH>1x</TH></TR></THEAD><TBODY><TR><TD>10x ligase buffer</TD><TD>1.0µl</TD></TR><TR><TD>DNA Backbone (PSV606)</TD><TD>0.4µl</TD></TR><TR><TD>Insert</TD><TD>0.2µl</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>0.5µl</TD></TR><TR><TD>H2O</TD><TD>7.9µl</TD></TR><TR><TD>Total</TD><TD>10 µl</TD></TR></TBODY></TABLE><P>-Added 0.2ul more Backbone.</P><P>Keep at room temperature for at least 2 hours.</P><H4>Transformation</H4><TABLE><THEAD><TR><TH>Materials</TH><TH>1x</TH></TR></THEAD><TBODY><TR><TD>Ligation (pSV606 and M3)</TD><TD>10µl</TD></TR><TR><TD>E. Coli Competent Cells</TD><TD>50ul</TD></TR></TBODY></TABLE><OL><LI>Mix both together and keep on ice for 30 minutes</LI><LI>42°C Heat shock for 45 seconds</LI><LI>Put immediately on ice for at least 2 minutes</LI><LI>Add 250 ul of SOC media, shake for 1 hr at 37°C.</LI><LI>Plate on selective media and store at 37°C. </LI></OL><H2>06/13/14</H2><H3>Yeast Transformation (CB008 + Inducible Promoters)</H3><P>Assisted Ianto and others in Transforming Inducible Promoters into Yeast.</P><PRE><CODE>| Reagents                  |
| ------------------------- |
| YPD                       |
| 1 M LiOAc                 |
| 10X TE pH 7.5             |
| 1X TE pH 7.5, 0.1 M LiOAc |
| 50% PEG 3350              |
| DMSO                      |
| Salmon Sperm DNA (ssDNA)  | 
</CODE></PRE><UL><LI>PEG is viscous, so pipette slowly to prevent air bubbles from forming.</LI><LI>boil ssDNA aliquots for 10 min, then ice down for 10 min before use.</LI></UL><P>Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C</P><OL><LI>Set up digest to linearize DNA </LI><LI>Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C</LI><LI>Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)</LI><LI>Harvest cells in centrifuge - 3000 rpm, 2-5 min</LI><LI>Wash with 1 ml 0.1 M LiOAin TE</LI><LI>Pellet cells - 3000 rpm , 2-5 min</LI><LI>Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation</LI><LI>to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA</LI><LI>Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc)  <EM>Optional</EM>: Add 75 µl DMSO <EM>definitely did</EM></LI><LI>Vortex</LI><LI>Incubate 42° C for 30m &amp; begin drying plates</LI><LI>Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)</LI><LI>Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD</LI><LI>Plate on selective media</LI><LI>Incubate 1-3 days</LI></OL><H2>06/16/14</H2><H3>Colony PCR for Screening E.Coli Procedure:</H3><OL><LI>Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 25ul of water.Do this for 5 colonies. Use 5ul for PCR reaction below. </LI><LI><P>Set up PCR Reaction below:</P><TABLE><THEAD><TR><TH>Reagents</TH><TH>1X</TH><TH>6X</TH></TR></THEAD><TBODY><TR><TD>2X GoTaq Green PCR Master Mix</TD><TD>10 µl</TD><TD>60 µl</TD></TR><TR><TD>10 µM Forward primer</TD><TD>1 µl</TD><TD>6 µl</TD></TR><TR><TD>10 µM Reverse primer</TD><TD>1 µl</TD><TD>6 µl</TD></TR><TR><TD>Water</TD><TD>3 µl</TD><TD>18 µl</TD></TR><TR><TD>Bacterial cells (template)</TD><TD>5 µl</TD><TD>-----</TD></TR><TR><TD>Total</TD><TD>90 ul</TD></TR></TBODY></TABLE></LI><LI><P>Add 15 ul of Master Mix into each tube.</P><PRE><CODE>    Initial Denaturation   | 95° C | 5 min 
  35 Cycles of: 
      Denaturation | 95° C | 45s
      Annealing    | 55° C | 30s
      Extension    | 72° C | 1min per kb
Final Extension    | 72° C | 10m
Hold               |  4°C  | Forever
</CODE></PRE></LI></OL><UL><LI><P>Load PCR products on a gel. (5ul) (GoTaq already has loading dye)</P></LI><LI><P>For all positive bands on the gel, add rest of bacterial cells from PCR tubes to 5ml LB and grow overnight at 37°C for miniprep on the following day. </P></LI></UL><H5>Result of E.Coli Colony PCR (Constitutive Promoters): All lanes worked.</H5><UL><LI>Grew 2 out of 5 Successful Colony PCR E.Coli Colonies in liquid culture. 

</LI><LI> Unfortunately, the rack which contained our cultures was not secured and caused our cultures to fall down and possibly be contaminated. </LI><LI> We re-inocculated the colonies due to possible contamination.</LI></UL><H3>Colony PCR for Yeast (Inducible Promoters CB008 and CB008DB) Procedure:</H3><UL><LI>Boil Yeast at 95°C for 1 hour.</LI></UL><OL><LI>Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 15ul of NaOH. Take 3 colonies from each plate. Use 5ul for PCR reaction below. </LI><LI><P>Set up PCR reaction below: </P><TABLE><THEAD><TR><TH>Reagents</TH><TH>1X</TH><TH>67X</TH></TR></THEAD><TBODY><TR><TD>2X GoTaq Green PCR Master Mix</TD><TD>10 µl</TD><TD>670 µl</TD></TR><TR><TD>10 µM Forward primer</TD><TD>1 µl</TD><TD>67 µl</TD></TR><TR><TD>10 µM Reverse primer</TD><TD>1 µl</TD><TD>67 µl</TD></TR><TR><TD>Water</TD><TD>3 µl</TD><TD>201 µl</TD></TR><TR><TD>cells (template)</TD><TD>5 µl</TD><TD>-------</TD></TR><TR><TD>Total</TD><TD>1005 ul</TD></TR></TBODY></TABLE></LI><LI><P>Add 15 ul of Master Mix into each tube.</P><PRE><CODE>Initial Denaturation    | 95°C | 5 min 
  35 Cycles of: 
      Denaturation | 95°C | 45s
      Annealing    | 55°C | 30s
      Extension    | 72°C | 1min per kb
Final Extension    | 72°C | 10m
Hold               |  4°C | Forever
</CODE></PRE></LI></OL><UL><LI><P>Load PCR products on a gel. (5ul) (GoTaq already has loading dye)</P></LI><LI><P>Results: only 4 positive lanes </P></LI><LI>Possible Explanation: Not enough DNA in PCR tubes.</LI></UL><H2>06/17/14</H2><H3>Miniprep of Constitutive Promoters</H3><H4>Miniprep Precedure:</H4><UL><LI>Obtain E.Coli Cultures from 06/16/14 in 37°C  Incubation Room.</LI></UL><OL><LI>Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.</LI><LI>Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.</LI><LI>Add 350 µl Buffer N3 and invert immediately 4-6 times.</LI><LI>Centrifuge for 10 min at 13,000 rpm.</LI><LI>Add supernatant to QIAprep spin column.</LI><LI>Centrifuge for 30-60s - Discard flow through.</LI><LI>Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.</LI><LI>Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s</LI><LI>Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.</LI><LI>Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.</LI></OL><UL><LI>Sent miniprep into sequencing. Results came out positive. Constitutive promoters successfully transformed into plasmids.</LI></UL><H2>06/18/14</H2><H3>Cultures Set up for Yeast Tranformation</H3><P>Made cultures of CB008 and CB008DB for each of our constitutive promoters. </P><H2>06/20/14</H2><H3>Linearization of Plasmids with Constitutive Promoters - PSV606</H3><H4>Linearization Materials:</H4><PRE><CODE>10 µl DNA (miniprep in my box in -20°C Fridge)
5 µl Cutsmart Buffer
1 µl PME1
34 µl H2O
</CODE></PRE><UL><LI>Incubate at 37 ° C for 2 hours</LI></UL><H3>Transformation of Constitutive Promoters into Yeast (CB008 and CB008DB)</H3><PRE><CODE>| Reagents                  |
| ------------------------- |
| YPD                       |
| 1 M LiOAc                 |
| 10X TE pH 7.5             |
| 1X TE pH 7.5, 0.1 M LiOAc |
| 50% PEG 3350              |
| DMSO                      |
| Salmon Sperm DNA (ssDNA)  | 
</CODE></PRE><UL><LI>PEG is viscous, so pipette slowly to prevent air bubbles from forming.</LI><LI>boil ssDNA aliquots for 10 min, then ice down for 10 min before use.</LI><LI>Make sure to add reagents in order!</LI></UL><P>Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C</P><OL><LI>Set up digest to linearize DNA (see procedure above)</LI><LI>Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C</LI><LI>Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)</LI><LI>Harvest cells in centrifuge - 3000 rpm, 2-5 min</LI><LI>Wash with 1 ml 0.1 M LiOAin TE</LI><LI>Pellet cells - 3000 rpm , 2-5 min</LI><LI>Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation</LI><LI>to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA</LI><LI>Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc)  <EM>Optional</EM>: Add 75 µl DMSO <EM>definitely did</EM></LI><LI>Vortex</LI><LI>Incubate 42° C for 30m &amp; begin drying plates</LI><LI>Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)</LI><LI>Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD</LI><LI>Plate on selective media</LI><LI>Incubate 1-3 days</LI></OL><H2>06/23/14</H2><H3>Yeast Colony PCR of Constitutive Promoters</H3><UL><LI>Boil Yeast at 95°C for 1 hour.</LI></UL><OL><LI>Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 15ul of NaOH. Take 3 colonies from each plate. Use 5ul for PCR reaction below. </LI><LI><P>Set up PCR reaction below: </P><TABLE><THEAD><TR><TH>Reagents</TH><TH>1X</TH><TH>67X</TH></TR></THEAD><TBODY><TR><TD>2X GoTaq Green PCR Master Mix</TD><TD>10 µl</TD><TD>670 µl</TD></TR><TR><TD>10 µM Forward primer</TD><TD>1 µl</TD><TD>67 µl</TD></TR><TR><TD>10 µM Reverse primer</TD><TD>1 µl</TD><TD>67 µl</TD></TR><TR><TD>Water</TD><TD>3 µl</TD><TD>201 µl</TD></TR><TR><TD>cells (template)</TD><TD>5 µl</TD><TD>-------</TD></TR><TR><TD>Total</TD><TD>1005 ul</TD></TR></TBODY></TABLE></LI><LI><P>Add 15 ul of Master Mix into each tube.</P><PRE><CODE>Initial Denaturation    | 95°C | 5 min 
  35 Cycles of: 
      Denaturation | 95°C | 45s
      Annealing    | 55°C | 30s
      Extension    | 72°C | 1min per kb
Final Extension    | 72°C | 10m
Hold               |  4°C | Forever
</CODE></PRE></LI></OL><UL><LI>Load 5ul of PCR products on a gel.  (GoTaq already has loading dye)</LI><LI>Results: All Constitutive Promoters were successfully Transformed. </LI><LI>Made cultures to be diluted for glycerol stocks. Cultures made of the yeast transformations of the constitutive promoters.</LI></UL><H2>06/24/14</H2><H3>Flow Cytometry Protocol:</H3><UL><LI><P>Day Before: </P></LI><LI>Start overnight cultures of your strains to be tested in 5-10ml SD Complete media. (Do not use YPD. YPD has fluorescence background which would cause problems for FACS testing.)</LI></UL><LI><P>Day Of: </P></LI><H5>Flow Procedures:</H5><OL><LI>Dilute overnight cultures 1:100 in the 96 well shaker plate. (Shaker plates can be found in the Lim Lab in the back glass cabinet where large glass bottles can also be found.) </LI><LI>Allow cells to enter growth stage by putting on plate shaker at 30°C going 1000rpm (found in Flow room or the 30°C room in the other hall)for 2-3 hours after dilution.</LI><LI>Induce with alpha factor. The stock alpha factor in the freezer is 3mM and we use it at the final concentrations of: 0, 1nM, 10nM, 100nM, and 1uM.</LI><LI>Allow introduction to proceed on plate shaker for 90 min to 2 hrs, no longer. </LI><LI>Transfer 250ul of each culture to the 96 well flow cytometry v-bottom plate. (found in flow room on shelf) Arrest cells with 10ul of cycloheximide. (stops translations of proteins; stops growth)</LI><LI>Run on flow cytometer. </LI><LI>Analyze Data.</LI></OL><H5>Things to check for Flow:</H5><OL><LI>Check that machine is on standby and not run. (Should be on standby when not in use) </LI><LI>Make sure that a clean plate has been ran by the person before you. Otherwise, run a clean plate. (A1-A4 bleach, B1-B4 water)</LI><LI>Check fluid buffer box and storage tanks. Discard waste fluid in sink 2 doors down (remember to add bleach to tank when done) and replace buffer box if needed.</LI><LI>Make sure you have a positive and negative control so that you can set the parameters.</LI><LI>Open up FACSDiva. Sign into iGEM account. </LI><LI>Create new FACS experiment. Highlight needed wells and use the blue button to create wells for use. Create specimens among the wells and rename them. </LI><LI>Open up inspector to check parameters. </LI><LI>Run well. (Do not click 'Run Plate'.)</LI><LI>Once finished, export data onto a USB.</LI><LI>Run a clean plate.</LI></OL><H5>* Note: NEVER TURN THE WIFI ON! THE COMPUTER WILL CRASH!</H5><P>Voltage Parameters:</P><PRE><CODE>FSC:  250 
SSC:  280 
FITC: 550 
</CODE></PRE><P>Sample Loading Parameters:</P><PRE><CODE>Sample Volume: 200
Mixing Volume: 150
Flow Speed: 0.5ul/sec when adjusting, otherwise 1ul/sec
Mixing quantity: 3
</CODE></PRE><H3>Using Flowjo and Matlab to Analyze Data</H3><H4>Flowjo:</H4><OL><LI>Export Data from USB and transfer to iGEM2014 folder.</LI><LI>Open Flowjo and drag data over to box.</LI><LI>Set an appropriate gate.</LI><LI>Go to data, select all data and press 'Σ'. Select median, mean, and count. </LI><LI>Press the refresh button. Additional numerical data should be present.</LI></OL><H4>Matlab:</H4><OL><LI>Open up a new script. and comment the title of the script. (Use '%' for comments.)</LI><LI>Enter the alpha factor info that will serve as the x-axis. (Put in log form.)</LI><LI>Enter the yGEM data. This will serve as the y-axis. </LI><LI>Type your figure info and press the &quot;go&quot;.</LI></OL><H6>Example of Matlab script to create graphs based on Flowjo Data:</H6><P>%14-6-19 Alpha-Factor Promoter ASG7</P><P>clear % this will get rid of all previous graphs</P><P>Clear all % same as above but for ALL graphs</P><P>alpha = [0 1 10 100 1000] % nanomolar nM, spaces show a new number</P><P>alpha = [0.01 1 10 100 1000] % 0.01 is really 0. Flowjo cannot comput 0s</P><P>yGEM = [558 715 1040 2230 4731] %mean GFP C1-C5 ASG7</P><P>figure(1)</P><P>semilogx(alpha,yGEM4,'--c<EM>'), %('--c</EM>') means dotted line, cyan colored line, and star for points.</P><P>hold on</P><P>xlabel'<A href="nM">alpha</A>') % [] denotes concentration, () denotes &quot;in&quot;</P><P>ylabel('mean GFP (au)') % au means arbitrary unit</P><P>title('Alpha-factor vs mean GDP yGEM4')</P><P>legend('yGEM4')</P><P>axis([0.01 1500 200 4000]); % axis([xmin xmax ymin ymax])</P><UL><LI><P>Use Adobe Illustrator to beautify graphs. Newer Matlab versions may have the option to beautify graph without the use of Illustrator. </P></LI><LI><P>Code available in dropbox.</P></LI></UL><H2>06/24/14</H2><H3>Plate Map for Flow Cytometry of Inducible Promoters:</H3><H3>PCR of rtTA Promoter</H3><UL><LI><P>Became part of Jessica's rtTA portion of the project. </P></LI><LI>Inducible Promoters + rtTA into CB008 and CB008DB</LI></UL><LI><P>PCR Purification of remaining pHY4 + inducible promoters + GFP (Digestion done by Derrick) (Backbone)</P></LI><LI><P>PCR of inducible promoter primer + rtTA with new rtTA RV primer  </P><TABLE><THEAD><TR><TH>Materials</TH><TH>1x reaction</TH><TH>12x Master Mix</TH></TR></THEAD><TBODY><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD><TD>120 µl</TD></TR><TR><TD>dNTP's (10 mM)</TD><TD>1 µl</TD><TD>12  µl</TD></TR><TR><TD>Forward Primer (10µm)</TD><TD>2.5 µl</TD><TD>30  µl</TD></TR><TR><TD>Reverse Primer (10µm)</TD><TD>2.5 µl</TD><TD>30  µl</TD></TR><TR><TD>Template DNA: rtTA</TD><TD>0.5 µl</TD><TD>6   µl</TD></TR><TR><TD>Phusion Polymerase</TD><TD>0.5 µl</TD><TD>6   µl</TD></TR><TR><TD>Water</TD><TD>33 µl</TD><TD>396 µl</TD></TR><TR><TD>Total</TD><TD>50 µl</TD><TD>600 µl</TD></TR></TBODY></TABLE></LI><P>(Note: Add enzymes last)</P><H5>PCR Reaction Procedures:</H5><OL><LI>Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom. </LI><LI>Pipette 50ul from the Master Mix into four labled PCR tubes</LI><LI><P>Put in the thermocycler for the following cycles:</P><PRE><CODE>Initial Denaturation  | 95°C | 5 min 
  30 Cycles of: 
      Denaturation | 95°C | 30s
      Annealing    | 50°C | 20s
      Extension    | 72°C | 2 min
Final Extension    | 72°C | 5 min
Hold               | 4°C  | Forever
</CODE></PRE></LI><LI><P>Keep samples for gel extraction on the following day.</P><TABLE><THEAD><TR><TH>Promoter</TH><TH>Concentration</TH></TR></THEAD><TBODY><TR><TD>PRM2</TD><TD>0.451</TD></TR><TR><TD>YDR124W</TD><TD>15.46</TD></TR><TR><TD>SAG1</TD><TD>12.10</TD></TR><TR><TD>PRM6</TD><TD>21.74</TD></TR><TR><TD>PRM1</TD><TD>4.186</TD></TR><TR><TD>PCL2</TD><TD>6.348</TD></TR></TBODY></TABLE></LI></OL><H3>E. Coli Transformation for Increase of Plasmids:</H3><UL><LI>Plasmids: HY3E (rtTA), HY130E (TRP), HY12E1 (URA3), HY34E2 (HIS2), HY111E2 (LEU2), HY4 (stock plasmid)</LI></UL><H2>06/25/14</H2><H3>Glycerol Stocks of Constitutive Promoters in Yeast (CB008 and CB008 DB)</H3><P>Diluted overnight cultures of constitutive promoters in yeast (CB008 and CB008DB)</P><UL><LI>added 250ul culture to 5ml YPD</LI><LI>put in 30°C incubator for 2 hours</LI></UL><H4>Glycerol Stocks Procedure:</H4><OL><LI>Obtain Glycerol Stock tubes from drawer near autoclave box. (Tubes have orange caps)</LI><LI>Add 350ul of 60% Glycerol or 420ul of 50% Glycerol to 350ul of cells. </LI><LI>Vortex well. </LI><LI>Label tubes and put into -70°C Freezer. (3rd row, 1st drawer)</LI></OL><H3>Graphs of Flow for yGEM4:</H3><H3>Gel Map of pTEF1 Strains Yeast Homology:</H3><H3>Graph of Flow for All Inducible Promoters:</H3><PRE><CODE>|  yGEM#  | Promoter|                       
|---------|---------|
|yGEM4    |   ASG7  |
|yGEM5    |   PCL2  |
|yGEM6    |   SAG1  |
|yGEM10   |   PRM2  |
|yGEM11   |   CLG1  |
|yGEM12   | YDR124W |
|yGEM13   |   PRM6  |
|yGEM14   |   PRM2  |
|yGEM15   |   ECM18 |
|yGEM16   |   PRM3  |

|  yGEM#  |          Strain            |
|---------|----------------------------|
|yGEM23   |CB008 pTEF1-GFP-URA3        |
|yGEM24   |CB008 pTEF1(M3)-GFP-URA3    |
|yGEM25   |CB008 pTEF1(M6)-GFP-URA3    |
|yGEM26   |CB008 pTEF1(M7)-GFP-URA3    |
|yGEM27   |CB008 pTEF1(M10)-GFP-URA3   |
|yGEM28   |CB008DB pTEF1-GFP-URA3      |
|yGEM29   |CB008DB pTEF1(M3)-GFP-URA3  | 
|yGEM30   |CB008DB pTEF1(M6)-GFP-URA3  |
|yGEM31   |CB008DB pTEF1(M7)-GFP-URA3  |
|yGEM32   |CB008DB pTEF1(M10)-GFP-URA3 |

| Promoter|  Owner  |                   
|---------|---------|
|PRM2     | George  |
|ASG7     | Sabrina |
|PCL2     | Eric    |
|CLG1     | Derrick |
|YDR124W  | Ianto   |
|PRM6     | Ianto   |
|PRM1     | Robert  |
|ECM18    | Robert  |
|PRM3     | Eleanor |
|SAG1     | Jessica |
</CODE></PRE><H5>Diluted Overnight Cultures - Constitutive Promoters</H5><UL><LI>Added 250ul of culture to 5ml YPD - put in incubator 30°C. Finished ~ 9:30am</LI></UL><H2>06/26/14</H2><H3>Miniprep of Plasmids for rtTA</H3><H4>Miniprep Precedure:</H4><UL><LI>Obtain E.Coli Cultures from 06/25/14 in 37°C  Incubation Room.</LI></UL><OL><LI>Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.</LI><LI>Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.</LI><LI>Add 350 µl Buffer N3 and invert immediately 4-6 times.</LI><LI>Centrifuge for 10 min at 13,000 rpm.</LI><LI>Add supernatant to QIAprep spin column.</LI><LI>Centrifuge for 30-60s - Discard flow through.</LI><LI>Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.</LI><LI>Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s</LI><LI>Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.</LI><LI>Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.</LI></OL><H5>Nanodrop Concentrations:</H5><OL><LI>PRM2 = 107.0 ng/ul</LI><LI>ASG7 = 111.2 ng/ul</LI><LI>PCL2 = 96.49 ng/ul</LI><LI>CLG1 = 114.8 ng/ul</LI><LI>YDR124W = 90.53 ng/ul</LI><LI>HYM1 = 77.29 ng/ul</LI><LI>PRM6 = 99.83 ng/ul</LI><LI>PRM1 = 75.35 ng/ul</LI><LI>ECM18 = 18.81 ng/ul (to be remade)</LI><LI>PRM3 = 130.8 ng/ul</LI><LI>SAG1 = 104.1 ng/ul</LI><LI>pTEF1 = 182.3 ng/ul</LI></OL><H3>PCR of rtTA Promoter</H3><UL><LI>PCR done on 6/24/14 failed. </LI><LI><P>Added DMSO this time.</P></LI><LI><P>Became part of Jessica's rtTA portion of the project. </P></LI><LI>Inducible Promoters + rtTA into CB008 and CB008DB</LI></UL><LI><P>PCR Purification of remaining pHY4 + inducible promoters + GFP (Digestion done by Derrick) (Backbone)</P></LI><LI><P>PCR of inducible promoter primer + rtTA with new rtTA RV primer  </P><TABLE><THEAD><TR><TH>Materials</TH><TH>1x reaction</TH><TH>12x Master Mix</TH></TR></THEAD><TBODY><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD><TD>120 µl</TD></TR><TR><TD>dNTP's (10 mM)</TD><TD>1 µl</TD><TD>12  µl</TD></TR><TR><TD>Forward Primer (10µm)</TD><TD>2.5 µl</TD><TD>30  µl</TD></TR><TR><TD>Reverse Primer (10µm)</TD><TD>2.5 µl</TD><TD>30  µl</TD></TR><TR><TD>Template DNA: rtTA</TD><TD>0.5 µl</TD><TD>6   µl</TD></TR><TR><TD>Phusion Polymerase</TD><TD>0.5 µl</TD><TD>6   µl</TD></TR><TR><TD>Water</TD><TD>31.5 µl</TD><TD>396 µl</TD></TR><TR><TD>DMSO</TD><TD>1.5ul</TD><TD>30  ul</TD></TR><TR><TD>---------------------</TD><TD>-------------</TD><TD>----------------</TD></TR><TR><TD>Total</TD><TD>50 µl</TD><TD>600 µl</TD></TR></TBODY></TABLE></LI><P>(Note: Add enzymes last)</P><H5>PCR Reaction Procedures:</H5><OL><LI>Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom. </LI><LI>Pipette 50ul from the Master Mix into four labled PCR tubes</LI><LI><P>Put in the thermocycler for the following cycles:</P><PRE><CODE>Initial Denaturation  | 95°C | 5 min 
  30 Cycles of: 
      Denaturation | 95°C | 30s
      Annealing    | 50°C | 20s
      Extension    | 72°C | 2 min
Final Extension    | 72°C | 5 min
Hold               | 4°C  | Forever
</CODE></PRE></LI><LI><P>Keep samples for gel extraction on the following day.</P></LI></OL><H2>06/27/14</H2><H3>rtTA PCR Purification</H3><H4>PCR Purification Prodedure:</H4><OL><LI>5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample</LI><LI>To bind DNA, apply sample to column and spin for 1 min</LI><LI>Discard flow through </LI><LI>To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min</LI><LI>Discard flow through and place column back in the same tube</LI><LI>Cetrifuge for an addtitional minute (Dry Spin)</LI><LI>Place column in clean 1.5 ml tube</LI><LI>Add 30µl water to column, let stand for 1 min and centrifuge for 1 min </LI></OL><H3>Streak Plate of Constitutive Promoters</H3><UL><LI><P>As instructed by Kara, I made a streak plate with all 5 Constitutive Promoters on it. </P></LI><LI>Accidentally made the plate by stamping it instead of actually streaking it. It looks kinda funny. </LI></UL><LI><P>Along with the streak plate, I also made cultures of of CB008 SAG1 and CB008 ECM18, along with two negative cultures of CB008. Cultures of the Contitutive Promoters were made also. </P></LI><LI>All cultures made with SD-Complete. (for Flow)</LI><H3>Gibson Assembly and Transformation</H3><H4>Gibson Assembly Procedures:</H4><PRE><CODE>|        Materials       |  Amount  |   
|------------------------|----------|
|Gibson Master Mix (2x)  |   5ul    |
|Backbone (pHY4+AFRP+GFP)|   3ul    |
|Insert (rtTA + primer)  | Varies   |
|ddH2O                   | Varies   |

    Promoter|Concentration of Insert| Amount of Insert Needed  |H2O Needed| 
    --------| --------------------- | ------------------------ | -------- |
    PRM2    | 107.0 ng/ul           | 1 ul                     |    1ul   |
    ASG7    | 111.2 ng/ul           | 1 µl                     |    1ul   |
    PCL2    | 96.49 ng/ul           | 1 µl                     |    1ul   |
    CLG1    | 114.8 ng/ul           | 1 µl                     |    1ul   |
    YDR124W | 90.53 ng/ul           | 1 µl                     |    1ul   |
    HYM1    | 72.29 ng/ul           | 1 µl                     |    1ul   |
    PRM6    | 99.83 ng/ul           | 1 µl                     |    1ul   |
    PRM1    | 75.35 ng/ul           | 1 µl                     |    1ul   |
    ECM18   | 18.87 ng/ul           | 3 µl                     |    0ul   |
    PRM3    | 130.8 ng/ul           | 1 µl                     |    1ul   |
    SAG1    | 104/1 ng/ul           | 1 µl                     |    1ul   |
    Pos Ctrl| Gibson Positive Ctrl  | 2 µl Backbone, 2ul Insert|    1ul   |
    Neg Ctrl| ASG7 Backbone         | 3 ul Backbone, 0ul Insert|    2ul   |
</CODE></PRE><H5>PRM2 did not have a backbone and thus could not be transformed yet.</H5><UL><LI>Obtain Mach1 Cells from -70°C Freezer

</LI><LI>25ul for 10ul Gibson Mix.</LI></UL><OL><LI>Mix Materials.</LI><LI>Incubate at 50°C for 1 hour.</LI><LI>Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE) </LI><LI>Incubate on ice for 30 min.</LI><LI>Incubate at 42°C for 45 sec.</LI><LI>Incubate on ice for at least 2 min.</LI><LI>Add 250ul SOC Media (Make sure it isn't cloudy) to cells.</LI><LI>Shake in 37°C Incubator Room for 1 hr. (Warm plates also)</LI><LI>Plate on LB-CARB and keep in drawer over the weekend.</LI></OL><H2>06/30/14</H2><H3>Results of Transformation:</H3><PRE><CODE>| Promoter | Colonies|                  
|----------|---------|
| PRM2     |   N/A   |
| ASG7     |    7    |
| PCL2     |    6    |
| CLG1     |   15    |
| YDR124W  |    2    |
| HYMI     |    0    |
| PRM6     |    9    |
| PRM1     |    8    |
| ECM18    |    8    |
| PRM3     |    5    |
| SAG1     |    7    |
| Pos Ctrl |  TMTC   |
| Neg Ctrl |   10    |
</CODE></PRE><H3>Colony PCR of rtTA Transformation:</H3><OL><LI>Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 25ul of water. Do this for 3-5 colonies. Use 5ul for PCR reaction below. </LI><LI><P>Set up PCR Reaction below:</P><TABLE><THEAD><TR><TH>Reagents</TH><TH>1X</TH><TH>6X</TH></TR></THEAD><TBODY><TR><TD>2X GoTaq Green PCR Master Mix</TD><TD>10 µl</TD><TD>60 µl</TD></TR><TR><TD>10 µM Forward primer</TD><TD>1 µl</TD><TD>6 µl</TD></TR><TR><TD>10 µM Reverse primer</TD><TD>1 µl</TD><TD>6 µl</TD></TR><TR><TD>Water</TD><TD>3 µl</TD><TD>18 µl</TD></TR><TR><TD>Bacterial cells (template)</TD><TD>5 µl</TD><TD>-----</TD></TR><TR><TD>Total</TD><TD>90 ul</TD></TR></TBODY></TABLE></LI><LI><P>Add 15 ul of Master Mix into each tube.</P><PRE><CODE>    Initial Denaturation   | 95°C | 5 min 
  30 Cycles of: 
      Denaturation | 95°C | 45s
      Annealing    | 55°C | 30s
      Extension    | 72°C | 1min per kb
Final Extension    | 72°C | 10m
Hold               |  4°C  | Forever
</CODE></PRE></LI></OL><UL><LI><P>Load PCR products on a gel. (5ul) (GoTaq already has loading dye)</P></LI><LI><P>For all positive bands on the gel, add rest of bacterial cells from PCR tubes to 5ml LB and grow overnight at 37°C for miniprep on the following day. </P></LI></UL><H4>Gel of rtTA Tranformation Colony PCR:</H4><UL><LI><P>Original Gel failed due to the bottom part of the gel disappearing. (possibly due to the bottom part not solidifying enough)</P></LI><LI><P>Jessica made PRM2 Backbone</P></LI><LI><P>ECM18 PCR'ed</P></LI><LI><P>Both were Gibson'ed and transformed. </P></LI></UL><H5>Cultures were made of pHY4 AFRP's + rtTA.</H5><UL><LI>5ml YPD + Transformed Colonies</LI><LI>Put in 37°C Incubator Room Shaker overnight.</LI></UL><H2>07/01/14</H2><H3>Miniprep of Inducible Promoters + rtTA</H3><H6>Note: Try not to do more than one miniprep per promoter.</H6><H4>Miniprep Precedure: (did in triplicate for some reason)</H4><UL><LI>Obtain E.Coli Cultures from 06/30/14 in 37°C  Incubation Room.</LI></UL><OL><LI>Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.</LI><LI>Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.</LI><LI>Add 350 µl Buffer N3 and invert immediately 4-6 times.</LI><LI>Centrifuge for 10 min at 13,000 rpm.</LI><LI>Add supernatant to QIAprep spin column.</LI><LI>Centrifuge for 30-60s - Discard flow through.</LI><LI>Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.</LI><LI>Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s</LI><LI>Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.</LI><LI>Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.</LI></OL><H5>Nanodrop Concentrations (ng/ul):</H5><OL><LI>ASG7 1 = 203.0 ng/ul</LI><LI>ASG7 2 = 282.9 ng/ul</LI><LI>ASG7 3 = 244.9 ng/ul</LI><LI>PCL2 1 = 205.5 ng/ul</LI><LI>PCL2 2 = 205.3 ng/ul</LI><LI>PCL2 3 = 305.3 ng/ul</LI><LI>CLG1 1 = 155.5 ng/ul</LI><LI>CLG1 2 = 187.6 ng/ul</LI><LI>CLG1 3 = 230.0 ng/ul</LI><LI>YDR124W 1 = 89.44 ng/ul</LI><LI>PRM6 1 = 246.4ng/ul</LI><LI>PRM6 2 = 17.32 ng/ul</LI><LI>PRM6 3 = 154.0 ng/ul</LI><LI>PRM3 1 = 244.9 ng/ul</LI><LI>PRM3 3 = 180.1 ng/ul</LI><LI>SAG1 1 = 146.5 ng/ul</LI><LI>SAG1 2 = 243.8 ng/ul</LI><LI>SAG1 3 = 137.8 ng/ul</LI></OL><P>*Did not use PRM 1 #1, 2, and 3 or PRM3 #2 because the cultures did not grow.</P><H5>Sent into sequencing.</H5><H3>Colony PCR for Screening Yeast - Constitutive Promoters + GFP</H3><UL><LI><P>Used Streak plate in 4°C Fridge.</P><TABLE><THEAD><TR><TH>Reagents</TH><TH>1X</TH><TH>6X</TH></TR></THEAD><TBODY><TR><TD>2X GoTaq Green PCR Master Mix</TD><TD>10 µl</TD><TD>60 µl</TD></TR><TR><TD>10 µM Forward primer</TD><TD>1 µl</TD><TD>-----</TD></TR><TR><TD>10 µM Reverse primer</TD><TD>1 µl</TD><TD>6 µl</TD></TR><TR><TD>Water</TD><TD>5 µl</TD><TD>30 µl</TD></TR><TR><TD>Bacterial cells (template)</TD><TD>3 µl</TD><TD>-----</TD></TR><TR><TD>Total</TD><TD>96 ul</TD></TR></TBODY></TABLE></LI><LI><P>add 15ul of master mix to each tube.</P><PRE><CODE>    Initial Denaturation   | 95°C | 5 min 
  30 Cycles of: 
      Denaturation | 95°C | 45s
      Annealing    | 55°C | 30s
      Extension    | 72°C | 1min per kb
Final Extension    | 72°C | 10m
Hold               |  4°C  | Forever
</CODE></PRE></LI></UL><H5>Load 5ul of PCR products on a gel. (GoTaq already has loading dye)</H5><UL><LI>10uM RV Primer = GFP RV Primer - kept in Primer Working Stock (2014 Primers)</LI><LI>10uM FW Primer = Individual pTEF1/mutant Primers kept in the -20°C freezer boxes of the respective owners. (pTEF1=Jeffrey, M3=Sabrina, M6=Eleanor, M7=Robert, M10=George)

</LI><LI>M7 used M10 primer due to the primers being so similar. M7 ran out of its own primer.</LI></UL><P>Gel of Yeast Colony PCR (Constitutive Promoters)</P><UL><LI>Gel mostly failed - to be redone the following day. </LI></UL><H5>Sent into sequencing.</H5><UL><LI>Sequencing for pTEF promoters cancelled due to the absence of primers that were supposed to be sent in or selected on the promoter catalog.</LI></UL><H2>07/02/14</H2><H3>Re-Colony PCR for Yeast - Constitutive Promoters + GFP</H3><H3>Colony PCR for Screening Yeast - Constitutive Promoters + GFP</H3><UL><LI><P>Used Streak plate in 4°C Fridge. </P></LI><LI><P>picked 3 different colonies.</P><TABLE><THEAD><TR><TH>Reagents</TH><TH>1X</TH><TH>6X</TH></TR></THEAD><TBODY><TR><TD>2X GoTaq Green PCR Master Mix</TD><TD>10 µl</TD><TD>60 µl</TD></TR><TR><TD>10 µM Forward primer</TD><TD>1 µl</TD><TD>-----</TD></TR><TR><TD>10 µM Reverse primer</TD><TD>1 µl</TD><TD>6 µl</TD></TR><TR><TD>Water</TD><TD>5 µl</TD><TD>30 µl</TD></TR><TR><TD>Bacterial cells (template)</TD><TD>3 µl</TD><TD>-----</TD></TR><TR><TD>Total</TD><TD>96 ul</TD></TR></TBODY></TABLE></LI></UL><LI><P>add 15ul of master mix to each tube.</P><PRE><CODE>    Initial Denaturation   | 95°C | 5 min 
  30 Cycles of:
      Denaturation | 95°C | 45s
      Annealing    | 55°C | 30s
      Extension    | 72°C | 1min per kb
Final Extension    | 72°C | 10m
Hold               |  4°C  | Forever
</CODE></PRE></LI><H5>Load 5ul of PCR product on a gel. (GoTaq already has loading dye)</H5><UL><LI>10uM RV Primer = GFP RV Primer - kept in Primer Working Stock (2014 Primers)</LI><LI>10uM FW Primer = Individual pTEF1/mutant Primers kept in the -20°C freezer boxes of the respective owners. (pTEF1=Jeffrey, M3=Sabrina, M6=Eleanor, M7=Robert, M10=George)

</LI><LI>M7 used M10 primer due to the primers being so similar. M7 ran out of its own primer.</LI></UL><H4>Gel of Yeast Colony PCR (Const. + GFP):</H4><H3>Yeast PCR Purification</H3><UL><LI>To extract Yeast DNA for Sequencing</LI></UL><H4>PCR Purification Prodedure:</H4><OL><LI>5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample</LI><LI>To bind DNA, apply sample to column and spin for 1 min</LI><LI>Discard flow through </LI><LI>To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min</LI><LI>Discard flow through and place column back in the same tube</LI><LI>Cetrifuge for an addtitional minute (Dry Spin)</LI><LI>Place column in clean 1.5 ml tube</LI><LI>Add 30µl water to column, let stand for 1 min and centrifuge for 1 min </LI></OL><H5>Nanodrop Concentrations:</H5><OL><LI>pTEF1       = 12.30 ng/ul</LI><LI>pTEF1 (M3)  = 6.242 ng/ul</LI><LI>pTEF1 (M6)  = 33.68 ng/ul</LI><LI>pTEF1 (M7)  = 39.72 ng/ul</LI><LI>pTEF1 (M10) = 19.43 ng/ul</LI></OL><H5>Did not send into sequencing due to the conclusion that the constitutive promoters were successfully transformed based on past sequencing and flow cytometry.</H5><UL><LI>I restreaked the streak plate from the patch plate with the colonies marked (✓) on them. I did not trust the other streak plate. Incubated in 30°C incubator.</LI></UL><H3>Re-Transformation of pHY4 Inducuble Promoters + rtTA</H3><H4>Gibson Assembly Procedures:</H4><PRE><CODE>|        Materials       |  Amount  |   
|------------------------|----------|
|Gibson Master Mix (2x)  |   5ul    |
|Backbone (pHY4+AFRP+GFP)|   4ul    |
|Insert (rtTA + primer)  |   1ul    |
|ddH2O                   |   0ul    |
|Total                   |   10ul   |
</CODE></PRE><UL><LI>Obtain NEB5α (C2987) Cells instead of Mach1 cells from -70°C Freezer

</LI><LI>25ul for 10ul Gibson Mix.</LI></UL><OL><LI>Mix Materials.</LI><LI>Incubate at 50°C for 1 hour.</LI><LI>Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE) </LI><LI>Incubate on ice for 30 min.</LI><LI>Incubate at 42°C for 45 sec.</LI><LI>Incubate on ice for at least 2 min.</LI><LI>Add 250ul SOC Media (Make sure it isn't cloudy) to cells.</LI><LI>Shake in 37°C Incubator Room for 30 min rather than 1 hr. (Warm plates also)</LI><LI>Plate on LB-CARB and incubate in 37°C Incubator Room.</LI></OL><H5>Re-Transformed Promoters: (never been transformed, not enough colonies, etc)</H5><OL><LI>PCL2</LI><LI>YDR124W</LI><LI>HYM1</LI><LI>PRM6</LI><LI>PRM1</LI><LI>ECM18</LI><LI>PRM3</LI><LI>PRM2</LI><LI>Negative Control (heated at 55°C for 40 min due to being made late)</LI></OL><H5>ALWAYS MAKE A NEGATIVE CONTROL!</H5><UL><LI>Jessica did a Colony PCR for rtTA Promoters and a Miniprep for the 1st Transformation.</LI></UL><H2>07/03/14</H2><H3>Results of Re-Transformtation of Inducible Promoters + GFP</H3><UL><LI>Few to no colonies grew onthe transformed plates, which leads me to think that the Gibson Assembly failed because only PRM2 worked and was Gibson'ed prior to the other transformed promoters.</LI><LI><P>PRM2 grew many tiny colonies.</P><TABLE><THEAD><TR><TH>Promoter</TH><TH>Colonies</TH></TR></THEAD><TBODY><TR><TD>PRM2</TD><TD>TMTC</TD></TR><TR><TD>PCL2</TD><TD>2</TD></TR><TR><TD>PRM6</TD><TD>3</TD></TR><TR><TD>PRM3</TD><TD>2</TD></TR><TR><TD>YDR124W</TD><TD>~ 40</TD></TR><TR><TD>PRM1</TD><TD>0</TD></TR><TR><TD>ECM18</TD><TD>0</TD></TR><TR><TD>HYM1</TD><TD>0</TD></TR><TR><TD>Neg Ctrl</TD><TD>0</TD></TR></TBODY></TABLE></LI></UL><H5>Possible Explanations for Failure:</H5><UL><LI>Gibson Assembly failed due to bad Gibson Master Mix</LI><LI>Backbone concentrations were not high enough, which would make sense because PRM2 had a new backbone which had a higher concentration than the rest of the promoters.

</LI><LI>PRM2 Backbone Concentration: 208.7 ng/ul</LI></UL><H3>Backbone Digestion</H3><UL><LI>Jessica remade the backbones, but the concentrations are still very low. 

</LI><LI>Used the minipreps from my box in the -20°C fridge.</LI></UL><H3>Colony PCR of Miniprep (Inducible Promoter + GFP)</H3><PRE><CODE>|Reagents                      | 1X   | 20X   |
|----------------------------- |------| ----- |
|2X GoTaq Green PCR Master Mix | 10µl | 200µl |
|10 µM Forward primer          | 1µl  | ----- |
|10 µM Reverse primer          | 1µl  | 20µl  |
|Water                         | 5µl  | 100µl |
|Miniprep                      | 1µl  | ----- | 
|Total                         | ---- | 320ul |

    Initial Denaturation   | 95°C | 5 min 
      30 Cycles of:
          Denaturation | 95°C | 45s
          Annealing    | 55°C | 30s
          Extension    | 72°C | 1min per kb
    Final Extension    | 72°C | 10m
    Hold               |  4°C  | Forever
</CODE></PRE><UL><LI>10uM FW Primer = individual inducible promoter primers - kept in Jessica's box.</LI><LI>10uM RV Primer = rtTA primer - kept in 2014 iGEM primer working stock box.</LI></UL><H5>Sent into sequencing.</H5><H4>Gel of Miniprep Colony PCR (Inducible Promoter + GFP):</H4><H2>07/07/14</H2><H3>Confusion with Minipreps</H3><UL><LI>Due to a confusion on 7/3/14, all minipreps from before 7/7/14 are to be thrown away from my box. Only the needed minipreps are to be kept.</LI><LI>Inserts were remade for no reason. (old inserts were fine) </LI><LI>Backbones may not have been digested properly. (minimum of 2 hrs needed)</LI></UL><H2>07/08/14</H2><H3>PCR Amplification of new rtTA</H3><UL><LI><P>Amplification of new rtTA with homology to AFRP's.</P></LI><LI><P>extract rtTA from pTS47 (~100ng/ul)</P><TABLE><THEAD><TR><TH>Materials</TH><TH>1x reaction</TH><TH>12x Master Mix</TH></TR></THEAD><TBODY><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD><TD>120 µl</TD></TR><TR><TD>dNTP's (10 mM)</TD><TD>1 µl</TD><TD>12 µl</TD></TR><TR><TD>Forward Primer (10µm)</TD><TD>2.5 µl</TD><TD>individually added</TD></TR><TR><TD>Reverse Primer (10µm)</TD><TD>2.5 µl</TD><TD>30 µl</TD></TR><TR><TD>Template DNA:rtTA</TD><TD>1 µl</TD><TD>12 µl</TD></TR><TR><TD>Phusion Polymerase</TD><TD>0.5 µl</TD><TD>6 µl</TD></TR><TR><TD>Water</TD><TD>31 µl</TD><TD>372 µl</TD></TR><TR><TD>DMSO</TD><TD>1.5 µl</TD><TD>18 µl</TD></TR><TR><TD>Total</TD><TD>50 ul</TD><TD>570 ul</TD></TR></TBODY></TABLE></LI></UL><P>(Note: Add enzymes last)</P><H5>PCR Reaction Procedures:</H5><OL><LI>Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom. </LI><LI>Pipette 50ul from the Master Mix into four labled PCR tubes</LI><LI><P>Put in the thermocycler for the following cycles:</P><PRE><CODE>Initial Denaturation   | 95°C | 5 min 
  30 Cycles of: 
      Denaturation | 95°C | 30 sec
      Annealing    | 55°C | 30 sec
      Extension    | 72°C | 2 min
Final Extension    | 72°C | 5 min
Hold               | 4° C | Forever
</CODE></PRE></LI><LI><P>Keep samples for gel extraction on the following day.</P></LI></OL><H5>Nanodrop Concentration:</H5><OL><LI>PRM1 = 115.9 ng/ul</LI><LI>PRM2 = 176.6 ng/ul</LI><LI>PRM3 = 169.2 ng/ul</LI><LI>PRM6 = 154.9 ng/ul</LI><LI>ECM18 = 122.5 ng/ul</LI><LI>SAG1 = 161.9 ng/ul</LI><LI>YDR124W = 99.70 ng/ul</LI><LI>CLG1 = 153.5 ng/ul</LI><LI>ASG7 = 139.9 ng/ul</LI><LI>HYM1 = 123.5 ng/ul</LI><LI>PCL2 = 1-2.3 ng/ul</LI></OL><H2>07/09/14</H2><H3>Gibson Assembly of rtTA Inducible Promoters (Transformation #3)</H3><H4>Gibson Assembly Procedures:</H4><PRE><CODE>|        Materials       |  Amount  |   
|------------------------|----------|
|Gibson Master Mix (2x)  |   5ul    |
|Backbone (pHY4+AFRP+GFP)|   3ul    |
|Insert (rtTA + primer)  |   1ul    |
|ddH2O                   |   1ul    |
|Total                   |   10ul   |
</CODE></PRE><OL><LI>Mix Materials.</LI><LI>Incubate at 50°C for 1 hour.</LI><LI>Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE) </LI><LI>Incubate on ice for 30 min.</LI><LI>Incubate at 42°C for 45 sec.</LI><LI>Incubate on ice for at least 2 min.</LI><LI>Add 250ul SOC Media (Make sure it isn't cloudy) to cells.</LI><LI>Shake in 37°C Incubator Room for 1 hr. (Warm plates also)</LI><LI>Plate on LB-CARB and incubate in 37°C Incubator Room overnight.</LI></OL><H2>07/10/14</H2><H3>Results of Transformation from 7/9/14</H3><UL><LI><P>Most of the plates grew well except:</P></LI><LI>PRM1</LI><LI>PRM2</LI><LI>PRM6</LI><LI>HYM1</LI></UL><LI><P>Positive control worked.</P></LI><LI><P>Negative control also worked. </P></LI><LI>maybe the backbones were not digested enough?</LI><LI>Going to redo backones for PRM1 and PRM2 (to be redigested)</LI><H3>Backbone Digest for PRM1 and PRM2</H3><TABLE><THEAD><TR><TH>Materials</TH><TH>PRM1</TH><TH>PRM2</TH></TR></THEAD><TBODY><TR><TD>NotI</TD><TD>0.5 ul</TD><TD>0.5 µl</TD></TR><TR><TD>XhoI</TD><TD>0.5 µl</TD><TD>0.5 µl</TD></TR><TR><TD>CutSmart</TD><TD>5 µl</TD><TD>5 ul</TD></TR><TR><TD>pGEM</TD><TD>0 µl</TD><TD>4 µl</TD></TR><TR><TD>Original Digest</TD><TD>28 µl</TD><TD>40 µl</TD></TR><TR><TD>H2O</TD><TD>16 µl</TD><TD>0 µl</TD></TR><TR><TD>Total</TD><TD>50 ul</TD><TD>50 ul</TD></TR></TBODY></TABLE><UL><LI>Incubate at 37°C for at least 2 hours. </LI><LI>PCR Purify. </LI></UL><H4>PCR Purification Procedure:</H4><OL><LI>5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample</LI><LI>To bind DNA, apply sample to column and spin for 1 min</LI><LI>Discard flow through </LI><LI>To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min</LI><LI>Discard flow through and place column back in the same tube</LI><LI>Cetrifuge for an addtitional minute (Dry Spin)</LI><LI>Place column in clean 1.5 ml tube</LI><LI>Add 30µl water to column, let stand for 1 min and centrifuge for 1 min </LI></OL><UL><LI>Concentrations of Backbone Digestions of PRM1 and PRM2 were too low; not usable. </LI></UL><H5>Colony PCR Gel of Transformation #3:</H5><H2>07/11/14</H2><H3>Miniprep of Transformation #3</H3><UL><LI>Made cultures of the the transformations.</LI></UL><H5>Sent Transformation #3 into sequencing.</H5><H2>07/14/14</H2><H3>Results of Sequencing and Colony PCR</H3><UL><LI><P>PCR showed that rtTA and GFP were both present. </P></LI><LI><P>Sequencing showed that GFP was in the backbone, not rtTA</P></LI><LI>rtTA was probably not in the backbone and was floating near the colonies instead, so when we picked up the colonies for Colony PCR, we also picked up the rtTA with them. </LI><LI>Will use Backbone set #2 in Jessica's box along with Derrick's PRM1 and PRM2 backbones from 6/09/14.</LI><LI>Will use the new inserts from 7/11/14 in my box.</LI></UL><H3>Gibson Assembly and Transformation #4</H3><H4>Gibson Assembly Procedures:</H4><PRE><CODE>|        Materials       |  Amount  |   
|------------------------|----------|
|Gibson Master Mix (2x)  |   5ul    |
|Backbone (pHY4+AFRP+GFP)|  Varies  |
|Insert (rtTA + primer)  |   1ul    |
|ddH2O                   |  Varies  |
|Total                   |   10ul   |

Promoter |Concentration of Backbone| Amount of Insert Needed  |H2O Needed| 
|------- | ----------------------- | ------------------------ | -------- |
PRM1     | ~50.0 ng/ul             | 1 ul                     |   3 ul   |
PRM2     | ~50.0 ng/ul             | 1 µl                     |   3 ul   |
PRM3     | 15.20 ng/ul             | 1 µl                     |   1 ul   |
PRM6     | 21.74 ng/ul             | 1 µl                     |   1 ul   |
ECM18    | 21.39 ng/ul             | 1 µl                     |   1 ul   |
SAG1     | 12.10 ng/ul             | 1 µl                     |   1 ul   |
YDR124W  | 15.46 ng/ul             | 1 µl                     |   1 ul   |
CLG1     | 16.59 ng/ul             | 1 µl                     |   1 ul   |
ASG7     | 23.77 ng/ul             | 3 µl                     |   1 ul   |
HYM1     | 12.96 ng/ul             | 1 µl                     |   1 ul   |
PCL2     | 6.348 ng/ul             | 1 µl                     |   0 ul   |
Pos Ctrl | Gibson Positive Ctrl    | 3 µl Backbone 1 ul Insert|   0 ul   |
Neg Ctrl | PRM1 Backbone           | 1 ul Backbone 0 ul Insert|   4 ul   |
</CODE></PRE><UL><LI>Concentrations of PRM1 and PRM2 Backbones were around 200 ng/ul. I diluted them 1:4 down to 50 ng/ul. </LI></UL><OL><LI>Mix Materials.</LI><LI>Incubate at 50°C for 1 hour.</LI><LI>Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE) </LI><LI>Incubate on ice for 30 min.</LI><LI>Incubate at 42°C for 45 sec.</LI><LI>Incubate on ice for at least 2 min.</LI><LI>Add 250ul SOC Media (Make sure it isn't cloudy) to cells.</LI><LI>Shake in 37°C Incubator Room for 1 hr. (Warm plates also)</LI><LI>Plate on LB-CARB and keep in drawer over the weekend.</LI></OL><H2>07/15/14</H2><H3>Glycerol Stocks for pTET + GFP + rtTA</H3><UL><LI>Made cultures of pTET + GFP + Const. Promoter + rtTA.</LI><LI>CB008: pTEF1, M6, M7, and M10</LI><LI>CB008DB: M6, and M7</LI></UL><P>Made a streak plate with the strains on them. </P><UL><LI>Did a PCR of the rtTA Backbone and Inducible Promoter Insert. 

</LI><LI>Used pGEM20 rtTA backbone (~100 ng/ul dilution)</LI></UL><H5>Gel Picture of PCR:</H5><H3>Transformation of HY130E into DH5Fα Cells</H3><OL><LI>All of DNA (~2ul) + 50ul DH5α.</LI><LI>Put on ice for 10 min.</LI><LI>Heatshock for 45 sec at 42°C.</LI><LI>Put on ice for at least 2 min.</LI><LI>Add 250ul SOC.</LI><LI>Incubate for 30 min in 37°C Room Shaker.</LI><LI>Plate 100ul on LB-CARB plate.</LI><LI>Plate 200ul on another LB-CARB plate.</LI></OL><H2>07/16/14</H2><H3>rtTA Insert</H3><UL><LI>Used pGEM16 Template to make rtTA insert. </LI></UL><H3>Gel Extraction of rtTA PCR</H3><H4>Gel Extraction Procedure:</H4><OL><LI>Cut Gel</LI><LI>Weigh it in a colorless tube</LI><LI>Add 3 volumes Buffer QG to 1 volime Gel (100mg ~ 100µl)</LI><LI>Incubate @ 50°C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)</LI><LI>Add 1 gel volume isopropanal to the sample and mix</LI><LI>Place a QlAquick soin column in a provided 2ml collection tube</LI><LI>Place sample in column &amp; spin for 1 min -&gt; discard flow through</LI><LI>To wash add 0.75 ml Buffer PE to column &amp; centrifue for 1 min, then dry spin</LI><LI>Place column in 1.5 ml tube</LI><LI>Add 30µl H2O &amp; centrifuge for 1 min.</LI></OL><H5>Gel #1:</H5><H5>Gel #2:</H5><H5>Fragment Weights: (try to cut smaller pieces)</H5><OL><LI>PRM1 = 0.54 g = 540ul</LI><LI>PRM2 = 0.56 g = 560ul</LI><LI>PRM3 = 1.15 g = 1150ul</LI><LI>PRM6 = 1.21 g = 1210ul</LI><LI>ECM18 = 0.61 g = 610ul</LI><LI>SAG1 = 0.68 g = 680ul</LI><LI>YDR124W = 0.45 g = 450ul</LI><LI>CLG1 = 0.4 g = 400ul</LI><LI>ASG7 = 0.31 g = 310ul</LI><LI>HYM1 = 0.50 g = 500ul</LI><LI>PCL2 = 0.42 g = 420ul</LI><LI>M3 = 0.69 g = 690ul</LI></OL><UL><LI><P>Initial Concentrations of Gel Extraction were too low. Re-PCRed and obtained new concentrations. See Jeffrey's notebook for initial concentrations.</P></LI><LI><P>Assisted Jeffrey in doing gel extraction for M3.</P></LI></UL><H5>New Concentrations:</H5><OL><LI>PRM1 = 102.2 ng/ul</LI><LI>PRM2 = 110.6 ng/ul</LI><LI>PRM3 = 135.5 ng/ul</LI><LI>PRM6 = 130.7 ng/ul</LI><LI>ECM18 = 136.8 ng/ul</LI><LI>SAG1 = 152.4 ng/ul</LI><LI>YDR124W = 78.89 ng/ul</LI><LI>CLG1 = 133.7 ng/ul</LI><LI>ASG7 = 68.15 ng/ul</LI><LI>HYM1 = 91.08 ng/ul</LI><LI>PCL2 = 86.63 ng/ul</LI><LI>M3 = 47.19 ng/ul</LI></OL><UL><LI>Stored in Jessica's box in -20°C fridge.</LI></UL><H2>07/17/14</H2><H3>Gibson and Transformation (Inducible Promoter + rtTA) #5</H3><H4>Gibson Assembly Procedure:</H4><PRE><CODE>|        Materials       |  Amount  |   
|------------------------|----------|
|Gibson Master Mix (2x)  |   5ul    |
|Backbone (pHY4+AFRP+GFP)|  Varies  |
|Insert (rtTA + primer)  |  Varies  |
|ddH2O                   |  Varies  |
|Total                   |   10ul   |

Promoter |Amount of Backbone Needed| Amount of Insert Needed  |H2O Needed| 
|--------| ----------------------- | ------------------------ | -------- |
PRM1     | 1ul                     | 1 ul                     |   3 ul   |
PRM2     | 3ul                     | 1 µl                     |   3 ul   |
PRM3     | 3ul                     | 1 µl                     |   1 ul   |
PRM6     | 3ul                     | 1 µl                     |   1 ul   |
ECM18    | 3ul                     | 1 µl                     |   1 ul   |
SAG1     | 3ul                     | 1 µl                     |   1 ul   |
YDR124W  | 3ul                     | 1 µl                     |   1 ul   |
CLG1     | 3ul                     | 1 µl                     |   1 ul   |
ASG7     | 3ul                     | 2 µl                     |   0 ul   |
HYM1     | 3ul                     | 1 µl                     |   1 ul   |
PCL2     | 4ul                     | 1 µl                     |   0 ul   |
Pos Ctrl | 3ul                     | 2 ul                     |   0 ul   |
Neg Ctrl | 3ul                     | 1 ul                     |   1 ul   |
</CODE></PRE><UL><LI>Used YDR124W Backbone for Negative Control</LI></UL><OL><LI>Mix Materials.</LI><LI>Incubate at 50°C for 1 hour.</LI><LI>Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE) </LI><LI>Incubate on ice for 30 min.</LI><LI>Incubate at 42°C for 45 sec.</LI><LI>Incubate on ice for at least 2 min.</LI><LI>Add 250ul SOC Media (Make sure it isn't cloudy) to cells.</LI><LI>Shake in 37°C Incubator Room for 1 hr. (Warm plates also)</LI><LI>Plate on LB-CARB and keep in drawer over the weekend.</LI></OL><H2>07/18/14</H2><H3>Results of Transformation #5</H3><PRE><CODE>| Promoter | Colonies|                  
|----------|---------|
| PRM1     |    0    |
| PRM2     |    4    |
| PRM3     |    0    |
| PRM6     |  TMTC   |
| ECM18    |    0    |
| SAG1     |  ~25    |
| YDR124W  |    0    |
| CLG1     |    0    |
| ASG7     |    0    |
| HYM1     |    0    |
| PCL2     |    0    |
| Pos Ctrl |    0    |
| Neg Ctrl |    0    |
</CODE></PRE><UL><LI>Used only 100ul for transformation.</LI><LI>Going to try again with the remaining 150ul. </LI><LI>Stored in drawer at room temperature for transformation over the weekend.</LI><LI>Since Positive Control did not work, I suspect that either the positive control was bad or that the Gibson was bad.</LI></UL><H2>07/21/14</H2><H3>Results of Transformation #6</H3><PRE><CODE>| Promoter | Colonies|                  
|----------|---------|
| PRM1     |   30    |
| PRM2     |   30    |
| PRM3     |   40    |
| PRM6     |  TMTC   |
| ECM18    |   30    |
| SAG1     |   50    |
| YDR124W  |   20    |
| CLG1     |   20    |
| ASG7     |   20    |
| HYM1     |   30    |
| PCL2     |   30    |
| Pos Ctrl |   20    |
| Neg Ctrl |   20    |
</CODE></PRE><UL><LI>Since Positive Control did grow very much and the Negative Control worked, I suspect that either the positive control was bad or that the Gibson was bad.</LI><LI><P>Incubated for 4 more hours in the 37°C Incubator Room so that the colonies could grow slightly bigger. (All the colonies are very tiny)</P></LI><LI><P>Made cultures for PRM2, PRM6, and SAG1 for Colony PCR/Miniprep.</P></LI></UL><H5>Sent Colony PCR/Miniprep of PRM2, PRM6, and SAG1 into sequencing.</H5><UL><LI>Since most of the transformations did not actually seem to work, we will be trying Seamless to see if the transformations are any better.</LI></UL><H3>Backbone Redigestion</H3><UL><LI><P>Decided to redigest backbones for higher concentrations</P></LI><LI>Backbones: PRM3, SAG1, YDR124W, HYM1, PCL2</LI><LI><P>pGEM #: pGEM3, pGEM6, pGEM7, pGEM10, pGEM11</P><TABLE><THEAD><TR><TH>Materials</TH><TH>PRM3</TH><TH>SAG1</TH><TH>YDR124W</TH><TH>HYM1</TH><TH>PCL2</TH></TR></THEAD><TBODY><TR><TD>NotI</TD><TD>0.5 ul</TD><TD>0.5 µl</TD><TD>0.5 ul</TD><TD>0.5 ul</TD><TD>0.5 ul</TD></TR><TR><TD>XhoI</TD><TD>0.5 µl</TD><TD>0.5 µl</TD><TD>0.5 ul</TD><TD>0.5 ul</TD><TD>0.5 ul</TD></TR><TR><TD>CutSmart</TD><TD>5 µl</TD><TD>5 ul</TD><TD>5 ul</TD><TD>5 ul</TD><TD>5 ul</TD></TR><TR><TD>pGEM</TD><TD>5 µl</TD><TD>4 µl</TD><TD>4 ul</TD><TD>7 ul</TD><TD>7 ul</TD></TR><TR><TD>H2O</TD><TD>34 µl</TD><TD>40 µl</TD><TD>40 ul</TD><TD>37 ul</TD><TD>37 ul</TD></TR><TR><TD>Total</TD><TD>50 ul</TD><TD>50 ul</TD><TD>50 ul</TD><TD>50 ul</TD><TD>50 ul</TD></TR></TBODY></TABLE></LI></UL><OL><LI>Incubate at 37°C for at least 2 hours.</LI><LI>Heat inactivate at 65°C for 20 min.</LI><LI>Add 10ul of 6x loading dye to each sample.</LI><LI>Gel extract.</LI></OL><H4>Gel Extraction Procedure:</H4><OL><LI>Cut Gel</LI><LI>Weigh it in a colorless tube</LI><LI>Add 3 volumes Buffer QG to 1 volime Gel (100mg ~ 100µl)</LI><LI>Incubate @ 50° C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)</LI><LI>Add 1 gel volume isopropanal to the sample and mix</LI><LI>Place a QlAquick soin column in a provided 2ml collection tube</LI><LI>Place sample in column &amp; spin for 1 min -&gt; discard flow through</LI><LI>To wash add 0.75 ml Buffer PE to column &amp; centrifue for 1 min, then dry spin</LI><LI>Place column in 1.5 ml tube</LI><LI>Add 30µl H2O &amp; centrifuge for 1 min.</LI></OL><H5>New Concentrations of Backbones:</H5><TABLE><THEAD><TR><TH>Promoter</TH><TH>Concentration</TH></TR></THEAD><TBODY><TR><TD>PRM3</TD><TD>13.48 ng/ul</TD></TR><TR><TD>SAG1</TD><TD>44.26 ng/ul</TD></TR><TR><TD>YDR124W</TD><TD>82.00 ng/ul</TD></TR><TR><TD>HYM1</TD><TD>39.66 ng/ul</TD></TR><TR><TD>PCL2</TD><TD>82.97 ng/ul</TD></TR></TBODY></TABLE><H3>Seamless Cloning and Assembly Reaction</H3><H4>Seamless Procedure:</H4><PRE><CODE>Promoter |Amount of Backbone Needed| Amount of Insert Needed  | Seamless | 
|--------| ----------------------- | ------------------------ | -------- |
PRM1     | 1ul                     | 2 ul                     |   3 ul   |
PRM2     | 3ul                     | 2 µl                     |   3 ul   |
PRM3     | 3ul                     | 1.5 µl                   |   1 ul   |
PRM6     | 3ul                     | 1.5 ul                   |   1 ul   |
ECM18    | 3ul                     | 1.5 µl                   |   1 ul   |
SAG1     | 3ul                     | 1.3 µl                   |   1 ul   |
YDR124W  | 3ul                     | 2.5 µl                   |   1 ul   |
CLG1     | 3ul                     | 1.5 µl                   |   1 ul   |
ASG7     | 3ul                     | 3 µl                     |   0 ul   |
HYM1     | 3ul                     | 2.2 µl                   |   1 ul   |
PCL2     | 4ul                     | 2.3 µl                   |   0 ul   |

|        Materials                |  Amount  |
|---------------------------------|----------|
|Inserts (200ng each)             |   x ul   |
|Linear Cloning Vector (50ng each)|   y ul   |
|GeneArt 2X Enzyme Mix            | x+y ul   |
</CODE></PRE><OL><LI>In a microcentrifuge tube, set up the seamless cloning and assembly reaction. It is CRUTIAL that you add the GeneArt 2x Enzyme Mix as the LAST COMPONENT. </LI><LI>Quickly thaw the GeneArt 2x Enzyme Mix on ice and pipette up and down to mix thoroughly. Add (x+y) ul of thawed enzyme. Mix to each reaction. Immediately return GeneArt 2x Enzyme Mix to -80°C Freezer. </LI><LI>Mix the reaction components completely by pipetting up and down SLOWLY 3 times and then gently tap the side of the tubed 3-5 times to mix. </LI><LI>If needed, briefly centrifge (&lt; 500rpm for &lt; 5sec) to collect reaction components to the bottom of the microcentrifuge tube.</LI><LI>Incubate the reaction mix at room temperature for 15-30 min. Do not incubate longer than 30 min!</LI><LI>After incubation is complete, plate the reaction mix on ice for 2-5 min before proceeding to transformation step. Do not let the samples stay on ice for more than 5 min before transformation.</LI></OL><H6>Transformation:</H6><OL><LI>Use 3ul of the seamless cloning and assembly reaction to transform NEB5α cells. Do not use electrocompetent cells.</LI><LI>Follow normal Transformation procedures.</LI><LI>Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE) </LI><LI>Incubate on ice for 30 min.</LI><LI>Incubate at 42°C for 45 sec.</LI><LI>Incubate on ice for at least 2 min.</LI><LI>Add 250ul SOC Media (Make sure it isn't cloudy) to cells.</LI><LI>Shake in 37°C Incubator Room for 1 hr. (Warm plates also)</LI><LI>Plate on LB-CARB and keep in drawer over the weekend.</LI></OL><H3>Miniprep of Transformations (PRM2, PRM6, SAG1)</H3><UL><LI> Used cultures made on 7/18/14 (put in incubator on 7/20/14)</LI></UL><H4>Miniprep Precedure:</H4><UL><LI>Obtain E.Coli Cultures from 06/16/14 in 37°C  Incubation Room.</LI></UL><OL><LI>Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.</LI><LI>Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.</LI><LI>Add 350 µl Buffer N3 and invert immediately 4-6 times.</LI><LI>Centrifuge for 10 min at 13,000 rpm.</LI><LI>Add supernatant to QIAprep spin column.</LI><LI>Centrifuge for 30-60s - Discard flow through.</LI><LI>Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.</LI><LI>Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s</LI><LI>Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.</LI><LI><P>Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.</P><TABLE><THEAD><TR><TH>Promoter</TH><TH>Concentration</TH></TR></THEAD><TBODY><TR><TD>PRM2</TD><TD>225.0 ng/ul</TD></TR><TR><TD>PRM6</TD><TD>491.5 ng/ul</TD></TR><TR><TD>SAG1</TD><TD>439.0 ng/ul</TD></TR></TBODY></TABLE></LI></OL><UL><LI>Stored in my box in -20°C fridge.</LI></UL><H5>Sent into sequencing.</H5><UL><LI>Anasuya came in and talked to us about some useful articles to read: mine is the one about &quot;Binding of the B cell Activation...&quot;</LI></UL><H2>07/22/14</H2><H3>Results of Transformation</H3><UL><LI><P>Seamless Transformation from 7/21/14</P><PRE><CODE>| Promoter | Colonies|                  
|----------|---------|
| PRM1     |  some   |
| PRM2     |  some   |
| PRM3     |    0    |
| PRM6     |    0    |
| ECM18    |    0    |
| SAG1     |    0    |
| YDR124W  |    0    |
| CLG1     |    0    |
| ASG7     |    0    |
| HYM1     |    0    |
| PCL2     |    0    |
| Pos Ctrl |  some   |
| Neg Ctrl |  some   |
</CODE></PRE></LI></UL><H5>Possible Explanations:</H5><UL><LI>Used wrong inserts</LI><LI>Seamless went wrong</LI><LI>Mutations of template</LI><LI>Backbones not digested properly (not likely)</LI><LI>PRM1 and PRM2 backbones have GFP</LI><LI>Concentrations of both insert and backbone too low</LI></UL><H5>Final Concensus: We will use another method to transform the cells:</H5><UL><LI>Instead of using Gibson or Seamless, we will PCR with primers for all 7kb.</LI><LI>Currently ordering primers.</LI></UL><P>Meanwhile, transformations from 7/18/14 worked - colonies grew for all plates</P><UL><LI>will attempt colony PCR to determine the presence of rtTA</LI><LI>plates to be PCR'ed: PRM3, ECM18, YDR124W, CLG1, ASG7, HYM1, and PCL2</LI><LI>plates to be excluded: PRM1 and PRM2 (both backbones have GFP), PRM6 (has both mutation and rtTA, determined by previous sequencing), SAG1 (backbone has GFP but has recently been redigested), '+' control, '-' control</LI></UL><P>Culturing pGEM #1-11 (except pGEM7) to make pGEM for backbone digestion tomorrow.</P><H2>07/23/14</H2><H3>pGEM Miniprep and Overnight Backbone Digestion</H3><UL><LI>Collected cultures from 37°C incubator room - shaker was off for some reason - I turned it back on</LI><LI>Cells grew ok despite shaker incident</LI><LI>pGEM 4 and pGEM6 did not grow very well</LI><LI>Did a miniprep and gel extraction for backbone digestion</LI></UL><H2>07/24/14</H2><H3>Digested Backbones</H3><UL><LI>Did a gel extract.</LI></UL><P>Concentrations of Backbones:</P><OL><LI>PRM1 = 60.10 ng/ul</LI><LI>PRM2 = 61.79 ng/ul</LI><LI>PRM3 = 41.36 ng/ul</LI><LI>PRM6 = 55.34 ng/ul</LI><LI>ECM18 = 55.11 ng.ul</LI><LI>SAG1 = 89.55 ng/ul</LI><LI>YDR124W = 35.55 ng/ul</LI><LI>CLG1 = 49.42 ng/ul</LI><LI>ASG7 = 40.27 ng/ul</LI><LI>HYM1 = 33.18 ng/ul</LI><LI>PCL2 = 29.47 ng/ul</LI></OL><UL><LI>Backbones stored in my bix in -20°C fridge.</LI></UL><H2>07/25/14</H2><H3>Transformation</H3><UL><LI>Collected overnight digests of rtTA and Msn2 from 37°C incubator room.</LI></UL><H4>PCR Purification</H4><OL><LI>5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample</LI><LI>To bind DNA, apply sample to column and spin for 1 min</LI><LI>Discard flow through </LI><LI>To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min</LI><LI>Discard flow through and place column back in the same tube</LI><LI>Cetrifuge for an addtitional minute (Dry Spin)</LI><LI>Place column in clean 1.5 ml tube</LI><LI>Add 30µl water to column, let stand for 1 min and centrifuge for 1 min </LI></OL><UL><LI>Concentrations of digests:

</LI><LI>rtTA = 28.98 ng/ul</LI><LI>Msn2 = 34.86 ng/ul</LI></UL><H3>Ligation</H3><P>1:3 Backbone to Insert</P><P>Materials:</P><TABLE><THEAD><TR><TH>Reagents</TH><TH>1x</TH></TR></THEAD><TBODY><TR><TD>10x ligase buffer</TD><TD>1 µl</TD></TR><TR><TD>Backbone</TD><TD>2 µl</TD></TR><TR><TD>Msn2</TD><TD>1 µl</TD></TR><TR><TD>rtTA</TD><TD>1 ul</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 µl</TD></TR><TR><TD>H2O</TD><TD>4 µl</TD></TR><TR><TD>Total</TD><TD>10 µl</TD></TR></TBODY></TABLE><P>Keep at room temperature for at least 2 hours.</P><H3>Transformation</H3><TABLE><THEAD><TR><TH>Materials</TH><TH>1x</TH></TR></THEAD><TBODY><TR><TD>Ligation</TD><TD>10µl</TD></TR><TR><TD>E. Coli Competent Cells</TD><TD>25ul</TD></TR></TBODY></TABLE><OL><LI>Mix both together and keep on ice for 30 minutes.</LI><LI>42°C Heat shock for 45 seconds.</LI><LI>Put immediately on ice for at least 2 minutes.</LI><LI>Add 250 ul of SOC media, shake for 1 hr at 37°C.</LI><LI>Plate on selective media and store at room temperature over the weekend. </LI></OL><UL><LI>Had extra cells, so I added 50ul to PCL2.</LI></UL><H2>07/29/14</H2><H3>Colony PCR of AFRP + rtTA</H3><UL><LI><P>Use rtTA FW primer instead of individual promoter primers</P></LI><LI><P>Cultures to use for Colony PCR: PRM2-1, PRM3-2, PRM6-1, ECM18-1, SAG1-3, ASG7-2, HYM1-3</P></LI><LI>use plates for the rest of the AFRPs</LI></UL><LI><P>Use primers 40 and 41, rtTA FW and RV</P></LI><H5>Colony PCR Procedure:</H5><OL><LI>Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 25ul of water.Do this for 5 colonies. Use 5ul for PCR reaction below. </LI><LI>Set up PCR Reaction below:</LI></OL><TABLE><THEAD><TR><TH>Reagents</TH><TH>1X</TH><TH>34X</TH></TR></THEAD><TBODY><TR><TD>2X GoTaq Green PCR Master Mix</TD><TD>10 µl</TD><TD>340 µl</TD></TR><TR><TD>10 µM Forward primer</TD><TD>1 µl</TD><TD>34 µl</TD></TR><TR><TD>10 µM Reverse primer</TD><TD>1 µl</TD><TD>34 µl</TD></TR><TR><TD>Water</TD><TD>1.5µl</TD><TD>51 µl</TD></TR><TR><TD>Bacterial cells (template)</TD><TD>5 µl</TD><TD>------</TD></TR><TR><TD>DMSO</TD><TD>1.5 µl</TD><TD>51 ul</TD></TR></TBODY></TABLE><UL><LI><P>Add 15 ul of Master Mix into each tube.</P><PRE><CODE>Initial Denaturation   | 95° C | 5 min 
  35 Cycles of: 
      Denaturation | 95° C | 45s
      Annealing    | 55° C | 30s
      Extension    | 72° C | 1min per kb
Final Extension    | 72° C | 10m
Hold               |  4°C  | Forever
</CODE></PRE></LI><LI><P>Load PCR products on a gel. (5ul) (GoTaq already has loading dye)</P></LI><LI><P>For all positive bands on the gel, add rest of bacterial cells from PCR tubes to 5ml LB and grow overnight at 37°C for miniprep on the following day. </P></LI></UL><H5>Gel Map of Colony PCR:</H5><H2>07/30/14</H2><H3>Re-Colony PCR of SAG1</H3><UL><LI>Redid Colony PCR for SAG1</LI></UL><H3>Miniprep of Colony PCR</H3><UL><LI>Cultures to be miniprep-ed: 

</LI><LI>PRM1.6</LI><LI>PRM2.1</LI><LI>PRM3.2</LI><LI>PRM6.4</LI><LI>ECM18.1</LI><LI>YDR124W.6</LI><LI>CLG1.5</LI><LI>ASG7.4</LI><LI>HYM1.3</LI><LI>PCL2.6</LI></UL><OL><LI>Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.</LI><LI>Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.</LI><LI>Add 350 µl Buffer N3 and invert immediately 4-6 times.</LI><LI>Centrifuge for 10 min at 13,000 rpm.</LI><LI>Add supernatant to QIAprep spin column.</LI><LI>Centrifuge for 30-60s - Discard flow through.</LI><LI>Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.</LI><LI>Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s</LI><LI>Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.</LI><LI>Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.</LI></OL><H2>07/31/14</H2><H3>Sequencing of AFRP + rtTA</H3><UL><LI><P>Sequencing of 7/30/14 Minipreps</P></LI><LI>Sent in: PRM1, PRM2, PRM3, PRM6, ECM18, YDR124W, CLG1, ASG7, HYM1, PCL2</LI><LI>SAG1 on hold - PRM6 and PCL2 need to be re-Colony PCRed.</LI></UL><LI><P>Did a Colony PCR for PRM6, SAG1, and PCL2</P></LI><LI>Made cultures with 5ml LB+CARB and 20ul cells</LI><LI>PRM6 = 7, 8, 9</LI><LI>SAG1 = 10, 11, 12</LI><LI>PCL2 = 7, 8, 9 (plate may have been contaminated, used white colonies)</LI><LI><P>Colony PCR failed - no rtTA - decided to re-transform</P></LI><LI><P>Ligation </P></LI><LI>see procedure above</LI><LI>stored at 16°C overnight (room temperature)</LI><H2>08/01/14</H2><H3>Transformation of Ligation into NEB5α (C2987) cells</H3><UL><LI><P>see procedure above - used 25ul cells - stored in 37°C incubator room
-Transformed PCL2, PRM6, SAG1, and negative control</P></LI><LI><P>Yeast Transformation </P></LI><LI>linearized DNA</LI><LI>boiled salmon sperm at 100°C for 10 min</LI></UL><H5>Yeast Transformation Procedure:</H5><TABLE><THEAD><TR><TH>Reagents</TH></TR></THEAD><TBODY><TR><TD>YPD</TD></TR><TR><TD>1 M LiOAc</TD></TR><TR><TD>10X TE pH 7.5</TD></TR><TR><TD>1X TE pH 7.5, 0.1 M LiOAc</TD></TR><TR><TD>50% PEG 3350</TD></TR><TR><TD>DMSO</TD></TR><TR><TD>Salmon Sperm DNA (ssDNA)</TD></TR></TBODY></TABLE><UL><LI>PEG is viscous, so pipette slowly to prevent air bubbles from forming.</LI><LI>boil ssDNA aliquots for 10 min, then ice down for 10 min before use.</LI></UL><P>Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C</P><OL><LI>Set up digest to linearize DNA </LI><LI>Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C</LI><LI>Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)</LI><LI>Harvest cells in centrifuge - 3000 rpm, 2-5 min</LI><LI>Wash with 1 ml 0.1 M LiOAin TE</LI><LI>Pellet cells - 3000 rpm , 2-5 min</LI><LI>Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation</LI><LI>to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA</LI><LI>Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc)  <EM>Optional</EM>: Add 75 µl DMSO <EM>definitely do</EM></LI><LI>Vortex</LI><LI>Incubate 42° C for 30m &amp; begin drying plates</LI><LI>Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)</LI><LI>Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD</LI><LI>Plate on selective media</LI><LI>Incubate 1-3 days at 30°C</LI></OL><H2>08/04/14</H2><H3>Results of E.Coli and Yeast Transformations</H3><UL><LI><P>E.Coli plates:</P></LI><LI>PRM6 = 30</LI><LI>SG1 = 20</LI><LI>PCL2 = 20</LI><LI>Negative Control = 20</LI></UL><LI><P>Yeast plates:</P></LI><LI>CB008: all plates grew at least 30+ colonies </LI><LI>CB008DB: all plates grew at least 30+ colonies</LI><LI>YPD became contaminates - plates may be compromised</LI><H5>Did a Colony PCR on successful E.Coli plates.</H5><UL><LI>FW primer = rtTA FW #40</LI><LI>RV primer = rtTA RV #41</LI><LI>cultures = 5ml LB+CARB + 20ul cells</LI></UL><H5>SD, YPD, and LB were contaminated, as a result, some of our yeast plates became contaminated.</H5><UL><LI>Jessica did a Colony PCR for the successful Yeast Plates</LI><LI>Used the uncontaminated colonies for Colony PCR</LI></UL><H5>See Colony PCR procedures for both E.Coli and Yeast above.</H5><H2>08/05/14</H2><H3>Miniprep of E.Coli Cultures</H3><UL><LI>Cultures = PRM6, SAG1, PCL2

</LI><LI>AFRP+rtTA</LI></UL><H4>Miniprep Precedure:</H4><UL><LI>Obtain E.Coli Cultures from 8/4/14 in 37°C  Incubation Room.</LI></UL><OL><LI>Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.</LI><LI>Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.</LI><LI>Add 350 µl Buffer N3 and invert immediately 4-6 times.</LI><LI>Centrifuge for 10 min at 13,000 rpm.</LI><LI>Add supernatant to QIAprep spin column.</LI><LI>Centrifuge for 30-60s - Discard flow through.</LI><LI>Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.</LI><LI>Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s</LI><LI>Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.</LI><LI>Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.</LI></OL><H5>Miniprep Concentrations:</H5><UL><LI>PRM6 = 611.2 ng/ul</LI><LI>SAG1 = 478.1 ng/ul</LI><LI>PCL2 = 384.8 ng/ul</LI></UL><H5>Sent into sequencing</H5><H3>Linearization of E.Coli Minipreps</H3><P>DIgestion Materials:</P><TABLE><THEAD><TR><TH>Reaction</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>DNA</TD><TD>~2000ng</TD></TR><TR><TD>Pme1</TD><TD>0.5ul</TD></TR><TR><TD>CutSmart</TD><TD>2.5ul</TD></TR><TR><TD>H2O</TD><TD>x ul</TD></TR><TR><TD>Total</TD><TD>25ul</TD></TR></TBODY></TABLE><P>Concentration Calculation:</P><TABLE><THEAD><TR><TH>Promoter</TH><TH>Concentration of Miniprep</TH><TH>Amount of Miniprep Needed</TH><TH>H2O</TH></TR></THEAD><TBODY><TR><TD>PRM1</TD><TD>376.7 ng/ul</TD><TD>6 ul</TD><TD>16 ul</TD></TR><TR><TD>PRM2</TD><TD>356.8 ng/ul</TD><TD>6 µl</TD><TD>16 ul</TD></TR><TR><TD>PRM3</TD><TD>364.5 ng/ul</TD><TD>6 µl</TD><TD>16 ul</TD></TR><TR><TD>PRM6</TD><TD>611.2 ng/ul</TD><TD>4 ul</TD><TD>18 ul</TD></TR><TR><TD>ECM18</TD><TD>371.7 ng/ul</TD><TD>6 µl</TD><TD>16 ul</TD></TR><TR><TD>SAG1</TD><TD>478.1 ng/ul</TD><TD>5 µl</TD><TD>17 ul</TD></TR><TR><TD>YDR124W</TD><TD>328.4 ng/ul</TD><TD>6 µl</TD><TD>16 ul</TD></TR><TR><TD>CLG1</TD><TD>543.1 ng/ul</TD><TD>4 µl</TD><TD>18 ul</TD></TR><TR><TD>ASG7</TD><TD>348.8 ng/ul</TD><TD>6 µl</TD><TD>16 ul</TD></TR><TR><TD>HYM1</TD><TD>449.6 ng/ul</TD><TD>5 µl</TD><TD>17 ul</TD></TR><TR><TD>PCL2</TD><TD>384.8 ng/ul</TD><TD>6 µl</TD><TD>16 ul</TD></TR></TBODY></TABLE><UL><LI>Add reagents and incubate for 2 hours at 37°C  Incubation Room.</LI><LI>Also assisted Jeffrey in lineaerizing 11E3128 (423.0 ng/ul) and pGEM22 (384.8 ng/ul).</LI></UL><H5>Transformed linearizations into yeast</H5><UL><LI>CB008: PRM6, ECM18, SAG1, PCL2</LI><LI>CB008DB: all 11 promoters</LI><LI>Forgot to make negative control</LI></UL><H5>Yeast Transformation Procedure:</H5><TABLE><THEAD><TR><TH>Reagents</TH></TR></THEAD><TBODY><TR><TD>YPD</TD></TR><TR><TD>1 M LiOAc</TD></TR><TR><TD>10X TE pH 7.5</TD></TR><TR><TD>1X TE pH 7.5, 0.1 M LiOAc</TD></TR><TR><TD>50% PEG 3350</TD></TR><TR><TD>DMSO</TD></TR><TR><TD>Salmon Sperm DNA (ssDNA)</TD></TR></TBODY></TABLE><UL><LI>PEG is viscous, so pipette slowly to prevent air bubbles from forming.</LI><LI>boil ssDNA aliquots for 10 min, then ice down for 10 min before use.</LI></UL><P>Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C</P><OL><LI>Set up digest to linearize DNA </LI><LI>Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C</LI><LI>Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)</LI><LI>Harvest cells in centrifuge - 3000 rpm, 2-5 min</LI><LI>Wash with 1 ml 0.1 M LiOAin TE</LI><LI>Pellet cells - 3000 rpm , 2-5 min</LI><LI>Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation</LI><LI>to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA</LI><LI>Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc)  <EM>Optional</EM>: Add 75 µl DMSO <EM>definitely do</EM></LI><LI>Vortex</LI><LI>Incubate 42° C for 30m &amp; begin drying plates</LI><LI>Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)</LI><LI>Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD</LI><LI>Plate on selective media</LI><LI>Incubate 1-3 days</LI></OL><H2>08/08/14</H2><H3>Results of Transformation</H3><UL><LI>CB008 = all plates grew 20-30 colonies except for PRM6, which did not grow.</LI><LI>CB008DB = all plates grew 20-30 colonies</LI><LI>Both plates had small colonies</LI></UL><H5>Did a Colony PCR on the Transformations</H5><H3>Flow Testing</H3><H5>CB008 Const+rtTA+GFP+mfa+pAGA mCherry</H5><P>Plate map for all time points (4 plates):</P><PRE><CODE>  1   2   3   4   5   6   7   8   9   10  11  12
A [CB008 m6-1]----]       [CB008 m10-3]----]
B [CB008 m6-2]----]       [CB008DB m7-1]---]
C [CB008 m6-3]----]       [CB008DB m7-2]---]
D [CB008 m7-1]----]       [CB008DB m7-3]---]
E [CB008 m7-2]----] 
F [CB008 m7-3]----]           
G [CB008 m10-1]---] 
H [CB008 m10-2]---]
</CODE></PRE><P>[Dox] Concentration Map:</P><PRE><CODE>        H   G   F   E   D   C   B   A   
1       [--------0 µg/ml------------]
2       [--------0.03 µg/ml---------]
3       [--------0.06 µg/ml---------]
4       [--------0.09 µg/ml---------]
5       [--------0.6 µg/ml----------]
6 
7       [--------0 µg/ml------------]
8       [--------0.03 µg/ml---------]
9       [--------0.06 µg/ml---------]
10      [--------0.09 µg/ml---------]
11      [--------0.6 µg/ml----------]
12                     
</CODE></PRE><UL><LI>See notes on flow in Eric's notebook.</LI></UL><H2>08/12/14</H2><H3>Colony PCR of Yeast for rtTA and mfa</H3><UL><LI>Boil Yeast at 95°C for 20 minutes.</LI></UL><OL><LI>Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 15ul of NaOH. Take 3 colonies from each plate. Use 5ul for PCR reaction below. </LI><LI>Set up PCR reaction below: </LI></OL><TABLE><THEAD><TR><TH>Reagents (rtTA)</TH><TH>1X</TH><TH>43X</TH></TR></THEAD><TBODY><TR><TD>2X GoTaq Green PCR Master Mix</TD><TD>10 µl</TD><TD>430 µl</TD></TR><TR><TD>10 µM Forward primer</TD><TD>1 µl</TD><TD>43 µl</TD></TR><TR><TD>10 µM Reverse primer</TD><TD>1 µl</TD><TD>43 µl</TD></TR><TR><TD>Water</TD><TD>1.5µl</TD><TD>64.5 µl</TD></TR><TR><TD>cells (template)</TD><TD>5 µl</TD><TD>-------</TD></TR><TR><TD>DMSO</TD><TD>1.5ul</TD><TD>64.5 ul</TD></TR></TBODY></TABLE><UL><LI>FW primer = pSV606</LI></UL><TABLE><THEAD><TR><TH>Reagents (rtTA)</TH><TH>1X</TH><TH>7X</TH></TR></THEAD><TBODY><TR><TD>2X GoTaq Green PCR Master Mix</TD><TD>10 µl</TD><TD>70 µl</TD></TR><TR><TD>10 µM Forward primer</TD><TD>1 µl</TD><TD>7 µl</TD></TR><TR><TD>10 µM Reverse primer</TD><TD>1 µl</TD><TD>7 µl</TD></TR><TR><TD>Water</TD><TD>1.5µl</TD><TD>10.5 µl</TD></TR><TR><TD>cells (template)</TD><TD>5 µl</TD><TD>-------</TD></TR><TR><TD>DMSO</TD><TD>1.5ul</TD><TD>10.5 ul</TD></TR></TBODY></TABLE><UL><LI><P>Add 15 ul of Master Mix into each tube.</P><PRE><CODE>Initial Denaturation    | 95°C | 5 min 
  35 Cycles of: 
      Denaturation | 95°C | 45s
      Annealing    | 55°C | 30s
      Extension    | 72°C | 1min per kb
Final Extension    | 72°C | 10m
Hold               |  4°C | Forever
</CODE></PRE></LI><LI><P>Load PCR products on a gel. (5ul) (GoTaq already has loading dye)</P></LI></UL><H5>Results of Colony PCR:</H5><H2>08/13/14</H2><H3>Doxycyline Dilutions (Dilutions by Ianto)</H3><UL><LI><P>Stock in Siberia Fridge (Middle Shelf, Middle Rack, Top Drawer, 3rd Box) 100 mg/ml</P><TABLE><THEAD><TR><TH>x</TH><TH>Concentration</TH><TH>Take from x</TH><TH>Amount of x Needed</TH><TH>H2O Needed</TH></TR></THEAD><TBODY><TR><TD>A</TD><TD>60</TD><TD>Stock</TD><TD>60 ul</TD><TD>940 ul</TD></TR><TR><TD>B</TD><TD>30</TD><TD>Stock</TD><TD>30 ul</TD><TD>970 ul</TD></TR><TR><TD>C</TD><TD>9</TD><TD>A</TD><TD>150 ul</TD><TD>850 ul</TD></TR><TR><TD>D</TD><TD>6</TD><TD>A</TD><TD>50 ul</TD><TD>900 ul</TD></TR><TR><TD>E</TD><TD>3</TD><TD>A</TD><TD>50 ul</TD><TD>950 ul</TD></TR><TR><TD>F</TD><TD>0.9</TD><TD>C</TD><TD>100 ul</TD><TD>900 ul</TD></TR><TR><TD>G</TD><TD>0.6</TD><TD>D</TD><TD>100 ul</TD><TD>900 ul</TD></TR><TR><TD>H</TD><TD>0.3</TD><TD>E</TD><TD>100 ul</TD><TD>900 ul</TD></TR><TR><TD>I</TD><TD>0.09</TD><TD>F</TD><TD>100 ul</TD><TD>900 ul</TD></TR><TR><TD>J</TD><TD>0.06</TD><TD>G</TD><TD>100 ul</TD><TD>900 ul</TD></TR><TR><TD>K</TD><TD>0.03</TD><TD>H</TD><TD>100 ul</TD><TD>900 ul</TD></TR></TBODY></TABLE></LI><LI><P>Vortex while like making and before adding to each dilution.</P></LI></UL><H5>Assisted Derrick in running plates (time points: 0, 1.5, 3, 5)</H5><UL><LI>See Derrick's lab norebook for reference.</LI></UL><H2>08/17/14</H2><UL><LI>Due to contamination of the plates from the flow run of 8/13/14, another run will be needed of the same experiment.

</LI><LI>Plate had large clumps of cells due to not being cleaned out well.</LI><LI>Make sure to put finished shaker plate in bleach bin (near Hyun's desk, under the table in a blue bin)</LI></UL><P>prepared cultures of pTEF1, M6, M7, and M10 to be diluted on the following day for flow.</P><UL><LI>CB008 const+rtTA+GFP+mfa+pAGA1+mCherry</LI></UL><H5>Plate Map</H5><TABLE><THEAD><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR></THEAD><TBODY><TR><TD>A</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR><TR><TD>B</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>C</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>D</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>E</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD></TR><TR><TD>F</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD></TR><TR><TD>G</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR><TR><TD>H</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR></TBODY></TABLE><P>1 = PTEF1
6 = M6
7 = M7
10 = M10</P><P>[Dox] Concentration Map:</P><PRE><CODE>        H   G   F   E   D   C   B   A   
1       [--------0 µg/ml------------]
2       [--------0.03 µg/ml---------]
3       [--------0.06 µg/ml---------]
4       [--------0.09 µg/ml---------]
5       [--------0.6 µg/ml----------]
6       [----------6 µg/ml----------]
7       [--------0 µg/ml------------]
8       [--------0.03 µg/ml---------]
9       [--------0.06 µg/ml---------]
10      [--------0.09 µg/ml---------]
11      [--------0.6 µg/ml----------]
12      [----------6 µg/ml----------]
</CODE></PRE><H2>08/18/14</H2><H3>CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2</H3><P>Ran plate with 5 time points (0, 1.5, 3, 5, 8)</P><UL><LI>added 15ul of dox to induce </LI><LI>Added 10ul of cycloheximide to each well to stop growth.</LI></UL><P>Finished running the first two plates (0hr and 1.5hr)</P><UL><LI>Slow run may be due to the cells being overdilute. (source: Kara)</LI><LI>Plate run to be continued and finished on 8/19/14</LI></UL><H3>CB008DB Const+rtTA+GFP+mfa+pAGA+mCherry</H3><P>Started cultures for this experiment.</P><UL><LI>To be ran on 8/20/14</LI></UL><P>All 5 Const. Promoters available.</P><TABLE><THEAD><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR></THEAD><TBODY><TR><TD>A</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD></TR><TR><TD>B</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR><TR><TD>C</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR><TR><TD>D</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR><TR><TD>E</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>F</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>G</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>H</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD></TR></TBODY></TABLE><P>1 = PTEF1
3 = M3
6 = M6
7 = M7
10 = M10</P><P>[Dox] Concentration Map:</P><PRE><CODE>        H   G   F   E   D   C   B   A   
1       [--------0 µg/ml------------]
2       [--------0.03 µg/ml---------]
3       [--------0.06 µg/ml---------]
4       [--------0.09 µg/ml---------]
5       [--------0.6 µg/ml----------]
6       [----------6 µg/ml----------]
7       [--------0 µg/ml------------]
8       [--------0.03 µg/ml---------]
9       [--------0.06 µg/ml---------]
10      [--------0.09 µg/ml---------]
11      [--------0.6 µg/ml----------]
12      [----------6 µg/ml----------]
</CODE></PRE><H2>08/20/14</H2><H3>CB008DB Const+rtTA+GFP+pAGA1+mCherry Run</H3><P>Ran plate with 5 time points (0, 1.5, 3, 5, 8)</P><UL><LI><P>Ran first plate and noticed that specimen 5 only had 10ul loaded due to the addition of a new speciment (parameters were reset) </P></LI><LI>Readjusted the parameters and reran the well that only had 10ul loaded </LI><LI>Make sure to check parameters for all wells before running plate.</LI></UL><LI><P>Stopped plate run at 3hr plate - D7</P></LI><H2>08/21/14</H2><H3>CB008DB Const+rtTA+GFP+pAGA1+mCherry Run (Cont.)</H3><UL><LI>Continued running 3hr plate from 8/20/14</LI><LI>Finished 3 and 5 hr plates but could not do 8 hr plate because the plate was dropped.</LI></UL><H5>Hyun said not to run DB plates</H5><H2>08/25/14</H2><H3>CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Preparation</H3><H5>Plate Map</H5><TABLE><THEAD><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH><TH>12</TH></TR></THEAD><TBODY><TR><TD>A</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR><TR><TD>B</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>C</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>D</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>E</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD></TR><TR><TD>F</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD><TD>6</TD></TR><TR><TD>G</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR><TR><TD>H</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR></TBODY></TABLE><P>1 = PTEF1
6 = M6
7 = M7
10 = M10</P><UL><LI>Grew cultures and put into 30°C incubator.</LI></UL><P>[Dox] Concentration Map:</P><PRE><CODE>        H   G   F   E   D   C   B   A   
1       [--------0 µg/ml------------]
2       [--------0.03 µg/ml---------]
3       [--------0.06 µg/ml---------]
4       [--------0.09 µg/ml---------]
5       [--------0.6 µg/ml----------]
6       [----------6 µg/ml----------]
7       [--------0 µg/ml------------]
8       [--------0.03 µg/ml---------]
9       [--------0.06 µg/ml---------]
10      [--------0.09 µg/ml---------]
11      [--------0.6 µg/ml----------]
12      [----------6 µg/ml----------]
</CODE></PRE><H2>08/26/14</H2><H3>CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Run</H3><UL><LI>Was only able to finish running 0 hr plate and 5 hr plate up to C2.</LI><LI>Wells D4 and G3 on plates should not have sample due to error during dilutions.</LI><LI>All of M10 seems to not have grown compared to the other strains.</LI></UL><H2>08/27/14</H2><H3>CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Run (Cont.)</H3><UL><LI>Accidentally did not run the last specimen (M10) on the 5hr plate due to Eric's advice on not wasting time because M10 did not grow.</LI></UL><H3>Re-streaking of old plates</H3><UL><LI>Started restreaking old streak plates onto organized plates. (organized by pGEM #s)</LI></UL><H2>08/28/14</H2><H3>Data Analysis of CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Run</H3><UL><LI>After using FlowJo and Matlab to analyze data, we have come to the conclusion that CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry does not have RFP. </LI><LI>Compared pPCL2 RFP to pAGA1 RFP runs. pAGA1 has much RFP levels.</LI></UL><P>Prepared cultures of CB008DB Const+rtTA+GFP+mfa+pPCL2+mCherry M3, M6, M7, and M10 to be ran on the following day to confirm the presence or lack of presence of RFP.</P><P>pTEF1 missing, may need to grow a glycerol stock streak plate of it.</P><H2>09/02/14</H2><H3>CB008DB Const+rtTA+GFP+mfa+pPCL2+mCherry Run</H3><UL><LI>Shaker plate not completely clean, I attemped to clean it out with water. Saw chunks of dried cells on the bottom of some of the wells. MAKE SURE YOU PUT THE SHAKER PLATE IN THE BLEACH BIN AFTER USE!!!</LI><LI>M10 may be contaminated due to the shaker plate.</LI><LI>M6 culture was not labeled correctly in the incubator shaker, M6 may not actually be the right specimen.</LI></UL><P>Ran 2 time points: 0 hr and 5 hrs.</P><UL><LI>Data to be analyzed in the near future after asking Ianto how to run only two time points.</LI></UL><H2>09/04/14</H2><H3>Co-culture of CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry (pTEF1, M7, M10)</H3><P>Plate Map: </P><UL><LI>A1-A6 = pTEF1+M7</LI><LI>B1-B6 = pTEF1+M7</LI><LI>C1-C6 = pTEF1+M7</LI><LI>D1-D6 = M7+M10</LI><LI>E1-E6 = M7+M10</LI><LI>F1-F6 = M7+M10</LI><LI>G1-G6 = M10+pTEF1</LI><LI>H1-H6 = M10+pTEF1</LI><LI>A7-A12 = M10+pTEF1</LI></UL><P>Dox Map:</P><UL><LI>Lane 1 = 0</LI><LI>Lane 2 = 0.03</LI><LI>Lane 3 = 0.06</LI><LI>Lane 4 = 0.09</LI><LI>Lane 5 = 0.6</LI><LI>Lane 6 = 6</LI><LI>Lane 7 = 0</LI><LI>Lane 8 = 0.03</LI><LI>Lane 9 = 0.06</LI><LI>Lane 10 = 0.09</LI><LI>Lane 11 = 0.6</LI><LI><P>Lane 12 = 6</P></LI><LI><P>Grew cultures of CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry</P></LI><LI>pTEF1</LI><LI>M7</LI><LI>M10

</LI><LI>stored in 30°C incubator shaker.</LI></UL><H2>09/05/14</H2><H3>Co-culture of CB008 Const+rtTA+GFP+mfa+pAGA+mCherry Run</H3><UL><LI><P>5 Timepoints: 0, 1.5, 3, 5, 8.</P></LI><LI><P>Successfully ran all the plates.</P></LI></UL><H2>09/10/14</H2><H3>Data Analysis of Co-culture of CB008 Const+rtTA+GFP+mfa+pAGA+mCherry Run</H3><UL><LI>After running the data on FlowJo and Matlab, we compared the data to the original pAga1+mCherry pTEF1, M7, and M10 data. </LI><LI><P>The comparison showed slight variation and unexpected results. </P></LI><LI>Maybe due to the larger cells being too stressed from producing so many proteins, and the overall population producing less as a result of this.</LI></UL><LI><P>After comparing the data, it was decided that the best option would be to rerun this experiment but with pTEF1, M7, and M10 separately put on the plate as a comparison.</P></LI><LI><P>Started growing cultures of pTEF1, M7, and M10 in the incubator.</P></LI><H2>09/11/14</H2><H3>Co-Culture of CB009 Const+rtTA+GFP+mfa+pAGA+mCherry Round 2</H3><P>Plate Map:</P><UL><LI>A1-A6 = pTEF1</LI><LI>B1-B6 = pTEF1</LI><LI>C1-C6 = pTEF1</LI><LI>D1-D6 = M7</LI><LI>E1-E6 = M7</LI><LI>F1-F6 = M7</LI><LI>G1-G6 = M10</LI><LI>H1-H6 = M10</LI><LI>A7-A12 = M10</LI><LI>B7-B12 = M7+pTEF1</LI><LI>C7-C12 = M7+pTEF1</LI><LI>D7-D12 = M7+pTEF1</LI><LI>E7-E12 = M7+M10</LI><LI>F7-F12 = M7+M10</LI><LI>G7-G12 = M7+M10</LI></UL><P>Dox Map:</P><UL><LI>Lane 1 = 0</LI><LI>Lane 2 = 0.03</LI><LI>Lane 3 = 0.06</LI><LI>Lane 4 = 0.09</LI><LI>Lane 5 = 0.6</LI><LI>Lane 6 = 6</LI><LI>Lane 7 = 0</LI><LI>Lane 8 = 0.03</LI><LI>Lane 9 = 0.06</LI><LI>Lane 10 = 0.09</LI><LI>Lane 11 = 0.6</LI><LI>Lane 12 = 6</LI></UL><H5>Did not make a co-culture of pTEF1+M10 because the two promoters are too close in activity to make much of a difference.</H5><H2>09/12/14</H2><H3>Attempt of Co-Culture CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2 Run</H3><UL><LI>Cancelled due to wrong culture being grown and due to the absence of induction.</LI><LI>Make sure that Eleanor and George pay attention when growing cultures and when inducing with doxycycline.</LI><LI>pPCL2 was grown instead of pAGA1. pPCL2 cannot be used in a co-culture due to its lack of RFP. </LI></UL><P>Experiment to be continued at another time.</P><H2>09/14/14</H2><H3>Co-Culture CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2 Preparation</H3><P>Plate Map:</P><UL><LI>A1-A6 = pTEF1</LI><LI>B1-B6 = pTEF1</LI><LI>C1-C6 = pTEF1</LI><LI>D1-D6 = M7</LI><LI>E1-E6 = M7</LI><LI>F1-F6 = M7</LI><LI>G1-G6 = M10</LI><LI>H1-H6 = M10</LI><LI>A7-A12 = M10</LI><LI>B7-B12 = M7+pTEF1</LI><LI>C7-C12 = M7+pTEF1</LI><LI>D7-D12 = M7+pTEF1</LI><LI>E7-E12 = M7+M10</LI><LI>F7-F12 = M7+M10</LI><LI>G7-G12 = M7+M10</LI></UL><P>Dox Map:</P><UL><LI>Lane 1 = 0</LI><LI>Lane 2 = 0.03</LI><LI>Lane 3 = 0.06</LI><LI>Lane 4 = 0.09</LI><LI>Lane 5 = 0.6</LI><LI>Lane 6 = 6</LI><LI>Lane 7 = 0</LI><LI>Lane 8 = 0.03</LI><LI>Lane 9 = 0.06</LI><LI>Lane 10 = 0.09</LI><LI>Lane 11 = 0.6</LI><LI>Lane 12 = 6</LI></UL><H5>Did not make a co-culture of pTEF1+M10 because the two promoters are too close in activity to make much of a difference.</H5><UL><LI>Made sure to use pAGA1 this time. </LI><LI>Cultures growing in yeast incubator.</LI></UL><H2>09/15/14</H2><H3>Co-Culture CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2 Run</H3><P>Time Points: 0, 1.5, 3, 5, 8.</P><UL><LI>Made sure to induce with doxycycline this time.</LI><LI>Also made sure to create the right file name this time.</LI></UL><P>Successfully finished running the plate </P><P>Analyzed the data and made scatter plots</P><H5>Made cultures of AFRP+rtTA+GFP+mfa+pAGA+mCherry</H5><OL><LI>PRM1</LI><LI>PRM2</LI><LI>PRM3</LI><LI>YDR124W</LI><LI>CLG1</LI></OL><H3>AFRP+rtTA+GFP+mfa+pAGA+mCherry Preparation</H3><P>Plate Map:</P><UL><LI>A1-A6 = PRM1</LI><LI>B1-B6 = PRM1</LI><LI>C1-C6 = PRM1</LI><LI>D1-D6 = PRM2</LI><LI>E1-E6 = PRM2</LI><LI>F1-F6 = PRM2</LI><LI>G1-G6 = PRM3</LI><LI>H1-H6 = PRM3</LI><LI>A7-A12 = PRM3</LI><LI>B7-B12 = YDR124W</LI><LI>C7-C12 = YDR124W</LI><LI>D7-D12 = YDR124W</LI><LI>E7-E12 = CLG1</LI><LI>F7-F12 = CLG1</LI><LI>G7-G12 = CLG1</LI></UL><P>Dox Map:</P><UL><LI>Lane 1 = 0</LI><LI>Lane 2 = 0.03</LI><LI>Lane 3 = 0.06</LI><LI>Lane 4 = 0.09</LI><LI>Lane 5 = 0.6</LI><LI>Lane 6 = 6</LI><LI>Lane 7 = 0</LI><LI>Lane 8 = 0.03</LI><LI>Lane 9 = 0.06</LI><LI>Lane 10 = 0.09</LI><LI>Lane 11 = 0.6</LI><LI>Lane 12 = 6</LI></UL><H2>09/16/14</H2><H3>AFRP+rtTA+GFP+mfa+pAGA+mCherry Run</H3><P>Time Points = 0, 1.5, 3, 5, 8.</P><P>Successfully ran all plates.</P><H2>09/17/14</H2><P>Absent due to doctor's appointment.</P><H2>09/18/14</H2><H3>Data Analysis of AFRP+rtTA+GFP+mfa+pAGA+mCherry Run</H3><P>Ran the data on FlowJo and Matlab. </P><UL><LI>Got interesting results like bimodal populations and large amounts of noise.</LI></UL><H3>Eleanor's Transformations</H3><P>Assisted Eleanor in her transformations of CB008 M3 and M6 with pTET+mfa, pAGA1+mCherry, and pPCL2+mCherry.</P><UL><LI>None of her transformations worked other than M3+mfa which had 2 colonies. I put the plates back into the incubator in case they worked.</LI></UL><P>Made cultures of M3+rtTA and M6+mfa to re-transform the following day.</P><H2>09/19/14</H2><H3>Results of Eleanor's Transformations</H3><P>Plates that grew colonies:</P><UL><LI>CB008 M3 pAGA1+mCherry = 1 colony</LI><LI>CB008 M3 mfa = 2 colonies</LI><LI><P>CB008 M6 mfa+pPCL2+mCherry = 4 colonies</P></LI><LI><P>Will do a Colony PCR to check plates - unlikely that plates worked</P></LI><LI><P>Linearized plasmids: mfa and pAGA1+mCherry</P></LI></UL><H5>Kara advises us to not used previously linearized plasmid.</H5><UL><LI>will attempt to use Jeffret's digested pPCL2+mCherry for M6.</LI></UL><P>Re-transformed and put into incubator:</P><UL><LI>CB008 M3+mfa</LI><LI>CB008 M3+pAGA1+mCherry</LI><LI>CB008 M3+mfa+pAGA1+mCherry</LI><LI>CB008 M6+mfa+pAGA1+mCherry</LI><LI>CB008 M6+mfa+pPCL2+mCherry</LI></UL><H2>9/22/14</H2><H3>Colony PCR of Transformations</H3><P>Ran colony PCR's of the Transformations from 9/19 and 9/15 of the CB008 Constitutive Promoters.</P><P>Checked for mfa and pPCL2/pAGA1+mCherry separately.</P><H5>Colony PCR Gel: (Eleanor said that M6 already has mfa even if there is no band)</H5><H2>09/25/14</H2><H3>PCR Amplification of Bar1, mfa, and pGEM1-16 (except 7 and 10)</H3><P>Due to the primers not covering all of the needed insert parts, new primers with new overhangs were ordered. I was put in charge of PCR amplifying the plasmids with these new primers.</P><P>Used primers #103-110.</P><P>mfa Template: Shuaixin's PRM2 + mfa (7.606) Plasmid</P><P>Bar1 Template: Shuaixin's PCL2 + Bar1 (6.62) Plasmid</P><P>George did the PCR for Ste2. </P><P>Used Seamless to transform all the plasmids. George made a pGEM10 PCR insert, which I used for transformation.</P><P>Used pSB1C3 as backbone. Used cleaned up undigested one. </P><H2>09/26/14</H2><H3>Results of Transformation</H3><P>All the pGEM plates transformed very well.</P><P>Bar1, Ste2, and mfa did not transform very well.</P><P>Negative control grew for some reason also, hopefully because of a contamination of PCR product. </P><H3>Colony PCR of Transformation</H3><P>After a few more hours of growing in the incubator, Bar1, Ste2, and mfa grew more cells. </P><P>Did a colony PCR on all the plates (3 colonies from each plate) (a total of 62 Colony PCRs including 5 colonies of Eleanor's Bar1 Transformations)</P><P>Jeffrey ran the gel for my Colony PCR. </P><P>Colony PCR Gel:</P><H2>09/28/14</H2><P>I moved to UC Davis and will be taking a break from lab work to concentrate on classes.</P><P>Kara made cultures from the plates for the successful Colony PCR's to miniprep on Monday.</P><H2>09/29/14</H2><H3>Miniprep of Cultures</H3><P>Eleanor did the miniprep of the cultures from 9/28/14 and sent them into sequencing. </P><H2>09/30/14</H2><H3>Results of Sequencing</H3><P>Sequencing showed that the miniprep was wrong. May have to redo everything or pick new colonies off the plates? Need to discuss with Kara on this matter.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCSF_UCB/sabrina">http://2014.igem.org/Team:UCSF_UCB/sabrina</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCSF_UCB/sabrina" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCSF_UCB/sabrina" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCSF_UCB/sabrina&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCSF_UCB/sabrina&amp;oldid=197598" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>