"
Page
Discussion
View source
History
teams
Log in
 
Team:UCSF UCB/notebooks.html
From 2014.igem.org
Project
Judging
Human Practices
Notebooks & Protocols
Team
Notebooks
Protocols
Lab Members
Eleanor Amidei
Sabrina Chu
Shuaixin He
Jessica Hsueh
Derrick Lee
Jeffrey Shu
Eric Wong
Robert Wong
Ianto Lin Xi
George Yip
Select a Lab Member
Eleanor's Lab Notebook
June 9, 2014
PCR for Constitutive Promoters (pTEF1(M6))
Materials
1x reaction
4.5x Master Mix
5x Phusion HF Buffer
10 µl
45 µl
dNTP's (10 mM)
1 µl
4.5 µl
Forward Primer (10µm)
2.5 µl
11.25 µl
Reverse Primer (10µm)
2.5 µl
11.25 µl
*Template DNA
0.3 µl
1.35 µl
Phusion Polymerase
0.5 µl
2.25 µl
Water
33.2 µl
149.4 µl
Total
50 µl
225 µl
*Added everything to template DNA. oops
Mix materials in a 4.5x Master Mix on ice. Mix well.
Pipetter 50 µl from the Master Mix into 4 labeled PCR tubes
Thermocycler for :
Initial Duration   | 98° C | 30s    35 Cycles of:        Denaturation | 98° C | 10s       Annealing    | 55° C | 20s       Extension    | 72° C | 30s Final Extension    | 72° C | 5m Hold               | 4°  C | Forever
Keep Samples for gel extraction on tje next day
June 10, 2014
Gel of Constitutive Promoters
Add 5 µl Bluejuice 10x to each of the tubes
Load all on gel.
Gel Extraction
QlAquick Gel Extraction Kit
Cut Gel
Weigh it in a colorless tube
Add 3 volumes Buffer Q G to 1 volime Gel (100mg ~ 100µl)
Incubate @ 50° C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)
Add 1 gel volume isopropanal to the sample and mix
Place a QlAquick soin column in a provided 2ml collection tube
Place sample in column & spin for 1 min --> discard flow through
To wash add 0.75 ml Buffer PE to column & centrifue for 1 min, then dry spin
Place column in 1.5 ml tube
Add 35µl H2O & centrifuge for 1 min.
Restriction Enzyme Digest with APA1
40 µl DNA
5 µl Cutsmart
0.5 µl APA1  Room Teperature Overnight
June 11, 2014
Digest with Xho1
-0.5µl  let sit for a few hours
PCR Purification
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
Apply sample to column and spin for 1 min
Discard flow through
To wash add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
[Elute DNA] Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
M6 + PSV606 ligation
1:3 backbone + insert
insert concentration: 138 ng/µl
Reagents
1x
10x ligase buffer
1.0µl
DNA Backbone (PSV606)
0.2µl
Insert
0.2µl
T4 DNA Ligase
0.5µl
H2O
8.1µl
Total
10 µl
Room Temperature for 2 hours
E. Coli Transformation
- 10 µl ligation  - 50 µl competent cells      --> 30m on ice      --> 45s heatshock 42° C      --> 2m on ice  - 250 µl of SOC media      --> 1h shake 37° C  Plate
June 12, 2014
Colonies didn't grow
Possibly because Kara gave us cheap cells to use
Redo:
ligation - 0.4 µl backbone & 7.9 µl H2O
transformation - used more expensive competent cells
June 13, 2014
Yeast Transformation (CB008 + Inducible Promoters)
Reagents
YPD
1 M LiOAc
10X TE pH 7.5
1X TE pH 7.5, 0.1 M LiOAc
50% PEG 3350
DMSO
Salmon Sperm DNA (ssDNA)
PEG = viscous, pipette slow
boil ssDNA aliquots
Previous Day : Grow yeast strain to be transformed in 5-10 mL YPD overnight at 30° C
Set up digest to linearize DNA
Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C
Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)
Harvest cells in centrifuge - 3000 rpm, 2-5 min
Wash with 1 ml 0.1 M LiOAin TE
Pellet cells - 3000 rpm , 2-5 min
Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation
to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA
Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc) Optional: Add 75 µl DMSO definitely did
Vortex
Incubate 42° C for 30m & begin drying plates
Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)
Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD
Plate on selective media
Incubate 1-3 days
June 16, 2014
Colony PCR for screening E. Coli (Constitutive promoters + GFP)
Pick single colonies ( 5 from each plate ) mix in 25 µl H2O in a tube. Use 5 µl in PCR reaction
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
6 µl
10 µM Reverse primer
1 µl
6 µl
Water
3 µl
18 µl
Bacterial cells (template)
5 µl
-----
Cycle:
95° C | 5m     30x:         95° C | 45s         55° C | 30s         72° C | 1m per kb  72° C | 10m   4° C | Forever
load 5 µl onto gel
for all positive bands - take the rest of the cells from step 1 and inoculate them into an overnight LB (+antibiotic) for miniprep
Colony PCR for Yeast & Patching
Number colonies
Patch on plate
Mix in 10 µl NaOH
Boil for 20m
PCR
Cycle:
95° C | 5m     30x:         95° C | 45s         55° C | 30s         72° C | 1m per kb  72° C | 10m   4° C | Forever
Only 9 lanes worked Gel 1: SAG(1), ~~PRM3(1)~~, ASG7(1), ~~PRM3(3)~~, PCL2(2) [PRM3 mislabeled possibility of one being PRM6. unusable. Gel 2: EXM18(1), ECM18(2), PRM2(3), CLG(3) will redo
June 17, 2014
CB008(x)
CB008DB(DB)
CLG1
PRM1
PRM1
YDR124W
PRM3
PRM6
PRM2
HYM1
ECM18
PCL2
YDR124W
SAG1
PRM6
PRM3
--------
ASG7
Create new template by picking off patch plate
also pick 2 new colonies (4&5) from original growth plate and create new patch plate
Gel Photos
June 18, 2014
Frozen Glycerol Stocks Yeast
1) Grow Overnight in YPD (2-5 mL) then dilute 1:20 in YPD, grow to OD 0.4-0.5 2) Add 350 µl cells to 350 µl sterile 60% glycerol in cryovial, vortex to mix, snap freeze in liquid nitrogen and store at -80° C (but actually we just stuck it in the freezer because nobody will let us use liquid nitrogen).
Constitutive Promoters Miniprep
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix
Add 350 µl Buffer N3 and invert immediately 4-6 times
Centrifuge for 10 min at 13,000 rpm
Add supernatant to QIAprep spin column
Centrifuge for 30-60s - Discard flow through
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
June 19, 2014
Adding α Factor to CB008 Promoters and Flow Cytometry
Concentrations of α factor 3mM stock 0, 1 nm, 10 nm, 100 nm, 1 µm We want to use 10 µl of 100x concentration  100nm, 1000nm, 10 µm, 100 µm
Amount of reagents
desired concentration
30µl of 3mM + 870 µl H2O
100 uM
50 µl of 100 µm + 450 H2O
10 µM
50 µl of 10 µm + 450 H20
1000 nM
50 µl of 1000 nm + 450 H2O
100nM
1   2   3   4   5   6   7   8   9   10  11  12 A   [------NEG------]       [------NEG-------] B   [------PRM2-----]       [------PRM1------] C   [------ASG7-----]       [------EMC18-----] D   [------PCL2-----]       [------PRM3------] E   [------NEG------]       [------SAG1------] F   [------CLG1-----] G   [------YDR124W--] H   [------PRM6-----]
Protocol : Night Before - Start overnight culture in SD coplete media (not YPD)
Day of:  1. Dilute cultures to final ~OD 0.05 - 0.1 in the 96 well shaker plate (saturated culture) should be ~OD 7 2. Allow cells to enter growth stage by putting on plate shaker for 3 hrs at 100 rpm 30° C 3. Induce w/    α factor concentrations [0 , 1nm, 10nm, 100nm, 1µm] 4. Allow induction to proceed on shaker for 90 min to 2 hrs 5. Transfer 250 µl of each culture to 96 well flow cytometry V-bottom plate using a multichannel pipette. Also arrested cells w/ cyclohexiimide - 10ml/well 6. Run on Flow Cytometer - Check fluid levels, press run - check voltage 7. Analyze data -> FloJo/MatLab
Flow Voltages Inspector :  FSC: 250 SSC: 280 FITC: 550
June 20, 2014
Yeast Transformation w/ Constitutive Promoters
PTEF1, M3, M6, M7, M10 in CB008 and CB008DB
Linearize Plasmids - PSV606 10 µl DNA 5 µl Cutsmart Buffer 1 µl PME1 34 µl H2O 37 ° C for 2 hours
Procedure the same as June 13, 2014
June 23, 2014
Colony PCR and Gels (Constitutive Yeast)
protocol same as June 16
Overnight Cultures in 5 ml YPD
June 25, 2014
Glycerol Stocks of Constitutive Promoters in Yeast
dilute cultures ~ 1:20 4-5 hours before
420 µl of 50% glycerol + 350 µl cells
vortex
store in -80° C freezer
m3 leaked during dilution
Parts Registry Inducible Promoters PCR with pSB1C3 overhang
Reagents
1x
12x
5x phusion HF Buffer
10 µl
120 µl
dNTP's (10 mM)
1 µl
12 µl
Phusion Polymerase
0.5 µl
6 µl
Water
33.2 µl
398.4
.3 µl template DNA    44.7 µl master mix   2.5 µl FW Primer   2.5 µl RV Primer       Initial Denaturation  | 98° C | 30s        35 Cycles of:            Denaturation    | 98° C | 10s           Annealing       | 55° C | 20s           Extension       | 72° C | 30s     Final Extension       | 72° C | 5m     Hold                  | 4°  C | Forever
25µl sybersafe used when pouring gel
«Only pGEM's 2 & 3 worked»
June 26, 2014
PCR Constitutive and Inducible Promoters with PSB1C3 overhang
« pGEM's 1,4,5,6,7,8,9,10,11,12,13,14,15,16 »
Reagents
1x
17X
Template DNA
0.3 µl
N/A
FW Primer
2.5 µl
N/A
RV Primer
2.5 µl
N/A
DMSO
1.5 µl
25.5 µl
5x Phusion HF Buffer
10 µl
170 µl
dNTP's
1 µl
17 µl
Phusion Polymerase
0.5 µl
8.5 µl
Water
31.7 µl
538.9 µl
Initial Denaturation   | 98° C | 30s        35 Cycles of:            Denaturation    | 98° C | 10s           Annealing       | 55° C | 20s           Extension       | 72° C | 30s     Final Extension       | 72° C | 5m     Hold                  | 4°  C | Forever
June 27, 2014
«Still need to redo pGEM 5,6,8»
Constitutive Promoter Digest
pGEM12 - pGEM16
Removing GFP to insert rtTA
Not1 HF & Xho1
Reagents
1x
DNA
15 µl
Cutsmart
2.5 µl
Xho1
0.5 µl
Not1 HF
0.5 µl
6.5 µl
H20
incubate at 37° C for ~ 2 hours
Parts Registry PCR Redo & PCR Purification
«Same protocol as on the 26th»
#finally
PCR Purification (pGEM 1,2,3,4,7,9,10-16)   ( Same as June 11)
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
Apply sample to column and spin for 1 min
Discard flow through
To wash add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
[Elute DNA] Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Gibson Assembly
Reagents
1X
pSB1C3 (25ng/µl)
2 µl
insert
3 µl
Gibson Master Mix
5 µl
50° C fro 1 hour -> ice
Transformation
10µl Gibson     25 µl Mach I cells      30 m ice     45 s heat shock (42° C)      2 m ice     +250 SOC Media      1 hr. 37°C      Plate on LB + CAM      Keep in drawer over the weekend
Didn't Work
June 30, 2014
Colony PCR for Yeast ( pTET GFP & pAGA GFP CB008 + CB008 DB )
Number colonies
Patch on plate
Mix in 10 µl NaOH
Boil for 20m
PCR
Cycle:
95° C | 5m     30x:         95° C | 45s         55° C | 30s         72° C | 1m per kb  72° C | 10m   4° C | Forever
pTET patch plate -leu had an orange and tellow stripe which was kind of sketchy. It isnt growing either so we are going to replate from one of the original colonies just to check. (7/2/14)
Primers Used -       pAGA - regular pSV606 primers      pTET - pTET primers
July 1, 2014
pTET-GFP/pAGA-GFP Colony PCR's
Making iGEM Backbone
5 tubes - 50 µl reaction each
Reagents
1x
5x
Phusion Polymerase
0.5 µl
2.5 µl
Template DNA
1 µl
5 µl
Phusion HF Buffer
10 µl
50 µl
dNTP's
1 µl
5 µl
FW Primer (SB prep 3P1)
2.5 µl
12.5 µl
RV Primer (SB Prep 2-Ga)
2.5 µl
12.5 µl
Water
32.5 µl
162.5 µl
*iGEM provides different template DNA than pSB1C3
Initial Denaturation   | 98° C | 30s        35 Cycles of:            Denaturation    | 98° C | 10s           Annealing       | 55° C | 20s           Extension       | 72° C | 30s     Final Extension       | 72° C | 5m     Hold                  | 4°  C | Forever
Different times for a bulk reaction but we went with the one above
-Failed
Redone 7/2/14 using pSB1C3 as template 7/2/14
July 2, 2014
Glycerol Stocks for pTET-GFP & pAGA-GFP
«still need to check if pTET is okay (patch plate)»
PCR Cleanup of pSB1C3
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
Apply sample to column and spin for 1 min
Discard flow through
To wash add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
[Elute DNA] Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Digestion of pGEM insert with pSB1C3 overhang and pSB1C3 backbone
Gibson didn't work, so we are going the standard "cut & paste" route
Reaction is as follows:              5 µl cutsmart             1 µl enzymes             _ µl H20             _ µl DNA         + ___________            50 µl Total    We want 1000 ng of DNA
pGEM #
concentration
µl for digestion
µl H20
1
245.1 ng/µl
4.1 µl
39.9 µl
2
116.9 ng/µl
8.6 µl
35.4 µl
3
81.77 ng/µl
12.3 µl
31.7 µl
4
87.00 ng/µl
11.5 µl
32.5 µl
5
12.26 ng/µl
81µl* (30µl)
14 µl
6
60.79 ng/µl
16.45 µl
27.5 µl
8
55.62 ng/µl
18 µl
26 µl
9
87.86 ng/µl
11.38 µl
32.62 µl
10
25.02 ng/µl
40µl* (30µl)
14 µl
11
54.95 ng/µl
18.2 µl
25.8 µl
12
45.29 ng/µl
22.1 µl
21.9 µl
13
55.37 ng/µl
18.06 µl
25.94 µl
14
35.62 ng/µl
28.07 µl
15.93 µl
15
107.1 ng/µl
9.3 µl
34.7 µl
16
24.08 ng/µl
41.52µl* (30 µl)
14 µl
37° C for 2 hours or overnight
Wanted to keep DNA 30 µl or under*
«pGEM7 wasn't used because it isnt biobrick compatible»
July 3 - July 6, 2014
-out- Check Roberts Lab Notebook for what happened
(He ligated and transformed)
July 7, 2014
All plates grew including the negative control.
Colony PCR
Pick single colonies ( 5 or so from each plate ) mix in 25 µl H2O in a tube. Use 5 µl in PCR reaction
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
6 µl
10 µM Reverse primer
1 µl
6 µl
Water
3 µl
18 µl
Bacterial cells (template)
5 µl
-----
Cycle:
95° C | 5m     30x:         95° C | 45s         55° C | 30s         72° C | 1m per kb  72° C | 10m   4° C | Forever
Made liquid cultures of ones that worked  Re - Colony PCR pGEM 3,10,13
July 8, 2014
Liquid cultures didn't work
Used LB-Carb instead of LB Cam -oops- Re-Culturing
Colony PCR (Redo pGEM's 3, 10, 13)
Made cultures of 3 & 10
Redo 13 (13.7-13.11 labelled 1-5)
July 9, 2014
Parts Registry Miniprep
pGEM #
Colony #
1
1
2
1
3
4
4
1
5
1
6
1
7
1
8
1
9
1
10
4
11
1
12
1
14
1
15
1
16
1
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix
Add 350 µl Buffer N3 and invert immediately 4-6 times
Centrifuge for 10 min at 13,000 rpm
Add supernatant to QIAprep spin column
Centrifuge for 30-60s - Discard flow through
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Transformation of pGEM 13
10µl plasmid 25 µl cells  30m on ice 45s heat shock 2m on ice  +250 µl SOC Media  1hr. 37° C Plate on LB CAM
July 10, 2014
Colony PCR and overnight cultures
( shared gel with Jessica )
July 11, 2014
Miniprep pGEM 13
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix
Add 350 µl Buffer N3 and invert immediately 4-6 times
Centrifuge for 10 min at 13,000 rpm
Add supernatant to QIAprep spin column
Centrifuge for 30-60s - Discard flow through
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
July 12, 2014
Check Sequencing and nanodrop of all parts(concentrations written on tubes)
July 17, 2014
500µl Carb + 500ml LB = LB + Carb
July 21, 2014
Colony PCR PTET-GFP + pTET-mfα + Constitutive Promoter-rtTA
CB008
CB008DB
PTEF1
m6
m6
m7
m7
-
m10
-
others are still being made
only 3 worked - m7 #2 & m10 #3 & #4
Will repeat
July 22, 2o14
July 23, 2014
Fluorescent Protein Transformations for Exploratoreum
GFP
Citrine
mT BFP2
mRuby
ECFP
mKitr
EGFP
.5 µl DNA  25 µl cells   30m on ice  45s heat shock  2m on ice  +250 µl SOC Media  1hr. 37° C Plate
Didn't work. Found in old UCSF iGEM teams stuff. Possibly for yeast.
July 25, 2014
Yeast Transformations
pTET-mfα in 11E3128 into
pAGA1-mCherry into
pTEF1 DB
pTEF1
pTEF1(m10) DB
pTEF1(m6)
-
pTEF1(m7)
-
pTEF1(m10)
-
pTEF1(m6) DB
-
pTEF1(m7) DB
Digest w/ PME1
5µl 11E3128  1µl PME1 1µl cutsmart 3µl H20  and   10µl pTET-GFP pTET mfα 1 µl PME1 1.5µl cutsmart 2.5µl H20  37° C for 1h
E. Coli Transformation pGEM 22 ( pAGA1-mCherry)
0.2 µl plasmid 25 µl cells (DH5α)  30m ice 45s 42°C 2m ice  +250 µl SOC Media 1h 37°C  Plate on LB Carb Place in drawer over the weekend
August 5, 2014
Streak Plates for Jessica
CB008 pTET+GFP AFRP +rtTA
PRM1
PRM2
PRM3
YDR124W
CLG1
ASG7
HYM1
Glycerol Stocks of
pTET+GFP + m3+rtTA + pTET+mfAlpha
pTET+GFP + m10+rtTA + pTET+mfAlpha + pAGA mCherry
Streak & Overnight culture of m6 & m7 CB008 & DB pTET GFP +rtTA +mfalpha + mCherry
August 6, 2014
Glycerol Stocks of
CB008 m6/m7+(GFP rtTA mfalpha)+pAGA mCherry
CB008DB m7+(GFP rtTA mfalpha)+pAGA mCherry
August 8, 2014
BFP Miniprep
August 11, 2014
Flow Cytometry Set Up
pTET GFP
Dox and alpha factor inductions
YDR124W, pASG7, HYM1, CLG1
Dox Dilutions by Ianto:
Stock: 100mg/mL  **1x**           **100x**          Prep 60µg/mL       A: 6mg/mL            (6:100) 60µL of Stock in 940µL of water 30µg/mL       B: 3mg/mL            30µL of Stock in 970µL of waterb 9µg/mL        C: 900µg/mL          150µL of A in 850µL of water 6µg/mL        D: 600µg/mL          100µL of A in 900µL of water 3µg/mL        E: 300µg/mL          50µL of A in 950µL of water          0.9µg/mL      F: 90µg/mL           100µL of C in 900µL of water 0.6µg/mL      G: 60µg/mL           100µL of D in 900µL of water 0.3µg/mL      H: 30µg/mL           100µL of E in 900µL of water 0.09µg/mL     I: 9µg/mL            100µL of F in 900µL of water 0.06µg/mL     J: 6µg/mL            100µL of G in 900µL of water 0.03µg/mL     K: 3µg/mL            100µL of H in 900µL of water
Alpha Factor Dilutions :
1x
100x
0 nm
0 nm
0.5 nm
50 nm
1 nm
100 nm
10 nm
1000 nm
100 nm
10000 nm
1000 nm
100000 nm
3000 nm
300000 nm
30 µl 3Mm stock + 270 µl H2O = 300,000 nm
100 µl 300,000 nm stock + 200 µl H2O = 100,000 nm
50 µl 100,000 nm + 450 µl H2O = 10,000 nm
50 µl 10,000 nm + 450 µl H2O = 1,000 nm
50 µl 1000 nm + 450 µl H2O = 100 nm
200 µl 100 nm + 200 µl H20 + 50 nm
Cutures of AFRP's + rtTA & pTET GFP (PRM1, PRM2 , PRM3)
August 13, 2014
Flow
Const. +rtTA mCherry pTET GFP
In triplicate
Time Course
0, 1.5, 3, 5 hours
dox concentrations
0, 0.03, 0.06, 0.09, .6, 6
pTEF1, m3, m6, m7, m10
August 14, 2014
Flow
Const. + mCherry
Same as August 13 except includes 8hr time point.
Weird clumps found in shaker plate - cancelled
August 15, 2014
1:200 dilutions 7:50 AM
Redo of August 14
1  2  3  4  5  6  7  8  9  10 11 12  A { --- pTEF 1 ---} { ----- m7 ------} B { --- pTEF 1 ---} { ----- m10 -----} C { --- pTEF 1 ---} { ----- m10 -----} D { ----- m6 -----} { ----- m10 -----} E { ----- m6 -----} F { ----- m6 -----} G { ----- m7 -----} H { ----- m7 -----}   dox concentration increases from left to right.   0, 0.03, 0.06, 0.09, 0.6, 6
August 19, 2014
Flow
1:200 dilution 7:40 AM
mCherry( +pTET GFP mfAlpha const. rtTA) DB strains
1  2  3  4  5  6  7  8  9  10 11 12  A { --- pTEF 1 ---} { ----- m6 ------} B { --- pTEF 1 ---} { ----- m7 ------} C { --- pTEF 1 ---} { ----- m7 ------} D { ----- m3 -----} { ----- m7 ------} E { ----- m3 -----} { ----- m10 -----} F { ----- m3 -----} { ----- m10 -----} G { ----- m6 -----} { ----- m10 -----} H { ----- m6 -----}   dox concentration increases from left to right.   0, 0.03, 0.06, 0.09, 0.6, 6   0, 1.5, 3, 5, 8 hours
August 28, 2014
Start cultures for CB008 const. +rtTA pTET GFP pTET mfAlpha for PCL2 mCherry transformation
Flow data does not show RFP - same for m6 pAGA mCherry
streak yeast strains from glycerol stocks - yGEM's 17, 20, 53, 54, 68, 69, 70, 88, 96
August 29, 2014
Transform yGEM 47-50 + PCL2 mCherry  Const. rtTA, pTET GFP, pTET mfAlpha
September 2, 2014
Forgot to linearize plasmid August 29 - will colony PCR since some colonies grew but probably didn't work
Prepared dilutions to retransform using linearized plasmid in Jeffrey's box
September 4, 2014
Colony PCR PCL2 mCherry's
Transform :
CB008 m3 rtTA w/ pTET mfAlpha
CB008 m6 mfAlpha w/ pAGA1 mCherry
linearized plasmid in Jeffrey's box
mfAlpha auxotrophic marker - HIS
mCherry on TRP site
Gel Photo machine wasn't working - pTEF (1,2,3) and m7 (3) worked
Redo m6 & m10
Overnight cultures od pTEF&m7&m10 pAGA mCherry for flow
September 5, 2014
Flow (Co-cultures) pAGA mCherry CBoo8
Same Dox concentrations
1.5 ml SD in the growth plate
final dilution OD 0.03 so OD 0.015 for each strain
1  2  3  4  5  6  7  8  9 10 11 12 A { -pTEF1 x M7- }  { -pTEF1 x M10- } B { -pTEF1 x M7- }  { -pTEF1 x M10- } C { -pTEF1 x M7- }  { -pTEF1 x M10- } D { --M7 x M10-- } E { --M7 x M10-- } F { --M7 x M10-- } G H
x
ptef1
m7
m10
OD
0.39
0.39
0.39
Actual OD
3.9
3.9
3.9
Actual OD / Target OD = Dilution Factor
3.9/0.015 = 260
µl / Dilution Factor = µl to be added
1500 µl/260 = 5.77 µl
Colony PCR
Failed
Try again with Zymalase
25 µl Zymalase  + Cells   37 °C 10m   5 µl ^ in PCR reaction    Only m10 b/c m6 is growing weird
Need to:
Retransform M6 with PCL2 mCherry
Colony PCR m3 mfAlpha & m6 pAGA RFP
All m6-mfAlpha strains when transformed grow weird
September 8, 2014
Streak Plates of yGEM 75 & 76
Colony PCR - m3 w/ mfAlpha retransform m6 w/ pAGA1/pPCL2 +mCherry
-Grows weird
Transform m3+mfAlpha
September 9, 2014
Flow - New PCL2s -m7 m10 pTEF1
September 10, 2014
yGEM 48 (m6-mfAlpha) developed a weird mutation allowing it to grow strangely on an SD-TRP plate without integrating the plasmid.
Will retransform m6-rtTA with mfAlpha
3 Transformations:
m6+rtTA w/mfAlpha
m6+rtTA w/mfAlpha & pCL2 mCherry
- m6+rtTA w/mfAlpha & pAGA1 mCherry
September 11, 2014
Flow and Transformations
1  2  3  4  5  6  7  8  9  10 11 12  A { --- pTEF 1 ---} { ----- m10 -----} B { --- pTEF 1 ---} { --- m7+pTEF1 --} C { --- pTEF 1 ---} { --- m7+pTEF1 --} D { ----- m7 -----} { --- m7+pTEF1 --} E { ----- m7 -----} { --- m10 + M7---} F { ----- m7 -----} { --- m10 + M7---} G { ----- m10 ----} { --- m10 + M7---} H { ----- m10 ----}
dox concentration increases from left to right.       0, 0.03, 0.06, 0.09, 0.6, 6
0, 1.5, 3, 5, 8 hours Cancelled
September 14, 2014
Colony PCR with Zymalase
Reagents
1X
10X
2X GoTaq Green PCR Master Mix
10 µl
100µl
10 µM Forward primer
1 µl
10 µl
10 µM Reverse primer
1 µl
10 µl
Water
3 µl
30 µl
Bacterial cells (template)
5 µl
------
DMSO
1.5 µl
15 µl
Zymalase
0.5 µl
5 µl
37° C | 5m 95° C | 5m     30x:         95° C | 45s         55° C | 30s         72° C | 1m per kb  72° C | 10m   4° C | Forever
Didnt Work
September 15, 2014
Flow Repeat (9/11)
1  2  3  4  5  6  7  8  9  10 11 12  A { --- pTEF 1 ---} { ----- m10 -----} B { --- pTEF 1 ---} { --- m7+pTEF1 --} C { --- pTEF 1 ---} { --- m7+pTEF1 --} D { ----- m7 -----} { --- m7+pTEF1 --} E { ----- m7 -----} { --- m10 + M7---} F { ----- m7 -----} { --- m10 + M7---} G { ----- m10 ----} { --- m10 + M7---} H { ----- m10 ----}
x
ptef1
m7
m10
OD
0.64
0.67
0.67
Actual OD
6.4
6.7
6.7
Actual OD / Target OD = Dilution Factor
6.4/0.015 = 427
6.7/0.015 = 447
µl / Dilution Factor = µl to be added
1500/427 = 3.5µl
1500/447 = 3.4µl
Colony PCR from day before failed.
Repeat
To Check for mfAlpha
Reagents
1X
10X
2X GoTaq Green PCR Master Mix
10 µl
100 µl
10 µl RA146
1 µl
10 µl
10 µl RA145
1 µl
10 µl
Water
3 µl
30 µl
Bacterial cells (template)
5 µl
------
pAGA
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
50 µl
10 µl RA148
1 µl
5 µl
10 µl RA145
1 µl
5 µl
Water
3 µl
15 µl
Bacterial cells (template)
5 µl
-----
pTET mfAlpha Digestion
Reagents
1X
plasmid
23 µl
Cutsmart
3 µl
PME1
1 µl
H2O
3 µl
September 22, 2014
Digest
PCL2 - Bar1 w/ APA1 & Not1
CLG1 - Bar1 w/ APA1 & Not1
CLG - Bar1 w/ Xho1 & Not 1
APA1 - 1hr RT Not 1 37° O/n
Not1 + Xho1 - 37° O/n
PJW608 APA1/Not1 HF
September 23, 2014
Gel Extraction
CLG - Bar1
PCL2 - Bar1
Bar1
PCR Purification
PJW608
Reagents
Bar1
PCL2
CLG1
10x Buffer
2 µl
2 µl
2 µl
Vector DNA
1 µl
1 µl
1 µl
Insert DNA
6 µl
2.3 µl
1.6 µl
H20
10 µl
13.7 µl
14.4 µl
Ligase
1 µl
1 µl
1 µl
September 24, 2014
pAGA1 + Bar1 + PJW608
1 µl Ligase 2 µl Buffer 1 µl Vector 6.2 + x µl insert 9.8 µl H2O
Colony PCR
pCL2 Bar1 CLG Bar1
September 25, 2014
Colony PCR  pAGA Bar1
LB Cultures: CLG Bar1 PCL2 Bar1
September 26, 2014
Miniprep PJW608 - CLG (or PCL2) Bar1
Transform into DB strains
linearize plasmids
6 µl Plasmid
2.5 µl cutsmart
1 µl PME1
13.5 µl H2O
Colony PCR pAGA Bar1 again
September 29, 2014
Colony PCR
CLG1
PCL2
m6
pTEF1
m3
m3
pTEF1
m6
-----
m10
m10 CLG did not grow
September 30, 2014
Bar 1 Flow
1  2  3  4  5  6  7  8  9  10 11 12  A { --- pTEF 1 ---} { ----- m6 ------} B { --- pTEF 1 ---} { ----- m10 -----} C { --- pTEF 1 ---} { ----- m10 -----} D { ----- m3 -----} { ----- m10 -----} E { ----- m3 -----} { ---- m3+m6 ----} F { ----- m3 -----} { ---- m3+m6 ----} G { ----- m6 -----} { ---- m3+m6 ----} H { ----- m6 -----}
x
m3
m6
OD
0.4
0.45
Actual OD
4
4.5
Actual OD / Target OD = Dilution Factor
4/0.015 = 266.66
4.5/0.015 = 300
µl / Dilution Factor = µl to be added
1500/266.66 = 5.6 µl
1500/300 = 5 µl
October 2, 2014
Flow Cocultures w/ Dye
Stain m7 with blue dye (Cell Tracker?)
Find OD.
OD/0.015 = DF
1000µl/DF = x per well
xµlperwell x 100 = y amount of overnight culture to dye
Split y into 2 1.5 ml tubes
Pellet - 8000 rpm for 3 min
Remove liquid, no cells
add 1 ml HEPES and 10 µl Cell Tracker to each tube
gently resuspend and keep at RT for 30m
Flick tube every 5m or so (protect from light)
Pellet
Remove Liquid and Replace w/ 1ml SD media
Pellet
Remove liquid
Add y/2 µl SD to each tube.
Resuspend 2xµl in m7 only wells (OD 0.03) xµl in coculture wells (OD 0.015)
1  2  3  4  5  6  7  8  9  10 11 12  A { --- pTEF 1 ---} { ----- m10 -----} B { --- pTEF 1 ---} { --- m7+pTEF1 --} C { --- pTEF 1 ---} { --- m7+pTEF1 --} D { ----- m7 -----} { --- m7+pTEF1 --} E { ----- m7 -----} { --- m10 + M7---} F { ----- m7 -----} { --- m10 + M7---} G { ----- m10 ----} { --- m10 + M7---} H { ----- m10 ----}
x
ptef1
m7
m10
OD
0.44
0.62
0.47
Actual OD
4.4
6.2
4.7
Actual OD / Target OD = Dilution Factor
4.4/0.015 = 293.33
6.2/0.015 = 413.33
4.7/0.015 = 313.33
µl / Dilution Factor = µl to be added
1000/293.33 = ~3.41µl
1000/413.33 = ~2.42µl
1000/313.33 = ~3.19µl
Sabrina's Lab Notebook
06/09/14
PCR for Constitutive Promoters - pTEF1 (M3)
Materials
1x reaction
4.5x Master Mix
5x Phusion HF Buffer
10 µl
45 µl
dNTP's (10 mM)
1 µl
4.5 µl
Forward Primer (10µm)
2.5 µl
11.25 µl
Reverse Primer (10µm)
2.5 µl
11.25 µl
*Template DNA
0.3 µl
1.35 µl
Phusion Polymerase
0.5 µl
2.25 µl
Water
33.2 µl
149.4 µl
Total
50 µl
225 µl
(Note: Add enzymes last)
PCR Reaction Procedures:
Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom.
Pipette 50ul from the Master Mix into four labled PCR tubes
Put in the thermocycler for the following cycles:
Initial Denaturation   | 98° C | 30s    35 Cycles of:        Denaturation | 98° C | 10s       Annealing    | 55° C | 20s       Extension    | 72° C | 30s Final Extension    | 72° C | 5m Hold               | 4°  C | Forever
Keep samples for gel extraction on the following day.
06/10/14
Loading the Gel
*Add 5ul of Loading Dye (10X Blue Juice) to samples.
Gel Map:
1  2  3  4  5  6  7  8  9  10  11  12  13  14  15    [----M6---][----M7----] [-----M3-----]  1  2  3  4  5  6  7  8  9  10  11  12  13  14  15    [----TEF1----]   [-M10]
Both lane #1 are 100 bp ladder
*George had less to PCR due to a spill accident when spinning tubes
Gel Extraction Procedure:
Cut Gel
Weigh it in a colorless tube
Add 3 volumes Buffer QG to 1 volime Gel (100mg ~ 100µl)
Incubate @ 50° C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)
Add 1 gel volume isopropanal to the sample and mix
Place a QlAquick soin column in a provided 2ml collection tube
Place sample in column & spin for 1 min -> discard flow through
To wash add 0.75 ml Buffer PE to column & centrifue for 1 min, then dry spin
Place column in 1.5 ml tube
Add 30µl H2O & centrifuge for 1 min.
Restriction Enzyme Digest with ApaI (Backbone Digestion):
40ul of Gel Extraction DNA
5ul of Cutsmart Buffer
0.5ul of ApaI
*Digest at room temperature overnight
06/11/14
PCR Purification and Ligation
Digest with XhoI
Add 0.5ul of XhoI Enzyme (in Enzyme Box) to ApaI Digest from 06/10/14
Incubate in the 37°C shaker for ~2 hours.
PCR Purification
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
To bind DNA, apply sample to column and spin for 1 min
Discard flow through
To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Ligation (pSV606 and pTEF1-M3)
1:3 Backbone to Insert
Concentration of M3 Digest = 127.1 ng/ul
Materials:
Reagents
1x
10x ligase buffer
1.0µl
DNA Backbone (PSV606)
0.2µl
Insert
0.2µl
T4 DNA Ligase
0.5µl
H2O
8.1µl
Total
10 µl
Keep at room temperature for at least 2 hours.
Transformation
Materials
1x
Ligation (pSV606 and M3)
10µl
E. Coli Competent Cells
50ul
Mix both together and keep on ice for 30 minutes
42°C Heat shock for 45 seconds
Put immediately on ice for at least 2 minutes
Add 250 ul of SOC media, shake for 1 hr at 37°C.
Plate on selective media and store at 37°C.
06/12/14
Re-Ligation and Tranformation of M3
Note: Most of our plates did not form colonies, with the exceptions of pTEF1-M10 and pTEF1-M3.
Due to this, we will all have to re-ligate using higher concentration of the pSV606 backbone.
Kara belives that the failure in our transformation was due to mediocre competent cells. We will be using better competent cells.
Forgot to make a Negative control. Make sure to make one next time.
Ligation #2:
Materials:
Reagents
1x
10x ligase buffer
1.0µl
DNA Backbone (PSV606)
0.4µl
Insert
0.2µl
T4 DNA Ligase
0.5µl
H2O
7.9µl
Total
10 µl
-Added 0.2ul more Backbone.
Keep at room temperature for at least 2 hours.
Transformation
Materials
1x
Ligation (pSV606 and M3)
10µl
E. Coli Competent Cells
50ul
Mix both together and keep on ice for 30 minutes
42°C Heat shock for 45 seconds
Put immediately on ice for at least 2 minutes
Add 250 ul of SOC media, shake for 1 hr at 37°C.
Plate on selective media and store at 37°C.
06/13/14
Yeast Transformation (CB008 + Inducible Promoters)
Assisted Ianto and others in Transforming Inducible Promoters into Yeast.
| Reagents                  | | ------------------------- | | YPD                       | | 1 M LiOAc                 | | 10X TE pH 7.5             | | 1X TE pH 7.5, 0.1 M LiOAc | | 50% PEG 3350              | | DMSO                      | | Salmon Sperm DNA (ssDNA)  |
PEG is viscous, so pipette slowly to prevent air bubbles from forming.
boil ssDNA aliquots for 10 min, then ice down for 10 min before use.
Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C
Set up digest to linearize DNA
Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C
Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)
Harvest cells in centrifuge - 3000 rpm, 2-5 min
Wash with 1 ml 0.1 M LiOAin TE
Pellet cells - 3000 rpm , 2-5 min
Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation
to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA
Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc) Optional: Add 75 µl DMSO definitely did
Vortex
Incubate 42° C for 30m & begin drying plates
Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)
Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD
Plate on selective media
Incubate 1-3 days
06/16/14
Colony PCR for Screening E.Coli Procedure:
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 25ul of water.Do this for 5 colonies. Use 5ul for PCR reaction below.
Set up PCR Reaction below:
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
6 µl
10 µM Reverse primer
1 µl
6 µl
Water
3 µl
18 µl
Bacterial cells (template)
5 µl
-----
Total
90 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation   | 95° C | 5 min    35 Cycles of:        Denaturation | 95° C | 45s       Annealing    | 55° C | 30s       Extension    | 72° C | 1min per kb Final Extension    | 72° C | 10m Hold               |  4°C  | Forever
Load PCR products on a gel. (5ul) (GoTaq already has loading dye)
For all positive bands on the gel, add rest of bacterial cells from PCR tubes to 5ml LB and grow overnight at 37°C for miniprep on the following day.
Result of E.Coli Colony PCR (Constitutive Promoters): All lanes worked.
Grew 2 out of 5 Successful Colony PCR E.Coli Colonies in liquid culture.
Unfortunately, the rack which contained our cultures was not secured and caused our cultures to fall down and possibly be contaminated.
We re-inocculated the colonies due to possible contamination.
Colony PCR for Yeast (Inducible Promoters CB008 and CB008DB) Procedure:
Boil Yeast at 95°C for 1 hour.
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 15ul of NaOH. Take 3 colonies from each plate. Use 5ul for PCR reaction below.
Set up PCR reaction below:
Reagents
1X
67X
2X GoTaq Green PCR Master Mix
10 µl
670 µl
10 µM Forward primer
1 µl
67 µl
10 µM Reverse primer
1 µl
67 µl
Water
3 µl
201 µl
cells (template)
5 µl
-------
Total
1005 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation    | 95°C | 5 min    35 Cycles of:        Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C | Forever
Load PCR products on a gel. (5ul) (GoTaq already has loading dye)
Results: only 4 positive lanes
Possible Explanation: Not enough DNA in PCR tubes.
06/17/14
Miniprep of Constitutive Promoters
Miniprep Precedure:
Obtain E.Coli Cultures from 06/16/14 in 37°C  Incubation Room.
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Sent miniprep into sequencing. Results came out positive. Constitutive promoters successfully transformed into plasmids.
06/18/14
Cultures Set up for Yeast Tranformation
Made cultures of CB008 and CB008DB for each of our constitutive promoters.
06/20/14
Linearization of Plasmids with Constitutive Promoters - PSV606
Linearization Materials:
10 µl DNA (miniprep in my box in -20°C Fridge) 5 µl Cutsmart Buffer 1 µl PME1 34 µl H2O
Incubate at 37 ° C for 2 hours
Transformation of Constitutive Promoters into Yeast (CB008 and CB008DB)
| Reagents                  | | ------------------------- | | YPD                       | | 1 M LiOAc                 | | 10X TE pH 7.5             | | 1X TE pH 7.5, 0.1 M LiOAc | | 50% PEG 3350              | | DMSO                      | | Salmon Sperm DNA (ssDNA)  |
PEG is viscous, so pipette slowly to prevent air bubbles from forming.
boil ssDNA aliquots for 10 min, then ice down for 10 min before use.
Make sure to add reagents in order!
Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C
Set up digest to linearize DNA (see procedure above)
Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C
Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)
Harvest cells in centrifuge - 3000 rpm, 2-5 min
Wash with 1 ml 0.1 M LiOAin TE
Pellet cells - 3000 rpm , 2-5 min
Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation
to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA
Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc) Optional: Add 75 µl DMSO definitely did
Vortex
Incubate 42° C for 30m & begin drying plates
Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)
Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD
Plate on selective media
Incubate 1-3 days
06/23/14
Yeast Colony PCR of Constitutive Promoters
Boil Yeast at 95°C for 1 hour.
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 15ul of NaOH. Take 3 colonies from each plate. Use 5ul for PCR reaction below.
Set up PCR reaction below:
Reagents
1X
67X
2X GoTaq Green PCR Master Mix
10 µl
670 µl
10 µM Forward primer
1 µl
67 µl
10 µM Reverse primer
1 µl
67 µl
Water
3 µl
201 µl
cells (template)
5 µl
-------
Total
1005 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation    | 95°C | 5 min    35 Cycles of:        Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C | Forever
Load 5ul of PCR products on a gel.  (GoTaq already has loading dye)
Results: All Constitutive Promoters were successfully Transformed.
Made cultures to be diluted for glycerol stocks. Cultures made of the yeast transformations of the constitutive promoters.
06/24/14
Flow Cytometry Protocol:
Day Before:
Start overnight cultures of your strains to be tested in 5-10ml SD Complete media. (Do not use YPD. YPD has fluorescence background which would cause problems for FACS testing.)
Day Of:
Flow Procedures:
Dilute overnight cultures 1:100 in the 96 well shaker plate. (Shaker plates can be found in the Lim Lab in the back glass cabinet where large glass bottles can also be found.)
Allow cells to enter growth stage by putting on plate shaker at 30°C going 1000rpm (found in Flow room or the 30°C room in the other hall)for 2-3 hours after dilution.
Induce with alpha factor. The stock alpha factor in the freezer is 3mM and we use it at the final concentrations of: 0, 1nM, 10nM, 100nM, and 1uM.
Allow introduction to proceed on plate shaker for 90 min to 2 hrs, no longer.
Transfer 250ul of each culture to the 96 well flow cytometry v-bottom plate. (found in flow room on shelf) Arrest cells with 10ul of cycloheximide. (stops translations of proteins; stops growth)
Run on flow cytometer.
Analyze Data.
Things to check for Flow:
Check that machine is on standby and not run. (Should be on standby when not in use)
Make sure that a clean plate has been ran by the person before you. Otherwise, run a clean plate. (A1-A4 bleach, B1-B4 water)
Check fluid buffer box and storage tanks. Discard waste fluid in sink 2 doors down (remember to add bleach to tank when done) and replace buffer box if needed.
Make sure you have a positive and negative control so that you can set the parameters.
Open up FACSDiva. Sign into iGEM account.
Create new FACS experiment. Highlight needed wells and use the blue button to create wells for use. Create specimens among the wells and rename them.
Open up inspector to check parameters.
Run well. (Do not click 'Run Plate'.)
Once finished, export data onto a USB.
Run a clean plate.
* Note: NEVER TURN THE WIFI ON! THE COMPUTER WILL CRASH!
Voltage Parameters:
FSC:  250  SSC:  280  FITC: 550
Sample Loading Parameters:
Sample Volume: 200 Mixing Volume: 150 Flow Speed: 0.5ul/sec when adjusting, otherwise 1ul/sec Mixing quantity: 3
Using Flowjo and Matlab to Analyze Data
Flowjo:
Export Data from USB and transfer to iGEM2014 folder.
Open Flowjo and drag data over to box.
Set an appropriate gate.
Go to data, select all data and press 'Σ'. Select median, mean, and count.
Press the refresh button. Additional numerical data should be present.
Matlab:
Open up a new script. and comment the title of the script. (Use '%' for comments.)
Enter the alpha factor info that will serve as the x-axis. (Put in log form.)
Enter the yGEM data. This will serve as the y-axis.
Type your figure info and press the "go".
Example of Matlab script to create graphs based on Flowjo Data:
%14-6-19 Alpha-Factor Promoter ASG7
clear % this will get rid of all previous graphs
Clear all % same as above but for ALL graphs
alpha = [0 1 10 100 1000] % nanomolar nM, spaces show a new number
alpha = [0.01 1 10 100 1000] % 0.01 is really 0. Flowjo cannot comput 0s
yGEM = [558 715 1040 2230 4731] %mean GFP C1-C5 ASG7
figure(1)
semilogx(alpha,yGEM4,'--c'), %('--c') means dotted line, cyan colored line, and star for points.
hold on
xlabel' alpha ') % [] denotes concentration, () denotes "in"
ylabel('mean GFP (au)') % au means arbitrary unit
title('Alpha-factor vs mean GDP yGEM4')
legend('yGEM4')
axis([0.01 1500 200 4000]); % axis([xmin xmax ymin ymax])
Use Adobe Illustrator to beautify graphs. Newer Matlab versions may have the option to beautify graph without the use of Illustrator.
Code available in dropbox.
06/24/14
Plate Map for Flow Cytometry of Inducible Promoters:
PCR of rtTA Promoter
Became part of Jessica's rtTA portion of the project.
Inducible Promoters + rtTA into CB008 and CB008DB
PCR Purification of remaining pHY4 + inducible promoters + GFP (Digestion done by Derrick) (Backbone)
PCR of inducible promoter primer + rtTA with new rtTA RV primer
Materials
1x reaction
12x Master Mix
5x Phusion HF Buffer
10 µl
120 µl
dNTP's (10 mM)
1 µl
12  µl
Forward Primer (10µm)
2.5 µl
30  µl
Reverse Primer (10µm)
2.5 µl
30  µl
Template DNA: rtTA
0.5 µl
6   µl
Phusion Polymerase
0.5 µl
6   µl
Water
33 µl
396 µl
Total
50 µl
600 µl
(Note: Add enzymes last)
PCR Reaction Procedures:
Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom.
Pipette 50ul from the Master Mix into four labled PCR tubes
Put in the thermocycler for the following cycles:
Initial Denaturation  | 95°C | 5 min    30 Cycles of:        Denaturation | 95°C | 30s       Annealing    | 50°C | 20s       Extension    | 72°C | 2 min Final Extension    | 72°C | 5 min Hold               | 4°C  | Forever
Keep samples for gel extraction on the following day.
Promoter
Concentration
PRM2
0.451
YDR124W
15.46
SAG1
12.10
PRM6
21.74
PRM1
4.186
PCL2
6.348
E. Coli Transformation for Increase of Plasmids:
Plasmids: HY3E (rtTA), HY130E (TRP), HY12E1 (URA3), HY34E2 (HIS2), HY111E2 (LEU2), HY4 (stock plasmid)
06/25/14
Glycerol Stocks of Constitutive Promoters in Yeast (CB008 and CB008 DB)
Diluted overnight cultures of constitutive promoters in yeast (CB008 and CB008DB)
added 250ul culture to 5ml YPD
put in 30°C incubator for 2 hours
Glycerol Stocks Procedure:
Obtain Glycerol Stock tubes from drawer near autoclave box. (Tubes have orange caps)
Add 350ul of 60% Glycerol or 420ul of 50% Glycerol to 350ul of cells.
Vortex well.
Label tubes and put into -70°C Freezer. (3rd row, 1st drawer)
Graphs of Flow for yGEM4:
Gel Map of pTEF1 Strains Yeast Homology:
Graph of Flow for All Inducible Promoters:
|  yGEM#  | Promoter|                        |---------|---------| |yGEM4    |   ASG7  | |yGEM5    |   PCL2  | |yGEM6    |   SAG1  | |yGEM10   |   PRM2  | |yGEM11   |   CLG1  | |yGEM12   | YDR124W | |yGEM13   |   PRM6  | |yGEM14   |   PRM2  | |yGEM15   |   ECM18 | |yGEM16   |   PRM3  |  |  yGEM#  |          Strain            | |---------|----------------------------| |yGEM23   |CB008 pTEF1-GFP-URA3        | |yGEM24   |CB008 pTEF1(M3)-GFP-URA3    | |yGEM25   |CB008 pTEF1(M6)-GFP-URA3    | |yGEM26   |CB008 pTEF1(M7)-GFP-URA3    | |yGEM27   |CB008 pTEF1(M10)-GFP-URA3   | |yGEM28   |CB008DB pTEF1-GFP-URA3      | |yGEM29   |CB008DB pTEF1(M3)-GFP-URA3  |  |yGEM30   |CB008DB pTEF1(M6)-GFP-URA3  | |yGEM31   |CB008DB pTEF1(M7)-GFP-URA3  | |yGEM32   |CB008DB pTEF1(M10)-GFP-URA3 |  | Promoter|  Owner  |                    |---------|---------| |PRM2     | George  | |ASG7     | Sabrina | |PCL2     | Eric    | |CLG1     | Derrick | |YDR124W  | Ianto   | |PRM6     | Ianto   | |PRM1     | Robert  | |ECM18    | Robert  | |PRM3     | Eleanor | |SAG1     | Jessica |
Diluted Overnight Cultures - Constitutive Promoters
Added 250ul of culture to 5ml YPD - put in incubator 30°C. Finished ~ 9:30am
06/26/14
Miniprep of Plasmids for rtTA
Miniprep Precedure:
Obtain E.Coli Cultures from 06/25/14 in 37°C  Incubation Room.
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Nanodrop Concentrations:
PRM2 = 107.0 ng/ul
ASG7 = 111.2 ng/ul
PCL2 = 96.49 ng/ul
CLG1 = 114.8 ng/ul
YDR124W = 90.53 ng/ul
HYM1 = 77.29 ng/ul
PRM6 = 99.83 ng/ul
PRM1 = 75.35 ng/ul
ECM18 = 18.81 ng/ul (to be remade)
PRM3 = 130.8 ng/ul
SAG1 = 104.1 ng/ul
pTEF1 = 182.3 ng/ul
PCR of rtTA Promoter
PCR done on 6/24/14 failed.
Added DMSO this time.
Became part of Jessica's rtTA portion of the project.
Inducible Promoters + rtTA into CB008 and CB008DB
PCR Purification of remaining pHY4 + inducible promoters + GFP (Digestion done by Derrick) (Backbone)
PCR of inducible promoter primer + rtTA with new rtTA RV primer
Materials
1x reaction
12x Master Mix
5x Phusion HF Buffer
10 µl
120 µl
dNTP's (10 mM)
1 µl
12  µl
Forward Primer (10µm)
2.5 µl
30  µl
Reverse Primer (10µm)
2.5 µl
30  µl
Template DNA: rtTA
0.5 µl
6   µl
Phusion Polymerase
0.5 µl
6   µl
Water
31.5 µl
396 µl
DMSO
1.5ul
30  ul
---------------------
-------------
----------------
Total
50 µl
600 µl
(Note: Add enzymes last)
PCR Reaction Procedures:
Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom.
Pipette 50ul from the Master Mix into four labled PCR tubes
Put in the thermocycler for the following cycles:
Initial Denaturation  | 95°C | 5 min    30 Cycles of:        Denaturation | 95°C | 30s       Annealing    | 50°C | 20s       Extension    | 72°C | 2 min Final Extension    | 72°C | 5 min Hold               | 4°C  | Forever
Keep samples for gel extraction on the following day.
06/27/14
rtTA PCR Purification
PCR Purification Prodedure:
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
To bind DNA, apply sample to column and spin for 1 min
Discard flow through
To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Streak Plate of Constitutive Promoters
As instructed by Kara, I made a streak plate with all 5 Constitutive Promoters on it.
Accidentally made the plate by stamping it instead of actually streaking it. It looks kinda funny.
Along with the streak plate, I also made cultures of of CB008 SAG1 and CB008 ECM18, along with two negative cultures of CB008. Cultures of the Contitutive Promoters were made also.
All cultures made with SD-Complete. (for Flow)
Gibson Assembly and Transformation
Gibson Assembly Procedures:
|        Materials       |  Amount  |    |------------------------|----------| |Gibson Master Mix (2x)  |   5ul    | |Backbone (pHY4+AFRP+GFP)|   3ul    | |Insert (rtTA + primer)  | Varies   | |ddH2O                   | Varies   |      Promoter|Concentration of Insert| Amount of Insert Needed  |H2O Needed|      --------| --------------------- | ------------------------ | -------- |     PRM2    | 107.0 ng/ul           | 1 ul                     |    1ul   |     ASG7    | 111.2 ng/ul           | 1 µl                     |    1ul   |     PCL2    | 96.49 ng/ul           | 1 µl                     |    1ul   |     CLG1    | 114.8 ng/ul           | 1 µl                     |    1ul   |     YDR124W | 90.53 ng/ul           | 1 µl                     |    1ul   |     HYM1    | 72.29 ng/ul           | 1 µl                     |    1ul   |     PRM6    | 99.83 ng/ul           | 1 µl                     |    1ul   |     PRM1    | 75.35 ng/ul           | 1 µl                     |    1ul   |     ECM18   | 18.87 ng/ul           | 3 µl                     |    0ul   |     PRM3    | 130.8 ng/ul           | 1 µl                     |    1ul   |     SAG1    | 104/1 ng/ul           | 1 µl                     |    1ul   |     Pos Ctrl| Gibson Positive Ctrl  | 2 µl Backbone, 2ul Insert|    1ul   |     Neg Ctrl| ASG7 Backbone         | 3 ul Backbone, 0ul Insert|    2ul   |
PRM2 did not have a backbone and thus could not be transformed yet.
Obtain Mach1 Cells from -70°C Freezer
25ul for 10ul Gibson Mix.
Mix Materials.
Incubate at 50°C for 1 hour.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 1 hr. (Warm plates also)
Plate on LB-CARB and keep in drawer over the weekend.
06/30/14
Results of Transformation:
| Promoter | Colonies|                   |----------|---------| | PRM2     |   N/A   | | ASG7     |    7    | | PCL2     |    6    | | CLG1     |   15    | | YDR124W  |    2    | | HYMI     |    0    | | PRM6     |    9    | | PRM1     |    8    | | ECM18    |    8    | | PRM3     |    5    | | SAG1     |    7    | | Pos Ctrl |  TMTC   | | Neg Ctrl |   10    |
Colony PCR of rtTA Transformation:
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 25ul of water. Do this for 3-5 colonies. Use 5ul for PCR reaction below.
Set up PCR Reaction below:
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
6 µl
10 µM Reverse primer
1 µl
6 µl
Water
3 µl
18 µl
Bacterial cells (template)
5 µl
-----
Total
90 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation   | 95°C | 5 min    30 Cycles of:        Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C  | Forever
Load PCR products on a gel. (5ul) (GoTaq already has loading dye)
For all positive bands on the gel, add rest of bacterial cells from PCR tubes to 5ml LB and grow overnight at 37°C for miniprep on the following day.
Gel of rtTA Tranformation Colony PCR:
Original Gel failed due to the bottom part of the gel disappearing. (possibly due to the bottom part not solidifying enough)
Jessica made PRM2 Backbone
ECM18 PCR'ed
Both were Gibson'ed and transformed.
Cultures were made of pHY4 AFRP's + rtTA.
5ml YPD + Transformed Colonies
Put in 37°C Incubator Room Shaker overnight.
07/01/14
Miniprep of Inducible Promoters + rtTA
Note: Try not to do more than one miniprep per promoter.
Miniprep Precedure: (did in triplicate for some reason)
Obtain E.Coli Cultures from 06/30/14 in 37°C  Incubation Room.
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Nanodrop Concentrations (ng/ul):
ASG7 1 = 203.0 ng/ul
ASG7 2 = 282.9 ng/ul
ASG7 3 = 244.9 ng/ul
PCL2 1 = 205.5 ng/ul
PCL2 2 = 205.3 ng/ul
PCL2 3 = 305.3 ng/ul
CLG1 1 = 155.5 ng/ul
CLG1 2 = 187.6 ng/ul
CLG1 3 = 230.0 ng/ul
YDR124W 1 = 89.44 ng/ul
PRM6 1 = 246.4ng/ul
PRM6 2 = 17.32 ng/ul
PRM6 3 = 154.0 ng/ul
PRM3 1 = 244.9 ng/ul
PRM3 3 = 180.1 ng/ul
SAG1 1 = 146.5 ng/ul
SAG1 2 = 243.8 ng/ul
SAG1 3 = 137.8 ng/ul
*Did not use PRM 1 #1, 2, and 3 or PRM3 #2 because the cultures did not grow.
Sent into sequencing.
Colony PCR for Screening Yeast - Constitutive Promoters + GFP
Used Streak plate in 4°C Fridge.
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
-----
10 µM Reverse primer
1 µl
6 µl
Water
5 µl
30 µl
Bacterial cells (template)
3 µl
-----
Total
96 ul
add 15ul of master mix to each tube.
Initial Denaturation   | 95°C | 5 min    30 Cycles of:        Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C  | Forever
Load 5ul of PCR products on a gel. (GoTaq already has loading dye)
10uM RV Primer = GFP RV Primer - kept in Primer Working Stock (2014 Primers)
10uM FW Primer = Individual pTEF1/mutant Primers kept in the -20°C freezer boxes of the respective owners. (pTEF1=Jeffrey, M3=Sabrina, M6=Eleanor, M7=Robert, M10=George)
M7 used M10 primer due to the primers being so similar. M7 ran out of its own primer.
Gel of Yeast Colony PCR (Constitutive Promoters)
Gel mostly failed - to be redone the following day.
Sent into sequencing.
Sequencing for pTEF promoters cancelled due to the absence of primers that were supposed to be sent in or selected on the promoter catalog.
07/02/14
Re-Colony PCR for Yeast - Constitutive Promoters + GFP
Colony PCR for Screening Yeast - Constitutive Promoters + GFP
Used Streak plate in 4°C Fridge.
picked 3 different colonies.
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
-----
10 µM Reverse primer
1 µl
6 µl
Water
5 µl
30 µl
Bacterial cells (template)
3 µl
-----
Total
96 ul
add 15ul of master mix to each tube.
Initial Denaturation   | 95°C | 5 min    30 Cycles of:       Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C  | Forever
Load 5ul of PCR product on a gel. (GoTaq already has loading dye)
10uM RV Primer = GFP RV Primer - kept in Primer Working Stock (2014 Primers)
10uM FW Primer = Individual pTEF1/mutant Primers kept in the -20°C freezer boxes of the respective owners. (pTEF1=Jeffrey, M3=Sabrina, M6=Eleanor, M7=Robert, M10=George)
M7 used M10 primer due to the primers being so similar. M7 ran out of its own primer.
Gel of Yeast Colony PCR (Const. + GFP):
Yeast PCR Purification
To extract Yeast DNA for Sequencing
PCR Purification Prodedure:
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
To bind DNA, apply sample to column and spin for 1 min
Discard flow through
To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Nanodrop Concentrations:
pTEF1       = 12.30 ng/ul
pTEF1 (M3)  = 6.242 ng/ul
pTEF1 (M6)  = 33.68 ng/ul
pTEF1 (M7)  = 39.72 ng/ul
pTEF1 (M10) = 19.43 ng/ul
Did not send into sequencing due to the conclusion that the constitutive promoters were successfully transformed based on past sequencing and flow cytometry.
I restreaked the streak plate from the patch plate with the colonies marked (✓) on them. I did not trust the other streak plate. Incubated in 30°C incubator.
Re-Transformation of pHY4 Inducuble Promoters + rtTA
Gibson Assembly Procedures:
|        Materials       |  Amount  |    |------------------------|----------| |Gibson Master Mix (2x)  |   5ul    | |Backbone (pHY4+AFRP+GFP)|   4ul    | |Insert (rtTA + primer)  |   1ul    | |ddH2O                   |   0ul    | |Total                   |   10ul   |
Obtain NEB5α (C2987) Cells instead of Mach1 cells from -70°C Freezer
25ul for 10ul Gibson Mix.
Mix Materials.
Incubate at 50°C for 1 hour.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 30 min rather than 1 hr. (Warm plates also)
Plate on LB-CARB and incubate in 37°C Incubator Room.
Re-Transformed Promoters: (never been transformed, not enough colonies, etc)
PCL2
YDR124W
HYM1
PRM6
PRM1
ECM18
PRM3
PRM2
Negative Control (heated at 55°C for 40 min due to being made late)
ALWAYS MAKE A NEGATIVE CONTROL!
Jessica did a Colony PCR for rtTA Promoters and a Miniprep for the 1st Transformation.
07/03/14
Results of Re-Transformtation of Inducible Promoters + GFP
Few to no colonies grew onthe transformed plates, which leads me to think that the Gibson Assembly failed because only PRM2 worked and was Gibson'ed prior to the other transformed promoters.
PRM2 grew many tiny colonies.
Promoter
Colonies
PRM2
TMTC
PCL2
2
PRM6
3
PRM3
2
YDR124W
~ 40
PRM1
0
ECM18
0
HYM1
0
Neg Ctrl
0
Possible Explanations for Failure:
Gibson Assembly failed due to bad Gibson Master Mix
Backbone concentrations were not high enough, which would make sense because PRM2 had a new backbone which had a higher concentration than the rest of the promoters.
PRM2 Backbone Concentration: 208.7 ng/ul
Backbone Digestion
Jessica remade the backbones, but the concentrations are still very low.
Used the minipreps from my box in the -20°C fridge.
Colony PCR of Miniprep (Inducible Promoter + GFP)
|Reagents                      | 1X   | 20X   | |----------------------------- |------| ----- | |2X GoTaq Green PCR Master Mix | 10µl | 200µl | |10 µM Forward primer          | 1µl  | ----- | |10 µM Reverse primer          | 1µl  | 20µl  | |Water                         | 5µl  | 100µl | |Miniprep                      | 1µl  | ----- |  |Total                         | ---- | 320ul |      Initial Denaturation   | 95°C | 5 min        30 Cycles of:           Denaturation | 95°C | 45s           Annealing    | 55°C | 30s           Extension    | 72°C | 1min per kb     Final Extension    | 72°C | 10m     Hold               |  4°C  | Forever
10uM FW Primer = individual inducible promoter primers - kept in Jessica's box.
10uM RV Primer = rtTA primer - kept in 2014 iGEM primer working stock box.
Sent into sequencing.
Gel of Miniprep Colony PCR (Inducible Promoter + GFP):
07/07/14
Confusion with Minipreps
Due to a confusion on 7/3/14, all minipreps from before 7/7/14 are to be thrown away from my box. Only the needed minipreps are to be kept.
Inserts were remade for no reason. (old inserts were fine)
Backbones may not have been digested properly. (minimum of 2 hrs needed)
07/08/14
PCR Amplification of new rtTA
Amplification of new rtTA with homology to AFRP's.
extract rtTA from pTS47 (~100ng/ul)
Materials
1x reaction
12x Master Mix
5x Phusion HF Buffer
10 µl
120 µl
dNTP's (10 mM)
1 µl
12 µl
Forward Primer (10µm)
2.5 µl
individually added
Reverse Primer (10µm)
2.5 µl
30 µl
Template DNA:rtTA
1 µl
12 µl
Phusion Polymerase
0.5 µl
6 µl
Water
31 µl
372 µl
DMSO
1.5 µl
18 µl
Total
50 ul
570 ul
(Note: Add enzymes last)
PCR Reaction Procedures:
Mix the regents in a 4.5x Master mix on ice. Make sure to mix well since enzyme is viscous and sinks to the bottom.
Pipette 50ul from the Master Mix into four labled PCR tubes
Put in the thermocycler for the following cycles:
Initial Denaturation   | 95°C | 5 min    30 Cycles of:        Denaturation | 95°C | 30 sec       Annealing    | 55°C | 30 sec       Extension    | 72°C | 2 min Final Extension    | 72°C | 5 min Hold               | 4° C | Forever
Keep samples for gel extraction on the following day.
Nanodrop Concentration:
PRM1 = 115.9 ng/ul
PRM2 = 176.6 ng/ul
PRM3 = 169.2 ng/ul
PRM6 = 154.9 ng/ul
ECM18 = 122.5 ng/ul
SAG1 = 161.9 ng/ul
YDR124W = 99.70 ng/ul
CLG1 = 153.5 ng/ul
ASG7 = 139.9 ng/ul
HYM1 = 123.5 ng/ul
PCL2 = 1-2.3 ng/ul
07/09/14
Gibson Assembly of rtTA Inducible Promoters (Transformation #3)
Gibson Assembly Procedures:
|        Materials       |  Amount  |    |------------------------|----------| |Gibson Master Mix (2x)  | 5ul    | |Backbone (pHY4+AFRP+GFP)|   3ul    | |Insert (rtTA + primer)  |   1ul    | |ddH2O                   |   1ul    | |Total                   |   10ul   |
Mix Materials.
Incubate at 50°C for 1 hour.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 1 hr. (Warm plates also)
Plate on LB-CARB and incubate in 37°C Incubator Room overnight.
07/10/14
Results of Transformation from 7/9/14
Most of the plates grew well except:
PRM1
PRM2
PRM6
HYM1
Positive control worked.
Negative control also worked.
maybe the backbones were not digested enough?
Going to redo backones for PRM1 and PRM2 (to be redigested)
Backbone Digest for PRM1 and PRM2
Materials
PRM1
PRM2
NotI
0.5 ul
0.5 µl
XhoI
0.5 µl
0.5 µl
CutSmart
5 µl
5 ul
pGEM
0 µl
4 µl
Original Digest
28 µl
40 µl
H2O
16 µl
0 µl
Total
50 ul
50 ul
Incubate at 37°C for at least 2 hours.
PCR Purify.
PCR Purification Procedure:
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
To bind DNA, apply sample to column and spin for 1 min
Discard flow through
To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Concentrations of Backbone Digestions of PRM1 and PRM2 were too low; not usable.
Colony PCR Gel of Transformation #3:
07/11/14
Miniprep of Transformation #3
Made cultures of the the transformations.
Sent Transformation #3 into sequencing.
07/14/14
Results of Sequencing and Colony PCR
PCR showed that rtTA and GFP were both present.
Sequencing showed that GFP was in the backbone, not rtTA
rtTA was probably not in the backbone and was floating near the colonies instead, so when we picked up the colonies for Colony PCR, we also picked up the rtTA with them.
Will use Backbone set #2 in Jessica's box along with Derrick's PRM1 and PRM2 backbones from 6/09/14.
Will use the new inserts from 7/11/14 in my box.
Gibson Assembly and Transformation #4
Gibson Assembly Procedures:
|        Materials       |  Amount  |    |------------------------|----------| |Gibson Master Mix (2x)  |   5ul    | |Backbone (pHY4+AFRP+GFP)|  Varies  | |Insert (rtTA + primer)  |   1ul    | |ddH2O                   |  Varies  | |Total                   |   10ul   |  Promoter |Concentration of Backbone| Amount of Insert Needed  |H2O Needed|  |------- | ----------------------- | ------------------------ | -------- | PRM1     | ~50.0 ng/ul             | 1 ul                     |   3 ul   | PRM2     | ~50.0 ng/ul             | 1 µl                     |   3 ul   | PRM3     | 15.20 ng/ul             | 1 µl                     |   1 ul   | PRM6     | 21.74 ng/ul             | 1 µl                     |   1 ul   | ECM18    | 21.39 ng/ul             | 1 µl                     |   1 ul   | SAG1     | 12.10 ng/ul             | 1 µl |   1 ul   | YDR124W  | 15.46 ng/ul             | 1 µl                     |   1 ul   | CLG1     | 16.59 ng/ul             | 1 µl                     |   1 ul   | ASG7     | 23.77 ng/ul             | 3 µl                     |   1 ul   | HYM1     | 12.96 ng/ul             | 1 µl                     |   1 ul   | PCL2     | 6.348 ng/ul             | 1 µl                     |   0 ul   | Pos Ctrl | Gibson Positive Ctrl    | 3 µl Backbone 1 ul Insert|   0 ul   | Neg Ctrl | PRM1 Backbone           | 1 ul Backbone 0 ul Insert|   4 ul   |
Concentrations of PRM1 and PRM2 Backbones were around 200 ng/ul. I diluted them 1:4 down to 50 ng/ul.
Mix Materials.
Incubate at 50°C for 1 hour.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 1 hr. (Warm plates also)
Plate on LB-CARB and keep in drawer over the weekend.
07/15/14
Glycerol Stocks for pTET + GFP + rtTA
Made cultures of pTET + GFP + Const. Promoter + rtTA.
CB008: pTEF1, M6, M7, and M10
CB008DB: M6, and M7
Made a streak plate with the strains on them.
Did a PCR of the rtTA Backbone and Inducible Promoter Insert.
Used pGEM20 rtTA backbone (~100 ng/ul dilution)
Gel Picture of PCR:
Transformation of HY130E into DH5Fα Cells
All of DNA (~2ul) + 50ul DH5α.
Put on ice for 10 min.
Heatshock for 45 sec at 42°C.
Put on ice for at least 2 min.
Add 250ul SOC.
Incubate for 30 min in 37°C Room Shaker.
Plate 100ul on LB-CARB plate.
Plate 200ul on another LB-CARB plate.
07/16/14
rtTA Insert
Used pGEM16 Template to make rtTA insert.
Gel Extraction of rtTA PCR
Gel Extraction Procedure:
Cut Gel
Weigh it in a colorless tube
Add 3 volumes Buffer QG to 1 volime Gel (100mg ~ 100µl)
Incubate @ 50°C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)
Add 1 gel volume isopropanal to the sample and mix
Place a QlAquick soin column in a provided 2ml collection tube
Place sample in column & spin for 1 min -> discard flow through
To wash add 0.75 ml Buffer PE to column & centrifue for 1 min, then dry spin
Place column in 1.5 ml tube
Add 30µl H2O & centrifuge for 1 min.
Gel #1:
Gel #2:
Fragment Weights: (try to cut smaller pieces)
PRM1 = 0.54 g = 540ul
PRM2 = 0.56 g = 560ul
PRM3 = 1.15 g = 1150ul
PRM6 = 1.21 g = 1210ul
ECM18 = 0.61 g = 610ul
SAG1 = 0.68 g = 680ul
YDR124W = 0.45 g = 450ul
CLG1 = 0.4 g = 400ul
ASG7 = 0.31 g = 310ul
HYM1 = 0.50 g = 500ul
PCL2 = 0.42 g = 420ul
M3 = 0.69 g = 690ul
Initial Concentrations of Gel Extraction were too low. Re-PCRed and obtained new concentrations. See Jeffrey's notebook for initial concentrations.
Assisted Jeffrey in doing gel extraction for M3.
New Concentrations:
PRM1 = 102.2 ng/ul
PRM2 = 110.6 ng/ul
PRM3 = 135.5 ng/ul
PRM6 = 130.7 ng/ul
ECM18 = 136.8 ng/ul
SAG1 = 152.4 ng/ul
YDR124W = 78.89 ng/ul
CLG1 = 133.7 ng/ul
ASG7 = 68.15 ng/ul
HYM1 = 91.08 ng/ul
PCL2 = 86.63 ng/ul
M3 = 47.19 ng/ul
Stored in Jessica's box in -20°C fridge.
07/17/14
Gibson and Transformation (Inducible Promoter + rtTA) #5
Gibson Assembly Procedure:
| Materials       |  Amount  |    |------------------------|----------| |Gibson Master Mix (2x)  |   5ul    | |Backbone (pHY4+AFRP+GFP)|  Varies  | |Insert (rtTA + primer)  |  Varies  | |ddH2O                   |  Varies  | |Total                   |   10ul   |  Promoter |Amount of Backbone Needed| Amount of Insert Needed  |H2O Needed|  |--------| ----------------------- | ------------------------ | -------- | PRM1     | 1ul                     | 1 ul                     |   3 ul   | PRM2     | 3ul                     | 1 µl                     |   3 ul   | PRM3     | 3ul                     | 1 µl                     |   1 ul   | PRM6     | 3ul                     | 1 µl                     |   1 ul   | ECM18    | 3ul                     | 1 µl                     |   1 ul   | SAG1     | 3ul                     | 1 µl                     |   1 ul   | YDR124W  | 3ul                     | 1 µl                     |   1 ul   | CLG1     | 3ul                     | 1 µl                     |   1 ul   | ASG7     | 3ul                     | 2 µl                     |   0 ul   | HYM1     | 3ul                     | 1 µl                     |   1 ul   | PCL2     | 4ul                     | 1 µl                     |   0 ul   | Pos Ctrl | 3ul                     | 2 ul                     |   0 ul   | Neg Ctrl | 3ul                     | 1 ul                     |   1 ul   |
Used YDR124W Backbone for Negative Control
Mix Materials.
Incubate at 50°C for 1 hour.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 1 hr. (Warm plates also)
Plate on LB-CARB and keep in drawer over the weekend.
07/18/14
Results of Transformation #5
| Promoter | Colonies|                   |----------|---------| | PRM1     |    0    | | PRM2     |    4    | | PRM3     |    0    | | PRM6     |  TMTC   | | ECM18    |    0    | | SAG1     |  ~25    | | YDR124W  |    0    | | CLG1     |    0    | | ASG7     |    0    | | HYM1     |    0    | | PCL2     |    0    | | Pos Ctrl |    0    | | Neg Ctrl |    0    |
Used only 100ul for transformation.
Going to try again with the remaining 150ul.
Stored in drawer at room temperature for transformation over the weekend.
Since Positive Control did not work, I suspect that either the positive control was bad or that the Gibson was bad.
07/21/14
Results of Transformation #6
| Promoter | Colonies|                   |----------|---------| | PRM1     |   30    | | PRM2     |   30    | | PRM3     |   40    | | PRM6     |  TMTC   | | ECM18    |   30    | | SAG1     |   50    | | YDR124W  |   20    | | CLG1     |   20    | | ASG7     |   20    | | HYM1     |   30    | | PCL2     |   30    | | Pos Ctrl |   20    | | Neg Ctrl |   20    |
Since Positive Control did grow very much and the Negative Control worked, I suspect that either the positive control was bad or that the Gibson was bad.
Incubated for 4 more hours in the 37°C Incubator Room so that the colonies could grow slightly bigger. (All the colonies are very tiny)
Made cultures for PRM2, PRM6, and SAG1 for Colony PCR/Miniprep.
Sent Colony PCR/Miniprep of PRM2, PRM6, and SAG1 into sequencing.
Since most of the transformations did not actually seem to work, we will be trying Seamless to see if the transformations are any better.
Backbone Redigestion
Decided to redigest backbones for higher concentrations
Backbones: PRM3, SAG1, YDR124W, HYM1, PCL2
pGEM #: pGEM3, pGEM6, pGEM7, pGEM10, pGEM11
Materials
PRM3
SAG1
YDR124W
HYM1
PCL2
NotI
0.5 ul
0.5 µl
0.5 ul
0.5 ul
0.5 ul
XhoI
0.5 µl
0.5 µl
0.5 ul
0.5 ul
0.5 ul
CutSmart
5 µl
5 ul
5 ul
5 ul
5 ul
pGEM
5 µl
4 µl
4 ul
7 ul
7 ul
H2O
34 µl
40 µl
40 ul
37 ul
37 ul
Total
50 ul
50 ul
50 ul
50 ul
50 ul
Incubate at 37°C for at least 2 hours.
Heat inactivate at 65°C for 20 min.
Add 10ul of 6x loading dye to each sample.
Gel extract.
Gel Extraction Procedure:
Cut Gel
Weigh it in a colorless tube
Add 3 volumes Buffer QG to 1 volime Gel (100mg ~ 100µl)
Incubate @ 50° C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)
Add 1 gel volume isopropanal to the sample and mix
Place a QlAquick soin column in a provided 2ml collection tube
Place sample in column & spin for 1 min -> discard flow through
To wash add 0.75 ml Buffer PE to column & centrifue for 1 min, then dry spin
Place column in 1.5 ml tube
Add 30µl H2O & centrifuge for 1 min.
New Concentrations of Backbones:
Promoter
Concentration
PRM3
13.48 ng/ul
SAG1
44.26 ng/ul
YDR124W
82.00 ng/ul
HYM1
39.66 ng/ul
PCL2
82.97 ng/ul
Seamless Cloning and Assembly Reaction
Seamless Procedure:
Promoter |Amount of Backbone Needed| Amount of Insert Needed  | Seamless |  |--------| ----------------------- | ------------------------ | -------- | PRM1     | 1ul                     | 2 ul                     |   3 ul   | PRM2     | 3ul                     | 2 µl                     |   3 ul   | PRM3     | 3ul                     | 1.5 µl                   |   1 ul   | PRM6     | 3ul                     | 1.5 ul                   |   1 ul   | ECM18    | 3ul                     | 1.5 µl                   |   1 ul   | SAG1     | 3ul                     | 1.3 µl                   |   1 ul   | YDR124W  | 3ul                     | 2.5 µl                   |   1 ul   | CLG1     | 3ul                     | 1.5 µl                   |   1 ul   | ASG7     | 3ul                     | 3 µl                     |   0 ul   | HYM1     | 3ul                     | 2.2 µl                   |   1 ul   | PCL2     | 4ul                     | 2.3 µl                   |   0 ul   |  |        Materials                |  Amount  | |---------------------------------|----------| |Inserts (200ng each)             |   x ul   | |Linear Cloning Vector (50ng each)|   y ul   | |GeneArt 2X Enzyme Mix            | x+y ul   |
In a microcentrifuge tube, set up the seamless cloning and assembly reaction. It is CRUTIAL that you add the GeneArt 2x Enzyme Mix as the LAST COMPONENT.
Quickly thaw the GeneArt 2x Enzyme Mix on ice and pipette up and down to mix thoroughly. Add (x+y) ul of thawed enzyme. Mix to each reaction. Immediately return GeneArt 2x Enzyme Mix to -80°C Freezer.
Mix the reaction components completely by pipetting up and down SLOWLY 3 times and then gently tap the side of the tubed 3-5 times to mix.
If needed, briefly centrifge (< 500rpm for < 5sec) to collect reaction components to the bottom of the microcentrifuge tube.
Incubate the reaction mix at room temperature for 15-30 min. Do not incubate longer than 30 min!
After incubation is complete, plate the reaction mix on ice for 2-5 min before proceeding to transformation step. Do not let the samples stay on ice for more than 5 min before transformation.
Transformation:
Use 3ul of the seamless cloning and assembly reaction to transform NEB5α cells. Do not use electrocompetent cells.
Follow normal Transformation procedures.
Add 25ul Competent Cells to 10ul Gibson Mix. (ADD ON ICE)
Incubate on ice for 30 min.
Incubate at 42°C for 45 sec.
Incubate on ice for at least 2 min.
Add 250ul SOC Media (Make sure it isn't cloudy) to cells.
Shake in 37°C Incubator Room for 1 hr. (Warm plates also)
Plate on LB-CARB and keep in drawer over the weekend.
Miniprep of Transformations (PRM2, PRM6, SAG1)
Used cultures made on 7/18/14 (put in incubator on 7/20/14)
Miniprep Precedure:
Obtain E.Coli Cultures from 06/16/14 in 37°C  Incubation Room.
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Promoter
Concentration
PRM2
225.0 ng/ul
PRM6
491.5 ng/ul
SAG1
439.0 ng/ul
Stored in my box in -20°C fridge.
Sent into sequencing.
Anasuya came in and talked to us about some useful articles to read: mine is the one about "Binding of the B cell Activation..."
07/22/14
Results of Transformation
Seamless Transformation from 7/21/14
| Promoter | Colonies|                   |----------|---------| | PRM1     |  some   | | PRM2     |  some   | | PRM3     |    0    | | PRM6     |    0    | | ECM18    |    0    | | SAG1     |    0    | | YDR124W  |    0    | | CLG1     |    0    | | ASG7     |    0    | | HYM1     |    0    | | PCL2     |    0    | | Pos Ctrl |  some   | | Neg Ctrl |  some   |
Possible Explanations:
Used wrong inserts
Seamless went wrong
Mutations of template
Backbones not digested properly (not likely)
PRM1 and PRM2 backbones have GFP
Concentrations of both insert and backbone too low
Final Concensus: We will use another method to transform the cells:
Instead of using Gibson or Seamless, we will PCR with primers for all 7kb.
Currently ordering primers.
Meanwhile, transformations from 7/18/14 worked - colonies grew for all plates
will attempt colony PCR to determine the presence of rtTA
plates to be PCR'ed: PRM3, ECM18, YDR124W, CLG1, ASG7, HYM1, and PCL2
plates to be excluded: PRM1 and PRM2 (both backbones have GFP), PRM6 (has both mutation and rtTA, determined by previous sequencing), SAG1 (backbone has GFP but has recently been redigested), '+' control, '-' control
Culturing pGEM #1-11 (except pGEM7) to make pGEM for backbone digestion tomorrow.
07/23/14
pGEM Miniprep and Overnight Backbone Digestion
Collected cultures from 37°C incubator room - shaker was off for some reason - I turned it back on
Cells grew ok despite shaker incident
pGEM 4 and pGEM6 did not grow very well
Did a miniprep and gel extraction for backbone digestion
07/24/14
Digested Backbones
Did a gel extract.
Concentrations of Backbones:
PRM1 = 60.10 ng/ul
PRM2 = 61.79 ng/ul
PRM3 = 41.36 ng/ul
PRM6 = 55.34 ng/ul
ECM18 = 55.11 ng.ul
SAG1 = 89.55 ng/ul
YDR124W = 35.55 ng/ul
CLG1 = 49.42 ng/ul
ASG7 = 40.27 ng/ul
HYM1 = 33.18 ng/ul
PCL2 = 29.47 ng/ul
Backbones stored in my bix in -20°C fridge.
07/25/14
Transformation
Collected overnight digests of rtTA and Msn2 from 37°C incubator room.
PCR Purification
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
To bind DNA, apply sample to column and spin for 1 min
Discard flow through
To wash, add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Concentrations of digests:
rtTA = 28.98 ng/ul
Msn2 = 34.86 ng/ul
Ligation
1:3 Backbone to Insert
Materials:
Reagents
1x
10x ligase buffer
1 µl
Backbone
2 µl
Msn2
1 µl
rtTA
1 ul
T4 DNA Ligase
1 µl
H2O
4 µl
Total
10 µl
Keep at room temperature for at least 2 hours.
Transformation
Materials
1x
Ligation
10µl
E. Coli Competent Cells
25ul
Mix both together and keep on ice for 30 minutes.
42°C Heat shock for 45 seconds.
Put immediately on ice for at least 2 minutes.
Add 250 ul of SOC media, shake for 1 hr at 37°C.
Plate on selective media and store at room temperature over the weekend.
Had extra cells, so I added 50ul to PCL2.
07/29/14
Colony PCR of AFRP + rtTA
Use rtTA FW primer instead of individual promoter primers
Cultures to use for Colony PCR: PRM2-1, PRM3-2, PRM6-1, ECM18-1, SAG1-3, ASG7-2, HYM1-3
use plates for the rest of the AFRPs
Use primers 40 and 41, rtTA FW and RV
Colony PCR Procedure:
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 25ul of water.Do this for 5 colonies. Use 5ul for PCR reaction below.
Set up PCR Reaction below:
Reagents
1X
34X
2X GoTaq Green PCR Master Mix
10 µl
340 µl
10 µM Forward primer
1 µl
34 µl
10 µM Reverse primer
1 µl
34 µl
Water
1.5µl
51 µl
Bacterial cells (template)
5 µl
------
DMSO
1.5 µl
51 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation   | 95° C | 5 min    35 Cycles of:        Denaturation | 95° C | 45s       Annealing    | 55° C | 30s       Extension    | 72° C | 1min per kb Final Extension    | 72° C | 10m Hold               |  4°C  | Forever
Load PCR products on a gel. (5ul) (GoTaq already has loading dye)
For all positive bands on the gel, add rest of bacterial cells from PCR tubes to 5ml LB and grow overnight at 37°C for miniprep on the following day.
Gel Map of Colony PCR:
07/30/14
Re-Colony PCR of SAG1
Redid Colony PCR for SAG1
Miniprep of Colony PCR
Cultures to be miniprep-ed:
PRM1.6
PRM2.1
PRM3.2
PRM6.4
ECM18.1
YDR124W.6
CLG1.5
ASG7.4
HYM1.3
PCL2.6
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
07/31/14
Sequencing of AFRP + rtTA
Sequencing of 7/30/14 Minipreps
Sent in: PRM1, PRM2, PRM3, PRM6, ECM18, YDR124W, CLG1, ASG7, HYM1, PCL2
SAG1 on hold - PRM6 and PCL2 need to be re-Colony PCRed.
Did a Colony PCR for PRM6, SAG1, and PCL2
Made cultures with 5ml LB+CARB and 20ul cells
PRM6 = 7, 8, 9
SAG1 = 10, 11, 12
PCL2 = 7, 8, 9 (plate may have been contaminated, used white colonies)
Colony PCR failed - no rtTA - decided to re-transform
Ligation
see procedure above
stored at 16°C overnight (room temperature)
08/01/14
Transformation of Ligation into NEB5α (C2987) cells
see procedure above - used 25ul cells - stored in 37°C incubator room -Transformed PCL2, PRM6, SAG1, and negative control
Yeast Transformation
linearized DNA
boiled salmon sperm at 100°C for 10 min
Yeast Transformation Procedure:
Reagents
YPD
1 M LiOAc
10X TE pH 7.5
1X TE pH 7.5, 0.1 M LiOAc
50% PEG 3350
DMSO
Salmon Sperm DNA (ssDNA)
PEG is viscous, so pipette slowly to prevent air bubbles from forming.
boil ssDNA aliquots for 10 min, then ice down for 10 min before use.
Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C
Set up digest to linearize DNA
Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C
Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)
Harvest cells in centrifuge - 3000 rpm, 2-5 min
Wash with 1 ml 0.1 M LiOAin TE
Pellet cells - 3000 rpm , 2-5 min
Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation
to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA
Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc) Optional: Add 75 µl DMSO definitely do
Vortex
Incubate 42° C for 30m & begin drying plates
Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)
Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD
Plate on selective media
Incubate 1-3 days at 30°C
08/04/14
Results of E.Coli and Yeast Transformations
E.Coli plates:
PRM6 = 30
SG1 = 20
PCL2 = 20
Negative Control = 20
Yeast plates:
CB008: all plates grew at least 30+ colonies
CB008DB: all plates grew at least 30+ colonies
YPD became contaminates - plates may be compromised
Did a Colony PCR on successful E.Coli plates.
FW primer = rtTA FW #40
RV primer = rtTA RV #41
cultures = 5ml LB+CARB + 20ul cells
SD, YPD, and LB were contaminated, as a result, some of our yeast plates became contaminated.
Jessica did a Colony PCR for the successful Yeast Plates
Used the uncontaminated colonies for Colony PCR
See Colony PCR procedures for both E.Coli and Yeast above.
08/05/14
Miniprep of E.Coli Cultures
Cultures = PRM6, SAG1, PCL2
AFRP+rtTA
Miniprep Precedure:
Obtain E.Coli Cultures from 8/4/14 in 37°C  Incubation Room.
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix. DO NOT VORTEX.
Add 350 µl Buffer N3 and invert immediately 4-6 times.
Centrifuge for 10 min at 13,000 rpm.
Add supernatant to QIAprep spin column.
Centrifuge for 30-60s - Discard flow through.
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through.
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer.
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
Miniprep Concentrations:
PRM6 = 611.2 ng/ul
SAG1 = 478.1 ng/ul
PCL2 = 384.8 ng/ul
Sent into sequencing
Linearization of E.Coli Minipreps
DIgestion Materials:
Reaction
Amount
DNA
~2000ng
Pme1
0.5ul
CutSmart
2.5ul
H2O
x ul
Total
25ul
Concentration Calculation:
Promoter
Concentration of Miniprep
Amount of Miniprep Needed
H2O
PRM1
376.7 ng/ul
6 ul
16 ul
PRM2
356.8 ng/ul
6 µl
16 ul
PRM3
364.5 ng/ul
6 µl
16 ul
PRM6
611.2 ng/ul
4 ul
18 ul
ECM18
371.7 ng/ul
6 µl
16 ul
SAG1
478.1 ng/ul
5 µl
17 ul
YDR124W
328.4 ng/ul
6 µl
16 ul
CLG1
543.1 ng/ul
4 µl
18 ul
ASG7
348.8 ng/ul
6 µl
16 ul
HYM1
449.6 ng/ul
5 µl
17 ul
PCL2
384.8 ng/ul
6 µl
16 ul
Add reagents and incubate for 2 hours at 37°C  Incubation Room.
Also assisted Jeffrey in lineaerizing 11E3128 (423.0 ng/ul) and pGEM22 (384.8 ng/ul).
Transformed linearizations into yeast
CB008: PRM6, ECM18, SAG1, PCL2
CB008DB: all 11 promoters
Forgot to make negative control
Yeast Transformation Procedure:
Reagents
YPD
1 M LiOAc
10X TE pH 7.5
1X TE pH 7.5, 0.1 M LiOAc
50% PEG 3350
DMSO
Salmon Sperm DNA (ssDNA)
PEG is viscous, so pipette slowly to prevent air bubbles from forming.
boil ssDNA aliquots for 10 min, then ice down for 10 min before use.
Previous Day : Grew yeast strain to be transformed in 5-10 mL YPD overnight at 30° C
Set up digest to linearize DNA
Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C
Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)
Harvest cells in centrifuge - 3000 rpm, 2-5 min
Wash with 1 ml 0.1 M LiOAin TE
Pellet cells - 3000 rpm , 2-5 min
Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation
to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA
Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc) Optional: Add 75 µl DMSO definitely do
Vortex
Incubate 42° C for 30m & begin drying plates
Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)
Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD
Plate on selective media
Incubate 1-3 days
08/08/14
Results of Transformation
CB008 = all plates grew 20-30 colonies except for PRM6, which did not grow.
CB008DB = all plates grew 20-30 colonies
Both plates had small colonies
Did a Colony PCR on the Transformations
Flow Testing
CB008 Const+rtTA+GFP+mfa+pAGA mCherry
Plate map for all time points (4 plates):
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 m6-1]----]       [CB008 m10-3]----] B [CB008 m6-2]----]       [CB008DB m7-1]---] C [CB008 m6-3]----]       [CB008DB m7-2]---] D [CB008 m7-1]----]       [CB008DB m7-3]---] E [CB008 m7-2]----]  F [CB008 m7-3]----]            G [CB008 m10-1]---]  H [CB008 m10-2]---]
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12
See notes on flow in Eric's notebook.
08/12/14
Colony PCR of Yeast for rtTA and mfa
Boil Yeast at 95°C for 20 minutes.
Using a sterile pipette tip (p200 tip is good), innoculate a single colony and mix in 15ul of NaOH. Take 3 colonies from each plate. Use 5ul for PCR reaction below.
Set up PCR reaction below:
Reagents (rtTA)
1X
43X
2X GoTaq Green PCR Master Mix
10 µl
430 µl
10 µM Forward primer
1 µl
43 µl
10 µM Reverse primer
1 µl
43 µl
Water
1.5µl
64.5 µl
cells (template)
5 µl
-------
DMSO
1.5ul
64.5 ul
FW primer = pSV606
Reagents (rtTA)
1X
7X
2X GoTaq Green PCR Master Mix
10 µl
70 µl
10 µM Forward primer
1 µl
7 µl
10 µM Reverse primer
1 µl
7 µl
Water
1.5µl
10.5 µl
cells (template)
5 µl
-------
DMSO
1.5ul
10.5 ul
Add 15 ul of Master Mix into each tube.
Initial Denaturation    | 95°C | 5 min    35 Cycles of:        Denaturation | 95°C | 45s       Annealing    | 55°C | 30s       Extension    | 72°C | 1min per kb Final Extension    | 72°C | 10m Hold               |  4°C | Forever
Load PCR products on a gel. (5ul) (GoTaq already has loading dye)
Results of Colony PCR:
08/13/14
Doxycyline Dilutions (Dilutions by Ianto)
Stock in Siberia Fridge (Middle Shelf, Middle Rack, Top Drawer, 3rd Box) 100 mg/ml
x
Concentration
Take from x
Amount of x Needed
H2O Needed
A
60
Stock
60 ul
940 ul
B
30
Stock
30 ul
970 ul
C
9
A
150 ul
850 ul
D
6
A
50 ul
900 ul
E
3
A
50 ul
950 ul
F
0.9
C
100 ul
900 ul
G
0.6
D
100 ul
900 ul
H
0.3
E
100 ul
900 ul
I
0.09
F
100 ul
900 ul
J
0.06
G
100 ul
900 ul
K
0.03
H
100 ul
900 ul
Vortex while like making and before adding to each dilution.
Assisted Derrick in running plates (time points: 0, 1.5, 3, 5)
See Derrick's lab norebook for reference.
08/17/14
Due to contamination of the plates from the flow run of 8/13/14, another run will be needed of the same experiment.
Plate had large clumps of cells due to not being cleaned out well.
Make sure to put finished shaker plate in bleach bin (near Hyun's desk, under the table in a blue bin)
prepared cultures of pTEF1, M6, M7, and M10 to be diluted on the following day for flow.
CB008 const+rtTA+GFP+mfa+pAGA1+mCherry
Plate Map
1
2
3
4
5
6
7
8
9
10
11
12
A
1
1
1
1
1
1
7
7
7
7
7
7
B
1
1
1
1
1
1
10
10
10
10
10
10
C
1
1
1
1
1
1
10
10
10
10
10
10
D
6
6
6
6
6
6
10
10
10
10
10
10
E
6
6
6
6
6
6
F
6
6
6
6
6
6
G
7
7
7
7
7
7
H
7
7
7
7
7
7
1 = PTEF1 6 = M6 7 = M7 10 = M10
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6       [----------6 µg/ml----------] 7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12      [----------6 µg/ml----------]
08/18/14
CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2
Ran plate with 5 time points (0, 1.5, 3, 5, 8)
added 15ul of dox to induce
Added 10ul of cycloheximide to each well to stop growth.
Finished running the first two plates (0hr and 1.5hr)
Slow run may be due to the cells being overdilute. (source: Kara)
Plate run to be continued and finished on 8/19/14
CB008DB Const+rtTA+GFP+mfa+pAGA+mCherry
Started cultures for this experiment.
To be ran on 8/20/14
All 5 Const. Promoters available.
1
2
3
4
5
6
7
8
9
10
11
12
A
1
1
1
1
1
1
6
6
6
6
6
6
B
1
1
1
1
1
1
7
7
7
7
7
7
C
1
1
1
1
1
1
7
7
7
7
7
7
D
3
3
3
3
3
3
7
7
7
7
7
7
E
3
3
3
3
3
3
10
10
10
10
10
10
F
3
3
3
3
3
3
10
10
10
10
10
10
G
6
6
6
6
6
6
10
10
10
10
10
10
H
6
6
6
6
6
6
1 = PTEF1 3 = M3 6 = M6 7 = M7 10 = M10
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6       [----------6 µg/ml----------] 7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12      [----------6 µg/ml----------]
08/20/14
CB008DB Const+rtTA+GFP+pAGA1+mCherry Run
Ran plate with 5 time points (0, 1.5, 3, 5, 8)
Ran first plate and noticed that specimen 5 only had 10ul loaded due to the addition of a new speciment (parameters were reset)
Readjusted the parameters and reran the well that only had 10ul loaded
Make sure to check parameters for all wells before running plate.
Stopped plate run at 3hr plate - D7
08/21/14
CB008DB Const+rtTA+GFP+pAGA1+mCherry Run (Cont.)
Continued running 3hr plate from 8/20/14
Finished 3 and 5 hr plates but could not do 8 hr plate because the plate was dropped.
Hyun said not to run DB plates
08/25/14
CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Preparation
Plate Map
1
2
3
4
5
6
7
8
9
10
11
12
A
1
1
1
1
1
1
7
7
7
7
7
7
B
1
1
1
1
1
1
10
10
10
10
10
10
C
1
1
1
1
1
1
10
10
10
10
10
10
D
6
6
6
6
6
6
10
10
10
10
10
10
E
6
6
6
6
6
6
F
6
6
6
6
6
6
G
7
7
7
7
7
7
H
7
7
7
7
7
7
1 = PTEF1 6 = M6 7 = M7 10 = M10
Grew cultures and put into 30°C incubator.
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6       [----------6 µg/ml----------] 7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12      [----------6 µg/ml----------]
08/26/14
CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Run
Was only able to finish running 0 hr plate and 5 hr plate up to C2.
Wells D4 and G3 on plates should not have sample due to error during dilutions.
All of M10 seems to not have grown compared to the other strains.
08/27/14
CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Run (Cont.)
Accidentally did not run the last specimen (M10) on the 5hr plate due to Eric's advice on not wasting time because M10 did not grow.
Re-streaking of old plates
Started restreaking old streak plates onto organized plates. (organized by pGEM #s)
08/28/14
Data Analysis of CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry Run
After using FlowJo and Matlab to analyze data, we have come to the conclusion that CB008 Const+rtTA+GFP+mfa+pPCL2+mCherry does not have RFP.
Compared pPCL2 RFP to pAGA1 RFP runs. pAGA1 has much RFP levels.
Prepared cultures of CB008DB Const+rtTA+GFP+mfa+pPCL2+mCherry M3, M6, M7, and M10 to be ran on the following day to confirm the presence or lack of presence of RFP.
pTEF1 missing, may need to grow a glycerol stock streak plate of it.
09/02/14
CB008DB Const+rtTA+GFP+mfa+pPCL2+mCherry Run
Shaker plate not completely clean, I attemped to clean it out with water. Saw chunks of dried cells on the bottom of some of the wells. MAKE SURE YOU PUT THE SHAKER PLATE IN THE BLEACH BIN AFTER USE!!!
M10 may be contaminated due to the shaker plate.
M6 culture was not labeled correctly in the incubator shaker, M6 may not actually be the right specimen.
Ran 2 time points: 0 hr and 5 hrs.
Data to be analyzed in the near future after asking Ianto how to run only two time points.
09/04/14
Co-culture of CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry (pTEF1, M7, M10)
Plate Map:
A1-A6 = pTEF1+M7
B1-B6 = pTEF1+M7
C1-C6 = pTEF1+M7
D1-D6 = M7+M10
E1-E6 = M7+M10
F1-F6 = M7+M10
G1-G6 = M10+pTEF1
H1-H6 = M10+pTEF1
A7-A12 = M10+pTEF1
Dox Map:
Lane 1 = 0
Lane 2 = 0.03
Lane 3 = 0.06
Lane 4 = 0.09
Lane 5 = 0.6
Lane 6 = 6
Lane 7 = 0
Lane 8 = 0.03
Lane 9 = 0.06
Lane 10 = 0.09
Lane 11 = 0.6
Lane 12 = 6
Grew cultures of CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry
pTEF1
M7
M10
stored in 30°C incubator shaker.
09/05/14
Co-culture of CB008 Const+rtTA+GFP+mfa+pAGA+mCherry Run
5 Timepoints: 0, 1.5, 3, 5, 8.
Successfully ran all the plates.
09/10/14
Data Analysis of Co-culture of CB008 Const+rtTA+GFP+mfa+pAGA+mCherry Run
After running the data on FlowJo and Matlab, we compared the data to the original pAga1+mCherry pTEF1, M7, and M10 data.
The comparison showed slight variation and unexpected results.
Maybe due to the larger cells being too stressed from producing so many proteins, and the overall population producing less as a result of this.
After comparing the data, it was decided that the best option would be to rerun this experiment but with pTEF1, M7, and M10 separately put on the plate as a comparison.
Started growing cultures of pTEF1, M7, and M10 in the incubator.
09/11/14
Co-Culture of CB009 Const+rtTA+GFP+mfa+pAGA+mCherry Round 2
Plate Map:
A1-A6 = pTEF1
B1-B6 = pTEF1
C1-C6 = pTEF1
D1-D6 = M7
E1-E6 = M7
F1-F6 = M7
G1-G6 = M10
H1-H6 = M10
A7-A12 = M10
B7-B12 = M7+pTEF1
C7-C12 = M7+pTEF1
D7-D12 = M7+pTEF1
E7-E12 = M7+M10
F7-F12 = M7+M10
G7-G12 = M7+M10
Dox Map:
Lane 1 = 0
Lane 2 = 0.03
Lane 3 = 0.06
Lane 4 = 0.09
Lane 5 = 0.6
Lane 6 = 6
Lane 7 = 0
Lane 8 = 0.03
Lane 9 = 0.06
Lane 10 = 0.09
Lane 11 = 0.6
Lane 12 = 6
Did not make a co-culture of pTEF1+M10 because the two promoters are too close in activity to make much of a difference.
09/12/14
Attempt of Co-Culture CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2 Run
Cancelled due to wrong culture being grown and due to the absence of induction.
Make sure that Eleanor and George pay attention when growing cultures and when inducing with doxycycline.
pPCL2 was grown instead of pAGA1. pPCL2 cannot be used in a co-culture due to its lack of RFP.
Experiment to be continued at another time.
09/14/14
Co-Culture CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2 Preparation
Plate Map:
A1-A6 = pTEF1
B1-B6 = pTEF1
C1-C6 = pTEF1
D1-D6 = M7
E1-E6 = M7
F1-F6 = M7
G1-G6 = M10
H1-H6 = M10
A7-A12 = M10
B7-B12 = M7+pTEF1
C7-C12 = M7+pTEF1
D7-D12 = M7+pTEF1
E7-E12 = M7+M10
F7-F12 = M7+M10
G7-G12 = M7+M10
Dox Map:
Lane 1 = 0
Lane 2 = 0.03
Lane 3 = 0.06
Lane 4 = 0.09
Lane 5 = 0.6
Lane 6 = 6
Lane 7 = 0
Lane 8 = 0.03
Lane 9 = 0.06
Lane 10 = 0.09
Lane 11 = 0.6
Lane 12 = 6
Did not make a co-culture of pTEF1+M10 because the two promoters are too close in activity to make much of a difference.
Made sure to use pAGA1 this time.
Cultures growing in yeast incubator.
09/15/14
Co-Culture CB008 Const+rtTA+GFP+mfa+pAGA1+mCherry Round 2 Run
Time Points: 0, 1.5, 3, 5, 8.
Made sure to induce with doxycycline this time.
Also made sure to create the right file name this time.
Successfully finished running the plate
Analyzed the data and made scatter plots
Made cultures of AFRP+rtTA+GFP+mfa+pAGA+mCherry
PRM1
PRM2
PRM3
YDR124W
CLG1
AFRP+rtTA+GFP+mfa+pAGA+mCherry Preparation
Plate Map:
A1-A6 = PRM1
B1-B6 = PRM1
C1-C6 = PRM1
D1-D6 = PRM2
E1-E6 = PRM2
F1-F6 = PRM2
G1-G6 = PRM3
H1-H6 = PRM3
A7-A12 = PRM3
B7-B12 = YDR124W
C7-C12 = YDR124W
D7-D12 = YDR124W
E7-E12 = CLG1
F7-F12 = CLG1
G7-G12 = CLG1
Dox Map:
Lane 1 = 0
Lane 2 = 0.03
Lane 3 = 0.06
Lane 4 = 0.09
Lane 5 = 0.6
Lane 6 = 6
Lane 7 = 0
Lane 8 = 0.03
Lane 9 = 0.06
Lane 10 = 0.09
Lane 11 = 0.6
Lane 12 = 6
09/16/14
AFRP+rtTA+GFP+mfa+pAGA+mCherry Run
Time Points = 0, 1.5, 3, 5, 8.
Successfully ran all plates.
09/17/14
Absent due to doctor's appointment.
09/18/14
Data Analysis of AFRP+rtTA+GFP+mfa+pAGA+mCherry Run
Ran the data on FlowJo and Matlab.
Got interesting results like bimodal populations and large amounts of noise.
Eleanor's Transformations
Assisted Eleanor in her transformations of CB008 M3 and M6 with pTET+mfa, pAGA1+mCherry, and pPCL2+mCherry.
None of her transformations worked other than M3+mfa which had 2 colonies. I put the plates back into the incubator in case they worked.
Made cultures of M3+rtTA and M6+mfa to re-transform the following day.
09/19/14
Results of Eleanor's Transformations
Plates that grew colonies:
CB008 M3 pAGA1+mCherry = 1 colony
CB008 M3 mfa = 2 colonies
CB008 M6 mfa+pPCL2+mCherry = 4 colonies
Will do a Colony PCR to check plates - unlikely that plates worked
Linearized plasmids: mfa and pAGA1+mCherry
Kara advises us to not used previously linearized plasmid.
will attempt to use Jeffret's digested pPCL2+mCherry for M6.
Re-transformed and put into incubator:
CB008 M3+mfa
CB008 M3+pAGA1+mCherry
CB008 M3+mfa+pAGA1+mCherry
CB008 M6+mfa+pAGA1+mCherry
CB008 M6+mfa+pPCL2+mCherry
9/22/14
Colony PCR of Transformations
Ran colony PCR's of the Transformations from 9/19 and 9/15 of the CB008 Constitutive Promoters.
Checked for mfa and pPCL2/pAGA1+mCherry separately.
Colony PCR Gel: (Eleanor said that M6 already has mfa even if there is no band)
09/25/14
PCR Amplification of Bar1, mfa, and pGEM1-16 (except 7 and 10)
Due to the primers not covering all of the needed insert parts, new primers with new overhangs were ordered. I was put in charge of PCR amplifying the plasmids with these new primers.
Used primers #103-110.
mfa Template: Shuaixin's PRM2 + mfa (7.606) Plasmid
Bar1 Template: Shuaixin's PCL2 + Bar1 (6.62) Plasmid
George did the PCR for Ste2.
Used Seamless to transform all the plasmids. George made a pGEM10 PCR insert, which I used for transformation.
Used pSB1C3 as backbone. Used cleaned up undigested one.
09/26/14
Results of Transformation
All the pGEM plates transformed very well.
Bar1, Ste2, and mfa did not transform very well.
Negative control grew for some reason also, hopefully because of a contamination of PCR product.
Colony PCR of Transformation
After a few more hours of growing in the incubator, Bar1, Ste2, and mfa grew more cells.
Did a colony PCR on all the plates (3 colonies from each plate) (a total of 62 Colony PCRs including 5 colonies of Eleanor's Bar1 Transformations)
Jeffrey ran the gel for my Colony PCR.
Colony PCR Gel:
09/28/14
I moved to UC Davis and will be taking a break from lab work to concentrate on classes.
Kara made cultures from the plates for the successful Colony PCR's to miniprep on Monday.
09/29/14
Miniprep of Cultures
Eleanor did the miniprep of the cultures from 9/28/14 and sent them into sequencing.
09/30/14
Results of Sequencing
Sequencing showed that the miniprep was wrong. May have to redo everything or pick new colonies off the plates? Need to discuss with Kara on this matter.
Jessica's Notes
Protocols and Procedures in Lab Notebook, Eric's Notebook
TO-DO:
[ ] Chronicle attempts at alpha-inducible promoters cloning, link related tasks
[ ] link/update pictures, graphs, data (if possible/desirable)
[ ] add gel photos
WEEK i
Tuesday May 27, 2014
Meet Kara, Derrick
Go around lab
Generally got set up for iGEM, lab work.
Wednesday May 28, 2014
Meet Eric
I. PCR 13 alpha-inducible yeast promoters
PRM2
ASG7
FUS2
PCL2
FUS3
CLG1
YDR124W
HYM1
PRM6
PRM1
ECM18
PRM3
SAG1
II. Transformation of E. coli with backbone plasmid
Mach 1 cells with pHY4 plasmid
Thursday May 29, 2014
Come in late due to dentist appointment
I. Gel Extraction of 5/28 PCR
Friday May 30, 2014
I. Mini-Prep E. coli cultures from 5/29 Transformation
       - Extract plasmid backbone
II. Backbone Digestion
WEEK ii
Monday June 2, 2014
Ianto returns
I. Ligation of pHY4 Backbone + promoters
II. PCR unsuccessful/low-yield promoters from 5/28
III. Gel Extract 6/02 Primers
Promoter
Concentration
FUS2
failed
PCL2
84.35
YDR124W
134.3
PRM6
84.14
ECM18
91.99
PRM1
64.64
Tuesday June 3, 2014
I. PCR Yeast Promoters
       - redo of 5/28 PCR due to low yield
II. Gel Extraction of PCR
Wednesday June 4, 2014
I. Mini-prep Backbone DNA + promoter
       - transformation done previously
II. Digestion of plasmids in relation to 6/03 PCR (?)
Thursday June 5, 2014
I. Mini-prep Promoters from 6/02 PCR
II. Ligation of 6/03 Promoters
III. Transformation of E. coli with 6/05 Ligation DNA
IV. Transformation of Yeast with linearized promoters
       - YTS18 strain
Friday June 6, 2014
I. Colony PCR of 6/05 E. coli transformation
WEEK 1
Monday June 9, 2014
First day of bootcamp
Meet rest of iGEM: Eleanor, Sabrina, Robert, George, Jeffrey
Sometime before here, decide to give up on FUS2 and FUS3
I. Mini-prep Colonies from 6/06 Colony PCR
II. Yeast Colony PCR of 6/05 Yeast Transformation
Tuesday June 10, 2014
I. Digestion of pHY4 + alpha promoter + GFP
Wednesday June 11, 2014
I. PCR Purification of 6/10 digestion
Promoter
Concentration
PRM2
59.01
ASG7
89.23
PCL2
101.3
CLG1
93.58
YDR124W
127.5
HYM1
16.55
PRM6
400.0
PRM1
19.4
ECM18
118.7
PRM3
87.11
SAG1
179.2
II. Gibson Assembly of pHY4 + alpha promoter (I.) and rtTA inserts (made by Derrick 6/10)
III. Transformation of E. coli with Gibson Assembly (II.) {Attempt 1}
       - DH5alpha E. coli strain
       - 6/12 Results: failed. Use higher efficiency cells?
Thursday June 12, 2014
I. Seamless Cloning Assembly reaction of alpha promoter + rtTA
II. Transformation of E. coli with Seamless Cloning (I.) {Attempt 1.2}
       - Top 10 E. coli strain
Friday June 13, 2014
did not come in - attending cousin's graduation
NASA field trip
WEEK 2
Monday June 16, 2014
Lab meeting
presentation on ongoing work
I. Colony PCR of 6/11 E. coli transformation
II. Yeast Colony PCR of 6/13 yeast transformations (done by ?)
Tuesday June 17, 2014
I. Colony PCR of failed yeast + alpha promoter + GFP from 6/16
II. Mini-prep of pHY4 + alpha promoter + rtTA E. coli successful colonies from 6/16 Colony PCR/ 6/11 Transformation
Promoter+rtTA
Concentration
ECM18 1
308.3
ECM18 2
231.6
PCL2 1
162.7
PCL2 2
247.3
PRM6 1
248.1
PRM6 2
327.9
ASG7 1
351.7
ASG7 2
593.5
HYM1 1
221.7
HYM1 2
413.0
PRM2 1
324.3
PRM2 2
339.6
PRM3 1
351.5
PRM3 2
316.7
CLG1 1
289.0
CLG1 2
276.8
YDR124W 1
366.0
YDR124W 2
239.3
       - sent to sequencing {Attempt 1}
       - 6/18 Results: failed
Wednesday June 18, 2014
I. Transformation of yeast with alpha promoter + GFP (from 5/30 I.)
       - yeast strains CB008 and CB008DB
II. Overnight cultures of yeast from (I.)
Thursday June 19, 2014
I. Culture colonies from 6/12 transformation (6/12 II.)
II. Glycerol stocks of successful yeast strains from 6/18 transformation
Friday June 20, 2014
I. Learned to use flow cytometer
       - Characterization of alpha-inducible promoter + GFP in CB008 yeast
II. Mini-prep successful colonies from 6/19 cultures (6/19 I.)
       - sent to sequencing {Attempt 2}
       - 6/23 Results: failed, determined rtTA gene was incomplete
WEEK 3
Monday June 23, 2014
Lab meeting
presentation on iGEM Parts Registry and Parts Submission
I. Digestion of some pHY4 + alpha promoters - GFP (redo of 6/10 Digestion)
II. Culture DH5alpha + pHY4 + HYM1 cells for mini-prep (runnning low)
III. Gel Extraction of Digestion (I.)
Promoter
Concentration
ASG7
23.77
CLG1
16.59
HYM1
12.96
ECM18
21.39
PRM3
15.20
Tuesday June 24, 2014
I. PCR Purification of remaining pHY4 + alpha promoters - GFP (Digestion done by Derrick)
Promoter
Concentration
PRM2
0.4510
YDR124W
15.46
SAG1
12.10
PRM6
21.74
PRM1
4.186
PCL2
6.348
II. PCR of alpha promoter primer + rtTA with new rtTA RV primer
Wednesday June 25, 2014
I. Gel Check of 6/24 PCR - failed
II. Mini-prep of rtTA, other plasmids (transformed by Eric and Derrick 6/24)
Thursday June 26, 2014
I. PCR of alpha promoter primer + rtTA (Redo of 6/24 II.)
Friday June 27, 2014
I. Gel Check and PCR Purification of 6/26 PCR (6/26 I.)
Promoter
Concentration
PRM2
107.0
ASG7
111.2
PCL2
96.49
CLG1
114.8
YDR124W
90.53
HYM1
72.29
PRM6
99.83
PRM1
75.35
ECM18
18.81
SAG1
104.1
pTEF
182.3
II. Digest Excess plasmid from (I.)
III. Gibson Assembly of primer + rtTA (I.) and pHY4 + promoter (6/23, 6/24) {Attempt 3}
IV. Transformation of E. coli with pHY4 + alpha promoter + rtTA {Attempt 3}
       - use Mach 1 E. coli strain and Gibson Assembly (III.)
WEEK 4
Monday June 30, 2014
Lab meeting
update on alpha promoters cloning
continue to clone alpha-inducible promoters + rtTA with Sabrina
I. Gibson Assembly of rtTA + PRM2 (related to 6/27 III.)
II. Colony PCR of 6/27 E. coli transformation (6/27 IV.)
III. PCR of rtTA + ECM 18 primer (related to 6/26 I.)
Tuesday July 1, 2014
I. Gel Check and PCR Purification of 6/30 PCR of rtTA + ECM18
       - rtTA+ECM18 Concentration: 173.9
II. Mini-prep of alpha promoter + rtTA from 6/30 Colony PCR (6/30 II.)
       - Sent to sequencing {Attempt 3}
       - 7/02 Results: all had CTATTCTCACTCTTTGGACCT interrupting stop codon or failed
Promoter
Colony
Concentration
ASG7
1
203.0
2
282.9
3
244.9
PCL2
1
205.5
2
205.3
3
305.3
CLG1
1
155.5
2
187.0
3
230.0
YDR124W
1
89.44
PRM6
1
246.4
2
17.32
3
154.0
PRM3
1
244.9
3
180.1
SAG1
1
146.5
2
243.8
3
137.8
III. Culture PRM1 + rtTA due to low cell count
IV. Yeast Colony PCR of CB008 + constitutive promoters + GFP (made previously by Jeffrey, Sabrina, George, Robert, Eleanor)
V. Gel Check and PCR Purification of (IV.)
Wednesday July 2, 2014
I. Yeast Colony PCR and Gel Check of CB008 + constitutive promoters + GFP (Attempt 2, redo of 7/01 IV.)
II. Colony PCR and Gel Check of 7/01 Mini-preps (7/01 II.)
III. Cultures of remaining pHY4 + alpha promoters + rtTA E. coli colonies from 6/27 transformation (6/27 IV.)
IV. Gibson Assembly of failed alpha promoters + rtTA {Attempt 4}
V. Transformation of E. coli with (IV.) {Attempt 4}
       - Result: low or no yield on plates, possibly due to low backbone concentration
       - PRM2 overgrown, sequencing contained GFP
Thursday July 3, 2014
I. Mini-prep of (7/02 III.) cultures
       - Sent to sequencing {Attempt 4}
       - 7/07 Results: all had CTATTCTCACTCTTTGGACCT interrupting stop codon or failed
II. Digestion and Gel Extraction of pHY4 + alpha promoter + GFP (Redo of 6/23 I. and 6/24 I.)
Monday July 7, 2014
No lab meeting
I. Gel Check of alpha promoter + rtTA Colony PCR (done by Sabrina)
II. Linearization of HY86E3
       - contains pTET + GFP
III. PCR Purification of (II.)
WEEK 5
Tuesday July 8, 2014
I. PCR & PCR Purification of alpha promoter primer + rtTA (redo of 6/26)
Wednesday July 9, 2014
I. Gibson Assembly of pHY4 + alpha promoter + rtTA {Attempt 5}
       - use backbone from 7/03 II.
       - use rtTA insert from 6/08 I.
II. Transformation of E. coli with pHY4 + alpha promoter + rtTA {Attempt 5}
       - 7/10 Results: failed negative control, GFP may still be present
Thursday July 10, 2014
Attended Lim Lab meeting
Work on lab meeting presentation - Parts Registry and Human Practices
I. Digestion of pHY4 + PRM1 + GFP and pHY4 + PRM2 + GFP (redo of 7/03 II.)
II. Colony PCR of 6/09 E. coli transformation (6/09 II.)
Friday Jul 11, 2014
Work on presentation
Continual edits to HP letters
I. Gel Check and mini-prep pHY4 + alpha promoter + rtTA plasmids from 6/10 II.
       - Sent to sequencing {Attempt 5}
Promoter
Concentration
7/14 Sequencing Results
PRM1
73.71
contains GFP
PRM2
143.0
contains GFP
PRM3
65.07
contains GFP
PRM6
80.45
contains GFP
ECM18
143.0
contains GFP
SAG1
335.9
contains GFP
YDR124W
50.44
contains GFP
CLG1
79.84
contains GFP
ASG7
36.22
contains GFP
HYM1
41.63
contains GFP
PCL2
162.3
contains GFP
       - 7/14 Results: all contained GFP, backbone not properly digested
WEEK 6
Monday July 14, 2014
Lab meeting - meet Wendell officially for first time
present on Parts Registry and Human Practices
PTS47 mini-prep sequencing returned - contains CTATTCTCACTCTTTGGACCT
I. Gibson Assembly using previous backbone - ABANDONED
Tuesday July 15, 2014
Meet Shuaixin
I. PCR of rtTA + alpha promoter primer homology
       - use rtTA template from Jeffrey's successful transformation (7/14)
II. Set up cultures for Glycerol stocks of new yeast strains
yGEM37 CB008   pTEF1
yGEM39 CB008   pTEFm6
yGEM40 CB008   pTEFm7
yGEM41 CB008   pTEFm10
yGEM44 CB008DB pTEFm6
yGEM45 CB008DB pTEFm7
Wednesday July 16, 2014
Kara gone
PTS47 stock sequencing returned - contains CTATTCTCACTCTTTGGACCT sequence ):
I. Gel Extract of 7/15 PCR        - done by Jeffrey+Sabrina        - low concentrations, decide to rePCR
Promoter
Concentration
PRM1
TBA
PRM2
PRM3
PRM6
ECM18
SAG1
YDR124W
CLG1
ASG7
HYM1
PCL2
pTEFm3
II. Glycerol Stocks of yeast strains: pTET+GFP pTEF#+rtTA
       - use cultures from 7/15 II.
yGEM37    CB008   pTEF1
yGEM3 9CB008  pTEFm6
yGEM40    CB008   pTEFm7
yGEM41    CB008   pTEFm10
yGEM44    CB008DB pTEFm6
yGEM45    CB008DB pTEFm7
Thursday July 17, 2014
Kara, Sabrina gone
Email Kara about progress
Talk with Anasuya
start this online lab journal
Wiki pages for website - start updating/saving now, update along the way (papers, figures, schematics)
understand biological basis of modeling goal - specific examples (specific disease - neuroinflammation)
4 slides with 4 potential examples
analogue to alpha-inducible promoter + rtTA?
Check if we're on track for medal requirements
Quick-change? (methylated digest?multiple Quick-change)
PTS47 stock sequencing rerun and returned - still contains CTATTCTCACTCTTTGGACCT sequence
I. PCR Purification of rtTA PCR     Results:
Promoter
Concentration
PRM1
102.2
PRM2
110.6
PRM3
135.5
PRM6
130.7
ECM18
136.8
SAG1
152.4
YDR124W
78.89
CLG1
133.7
ASG7
68.15
HYM1
91.08
PCL2
86.63
pTEFm3
47.19
II. Gibson Assembly using inserts from 1. {Attempt 6}
III. Transformation: E. coli {Attempt 6}
       - NEB5alpha E. coli
       - 7/18 Results:
             Plates with no colonies: PRM1, PRM3, ECM18, YDR124W, CLG1, ASG7, HYM1, PCL2, positive control, negative control
            Plates with colonies: PRM2 (4), PRM6 (countless), SAG1 (25)
TO-DO:
Research examples of cell-cell community communication -> 4 slides with 4 examples
Friday Jul 18, 2014
Kara & Sabrina gone
Prepare for Monday's lab meeting
I. Colony PCR and culture successful colonies from 6/17 transformation (6/17 III.)
       - cultured by Eleanor Sunday 7/20
       - 3 cultures from PRM2, PRM6, and SAG1
II. Replate failed transformations {Attempt 6.2}
       - set in drawer over weekend
       - 7/21 Results: almost all did not grow any colonies
             - PRM6 overgrown (similar to 6/17 transformation)
WEEK 7
Monday July 21, 2014
Lab meeting -  RM S436a
I. Mini-prep 6/18 I. cultures
Promoter+rtTA
Concentration
7/22 Results
PRM2
225.0
failed sequencing
PRM6
491.5
has 21 bp mutation?!?!
SAG1
439.0
has GFP (backbone not properly digested)
II. Digestion and Gel Extraction of low-concentration backbones
pGEM
pHY4+Promoter
Concentration
3
PRM3
6
SAG1
7
YDR124W
10
HYM1
11
PCL2
III. Seamless Cloning of alpha-inducible promoters+rtTA {Attempt 7}
Promoter
Insert
Backbone
PRM1
7/17 I.
6/10 I.
PRM2
7/17 I.
6/10 I.
PRM3
7/17 I.
7/21 II.
PRM6
7/17 I.
6/24  I.
ECM18
7/17 I.
6/23  I.
SAG1
7/17 I.
7/21 II.
YDR124W
7/17 I.
7/21 II.
CLG1
7/17 I.
6/23  I.
ASG7
7/17 I.
6/23  I.
HYM1
7/17 I.
7/21 II.
PCL2
7/17 I.
7/21 II.
IV. Transformation of E. coli with III. {Attempt 7}
       - NEB5alpha strain
TO-DO:
Determine Exploratorium HP topic/activity plan
Research examples of cell-cell community communication -> 4 slides with 4 examples
Tuesday July 22, 2014
finalize Exploratorium activity
explain University of Virginia collaboration
I. Colony PCR of 6/18 Transformation {Attempt 6.2}
       - Abandoned due to poor Gel Photo, unlikely to contain desired gene
II. Culture pGEM colonies
       - restock pGEM for future use
III. Transformation of E. coli with pGEM
       - use pGEM4 (PRM6) and pGEM6 (SAG1)
Wednesday July 23, 2014
I. Mini-prep of cultures from 6/22 II.
       - stored in Parts and Plasmids Box
pGEM#
Promoter
Concentration
1
PRM1
269.5
2
PRM2
294.9
3
PRM3
170.1
5
ECM18
285.4
8
CLG1
297.4
9
ASG7
259.7
10
HYM1
240.2
11
PCL2
261.0
II. Digestion of pGEM plasmids
       - remove GFP
       - overnight digestion at 37 C
TO-DO:
Digestion of pGEMs
use Xho1 and Not1 to remove GFP
PCR rtTA with Xho1 and BamH1
PCR Msn2 activating domain with BamH1 and Not1
Thursday July 24, 2014
I. Mini-prep of cultures from 6/22 III. Transformation
pGEM#
Promoter
Concentration 1
Concentration 2
4
PRM6
398.3
527.8
6
SAG1
375.2
399.1
II. Gel Extraction of 7/23 II. Digestion
       - done by Sabrina
pGEM#
Promoter
Concentration
1
PRM1
2
PRM2
3
PRM3
4
PRM6
5
ECM18
6
SAG1
7
YDR124W
8
CLG1
9
ASG7
10
HYM1
11
PCL2
III. PCR and PCR Purification of rtTA and Msn2 inserts
       - obtain rtTA with XhoI and BamHI cut sites
       - obtain Msn2 with BamHI and NotI cut sites
Insert
Concentration
rtTA
153.4
Msn2
100.9
IV. Digestion of rtTA and Msn2
       - use inserts from III.
       - rtTA: XhoI and BamHI
       - Msn2: BamHI and NotI
       - overnight digestion at 37 C
Friday July 25, 2014
I. PCR Purification of 7/24 IV. Digestion
       - done by Sabrina
II. Ligation for pHY4+alpha promoter+rtTA
       - done by Sabrina
       - use inserts from 7/25 I.
       - use backbone from 7/24 II.
III. Transformation of E. coli with 7/25 II. plasmids
       - done by Sabrina
       - DH5alpha strain
       - culture on Sunday
WEEK 8
Monday July 28, 2014
I. Colony PCR of successful colonies from 7/25 III. Transformation
       - Gel inconclusive, but seems as if most failed        - Pick new colonies to redo
Tuesday July 29, 2014
I. Colony PCR of 7/25 III. Transformation (redo of 7/28 I.)
       - 1st gel ran off, inconclusive        - 2nd gel
Wednesday July 30, 2014
worked on Exploratorium Human Practices
I. Colony PCR Gel Check of 7/25 I. (redo 3 of 7/29 I.)
II. Miniprep successful colonies from I. Colony PCR
       - Sequencing Results 7/31
Promoter
Colony
Concentration
Sequencing
PRM1
6
376.7
Successful
PRM2
1
356.8
Successful
PRM3
2
364.5
Successful
PRM6
4
377.5
Failed
ECM18
1
371.7
Successful
YDR124W
5
328.4
Successful
CLG1
5
543.1
Successful
ASG7
4
348.8
Successful
HYM1
3
449.6
Successful
PCL2
6
444.4
Failed
Thursday July 31, 2014
worked on Exploratorium Human Practices
work on introduction for iGEM newsletter by Paris-Bethencourt
I. Colony PCR of unsuccessful AFRP+rtTA plates
       - PRM6, SAG1, and PCL2 all failed
II. Linearization of successful plasmids from 7/30 II.
       - Digested with PMEI for 2 hours at 37 degrees C
III. Ligation of pHY4+AFRP, rtTA, and Msn2 (redo of 7/25 II. for promoters from I.)
       - negative control used PRM6 backbone
       - ligated with T4 ligase overnight at 16 degrees C
Friday August 1, 2014
Anasuya: various examples of how system will respond (up, down, stays, converge, diverge) depending on stimulus
drop intermediate values when testing? Allow for more samples, less accuracy
I. Transformation of E. coli with pHY4+AFRP+rtTA plasmid
       - use 7/31 III. ligations
       - plated overnight at room temperature
II. Transformation of yeast with AFRP+rtTA
       - use 7/31 II. linearized plasmids
       - CB008 pTET+GFP and CB008DB pTET+GFP yeast strains
WEEK 9
Monday August 4, 2014
Lim Lab meeting
I. Colony PCR of E. coli Transformation (8/01 I.)
       - PRM6, SAG1, PCL2 all successful
II. Colony PCR of Yeast Transformation (8/01 II.)
       - CB008(DB) pTET+GFP AFRP+rtTA plates
       - CB008 all successful except ECM18
       - CB008DB all failed
III. Colony PCR of Yeast Transformation (redo of II.)
       - CB008DB AFRP+rtTA, CB008 ECM18
       - all failed
Tuesday August 5, 2014
Work on iGEM Collaborations with Paris-Bettencourt
I. Colony PCR of Yeast Transformation (redo of 8/04 III.)
       - CB008DB AFRP+rtTA, CB008 ECM18
       - on hold, need more primers
II. Streak successful colonies from 8/04 II.
Wednesday August 6, 2014
Exploratorium Presentation Practice
Work on iGEM Collaborations with Paris-Bettencourt
I. Transformation of Yeast with AFRP + rtTA
       - CB008 pTET+GFP, PRM6, ECM18, SAG1, PCL2 +rtTA (use Mini-prep of 8/01 I., [for ECM18, redo of 8/01 II.)
       - CB008DB pTET+GFP with all 11 AFRP+rtTA (redo of 8/01 II.)
       - 8/08 Results: all grew colonies except CB008 PRM6+rtTA
Thursday August 7, 2014
Human Practices: Exploratorium After Night Event
Super Science: the science behind superheroes
I. Culture 8/05 II. streaks for glycerol stocks
Friday August 8, 2014
send in iGEM newsletter information
I. Yeast Colony PCR of 8/06 I. Transformation
       - all failed
II. Glycerol stocks of 8/05 II. yeast strains
       - CB008 pTET+GFP PRM1, PRM2, PRM3, YDR124W, CLG1, ASG7, HYM1 + rtTA
III. Transformation of Yeast with pTET+mfalpha
       - use strains: CB008 pTET+GFP, PRM1, PRM2, PRM3, YDR124W, CLG1, ASG7, HYM1+ rtTA
       - 8/11 Results: all grew colonies
Week 10
Monday August 11, 2014
lab meeting
email Kara about upcoming class schedules, days you can't go in
re-organize box
brainstorm project titles
Parts Registry - help Eleanor
finish Santa Cruz poster by Wed. - help Ianto
I. Yeast Colony PCR of 8/06 I. Transformation (redo of 8/08 I.)
       - all failed again
II. Yeast Colony PCR of 8/08 III. Transformation
       - all successful except CLG1 and HYM1
Tuesday August 12, 2014
I. Yeast Colony PCR of 8/06 and 8/08 Transformation (redo of 8/11 I. and II.)
       - CLG1+rtTA pTET+mfalpha successful; all others failed
II. Linearization of plasmids
       - use PmeI
III. Start cultures for retransformation (8/13 II.)
Wednesday August 13, 2014
Poster arrived!
I. Yeast Colony PCR
       - check CB008 pTET+GFP HYM1+rtTA pTET+mfalpha
       - colony 10 worked; culture made for glycerol stocks and transformation
II. Yeast Tranformation
       - a. add pPCL2+RFP, pAGA+RFP, pPCL2+BFP to CB008 pTET+GFP AFRP+rtTA pTET+mfalpha
       - b. add AFRP+rtTA to CB008 pTET+GFP: PRM6, ECM18, SAG1, PCL2
       - suspend CB008DB pTET+GFP AFRP+rtTA transformations
       - Results (Checked 8/18): b. all failed. BFP suspended due to lack of supplies. a. first Colony PCR failed, HYM1 failed, rest successful
III. Glycerol Stocks of CB008 pTET+GFP AFRP+rtTA pTET+mfalpha        - PRM1, PRM2, PRM3, YDR124W, CLG1, ASG7
Thursday August 14, 2014
Practice presentations 2-4PM
Friday August 15, 2014
last official day of iGEM
visit Santa Cruz for iGEM meet-up
Week 11
Monday August 18, 2014
Kilobot workshop! no group meeting
Sunday August 17: Jeffrey did Colony PCR for 8/13 II. Yeast Transformations
Successful yeast strains (Colony PCR'ed by Jeffrey)
yGEM Number
Characteristics
105
CB008 pTET-GFP::LEU2 pPRM1-rtTA::URA3 pTET-mfalpha::HIS  pAGA1+mCherry::TRP
106
CB008 pTET-GFP::LEU2 pPRM2-rtTA::URA3 pTET-mfalpha::HIS  pAGA1+mCherry::TRP
107
CB008 pTET-GFP::LEU2 pPRM3-rtTA::URA3 pTET-mfalpha::HIS  pAGA1+mCherry::TRP
111
CB008 pTET-GFP::LEU2 pYDR124W-rtTA::URA3 pTET-mfalpha::HIS  pAGA1+mCherry::TRP
112
CB008 pTET-GFP::LEU2 pCLG1-rtTA::URA3 pTET-mfalpha::HIS  pAGA1+mCherry::TRP
113
CB008 pTET-GFP::LEU2 pASG7-rtTA::URA3 pTET-mfalpha::HIS  pAGA1+mCherry::TRP
116
CB008 pTET-GFP::LEU2 pPRM1-rtTA::URA3 pTET-mfalpha::HIS pPCL2+mCherry::TRP
117
CB008 pTET-GFP::LEU2 pPRM2-rtTA::URA3 pTET-mfalpha::HIS  pPCL2+mCherry::TRP
118
CB008 pTET-GFP::LEU2 pPRM3-rtTA::URA3 pTET-mfalpha::HIS  pPCL2+mCherry::TRP
122
CB008 pTET-GFP::LEU2 pYDR124W-rtTA::URA3 pTET-mfalpha::HIS  pPCL2+mCherry::TRP
123
CB008 pTET-GFP::LEU2 pCLG1-rtTA::URA3 pTET-mfalpha::HIS  pPCL2+mCherry::TRP
124
CB008 pTET-GFP::LEU2 pASG7-rtTA::URA3 pTET-mfalpha::HIS  pPCL2+mCherry::TRP
I. Yeast Transformation
       - redo of 8/13 II.
       - Results (8/20): all failed, redo?
Tuesday August 19, 2014
Kilobot workshop!
Wednesday August 20, 2014
Kilobot workshop!
I. Yeast Transformation
       - CB008 AFRP+rtTA pTET+GFP pTET+mfalpha add in pPCL2+BFP
       - PRM1, PRM2, PRM3, YDR124W, CLG1, ASG7, HYM1
Thursday August 21, 2014
Kilobot workshop!
I. Yeast Transformation
       - CB008 pTET+GFP, add in AFRP+rtTA: SAG1 nd PCL2
       - PRM6 and ECM18 suspended due to poor expression
Friday August 22, 2014
Kilobot workshop!
Week 12
Monday August 25, 2014
Results: Friday 8/22 Transformation failed
I. Make cultures for Yeast Transformation tomorrow
II. Linearize plasmids for Yeast Transformation
Culture
Added part
CB008 pTET+GFP m3+rtTA
pTET+mfalpha
CB008 pTET+GFP m6+rtTA pTET+mfalpha
pAGA1+mCherry
CB008 pTET+GFP
SAG1+rtTA
CB008 pTET+GFP
PCL2+rtTA
CB008 pTET+GFP PRM1+rtTA pTET+mfalpha
pPCL2+BFP
CB008 pTET+GFP PRM2+rtTA pTET+mfalpha
pPCL2+BFP
CB008 pTET+GFP PRM3+rtTA pTET+mfalpha
pPCL2+BFP
CB008 pTET+GFP YDR124W+rtTA pTET+mfalpha
pPCL2+BFP
CB008 pTET+GFP CLG1+rtTA pTET+mfalpha
pPCL2+BFP
CB008 pTET+GFP ASG7+rtTA pTET+mfalpha
pPCL2+BFP
CB008 pTET+GFP HYM1+rtTA pTET+mfalpha
pPCL2+BFP
CB008 pTET+GFP HYM1+rtTA pTET+mfalpha
pPCL2+mCherry
CB008 pTET+GFP HYM1+rtTA pTET+mfalpha
pAGA1+mCherry
Tuesday August 26, 2014
I. Yeast Transformation (redo of a lot of failed transformations II. E. coli Transformation (redo of 8/85, which failed due to plating on wrong antibiotic)
Thursday August 28, 2014
First day of class at UC Berkeley
Week 13+
9/08/2014 - 10/29/2014 Came in on Mondays and Wednesdays to work on Kilobots modeling, iGEM details, reduced wet lab work.
Derrick's Lab Notebook
5/28/14
PCR of Yeast Promoters
Materials:                          1X          4.5x               5x HF Phusion Buffer     10 µl       45 µl 10mM dNTPs               1 µl        4.5 µl 10mM FW Primer           2.5 µl      11.25 µl 10mM REV Primer          2.5 µl      11.25 µl Phusion Polymerase       0.5 µl      2.25 µl Template DNA             0.5 µl      2.25 µl ddH20                    33 µl       148.5 µl  PCR Cycle:  98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
Notes:
from yeast genomic DNA
13 different promoters:
1KB: YDR124W, SAG1, PCL2, HYM1, FUS3
700BP: CLG1, PRM2
500BP: PRM6, PRM3, PRM1, FUS2, ECM18, ASG7
Promoters
1) PRM2
2) ASG7
3) FUS2
4) PCL2
5) FUS3
6) CLG1
7) YDR124W
8) HYM1
9) PRM6
10) PRM1
11) ECM18
12) PRM3
13) SAG1
numbering system for promoters
5/29/14
Gel:
Transformation of pHY4:
used 0.2 µl of pHY4
sat on ice for 10 minutes before heat shock
incubate at 37°C for 30 minutes
Gel Extraction of Alpha Inducible Promoters:
Qiagen Gel Extraction Protocol used
Qiagen column was not used, but protocol was used. may have yielded less efficency
Promoters       ng/µl  1) PRM2         13.49 2) ASG7         12.50 3) FUS2         n/a 4) PCL2         9.489 5) FUS3         11.77 6) CLG1         16.30 7) YDR124W      35.75 8) HYM1         23.61 9) PRM6         79.03 10) PRM1        n/a 11) ECM18       14.17 12) PRM3        n/a 13) SAG1        26.07
5/30/14
Miniprep of pHY4
centrifuge at 4000 RPM
followed Qiagen protocol
Concentrations: ng/µl
51.15
65.40
52.89
31.65
Digestion of promoters
Promoters
29 µl of PCR product
3.3 µl of 10x Cutsmart Buffer
1) Add 0.5 µl of ApaI. Mix gently, spin down, incubate at room temp. for 2 hours
2) Add 0.5 µl of XhoI. Mix gently, spin down, incubate at 37ºC for 1 hour
3) PCR Purification
Digestion of pHY4
pHY4
43 µl of PCR product
5 µl of 10x Cutsmart Buffer
1) Add 1 µl of ApaI. Mix gently, spin down, incubate at room temp. for 2 hours
2) Add 1 µl of XhoI. Mix gently, spin down, incubate at 37ºC for 1 hour
3) Gel Extraction
Concentrations from PCR Purification: 1) 14.54 2) 4.326 redo, 5.4890 4) -0.05256 redo, 1.730 5) 1.886 redo, 2.642 6) 41.05 7) 3.187 8) 19.09 9) 9.677 11) 1.868 13) 23.78  Concentrations from Gel Extraction 1) 7.749 -pHY4 1 2) 14.87 -pHY4 2
Gel:
6/2/14
ligated #1,2,5,6,7,8,9,13 with the 6kb pHY4 backbone
only had enough backbone for 8 reactions so chose the 8 promoters with the highest concentration
used Gibson ligation calculator to calculate mLs of reagents of our reaction
Protocol: See ligation protocol
PCR of Yeast Alpha Inducible Promoters 3,4,7,10,11,12
Materials:                          1X          4.5x               5x HF Phusion Buffer     10 µl       45 µl 10mM dNTPs               1 µl        4.5 µl 10mM FW Primer           2.5 µl      11.25 µl 10mM REV Primer          2.5 µl      11.25 µl Phusion Polymerase       0.5 µl      2.25 µl Template DNA             0.5 µl      2.25 µl ddH20                    32.5 µl     146.25 µl DSMO                     0.5 µl      2.25 µl     PCR Cycle:  98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
Gel:
Gel Extraction:
follow Qiagen protocol
Concentrations (ng/µl):
4) 84.15
7) 134.3
10) 84.14
11) 91.99
12) 64.64
6/3/14
Miniprep of pHY4
follow Qiagen protocol
low yields of plasmid previously due to PE Buffer not containing ethanol
Concentrations (ng/µl):
360.2
398.8
322.2
240.2
pHY4 + Promoters:
did not grow very well after ligation
7,9,13 grew some colonies
1,2,5,6,8,9,13 will be re-PCRed
Digestion:
Digestion of promoters
Promoters
50 µl of PCR product
5 µl of 10x Cutsmart Buffer
1) Add 1 µl of ApaI. Mix gently, spin down, incubate at room temp. for 2 hours
2) Add 1 µl of XhoI. Mix gently, spin down, incubate at 37ºC for 1 hour
3) PCR Purification
Digestion of pHY4
pHY4
50 µl of PCR product
5 µl of 10x Cutsmart Buffer
1) Add 1 µl of ApaI. Mix gently, spin down, incubate at room temp. for 2 hours
2) Add 2 µl of XhoI. Mix gently, spin down, incubate at 37ºC for 1 hour
3) Gel Extraction
Gel:
gel extraction was not performed b/c DNA was chewed up, no clear cut
PCR Purification Concentrations (ng/µl):
follow Qiagen Protocol
4) 165.1
7) 146.8
11) 68.84
12) 76.15
6/4/13
Digestion of pHY4
pHY4 was redigested since the first two attempts were unsuccessful
Digestion of pHY4
pHY4
40 µl of PCR product
5 µl of 10x Cutsmart Buffer
1) Add 1 µl of ApaI. Mix gently, spin down, incubate at room temp. for 2 hours
2) Add 2 µl of XhoI. Mix gently, spin down, incubate at 37ºC for 1 hour
3) Gel Extraction
Miniprep of pHY4
pHY4 was cultured last night, more needed for ligation w/ promoters
follow Qiagen protocol
Concentrations (ng/ul):
pHY4-1: 178.8
pHY4-2: 155.6
pHY4-4: 231.7
Gel:
Digestion of pHY4 +positive control
positive control from Hyun's stock plasmid
Digestion of positive control
pHY4
5 µl of PCR product
2.5 µl of 10x Cutsmart Buffer
16.5 µl H20
1) Add 1 µl of ApaI. Mix gently, spin down, incubate at room temp. for 2 hours
2) Add 1 µl of XhoI. Mix gently, spin down, incubate at 37ºC for 1 hour
Digestion of pHY4
pHY4
40 µl of PCR product
5 µl of 10x Cutsmart Buffer
1) Add 1 µl of ApaI. Mix gently, spin down, incubate at room temp. for 2 hours
2) Add 2 µl of XhoI. Mix gently, spin down, incubate at 37ºC for 1 hour
3) Gel Extraction
6/5/14
Gel:
Gel Extraction
follow Qiagen gel extraction protocol
concentrations (ng/µl)
+ control: 33.23
pHY4-1: 50.33
pHY4-3: 45.38
pHY4-4: 56.06
Ligation of Promoters w/pHY4
0.2 µl of backbone _ µl of insert 1 µl of T4 ligase buffer 0.5 µl _ µl of H20 --------------- 10 µl total      - sit at room temp for 2 hours     - 1,2,4,8,10,11,12: 8.1 µl H20 + 0.2 µl insert     - 6: 8.0 µl H20 + 0.3 µl insert     - negative control: 8 µl H20 + no insert transform into 50 µl dH5
Yeast Transformation of 7,9,13
6/6/14
Digestion of pHY4 -negative control had around 29 colonies -redigesting plasmids of control, pHY4 1,3,4 -follow same digestion protocol     -note: 50µl of DNA
Colony PCR of promoters: 1,2,4,6,10,11,12
Materials:                              1X          8x
GoTAQ Mix                12.5 µl       10mM FW Primer           1.25 µl       10mM REV Primer          1.25 µl       Template DNA             5 µl       ddH20                    5 µl       PCR Cycle:  98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
Gel 1:
Gel 2:
Miniprep of Ligation cultures -miniprep protocol -used vacuum manifold -ethanol not completely eluted, may affect sequencing
Concentrations (ng/ul):
1-4: 142.4
1-5: 203.7
2-4: 317.3
2-5: 205.9
4-4: 321.5
4-5: 284.9
6-4: 192.9
6-5: 432.9
10-2: 103.8
10-3: 203.2
11-4: 144.7
11-5: 226.8
12-4: 267.7
12-5: 267.7
6/10/14
PCR of Gibson Homology of rtTA to promoters
Materials:                              1X
5x HF Phusion Buffer     10 µl       10mM dNTPs               1 µl         10mM FW Primer           2.5 µl      10mM REV Primer          2.5 µl       Phusion Polymerase       0.5 µl       Template DNA             0.5 µl       ddH20                    33 µl         PCR Cycle:  98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
PCR Purification -follow PCR purification protocol
Concentrations (ng/ul): 1.PRM2: 190.9 4.PCL2: 149.1 6.CLG1: 412.1 7.YDR124W: 90.37 8.HYM1: 248.9 9.PRM6: 145.7 10.PRM1: 71.88 11.ECM18: 199.7 13.SAG1: 65.55
Gel:
6/11/14
PCR of Gibson Homology of rtTA to promoters
Materials:                              1X
5x HF Phusion Buffer     10 µl       10mM dNTPs               1 µl         10mM FW Primer           2.5 µl      10mM REV Primer          2.5 µl       Phusion Polymerase       0.5 µl       Template DNA             0.5 µl       ddH20                    33 µl         PCR Cycle:  98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
PCR Purification -follow PCR purification protocol
Concentrations (ng/ul): 2.ASG7: 179.6 12.PRM3: 236.6
Gel
6/12/14
Seamless Cloning of rtTA + promoters & pHY4 -follow seamless cloning procedure Promoters
1) PRM2
2) ASG7
4) PCL2
6) CLG1
7) YDR124W
8) HYM1
9) PRM6
10) PRM1
11) ECM18
12) PRM3
13) SAG1
6/16/14
Colony PCR of rtTA -transformations of SAG1& PRM1 didn't grow -inoculate colony in 25µl H20 Materials:                              1X          4x
GoTAQ Mix                12.5 µl       10mM FW Primer           1.25 µl       10mM REV Primer          1.25 µl       Template DNA             5 µl       ddH20                    5 µl
PCR Cycle:
98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
Gel:
Colony PCR pf yeast transformations (promoter+GFP) Materials:                              1X          70x
GoTAQ Mix                12.5 µl       10mM FW Primer           1.25 µl       10mM REV Primer          1.25 µl       Template DNA             5 µl       ddH20                    5 µl
PCR Cycle:
98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
Gel 1:
Gel 2:
Gel 3:
6/17/14
Colony PCR of Yeast transformations (promoter +GFP) -previous colony PCR failed, need to redo Materials:                              1X          80x
GoTAQ Mix                12.5 µl       10mM FW Primer           1.25 µl       10mM REV Primer          1.25 µl       Template DNA             5 µl       ddH20                    5 µl
PCR Cycle:
98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
Miniprep of rtTA + promoter Concentrations (ng/ul):      ECM18+rtTA #1    308.3 ng/ul     ECM18+rtTA #2    231.6 ng/ul     PCL2+rtTA #1     162.7 ng/ul     PCL2+rtTA #2     247.3 ng/ul     PRM6+rtTA #1     248.1 ng/ul     PRM6+rtTA #2     327.9 ng/ul     ASG7+rtTA #1     351.7 ng/ul     ASG7+rtTA #2     593.5 ng/ul     HYM1+rtTA #1     221.7 ng/ul     HYM1+rtTA #2     413.0 ng/ul     PRM2+rtTA #1     324.3 ng/ul     PRM2+rtTA #2     339.6 ng/ul     PRM3+rtTA #1     351.5 ng/ul     PRM3+rtTA #2     316.7 ng/ul     CLG1+rtTA #1     289.0 ng/ul     CLG1+rtTA #2     276.8 ng/ul     YDR124W+rtTA #1  366.0 ng/ul     YDR124W+rtTA #2  239.3 ng/ul
6/18/14
Yeast Transformations  -two strains (CB008) & (CB008DB) 9 rtTA Promoters:     PRM2      ASG7      PCL2      CLG1      YDR124W      PRM6      PRM1      ECM18      PRM3      SAG1  GFP + Promoter:     HYM1     PRM3 (only DB)
6/19/14
Flow Cytometry of AFRP + GFP
Overnight cultures of CB008+AFRP+GFP diluted ~100x (to a final concentration of OD600 0.5-0.1, Saturated overnight cultures should be OD600 of ~7) in SD complete media, and grown for 3 hours, 1000rpm shaker, 30ºC. Growing in 2mL well plates
Induce with Alpha-factor. Stock is in 3mM. Final concentrations are 0, 1nM, 10nM, 100nM, 1000nM.  Alpha-factor cannot be refrozen, so throw leftover away.
Induce for 90mins, but no longer than 120mins
Transfer 250u of each well into a V-bottom 96-well plate containing 10ul of the fixing chemical Cyclohexamide. (thats 4 ul for every 100ul of culture)  Cyclohexamide stops protein production by inhibiting ribosomes.
Run on the flow cytometer.
Parameters of flow
FSC: 250 SSC: 280 FITC(GFP): 550 B(RFP): 650  Flow rate: 1µL/sec Sample Volume: 200µL Mixing Volume: 100µL Mixing speed: 180µL/sec
Plate map
1   2   3   4   5   6   7   8   9   10  11  12 A   [------NEG------]       [------NEG-------] B   [------PRM2-----]       [------PRM1------] C   [------ASG7-----]       [------EMC18-----] D   [------PCL2-----]       [------PRM3------] E   [------NEG------]       [------SAG1------] F   [------CLG1-----] G   [------YDR124W--] H   [------PRM6-----]
Notes: -CLG1 x2 Alpha Factor -10ul culture diluted in 1ml of SD Complete
6/20/14
Minipreps of Yeast Colonies (rtTA + promoter) -send for sequencing since previous sequencing failed. colony PCR yielded false positives -follow Miniprep protocol Concentrations (ng/ul):      YDR124W.3   241.7 ng/ul     YDR124W.4   386.5 ng/ul     YDR124W.5   439.6 ng/ul     ASG7.3      207.7 ng/ul     ASG7.4      319.8 ng/ul     ASG7.5      287.2 ng/ul     ASG7.6      300.2 ng/ul     ASG7.7      338.5 ng/ul     CLG1.3      408.0 ng/ul     CLG1.4      306.0 ng/ul     CLG1.5      277.6 ng/ul     CLG1.6      249.8 ng/ul     CLG1.7      277.1 ng/ul     HYM1.3      440.1 ng/ul     HYM1.4      342.4 ng/ul     HYM1.5      388.7 ng/ul     HYM1.6      226.7 ng/ul     HYM1.7      351.2 ng/ul     ECM18.3     377.0 ng/ul     ECM18.4     426.8 ng/ul     ECM18.5     368.8 ng/ul     ECM18.6     459.7 ng/ul     ECM18.7     372.9 ng/ul     PRM3.3      430.4 ng/ul     PRM3.4      441.8 ng/ul     PRM3.5      361.7 ng/ul     PRM3.6      375.6 ng/ul     PRM3.7      333.1 ng/ul
6/23/14
Digestion of GFP from pHY4 + promoters -sequencing failed fpr nearly all promoters -GFP possibly not cut all the way -sequencing shows that GFP not fully digested
Already cut plasmid   49ul plasmid               5ul Cutsmart              6ul Not1                  0.5ul Xho1                  1ul  incubate at 37ºC for 2 hours
Gel Extraction Concentrations (ng/ul):      ASG7    23.77 ng/ul     CLG1    16.59 ng/ul     HMY1    12.96 ng/ul     ECM18   21.39 ng/ul     PRM3    15.20 ng/ul     -done by Ianto & Jessica
Concentrations (ng/ul):  YDR124W   15.46 ng/ul SAG1      12.10 ng/ul PRM6      21.74 ng/ul PRM1      4.186 ng/ul PCL2      6.348 ng/ul PRM2      0.451 ng/ul
Gel:
6/24/14
PCR of rtTA & promoters -previously only been amplifying half of rtTA -reorderedplasmids to include activation domain of rtTA
Materials:                              1X
5x HF Phusion Buffer     10 µl       10mM dNTPs               1 µl         10mM FW Primer           2.5 µl      10mM REV Primer          2.5 µl       Phusion Polymerase       0.5 µl       Template DNA             0.5 µl       ddH20                    33 µl         PCR Cycle:  98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
6/26/14
Flow cytometry, testing (triplicate) PRM2, ASG7, PLC2, CLG1
Alpha-factor concentrations: (Induce for 90mins)
(0nm, 0.5nm, 1nm, 10nm, 100nm, 1000nm, 3000nm)
Starting concentration is 3mM or 3,000,000nM Make 100x stocks and add 10ul to the 1mL cultures: 0nM 50nM 100nM 1000nM 10,000nM 100,000nM 300,000nM
Plate 1 & 2 Alpha Factor Concentration Map:
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H     Plate 1:     H   G   F   E   D   C   B   A    1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------PRM2-1---------] 5       [--------PRM2-2---------] 6       [--------PRM2-3---------] 7       [--------ASG7-1---------] 8       [--------ASG7-2---------] 9       [--------ASG7-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]  Plate 2:     H    G   F   E   D   C   B   A    1       [--------CLG1-1---------] 2       [--------CLG1-2---------] 3       [--------CLG1-3---------] 4 5 6 7 8 9 10 11 12
6/27/14
Continuation of CB008+AFRP+GFP FACs.
PRM6+GFP did not grow up yesterday.
ROUND 3 PLATE 1 PROMOTER MAP
H   G   F   E   D   C   B   A    1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------YDR124W-1------] 5       [--------YDR124W-2------] 6       [--------YDR124W-3------] 7       [--------PRM3-1---------] 8       [--------PRM3-2---------] 9       [--------PRM3-3---------] 10      [--------PRM1-1---------] 11      [--------PRM1-2---------] 12      [--------PRM1-3---------]
ROUND 2 PLATE 2 PROMOTER MAP [DID NOT COMPLETE]
H    G   F   E   D   C   B   A    1       [--------ECM18-1--------] 2       [--------ECM18-2--------] 3       [--------ECM18-3--------] 4       [--------SAG1-1---------] 5       [--------SAG1-2---------] 6       [--------SAG1-3---------] 7 8 9 10 11 12
Instructions for starting flow:
Turn on and wait 30mins for the machine to warm up
Re-initialize the HTS, them prime it three times.  You'll get a bubble in the lines if you don't
Run CST!!! Follow the sheet in front of the monitor for more instructions for that part. (CST uses beads to calibrate the laser detection.) add one drop into 250ul of sheath fluid in A1.
Bead LOT ID: use the one that is most current
Load A1-A4 with bleach and B1-B4 with water [flip for the opposite corner] -Cytometer>CST.  Make sure cytometer performance results passed
Run Clean Plate: 1. Click on experiment. Experiment>Open experiment 2. Open clean plate "Daily Clean" - 96 well U-bottom" 3. Be here to see if the cleaning is going correctly, low events (less than 100events/sec for bleach and less than 10events/sec for water) 4. If over the events, run another clean plate 5. View events in aquisition dashboard in view>Acquisition dashboard 6. Can also make a clean plate using HTS>Clean
6/14/30
Continuation of CB008+AFRP+GFP FACS.  Also testing Constitutive pTEF1 promoters +GFP today
ROUND 4 MAP
H    G   F   E   D   C   B   A    1       [--------ECM18-1--------] 2       [--------ECM18-2--------] 3       [--------ECM18-3--------] 4       [--------SAG1-1---------] 5       [--------SAG1-2---------] 6       [--------SAG1-3---------] 7       [--------CB008-1--------] 8       [--------CB008-2--------] 9       [--------CB008-3--------] 10 11 12
Constitutive Promoter MAP
1   2   3   4   5   6   7   8   9   10  11  12 A   [-CB008-]        B   [-pTEF1-]        C   [-m3----]        D   [-m6----]        E   [-m7----]        F   [-m10---] G    H
7/2/14
Flow of AFRP +GFP CB008 -redoing since data from previous FACS run had some odd behaviors and high standard error
Plate 1:         H    G   F   E   D   C   B   A
1       [--------CB008-1--------]     2       [--------CB008-2--------]     3       [--------CB008-3--------]     4       [--------PRM2-1---------]     5       [--------PRM2-2---------]     6       [--------PRM2-3---------]     7       [--------ASG7-1---------]     8       [--------ASG7-2---------]     9       [--------ASG7-3---------]     10      [--------PCL2-1---------]     11      [--------PCL2-2---------]     12      [--------PCL2-3---------]
Plate 2:     H    G   F   E   D   C   B   A    1       [--------CLG1-1---------] 2       [--------CLG1-2---------] 3       [--------CLG1-3---------] 4       [--------YDR124W-1------] 5       [--------YDR124W-2------] 6       [--------YDR124W-3------] 7       [--------PRM6-1---------] 8       [--------PRM6-2---------] 9       [--------PRM6-3---------]   10 11 12
Alpha Factor Concentrations:
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{3000 nM}----------------] B   [-------------------{1000 nM}----------------] C   [-------------------{100 nM}-----------------] D   [-------------------{10 nM}------------------] E   [-------------------{1 nM}-------------------] F   [-------------------{0.5 nM}-----------------] G   [-------------------{0 nM}-------------------] H
-CB008-1, PRM2-1-3, CLG1-1-3 didn't grow well, cultures very dilute and clear. added 20µl of cells instead of 10 -plate 1&2 follow same concentrations
7/7/14
Flow of AFRP +GFP CB008 & pTEF1 + Mutants + GFP Plate 1:         H    G   F   E   D   C   B   A
1       [--------CB008-1--------]     2       [--------CB008-2--------]     3       [--------CB008-3--------]     4 [--------PRM1-1---------]     5       [--------PRM1-2---------]     6       [--------PRM1-3---------]     7       [--------ECM18-1--------]     8       [--------ECM18-2--------]     9       [--------ECM18-3--------]     10      [--------PRM3-1---------]     11      [--------PRM3-2---------]     12      [--------PRM3-3---------]
Plate 2:    H    G   F   E   D   C   B   A    1       [--------SAG1-1---------] 2       [--------SAG1-2---------] 3       [--------SAG1-3---------] 4       [--------AGA1-1---------] 5       [--------AGA1-2---------] 6       [--------AGA1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10      [--------CLG1-1---------] 11      [--------CLG1-2---------] 12      [--------CLG1-3---------]  Alpha Factor Concentrations:       1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{3000 nM}----------------] B   [-------------------{1000 nM}----------------] C   [-------------------{100 nM}-----------------] D   [-------------------{10 nM}------------------] E   [-------------------{1 nM}-------------------] F   [-------------------{0.5 nM}-----------------] G   [-------------------{0 nM}-------------------] H     Plate 3:    1   2   3   4   5   6   7   8   9   10  11  12 A [-CB008-]   B [-pTEF1-]    C [-m3----]    D [-m6----]  E [-m7----]    F [-m10---]    G              H
-PRM2 had an OD600 of 0.3, added 100µl to 1ml of media
7/8/14
Colony PCR of pTET + GFP in CB008 & CB008DB -leu2 integration site -primers RA151 FW & RA 145 REV -boiled colony in 5µl 20 mM NaOH for 20min @ 95°C
Materials:                              1X
GoTAQ Mix                10 µl       10mM FW Primer           1 µl       10mM REV Primer          1 µl       Template DNA             3 µl       ddH20                    5 µl
PCR Cycle:
98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
-no colonies worked for the PCR
7/9/14
-Jeffrey previously cloned all constiutive promoters & rtTA (except for m3) & digested w/ PME1 -Eric & I transformed this into yeast CB008 & CB008DB containg pTET + GFP in the leu integration site -consti. & rtTA went into the ura site -follow yeast transformation protocol
7/10/14
Flow of CB008DB+AFRP+GFP -testing in strain CB008DB -strain has no Bar1 which degrades alpha factor -CB008 has Bar1 -hope to reach plateau in flow analysis
Plate 1:
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4       [--------CLG1-1---------] 5       [--------CLG1-2---------] 6       [--------CLG1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]
Plate 2:
H   G   F   E   D   C   B   A    1       [--------SAG1-1---------] 2       [--------SAG1-2---------] 3       [--------SAG1-3---------] 4       [--------AGA1-1---------] 5       [--------AGA1-2---------] 6       [--------AGA1-3---------] 7        8        9        10       11       12
Alpha-Factor Concentrations
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H
Miniprep of PTS47 -PTS47 pNH406 + rtTA -11E3178 pTET-MFAlpha -follow Miniprep protocol
Concentrations (ng/ul): PTS47 pNH406 + rtTA: #1 637.7 #2 387.7 11E3178 pTET-MFAlpha: 423.0
7/11/14
Flow Cytometry with CB008DB+AFRP+GFP
H  G   F   E   D   C   B   A    1       [-------CB008DB-1-------] 2       [-------CB008DB-2-------] 3       [-------CB008DB-3-------] 4       [---------PRM1-1--------] 5       [---------PRM1-2--------]    6       [---------PRM1-3--------] 7       [--------ECM18-1--------] 8       [--------ECM18-2--------] 9       [--------ECM18-3--------]    10       11       12
Alpha-Factor Concentrations
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H
Glycerol stocks of YDR124W, PRM6, & ASG7 did not grow in SD complete media overnight
7/14/14
Flow Cytometry with CB008DB+AFRP+GFP
Plate 1:
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4       [--------ASG7-1---------] 5       [--------ASG7-2---------] 6       [--------ASG7-3---------] 7       [--------YDR124W-1------] 8       [--------YDR124W-2------] 9       [--------YDR124W-3------] 10      [--------PRM6-1---------] 11      [--------PRM6-2---------] 12      [--------PRM6-3---------]
Alpha-Factor Concentrations
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H
E3 & F3 had some weird clumping on the bottom of the well and can't be resuspended
A2 skipped, human error.
7/16/14
[Doxycycline]: 100 mg/ml stock or 100,000 ug/ml (in 100ul Aliquots)  Induction time: 6hrs Concentrations needed for 100x:
0 ug/ml 3 ug/ml 6 ug/ml 9 ug/ml 30 ug/ml 60 ug/ml 90 ug/ml 300 ug/ml 600 ug/ml 900 ug/ml 3000 ug/ml 6000 ug/ml
Dilutions by Ianto:
Stock: 100mg/mL  **1x**           **100x**          Prep 60µg/mL       A: 6mg/mL            (6:100) 60µL of Stock in 940µL of water 30µg/mL       B: 3mg/mL            30µL of Stock in 970µL of water 9µg/mL        C: 900µg/mL          150µL of A in 850µL of water 6µg/mL        D: 600µg/mL          100µL of A in 900µL of water 3µg/mL        E: 300µg/mL          50µL of A in 950µL of water      Can also do 100ul of B in 900ul of water 0.9µg/mL      F: 90µg/mL           100µL of C in 900µL of water 0.6µg/mL      G: 60µg/mL           100µL of D in 900µL of water 0.3µg/mL      H: 30µg/mL           100µL of E in 900µL of water 0.09µg/mL     I: 9µg/mL            100µL of F in 900µL of water 0.06µg/mL     J: 6µg/mL            100µL of G in 900µL of water 0.03µg/mL     K: 3µg/mL            100µL of H in 900µL of water
Doxycycline concentrations plate map:
H   G   F   E   D   C   B   A    1                   [----0 µg/ml----] 2                   [---0.03 µg/ml--] 3                   [---0.06 µg/ml--] 4                   [---0.09 µg/ml--] 5                   [---0.3 µg/ml---] 6                   [---0.6 µg/ml---] 7                   [---0.9 µg/ml---] 8                   [---3.0 µg/ml---] 9                   [---6.0 µg/ml---] 10                  [---9.0 µg/ml---] 11                  [---30 µg/ml----] 12                  [---60 µg/ml----]
pTEF1 promoters
Plate 1:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008 Control]--------------]   B [---------------[CB008 pTET_GFP Control]-----]   C [---------------[CB008 pTET_GFP pTEF1_rtTA]--]   D [---------------[CB008 pTET_GFP pTEF1 m6]----]      E [---------------[CB008 pTET_GFP pTEF1 m7]----]     F [---------------[CB008 pTET_GFP pTEF1 m10]---]   G H
Plate 2:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB Control]------------]   B [---------------[CB008DB pTET_GFP Control]---]   C [---------------[CB008DB pTET_GFP pTEF1 1of2 ]  D [---------------[CB008DB pTET_GFP pTEF1 2of2 ] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G  H
7/22/14
Plate map
Flow Cytometry with CB008DB+AFRP+GFP
Plate 1:
H      G   F   E   D   C   B   A    1  pTEF1-1 [--------CB008DB-1------] 2  pTEF1-2 [--------CB008DB-2------] 3  pTEF1-3 [--------CB008DB-3------] 4  M7-1    [--------ASG7-1---------] 5  M7-2    [--------ASG7-2---------] 6  M7-3    [--------ASG7-3---------] 7          [--------YDR124W-1------] 8          [--------YDR124W-2------] 9          [--------YDR124W-3------] 10         [--------PRM6-1---------] 11         [--------PRM6-2---------] 12         [--------PRM6-3---------]
Plate 2:
H   G   F   E   D   C   B   A    1       [--------AGA1-1---------] 2       [--------AGA1-2---------] 3       [--------AGA1-3---------] 4       [--------CLG1-1---------] did not grow as much as the others 5       [--------CLG1-2---------] 20ul instead of 10ul for CLG1 6       [--------CLG1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]
Plate 3:
H    G   F   E   D   C   B   A    1        [--------ECM18-1--------] 2        [--------ECM18-2--------] 3        [--------ECM18-3--------] 4        [--------SAG1-1---------] 5        [--------SAG1-2---------] 6        [--------SAG1-3---------] 7         8         there is no PRM3 made, in glycerol stock 9         10       [--------PRM1-1---------] 11       [--------PRM1-2---------] 12       [--------PRM1-3---------]
Alpha-Factor Concentrations
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H
7/29/14
Flow Cytometry of Consti+rtTA+pTET+GFP CB008DB Dox run
-All plates are identical
Plate 1:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------] C [---------------[CB008DB pTET_GFP pTEF1]-----] D [---------------[CB008DB pTET_GFP pTEF1 m3]--] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G [---------------[CB008DB pTET_GFP pTEF1 m10]-] H
Plate 2:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------]   C [---------------[CB008DB pTET_GFP pTEF1]-----]  D [---------------[CB008DB pTET_GFP pTEF1 m3]--] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G [---------------[CB008DB pTET_GFP pTEF1 m10]-] H
Plate 3:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------]   C [---------------[CB008DB pTET_GFP pTEF1]-----] entire lane 10 is induced D [---------------[CB008DB pTET_GFP pTEF1 m3]--] with wrong concentration. E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G [---------------[CB008DB pTET_GFP pTEF1 m10]-] H
Doxycycline concentrations plate map:
H   G   F   E   D   C   B   A    1                   [----0 µg/ml----] 2                   [---0.03 µg/ml--] 3                   [---0.06 µg/ml--] 4                   [---0.09 µg/ml--] 5                   [---0.3 µg/ml---] 6                   [---0.6 µg/ml---] 7                   [---0.9 µg/ml---] 8                   [---3.0 µg/ml---] 9                   [---6.0 µg/ml---] 10                  [---9.0 µg/ml---] 11                  [---30 µg/ml----] 12                  [---60 µg/ml----]
-plate 1 concentrations backwards -plate 3 had 2x exposure of diff concentrations, not run on flow
7/30/14
Flow Cytometry of PTEF1 + rtTA + pTET + GFP CB008 -could not run b/c cultures did not grow -original patch plates did not have much growth at all -restreaked plates
8/1/14
Cloning AFRP + Ste2 -doing this to increase receptor levels to alpha factor
Materials:                              1X
5x HF Phusion Buffer     10 µl       10mM dNTPs               1 µl         10mM FW Primer           2.5 µl      10mM REV Primer          2.5 µl       Phusion Polymerase       0.5 µl       Template DNA             0.5 µl       ddH20                    33 µl         PCR Cycle:  98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
Gel:
8/4/14
Flow Cytometry of Consti+rtTA+pTET+GFP CB008 Dox run
Plate 1:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008]--------------------]   B [---------------[CB008 pTET_GFP]-----------] C [---------------[CB008 pTET_GFP pTEF1]-----] D [---------------[CB008 pTET_GFP pTEF1 m6]--]  E [---------------[CB008 pTET_GFP pTEF1 m7]--]   F [---------------[CB008 pTET_GFP pTEF1 m10]-]           G  H
Plate 2:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008]--------------------]   B [---------------[CB008 pTET_GFP]-----------] C [---------------[CB008 pTET_GFP pTEF1]-----] D [---------------[CB008 pTET_GFP pTEF1 m6]--]  E [---------------[CB008 pTET_GFP pTEF1 m7]--]   F [---------------[CB008 pTET_GFP pTEF1 m10]-]           G  H
Plate 3:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008]--------------------]   B [---------------[CB008 pTET_GFP]-----------] C [---------------[CB008 pTET_GFP pTEF1]-----] D [---------------[CB008 pTET_GFP pTEF1 m6]--]  E [---------------[CB008 pTET_GFP pTEF1 m7]--]   F [---------------[CB008 pTET_GFP pTEF1 m10]-]           G  H
Doxycycline concentrations plate map:
H   G   F   E   D   C   B   A    1                   [----0 µg/ml----] 2                   [---0.03 µg/ml--] 3                   [---0.06 µg/ml--] 4                   [---0.09 µg/ml--] 5                   [---0.3 µg/ml---] 6                   [---0.6 µg/ml---] 7                   [---0.9 µg/ml---] 8                   [---3.0 µg/ml---] 9                   [---6.0 µg/ml---] 10                  [---9.0 µg/ml---] 11                  [---30 µg/ml----] 12                  [---60 µg/ml----]
8/6/14
Colony PCR of AFRP + Ste2 Materials:                              1X
GoTAQ Mix                10 µl       10mM FW Primer           1 µl       10mM REV Primer          1 µl       Template DNA             3 µl       ddH20                    5 µl
PCR Cycle:
98°C    30 sec 98°C    10 sec 55°C    20 sec 72°C    30 sec 72°C    5 min 4°C     ∞
Gel:
8/8/14
CB008 pTEF1+rtTA pTET+GFP pTET+MFalpha pAGA1+mCherry
Time points with [Dox] induction: 0, 1.5hr, 3hr, 5hr
Plate map for all time points (4 plates):
1   2   3   4   5   6   7   8   9 10  11  12 A [CB008 m6-1]----]       [CB008 m10-3]----] B [CB008 m6-2]----]       [CB008DB m7-1]---] C [CB008 m6-3]----]       [CB008DB m7-2]---] D [CB008 m7-1]----]       [CB008DB m7-3]---] E [CB008 m7-2]----]  F [CB008 m7-3]----]            G [CB008 m10-1]---]  H [CB008 m10-2]---]
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12
-diluted 1:200 to get OD of 0.1 -fix with cycloheximide at diff. time points
8/11/14
Flow Cytometry of AFRP + rtTA + pTET + GFP -flow of 4 AFRPs: pHYM1, pYDR124W, pCLG1, pASG7 -plates are the same for all 4 promoters -1:200 dilution
Plate Map:
H   G   F   E   D   C   B   A    1           [----0 µg/ml------------] 2           [----0.03 µg/ml---------] 3           [----0.06 µg/ml---------] 4           [----0.09 µg/ml---------] 5           [----0.3 µg/ml----------] 6           [----0.6 µg/ml----------] 7           [----0.9 µg/ml----------] 8           [----3.0 µg/ml----------] 9           [----6.0 µg/ml----------] 10          [----9.0 µg/ml----------] 11          [----30 µg/ml-----------] 12          [----60 µg/ml-----------]
Alpha Factor Concentration Map
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H
8/12/14
Flow Cytometry of AFRP + rtTA + pTET + GFP -flow of 3 AFRPs: PRM1, PRM2, PRM3 -plates are the same for all 3 promoters
Plate Map:
H   G   F   E   D   C   B   A    1           [----0 µg/ml------------] 2           [----0.03 µg/ml---------] 3           [----0.06 µg/ml---------] 4           [----0.09 µg/ml---------] 5           [----0.3 µg/ml----------] 6           [----0.6 µg/ml----------] 7           [----0.9 µg/ml----------] 8           [----3.0 µg/ml----------] 9           [----6.0 µg/ml----------] 10          [----9.0 µg/ml----------] 11          [----30 µg/ml-----------] 12          [----60 µg/ml-----------]
Alpha Factor Concentration Map
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H
8/13/14
Flow Cytometry of pTEF1 + rtTA pTET + GFP pTET + MFalpha pAGA1_mCherry CB008DB -1:200 dilution -Time points for Dox induction: 0, 1.5hr, 3hr, 5hr -plate map same for all 4 plates
Plate Map: 1   2   3   4   5   6   7   8   9   10  11  12     A [CB008 pTEF1-1]-]       [CB008DB m3-3]--]      B [CB008 pTEF1-2]-]       [CB008DB m6-1]---]     C [CB008 pTEF1-3]-]       [CB008DB m6-2]---]     D [CB008DB pTEF1-1]       [CB008DB m6-3]---]     E [CB008DB pTEF1-2]
F [CB008DB pTEF1-3]
G [CB008DB m3-1]--]      H [CB008DB m3-2]--]
Dox Concentrations:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12
-0 time point plate was not fully run. Due to mistakes, only 10µl run on flow. plates saved to run flow on 0 time point plate tomorrow
8/14/14
Flow Cytometry of pTEF1 + rtTA pTET + GFP pTET + MFalpha pAGA1_mCherry CB008 -1:200 dilution -Time points for Dox induction: 0, 1.5hr, 3hr, 5hr -plate map same for all 4 plates -in 2ml culture plate there was a lot of debris/contamination in rows A,B,C -tried to avoid it by pipeting one at a time
Plate Map  1   2   3   4   5   6   7   8   9   10  11  12     A [CB008 pTEF1-1]-]       [CB008 m7-3]-----]     B [CB008 pTEF1-2]-]       [CB008 m10-1]----]     C [CB008 pTEF1-3]-]       [CB008 m10-2]----]     D [CB008 m6-1]----]       [CB008 m10-3]----]     E [CB008 m6-2]----]      F [CB008 m6-3]----]
G [CB008 m7-1]----]      H [CB008 m7-2]----]
Dox Concentrations:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12
-flow experiment completely failed due to high bacterial contamination
8/15/14
Official Last Day of iGEM
Jeffrey's Lab Notebook
6/9/14
PCR for Constitutive Promoters (pTEF1)
Materials
1x reaction
4.5x Master Mix
x Phusion HF Buffer
10 µl
45 µl
dNTP's (10 mM)
1 µl
4.5 µl
Forward Primer (10µm)
2.5 µl
11.25 µl
Reverse Primer (10µm)
2.5 µl
11.25 µl
*Template DNA
0.3 µl
1.35 µl
Phusion Polymerase
0.5 µl
2.25 µl
Water
33.2 µl
149.4 µl
Total
50 µl
225 µl
1. Mix materials in a 4.5x Master Mix on ice. Mix well. 2. Pipetter 50 µl from the Master Mix into 4 labeled PCR tubes 3. Thermocycler for :       Initial Duration   | 98° C | 30s        35 Cycles of:            Denaturation | 98° C | 10s           Annealing    | 55° C | 20s           Extension    | 72° C | 30s     Final Extension    | 72° C | 5m     Hold               | 4°  C | Forever  POSSIBLE ERRORS: Incomplete thawing of dNTPs
Working Stock of Reverse and Forward Primers for pTEF1 kept in my freezer box. 6/9/14
6/10/14
Gel Extraction
Cut DNA band from PCR of constitutive promoter gel Weigh gel: .720 g QG Buffer: 2160 ul  1. Mix gel slice with QG Buffer in 50C heat bath for 10 min. 2. Add 1 ml isopropanol. Mix. 3. Add 750 ul of mixture into purple Qiagen spin column. Spin for 30 sec. Discard liquid.  Repeat. 4. Add 750 ul of Buffer PE and spin for 1 min. Discard waste. 5. Dry spin the column for 1 min. Replace the spin column in a new microcentrifuge tube 6. Elute in 40 ul of ddH2O. Wait 1 min and then spin for 1 min.
Restriction Digest with ApaI
40 ul of DNA (pTEF1 gel extraction) 5 ul of 10x CutSmart Buffer .5 ul of ApaI Enzyme  *digest overnight at room temperature*
6/11/14
Restriction Digest with XhoI
Add .5 ul of XhoI enzyme to ApaI digestion from 6/10/14 Incubate in the 37°C shaker.
PCR Purification
1. Add 250 ul of Buffer PB to digestion. 2. Place sample in a purple QIAquick spin column. Spin for 1 min. Discard waste. 3. Wash with 750 ul of Buffer PE. Spin for 1 min. Discard waste. 4. Dry spin for 1 min. Replace spin column in a new microcentrifuge tube. 5. 6. Elute in 50 ul of ddH2O. Wait 1 min and then spin for 1 min.  Total concentration: 141.4 ng/ul
DNA Ligation
Materials
Volume
10x Ligase Buffer
1 ul
DNA Backbone PSV606
.2 ul
DNA Insert (PCR Purication)
.2 ul
T4 DNA Ligase
.5 ul
H2O
8.1 ul
Total                       | 10 ul ---------------------------------------------------- Mix reagents and incubate at room temperature for 2 hrs
Ligation of pTEF1 into PSV606 kept in my freezer box. 6/11/14
Transformation
10 ul of ligation 50 ul of E. Coli competent cells  30 min    | ice 45 sec    | 42°C heat shock 2 min     | ice  Add 250 ul of SOC media. Incubate at 37°C for 1 hr. Plate on LB+Carb.   NO COLONIES
6/12/14
Redo Transformation
Follow procedure from 6/11/14 with minor alterations. 1. Use .4 ul of ligation instead of .2 ul. 2. Use more expensive competent cells.
6/13/14
Transforming α-inducible promoters
Transforming 11 different inducible promoters into CB008 and CB008DB strains of yeast.  1. Boil salmon sperm DNA (ssDNA) for 10 min.-->10 ul of 10 mg/ml stock per transformation 2. Cool on ice for 10 min.  3. Pellet yeast cultures in centrifuge. (3000 rpm for 2-5 min.) 4. Resuspend with with .1M LiOAc in TE. 5. Pellet cells (3000 rpm for 2-5 min.) 6. Resuspend in 100 ul .1M LiOAc in TE per 2.5 ml culture.  7. Aliquot 100 ul into each microcentrifuge per transformation. (22 tubes) Per Tube: 8. Add 100 ug ssDNA, 1 ug of target DNA (1~5 ul)  ADD IN ORDER 9. 480 ul 50% PEG 3350 10. 60 ul 10x TE 11. 60 ul 1 M LiOAc 12. 75 ul DMSO  13. Vortex 14. Incubate at 42°C for 30 min.  15. Pellet cells (6000 rpm for 2 min) 16. Resuspend with 500 ul YPD. 17. Pellet cells. 18. Resuspend with 50 ul  YPD 19. Plate on SDS-Ura.   Incubate 2 days at 30°C.
6/16/14
Colony PCR for Screening Yeast
1. Pick a single colony using a sterile wooden stick and patch on a dropout plate.     Take stick and rub into a dry PCR tube. 2. Add 10 ul of 20mM NaOH to each of the PCR tubes.    Boil in pCR machine at 95°C for 20 min.  3. Set up PCR reaction as below:                                         1x Reaction      7x Master Mix                                         ---------------------------------     2X Go Taq Green PCR Master Mix              10ul               70ul     10 uM FW primer                              1ul                7ul     10uM RV primer                               1ul                7ul     Water                                        5ul               35ul      ----------------------------------------------------------------------     Boiled Yeast cells (template)                3ul            4. Set up Thermocycler for:     95°C       | 5 min     30x |95°C   | 45 sec         |50°C   | 30 sec         |72°C   | 1 min per kb     72°C       | 10 min     4°C            | hold 5. Run on 1% Agarose Gel  We ran 66 lanes for colony PCR RESULTS: only 4 lanes were positive POSSIBLE ERRORS: not enough DNA in tubes
Colony PCR on E.Coli for Constitutive Promoters
1. Pick a single colony using a sterile wooden stick and mix in 25ul of water in a PCR tube. Do this for about 4-6 colonies. Use 5ul for the PCR reaction below, adn save the rest for later.  2. Set up PCR reaction as below:                                             1x Reaction      6x master mix                                         ---------------------------------- 2X Go Taq Green PCR Master Mix                  10ul               60ul 10 uM FW primer                                  1ul                6ul 10uM RV primer                                   1ul                6ul Water                                            3ul               18ul  Bacterial cells (template)                       5ul                --         Cycles:             95°C               | 5 min             30X:    95°C       | 45 s                     55°C       | 30 s                     72°C       | 1 min per kb              72°C | 10 min  3. Analyze products on a 1% agarose gel.   RESULTS: All lanes worked  4. Inoculate the colonies in 5 ml LB+Carb and incubate 37°C shaker overnight.
6/17/14
Re-inoculate the E.Coli with Constitutive Promoters
6/18/14
Dilution of Yeast Strains
We diluted CB008 and CB008DB strains 1:20 times in YPD media.
Miniprep Constitutive Promoters from E.Coli
1. Pellet bacterial cells 2. Resuspend in 250 ul in Buffer P1. Transfer to a microcentrifuge tube. 3. Add 250 ul Buffer P1. Invert to mix. 4. Add 350 ul Buffer P2 and invert immediately but gently. 5. Centrifuge for 10 min at 130000 rpm. Apply supernatants to QIAprep spin columns. 6. Centrifuge for 30-60 sec. Discard flow through. 7. Wash with 750 ul Buffer PE and centrifuge for 60 sec. 8. Discard flow through and dry spin for 1 min. 9. Place spin column in microcentrifuge tube and elute in 25 ul ddH20. Spin for 1 min.
6/20/14-6/21/14
Out of the lab. Other lab members ran the flow cytometer on AFRPs and transformed yeast with constitutive promoters.
6/23/14
Yeast Colony PCR on Transformed Yeast
used wrong primers so we need to redo on Tuesday
6/24/14
Re-do Yeast Colony PCR on Transformed Yeast
Innoculated Yeast colonies in 5ml of YPD
Flow Cytometer Notes
Overnight cultures of CB008+AFRP+GFP diluted ~100x (to a final concentration of OD600 0.5-0.1, Saturated overnight cultures should be OD600 of ~7) in SD complete media, and grown for 3 hours, 1000rpm shaker, 30ºC. Growing in 2mL well plates
Induce with Alpha-factor. Stock is in 3mM. Final concentrations are 0, 1nM, 10nM, 100nM, 1000nM.  Alpha-factor cannot be refrozen, so throw leftover away.
Induce for 90mins, but no longer than 120mins
Transfer 250u of each well into a V-bottom 96-well plate containing 10ul of the fixing chemical Cyclohexamide. (thats 4 ul for every 100ul of culture)  Cyclohexamide stops protein production by inhibiting ribosomes.
Run on the flow cytometer.
Flow Cytometer things to remember:
check the sheath fluid box to see if its empty
check the waster container.  If full, dispose and add new bleach
take care not to leave the machine on run. Save sheath fluid by keeping on standby when not in use
find parameters using negative controls. (FSC, SSC, Fluorescense marker)
DO NOT RUN PLATE. RUN WELLS instead. (Run the first well by itself first b/c it needs time to create file folders.) this causes problems.
Run plate only saves data of the first well.
Can rerun through the first well again which would suck up air.  Air no good for the machine.
highlight wells and change settings on the right in the aquisition dashboard screen
mixing speed 180ul/sec
mix 3 times rather than 2
run at 1ul/sec for samples and 0.5ul/sec for negative controls when finding parameters.
make sre uL doesn't exceed samle to prevent sucking up air. (Rule of thumb is to run 50ul less than allotted sample sizes for wells.)
Voltage settings were gotten by running negative controls and adjusting to readouts seen in histogram form of FSC, SSC, FITC
Parameters of This Flow Experiment
FSC: 250 SSC: 280 FITC(GFP): 550 B(RFP): 650  Flow rate: 1µL/sec Sample Volume: 200µL Mixing Volume: 100µL Mixing speed: 180µL/sec
Flowjo Notes
Export data from USB and transfer to iGEM2014 folder
Open FlowJo and drag data over to box
Set an appropriate gate
Go to data, select all data and press E. Select mean, geometric mean, and count.
Hit refresh. Addiction numerical data should be present.
Use mean of FITC(GFP).
MatLab Notes
Open up new script and comment the title of the experiment
Enter the alpha factor concentrations info. (X axis). Put in log form.
Enter the yGEM data. This will serve as y axis. (FITC means)
Define variables
Tyope figure info and run to show plot.
6/25/14
Transformations of DH5a
HY86E3
HY67E1
HYGE1
PTS98
PTS108
PTS133
PTS97
plasmids for backbones in different integration sites and fluorescent tags
Glycerol stocks of Yeast Transformations (yGEM23-32)
420 ul 50% glycerol 350 ul cells in YPD vortex, store in -80C freezer
PCR for Parts Registry Promoters
ASG7, CLG1, ECM18, HYM1, PCL2, PRM1, PRM2, PRM3, PRM6, SAG1, YDR124W
Materials
1x reaction
x Phusion HF Buffer
10 µl
dNTP's (10 mM)
1 µl
Forward Primer (10µm)
2.5 µl
Reverse Primer (10µm)
2.5 µl
*Template DNA
0.3 µl
Phusion Polymerase
0.5 µl
Water
33.2 µl
Total
50 µl
-----
------
3. Thermocycler for :       Initial Duration   | 98° C | 30s        35 Cycles of:            Denaturation | 98° C | 10s           Annealing    | 55° C | 20s           Extension    | 72° C | 30s     Final Extension    | 72° C | 5m     Hold               | 4°  C | Forever  Ran gel on PCR products. 1% Agarose, 125 volts for 15 min, 5 ul SybrSafe
6/26/14
Only pGEM2 and pGEM3 worked. RETRY with DMSO (1.5 ul per reaction).
PCR
Master Mix: 170 ul HF Buffer 17 ul dNTPs 8.5 ul Phusion Polymerase 25.5 DMSO 539 ul Water ------------- 44.7 per tube   Also add 2.5 ul FW primers          2.5 ul RV primers          .3 ul template DNA
Made liquid cultures of transformed DH5a E. Coli cells with HY+PTS plasmid
6/27/14
Double Digestion of Constitutive Promoters-GFP
6.5 ul water 15 ul DNA template (pTEF1 plasmids) .5 ul Xho1 enzyme .5 ul Not1 enzyme    2.5 ul CutSmart Buffer  Incubate at 37C for 2hrs Load into agarose gel. *pTEF1 appers to be smaller than the others.
Gel Extraction
Exise the DNA bands from the gel. Weigh using a scale. pGEM12- 120 mg      -360 ul         -120 ul pGEM13- 110 mg      -360 ul         -110 ul pGEM14- 110 mg      -330 ul         -110 ul pGEM15- 100 mg      -300 ul         -100 ul pGEM16- 170 mg      -510 ul         -170 ul         Weights     QG Buffer       Isopropanol 1. Melt gels in 50C for 10 min in QG Buffer 2. Add isopropanol. 3. Place in a spin column and spin for 1 min 13000rpm 4. Add 500 ul QG Buffer 5. Spin for 1 min. Dump liquid. 6. Add 750 ul PE Buffer 7. Spin for 1 min. Dump Liquid. 8. Dry spin for 1 min. 9. Elute in 25 ul ddH20.   Concentrations (ng/ul) through NanoDrop:  1. pGEM12-25.77  2. pGEM13-47.93  3. pGEM14-18.31 4. pGEM15-10.6 5. pGEM16-32.45
Gibson Assembly
50 ng of DNA Backbone 2 ul of rtTA insert 5 ul Gibson Assembly Master Mix  pGEM12-2 ul         1 ul pGEM13-1 ul         2 ul pGEM14-3 ul         - pGEM15-4 ul         - pGEM16-1.5 ul       1.5 ul       DNA backbone    water All have 2 ul rtTA insert and 5 ul GAMM Incubate in thermocycler 50C for 15 min.
Transforming E. Coli with Gibson Assembly
10 ul Gibson mix 25 ul Mach1 cells ----------- 250 ul SOC media  30 min ice 45 sec heatshock at 42C 2 min ice Incubate at 37C for 1 hr. Plated 250 ul on LB-Cam (WRONG SELECTION MARKER- LB-Carb) Incubate in a drawer overnight
6/29/14
pTEF1+rtTA transformations failed!  Dilute CB008 constututive promoters (pTEF1)1:20 times Incubate overnight 30C shaker/
6/30/14
Repeat pGEM12,pGEM15, rtTA insert digestions with Minipreps from freezer box Same procedure as from 6/27/14  pGEM12-21.05 ng/ul pGEM15-23.03 ng/ul rtTA-182.3 ng/ul  REDO Gibson Assembly (procedure from 6/27/14). Plate all Gibson mix and incubate 37C overnight on LB-CAM.
7/1/14
Seamless Cloning
pGEM12- 2 ul            3 ul pGEM13- 1 ul            2 ul pGEM14- 3 ul            4 ul pGEM15- 2 ul            3 ul pGEM16- 1.5 ul          2.5 ul Positive- 1 ul PUC      3 ul        2 ul control insert Negative- 2 ul          2 ul         DNA backbone    Enzyme Mix  All with 1 ul of rtTA insert except for positive and negative control.
Transform into C2987 competent cells
25 ul competent cells 3 ul Seamless Assembly Mix  30 min ice 45 sec heatshock 42C 2 min ice Add 250 ul SOC media 1 hr incubation 37C  Plate all on LB+Cam
Miniprep HY/PTS cultures
Follow same miniprep procedure as previous experiments. Concentrations (in ng/ul) 1. PTS98    66.8 2. PTS97    88.2 3. HY67E1   63.43 4. HY6E1    22.05 5. PTS108   120.8 6. HY86E3   118.6 7. PTS133   701.3
Transforming pTEF1+GFP into DH5a
.5 ul plasmid DNA (pGEM12 to pGEM16)
7/2/14
Plated Seamless Cloning Transformation on LB+Carb, not LB+Cam.
7/3/14
Colonies found on LB+Carb plates.
Negative control has 5 colonies(?)
Positive control has too numerous to count.
All other colonies had ~100 colonies.
GFP
Innoculated GFP+pGEM12 to pGEM16 colonies. Stored in -80C freezer. Move plates from 4C to 37C incubator on Sunday.
rtTA
We ran a colony PCR on pGEM12-16 + rtTA. Follow the same procedure as before, using forward/reverse primers for rtTA. Used a 50C annealing temperature instead of 55C due to melting point of rtTA. Colonies all worked! Innoculated all colonies and placed in 4C freezer.
7/6/14
Moved all tubes from 4C freezer to 37C incubator shaker.
7/7/14
Miniprepped all tubes
Concentrations (in ng/ul)of rtTA plasmids 1. pGEM12-1     540.2 2. pGEM12-2     560.4 3. pGEM12-3     375.6 4. pGEM13-1     360.3 5. pGEM13-2     488.9 6. pGEM13-3     ------ 7. pGEM14-1     302.3 8. pGEM14-2     425.5 9. pGEM14-3     513.5 10. pGEM15-1    595.8 11. pGEM15-2    554.4 12. pGEM15-2    510.6 13. pGEM16-1    592.3 14. pGEM16-2    545.8 15. pGEM16-3    302.3  Concentrations (in ng/ul) of GFP plasmids 1. pGEM12- 425.5 2. pGEM13- 620.3 3. pGEM14- 521.9 4. pGEM15- 567.7     5. pGEM16- 534.3
*discovered that rtTA inserts are wrong because activation domains may not have been copied with PCR
7/8/14
Transformed 11E-3128 into DH5a (pTET-mfa)
50 ul DH5a .5 ul plasmid DNA Heatshock transformation Plate 100 ul on LB+Carb
pGEM13+rtTA sequencing did not work (weird insertion in rtTA before terminator sequence)
Innoculate pGEM16+GFP again.
7/9/14
Miniprepped pGEM16+GFP
Re-PCR of rtTA insert
Materials
1x reaction
5x Phusion HF Buffer
10 µl
dNTP's (10 mM)
1 µl
Forward Primer (#70)
2.5 µl
Reverse Primer (#39)
2.5 µl
*Template DNA
1 µl
Phusion Polymerase
0.5 µl
Water
31 µl
DMSO
1.5 ul
Total
50 µl
Thermocycler Cycles:     Initial Duration   | 95° C | 5 min        30 Cycles of:            Denaturation | 95° C | 30s           Annealing    | 50° C | 30s           Extension    | 72° C | 2 min     Final Extension    | 72° C | 5m     Hold               | 4°  C | Forever  *used 10 ul of dNTPs instead of 1 ul
Pme1 Digest of pGEM12-16 (without pGEM13) +rtTA
Digest 2000ng of template DNA Pme1 enzyme-        .5 ul Cutsmart Buffer-    2.5 ul DNA plasmid-        4 ul  Incubate 1 hr 37C.
Ran a gel for rtTA PCR
PCR Purification of rtTA
Eluted in 50 ul
Gibson Assembly
pGEM12-         1 ul rtTA insert-    3 ul GA MM-          4 ul  Incubate 15 min at 50C.
Transform
25 ul Mach1 cells .5 ul pGEM13+rtTA plasmid  Plate all on LB+Carb
7/10/14
Colony PCR of pGEM13+rtTA
-same procedure as 7/9/14 -successful
Miniprep 11E 3128 (pTET+mfalpha)
-followed standard miniprep procedure. Concentration: 430 ng/ul SEQUENCING CORRECT! (7/14/14)
7/11/14
Miniprep of pGEM13+rtTA
Sequencing correct
pGEM16 (m10+GFP mislabeled m3+GFP) and pTET+mfalpha
7/15/14
PCR rtTA out of correct pGEM20
*threw away all incorrect rtTA inserts 5x HF Buffer 10 ul dNTPS                               1 ul Phusion Polymerase                  .5 ul pGEM20- rtTA template               1 ul H2O                                 21 ul FW Primer (different for pTEFs)     2.5 ul RV Primer (rtTA reverse primer)     2.5 ul DMSO                                1.5 ul  Thermocycler:     95  5 min     30x cycle:         95  45 sec         50  30 sec         72  2 min     72  5 min     4   hold
7/16/14
Load pGEM1-13+rtTA on gel
Gels were leaky and had spillage on contents
Gel extractions
1. PRM1     .54g 2. PRM2     .56g 3. PRM3     1.15g 4. PRM6     1.21g 5. ECM18    .61g 6. SAG1     .68g 7. YDR124W  .45g 8. CLG1     .40g 9. ASG7     .31g 10. HYM1    .50g 11. PCL2    .42g 12. M3      .69g Followed same gel extraction procedure
Gibson Assembly for m3
3 ul insert 1 ul backbone 5 ul GA Master Mix 1 ul H2O
PCR of rtTAs (again)
same procedure as before, except: use 20 ul gel extraction as template
Transform m3+rtTA cells from Seamless Cloning
25 ul cells 1 ul Gibson DNA
7/17/14
Colony PCR on m3+rtTA
all had the rtTA insert! made liquid cultures and incubated overnight
7/18/14
Miniprep pGEM13+rtTA
eluted in 50 ul 1. 500.2 ng/ul 2. 386.9 ng/ul 3. 433.4 ng/ul 4. 400.5 ng/ul
7/20/14
Sequencing came back conclusive! Silent mutation at 417 bp.
7/21/14
Innoculated pTET+GFP yeast strains in 5ml YPD for transformation
7/22/14
Prepare cultures for transformation
Diluted pTET+GFP yeast strains 1:20. Incubated for 3 hrs.
Pme1 Digest
Pme1 digest to linearize the pTEF1 m3+rtTA circular plasmid for yeast transformation.  Digest 2000 ng DNA. 4 ul DNA .5 ul Pme1 Enzyme 2.5 ul Cutsmart Buffer 18 ul H2O  Incubate at 37C for 1 hr.
Transform Yeast
Transformed pTEF1-mr+rtTA into CB008/CB008DB pTET+GFP. Follow standard procedure for transforming yeast found in our protocols page.  -maybe not enough salmon sperm Plated on SD URA in the 30C incubator for 2 days.
7/23/14
Transforming E.coli with Fluorescent Proteins
25 ul DH5a cells .5 ul DNA
7/24/14
Yeast Colony PCR
Followed colony PCR procedure from before Used primers for PSV606 Used 1.5 ul DMSO per reaction 3 min annealing step  FAILED
7/25/14
Diluted CB008/DB pTET+GFP strains for transformations
Transformations of m3+rtTA
used 4 ul DNA Plate on SD URA
7/28/14
Yeast Colony PCRS
1 ul of FW primer 1 ul of RV primer 10 ul Green MM 10 ul Yeast cells 1.5 ul DMSO  Ura: 606/606 His: RA145/RA146 Trp: 604v2/604v2  -FAILED-
Yeast Transformations
Transformed pAGA+mCherry into yeast strains. (All with pTET+GFP, and pTET+mfalpha) CB008DB m6+rtTA         m7+rtTA CB008   m10+rtTA         m6+rtTA         m7+rtTA         pTEF1+rtTA  Transformed pTET mfalfpha into CB008DB pTET+GFP         pTEF1+rtTA  Transformed m3+rtTA into pTET+GFP CB008/DB
Miniprep
PTS94- 150 ng/ul PSW696- 60 ng/ul PS2607- 125 ng/ul
Colony PCR
Colony PCR on URA, HIS, and TRP sites. Repeat Colony PCR done earlier today.
7/29/14
Colony PCR
Repeat Colony PCR from yesterday. FAILED. Possibly errors in the primers for TRP. should show 300 bp. Use a 100 bo ladder?
More Colony PCR
Repeat URA and TRP colony PCRs Made own Green MM from (per reaction)     .125 ul Gotaq enzyme     5 ul 5x Buffer     .5 ul DNTPs Still FAILED.
7/29/14
More failed colony PCRs
7/31/14
PCR Purification
BFP-118.4 ng/ul     digest with Xho1/Not1 Aga-144.2 ng/ul     digest with Apa1/Xho1
Double Digestion of BFP
Digest 2000ng 17 ul   BFP PCR Purification .5 ul   Xho1 Enzyme .5 ul   Not1 Enzyme 2.5 ul  Cutsmart Buffer 4.5 ul  Water  Incubate 37C for 3 hrs
Digestion of RFP
Digest 2000ng 15 ul   pAGA1 PCR Purification .5 ul   Apa1 Enzyme 2.5 ul  Cutsmart Buffer 7 ul    Water  Incubate Room Temperature for 2 hrs  Add .5 ul Xho1 Enzyme     Incubate 37C for 1 hr
Digest mCherry+pAGA1 with Pme1 for Transformation
6 ul    pGEM22 .5 ul   Pme1 Enzyme 2.5 ul  Cutsmart Buffer 16 ul   Water  Incubate 37C for 1 hr
Transformations (Yeast)
Transformed pAGA1+mCherry into (CB008):    pTEF1             m6               m7 (CB008DB)   m6             m7  All contain the other motifs in the circuit.
PCR Purify Digestions
Followed procedure for PCR Purifications in protocols. Final Concentrations: BFP: 17.71 ng/ul RFP: 21.75 ng/ul
Ligations
1. HY130E Backbone with pAGA1+BFP (TRP) 2. HY130E Backbone with pPCL2+BFP (TRP)  1)  3 ul    Backbone     1 ul    BFP     1 ul    pAGA1     1 ul    T4 Ligase     1 ul    10X Buffer     3 ul    Water  2)  3 ul    Backbone     1 ul    BFP     .5 ul   pPCL2     1 ul    T4 Ligase     1 ul    10X Buffer     3.5 ul  Water  Incubate room temperature Kept in Ianto's freezer box
8/1/14
Pme1 Digest of pTET+mfalpha
Same protocol as before
Transformations (Yeast)
CB008 m3 transform- m3+rtTA CB008DB m10 transform- pTET+mfalpha
8/4/14
Remade 20mM NaOH
Colony PCR
CB008 pAGA+mCherry:     m6, m7 CB008DB pAGA+mCherry:     m6, m7
Made Liquid Cultures
CB008:     pTET+mfalpha         pTEF1, m3, m6, m7     m10+rtTA CB008DB      pTET+mfalpha         pTEF1, m6, m7     pTET+GFP (for m3+rtTA transformations)     pAGA1+mCherry m10
8/5/14
Colony PCR
CB008/DB pAGA1+mCherry     m6, m7 Most of the colonies worked
Gel
Ran a gel of: Pme1 Digest of pAGA1+mCherry                pTET+mfalpha
8/7/14
Liquid Cultures
5 ml YPD cultures of the 8 mfalpha strains from 8/4/14 to transform pPCL2+mCherry
Exploratorium Presentation
8/8/14
Dilutions
Made 1:20 dilutions of pTET+mFalpha strains: CB008: pTEF1, m6, m7, m10 CB008DB: pTEF1, m3, m6, m10 *for pPCL2+RFP transformations
Colony PCR
CB008:     m3+rtTA      pTEF1, pAGA1+mCherry CB008DB:     m3, pAGA1+mCherry     pTEF1, pAGA1+mCherry     m6, pAGA1+mCherry     m10, pTET+mfalpha  Boiled 100C for 20 min Primers-    URA: 606             HIS: RA 145/146             TRP: RA 145/148
Pme1 Digests
(Same for all digests) .5 ul   Pme1 Enzyme 2.5 ul  Cutsmart Buffer  RFP:    9 ul of pPCL2+mCherry          13 ul of water BFP:    4 ul of pPCL2+BFP         18 ul of water  Did twice for more transformations
Transformations
Tranformed pPCL2+BFP and pPCL2+mCherry into mfalpha strains
8/11/14
Colony PCR
BFP and RFP transformations from 8/8/14 Used RA145/148 for TRP 1.5 ul DMSO per reaction
Transformations (Yeast)
pTET+mfalpha into CB008 m3 pAGA1+mCherry into CB008DB m10 pPCL2+RFP into CB008DB m10 pPCL2+BFP into CB008DB m10
Streaked Plates
CB008     pTEF1, pAGA1+mCherry     m3+rtTA CB008DB     pAGA1+mCherry         pTEF1, m3, m6     m10, pTET+mfalpha
8/12/14
Glycerol stocks
CB008     pTEF1, pAGA1+mCherry     m3+rtTA CB008DB     pAGA1+mCherry         pTEF1, m3, m6     m10, pTET+mfalpha  (yGEM69-74)
Pme1 Digestions
pPCL2+BFP pPCL2+mCherry pAGA1+mCherry pTET+mfalpha
Colony PCR
pPCL2+BFP pPCL2+mCherry
Transformations (DH5a cells for Miniprepping)
pTET+mfalpha pPCL2+BFP pPCL2+mCherry pAGA1+mCherry
Transformations (Yeast)
CB008 m3, pTET+mfalpha CB008DB m10     pPCL2+mCherry, pAGA1+mCherry, pPCL2+BFP
Pme1 Digest
pTET+mfalpha
Streaked Plates
Streaked mCherry and BFP successful colonies from Colony PCR
Liquid Cultures of DH5a transformations
8/13/14
Dilutions
Made 1:20 dilutions of BFP/RFP cultures CB008 BFP     pTEF1, m6, m7, m10 CB008 RFP     m6 CB008DB RFP     m7, m10 CB008DB BFP     m10
Colony PCR
Colony PCR of BFP/RFP transformations -FAILED-
Miniprepped DH5a transformations for plasmids
pTET+mfalpha RFP BFP
Streak/Glycerol Stocks
CB008      pTEF1- RFP/BFP     m6- RFP/BFP     m7- RFP/BFP     m10- BFP CB008DB     pTEF1- RFP/BFP     m3- RFP/BFP     m6- RFP/BFP     m7- RFP/BFP
Transformations
CB008     m6, RFP     m10, BFP
8/14/14
Colony PCR
CB008 m3, pTET+mfalpha CB008DB m10, pPCL2+RFP              pAGA1+RFP              pPCL2+BFP *used .5 ul Zymolyase per reaction Thermocycle:     37C         5 min     95C         5 min     30x 95C     45 sec         50C     30 sec         72C     1 min     72C         10 min     4C          hold
8/17/14
Colony PCR
RFP/BFP strains 56 colonies used RA145/148 and zymolyase
8/18/14
Colony PCR Check
pAGA1s successful pPCL2 failed
Colony PCR
pPCL2+mCherry pPCL2+BFP pSAG1+Ste2 (use 87/91 primers. around 600bp)
Transformations (Yeast)
AFRPS: 7 pPCL2+mCherry  1 pAGA1+mCherry 4 rtTAs  m3 CB008 pTET+mfalpha
8/19/14
Streaked Plates
pPCL2+mCherry AFRPS m10, pPCL2+BFP
Liquid Cultures
23 things for glycerol stocks and transformations
8/20/14
Pme1 Digests
Digest 4000 ng pPCL2+BFP pAGA1+RFP pTET+mfalpha  Incubate 2 hrs 37C
Colony PCR
*extended extension time to 2 min* ~940bp Ste2 in NAT integration site 87/91 primers
Transformations (Yeast)
pPCL2+BFP: PRM1, PRM2, PRM3, YDR124W, CLG1, ASG7, HYM1
Glycerol Stocks of yGEM101-126
8/21/14
Restreak ALL Plates
Yeast plates die after 2 weeks Restreak on new YPD
Colony PCR
CB008/DB m6 for:     pTET+mfalpha, pAGA1+RFP *flow data suggests false positives of RFP
Transformations (Yeast)
pSAG1+rtTA pPCL2+rtTA pPRM6+rtTA, suspended pECM18+rtTA, suspended m3, pTET+mfalpha
Made new LiOAC, 10x TE Buffer, PEG 3350
ShuaiXin's Lab Notebook is unfortunately not available in digital form.
Thurs 14/05/29
1-13 Promoter Number Key: 1) PRM2 2) ASG7 3) FUS2 4) PCL2 5) FUS3 6) CLG1 7) YDR124W 8) HYM1 9) PRM6 10) PRM1 11) ECM18 12) PRM3 13) SAG1
PCR of 13 Alpha-factor inducible promoters
Reagents
1x reaction
4.5x MM
1) 5x HF Phusion Buffer
10ul
45ul
2) 10mM dNTPs
1ul
4.5ul
3) 10mM FW primers
2.5ul
11.25ul
4) 10mM Rev primers
2.5ul
11.25ul
5) Phusion Polymerase
0.5ul
2.25ul
6) Template DNA
0.5ul
2.25ul
7) ddH2O
33ul
148.5ul
~1kb: YDR124W, SAG1, PCL2, HYM1, FUS3
~700bp: CLG1, PRM2
~500bp: PRM6, PRM3, PRM1, FUS2, ECM18, ASG7
PCR Cycle (30) 98ºC    30secs 98ºC    10secs 55ºC    20secs 72ºC    30secs 72ºC    5mins 4ºC     hold  PCR products of #3,10,12 did not work. We Purified and nanodropped for concentrations:                 ng/ul 1) PRM2         13.49 2) ASG7 12.50 3) FUS2 4) PCL2         9.489 5) FUS3         11.77 6) CLG1         16.30 7) YDR124W      35.75 8) HYM1         23.61 9) PRM6         79.03 10) PRM1 11) ECM18       14.17 12) PRM3 13) SAG1        26.07
Fri 14/05/30
Miniprepped overnight cultures of phy4 (4)
Conentrations:
55.15 ng/ul
65.40 ng/ul <-- pipette error, less than 43ul so we added 5ul H2O
52.89 ng/ul
31.65 ng/ul
Used #1,2 for digest with ApaI and XhoI
Digestion (Yesterdays Promoter PCR products, 10)
29ul of product 33ul of 10x Cutsmart buffer  1) Add 0.5ul of ApaI. Mix gently, spin down, incubate at room temp. for 2 hours 2) Add 0.5ul of XhoI. Mix gently, spin down, incubate at 37ºC for 1 hour 3) PCR cleanup (purple tubes) for PCR reactions (Promoters)  Concentrations after PCR cleanup: 1) 14.54 2) 4.326 redo, 5.489 4) -0.05256 redo, 1.730 5) 1.886 redo, 2.642 6) 41.05 7) 3.187 8) 19.09 9) 9.677 11) 1.868 13) 23.78
Digestion (Backbone pHY4) Cutting out present promoter to replate with ours
43ul of DNA 5ul of Cutsmart  1) Add 1ul ApaI. Mix gently, spin down, incubate at room temp for 2 hours 2) Add 1ul of XhoI, Mix gently, spin down, incubate at 37ºC for 1 hour 3) Gel extract backbone, don't want the promoter cutout
Mon 14/06/02
Ligated #1,2,5,6,7,8,9,13 with the 6kb pHY4 backbone
We only had enough backbone for 8 reactions so we chose the 8 promoters with the highest concentration.
Used Gibson ligation calculator to calculate mLs of reagents of our reaction.
Genes
Concentrations
Backbone(ul)
Insert(ul)
Water(ul)
PRM2
14.54
3
1.1
12.9
ASG7
5.489
3
2
12
FUS3
1.73
2.3
10
4.7
CLG1
41.05
3
0.4
13.6
YDR124W
3.187
3
7.1
6.9
HYM1
19.09
3
1.2
12.8
PRM6
9.677
3
1.2
12.8
SAG1
23.78
3
0.9
13.1
Transformed α-inducible Promoters in pHY4 into DH5α competent cells. 50uls for each transformation instead of 25uls.
9 Plates for 8 promoters and 1 negative control.
Redo-PCR of AFRPs that didn't work or had low yield. Redoing: 3,4,7,10,11,12
After PCR with the same primers, we ran it through a 1% gel
Tues 14/06/03
Transformation of pHY4 + AFRP
#7  ⎫ #9  | Grew at least a few colonies so they did okay #13 ⎭
The rest, 1,2,5,6,8 didn't grow likely because of our low yield template promoter DNA.  We need to restart and PCR promoters again for higher yield DNA. Then ligate and transform
Yesterdays AFRP PCR yields: #4   84.35 ng/ul #7   134.3 ng/ul #10  84.14 ng/ul #11  91.99 ng/ul #12  64.64 ng/ul
Miniprepped cultures of Mach1 with pHY4 to get more pHY4 template.
*The PE buffer we used previously did not contain ethanol which was why we were having low yields for minipreps and other purifications.
Concentraions of pHY4 1) 360.2 ng/ul 2) 398.8 ng/ul 3) 322.2 ng/ul 4) 240.2 ng/ul
Digestion of AFRP PCR products (4,7,10,11,12)
50ul PCR product 5ul of 10x Cutsmart buffer
1) add 1ul of ApaI, incubate at room temp for 2 hours
2) add 1ul of Xho1, incubate at 37ºC for 1 hour
Digestion of pHY4 backbone (2 tubes)
43ul of DNA 5ul Cutsmart buffer  * 1 unit of enzyme = 1ug of DNA = 1000ng     * 300ng/ul *43ul = 12,900 = Need 13units     * Unit count depends on the RE. Need to check the product.     * Always want excess RE  1) add 1ul ApaI, incubate at room temp for 2 hours 2) add 1ul of Xho1, incubate at 37ºC for 1 hour
Ran the backbone digest through a gel.
The streaks of pHY4 suggests that its been cut up by other nucleases so we can't purify this as our backbone.  So we created overnight cultures of Mach1+pHY4 and are sequencing leftover digested phy4 to see if we have the backbone we want.
Created overnight cultures of the transformations that seemed to work: 7,9,13
Tomorrow we will miniprep and purify the construct for promoter testing.
Weds 14/06/04
Miniprepped yesterdays cultures of YDR124W, SAG1, and PRM6, pHY4
Concentrations of minipreps YDR124W.1     539.4 ng/ul YDR124W.2     555.4 ng/ul YDR124W.3   594.3 ng/ul YDR124W.4   652.5 ng/ul YDR124W.5   481.5 ng/ul YDR124W.6   377.7 ng/ul  PRM6.1      545.3 ng/ul PRM6.2      569.1 ng/ul PRM6.3        656.8 ng/ul PRM6.4      570.8 ng/ul PRM6.5      558.6 ng/ul  SAG1.1      538.1 ng/ul SAG1.2      532.1 ng/ul SAG1.3      505.8 ng/ul  pHY4.1      179.8 ng/ul pHY4.2      155.6 ng/ul pHY4.3      231.7 ng/ul
Ianto and Jessica digested and PCR purified 1,2,6,8,9,10,13
AFRP PCR Concentrations: 1. PRM2   58.91 ng/ul 2. ASG7   68.53 ng/ul 6. CLG1   15.13 ng/ul 8. HYM1   78.56 ng/ul 9. PRM6   132.6 ng/ul 10. PRM1  41.72 ng/ul 13. SAG1  91.24 ng/ul
Made a positive control digest of Hyun's pHY4 backbone.
Tomorrow we will run it through a gel and gel extracted it.
Thurs 14/06/05
Yesterday we made solutions for the yeast transformations we will do today.
Check chemical container information to confirm molar mass, sometimes they may be different.  50mL of LiOAc: Molar mass = 60.99g/mol / 20 = 3.3 grams in 50mL of H2O ~pH8  50mL Trizma: Molar mass = 157.60g/mol / 10 / 20 = 0.788 grams  50mL EDTA Molar mass = 292.24g/mol.  Used 0.146 grams in 50mL of H2O ~pH7.5  50% PEG.  Used 25grams of PEG + H2O until 50mL.  Heat in 50ºC water bath to melt all PEG.  Store tube in foil to protect from light at 4ºC
Fri 14/06/06
Out for the day. What I missed:
1-13 Promoter Number Key:     1) PRM2     2) ASG7     3) FUS2     4) PCL2     5) FUS3     6) CLG1     7) YDR124W     8) HYM1     9) PRM6     10) PRM1     11) ECM18     12) PRM3     13) SAG1
Transformed 1,2,4,6,7,10,11,12, and negative control.  The negative control grew colonies so the backbone may have liated back by itself so we are using phosphitase to remove phosphates so that it cannot ligate together.
They then colony PCRed and sequenced.
AFRP
1
2
3
4
5
1
✓
X
X
✓
✓
2
✓
✓
X
✓
✓
4
✓
✓
✓
✓
✓
6
✓
✓
✓
✓
✓
10
X
✓
✓
X
X
11
X
✓
X
✓
✓
12
✓
✓
✓
✓
✓
Mon 14/06/09
Today we are miniprepping 2 of each successful colonies from the colony PCR
YDR124W ⎫ PRM6    | All grew 100+ colonies SAG1    ⎭
We are doing colony PCR on 5 colonies of each transformaiton. All 5 colonies are also patched as a stock and are circled on the plate when colony PCR confirms that they have the gene knock in.
Anneling temp at 50ºC for 30secs Polymerase extension at 72ºC for 1min Length of PCR product should be 940bp
Tues 14/06/10
Ianto and Jessica are digesting pHY4 + AFRP, all 11, and are PCRing out the bakbone w/o the GFP.  Derrick has PCRed the rtTA with ends that have homolog to the digest ends NotI and XhoI of the plasmid being digested. 9 of 11 PCRs have worked for Derrick.  Digest overnight at 37ºC
Finished product should be pHY4+AFRP+rtTA. Then they will be digested with PmeI ad transformed into yeast.
Weds 14/06/11
Redoing PCR of 2 and 12 of rtTA with homology ends to the 2, 12 promoters and the backbone.
PCR of gibson homology of rtTA to ASG7 & PRM3
5x HF buffer            10ul dnTPs                   1ul FW primer               2.5ul Rev primer          2.5ul Phusion Polymerase  0.5ul Template                0.5ul ddH2O                   33ul  PCR cycle (30) 98ºC   30secs 98ºC   10secs 55ºC   20secs 72ºC   30secs 72ºC   5mins 4ºC        hold
Concentrations of the 9 successful so far of PCR of gibson homology
1.PRM2       190.9 ng/ul 4.PCL2   149.1 ng/ul 6.CLG1       412.1 ng/ul 7.YDR124W  90.37 ng/ul 8.HYM1       248.9 ng/ul 9.PRM6      145.7 ng/ul 10.PRM1    71.88 ng/ul 11.ECM18   199.7 ng/ul 13.SAG1    65.55 ng/ul
We gibsoned inserts of rtTA with homology to the PCR backbone w/o GFP.  The gibson reaction was in 10ul with 2 controls (backbone w/o insert)-1)CLG1, 2)PCL2
Incubated samples at 50ºC for 1hour
Thurs 14/06/12
The gibson reaction with backbone w/o GFP and insert of rtTA did not work
Today we will do a seamless reaction which is similar to gibson but is better with inserts that have shorter homology sequences.
Mon 14/06/16
Transformations of rtTA+AFRP in E.coli are going to be colony PCRed.  Of the 11, SAG1 & PRM1 did not grow.  We are PCRing rtTA, seeing if it has successfully transformed.
Protocol: Inoculate colony in 25ul of H2O, use 5ul for PCR template and the rest can be use for making 5mL cultures
Insert E.coli colony PCR
PCR mastermix
Reagents
1x
4x
Template DNA
5ul
n/a
Gotaq mix (2x)
12.5ul
50ul
FW primer
1.25ul
5ul
Rev Primer
1.25ul
5ul
H2O
5ul
20ul
PCR cycle (30x) 98ºC   2min 98ºC   30secs 55ºC   30secs 72ºC   1.5mins 72ºC   5mins 4ºC        hold
Colony PCR of the 22 transformations.
Insert yeast colony PCR
PCR mastermix
1x
70x
2x GoTaq
10ul
700ul
10uM FW
1ul
70ul
10uM Rev
1ul
70ul
H2O
5ul
350ul
boiled yeast template
3ul
n/a
PCR cycle (30x) 95ºC   5mins 95ºC   45secs 50ºC   30secs 72ºC   1min/kb 72ºC   10min 4ºC        hold
Every PCR is being run on 1% agarose gel
Number of colony PCRs: 66 colonies from yeast, 27 from 9 rtTA+AFRP, 25 from constitutive promoters
Tues 14/06/17
Since colony PCR of the 22 transformations didn't all work, we are redoing it.  Only 5 strains of the 22 strains were successful (11 CB008, 11 CB008DB).  Checking URA site integration, expecting 940bp bands.
Colony PCRs to redo:
CB008DB
CB008
PRM1
PRM1
YDR124W
YDR124W
PRM3
PRM3
PRM6
PRM6
PCL2
ECM18
SAG1
PRM2
HYM1
CLG1
ASG7
Mastermix
1x
80x
2x GoTaq
10ul
800ul
10uM FW
1ul
80ul
10uM Rev
1ul
80ul
H2O
5ul
400ul
Boiled Yeast template
3ul
n/a
FW/REV primer: "606V2"
Yesterday we made culturesof the AFRP+rtTA (all successful), 2 of each.  Today we miniprepped them to prepare to transform into yeast.  Transform into yeast after linearization/digest with Pme1.
Concentrations of AFRP+rtTA constructs:
ECM18+rtTA #1    308.3 ng/ul ECM18+rtTA #2    231.6 ng/ul PCL2+rtTA #1     162.7 ng/ul PCL2+rtTA #2     247.3 ng/ul PRM6+rtTA #1     248.1 ng/ul PRM6+rtTA #2     327.9 ng/ul ASG7+rtTA #1     351.7 ng/ul ASG7+rtTA #2     593.5 ng/ul HYM1+rtTA #1     221.7 ng/ul HYM1+rtTA #2     413.0 ng/ul PRM2+rtTA #1     324.3 ng/ul PRM2+rtTA #2     339.6 ng/ul PRM3+rtTA #1     351.5 ng/ul PRM3+rtTA #2     316.7 ng/ul CLG1+rtTA #1     289.0 ng/ul CLG1+rtTA #2     276.8 ng/ul YDR124W+rtTA #1  366.0 ng/ul YDR124W+rtTA #2  239.3 ng/ul
Weds 14/06/18
Transforming 9 AFRP+rtTA plasmids into yeast.  Ianto linearized the plasmids #1 and some of #2 depending on the sequences that came back (most won't work)
Preparing for the first flow experiment tomorrow.  Everyone gets a promoter.
AFRP+rtTA owners PRM2 - George ASG7 - Sabrina PCL2 - Eric CLG1 - Derrick YDR124W - Ianto PRM6 - Ianto PRM1 - Robert ECM18 - Robert PRM3 - Eleanor SAG1 -Jessica
Also transformed the 2 AFRP+GFP that didnt work last time HYM1 (08/DB) & PRM3 (DB)
Made overnight cultures of the strain CB008+AFRP to test in on the flow after induction with 5 alpha-factor concentrations (0nM, 10nM, 1000nM, 10,000nM, 100,000nM)
Thurs 14/06/19
Overnight cultures of CB008+AFRP+GFP diluted ~100x (to a final concentration of OD600 0.5-0.1, Saturated overnight cultures should be OD600 of ~7) in SD complete media, and grown for 3 hours, 1000rpm shaker, 30ºC. Growing in 2mL well plates
Induce with Alpha-factor. Stock is in 3mM. Final concentrations are 0, 1nM, 10nM, 100nM, 1000nM.  Alpha-factor cannot be refrozen, so throw leftover away.
Induce for 90mins, but no longer than 120mins
Transfer 250u of each well into a V-bottom 96-well plate containing 10ul of the fixing chemical Cyclohexamide. (thats 4 ul for every 100ul of culture)  Cyclohexamide stops protein production by inhibiting ribosomes.
Run on the flow cytometer.
Flow Cytometer things to remember:
check the sheath fluid box to see if its empty
check the waster container.  If full, dispose and add new bleach
take care not to leave the machine on run. Save sheath fluid by keeping on standby when not in use
find parameters using negative controls. (FSC, SSC, Fluorescense marker)
DO NOT RUN PLATE. RUN WELLS instead. (Run the first well by itself first b/c it needs time to create file folders.) this causes problems.
Run plate only saves data of the first well.
Can rerun through the first well again which would suck up air.  Air no good for the machine.
highlight wells and change settings on the right in the aquisition dashboard screen
mixing speed 180ul/sec
mix 3 times rather than 2
run at 1ul/sec for samples and 0.5ul/sec for negative controls when finding parameters.
make sre uL doesn't exceed samle to prevent sucking up air. (Rule of thumb is to run 50ul less than allotted sample sizes for wells.)
Voltage settings were gotten by running negative controls and adjusting to readouts seen in histogram form of FSC, SSC, FITC
Parameters of todays flow
FSC: 250 SSC: 280 FITC(GFP): 550 B(RFP): 650  Flow rate: 1µL/sec Sample Volume: 200µL Mixing Volume: 100µL Mixing speed: 180µL/sec
Plate map
1   2   3   4   5   6   7   8   9   10  11  12 A   [------NEG------]       [------NEG-------] B   [------PRM2-----]       [------PRM1------] C   [------ASG7-----]       [------EMC18-----] D   [------PCL2-----]       [------PRM3------] E   [------NEG------]       [------SAG1------] F   [------CLG1-----] G   [------YDR124W--] H   [------PRM6-----]
Analyze data.
Fri 14/06/20
Continuation of Weds 14/06/18:
6 of the 9 transformations in yeast failed for the AFRP+rtTA
Those that failed are: ASG7 CLG1 YDR124W HYM1 ECM18 PRM3
This means we need to go back to our E.Coli plates and miniprep colonies for sequencing again, then linearize and transform into yeast.  Yesterday cultures were made of these 6, 5 colonies each except YDR124W which only had 3.  Today we are miniprepping them and sending them for sequencing.
Nanodrop Concentrations: YDR124W.3   241.7 ng/ul YDR124W.4   386.5 ng/ul YDR124W.5   439.6 ng/ul ASG7.3      207.7 ng/ul ASG7.4      319.8 ng/ul ASG7.5      287.2 ng/ul ASG7.6      300.2 ng/ul ASG7.7      338.5 ng/ul CLG1.3      408.0 ng/ul CLG1.4      306.0 ng/ul CLG1.5      277.6 ng/ul CLG1.6      249.8 ng/ul CLG1.7      277.1 ng/ul HYM1.3      440.1 ng/ul HYM1.4      342.4 ng/ul HYM1.5      388.7 ng/ul HYM1.6      226.7 ng/ul HYM1.7      351.2 ng/ul ECM18.3     377.0 ng/ul ECM18.4     426.8 ng/ul ECM18.5     368.8 ng/ul ECM18.6     459.7 ng/ul ECM18.7     372.9 ng/ul PRM3.3      430.4 ng/ul PRM3.4      441.8 ng/ul PRM3.5      361.7 ng/ul PRM3.6      375.6 ng/ul PRM3.7      333.1 ng/ul
Mon 14/06/23
Found out that we were missing the activation domain of rtTA, the rtTA we've been cloning for the past 2-3 weeks has been wrong.
New primers for the full rtTA has been made
Digested more backbone with AFRP+GFP to prepare for correct rtTA insert.
Tues 14/06/24
Today we did Flowjo data analysis with Hyun and Kara with data from our last flow experiment, Thur 14/06/19
Data from usb transfer to save file folder in nanodrop room computer.
Drag all well data into flojo
Set up a gate for all data to select for appropriate size cells. Excludes large cells which are clumped together and small debree or bacteria, which are small.
Find the mean fluorescense and use that data to make a graph in matlab.
Code for my PCL2 graph:
alpha = [0.01 1 10 100 1000] CB008 = [446 438 532 592 590] yGEM5 = [2568 2759 3487 5829 7935]  figure(1) semilogx(alpha,CB008,'--bo') hold all semilogx(alpha,yGEM5,'--rs') xlabel('[Alpha-Factor][nM]') ylabel('MeanGFP[Au]') title('yGEM5 GFP vs [Alpha-Factor]') legend('CB008','yGEM5')
Primers for the correct rtTA with the activation domain came.  Jessica and Sabrina are doing the PCR for rtTA.
Gel photo of backbone digest +promoter+GFP
Already cut plasmid   49ul Fresh plasmid         5ul Cutsmart              6ul Not1                  0.5ul Xho1                  1ul  incubate at 37ºC for 2 hours
After digestion, Ianto & Jessica cut out and gel extracted the ~8kb backbone
Concentrations: (Ianto & Jessica) pHY4+ASG7+GFP   23.77 ng/ul pHY4+CLG1+GFP   16.59 ng/ul pHY4+HMY1+GFP   12.96 ng/ul pHY4+ECM18+GFP  21.39 ng/ul pHY4+PRM3+GFP   15.20 ng/ul  Concentrations: (Derrick) pHY4+YDR124W+GFP   15.46 ng/ul pHY4+SAG1+GFP      12.10 ng/ul pHY4+PRM6+GFP      21.74 ng/ul pHY4+PRM1+GFP      4.186 ng/ul pHY4+PCL2+GFP      6.348 ng/ul pHY4+PRM2+GFP      0.451 ng/ul
I was asked to transform 6 things in DH5α: 1. HY3E (has rtTA) 2. pHY4 (for stock) 3. HY130E (TRP) 4. HY121E1 (URA3) 5. HYS4E2 (HIS3) 6. HY111E2 (LEU2)
Weds 14/06/25
11:30am, made cultures my transformations done yesterday for minipreps.
DH5α Transformations
Colony Count
1. HY3E (has rtTA)
8
2. pHY4 (for stock)
100+
3. HY130E (TRP)
100+
4. HY121E1 (URA3)
100+
5. HYS4E2 (HIS3)
100+
6. HY111E2 (LEU2)
100+
Derrick and Ianto made yeast cultures to test promoters in flow tomorrow.
The yeast was taken from patch plates.
They created streak plates to get individual colonies as well.  We need individual colonies for biological replicates.
Thurs 14/06/26
Yesterday I made cultures of 6 transformations for minipreps. 1. HY3E (has rtTA) 2. pHY4 (for stock) 3. HY130E (TRP) 4. HY121E1 (URA3) 5. HYS4E2 (HIS3) 6. HY111E2 (LEU2) They grew for 22 hours so their concentrations are very high.
Miniprep concentrations (minipreps done by Jessica):
HY3E #1      628.1 ng/ul HY3E #2      875.2 ng/ul HY130E #1    549.3 ng/ul HY130E #2    696.8 ng/ul HY121E1 #1   152.9 ng/ul HY121E1 #2   539.7 ng/ul HYS4E2 #1    424.4 ng/ul HYS4E2 #2    672.7 ng/ul HY111E2 #1   555.3 ng/ul HY111E2 #2   679.0 ng/ul pHY4 #1      487.0 ng/ul pHY4 #2      554.2 ng/ul
Flow cytometry, testing (triplicate) PRM2, ASG7, PLC2, CLG1
Alpha-factor concentrations: (Induce for 90mins)
(0nm, 0.5nm, 1nm, 10nm, 100nm, 1000nm, 3000nm)
Starting concentration is 3mM or 3,000,000nM Make 100x stocks and add 10ul to the 1mL cultures: 0nM 50nM 100nM 1000nM 10,000nM 100,000nM 300,000nM
Plate 1 & 2 Alpha Factor Concentration Map:
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H     Plate 1:     H   G   F   E   D   C   B   A    1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------PRM2-1---------] 5       [--------PRM2-2---------] 6       [--------PRM2-3---------] 7       [--------ASG7-1---------] 8       [--------ASG7-2---------] 9       [--------ASG7-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]  Plate 2:     H    G   F   E   D   C   B   A    1       [--------CLG1-1---------] 2       [--------CLG1-2---------] 3       [--------CLG1-3---------] 4 5 6 7 8 9 10 11 12
Fri 14/06/27
Continuation of CB008+AFRP+GFP FACs.
PRM6+GFP did not grow up yesterday.
ROUND 3 PLATE 1 PROMOTER MAP
H   G   F   E   D   C   B   A    1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------YDR124W-1------] 5       [--------YDR124W-2------] 6       [--------YDR124W-3------] 7       [--------PRM3-1---------] 8       [--------PRM3-2---------] 9       [--------PRM3-3---------] 10      [--------PRM1-1---------] 11      [--------PRM1-2---------] 12      [--------PRM1-3---------]
ROUND 2 PLATE 2 PROMOTER MAP [DID NOT COMPLETE]
H    G   F   E   D   C   B   A    1       [--------ECM18-1--------] 2       [--------ECM18-2--------] 3       [--------ECM18-3--------] 4       [--------SAG1-1---------] 5       [--------SAG1-2---------] 6       [--------SAG1-3---------] 7 8 9 10 11 12
Instructions for starting flow:
Turn on and wait 30mins for the machine to warm up
Re-initialize the HTS, them prime it three times.  You'll get a bubble in the lines if you don't
Run CST!!! Follow the sheet in front of the monitor for more instructions for that part. (CST uses beads to calibrate the laser detection.) add one drop into 250ul of sheath fluid in A1.
Bead LOT ID: use the one that is most current
Load A1-A4 with bleach and B1-B4 with water [flip for the opposite corner] -Cytometer>CST.  Make sure cytometer performance results passed
Run Clean Plate: 1. Click on experiment. Experiment>Open experiment 2. Open clean plate "Daily Clean" - 96 well U-bottom" 3. Be here to see if the cleaning is going correctly, low events (less than 100events/sec for bleach and less than 10events/sec for water) 4. If over the events, run another clean plate 5. View events in aquisition dashboard in view>Acquisition dashboard 6. Can also make a clean plate using HTS>Clean
Mon 14/06/30
Continuation of CB008+AFRP+GFP FACS.  Also testing Constitutive pTEF1 promoters +GFP today
ROUND 4 MAP
H    G   F   E   D   C   B   A    1       [--------ECM18-1--------] 2       [--------ECM18-2--------] 3       [--------ECM18-3--------] 4       [--------SAG1-1---------] 5       [--------SAG1-2---------] 6       [--------SAG1-3---------] 7       [--------CB008-1--------] 8       [--------CB008-2--------] 9       [--------CB008-3--------] 10 11 12
Constitutive Promoter MAP
1   2   3   4   5   6   7   8   9   10  11  12 A   [-CB008-]        B   [-pTEF1-]        C   [-m3----]        D   [-m6----]        E   [-m7----]        F   [-m10---] G    H
Tues 14/07/08
Meeting with Anusuya:
Site directed mutagenesis of ribosome binding sites to fine tune expression of proteins.  Tune down the effect of the promoters
Two promoters back to back to increase expression?
Find out how to differentiate yeast strains when they are co-cultured.
Maybe work more with constitutive rather than the inducible because there might be variable behaviors and its hard to model.
Email ben and ask for code from last year.
Start modeling progress, promoters and outputs. Co-opt code that was written last year
Flow with Bar1 deleted yeast.
Find examples of Tcells that start out with slightly different inputs that strongly converge over time.  Chaotic phenomenon.
look up Chaotic phenomenon in immune cells.
Weds 14/07/09
1) Transforming pTEF1 + rtTA (and mutants m6, m7, m10) into CB008+pTET+GFP and CB008DB+pTET+GFP.  8 Transformations. 2) Creating cultures of 5 CB008DB+AFRP+GFP strains for flow analysis on Thursday.
[Flow with CB008DB AFRP+GFP] https://www.dropbox.com/s/ymmlrcgm3zkoxr0/Flow%20with%20CB008DB%20AFRP%2BGFP.xlsx [pTEF+rtTA progress] https://www.dropbox.com/s/wilm904hbvosox6/pTEF%2BrtTA%20progress.xlsx
Thurs 14/07/10
Flow of CB008DB strains +AFRP+GFP
plate 1
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4       [--------CLG1-1---------] 5       [--------CLG1-2---------] 6       [--------CLG1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]
Plate 2
H   G   F   E   D   C   B   A    1       [--------SAG1-1---------] 2       [--------SAG1-2---------] 3       [--------SAG1-3---------] 4       [--------AGA1-1---------] 5       [--------AGA1-2---------] 6       [--------AGA1-3---------] 7        8        9        10       11       12
Dilutions: 3 hour incubation with 100x dilutions in 2mL well plates. 9:35am -> 12:35pm
Added Alpha-factor.  There are several possible mistakes when inducing. On 2-3 of the wells of the 100nm and 1000nm concentration, alpha factor was on the walls of the plate and may not have been added to the culture fully.  Also, when using the multi-dispenser, the initial 10ul doesnt all fall off the tip, leaving residual liquid hanging. 12:50 -> 2:20pm
Alpha-Factor Concentrations
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H
Fri 14/07/11
CB008DB AFRP+GFP Round 2 Plate 1of1
Glycerol stocks of YDR124W, PRM6, & ASG7 did not grow in SD complete media overnight. The cells may have settled before freezing.
3 Hour incubation plate
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4        5       [--------PRM1-1---------] 6       [--------PRM1-2---------] 7       [--------PRM1-3---------] 8       [--------ECM18-1--------] 9       [--------ECM18-2--------] 10      [--------ECM18-3--------] 11       12
Incubated for an extra 25mins by accident. 9:30am -> 12:55pm
Added alpha-factor 1:05 -> 2:35PM
Flow Plate
H   G   F   E   D   C   B   A    1       [-------CB008DB-1-------] 2       [-------CB008DB-2-------] 3       [-------CB008DB-3-------] 4       [---------PRM1-1--------] 5       [---------PRM1-2--------]    6       [---------PRM1-3--------] 7       [--------ECM18-1--------] 8       [--------ECM18-2--------] 9       [--------ECM18-3--------]    10       11       12
Concentrations of Alpha-Factor are the same.
Mon 14/07/14
Group Meeting at 1pm Until then, everyone was working on their powerpoints for the presentation.
~3:00pm Colony PCR of CB008/CB008DB (+Constitutive Promoters +rtTA) (+pTET +GFP) 3 colonies of each were patched and PCRed onto URA knockout plates.
PCR Reaction: 10ul 2x GoTaq Mix 3ul ddH2O 1ul FW primer 606V2 (primers check for chromosomal integration at the URA2 site) 1ul RV primer 606V2 5ul boiled Template in 20mM NaOH  PCR Protocol: 95ºC 5mins 95ºC 30secs  | 50ºC 30secs  | x30 72ºC 1min    | 4ºC  hold
Running colony PCR on a gel tomorrow.
Tues 14/07/15
1). Running 24 colony PCRs. Expected band size is 940bp.
Gel Photo:
2). 6 Strains worked, the two that didn't are CB008DB+pTEF1+rtTA and CB008DB+m10+rtTA
3). We are redoing colony PCR of those 2 strains that failed, each with 5 colonies each this time. Update: Second colony PCR did not work so we will be doing a third colony PCR
Gel Photo:
4). Redoing Colony PCR for a third time, 5 colonies each again.  This time we included 2 positive controls to check if the PCRs are at fault.  If this doesn't work we will have to redo transformations for these two and we will most likely not run the DB strains on the flow tomorrow. Update: Colony PCR did not work. Gel Photo:
Flow Cytometry with CB008DB +AFRP+GFP 100x dilutions, 3 hour growth: 9:40am -> 12:40pm, alpha-factor induction 12:50 -> 2:20pm
plate 1
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4       [--------ASG7-1---------] 5       [--------ASG7-2---------] 6       [--------ASG7-3---------] 7       [--------YDR124W-1------] 8       [--------YDR124W-2------] 9       [--------YDR124W-3------] 10      [--------PRM6-1---------] 11      [--------PRM6-2---------] 12      [--------PRM6-3---------]
NOTE E3 & F3 had some weird clumping on the bottom of the well and somehow can't be resuspended, even with pipetting.  A2 skipped, human error.
Wed 14/07/16
1). None of 13 Colonies chosen from m10 worked so today George did another 10 colony PCRs.  Again, did not work so we need to retransform m10+rtTA into CB008DB+pTET+GFP. Gel Photo:
2). Flow plate maps today:  Attenuating GFP with 12 concentrations of doxycycline Both strains with Constitutive Promoter+rtTA +pTET+GFP
Ianto made 100x diluted cultures in the morning Incubation 7:25am -> 10:25am Induction 11:17am -> 5:17pm
[Doxycycline]: 100 mg/ml stock or 100,000 ug/ml (in 100ul Aliquots)  Induction time: 6hrs Concentrations needed for 100x:
0 ug/ml 3 ug/ml 6 ug/ml 9 ug/ml 30 ug/ml 60 ug/ml 90 ug/ml 300 ug/ml 600 ug/ml 900 ug/ml 3000 ug/ml 6000 ug/ml
Dox Dilutions:
Stock: 100mg/mL  **1x**           **100x**          Prep 60µg/mL       A: 6mg/mL            (6:100) 60µL of Stock in 940µL of water 30µg/mL       B: 3mg/mL            30µL of Stock in 970µL of water 9µg/mL        C: 900µg/mL          150µL of A in 850µL of water 6µg/mL        D: 600µg/mL          100µL of A in 900µL of water 3µg/mL        E: 300µg/mL          50µL of A in 950µL of water      Can also do 100ul of B in 900ul of water 0.9µg/mL      F: 90µg/mL           100µL of C in 900µL of water 0.6µg/mL      G: 60µg/mL           100µL of D in 900µL of water 0.3µg/mL      H: 30µg/mL           100µL of E in 900µL of water 0.09µg/mL     I: 9µg/mL            100µL of F in 900µL of water 0.06µg/mL     J: 6µg/mL            100µL of G in 900µL of water 0.03µg/mL     K: 3µg/mL            100µL of H in 900µL of water
Doxycycline concentrations plate map:
H   G   F   E   D   C   B   A    1                   [----0 µg/ml----] 2                   [---0.03 µg/ml--] 3                   [---0.06 µg/ml--] 4                   [---0.09 µg/ml--] 5                   [---0.3 µg/ml---] 6                   [---0.6 µg/ml---] 7                   [---0.9 µg/ml---] 8                   [---3.0 µg/ml---] 9                   [---6.0 µg/ml---] 10                  [---9.0 µg/ml---] 11                  [---30 µg/ml----] 12                  [---60 µg/ml----]
pTEF1 promoters
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008 Control]--------------]   B [---------------[CB008 pTET_GFP Control]-----]   C [---------------[CB008 pTET_GFP pTEF1_rtTA]--]   D [---------------[CB008 pTET_GFP pTEF1 m6]----]      E [---------------[CB008 pTET_GFP pTEF1 m7]----]     F [---------------[CB008 pTET_GFP pTEF1 m10]---]   G H    1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB Control]------------]   B [---------------[CB008DB pTET_GFP Control]---]   C [---------------[CB008DB pTET_GFP pTEF1 1of2 ]  D [---------------[CB008DB pTET_GFP pTEF1 2of2 ] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G  H
All wells ran its 10k events threshhold within seconds due to the amount of time they've been allowed to grow, 9 hrs.
3). Making cultures of CB008DB+pTET+GFP for tranformations of m10+rtTA tomorrow.
4). Yesterday's flow plate was contaminated and our data is very weird. (refer to Ianto's lab notebook on the same date) We will be redoing that experiment so today we should make more cultures for flow tomorrow.
Thu 14/07/17
Anusuya meeting: We should have everyone create a labnotebook/blog of what they do everyday so that we can keep track of everything. Background of T-cell signalling. Need good examples of what naturally happens.  We need a specific thing we want to focus on and model for our project.
Read:  1). interaction between T-reg cells and T-helper cells.  How T-reg with macrophages, their interaction represent some negative feedback.  Macrophage somehow suppresses T-regs. 2). T-reg, dentridic cells.  relates to neural.  Stanford 2010, interaction btwn T-reg, T-helper, Th1-th2. 3). Topic of Neural information? microglia interaction? Might play a role in recruiting and regulating T-cells when there is injury to the brain.
1). Read some wiki and review papers, dont need data papers be able to understand whats going on with these interactions so that we have many examples where interactions play important roles.
2). Look from a non-immune viewpoint.  There are interactions beyond the immune system so look into those and come up with examples. Perhaps we can build on the work from UCSF iGEM 2010.
3). COME UP WITH EXAMPLES, need an elavator pitch.  Whats still unknown about these cell types? and how does this model we are making with yeast help us to understand whats going on.  Specific termonology.  Have some slides with these examples set up for the next meeting.  Layout a rational.
4). Look up alpha factor diffusion rates.  Delegate tasks for all of these points.
5). Have a point person for the medal requirements.  Need someone to keep track of everything.
6). Maybe dedicate a 10min session to brief everyone about what has been done for the day and the end of every day.  Have it written day and keep it in a text file for people to access.
7). Folders for all our files should be put into folders for the wiki.  Update them!  Intellecutal material, protocols, etc.
8). Create a library of motifs we've constructed and explain how its valuable.  Find examples of them where they exist in the immune system.
Type 2 restriction sites. AAR1  BSA1
Modeling: Write code for both strains and their find out their output of alpha-factor.  Then have both sets of code and add them together to see RFP output when those two cells are together.
factors to account for the model: diffusion constants, degredation rate, cell density (later)
Overnight cultures of CB008DB+pTET+GFP grew well. Continuing to tranform m10+rtTA. Diluted 20x at 9:45am -> 11:45am
Fri 14/07/18
Transferred labnotebook to this textfile.
Looked up examples related to our project
Prepared for monday's group meeting
Helped Robert with what he should be doing in the lab
Mon 14/07/21
100+ Colonies grew for our m10+rtTA transformations in CB008DB+pTET+GFP.  We are now colony PCRing 10 colonies.
2-4pm group meeting
4 out of the 10 colony PCRs worked! m10+rtTA is now available for flow testing.
Everyone is assigned an article to read for an example related to our iGEM project. They were told to make slides of the signalling pathway if they could.
Tues 14/07/22
Made a streak plate of CB008DB+m10+rtTA+pTET+GFP from the patch plate.
100x dilutions done at 9:20am 9:20am -> 12:30pm Alpha-factor induction 12:47pm -> 2:17pm
Plate 1:
H      G   F   E   D   C   B   A    1  pTEF1-1 [--------CB008DB-1------] 2  pTEF1-2 [--------CB008DB-2------] 3  pTEF1-3 [--------CB008DB-3------] 4  M7-1    [--------ASG7-1---------] 5  M7-2    [--------ASG7-2---------] 6  M7-3    [--------ASG7-3---------] 7          [--------YDR124W-1------] 8          [--------YDR124W-2------] 9          [--------YDR124W-3------] 10         [--------PRM6-1---------] 11         [--------PRM6-2---------] 12         [--------PRM6-3---------]
Plate 2:
H   G   F   E   D   C   B   A    1       [--------AGA1-1---------] 2       [--------AGA1-2---------] 3       [--------AGA1-3---------] 4       [--------CLG1-1---------] did not grow as much as the others 5       [--------CLG1-2---------] 20ul instead of 10ul for CLG1 6       [--------CLG1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]
Plate 3:
H    G   F   E   D   C   B   A    1        [--------ECM18-1--------] 2        [--------ECM18-2--------] 3        [--------ECM18-3--------] 4        [--------SAG1-1---------] 5        [--------SAG1-2---------] 6        [--------SAG1-3---------] 7         8         there is no PRM3 made. They're in glycerol stock. 9         10       [--------PRM1-1---------] 11       [--------PRM1-2---------] 12       [--------PRM1-3---------]
Weds 14/07/23
Came up with more cards for Cards against humanity_UCSF iGEM
Working on the CRISPR paper, discussion.  Research for what has been done with CRISPR delivery and how our work is valuable.
Thurs 14/07/24
Working on the CRISPR paper and the Explo Powerpoint.
Made cultures for glycerol stocks of 8 things:
1. CB008 +pTEF1+rtTA +pTET+GFP +pTET+mfalpha 2. CB008 +m6+rtTA +pTET+GFP +pTET+mfalpha 3. CB008 +m7+rtTA +pTET+GFP +pTET+mfalpha 4. CB008 +m10+rtTA +pTET+GFP +pTET+mfalpha 5. CB008DB +m6+rtTA +pTET+GFP +pTET+mfalpha 6. CB008DB +m7+rtTA +pTET+GFP +pTET+mfalpha 7. CB008DB +pTEF1+rtTA +pTET+GFP 8. CB008DB +m10+rtTA +pTET+GFP
7 & 8 need +pTET+mfalpha
In the coming days we will also transfrom everything with pAGA+mCherry.
Fri 14/07/25
Stuff to do other than lab:
CRISPR paper. Read, Write
Powerpoint for Explo due today
Research for examples relating to the project
Cards Against Humanity: UCSF_UCB iGEM 2014 (illustrator/InDesign)
Make glycerol stocks of:
1:50 dilutions of each and grow for 3 hours in 5ml cultures to .4 ~ .5 OD600.  Then follow glycerol stock protocol.
9am -> 12pm growth to log phase
1. CB008 +pTEF1+rtTA +pTET+GFP +pTET+mfalpha 2. CB008 +m6+rtTA +pTET+GFP +pTET+mfalpha 3. CB008 +m7+rtTA +pTET+GFP +pTET+mfalpha 4. CB008 +m10+rtTA +pTET+GFP +pTET+mfalpha 5. CB008DB +m6+rtTA +pTET+GFP +pTET+mfalpha 6. CB008DB +m7+rtTA +pTET+GFP +pTET+mfalpha 7. CB008DB +pTEF1+rtTA +pTET+GFP 8. CB008DB +m10+rtTA +pTET+GFP
1:20 dilutions for transformations after 3 hours growth in 5ml cultures.  Transforming pTET+mfalpha into #7,8 and pAGA+mCherry into #1-6.
9am -> 12pm growth to log phase
1. CB008 +pTEF1+rtTA +pTET+GFP +pTET+mfalpha 2. CB008 +m6+rtTA +pTET+GFP +pTET+mfalpha 3. CB008 +m7+rtTA +pTET+GFP +pTET+mfalpha 4. CB008 +m10+rtTA +pTET+GFP +pTET+mfalpha 5. CB008DB +m6+rtTA +pTET+GFP +pTET+mfalpha 6. CB008DB +m7+rtTA +pTET+GFP +pTET+mfalpha 7. CB008DB +pTEF1+rtTA +pTET+GFP 8. CB008DB +m10+rtTA +pTET+GFP
~23 primers are coming in to clone AFRP upstream of BAR1 and Ste2 separately.
Note:  1. PCL2+mCherry in HY130 backbone is being constructed by Robert.  We will probably make additional strains with this in place of pAGA+mCherry. 2. Jeffrey is working on getting m3+rtTA into CB008DB +pTET+GFP.
Mon 14/07/28
Last friday, Eleanor transformed mCherry into 6 strains with Constitutive Promoter, GFP, Alpha factor.  Today I am colony PCRing them along with 3 other strains that have other things being integrated: 1) +rtTA for the m3s 2) +mfalpha for DB pTEF1 & m10.
Next steps: 1. Need to retransform DB m10 and CB008 m7 because they have no colonies. 2. Need to make glycerol stocks of the ones that worked for Colony PCR 3. For tomorrow, make cultures for tomorrow's flow (9 strains + control)
Auxotrophic marker Colony PCR primer names: His: FW RA145(#87) & REV RA146(#89) Trp: FW RA145(#87) & REV
Today I also worked on the Cards Against iGEM inDesign layout.
Tues 14/07/29
Flow scheduled for 5-8pm. Doing Consti+rtTA+pTET+GFP Dox run. No alpha, no RFP. Runnin all 5 DB strains today (and 4-5 CB008 strains tomorrow pending successful colony PCR of CB008 m3.)
Dilutions start 7:16am -> 10:16am Induction 10:45 -> 4:45pm
Dox concentrations are the same as last time.
All plates identical assortment:
Plate 1:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------] C [---------------[CB008DB pTET_GFP pTEF1]-----] D [---------------[CB008DB pTET_GFP pTEF1 m3]--] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G [---------------[CB008DB pTET_GFP pTEF1 m10]-] H
Plate 2:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------]   C [---------------[CB008DB pTET_GFP pTEF1]-----]  D [---------------[CB008DB pTET_GFP pTEF1 m3]--] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G [---------------[CB008DB pTET_GFP pTEF1 m10]-] H
Plate 3:
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------]   C [---------------[CB008DB pTET_GFP pTEF1]-----] entire lane 10 is induced D [---------------[CB008DB pTET_GFP pTEF1 m3]--] with wrong concentration. E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G [---------------[CB008DB pTET_GFP pTEF1 m10]-] H
time course flow allows us to create a stocastic model of our products: rtTA, GFP, alpha-factor, RFP.
Colony PCR2 redo from yesterday because only 1 worked
Constitutive Promoter Constructs progress:
Today George and I are retransforming:
CB008DB+pTET+GFP+m3+rtTA[+pTET+mfalpha]
CB008+pTET+GFP+m7+rtTA+pTET+mfalpha[+pAGA+mCherry]
CB008+pTET+GFP+m10+rtTA+pTET+mfalpha[+pAGA+mCherry]
The salmon sperm boiled down to about 5ul for all 3 transformations. It also seemed a little viscous.
CB008+pTET+GFP+m10+rtTA+pTET+mfalpha[+pAGA+mCherry] already exists so our transformation of this (#3) was pointless.  What we meant to do was CB008DB+m10+rtTA+pTET+GFP[+pTET+mfalpha]
After all our colony PCRs for the day, this is the update of the excell progress sheet:
Made cultures of all 10 things on the Consti list. 9 for transformations and 1 for glycerol stock.
Made cultures for tomorrows flow.
Weds 14/07/30
None of the cultures for today's flow grew. Only the controls grew. Looking back at the URA knock out patch plates that we inoculated from, they didn't grow very well. I think they moreso had a hard time growing than it not being correctly transformed because they came back positive in the colony PCRs (14/7/15). I transferred them onto a YPD patch plate to see if they would grow.
Derrick and I plated on YPD CB008/DB strains with pTET+GFP+consti+rtTA because the current patch plate on URA of CB008s does not have good growth.  Cultures made from this patch plate did not grow either so holding out flow back.
Jeffrey and George are retransforming the ones that failed.
Thurs 14/07/31
Our steak plates of CB008/DB +pTET+GFP+Consti+rtTA over grew only on CB008DB.  All the CB008s did not grow and have no sign of growth.  This may mean that we will have to retransform consti+rtTA into our CB008 strains.  So far, our glycerol stocks have not been working well for us; we don't know what the problem is.
Worked on the Explo powerpoint today.
Fri 14/08/01
Made streak plates with Derrick of everything we have so that we have fresh/organized cultures.
Worked on Cards Against iGEM InDesign. Printed and cut out our first deck for testing.
Mon - Weds 14/08/04-06
Out on vacation.
Thurs 14/08/07
Tomorrow we are running flow. Testing the final circuit with 5 strains. triplicate, 5 Dox concentrations (Shaved off 7 of the top concentrations of Dox), 4 time points.
Concentrations:  0 ug/ml 0.03 ug/ml 0.06 ug/ml 0.09 ug/ml 0.6 ug/ml
Exploratorium After Dark event 6pm-10pm
Fri 14/08/08
OD600 of .1
Flow today with CB008 pTEF1+rtTA pTET+GFP pTET+MFalpha pAGA1+mCherry. Signed up from 4:30pm -> 9pm
Diluted 1:200 at 7:45am and grown for 2 hours
Time points with [Dox] induction: 0, 1.5hr, 3hr, 5hr
Plate map for all time points (4 plates):
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 m6-1]----]       [CB008 m10-3]----] B [CB008 m6-2]----]       [CB008DB m7-1]---] C [CB008 m6-3]----]       [CB008DB m7-2]---] D [CB008 m7-1]----]       [CB008DB m7-3]---] E [CB008 m7-2]----]  F [CB008 m7-3]----]            G [CB008 m10-1]---]  H [CB008 m10-2]---]
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12
Flow notes: The first plate for time 0 contains the concentrations of doxycycline so we saw a bimodal RFP response.  An explanation for this can be that the addition of doxycline had driven expression of mfalpha -> pAGA+mcherry before the cyclohexamide fixed the cells.
Finished flow at 9:40pm
Mon 14/08/11
Group Meeting at 10am -> 12pm
Flow with AFRP+rtTA +pTET+GFP 7 strains (Mon-Tues Flow), 7 alpha concentrations. 7 AFRP promoters+rtTA sucessfully transformed into CB008.
The AFRP strains that have successfully been transformed (7):
CB008 +CLG1+rtTA +pTET+GFP CB008 +YDR124W+rtTA +pTET+GFP CB008 +ASG7+rtTA +pTET+GFP CB008 +HYM1+rtTA +pTET+GFP CB008 +PRM1+rtTA +pTET+GFP CB008 +PRM2+rtTA +pTET+GFP CB008 +PRM3+rtTA +pTET+GFP
HYM1 plate, 0.6ug/ml & 0.06ug/ml concentrations were switched.
1:200 dilution grown for 2hrs 8:00am -> 10:00am induced at 10:30am Fix at 3:30pm Flow signed up from 4:30pm -> 9:00pm Finished flow at 8:40pm
Doxycycline concentrations plate map (12):
H   G   F   E   D   C   B   A    1          [--------0 µg/ml----------] 2          [--------0.03 µg/ml-------] 3          [--------0.06 µg/ml-------] 4          [--------0.09 µg/ml-------] 5          [--------0.3 µg/ml--------] 6          [--------0.6 µg/ml--------] 7          [--------0.9 µg/ml--------] 8          [--------3.0 µg/ml--------] 9 [--------6.0 µg/ml--------] 10         [--------9.0 µg/ml--------] 11         [--------30 µg/ml---------] 12         [--------60 µg/ml---------]
Alpha-Factor Concentrations (7):
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}------------] B   [-------------------{0.5 nM-----------] C   [-------------------{1 nM}------------] D   [-------------------{10 nM}-----------] E   [-------------------{100 nM}----------] F   [-------------------{1000 nM}---------] G   [-------------------{3000 nM}---------] H
Plates:
Plate #1 (ASG7) Plate #2 (CLG1) Plate #3 (HYM1) Plate #4 (YDR124W)
Play tested the printed Cards Against iGEM. Eliminated about 21 black cards. We have 24 good black cards, Goal 40-50.µ
Tues 14/08/12
Flow with the other 3 CB008 +AFRP+rtTA +pTET+GFP (PRM1, PRM2, PRM3) constructs. Signed up from 4pm -> 8pm.
1:200 dilution at 8:11am -> 10:11am
Same Dox and alpha-factor concentrations as yesterday.
Plate#1 (PRM1) Plate#2 (PRM2) Plate#3 (PRM3)
Helped Jessica with her progress with AFRP+rtTA constructs. Streaked out the 5 successful transformations of pTET_mfalpha for the CB008 (12-Aug) and made cultures of those for tranformations (+pAGA+mcherry/+pPCL2+mcherry/+pPCL2+BFP).
AFRP+rtTA progress chart:
Weds 14/08/13
1:200 dilutions, 2hr growth 7:50am -> 9:50am
Time points with [Dox] induction: 0, 1.5hr, 3hr, 5hr
Making Dox dilutions, follow this:
Stock: 100mg/mL  **1x**           **100x**          Prep 60µg/mL       A: 6mg/mL            (6:100) 60µL of Stock in 940µL of water 30µg/mL       B: 3mg/mL            30µL of Stock in 970µL of water 9µg/mL        C: 900µg/mL          150µL of A in 850µL of water 6µg/mL        D: 600µg/mL          100µL of A in 900µL of water 3µg/mL        E: 300µg/mL          50µL of A in 950µL of water      Can also do 100ul of B in 900ul of water 0.9µg/mL      F: 90µg/mL           100µL of C in 900µL of water 0.6µg/mL      G: 60µg/mL           100µL of D in 900µL of water 0.3µg/mL      H: 30µg/mL           100µL of E in 900µL of water 0.09µg/mL     I: 9µg/mL            100µL of F in 900µL of water 0.06µg/mL     J: 6µg/mL            100µL of G in 900µL of water 0.03µg/mL     K: 3µg/mL            100µL of H in 900µL of water
Plate map for all time points (4 plates):
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 pTEF1-1]-]       [CB008DB m3-3]---]  B [CB008 pTEF1-2]-]       [CB008DB m6-1]---] C [CB008 pTEF1-3]-]       [CB008DB m6-2]---] D [CB008DB pTEF1-1]       [CB008DB m6-3]---] E [CB008DB pTEF1-2]   F [CB008DB pTEF1-3]            G [CB008DB m3-1]--]  H [CB008DB m3-2]--]
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7       [--------0 µg/ml------------] 8       [--------0.03 µg/ml---------] 9       [--------0.06 µg/ml---------] 10      [--------0.09 µg/ml---------] 11      [--------0.6 µg/ml----------] 12
Deleted and exported files on the flow to prevent data crowding. (All files exported onto iGEM2 usb) Note: 140627 Alpha Factor Responsive Promoter 2 of 2 gives an error when exporting. "There are no tubes with data to export". Data may have been deleted? idk
Thurs 14/08/14
1:200 dilutions at 7:50 -> 9:50am
Time points with [Dox] induction: 0, 1.5hr, 3hr, 5hr, 8hr
Plate map for all time points (5 plates):
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 pTEF1-1]-]       [CB008 m7-3]-----] B [CB008 pTEF1-2]-]       [CB008 m10-1]----] C [CB008 pTEF1-3]-]       [CB008 m10-2]----] D [CB008 m6-1]----]       [CB008 m10-3]----] E [CB008 m6-2]----]  F [CB008 m6-3]----]            G [CB008 m7-1]----]  H [CB008 m7-2]----]
[Dox] Concentration Map:
H   G   F   E   D   C   B   A    1       [--------0 µg/ml------------] 2       [--------0.03 µg/ml---------] 3       [--------0.06 µg/ml---------] 4       [--------0.09 µg/ml---------] 5       [--------0.6 µg/ml----------] 6  7                       [--0 µg/ml--] 8                       [-0.03 µg/ml] 9                       [-0.06 µg/ml] 10                      [-0.09 µg/ml] 11                      [-0.6 µg/ml-] 12
In the 2ml culture plate, we noticed what seems to look like debri or contamination on the bottom of the wells from rows A,B,C.  We tried to make the time point plates by pipetting everything singly.
After taking a look at the 0 time point plate through the flow, we can see that the size of molecules running through are not yeast but bacterial contamination.  We need to totally redo this experiment again because this contamination gives us inaccurate data.
Yesterday, the 0 time point plate was not run all the way (only 10ul was run and there were not enough events).  Derrick saved the plate and I will run 100uls of each of those wells. The wells are G,H (1-5), and A,B,C,D (7-11).
2pm -> 4pm Practiced presenting poster with the group and Anusuya.
Fri 14/08/15
Last official day of iGEM
All day event.  Driving down to Santa Cruz for the Symposium held by UC Santa Cruz.
Sun 14/08/24
Made cultures of Contsti+full contruct + pPCL2+mCherry for Monday's flow (4).
pTEF1, m6, m7, m10
Robert's Lab Notebook
WELCOME TO THIS AWESOME MESSY NOTEBOOK
6/9/14 (BootCamp)
PCR for Constitutive Promoters
PCR Reaction:  1 Reaction                        4.5X Master Mix    10 ul   5X Phusion Buffer         45 ul  1 ul    dNTP's (10uM)             4.5 ul 2.5 ul  Foward Primer (10uM)      11.25 ul 2.5 ul  Reverse Primer (10uM)     11.25 ul .3 ul   Template DNA              1.35 ul .5 ul   Phusion DNA Polymerase    2.25 ul 332 ul  Water                     150 ul
Steps
1) Mix Reagents in a 4.5X Master Mix on ice
2) Pipette 50ul of Master Mix into 4 labeled PCR tubes
3) Put into thermocycler for following cycle:
Initial Denaturation       98ºC          30s     35 cycles of          Denaturation       98ºC          10s         Annealing          55ºC          20s         Extension          72ºC          30s Final Extension            72ºC          5 min Hold                       4ºC           FOREVERRRRRR
4) Keep samples for gel extraction
*Forgot to keep 4.5X Master Mix tube on ice *I also made a 1/10ul dilution of each primer when I was supposed to make a 10/100ul dilution
6/10/14
Loading PCR'ed Constitutive Promoters on Gel
50 ul          PCR Tube 5 ul           10X Blue Juice Dye
Steps
1) Make a Gel with Cybr Safe
2) Add Blue Juice into PCR tube
3) Load and run gel at 100 volts for 30 min
Gel Photo PCR'ed Constitutive Promoters
Constitutive Promoters pTEF1, m3, m6, m7, m10
6/10/14
Gel Extraction Kit
Steps
1) Cut out DNA from Gel
2) Weigh the slice of gel and and 3X volume of QG Buffer to 1X volume gel.  For >2% gels add 6X Buffer.
3) Incubate at 50ºC or 10 mins.  Vortex every 2-3 mins.
4) Solution should be yellow.  If it is orange or violet add 10ul 3M sodium accetate pH 5 and mix
5) Add 1X gel volume of isoproponal and mix
6) Get a QIAquick spin column (Purple column with cap)
7) Put sample into column and centrifuge for 1 min.  Discard flow-through and place column back into tube.  for samples >800ul, re-spin the liquid.
8) Add 500ul QG Buffer and spin for 1 min.  Discard flow-through and put column back into tube.
9) Add 750ul PE Buffer and spin for 1 min.  Discard flow-through and put column back into tube.
10) Do a dry spin for 1 min.
11) Place column into a new 1.5ml microcentrifuge tube.
12) Elute DNA in 50ul (or whatever).
6/11/14
Restricton Enzyme Digest with ApaI
Steps
1) Put all the following into a tube:
40ul       DNA 5ul        Cutsmart Buffer 0.5ul      ApaI
2) Let it digest overnight at room temperature
6/12/14
Restriction Digest with XhoI
Steps
1) Add 0.5ul XhoI
2) Incubate in 37ºC in shaker
3) Run a PCR purification
*DNA concentration: 153ng/ul
DNA Ligation
1 ul            10X Ligase Buffer  0.2 ul          DNA backbone 0.2 ul          Insert 0.5 ul          T4 DNA Ligase 8.1 ul          Water
10ul total
-Incubate at room temperature for 2 hours
Transformation
10 ul    Ligation 50 ul    E.Coli compitant cells
Steps
1) 30 mins on ice
2) 42ºC Heat shock
3) Ice for 2 mins
4) Add 250 ul SOC media and shake/sit at 37ºC for an hour
5) Plate
Re-do Ligation (This time use 0.4ul backbone)
-Also use better competant cells
Re-do Transformation (above)
6/13/14
Yeast Transformation
Steps
Refer to protocal in my binder
6/16/14
Colony PCR for E.Coli
Steps
1) Pick a single colony using a pipette tip and mix in 25ul water in a PCR tube.  Do ths for 4-6 colonies on each plate.  Save rest of the tube just incase.
2) Set up PCR reaction as below:
**1X Reaction**      **6X Reaction** 2X GoTaq Green PCR Master Mix             10ul                 60ul 10uM FW Primer                             1ul                  6ul 10uM RV Primer                             1ul                  6ul Water                                      3ul                 18ul   Bacterial cells (Template)                 5ul                 ___ul
*Add 15ul of the 6X Reaction to each tube
Cycles
95           5min 30X: 95      45s      55      30s      72      1min per kb 72           10min
3) Anaylze products on a gel.  GoTaq has loading gel in it already.  Load 5ul of PCR reaction.
4) For all positive bands, take the rest o the bacterial cells from step 1 and grow them in LB overnight (+antibiotic) for miniprep next day.
6/16/14
Send DNA to be sequenced
-They came back good :)
Colony PCR for Yeast & Patching (Refer to binder for protocol)
Steps
1) Pick 3 colonies and number them
2) Patch colonies onto a plate
3) Mix Yeast cells in 10ul NaOH in PCR tube
4) Boil for 10 min at 95ºC
5) PCR:
95ºC         5min  30X:     95ºC     45s     55ºC     30s     72ºC     1min 72ºC         10min 4ºC          FOREVERRRRRRR
Gels
SAG1     ECM18    PRM3   HYM1     ASG7      CLO7    PRM1
1KB
1  2  3
1  2  3
1  2  3
1  2  3
1  2  3
1  2  3
1
----
---------
----------
---------
---------
--------
---------
---
1KB
2  3
1  2  3
1  2  3
1  2  3
1  2  3
----
-----
------
---------
---------
---------
---------
6/18/14
MiniPrep Constitutive Promoters (Refer to binder for protocol)
Made CBOO8 & CBOO8DB stock dilutions
Glycerol Stocks of CBOO8 & CBOO8DB (Refer to binder for protocol)
6/19/14
Flow Cytometry
Night Before:
1) Start overnight cultures using SD complete media
Day of:
1) Dilute cultures to final ~OD (Optimal Density) in the 96 well shaker plate (overnight cultures should be around ~OD7, so about a 1:100 dilution). Use SD media to make each well a total volume of 1mL
CBOO8
1   2   3   4   5   6   7   8   9   10  11  12 A   [------NEG------]       [------NEG-------] B   [------PRM2-----]       [------PRM1------] C   [------ASG7-----]       [------EMC18-----] D   [------PCL2-----]       [------PRM3------] E   [------NEG------]       [------SAG1------] F   [------CLG1-----] G   [------YDR124W--] H   [------PRM6-----]
in DB008
1   2   3   4   5   6   7   8   9   10  11  12 A   [------NEG------]       [------NEG-------] B   [------yGEM10---]       [------yGEM14----] C   [------yGEM4----]       [------yGEM15----] D   [------yGEM5----]       [------yGEM16----] E   [------NEG------]       [------yGEM6-----] F   [------yGEM11---] G   [------yGEM12---] H   [------yGEM13---]
2) Allow cells to grow on plate shaker for 3 hours at 1000 rpm
3) Induce with Alpha Factor.  The Alpha Factor in the freezer is at 3mM and we're using concentrations at 0.1nm, 1nm, 10nm, 100nm, and 1uM
-Concentrations
3mM --> 30ul + 870ul water
⍒
100uM --> 50ul + 450ul water
⍒
10um --> 50ul + 450ul water
⍒
1000nm 50ul + 450ul water
⍒
100nm
*Alpha Factor cannot be refrozen so dump after use
4) Allow induction to proceed on plate shacker for 1.5 - 2 hours
5) Transfer 250 ul of each culture into a 96 V-Well
6) RUn on flow cytometer
-Checklist ✓:
Check status of machine (should be on standyby) Check fluid boxes and stroage tanks Do not run plate. Run well. Have a positive & negative control Open iGEM account (pw: biobricks) Make new FACS experiment
-Voltage:
CHeck inspector to see if everything is on  FCS: 250 SSC: 280 FITC: 550  Sample Flow Rate: 1 Sample Volume: 200 Mixing Volume: 100 Mixing Speed: 180 Number of Mixes: 3 Wash Volume: 400
-Wells
Select wells to run (make sure volume is 50ul below actual amount) and there will be a side bar of options on how to adjust your gels (mixing, flow speed) Run well & save data onto USB *Don't turn on wifi
7) Analyze Data
6/23/14
Colony PCR (refer to 6/16/14)
Gel result of Yeast Colony PCR:
MON 19 MAY 2014
Cleaned up lab. Read lots of literature.
TUE 20 MAY 2014
Looking for alpha-factor responsive promoters. Results can be found in "iGEM 2014/Parts and Plasmids/Mating Responsive Promoters/"
WED 21 MAY 2014
Continuing search for more alpha-factor responsive promoters to add to the 16 that we have found so far.
Also, designing primers for the promoters that we have found so far. For primer design, see "iGEM 2014/Protocols/cloning_DH5aT.docx"
MON 02 JUN 2014
I. Calculated Ligation concentrations found in 140602 Ligation Calculation.xlsx and performed ligation reactions.
II. Also re-PCR'd promoters that did not work before.  Please see paper notebook page 3 for gel map. Gel image:
TUE 03 JUN 2014
I. Re-PCR'd promoters that we did not do yesterday. See gel photos:
Also gel extracted.
II. Selected colonies and inoculated cultures.
WED 04 JUN 2014
I. Digest PCR reactions and PCR cleanup. Completed.  Concentration of Promoter 6 is pretty bad compared to others.
II. Inoculated single colony plates.
THURS 05 JUN 2014
I. Eric and Derrick are retrying digestion of backbone for the third time after two failed attempts.  Will try ligation and transformation into e. coli today.
II. Need to miniprep plates with single colonies.
III. Linearize DNA for Yeast Transformation Linearized PRM1, SAG1, YDR124W. See concentrations here:
FRI 06 JUN 2014
I. Designed Gibson Primers and produced Promoter + rtTA ApE files.
II. Performed gel extraction of re-digested backbone.  See gel photo below:
.
See how the band is at the right place now at around 8kb, but that there isn't really a smaller band. Possible that the smaller band ran off the gel.
Overall:  Found that most of the promoters worked out in the end with cloning. Ready for yeast transformation and integration into yeast genome. Now trying to figure out next steps.
MON 09 JUN 2014
I. iGEM Bootcamp started.
II. Minipreps of Transformations from attempt 2.  Vacuum manifold does not clear PE Buffer as well as centrifuges. May have ended with more than 50microliters of product. Nanodrop showed that our concentrations are okay still. We are now sending out these samples for sequencing.
TUE 10 JUN 2014
I. iGEM Bootcamp.
II. Dual Digest of miniprepped backbones to remove GFP from built vectors → will digest overnight.
III. Worked on iGEM website.
IV. Sequencing came back and found that all miniprepped vectors turned out well → moving forward with yeast transformation and Dual digest for rtTA incorporation as seen in II.
Lecture on E. Coli and Yeast by Zairan Liu from UCSF.
WED 11 JUN 2014
I. CIP treatment of backbone Lecture on Flow Cytometry by
Project introduction lecture - Kara.
Cell-cell communication lecture - Leo
Planned:  II. Gibson reaction III. Transformation into Bacteria
Brainstorm:  Possible human practices: bands! Playing music without a conductor! Collective structures. The role of more effective communication, or communication training in establishing structures.
Possibility of not just excreting more Alpha factor, but increasing number of receptors.
THURS 13 JUN 2014
I. No colonies on plate. Have to re-gibson and re-transform.  Lots of lectures
FRI 13 JUN 2014
I. Linearization of Promoter + GFP plasmids for yeast transformation  II. Yeast transformation today, transformed 11 promoters + GFP into two different yeast strains.  III. Field trip to NASA today. Meet up with Brown/Stanford/Spielgal iGEM team!
Questions:  What is the difference between the two yeast strains that we transformed?
MON 16 JUN 2014
I. Colony PCR of E. Coli Transformations of Promoter + rtTA (9) II. Colony PCR of Yeast Transformations of Promoter + GFP (11) III. Over 100 lanes of gels to run today! Woooooooo!
IV. Inoculated Successful Colony PCRs.
Results of E Coli Colony PCR of rtTA + promoter
TUE 17 JUN 2014
I. Redo Yeast Colony PCRs that failed yesterday.  II. Miniprep of E. Coli rtTA + promoters inoculated yesterday.
CB008 genotype: W303 MATa far1Δ his3 trp1 leu2 ura3
CB008DB genotype: W303 MATa far1Δ his3 trp1 leu2 ura3 bar1
Results of Yeast Colony PCR of GFP + Promoter in CB008:
WED 18 JUN 2014
I. Linearize promoter + rtTA and transform into Yeast II. Inoculate and incubate promoter + GFP transformed yeast. Prepare patch plate with promoters that worked.  III. Lincoln: miniprep constitutive + GFP vectors and send for sequencing.  IV. Overnight cultures of alpha promoters CB008
THUR 19 JUN 2014
I. Dilute inoculated cultures. (2 hours)
Flow Cytometer Preparation
Day Before: start overnight cultures of your strains to be tested in SD complete media (NOT YPD, since it has fluorescence background and is problematic for FACS testing)
Day Of: 1. Dilute overnight cultures to final ~OD 0.05-0.1 in the 96 well shaker plate (the saturated overnight cultures should be ~OD 7, so this is approximately a 1:100 dilution. Also, where are the plates? If you don't know, ask and make a note.). Again, use SD complete media and the total volume of each well should be 1 mL. Make a plate map!
Allow cells to enter growth stage by putting on plate shaker for 3 hours at 1000 rpm, 30 degrees C.
Induce with alpha factor. The stock alpha factor in the freezer is 3 mM, and we used it at final concentrations of 0, 1 nM, 10 nM, 100 nM, and 1 uM. **Please include your concentrations for making the dilutions we used to pipette so that you can quickly and easily refer to this for next time!** Alpha factor cannot be refrozen, so throw away stocks after use.
Allow induction to proceed on plate shaker for 90 min to 2 hours, no longer.
Transfer 250 uL of each culture to the 96 well flow cytometry V-bottom plate using the multichannel pipette. Add 4µl of cyclohexamide for every 100µl of cells (10µl total) to kill cells and stop protein production.
Run on flow cytometer. Things you should note:
Check waste, turn machine on, check sheathing fluid.
Open FACS diva. Use iGEM account.
Make new plate or new experiment.
Make sure settings for each well is accurate. Ensure that pick up volume is not more than volume in each well, that the mixing volume is not more than volume in each well.
Make sure laser settings, voltage settings are correct. (View>inspector)
FSC: 250 SSD: 280 FITC: 550 B: 650
Click blue button to tag selected wells as wells with samples to test.
Flow rate: 1µL/sec Sample Volume: 200µL Mixing Volume: 100µL Mixing speed: 180µL/sec
Always run wells, not run plate.
FIRST what things you need to check on the machine before running!, how to open the iGEM account and create a new FACS experiment, voltage settings (and tips on how to set an appropriate voltage setting), how to select wells and alter the volume of sample, flow speed, mixing volume, etc, how to run the plate/wells, how to export data, and anything else you might need)
Analyze data. Notes on that from what you learned in FlowJo and MatLab today.
II. Expose to alpha factor (~90 minutes) (0, 10nM, 1µM, 10µM, 100µM)
Correction 24 JUNE 2014: (0, 1nM, 10nM, 100nM, 1µM)
Flow Cytometer 1-4pm.
How to use flow cytometer:  1. Check the sheathing fluid levels! 2. Press run  3. On computer, use flowjo.  4. Check voltages! (in notebook) 5. Select wells 6. run! 7. Make sure you see a good number of events.
PRELIM Plate map:
1   2   3   4   5   6   7   8   9   10  11  12 A   [------NEG------]       [------NEG-------] B   [------PRM2-----]       [------PRM1------] C   [------ASG7-----]       [------EMC18-----] D   [------PCL2-----]       [------PRM3------] E   [------NEG------]       [------SAG1------] F   [------CLG1-----] G   [------YDR124W--] H   [------PRM6-----]
in DB008
1   2   3   4   5   6   7   8   9   10  11  12 A   [------NEG------]       [------NEG-------] B   [------yGEM10---]       [------yGEM14----] C   [------yGEM4----]       [------yGEM15----] D   [------yGEM5----]       [------yGEM16----] E   [------NEG------]       [------yGEM6-----] F   [------yGEM11---] G   [------yGEM12---] H   [------yGEM13---]
I will be working with YDR124W and PRM6.
III. Prepare glycerol stocks of DB promoter + GFP yeast strains.  IV. Pick and inoculate cultures with promoter + rtTA again since most of the sequencing failed. Will need to re-linearize most of the yeast.
FRI 20 JUN 2014
I. Leaving early today for Doctor's appointment.  II. Minipreps for rtTA+promoter.
MON 23 JUN 2014
I. Found that the rtTA that we cloned were incorrect. Did not including activating domain.  II. Inoculating HYM1 DH5alpha + GFP again for more plasmid.
Now working on Flow Cytometry data analysis of alpha responsive promoters.  Now working on pTET/GFP/mFalpha. p2A?
TUE 24 JUN 2014
I. Literature search on feasibility of P2A incorportation of alpha factor and other sequences.  II. FlowJo notes:
Import data. Save in save folder on desktop.  Double click to open a sample.  Gate the sample population.  Activate gate: gives a percentage of samples in gate.   Click on sample list and click on gate name for analysis on just gated region.   Save workspaces so that you can continue analysis later on.  Displaying mean: click on statistics and click ∑ for which stats that you want.   Look through individual samples for obscurities.   Matlab: comment with what workspace your data comes from.   Enter in data by hand.
Analysis of Promoters+GFP in different alpha factor concentrations:
Translation:  Name    Promoter    PREs yGEM4:  ASG7        2 yGEM5:  PCL2        5 yGEM6:  SAG1        4 yGEM10: PRM2        2 yGEM11: CLG1        2 yGEM12: YDR124W     2 yGEM13: PRM6        2 yGEM14: PRM1        2 yGEM15: ECM18       1 yGEM16: PRM3        3
Continue to learn canvas!  Game brainstorming: recording how many rounds someone plays a game depending on how much on succeeds. Simulates positive feedback in gameplay. Building of circuits of games.
WED 25 JUN 2014
I. Dilution of CB008 and CB008DB for yeast transformation of pTETGFP today @ ~1pm.      Need to linearize pTETGFP for transformation later today.
Given two vectors:  hy86E3(pTETGFP LEU2)                     pTS97(pAGA1GFP URA3)
linearization @ 37°C started at 11:57am → 12:57pm
No need to PCR cleanup
Straight to transformation.
Yeast Transformation Protocol
T-(1 day) Grow yeast strains in 5-10ml YPD overnight at 30°C
T-(2–4hours) Dilute cultures ~1:20 in YPD.  Grow for 2-4 hours @ 30°C to OD_600 = 0.4-0.6. Make sure to inoculate 2.5ml per transformation reaction.
T-(20 minutes) Boil salmon sperm DNA for 10 minutes and cool on ice for 10 minutes. (This can be done in the thermocycler with the ssDNA Heat-Freeze protocol) Thaw DMSO! Pellet cells (3000rpm 2-5min). Discard supernatant.  Wash with 1:1 culture volume of LOATE(0.1M LiOAc in TE) or water.  Pellet cells (3000rpm 2-5min). Discard supernatant.  Resuspend pellet in 100µl per reaction (aka per 2.5mL of culture volume)  Aliquot 100µl into each transformation reaction tube.  To each tube of 100µL yeast, add:
10µL of ssDNA
8µL of target DNA(Hyun's standard amount after linearization)
480µL of 50% PEG 3350
60µL TE(10XTE)
60µL of LOA(1M LiOAc)
75µL of DMSO
Vortex! Incubate at 42°C for 30 minutes.
T+(30 minutes) Pellet (6000rpm for 2 minutes). Use pipette to remove media! Resuspend in 500µL YPD (or selective media) Pellet (6000rpm for 2 minutes). Discard media.  Resuspend in residual ~50µL of YPD.  Plate on selective media.  Incubate for 1-3 days at 30°C.
T+(1-3 days) Pick colonies, colony PCR, or continue with experiment in some other way.
Incubation at 42°C started at 1:56pm → 2:26pm.
II. Until then, still learning more javascript and canvas.
III. Inoculate yeast strains for flow cytometry tomorrow. Do in triplicates tomorrow! Measuring the following promoters+GFPs in triplicate tomorrow (start @ 8am):
PRM2
ASG7
PCL2
CLG1
IV. Design gibson primers for constitutive promoters + rtTA
THURS 26 JUN 2014
Alpha factor:  3mM starting concentration. 30µL in them.  0, 0.5nM, 1nM, 10nM, 100nM, 1000nM, 3000nM
Make 100X stocks of each concentration and aliquot 10µL into each well.
ROUND 2 Promoter Map:
Plate 1:     H   G   F   E   D   C   B   A    1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------PRM2-1---------] 5       [--------PRM2-2---------] 6       [--------PRM2-3---------] 7       [--------ASG7-1---------] 8       [--------ASG7-2---------] 9       [--------ASG7-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------] Plate 2:     H    G   F   E   D   C   B   A    1       [--------CLG1-1---------] 2       [--------CLG1-2---------] 3       [--------CLG1-3---------] 4 5 6 7 8 9 10 11 12
Input into 30°C + 1000rpm shaker @ 8:14am -> 11:14am
Plate 1 & 2 Alpha Factor Concentration Map:
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H
Induce with alpha factor for 90 minutes.
Make 100X stocks:  0nM 50nM 100nM 1000nM 10000nM 100000nM 300000nM
3000000nM -> 30000nM
Added alpha factor 11:45am -> 1:15pm
II. Second round of alpha factor promoter + GFP characterization. Now characterizing:
YDR124W
PRM6
PRM1
ECM18
PRM3
SAG1
[Plate maps updated in FRI Jun 27 below]~~PLATE 1 PROMOTER MAP
H   G   F   E   D   C   B   A    1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------YDR124W-1------] 5       [--------YDR124W-2------] 6       [--------YDR124W-3------] 7       [--------PRM6-1---------] 8       [--------PRM6-2---------] 9       [--------PRM6-3---------] 10      [--------PRM1-1---------] 11      [--------PRM1-2---------] 12      [--------PRM1-3---------]
PLATE 2 PROMOTER MAP
H    G   F   E   D   C   B   A    1       [--------ECM18-1--------] 2       [--------ECM18-2--------] 3       [--------ECM18-3--------] 4       [--------PRM3-1---------] 5       [--------PRM3-2---------] 6       [--------PRM3-3---------] 7       [--------SAG1-1---------] 8       [--------SAG1-2---------] 9       [--------SAG1-3---------] 10 11 12~~
PLATE 1 & 2 ALPHA FACTOR CONCENTRATION MAP: same as before
III. Dilution of FW pTEF1 + rtTA primers     1. Resuspend in 10µL * amount ng of DNA.      2. 1/10 dilution of stock made in (1)
FRI 27 JUN 2014
7:55AM: 1/100 dilutions of alpha responsive promoters + GFP for flow cytometry later today.
PRM6+GFP did not grow up yesterday. Updated Plate maps below:
ROUND 3 PLATE 1 PROMOTER MAP
H   G   F   E   D   C   B   A    1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------YDR124W-1------] 5       [--------YDR124W-2------] 6       [--------YDR124W-3------] 7       [--------PRM3-1---------] 8       [--------PRM3-2---------] 9       [--------PRM3-3---------] 10      [--------PRM1-1---------] 11      [--------PRM1-2---------] 12      [--------PRM1-3---------]
~~ROUND 2 PLATE 2 PROMOTER MAP [DID NOT COMPLETE]
H    G   F   E   D   C   B   A    1       [--------ECM18-1--------] 2       [--------ECM18-2--------] 3       [--------ECM18-3--------] 4       [--------SAG1-1---------] 5       [--------SAG1-2---------] 6       [--------SAG1-3---------] 7 8 9 10 11 12~~
8:10AM: 1/100 dilutions set in 30°C room, shaking at 1000RPM. Incubate until 11:10AM.
8:45AM: Set up and started PCR for rtTA + Constitutive Promoter_SV606 homology. End time: 9:40AM.
Need to continue on with Gel verification and PCR purification. (Jessica will include this in her gel samples.)
9:47AM
Flow Cytometer Start Up Procedure
30 minutes prior, push big green start button on right side of machine. Make sure machine is on "Stand-by."
Turn machine to "Run."
Re-initialize HTS
Prime three times
Run CST Beads
250µL sheath fluid into A1 of a 96 well plate.
Load A'1 - A'4 with Bleach and B'1-B'4 with Water. (Opposite corner: H12 = A'1).
Cytometer > CST
Make sure Cytometer Performance Results: Passed
Beads
Kept in fridge in flow cytometry room.
Make sure lot number in CST program matches lot number on box.
Shake bottle
One drop of bead bottle into A1 with sheath fluid
Load into machine and press run
Run clean plate
Click on experiment. Experiment > Open experiment.
Open clean plate "Daily Clean - 96 well U-bottom."
Be here to see how clean the machine is. Under 100 events/sec max!
If over 100 events/sec, run another clean plate!
View events on acquisition dashboard in View > Acquisition Dashboard.
Can also make a clean plate using HTS > Clean…>
11:18AM: Gel Photo of rtTA + TEF1 Homology PCR
11:37AM: Second round of Alpha Response Promoter + GFP characterization induced with alpha factor -> 1:07pm
1:07PM: Treated each sample with cyclohexamide. Running through FACS right now.
1:20PM: Streaked out pAGA_GFP for single colonies. Not sure why plate overgrew so much.
1:30PM: Worked on FlowJo to extract data from fcs files.
4:56PM: Finished with analysis. Created program to easily import data from FACS.
MON 30 JUN 2014
Redoing FACS plate two from Friday since it was left for over two days.
ROUND 4 PROMOTER MAP
H    G   F   E   D   C   B   A    1       [--------ECM18-1--------] 2       [--------ECM18-2--------] 3       [--------ECM18-3--------] 4       [--------SAG1-1---------] 5       [--------SAG1-2---------] 6       [--------SAG1-3---------] 7       [--------CB008-1--------] 8       [--------CB008-2--------] 9 [--------CB008-3--------] 10 11 12
Constitutive Promoter PLATE MAP
1   2   3   4   5   6   7   8   9   10  11  12 A   [-CB008-]        B   [-pTEF1-]        C   [-m3----]        D   [-m6----]        E   [-m7----]        F   [-m10---] G    H
Will be able to run Flow on Wednesday.
II. Streaked out for biological replicate inoculation on wednesday.
III. Analysis of Today's AFRP Plate
TUE 01 JUN 2014
Meeting with Anusuya
Verse yourself in Alpha Factor Pathway. Make barebones alpha factor slide.
Make content folders for website. Collect pieces of information right now.
Characterize promoters by time exposure.
Use Hill curves to fit. Take parameters from curves for models.
What in nature uses positive feedback loops. Comparison to natural systems.
anusuyar@berkeley.edu
Analyzed data from Constitutive Promoters + GFP:
WED 02 JUN 2014
ROUND 5 PROMOTER MAP
Plate 1:     H    G   F   E   D   C   B   A    1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------PRM2-1---------] 5       [--------PRM2-2---------] 6       [--------PRM2-3---------] 7       [--------ASG7-1---------] 8       [--------ASG7-2---------] 9       [--------ASG7-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]  Plate 2:     H    G   F   E   D   C   B   A    1 2 3 4       [--------YDR124W-1------] 5       [--------YDR124W-2------] 6       [--------YDR124W-3------] 7       [--------PRM6-1---------] 8       [--------PRM6-2---------] 9       [--------PRM6-3---------]   10 11 12
Alpha Concentrations Flipped Today
1   2   3   4   5   6   7   8 9   10  11  12 A   [-------------------{3000 nM}----------------] B   [-------------------{1000 nM}----------------] C   [-------------------{100 nM}-----------------] D   [-------------------{10 nM}------------------] E   [-------------------{1 nM}-------------------] F   [-------------------{0.5 nM}-----------------] G   [-------------------{0 nM}-------------------] H
For CB008-1, PRM2-1:3, CLG1-1:3, added 20µL instead of 10µL to 1mL due to low density of cultures. Possibly due to selection of very small colonies.
9:07AM - Started inoculation of 1/100 dilutions.
9:37AM - Plates shaking at 1000RPM in 30°C room.
12:37PM - Alpha Factor Exposures Start 12:57PM - Finished Alpha Factor Exposures
2:37PM - Cyclohexamide treatment
3:00PM - Flow Cytometry Plate Reading
6:00PM - Finish Plate Reading
Alpha Factor Concentration Map, same as before.
THURS 03 JUN 2014
Sick today :/ Tummy hurts
See below for repeat of AFRP GFP measurements in biological replicates.
Some questions that we still need to answer:
Why yeast instead of bacteria? Other studies have made similar circuits in bacteria before (see "Building Biological Memory by Linking Positive Feedback Loops" by Dong-Eun Chang et al.) The argument that yeast are slightly more representative of eukaryotic cells here seems far-fetched as we are not really looking at anything endogenous to yeast. Not that I think we should drop everything and start working in bacteria. We should have a solid response to this though.
SAT 05 JUL 2014
Consolidation of Flow Experiments
Prelim ASG7 PCL2 SAG1 PRM2 CLG1 YDR124W PRM6 PRM1 ECM18 PRM3
ROUND 2 PRM2 CLG1 ASG7 PCL2
ROUND 3 PRM1  PRM3 YDR124W PRM6 did not complete
ROUND 4 SAG1 ECM18
ROUND 5 PRM2 ASG7 PCL2 YDR124W PRM6
ROUND 6 PRM1 ECM18 PRM3 SAG1 AGA1 PRM2 CLG1
MON 07 JUN 2014
Alpha Concentration Map: same as before
Updated ROUND 6 Plate 2mL Growth Plate Map:
Plate 1:     H    G   F   E   D   C   B   A   1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4 5 6 7 8       [--------ECM18-1--------] 9       [--------ECM18-2--------] 10      [--------ECM18-3--------] 11       12     Plate 2:     H    G   F   E   D   C   B   A    1       [--------SAG1-1---------] 2       [--------SAG1-2---------] 3       [--------SAG1-3---------] 4       [--------AGA1-1---------] 5       [--------AGA1-2---------] 6       [--------AGA1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10      [--------CLG1-1---------] 11      [--------CLG1-2---------] 12      [--------CLG1-3---------]  Plate 3   1   2   3   4   5   6   7   8   9   10  11  12 A [-CB008-]   ⨅   ⨅   ⨅   ⨅   ⨅   ⨅ B [-pTEF1-]   |  PRM1 |   |  PRM3 |  C [-m3----]   |   |   |   |   |   | D [-m6----]   1   2   3   1   2   3   E [-m7----]   |   |   |   |   |   | F [-m10---]   |   |   |   |   |   | G             ⨆   ⨆   ⨆   ⨆   ⨆   ⨆ H
For PRM2, we added 100µL of overnight yeast due to OD600 of 0.3. All others added 10µL to 1mL of SD complete
Incubation at 30°C 1000RPM at 9:45am -> 12:45PM
Exposure to Alpha factor at 1:05PM -> 2:35PM.
ROUND 6 96 Well Reading Map
Plate 1:     H    G   F   E   D   C   B   A   1       [--------CB008-1--------] 2       [--------CB008-2--------] 3       [--------CB008-3--------] 4       [--------PRM1-1---------] 5       [--------PRM1-2---------] 6       [--------PRM1-3---------] 7       [--------ECM18-1--------] 8       [--------ECM18-2--------] 9       [--------ECM18-3--------] 10      [--------PRM3-1---------] 11      [--------PRM3-2---------] 12      [--------PRM3-3---------]     Plate 2:    H    G   F   E   D   C   B   A    1       [--------SAG1-1---------] 2       [--------SAG1-2---------] 3       [--------SAG1-3---------] 4       [--------AGA1-1---------] 5       [--------AGA1-2---------] 6       [--------AGA1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10      [--------CLG1-1---------] 11      [--------CLG1-2---------] 12      [--------CLG1-3---------]  Plate 3:    1   2   3   4   5   6   7   8   9   10  11  12 A [-CB008-]   B [-pTEF1-]    C [-m3----]    D [-m6----]  E [-m7----]    F [-m10---]    G              H
Combined analysis of most* promoters:
*Does not include PRM6 or AGA1!
TUE 08 JUL 2014
Out sick :(
WED 09 JUL 2014
Out sick :( To do:
-Website stuff -Modeling stuff -> make graphs out of data from found functions.
THURS 10 JUL 2014
Results of Constitutive Promoters
Results of Inducible Promoters Second set 2 of try 2
Today:  Recharacterizing inducible promoters in CB008DB strains
GFP = before signal processing RFP = after signal processing
Flow with AFRP in CB008DB cells
plate 1
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4       [--------CLG1-1---------] 5       [--------CLG1-2---------] 6       [--------CLG1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10      [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]
Plate 2
H   G   F   E   D   C   B   A    1       [--------SAG1-1------------] 2       [--------SAG1-2------------] 3       [--------SAG1-3------------] 4       [--------AGA1-1------------] 5       [--------AGA1-2------------] 6       [--------AGA1-3------------] 7        8        9        10       11       12
Results of Inducible Promoters in DB:
FRI 11 JUL 2014
AFRP GFP DB ROUND 2
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4        5       [--------PRM1-1---------] 6       [--------PRM1-2---------] 7       [--------PRM1-3---------] 8       [--------ECM18-1--------] 9       [--------ECM18-2--------] 10      [--------ECM18-3--------] 11       12
Today:
Presentation for Wendell on Monday
Analyze Flow Data from today
Start modeling previous data
MON 14 JUL 2014
Group Meeting today.
Prep for flow cytometry tomorrow. Waiting on Colony PCR of transformants to ensure correct insertions. Will be characterizing yeast strains with pTETGFP and pTEF1rtTA under 12 different concentrations of Doxycycline.
TUE 14 JUL 2014
Flow with AFRP in CB008DB cells
H   G   F   E   D   C   B   A    1       [--------CB008DB-1------] 2       [--------CB008DB-2------] 3       [--------CB008DB-3------] 4       [--------ASG7-1---------] 5       [--------ASG7-2---------] 6       [--------ASG7-3---------] 7       [--------YDR124W-1------] 8       [--------YDR124W-2------] 9       [--------YDR124W-3------] 10      [--------PRM6-1---------] 11      [--------PRM6-2---------] 12      [--------PRM6-3---------]
Alpha factor concentrations same as before 9:40AM Dilution -> 12:40PM Induction with Alpha Factor
Analysis of FSC-SSC plots from 140715 show significant oddities.
Wells with "S Curve" in SSC by FSC plot
1   2   3   4   5   6   7   8   9   10  11  12 A X B X C X D X E X F X             G X H
Wells with "Contamination" in SSC by FSC plot
1   2   3   4   5   6   7   8   9   10  11  12 A           B  C  D         X   X   X       X   X E         X       X   X       X F         X       X   X   X   X G                     X   X   X   X  H
Conclusion: will redo this set of promoters tomorrow.
WED 15 JUL 2014
[CB008 CB008DB] + pTEF1rtTA + pTETGFP
Flow plate maps today:  Doxycycline concentrations:
H   G   F   E   D   C   B   A    1               [0 µg/ml------------] 2               [0.03 µg/ml---------] 3               [0.06 µg/ml---------] 4               [0.09 µg/ml---------] 5               [0.3 µg/ml----------] 6               [0.6 µg/ml----------] 7               [0.9 µg/ml----------] 8               [3.0 µg/ml----------] 9               [6.0 µg/ml----------] 10              [9.0 µg/ml----------] 11              [30 µg/ml-----------] 12              [60 µg/ml-----------]
pTEF1 promoters
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008]----------------------]   B [---------------[CB008 pTET_GFP]-------------] C [---------------[CB008 pTET_GFP pTEF1_rtTA]--]   D [---------------[CB008 pTET_GFP pTEF1 m6]----]      E [---------------[CB008 pTET_GFP pTEF1 m7]----]     F [---------------[CB008 pTET_GFP pTEF1 m10]---]   G H    1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------]   C [---------------[CB008DB pTET_GFP pTEF1 1of2 ]  D [---------------[CB008DB pTET_GFP pTEF1 2of2 ] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G  H
1/100 dilution of cultures started at 7:25AM -> 10:25AM
Dilutions
Stock: 100mg/mL  **1x**           **100x**         **Ratios**      **Prep** 60µg/mL       A: 6mg/mL           06% S       60µL of Stock in 940µL of water 30µg/mL       B: 3mg/mL           03% S 30µL of Stock in 970µL of water   9µg/mL        C: 900µg/mL         15% A       150µL of A in 850µL of water   6µg/mL        D: 600µg/mL         10% A       100µL of A in 900µL of water 3µg/mL        E: 300µg/mL         10% B       100µL of B in 900µL of water 0.9µg/mL      F: 90µg/mL          10% C       100µL of C in 900µL of water 0.6µg/mL      G: 60µg/mL          10% D       100µL of D in 900µL of water 0.3µg/mL      H: 30µg/mL          10% E       100µL of E in 900µL of water 0.09µg/mL     I: 9µg/mL           10% F       100µL of F in 900µL of water 0.06µg/mL     J: 6µg/mL           10% G       100µL of G in 900µL of water 0.03µg/mL     K: 3µg/mL           10% H       100µL of H in 900µL of water 0             N/A                 N/A         N/A
Induction with Doxy at 11:16AM -> 5:16PM
THURS 17 JUL 2014
TODO:
[X] Working on generating noise graphs for Hyun
[X] Need to ensure that all the data is correct and current. Will need to redownload all files. Ugh.
[X] Need to decide if I should first divide SD by Mean GFP then average triplicates or first average SD and Mean GFP triplicates and then divide.
[X] Need to also understand if we are using Geometric or Arithmetic Means
[X] Generate pTEFrtTA + pTETGFP v. Doxy graphs.
[X] Confirm which plate is which strain
Meeting with Anasuya
[ ] Understanding background
[ ] Focus on how T-reg cells interact with macrophages. Macrophages can play a suppressive role on T-regs.
[ ] Dendritic cells and T-reg cells as an example of a divergent system.
[ ] Stanford iGEM 2010: interaction between t-reg and t-helper cells.
[ ] How microglia interact with different cell types: might play a role in regulating T-helper cells.
[ ]  Non-immune standpoint
[ ] Examples of how cells respond and coordinate using secondary post-cellular signaling.
[ ] OVERALL GOAL: Understand how your work can help elucidate and understand other cell/organism interactions through community signaling
[ ] Model
[ ] Website
[ ] Generate content for website.
FRI 18 JUL 2014
[ ] Model for pTEFrtTA pTETGFP pTETmFα ARFPmFα circuit
[ ] Functions
[ ] Develop function to model pTEFrtTA -> pTETGFP/pTET_mFα.
[ ] Need to understand how [Doxycycline] alters [α]. Does [GFP] correlate directly with [α]?
[X] Develop function to model AFRP_RFP
[ ] Develop function for Diffusion of Alpha factor as as F(Initial Alpha Factor Concentration, Time, Distance)
[X] Data for Model
[X] Export all sigmoid fits
Conclusions from today:  To understand Alpha Factor Secretion, need to understand how [Doxy] -> pTEF1rtTA -> pTETGFP results translate to [Doxy] -> pTEF1rtTA -> pTETmFα -> [α].  Need this circuit: ([Doxy] -> pTEF1rtTA -> pTETmFα -> [α]) + (pAGA1RFP -> [RFP]) and then use ([α] -> pAGA1GFP -> [GFP]) measurements to understand the local concentrations of Alpha Factor from each cell.
Need to construct: ([Doxy] -> pTEF1rtTA -> pTETGFP + pTETMFα -> pAGA1RFP -> [RFP])
Questions for Hyun:
How does one find the phenomological alpha factor secretion from the data that we currently have: [Doxy] -> (pTEF1rtTA + pTETGFP) -> [GFP] and [α] -> (AFRP_GFP) -> [GFP].
Even though there seems to be a one-to-one correlation between [GFP] and [α], leading us to an understanding of local [α], what does that say about [rtTA-Dox] -> pTET_GFP production of GFP, and eventually [α]?
SAT 19 JUL 2014
See checklist in ##FRI 18 JUL 2014.
MON 21 JUL 2014
Working on Group Presentation. Made graphics for yeast modeling.
Working on two cell models.
TUES 22 JUL 2014
100x dilutions done at 9:20am 9:20am -> 12:20pm
Plate 1:
H           G   F   E   D   C   B   A    1   [pTEF1-1]   [--------CB008DB-1------] 2   [pTEF1-2]   [--------CB008DB-2------] 3   [pTEF1-3]   [--------CB008DB-3------] 4   [M7-1]      [--------ASG7-1---------] 5   [M7-2]      [--------ASG7-2---------] 6   [M7-3]      [--------ASG7-3---------] 7               [--------YDR124W-1------] 8               [--------YDR124W-2------] 9               [--------YDR124W-3------] 10              [--------PRM6-1---------] 11              [--------PRM6-2---------] 12              [--------PRM6-3---------]
Plate 2:
H   G   F   E   D   C   B   A    1       [--------AGA1-1---------] 2       [--------AGA1-2---------] 3       [--------AGA1-3---------] 4       [--------CLG1-1---------] did not grow as much as the others 5       [--------CLG1-2---------] 20ul instead of 10ul for CLG1 6       [--------CLG1-3---------] 7       [--------PRM2-1---------] 8       [--------PRM2-2---------] 9       [--------PRM2-3---------] 10  [--------PCL2-1---------] 11      [--------PCL2-2---------] 12      [--------PCL2-3---------]
Plate 3:
H    G   F   E   D   C   B   A    1        [--------ECM18-1--------] 2        [--------ECM18-2--------] 3        [--------ECM18-3--------] 4        [--------SAG1-1---------] 5        [--------SAG1-2---------] 6        [--------SAG1-3---------] 7         8        PRM3 MIA 9         10       [--------PRM1-1---------] 11       [--------PRM1-2---------] 12       [--------PRM1-3---------]
WED 23 JUL 2014
Reading up on Stochastic Modeling of our genetic circuits
Illustrations and style design of website and poster materials
TODO:
[ ] Make signs by Friday for Exploratorium Exhibition
[ ] Make pPCL2BFP and pAGA1BFP
[X] Transform BFP to make more of it. Grow up, miniprep. (Will be transforming two more backbones with antibiotic resistance)
[X] Digest BFP using Xho1 and Not1
[X] Digest pAGA1RFP and pPCL2RFP with Xho1 and Not1.
[ ] Ligate BFP with pAGA1backbone and pPCL2backbone.
THUR 24 JUL 2014
See todo list in 23 JUL 2014
E. Coli Transformation Protocol
Materials DNA       1µL DH5α      50µL
Add DNA and DH5α.
Let sit on ice for 10 mins if whole plasmid/30 min if ligation reaction
Heat shock @ 42°C for 45 seconds
Add 250µL of SOC.
Incubate at 37°C for 1 hour (shaking)
Plate on appropriate plate.
Total Time: 80-100 minutes
Incubated at 10:30AM -> 11:30PM
FRI 25 JUL 2014
Pick colonies for miniprep on Monday -> refrigerated and then set to grow on Sunday. Minipreps today.
Developed design guide for website and other materials
Using 4µL of pAGA1_RFP on Monday
Set up Balsamiq for the team. Designing flyers for Exploratorium event.
MON 28 JUL 2014
Miniprep transformations of BFP and shuttle vectors. - Jeffrey
Prepare for presentation today.
Digest pTS94(BFP), pGEM22(pAGA1+mCherry), and (pPCL2+mCherry)x2 with Xho1 and Not1 double overnight digest @ 37°C overnight.
TUES 29 JUL 2014
Gel Extract Digestions from Yesterday.  BFP Length: 696 Basepairs.
[CB008DB] + pTEF1rtTA + pTETGFP
Flow plate maps today:  Doxycycline concentrations:
H   G   F   E   D   C   B   A    1           [----0 µg/ml------------] 2           [----0.03 µg/ml---------] 3           [----0.06 µg/ml---------] 4           [----0.09 µg/ml---------] 5           [----0.3 µg/ml----------] 6           [----0.6 µg/ml----------] 7           [----0.9 µg/ml----------] 8           [----3.0 µg/ml----------] 9           [----6.0 µg/ml----------] 10          [----9.0 µg/ml----------] 11          [----30 µg/ml-----------] 12          [----60 µg/ml-----------]
pTEF1 promoters
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008DB]--------------------]   B [---------------[CB008DB pTET_GFP]-----------]   C [---------------[CB008DB pTET_GFP pTEF1]-----]  D [---------------[CB008DB pTET_GFP pTEF1 m3]--] E [---------------[CB008DB pTET_GFP pTEF1 m6]--]  F [---------------[CB008DB pTET_GFP pTEF1 m7]--]              G [---------------[CB008DB pTET_GFP pTEF1 m10]-] H
x3 plates
100X Dilution started at 7:16AM -> 10:16AM
10:50AM Dox Dose.  Plate 1 concentrations backwards.  Plate 3 lane 10 had double exposure of two different concentrations.
Gel Extractions of Digestions
Lanes:  1: 1kb Ladder 2: BFP Digestion 3: BFp Digestion 4: pAGA1 mCherry Digestion 5: pAGA1 mCherry Digestion 6: pPCL2 mCherry Digestion  7: pPCL2 mCherry Digestion
Digestion smeared.
New strategy:  PCR BFP
Ligate with Backbone(pHY130E), and PCL2 and AGA1.
Today, Digest pAGA1 from pTS108 with APA1 (11:35AM-1:35PM) and Xho1 (1:35PM -> 2:35PM) and gel extract (2:35PM -?> 4PM)
Digestion of pAGA1 from pAGA1-SAG1 (pTS108) failed, showing very faint bands and smearing. Will try with new ddH20.
Will now also PCR pAGA1.
WED 30 JUL 2014
Could not run flow cytometry today because no cells grew.
1   2   3   4   5   6   7   8   9   10  11  12 A [---------------[CB008]----------------------]   B [---------------[CB008 pTET_GFP]-------------] C [---------------[CB008 pTET_GFP pTEF1_rtTA]--]   D [---------------[CB008 pTET_GFP pTEF1 m3]----]      E [---------------[CB008 pTET_GFP pTEF1 m6]----]     F [---------------[CB008 pTET_GFP pTEF1 m7]----]   G [---------------[CB008 pTET_GFP pTEF1 m10]---]  H
PCR'd pAGA1 and BFP overnight.
Thanks to Jeffrey for his assistance today while I'm out sick:
[X] PCR Clean Up
[X] Digests
[X] BFP using Xho1 Not1 double digest for 3 hours w/ cutsmart @ 37°C
[X] pAGA1 using Apa1 w/ cutsmart for 2 hours @ room temp
[X] pAGA1 using Xho1 w/ cutsmart for 1 hour @ 37°C
[X] Ligations (Overnight)
[X] Ligate: Hy130E - pAGA1 - BFP (Hy130E in my box! should already be digested!)
[X] Ligate: Hy130E - pPCL2 - BFP (pPCL2 should also be in my box, already digested!)
THURS 31 JUL 2014
Out sick: Jeffery altruistically offered to help.
FRI 01 AUG 2014
Transformed Ligations from yesterday.  Now working on website.
Finished landing page with petri dish now.  Will now work on Team and Protocols pages.
Incorporate iGEM logo into top right corner.
Website Map
Home
Project
Background
Achievements
Implications
Parts
Safety
Models
Human Practices
Collaborations
ALHS
Super Science!
Notebooks & Protocols
Notebooks
Protocols
Team
Attributions
Advisors
Mentors
Students
Contact
iGEM Profile
MON 04 AUG 2014
Worked on website.  Picked colonies for miniprep tomorrow. Should have performed a e. coli colony PCR.
Colony PCR for screening E. Coli
Pick single colonies ( 5 or so from each plate ) mix in 25 µl H2O in a tube. Use 5 µl in PCR reaction
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
6 µl
10 µM Reverse primer
1 µl
6 µl
Water
3 µl
18 µl
Bacterial cells (template)
5 µl
-----
Cycle (Varies):
95° C | 5m
30x:       95° C | 45s        55° C | 30s        72° C | 1m per kb
72° C | 10m      4° C | Forever
load 5 µl onto gel
for all positive bands - take the rest of the bands and inoculate them into an overnight LB (+antibiotic) for miniprep
Started at 10:18AM -> 12:35PM
TUES 05 AUG 2014
Colony PCR of pPCL2BFP and pAGA1BFP
Miniprep'd pAGA1 in lanes 5 and 6.
Picked new colonies from pPCL2 plate and ran colony PCR.
WED 06 AUG 2014
Working on Poster for Santa Cruz today.  Found mixed peaks in sequencing reaction. Otherwise, sequences were correct. Will transform minipreps and re-miniprep. Miniprep pPCL2 today.
THURS 07 AUG 2014
Sequences from pPCL2 BFP worked out very well. Colonies grew on pAGA1BFP  plate and not on negative control. Will miniprep pAGA1BFP tomorrow.
FRI 08 AUG 2014
1:200 dilution at 7:45AM.
[Dox] dose response of pTEF1rtTA pTETGFP pTETMFalpha pAGA1mCherry
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 m6-1]----]       [CB008 m10-3]----] B [CB008 m6-2]----]       [CB008DB m7-1]---] C [CB008 m6-3]----]       [CB008DB m7-2]---] D [CB008 m7-1]----]       [CB008DB m7-3]---] E [CB008 m7-2]----]  F [CB008 m7-3]----]            G [CB008 m10-1]---]  H [CB008 m10-2]---]
[Dox] Map:
H   G   F   E   D   C   B   A    1           [----0 µg/ml------------] 2           [----0.03 µg/ml---------] 3           [----0.06 µg/ml---------] 4           [----0.09 µg/ml---------] 5           [----0.6 µg/ml----------] 6  7                      [--0 µg/ml---] 8                      [--0.03 µg/ml] 9                      [--0.06 µg/ml] 10                     [--0.09 µg/ml] 11                     [--0.6 µg/ml-] 12
MON 11 AUG 2014
1:200 dilution of 4 different strains into 4 96 well plates.
Need to prepare presentation for group meeting tomorrow.
Dilution of cultures at 7:30AM.
4 different promoters in front of rtTA: pHYM1, pYDR124W, pCLG1, pASG7
Dox
H   G   F   E   D   C   B   A    1           [----0 µg/ml------------] 2           [----0.03 µg/ml---------] 3           [----0.06 µg/ml---------] 4           [----0.09 µg/ml---------] 5           [----0.3 µg/ml----------] 6           [----0.6 µg/ml----------] 7           [----0.9 µg/ml----------] 8           [----3.0 µg/ml----------] 9           [----6.0 µg/ml----------] 10          [----9.0 µg/ml----------] 11          [----30 µg/ml-----------] 12          [----60 µg/ml-----------]
Alpha Factor Concentration Map
1   2   3   4   5   6   7   8   9   10  11  12 A   [-------------------{0 nM}-------------------] B   [-------------------{0.5 nM}-----------------] C   [-------------------{1 nM}-------------------] D   [-------------------{10 nM}------------------] E   [-------------------{100 nM}-----------------] F   [-------------------{1000 nM}----------------] G   [-------------------{3000 nM}----------------] H
Single cell scatter.
Correlation Coefficient between GFP and RFP.
CV (noise) for all strains.
TUES 12 AUG 2014
Rest of the AFRPrtTA pTETGFP
Set in induction at 8:11AM.
Notes about pTEF1rtTA pTETGFP pTETMFa pAGA1RFP data from Friday:
GFP Noise:  Very consistent pattern across almost all strains(mutations of pTEF1). High  [Doxycycline] induces noise to decrease over time, while lower doxycycline concentrations maintain similar noise levels over time, and some even increase.
WEDS 13 AUG 2014
Pick colonies for Jeffrey.  Rest of pTEF1rtTA pTETGFP pTETMFa pAGA1mCherry
1:200 dilution at 7:50AM.
[Dox] dose response of pTEF1rtTA pTETGFP pTETMFalpha pAGA1mCherry
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 pTEF1-1]-]       [CB008DB m3-3]--]  B [CB008 pTEF1-2]-]       [CB008DB m6-1]---] C [CB008 pTEF1-3]-]       [CB008DB m6-2]---] D [CB008DB pTEF1-1]       [CB008DB m6-3]---] E [CB008DB pTEF1-2]   F [CB008DB pTEF1-3]            G [CB008DB m3-1]--]  H [CB008DB m3-2]--]
THURS 14 AUG 2014
1:200 Dilution at 7:52AM
[Dox] dose response of pTEF1rtTA pTETGFP pTETMFalpha pAGA1mCherry
1   2   3   4   5   6   7   8   9   10  11  12 A [CB008 pTEF1-1]-]       [CB008 m7-3]-----]  B [CB008 pTEF1-2]-]       [CB008 m10-1]----] C [CB008 pTEF1-3]-]       [CB008 m10-2]----] D [CB008 m6-1]----]       [CB008 m10-3]----] E [CB008 m6-2]----]   F [CB008 m6-3]----]           G [CB008 m7-1]----]  H [CB008 m7-2]----]
Plate failed due to large amounts of contamination. Flow FSC/SSC plots showed large amounts of contamination.
FRI 15 AUG 2014
UCSC Meetup!
MON 18 AUG 2014
Moving!
TUES 19 AUG 2014
Moving!
WED 20 AUG 2014
Back at it!  Taught people how to generate graphs from flow data. Also taught people how to generate scatter plots from flow data. Produced graphs for flow data from many days ago.
Decided to not continue with pECM18 and pPRM6 as they did not seem to express more than basal level.
THURS 21 AUG 2014
Attempted to use R for further data analysis. Importing data into R may be too complicated and time-consuming.
FRI 22 AUG 2014
Seems like all DB strains have RFP expression without induction with Doxy. Most likely due to leakiness in Tet-On circuit.
Decided to not analyze data from Tuesday and will be running experiments again on Monday.
Downloaded and install flowjo on my computer for easier remote work.
George's Lab Notebook
6/9/14
PCR for Constitutive Promoters (m10)
Materials
1x reaction
4.5x Master Mix
x Phusion HF Buffer
10 µl
45 µl
dNTP's (10 mM)
1 µl
4.5 µl
Forward Primer (10µm)
2.5 µl
11.25 µl
Reverse Primer (10µm)
2.5 µl
11.25 µl
*Template DNA (m10)
0.3 µl
1.35 µl
Phusion Polymerase
0.5 µl
2.25 µl
Water
33.2 µl
149.4 µl
Total
50 µl
225 µl
1. Mix materials in a 4.5x Master Mix on ice. Mix well. 2. Pipetter 50 µl from the Master Mix into 4 labeled PCR tubes 3. Thermocycler for :       Initial Duration   | 98° C | 30s        35 Cycles of:            Denaturation | 98° C | 10s           Annealing    | 55° C | 20s           Extension    | 72° C | 30s     Final Extension    | 72° C | 5m     Hold               | 4°  C | Forever  Possible Errors: Incomplete thawing of dNTPs, not well mixed
Working Stock of Reverse and Forward Primers for pTEF1 kept in my freezer box. 6/9/14
6/10/14
Gel loading of pTEF1 in Gel
Loading order: 15Wells 100bp ladder|Eleanorx4|Robertx4|Sabrinax4|100bp ladder
20wells 100bp ladder|Jeffreyx5|skip|Georgex2|100bp ladder
*m10 only had two wells loaded because a centrifuge spin burst onpen my caps and 2 flew out. Luckily one was unharmed and I was able to load two samples in.
Gel Extraction
Cut DNA band from PCR of constitutive promoter gel Weigh gel: .280g QG Buffer: 840ul     *QG buffer amount is 3:1 of gel weight  1. Mix gel slice with QG Buffer in 50C heat bath for 10 min. 2. Add 1 ml isopropanol. Mix. 3. Add 750 ul of mixture into purple Qiagen spin column. Spin for 30 sec. Discard liquid.  Repeat. 4. Add 750 ul of Buffer PE and spin for 1 min. Discard waste. 5. Dry spin the column for 1 min. Replace the spin column in a new microcentrifuge tube 6. Elute in 50 of ddH2O. Wait 1 min and then spin for 1 min. 7.Keep the runthrough and discard the column
Restriction Digest with ApaI
40 ul of DNA (m10 gel extraction) 5 ul of 10x CutSmart Buffer .5 ul of ApaI Enzyme  *digest overnight at room temperature*
6/11/14
Restriction Digest with XhoI
Add .5 ul of XhoI enzyme to ApaI digestion from 6/10/14 Incubate in the 37°C shaker.
PCR Purification
1. Add 250 ul of Buffer PB to digestion. 2. Place sample in a purple QIAquick spin column. Spin for 1 min. Discard waste. 3. Wash with 750 ul of Buffer PE. Spin for 1 min. Discard waste. 4. Dry spin for 1 min. Replace spin column in a new microcentrifuge tube. 5. 6. Elute in 50 ul of ddH2O (add directly into the column film). Wait 1 min and then spin for 1 min.  Total concentration: 40.28ul
DNA Ligation
Materials
Volume
10x Ligase Buffer
1 ul
DNA Backbone PSV606
.2 ul
DNA Insert (PCR Purication)
.2 ul
T4 DNA Ligase
.5 ul
H2O
8.1 ul
Total                       | 10 ul ---------------------------------------------------- Mix reagents and incubate at room temperature for 2 hrs
Ligation of pTEF1 into PSV606 kept in my freezer box. 6/11/14
Transformation
10 ul of ligation 50 ul of E. Coli competent cells  30 min    | ice 45 sec    | 42°C heat shock 2 min     | ice  Add 250 ul of SOC media. Incubate at 37°C for 1 hr. Plate on LB+Carb.   Only 2 colonies formed on plate
6/12/14
Redo Transformation
Follow procedure from 6/11/14 with minor alterations. 1. Use .4 ul of ligation instead of .2 ul. 2. Use more competant cells.     *REMEMBER NEGATIVE CONTROL GUYS
6/13/14
Transforming α-inducible promoters
Transforming 11 different inducible promoters into CB008 and CB008DB strains of yeast.  1. Boil salmon sperm DNA (ssDNA) for 10 min.-->10 ul of 10 mg/ml stock per transformation 2. Cool ssdna on ice for at least 10 minutes 3. Pellet yeast cultures in centrifuge. (3000 rpm for 2-5 min.) 4. Resuspend with 1ml .1M LiOAc in TE. 5. Pellet cells (3000 rpm for 2-5 min.) 6. Resuspend in 100 ul .1M LiOAc in TE per 2.5 ml culture.  7. Aliquot 100 ul into each microcentrifuge per transformation. (22 tubes) Per Tube: 8. Add 100 ug ssDNA, 1 ug of target DNA (1~5 ul)  ADD IN ORDER 9. 480 ul 50% PEG 3350 10. 60 ul 10x TE 11. 60 ul 1 M LiOAc 12. 75 ul DMSO  13. Vortex 14. Incubate at 42°C for 30 min.  15. Pellet cells (6000 rpm for 2 min) 16. Resuspend with 500 ul YPD. 17. Pellet cells. 18. Resuspend with 50 ul  YPD 19. Plate on SDS-Ura.   Incubate 2 days at 30°C.
6/16/14
Colony PCR for Screening E. coli
1. Pick a single colony using a sterile wooden stick and mix in 25ul of water in a PCR tube. Do this for about 4-6 colonies. Use 5ul for the PCR reaction below, adn save the rest for later.  2. Set up PCR reaction as below:                                             1x Reaction      6x master mix                                         ---------------------------------- 2X Go Taq Green PCR Master Mix                  10ul               60ul 10 uM FW primer                                  1ul                6ul 10uM RV primer                                   1ul                6ul Water                                            3ul               18ul  Bacterial cells (template)                       5ul                --         Cycles:             95                      5 min             30X:    95              45 s                     55              30 s                     72              1 min per kb (adjust accordingly)             72                      10 min  3. Analyze products on a 1% agarose gel. The GoTaq mix alread has gel loading dye in it, so you can just directly load 5ul of your PCR readction into a gel.  4. For all positive bands on the gel, take the rest of the bacterial cells from step 1 and inoculate them into and overnight LB (+antibiotic) for miniprep the next day.
E. coli Colony PCR results
*refer to gel photo on 6/13/14 in George's Lab notebook-> Gel Photos -> 6-13-14 E. Coli Colony PCR
**Additional Notes: I was m10 so I occupied lanes 11-15 on row 2 and all the results looked quite bomb.
6/16/14
Sequencing
All sucessful sequences (all of them)
Colony PCR for Yeast + Patching
1. Label the patch plate with the strain and promoter along with the colony number picked.  2. Pick 3 big isolated colonies and patch with a stick to the patch plate. 3. # the colonies that you picked on the plate. 4. Fill PCR tubes with 10ul of NaOH 5. Stick the rest of the yeast on the stick into the PCR tubes 6. Boil for 10 mins@ 95 7. PCR TIME 8. Set up reactions as followed:  1x Reaction      7x Master Mix                                         ---------------------------------     2X Go Taq Green PCR Master Mix              10ul               70ul     10 uM FW primer                              1ul                7ul     10uM RV primer                               1ul                7ul     Water                                        5ul               35ul      ----------------------------------------------------------------------     Boiled Yeast cells (template)                3ul            9. Set up Thermocycler for:     95°C       | 5 min     30x |95°C   | 45 sec         |50°C   | 30 sec         |72°C   | 1 min per kb     72°C       | 10 min     4°C            | hold
Running a gel for AFRPs
1st gel  1st row 20wells  1       2       3       4       5       6       7       8       9       10      11      12      13      14      15      16      17      18      19      20 100bb          SAG1                   ECM18                    PRM3                     HYM1                  ASG7 CLO7            PRM1                                               ladder  1       2       3       1       2       3       1       2       3       1       2       3       1       2       3       1       2       3       1   2nd row 15 wells I give up
Refer to the gel photo in George's noteboot -> Gel photos->6-16-14 AFRPS stuffs i'll update when i find the picture.
6/18/14
Colony Dilutions
20x dilutions to all CB008 and CB008DB alpha-promoter yeast.  *Note: CB008DB PRM1-2 sticker was labled on top of CB008 SAG1-3 and could've resulted in a sticker coming off at night.
Glycerol stocks
1:20 Dillution 200ul cells + 9.8ml YPD media. FREEZE
6/19/14
Flow Cytometry
1. Start overnight cultures of our strains to be tested om SD complete media instead of YPD (YPD has fluorescence). 2. Dilute overnight cultures to OD 0.05-0.1 in the 96-well shakeplate.  The plates are in a cabinet in the limlab wrapped in shiny paper. USe SD complete media and the total volume of the well should be 1mL. Make a plate map now. 3. Allow cells to enter growth phase by putting wells in plate shakers for 3 hours at 1000rpm@30 degrees. 4. Allow induction with alpha factor. The stock alpha factor is 3mM and we used 0, 1, 10, 100, 1000nM. (Stocks cannot be refrozen, discard after use) 5. Allow cells to enter growth stage by putting on plate for 90-120 minutes, no more.96  6. Transfer 20ul of each culture to the 96 well flow-cytometry v-bottom plate. Use cycloheximide to arrest cells. Cycloheximide suspends cell division. 7. Run on Flow Cytometer     1. Check liquid box for sufficient amount of liquid, if too low, scream for help.     2. Check the second tub for liquid with clear top. It should also have some liquid, don;t shake too harshly     3. Check the gasoline tank, if kinda full dump out and add some bleach to the bottom of the container.     4. You can open the iGEM 2014 path and the default settings will be default, but check it anyway to be sure.     5. For each new experiment, load 2 neg control wells and run those first as a blank.
(Voltage settings: FSC:250, SSC:280, FTC:550
Select wells by double-clicking until it is green and you can just add more. The volume can be modified, but 200ul is suggested (decrease by 50ul from our load)
Flow speed is usually kept at either 0.5ul per second or 1ul/second and can be adjusted relatively quickly. VOlume can be less than our sample volume.
Data is exported by usb, no internet is allowed.)
8. Analyze data (graphs: histogram vs FTC,FSC vs SSC) Flowjo note on page 17 and export to matlab yo.
96 well plates
Notebooks
Protocols
Protocols
Minipreps
PCR Purification
Gel Extractions
E-Coli Transformation
Yeast Transformation
E-Coli Colony PCR
Yeast Colony PCR
Yeast Frozen Glycerol Stocks
Select a Protocol
Miniprep
Resuspend pelleted bacterial cells in 250 µl Buffer PI and transfer to a microcentrifuge tube.
Add 250 µl Buffer P2 and gently invert tube 4-6 times to mix
Add 350 µl Buffer N3 and invert immediately 4-6 times
Centrifuge for 10 min at 13,000 rpm
Add supernatant to QIAprep spin column
Centrifuge for 30-60s - Discard flow through
Wash QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30-60s - Discard flow through
Wash column by adding 0.75 ml Buffer PE and centrifuginh for 30-60s
Discard flow through and centrifuge for an additional 1 min to remove residual wash buffer
Place column in clean 1.5 ml microcentrifuge tube. To elute DNA add 50 µl water or buffer EB, let stand for 1 min and centrifuge for 1 min.
PCR Purification
5 volumes of Buffer PB (with ethanol) to 1 volume PCR sample
Apply sample to column and spin for 1 min
Discard flow through
To wash add 0.75 ml of Buffer PE to column and centrifuge for 1 min
Discard flow through and place column back in the same tube
Cetrifuge for an addtitional minute (Dry Spin)
Place column in clean 1.5 ml tube
[Elute DNA] Add 30µl water to column, let stand for 1 min and centrifuge for 1 min
Gel Extraction
QlAquick Gel Extraction Kit
Cut Gel
Weigh it in a colorless tube
Add 3 volumes Buffer Q G to 1 volime Gel (100mg ~ 100µl)
Incubate @ 50° C for 10 min or until completely dissolved (vortex every 2-3 min to help dissolve)
Add 1 gel volume isopropanal to the sample and mix
Place a QlAquick soin column in a provided 2ml collection tube
Place sample in column & spin for 1 min --> discard flow through
To wash add 0.75 ml Buffer PE to column & centrifue for 1 min, then dry spin
Place column in 1.5 ml tube
Add 35µl H2O & centrifuge for 1 min.
E. Coli Transformation (Digestion/Ligation)
- 10 µl ligation - 50 µl competent cells --> 30m on ice --> 45s heatshock 42° C --> 2m on ice - 250 µl of SOC media --> 1h shake 37° C
Yeast Transformation
Reagents
YPD
1 M LiOAc
10X TE pH 7.5
1X TE pH 7.5, 0.1 M LiOAc
50% PEG 3350
DMSO
Salmon Sperm DNA (ssDNA)
PEG = viscous, pipette slow
boil ssDNA aliquots
Previous Day : Grow yeast strain to be transformed in 5-10 mL YPD overnight at 30° C
Set up digest to linearize DNA
Dilute O/N culture ~ 1:20 in YPD grow 2-4 hours at 30° C
Prepare ssDNA - boil for 10m cool on ice for at least 10m (10 µl of 10 mg/mL stock per transformation)
Harvest cells in centrifuge - 3000 rpm, 2-5 min
Wash with 1 ml 0.1 M LiOAin TE
Pellet cells - 3000 rpm , 2-5 min
Resuspend pellet in 100 µl 0.1 M LiOAc in TE per 2.5 ml culture, split into 100 µl per epindorph tube for each transformation
to 100 µl cells add 100 µg ssDNA (10 µl of 10 mg/mL sock), 1-5 µl target DNA
Add (in order): 480 µl 50% PEG 3350, 60 µl 10X TE, 60 µl 1 M LiOAc (for final 40% PEG, 1X TE, 0.1 M LiOAc) Optional: Add 75 µl DMSO definitely did
Vortex
Incubate 42° C for 30m & begin drying plates
Pellet (6000 rpm - 2m), discard supernatant (remove PEG completely by pipetting), resuspend in 500 µl YPD (or selective media)
Pellet, discard supernatant, resuspend in risidual ~ 50 µl YPD
Plate on selective media
Incubate 1-3 days
Colony PCR for screening E. Coli
Pick single colonies ( 5 or so from each plate ) mix in 25 µl H2O in a tube. Use 5 µl in PCR reaction
Reagents
1X
6X
2X GoTaq Green PCR Master Mix
10 µl
60 µl
10 µM Forward primer
1 µl
6 µl
10 µM Reverse primer
1 µl
6 µl
Water
3 µl
18 µl
Bacterial cells (template)
5 µl
-----
Cycle (Varies):
95° C | 5m     30x:         95° C | 45s         55° C | 30s         72° C | 1m per kb  72° C | 10m   4° C | Forever
load 5 µl onto gel
for all positive bands - take the rest of the bands and inoculate them into an overnight LB (+antibiotic) for miniprep
Colony PCR for Yeast & Patching
Number colonies
Patch on plate
Mix in 10 µl NaOH
Boil for 20m
PCR
Cycle (varies):
95° C | 5m 30x:         95° C | 45s         55° C | 30s         72° C | 1m per kb  72° C | 10m   4° C | Forever
Frozen Glycerol Stocks Yeast
Grow Overnight in YPD (2-5 mL) then dilute 1:20 in YPD, grow to OD 0.4-0.5
Add 350 µl cells to 350 µl sterile 60% glycerol in cryovial, vortex to mix, snap freeze in liquid nitrogen and store at -80° C (but actually we just stuck it in the freezer because nobody will let us use liquid nitrogen).
Home
Project
Background
Design
Achievements
Models
Judging
Medal Requirements
Safety
Implications
Human Practices
Super Science!
Collaborations
Notebooks & Protocols
Notebooks
Protocols
Team
Team Members
Advisors
Sponsors
Attributions
Contact
iGEM Profile
@iGEM_UCSF
UCSF iGEM
igem.ucsf.edu
© 2014 UCSF-UCB iGEM
Retrieved from " http://2014.igem.org/Team:UCSF_UCB/notebooks.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
