<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCSF_UCB_ianto"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCSF_UCB/ianto">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCSF_UCB/ianto&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF_UCB/ianto&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF_UCB/ianto&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCSF_UCB/ianto" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCSF UCB/ianto</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H2>MON 19 MAY 2014</H2><P>Cleaned up lab. Read lots of literature. </P><H2>TUE 20 MAY 2014</H2><P>Looking for alpha-factor responsive promoters. Results can be found in &quot;iGEM 2014/Parts and Plasmids/Mating Responsive Promoters/&quot;</P><H2>WED 21 MAY 2014</H2><P>Continuing search for more alpha-factor responsive promoters to add to the 16 that we have found so far. </P><P>Also, designing primers for the promoters that we have found so far. For primer design, see &quot;iGEM 2014/Protocols/cloning_DH5aT.docx&quot;</P><H2>MON 02 JUN 2014</H2><P>I. Calculated Ligation concentrations found in 140602 Ligation Calculation.xlsx and performed ligation reactions. </P><P>II. Also re-PCR'd promoters that did not work before. 
Please see paper notebook page 3 for gel map. Gel image: </P><H2>TUE 03 JUN 2014</H2><P>I. Re-PCR'd promoters that we did not do yesterday. See gel photos: </P><P>Also gel extracted. </P><P>II. Selected colonies and inoculated cultures. </P><H2>WED 04 JUN 2014</H2><P>I. Digest PCR reactions and PCR cleanup. Completed. 
Concentration of Promoter 6 is pretty bad compared to others. </P><P>II. Inoculated single colony plates. </P><H2>THURS 05 JUN 2014</H2><P>I. Eric and Derrick are retrying digestion of backbone for the third time after two failed attempts. 
Will try ligation and transformation into e. coli today. </P><P>II. Need to miniprep plates with single colonies. </P><P>III. Linearize DNA for Yeast Transformation
Linearized PRM1, SAG1, YDR124W. See concentrations here: 
</P><H2>FRI 06 JUN 2014</H2><P>I. Designed Gibson Primers and produced Promoter + rtTA ApE files. </P><P>II. Performed gel extraction of re-digested backbone. 
See gel photo below: . </P><P>See how the band is at the right place now at around 8kb, but that there isn't really a smaller band. Possible that the smaller band ran off the gel. </P><P>Overall: 
Found that most of the promoters worked out in the end with cloning. Ready for yeast transformation and integration into yeast genome. Now trying to figure out next steps. </P><H2>MON 09 JUN 2014</H2><P>I. iGEM Bootcamp started. </P><P>II. Minipreps of Transformations from attempt 2. 
Vacuum manifold does not clear PE Buffer as well as centrifuges. May have ended with more than 50microliters of product. Nanodrop showed that our concentrations are okay still. We are now sending out these samples for sequencing. </P><H2>TUE 10 JUN 2014</H2><P>I. iGEM Bootcamp. </P><P>II. Dual Digest of miniprepped backbones to remove GFP from built vectors → will digest overnight. </P><P>III. Worked on iGEM website. </P><P>IV. Sequencing came back and found that all miniprepped vectors turned out well → moving forward with yeast transformation and Dual digest for rtTA incorporation as seen in II. </P><P>Lecture on E. Coli and Yeast by Zairan Liu from UCSF. </P><H2>WED 11 JUN 2014</H2><P>I. CIP treatment of backbone
Lecture on Flow Cytometry by </P><P>Project introduction lecture - Kara.</P><P>Cell-cell communication lecture - Leo </P><P>Planned: 
II. Gibson reaction
III. Transformation into Bacteria</P><P>Brainstorm: 
Possible human practices: bands! Playing music without a conductor! Collective structures. The role of more effective communication, or communication training in establishing structures. </P><P>Possibility of not just excreting more Alpha factor, but increasing number of receptors. </P><H2>THURS 13 JUN 2014</H2><P>I. No colonies on plate. Have to re-gibson and re-transform. 
Lots of lectures </P><H2>FRI 13 JUN 2014</H2><P>I. Linearization of Promoter + GFP plasmids for yeast transformation 
II. Yeast transformation today, transformed 11 promoters + GFP into two different yeast strains. 
III. Field trip to NASA today. Meet up with Brown/Stanford/Spielgal iGEM team! </P><P>Questions: 
What is the difference between the two yeast strains that we transformed? </P><H2>MON 16 JUN 2014</H2><P>I. Colony PCR of E. Coli Transformations of Promoter + rtTA (9)
II. Colony PCR of Yeast Transformations of Promoter + GFP (11)
III. Over 100 lanes of gels to run today! Woooooooo!</P><P>IV. Inoculated Successful Colony PCRs. </P><P>Results of E Coli Colony PCR of rtTA + promoter 
</P><H2>TUE 17 JUN 2014</H2><P>I. Redo Yeast Colony PCRs that failed yesterday. 
II. Miniprep of E. Coli rtTA + promoters inoculated yesterday. </P><P>CB008 genotype: <EM>W303 MATa far1Δ his3 trp1 leu2 ura3</EM></P><P>CB008DB genotype: <EM>W303 MATa far1Δ his3 trp1 leu2 ura3 bar1</EM></P><P>Results of Yeast Colony PCR of GFP + Promoter in CB008: 
</P><H2>WED 18 JUN 2014</H2><P>I. Linearize promoter + rtTA and transform into Yeast
II. Inoculate and incubate promoter + GFP transformed yeast. Prepare patch plate with promoters that worked. 
III. Lincoln: miniprep constitutive + GFP vectors and send for sequencing. 
IV. Overnight cultures of alpha promoters CB008</P><H2>THUR 19 JUN 2014</H2><P>I. Dilute inoculated cultures. (2 hours)</P><BLOCKQUOTE><H3>Flow Cytometer Preparation</H3><P>Day Before:
start overnight cultures of your strains to be tested in SD complete media (NOT YPD, since it has fluorescence background and is problematic for FACS testing)</P><P>Day Of:
1. Dilute overnight cultures to final ~OD 0.05-0.1 in the 96 well shaker plate (the saturated overnight cultures should be ~OD 7, so this is approximately a 1:100 dilution. Also, where are the plates? If you don't know, ask and make a note.). Again, use SD complete media and the total volume of each well should be 1 mL. Make a plate map!</P><OL><LI><P>Allow cells to enter growth stage by putting on plate shaker for 3 hours at 1000 rpm, 30 degrees C.</P></LI><LI><P>Induce with alpha factor. The stock alpha factor in the freezer is 3 mM, and we used it at final concentrations of 0, 1 nM, 10 nM, 100 nM, and 1 uM. **<STRONG><EM>Please include your concentrations for making the dilutions we used to pipette so that you can quickly and easily refer to this for next time!</EM></STRONG>** Alpha factor cannot be refrozen, so throw away stocks after use.</P></LI><LI><P>Allow induction to proceed on plate shaker for 90 min to 2 hours, no longer.</P></LI><LI><P>Transfer 250 uL of each culture to the 96 well flow cytometry V-bottom plate using the multichannel pipette. Add 4µl of cyclohexamide for every 100µl of cells (10µl total) to kill cells and stop protein production. </P></LI><LI><P>Run on flow cytometer. Things you should note: </P></LI><LI><P>Check waste, turn machine on, check sheathing fluid.
Open FACS diva. Use iGEM account. </P></LI></OL><P>Make new plate or new experiment. </P><P>Make sure settings for each well is accurate. Ensure that pick up volume is not more than volume in each well, that the mixing volume is not more than volume in each well. </P><P>Make sure laser settings, voltage settings are correct. (View&gt;inspector)</P><P>FSC: 250
SSD: 280
FITC: 550
B: 650</P><P>Click blue button to tag selected wells as wells with samples to test. </P><P>Flow rate: 1µL/sec
Sample Volume: 200µL
Mixing Volume: 100µL
Mixing speed: 180µL/sec</P><P>Always run wells, not run plate. </P><P>FIRST what things you need to check on the machine before running!, how to open the iGEM account and create a new FACS experiment, voltage settings (and tips on how to set an appropriate voltage setting), how to select wells and alter the volume of sample, flow speed, mixing volume, etc, how to run the plate/wells, how to export data, and anything else you might need)</P><OL><LI>Analyze data. Notes on that from what you learned in FlowJo and MatLab today.</LI></OL></BLOCKQUOTE><P>II. Expose to alpha factor (~90 minutes)
(0, 10nM, 1µM, 10µM, 100µM)</P><P><STRONG>Correction</STRONG> 24 JUNE 2014: (0, 1nM, 10nM, 100nM, 1µM)</P><P>Flow Cytometer 1-4pm. </P><P>How to use flow cytometer: 
1. Check the sheathing fluid levels!
2. Press run 
3. On computer, use flowjo. 
4. Check voltages! (in notebook)
5. Select wells
6. run!
7. Make sure you see a good number of events. </P><P>PRELIM Plate map: </P><PRE><CODE>    1   2   3   4   5   6   7   8   9   10  11  12
A   [------NEG------]       [------NEG-------]
B   [------PRM2-----]       [------PRM1------]
C   [------ASG7-----]       [------EMC18-----]
D   [------PCL2-----]       [------PRM3------]
E   [------NEG------]       [------SAG1------]
F   [------CLG1-----]
G   [------YDR124W--]
H   [------PRM6-----]
</CODE></PRE><P>in DB008</P><PRE><CODE>    1   2   3   4   5   6   7   8   9   10  11  12
A   [------NEG------]       [------NEG-------]
B   [------yGEM10---]       [------yGEM14----]
C   [------yGEM4----]       [------yGEM15----]
D   [------yGEM5----]       [------yGEM16----]
E   [------NEG------]       [------yGEM6-----]
F   [------yGEM11---]
G   [------yGEM12---]
H   [------yGEM13---]
</CODE></PRE><P>I will be working with YDR124W and PRM6.</P><P>III. Prepare glycerol stocks of DB promoter + GFP yeast strains. 
IV. Pick and inoculate cultures with promoter + rtTA again since most of the sequencing failed. Will need to re-linearize most of the yeast. </P><H2>FRI 20 JUN 2014</H2><P>I. Leaving early today for Doctor's appointment. 
II. Minipreps for rtTA+promoter. </P><H2>MON 23 JUN 2014</H2><P>I. Found that the rtTA that we cloned were incorrect. Did not including activating domain. 
II. Inoculating HYM1 DH5alpha + GFP again for more plasmid. </P><P>Now working on Flow Cytometry data analysis of alpha responsive promoters. 
Now working on pTET/GFP/mFalpha. p2A? </P><H2>TUE 24 JUN 2014</H2><P>I. Literature search on feasibility of P2A incorportation of alpha factor and other sequences. 
II. FlowJo notes: </P><PRE><CODE>Import data. Save in save folder on desktop. 
Double click to open a sample. 
Gate the sample population. 
Activate gate: gives a percentage of samples in gate. 

Click on sample list and click on gate name for analysis on just gated region. 

Save workspaces so that you can continue analysis later on.

Displaying mean: click on statistics and click ∑ for which stats that you want. 

Look through individual samples for obscurities. 

Matlab: comment with what workspace your data comes from. 

Enter in data by hand. 
</CODE></PRE><P>Analysis of Promoters+GFP in different alpha factor concentrations: 
</P><P>Translation: 
Name    Promoter    PREs
yGEM4:  ASG7        2
yGEM5:  PCL2        5
yGEM6:  SAG1        4
yGEM10: PRM2        2
yGEM11: CLG1        2
yGEM12: YDR124W     2
yGEM13: PRM6        2
yGEM14: PRM1        2
yGEM15: ECM18       1
yGEM16: PRM3        3</P><P>Continue to learn canvas! 
Game brainstorming: recording how many rounds someone plays a game depending on how much on succeeds. Simulates positive feedback in gameplay. Building of circuits of games. </P><H2>WED 25 JUN 2014</H2><P>I. Dilution of CB008 and CB008DB for yeast transformation of pTET<EM>GFP today @ ~1pm. 
    Need to linearize pTET</EM>GFP for transformation later today. </P><P>Given two vectors:  hy86E3(pTET<EM>GFP LEU2)
                    pTS97(pAGA1</EM>GFP URA3)</P><UL><LI>linearization @ 37°C started at 11:57am → 12:57pm</LI><LI>No need to PCR cleanup</LI><LI>Straight to transformation. </LI></UL><BLOCKQUOTE><H3>Yeast Transformation Protocol</H3><P><STRONG>T-(1 day)</STRONG>
Grow yeast strains in 5-10ml YPD overnight at 30°C</P><P><STRONG>T-(2–4hours)</STRONG>
Dilute cultures ~1:20 in YPD. 
Grow for 2-4 hours @ 30°C to OD_600 = 0.4-0.6. 
<EM>Make sure to inoculate 2.5ml per transformation reaction.</EM></P><P><STRONG>T-(20 minutes)</STRONG>
Boil salmon sperm DNA for 10 minutes and cool on ice for 10 minutes.
(This can be done in the thermocycler with the ssDNA Heat-Freeze protocol)
Thaw DMSO!
Pellet cells (3000rpm 2-5min). Discard supernatant. 
Wash with 1:1 culture volume of LOATE(0.1M LiOAc in TE) or water. 
Pellet cells (3000rpm 2-5min). Discard supernatant. 
Resuspend pellet in 100µl per reaction (aka per 2.5mL of culture volume) 
Aliquot 100µl into each transformation reaction tube. 
To each tube of 100µL yeast, add:    </P><UL><LI>10µL of ssDNA</LI><LI>8µL of target DNA(Hyun's standard amount after linearization)</LI><LI>480µL of 50% PEG 3350</LI><LI>60µL TE(10XTE)</LI><LI>60µL of LOA(1M LiOAc)</LI><LI>75µL of DMSO</LI></UL><P>Vortex!
Incubate at 42°C for 30 minutes. </P><P><STRONG>T+(30 minutes)</STRONG>
Pellet (6000rpm for 2 minutes). Use pipette to remove media!
Resuspend in 500µL YPD (or selective media)
Pellet (6000rpm for 2 minutes). Discard media. 
Resuspend in residual ~50µL of YPD. 
Plate on selective media. 
Incubate for 1-3 days at 30°C. </P><P><STRONG>T+(1-3 days)</STRONG>
Pick colonies, colony PCR, or continue with experiment in some other way. </P></BLOCKQUOTE><P>Incubation at 42°C started at 1:56pm → 2:26pm. </P><P>II. Until then, still learning more javascript and canvas. </P><P>III. Inoculate yeast strains for flow cytometry tomorrow. Do in triplicates tomorrow!
Measuring the following promoters+GFPs in triplicate tomorrow (start @ 8am): </P><UL><LI>PRM2</LI><LI>ASG7</LI><LI>PCL2</LI><LI>CLG1</LI></UL><P>IV. Design gibson primers for <STRONG>constitutive promoters + rtTA</STRONG></P><H2>THURS 26 JUN 2014</H2><P>Alpha factor: 
3mM starting concentration. 30µL in them. 
0, 0.5nM, 1nM, 10nM, 100nM, 1000nM, 3000nM</P><P>Make 100X stocks of each concentration and aliquot 10µL into each well. </P><P>ROUND 2 Promoter Map: </P><PRE><CODE>Plate 1:
    H   G   F   E   D   C   B   A   
1       [--------CB008-1--------]
2       [--------CB008-2--------]
3       [--------CB008-3--------]
4       [--------PRM2-1---------]
5       [--------PRM2-2---------]
6       [--------PRM2-3---------]
7       [--------ASG7-1---------]
8       [--------ASG7-2---------]
9       [--------ASG7-3---------]
10      [--------PCL2-1---------]
11      [--------PCL2-2---------]
12      [--------PCL2-3---------]
Plate 2: 
   H    G   F   E   D   C   B   A   
1       [--------CLG1-1---------]
2       [--------CLG1-2---------]
3       [--------CLG1-3---------]
4
5
6
7
8
9
10
11
12
</CODE></PRE><P>Input into 30°C + 1000rpm shaker @ 8:14am -&gt; 11:14am</P><P>Plate 1 &amp; 2 Alpha Factor Concentration Map: </P><PRE><CODE>    1   2   3   4   5   6   7   8   9   10  11  12
A   [-------------------{0 nM}-------------------]
B   [-------------------{0.5 nM}-----------------]
C   [-------------------{1 nM}-------------------]
D   [-------------------{10 nM}------------------]
E   [-------------------{100 nM}-----------------]
F   [-------------------{1000 nM}----------------]
G   [-------------------{3000 nM}----------------]
H   
</CODE></PRE><P>Induce with alpha factor for 90 minutes. </P><P>Make 100X stocks: 
0nM
50nM
100nM
1000nM
10000nM
100000nM
300000nM</P><P>3000000nM -&gt; 30000nM</P><P>Added alpha factor 11:45am -&gt; 1:15pm</P><P>II. Second round of alpha factor promoter + GFP characterization.
Now characterizing: </P><UL><LI>YDR124W</LI><LI>PRM6</LI><LI>PRM1</LI><LI>ECM18</LI><LI>PRM3</LI><LI>SAG1</LI></UL><P>[Plate maps updated in FRI Jun 27 below]~~PLATE 1 PROMOTER MAP </P><PRE><CODE>    H   G   F   E   D   C   B   A   
1       [--------CB008-1--------]
2       [--------CB008-2--------]
3       [--------CB008-3--------]
4       [--------YDR124W-1------]
5       [--------YDR124W-2------]
6       [--------YDR124W-3------]
7       [--------PRM6-1---------]
8       [--------PRM6-2---------]
9       [--------PRM6-3---------]
10      [--------PRM1-1---------]
11      [--------PRM1-2---------]
12      [--------PRM1-3---------]
</CODE></PRE><P>PLATE 2 PROMOTER MAP</P><PRE><CODE>   H    G   F   E   D   C   B   A   
1       [--------ECM18-1--------]
2       [--------ECM18-2--------]
3       [--------ECM18-3--------]
4       [--------PRM3-1---------]
5       [--------PRM3-2---------]
6       [--------PRM3-3---------]
7       [--------SAG1-1---------]
8       [--------SAG1-2---------]
9       [--------SAG1-3---------]
10
11
12~~
</CODE></PRE><P>PLATE 1 &amp; 2 ALPHA FACTOR CONCENTRATION MAP: same as before</P><P>III. Dilution of FW pTEF1 + rtTA primers
    1. Resuspend in 10µL * amount ng of DNA. 
    2. 1/10 dilution of stock made in (1)</P><H2>FRI 27 JUN 2014</H2><P>7:55AM: 1/100 dilutions of alpha responsive promoters + GFP for flow cytometry later today. </P><P>PRM6+GFP did not grow up yesterday. Updated Plate maps below: </P><P>ROUND 3 PLATE 1 PROMOTER MAP</P><PRE><CODE>    H   G   F   E   D   C   B   A   
1       [--------CB008-1--------]
2       [--------CB008-2--------]
3       [--------CB008-3--------]
4       [--------YDR124W-1------]
5       [--------YDR124W-2------]
6       [--------YDR124W-3------]
7       [--------PRM3-1---------]
8       [--------PRM3-2---------]
9       [--------PRM3-3---------]
10      [--------PRM1-1---------]
11      [--------PRM1-2---------]
12      [--------PRM1-3---------]
</CODE></PRE><P>~~ROUND 2 PLATE 2 PROMOTER MAP [DID NOT COMPLETE]</P><PRE><CODE>   H    G   F   E   D   C   B   A   
1       [--------ECM18-1--------]
2       [--------ECM18-2--------]
3       [--------ECM18-3--------]
4       [--------SAG1-1---------]
5       [--------SAG1-2---------]
6       [--------SAG1-3---------]
7
8
9
10
11
12~~
</CODE></PRE><P>8:10AM: 1/100 dilutions set in 30°C room, shaking at 1000RPM. Incubate until 11:10AM. </P><P>8:45AM: Set up and started PCR for rtTA + Constitutive Promoter_SV606 homology. End time: 9:40AM. </P><P>Need to continue on with Gel verification and PCR purification. (Jessica will include this in her gel samples.)</P><P>9:47AM</P><BLOCKQUOTE><H3>Flow Cytometer Start Up Procedure</H3><OL><LI>30 minutes prior, push big green start button on right side of machine. Make sure machine is on &quot;Stand-by.&quot;</LI><LI>Turn machine to &quot;Run.&quot;</LI><LI>Re-initialize HTS</LI><LI>Prime three times</LI><LI>Run CST Beads

</LI><LI>250µL sheath fluid into A1 of a 96 well plate. </LI><LI>Load A'1 - A'4 with Bleach and B'1-B'4 with Water. (Opposite corner: H12 = A'1). </LI><LI>Cytometer &gt; CST</LI><LI>Make sure Cytometer Performance Results: Passed</LI><LI>Beads</LI><LI>Kept in fridge in flow cytometry room. </LI><LI>Make sure lot number in CST program matches lot number on box. </LI><LI>Shake bottle</LI><LI>One drop of bead bottle into A1 with sheath fluid</LI><LI>Load into machine and press run </LI></OL><LI>Run clean plate

</LI><LI>Click on experiment. Experiment &gt; Open experiment. </LI><LI>Open clean plate &quot;Daily Clean - 96 well U-bottom.&quot;</LI><LI>Be here to see how clean the machine is. Under 100 events/sec max!</LI><LI>If over 100 events/sec, run another clean plate!</LI><LI>View events on acquisition dashboard in View &gt; Acquisition Dashboard. </LI><LI>Can also make a clean plate using HTS &gt; Clean…&gt;</LI></BLOCKQUOTE><P>11:18AM: Gel Photo of rtTA + TEF1 Homology PCR
</P><P>11:37AM: Second round of Alpha Response Promoter + GFP characterization induced with alpha factor -&gt; 1:07pm</P><P>1:07PM: Treated each sample with cyclohexamide. Running through FACS right now. </P><P>1:20PM: Streaked out pAGA_GFP for single colonies. Not sure why plate overgrew so much. </P><P>1:30PM: Worked on FlowJo to extract data from fcs files. </P><P>4:56PM: Finished with analysis. Created program to easily import data from FACS. </P><H2>MON 30 JUN 2014</H2><P>Redoing FACS plate two from Friday since it was left for over two days. </P><P>ROUND 4 PROMOTER MAP</P><PRE><CODE>   H    G   F   E   D   C   B   A   
1       [--------ECM18-1--------]
2       [--------ECM18-2--------]
3       [--------ECM18-3--------]
4       [--------SAG1-1---------]
5       [--------SAG1-2---------]
6       [--------SAG1-3---------]
7       [--------CB008-1--------]
8       [--------CB008-2--------]
9       [--------CB008-3--------]
10
11
12
</CODE></PRE><P>Constitutive Promoter PLATE MAP</P><PRE><CODE>    1   2   3   4   5   6   7   8   9   10  11  12
A   [-CB008-]       
B   [-pTEF1-]       
C   [-m3----]       
D   [-m6----]       
E   [-m7----]       
F   [-m10---]
G   
H   
</CODE></PRE><P>Will be able to run Flow on Wednesday. </P><P>II. Streaked out for biological replicate inoculation on wednesday. </P><P>III. Analysis of Today's AFRP Plate
</P><H2>TUE 01 JUN 2014</H2><P>Meeting with Anusuya</P><UL><LI>Verse yourself in Alpha Factor Pathway. Make barebones alpha factor slide. </LI><LI>Make content folders for website. Collect pieces of information right now. </LI><LI>Characterize promoters by time exposure. </LI><LI>Use Hill curves to fit. Take parameters from curves for models. </LI><LI>What in nature uses positive feedback loops. Comparison to natural systems. </LI></UL><P>anusuyar@berkeley.edu</P><P>Analyzed data from Constitutive Promoters + GFP: 
</P><H2>WED 02 JUN 2014</H2><P>ROUND 5 PROMOTER MAP</P><PRE><CODE>Plate 1: 
   H    G   F   E   D   C   B   A   
1       [--------CB008-1--------]
2       [--------CB008-2--------]
3       [--------CB008-3--------]
4       [--------PRM2-1---------]
5       [--------PRM2-2---------]
6       [--------PRM2-3---------]
7       [--------ASG7-1---------]
8       [--------ASG7-2---------]
9       [--------ASG7-3---------]
10      [--------PCL2-1---------]
11      [--------PCL2-2---------]
12      [--------PCL2-3---------]

Plate 2: 
   H    G   F   E   D   C   B   A   
1
2
3
4       [--------YDR124W-1------]
5       [--------YDR124W-2------]
6       [--------YDR124W-3------]
7       [--------PRM6-1---------]
8       [--------PRM6-2---------]
9       [--------PRM6-3---------]  
10
11
12
</CODE></PRE><P>Alpha Concentrations Flipped Today</P><PRE><CODE>    1   2   3   4   5   6   7   8   9   10  11  12
A   [-------------------{3000 nM}----------------]
B   [-------------------{1000 nM}----------------]
C   [-------------------{100 nM}-----------------]
D   [-------------------{10 nM}------------------]
E   [-------------------{1 nM}-------------------]
F   [-------------------{0.5 nM}-----------------]
G   [-------------------{0 nM}-------------------]
H   
</CODE></PRE><P>For CB008-1, PRM2-1:3, CLG1-1:3, added 20µL instead of 10µL to 1mL due to low density of cultures. Possibly due to selection of very small colonies. </P><P>9:07AM - Started inoculation of 1/100 dilutions. </P><P>9:37AM - Plates shaking at 1000RPM in 30°C room. </P><P>12:37PM - Alpha Factor Exposures Start
12:57PM - Finished Alpha Factor Exposures</P><P>2:37PM - Cyclohexamide treatment</P><P>3:00PM - Flow Cytometry Plate Reading</P><P>6:00PM - Finish Plate Reading </P><P>Alpha Factor Concentration Map, same as before. </P><H2>THURS 03 JUN 2014</H2><P>Sick today :/ Tummy hurts</P><P>See below for repeat of AFRP GFP measurements in biological replicates. </P><P>Some questions that we still need to answer: </P><OL><LI><STRONG>Why yeast instead of bacteria?</STRONG> Other studies have made similar circuits in bacteria before (see &quot;Building Biological Memory by Linking Positive Feedback Loops&quot; by Dong-Eun Chang et al.) The argument that yeast are slightly more representative of eukaryotic cells here seems far-fetched as we are not really looking at anything endogenous to yeast. Not that I think we should drop everything and start working in bacteria. We should have a solid response to this though. </LI></OL><H2>SAT 05 JUL 2014</H2><P>Consolidation of Flow Experiments</P><P>Prelim
ASG7
PCL2
SAG1
PRM2
CLG1
YDR124W
PRM6
PRM1
ECM18
PRM3</P><P>ROUND 2
PRM2
CLG1
ASG7
PCL2</P><P>ROUND 3
PRM1 
PRM3
YDR124W
<EM>PRM6 did not complete</EM></P><P>ROUND 4
SAG1
ECM18</P><P>ROUND 5
PRM2
ASG7
PCL2
YDR124W
PRM6</P><P>ROUND 6
PRM1
ECM18
PRM3
SAG1
<EM>AGA1</EM>
PRM2
CLG1</P><H2>MON 07 JUN 2014</H2><P>Alpha Concentration Map: same as before </P><P>Updated ROUND 6 Plate 2mL Growth Plate Map: </P><PRE><CODE>Plate 1: 
   H    G   F   E   D   C   B   A  
1       [--------CB008-1--------]
2       [--------CB008-2--------]
3       [--------CB008-3--------]
4
5
6
7
8       [--------ECM18-1--------]
9       [--------ECM18-2--------]
10      [--------ECM18-3--------]
11      
12   

Plate 2: 
   H    G   F   E   D   C   B   A   
1       [--------SAG1-1---------]
2       [--------SAG1-2---------]
3       [--------SAG1-3---------]
4       [--------AGA1-1---------]
5       [--------AGA1-2---------]
6       [--------AGA1-3---------]
7       [--------PRM2-1---------]
8       [--------PRM2-2---------]
9       [--------PRM2-3---------]
10      [--------CLG1-1---------]
11      [--------CLG1-2---------]
12      [--------CLG1-3---------]

Plate 3
  1   2   3   4   5   6   7   8   9   10  11  12
A [-CB008-]   ⨅   ⨅   ⨅   ⨅   ⨅   ⨅
B [-pTEF1-]   |  PRM1 |   |  PRM3 | 
C [-m3----]   |   |   |   |   |   |
D [-m6----]   1   2   3   1   2   3  
E [-m7----]   |   |   |   |   |   |
F [-m10---]   |   |   |   |   |   |
G             ⨆   ⨆   ⨆   ⨆   ⨆   ⨆
H   
</CODE></PRE><P><EM>For PRM2, we added 100µL of overnight yeast due to OD600 of 0.3. All others added 10µL to 1mL of SD complete</EM></P><P>Incubation at 30°C 1000RPM at 9:45am -&gt; 12:45PM </P><P>Exposure to Alpha factor at 1:05PM -&gt; 2:35PM. </P><P>ROUND 6 96 Well Reading Map</P><PRE><CODE>Plate 1: 
   H    G   F   E   D   C   B   A  
1       [--------CB008-1--------]
2       [--------CB008-2--------]
3       [--------CB008-3--------]
4       [--------PRM1-1---------]
5       [--------PRM1-2---------]
6       [--------PRM1-3---------]
7       [--------ECM18-1--------]
8       [--------ECM18-2--------]
9       [--------ECM18-3--------]
10      [--------PRM3-1---------]
11      [--------PRM3-2---------]
12      [--------PRM3-3---------]   

Plate 2:
   H    G   F   E   D   C   B   A   
1       [--------SAG1-1---------]
2       [--------SAG1-2---------]
3       [--------SAG1-3---------]
4       [--------AGA1-1---------]
5       [--------AGA1-2---------]
6       [--------AGA1-3---------]
7       [--------PRM2-1---------]
8       [--------PRM2-2---------]
9       [--------PRM2-3---------]
10      [--------CLG1-1---------]
11      [--------CLG1-2---------]
12      [--------CLG1-3---------]

Plate 3: 
  1   2   3   4   5   6   7   8   9   10  11  12
A [-CB008-]  
B [-pTEF1-]   
C [-m3----]   
D [-m6----] 
E [-m7----]   
F [-m10---]   
G             
H   
</CODE></PRE><P>Combined analysis of most* promoters: 

*Does not include PRM6 or AGA1! </P><H2>TUE 08 JUL 2014</H2><P>Out sick :(</P><H2>WED 09 JUL 2014</H2><P>Out sick :(
To do: </P><P>-Website stuff
-Modeling stuff -&gt; make graphs out of data from found functions.</P><H2>THURS 10 JUL 2014</H2><P>Results of Constitutive Promoters
</P><P>Results of Inducible Promoters Second set 2 of try 2
</P><P>Today: 
Recharacterizing inducible promoters in CB008DB strains</P><P>GFP = before signal processing
RFP = after signal processing</P><P>Flow with AFRP in CB008DB cells</P><P>plate 1</P><PRE><CODE>    H   G   F   E   D   C   B   A   
1       [--------CB008DB-1------]
2       [--------CB008DB-2------]
3       [--------CB008DB-3------]
4       [--------CLG1-1---------]
5       [--------CLG1-2---------]
6       [--------CLG1-3---------]
7       [--------PRM2-1---------]
8       [--------PRM2-2---------]
9       [--------PRM2-3---------]
10      [--------PCL2-1---------]
11      [--------PCL2-2---------]
12      [--------PCL2-3---------]
</CODE></PRE><P>Plate 2</P><PRE><CODE>        H   G   F   E   D   C   B   A   
1       [--------SAG1-1------------]
2       [--------SAG1-2------------]
3       [--------SAG1-3------------]
4       [--------AGA1-1------------]
5       [--------AGA1-2------------]
6       [--------AGA1-3------------]
7       
8       
9       
10      
11      
12      
</CODE></PRE><P>Results of Inducible Promoters in DB: 
</P><H2>FRI 11 JUL 2014</H2><P>AFRP GFP DB ROUND 2</P><PRE><CODE>    H   G   F   E   D   C   B   A   
1       [--------CB008DB-1------]
2       [--------CB008DB-2------]
3       [--------CB008DB-3------]
4       
5       [--------PRM1-1---------]
6       [--------PRM1-2---------]
7       [--------PRM1-3---------]
8       [--------ECM18-1--------]
9       [--------ECM18-2--------]
10      [--------ECM18-3--------]
11      
12      
</CODE></PRE><P>Today: </P><UL><LI>Presentation for Wendell on Monday</LI><LI>Analyze Flow Data from today </LI><LI>Start modeling previous data </LI></UL><H2>MON 14 JUL 2014</H2><P>Group Meeting today. </P><P>Prep for flow cytometry tomorrow. Waiting on Colony PCR of transformants to ensure correct insertions. Will be characterizing yeast strains with pTET<EM>GFP and pTEF1</EM>rtTA under 12 different concentrations of Doxycycline. </P><H2>TUE 14 JUL 2014</H2><P>Flow with AFRP in CB008DB cells</P><PRE><CODE>    H   G   F   E   D   C   B   A   
1       [--------CB008DB-1------]
2       [--------CB008DB-2------]
3       [--------CB008DB-3------]
4       [--------ASG7-1---------]
5       [--------ASG7-2---------]
6       [--------ASG7-3---------]
7       [--------YDR124W-1------]
8       [--------YDR124W-2------]
9       [--------YDR124W-3------]
10      [--------PRM6-1---------]
11      [--------PRM6-2---------]
12      [--------PRM6-3---------]
</CODE></PRE><P>Alpha factor concentrations same as before
9:40AM Dilution -&gt; 12:40PM Induction with Alpha Factor</P><P>Analysis of FSC-SSC plots from 140715 show significant oddities. </P><P><STRONG>Wells with &quot;S Curve&quot; in SSC by FSC plot</STRONG></P><PRE><CODE>  1   2   3   4   5   6   7   8   9   10  11  12
A X
B X
C X
D X
E X
F X            
G X
H
</CODE></PRE><P><STRONG>Wells with &quot;Contamination&quot; in SSC by FSC plot</STRONG></P><PRE><CODE>  1   2   3   4   5   6   7   8   9   10  11  12
A          
B 
C 
D         X   X   X       X   X
E         X       X   X       X
F         X       X   X   X   X
G                     X   X   X   X 
H   
</CODE></PRE><P>Conclusion: will redo this set of promoters tomorrow. </P><H2>WED 15 JUL 2014</H2><P>[CB008 CB008DB] + pTEF1<EM>rtTA + pTET</EM>GFP</P><P>Flow plate maps today: 
Doxycycline concentrations: </P><PRE><CODE>        H   G   F   E   D   C   B   A   
1               [0 µg/ml------------]
2               [0.03 µg/ml---------]
3               [0.06 µg/ml---------]
4               [0.09 µg/ml---------]
5               [0.3 µg/ml----------]
6               [0.6 µg/ml----------]
7               [0.9 µg/ml----------]
8               [3.0 µg/ml----------]
9               [6.0 µg/ml----------]
10              [9.0 µg/ml----------]
11              [30 µg/ml-----------]
12              [60 µg/ml-----------]
</CODE></PRE><P>pTEF1 promoters</P><PRE><CODE>  1   2   3   4   5   6   7   8   9   10  11  12
A [---------------[CB008]----------------------]  
B [---------------[CB008 pTET_GFP]-------------]
C [---------------[CB008 pTET_GFP pTEF1_rtTA]--]  
D [---------------[CB008 pTET_GFP pTEF1 m6]----]     
E [---------------[CB008 pTET_GFP pTEF1 m7]----]    
F [---------------[CB008 pTET_GFP pTEF1 m10]---]  
G
H

  1   2   3   4   5   6   7   8   9   10  11  12
A [---------------[CB008DB]--------------------]  
B [---------------[CB008DB pTET_GFP]-----------]  
C [---------------[CB008DB pTET_GFP pTEF1 1of2 ] 
D [---------------[CB008DB pTET_GFP pTEF1 2of2 ]
E [---------------[CB008DB pTET_GFP pTEF1 m6]--] 
F [---------------[CB008DB pTET_GFP pTEF1 m7]--]             
G 
H
</CODE></PRE><P>1/100 dilution of cultures started at 7:25AM -&gt; 10:25AM</P><P>Dilutions</P><PRE><CODE>Stock: 100mg/mL 
**1x**           **100x**         **Ratios**      **Prep**
60µg/mL       A: 6mg/mL           06% S       60µL of Stock in 940µL of water
30µg/mL       B: 3mg/mL           03% S       30µL of Stock in 970µL of water  
9µg/mL        C: 900µg/mL         15% A       150µL of A in 850µL of water  
6µg/mL        D: 600µg/mL         10% A       100µL of A in 900µL of water
3µg/mL        E: 300µg/mL         10% B       100µL of B in 900µL of water
0.9µg/mL      F: 90µg/mL          10% C       100µL of C in 900µL of water
0.6µg/mL      G: 60µg/mL          10% D       100µL of D in 900µL of water
0.3µg/mL      H: 30µg/mL          10% E       100µL of E in 900µL of water
0.09µg/mL     I: 9µg/mL           10% F       100µL of F in 900µL of water
0.06µg/mL     J: 6µg/mL           10% G       100µL of G in 900µL of water
0.03µg/mL     K: 3µg/mL           10% H       100µL of H in 900µL of water
0             N/A                 N/A         N/A
</CODE></PRE><P>Induction with Doxy at 11:16AM -&gt; 5:16PM</P><H2>THURS 17 JUL 2014</H2><P>TODO: </P><UL><LI>[X] Working on generating noise graphs for Hyun

</LI><LI>[X] Need to ensure that all the data is correct and current. Will need to redownload all files. Ugh.</LI><LI>[X] Need to decide if I should <STRONG>first divide SD by Mean GFP then average triplicates</STRONG> or first average SD and Mean GFP triplicates and then divide. </LI><LI>[X] Need to also understand if we are using Geometric or <STRONG>Arithmetic Means</STRONG></LI></UL><LI>[X] Generate <STRONG>pTEF<EM>rtTA + pTET</EM>GFP v. Doxy</STRONG> graphs. 

</LI><LI>[X] Confirm which plate is which strain</LI><P><STRONG>Meeting with Anasuya</STRONG></P><UL><LI>[ ] Understanding background 

</LI><LI>[ ] Focus on how T-reg cells interact with macrophages. Macrophages can play a suppressive role on T-regs. 

</LI><LI>[ ] Dendritic cells and T-reg cells as an example of a divergent system. </LI><LI>[ ] Stanford iGEM 2010: interaction between t-reg and t-helper cells. </LI><LI>[ ] How microglia interact with different cell types: might play a role in regulating T-helper cells. </LI></UL><LI>[ ]  Non-immune standpoint

</LI><LI>[ ] Examples of how cells respond and coordinate using secondary post-cellular signaling. </LI><LI>[ ] <STRONG>OVERALL GOAL: Understand how your work can help elucidate and understand other cell/organism interactions through community signaling</STRONG></LI><LI>[ ] Model </LI><LI>[ ] Website

</LI><LI>[ ] Generate content for website. </LI><H2>FRI 18 JUL 2014</H2><UL><LI>[ ] Model for pTEF<EM>rtTA pTET</EM>GFP pTET<EM>mFα ARFP</EM>mFα circuit

</LI><LI>[ ] Functions

</LI><LI>[ ] Develop function to model pTEF<EM>rtTA -&gt; pTET</EM>GFP/pTET_mFα. 

</LI><LI>[ ] Need to understand how [Doxycycline] alters [α]. Does [GFP] correlate directly with [α]? </LI></UL><LI>[X] Develop function to model AFRP_RFP </LI><LI>[ ] Develop function for Diffusion of Alpha factor as as F(Initial Alpha Factor Concentration, Time, Distance)</LI><LI>[X] Data for Model

</LI><LI>[X] Export all sigmoid fits</LI><P>Conclusions from today: 
To understand Alpha Factor Secretion, need to understand how [Doxy] -&gt; pTEF1<EM>rtTA -&gt; pTET</EM>GFP results translate to [Doxy] -&gt; pTEF1<EM>rtTA -&gt; pTET</EM>mFα -&gt; [α].  Need this circuit: ([Doxy] -&gt; pTEF1<EM>rtTA -&gt; pTET</EM>mFα -&gt; [α]) + (pAGA1<EM>RFP -&gt; [RFP]) and then use ([α] -&gt; pAGA1</EM>GFP -&gt; [GFP]) measurements to understand the local concentrations of Alpha Factor from each cell. </P><P>Need to construct: ([Doxy] -&gt; pTEF1<EM>rtTA -&gt; pTET</EM>GFP + pTET<EM>MFα -&gt; pAGA1</EM>RFP -&gt; [RFP])</P><P>Questions for Hyun: </P><UL><LI>How does one find the phenomological alpha factor secretion from the data that we currently have: [Doxy] -&gt; (pTEF1<EM>rtTA + pTET</EM>GFP) -&gt; [GFP] and [α] -&gt; (AFRP_GFP) -&gt; [GFP].

</LI><LI>Even though there seems to be a one-to-one correlation between [GFP] and [α], leading us to an understanding of local [α], what does that say about [rtTA-Dox] -&gt; pTET_GFP production of GFP, and eventually [α]? </LI></UL><H2>SAT 19 JUL 2014</H2><P>See checklist in ##FRI 18 JUL 2014. </P><H2>MON 21 JUL 2014</H2><P>Working on Group Presentation. Made graphics for yeast modeling. </P><P>Working on two cell models. </P><H2>TUES 22 JUL 2014</H2><P>100x dilutions done at 9:20am
9:20am -&gt; 12:20pm</P><P>Plate 1:</P><PRE><CODE>    H           G   F   E   D   C   B   A   
1   [pTEF1-1]   [--------CB008DB-1------]
2   [pTEF1-2]   [--------CB008DB-2------]
3   [pTEF1-3]   [--------CB008DB-3------]
4   [M7-1]      [--------ASG7-1---------]
5   [M7-2]      [--------ASG7-2---------]
6   [M7-3]      [--------ASG7-3---------]
7               [--------YDR124W-1------]
8               [--------YDR124W-2------]
9               [--------YDR124W-3------]
10              [--------PRM6-1---------]
11              [--------PRM6-2---------]
12              [--------PRM6-3---------]
</CODE></PRE><P>Plate 2:</P><PRE><CODE>    H   G   F   E   D   C   B   A   
1       [--------AGA1-1---------]
2       [--------AGA1-2---------]
3       [--------AGA1-3---------]
4       [--------CLG1-1---------] did not grow as much as the others
5       [--------CLG1-2---------] 20ul instead of 10ul for CLG1
6       [--------CLG1-3---------]
7       [--------PRM2-1---------]
8       [--------PRM2-2---------]
9       [--------PRM2-3---------]
10  [--------PCL2-1---------]
11      [--------PCL2-2---------]
12      [--------PCL2-3---------]
</CODE></PRE><P>Plate 3:</P><PRE><CODE>    H    G   F   E   D   C   B   A   
1        [--------ECM18-1--------]
2        [--------ECM18-2--------]
3        [--------ECM18-3--------]
4        [--------SAG1-1---------]
5        [--------SAG1-2---------]
6        [--------SAG1-3---------]
7        
8        PRM3 MIA
9        
10       [--------PRM1-1---------]
11       [--------PRM1-2---------]
12       [--------PRM1-3---------]
</CODE></PRE><H2>WED 23 JUL 2014</H2><P>Reading up on Stochastic Modeling of our genetic circuits</P><P>Illustrations and style design of website and poster materials</P><P>TODO: </P><UL><LI>[ ] Make signs by Friday for Exploratorium Exhibition</LI><LI>[ ] Make pPCL2<EM>BFP and pAGA1</EM>BFP

</LI><LI>[X] Transform BFP to make more of it. Grow up, miniprep. (Will be transforming two more backbones with antibiotic resistance)</LI><LI>[X] Digest BFP using Xho1 and Not1</LI><LI>[X] Digest pAGA1<EM>RFP and pPCL2</EM>RFP with Xho1 and Not1. </LI><LI>[ ] Ligate BFP with pAGA1<EM>backbone and pPCL2</EM>backbone. </LI></UL><H2>THUR 24 JUL 2014</H2><P>See todo list in 23 JUL 2014</P><BLOCKQUOTE><P><STRONG>E. Coli Transformation Protocol</STRONG></P><P><EM>Materials</EM>
DNA       1µL
DH5α      50µL</P><P>Add DNA and DH5α.
Let sit on ice for 10 mins if whole plasmid/30 min if ligation reaction</P><P>Heat shock @ 42°C for 45 seconds</P><P>Add 250µL of SOC. </P><P>Incubate at 37°C for 1 hour (shaking)</P><H2>Plate on appropriate plate. </H2><P>Total Time: 80-100 minutes</P></BLOCKQUOTE><P>Incubated at 10:30AM -&gt; 11:30PM</P><H2>FRI 25 JUL 2014</H2><P>Pick colonies for miniprep on Monday -&gt; refrigerated and then set to grow on Sunday. Minipreps today. </P><P>Developed design guide for website and other materials</P><P>Using 4µL of pAGA1_RFP on Monday </P><P>Set up Balsamiq for the team. Designing flyers for Exploratorium event. </P><H2>MON 28 JUL 2014</H2><P>Miniprep transformations of BFP and shuttle vectors. - Jeffrey</P><P>Prepare for presentation today.</P><P>Digest pTS94(BFP), pGEM22(pAGA1+mCherry), and (pPCL2+mCherry)x2 with Xho1 and Not1 double overnight digest @ 37°C overnight.</P><H2>TUES 29 JUL 2014</H2><P>Gel Extract Digestions from Yesterday. 
BFP Length: 696 Basepairs. </P><P>[CB008DB] + pTEF1<EM>rtTA + pTET</EM>GFP</P><P>Flow plate maps today: 
Doxycycline concentrations: </P><PRE><CODE>        H   G   F   E   D   C   B   A   
1           [----0 µg/ml------------]
2           [----0.03 µg/ml---------]
3           [----0.06 µg/ml---------]
4           [----0.09 µg/ml---------]
5           [----0.3 µg/ml----------]
6           [----0.6 µg/ml----------]
7           [----0.9 µg/ml----------]
8           [----3.0 µg/ml----------]
9           [----6.0 µg/ml----------]
10          [----9.0 µg/ml----------]
11          [----30 µg/ml-----------]
12          [----60 µg/ml-----------]
</CODE></PRE><P>pTEF1 promoters</P><PRE><CODE>  1   2   3   4   5   6   7   8   9   10  11  12
A [---------------[CB008DB]--------------------]  
B [---------------[CB008DB pTET_GFP]-----------]  
C [---------------[CB008DB pTET_GFP pTEF1]-----] 
D [---------------[CB008DB pTET_GFP pTEF1 m3]--]
E [---------------[CB008DB pTET_GFP pTEF1 m6]--] 
F [---------------[CB008DB pTET_GFP pTEF1 m7]--]             
G [---------------[CB008DB pTET_GFP pTEF1 m10]-]
H
</CODE></PRE><P>x3 plates</P><P>100X Dilution started at 7:16AM -&gt; 10:16AM</P><P>10:50AM Dox Dose. 
Plate 1 concentrations backwards. 
Plate 3 lane 10 had double exposure of two different concentrations. </P><P>Gel Extractions of Digestions</P><PRE><CODE>Lanes: 
1: 1kb Ladder
2: BFP Digestion
3: BFp Digestion
4: pAGA1 mCherry Digestion
5: pAGA1 mCherry Digestion
6: pPCL2 mCherry Digestion 
7: pPCL2 mCherry Digestion 
</CODE></PRE><P>
Digestion smeared. </P><P>New strategy: 
PCR BFP</P><P>Ligate with Backbone(pHY130E), and PCL2 and AGA1. </P><P>Today, Digest pAGA1 from pTS108 with APA1 (11:35AM-1:35PM) and Xho1 (1:35PM -&gt; 2:35PM) and gel extract (2:35PM -?&gt; 4PM)</P><P>Digestion of pAGA1 from pAGA1-SAG1 (pTS108) failed, showing very faint bands and smearing. Will try with new ddH20.  </P><P>Will now also PCR pAGA1. </P><H2>WED 30 JUL 2014</H2><P>Could not run flow cytometry today because no cells grew. </P><PRE><CODE>  1   2   3   4   5   6   7   8   9   10  11  12
A [---------------[CB008]----------------------]  
B [---------------[CB008 pTET_GFP]-------------]
C [---------------[CB008 pTET_GFP pTEF1_rtTA]--]  
D [---------------[CB008 pTET_GFP pTEF1 m3]----]     
E [---------------[CB008 pTET_GFP pTEF1 m6]----]    
F [---------------[CB008 pTET_GFP pTEF1 m7]----]  
G [---------------[CB008 pTET_GFP pTEF1 m10]---] 
H
</CODE></PRE><P>PCR'd pAGA1 and BFP overnight. </P><P>Thanks to Jeffrey for his assistance today while I'm out sick: </P><UL><LI>[X] PCR Clean Up</LI><LI>[X] Digests

</LI><LI>[X] BFP using Xho1 Not1 double digest for 3 hours w/ cutsmart @ 37°C</LI><LI>[X] pAGA1 using Apa1 w/ cutsmart for 2 hours @ room temp</LI><LI>[X] pAGA1 using Xho1 w/ cutsmart for 1 hour @ 37°C</LI></UL><LI>[X] Ligations (Overnight)

</LI><LI>[X] Ligate: Hy130E - pAGA1 - BFP (Hy130E in my box! should already be digested!)</LI><LI>[X] Ligate: Hy130E - pPCL2 - BFP (pPCL2 should also be in my box, already digested!)</LI><H2>THURS 31 JUL 2014</H2><P>Out sick: Jeffery altruistically offered to help. </P><H2>FRI 01 AUG 2014</H2><P>Transformed Ligations from yesterday. 
Now working on website. </P><P>Finished landing page with petri dish now. 
Will now work on Team and Protocols pages. </P><P>Incorporate iGEM logo into top right corner. </P><P>Website Map</P><UL><LI>Home</LI><LI>Project

</LI><LI>Background</LI><LI>Achievements</LI><LI>Implications</LI><LI>Parts</LI><LI>Safety</LI></UL><LI>Models</LI><LI>Human Practices

</LI><LI>Collaborations</LI><LI>ALHS</LI><LI>Super Science!</LI><LI>Notebooks &amp; Protocols

</LI><LI>Notebooks</LI><LI>Protocols</LI><LI>Team

</LI><LI>Attributions</LI><LI>Advisors</LI><LI>Mentors</LI><LI>Students</LI><LI>Contact</LI><LI>iGEM Profile</LI><H2>MON 04 AUG 2014</H2><P>Worked on website. 
Picked colonies for miniprep tomorrow. Should have performed a e. coli colony PCR. </P><BLOCKQUOTE><H3>Colony PCR for screening E. Coli</H3><P>Pick single colonies ( 5 or so from each plate ) mix in 25 µl H2O in a tube. Use 5 µl in PCR reaction</P><TABLE><THEAD><TR><TH>Reagents</TH><TH>1X</TH><TH>6X</TH></TR></THEAD><TBODY><TR><TD>2X GoTaq Green PCR Master Mix</TD><TD>10 µl</TD><TD>60 µl</TD></TR><TR><TD>10 µM Forward primer</TD><TD>1 µl</TD><TD>6 µl</TD></TR><TR><TD>10 µM Reverse primer</TD><TD>1 µl</TD><TD>6 µl</TD></TR><TR><TD>Water</TD><TD>3 µl</TD><TD>18 µl</TD></TR><TR><TD>Bacterial cells (template)</TD><TD>5 µl</TD><TD>-----</TD></TR></TBODY></TABLE><P>Cycle (Varies): </P><P>95° C | 5m</P><PRE><CODE>  30x:
      95° C | 45s
       55° C | 30s
       72° C | 1m per kb
</CODE></PRE><P>72° C | 10m 
    4° C | Forever</P><P>load 5 µl onto gel</P><P>for all positive bands - take the rest of the bands and inoculate them into an overnight LB (+antibiotic) for miniprep</P><P>Started at 10:18AM -&gt; 12:35PM</P><H2>TUES 05 AUG 2014</H2></BLOCKQUOTE><P>Colony PCR of pPCL2<EM>BFP and pAGA1</EM>BFP</P><P>
Miniprep'd pAGA1 in lanes 5 and 6. </P><P>Picked new colonies from pPCL2 plate and ran colony PCR.
</P><H2>WED 06 AUG 2014</H2><P>Working on Poster for Santa Cruz today. 
Found mixed peaks in sequencing reaction. Otherwise, sequences were correct. Will transform minipreps and re-miniprep. Miniprep pPCL2 today. </P><H2>THURS 07 AUG 2014</H2><P>Sequences from pPCL2 BFP worked out very well. Colonies grew on pAGA1<EM>BFP  plate and not on negative control. Will miniprep pAGA1</EM>BFP tomorrow. </P><H2>FRI 08 AUG 2014</H2><P>1:200 dilution at 7:45AM. </P><P>[Dox] dose response of pTEF1<EM>rtTA pTET</EM>GFP pTET<EM>MFalpha pAGA1</EM>mCherry</P><PRE><CODE>  1   2   3   4   5   6   7   8   9   10  11  12
A [CB008 m6-1]----]       [CB008 m10-3]----]
B [CB008 m6-2]----]       [CB008DB m7-1]---]
C [CB008 m6-3]----]       [CB008DB m7-2]---]
D [CB008 m7-1]----]       [CB008DB m7-3]---]
E [CB008 m7-2]----] 
F [CB008 m7-3]----]           
G [CB008 m10-1]---] 
H [CB008 m10-2]---]
</CODE></PRE><P>[Dox] Map:</P><PRE><CODE>        H   G   F   E   D   C   B   A   
1           [----0 µg/ml------------]
2           [----0.03 µg/ml---------]
3           [----0.06 µg/ml---------]
4           [----0.09 µg/ml---------]
5           [----0.6 µg/ml----------]
6 
7                      [--0 µg/ml---]
8                      [--0.03 µg/ml]
9                      [--0.06 µg/ml]
10                     [--0.09 µg/ml]
11                     [--0.6 µg/ml-]
12                     
</CODE></PRE><H2>MON 11 AUG 2014</H2><P>1:200 dilution of 4 different strains into 4 96 well plates. </P><P>Need to prepare presentation for group meeting tomorrow. </P><P>Dilution of cultures at 7:30AM. </P><P>4 different promoters in front of rtTA: pHYM1, pYDR124W, pCLG1, pASG7</P><P>Dox</P><PRE><CODE>        H   G   F   E   D   C   B   A   
1           [----0 µg/ml------------]
2           [----0.03 µg/ml---------]
3           [----0.06 µg/ml---------]
4           [----0.09 µg/ml---------]
5           [----0.3 µg/ml----------]
6           [----0.6 µg/ml----------]
7           [----0.9 µg/ml----------]
8           [----3.0 µg/ml----------]
9           [----6.0 µg/ml----------]
10          [----9.0 µg/ml----------]
11          [----30 µg/ml-----------]
12          [----60 µg/ml-----------]
</CODE></PRE><P>Alpha Factor Concentration Map</P><PRE><CODE>    1   2   3   4   5   6   7   8   9   10  11  12
A   [-------------------{0 nM}-------------------]
B   [-------------------{0.5 nM}-----------------]
C   [-------------------{1 nM}-------------------]
D   [-------------------{10 nM}------------------]
E   [-------------------{100 nM}-----------------]
F   [-------------------{1000 nM}----------------]
G   [-------------------{3000 nM}----------------]
H   
</CODE></PRE><P>Single cell scatter. </P><P>Correlation Coefficient between GFP and RFP. </P><P>CV (noise) for all strains. </P><H2>TUES 12 AUG 2014</H2><P>Rest of the AFRP<EM>rtTA pTET</EM>GFP </P><P>Set in induction at 8:11AM. </P><P>Notes about pTEF1<EM>rtTA pTET</EM>GFP pTET<EM>MFa pAGA1</EM>RFP data from Friday: </P><P>GFP Noise: 
Very consistent pattern across almost all strains(mutations of pTEF1). High 
[Doxycycline] induces noise to decrease over time, while lower doxycycline concentrations maintain similar noise levels over time, and some even increase. </P><H2>WEDS 13 AUG 2014</H2><P>Pick colonies for Jeffrey. 
Rest of pTEF1<EM>rtTA pTET</EM>GFP pTET<EM>MFa pAGA1</EM>mCherry </P><P>1:200 dilution at 7:50AM. </P><P>[Dox] dose response of pTEF1<EM>rtTA pTET</EM>GFP pTET<EM>MFalpha pAGA1</EM>mCherry</P><PRE><CODE>  1   2   3   4   5   6   7   8   9   10  11  12
A [CB008 pTEF1-1]-]       [CB008DB m3-3]--] 
B [CB008 pTEF1-2]-]       [CB008DB m6-1]---]
C [CB008 pTEF1-3]-]       [CB008DB m6-2]---]
D [CB008DB pTEF1-1]       [CB008DB m6-3]---]
E [CB008DB pTEF1-2]  
F [CB008DB pTEF1-3]           
G [CB008DB m3-1]--] 
H [CB008DB m3-2]--] 
</CODE></PRE><H2>THURS 14 AUG 2014</H2><P>1:200 Dilution at 7:52AM</P><P>[Dox] dose response of pTEF1<EM>rtTA pTET</EM>GFP pTET<EM>MFalpha pAGA1</EM>mCherry</P><PRE><CODE>  1   2   3   4   5   6   7   8   9   10  11  12
A [CB008 pTEF1-1]-]       [CB008 m7-3]-----] 
B [CB008 pTEF1-2]-]       [CB008 m10-1]----]
C [CB008 pTEF1-3]-]       [CB008 m10-2]----]
D [CB008 m6-1]----]       [CB008 m10-3]----]
E [CB008 m6-2]----]  
F [CB008 m6-3]----]          
G [CB008 m7-1]----] 
H [CB008 m7-2]----] 
</CODE></PRE><P>Plate failed due to large amounts of contamination. Flow FSC/SSC plots showed large amounts of contamination. </P><H2>FRI 15 AUG 2014</H2><P>UCSC Meetup!</P><H2>MON 18 AUG 2014</H2><P>Moving!</P><H2>TUES 19 AUG 2014</H2><P>Moving!</P><H2>WED 20 AUG 2014</H2><P>Back at it! 
Taught people how to generate graphs from flow data. Also taught people how to generate scatter plots from flow data. Produced graphs for flow data from many days ago. </P><P>Decided to not continue with pECM18 and pPRM6 as they did not seem to express more than basal level. </P><H2>THURS 21 AUG 2014</H2><P>Attempted to use R for further data analysis. Importing data into R may be too complicated and time-consuming. </P><H2>FRI 22 AUG 2014</H2><P>Seems like all DB strains have RFP expression without induction with Doxy. Most likely due to leakiness in Tet-On circuit. </P><P>Decided to not analyze data from Tuesday and will be running experiments again on Monday. </P><P>Downloaded and install flowjo on my computer for easier remote work. </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCSF_UCB/ianto">http://2014.igem.org/Team:UCSF_UCB/ianto</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCSF_UCB/ianto" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCSF_UCB/ianto" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCSF_UCB/ianto&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCSF_UCB/ianto&amp;oldid=139909" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>