<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCSF_UCB_project_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCSF_UCB/project.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCSF_UCB/project.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF_UCB/project.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF_UCB/project.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCSF_UCB/project.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCSF UCB/project.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>    UCSF iGEM 2014  </TITLE><SVG height="0px" width="0px"><DEFS><G id="BaseCircuit-Background"><TEXT transform="matrix(1 0 0 1 325.7002 3.168)" style="fill:#FFFFFF; font-family:'HelveticaNeue-Light'; font-size:24;">Community Response</TEXT><TEXT transform="matrix(1 0 0 1 31.7002 5.501)" style="fill:#FFFFFF; font-family:'HelveticaNeue-Light'; font-size:24;">Individual Response</TEXT></G><G id="Implementation-MFalpha"><TEXT transform="matrix(1 0 0 1 511.085 323.2002)" style="fill:#FFFFFF; font-family:'HelveticaNeue-Bold'; font-size:10;">ALPHA FACTOR</TEXT></G><G id="Implementation-Doxy"><TEXT transform="matrix(1 0 0 1 11.8994 322.1006)" style="fill:#FFFFFF; font-family:'HelveticaNeue-Bold'; font-size:10;">DOXYCYCLINE</TEXT></G><G id="Conceptual-CommunityResponse"><TEXT transform="matrix(1 0 0 1 298.8994 115.1338)" style="fill:#FFFFFF; font-family:'HelveticaNeue-Bold'; font-size:12;">RESPONSE</TEXT></G><G id="Conceptual-Secondary"><TEXT transform="matrix(1 0 0 1 465.8994 140.2998)" style="fill:#FFFFFF; font-family:'HelveticaNeue-Bold'; font-size:12;">SECONDARY SIGNAL</TEXT></G><G id="Conceptual-Communication"><TEXT transform="matrix(1 0 0 1 295.8994 74.1338)" style="fill:#FFFFFF; font-family:'HelveticaNeue-Bold'; font-size:12;">COMMUNICATION</TEXT></G><G id="Conceptual-IndividualResponse"><TEXT transform="matrix(1 0 0 1 139.8994 73.6338)" style="fill:#FFFFFF; font-family:'HelveticaNeue-Bold'; font-size:12;">RESPONSE</TEXT></G><G id="Conceptual-Stimulus"><TEXT transform="matrix(1 0 0 1 11.8994 101.2998)" style="fill:#FFFFFF; font-family:'HelveticaNeue-Bold'; font-size:12;">STIMULUS</TEXT></G></DEFS></SVG></P><DIV class="header-container"><HEADER class="wrapper clearfix"><NAV><UL><LI class="project"><A class="on-page" href="./project.html">Project            </A></LI><LI class="judging"><A href="./judging.html">Judging</A></LI><LI class="hp"><A href="./hp.html">Human Practices            </A></LI><LI class="notebooks"><A href="./notebooks.html">Notebooks &amp; Protocols            </A></LI><LI class="team"><A href="./team.html">Team            </A></LI></UL></NAV></HEADER></DIV><DIV id="background"><DIV class="project-container"><DIV class="wrapper clearfix menu"><H1>          Background        </H1><A href="#design"><H2>            Design            
                                                    </H2></A><A href="#achievements"><H2>            Achievements            
                                                    </H2></A><A href="#model"><H2>            Models            
                                                    </H2></A></DIV></DIV><DIV class="main-container"><DIV class="wrapper clearfix"><P>Cells in a local population have a wide range of responses to a given stimulus. This variation in cell response may be due to differences in immediate extracellular environment or intracellular molecular makeup. Nevertheless, in spite of local differences in cellular or environmental conditions, cellular systems often hinge on their component cells’ ability to coordinate and respond in a concerted way. For example, all T cells in a lymph node respond to an antigen, yet only those with a strong initial reaction proliferate and activate an immune response [<A href="#footnote-3">1</A>]. </P><P>This phenomenon is also exhibited in quorum sensing by the bioluminescent bacteria <EM>V. fischeri</EM>. Through symbiosis with the Hawaiian Bobcat Squid, <EM>V. fischeri</EM> live in light organs at high cell densities. In these high densities, the <EM>V. fischeri</EM> bacteria can change the concentrations of certain chemicals in their local environment and activate bioluminescence in the entire bacterial population.</P><P>In some cases, the decision reaches consensus, and all cells respond the same -- a phenomenon seen in quorum sensing by <EM>V. fischeri</EM>, for example, where gene expression is coordinated based on signals sensed at particular cell densities. In this example, cell populations somehow overcome the intrinsic biological noise found in all cellular populations and are able to respond in a concerted fashion. Alternatively, decisions can also favor the success of one individual in the group, namely the behavior of T cells in the human immune system. Again, intrinsic biological noise is suppressed as a coordinated and specific response is delivered by the population. </P><UL class="foot"><LI id="footnote-3">1. Höfer, T., Krichevsky, O., &amp; Altan-Bonnet, G. (2012). Competition for IL-2 between Regulatory and Effector T Cells to Chisel Immune Responses. Frontiers in Immunology, 3(September), 268. doi:10.3389/fimmu.2012.00268</LI></UL></DIV></DIV></DIV><DIV id="design"><DIV class="project-container"><DIV class="wrapper clearfix menu"><A href="#background"><H2>            Background            
                                                    </H2></A><H1>          Design        </H1><A href="#achievements"><H2>            Achievements            
                                                    </H2></A><A href="#model"><H2>            Models            
                                                    </H2></A></DIV></DIV><DIV class="main-container"><DIV class="wrapper clearfix"><DIV id="project-circuit" class="clearfix"><H2>Circuit Development</H2><ARTICLE id="CircuitDevelopment-1" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-2">Next</A></DIV><DIV class="slide-content"><H3>Conceptual Framework</H3><P>All multicellular systems, and many examples of collectives of unicellular organisms, communicate in order to respond and adapt to their environment. However, biological systems are pervaded by noise, and surprisingly in the face of that, many biological responses are robust and withstand cell-to-cell variation [<A href="#footnote-9">1</A>]. </P><P>Using endogenous signaling pathways found in yeast, communicating “sense-and-secrete” circuits can be designed to interrogate cell-to-cell signaling, and collective response, in these organisms. Depending on the communication parameters — signal strength, receptor abundance, and signal degradation — researchers were able to tune the strength and output of these communicating cells [<A href="#footnote-10">2</A>]. </P><P>We expand on this previous work by investigating further the relationship between cell-to-cell variation and the collective community response of cells. We have engineered yeast strains to report two responses by fluorescence: 1) a cell autonomous response triggered by an experimental induction and 2) a collective community response driven by cell-cell signaling. We are thus able to read simultaneously the relationship between the individual cell response and the post-cellular signal processing that occurs through extracellular signaling.</P><P>Please click &quot;next&quot; in the upper right corner of this box for an explanation of each component of this theoretical circuit and our implementation of this circuit.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-2" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-3">Next</A><A class="slide-left" href="#CircuitDevelopment-1">Prev</A></DIV><DIV class="slide-content"><H3>Conceptual Framework: Stimulus</H3><P>In our generalized circuit design, we would like to be able to stimulate the cell response reproducibly, and will therefore use a cell-permeable small molecule inducer to drive gene expression. Cells should have a range of responses given different amounts of stimuli.</P><P>An analoguous native stimulus includes antigen that stimulate immune response.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-3" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-4">Next</A><A class="slide-left" href="#CircuitDevelopment-2">Prev</A></DIV><DIV class="slide-content"><H3>Conceptual Framework: Autonomous Response</H3><P>To read an output that allows us to measure the natural variation in genetically identical cells, we need to engineer an output signal or response. This external stimulus will directly induce a fluorescence output, allowing us to read the natural variation in a population.</P><P>Often, there is no analogy to this in natural systems as this variation is suppressed for a coordinated community response.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-4" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-5">Next</A><A class="slide-left" href="#CircuitDevelopment-3">Prev</A></DIV><DIV class="slide-content"><H3>Conceptual Framework: Communication Response</H3><P>The stimulus will also trigger a communication circuit, driven by the same induction system as the cell autonomous fluorescence response.</P><P>In T regulatory and T killer cells, this communication response pathway corresponds to the IL-2 downstream pathway that includes upregulation of IL-2R alpha and IL-2 secretion.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-5" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-6">Next</A><A class="slide-left" href="#CircuitDevelopment-4">Prev</A></DIV><DIV class="slide-content"><H3>Conceptual Framework: Secondary Signal</H3><P>The communication circuit will incorporate a secreted signal produced by the induced cell. </P><P>This communication signal is IL-2 in T cells and autoinducer molecules in <EM>V. fischeri</EM>.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-6" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-7">Next</A><A class="slide-left" href="#CircuitDevelopment-5">Prev</A></DIV><DIV class="slide-content"><H3>Conceptual Framework: Community Integration</H3><P>The secreted signal will be produced and is transmitted between all cells in the population.</P><P>While in our experiment, we will stick with simple culture tubes, the location and structure of how cells are arranged and allowed to move can impact how the secondary signal is integrated. T cells are usually arranged in the highly structured lymph node when first exposed to pathological antigen.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-7" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-8">Next</A><A class="slide-left" href="#CircuitDevelopment-6">Prev</A></DIV><DIV class="slide-content"><H3>Conceptual Framework: Community Response</H3><P>Each cell in the population will also have receptors to recognize the secondary signal. Receptor binding will trigger a signaling cascade to induce expression of a downstream reporter gene to indicate cellular communication levels — in this case, fluorescence.</P><P>In <EM>V. fischeri</EM>, one can find an analogous community response once the population has reached quorum. This response, however, is processed to binary form while our circuit will strive to output a signal that reports the continuous distribution of community responses.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-8" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-9">Next</A><A class="slide-left" href="#CircuitDevelopment-7">Prev</A></DIV><DIV class="slide-content"><H3>Implementation</H3><P>Here we diagram the realized implementation of our circuit design.</P><P>There are two components to the overall circuit. The first (left), is a cell autonomous response driven by induction by a small-molecule (doxycycline). This induction is read out by the reporter GFP.</P><P>The second component integrates communication by stimulating the production of a secreted signaling factor after doxycycline induction. This factor can communicate to the originating cell as well as neighboring cells, and receptor binding of this signal stimulates RFP reporter expression.</P><P>This design thus allows us to view both individual cell responses (GFP) and community responses (RFP) simultaneously in a population of cells.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-9" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-10">Next</A><A class="slide-left" href="#CircuitDevelopment-8">Prev</A></DIV><DIV class="slide-content"><H3>Implementation: Doxycycline</H3><P>For our stimulus, we have opted to use the well-characterized small molecule antibiotic, doxycycline, as an inducer.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-10" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-11">Next</A><A class="slide-left" href="#CircuitDevelopment-9">Prev</A></DIV><DIV class="slide-content"><H3>Implementation: rtTA</H3><P>Doxycycline is able to induce gene expression by binding to the engineered transcription factor rtTA. This protein is a fusion between the <EM>E. coli</EM> Tet repressor (which binds doxycycline) and the <EM>S. cerevisiae</EM> transcription factor Msn2 [<A href="#footnote-11">3</A>]. </P><P>We have further elaborated on this component of the circuit by engineering yeast strains that respond differently to doxycycline induction. We have achieved this by altering the constitutive promoter driving rtTA expression. We have implemented the endogenous pTEF1 promoter as well as several mutants (see promoter characterization below) which have altered expression levels [<A href="#footnote-2">4</A>]. The altered doxycycline dose response can be viewed in the Achievements section.
																												</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-11" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-12">Next</A><A class="slide-left" href="#CircuitDevelopment-10">Prev</A></DIV><DIV class="slide-content"><H3>Implementation: GFP</H3><P>The doxycycline-rtTA complex is able to initiate transcription at the TET promoter. We have placed GFP behind pTET in order to read out the direct doxycycline response.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-12" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-13">Next</A><A class="slide-left" href="#CircuitDevelopment-11">Prev</A></DIV><DIV class="slide-content"><H3>Implementation: Mating Factor Alpha</H3><P>Our communication circuit is also driven by the TET promoter — doxycycline induction results in expression of a secreted signaling molecule. We have repurposed an endogenous yeast signaling system for this circuit, the mating pathway. Our yeast strains contain the receptors and downstream signaling cascade to be able to respond to the presence of mating factor alpha; however, the only alpha production gene is controlled by our communication circuit [<A href="#footnote-1">5</A>,<A href="#footnote-12">6</A>]. </P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-13" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-14">Next</A><A class="slide-left" href="#CircuitDevelopment-12">Prev</A></DIV><DIV class="slide-content"><H3>Implementation: Yeast Population Integration</H3><P>Alpha factor is secreted by all of the cells in the population, and can be similarly sensed by all cells.</P></DIV></ARTICLE><ARTICLE id="CircuitDevelopment-14" class="clearfix"><DIV class="slide-controls"><A class="slide-right" href="#CircuitDevelopment-8">Next</A><A class="slide-left" href="#CircuitDevelopment-13">Prev</A></DIV><DIV class="slide-content"><H3>Implementation: RFP</H3><P>Binding of alpha factor to its receptor, Ste2, triggers a signaling cascade that stimulates induction at alpha factor responsive promoters, of which we characterized and submitted 11 to the Parts Registry (see <A href="#biobricks">below</A>). We chose the highly expressing pAGA1 as the promoter to drive the reporter RFP in order to read the community-stimulated response.</P></DIV></ARTICLE><TABLE><THEAD><TR><TH class="active"><A href="#CircuitDevelopment-1">Conceptual Framework</A></TH></TR></THEAD><TBODY><TR><TD><A href="#CircuitDevelopment-2">Stimulus</A></TD></TR><TR><TD><A href="#CircuitDevelopment-3">Autonomous Response</A></TD></TR><TR><TD><A href="#CircuitDevelopment-4">Communication Response</A></TD></TR><TR><TD><A href="#CircuitDevelopment-5">Secondary Signal</A></TD></TR><TR><TD><A href="#CircuitDevelopment-6">Community Integration</A></TD></TR><TR><TD><A href="#CircuitDevelopment-7">Community Response</A></TD></TR></TBODY><THEAD><TR><TH><A href="#CircuitDevelopment-8">Implementation</A></TH></TR></THEAD><TBODY><TR><TD><A href="#CircuitDevelopment-9">Doxycycline</A></TD></TR><TR><TD><A href="#CircuitDevelopment-10">rtTA</A></TD></TR><TR><TD><A href="#CircuitDevelopment-11">GFP</A></TD></TR><TR><TD><A href="#CircuitDevelopment-12">Mating Factor Alpha</A></TD></TR><TR><TD><A href="#CircuitDevelopment-13">Yeast Population Integration</A></TD></TR><TR><TD><A href="#CircuitDevelopment-14">RFP</A></TD></TR></TBODY></TABLE></DIV><DIV id="experiment-design"><H2>Experiment Design</H2><P>To confirm that our circuits behave as we have designed them to behave, we set up two different experiments: </P><P>1. Simple doxycycline dose response to test cell autonomous behavior. The results of this experiment can be found <A href="#results">here</A>.</P><P>2. <STRONG>Mixing experiment</STRONG> (on the right), where we test the ability of our circuit to produce convergent behavior with artificially induced divergence. In our experiment, we mix our lowest responding cell autonomous circuit with rtTA expressed by pTEF1(m7) with our higest responding circuit with pTEF1. We expect that through intercellular signaling and the community circuit, the two populations will look in distinguishable by the community response reporter. Our results for this experiment can be found <A href="#mix-result">here</A>.</P></DIV><DIV id="project-promoter" class="clearfix"><H2>Promoter Characterization&gt;</H2><P>We have characterized two types of promoters endogenous to <EM>S. Cerevisiae</EM>; eleven alpha-factor responsive promoters (AFRP) and five constitutive promoters. To characterize these promoters, we cloned them into a shuttle vector plasmid to drive expression of GFP, which was subsequently integrated into yeast to create stable strains. Each construct’s fluorescence was measured using flow cytometry. </P><H3>Alpha Factor Responsive Promoters (AFRP)</H3><SVG class="diagram" version="1.1" width="250px" height="136px" id="Promoter-AFRPGFP" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="-89.8 -86.7 165.3 90.3"><G id="AFRPGFP-AFRP_x5F_GFP"><TEXT transform="matrix(1 0 0 1 19.958 -37.375)" fill="#FFFFFF" font-family="'HelveticaNeue-Bold'" font-size="9">GFP</TEXT><TEXT transform="matrix(1 0 0 1 -18.292 -35.875)" fill="#FFFFFF" font-family="'HelveticaNeue-MediumItalic'" font-size="9">AFRP</TEXT></G><TEXT transform="matrix(1 0 0 1 -85.2603 -77.1665)"><TSPAN x="0" y="0" fill="#FFFFFF" font-family="'HelveticaNeue-Thin'" font-size="7">Mating Factor </TSPAN><TSPAN x="12.7" y="8.4" fill="#FFFFFF" font-family="'HelveticaNeue-Thin'" font-size="7">Alpha</TSPAN></TEXT></SVG><P>AFRPs were selected from a microarray study where Roberts <EM>et. al.</EM> searched for genes responsive to yeast MAPK pathways including the alpha factor response pathway [<A class="footnote" href="#footnote-1">5</A>]. We picked the promoters of the genes that were shown to be up-regulated in the presence of alpha factor. For each AFRP, we measured its construct’s fluorescence over 7 concentrations of alpha factor [0nM, 0.5nM, 1nM, 10nM, 100nM, 1000nM, 3000nM] after 90 minutes of induction. Through multiple rounds of testing, we found pAGA1 to have the highest expression level and with very low basal expression and pPCL2 to have the highest basal expression and the second highest induced expression.</P><H3>Constitutive Promoters</H3><SVG class="diagram" version="1.1" width="171px" height="136px" id="Promoter-pTEF1GFP" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="-37.2 -86.7 112.7 90.3" enable-background="new -37.2 -86.7 112.7 90.3" xml:space="preserve"><G id="pTEF1GFP-pTEF1GFP"><TEXT transform="matrix(1 0 0 1 19.958 -37.375)" fill="#FFFFFF" font-family="'HelveticaNeue-Bold'" font-size="9">GFP</TEXT><TEXT transform="matrix(1 0 0 1 -18.292 -35.875)" fill="#FFFFFF" font-family="'HelveticaNeue-MediumItalic'" font-size="9">pTEF1</TEXT></G></SVG><P>Constitutive promoters were selected from a study where Nevoigt et. al. used error prone PCR to create 14,000 pTEF1 mutants [<A class="footnote" href="#footnote-2">4</A>]. The study then selected 11 mutants with a wide range of promoter activity for further characterization. Out of these 11, we chose 5 of these mutants to use in our circuit. We confirmed that constitutive promoter pTEF1 m6 had the highest expression of GFP whereas constitutive promoter m7 had the lowest expression of GFP. Our data closely correlates with the results of the Nevoigt study. </P><P> Characterization data for all of our promoters can be found in <A href="#biobricks">Biobricks Submitted</A> of our <A href="#achievements">Achievements Section</A>.</P><UL class="foot"><LI id="footnote-9">1. Cotari, J. W., Voisinne, G., &amp; Altan-Bonnet, G. (2013). Diversity training for signal transduction: leveraging cell-to-cell variability to dissect cellular signaling, differentiation and death. Current Opinion in Biotechnology, 24(4), 760–6. doi:10.1016/j.copbio.2013.05.002</LI><LI id="footnote-10">2. Youk, H., &amp; Lim, W. A. (2014). Secreting and Sensing the Same Molecule Allows Cells to Achieve Versatile Social Behaviors. Science (New York, N.Y.), 343(6171), 1242782. doi:10.1126/science.1242782</LI><LI id="footnote-11">3. Gossen, M., Pak, D.T.S., Hansen, S.K., Acharya, J.K., Botchan, M.R. (1995). A Drosophila homolog of the yeast origin recognition complex.  Science 270(5242): 1674--1677</LI><LI id="footnote-2">4. Nevoigt, E., Kohnke, J., Fischer, C. R., Alper, H., Stahl, U., &amp; Stephanopoulos, G. (2006). Engineering of promoter replacement cassettes for fine-tuning of gene expression in Saccharomyces cerevisiae. Applied and Environmental Microbiology, 72(8), 5266–73. doi:10.1128/AEM.00530-06</LI><LI id="footnote-1">5. Roberts, C. J. (2000). Signaling and Circuitry of Multiple MAPK Pathways Revealed by a Matrix of Global Gene Expression Profiles. Science, 287(5454), 873–880. doi:10.1126/science.287.5454.873</LI><LI id="footnote-12">6. Gross, a, Rödel, G., &amp; Ostermann, K. (2011). Application of the yeast pheromone system for controlled cell-cell communication and signal amplification. Letters in Applied Microbiology, 52(5), 521–6. doi:10.1111/j.1472-765X.2011.03035</LI></UL></DIV></DIV></DIV></DIV><DIV id="achievements"><DIV class="project-container"><DIV class="wrapper clearfix menu"><A href="#background"><H2>            Background            
                                                    </H2></A><A href="#design"><H2>            Design            
             </H2></A><H1>          Achievements        </H1><A href="#model"><H2>Models            
                                                    </H2></A></DIV></DIV><DIV class="main-container"><DIV class="wrapper clearfix"><DIV id="results" class="clearfix"><DIV class="clearfix"><H2>Results: Autonomous Circuit Verification</H2><DIV class="result-graph"><H3>Doxycycline Dose Response of Different Autonomous Circuits</H3><SVG version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="650px" height="505px" viewbox="0 30 850 646" xml:space="preserve"><G><TEXT transform="matrix(1 0 0 1 259.0059 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">0</TEXT><TEXT transform="matrix(1 0 0 1 316.1699 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">0.03</TEXT><TEXT transform="matrix(1 0 0 1 360.0918 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">0.06</TEXT><TEXT transform="matrix(1 0 0 1 398.584 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">0.09</TEXT><TEXT transform="matrix(1 0 0 1 467.1699 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">0.3</TEXT><TEXT transform="matrix(1 0 0 1 511.1699 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">0.6</TEXT><TEXT transform="matrix(1 0 0 1 536.834 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">0.9</TEXT><TEXT transform="matrix(1 0 0 1 620.748 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">3</TEXT><TEXT transform="matrix(1 0 0 1 664.6699 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">6</TEXT><TEXT transform="matrix(1 0 0 1 690.4199 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">9</TEXT><TEXT transform="matrix(1 0 0 1 761.7559 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">30</TEXT><TEXT transform="matrix(1 0 0 1 805.748 643.4199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">60</TEXT><TEXT transform="matrix(1 0 0 1 105.084 630.5059)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">0</TEXT><TEXT transform="matrix(1 0 0 1 105.084 531.6699)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">2</TEXT><TEXT transform="matrix(1 0 0 1 105.084 432.9199)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">4</TEXT><TEXT transform="matrix(1 0 0 1 105.084 334.0918)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">6</TEXT><TEXT transform="matrix(1 0 0 1 105.084 235.3418)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">8</TEXT><TEXT transform="matrix(1 0 0 1 95.084 136.5059)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">10</TEXT><TEXT transform="matrix(1 0 0 1 117.998 48.9277)" fill="#6D6E71" font-family="'Helvetica'" font-size="18">x 10</TEXT><TEXT transform="matrix(1 0 0 1 151.998 37.7559)" fill="#6D6E71" font-family="'Helvetica'" font-size="12">4</TEXT></G><G><TEXT transform="matrix(1 0 0 1 377.5059 673.2559)" fill="#6D6E71" font-family="'Helvetica'" font-size="24">[Doxycycline](µg/µl)</TEXT><TEXT transform="matrix(0 -1 1 0 86.5059 402.7402)" fill="#6D6E71" font-family="'Helvetica'" font-size="24">Mean GFP (AU)</TEXT><TEXT transform="matrix(1 0 0 1 162.9199 82.584)" fill="#231F20" font-family="'Helvetica'" font-size="24">Control</TEXT><TEXT transform="matrix(1 0 0 1 162.9199 111.0059)" fill="#231F20" font-family="'Helvetica'" font-size="24">pTEF1</TEXT><TEXT transform="matrix(1 0 0 1 162.9199 139.5059)" fill="#231F20" font-family="'Helvetica'" font-size="24">pTEF1(m3)</TEXT><TEXT transform="matrix(1 0 0 1 162.9199 167.9199)" fill="#231F20" font-family="'Helvetica'" font-size="24">pTEF1(m6)</TEXT><TEXT transform="matrix(1 0 0 1 162.9199 196.4199)" fill="#231F20" font-family="'Helvetica'" font-size="24">pTEF1(m7)</TEXT><TEXT transform="matrix(1 0 0 1 162.9199 224.834)" fill="#231F20" font-family="'Helvetica'" font-size="24">pTEF1(m10)</TEXT></G></SVG></DIV><DIV class="graph-description" class="clearfix"><P>Here, we demonstrate that the cell autonomous portion of our circuit behaves as designed. Each line represents a different version of the rtTA autonomous circuit as shown above. The circuits differ by the constitutive promoter driving rtTA expression; five mutants of pTEF1 were used to drive rtTA expression, creating a library of Doxycycline responsive circuits. In this graph, one can see that pTEF1(m7) driving rtTA results in the least sensitive circuit. At a doxycycline concentration of 0.3µg/µl, we achieve the greatest resolution between different cell autonomous circuits.</P></DIV></DIV><DIV id="mix-result" class="clearfix"><H2>Results: Community Circuit Verification</H2><DIV class="result-graph"><H3>Convergence of Community Response</H3><SVG xmlns="http://www.w3.org/2000/svg" width="658.4px" height="371.6px" viewbox="0 0 658.4 371.6"><G id="Layer_2"><TEXT transform="matrix(1 0 0 1 108.5005 355.1008)" fill="#FFFFFF" font-family="'HelveticaNeue'" font-size="20">Normalized GFP (A.U.)</TEXT><TEXT transform="matrix(1 0 0 1 420.5015 356.1008)" fill="#FFFFFF" font-family="'HelveticaNeue'" font-size="20">Normalized RFP (A.U.)</TEXT><TEXT transform="matrix(-4.371139e-08 -1 1 -4.371139e-08 40.2783 209.3586)" fill="#FFFFFF" font-family="'HelveticaNeue'" font-size="20">Count</TEXT></G><TEXT transform="matrix(1 0 0 1 112.7007 32.1008)" fill="#FFFFFF" font-family="'HelveticaNeue-Light'" font-size="24">Individual Response</TEXT><TEXT transform="matrix(1 0 0 1 399.7007 31.7678)" fill="#FFFFFF" font-family="'HelveticaNeue-Light'" font-size="24">Community Response</TEXT></SVG></DIV><DIV class="graph-description" class="clearfix"><P>
	                                                	In this figure, we show that we have successfully constructed a community averaging circuit in yeast. The blue histogram corresponds to yeast cells with pTEF1(m7) driving expression of rtTA, while the yellow histogram corresponds to yeast cells with pTEF1(m10) driving expression of rtTA. These two yeast strains were mixed and grown in the same well to produce a divergent population as shown in the divergent GFP peaks. This genetic divergence is averaged upon alpha factor signaling as shown in the overlapping RFP peaks. 

	                                                </P></DIV></DIV></DIV><DIV id="future" class="clearfix"><H2>Future Directions</H2><P>To allow us to tune intercellular communication, we plan to add the following feedback loops to our base circuit above: </P><DIV class="direction clearfix"><H3>Positive Feedback: Mating Factor Alpha</H3><SVG xmlns="http://www.w3.org/2000/svg" width="280px" height="173px" viewbox="0 0 129.5 79.8"><G id="MFalpha-gene"><TEXT transform="matrix(1 0 0 1 50.6582 51.9917)" fill="#FFFFFF" font-family="'HelveticaNeue-Bold'" font-size="9">MFα</TEXT><TEXT transform="matrix(1 0 0 1 12.4082 53.4917)" fill="#FFFFFF" font-family="'HelveticaNeue-MediumItalic'" font-size="9">AFRP</TEXT></G></SVG><P>In this positive feedback circuit, we express mating factor alpha under an alpha factor responsive promoter (AFRP). This generates a direct feedback loop where more alpha factor sensed effects more alpha factor secreted. </P></DIV><DIV class="direction clearfix"><H3>Positive Feedback: STE2</H3><SVG xmlns="http://www.w3.org/2000/svg" width="280px" height="175px" viewbox="0 0 199.3 124.5" enable-background="new 0 0 199.3 124.5"><G id="STE2-elements"><G id="STE2-AFRP_x5F_STE2"><TEXT transform="matrix(1 0 0 1 74.4365 50.5264)" font-family="'HelveticaNeue-Bold'" font-size="13.8675">Ste2</TEXT><TEXT transform="matrix(1 0 0 1 15.499 52.8379)" fill="#FFFFFF" font-family="'HelveticaNeue-MediumItalic'" font-size="13.8675">AFRP</TEXT></G></G></SVG><P>In this positive feedback circuit, we express the mating factor alpha receptor, STE2, under the control of an alpha factor responsive promoter. While yeast endogenously upregulate STE2 in response to alpha factor, our circuit would exacerbate that effect. Compared to the MFalpha positive feedback circuit (on the left), this feedback circuit is less direct as it upregulates a receptor that senses alpha factor. This does not change the local concentration of alpha factor, and may be thought of as a &quot;selfish&quot; feedback loop. </P></DIV><DIV class="direction clearfix"><H3>Negative Feedback: BAR1</H3><SVG id="BAR1" xmlns="http://www.w3.org/2000/svg" width="280px" height="194px" viewbox="0 0 129.5 89.9"><G id="BAR1-BAR1"><TEXT transform="matrix(1 0 0 1 47.708 41.8914)" font-family="'HelveticaNeue-Bold'" font-size="9">Bar1</TEXT><TEXT transform="matrix(1 0 0 1 10.708 43.3914)" fill="#FFFFFF" font-family="'HelveticaNeue-MediumItalic'" font-size="9">AFRP</TEXT></G><G id="BAR1-BAR1protein"><TEXT transform="matrix(1 0 0 1 109.958 10.8914)" fill="#FFFFFF" font-family="'HelveticaNeue-CondensedBold'" font-size="4">BAR1</TEXT></G></SVG><P>In this negative feedback circuit, we express BAR1, an alpha factor degrading protease, under the control of an alpha factor responsive promoter. With BAR1, we expect to be able to decrease cellular noise and model divergent populations. Our computational models that demonstrate this expectation can be found in our <A href="#model">Models section</A>. </P></DIV></DIV><DIV id="biobricks" class="clearfix"><H2>            BioBricks Used and Submitted          
                                                <TABLE><THEAD><TR><TH>                  AFRP                </TH><TH>                  Biobrick #                </TH></TR></THEAD><TBODY><TR><TD>                  pASG7                </TD><TD class="active"><A href="#pASG7">      BBa_K1346011                  </A></TD></TR><TR><TD>                  pPCL2                </TD><TD><A href="#pPCL2">      BBa_K1346013                  </A></TD></TR><TR><TD>                  pPRM1                </TD><TD><A href="#pPRM1">      BBa_K1346004                  </A></TD></TR><TR><TD>                  pPRM2                </TD><TD><A href="#pPRM2">      BBa_K1346005                  </A></TD></TR><TR><TD>                  pPRM3                </TD><TD><A href="#pPRM3">      BBa_K1346006                  </A></TD></TR><TR><TD>                  pPRM6                </TD><TD><A href="#pPRM6">      BBa_K1346007                  </A></TD></TR><TR><TD>                  pSAG1                </TD><TD><A href="#pSAG1">      BBa_K1346009                  </A></TD></TR><TR><TD>                  pCLG1                </TD><TD><A href="#pCLG1">      BBa_K1346010                  </A></TD></TR><TR><TD>                  pECM18                </TD><TD><A href="#pECM18">      BBa_K1346008                  </A></TD></TR><TR><TD>                  pPRM6                </TD><TD><A href="#pPRM6">      BBa_K1346007                  </A></TD></TR><TR><TD>                  pHYM1                </TD><TD><A href="#pHYM1">      BBa_K1346012                  </A></TD></TR></TBODY><THEAD><TR><TH>                  pTEF1                </TH><TH>                  Biobrick #                </TH></TR></THEAD><TBODY><TR><TD>                  pTEF1                </TD><TD><A href="#pTEF1">      BBa_K431008                  </A></TD></TR><TR><TD>                  pTEF1.m3                </TD><TD><A href="#pTEF1_m3">      BBa_K1346000                  </A></TD></TR><TR><TD>                  pTEF1.m6                </TD><TD><A href="#pTEF1_m6">      BBa_K1346001                  </A></TD></TR><TR><TD>                  pTEF1.m7                </TD><TD><A href="#pTEF1_m7">      BBa_K1346002                  </A></TD></TR><TR><TD>                  pTEF1.m10                </TD><TD><A href="#pTEF1_m10">      BBa_K1346003                  </A></TD></TR></TBODY></TABLE></H2><H3 class="graph-title">  Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 619.4121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 562.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      2000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 505.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 448.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      4000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 391.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      5000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 334.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      6000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 277.1699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      7000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 220.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      8000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 163.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      9000                  </TEXT><TEXT transform="matrix(1 0 0 1 -79.9082 106.0059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      10000                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      Mean GFP (AU                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 306.0371)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      )                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 2              </P><P><A href="http://parts.igem.org/Part:BBa_K1346005">    Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                PRM2: Pheromone-regulated protein; transmembrane domains; regulated by Ste12p; required for efficient nuclear fusion              </P></DIV><ARTICLE id="pASG7" class="biobrick"><DIV class="graph"><H3 class="graph-title">  Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 619.4121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 562.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      2000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 505.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 448.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      4000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 391.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      5000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 334.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      6000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 277.1699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      7000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 220.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      8000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 163.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      9000                  </TEXT><TEXT transform="matrix(1 0 0 1 -79.9082 106.0059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      10000                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      Mean GFP (AU                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 306.0371)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      )                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 2              </P><P><A href="http://parts.igem.org/Part:BBa_K1346011">    Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                ASG7: Protein that regulates signaling from a G protein beta subunit Ste4p and its relocalization within the cell; specific to mating type a-cells and induced by alpha-factor              </P></DIV></ARTICLE><ARTICLE id="pPCL2" class="biobrick"><DIV class="graph"><H3 class="graph-title">  Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 619.4121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 562.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      2000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 505.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 448.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      4000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 391.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      5000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 334.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      6000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 277.1699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      7000     </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 220.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      8000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 163.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      9000                  </TEXT><TEXT transform="matrix(1 0 0 1 -79.9082 106.0059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      10000                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      Mean GFP (AU                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 306.0371)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      )                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 4              </P><P><A href="http://parts.igem.org/Part:BBa_K1346013">    Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                PCL2: Cyclin; interacts with cyclin-dependent kinase Pho85p;  involved in the regulation of polarized growth and morphogenesis and progression through the cell cycle              </P></DIV></ARTICLE><ARTICLE id="pCLG1" class="biobrick"><DIV class="graph"><H3 class="graph-title">  Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      0                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 619.4121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      1000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 562.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      2000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 505.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 448.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      4000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 391.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      5000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 334.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      6000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 277.1699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      7000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 220.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      8000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 163.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      9000                  </TEXT><TEXT transform="matrix(1 0 0 1 -79.9082 106.0059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      10000                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      Mean GFP (AU                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 306.0371)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">      )                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 2              </P><P><A href="http://parts.igem.org/Part:BBa_K1346010">    Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                CLG1: Cyclin-like protein that interacts with Pho85p; has sequence similarity to G1 cyclins PCL1 and PCL2              </P></DIV></ARTICLE><ARTICLE id="pHYM1" class="biobrick"><DIV class="graph"><H3 class="graph-title">                Promoter Activity By Alpha Factor Dose               </H3><P>                Data Not Available :(              </P></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 8              </P><P><A href="http://parts.igem.org/Part:BBa_K1346012">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                HYM1: Component of the RAM signaling network that is involved in regulation cellular morphogenesis; localizes to sites of polarized growth during budding and during the mating response              </P></DIV></ARTICLE><ARTICLE id="pPRM6" class="biobrick"><DIV class="graph"><H3 class="graph-title">                Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 619.4121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 562.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    2000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 505.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 448.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    4000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 391.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    5000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 334.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    6000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 277.1699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    7000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 220.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    8000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 163.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    9000                  </TEXT><TEXT transform="matrix(1 0 0 1 -79.9082 106.0059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10000                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    Mean GFP (AU        </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 306.0371)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 2              </P><P><A href="http://parts.igem.org/Part:BBa_K1346007">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                PRM6: Pheromone-regulated protein; predicted to have 2 transmembrane segments; regulated by Ste12p during mating              </P></DIV></ARTICLE><ARTICLE id="pPRM1" class="biobrick"><DIV class="graph"><H3 class="graph-title">                Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 619.4121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 562.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    2000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 505.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 448.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    4000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 391.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    5000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 334.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    6000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 277.1699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    7000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 220.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    8000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 163.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    9000                  </TEXT><TEXT transform="matrix(1 0 0 1 -79.9082 106.0059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10000                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    Mean GFP (AU                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 306.0371)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 3              </P><P><A href="http://parts.igem.org/Part:BBa_K1346004">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                PRM1: Pheromone-regulated multispanning membrane protein involved in membrane fusion during mating; localizes to the shmoo tip; regulated by Ste12p              </P></DIV></ARTICLE><ARTICLE id="pECM18" class="biobrick"><DIV class="graph"><H3 class="graph-title">                Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">      100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 619.4121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 562.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    2000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 505.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 448.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    4000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 391.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    5000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 334.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    6000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 277.1699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    7000 </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 220.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    8000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 163.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    9000                  </TEXT><TEXT transform="matrix(1 0 0 1 -79.9082 106.0059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10000                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    Mean GFP (AU                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 306.0371)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 1              </P><P><A href="http://parts.igem.org/Part:BBa_K1346008">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                ECM18: Protein of unknown function; similar to ribosomal-like protein 24              </P></DIV></ARTICLE><ARTICLE id="pPRM3" class="biobrick"><DIV class="graph"><H3 class="graph-title">                Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 619.4121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 562.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    2000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 505.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 448.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    4000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 391.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    5000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 334.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    6000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 277.1699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    7000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 220.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    8000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 163.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    9000                  </TEXT><TEXT transform="matrix(1 0 0 1 -79.9082 106.0059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10000                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    Mean GFP (AU                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 306.0371)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 3              </P><P><A href="http://parts.igem.org/Part:BBa_K1346006">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                PRM3: Pheromone-regulated protein required for nuclear envelope fusion during karyogamy; localizes to the outer face of the nuclear membrane; interacts with Kar5p at the spindle pole body              </P></DIV></ARTICLE><ARTICLE id="pSAG1" class="biobrick"><DIV class="graph"><H3 class="graph-title">                Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 619.4121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 562.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    2000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 505.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 448.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    4000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 391.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    5000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 334.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    6000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 277.1699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    7000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 220.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    8000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 163.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">  9000                  </TEXT><TEXT transform="matrix(1 0 0 1 -79.9082 106.0059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10000                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    Mean GFP (AU                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 306.0371)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 4              </P><P><A href="http://parts.igem.org/Part:BBa_K1346009">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                SAG1: Alpha-agglutinin of alpha-cells; binds to Aga1p during agglutination              </P></DIV></ARTICLE><ARTICLE id="pYDR124W" class="biobrick"><DIV class="graph"><H3 class="graph-title">                Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 619.4121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 562.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    2000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 505.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 448.334)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    4000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 391.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    5000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 334.2559)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    6000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 277.1699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    7000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 220.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    8000                  </TEXT><TEXT transform="matrix(1 0 0 1 -69.9082 163.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    9000                  </TEXT><TEXT transform="matrix(1 0 0 1 -79.9082 106.0059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10000                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    Mean GFP (AU                  </TEXT><TEXT transform="matrix(0 -1 1 0 -89.8223 306.0371)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 2              </P><P><A href="http://parts.igem.org/Part:">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                ORF of unknown function              </P></DIV></ARTICLE><ARTICLE id="pAGA1" class="biobrick"><DIV class="graph"><H3 class="graph-title">                Promoter Activity By Alpha Factor Dose               </H3><SVG version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" width="400px" viewbox="-130 80 850 650"><G><TEXT transform="matrix(1 0 0 1 28.3262 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 66.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 119.5918 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 266.1699 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    10                  </TEXT><TEXT transform="matrix(1 0 0 1 412.8418 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    100                  </TEXT><TEXT transform="matrix(1 0 0 1 559.4199 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1000                  </TEXT><TEXT transform="matrix(1 0 0 1 631.7559 689.4199)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3000                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 676.5059)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 -54.9082 583.584)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 490.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 -54.9082 397.748)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    1.5                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 304.8418)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    2                  </TEXT><TEXT transform="matrix(1 0 0 1 -54.9082 211.9121)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    2.5                  </TEXT><TEXT transform="matrix(1 0 0 1 -39.916 119.0918)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    3                  </TEXT><TEXT transform="matrix(1 0 0 1 -26.9941 94.8418)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:18;">                    x 10                  </TEXT><TEXT transform="matrix(1 0 0 1 6.998 83.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:12;">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 223.0059 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    [Alpha Factor](nM                  </TEXT><TEXT transform="matrix(1 0 0 1 413.7402 711.6699)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    )                  </TEXT><TEXT transform="matrix(0 -1 1 0 -64.8301 470.084)" style="fill:#6D6E71; font-family:'Helvetica'; font-size:24;">                    Mean GFP (AU)                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P>                Pheromone Response Elements: 7              </P><P><A href="http://parts.igem.org/Part:">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                AGA1: Anchorage subunit of a-agglutinin of a-cells;linked to adhesion subunit Aga2p via two disulfide bonds              </P></DIV></ARTICLE><ARTICLE id="pTEF1" class="biobrick"><DIV class="graph"><H3 class="graph-title">                pTEF1 Promoter Activity by Mutant              </H3><SVG version="1.1" id="pTEF-graphs" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" x="0px" y="0px" width="400px" viewbox="0 0 900.4 700" enable-background="new 0 0 900.4 700" xml:space="preserve"><G><TEXT transform="matrix(1 0 0 1 196.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    N/A                  </TEXT><TEXT transform="matrix(1 0 0 1 280.4219 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    pTEF1                  </TEXT><TEXT transform="matrix(1 0 0 1 399.4141 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m3                  </TEXT><TEXT transform="matrix(1 0 0 1 499.0859 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m6                  </TEXT><TEXT transform="matrix(1 0 0 1 598.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m7                  </TEXT><TEXT transform="matrix(1 0 0 1 691.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m1                  </TEXT><TEXT transform="matrix(1 0 0 1 725.0859 648.0938)" fill="#231F20" font-family="'Helvetica'" font-size="24">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 631.8281)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 568.4141)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 505)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 441.6641)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    1.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 378.25)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    2                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 314.9219)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    2.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 251.5)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    3                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 188.1719)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    3.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 124.75)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 61.4141)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    4.5                  </TEXT><TEXT transform="matrix(1 0 0 1 117.0859 47.9922)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    x 10                  </TEXT><TEXT transform="matrix(1 0 0 1 162.4219 33.1641)" fill="#51646B" font-family="'Helvetica'" font-size="16">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 360.8359 685.3203)" fill="#51646B" font-family="'Helvetica'" font-size="36">                    pTEF1 Mutants                  </TEXT><TEXT transform="matrix(0 -1 1 0 60.9219 463.1641)" fill="#51646B" font-family="'HelveticaNeue'" font-size="36">                    Mean GFP (AU)                </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P><A href="http://parts.igem.org/Part:BBa_K431008">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                TEF1: Translational elongation factor EF-1; constitutive expression              </P></DIV></ARTICLE><ARTICLE id="pTEF1_m3" class="biobrick"><DIV class="graph"><H3 class="graph-title">                pTEF1 Promoter Activity by Mutant              </H3><SVG version="1.1" id="pTEF-graphs" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" x="0px" y="0px" width="400px" viewbox="0 0 900.4 700" enable-background="new 0 0 900.4 700" xml:space="preserve"><G><TEXT transform="matrix(1 0 0 1 196.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    N/A                  </TEXT><TEXT transform="matrix(1 0 0 1 280.4219 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    pTEF1    </TEXT><TEXT transform="matrix(1 0 0 1 399.4141 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m3                  </TEXT><TEXT transform="matrix(1 0 0 1 499.0859 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m6                  </TEXT><TEXT transform="matrix(1 0 0 1 598.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m7                  </TEXT><TEXT transform="matrix(1 0 0 1 691.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m1                  </TEXT><TEXT transform="matrix(1 0 0 1 725.0859 648.0938)" fill="#231F20" font-family="'Helvetica'" font-size="24">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 631.8281)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 568.4141)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 505)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 441.6641)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    1.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 378.25)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    2                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 314.9219)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    2.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 251.5)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    3                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 188.1719)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    3.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 124.75)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 61.4141)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    4.5                  </TEXT><TEXT transform="matrix(1 0 0 1 117.0859 47.9922)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    x 10                  </TEXT><TEXT transform="matrix(1 0 0 1 162.4219 33.1641)" fill="#51646B" font-family="'Helvetica'" font-size="16">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 360.8359 685.3203)" fill="#51646B" font-family="'Helvetica'" font-size="36">                    pTEF1 Mutants                  </TEXT><TEXT transform="matrix(0 -1 1 0 60.9219 463.1641)" fill="#51646B" font-family="'HelveticaNeue'" font-size="36">                    Mean GFP (AU)                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P><A href="http://parts.igem.org/Part:BBa_K1346000">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                pTEF1 mutant by error-prone PCR              </P></DIV></ARTICLE><ARTICLE id="pTEF1_m6" class="biobrick"><DIV class="graph"><H3 class="graph-title">                pTEF1 Promoter Activity by Mutant              </H3><SVG version="1.1" id="pTEF-graphs" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" x="0px" y="0px" width="400px" viewbox="0 0 900.4 700" enable-background="new 0 0 900.4 700" xml:space="preserve"><G><TEXT transform="matrix(1 0 0 1 196.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    N/A                  </TEXT><TEXT transform="matrix(1 0 0 1 280.4219 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    pTEF1                  </TEXT><TEXT transform="matrix(1 0 0 1 399.4141 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m3                  </TEXT><TEXT transform="matrix(1 0 0 1 499.0859 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m6                  </TEXT><TEXT transform="matrix(1 0 0 1 598.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m7                  </TEXT><TEXT transform="matrix(1 0 0 1 691.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m1                  </TEXT><TEXT transform="matrix(1 0 0 1 725.0859 648.0938)" fill="#231F20" font-family="'Helvetica'" font-size="24">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 631.8281)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 568.4141)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 505)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 441.6641)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    1.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 378.25)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    2                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 314.9219)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    2.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 251.5)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    3                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 188.1719)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    3.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 124.75)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 61.4141)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    4.5                  </TEXT><TEXT transform="matrix(1 0 0 1 117.0859 47.9922)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    x 10                  </TEXT><TEXT transform="matrix(1 0 0 1 162.4219 33.1641)" fill="#51646B" font-family="'Helvetica'" font-size="16">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 360.8359 685.3203)" fill="#51646B" font-family="'Helvetica'" font-size="36">                    pTEF1 Mutants                  </TEXT><TEXT transform="matrix(0 -1 1 0 60.9219 463.1641)" fill="#51646B" font-family="'HelveticaNeue'" font-size="36">                    Mean GFP (AU)                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P><A href="http://parts.igem.org/Part:BBa_K1346001">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                pTEF1 mutant by error-prone PCR              </P></DIV></ARTICLE><ARTICLE id="pTEF1_m7" class="biobrick"><DIV class="graph"><H3 class="graph-title">                pTEF1 Promoter Activity by Mutant              </H3><SVG version="1.1" id="pTEF-graphs" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" x="0px" y="0px" width="400px" viewbox="0 0 900.4 700" enable-background="new 0 0 900.4 700" xml:space="preserve"><G><TEXT transform="matrix(1 0 0 1 196.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    N/A                  </TEXT><TEXT transform="matrix(1 0 0 1 280.4219 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    pTEF1                  </TEXT><TEXT transform="matrix(1 0 0 1 399.4141 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m3                  </TEXT><TEXT transform="matrix(1 0 0 1 499.0859 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m6                  </TEXT><TEXT transform="matrix(1 0 0 1 598.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m7                  </TEXT><TEXT transform="matrix(1 0 0 1 691.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m1                  </TEXT><TEXT transform="matrix(1 0 0 1 725.0859 648.0938)" fill="#231F20" font-family="'Helvetica'" font-size="24">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 631.8281)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 568.4141)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 505)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 441.6641)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    1.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 378.25)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    2                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 314.9219)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    2.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 251.5)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    3                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 188.1719)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    3.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 124.75)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 61.4141)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    4.5                  </TEXT><TEXT transform="matrix(1 0 0 1 117.0859 47.9922)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    x 10                  </TEXT><TEXT transform="matrix(1 0 0 1 162.4219 33.1641)" fill="#51646B" font-family="'Helvetica'" font-size="16">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 360.8359 685.3203)" fill="#51646B" font-family="'Helvetica'" font-size="36">                    pTEF1 Mutants                  </TEXT><TEXT transform="matrix(0 -1 1 0 60.9219 463.1641)" fill="#51646B" font-family="'HelveticaNeue'" font-size="36">                    Mean GFP (AU)                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P><A href="http://parts.igem.org/Part:BBa_K1346002">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                pTEF1 mutant by error-prone PCR              </P></DIV></ARTICLE><ARTICLE id="pTEF1_m10" class="biobrick"><DIV class="graph"><H3 class="graph-title">                pTEF1 Promoter Activity by Mutant              </H3><SVG version="1.1" id="pTEF-graphs" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" x="0px" y="0px" width="400px" viewbox="0 0 900.4 700" enable-background="new 0 0 900.4 700" xml:space="preserve"><G><TEXT transform="matrix(1 0 0 1 196.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    N/A                  </TEXT><TEXT transform="matrix(1 0 0 1 280.4219 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    pTEF1                  </TEXT><TEXT transform="matrix(1 0 0 1 399.4141 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m3                  </TEXT><TEXT transform="matrix(1 0 0 1 499.0859 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m6                  </TEXT><TEXT transform="matrix(1 0 0 1 598.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m7                  </TEXT><TEXT transform="matrix(1 0 0 1 691.75 648.0938)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    m1                  </TEXT><TEXT transform="matrix(1 0 0 1 725.0859 648.0938)" fill="#231F20" font-family="'Helvetica'" font-size="24">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 631.8281)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    0                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 568.4141)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    0.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 505)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    1                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 441.6641)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    1.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 378.25)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    2                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 314.9219)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    2.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 251.5)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    3                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 188.1719)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    3.5                  </TEXT><TEXT transform="matrix(1 0 0 1 100.8281 124.75)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 80.8281 61.4141)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    4.5                  </TEXT><TEXT transform="matrix(1 0 0 1 117.0859 47.9922)" fill="#51646B" font-family="'Helvetica'" font-size="24">                    x 10                  </TEXT><TEXT transform="matrix(1 0 0 1 162.4219 33.1641)" fill="#51646B" font-family="'Helvetica'" font-size="16">                    4                  </TEXT><TEXT transform="matrix(1 0 0 1 360.8359 685.3203)" fill="#51646B" font-family="'Helvetica'" font-size="36">                    pTEF1 Mutants                  </TEXT><TEXT transform="matrix(0 -1 1 0 60.9219 463.1641)" fill="#51646B" font-family="'HelveticaNeue'" font-size="36">                    Mean GFP (AU)                  </TEXT></G></SVG></DIV><DIV class="biobrick-tidbit"><P><A href="http://parts.igem.org/Part:BBa_K1346003">                  Biobrick Specification                </A></P></DIV><DIV class="biobrick-info"><P>                pTEF1 mutant by error-prone PCR              </P></DIV></ARTICLE></DIV></DIV></DIV></DIV><DIV id="model"><DIV class="project-container"><DIV class="wrapper clearfix menu"><A href="#background"><H2>            Background            
                                                    </H2></A><A href="#design"><H2>            Design            
                                                    </H2></A><A href="#achievements"><H2>            Achievements            
                                                    </H2></A><H1>Models</H1></DIV></DIV><DIV class="main-container"><DIV class="wrapper clearfix"><DIV id="computer-model" class="clearfix"><H2>Computational Models of Bar1 Negative Feedback</H2><H3>Part 1: Negative Feedback and Noise Reduction</H3><P><STRONG>Background:</STRONG> Noise can be defined as extracellular or intracellular perturbation which affects signal transduction as well as gene expression[<A href="#footnote-5">1</A>]. A signaling pathway can include mechanisms that reduce noise. Such reduction in biological noise may be necessary for pathway functionality. For example, the variety of T cell receptors (TCR) in a population of T cells generates a wide range of responses to a particular antigen, yet somehow, a very specific, low noise, response is achieved through its biological circuitry [<A href="footnote-6">2</A>]. The regulatory mechanisms downstream of TCR include negative feedback which allows stimulation signals to be controlled, guaranteeing appropriate responses to external perturbations. Also, noise is regarded as a quantitative index of variability within a population, which can be shown by the distribution of population response when stimulated. We wonder if adding a negative feedback to the production of extracellular communication factor will reduce population noise.</P><P><STRONG>Circuit Diagram:</STRONG> We have added the Bar1 gene, which is activated by alpha factor in the primary circuit [<A href="#footnote-7">3</A>]. Bar1 is a protease that cleaves alpha factor in the extracellular zone. So, alpha factor inhibits the accumulation of itself by producing Bar1, which forms a negative feedback for extracellular alpha factor. We can tune the strength of negative feedback with different alpha factor responsive promoters. </P><P><STRONG>Modeling Result:</STRONG> Here is the data plot using a stochastic model based on Chemical Langevin Equation  [<A href="#footnote-8">4</A>]. The data along the whole time period is integrated into histograms on the right. From the results, we find that strains with negative feedback loop have lower noise (Upper panel), while strains without it have higher noise (Lower panel). </P><H3>Part 2: Bimodal Response</H3><P><STRONG>Background: </STRONG>We explored another behavior pattern, bimodal response, which means when stimulated, the whole cell group differentiates into two subgroups - one of them activated while the other one remains in OFF state. </P><P><STRONG>Circuit Design:</STRONG> This is the circuit we designed to achieve bimodal response. Based on the primary secrete-sense circuit, we add a positive feedback, and a degradation factor Bar1. The strength of positive feedback can be tuned by doxycycline concentration, and the strength of degradation factor can be tuned by changing pTEF promoter in front of Bar1, which we characterized experimentally.</P><P><STRONG>Modeling Formula:</STRONG> In order to describe the dynamic of extracellular alpha factor concentration, we build a deterministic model based on this formula. </P><P>The first two terms represents generation of alpha factor: the first is the basal synthesis, and the second describes the positive feedback for generating alpha factor. We use sigmoidal form based on our pre-experiment results. The last two terms represents degradation of alpha factor. In this circuit, Bar1 is constitutively expressed, and the degradation of alpha factor by Bar1 is of first-order. The last term is natural degradation. </P><P>If we define this formula as the driving force, then the negative value of its integration can be defined as potential.</P><P><STRONG>Modeling Results: </STRONG>Assuming the basal synthesis of alpha factor is zero, we plot the secretion term and degradation term against concentration of alpha factor respectively.</P><P>Tuning the parameters to change the strength of positive feedback and degradation term, when extracellular alpha factor concentration stops changing at last, there are three fixed points – among which two are stable and one is unstable judged from Jacobian matrix. By calculating the potential (defined above), we correspond two of fixpoints to the two potential barriers where cells may gather. Thus a bimodal response appears. </P><P><STRONG>Parameter Tuning: </STRONG>In this model, the strength of positive feedback changes by tuning V, while the strength of Bar1 degradation changes by tuning deltaBar1. So, we set a range for these two parameters to see under which condition bimodal response appears. This figure shows tuning deltaBar1, the slope of the degradation line. For different values, the pattern is distinctive. Along with its decrease, the potential difference between the barrier and that of the well decreases, which means cells may have a bigger probability to jump out of the well by stochastic events, causing the bimodal pattern to collapse.</P><P>This figure shows the results of tuning V, the highest value of secretion curve. Also, along with its decrease, the trend is similar with changing deltaBar1, resulting in difficulty maintaining bimodal response within a cell population. </P></DIV><H2 style="border-top: 1px #ccc solid">Kilobots</H2><H3>What are Kilobots?</H3><P>Kilobots are small robots designed by the Self-Organizing Research Group at Harvard University to operate in large groups[<A href="#footnote-4">5</A>]. When programmed, the kilobots can cooperate through local interactions to perform collective behaviors such as swarming and self-assembly. They can flash LED lights, transmit and receive electronic signals to and from each other, and move using two vibration modules. For our purposes, we only need to use the LED light to report robot response.</P><H3>Why model with Kilobots?</H3><P>With their simple design, we hope to produce complex collective behavioral models that mimic our yeast circuits. By programming Kilobots to behave as our yeast cells, we hope to produce predicted community responses such as converging or diverging behaviors.</P><P>In our models, we utilized their ability to sense and secrete signals to produce community responses. A Kilobot can be in one of five states: NONE*, LOW, MED, HIGH, or MAX, which describe how frequently the cell transmits signals. A Kilobot’s state may vary depending on its program and the states of nearby Kilobots. </P><P>A Kilobot’s internal state is reported using its LEDs, which correspond to the following key:</P><UL><LI>None - Blue</LI><LI>Low - Cyan</LI><LI>Medium - White</LI><LI>High - Orange</LI><LI>Maximum - Red</LI></UL><P>*Kilobots in the NONE state send signals at a very low frequency; this is so that if isolated, the Kilobot will still be able to self-signal.</P><H3>Circuit Components</H3><OL><LI><P>i_sense_rand_bar</P><P>This code allows Kilobots (yeast cells) to sense and transmit electronic signals (mating factor alpha) that will influence other Kilobots’ states and may influence its own state (self-sensing). The Kilobots are randomized into different initial states, representing different initial constitutive expressions. There are five possible states, which describe the Kilobot’s output: NONE* [blue], LOW [cyan], MED [white], HIGH [orange], and MAX [red]. At each state, the cell lights up in the assigned color and transmits signals [value = 0] at the assigned frequency, NONE being the lowest level of frequency and MAX being highest. If a Kilobot senses enough signals transmitted by other Kilobots, it will go up in stage to the next highest level (so from NONE to LOW to MED to HIGH to MAX), until it reaches the MAX stage, at which point it will stay red. If the Kilobots sense enough signals transmitted by a i_bar1 Kilobot [value =1], it will go down a stage. </P></LI><LI><P>i_bar1</P><P>This code is meant to mimic yeast cells that produce Bar1, which degrades mating alpha factor and can cause negative feedback to reduce activity in nearby cells. The Kilobot will flash magenta every time it transmits a signal [value = 1], which may inhibit nearby bar1-sensitive Kilobots to reduce signal if exposed for long enough.</P></LI><LI><P>i_sense_average_bar_rand</P><P>This code allows Kilobots (yeast cells) to sense and transmit electronic signals (mating factor alpha) that will influence other Kilobots’ states and may influence its own state (self-sensing) in order to converge to a medium level of expression. The Kilobots are randomized into different initial states, representing different initial constitutive expressions. There are five possible states, which describe the Kilobot’s output: NONE [blue], LOW [cyan], MED [white], HIGH [orange], and MAX [red]. At each state, the cell lights up in the assigned color and transmits signals [value = 0] at the assigned frequency, NONE being the lowest level of frequency and MAX being highest. If a Kilobot senses enough signals transmitted by other Kilobots, it will go up in stage to the next highest level (so from NONE to LOW to MED). If the Kilobot initially is at a HIGH or MAX state, it will gradually reduce its expression until it matches that of other cells at the MED.

																				</P></LI></OL><H3>Results</H3><DIV class="clearfix"><H4>ConvergeHigh</H4><P>When all the Kilobots were programmed with code 1, they each started at different initial states, but they all reached maximum level of expression very quickly. Note that robots in the center seem to turn red faster while the slowest robot to turn red is in the bottom corner, where it gets less signal due to having fewer adjacent neighbors.</P><P>This progression from random to maximum is similar to how a noisy but highly expressing yeast strain would respond to mating factor alpha.</P></DIV><DIV class="clearfix"><H4>Diverging Bar1</H4><P>In this clip, seven Kilobots were given program 1 and three were given program 2 (flashing magenta) to mimic Bar1 cells. This produced a diverging response, in which robots further away from the Bar1 Kilobots still reached maximum expression level, but those closer to the Bar1 Kilobots stayed at low levels of expression, eventually settling at a medium level of expression. 
																					This diverging response may be similar to what happens when different strains of yeast with and without bar1 are co-cultured.</P></DIV><DIV class="clearfix"><H4>RandomtoBimodal</H4><P>In this clip, half of the Kilobots were programmed with code 1 and the other half were programmed with code 2. The initial state is similar to what happens in the first scenario (ConvergeHigh), in that initial response is random and noisy; however, the robots have clearly differentiated into medium and high levels of expression (white and red, respectively) by the end. 
																					This diverging yet coordinated response may be similar to what happens when two noisy strains of yeast are intermixed.</P></DIV><DIV class="clearfix"><H4>LowtoBimodal</H4><P>In this clip, half of the Kilobots were programmed with code 1 and the other half were programmed with code 2, this time with a few modifications such that they all started at the lowest level of expression (blue for NONE). Again, the robots ultimately produced a bimodal response, with half of the robots emitting white light for medium expression and the other half emitting red light for maximum expression.</P><P>This diverging response may model what happens when two strains of yeast with the same initial response but different secondary responses are mixed together and induced with mating factor alpha.</P></DIV><UL class="foot"><LI id="footnote-5">1. Blake, W. J., KAErn, M., Cantor, C. R., &amp; Collins, J. J. (2003). Noise in eukaryotic gene expression. Nature, 422(6932), 633–7. doi:10.1038/nature01546</LI><LI id="footnote-6">2. Höfer, T., Krichevsky, O., &amp; Altan-Bonnet, G. (2012). Competition for IL-2 between Regulatory and Effector T Cells to Chisel Immune Responses. Frontiers in Immunology, 3(September), 268. doi:10.3389/fimmu.2012.00268</LI><LI id="footnote-7">3. Gross, a, Rödel, G., &amp; Ostermann, K. (2011). Application of the yeast pheromone system for controlled cell-cell communication and signal amplification. Letters in Applied Microbiology, 52(5), 521–6. doi:10.1111/j.1472-765X.2011.03035.x</LI><LI id="footnote-8">4. Gillespie, D. T., &amp; Introduction, I. (2001). Approximate accelerated stochastic simulation of chemically reacting systems, 115(4), 1716–1733.</LI><LI id="footnote-4">5. Rubenstein, M., Ahler, C., &amp; Nagpal, R. (2012). Kilobot: A low cost scalable robot system for collective behaviors. 2012 IEEE International Conference on Robotics and Automation, 3293–3298. doi:10.1109/ICRA.2012.6224638</LI></UL></DIV></DIV></DIV><DIV class="footer-container"><FOOTER class="wrapper clearfix"><UL id="sitemap"><LI class="page"><H4><A href="./index.html">Home</A></H4></LI><LI class="page"><H4><A href="./project.html">Project</A></H4></LI><LI><A href="./project.html#background">Background</A></LI><LI><A href="./project.html#design">Design</A></LI><LI><A href="./project.html#achievements">Achievements</A></LI><LI><A href="./project.html#model">Models</A></LI></UL><LI class="page"><H4><A href="./judging.html">Judging</A></H4></LI><LI><A href="./judging.html#medal">Medal Requirements</A></LI><LI><A href="./judging.html#safety">Safety</A></LI><LI><A href="./judging.html#implications">Implications</A></LI><LI class="page"><H4><A href="./hp.html">Human Practices</A></H4></LI><LI><A href="./hp.html#super_science">Super Science!</A></LI><LI><A href="./hp.html#collaborations">Collaborations</A></LI><LI class="page"><H4><A href="./notebooks.html">Notebooks &amp; Protocols</A></H4></LI><LI><A href="./notebooks.html#notebooks">Notebooks</A></LI><LI><A href="./notebooks.html#protocols">Protocols</A></LI><LI class="page"><H4><A href="./team.html">Team</A></H4></LI><LI><A href="./team.html#team-members">Team Members</A></LI><LI><A href="./team.html#advisors">Advisors</A></LI><LI><A href="./team.html#sponsors">Sponsors</A></LI><LI><A href="./team.html#attributions">Attributions</A></LI><LI><A href="./team.html#contact">Contact</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=UCSF_UCB">iGEM Profile</A></LI><DIV class="social clearfix"><A href="https://twitter.com/iGEM_UCSF">@iGEM_UCSF</A></DIV><DIV class="social clearfix"><A href="https://www.facebook.com/pages/UCSF-IGEM/290769630936213">UCSF iGEM</A></DIV><DIV class="social clearfix"><A href="http://igem.ucsf.edu">igem.ucsf.edu</A></DIV><DIV class="copyright clearfix">© 2014 UCSF-UCB iGEM</DIV></FOOTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCSF_UCB/project.html">http://2014.igem.org/Team:UCSF_UCB/project.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCSF_UCB/project.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCSF_UCB/project.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCSF_UCB/project.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCSF_UCB/project.html&amp;oldid=378898" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>