<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCSF_UCB_jeffrey"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCSF_UCB/jeffrey">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCSF_UCB/jeffrey&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF_UCB/jeffrey&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSF_UCB/jeffrey&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCSF_UCB/jeffrey" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCSF UCB/jeffrey</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1>Jeffrey's Lab Notebook</H1><H2>6/9/14</H2><H3>PCR for Constitutive Promoters (pTEF1)</H3><TABLE><THEAD><TR><TH>Materials</TH><TH>1x reaction</TH><TH>4.5x Master Mix</TH></TR></THEAD><TBODY><TR><TD>x Phusion HF Buffer</TD><TD>10 µl</TD><TD>45 µl</TD></TR><TR><TD>dNTP's (10 mM)</TD><TD>1 µl</TD><TD>4.5 µl</TD></TR><TR><TD>Forward Primer (10µm)</TD><TD>2.5 µl</TD><TD>11.25 µl</TD></TR><TR><TD>Reverse Primer (10µm)</TD><TD>2.5 µl</TD><TD>11.25 µl</TD></TR><TR><TD>*Template DNA</TD><TD>0.3 µl</TD><TD>1.35 µl</TD></TR><TR><TD>Phusion Polymerase</TD><TD>0.5 µl</TD><TD>2.25 µl</TD></TR><TR><TD>Water</TD><TD>33.2 µl</TD><TD>149.4 µl</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Total</TH><TH>50 µl</TH><TH>225 µl</TH></TR></THEAD></TABLE><PRE><CODE>1. Mix materials in a 4.5x Master Mix on ice. Mix well.
2. Pipetter 50 µl from the Master Mix into 4 labeled PCR tubes
3. Thermocycler for : 

    Initial Duration   | 98° C | 30s 
      35 Cycles of: 
          Denaturation | 98° C | 10s
          Annealing    | 55° C | 20s
          Extension    | 72° C | 30s
    Final Extension    | 72° C | 5m
    Hold               | 4°  C | Forever

POSSIBLE ERRORS: Incomplete thawing of dNTPs
</CODE></PRE><P><STRONG>Working Stock of Reverse and Forward Primers for pTEF1 kept in my freezer box. 6/9/14</STRONG></P><H2>6/10/14</H2><H3>Gel Extraction</H3><PRE><CODE>Cut DNA band from PCR of constitutive promoter gel
Weigh gel: .720 g
QG Buffer: 2160 ul

1. Mix gel slice with QG Buffer in 50C heat bath for 10 min.
2. Add 1 ml isopropanol. Mix.
3. Add 750 ul of mixture into purple Qiagen spin column. Spin for 30 sec. Discard liquid.  Repeat.
4. Add 750 ul of Buffer PE and spin for 1 min. Discard waste.
5. Dry spin the column for 1 min. Replace the spin column in a new microcentrifuge tube
6. Elute in 40 ul of ddH2O. Wait 1 min and then spin for 1 min.
</CODE></PRE><H3>Restriction Digest with ApaI</H3><PRE><CODE>40 ul of DNA (pTEF1 gel extraction)
5 ul of 10x CutSmart Buffer
.5 ul of ApaI Enzyme

*digest overnight at room temperature*
</CODE></PRE><H2>6/11/14</H2><H3>Restriction Digest with XhoI</H3><PRE><CODE>Add .5 ul of XhoI enzyme to ApaI digestion from 6/10/14
Incubate in the 37°C shaker.
</CODE></PRE><H3>PCR Purification</H3><PRE><CODE>1. Add 250 ul of Buffer PB to digestion.
2. Place sample in a purple QIAquick spin column. Spin for 1 min. Discard waste.
3. Wash with 750 ul of Buffer PE. Spin for 1 min. Discard waste.
4. Dry spin for 1 min. Replace spin column in a new microcentrifuge tube.
5. 6. Elute in 50 ul of ddH2O. Wait 1 min and then spin for 1 min.

Total concentration: 141.4 ng/ul
</CODE></PRE><H3>DNA Ligation</H3><TABLE><THEAD><TR><TH>Materials</TH><TH>Volume</TH></TR></THEAD><TBODY><TR><TD>10x Ligase Buffer</TD><TD>1 ul</TD></TR><TR><TD>DNA Backbone PSV606</TD><TD>.2 ul</TD></TR><TR><TD>DNA Insert (PCR Purication)</TD><TD>.2 ul</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>.5 ul</TD></TR><TR><TD>H2O</TD><TD>8.1 ul</TD></TR></TBODY></TABLE><PRE><CODE>Total                       | 10 ul
----------------------------------------------------
Mix reagents and incubate at room temperature for 2 hrs
</CODE></PRE><P><STRONG>Ligation of pTEF1 into PSV606 kept in my freezer box. 6/11/14</STRONG></P><H3>Transformation</H3><PRE><CODE>10 ul of ligation
50 ul of E. Coli competent cells

30 min    | ice
45 sec    | 42°C heat shock
2 min     | ice

Add 250 ul of SOC media. Incubate at 37°C for 1 hr.
Plate on LB+Carb. 

NO COLONIES
</CODE></PRE><H2>6/12/14</H2><H3>Redo Transformation</H3><PRE><CODE>Follow procedure from 6/11/14 with minor alterations.
1. Use .4 ul of ligation instead of .2 ul.
2. Use more expensive competent cells.
</CODE></PRE><H2>6/13/14</H2><H3>Transforming α-inducible promoters</H3><PRE><CODE>Transforming 11 different inducible promoters into CB008 and CB008DB strains of yeast.

1. Boil salmon sperm DNA (ssDNA) for 10 min.--&gt;10 ul of 10 mg/ml stock per transformation
2. Cool on ice for 10 min. 
3. Pellet yeast cultures in centrifuge. (3000 rpm for 2-5 min.)
4. Resuspend with with .1M LiOAc in TE.
5. Pellet cells (3000 rpm for 2-5 min.)
6. Resuspend in 100 ul .1M LiOAc in TE per 2.5 ml culture. 
7. Aliquot 100 ul into each microcentrifuge per transformation. (22 tubes)
Per Tube:
8. Add 100 ug ssDNA, 1 ug of target DNA (1~5 ul)

ADD IN ORDER
9. 480 ul 50% PEG 3350
10. 60 ul 10x TE
11. 60 ul 1 M LiOAc
12. 75 ul DMSO

13. Vortex
14. Incubate at 42°C for 30 min. 
15. Pellet cells (6000 rpm for 2 min)
16. Resuspend with 500 ul YPD.
17. Pellet cells.
18. Resuspend with 50 ul  YPD
19. Plate on SDS-Ura. 

Incubate 2 days at 30°C.
</CODE></PRE><H2>6/16/14</H2><H3>Colony PCR for Screening Yeast</H3><PRE><CODE>1. Pick a single colony using a sterile wooden stick and patch on a dropout plate. 
   Take stick and rub into a dry PCR tube.
2. Add 10 ul of 20mM NaOH to each of the PCR tubes.
   Boil in pCR machine at 95°C for 20 min.

3. Set up PCR reaction as below:
                                        1x Reaction      7x Master Mix
                                        ---------------------------------
    2X Go Taq Green PCR Master Mix              10ul               70ul
    10 uM FW primer                              1ul                7ul
    10uM RV primer                               1ul                7ul
    Water                                        5ul               35ul 
    ----------------------------------------------------------------------
    Boiled Yeast cells (template)                3ul          

4. Set up Thermocycler for:
    95°C       | 5 min
    30x |95°C   | 45 sec
        |50°C   | 30 sec
        |72°C   | 1 min per kb
    72°C       | 10 min
    4°C            | hold
5. Run on 1% Agarose Gel

We ran 66 lanes for colony PCR
RESULTS: only 4 lanes were positive
POSSIBLE ERRORS: not enough DNA in tubes
</CODE></PRE><H3>Colony PCR on E.Coli for Constitutive Promoters</H3><PRE><CODE>1. Pick a single colony using a sterile wooden stick and mix in 25ul of water in a PCR tube. Do this for about 4-6 colonies. Use 5ul for the PCR reaction below, adn save the rest for later.

2. Set up PCR reaction as below:
                                            1x Reaction      6x master mix
                                        ----------------------------------
2X Go Taq Green PCR Master Mix                  10ul               60ul
10 uM FW primer                                  1ul                6ul
10uM RV primer                                   1ul                6ul
Water                                            3ul               18ul 
Bacterial cells (template)                       5ul                --      


Cycles:
            95°C               | 5 min
            30X:    95°C       | 45 s
                    55°C       | 30 s
                    72°C       | 1 min per kb 
            72°C               | 10 min

3. Analyze products on a 1% agarose gel. 

RESULTS: All lanes worked

4. Inoculate the colonies in 5 ml LB+Carb and incubate 37°C shaker overnight.
</CODE></PRE><H2>6/17/14</H2><H3>Re-inoculate the E.Coli with Constitutive Promoters</H3><H2>6/18/14</H2><H3>Dilution of Yeast Strains</H3><PRE><CODE>We diluted CB008 and CB008DB strains 1:20 times in YPD media.
</CODE></PRE><H3>Miniprep Constitutive Promoters from E.Coli</H3><PRE><CODE>1. Pellet bacterial cells
2. Resuspend in 250 ul in Buffer P1. Transfer to a microcentrifuge tube.
3. Add 250 ul Buffer P1. Invert to mix.
4. Add 350 ul Buffer P2 and invert immediately but gently.
5. Centrifuge for 10 min at 130000 rpm. Apply supernatants to QIAprep spin columns.
6. Centrifuge for 30-60 sec. Discard flow through.
7. Wash with 750 ul Buffer PE and centrifuge for 60 sec.
8. Discard flow through and dry spin for 1 min.
9. Place spin column in microcentrifuge tube and elute in 25 ul ddH20. Spin for 1 min.
</CODE></PRE><H2>6/20/14-6/21/14</H2><P>Out of the lab. Other lab members ran the flow cytometer on AFRPs and transformed yeast with constitutive promoters.</P><H2>6/23/14</H2><H3>Yeast Colony PCR on Transformed Yeast</H3><P><EM>used wrong primers so we need to redo on Tuesday</EM></P><H2>6/24/14</H2><H3>Re-do Yeast Colony PCR on Transformed Yeast</H3><H3>Innoculated Yeast colonies in 5ml of YPD</H3><H3>Flow Cytometer Notes</H3><OL><LI><P>Overnight cultures of CB008+AFRP+GFP diluted ~100x (to a final concentration of OD600 0.5-0.1, Saturated overnight cultures should be OD600 of ~7) in SD complete media, and grown for 3 hours, 1000rpm shaker, 30ºC. Growing in 2mL well plates</P></LI><LI><P>Induce with Alpha-factor. Stock is in 3mM. Final concentrations are 0, 1nM, 10nM, 100nM, 1000nM.  Alpha-factor cannot be refrozen, so throw leftover away.</P></LI><LI><P>Induce for 90mins, but no longer than 120mins</P></LI><LI><P>Transfer 250u of each well into a V-bottom 96-well plate containing 10ul of the fixing chemical Cyclohexamide. (thats 4 ul for every 100ul of culture)  Cyclohexamide stops protein production by inhibiting ribosomes.</P></LI><LI><P>Run on the flow cytometer.</P></LI></OL><P>Flow Cytometer things to remember:</P><UL><LI>check the sheath fluid box to see if its empty</LI><LI>check the waster container.  If full, dispose and add new bleach</LI><LI>take care not to leave the machine on run. Save sheath fluid by keeping on standby when not in use</LI><LI>find parameters using negative controls. (FSC, SSC, Fluorescense marker)</LI><LI>DO NOT RUN PLATE. RUN WELLS instead. (Run the first well by itself first b/c it needs time to create file folders.) this causes problems.

</LI><LI>Run plate only saves data of the first well.</LI><LI>Can rerun through the first well again which would suck up air.  Air no good for the machine.</LI></UL><LI>highlight wells and change settings on the right in the aquisition dashboard screen</LI><LI>mixing speed 180ul/sec</LI><LI>mix 3 times rather than 2</LI><LI>run at 1ul/sec for samples and 0.5ul/sec for negative controls when finding parameters.</LI><LI>make sre uL doesn't exceed samle to prevent sucking up air. (Rule of thumb is to run 50ul less than allotted sample sizes for wells.)</LI><P>Voltage settings were gotten by running negative controls and adjusting to readouts seen in histogram form of FSC, SSC, FITC</P><H3>Parameters of This Flow Experiment</H3><PRE><CODE>FSC: 250
SSC: 280
FITC(GFP): 550
B(RFP): 650

Flow rate: 1µL/sec
Sample Volume: 200µL
Mixing Volume: 100µL
Mixing speed: 180µL/sec
</CODE></PRE><H3>Flowjo Notes</H3><OL><LI>Export data from USB and transfer to iGEM2014 folder</LI><LI>Open FlowJo and drag data over to box</LI><LI>Set an appropriate gate</LI><LI>Go to data, select all data and press E. Select mean, geometric mean, and count.</LI><LI>Hit refresh. Addiction numerical data should be present.</LI><LI>Use mean of FITC(GFP).</LI></OL><H3>MatLab Notes</H3><OL><LI>Open up new script and comment the title of the experiment</LI><LI>Enter the alpha factor concentrations info. (X axis). Put in log form.</LI><LI>Enter the yGEM data. This will serve as y axis. (FITC means)</LI><LI>Define variables</LI><LI>Tyope figure info and run to show plot.</LI></OL><H2>6/25/14</H2><H3>Transformations of DH5a</H3><OL><LI>HY86E3</LI><LI>HY67E1</LI><LI>HYGE1</LI><LI>PTS98</LI><LI>PTS108</LI><LI>PTS133</LI><LI>PTS97</LI></OL><P><EM>plasmids for backbones in different integration sites and fluorescent tags</EM></P><H3>Glycerol stocks of Yeast Transformations (yGEM23-32)</H3><PRE><CODE>420 ul 50% glycerol
350 ul cells in YPD
vortex, store in -80C freezer
</CODE></PRE><H3>PCR for Parts Registry Promoters</H3><P>ASG7, CLG1, ECM18, HYM1, PCL2, PRM1, PRM2, PRM3, PRM6, SAG1, YDR124W</P><TABLE><THEAD><TR><TH>Materials</TH><TH>1x reaction</TH></TR></THEAD><TBODY><TR><TD>x Phusion HF Buffer</TD><TD>10 µl</TD></TR><TR><TD>dNTP's (10 mM)</TD><TD>1 µl</TD></TR><TR><TD>Forward Primer (10µm)</TD><TD>2.5 µl</TD></TR><TR><TD>Reverse Primer (10µm)</TD><TD>2.5 µl</TD></TR><TR><TD>*Template DNA</TD><TD>0.3 µl</TD></TR><TR><TD>Phusion Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD>Water</TD><TD>33.2 µl</TD></TR><TR><TD>Total</TD><TD>50 µl</TD></TR><TR><TD>-----</TD><TD>------</TD></TR></TBODY></TABLE><PRE><CODE>3. Thermocycler for : 

    Initial Duration   | 98° C | 30s 
      35 Cycles of: 
          Denaturation | 98° C | 10s
          Annealing    | 55° C | 20s
          Extension    | 72° C | 30s
    Final Extension    | 72° C | 5m
    Hold               | 4°  C | Forever

Ran gel on PCR products. 1% Agarose, 125 volts for 15 min, 5 ul SybrSafe
</CODE></PRE><H2>6/26/14</H2><P>Only pGEM2 and pGEM3 worked. RETRY with DMSO (1.5 ul per reaction).</P><H3>PCR</H3><PRE><CODE>Master Mix:
170 ul HF Buffer
17 ul dNTPs
8.5 ul Phusion Polymerase
25.5 DMSO
539 ul Water
-------------
44.7 per tube


Also add 2.5 ul FW primers
         2.5 ul RV primers
         .3 ul template DNA
</CODE></PRE><P>Made liquid cultures of transformed DH5a E. Coli cells with HY+PTS plasmid</P><H2>6/27/14</H2><H3>Double Digestion of Constitutive Promoters-GFP</H3><PRE><CODE>6.5 ul water
15 ul DNA template (pTEF1 plasmids)
.5 ul Xho1 enzyme
.5 ul Not1 enzyme   
2.5 ul CutSmart Buffer

Incubate at 37C for 2hrs
Load into agarose gel. *pTEF1 appers to be smaller than the others.
</CODE></PRE><H3>Gel Extraction</H3><PRE><CODE>Exise the DNA bands from the gel. Weigh using a scale.
pGEM12- 120 mg      -360 ul         -120 ul
pGEM13- 110 mg      -360 ul         -110 ul
pGEM14- 110 mg      -330 ul         -110 ul
pGEM15- 100 mg      -300 ul         -100 ul
pGEM16- 170 mg      -510 ul         -170 ul
        Weights     QG Buffer       Isopropanol
1. Melt gels in 50C for 10 min in QG Buffer
2. Add isopropanol.
3. Place in a spin column and spin for 1 min 13000rpm
4. Add 500 ul QG Buffer
5. Spin for 1 min. Dump liquid.
6. Add 750 ul PE Buffer
7. Spin for 1 min. Dump Liquid.
8. Dry spin for 1 min.
9. Elute in 25 ul ddH20.


Concentrations (ng/ul) through NanoDrop: 
1. pGEM12-25.77 
2. pGEM13-47.93 
3. pGEM14-18.31
4. pGEM15-10.6
5. pGEM16-32.45
</CODE></PRE><H3>Gibson Assembly</H3><PRE><CODE>50 ng of DNA Backbone
2 ul of rtTA insert
5 ul Gibson Assembly Master Mix

pGEM12-2 ul         1 ul
pGEM13-1 ul         2 ul
pGEM14-3 ul         -
pGEM15-4 ul         -
pGEM16-1.5 ul       1.5 ul  
    DNA backbone    water
All have 2 ul rtTA insert and 5 ul GAMM
Incubate in thermocycler 50C for 15 min.
</CODE></PRE><H3>Transforming E. Coli with Gibson Assembly</H3><PRE><CODE>10 ul Gibson mix
25 ul Mach1 cells
-----------
250 ul SOC media

30 min ice
45 sec heatshock at 42C
2 min ice
Incubate at 37C for 1 hr.
Plated 250 ul on LB-Cam (WRONG SELECTION MARKER- LB-Carb)
Incubate in a drawer overnight
</CODE></PRE><H2>6/29/14</H2><PRE><CODE>pTEF1+rtTA transformations failed! 
Dilute CB008 constututive promoters (pTEF1)1:20 times
Incubate overnight 30C shaker/
</CODE></PRE><H2>6/30/14</H2><PRE><CODE>Repeat pGEM12,pGEM15, rtTA insert digestions with Minipreps from freezer box
Same procedure as from 6/27/14

pGEM12-21.05 ng/ul
pGEM15-23.03 ng/ul
rtTA-182.3 ng/ul

REDO Gibson Assembly (procedure from 6/27/14).
Plate all Gibson mix and incubate 37C overnight on LB-CAM.
</CODE></PRE><H2>7/1/14</H2><H3>Seamless Cloning</H3><PRE><CODE>pGEM12- 2 ul            3 ul
pGEM13- 1 ul            2 ul
pGEM14- 3 ul            4 ul
pGEM15- 2 ul            3 ul
pGEM16- 1.5 ul          2.5 ul
Positive- 1 ul PUC      3 ul        2 ul control insert
Negative- 2 ul          2 ul
        DNA backbone    Enzyme Mix

All with 1 ul of rtTA insert except for positive and negative control.
</CODE></PRE><H3>Transform into C2987 competent cells</H3><PRE><CODE>25 ul competent cells
3 ul Seamless Assembly Mix

30 min ice
45 sec heatshock 42C
2 min ice
Add 250 ul SOC media
1 hr incubation 37C

Plate all on LB+Cam
</CODE></PRE><H3>Miniprep HY/PTS cultures</H3><PRE><CODE>Follow same miniprep procedure as previous experiments.
Concentrations (in ng/ul)
1. PTS98    66.8
2. PTS97    88.2
3. HY67E1   63.43
4. HY6E1    22.05
5. PTS108   120.8
6. HY86E3   118.6
7. PTS133   701.3
</CODE></PRE><H3>Transforming pTEF1+GFP into DH5a</H3><PRE><CODE>.5 ul plasmid DNA (pGEM12 to pGEM16)
</CODE></PRE><H2>7/2/14</H2><P>Plated Seamless Cloning Transformation on LB+Carb, not LB+Cam.</P><H2>7/3/14</H2><OL><LI>Colonies found on LB+Carb plates.</LI><LI>Negative control has 5 colonies(?)</LI><LI>Positive control has too numerous to count.</LI><LI>All other colonies had ~100 colonies.</LI></OL><H3>GFP</H3><PRE><CODE>Innoculated GFP+pGEM12 to pGEM16 colonies. Stored in -80C freezer.
Move plates from 4C to 37C incubator on Sunday.
</CODE></PRE><H3>rtTA</H3><PRE><CODE>We ran a colony PCR on pGEM12-16 + rtTA.
Follow the same procedure as before, using forward/reverse primers for rtTA.
Used a 50C annealing temperature instead of 55C due to melting point of rtTA.
Colonies all worked! Innoculated all colonies and placed in 4C freezer.
</CODE></PRE><H2>7/6/14</H2><P>Moved all tubes from 4C freezer to 37C incubator shaker.</P><H2>7/7/14</H2><H3>Miniprepped all tubes</H3><PRE><CODE>Concentrations (in ng/ul)of rtTA plasmids
1. pGEM12-1     540.2
2. pGEM12-2     560.4
3. pGEM12-3     375.6
4. pGEM13-1     360.3
5. pGEM13-2     488.9
6. pGEM13-3     ------
7. pGEM14-1     302.3
8. pGEM14-2     425.5
9. pGEM14-3     513.5
10. pGEM15-1    595.8
11. pGEM15-2    554.4
12. pGEM15-2    510.6
13. pGEM16-1    592.3
14. pGEM16-2    545.8
15. pGEM16-3    302.3

Concentrations (in ng/ul) of GFP plasmids
1. pGEM12- 425.5
2. pGEM13- 620.3
3. pGEM14- 521.9
4. pGEM15- 567.7    
5. pGEM16- 534.3
</CODE></PRE><P>*discovered that rtTA inserts are wrong because activation domains may not have been copied with PCR</P><H2>7/8/14</H2><H3>Transformed 11E-3128 into DH5a (pTET-mfa)</H3><PRE><CODE>50 ul DH5a
.5 ul plasmid DNA
Heatshock transformation
Plate 100 ul on LB+Carb
</CODE></PRE><OL><LI>pGEM13+rtTA sequencing did not work (weird insertion in rtTA before terminator sequence)</LI><LI>Innoculate pGEM16+GFP again.</LI></OL><H2>7/9/14</H2><H3>Miniprepped pGEM16+GFP</H3><H3>Re-PCR of rtTA insert</H3><TABLE><THEAD><TR><TH>Materials</TH><TH>1x reaction</TH></TR></THEAD><TBODY><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD></TR><TR><TD>dNTP's (10 mM)</TD><TD>1 µl</TD></TR><TR><TD>Forward Primer (#70)</TD><TD>2.5 µl</TD></TR><TR><TD>Reverse Primer (#39)</TD><TD>2.5 µl</TD></TR><TR><TD>*Template DNA</TD><TD>1 µl</TD></TR><TR><TD>Phusion Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD>Water</TD><TD>31 µl</TD></TR><TR><TD>DMSO</TD><TD>1.5 ul</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Total</TH><TH>50 µl</TH></TR></THEAD></TABLE><PRE><CODE>Thermocycler Cycles:
    Initial Duration   | 95° C | 5 min 
      30 Cycles of: 
          Denaturation | 95° C | 30s
          Annealing    | 50° C | 30s
          Extension    | 72° C | 2 min
    Final Extension    | 72° C | 5m
    Hold               | 4°  C | Forever

*used 10 ul of dNTPs instead of 1 ul
</CODE></PRE><H3>Pme1 Digest of pGEM12-16 (without pGEM13) +rtTA</H3><PRE><CODE>Digest 2000ng of template DNA
Pme1 enzyme-        .5 ul
Cutsmart Buffer-    2.5 ul
DNA plasmid-        4 ul

Incubate 1 hr 37C.
</CODE></PRE><H3>Ran a gel for rtTA PCR</H3><H3>PCR Purification of rtTA</H3><PRE><CODE>Eluted in 50 ul
</CODE></PRE><H3>Gibson Assembly</H3><PRE><CODE>pGEM12-         1 ul
rtTA insert-    3 ul
GA MM-          4 ul

Incubate 15 min at 50C.
</CODE></PRE><H3>Transform</H3><PRE><CODE>25 ul Mach1 cells
.5 ul pGEM13+rtTA plasmid

Plate all on LB+Carb
</CODE></PRE><H2>7/10/14</H2><H3>Colony PCR of pGEM13+rtTA</H3><PRE><CODE>-same procedure as 7/9/14
-successful
</CODE></PRE><H3>Miniprep 11E 3128 (pTET+mfalpha)</H3><PRE><CODE>-followed standard miniprep procedure.
Concentration: 430 ng/ul
SEQUENCING CORRECT! (7/14/14)
</CODE></PRE><H2>7/11/14</H2><H3>Miniprep of pGEM13+rtTA</H3><H3>Sequencing correct</H3><PRE><CODE> pGEM16 (m10+GFP mislabeled m3+GFP)
and pTET+mfalpha
</CODE></PRE><H2>7/15/14</H2><H3>PCR rtTA out of correct pGEM20</H3><PRE><CODE>*threw away all incorrect rtTA inserts
5x HF Buffer                        10 ul
dNTPS                               1 ul
Phusion Polymerase                  .5 ul
pGEM20- rtTA template               1 ul
H2O                                 21 ul
FW Primer (different for pTEFs)     2.5 ul
RV Primer (rtTA reverse primer)     2.5 ul
DMSO                                1.5 ul

Thermocycler:
    95  5 min
    30x cycle:
        95  45 sec
        50  30 sec
        72  2 min
    72  5 min
    4   hold
</CODE></PRE><H2>7/16/14</H2><H3>Load pGEM1-13+rtTA on gel</H3><PRE><CODE>Gels were leaky and had spillage on contents
</CODE></PRE><H3>Gel extractions</H3><PRE><CODE>1. PRM1     .54g
2. PRM2     .56g
3. PRM3     1.15g
4. PRM6     1.21g
5. ECM18    .61g
6. SAG1     .68g
7. YDR124W  .45g
8. CLG1     .40g
9. ASG7     .31g
10. HYM1    .50g
11. PCL2    .42g
12. M3      .69g
Followed same gel extraction procedure
</CODE></PRE><H3>Gibson Assembly for m3</H3><PRE><CODE>3 ul insert
1 ul backbone
5 ul GA Master Mix
1 ul H2O
</CODE></PRE><H3>PCR of rtTAs (again)</H3><PRE><CODE>same procedure as before, except:
use 20 ul gel extraction as template
</CODE></PRE><H3>Transform m3+rtTA cells from Seamless Cloning</H3><PRE><CODE>25 ul cells
1 ul Gibson DNA
</CODE></PRE><H2>7/17/14</H2><H3>Colony PCR on m3+rtTA</H3><PRE><CODE>all had the rtTA insert!
made liquid cultures and incubated overnight
</CODE></PRE><H2>7/18/14</H2><H3>Miniprep pGEM13+rtTA</H3><PRE><CODE>eluted in 50 ul
1. 500.2 ng/ul
2. 386.9 ng/ul
3. 433.4 ng/ul
4. 400.5 ng/ul
</CODE></PRE><H2>7/20/14</H2><PRE><CODE>Sequencing came back conclusive! Silent mutation at 417 bp.
</CODE></PRE><H2>7/21/14</H2><PRE><CODE>Innoculated pTET+GFP yeast strains in 5ml YPD for transformation
</CODE></PRE><H2>7/22/14</H2><H3>Prepare cultures for transformation</H3><PRE><CODE>Diluted pTET+GFP yeast strains 1:20. Incubated for 3 hrs.
</CODE></PRE><H3>Pme1 Digest</H3><PRE><CODE>Pme1 digest to linearize the pTEF1 m3+rtTA circular plasmid for yeast transformation.

Digest 2000 ng DNA.
4 ul DNA
.5 ul Pme1 Enzyme
2.5 ul Cutsmart Buffer
18 ul H2O

Incubate at 37C for 1 hr.
</CODE></PRE><H3>Transform Yeast</H3><PRE><CODE>Transformed pTEF1-mr+rtTA into CB008/CB008DB pTET+GFP.
Follow standard procedure for transforming yeast found in our protocols page. 
-maybe not enough salmon sperm
Plated on SD URA in the 30C incubator for 2 days.
</CODE></PRE><H2>7/23/14</H2><H3>Transforming E.coli with Fluorescent Proteins</H3><PRE><CODE>25 ul DH5a cells
.5 ul DNA
</CODE></PRE><H2>7/24/14</H2><H3>Yeast Colony PCR</H3><PRE><CODE>Followed colony PCR procedure from before
Used primers for PSV606
Used 1.5 ul DMSO per reaction
3 min annealing step

FAILED
</CODE></PRE><H2>7/25/14</H2><PRE><CODE>Diluted CB008/DB pTET+GFP strains for transformations 
</CODE></PRE><H3>Transformations of m3+rtTA</H3><PRE><CODE>used 4 ul DNA
Plate on SD URA
</CODE></PRE><H2>7/28/14</H2><H3>Yeast Colony PCRS</H3><PRE><CODE>1 ul of FW primer
1 ul of RV primer
10 ul Green MM
10 ul Yeast cells
1.5 ul DMSO

Ura: 606/606
His: RA145/RA146
Trp: 604v2/604v2

-FAILED-
</CODE></PRE><H3>Yeast Transformations</H3><PRE><CODE>Transformed pAGA+mCherry into yeast strains.
(All with pTET+GFP, and pTET+mfalpha)
CB008DB m6+rtTA
        m7+rtTA
CB008   m10+rtTA
        m6+rtTA
        m7+rtTA
        pTEF1+rtTA

Transformed pTET mfalfpha into CB008DB pTET+GFP
        pTEF1+rtTA

Transformed m3+rtTA into pTET+GFP CB008/DB
</CODE></PRE><H3>Miniprep</H3><PRE><CODE>PTS94- 150 ng/ul
PSW696- 60 ng/ul
PS2607- 125 ng/ul
</CODE></PRE><H3>Colony PCR</H3><PRE><CODE>Colony PCR on URA, HIS, and TRP sites.
Repeat Colony PCR done earlier today.
</CODE></PRE><H2>7/29/14</H2><H3>Colony PCR</H3><PRE><CODE>Repeat Colony PCR from yesterday.
FAILED.
Possibly errors in the primers for TRP. should show 300 bp.
Use a 100 bo ladder?
</CODE></PRE><H3>More Colony PCR</H3><PRE><CODE>Repeat URA and TRP colony PCRs
Made own Green MM from (per reaction)
    .125 ul Gotaq enzyme
    5 ul 5x Buffer
    .5 ul DNTPs
Still FAILED.
</CODE></PRE><H2>7/29/14</H2><PRE><CODE>More failed colony PCRs
</CODE></PRE><H2>7/31/14</H2><H3>PCR Purification</H3><PRE><CODE>BFP-118.4 ng/ul
    digest with Xho1/Not1
Aga-144.2 ng/ul
    digest with Apa1/Xho1
</CODE></PRE><H3>Double Digestion of BFP</H3><PRE><CODE>Digest 2000ng
17 ul   BFP PCR Purification
.5 ul   Xho1 Enzyme
.5 ul   Not1 Enzyme
2.5 ul  Cutsmart Buffer
4.5 ul  Water

Incubate 37C for 3 hrs
</CODE></PRE><H3>Digestion of RFP</H3><PRE><CODE>Digest 2000ng
15 ul   pAGA1 PCR Purification
.5 ul   Apa1 Enzyme
2.5 ul  Cutsmart Buffer
7 ul    Water

Incubate Room Temperature for 2 hrs

Add .5 ul Xho1 Enzyme   

Incubate 37C for 1 hr
</CODE></PRE><H3>Digest mCherry+pAGA1 with Pme1 for Transformation</H3><PRE><CODE>6 ul    pGEM22
.5 ul   Pme1 Enzyme
2.5 ul  Cutsmart Buffer
16 ul   Water

Incubate 37C for 1 hr
</CODE></PRE><H3>Transformations (Yeast)</H3><PRE><CODE>Transformed pAGA1+mCherry into
(CB008):    pTEF1
            m6  
            m7
(CB008DB)   m6
            m7

All contain the other motifs in the circuit.
</CODE></PRE><H3>PCR Purify Digestions</H3><PRE><CODE>Followed procedure for PCR Purifications in protocols.
Final Concentrations:
BFP: 17.71 ng/ul
RFP: 21.75 ng/ul
</CODE></PRE><H3>Ligations</H3><PRE><CODE>1. HY130E Backbone with pAGA1+BFP (TRP)
2. HY130E Backbone with pPCL2+BFP (TRP)

1)  3 ul    Backbone
    1 ul    BFP
    1 ul    pAGA1
    1 ul    T4 Ligase
    1 ul    10X Buffer
    3 ul    Water

2)  3 ul    Backbone
    1 ul    BFP
    .5 ul   pPCL2
    1 ul    T4 Ligase
    1 ul    10X Buffer
    3.5 ul  Water

Incubate room temperature
Kept in Ianto's freezer box
</CODE></PRE><H2>8/1/14</H2><H3>Pme1 Digest of pTET+mfalpha</H3><PRE><CODE>Same protocol as before
</CODE></PRE><H3>Transformations (Yeast)</H3><PRE><CODE>CB008 m3 transform- m3+rtTA
CB008DB m10 transform- pTET+mfalpha
</CODE></PRE><H2>8/4/14</H2><H2>Remade 20mM NaOH</H2><H2>Colony PCR</H2><PRE><CODE>CB008 pAGA+mCherry:
    m6, m7
CB008DB pAGA+mCherry:
    m6, m7
</CODE></PRE><H2>Made Liquid Cultures</H2><PRE><CODE>CB008:
    pTET+mfalpha
        pTEF1, m3, m6, m7
    m10+rtTA
CB008DB 
    pTET+mfalpha
        pTEF1, m6, m7
    pTET+GFP (for m3+rtTA transformations)
    pAGA1+mCherry m10
</CODE></PRE><H2>8/5/14</H2><H3>Colony PCR</H3><PRE><CODE>CB008/DB pAGA1+mCherry
    m6, m7
Most of the colonies worked
</CODE></PRE><H3>Gel</H3><PRE><CODE>Ran a gel of:
Pme1 Digest of pAGA1+mCherry
               pTET+mfalpha
</CODE></PRE><H2>8/7/14</H2><H3>Liquid Cultures</H3><PRE><CODE>5 ml YPD cultures of the 8 mfalpha strains from 8/4/14 to transform pPCL2+mCherry
</CODE></PRE><H3>Exploratorium Presentation</H3><H2>8/8/14</H2><H3>Dilutions</H3><PRE><CODE>Made 1:20 dilutions of pTET+mFalpha strains:
CB008: pTEF1, m6, m7, m10
CB008DB: pTEF1, m3, m6, m10
*for pPCL2+RFP transformations
</CODE></PRE><H3>Colony PCR</H3><PRE><CODE>CB008:
    m3+rtTA 
    pTEF1, pAGA1+mCherry
CB008DB:
    m3, pAGA1+mCherry
    pTEF1, pAGA1+mCherry
    m6, pAGA1+mCherry
    m10, pTET+mfalpha

Boiled 100C for 20 min
Primers-    URA: 606
            HIS: RA 145/146
            TRP: RA 145/148
</CODE></PRE><H3>Pme1 Digests</H3><PRE><CODE>(Same for all digests)
.5 ul   Pme1 Enzyme
2.5 ul  Cutsmart Buffer

RFP:    9 ul of pPCL2+mCherry 
        13 ul of water
BFP:    4 ul of pPCL2+BFP
        18 ul of water

Did twice for more transformations
</CODE></PRE><H3>Transformations</H3><PRE><CODE>Tranformed pPCL2+BFP and pPCL2+mCherry into mfalpha strains
</CODE></PRE><H2>8/11/14</H2><H3>Colony PCR</H3><PRE><CODE>BFP and RFP transformations from 8/8/14
Used RA145/148 for TRP
1.5 ul DMSO per reaction
</CODE></PRE><H3>Transformations (Yeast)</H3><PRE><CODE>pTET+mfalpha into CB008 m3
pAGA1+mCherry into CB008DB m10
pPCL2+RFP into CB008DB m10
pPCL2+BFP into CB008DB m10
</CODE></PRE><H3>Streaked Plates</H3><PRE><CODE>CB008
    pTEF1, pAGA1+mCherry
    m3+rtTA
CB008DB
    pAGA1+mCherry
        pTEF1, m3, m6
    m10, pTET+mfalpha
</CODE></PRE><H2>8/12/14</H2><H3>Glycerol stocks</H3><PRE><CODE>CB008
    pTEF1, pAGA1+mCherry
    m3+rtTA
CB008DB
    pAGA1+mCherry
        pTEF1, m3, m6
    m10, pTET+mfalpha

(yGEM69-74)
</CODE></PRE><H3>Pme1 Digestions</H3><PRE><CODE>pPCL2+BFP
pPCL2+mCherry
pAGA1+mCherry
pTET+mfalpha
</CODE></PRE><H3>Colony PCR</H3><PRE><CODE>pPCL2+BFP
pPCL2+mCherry 
</CODE></PRE><H3>Transformations (DH5a cells for Miniprepping)</H3><PRE><CODE>pTET+mfalpha
pPCL2+BFP
pPCL2+mCherry
pAGA1+mCherry
</CODE></PRE><H3>Transformations (Yeast)</H3><PRE><CODE>CB008 m3, pTET+mfalpha
CB008DB m10
    pPCL2+mCherry, pAGA1+mCherry, pPCL2+BFP
</CODE></PRE><H3>Pme1 Digest</H3><PRE><CODE>pTET+mfalpha
</CODE></PRE><H3>Streaked Plates</H3><PRE><CODE>Streaked mCherry and BFP successful colonies from Colony PCR
</CODE></PRE><H3>Liquid Cultures of DH5a transformations</H3><H2>8/13/14</H2><H3>Dilutions</H3><PRE><CODE>Made 1:20 dilutions of BFP/RFP cultures
CB008 BFP
    pTEF1, m6, m7, m10
CB008 RFP
    m6
CB008DB RFP
    m7, m10
CB008DB BFP
    m10
</CODE></PRE><H3>Colony PCR</H3><PRE><CODE>Colony PCR of BFP/RFP transformations
-FAILED-
</CODE></PRE><H3>Miniprepped DH5a transformations for plasmids</H3><PRE><CODE>pTET+mfalpha
RFP
BFP
</CODE></PRE><H3>Streak/Glycerol Stocks</H3><PRE><CODE>CB008 
    pTEF1- RFP/BFP
    m6- RFP/BFP
    m7- RFP/BFP
    m10- BFP
CB008DB
    pTEF1- RFP/BFP
    m3- RFP/BFP
    m6- RFP/BFP
    m7- RFP/BFP
</CODE></PRE><H3>Transformations</H3><PRE><CODE>CB008
    m6, RFP
    m10, BFP
</CODE></PRE><H2>8/14/14</H2><H3>Colony PCR</H3><PRE><CODE>CB008 m3, pTET+mfalpha
CB008DB m10, pPCL2+RFP
             pAGA1+RFP
             pPCL2+BFP
*used .5 ul Zymolyase per reaction
Thermocycle:
    37C         5 min
    95C         5 min
    30x 95C     45 sec
        50C     30 sec
        72C     1 min
    72C         10 min
    4C          hold
</CODE></PRE><H2>8/17/14</H2><H3>Colony PCR</H3><PRE><CODE>RFP/BFP strains
56 colonies
used RA145/148 and zymolyase
</CODE></PRE><H2>8/18/14</H2><H3>Colony PCR Check</H3><PRE><CODE>pAGA1s successful
pPCL2 failed
</CODE></PRE><H3>Colony PCR</H3><PRE><CODE>pPCL2+mCherry
pPCL2+BFP
pSAG1+Ste2 (use 87/91 primers. around 600bp)
</CODE></PRE><H3>Transformations (Yeast)</H3><PRE><CODE>AFRPS:
7 pPCL2+mCherry 
1 pAGA1+mCherry
4 rtTAs

m3 CB008
pTET+mfalpha
</CODE></PRE><H2>8/19/14</H2><H3>Streaked Plates</H3><PRE><CODE>pPCL2+mCherry AFRPS
m10, pPCL2+BFP
</CODE></PRE><H3>Liquid Cultures</H3><PRE><CODE>23 things for glycerol stocks and transformations
</CODE></PRE><H2>8/20/14</H2><H3>Pme1 Digests</H3><PRE><CODE>Digest 4000 ng
pPCL2+BFP
pAGA1+RFP
pTET+mfalpha

Incubate 2 hrs 37C
</CODE></PRE><H3>Colony PCR</H3><PRE><CODE>*extended extension time to 2 min* ~940bp
Ste2 in NAT integration site
87/91 primers
</CODE></PRE><H3>Transformations (Yeast)</H3><PRE><CODE>pPCL2+BFP:
PRM1, PRM2, PRM3, YDR124W, CLG1, ASG7, HYM1
</CODE></PRE><H3>Glycerol Stocks of yGEM101-126</H3><H2>8/21/14</H2><H3>Restreak ALL Plates</H3><PRE><CODE>Yeast plates die after 2 weeks
Restreak on new YPD
</CODE></PRE><H3>Colony PCR</H3><PRE><CODE>CB008/DB m6 for:
    pTET+mfalpha, pAGA1+RFP
*flow data suggests false positives of RFP
</CODE></PRE><H3>Transformations (Yeast)</H3><PRE><CODE>pSAG1+rtTA
pPCL2+rtTA
pPRM6+rtTA, suspended
pECM18+rtTA, suspended
m3, pTET+mfalpha
</CODE></PRE><H3>Made new LiOAC, 10x TE Buffer, PEG 3350</H3><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCSF_UCB/jeffrey">http://2014.igem.org/Team:UCSF_UCB/jeffrey</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCSF_UCB/jeffrey" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCSF_UCB/jeffrey" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCSF_UCB/jeffrey&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCSF_UCB/jeffrey&amp;oldid=205837" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>