"
Page
Discussion
View source
History
teams
Log in
 
Team:UCSF UCB/jeffrey
From 2014.igem.org
Jeffrey's Lab Notebook
6/9/14
PCR for Constitutive Promoters (pTEF1)
Materials
1x reaction
4.5x Master Mix
x Phusion HF Buffer
10 µl
45 µl
dNTP's (10 mM)
1 µl
4.5 µl
Forward Primer (10µm)
2.5 µl
11.25 µl
Reverse Primer (10µm)
2.5 µl
11.25 µl
*Template DNA
0.3 µl
1.35 µl
Phusion Polymerase
0.5 µl
2.25 µl
Water
33.2 µl
149.4 µl
Total
50 µl
225 µl
1. Mix materials in a 4.5x Master Mix on ice. Mix well. 2. Pipetter 50 µl from the Master Mix into 4 labeled PCR tubes 3. Thermocycler for :       Initial Duration   | 98° C | 30s        35 Cycles of:            Denaturation | 98° C | 10s           Annealing    | 55° C | 20s           Extension    | 72° C | 30s     Final Extension    | 72° C | 5m     Hold               | 4°  C | Forever  POSSIBLE ERRORS: Incomplete thawing of dNTPs
Working Stock of Reverse and Forward Primers for pTEF1 kept in my freezer box. 6/9/14
6/10/14
Gel Extraction
Cut DNA band from PCR of constitutive promoter gel Weigh gel: .720 g QG Buffer: 2160 ul  1. Mix gel slice with QG Buffer in 50C heat bath for 10 min. 2. Add 1 ml isopropanol. Mix. 3. Add 750 ul of mixture into purple Qiagen spin column. Spin for 30 sec. Discard liquid.  Repeat. 4. Add 750 ul of Buffer PE and spin for 1 min. Discard waste. 5. Dry spin the column for 1 min. Replace the spin column in a new microcentrifuge tube 6. Elute in 40 ul of ddH2O. Wait 1 min and then spin for 1 min.
Restriction Digest with ApaI
40 ul of DNA (pTEF1 gel extraction) 5 ul of 10x CutSmart Buffer .5 ul of ApaI Enzyme  *digest overnight at room temperature*
6/11/14
Restriction Digest with XhoI
Add .5 ul of XhoI enzyme to ApaI digestion from 6/10/14 Incubate in the 37°C shaker.
PCR Purification
1. Add 250 ul of Buffer PB to digestion. 2. Place sample in a purple QIAquick spin column. Spin for 1 min. Discard waste. 3. Wash with 750 ul of Buffer PE. Spin for 1 min. Discard waste. 4. Dry spin for 1 min. Replace spin column in a new microcentrifuge tube. 5. 6. Elute in 50 ul of ddH2O. Wait 1 min and then spin for 1 min.  Total concentration: 141.4 ng/ul
DNA Ligation
Materials
Volume
10x Ligase Buffer
1 ul
DNA Backbone PSV606
.2 ul
DNA Insert (PCR Purication)
.2 ul
T4 DNA Ligase
.5 ul
H2O
8.1 ul
Total                       | 10 ul ---------------------------------------------------- Mix reagents and incubate at room temperature for 2 hrs
Ligation of pTEF1 into PSV606 kept in my freezer box. 6/11/14
Transformation
10 ul of ligation 50 ul of E. Coli competent cells  30 min    | ice 45 sec    | 42°C heat shock 2 min     | ice  Add 250 ul of SOC media. Incubate at 37°C for 1 hr. Plate on LB+Carb.   NO COLONIES
6/12/14
Redo Transformation
Follow procedure from 6/11/14 with minor alterations. 1. Use .4 ul of ligation instead of .2 ul. 2. Use more expensive competent cells.
6/13/14
Transforming α-inducible promoters
Transforming 11 different inducible promoters into CB008 and CB008DB strains of yeast.  1. Boil salmon sperm DNA (ssDNA) for 10 min.-->10 ul of 10 mg/ml stock per transformation 2. Cool on ice for 10 min.  3. Pellet yeast cultures in centrifuge. (3000 rpm for 2-5 min.) 4. Resuspend with with .1M LiOAc in TE. 5. Pellet cells (3000 rpm for 2-5 min.) 6. Resuspend in 100 ul .1M LiOAc in TE per 2.5 ml culture.  7. Aliquot 100 ul into each microcentrifuge per transformation. (22 tubes) Per Tube: 8. Add 100 ug ssDNA, 1 ug of target DNA (1~5 ul)  ADD IN ORDER 9. 480 ul 50% PEG 3350 10. 60 ul 10x TE 11. 60 ul 1 M LiOAc 12. 75 ul DMSO  13. Vortex 14. Incubate at 42°C for 30 min.  15. Pellet cells (6000 rpm for 2 min) 16. Resuspend with 500 ul YPD. 17. Pellet cells. 18. Resuspend with 50 ul  YPD 19. Plate on SDS-Ura.   Incubate 2 days at 30°C.
6/16/14
Colony PCR for Screening Yeast
1. Pick a single colony using a sterile wooden stick and patch on a dropout plate.     Take stick and rub into a dry PCR tube. 2. Add 10 ul of 20mM NaOH to each of the PCR tubes.    Boil in pCR machine at 95°C for 20 min.  3. Set up PCR reaction as below:                                         1x Reaction      7x Master Mix                                         ---------------------------------     2X Go Taq Green PCR Master Mix              10ul               70ul     10 uM FW primer                              1ul                7ul     10uM RV primer                               1ul                7ul     Water                                        5ul               35ul      ----------------------------------------------------------------------     Boiled Yeast cells (template)                3ul            4. Set up Thermocycler for:     95°C       | 5 min     30x |95°C   | 45 sec         |50°C   | 30 sec         |72°C   | 1 min per kb     72°C       | 10 min     4°C            | hold 5. Run on 1% Agarose Gel  We ran 66 lanes for colony PCR RESULTS: only 4 lanes were positive POSSIBLE ERRORS: not enough DNA in tubes
Colony PCR on E.Coli for Constitutive Promoters
1. Pick a single colony using a sterile wooden stick and mix in 25ul of water in a PCR tube. Do this for about 4-6 colonies. Use 5ul for the PCR reaction below, adn save the rest for later.  2. Set up PCR reaction as below:                                             1x Reaction      6x master mix                                         ---------------------------------- 2X Go Taq Green PCR Master Mix                  10ul               60ul 10 uM FW primer                                  1ul                6ul 10uM RV primer                                   1ul                6ul Water                                            3ul               18ul  Bacterial cells (template)                       5ul                --         Cycles:             95°C               | 5 min             30X:    95°C       | 45 s                     55°C       | 30 s                     72°C       | 1 min per kb              72°C               | 10 min  3. Analyze products on a 1% agarose gel.   RESULTS: All lanes worked  4. Inoculate the colonies in 5 ml LB+Carb and incubate 37°C shaker overnight.
6/17/14
Re-inoculate the E.Coli with Constitutive Promoters
6/18/14
Dilution of Yeast Strains
We diluted CB008 and CB008DB strains 1:20 times in YPD media.
Miniprep Constitutive Promoters from E.Coli
1. Pellet bacterial cells 2. Resuspend in 250 ul in Buffer P1. Transfer to a microcentrifuge tube. 3. Add 250 ul Buffer P1. Invert to mix. 4. Add 350 ul Buffer P2 and invert immediately but gently. 5. Centrifuge for 10 min at 130000 rpm. Apply supernatants to QIAprep spin columns. 6. Centrifuge for 30-60 sec. Discard flow through. 7. Wash with 750 ul Buffer PE and centrifuge for 60 sec. 8. Discard flow through and dry spin for 1 min. 9. Place spin column in microcentrifuge tube and elute in 25 ul ddH20. Spin for 1 min.
6/20/14-6/21/14
Out of the lab. Other lab members ran the flow cytometer on AFRPs and transformed yeast with constitutive promoters.
6/23/14
Yeast Colony PCR on Transformed Yeast
used wrong primers so we need to redo on Tuesday
6/24/14
Re-do Yeast Colony PCR on Transformed Yeast
Innoculated Yeast colonies in 5ml of YPD
Flow Cytometer Notes
Overnight cultures of CB008+AFRP+GFP diluted ~100x (to a final concentration of OD600 0.5-0.1, Saturated overnight cultures should be OD600 of ~7) in SD complete media, and grown for 3 hours, 1000rpm shaker, 30ºC. Growing in 2mL well plates
Induce with Alpha-factor. Stock is in 3mM. Final concentrations are 0, 1nM, 10nM, 100nM, 1000nM.  Alpha-factor cannot be refrozen, so throw leftover away.
Induce for 90mins, but no longer than 120mins
Transfer 250u of each well into a V-bottom 96-well plate containing 10ul of the fixing chemical Cyclohexamide. (thats 4 ul for every 100ul of culture)  Cyclohexamide stops protein production by inhibiting ribosomes.
Run on the flow cytometer.
Flow Cytometer things to remember:
check the sheath fluid box to see if its empty
check the waster container.  If full, dispose and add new bleach
take care not to leave the machine on run. Save sheath fluid by keeping on standby when not in use
find parameters using negative controls. (FSC, SSC, Fluorescense marker)
DO NOT RUN PLATE. RUN WELLS instead. (Run the first well by itself first b/c it needs time to create file folders.) this causes problems.
Run plate only saves data of the first well.
Can rerun through the first well again which would suck up air.  Air no good for the machine.
highlight wells and change settings on the right in the aquisition dashboard screen
mixing speed 180ul/sec
mix 3 times rather than 2
run at 1ul/sec for samples and 0.5ul/sec for negative controls when finding parameters.
make sre uL doesn't exceed samle to prevent sucking up air. (Rule of thumb is to run 50ul less than allotted sample sizes for wells.)
Voltage settings were gotten by running negative controls and adjusting to readouts seen in histogram form of FSC, SSC, FITC
Parameters of This Flow Experiment
FSC: 250 SSC: 280 FITC(GFP): 550 B(RFP): 650  Flow rate: 1µL/sec Sample Volume: 200µL Mixing Volume: 100µL Mixing speed: 180µL/sec
Flowjo Notes
Export data from USB and transfer to iGEM2014 folder
Open FlowJo and drag data over to box
Set an appropriate gate
Go to data, select all data and press E. Select mean, geometric mean, and count.
Hit refresh. Addiction numerical data should be present.
Use mean of FITC(GFP).
MatLab Notes
Open up new script and comment the title of the experiment
Enter the alpha factor concentrations info. (X axis). Put in log form.
Enter the yGEM data. This will serve as y axis. (FITC means)
Define variables
Tyope figure info and run to show plot.
6/25/14
Transformations of DH5a
HY86E3
HY67E1
HYGE1
PTS98
PTS108
PTS133
PTS97
plasmids for backbones in different integration sites and fluorescent tags
Glycerol stocks of Yeast Transformations (yGEM23-32)
420 ul 50% glycerol 350 ul cells in YPD vortex, store in -80C freezer
PCR for Parts Registry Promoters
ASG7, CLG1, ECM18, HYM1, PCL2, PRM1, PRM2, PRM3, PRM6, SAG1, YDR124W
Materials
1x reaction
x Phusion HF Buffer
10 µl
dNTP's (10 mM)
1 µl
Forward Primer (10µm)
2.5 µl
Reverse Primer (10µm)
2.5 µl
*Template DNA
0.3 µl
Phusion Polymerase
0.5 µl
Water
33.2 µl
Total
50 µl
-----
------
3. Thermocycler for :       Initial Duration   | 98° C | 30s        35 Cycles of:            Denaturation | 98° C | 10s           Annealing    | 55° C | 20s           Extension    | 72° C | 30s     Final Extension    | 72° C | 5m     Hold               | 4°  C | Forever  Ran gel on PCR products. 1% Agarose, 125 volts for 15 min, 5 ul SybrSafe
6/26/14
Only pGEM2 and pGEM3 worked. RETRY with DMSO (1.5 ul per reaction).
PCR
Master Mix: 170 ul HF Buffer 17 ul dNTPs 8.5 ul Phusion Polymerase 25.5 DMSO 539 ul Water ------------- 44.7 per tube   Also add 2.5 ul FW primers          2.5 ul RV primers          .3 ul template DNA
Made liquid cultures of transformed DH5a E. Coli cells with HY+PTS plasmid
6/27/14
Double Digestion of Constitutive Promoters-GFP
6.5 ul water 15 ul DNA template (pTEF1 plasmids) .5 ul Xho1 enzyme .5 ul Not1 enzyme    2.5 ul CutSmart Buffer  Incubate at 37C for 2hrs Load into agarose gel. *pTEF1 appers to be smaller than the others.
Gel Extraction
Exise the DNA bands from the gel. Weigh using a scale. pGEM12- 120 mg      -360 ul         -120 ul pGEM13- 110 mg      -360 ul         -110 ul pGEM14- 110 mg      -330 ul         -110 ul pGEM15- 100 mg      -300 ul         -100 ul pGEM16- 170 mg      -510 ul         -170 ul         Weights     QG Buffer       Isopropanol 1. Melt gels in 50C for 10 min in QG Buffer 2. Add isopropanol. 3. Place in a spin column and spin for 1 min 13000rpm 4. Add 500 ul QG Buffer 5. Spin for 1 min. Dump liquid. 6. Add 750 ul PE Buffer 7. Spin for 1 min. Dump Liquid. 8. Dry spin for 1 min. 9. Elute in 25 ul ddH20.   Concentrations (ng/ul) through NanoDrop:  1. pGEM12-25.77  2. pGEM13-47.93  3. pGEM14-18.31 4. pGEM15-10.6 5. pGEM16-32.45
Gibson Assembly
50 ng of DNA Backbone 2 ul of rtTA insert 5 ul Gibson Assembly Master Mix  pGEM12-2 ul         1 ul pGEM13-1 ul         2 ul pGEM14-3 ul         - pGEM15-4 ul         - pGEM16-1.5 ul       1.5 ul       DNA backbone    water All have 2 ul rtTA insert and 5 ul GAMM Incubate in thermocycler 50C for 15 min.
Transforming E. Coli with Gibson Assembly
10 ul Gibson mix 25 ul Mach1 cells ----------- 250 ul SOC media  30 min ice 45 sec heatshock at 42C 2 min ice Incubate at 37C for 1 hr. Plated 250 ul on LB-Cam (WRONG SELECTION MARKER- LB-Carb) Incubate in a drawer overnight
6/29/14
pTEF1+rtTA transformations failed!  Dilute CB008 constututive promoters (pTEF1)1:20 times Incubate overnight 30C shaker/
6/30/14
Repeat pGEM12,pGEM15, rtTA insert digestions with Minipreps from freezer box Same procedure as from 6/27/14  pGEM12-21.05 ng/ul pGEM15-23.03 ng/ul rtTA-182.3 ng/ul  REDO Gibson Assembly (procedure from 6/27/14). Plate all Gibson mix and incubate 37C overnight on LB-CAM.
7/1/14
Seamless Cloning
pGEM12- 2 ul            3 ul pGEM13- 1 ul            2 ul pGEM14- 3 ul            4 ul pGEM15- 2 ul            3 ul pGEM16- 1.5 ul          2.5 ul Positive- 1 ul PUC      3 ul        2 ul control insert Negative- 2 ul          2 ul         DNA backbone    Enzyme Mix  All with 1 ul of rtTA insert except for positive and negative control.
Transform into C2987 competent cells
25 ul competent cells 3 ul Seamless Assembly Mix  30 min ice 45 sec heatshock 42C 2 min ice Add 250 ul SOC media 1 hr incubation 37C  Plate all on LB+Cam
Miniprep HY/PTS cultures
Follow same miniprep procedure as previous experiments. Concentrations (in ng/ul) 1. PTS98    66.8 2. PTS97    88.2 3. HY67E1   63.43 4. HY6E1    22.05 5. PTS108   120.8 6. HY86E3   118.6 7. PTS133   701.3
Transforming pTEF1+GFP into DH5a
.5 ul plasmid DNA (pGEM12 to pGEM16)
7/2/14
Plated Seamless Cloning Transformation on LB+Carb, not LB+Cam.
7/3/14
Colonies found on LB+Carb plates.
Negative control has 5 colonies(?)
Positive control has too numerous to count.
All other colonies had ~100 colonies.
GFP
Innoculated GFP+pGEM12 to pGEM16 colonies. Stored in -80C freezer. Move plates from 4C to 37C incubator on Sunday.
rtTA
We ran a colony PCR on pGEM12-16 + rtTA. Follow the same procedure as before, using forward/reverse primers for rtTA. Used a 50C annealing temperature instead of 55C due to melting point of rtTA. Colonies all worked! Innoculated all colonies and placed in 4C freezer.
7/6/14
Moved all tubes from 4C freezer to 37C incubator shaker.
7/7/14
Miniprepped all tubes
Concentrations (in ng/ul)of rtTA plasmids 1. pGEM12-1     540.2 2. pGEM12-2     560.4 3. pGEM12-3     375.6 4. pGEM13-1     360.3 5. pGEM13-2     488.9 6. pGEM13-3     ------ 7. pGEM14-1     302.3 8. pGEM14-2     425.5 9. pGEM14-3     513.5 10. pGEM15-1    595.8 11. pGEM15-2    554.4 12. pGEM15-2    510.6 13. pGEM16-1    592.3 14. pGEM16-2    545.8 15. pGEM16-3    302.3  Concentrations (in ng/ul) of GFP plasmids 1. pGEM12- 425.5 2. pGEM13- 620.3 3. pGEM14- 521.9 4. pGEM15- 567.7     5. pGEM16- 534.3
*discovered that rtTA inserts are wrong because activation domains may not have been copied with PCR
7/8/14
Transformed 11E-3128 into DH5a (pTET-mfa)
50 ul DH5a .5 ul plasmid DNA Heatshock transformation Plate 100 ul on LB+Carb
pGEM13+rtTA sequencing did not work (weird insertion in rtTA before terminator sequence)
Innoculate pGEM16+GFP again.
7/9/14
Miniprepped pGEM16+GFP
Re-PCR of rtTA insert
Materials
1x reaction
5x Phusion HF Buffer
10 µl
dNTP's (10 mM)
1 µl
Forward Primer (#70)
2.5 µl
Reverse Primer (#39)
2.5 µl
*Template DNA
1 µl
Phusion Polymerase
0.5 µl
Water
31 µl
DMSO
1.5 ul
Total
50 µl
Thermocycler Cycles:     Initial Duration   | 95° C | 5 min        30 Cycles of:            Denaturation | 95° C | 30s           Annealing    | 50° C | 30s           Extension    | 72° C | 2 min     Final Extension    | 72° C | 5m     Hold               | 4°  C | Forever  *used 10 ul of dNTPs instead of 1 ul
Pme1 Digest of pGEM12-16 (without pGEM13) +rtTA
Digest 2000ng of template DNA Pme1 enzyme-        .5 ul Cutsmart Buffer-    2.5 ul DNA plasmid-        4 ul  Incubate 1 hr 37C.
Ran a gel for rtTA PCR
PCR Purification of rtTA
Eluted in 50 ul
Gibson Assembly
pGEM12-         1 ul rtTA insert-    3 ul GA MM-          4 ul  Incubate 15 min at 50C.
Transform
25 ul Mach1 cells .5 ul pGEM13+rtTA plasmid  Plate all on LB+Carb
7/10/14
Colony PCR of pGEM13+rtTA
-same procedure as 7/9/14 -successful
Miniprep 11E 3128 (pTET+mfalpha)
-followed standard miniprep procedure. Concentration: 430 ng/ul SEQUENCING CORRECT! (7/14/14)
7/11/14
Miniprep of pGEM13+rtTA
Sequencing correct
pGEM16 (m10+GFP mislabeled m3+GFP) and pTET+mfalpha
7/15/14
PCR rtTA out of correct pGEM20
*threw away all incorrect rtTA inserts 5x HF Buffer                        10 ul dNTPS                               1 ul Phusion Polymerase                  .5 ul pGEM20- rtTA template               1 ul H2O                                 21 ul FW Primer (different for pTEFs)     2.5 ul RV Primer (rtTA reverse primer)     2.5 ul DMSO 1.5 ul  Thermocycler:     95  5 min     30x cycle:         95  45 sec         50  30 sec         72  2 min     72  5 min     4   hold
7/16/14
Load pGEM1-13+rtTA on gel
Gels were leaky and had spillage on contents
Gel extractions
1. PRM1     .54g 2. PRM2     .56g 3. PRM3     1.15g 4. PRM6     1.21g 5. ECM18    .61g 6. SAG1     .68g 7. YDR124W  .45g 8. CLG1     .40g 9. ASG7     .31g 10. HYM1    .50g 11. PCL2    .42g 12. M3      .69g Followed same gel extraction procedure
Gibson Assembly for m3
3 ul insert 1 ul backbone 5 ul GA Master Mix 1 ul H2O
PCR of rtTAs (again)
same procedure as before, except: use 20 ul gel extraction as template
Transform m3+rtTA cells from Seamless Cloning
25 ul cells 1 ul Gibson DNA
7/17/14
Colony PCR on m3+rtTA
all had the rtTA insert! made liquid cultures and incubated overnight
7/18/14
Miniprep pGEM13+rtTA
eluted in 50 ul 1. 500.2 ng/ul 2. 386.9 ng/ul 3. 433.4 ng/ul 4. 400.5 ng/ul
7/20/14
Sequencing came back conclusive! Silent mutation at 417 bp.
7/21/14
Innoculated pTET+GFP yeast strains in 5ml YPD for transformation
7/22/14
Prepare cultures for transformation
Diluted pTET+GFP yeast strains 1:20. Incubated for 3 hrs.
Pme1 Digest
Pme1 digest to linearize the pTEF1 m3+rtTA circular plasmid for yeast transformation.  Digest 2000 ng DNA. 4 ul DNA .5 ul Pme1 Enzyme 2.5 ul Cutsmart Buffer 18 ul H2O  Incubate at 37C for 1 hr.
Transform Yeast
Transformed pTEF1-mr+rtTA into CB008/CB008DB pTET+GFP. Follow standard procedure for transforming yeast found in our protocols page.  -maybe not enough salmon sperm Plated on SD URA in the 30C incubator for 2 days.
7/23/14
Transforming E.coli with Fluorescent Proteins
25 ul DH5a cells .5 ul DNA
7/24/14
Yeast Colony PCR
Followed colony PCR procedure from before Used primers for PSV606 Used 1.5 ul DMSO per reaction 3 min annealing step  FAILED
7/25/14
Diluted CB008/DB pTET+GFP strains for transformations
Transformations of m3+rtTA
used 4 ul DNA Plate on SD URA
7/28/14
Yeast Colony PCRS
1 ul of FW primer 1 ul of RV primer 10 ul Green MM 10 ul Yeast cells 1.5 ul DMSO  Ura: 606/606 His: RA145/RA146 Trp: 604v2/604v2  -FAILED-
Yeast Transformations
Transformed pAGA+mCherry into yeast strains. (All with pTET+GFP, and pTET+mfalpha) CB008DB m6+rtTA         m7+rtTA CB008   m10+rtTA         m6+rtTA         m7+rtTA         pTEF1+rtTA  Transformed pTET mfalfpha into CB008DB pTET+GFP         pTEF1+rtTA  Transformed m3+rtTA into pTET+GFP CB008/DB
Miniprep
PTS94- 150 ng/ul PSW696- 60 ng/ul PS2607- 125 ng/ul
Colony PCR
Colony PCR on URA, HIS, and TRP sites. Repeat Colony PCR done earlier today.
7/29/14
Colony PCR
Repeat Colony PCR from yesterday. FAILED. Possibly errors in the primers for TRP. should show 300 bp. Use a 100 bo ladder?
More Colony PCR
Repeat URA and TRP colony PCRs Made own Green MM from (per reaction)     .125 ul Gotaq enzyme     5 ul 5x Buffer     .5 ul DNTPs Still FAILED.
7/29/14
More failed colony PCRs
7/31/14
PCR Purification
BFP-118.4 ng/ul     digest with Xho1/Not1 Aga-144.2 ng/ul     digest with Apa1/Xho1
Double Digestion of BFP
Digest 2000ng 17 ul   BFP PCR Purification .5 ul   Xho1 Enzyme .5 ul   Not1 Enzyme 2.5 ul  Cutsmart Buffer 4.5 ul  Water  Incubate 37C for 3 hrs
Digestion of RFP
Digest 2000ng 15 ul   pAGA1 PCR Purification .5 ul   Apa1 Enzyme 2.5 ul  Cutsmart Buffer 7 ul    Water  Incubate Room Temperature for 2 hrs  Add .5 ul Xho1 Enzyme     Incubate 37C for 1 hr
Digest mCherry+pAGA1 with Pme1 for Transformation
6 ul    pGEM22 .5 ul   Pme1 Enzyme 2.5 ul  Cutsmart Buffer 16 ul   Water  Incubate 37C for 1 hr
Transformations (Yeast)
Transformed pAGA1+mCherry into (CB008):    pTEF1             m6               m7 (CB008DB)   m6             m7  All contain the other motifs in the circuit.
PCR Purify Digestions
Followed procedure for PCR Purifications in protocols. Final Concentrations: BFP: 17.71 ng/ul RFP: 21.75 ng/ul
Ligations
1. HY130E Backbone with pAGA1+BFP (TRP) 2. HY130E Backbone with pPCL2+BFP (TRP)  1)  3 ul    Backbone     1 ul    BFP     1 ul    pAGA1     1 ul    T4 Ligase     1 ul    10X Buffer     3 ul    Water  2)  3 ul    Backbone     1 ul    BFP     .5 ul   pPCL2     1 ul    T4 Ligase     1 ul    10X Buffer     3.5 ul  Water  Incubate room temperature Kept in Ianto's freezer box
8/1/14
Pme1 Digest of pTET+mfalpha
Same protocol as before
Transformations (Yeast)
CB008 m3 transform- m3+rtTA CB008DB m10 transform- pTET+mfalpha
8/4/14
Remade 20mM NaOH
Colony PCR
CB008 pAGA+mCherry:     m6, m7 CB008DB pAGA+mCherry:     m6, m7
Made Liquid Cultures
CB008:     pTET+mfalpha         pTEF1, m3, m6, m7     m10+rtTA CB008DB      pTET+mfalpha         pTEF1, m6, m7     pTET+GFP (for m3+rtTA transformations)     pAGA1+mCherry m10
8/5/14
Colony PCR
CB008/DB pAGA1+mCherry     m6, m7 Most of the colonies worked
Gel
Ran a gel of: Pme1 Digest of pAGA1+mCherry                pTET+mfalpha
8/7/14
Liquid Cultures
5 ml YPD cultures of the 8 mfalpha strains from 8/4/14 to transform pPCL2+mCherry
Exploratorium Presentation
8/8/14
Dilutions
Made 1:20 dilutions of pTET+mFalpha strains: CB008: pTEF1, m6, m7, m10 CB008DB: pTEF1, m3, m6, m10 *for pPCL2+RFP transformations
Colony PCR
CB008:     m3+rtTA      pTEF1, pAGA1+mCherry CB008DB:     m3, pAGA1+mCherry     pTEF1, pAGA1+mCherry     m6, pAGA1+mCherry     m10, pTET+mfalpha  Boiled 100C for 20 min Primers-    URA: 606             HIS: RA 145/146             TRP: RA 145/148
Pme1 Digests
(Same for all digests) .5 ul   Pme1 Enzyme 2.5 ul  Cutsmart Buffer  RFP:    9 ul of pPCL2+mCherry          13 ul of water BFP:    4 ul of pPCL2+BFP         18 ul of water  Did twice for more transformations
Transformations
Tranformed pPCL2+BFP and pPCL2+mCherry into mfalpha strains
8/11/14
Colony PCR
BFP and RFP transformations from 8/8/14 Used RA145/148 for TRP 1.5 ul DMSO per reaction
Transformations (Yeast)
pTET+mfalpha into CB008 m3 pAGA1+mCherry into CB008DB m10 pPCL2+RFP into CB008DB m10 pPCL2+BFP into CB008DB m10
Streaked Plates
CB008     pTEF1, pAGA1+mCherry     m3+rtTA CB008DB     pAGA1+mCherry         pTEF1, m3, m6     m10, pTET+mfalpha
8/12/14
Glycerol stocks
CB008     pTEF1, pAGA1+mCherry     m3+rtTA CB008DB     pAGA1+mCherry         pTEF1, m3, m6     m10, pTET+mfalpha  (yGEM69-74)
Pme1 Digestions
pPCL2+BFP pPCL2+mCherry pAGA1+mCherry pTET+mfalpha
Colony PCR
pPCL2+BFP pPCL2+mCherry
Transformations (DH5a cells for Miniprepping)
pTET+mfalpha pPCL2+BFP pPCL2+mCherry pAGA1+mCherry
Transformations (Yeast)
CB008 m3, pTET+mfalpha CB008DB m10     pPCL2+mCherry, pAGA1+mCherry, pPCL2+BFP
Pme1 Digest
pTET+mfalpha
Streaked Plates
Streaked mCherry and BFP successful colonies from Colony PCR
Liquid Cultures of DH5a transformations
8/13/14
Dilutions
Made 1:20 dilutions of BFP/RFP cultures CB008 BFP     pTEF1, m6, m7, m10 CB008 RFP     m6 CB008DB RFP     m7, m10 CB008DB BFP     m10
Colony PCR
Colony PCR of BFP/RFP transformations -FAILED-
Miniprepped DH5a transformations for plasmids
pTET+mfalpha RFP BFP
Streak/Glycerol Stocks
CB008      pTEF1- RFP/BFP     m6- RFP/BFP     m7- RFP/BFP     m10- BFP CB008DB     pTEF1- RFP/BFP     m3- RFP/BFP     m6- RFP/BFP     m7- RFP/BFP
Transformations
CB008     m6, RFP     m10, BFP
8/14/14
Colony PCR
CB008 m3, pTET+mfalpha CB008DB m10, pPCL2+RFP              pAGA1+RFP              pPCL2+BFP *used .5 ul Zymolyase per reaction Thermocycle:     37C         5 min     95C         5 min     30x 95C     45 sec 50C     30 sec         72C     1 min     72C         10 min     4C          hold
8/17/14
Colony PCR
RFP/BFP strains 56 colonies used RA145/148 and zymolyase
8/18/14
Colony PCR Check
pAGA1s successful pPCL2 failed
Colony PCR
pPCL2+mCherry pPCL2+BFP pSAG1+Ste2 (use 87/91 primers. around 600bp)
Transformations (Yeast)
AFRPS: 7 pPCL2+mCherry  1 pAGA1+mCherry 4 rtTAs  m3 CB008 pTET+mfalpha
8/19/14
Streaked Plates
pPCL2+mCherry AFRPS m10, pPCL2+BFP
Liquid Cultures
23 things for glycerol stocks and transformations
8/20/14
Pme1 Digests
Digest 4000 ng pPCL2+BFP pAGA1+RFP pTET+mfalpha  Incubate 2 hrs 37C
Colony PCR
*extended extension time to 2 min* ~940bp Ste2 in NAT integration site 87/91 primers
Transformations (Yeast)
pPCL2+BFP: PRM1, PRM2, PRM3, YDR124W, CLG1, ASG7, HYM1
Glycerol Stocks of yGEM101-126
8/21/14
Restreak ALL Plates
Yeast plates die after 2 weeks Restreak on new YPD
Colony PCR
CB008/DB m6 for:     pTET+mfalpha, pAGA1+RFP *flow data suggests false positives of RFP
Transformations (Yeast)
pSAG1+rtTA pPCL2+rtTA pPRM6+rtTA, suspended pECM18+rtTA, suspended m3, pTET+mfalpha
Made new LiOAC, 10x TE Buffer, PEG 3350
Retrieved from " http://2014.igem.org/Team:UCSF_UCB/jeffrey "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
