Team:Newcastle/Model
MENU
HOME
TEAM
Team
Collaborations
PROJECT
Description
Design
Experiments
Notebook
InterLab
Contribution
Model
Results
Demonstrate
Improve
Attributions
PARTS
Parts
Basic Parts
Composite Parts
Part Collection
SAFETY
HUMAN PRACTICES
Silver HP
Integrated and Gold
Public Engagement
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Plant
Software
JUDGING FORM
Show Navbar
spacefill
Project
Description Results Demonstration Protocols Parts Interlab Study
Modelling
Overview Simbiotics Design of Experiments Microfluidic Agent-based Model
Human Practices
Silver Gold and Integrated Education and Public Engagement Science Communication
Collaborations
Overview Edinburgh OG Exeter Evry Paris-Saclay
Judging
Medal Requirements Safety Measurement Applied Design Part Collection Improvement
Team
Meet the Team Attributions
Collapse Navbar
spacefill
Our Models
For our project, we built three types of models. The first was an agent-based model which simulated our multicellular biosensor framework. This model gave insight into the optimal ratio of cell-types to have in the system. This information was used during experimental characterisation to optimise our system.
Our second model was a statistical, multifactorial Design of Experiments (DoE) approach towards optimising Cell-Free Protein Synthesis (CFPS) systems. This statistical model was used to generate an experimental design to gather data on the importance of certain supplements in CFPS systems, and then use the experimental data to optimise CFPS systems.
Our third model was an agent-based model designed to replicate the functions of a digital microfluidic chip and schedule the tasks for the device. The final piece of software controls agents which are the microfluidic droplets and moves them around the simulated chip according to predefined movement plans which can be read from either the program itself or custom external files. This provides a quicker, more inexpensive means of testing the chip than repeated real-world experiments.
Please click on the links below to find out more about our models.
Multicellular Modelling: Simbiotics
Cell Free Protein Synthesis Systems Optimisation Microfluidic Agent-Based Model
Multicellular Modelling: Simbiotics
Rationale and Aim
After the initial design of the Sensynova platform, it was important to determine, in silico, if multicellular biosensor systems constructed according to our paradigm would be able to produce responses to target molecules which were comparable to traditional whole cell sensors. Therefore, a 3D, spatially explicit, stochastic model was constructed, in which each cell was modelled as a separate agent containing kinetic equations specific to the biosensor components present in that cell type. To enable the application of experimentally derived rate constants, an IPTG sensor was designed according to our platform and modelled. This design was later used as our proof-of- concept in vitro system. Additionally, in traditionally engineered biosensor systems, biosensor components are often present in equal amounts, mostly one detection device to one processing device to one reporter device. However, other than ease of production, there is no evidence that a component ratio of 1:1:1 is optimum for all systems. An unexpected side effect of splitting biosensor components into different cells was the production of a new design space in which biosensor behaviour could be altered by varying the ratios of cell types, and therefore biosensor components, in a multicellular system. We wanted to harness this new method of fine-tuning biosensor circuits through the in silico exploration of cell type ratios and subsequent in vitro confirmation of optimum component ratios.
Background Information
Simbiotics is a novel 3D multicellular simulation tool developed by Jonny Naylor at Newcastle University (Naylor et al. 2017). This tool uses stochastic agent-based modelling to simulation interactions between different cell types within a defined 3-dimensional space. Each cell type defined in the model can run its own deterministic SBML model, which gives each cell its own behaviour. The simbiotics tool can give certain molecules defined within the SBML models (e.g. quorum sensing molecules) the ability to pass through the cell membrane, diffuse across the defined space, and enter another cell type. This enables the simulation of communication between members of a multicellular community.
Visual Simulation of Simbiotics Model: Red dot shows a detector cell, blue dot shows a processor cell. Black dots show reporter cells, green dots show reporter cells expressing sfGFP. Orange circles show the diffusion of C12-AHL, and pink circles show diffusion of C4-AHL.
Simbiotics was an obvious choice for modelling the Sensynova development framework , as it is a multicellular community with three defined cell types which communicate via quorum sensing molecules.
Deterministic SBML Models
A model for each cell type in the Sensynova framework (the detector, the processor, and the reporter) was initially made using SBML (Hucka et al., 2003) and COPASI (Hoops et al., 2006). The code for the SBML models can be downloaded here .
Detector Cell SBML Model
The deterministic SBML model for the detector cell was tested separately from the other two cell types in COPASI. Figure 1 shows the model schematic. When IPTG is absent, LacI is produced constitutively and inhibits the pLac promoter. This prevents LasI from being made, which means that the C12 quorum sensing molecule can not be synthesised. When IPTG is present, it can bind to the LacI and stop the pLac promoter from being repressed. This allows the production of LasI, and hence the synthesis of C12.
Figure 1: Schematic for the detector cell model. Crossed circle represents a degradation reaction. A blunt-ended line represents repression.
Figures 2 and 3 below show simulations by COPASI of the detector cell with and without IPTG present. As was expected, when IPTG is not present, C12 has a low value, and when IPTG is present C12 has a high value.
Figure 2: Detector cell SBML model simulated by COPASI. Graph shows production of C12 quorum sensing molecule over time in the presence of IPTG.
Figure 3: Detector cell SBML model simulated by COPASI. Graph shows production of C12 quorum sensing molecule over time in the absence of IPTG.
Processor Cell SBML Model
The deterministic SBML model for the processor cell was tested separately from the other two cell types in COPASI. Figure 4 shows the model schematic. LasR is produced constitutively. When the C12 quorum sensing molecule is absent, LasR is unable to activate expression from the pLas promoter, resulting in no production of RhlI and no synthesis of the C4 quorum sensing molecule. When C12 is present, it binds to LasR, allowing expression from pLas, production of RhlI, and synthesis of C4.
Figure 4: Schematic for the processor cell model. Crossed circle represents a degradation reaction.
Figures 5 and 6 below show simulations by COPASI of the processor cell with and without the C12 quorum sensing molecule present. As was expected, when C12 is not present, C4 has a low value, and when C12 is present C4 has a high value.
Figure 5: Processor cell SBML model simulated by COPASI. Graph shows production of C4 quorum sensing molecule over time in the absence of C12 quorum sensing molecule.
Figure 6: Processor cell SBML model simulated by COPASI. Graph shows production of C4 quorum sensing molecule over time in the presence of C12 quorum sensing molecule.
Reporter Cell SBML Model
The deterministic SBML model for the reporter cell was tested separately from the other two cell types in COPASI. Figure 7 shows the model schematic. RhlR is produced constitutively. When the C4 quorum sensing molecule is absent, RhlR is unable to activate expression from the pRhl promoter, resulting in no production of sfGFP. When C4 is present, it binds to RhlR, allowing expression from pRhl and production of sfGFP.
Figure 7 Schematic for the reporter cell model. Crossed circle represents a degradation reaction.
Figures 8 and 9 below show simulations by COPASI of the reporter cell with and without the C4 quorum sensing molecule present. As was expected, when C4 is not present, sfGFP has a low value, and when C4 is present sfGFP has a high value.
Figure 8: Reporter cell SBML model simulated by COPASI. Graph shows production of sfGFP over time in the presence of C4 quorum sensing molecule.
Figure 9: Reporter cell SBML model simulated by COPASI. Graph shows production of sfGFP over time in the absence of C4 quorum sensing molecule.
Multicellular Simbiotics Simulation
The three deterministic SBML models described and tested above were used in the Simbiotics tool to simulate the entire multicellular biosensor framework. The cells were tested as three ratios to determine which yielded the highest sfGFP production in the presence of IPTG, compared to in the absence of IPTG (i.e. the optimal signal-to-noise ratio).
Ratio 1 - 200 detectors : 200 processors : 200 reporters
The graph below (figure 10) was created using data generated by the Simbiotics software tool using default values for diffusion of the quorum sensing molecules (C12 and C4), and the inducer IPTG, and default values for membrane permeation by the three molecules. The system was simulated in the presence (blue line) and absence (orange line) of IPTG.
When the three cell types were in equal amounts, there was no difference in sfGFP production when IPTG was present vs. when it was absent.
Figure 10: Simbiotics simulation of Sensynova framework with 200 detector cells, 200 processor cells, and 200 reporter cells. Blue line shows production of sfGFP by reporter cells over time in the presence of IPTG, and the orange line shows the same in the absence of IPTG.
Ratio 2 - 1 detector : 200 processors : 200 reporters
The graph below (figure 11) was created using data generated by the Simbiotics software tool using default values for diffusion of the quorum sensing molecules (C12 and C4), and the inducer IPTG, and default values for membrane permeation by the three molecules. The system was simulated in the presence (blue line) and absence (orange line) of IPTG.
When the three cell types were in the ratio of 1:200:200 for detectors:processors:reporters, there was a small difference in sfGFP production when IPTG was present vs. when it was absent.
Figure 11: Simbiotics simulation of Sensynova framework with 1 detector cell, 200 processor cells, and 200 reporter cells. Blue line shows production of sfGFP by reporter cells over time in the presence of IPTG, and the orange line shows the same in the absence of IPTG.
Ratio 3 - 1 detector : 1 processor : 200 reporters
The graph below (figure 12) was created using data generated by the Simbiotics software tool using default values for diffusion of the quorum sensing molecules (C12 and C4), and the inducer IPTG, and default values for membrane permeation by the three molecules. The system was simulated in the presence (blue line) and absence (orange line) of IPTG.
When the three cell types were in the ratio of 1:1:200 for detectors:processors:reporters, there was a large difference in sfGFP production when IPTG was present vs. when it was absent.
Figure 12: Simbiotics simulation of Sensynova framework with 1 detector cell, 1 processor cell, and 200 reporter cells. Blue line shows production of sfGFP by reporter cells over time in the presence of IPTG, and the orange line shows the same in the absence of IPTG.
Integration into Experimental Design
As shown above, the Simbiotics simulation predicted that the system worked most optimally when the reporter cells were far in excess of the detector and processor cells. Therefore, when the system was tested in the laboratory the reporter cells were co-cultured in excess of the detector and processor cells. The wet-lab data showed that the model was correct in its prediction, as a large signal-to-noise ratio was only seen when the system contained more reporter cells than detector and processor cells. Click here to go to the experimental testing for this system.
Discussions and Conclusions
As shown above, the model correctly predicted that the system worked most optimally when the reporter cells were in excess of both the detector and processor cells. One of the reasons that this configuration was the best may be because of signal amplification at each of the quorum sensing communication stages. The quorum sensing mechanism used here is the acyl homoserine lactone (AHL) system in gram negative bacteria. This system works by one cell producing a quorum sensing molecule which can diffuse out through its membrane. Once the extracellular space reaches a certain threshold concentration of AHL molecule, the AHL will begin to diffuse into other cells in the community. If the cell the AHL molecule enters has the appropriate transcription factor present (e.g. LasR for the C12 AHL), then transcription of a gene under the control of the pLas promoter can occur. Therefore, if background expression of the AHL is high enough to reach above the threshold level, then expression of the next quorum sensing molecule in another cell (in this case C4 AHL) will occur. By reducing the amount of detector and processor cells present in the system, the background expression levels of C12 and C4 will be lower, and hence expression of sfGFP by the reporter cell will be lower.
Future Work
The Simbiotics tool has many extra functions that could be useful in aiding the design of Sensynova biosensors. One of the features in the ability to organise cells in a pattern on a 2-dimensional surface (figure 13). This would allow modelling of the spatial organisation of the three cell type on a test dip-stick, which could be used similarly to tests such as litmus paper (for detecting pH), or pregnancy tests (for detecting human chorionic gonadotropin [hCG]). This may or may not provide a more optimal system for the biosensor compared to a mixed community in solution. Unfortunately, there was not time to complete this analysis.
Figure 13: Rendering of three cell types in a 2-dimensional spatial arrangement by the Simbiotics tool. Red dots are detector cells, blue dots are processing cells, and black dots are reporter cells. Cyan dots are C12 quorum sensing molecules.
References
Hoops, S., Sahle, S., Gauges, R., Lee, C., Pahle, J., Simus, N., Singhal, M., Xu, L., Mendes, P., Kummer, U., (2006),  COPASI -- A COmplex PAthway SImulator, Bioinformatics, 22(24), pp 3067-3074
Hucka, M., et al., (2003), The Systems Biology Markup Language (SBML): A Medium for Representation and Exchange of Biochemical Network Models, Bioinformatics, 19(4), pp 524-531
Naylor, J., Fellerman, H., Ding, Y., Mohammed, W.K., Jakubovics, N.S., Mukherjee, J., Biggs, C.A., Wright, P.C., Krasnogor, N., (2017), Simbiotics: A Multiscale Integrative Platform for 3D modeling of Bacterial Populations, ACS Synth. Biol., 6(7), pp 1194-1210
Cell Free Protein Synthesis Systems Optimisation: Design of Experiments (JMP)
Abstract Overview of DoE for CFPS Systems: Components of the CFPS supplement solution premix were the factors of CFPS systems which were analysed, and the software program JMP (SAS Institute Inc., 2016) was used to apply Design of Experiments. In the first step, each supplement was created as a continuous factor within the JMP software, and the second step was to create an experimental design using DoE (the type of design depends on the question being asked; e.g. which supplements are the most important or what are the optimal concentrations for supplements A-D). CFPS reactions were then performed according to the generated design (step 3), and data was collected to be input back into the JMP software (step 4). The fifth step was to use the JMP software to produce models and statistical analyses of the data. The results could then be used to create a new experimental design (e.g. to further analyse factors highlighted as important), or predictions could be directly tested through experimental testing and observed results compared to said predictions.
Rationale and Aim
Previous research has shown that the concentration of some components of the supplement solution are crucial for efficient protein synthesis, and that for each batch of extract produced the optimal concentration may need to be found (Yang, et al., 2012). Studies which have explored this have only focused on, at most, a few components at a time (Garamella, et al., 2016; Kelwick, et al., 2016), which means that important interactions between the components may have been missed. In this study, a multifactorial approach will be taken to investigate the effect that all supplements have on the protein synthesis activity of CFPS systems simultaneously. The specific aims for this section of the project were: (i) to demonstrate the applicability of DoE to determine important components of the supplement solution premix, and (ii) to demonstrate the ability of DoE to predict concentrations of CFPS supplements which yield optimal protein synthesis activity.
Background Information
Traditionally, biologists tend to use One Factor At a Time (OFAT) approaches to determine the effect and importance of factors on a system, which can sometimes be a poor method. By only determining the effect that a single factor has on a system at a time, important interactions can be missed. For example, removing only factor A may have no effect, and removing only factor B may also have no effect, but removing both may cause an adverse effect. Therefore, it is important to take a multifactorial approach when investigating the importance of conditions or components of a system, or when trying to optimise a system. An issue with this approach is that a large number of experiments may be required to fully investigate all factors. By using statistical methods, a Design of Experiments (DoE) can be determined which has the minimum number of experiments required to explore questions such as the importance of factors in a system. This approach also allows for robustness testing or determining batch-batch variation (Anderson & Whitcomb, 2010). As discussed here , CFPS systems can be plagued with issues rising from variation, so this approach offers a method to investigate the causes. It could also be used to determine less important components of the supplement solution premix which is added to CFPS systems, and hence a minimal supplement premix could be determined. There are several different types of DoE designs. One of these is the screening design (SD), which is used to create experimental designs to determine the factors with the highest effect on a system. Another design is the surface response design (SRD), which makes experimental designs to collect data for generating models which can predict optimal settings for many factors (SAS Institute Inc., 2016). Software tools, such as JMP (SAS Institute Inc., 2016), can assist in creating these experimental designs.
Screening Design for Salt Supplements
Previous research has shown that the concentration of certain salts in the CFPS supplement premix are crucial for maximal protein synthesis activity (Yang et al. 2012). A Design of Experiments approach was used to determine which of the four salts (magnesium glutamate, potassium glutamate, sodium oxalate, and ammonium acetate) are the most important using the JMP software. A classical screening design was created with all four salts as continuous factors and CFPS activity as the response to be maximised. A concentration of ‘0’ was used as the lower limit for each factor, and the concentration used normally in CFPS supplement premixes was used as the upper limit (Figure 1). The screening design generated is shown in table 1.
Figure 1: Response and Factors for CFPS Salt Supplement Screening Design: Screenshot of JMP software program. CFPS activity was set as the response, and the response was set to be maximised. All four salts were input as continuous factors. The “Construct a main effects screening design” radio button was selected.
Table 1: Main effects screening experimental design: CFPS reactions contained concentrations of each salt supplement according to the table above. The pattern column shows whether a salt was present (✔)  or absent (X) from the system.
CFPS reactions were performed using supplement solution premixes with salt concentrations as determined by the main effects screening design. Reactions were incubated with 1.7 μg plasmid DNA encoding sfGFP (superfolder Green Fluorescent Protein) at 37 o C for 13 hours. CFPS activity was calculated as fluorescence intensity at 13 hours minus fluorescence intensity at 15 mins. This data was then used to generate a bar chart of Contrast values and a Half-Norma Plot (Figure 2 and 3) to determine which factors were having the most effect on CFPS activity. It should be noted that predictions for non-primary factors (i.e. interactions) may be inaccurate as they were forced-orthogonal. Considering the primary factors, magnesium glutamate was found to be the salt supplement with the largest contrast value, followed by potassium glutamate. This suggests that these two salt supplements were the most important. Sodium oxalate had a lower contrast value than either of the two glutamate salts, and was considered to have moderate importance in terms of CFPS activity. Ammonium acetate had an extremely low contrast value, suggesting that it may be unimportant for enhancing CFPS activity.
Figure 2: Contrast Values for CFPS Salt Supplements: Bar chart of contrast values for each term in the CFPS salt supplements Main Effects Screening Design generated by the JMP software. Contrast values were used as an estimate of a factor’s effect on the response. Interactions (*) were forced orthogonal.
Figure 3: E. coli Half-Normal Plot for Salt Supplements: Half-normal plot generated by the JMP software from data collected according to the salt supplement screening design. Blue line is the Lenth’s Pseudo-Standard Error (PSE). Primary terms are shown on the plot as circles (•), other terms are shown as crosses (+).
Surface Response Designs for Salt Supplements
The DoE software, JMP, was used to create a surface response design (SRD) for the three salts which were found by the screening design to have the most effect on CFPS activity (magnesium glutamate, potassium glutamate, and sodium oxalate). Ammonium acetate was kept at the default concentration and was not varied. Four SRDs were created using JMP; Central Composite Design-Uniform Precision design (CCD-UP), Box-Behnken (BB), Central Composite Design-Orthogonal (CCD-O), and Central Composite Design (CCD). The design diagnostics feature was used to compare the designs (Figure 4). Specifically, the colour map on correlations, power analysis for each factor and interaction, D, G, and A efficiencies, average variance of prediction, and number of reactions were compared to determine which design would be used. The colour map on correlations shows how correlated two terms are (red is highly correlated, blue is highly un-correlated). The more correlated two terms are, the more difficult it is to determine which is responsible for the effect on the response (Anderson & Whitcomb, 2010). As would be expected, in each design, terms are highly correlated with themselves (observed as a diagonal red line). Other terms are generally very lowly correlated with different terms. For the CCD-UP, BB, and CCD, the terms at the bottom right of the map have correlations above 0. For CCD-UP and BB, these correlations are still very low, but for CCD they are at about 0.5. Power analysis shows the likelihood of detecting an active effect for terms in the design (Anderson & Whitcomb, 2010). The CCD-O had a higher Power for all terms, with CCD-UP having the next highest. BB and CCD had lower Power for all terms, but some terms were higher in the BB design than the CC design, and some higher in CC design than the BB design.
Figure 4: CFPS Salt Supplement Surface Response Design Comparison : Comparison of four surface response designs generated by the JMP software using the compare designs feature. From top to bottom: design type, number of reactions required by the design, colour map on correlations, Power analysis of terms, efficiencies, and average prediction variance. For the colour map on correlations, red is highly correlated and blue is highly un-correlated.
D, G, and A efficiencies are a measure of each design to be D, G, and A optimised. A design is D optimal if confidence regions for the vector of regression coefficients are minimized, G optimal if maximum prediction variance over the design region is minimized, and A optimal if the sum of the regression coefficient variance is minimized (Anderson & Whitcomb, 2010). The CCD-UP design has the highest D efficiency and the BB design has the lowest. The CCD design has the highest G efficiency and the BB design has the lowest. The CCD-UP design has the highest A efficiency and the BB design has the lowest. The last two values analysed to determine which design would be used were the average variance of prediction, for which CCD-O had the lowest and BB had the highest, and the number of reactions required by each design, for which CCD-O had the highest and BB had the lowest. Taking all of the information into account, the CCD-Orthogonal design was chosen as it has no correlations between non-identical terms, high Power for all terms, relatively high efficiency scores, and low prediction variation. CFPS reactions with salt supplement amounts according to the Surface Response experimental design (Table 2) were performed (Figure 5). Five reactions (12-16) were discarded from analysis. These were all repeats of CFPS systems with default amounts of salts and showed no CFPS activity due to an error during set-up. Discarding these results had a minor effect on the diagnostics of the surface response design, with some terms becoming more correlated (but all values were still below 0.2), the Power for terms decreasing slightly, and average variance of prediction increasing slightly. Despite this, the D, G, and A efficiencies all increased.
Table 2: Salt Supplements Surface Response Experimental Design: Table of reactions performed according to the DoE salt supplement surface response design. CFPS reactions contained concentrations of magnesium glutamate, potassium glutamate, and sodium oxalate, according to the table above. The pattern column shows how much of each supplement was present in each reaction; very low concentration (a), low concentration (−), usual concentration (0), high concentration (+), and very high concentration (A).
Figure 5: CFPS Salt Supplements Surface Response Design : CFPS activity (fluorescence at each time point minus fluorescence at 15 mins) of reactions with salt supplements in amounts according to the DoE salt surface response design. Legend shows the reaction number and ‘pattern’ for three of the salt supplements in the order of magnesium glutamate, potassium glutamate, and sodium oxalate (Table 2).
Results for the remaining reactions were used to build a model in JMP to predict an optimal composition for the three salts. The model predicted that at high amounts, magnesium glutamate and sodium oxalate were having an inhibitory effect, and potassium glutamate was having an enhancing effect on CFPS activity (Figure 6). It is well known that magnesium ions are crucial for protein synthesis, for example in the functioning of ribosomes, however at high amounts magnesium can become inhibiting to protein synthesis by stalling translation at the translocation step (Li, et al., 2014). Therefore, it is not unexpected that magnesium glutamate causes a decrease in protein synthesis activity at certain concentrations.
Figure 6: Cube plot generated by JMP using data collected for the SRD. The x-axis shows magnesium glutamate concentration, the y-axis shows potassium glutamate concentration, and the z-axis shows sodium oxalate concentration. The ovals shows predicted CFPS concentration. The maximal CFPS activity was found at 195 mM potassium glutamate, 6 mM magnesium glutamate, and 2 mM sodium oxalate. The minimal point was found at 65 mM potassium glutamate, 18 mM magnesium glutamate, and 2 mM sodium oxalate. This was verified as the maximum and minimum points using the surface profiler function in JMP.
As mentioned before, sodium oxalate is used as an inhibitor of the enzyme which converts pyruvate to PEP. However, pyruvate is also con verted to desirable metabolites during the pathway which generates ATP for protein synthesis, and these reactions should not be inhibited. Sodium oxalate acts as a pyruvate mimic, and therefore at high concentrations it may not only inhibit conversion of pyruvate to PEP, but also pyruvate to acetyl-CoA, which would decrease the amount of ATP generate, and hence reduce protein synthesis activity. This may be one explanation for why sodium pyruvate appears to have an inhibitory affect. Using this data, a maximum protein synthesis activity within the range of concentrations used for each salt was found at 6 mM magnesium glutamate, 195 mM potassium glutamate, and 2 mM sodium oxalate (Figure 6). A CFPS supplement solution with these revised amounts was made and used to perform CFPS reactions. Two types of CFPS system were used; one which contained cell extract from the same batch that was used to build the SRD model (B1), and extract from a separate batch, but which was prepared in an identical way (D1). The results showed that CFPS reactions using extract from the same batch that was used to build the model (B1) did indeed increase the CFPS activity of that extract, and the activity was within the confidence intervals (CIs) predicted by the model (Figure 7a). For systems using a separate batch of extract (D1), the new supplement solution caused a decrease in activity (Figure 7b). This backs up previous research which suggests that each batch of cell free extract requires its own optimal conditions for high protein synthesis activity. It also shows that a multifactorial Design of Experiments approach can easily determine important factors in CFPS systems, and accurately predict optimal supplement amounts.
Figure 7: CFPS activity for two CFPS systems utilising two different cell extract batches prepared identically. A) Results for a system utilising the same extract batch used to test the SRD. B) Results for a system utilising a new batch of cell extract. The blue lines show systems using the normal CFPS supplement premix, and the purple lines show the systems with supplement premix with ‘optimised’ magnesium glutamate, potassium glutamate, and sodium oxalate as identified above. The system utilising extract from the same batch used in the SRD testing had a higher CFPS activity with the ‘optimised’ premix (purple) than with the original premix (blue), whereas the system utilising extract from a separate batch had higher activity with the original premix.
Screening Design for CFPS Supplements
Following the success of using a DoE approach to optimise CFPS systems focusing only on salt supplements, the JMP software was used to create a screening design for all 15 components of the CFPS supplement solution. The 15 components were made into 13 factors for the design, with UTP, CTP, and GTP being combined. The experiments determined by the design are shown in Table 3 The experimental design was performed on two different extracts; one with a moderate CFPS activity (D2), and one with a low CFPS activity (B2). For this set of reactions, cell extract was added such that 1.5 mg (30 mg/mL) of total cell extract protein was present in each CFPS reaction. This was done so that the factors predicted as being important could be compared between cell batches, without any small differences in total protein concentration potentially affecting the results. Figure 8 shows the contrast values (used here as an estimate of a factor’s effect on the response) for each supplement in CFPS systems using the moderately active cell free extract (D2), and Figure 9 shows the contrast values for each supplement in CFPS systems using the low active cell free extract (B2).
While some conclusions can be drawn from this data, care should be taken as the inherently low activity of systems using this data resulted in a high noise-to-signal ratio. Nonetheless, the DoE screening design identified 7 out of 13 factors which may have a negative effect on CFPS activity. This might have been expected because the CFPS system had a very low activity, which suggested that the supplement solution was far from optimised for that extract. Three of the factors with an inhibitory affect were potassium glutamate, sodium oxalate, and ammonium acetate, which, as has already been explained earlier, is not unusual as cell extract require specific amounts of these salts for high activity. Other supplements which appear to be having a strong negative effect on CFPS activity, such as the tRNAs and Co-enzyme A, are harder to explain.
Due to a lack of time, repeats of these experimental designs were not performed, and therefore the results of this analysis are only indicative. Further experimentation should be performed to validate these results, and hence the conclusions drawn.
Conclusions and Future Work
This study has begun multifactorial analysis on the components of the supplemental solution for cell free protein synthesis systems. It has provided evidence that some supplements have a greater effect on protein synthesis activity than others, and that the important factors may differ between cell extract batches. The ability to use a Design of Experiments approach towards the optimisation of CFPS systems has also been demonstrated. While this study has provided evidence to-wards these claims, further work should be performed to validate the findings. A DoE screening design for the supplements of CFPS sys-tems should be used on the same cell extract batch repeatedly. This will help confirm that the screening models derived from the experi-mental design data are accurate. The screening design should also be performed on many different batches of at least moderately active cell extracts to confirm that important supplements do differ between batches. Following the above, a surface response design could be used for all commonly important supplements of the CFPS system to deter-mine its effectiveness at optimising CFPS activity. The information could also be used to determine commonly unimportant supplements so they can be eliminated from the supplement solution, hence de-creasing the cost per reaction.
References
Anderson, M. J. & Whitcomb, P. J., 2010. Design of Experiments. In: Kirk-Othmer Encyclopedia of Chemical Technology. s.l.:John Wiley & Sons, Inc, pp. 1-22.
Garamella, J., Marshall, R., Rustad, M. & Noireaux, V., 2016. The All E. coli TX-TL Toolbox 2.0: A Platform for Cell-Free Synthetic Biology. ACS Syn. Biol., 5(4), pp. 344-355.
Kelwick, R., Webb, A. J., MacDonald, J. & Freemont, P. S., 2016. Development of a Bacillus subtilis cell-free transcription-translation system for prototyping regulatory elements. Metab. Eng., Volume 38, pp. 370-381.
Li, J., Gu, L., Aach, J. & Church, G. M., 2014. Improved Cell-Free RNA and Protein Synthesis System. PLoS ONE, 9(9).
SAS Institute Inc., 2016. JMP® 13 Design of Experiments Guide. Cary, NC, USA: SAS Institute Inc.
Yang, W. C., Patel, K. & Wong, H. E., 2012. Simplifying and streamlining Escherichia coli-based cell-free protein synthesis. Biotechnol. Prog., 28(2), pp. 413-420.
Microfluidic Agent-Based Model
Motivation and Aim
Digital microfluidics is an area of study intersecting biology, computer science, electronics and several different engineering disciplines. The technology has seen a number of advances and improvements over recent years, with the dream of a “lab on a chip” inching ever closer. Digital microfluidic devices satisfy the requirements of our project very well as they allow the process of switching modular components to be automated. We explored how digital microfluidics could be used to automatically mix the the different modular cellular components in our biosensors in different ratios to optimise the biosensor response characteristics. Our first step was to develop a simulator to model the behaviour of droplet movement on droplet-based devices.       The aim of creating this model was to eventually create software to be used alongside microfluidic devices to continue the theme of automation of production of modular components in the project.
Why Digital Microfluidics?
The motivation behind working on a project creating software for digital microfluidics over other microfluidic techniques stems from the numerous advantages that it confers over a more traditional setting involving continuous flow microfluidics. The most basic of these advantages is very simple in that it allows for a reduction in consumed reagents and samples. A natural bonus of this is the expenses saved in using lower quantities of reagents. This also offers another less obvious advantage in that as other microfluidic techniques have shown, a lower reagent volume causes a faster overall result (Whitesides, 2006). The comparatively larger surface to volume ratio which can be achieved with the droplets also aids the speed with which reactions can occur. (Haeberle and Zengerle, 2007). As such, when dealing with single droplets of reagents, as is always the case in a digital microfluidic setting, digital microfluidic technologies are therefore able to achieve results faster than is otherwise possible. Alongside the innate benefits derived from the physics of using single droplets as described above, the much greater degree of automation conferred from use of a digital microfluidics system is also significant in speeding up laboratory processes. Even when introducing sensing systems and feedback control a very high degree of precision in the generation of droplet volumes (Liu, Banerjee and Papautsky, 2014) and also the control of mixing processes (Gong and Kim, 2008) is attainable. Combining this with the ability to then parallelise multiple versions of the same functions on the same chip all at one time, providing simultaneous output of multiple reactions, allows for a vast decrease in the time-scale required for comparatively larger amounts of reagents in reactions. This provides a very valid use case for the automation of modular variants in our project. When creating the programs both for the system to operate from and to simulate the system, an agent-based approach to modelling was used. An agent-based system is one in which each entity in the system is considered its own unique “agent”, where each agent of the same type holds the same properties but each with their own values. An agent-based system assesses the effects on the system as a whole by observing each individual agent and monitoring their own actions and their interactions with other agents (Macal and North, 2010). In a synthetic biology context, agent based models have been used to model the discrete elements of different systems and are able to capture even some of the most minor differences between agents inside the system. (Gorochowski, 2016).
Simulator Software
The goal of this section of the project is to create a piece of scalable real-time software capable of simulating the full functionality of our EWOD microfluidic device. It should additionally be able to schedule its own microfluidic operation sets to simulate. This program will then allow for a controller on the hardware device to enact the given schedule, whereby a set of operations can be carried out on the chip. These operations can be customised according to the individual capabilities of the version of hardware being used. This therefore has extended capabilities configurable to the breadth of operations of the device being simulated, including more complex operations in the field of digital microfluidics such as mixing and extensibility for zones capable of manipulating temperature. The software provides a separate but fundamentally similar interface alongside that which the chip will be running, creating a simulator able to mimic the effects of running a process on the chip. This serves as an inexpensive and rapid means of testing a real EWOD system. In order to serve a real purpose, the simulation software must not simply function correctly but also provide a simple, fast means of displaying, logging and outputting the information it processes. As such the software must not only be correct, verified, and validated, but also intuitive both to extend and use the full range of the base functionality. This introduces a key component of human-computer interaction to the task whereby the software must be able to cater to the precise needs of the user. It must also be possible to tailor it accordingly.
Agent-Based Modelling
The use of an agent-based system for this task is advantageous in a number of key areas. Agent-based systems excel in giving a very clear idea of how the properties of an individual are affecting the greater system, especially when individuals are heterogeneous (Bonabeau, 2002). In this project for example, if scheduling is not optimal for a chip layout then there may be significant bottlenecking in a particular area. Bottlenecking is an example of emergent phenomena in that it can only occur as a result of the properties of many individuals. With only access to the behaviour of the entire system, it can be very difficult to determine the cause of a bottleneck. With an agent based approach however, and analysis of several individuals in the affected area, the cause of the problem will in most cases quickly become apparent. Another advantage of using an agent-based system for this project is that we are also implementing the scheduling for the software. Our scheduling system requires knowledge of each of the individual droplets in the system, and it must also be able to predict their movements for a certain period of time into the future. Computationally we are therefore already expending time and resources upon acquiring and manipulating these data sets and as such it makes sense to extend this knowledge into the agent-based model.
References
Bonabeau, E. (2002). Agent-based modelling: Methods and techniques for simulating human systems. Proceedings of the National Academy of Sciences, 99(Supplement 3), pp.7280-7287. Gong, J. and Kim, C. (2008). All-electronic droplet generation on-chip with real-time feedback control for EWOD digital microfluidics. Lab on a Chip, 8(6), p.898. Gorochowski, T. (2016). Agent-based modelling in synthetic biology. Essays In Biochemistry, 60(4), pp.325-336. Haeberle, S. and Zengerle, R. (2007). Microfluidic platforms for lab-on-a-chip applications. Lab on a Chip, 7(9), p.1094. Liu, Y., Banerjee, A. and Papautsky, I. (2014). Precise droplet volume measurement and electrode-based volume metering in digital microfluidics. Microfluidics and Nanofluidics, 17(2), pp.295-303. Macal, C. and North, M. (2010). Tutorial on agent-based modelling and simulation. Journal of Simulation, 4(3), pp.151-162. Whitesides, G. (2006). The origins and the future of microfluidics. Nature, 442(7101), pp.368-373.
