<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper" style="display:None"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Newcastle"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Newcastle/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Newcastle/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Newcastle/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:Newcastle/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Newcastle/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Newcastle/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Software"><DIV class="submenu_button" id="Software_page">
					Software
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=Newcastle"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><BUTTON class="btn btn-primary collapsed" type="button" data-toggle="collapse" data-target="#jack" aria-expanded="true" aria-controls="jack" style="max-height: 30px; width: 100%; position: fixed; font-size: 0.6em; background-color: #8db2e7; border-color: #222222; color: #222222">Show Navbar</BUTTON><DIV style="z-index: -10; height: 30px; background-color: #e8e8e8; margin-bottom: 0"><P style="font-size: 0.1em; margin-bottom: 0">spacefill</P></DIV><DIV id="jack" class="collapse show" style="margin-bottom: 0"><NAV id="nav-jack" class="navbar fixed-top navbar-expand-lg navbar-custom" style="margin-top: 16px; margin-bottom: 0"><UL class="navbar-nav mr-auto"><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
            Project
          </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Description">Description</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Demonstrate">Demonstration</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Protocols">Protocols</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Parts">Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/InterLab">Interlab Study</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
            Modelling
          </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Model">Overview</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Model#sim">Simbiotics</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Model#doe">Design of Experiments</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Model#mf">Microfluidic Agent-based Model</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
              Human Practices
            </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/HP/Silver">Silver</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/HP/Gold_Integrated">Gold and Integrated</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Engagement">Education and Public Engagement</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/ScienceCommunication">Science Communication</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
                Collaborations
              </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Collaborations">Overview</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Collaborations#edi_collab">Edinburgh OG</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Collaborations#exe_collab">Exeter</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Collaborations#eps_collab">Evry Paris-Saclay</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
                  Judging
                </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/MedalRequirements">Medal Requirements</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Safety">Safety</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Measurement">Measurement</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Applied_Design">Applied Design</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Part_Collection">Part Collection</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Improve">Improvement</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
                    Team
                  </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Team">Meet the Team</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI></UL><BUTTON class="btn btn-primary collapsed" type="button" data-toggle="collapse" data-target="#jack" aria-expanded="true" aria-controls="jack" style="color: #222222; background-color: #8db2e7; border-color: #222222">Collapse Navbar</BUTTON></NAV><DIV style="z-index: -10; height: 100px; background-color: #e8e8e8; margin-bottom: 0"><P style="font-size: 0.1em; margin-bottom: 0">spacefill</P></DIV></DIV><DIV class="container-fluid" id="main" style="width: 100%; padding-top: 0.5%"><DIV class="container-fluid" style="max-width: 60%"><H1 style="font-family: Rubik">Our Project</H1><P style="font-family: Rubik">
We have developed a multicellular, modular biosensor development platform to usher in a new era of biosensors. The platform aims to ease the design, implementation, and characterisation/optimisation stages of biosensor development: Click NEXT on the diagram below for an overview of the project.
</P><DIV style="border-style:solid; border-color:black; border-width: thick; padding: 10px;"><TABLE style="background:none;"><TBODY><TR><TD><BUTTON id="prev" class="w3-button w3-display-left" onclick="plusDivs(-1)">Previous</BUTTON></TD><TD><BUTTON id="next" class="w3-button w3-display-right" onclick="plusDivs(+1)" style="float:right;">Next</BUTTON></TD></TR></TBODY></TABLE></DIV><H1 style="font-family: Rubik">What is a Biosensor?</H1><P style="font-family: Rubik">
        Biosensors can be thought of as any device which is capable of sensing an analyte (e.g. a molecule or compound) or certain condition (e.g. pH or temperature) through the use of a biological component (Turner, 2013). One example of this would be a canary in a coal mine, where in the presence of carbon monoxide, the canary dies. A perhaps less morbid and more advanced biosensor example are those which have been developed by synthetic biologists. All organisms use native biosensing devices to monitor molecules of interest and initiate cell responses. For example, maintenance of cell homeostasis requires the sensitive detection and subsequent regulation of many molecules, such as metals, fatty acids and hydrogen peroxide (Rensing &amp; Grass, 2003, Zhang &amp; Rock, 2009 and Marinho <I>et al</I>., 2014). Two-component systems are common biosensing systems in bacteria. These systems allow bacteria to respond to extracellular signals by the phosphorylation of a sensor kinase in the presence of a target molecule, which subsequently phosphorylates further response regulator proteins. These response regulators can alter cell behaviour through protein interactions, transcriptional regulation, or RNA binding (Gao <I>et al</I>., 2007).
        
        In recent years, there has been a substantial increase in the number of biosensors produced using synthetic biology methods. Synthetic biology involves the application of engineering principles to the manipulation of biological systems. Biosensors constructed using these methods adapt the native cellular biosensing processes discussed previously, such as protein or RNA binding, and use these interactions to induce transcription of a reporter gene, such as a fluorescent protein.
        
        These sensors may be expressed as living whole-cell sensors, but are also increasingly being expressed in cell-free protein synthesis systems. However, thus far, the costs of these systems has been prohibitive to wide-spread use in synthetic biology (Smith <I>et al</I>., 2017).
      </P><H1 style="font-family: Rubik">Why are Biosensors Useful?</H1><P style="font-family: Rubik">
        One main advantage of synthetic biology based biosensors is their cost-effectiveness. After the research stages, production of the biosensor relies only on the maintenance of a population of cells expressing an engineered system, which is a relatively cheap process in comparison to other traditional methods such as immunoassays or mass spectrometry. Synthetic biology biosensors can be designed to have no dependence on additional equipment, which not only adds to their cost-effectiveness, but also enables onsite diagnostics (Bhatia &amp; Chugh, 2013). Synthetic biology approaches also enable the introduction of more complex behaviour into biosensor designs, such as logic gates which allow for signal generation in response to a variety of simultaneous triggers (Chappel &amp; Freemont, 2011).
        
        One specific example of a biosensor is an arsenic biosensor, developed by Aleksic <I>et al</I>. (2007). This sensor was able to generate pH changes in response to the presence of arsenic in drinking water. In this system, ArsR, an arsenic responsive transcription factor, represses the pArs promoter in the absence of arsenic. When arsenic is present and bound to ArsR, the protein no longer binds and represses the promoter, enabling the transcription of downstream genes. In this example, the downstream gene is urease, which generates a detectable pH change. Therefore, the presence of arsenic can be detected by the monitoring of pH.
      </P><H1 style="font-family: Rubik">What problems do Biosensor Developers Face?</H1><P style="font-family: Rubik">
        Our project focuses on the challenges of biosensor development: If synthetic biology biosensors are so much better than the alternatives, which are often expensive and not capable of onsite diagnosis, why isn’t their use more widespread?
        
        In an attempt to answer this question, we consulted various stakeholders in biosensor development: both in and outside the field of synthetic biology, from the early research stage to end-users. We <A href="https://2017.igem.org/Team:Newcastle/HP/Gold_Integrated">skyped, emailed, attended conferences.</A> It was determined the problems faced by biosensor developers are in 5 main areas, detailed below.
        <TABLE style="background:none;"><TBODY><TR><TD><BUTTON class="w3-button w3-display-left" onclick="plusDivsprob(1)"><FONT size="2.5">Part Reusability</FONT></BUTTON></TD><TD><BUTTON class="w3-button w3-display-right" onclick="plusDivsprob(2)" style="float:right;"><FONT size="2.5">Extensive Optimisation</FONT></BUTTON></TD><TD><BUTTON class="w3-button w3-display-right" onclick="plusDivsprob(3)" style="float:right;"><FONT size="2.5">Range of Analytes &amp; Outputs</FONT></BUTTON></TD><TD><BUTTON class="w3-button w3-display-right" onclick="plusDivsprob(4)" style="float:right;"><FONT size="2.5">Uptake of Technology</FONT></BUTTON></TD><TD><BUTTON class="w3-button w3-display-right" onclick="plusDivsprob(5)" style="float:right;"><FONT size="2.5">GMO Legislation</FONT></BUTTON></TD></TR></TBODY></TABLE></P><H1 style="font-family: Rubik">About the Sensynova Framework</H1><H2 class="text-left">Overview</H2><P style="font-family: Rubik" class="text-justify">
        In this project, it is proposed that modularity, and therefore the ability to use parts “off-the-shelf” without further genetic engineering, could be improved by splitting components of biosensors into different cells which communicate to coordinate responses. The orthogonal quorum sensing systems Rhl and Las will be used as biological “wires”, linking different biosensor components together. This separation of components will enable the decoupling of non-specific components from specific detection systems. Using this approach, production of biosensor variants will not require subsequent engineering steps: cells containing desired components will simply be mixed together.
        </P><H2 class="text-left">Background Information</H2><H3>Multicellular systems</H3><P style="font-family: Rubik" class="text-justify">

        The splitting of biosensor components into separate cells may have additional advantages besides ease of variant production. Goni-Moreno <I>et al</I>. (2011) have previously suggested that the use of synthetic quorum sensing circuits enables each cell to be considered an independent logic gate, which may rectify the “fuzzy logic” seen in some biosensors, where stochastic cellular processes may produce false positive results. Another advantage of splitting biosensors into separate cells is that it reduces the load on any one cell. If one cell is required to express the entire biosensor system on a high copy number plasmid like pSB1C3, then resources in the cell can become limited and cell growth rates can become problematic. By splitting the network into thirds and expressing them in different cells, then no one cell is required to deal with that level of stress.
</P><DIV><P class="legend"><STRONG>Figure 1:</STRONG> Three-input multicellular biosensor design by Wang <I>et al.</I> 2013. RFP is produced in the presence of arsenic, mercury, and copper. Figure taken from Wang <I>et al.</I> 2013 (Figure 4a).</P></DIV><P style="font-family: Rubik">
The concept of biosensors in a multicellular environment was described by  Wang and co-workers in (2013). Wang and co-workers used genetic logic gates in multiple cells to integrate signals from the detection of multiple analytes to one output. They used this concept to design a three-input heavy-metal biosensor, which produced a signal only in the presence of mercury, arsenic, and copper (Figure 1). One cell type in the community used a genetic AND gate to activate the expression of <I>luxI</I> in the presence of arsenic <I>and</I> mercury. LuxI synthesises the quorum sensing molecule 3OC<SUB>6</SUB>HSL. A second cell type in the community used the same genetic AND gate to produce red fluorescent protein (RFP) in the presence of the HSL and copper. It was proposed in this study that this approach could lead to easily customisable and modular biosensors. While this design does allow the biosensor to be customised to some extent (e.g. the <I>P<SUB>hrpL</SUB>-rbs30-rfp</I> construct could be replaced with a <I>P<SUB>hrpL</SUB>-rbs30-sfGFP</I> construct), the individual parts are still coupled tightly together on the same DNA molecule, and mostly still within the same cell. Additionally, this design only allows for the design of biosensors with AND gates, and there is no capability to add additional signal processing modules into the system. Nevertheless, this study demonstrates that the principle of making biosensors multicellular and modular is both possible and useful.  We drew inspiration for our project from this paper and extended their concept with the idea of an off the shelf set of module components, enhanced separation of the design through systematic analysis of previous configurations and the idea of using different ratios of cellular components to optimise the sensor response characteristics. We also included the idea of a cell-free adaptor system.</P><H3>Quorum sensing mechanisms</H3><P style="font-family: Rubik">
In the Sensynova framework, two quorum sensing (QS) mechanisms are employed; the LasIR system and the RhlIR system. Both systems are acyl-homoserine lactone (AHL) based systems found in gram negative bacteria; specifically the Las and Rhl systems were orignially characterised in <I>Pseudomonas aeruginosa</I> as a regulatory mechanism for virulence factors (Pearson <I>et al.</I> 1997). The mechanism for the Las QS system is shown in Figure 2. The mechanism for the RHL system is identical, except LasI is replaced with RhlI (which produces C4-HSL instead of C12-HSL), and LasR is replaced with RhlR (which is activated by C4-HSL and activates transcription from the <I>P<SUB>Rhl</SUB></I> promoter).</P><DIV><P class="legend"><STRONG>Figure 2:</STRONG> LasIR quorum sensing mechanism. LasI is produced constitutively by a cell. LasI synthesises the C12-HSL quorum sensing molecule. C12-HSL is able to diffuse through the membrane into the extracellular space.
 When enough LasI positive cells (cells producing LasI) are present in one location, the amount of C12-HSL in the extracellular space reaches a threshold concentration. At this point, the quorum sensing molecule diffuses into the surrounding cells through their membranes. These cells also produce the LasR transcription factor. C12-HSL can bind to LasR,
 enabling the transcription factor to activate transcription from the <I>P<SUB>Las</SUB></I> promoter.</P></DIV><P class="text-justify style=" margin-top:="" 2="">
In nature, QS systems tend to be used by pathogens as a method of synchronising virulence factor production (Rutherford and Bassler, 2012). When enough pathogens are present that infection is likely to succeed (dictated by the amount of cells required to reach the QS molecule threshold), virulence factors under the control of a promoter regulated by the QS transcription factor (e.g. LasR or RhlR) are expressed. This enables a co-ordinated attack, instead of individual or a small population of cells producing virulence factors, signalling their presence to the immune system, and reducing the likelihood of a successful infection.
QS mechanisms provide a very convenient mechanism to engineer cell-to-cell communication into synthetic microbial communities. Some cells in the population can be engineered to produce the AHL synthase (e.g. LasI or RhlI), while others to produce the AHL transcription factor (TF) (LasR or RhlR). Desired parts can then be assembled after the AHL-TF regulated promoter, such that they are only produced when the sender cell is present or activated. One example of quorum sensing in synthetic biology is to synchronise gene expression, which leads to reduced variability within a population (Danino <I>et al</I>., 2010).
In the Sensynova framework, the quorum sensing mechanisms are used as follows. The <I>lasR</I> gene is encoded by the detector cell under the control of a promoter regulated by the target analyte/condition of choice. In the presence of the target, LasR is produced and C12-HSL is synthesised and released into the extracellular space. The LasR transcription factor is produced constantly by the processor cells, and when the C12-HSL is present transcription from the <I>P<SUB>las</SUB></I> promoter is activated. The desired processing device is placed under the control of the <I>P<SUB>las</SUB></I> promoter, and RhlI synthase is produced by the processing device. The RhlI device can then synthesise the C4-AHL, which passes through into the extracellular space. The reporter cell encodes and expresses the transcription factor RhlR. When enough C4-AHL has been produced by the processor cells, the AHL will pass through the membrane into cells in the community. The C4-AHL which passes into the reporter cells will bind to the RhlR, and activate transcription from the <I>P<SUB>rhl</SUB></I> promoter, which is upstream from a reporter coding sequence (e.g. superfolder GFP). This process is shown diagrammatically in Figure 3.
</P><DIV><P class="legend"><STRONG>Figure 3:</STRONG> Diagrammatic overview of quorum sensing in the Sensynova framework. Description in main text.</P></DIV><H1 style="font-family: Rubik">References:</H1><P style="font-family: Rubik">  
Aleksic, J., Bizzari, F., Cai, Y., Davidson, B., De Mora, K., Ivakhno, S., Seshasayee, S.L., Nicholson, J., Wilson, J., Elfick, A., French, C., Kozma-Bognar, L., Ma, H. &amp; Millar, A. (2007)  Development of a novel biosensor for the detection of arsenic in drinking water. <I>IET Synthetic Biology</I> 1: 87–90

Bhatia, P. &amp; Chugh, A. (2013) Synthetic Biology Based Biosensors and the Emerging Governance Issues Current Synthetic and Systems Biology 1: 108 

Chappel, J. &amp; Freemont, P. (2011) Synthetic Biology – A new generation of biofilm biosensors Forum on Microbial Threats. The Science and Applications of Synthetic and Systems Biology: Workshop Summary. <I>National Academies: Washington (DC)</I>

Danino, T., Mondragon-Palomino, O., Tsimring, L. &amp; Hasty, J. (2010) A synchronized quorum of genetic clocks Nature 463: 326 - 330 

Gao, R., Mack, T.R., and Stock, A.M. (2007) Bacterial response regulators: Versatile regulatory strategies from common domains. <I>Trends Biochem. Sci.</I> 32: 225–234  

Goni-Moreno, A., Redondo, M., Arroyo, F. &amp; Castellanos, J. (2011) Biocircuit design through engineering bacterial logic gates Natural Computing 10: 119 – 127 

Rensing, C. &amp; Grass, G. (2003) <I>Escherichia coli</I> mechanisms of copper homeostasis in a changing environment FEMS Microbiology Reviews 27: 197 – 213

Marinho, S., Real, C., Cyrne, L., Soares, H. &amp; Antunes, F. (2014) Hydrogen Peroxide sensing, signalling and regulation of transcription factors Redox biology 2: 535 – 562

Smith, R., Marris, C., Berry, D., Sundaram, L. &amp; Rose, N. (2017) Synthetic Biology Biosensors for Global Health Challenges Workshop Report of the Flowers Consortium King’s College London 

Turner, A. P. F. (2013) Biosensors: sense and sensibility, Chem. Soc. Rev., DOI: 10.1039/C3CS35528D 

Zhang, Y. &amp; Rock, C. (2009) Transcriptional regulation in bacterial membrane lipid synthesis Journal of Lipid Research 50: S115 </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>