Team:Newcastle/Description merge
MENU
HOME
TEAM
Team
Collaborations
PROJECT
Description
Design
Experiments
Notebook
InterLab
Contribution
Model
Results
Demonstrate
Improve
Attributions
PARTS
Parts
Basic Parts
Composite Parts
Part Collection
SAFETY
HUMAN PRACTICES
Silver HP
Integrated and Gold
Public Engagement
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Plant
Software
JUDGING FORM
Project
Description Results Parts Interlab Study
Modelling
Overview Simbiotics Design of Experiment
Human
Practices
Overview Silver Gold and Integrated Education and Public Engagement Science Communication Legislation
Collaboration
Edinburgh_OG Exeter Evry Paris-Saclay
Judging
Medal Requirements Awards Safety
Team
Meet the Team Attributions Acknowledgements Lab Book
Show Menu
Click to reset page
What is a Biosensor?
Click to learn more
Biosensors can be thought of as any device which is capable of sensing an analyte (e.g. a molecule or compound) or certain condition (e.g. pH or temperature) through the use of a biological component (Turner, 2013). One example of this would be a canary in a coal mine, where in the presence of carbon monoxide, the canary dies. A perhaps less morbid and more advanced biosensor example are those which have been developed by synthetic biologists. All organisms use native biosensing devices to monitor molecules of interest and initiate cell responses. For example, maintenance of cell homeostasis requires the sensitive detection and subsequent regulation of many molecules, such as metals, fatty acids and hydrogen peroxide (Rensing & Grass, 2003, Zhang & Rock, 2009 and Marinho et al., 2014). Two-component systems are common biosensing systems in bacteria. These systems allow bacteria to respond to extracellular signals by the phosphorylation of a sensor kinase in the presence of a target molecule, which subsequently phosphorylates further response regulator proteins. These response regulators can alter cell behaviour through protein interactions, transcriptional regulation, or RNA binding (Gao et al., 2007).
In recent years, there has been a substantial increase the number biosensors produced using synthetic biology methods. Synthetic biology involves the application of engineering principles to the manipulation of biological systems. Biosensors constructed using these methods adapt the native cellular biosensing processes discussed previously, such as protein or RNA binding, and use these interactions to induce transcription of a reporter gene, such as a fluorescent protein.
These sensors may be expressed as living whole-cell sensors, but are also increasingly being expressed in cell-free protein synthesis systems . However, thus far, the costs of these systems has been prohibitive to wide-spread use in synthetic biology (Smith et al., 2017).
Why are Biosensors Useful?
Click to learn more
One main advantage of synthetic biology based biosensors is their cost-effectiveness. After the research stages, production of the biosensor relies only on the maintenance of a population of cells expressing an engineered system, which is a relatively cheap process in comparison to other traditional methods such as immunoassays or mass spectrometry. Synthetic biology biosensors can be designed to have no dependence on additional equipment, which not only adds to their cost-effectiveness, but also enables onsite diagnostics (Bhatia & Chugh, 2013). Synthetic biology approaches also enable the introduction of more complex behaviour into biosensor designs, such as logic gates which allow for signal generation in response to a variety of simultaneous triggers (Chappel & Freemont, 2011).
One specific example of a biosensor is an arsenic biosensor, developed by Aleksic et al. (2007). This sensor was able to generate pH changes in response to the presence of arsenic in drinking water. In this system, ArsR, an arsenic responsive transcription factor, represses the pArs promoter in the absence of arsenic. When arsenic is present and bound to ArsR, the protein no longer binds and represses the promoter, enabling the transcription of downstream genes. In this example, the downstream gene is urease, which generates a detectable pH change. Therefore, the presence of arsenic can be detected by the monitoring of pH.
What Problems do Biosensor Developers Face?
Click to learn more
The Sensynova Framework
Click to learn more
In this project, it is proposed that modularity, and therefore the ability to use parts “off-the-shelf” without further genetic engineering, could be improved by splitting components of biosensors into different cells which communicate to coordinate responses. The orthogonal quorum sensing systems Rhl and Las will be used as biological “wires”, linking different biosensor components together. This separation of components will enable the decoupling of non-specific components from specific detection systems. Using this approach, production of biosensor variants will not require subsequent engineering steps: cells containing desired components will simply be mixed together.
The splitting of biosensor components into separate cells may have additional advantages besides ease of variant production. Goni-Moreno et al. (2011) have previously suggested that the use of synthetic quorum sensing circuits enables each cell to be considered an independent logic gate, which may rectify the “fuzzy logic” seen in some biosensors, where stochastic cellular processes may produce false positive results. Quorum sensing has also been previously used to synchronise gene expressions, leading to reduced variability within a population (Danino et al., 2010).
References:
Turner, A. P. F., 2013, Biosensors: sense and sensibility, Chem. Soc. Rev., DOI: 10.1039/C3CS35528D
Email Address: newcastle.igem@outlook.com
Twitter: https://twitter.com/newcastle_igem
