<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Description_Jack skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Description Jack</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper" style="display:None"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Newcastle"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Newcastle/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Newcastle/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Newcastle/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:Newcastle/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Newcastle/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement 
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Newcastle/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:Newcastle/Software"><DIV class="submenu_button" id="Software_page">
					Software
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=Newcastle"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><BUTTON class="btn btn-primary collapsed" type="button" data-toggle="collapse" data-target="#jack" aria-expanded="true" aria-controls="jack" style="max-height: 30px; width: 100%; position: fixed; font-size: 0.6em; background-color: #8db2e7; border-color: #222222; color: #222222">Show Navbar</BUTTON><DIV style="z-index: -10; height: 30px; background-color: #e8e8e8; margin-bottom: 0"><P style="font-size: 0.1em; margin-bottom: 0">spacefill</P></DIV><DIV id="jack" class="collapse show" style="margin-bottom: 0"><NAV id="nav-jack" class="navbar fixed-top navbar-expand-lg navbar-custom" style="margin-top: 16px; margin-bottom: 0"><UL class="navbar-nav mr-auto"><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
            Project
          </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Description">Description</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Demonstrate">Demonstration</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Protocols">Protocols</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Parts">Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/InterLab">Interlab Study</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">      Modelling
          </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Model">Overview</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Model#sim">Simbiotics</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Model#doe">Design of Experiments</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Model#mf">Microfluidic Agent-based Model</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
              Human Practices
            </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/HP/Silver">Silver</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/HP/Gold_Integrated">Gold and Integrated</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Engagement">Education and Public Engagement</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/ScienceCommunication">Science Communication</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
                Collaborations
              </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Collaborations">Overview</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Collaborations#edi_collab">Edinburgh OG</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Collaborations#exe_collab">Exeter</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Collaborations#eps_collab">Evry Paris-Saclay</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
                  Judging
                </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/MedalRequirements">Medal Requirements</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Safety">Safety</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Measurement">Measurement</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Applied_Design">Applied Design</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Part_Collection">Part Collection</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Improve">Improvement</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none; display: inline"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
                    Team
                  </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Team">Meet the Team</A><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI></UL><BUTTON class="btn btn-primary collapsed" type="button" data-toggle="collapse" data-target="#jack" aria-expanded="true" aria-controls="jack" style="color: #222222; background-color: #8db2e7; border-color: #222222">Collapse Navbar</BUTTON></NAV><DIV style="z-index: -10; height: 100px; background-color: #e8e8e8; margin-bottom: 0"><P style="font-size: 0.1em; margin-bottom: 0">spacefill</P></DIV></DIV><DIV class="container-fluid" id="main" style="width: 100%; padding-top: 0.5%"><DIV class="container-fluid" style="max-width: 60%"><H1 style="font-family: Rubik">What is a Biosensor?</H1><P style="font-family: Rubik">
        Biosensors can be thought of as any device which is capable of sensing an analyte (e.g. a molecule or compound) or certain condition (e.g. pH or temperature) through the use of a biological component (Turner, 2013). One example of this would be a canary in a coal mine, where in the presence of carbon monoxide, the canary dies. A perhaps less morbid and more advanced biosensor example are those which have been developed by synthetic biologists. All organisms use native biosensing devices to monitor molecules of interest and initiate cell responses. For example, maintenance of cell homeostasis requires the sensitive detection and subsequent regulation of many molecules, such as metals, fatty acids and hydrogen peroxide (Rensing &amp; Grass, 2003, Zhang &amp; Rock, 2009 and Marinho et al., 2014). Two-component systems are common biosensing systems in bacteria. These systems allow bacteria to respond to extracellular signals by the phosphorylation of a sensor kinase in the presence of a target molecule, which subsequently phosphorylates further response regulator proteins. These response regulators can alter cell behaviour through protein interactions, transcriptional regulation, or RNA binding (Gao et al., 2007).
        
        In recent years, there has been a substantial increase the number biosensors produced using synthetic biology methods. Synthetic biology involves the application of engineering principles to the manipulation of biological systems. Biosensors constructed using these methods adapt the native cellular biosensing processes discussed previously, such as protein or RNA binding, and use these interactions to induce transcription of a reporter gene, such as a fluorescent protein.
        
        These sensors may be expressed as living whole-cell sensors, but are also increasingly being expressed in cell-free protein synthesis systems. However, thus far, the costs of these systems has been prohibitive to wide-spread use in synthetic biology (Smith et al., 2017).
      </P><H1 style="font-family: Rubik">Why are Biosensors Useful?</H1><P style="font-family: Rubik">
        One main advantage of synthetic biology based biosensors is their cost-effectiveness. After the research stages, production of the biosensor relies only on the maintenance of a population of cells expressing an engineered system, which is a relatively cheap process in comparison to other traditional methods such as immunoassays or mass spectrometry. Synthetic biology biosensors can be designed to have no dependence on additional equipment, which not only adds to their cost-effectiveness, but also enables onsite diagnostics (Bhatia &amp; Chugh, 2013). Synthetic biology approaches also enable the introduction of more complex behaviour into biosensor designs, such as logic gates which allow for signal generation in response to a variety of simultaneous triggers (Chappel &amp; Freemont, 2011).
        
        One specific example of a biosensor is an arsenic biosensor, developed by Aleksic et al. (2007). This sensor was able to generate pH changes in response to the presence of arsenic in drinking water. In this system, ArsR, an arsenic responsive transcription factor, represses the pArs promoter in the absence of arsenic. When arsenic is present and bound to ArsR, the protein no longer binds and represses the promoter, enabling the transcription of downstream genes. In this example, the downstream gene is urease, which generates a detectable pH change. Therefore, the presence of arsenic can be detected by the monitoring of pH.
      </P><H1 style="font-family: Rubik">What problems do Biosensor Developers Face?</H1><P style="font-family: Rubik">
        Our project focuses on the challenges of biosensor development: If synthetic biology biosensors are so much better than the alternatives, which are often expensive and not capable of onsite diagnosis, why isn’t their use more widespread?
        
        attempt to answer this question, we consulted various stakeholders in biosensor development: both in and outside the field of synthetic biology, from the early research stage to end-users. We skyped, emailed, attended conferences, detailed here (link to integrated HP) and even performed our own experiments (link to Biotech stuff). It was determined the problems faced by biosensor developers are in 5 main areas, detailed below. We spent the summer addressing these issues, and our solutions can be found on the following links:
        
        1) <B>Part Reusability - </B> The sometimes tedious techniques used to construct genetic circuits mean that interesting, reusable parts are often so tightly coupled inside the system they were developed in that they never get reused
        2) <B>Output Comparability - </B> The lack of any standards in biosensor development means that day-to- day background differences can have a huge effect on reporter values – without any change in input concentrations
        3) <B>Binding Sequences - </B> The bulk of the time spent in the design phase of biosensor development is in the search for a system which can interact with your target molecule. The more we can take the emphasis off the biology, the less we are limited by what nature has already created and the more we can move synthetic biology into a true engineering discipline
        4) <B>Uptake of Technology - </B> You’ve got your biosensor - great! But how are you going to make sure it gets used by the people it’s designed for?
        5) <B>Legislation  - </B> One of the main advantages of synbio biosensors over traditional methods is their reduced reliance on additional equipment, meaning they are perfect for onsite diagnostics and use in resource poor settings. However, no biosensor has ever successfully made it through european legislation for use outside of a laboratory</P><H1 style="font-family: Rubik">The Sensynova Framework</H1><P style="font-family: Rubik">
        In this project, it is proposed that modularity, and therefore the ability to use parts “off-the-shelf” without further genetic engineering, could be improved by splitting components of biosensors into different cells which communicate to coordinate responses. The orthogonal quorum sensing systems Rhl and Las will be used as biological “wires”, linking different biosensor components together. This separation of components will enable the decoupling of non-specific components from specific detection systems. Using this approach, production of biosensor variants will not require subsequent engineering steps: cells containing desired components will simply be mixed together.
        
        The splitting of biosensor components into separate cells may have additional advantages besides ease of variant production. Goni-Moreno et al. (2011) have previously suggested that the use of synthetic quorum sensing circuits enables each cell to be considered an independent logic gate, which may rectify the “fuzzy logic” seen in some biosensors, where stochastic cellular processes may produce false positive results. Quorum sensing has also been previously used to synchronise gene expressions, leading to reduced variability within a population (Danino et al., 2010).
        </P><H1 style="font-family: Rubik">References:</H1><P style="font-family: Rubik">
          Turner, A. P. F., 2013, Biosensors: sense and sensibility, Chem. Soc. Rev., DOI: 10.1039/C3CS35528D
        </P><H1 style="font-family: Rubik">FOOTER GOES HERE</H1><P style="font-family: Rubik">
          footer footer footer footer footer footer footer footer footer footer footer footer footer footer footer footer footer footer footer footer footer
        </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>