<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Results_Brad skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Results Brad</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="jack"><NAV class="navbar navbar-expand-lg navbar-custom"><UL class="navbar-nav mr-auto"><LI class="nav-item dropdown" style="border-style: none"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
            Project
          </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Newcastle/Description_Jack">Description</A><A class="dropdown-item" href="#">Results</A><A class="dropdown-item" href="#">Parts</A><A class="dropdown-item" href="#">Interlab Study</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
            Modelling
          </A><DIV class="dropdown-menu"><A class="dropdown-item" href="#">Simbiotics</A><A class="dropdown-item" href="#">Design of Experiment</A><A class="dropdown-item" href="#">Microfluidic Agent-based Model</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
              Human Practices
            </A><DIV class="dropdown-menu"><A class="dropdown-item" href="#">Silver</A><A class="dropdown-item" href="#">Gold and Integrated</A><A class="dropdown-item" href="#">Education and Public Engagement</A><A class="dropdown-item" href="#">Science Communication</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
                Collaborations
              </A><DIV class="dropdown-menu"><A class="dropdown-item" href="#">Collaborations Map</A><A class="dropdown-item" href="#">Edinburgh OG</A><A class="dropdown-item" href="#">Exeter</A><A class="dropdown-item" href="#">Evry Paris-Saclay</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
                  Judging
                </A><DIV class="dropdown-menu"><A class="dropdown-item" href="#">Medal Requirements</A><A class="dropdown-item" href="#">Awards</A><A class="dropdown-item" href="#">Safety</A></DIV></LI><LI class="nav-item dropdown" style="border-style: none"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">
                    Team
                  </A><DIV class="dropdown-menu"><A class="dropdown-item" href="#">Meet the Team</A><A class="dropdown-item" href="#">Attributions</A><A class="dropdown-item" href="#">Acknowledgements</A><A class="dropdown-item" href="#">Lab Book</A></DIV></LI></UL></NAV></DIV></DIV><DIV class="container-fluid" id="main" style="width: 100%; padding-top: 0.5%"><DIV class="container-fluid text-center" style="max-width: 80%; font-size: 1em"><H1 style="font-weight:normal;">Our Experimental Results</H1><DIV style="border: 1px solid black"><P3>
Hover over elements of the diagram below to see what each part represents. 

Click elements of the diagram below to see results for each section of our project.</P3><CENTER><TABLE class="result_picture" style="background:none; padding:0px; border:0px;"><TBODY><TR><TD><CENTER><P>Biochemical Adaptor</P></CENTER></TD></TR><TR><TD><CENTER><P>Target</P></CENTER></TD><TD><CENTER><P>Detector Modules</P></CENTER></TD><TD rowspan="4"><CENTER><P>Multicellular Framework Testing</P></CENTER></TD></TR><TR><TD><CENTER><P>C12 HSL: Connector 1</P></CENTER></TD></TR><TR><TD><CENTER><P>Processor Modules</P></CENTER></TD></TR><TR><TD rowspan="3"><CENTER><P>Framework in Cell Free Protein Synthesis Systems</P></CENTER></TD></TR><TR><TD><CENTER><P>C4 HSL: Connector 2</P></CENTER></TD></TR><TR><TD><CENTER><P>Reporter Modules</P></CENTER></TD></TR><TR><TD colspan="5"><CENTER><P2 width="30%">Looking for Interlab Study related results? Click below!</P2></CENTER></TD></TR></TBODY></TABLE></CENTER><P3 id="all" class="hoverable">
Alternatively, use the tabs below to navigate through our results and experiments.
</P3><P3>
Want to learn more about our framework (above)? Head over to our <A href="https://2017.igem.org/Team:Newcastle/Description#Sensynova">description page</A>!
</P3></DIV><NAV class="nav nav-tabs" id="myTab" role="tablist"><A class="nav-item nav-link active" id="nav-adaptor-tab" data-toggle="tab" href="#nav-adaptor" role="tab" aria-controls="nav-adaptor" aria-selected="true" style="font-weight:normal; font-size: 0.8em">Adaptor Modules</A><A class="nav-item nav-link" id="nav-detector-tab" data-toggle="tab" href="#nav-detector" role="tab" aria-controls="nav-detector" aria-selected="false" style="font-weight:normal; font-size: 0.8em">Detector Modules</A><A class="nav-item nav-link" id="nav-processor-tab" data-toggle="tab" href="#nav-processor" role="tab" aria-controls="nav-processor" aria-selected="false" style="font-weight:normal; font-size: 0.8em">Processor Modules</A><A class="nav-item nav-link" id="nav-reporter-tab" data-toggle="tab" href="#nav-reporter" role="tab" aria-controls="nav-reporter" aria-selected="false" style="font-weight:normal; font-size: 0.8em">Reporter Modules</A><A class="nav-item nav-link" id="nav-framework-tab" data-toggle="tab" href="#nav-framework" role="tab" aria-controls="nav-framework" aria-selected="false" style="font-weight:normal; font-size: 0.8em">Framework Testing (IPTG Sensor)</A><A class="nav-item nav-link" id="nav-cellfree-tab" data-toggle="tab" href="#nav-cellfree" role="tab" aria-controls="nav-cellfree" aria-selected="false" style="font-weight:normal; font-size: 0.8em">Cell Free Protein Synthesis System Optimisation</A></NAV><DIV class="tab-content" id="nav-tabContent"><DIV class="tab-pane fade show active" id="nav-adaptor" role="tabpanel" aria-labelledby="nav-adaptor-tab">
        ADAPT
      </DIV><DIV class="tab-pane fade" id="nav-detector" role="tabpanel" aria-labelledby="nav-detector-tab">
        DETECT
      </DIV><DIV class="tab-pane fade info_box" id="nav-processor" role="tabpanel" aria-labelledby="nav-processor-tab"><A class="nav-item nav-link active" id="FIM_title" data-toggle="tab" href=".FIM" role="tab" aria-controls="nav-adaptor" aria-selected="true" style="font-weight:normal; font-size: 0.8em"><H2 class="hoverable" id="FIM_title" style="text-align: left"> Fim Standby Switch </H2></A><P class="FIM">BioBricks used: BBa_0123456 (New), BBa_7890123 (Team_Name 20XX)</P><P class="FIM">Diagrammatic Overview: Representation of the switching mechanism of the Fim Switch, in the native [OFF] state the eforRED reporter is expressed (shown in red) allowing direct visualisation of the cells. Following the inversion of the promoter region (K1632004), eforRED expression is halted and the RhlI gene is expressed (J64718), this is now the [ON] state.</P><H2 class="FIM" style="text-align: left"> Rationale and Aim </H2><P class="FIM">Sensynova multicellular biosensor platform has been developed to overcome the limitations that hamper the success in biosensors development. One of these limits regards the lack of modularity and reusability of the various components. Our platform design, based on the expression of three main modules (Detector, Processor and Output) by three E.coli strains in co-culture, allows the switch of possible variances for each module and the production of multiple customised biosensors. 

This part can be used within the platform as a Processor unit. Real world applications of biosensors are limited by many factors, one of which is that with most biosensors there is not a readout signal showing if the biosensor is working when not in use, i.e that the cells are still alive and have not lost their biosensor phenotypes. This can make them difficult to use, as well as market, since their viability comes into question as well as leading to false negatives/positives. Biosensors which rely on expression of a reporter signal may also suffer from unobserved activation due to weak or inconstant induction. 

For this section of the project, as an improvement on a part by the Tokyo Tech team, we aim to produce a biobrick compatible part which is able to constitutively express a reporter signal prior to activation (to show that it is functioning) and to amplify a weak or inconsistent induction signal by permanently switching from an [OFF] to [ON] state after induction.</P><H2 class="FIM" style="text-align: left"> Background Information </H2><P class="FIM">Expression of the E. coli type 1 fimbriae gene is tightly regulated and phase dependent, i.e expression is either completely [ON] or [OFF] &gt;&gt;FimB FimE fimbrae control - Klemm 1986&lt;&lt;. This change in expression is controlled by the action of two proteins FimB and FimE which independently act upon a 300bp promoter region upstream of the fimbriae gene. The 300bp promoter region is inverted to either activate or suppress expression &gt;&gt; Roles of FimB FimE in site specific inversion - McClain 1991&lt;&lt;. Typical gene regulation mechanisms rely on up or down regulation of a promoter from a baseline expression, the fimbriae mechanism of ‘ALL’ or ‘NONE’ makes it a useful tool for synthetic biology applications. While the FimB protein inverts the promoter back and forth between [ON] and [OFF] states the FimE protein permanently inverts the promoter from [ON] to [OFF]. This inversion can be used to amplify weak or inconsistent induction signals. 

The part we are producing is a combination of the 2015 Tokyo Tech part (BBa_K1632013, BBa_K1632007) and the Voight lab 2007 part (BBa_J64718) quorum sensing RhlI an autoinducer synthesis protein which produces C4-HSL. The part by Tokyo Tech used FimE in combination with GFP to switch from an [ON] to [OFF] state, we have produced a construct where in the [ON] state the non-fluorescense based reporter protein eforRed (Uppsala 2011 BBa_K592012) is expressed. In the [OFF] state the RhlI quorum sensing protein from P. aeruginosa is expressed. The eforRed chromoprotein was chosen as fluorescence based reporters often require complex laboratory equipment to measure expression. 

Using a chromoprotein allows for the switch from [ON] to [OFF] to be verified by sight alone. The promoter flip also creates expression of the quorum sensing protein RhlI. This allows for a downstream activation of a reporter cell, meaning that the Fim Standby Switch has dual reporting properties. 

The use of the FimE switch as opposed to FimB (which is reversible) means that even with a low or inconsistent induction signal, the permanent inversion of the 300bp fim promoter region leads to a signal which can be observed. This may have applications in agricultural biosensors where there can be plant uptake of the potential induction compound (reference ). FimE was also chosen because FimB can switch the promoter back and forth, so it is likely to result in about half the plasmids with the promoter in one direction and half in the other over time, resulting in no strong signal either direction.

The inactivation of the eforRed chromoprotein following an induction signal visibly displays the [ON] to [OFF] inversion. Expression of this chromoprotein also displays that the [ON] section of the fim switch is working correctly.</P><H2 class="FIM" style="text-align: left"> Design Stage </H2><P class="FIM">To construct the Fim reporter switch 3 separate gBlocks were designed with overlapping adapter regions homologous to the iGEM prefix and suffix to allow for Gibson assembly into the pSB1C3 backbone whilst retaining biobrick compatibility. The individual genes and other components are shown in (Table 1). The 1st gBlock sequence starts with a RBS (B0034) upstream of the fimE ORF (K137007) with no promoter region, this is to allow for other promoters to be cloned in upstream of the part. Downstream of the fimE gene is a double terminator (B0015). All RBS and terminator sequences used are B0034 and B0015 respectively.</P><H2 class="FIM" style="text-align: left"> Implementation </H2><P class="FIM"> Text goes here.</P><H2 class="FIM" style="text-align: left"> Characterisation </H2><P class="FIM">The fim Standby Switch has two main functions; a visual signal to show that the target compound has been detected and AHL production so that the part can be detected by a reporter cell. To characterise the part these functions are individually tested, in aim to further isolate issues if they occur.
To test the detecting function of the Fim Standby Switch it was assembled with the PBAD/AraC promotor. The PBAD/AraC promotor with Standby Switch parts were plated out onto four different LB plates containing chloramphenicol, two plates with different concentrations of glucose and chloramphenicol, and another containing arabinose. As colonies for the Fim Switch section were red and white when plated onto chloramphenicol plates due to leakiness, the plates with glucose in theory should suppress this switching and a greater percentage colonies on these plates should be red after transformation. The colonies on the arabinose plate should be white as translation of fimE leads to the flipping fimS, and expression of RHlI.

The number of colonies on the plate that are white and red confirm inversion, this will show the percentage of colonies in the [ON] and [OFF] states. DNA sequencing will show inversion of the switch. As fimE is unidirectional over time the colonies should all become white on the plate containing arabinose. The plate containing glucose should repress leakage and the medium is supplemented by some percentage glucose. 

To ensure that AHL is produced a white colony is chosen, where the fimS section has flipped, and a red colony which has not yet inverted. These colonies are then co-cultured with a successfully independently tested reporter cell. This reporter cell detects AHL production and as a result GFP produced. This also shows that the reversed sequences for RHLI and B0034 are working as expected.</P><H2 class="FIM" style="text-align: left; clear: both"> Conclusions and Future Work </H2><H2 class="FIM" style="text-align: left"> References </H2></DIV><DIV class="tab-pane fade" id="nav-reporter" role="tabpanel" aria-labelledby="nav-reporter-tab">
        REPORT
      </DIV><DIV class="tab-pane fade" id="nav-framework" role="tabpanel" aria-labelledby="nav-framework-tab">
        FRAMEWORK
      </DIV><DIV class="tab-pane fade" id="nav-cellfree" role="tabpanel" aria-labelledby="nav-cellfree-tab">
        CELL-FREE
      </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>