Team:Newcastle/Improve
MENU
HOME
TEAM
Team
Collaborations
PROJECT
Description
Design
Experiments
Notebook
InterLab
Contribution
Model
Results
Demonstrate
Improve
Attributions
PARTS
Parts
Basic Parts
Composite Parts
Part Collection
SAFETY
HUMAN PRACTICES
Silver HP
Integrated and Gold
Public Engagement
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Plant
Software
JUDGING FORM
Show Navbar
spacefill
Project
Description Results Demonstration Protocols Parts Interlab Study
Modelling
Overview Simbiotics Design of Experiments Microfluidic Agent-based Model
Human Practices
Silver Gold and Integrated Education and Public Engagement Science Communication
Collaborations
Overview Edinburgh OG Exeter Evry Paris-Saclay
Judging
Medal Requirements Safety Measurement Applied Design Part Collection Improvement
Team
Meet the Team Attributions
Collapse Navbar
spacefill
Part Improvement
Fim Standby Switch
BioBricks made and used: BBa_K2205005 (New) , BBa_K1632013 (2015 Tokyo Tech part), BBa_K1632007(2015 Tokyo Tech part)
The Original Part and Our Improvements
The Tokyo Tech iGEM team (2015) standardised a 'flippable promoter' part, which produces GFP ( BBa_E0040 ) constitutively. When FimE (a transposase) is produced, the promoter driving expression of the GFP is 'cut' out of the construct and flipped, such that the promoter is now on the opposite strand. This means that when FimE is absent, GFP is produced. When FimE is present, GFP is not produced. While this part is useful as it provides a reporter which can be switch off, it has two major drawbacks. Firstly, the GFP variant used in this part has very low fluorescent intensity, and cannot be seen with the naked eye. This prevents its use in biosensors, where most applications are outside of a laboratory environment and away from specialist equipment which could be used to measure the expression of the GFP. To remedy this issue, we have replaced the E0040 GFP variant with a red chromoprotein ( eforRed ). This chromoprotein is easily visible to the naked eye, making it much more useful for applications outside of the laboratory.
A second issue with this original part is that it is only able to give a single output (GFP). When the promoter is flipped, nothing else is produced. This completely misses one of the most useful characteristics of a device which has a promoter that can be flipped: dual outputs. We have made use of this characteristic by encoding the rhlI gene on the antisense strand of the Fim switch upstream of the flippable promoter. This enables any additional output to be easily added by using the RhlIR two component system mechanism. As a proof-of-concept for this, we have included the Fim switch as a processing variant in our Sensynova Framework. The Fim switch is able to act as a 'standby switch' by expressing eforRed (red output) when no signal is detected (i.e. when no analyte is detected by the detector cell). When a signal is detected, expression of fimE can be activated, and the promoter is flipped, resulting in expression of RhlI. This leads to the synthesis of the C4-AHL quorum sensing molecule, which is sensed by the reporter cells present in the community and its reporter gene is produced. This is shown in Figure 1.
Figure 1: SBOL Visual of the switching mechanism of the Fim Switch, in the native [OFF] state the eforRED reporter is expressed (shown in red) allowing direct visualisation of the cells.  Following the inversion of the promoter region ( K1632004 ), eforRED expression is halted and the rhlI gene is expressed ( J64718 ), this is now the [ON] state.
Characterisation
To test the functionality of the Fim switch, ensuring that C4 AHL is produced, the strain was cultured with a reporter strain ( K2205015 ) which produces GFP in response to the quorum sensing molecule C4 AHL.  Due to a small sub-population of the Fim switch strain being white, a single white colony was picked and cultured separately.  This strain was used as a positive control as it should produce C4 AHL.  Both the majority (red) Fim switch strains and minority flipped (white) Fim switch strains were tested for C4 AHL production by co-culture with the reporter strain.  Initially the Fim switch strains were spotted onto a lawn of the reporter strain (Figure 2) followed by quantitative analysis of the strains by co-culture in a 96 well microplate (Figure 3).    This culture is then co-cultured with the reporter cell. This reporter cell detects C4 AHL and expresses GFP in response.
Figure 2: Initial test of the red and white fim switch strains which were spotted onto a lawn of the reporter strain ( BBa_K2205015 ).
Figure 3: Expression of GFP in the reporter ( BBa_K2205015 )strain in co-culture with the Fim switch strains.  The assay was performed using methods described in Fim 96 Plate assay Protocol .  The data shows the expression of GFP in the reporter strain over a standard growth curve.  The FimW and FimR strains represent the white and red variants of the Fim switch strain respectively, these were co-cultured with the reporter strain in a 1:14 ratio.  Each data point is the mean of 3 biological repeats.  RFU stands for relative fluorescence units.
Conclusions
The aim of the Fim switch part was to make a processor module which can be visually inspected for functionality.  The Fim switch has been shown to expresses the eforRed chromoprotein under normal (uninduced) conditions which allows the user to both determine that the strain is alive and has maintained the Fim switch plasmid.  Following induction, the Fim promoter flips direction and begins expressing RhlI which synthesises the C4-AHL quorum sensing molecule.  This has been shown to successfully induce expression of sfGFP in the reporter strain ( BBa_K2205015 ).
Despite several attempts we were unable to produce a Fim switch testing construct where fimE expression could be controlled using the E. coli arabinose inducible promoter.  Though transformations did yield some colonies when trying to make this part, none were red in colour.  This possibly indicates that the arabinose inducible promoter (even when grown on 0.5% w/v glucose) is still too active.  The design for this construct has been submitted ( BBa_K2205006 ).
Future Work
Since there is some leaky expression of the fimE gene (even without a promoter) to fine tune the Fim switch the lac operator could be inserted upstream of the fimE RBS to repress unwanted expression.  The part could then be used following the addition of IPTG.  An alternative method would be to clone a transcriptional terminator upstream of the fimE RBS to prevent leaky expression from elsewhere in the plasmid.
References
P. Klemm, Two regulatory fim genes, fimB and fimE, control the phase variation of type 1 fimbriae in Escherichia coli. EMBO J 5, 1389-1393 (1986).
M. S. McClain, I. C. Blomfield, B. I. Eisenstein, Roles of fimB and fimE in site-specific DNA inversion associated with phase variation of type 1 fimbriae in Escherichia coli. J Bacteriol 173, 5308-5314 (1991).
M. R. Parsek, E. P. Greenberg, Acyl-homoserine lactone quorum sensing in gram-negative bacteria: a signaling mechanism involved in associations with higher organisms. Proc Natl Acad Sci U S A 97, 8789-8793 (2000).
Promoter Probe Plasmid
BioBricks made and used: BBa_K2205026 (New) , BBa_J61002 (Arkin Lab, 2006)
The Original Part and Our Improvements
The Arkin Lab team (2006) produced the promoter probe plasmid ( BBa_J61002 ) for the testing and characterisation of the Anderson promoter library ( BBa_J23100 - BBa_J23119 ). When using this part to characterise the synthetic promoter library produced in this project, many issues concerning expression arose.
In order to combat these limitations, a new plasmid has been designed to improve the expression of the BBa_J61002 plasmid. Firstly, a bidirectional terminator has been inserted upstream of the iGEM cloning site in order to prevent any leaky expression from the plasmid backbone in which contains a promoter in the reverse direction.  In order to obtain the ability to repress the expression of the promoter being tested, a Lac Operator sequence has been inserted immediately downstream of the promoter insertion site.  The ribosome binding site BBa_B0034 which is present within the plasmid, is not flanked by any restriction site in order for the swapping out of RBSs. In order to give the plasmid this ability, two restriction sites SacI and BamHI were inserted flanking its sequence.  This is to allow for the RBS to be changed at will if the user wishes to screen a library of RBS as well as promoters.  Finally, the reporter gene currently present in the promoter probe plasmid mRFP, was been replaced with a sfGFP coding sequence. As the sfGFP coding sequence is a significantly shorter than its predecessor, therefore allowing for a faster transcription. This gene sequence also produces a brighter fluorescence than mRFP, with emission wavelengths of 509 nm, interference that can arise when using techniques such as plate readers from high cell density (wavelength of 600 nm) samples become less likely when compared to the emission wavelength of mRFP (584 nm).
Figure 1: Improved Promoter Probe Plasmid ( BBa_K2205026 ).
Characterisation
To test the functionality of the improved promoter plasmid probe, a sample of the synthetic promoter library produced in this project was assembled into the plasmid using BioBrick assembly. The resulting colonies were miniprepped and insertion was confirmed through sequencing.
Conclusions
Though the insertion of the promoters were successful into the promoter probe plasmid, due to time constraints, we lacked the time to further characterise this part into the Sensynova platform within the lab.
The new improved promoter probe plasmid, part BBa_K2205026 was submitted to the registry for further work and characterisation by future teams.
