<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BCCS-Bristol_Team_Brainstorm"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:BCCS-Bristol/Team/Brainstorm">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BCCS-Bristol/Team/Brainstorm&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BCCS-Bristol/Team/Brainstorm&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BCCS-Bristol/Team/Brainstorm&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BCCS-Bristol/Team/Brainstorm" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BCCS-Bristol/Team/Brainstorm</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><UL id="nav"><LI><A href="https://2009.igem.org/Team:BCCS-Bristol">Home</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Project">VESECURE</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Project/Wet_lab">Wet lab</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Project/Wet_lab/BioBricks">BioBricks</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Project/Wet_lab/Notebook">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Project/Wet_lab/Lab_photos">Lab photos</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Project/Wet_lab#Safety_Issues">Safety</A></LI></UL><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Project/Modelling">Modelling</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/BSim">BSim</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/BSim#Features">Features</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/BSim/Tutorials">Tutorials</A></LI><LI><A href="http://code.google.com/p/bsim-bccs/downloads">Download</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/BSim/Case_studies">Case studies</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/BSim/Case_studies/VESECURE">VESECURE</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/BSim/Case_studies/Magnetotaxis">Magnetotaxis</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/BSim/Case_studies/Repressilators">Repressilators</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Bioscaffold">Bioscaffold</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Bioscaffold/Protein_Fusion_Problem">Fusion Problem</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Bioscaffold/BCCS_Bioscaffold">BCCS Bioscaffold</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Team">Team</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Team/Calendar">Calendar</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Team/Brainstorm">Brainstorm</A></LI><LI><A href="https://2009.igem.org/Team:BCCS-Bristol/Team/Preliminary_meetings">Preliminary meetings</A></LI></P><DIV style="position: relative; width: 965px; height: 150px; background: url('https://static.igem.org/mediawiki/2009/c/c2/BCCSBanner.jpg'); clear: both;  "><DIV style="position: absolute; bottom: 12px; right: 25px; color: black; z-index: 3; font-size: 1.5em">
BCCS-Bristol
<SPAN style="font-size: small">iGEM 2009</SPAN></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Magnetosomes"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Magnetosomes</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Bacterial_versioned_upgrades"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Bacterial versioned upgrades</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Vesicle_Secretion"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Vesicle Secretion</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#RNA_Editing_.26_Reverse_Trascription_.28Turing_machine.29"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">RNA Editing &amp; Reverse Trascription (Turing machine)</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#RNA_Editing"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">RNA Editing</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Reverse_Trascription"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Reverse Trascription</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Synthetic_Proteins_.28Dek.27s_Lab.29"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Synthetic Proteins (Dek's Lab)</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Viral_Coat_Modifications"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Viral Coat Modifications</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Actin_Modification.2FMovement"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Actin Modification/Movement</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Improving.2FCompleting_Previous_iGEM_Networks"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Improving/Completing Previous iGEM Networks</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Multi-Functional_Networks"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Multi-Functional Networks</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Switches.2FSensors"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Switches/Sensors</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Symbiosis_-_Host.2FParasite_Symbiosis"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Symbiosis - Host/Parasite Symbiosis</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Bacterial_computation"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Bacterial computation</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Error_Correction"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Error Correction</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Simulated_Economy"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Simulated Economy</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Modelling_Projects"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Modelling Projects</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#GRN_Modelling"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">GRN Modelling</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#BSim_Version_2.0"><SPAN class="tocnumber">15.2</SPAN><SPAN class="toctext">BSim Version 2.0</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Diffusion_Modelling"><SPAN class="tocnumber">15.3</SPAN><SPAN class="toctext">Diffusion Modelling</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Statistical_Mechanics"><SPAN class="tocnumber">15.4</SPAN><SPAN class="toctext">Statistical Mechanics</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Concurrent_Discrete_Process_Modelling"><SPAN class="tocnumber">15.5</SPAN><SPAN class="toctext">Concurrent Discrete Process Modelling</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Petri_Nets"><SPAN class="tocnumber">15.6</SPAN><SPAN class="toctext">Petri Nets</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#P_Systems_to_Understand_Localisation"><SPAN class="tocnumber">15.7</SPAN><SPAN class="toctext">P Systems to Understand Localisation</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Novel_Visualisation_of_Time_Series_or_Agent_Based_Models"><SPAN class="tocnumber">15.8</SPAN><SPAN class="toctext">Novel Visualisation of Time Series or Agent Based Models</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Magnetosomes"> Magnetosomes </SPAN></H2><P>Some prokaryotic organisms have membrane bound organelles within their cell which contain single crystal ferromagnetic material known as magnetite (Fe3O4). If the gene can be manipulated and expressed in E.Coli then the possibilities of applications for single crystal magnetic material are endless ranging from medicine to the industry. Many species have magnetic properties but the most studied are from prokaryotic organisms of aquatic nature and are of the genus of magnetotacticum.
</P><H2><SPAN class="mw-headline" id="Bacterial_versioned_upgrades"> Bacterial versioned upgrades </SPAN></H2><P>Have the ability for bacteria to naturally propogate updates to syntheic GRNs that need updating.
</P><H2><SPAN class="mw-headline" id="Vesicle_Secretion"> Vesicle Secretion </SPAN></H2><P>Hormone signalling and plasmids have limited capabilities in communication and signalling.
Hormone - diffusive and not fast and limited radius of communication.
</P><P>By packing chemicals in vesicles u can extend the radius of communication and increase signalling. Also you can design exclusive communication schemes by specialised recognition from bacteria.
</P><H2><SPAN class="mw-headline" id="RNA_Editing_.26_Reverse_Trascription_.28Turing_machine.29"> RNA Editing &amp; Reverse Trascription (Turing machine) </SPAN></H2><H3><SPAN class="mw-headline" id="RNA_Editing"> RNA Editing </SPAN></H3><P>Creating a similar editing that occurs in eukaryotic cells where splicing can give different permutations of a protein (or in this case different proteins). 
This can be implemented by testing the functionality of current eukaryotic proteins in bacteria or by implementing a scheme from scratch that does a similar function (restriction enzyme usage?) 
</P><P>This can then self-insert into the genome at desired locations by homologous recombination. 
</P><H3><SPAN class="mw-headline" id="Reverse_Trascription"> Reverse Trascription </SPAN></H3><P>Attempt to translate the principlies of Turing Machines in bacterial organisms to allow for the creation of an adaptive genome using both RNA Editing and Reverse Transcription. This can be affected by environmental factors. Also this is a useful mechanism for reducing basal transcription levels in gene networks.
</P><H2><SPAN class="mw-headline" id="Synthetic_Proteins_.28Dek.27s_Lab.29"> Synthetic Proteins (Dek's Lab) </SPAN></H2><P>Using completely lab designed proteins and implementing them along with natural proteins found in bacterial cells to carry out novel functions.
</P><H2><SPAN class="mw-headline" id="Viral_Coat_Modifications"> Viral Coat Modifications </SPAN></H2><P>Viruses assembly their protein coats by using certain geometric monomers (e.g triangular) of proteins to create different shapes (dodecahedrons,icosa-hedrons). One can model and exploit the thermodynamics and geometry to create novel shapes or extend the volume of the protein coat to allow for extended capacity in terms of DNA or proteins/biologics.
</P><P>VERY VERY VERy VEry Very very hardÂ !
</P><H2><SPAN class="mw-headline" id="Actin_Modification.2FMovement"> Actin Modification/Movement </SPAN></H2><P>Try to mimick the actin polymerisation and cytoskeletal arrangements of eukaryotic cells into bacteria to try and promote movement into specific directions or affect the shape of bacterial cells.
</P><H2><SPAN class="mw-headline" id="Improving.2FCompleting_Previous_iGEM_Networks"> Improving/Completing Previous iGEM Networks </SPAN></H2><P>Simply taking previous iGEM networks that almost work but do not, due to the team running out of time etc, and just finishing them off, ideally in as automated fashion as possible, or under some general technique of completing networks i.e. you try to see what the main reasons where for the networks not functioning correctly, and then concentrate on the the biggest group, and come up with a general method of fixing that problem. good points are, most of the lab work is done, and potentially allows a large number of new networks to be created.  
</P><H2><SPAN class="mw-headline" id="Multi-Functional_Networks">  Multi-Functional Networks </SPAN></H2><P>This could be done in a number of possible ways:
</P><P>!Inherent Multi-functionality
Creating networks that have two or more distinct useful behaviours/functions dependent on the input into the network. 
</P><P>!Exploiting Stochastic Coherence
Network dynamics can vary depending on the number of copies of the network that are present in the cell. Therefore it is should be possible to create networks whose function changes depending on the nunmber of copies that are present. 
</P><P>!Controlling Dynamics Through Temperature - Arrenhius Relationship for Rate Constants. 
Rate constants are dependent on temperature through the Arrenhius equation. Therefore it may be possible to design networks whos function depends on the temperature of the network environment. Numerous applications here. 
</P><P>!Concentration Modification. 
In a mass action formalism of chemical reaction there are rate constants and concentrations. Therefore in addition to modifying rate constants through temperature, one could modify the behavior/function by modifying the concentration of the network components. That is, you get the network doing one function with a certain constant input, then maintaining that input, you switch on a gene which modifys the level of a network component(s), which then switches the behaviour/function of the network. In essence, having two functions for one input. 
</P><H2><SPAN class="mw-headline" id="Switches.2FSensors"> Switches/Sensors </SPAN></H2><P>Potential use as a non-invasive toxin detection mechanism for organisms. Put a number of existing detection networks into one bacteria, which you then pass through an organism, and analyse them for the presence of the toxin(s) in the organism. 
</P><H2><SPAN class="mw-headline" id="Symbiosis_-_Host.2FParasite_Symbiosis"> Symbiosis - Host/Parasite Symbiosis </SPAN></H2><P>Find two toxins, one which bacteria sense better than yeast and another that yeast sense better than bacteria, but both which they have a response that protects themselves again the toxin. Then create an interface by whch the bacteria inform the yeast when they detect the toxin theyu are more sensitive to, and the yeast inform the bacteria when they sense the toxin that they are more sensitive to. 
</P><H2><SPAN class="mw-headline" id="Bacterial_computation">  Bacterial computation </SPAN></H2><P>Can we come up with a simple problem, like the ones solved by the Davison team, to be solved by our bacteria? 
</P><P>Perhaps the <A href="http://en.wikipedia.org/wiki/Majority_problem_(cellular_automaton)" class="external text" rel="nofollow">Majority problem</A>?
</P><UL><LI><A href="https://2006.igem.org/Davidson_2006" class="external text" rel="nofollow">Davison 2006 Burnt Pancake</A>, <A href="http://www.jbioleng.org/content/pdf/1754-1611-2-8.pdf" class="external text" rel="nofollow">Burnt pancake journal paper</A></LI><LI><A href="https://2007.igem.org/Davidson_Missouri_W" class="external text" rel="nofollow">Davison 2007 Hamiltonian Path</A>, <A href="http://openwetware.org/images/3/3a/DMW_IBE_Presentation.pdf" class="external text" rel="nofollow">Presentation</A></LI><LI><A href="https://2008.igem.org/Team:Davidson-Missouri_Western" class="external text" rel="nofollow">Davison 2008 hash function</A></LI></UL><UL><LI><A href="http://medgadget.com/archives/2007/07/scientists_create_biomolecular_computing_device_with_bacterial_phenotype_output.html" class="external text" rel="nofollow">A Programmable Biomolecular Computing Machine with Bacterial Phenotype Output, J. ChemBioChem</A></LI></UL><P>Writings on bacterial computation by Martyn Amos:
</P><UL><LI><A href="http://www.docm.mmu.ac.uk/STAFF/M.Amos/Docs/baccomp.pdf" class="external text" rel="nofollow">Bacterial computing in the Encyclopedia of Complexity and System Science</A></LI><LI><A href="http://arxiv.org/PS_cache/q-bio/pdf/0512/0512017v1.pdf" class="external text" rel="nofollow">Bacterial self-organisation and computation</A></LI><LI><A href="http://www.cs.york.ac.uk/nature/workshop/papers/Amos.pdf" class="external text" rel="nofollow">Bacterial Self-Assembly and Computation</A></LI></UL><H2><SPAN class="mw-headline" id="Error_Correction">  Error Correction </SPAN></H2><P>Would it be possible for a population of bacteria to always maintain specific proportions that are in different states.
</P><H2><SPAN class="mw-headline" id="Simulated_Economy"> Simulated Economy </SPAN></H2><P>Using a population of bacteria as a way to simulate the highly distributed economy.
</P><H2><SPAN class="mw-headline" id="Modelling_Projects"> Modelling Projects </SPAN></H2><P>Although modelling will form a vital part of understanding how the wetlab GRN will behave it would be interesting to carry out some additional modelling into some novel area that has not previously been considered. This will then allow us to add to the toolbox of methods we are developing to help the sythetic biological engineer in the design process. Need to make sure this is aligned to the interests and abilities of the team to ensure the best possible outcome is achieved. 
</P><H3><SPAN class="mw-headline" id="GRN_Modelling"> GRN Modelling </SPAN></H3><P>As a GRN is likely to form the biggest part of the project we will need to carry out some modelling of its dynamics. This could include ODE models, bifurcation analysis, stochastic simulation or other novel techniques we find.
</P><H3><SPAN class="mw-headline" id="BSim_Version_2.0"> BSim Version 2.0 </SPAN></H3><P>Continuing on for the work done in 2008 there are several extensions to BSim that could be performed to improve accuracy and performance on large scale simulations. These include:
</P><UL><LI> Incorporate GRN Dynamics - Currently the state of a cell is determined using very simple discrete rules. It would be possbile to incorporate ODE type GRN models to give detailed information about the protein/mRNA concentrations over time for each cell. One possible method would be to create a plugin for MATLAB that would allow for a model to be accessed by BSim and for it to provide environmental information such as signalling chemical concentrations. Alternatively, a simple ODE solver could be written in Java and the models transferred directly to BSim which would remove any dependancy on having MATLAB available and may work better for running on high performance clusters.
</LI></UL><UL><LI> Population Dynamics - Bacteria in BSim version 1 are immortal! Unfortunately in real life a bacterium version of the &quot;highlander&quot; has yet to be found and so the dynamics of population growth and death can be important. Adding population dynamics could be achieved through simple rules for bacteria duplication or more advanced methods based on nutrient distribution and use could be sought. This is probably the easiest out of the extensions to implement as it is merely creating new elements during the simulations which is very simple.
</LI></UL><UL><LI> Large Scale Computations - Multi-threading is used by BSim to help improve the performance of a simulation running on a single computing node. This does not, however, provide any advantage when the available computer resources are in a cluster form. In 2008 this did not matter as we wanted to run large numbers of simulations to gain statistical results. If instead you wanted to run on humoungous simulation with millions of bacteria it would run relatively slowly. One extension that would allow for this limitation to be removed would be including support for message passing computing architectures such as <A href="http://en.wikipedia.org/wiki/Message_Passing_Interface" class="external text" rel="nofollow">MPI</A>. These allow for parts of the program to run on physically separated machines that are co-ordinated by messages sent between each of the machines during execution. There are Java libraries which give access to this technology, however, this is a fairly large undertaking as parallel computation has a lot of gotchas and some large changes would be required in BSim to fully support this. If undertaken it would then allow in theory for simulations will millions of bacteria to run a reasonable time (e.g. days not years!) but if the project is not concerned with populations of bacteria it may not be that useful to support the wetlab project. 
</LI></UL><P>Considering that the code is in good shape the effort to implement at least one of these extensions would be fairly simple and allow for a lot of previous effort to be utilised. It also should allow for yet more pretty videos to be generated which always go down well in presentations.
</P><H3><SPAN class="mw-headline" id="Diffusion_Modelling"> Diffusion Modelling </SPAN></H3><P>Before synthesising a gene network in the lab, one creates a computer model of the network in order to assess whether the network is likely to function when synthesised, and for what parameter values (transcription rates, translation rates etc etc) the network is most likely to function for. However, it is commonly accepted that current models of gene networks are quite poor at predicting the dynamics of the real network. This is of course because the model is a necessary simplification of the real network, and one cannot represent or even know all the features of the real system. The logical (and current) approach is to start with as simple model as possible, and then add in detail when the experimental results from the synthesised network differ significantly from the modelling results, suggesting that there is a discrepancy between the model representation and the real situation that leads to the model giving prediction of the dynamics that are qualitatively or quantitatively incorrect to a degree that makes the model practically useless. This project would, based on knowledge from those who understand the real systems well (Claire, Nigel etc), involve picking a process of the real system that is common to all GRNs, that which an increase in the accuracy of the representation of that process in the model would be predicted to lead to the greatest increase in correlation between model and experimental dynamics. 
</P><P>One possibility is the representation of the diffusion of network components within the cell. That is, at the moment there appears to have been no consideration of whether an accurate representation of the diffusion of the networks and the transcription factors of the GRN can significantly increase the predictive power of a GRN mode. One approach would involve creating a sophisticated model of protein and plasmid diffusion within E.coli, that could be combined with a standard ODE model of the rest of the network components. However, this may be computationally intractable for large networks, so another approach may be to attempt to create, based on a simulations and theory, a way of estimating the delay parameter between production of a transcription factor and its reaching its target. This will of course be a distribution, but it is the mean that would be required. Currently delay parameters are used in some instances, although these are free parameters that are optimised to match the network dynamics with what is seen experimentally. If we could go from first principles, and give some a priori values for these parameters it could either speed up the optimisation process, or remove the need for it all together.
</P><P>This would be attempted for a very simple network comprising two genes; One that produced a transcription factor in response to an external stimulus that we can control, and another that produces a GFP reporter in response to this transcription factor. Experimentally, we would need to be able to measure, on a single-cell level, the time between the addition of the stimulus, and the production of the GFP (one copy would be fine). If this could be done one a large enough scale, then a distribution of `stimulus addition to GFP production' times could be obtained.
</P><P>
We would hope that if we compared a standard ODE model with delays (that we optimise in the standard fashion), to our ODE-Diffusion hybrid or time-delay estimation technique, we would see an increase in terms of the correlation between the predictions of the distribution of times of the model, and the distribution of times in the experiment, when we use our model relative to the standard optimisation process. 
</P><P>The issue with this is that as the network is so simple, the optimisation process has a big advantage i.e. on a simple network it should be easy to find parameters that match the experimental data as we only have one link. The problem with optimisation comes when there are a large number of time-delay parameters to tune, and is where the approach outlined here may come in useful, although this would require a larger network that would be much harder to optimise on. This is possibly feasible if a network is used that has been created elsewhere. 
</P><P>Anyway, diffusion modelling is only one possible process that when modelled far more accurately could provide significant accuracy/predictive power increases. Others have not yet been considered. 
</P><H3><SPAN class="mw-headline" id="Statistical_Mechanics"> Statistical Mechanics </SPAN></H3><P>Again coming from the complexity perspective, are there any aspects for synthetic biology that lend themselves to modelling through statistical mechanics. There are a few papers that have used it to look a bacteria movement (chemotaxis), however, are there intercellular processes that may be amenable to this approach?
</P><H3><SPAN class="mw-headline" id="Concurrent_Discrete_Process_Modelling"> Concurrent Discrete Process Modelling </SPAN></H3><H3><SPAN class="mw-headline" id="Petri_Nets"> Petri Nets </SPAN></H3><H3><SPAN class="mw-headline" id="P_Systems_to_Understand_Localisation"> P Systems to Understand Localisation </SPAN></H3><H3><SPAN class="mw-headline" id="Novel_Visualisation_of_Time_Series_or_Agent_Based_Models"> Novel Visualisation of Time Series or Agent Based Models </SPAN></H3><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:BCCS-Bristol/Team/Brainstorm">http://2009.igem.org/Team:BCCS-Bristol/Team/Brainstorm</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BCCS-Bristol/Team/Brainstorm" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BCCS-Bristol/Team/Brainstorm" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BCCS-Bristol/Team/Brainstorm&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BCCS-Bristol/Team/Brainstorm&amp;oldid=135534" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>