"
Page
Discussion
View source
History
teams
Log in
 
Team:BCCS-Bristol/Project/Wet lab/Notebook
From 2009.igem.org
Home
VESECURE
Wet lab
BioBricks
Notebook
Lab photos
Safety
Modelling
BSim
Features
Tutorials
Download
Case studies
VESECURE
Magnetotaxis
Repressilators
Bioscaffold
Fusion Problem
BCCS Bioscaffold
Team
Calendar
Brainstorm
Preliminary meetings
BCCS-Bristol
iGEM 2009
Please select a week from below to view our weekly lab progress and methods used in developing VESECURE in the wet-lab.
Contents
1 Week 0
2 Week 1
2.1 Canditate Proteins for Biobricks
2.2 Reporters,RBS,Backbones
2.2.1 Reporters
2.2.2 RBS
2.2.3 Plasmid Backbone
3 Week 2
3.1 PCR
3.2 In frame protein fusions
4 Week 3
5 Week 4
6 Week 5
6.1 Alternative Assembly Method
7 Week 6
8 Week 7
9 Week 8
9.1 Promoter-RBS-GFP-terminator ligation
10 Week 9
10.1 Constructing a simple fusion to test the promoter
10.2 Alternative Assembly method (a.k.a. Quick and Dirty)
10.3 Bioscaffold Assemblies
10.4 Alternative Test for the Bioscaffold
11 Week 10
11.1 Bioscaffold Tests
11.2 Alternative Assembly Constructs
12 Week 11
12.1 Alternative Assembly Constructs
12.2 Bioscaffold Tests
13 Week 12
Week 0
Familiarising with standard lab procedures for the past week (bacterial culture growth, restriction enzyme usage, agarose gel electrophoresis).
Will start designing some biobricks for the project today.
Week 1
Canditate Proteins for Biobricks
Isolated 3 canditate proteins to act as carriers for our biobricks. These are FhuA,Fiu & OsmE. Started to design primers to amplify the selected genes via PCR.
Primers designed and ordered. Waiting for their arrival to do PCR! :D
Reporters,RBS,Backbones
Decided to use 3 reporter genes, 1 RBS, 1 High Copy plasmid backbone for now.
Reporters
*RFP(Bba_E1010) *GFP(Bba_E1040) *LacZ(Bba_I732005)
RBS
*Bba_J61100 - From Anderson Family
Plasmid Backbone
*BBa_J04450 ; pSB1A3
Tried to extract DNA from the iGEM biobricks and transfrom into bacteria.
Transformations do not work properly with non-commercial E.coli strain (XL-1).
Transformations worked the 2nd time round with commercial Nova Blue E.coli Strain. DNA samples in toolkit must be of low concentrations!
Regrew bacterial colonies to amplify DNA of reporters,RBS,backbone.
Miniprepped the DNA of Reporters,RBS,plasmid backbone and made glycerol stocks.
Realised that we are faced with a problem when wanting to assemble biobricks for protein fusions.
Week 2
PCR
Primers finally arrived. Did PCR to amplify carrier genes.
PCR worked. PCR products (3 candidate proteins)ligated onto biobrick backbones pSB1A3 and pSB1A2 (contains RBS BBa_J61100).
The plasmid backbones with the genes of interest inserted into them were used to transform the XL1-BLUE E.coli strain (although not competent enough compared to NovaBlue cells they are much cheaper!!)
Most transformations are successful. Used transformed colonies to prepare liquid cultures so that we can proceed with minipreping them.
In frame protein fusions
Started working on finding an easy assembly method for in-line protein fusions.
Developed the design for a Bioscaffold-Linker transformer family (inspired by Bioscaffolds). Should allow fusions of proteins and all RFC10 biobricks in-frame after using Bioscaffold specific restriction enzymes.
Prepared different versions of this Bioscaffold to be ordered and tested in the lab for actual functionality.
Novel protein additions were cloned from the MG1655 genome with PCRt. Lane 1 shows the presence of fhuA, lane 2 fiu and lane 3 osmE. The ladder is Lambda-DNA HindIII digest from NEB (In descending order; 23130, 9416, 6557, 4361,2322, 2027. Carrier proteins were then placed onto pSB1A3 plasmid backbones to create biobricks novel protein carriers.
Week 3
Ligation successful only for the OsmE and FhuA inserts on pSb1A3 backbone. Unfortunately we couldn't get any of the 3 proteins on the pSB1A2 backbone.
Seems that overxpression of these proteins on pSB1A2 is unfavourable for the cells. Hence decided to replace the existing promoter with an inducible one of our choice such  as the AraC promoter.
Promoter removed from pSB1A2 and religated. Fiu still not able to be ligated on any of the two backbones.
In the meantime the two parts comprising of the AraC promoter were extracted from the iGEM kit plates and used to transform NovaBlue competent cells. Also liquid cultures of the transformed cells were prepared.
Finalised designs for the Bioscaffold-Linker biobrick and ordered from GeneART/Mr.Gene the construct.
Finding canditate primers for sequencing the first 3 carrier biobrick proteins (fhuA/osmE/fiu).
Started thinking of a quick and dirty in-frame fusion for testing functional carrier-reporter gene fusions.
Found easy fusion way for in line testing. Will take out end of FhuA and scar formed after RFC10 assembly of FhuA-GFP using RE's to take out FhuA end ( including TAA TAA SCAR) and part of GFP start. Will replace lost coding sequences with PCR.
Ordered primers for the quick-n-dirty assembly method.
Week 4
One more try to get Fiu protein on pSB1A3 backbone. NovaBlue transformed with ligation products of pSB1A3 and Fiu.
Grew colonies for FhuA and OsmE midi preps.
Carried out midi preps for FhuA and OsmE ( [FhuA]=378.9 ng/ul, [OsmE]=113.2 ng/ul).
Transformed Nova Blue with biobrick part BBa_B0014 (double terminator)
Culture colonies of BBa_B0014 transformants to get ready for mini prep.
PCR FhuA and GFP for assembling together a quick and dirty fusion to be ready prior to BioScaffold arrival.
Carried out colony growth (agar & liquid) for the AraC promoter (2 parts BBa_R0080 (lacking O2 region) and BBa_R0081 (the O2 region).
Miniprepped DNA from cultures of BBa_R0080 and R0081.
Attempted a 2-way ligation and of the BBa_R0080 and R0081 parts. The final product should be a 2-component biobrick on a plasmid in 5'-R0081-R0080-3' direction which can be transformed into cells.
R0081 part was purified by gel extraction after being treated by double digest. R0080 and R0081 were ligated together and used to transform NovaBlue cells.
First attempt to make the AraC promoter from the two parts failed.
Fiu still not being ligated on ANY backbone.
Standard 1kb NEB Ladder. Lanes 1-2 FhuA Midipreps, Lanes3-4 OsmE midipreps.
Standard 1kb NEB Ladder. Lanes 1-2 FhuA PCR fragment for Quick and Dirty assembly Method, Lanes3-4 GFP PCR fragments for quick and dirty assembly method.
The figure indicates the presence of the AraC promoter subunits. Lanes 1-6 show the unsuccesful ligation of the Fiu Carrier protein. Lanes 7-9 illustrate Double Digest with XbaI and SpeI,Single Digest with XbaI and Uncut versions of the plasmid carrying R0080. Similarly lanes 10-12 show the same for R0081. The promoter fragments are 149bp and 183bp respectively and do not appear on the gel but the presence is illustrated by the band shift between the Double and Single Digests.
Week 5
The double terminator BBa_B0014 was taken out of the iGEM kit plates and used to transform NovaBlue cells.
Minipreps and restriction digests were performed for terminator cultures (B0014) to check that they have the correct insert.
Ligations set up between R0080 (on pSB1A2) and R0081 and used to transform NovaBlue competent cells.
Alternative Assembly Method
In this method we will use standard non-iGEM-procedures to fuse FhuA and GFP. This construct will be placed downstream of an AraC-RBS fusion and upstream of a terminator. The final assembly product will be RFC10 compliant.
The aim is to check whether a cargo protein like GFP will be transported to vesicles by one of our carrier proteins of interest like FhuA. It should be completed more quickly than the other method(at least in principle).
It can also be used as a back up in case the bioscaffold method does not work.
In general  a PCR-amplified GFP fragment will be inserted in a pQE31 plasmid. Then a PCR-amplified FhuA fragment will be inserted upstream of the GFP.
The whole construct will be cut and ligated upstream of an already made AraC-RBS fusion and at the end a terminator will be inserted upstream.
Performed restriction digests of pQE31 and GFP PCR product.
Make liquid cultures of NovaBlue transformed with complete AraC promoter.
PCR reactions to amplify FhuA, GFP and LacZ. GFP, FhuA and pQE31 are cut with appropriate enzymes for subsequent reactions and then purified.
LacZ reactions didn't work. We will proceed only with GFP.
Minipreps for colonies with Arac promoter.
Ligations set up between:
1. AraC and RBS 2. pQE31 and GFP
1kb NEB Standard Ladder. Lanes run in triplets featuring Double Digest (XbaI/SpeI), Single Digest(XbaI) and uncut versions of DNA plasmid carrying the AraC promoter. The promoter is seen as a single band at the bottom of the Double Digest lanes. The top band is the pSB1A2 Carrier plasmid.
1kb NEB Standard Ladder. Lanes run in triplets featuring  uncut versions of DNA plasmid carrying the Terminator B0014, Single Digest(XbaI) and Double Digest (XbaI/SpeI). The terminator is seen as a single band at the bottom of the Double Digest lanes. The top band is the pSB1A2 Carrier plasmid.
Week 6
XL1-Blue transformations with ligations 1 and 2 from Week 5.
Restriction digests for FhuA and OsmE to prepare their insertion downstream of the AraC-RBS costruct. Also GFP cut to fuse it with BOO14 terminator.
Ligation set up between GFP and B0014 terminator (on pSB1AK3) and used to transform XL1-Blue.
Transformations with AraC-RBS and pQE31-GFP didn't work. Repeated with NovaBlue cells.
AraC-RBS transformations worked very well in contrast with pQE31-GFP and GFP-terminator that didn't work again.
Repeat ligations between pQE31-GFP and GFP-terminator and transform XL1-Blue.
Minipreps and Restr. Digests for AraC-RBS constructs.
pQE31-GFP and GFP-terminator transformations failed.
Ligations set up:
1. pQE31+GFP 2. AraC-RBS + FhuA 3. AraC-RBS + OsmE 4. GFP-terminator
1kb NEB Standard Ladder. Lanes run in triplets featuring Double Digest (XbaI/SpeI), Single Digest(XbaI) and uncut versions of DNA plasmid carrying the AraC-RBS ligation product. The product is seen as a single band at the bottom of the Double Digest lanes. The top band is the pSB1A2 Carrier plasmid.
Week 7
Transform XL1-Blue with the 4 ligations (pQE31+GFP, AraC-RBS + FhuA, AraC-RBS + OsmE, GFP-terminator).
Transformations with AraC-RBS + FhuA/OsmE worked but NOT for pQE31-GFP and GFP-terminator.Repeat transformations for the ones that didn't worked.
Trasformation for GFP-Terminator worked and liquid cultures were prepared.
pQE31+GFP trtansformation didn't worked. Fresh stock of pQE31 was cut open, phosphatase treated and purified. PCR set up to amplify new stocks of GFP to be ligated to the new stock of pQE31. Ligations between pQE31 and GFP with varying ratios of plasmid vector to GFP insert were carried out.
i.e. Plasmid: Insert
1:1              1:2              1:3              1:4              1:5
pSB1A2 plasmids containing AraC-RBS-FhuA and AraC-RBS-OsmE were extracted using minipreps and later were cut with restriction enzymes and phosphatase treated. The first part of the conventional method (with bioscaffold) is finally completed.
Minipreps were performed for the GFP-terminator-containing cultures and the plasmid on which they are found was digested with restriction enzymes to allow the incorporation of the bioscaffold, which is the next step.
XL1-Blues were transformed with the pQE31+GFP ligations with the varying ratios of vector to insert. Moreover the bioscaffold arrived and its DNA was used to transform XL1-Blues.
Both of the above transformations worked, liquid cultures were set up and used to carry out minipreps.
The 4 different bioscaffold versions were cut with restriction enzymes and isolated using the gel extraction protocol. At the end of the day ligations between the cut GFP-terminator and the 4 bioscaffold versions were set up.
Ladder is 1kb DNA marker. Lanes run in triplets running in Double Digest (XbaI/OsmE), Single Digest(XbaI) and Uncut versions of the DNA plasmid carrying the AraC-RBS-FhuA construct.
Ladder is 1kb DNA marker. Lanes run in triplets running in Double Digest (XbaI/OsmE), Single Digest(XbaI) and Uncut versions of the DNA plasmid carrying the AraC-RBS-OsmE construct.
Ladder is 1kb DNA marker. Lanes run in triplets running in Double Digest (XbaI/OsmE), Single Digest(XbaI) and Uncut versions of the DNA plasmid carrying the GFP terminator construct.
Ladder is 1kb DNA marker. Lanes run in triplets running in Double Digest (XbaI/OsmE), Single Digest(XbaI) and Uncut versions of the DNA plasmid carrying the different Bioscaffold Versions.
Week 8
pQE31-GFP was digested and ligations with digested and purified FhuA were performed.
XL1-Blue competent cells were transformed with the pQE31-GFP+FhuA ligations.
The concentration of the DNA parts that we are going to send for sequencing was determined by spectrophotometry. These parts include: AraC-RBS, AraC-RBS-FhuA/OsmE, GFP-terminator and bioscaffolds-GFP-terminator.
Liquid cultures of the bioscaffolds-GFP-terminator were prepared and minipreps performed to isolated the plasmid DNA. Then the whole bioscaffold-GFP-terminator constructs were purified by gel extraction and ligations with the AraC-RBS-FhuA/OsmE were prepared.
Overnight ligations set-up;
1) AraC-RBS-FhuA-Bioscaffold-GFP-Terminator 2) AraC-RBS-OsmE-Bioscaffold-GFP-Terminator
Prepare liquid cultures for pQE31-GFP-FhuA.
Promoter-RBS-GFP-terminator ligation
This will be used to check that GFP works and also to check if the AraC promoter works by trying to control its level of activity using arabinose.
Restriction digest and gel purify GFP-terminator part and then ligate it to the already cut AraC-RBS on pSB1A2 plasmid.
The figure illustrates the construct AraC-RBS-GFP-TErminator created to assess promoter activity. The ladder is 1kb DNA marker from NEB. Lanes run in triplets with Double Digest (XbaI/Osme), Single Digest(XbaI) and Uncut versions of DNA plasmids carrying the construct. Presence is illustrated by the bands created in the double digest lane (top:plasmid backbone bottom:construct)
Week 9
Sequencing results indicate that our fusions were corrent and we are working with the correct proteins/DNA parts.
Constructing a simple fusion to test the promoter
XL1-Blue compet<math>Insert formula here</math>ent cells transformed with plasmid containing AraC-RBS-GFP-Terminator insert.
Alternative Assembly method (a.k.a. Quick and Dirty)
Minipreps of cultures of cells containing pQE31 with FhuA-GFP insert, restriction digests and gel electrophoresis were performed to certify that the insert of interest is there.
FhuA-GFP insert was then cut and isolated using gel extraction and ligations were prepared with open pSB1A2 containing the AraC-RBS.
Tranformations of XL1-Blue with pSB1A2 containing AraC-RBS-FhuA-GFP (ligations) were done. Moreover liquid cultures of cells containing the AraC-RBS-GFP-termninator insert were prepared.
DNA from the AraC-RBS-FhuA-GFP liquid cultures was miniprepped and assayed with restriction enzymes to test for the presence of the construct.
Bioscaffold Assemblies
* AraC-RBS-FhuA-Bioscaffold-GFP-Terminator * AraC-RBS-OsmE-Bioscaffold-GFP-Terminator
The constructs above were used to transform XL1-Blue cells.
Transformations of the Bioscaffold assemblies above were successful. Restrikes and liquid cultures were set up.
Restrikes and liquid cultures were successful. DNA was miniprepped from the liquid cultures and assayed with restriction enzymes to test for the presence of the entire construct. Results showed that the construct was successfully incorporated in the plasmids.
Site directed mutagenesis needs to be done to mutate an EcoRI site in OsmE and a BpuEI site in GFP. The former in order to make OsmE RFC10 compatible and the latter in order to be able to use the Bioscaffold.
Primers for site-directed mutagenesis were designed and ordered.
Alternative Test for the Bioscaffold
To be able to characterize the action of the Bioscaffold a simple construct of two proteins would be created with the Bioscaffold in the middle and the relevant assembly taking place.
For this purpose we are going to try and use again LacZ reporter gene to be the protein downstream of the Bioscaffold. If the process works then the colonies would appear blue in X-Gal plated agar plates.
Week 10
Bioscaffold Tests
The LacZ reporter gene seems not to be cut by XbaI enzyme and hence cannot be used in the Alternative Bioscaffold tests.
To replace LacZ we decided to use RFP (E1010) as a downstream protein for the Bioscaffold alternative tests. Ligations and transformations were set up for the 4 different bioscaffold versions, upstream of RFP on the pSB2k3 plasmid backbone. Only A1(CspCI st.) and A2(BseRI) were succesful.
Site-Directed mutagenesis was set up to mutate the illegal EcoRI site in OsmE and the Bioscaffold illegal site in GFP. This was performed only on the full constructs shown below (* indicates mutagenesis site):
1) AraC-RBS-FhuA-Bioscaffold-GFP*-Terminator (for 4 different bioscaffold versions) 2) AraC-RBS-OsmE*-Bioscaffold-GFP-Terminator (for 4 different bioscaffold versions)
Transformations with XL-1 Blue were set up for the above mutants. Transformations with 2ul of the mutated products were unsuccessful.
Transformed versions A1(CspCI St.) A2(BseRI) A3(CspCI alternative) of the bioscaffold constructs with FhuA. 7ul in Nova-Blue strain. Transformations were successful but only few colonies appeared.
Alternative Assembly Constructs
A finished version of alternative assembly was created featuring:
1) AraC-RBS-FhuA-GFP
The finished construct was tested by setting up overnight 5ml liquid cultures in LB Broth (Ampicillin only). The following was set up:
1) AraC-RBS-FhuA-GFP 2) AraC-RBS-GFP-Term (control and promoter testing)
In the morning 100ul aliquots were transferred in fresh medium (5ml LB+Ampicillin) and were induced with arabinose in varying concentrations;
1) 0% 2) 0.05% 3) 0.1% 4) 0.2% 5) 0.5% 6) 1%
No GFP was seen to be produced in both the control and the FhuA-GFP construct. Also the FhuA-GFP construct seem not to be of high concentration.
The graph shows on the y-axis Optical Density at absorbance of 600nm (representative of cell concentration). The higher the value, the higher the cell concentration. The x-axis is the percentage of L-Arabinose used for promoter induction. ARF is AraC-RBS-FhuA-GFP and results show drop in optical density after promoter induction. ARG is AraC-RBS-GFP-Terminator and after promoter induction there is no drop in optical density. Clearly the promoter (AraC-RBS) works whilst the FhuA-GFP protein fusion inhibits cell growth.
The ladder is standard 1kb NEB DNA marker. Lanes run in quadruplets illustrating Uncut, Single Digest with BpuEI, Single Digest with XbaI and Double digest with XbaI and BpuEI of plasmid DNA carrying AraC-RBS-FhuA-Bioscaffold-GFP-Term on a pSB2K3 plasmid. Lanes 1-4 show pre-mutated DNA of the construct. It is apparent that lanes with BpuEI cause star activity for prolonged periods of incubation, exceeding 60min. Successful ligations and mutations are shown by lanes 5-8 where only two bands are produced in double digests.
: Lanes run in quadruplets illustrating Uncut, Single Digest with BpuEI, Single Digest with XbaI and Double digest with XbaI and BpuEI of plasmid DNA carrying AraC-RBS-FhuA-Bioscaffold-GFP-Term on a pSB2K3 plasmid. Lanes 1-4, 25-27 show successful ligations and also succesful removal of the BpuEI site in GFP that conflicted the Bioscaffold specific enzyme.
Week 11
Alternative Assembly Constructs
Last week's experiment was repeated with ARabinose induction occuring overnight. Again the FhuA-GFP construct seemed not to be growing.
Cell concentration assays revealed that the fusion construct was inhibiting E.coli growth. The only GFP control seemed to be growing normally. (see results here)
To test the inhibition fresh transformants of :
1) AraC-RBS-OsmE 2) AraC-RBS-FhuA 3) AraC-RBS-FhuA-GFP 4) AraC-RBS-GFP-Terminator
were set up. 3 colonies of each were picked and grown overnight. The next day 20ml cultures were set-up for each colony and growth was tested with A600 O.D. every half an hour (pre and post arabinose induction).
Results indicate that construct 1 and 3 lyse the cells.
Bioscaffold Tests
To test for successful mutation 3 colonies of each mutated construct (BpuEI site mutated out) were miniprepped and tested with BpuEI. Unfortunately the enzyme was not working.
To test the Bioscaffold the mutated construct of AraC-RBS-FhuA-Bioscaffold-GFP was transferred to the pSB2k3 backbone that is compatible with the Bioscaffold specific enzymes. (enzymes used by the bioscaffold A2(BseRI and BpuEI) do not appear in this backbone.
As a back-up plan, if the mutation was unsuccessful the C-terminal region of GFP will be cut out in order to assess Bioscaffold efficiency.
The graph illustrates the assessment of growth post promoter induction with L-Arabinose at 0.1%. The y-axis is time(min) and the x-axis log10 A600 Optical Density, representing cell concentration. Promoter induction occurred at 95min. ARG is AraC-RBS-GFP-Terminator, ARFG is AraC-RBS-FhuA-GFP ARO is AraC-RBS-OsmE and ARF is AraC-RBS-FhuA. ARO and ARFG indicate drop in OD A600 whilst ARF and ARG keep growing normaly post-induction.
Week 12
During week 11 tests for mutation success for GFP illegal Bioscaffold sitewere carried out and successful mutants were identified to proceed with the bioscaffold tests
The entire construct AraC-RBS-FhuA-Bioscaffold-GFP-Terminator was inserted in a pSB2K3 plasmid. pSB2k3 has no restriction enzyme recognition sequences for the Bioscaffold specific enzymes.
Different liquid cultures carrying the entire construct were harvested by centrifugation and DNA was miniprepped.
Successful mutants were identified. Also prolonged incubation leads to BpuEI having star activity. As a result we decided to incubate for 45min. (Bioscaffold enzymes are time-saver qualified and can also be used for 5min incubation).
The Bioscaffold was tested by following its application instructions:
Step1: Restrict DNA with BseRI to remove stop codons and DNA scar.  Step2: Ligate to convert the remaining stop codon into tyrosine aminoacid (From TA to TAC).  Step3: Restrict Digest with BpuEI to collapse bioscaffold.   Step4: Ligate to obtain a seemingly scarless fusion.
All incubations were for 60min/37oC. After each step, the relevant enzymes were inactivated and DNA purification was implemented.
Aliquots from each step were kept and run on an agarose gel to assess Bioscaffold viability.
Transformations with XL-1 Blue cells from each restriction-ligation pair and subsequent restreak and liquid cultures were made. Cells were harvested by centrifugation and DNA plasmid obtained by minipreps.
Aliquots from each step were assessed by BpuEI single digest, BseRI single digest and Double Digest. Preliminary results show bioscaffold to be successful.
Samples were sent for sequencing to confirm if Bioscaffold application was successful.
The ladder is a standard 1kb NEB DNA marker. Lanes run in pairs of four illustrating Uncut, Single Digest (BpuEI), Single Digest (BseRI) and Double Digest of plasmid DNA carrying the entire AraC-RBS-FhuA-Bioscaffold-GFP-Terminator construct (left) and the product post application of the Bioscaffold as outlined in text (right). Band shift are seen on samples on the gel still carrying the Bioscaffold but no band shifts are seen on the gel from samples post-scaffold application indicating collapse of the scaffold.
Retrieved from " http://2009.igem.org/Team:BCCS-Bristol/Project/Wet_lab/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
