<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGem Groningen 2018</TITLE></P><DIV class="navbar-fixed"><NAV class="row green"><UL id="dropdown-project" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Description" class="green-text">Description</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Design" class="green-text">Design</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Results" class="green-text">Results</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Demonstrate" class="green-text">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Entrepreneurship" class="green-text">Entrepreneurship</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Awards" class="green-text">Awards</A></LI></UL><UL id="dropdown-notebook" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Notebook" class="green-text">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Protocols" class="green-text">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Safety" class="green-text">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Measurement" class="green-text">Measurement</A></LI></UL><UL id="dropdown-parts" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Parts" class="green-text">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Primers" class="green-text">Primers</A></LI></UL><UL id="dropdown-HP" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Human_Practices" class="green-text">Integrated Human Practices (Gold)</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Public_Engagement" class="green-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Applied_Design" class="green-text">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Ethics" class="green-text">Ethics</A></LI></UL><UL id="dropdown-attribution" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Attributions" class="green-text">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Sponsors" class="green-text">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Crowdfunding" class="green-text">Crowdfunding</A></LI></UL><UL id="dropdown-help" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/InterLab" class="green-text">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Software" class="green-text">Software</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Collaborations" class="green-text">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Embassy" class="green-text">Consulate Meetup</A></LI></UL><UL id="dropdown-team" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Team" class="green-text">Team</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Supervisors" class="green-text">Supervisors</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Team_Building" class="green-text">Team Building</A></LI></UL><UL id="dropdown-model" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Model" class="green-text">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Molecular_Dynamics" class="green-text">Proximity by affinity</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Mathematical_Modeling" class="green-text">Cellulose Degradation</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Flux_Based_Analysis" class="green-text">Optimizing StyGreen Production</A></LI></UL><DIV class="nav-wrapper" id="nav"><UL class="left" id="nav-buttons"><LI><A href="https://2018.igem.org/Team:Groningen" class="waves-effect waves-light block"><I class="material-icons">home</I></A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-project">Project</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-notebook">Lab</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-model">Model</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-parts">Sequences/Parts</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-HP">Human Practices</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-help">Contributions</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-team">Team</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-attribution">Acknowledgements</A></LI></UL><UL class="right" id="contact"><LI title="Click to visit our Facebook page"><A target="_blank " href="https://www.facebook.com/igemgroningen2018/" class="menu-hover waves-effect waves-light block"><SPAN>Groningen.iGEM2018</SPAN></A></LI><LI title="Click to e-mail us"><A target="_blank" href="mailto:igem2018rug@gmail.com" class="menu-hover waves-effect waves-light block"><SPAN>igem2018rug@gmail.com</SPAN></A></LI><LI><A class="menu-hover" id="cross"><I class="material-icons">close</I></A></LI></UL></DIV></NAV></DIV><A href="#!" class="btn-floating btn-large waves-effect waves-light green" id="menu-btn"><I class="material-icons">menu</I></A><DIV class="igem_2018_team_content" style="background-color: white"><DIV class="igem_2018_team_column_wrapper"><UL class="collapsible popout"><LI><DIV class="collapsible-header" id="ecoli-transformation">E. coli transformation protocol</DIV><DIV class="collapsible-body"><P>The protocol for making E. coli competent cells can be found <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli">here</A>The protocol for making LB agar plates can be found <A href="https://2018.igem.org/Team:Groningen/Protocols#Yeast-transformation">here</A>The source of this protocol can be found <A href="http://parts.igem.org/Help:Protocols/Transformation" target="_blank">here</A></P><P><U>Method</U></P></DIV></LI><LI>Thaw the competent cells on ice.</LI><LI>Pre-chill 1,5 ml eppendorf tubes on ice.</LI><LI>Spin down the DNA.</LI><LI>Transfer 1-5 µl of the DNA into an eppendorf tube.</LI><LI>Add 50 µl of the competent cells.</LI><LI>Incubate the cells with the DNA on ice for 30 minutes.</LI><LI>Perform a heat-shock in a waterbath, 42℃, 45 seconds.</LI><LI>Immediately place the tubes on ice for 5 minutes.</LI><LI>Add 950 µl of LB per transformation.</LI><LI>Incubation at 37℃, 200 rpm for 1 hour.</LI><LI>Plate 100 µl on plates containing the appropriate antibiotic, spin down the rest of the cells and plate them on a second plate.</LI><LI>Incubate the plates overnight at 37℃.</LI><LI><DIV class="collapsible-header" id="Yeast-SDS">Extract of yeast cells with TCA for SDS-PAGE</DIV><DIV class="collapsible-body"><P><U>Materials:</U>- Sample buffer: ( 2.5 ml 1 M Tris-HCl pH 6.8, 0.5 ml of ddH20, 1.0 g SDS, 0.8 ml 0.1% Bromophenol Blue, 4 ml 100% glycerol, 2 ml 14.3 M β-mercaptoethanol (100% stock)) - Trichloroacetic acid 12,5% - 80% acetone ice cold - Resuspension buffer: (1% SDS, 0.1M NaOH)</P><P><U>Method</U></P></DIV></LI><LI>Cultivate cells and harvest 3 OD units.</LI><LI>Spin down the cells (1 min, 14.000 rpm).</LI><LI>Wash the cells in 900 µl water (vortex) and centrifuge.</LI><LI>Resuspend the cells in 400 µl 12,5% TCA.</LI><LI>Freeze the samples at -80℃ for minimal 30 minutes.</LI><LI>When running gel thaw cells and centrifuge 14000 rpm, 5 minutes.</LI><LI>Wash the pellet twice with ice-cold 80% acetone, spin 5 min 14000 rpm.</LI><LI>Remove the supernatant carefully and dry the pellets.</LI><LI>Dissolve the pellet in 75 µl resuspension buffer.</LI><LI>Add 25 µl SDS sample buffer to 75 µl sample and boil at 100℃ for 5 min.</LI><LI>Use the sample immediately for SDS-PAGE.</LI><P><U>Notes:</U> steps 1-3 as quick as possible Watch out, acetone can remove ink labels from tubes!</P><LI><DIV class="collapsible-header" id="SDS-PAGE">Standard SDS-PAGE protocol </DIV></LI><LI><U>Materials</U>:</LI><LI>- 30% acrylamide/Bisacrylamide </LI><LI>- 1M Tris∘HCl pH 8.8</LI><LI>- 1M Tris∘HCl pH 6.8</LI><LI>- 1% SDS</LI><LI>- 10% ammonium persulfate (APS)</LI><LI>- TEMED</LI><LI>- 10X electrophoresis buffer</LI><LI>- 1% SDS (10g/L)</LI><LI>- 0,25M Tris (15g/L)</LI><LI>- 1,92M glycine (72g/L)</LI></UL><OL><LI>Clean the glass and spacers with ethanol and dry them.</LI><LI>Test leakage by pouring some water in.</LI><LI>Prepare the gel mixture.</LI><TABLE><TBODY><TR><TH>Running gel (10 ml total, 2 gels)</TH><TH>7,5 % (&gt;100kDa)</TH><TH>10%</TH><TH>12,5%</TH><TH>15% (&lt;40 kDa)</TH></TR><TR><TD>1M Tris pH 8.8</TD><TD>4 ml</TD><TD>4 ml</TD><TD>4 ml</TD><TD>4 ml</TD></TR><TR><TD>1% SDS</TD><TD>1 ml</TD><TD>1 ml</TD><TD>1 ml</TD><TD>1 ml</TD></TR><TR><TD>Acrylamide</TD><TD>2,5 ml</TD><TD>3,3 ml</TD><TD>4,2 ml</TD><TD>5 ml</TD></TR><TR><TD>Water</TD><TD>2,5 ml</TD><TD>1,7 ml</TD><TD>0,8 ml</TD><TD>0</TD></TR><TR><TD>APS</TD><TD>100 µl</TD><TD>100 µl</TD><TD>100 µl</TD><TD>100 µl</TD></TR><TR><TD>TEMED</TD><TD>10 µl</TD><TD>10 µl</TD><TD>10 µl</TD><TD>10 µl</TD></TR></TBODY></TABLE><LI>Pour approximately 4,5 ml running gel into the cassette, with a small volume of isopropanol on top to ensure an even surface. Wait approximately 30 minutes before removing the isopropanol, pouring the stacking gel on top and adding the comb.</LI><TABLE><TBODY><TR><TH>Stacking gel (5 ml total, 2 gels)</TH><TH>4%</TH></TR><TR><TD>1M Tris pH 6.8</TD><TD>0,6 ml</TD></TR><TR><TD>1% SDS</TD><TD>0,5 ml</TD></TR><TR><TD>Acrylamide</TD><TD>0,7 ml</TD></TR><TR><TD>Water</TD><TD>3,2 ml</TD></TR><TR><TD>APS</TD><TD>50 µl</TD></TR><TR><TD>TEMED</TD><TD>5 µl</TD></TR></TBODY></TABLE><LI>Assemble the gel and pour 1X electrophoresis buffer into the SDS-PAGE system.</LI><LI>Load samples on the gel together with a ladder. (link to sample preparation protocol)</LI><LI>Run the gel at a constant voltage around 200V until the loading dye front reaches the bottom of the gel</LI></OL><P><U>Staining</U>Take the gel out of the system and incubate for the appropriate amount of time with a dye to visualize the proteins.</P><P><U>Notes</U>- The 1X running buffer can be reused. It should be discarded when it becomes blue-ish. - Gels can be wrapped in wet tissues and stored in a fridge for 1-2 months.</P></DIV><LI><DIV class="collapsible-header" id="Agarose-gel">Standard Agarose gel protocol </DIV><DIV class="collapsible-body"><P><U>Materials</U></P></DIV></LI><LI>50x TAE buffer: (Tris-base: 242 g, acetic acid: 57.1 ml, EDTA: 100 ml, 0.5M sodium EDTA and add dH2O up to one litre).</LI><LI>1 % Agarose gel: (10 gr agarose/L, 10 ul DNA stain serva/L add 1X TAE buffer till 1 litre). Microwave till components are dissolved and store at 60 degrees C.</LI><P><U>Method</U></P><OL><LI>Pour 1% agarose gel and wait till polymerization. </LI><LI>Load samples mixed with 6X loading dye if loading dye is not present yet. (6-10 ul).</LI><LI>Load 5 ul 1kB ladder.</LI><LI>Run gel in 1X TAE buffer for 30 min, 90V. </LI><LI>Image gel using Gel doc. EZ Gel documentation system. With automatic exposure setting for faint bands.</LI></OL></DIV><LI><DIV class="collapsible-header" id="Yeast-transformation">High efficiency yeast transformation of intact yeast cells </DIV><DIV class="collapsible-body"><P><U>Chemicals:</U>• 0.1 M LiAc pH 7.5, adjusted with diluted acetic acid. • 1 M LiAc pH 7.5, adjusted with diluted acetic acid. • 50% PEG6000 solution. • 10 mg/ml herring sperm.</P><P><U>Method</U></P></DIV></LI><LI>Inoculate cells in 20 ml LB. </LI><LI>Measure OD600 next day and use 7.5 OD units (OD600*ml culture) per transformation. Ideally use a culture of OD600: 0.4.</LI><LI>Spin 3 min at 3,000 rpm.</LI><LI>Wash the cells twice with 25 ml sterile water.</LI><LI>Resuspend the pellet in 1 ml sterile 0.1 M LiAc and transport to an eppendorf tube.</LI><LI>Spin at 1,300g for 1 min.</LI><LI>Resuspend the pellet in 1 ml 0.1 M LiAc and divide the cells over 10 tubes.</LI><LI>Spin at 1,300g for 1 min.</LI><LI>Remove the supernatant and add to the dry pellet in this order:</LI><LI>240 μl 50% PEG6000</LI><LI>35 μl 1 M LiAc</LI><LI>75 μl DNA mix (200 ng pDNA/ plasmid + 50 μg [5 μl] of denatured herring sperm (denaturing steps specified further in protocol), with sterile water added to a final volume of 75 μl).</LI><LI>Vortex until the pellet is completely resuspended.</LI><LI>Incubate 30 min at 30°C.</LI><LI>Heat shock 20-25 min at 42°C.</LI><LI>Centrifuge tubes for 15 s at full speed and discard the supernatant.</LI><LI>If an antibiotic resistance marker is introduced, carefully resuspend the cells in 1 ml YPD and incubate shaking at 30ºC for the time specified as specified further in the protocol. If not, continue with step 16.</LI><LI>Resuspend the cells carefully in 1 ml sterile water and plate 100 μl onto one selection</LI><LI>plate (1/10).</LI><LI>Spin the remaining cells for 1 min at 1,300 rpm.</LI><LI>Discard supernatant and resuspend the pellet in 100 μl sterile water and plate onto a second selection plate (9/10).</LI><LI>Incubate 2-3 days at 30°C until transformants appear</LI><P><U>Antibiotics Minimal incubation time:</U>- G418 (kanMX) 2 hours - Hygromycine (hphMX) 2 hours - Phleomycine (ZEO) 6 hours - Nourseothrycine (natMX) 2 hours</P><P><U>Denaturizing herring sperm:</U></P><OL><LI>Defrost tube.</LI><LI>Incubate 5 min at 100°C.</LI><LI>Place on ice immediately after denaturization.</LI><LI>Solution is ready for use</LI><LI>Remaining solution can be stored at -20°C.</LI><LI>After 4 freeze-thaw cycles the solution has to be denatured again</LI></OL></DIV><LI><DIV class="collapsible-header" id="LB-plates">Making LB agar plates </DIV><DIV class="collapsible-body"><P>Preparation of 2% LB agar plates.</P><P><U>Method</U></P></DIV></LI><LI>Add agar to obtain weight/volume 2%.</LI><LI>Add 15.5 g/L L-broth.</LI><LI>Add water up to the appropriate total volume for the concentrations.</LI><LI>Autoclave the solution and cool till approx 60 degrees C.</LI><LI>Optional: Add antibiotic.</LI><LI>Pour plates and store at 4℃ until further use.</LI></DIV><LI><DIV class="collapsible-header" id="Verduyn">Making Verduyn medium </DIV><DIV class="collapsible-body"><P><U>Verduyn medium:</U></P></DIV></LI><LI>Carbon source (Dextrose) - 20,0 g/L</LI><LI>(NH4)2SO4 - 5,0 g/L</LI><LI>KH2PO4 - 3,0 g/L</LI><LI>MgSO4⋅7H20 - 0.5 g/L</LI><LI>1000x vitamin solution* - 1 mL/L</LI><LI>100x trace elements - 10 mL/L</LI><P>*Vitamin solution is added after autoclaving the medium</P><P><U>1000x vitamin solution:</U></P><UL><LI>D-biotin - 0.05 g/L</LI><LI>Ca-D-pantothenate - 1.00 g/L</LI><LI>Nicotonic acid - 1.00 g/L</LI><LI>Myo-inositol - 25.00 g/L</LI><LI>Thiamine hydrochloride - 1.00 g/L</LI><LI>Pyridoxal hydrochloride - 1.00 g/L</LI><LI>p-aminobenzoic acid - 0.20 g/L</LI></UL><P><U>100x trace elements for Verduyn medium</U></P><UL><LI>Na2EDTA - 1.50 g/L</LI><LI>ZnSO4·7H2O - 0.45 g/L</LI><LI>MnCl2·2H2O - 0.10 g/L</LI><LI>CoCl2·6H2O - 0.03 g/L</LI><LI>CuSO4·5H2O - 0.03 g/L</LI><LI>Na2MoO4·2H2O - 0.04 g/L</LI><LI>CaCl2·2H2O - 0.45 g/L</LI><LI>FeSO4·7H2O - 0.30 g/L</LI><LI>H3BO3 - 0.10 g/L</LI><LI>KI - 0.01 g/L</LI></UL><P>Amino acid concentrations for auxotrophs:</P><TABLE><TBODY><TR><TH>constituent</TH><TH>ug/liter in medium</TH><TH>stock mg/ml (dilution of stock)</TH></TR><TR><TD>adenine-sulphate</TD><TD>20</TD><TD>2 (100x)</TD></TR><TR><TD>uracil</TD><TD>30</TD><TD>3 (100x)</TD></TR><TR><TD>1-tryptophane</TD><TD>20</TD><TD>10 (500 x)</TD></TR><TR><TD>1-histidine.HCl</TD><TD>20</TD><TD>10 (500x)</TD></TR><TR><TD>1-arginine.HCl</TD><TD>20</TD><TD>10 (500x)</TD></TR><TR><TD>1-methionine</TD><TD>20</TD><TD>10(500x)</TD></TR><TR><TD>1-tyrosine</TD><TD>30</TD><TD>2(66x)</TD></TR><TR><TD>1-leucine</TD><TD>30</TD><TD>3(100x)</TD></TR><TR><TD>1-isoleucine</TD><TD>30</TD><TD>3(100x)</TD></TR><TR><TD>1-lysine.HCl</TD><TD>30</TD><TD>10(300x)</TD></TR><TR><TD>1-phenylalanine</TD><TD>50</TD><TD>10(200x)</TD></TR><TR><TD>1-glutamate</TD><TD>100</TD><TD>10(100x)</TD></TR><TR><TD>1-aspartate</TD><TD>100</TD><TD>10(100)</TD></TR><TR><TD>1-valine</TD><TD>150</TD><TD>30(200x)</TD></TR><TR><TD>1-threonine</TD><TD>200</TD><TD>40(200x)</TD></TR><TR><TD>1-serine</TD><TD>400</TD><TD>80(200x)</TD></TR></TBODY></TABLE></DIV><LI><DIV class="collapsible-header" id="competent-ecoli">Standard protocol for production of competent E. coli </DIV><DIV class="collapsible-body"><P>You can find the source of the protocol <A href="http://parts.igem.org/Help:Protocols/Competent_Cells" target="_blank">here</A></P><P><U>Materials:</U></P></DIV></LI><LI>CCMB80 medium</LI><LI>10 mM KoAc pH 7.0</LI><LI>80 mM CaCl2 </LI><LI>20 mM MnCl2 </LI><LI>10 mM MgCl2</LI><LI>10% glycerol</LI><P>Filter sterilized and stored at 4℃.</P><P><U>Method</U></P><OL><LI>Inoculate 250 ml of LB medium with the seed stock of E. coli DH5ɑ and grow overnight to an OD600 of 0.3 (better lower than higher).</LI><LI>Fill an ice bucket with ice and pre-chill flat bottom centrifuge bottles (these make it easier to resuspend the cells).</LI><LI>Transfer the culture to the flat bottom centrifuge tubes, weigh and balance the tubes.</LI><LI>Centrifuge at 3000g for 10 minutes, at 4℃.</LI><LI>Decant the supernatant.</LI><LI>Resuspend the cells in 80 ml of ice-cold CCMB80 buffer.</LI><LI>Incubate on ice for 20 minutes.</LI><LI>Centrifuge again at 3000g for 10 minutes, 4℃.</LI><LI>Resuspend the cell pellet in 10 ml of ice-cold CCMB80 buffer.</LI><LI>Measure the OD of a mixture of 200 µl LB and 50 µl of the cells.</LI><LI>Add chilled CCMB80 to target a final OD of 1.0-1.5.</LI><LI>Incubate on ice for 20 minutes.</LI><LI>Aliquot 300ul into eppendorf tubes.</LI><LI>Store at -80℃ indefinitely.</LI></OL></DIV><LI><DIV class="collapsible-header" id="restriction-digest">Standard protocol for restriction digest </DIV><DIV class="collapsible-body"><P><U>Restriction analysis:</U></P></DIV></LI><LI>Mix: </LI><LI>400 ng of pDNA. </LI><LI>5 µl Fastdigest buffer green </LI><LI>0,5 µl per restriction enzyme used. </LI><LI>MQ to final volume 50 µl.</LI><LI>Incubate at 37℃ for 1 hour.</LI><LI>Run restriction product on an agarose gel to confirm the length of the product. See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE">agarose gel protocol</A></LI><P>Make sure DNA for transformation is PCR cleaned. If not use PCR clean up kit to clean DNA.</P><P><U>Plasmid restriction for transformation:</U></P><OL><LI>Mix: </LI><LI>2*50 ng plasmid (2kB plasmid) or 2*100ng plasmid (5kB plasmid). Half for ligation with insert and half as negative control.</LI><LI>5 µl Fastdigest buffer </LI><LI>0,5 µl per restriction enzyme used. </LI><LI>0,5 ul FAST AP</LI><LI>MQ to final volume 50 µl.</LI><LI>Incubate at 37℃ for 1 hour.</LI><LI>Heat-inactivate the enzymes by incubation at 75℃ for 10 minutes.</LI></OL><P><U>Insert restriction for transformation:</U></P><OL><LI>Mix: </LI><LI>3x equimolar amount of vector DNA.</LI><LI>5 µl Fastdigest buffer </LI><LI>0,5 µl per restriction enzyme used. </LI><LI>MQ to final volume 50 µl.</LI><LI>Incubate at 37℃ for 1 hour.</LI></OL></DIV><LI><DIV class="collapsible-header" id="T4-ligation">Standard protocol for T4 ligation </DIV></LI><LI>Mix </LI><LI>50 ng (2kB plasmid) or 100 ng (5kB plasmid) of digested plasmid backbone (treated with Fast AP).</LI><LI>3:1 molar amount of insert: plasmid backbone.</LI><LI>2 µl T4 Ligase buffer (10X).</LI><LI>0,5 µl T4 ligase.</LI><LI>MQ up to final volume of 20 µl.</LI><LI>Incubation at room temperature for at least 1 hour.</LI><LI>Use ligation mix for transformation.</LI><LI><DIV class="collapsible-header" id="WB">Standard Western Blot protocol</DIV><DIV class="collapsible-body"><P><U>Materials:</U></P></DIV></LI><LI>A1 buffer (0.3M Tris, 20% MeOH)</LI><LI>A2 buffer (25mM Tris, 20% MeOH)</LI><LI>K buffer (25mM, 40mM 6-amino-n-hexanoic acid, 20% MeOH)</LI><P><U>Method</U></P><OL><LI>Wash the Blotting plates with demi water</LI><LI>Place on the anode in the following order: </LI><LI>Pad soaked in A1</LI><LI>Pad soaked in A2</LI><LI>PVDF membrane soaked in MeOH</LI><LI>Running gel</LI><LI>Pad soaked in K buffer</LI><LI>Gently remove air bubbles with a roller</LI><LI>Place the cathode on top</LI><LI>Blot for 45-60 minutes </LI><LI>1 gel 43 mA</LI><LI>2 gels 86 mA</LI><LI>4 gels 172 mA</LI><LI>Remove the blot and stain the western blot</LI></OL><LI><DIV class="collapsible-header" id="WB-His">Protocol for CBP-Star visualisation of His-tagged proteins on Western Blot  </DIV><DIV class="collapsible-body"><P><U>Materials:</U></P></DIV></LI><LI>PBS (For 1L of 10X: 207.7g Na2HPO412H20, 23.5g NaH2PO4*H2O, 39.7g NaCl)</LI><LI>PBST (0.1% Tween-20 in PBS)</LI><LI>Blocking buffer (PBST + 0.2% I-block. If I-block is added to PBST, it must be heated in the microwave to dissolve)</LI><LI>Antibodies: 2µl 5000x antibody into 10ml PBST with 0.1% I-block</LI><LI>EDTA: 2ml 0.5M EDTA into 8ml PBST with 0.1% I-block</LI><LI>Assay buffer: 5.26g diethanolamine 1mM MgCl2 0.5ml concentrated HCl in 500ml tridist H2O</LI><LI>CSPD-star: 40µl CSPD-star into 2ml assay buffer</LI><P><U>Method</U></P><OL><LI>Incubate the PVDF membrane in PBST+0.2% I-block 1h or overnight at 4℃</LI><LI>Incubate for 1h in PBST+0.1% I-block+EDTA</LI><LI>Wash twice 10 min with PBST+0.1% I-block</LI><LI>Incubate 2h in PBST+0.1% I-block+first antibody</LI><LI>Wash twice 10 min with PBST+0.1% I-block</LI><LI>Incubate 1h in PBST+0.1% I-block+second antibody</LI><LI>Wash three times 10 min with PBST+0.1% I-block</LI><LI>Incubate twice 5 min in assay buffer, drain the excess liquid from the blot with paper</LI><LI>Add assay buffer + CSPD-star and incubate for 5 minutes, drain the excess liquid</LI><LI>Image (chemiluminescence imaging, high sensitivity, increment 30-60 sec intervals)</LI></OL><LI><DIV class="collapsible-header" id="cellulosephosphorylation">Phosphorylation of cellulose I &amp; II</DIV><DIV class="collapsible-body"><P><B>Phosphorylation of cellulose I</B></P><FIGURE><FIGCAPTION><I>Cellulose 6-phosphate</I></FIGCAPTION></FIGURE><P>A mixture of cellulose (500 mg, 1.54 mmol), o-phosphoric acid (80.3 μl, 1.54 mmol), urea (278 mg, 4.62 mmol) and water (279 μl, 15.4 mmol) was stirred for 4 hours at 60°C. Subsequently it was stirred for another 2 hours at 120°C. Then the mixture was washed with 50% ethanol, 0.5 N HCL, 50% ethanol and dried for 18h under vacuum at 60°C.</P><P>[1] Nehls, I. and Loth, F. 13C-NMR-spektroskopische Untersuchungen zur Phosphatierung von Celluloseprodukten im system H3PO4/Harnstoff. Acta Polymerica 42, 5, (1991).
		[2] Illy, N., Fache, M., Ménard, R., Negrell, C., Caillol, S. and David, G. Phosphorylation of bio-based compounds: the state of the art. Polym. Chem., 6, 6257, (2015).</P><P><B>Phosphorylation of cellulose II</B></P><P>Cellulose (200 mg, 0.62 mmol) and distilled water (0.6 mL) were added to a 50 mL glass tube. Slowly, ice cold o-phosphoric acid (15.2M, 10 mL) was added to the mixture under vigorous stirring. This mixture was stirred for one hour on ice. Subsequently, 40 mL of ice cold water was added. This mixture was centrifuged at 5000g at 4°C for 20 minutes. Then the supernatant was discarded and the pellet re-suspended in ice cold water. This mixture was centrifuged at 5000g at 4°C for 10 minutes. The washing cycle was repeated 4 times. Afterwards the pellet was washed one time with Na2CO3 (2M, 1 mL) and ice cold distilled water (39 mL). Finally, the pellet was washed with distilled water until pH 5-7.</P><P>[1] Zhang, Y. H. P., Cui, J., Lynd, L. R. and Kuang, L. R. A transition from Cellulose Swelling to Cellulose Dissolution by o-Phosphoric Acid: Evidence from Enzymatic Hydrolysis and Supramolecular Structure. Biomacromolecules, 7, 644-648, (2006).
		[2] Wen, F., Sun, J. and Zhao, H. Yeast Surface Display of Trifunctional Minicellulosomes for Simultaneous Saccharification and Fermentation of Cellulose to Ethanol. Appl. Environ. Microbiol., 76, 1251-120, (2010).</P></DIV></LI><LI><DIV class="collapsible-header" id="colony-PCR">Colony PCR protocol </DIV><DIV class="collapsible-body"><P>Add each colony to 15 ul MQ and mix solution.</P><P><U>Prepare PCR mastermix per colony:</U></P></DIV></LI><LI>10 ul Phire mastermix</LI><LI>0,1 ul primer FW 100 uM</LI><LI>0,1 ul primer RV 100 uM</LI><LI>1 ul colony mix</LI><LI>8,8 ul MQ</LI><P>Run the following PCR protocol:</P><TABLE><TBODY><TR><TH>1</TH><TH>98°C</TH><TH>5min</TH></TR><TR><TD>2</TD><TD>98°C</TD><TD>5s</TD></TR><TR><TD>3</TD><TD>X</TD><TD>5s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>10-15s/kB</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>2x step4</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE><P>Run colony PCR on gel: See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE">agarose gel protocol</A></P><LI><DIV class="collapsible-header" id="Phire-PCR">Phire PCR protocol </DIV><DIV class="collapsible-body"><P><U>Prepare 50 ul Mastermix per amplification:</U></P></DIV></LI><LI>25 ul Phire mastermix</LI><LI>0,25 ul primer FW 100uM</LI><LI>0,25 ul primer RV 100uM</LI><LI>5-200 pg template.</LI><LI>Add MQ till 50 ul.</LI><P>Run the following PCR protocol:</P><TABLE><TBODY><TR><TH>1</TH><TH>98°C</TH><TH>30s</TH></TR><TR><TD>2</TD><TD>98°C</TD><TD>5s</TD></TR><TR><TD>3</TD><TD>x</TD><TD>5s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>10-15s/kB</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>2x step4</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE><P>To verify correct amplification of fragment run on an agaros gel. See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE">agarose gel protocol</A>. </P><LI><DIV class="collapsible-header" id="Phusion-PCR">Phusion PCR protocol </DIV><DIV class="collapsible-body"><P><U>Prepare 50 ul mastermix per PCR reaction</U></P></DIV></LI><LI>25 ul Phusion master mix.</LI><LI>0,25 ul primer FW 100uM.</LI><LI>0,25 ul primer RV 100uM.</LI><LI>5-200 pg DNA</LI><LI>Add MQ till 50 ul.</LI><P>PCR program:</P><TABLE><TBODY><TR><TH>1</TH><TH>98°C</TH><TH>30s</TH></TR><TR><TD>2</TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD>3</TD><TD>X</TD><TD>30s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>15-30s/kB</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>10 min</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE><P>To verify correct amplification of fragment run on an agarose gel. See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE">agarose gel protocol</A>.</P><LI><DIV class="collapsible-header" id="HPLC-validation">HPLC System Validation Protocol </DIV></LI><LI>Start up the HPLC machine and the connected computer</LI><LI>Insert apolar C18 column </LI><LI>Make sure solvent tank A is filled with ultrapure (UP) water</LI><LI>Make sure solvent tank B is filled with acetonitrile</LI><LI>Make sure solvent tank C is filled with methanol</LI><LI>Set up procedure for run 45 min, Temp: 40 °C, Flow: 20 µl/min, gradient 100 % A → 100 % B</LI><LI>Put HPLC vial X with 10 mM styrene in ethyl acetate solution into autosampler</LI><LI>Put HPLC vial Y with 10 mM styrene in ethyl acetate + cell lysate into autosampler</LI><LI>Start Run</LI><LI>Measure the retention time of all peaks (should be only 1)</LI><LI>Check the DAD recorded UV spectrum at all peaks</LI><LI>The peak whos UV spectrum matches styrenes (absorption peak at ~245 nm) is styrene</LI><LI>Note the retention time and the % B where it elutes</LI><LI>The retention time should be the same for vial X and vial Y</LI><LI>Save the chromatograms as pdf</LI><LI>Optimize the running conditions by altering run length, flow, gradient and mobile phase</LI><LI><DIV class="collapsible-header" id="styrene-HPLC">HPLC Styrene detection protocol </DIV></LI><LI>Start up the shimadzu HPLC UV DAD machine and the connected computer</LI><LI>Insert apolar C18 column </LI><LI>Make sure solvent tank A is filled with ultrapure (UP) water</LI><LI>Make sure solvent tank B is filled with acetonitrile</LI><LI>Make sure solvent tank C is filled with methanol</LI><LI>Load HPLC method:</LI><LI>Column length: 150 mm, Temp: 40 °C, Flow: 20 µl/min, gradient A &amp; C with t=0 C 30 % to t=45 C 100 %</LI><LI>Prepare sample as follows (8 - 17):</LI><LI>Bring 900 µl of liquid culture with cells of interest into 2 ml Eppendorf cup</LI><LI>Add ~200 µl volume worth of glass beads and close the Eppendorf cup</LI><LI>Lysate the cells by vortexing the Eppendorf for 10 minutes</LI><LI>Spin the glass beads down on the centrifuge in 1 min at 1.000 rpm</LI><LI>Transfer 800 µl of supernatant into another 2 ml Eppendorf cup</LI><LI>Add 800 µl of ethyl acetate</LI><LI>Extract the styrene from the cell lysate by vortexing for 10 minutes</LI><LI>Transfer the apolar (upper) phase into a separation Eppendorf cup with a 0,22 nm nylon membrane</LI><LI>Filter the apolar phase by centrifuging the Eppendorf cup at 13.000 rpm for 5 min</LI><LI>Transfer the filtered apolar phase into a HPLC vial</LI><LI>Place the HPLC vial into the Autosampler of the HPLC system</LI><LI>Start the run</LI><LI>If a peak shows at 17,5 min / 50 % MeOH and the UV spectrum peaks at 245 nm the presence of styrene is identified</LI><LI><DIV class="collapsible-header" id="styrene-extraction">Styrene Extraction from Liquid Culture Batch for Harvesting </DIV></LI><LI>Measure the OD of liquid culture and make sure it’s in stagnation phase</LI><LI>Bring the liquid culture into a French Press setup for homogenization and cell lysis</LI><LI>Bring the cell lysate into a separatory funnel</LI><LI>Extract the cell lysate 5 times with ethyl acetate (EtOAc) of equal volume and gather the EtOAc fractions</LI><LI>In case of excessive foam formation phase separation can be supported by cooling the separatory funnel and increasing the osmolarity of the aqueous phase through addition of NaCl</LI><LI>Discard the aqueous phase</LI><LI>Extract the collected organic phase twice with an equal volume of water</LI><LI>Transfer the organic phase to a round bottom flask (RBF)</LI><LI>Add 4-tert-Butylcatechol to the organic phase and to the gathering vessel of a rotary evaporator</LI><LI>Attach the RBF to the rotary evaporator and evaporate the ethyl acetate at a pressure of 153 mbar and bath temperature of 40 °C</LI><LI>The purified styrene has to be kept concealed, cool and away from sunlight to prevent runaway polymerisation or evaporation</LI><LI><DIV class="collapsible-header" id="GC-FID">GC-FID calibration for styrene detection </DIV></LI><LI>2-methylanisole stock solution was prepared by dissolving 30,54 mg of 2-methylanisole in 25 ml benzene</LI><LI>Styrene stock solution was prepared by dissolving 5,2075 mg of styrene in 5 ml of benzene</LI><LI>10 calibration points of 200 µl were prepared by pipetting 20 µl of 2-methylanisole stock and X * 10 µl of styrene stock with X going from 1 to 10 and the rest of the volume to 200 µl being filled with benzene</LI><LI>The most concentrated calibration point is injected into the system with a mass spectrometer attached (GC-MS) and the retention times of styrene and 2-methylanisole are recorded</LI><LI>The 10 calibration points are injected into the system with a Flame Ionization Detector (GC-FID)</LI><LI>For each calibration point the Response Ratio is calculated by dividing the AUC of the styrene peak by the AUC of the 2-methylanisole peak</LI><LI>The measurements and calculations of step 5) and 6) are executed in triplo</LI><LI>The r² of the three calibration curves are calculated</LI><LI>In case one of the curves had an r² of &lt; 0,99 the measurements were repeated</LI><LI>For calculating styrene concentration the average of 3 valid slope values and the average of the y-axis intercepts are combined into one equation</LI><LI>The final equation of the calibration curve for our system came out to be:</LI><P>styrene (mmol) = 0,9879494 x RR - 0,0857551</P><LI><DIV class="collapsible-header" id="GC-extraction">GC Styrene Extraction from Liquid Culture for Measurement </DIV></LI><LI>Measure the OD of liquid culture and make sure it’s in exponential growth phase</LI><LI>Bring 900 µl liquid culture into a 2 ml Eppendorf cup</LI><LI>Add 200 µl volume worth of glass beads</LI><LI>Vortex the Eppendorf for 1 minute</LI><LI>Put the Eppendorf on ice for 1 minute</LI><LI>Repeat step 4) and 5) a total of 7 times</LI><LI>Centrifuge the Eppendorf at 1.000 rpm for 1 minute</LI><LI>Transfer 800 µl of the supernatant into a 2 ml Eppendorf cup</LI><LI>Add 80 µl of 10 mM 2-methylanisole stock solution in benzene and 720 µl benzene</LI><LI>Vortex the cup for 10 minutes</LI><LI>Centrifuge at 13.000 rpm for 10 minutes</LI><LI>Transfer the supernatant into a Gas Chromatography vial</LI><LI>Seal the GC Vial and put it into the GC sampler, 10 µl will be </LI><LI>Run the GC at 100 °C, </LI><LI>Measure the AUC of styrene at t=3,741 min and the AUC of 2-methylanisole at t=4,381 min</LI><LI>Calculate the Response Ratio by dividing the AUC of styrene by the AUC of 2-methylanisole</LI><LI>Calculate the styrene concentration from the calibration curve</LI><LI>Calculate the resolution of the peaks by dividing the difference in retention times of the peaks by the average of the two peak widths at their base</LI><LI><DIV class="collapsible-header" id="cellulase-activity">Cellulase activity assay </DIV><DIV class="collapsible-body"><P>This assay has the purpose to measure activity of cellulases on cellulose. In case cellulases are active, they will have produced hydrolytic ends on the degraded cellulose strands. The assay is based on this feature in a two-step oxidative pathway towards a chromogenic output. First, a mutated chito-oligosaccharide oxidase (ChitO-Q268R) releases hydrogen peroxide upon oxidation of the cellulase-produced hydrolytic products. Second, the hydrogen peroxide produced can be monitored by the enzyme horseradish peroxidase (HRP) and a chromogenic peroxidase substrate (DCHBS and AAP). As a result, the intensity of the pink color is proportional to the concentration of the available oxidase substrates and hence the amount of cellulose degradation.</P><P>Incubation buffer: 50 mM KPi pH 4.5 Assay buffer: 50 mM KPi pH 6.0 DCHBS: 3,5-dichloro-2-hydroxybenzenesulfonic acid  AAP: 4-aminoantipyrine</P></DIV></LI><LI>The cellulose and cellulases were incubated in incubation buffer at 40℃, 600 rpm in 2 ml eppendorf tubes in a heat block </LI><LI>The incubation mix of cellulose and cellulases is centrifuged for 1 min at 14000 rpm</LI><LI>50 µl of the supernatant is transferred into a 96 well plate</LI><LI>20 µl of 10 mM DCHBS is added</LI><LI>20 µl of 1 mM AAP is added</LI><LI>4 µl of HRP (200 U/mL) is added</LI><LI>6 µl assay buffer is added</LI><LI>Finally, to start the reaction, 6 µl ChitO (20 U/mL) is added, right before starting the absorbance measurement in a plate reader at 515 nm</LI><H4>References</H4><P>Ferrari, A. R., Gaber, Y. &amp; Fraaije, M. W. A fast, sensitive and easy colorimetric assay for chitinase and cellulase activity detection. Biotechnol. Biofuels 7, 37 (2014).</P><LI><DIV class="collapsible-header" id="galactose-induction">Protocol for galactose induction and sample preparation to make a growth curve</DIV><DIV class="collapsible-body"><P>This protocol describes how to perform galactose induction, and prepare the cells to be loaded into a 96-well plate to make a growth curve using a plate reader.
		This protocol has been developed for use with our Saccharomyces cerevisiae strain containing plasmids CB and 1883, to express an artificial cellulosome. All genes are behind a Gal1-Gal10 bidirectional galactose inducible promotor.</P><P><U>Materials</U></P><P>Verduyn medium with 0.5% fructoseVerduyn medium without carbon sourceAmino acids, as needed to make your cells grow without losing the plasmids40% galactose solutionCellulose, or other carbon source of interest96-well microtiter plateSpectrophotometer, e.g. an Implen Nanophotometerplate reader</P><P><U>Method</U></P><P>Culturing and induction</P></DIV></LI><LI>Culture strains on Verduyn medium with 0,5% fructose and appropriate amino acids. Fructose is chosen as carbon source, as glucose actively represses the Gal1-Gal10 promotor. Incubate at 30 ℃ with agitation.</LI><LI>When the cultures reach an OD600 of between 0.5-0.8, induce the cultures with 2% final concentration galactose (50 µl per ml of 40% stock solution).</LI><LI>Incubate the cultures for 2-6 hours at 30 ℃ with agitation.</LI><LI>Transfer 1ml of culture to a microcentrifuge tube.</LI><LI>Pellet the cells in a microcentrifuge, 5 minutes at 6000 RPM.</LI><LI>Remove the supernatant and resuspend in fresh Verduyn medium without carbon source.</LI><LI>Pellet the cells as in step 5.</LI><LI>Remove the supernatant and resuspend in fresh Verduyn medium without carbon source.</LI><LI>Pellet the cells as in step 5.</LI><LI>Remove the supernatant and resuspend in fresh Verduyn medium without carbon source.</LI><LI>Measure the OD of the resuspended pellet using a spectrophotometer. Due to the small sample volume, a Nanophotometer or Nanodrop style spectrophotometer would be ideal.</LI><LI>Dilute the cells to an OD of 1.0. A volume of 20 µl per well is needed.</LI><P>Preparing a 96-well plate:</P><OL><LI>Load each well with the desired carbon source. In our situation 25 ul of 0.8% phosphorylated cellulose or 20 µl ball-milled ReCell is added.</LI><LI>Add Verduyn medium with appropriate amino acids, without carbon source to each well, up to a total volume of 180 µl.</LI><LI>Add 20 µl of cells from step 12 to each well.  Each well should now contain 200 µl of sample.</LI><LI>Insert the 96-well plate into the plate reader, and setup the desired machine settings.
In our situation: Kinetic measurement, measure every 20 minutes or every hour, for a period of 24 or 48 hours, with continuous agiation, and incubator temperature set at 30 ℃.</LI></OL><LI><DIV class="collapsible-header" id="His-purification">His-tagged protein purification </DIV><DIV class="collapsible-body"><P>- The culturing conditions are 37℃, 200 rpm unless mentioned differently. - Buffers were cooled to 4℃. - From step 3 onwards all steps were carried out on ice or at 4℃.</P><P>Buffer A: 50 mM Tris-HCl pH 8, 100 mM KCl, 20% glycerol Wash Buffer: 50 mM Tris-HCl pH 8, 100 mM KCl, 20% glycerol, 10 mM imidazole Elution Buffer: 50 mM Tris-HCl pH 8, 100 mM KCl, 20% glycerol, 300 mM imidazole</P></DIV></LI><LI>A 25 mL overnight culture was inoculated into 1 L fresh culturing medium with the appropriate antibiotics </LI><LI>The cultures were induced with a final concentration of 1mM IPTG at an OD600 between 0.6 and 0.8</LI><LI>The cultures were induced for 2.5 hours and transferred to ice subsequently</LI><LI>Cultures were centrifuged in 1 L centrifuge bottles for 10 min at 6500 rpm to pellet the cells (JLA 9.1000)</LI><LI>The supernatant is discarded </LI><LI>The cell pellet is resuspended into approximately 6 mL of buffer A </LI><LI>An appropriate amount of DNase is added and the cells are disrupted in a cell disrupter (13 kPsi, 1 passage)</LI><LI>The cell lysate is transferred into 50 ml centrifuge tubes and filled up to 50 ml with buffer A</LI><LI>The cell lysate is centrifuged for 10 min at 3500 rpm (JA 25.50) to pellet cell debris</LI><LI>The supernatant was transferred to new tubes and centrifuged again for 15 min at 19.000 rpm (JA 25.50)</LI><LI>The supernatant was incubated with washed Nickel-charged affinity beads for 1 hour. His-tag purification: samples were taken from all fractions during the purification process for analysis by SDS-PAGE</LI><LI>The suspension was filtered through a Biorad disposable column to collect the Nickel beads</LI><LI>The beads were washed twice with 8 mL wash buffer</LI><LI>Protein was eluted twice with 200 µl elution buffer</LI><LI>The elution fractions were snap frozen in liquid nitrogen and stored at -80℃ until further use</LI><LI><DIV class="collapsible-header" id="PAL2-assay">PAL2 assay raw lysate</DIV><DIV class="collapsible-body"><P>Requirements: -TRIS HCl 20 mM pH 8.0 -Stock phenylalanine 15 mM -Stock trans-cinnamic acid 3 mM -Nanophotometer (or plate reader) -Thermocycler (or water bath/heat block) -~3,26 mg/ml final concentration of total protein</P><P>Steps:</P></DIV></LI><LI>Make a dilution series for trans-cinnamic acid, in the range of the final concentration of phenylalanine (will be diluted 5 times).</LI><LI>Measure absorbance at 268 nm, derive extinction coefficient from calibration curve.</LI><LI>Make phenylalanine dilution series (15 - 1.25 mM).</LI><LI>Setup reactions on ice: (example) </LI><LI>a. 75 µl TRIS</LI><LI>b. 5 µl enzyme solution (20x)</LI><LI>c. 20 µl phenylalanine solution (5x)</LI><LI>d. 100 µl total volume</LI><LI>Incubate at 37℃ for 20 minutes.</LI><LI>Stop reaction by heating at 80+℃ for 5 minutes</LI><LI>Measure absorbance of product at 268 nm, calculate concentrations.</LI><P>If purified enzymes were used, kinetic parameters could be derived from this information, however we used PAL2 as a native enzyme.</P><FOOTER><H4 class="center">We thank our sponsors:</H4></FOOTER></BODY></HTML>