<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Model_Flux_Based_Analysis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Model/Flux Based Analysis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGem Groningen 2018</TITLE></P><DIV class="navbar-fixed"><NAV class="row green"><UL id="dropdown-project" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Description" class="green-text">Description</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Design" class="green-text">Design</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Results" class="green-text">Results</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Demonstrate" class="green-text">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Entrepreneurship" class="green-text">Entrepreneurship</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Awards" class="green-text">Awards</A></LI></UL><UL id="dropdown-notebook" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Notebook" class="green-text">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Protocols" class="green-text">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Safety" class="green-text">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Measurement" class="green-text">Measurement</A></LI></UL><UL id="dropdown-parts" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Parts" class="green-text">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Primers" class="green-text">Primers</A></LI></UL><UL id="dropdown-HP" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Human_Practices" class="green-text">Integrated Human Practices (Gold)</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Public_Engagement" class="green-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Applied_Design" class="green-text">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Ethics" class="green-text">Ethics</A></LI></UL><UL id="dropdown-attribution" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Attributions" class="green-text">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Sponsors" class="green-text">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Crowdfunding" class="green-text">Crowdfunding</A></LI></UL><UL id="dropdown-help" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/InterLab" class="green-text">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Software" class="green-text">Software</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Collaborations" class="green-text">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Embassy" class="green-text">Consulate Meetup</A></LI></UL><UL id="dropdown-team" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Team" class="green-text">Team</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Supervisors" class="green-text">Supervisors</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Team_Building" class="green-text">Team Building</A></LI></UL><UL id="dropdown-model" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Model" class="green-text">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Molecular_Dynamics" class="green-text">Proximity by affinity</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Mathematical_Modeling" class="green-text">Cellulose Degradation</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Flux_Based_Analysis" class="green-text">Optimizing StyGreen Production</A></LI></UL><DIV class="nav-wrapper" id="nav"><UL class="left" id="nav-buttons"><LI><A href="https://2018.igem.org/Team:Groningen" class="waves-effect waves-light block"><I class="material-icons">home</I></A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-project">Project</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-notebook">Lab</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-model">Model</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-parts">Sequences/Parts</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-HP">Human Practices</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-help">Contributions</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-team">Team</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-attribution">Acknowledgements</A></LI></UL><UL class="right" id="contact"><LI title="Click to visit our Facebook page"><A target="_blank " href="https://www.facebook.com/igemgroningen2018/" class="menu-hover waves-effect waves-light block"><SPAN>Groningen.iGEM2018</SPAN></A></LI><LI title="Click to e-mail us"><A target="_blank" href="mailto:igem2018rug@gmail.com" class="menu-hover waves-effect waves-light block"><SPAN>igem2018rug@gmail.com</SPAN></A></LI><LI><A class="menu-hover" id="cross"><I class="material-icons">close</I></A></LI></UL></DIV></NAV></DIV><A href="#!" class="btn-floating btn-large waves-effect waves-light green" id="menu-btn"><I class="material-icons">menu</I></A><DIV class="igem_2018_team_content" style="background-color: white"><DIV class="igem_2018_team_column_wrapper"><DIV class="column" style="float:none !important;"><H4>Introduction</H4><P>The increase in availability of genome scale databases of microorganisms has given rise to the possibility of creating genome scale reaction networks that include corresponding metabolites and relevant genes[1]. One of the methods to study metabolic fluxes through such a wide network is called flux based analysis or FBA. As part of our project we used FBA on a genome scale model (GEM) of S. cerevisiae in order to achieve the following two goals:</P><B><P>Goal 1:  Study the relation between cell growth and styrene production	Goal 2:  Find and visualize interventions that lead to a higher styrene production </P></B><P>An overview of the different methods and software packages that were used to achieve these goals can be seen in figure 1. </P><FIGURE><FIGCAPTION><I>Figure 1: overview depicting various methods used in our study</I></FIGCAPTION></FIGURE><H4>Theory</H4><P>Let us consider a small network consisting of the five reactions (r1 to r5), with mock metabolites A, B ,C and D (figure 2). </P><FIGURE><FIGCAPTION><I>Figure 2: example of a simple metabolic network</I></FIGCAPTION></FIGURE><P>The relative amount of a particular metabolite participating in a reaction is represented by its stoichiometric coefficient, for instance metabolite B has a stoichiometric coefficient of 2 in reaction r2. The whole network of metabolic reactions can be represented in a stoichiometric matrix S of m rows and r columns, where m is the number of metabolites and r is the number of reactions in the network. The matrix then contains on the i,jth entry the stoichiometric coefficient of metabolite i in reaction j. The stoichiometric coefficient is negative when the metabolite is a substrate (traditionally written on the left side of the reaction) and positive when a metabolite is a product (on the right side of the reaction).Therefore our example network can thus be represented as a matrix S of 4 rows and 5 columns as can be seen in figure 3. To illustrate let us consider position 1,1 in the matrix, since the first column represents reaction r1 and the first row the metabolite A, we put in the stoichiometric coefficient of A in r1 which is 1. The complete matrix S for the mock network will look as follows:</P><FIGURE><FIGCAPTION><I>Figure 3: stochiometric matrix describing the network in figure 2</I></FIGCAPTION></FIGURE><P>The flux or reaction rate through each reaction can be represented as a vector v of length r, where r is the amount of reactions in the network and every position in the vector represents the flux through a single reaction.  If the network is at a steady state (all systems variables are constant) the following holds:</P><P>Where S is the stoichiometric matrix and v is the reaction rate vector. Since what we want to know is the flux through each reaction, given a stoichiometric matrix S that represents the whole metabolic network of our cell we have to solve the above equation for v. In a genome-scale model there are usually fewer equations than variables making the system underdetermined. This means that usually there is more than one  v that satisfy the above equation, creating a whole solution space.  In order make this solution space smaller, we choose a physiologically relevant objective function, such as a maximum biomass growth and optimize network fluxes to satisfy this criterion.  </P><H4>The model</H4><P>TIn our study we used to publicly available YeastGem8: “the current consensus genome-scale metabolic model of Saccharomyces cerevisiae”.[2] This model contains three main types of information: 2241 metabolites,  3520 reactions and  1019 genes. The relation between the metabolites and reactions is represented by a stoichiometric matrix, as described above. The interactions between genes and reactions are described by Boolean relationships, meaning that genes can turn reactions on or off.  To accomplish our first goal we used a version of the YeastGem model that is further constrained based on the GECKO method: ecYeast7[3].  </P><P>The GECKO method enhances a model by adding enzymes as part of its reactions. This ensures that fluxes are limited based on the enzymes abundance and their turnover numbers. Since the maximum flux through an enzyme catalysed reaction is equal to the turnover number times its abundance the following holds (for enzymes which catalyse only a single reaction)</P><P>Where v is the flux through a reaction Kcat is the enzyme’s turnover number and [E] its concentration. In practice this means that enzymes have to be included in the model as metabolites with a 1/Kcat stoichiometric coefficient in the reactions they catalyse. </P><P>Now we have to include the enzyme abundance in our model. Since we do not have the proteomic data to limit each enzyme directly based on its abundance we make use of the single pool assumption. We include a pseudo metabolite “protein pool” representing the total amount of enzymes in the cell and then for each enzyme a reaction in which the enzyme is drawn from the protein pool with its molecular weight as a coefficient. Using this method the following reactions were added to the model:</P><FIGURE><FIGCAPTION><I>TCA: trans-cinnamic acid, PAL2: Phenylalanine ammonia-lyase 2, FDC1: Ferulic acid decarboxylase 1, ppool: pseudo metabolite representing the total amount of enzymes</I></FIGCAPTION></FIGURE><P><I>Table 1. Enzymes</I></P><TABLE><TBODY><TR><TH>Enzyme</TH><TH>Kcat (/s)</TH><TH>Weight(kDa)</TH><TH>uniprot</TH><TH>Gene</TH></TR><TR><TD>Ferulic acid decarboxylase 1</TD><TD>4.6 [4]</TD><TD>56.164</TD><TD>Q03034</TD><TD>FDC1 (YDR539W)</TD></TR><TR><TD>Phenylalanine ammonia-lyase 2</TD><TD>3.2 [5]&gt;</TD><TD>77.860</TD><TD>P45724</TD><TD>PAL2 (YNL294C)</TD></TR></TBODY></TABLE><H4>Goal 1</H4><P>First, we studied the relationship between styrene production and cell growth using our updated model. If cell growth is heavily affected by the styrene production, an inducible expression system should be considered. To accomplish it we would have to design our yeast cell so that first the population grows and then induce a switch that activates styrene production and slows down/stops biomass growth.  On the other hand, if styrene production does not hinder cell growth significantly our cells can produce styrene continuously through the cell cycle (growth and reproduction).</P><P>In order to perform flux based analysis we made used functions included in theCOBRA toolbox, a package for MATLAB that implements the COnstraint- Based Reconstruction and Analysis (COBRA) method  [6] The COBRA toolbox contains various tools valuable for modelling and analysing genome-scale networks. In order to solve optimization problems IBM ILOG CPLEX Optimizer (v 12.8.0) was used. </P><P>In the beginning we wanted to see the maximum theoretical growth rate and the maximum theoretical production flux of styrene. Therefore, we run a flux based analysis twice on the model, using first biomass growth as objective function, and secondly optimizing for styrene production:  </P><P><I>Table 2: FBA results </I></P><TABLE><TBODY><TR><TD>Input flux of glucose (goal = biomass)</TD><TD>1 mmol gDWh<SUP>-1</SUP></TD></TR><TR><TD>Max biomass growth</TD><TD>0.097 mmol gDWh<SUP>-1</SUP></TD></TR><TR><TD>Input flux of glucose (goal = styrene)</TD><TD>0.359 mmol gDWh<SUP>-1</SUP></TD></TR><TR><TD>Max output flux of styrene</TD><TD>0.00213 mmol gDWh<SUP>-1</SUP></TD></TR></TBODY></TABLE><P>The analysis showed us that without additional interventions we maximally produce about one mole of styrene per 167 moles of glucose meaning that 0.35% of the mass of glucose can be turned into styrene.  This means that we if we would grow the yeast cells in a medium of 20 g glucose/L, and all glucose is taken up by the cell, the final concentration of styrene will reach at its highest a concentration of <B>70 mg/L</B>. This is well within toxicity levels[7] (Mckenna et al. found that growth was only minimally disrupted with styrene levels of 200 mg/L). However if we increase styrene yield with over expressions/knockouts we will have to look at minimising toxicity, for instance with a system that continuously takes styrene out of the medium </P><P>Next we tested how styrene production influences cell growth. To this end we ran FBA, optimizing for biomass while iteratively constraining the lower bound of the styrene sink reaction from 0 flux to its max flux of 0.00213 mmol gDWh-1. The result of this analysis is seen in the figure 4. The script used for analysis is available on <A href="https://github.com/Ezmagon/igem_modeling/tree/master/igem_GECKO">Github</A></P><FIGURE><FIGCAPTION><I>Figure 4</I></FIGCAPTION></FIGURE><P>As seen in the figure the cells can continue to grow at almost optimal rate when the flux through the styrene pathway is at 65% of its theoretical maximum. We can therefore argue that there is no need to create an inducible system in the cells, but rather let the cells produce styrene continuously, since styrene production (up to a reasonable level) barely hinders cell growth. However, it shows that we should be careful with the chosen promoters for gene expression to not overburden the system with too high expression which could hinder the growth (and as a consequence also styrene production).</P><H4>OptForce</H4><P>Next we looked at how the styrene production can be optimized further in the cells. We used an algorithm called OptForce8, which has been employed successfully to find interventions leading to overproduction of a target metabolite. OptForce compares the flux patterns observed in an initial wildtype strain and a mutant strain which overproduces the chemical at the target yield. In order to compare the two strains it makes use of the Flux Variance Analysis (FVA). As noted before the steady state equation usually results in a solution space, meaning that there is a whole range of possible fluxes that satisfy the steady state criterion. With FVA the reactions in the network are iteratively minimized and maximised subject to the constraints. This creates a lower and upper bound for every reaction: a flux range for which the reaction can still satisfy all constraints.</P><P>The key of OptForce is identification of so called MUST sets (sets of fluxes that <I>must</I> change in order to reach the target optimization). The flux ranges for every reaction is first calculated for both the wildtype strain which is constrained to grow at a certain level and the mutant strain which is constrained to produce a target chemical at a certain level. Next it compares the flux ranges of both strains for every reaction. If the flux ranges have some overlap (figure 5a), the target production is possible without having to intervene in this reaction. However if the flux range for a reaction of the wildtype strain are completely to the left (Figure 5b) or to the right (Figure 5c) of the mutant strain then we should directly or indirectly change the flux of this reaction in order to be able to achieve the overproduction target. Reaction fluxes that must increase are referred to as MUSTU sets while reaction fluxes that must decrease are referred to as MUSTL sets. Furthermore we also look at the sums of two reaction fluxes and compare the resulting flux range between the wildtype and mutant type. Now non-overlapping flux ranges imply that one of the two reaction flux must increase or decrease. These second order sets are referred to as respectively MUSTUU and MUSTLL  sets.  </P><FIGURE><FIGCAPTION><I>Figure 5: MUSTsets (wildtype flux range for a reaction in green, mutant-type flux range in orange)</I></FIGCAPTION></FIGURE><P>Finally after identifying the MUST sets the OptForce algorithm looks at how these can be imparted on the wildtype strain with the minimum amount of interventions. In other words it finds the minimum amount of upregulations or knockouts which will make the wildtype produce the target chemical at the same rate as the mutant strain.  This set of interventions that must actively be <I>forced</I> through genetic manipulations is referred to as the FORCE set. </P><H4>Goal 2</H4><P>The OptForce algorithm was used with YeastGem8 as a wildtype model. As it is suggested to exclude reactions in a linear pathway leading up to the target chemical we will try to find interventions leading to an increased phenylalanine production, precursor of styrene. The wildtype model had its growth constrained to 0.07 mmol gDWh <SUP>-1</SUP> and the mutant type had phenylalanine production constrained to 0.5 mmol gDWh<SUP>-1</SUP>, both close to their maximum values. In order to solve the optimization problem IBM ILOG CPLEX Optimizer  (v 12.8.0) was used.  </P><P>As finding the second order MUST sets is computationally heavy we use the cluster (see introduction page) to run the algorithm. The complete output in the form of the first and second order MUST sets and FORCE sets together with the algorithm that was run to find them can be found on our <A href="https://github.com/Ezmagon/igem_modeling/tree/master/igem_optforce">Github page</A>. Note that we excluded transport, exchange and ATP related reactions as we do not want those to end up in the FORCE sets</P><P>Next we visualized the found MUST sets; the candidates for downregulation/knockouts or upregulations, using Paint4Net (included in the Cobra Toolbox). Paint4Net is able to automatically generate a graph layout of (part of) the metabolism of a COBRA model, including data such as flux rate and direction, and the possibility to show detailed information about reactions and metabolites.[9]  </P><P>As the model consists of over 3500 reactions visualizing the whole map at once would not be informative, chose to visualize the cellulars metabolism per subsystem. Here we will show the map of: Phenylalanine, tyrosine and tryptophan biosynthesis (KEGG pathway sce00400) and Phenylalanine metabolism (KEGG pathway sce00360). This part seems most relevant to our analysis and allows us to compare interventions found by the model with well-established interventions from literature. (Do note that our method allows one to visualize any subsystem).</P><P>Once the Paint4net map was created the MUST sets had to be visualised. For this a simple MATLAB script was written  that goes Once the Paint4net map was created the MUST sets were visualised. For this a simple MATLAB script was written  that goes over all nodes and edges in the network and makes them red if they correspond to a reaction in MUSTL  or MUSTLL sets (candidatse for downregulation/knockout) , green if they belong to a MUSTU  or MUSTUU set  (are a candidate for upregulation) and blue if they belong to no MUST set (should be left unchanged). Furthermore to make a clearer visualization the script changes the labels of the reaction nodes to display the corresponding genes and the metabolite of interest is coloured in orange. The algorithm <A href="https://github.com/Ezmagon/igem_modeling/blob/master/igem_optforce/visualization.m">visualization.m</A> is available on our Github. The result of applying this to the map of the subsystem described above can be seen in figure 6. </P><B> mouse over the image!</B><P>Furthermore we verified our model predictions by comparing the computed interventions with literature. McKenna et al. showed that Styrene yield can be increased by deletion of ARO10.[7] The reaction catalysed by ARO10 (phenylpyruvate decarboxylase) is indeed marked in the graph as a possible candidate for downregulation (included in a MUST¬L set) . Looking at the graph we noticed that our model additionally predicts the genes ARO1, ARO2 and ARO7, that are part of `the shikimate pathway in S. cerevisiae as possible candidates for upregulation. This is in accordance with earlier research showing that styrene production in E. Coli can be increased by overexpressing the two upstream shikimate pathway genes, aroF and pheA. [10]</P><P>Next we look at the found force sets:</P><B>
set1: PHA2 (upregulation)    ARO10    (Knockout) 
set2: PHA2 (upregulation)    ARO10    (downregulation)
set3: ARO7 (upregulation)    FOX2     (knockout)
set4: ARO1 (upregulation)    IFA38    (knockout) </B><P>The first two FORCE set include either a knockout or the downregulation of ARO10, which is as noted before a well-established way to increase Styrene yield yeast 7  Furthermore these sets both increase the upregulation of PHA2, which encodes Putative prephenate dehydratase involved in the catalysis of phenylpyruvate a precursor to phenylalanine. The other two FORCE sets include the upregulation of ARO7 and ARO1, which are as noted part of the shikimate pathway. They also include respectively FOX2 knockout, encoding an enzyme involved in Fatty acid Oxidation and IFA38 knockout evolved in  pathway fatty acid biosynthesis. These reactions are bit less intuitive, and further experimental analysis should be done to see if those interventions would actually increase phenylalanine production and are not just the results of an under/over constrained model.  
</P><P>To summarize we have first shown, using a genome scale model with enzyme based constraints based on the GECKO method, that cell growth and styrene production (up to a certain level) are not mutually competitive, leading us to design cells that will express necessary enzymes constitutively. Furthermore we used the OptForce algorithm to find interventions that increase phenylalanine yield. Finally we wrote a script that can visualize these interventions on a graph drawn by the Paint4Net method, allowing us to gain more insight in how we can increase styrene production in our cells.   </P><H4>References</H4><P>[1] Smallboe K, Simeonidis E. Flux balance analysis: A geometric perspective. 2009. doi:10.1016/j.jtbi.2009.01.027 </P><P>[2] Aung HW, Henry SA, Walker LP. Revising the Representation of Fatty Acid, Glycerolipid, and Glycerophospholipid Metabolism in the Consensus Model of Yeast Metabolism. Ind Biotechnol (New Rochelle N Y). 2013;9(4):215-228. doi:10.1089/ind.2013.0013</P><P>[3] Sánchez BJ, Zhang C, Nilsson A, Kerkhoven EJ, Nielsen J, Lahtvee P. Improving the phenotype predictions of a yeast genome-scale metabolic model by incorporating enzymatic constraints. 2017:1-16. doi:10.15252/msb.20167411</P><P>[4] Bateman A, Martin MJ, O’Donovan C, et al. UniProt: the universal protein knowledgebase. Nucleic Acids Res. 2017;45(D1):D158-D169. doi:10.1093/nar/gkw1099</P><P>[5] Cochrane FC, Davin LB, Lewis NG. The Arabidopsis phenylalanine ammonia lyase gene family: kinetic characterization of the four PAL isoforms. Phytochemistry. 2004;65(11):1557-1564. doi:10.1016/j.phytochem.2004.05.006</P><P>[6] Heirendt L, Arreckx S, Pfau T, et al. Creation and analysis of biochemical constraint-based models: the COBRA Toolbox v3.0. October 2017. http://arxiv.org/abs/1710.04038. Accessed October 6, 2018.</P><P>[7] Mckenna R, Thompson B, Pugh S, Nielsen DR. Rational and Combinatorial Approaches to Engineering Styrene Production by Saccharomyces Cerevisiae.; 2011. doi:10.1186/2046-1682-4-13</P><P>[8] Ranganathan S, Suthers PF, Maranas CD. OptForce: An Optimization Procedure for Identifying All Genetic Manipulations Leading to Targeted Overproductions. Price ND, ed. PLoS Comput Biol. 2010;6(4):e1000744. doi:10.1371/journal.pcbi.1000744</P><P>[9] Kostromins A, Stalidzans E. Paint4Net: COBRA Toolbox extension for visualization of stoichiometric models of metabolism. Biosystems. 2012;109(2):233-239. doi:10.1016/J.BIOSYSTEMS.2012.03.002</P><P>[10] Liu C, Men X, Chen H, et al. A systematic optimization of styrene biosynthesis in Escherichia coli BL21(DE3). Biotechnol Biofuels. 2018;11. doi:10.1186/s13068-018-1017-z</P></DIV></DIV></DIV></DIV></DIV><FOOTER><H4 class="center">We thank our sponsors:</H4></FOOTER></DIV></DIV></DIV></BODY></HTML>