<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGem Groningen 2018</TITLE></P><DIV class="navbar-fixed"><NAV class="row green"><UL id="dropdown-project" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Description" class="green-text">Description</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Design" class="green-text">Design</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Results" class="green-text">Results</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Demonstrate" class="green-text">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Entrepreneurship" class="green-text">Entrepreneurship</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Awards" class="green-text">Awards</A></LI></UL><UL id="dropdown-notebook" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Notebook" class="green-text">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Protocols" class="green-text">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Safety" class="green-text">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Measurement" class="green-text">Measurement</A></LI></UL><UL id="dropdown-parts" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Parts" class="green-text">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Primers" class="green-text">Primers</A></LI></UL><UL id="dropdown-HP" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Human_Practices" class="green-text">Integrated Human Practices (Gold)</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Public_Engagement" class="green-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Applied_Design" class="green-text">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Ethics" class="green-text">Ethics</A></LI></UL><UL id="dropdown-attribution" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Attributions" class="green-text">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Sponsors" class="green-text">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Crowdfunding" class="green-text">Crowdfunding</A></LI></UL><UL id="dropdown-help" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/InterLab" class="green-text">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Software" class="green-text">Software</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Collaborations" class="green-text">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Embassy" class="green-text">Consulate Meetup</A></LI></UL><UL id="dropdown-team" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Team" class="green-text">Team</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Supervisors" class="green-text">Supervisors</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Team_Building" class="green-text">Team Building</A></LI></UL><UL id="dropdown-model" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Model" class="green-text">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Molecular_Dynamics" class="green-text">Proximity by affinity</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Mathematical_Modeling" class="green-text">Cellulose Degradation</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Flux_Based_Analysis" class="green-text">Optimizing StyGreen Production</A></LI></UL><DIV class="nav-wrapper" id="nav"><UL class="left" id="nav-buttons"><LI><A href="https://2018.igem.org/Team:Groningen" class="waves-effect waves-light block"><I class="material-icons">home</I></A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-project">Project</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-notebook">Lab</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-model">Model</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-parts">Sequences/Parts</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-HP">Human Practices</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-help">Contributions</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-team">Team</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-attribution">Acknowledgements</A></LI></UL><UL class="right" id="contact"><LI title="Click to visit our Facebook page"><A target="_blank " href="https://www.facebook.com/igemgroningen2018/" class="menu-hover waves-effect waves-light block"><SPAN>Groningen.iGEM2018</SPAN></A></LI><LI title="Click to e-mail us"><A target="_blank" href="mailto:igem2018rug@gmail.com" class="menu-hover waves-effect waves-light block"><SPAN>igem2018rug@gmail.com</SPAN></A></LI><LI><A class="menu-hover" id="cross"><I class="material-icons">close</I></A></LI></UL></DIV></NAV></DIV><A href="#!" class="btn-floating btn-large waves-effect waves-light green" id="menu-btn"><I class="material-icons">menu</I></A><DIV class="igem_2018_team_content" style="background-color: white"><DIV class="igem_2018_team_column_wrapper"><UL class="collapsible popout"><LI><DIV class="collapsible-header">Week 25</DIV></LI><LI><DIV class="collapsible-header"> Thursday June 21st </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Preparation of 200 ml YPD and 200 ml YPD with agar. Preparation of 200 ml LB medium and 200 ml LB medium with 2% agar</P><P>For preparation of 200 ml YPD add to demi water:
            - 2 g yeast extract
            - 4 g peptone
            - 4 g glucose  </P><P>For preparation of 200 ml YPD 2% agar add to demi water:
            - 2 g yeast extract
            - 4 g peptone
            - 4 g glucose
            - 4 g agar</P><P>For preparation of 200 ml LB add to demi water:
            - 4 g LB</P><P>For preparation of 200 ml LB 2% agar add to demi water:
            - 4 g LB
            - 4 g agar</P></DIV></LI><LI><DIV class="collapsible-header"> Friday June 22nd </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Prepare 4 ml 1000x ampicillin stock (100mg/ml in MQ)</P><P>- Add 400 mg ampicillin to 4 ml MQ and filter sterilize. 
            - Aliquot and store at - 20 degrees C.  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Pour YPD plates and LB+Amp plates</P><P>Autoclave media at 121 degrees C for 15 minutes and cool to hand warm.</P><P>- Add 200 ul 1000x ampicillin to 200 ml LB medium with agar.
            - Pour LB + Amp plates.
            - Pour YPD plates  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Grow strain BY4741 for cloning</P><P>Strain provided by Molecular Microbiology group RUG. - Plate BY4741 on YPD and grow over the weekend at 30 degrees. 
            25-6-18 Colonies visible for BY4741 on YPD plate. </P></DIV></LI></UL></DIV><LI><DIV class="collapsible-header">Week 26</DIV></LI><LI><DIV class="collapsible-header"> Monday June 25th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Preparation of 400 ml SOP Verduyn medium with glucose and no pH adjustment </P><P>Prepare 400 ml SOP verduyn medium with glucose and 400 ml SOP verduyn medium with glucose and 2% agar (8g). See protocol: <A href="https://2018.igem.org/Team:Groningen/Protocols#LB-plates" target="_blank">Verduyn medium</A></P><P>Add 250 x G418 1,6 ml and 4ml 100x trace element solution to Verduyn medium with agar and pour plates.
            + Burner went off during pouring of plates.
            + Vitamin solution was not added.</P><P><B>Who: Owen</B></P><P><B>Aim</B>  Restreak DH5α pYD1-CipA1-EGII, pYD1-CipA3-EGII and DH5alpha pRS425-BGLI-CBHII on LB + Amp plates because previous plates were more than 1 month old and grow overnight at 37 degrees C</P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate a colony BY4741 in 20 ml YPD and grow overnight at 30 degrees</P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday June 26th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Take 50 ul overnight culture BY4741 add to 20 ml YPD and grow overnight at 30 degrees</P></DIV></LI><LI><DIV class="collapsible-header"> Thursday June 28th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Prepare G418 Verduyn plates with uracil, methionine, leucine and histidine</P><P>Add 2x amino acid solution and vitamin solution to 3 x Verduyn G418 plates: 400 ul 100x URA, MET, LEU solution, 80 ul 500x His solution and 40 ul 1000x vitamin solution. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate BY4741 for transformation</P><P>Take 50 ul overnight culture BY4741, add to 20 ml YPD and grow overnight 30 degrees for transformation. </P></DIV></LI><LI><DIV class="collapsible-header"> Friday June 29th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of p414 KanMX TEF1-Cas9-Tcyc into BY4741</P><P>See protocol: <A href="https://2018.igem.org/Team:Groningen/Protocols#Agarose-gel" target="_blank">High efficiency yeast transformation of intact yeast cells</A> v2. Altered steps or specifics for a step are mentioned below. </P><P>- step 3: OD600 was not measured but estimated to be around 15. 
            - step 5: cells washed 2x in 10 ml dH2O. 
            - step 8: cells resuspended in 0,4 ml 0.1M LiAc and divided over 4 tubes. 
      - step 10: Transformation: 10 ul pDNA, 10 ul ssDNA and 55 ul dH2O as DNA mix. Control: 10 ul ssDNA and 65 ul dH2O.
            - step 16:  plate 100 ul and centrifuge and resuspend remaining 900 ul and plate on Verduyn G418 plates.</P><P>Plasmid DNA contains precipitate. </P></DIV></LI></DIV><LI><DIV class="collapsible-header">Week 27</DIV></LI><LI><DIV class="collapsible-header"> Monday July 2nd </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate BY4741</P><P>Transformation plates of BY4741 with p414 KanMX TEF1-Cas9-Tcyc into BY4741 from 29-6-18 contain a lot of background as the plates were not completely dry and OD600 of transformation culture was very high. Colonies are visible, to increase colony size plates put at 30 degrees C for 1 more day. A culture for a second transformation is prepared in case the first one failed. </P><P>Inoculate 2x colony BY4741 in 20 ml YPD and grow overnight at 30 degrees C.</P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday July 3rd </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Prepare G418 Verduyn plates with uracil, methionine, leucine (and histidine)</P><P>Add 2x amino acid solution and vitamin solution to 6 x Verduyn G418 plates. 400 ul 100x URA, MET, LEU solution, 80 ul 500x His solution and 40 ul 1000x vitamin solution. </P><P>Add 2 amino acid solution and vitamin solution to 3 x Verduyn G418 plates. Creating verduyn G418 –His plates. 400 ul 100x URA, MET, LEU solution and 40 ul 1000x vitamin solution.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Transform p414 KanMX TEF1-Cas9-Tcyc into BY4741</P><P>See protocol: <A href="https://2018.igem.org/Team:Groningen/Protocols#Agarose-gel" target="_blank">High efficiency yeast transformation of intact yeast cells</A> v2. Altered steps or specifics for a step are mentioned below. </P><P>- step3: OD600 of overnight culture measured: approx. 15. - step 4: Spin down 5 ml culture. - step 5: wash 1x with 25 ml dH2O. - step 10: Transformation: 10 ul pDNA, 10 ul ssDNA and 55 ul dH2O as DNA mix. No negative control used. - step 16:  plate 100 ul and centrifuge and resuspend remaining 900 ul and plate on Verduyn G418 plates.</P><P>On 8-7-18 colonies are visible and plate is transferred to fridge.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Verify quality of p414 KanMX TEF1-Cas9-Tcyc</P><P>To check p414 KanMX TEF1-Cas9-Tcyc with precipitate, it is run on a 1% agarose with Serva stain G at 90 V for 35 min. </P><P><B>Results</B></P><P>Big band is visible confirming that the DNA is still ok. Gel not shown.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Transform p414 KanMX TEF1-Cas9-Tcyc into DH5alpha to amplify plasmid</P><P>- Thaw competent cells 300 ul. - Add 1 ul p414 KanMX TEF1-Cas9-Tcyc. - Put cells on ice for 5 minutes. - Put cells at 1 min for 42 degrees C. - Put cells on ice for 5 min. - Add 1 ml LB to cells and grow at 37 degrees C for 30 min. - Plate 50 ul culture on a LB+Amp plate. - Grow cells overnight at 37 degrees.</P><P>Next day: p414 KanMX TEF1-Cas9-Tcyc to DH5alpha, many colonies visible.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Verify transformation of p414 KanMX TEF1-Cas9-Tcyc into BY4741 by Colony PCR</P><P>- Add 10x colony to 15 ul demiwater. - Put colony mix at 95 degrees C for 10 minutes. - Prepare 11x Phire Mastermix: 137,5 ul Phire green, 5,5 ul primer FW 588, 5,5 ul primer RV 589, 104, 5 ul demiwater. - Add 23 ul mastermix to 10 x PCR tube. - Add 2 ul lysed colony mix to each PCR tube. </P><P>Run PCR protocol:</P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>03:00</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>20s</TD></TR><TR><TD>3</TD><TD>58°C</TD><TD>20s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>90s</TD></TR><TR><TD>25x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>03:00</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE></DIV></LI><LI><DIV class="collapsible-header"> Wednesday July 4th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Run colony PCR transformation 29-6 on gel</P><P>- Pour 2 % agarose gel containing Serva DNA stain G. - load 10 ul sample, 5 ul ladder 1kB O’generuler thermofisher. - Run gel 20 min 90 V. </P><P><B>Results</B></P><P>Band should be visible at 151 bp, no bands visible. Apparent bands in middle are caused by gel casting tray</P><P><B>Who: Owen</B></P><P><B>Aim</B>  Restreak 6 transformation colonies 29-6 on Verduyn G418 plates to get more apparent colonies</P><P>- Restreak 6 transformation colonies 29-6 on Verduyn G418 plates. - Grow overnight 30 degrees C. </P></DIV></LI><LI><DIV class="collapsible-header"> Friday July 6th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne &amp; Phillip</B></P><P><B>Aim</B> Competent cell test</P><P>To test the competence of our cells, we performed a competent cell test according to the IGEM Competent Cell test kit <A href="http://parts.igem.org/Help:2018_Competent_Cell_Test_Kit" target="_blank">protocol</A>.</P><P>We dissolved the dried-down purified plasmid DNA from <A href="http://parts.igem.org/Part:BBa_J04450" target="_blank">BBa_J04450</A> into 50 µl of water to obtain 100 pg/µl and 10 pg/µl concentrations. In short, 1 µl of each DNA concentration was added to 50 µl of competent cells in duplicates. We obtained plenty of colonies and the efficiency of transformation was determined sufficient. This batch of competent E. coli DH5ɑ cells was used for all of the following E. coli transformations.  </P></DIV></LI><LI><DIV class="collapsible-header"> Sunday July 8th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Colony PCR on restreaked colonies 29-6 a to f and colonies 1 to 6 yeast transformation 3-7 to verify transformation of p414 KanMX TEF1-Cas9-Tcyc into BY4741</P><P>- Add 12x colony to 15 ul demiwater. - Put at 95 degrees C for 10 minutes. - Prepare 14x Phire Mastermix: 175 ul Phire green, 7 ul primer FW 588, 7 ul primer RV 589, 133 ul demiwater. - Add 23 ul mastermix to 12 x PCR tube. - Add 2 ul lysed colony mix to each PCR tube. - cut off left, caps down left, 1 to 6. - cut off right, caps down left, a to f. </P><P>Run PCR protocol:</P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>03:00</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>20s</TD></TR><TR><TD>3</TD><TD>58°C</TD><TD>20s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>15s</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE><P>- pour 2% agarose gel. - Load 8 ul for each colony PCR sample. - Load 5 ul ladder 1kB O’Generuler thermofisher. - Run gel 20 min 90V.</P><P><B>Results</B></P><P>Expected band size: 151bp. Lane 2-7: colony 1-6. Lane 9-14: colony a-f. Colony 1 and 3 appear to be the expected size, but the bands are vague. </P></DIV></LI></DIV><LI><DIV class="collapsible-header">Week 28</DIV></LI><LI><DIV class="collapsible-header"> Monday July 9th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne, Phillip &amp; Owen</B></P><P><B>Aim</B> Production of competent E. coli DH5ɑ </P><P><A href="https://2018.igem.org/Team:Groningen/Protocols#Verduyn">Standard Protocol For Competent Cell Production</A> Was used.</P><P>A sample of E. coli DH5ɑ was kindly provided by Niels de Kok and grown overnight in LB.</P><P>CCMB80 medium - 10 mM KoAc pH 7.0 - 80 mM CaCl2  - 20 mM MnCl2  - 10 mM MgCl2 - 10% glycerol</P><P>CCMB80 was filter sterilized and stored at 4℃</P></DIV></LI><LI>Seed cells were inoculated into 250 mL of LB and grown overnight, to OD600 of 0.3.</LI><LI>Cells were transferred to pre-chilled weighted flat-bottomed tubes and weighed.</LI><LI>Tubes were centrifuged at 3000g for 10 minutes, supernatant was thrown away</LI><LI>Cells were resuspended in 80 mL of ice-cold CCMB80 medium and incubated on ice for 20 minutes. </LI><LI>Step 3 was repeated.</LI><LI> 200uL of LB and 50uL of cells were mixed and CCMB80 was added to reach a final OD of 1.0-1.5</LI><LI>Mixture was incubated on ice for 20 minutes and then measured out into aliquots of 300uL</LI><LI>Aliquots of cells were kept at -80 degrees celsius indefinitely.</LI></DIV><LI><DIV class="collapsible-header"> Tuesday July 10th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne &amp; Phillip</B></P><P><B>Aim</B> Transformation of competent E. coli DH5ɑ with the devices for the iGEM Interlab study</P><P>The next devices were provided to us by iGEM in the plates in the distribution kit. The following devices were resuspended into 10 µl sterilized deionized water. </P><TABLE><TBODY><TR><TH>Device</TH><TH>Location on plate 7</TH></TR><TR><TD>Negative control</TD><TD>2D</TD></TR><TR><TD>Positive control</TD><TD>2B</TD></TR><TR><TD>Device 1</TD><TD>2F</TD></TR><TR><TD>Device 2</TD><TD>2H</TD></TR><TR><TD>Device 3</TD><TD>2J</TD></TR><TR><TD>Device 4</TD><TD>2L</TD></TR><TR><TD>Device 5</TD><TD>2N</TD></TR><TR><TD>Device 6</TD><TD>2P</TD></TR></TBODY></TABLE><P>1 µl of this solution was transformed into 50 µl of competent DH5ɑ cells using the following protocol:</P></DIV></LI><LI>LB plates with the appropriate antibiotics were prepared as described here (link to our protocol page LB agar plates)</LI><LI>Eppendorf tubes (1.5 ml) were pre-chilled on ice and competent cells were thawed on ice</LI><LI>50 µl of competent cells was added to the pre-chilled eppendorf tubes, together with the DNA</LI><LI>30 min incubation on ice</LI><LI>Heat shock for 45 sec in a 42℃ water bath</LI><LI>Incubation on ice for 5 min</LI><LI>950 µl of LB broth was added to the cells</LI><LI>Incubation for 1 hour, 37℃, 200 rpm</LI><LI>Plating of the cells on the LB-agar plates. 100 µl one one plate, then the culture is centrifuged and the supernatant is partially discarded. The cell pellet is resuspended in approximately 100 µl and plated on a separate plate. </LI><LI>Overnight incubation at 37℃ (agar side up)</LI></DIV><LI><DIV class="collapsible-header"> Wednesday July 11th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne </B></P><P><B>Aim</B> Growth of the colonies used in the iGEM Interlab study </P><P>We obtained between 10-100 colonies per transformation. Two colonies were picked with a sterile pipette tip and transferred into 5 ml of LB with chloramphenicol. The cultures were grown in the appropriate broth overnight at 37℃ in a shaking incubator at 200 rpm. </P><P><B>Who: Rianne</B></P><P><B>Aim</B> To purify the plasmids from Wen. et al out of E. coli for transformation into yeast</P></DIV></LI><LI>1882: PYD1 - CipA1 - EGII</LI><LI>1883: PYD1 - CipA3 - EGII</LI><LI>CB: PRS425 - CBHII - BGLI</LI><P> These three plasmids that we received from Wen. et al were kindly transformed by Vakhil Takhaveev into E. coli and plated onto LB agar plates. In order to purify the plasmids out of these cells, I grew three colonies of each strain into 10 ml of LB with the appropriate antibiotics. The next day, the plasmid was extracted from the full-grown cultures using a plasmid extraction kit. The following concentrations (ng/µl) were obtained: </P><P><B>Results</B></P><TABLE><TBODY><TR><TH>1882</TH><TH>1883</TH><TH>CB</TH></TR><TR><TD>1</TD><TD>278,15</TD><TD>572,80</TD><TD>443,80</TD></TR><TR><TD>2</TD><TD>664,10</TD><TD>98,0</TD><TD>361,70</TD></TR><TR><TD>3</TD><TD>321,25</TD><TD>652,0</TD><TD>439,95</TD></TR></TBODY></TABLE><P>The plasmids were stored at -20℃ until further use.</P></DIV><LI><DIV class="collapsible-header"> Thursday July 12th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Production of glycerol stocks for the strains used in the iGEM Interlab study</P><P> 250 µl of culture was mixed with 250 µl of 50% glycerol and stored at -80℃. Simultaneously, a small amount of the glycerol stock was transferred into 5 ml of fresh medium supplemented with chloramphenicol and grown overnight at 37℃, 200 rpm. </P><P><B>Who: Owen</B></P><P><B>Aim</B>  Colonies 1,3 and 5 from transformation 3-7 and colonies d and f from 29-6 transformation are restreaked on Verduyn+ G418 plates</P><P><B>Who: Owen</B></P><P><B>Aim</B> Verify presence of p414-KanMX-Cas9 by colony PCR</P><P>As previous colony PCR was inconclusive, because of the vague bands, a second colony PCR with more colony material is performed to check the most promising colonies from the last colony PCR: Colonies 1,3,5, d and f. - Add a lot of colony material to 15 ul MQ each. - Boil colony mix 95 degrees C for 10 min. - Prepare 9 MM without MQ: 112,5 ul phire green, 4,5 ul primer FW 588, 4,5 ul primer RV 589. - Add 13,5 ul to 8x a PCR tube. - Add 1, 2, 4 ul template solution of col 1 and fill the respective PCR tube up to 25 ul. - Add 4 ul template for colonies 3, 5, d and f and fill the respective PCR tube up to 25 ul. - Prepare positive control: 0,5 ul primer FW 588, 0,5 ul 589 primer RV 589, 2 ul p414-Cas9-KanMx 100x diluted, 9,5 ul MQ and 12,5 ul Phire green mix. - Run PCR protocol:   </P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>03:00</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>20s</TD></TR><TR><TD>3</TD><TD>58°C</TD><TD>20s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD>6</TD><TD>4°C</TD><TD>till end</TD></TR></TBODY></TABLE><P>- Prepare 2,5% agarose gel: solubilize 2,5 gr agarose in 100 ml 1x TAE in microwave, wait till handwarm and add 1 ul 100.000x Serva DNA stain. - Pour 2,5 % agarose gel. - Load 10 ul of each colony PCR sample. - Load 10 ul p414-KanMX-Cas9 as a positive control. - Load 5 ul 1kB ladder. - Run gel 90V 10 min.</P><P><B>Results</B></P><P>Expected band size 151 bp. Lane 2-9 from left to right: col 1 1ul, col 1 2ul, col 1 4ul, col 3, col 5, col d, col f, p414-KanMX-Cas9. As bands of colony 1 and 3 look similar to the control band in shape the presence of the plasmid is confirmed in colony 1 and 3. </P><P><B>Who: Owen</B></P><P><B>Aim</B> PCR reaction to amplify pMEL16 without target DNA sequence to create Aga1, Aga2 and Aro10 gRNA plasmids</P><P>- Prepare 2x 50 ul PCR mix phire green: + 50 ul phire green buffer. + 2 ul primer FW 6006  + 2 ul primer RV 6005 + 4 ul pMEL16 OR1 + 42 ul MQ. - Add 2 x 50 ul to a PCR tube.  </P><P>- Prepare 2x 50 ul PCR mix Phusion: + 50 ul phusion buffer. + 2 ul primer FW 6006 + 2 ul primer RV 6005 + 4 ul pMEL16 OR1 + 42 ul MQ.</P><P>- Run PCR protocol Phusion:</P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>00:30</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD>3</TD><TD>62°C</TD><TD>25s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>04:00</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>06:00</TD></TR><TR><TD>6</TD><TD>4°C</TD><TD>till end</TD></TR></TBODY></TABLE><P>- Run PCR protocol Phire:</P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>03:00</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>20s</TD></TR><TR><TD>3</TD><TD>62°C</TD><TD>20s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>04:00</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>06:00</TD></TR><TR><TD>6</TD><TD>4°C</TD><TD>till end</TD></TR></TBODY></TABLE></DIV></LI><LI><DIV class="collapsible-header"> Friday July 13th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne &amp; Phillip</B></P><P><B>Aim</B> First cell measurement for the iGEM Interlab study</P></DIV></LI><LI>500 µl of the overnight culture was transferred into 4,5 ml of LB with chloramphenicol </LI><LI>OD600 measurement of 1:10 dilutions</LI><TABLE><TBODY><TR><TH>Colony 1</TH><TH>Colony 2</TH></TR><TR><TD>Negative control</TD><TD>0.191</TD><TD>0.192</TD></TR><TR><TD>Positive control</TD><TD>0.198</TD><TD>0.204</TD></TR><TR><TD>Device 1</TD><TD>0.152</TD><TD>0.149</TD></TR><TR><TD>Device 2</TD><TD>0.189</TD><TD>0.208</TD></TR><TR><TD>Device 3</TD><TD>0.19</TD><TD>0.191</TD></TR><TR><TD>Device 4</TD><TD>0.15</TD><TD>0.171</TD></TR><TR><TD>Device 5</TD><TD>0.094</TD><TD>0.109</TD></TR><TR><TD>Device 6</TD><TD>0.204</TD><TD>0.195</TD></TR></TBODY></TABLE><LI>Dilution of these 1:10 cultures to target OD600 0.2 in a final volume of 12 ml</LI><P>10 ml of culture was already pipetted into the 50 ml falcon tubes for all cultures, except for cultures of device 5, for which 9 ml was used. To these tubes, the following volumes (µl) were added:</P><TABLE><TBODY><TR><TH>Colony 1</TH><TH>Cell culture</TH><TH>LB</TH><TH>Colony 2</TH><TH>Cell culture</TH><TH>LB</TH></TR><TR><TD>Negative control</TD><TD>1257</TD><TD>743</TD><TD>Negative control</TD><TD>1249</TD><TD>751</TD></TR><TR><TD>Positive control</TD><TD>1212</TD><TD>788</TD><TD>Positive control</TD><TD>1176</TD><TD>824</TD></TR><TR><TD>Device 1</TD><TD>1579</TD><TD>421</TD><TD>Device 1</TD><TD>1611</TD><TD>389</TD></TR><TR><TD>Device 2</TD><TD>1269</TD><TD>731</TD><TD>Device 2</TD><TD>1154</TD><TD>846</TD></TR><TR><TD>Device 3</TD><TD>1263</TD><TD>737</TD><TD>Device 3</TD><TD>1257</TD><TD>743</TD></TR><TR><TD>Device 4</TD><TD>1599</TD><TD>401</TD><TD>Device 4</TD><TD>1404</TD><TD>596</TD></TR><TR><TD>Device 5</TD><TD>2553</TD><TD>447</TD><TD>Device 5</TD><TD>2202</TD><TD>798</TD></TR><TR><TD>Device 6</TD><TD>1176</TD><TD>824</TD><TD>Device 6</TD><TD>1231</TD><TD>769</TD></TR></TBODY></TABLE><LI>1000 µl of the 0 hour time point culture was pipetted into an eppendorf tube and kept on ice until further use in a box with ice at 4℃.</LI><LI>Six hours of incubation at 37℃, 200 rpm</LI><LI>1000 µl of the 6 hour time point culture was pipetted into an eppendorf tube and kept on ice until further use in a box with ice at 4℃.</LI><LI>100 µl of these cell cultures were transferred into wells of a 96-microtiter plate.</LI><P>However, while analyzing our results, we realized that we did not set the ‘gain’ setting for fluorescence measurement in the plate reader to a fixed number across the different measurements. Therefore, we need to repeat this measurement. </P><P><B>Aim</B> Correlate OD600 measurements in our plate reader to colony forming units (CFU)</P><P>We first diluted the overnight cultures of the negative and positive control 1:8 and measured OD600 in our plate reader. We then further diluted these cultures to target OD600 0.1, confirmed by our plate reader and plated onto plates in several dilutions.</P><P><B>Aim</B> Calibration experiments for the Interlab study</P><P>LUDOX, silica beads and fluorescein experiments as described on our interlab page were also performed. As described above, the fluorescein calibration needs to be repeated.</P><P>The exact protocol can be found <A href="https://static.igem.org/mediawiki/2018/0/09/2018_InterLab_Plate_Reader_Protocol.pdf" target="_blank">here</A> and the results of these experiments can be found on our <A href="https://2018.igem.org/Team:Groningen/InterLab" target="_blank">Interlab page</A>.</P></DIV><LI><DIV class="collapsible-header"> Saturday July 14th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne &amp; Phillip</B></P><P><B>Aim</B> Counting colony forming units</P><P>We checked the colonies and counted them, to calculate how many cells were present in our culture of which the plate reader displayed the OD600 to be 0.1. The following numbers of colonies were found, where + and - stand for positive and negative control, respectively, and 1 and 2 stand for the two colonies that were picked at day 11-7-18. (4) and (5) stand for the dilutions plated as performed on 13-7-18. </P><TABLE><TBODY><TR><TH>+1(4)</TH><TH>+1(5)</TH><TH>+2(4)</TH><TH>+2(5)</TH><TH>-1(4)</TH><TH>-1(5)</TH><TH>-2(4)</TH><TH>-2(5)</TH></TR><TR><TD>1</TD><TD>181</TD><TD>11</TD><TD>192</TD><TD>23</TD><TD>191</TD><TD>9</TD><TD>170</TD><TD>9</TD></TR><TR><TD>2</TD><TD>150</TD><TD>11</TD><TD>208</TD><TD>14</TD><TD>112</TD><TD>17</TD><TD>202</TD><TD>19</TD></TR><TR><TD>3</TD><TD>166</TD><TD>19</TD><TD>184</TD><TD>22</TD><TD>213</TD><TD>11</TD><TD>205</TD><TD>16</TD></TR></TBODY></TABLE></DIV></LI><LI><DIV class="collapsible-header">Week 29</DIV></LI><LI><DIV class="collapsible-header"> Monday July 16th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Growth of yeast strain YGEN 0013 and the interlab strains for second measurement</P><P>YGEN 0013 colonies were picked and grown in Verduyn medium supplemented with the appropriate vitamins and trace elements, with addition of uracil, tryptophan and leucin. The strain was grown in 5 ml overnight at 30℃, 200 rpm. </P><P>To repeat the Interlab fluorescence measurements, the glycerol stocks of the interlab strains 12-7-18 were taken out of the -80℃, and grown in 5 ml of LB and the appropriate antibiotic overnight at 37℃, 220 rpm.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Run PCR Phusion and Phire PCR pMEL16 on gel to determine whether backbone is amplified correctly</P><P>- Pour 1 % agarose gel. - load 5 ul Phire PCR.  - Add 4 ul MQ and 1 ul 6x Loading dye to 1 ul phusion PCR mix.  - Load 5 ul Phusion PCR mix on gel - Load 5 ul 1kB ladder on gel - Run gel 20 min 90 V</P><P><B>Results</B></P><P>Lane 5 and 6 contain phire PCR of pMEL16. Lane 8 and 9 contain phusion PCR of pMEL16, but nothing can be concluded from the gel as both lanes only show a light smear.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Run a second gel with PCR Phusion and Phire. As the first gel was inconclusive</P><P>New agarose is prepared for this gel.</P><P>- Prepare 400 ml 1 % agarose: Add 4 gr agarose to 400 ml TAE, heat till dissolved. Wait until agarose is handwarm and add 4 ul DNA stain serva 100.000x. - Load 2,5 ul Phire PCR mix. - Load 6 ul Phusion PCR mix. Consisting of 1 ul PCR mix, 1 ul 6x loading dye and 4 ul MQ. - Run gel 90 V, 20 min.</P><P><B>Results</B></P><P>Lane 1 and 2 contain Phusion PCR of pMEL16. Lane 4 and 5 contain Phire PCR of pMEL16. Size of pMEL16 amplification product: 5755 bp. Phusion bands are on the desired height. Phire bands do not run well on the gel. Phire PCR may be overloaded.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Clean phire &amp; Phusion PCR mix pMEL16 using PCR clean up kit</P><P>- Add 2x 50 ul Phire PCR mix together. - Add 2x 50 ul Phusion PCR mix together. - Follow protocol of kit.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Dpn1 digestion of pMEL16 to remove template plasmid</P><P>- Add 5,5 ul tango buffer 10x and 1 ul DpnI to 50 ul cleaned up DNA Phusion. - Add 5,5 ul tango buffer 10x and 1 ul DpnI to 50 ul cleaned up DNA Phire. - Overnight digestion at 37 degrees C.</P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday July 17th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne &amp; Phillip</B></P><P><B>Aim</B> Second Interlab measurement </P><P>The same protocol as described on 13-8-18 and in the Interlab study was used. The starting OD600 of the 1:10 diluted overnight cultures were: </P><TABLE><TBODY><TR><TH>Colony 1</TH><TH>Colony 2</TH></TR><TR><TD>Negative control</TD><TD>0.19</TD><TD>0.182</TD></TR><TR><TD>Positive control</TD><TD>0.167</TD><TD>0.168</TD></TR><TR><TD>Device 1</TD><TD>0.205</TD><TD>0.181</TD></TR><TR><TD>Device 2</TD><TD>0.17</TD><TD>0.175</TD></TR><TR><TD>Device 3</TD><TD>0.185</TD><TD>0.19</TD></TR><TR><TD>Device 4</TD><TD>0.168</TD><TD>0.184</TD></TR><TR><TD>Device 5</TD><TD>0.178</TD><TD>0.185</TD></TR><TR><TD>Device 6</TD><TD>0.195</TD><TD>0.174</TD></TR></TBODY></TABLE><P>Dilution of these 1:10 cultures to target OD600 0.2 in a final volume of 12 ml: 10 ml of culture was already pipetted into the 50 ml falcon tubes for all cultures. To these tubes, the following volumes (µl) were added:</P><TABLE><TBODY><TR><TH>Colony 1</TH><TH>Cell culture</TH><TH>LB</TH><TH>Colony 2</TH><TH>Cell culture</TH><TH>LB</TH></TR><TR><TD>Negative control</TD><TD>1263</TD><TD>737</TD><TD>Negative control</TD><TD>1319</TD><TD>681</TD></TR><TR><TD>Positive control</TD><TD>1437</TD><TD>563</TD><TD>Positive control</TD><TD>1429</TD><TD>571</TD></TR><TR><TD>Device 1</TD><TD>1171</TD><TD>829</TD><TD>Device 1</TD><TD>1326</TD><TD>674</TD></TR><TR><TD>Device 2</TD><TD>1412</TD><TD>588</TD><TD>Device 2</TD><TD>1371</TD><TD>629</TD></TR><TR><TD>Device 3</TD><TD>1297</TD><TD>703</TD><TD>Device 3</TD><TD>1263</TD><TD>737</TD></TR><TR><TD>Device 4</TD><TD>1429</TD><TD>571</TD><TD>Device 4</TD><TD>1304</TD><TD>696</TD></TR><TR><TD>Device 5</TD><TD>1348</TD><TD>652</TD><TD>Device 5</TD><TD>1297</TD><TD>703</TD></TR><TR><TD>Device 6</TD><TD>1231</TD><TD>769</TD><TD>Device 6</TD><TD>1379</TD><TD>621</TD></TR></TBODY></TABLE><P>The cell measurements and the fluorescein measurements were repeated, but now the gain settings were fixed between the measurements. You can find the results on our <A href="https://2018.igem.org/Team:Groningen/InterLab" target="_blank">Interlab page</A>.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Clean up DpnI restriction of pMEL16 Phusion PCR and Phire PCR using PCR clean up kit</P><P>- Follow protocol.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Run a gel with the amplified pMEL16 backbone Phusion and Phire PCR to verify PCR product</P><P>As second gel was also inconclusive for Phire PCR of pMEL16 a third run. As the DNA is cleaned with a PCR clean up kit now the hypotheses is that it will run better on the gel.</P><P>- Pour 1% agarose gel. - Add 2 ul Phire PCR pMEL16 to 1 ul 6x loading dye and 3 ul MQ. - Add 2 ul Phusion PCR pMEL16 to 1 ul 6x loading dye and 3 ul MQ. - Load samples on gel. - Load 5 ul 1kB ladder. - Determine DNA concentration of pMEL16 Phusion using nanodrop: 19,9 ng/ul</P><P><B>Results</B></P><P>Lane 3: Phire PCR pMEL16. Lane4: Phusion PCR pMEL16. pMEL16 expected size: 5755 bp. Phire PCR shows no band, but Phusion band appears at the correct size. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Anneal FW and RV primers for gRNA to create dsDNA for Gibson assembly</P><P>- Add 100uM 25 ul primer F ARO10 tar &amp; R ARO10 tar together, F AGA2 KO tar &amp; R AGA2 KO tar together and F AGA1 OE tar &amp; R AGA1 OE tar together. - Put at 97 degrees C for 5 min. - Dilute dsDNA for each target 10x, 2 ul in 18 ul MQ.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Gibson assembly of pMEL16 linearized backbone with ARO10, AGA2 and AGA1 gRNA’s</P><P>Plasmids pMEL16-ARO10tar, pMEL16-AGA2tar and pMEL16-AGA1tar are created.</P><P>- For each plasmid to construct: 1,5 ul backbone, 1 ul target dsDNA, 2,5 ul Gibson assembly mix 2x. - Put at 50 degrees C for 1 hour. </P><P>Tubes look empty after 1 hour</P><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of gRNA plasmids pMEL16-ARO10tar, pMEL16-AGA2tar or pMEL16-AGA1tar target into DH5alpha</P><P>- Thaw competent cells DH5α 600 ul. - Add 200 ul competent cells to each Gibson assembly mix. - 5 min on ice. - 1 min 42 degrees C. - 5 min on ice. - Add 1 ml LB to each transformation mix.  - Grow at 37 degrees C for 1,5h.  - Plate cell mix on LB+Amp 100 ul and 900 ul (spin down 900 ul and resuspend in 100 ul). - Grow overnight at 37 degrees C. </P><P>18-7-18 - Aga1: medium amount of colonies. - Aga2: low amount of colonies. - Aro10: low amount of colonies.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate 100 ul BJ1991 colony 1 &amp; 2 in 20 ml YPD and grow overnight at 30 degrees C </P></DIV></LI><LI><DIV class="collapsible-header"> Wednesday July 18th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne &amp; Ingeborg</B></P><P><B>Aim</B> Purification of PhipZ </P><P><I>E. coli</I> containing PhipZ (with both an ampicillin and zeocin marker) was grown in LB containing ampicillin and the plasmid was extracted using a plasmid extraction kit. Four parallel cultures were used to purify PhipZ and the following concentrations were obtained (ng/µl): 381.7, 319.9, 419.2 and 389.05. This will be sufficient for the rest of our project. The plasmids were stored at -20℃. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of plasmid pYD1-CipA1-EGII, pYD1-CipA3-EGII, pRS425-CBHII-BGL1, pYD1-CipA1-EGII &amp; pRS425-CBHII-BGL1 and pYD1-CipA3-EGII &amp; pRS425-CBHII-BGL1 into BJ1991 </P><P>See protocol: <A href="https://2018.igem.org/Team:Groningen/Protocols#Agarose-gel" target="_blank">High efficiency yeast transformation of intact yeast cells</A> v2. Altered steps are mentioned. </P><P>- Measure OD600 of BJ1991 colony 1 and 2. col1: 13,8, col2: 14 - Pellet was difficult to resuspend. - Prepare the following transformation mixes:</P><TABLE><TBODY><TR><TH>ssDNA</TH><TH>pYD1-CipA1-EGII p DNA</TH><TH>pYD1-CipA3-EGII pDNA</TH><TH>pRS425-CBHII-BGL1 pDNA</TH><TH>MQ</TH></TR><TR><TD>1882</TD><TD>5 ul</TD><TD>8 ul</TD><TD>62 ul</TD></TR><TR><TD>1883</TD><TD>5 ul</TD><TD>4 ul</TD><TD>66 ul</TD></TR><TR><TD>CBHI</TD><TD>5 ul</TD><TD>5 ul</TD><TD>65 ul</TD></TR><TR><TD>1882 &amp; CBHI</TD><TD>5 ul</TD><TD>8 ul</TD><TD>5 ul</TD><TD>57 ul</TD></TR><TR><TD>1883 &amp; CBHI</TD><TD>5 ul</TD><TD>4 ul</TD><TD>5 ul</TD><TD>61 ul</TD></TR><TR><TD>Negative control</TD><TD>5 ul</TD><TD>70 ul</TD></TR></TBODY></TABLE></DIV></LI><LI><DIV class="collapsible-header">Week 30</DIV></LI><LI><DIV class="collapsible-header"> Monday July 23rd </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> PCR on EGII and CBHI gene fragments</P><P>Primers for the gene fragments were diluted in MQ to 100 µM (a working stock of 50 µM was also prepared). Synthesized gene fragments were diluted in MQ to 5 ng/µl. For both EGII and CBHI, three PCR reactions were performed: </P><TABLE><TBODY><TR><TH>A</TH><TH>B</TH><TH>C</TH></TR><TR><TD>PCR buffer (5X)</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>Primer A (50 µM)</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Primer B (50 µM)</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Polymerase</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Template (5ng/µl)</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Q buffer</TD><TD>10</TD></TR><TR><TD>Mg2+</TD><TD>2</TD></TR><TR><TD>MQ</TD><TD>36</TD><TD>26</TD><TD>34</TD></TR></TBODY></TABLE><P>Primer melting temperatures:</P><P>EGII:FW = 51,3 Rev = 49,2</P><P>CBHI:Fw = 51,3 Rev = 48,6</P><P>PCR programme run in a thermocycler:</P></DIV></LI><LI>95℃ - 5 min</LI><LI>94℃ - 15 sec</LI><LI>44℃ - 1 min</LI><LI>72℃ - 1 min</LI><LI>72℃ - 10 min</LI><LI>4℃ on hold</LI><P>Steps 2-4 were repeated 35 times</P><P><B>Who: Rianne</B></P><P><B>Aim</B> Count colonies &amp; pick colonies from yeast transformation </P><P>Negative control-Leu, -Trp: 0 colonies-Trp: at least 10 colonies-Leu: 0 colonies</P><TABLE><TBODY><TR><TH>1882+CB</TH><TH>1883+CB</TH><TH>1882</TH><TH>1883</TH><TH>CB</TH></TR><TR><TD>10%</TD><TD>1</TD><TD>2</TD><TD>61</TD><TD>55</TD><TD>21</TD></TR><TR><TD>90%</TD><TD>32</TD><TD>32</TD><TD>many</TD><TD>many</TD><TD>many</TD></TR></TBODY></TABLE><P><B>Note: -trp plates look different than others. They are white-ish and the colonies are small.</B></P><P>Two colonies of each plate were used to inoculate 5 ml of Verduyn medium with the appropriate uracil, leucin or tryptophan supplementation. The cultures were grown overnight at 30℃ and 200 rpm</P><LI><DIV class="collapsible-header"> Tuesday July 24th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Agarose gel of 23-07-18 PCR products </P><P> 3 µl of the ladder was loaded, 12 µl of the PCR products were mixed with 3 µl of sample buffer. NC = negative control (no template added). L = ladder.</P></DIV></LI><LI>Upper lane: L - EGII(A)-NC(A)-EGII(B)-NC(B)-EGII(C)-EGII(C)-NC(C)-CBHII(A)-NC(A)</LI><LI>Lower lane: L- CBHII(B)-NC(B)-CBHII(C)-NC(C)-ctrl EGII-ctrl CBHII</LI><LI><DIV class="collapsible-header"> Wednesday July 25th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Sending plasmids pNZ8048 and pPMK4 to Leiden for collaboration </P><P>A small volume of purified plasmid pNZ8048 (obtained from Alisa Garaeva) and PMK4 (obtained from Buu Minh Tran) were transferred to screw-capped microtubes, and, accompanied by a dried drop on Whatman filter paper, sent to Leiden by mail. </P><P>The plasmids arrived in Leiden soon!</P><FIGURE><FIGCAPTION><I>The Leiden Igem team with the eppendorf tubes containing the plasmids.</I></FIGCAPTION></FIGURE></DIV></LI><LI><DIV class="collapsible-header"> Thursday July 26th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne &amp; Bram</B></P><P><B>Aim</B> PCR of EGII </P><P> Because two bands appeared in the previous PCR reaction, we want to change the reaction conditions to see if we can obtain a single product with the right length. Two conditions were tested. The temperature was increased to prevent non-specific binding and DMSO was added.</P><TABLE><TBODY><TR><TH>A</TH><TH>B</TH></TR><TR><TD>PCR buffer (5X)</TD><TD>10</TD><TD>10</TD></TR><TR><TD>Primer A (50 µM)</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Primer B (50 µM)</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Polymerase</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Template (5ng/µl)</TD><TD>2</TD><TD>2</TD></TR><TR><TD>DMSO</TD><TD>3</TD></TR><TR><TD>MQ</TD><TD>35</TD><TD>32</TD></TR></TBODY></TABLE><P>Primer melting temperatures:</P><P>EGII:FW = 51,3 Rev = 49,2</P><P>PCR programme run in a thermocycler:</P></DIV></LI><LI>95℃ - 5 min</LI><LI>94℃ - 15 sec</LI><LI><B>46℃</B> - 1 min</LI><LI>72℃ - 1 min</LI><LI>72℃ - 10 min</LI><LI>4℃ on hold</LI><P>Steps 2-4 were repeated 35 times</P><FIGURE><FIGCAPTION><I>Ladder - EGII A - EGII B</I></FIGCAPTION></FIGURE><P><I>Gel was kept in the fridge overnight before use, which is probably the reason why the bands are not as clear as desired</I></P><LI><DIV class="collapsible-header"> Friday July 27th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne &amp; Phillip</B></P><P><B>Aim</B> Grow transformed yeast strains for SDS-PAGE analysis of protein profiles </P><P>Strains are grown in Verduyn medium and induced with galactose 1 ml samples were stored at -20℃ at 0, 24 and 48 hours</P></DIV></LI><LI>OD 600 were measured of 10x diluted transformant cultures.</LI><LI>4mL of each culture was removed and spun down, the rest was left to grow diluted in 4mL of auxotrophic media.</LI><LI>Spun-down cells were resuspended in 5mL of their respective auxotrophic media. 260 uL of 40% galactose was added. </LI><LI>Samples were incubated overnight.</LI><P>The OD measurements were found to be inaccurate halfway through the experiment, these were not taken into account during sample preparation, meaning that different concentrations of cells were exposed to the same concentration of Galactose.</P><LI><DIV class="collapsible-header">Week 31</DIV></LI><LI><DIV class="collapsible-header"> Thursday August 2nd </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Preparation of yeast cultures for SDS-PAGE </P><P>Cultures induced for 0, 24 or 48 hours were taken from the -20℃ storage, thawed and OD600s were measured (1:10 dilutions)</P><TABLE><TBODY><TR><TH>0 hrs</TH><TH>24 hrs</TH><TH>48 hrs</TH></TR><TR><TD>1883 + CB, 1</TD><TD>0.46</TD><TD>0.42</TD><TD>0.46</TD></TR><TR><TD>1883 + CB, 2</TD><TD>0.34</TD><TD>0.25</TD><TD>0.43</TD></TR><TR><TD>1882 + CB, 1</TD><TD>0.51</TD><TD>0.46</TD><TD>0.5</TD></TR><TR><TD>1882 + CB, 2</TD><TD>0.48</TD><TD>0.35</TD><TD>0.55</TD></TR><TR><TD>CB, 1</TD><TD>0.24</TD><TD>0.29</TD><TD>0.35</TD></TR><TR><TD>CB, 2</TD><TD>0.38</TD><TD>0.3</TD><TD>0.38</TD></TR></TBODY></TABLE><P>900 µl of culture was then used for processing for SDS-PAGE analysis as described <A href="https://2018.igem.org/Team:Groningen/Protocols#ecoli-transformation" target="_blank">here</A>. After preparation, the samples were all clear, except for all the samples CB1 and CB2, which were both a bit purple-ish. The samples were boiled for 6 minutes at 99℃ and subsequently stored at 4℃.</P></DIV></LI><LI><DIV class="collapsible-header"> Friday August 3rd </DIV><DIV class="collapsible-body"><P><B>Who: Rianne, Phillip &amp; Jan Marten</B></P><P><B>Aim</B> SDS-PAGE of yeast extracts </P><P>The SDS-PAGE gels were prepared following <A href="https://2018.igem.org/Team:Groningen/Protocols#Yeast-SDS" target="_blank">this protocol</A>. </P><P>10 µl of each sample is loaded on the gel. Gel is run at 200V, with maximum amperes, until the blue front exits the gels. 
            Gels are stained with Coomassie brilliant blue.</P><P>Proteins are transferred from gel to a PVDF membrane according to the standard western blotting protocol. Membranes are later discarded because of difficulties acquiring antibodies.</P></DIV></LI><LI><DIV class="collapsible-header">Week 32</DIV></LI><LI><DIV class="collapsible-header"> Monday August 6th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Restrictions of cellulosome PCR products or gene fragments </P><P>Restriction reactions were performed in a 50 µl total reaction volume. (1µl of each restriction enzyme, 5 µl of 10X restriction buffer, 1 µg of DNA). For all reactions, NEB buffer 3.1 was used with enzymes BamH1 and Xho1.</P><TABLE><TBODY><TR><TH>Concentration stock (ng/µl)</TH><TH>For ~1 µg (µl)</TH><TH>MQ</TH></TR><TR><TD>CBHI</TD><TD>225,5</TD><TD>5</TD><TD>38</TD></TR><TR><TD>EGII (PCR with DMSO, 26-7)</TD><TD>145</TD><TD>7,5</TD><TD>35,5</TD></TR><TR><TD>EGII (no DMSO, 26-7)</TD><TD>150</TD><TD>7,5</TD><TD>35,5</TD></TR><TR><TD>PhipZ</TD><TD>300</TD><TD>4</TD><TD>39</TD></TR></TBODY></TABLE><P>Reactions were incubated at 37℃ for 1,5 hours.</P></DIV></LI><LI>PCR clean-up gave the following concentrations of restricted fragments:</LI><LI>-CBHI31,35</LI><LI>-EGII (DMSO)27,05</LI><LI>-EGII 28,35</LI><LI>-PhipZ32,55</LI><P>The restricted clean fragments were stored at -20℃ until further use. The genes Scaffold part 1 and Scaffold part 2 were diluted to 5 ng/µl by addition of 200 µl MQ to the delivered dried genes. One third (~300 ng in 65,5 µl) was used for a 75 µl restriction reaction with the following additions:</P><TABLE><TBODY><TR><TH>Enzyme 1 (1 µl)</TH><TH>Enzyme 2 (1 µl)</TH><TH>Buffer (7,5 µl)</TH></TR><TR><TD>Scaffold part 1</TD><TD>BamH1</TD><TD>Cla1</TD><TD>NEB 3</TD></TR><TR><TD>Scaffold part 2</TD><TD>Xho1</TD><TD>Cla1</TD><TD>Cutsmart</TD></TR></TBODY></TABLE><P>The mixtures were incubated for 2,5 hrs at 37℃ and then kept at 4℃ overnight. The genes BGL part 1 and BGL part 2 were diluted to 10 ng/µl by addition of 100 µl MQ to the delivered dried genes. One third (~300 ng in 30 µl) was used for a 50 µl restriction reaction with the following additions:</P><TABLE><TBODY><TR><TH>Enzyme 1 (1 µl)</TH><TH>Enzyme 2 (1 µl)</TH><TH>Buffer (5 µl)</TH></TR><TR><TD>BGL part 1</TD><TD>BamH1</TD><TD>Nru1</TD><TD>NEB 3</TD></TR><TR><TD>BGL part 2</TD><TD>Xho1</TD><TD>Nru1</TD><TD>NEB 3</TD></TR></TBODY></TABLE><P>The mixtures were filled up to 50 µl by addition of 13 µl MQ and incubated for 1,5 hrs at 37℃ and then kept at 4℃ overnight.</P><LI><DIV class="collapsible-header"> Tuesday August 7th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Restriction cleanup and ligation </P><P>PCR cleanup of the previous restriction reactions gave the following concentrations:</P></DIV></LI><LI>-Scaffold part 13,2 ng/µl</LI><LI>-Scaffold part 24,8 ng/µl</LI><LI>-BGL part 12,55 ng/µl</LI><LI>-BGL part 22,9 ng/µl</LI><P>For a vector:insert(:insert) equimolar ratio of around 1:3(:3), the following ligation mixtures were prepared:</P><P>A, B: 100 ng plasmid. C, D: 50 ng plasmid reactions.</P><TABLE><TBODY><TR><TH>A (µl)</TH><TH>B (µl)</TH><TH>C (µl)</TH><TH>D (µl)</TH><TH>Control</TH></TR><TR><TD>PhipZ</TD><TD>3.1</TD><TD>3.1</TD><TD>1,54</TD><TD>1,54</TD><TD>3,1</TD></TR><TR><TD>CBH</TD><TD>3</TD></TR><TR><TD>EGII</TD><TD>3.12</TD></TR><TR><TD>BGLI part 1</TD><TD>13.2</TD></TR><TR><TD>BGLI part 2</TD><TD>16.1</TD></TR><TR><TD>Scaffold part 1</TD><TD>11</TD></TR><TR><TD>Scaffold part 2</TD><TD>7</TD></TR><TR><TD>T4 ligase</TD><TD>1</TD><TD>1</TD><TD>2</TD><TD>2</TD><TD>1</TD></TR><TR><TD>Ligase buffer</TD><TD>1</TD><TD>1</TD><TD>3.5</TD><TD>2.5</TD><TD>1</TD></TR><TR><TD>MQ</TD><TD>1.9</TD><TD>1.78</TD><TD>1</TD><TD>4.9</TD></TR><TR><TD>Total volume</TD><TD>10</TD><TD>10</TD><TD>35</TD><TD>25</TD><TD>10</TD></TR></TBODY></TABLE><P>The ligation mixtures were incubated for 2 hours at 37℃, following 20 minutes at 80℃ to heat-kill the enzymes.</P><P><B>Who: Rianne</B></P><P><B>Aim</B> Transformation </P><TABLE><TBODY><TR><TH>Ligated amount</TH><TH>Into X competent cells (µl)</TH></TR><TR><TD>A</TD><TD>all</TD><TD>75</TD></TR><TR><TD>B</TD><TD>all</TD><TD>75</TD></TR><TR><TD>C</TD><TD>35</TD><TD>200</TD></TR><TR><TD>D</TD><TD>25</TD><TD>150</TD></TR><TR><TD>V</TD><TD>all</TD><TD>75</TD></TR></TBODY></TABLE><P>Following the same protocol as previously used. Cells were plated on LB-ampicillin plates (100µl, 250µl and rest) and incubated at 37℃ overnight.</P><LI><DIV class="collapsible-header"> Thursday August 9th </DIV><DIV class="collapsible-body"><P><B>Who: Phillip</B></P><P><B>Aim</B> Transformation of restricted PhipZ plasmid </P><P>Standard Protocol for <A href="https://2018.igem.org/Team:Groningen/Protocols#Protocols">transformation into e.coli</A> was used. </P></DIV></LI><LI>Competent cells were thawed on ice along with several empty eppendorf tubes.</LI><LI>DNA was spun down and 1.5uL was transferred into an empty eppendorf tube.</LI><LI>50uL of competent cells were added to the DNA.</LI><LI>Mixture was left to incubate for 30 min on ice.</LI><LI>Mixture was heat shocked at 42 degrees for 45 seconds</LI><LI>Immediately after mixture was left to incubate on ice.</LI><LI>950 uL of LB was added to mixture.</LI><LI>Mixture was incubated at 37 degrees and 37 rpm for 1 hour.</LI><LI>Cells were plated on appropriate plates and left at 37 degrees overnight.</LI><LI><DIV class="collapsible-header"> Friday August 10th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Grow the cellulosome-strains on cotton </P><P>Common cotton pads were purchased from a local supermarket and 20 mg cotton was weighed and transferred into glass tubes. The tubes were subsequently autoclaved.  </P><P>Glass tubes with 2%, 0,2%, 0,02%, 0,002% and 0,0002% galactose were prepared. Both with and without the cotton. 10 ml of Verduyn medium (supplemented with the appropriate vitamins, trace elements and with uracil) was added to each tube. As a positive control, LB and E. coli bacteria were added to a glass tube containing cotton, to exclude non-growth because of chemicals in the cotton pads. Also, Verduyn medium containing 2% glucose was added to one of the glass tubes with cotton, to ensure viability of the yeast strains. </P><P>Colonies of CB1883 and CB1882 were taken from the plate and inoculated into the tubes. The cultures were incubated at 30℃, 150 rpm.</P><P><B>Results</B> Growth in both positive controls, no growth in the remaining tubes.</P><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Colony PCR </P><P>Colony PCR is performed on e. Coli DH5a containing pHIPZ7 EGII, pHIPZ7 BGLI, and pHIPZ7 CBHII. 10 colonies from each strain are checked. The following primers are used: pHIPZ7 EGII: EGII1fw, EGII1rv pHIPZ7 BGLI: BGLI1fw, BGL1rv pHIPZ7 CBHII: CBHII1fw, CBHII1rv</P><P>PCR is performed with Qiagen Taq polymerase, in a final volume of 20 µl per sample, with primer concentrations of 0.5 µM per primer. Other ingredients according to the manual. Colonies are touched with a pipette tip and resuspended into the PCR mix.</P><P>PCR protocol: Step 1: 94℃ - 3 min Step 2: 94℃ - 45 sec Step 3: 53℃ - 45 sec Step 4: 72℃ - 2 min Step 5: 72℃ - 10 min Steps 2-4 were repeated 35 times</P><FIGURE><FIGCAPTION><I>Top: EGII 1-10, bottom CBHI 1-10. For BGLI no bands were observed.</I></FIGCAPTION></FIGURE><P><B>Who: Owen &amp; Rianne</B></P><P><B>Aim</B> Run gel to check the PCR on the ordered fragments and the quality of obtained A. thaliana DNA </P><P>- Load 10 ul samples colony PCR pHIPZ7-EGII 1-10 &amp; 5 ul ladder 1kB on gel 1. - Load 10ul sample 11 &amp; 8 ul samples colony PCR pHIPZ7-CBHI 12-20 on gel 1. - load 8 ul samples colony PCR pHIPZ7-BGLI 21-30 on gel 2. - Mix 1 ul genomic DNA A. thaliana with 9 ul MQ, for both gDNA samples. - Add 1 ul genomic DNA 1 A. thaliana to 4 ul MQ and 1 ul loading dye 6x. - Add 1 ul genomic DNA 2 A. thaliana to 4 ul MQ and 1 ul loading dye 6x. - Add 1 ul genomic DNA 1 A. thaliana 10x diluted to 4 ul MQ and 1 ul loading dye 6x. - Add 1 ul genomic DNA 2 A. thaliana 10x diluted to 4 ul MQ and 1 ul loading dye 6x. - Load gDNA samples on gel 2. - Load 5 ul 1kB ladder. - Run gel 1 25 min 90 V. - Run gel 2 30 min 90 V. </P><P><B>Results</B></P><P>Samples EGII (samples 1-10) and CBHI (11-20) and BGLI (21-30) were run on gel 1 upper lane, lower lane and the upper half of gel 2, respectively. For BGLI, no bands were visible. EGII and CBHI showed unexpected bands. Lane 2, 3, 5 &amp; 6 of the lower part of gel 2 contain genomic DNA samples of A. thaliana. No smear across the whole lane can be seen. This indicates that the genomic DNA is partially or fully degraded and likely cannot be used for PCR anymore.</P></DIV></LI><LI><DIV class="collapsible-header">Week 33</DIV></LI><LI><DIV class="collapsible-header"> Monday August 13th </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Negative controls for 10-08-2018 </P><P>Colony PCR on e.coli pHIPZ7 as negative controls. Same primers and PCR settings as on 10-08-2018. A sample without primers is also made.</P><P>Gel was imaged on a UV lightbox, as the imaging PC was acting strange. No bands were visible on the negative controls.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Preparation overnight cultures of DH5α pMEL16-ARO10tar, pMEL16-AGA2tar and pMEL16-AGA1tar  </P><P>- Inoculate 8 colonies of each transformation in 4 ml LB with 4 ul 1000x Amp. - Grow overnight at 37 degrees C. </P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday August 14th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Pick and grow colonies </P><P>Colonies 9 and 10 of both EGII and CBHI (of 10-08-18) were cultured, as well as 10 colonies from the plated with the transformed scaffold. The colonies were grown overnight in 5 ml LB supplemented with ampicillin.  </P><P><B>Who: Matthijs</B></P><P><B>Aim</B> Restriction digest of gBlock genes + restriction analysis on scaffold </P><P><U>Restriction digest of gBlock genes</U></P><P>Since the genes PAL2, CipA3 and BGLI were synthesized in two parts, they had to be ligated to use for further experiments.</P></DIV></LI><LI>The CipA3 fragment contained a ClaI site</LI><LI>BGLI contained ApaI and NruE, but since NruE cuts blunt ends, ApaI was used</LI><LI>PAL2 contained NcoI</LI><P>The fragments were hydrated to arrive at a final concentration of 10 ng/µl, the CipA3 fragments were already hydrated to a concentration of 5 ng/µl. The final reaction mixture contained the following ingredients:</P><TABLE><TBODY><TR><TH>Sample</TH><TH>µl DNA</TH><TH>µl Enzyme</TH><TH>µl Buffer</TH><TH>µl MQ</TH><TH>µl Final</TH></TR><TR><TD>PAL2</TD><TD>30</TD><TD>1</TD><TD>5</TD><TD>14</TD><TD>50</TD></TR><TR><TD>BGLI</TD><TD>30</TD><TD>1</TD><TD>5</TD><TD>14</TD><TD>50</TD></TR><TR><TD>CipA3</TD><TD>65</TD><TD>1.1</TD><TD>7.5</TD><TD>0</TD><TD>~75</TD></TR></TBODY></TABLE><P>Where the enzyme used was corresponds to the enzyme as described above, and the buffer corresponds to the appropriate buffer for the enzyme. Both fragments for each gene are here already mixed.</P><P>Restriction digest was incubated at 37℃ for 2 hours and inactivated by heating to 80℃ for 20 minutes. The mixture was finally held at 4℃</P><P><U>Restriction analysis on scaffold </U></P><P>Colonies were picked previously by Rianne. 10 of these were selected for restriction analysis to verify the transformation.
For the plasmid containing CipA3, the enzyme SmaI was used.
SmaI has two recognition sites on the empty plasmid, pHIPZ7, producing one fragment of 4.6kb and one of 900b. When the gene is successfully inserted it will remove one of the recognition sites, producing only one large fragment of roughly 8.2kb.
</P><P>The following scheme was used to prepare the reaction mixtures for all restriction digests of the CipA3 containing plasmids.</P><TABLE><TBODY><TR><TH>Sample</TH><TH>µl DNA</TH><TH>µl Enzyme</TH><TH>µl Buffer</TH><TH>µl MQ</TH><TH>µl Final</TH></TR><TR><TD>CipA3 1</TD><TD>1.3</TD><TD>0.5</TD><TD>2.5</TD><TD>15.7</TD><TD>20</TD></TR><TR><TD>CipA3 2</TD><TD>1.7</TD><TD>0.5</TD><TD>2.5</TD><TD>15.3</TD><TD>20</TD></TR><TR><TD>CipA3 3</TD><TD>1.43</TD><TD>0.5</TD><TD>2.5</TD><TD>15.6</TD><TD>20</TD></TR><TR><TD>CipA3 4</TD><TD>2.93</TD><TD>0.5</TD><TD>2.5</TD><TD>14</TD><TD>20</TD></TR><TR><TD>CipA3 5</TD><TD>2.96</TD><TD>0.5</TD><TD>2.5</TD><TD>14</TD><TD>20</TD></TR></TBODY></TABLE><P>The amount of DNA to add was calculated such that a final content of 250 ng of DNA would be reached.
The restriction digest was incubated at 37℃ for 2 hours, after which the enzymes were inactivated by heating at 80℃ for 20 minutes.
</P><P><B>Who: Matthijs</B></P><P><B>Aim</B> Ligation of gBlock fragments + Restriction analysis on gel </P><P><U>Ligation of gBlock fragments</U></P><P>After the restriction digest, the fragments had to be ligated together. For this we use T4 ligase. The reaction was mixed as follows:</P><TABLE><TBODY><TR><TH>Sample</TH><TH>µl DNA</TH><TH>µl T4</TH><TH>µl Buffer</TH><TH>µl MQ</TH><TH>µl Final</TH></TR><TR><TD>PAL2</TD><TD>6</TD><TD>0.5</TD><TD>1</TD><TD>2.5</TD><TD>10</TD></TR><TR><TD>BGLI</TD><TD>6</TD><TD>0.5</TD><TD>1</TD><TD>2.5</TD><TD>10</TD></TR><TR><TD>CipA3</TD><TD>8.5</TD><TD>0.5</TD><TD>1</TD><TD>1</TD><TD>10</TD></TR></TBODY></TABLE><P>The amount of DNA to add was calculated such that a total of 25 ng for each fragment was present. The ligation reaction was incubated at 16℃ for 16 hours overnight, after which the enzyme was heat inactivated by heating to 80℃ for 20 minutes.</P><P><U>Restriction analysis on gel</U></P><P>The restriction analysis of the CipA3 containing plasmids was run on an agarose gel. 1% of agarose was used in TAE buffer. 5 µl of SYBR green was added to 50 ml of agarose in TAE as prescribed by the manual. The gel was run at 100V for about 30 minutes after which it was imaged using a UV light source. As a standard DNA ladder, 1 kb generuler was added to the well in the middle.</P><FIGURE><FIGCAPTION><I>CipA3 1-CipA3 2-CipA3 3-CipA3 4-CipA3 5-Ladder-CipA3 6-CipA3 7-CipA3 8-CipA3 9-CipA3 10</I></FIGCAPTION></FIGURE><P><B>Who: Owen</B></P><P><B>Aim</B> Isolation of genomic DNA from A. thaliana  </P><P>A. thaliana leaves were kindly provided by Tau Jang of the Genomics Research in Ecology &amp; Evolution in Nature group. </P><P>Protocol used adapted from: <A href="https://bio-protocol.org/bio101/e90" target="_blank">https://bio-protocol.org/bio101/e90</A></P><P>- Put a leaf in a 2 ml Eppendorf, add a bead for grinding the leaves. Submerge tube in liquid N2 for 5 min. - Add 400 ul Edward buffer: 200 mM Tris (pH 7.5), 250 mM NaCl, 25 mM EDTA, 0.5% SDS. - Take 200 ul of suspension and transfer to a second Eppendorf tube and add 400 ul Edward buffer: Sample 2. Remaining 200 ul: Sample 1. - Vortex both samples 5 sec, set at room temperature until all preps are ready. - Spin both samples at 16,000 rpm for 2 min.</P><P>Sample1:  - Transfer 100 μl of suspension to a fresh tube. - Add 100 μl of isopropanol at room temperature for 2 min. - Spin 5 min, wash pellet with 300 ul 70% EtOH, and dry at room temperature. - Resuspend in 100 μl H2O and store at -20 °C.</P><P>Sample2: - Transfer 300 μl of suspension to a fresh tube. - Spin 5 min max speed.  - Transfer supernatant to a fresh tube.   - Spin 5 min max speed.  - Transfer supernatant to a fresh tube (100 ul).   - Add 100 μl of isopropanol at room temperature for 2 min. - Spin 5 min, wash pellet with 300 ul 70% EtOH, and dry at room temperature. - Resuspend in 50 μl H2O and store at -20 °C.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Plasmid isolation of pMEL16-ARO10tar, pMEL16-AGA2tar &amp; pMEL16-AGA1tar from DH5α </P><P>Isolate plasmids using plasmid isolation kit. - Elute pDNA using 50 ul MQ </P><LI><DIV class="collapsible-header"> Wednesday August 15th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Glycerol stocks and plasmid minipreps </P><P>The cultures of the transformed EGII, CBHI and the scaffold were taken out of the incubator and glycerol stocks were prepared and stored. A plasmid miniprep of the cultures resulted in the following concentrations (ng/µl) of plasmid for restriction analysis and stored at -20℃: </P></DIV></LI><LI>EGII 9 - 144,50</LI><LI>EGII 10 - 160,05</LI><UL><LI>CBHI 9 - 115,20</LI><LI>CBHI 10 - 207,40</LI></UL><UL><LI>Scaffold 1 - 193,59</LI><LI>Scaffold 2 - 146,95</LI><LI>Scaffold 3 - 175,05</LI><LI>Scaffold 4 - 85,35</LI><LI>Scaffold 5 - 84,50</LI><LI>Scaffold 6 - 96,75</LI><LI>Scaffold 7 - 103,95</LI><LI>Scaffold 8 - 95,65</LI><LI>Scaffold 9 - 105,05</LI><LI>Scaffold 10 - 108,0</LI></UL><LI><DIV class="collapsible-header"> Thursday August 16th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction analysis of pMEL16-ARO10tar and pMEL16-AGA2tar </P><P>Restriction analysis is performed using approximately 400ng pDNA per analysis. </P><P>- Prepare 18x restriction mastermix: + 2*18= 36 ul Fastdigest buffer + 0,25*18= 4,5 ul Bcl1 + 12,75*18= 229,5 ul MQ - Add 15 ul mastermix to 16 tubes.  -- Add 5 ul pDNA to 15 ul mastermix and vortex. - Restriction @ 37 degrees C overnight.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Verify quality of isolated A. thaliana DNA 14-8 </P><P>- Pour 1 % agarose gel. - Load gDNA A. thaliana samples 1 and 2 obtained on  14-8-18. 2 ul sample, 3 ul MQ, 1 ul 6x Loading dye. - Run gel 90 V, 40 min. </P><P><B>Results</B></P><P>Lane 1 and 2 contain sample 1 and 2 of the isolated genomic DNA. As no signal is visible in either lane no DNA appears to be isolated. </P><P><B>Who: Owen</B></P><P><B>Aim</B> PCR to amplify PAL2 with N-terminal histag and C-terminal histag from isolated gDNA A. thaliana 14-8-18 </P><P>Prepare C-term histag PAL2 PCR reaction mix: - 10 ul 5x buffer Pfu polymerase  - 0,5 ul PAL2C-FW primer - 0,5 ul PAL2D-RV primer - 1 ul Pfu polymerase  - 1 ul gDNA sample 2 14-8-18 - 37 ul MQ</P><P>Prepare N-term histag PAL2 PCR reaction mix: - 10 ul 5x buffer Pfu polymerase  - 0,5 ul PAL2D-FW primer - 0,5 ul PAL2A-RV primer - 1 ul Pfu polymerase  - 1 ul gDNA sample 2 14-8-18 - 37 ul MQ</P><P>Run the following PCR protocol:</P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>95°C</TD><TD>5 min</TD></TR><TR><TD>2</TD><TD>94°C</TD><TD>15s</TD></TR><TR><TD>3</TD><TD>58°C</TD><TD>1 min</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE></DIV></LI><LI><DIV class="collapsible-header">Week 34</DIV></LI><LI><DIV class="collapsible-header"> Monday August 20th </DIV><DIV class="collapsible-body"><P><B>Who: Matthijs</B></P><P><B>Aim</B> HF PCR on gblocks fragments </P><P>High fidelity PCR was used to amplify the ligated gblock fragments. The following primers were used: </P><TABLE><TBODY><TR><TH>Sample</TH><TH>FW</TH><TH>RV</TH></TR><TR><TD>PAL2</TD><TD>PAL2.1-FW</TD><TD>PAL2.2-REV</TD></TR><TR><TD>BGLI</TD><TD>BGLI-FW</TD><TD>BGLI-RV</TD></TR><TR><TD>CIPA3</TD><TD>CIP3A-FW</TD><TD>CIP3F-REV</TD></TR></TBODY></TABLE><P>Primers were diluted to 30 pmol/μl. The pipetting scheme was as follows:</P><TABLE><TBODY><TR><TH>Sample</TH><TH>DNA µl</TH><TH>FW µl</TH><TH>RV µl</TH><TH>Buffer µl</TH><TH>HF µl</TH><TH>H2O µl</TH></TR><TR><TD>PAL2</TD><TD>2</TD><TD>1,7</TD><TD>1,7</TD><TD>10</TD><TD>2</TD><TD>32,6</TD></TR><TR><TD>BGLI</TD><TD>2</TD><TD>1,7</TD><TD>1,7</TD><TD>10</TD><TD>2</TD><TD>32,6</TD></TR><TR><TD>CIPA3</TD><TD>2</TD><TD>1,7</TD><TD>1,7</TD><TD>10</TD><TD>2</TD><TD>32,6</TD></TR><TR><TD>Control</TD><TD>0</TD><TD>1,7</TD><TD>1,7</TD><TD>10</TD><TD>2</TD><TD>34,6</TD></TR></TBODY></TABLE><P>The PCR protocol was as follows:</P></DIV></LI><LI>activation - 95C - 5m</LI><LI>denaturing - 94C - 15s,30x</LI><LI>annealing - 50C - 1m, 30x</LI><LI>extension - 68C - 2m, 30x</LI><LI>Final extension - 72C - 10m</LI><FIGURE><FIGCAPTION><I>HF PCR on PAL2, BGLI and CIPA3</I></FIGCAPTION></FIGURE><P>The BGLI fragment seems to have a size of about 3kb which is as expected. The size of PAL2 is too small and is most likely not correct. CIPA3 did not give any signal. BGLI concentration after PCR: 69.2 ng/µl</P><LI><DIV class="collapsible-header"> Tuesday August 21st </DIV><DIV class="collapsible-body"><P><B>Who: Matthijs</B></P><P><B>Aim</B> Ligation of BGLI into pHIPZ7 </P><P> Since BGLI was the only sample with a proper PCR signal, it was attempted to ligate into pHIPZ7, our standard yeast expression plasmid. First, double restriction with BamHI and XhoI was done using the following scheme:</P><TABLE><TBODY><TR><TH>sample</TH><TH>DNA</TH><TH>Buffer</TH><TH>BamHI</TH><TH>XhoI</TH><TH>H2O</TH></TR><TR><TD>BGLI</TD><TD>3.6 µl</TD><TD>2.5 µl</TD><TD>0.5 µl</TD><TD>0.5 µl</TD><TD>12.9 µl</TD></TR></TBODY></TABLE><P>The reaction was performed at 37℃ for two hours after which the enzymes were inactivated by heating the sample at 80℃ for 20 minutes. This product was purified by using a PCR purification column to remove the enzymes and buffer components. Next the product was mixed with linearized plasmid at a 1:3 weight ratio and T4 ligase was added to ligate the product into the plasmid.</P><TABLE><TBODY><TR><TH>Sample</TH><TH>Insert DNA</TH><TH>pHIPZ7</TH><TH>T4 buffer</TH><TH>T4</TH><TH>H2O</TH></TR><TR><TD>BGLI</TD><TD>6 µl</TD><TD>5 µl</TD><TD>1.5 µl</TD><TD>1 µl</TD><TD>1.75 µl</TD></TR></TBODY></TABLE><P>The reaction was performed overnight at 16℃ after which the enzyme was inactivated by heat treatment at 80℃ for 20 minutes.</P><P>The product was again cleaned up and Taq polymerase was performed to verify insertion. The general sequence primers were used as these should give a large product when a sequence has inserted at the restriction site and a small product otherwise.</P><P>PCR conditions were standard Taq conditions with an annealing temperature of 50℃ as the melting temperature of the primers was quite low.</P><P><B>Who: Ingeborg</B></P><P><B>Aim</B> Restriction digest of EGII and CBHII </P><P><U>Restriction digest</U>For the plasmids containing EGII and CBHII, the enzyme NCOI was used.
            NCOI has one recognition site on the empty plasmid, pHIPZ7, producing one fragment of 5.8kb. When the EGII gene is successfully inserted it will add one recognition site, producing one large fragment of roughly 6.4kb and a smaller fragment of 967b.
            When the CBHII gene is successfully inserted it will add two recognition sites, producing one large fragment of roughly 5.5kb and two smaller fragments, a fragment of 1186b and one of 766b.</P><P>The following scheme was used to prepare the reaction mixtures for all restriction digests of the EGII and CBHII containing plasmids.</P><TABLE><TBODY><TR><TH>Sample</TH><TH>µl DNA</TH><TH>µl Enzyme</TH><TH>µl Buffer</TH><TH>µl MQ</TH><TH>µl Final</TH></TR><TR><TD>EGII - E9</TD><TD>2.18</TD><TD>0.5</TD><TD>2.5</TD><TD>14.82</TD><TD>20</TD></TR><TR><TD>EGII - E10</TD><TD>1.56</TD><TD>0.5</TD><TD>2.5</TD><TD>15.44</TD><TD>20</TD></TR><TR><TD>CBHII - C9</TD><TD>2.17</TD><TD>0.5</TD><TD>2.5</TD><TD>14.83</TD><TD>20</TD></TR><TR><TD>CBHII - C10</TD><TD>1.2</TD><TD>0.5</TD><TD>2.5</TD><TD>15.8</TD><TD>20</TD></TR></TBODY></TABLE><P>The amount of DNA to add was calculated such that a final content of 250 ng of DNA would be reached.</P><P>The restriction digest was incubated at 37℃ for 2 hours, after which the enzymes were inactivated by heating at 80C for 20 minutes.</P><P><U>Restriction analysis on gel</U>The restriction analysis of the EGII and CBHII containing plasmids was run on an agarose gel. 1% of agarose was used in TAE buffer. 5 µl of SYBR green was added to 50 ml of agarose in TAE as prescribed by the manual. The gel was run at 100V for about 55 minutes after which it was imaged using a UV light source. As a standard DNA ladder, 1 kb generuler was added to the well in the middle.</P><FIGURE><FIGCAPTION><I>E9-E10-C9-C10-Ladder</I></FIGCAPTION></FIGURE></DIV></LI><LI><DIV class="collapsible-header"> Thursday August 23rd </DIV><DIV class="collapsible-body"><P><B>Who: Ingeborg</B></P><P><B>Aim</B> Transformation of BGL1 and PAL2 </P><P>Cells were plated on LB-ampicillin plates (250µl and 100µl concentrated) and incubated at 37℃ overnight. </P></DIV></LI><LI><DIV class="collapsible-header"> Friday August 24th </DIV><DIV class="collapsible-body"><P><B>Who: </B></P><P><B>Aim</B></P></DIV></LI><LI><DIV class="collapsible-header"> Saturday August 25th </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Restriction analysis on BGL1 and PAL2 </P><P>5 colonies of BGL1 and 5 colonies of the PAL2 transformations are inoculated into LB ampicillin medium, and incubated overnight with agitation at 37℃. </P></DIV></LI><LI><DIV class="collapsible-header"> Sunday August 26th </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Restriction analysis of BGL1 and PAL2</P><P>The colonies have grown overnight and are miniprepped to isolate the plasmids. The BGL1 plasmids, along with a pHIPZ7 control plasmid are restricted with XbaI. The PAL2 plasmids are not processed, as it turns out the PCR reaction didn’t add the needed restriction sites. Therefore these plasmids cannot be correct. 
            The restriction tubes are placed in a PCR machine. 2 hours at 37℃, then infinite time at 4℃.</P></DIV></LI><LI><DIV class="collapsible-header">Week 35</DIV></LI><LI><DIV class="collapsible-header"> Monday August 27th </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Restriction analysis of BGL1</P><P>The restriction products are mixed with 5x loading dye and loaded onto a gel. Gel is run at 90 Volt for 25 minutes. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Run PCR PAL2 16-8 and restriction gRNA plasmids 16-8 on gel to verify successful amplification of PAL2 and sequence of gRNA plasmids </P><P>- Prepare new 1% agarose gel. 4 gr agarose in 400 ml TAE buffer, with 4 ul Serva stain 100.000x. - Pour 1 % agarose gel. - Mix 2 ul PCR reaction with 2 ul 6x loading dye and 8 ul MQ. - Add 4 ul 6x loading dye to restriction reaction mixtures. - Load 5 ul 1 kB ladder o Generuler, 12 ul of each PCR sample and 12 ul of each restriction sample on gel. - Run gel 90V 40min. </P><P><B>Results</B></P><P>Lane 1 and 2 contain the PCR reaction PAL2. No fragment was amplified, this is most likely due to incorrect purification of the genomic DNA. Lane 3-6, 8-13, 15-20 contain the restriction samples of both pMEL16-Aga2tar and pMEL16-Aro10tar. 2 bands are expected due to Bcl1 cutting the plasmid 3 times with insertion of the target sequence. As only one band is visible no positive hits seem to be present. However, after further inspection it was found out that Bcl1 cannot cut methylated DNA. It was decided that the region of interest would be sequenced to verify. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Amplifying ss1-V5-EGII-DocS and ss1-FLAG-CBHI-DocS using PCR </P><P>Prepare PCR reaction mix EGII: - 25 ul phire mastermix - 1 ul ss1-V5-EGII-DocS fragment obtained from IDTA 5pg/ul - 1 ul EGII1-FW 100uM - 1 ul EGII1-RV 100uM - 23 ul MQ </P><P>Prepare PCR reaction mix CBHI: - 25 ul phire mastermix - 1 ul ss1-FLAG-CBHI-DocS fragment obtained from IDTA 5pg/ul - 1 ul CBHII1-FW 100uM - 1 ul CBHII1-RV 100uM - 23 ul MQ</P><P>- Add 25 ul PCR reaction mix to a PCR tube for both EGII and CBHI.- Run following PCR protocols on both samples:</P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>5s</TD></TR><TR><TD>3</TD><TD>55</TD><TD>5s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>PCR program 2:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>5s</TD></TR><TR><TD>3</TD><TD>55</TD><TD>5s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE></DIV></LI><LI><DIV class="collapsible-header"> Tuesday August 28th </DIV><DIV class="collapsible-body"><P><B>Who: Ingeborg</B></P><P><B>Aim</B> Transformation of BGL1 (ligation product made by Matthijs) </P><P>Cells were plated on LB-ampicillin plates (250µl and 100µl concentrated) and incubated at 37℃ overnight. </P><P><B>Who: Rianne</B></P><P><B>Aim</B> Agarose gel of Matthijs’ ligation products of BGLI fragments </P><P>4µl ladder, 8µl of the samples </P><FIGURE><FIGCAPTION><I>Ladder-B1-B2-Control</I></FIGCAPTION></FIGURE><P><B>Who: Rianne</B></P><P><B>Aim</B> Grow colonies EGII and BGLI 9 </P><P>EGII and BGLI colonies 9 were take from the glycerol stocks and inoculated into 5 ml LB with ampicillin and grown overnight in a 37℃ incubator, whilst shaking at 220 rpm. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Send pMEL16-Aro10tar, pMEL16-AGA2tar and pMEL16-AGA1tar for sequencing </P><P>Plasmids from 2 colonies of each pMEL16 variant are sent for sequencing. </P><P>Prepare sequencing samples: - 2 ul MQ - 3 ul pDNA - 5 ul pMEL16-seq-FW </P><P><B>Who: Owen</B></P><P><B>Aim</B> Run agarose gel to verify A. thaliana gDNA and EGII and CBHI PCR </P><P>- Pour 1 % agarose gel. - Load 1 ul loading dye with 4 ul MQ and 1ul sample. - Load 5 ul 1kB ladder o Generuler - Run gel 90V 35 min.  </P><P><B>Results</B></P><P>Lane 1-2 gDNA samples of A. thaliana gDNA isolation 14-8-18. Lane 3-4 contain A. thaliana gDNA samples obtained from Tau Jang. None of these samples show a band across the whole lane. This indicates that the quality of the gDNA is not sufficient. Lane 5, 6, 7 and 9 contain the EGII and CBHI PCR reactions of 28-8-18. No clear bands are visible in these lanes indicating that the PCR has failed.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate BJ1991 containing pYD1-CipA3-EGII &amp; pRS425-CBHII-BGL1 and BJ1991 containing pYD1-CipA1-EGII &amp; pRS425-CBHII-BGL1 </P><P>- Inoculate a colony in 5 ml Verduyn with 50 ul mineral solution, 5 ul vitamin solution and 50 ul 100x URA solution. - Grow at 30 degrees C overnight while shaking </P><P>On 29/8/18 minimal growth was observed.</P></DIV></LI><LI><DIV class="collapsible-header"> Wednesday August 29th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Taq colony PCR of EGII, BGLI and CBHI colonies/cultures</P><P>20µl reaction volume, taq polymerase (according to manual manufacturer)</P><P>Different sets of repair fragment primers:
            A) CasR2 Fw &amp; Rv
            B) CasR3 Fw &amp; Rv
            C) CasR5 Fw &amp; Rv
            D) CasR4 Fw &amp; CasR1 Rv</P><P>Expected sizes:
            Insert + promotor, terminator, docking module (2150 bp)
            BGLI = 3150 bp + 2150 = 5300
            EGII = 1640 bp + 2150 = 3790
            CBHI = 1820 bp + 2150 = 3970</P><P>All primer sets are tested on the empty PhipZ plasmid as a control reaction.
            For the CBHI colony 9 primer set B was used and EGII was tested with primer set A. A small amount of culture was added to the reaction mix to provide the template DNA. 30 colonies of the BGLI plates were tested with primer set A.</P><P>Cycling programme:
            94℃   5 min</P><P>15 cycles:
            94℃   45 sec
            64-57℃  45 sec
            72℃   5:30 min</P><P>15 cycles:
            94℃   45 sec
            57℃   45 sec
            72℃   5:30 min</P><P>72℃   10 min
            12℃   indefinite</P><P><B>Who: Owen</B></P><P><B>Aim</B> Amplifying ss1-V5-EGII-DocS and ss1-FLAG-CBHI-DocS using PCR2 </P><P>Prepare PCR reaction mix EGII Phire: - 25 ul phire mastermix - 1 ul ss1-V5-EGII-DocS fragment obtained from IDTA 5pg/ul - 1 ul EGII1-FW 100uM - 1 ul EGII1-RV 100uM - 23 ul MQ </P><P>Prepare PCR reaction mix EGII Phusion:  - 25 ul phusion mastermix  - 1 ul ss1-V5-EGII-DocS fragment obtained from IDTA 5pg/ul - 1 ul EGII1-FW 100uM - 1 ul EGII1-RV 100uM - 23 ul MQ</P><P>Prepare PCR reaction mix CBHI Phire: - 25 ul phire mastermix - 1 ul ss1-FLAG-CBHI-DocS fragment obtained from IDTA 5pg/ul - 1 ul CBHII1-FW 100uM - 1 ul CBHII1-RV 100uM - 23 ul MQ</P><P>Prepare PCR reaction mix CBHI Phusion: - 25 ul phusion mastermix - 1 ul ss1-FLAG-CBHI-DocS fragment obtained from IDTA 5pg/ul - 1 ul CBHII1-FW 100uM - 1 ul CBHII1-RV 100uM - 23 ul MQ</P><P>Run following Phire PCR protocol for both samples:</P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>5s</TD></TR><TR><TD>3</TD><TD>52°C</TD><TD>5s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE><P>Run the following Phusion protocol for both samples:</P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD>3</TD><TD>50°C</TD><TD>30s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE><P>- Pour 1 % agarose gel. - Load 2 ul PCR sample with 3 ul MQ and 1 ul 6x loading dye. - Load 5 ul 1kB ladder o Generuler. - Run gel 100 V 35 min.</P><P><B>Results</B></P><P>No bands besides the ladder in lane 6 can be seen indicating that the PCR reaction was unsuccessful. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate overnight culture BY4741 p414 KanMX TEF1-Cas9-Tcyc </P><P>- Inoculate a colony in 10 ml YPD with 40 ul 250x G418 - Grow overnight at 30 degrees C while shaking. </P></DIV></LI><LI><DIV class="collapsible-header"> Thursday August 30th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Agarose gel of 29-8 pcr products </P><FIGURE><FIGCAPTION><I>APhipZ - BPhipZ - CPhipZ - DPhipZ - Ladder - CBHI - EGII</I></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><I>BGLI: Ladder - 1 - 2 - 3 - 4 - 5 - 6/10 - 11/15 - 16/20 - 21/25 - 26-30</I></FIGCAPTION></FIGURE><P><B>Who: Jan Marten</B></P><P><B>Aim</B> PCR on the PAL2 and CIPA3 fragments </P><P>PCR is performed on the PAL2 and CIPA3 gene fragments. 
            Primers: For PAL2, PAL2f-fw and pal2f-rv are used to attach a C-terminal His-tag, and pal2g-fw and pal2g-rv are used to attach a N-terminal His-tag.
            For CIPA3, primers cipa3a-fw and cipa3c-rv are used.
            Master mix: 5 ul buffer, 1 ul dntp’s, 0.2 ul of each primer, 1 ul template, 0,5 ul Qiagen high fidelity polymerase, 42 ul MQ water.
            9 mixes are made:  </P></DIV></LI><LI>CIPA3</LI><LI>CIPA3</LI><LI>CIPA3</LI><LI>PAL2 primer set F</LI><LI>PAL2 primer set F</LI><LI>PAL2 primer set F</LI><LI>PAL2 primer set G</LI><LI>PAL2 primer set G</LI><LI>PAL2 primer set G</LI><P>Machine settings:
            94 degrees, 3 minutes initial denaturation
            94 degrees, 45 seconds denaturation
            52 degrees, 45 seconds, annealing
            72 degrees, 2 minutes, extension
            Repeat denaturation, annealing, extension 35 times
           72 degrees, 10 minutes, final extension</P><P><B>Who: Owen</B></P><P><B>Aim</B> Amplifying ss1-V5-EGII-DocS and ss1-FLAG-CBHI-DocS using PCR3 </P><P>Prepare PCR reaction mix CBHI: - 25 ul Phire mastermix - 1 ul ss1-FLAG-CBHI-DocS fragment 5pg/ul obtained from IDTA - 1 ul CBHII1-FW 100uM - 1 ul CBHII1-RV 100uM - 23 ul MQ </P><P>Prepare PCR reaction mix EGII Phire: - 25 ul Phiremastermix - 1 ul ss1-V5-EGII-DocS fragment 5pg/ul obtained from IDTA - 1 ul EGII1-FW 100uM - 1 ul EGII1-RV 100uM - 23 ul MQ</P><P>Split 50 ul mastermix into 2x 25 ul in a PCR tube. </P><P>Run a EGII and CBHI PCR reaction according to the following 2 protocols:</P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>5s</TD></TR><TR><TD>3</TD><TD>49°C</TD><TD>5s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>5s</TD></TR><TR><TD>3</TD><TD>48°C</TD><TD>5s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD>30x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE><P>- Pour 1 % agarose gel. - Load 1 ul PCR sample with 4 ul MQ and 1 ul 6x loading dye. - Load 5 ul 1kB ladder o Generuler. - Run gel 100 V 35 min. - Clean EGII PCR using PCR cleanup kit.</P><P><B>Results</B></P><P>Lane 1 and 2 contain EGII PCR samples, a band can be seen at 1700 bp which corresponds to the size of the amplified fragment. Lane 4 and 5 contain CBHI PCR samples. Multiple bands can be seen, indicating a non pure PCR reaction.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate BJ1991 containing pYD1-CipA3-EGII &amp; pRS425-CBHII-BGL1 </P><P>- Inoculate a colony in 5 ml Verduyn with 50 ul mineral solution, 5 ul vitamin solution and 50 ul 100x URA solution. - Grow at 30 degrees C overnight while shaking </P><LI><DIV class="collapsible-header"> Friday August 31st </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Prepare Verduyn medium and 10% raffinose and fructose solutions </P><P>Prepare 400 ml Verduyn medium as described in the Protocol: <A href="https://2018.igem.org/Team:Groningen/Protocols#LB-plates" target="_blank">Verduyn medium.</A></P><P>Prepare 100 ml 10% fructose and raffinose solutions. 10 gram in 100 ml MQ.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction of pHIPZ7 and ss1-V5-EGII-DocS PCR fragment </P><P>A 3:1 cut insert: cut vector ratio is used. 100 ng cut vector DNA is used. This translates to 88 ng insert. </P><P>- Prepare the pHIPZ7 restriction mixture: 0,62 ul pDNA 318ng/ul, 0,5 BamHI, 0,5 XhoI, 2 ul fast digest buffer, 16,4 ul MQ. - Prepare the ss1-V5-EGII-DocS restriction mixture: 0,82 ul insert, 0,5 BamHI, 0,5 XhoI, 2 ul fast digest buffer, 16,2 ul MQ. - Restrict overnight at 37 degrees C.</P></DIV></LI><LI><DIV class="collapsible-header"> Saturday September 1st </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B></P><P><B>Who: Owen</B></P><P><B>Aim</B> PCR to amplify ss1-FLAG-CBHI-DocA </P><P>Prepare the following PCR mixture: - 37,5 ul Phire master mix - 1,5 ul CBHII1-FW - 1,5 ul CBHII1-RV - 3 ul ss1-FLAG-CBHI-DocS 5uM - 31,5 ul MQ </P><P>Run the following PCR protocol:</P><TABLE><TBODY><TR><TH>PCR program:</TH></TR><TR><TD>1</TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD>2</TD><TD>98°C</TD><TD>5s</TD></TR><TR><TD>3</TD><TD>51°C</TD><TD>5s</TD></TR><TR><TD>4</TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD>34x back to 2</TD></TR><TR><TD>5</TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD>6</TD><TD>12°C</TD><TD>till end</TD></TR></TBODY></TABLE><P>Run gel according to standard <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE" target="_blank">agarose gel protocol</A>.</P><P><B>Results</B></P><P>Lane 1-3 AGA2-CIPA3-His PCR product. A band is visible at the expected height of approx 2.600. Lane 4-6 contain synthetic PAL2 gene part1 PCR product. No product visible. Lane 8-10 contain synthetic PAL2 gene part2. No product visible. Lane 11-13 contain ss1-FLAG-CBHI-DocS PCR product 1-9-18. 3 bands are visible. The size of ss1-FLAG-CBHI-DocS is 1859 bp, therefore it is assumed that the uppermost band is the correct band. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Ligation of ss1-V5-EGII-DocS into pHIPZ7. </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#restriction-digest" target="_blank">standard ligation protocol</A>. Restriction of 31-8-18 is used. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of pHIPZ7-ss1-V5-EGII-DocS ligation product into DH5alpha </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Protocols" target="_blank">Ecoli transformation protocol</A>. LB amp plates used as selection plates. Empty vector used as control and plated on 900 ul as well. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Prepare overnight culture BJ1991 with pYD1-CipA3-EGII &amp; pRS425-CBHII-BGL1 (Also BJA3CB from here on) and BJ1991 with pYD1-CipA1-EGII &amp; pRS425-CBHII-BGL1 (Also BJA1CB from here on) </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#LB-plates" target="_blank">Verduyn medium protocol</A></P><P>- Inoculate a colony of each strain in Verduyn medium with uracil and fructose (5ml). - Inoculate a colony of each strain in Verduyn medium with uracil and raffinose(5ml).</P></DIV></LI><LI><DIV class="collapsible-header"> Sunday September 2nd </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Preculturing of BJC3CB &amp; BJC1CB for growth on cellobiose </P><P>See protocol <A href="https://2018.igem.org/Team:Groningen/Protocols#LB-plates" target="_blank">Verduyn medium</A>. </P><P>- Measure OD600 of overnight cultures using a photospectrometer 14:00. Use 10X diluted culture medium as blank.</P><TABLE><TBODY><TR><TH>BJC3CB</TH><TH>fruc</TH><TH>1,7</TH></TR><TR><TD>raf</TD><TD>0,67</TD></TR><TR><TD>BJC1CB</TD><TD>fruc</TD><TD>1,37</TD></TR><TR><TD>raf</TD><TD>0,56</TD></TR></TBODY></TABLE><P>- Calculate dilutions of cultures to have OD600 0,4 at 10:00 3-9-18 assuming a doubling time of 2, 3 and 4 hours.
            - Add 5, 45 and 140 ul culture BJC1CB fructose to 20 ml Verduyn with fructose and uracil. 
            - Add 4, 37 and 110 ul culture BJC3CB fructose to 20 ml Verduyn with fructose and uracil.
            - Add 11, 110 and 336 ul culture BJC1CB raffinose to 20 ml Verduyn with fructose and uracil.
            - Add 9, 92 and 280 ul culture BJC3CB raffinose to 20 ml Verduyn with fructose and uracil.</P></DIV></LI><LI><DIV class="collapsible-header">Week 36</DIV></LI><LI><DIV class="collapsible-header"> Monday September 3rd </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> measuring OD of cultures, and galactose induction </P><P>The OD of the cultures are measured: </P><TABLE><TBODY><TR><TH>Strain</TH><TH>fructose</TH><TH>raffinose</TH></TR><TR><TD>1882 1</TD><TD>0.05</TD><TD>0.02</TD></TR><TR><TD>1882 2</TD><TD>0.16</TD><TD>0.09</TD></TR><TR><TD>1882 3</TD><TD>0.52</TD><TD>0.22</TD></TR><TR><TD>1883 1</TD><TD>0.02</TD><TD>0.02</TD></TR><TR><TD>1883 2</TD><TD>0.2</TD><TD>0.07</TD></TR><TR><TD>1883 2</TD><TD>0.55</TD><TD>0.25</TD></TR></TBODY></TABLE><P>At 13.10, 0.5% final concentration galactose is added to 1882.3 and 1883.3 cultured on fructose.
            At 15.10, 0.5% final concentration galactose is added to 1882.3 and 1883.3 cultured on raffinose.
            After 2 hours of incubation, the cells are spun down at 6000 rcf, 5 minutes, and resuspended in 10 ml of Verduyn medium. Spun down again, and resuspended in 20 ml Verduyn medium. 
            10 ml of this is transferred to a culture flask, more Verduyn medium is added, to a final volume of 20 ml. 2% cellobiose final concentration is added, along with uracil.</P><P>At 17.10, the OD of the cultures is measured:</P><TABLE><TBODY><TR><TH>fructose</TH><TH>raffinose</TH></TR><TR><TD>1882.3</TD><TD>0.44</TD><TD>0.108</TD></TR><TR><TD>1883.3</TD><TD>0.456</TD><TD>0.134</TD></TR></TBODY></TABLE><P>The positive control had an OD of 0.432, the negative control was 0.514</P><P><B>Who: Owen</B></P><P><B>Aim</B> Colony PCR transformation of pHIPZ7-ss1-V5-EGII-DocS 1-9-18 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#cellulosephosphorylation">Colony PCR protocol</A>, annealing temperature 50 degrees C. Primer FW: EGII1-FW. Primer RV: EGII1-RV. 5 colonies are added together in 1 tube instead of 1 colony per tube. </P><P><B>Results</B></P><P>Lane 1-3 contain larger volumes of earlier PCR reactions of ss1-FLAG-CBHI-DocS. three bands are visible in lane 3. The expected size of the fragment is 1800bp. The uppermost band is assumed to be the desired fragment. This gel shows that the one of the earlier PCR reactions was successfull. Lane 5-10 contain colony PCR samples 3-9-18. The ss1-V5-EGII-DocS fragment is 1700 bp. No band at this height is observed indicating that none of the colonies are positive.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Second colony PCR transformation of pHIPZ7-ss1-V5-EGII-DocS 1-9-18 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#cellulosephosphorylation" target="_blank">Colony PCR protocol</A>, annealing temperature 50 degrees C. Primer FW: EGII1-FW. Primer RV: EGII1-RV. Gel run later. </P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday September 4th </DIV><DIV class="collapsible-body"><P><B>Who: Ingeborg</B></P><P><B>Aim</B> Transformation of the PAL2 plasmid (obtained from Shreyans) </P><P>Cells were plated on LB-kanamycin plates (250µl and 250µl concentrated) and incubated at 37℃ overnight. </P><P><B>Who: Rianne</B></P><P><B>Aim</B> PCR PAL2 on pAL2 syn gene in pUC57-kan obtained from Shreyans (Roelfes group) </P><P>Phusion polymerase with primer sets:
            PAL2C Fw &amp; PAL2B Rv
            PAL2B Fw &amp; PAL2A Rv </P><P>Mix (50 µl):
            PCR buffer    10 µl
            Primer A    5 µl  (final concentration 1µM)
            Primer B    5 µl
            Polymerase    1,5 µl
            Template    1 µl  (from 51,3 ng/µl stock)
            MQ      27,5 µl</P></DIV></LI><LI>95℃ - 5 min</LI><LI>94℃ - 15 sec</LI><LI>65℃ - 1 min</LI><LI>72℃ - 4.5 min</LI><LI>2-4 - 35X</LI><LI>72℃ - 10 min</LI><LI>4℃ - indefinite</LI><P><B>Who: Rianne</B></P><P><B>Aim</B> Culture BGLI colony number 2 </P><P>Colony number 2 (see 30-8-18) was grown overnight in 5 ml LB with ampicillin, at 37℃, 200 rpm. </P><P><B>Who: Rianne</B></P><P><B>Aim</B> Prepare yeast extract samples for SDS-PAGE </P><P>Samples prepared by Jan Marten were taken from the -20℃ storage and thawed. The samples include the 1883 pellet and 1883 supernatant, and 1882 pellet and 1882 supernatant. </P><P>1ml of the supernatant was added to 667 µl of 12,5% TCA. After washing, the pellet was resuspended into 400 µl of 12,5% TCA. The samples were stored at -20℃ overnight. </P><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Measure OD of yesterday’s cultures, and to start new cultures of the 1882 and 1883 strains on Verduyn medium with fructose </P><P>At 14.00 hrs the OD of yesterday’s cultures are measured. </P><TABLE><TBODY><TR><TH>fructose</TH><TH>raffinose</TH></TR><TR><TD>1882.3</TD><TD>3.9</TD><TD>3.6</TD></TR><TR><TD>1883.3</TD><TD>3.5</TD><TD>4.3</TD></TR></TBODY></TABLE><P>The positive control had an OD of 4.5, the negative control was 4.4. Since the negative control is not supposed to grow, the experiment is performed again.
            20 ml of Verduyn medium with uracil and 1% fructose is used as a growth medium. Inoculate 2 cultures of 1882 and 1883 each. (1882.1, 1882.2, 1883.1, 1883.2)</P><P><B>Who: Owen</B></P><P><B>Aim</B> Running an agarose gel to verify second colony PCR 3-9-18 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE" target="_blank">agarose gel protocol</A>. 0,8% agarose gel is used, because of the gel extraction of the ss1-FLAG-CBHI-DocS PCR sample.  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Running ss1-FLAG-CBHI-DocS PCR sample from lane 3 gel 3-9-18 on gel and extracting it from the gel </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE" target="_blank">agarose gel protocol</A>. 0,8% agarose gel is used. 25 ul sample is loaded in 2 wells. Band at 1800bp is cut out of gel under blue light and DNA is isolated using gel extraction kit. DNA concentration measured using nanodrop. No DNA was found to be present. </P><P><B>Results</B></P><P>Lane 1-5 contain the colony PCR. The ss1-V5-EGII-DocS fragment is 1700 bp. Lane 1 and 3 contain a band of the desired size. Lane 8 and 9 contain ss1-FLAG-CBHI-DocS PCR. Upper band is cut out for DNA extraction.</P><P><B>Who: Owen</B></P><P><B>Aim</B> PCR on ss1-FLAG-CBHI-DocS fragment to amplify for cloning </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#colony-PCR" target="_blank">Phire PCR protocol</A>. Annealing temp: 52 degrees C. Primer FW: CBHII1-FW. Primer RV: CBHII1-RV. Template: ss1-FLAG-CBHI-DocS.  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Gel extraction of ss1-FLAG-CBHI-DocS PCR 4-9-10 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE" target="_blank">agarose gel protocol</A>. 0,8% agarose gel used. Load 4*25 ul sample on gel. Cut out bands at 1800 bp and isolate DNA using gel extraction kit. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction and ligation of ss1-V5-EGII-DocS into pHIPZ7 and ss1-FLAG-CBHI-DocS into pHIPZ7 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">protocol restriction</A>. Use BamHI and XhoI restriction enzymes. See <A href="https://2018.igem.org/Team:Groningen/Protocols#restriction-digest" target="_blank">ligation protocol</A>. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Overnight culture positive colonies transformation 3-9-18 </P><P>- Inoculate the 2 positive colonies in 4 ml LB + amp. - Grow overnight at 37 degrees C. </P><LI><DIV class="collapsible-header"> Wednesday September 5th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Gel of 4-9-18 PAL2 PCR </P><P>4 µl ladder - 15 µl PCR sample (12 + 3 loading dye) </P><FIGURE><FIGCAPTION><I>Ladder - PAL2A - PAL2C</I></FIGCAPTION></FIGURE><P><B>Who: Rianne</B></P><P><B>Aim</B> Gel of 4-9-18 protein samples SDS-PAGE </P><P>The protein samples of 4-9-18 were further processed as described in the <A href="https://2018.igem.org/Team:Groningen/Protocols#Protocols" target="_blank">protocol</A> and stored at 4℃. </P><P><B>Who: Rianne</B></P><P><B>Aim</B> Plasmid prep BGLI and sequencing of plasmid </P><P>The culture inoculated on 4-9-18 was used to miniprep the plasmid containing BGLI. The yield was 640 ng/µl, of which 7,8 µl was taken into 42,2 µl MQ to obtain a concentration of 100 ng/µl for sequencing. BGLI Rv seq : seq ID = 98FB34 BGLI Fw seq : seq ID = 98FB35 </P><P>Sequencing results show non-silent mutations in the fragments.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of ligations 4-9-18 into DH5alpha for amplification of plasmid </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Protocols" target="_blank">standard transformation protocol</A>. </P></DIV></LI><LI><DIV class="collapsible-header"> Thursday September 6th </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Transfer cultures to fresh medium </P><P>100 ul of each culture started on 04-09-2018 is transferred to 20 ml of fresh Verduyn medium with uracil and fructose. Incubate the cultures at 30 degrees celsius. </P><P><B>Who: Rianne</B></P><P><B>Aim</B> SDS-PAGE of protein samples 5-9-18 </P><P>4µl ladder, other samples 20 µl loaded onto a 12,5% SDS-gel. The order of the samples was: ladder - 1882 pellet - 1882 supernatant - 1883 pellet - 1883 supernatant. Unfortunately, no overexpressed protein production was visible on the gel after staining. </P><P><B>Who: Rianne</B></P><P><B>Aim</B> PAL2C PCR purification and restriction </P><P>Concentration after purification: 14,55 ng/µl PCR product Restriction: Xho1 and BamH1 enzymes both 1 µl, 50 µl of the purified pcr product, 6 µl of NEB buffer. Incubation at 37℃ for 3 hours. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Colony PCR transformation 5-9-18 verify transformation </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#cellulosephosphorylation" target="_blank">Colony PCR protocol</A>. Annealing temp: 52 degrees C. Primer FW: CBHII1-FW. Primer RV: CBHII1-RV. 8 colonies.  </P><P><B>Results</B></P><P>ss1-FLAG-CBHI-DocS is 1800 bp. Lane 2, 7 and 8 show a band at the desired height.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction and ligation of AGA2-CIPA3-His into pHIPZ7 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">restriction protocol</A>. BamHI and XhoI used as restriction enzymes.  See <A href="https://2018.igem.org/Team:Groningen/Protocols#restriction-digest" target="_blank">ligation protocol</A>. PCR clean up with DNA for ligation accidentally thrown away another round of restriction and ligation is started as described before.  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Prepare overnightculture colonies lane 2, 7 and 8 </P><P>pHIPZ7-ss1-FLAG-CBHI-DocS (pHIPZ7-CBHI from here on) for plasmid isolation.</P><P>- Add each culture to 4 ml LB + Amp. Grow overnight at 37 degrees C. </P></DIV></LI><LI><DIV class="collapsible-header"> Friday September 7th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> PCR purification of the restricted product and ligation into PhipZ </P><P>Vector (5800 bp) 25 ng: insert (2200 bp) 28,5 ng </P></DIV></LI><LI>2 µl T4 ligation buffer</LI><LI>2 µl T4 ligase</LI><LI>2.4 µl restricted PhipZ (21.2 ng/µl)</LI><LI>6.1 µl restricted PAL2 (9.4 ng/µl)</LI><LI>7.5 µl MQ</LI><P>Incubation for 3 hours at 37℃</P><P>Transformation: 5 µl of the ligation product into 50 µl of competent E. coli cells.
            As a control, 1.5 µl of the restricted PhipZ plasmid was transformed in parallel. 
            The transformed cells were plated 100 µl, 200 µl, 350 µl and Rest. Of the control 200 µl was plated only. The plates were incubated at 37℃ overnight.</P><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Measure OD and galactose induction </P><P>At 12.00 hrs, the OD of the cultures is measured.
            1882.1: OD 1.35
            1882.2: OD 1,26
            1883.1: OD 1,53
            1883.2: OD 1,29 </P><P>Induce with 0.25 ml 40% galactose solution. This leads to a concentration of 0.5% in the medium.
            Dilute: 10 ml culture + 10 ml fresh medium. Incubate the cultures at 30 degrees celsius.</P><P>At 18.00 hrs, 1 sample of each strain is spinned down, washed 2x with Verduyn medium without carbon source, and inoculated in Verduyn medium with uracil and cellobiose (0.5% final concentration).</P><P>OD at T=0:
            1882.1: 0,59
            1882.2: 0,63
            1883.1: 0,54
            1883.2: 0,36</P><P><B>Who: Owen</B></P><P><B>Aim</B> Isolate plasmids pHIPZ7-CBHI overnight culture </P><P>Use plasmid miniprep kit to isolate plasmid. </P><LI><DIV class="collapsible-header"> Saturday September 8th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Taq colony PCR on PAL2-PhipZ colonies </P><P>Primer set A was used (see 29-08-18). Taq reaction mixture following manufacturers manual.
            50 colonies were screened. The first 5 were transferred to a single reaction tube (A-E), subsequently 5 colonies per reaction tube (F-N), resulting in 14 reactions, 20 µl each. </P><P>Expected size: approximately 4400 bp</P><P>Cycling programme:94℃   5 min</P><P>15 cycles:
            94℃   45 sec
            64-57℃  45 sec
            72℃   5 min</P><P>15 cycles:
            94℃   45 sec
            57℃   45 sec
            72℃   5 min</P><P>72℃    10 min
            12℃   indefinite</P><P><B>Who: Jan Marten</B></P><P><B>Aim</B> measure OD and inoculate cellobiose cultures </P><P>At 12.00 hrs the 24 hrs galactose induction cultures are washed with Verduyn medium without carbon source, and inoculated into Verduyn medium with 0.5% cellobiose. As described yesterday.
            Some of the leftover cultures is pelleted and plated onto ReCell plates. </P><P>At 13.00 hrs the OD of all cultures is measured</P><TABLE><TBODY><TR><TH>Strain</TH><TH>Evening culture (T=18 hrs)</TH><TH>Morning culture (T=0 hrs)</TH></TR><TR><TD>1882.1</TD><TD>1.64</TD><TD>0.85</TD></TR><TR><TD>1882.2</TD><TD>1.47</TD><TD>1.04</TD></TR><TR><TD>1883.1</TD><TD>1.65</TD><TD>0.81</TD></TR><TR><TD>1883.2</TD><TD>1.22</TD><TD>0.81</TD></TR></TBODY></TABLE><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of ligation product 6-9-18 into DH5alpha for plasmid amplification </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Protocols" target="_blank">Ecoli transformation protocol</A>. LB+Amp plates used. </P></DIV></LI><LI><DIV class="collapsible-header"> Sunday September 9th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Gel of the PCR samples of 8-9-18 PAL2 </P><P>Ladder: 4µl, samples: 10µl </P><FIGURE><FIGCAPTION><I>Gel 1: Ladder - A - B - C - D - E - F - G</I></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><I>Gel 2: Ladder - H - I - J - K - L - M - N </I></FIGCAPTION></FIGURE><P>All colonies are negative.</P><P><B>Who: Rianne</B></P><P><B>Aim</B> BGL1 another colony PCR and agarose gel </P><P>Another colony PCR on the BGL1 colonies was performed similar to the PCR on 29-8-18. Samples A-E contain 1 colony (31-35), samples F-N contain 5 each (36-80). Primer set A was used.  </P><FIGURE><FIGCAPTION><I>Gel 1: Ladder - A - B - C - D - E - F- G</I></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><I>Gel 2: Ladder - H - I - J - K - L - M - N</I></FIGCAPTION></FIGURE><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Measure OD of cellobiose grown cultures </P><P>At 13.00 hrs the OD of the cultures is measured. </P><TABLE><TBODY><TR><TH>Strain</TH><TH>Evening (T=40 hrs)</TH><TH>Morning (T=24 hrs)</TH></TR><TR><TD>1882.1</TD><TD>2</TD><TD>2.04</TD></TR><TR><TD>1882.2</TD><TD>1.84</TD><TD>2.12</TD></TR><TR><TD>1883.1</TD><TD>2</TD><TD>1.85</TD></TR><TR><TD>1883.2</TD><TD>1.62</TD><TD>1.94</TD></TR></TBODY></TABLE></DIV></LI><LI><DIV class="collapsible-header">Week 37</DIV></LI><LI><DIV class="collapsible-header"> Monday September 10th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Prepare yeast extract samples for SDS-PAGE </P><P>Jan Marten prepared samples and stored them at -20℃. The samples included 1882 1 and 2, 1883 1 and 2, both 0 hours and 24 hours induction. For each sample, pellet and supernatant were stored. The pellets were transferred into 400 µl 12.5% TCA. 1 ml of the supernatant was added to 667 µl 12.5% TCA. Except for the 24 hours samples, where only approximately 800 µl of supernatant was available but added to the same amount of TCA. The samples were stored at -20℃ overnight. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Colony PCR AGA2-CIPA3-His into pHIPZ7 transformation 8-9-18 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#cellulosephosphorylation" target="_blank">colony PCR protocol</A>. FW primer: CIPA3A-FW. RV primer: CIPA3C-RV. Annealing temp: 55 degrees C. </P><P><B>Results</B></P><P>The size of the fragment is approximately 2600. Lane 3 and 8 show a band at the desired height.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculation of overnight cultures positive hits colony PCR </P><P>Inoculate the positive colonies in 4 ml LB+Amp. Grow overnight at 37 degrees C. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Preparation of Verduyn+ G418 plates with appropriate amino acids </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#LB-plates" target="_blank">Verduyn protocol</A>. Add 60 ul Histidine stock solution and 300 ul Methionine, Leucine and uracil stock solutions to plate and let plates dry. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Restreak colonies 1 and 3 BY4741-p414 KanMX TEF1-Cas9-Tcyc </P><P>Restreak colonies onto fresh YPD+g418 plate.  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate overnight culture BY4741-p414 KanMX TEF1-Cas9-Tcyc to check viability </P><P>- Add colony to 5 ml + 20 ul 250x G418. </P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday September 11th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> SDS-PAGE and Western blot of protein samples 10-09-18 </P><P>The samples were taken from the -20℃ storage and processed further according to the protocol. The samples were loaded onto two SDS-PAGE gels. One of the gels was stained, the other was used for Western blotting. The samples were loaded in the following order: Ladder - 1882(1)(Pellet)(0 hours), 1882(2)(P)(0), 1883(1)(P)(0), 1883(2)(P)(0), 1882(1)(P)(24), 1882(2)(P)(24), 1883(1)(P)(24), 1883(2)(P)(24) - empty well -  1µM OsmY positive control (obtained from A. Iyer).</P><P>Ladder 4 µl, sample 1&amp;2 20 µl, the rest 15 µl.</P><P>The PVDF membrane was incubated overnight in Blocking buffer in the fridge.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Run PCR on ligation of synthetic PAL2 fragment to amplify PAL2 gene with and without C terminal Histag </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Phire-PCR" target="_blank">Phusion PCR protocol</A>. PCR1: primers: PAL2C-FW and PAL2A-RV, template PAL2 ligation, annealing temp 55 degrees C. PCR2: primers: PAL2C-FW and PAL2B-RV, PAL2 ligation, annealing temp 55 degrees C.  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction and ligation of ss1-myc-BGLI-DocA into pHIPZ7 (product from here on out pHIPZ7-BGLI) </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">Restriction protocol</A>. BamHI and XhoI used as restriction enzymes. See <A href="https://2018.igem.org/Team:Groningen/Protocols#restriction-digest" target="_blank">ligation protocol</A>. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of pHIPZ7-BGLI ligation product into DH5alpha </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Protocols" target="_blank">Ecoli transformation protocol</A>. Use LB+Amp plates. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Isolate plasmids overnight cultures positive hits colony PCR 10-9-18 CIPA3 </P><P>Use plasmid miniprep kit.  </P></DIV></LI><LI><DIV class="collapsible-header"> Wednesday September 12th </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Transforming 2 plasmids from the distribution kit into E.coli </P><P>From the distribution kit plate 7, the contents of wells 3a and 3c are resuspended into 10 ul of MQ water. 1 ul of each strain is transformed into E.coli DH5a, according to E.coli standard transformation protocol. The transformed cultures are plated onto LB chloramphenicol plates and incubated overnight at 37 degrees celsius. </P><P>A CIPA3 sample is sent away for sequencing to Eurofins Genomics, using the GenSeq-fw primer. Colony 3: Seq. ID 98FB00 Colony 7: Seq. ID 98FB01</P><P>Riannes blot from yesterday is incubated in α-His primary antibodies, according to the <A href="https://2018.igem.org/Team:Groningen/Protocols#WB-His" target="_blank">antibody incubation protocol.</A></P><P>The gel is imaged, but no bands are visible.</P></DIV></LI><LI><DIV class="collapsible-header"> Thursday September 13th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Colony PCR BGLI </P><P>The same protocol as described on 29-8-18 was used. Reactions A-J contained 1 single colony, reactions K-N five. </P></DIV></LI><LI><DIV class="collapsible-header"> Friday September 14th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Colony PCR BGLI agarose gel </P><P>4 µl ladder, 10 µl sample loaded </P><FIGURE><FIGCAPTION><I>Gel 1: Ladder - A - B - C - D - E - F - G</I></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><I>Gel 2: Ladder - H - I - J - K - L - M - N</I></FIGCAPTION></FIGURE></DIV></LI><LI><DIV class="collapsible-header"> Sunday September 16th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> PCR on BGLI fragment ligation product and PAL2 fragment ligation product </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#colony-PCR" target="_blank">Phire PCR protocol</A>.  </P><P>- BGLI primers: BGLI-fragA-fw and BGLI-fragB-rv, annealing temp 52 degrees C, template: BGLI fragment ligation product. 
            - PAL2 primers: PAL2A-FW and PAL2A-RV, annealing temp 55 degrees C. template: PAL2 fragment ligation product.</P></DIV></LI><LI><DIV class="collapsible-header">Week 38</DIV></LI><LI><DIV class="collapsible-header"> Monday September 17th </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Start cultures to make a growth curve for strains 1882 and 1883 in a plate reader </P><P>11 cultures are started, including induced and uninduced cultures, positive and negative controls.
            1883.1 2 hrs galactose induction, to be cultured on cellobiose
            1883.2 2 hrs galactose induction, to be cultured on cellobiose
            1883.1 6 hrs galactose induction, to be cultured on cellobiose
            1883.2 6 hrs galactose induction, to be cultured on cellobiose
            1883.1 not induced, to be cultured on cellobiose
            1883.2 not induced, to be cultured on cellobiose
            1883.1 6 hrs galactose induction, to be cultured without carbon source
            1883.2 6 hrs galactose induction, to be cultured without carbon source
         1883.1 positive control, 6 hrs galactose induction, to be cultured with 1.25% glucose
            1883.2 positive control, 6 hrs galactose induction, to be cultured with 1.25% glucose
            negative control (wildtype s. cerevisiae, 6 hrs galactose induction, cultured on cellobiose) </P><P>All cultures are grown on 20 ml Verduyn medium with 0.5% fructose and uracil, and leucine and tryptophan added for the negative control. Culture at 30 degrees Celsius with agitation.</P><P><B>Who: Owen</B></P><P><B>Aim</B> PCR on pUC57-PAL2, ligation product BGLI, synthetic PAL2 gene part1 and synthetic PAL2 gene part2 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#colony-PCR" target="_blank">PCR protocol Phire</A>. Different annealing temperatures are used on all PCR mixes to increase the chance of amplifying a fragment 50, 51, 52 and 53 degrees C. 
            - Mix1: primers: PAL2-BB-fix-fw, PAL2-BB-fix-rv, template: pUC57-PAL2. 
            - Mix2: primers: BGLI-fragA-fw, BGLI-fragB-rv, template: ligation product BGLI.
            - Mix3: primers: PAL2.1-FW, PAL2.1-RV, template: synthetic PAL2 gene part1.
            - Mix4: primers: PAL2.2-FW, PAL2.2-RV, template: synthetic PAL2 gene part2. </P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday September 18th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Results</B></P><P>Lane 2-5 contain synthetic PAL2 gene part1 PCR fragments. Very faint bands can be observed at the desired size of 1300 bp. Lane 6-9 contain synthetic PAL2 gene part2 PCR fragments. No bands can be observed. Lane 11 to 14 contain pUC57-PAL2 PCR fragment. Amplified fragment should be 2600, bands are present at the desired height. Lane 15 to 18 contain ligation product BGLI PCR fragments. Bands can be seen at 3kB, the height of the desired fragment. </P></DIV></LI><LI><DIV class="collapsible-header"> Wednesday September 19th </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> to induce, wash, and distribute the cultures from 17-09-2018 </P><P>At 10.30 hrs the cultures to be induced for 6 hrs are induced by adding 2.0% final concentration of galactose.
            At 14.30 hrs the cultures to be induced for 2 hrs are induced by adding 2.0% final concentration of galactose.
            At 16.30 hrs the cultures are spinned down in falcon tubes at 4000 rpm in a large centrifuge. Samples are washed 2x with Verduyn medium without a carbon source.
            The cultures are mixed with supplements as described on 17-09-2018 and 200 ul of each culture are loaded into a 96 well plate, in duplo, with starting OD’s of 0.2, 0.1, and 0.05. 3 Verduyn cellobiose blanks and 2 Verduyn cellobiose glucose blanks are loaded as well.</P><P>The plate reader is set to measure, for 48 hrs, once every 20 minutes. The plate is agitated for 30 seconds before every measurement, and the interior is kept at 30 degrees Celsius over the duration of the experiment.
            The plate reader is a Tecan Geneos.</P><P>1 ml of each culture is taken before the wash step and frozen, to be used to check gene expression via Western Blot.</P><P><B>Who: Benno</B></P><P><B>Aim</B> The aims of the RP-HPLC validation runs are to evaluate the available RP-HPLC system in the Linnaeusborg, to discover the retention time of styrene, to test if styrene elutes at all, to optimize the available parameters of the RP-HPLC for optimal measurement, to validate the sample preparation and to come up with a standardizeable procedure that can be employed for qualitative styrene analysis for the rest of the project </P><P>The shimadzu HPLC system was equipped with a C18 column and with water plus 0 - 100 % acetonitrile as gradient mobile phase. Two runs were done. For the first run 0,1 molar styrene in ethyl acetate solution was injected to find out the retention time and test the Diode Array Detector.
            For the second run 0,5 ml ethylacetate, 0,5 ml lysed yeast cell culture and a drop 20 µl styrene were vortexed for 15 minutes. The apolar phase was then filtered over a 0,22 nm nylon membrane and injected into the HPLC system.  </P><P>The styrene only eluted at a high percentage of acetonitrile as is expected, looking at the log P of 2,7. Therefore another gradient mobile phase was chosen: water with methanol 0 - 100 %. As styrene eluted at a composition of 50 % water 50 % methanol it was chosen to validate a method with a gradient of 30 - 100 % methanol. The retention time of styrene in this run was 17,5 minutes. The UV spectrum recorded at this point in time confirms that styrene is what elutes from the column.</P></DIV></LI><LI><DIV class="collapsible-header"> Thursday September 20th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Colony PCR BGLI </P><P>The same protocol as described on 29-8-18 was used. Reactions A-O contained 1 single colony, reactions P-Y 2. The reaction volume was 15 µl. The colonies were taken from the plates of 28.08.18. Gels were run by Owen. </P><FIGURE><FIGCAPTION><I>Samples A-R</I></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><I>Samples S-Y</I></FIGCAPTION></FIGURE><P><B>Who: Owen</B></P><P><B>Aim</B> Run gel colony PCR Rianne on pHIPZ7-ss1-myc-BGLI-DocA (also pHIPZ7-BGLI from here on) 26 samples </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE" target="_blank">Protocol agarose gel</A></P><P><B>Results</B></P><P>No bands are visible at 3kB the desired height. Gels not shown.</P><P><B>Who: Rianne</B></P><P><B>Aim</B> Preparation SDS-PAGE samples </P><P>Samples from Jan Marten from 19-09-18 were processed following the <A href="https://2018.igem.org/Team:Groningen/Protocols#ecoli-transformation" target="_blank">SDS-PAGE yeast extract protocol</A> until the -80 step. Then left in the -80 overnight. </P></DIV></LI><LI><DIV class="collapsible-header"> Friday September 21st </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Colony PCR BGLI </P><P>PCR was performed on the plates of 11-9-18. 50 colonies were picked in 15 µl reactions. The agarose gels were run by Owen.  </P><FIGURE><FIGCAPTION><I>Samples 1-36</I></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><I>Samples 37-50</I></FIGCAPTION></FIGURE><P><B>Who: Rianne</B></P><P><B>Aim</B> SDS-PAGE and Western blot </P><P>Samples of 20-9-18 were taken out of -80 and processed further according to the sample preparation protocol. SDS-PAGE and blotting were performed according to the <A href="https://2018.igem.org/Team:Groningen/Protocols#Yeast-SDS" target="_blank">SDS protocol</A> and <A href="https://2018.igem.org/Team:Groningen/Protocols#T4-ligation" target="_blank">blotting protocol</A>. After blotting, the PVDF membrane was incubated in blocking buffer in the fridge over the weekend. </P></DIV></LI><LI><DIV class="collapsible-header"> Saturday September 22nd </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> To miniprep and restrict Puc57kan Pal2 </P><P>1.5 ml of e.coli containing Puc57kan Pal2 (induced yesterday) is miniprepped. Eluate in 50 ul.
            1) 25 ul is mixed with 10 ul Jena 10x universal buffer, 1 ul BamHI and 1 ul PstI, and 63 ul MQ
            2) 12.5 ul is mixed with 5 ul Jena 10x universal buffer, 1 ul BamHI, and 31.5 ul MQ
            3) 12.5 ul is mixed with 5 ul Jena 10x universal buffer, 1 ul PstI, and 31.5 ul MQ
            Incubate 2 hrs at 37 degrees, and store overnight at 4 degrees. </P><P>E.coli pHIPZ7 is inoculated into LB ampicillin and grown overnight at 37 degrees.</P></DIV></LI><LI><DIV class="collapsible-header"> Sunday September 23rd </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> To miniprep e.coli pHIPZ7, restrict, and load and run an agarose gel </P><P>E. coli pHIPZ7 is miniprepped.
            A gel is loaded:
            Slot 1: Ladder (Jena 1kb ladder). Load 3 ul.
            Slot 2: Unrestricted pHIPZ7 plasmid
            Slot 3: Sample 1 from yesterday, 5 ul, mixed with 1 ul 5x loading dye
            Slot 4: Sample 2 from yesterday, 5 ul, mixed with 1 ul 5x loading dye
            Slot 5: Sample 3 from yesterday, 5 ul, mixed with 1 ul 5x loading dye
            Run for 30 minutes at 90 mA. </P><P>It appears that 1) from yesterday has cut only once, the same goes for 2) and 3).
            Add 2 ul of PstI to 1) and 2), and add 1 ul BamHI to 3).
            Incubate 2 hrs at 37 degrees Celsius.</P><P>Mix 5 ul of each sample with 1 ul 5x loading dye, load in the same order as above, and run for 30 minutes at 90 mA.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Run gel colony PCR Rianne on pHIPZ7-BGLI 50 samples </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE" target="_blank">Protocol agarose gel</A>. </P><P>Results</P><P>Lane 3 shows a band at the desired height of 3kB. Other samples not shown as they were all negative. </P><P><B>Who: Owen</B></P><P><B>Aim</B> PCR on ss1-FLAG-CBHI-DocS (also CBHI from here on) fragment and ss1-V5-EGII-DocS (also ss1-EGII from here on) </P><P>As previous PCR reactions were carried out using phire polymerase mutations were observed in sequencing runs of the fragments. To create fragments without mutations Phusion polymerase is used here. </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Phire-PCR" target="_blank">Phusion PCR protocol</A>. 
            - PCR CBHI: primers: CBHII1-FW, CBHII1-RV, template: ss1-FLAG-CBHI-DocS fragment. Annealing temperature: 52 degrees C. 
            - PCR ss1-EGII: primers: EGII1-FW, EGII1-RV, template: ss1-V5-EGII-DocS fragment. Annealing temperature: 55 degrees C. </P></DIV></LI><LI><DIV class="collapsible-header">Week 39</DIV></LI><LI><DIV class="collapsible-header"> Monday September 24th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Western blot </P><P>The PVDF membranes were stained following the <A href="https://2018.igem.org/Team:Groningen/Protocols#T4-ligation" target="_blank">Western Blot protocol</A>. The first antibody was incubated for 1 hour instead of 2. </P><P>Unfortunately, the Western blot was not successful. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Run 0,8% agarose gel with CBHI and ss1-EGII PCR reaction 23-9-18 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE" target="_blank">agarose gel protocol</A>. 0,8 % agarose gel run to cut out CBHI bands for gel extraction. Bands could not be seen with a blue light flashlight.  </P><P><B>Results</B></P><P>Lane 1 and 2 contain ss1-EGII PCR. No clear band can be seen. Lane 4 and 5 contain CBHI PCR reaction. Faint bands can be observed, but the DNA concentration is not high enough for gel extraction.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction of pSB1C3-BBa_K525998 (vector) and pUC57-PAL2 PCR fragment 18-9-18 for cloning </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">Restriction protocol</A></P><P><B>Who: Owen</B></P><P><B>Aim</B> PCR reaction to amplify CBHI: second attempt </P><P>As the previous attempt on 23-9-18 didn't give enough signal a second PCR reaction is attempted with more cycles.  </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Phire-PCR" target="_blank">PCR protocol Phusion</A>. 
            - PCR CBHI: primers: CBHII1-FW, CBHII1-RV, template: ss1-FLAG-CBHI-DocS fragment. Annealing temperature: 52 and 55 degrees C. 34 cycles.</P><P><B>Who: Owen</B></P><P><B>Aim</B> PCR reaction to amplify PAL2 and ss1-EGII: second attempt </P><P>As the first attempt to amplify ss1-EGII failed and the primers for the first PAL2 amplification were incorrect, 4 PCR reactions with a decreasing annealing temperature over the first 15 cycles was attempted. When the lowest temperature is reached the annealing temperature is kept constant for the next 15 cycles. </P><P>- Mastermix PCR PAL2: primers: PAL2-BB-fix-fw, PAL2-BB-fix-rv. template: pUC57-PAL2.
            - Mastermix PCR ss1-EGII: primers: EGII1-FW, EGII1-RV, template: ss1-V5-EGII-DocS fragment.
            - Temperature gradients used: 63-55, 60-56, 56-52, 52-48 degrees C over 15 cycles. </P><P><B>Who: Jacques</B></P><P><B>Aim</B> Phosphorylation and characterization of cellulose at position 6</P><P><U>Protocol followed as described in the paper mentioned.</U><I>Percival Zhang, Y. H., Cui, J., Lynd, L. R. and Kuang, L. R. A transition from Cellulose Swelling to Cellulose Dissolution by o-Phosphoric Acid: Evidence from Enzymatic Hydrolysis and Supramolecular Structure. Biomacromolecules, 7, 644-648 (2006)</I></P><P>Cellulose (200 mg, 0.62 mmol) and distilled water (0.6 mL) were added to a 50 mL glass tube. Slowly, ice cold o-phosphoric acid (15.2M, 10 mL) was added to the mixture under vigorous stirring. The suspension turned into transparent slurry within minutes.
            This mixture was stirred for one hour on ice. Subsequently, 40 mL of ice cold water was added resulting in a white clouded suspension. This mixture was centrifuged at 5000g at 4°C for 20 minutes. Then the supernatant was discarded and the pellet resuspended in ice cold water. This mixture was centrifuged at 5000g at 4°C for 10 minutes. The washing cycle was repeated 4 times. Afterwards the pellet was washed one time with Na<SUB>2</SUB>CO<SUB>3</SUB> (2M, 1 mL) and ice cold distilled water (39 mL). Finally, the pellet was washed with distilled water until pH 7.</P><P><B>Assumptions made for a successful phosphorylation reaction</B></P><P>The <SUP>31</SUP>p-NMR spectrum showed one peak (-4 ppm), expected to be from phosphoric acid. The lack of the phosphate peak from cellulose is supposed to be due to lack of solubility in D<SUB>2</SUB>O. Nevertheless, it is suspected that the phosphorylation of cellulose at the 6th position was successful. Based on the same observation made as were described by the protocol followed[1]. Furthermore, the phosphorylated cellulose dissolved in phosphate buffer, while cellulose was insoluble. Moreover, the cellulase activity assay showed the highest activity for the phosphorylated cellulose, expected to be due to solubility. As a negative control cellulose was chosen, which showed significant less activity.</P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday September 25th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B>  Run 0,8 % agarose gel with CBHI PCR 24-9-18 and 1% agarose gel with ss1-EGII and PAL2 PCR reaction 24-9-18 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE" target="_blank">Agarose gel protocol</A>.  </P><P><B>Results</B></P><P>0.8%gel: No clear bands were visible under blue light for the CBHI PCR reaction. (Gel not shown)</P><P>1%gel: Lane 1-4 contain ss1-EGII PCR fragments, bands are visible at the desired size of 1700 bp. Lane 6-9 contain PAL2 PCR fragments, bands are visible at the desired size of 2300 bp.</P><P><B>Who: Owen</B></P><P><B>Aim</B> PCR reaction on ss1-myc-BGLI-DocA part1 and ss1-myc-BGLI-DocA part2 to amplify fragments for ligation </P><P>To increase the chance of amplification 4 different PCR reactions with decreasing annealing temperature over 15 cycles are performed. See <A href="https://2018.igem.org/Team:Groningen/Protocols#Phire-PCR" target="_blank">Phusion protocol</A>. 
            - PCR mix1: primers: BGLI-fragA-fw, BGLI-fragA-rv, template: ss1-myc-BGLI-DocA part1.
            - PCR mix2: primers: BGLI-fragB-fw, BGLI-fragB-rv, template: ss1-myc-BGLI-DocA part2.
            - Temperature gradients used: 63-55, 55-50, 52-48 and 50-46 degrees C decreasing from the highest to the lowest temperature over 15 cycles.  </P></DIV></LI><LI><DIV class="collapsible-header"> Wednesday September 26th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Verifying PCR on BGLI fragments 25-9-18 by running them on an agarose gel </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE" target="_blank">Agarose gel protocol</A></P><P><B>Results</B></P><P>Lane 1-4 contain samples of ss1-myc-BGLI-DocA part1 PCR reactions 25-9-18. A band can be seen at the desired height of 1300 bp. Lane 6-9 contain samples of ss1-myc-BGLI-DocA part2 PCR reactions 25-9-18. A band can be seen at the desired height of 1700 bp. </P></DIV></LI><LI><DIV class="collapsible-header"> Thursday September 27th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction and ligation of BGLI fragments and subsequent PCR to amplify the entire fragment </P><P>- Add PCR reactions of BGLI fragment 1 together. Do the same for fragment 2.
            - Use PCR clean up kit to clean DNA.
            - See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">restriction protocol:</A> transformation insert part. Use 2 ul frag 1 and 2 ul frag 2 for restriction.
            - See <A href="https://2018.igem.org/Team:Groningen/Protocols#restriction-digest" target="_blank">ligation protocol</A>. 
            - See <A href="https://2018.igem.org/Team:Groningen/Protocols#Phire-PCR" target="_blank">Phusion polymerase protocol</A>. Perform PCR on the ligated product. 
            - PCR mix: primers: PAL2-fragA-fw, PAL2-fragB-rv, template: 1 ul ligation product. Annealing temp: 55 degrees C. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Amplification of His-EGII from plasmid pYD1-CipA3-EGII for cloning </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Phire-PCR" target="_blank">Phusion PCR protocol</A>. </P><P>- PCR mix: Bio-His-EG-FW1, Bio-His-EG-RV1, template: pYD1-CipA3-EGII. Annealing temp 55 degrees C.</P><P><B>Results</B></P><P>Lane 1-5, 7-9 contain PCR reaction of His-EGII. A band is observed at the expected height of 1300 bp. Lane 10-12, 14-18 contain PCR reaction of BGLI fragment ligation PCR 27-9-18. No bands can be observed indicating a failed PCR.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Amplification of zeocin cassette from pHIPZ7 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Phire-PCR" target="_blank">Phusion PCR protocol</A>. 
            - PCR mix1: primers: Zcas1-FW, Zcas4-RV, template: pHIPZ7.
            - PCR mix2: primers: Zcas2-FW, Zcas3-RV, template: pHIPZ7.
            - PCR mix3: primers: Zcas3-FW, Zcas2-RV, template: pHIPZ7.
            - PCR mix4: primers: Zcas4-FW, Zcas1-RV, template: pHIPZ7.
            - All PCR mixes are amplified at 52, 53, 54...60 degrees C. </P><P><B>Results</B></P><P>All Lanes contain PCR product of the zeocin cassette. A band can be seen at 1300 bp indicating a succesffull PCR. Samples from lane 1-7 are used for subsequent cloning.</P></DIV></LI><LI><DIV class="collapsible-header"> Friday September 28th </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> PCR on BGL1 </P><P>PCR is performed on the ligated BGL1 fragments, using primers BGL1-fragA-fw and BGL1-fragB-rv. In a mix: 100 ul Thermo Fisher 2x Phusion Master mix 1 ul template 0.5 ul of each primer 98 ul MQ water Mix and pipette 20 ul into 8 pcr cups. </P><P>Perform gradient PCR: 98 degrees for 3 minutes initial denaturation 98 degrees for 10 seconds denaturation 60-52 degrees gradient 30 seconds annealing 72 degrees 2 minutes extension Repeat denaturation, annealing, extension 30x 72 degrees 5 minutes final extension</P><P>Samples are run on gel and imaged by Owen. No bands were visible, gel not shown</P><P><B>Who: Owen</B></P><P><B>Aim</B>  Restriction and ligation of: ss1-EGII into pHIPZ7, ss1-EGII into pSB1C3, His-EGII into pSB1C3, PAL2 into pSB1C3, His-EGII into pSB1C3T7(Part:BBa_K525998), PAL2 into pSB1C3T7, PAL2 into pRS313+ s promHxt7-MCS-terHxt7 (pRS313-Hxt7 from here on),  and the zeocin cassette(Zcas from here on) into pSB1C3 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">Restriction</A> and <A href="https://2018.igem.org/Team:Groningen/Protocols#restriction-digest" target="_blank">ligation</A> protocols. 
            Restriction enzymes used:
            - ss1-EGII → pHIPZ7 BamHI,XhoI
            - ss1-EGII → pSB1C3 XbaI, SpeI
            - His-EGII → pSB1C3-T7-RBS SpeI, Pst1
            - Pal2 → pSB1C3-T7-RBS SpeI, PstI
            - PAL2 → pRS313-Ht7 SpeI, XbaI
            - Zcas→ pSB1C3 EcoRI, PstI
            - HisEGII → pSB1C3 EcoRI, PstI
            - PAL2 → pSB1C3 EcoRI, PstI</P><P>FastAP step was forgotten, another restriction and ligation reaction are carried out the next day.</P></DIV></LI><LI><DIV class="collapsible-header"> Saturday September 29th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction and ligation of: ss1-EGII into pHIPZ7, ss1-EGII into pSB1C3, His-EGII into pSB1C3, PAL2 into pSB1C3, His-EGII into pSB1C3T7(Part:BBa_K525998), PAL2 into pSB1C3T7, PAL2 into pRS313+ s promHxt7-MCS-terHxt7 (pRS313-Hxt7 from here on),  and the zeocin cassette(Zcas from here on) into pSB1C3. Second attempt </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">Restriction</A> and <A href="https://2018.igem.org/Team:Groningen/Protocols#restriction-digest" target="_blank">ligation</A> protocols. Restriction enzymes used: - ss1-EGII → pHIPZ7 BamHI,XhoI - ss1-EGII → pSB1C3 XbaI, SpeI - His-EGII → pSB1C3-T7-RBS SpeI, Pst1 - Pal2 → pSB1C3-T7-RBS SpeI, PstI - PAL2 → pRS313-Ht7 SpeI, XbaI - Zcas→ pSB1C3 EcoRI, PstI - HisEGII → pSB1C3 EcoRI, PstI - PAL2 → pSB1C3 EcoRI, PstI </P></DIV></LI><LI><DIV class="collapsible-header"> Sunday September 30th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of ligation products 29-9-18 into DH5alpha and BL21(DE3) and BBa_K1692004 into BL21(DE3) </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Protocols" target="_blank">E coli transformation protocol</A>.</P><P>E. coli strains and plates used for each transformation:
            - EGII → pHIPZ7 DH5 // LB Amp 3
            - EGII → pSB1C3 DH5 // LB chlor 3
            - His-EGII → pSB1C3-T7-RBS BL21(DE3) // LB chlor 3
            - Pal2 → pSB1C3-T7-RBS BL21(DE3) // LB chlor 3
            - His-EGII → pSB1C3 DH5 // LB chlor
            - PAL2 → pSB1C3 DH5 // LB chlor
            - pSB1C3-T7-RBS-PAL-Stan → BL21 (DE3) // LB chlor 1
            - PAL2 → pRS313 DH3 prom DH5 // LB Amp 3
            - Zeocin → pSB1C3 DH5 // LB chlor
            DH5=DH5alpha
            Amp=Ampicillin
            Chlor=Chloramphenicol</P></DIV></LI><LI><DIV class="collapsible-header">Week 40</DIV></LI><LI><DIV class="collapsible-header"> Monday October 1st </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction and ligation of pSB1C3T7 as control for transforming His-EGII → pSB1C3-T7-RBS and Pal2 → pSB1C3-T7-RBS </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">Restriction</A> and <A href="https://2018.igem.org/Team:Groningen/Protocols#restriction-digest" target="_blank">ligation</A> protocols. Only 50 ng of pDNA is used as only production of a control is necessary.  </P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday October 2nd </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Colony PCR on pSB1C3T7-His-EGII ligation product transformation and pSB1C3T7-PAL2 1-10-18 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#cellulosephosphorylation" target="_blank">Colony PCR protocol</A>. - Primers and annealing temperature for both colony PCR’s: VF2, RV and 55 degrees C. 8 colonies of each transformation tested.  </P><P><B>Results</B></P><P>Lane 1-5, 7-9 show colony PCR of His-EGII. A band is observed at the desired size of 1600 bp. Lane 10-12, 14-18 contain colony PCR of PAL2. Lane 10 and 17 show a single band at the desired size of 2600 bp. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Plasmid isolation for restriction analysis of pSB1C3-Zcas colony 1-8, pRS313-Hxt7-PAL2 colony 1-8. pHIPZ7-ss1-EGII colony 1-4</P><P>Use plasmid miniprep kit to isolate plasmids.</P></DIV></LI><LI><DIV class="collapsible-header"> Wednesday October 3rd </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Transform E. coli BL21 DE3 Star with Pal2 and His-EGII </P><P>E. coli BL21 DE3 Star is transformed with pSB1C3 T7 promotor + Pal2 colonies 1 and 7, pSB1C3 T7 promotor + His-EGII colonies 1 and 2, and Bba_K1692003 (pSB1C3 T7 promotor + Pal2).
            Thaw cells on ice, make 100 ul aliquots, add 100 ng of DNA to each aliquot, incubate for 30 minutes. Heat shock 1 minute at 42 degrees celsius, and incubate on ice for 2 minutes. Add 900 ul LB medium, and incubate at 37 degrees celsius for 1 hour. Plate on LB chloramphenicol plates, in 10% and 90% fractions. Incubate overnight at 37 degrees celsius.</P><P>Strains are inoculated into 20 ml of Verduyn medium with 0.5% fructose and appropriate amino acids. </P></DIV></LI><LI>Negative control (strain without cellulosome plasmids)</LI><LI>CB + 1883.1 in 3 cultures (for 6 hrs and 2 hrs galactose induction, and for + control)</LI><LI>CB + 1883.2 in 3 cultures</LI><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction analysis of isolated plasmids pSB1C3-Zcas colony 1-8 and pRS313-Hxt7-PAL2 colony 1-8 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">restriction protocol</A>. Restriction enzymes SpeI and XbaI used for pRS313-Hxt7-PAL2. Restriction enzymes EcoRI and PstI used for pSB1C3-Zcas. </P><P><B>Results</B></P><P>No positive hits observed. (Gel not shown).</P><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of pHIPZ7 into YPH499 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Agarose-gel" target="_blank">yeast transformation protocol</A>. 200ng pHIPZ7 used for transformation. Transformation with MQ as negative control. YPD Zeocin plates used for screening.  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Colony PCR on pSB1C3-Zcas and pRS313-Hxt7-PAL2 transformations 1-10-18 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#cellulosephosphorylation" target="_blank">Colony PCR protocol</A>. - Colony PCR Zcas: primers: VF2 and VR, annealing temp 55 degrees C. 20 colonies tested. - Colony PCR pRS313-PAL2:PAL2-seq-fw-internal, R T7, annealing temp 55 degrees C. 20 colonies tested. </P><P><B>Results</B></P><P>Lane 1-6 contain pRS313-Hxt7-PAL2 colony PCR samples. lane 2 and 3 contain a band at the desired height of 1600 bp. Other Lanes not shown as no positives were present. Primers are chosen in such a manner that PCR amplification is only possible with orientation of the insert into the backbone. </P><P>Lane 1-4, 6-11 contain colony PCR pSB1C3-Zcas. Lane 1 and 11 show 2 bands. 1 at the desired height and one at 300 bp which corresponds to an empty vector. Colonies corresponding to lane 1 and 11 are analysed further using restriction analysis. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Propagating culture of YPH499 for transformation of pRS313-Hxt7-PAL2 </P><P>- Take 100 ul YPH499 overnight culture and transfer it to 20 ml YPD and growing it at 30 degrees C overnight. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Overnight culture positive hits colony PCR pRS313-Hxt7-PAL2 colony 2 and 3 for plasmid isolation </P><P>- Inoculate the cultures is 4 ml LB+Amp and grow at 37 degrees C overnight. </P><LI><DIV class="collapsible-header"> Thursday October 4th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Plasmid prep PAL2 for Owen </P><P>E. coli cultures inoculated by Owen were taken from the incubator and plasmids were miniprepped, resulting in the following concentrations: </P><P>PAL2 2 = 571.85 ng/µl
            PAL2 3 = 696.20 ng/µl</P><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction analysis pRS313-Hxt7-PAL2 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">restriction protocol</A>. SpeI and XbaI used as restriction enzymes. </P><P><B>Results</B></P><P>Only one band can be seen on the gel for each colony analyzed indicating that restriction of the plasmid failed. (Gel not shown).</P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculation of colonies corresponding to lane 1 and 11 of the pHIPZ7-Zcas colony PCR for plasmid isolation </P><P>Inoculate colonies in 4 ml LB+Chlor. Grow overnight at 37 degrees C.  </P></DIV></LI><LI><DIV class="collapsible-header"> Friday October 5th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> Plasmid prep Zeocin cassette for Owen </P><P>E. coli cultures inoculated by Owen were taken from the incubator and plasmids were miniprepped, resulting in the following concentrations: </P><P>Zeocin 9 = 152.38 ng/µl
 Zeocin 18 = 206.75 ng/µl</P><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction analysis of pHIPZ7-Zcas to confirm correct integration of fragment </P><P>Isolate plasmids from the overnight cultures of colonies corresponding to lane 1 and 11 of the pHIPZ7-Zcas colony PCR using a plasmid miniprep kit. </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">Restriction analysis protocol</A>. EcoRI and SpeI are the restriction enzymes used. </P><P><B>Results</B></P><P>Lane 1 and 2 contain negative controls. Lane 4 and 5 contain restricted pHIPZ7-Zcas. Restricting the plasmid with EcoRI and SpeI should give a fragment of 2 kB and 1300 bp. Lane 5 shows the desired bands, but the intensity of the upper band is higher than the lower band. This could mean that the colony consist of 2 different transformants. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Restreaking the ahlf positive colony of the pHIPZ7-Zcas colony to obtain the correct transformant </P><P>Restreak colony on a LB+chlor plate and grow at 30 degrees overnight.  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of pRS313-Hxt7-PAL2 and pRS313-Hxt7-PAL3+pYD1-CipA3-EGII+pRS425-CBHII-BGL1 into YPH499 </P><P>Even though the restriction analysis on the 2 positive colony PCR hits was negative the plasmids are still transformed into YPH499. This is because a positive colony PCR was only possible when the insert was integrated into the plasmid, due to the selection of primers. Both positive hits are transformed individually and together with the cellulase plasmids. </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#Agarose-gel" target="_blank">high efficiency yeast transformation protocol</A>. 200 ng of each plasmid is used for transformation. See <A href="https://2018.igem.org/Team:Groningen/Protocols#LB-plates" target="_blank">Verduyn protocol</A>. Verduyn plates with URA, LYS and ADE are used for transformation of the triple plasmid combo. Verduyn plates with URA, LYS, ADE, TRP and LEU are used for transformation of pRS313-Hxt7-PAL2. </P><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Start a cellulose degradation assay in a plate reader </P><P>At 10.00, the negative control, the positive controls, and the 6 hrs induction strains are induced with 2% galactose
            At 14.00, the 2 hrs induction strains are induced with 2% galactose.
            At 16.00, all strains are washed 2x with Verduyn medium without amino acids and carbon source.
            Resuspend in Verduyn medium without carbon source containing appropriate amino acids.
            All strains are loaded in triplo, with 1% phosphorylated cellulose, in a final volume of 200 µl. The positive control strains also contain 0.5% final concentration of glucose.</P><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Analyze results of cellulose degradation assay</P><P>The plate is removed from the plate reader, and the data is downloaded. Immediately a problem is clear. When looking at the plate it is obvious that all the cellulose in the cellulose samples has sticked to one corner of the well, and hasn’t been mixed through the sample properly. This likely happened during the loading of the 96-well plate, as the carbon sources were the first liquids pipetted into the wells. Since these were only 10-20 µl drops, they would have sticked to one corner and not mixed properly throughout the sample. No effort is made to process the data, as no useable information on cellulose will be available.</P></DIV></LI><LI><DIV class="collapsible-header"> Saturday October 6th </DIV><DIV class="collapsible-body"><P><B>Who: </B></P><P><B>Aim</B> Colony PCR of restreaked colony pSB1C3-Zcas 5-10-18 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#cellulosephosphorylation" target="_blank">Colony PCR protocol</A>. Primers: VF2, RV. Annealing temp: 55 degrees C. 18 colonies are tested. See <A href="https://2018.igem.org/Team:Groningen/Protocols#SDS-PAGE" target="_blank">agarose gel protocol</A>.</P><P><B>Results</B></P><P>All numbered lanes contain colony PCR product. A band can be observed at the desired height of 1600 bp, but also at approx 300 bp. The band at 300 bp corresponds to an empty vector. These results show that both plasmids are integrated into the transformant, but to verify this samples are send for sequencing. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction analysis of pSB1C3-PAL2 plasmids obtained during transformation 30-9-18 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">restriction analysis protocol</A>. EcoRI and SpeI used as restriction enzymes. </P><P><B>Results</B></P><P>Lane 1-6 contain restriction analyses of pSB1C3-PAL2. Restriction of the plasmid should form a fragment of 2kB and 2300bp. Samples from lane 1, 4 and 5 show a broader band than the other lanes, this is a good indication of the presence of the correct fragment.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction analysis of positive hits restriction analysis pSB1C3-PAL2 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">restriction protocol. Use restriction enzyme EcoRI. </A></P><P><B>Results</B></P><P>Lane 1-3 contain restricted plasmid. Lane 4-6 contain negative control. All bands appear at the desired height, giving no information as to the correctness of the plasmid. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Colony PCR Transformation of Zcas into pSB1C3 Matthijs </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#cellulosephosphorylation" target="_blank">Colony PCR protocol</A>. 18 colonies tested. Primers: VF2, VR. Annealing temp: 55 degrees C. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate colonies 1, 10 and 14 of the colony PCR pSB1C3-Zcas. For plasmid isolation </P><P>Inoculate cultures in 4 ml LB+Chlor and grow overnight at 37 degrees C.  </P></DIV></LI><LI><DIV class="collapsible-header"> Sunday October 7th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction analysis of overnight cultures 1,10 and 14 of the colony PCR pSB1C3-Zcas and pSB1C3-His-EGII from transformation 30-9-18 </P><P>Use plasmid miniprep kit to isolate plasmids. See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">restriction protocol</A>. One restriction analysis is performed using NotI creating 2 fragments and one using EcoRI to create a linearized backbone. The second reaction is only performed for pSB1C3-Zcas.   </P><P><B>Results</B></P><P>Lane 1-6 contain restriction analyses of pSB1C3-His-EGII. Cutting pSB1C3-His-EGII should gives a fragment of 2kB and a fragment of 1300 bp. This can be seen in the first 2 lanes and a very faint band at 1300 bp in the third lane. Lane 7-9 contain pSB1C3-Zcas restricted with NotI. This should give 2 bands, one of 2kB and 1 of 1300 bp. No bands are present at 1300bp. Lane 10-12 contain pSB1C3-Zcas restricted with EcoRI. A band at 3.3 is expected. This can be seen in all 3 lanes, but there is also a band at 2kB. This band has the same size as linearized pSB1C3. As the bottom ladder bands are less faint than the upper ones, we hypothesized that the imager might detect higher bands better. To test this we turned the gel upside down and imaged it. Bands at 1300 bp can be seen now in lane 1-9. (Gel not shown).</P><P><B>Who: Owen</B></P><P><B>Aim</B> Restriction and ligation of Zcas into pSB1C3 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">restriction protocol</A>. EcoRI and PstI are the restriction enzymes used. See <A href="https://2018.igem.org/Team:Groningen/Protocols#restriction-digest" target="_blank">ligation protocol</A>. </P></DIV></LI><LI><DIV class="collapsible-header">Week 41</DIV></LI><LI><DIV class="collapsible-header"> Monday October 8th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Checking YPH499 transformations 5-10-18 for contamination </P><P>As one of our other yeast transformations using the same strain was contaminated we decided to check the transformations for contamination. </P><P>One colony was added to 10ul MQ and pipetted onto a glass slide. A microscope using a 400x lense was used to check for contamination.</P><P><B>Results</B></P><P>No contamination was observed. No image shown as microscope did not contain imager. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Transformation of ligation 7-10-18 Zcas into pSB1C3 to E. coli DH5alpha </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#competent-ecoli" target="_blank">transformation protocol</A>. LB + Chlor, LB + Zeocin and LB + Zeocin + Chlor plates used. 100 ul and 200 ul plated on a plate of each kind for the transformation. 200 ul control plated on a plate of each kind. </P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculation of YPH499, YPH499 + pRS313-Hxt7-PAL2 (YPHPAL2 from here on) and YPH499 + pRS313-Hxt7-PAL2+ pYD1-CipA3-EGII+ pRS425-CBHII-BGL1 (YPHCON from here on) for styrene detection using HPLC </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#LB-plates" target="_blank">Protocol Verduyn medium</A>. 
            - YPHPAL2 is grown in 20 ml 2% glucose with URA, ADE, LYS, TRP, LEU with and without 0,5mg/ml phenylalanine. A colony from a transformation with both putative positive plasmids is precultured. 
            - YPHCON is grown in 20 ml 2% glucose with URA, ADE, LYS and 0,5mg/ml phenylalanine. 
            - YPH499 is grown in 20 ml 2% glucose with URA, ADE, LYS, TRP, LEU, HIS with and without 0,5mg/ml phenylalanine. 
            - Cultures are incubated overnight at 30 degrees C.  </P><P><B>Who: Benno</B></P><P><B>Aim</B> The goal of sieving Recell cellulose was to turn the slightly inconsistent Recell starting material into a standardized starting material for ball milling so the results of multiple ball milling runs can be compared to each other </P><P>Cellulose obtained from KNN cellulose was sieved over a 150 µm sieve until 1,5 g of fine cellulose powder were obtained. The powder was used for growth medium to quantify the beneficial effects of various cellulose pretreatments. </P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday October 9th </DIV><DIV class="collapsible-body"><P><B>Who: Rianne</B></P><P><B>Aim</B> EGII-His purification and PAL2 cell lysate preparation</P><P>Resuspension buffer
            - 50 mM Tris-HCl
            - 100 mM KCl
            - 20% glycerol </P><P>Washing buffer
            - 50 mM Tris-HCl
            - 100 mM KCl
            - 20% glycerol
            - 10 mM imidazole</P><P>Elution buffer
            - 50 mM Tris-HCl
            - 100 mM KCl
            - 20% glycerol
            - 300 mM imidazole</P><P>25 ml overnight cultures of EGII-His and PAL2 from Stanford-Brown and our own PAL2 were transferred into 1 L fresh LB medium with chloramphenicol. The culture flasks were incubated at 200 rpm, 37℃. 
            The cultures were induced at an OD600 of 0.6 for 2.5 hours. Subsequently, the cultures were put on ice and transferred into 1 L centrifuge tubes for centrifugation at 6500 rpm for 10 min (JLA 9.1000 rotor, 4℃).
            The supernatant was discarded and the pellet was resuspended into approximately 6 ml resuspension buffer. An appropriate amount of Dnase was added and the mixture was processed in a cell disrupter (13 kPsi). 
            The cell lysates were centrifuged at 3500 rpm for 10 min (JA 25.50,4℃) and the supernatant was transferred into new centrifuge tubes and centrifuged in the same rotor at 19000 rpm, 15 min. Of the PAL2 cultures, a sample was stored on ice and transferred to the fridge for later use.
            The cell lysate of EGII was mixed with Nickel beads for 1 hour in the cold room. The mixture was loaded into a disposable column. The beads were washed twice with 8 ml of washing buffer and eluted twice with 200 µl of elution buffer. Samples were collected throughout the purification process and stored in the fridge. The elution fractions were snap-frozen in liquid nitrogen and stored in a -80℃ freezer.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Colony PCR on transformation 8-10-18 Zcas into pSB1C3 </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#cellulosephosphorylation" target="_blank">colony PCR protocol</A>. Primers: VF2, VR. Annealing temp: 55 dergees C. 9 colonies from LB+Chlor and 9 colonies from LB+Zeocin checked.  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Dilution of overnight YPH499 cultures for styrene detection using HPLC </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#LB-plates" target="_blank">Verduyn medium protocol</A>. 
            OD600 is measured using blanks corresponding to the culture medium. Cultures are diluted as to reach OD600 0,4 at 9:00 in the morning assuming a 2 hour doubling time. 
            - YPHPAL2 is grown in 20 ml 2% glucose with URA, ADE, LYS, TRP, LEU with and without 0,5mg/ml phenylalanine. A colony from a transformation with both putative positive plasmids is precultured. 
            - YPHCON is grown in 20 ml 2% glucose with URA, ADE, LYS and 0,5mg/ml phenylalanine. - YPH499 is grown in 20 ml 2% glucose with URA, ADE, LYS, TRP, LEU, HIS with and without 0,5mg/ml phenylalanine. 
            - Cultures are incubated overnight at 30 degrees C. </P><P><B>Who: Benno</B></P><P><B>Aim</B> The goal of ball milling cellulose was to evaluate the sample technique ball milling for preparation of Recell cellulose for medium usage. </P><P>10 g of unsieved cellulose was ball milled in a Fritsch Pulverisette 6 machine using a ZrO2 cylinder and ZrO2 balls. The choice for zirconia was made based upon its hardness (&gt;8 on Mohs hardness scale). The ball mill required periodical cool down times of 15-20 minutes per 15 minutes of running time. The Ball Mill was started at 9:55 and stopped at 10:10. A small sample (200 mg) was taken (ReCellBM15). The sample taken after 15 minutes was not significantly visibly different from the starting material. The milling process was continued at 10:30. </P><P>At 11:00 the Ball Mill was still very warm so it was decided to keep it cooling for another 15 minutes and start the next run at 11:30.</P><P>From there on out a 15 minute running and 30 minute cooling pattern was kept until 14:15 which samples taken every time the Ball Mill was stopped (ReCellBM30 - ReCellBM90).</P><FIGURE><FIGCAPTION><I>A visual comparison of cellulose with different degrees of milling. All tubes contain 200 mg of cellulose but finer grinded cellulose collapses into a much smaller volume then the original, more fibery cellulose.</I></FIGCAPTION></FIGURE><P>The samples of different milling times were also compared by making plates of them and comparing colony number and size after 3 days of cultivating.</P><P>After milling and taking samples 8,5 g of very fine, colloid cellulose powder were obtained. Parts of the cellulose were further treated with phosphorylation, hydroxylation and ultra-sonication. All prepared cellulose was ultimately used for growth medium, mostly for liquid cultures with a concentration of 2 g/ 100 ml.</P></DIV></LI><LI><DIV class="collapsible-header"> Wednesday October 10th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Isolating plasmids (Z2, 3, 5 and 8) positive hits colony PCR pSB1C3-Zcas 9-10-18 for biobrick submission </P><P>Isolate plasmids using plasmid miniprep kit. Measure DNA concentration using nanodrop. Pipet 250 ng DNA in the appropriate wells.  </P><P><B>Who: Owen</B></P><P><B>Aim</B> Send plasmids pSB1C3-Zcas (Z2, 3 and 5) for sequencing </P><P>Prepare sequencing samples containing 500 ng pDNA and 5uM final concentration primer. 
            - Z2 with primer VF2.
            - Z3 with primer VF2.
            - Z5 with primer VF2.
            - Z2 with primer VR.
            - Z3 with primer VR.
            - Z5 with primer VR. </P><P><B>Results</B></P><P>All sequencing samples are negative.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate plasmid (Z8) positive hit colony PCR pSB1C3-Zcas 9-10-18 </P><P>Inoculate colony in 4 ml LB+Chlor. Grow overnight at 37 degrees C. </P><P><B>Who: Benno</B></P><P><B>Aim</B> The goal of the GC-MS and GC-FID runs is to have a second qualitative measurement technique for styrene to ensure a reliable proof of concept. Also we wanted to evaluate GC-FID for quantitative information </P><P>Sample preparation for GM-HS and GC-FID:
            800 µl of liquid culture is pipetted into a 2 ml Eppendorf cup. 720 µl benzene and 80 µl 10 mM 2-methylanisole benzene solution (internal standard) are added. The Eppendorf is then vortexed for 20 minutes and centrifuged for 10 minutes at 13.000 rpm. The organic phase is filtered over a 0,2 µM polyethersulfone (PES) disc membrane and brought into cealed GC vial. The GC vial is placed into the GC autosampler and the run is started. </P><P>GC-FID calibration curve:
            For quantification with the Flame Ionization Detector a calibration curve has to be taken. Eight samples were prepared in duplo with known amounts of styrene, ranging from 0,5 mmol to 5,0 mmol. The samples were injected into the GC-FID system. The retention time of styrene was 3,741 min, the rt of the internal standard was 4,381 min. From the AUCs the Response Ratios (RR) were calculated and a calibration curve was fitted using linear regression.</P><P>The result for overnight grown culture that wasn’t fed extra phenylalanine was negative. The result for culture that was fed phenylalanine gave a miniscule peak at the correct retention time (3,7 minutes) but the AUC fell outside of the calibration curve.</P><P>The peak of the phenylalanine fed culture was very small and was therefore not viewed as proof of concept of styrene production. It was decided to grow another culture, also fed with phenylalanine, that would grow for 2 days and would be prepared for GC differently. The used GC protocol was made for e.coli and therefore didn’t account for the existence of the yeast cell wall which could hinder desorption of styrene from the lipid bilayer into the aromatic organic phase. It was therefore altered by adding a mechanical cell lysis step.</P><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Lyse cultures and measure styrene levels </P><P>Owen has inoculated a number of cultures, these need to be lysed and the styrene levels need to be measured. First, the OD’s are measured. </P></DIV></LI><LI>Pal2 3 8/10: 4.2</LI><LI>Prs313 Pal2 2 +phen 8/10: 1.53</LI><LI>Pal2 + cellulases + phen 8/10: 0.38</LI><LI>Prs313-Pal2 3 + phen 8/10: 2.41</LI><LI>Pal2 2 8/10: 4.1</LI><LI>Pal2 2 phen: 0.47</LI><LI>Pal2 2 9/10: 0.38</LI><LI>Pal2 3 phen: 0.46</LI><LI>Pal2 cellulases phen 9/10: 0.33</LI><LI>Pal2 3 9/10: 0.35</LI><LI>YPH499 8/10 ctrl: 5.4</LI><LI>YPH499 9/10 ctrl: 0.41</LI><P>800 µl of each culture is pelletet. 200 µl of the supernatant is used as sample for HPLC analysis.</P><P>800 µl of each culture is lysed using glass beads and a beadbeater. 7 cycles of 1 minute vortexing and 1 minute rest.</P><P>After beadbeating, 200 µl of the sample is mixed with 200 µl ethylacetate. Centrifuge 5 minutes at maximum speed. 200 µl of the top layer of each sample is used as sample for HPLC analysis.</P><P>As positive control, 0.5% final concentration is added to the YPH499 8/10 ctrl sample.</P><P><B>Who: Rianne &amp; Jacques</B></P><P><B>Aim</B> Cellulase activity assay on (preprocessed) cellulose </P><P>Cellulose sources used were: Pure ReCell (R), Ball-milled ReCell (BR), pure cellulose (C) and phosphorylated cellulose (PC). Since ReCell consists of 99% cellulose, of R, BR and C 1% solutions were prepared in incubation buffer (https://2018.igem.org/Team:Groningen/Protocols). Because PC was in an aqueous mixture, the exact percentage could not be estimated precisely. We therefore dissolved 1,23 grams into 5 ml of incubation buffer. </P><FIGURE><FIGCAPTION><I>From left to right: R, BR, C, PC. Left before vortexing: only PC stays more in solution. Right: by vortexing, most solutions become evenly distributed except for R.</I></FIGCAPTION></FIGURE><P>We purchased enzymes representing our designed CBHI, EGII and BGLI from Megazyme. The purchased enzymes are: cellobiohydrolase I (CBHI) from Trichoderma sp., cellulase (endo-I,4-ꞵ-D-glucanase) from A. niger, and ꞵ-glucosidase from Aspergillus niger. We will refer to them as CBH, EG and BGL, respectively. We furthermore purchased a cellulase enzyme blend as a positive control from Sigma-Aldrich (a product of Novozyme).</P><P>CBH: 0.05 U/mg EG: 70 U/mg, 1200 U/mL BGL: 90 U/mg, 40 U/mL Cellulase blend: ~1.15 g/mL</P><P>Since the assay is suitable for cellulase activity between 6-375 mU, but CBH came in a low concentration, we decided to use a final enzyme concentration of 10 mU for all enzymes to be able to compare their activities.</P><P>For these purposes, we diluted all enzymes to 0.05 U/mL concentration, of which we took 20 µl into a final assay volume of 100 µl. For the negative control, 20 µl of buffer was added. The other 80 µl consisted of the cellulose sources suspended in incubation buffer. The assay was further performed according to the <A href="https://2018.igem.org/Team:Groningen/Protocols#GC-extraction" target="_blank">protocol</A>. For the image below, the enzymes were incubated with the substrates overnight. </P><P>This resulted in the chromogenic pattern as can be seen below. The positive control contains such large amounts of cellulases that it becomes pink in all cases (it is brownish because the blend has a brown color). It appears that the phosphorylated cellulose is best degraded, especially by CBH. The cellulose and ball-milled ReCell can be degraded as well, but need more time.</P><FIGURE><FIGCAPTION><I>Row A to D contain R, BR, C and PC, respectively. Lanes 1 to 6 contain BGL, EG, CBH, a mix of 1-3, cellulase blend, negative control, respectively. </I></FIGCAPTION></FIGURE><LI><DIV class="collapsible-header"> Thursday October 11th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Send plasmid pSB1C3-Zcas (Z8) for sequencing </P><P>Isolate plasmids using plasmid miniprep kit. Measure DNA concentration using nanodrop. Prepare sequencing samples containing 500 ng pDNA and 5uM final concentration primer. 
            - Z8 with primer VF2.
            - Z8 with primer VR. </P></DIV></LI><LI><DIV class="collapsible-header"> Friday October 12th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Inoculate overnight cultures YPHPAL2, YPHCON and YPH499 for growth on cellobiose &amp; cellulose for styrene detection using HPLC </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#LB-plates" target="_blank">Verduyn protocol</A>. 
            - YPHPAL2 is grown in 10 ml 1% fructose with URA, ADE, LYS, TRP, LEU.
            - YPHCON is grown in 10 ml 1% fructose with URA, ADE, LYS.
            - YPH499 is grown in 20 ml 1% fructose with URA, ADE, LYS, TRP, LEU, HIS. 
            - Cultures are incubated overnight at 30 degrees C. </P><P><B>Who: Rianne</B></P><P><B>Aim</B> SDS-PAGE and Western blot analysis of PAL2 cell lysate and EGII purification </P><P>An SDS-PAGE gel and sample buffer was kindly provided by Marten Exterkate. 15 µl of each sample was mixed with 5 µl of sample buffer and 15 µl was loaded onto the gel for analysis. 
            PAL2(US) is the soluble fraction of the cell lysate from our culture, whereas PAL2(SB) refers to that of Stanford-Brown and EGII to that of our EGII expressing cell culture. The samples afterwards are fractions during the purification process of EGII. SF is the soluble lysate fraction after incubation with the Nickel beads, FT the flow though after passage through the disposable column. W1 and W2 fractions after the two wash steps, and EF1 and EF2 the two elution fractions. </P><FIGURE><FIGCAPTION><I>Coomassie stained SDS-PAGE of cell lysates and purification fractions. Order of the samples: Page Ruler protein ladder, PAL2(US), PAL2(SB), EGII, SF, FT, W1, W2, EF1, EF2. Page Ruler ladder: lowest band upwards: 15 (flat line below), 25, 35, 40, 55, 70, 100, 130, 180 kDa.</I></FIGCAPTION></FIGURE><P>We observe a large band in PAL2(US) around the 70 kDa band, which corresponds to the 78 kDa expected height for PAL2. This band is not visible in PAL2(SB), suggesting the protein did not come to (as high) expression in the Stanford Brown strain. For EGII, the second wash is clean, suggesting most non-specific binding proteins have been washed from the beads. The elution fractions however contain multiple bands. EGII has an expected molecular weight of 44 kDa, but the bands on the gel show a larger sized protein. Hence we decided to perform an anti-His Western blot to detect the presence of a His-tagged protein.</P><P>The same protein samples were used for another SDS-PAGE gel (15%) and followed by a Western blot which was kindly performed by Dr. Aditya Iyer according to <A href="https://docs.google.com/document/d/1UMabJpKOg9ukW-qTmK0ZssQxaB19ZL_LgqrKfKlBn-U/edit" target="_blank">https://docs.google.com/document/d/1UMabJpKOg9ukW-qTmK0ZssQxaB19ZL_LgqrKfKlBn-U/edit</A>. In this case, also a sample of the insoluble pellet fraction was loaded to see if there was any EGII presence (EGII(P)). The other samples are the same as described above.</P><FIGURE><FIGCAPTION><I>Western blot EGII. Order of the samples: ladder, EGII(P), EGII, SF, FT, W1, (empty lane), W2, EF1, EF2. The ladder on this blot is Page Ruler Plus, so lowest band of the ladder upwards: 10, 15, 25, 35, 55, 70, 100  kDa.</I></FIGCAPTION></FIGURE><P>This shows that the detected His-tagged protein bands are between 35 and 55 kDa, which corresponds with the expected 44 kDa for EGII. The EGII elution fractions and the PAL2 cell lysates were further used in enzyme activity assays.</P><P><B>Who: Rianne &amp; Jacques</B></P><P><B>Aim</B> EGII activity assay on (preprocessed) cellulose </P><P>The cellulase test of 10-10-18 was repeated, but in this case with different enzymes on the same substrates. The total incubation volume was 200 µl instead of 100 µl. </P><P>We measured the concentration of the purchased EG and our purified EGII elution fraction on nanodrop (blank MQ): EG = 9 mg/mL His-EGII = 0.66 mg/mL</P><P>We decided to assume the amount of units in our purified elution fraction will be comparable to the purchased enzyme, and therefore diluted the purchased enzyme tenfold to approximate the His-EGII concentration. We tested undiluted His-EGII, 1:1000 and 1:10.000 dilutions. The samples were measured after 1 hour and overnight incubation, and both gave the following result:</P><FIGURE><FIGCAPTION><I>Row A to D contain R, BR, C and PC, respectively. Lanes 1 to 6 contain EG 1:10, His-EGII, His-EGII 1:1000, His-EGII 1:10.000, cellulase blend, negative control, respectively. </I></FIGCAPTION></FIGURE><P>We see that the positive control worked on all substrates. The purchased EG enzyme however did not show much activity. Our undiluted EGII showed activity on all substrates except for unprocessed ReCell</P><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Inoculate cultures for a new cellulose degradation assay </P><P>Aim: Inoculate cultures for a new cellulose degradation assay. The following strains are inoculated into 20 ml Verduyn medium with appropriate amino acids, and fructose as carbon source.  CB + 1883 colony 1  CB + 1883 colony 1 CB + 1883 + Pal2 colony 1 CB + 1883 + Pal2 colony 2 Negative control (strain not containing any plasmids) Incubate at 30 ℃ with agitation. </P></DIV></LI><LI><DIV class="collapsible-header"> Sunday October 14th </DIV><DIV class="collapsible-body"><P><B>Who: Jan Marten</B></P><P><B>Aim</B> Galactose induce the cultures from 12-10-’18, and prepare the samples for the plate reader to make a growth curve </P><P>The cultures have grown, and are in the exponential phase. At 11.00, the cultures are induced with 2% final concentration galactose. At 17.00 hrs, the cultures are washed and samples are prepared according to the <A href="https://2018.igem.org/Team:Groningen/Protocols#galactose-induction">galactose induction and sample preparation protocol</A>. </P><P>96-well plate layout: All wells in the same row (A, B, C, etc.) contain the same cell culture. All wells in the same column (1, 2, 3, etc.) contain the same carbon source. A: blank B: BJ1991 with plasmids CB + 1883 colony 1 C: BJ1991 with plasmids CB + 1883 colony 2 D: YPH499 with plasmids CB + 1883 + pRS313-Hxt7-PAL2 colony 1 E: YPH499 with plasmids CB + 1883 + pRS313-Hxt7-PAL2 colony 2 F: negative control (YPH499 without plasmids)</P><P>1: No carbon source 2: No carbon source 3: 0.5% glucose + 1% phosphorylated cellulose 4: 0.5% glucose + 1% phosphorylated cellulose 5: 1% phosphorylated cellulose 6: 1% phosphorylated cellulose 7: 1% 90 minutes ball-milled ReCell 8: 1% 90 minutes ball-milled ReCell 9: 1% 45 minutes ball-milled ReCell 10: 1% 45 minutes ball-milled ReCell 11: 1% cellobiose 12: 1% cellobiose</P><P>The plate is inserted into a Thermo Scientific Varioskan Lux platereader. Settings: Kinetic loop. Duration: 23:59:59, measure every 01:00:00. Agitation: continuous, low, 180 RPM. Incubator set at 30℃.</P></DIV></LI><LI><DIV class="collapsible-header">Week 42</DIV></LI><LI><DIV class="collapsible-header"> Monday Oktober 15th </DIV><DIV class="collapsible-body"><P><B>Who: Benno</B></P><P><B>Aim</B> The goal of repeating the GC-FID runs with a different sample preparation was to investigate the negative results of the previous run and to obtain reliable styrene quantification</P><P>The GC runs were repeated with an improved sample preparation involving mechanical cell lysis through glass beads. The results were analysed on the HPLC by Owen and Jeroen Nijland and by Benno on the GC-FID. </P><P>For cell lysis 200 µl worth of glass beads were added to 900 µl worth of liquid culture in a 1,5 ml Eppendorf cup. The Eppendorf was vortexed for 10 minutes and subsequently spun down at 13.000 rpm for 1 minute. 800 µl of the supernatant were used for the described GC sample preparation.</P><P>Liquid cultures of two promising PAL2 positive colonies were grown to OD , a negative control sample was also included.</P><P>Due to a miscommunication these supposedly superior samples originated from cells that were not supported with Phenylalanine in their grow medium.</P><P>The GC-FID results of all three measurements were negative. One potential explanation of this result is, that glass bead cell lysis is usually performed in 1 minute steps with 1 min ice cooling steps in between to prevent enzyme degradation. As enzymes are of no interest to our measurement this was skipped. During the 10 minutes of vortexing the movement of the glass beads could have risen the temperature of the contained liquid so much that styrene evaporated and left the Eppendorf immediately after opening.</P><P>The GC-FID runs therefore only give qualitative and not quantitative styrene measurement. The absence of styrene in the GC runs from 15 October cannot be interpreted as the yeast strain not working as the sample preparation was suboptimal and not validated with blanco cell material with exogenous styrene.</P><P><B>Who: Owen</B></P><P><B>Aim</B> Calculation of styrene production per amount of dry weight (DW) S. cerevisiae </P><P>Using peak area of the peak at the retention time of styrene (17,5min) and OD600 of cell cultures an estimate was made of the amount of styrene produced per DW.</P><P>Values: - Kdw: 300mgDW/L/UnitOD600 is assumed as value obtained from the Molmic group. - D: density of styrene: 0,909 - P: is peak area- peak area negative control - Pref: is the peak area of 0,5% styrene</P><P>Formula: styrene/DW(ug/mg)=((P/Pref)*0,5%*D*1.000.000)/(Kdw*OD600)</P><TABLE><TBODY><TR><TH>OD600</TH><TH>Peak area (arbitrary units)</TH><TH>styrene/Dryweight cells (ug/mg)</TH></TR><TR><TD>negative control</TD><TD>-</TD><TD>2699</TD><TD>-</TD></TR><TR><TD>PAL2</TD><TD>4.2</TD><TD>20586</TD><TD>3.773894298</TD></TR><TR><TD>PAL2+</TD><TD>2.41</TD><TD>38628</TD><TD>13.21081475</TD></TR><TR><TD>PAL2++</TD><TD>0.38</TD><TD>26467</TD><TD>55.4256197</TD></TR><TR><TD>control + 0.5% styrene</TD><TD>-</TD><TD>1692738</TD><TD>-</TD></TR></TBODY></TABLE><P><B>Who: Owen</B></P><P><B>Aim</B> Styrene detection in YPH499 with pRS313-Hxt7-PAL2+pYD1-CipA3-EGII+pRS425-CBHII-BGL1 and growth on cellobiose and phosphorylated cellulose </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#HPLC-validation" target="_blank">HPLC styrene detection protocol</A>. Two biological duplicates were tested grown on cellobiose, together with controls. 1 culture was tested grown on phosphorylated cellulose. OD600 cultures: - 1 ctrl: 0,06 - 2 ctrl: 0,06 - 1 cellobiose: 0,15 - 2 cellobiose: 0,15 - phosphorylated cellulose: 0,55 - phosphorylated cellulose control: 0,51</P><P><B>Results</B></P><P>No cinnamate or styrene peaks can be seen.</P></DIV></LI><LI><DIV class="collapsible-header"> Tuesday Oktober 16th </DIV><DIV class="collapsible-body"><P><B>Who: Owen</B></P><P><B>Aim</B> Styrene detection in YPH499 with pRS313-Hxt7-PAL2+pYD1-CipA3-EGII+pRS425-CBHII-BGL1 grown on cellobiose and phosphorylated cellulose </P><P>See <A href="https://2018.igem.org/Team:Groningen/Protocols#HPLC-validation" target="_blank">HPLC styrene detection protocol</A>. Two biological duplicates were tested grown on cellobiose, together with controls. 1 culture was tested grown on phosphorylated cellulose. Ctrl 1 with 0.01% styrene was also loaden on the HPLC. OD600 cultures: - 1 ctrl: 0,1 - 2 ctrl: 0,088 - 1 cellobiose: 2,2 - 2 cellobiose: 1,5 - phosphorylated cellulose: 1,36 </P><P><B>Results</B></P><P>See our <A href="https://2018.igem.org/Team:Groningen/Results" target="_blank">results page</A></P></DIV></LI><FOOTER><H4 class="center">We thank our sponsors:</H4></FOOTER></BODY></HTML>