<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGem Groningen 2018</TITLE></P><DIV class="navbar-fixed"><NAV class="row green"><UL id="dropdown-project" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Description" class="green-text">Description</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Design" class="green-text">Design</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Results" class="green-text">Results</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Demonstrate" class="green-text">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Entrepreneurship" class="green-text">Entrepreneurship</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Awards" class="green-text">Awards</A></LI></UL><UL id="dropdown-notebook" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Notebook" class="green-text">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Protocols" class="green-text">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Safety" class="green-text">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Measurement" class="green-text">Measurement</A></LI></UL><UL id="dropdown-parts" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Parts" class="green-text">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Primers" class="green-text">Primers</A></LI></UL><UL id="dropdown-HP" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Human_Practices" class="green-text">Integrated Human Practices (Gold)</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Public_Engagement" class="green-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Applied_Design" class="green-text">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Ethics" class="green-text">Ethics</A></LI></UL><UL id="dropdown-attribution" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Attributions" class="green-text">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Sponsors" class="green-text">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Crowdfunding" class="green-text">Crowdfunding</A></LI></UL><UL id="dropdown-help" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/InterLab" class="green-text">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Software" class="green-text">Software</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Collaborations" class="green-text">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Embassy" class="green-text">Consulate Meetup</A></LI></UL><UL id="dropdown-team" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Team" class="green-text">Team</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Supervisors" class="green-text">Supervisors</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Team_Building" class="green-text">Team Building</A></LI></UL><UL id="dropdown-model" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Model" class="green-text">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Molecular_Dynamics" class="green-text">Proximity by affinity</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Mathematical_Modeling" class="green-text">Cellulose Degradation</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Flux_Based_Analysis" class="green-text">Optimizing StyGreen Production</A></LI></UL><DIV class="nav-wrapper" id="nav"><UL class="left" id="nav-buttons"><LI><A href="https://2018.igem.org/Team:Groningen" class="waves-effect waves-light block"><I class="material-icons">home</I></A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-project">Project</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-notebook">Lab</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-model">Model</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-parts">Sequences/Parts</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-HP">Human Practices</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-help">Contributions</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-team">Team</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-attribution">Acknowledgements</A></LI></UL><UL class="right" id="contact"><LI title="Click to visit our Facebook page"><A target="_blank " href="https://www.facebook.com/igemgroningen2018/" class="menu-hover waves-effect waves-light block"><SPAN>Groningen.iGEM2018</SPAN></A></LI><LI title="Click to e-mail us"><A target="_blank" href="mailto:igem2018rug@gmail.com" class="menu-hover waves-effect waves-light block"><SPAN>igem2018rug@gmail.com</SPAN></A></LI><LI><A class="menu-hover" id="cross"><I class="material-icons">close</I></A></LI></UL></DIV></NAV></DIV><A href="#!" class="btn-floating btn-large waves-effect waves-light green" id="menu-btn"><I class="material-icons">menu</I></A><MAIN class="row HP"><DIV class="igem_2018_team_content" style="background-color: white"><DIV class="igem_2018_team_column_wrapper"><DIV class="col s12 m10 l10"><DIV class="column"><P>
    In the process of designing the route from cellulose to StyGreen, there are many factors to consider. There are several approaches that can be taken to degrade cellulose, metabolic pathways to styrene as well as different interesting sources of cellulose. Discussions with many <A href="https://2018.igem.org/Team:Groningen/Human_Practices#porter" target="_blank">experts</A> have aided us in making the right choices or adjustments during the design process. Furthermore, we connected with the upstream and downstream processes of our design and approached different stakeholders, connecting our project to <A href="https://2018.igem.org/Team:Groningen/Human_Practices" target="_blank">cellulose suppliers and styrene buyers</A>. 
</P><P>
    The production of StyGreen from cellulose is a multi-step pathway. As such, we have the choice of using either one or multiple organisms to perform the different reactions. For example, we could engineer a cellulose degrading organism and a styrene producing organism; these organisms can be co-cultured to produce styrene from cellulose. However, after discussing this idea with <A href="https://2018.igem.org/Team:Groningen/Human_Practices#porter" target="_blank">experts</A>, we concluded that creation of a single organism performing both steps would be the most efficient and industrially desirable choice. In this manner, we produce a consolidated bioprocessing system which is not currently available. The combined process saves substrate, raw materials and utilities such as separate cellulase enzyme production. Moreover, the complete process could take place in one reactor and, in case of higher overall efficiency, this results in a reduced reactor volume and thus lower capital investment [A]. Hence, our ultimate goal is the production of a single robust cell factory that can grow on cellulose and produce styrene effectively in an industrial bioreactor setting.
</P><P>
    Besides ‘wet lab’ experiments, we also <A href="https://2018.igem.org/Team:Groningen/Model" target="_blank">modelled</A> parts of our system  in the 'dry lab' with help from experts to substantiate or improve our design. Want to know more details about our design? Click on the questions below to find out!
</P><UL class="collapsible popout" data-collapsible="expandable"><LI id="timeline-scroll" class="scrollspy"><DIV class="collapsible-header active">
                        Design
                    </DIV><DIV class="collapsible-body"><DIV class="row"><DIV class="col s12"><P><B> Click on the icons in the timeline</B>, and find out about all the insights we gained from our stakeholders and how the dialogues shaped our project. </P><DIV class="col m10 s12"><DIV id="cellulose_input" class="modal" style="z-index: 1003;"><DIV class="modal-content"><H4>Cellulose input</H4></DIV></DIV></DIV></DIV></DIV></DIV></LI><LI><DIV class="collapsible-header">1.1 What is a good cellulose source for our product?</DIV><DIV class="collapsible-body"><P>Since cellulose is a common material in nature, many organic materials contain significant amounts of cellulose. We investigated several options of cellulose materials and compared the advantages and disadvantages. In our quest for the ideal cellulose source, wide availability, suitability for our system, financial feasibility and non-competition with food sources were important factors (also read more <A href="https://2018.igem.org/Team:Groningen/Applied_Design" target="_blank">here</A>.</P><P>An important widely available source of cellulose is wood, which is also relatively inexpensive. The infrastructure for the wood market is already in place. During our company visit at <A href="https://2018.igem.org/Team:Groningen/Human_Practices#avantiuminterview" target="_blank">Avantium</A>, we learned about the degradation process of wood into separate components. Wood does not solely contain cellulose, but also compounds as hemicellulose and lignin, which would interfere with our process. In our current design in particular lignin would be a tough material to process, leading to unused waste material. </P><P>The second potential source of cellulose we considered are algae [2]. The growth of algae is fast and they can grow under various conditions [3]. At the moment, the market for algae cultivation is still relatively small [4]. Therefore, the price of algae biomass is high. As we aim to develop a financially feasible product, algae biomass is currently not interesting for us.  In case the market of algae cultivation increases in the future, decreasing the price of algae biomass, it will be an interesting feedstock to consider. </P><P>Another source of cellulose that  caught our attention is used toilet paper. During a meeting with a representative of the company KNN we learned that cellulose-rich biomass of high quality such as wood can still be valorized in quality products such as a table, whereas other cellulose sources such as used toilet paper are not of sufficient quality. The  quality of the cellulose strands is too low making them unsuitable for many conventional recycling processes. Therefore, this cellulose waste material is more difficult to valorize, making it an interesting feedstock for our process from an economic and ecological perspective. Up to now, there are very few applications for used toilet paper from sewage. However, as it still consists largely of cellulose,  we can use it as input for our system. The company <A href="https://2018.igem.org/Team:Groningen/Human_Practices#knninterview" target="_blank">KNN</A> produces a product called Recell®, which is cleaned and processed toilet paper filtered out of the sewage. Our process could greatly increase the value of this newly developed material making it an interesting cellulose source! Therefore, we decided to focus on the use of cellulose waste in the form of used toilet paper as input for our system.</P></DIV></LI><LI><DIV class="collapsible-header">1.2 How do we preprocess the cellulose-containing biomass for optimal use?</DIV><DIV class="collapsible-body"><P>Some materials contain very dense cellulose, which hinder the cellulases to work on the cellulose strands. In our project, we decided to focus on one of our most important partners, KNN Cellulose, and their product ReCell. ReCell consists of 99% cellulose. Like most materials made from cellulose, ReCell is difficult to dissolve in an aqueous solution, consequently diminishing access for cellulases. This makes it difficult for cells expressing a cellulosome to sustain growth on raw cellulosic materials, which we also observed in our own lab. To optimize the capability of cellulases and our cells to utilize cellulose, we investigated phosphorylation and ball milling as pretreatments [5, 6, 7, 8]. The results of the improved enzymatic degradation of these pretreated types of cellulose can be found <A href="https://2018.igem.org/Team:Groningen/Results#cellulose_assay" target="_blank">here</A>. In this manner, we make the raw cellulose material more suitable for use in growth medium and eventually in bioreactors.</P></DIV></LI></UL></DIV></DIV><DIV id="towards_glucose" class="modal" style="z-index: 1003;"><DIV class="modal-content"><H4>Cellulose towards glucose</H4><UL class="collapsible" data-collapsible="expandable"><LI><DIV class="collapsible-header">2.1 Which enzymes do we need to degrade cellulose effectively?</DIV><DIV class="collapsible-body"><P><I>Saccharomyces cerevisiae</I> is not able to degrade cellulose into glucose by itself. Therefore, we enable expression of a set of enzymes called cellulases, that can break the β-(1,4)-glycosidic bonds between the glucose molecules in the cellulose strands. These cellulases exist in different types, and a combination of three of them can effectively degrade cellulose: an endoglucanase, a cellobiohydrolase and a β-glucosidase. In previous studies, EGII, CBHI and BGL1, respectively, have proven to form a good cellulose degrading team, hence we decided to use these enzymes in our project [9].</P></DIV></LI><LI><DIV class="collapsible-header">2.2 What is a mini-cellulosome?</DIV><DIV class="collapsible-body"><P>We could make our cells secrete the cellulases and have them float around in the medium, but the closer the enzymes are in vicinity of each other, the more efficiently they cooperate to degrade cellulose. We also <A href="https://2018.igem.org/Team:Groningen/Model/Stochastic_Modeling" target="_blank">modeled</A> these interactions. Additionally, our cells need to use the freed glucose, thus the closer the enzymes are to the cell, the faster the glucose is taken up and the subsequent styrene synthesis takes place. To achieve this we make use of a mini-cellulosome complex. A cellulosome is a high molecular structure used by cellulose degrading microorganisms in nature to make the hydrolysis of cellulose efficient enough for them to survive on it. As heterologously expressing the entire complex is a daunting task that has never been accomplished before, we use only a subset of the enzymes contained in the natural cellulosome. In our mini-cellulosome three enzymes are bound next to each other on a scaffold that is in turn attached to the cell wall of our yeast. The enzymes are fused to a dockerin by which they are connected to the scaffold via cohesin domains. The scaffold is fused to the mating receptor AGA2, which binds to AGA1 on the cell wall. In this manner, we create both enzyme-enzyme and cellulosome-yeast proximity. The cellulosome-yeast proximity is so effective no reducing sugars can be detected when it degrades cellulose (C). As such there are almost no free sugars for competing organisms, potentially removing the need to sterilize the feedstock.</P><P>The design of the scaffold has another important feature to improve degradation, because it contains a cellulose binding domain. With the use of molecular dynamics we have <A href="https://2018.igem.org/Team:Groningen/Model/Molecular_Dynamics" target="'_blank">modelled</A> and characterized the affinity of the domain for the cellulose fibers. The cellulose binding domain does not only enhance enzyme proximity to the substrate, but also has benefits for the final bioreactor design. Cellulose-adherence makes it more likely that the genetically modified organism can compete for cellulose with non-adhered microbes. This together with the cellulosome-yeast proximity could potentially remove the need for sterilizing the feedstock in a bioreactor, greatly reducing CO2 emission and costs [1]. </P></DIV></LI></UL></DIV></DIV><DIV id="towards_stygreen" class="modal" style="z-index: 1003;"><DIV class="modal-content"><H4>Glucose towards StyGreen</H4><UL class="collapsible" data-collapsible="expandable"><LI><DIV class="collapsible-header">3.1 How to convert glucose into styrene?</DIV><DIV class="collapsible-body"><P>The second part of our process is production of styrene. There are several steps which the cell can perform endogenously, but some need exogenous aid. In order for glucose to be converted to styrene, it first has to enter the glycolysis where it is converted into phosphoenolpyruvate (PEP). Next, PEP is converted into phenylalanine via the shikimate pathway. For the conversion of phenylalanine to trans-cinnamate, which is the next step in our process, we need to add a catalyst in the form of phenylalanine ammonia lyase 2 (PAL2). Subsequently, the trans-cinnamate is converted into styrene by the endogenous enzyme ferulic acid decarboxylase 1 (FDC1) [10].  </P><FIGURE><FIGCAPTION><I>Figure 2. The phenylalanine to styrene pathway. Phenylalanine is de-aminated by Pal2 to form trans-cinnamate, which is converted to styrene by a Fdc1-catalyzed carboxylation</I></FIGCAPTION></FIGURE><P>To increase styrene production, one of the most common strategies is to look for the enzyme with the highest activity. In order to do so one can screen isoenzymes from different organisms and compared their activity. Since PAL2 from <I>Arabidopsis thaliana</I> had the highest enzyme activity in <I>E. coli</I> compared to several other phenylalanine ammonia lyases, we decided to use this enzyme for our project [11]. Earlier work also shows successful styrene production in yeast using this PAL2 enzyme [10].</P></DIV></LI></UL></DIV></DIV><DIV id="host_strain" class="modal" style="z-index: 1003;"><DIV class="modal-content"><H4>Host strain</H4><UL class="collapsible" data-collapsible="expandable"><LI><DIV class="collapsible-header">4.1 Why Saccharomyces cerevisiae ?</DIV><DIV class="collapsible-body"><P>In synthetic biology, one can either choose a cellulolytic organism and improve the production-related properties, or choose an organism with high product yield and enable it to express a cellulase system [1]. For our project, we chose <I>Saccharomyces cerevisiae</I> as a host organism and aim to enable it with a set of heterologous cellulases. There are four reasons for this decision: Firstly, the cellulose degrading system of our choice has already been expressed successfully in <I>S. cerevisiae</I> [9]. Secondly, the cellulose degrading system contains an anchor specific to <I>S. cerevisiae</I>, removing the need to find a different anchoring mechanism [9]. Thirdly, <I>S. cerevisiae</I> has an improved industrial phenotype compared to <I>E. coli</I> [10]. The final added benefit of using <I>S. cerevisiae</I> is that it already contains an endogenous enzyme needed for our production process, namely Ferulic acid decarboxylase 1 (FDC1) [10]. </P></DIV></LI><LI><DIV class="collapsible-header">4.2 How do we genetically modify S. cerevisiae ?</DIV><DIV class="collapsible-body"><P>Instead of providing the cell with multiple plasmids containing the cellulases, scaffold and PAL2, <A href="https://2018.igem.org/Team:Groningen/Human_Practices#driesseninterview">Prof. Driessen</A> suggested to use CRISPR-Cas9 combined with homologous recombination to introduce our system into the genome. This would make the cellulosome more genetically stable, according to Driessen. Furthermore, this would eliminate the complicated cloning steps needed to create large plasmids for our system. Saccharomyces cerevisiae is excellent at homologous recombination of multiple fragments simultaneously, a property we can take advantage of in our project [12]. By use of this strategy, multiple genes (three cellulases, a scaffold and PAL2) with 60 base pairs overhangs can be transformed simultaneously and subsequently, homologous recombination will take place at the site of the CRISPR-Cas9 induced double strand break. The use of CRISPR-Cas9 has the added benefit of eliminating the need for addition of antibiotics or amino acids to the growth medium. They are no longer necessary for plasmid maintenance or screening, as cells with unrepaired double stranded breaks should not grow and the double stranded break can only be repaired with all fragments combined [13]. This is especially beneficial when our strain will be upscaled towards bioreactor volumes.</P><FIGURE><FIGCAPTION><I>Source: <A href="https://sites.tufts.edu/crispr/genome-editing/homology-directed-repair" target="_blank">https://sites.tufts.edu/crispr/genome-editing/homology-directed-repair/</A></I></FIGCAPTION></FIGURE></DIV></LI></UL></DIV></DIV><DIV id="improving_our_yeast" class="modal" style="z-index: 1003;"><DIV class="modal-content"><H4>Improving our yeast strain</H4><UL class="collapsible" data-collapsible="expandable"><LI><DIV class="collapsible-header">5.1 How to adapt the expression system for sustained growth on cellulose?</DIV><DIV class="collapsible-body"><P>Eventual implementation of our system ideally involves sustained growth on cellulose of our microbe. However, as pointed out by Prof. Driessen during an <A href="https://2018.igem.org/Team:Groningen/Human_Practices#driesseninterview" target="_blank">expert meeting</A>, our initial mini-cellulosome expression system was under control of galactose promoters [9]. This prevents efficient sustained growth on cellulosic biowaste twofold. Firstly, it requires the addition of galactose to to express the system. However, as we do not want to compete with the food industry we would like to prevent addition of galactose to our process. Secondly, galactose promoters are regulated by negative feedback from glucose [14]. Consequently, the cellulosome downregulates its own expression as it produces glucose. To remedy this problem we chose to express the components of our system under the TEF1 promoter. This is a well characterized promoter giving high levels of expression [15]. </P></DIV></LI><LI><DIV class="collapsible-header">5.2 How to improve the flux to styrene?</DIV><DIV class="collapsible-body"><P>Metabolic engineering is a powerful way to improve microbial production processes. In published studies in PAL2- and FDC1-expressing <I>E. coli</I>, the limiting condition for higher styrene production was the activity of PAL2. Although addition of exogenous phenylalanine significantly improves the net production of styrene, this approach is not economically feasible. However, this indicates that enhancing the endogenous phenylalanine production in the host organism could increase product formation [11, 16].</P><P>Therefore we investigated options to metabolically engineer our strain to optimize the process from glucose to phenylalanine. Since the CRISPR-Cas9 technique enables us to choose a site of integration, it simultaneously enables us to make a knock-out in an endogenous gene. In previous studies, deletion of ARO10 eliminated a competing pathway (Ehrlich pathway) and promoted the availability of phenylalanine. This leads to an enhanced flux towards styrene [10]. Interestingly, we also found this in our <A href="https://2018.igem.org/Team:Groningen/Model/Flux_Based_Analysis" target="_blank">flux balance analysis</A>. For these reasons, we choose to partly integrate our cellulosome into the locus of ARO10. Finally, part of our team works on <A href="https://2018.igem.org/Team:Groningen/Model" target="_blank">models</A> to find additional ways of metabolic engineering that would make our strain more robust and that would increase the flux towards styrene.</P></DIV></LI><LI><DIV class="collapsible-header">5.3 How to further improve our StyGreen production?</DIV><DIV class="collapsible-body"><P>With a potential mechanism for sustainable growth on cellulose in hand, we looked into enhancing the cellulose utilization by our microbe. An interesting way to achieve a more efficient microbe is conducting an adaptive laboratory experiment (ALE), as suggested by prof. Driessen. A strong selective pressure is needed for ALEs, which we possess in the form of growth on cellulose. There are two approaches to conducting ALEs; the first involves use of a chemostat and the second serial dilution [17]. Due to the low solubility of cellulose the option of serial dilution was chosen as ALE.</P><P>An evolution experiment also fits well with our decision to use CRISPR-Cas9 to introduce our system genomically. By use of genomic integration there is no need for addition of antibiotics or essential amino acids for selection or maintainance of the introduced genes, so these factors will not influence the evolution experiment. Based on his experience with earlier ALEs Prof. Driessen hypothesized that the ALE would yield a strain showing increased growth on cellulose after 2 months.</P><P>Inherent to the use of the CRISPR-Cas9 technique is the creation of double strand breaks. By tactically choosing the sites of these double strands breaks and designing our repair fragments accordingly, we can simultaneously insert our system and knock out unfavorable genes. We chose <I>AGA2</I> and ARO10 as knockout sites, while we selected the scaffold-binding AGA1 for overexpression by using CRISPR-Cas9 to insert a TEF1 promoter. Our cellulosome binds through AGA1-AGA2 interaction, thus this overexpression and knockout would increase the amount of binding places (AGA1) while simultaneously removing competitors for binding (AGA2) [9]. </P></DIV></LI><LI><DIV class="collapsible-header">5.4 How do we combat styrene toxicity?</DIV><DIV class="collapsible-body"><P>A limitation in published studies of styrene production in <I>E. coli</I> is the styrene toxicity. This is an important consideration in the perspective of bio-styrene as a reputable alternative to traditional chemical processes [11, 16]. To overcome styrene toxicity, we investigated ways to extract styrene during the process. Another solution was was suggested by <A href="https://2018.igem.org/Team:Groningen/Human_Practices#poolmaninterview" target="_blank">Prof. dr Poolman</A> in the form of an adaptive evolution experiment. He proposed growing our strain on increasing concentrations of styrene to increase resistance to styrene toxicity. This would enable our strain to still grow effectively on cellulose while producing high concentrations of styrene.</P></DIV></LI></UL></DIV></DIV><DIV id="experimental_plan" class="modal" style="z-index: 1003;"><DIV class="modal-content"><H4>Experimental plan to test designs</H4><UL class="collapsible" data-collapsible="expandable"><LI><DIV class="collapsible-header">6.1 How to improve the styrene yield?</DIV><DIV class="collapsible-body"><P>At the <A href="https://2018.igem.org/Team:Groningen/Public_Engagement" target="_blank">Dutch Biotechnology Conference</A> we met Hemant Kumar who presented a poster about a cellulase activity assay. Coincidentally, he works in the laboratory of <A href="https://2018.igem.org/Team:Groningen/Human_Practices#fraaijeinterview" target="_blank">Prof. Dr. Marco Fraaije</A> in Groningen, where we had a fruitful meeting about the use of this assay in our experiments. They provided us with a sensitive colorimetric assay to detect cellulase activity. This assay works as follows. First, degradation of cellulose by cellulases produces hydrolytic products. During the assay, the mutant oxidase ChitO-Q268R releases an equimolar amount of hydrogen peroxide upon the oxidation of the hydrolytic product. The hydrogen peroxide can subsequently be monitored by the horseradish peroxidase (HRP) enzyme and a chromogenic peroxidase substrate, which results in a pink color. The intensity of the pink color represents the degree of cellulose degradation [18]. We used this assay to show activity of our own cellulases, click <A href="https://2018.igem.org/Team:Groningen/Results" target="_blank">here</A> if you want to see some pink!</P></DIV></LI><LI><DIV class="collapsible-header">6.2 How do we detect styrene production?</DIV><DIV class="collapsible-body"><P>To prove our concept of a cellulolytic, styrene producing <I>S. cerevisiae</I> strain the detection of styrene is an essential part. For this purpose we employed multiple state-of-the-art measuring techniques to identify and quantify styrene as specifically and precisely as possible. Chromatographic methods are an apparent option as they allow separation of styrene from the biological matrix of lysed yeast cells based upon characteristics like hydrophobicity (log P styrene = 2,7) and volatility.</P><P>Reverse Phase High Performance Liquid Chromatography (RP-HPLC) was employed to detect the presence of styrene in cell lysate by attaching it to a UltraViolet (UV) Diode Array Detector (DAD) and measuring the UV absorption spectrum of the eluting mobile phase. Both the retention time as well as the recorded UV spectrum at that point in time serve to detect styrene. Gas Chromatography Mass Spectrometry (GC-MS) was used to detect the retention time of styrene on a GC system. By attaching a Flame Ionization Detector (GC-FID) the system is able to give both very specific qualitative as well as very precise quantitative information on our analyte styrene. The GC-FID system utilizes an internal standard approach with 2-methylanisole to correct for variance in the sample preparation steps. Styrene was <A href="https://2018.igem.org/Team:Groningen/Measurement" target="_blank">successfully detected</A> in cell cultures having the PAL2 gene with both methods. </P></DIV></LI><LI><DIV class="collapsible-header">6.3 How do we test our strain?</DIV><DIV class="collapsible-body"><P>An integral part of the design of our strain is that it can survive and grow using cellulose as the sole carbon source. This means that the expression of our artificial cellulosome needs to be high enough and needs to break down cellulose to glucose fast enough. The easiest way to test whether the cellulosome is produced in a sufficient quantity and has a high enough activity, is to test for growth of the engineered <I>S. cerevisiae</I> strain on cellulose. </P><P>First a proof of concept experiment is devised. We culture the cellulolytic <I>S. cerevisiae</I> strain on Verduyn medium and induce the cultures with galactose for several hours to express the cellulosome. Subsequently, we detect growth over two days on the cellulose sources by  measurement of the optical density in 96-well plates in a plate reader. First we will test with cellobiose. Cellobiose is a ꞵ-1,4 glycosidic bond glucose dimer, the shortest existing cellulose polymer. Wild-type <I>S. cerevisiae</I> strains cannot grow on this, but with the presence of the artificial cellulosome it can be hydrolyzed into glucose. This setup will function as a proof of concept before testing other carbon sources.</P><P>After a successful proof of concept experiment with cellobiose, we will repeat the experiment with cellulose.  The tested cellulose sources also include the pretreated phosphorylated cellulose and ball-milled ReCell, as well as cellobiose. From these cultures, we can also measure whether they produce styrene. Interested in the results? Read more about our proof of concept <A href="https://2018.igem.org/Team:Groningen/Results">here</A>!</P></DIV></LI></UL></DIV></DIV><DIV id="upscaling" class="modal" style="z-index: 1003;"><DIV class="modal-content"><H4>Upscaling towards a bioreactor and styrene harvesting</H4><UL class="collapsible" data-collapsible="expandable"><LI><DIV class="collapsible-header">7.1 How to upscale towards a sustainable bioreactor application</DIV><DIV class="collapsible-body"><P>During our visits to <A href="https://2018.igem.org/Team:Groningen/Human_Practices#avantiuminterview" target="_blank">Avantium</A> and <A href="https://2018.igem.org/Team:Groningen/Human_Practices#photanolinterview" target="_blank">Photanol</A> we saw working bioreactor systems and learned about the challenges of upscaling towards an industrial scale. We decided that designing a bioreactor system for StyGreen is the next logical step towards a working real world application. Our production plant design has three parts: the cellulose preprocessing, the main bioreactor tank and a styrene purification module.</P><FIGURE><FIGCAPTION><I>Figure 1. Production plant design, consisting of a cellulose preprocessing module (left), a main bioreactor tank (mid) and a styrene purification module (right) </I></FIGCAPTION></FIGURE><P>In our production plant, the cellulose preprocessing is performed by ball milling which proved to be effective for <A href="https://2018.igem.org/Team:Groningen/Results#cellulose_assay">Recell</A> from <A href="https://2018.igem.org/Team:Groningen/Human_Practices#knninterview" target="_blank">KNN cellulose</A>, our main cellulose supplier. Ball milling also does not require strong chemicals and leads to an increased solubility and more amorphous regions, increasing cellulosome activity [6]. </P><P>The second part of our production plant is the main bioreactor tank. We chose for a continuous production approach over a batch or fed-batch approach because of two reasons. Firstly, <I>S. cerevisiae</I> only grows slowly on cellulose, which makes growing batches time consuming and less economic. Secondly, continuous production enables us to harvest the styrene without having to destroy the laboriously expressed cellulosomes. The bioreactor contains a biphasic medium of water and an apolar solvent. Ethyl acetate was chosen as it has the right density, doesn’t form micelles and doesn’t denature proteins like other solvents might. The produced styrene has a strong preference for an apolar environment (log P = 2,7) and will therefore localize into the ethyl acetate phase. The difference in density of water and ethyl acetate are just right so that they will mix thoroughly under stirring while separating into two phases once stirring is stopped. The apolar phase can then be siphoned off and be led to the styrene purification module.</P><P>In the styrene purification module the apolar phase is reverse extracted with water to remove polar impurities. The washing water is recycled into the main bioreactor tank to avoid losing nutrients. Next, the ethyl acetate is evaporated under medium vacuum and is also recycled into the main tank; this way only a small pool apolar solvent is required. What is left from the apolar phase is only styrene and some oily impurities. Styrene can be purified from this mixture by evaporation under stronger vacuum. The leftover oil is also recycled into the tank. Once yeast biomass in the many tank exceeds the ideal fermenting conditions, it can be let out and be recycled into new medium as the YPD medium contains yeast extract. Therefore everything that is removed from the tank, except for our product styrene, is fed back into the tank. This makes for an industrial process that creates very little waste while yielding considerable amounts of styrene. For safe storage and further processing of styrene the stabilizator 4-tert-butylcatechol has to be added to prevent potential runaway polymerisations that could be triggered by heat, pressure or light [19].</P><P>In summary, the production plant requires a constant input of cellulosic medium, little input of new water and ethyl acetate, stirring and cooling while outputting styrene and warm water.</P></DIV></LI></UL></DIV></DIV></DIV></DIV></MAIN></DIV></DIV><H4>References</H4><P>[1] Lynd, L. R., Zyl, W. H. van, McBride, J. E., &amp; Laser, M. (2005). Consolidated bioprocessing of cellulosic biomass: an update. Current Opinion in Biotechnology, 16(5), 577–583. http://doi.org/10.1016/J.COPBIO.2005.08.009</P><P>[2] Domozych, D. S., Ciancia, M., Fangel, J. U., Mikkelsen, M. D., Ulvskov, P., &amp; Willats, W. G. (2012). The Cell Walls of Green Algae: A Journey through Evolution and Diversity. Frontiers in Plant Science, 3. doi:10.3389/fpls.2012.00082 </P><P>&lt;[3] Roostaei, J., Zhang, Y., Gopalakrishnan, K., &amp; Ochocki, A. J. (2018). Mixotrophic Microalgae Biofilm: A Novel Algae Cultivation Strategy for Improved Productivity and Cost-efficiency of Biofuel Feedstock Production. Scientific Reports, 8(1). doi:10.1038/s41598-018-31016-1</P><P>[4] Algae Biofuel Market Estimates &amp; Trend Analysis By Application (Transportation, Others), By Region (North America, Europe, Asia Pacific, Rest of World), By Country, And Segment Forecasts, 2018 - 2025. (n.d.). Retrieved from https://www.grandviewresearch.com/industry-analysis/algae-biofuel-market</P><P>[5] Mandels, M., Hontz, L., Nystrom, J.: Enzymatic Hydrolysis of Waste Cellulose. Biotechnology And Bioengineering 16, 1471-1493 (1974)</P><P>[6] Da Silva, A., Inoue, H., Endo, T., Yano, S., Bon, E.P.S.: Milling and pretreatment of sugarcane bagasse and straw for enzymatic hydrolysis and ethanol fermentation. Bioresource Technology 101, 7402-7409 (2010)</P><P>[7] Tianjiao Qu et al.: Ball Milling for Biomass Fractionation and Pretreatment with Aqueous Hydroxide Solutions. ACS Sustainable Chemistry and Engineering 5, 7733-7742 (2017)</P><P>[8] Percival Zhang, Y. H., Cui, J., Lynd, L. R. and Kuang, L. R. A transition from Cellulose Swelling to Cellulose Dissolution by o-Phosphoric Acid: Evidence from Enzymatic Hydrolysis and Supramolecular Structure. Biomacromolecules, 7, 644-648 (2006)</P><P>[9] Wen, F., Sun, J., &amp; Zhao, H. (2010). Yeast surface display of trifunctional minicellulosomes for simultaneous saccharification and fermentation of cellulose to ethanol. Applied and Environmental Microbiology, 76(4), 1251–60. http://doi.org/10.1128/AEM.01687-09 </P><P>[10] McKenna, R., Thompson, B., Pugh, S., &amp; Nielsen, D. R. (2014). Rational and combinatorial approaches to engineering styrene production by Saccharomyces cerevisiae. Microbial Cell Factories, 13(1), 123. http://doi.org/10.1186/s12934-014-0123-2 </P><P>[11] McKenna, R., &amp; Nielsen, D. R. (2011). Styrene biosynthesis from glucose by engineered E. coli. Metabolic Engineering, 13(5), 544–554. http://doi.org/10.1016/J.YMBEN.2011.06.005 </P><P>[12] van Leeuwen, J., Andrews, B., Boone, C., &amp; Tan, G. (2015). Rapid and Efficient Plasmid Construction by Homologous Recombination in Yeast. Cold Spring Harbor Protocols, 2015(9), pdb.prot085100. http://doi.org/10.1101/pdb.prot085100</P><P>[13] Stovicek, V., Holkenbrink, C., &amp; Borodina, I. (2017). CRISPR/Cas system for yeast genome engineering: Advances and applications. FEMS Yeast Research, 17(5). doi:10.1093/femsyr/fox030 </P><P>[14] West, R. W., Chen, S. M., Putz, H., Butler, G., &amp; Banerjee, M. (1987). GAL1-GAL10 divergent promoter region of Saccharomyces cerevisiae contains negative control elements in addition to functionally separate and possibly overlapping upstream activating sequences. Genes &amp; Development, 1(10), 1118-1131. doi:10.1101/gad.1.10.1118</P><P>[15] Peng, B., Williams, T. C., Henry, M., Nielsen, L. K., &amp; Vickers, C. E. (2015). Controlling heterologous gene expression in yeast cell factories on different carbon substrates and across the diauxic shift: A comparison of yeast promoter activities. Microbial Cell Factories, 14(1). doi:10.1186/s12934-015-0278-5  </P><P>[16] Liu, C., Men, X., Chen, H., Li, M., Ding, Z., Chen, G., … Xian, M. (2018). A systematic optimization of styrene biosynthesis in Escherichia coli BL21(DE3). Biotechnol Biofuels, 11, 14. http://doi.org/10.1186/s13068-018-1017-z </P><P>[17] Dragosits, M., &amp; Mattanovich, D. (2013). Adaptive laboratory evolution – principles and applications for biotechnology. Microbial Cell Factories, 12, 64. doi.org/10.1186/1475-2859-12-64</P><P>[18] Ferrari, A. R., Gaber, Y., &amp; Fraaije, M. W. (2014). A fast, sensitive and easy colorimetric assay for chitinase and cellulase activity detection. Biotechnology for Biofuels, 7(1), 37. http://doi.org/10.1186/1754-6834-7-37 </P><P>[19] Kemmere, M. F., Mayer, M. J., Meuldijk, J. and Drinkenburg, A. A. (1999), The influence of 4‐tert‐butylcatechol on the emulsion polymerization of styrene. J. Appl. Polym. Sci., 71: 2419-2422. doi:10.1002/(SICI)1097-4628(19990404)71:14&lt;2419::AID-APP14&gt;3.0.CO;2-W</P></DIV></DIV></DIV><FOOTER><H4 class="center">We thank our sponsors:</H4></FOOTER></BODY></HTML>