<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Model_Molecular_Dynamics skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Model/Molecular Dynamics</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGem Groningen 2018</TITLE></P><DIV class="navbar-fixed"><NAV class="row green"><UL id="dropdown-project" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Description" class="green-text">Description</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Design" class="green-text">Design</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Results" class="green-text">Results</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Demonstrate" class="green-text">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Entrepreneurship" class="green-text">Entrepreneurship</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Awards" class="green-text">Awards</A></LI></UL><UL id="dropdown-notebook" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Notebook" class="green-text">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Protocols" class="green-text">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Safety" class="green-text">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Measurement" class="green-text">Measurement</A></LI></UL><UL id="dropdown-parts" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Parts" class="green-text">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Primers" class="green-text">Primers</A></LI></UL><UL id="dropdown-HP" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Human_Practices" class="green-text">Integrated Human Practices (Gold)</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Public_Engagement" class="green-text">Education &amp; Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Applied_Design" class="green-text">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Ethics" class="green-text">Ethics</A></LI></UL><UL id="dropdown-attribution" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Attributions" class="green-text">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Sponsors" class="green-text">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Crowdfunding" class="green-text">Crowdfunding</A></LI></UL><UL id="dropdown-help" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/InterLab" class="green-text">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Software" class="green-text">Software</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Collaborations" class="green-text">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Embassy" class="green-text">Consulate Meetup</A></LI></UL><UL id="dropdown-team" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Team" class="green-text">Team</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Supervisors" class="green-text">Supervisors</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Team_Building" class="green-text">Team Building</A></LI></UL><UL id="dropdown-model" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:Groningen/Model" class="green-text">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Molecular_Dynamics" class="green-text">Proximity by affinity</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Mathematical_Modeling" class="green-text">Cellulose Degradation</A></LI><LI><A href="https://2018.igem.org/Team:Groningen/Model/Flux_Based_Analysis" class="green-text">Optimizing StyGreen Production</A></LI></UL><DIV class="nav-wrapper" id="nav"><UL class="left" id="nav-buttons"><LI><A href="https://2018.igem.org/Team:Groningen" class="waves-effect waves-light block"><I class="material-icons">home</I></A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-project">Project</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-notebook">Lab</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-model">Model</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-parts">Sequences/Parts</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-HP">Human Practices</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-help">Contributions</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-team">Team</A></LI><LI><A href="#!" class="dropdown-button waves-effect waves-light block" data-activates="dropdown-attribution">Acknowledgements</A></LI></UL><UL class="right" id="contact"><LI title="Click to visit our Facebook page"><A target="_blank " href="https://www.facebook.com/igemgroningen2018/" class="menu-hover waves-effect waves-light block"><SPAN>Groningen.iGEM2018</SPAN></A></LI><LI title="Click to e-mail us"><A target="_blank" href="mailto:igem2018rug@gmail.com" class="menu-hover waves-effect waves-light block"><SPAN>igem2018rug@gmail.com</SPAN></A></LI><LI><A class="menu-hover" id="cross"><I class="material-icons">close</I></A></LI></UL></DIV></NAV></DIV><A href="#!" class="btn-floating btn-large waves-effect waves-light green" id="menu-btn"><I class="material-icons">menu</I></A><DIV class="igem_2018_team_content" style="background-color: white"><DIV class="igem_2018_team_column_wrapper"><DIV class="column"><H4>Introduction</H4><P>The minicellulosome scaffold complex used in our project contains a domain that is thought to bind the scaffold to a cellulose fiber, called the cellulose binding domain (CBD). By doing so it is thought to greatly enhance the efficiency of the bound enzymes simply by proximity to the substrate. Molecular dynamics was used to simulate a CBD protein in proximity to cellulose to gain insight in the molecular interactions between them. </P><P>The CBD structure we are using was crystallized by Tormo et al. [1]. They hypothesize that the binding interface between CBD and cellulose must be at a planar strip of aromatic residues on one side of the beta-sheet sandwich. It is thought that the CBD binds to cellulose by stacking these aromatic amino acids on the ring structure of the glucose monomers in cellulose, as this is also the case in other known cellulose binding domains.</P><P>To verify the binding of CBD to cellulose, and to validate the proposed mechanism of binding, molecular dynamics modeling (MD) was used to simulate the binding of the CBD to cellulose. Furthermore, the resulting structure can be used to determine a free energy of binding.</P><P>MD is ideally suited for subjects involving biomolecules as these are often difficult to visualize experimentally. In particular, version 2.2 of the MARTINI forcefield is used [2]. The forcefield in MD describes the interactions between all the different particles. Many different forcefields exist, but the coarse grained MARTINI forcefield is ideal for cheap, lengthy simulations of proteins. More recently polysaccharides have been properly parameterized for Martini 2 as well [3].</P><P>
Figure 0: Our minicellulosome contains 3 enzymes and a cellulose binding domain bound together by a scaffold protein. An AGA2 mating receptor is fused to the scaffold to allow the scaffold to adhere to the cell surface. </P><H4>Methods</H4><P>To start, the CBD structure from Tormo et al. was used for its structural information. The structure was then coarse grained using the <A href="https://github.com/cgmartini/martinize.py" target="_blank">martinize tool</A>. An elastic network was added to preserve the secondary structure of the protein, at a cutoff of 0.5 nm. The cutoff was chosen to minimize artificial restraints, while keeping the structure of the beta sheets intact.</P><P>Next, the cellulose model from Lopez et al. [4] was used to model cellulose in MARTINI 2. An infinite fiber spanning over the periodic boundary condition was realized by adding bonds between the first and last glucose monomers, effectively realizing an infinite fiber.</P><P>Cellulose and CBD were put together in the same simulation and water and ions were added (150mM NaCl). As Martini 2 water is represented as one spherical bead for every 4 water molecules, it has a tendency to pack in a crystalline lattice structure. This system in particular had a high tendency to freeze, even at 30℃. Most likely this is due to the regular crystalline structure of the cellulose acting as a nucleation point for crystallization of the water beads. To prevent this, 10% of antifreeze beads were added. Antifreeze beads act just like regular water, except other water beads perceive them as slightly larger, thus disrupting crystalline packing. This should not impact the accuracy of the simulation.</P><P>Common parameters for martini simulations were used for dealing with electrostatics [5]. Temperature coupling was achieved by using the Berendsen thermostat at a ref-t of 303.15 K. Pressure coupling was done semiisotropically with Parenello-Rahman at a ref-p of 1.0 bar. While the compressibility in the xy plane was set to 3e-4, the compressibility in the z-direction was set to 0. This was done to keep the infinite fiber in tact and to keep the box from collapsing. The simulations were run for at least 400ns which should provide plenty of sampling. This took about 6 hours of wall time on the Groningen University’s <A href="https://www.rug.nl/society-business/centre-for-information-technology/research/services/hpc/facilities/peregrine-hpc-cluster" target="_blank">Peregrine cluster</A>.</P><H4>Results</H4><P>4 simulations were setup where the CBD was inserted in different orientations around the cellulose fiber. Figure 1 shows the distance between CBD and cellulose over time. In all 4 simulations CBD binds to cellulose relatively quickly and never leaves.</P><P>Figure 1: Distance between CBD and cellulose over time. Every run is plotted separately, all bind to cellulose and never leave within the timeframe of the simulation.</P><P>With adequate sampling, a contact map between CBD and cellulose can be computed (Figure 2). By calculating a distance matrix between CBD and cellulose, and averaging this matrix, a visual representation of how CBD binds to cellulose on average. By taking the minimum distance along one axis of this matrix, a plot can be generated showing which residues of CBD are on average closer to cellulose (Figure 3).</P><P>Figure 2: Distance matrix between the backbone beads of CBD and all cellulose atoms. Distance (nm) is represented by yellow (close) to blue (far). Patches between indices 45-70 and 100-120 of CBD show the closest distance. </P><P>Figure 3: The average distance to cellulose for every backbone bead in the sequence of CBD. There is a clear contact patch around indices 56-67 and around index 112.</P><P>When these close residues are colored on the atomistic structure (Figure 4), it is striking to see that they correspond to planar strip that was hypothesized by Tormo et al. to bind to cellulose. Furthermore this binding patch contains many aromatic amino acids as was also hypothesized. </P><P>Figure 4: Structural visualization of the binding patch of CBD as predicted by Figure 3. Yellow beads are cellulose, the protein backbone of CBD is colored cyan. The space filling protein sequence represents the lowest valleys in the by-sequence distance data. This patch includes 4 of the 5 binding residues as hypothesized by Tormo et al. and contains many aromatic residues.</P><P>To gain more insight in the hypothetical molecular interactions, the atomistic structure of CBD was overlaid over the coarse grained structure, yielding Figure 5. When residues 118, 112, 56, 57 and 67 are visualized as “sticks”, it is striking to see that they would be in contact with the cellulose fiber exactly in the way as described by Tormo et al. This result strongly implies that these residues are indeed responsible for binding to cellulose, at the very least to straight, singular fibers. The planar patch of amino acids likely maximizes the contact area between CBD and cellulose, providing a strong binding affinity between the two.</P><P>Figure 5: Interface between cellulose and CBD. Non binding CBD residues are not visible. The backbone of the protein is in cyan, cellulose is in yellow. The binding patch of CBD consists of Trp 118, Arg 112, Asp 56, His 57 and Tyr 67.</P><P>Additionally a movie was rendered illustrating the strong binding between CBD and cellulose. The movie shows the separate CBD floating in aqueous solution. After some time it finds the cellulose fiber and binds at the binding site as hypothesized by Tormo et al.</P><P>
Now that we have a configuration of the CBD strongly bound to cellulose, it would be interesting to measure the free energy of binding between the CBD and cellulose. This free energy of binding can be derived from the Potential of Mean Force (PMF). The PMF is a property that cannot be directly calculated from mere configurations and trajectories, but rather it has to be determined ‘experimentally’. By using steered MD simulations (SMD), an artificial force is applied on the CBD with the goal of increasing the distance between CBD and cellulose. As restorative forces keep the two molecules together, a high force is initially required to separate the two molecules. When the CBD is far enough away from cellulose, the force required to pull the CBD further away is low. The difference between these forces is known as the Potential of Mean Force. 
</P><P>
The first pulling simulation was started to generate starting configurations for umbrella sampling. Figure 6 shows the pulling force increases over time to overcome the restorative forces of the molecular interactions, and decreases when CBD is pulled loose from cellulose. It is important to note that this pulling force is <B>not</B> directly related to the free energy of binding. The binding free energy is the difference in free energy between the bound and unbound states, and this is not what is measured here. Instead, umbrella sampling of the free energy landscape along the reaction coordinate is needed to probe the real difference in free energy between bound and unbound states.
</P><P>
Figure 6: Pulling force along the reaction coordinate over time. At first pulling force increases linearly. After CBD is pulled loose, the pulling force is allowed to decrease until the specified rate of pulling is reached.
</P><P>
Umbrella sampling was used to determine the potential of mean force of the interaction between cellulose and CBD. Umbrella sampling is a widely established method by which higher energy configurations can be sampled [6]. When the reaction coordinate of the desired event is known, a molecule can be forced along this coordinate by steered MD. Then, separate configurations along the reaction coordinate are taken and restrained by an umbrella potential that mirrors the underlying local energy landscape. This makes it possible to sample the energy landscape along the entire reaction coordinate. This difference between the energy minimum and maximum in this landscape represents the free energy between both states (Figure 7).
</P><P>
Figure 7: Schematic representation of umbrella sampling. First starting configurations are generated by pulling along the reaction coordinate. Then these configurations are restrained by overlapping umbrella potentials, thereby sampling the energy landscape along the entire reaction coordinate. Figure taken from http://www.mdtutorials.com/gmx/umbrella/05_pull.html .
</P><P>
To setup the system for umbrella sampling, first the box from the finished simulation was enlarged as to provide plenty of room for pulling the CBD away from cellulose. The x axis of the box was changed to 30 nm, and the x axis was also used as the direction for pulling. Next, the system was resolvated and equilibrated. 
</P><P>
Umbrella sampling windows of ~0.2 nm were chosen. These configurations were then equilibrated shortly and subjected to the harmonic umbrella potential for 3 ns with a force of 900 kJ/mol. Next gmx wham was used to extract the PMF over distance (Figure 8). As the Center Of Mass (COM) was some distance away from the COM of cellulose from the beginning, the PMF plot does not start at 0. The energy distance between the lowest energy state (binding to cellulose) and the highest energy state (unbound CBD) was determined to be -41.74 kJ/mol.
</P><P>
Figure 8: Plot of the potential of mean force for CBD-cellulose dissociation. PMF converges after around 2.5 nm of pulling. The energy difference between the lowest and highest energy states is roughly -41.74 kJ/mol.
</P><P>
When compared to partitioning free energies of around -30 kJ/mol, measured by Arola et al. [7], a dG of -41.74 seems a bit high. As this particular CBD was not measured by Arola et al. it is possible that our CBD of CipA3 Clostridium thermocellum does bind more strongly to cellulose than these other CBDs. It may be worthwhile to repeat the experiment in the soon to be released Martini 3 forcefield [8], as it is better equipped to handle ring structures and would allow for a more accurate description of both cellulose and the interaction site. This would require reparameterizing cellulose for Martini 3 however.
</P><H4>Conclusion</H4><P>In conclusion, CBD binding to cellulose was modeled using coarse grained molecular dynamics. The results verify that CBD indeed binds to cellulose using the mechanism described by Tormo et al. The binding seems to be very strong as the simulations show CBD never leaves cellulose after it has bound. This observation was verified by PMF calculations, which determined the binding free energy between CBD and cellulose to be -41.74 kJ/mol which is a very strong bond according to experimental data from literature.  These results justify keeping the CBD in our minicellulosome as the strong binding to cellulose will most likely enhance activity of the complex by proximity to cellulose.</P><H4>References</H4><P>[1] Tormo, J., Lamed, R., Chirino, A. J., Morag, E., Bayer, E. A., Shoham, Y., &amp; Steitz, T. A. (1996). Crystal structure of a bacterial family-III cellulose-binding domain: a general mechanism for attachment to cellulose. The EMBO Journal, 15(21), 5739–5751. Retrieved from <A href="http://www.ncbi.nlm.nih.gov/pubmed/8918451" target="_blank">http://www.ncbi.nlm.nih.gov/pubmed/8918451</A></P><P>[2] Monticelli, L., Kandasamy, S. K., Periole, X., Larson, R. G., Tieleman, D. P., &amp; Marrink, S. J. (2008). The MARTINI coarse-grained force field: Extension to proteins. Journal of Chemical Theory and Computation, 4(5), 819–834. <A href="https://doi.org/10.1021/ct700324x" target="_blank">https://doi.org/10.1021/ct700324x</A></P><P>[3] Schmalhorst, P. S., Deluweit, F., Scherrers, R., Heisenberg, C. P., &amp; Sikora, M. (2017). Overcoming the Limitations of the MARTINI Force Field in Simulations of Polysaccharides. Journal of Chemical Theory and Computation, 13(10), 5039–5053. <A href="https://doi.org/10.1021/acs.jctc.7b00374" target="_blank">https://doi.org/10.1021/acs.jctc.7b00374</A></P><P>[4] López, C. A., Bellesia, G., Redondo, A., Langan, P., Chundawat, S. P. S., Dale, B. E., … Gnanakaran, S. (2015). MARTINI coarse-grained model for crystalline cellulose microfibers. Journal of Physical Chemistry B. <A href="https://doi.org/10.1021/jp5105938" target="_blank">https://doi.org/10.1021/jp5105938</A></P><P>[5] De Jong, D. H., Baoukina, S., Ingólfsson, H. I., &amp; Marrink, S. J. (2016). Martini straight: Boosting performance using a shorter cutoff and GPUs. Computer Physics Communications. <A href="https://doi.org/10.1016/j.cpc.2015.09.014" target="_blank">https://doi.org/10.1016/j.cpc.2015.09.014</A></P><P>[6] Domański, J., Hedger, G., Best, R. B., Stansfeld, P. J., &amp; Sansom, M. S. P. (2017). Convergence and Sampling in Determining Free Energy Landscapes for Membrane Protein Association. Journal of Physical Chemistry B. https://doi.org/10.1021/acs.jpcb.6b08445 </P><P>[7] Arola, S., &amp; Linder, M. B. (2016). Binding of cellulose binding modules reveal differences between cellulose substrates. Scientific Reports. https://doi.org/10.1038/srep35358 </P></DIV></DIV></DIV></DIV></DIV><FOOTER><H4 class="center">We thank our sponsors:</H4></FOOTER></DIV></DIV></DIV></BODY></HTML>