Team:XJTLU-CHINA/Exosome Biogenesis
Toggle navigation
Home
Project
Exosome Biogenesis
RNA Packaging
Targeting and Delivery
RNA Stabilization
Results
Modeling
Our Model
Pharmacokinetic model
mRNA delivery simulation
Functional exosome production system
Human Practice
Silver
Gold
Education
Collaborations
Survey and Policy
Notebook
Lab Book
Protocols
Parts
Basic Parts
Composite Parts
Team
Members
Attribution
Safety
InterLab
Exosome Biogenesis
Design Overview
We constructed the tricistronic STEAP3-SDC4-NadB construct at a 1:1:1 ratio by which exosome secretion is supposed to enhanced by 15-fold to 40-fold in yield in the study of Kojima et al. (2018).                     The nSMase is another gene proved to be effective in increasing exosome production by promoting the budding of intraluminal vesicles.                     These 4 genes regulate exosome biogenesis through a different pathway.                     Thus, the tentative combination of these 4 genes became our Booster of exosome production.
Aim of the design
We used human embryonic kidney cell 293T (HEK293T) line to produce exosomes in vitro,                     but the HEK293T cells are only able to produce exosomes at a low rate.                     Without a exosome production booster,                     even though we enrich our therapeutic mRNA relative to exosome quantity,                     the quantity of exosomes may still not be enough for investigating the therapeutic effect because the theraputic mRNA is loaded in the process of biogenesis.                     Especially, large quantity of exosomes generated may lose their activity during recommended −80°C storage due to destabilizing of the surface protein (Jeyaram & Jay, 2017).                     To ensure the efficacy in treatment, utilizing a booster to enhance exosome production would be helpful.
Booster：STEAP3-SDC4-NadB/nSmase
The maturation process of the endosomal system requires the involvement of several pathways and is simultaneously regulated by cellular conditions.                     These four genes in our booster construct are involved in both classic and non-classic biogenesis routes.                     Their roles in the booster will be demonstrated in the following:
STEAP3 (Homo sapiens)
STEAP3 metalloreductase, also known as TSAP6,                     encodes a ferrireductase that is involved in promoting apoptosis and exosome biogenesis in the Endoplasmic reticulum/Golgi-independent or nonclassical pathway.                     TSAP6 is a glycosylated protein present in the trans-Golgi network,                     endosomal–vesicular compartment and cytoplasmic membrane (Lespagnol et al., 2008).                     One possible functions of TSAP6 is that it regulates the expression of the transferrin receptor genes,                     which is needed in exosomal secretion. In addition, cellular homeostasis can affect exosome release,                     and the Ferrireductase activity decrease caused by STEAP3 deficiency may be an explanation for impaired exosome yield.                     Although its detailed mechanism of promoting exosome secretion is still unknown, Lespagnol et al.                     (2008) have shown that exosome production is severely compromised in TSAP6-null cells.
In addition, according to Amzallag et al. (2004), TSAP6 was found to control Translationally controlled tumor protein (TCTP),                     which is involved in proliferation of tumor cells as an anti –apoptosis mediator.                     Furthermore, TCTP is related to preparation of exosome biogenesis for binding to several exosomal resident proteins as a potential binding partner.                     TASP6 is a p53-inducible transmembrane protein and TASP6 expression leads to enhanced TCTP secretion in exosomes.                     Therefore,                     tumor cells export TCTP via large quantities of exosomes in response to survival stress may explain the enhanced exosome secretion effect of the overexpression of STEAP3.
SDC4 (Homo sapiens)
SDC4 is involved in the syndecan-4/syntenin (ESCRT-dependent classical) pathway,                     which can positively regulate specific protein secretion via exosomes from endothelial cells (Ju et al., 2014).                     Syndecan controll exosome formation by its heparan sulphate proteoglycans and binding with cytoplasmic adaptor syntenin.                     This classic secretion pathway requires the interaction of the sydecan-syntenin complex with ALIX which is an ESCRT (endosomal-sorting complex required for transport)-III-binding protein (Baietti et al., 2012).                     Syndecan-4 supports the budding of endosomal membranes to form multivesicular bodies by interacting direactly with ALIX through the LYPX(n)L motif (Baietti et al., 2012).                     Exosome biosynthesis can be further regulated by The activation of Rac1 and RhoG which requires the interactions of syndecan-4 and its cytoplasmic partners synectin and syntenin (Baietti et al., 2012).
(Hessvik & Llorente, 2017)
NadB (Escherichia coli str. K-12 substr. MG1655)
L-aspartate oxidase, encoded by NadB,                     is inferred to possibly boosts cellular metabolism by tuning up the citric acid cycle. Based on the study of Li., et al., (2012),                     we hypothesized that NadB accelerates the citric cycle by integrating the quinolinic acid production pathway of prokaryotes to the NAD+ biosynthesis system in eukaryotes.                     Quinolinic acid is a central metabolite in both prokaryotes and eukaryotes.                     However, in eukaryotes, quinolinic acid is produced from L-tryptophan,                     while in prokaryote it is oxidized from L-aspartate instead. NadB provides this alternative pathway to produce more NAD+.                     Large quantity of the coenzyme NAD+ will then join in the redox reactions and enhance the NAD-dependent metabolic pathways.
(Li., et al., 2012)
nSMase (Homo sapiens)
Sphingomyelin phosphodiesterase 3 (SMPD3) is a neutral sphingomyelinase (nSMase) that is able to make ceramide which can be used for exosome membrane production.                     This non-classic pathway uses lipid modifying enzymes to change the shape of membrane lipid to enhance membrane curvature.                     Ceramide is involved in many biological functions such as cell proliferation,                     apoptosis, differentiation and inflammation (Canals et al., 2011).                     It can promote domain-induced budding of intraluminal vesicles (ILVs) of multivesicular endosomes (MVEs) in an alternative pathway which is independent of the ESCRT machinery,                     but it seems to depend on raft-based microdomains for the lateral segregation of cargo within the endosomal membrane.                     This is due to the fact that ceramide can induce the coalescence of small microdomains into larger domains, and in addition,                     the cone-shaped structure of ceramide might induce spontaneous negative curvature by creating an area difference between the membrane leaflets (Trajkovic et al., 2008).                     It is previously proved that reduction of nSMases will block the ceramide-dependent budding of intraluminal vesicles (ILV) into the lumen of multivesicular bodies (MVB) (Canals et al.,2011). 
Quantitative marker CD63-nluc
CD63-nluc
CD63 is one of the untargeted proteomics-detected markers of exosomes. It is a cell-surface protein that belongs to the tetraspanin family.                     This exosomal tetraspanin is usually used as a marker to quantify CD63-containing extracellular vesicles (EVs) (Kanada et al., 2015).                     Therefore, we employed the nanoluciferase (nluc) bioluminescent systems to quantify the exosomes by combining nluc with CD63. Moreover,                     CD63 was also used as an EV-enriched protein to combine the packaging device to the exsome membrane to enhance the loading process.                     This loading process will be discussed in the Packaging section.
Testing of design
We constructed a pcDNA3.1-STEAP3-IRES-SDC4-IRES-NadB plasmid, pcDNA3.1-nSMase plasmid and pcDNA3.1-CD63-nanoluc plasmid (nanoluc encoding nanoluciferase) to test the efficiency of exosome biogenesis.
1. pcDNA3.1-CD63-nanoluc and two portions of pcDNA3.1 empty plasmid were co-transfected into HEK293T cells as a negative control.
2. pcDNA3.1-CD63-nanoluc plasmid, cDNA3.1-STEAP3-IRES-SDC4-IRES-NadB plasmid and pcDNA3.1 empty plasmid were co-transfected into HEK293T cells as a booster test group.
3. pcDNA3.1-CD63-nanoluc plasmid, pcDNA3.1-STEAP3-IRES-SDC4-IRES-NadB plasmid and pcDNA3.1-nSMase plasmid were co-transfected into HEK293T cells as a booster+nSMase test group.
4. pcDNA3.1-CD63-nanoluc plasmid, pcDNA3.1-STEAP3-IRES-SDC4-IRES-NadB plasmid and pcDNA3.1 empty plasmid were co-transfected into HEK293T cells as a booster test group.
5.
pcDNA3.1-CD63-nanoluc
pcDNA3.1-STEAP3-IRES-SDC4-IRES-NadB
pcDNA3.1-nSMase
Group 1
+
-
-
Group 2
+
+
-
Group 3
+
+
+
Group 4
+
-
+
The exosomes in these two groups were purified by ultracentrifugation from the culture media and quantified by the luminescence output.
References
Amzallag, N., Passer, B. J., Allanic, D., Segura, E., Thery, C., Goud, B., Amson, R., and Telerman, A.                     (2004) TSAP6 facilitates the secretion of translationally controlled tumor protein/histamine-releasing factor via a nonclassical pathway.                     J Biol Chem 279, 46104-46112
Baietti, M. F., Zhang, Z., Mortier, E., Melchior, A., Degeest, G., Geeraerts, A., Ivarsson, Y., Depoortere, F., Coomans, C., Vermeiren, E., Zimmermann, P., and David, G. (2012) Syndecan-syntenin-ALIX regulates the biogenesis of exosomes. Nat Cell Biol 14, 677-685
Canals, D., Perry, D. M., Jenkins, R. W., and Hannun, Y. A. (2011) Drug targeting of sphingolipid metabolism: sphingomyelinases and ceramidases. Br J Pharmacol 163, 694-712
England, C. G., Ehlerding, E. B., and Cai, W. (2016) NanoLuc: A Small Luciferase Is Brightening Up the Field of Bioluminescence. Bioconjug Chem 27, 1175-1187
Hessvik, N. P., and Llorente, A. (2017) Current knowledge on exosome biogenesis and release. Cellular and Molecular Life Sciences 75, 193-208
Kanada, M., Bachmann, M. H., Hardy, J. W., Frimannson, D. O., Bronsart, L., Wang, A., Sylvester, M. D., Schmidt, T. L., Kaspar, R. L., Butte, M. J., Matin, A. C., and Contag, C. H. (2015) Differential fates of biomolecules delivered to target cells via extracellular vesicles. Proc Natl Acad Sci U S A 112, E1433-1442
Kojima, R., Bojar, D., Rizzi, G., Hamri, G. C., El-Baba, M. D., Saxena, P., Auslander, S., Tan, K. R., and Fussenegger, M. (2018) Designer exosomes produced by implanted cells intracerebrally deliver therapeutic cargo for Parkinson's disease treatment. Nat Commun 9, 1305
Lespagnol, A., Duflaut, D., Beekman, C., Blanc, L., Fiucci, G., Marine, J. C., Vidal, M., Amson, R., and Telerman, A. (2008) Exosome secretion, including the DNA damage-induced p53-dependent secretory pathway, is severely compromised in TSAP6/Steap3-null mice. Cell Death Differ 15, 1723-1733
Li, Y., Ogola, H. J., and Sawa, Y. (2012) L-aspartate dehydrogenase: features and applications. Appl Microbiol Biotechnol 93, 503-516
Trajkovic, K., Hsu, C., Chiantia, S., Rajendran, L., Wenzel, D., Wieland, F., Schwille, P., Brugger, B., and Simons, M. (2008) Ceramide triggers budding of exosome vesicles into multivesicular endosomes. Science 319, 1244-1247
Collaborators and Supporters
Location
Rm 363, Science Building
Xi'an Jiaotong-Liverpool University
111 Ren'ai Road, Suzhou, China
215123
Social
FB
Get in touch
igem@xjtlu.edu.cn
