<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_XJTLU-CHINA_Exosome_Biogenesis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:XJTLU-CHINA/Exosome Biogenesis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><NAV class="navbar navbar-default navbar-fixed-top" style="box-shadow: 1px 2px 4px rgba(43,43,43,0.5); margin-bottom: 0px"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI style="margin-left: 100px"><A href="https://2018.igem.org/Team:XJTLU-CHINA">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Exosome_Biogenesis">Exosome Biogenesis</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/RNA_Packaging">RNA Packaging</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Targeting_And_Delivery">Targeting and Delivery</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/RNA_Stabilization">RNA Stabilization</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Demonstrate">Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Modeling </A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Model">Our Model</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Modelling1">Pharmacokinetic model</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Modelling2">mRNA delivery simulation</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Modelling3">Functional exosome production system</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/HP/Silver">Silver</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/HP/Gold">Gold</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Education">Education</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Survey_And_Policy">Survey and Policy</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Labbook">Lab Book</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Attributions">Attribution</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:XJTLU-CHINA/InterLab">InterLab</A></LI></DIV></DIV></NAV></DIV><DIV class="container-fluid" style="background-color: rgb(84,146,197); padding: 40px 30px; margin-top: 35px"><H1 style="font-family:'Impact'; color: white; font-size: 80px; margin-top: 20px; margin-bottom: 50px; float: left; margin-left: 20px">
        Exosome Biogenesis
    </H1><DIV class="container-fluid" style="background-color: white; box-shadow: 0px 0px 4px rgba(0,0,0,0.56); border-radius: 5px; margin-top: 200px; padding-bottom: 50px"><DIV style="text-align: center; margin-top: 50px; border-bottom: 0.8px solid #337ab7; padding-bottom: 20px"><H1 style="font-family:'Impact'; color: #337ab7">
                Design Overview
            </H1></DIV><DIV class="row"><DIV class="col-xs-10"><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    We constructed the tricistronic STEAP3-SDC4-NadB construct at a 1:1:1 ratio by which exosome secretion is supposed to enhanced by 15-fold to 40-fold in yield in the study of Kojima et al. (2018).
                    The nSMase is another gene proved to be effective in increasing exosome production by promoting the budding of intraluminal vesicles.
                    These 4 genes regulate exosome biogenesis through a different pathway.
                    Thus, the tentative combination of these 4 genes became our Booster of exosome production.
                </P></DIV></DIV><DIV style="text-align: center; margin-top: 50px; border-bottom: 0.8px solid #337ab7; padding-bottom: 20px"><H1 style="font-family:'Impact'; color: #337ab7">
                Aim of the design
            </H1></DIV><DIV class="row"><DIV class="col-xs-10"><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    We used human embryonic kidney cell 293T (HEK293T) line to produce exosomes in vitro,
                    but the HEK293T cells are only able to produce exosomes at a low rate.
                    Without a exosome production booster,
                    even though we enrich our therapeutic mRNA relative to exosome quantity,
                    the quantity of exosomes may still not be enough for investigating the therapeutic effect because the theraputic mRNA is loaded in the process of biogenesis.
                    Especially, large quantity of exosomes generated may lose their activity during recommended −80°C storage due to destabilizing of the surface protein (Jeyaram &amp; Jay, 2017).
                    To ensure the efficacy in treatment, utilizing a booster to enhance exosome production would be helpful.
                </P></DIV></DIV><DIV style="text-align: center; margin-top: 50px; border-bottom: 0.8px solid #337ab7; padding-bottom: 20px"><H1 style="font-family:'Impact'; color: #337ab7">
                Booster：STEAP3-SDC4-NadB/nSmase
            </H1></DIV><DIV class="row"><DIV class="col-xs-10"><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    The maturation process of the endosomal system requires the involvement of several pathways and is simultaneously regulated by cellular conditions.
                    These four genes in our booster construct are involved in both classic and non-classic biogenesis routes.
                    Their roles in the booster will be demonstrated in the following:
                </P></DIV></DIV><DIV class="row"><DIV class="col-xs-10"><H2 style="font-family:'Impact'; color: #337ab7; margin-bottom: 20px">STEAP3 (Homo sapiens)</H2><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    STEAP3 metalloreductase, also known as TSAP6,
                    encodes a ferrireductase that is involved in promoting apoptosis and exosome biogenesis in the Endoplasmic reticulum/Golgi-independent or nonclassical pathway.
                    TSAP6 is a glycosylated protein present in the trans-Golgi network,
                    endosomal–vesicular compartment and cytoplasmic membrane (Lespagnol et al., 2008).
                    One possible functions of TSAP6 is that it regulates the expression of the transferrin receptor genes,
                    which is needed in exosomal secretion. In addition, cellular homeostasis can affect exosome release,
                    and the Ferrireductase activity decrease caused by STEAP3 deficiency may be an explanation for impaired exosome yield.
                    Although its detailed mechanism of promoting exosome secretion is still unknown, Lespagnol et al.
                    (2008) have shown that exosome production is severely compromised in TSAP6-null cells.
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    In addition, according to Amzallag et al. (2004), TSAP6 was found to control Translationally controlled tumor protein (TCTP),
                    which is involved in proliferation of tumor cells as an anti –apoptosis mediator.
                    Furthermore, TCTP is related to preparation of exosome biogenesis for binding to several exosomal resident proteins as a potential binding partner.
                    TASP6 is a p53-inducible transmembrane protein and TASP6 expression leads to enhanced TCTP secretion in exosomes.
                    Therefore,
                    tumor cells export TCTP via large quantities of exosomes in response to survival stress may explain the enhanced exosome secretion effect of the overexpression of STEAP3.
                </P></DIV></DIV><DIV class="row"><DIV class="col-xs-10"><H2 style="font-family:'Impact'; color: #337ab7; margin-bottom: 20px">SDC4 (Homo sapiens)</H2><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    SDC4 is involved in the syndecan-4/syntenin (ESCRT-dependent classical) pathway,
                    which can positively regulate specific protein secretion via exosomes from endothelial cells (Ju et al., 2014).
                    Syndecan controll exosome formation by its heparan sulphate proteoglycans and binding with cytoplasmic adaptor syntenin.
                    This classic secretion pathway requires the interaction of the sydecan-syntenin complex with ALIX which is an ESCRT (endosomal-sorting complex required for transport)-III-binding protein (Baietti et al., 2012).
                    Syndecan-4 supports the budding of endosomal membranes to form multivesicular bodies by interacting direactly with ALIX through the LYPX(n)L motif (Baietti et al., 2012).
                    Exosome biosynthesis can be further regulated by The activation of Rac1 and RhoG which requires the interactions of syndecan-4 and its cytoplasmic partners synectin and syntenin (Baietti et al., 2012).
                </P></DIV></DIV><DIV class="row" style="margin-top: 0px"><DIV class="col-xs-6" style="text-align: center"><P style="margin-top: 20px; text-align: center"><STRONG>(Hessvik &amp; Llorente, 2017)</STRONG></P></DIV></DIV><DIV class="row"><DIV class="col-xs-10"><H2 style="font-family:'Impact'; color: #337ab7; margin-bottom: 20px">NadB (Escherichia coli str. K-12 substr. MG1655)</H2><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    L-aspartate oxidase, encoded by NadB,
                    is inferred to possibly boosts cellular metabolism by tuning up the citric acid cycle. Based on the study of Li., et al., (2012),
                    we hypothesized that NadB accelerates the citric cycle by integrating the quinolinic acid production pathway of prokaryotes to the NAD+ biosynthesis system in eukaryotes.
                    Quinolinic acid is a central metabolite in both prokaryotes and eukaryotes.
                    However, in eukaryotes, quinolinic acid is produced from L-tryptophan,
                    while in prokaryote it is oxidized from L-aspartate instead. NadB provides this alternative pathway to produce more NAD+.
                    Large quantity of the coenzyme NAD+ will then join in the redox reactions and enhance the NAD-dependent metabolic pathways.
                </P></DIV></DIV><DIV class="row" style="margin-top: 0px"><DIV class="col-xs-6" style="text-align: center"><P style="margin-top: 20px; text-align: center"><STRONG>(Li., et al., 2012)</STRONG></P></DIV></DIV><DIV class="row"><DIV class="col-xs-10"><H2 style="font-family:'Impact'; color: #337ab7; margin-bottom: 20px">nSMase (Homo sapiens)</H2><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    Sphingomyelin phosphodiesterase 3 (SMPD3) is a neutral sphingomyelinase (nSMase) that is able to make ceramide which can be used for exosome membrane production.
                    This non-classic pathway uses lipid modifying enzymes to change the shape of membrane lipid to enhance membrane curvature.
                    Ceramide is involved in many biological functions such as cell proliferation,
                    apoptosis, differentiation and inflammation (Canals et al., 2011).
                    It can promote domain-induced budding of intraluminal vesicles (ILVs) of multivesicular endosomes (MVEs) in an alternative pathway which is independent of the ESCRT machinery,
                    but it seems to depend on raft-based microdomains for the lateral segregation of cargo within the endosomal membrane.
                    This is due to the fact that ceramide can induce the coalescence of small microdomains into larger domains, and in addition,
                    the cone-shaped structure of ceramide might induce spontaneous negative curvature by creating an area difference between the membrane leaflets (Trajkovic et al., 2008).
                    It is previously proved that reduction of nSMases will block the ceramide-dependent budding of intraluminal vesicles (ILV) into the lumen of multivesicular bodies (MVB) (Canals et al.,2011). 
                </P></DIV></DIV><DIV style="text-align: center; margin-top: 50px; border-bottom: 0.8px solid #337ab7; padding-bottom: 20px"><H1 style="font-family:'Impact'; color: #337ab7">
                Quantitative marker CD63-nluc
            </H1></DIV><DIV class="row"><DIV class="col-xs-10"><H2 style="font-family:'Impact'; color: #337ab7; margin-bottom: 20px">CD63-nluc</H2><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    CD63 is one of the untargeted proteomics-detected markers of exosomes. It is a cell-surface protein that belongs to the tetraspanin family.
                    This exosomal tetraspanin is usually used as a marker to quantify CD63-containing extracellular vesicles (EVs) (Kanada et al., 2015).
                    Therefore, we employed the nanoluciferase (nluc) bioluminescent systems to quantify the exosomes by combining nluc with CD63. Moreover,
                    CD63 was also used as an EV-enriched protein to combine the packaging device to the exsome membrane to enhance the loading process.
                    This loading process will be discussed in the Packaging section.
                </P></DIV></DIV><DIV style="text-align: center; margin-top: 50px; border-bottom: 0.8px solid #337ab7; padding-bottom: 20px"><H1 style="font-family:'Impact'; color: #337ab7">
                Testing of design
            </H1></DIV><DIV class="row"><DIV class="col-xs-10"><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">                   We constructed a <STRONG>pcDNA3.1-STEAP3-IRES-SDC4-IRES-NadB</STRONG> plasmid, <STRONG>pcDNA3.1-nSMase</STRONG> plasmid and <STRONG>pcDNA3.1-CD63-nanoluc</STRONG> plasmid (nanoluc encoding nanoluciferase) to test the efficiency of exosome biogenesis.
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    1. <STRONG>pcDNA3.1-CD63-nanoluc</STRONG> and <STRONG>two portions of pcDNA3.1</STRONG> empty plasmid were co-transfected into HEK293T cells as a negative control.
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    2. <STRONG>pcDNA3.1-CD63-nanoluc</STRONG> plasmid, <STRONG>cDNA3.1-STEAP3-IRES-SDC4-IRES-NadB</STRONG> plasmid and <STRONG>pcDNA3.1</STRONG> empty plasmid were co-transfected into HEK293T cells as a booster test group.
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    3. <STRONG>pcDNA3.1-CD63-nanoluc</STRONG> plasmid, <STRONG>pcDNA3.1-STEAP3-IRES-SDC4-IRES-NadB</STRONG> plasmid and <STRONG>pcDNA3.1-nSMase</STRONG> plasmid were co-transfected into HEK293T cells as a booster+nSMase test group.
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    4. <STRONG>pcDNA3.1-CD63-nanoluc</STRONG> plasmid, <STRONG>pcDNA3.1-STEAP3-IRES-SDC4-IRES-NadB</STRONG> plasmid and <STRONG>pcDNA3.1</STRONG> empty plasmid were co-transfected into HEK293T cells as a booster test group.
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    5.
                </P><TABLE class="table" style="border-right: 1px solid rgba(0,0,0,0.1); border-left: 1px solid rgba(0,0,0,0.1); border-bottom: 1px solid rgba(0,0,0,0.1)"><TBODY><TR><TH style="text-align: center">pcDNA3.1-CD63-nanoluc</TH><TH style="text-align: center">pcDNA3.1-STEAP3-IRES-SDC4-IRES-NadB</TH><TH style="text-align: center">pcDNA3.1-nSMase</TH></TR><TR><TH>Group 1</TH><TH style="font-weight: 400; text-align: center">+</TH><TH style="font-weight: 400; text-align: center">-</TH><TH style="font-weight: 400; text-align: center">-</TH></TR><TR><TH>Group 2</TH><TH style="font-weight: 400; text-align: center">+</TH><TH style="font-weight: 400; text-align: center">+</TH><TH style="font-weight: 400; text-align: center">-</TH></TR><TR><TH>Group 3</TH><TH style="font-weight: 400; text-align: center">+</TH><TH style="font-weight: 400; text-align: center">+</TH><TH style="font-weight: 400; text-align: center">+</TH></TR><TR><TH>Group 4</TH><TH style="font-weight: 400; text-align: center">+</TH><TH style="font-weight: 400; text-align: center">-</TH><TH style="font-weight: 400; text-align: center">+</TH></TR></TBODY></TABLE><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    The exosomes in these two groups were purified by ultracentrifugation from the culture media and quantified by the <STRONG>luminescence output</STRONG>.
                </P></DIV></DIV><DIV style="text-align: center; margin-top: 50px; border-bottom: 0.8px solid #337ab7; padding-bottom: 20px"><H1 style="font-family:'Impact'; color: #337ab7">
                References
            </H1></DIV><DIV class="row" style="margin-bottom: 40px"><DIV class="col-xs-10"><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    Amzallag, N., Passer, B. J., Allanic, D., Segura, E., Thery, C., Goud, B., Amson, R., and Telerman, A.
                    (2004) TSAP6 facilitates the secretion of translationally controlled tumor protein/histamine-releasing factor via a nonclassical pathway.
                    J Biol Chem 279, 46104-46112
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    Baietti, M. F., Zhang, Z., Mortier, E., Melchior, A., Degeest, G., Geeraerts, A., Ivarsson, Y., Depoortere, F., Coomans, C., Vermeiren, E., Zimmermann, P., and David, G. (2012) Syndecan-syntenin-ALIX regulates the biogenesis of exosomes. Nat Cell Biol 14, 677-685
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    Canals, D., Perry, D. M., Jenkins, R. W., and Hannun, Y. A. (2011) Drug targeting of sphingolipid metabolism: sphingomyelinases and ceramidases. Br J Pharmacol 163, 694-712
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    England, C. G., Ehlerding, E. B., and Cai, W. (2016) NanoLuc: A Small Luciferase Is Brightening Up the Field of Bioluminescence. Bioconjug Chem 27, 1175-1187
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    Hessvik, N. P., and Llorente, A. (2017) Current knowledge on exosome biogenesis and release. Cellular and Molecular Life Sciences 75, 193-208
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    Kanada, M., Bachmann, M. H., Hardy, J. W., Frimannson, D. O., Bronsart, L., Wang, A., Sylvester, M. D., Schmidt, T. L., Kaspar, R. L., Butte, M. J., Matin, A. C., and Contag, C. H. (2015) Differential fates of biomolecules delivered to target cells via extracellular vesicles. Proc Natl Acad Sci U S A 112, E1433-1442
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    Kojima, R., Bojar, D., Rizzi, G., Hamri, G. C., El-Baba, M. D., Saxena, P., Auslander, S., Tan, K. R., and Fussenegger, M. (2018) Designer exosomes produced by implanted cells intracerebrally deliver therapeutic cargo for Parkinson's disease treatment. Nat Commun 9, 1305
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    Lespagnol, A., Duflaut, D., Beekman, C., Blanc, L., Fiucci, G., Marine, J. C., Vidal, M., Amson, R., and Telerman, A. (2008) Exosome secretion, including the DNA damage-induced p53-dependent secretory pathway, is severely compromised in TSAP6/Steap3-null mice. Cell Death Differ 15, 1723-1733
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    Li, Y., Ogola, H. J., and Sawa, Y. (2012) L-aspartate dehydrogenase: features and applications. Appl Microbiol Biotechnol 93, 503-516
                </P><P style="font-size: 17px; text-align: justify; font-weight: normal; margin-top: 20px; margin-bottom: 20px">
                    Trajkovic, K., Hsu, C., Chiantia, S., Rajendran, L., Wenzel, D., Wieland, F., Schwille, P., Brugger, B., and Simons, M. (2008) Ceramide triggers budding of exosome vesicles into multivesicular endosomes. Science 319, 1244-1247
                </P></DIV></DIV></DIV><DIV style="background-color: white; box-shadow: 0px 0px 4px rgba(0,0,0,0.56); border-radius: 5px; margin-top: 50px; background-color: white; padding-bottom: 40px"><DIV style="text-align: center; padding-top: 40px"><H1 style="margin: 0px auto; font-family:'Impact'; color: #337ab7">Collaborators and Supporters</H1></DIV></DIV><DIV class="container-fluid" style="border-top: 1px solid rgba(0,0,0,0.15); height: 240px; padding-top: 30px; background-color: rgb(84,146,197);"><DIV class="row" style="padding: 0px 50px"><DIV class="col-md-4"><DIV><P style="font-size: 26px; color: white;">Location</P></DIV><DIV><P style="line-height: 15px; font-size: 16px; color: white;">Rm 363, Science Building</P><P style="line-height: 15px; font-size: 16px; color: white;">Xi'an Jiaotong-Liverpool University</P><P style="line-height: 15px; font-size: 16px; color: white;">111 Ren'ai Road, Suzhou, China</P><P style="line-height: 16px; font-size: 16px; color: white;">215123</P></DIV></DIV><DIV class="col-md-4"><DIV><P style="font-size: 26px; color: white;">Social</P></DIV><DIV><P style="line-height: 15px; font-size: 16px; color: white;">FB</P></DIV></DIV><DIV class="col-md-3"><DIV><P style="font-size: 26px; color: white;">Get in touch</P></DIV><DIV><P style="line-height: 15px; font-size: 16px; color: white;">igem@xjtlu.edu.cn</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>