<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UGA-Georgia_Description_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UGA-Georgia/Description/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><UL class="nav navbar-nav uga-nav-li"><LI><A class="navbar-brand" href="https://2016.igem.org/Team:UGA-Georgia">Home</A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Modeling">Modeling</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Description/Experiment">Experiments and Protocol</A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Description/Results">Results</A></LI></UL><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Interlab">InterLab</A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Lab_Safety">Lab Safety</A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Human">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:UGA-Georgia/Notebook">Notebook</A></LI><UL class="nav navbar-nav navbar-right uga-nav-li"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><B>iGEM 2016</B></A></LI><LI><A href="https://2016.igem.org/">Main Menu</A></LI><LI class="dropdown-submenu"><A tabindex="-1" href="#">Teams</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/Team_List?year=2016">Team List</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Community/Map">Team Map</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/Team_Wikis?year=2016">Team Wikis</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/Team_Parts?year=2016">Team Parts</A></LI></UL><LI class="dropdown-submenu"><A tabindex="-1" href="#">Community</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/Community_Bulletin">Community Bulletin</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Community/Meetups">Meetups</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Community/Connect_with_the_iGEM_Community">Connet with iGEMers</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/Newsletters">Newsletters</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Community/Mentorship">Mentorship</A></LI><LI><A class="uga-nav-a-small" href="http://forum.igem.org/">iGEM Forum</A></LI><LI class="dropdown-submenu"><A tabindex="-1" href="#">Competition</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Requirements">Requirements</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/IGEM_2016_Registration">Register A Team</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Registration_Handbook">Registration Handbook</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Requirements/Team_Registration_Fee">Team Registration Fee</A></LI><LI><A class="uga-nav-a-small" href="http://parts.igem.org/Help:2016_DNA_Distribution">Distribution Kit</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Human_Practices">Human Practices</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Tracks">Tracks</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Tracks/Measurement/Interlab_study">InterLab Study</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Survey">Survey</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Tracks/High_School">High School</A></LI><LI><A href="https://2016.igem.org/Calendar">Calendar</A></LI><LI><A href="https://2016.igem.org/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Giant_Jamboree">Giant Jamboree</A></LI><LI><A href="https://2016.igem.org/Judging">Judging</A></LI><LI class="dropdown-submenu"><A tabindex="-1" href="#">Resources</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Resources/Start_A_Team">Start A Team</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Resources/Funding">Funding</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Resources/Visa_Help">Visa Help</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Sponsors/Partner_Special_Offers">Partner Special Offers</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Resources/Telling_your_Story">Telling Your Story</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/Registry">Parts Registry</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Resources/Plant_Synthetic_Biology">Plant Synthetic Biology</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Resources/FAQ">FAQs</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Resources/PLOS_iGEM_Collection">iGEM PLOS Publication</A></LI><LI class="dropdown-submenu"><A tabindex="-1" href="#">About</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/About">About iGEM</A></LI><LI><A class="uga-nav-a-small" href="https://2016.igem.org/Contact">Contact</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/Sponsors">Sponsors</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/Previous_iGEM_Competitions">Previous Competitions</A></LI><LI><A class="uga-nav-a-small" href="https://igem.org/">iGEM.org</A></LI><LI><A href="https://2016.igem.org/Sitemap">Sitemap</A></LI><UL><DIV class="row"><DIV class="col-md-8 col-xs-8"><H1>Ribosome Binding Site Library Results</H1>
Last year our team was able to successfully characterize thirteen mutants to add to this library. To continue this library, extracts were gathered from the remaining mutants. Of those mutants one came back positive for the mCherry gene. Fluorescence data and sequencing was done on the mCherry extracts of this mutants. The data showed an expression level of 132% of the native. In this mutant an adenine nucleotide  was substituted for a guanine nucleotide in the first base pair of the space region. This resulted in an expression level of 132% when compared to the native. 
</DIV></DIV><DIV class="row"><DIV class="container"><DIV class="col-md-12 col-xs-12"><H1>Spacer Library Results</H1><B>Table 1.</B> Spacer Library Sequences with Fluorescence Values and ∆G Values
<H2>Spacer Library Methods Development</H2>
The mutations for the new spacer library were formed using the primers to essentially change the location of the RBS in the AT or TA background along with introducing complementary sequences to form mRNA secondary structure mutations. As stated before, this method of introducing secondary mRNA structures in our model was derived from mRNA modeling with delta G values of the folding kept in mind so that there was a range of delta G values in the experiment to see if delta G values have an effect on fluorescence.
In order to add onto to the novel protocol we developed last year for measuring fluorescence for mCherry, we optimized the lysis of the cells for extraction of mCherry by changing the lysis agent from sonication to chemical lysis buffers. This would enable us to obtain more precise and accurate data as there is less human error involved.
<H2>Preliminary results</H2>
The process of transformation into <I>Methanococcus maripaludis</I> is very complex and difficult. This year the team was able to successfully gather complete sets of data for five of the fifteen constructs that we originally planned to integrate into our spacer library, while completing the other 10 constructs completely in<I> E. coli</I> and partially in<I> M. maripaludis.</I> They just lack the last step of extracting from and isolating pure colonies from the anaerobic cultures, many of them are mixed without the ideal genetic end product.
Since our model utilizes fluorescence values to measure protein expression levels, we can generalize from our data that within base pairs for the length of the spacer region, 7 to 5 bp that 5 base-pairs would produce the largest amount a given protein. From comparing the AT and TA backgrounds we see that there is little difference between the two suggesting the background may not be significant. Furthermore the data suggest having a structure with two  Ribosome Binding Sites and the complementary structure to the RBS in the loop decreases the fluorescence values. Seeing that the construct with an RBS and complementary RBS in the loop lead to lower fluorescence values, it can be theorized that because the ribosomes have two locations to bind it may cause competitive binding between the two locations. Based on the data on hand it seems like there is no significant correlation between delta G values and fluorescence. Within the next year we hope to complete our characterization of the Spacer Mutant and mRNA secondary structure library complete with fluorescence values for protein expression levels. This will allow us to make more precise extrapolations as to the impact the spacer region and mRNA secondary structure has on protein expression levels. This would culminate in a variety of new genetic tools in Archaea to alter protein expression levels for different types of experiments and designs.</DIV></DIV></DIV><DIV class="row"><DIV class="container"><DIV class="col-md-12 col-xs-12"><H1>Methods Development Results</H1><H2>Lysis Optimization</H2><P>Key: SP- SDS + PIPES; TP- Triton + PIPES; P- PIPES alone; Pson- PIPES + sonication; F- Formaldehyde</P><P><B>Figure 7</B>. Summary of different extraction methods performed on L1C12 culture. A plate reader was used to obtain the fluorescent readings of mCherry protein displayed above.</P>
The fluorescence values in the figures were calculated the value with the background of the wild type plasmid “fluorescence” values being subtracted from the fluorescence reading. This gave a more accurate record of the fluorescence reading of the mCherry protein for each treatment.
<B>Figure 7</B> is representative of this experiment. The results from the lysing treatments include: SDS + PIPES, Triton + PIPES, PIPES, PIPES + sonication and formaldehyde treatment. Lysing the cell was more successful than performing sonication, and it was shown that SDS + PIPES was the most reproducible for obtaining significant fluorescence reading. Also, fixing the cells with formaldehyde caused a decrease in the fluorescent reading for every treatment. The lower fluorescence may be due to the fact that the cell extracts were fixed with formaldehyde before oxygen exposure, which may have prevented mCherry from fluorescing. Since formaldehyde is a fixer, it may be best to add formaldehyde after oxygen exposure after the protein has fluorescence. This point is investigated in our “mCherry Extraction Maturation for Fixation Experiment.”
The results of the lysing experiment, the reproducibility of SDS and PIPES, was incorporated into the Spacer Library Experiment for the extraction of mCherry.



<H2>mCherry Extract Maturation for Fixation</H2><DIV class="row"><DIV class="container"><DIV class="col-md-6 col-xs-6"><P><B>Figure 8</B>. Summary of the fluorescent readings of the mCherry protein is displayed above for the L2C16 cells resuspended in PIPES for each treatment before and after oxygen exposure.</P></DIV><DIV class="col-md-6 col-xs-6"><P><B>Figure 9</B>. Summary of the fluorescent readings of the mCherry protein is displayed above for the L2C16 cells resuspended in media salt for each treatment before and after oxygen exposure.</P></DIV></DIV></DIV>
Overall, the results proved that adding formaldehyde after oxygen exposure for both media salt and PIPES was more successful in the fluorescence of the protein than before oxygen exposure. For the other treatments, SDS and Triton, there was no significant difference like what was seen with the formaldehyde treatment whether the mCherry was exposed to oxygen before or after treatment. This was likely the result because formaldehyde is a fixer, and it was best to add formaldehyde after oxygen exposure since it is after the fluorescence of the protein. This method was then incorporated into the Archaeal Interlab study protocol.

<H1>Conclusion</H1>
For the RBS library that was started last year in 2015, the UGA iGEM team added another mutant with an adenine nucleotide substituted for a guanine nucleotide in the first base pair of the spacer region. This resulted in an expression level of 132% when compared to the native. 
For the Spacer library that was started this year, the team managed to add 4 spacer mutants and one mRNA secondary structure mutant. The fluorescence data seems to favor the 5 base pair spacer region compared to a 7 or 6 base pair spacer region. The mRNA secondary structure mutant with an RBS sequence and its complement in the loop seemed to inhibit protein expression levels.
In further developing our methods in working with Archaea and within our protocol, the team optimized our lysis procedure for mCherry extraction by replacing sonication with SDS-PIPES buffer. 
We also further developed the interlab protocol by experimenting with the oxygen maturation step for mCherry. Our experiments showed that oxygen exposure before fixation with formaldehyde produced better fluorescence results.



  </DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>