Team:UGA-Georgia/Description
Home
Parts
Modeling
Project
Description
Experiments and Protocol
Results
InterLab
Lab Safety
Team
Attributions
Software
Human Practices
Notebook
iGEM 2016
Main Menu
Teams
Team List
Team Map
Team Wikis
Team Parts
Community
Community Bulletin
Meetups
Connet with iGEMers
Newsletters
Mentorship
iGEM Forum
Competition
Requirements
Register A Team
Registration Handbook
Team Registration Fee
Distribution Kit
Human Practices
Tracks
InterLab Study
Survey
High School
Calendar
Safety
Giant Jamboree
Judging
Resources
Start A Team
Funding
Visa Help
Partner Special Offers
Telling Your Story
Parts Registry
Plant Synthetic Biology
FAQs
iGEM PLOS Publication
About
About iGEM
Contact
Sponsors
Previous Competitions
iGEM.org
Sitemap
Description
The goal of the University of Georgia’s iGEM team is develop tools and methods to further the utility of Archaea in the field of synthetic biology. In 2013, the iGEM team established the ability of Methanococcus Maripaludis to produce the isoprenoid compound, Geraniol, with low yields. Then, in 2014, our team developed a method to quantify protein expression using the fluorescent protein mCherry as a reporter system. The following year we used this reporter system to quantify mCherry expression for single-base pair nucleotide substitutions in M. Maripaludis to create a ribosome binding site library. In a fruitful Interlab collaboration with eight other iGEM teams, we have quantified the mCherry fluorescence for a library of ribosomal binding sites or RBS with single-base pair nucleotide substitutions in Methanococcus maripaludis, a model organism for Archaea. The goal of our Interlab was to evaluate the reliability of our fluorescent measurements, which we then determined to have high reliability. Furthermore, we continued with our Interlab study by this time giving participating teams a standardized protocol to follow with two choices for fluorescence values: flow cytometry and plate reader.
RBS Library
Last year our team was able to successfully characterize thirteen mutants to add to this library.  Wanting to complete this library, the students gathered extracts from the remaining mutants. Of those mutants, one came back positive for the mCherry gene. Fluorescence data and sequencing was done on the mCherry extracts of this mutants. As a continuation of last year’s project, this year the University of Georgia’s iGEM team added another single base pair mutant to the RBS library (See Figure 6 in results section).
Spacer Library
Furthermore, we started a new spacer library, adding a 7 base pair spacer region consisting of adenine and thymine nucleotides in the spacer region located between the Ribosome Binding Site (RBS) and Start Codon. It was hypothesized that the length of the spacer region between the ribosome binding site and the start codon would have an effect on the translational efficiency of protein expression due to proximity of the RBS to the start codon.
It was also thought that the secondary structure of the mRNA may play a role in translational efficiency. One interesting mutation we incorporated was a two RBS model in order to see if the two RBS would inhibit each other as predicted. As a control, a few secondary structures were modeled in order to produce mutations where the RBS would be in the stem and in the loop of the secondary structure where they don’t anneal to each or any other sequences. This would presumably allow the ribosome to bind to those sequences.
We have systematically altered base pairs within this region in our mutants with the aim of determining the optimal length of the region through our fluorescence measurements. In order to make this model more encompassing, we decided to include a few mutations including the AT  background in addition to the TA background. Presumably, the strength of mCherry fluorescence would change when varying the length of the spacer region.
The mRNA secondary structure mutations include alterations of the sequences to produce mutations with 2 ribosome binding sites adjacent to each other and mutants in which the genetic sequences of the target regions were altered to contain both an RBS and a sequence complementary to RBS within the stem and loop portion of the mRNA. These were constructed using the Vienna RNA Package 2 Model.
Figure 3. Model of mRNA secondary structure for M15 contain two ribosome binding sites
Figure 2. Conceptual Model of Spacer Mutations
Method Development
mCherry Extraction Optimization
The goal was to optimize the lysing process of the cell. In the past, after harvesting our methanococcus cells, we would re-suspend the cell pellets in a lysing buffer known as PIPES followed by sonication. However, there are better and easier methods to lyse the cell since performing sonication is more prone to human error. To do so, four different lysis treatments were experimented on 3 different cultures. The cultures, L1C12, L1C18 and L2C16, were chosen. The lysis treatments included re-suspension in PIPES solution (1), Triton added to PIPES solution (2), Sodium Dodecyl Sulfate (SDS) added to PIPES solution (3), and PIPES solution and sonication (4). The cell extracts were exposed to oxygen to prepare to measure the fluorescence of mCherry using a plate reader. For each treatment (PIPES, SDS + PIPES, Triton + PIPES and PIPES + sonication), an aliquot was treated with 5% formaldehyde. The fluorescence readings of these cell extracts were compared to the cell extracts that were not treated with formaldehyde. The same procedure was repeated for the wild type culture as our negative control.
Figure 4. Cell Lysis for mCherry Extraction
mCherry Oxidation Maturation for Fixation
After the lysing portion of the experiment the fluorescence of mCherry protein is measured to determine which treatment worked best. The higher the fluorescence, the more effective the lysing treatment. However, we figured that the fluorescence of the protein might also be dependent on the timing of the oxygen exposure. Depending on the lysing treatment, it may produce strong signals that prevent the protein from fluorescing to its fullest. Therefore, we decided to also test whether the addition of the lysing treatment is best before or after oxygen exposure.
The culture, L2C16, was used to perform all the treatments. The cells were first centrifuged and re- suspended in PIPES, a lysing buffer.
The following treatments were added to a set of cell samples before oxygen exposure and to another set of cell samples after oxygen exposure:
Formaldehyde
SDS
Triton
Freeze for 20 minutes + No treatment
No treatment.
To test the effect of the lysing treatment alone without using a lysing solution such as PIPES, we re- suspended cells in a suspension buffer known as media salt.
The following treatments were added to a set of cell samples before oxygen exposure and to another set of cell samples after oxygen exposure:
Formaldehyde
SDS
Triton
No treatment.
A plate reader was used to measure the fluorescence of mCherry protein.
Figure 5. mCherry Extract oxygen maturation
