<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Valencia_Parts"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Valencia/Parts">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Valencia/Parts&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia/Parts&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia/Parts&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Valencia/Parts" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Valencia/Parts</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><UL id="topnav"><LI><A href="https://2010.igem.org/Team:Valencia">Home</A><SPAN><A href="https://2010.igem.org/Team:Valencia">Mad yeasts on Mars?</A> |
            <A href="https://2010.igem.org/Team:Valencia#The Valencia IGEM Team 2010">The Valencia Team</A> |
            <A href="https://2010.igem.org/Team:Valencia#News">News</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:Valencia/Team">The Team</A><SPAN><A href="https://2010.igem.org/Team:Valencia/Team">The Team</A> |
            <A href="https://2010.igem.org/Team:Valencia/Universities">The Universities</A> |
            <A href="https://2010.igem.org/Team:Valencia/Akw">Acknowledgements</A> |
            <A href="https://2010.igem.org/Team:Valencia/Attributions">Attributions</A> |
            <A href="https://2010.igem.org/Team:Valencia/Gallery">Gallery</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:Valencia/Project">The Project</A><SPAN><A href="https://2010.igem.org/Team:Valencia/Project">Overview</A> |
            <A href="https://2010.igem.org/Team:Valencia/Terraforming">Terraforming</A> |
            <A href="https://2010.igem.org/Team:Valencia/Technologies">Creating Technologies</A> |
            <A href="https://2010.igem.org/Team:Valencia/lea">Surviving Mars</A> |
            <A href="https://2010.igem.org/Team:Valencia/prion">Regulating Mars temperature</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:Valencia/Parts">The Parts</A><SPAN><A href="https://2010.igem.org/Team:Valencia/Parts">Submitted Biobricks</A> |
            <A href="https://2010.igem.org/Team:Valencia/Parts#Making our Biobricks">Making our Biobricks</A> |
            <A href="https://2010.igem.org/Team:Valencia/Notebook/Calender">Notebook</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:Valencia/Modeling">Modeling</A><SPAN><A href="https://2010.igem.org/Team:Valencia/Modeling">Modeling Mad Yeast On Mars</A>|
            <A href="https://2010.igem.org/Team:Valencia/Modeling#Welcome to the Yeastworld: theoretical background ">Yeastworld: theoretical background</A>|
            <A href="https://2010.igem.org/Team:Valencia/Modeling#Yeastworld Mathematics">Yeastworld Mathematics</A></SPAN></LI><LI><A href="#">Human Practices</A><SPAN><A href="https://2010.igem.org/Team:Valencia/Ethics">Ethics in terraforming</A> |
            <A href="https://2010.igem.org/Team:Valencia/Safety">Safety</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:Valencia/Achievements">Achievements</A></LI><LI><A href="https://2010.igem.org/Team:Valencia/Contact">Contact</A></LI></UL><DIV id="HomeRight"><P>
Time goes by... 



(El tiempo pasa...)

Follow us: 
</P><P>Our main sponsors:
</P><P>Our institutions:
</P><P>
Visitor location:
</P></DIV><DIV id="HomeCenter"><DIV id="Titulos"><P>Submitted Biobricks
</P></DIV><H2><SPAN class="mw-headline" id="Parts">Parts</SPAN></H2><P>We placed the first prion-based inheritance element construction into a <I>standard biological parts</I>, also called <I>BioBricks</I>. These are nucleic acid coding molecular biological functions, along with the associated information defining and describing the parts. 
You can find more information about this on the website of <A href="http://biobricks.org/" class="external text" rel="nofollow">The BioBricks Foundation</A>.
</P><P>Using these molecules any synthetic biologist or biological engineer can program living organisms. All our parts are available to anyone for free via MIT's <A href="http://partsregistry.org" class="external text" rel="nofollow">Registry of Standard Biological Parts</A>.
</P><P>We also placed the PM2 gene, coding the LEA3 protein, in another Biobrick, and probed that it conferes to our <I>E. coli</I> protection against extreme temperature.
</P><P>&lt;groupparts&gt;iGEM010 Valencia&lt;/groupparts&gt;</P><H2><SPAN class="mw-headline" id="LEA"><B>LEA</B></SPAN></H2><H3><SPAN class="mw-headline" id="Function"> Function </SPAN></H3><P>PM2 corresponds to the LEA3 protein from soybean (<I>Glycine max</I>). Late Embryogenesis Abundant (LEA) proteins are known for their roles in plant embryogenesis, as important dessication-resistance factors. It has also been shown that they confer tolerance under several stress conditions in <I>Escherichia coli</I>. You can read more about it in our wiki and in several references, as, for example, the following: Liu <I>et al.</I> (2010).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>LEA14</B> Three dimensional representation of the LEA14 proteinn.(EPDB)</DIV></DIV></DIV></DIV><P>
PM2 can be useful as a general stress-resistance factor. It could improve survival under low or high temperatures, water stress and maybe other atypical conditions.
</P><H3><SPAN class="mw-headline" id="Original_source">Original source</SPAN></H3><P>Our PM2 was inserted into the pET28a vector and transformed into <I>E. coli</I> BL21 Star cells. 
We sincerely thank professor Yizhi Zheng and Yun Liu (College of Life Science, Shenzhen University. Guangdong, China) for sending us the plasmid with the gene, and, in consequence, let us develop our project.
</P><H3><SPAN class="mw-headline" id="Sequence">Sequence</SPAN></H3><P>The amplification previous to the clonation into the pSB1C3, was made using the primers previously employed in Liu and Zheng (2005):
</P><UL><LI> Forward actagtagcggccgctgcagATGGCGTCCAAGAAAC
</LI><LI> Reverse tctagaagcggccgcgaattcTGCGTCTATATATAC
</LI></UL><P>(capital letters indicate the region of the sequence that pairs with the coding sequence of PM2).
</P><P>The sequence of the part, the  PM2 gene, was first reported by Z.-Y. Chen, .Y-I.C. Hsing and T.-Y. Chow and it is available at the NCBI Nucleotide database (accession number AF532313). The complete sequence is pasted below:
</P><PRE>       1 cacaaaagtg ttccacttga gtgaaaagta gtgtgttaag aactaaacaa tttttcaatg
      61 gcgtccaaga aacaagagga gcgagctgag gcagctgcga aagttgctgc caaagaactc
     121 gaacaagtca acagagaaag aagagaccgt gatttcggtg ttgttgctga acaacaacaa
     181 caacatcatc aggaagatca acaaaaacgt ggtgtaatcg ggtccatgtt taaggcggtg
     241 caagacacct acgagaacgc caaggaagct gtcgttggca agaaagaagc tactaataac
     301 gcgtacagta atacagaggt tattcacgat gttaacattc agcccgatga cgtgtcggca
     361 acgggggaag taagggacat atcagccaca aagactcatg atatctacga ttctgccacg
     421 gacaacaaca acaacaaaac cggttccaag gtcggagagt acgcagatta cgcttctcag
     481 aaggccaagg aaacaaaaga tgcaacgatg gaaaaagctg gagagtacac agattatgct
     541 tcgcagaaag cgaaggaagc gaagaagacg accatggaga agggtggaga atacaaggat
     601 tactctgcgg agaaagctaa ggagagaaaa gatgctactg tgaataagat gggagagtat
     661 aaggactatg ctgcggagaa agccaaagag gggaaagatg ctactgtgaa taaaatggga
     721 gagtataagg actatgctgc ggagaaaacg aaagagggga aagatgccac tgtgaataag
     781 atgggagagt ataaggatta cactgcggag aaggcgaaag aggggaaaga tacgacgttg
     841 gggaagcttg gggagctgaa ggacacggct tcggatgcgg cgaagagggc cgtgggttac
     901 ttgagcggca agaaagagga aactaaagag atggcttcgg agaccgccga ggcgacggcg
     961 aataaggcag gggagatgaa ggaggcaaca aagaaaaaga cggcggagac cgcggaggcg
    1021 gcgaagaata aggcggggga gatcaaggac agagccgcgg agacggcgga ggccgcgaaa
    1081 aacaagaccg cggagaccgc ggaagtgacg aagaataagg ctttggagat gaaggatgca
    1141 gcgaaggaca ggaccgctga gacaacggat gcggcgaagc agaaaactgc acaggcaaag
    1201 gagaacacca aggaaaatgt gagtggtgca ggtgaaactg caaggaggaa aatggaagag
    1261 ccaaagcttc aaggtaaaga agggtatggg ggccgtggag acaaggtggt ggtgaaagtg
    1321 gaagagagtc gaccaggggc aattgcggaa acgctgaaag ccgccgacca gattgcggga
    1381 cagaccttca acgatgtagg acgcttcgat gaagagggtg tcgtcaatgt ggagcgccgc
    1441 aagaaataat taaaacgtga tctatgatac aacaatatta gtatatatag acgcatgcag
    1501 tttatatagt atatattgtc atgttgtatg tttttacatt ttggtttgct tgtttacatt
    1561 ctcttcaaaa aaaaaaaaaa tgtgtagtac gtgtaaggtt ttgaagattg gttctaggct
    1621 ccgtgggaac catttcaaca ataaacattt tgcgcgttct tgtacacgta gtgatgagaa
    1681 gagatgcctt atgggcagta tcatctaaaa cttattttca tccatcatag aatttggatc
    1741 t
</PRE><H2><SPAN class="mw-headline" id="Prionic_Switch"><B>Prionic Switch</B></SPAN></H2><H3><SPAN class="mw-headline" id="Components">Components</SPAN></H3><P>The switch is formed by two different parts: the activator and the reporter. The activator part is a construction of two fragments: the NM domains of the protein Sup35, which confers to the protein the prionic activity, and the GR<SUP>526</SUP>portion, which contains the DNA-binding and transcription-activation domains. The ligand-binding domain of the protein GR was eliminated, decoupling the response of the protein to the presence of glucocorticoids, and thus generating a constitutive transcription activation factor. The normal activity of this protein results in the activation of the genes preceded by the GRE (Glucocorticoid Response Element). When exposed to heat shock or other stress conditions, the NM domains start the prionic activity, eventually inhibiting the activation of transcription.
</P><P>This part was amplified by using the primers indicated by Li and Lindquist (2000), together with the sequence recommended to use for the ligation protocol with the plasmid pSB1C3. Those primers are: 
</P><UL><LI> Forward actagtagcggccgctgcagATGTCGGATTCAAAC 
</LI><LI> Reverse tctagaagcggccgcgaattcTCCTGCAGTGGCTTG 
</LI></UL><P>(again, capital letters represent the region that pairs with the coding sequence of NMGR<SUP>526</SUP>).
</P><P>The second part consists of the GRE followed by the reporter gene. In our experiments, we used LacZ for this purpose. The amplification of this part could not be made because of some problems found when trying to find the sequence of the GRE.
</P><H3><SPAN class="mw-headline" id="Sequence_2">Sequence</SPAN></H3><P>The sequence of NMGR<SUP>526</SUP> can be reconstructed from the separated parts that compose it. The NM domain sequence can be found in the SUP35/YDR172W first fragments. The GR<SUP>526</SUP> sequence comes from the nucleotides 14-2398 of the NM_012576 (GI:158303299) locus. The sequence of the part is shown below:
</P><PRE>   1 atgtcggattcaaaccaaggcaacaatcagcaaaactaccagcaatacagccagaacggt
  61 aaccaacaacaaggtaacaacagataccaaggttatcaagcttacaatgctcaagcccaa
 121 cctgcaggtgggtactaccaaaattaccaaggttattctgggtaccaacaaggtggctat
 181 caacagtacaatcccgacgccggttaccagcaacagtataatcctcaaggaggctatcaa
 241 cagtacaatcctcaaggcggttatcagcagcaattcaatccacaaggtggccgtggaaat
 301 tacaaaaacttcaactacaataacaatttgcaaggatatcaagctggtttccaaccacag
 361 tctcaaggtatgtctttgaacgactttcaaaagcaacaaaagcaggccgctcccaaacca
 421 aagaagactttgaagcttgtctccagttccggtatcaagttggccaatgctaccaagaag
 481 gttggcacaaaacctgccgaatctgataagaaagaggaagagaagtctgctgaaaccaaa
 541 gaaccaactaaagagccaacaaaggtcgaagaaccagttaaaaaggaggagaaaccagtc
 601 cagactgaagaaaagacggaggaaaaatcggaacttccaaaggtagaagaccttaaaatc
 661 tctgaatcaacacataataccaacaatgccaatgttaccagtgctgatgccttgatcaag
 721 gaacaggaagaagaagtggatgacgaagttgttaacgatatggactccaaagaatcctta
 781 gctccccctggtagagacgaagtccctggcagtttgcttggccaggggagggggagcgta
 841 atggacttttataaaagcctgaggggaggagctacagtcaaggtttctgcatcttcgccc
 901 tcagtggctgctgcttctcaggcagattccaagcagcagaggattctccttgatttctcg
 961 aaaggctccacaagcaatgtgcagcagcgacagcagcagcagcagcagcagcagcagcag
1021 cagcagcagcagcagcagcagcagccagacttatccaaagccgtttcactgtccatgggg
1081 ctgtatatgggagagacagaaacaaaagtgatggggaatgacttgggctacccacagcag
1141 ggccaacttggcctttcctctggggaaacagactttcggcttctggaagaaagcattgca
1201 aacctcaataggtcgaccagcgttccagagaaccccaagagttcaacgtctgcaactggg
1261 tgtgctaccccgacagagaaggagtttcccaaaactcactcggatgcatcttcagaacag
1341 aaaatcgaaaaagccagaccggcaccaacggaggcagtgtgaaattgtatcccacagacc
1401 aaagcacctttgacctcttgaaggatttggagttttccgctgggtccccaggtaaagaca
1461 caaacgagagtccctggagatcagacctgttgatagatgaaaacttgctttctcctttgg
1521 cgggagaagatgatccattccttctcgaaggggacacgaatgaggattgtaagcctctta
1581 ttttaccggacactaaacctaaaattaaggatactggagatacaatcttatcaagtccca
1641 gcagtgtggcactgccccaagtgaaaacagaaaaagatgatttcattgaactttgcaccc
1701 ccggggtaattaagcaagagaaactgggcccagtttattgtcaggcaagcttttctggga
1761 caaatataattggtaataaaatgtctgccatttctgttcatggtgtgagtacctctggag
1821 gacagatgtaccactatgacatgaatacagcatccctttctcagcagcaggatcagaagc
1881 ctgtttttaatgtcattccaccaattcctgttggttctgaaaactggaataggtgccaag
1941 gctccggagaggacagcctgacttccttgggggctctgaacttcccaggccggtcagtgt
2001 tttctaatgggtactcaagccctggaatgagaccagatgtaagctctcctccatccagct
2061 cgtcagcagccacgggaccacctcccaagctctgcctggtgtgctccgatgaagcttcag
2121 gatgtcattacggggtgctgacatgtggaagctgcaaagtattctttaaaagagcagtgg
2181 aaggacagcacaattacctttgtgctggaagaaacgattgcatcattgataaaattcgaa
2241 ggaaaaactgcccagcatgccgctatcggaaatgtcttcaggctggaatgaaccttgaag
2301 ctcgaaaaacaaagaaaaaaatcaaagggattcagcaagccactgcaggagtctca
</PRE><H3><SPAN class="mw-headline" id="Sup35p">Sup35p</SPAN></H3><P>[<I>PSI<SUP>+</SUP></I>] is a non-Mendelian trait of <I>Saccharomyces cerevisae</I> that supress nonsense codons.  This phenotype is due to a self-replication conformation (prion state) of a protein encoded by the gene Sup35. This protein, Sup35p, is the yeast eukariotyc release factor 3 (eRF3) and forms the translation termination complex with Sup45p (eRF1). The function of Sup35p is releasing the nascent polypeptide chain from the ribosome through GTP hydrolysis when Sup45p recognize a stop codon (Shorter and Lindquist, 2005).
</P><P>Sup35p is 685 amino acids long and has three distinct parts (Fig.2). The NH2-terminus (N) is termed the prion-forming domain (PrD) because plays a critical role in Sup35p’s changes in proteic conformation and it is responsible for its prion behaviour. This domain is 114 amino acids long and has a high content in glutamine and asparagine. The middle region (M) provides a solubilizing and/or spacing function. Finally, the COOH-terminus (C) is responsible for the translation-termination activity. 
</P><P><A href="/File:Valencia_prion_sup35.jpg" title="File:Valencia prion sup35.jpg">File:Valencia prion sup35.jpg</A></P><P>In [<I>PSI<SUP>+</SUP></I>] cells, most Sup35p is insoluble and nonfunctional, causing an increase in the translational read-through of stop codons. This trait is heritable because Sup35p in the amyloid state as every prion influences new Sup35p to adopt the same conformation and passes from mother cell to daughter. In [<I>psi<SUP>-</SUP></I>] cells, the translation-termination factor Sup35p is soluble and functional.
</P><H3><SPAN class="mw-headline" id="Behaviour">Behaviour</SPAN></H3><P>Sup35p is a subunit of the translation termination complex. Its prionic nature has been proposed to have some effect on the stress response, as a possible mechanism to obtain modified genetic expression products. When the prionic conformation is activated, the termination of translation is less effective and thus new longer proteins form (True and Lindquist, 2000; Tyedmers <I>et al.</I>, 2008). When the sequence corresponding to the NM domains of Sup35p is fused to other gene, the protein resulting of this construction acquires the prionic behaviour (Li and Lindquist, 2000).
</P><P>On the other side, GR activates the transcription of genes preceded by GRE (Glucocorticoid Receptor Element) when steroid hormones are present (Heitzer <I>et al.</I>, 2007). However, it becomes a constitutive transcription activator when it lacks its C terminal ligand-binding domain (Schena and Yamamoto, 1988). Because of the length of amino acids of the cut protein, this short version of GR is named GR<SUP>526</SUP>. 
</P><P>Li and Lindquist (2000) showed that the fused protein (NMGR<SUP>526</SUP>) is a functional constitutive transcription activator. In addition, when the prionic conformation is reached because of the presence of a certain stimulus, NMGR<SUP>526</SUP> is no longer capable of inducing the activation of the gene preceded by GRE. Tyedmers <I>et al.</I> (2000) checked the conditions that trigger the prionic conformation and they found that heat shock is a significantly relevant factor. The cells in which the prionic conformation is induced, the process is promoted in an autocatalytic manner and all the protein is found in the prionic conformation. The cells resulting show the phenotype [<I>PSI<SUP>+</SUP></I>].
</P><P>It is important to note that the rate of the change of conformation is not equal to zero even at optimal growth conditions, and that not all the cells become [<I>PSI<SUP>+</SUP></I>]. The rate of spontaneous activation of the switch is thought to be around 10<SUP>-6</SUP> or 10<SUP>-7</SUP> (Alberti <I>et al.</I>, 2009). This process is promoted under heat stress (Tyedmers <I>et al.</I>, 2008), probably because of the important role of heat shock proteins like Hsp104 in the formation and maintenance of the amyloid fiber (Halfmann <I>et al.</I>, 2009). These approximate rates will have very important implications for the yeastworld model that we briefly describe in the following subsection, and with more detail in our wiki <A href="https://2010.igem.org/Team:Valencia/Modeling" class="external text" rel="nofollow">Modeling</A> section.
</P><H3><SPAN class="mw-headline" id="Original_source_2">Original source</SPAN></H3><P>We thank D. Susan Lindquist (Whitehead Institute for Biomedical Research / Howard Hughes Medical Institute / Department of  Biology, MIT) for sending us the prionic system transformed E.coli strains.
</P><H2><SPAN class="mw-headline" id="Making_our_Biobricks">Making our Biobricks</SPAN></H2><P>After a lot of attempts trying to make our BioBricks (we had some dificulties at different steps) we achieved our purpose: we built 2 BioBricks  and we sent them to de Registry.
</P><H3><SPAN class="mw-headline" id="PCR">PCR</SPAN></H3><P>First of all, we needed enough amounts of DNA in order to insert them as our Biobrick into the pSB1C3 provided by the Registry.
We had purified DNA from PM2 E.coli strains and from NMGR526 E.coli strains. Using these as our DNA template we amplified the quoted genes: PM2 and NMGR526.
The chosen program was:
</P><UL><LI>A first denaturation cycle: 94ºC 3min
</LI><LI>30 amplification cycles, made up of: 
</LI><LI>94ºC 30s
</LI><LI>55ºC 1min
</LI><LI>72ºC 1min
</LI></UL><LI>Final step: 72ºC 7min
</LI><H3><SPAN class="mw-headline" id="Digestion">Digestion</SPAN></H3><P>Next step was the digestion of the amplified material with the enzimes EcoRI and PstI, keeping the reaction tubes during 4-5 hours at 37ºC with:
</P><P><B>PM2</B></P><UL><LI>1 μL PstI
</LI><LI>1μL EcoRI
</LI><LI>2,5 μL of insert PM2 (at 370,9 ng/ μL)
</LI><LI>4 μL H buffer
</LI><LI>31,5 μL water
</LI></UL><P><B>NMGR<SUP>526</SUP></B></P><UL><LI>1 μL PstI
</LI><LI>1 μL EcoRI
</LI><LI>1,8 μL of insert NMGR<SUP>526</SUP> (at 504,3 ng/ μL)
</LI><LI>4 μL H buffer
</LI><LI>32,2 μL water
</LI></UL><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><P>After purificating the DNA using a simple etanol precipitation protocol, we resuspended the DNA into 12 μL of water. Then, we put both plasmid and PM2 in one tube, and both plasmid and NMGR526 in another one, using these amounts of components:
</P><UL><LI>1 μL T4 ligase
</LI><LI>3 μL pSB1C3
</LI><LI>4 μL ligation buffer
</LI><LI>Whole 12 μL of our DNA
</LI></UL><P>We kept it O/N in the fridge at 4ºC.
</P><H3><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H3><P>We transformed DH5α <I>E.coli</I> competent strain with our ligation product using a standard heat shock transformation protocol.
</P><H3><SPAN class="mw-headline" id="Miniprep">Miniprep</SPAN></H3><P>Before the tranformation ending O/N step, we noticed that we had grown colonies in cloramphenicol LB plaques, so we used them to make a plasmidic DNA extraction (Miniprep: High Pure Plasmid Isolation Kit, Roche. 11 754 785 001).
In order to check the process had been successful, we made an electrophoresis gel, putting  four careers: 
</P><OL><LI>Ligation product digested with EcoRI 
</LI><LI>Ligation product digested with PstI
</LI><LI>Ligation product digested with EcoRI and PstI
</LI><LI>Ligation product without any restriction enzyme
</LI></OL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI>Alberti, S., Halfmann, R., King, O., Kapila, A., Lindquist, S. (2009). A systematic survey identifies prions and illuminates sequence features of prionogenic proteins. <I>Cell</I>. 137: 146-158.
</LI></UL><UL><LI>Cobb, N.J., Surewicz, W. K. (2009). Prion diseases and their biochemical mechanisms. <I>Biochemistry</I>. 48: 2574–2585.
</LI></UL><UL><LI>Cox,B.S. (1965). A cytoplasmic suppressor of super-suppressor in yeast. <I>Heredity</I>. 20: 505–521.
</LI></UL><UL><LI>Halfmann, R., Alberti, S., Lindquist, S. (2009). Prions, protein homeostasis and phenotypic diversity. <I>Trends in Cell Biology</I>. 20: 125-133.
</LI></UL><UL><LI>Heitzer, M.D., Wolf, I.M., Sanchez, E.R., Witchel, S.F., DeFranco, D.B. (2007). Glucocorticoid receptor phisiology. <I>Rev. Endocr. Metab. Disord.</I>. 8: 321-330.
</LI></UL><UL><LI>Li, L., Lindquist, S. (2000). Creating a protein-based element of inheritance. <I>Science</I>. 287: 661-664.
</LI></UL><UL><LI>Liu, Y., Zheng, Y. (2005). PM2, a group 3 LEA protein from soybean, and its 22-mer repeating region confer salt tolerance in Escherichia coli. <I>Biochem. Biophys. Res. Com.</I> 331: 325-332.
</LI></UL><UL><LI>Liu, Y., Zheng, Y., Zhang, Y., Wang, W., Li, R. (2010). Soybean PM2 protein (LEA3) confers the tolerance of <I>Escherichia col</I>i and stabilization of enzyme activity under diverse stresses. <I>Current Microbiology</I>. 60: 373-378.
</LI></UL><UL><LI>Prusiner, S.B. (1982). Novel proteinaceous infectious particles cause scrapie. Science. 216: 136–144.
</LI></UL><UL><LI>Prusiner, S.B. (1998). Prions. <I>PNAS</I>. 95: 13363–13383.
</LI></UL><UL><LI>Si, K., Lindquist, S. Kandel, E.R. (2003). A neuronal isoform of the Aplysia CPEB has prion-like properties. <I>Cell</I>. 115: 879-891.
</LI></UL><UL><LI>Schena, M., Yamamoto, K.R. (1988). Mammalian glucocorticoid receptor derivatives enhance transcription in yeast. <I>Science</I>. 241: 965-967.
</LI></UL><UL><LI>Shorter, J., Lindquist, S. (2005). Prions as adaptive conduits of memory and inheritance. <I>Nature</I>. 6: 435-450.
</LI></UL><UL><LI>True, H.L., Lindquist, S. (2000). A yeast prion provides a mechanism for genetic variation and phenotypic diversity. <I>Nature</I>. 407: 477-483.
</LI></UL><UL><LI>Tyedmers, J., Madariaga, M.L., Lindquist, S. (2008). Prion switching in response to environmental stress. <I>PLoS Biology</I>. 6: e294.
</LI></UL><UL><LI>Wickner, R.B. (1994). [URE3] as an altered <I>URE2</I> protein: evidence for a prion analog in <I>Saccharomyces cerevisiae</I>. <I>Science</I>. 264: 566-569.
</LI></UL><UL><LI>Wickner, R.B., Shewmaker, F., Kryndushkin, D., Edskes, H.K. (2008). Protein inheritance (prions) based on parallel in-register β-sheet amyloid structures. <I>BioEssays</I>. 30: 955-964.
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Valencia/Parts">http://2010.igem.org/Team:Valencia/Parts</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Valencia/Parts" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Valencia/Parts" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Valencia/Parts&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Valencia/Parts&amp;oldid=209551" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>