<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Valencia_prion"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Valencia/prion">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Valencia/prion&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia/prion&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia/prion&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Valencia/prion" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Valencia/prion</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><UL id="topnav"><LI><A href="https://2010.igem.org/Team:Valencia">Home</A><SPAN><A href="https://2010.igem.org/Team:Valencia">Mad yeasts on Mars?</A> |
            <A href="https://2010.igem.org/Team:Valencia#The Valencia IGEM Team 2010">The Valencia Team</A> |
            <A href="https://2010.igem.org/Team:Valencia#News">News</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:Valencia/Team">The Team</A><SPAN><A href="https://2010.igem.org/Team:Valencia/Team">The Team</A> |
            <A href="https://2010.igem.org/Team:Valencia/Universities">The Universities</A> |
            <A href="https://2010.igem.org/Team:Valencia/Akw">Acknowledgements</A> |
            <A href="https://2010.igem.org/Team:Valencia/Attributions">Attributions</A> |
            <A href="https://2010.igem.org/Team:Valencia/Gallery">Gallery</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:Valencia/Project">The Project</A><SPAN><A href="https://2010.igem.org/Team:Valencia/Project">Overview</A> |
            <A href="https://2010.igem.org/Team:Valencia/Terraforming">Terraforming</A> |
            <A href="https://2010.igem.org/Team:Valencia/Technologies">Creating Technologies</A> |
            <A href="https://2010.igem.org/Team:Valencia/lea">Surviving Mars</A> |
            <A href="https://2010.igem.org/Team:Valencia/prion">Regulating Mars temperature</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:Valencia/Parts">The Parts</A><SPAN><A href="https://2010.igem.org/Team:Valencia/Parts">Submitted Biobricks</A> |
            <A href="https://2010.igem.org/Team:Valencia/Parts#Making our Biobricks">Making our Biobricks</A> |
            <A href="https://2010.igem.org/Team:Valencia/Notebook/Calender">Notebook</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:Valencia/Modeling">Modeling</A><SPAN><A href="https://2010.igem.org/Team:Valencia/Modeling">Modeling Mad Yeast On Mars</A>|
            <A href="https://2010.igem.org/Team:Valencia/Modeling#Welcome to the Yeastworld: theoretical background ">Yeastworld: theoretical background</A>|
            <A href="https://2010.igem.org/Team:Valencia/Modeling#Yeastworld Mathematics">Yeastworld Mathematics</A></SPAN></LI><LI><A href="#">Human Practices</A><SPAN><A href="https://2010.igem.org/Team:Valencia/Ethics">Ethics in terraforming</A> |
            <A href="https://2010.igem.org/Team:Valencia/Safety">Safety</A></SPAN></LI><LI><A href="https://2010.igem.org/Team:Valencia/Achievements">Achievements</A></LI><LI><A href="https://2010.igem.org/Team:Valencia/Contact">Contact</A></LI></UL><DIV id="HomeRight"><P>
Time goes by... 



(El tiempo pasa...)

Follow us: 
</P><P>Our main sponsors:
</P><P>Our institutions:
</P><P>
Visitor location:
</P></DIV><DIV id="HomeCenter"><P><SMALL> » <A href="/Team:Valencia" title="Team:Valencia">Home</A> » <A href="/Team:Valencia/Project" title="Team:Valencia/Project"> The Project </A></SMALL></P><DIV id="Titulos"><P>Regulating Mars temperature 
using a prionic switch
</P></DIV><H2><SPAN class="mw-headline" id="Prions">Prions</SPAN></H2><P>In (<A href="#References">1982</A>) Stanley B. Prusiner created the term “prion” (or proteinacius infectious particle) to name the exclusively proteic infectious agent responsible of the transmissible spongiform encephalopathies (TSEs), a group of mammalian neurodegenerative disorders. According to the widely supported “protein-only” model, the prion mechanism of transmissibility arise from the ability of the prion form of the protein to promote the conformational change of the normal cellular form to the infectious prion forms (<A href="#References">Prusiner, 1998</A>). This results in an autocatalytic process that is represented in the animation <I>How the prion works?</I>. The infectious forms are mis-folded proteins that induce by polymerization the formation of an amyloid fold constituted by tightly packed beta sheets. These aggregates are insoluble fibrils that display resistance to proteolytic digestion and have affinity for aromatic dyes (<A href="#References">Cobb and Surewicz 2009</A>). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>How the prion works?</B> Schematic explanation on prionic systems.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Fungal_prions">Fungal prions</SPAN></H3><P>Sixteen years ago, Reed Wickner (<A href="#References">1994</A>) proposed the prion nature of Ure2, a protein involved in the nitrogen metabolism of the yeast <I>Saccharomyces cerevisae</I>, to explain the unusual dominant and cytoplasmatic inheritance of the phenotype [<I>URE3</I>] first described by Cox (<A href="#References">1965</A>). Other publications of a wide array of genetic and biochemical evidence have supported that the prionic behaviour is present in other yeast proteins such as Sup35, Rnq1 and Swi1 and in HET-s, a protein involved in the mechanism of genetic incompatibility between strains of <I>Podospora anserina</I> (<A href="#References">Shorter and Lindquist 2005</A>). 
</P><P>These prions can produce amyloid-like fibrils similar to those associated with the mammalian prions. The molecular architecture of these amyloids have been studied using solid-state NMR spectroscopy and it has been found that the fibrils formed by Ure2p, Rnq1, and Sup35 share a common parallel and in-register β-structure (Fig.1) (<A href="#References">Wickner <I>et al.</I> 2008</A>). However, these fungal prions have no sequence similarity with their mammalian counterparts (<A href="#References">Cobb and Surewicz 2009</A>). Besides, they are not generally pathogenic and might have a beneficial role providing an evolutionary advantage to their hosts (<A href="#References">Halfmann <I>et al.</I> 2009</A>). The prion domains of these fungal proteins have an exceptionally high content (~40%) of two polar amyloidogenic amino acids: glutamine and asparagine (<A href="#References">Cobb and Surewicz 2009</A>).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption"><B>Figure 1</B>. Amyloid fibrils of fungal prions. At the left, electron microscopy of amyloid fibres formed by Sup35p (Shorter and Lindquist 2005). At the right, parallel in-register  &amp;beta-sheet model of yeast prion amyloids [URE3] (Ure2), [PSI+] (Ure2p) and [PIN+] (Rnq1). The hydrogen bonds of the main chain run parallel to the filament axis, while the main chains themselves are perpendicular to the filament axis (Wickner et al 2008).</DIV></DIV></DIV></DIV><P>Fungal prions generally produce changes in phenotype that seem loss-of-function mutations, but with a non-Mendelian dominant inheritance. This abnormal genetic behaviour is due to the transference of the amyloidal fibers from mother cells to her daughters and mating partners. Once this prionic infective form is present, it catalyzes the change of conformation of all the proteins of the same type that were still in the functional non-prionic form. Therefore, this causes a dominant change in the phenotype that is perpetuated through generations (<A href="#References">Si <I>et al.</I> 2003</A>).
</P><H4><SPAN class="mw-headline" id="Sup35p">Sup35p</SPAN></H4><P>[<I>PSI<SUP>+</SUP></I>] is a non-Mendelian trait of <I>Saccharomyces cerevisae</I> that supress nonsense codons. This phenotype is due to a self-replication conformation (prion state) of a protein encoded by the gene Sup35. This protein, Sup35p, is the yeast eukaryotic release factor 3 (eRF3) and constitutes the translation termination complex together with the protein Sup45p (eRF1). Sup35p releases the nascent polypeptide chain from the ribosome through GTP hydrolysis when Sup45p recognize a stop codon (<A href="#References">Shorter and Tyadmers 2005</A>).
</P><P>Sup35p is 685 amino acids long and has three distinct parts (Fig.2) (<A href="#References">Wickner et al 2008</A>). The NH2-terminus (N) is termed the prion-forming domain (PrD) because plays a critical role in Sup35p’s changes in proteic conformation and it is responsible for its prion behaviour. This domain is 114 amino acids long and has a high content in glutamine and asparagine. The middle region (M) provides a solubilizing and/or spacing function. Finally, the COOH-terminus (C) is responsible for the translation-termination activity. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 2</B>. The prion domain of Sup35 is Q/N Rich and has the ability to propogate the corresponding prion in the absence of the rest of the molecule. Source: Wickner <I>et al</I>. 2008.</DIV></DIV></DIV></DIV><P>
In [<I>PSI<SUP>+</SUP></I>] cells, most Sup35p is insoluble and nonfunctional, causing an increase in the translational read-through of stop codons. This trait is heritable because Sup35p in the  amyloid state as every prion influences new Sup35p to adopt the same conformation and passes from mother cell to daughter. In [<I>psi<SUP>-</SUP></I>] cells, the translation-termination factor Sup35p is soluble and functional (<A href="#References">Li and Lindquist 2000</A>).
</P><H2><SPAN class="mw-headline" id="Prionic_switch">Prionic switch</SPAN></H2><H3><SPAN class="mw-headline" id="Components">Components</SPAN></H3><P>The switch is formed by two different parts: the activator and the reporter. The activator part is a construction of two fragments: the NM domains of the protein Sup35, which confers to the protein the prionic activity, and the GR<SUP>526</SUP> portion, which contains the DNA-binding and transcription-activation domains. The ligand-binding domain of the protein GR was eliminated, decoupling the response of the protein to the presence of glucocorticoids, and thus generating a constitutive transcription activation factor. The normal activity of this protein results in the activation of the genes preceded by the GRE (Glucocorticoid Response Element). When exposed to heat shock or other stress conditions, the NM domains start the prionic activity, eventually inhibiting the activation of transcription.
</P><P>This part was amplified by using the primers indicated by <A href="#References">Li and Lindquist 2000</A>, together with the sequence recommended to use for the ligation protocol with the plasmid pSB1C3. Those primers are: 
</P><UL><LI> Forward actagtagcggccgctgcagATGTCGGATTCAAAC 
</LI><LI> Reverse tctagaagcggccgcgaattcTCCTGCAGTGGCTTG 
</LI></UL><P>(again, capital letters represent the region that pairs with the coding sequence of NMGR<SUP>526</SUP>).
</P><P>The second part consists of the GRE followed by the reporter gene. In our experiments, we used LacZ for this purpose. The amplification of this part could not be made because of some problems found when trying to find the sequence of the GRE.
</P><H3><SPAN class="mw-headline" id="Behaviour">Behaviour</SPAN></H3><P>Sup35p is a subunit of the translation termination complex. Its prionic nature has been proposed to have some effect on the stress response, as a possible mechanism to obtain modified genetic expression products. When the prionic conformation is activated, the termination of translation is less effective and thus new longer proteins form (<A href="#References">True and Lindquist 2000</A>; <A href="#References">Tyedmers <I>et al.</I> 2008</A>). When the sequence corresponding to the NM domains of Sup35p is fused to other gene, the protein resulting of this construction acquires the prionic behaviour (<A href="#References">True and Lindquist 2000</A>).
</P><P>On the other side, GR (Glucocorticoid Receptor) activates the transcription of genes preceded by GRE (Glucocorticoid Receptor Element) when steroid hormones are present (<A href="#References">Heizer <I>et al.</I> 2007</A>). However, it becomes a constitutive transcription activator when it lacks its C terminal ligand-binding domain (<A href="#References">Schena and Yamamoto 1988</A>). Because of the length of amino acids of the cut protein, this short version of GR is named GR<SUP>526</SUP>. 
</P><P><A href="#References">Li and Lindquist 2000</A> showed that the fused protein (NMGR<SUP>526</SUP>) is a functional constitutive transcription activator. In addition, when the prionic conformation is reached because of the presence of a certain stimulus, NMGR<SUP>526</SUP> is no longer capable of inducing the activation of the gene preceded by GRE. <A href="#References">Tyedmers <I>et al.</I> 2000</A> checked the conditions that trigger the prionic conformation and they found that heat shock is a significantly relevant factor. The cells in which the prionic conformation is induced, the process is promoted in an autocatalytic manner and all the protein is found in the prionic conformation. The cells resulting show the phenotype [PSI+].
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>How the prionic switch works?</B> Explanation on how does the prionic switch fuction.</DIV></DIV></DIV></DIV><P>
It is important to note that the rate of the change of conformation is not equal to zero even at optimal growth conditions, and that not all the cells become [PSI+]. The rate of spontaneous activation of the switch is thought to be around 10<SUP>-6</SUP> or 10<SUP>-7</SUP> (<A href="#References">Alberti <I>et al.</I> 2009</A>). This process is promoted under heat stress (<A href="#References">Tyedmers <I>et al.</I> 2008</A>), probably because of the important role of heat shock proteins like Hsp104 in the formation and maintenance of the amyloid fiber (<A href="#References">Halfmann <I>et al.</I> 2009</A>). These approximate rates will have very important implications for the yeastworld model that we briefly describe in the following subsection, and with more detail in the <A href="/wiki/index.php?title=Team:Valencia/modelling&amp;action=edit&amp;redlink=1" class="new" title="Team:Valencia/modelling (page does not exist)"> Modelling</A> section.
</P><H2><SPAN class="mw-headline" id="Introduction_to_the_Yeastworld">Introduction to the Yeastworld</SPAN></H2><P>A model population of yeast containing the prionic switch growing under low temperatures would contain a pool of [psi -] (black) cells and a minimal amount of [PSI+] (white) cells. The high proportion of black yeasts will cause an increase of the planet mean temperature (as can be deduced by the <A href="/Team:Valencia/WB" title="Team:Valencia/WB"> Microbial Albedo Recorder</A> section of the project). As we explained in the section above, when the temperature is high enough, the prionic switch will tend to become activated. Then, there will be a much higher proportion of non-melanic [PSI+] cells. In addition to the underlying molecular tendency, the black cells will be selected against in this situation, because of the increasing of their inner temperature, in comparison to the white cells, that will provoke a reduction of their growth rate. The white cells, on the other hand, have an intrinsic protection against overheating due to their higher albedo. Thus, these cells will grow faster and become more and more abundant. The eventual high proportion of white cells will lead to an increase of the planetary albedo effect and a decrease of the mean temperature. This situation will become autoregulated and reach an equilibrium through several oscillatory cycles. The background and the dynamics of this process can be regarded with much more detail in the <A href="/wiki/index.php?title=Team:Valencia/Modelling&amp;action=edit&amp;redlink=1" class="new" title="Team:Valencia/Modelling (page does not exist)"> Modelling</A> section of the project, but the conclusion is quite remarkable: a pleasant equilibrium temperature of about 30ºC.
</P><H2><SPAN class="mw-headline" id="Experimental_procedure">Experimental procedure</SPAN></H2><H3><SPAN class="mw-headline" id="Plasmids_and_strains">Plasmids and strains</SPAN></H3><P>As we mentioned in the <A href="/Team:Valencia/prion#Components" title="Team:Valencia/prion"> Components</A> subsection, the switch is formed by two different parts, which were handled independently. The prionic transcription activator was inserted into a multicopy (2μ) constitutive vector. The construct is called PG1-NMGR<SUP>526</SUP> (<B>Fig 1:</B><A href="#References">Schena and Yamamoto 2008</A>; see its card in <A href="http://www.addgene.org/pgvec1?f=c&amp;cmd=findpl&amp;identifier=1122&amp;attag=n" class="external text" rel="nofollow">Addgene</A>).
</P><P>The GRE+reporter portion was cloned into another multicopy vector, naming the construct pL2/GZ (<B>Fig 2:</B>; <A href="#References">Kimura <I>et al.</I> 2008</A>; see its card in <A href="http://www.addgene.org/pgvec1?f=c&amp;cmd=findpl&amp;identifier=1282&amp;attag=n" class="external text" rel="nofollow">Addgene</A>).
</P><P>The plasmids were cloned into chemically competent BL21 Star <I>E. coli</I> cultures (read more about this bacterial strain in its <A href="http://tools.invitrogen.com/content/sfs/manuals/oneshotbl21star_man.pdf" class="external text" rel="nofollow">Invitrogen</A> manual). The cultures were grown overnight at 37ºC and then the plasmids were isolated. Finally, yeast strain 5523 ([<I>psi<SUP>-</SUP></I>]) was transformed using the lithium acetate protocol we describe in XXXXXX. The yeast strain 5523 has several auxotrophies (genotype <I>ade1-14(UGA), trp1-289(UAG), his3Δ-200, ura3-52, leu2-3,112</I>) and the NM domains of Sup35 deleted from the background 74D-694 [<I>PSI<SUP>+</SUP></I>] (read more about the obtaining of this strain in <A href="#References">True and Lindquist 2000</A>).
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>We succeeded to transform the yeast strain 5523 with the two plasmids: PG1-NMGR<SUP>526</SUP> and pL2/GZ. In the Fig.3 we demonstrate the obtaining of this double transformation, where it can be seen that this strain was the only one that could grow on a selective medium, compared to others that did not possessed both plasmids.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 3</B>. Transformation results. The three dishes at the top of the picture contain a rich (YPD) medium. The three dishes at the bottom contain a selective (SD+Ade+His+Ura) medium. The yeast strains growing on ech plate are (A and D) 5523 with no plamids, (B and E) 5523 with only the pL2/GZ plasmid and (C and F) 5523 with the complete prionic system. Because the plasmids confer the ability to overcome the triptophan (pG1-NMGR<SUP>526</SUP>) and leucine (pL2/GZ) auxotrophies, only the prionic system-containing yeasts could grow on the selective medium. The rich medium impose no restriction for the growth of the three strains.</DIV></DIV></DIV></DIV><P>Where growing in presence of X-gal, the proper substrate for the beta-galactosidase enzyme (our reporter gene for the behaviour of the prionic switch), we obtained both blue and white colonies. These colonies can be identified as the [<I>psi<SUP>-</SUP></I>] and [<I>PSI<SUP>+</SUP></I>] phenotypes. In contrast with the results of Li and Lindquist (2000) or Tyedmers <I>et al.</I> (2008), the proportion of white yeasts growing in our plates was unexpectedly high. Further optimization of the experimental manipulation would be needed for the proper characterization of the system.
</P><P>Nevertheless, the presence of both blue and white colonies, in relation with the results explained in the <A href="/Team:Valencia/WB" title="Team:Valencia/WB"> Microbial Albedo Recorder</A> section, would be sufficient to recall the properties of the system at the population level. Then, the Yeastworld model (described above and, in more detail, in the <A href="/Team:Valencia/Modeling" title="Team:Valencia/Modeling"> Modeling</A> section) could depict the evolution and fate of a population of yeast containing the prionic system.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI>Alberti, S., Halfmann, R., King, O., Kapila, A., Lindquist, S. (2009). A systematic survey identifies prions and illuminates sequence features of prionogenic proteins. <I>Cell</I>. 137: 146-158.
</LI></UL><UL><LI>Cobb, N.J., Surewicz, W. K. (2009). Prion diseases and their biochemical mechanisms. <I>Biochemistry</I>. 48: 2574–2585.
</LI></UL><UL><LI>Cox,B.S. (1965). A cytoplasmic suppressor of super-suppressor in yeast. <I>Heredity</I>. 20: 505–521.
</LI></UL><UL><LI>Halfmann, R., Alberti, S., Lindquist, S. (2009). Prions, protein homeostasis and phenotypic diversity. <I>Trends in Cell Biology</I>. 20: 125-133.
</LI></UL><UL><LI>Heitzer, M.D., Wolf, I.M., Sanchez, E.R., Witchel, S.F., DeFranco, D.B. (2007). Glucocorticoid receptor phisiology. <I>Rev. Endocr. Metab. Disord.</I>. 8: 321-330.
</LI></UL><UL><LI>Li, L., Lindquist, S. (2000). Creating a protein-based element of inheritance. <I>Science</I>. 287: 661-664.
</LI></UL><UL><LI>Prusiner, S.B. (1982). Novel proteinaceous infectious particles cause scrapie. Science. 216: 136–144.
</LI></UL><UL><LI>Prusiner, S.B. (1998). Prions. <I>PNAS</I>. 95: 13363–13383.
</LI></UL><UL><LI>Si, K., Lindquist, S. Kandel, E.R. (2003). A neuronal isoform of the Aplysia CPEB has prion-like properties. <I>Cell</I>. 115: 879-891.
</LI></UL><UL><LI>Schena, M., Yamamoto, K.R. (1988). Mammalian glucocorticoid receptor derivatives enhance transcription in yeast. <I>Science</I>. 241: 965-967.
</LI></UL><UL><LI>Shorter, J., Lindquist, S. (2005). Prions as adaptive conduits of memory and inheritance. <I>Nature</I>. 6: 435-450.
</LI></UL><UL><LI>True, H.L., Lindquist, S. (2000). A yeast prion provides a mechanism for genetic variation and phenotypic diversity. <I>Nature</I>. 407: 477-483.
</LI></UL><UL><LI>Tyedmers, J., Madariaga, M.L., Lindquist, S. (2008). Prion switching in response to environmental stress. <I>PLoS Biology</I>. 6: e294.
</LI></UL><UL><LI>Wickner, R.B. (1994). [URE3] as an altered <I>URE2</I> protein: evidence for a prion analog in <I>Saccharomyces cerevisiae</I>. <I>Science</I>. 264: 566-569.
</LI></UL><UL><LI>Wickner, R.B., Shewmaker, F., Kryndushkin, D., Edskes, H.K. (2008). Protein inheritance (prions) based on parallel in-register β-sheet amyloid structures. <I>BioEssays</I>. 30: 955-964.
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Valencia/prion">http://2010.igem.org/Team:Valencia/prion</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Valencia/prion" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Valencia/prion" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Valencia/prion&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Valencia/prion&amp;oldid=209053" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>