Team:SPSingapore/home
Classic parallax page
Home
Team
Project
Protocol
Part
Notebook
Human Practice
Safety
Requirement
Home
Kenneth Lim Kun Ming
Research Interest: Bioinformatics, Genetic analysis Random fact: Is apparently Schrˆdinger's Biologist
Clarice Hong Kit Yee
Research Interest: genetics, RNA, cancer Past research projects: (can't rmb, tell you later)
Current project: Differential roles of SALL4A and SALL4B in HCC
Wong Chi Yan
Research Interest: Microbiology, molecular biology, proteomics
Past research project: Genetic studies on Salmonella biofilms
Current project: Role of fumarase and cysteine dehydrogenase in DNA damage response
Random fact: Likes statistics and playing volleyball :)
Yeo Xin Yi
Research Interests: Neurobiology, Neurosciences
Past research projects: Role of STAT in neuroinflammation and the pathogenesis of Alzheimer's Disease
Current research projects: Synaptic plasticity threshold in hippocampal CA1 pyramidal neurons, Role of WNK1 in neuronal survival and development
Random fact: Blah ~
Tan Yi Han
Research Interests: Pathogens, Immunology, Synthetic biology
Past research projects: Genetic studies on plant pathogenic fungi, Drug screening for Acute Lymphoblastic Leukemia
Current project: Characterisation of Klebsiella pneumoniae isolates from liver abscess
Random fact: Knits and bakes in spare time. =)
Hee Yanting
Research Interests: RNA, genomics, epigenetics
Past research projects: microRNA as a potential therapeutic strategy for colorectal cancer, Characterising LPA1 antagonists using calcium imaging
Current project: The role and targeting of EZH2 in lymphoma
Random fact: Plays the erhu and self-learning cello and classical guitar
Adrian Tan Hong Ji
Research Interest: Genetic Engineering, Cancer, Immunology
Random fact: 500 Hours in Terraria
Soong Yun Ting
Research Interests: Proteomics, Past research projects: Genetic studies on Salmonella biofilms
Current project: Identification of protein players in metastasis
Random fact: Plays the harmonica
Nguyen Duy
Research Interest: Pharmacoinfomatics, Bioinformatics, Neurosciences
Past research project: Genetic linkage analysis of asthma
Current project: Neurodegeneration of Drosophila Melanogaster.
Random fact: 0 hour in Terraria
Leslie Gapter
Dr. Leslie is trained as a molecular biologist and her dissertation focused on breast development and tumorigenesis. Leslie joined NUS in 2005 and her past research has focused on analyzing botanical products for breast and prostate cancer treatment.
In 2008, Leslie became a full time scientific writer at the Mechanobiology Institute, Singapore, before moving into her current position as a Lecturer in 2010. Leslie teaches 'The Cell' module, which examines the universal mechanics and functions of cells from an integrated science perspective, for the Special Program in Science.
Linda J Kenney
Dr Kenney is a Professor of Microbiology at the University of Illinois-Chicago. Her laboratory studies two-component systems in bacteria that control gene expression at a single cell and nanometer level.
Stuti Desai
She joined the Kenney group in May, 2012 with a strong urge to amalgamate her doctoral training in studying silent genetic systems in enterics to decipher the behavior of bacteria under various environmental challenges. She obtained her doctorate from the Indian Institute of Science, Bangalore, India, under the guidance of Prof Subramony Mahadevan. She studied Biochemistry for my Master's degree and Chemistry, Physics and Zoology for her Bachelor's degree at the Maharaja Sayajirao University of Baroda, Baroda, India.
Introduction
For description
Esa Quorum sensing
Invasin + lysteriolysin
FNR hypoxic promoter
Design
For design
Restriction Digest with NEB enzymes
PCR Purification
Transfection-Expression
Cloning
Cell Number and Viability Determination
Freezing down cells
Making Bacterial Media
Miniprep of plasmid DNA
Subculturing for adherent cell lines
Ligation Protocol with T4 DNA ligase
Antibiotic Preparation
Agarose Electrophoresis
Link to protocol
Restriction Digest with NEB enzymes
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Adjust Water bath to 37 degrees.
Electric shock, heat shock
Possibility of electric shock
wear proper PPE (gloves, lab coat, covered shoes)
1
1
1
 
 
 
2
Add aliquots of DNA to water and Cut Smart buffer in a 1.5mL centrifuge tube.
Biological exposure
Spillage of buffer and DNA
wear proper PPE (gloves, lab coat, covered shoes)
1
1
1
 
 
 
3
Lightly centrifuge the centrifuge tube in a microcentrifuge. Add an appropriate volume of NEB restriction enzymes and centrifuge again.
Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers
Injury due to imbalanced centrifuge and trapping of limbs or fingers
Internal training is compulsory.wear proper PPE (gloves, lab coat, covered shoes); handle cells in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand. When using the centrifuge, ensure centrifuge is balanced and rotor is placed correctly, and that all tubes are capped tightly. Close centrifuge properly and ensure that there are no funny sounds when centrifuge is running.
1
1
1
 
 
 
4
Place 1.5mL tube into waterbath and digest for 2 hours at 37 degrees. When Restriction digest is done, take thetube out and stop reaction by adding Purple loading dye and cooling on ice.
Electric shock, heat shock
Spillage of buffer and DNA
wear proper PPE (gloves, lab coat, covered shoes)
1
1
1
 
 
Link to protocol
PCR Purification
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Adjust Water bath to 55 degrees to preheat.
Electric shock, heat shock
Possibility of electric shock
wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day.
1
1
1
 
 
 
2
Excise gel slices with DNA of the correct band into a 1.5mL centrufuge tube, add an appropriate volume of DNA binding buffer and melt the gel slice by incubation in the water bath at 55 degrees. Add a equal volume of DNA binding buffer to a completed pcr reation.
Biological exposure
Spillage of buffer and DNA
wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day.
1
1
1
 
 
 
3
Add the melted agarose in DNA binding buffer or pcr reaction and DNA binding buffer to DNA bidning columns resting in 1.5mL tubes. Centrifuge the tube in a microcentrifuge and allow the DNA mixtiure to flow through.
Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers
Injury due to imbalanced centrifuge and trapping of limbs or fingers
wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.
1
1
1
 
 
 
4
Add wash buffer to wash through the colomns and precipitate DNA. Centrifuge for 1 min, then pour out the remaining wash buffer in the tube. Centrifuge for a further 5 minutes to dry the coloumn.
Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers
Injury due to imbalanced centrifuge and trapping of limbs or fingers
wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.
1
1
1
 
 
 
5
Add 30-50ul of nuclease free water to the column, and incubate the columns in the 55 degrees water bath for 5 minutes. Spin down the DNA and quantify concentration using the Thermo Scientific nanodrop.
Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers
Injury due to imbalanced centrifuge and trapping of limbs or fingers
wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.
1
1
1
 
 
Link to protocol
TransfectionExpression
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Transferring aliquot of competent E.coli (dH5a or BL21 from Life Technologies) from  -80 C to thaw on ice with an appropriate aliquot of plasmid (100ng) for 30min.
Skin contact with extremely cold materials; biological exposure
Cold burn injury; handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested
Internal training is compulsory. Handle materials from -80 C using insulated / thermal gloves (or double layer latex gloves); wear proper PPE (safety goggles, lab coat, covered shoes); transfer samples in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.
2
1
1
 
 
 
2
Adjust water bath to 42 degrees.
Electric shock, heat burns
Possibility of electric shock
wear proper PPE (gloves, lab coat, covered shoes)/
1
1
1
 
 
 
3
Heatshock bacteria for 30 seconds exactly at 42degrees
Biological exposure, Electric shock, heat burns
Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested
Internal training is compulsory. Conduct routine bacteria work in a Class I Biohazard Safety Cabinet. Wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90. Have training on how to manage biological spill with appropriate spill kit reagetns (10% bleach or 70% Ethanol)
1
2
2
 
 
 
4
Reviive bacteria at 37degrees with shaking for 1 hour.
Biological exposure to bacteria
Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested
Internal training is compulsory. Conduct routine bacteria work in a Class I Biohazard Safety Cabinet. Wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90. Have training on how to manage biological spill with appropriate spill kit reagetns (10% bleach or 70% Ethanol)
1
2
2
 
 
 
5
Plate bacteria on LB+ appropriate antibiotic for selection, grow overnight in a small bacteria incubator.
Biological exposure to bacteria
Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested
Internal training is compulsory. Conduct routine bacteria work in a Class I Biohazard Safety Cabinet. Wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90. Have training on how to manage biological spill with appropriate spill kit reagetns (10% bleach or 70% Ethanol)
1
2
2
 
 
Link to protocol
Cloning
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Transferring bacteria from glycerol stock stored @ -80 C to plastic culture tubes
Skin contact with extremely cold materials; biological exposure
Cold burn injury; handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested
Internal training is compulsory. Handle materials from -80 C using insulated / thermal gloves (or double layer latex gloves); wear proper PPE (safety goggles, lab coat, covered shoes); transfer samples in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.
2
1
1
 
 
 
2
Transferring culture samples to glassware
Breakage of glassware; biological exposure
Injury from broken glassware; handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested
Internal training is compulsory. Handle glassware over the lab bench (or in the biosafety cabinet); wear proper PPE (gloves, lab coat, covered shoes); transfer samples in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.
1
2
1
 
 
 
3
Routine maintenance of bacteria culture
Biological exposure, breakage of glassware
Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested
Internal training is compulsory. Conduct routine bacteria work in a Class I Biohazard Safety Cabinet. Wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90.
1
2
2
 
 
 
4
Single day experiements using bacteria culture (students)
Biological exposure; breakage of glassware
Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested
Internal training is compulsory. Conduct routine bacteria work at the lab bench over absorbant materials; wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90.
1
2
2
 
 
 
5
 
 
 
 
 
 
0
 
 
Link to protocol
Cell Number and Viability Determination
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Cleaning the glass slide with detergent.
1) Breakage of the glass slide; 2)naturally sharp edges on the glass slide.
Injury from broken slide or sharp edges
Handle glassware over the lab bench; wear proper PPE (gloves, lab coat, covered shoes); the usual precautions outlined in the "Use of Laboratory Glassware" SOP must be taken.
1
1
1
 
 
 
2
Transferring sample to the glass slide.
Breakage of glass slide.
Injury from broken slide.
Handle glassware over the lab bench; wear proper PPE; the usual precautions outlined in the "Use of Laboratory Glassware" SOP must be taken.
1
1
1
 
 
 
3
Overlay sample with a glass cover slip.
Dropping the cover slip onto the bench top or floor.
Breaking the glass coverslip when retrieving it & subsequent injury.
Wear proper PPE; use a brush and dustpan to retrieve the glass coverslip (do not attempt to use the coverslip- discard it into the glass waste).
1
2
2
 
 
 
4
Transferring the glass slide to/from the microscope for viewing.
1) finger pinched (or cut) by the stage clip; 2) breakage of glass slide
Injury to fingers (pinch or cut).
1) Use both hands to position the slide, wear proper PPE; 2) handle glassware over the lab bench and use the precautions as outlined in the "Use of Laboratory Glassware" SOP.
1
1
1
 
 
 
5
Looking through the eye piece to observe samples.
1) Accidental knocking of the one's head onto the eye piece; 2) backlight too intense
Injury of the eye.
1) Position hands around the eye pieces to help serve as a guide for lowering your head and eyes; 2) before viewing the sample, reduce the backlight to zero, then look into the eyepieces and raise the light intensity.
2
1
2
 
 
 
6
Leaving the backlight on to view the sample.
Overheating of microscope parts due to the backlight.
Slight burns upon skin contact with the heated light element.
Switch off backlights whenever not in use.
2
1
2
 
 
 
7
(Optional) Viewing fluorescence using the mercury lamp (aka Intensilight)
Physical hazard- Mercury lamps release extremely potent and visible UV radiation.
Injury of the eye and skin.
Avoid looking at the microscope stage and slide directly- always view them through the UV light shield; note the number of hours the mercury lamp has been used (shown on the unit itself)- going beyond its expected lifespan (~2000 hrs) is an increased fire risk (see Nikon Intensilight manual).
2
1
2
 
 
Link to protocol
Freezing down cells
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Grow your cells in a large plate (150 mm) to a confluent monolayer.
Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,
Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury
All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.
1
1
2
 
 
 
2
Prepare 10 ml of cryo media. This is just 95% FBS and 5% DMSO. In a 15 ml tube, add 10 ml FBS and 0.555 ml DMSO. Mix well.
Spillage and exposure to DMSO which is an irritant
Eye contact with DMSO may cause blurry vision, burning sensation, redness, tearing, and vasodilation.
Wear proper PPE (goggles, gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; dispose of any spillage using the Chemical Spill SOPs.
1
1
2
 
 
 
3
You will need to wash with ~20 ml of media, and use 5 ml of trypsin (3 ml media, 2 ml trypsin). Collect the trypsin/cells into a 50 ml tube (instead of 15 ml, using 5 ml serological pipette), and wash the plate with 20 ml of serum-containing media. After you remove the media from your cells, resuspend the pellet in the 10 ml of cryo media. Aliquot 1 ml of this into each of the 10 cryotubes.
Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens, injury due to improper usage of centrifuge and fingers
Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury. Injury due to imbalanced centrifuge and trapping of limbs or fingers
Internal training is compulsory for centrifuge use and the centrifuge key.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand. When using the centrifuge, ensure centrifuge is balanced and rotor is placed correctly, and that all tubes are capped tightly. Close centrifuge properly and ensure that there are no funny sounds when centrifuge is running. Decontaminate waste using Presept tablets (dilute to 10% solution of activaed bleach to kill cells) and let the bleach decontaminate for half an hour before discarding with plenty of water and dilution.
1
1
2
 
 
 
4
Put the tubes into the Mr. Frosty and place in the -80ºC freezer. Keep in the freezer for 24 – 48 hours.
Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,
Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury
All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.
1
1
2
 
 
Link to protocol
Making Bacterial Media
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Weigh out Lennox Broth/Miller Broth solid: 15g/L, add 10g/L agar for (1%) agar. (From scratch, weigh 1% peptone, 0.5% yeast extract, and 1% NaCl. ( NaCl 10 g/L, Peptone 10 g/L,  Yeast extract 5 g/L) LB broth suggested to be pH 7-7.2 (adjust with 1N NaOH). ). Add Bacterial agarose for 10g/L agar (1%).
Spillage of media
Accidental inhalation or ingestion of media powder.
wear proper PPE (gloves, lab coat, covered shoes).
1
1
1
 
 
 
2
Add Milliq water to media bottles. Cap tightly and shake throughly, making sure that the solid is all well distributed in solution. Loosen caps by a full turn and place a piece of dog-eared autoclave tape on the bottle. Never autoclave flammable liquids or tightly capped bottles.
Spillage of media
Accidental inhalation or ingestion of media powder.
wear proper PPE (gloves, lab coat, covered shoes).
1
1
1
 
 
 
3
Turn on the autoclave. Ensure that enough water is in the bottle, and in the bottom of autoclave, drain bottle empty, drain valve closed. Top up with Milliq water as required.Turn the autoclave machine to agar sterilisation mode, place balanced autoclave baskets of media bottles in the autoclave. Do not close the autoclave machine by hand. Use machine controls to close machine and start heating cycle.
Electric shock, heat burns, explosion
Heat or electrical burns from autoclave.
wear proper PPE (gloves, lab coat, covered shoes).
1
1
1
 
 
 
3
After the autoclave cycle is over and has cooled down, press the open button on the machine and let machine open and cool for 15 minutes. Using heatproof gloves, lift the autoclave basket onto trolley and push trolley into main lab area. Allow the agar to cool overnight before capping tightly and sealing with parafilm.
Electric shock, heat burns from steam
Heat burns from hot steam
wear proper PPE (goggles, gloves, lab coat, covered shoes). Stand back when autoclave opens and allow steam to escape and machine to cool for 15min before removing baskets. Use heatproof gloves to handle autoclave baskets.
1
1
1
 
 
Link to protocol
Miniprep of Plasmid DNA
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Inoculate bateria in 3mL of LB+antibiotic to grow for 16 hours at 37 degrees in a shaking incubator.
Biological exposure, breakage of glassware
Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested
Internal training is compulsory. Conduct routine bacteria work in a Class I Biohazard Safety Cabinet. Wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90.
1
1
1
 
 
 
2
Pellet Bacteria in 1.5mL tubes in a microcentrifuge at maximum speed for 5 minutes. Pour away the LB media into a waste containter containing 10% bleach.
Biological exposure
Spillage of bacteria culture.
wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day.
1
1
1
 
 
 
3
Resuspend the bacterial cells in Resuspension solution, vortexing lightly to ensure pellet is fully resuspended in solution. Add Lysis buffer and incubate till solution is clear and there is no cloudy bacteria for 2 minutes. Add Neutralisation solution and invert 4-6 times to mix immediately.
Biological exposure
Spillage of bacteria culture, and buffers
wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day.
1
1
1
 
 
 
4
Pellet the lysed bacteria by centrifugation for  5 minutes. Run the flow through in DNA binding columns.
Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers
Injury due to imbalanced centrifuge and trapping of limbs or fingers
wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.
1
1
1
 
 
 
5
Add wash buffer to wash through the colomns and precipitate DNA. Centrifuge for 1 min, then pour out the remaining wash buffer in the tube. Centrifuge for a further 2 minutes to dry the coloumn.
Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers
Injury due to imbalanced centrifuge and trapping of limbs or fingers
wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.
1
1
1
 
 
 
6
Add 30-50ul of nuclease free water to the column. Spin down the DNA and quantify concentration using the Thermo Scientific nanodrop.
Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers
Injury due to imbalanced centrifuge and trapping of limbs or fingers
wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.
1
1
1
 
 
Link to protocol
Subculturing for adherent cell lines
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Remove and discard old media. Avoid disturbing the adhered cells.
Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,
Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury
All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.
1
1
2
 
 
 
2
Wash flask/dish with 10 ml of DMEM (-), then remove. Avoid washing off cells.
Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,
Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury
All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.
1
1
2
 
 
 
3
Trypsinize the cells by adding 2 ml of DMEM (-) and 1ml of trypsin. Swirl the flask/dish, then incubate for 4 – 7 minutes.
Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,
Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury
All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.
1
1
2
 
 
 
4
Add 10 ml of DMEM (+) to stop the action of trypsin, then wash the flask/dish to wash off more cells. Transfer the cells into a 15 ml Falcon tube.
Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,
Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury
All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.
1
1
2
 
 
 
5
Spin down cells by centrifuging at room temperature (25oC), 800 rpm for 5 minutes. After centrifuging, pour out the media without disturbing the cell pellet. Resuspend cell pellet with 1 ml DMEM (+) by pipetting up and down at least 10 – 15 times. Split cells into new culture dish/flask at the desired and recommended density (usually ~30-40% for cell maintenance).
Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens, injury due to improper usage of centrifuge and fingers
Injury due to imbalanced centrifuge and trapping of limbs or fingers
Internal training is compulsory for centrifuge use and the centrifuge key.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand. When using the centrifuge, ensure centrifuge is balanced and rotor is placed correctly, and that all tubes are capped tightly. Close centrifuge properly and ensure that there are no funny sounds when centrifuge is running. Decontaminate waste using Presept tablets (dilute to 10% solution of activaed bleach to kill cells) and let the bleach decontaminate for half an hour before discarding with plenty of water and dilution.
1
1
2
 
 
Link to protocol
Ligation Protocol with T4 DNA ligase
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Adjust Thermocycler to 16 degrees hold.
Electric shock
Possibility of electric shock
wear proper PPE (gloves, lab coat, covered shoes)/
1
1
1
 
 
 
2
Add aliquots of cut vector and insert to water and T4 ligase buffer in a 0.2mL pcr tube.
Biological exposure
Spillage of buffer and DNA
wear proper PPE (gloves, lab coat, covered shoes)/
1
1
1
 
 
 
3
Lightly centrifuge the pcr tube in a microcentrifuge. Add an appropriate volume of ligase and centrifuge again.
Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers
Injury due to imbalanced centrifuge and trapping of limbs or fingers
Internal training is compulsory.wear proper PPE (gloves, lab coat, covered shoes); handle cells in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand. When using the centrifuge, ensure centrifuge is balanced and rotor is placed correctly, and that all tubes are capped tightly. Close centrifuge properly and ensure that there are no funny sounds when centrifuge is running.
1
2
2
 
 
 
4
Place pcr tube into thermocycler and ligate overnight at 16 degrees.
Biological exposure
Spillage of buffer and DNA
wear proper PPE (gloves, lab coat, covered shoes)/
1
2
2
 
 
Link to protocol
Antibiotic Preparation
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Weigh out antibiotics in a blue falcon tube. Prepare atibotic concentrations to 100ug/mL.
Spillage of antibiotics
Accidental inhalation or ingestion of antibiotics
wear proper PPE (gloves, lab coat, covered shoes).
1
1
1
 
 
 
2
Add Milliq water to blue falcon tube. Vortex and allow to incubate at room temperature till antibiotics are dissolved.
Spillage of antibiotics
Accidental inhalation or ingestion of antibiotics
wear proper PPE (gloves, lab coat, covered shoes).
1
1
1
 
 
 
3
Filter the antibiotics through a 0.2um filter using a sterile syringe in the bacterial BSC. Prepare aliquots of 250ul or less of antibiotics as working stock for concentration saccording to the antibiotic concentrations listed in the recommended amounts  and store in antibiotics drawer in the -20 fridge.
Spillage of antibiotics
Accidental inhalation or ingestion of antibiotics
wear proper PPE (gloves, lab coat, covered shoes).
1
1
1
 
 
Risk assessment
No
Desription/Details of Steps in Activity
Hazards
Possible Accident / Ill Health & Persons-at-Risk
Existing Risk Control (Mitigation)
Severity
Likelihood (Probability)
Risk Level
Additional Risk Control
Person Responsible
By (Date)
1
Heat agarose powder in TAE buffer until dissolved. Cool agarose to 50 degrees and add Sybr Safe DNA stain.
Chemical hazard- Sybr safe is an intercalating dye but unable to diffuse across cell membranes
Spillage of heated agarose
A work bench is specially allocated for DNA gel electrophoresis. Sybr Safe is used as it is not as toxic as Etbr. Wear appropriate PPE (nitrile gloves, lab coat, mask). Heat agarose slowly to avoid boiling and spillage out of the flask. Sybr safe solution is added to the flask in a fume hood after the agarose has cooled to reduce the chance of producing vapors.
2
1
2
 
 
 
2
Pour the agarose into the casting tray and allow the gel to solidify over 30 minutes.
Chemical hazard- Sybr safe is an intercalating dye but unable to diffuse across cell membranes
Spillage of heated agarose
Wear appropriate PPE (nitrile gloves, lab coat, mask). Casting tray is contained in appropriate trays to contain any accidental spillage. Handle hot flask of agarose with a heatprooof glove.
2
1
2
 
 
 
3
Remove gel tray and fillthe electrophoresis apparatus with TAE buffer. Load the DNA sample and resolve through the gel
Chemical hazard- Sybr safe is an intercalating dye but unable to diffuse across cell membranes; electrical hazard
Electric shock may occur if connecting and disconnecting the apparatus without powering off the machine. Accidental physical contact with Sybr Safe containing agarose gel
Wear appropriate PPE (nitrile gloves, lab coat). All procedures are performed with electrophoresis equipment contained within appropriate trays to contain any accidental spillage.  Connect and disconnect the apparatus only when it is powered off. Do not touch the apparatus while the gel is running.
2
1
2
 
 
 
5
Visualize the gel under UV light. Discard gel for incineration at medical waste disposal plant.
Chemical hazard- Sybr safe is an intercalating dye but unable to diffuse across cell membranes; biological hazard- UV light is mutagenic
Electric shock may occur if connecting and disconnecting the apparatus without powering off the machine. Accidental physical contact with Sybr Safe containing agarose gel
Wear appropriate PPE (nitrile gloves, lab coat, UV resistant goggles).
2
1
2
 
 
Team part
For Team part
Basic part
For basic part
Composite part
For composite part
Part collection
For part collection
esaGFP quorum sensing [Adrian, Clarice, Kenneth]
Invasin + Listeriolysin [YanTing, YunTing]
Maintenance
FNRgfp [YiHan, ChiYan]
Yi Han
Received 2 bacterial stab cultures, EsaR/I plasmid from addgene (CHL) and BBa_K299812 from iGEM HQ.
Streaked out on plates with amp.
Adrian
Transformed:
+Kit plate 1 9N Ba_K763002 chl
+Kit plate 4 13L BBa_E0040 amp
DNa was received in powder form in plates, and resuspended in 10ul ultrapure H2O respectively. Plates were stored in -20/
Yi Han
Plates cracked in incubator as they dried from lack of humidity.
Transfer to small incubator with beaker of water for humidity no single colonies for inv plasmid -> streak again.
Wrong antibiotic for EsaR/I plasmid-> streak out again
Yi Han
Inoculate single colony of invasin plasmid carrying bacteria in 3mL LB+amp
Transformation of 13L repeated with 1ul of DNA.
Xin Yi
No colonies grew for 13L on all plates ->adjust incubator, make new media
Miniprep of inoculated bacteria for invasin plasmid, incubate sample in 37degC for 1h 20min
RE of invasin plasmid
RE control
Rsal 1ul
Rsal 0 ul
EcoRI 1 ul
EcoRI 0 ul
INv plasmid 7.5ul
Inv plasmid 7.5 ul
H2O 35.5 ul
H2O 37.5 ul
NEB buffer 4.5 ul
Buffer 4.5 ul
 
Repeat transformation of 13L with 1ul
Xin Yi and Yi Han
Miniprep of EsaR/I plasmid for 4 colonies
Restriction digest for EsaR with XbaI/BamHI
RE reaction
5ul plasmid
5ul buffer
1ul XbaI
1ul BamHI
Add H2O to 50ul
Xin Yi
Sent EsaR clone 4 and INv-4 for sequencing.
Yi Han
YFP and GFP transformation results - no colonies for YFP
The GFP transformation repeated with 100ng of plasmid was sucessful.
4 colonies of gfp plasmid were inoculated in 3mL LB+amp and grown overnight.
Yi Han
Miniprep of gfp plasmids
RE digest with EcoRI and RsaI
Chi Yan
RE digest indicated a very faint smaller band for colonies 2-4, and hence these were likely to be positive clones
Yanting
Inoculated 3mLS of Inv-4 and EsaR-4 plasmid carrying bacteria into 100mL LB+ appropriate Antibiotic
Cell culture-> HEK293 cells revived from freezing down appeared detached.
Yi Han + Yanting
Storage of bacterial glycerol stocks for Inv-4, EsaR4 in 25% glycerol
Yanting: grew HEK293T cells in T25 glask
Gel extract to clean up gfp plasmids which loading dye had accidentally been added to.
Yi Han
Miniprep of gfp plasmids, preparation of samples for sequencing
Yi Han
All gfp plasmids had a correct sequence
Yunting kept glycerol stocok for all, and inoculation of 3mL of gfp3 into 100mL LB+amp for midiprep
Yi Han + Yunting
Midiprep of gfp plasmid PCR of gfp with KpnI-gfp and XhoI-gfp primers
Yunting + Duy
Nanodrop of gfp product-> 595.5ng/ul
RE digest of EsaR vector with KpnI/XHoI
Gel electrophoreseis at 1000V for 30min
Gel extraction: 3.9ng/ul and 6.9ng/ul for Esa fragment and GFP --> low yield
Yunting
Gel extraction using Promega binding solution to melt gel, followed by thermo scientific kit
Adrian
Gel extraction optimisation
Hypothesised that the Binding buffer has a problem/DNA does not bind to column
1: 2X promega binding buffer volume
2: Increase incubation time for binding to 5min
Switch binding buffer to that of thermo scientific PCR purification kit
Use sodium acetate if available? TO facilitate stronger binding to column
Results
1: ~10ng/ul
2: ~9ng/ul
not succesful
further optimisation-> warm buffer, incubate for 5min
elute in 30/20ul smaller volumes
Yihan
1: Thermoscientific miniprep columns with 2XThermoscientific binding buffer
2: Thermoscientific PCR purification kit 2X buffer
3: Promega kit 2X buffer
Yihan
PCR (mastermix: 8)
Reagent
Amount
PCR buffer
10 ul * 8 = 8-ul
primers
0.8ul, 0.8ul
DNA polymerase
4ul
dH2O
61.75 * 8 = 494ul
Templates
5.55*7 = 38.75ul
Digest more plasmid (Esa plasmid)
4 rxns ( KpnI/XhoI digest) 50ul each
Reagent
Amount
Buffer
 
KpnI
2ul
XhoI
2ul
DNA
34.8ul
H2O
141.2ul
 
Yihan
PCR
RP_XhoI_GFP and FP_KpnI_GFP with GFP midiprep
For 15 reactions with control .: Mastermix * 17
Reagent
Amount
PCR buffer
170ul
primers
1.7ul, 1.7ul
dNTP
34 ul
DNA polymerase
8.5 ul
dH2O
66.3*17 = 1127.1ul
templates
17ul
total
1360ul
PCR protocol “GFP 1” (32 cycles)
GFP PCR product -> 448ng/ul
Gel extraction
Qiagen gel extraction kit at MBI
Esa gel band from 14/6
Nanodrop: 16.4 ng/ul, 260/280 = 2.60
Plasmid construction
RE digest (KpnI, XhoI)
3 Replicates of the following
Reagent
Amount
DNA
2.6ul
Buffer
5ul
H2O
41.5ul
KpnI
0.5ul
XhoI
0.5ul
Ligation (2x reaction) 40ul
Reagent
Amount
10X T4 DNA ligase buffer
4ul
Vector DNA (16ng/ul)
100ng -> 6.25ul
insert DNA
75nl -> 12 ul
T4 DNA Ligase
2ul
H2O
16ul
Note: ALL digested DNA in tube labeled lacGFP.ligation was used
Nanodrop of ligation: 1114.0ng/ul. 260/280 = 3.7
Transformation
Transformation of ligated esa-GFP plasmid into DH5\alpha cells
DH5\alpha ,<- unlabelled brown vial inside DH5\alpha box at -80
Plates spread at 11:40h
LB + chloroamphenicol
+4x (100ul)
+10x (40ul)
+20x (20ul)
+LB - 20x (20ul)
remaining transformed cells (~220ul) are kept at 4C
--> labeled as placGFP in brown tube
Oservation: only 4x placGFP plate has colonies (7)
spread one new LB + chlor plate with 200ul of transformed bacteria
picked 6 colonies to grow for miniprep in LB + chl liq media
Conclusion:
+protocol works
+optimization for increase vol needed
cloning of synparts in amp vector
+comes as 4mg dry DNA
+40ul of DI/RNAse free H20
+for transformation, 150ml on wed
Unknown
Miniprep of placGFP followed by RE digest (Kpn, XhoI): 50 ul total
Reagent
Amount
Buffer
2.5ul
KpnI
<ILLEGIBLE>
XhoI
<ILLEGIBLE>
DNA
10
H2O
12025
Colony PCR for synparts
+ Failed
+ No specific <illegible> produced
+ Might need optimization
 
Yihan
RE digest (XhoI, KpnI) of esa and GFP
Yunting
Gel electrophoresis (100V, 40min)
+ Lane 2: gpf: no bands
+ Lane 3: 100bp ladder
+ Lane 4: blank
+ Lane 5: esa: 2 bands
+ Lane 6: 1kb ladder
+ Lane 7:blank
<Picture of gel>
Optimizing gel extract protocol
+ RE GFP from PCR (directly RE)
+ promega agarose 1% gel
+ add NaAC in binding buffer
Result:
+ yield for cut plasmid was 12.8ng/ul
+ GFP was 8.6ng/ul
Ligation reaction 6 reactions
Reagent
Amount
Buffer
12ul
Vector
30ul
Insert
30ul
 
+ 6x ligation result transformed into competent DH5\alpha 20 ul, 50ul, 100ul plated on LB + chl
Yunting
Ligation of GFP tester plasmid.
Performed 4x ligation using RE- digested esa and GFP (CY, 22/6). Ligation rxn at room temperature for 30min instead of 10min.
Included vector-only and insert-only controls.
Stored in -20deg.
Will run gel tmr. Insert only control should be same size as gfp product. Same for the other control. Can try to plate vector only to see re-ligation??
Nanodrop:
PlacGFP - 642.8ng/ul. 260/280=3.89
Vector ctrl - 756.6ng/ul. 260/280=4.11
Insert ctrl - 557 ng/ul. 260/280=3.86
Yanting
Ran 0.8% pre-cast gel at 100V for 45min.
10ul of each sample to 2ul of loading dye.
Gel lanes:
100bp; uncut esa (used esa4 from -20);
pLacGFP; vector ctrl; insert ctrl; 1kb.
[YH] Re-run gel. 35ul of each sample. No bands.
<insert gel picture>
RE of esa and Gfp pcr product.
Gel electrophoresis.
Cast a thick 1% gel (60ml) with combined wells. [Don't need to cast thick gel next time, takes too long to melt]
Gel run at 100V,  45min. Lanes: 100bp, esa, gfp 1&2, 1kb
Image after cutting is also saved.
Gel extraction of RE esa & gfp.
Used Promega kit, loaded both gfp bands into one column. Eluted with 30ul water for 5min before centrifugation.
Nanodrop:
Esa- 37.7ng/ul. 260/280= 1.83
Gfp - 25.1ng/ul. 260/280= 1.84
Overnight ligation
6x ligation:
Reagent
Amount
Buffer
6ul
Vector (37.7ng/ul)
50ng = 7.8ul
Insert (25.1ng/ul)
37.7 ng = 6.6ul
H2O
87.6ul
Ligase
6ul
Ligation reaction run overnight in 16 deg hold in thermocycler - 15h, 8pm - 11am.
 
[Chi Yan + Yunting] Midiprep of EsaR plasmid
[Yihan Yunting] Gel electrophoresis of inv colony PCR
insert picture gel
[Xinyi] Colony pcr for placgfp for >18 colonies
insert gel picture
[Adrian] Gfp expression observed using gfp filter with the SPS microscope
Yanting
Subculture of HEK 293
P4 -> p5
Grow/split into 150mm dish for freezing on sat(20/6)
-->30ml DMEM + 1ml cells
90mm dish for maintenance (buffer)
-->10ml DMEM + 30ul cells
Cells combined from 3 90mm dishes
Yanting
Freezing of HEK293
P6
Cells from a 150 mm dish and 90mm dish
Adrian
Prepared restreak of inv/hly plasmid from original stab culture
Yanting and Yunting
Colony PCR of invasin
Picked out 8 colonies (marked 1-8) from BBa_K299812 plate stored at 4deg (Adrian, 12/7).
Colony PCR using prefix suffix primers for first 4 rxn and universal primers VP & VF2 for last 4 rxn. Used thermocycler "Colony" protocol.
Also constituted dNTP w 2.5mM of each atcg triphosphate.
Yanting
100V at 30min. Ladder, 4 prefix suffix rxn, 4 universal primers rxn. Saved as “7.15_inv colony”.
When I ran for longer (after storing gel at 4deg), the bands were longer but the 250bp marker as well as the primer-dimers are pretty close to dye front.
Yunting
Placed BBa_K299812 plate in incubator for overnight growth.
Yanting and Yunting
Colony PCR of 8 colonies (marked A-H) from BBa_K299812 plate, and also spotted on a save plate. Both plates placed back in incubator.
Primer conc=0.5 uM, template DNA dissolved in 10ul h20.
Thermocycler “colony_inv” protocol, with adjusted extension time and annealing time&temp from standard “colony” protocol.
100V for 34min.
Tubes 1-2: FP-prefix, RP-suffix.
3-4: FP-VF2, RP-VR.
5-6: FP-prefix, RP_Inv_M2.
7-8: RP-suffix, FP_Inv_M1.
No template control with FP-prefix, RP-suffix.
[Note: RP_M2 = FP_M2. Check future uses agn seq on the primer master file. Just in case, the sequence used in this expt is INV_FP_M2->GCTCATTATAGTCCGCGAAATCACG].
Gel image saved as “7.17_inv colony.sgd” (handphone pic below).
Results: Tubes 3&4 with universal primers VF2 VR have a band between 4&5kb - Inv+LLO is 4.1kb, probably are positive colonies.
Tubes 1&2, 5&6 have bands <750bp as well as primer dimers (but the annealing temperature was calculated for prefix suffix primers not universal ones…I’ll try thermo-gradient thermocycler protocol next time).
Expected band size for 5&6 (FP-prefix, RP_Inv_M2 (ends 1928)) = 1.9kb.
No bands for tubes 7&8. Expected band size for 7&8 (RP-suffix, FP_Inv_M1 (starts 1046)) = 2.2kb.
 
Yihan
Inoculation of positive colonies in liquid culture. 3mL and shaking incubation.
Yunting
Miniprep of positive colonies from 30h liquid culture.
Eluted in 50ul elution buffer and stored at -20.
C= 225.9ng/ul. 260/280=1.86
D= 297.7ng/ul. 1.87
Yunting
Colony PCR with thermogradient: 14rxn
Reagent
Amount
H2O
70ul
dNTP
14ul
MgCl2
35ul
Taq
3.25ul
primers (forward + reverse)
14ul*2
Template DNA
14ul
C/D-1,2: Col3.
C/D-3,4: Col12.
D-5,6: Col 9.
D-7,8: Col 10.
Negative control (no template): Col 8.
Gel ran for 80V, 1h.
D-1,2: VF2, VR.
D-3,4: FP-prefix, RP-suffix.
D-5,6: FP-prefix, FP_M2.
D-7,8: FP-prefix, FP_invF.
C-1,2: VF2, VR.
C-3,4: FP-prefix, RP-suffix.
Results Notes:
Too much template DNA (~250ng). Separate the universal primers (to avoid differing extension time).
Yanting
RE digest of inv/hyl plasmid with EcoRI and PstI for 2 hours at 37oC.
Lane 1-4: 1kb ladder, 100bp ladder, RE of colony C, RE of colony D.
Size is correct: 4kb main band (inv+hly part) and 2kb (plasmid backbone). These plasmid DNA should have the part.
Yanting and Yunting
Colony PCR, used with temperature gradient to vary annealing temperature. 10rxn
Reagent
Amount
5x buffer
50ul
H2O
127.5ul
dNTP
50ul
MgCl2
10ul
Taq
3.25ul
primers (forward + reverse)
14ul*2
Template DNA
14ul
100V for 45min (can run for longer).
Result:
Lane 1: 1kb ladder; lane 2: 100bp ladder;
lane 3-7: FP prefix + RP suffix (5 repeats with increasing annealing temperatures) has 2 bands ~800 bp & 100-200bp (probably non specific bands, will lower # of cycles in future, use higher annealing temp, lower annealing time).
Expected size of inv+hly part is 4.1kb.
lane 8-9: VF + M2 has a thick band 800-900bp.
Expected size is 600+130bp (VF adds ~130bp compared to FP_prefix) .: doesn’t seem to be correct….
lane 10-11: VF + inv F has no bands. Expected is at least 200bp (bcos universal primers).
lane 12: VR + inv F seems to have a small band <100bp. Non specific amplification?
Expected is 1.5+0.1 kb (from VR).
Yanting
RE with XbaI and PstI
Reagent
Amount
Restriction enzyme
2ul
Buffer
5ul
DNA (4x of miniprep=74.5ng/ul)
13.4ul
H2O
29.6ul
Incubate at 37degC for 2.5h (1130-1400)
Results: Gel loaded 1kb ladder, uncut, RE digested. 100V for 1h.
Plasmid doesn't have XbaI site - RE digested DNA is a linear 6kb band.
Yunting
Cast a big gel. Stored in 4 deg. [Used up 26/6]
Pre-cast two 0. 8% gels. Stored in 4deg. [Used up on 24 & 25/6]
Chi Yan and Yunting
Midiprep of esaR plasmid.
Airdry ON.
Safety Inspection for our lab by OSHE. We passed with flying colours!
Transformation of ligated FNRgfp plasmid into dH5alpha
Cleaned waterbath, de-iced the fridge.
Today, we also did a spring cleaning for the lab
Pre-cast big and small gels and stored in 4deg in the blue tupperware. [Used up on 20 & 22/7]
Adrian
BBPrefix_esaRBS PCR
Reagent
Amount
H2O
221ul
Buffer
80ul
MgCl
32ul
dNTP
32ul
Forward Primer_Biobrick Prefix
16ul
ORP_esaRBS_fragsyn
16ul
synpart_BBP_esaRBS
1ul
gotaq
2ul
total
400ul
success-> PCR purification
esaRBS_GFP_BBsuffix
Reagent
Amount
H2O
237ul
Buffer
80ul
MgCl2
32ul
dNTP
32ul
Reverse primer_BB_suffix
??
ORP_esaRS_GFP
16ul
Plac_GFPPLASMID
1ul
gotaq
2ul
failed -> troubleshooting-> new OFP_esaRBBS_GFP
Redid 30/6 with new primers
success with new primers-> fusion pcr
PCR with inv primers (adrian’s primers)
 
Clarice
Fusion PCR with esaRBS, GFP
Reagent
Amount
H2O
104.07ul
Buffer
40ul
MgCl2
16ul
dNTP
16ul
Forward primer_BB_prefix
8ul
Reverse primer_BB_prefix
8ul
400ng esaRBS
2.13ul
400ng esaRBSgfp
4.8ul
gotaq
1ul
Total
200ul
annealing temp 50degC
Gel electrophoresis: looks correct-> PCR Purification
RE digest of esaRBS+GFP
Reagent
Insert
Vector
Buffer
2ul
2ul
EcoRI/PstI
1ul
1ul
Template(2ug)
5ul
4ul
dH2O
11ul
12ul
total
20ul
20ul
Clarice
COlny PCR with different colonies (15 colonies)
FP_BB_Prefix and RP_BB_Suffix
Success
Clarice
Colony PCR successful - inouclated colony 2,3,4,12 in 3mL LB
PCR-FNRsynpart BBP
Reagent
Amount
dH2O
221ul
Buffer
80ul
MgCl2
32ul
dNTPs
32ul
FP_RNE_PromoterBBP
16ul
GFP_FNR_Prom_GFP
16ul
synpart FNR
1ul
gotaq
2ul
 
Clarice
Verify minipreppped esaGFP plasmid
Reagent
Amount
Buffer
2ul
EcoRI
1ul
Pst1
1ul
Plasmid
5ul
dH2O
11ul
total
20ul
Clarice
Transformatin of EsaGFP plasmid (30ul BL21 + 5ul ligation reaction)
plate O/N
Yihan
Ran gel of gfp plasmid EcoRI/PstI, took out gel slice for vector - 4kb
Gel extraction:
+gfpvector->38.7ng/ul fragment-> direction purification 226.4ng/ul
+LIgation reaction (5:1) 6X
Reagent
Amount
300ng vector
7.8ng/ul
531.2ng of insert
2.4ul
Buffer
12ul
Enzyme
6ul
H2O
91.8ul
ligate for 2 hours at 16 hours transform, plate, grow ON
Yihan
No colonies-> ligation did not work
Recalcualte ligation reaction (3:1),7x
350ng vector->9ul vector
446.3ng insert-> 2ul 7ul
ligase 110
H2O
12ul buffer
Yihan and Clarice
colonies for EsaGFP
grew Colony PCR for 14 colonies
Unsuccessful-> no bands observed
Yihan
Trying out an idea for workshop
Trial for blue/white screen
Streak out pGFPuv and pGEMT on plate with amp added spread 40ul of 0.1M IPTG and 30ul of 5%xgal, dried and on plate without amp
Yihan
Ran gel with FNR Pcr and gfp pcr
size of pcr product was correct but gel picture was not saved
FNR Product was pcr purified
Plasmid extraction for colony 2,3,4 of esaGFP - > 2,3 were sent for sequencing and primers did not bind, chromatogram was messed up.
Fusion PCR for FNRGFP:
Reagent
Amount
dH2O
218ul
10X buffer
80ul
MgCl2
23ul
dNTP
32ul
Template FNR PCR (40ng)
3ul
gfp PCR (400ng)
1ul
FP_BB_Suffix
16ul
RP_BB_Suffix
16ul
gotaq
2ul
Chi yan
Gel was run for fusion pcr size was correct
Clarice RE digest:
Reagent
Amount
PstI
1.5ul
EcoRI
1.5ul
Buffer
4ul
Template
3ul
H2O
20ul
Total
40ul
Chi yan
Ran gel for gfpp plasmid
gel purificaiton and extraction of 4kb fragment
Reagent
Amount
ligation reaction (2X)
40ul
Buffer
4ul
100ng of DNA vector
3.4ul
157ng insert
16ul
enzyme
2ul
H2O
14.6ul
Yihan
no colonies for FNRgfp plasmid
Inv plasmid verification
Reagent
Amount
buffer
1ul
EcoRI
0.5ul
invasin plasmid
0.5ul
dH2O
8ul
Total
10ul
Gel extraction RE digest of EsaR-GFP plasmid
Reagent
Ligation
Control
Vector (45ng)
1ul
1ul
Insert
1ul
1ul
Buffer
1ul
1ul
dH2O
6ul
7ul
ligase
1ul
1ul
Transformation of FNRgfp anaerobe jar
E. coli grew in both conditions p putida-> some growth in anaerobe chamber proper streak plate in aerobic conditions
packet runs out after 16hours
Yihan and Yunting
LIgation for FNRgfp (4X)
Reagent
Amount
200ng Gfp plasmid
4.36ul
297ng insert (7:1)
10.3ul
Buffer
53.34ul
T4 ligase
4ul
Trial run of anaerobe chamber:
+ open sachet to decrease O2 at 3.30pm
+ takes 2.5h to activate
place streak plate of p putida a strict aerobe and e coli, facultative aerobe in chamber at 30deg C to grow O/N
P. putida is an abligate aerobe and if chmaber works ,it will not grow
E. coli should grow in both conditions
controls grown outside chamber
Yihan
FNRgfp-> no colonies after transformation
religation (4X reaction)
Reagent
Amount
200ng GFP plasmid
4.4ul
2125ng of insert
8ul
T4 ligase
4ul
H2O
55.6ul
transformation of FNR gfp into 20ul of BL21
Yihan
No colonies grew
Ran a gel, FNRgfp pcr, gfp pcr, fnr, 100bp ladder
Chi Yan
RE digest of FNR-GFP fusion PCR with EcoRI and PstI
Reagent
Amount
EcoRI
1ul
PstI
1ul
Buffer
5ul
FNR-GFP from 12/7 in RIP box 1ug
2.5ul
H2O
40.5ul
Total
50ul
Direct purification using Promega kit
Overnight ligation (4X reaction)
Reagent
Amount
200ng of GFP plasmid
4.4ul
400ng of insert (5:1)
?
T4 ligase
4ul
T4 ligase buffer
8ul
H2O
?
 
200ng of gfp plasmid (4.4ul)
400ng of insert (5:1) (?ul)
4ul T4 ligase
8ul T4 ligase buffer
? H2O
Yihan
Ncbi search in BL21 genome revealed that it does have FNR transcriptional regulator
http://www.ncbi.nlm.nih.gov/nuccore/CP010816.1
In dH5alpha as well
http://www.ncbi.nlm.nih.gov/gene/945908
However no direct data on our specific strains of dH5alpha and BL21
Adrian 1_8
PCR
reaction
H2O
222ul
Buffer
80ul
MgCl2
32ul
dNTPs
32ul
VF2
16ul
VR
16ul
GoTaq
2ul
Cycle conditions 95ºC (3min) -> 95ºC (30s) -> 60ºC (30s) -> 72ºC (1.5min) -> 72ºC (5min) for 25 cycles Result: FP_Biobricks_Suffix and RP_Biobrick_Suffix primers are contaminated -> rediute from stock Maintenance Red Workshop Protocol
To make Fragment 1 for the
workshop
H2O
222ul
Buffer
80ul
MgCl2
32ul
dNTP
32ul
Primer 1 (FP_retarded)
16ul
Primer 2 (RP_retarded)
16ul
GoTaq
2ul
GFP plasmid
1ul
To make Fragment 2 for the
workshop
H2O
222ul
Buffer
80ul
MgCl2
32ul
dNTP
32ul
Primer 1 (OFP_esaRBSver2)
16ul
Primer 2 (RP_VR)
16ul
GoTaq
2ul
GFP plasmid
1ul
Cycle conditions 95ºC (3min) -> 95ºC (30s) -> 60ºC (30s) -> 72ºC (2.5min) -> 72ºC (10min) for 25 cycles
Adrian 2/8
Recreating backbone
pSB1A2
EcoRI/PstI
H2O
22ul
pSB1A2-GFP
10ul
Buffer
4ul
EcoRI
2ul
PstI
2ul
Total
40ul
Adrian 3/8
Fusion PCR for
workshop
H2O
111ul
Buffer
40ul
MgCl2
16ul
dNTP
16ul
OFP_esaRBS_GFP_ver2
8ul
ORP_XhoI_retarded
8ul
GoTaq
1ul
Fragment 1
1ul
Fragment 2
1ul
95ºC (3min) -> 95ºC (30s) -> 60ºC (30s) -> 72ºC (1.5min) -> 72ºC (10min) for 25 cycles
EsaR Master Plate
Test
Colony PCR
H2O
222ul
Buffer
80ul
MgCl2
32ul
dNTP
32ul
VF2
16ul
VR
16ul
GoTaq
2ul
400ul
For 10 colonies
Ligation of esaRBS-GFP (EcoRI/PstI) and RNG-GFP (EcoRI/PstI) into pSB1A2 (EcoRI/PstI)
10x T4 DNA ligase Buffer
1ul
Vector
1ul
Insert
5ul
H2O
10ul
T4 DNA ligase
1ul
Reactions were incubated for 30min at room temperature Transformation pSB1A2 esaRBSGFP pSB1A2 RNG GFP Control cut pSB1A2
Adrian 17/7
Ligation of esaRBS-GFP (EcoRI/PstI) and RNG-GFP (EcoRI/PstI) into pSB1A2 (EcoRI/PstI)
10x T4 DNA ligase Buffer
1ul
Vector
1ul
Insert
5ul
H2O
10ul
T4 DNA ligase
1ul
Reactions were incubated for 30min at room temperature Transformation pSB1A2 esaRBSGFP pSB1A2 RNG GFP Control cut pSB1A2
Adrian 18/7
Vector dimer check 8 colonies were selected from the control plate (26/7)
PCR
mastermix
H2O
222ul
Buffer
80ul
MgCl2
32ul
dNTP
32ul
FP_Biobricks_Prefix
16ul
RP_Biobricks_Prefix
16ul
GoTaq
2ul
Total
400ul
The control plate had plasmids with a 800bp insert Gel extract was performed on a 2kb band Ligation Troubleshooting
Ligation
Mastermix
H2O
222ul
Buffer
80ul
MgCl2
32ul
dNTP
32ul
FP_Biobricks_Suffix
16ul
RP_Biobricks_Suffix
16ul
GoTaq
2ul
Total
400ul
Colony PCR 8 colonies were selected from esaR and RNG each, total 16 colonies
PCR
Mastermix
H2O
444ul
Buffer
160ul
MgCl2
64ul
dNTP
64ul
FP_Biobricks_Suffix
32ul
RP_Biobricks_Suffix
32ul
GoTaq
4ul
Total
800ul
Ran gel Lanes Vector Backbone (EcoRI/PstI Control Ligation mix esaR ligation mix RNG ligation mix T4 Ligation buffer Blank
Adrian 26/7
BBPprefix_esaRBS PCR synthesis
PCR
Reaction
H2O
22ul
Buffer
80ul
MgCl2
32ul
dNTP
32ul
FP_Biobricks_Prefix
16ul
ORP_esaRBS_fragsyn
16ul
synpartBBP_esaRBS
1ul
GoTaq polymerase
2ul
Total
400ul
esaRBS_GFP_BBsuffix
H2O
221ul
Buffer
80ul
MgCl2
32ul
dNTP
32ul
RP_BiobricksSuffix
16ul
OFP_esaRBS_GFPver2
16ul
pSB1A2_BBa_0040
1ul
GoTaq
2ul
Total
400ul
PCR for
esa
H2O
220ul
Buffer
80ul
MgCl2
32ul
dNTPs
32ul
FP_BB_Prefix
16ul
RP_BB_Suffix
16ul
BBPrefix_esaRBS
1.2ul
esaRBS_GFPBBSuffix
0.5ul
GoTaq
2ul
Total
400ul
PCR for
RNG
H2O
220ul
Buffer
80ul
MgCl2
32ul
dNTPs
32ul
FP_BB_Prefix
16ul
RP_BB_Suffix
16ul
BBPrefix_esaRBS
1.5ul
esaRBS_GFPBBSuffix
0.5ul
GoTaq
2ul
Total
400ul
Adrian Clarice
REdigest of esaR and RNF after PCR purification and gel extraction
esaR-gfp RE
digest
Bffer
2ul
PstI
1ul
EcoRI
1ul
2ug Template
5.4ul
H2O
10.6ul
Total
20ul
RNG RE
digest
Bffer
2ul
PstI
1ul
EcoRI
1ul
2ug Template
3.9ul
H2O
12.1ul
Total
20ul
Ligation of pSB1A2 with esaR and
RNG
10X T4 Ligase Buffer
2ul
Vector (pSB1A2 EcoRI/PstI)
1ul
Insert (esaR and RNG)
5ul
H2O
11ul
T4 DNA ligase
1ul
Total
20ul
Ligation for 30min at room temperature
Clarice 19/7
Colony PCR for RNG and esaR-GFP colonies from master plate
RNG
(10X)
H2O 13.8 + colony
each
Buffer
50
MgCl2
20
dNTPs
20
Primer 1
10
Primer 2
10
Gotaq
2
esaGFP
(9X)
H2O
45
Buffer
18
MgCl2
18
dNTPs
9
Primer 1
9
Primer 2
9
GoTaQ
1.8
CY 8/8
8 minipreps of 4 clones from BL21 pNirB+gfp and 4 clones from dH5alpha pNirB+gfp
RE
digest
EcoRI
9ul
PstI
9ul
Cutsmart Buffer
45ul
DNA of 1ug
6.7ul
H2O
36.3ul
Result after running gel: All clones are positive for pNirB+gfp
CY 13/8
Invasin + Listerolysin blue
Inoculated 4 colonies of BL21 transformed with the inv+hly clone D.
CY 14/8
Invasin + Listerolysin blue
Miniprep of the inv+hly clones
RE digest with
EcoRI/PstI
EcoRI
5ul
PstI
5ul
Buffer
25ul
DNA 1ug
(4ul)
H2O
11ul
Total
50ul
CY 19/8
Colony PCR for pEFYP plated on (17/8) Pcr for pNirBgfp as with (18/8) with pNirB-gfp primers, Biobricks prefix and suffix primers
PCR Master mix For
pEYFP
H2O
13.25ul
Buffer
5ul
dNTP
5ul
F/R primer
5ul
2.5mM MgCl2
12.5
GoTaq
11.25ul
PCR Master mix for pNirBgfp with pNirB-gfp primers, and Biobricks Prefix and Suffix Primers (Melting temperature increased to 53degrees) 3ul of plasmid in each tube
PCR Master mix for pNirB-gfp primers and BB prefix/suffix
primers
H2O
101.25ul
Buffer
45ul
dNTP
9ul
Forward Primer
9ul
Reverse Primer
9ul
2.5mM MgCl2
22.5ul
GoTaq polymerase
2.25ul
CY 21/7
Ran gel for 20/7 colony pcr for FNR gfp 1kb, 100bp, colonies negative control
CY 22/7
Anaerobic promoter Brown
PCR to test 'FNR-GFP plasmid'.
Colonies 1-5 from 20/7 + negative control 3 sets of primers FP_BB Prefix, RP BB Suffix GFP-F, GFP-R FP_FNRPromoter_BBP, ORP_FNR_Promoter GFP
For each reaction (6X)
H2O 73.5ul
dNTPs 6ul
Forward/Reverse Primer 6ul/6ul MgCl2 15ul
Taq polymerase1.5ul
For each plasmid 1ul
Buffer 30ul
CY 24/7
Ligation of Cut FNRGFP (17/7) with cut GFP vector
1X
reaction
Buffer
2ul
200ng vector
1.1ul
3:1
256.5ng
Water
3.4ul
T4 DNA ligase
1ul
Total
20ul
Incubation at room temperature for 2h Transformation into 10ul BL21 competent cells
CY 27/7
RE of extracted product with EcoRI/PstI for 300ng of DNA
RE
(1X)
FNR-gfp
0.9ul
EcoRI
1ul
PstI
1ul
Buffer
2ul
Water
15.1ul
Total
20ul
CY YH
When gel was run, primer dimers and 700-800bp band were present in the negative control with no template added
Fresh buffer, MgCL2, dNTP and fresh diluted primers were used to repeat colony PCR.
Diluting dATP, dCTP, dGTP, dTTP to 2.5mM
Repeated colony pcr with Prefix-suffix primers using TC pipettors
HYT 15/7
Invasin + Listerolysin blue
RE digest of the miniprepped inv/hly plasmid with EcoRI/PstI from (20/7) Tube C-> 226ng/ul Tube D-> 298ng/ul
RE
reaction
Buffer
3ul
Plasmid 4ul
each
EcoRI
1ul
PstI
1ul
H2O 21ul
each
Total
30ul
incubated at 37degC for 2h Add 6X loading dye to stop reaction in both tubes Ran gel (1kb, 100bp, cut vector C, D)
CY 24/7
Anaerobic promoter Brown
Ligation of Cut FNRGFP (17/7) with cut GFP vector
1X
reaction
Buffer
2ul
200ng vector
1.1ul
3:1
256.5ng
Water
3.4ul
T4 DNA ligase
1ul
Total
20ul
Incubation at room temperature for 2h Transformation into 10ul BL21 competent cells
XY 18/7
Anaerobic promoter Brown
Transformation of FNRGFP ligated product into BL21 (from CY 17/7) 1. 3ul pUC19 control +10ul BL21 2. 20ul Ligation product + 20ul BL21 3. 5ul Ligation product + 20ul BL21 Plated on LB+amp plates 40ul of transformed bacteria in SOC media on LB+amp 20ul of transformed bacteria in SOC media on LB alone
YH 2/8
Inoculated colony 3,7 of FNR GFp selected positive clones for miniprep
YH 3/8
Anaerobic promoter Brown
Miniprep of colony 3, 7 and sent for sequencing with GFP-R and FP_VF2 -> results show there is GFP but no FNR.
YH 6/8
Transform pNirB+gfp from the Biobricks Registry into BL21 and dH5alpha
YH 9/8
OD600 of 1 = 10^8 cells/cm^3
OD600 of 10X
dilution
BL21 clone 1 0.013 -> 1.04x10^8
cells/cm^3
BL21 clone 2 0.094 -> 7.52x10^8
cells/cm^3
dH5alpha clone 1 0.054 -> 4.32x10^8
cells/cm^3
dH5alpha clone 2 1.123 -> 9.84x10^8
cells/cm^3
BL21 0.084 -> 6.72x10^8
cells/cm^3
dH5alpha 0.046-> 3.68x10^8
cells/cm^3
BL21 + pGFPuv 0.036 -> 2.88x10^8
cells/cm^3
The cultures were diluted 100X and 500X in a volume of 200ul in a 24well plate and grown in the anaerobic chamber at 30degrees and 225rpm for 2, 4, 6, 8 hours
Replicating the iGEM Valencia Team - growing cultures under sterile oil (filtered through 0.22um filter) 0.5mL of stationary culture + 2.5mL LB, with 1mL of sterile oil added above
Grown at 28 degrees, 200rpm for 2 days
Results:
GFP signal is expressed in cultures with the anaerobic sensitive promoter grown in anaerobic conditions overnight, but signal was weak.
After 6 hours in anaerobic growth, the BL21 + pNirB clone 1 had highest GFP signal when observed with the GFP microscope.
BL21 strain had no signal
After 24 hours in anaerobic conditions, dH5alpha had highest signal compared to control
Control - colonies picked directly from plates (aerobic conditions) had no GFP signal
After 2 days there was no gfp expression
YH 14/8
Seeded cells for invasion assay
BL21 + inv-hly clones correct after running gel for digest of the plasmids with EcoRI and PstI
YH 15/8
Invasin + Listerolysin blue
Invasion assay with HEK293T cells and BL21, and BL21+inv-hly plasmid on 15/08/15
Rationale: The invasin gene in the invasin+listerolysin plasmid should remain intact and be expressed, leading to an invasion phenotype
Objective: Screen invasion phenotype of BL21 and BL21+invasin-listerolysin Biobricks part
Experiment details
Cells were passage 2 Seed 0.5mL of 5x10^5 HEK293T cells/mL and 1x10^6 in wells of 24-well plates, grow overnight.
Determine cell confluency with microscopy the next day - 80-90% confluency
All strains were grown overnight in 3mL LB broth
For 5x10^5 cells
100ul of 10^9 cells/mL was used for moi 200
100ul of 2X dilution of 10^9 cells/mL was used for moi 100
100ul of 4X dilution of 10^9 cells/mL was used for moi 50
100ul of 5X dilution of 10^9 cells/mL was used for moi10
Result Lawn on all plates HEK293T cells detach too easily.
Grow for 24 hours in the future and do minimal, and gentle washing. For next experiment, 0.5x10^6 and 1x10^6 cells wre seeded 1000ug/mL kanamycin for 1 hour is sufficient for kill step
no contamination in uninfected control
There is invasion in background BL21 strain as well
All mois showed high percentage of infection
In future use moi 10:1 and plate up 10^2 to 10^5 dilution for countable colonies
YH 17/7
For plates on LB alone there was a lawn
For pUC19, the transformation worked, and separated colonies were produced
No colonies were produced for the FNR-gfp plasmid
YH 18/8
PCR for the clones of pNirB-gfp from B21(1-4) and dH5alpha (1-4)with pNirB-gfp-F/R primers
5ul plasmid, 5ul H2O in each tube
PCR Mastermix
dNTP 8ul
MgCl2 20ul
Taq Polymerase 2ul
Buffer 40ul F/R
primers 0.8ul each
H2O 40ul
YH 19/7
inoculated colonies C,D for invasin plasmid in 3mL LB+amp at 1.30pm
Transformed remaining 40u ligation reaction for FNR gfp into 10ul BL21
Transformed 100ng pGFPuv plasmid into 10ul BL21
Poured new LB+amp plates
Plated 20, 40, 100ul of transformed bacteria in SOC media on plates
YH 20/7
pGFPuv transformation worked!
Single colony for FNR-GFP + 4 more from plating on 19/7
Sent FNR-GFP fusion product for sequencing with RP-Biobricks Suffx - result there is no FNR but there is no GFP
Colony PCR for FNR gfp (Colonies 1-5, (-) control)
Colony PCR
(6X)
H2O
19.5ul
Buffer
10ul
6ul
dNTP
6ul FP_Biobricks
Suffix
6ul RP_Biobricks
SUffix
15ul
MgCl2
1.5ul Taq
Polymerase
YH 21.7
Redid colony pcr for above as negative control had a positive band - contamination of PCR
negative control still has band - reagents contaminated
YH 22/7
Miniprepped colonies 1-5 for FNR-gfp plasmid
Did EcoRI/PstI digest
Digest for mastermix
(5X)
Buffer
10ul
EcoRI/PstI
2ul
H2O
60ul
Total 20ul per
tube
400ng plasmid in each
tube
Ran gel for RE samples, none were postive clones
YH 26/7
No colonies for FNRgfp
Hotstart fusion PCR (8X
reaction)
10X Buffer
80ul
dNTPs
16ul
FP Biobricks Prefix
8ul
RP Biobricks Suffix
8ul
FNR Pcr product
12ul
GFP PCR product
4ul
MgCl2
3ul
H2O
668ul
YH 27/7
Ran gel with cut vector and cut PCR product 1kb 100bp cut vector (wells 3-5) FNR-gfp (wells 6-8)
However, PCR product was not detectable
RE digest for FNRgfp
EcoRI/PstI
150ng PCR
5ul
Buffer
2ul
EcoRI
0.5ul
PstI
0.5ul
H2O
12ul
Total
20ul
Gel extracted Ligation reaction was performed overnight
Ligation reaction
(1X)
100ng vector
8.3ul
85ng insert
16ul
T4 ligase
2ul
Buffer
4ul
H2O
9.7ul
YH 28/7
Trialling transformation of 10ul BL21 with 100ng pGFPuv plasmid for workshop.
Using 200ul LB broth for the growth in 1hour step
10ul BL21 + 10ul plasmid Heatshock for 300s
Shake for 1hour at 37 degC 225rpm
plated 40ul on LB, 40ul on LB+amp plate
Transformation of ligation reaction into BL21 with control of cut vector (100ng) into BL21
plated 40ul on LB+amp plate, 20ul on LB plate
YH 29/7
Colonies on pGFPuv plate with amp -> LB works for reviving bacteria small colonies with FNRGFP plasmid
YH 30/7
12 colonies grew for FNRGFP plate no colonies on control with cut vector Colony PCR to screen 12 colonies and negative control using universal primers
PCR Mastermix
(13X)
H2O
42.25ul
PCR Buffer
65ul
VF2 Primer
13ul
RP VR Primer
13ul
MgCl2
32.5ul
GoTaq
3.25ul
[some introduction here]
Workshop
One of the participants taking a closer look at GFP-tagged E. coli under our very own SPS microscope.
The SPS iGEM Team of 2015 hosted a genetic engineering workshop for students from the Faculty of Science on 5th August 2015, in the Active Learning Room and the SPS Wet Lab. The workshop aimed to equip science students with an understanding of both the techniques of synthetic biology, and its risks and rewards. Participants were given the opportunity to be immersed in both the theoretical and wet lab components of synthetic biology.
Students were first guided through the concepts of genetic engineering, and the available wet lab tools and techniques used. After some light refreshments, they then got a chance to try their hands at designing their very own gene vectors with a fun set of theoretical puzzles.
 
One of the workshop facilitators explaining the process of constructing a genetic vector.
Participants and facilitators hard at work figuring out genetic puzzles
After lunch, the participants performed Fusion PCR (Polymerase Chain Reaction) and performed bacterial transformation in the SPS Wet Lab. They also had a look at green fluorescent protein (GFP) expressed in E. coli, as an example of one of the methods that are commonly used to quantify protein expression.
Participants beginning PCR in the SPS Wet Lab!
Loading a gel is hard work – participants ran a DNA gel to confirm if their PCR reaction was successful
All in all, both the workshop participants and facilitators spent an enjoyable day both learning and sharing about genetic engineering. The SPS iGEM Team of 2015 would like to thank all participants for spending their day with us! We would also like to thank Science Dean’s Office for their kind sponsorship, as well as the SPS staff and SPS community for their support.
A final group photograph with some of the facilitators and workshop participants
Video
Interview
Biosafety in our project involves minimising the risks to the researchers working in the laboratory, as well as the general public in future medical applications based off our research.
Safety when handling biological organisms
Non-pathogenic strains of E. coli K-12 strains BL21 and dH5α from Life Technologies were used for bacterial cloning of plasmids and expression of proteins of interest. These strains are Risk group 1 and were handled in a BSL2 Biosafety cabinet. The E. coli strain carrying the Biobrick BBaK299812 (containing parts derived from Risk group 2 organisms) was handled as a Risk Group 2 agent. Mammalian cell line HEK293T is classified under Risk group 2, and was also cultured in a BSL2 Biosafety cabinet.
Safety in Project Design
In our project, we aim to engineer non-pathogenic E. coli as a vector to deliver a potential drug into the tumour core. We use the Biobricks Part BBa_K299812 (http://parts.igem.org/wiki/index.php?title=Part:BBa_K299812), which contains the invasin gene from Yersinia pseudotuberculosis and the listerolysin O gene from Listeria monocytogenes. The Invasin protein allows for bacteria to enter mammalian cells, while Listerolysin O is a pore-forming protein that enable bacteria to escape the endosome. These two proteins are involved in pathogenesis of their respective bacterial species.
The Invasin and Listerolysin proteins enable our E. coli to enter mammalian cells, and escape the endosome, where they can subsequently deliver an encoded therapeutic to kill the tumour cell. To ensure that these proteins are only expressed under the conditions of the tumour microenvironment, the invasin and listerolysin proteins will be placed under the control of an anaerobic promoter, and a quorum sensing system.
Safety in Our Lab
All our team members have undergone Chemical, Biological and Fire Safety Training from the Office of Safety, Health and Environment (OSHE http://www.nus.edu.sg/osh/), the department in charge of Laboratory and Work Safety at the National University of Singapore
For each protocol used for our experiments, we have a separate risk assessment. Please refer to our ‘protocols’ page for more information.
Our laboratory is equipped with biological and chemical spill kits, and all members of our iGEM Team are trained to handle Biological and Chemical Spills. Our laboratory is classified as Biosafety Level 2, according to the classification by the Wolrd Health Organisation (WHO) and the Genetic Modification Advisory Committee of the government of Singapore (http://www.gmac.gov.sg/).
Bacterial work and Mammalian cell culture are performed in separate BSL2 Biosafety Cabinets, while DNA work is done on the bench. No cytotoxic reagents are used in our laboratory; Sybr Safe DNA stain is used rather than Ethidium Bromide. Liquid biological waste is decontaminated using 10% Bleach, while Solid biological waste is sent for incineration in a local incineration plant devoted to medical waste (Sembcorp http://www.sembcorp.com/en/business-on-site-services-solid_waste_management.aspx).
Safety Requirements for iGEM Participation
For the fulfillment of requirements for safety from the iGEM foundation, we have submitted the ‘About our lab’ safety forms (https://2015.igem.org/Safety/About_Our_Lab?team_id=1804) and the ‘Final Safety form (Yihan will complete this later to the deadline as currently can’t confirm what we are submitting to parts registry. Just put in first)’
We have also performed a check-in (https://2015.igem.org/Safety/Check_In) for the Biobricks Part (Bba_k299812 http://parts.igem.org/wiki/index.php?title=Part:BBa_K299812), which contains the invasin gene from Yersinia pseudotuberculosis and the listerolysin O gene from Listeria monocytogenes.
Gold
In addition to the Bronze and Silver Medal requirements, your team must convince the judges you have achieved at least two of the following goals:
1. Choose one of these two options: (1) Expand on your silver medal Human Practices activity by demonstrating how you have integrated the investigated issues into the design and/or execution of your project. OR (2) Demonstrate an innovative Human Practices activity that relates to your project (this typically involves educational, public engagement, and/or public perception activities; see the Human Practices Hub for information and examples of innovative activities from previous teams).
2. Help any registered iGEM team from a high-school, different track, another university, or institution in a significant way by, for example, mentoring a new team, characterizing a part, debugging a construct, modeling/simulating their system or helping validate a software/hardware solution to a synbio problem.
3. Improve the function OR characterization of a previously existing BioBrick Part or Device (created by another team, or by your own team in in a previous year of iGEM), and enter this information in the part's page on the Registry. Please see the Registry Contribution help page for help on documenting a contribution to an existing part. This part must not come from your team's 2015 range of part numbers.     Demonstrate a functional prototype of your project. Your prototype can derive from a previous project (that was not demonstrated to work) by your team or by another team. Show this system working under real-world conditions that you simulate in the lab.
Silver
In addition to the Bronze Medal requirements, your team must convince the judges you have achieved the following 3 goals:
1. Experimentally validate that at least one new BioBrick Part or Device of your own design and construction works as expected. Document the characterization of this part in the Main Page section of the Registry entry for that Part/Device. This working part must be different from the part you documented in Bronze medal.
2. Submit this new part to the iGEM Parts Registry. This part must be different from the part you documented in Bronze medal. (Submissions must adhere to the iGEM Registry guidelines.)
3. iGEM projects involve important questions beyond the bench, for example relating to (but not limited to) ethics, sustainability, social justice, safety, security, and intellectual property rights. We refer to these activities as Human Practices in iGEM. Demonstrate how your team has identified, investigated and addressed one or more of these issues in the context of your project.
Bronze
1. Register for iGEM, have a great summer, and attend the Giant Jamboree.
2. Complete the Judging form.     Create and share a Description of the team's project using the iGEM wiki, and document the team's parts using the Registry of Standard Biological Parts.
3. Present a poster and a talk at the iGEM Jamboree.
4. Create a page on your team wiki with clear attribution of each aspect of your project. This page must clearly attribute work done by the students and distinguish it from work done by others, including host labs, advisors, instructors, sponsors, professional website designers, artists, and commercial services.
5. Document at least one new standard BioBrick Part or Device central to your project and submit this part to the iGEM Registry (submissions must adhere to the iGEM Registry guidelines). You may also document a new application of a BioBrick part from a previous iGEM year, adding that documentation to the part's main page.
