<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SPSingapore_Interview_Full skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SPSingapore/Interview Full</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="cssmenu"><UL><LI class="active"><A href="https://2015.igem.org/Team:SPSingapore/"><SPAN>Home</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Team">Overview</A></LI><LI><A href="https://igem.org/Team.cgi?id=1804">Official Profile</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Team Bios">Team Bios</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Mentors">Mentors</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Attributions">Attributions</A></LI></UL><LI><A href="https://2015.igem.org/Team:SPSingapore/Project"><SPAN>Project</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Project">Overview</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Invasin">Invasin + Listerolysin</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/ESAQS">esa Quorum Sensing</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Anaerobic Promoter">Anaerobic Promoter</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Protocol">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Notebook">Entries</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Practices"><SPAN>Human Practices</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Workshop">Workshop</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Workshop Materials">Workshop Materials</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Interview">Consultations</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Safety"><SPAN>Safety</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:SPSingapore/Medals"><SPAN>Medals</SPAN></A></LI></DIV><DIV id="sidemenu" style="float:left"><UL><LI><A href="https://2015.igem.org/Team:SPSingapore/Practices"><SPAN>HUMAN PRACTICES</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Workshop"><SPAN>Workshop </SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Workshop Materials"><SPAN>Workshop Materials</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:SPSingapore/Interview"><SPAN>Consultations</SPAN></A></LI></UL><LI class="last"><A><SPAN style="background-color:midnightblue">&quot;An expert is someone who knows some of the worst mistakes that can be made in his subject and how to avoid them. - Werner Heisenberg</SPAN></A></LI></DIV><DIV id="maincontent"><CENTER><TABLE><TBODY><TR><TD colspan="1" style="box-shadow: 0 0 0; padding:0;border-top:5px white;font-size:15px"><H1> Full Interview Transcript</H1></TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
Introduction
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:30px;padding-top:30px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

These interviews have been edited for clarity and brevity. In both interviews, the interviewers Hee Yan Ting and Soong Yun Ting are referred to generally as SPSingapore. The interviewees from the Cheong lab are Dr Ian Cheong and Adrian Ng, from Temasek Life Sciences Laboratory. In the second interview, we consulted with Dr Matthew Chang and Adison Wong, who are respectively a principal investigator and synthetic biology program manager at the Centre for Life Sciences in the National University of Singapore. 

</DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
Ian Cheong
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:0px;padding-top:0px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px"> </DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:0px;"><A><B>▶ On controlling the system:
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px"><B>SPSingapore:</B> We think a problem can be leaky expression of our control system. How can we test for that?

<B>Prof Cheong:</B> Here the conceptual problem for you is more of an engineering question. What do you want the V-I characteristic of your system to be? Remember voltage-current?



So do you want it to be like diode where when it reaches a certain value, the voltage suddenly just lets everything through? Or do you want it to be like a resistor which slowly comes up? That is something you have to decide. Now based on that, you’re going have to figure out if there are components that would fit that kind of characteristics then translate it into biological entity. Do you what your gene to be expressed exactly at a certain threshold of inducer, or have some little amount of expression which greatly increases when the threshold is reached? Ideally, you would want the first case scenario but you will be faced with the second case in real life where the little amount of expression before threshold will cause you your collateral damage. So you would want something close to the first case as much as possible. If you can estimate where you need that trigger point to be, you can maybe start to figure out how stringent you need that leakage to be.


To measure this you’re going to have to make a variety of promoters, test it in the same condition, for example same cell line, and then select the desired expression characteristics. The key word here is variety.

</DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:0px;"><A><B>▶ On the proposed system:
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px"><B>Prof Cheong:</B> I think the system can be applied in other fields also and this can serve as both a test and another application for your system. While you have chosen to put in cytotoxic elements, you could also choose to put in things of metabolic importance. For example, instead of putting listeriolysin you could put a carbohydrate metabolising enzyme to digest a polymer such as chitin or cellulose. You can test this in a hypoxic environment, maybe in sewage? I mean what we’re trying to integrate is that you’re trying to take in a signal – whether it is it hypoxic or not. If yes, then the bacteria signals itself and expand, make more copies of itself and make effectors.


</DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:0px;"><A><B>▶ On immune response:
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px"><B>SPSingapore:</B> What do you think could be some potential flaws of our proposed cancer therapy? Indeed, our first concern is about immune response.
<B>Prof Cheong:</B> True. When you have bacteria swimming around the blood. It might not even be the bacteria that kills the patient. In general what you’d get is systemic response resulting in IL-2, TNF-α concentration skyrocketing in the patient and then what happens after that is you get systemic leakage in the vessels, blood vessels drops, shock and then death. That generally is what happens when you have a general benign bacteria and they induce a system wide immune response even if they are not making any toxins.
<B>SPSingapore:</B> But what if we (envision to) inject them intratumorally? That would avoid triggering septic shock.
<B>Prof Cheong:</B> If you inject intratumourally, then I think those concerns can be handled. Especially if you say these E. coli are extremely antibiotic sensitive. These bacteria will thrive in the tumour core where they will carry out their anti-cancer effect. The moment the bacteria escapes to other parts of the body and the patient starts having a fever, you can address it with antibiotics.

Another concern may be mutation. What if there’s a mutant in your E. coli that now hyperproduces AHL? Somewhere in your billion bacteria there would be one and that one is going to cause you huge amount of trouble because if it leaks out of the system where it is not inhibited. You might want to consider putting a brake on growth in aerobic conditions. In essence, the therapeutic effect of your bacteria is pretty much dependent on one ratio: the density of bacteria in your tumour versus that in your healthy tissue, right? You would want to have all the bacteria in the tumour, and they becomes kind of handicapped when they come out into aerobic environment. You’re trying to encourage growth in hypoxia, going against the natural gradient of what E. coli naturally does, while putting a break on aerobic growth.


</DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:0px;"><A><B>▶ On horizontal gene transfer:
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px"><B>SPSingapore:</B> One last thing that we heard was gene transfer. This is one thing that FDA also talks about – how do you prevent gene transfer if you do it in a patient?
<B>Prof Cheong:</B> That’s a famous, famous question that never gets answered. FDA always, always raises this up.
So your main issue is basically the antibiotic resistance cassettes… Okay to tell you the truth, the answer to this is that you need to have these elements integrated into the chromosome. You can’t have it as an extrachromosomal element. It’s just gonna share it right? So you have to put all the elements in the chromosome.

Then, you’re going to face the issue of having all these elements in there that could potentially recombine to other bacteria and make a superbug.

I think there are several strategies that can be done to reduce the chance of producing such a bug. First, design wise, the approach would have to be arranging things in such a way that recombination is very unlikely to occur. Because you have iGEM and BioBricks which you can play around like LEGO, I think it can be done.
<B>Adrian:</B> I think the biggest problem with the slippage of the genes to other systems, or other bacteria - I think it would probably happen during transformation where they take up free DNA. So if your E. coli die and then, regardless if it is in the chromosome or the plasmid itself, they can be picked up by another bacterium or bacteriophage. Can you introduce cleavage sites into the gene? So whenever the DNA get out of the bacteria, it will be chopped up by some common enzyme or a bacteriophage.
<B>Prof Cheong:</B> So another idea for you to consider is to think in modular terms. Many of the toxins are in subunits. Separate them, make them far apart and shuffle them. So if a phage clamps on to one, it’s very unlikely that successful transduction and carrying over of a gene to another bacteria strain can happen.

But that is too far down the road. The scope of this project is not to address that right now. The scope of the project is to show that we can make E. coli do something totally unnatural against its very nature.

If you make this platform, make sure you grow it over and over again, make sure everything is still there and not being depleted. Remember that the selection pressure is going to be very high. Just grow it. It will be an achievement if you can actually maintain the bacteria without it actually self-destructing. Beyond the utilitarian goals which is trying to push the technology to see if we can make E. coli an anti-tumour bug, the main thing is that  you can understand something about the basic biology about it that might actually further your aim.

</DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
Matthew Chang
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:30px;padding-top:30px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px"><B>SPSingapore:</B> What possible pitfalls do you see with regards to the control of our circuit system?
<B>Prof Chang:</B> Well, how specific is the promoter to anaerobic condition? You all may be testing its activation under anaerobic condition, but can it be activated by other stimuli? While you cannot possibly check for every stimuli, you can look this up in literature or databases, especially given that E. coli promoters are very well characterised. If you really want to express the downstream enzymes/effectors only under anaerobic condition, you really have to make sure that the promoter cannot be induced by other factors.
<B>Adison:</B> If you still have time, I suggest adding more specific control over your bacteria. A suicide mechanism would be great so that the bacteria can destroy itself after it performs its functions. Also, having only anaerobic condition to activate invasin expression is not enough. You need two or three simultaneous input for the activation, such as the prebiotic sugars that we eat. The patient has to eat these sugars first so that it will be present in the environment for the eventual invasin expression.
<B>SPSingapore:</B> We wanted to construct our system using two plasmids and then transfect them both into the bacteria. What do you think?
<B>Prof Chang:</B> Why not use the same plasmid?
<B>SPSingapore:</B> With lesser things in each plasmid, we think that it will be easier to do molecular cloning. Also, if we were to put all the components into one plasmid, the plasmid will be really large around 10 kilobases (kb), and such a huge size may be hard to perform bacteria transformation.
<B>Prof Chang:</B> I don’t think that would be a problem. Here, we usually handle 10 kb.
<B>Adison:</B> Or if you insist, the dual plasmids system will only be feasible if you use plasmid vectors of different backbone, because two plasmids with the same backbone will outcompete each other during replication. So let’s say if you use pSB1C3 backbone for one construct, then you must use something like pSC4A5 for the other construct.

</DIV></TD></TR></TBODY></TABLE></CENTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>