<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SPSingapore_Anaerobic_Promoter skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SPSingapore/Anaerobic Promoter</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="cssmenu"><UL><LI class="active"><A href="https://2015.igem.org/Team:SPSingapore/"><SPAN>Home</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Team">Overview</A></LI><LI><A href="https://igem.org/Team.cgi?id=1804">Official Profile</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Team Bios">Team Bios</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Mentors">Mentors</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Attributions">Attributions</A></LI></UL><LI><A href="https://2015.igem.org/Team:SPSingapore/Project"><SPAN>Project</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Project">Overview</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Invasin">Invasin + Listerolysin</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/ESAQS">esa Quorum Sensing</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Anaerobic Promoter">Anaerobic Promoter</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Protocol">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Notebook">Entries</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Practices"><SPAN>Human Practices</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Workshop">Workshop</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Workshop Materials">Workshop Materials</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Interview">Consultations</A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Safety"><SPAN>Safety</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:SPSingapore/Medals"><SPAN>Medals</SPAN></A></LI></DIV><DIV id="sidemenu" style="float:left"><UL><LI class="last"><A href="https://2015.igem.org/Team:SPSingapore/Project"><SPAN>PROJECT</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/ESAQS"><SPAN>ESA Quorum Sensing</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Invasin"><SPAN>Invasin + Listerolysin</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:SPSingapore/Anaerobic Promoter"><SPAN>Anaerobic Promoter</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:SPSingapore/Description"><SPAN>Parts</SPAN></A></LI></UL></DIV><DIV id="maincontent"><CENTER><TABLE><TBODY><TR><TD colspan="1" style="box-shadow: 0 0 0; padding:0;border-top:5px white;font-size:15px"><H1>Anaerobic Response System</H1></TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
The Anaerobic Response System
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:30px;padding-top:30px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

High metabolic activity and dysfunctional vasculature often result in oxygen deficiency in solid tumour cores (Smallbone, Gatenby &amp; Maini, 2005). This “hypoxic core” is an anaerobic environment, which contrasts with the aerobic environment in most other regions of the body. Therefore, we propose to use an anaerobic switch so that the target genes will specifically be transcribed at the tumour region in response to the anaerobic environment. 


In the facultative anaerobe <I>Escherichia coli</I> K-12, the transcription factor Fumarate and Nitrate reductase Regulatory (FNR) protein is encoded by the fnr gene (Spiro &amp; Guest, 1987). FNR activates various fnr-dependent promoters in the anaerobic pathway of E. coli (Spiro &amp; Guest, 1987), such as the nirB promoter (pnirB), which drives the expression of NADH-dependent nitrite reductase (Jayaraman et al., 1987). pnirB has partial activity when induced under anaerobic conditions and optimal activity in the presence of nitrite in addition to anaerobic growth conditions (Griffiths &amp; Cole, 1987; Jayaraman et al., 1987). We have selected pnirB as the switch to induce the expression of target genes under anaerobic conditions. As the non-pathogenic <A href="http://www.ncbi.nlm.nih.gov/nuccore/CP010816.1"><I>E. coli</I> K-12 strain BL21 (Life Technologies)</A> that we used expresses the fnr gene endogenously, pnirB will be regulated by FNR protein in this system. 




</DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
Part Design
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:30px;padding-top:30px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

Our aim was to first characterise the different parts used in the above two-switch regulatory system (Figure 1). As such, we designed the following four parts flanked by the BioBricks Prefix (BBPrefix) and BioBricks Suffix (BBSuffix):
<OL><LI>
pnirB-invasin (<A href="http://parts.igem.org/Part:BBa_K1804002">BBa_K1804002</A>)
</LI><LI>
pnirB-invasin-listeriolysin
</LI><LI>
plac-green fluorescent protein (gfp) (<A href="http://parts.igem.org/Part:BBa_K1804001 ">BBa_K1804001</A>)
</LI><LI>
plac-invasin
</LI></OL>

In pnirB-invasin (<A href="http://parts.igem.org/Part:BBa_K1804002">BBa_K1804002</A>), the invasin (inv) gene is placed under the control of pnirB. Similarly, the <A href="https://2015.igem.org/Team:SPSingapore/Invasin">invasin (inv) and listeriolysin O (LLO) genes</A> are placed under the control of pnirB in pnirB-invasin-listeriolysin. As such, bacteria transformed with plasmids encoding pnirB-invasin can invade mammalian cells under anaerobic conditions and bacteria cells transformed with plasmids encoding pnirB-invasin-listeriolysin can invade mammalian cells, as well as escape the endosome.



The constitutive lacI promoter (plac) (Shong &amp; Collins, 2013) driving the expression of invasin (plac-invasin) was chosen as a positive control for pnirB-invasin (<A href="http://parts.igem.org/Part:BBa_K1804002">BBa_K1804002</A>) to show that invasin expressed from pnirB-invasin was functional through an invasion assay. In order to ensure that plac was functional, plac-gfp (<A href="http://parts.igem.org/Part:BBa_K1804001">BBa_K1804001</A>) was constructed and functionally characterised by measuring fluorescence intensities.



Unfortunately, our team only managed to construct and functionally characterise plac-gfp (<A href="http://parts.igem.org/Part:BBa_K1804001">BBa_K1804001</A>) and pnirB-invasin (<A href="http://parts.igem.org/Part:BBa_K1804002">BBa_K1804002</A>).





</DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
Design and Construction of plac-gfp
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:30px;padding-top:30px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

plac-gfp was constructed by prepending the KpnI sequence and appending the XhoI sequence to the start and stop codons of gfp (<A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>) respectively. This was done via extension polymerase chain reaction (PCR). Restriction enzyme digest with KpnI and XhoI was then carried out to replace esaR and esaI with the PCR product in the pAC-esaR-esaI plasmid (Figure 1). This placed gfp under the control of the plac promoter. The resultant pAC-plac-gfp plasmid encoded gfp under the control of plac.

<FIGURE id="fig1"><FIGCAPTION><B>Figure 1 :</B> A. Plasmid map of pAC-EsaR-EsaI plasmid (Addgene plasmid #47660) with restriction sites. B. Plasmid map of pAC-plac-gfp.
</FIGCAPTION></FIGURE>


The BioBricks Prefix and Suffix were then prepended and appended to the start of plac and end of gfp specific primers, respectively, to produce BBPrefix-plac-gfp-BBSuffix by PCR. To increase the percentage of cleavage during restriction digest, primers with <A href="ttps://www.neb.com/~/media/NebUs/Files/Chart%20image/cleavage_olignucleotides_old.pdf">short (referred to as junk) sequences</A> prepended to BBPrefix (FP_BBP_Junk) and appended to BBSuffix (RP_BBS_Junk) were used to amplify BBPrefix-plac-gfp-BBSuffix by PCR. The resultant PCR product Junk-BBPrefix-plac-gfp-BBSuffix-Junk was then cloned into pSB1C3 by restriction digest with EcoRI and PstI followed by ligation and transformation into E. coli BL21 to produce plac-gfp in pSB1C3 (Figure 2). Figure 3 shows the gel electrophoresis of the PCR products formed after PCR of the correct pSB1C3-plac-gfp clone, PCR-purified Junk-BBPrefix-plac-gfp-BBSuffix-Junk (positive control), pAC-plac-gfp (negative control) and water (negative control) using the BBPrefix and BBSuffix primers. Colony PCR was performed with the BBPrefix and BBSuffix primers to confirm if clone was correct.

<FIGURE id="fig2"><FIGCAPTION><B>Figure 2 :</B> Plasmid map of plac-gfp in pSB1C3 vector.
</FIGCAPTION></FIGURE><FIGURE id="fig3"><FIGCAPTION><B>Figure 3 :</B> PCR of a correct pSB1C3-plac-gfp clone (lane 1), PCR-purified Junk-BBPrefix-plac-gfp-BBSuffix-Junk (positive control) (lane 2), pAC-plac-gfp (negative control) (lane 3) and water (negative control) (lane 4) using the BBPrefix and BBSuffix primers. 
</FIGCAPTION></FIGURE></DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
Design of plac-Inv
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:30px;padding-top:30px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

plac-invasin (Figure 3) was intended to be constructed by two rounds of extension PCR of the invasin gene from <A href="http://parts.igem.org/Part:BBa_K299812">BBa_K299812</A>. The first round of primers are FP1_placInv and RP2_invendBBsuffix. The second round of primers are FP2_BBprefixplac and RP2_invendBBsuffix. Unfortunately, due to time constraints, the construction of this part is incomplete.

<FIGURE id="fig3"><FIGCAPTION><B>Figure 4 :</B> Plasmid map of plac-invasin in pSB1C3 vector.
</FIGCAPTION></FIGURE></DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
Design and Construction of pnirB-Inv
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:30px;padding-top:30px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

pnirB-invasin (Figure 5) was constructed by extension PCR of the invasin gene from <A href="http://parts.igem.org/Part:BBa_K299812">BBa_K299812</A> using BBPrefix-pnirB as the forward primer and a reverse primer for the end of invasin fused to BBSuffix (invendBBSuffixLonger) (Figure 6). BBPrefix-pnirB-invasin-BBSuffix was then amplified into Junk-BBPrefix-pnirB-invasin-BBSuffix-Junk using FP_BBP_Junk and RP_BBS_Junk (Figure 7). Junk-BBPrefix-pnirB-invasin-BBSuffix-Junk was then cloned into pSB1C3 by restriction digest with EcoRI and PstI followed by ligation and transformation into E. coli BL21. The clone was confirmed with colony PCR of Biobricks Prefix/Suffix primers and internal primers invlloF1/InvendBBsuffixlonger (Figure 8).

<FIGURE id="fig5"><FIGCAPTION><B>Figure 5 :</B> Plasmid map of plac-invasin in pSB1C3 vector.
</FIGCAPTION></FIGURE><FIGURE id="fig6"><FIGCAPTION><B>Figure 6 :</B> PCR of the template plasmid BBa_K299812 was performed with BBPrefixpNirBinv and invendBBSuffix to add the NirB promoter to the invasin fragment. Lanes 1-3 show the 2.9kb product, while lane 4 is the 1kb ladder.
</FIGCAPTION></FIGURE><FIGURE id="fig7"><FIGCAPTION><B>Figure 7 :</B> PCR of the 2.9kb product to add short overhangs for Restriction digest to aid cloning. 1kb ladder Lane 1-7 PCR of 2.9kb fragment from Figure 1 with Junk_BBPrefix and BBSuffix_Junk primers 1kb ladder.
</FIGCAPTION></FIGURE><FIGURE id="fig8"><FIGCAPTION><B>Figure 8 :</B> Colony PCR with Biobricks Suffix/prefix primers, and internal primers to confirm if the clone contains the pnirb-invasin insert in vector.1kb ladder Lane 1-2 Colony PCR of colony 4 with negative control of water for Biobricks Prefix/Suffix primers Lane 3-4 Colony PCR of colony 4 with negative control of water for invlloF1/InvendBBsuffixlonger
</FIGCAPTION></FIGURE></DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
Results and Discussion
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:0px;padding-top:0px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px"> </DIV></TD></TR><TR><TD style="font-size:16px;padding-bottom:0px;"><A><B>▶ plac promoter is constitutively active in E. coli BL21
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

The plac promoter is constitutively active in our strain of E. coli BL21. After construction of the <A href="http://parts.igem.org/Part:BBa_K1804001">pSB1C3-plac-gfp</A> vector, the activity of the plac promoter was measured by quantification of fluorescence intensity. BL21 was used as the negative control and BL21 transformed with pAC-plac-gfp as the positive control. As seen in Figure 5, GFP is expressed in both strains of BL21 containing the plac-gfp vectors but not in the negative control. When fluorescence data was quantified in arbitrary units (A.U.), the intensity of plac-gfp in pSB1C3 was lower relative to that of plac-gfp in the pAC vector (Figure 9). Nonetheless, our results indicated that plac is indeed able to drive gene expression under normal growth conditions. Supplementary Table 1 shows the descriptive statistics of the fluorescence intensities determined.

<FIGURE id="fig9"><FIGCAPTION><B>Figure 9 :</B> Representative images of bacterial cultures were taken using a confocal laser scanning microscope at 488 nm, in duplicates A. BL21 pAC-plac-gfp, positive control. B. BL21 pSB1C3-plac-gfp. C. BL21, negative control.
</FIGCAPTION></FIGURE><FIGURE id="fig10"><FIGCAPTION><B>Figure 10 :</B> GFP is expressed in BL21 transformed with pAC-placgfp and pSB1C3. Data is represented in Mean +/- Standard Deviation of GFP fluorescence intensity (A.U.) for the positive control of BL21 pAC-plac-gfp, BL21 pSB1C3-plac-gfp (BBa_K1804001) and the negative control BL21. The experiment was carried out in duplicates (n=15).
</FIGCAPTION></FIGURE><FIGURE id="table1"><FIGCAPTION><B>Supplementary Table 1</B></FIGCAPTION></FIGURE></DIV></TD></TR><TR><TD style="font-size:16px;padding-bottom:0px;"><A><B>▶ Invasion Assay for pnirB-invasin in pSB1C3 (BBa_K1804002)
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">
The invasin protein from Yersinia facilitates bacterial invasion into mammalian cells via an integrin dependent mechanism (Leong et al., 1990). We induced the expression of invasin by growing B21 carrying pSB1C3-pnirB-invasin under anaerobic conditions. After 2 and 6 hours, the bacteria were allowed to invade a monolayer of HepG2 mammalian cells in an invasion assay, where results are represented in colony-forming units (CFU) of bacteria successful in invading mammalian cells (Figure 7). Our results show that BL21, and BL21 with pSB1C3-nirB-invasin grown under aerobic conditions have low invasion ability. BL21 has low invasion ability under both aerobic and anaerobic conditions . However, BL21 with pSB1C3-pnirB-invasin exhibits a 25-fold increase in invasion compared to the control grown at aerobic conditions, and this increase is statistically significant (p-value &lt; 0.05) (Figure 11). It is notable that after 6 hours of growth in anaerobic conditions, the pnirB-invasin construct can no longer confer an increased invasive phenotype, which may be due to the high fitness cost of producing the transgenic invasin protein.


<FIGURE id="fig11"><FIGCAPTION><B>Figure 10 :</B> Invasive ability of BL21, and BL21 with pSB1C3-pnirB-invasin. Mammalian cell invasion assay was conducted for BL21 and BL21 with pSB1C3-pnirB-invasin. The data was analysed using Graphpad Prism 6.0, and unpaired t-test was conducted for the above groups. * denotes p-value less than 0.05.
</FIGCAPTION></FIGURE></DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
Conclusions
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:30px;padding-top:30px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

From our experiments, we have demonstrated that the lacI promoter is able to drive the expression of a target gene (in this case, gfp) in pAC and pSB1C3. It also suggested that pnirB is able to control the invasion phenotype conferred by invasin expression in BL21 to some degree. However, there are some limitations, as we have not fully characterised the optimal conditions for activating the pnirB promoter. It is also unknown if the expression of the invasin gene results in high fitness cost that might affect the ability of transformed bacteria to serve as a drug delivery vehicle. This invasion phenotype can be further characterised with the construction of the pSB1C3-plac-invasin vector.



</DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
References
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:30px;padding-top:30px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

Anderson, J. C., Clarke, E. J., Arkin, A. P., &amp; Voigt, C. A. (2006). Environmentally controlled invasion of cancer cells by engineered bacteria.Journal of molecular biology, 355(4), 619-627.

Griffiths, L., &amp; Cole, J. A. (1987). Lack of redox control of the anaerobically-induced nirB+ gene of Escherichia coli K-12. Archives of microbiology, 147(4), 364-369.

Jayaraman, P. S., Peakman, T. C., Busby, S. J. W., Quincey, R. V., &amp; Cole, J. A. (1987). Location and sequence of the promoter of the gene for the NADH-dependent nitrite reductase of Escherichia coli and its regulation by oxygen, the Fnr protein and nitrite. Journal of molecular biology, 196(4), 781-788.

Jayaraman, P. S., Gaston, K. L., Cole, J. A., &amp; Busby, S. J. W. (1988). The nirB promoter of Escherichia coli: location of nucleotide sequences essential for regulation by oxygen, the FNR protein and nitrite. Molecular microbiology, 2(4), 527-530.

Leong, J. M., Fournier, R. S., &amp; Isberg, R. R. (1990). Identification of the integrin binding domain of the Yersinia pseudotuberculosis invasin protein. The EMBO journal, 9(6), 1979.

Shong, J., &amp; Collins, C. H. (2013). Engineering the esaR promoter for tunable quorum sensing-dependent gene expression. ACS synthetic biology, 2(10), 568-575.

Spiro, S., &amp; Guest, J. R. (1987). Regulation and over-expression of the fnr gene of Escherichia coli. Journal of general microbiology, 133(12), 3279-3288.

Smallbone, K., Gavaghan, D. J., Gatenby, R. A., &amp; Maini, P. K. (2005). The role of acidity in solid tumour 
growth and invasion. Journal of Theoretical Biology,235(4), 476-484.

</DIV></TD></TR><TR><TD style="font-size:20px;padding-bottom:10px;"><A><B>
Materials and Methods
</B></A></TD></TR><TR style="border-top:2px solid grey;"><TD style="padding-bottom:0px;padding-top:0px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px"> </DIV></TD></TR><TR><TD style="font-size:16px;padding-bottom:0px;"><A><B>▶ PCR and Cloning
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

Extension PCR was carried out to amplify target parts with specific flanking sequences, followed by deoxyribonucleic acid (DNA) gel electrophoresis to check for the amplification of the target parts. After the required part was created, restriction enzyme digest, ligation and transformation into bacteria (E. coli BL21 or dh5α) were carried out. The bacteria were then plated onto LB agar with the appropriate antibiotic and grown overnight in 37 ℃ in a stationary incubator. Colony PCR or PCR of miniprepped plasmids were then conducted to select for positive clones. For minipreps, bacteria was inoculated into LB media containing the appropriate antibiotic and grown overnight at 37 ℃ with constant agitation at 250 revolutions per minute (rpm).



</DIV></TD></TR><TR><TD style="font-size:16px;padding-bottom:0px;"><A><B>▶ Fluorescence Intensity Quantification
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

Green fluorescence protein (GFP) fluorescence intensity quantification was carried out after growth of transformed cells expressing GFP in LB media with the appropriate antibiotic. The optical density (OD) of the bacteria was then measured using a spectrophotometer (UV-1800 Shimadzu Spectrophotometer) at 600 nm and diluted to an OD of 0.5. The bacteria were then visualised under a confocal laser scanning microscope (Confocal Leica SPE2). The wavelength of the laser used was 488 nm, and the fluorescent signal detection range was between 500 and 540 nm, corresponding to the wavelength of the light emitted by GFP. Each experiment was carried in duplicates for the sample, positive control, and negative control.

The GFP fluorescence intensities were measured using ImageJ (Figure 12). For each group (sample, positive control and negative control), 15 bacteria cells were selected randomly from at least two images and their fluorescence intensities quantified, unless the fluorescence intensity was too low to be detected.  Graphpad Prism 6.0 was used to analyse the fluorescence intensities.

<FIGURE id="fig12"><FIGCAPTION><B>Figure 12 :</B> GFP fluorescence intensity quantification using ImageJ. The contours of bacteria cells were defined by the lowest intensity surrounding pixel.
</FIGCAPTION></FIGURE></DIV></TD></TR><TR><TD style="font-size:16px;padding-bottom:0px;"><A><B>▶ Mammalian Cell Invasion Assay
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

Bacteria were grown in Luria Broth (LB) at 37 ℃ with shaking at 250 rpm, while mammalian cells were maintained in Dulbecco’s Modified Eagle’s Medium (DMEM) supplemented with fetal bovine serum (FBS) in a humidified CO2 incubator. HepG2 cells were seeded in 24-well plates at a concentration of 2.5 x 105 cells/well and grown overnight. Stationary cultures of E. coli BL21 and BL21 + pSB1C3-pnirB-invasin were prepared by inoculating one colony in 3 mL LB and grown overnight. Bacteria were then diluted 10X in LB, and grown for 2 hours and 6 hours at 30 degrees with shaking at 250 rpm under anaerobic conditions in a BD GasPak EZ Incubation Container with activated Gazpak, and under aerobic conditions for 6 hours. The O.D. values of the cultures were then measured.

The cells in the 24-well plates were washed with 500 ul sterile phosphate buffered solution (PBS). Following which, 500 ul of fresh antibiotic-free DMEM with FBS was added into each well. The bacteria cells were then diluted in DMEM, and used to infect the cells with a multiplicity of infection of 100:1 for three hours in the CO2 incubator. Subsequently, the cells were washed with PBS, and DMEM with 1000 ug/mL kanamycin was added for a brief incubation of 1 hour in the CO2 incubator. Each well was then washed twice with PBS, and 100 ul of 0.1% Triton X-100 was subsequently added to lyse the cells. Serial dilutions of the lysate were carried out and plated on LB agar. After overnight growth at 37 ℃, the number of colonies on each plate was counted and colony forming units was calculated. Statistical tests were conducted using GraphPad Prism 6.0.

For further details on our experimental protocols and daily lab work, please refer to our <A href="https://2015.igem.org/Team:SPSingapore/Protocol">Protocols page</A> and <A href="https://2015.igem.org/Team:SPSingapore/Notebook">Notebook page </A>. 


</DIV></TD></TR><TR><TD style="font-size:16px;padding-bottom:0px;"><A><B>▶ Sources of Templates
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

The gfp gene was obtained from basic <A href="http://parts.igem.org/Part:BBa_E0040">BioBricks Part BBa_E0040</A>.

The plac sequence 5’ TTTACACTTTATGCTTCCGGCTCGTATGTT 3’ was obtained from the pAC-EsaR-EsaI plasmid (Figure 1) (Shong &amp; Collins, 2013), which was a gift from Cynthia Collins (Addgene plasmid # 47660). 

The pnirB sequence, which is 5’ CATTAAGGAGTATAAAGGTGAATTTGATTTACATCA
ATAAGCGGGGTTGCTGAATCGTTAAGGTAGGCGGTA 3’, was obtained from Jayaraman et al., 1988. 

</DIV></TD></TR><TR><TD style="font-size:16px;padding-bottom:0px;"><A><B>▶ Primer Sequences
</B></A></TD></TR><TR><TD style="padding-bottom:30px;padding-top:20px;padding-left:60px;padding-right:60px"><DIV style="text-align:justify;font-size:14px">

The sequences of the primers of our own design with start and end sites with respect to the submitted and theoretical parts used can be <A href="https://docs.google.com/spreadsheets/d/1H_dUvTqyt6uQRbFmEoEHDfufVBxGp7KuQEu2aNMYqZc/edit#gid=0" target="_blank">downloaded here</A>.


</DIV></TD></TR></TBODY></TABLE></CENTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>