<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SPSingapore_home skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SPSingapore/home</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Classic parallax page</TITLE></P><DIV id="bodyWiki"><NAV style="z-index:15"><UL style="z-index:15"><LI><A href="#home">Home</A></LI><LI><A href="#team">Team</A></LI><LI><A href="#project">Project</A></LI><LI><A href="#protocol">Protocol</A></LI><LI><A href="#part">Part</A></LI><LI><A href="#notebook">Notebook</A></LI><LI><A href="#human_practice">Human Practice</A></LI><LI><A href="#safety">Safety</A></LI><LI><A href="#requirement">Requirement</A></LI></UL></NAV><DIV id="skrollr-body"><DIV class="content" id="home">
Home
</DIV><DIV class="content content-full bigDiv" id="team" height="650px"><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TD width="39%"><DIV id="kenneth" class="memberPicture"><P><B>Kenneth Lim Kun Ming</B></P><P>
Research Interest: Bioinformatics, Genetic analysis
Random fact: Is apparently Schrˆdinger's Biologist
</P></DIV><DIV id="clarice" width="100%" height="auto"><P><B>Clarice Hong Kit Yee</B></P><P>Research Interest: genetics, RNA, cancer
Past research projects: (can't rmb, tell you later)
</P><P>Current project: Differential roles of SALL4A and SALL4B in HCC
</P></DIV><DIV id="chi_yan" width="100%" height="auto"><P><B>Wong Chi Yan</B></P><P>Research Interest: Microbiology, molecular biology, proteomics </P><P>Past research project: Genetic studies on Salmonella biofilms </P><P>Current project: Role of fumarase and cysteine dehydrogenase in DNA damage response </P><P>Random fact: Likes statistics and playing volleyball :)</P></DIV><DIV id="xin_yi" width="100%" height="auto"><P><B>Yeo Xin Yi</B></P><P>Research Interests: Neurobiology, Neurosciences </P><P>Past research projects: Role of STAT in neuroinflammation and the pathogenesis of Alzheimer's Disease</P><P> Current research projects: Synaptic plasticity threshold in hippocampal CA1 pyramidal neurons, Role of WNK1 in neuronal survival and development </P><P>Random fact: Blah ~</P></DIV><DIV id="yi_han" width="100%" height="auto"><P><B>Tan Yi Han</B></P><P>Research Interests: Pathogens, Immunology, Synthetic biology </P><P>Past research projects: Genetic studies on plant pathogenic fungi, Drug screening for Acute Lymphoblastic Leukemia </P><P>Current project: Characterisation of Klebsiella pneumoniae isolates from liver abscess </P><P>Random fact: Knits and bakes in spare time. =)</P></DIV><DIV id="yan_ting" width="100%" height="auto"><STRONG>Hee Yanting</STRONG><P>Research Interests: RNA, genomics, epigenetics</P><P>Past research projects: microRNA as a potential therapeutic strategy for colorectal cancer, Characterising LPA1 antagonists using calcium imaging</P><P>Current project: The role and targeting of EZH2 in lymphoma</P><P>Random fact: Plays the erhu and self-learning cello and classical guitar</P></DIV><DIV id="adrian" width="100%" height="auto"><P><STRONG>Adrian Tan Hong Ji
</STRONG></P><P>Research Interest: Genetic Engineering, Cancer, Immunology</P><P> Random fact: 500 Hours in Terraria </P></DIV><DIV id="yun_ting" width="100%" height="auto"><P><B>Soong Yun Ting</B></P><P>Research Interests: Proteomics,
Past research projects: Genetic studies on Salmonella biofilms </P><P>Current project: Identification of protein players in metastasis </P><P>Random fact: Plays the harmonica </P></DIV><DIV id="duy" width="100%" height="auto"><P><STRONG>Nguyen Duy</STRONG></P><P>Research Interest: Pharmacoinfomatics, Bioinformatics, Neurosciences </P><P>Past research project: Genetic linkage analysis of asthma</P><P> Current project: Neurodegeneration of Drosophila Melanogaster. </P><P>Random fact: 0 hour in Terraria </P></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="content content-full" id="mentor" style="height:150%"><TABLE width="100%" border="2"><TBODY><TR><TD width="33%"><P><STRONG>Leslie Gapter</STRONG></P><P>Dr. Leslie is trained as a molecular biologist and her dissertation focused on breast development and tumorigenesis. Leslie joined NUS in 2005 and her past research has focused on analyzing botanical products for breast and prostate cancer treatment. </P><P>In 2008, Leslie became a full time scientific writer at the Mechanobiology Institute, Singapore, before moving into her current position as a Lecturer in 2010. Leslie teaches 'The Cell' module, which examines the universal mechanics and functions of cells from an integrated science perspective, for the Special Program in Science.</P></TD><TD width="33%"><P><STRONG>Linda J Kenney</STRONG></P><P>Dr Kenney is a Professor of Microbiology at the University of Illinois-Chicago. Her laboratory studies two-component systems in bacteria that control gene expression at a single cell and nanometer level.</P></TD><TD width="33%"><P><STRONG>Stuti Desai </STRONG></P><P>She joined the Kenney group in May, 2012 with a strong urge to amalgamate her doctoral training in studying silent genetic systems in enterics to decipher the behavior of bacteria under various environmental challenges. She obtained her doctorate from the Indian Institute of Science, Bangalore, India, under the guidance of Prof Subramony Mahadevan. She studied Biochemistry for my Master's degree and Chemistry, Physics and Zoology for her Bachelor's degree at the Maharaja Sayajirao University of Baroda, Baroda, India. </P></TD></TR></TBODY></TABLE></DIV><DIV class="content content-full bigDiv" id="project" style="height:500px; overflow:scroll"><DIV id="accordionProject"><H3>Introduction</H3><DIV><P>
For description
</P></DIV><H3>Esa Quorum sensing</H3><H3>Invasin + lysteriolysin</H3><H3>FNR hypoxic promoter</H3><H3>Design</H3><DIV><P>
For design
</P></DIV></DIV></DIV><DIV class="content content-full bigDiv" id="protocol" style="height:600px; overflow:scroll"><DIV id="protocolTabs"><UL><LI><A href="#cutSmartBuffer">Restriction Digest with NEB enzymes</A></LI><LI><A href="#PCRPurification">PCR Purification</A></LI><LI><A href="#transfectionExpression">Transfection-Expression</A></LI><LI><A href="#cloning">Cloning</A></LI><LI><A href="#Cell_Number_and_Viability_Determination">Cell Number and Viability Determination</A></LI><LI><A href="#Freezing_down_cells">Freezing down cells</A></LI><LI><A href="#Making_Bacterial_Media">Making Bacterial Media</A></LI><LI><A href="#Plasmid_Making">Miniprep of plasmid DNA</A></LI><LI><A href="#Subculturing_for_adherent_cell_lines">Subculturing for adherent cell lines</A></LI><LI><A href="#Ligation_Protocol_with_DNA_ligase">Ligation Protocol with T4 DNA ligase</A></LI><LI><A href="#Antibiotic_Preparation">Antibiotic Preparation</A></LI><LI><A href="#Agarose_Electrophoresis">Agarose Electrophoresis</A></LI></UL><DIV id="cutSmartBuffer"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/1/14/SPSingapore_Protocols_CutSmartBuffer.pdf" target="_blank">Restriction Digest with NEB enzymes</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Adjust Water bath to 37 degrees.</TD><TD>Electric shock, heat shock</TD><TD>Possibility of electric shock</TD><TD>wear proper PPE (gloves, lab coat, covered shoes)</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Add aliquots of DNA to water and Cut Smart buffer in a 1.5mL centrifuge tube.</TD><TD>Biological exposure</TD><TD>Spillage of buffer and DNA</TD><TD>wear proper PPE (gloves, lab coat, covered shoes)</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Lightly centrifuge the centrifuge tube in a microcentrifuge. Add an appropriate volume of NEB restriction enzymes and centrifuge again.</TD><TD>Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers</TD><TD>Injury due to imbalanced centrifuge and trapping of limbs or fingers</TD><TD>Internal training is compulsory.wear proper PPE (gloves, lab coat, covered shoes); handle cells in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand. When using the centrifuge, ensure centrifuge is balanced and rotor is placed correctly, and that all tubes are capped tightly. Close centrifuge properly and ensure that there are no funny sounds when centrifuge is running.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>4</TD><TD>Place 1.5mL tube into waterbath and digest for 2 hours at 37 degrees. When Restriction digest is done, take thetube out and stop reaction by adding Purple loading dye and cooling on ice.</TD><TD>Electric shock, heat shock</TD><TD>Spillage of buffer and DNA</TD><TD>wear proper PPE (gloves, lab coat, covered shoes)</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="PCRPurification"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/b/b7/Vilnius15_PCRPurification.pdf" target="_blank">PCR Purification</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Adjust Water bath to 55 degrees to preheat.</TD><TD>Electric shock, heat shock</TD><TD>Possibility of electric shock</TD><TD>wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Excise gel slices with DNA of the correct band into a 1.5mL centrufuge tube, add an appropriate volume of DNA binding buffer and melt the gel slice by incubation in the water bath at 55 degrees. Add a equal volume of DNA binding buffer to a completed pcr reation.</TD><TD>Biological exposure</TD><TD>Spillage of buffer and DNA</TD><TD>wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Add the melted agarose in DNA binding buffer or pcr reaction and DNA binding buffer to DNA bidning columns resting in 1.5mL tubes. Centrifuge the tube in a microcentrifuge and allow the DNA mixtiure to flow through.</TD><TD>Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers</TD><TD>Injury due to imbalanced centrifuge and trapping of limbs or fingers</TD><TD>wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>4</TD><TD>Add wash buffer to wash through the colomns and precipitate DNA. Centrifuge for 1 min, then pour out the remaining wash buffer in the tube. Centrifuge for a further 5 minutes to dry the coloumn.</TD><TD>Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers</TD><TD>Injury due to imbalanced centrifuge and trapping of limbs or fingers</TD><TD>wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>5</TD><TD>Add 30-50ul of nuclease free water to the column, and incubate the columns in the 55 degrees water bath for 5 minutes. Spin down the DNA and quantify concentration using the Thermo Scientific nanodrop.</TD><TD>Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers</TD><TD>Injury due to imbalanced centrifuge and trapping of limbs or fingers</TD><TD>wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="transfectionExpression"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/4/42/SPSingapore_ProtocolsChemicallyCompetentCell.pdf" target="_blank">TransfectionExpression</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Transferring aliquot of competent E.coli (dH5a or BL21 from Life Technologies) from  -80 C to thaw on ice with an appropriate aliquot of plasmid (100ng) for 30min.</TD><TD> Skin contact with extremely cold materials; biological exposure</TD><TD>Cold burn injury; handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested</TD><TD>Internal training is compulsory. Handle materials from -80 C using insulated / thermal gloves (or double layer latex gloves); wear proper PPE (safety goggles, lab coat, covered shoes); transfer samples in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.</TD><TD>2</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Adjust water bath to 42 degrees.</TD><TD>Electric shock, heat burns</TD><TD>Possibility of electric shock</TD><TD>wear proper PPE (gloves, lab coat, covered shoes)/</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Heatshock bacteria for 30 seconds exactly at 42degrees</TD><TD>Biological exposure, Electric shock, heat burns</TD><TD>Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested</TD><TD>Internal training is compulsory. Conduct routine bacteria work in a Class I Biohazard Safety Cabinet. Wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90. Have training on how to manage biological spill with appropriate spill kit reagetns (10% bleach or 70% Ethanol)</TD><TD>1</TD><TD>2</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>4</TD><TD>Reviive bacteria at 37degrees with shaking for 1 hour.</TD><TD>Biological exposure to bacteria</TD><TD>Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested</TD><TD>Internal training is compulsory. Conduct routine bacteria work in a Class I Biohazard Safety Cabinet. Wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90. Have training on how to manage biological spill with appropriate spill kit reagetns (10% bleach or 70% Ethanol)</TD><TD>1</TD><TD>2</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>5</TD><TD>Plate bacteria on LB+ appropriate antibiotic for selection, grow overnight in a small bacteria incubator. </TD><TD>Biological exposure to bacteria</TD><TD>Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested</TD><TD>Internal training is compulsory. Conduct routine bacteria work in a Class I Biohazard Safety Cabinet. Wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90. Have training on how to manage biological spill with appropriate spill kit reagetns (10% bleach or 70% Ethanol)</TD><TD>1</TD><TD>2</TD><TD>2</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="cloning"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/7/75/SPSingapore_Protocols_Cloning.pdf" target="_blank">Cloning</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Transferring bacteria from glycerol stock stored @ -80 C to plastic culture tubes</TD><TD> Skin contact with extremely cold materials; biological exposure</TD><TD>Cold burn injury; handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested</TD><TD>Internal training is compulsory. Handle materials from -80 C using insulated / thermal gloves (or double layer latex gloves); wear proper PPE (safety goggles, lab coat, covered shoes); transfer samples in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.</TD><TD>2</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Transferring culture samples to glassware </TD><TD> Breakage of glassware; biological exposure</TD><TD>Injury from broken glassware; handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested</TD><TD>Internal training is compulsory. Handle glassware over the lab bench (or in the biosafety cabinet); wear proper PPE (gloves, lab coat, covered shoes); transfer samples in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.</TD><TD>1</TD><TD>2</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Routine maintenance of bacteria culture</TD><TD>Biological exposure, breakage of glassware</TD><TD>Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested</TD><TD>Internal training is compulsory. Conduct routine bacteria work in a Class I Biohazard Safety Cabinet. Wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90.</TD><TD>1</TD><TD>2</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>4</TD><TD>Single day experiements using bacteria culture (students)</TD><TD>Biological exposure; breakage of glassware</TD><TD>Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested</TD><TD>Internal training is compulsory. Conduct routine bacteria work at the lab bench over absorbant materials; wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90.</TD><TD>1</TD><TD>2</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>5</TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD>0</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="Cell_Number_and_Viability_Determination"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/b/b3/SPSingapore_Protocols_Cell_Number_and_Viability_Determination.pdf" target="_blank">Cell Number and Viability Determination</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Cleaning the glass slide with detergent.</TD><TD> 1) Breakage of the glass slide; 2)naturally sharp edges on the glass slide.</TD><TD>Injury from broken slide or sharp edges</TD><TD>Handle glassware over the lab bench; wear proper PPE (gloves, lab coat, covered shoes); the usual precautions outlined in the &quot;Use of Laboratory Glassware&quot; SOP must be taken.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Transferring sample to the glass slide.</TD><TD> Breakage of glass slide.</TD><TD>Injury from broken slide.</TD><TD>Handle glassware over the lab bench; wear proper PPE; the usual precautions outlined in the &quot;Use of Laboratory Glassware&quot; SOP must be taken.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Overlay sample with a glass cover slip.</TD><TD>Dropping the cover slip onto the bench top or floor.</TD><TD>Breaking the glass coverslip when retrieving it &amp; subsequent injury.</TD><TD>Wear proper PPE; use a brush and dustpan to retrieve the glass coverslip (do not attempt to use the coverslip- discard it into the glass waste).</TD><TD>1</TD><TD>2</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>4</TD><TD>Transferring the glass slide to/from the microscope for viewing.</TD><TD>1) finger pinched (or cut) by the stage clip; 2) breakage of glass slide</TD><TD>Injury to fingers (pinch or cut).</TD><TD>1) Use both hands to position the slide, wear proper PPE; 2) handle glassware over the lab bench and use the precautions as outlined in the &quot;Use of Laboratory Glassware&quot; SOP.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>5</TD><TD>Looking through the eye piece to observe samples.</TD><TD>1) Accidental knocking of the one's head onto the eye piece; 2) backlight too intense </TD><TD>Injury of the eye.</TD><TD>1) Position hands around the eye pieces to help serve as a guide for lowering your head and eyes; 2) before viewing the sample, reduce the backlight to zero, then look into the eyepieces and raise the light intensity.</TD><TD>2</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>6</TD><TD>Leaving the backlight on to view the sample.</TD><TD>Overheating of microscope parts due to the backlight.</TD><TD>Slight burns upon skin contact with the heated light element.</TD><TD>Switch off backlights whenever not in use.</TD><TD>2</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>7</TD><TD>(Optional) Viewing fluorescence using the mercury lamp (aka Intensilight)  </TD><TD>Physical hazard- Mercury lamps release extremely potent and visible UV radiation.</TD><TD>Injury of the eye and skin.</TD><TD>Avoid looking at the microscope stage and slide directly- always view them through the UV light shield; note the number of hours the mercury lamp has been used (shown on the unit itself)- going beyond its expected lifespan (~2000 hrs) is an increased fire risk (see Nikon Intensilight manual).</TD><TD>2</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="Freezing_down_cells"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/8/8d/SPSingapore_Protocols_Freezing_down_cells.pdf" target="_blank">Freezing down cells</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Grow your cells in a large plate (150 mm) to a confluent monolayer.</TD><TD>Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,</TD><TD>Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury</TD><TD>All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.</TD><TD>1</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Prepare 10 ml of cryo media. This is just 95% FBS and 5% DMSO. In a 15 ml tube, add 10 ml FBS and 0.555 ml DMSO. Mix well.</TD><TD>Spillage and exposure to DMSO which is an irritant</TD><TD>Eye contact with DMSO may cause blurry vision, burning sensation, redness, tearing, and vasodilation.</TD><TD>Wear proper PPE (goggles, gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; dispose of any spillage using the Chemical Spill SOPs.</TD><TD>1</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>You will need to wash with ~20 ml of media, and use 5 ml of trypsin (3 ml media, 2 ml trypsin). Collect the trypsin/cells into a 50 ml tube (instead of 15 ml, using 5 ml serological pipette), and wash the plate with 20 ml of serum-containing media. After you remove the media from your cells, resuspend the pellet in the 10 ml of cryo media. Aliquot 1 ml of this into each of the 10 cryotubes.</TD><TD>Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens, injury due to improper usage of centrifuge and fingers</TD><TD>Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury. Injury due to imbalanced centrifuge and trapping of limbs or fingers</TD><TD>Internal training is compulsory for centrifuge use and the centrifuge key.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand. When using the centrifuge, ensure centrifuge is balanced and rotor is placed correctly, and that all tubes are capped tightly. Close centrifuge properly and ensure that there are no funny sounds when centrifuge is running. Decontaminate waste using Presept tablets (dilute to 10% solution of activaed bleach to kill cells) and let the bleach decontaminate for half an hour before discarding with plenty of water and dilution.</TD><TD>1</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>4</TD><TD>Put the tubes into the Mr. Frosty and place in the -80ºC freezer. Keep in the freezer for 24 – 48 hours.</TD><TD>Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,</TD><TD>Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury</TD><TD>All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.</TD><TD>1</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="Making_Bacterial_Media"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/5/5d/SPSingapore_Protocols_Making_Bacterial_Media.pdf" target="_blank">Making Bacterial Media</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Weigh out Lennox Broth/Miller Broth solid: 15g/L, add 10g/L agar for (1%) agar. (From scratch, weigh 1% peptone, 0.5% yeast extract, and 1% NaCl. ( NaCl 10 g/L, Peptone 10 g/L,  Yeast extract 5 g/L) LB broth suggested to be pH 7-7.2 (adjust with 1N NaOH). ). Add Bacterial agarose for 10g/L agar (1%).</TD><TD>Spillage of media</TD><TD>Accidental inhalation or ingestion of media powder. </TD><TD>wear proper PPE (gloves, lab coat, covered shoes).</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Add Milliq water to media bottles. Cap tightly and shake throughly, making sure that the solid is all well distributed in solution. Loosen caps by a full turn and place a piece of dog-eared autoclave tape on the bottle. Never autoclave flammable liquids or tightly capped bottles.</TD><TD>Spillage of media</TD><TD>Accidental inhalation or ingestion of media powder. </TD><TD>wear proper PPE (gloves, lab coat, covered shoes).</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Turn on the autoclave. Ensure that enough water is in the bottle, and in the bottom of autoclave, drain bottle empty, drain valve closed. Top up with Milliq water as required.Turn the autoclave machine to agar sterilisation mode, place balanced autoclave baskets of media bottles in the autoclave. Do not close the autoclave machine by hand. Use machine controls to close machine and start heating cycle.</TD><TD>Electric shock, heat burns, explosion</TD><TD>Heat or electrical burns from autoclave.</TD><TD>wear proper PPE (gloves, lab coat, covered shoes).</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>After the autoclave cycle is over and has cooled down, press the open button on the machine and let machine open and cool for 15 minutes. Using heatproof gloves, lift the autoclave basket onto trolley and push trolley into main lab area. Allow the agar to cool overnight before capping tightly and sealing with parafilm.</TD><TD>Electric shock, heat burns from steam</TD><TD>Heat burns from hot steam</TD><TD>wear proper PPE (goggles, gloves, lab coat, covered shoes). Stand back when autoclave opens and allow steam to escape and machine to cool for 15min before removing baskets. Use heatproof gloves to handle autoclave baskets.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="Plasmid_Making"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/3/31/SPSingapore_Protocols_Plasmid_Making.pdf" target="_blank">Miniprep of Plasmid DNA</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Inoculate bateria in 3mL of LB+antibiotic to grow for 16 hours at 37 degrees in a shaking incubator.</TD><TD>Biological exposure, breakage of glassware</TD><TD>Handling of bacteria outside of a Class I Biohazard Safety Cabinet risks generating aerosol particles which may be inhaled or ingested</TD><TD>Internal training is compulsory. Conduct routine bacteria work in a Class I Biohazard Safety Cabinet. Wear suitable PPE (e.g. lab coat, covered shoes, gloves); decontaminate equipment and/or small spills with 70% ethanol or DeCon90.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Pellet Bacteria in 1.5mL tubes in a microcentrifuge at maximum speed for 5 minutes. Pour away the LB media into a waste containter containing 10% bleach.</TD><TD>Biological exposure</TD><TD>Spillage of bacteria culture.</TD><TD>wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Resuspend the bacterial cells in Resuspension solution, vortexing lightly to ensure pellet is fully resuspended in solution. Add Lysis buffer and incubate till solution is clear and there is no cloudy bacteria for 2 minutes. Add Neutralisation solution and invert 4-6 times to mix immediately.</TD><TD>Biological exposure</TD><TD>Spillage of bacteria culture, and buffers</TD><TD>wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>4</TD><TD>Pellet the lysed bacteria by centrifugation for  5 minutes. Run the flow through in DNA binding columns.</TD><TD>Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers</TD><TD>Injury due to imbalanced centrifuge and trapping of limbs or fingers</TD><TD>wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>5</TD><TD>Add wash buffer to wash through the colomns and precipitate DNA. Centrifuge for 1 min, then pour out the remaining wash buffer in the tube. Centrifuge for a further 2 minutes to dry the coloumn.</TD><TD>Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers</TD><TD>Injury due to imbalanced centrifuge and trapping of limbs or fingers</TD><TD>wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>6</TD><TD>Add 30-50ul of nuclease free water to the column. Spin down the DNA and quantify concentration using the Thermo Scientific nanodrop.</TD><TD>Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers</TD><TD>Injury due to imbalanced centrifuge and trapping of limbs or fingers</TD><TD>wear proper PPE (gloves, lab coat, covered shoes). Clean the bench with 70% ethanol after work for the day. Ensue that the centrifuge is balanced.</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="Subculturing_for_adherent_cell_lines"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/1/1e/SPSingapore_Protocols_Subculturing_for_adherent_cell_lines.pdf" target="_blank">Subculturing for adherent cell lines</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Remove and discard old media. Avoid disturbing the adhered cells.</TD><TD>Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,</TD><TD>Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury</TD><TD>All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.</TD><TD>1</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Wash flask/dish with 10 ml of DMEM (-), then remove. Avoid washing off cells.</TD><TD>Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,</TD><TD>Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury</TD><TD>All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.</TD><TD>1</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Trypsinize the cells by adding 2 ml of DMEM (-) and 1ml of trypsin. Swirl the flask/dish, then incubate for 4 – 7 minutes.</TD><TD>Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,</TD><TD>Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury</TD><TD>All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.</TD><TD>1</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>4</TD><TD>Add 10 ml of DMEM (+) to stop the action of trypsin, then wash the flask/dish to wash off more cells. Transfer the cells into a 15 ml Falcon tube.</TD><TD>Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens,</TD><TD>Exposure to mammalian cells and blood borne pathogens, self-inoculation by needlestick injury</TD><TD>All personel handling cell lines are to have Hep B vaccination, and passage numbers are not to be maintained too high. Internal training is compulsory.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the dedicated BSL2 biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand.</TD><TD>1</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>5</TD><TD>Spin down cells by centrifuging at room temperature (25oC), 800 rpm for 5 minutes. After centrifuging, pour out the media without disturbing the cell pellet. Resuspend cell pellet with 1 ml DMEM (+) by pipetting up and down at least 10 – 15 times. Split cells into new culture dish/flask at the desired and recommended density (usually ~30-40% for cell maintenance).</TD><TD>Spillage, biological exposure to mammalian cells and endogenous viruses/blood borne pathogens, injury due to improper usage of centrifuge and fingers</TD><TD>Injury due to imbalanced centrifuge and trapping of limbs or fingers</TD><TD>Internal training is compulsory for centrifuge use and the centrifuge key.  wear proper PPE (gloves, lab coat, covered shoes); handle cells in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand. When using the centrifuge, ensure centrifuge is balanced and rotor is placed correctly, and that all tubes are capped tightly. Close centrifuge properly and ensure that there are no funny sounds when centrifuge is running. Decontaminate waste using Presept tablets (dilute to 10% solution of activaed bleach to kill cells) and let the bleach decontaminate for half an hour before discarding with plenty of water and dilution.</TD><TD>1</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="Ligation_Protocol_with_DNA_ligase"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/f/f4/SPSingapore_Protocols_Ligation_Protocol_with_T4_DNA_Ligase.pdf" target="_blank">Ligation Protocol with T4 DNA ligase</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Adjust Thermocycler to 16 degrees hold.</TD><TD>Electric shock</TD><TD>Possibility of electric shock</TD><TD>wear proper PPE (gloves, lab coat, covered shoes)/</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Add aliquots of cut vector and insert to water and T4 ligase buffer in a 0.2mL pcr tube.</TD><TD>Biological exposure</TD><TD>Spillage of buffer and DNA</TD><TD>wear proper PPE (gloves, lab coat, covered shoes)/</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Lightly centrifuge the pcr tube in a microcentrifuge. Add an appropriate volume of ligase and centrifuge again.</TD><TD>Spillage, Biological exposure , injury due to improper usage of centrifuge and fingers</TD><TD>Injury due to imbalanced centrifuge and trapping of limbs or fingers</TD><TD>Internal training is compulsory.wear proper PPE (gloves, lab coat, covered shoes); handle cells in the biosafety cabinet; have disinfectant (e.g. 70 % ethanol) on hand. When using the centrifuge, ensure centrifuge is balanced and rotor is placed correctly, and that all tubes are capped tightly. Close centrifuge properly and ensure that there are no funny sounds when centrifuge is running.</TD><TD>1</TD><TD>2</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>4</TD><TD>Place pcr tube into thermocycler and ligate overnight at 16 degrees.</TD><TD>Biological exposure</TD><TD>Spillage of buffer and DNA</TD><TD>wear proper PPE (gloves, lab coat, covered shoes)/</TD><TD>1</TD><TD>2</TD><TD>2</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="Antibiotic_Preparation"><STRONG>Link to protocol</STRONG><DIV><P><A href="https://static.igem.org/mediawiki/2015/0/0f/SPSingapore_Protocols_Antibiotic_Preparation2.pdf" target="_blank">Antibiotic Preparation</A></P></DIV><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Weigh out antibiotics in a blue falcon tube. Prepare atibotic concentrations to 100ug/mL.</TD><TD>Spillage of antibiotics</TD><TD>Accidental inhalation or ingestion of antibiotics</TD><TD>wear proper PPE (gloves, lab coat, covered shoes).</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Add Milliq water to blue falcon tube. Vortex and allow to incubate at room temperature till antibiotics are dissolved.</TD><TD>Spillage of antibiotics</TD><TD>Accidental inhalation or ingestion of antibiotics</TD><TD>wear proper PPE (gloves, lab coat, covered shoes).</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Filter the antibiotics through a 0.2um filter using a sterile syringe in the bacterial BSC. Prepare aliquots of 250ul or less of antibiotics as working stock for concentration saccording to the antibiotic concentrations listed in the recommended amounts  and store in antibiotics drawer in the -20 fridge.</TD><TD>Spillage of antibiotics</TD><TD>Accidental inhalation or ingestion of antibiotics</TD><TD>wear proper PPE (gloves, lab coat, covered shoes).</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV><DIV id="Agarose_Electrophoresis"><STRONG>Risk assessment</STRONG><DIV><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>No</TH><TH>Desription/Details of Steps in Activity</TH><TH>Hazards</TH><TH>Possible Accident / Ill Health &amp; Persons-at-Risk</TH><TH>Existing Risk Control (Mitigation)</TH><TH>Severity</TH><TH>Likelihood (Probability)</TH><TH>Risk Level</TH><TH>Additional Risk Control</TH><TH>Person Responsible</TH><TH>By (Date)</TH></TR><TR><TD>1</TD><TD>Heat agarose powder in TAE buffer until dissolved. Cool agarose to 50 degrees and add Sybr Safe DNA stain. </TD><TD>Chemical hazard- Sybr safe is an intercalating dye but unable to diffuse across cell membranes</TD><TD>Spillage of heated agarose</TD><TD>A work bench is specially allocated for DNA gel electrophoresis. Sybr Safe is used as it is not as toxic as Etbr. Wear appropriate PPE (nitrile gloves, lab coat, mask). Heat agarose slowly to avoid boiling and spillage out of the flask. Sybr safe solution is added to the flask in a fume hood after the agarose has cooled to reduce the chance of producing vapors. </TD><TD>2</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>2</TD><TD>Pour the agarose into the casting tray and allow the gel to solidify over 30 minutes.</TD><TD>Chemical hazard- Sybr safe is an intercalating dye but unable to diffuse across cell membranes</TD><TD>Spillage of heated agarose</TD><TD>Wear appropriate PPE (nitrile gloves, lab coat, mask). Casting tray is contained in appropriate trays to contain any accidental spillage. Handle hot flask of agarose with a heatprooof glove.</TD><TD>2</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>3</TD><TD>Remove gel tray and fillthe electrophoresis apparatus with TAE buffer. Load the DNA sample and resolve through the gel </TD><TD>Chemical hazard- Sybr safe is an intercalating dye but unable to diffuse across cell membranes; electrical hazard</TD><TD>Electric shock may occur if connecting and disconnecting the apparatus without powering off the machine. Accidental physical contact with Sybr Safe containing agarose gel </TD><TD>Wear appropriate PPE (nitrile gloves, lab coat). All procedures are performed with electrophoresis equipment contained within appropriate trays to contain any accidental spillage.  Connect and disconnect the apparatus only when it is powered off. Do not touch the apparatus while the gel is running.</TD><TD>2</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD><TD> </TD></TR><TR><TD>5</TD><TD>Visualize the gel under UV light. Discard gel for incineration at medical waste disposal plant.</TD><TD>Chemical hazard- Sybr safe is an intercalating dye but unable to diffuse across cell membranes; biological hazard- UV light is mutagenic</TD><TD>Electric shock may occur if connecting and disconnecting the apparatus without powering off the machine. Accidental physical contact with Sybr Safe containing agarose gel </TD><TD>Wear appropriate PPE (nitrile gloves, lab coat, UV resistant goggles). </TD><TD>2</TD><TD>1</TD><TD>2</TD><TD> </TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="content content-full bigDiv" id="part" style="height:600px; ; overflow:scroll"><DIV id="accordionPart"><H3>Team part</H3><DIV><P>
For Team part
</P></DIV><H3>Basic part</H3><DIV><P>
For basic part
</P></DIV><H3>Composite part</H3><DIV><P>
For composite part
</P></DIV><H3>Part collection</H3><DIV><P>
For part collection
</P></DIV></DIV></DIV><DIV class="content content-full bigDiv" id="notebook"><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="row" width="50%"><DIV id="calendarLegend"><DIV id="project1Legend" style="width:100%;height:30px"><DIV style="float:left;display:inline;margin:3px">esaGFP quorum sensing [Adrian, Clarice, Kenneth]</DIV></DIV><DIV id="project2Legend" style="width:100%;height:30px"><DIV style="float:left;display:inline;margin:3px">Invasin + Listeriolysin [YanTing, YunTing]</DIV></DIV><DIV id="project4Legend" style="width:100%;height:30px"><DIV style="float:left;display:inline;margin:3px">Maintenance</DIV></DIV><DIV id="project6Legend" style="width:100%;height:30px"><DIV style="float:left;display:inline;margin:3px">FNRgfp [YiHan, ChiYan]</DIV></DIV></DIV></TH></TR></TBODY></TABLE><DIV id="calendarStorage"><DIV id="25_05"><P><STRONG>Yi Han </STRONG></P><P>Received 2 bacterial stab cultures, EsaR/I plasmid from addgene (CHL) and BBa_K299812 from iGEM HQ. </P><P>Streaked out on plates with amp. </P></DIV><DIV id="26_05"><P><STRONG>Adrian </STRONG></P><P>Transformed:</P><P> +Kit plate 1 9N Ba_K763002 chl </P><P>+Kit plate 4 13L BBa_E0040 amp </P><P>DNa was received in powder form in plates, and resuspended in 10ul ultrapure H2O respectively. Plates were stored in -20/ </P><P><STRONG>Yi Han </STRONG></P><P>Plates cracked in incubator as they dried from lack of humidity.
</P><P>Transfer to small incubator with beaker of water for humidity
no single colonies for inv plasmid -&gt; streak again.
</P><P>Wrong antibiotic for EsaR/I plasmid-&gt; streak out again
</P></DIV><DIV id="28_05"><P><STRONG>Yi Han </STRONG></P><P>Inoculate single colony of invasin plasmid carrying bacteria in 3mL LB+amp </P><P>Transformation of 13L repeated with 1ul of DNA. </P></DIV><DIV id="29_05"><STRONG>Xin Yi</STRONG><P>No colonies grew for 13L on all plates -&gt;adjust incubator, make new media
</P><P>Miniprep of inoculated bacteria for invasin plasmid, incubate sample in 37degC for 1h 20min </P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">RE of invasin plasmid</TH><TH scope="col">RE control</TH></TR><TR><TD>Rsal 1ul</TD><TD>Rsal 0 ul</TD></TR><TR><TD>EcoRI 1 ul</TD><TD>EcoRI 0 ul</TD></TR><TR><TD>INv plasmid 7.5ul</TD><TD>Inv plasmid 7.5 ul</TD></TR><TR><TD>H2O 35.5 ul</TD><TD>H2O 37.5 ul</TD></TR><TR><TD>NEB buffer 4.5 ul</TD><TD>Buffer 4.5 ul</TD></TR></TBODY></TABLE><P> </P>
Repeat transformation of 13L with 1ul
</DIV><DIV id="30_05"><P><STRONG>Xin Yi and Yi Han</STRONG></P><P>Miniprep of EsaR/I plasmid for 4 colonies </P><P>Restriction digest for EsaR with XbaI/BamHI </P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">RE reaction</TH></TR><TR><TD>5ul plasmid</TD></TR><TR><TD>5ul buffer</TD></TR><TR><TD>1ul XbaI</TD></TR><TR><TD>1ul BamHI</TD></TR><TR><TD>Add H2O to 50ul </TD></TR></TBODY></TABLE></DIV><DIV id="31_05"><P><STRONG>Xin Yi </STRONG></P><P>Sent EsaR clone 4 and INv-4 for sequencing. </P></DIV><DIV id="02_06"><STRONG>Yi Han</STRONG><P>YFP and GFP transformation results - no colonies for YFP </P><P>The GFP transformation repeated with 100ng of plasmid was sucessful.</P><P> 4 colonies of gfp plasmid were inoculated in 3mL LB+amp and grown overnight. </P></DIV><DIV id="03_06"><STRONG>Yi Han</STRONG><P>Miniprep of gfp plasmids </P><P>RE digest with EcoRI and RsaI </P></DIV><DIV id="04_06"><STRONG>Chi Yan</STRONG>
RE digest indicated a very faint smaller band for colonies 2-4, and hence these were likely to be positive clones
</DIV><DIV id="05_06"><P><STRONG>Yanting</STRONG></P><P>Inoculated 3mLS of Inv-4 and EsaR-4 plasmid carrying bacteria into 100mL LB+ appropriate Antibiotic
</P><P>Cell culture-&gt; HEK293 cells revived from freezing down appeared detached.
</P></DIV><DIV id="06_06"><P><STRONG> Yi Han + Yanting</STRONG></P><P>Storage of bacterial glycerol stocks for Inv-4, EsaR4 in 25% glycerol </P><P>Yanting: grew HEK293T cells in T25 glask </P><P>Gel extract to clean up gfp plasmids which loading dye had accidentally been added to. </P></DIV><DIV id="08_06"><P><STRONG>Yi Han</STRONG></P><P> Miniprep of gfp plasmids, preparation of samples for sequencing </P></DIV><DIV id="09_06"><P><STRONG>Yi Han </STRONG></P><P>All gfp plasmids had a correct sequence </P><P>Yunting kept glycerol stocok for all, and inoculation of 3mL of gfp3 into 100mL LB+amp for midiprep </P></DIV><DIV id="10_06"><P><STRONG>Yi Han + Yunting </STRONG></P><P>Midiprep of gfp plasmid
PCR of gfp with KpnI-gfp and XhoI-gfp primers </P></DIV><DIV id="11_06"><P><STRONG>Yunting + Duy
</STRONG></P><P>Nanodrop of gfp product-&gt; 595.5ng/ul
</P><P>RE digest of EsaR vector with KpnI/XHoI </P><P>Gel electrophoreseis at 1000V for 30min </P><P>Gel extraction: 3.9ng/ul and 6.9ng/ul for Esa fragment and GFP --&gt; low yield</P></DIV><DIV id="12_06"><P><STRONG>Yunting </STRONG></P><P>Gel extraction using Promega binding solution to melt gel, followed by thermo scientific kit</P><P><STRONG>Adrian</STRONG></P><P dir="ltr">Gel extraction optimisation</P><P dir="ltr">Hypothesised that the Binding buffer has a problem/DNA does not bind to column</P><P dir="ltr">1: 2X promega binding buffer volume</P><P dir="ltr">2: Increase incubation time for binding to 5min</P><P dir="ltr">Switch binding buffer to that of thermo scientific PCR purification kit</P><P dir="ltr">Use sodium acetate if available? TO facilitate stronger binding to column</P><P dir="ltr">Results</P><P dir="ltr">1: ~10ng/ul</P><P dir="ltr">2: ~9ng/ul</P><P dir="ltr">not succesful</P><P dir="ltr">further optimisation-&gt; warm buffer, incubate for 5min</P><P dir="ltr">elute in 30/20ul smaller volumes</P></DIV><DIV id="13_06"><P dir="ltr"><STRONG>Yihan</STRONG></P><P dir="ltr">1: Thermoscientific miniprep columns with 2XThermoscientific binding buffer</P><P dir="ltr">2: Thermoscientific PCR purification kit 2X buffer</P><P dir="ltr">3: Promega kit 2X buffer</P></DIV><DIV id="14_06"><P><STRONG>Yihan </STRONG></P><UL><P><STRONG>PCR </STRONG>(mastermix: 8)</P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>PCR buffer</TD><TD>10 ul * 8 = 8-ul</TD></TR><TR><TD>primers</TD><TD>0.8ul, 0.8ul </TD></TR><TR><TD>DNA polymerase</TD><TD>4ul</TD></TR><TR><TD>dH2O</TD><TD>61.75 * 8 = 494ul</TD></TR><TR><TD>Templates</TD><TD>5.55*7 = 38.75ul</TD></TR></TBODY></TABLE><P><STRONG>Digest more plasmid (Esa plasmid) </STRONG></P><P> 4 rxns ( KpnI/XhoI digest) 50ul each </P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>Buffer</TD><TD> </TD></TR><TR><TD><P dir="ltr">KpnI</P></TD><TD><P dir="ltr">2ul</P></TD></TR><TR><TD><P dir="ltr">XhoI</P></TD><TD><P dir="ltr">2ul</P></TD></TR><TR><TD><P dir="ltr">DNA</P></TD><TD><P dir="ltr">34.8ul</P></TD></TR><TR><TD><P dir="ltr">H2O</P></TD><TD><P dir="ltr">141.2ul</P></TD></TR></TBODY></TABLE></UL><P> </P></DIV><DIV id="15_06"><P><STRONG>Yihan</STRONG></P><P><STRONG>PCR </STRONG></P><P>RP_XhoI_GFP and FP_KpnI_GFP with GFP midiprep </P><P>For 15 reactions with control .: Mastermix * 17 </P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD><P dir="ltr">PCR buffer</P></TD><TD><P dir="ltr">170ul</P></TD></TR><TR><TD><P dir="ltr">primers</P></TD><TD><P dir="ltr">1.7ul, 1.7ul</P></TD></TR><TR><TD><P dir="ltr">dNTP</P></TD><TD><P dir="ltr">34 ul</P></TD></TR><TR><TD><P dir="ltr">DNA polymerase</P></TD><TD><P dir="ltr">8.5 ul</P></TD></TR><TR><TD><P dir="ltr">dH2O</P></TD><TD><P dir="ltr">66.3*17 = 1127.1ul</P></TD></TR><TR><TD><P dir="ltr">templates</P></TD><TD><P dir="ltr">17ul</P></TD></TR><TR><TD><P dir="ltr">total</P></TD><TD><P dir="ltr">1360ul</P></TD></TR></TBODY></TABLE><H3 dir="ltr"><STRONG>PCR protocol “GFP 1” (32 cycles)</STRONG></H3><H3 dir="ltr"><STRONG>GFP PCR product -&gt; 448ng/ul </STRONG></H3><P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-452f-67d8-ffd1-fb8e413164b0">Gel extraction</STRONG></P><P dir="ltr">Qiagen gel extraction kit at MBI</P><P dir="ltr">Esa gel band from 14/6</P><P dir="ltr">Nanodrop: 16.4 ng/ul, 260/280 = 2.60</P><STRONG id="docs-internal-guid-934bc9f1-4530-4a77-d62e-62ecf5445432"><P dir="ltr">Plasmid construction</P></STRONG><UL><LI dir="ltr"><P dir="ltr">RE digest (KpnI, XhoI)</P></LI><LI dir="ltr"><P dir="ltr">3 Replicates of the following</P></LI></UL><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD><P dir="ltr">DNA</P></TD><TD><P dir="ltr">2.6ul</P></TD></TR><TR><TD><P dir="ltr">Buffer</P></TD><TD><P dir="ltr">5ul</P></TD></TR><TR><TD><P dir="ltr">H2O</P></TD><TD><P dir="ltr">41.5ul</P></TD></TR><TR><TD><P dir="ltr">KpnI</P></TD><TD><P dir="ltr">0.5ul</P></TD></TR><TR><TD><P dir="ltr">XhoI</P></TD><TD><P dir="ltr">0.5ul</P></TD></TR></TBODY></TABLE><P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-4532-aecd-1923-3e0b25b1a403">Ligation (2x reaction) 40ul</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD><P dir="ltr">10X T4 DNA ligase buffer</P></TD><TD><P dir="ltr">4ul</P></TD></TR><TR><TD><P dir="ltr">Vector DNA (16ng/ul) </P></TD><TD><P dir="ltr">100ng -&gt; 6.25ul</P></TD></TR><TR><TD><P dir="ltr">insert DNA </P></TD><TD><P dir="ltr">75nl -&gt; 12 ul </P></TD></TR><TR><TD><P dir="ltr">T4 DNA Ligase</P></TD><TD><P dir="ltr">2ul</P></TD></TR><TR><TD><P dir="ltr">H2O</P></TD><TD><P dir="ltr">16ul</P></TD></TR></TBODY></TABLE><P dir="ltr">Note: ALL digested DNA in tube labeled lacGFP.ligation was used</P><P dir="ltr">Nanodrop of ligation: 1114.0ng/ul. 260/280 = 3.7</P></DIV><DIV id="16_06"><P><STRONG>Transformation
</STRONG></P><P>Transformation of ligated esa-GFP plasmid into DH5\alpha cells
</P><P>DH5\alpha ,&lt;- unlabelled brown vial inside DH5\alpha box at -80</P><BLOCKQUOTE><P> Plates spread at 11:40h </P></BLOCKQUOTE><P><STRONG>LB + chloroamphenicol </STRONG></P><P>+4x (100ul) </P><P>+10x (40ul) </P><P>+20x (20ul) </P><P>+LB - 20x (20ul) </P><P>remaining transformed cells (~220ul) are kept at 4C</P><P>--&gt; labeled as placGFP in brown tube </P></DIV><DIV id="17_06"><P><STRONG>Oservation: </STRONG>only 4x placGFP plate has colonies (7) </P><P>spread one new LB + chlor plate with 200ul of transformed bacteria </P><P>picked 6 colonies to grow for miniprep in LB + chl liq media </P><P><STRONG>Conclusion: </STRONG></P><P>+protocol works </P><P>+optimization for increase vol needed </P><P><STRONG>cloning of synparts in amp vector</STRONG></P><P>+comes as 4mg dry DNA </P><P>+40ul of DI/RNAse free H20 </P><P>+for transformation, 150ml on wed</P></DIV><DIV id="18_06"><P dir="ltr"><STRONG>Unknown</STRONG></P><P dir="ltr"><STRONG>Miniprep of placGFP followed by RE digest (Kpn, XhoI): 50 ul total</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD><P dir="ltr">Buffer</P></TD><TD><P dir="ltr">2.5ul</P></TD></TR><TR><TD><P dir="ltr">KpnI</P></TD><TD><P dir="ltr">&lt;ILLEGIBLE&gt;</P></TD></TR><TR><TD><P dir="ltr">XhoI</P></TD><TD><P dir="ltr">&lt;ILLEGIBLE&gt;</P></TD></TR><TR><TD><P dir="ltr">DNA</P></TD><TD><P dir="ltr">10</P></TD></TR><TR><TD><P dir="ltr">H2O</P></TD><TD><P dir="ltr">12025</P></TD></TR></TBODY></TABLE><P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-4554-de9f-2ea4-1efa0bb8416a">Colony PCR for synparts</STRONG></P><P dir="ltr">+ Failed</P>
+ No specific &lt;illegible&gt; produced
<P>+ Might need optimization</P><P dir="ltr"> </P></DIV><DIV id="19_06"><P><STRONG>Yihan </STRONG></P><P>RE digest (XhoI, KpnI) of esa and GFP </P><P><STRONG>Yunting</STRONG></P><P>Gel electrophoresis (100V, 40min)</P><P>+ Lane 2: gpf: no bands </P><P>+ Lane 3: 100bp ladder </P><P>+ Lane 4: blank </P><P>+ Lane 5: esa: 2 bands </P><P>+ Lane 6: 1kb ladder </P><P>+ Lane 7:blank </P><P>&lt;Picture of gel&gt;</P></DIV><DIV id="21_06"><P><STRONG>Optimizing gel extract protocol </STRONG></P><P>+ RE GFP from PCR (directly RE) </P><P>+ promega agarose 1% gel</P><P> + add NaAC in binding buffer</P><P><STRONG>Result:</STRONG></P><P>+ yield for cut plasmid was 12.8ng/ul </P><P>+ GFP was 8.6ng/ul</P><P><STRONG>Ligation reaction 6 reactions </STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD><P dir="ltr">Buffer</P></TD><TD><P dir="ltr">12ul</P></TD></TR><TR><TD><P dir="ltr">Vector</P></TD><TD><P dir="ltr">30ul</P></TD></TR><TR><TD><P dir="ltr">Insert</P></TD><TD><P dir="ltr">30ul</P></TD></TR></TBODY></TABLE><P> </P><P> + 6x ligation result transformed into competent DH5\alpha
20 ul, 50ul, 100ul plated on LB + chl </P></DIV><DIV id="23_06"><P><STRONG>Yunting</STRONG></P><P><STRONG>Ligation of GFP tester plasmid. </STRONG></P><P>Performed 4x ligation using RE- digested esa and GFP (CY, 22/6). Ligation rxn at room temperature for 30min instead of 10min. </P><P>Included vector-only and insert-only controls. </P><P>Stored in -20deg. </P><P>Will run gel tmr. Insert only control should be same size as gfp product. Same for the other control. Can try to plate vector only to see re-ligation?? </P><P><STRONG>Nanodrop: </STRONG></P><P>PlacGFP - 642.8ng/ul. 260/280=3.89</P><P> Vector ctrl - 756.6ng/ul. 260/280=4.11</P><P> Insert ctrl - 557 ng/ul. 260/280=3.86 </P></DIV><DIV id="25_06"><P><STRONG>Yanting</STRONG></P><P>Ran 0.8% pre-cast gel at 100V for 45min. </P><P>10ul of each sample to 2ul of loading dye. </P><P><STRONG>Gel lanes: </STRONG></P><P>100bp; uncut esa (used esa4 from -20); </P><P>pLacGFP; vector ctrl; insert ctrl; 1kb. </P><P><STRONG>[YH] Re-run gel.</STRONG> 35ul of each sample. No bands. </P><P>&lt;insert gel picture&gt;</P><P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-4578-e9aa-242f-73f4dc94cc3f">RE of esa and Gfp pcr product. </STRONG></P><P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-457a-7ead-b589-a735f588b766">Gel electrophoresis. </STRONG></P><P dir="ltr">Cast a thick 1% gel (60ml) with combined wells. [Don't need to cast thick gel next time, takes too long to melt]</P><P dir="ltr">Gel run at 100V,  45min. Lanes: 100bp, esa, gfp 1&amp;2, 1kb </P>
Image after cutting is also saved.
<P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-457b-7c6c-4056-14590d25f13e">Gel extraction of RE esa &amp; gfp.</STRONG></P>
Used Promega kit, loaded both gfp bands into one column. Eluted with 30ul water for 5min before centrifugation.
<P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-457c-0a29-4143-410f5a206639">Nanodrop:</STRONG></P><P dir="ltr">Esa- 37.7ng/ul. 260/280= 1.83</P><P dir="ltr">Gfp - 25.1ng/ul. 260/280= 1.84</P><STRONG id="docs-internal-guid-934bc9f1-457c-4799-66ba-fc8406b4fa38"><P dir="ltr">Overnight ligation</P></STRONG><P dir="ltr">6x ligation: </P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD><P dir="ltr">Buffer</P></TD><TD><P dir="ltr">6ul</P></TD></TR><TR><TD><P dir="ltr">Vector (37.7ng/ul)</P></TD><TD><P dir="ltr">50ng = 7.8ul</P></TD></TR><TR><TD><P dir="ltr">Insert (25.1ng/ul)</P></TD><TD><P dir="ltr">37.7 ng = 6.6ul</P></TD></TR><TR><TD><P dir="ltr">H2O</P></TD><TD><P dir="ltr">87.6ul</P></TD></TR><TR><TD><P dir="ltr">Ligase</P></TD><TD><P dir="ltr">6ul</P></TD></TR></TBODY></TABLE>
Ligation reaction run overnight in 16 deg hold in thermocycler - 15h, 8pm - 11am. <P dir="ltr"> </P></DIV><DIV id="26_06"><P><STRONG>[Chi Yan + Yunting] Midiprep of EsaR plasmid </STRONG></P><P><STRONG>[Yihan Yunting] Gel electrophoresis of inv colony PCR </STRONG></P><P>insert picture gel </P></DIV><DIV id="27_06"><P><STRONG>[Xinyi] Colony pcr for placgfp
for &gt;18 colonies
</STRONG></P><P> insert gel picture </P><STRONG>[Adrian] Gfp expression observed using gfp filter with the SPS microscope </STRONG></DIV><DIV id="17_06green"><P><STRONG>Yanting</STRONG></P><P><STRONG>Subculture of HEK 293
</STRONG></P><P>P4 -&gt; p5</P><P> Grow/split into 150mm dish for freezing on sat(20/6) </P><P>--&gt;30ml DMEM + 1ml cells </P><P>90mm dish for maintenance (buffer) </P><P>--&gt;10ml DMEM + 30ul cells </P><P>Cells combined from 3 90mm dishes </P></DIV><DIV id="20_06green"><P><STRONG>Yanting</STRONG></P><P><STRONG>Freezing of HEK293
</STRONG></P><P>P6 </P><P>Cells from a 150 mm dish and 90mm dish </P></DIV><DIV id="12_07blue"><P><STRONG>Adrian</STRONG></P><P>Prepared restreak of inv/hly plasmid from original stab culture </P></DIV><DIV id="14_07blue"><P><STRONG>Yanting and Yunting</STRONG></P><P>Colony PCR of invasin
</P><P>Picked out 8 colonies (marked 1-8) from BBa_K299812 plate stored at 4deg (Adrian, 12/7). </P><P>Colony PCR using prefix suffix primers for first 4 rxn and universal primers VP &amp; VF2 for last 4 rxn. Used thermocycler &quot;Colony&quot; protocol.</P><P> Also constituted dNTP w 2.5mM of each atcg triphosphate. </P></DIV><DIV id="15_07blue"><P><STRONG>Yanting</STRONG></P><P>100V at 30min. Ladder, 4 prefix suffix rxn, 4 universal primers rxn. Saved as “7.15_inv colony”. </P><P>When I ran for longer (after storing gel at 4deg), the bands were longer but the 250bp marker as well as the primer-dimers are pretty close to dye front. </P><P><STRONG>Yunting</STRONG></P><P>Placed BBa_K299812 plate in incubator for overnight growth. </P></DIV><DIV id="16_07blue"><P><STRONG>Yanting and Yunting</STRONG></P><P>Colony PCR of 8 colonies (marked A-H) from BBa_K299812 plate, and also spotted on a save plate. Both plates placed back in incubator. </P><P>Primer conc=0.5 uM, template DNA dissolved in 10ul h20.</P><P>Thermocycler “colony_inv” protocol, with adjusted extension time and annealing time&amp;temp from standard “colony” protocol.</P><P dir="ltr">100V for 34min. </P><P dir="ltr">Tubes 1-2: FP-prefix, RP-suffix. </P><P dir="ltr">3-4: FP-VF2, RP-VR. </P><P dir="ltr">5-6: FP-prefix, RP_Inv_M2.</P><P dir="ltr"> 7-8: RP-suffix, FP_Inv_M1. </P><P dir="ltr">No template control with FP-prefix, RP-suffix.</P><P dir="ltr">[Note: RP_M2 = FP_M2. Check future uses agn seq on the primer master file. Just in case, the sequence used in this expt is INV_FP_M2-&gt;GCTCATTATAGTCCGCGAAATCACG]. </P>
Gel image saved as “7.17_inv colony.sgd” (handphone pic below).
<P dir="ltr"><STRONG>Results</STRONG>: Tubes 3&amp;4 with universal primers VF2 VR have a band between 4&amp;5kb - Inv+LLO is 4.1kb, probably are positive colonies. </P><P dir="ltr">Tubes 1&amp;2, 5&amp;6 have bands &lt;750bp as well as primer dimers (but the annealing temperature was calculated for prefix suffix primers not universal ones…I’ll try thermo-gradient thermocycler protocol next time).</P><P dir="ltr"> Expected band size for 5&amp;6 (FP-prefix, RP_Inv_M2 (ends 1928)) = 1.9kb. </P>
No bands for tubes 7&amp;8. Expected band size for 7&amp;8 (RP-suffix, FP_Inv_M1 (starts 1046)) = 2.2kb.
<P> </P></DIV><DIV id="19_07blue"><P><STRONG>Yihan</STRONG></P><P>Inoculation of positive colonies in liquid culture. 3mL and shaking incubation. </P></DIV><DIV id="20_07blue"><P><STRONG>Yunting</STRONG></P><P>Miniprep of positive colonies from 30h liquid culture. </P><P>Eluted in 50ul elution buffer and stored at -20. </P><P>C= 225.9ng/ul. 260/280=1.86 </P><P>D= 297.7ng/ul. 1.87 </P></DIV><DIV id="21_07blue"><P><STRONG>Yunting</STRONG></P><P dir="ltr"><STRONG>Colony PCR with thermogradient: 14rxn</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD><P dir="ltr">H2O</P></TD><TD><P dir="ltr">70ul</P></TD></TR><TR><TD><P dir="ltr">dNTP</P></TD><TD><P dir="ltr">14ul</P></TD></TR><TR><TD><P dir="ltr">MgCl2</P></TD><TD><P dir="ltr">35ul</P></TD></TR><TR><TD><P dir="ltr">Taq</P></TD><TD><P dir="ltr">3.25ul</P></TD></TR><TR><TD><P dir="ltr">primers (forward + reverse)</P></TD><TD><P dir="ltr">14ul*2</P></TD></TR><TR><TD><P dir="ltr">Template DNA</P></TD><TD><P dir="ltr">14ul</P></TD></TR></TBODY></TABLE><P dir="ltr">C/D-1,2: Col3. </P><P dir="ltr">C/D-3,4: Col12. </P><P dir="ltr">D-5,6: Col 9. </P><P dir="ltr">D-7,8: Col 10.</P><P dir="ltr">Negative control (no template): Col 8.</P><P dir="ltr">Gel ran for 80V, 1h. </P><P dir="ltr">D-1,2: VF2, VR. </P><P dir="ltr">D-3,4: FP-prefix, RP-suffix. </P><P dir="ltr">D-5,6: FP-prefix, FP_M2. </P><P dir="ltr">D-7,8: FP-prefix, FP_invF.</P><P dir="ltr">C-1,2: VF2, VR. </P><P dir="ltr">C-3,4: FP-prefix, RP-suffix. </P><P><STRONG>Results Notes</STRONG>: </P><P>Too much template DNA (~250ng). Separate the universal primers (to avoid differing extension time).</P></DIV><DIV id="23_07blue"><P><STRONG>Yanting</STRONG></P><P>RE digest of inv/hyl plasmid with EcoRI and PstI for 2 hours at 37oC. </P><P dir="ltr">Lane 1-4: 1kb ladder, 100bp ladder, RE of colony C, RE of colony D.</P><P dir="ltr">Size is correct: 4kb main band (inv+hly part) and 2kb (plasmid backbone). These plasmid DNA should have the part.</P></DIV><DIV id="24_07blue"><P><STRONG>Yanting and Yunting</STRONG></P><P><STRONG>Colony PCR, used with temperature gradient to vary annealing temperature.
10rxn </STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>5x buffer</TD><TD>50ul</TD></TR><TR><TD><P dir="ltr">H2O</P></TD><TD><P dir="ltr">127.5ul</P></TD></TR><TR><TD><P dir="ltr">dNTP</P></TD><TD><P dir="ltr">50ul</P></TD></TR><TR><TD><P dir="ltr">MgCl2</P></TD><TD><P dir="ltr">10ul</P></TD></TR><TR><TD><P dir="ltr">Taq</P></TD><TD><P dir="ltr">3.25ul</P></TD></TR><TR><TD><P dir="ltr">primers (forward + reverse)</P></TD><TD><P dir="ltr">14ul*2</P></TD></TR><TR><TD><P dir="ltr">Template DNA</P></TD><TD><P dir="ltr">14ul</P></TD></TR></TBODY></TABLE><P>100V for 45min (can run for longer).</P><P><STRONG>Result:</STRONG></P><P dir="ltr">Lane 1: 1kb ladder; lane 2: 100bp ladder; </P><P dir="ltr">lane 3-7: FP prefix + RP suffix (5 repeats with increasing annealing temperatures) has 2 bands ~800 bp &amp; 100-200bp (probably non specific bands, will lower # of cycles in future, use higher annealing temp, lower annealing time). </P><P dir="ltr">Expected size of inv+hly part is 4.1kb.</P><P dir="ltr">lane 8-9: VF + M2 has a thick band 800-900bp. </P><P dir="ltr">Expected size is 600+130bp (VF adds ~130bp compared to FP_prefix) .: doesn’t seem to be correct…. </P><P dir="ltr">lane 10-11: VF + inv F has no bands. Expected is at least 200bp (bcos universal primers). </P><P dir="ltr">lane 12: VR + inv F seems to have a small band &lt;100bp. Non specific amplification? </P><P dir="ltr">Expected is 1.5+0.1 kb (from VR).</P></DIV><DIV id="31_07blue"><P><STRONG>Yanting</STRONG></P><P><STRONG>RE with XbaI and PstI </STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH width="36%" scope="col">Reagent</TH><TH width="64%" scope="col">Amount</TH></TR><TR><TD>Restriction enzyme</TD><TD>2ul</TD></TR><TR><TD>Buffer</TD><TD>5ul</TD></TR><TR><TD>DNA (4x of miniprep=74.5ng/ul)</TD><TD>13.4ul</TD></TR><TR><TD>H2O</TD><TD>29.6ul</TD></TR></TBODY></TABLE><P>Incubate at 37degC for 2.5h (1130-1400)</P><P dir="ltr"><STRONG>Results</STRONG>: Gel loaded 1kb ladder, uncut, RE digested. 100V for 1h. </P><P dir="ltr">Plasmid doesn't have XbaI site - RE digested DNA is a linear 6kb band. </P></DIV><DIV id="23_06red"><P><STRONG>Yunting</STRONG></P><P>Cast a big gel. Stored in 4 deg. [Used up 26/6] </P><P>Pre-cast two 0. 8% gels. Stored in 4deg. [Used up on 24 &amp; 25/6]
</P></DIV><DIV id="26_06red"><P><STRONG>Chi Yan and Yunting</STRONG></P><P>Midiprep of esaR plasmid. </P><P>Airdry ON. </P></DIV><DIV id="29_06red">
Safety Inspection for our lab by OSHE. We passed with flying colours!
</DIV><DIV id="14_07red"><P>Transformation of ligated FNRgfp plasmid into dH5alpha </P><P>Cleaned waterbath, de-iced the fridge. </P><P>Today, we also did a spring cleaning for the lab </P></DIV><DIV id="17_07red">
Pre-cast big and small gels and stored in 4deg in the blue tupperware. [Used up on 20 &amp; 22/7]
</DIV><DIV id="28_06yellow"><P><STRONG>Adrian</STRONG></P><P><STRONG>BBPrefix_esaRBS PCR</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD><P dir="ltr">H2O</P></TD><TD><P dir="ltr">221ul</P></TD></TR><TR><TD><P dir="ltr">Buffer</P></TD><TD><P dir="ltr">80ul</P></TD></TR><TR><TD><P dir="ltr">MgCl</P></TD><TD><P dir="ltr">32ul</P></TD></TR><TR><TD><P dir="ltr">dNTP</P></TD><TD><P dir="ltr">32ul</P></TD></TR><TR><TD><P dir="ltr">Forward Primer_Biobrick Prefix</P></TD><TD><P dir="ltr">16ul</P></TD></TR><TR><TD><P dir="ltr">ORP_esaRBS_fragsyn</P></TD><TD><P dir="ltr">16ul</P></TD></TR><TR><TD><P dir="ltr">synpart_BBP_esaRBS</P></TD><TD><P dir="ltr">1ul</P></TD></TR><TR><TD><P dir="ltr">gotaq</P></TD><TD><P dir="ltr">2ul</P></TD></TR><TR><TD><P dir="ltr">total</P></TD><TD><P dir="ltr">400ul</P></TD></TR></TBODY></TABLE><P><STRONG id="docs-internal-guid-934bc9f1-46c9-7554-40b8-48a3cad9d865">success-&gt; PCR purification</STRONG></P><P><STRONG id="docs-internal-guid-934bc9f1-46ca-0aee-b546-3982d573bbe8">esaRBS_GFP_BBsuffix</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD><P dir="ltr">H2O</P></TD><TD><P dir="ltr">237ul</P></TD></TR><TR><TD><P dir="ltr">Buffer</P></TD><TD><P dir="ltr">80ul</P></TD></TR><TR><TD><P dir="ltr">MgCl2</P></TD><TD><P dir="ltr">32ul</P></TD></TR><TR><TD><P dir="ltr">dNTP</P></TD><TD><P dir="ltr">32ul</P></TD></TR><TR><TD><P dir="ltr">Reverse primer_BB_suffix</P></TD><TD>
??</TD></TR><TR><TD><P dir="ltr">ORP_esaRS_GFP</P></TD><TD><P dir="ltr">16ul</P></TD></TR><TR><TD><P dir="ltr">Plac_GFPPLASMID</P></TD><TD><P dir="ltr">1ul</P></TD></TR><TR><TD><P dir="ltr">gotaq</P></TD><TD><P dir="ltr">2ul</P></TD></TR></TBODY></TABLE><P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-46cc-6a7d-ce73-e1023973a74a">failed -&gt; troubleshooting-&gt; new OFP_esaRBBS_GFP</STRONG></P><P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-46cc-6a7d-ce73-e1023973a74a">Redid 30/6 with new primers</STRONG></P><P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-46cc-6a7d-ce73-e1023973a74a">success with new primers-&gt; fusion pcr</STRONG></P>
PCR with inv primers (adrian’s primers)
<P> </P></DIV><DIV id="02_07yellow"><P><STRONG>Clarice</STRONG></P><P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-46cf-9174-adfe-47a39589e85a">Fusion PCR with esaRBS, GFP</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD><P dir="ltr">H2O</P></TD><TD><P dir="ltr">104.07ul</P></TD></TR><TR><TD><P dir="ltr">Buffer</P></TD><TD><P dir="ltr">40ul</P></TD></TR><TR><TD><P dir="ltr">MgCl2</P></TD><TD><P dir="ltr">16ul</P></TD></TR><TR><TD><P dir="ltr">dNTP</P></TD><TD><P dir="ltr">16ul</P></TD></TR><TR><TD><P dir="ltr">Forward primer_BB_prefix</P></TD><TD><P dir="ltr">8ul</P></TD></TR><TR><TD><P dir="ltr">Reverse primer_BB_prefix</P></TD><TD><P dir="ltr">8ul</P></TD></TR><TR><TD><P dir="ltr">400ng esaRBS</P></TD><TD><P dir="ltr">2.13ul</P></TD></TR><TR><TD><P dir="ltr">400ng esaRBSgfp</P></TD><TD><P dir="ltr">4.8ul</P></TD></TR><TR><TD><P dir="ltr">gotaq</P></TD><TD><P dir="ltr">1ul</P></TD></TR><TR><TD><P dir="ltr">Total</P></TD><TD><P dir="ltr">200ul</P></TD></TR></TBODY></TABLE><P dir="ltr">annealing temp 50degC</P><P dir="ltr"><STRONG id="docs-internal-guid-934bc9f1-46dd-d6fc-13e8-2e66bbd94e7a">Gel electrophoresis: looks correct-&gt; PCR Purification</STRONG></P><P><STRONG id="docs-internal-guid-934bc9f1-46dd-f312-fb45-0cdcebf3a920">RE digest of esaRBS+GFP</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Insert</TH><TH scope="col">Vector</TH></TR><TR><TD>Buffer</TD><TD>2ul</TD><TD>2ul</TD></TR><TR><TD>EcoRI/PstI</TD><TD>1ul</TD><TD>1ul</TD></TR><TR><TD>Template(2ug)</TD><TD>5ul</TD><TD>4ul</TD></TR><TR><TD>dH2O</TD><TD>11ul</TD><TD>12ul</TD></TR><TR><TD>total</TD><TD>20ul</TD><TD>20ul</TD></TR></TBODY></TABLE></DIV><DIV id="06_07yellow"><P><STRONG>Clarice</STRONG></P><P>COlny PCR with different colonies (15 colonies) </P><P> FP_BB_Prefix and RP_BB_Suffix </P><P><STRONG>Success </STRONG></P></DIV><DIV id="07_07yellow"><P><STRONG>Clarice</STRONG></P><P>Colony PCR successful - inouclated colony 2,3,4,12 in 3mL LB </P><P><STRONG id="docs-internal-guid-934bc9f1-46ea-c6bd-c68a-2b1fe837f12f">PCR-FNRsynpart BBP</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>dH2O</TD><TD>221ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTPs</TD><TD>32ul</TD></TR><TR><TD>FP_RNE_PromoterBBP</TD><TD>16ul</TD></TR><TR><TD>GFP_FNR_Prom_GFP</TD><TD>16ul</TD></TR><TR><TD>synpart FNR</TD><TD>1ul</TD></TR><TR><TD>gotaq</TD><TD>2ul</TD></TR></TBODY></TABLE><P> </P></DIV><DIV id="10_07yellow"><P><STRONG>Clarice</STRONG></P><P><STRONG id="docs-internal-guid-934bc9f1-46ed-4e6a-9565-a718a1856946">Verify minipreppped esaGFP plasmid</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>Buffer</TD><TD>2ul</TD></TR><TR><TD>EcoRI</TD><TD>1ul</TD></TR><TR><TD>Pst1</TD><TD>1ul</TD></TR><TR><TD>Plasmid</TD><TD>5ul</TD></TR><TR><TD>dH2O</TD><TD>11ul</TD></TR><TR><TD>total</TD><TD>20ul</TD></TR></TBODY></TABLE></DIV><DIV id="14_07yellow"><P><STRONG>Clarice</STRONG></P><P dir="ltr">Transformatin of EsaGFP plasmid (30ul BL21 + 5ul ligation reaction)</P>
plate O/N
</DIV><DIV id="03_07brown"><P><STRONG>Yihan</STRONG></P><P>Ran gel of gfp plasmid EcoRI/PstI, took out gel slice for vector - 4kb </P><P>Gel extraction: </P><P>+gfpvector-&gt;38.7ng/ul
fragment-&gt; direction purification 226.4ng/ul </P><P>+LIgation reaction (5:1) 6X </P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>300ng vector</TD><TD>7.8ng/ul</TD></TR><TR><TD>531.2ng of insert</TD><TD>2.4ul</TD></TR><TR><TD>Buffer</TD><TD>12ul</TD></TR><TR><TD>Enzyme</TD><TD>6ul</TD></TR><TR><TD>H2O</TD><TD>91.8ul</TD></TR></TBODY></TABLE><P>ligate for 2 hours at 16 hours
transform, plate, grow ON </P></DIV><DIV id="04_07brown"><P><STRONG>Yihan</STRONG></P><P>No colonies-&gt; ligation did not work
</P><P>Recalcualte ligation reaction (3:1),7x
</P><P>350ng vector-&gt;9ul vector
</P><P>446.3ng insert-&gt; 2ul
7ul </P><P>ligase
110</P><P>H2O </P><P>12ul buffer </P></DIV><DIV id="05_07brown"><P><STRONG>Yihan and Clarice</STRONG></P><P>colonies for EsaGFP </P><P>grew
Colony PCR for 14 colonies
</P><P>Unsuccessful-&gt; no bands observed
</P></DIV><DIV id="07_07brown"><P><STRONG>Yihan</STRONG></P><P>Trying out an idea for workshop </P><P>Trial for blue/white screen </P><P>Streak out pGFPuv and pGEMT on plate with amp added
spread 40ul of 0.1M IPTG and 30ul of 5%xgal, dried
and on plate without amp </P></DIV><DIV id="08_07brown"><P><STRONG>Yihan</STRONG></P><P>Ran gel with FNR Pcr and gfp pcr
</P><P>size of pcr product was correct but gel picture was not saved
</P><P>FNR Product was pcr purified
</P><P>Plasmid extraction for colony 2,3,4 of esaGFP - &gt; 2,3 were sent for sequencing and primers did not bind, chromatogram was messed up.
</P><P>Fusion PCR for FNRGFP:</P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>dH2O</TD><TD>218ul</TD></TR><TR><TD>10X buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>23ul</TD></TR><TR><TD>dNTP</TD><TD>32ul</TD></TR><TR><TD>Template FNR PCR (40ng)</TD><TD>3ul</TD></TR><TR><TD>gfp PCR (400ng)</TD><TD>1ul</TD></TR><TR><TD>FP_BB_Suffix</TD><TD>16ul</TD></TR><TR><TD>RP_BB_Suffix</TD><TD>16ul</TD></TR><TR><TD>gotaq</TD><TD>2ul</TD></TR></TBODY></TABLE></DIV><DIV id="09_07brown"><P><STRONG>Chi yan</STRONG></P><P>Gel was run for fusion pcr size was correct </P><P><STRONG>Clarice RE digest: </STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>PstI</TD><TD>1.5ul</TD></TR><TR><TD>EcoRI</TD><TD>1.5ul</TD></TR><TR><TD>Buffer</TD><TD>4ul</TD></TR><TR><TD>Template</TD><TD>3ul</TD></TR><TR><TD>H2O</TD><TD>20ul</TD></TR><TR><TD>Total</TD><TD>40ul</TD></TR></TBODY></TABLE></DIV><DIV id="10_07brown"><P><STRONG>Chi yan</STRONG></P><P>Ran gel for gfpp plasmid </P><P>gel purificaiton and extraction of 4kb fragment </P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>ligation reaction (2X)</TD><TD>40ul</TD></TR><TR><TD>Buffer</TD><TD>4ul</TD></TR><TR><TD>100ng of DNA vector</TD><TD>3.4ul</TD></TR><TR><TD>157ng insert</TD><TD>16ul</TD></TR><TR><TD>enzyme</TD><TD>2ul</TD></TR><TR><TD>H2O</TD><TD>14.6ul</TD></TR></TBODY></TABLE></DIV><DIV id="14_07brown"><P><STRONG>Yihan</STRONG></P><P>no colonies for FNRgfp plasmid
</P><P><STRONG>Inv plasmid verification
</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>buffer</TD><TD>1ul</TD></TR><TR><TD>EcoRI</TD><TD>0.5ul</TD></TR><TR><TD>invasin plasmid</TD><TD>0.5ul</TD></TR><TR><TD>dH2O</TD><TD>8ul</TD></TR><TR><TD>Total</TD><TD>10ul</TD></TR></TBODY></TABLE><P>Gel extraction RE digest of EsaR-GFP plasmid
</P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Ligation</TH><TH scope="col">Control</TH></TR><TR><TD>Vector (45ng)</TD><TD>1ul</TD><TD>1ul</TD></TR><TR><TD>Insert</TD><TD>1ul</TD><TD>1ul</TD></TR><TR><TD>Buffer</TD><TD>1ul</TD><TD>1ul</TD></TR><TR><TD>dH2O</TD><TD>6ul</TD><TD>7ul</TD></TR><TR><TD>ligase</TD><TD>1ul</TD><TD>1ul</TD></TR></TBODY></TABLE><P>Transformation of FNRgfp
anaerobe jar
</P><P>E. coli grew in both conditions
p putida-&gt; some growth in anaerobe chamber
proper streak plate in aerobic conditions
</P><P>packet runs out after 16hours </P></DIV><DIV id="13_07brown"><P><STRONG>Yihan and Yunting</STRONG></P><P>LIgation for FNRgfp (4X)</P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>200ng Gfp plasmid</TD><TD>4.36ul</TD></TR><TR><TD>297ng insert (7:1)</TD><TD>10.3ul</TD></TR><TR><TD>Buffer</TD><TD>53.34ul</TD></TR><TR><TD>T4 ligase</TD><TD>4ul</TD></TR></TBODY></TABLE><P>Trial run of anaerobe chamber:</P><P>+ open sachet to decrease O2 at 3.30pm </P><P>+ takes 2.5h to activate </P><P>place streak plate of p putida a strict aerobe and e coli, facultative aerobe in chamber at 30deg C to grow O/N </P><P>P. putida is an abligate aerobe and if chmaber works ,it will not grow </P><P>E. coli should grow in both conditions </P><P>controls grown outside chamber </P></DIV><DIV id="15_07brown"><P><STRONG>Yihan</STRONG></P><P>FNRgfp-&gt; no colonies after transformation
</P><P><STRONG>religation (4X reaction) </STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>200ng GFP plasmid</TD><TD>4.4ul</TD></TR><TR><TD>2125ng of insert</TD><TD>8ul</TD></TR><TR><TD>T4 ligase</TD><TD>4ul</TD></TR><TR><TD>H2O</TD><TD>55.6ul</TD></TR></TBODY></TABLE><P>transformation of FNR gfp into 20ul of BL21 </P></DIV><DIV id="16_07brown"><P><STRONG>Yihan</STRONG></P><P>No colonies grew </P><P>Ran a gel, FNRgfp pcr, gfp pcr, fnr, 100bp ladder </P></DIV><DIV id="17_07brown"><P><STRONG>Chi Yan</STRONG></P><P><STRONG>RE digest of FNR-GFP fusion PCR with EcoRI and PstI
</STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>EcoRI</TD><TD>1ul</TD></TR><TR><TD>PstI</TD><TD>1ul</TD></TR><TR><TD>Buffer</TD><TD>5ul</TD></TR><TR><TD>FNR-GFP from 12/7 in RIP box 1ug</TD><TD>2.5ul</TD></TR><TR><TD>H2O</TD><TD>40.5ul</TD></TR><TR><TD>Total</TD><TD>50ul</TD></TR></TBODY></TABLE><P>Direct purification using Promega kit</P><P><STRONG>Overnight ligation (4X reaction) </STRONG></P><TABLE width="100%" border="2" cellpadding="2"><TBODY><TR><TH scope="col">Reagent</TH><TH scope="col">Amount</TH></TR><TR><TD>200ng of GFP plasmid</TD><TD>4.4ul</TD></TR><TR><TD>400ng of insert (5:1)</TD><TD>?</TD></TR><TR><TD>T4 ligase</TD><TD>4ul</TD></TR><TR><TD>T4 ligase buffer</TD><TD>8ul</TD></TR><TR><TD>H2O</TD><TD>?</TD></TR></TBODY></TABLE><P> </P><P>200ng of gfp plasmid (4.4ul) </P><P>400ng of insert (5:1) (?ul) </P><P>4ul T4 ligase </P><P>8ul T4 ligase buffer </P><P>? H2O </P></DIV><DIV id="18_07brown"><P><STRONG>Yihan</STRONG></P><P>Ncbi search in BL21 genome revealed that it does have FNR transcriptional regulator </P><P>http://www.ncbi.nlm.nih.gov/nuccore/CP010816.1 </P><P>In dH5alpha as well </P><P>http://www.ncbi.nlm.nih.gov/gene/945908 </P><P>However no direct data on our specific strains of dH5alpha and BL21 </P></DIV><DIV id="01_08"><P>
Adrian 1_8
</P><TABLE><TBODY><TR><TD>PCR</TD><TD>reaction</TD></TR><TR><TD>H2O</TD><TD>222ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTPs</TD><TD>32ul</TD></TR><TR><TD>VF2</TD><TD>16ul</TD></TR><TR><TD>VR</TD><TD>16ul</TD></TR><TR><TD>GoTaq</TD><TD>2ul</TD></TR></TBODY></TABLE><P>
Cycle conditions
95ºC (3min) -&gt; 95ºC (30s) -&gt; 60ºC (30s) -&gt; 72ºC (1.5min) -&gt; 72ºC (5min)
for 25 cycles
Result: FP_Biobricks_Suffix and RP_Biobrick_Suffix primers are contaminated -&gt; rediute from stock
Maintenance Red
Workshop Protocol
</P><TABLE><TBODY><TR><TD>To make Fragment 1 for the</TD><TD>workshop</TD></TR><TR><TD>H2O</TD><TD>222ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTP</TD><TD>32ul</TD></TR><TR><TD>Primer 1 (FP_retarded)</TD><TD>16ul</TD></TR><TR><TD>Primer 2 (RP_retarded)</TD><TD>16ul</TD></TR><TR><TD>GoTaq</TD><TD>2ul</TD></TR><TR><TD>GFP plasmid</TD><TD>1ul</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>To make Fragment 2 for the</TD><TD>workshop</TD></TR><TR><TD>H2O</TD><TD>222ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTP</TD><TD>32ul</TD></TR><TR><TD>Primer 1 (OFP_esaRBSver2)</TD><TD>16ul</TD></TR><TR><TD>Primer 2 (RP_VR)</TD><TD>16ul</TD></TR><TR><TD>GoTaq</TD><TD>2ul</TD></TR><TR><TD>GFP plasmid</TD><TD>1ul</TD></TR></TBODY></TABLE><P>
Cycle conditions
95ºC (3min) -&gt; 95ºC (30s) -&gt; 60ºC (30s) -&gt; 72ºC (2.5min) -&gt; 72ºC (10min)
for 25 cycles
</P></DIV><DIV id="02_08">
Adrian 2/8
<P>
Recreating backbone </P><TABLE><TBODY><TR><TD>pSB1A2</TD><TD>EcoRI/PstI</TD></TR><TR><TD>H2O</TD><TD>22ul</TD></TR><TR><TD>pSB1A2-GFP</TD><TD>10ul</TD></TR><TR><TD>Buffer</TD><TD>4ul</TD></TR><TR><TD>EcoRI</TD><TD>2ul</TD></TR><TR><TD>PstI</TD><TD>2ul</TD></TR><TR><TD>Total</TD><TD>40ul</TD></TR></TBODY></TABLE></DIV><DIV id="03_08">
Adrian 3/8
<TABLE><TBODY><TR><TD>Fusion PCR for</TD><TD>workshop</TD></TR><TR><TD>H2O</TD><TD>111ul</TD></TR><TR><TD>Buffer</TD><TD>40ul</TD></TR><TR><TD>MgCl2</TD><TD>16ul</TD></TR><TR><TD>dNTP</TD><TD>16ul</TD></TR><TR><TD>OFP_esaRBS_GFP_ver2</TD><TD>8ul</TD></TR><TR><TD>ORP_XhoI_retarded</TD><TD>8ul</TD></TR><TR><TD>GoTaq</TD><TD>1ul</TD></TR><TR><TD>Fragment 1</TD><TD>1ul</TD></TR><TR><TD>Fragment 2</TD><TD>1ul</TD></TR></TBODY></TABLE><P>
95ºC (3min) -&gt; 95ºC (30s) -&gt; 60ºC (30s) -&gt; 72ºC (1.5min) -&gt; 72ºC (10min)
for 25 cycles
</P><TABLE><TBODY><TR><TD>EsaR Master Plate</TD><TD>Test</TD></TR><TR><TD>Colony PCR</TD></TR><TR><TD>H2O</TD><TD>222ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTP</TD><TD>32ul</TD></TR><TR><TD>VF2</TD><TD>16ul</TD></TR><TR><TD>VR</TD><TD>16ul</TD></TR><TR><TD>GoTaq</TD><TD>2ul</TD></TR><TR><TD>400ul</TD></TR></TBODY></TABLE><P>
For 10 colonies
</P></DIV><DIV id="17_07">
Ligation of esaRBS-GFP (EcoRI/PstI) and RNG-GFP (EcoRI/PstI) into pSB1A2 (EcoRI/PstI)
<TABLE><TBODY><TR><TD>10x T4 DNA ligase Buffer</TD><TD>1ul</TD></TR><TR><TD>Vector</TD><TD>1ul</TD></TR><TR><TD>Insert</TD><TD>5ul</TD></TR><TR><TD>H2O</TD><TD>10ul</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1ul</TD></TR></TBODY></TABLE><P>
Reactions were incubated for 30min at room temperature
Transformation
pSB1A2 esaRBSGFP
pSB1A2 RNG GFP
Control cut pSB1A2
</P><P>
Adrian 17/7
</P>
Ligation of esaRBS-GFP (EcoRI/PstI) and RNG-GFP (EcoRI/PstI) into pSB1A2 (EcoRI/PstI)
<TABLE><TBODY><TR><TD>10x T4 DNA ligase Buffer</TD><TD>1ul</TD></TR><TR><TD>Vector</TD><TD>1ul</TD></TR><TR><TD>Insert</TD><TD>5ul</TD></TR><TR><TD>H2O</TD><TD>10ul</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1ul</TD></TR></TBODY></TABLE><P>
Reactions were incubated for 30min at room temperature
Transformation
pSB1A2 esaRBSGFP
pSB1A2 RNG GFP
Control cut pSB1A2
</P></DIV><DIV id="18_07"><P>
Adrian 18/7
</P>
Vector dimer check
8 colonies were selected from the control plate (26/7)
<TABLE><TBODY><TR><TD>PCR</TD><TD>mastermix</TD></TR><TR><TD>H2O</TD><TD>222ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTP</TD><TD>32ul</TD></TR><TR><TD>FP_Biobricks_Prefix</TD><TD>16ul</TD></TR><TR><TD>RP_Biobricks_Prefix</TD><TD>16ul</TD></TR><TR><TD>GoTaq</TD><TD>2ul</TD></TR><TR><TD>Total</TD><TD>400ul</TD></TR></TBODY></TABLE><P>
The control plate had plasmids with a 800bp insert
Gel extract was performed on a 2kb band
Ligation Troubleshooting
</P><TABLE><TBODY><TR><TD>Ligation</TD><TD>Mastermix</TD></TR><TR><TD>H2O</TD><TD>222ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTP</TD><TD>32ul</TD></TR><TR><TD>FP_Biobricks_Suffix</TD><TD>16ul</TD></TR><TR><TD>RP_Biobricks_Suffix</TD><TD>16ul</TD></TR><TR><TD>GoTaq</TD><TD>2ul</TD></TR><TR><TD>Total</TD><TD>400ul</TD></TR></TBODY></TABLE><P>
Colony PCR
8 colonies were selected from esaR and RNG each, total 16 colonies
</P><TABLE><TBODY><TR><TD>PCR</TD><TD>Mastermix</TD></TR><TR><TD>H2O</TD><TD>444ul</TD></TR><TR><TD>Buffer</TD><TD>160ul</TD></TR><TR><TD>MgCl2</TD><TD>64ul</TD></TR><TR><TD>dNTP</TD><TD>64ul</TD></TR><TR><TD>FP_Biobricks_Suffix</TD><TD>32ul</TD></TR><TR><TD>RP_Biobricks_Suffix</TD><TD>32ul</TD></TR><TR><TD>GoTaq</TD><TD>4ul</TD></TR><TR><TD>Total</TD><TD>800ul</TD></TR></TBODY></TABLE><P>
Ran gel
Lanes
Vector Backbone (EcoRI/PstI
Control Ligation mix
esaR ligation mix
RNG ligation mix
T4 Ligation buffer
Blank
</P></DIV><DIV id="26_07"><P>
Adrian 26/7
</P>
BBPprefix_esaRBS PCR synthesis
<TABLE><TBODY><TR><TD>PCR</TD><TD>Reaction</TD></TR><TR><TD>H2O</TD><TD>22ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTP</TD><TD>32ul</TD></TR><TR><TD>FP_Biobricks_Prefix</TD><TD>16ul</TD></TR><TR><TD>ORP_esaRBS_fragsyn</TD><TD>16ul</TD></TR><TR><TD>synpartBBP_esaRBS</TD><TD>1ul</TD></TR><TR><TD>GoTaq polymerase</TD><TD>2ul</TD></TR><TR><TD>Total</TD><TD>400ul</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>esaRBS_GFP_BBsuffix</TD></TR><TR><TD>H2O</TD><TD>221ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTP</TD><TD>32ul</TD></TR><TR><TD>RP_BiobricksSuffix</TD><TD>16ul</TD></TR><TR><TD>OFP_esaRBS_GFPver2</TD><TD>16ul</TD></TR><TR><TD>pSB1A2_BBa_0040</TD><TD>1ul</TD></TR><TR><TD>GoTaq</TD><TD>2ul</TD></TR><TR><TD>Total</TD><TD>400ul</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>PCR for</TD><TD>esa</TD></TR><TR><TD>H2O</TD><TD>220ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTPs</TD><TD>32ul</TD></TR><TR><TD>FP_BB_Prefix</TD><TD>16ul</TD></TR><TR><TD>RP_BB_Suffix</TD><TD>16ul</TD></TR><TR><TD>BBPrefix_esaRBS</TD><TD>1.2ul</TD></TR><TR><TD>esaRBS_GFPBBSuffix</TD><TD>0.5ul</TD></TR><TR><TD>GoTaq</TD><TD>2ul</TD></TR><TR><TD>Total</TD><TD>400ul</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>PCR for</TD><TD>RNG</TD></TR><TR><TD>H2O</TD><TD>220ul</TD></TR><TR><TD>Buffer</TD><TD>80ul</TD></TR><TR><TD>MgCl2</TD><TD>32ul</TD></TR><TR><TD>dNTPs</TD><TD>32ul</TD></TR><TR><TD>FP_BB_Prefix</TD><TD>16ul</TD></TR><TR><TD>RP_BB_Suffix</TD><TD>16ul</TD></TR><TR><TD>BBPrefix_esaRBS</TD><TD>1.5ul</TD></TR><TR><TD>esaRBS_GFPBBSuffix</TD><TD>0.5ul</TD></TR><TR><TD>GoTaq</TD><TD>2ul</TD></TR><TR><TD>Total</TD><TD>400ul</TD></TR></TBODY></TABLE><P>
Adrian Clarice
</P><P>
REdigest of esaR and RNF after PCR purification and gel extraction
</P><TABLE><TBODY><TR><TD>esaR-gfp RE</TD><TD>digest</TD></TR><TR><TD>Bffer</TD><TD>2ul</TD></TR><TR><TD>PstI</TD><TD>1ul</TD></TR><TR><TD>EcoRI</TD><TD>1ul</TD></TR><TR><TD>2ug Template</TD><TD>5.4ul</TD></TR><TR><TD>H2O</TD><TD>10.6ul</TD></TR><TR><TD>Total</TD><TD>20ul</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>RNG RE</TD><TD>digest</TD></TR><TR><TD>Bffer</TD><TD>2ul</TD></TR><TR><TD>PstI</TD><TD>1ul</TD></TR><TR><TD>EcoRI</TD><TD>1ul</TD></TR><TR><TD>2ug Template</TD><TD>3.9ul</TD></TR><TR><TD>H2O</TD><TD>12.1ul</TD></TR><TR><TD>Total</TD><TD>20ul</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Ligation of pSB1A2 with esaR and</TD><TD>RNG</TD></TR><TR><TD>10X T4 Ligase Buffer</TD><TD>2ul</TD></TR><TR><TD>Vector (pSB1A2 EcoRI/PstI)</TD><TD>1ul</TD></TR><TR><TD>Insert (esaR and RNG)</TD><TD>5ul</TD></TR><TR><TD>H2O</TD><TD>11ul</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1ul</TD></TR><TR><TD>Total</TD><TD>20ul</TD></TR></TBODY></TABLE><P>
Ligation for 30min at room temperature
</P></DIV><DIV id="19_07"><P>
Clarice 19/7
</P><P>
Colony PCR for RNG and esaR-GFP colonies from master plate
</P><TABLE><TBODY><TR><TD>RNG</TD><TD>(10X)</TD></TR><TR><TD>H2O 13.8 + colony</TD><TD>each</TD></TR><TR><TD>Buffer</TD><TD>50</TD></TR><TR><TD>MgCl2</TD><TD>20</TD></TR><TR><TD>dNTPs</TD><TD>20</TD></TR><TR><TD>Primer 1</TD><TD>10</TD></TR><TR><TD>Primer 2</TD><TD>10</TD></TR><TR><TD>Gotaq</TD><TD>2</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>esaGFP</TD><TD>(9X)</TD></TR><TR><TD>H2O</TD><TD>45</TD></TR><TR><TD>Buffer</TD><TD>18</TD></TR><TR><TD>MgCl2</TD><TD>18</TD></TR><TR><TD>dNTPs</TD><TD>9</TD></TR><TR><TD>Primer 1</TD><TD>9</TD></TR><TR><TD>Primer 2</TD><TD>9</TD></TR><TR><TD>GoTaQ</TD><TD>1.8</TD></TR></TBODY></TABLE></DIV><DIV id="08_08brown"><P>
CY 8/8
</P><P>
8 minipreps of 4 clones from BL21 pNirB+gfp and 4 clones from dH5alpha pNirB+gfp
</P><TABLE><TBODY><TR><TD>RE</TD><TD>digest</TD></TR><TR><TD>EcoRI</TD><TD>9ul</TD></TR><TR><TD>PstI</TD><TD>9ul</TD></TR><TR><TD>Cutsmart Buffer</TD><TD>45ul</TD></TR><TR><TD>DNA of 1ug</TD><TD>6.7ul</TD></TR><TR><TD>H2O</TD><TD>36.3ul</TD></TR></TBODY></TABLE><P>
Result after running gel: All clones are positive for pNirB+gfp
</P></DIV><DIV id="13_08blue"><P>
CY 13/8
</P><P>
Invasin + Listerolysin blue
Inoculated 4 colonies of BL21 transformed with the inv+hly clone D.
</P></DIV><DIV id="14_08blue"><P>
CY 14/8
</P><P>
Invasin + Listerolysin blue
Miniprep of the inv+hly clones
</P><TABLE><TBODY><TR><TD>RE digest with</TD><TD>EcoRI/PstI</TD></TR><TR><TD>EcoRI</TD><TD>5ul</TD></TR><TR><TD>PstI</TD><TD>5ul</TD></TR><TR><TD>Buffer</TD><TD>25ul</TD></TR><TR><TD>DNA 1ug</TD><TD>(4ul)</TD></TR><TR><TD>H2O</TD><TD>11ul</TD></TR><TR><TD>Total</TD><TD>50ul</TD></TR></TBODY></TABLE></DIV><DIV id="19_08brown"><P>
CY 19/8
</P><P>
Colony PCR for pEFYP plated on (17/8)
Pcr for pNirBgfp as with (18/8) with pNirB-gfp primers, Biobricks prefix and suffix primers
</P><TABLE><TBODY><TR><TD>PCR Master mix For</TD><TD>pEYFP</TD></TR><TR><TD>H2O</TD><TD>13.25ul</TD></TR><TR><TD>Buffer</TD><TD>5ul</TD></TR><TR><TD>dNTP</TD><TD>5ul</TD></TR><TR><TD>F/R primer</TD><TD>5ul</TD></TR><TR><TD>2.5mM MgCl2</TD><TD>12.5</TD></TR><TR><TD>GoTaq</TD><TD>11.25ul</TD></TR></TBODY></TABLE><P>
PCR Master mix for pNirBgfp with pNirB-gfp primers, and Biobricks Prefix and Suffix Primers
(Melting temperature increased to 53degrees)
3ul of plasmid in each tube
</P><TABLE><TBODY><TR><TD>PCR Master mix for pNirB-gfp primers and BB prefix/suffix</TD><TD>primers</TD></TR><TR><TD>H2O</TD><TD>101.25ul</TD></TR><TR><TD>Buffer</TD><TD>45ul</TD></TR><TR><TD>dNTP</TD><TD>9ul</TD></TR><TR><TD>Forward Primer</TD><TD>9ul</TD></TR><TR><TD>Reverse Primer</TD><TD>9ul</TD></TR><TR><TD>2.5mM MgCl2</TD><TD>22.5ul</TD></TR><TR><TD>GoTaq polymerase</TD><TD>2.25ul</TD></TR></TBODY></TABLE></DIV><DIV id="21_07brown"><P>
CY 21/7
</P><P>
Ran gel for 20/7 colony pcr for FNR gfp
1kb, 100bp, colonies negative control
</P></DIV><DIV id="22_07brown"><P>
CY 22/7
</P><P>
Anaerobic promoter Brown
PCR to test 'FNR-GFP plasmid'.</P><P>Colonies 1-5 from 20/7 + negative control
3 sets of primers
FP_BB Prefix, RP BB Suffix
GFP-F, GFP-R
FP_FNRPromoter_BBP, ORP_FNR_Promoter GFP </P><P>For each reaction (6X) </P><P>H2O 73.5ul </P><P>dNTPs 6ul </P><P>Forward/Reverse Primer 6ul/6ul
MgCl2 15ul </P><P>Taq polymerase1.5ul </P><P>For each plasmid 1ul </P><P>Buffer 30ul </P></DIV><DIV id="24_07brown"><P>
CY 24/7
</P><P>
Ligation of Cut FNRGFP (17/7) with cut GFP vector
</P><TABLE><TBODY><TR><TD>1X</TD><TD>reaction</TD></TR><TR><TD>Buffer</TD><TD>2ul</TD></TR><TR><TD>200ng vector</TD><TD>1.1ul</TD></TR><TR><TD>3:1</TD><TD>256.5ng</TD></TR><TR><TD>Water</TD><TD>3.4ul</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1ul</TD></TR><TR><TD>Total</TD><TD>20ul</TD></TR></TBODY></TABLE><P>
Incubation at room temperature for 2h
Transformation into 10ul BL21 competent cells
</P></DIV><DIV id="27_07brown"><P>
CY 27/7
</P><P>
RE of extracted product with EcoRI/PstI for 300ng of DNA</P><TABLE><TBODY><TR><TD>RE</TD><TD>(1X)</TD></TR><TR><TD>FNR-gfp</TD><TD>0.9ul</TD></TR><TR><TD>EcoRI</TD><TD>1ul</TD></TR><TR><TD>PstI</TD><TD>1ul</TD></TR><TR><TD>Buffer</TD><TD>2ul</TD></TR><TR><TD>Water</TD><TD>15.1ul</TD></TR><TR><TD>Total</TD><TD>20ul</TD></TR></TBODY></TABLE><P>
CY YH
</P><P>
When gel was run, primer dimers and 700-800bp band were present in the negative control with no template added
</P><P>Fresh buffer, MgCL2, dNTP and fresh diluted primers were used to repeat colony PCR. </P><P>Diluting dATP, dCTP, dGTP, dTTP to 2.5mM </P><P>Repeated colony pcr with Prefix-suffix primers using TC pipettors </P></DIV><DIV id="15_07blue"><P>
HYT 15/7
</P><P>
Invasin + Listerolysin blue
RE digest of the miniprepped inv/hly plasmid with EcoRI/PstI from (20/7)
Tube C-&gt; 226ng/ul
Tube D-&gt; 298ng/ul
</P><TABLE><TBODY><TR><TD>RE</TD><TD>reaction</TD></TR><TR><TD>Buffer</TD><TD>3ul</TD></TR><TR><TD>Plasmid 4ul</TD><TD>each</TD></TR><TR><TD>EcoRI</TD><TD>1ul</TD></TR><TR><TD>PstI</TD><TD>1ul</TD></TR><TR><TD>H2O 21ul</TD><TD>each</TD></TR><TR><TD>Total</TD><TD>30ul</TD></TR></TBODY></TABLE><P>
incubated at 37degC for 2h
Add 6X loading dye to stop reaction in both tubes
Ran gel (1kb, 100bp, cut vector C, D)
</P></DIV><DIV id="24_07brown"><P>
CY 24/7
</P><P>
Anaerobic promoter Brown
Ligation of Cut FNRGFP (17/7) with cut GFP vector
</P><TABLE><TBODY><TR><TD>1X</TD><TD>reaction</TD></TR><TR><TD>Buffer</TD><TD>2ul</TD></TR><TR><TD>200ng vector</TD><TD>1.1ul</TD></TR><TR><TD>3:1</TD><TD>256.5ng</TD></TR><TR><TD>Water</TD><TD>3.4ul</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1ul</TD></TR><TR><TD>Total</TD><TD>20ul</TD></TR></TBODY></TABLE><P>
Incubation at room temperature for 2h
Transformation into 10ul BL21 competent cells
</P></DIV><DIV id="18_07brown"><P>
XY 18/7
</P><P>
Anaerobic promoter Brown
Transformation of FNRGFP ligated product into BL21 (from CY 17/7)
1. 3ul pUC19 control +10ul BL21
2. 20ul Ligation product + 20ul BL21
3. 5ul Ligation product + 20ul BL21
Plated on LB+amp plates
40ul of transformed bacteria in SOC media on LB+amp
20ul of transformed bacteria in SOC media on LB alone
</P></DIV><DIV id="02_08brown"><P>
YH 2/8
</P><P>
Inoculated colony 3,7 of FNR GFp selected positive clones for miniprep
</P></DIV><DIV id="03_08brown"><P>
YH 3/8
</P><P>
Anaerobic promoter Brown
Miniprep of colony 3, 7 and sent for sequencing with GFP-R and FP_VF2 -&gt; results show there is GFP but no FNR.
</P></DIV><DIV id="06_08brown"><P>
YH 6/8
</P><P>
Transform pNirB+gfp from the Biobricks Registry into BL21 and dH5alpha
</P></DIV><DIV id="09_08brown"><P>
YH 9/8
</P><P>
OD600 of 1 = 10^8 cells/cm^3
</P><TABLE><TBODY><TR><TD>OD600 of 10X</TD><TD>dilution</TD></TR><TR><TD>BL21 clone 1 0.013 -&gt; 1.04x10^8</TD><TD>cells/cm^3</TD></TR><TR><TD>BL21 clone 2 0.094 -&gt; 7.52x10^8</TD><TD>cells/cm^3</TD></TR><TR><TD>dH5alpha clone 1 0.054 -&gt; 4.32x10^8</TD><TD>cells/cm^3</TD></TR><TR><TD>dH5alpha clone 2 1.123 -&gt; 9.84x10^8</TD><TD>cells/cm^3</TD></TR><TR><TD>BL21 0.084 -&gt; 6.72x10^8</TD><TD>cells/cm^3</TD></TR><TR><TD>dH5alpha 0.046-&gt; 3.68x10^8</TD><TD>cells/cm^3</TD></TR><TR><TD>BL21 + pGFPuv 0.036 -&gt; 2.88x10^8</TD><TD>cells/cm^3</TD></TR></TBODY></TABLE><P>
The cultures were diluted 100X and 500X in a volume of 200ul in a 24well plate and grown in the anaerobic chamber at 30degrees and 225rpm for 2, 4, 6, 8 hours
</P><P>Replicating the iGEM Valencia Team - growing cultures under sterile oil (filtered through 0.22um filter)
0.5mL of stationary culture + 2.5mL LB, with 1mL of sterile oil added above </P><P>Grown at 28 degrees, 200rpm for 2 days </P><P>Results:</P><P> GFP signal is expressed in cultures with the anaerobic sensitive promoter grown in anaerobic conditions overnight, but signal was weak. 
After 6 hours in anaerobic growth, the BL21 + pNirB clone 1 had highest GFP signal when observed with the GFP microscope. </P><P>BL21 strain had no signal </P><P>After 24 hours in anaerobic conditions, dH5alpha had highest signal compared to control </P><P>Control - colonies picked directly from plates (aerobic conditions) had no GFP signal </P><P>After 2 days there was no gfp expression </P></DIV><DIV id="14_08blue"><P>
YH 14/8
</P><P>
Seeded cells for invasion assay
</P><P>BL21 + inv-hly clones correct after running gel for digest of the plasmids with EcoRI and PstI </P></DIV><DIV id="15_08blue"><P>
YH 15/8
</P><P>
Invasin + Listerolysin blue</P><P>
Invasion assay with HEK293T cells and BL21, and BL21+inv-hly plasmid on 15/08/15 </P><P>Rationale: The invasin gene in the invasin+listerolysin plasmid should remain intact and be expressed, leading to an invasion phenotype </P><P>Objective: Screen invasion phenotype of BL21 and BL21+invasin-listerolysin Biobricks part </P><P>Experiment details </P><P>Cells were passage 2
Seed 0.5mL of 5x10^5 HEK293T cells/mL and 1x10^6 in wells of 24-well plates, grow overnight. </P><P>Determine cell confluency with microscopy the next day - 80-90% confluency </P><P>All strains were grown overnight in 3mL LB broth </P><P>For 5x10^5 cells </P><P>100ul of 10^9 cells/mL was used for moi 200 </P><P>100ul of 2X dilution of 10^9 cells/mL was used for moi 100 </P><P>100ul of 4X dilution of 10^9 cells/mL was used for moi 50 </P><P>100ul of 5X dilution of 10^9 cells/mL was used for moi10 </P><P>Result
Lawn on all plates
HEK293T cells detach too easily. </P><P>Grow for 24 hours in the future and do minimal, and gentle washing.
For next experiment, 0.5x10^6 and 1x10^6 cells wre seeded
1000ug/mL kanamycin for 1 hour is sufficient for kill step </P><P>no contamination in uninfected control </P><P>There is invasion in background BL21 strain as well </P><P>All mois showed high percentage of infection </P><P>In future use moi 10:1 and plate up 10^2 to 10^5 dilution for countable colonies</P></DIV><DIV id="17_07brown"><P>
YH 17/7

For plates on LB alone there was a lawn
</P><P>For pUC19, the transformation worked, and separated colonies were produced </P><P>No colonies were produced for the FNR-gfp plasmid </P></DIV><DIV id="18_08brown"><P>
YH 18/8
</P><P>
PCR for the clones of pNirB-gfp from B21(1-4) and dH5alpha (1-4)with pNirB-gfp-F/R primers
</P><P>5ul plasmid, 5ul H2O in each tube </P><P>PCR Mastermix </P><P>dNTP 8ul </P><P>MgCl2 20ul </P><P>Taq Polymerase 2ul</P><P> Buffer 40ul
F/R </P><P>primers 0.8ul each </P><P>H2O 40ul </P></DIV><DIV id="19_07blue"><P>
YH 19/7

inoculated colonies C,D for invasin plasmid in 3mL LB+amp at 1.30pm </P><P>Transformed remaining 40u ligation reaction for FNR gfp into 10ul BL21 </P><P>Transformed 100ng pGFPuv plasmid into 10ul BL21 </P><P>Poured new LB+amp plates </P><P>Plated 20, 40, 100ul of transformed bacteria in SOC media on plates </P></DIV><DIV id="20_07brown"><P>
YH 20/7
pGFPuv transformation worked!
</P><P>Single colony for FNR-GFP + 4 more from plating on 19/7</P><P> Sent FNR-GFP fusion product for sequencing with RP-Biobricks Suffx - result there is no FNR but there is no GFP </P><P>Colony PCR for FNR gfp (Colonies 1-5, (-) control) </P><TABLE><TBODY><TR><TD>Colony PCR</TD><TD>(6X)</TD></TR><TR><TD>H2O</TD><TD>19.5ul</TD></TR><TR><TD>Buffer</TD><TD>10ul</TD></TR><TR><TD>6ul</TD><TD>dNTP</TD></TR><TR><TD>6ul FP_Biobricks</TD><TD>Suffix</TD></TR><TR><TD>6ul RP_Biobricks</TD><TD>SUffix</TD></TR><TR><TD>15ul</TD><TD>MgCl2</TD></TR><TR><TD>1.5ul Taq</TD><TD>Polymerase</TD></TR></TBODY></TABLE></DIV><DIV id="21_07brown"><P>
YH 21.7

Redid colony pcr for above as negative control had a positive band - contamination of PCR
</P><P>negative control still has band - reagents contaminated </P></DIV><DIV id="22_07brown"><P>
YH 22/7
</P><P>
Miniprepped colonies 1-5 for FNR-gfp plasmid
</P><P>Did EcoRI/PstI digest </P><TABLE><TBODY><TR><TD>Digest for mastermix</TD><TD>(5X)</TD></TR><TR><TD>Buffer</TD><TD>10ul</TD></TR><TR><TD>EcoRI/PstI</TD><TD>2ul</TD></TR><TR><TD>H2O</TD><TD>60ul</TD></TR><TR><TD>Total 20ul per</TD><TD>tube</TD></TR><TR><TD>400ng plasmid in each</TD><TD>tube</TD></TR></TBODY></TABLE><P>
Ran gel for RE samples, none were postive clones
</P></DIV><DIV id="26_07brown"><P>
YH 26/7

No colonies for FNRgfp
</P><TABLE><TBODY><TR><TD>Hotstart fusion PCR (8X</TD><TD>reaction)</TD></TR><TR><TD>10X Buffer</TD><TD>80ul</TD></TR><TR><TD>dNTPs</TD><TD>16ul</TD></TR><TR><TD>FP Biobricks Prefix</TD><TD>8ul</TD></TR><TR><TD>RP Biobricks Suffix</TD><TD>8ul</TD></TR><TR><TD>FNR Pcr product</TD><TD>12ul</TD></TR><TR><TD>GFP PCR product</TD><TD>4ul</TD></TR><TR><TD>MgCl2</TD><TD>3ul</TD></TR><TR><TD>H2O</TD><TD>668ul</TD></TR></TBODY></TABLE></DIV><DIV id="27_07brown"><P>
YH 27/7

Ran gel with cut vector and cut PCR product
1kb 100bp cut vector (wells 3-5) FNR-gfp (wells 6-8)
</P><P>However, PCR product was not detectable </P><TABLE><TBODY><TR><TD>RE digest for FNRgfp</TD><TD>EcoRI/PstI</TD></TR><TR><TD>150ng PCR</TD><TD>5ul</TD></TR><TR><TD>Buffer</TD><TD>2ul</TD></TR><TR><TD>EcoRI</TD><TD>0.5ul</TD></TR><TR><TD>PstI</TD><TD>0.5ul</TD></TR><TR><TD>H2O</TD><TD>12ul</TD></TR><TR><TD>Total</TD><TD>20ul</TD></TR></TBODY></TABLE><P>
Gel extracted
Ligation reaction was performed overnight
</P><TABLE><TBODY><TR><TD>Ligation reaction</TD><TD>(1X)</TD></TR><TR><TD>100ng vector</TD><TD>8.3ul</TD></TR><TR><TD>85ng insert</TD><TD>16ul</TD></TR><TR><TD>T4 ligase</TD><TD>2ul</TD></TR><TR><TD>Buffer</TD><TD>4ul</TD></TR><TR><TD>H2O</TD><TD>9.7ul</TD></TR></TBODY></TABLE></DIV><DIV id="28_07red"><P>
YH 28/7
</P><P>
Trialling transformation of 10ul BL21 with 100ng pGFPuv plasmid for workshop. </P><P>Using 200ul LB broth for the growth in 1hour step </P><P>10ul BL21 + 10ul plasmid
Heatshock for 300s </P><P>Shake for 1hour at 37 degC 225rpm </P><P>plated 40ul on LB, 40ul on LB+amp plate</P><P> Transformation of ligation reaction into BL21 with control of cut vector (100ng) into BL21 </P><P>plated 40ul on LB+amp plate, 20ul on LB plate </P></DIV><DIV id="29_07red"><P>
YH 29/7

Colonies on pGFPuv plate with amp -&gt; LB works for reviving bacteria
small colonies with FNRGFP plasmid
</P></DIV><DIV id="30_07brown"><P>
YH 30/7

12 colonies grew for FNRGFP plate
no colonies on control with cut vector
Colony PCR to screen 12 colonies and negative control using universal primers
</P><TABLE><TBODY><TR><TD>PCR Mastermix</TD><TD>(13X)</TD></TR><TR><TD>H2O</TD><TD>42.25ul</TD></TR><TR><TD>PCR Buffer</TD><TD>65ul</TD></TR><TR><TD>VF2 Primer</TD><TD>13ul</TD></TR><TR><TD>RP VR Primer</TD><TD>13ul</TD></TR><TR><TD>MgCl2</TD><TD>32.5ul</TD></TR><TR><TD>GoTaq</TD><TD>3.25ul</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="content content-full bigDiv" id="human_practice" style="height:700px;overflow:scroll"><P>[some introduction here]</P><DIV id="accordionHumanPractice"><H3>Workshop</H3><DIV><P><EM>One of the participants taking a closer look at GFP-tagged E. coli under our very own SPS microscope.</EM></P><P>The SPS iGEM Team of 2015 hosted a genetic engineering workshop for students from the Faculty of Science on 5th August 2015, in the Active Learning Room and the SPS Wet Lab. The workshop aimed to equip science students with an understanding of both the techniques of synthetic biology, and its risks and rewards. Participants were given the opportunity to be immersed in both the theoretical and wet lab components of synthetic biology.</P><P>Students were first guided through the concepts of genetic engineering, and the available wet lab tools and techniques used. After some light refreshments, they then got a chance to try their hands at designing their very own gene vectors with a fun set of theoretical puzzles.</P><P> </P><P><EM>One of the workshop facilitators explaining the process of constructing a genetic vector.</EM></P><P><EM>Participants and facilitators hard at work figuring out genetic puzzles</EM></P><P>After lunch, the participants performed Fusion PCR (Polymerase Chain Reaction) and performed bacterial transformation in the SPS Wet Lab. They also had a look at green fluorescent protein (GFP) expressed in <EM>E. coli</EM>, as an example of one of the methods that are commonly used to quantify protein expression.</P><P><EM>Participants beginning PCR in the SPS Wet Lab!</EM></P><P><EM>Loading a gel is hard work – participants ran a DNA gel to confirm if their PCR reaction was successful</EM></P><P>All in all, both the workshop participants and facilitators spent an enjoyable day both learning and sharing about genetic engineering. The SPS iGEM Team of 2015 would like to thank all participants for spending their day with us! We would also like to thank Science Dean’s Office for their kind sponsorship, as well as the SPS staff and SPS community for their support.</P><P><EM>A final group photograph with some of the facilitators and workshop participants</EM></P><P><STRONG>Video</STRONG></P></DIV><H3>Interview</H3></DIV></DIV><DIV class="content content-full bigDiv" id="safety" style="height:500px;overflow:scroll"><STRONG>Biosafety in our project involves minimising the risks to the researchers working in the laboratory, as well as the general public in future medical applications based off our research.</STRONG><DIV id="safetyAccordions"><H3>Safety when handling biological organisms</H3><DIV><P>
Non-pathogenic strains of E. coli K-12 strains BL21 and dH5α from Life Technologies were used for bacterial cloning of plasmids and expression of proteins of interest. These strains are Risk group 1 and were handled in a BSL2 Biosafety cabinet. The E. coli strain carrying the Biobrick BBaK299812 (containing parts derived from Risk group 2 organisms) was handled as a Risk Group 2 agent. Mammalian cell line HEK293T is classified under Risk group 2, and was also cultured in a BSL2 Biosafety cabinet.
</P></DIV><H3>Safety in Project Design</H3><DIV><P>
In our project, we aim to engineer non-pathogenic E. coli as a vector to deliver a potential drug into the tumour core. We use the Biobricks Part BBa_K299812 (http://parts.igem.org/wiki/index.php?title=Part:BBa_K299812), which contains the invasin gene from Yersinia pseudotuberculosis and the listerolysin O gene from Listeria monocytogenes. The Invasin protein allows for bacteria to enter mammalian cells, while Listerolysin O is a pore-forming protein that enable bacteria to escape the endosome. These two proteins are involved in pathogenesis of their respective bacterial species.
</P><P>
The Invasin and Listerolysin proteins enable our E. coli to enter mammalian cells, and escape the endosome, where they can subsequently deliver an encoded therapeutic to kill the tumour cell. To ensure that these proteins are only expressed under the conditions of the tumour microenvironment, the invasin and listerolysin proteins will be placed under the control of an anaerobic promoter, and a quorum sensing system. 
</P></DIV><H3>Safety in Our Lab</H3><DIV><P>All our team members have undergone Chemical, Biological and Fire Safety Training from the Office of Safety, Health and Environment (OSHE http://www.nus.edu.sg/osh/), the department in charge of Laboratory and Work Safety at the National University of Singapore</P><P> For each protocol used for our experiments, we have a separate risk assessment. Please refer to our ‘protocols’ page for more information. </P><P>Our laboratory is equipped with biological and chemical spill kits, and all members of our iGEM Team are trained to handle Biological and Chemical Spills.
Our laboratory is classified as Biosafety Level 2, according to the classification by the Wolrd Health Organisation (WHO) and the Genetic Modification Advisory Committee of the government of Singapore (http://www.gmac.gov.sg/).</P><P> Bacterial work and Mammalian cell culture are performed in separate BSL2 Biosafety Cabinets, while DNA work is done on the bench. No cytotoxic reagents are used in our laboratory; Sybr Safe DNA stain is used rather than Ethidium Bromide.
Liquid biological waste is decontaminated using 10% Bleach, while Solid biological waste is sent for incineration in a local incineration plant devoted to medical waste (Sembcorp http://www.sembcorp.com/en/business-on-site-services-solid_waste_management.aspx). </P></DIV><H3>Safety Requirements for iGEM Participation</H3><DIV><P>For the fulfillment of requirements for safety from the iGEM foundation, we have submitted the ‘About our lab’ safety forms (https://2015.igem.org/Safety/About_Our_Lab?team_id=1804) and the ‘Final Safety form (Yihan will complete this later to the deadline as currently can’t confirm what we are submitting to parts registry. Just put in first)’ </P><P>We have also performed a check-in (https://2015.igem.org/Safety/Check_In) for the Biobricks Part (Bba_k299812 http://parts.igem.org/wiki/index.php?title=Part:BBa_K299812), which contains the invasin gene from Yersinia pseudotuberculosis and the listerolysin O gene from Listeria monocytogenes. </P></DIV></DIV></DIV><DIV class="content" id="requirement" style="height:500px"><DIV id="requirementAccordions"><H3>Gold</H3><DIV><P><STRONG>In addition to the Bronze and Silver Medal requirements, your team must convince the judges you have achieved at least two of the following goals:    </STRONG></P><P>1. Choose one of these two options: (1) Expand on your silver medal Human Practices activity by demonstrating how you have integrated the investigated issues into the design and/or execution of your project. OR (2) Demonstrate an innovative Human Practices activity that relates to your project (this typically involves educational, public engagement, and/or public perception activities; see the Human Practices Hub for information and examples of innovative activities from previous teams).  </P><P>2. Help any registered iGEM team from a high-school, different track, another university, or institution in a significant way by, for example, mentoring a new team, characterizing a part, debugging a construct, modeling/simulating their system or helping validate a software/hardware solution to a synbio problem.  </P><P>3. Improve the function OR characterization of a previously existing BioBrick Part or Device (created by another team, or by your own team in in a previous year of iGEM), and enter this information in the part's page on the Registry. Please see the Registry Contribution help page for help on documenting a contribution to an existing part. This part must not come from your team's 2015 range of part numbers.
    Demonstrate a functional prototype of your project. Your prototype can derive from a previous project (that was not demonstrated to work) by your team or by another team. Show this system working under real-world conditions that you simulate in the lab. </P></DIV><H3>Silver</H3><DIV><P><STRONG>In addition to the Bronze Medal requirements, your team must convince the judges you have achieved the following 3 goals:    </STRONG></P><P>1. Experimentally validate that at least one new BioBrick Part or Device of your own design and construction works as expected. Document the characterization of this part in the Main Page section of the Registry entry for that Part/Device. This working part must be different from the part you documented in Bronze medal.    </P><P>2. Submit this new part to the iGEM Parts Registry. This part must be different from the part you documented in Bronze medal. (Submissions must adhere to the iGEM Registry guidelines.)    </P><P>3. iGEM projects involve important questions beyond the bench, for example relating to (but not limited to) ethics, sustainability, social justice, safety, security, and intellectual property rights. We refer to these activities as Human Practices in iGEM. Demonstrate how your team has identified, investigated and addressed one or more of these issues in the context of your project.</P></DIV><H3>Bronze</H3><DIV><P>1. Register for iGEM, have a great summer, and attend the Giant Jamboree.    </P><P>2. Complete the Judging form.
    Create and share a Description of the team's project using the iGEM wiki, and document the team's parts using the Registry of Standard Biological Parts.    </P><P>3. Present a poster and a talk at the iGEM Jamboree.    </P><P>4. Create a page on your team wiki with clear attribution of each aspect of your project. This page must clearly attribute work done by the students and distinguish it from work done by others, including host labs, advisors, instructors, sponsors, professional website designers, artists, and commercial services. </P><P>5. Document at least one new standard BioBrick Part or Device central to your project and submit this part to the iGEM Registry (submissions must adhere to the iGEM Registry guidelines). You may also document a new application of a BioBrick part from a previous iGEM year, adding that documentation to the part's main page. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>