"
Page
Discussion
View source
History
teams
Log in
 
Team:Macquarie Australia/WetLab/Notebook
From 2014.igem.org
Home
Project
Background
Overview
Results
Significance
Parts & Characterization
Modelling
Medal Requirements
Our Team
Students
Instructors & Advisors
The University
Attributions & Sponsors
Wet Lab
Notebook
Protocols
Safety
Policy & Practice
Overview
SYTYCS
MQU Open Day
Powerhouse Museum
MQ Research Interns
Business Proposal
Sponsorship Support
SynBio & AUS Law
iGEM
The Notebook
Welcome to our Notebook! This page has been created to detail our projects progress since its conception. Click on the section titles below to expand pages that contain information regarding what we worked on and achieved each week. For a more detailed summary of what we achieved please visit the Results page.
The full Team_Macquarie_2014 Lab Notebook may be found here:
Team_Macquarie_2014 Lab Notebook
November 2013
Week 1
Biobrick stocktake of 2013 iGEM Macquarie_Australia parts: 11/11/13
ChlG - sufficient plasmid stock
DVR1 - sufficient plasmid stock
ChlM - sufficient plasmid stock
ChlI2 - need more plasmid stock
POR- need more plasmid stock
YCF - need more plasmid stock
Plasto - need more plasmid stock
GUN4- need more plasmid stock
CTH1 - need more plasmid stock
ChlD - need more plasmid stock. Question whether the 2013 part is really the reported sequence - something appears to be missing.
Send all for re-sequencing to verify DNA sequence as per registry entries.
DVR1 re-tranformation: Was re-done using gibson assembly and then transformed.
Week 2
Sequencing Results: all parts except ChlD were correct.
ChlD Fix:  ChlD is missing 50 bp. Strategy to correct is to use ApaI and MluI restriction enzymes to cut out 50bp from clone of ChlD in pET vector from Willows group and re-insert into our BioBrick vector.
ApaI and MluI were used in a single digest according to manufacturer's instructions and as per ligation protocol on methods wiki. Fragments run on 1% agarose and gel purified. However, digestions were incomplete as viewed on agarose gel. Need to do separate digests for next attempt.
Double restriction enzyme digest was carried out to combine PCR1 and Gblock2. After the two sections were ligated and extended, straight PCR was done. The PCR worked as judged by agarose gel.
Week4
Tuesday: 26/11/13 Composite parts Assembly
Biobrick (BB) ChlI1 is combined with ChlI2 biobrick in AMP backbone. Method is via 3A assembly. Use 500ng of each part and insert into 500ng of amp backbone. Ligation for 16oC for 30 mins then 80oC for 20 mins. Leave plates over weekend at room temperature.
Growth on plates : 1 colony on low plate, hundreds on high plate.
Assembly of ChlH: PCR of individual fragments from ChlH: 29/11/13
G1 - G1F + G1R
G2 - G2F+ G2R
G3 - G3F + G4R
G4 - G4F+ G4R
G5 - G5F + G5R
G6 - G6F + G6R
PCR1+ G2- H1F+ G2R
(PCR1 + G2) + G1
G3 + PCR2- G3F+ H2R
G5 + G6- G5F +G6R
Increase stocks : Did plasmid preps to get more of: ChlI1; ChlI2; YCF54; ChlP, DVR1; POR
ChlH Biobrick correction
Attempt to make ChlH (BBa_K1080001) using combination of gblocks and PCR products, as designed by Macquarie_Australia 2013 iGEM team.
Assembly strategy is: G-Block –1 (470bp) + PCR-1 (304bp) + G-Block-2 (499bp) + G-Block-3 (499bp) + PCR-2 (984bp) + G-Block-4 (500bp) + G-Block-5 (481bp) + PCR-3 (673bp)
Extremely faint bands are seen for
Friday: 29/11/13
Another attempt to assemble chlD from PCR fragments
G Block 1
G Block 4
G4 + (G5-G6)
G1 + PCR1
G2 + (G3 + PCR2)
Figure 1 Extremely faint bands seen for ChlD amplification. G1 and G4 appear to work but bands are very faint on agarose gel. Faint to no bands viewed for G2 and G3+PCR2. Reattempt necessary.
December 2013
Week 1
Friday: 06/12/13 Digestion & Ligation of ChlM gene of lac promoter into CAM backbone
ChlM in AMP backbone vector and lac in backbone were digested using iGEM restriction digestion protocol EcoRI and Pst1 restriction enzymes.
Tuesday
10/12/13
The fragments to be PCR’d and the primers are presented on the following table;
Fragments to PCR
Primers
G1
BBF+G1R
G1+P1
BBF+H1R
G2+(G3-P2)
G2F+P2R
ChlI1
BBVF2+BBVR
ChlI2
BBVF2+BBVR
ChlD
BBVF2+BBVR
Small amount of growth seen on ChlM, lacA & lacB indicating that they were successfully incorporated into the DHS←  cells. Sequencing to confirm required
Transformation of Kanamycin Resistant backbone
We need more of the kanamycin biobrick. Transformation of kanamycin backbone into E. coli cells to produce large amounts of KAN backbone for future ligations.
Monday
09/12/13
PCR reaction for ChlH and ChlD
The overall of the aim of the week was to build ChlH fragment and PCR ChlD. Using the standard PCR protocol, G1+H1, G2 (G3+H2), G4 (G5+G6), ChlD (2) and ChlD (3) were run.
The result showed another G1+PCR1 failure. It was also suggested however to use BioBrick primers. Distinct bands for G2+(G3/PCR2) and G4+(G5/G6) were present and proved correct. This assumption was made that these results were correct.
ChlD 2 and 3 showed a band present at approximately 1500 bp which was also assumed to be correct in relation to the actual size of 1681 bp.
Figure2: The next step was to rePCR G1+PCR1 with BBF + HR2 and BBvF + HR2, gel extraction of G2+(G3/H2) and G4+(G5/G6) ,ChlD 2 and 3.
Tuesday
10/12/13
The fragments to be PCR’d and the primers are presented on the following table;
Fragments to PCR
Primers
G1
BBF+G1R
G1+P1
BBF+H1R
G2+(G3-P2)
G2F+P2R
ChlI1
BBVF2+BBVR
ChlI2
BBVF2+BBVR
ChlD
BBVF2+BBVR
The standard PCR Method was adopted to run the reaction
Figure 3: All but ChlI1 and ChlI2 failed
Continued ChlH construction
At this stage, the ChlH gene construct was continued;
G1-P1-G2-G3-P2-G4-G5-G6
The ChlD gene was cut from the gel and extracted with another attempt to PCR.
To test for protein expression, the successful 3A gene was combined with lac creating a composite.
Continuing the construct of ChlH, a PCR reaction was performed to identify the successful or unsuccessful attempt in the composite build in addition to DVR1 identification.
Table 2: The PCR reaction screening attempting to construct chlH failed
Gene fragment
Primers
P1+G2
H1F+G2R
(G3+P2)+(G4-G5-G6)
GBF+G6R
DVR1
BBVF2+BBVR
Figure 4 chlH and DVR1 Gibson assembly gel
Digest of DVR1
The next step was the insertion of DVR1 into the plasmid vector. The plasmid vector and the plasmid containing the gene of interest were ligated with EcoR1 and Pst1. The gene was introduced into the vector my means of 1 vector to 3 insert to maximise insertion efficiency.
ChlH construction by Gibson assembly
The failure of the construction of the ChlH gene subjected the attempt in the construction of the gene using Gibson assembly. The provided gel image proved the construction also failed.
Figure 5 PCR of ChlH Gibson Assembly
January 2014
Tuesday
ChlH construct PCR
In the attempt to yield a positive result in the construction of ChlH, each P1+G2, (G3+P2) and (G4-G5-G6) were PCR’d separately in the attempt to successfully join the individual components.
10/12/13
Table 3 PCR of ChlH fragments
Gene fragment
Primer
P1-G2
P1F + G2R
G3-P2
G3F + P2R
G4-G5-G6
G4F + G6R
(P1-G2) + (G3-P2)
P1F + P2R
(G3-P2) + (G4-G5-G6)
G3F + G6R
ChlD
ChlD F + ChlD R
Thursday: 09/01/14
Gel analysis of PCR gel of ChlH constructs and ChlD
The results obtained would indicate the band to extract for Gibson assembly. The gel image showed positive results
The marked were cut out and stored for gel extraction. Figure 6 PCR gel analysis of ChlH constructs and ChlD. To compare the sizes, 25-500ng of plasmid were digested with and without lac. The expected size was approximately 200 bp. The amplification of ChlD was faint indicating an issue with the construction of the gene.
Friday
10/01/14
ChlH screening
The bands on gel corresponding to ChlH were extracted to screen for the correct sizes. The result of the gel extraction showed low concentration indicating poor construction of gene.
Figure 7 ChlD/ ChlI Plastocyanin screening: Digests were performed with enzymes EcoR1 and Pst1 to comment on the sizes of the inserts including ChlD, ChlI1 and Plastocyanin. These were also run against the corresponding components including lac.
Saturday
11/01/14
Table 4: Continued construction for ChlH PCR
Gene fragment
Primers
P1-G2
F1 + G2R
G3-P2
G3F + P2R
G4-G5-G6
G4F + G6R
CHlD
NF2 + NR2
Figure 8 The results obtained from the gel yielded a successful result for P1-G2, responsible for the construction of CHlH and negative results for the remaining samples on the gel.
Thursday
30/01/14
PCR: It is thought that the excess template in the previous PCR may have been responsible for the failure of PCR amplification. Template dilutions of 1/10 and 1/100 were tested by running another pcr.
The PCRs carried out were:
ChlD (new template), diluted
G3 -P2 PCR template
ChlH gel run + extracted template
Figure 9 ChlD and ChlH G3-P2 PCR template did not work, however, ChlH G3 -P2 PCR template was successful.
PCR for ChlD blocks:
G4 + (G5-G6) X3 = G4F + G6R
P1- G2 (from the original templates) x3 = P1F +G2F.
Figure 10 G4 + (G5-G6) and P1- G2 PCRs worked. G4 + (G5-G6) showing a band of 1700 bp in length and P1-G2 showing 800 bp in length.
PCR continuation:
(P1-G2) + (G3-P2)
G3-P2 + G4-G5-G6
Figure 11 None of the PCRs from the ChlD blocks worked - clear, desired bands were not found
Feburary 2014
Week 1
Monday
3/2/2014
Digestion of DVR1 was run.
Following the digest, a ligase reaction was conducted and transformation performed. Note, the concentration of DVR part in comparison to the concentration of the plasmid was 1.5 times more. Plates incubated overnight.
Protein expression of lac+plasto & lac+ChlI
Expression of protein via lac promotor using 2uL of IPTG was done for each sample to amplify protein expression.SDS-PAGE was run according to methods.
Tuesday
4/2/14
SDS PAGE attempt #2
Here we conducted a second SDS PAGE for lac plasto and lac ChI1.
Lane order: 1-4 lac plasto, 5 is the ladder, 6-10 lac ChlI1
Expression of proteins was not visible by eye. No image of the gel was recorded. We think we need to do mass spec (MALDI/TOF/TOF) to identify proteins in bands. Discuss with APAF (Australian Proteomic Analysis Facility) at Macquarie University to ask if they can help us with performing mass spec.
Testing new ligase: New ligase purchased as concerns were that our ligase was old and the reason ligations were not successful
ChlH was digested with E+P restriction enzymes as per methods. To ligate, the ligation mixture comprised of 8.5uL DNA, 0.5uL ligase and 1uL of buffer.
Figure 12 The gel showed inconclusive results and requires further clarification. Further testing methods such as re-inserting the biobrick into another vector and growing it on a plate with the second vector antibiotic have been suggested.
PCR:
Fragment 1 - (P1-G2) + (G3-P2) = P1F, P2R
Fragment 2- (G3-P2) + (G4-G5-G6) = G3F, G6R
Fragment 3- (P1-G2) + (G3-P2) + (G4-G5-G6) = P1F, G6R.
For such large fragments, the preliminary melting step was completed twice prior to the addition of the primers because of the long fragments. The rest of the process was continued on the regular loop as in other PCR protocol.
Wednesday
5/2/14
Western Blot:
Western blot for ChlI1 and plasto were carried out.
Results: The plasto lanes did not show any expression, however ChlI showed good expression in lanes 2,4 and 5.
Figure 13 The gel showed inconclusive results and requires further clarification. Further testing methods such as re-inserting the biobrick into another vector and growing it on a plate with the second vector antibiotic have been suggested.
Thursday 6/2/14: Gibson Assembly of ChlH:
G1: 3uL
P1-G2 exosap: 0.5uL
G3-P2 exosap: 4.8uL
G4-G5-G6 exosap: 1.0uL
Cam vector: 29uL with Gibson mix: 12.3uL or AMP vector 1.4uL with Gibson mix: 10.8 uL
- Plated out
PCR of Gibson Assembly product for ChlH: Standed PCR x2 using BBF/ BBR/ BBVF2, BBVR.
Results:
Week 2
Wednesday
12/2/14
Nanodrop of ChlH fragments
ChlH Fragments:
P1-G2:
A= 19.3 ng/ml
B= 23.4 ng/m
C= 18.2 ng/m
G3-P2:
A = 27.8 A= 141ng/ml
B= 9.9 ng/ml
C= 47.5 ng/ml
G4-G5-G6:
A= 141ng/ml
B= 24.6 ng/ml
C= 62.5 ng/ml
ChlD Fragments: D1, D2, D3
Thursday
13/2/14
Gel Electrophoresis for ChlH and ChlD fragments:
Top Gel Lane order: 1- Ladder, 2- G1, 3-5 - P1-G2, 6-8- G3-P2, 9-11- G4-G5-G6, 12-14- ChlD
Bottom Gel Lane Order: 1+ 2- Ladder, 3-5- P1-G2, 6-8- G3-P2, 9-11 G4-G5-G6
Results: ChlH fragments appear not have been digested. P1-G2 and G4-G5-G6 didn’t have plasmids on the gel so they did not digest. ChlD has digested with MLU and partially APAI. ChlD plasmids from lanes 12 and 13 were added together for further re-digestion.
New Digestion using E+P from previous gel electrophoresis for ChlH:
Lane Order: 1- G1, 2- G3-P2 A, 3- G3-P2 B, 4- G3-P2 C, 5- G4-G5-G6 A, 6- G4-G5-G6 C, 7- Plasto Control
New Digest for ChlD with APAI: The ChlD being redigested is a combination of lanes 12 and 13 from the previous electrophoresis gel.
ChlD Ligation : ChlD was ligased and transformed into E.coli. 2 colonies grew on the 300uL plate and 1 colony on the 30uL plate.
Friday 14/2/14
Plasmid nanodrop
Lac
Gun
A
33.8 ng/µl
 
 
B
17.8 ng/µl
Lac
ChlM
A
33.5 ng/µl
 
 
B
26.8 ng/µl
Lac
ChlP
A
32.2 ng/µl
 
 
B
14.4 ng/µl
Lac
ChlI2
A
18.5 ng/µl
 
 
B
15.2 ng/µl
Lac
POR
A
35.1 ng/µl
 
 
B
24.5 ng/µl
Lac
ChlG
A
8.3 ng/µl
 
 
B
13.2 ng/µl
Lac
YCF54
A
29.3 ng/µl
 
 
B
33.9 ng/µl
Lac
CTH1
A
59.1 ng/µl
 
 
B
51.6 ng/µl
Table 6 ; : Biobrick concentrations
ChlH re-digest:
Digests checking biobricks:
With/without lac
Biobrick
Fragment
Expected weight
Measured
Lac
Gun
A
930
~930
 
 
B
 
~930
Lac
ChlM
A
873
~1050
 
 
B
 
~1050
Lac
ChlP
A
1299
~1500
 
 
B
 
~1500
Lac
ChlI2
A
1212
~1400
 
 
B
 
~1250
Lac
POR
A
1067
~1250
 
 
B
 
~1250
Lac
ChlG
A
1050
~1250
 
 
B
 
~1250
Lac
YCF54
A
471
~650
 
 
B
 
~650
Lac
CTH1
A
1152
~1350
 
 
B
 
~1350
 
DVR1
A
 
~1106
 
 
B
 
~1106
 
 
C
 
~1106
 
 
D
 
~1106
Results: Majority look as though they match the expected band length. All digests excluding DVR1 include lac.
Winter Lab Session
AUGUST (SEMESTER 2)
WEEK 1
Thursday
07/08/14
Off and running with the whole team of 12 Biomolecular Major students. We sat through a full day of learning about iGEM; we had a discussion of project goals and aims; as well as a refresher course on how the Chlorophyll pathway works. Roles were assumed by our wiki-chiefs and those interested in gaining sponsorship and promotional roles were also filled. The wet lab group started to discuss the plan for the first wet lab next week as well as looking over the protocols.
WET LAB WEEK 2
Thursday 14/08/14
Project Name: After brainstorming many names, we decided on "The Green Machine" as our title and the slogan "Follow the biobrick road" as our theme to carry throughout our wiki page.
Stocks: Made many stocks, plates and buffers as per methods.
Nanodrop: leant how to use the nanodrop to quantitate all parts
Gene Info: We did another stock-take of parts & checked that we had enough to perform ligations to assemble or planned three Operons. We discussed the strategy for how we were going to make each of the three Operons. The parts we require to assemble our pathway are as follows:
ChlD - 2240bp
ChlI1 - 1202bp
ChlI2 - 1298bp
GUN4 - 782bp
ChlH - 4207bp
CTH1 - 1382bp
YCF54 - 556bp
Plasto - 410bp
ChlM - 959bp
POR - 1154bp
DVR1 - 1193bp
ChlP - 1385bp
ChlG - 11366bp
ChlD BioBrick Correction
As before, previous BioBrick (BB) from 2013 had a 50bp deletion/error within the ChlD (900bp) from using a single restriction digest. Our Aim is to excise the entire ChlD gene using Apa1 and Mlu1 restriction enzymes and to insert a complete ChlD gene into a BB.  We attempted this experiment again but did restriction enzyme digestions with the two enzymes separately to improve efficiency of cutting.
To prevent re-joining after digestion of our cut vector, we treated our samples with Alkaline Phosphatase (Fast A.P.). The DNA was then run on a 1% Agarose gel. 5 bands were identified and using a 1Kb ladder a complete ChlD (900bp) band was found and excised for ligation.
WET LAB WEEK 3
Thursday
21/08/14
More plasmid prep was done, the following 6 genes were inserted into an Ampicillin backbone.  Cells were grown to extract more plasmid for stocks.
Chl1
Chl2
YCF54
ChlP
DVR1
POR
Composite Part Assembly: Trouble-shooting with the BioBrick assembly protocol, we found that if we ligated in a particular way then the plasmid linearises itself and then cannot be cut for the making of composite parts. We then started working on forming test composite parts. Our stocktake was also completed.
Composite parts were assembled of: lac + GUN4 + ChlI1 ; lac + GUN4 + ChlI2 ; lac + ChlI1 + GUN4. Digests were done with EcoRI & SpeI on first gene in part (with lac), and separately with XbaI and PstI for the second gene. These were then ligated into a KAN backbone which was digested with EcoRI & PstI. Digests and ligation steps ran at 37C for 1h and 80C for 20 minutes.
DH5-a electrocompetent E.coli were transformed via electroporation and plated out onto KAN LB-agar.
Transformations: Electroporation does not appear to be working well.  We changed to heat shock to transform our cells. There may be a problem with our electro-competent cells. Made more electro-competent cells to test. Also made chemical competent cells for heat-shock transformation
DRY LAB WEEK3
Thursday
21/08/14
Decided on Outreach ideas. FINALLY. Online reality contest “So You Think You Can Synthesise”. Had discussions of framework for competition, making a trailer.
Met up with MQ Media Team later during the week
Started to put together the sponsorship package
WET LAB WEEK 4
Thursday
28/08/14
Digests
Digests of GUN4+ChlI2 & ChlD to check results from last week
Competent cells
Electroporation does not seem to work and create viable competent cells ergo we shall stick to the heat shock methodology for further preps. More cells were made and used for plasmid preps, BB’s and composite parts.
Composite Parts
4:1 insert – vector ratio for Fast AP ligation steps to produce:
AMP backbone
CTH1 + YCF54
CTH1 + Plasto
ChlP + ChlG
CAM backbone
ChlD
KAN backbone
GUN4 + ChlI2
GUN4 + ChlI1
ChlI1 + GUN4
September 2014
Week 6
Thursday
04/09/14
A busy week!
Composite parts that had growth were digested with X & P and run on 1% Agarose gel to check insert size.
PCR was also performed with BioBrick Forward and BioBrick Reverse primers to see if we could confirm correct assembly of composite parts.
further composite parts were assembled on the AMP backbone
ChlM + YCF54
POR + DVR1
POR + ChlP
Figure Composite part RE digest images from plates that had growth
Friday
05/09/14
Liquid cultures of composite part transformants
Plasmid preps
RE digest of each part – into CAM BB
Competent cell prep: both chemical and electro-competent cells were made
New composite parts made:
/ChlM+YCF54/
/POR+DVR1/
/POR+ChlP/
September 2014
Week 7
WET LAB WEEK 6
Wednesday 10/09/14
Ran PCR products from last week
ChlD
CTH1 + YCF54
CTH1 + Plasto
ChlI2 + GUN4
GUN4 + ChlI2
Plasmid preps done
Thursday 11/09/14<
Sequencing
Plasmids were positive, sent to Macrogen for sequencing:
CTH1 + YCF54
ChlD
Gun4 + ChlI1
CTH1 + Plasto
Composite Checks
These composites were cut as a single digest and a double digest, and run on an agarose gel. Sizes were compared,
POR + ChlP
POR + DVR1
ChlM + YCF54
CTH1 + Plasto
Figure Composite part screenings
Composite Parts
New composite parts were made, and built upon, transformed and plated out:
/CTH1+YCF54/ + Plasto
/CTH1+YCF54/ + ChlM
ChlI2 + ChlI1
ChlI2 + GUN4
Open Day
Saturday 13/09/14
Set up chromatography reactions in preparation for open day on Saturday.
Fluorescent plates drawn, grown, ready to go for Saturday.
Plasmid prep done for previous composite parts. CHlH has finally worked
WET LAB WEEK 7
Monday
15/09/14
Nanodrops of plasmid preps.
single and double digests of every second plasmid (a, c, e, f) run gel. Send for sequencing if successful.
Wednesday
17/09/14
gels re-labelled
transformations
CTH1 + YCFS4 + Plasto <- ChlM
GUN4 + ChlD + CHlI2
<>Thursday  18/09/14
New Composites
POR + ChlP + ChlG
POR + DVR1 + ChlG
POR + DVR1 + ChlP
ChlD
All transformed and plated out.
ChlD Fix
Apa1 & Mlu1 digests with the backbone being treated with Fast AP in Mlu1 digest reaction. Ligations was as usual.
Then transformed and plated out onto CAM plates. Gel digests were run to resolve the 50bp difference
(850-900).
The resolution was seen.
^ Colony screen apaI mluI ChlD 850 and 900 + ChlH
^ pET ApaI MluI Digest Gel
ChlH PCR reaction was also run
WET LAB – MIDSEM BREAK W1
Monday
22/09/14
Liquid Cultures from Thursday plates.
POR + ChlP + ChlG
POR + DVR1 + ChlG
POR + DVR1 + ChlP
ChlD
Restriction Enzyme Digest of CTH1 + YCFS4 + Plasto + ChlM + ChlD composite part.
Gel run of ^ composite part and ChlD to compare with pET to confirm to presence of the 50bp.
Tuesday
23/09/14
EcoRI + X/P digests for plasmids from Mon.
ChlD A + M individual digests, compared against pET with same digest.
Re-screen CTH1 + YCFS4 + Plasto + ChlM colonies and grow in liquid culture
Wednesday 24/09/14
Recheck: ChlH in KAN and CAM both were not okay.
CTH...ChlM: not okay therefore re screen plates from liquid cultures.
Composite part-> POR + DVR1 + ChlP + ChlG
Transform: ChlI2 + GUN4 + ChlD} new composite and plate out.
Rescreen plated samples of CTHI + Plasto & ChlM + YCFS4 all in AMP and put into liquid culture.
Thursday 25/09/14
Liquid culture of ChlI2 + GUN4 + ChlD
prep done in afternoon
POR ...ChlG transformed and plated out
CTHI ...ChlM gel resolved and excited for friday purification.
ChlI2 + GUN4, CTH1 + YCF54 and POR + DVR1 into CAM backbones.
prepare ChlI2 + GUN4, CTH1 + YCF54, ChlD for sequencing
Plasmid prep of CTHI + Plasto, ChlM + YC5S4.
Friday 26/09/14
CTHI ..ChlM gel band purified
Liquid cultures of new composite parts (ChlI2 + GUN4 + ChlD POR..ChlG)
Plasmid prep of CTHI + plasto (A-D) and ChlM + YCF54 (A-D)
Restriction enzyme screen plasmid prep from thursday.
Transformation of GEl extracted plasmids (linear [total of 10 for transformation] + circular + ligation)
*1ul ligase and 4.5 ul ligase buffer.
37oC for 1 hour and 80oC for 20mins.
5ul linear plasmid → 50ul competent cells (chemical)
→ 10ul circular plasmid → 50ul competent cells
Gel Run for ChIl2 + GUN4 + ChlD, ChlH1 + Plasto & ChlM + YCF54 cuts.
LANE ORDER:
1.1….13.2-  ChIl2 + GUN4 + ChlD - ChlH1 + Plasto - ChlM + YCF54
*note: The wells for the last  3 lanes did not accept much of the sample. The sample would float to the surface when being inserted.
The transformants were plated out, the digest resolution wasn’t clear and needs to be re run on monday.  liquid cultures of Sunday transformants 28/9 (CTH1...ChlM gel purified CTH1 + YCF54 (CAM), ChlI2 + GUN4 (CAM).
WET LAB – MIDSEM BREAK W2
Monday 29/09/14
plasmid preps of:
CTH1..ChlM gel purified
CTH1..ChlM re-screen
POR..ChlG
POR + DVR AMP
CTH1 + YCF CAM
ChlI2 + GUN4 CAM
Re Digest and Gel which all results were good:
ChlI2 + GUN4 + ChlD
CTH1 + plasto AMP
ChlM + YCF54 AMP
Gel:
Figure ChlH linearized and gel purified.
Digest + gel: CTH1..ChlM and POR...ChlG
CTH..ChlM plasmid prep kept to H, I, J
POR ..ChlG plasmid prep kept to 1:1, 1:3, 1:6
ChlH upper and lower bands excised & gel purified
Tuesday 30/09/14
Re-transformed and plated out:
CTH..ChlM x 3
POR..ChlG x 3
ChlH x 2
CTH..+ POR.. mixed x 2
Gel run on CAM transformants and re-screen of:
POR + DVR, ChlI2 + GUN4, CTH + YCF
Composite part created, transformed and plated out: ChlI2 + GUN4 + ChlD + ChlI1
ChlM cut into CAM Backbone, transformed and plated out. Tested  for registry
PCR reactions run on final constructs and checked for
F &R from ends:
CTH...ChlM
POR...ChlG
October 2014
Week 9
Wednesday (1/10/14)
PCR:
1ul BB
5ul buffer (x10)
1ul F primer
1ul R primer
1ul dNTP
0.25 ul Taq
0.75 ul H2O (to Evel 50ul)
Master Mix:
50ul buffer
10ul dNTP
2.5ul Taq
407.5ul H2O
------ X ------
C1                  +          C2          → CTH1 + ChlM (H)
(CTF + BBR)       (CMR + BBR)
C3                  +          C4 → CTH1 + ChlM (I)
(CTF + BBR)       (CMR + BBR)
C5                  +          C6          → CTH1 + ChlM (J)
(CTF + BBR)       (CMR + BBR)
PI → POR A        P2→ PORB         P3 → PORC
-----X-----
Setup for functional assays and plasmid preps of ChlM & ChlI1 + ChlI2
Thursday
2/10/14
Cyclase assay (100ul)
MPE - 8ul
1mM NADP - 10ul
10mM G-P-P - 10ul
Assay buff 1 x (50mM tricine, 2mM MgCl2, 1mM DTT 10% glycerol, pH 8.0) - 50ul 2x
0.5ul of G-6-P-dehydrogenase
total volume = 78ul allowing 22ul for other additions.
#1 Blank (water)
#2 22ul CTH1 part 1
#3 11ul CTH1 part 2
#4 11ul CTH
Bradford functional assays were done on induced cell pellets
10% Glycerol + 5mM Tricine NaOH ph8.0 + 2mM MgCl2 + 1mM DTT
Results:
5µl         2.5 dilution
POR - ChlG   4         ~ 1.25        1               3      ~ 1.75         1.5               2        ~ 2        1.5               1     ~ 2         1.5
CTH - ChlM   1        ~ 1        1               2     ~ 1         1
Friday
3/10/14
Protein weight estimates:
ChlM - 30440 Da
CTH1 - 43873.3 Da
YCFS4 - 17073.7 Da
Plasto - 10339 Da
ChlI 1 - 39952 Da
GUN4 - 2450.6 Da
ChlD - 76420.1 Da
POR - 41871 Da
DVR1 - 37034 Da
ChlP - 47011 Da
ChlG - 36880 Da
Protein gels run of two complete composites.
Figure ^CTH1-ChlM composite
Figure ^POR-ChlG composite
The CTH1-ChlM composite showed good separation of products. The POR-ChlG composite had separation but only three parts were able to be easily identified. As Annotated the selected bands were cut-out for in-gel digestion & analysis by MALDI- TOF/TOF.
WET LAB WEEK 8
Wednesday
8/10/14
Plasmid preps were done on the ChlI1 - ChlD composite parts
Nanodrops:
Sample         nucleic acid conc (ng/µl)    260/280
ChlI1 + ChlID         179.6                 1.92
ChlI1 + ChlID         296.3                1.87
ChlI1 + ChlID         261.7                1.93
ChlI1 + ChlID         648                1.90
ChlI1 + ChlID         118.7                2.07
ChlI1 + ChlID         179.1                1.79
ChlI1 + ChlID         553.7                1.93
ChlI1 + ChlID         321.6                1.92
Figure All 8 were then digested and run on a gel, 5-10mL liquid cultures were also prepared and left to incubate overnight.
5-10mL liquid cultures were made of the following parts in an AMP backbone with a lac promoter. ChlI2, YCFS4, ChlP, POR, GUN4, ChlM, ChlG, CTH1, CTH1-ChlM, POR-ChlG for large scale growth (50mL) for functional assays.
Thursday (9/10/14)
New composite ChlI1 + ChlD + GUN4 transformed and plated out. Intermediates and final parts prepped and sent for sequencing to confirm that the inserts are what they’re supposed to be. Gylcerol stocks were made of the current intermediates and final composites. Final parts in an AMP backbone were induced (OD600 = 0.4-5) for functional assays.
WET LAB WEEK 9
Monday
13/10/14
Plasmid prep of ChlI1+ChlD+GUN4, All cultures were screened and prepped for sequencing.
Nanodrops:
Sample         nucleic acid conc (ng/µl)     260/280
ChlI1+ChlD+GUN4     539.2                1.86
ChlI1+ChlD+GUN4    315.4                1.91
ChlI1+ChlD+GUN4    493.7                1.90
ChlI1+ChlD+GUN4    164.0                1.91
ChlI1+ChlD+GUN4    217.5                1.90
ChlI1+ChlD+GUN4    447.0                1.90
ChlI1+ChlD+GUN4    499.2                1.90
All composites re-run on gels for results page, SDS-PAGE gels and MS/MS prep.
POR-ChlG and ChlI1-GUN4 parts were re-transformed for lysate harvesting.
Figure ^ ChlI - ChlM Final gels
Figure ^ POR-ChlG Final gel
Tuesday
14/10/14
POR assay’s were run with GUN4 activing as a negative control. We expected to see a major peak at ~630nm and a
secondary peak at ~670nm, but on both runs (a 20minute and 1.5hours) the second peak was still not visible.
Cyclase assays were also run mirroring experimentation from 2/10/14 on CTH1 and POR, these were run overnight
with an expected peak at ~590nm showing presence of the Mg - protoporphorin intermediate. The Mg is apparent but
no intermediates have been generated.
Wednesday
15/10/14
Liquid cultures were made from plate cultures from the previous day (POR-ChlG, CTH1, ChlI1-ChlD-GUN4, POR). The
liquid cultures from the POR-ChlG & ChlI1+ChlD+GUN4 composites were then induced for growth as 50mL cultures for
French Pressing. The resulting proteins were then run on SDS-PAGE gels and used for functional assays; gels were
destained and bands cut for in-gel digestion and MALDI-TOF/TOF.
Thursday
16/10/14
Figure ^ CTH1 - ChlM Final gel
Figure ^ POR-ChlG Final Gel
Thursday 16/10/14
Assaying CTH1, YCF54 and Plastocyanin
Upon building the second operon including CTH1, YCF54 and Plasto, a functional assay testing the successful or unsuccessful expression of the yielding protoChlorophyllide was made. The assay mix included;
Insert table code Reagents Volume 5µM magnesium-protoporphyrin IX monomethyl ester 8µl 1 µM NADPH 10µl 10 µM glucose 6 phosphate 10µl 1 µg/ml glucose 6 dehydrogenase 0.5µl 50 mM tricine 5 µl 2mM MgCl 5 µl 1mM DTT 5 µl
Extract from Chlamydamonas was added to;
1. Supernatant + membrane (1:1)
2. Supernatant only
3. Membrane
9 CTH1 supernatant samples were assayed, followed by the addition of 5 µl of water and centrifuged and was run on
HPLC for 1.5 hrs.
No significant protoChlorphyllide levels were detected using this assay, this technique however may be used in future assays,
Friday 18/10/14
Assaying the second operon CTH1, YCF54, Plasto including ChlM
Including the final component necessary for the production of protoChlorophyllide in the second operon was the addition of ChlM.
The assay mix was added to the pellet and was as follows;
Reagents
Volume
Resuspension buffer
Made to 44uL
SAM 1mM
1uL
Reagents Volume Resuspension buffer Made to 44 µl SAM 1mM 1 µl MgP 20 µM 2 µl
Figure x shows the MP standard elutes at 9.8 minutes, the precursor to the catalysis to MPE
Figure x shows the MPE standard shows a peak at 10.4 minutes
Figure x depicts the SAM control shows a significant peak at 9.8 minutes showing it does not result in the MPE peak at 10.4 minutes.
Therefore, figure x shows the assay presents a peak at 9.7 minutes. Presence of MP and the absence of MPE indicates the lack of conversion of MP to MPE. This may be due to a low level of expression or low ChlM activity.
Retrieved from " http://2014.igem.org/Team:Macquarie_Australia/WetLab/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
