<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Macquarie_Australia_Project_Model"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Macquarie_Australia/Project/Model">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Macquarie_Australia/Project/Model&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/Project/Model&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/Project/Model&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Macquarie_Australia/Project/Model" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Macquarie Australia/Project/Model</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><SECTION id="Page"><HEADER id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2014.igem.org/Team:Macquarie_Australia" target="_self">Home</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Background">Project</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Significance">Significance</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Parts">Parts &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Model">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Medal">Medal Requirements</A></LI></UL><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/OurTeam/TeamMembers">Our Team</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/OurTeam/TeamMembers">Students</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/OurTeam/Advisors">Instructors &amp; Advisors</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/OurTeam/TheUni">The University</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/OurTeam/Attributions">Attributions &amp; Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/WetLab/Notebook">Wet Lab</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/WetLab/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/WetLab/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/WetLab/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach">Policy &amp; Practice</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/SYTYCS">SYTYCS</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/OpenDay">MQU Open Day</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/Powerhouse">Powerhouse Museum</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/Undergrad">MQ Research Interns</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/BusinessProp">Business Proposal</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/SponsorSupport">Sponsorship Support</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/Law">SynBio &amp; AUS Law</A></LI><LI><A href="https://2014.igem.org/">iGEM</A></LI></HEADER></SECTION></P><DIV class="cont-menu"><H3>Project</H3><UL><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Significance">Significance</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Parts">Parts &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Model">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Medal">Medal Requirements</A></LI></UL></DIV><DIV class="cont-out"><H3>Modelling - Jumping the GUN</H3><H4>Modelling GUN4 binding to protoporphyrin IX in the heme-chlorophyll biosynthetic pathway</H4><P>Modelling biological and biochemical systems <I>in silico</I> is a central part of synthetic biology. The use of models to predict, test, and rationalise experimental observations is key in understanding the complex systems present in cells, even more so when those systems are being deliberately modified. </P><P>The issue when starting the modelling component of a project is often finding something to model, and finding something which will give useful results. When we set out to construct the chlorophyll pathway in our <I>E. coli</I> cells, we had a number of questions. How much chlorophyll are we going to end up making? How is the gene expression going to affect and result in that production? Is the chlorophyll going to kill the <I>E. coli</I>? Is the hydrogen production we’re proposing feasible? Questions from all sides of our project ended up having some component we could model, making the problem deciding what was the most important, central, and above all feasible part to construct mathematically.</P><P><I><B>Figure 1</B> The first part of the Chlorophyll biosynthesis pathway. Our modelling examined the effect of co-factor GUN4 on the incorporation of Mg into Protoporphyrin IX by Mg-Chelastase (Operon 1).</I></P><H4>Why GUN4?</H4><P>Our pathway begins at protoporphyrin IX (PPIX), an intermediate in the synthesis of heme in <I>E. coli</I>. Here, we’ve constructed a branch point, and are siphoning off this molecule from this pathway onto the chlorophyll production line (Figure 1). The amount of PPIX which we redirect onto the chlorophyll pathway dictates almost everything about our project, including our end-product formation, whether the gene product enzymes will function and to what extent, and how much gene expression to promote in order to have a functioning biosynthesis pathway. This redirection is primarily dependent on the binding of the GUN4 protein to PPIX, which makes it inaccessible to the ferrochelatase enzyme which begins the heme pathway, and allows the magnesium chelatase to insert the magnesium ion yielding magnesium protoporphyrin IX (Mg-PPIX). Since this is the “point of no return”, any amounts of PPIX which are metalated by magnesium chelatase are committed to the synthesis of chlorophyll, meaning all further questions about the project depend on this one branch step. Due to this importance, it is the system around this point which we have modelled.</P><H4>Constructing the Equations</H4><P>The first step in modelling our system was to find out more about its biochemistry, and let that knowledge dictate the mathematics. In the case of our system, there are two main catalytic equilibria involved, namely the ferrochelatase-catalysed production of heme, and the magnesium chelatase-catalysed production of Mg-PPIX. Linking these two pathways is the binding equilibrium of GUN4 with PPIX, with the detachment of GUN4 from Mg-PPIX the final step to be modelled. This product would then go on to react with the next enzyme of our pathway and continue to make chlorophyll, and is thus a quantity of interest.</P><P><I><B>Figure 2 </B> Equations used (<I>top</I>) to model incorporation of Mg into Protoporphyrin by GUN4 (co-factor) (<I>bottom</I>). Abbreviations: Protoporphyrin (PPIX); Magnesium Protoporphyrin IX (PPIX); Ferrochelatase (Ferro); Magnesium Chelatase (MgChel).</I></P><P>The reaction scheme in Figure 2 shows the main equations of the pathway which were included in the model. In addition to this, there is a feedback loop from protoheme to the production of PPIX; when the concentration of protoheme gets above a certain value, the synthesis of PPIX is turned off, while when the value drops below another value, the synthesis is started again. </P><P>Due to the insertion and expression of the GUN4 gene, there will also be some level of production of GUN4. However, this is likely to cease above a certain concentration due to transcription of the Operon 1. These two factors are then taken into account in the ordinary differential equations (ODEs) which are presented in Figure 2, with the code shown in Table 1.</P><P>This coupled set of 10 differential equations was then put into MATLAB, with constants and initial conditions as given and justified in Table 2.
We are primarily concerned with the production of MgPPIX, as well as the kinetics of the system as a whole. Figure 3 presents all ten molecules and complexes involved and their cellular concentrations over a five hour time scale. There are several interesting long-term behaviours present in the plot. First, the product which is being produced the most is in fact the complexed MgPPIX-GUN4, rather than the detached MgPPIX product. In the full reaction scheme, the MgPPIX would be further modified by other enzymes and thus would cause the MgPPIX-GUN4 equilibrium to move towards the right, leading to a decrease in the accumulation of this product. Initially, the PPIX concentration is highly dependent on the feedback mechanism by the protoheme regulation; however, after approximately 90 minutes, this concentration settles to an equilibrium which is more than the initial concentration. This is expected behaviour, as two systems are now relying on this pool of PPIX substrate, leading the cell to produce a higher standing concentration to compensate. Over long periods, the production of MgPPIX-GUN4, PPIX-GUN4, and GUN4 becomes linear, due to the constant production of GUN4 in our <I>in silico</I> cell. The feedback modelling of protoheme seems to function as desired, with a very fast feedback loop creating a very constant protoheme concentration in the cell. </P><P><I><B>Figure 3 </B>Modelling of concentration of all 10 molecules/proteins/protein complexes produced in chlorophyll biosynthesis pathway over a 5 hour time period.</I></P><P>Figure 4 shows in more detail the concentrations of the initial PPIX substrate, as well as our final desired product of the system, the MgPPIX. It is important to note that the fluctuating behaviour around 150-200 minutes is likely due to the time sampling of MATLAB rather than a behaviour of the system. This shows that the PPIX concentration indeed settles to an equilibrium, while the MgPPIX concentration follows quasi-hyperbolic kinetics.</P><P><I><B>Figure 4 </B>Modelling of initial concentration of PPIX substrate and final product (MgPPIX). Note: fluctuation likely due to sampling by MATLAB and not representative of biology of system.</I></P><H4>Linking to Experiment</H4><P>Initially, there were high hopes to measure the concetration of MgPPIX as a tool to verify the model. However, as the project neared its end, the best point of comparison to functional assays was to measure the impact of GUN4 only. This meant removing the MgChel component of the pathway, and looking at the measurable comparison between PPIX and the complexed PPIX-GUN4. The results of the model over an hour is shown in Figure 5. This illustrates that the assumption of constant production rate of GUN4 is likely invalid, due to a lack of equilibrium concentration of PPIX and thus an unlikely result for the system logically. From functional assays, the difference in PPIX concentration before and after the addition of GUN4 could be measured, and the model adjusted accordingly.</P><P><I><B>Figure 5</B> Modelling the effect of GUN4 on first step of chlorophyll biosynthesis pathway. Modelling measured the concentration of Protoporphyrin IX (PPIX) produced be removing Mg Chelatase from pathway. Modelling assumed a constant production rate of GUN4.</I></P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>Reagent</TH><TH>Code</TH></TR><TR><TD>Protoporphyrin IX (PPIX)</TD><TD>S<SUB>1</SUB></TD></TR><TR><TD>Ferrochelatase (Ferro)</TD><TD>E<SUB>1</SUB></TD></TR><TR><TD>PPIX-Ferro</TD><TD>E<SUB>1</SUB>S<SUB>1</SUB></TD></TR><TR><TD>Protoheme</TD><TD>P<SUB>1</SUB></TD></TR><TR><TD>GUN4</TD><TD>G</TD></TR><TR><TD>PPIX-GUN4</TD><TD>S<SUB>2</SUB></TD></TR><TR><TD>Magnesium chelatase (MgChel)</TD><TD>E<SUB>2</SUB></TD></TR><TR><TD>PPIX-GUN4-MgChel</TD><TD>E<SUB>2</SUB>S<SUB>2</SUB></TD></TR><TR><TD>Magnesium protoporphyrin IX (MgPPIX)-GUN4</TD><TD>P<SUB>2</SUB></TD></TR><TR><TD>MgPPIX</TD><TD>P<SUB>f</SUB></TD></TR></TBODY></TABLE><P><B>Table 1.</B> Acronyms of the components of the model.</P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>Parameter Symbol</TH><TH>Meaning</TH><TH>Value</TH><TH>Reference / Comments</TH></TR><TR><TD>K<SUB>D1</SUB></TD><TD>Dissociation constant of ferrochelatase with PPIX</TD><TD>110 nM/min </TD><TD>Ref. 1</TD></TR><TR><TD>K<SUB>D2</SUB></TD><TD>Dissociation constant of GUN4 and PPIX</TD><TD>11 nM/min</TD><TD>Ref. 2</TD></TR><TR><TD>K<SUB>D3</SUB></TD><TD>Dissociation constant of magnesium chelatase and PPIX-GUN4</TD><TD>10 nM/min</TD><TD>Thorough lit search yielded no value; estimated from Willows, R. 2014 (unpublished).</TD></TR><TR><TD>K<SUB>D4</SUB></TD><TD>Dissociation constant of MgPPIX and GUN4</TD><TD>1.6 nM/min</TD><TD>Ref. 2</TD></TR><TR><TD>K<SUB>cat,1</SUB></TD><TD>Turnover number for ferrochelatase for PPIX</TD><TD>0.78 /min</TD><TD>Ref. 1</TD></TR><TR><TD>K<SUB>cat,2</SUB></TD><TD>Turnover number for magnesium chelatase for PPIX-GUN4</TD><TD>0.8 /min</TD><TD>Ref. 3 &amp; 4</TD></TR><TR><TD>K<SUB>f</SUB></TD><TD>Rate of formation of PPIX due to protoheme regulation</TD><TD>20 nM/min</TD><TD>Thorough lit search yielded no value; estimated from Willows, R. 2014 (unpublished).</TD></TR><TR><TD>χ<SUB>f</SUB></TD><TD>Switch triggering PPIX production when protoheme concentration drops below 50 nM</TD><TD>1 (when [Protoheme] &lt; 50 nM), 0 (when [Protoheme] &gt; 50 nM)</TD><TD>Thorough lit search yielded no switch value; estimated from Willows, R. 2014 (unpublished).</TD></TR><TR><TD>β<SUB>deg</SUB></TD><TD>Degradation rate/rate at which protoheme is removed by conversion to heme</TD><TD>1 nM/min</TD><TD>Thorough lit search yielded no value; since we haven’t included the formation of heme in our system, this compensates for this process; estimated from Willows, R. 2014 (unpublished).</TD></TR><TR><TD>α<SUB>G</SUB></TD><TD>Formation rate of GUN4 from gene</TD><TD>1 nM/min</TD><TD>No value available due to lack of characterisation in E. coli; estimated from Willows, R. 2014 (unpublished).</TD></TR></TBODY></TABLE><P><B>Table 2.</B> Constant values of the model components based on reported literature.</P><H3>References</H3><P><UL style="list-style-type: decimal;"><LI>Storm, P., Tibiletti, T., Hall, M., Funk, C. Refolding and enzyme kinetic studies on the ferrochelatase of the cyanobacterium Synechocystis sp. PCC 6803. PloS one. 2013. 8.</LI><LI>Adhikari, N.D., Orler, R., Chory, J., Froehlich, J.E., Larkin, R.M. Porphyrins promote the association of GENOMES UNCOUPLED 4 and a Mg-chelatase subunit with chloroplast membranes. JBC. 2009. 284: 24783-96.</LI><LI>Reid, J.D., Hunter, C.N. Magnesium-dependent ATPase activity and cooperativity of magnesium chelatase from Synechocystis sp. PCC6803. JBC. 2004. 279: 26893-9.</LI><LI>Adams, N.B., Marklew, C.J., Brindley, A.A., Hunter, C.N., Reid, J.D. Characterization of the magnesium chelatase from Thermosynechococcus elongatus. Biochemical Journal. 2014. 457: 163-70. </LI></UL></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Macquarie_Australia/Project/Model">http://2014.igem.org/Team:Macquarie_Australia/Project/Model</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Macquarie_Australia/Project/Model" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Macquarie_Australia/Project/Model" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Macquarie_Australia/Project/Model&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Macquarie_Australia/Project/Model&amp;oldid=393516" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>