<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Macquarie_Australia_WetLab_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Macquarie_Australia/WetLab/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Macquarie_Australia/WetLab/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/WetLab/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/WetLab/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Macquarie_Australia/WetLab/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Macquarie Australia/WetLab/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><SECTION id="Page"><HEADER id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2014.igem.org/Team:Macquarie_Australia" target="_self">Home</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Background">Project</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Significance">Significance</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Parts">Parts &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Model">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Medal">Medal Requirements</A></LI></UL><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/OurTeam/TeamMembers">Our Team</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/OurTeam/TeamMembers">Students</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/OurTeam/Advisors">Instructors &amp; Advisors</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/OurTeam/TheUni">The University</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/OurTeam/Attributions">Attributions &amp; Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/WetLab/Notebook">Wet Lab</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/WetLab/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/WetLab/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/WetLab/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach">Policy &amp; Practice</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/SYTYCS">SYTYCS</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/OpenDay">MQU Open Day</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/Powerhouse">Powerhouse Museum</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/Undergrad">MQ Research Interns</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/BusinessProp">Business Proposal</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/SponsorSupport">Sponsorship Support</A></LI><LI><A href="https://2014.igem.org/Team:Macquarie_Australia/Outreach/Law">SynBio &amp; AUS Law</A></LI><LI><A href="https://2014.igem.org/">iGEM</A></LI></HEADER></SECTION></P><DIV id="accordion"><H3>The Notebook</H3><DIV><P>Welcome to our Notebook! This page has been created to detail our projects progress since its conception. Click on the section titles below to expand pages that contain information regarding what we worked on and achieved each week. For a more detailed summary of what we achieved please visit the <A href="https://2014.igem.org/Team:Macquarie_Australia/Project/Results">Results</A> page.</P><P>  The full Team_Macquarie_2014 Lab Notebook may be found here: </P><DIV id="pdfList"><UL><LI><A href="https://static.igem.org/mediawiki/2014/0/05/Team_Macquarie_2014_notebook_2014.pdf">Team_Macquarie_2014 Lab Notebook </A></LI></UL></DIV></DIV><H3>November 2013</H3><DIV><H4>Week 1</H4><P>Biobrick stocktake of 2013 iGEM Macquarie_Australia parts: 11/11/13</P><UL><LI>ChlG - sufficient plasmid stock</LI><LI>DVR1 - sufficient plasmid stock</LI><LI>ChlM - sufficient plasmid stock</LI><LI>ChlI2 - need more plasmid stock</LI><LI>POR- need more plasmid stock</LI><LI>YCF - need more plasmid stock</LI><LI>Plasto - need more plasmid stock</LI><LI>GUN4- need more plasmid stock</LI><LI>CTH1 - need more plasmid stock</LI><LI>ChlD - need more plasmid stock. Question whether the 2013 part is really the reported sequence - something appears to be missing.</LI></UL><P>Send all for re-sequencing to verify DNA sequence as per registry entries.DVR1 re-tranformation: Was re-done using gibson assembly and then transformed.</P><H4> Week 2</H4><U> Sequencing Results: </U> all parts except ChlD were correct.
<P>ChlD Fix:  ChlD is missing 50 bp. Strategy to correct is to use ApaI and MluI restriction enzymes to cut out 50bp from clone of ChlD in pET vector from Willows group and re-insert into our BioBrick vector. </P><P> ApaI and MluI were used in a single digest according to manufacturer's instructions and as per ligation protocol on methods wiki. Fragments run on 1% agarose and gel purified. However, digestions were incomplete as viewed on agarose gel. Need to do separate digests for next attempt.</P><P> Double restriction enzyme digest was carried out to combine PCR1 and Gblock2. After the two sections were ligated and extended, straight PCR was done. The PCR worked as judged by agarose gel. </P><H4> Week4 </H4><B> Tuesday: 26/11/13</B><U> Composite parts Assembly</U><P> Biobrick (BB) ChlI1 is combined with ChlI2 biobrick in AMP backbone. Method is via 3A assembly. Use 500ng of each part and insert into 500ng of amp backbone. Ligation for 16oC for 30 mins then 80oC for 20 mins. Leave plates over weekend at room temperature. </P><P> Growth on plates : 1 colony on low plate, hundreds on high plate.</P><U> Assembly of ChlH: PCR of individual fragments from ChlH: 29/11/13</U><UL><LI>G1 - G1F + G1R</LI><LI>G2 - G2F+ G2R </LI><LI>G3 - G3F + G4R</LI><LI>G4 - G4F+ G4R</LI><LI>G5 - G5F + G5R</LI><LI>G6 - G6F + G6R </LI><LI>PCR1+ G2- H1F+ G2R</LI><LI>(PCR1 + G2) + G1</LI><LI>G3 + PCR2- G3F+ H2R </LI><LI>G5 + G6- G5F +G6R </LI></UL><P><U> Increase stocks </U>: Did plasmid preps to get more of: ChlI1; ChlI2; YCF54; ChlP, DVR1; POR </P><U> ChlH Biobrick correction</U><P>Attempt to make ChlH (BBa_K1080001) using combination of gblocks and PCR products, as designed by Macquarie_Australia 2013 iGEM team.</P><P>Assembly strategy is: G-Block –1 (470bp) + PCR-1 (304bp) + G-Block-2 (499bp) + G-Block-3 (499bp) + PCR-2 (984bp) + G-Block-4 (500bp) + G-Block-5 (481bp) + PCR-3 (673bp)</P><P>Extremely faint bands are seen for </P><B> Friday: 29/11/13</B><P>Another attempt to assemble chlD from PCR fragments</P><UL><LI> G Block 1 </LI><LI> G Block 4  </LI><LI> G4 + (G5-G6) </LI><LI> G1 + PCR1 </LI><LI> G2 + (G3 + PCR2) </LI></UL><B>Figure 1</B><I>
Extremely faint bands seen for ChlD amplification. G1 and G4 appear to work but bands are very faint on agarose gel. Faint to no bands viewed for G2 and G3+PCR2. Reattempt necessary.
</I></DIV><H3>December 2013</H3><DIV><H4>Week 1</H4><B> Friday: 06/12/13</B><U>Digestion &amp; Ligation of ChlM gene of lac promoter into CAM backbone</U><P>ChlM in AMP backbone vector and lac in backbone were digested using iGEM restriction digestion protocol EcoRI and Pst1 restriction enzymes.</P><P><B>Tuesday </B></P><P><B>10/12/13</B></P><P>The fragments to be PCR’d and the primers are presented on the following table;</P><TABLE border="1" cellpadding="7" cellspacing="0"><TBODY><TR valign="TOP"><TD><P><B>Fragments to PCR</B></P></TD><TD><P><B>Primers</B></P></TD></TR><TR valign="TOP"><TD><P>G1</P></TD><TD><P>BBF+G1R</P></TD></TR><TR valign="TOP"><TD><P>G1+P1</P></TD><TD><P>BBF+H1R</P></TD></TR><TR valign="TOP"><TD><P>G2+(G3-P2)</P></TD><TD><P>G2F+P2R</P></TD></TR><TR valign="TOP"><TD><P>ChlI1</P></TD><TD><P>BBVF2+BBVR</P></TD></TR><TR valign="TOP"><TD><P>ChlI2</P></TD><TD><P>BBVF2+BBVR</P></TD></TR><TR valign="TOP"><TD><P>ChlD</P></TD><TD><P>BBVF2+BBVR</P></TD></TR></TBODY></TABLE><P>Small amount of growth seen on ChlM, lacA &amp; lacB indicating that they were successfully incorporated into the DHS←  cells. Sequencing to confirm required</P><P>Transformation of Kanamycin Resistant backbone</P><P>
We need more of the kanamycin biobrick. Transformation of kanamycin backbone into E. coli cells to produce large amounts of KAN backbone for future ligations. 
</P><H4>Monday</H4><P><B>09/12/13</B></P><P>PCR reaction for ChlH and ChlD</P><P>The overall of the aim of the week was to build ChlH fragment and PCR ChlD. Using the standard PCR protocol, G1+H1, G2 (G3+H2), G4 (G5+G6), ChlD (2) and ChlD (3) were run. </P><P>The result showed another G1+PCR1 failure. It was also suggested however to use BioBrick primers. Distinct bands for G2+(G3/PCR2) and G4+(G5/G6) were present and proved correct. This assumption was made that these results were correct.</P><P>ChlD 2 and 3 showed a band present at approximately 1500 bp which was also assumed to be correct in relation to the actual size of 1681 bp.</P><P><I><B>Figure2:</B> The next step was to rePCR G1+PCR1 with BBF + HR2 and BBvF + HR2, gel extraction of G2+(G3/H2) and G4+(G5/G6) ,ChlD 2 and 3. </I></P><P><B>Tuesday </B></P><P><B>10/12/13</B></P><P>The fragments to be PCR’d and the primers are presented on the following table;</P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>Fragments to PCR</TH><TH>Primers</TH></TR><TR><TD>G1</TD><TD>BBF+G1R</TD></TR><TR><TD>G1+P1</TD><TD>BBF+H1R</TD></TR><TR><TD>G2+(G3-P2)</TD><TD>G2F+P2R</TD></TR><TR><TD>ChlI1</TD><TD>BBVF2+BBVR</TD></TR><TR><TD>ChlI2</TD><TD>BBVF2+BBVR</TD></TR><TR><TD>ChlD</TD><TD>BBVF2+BBVR</TD></TR></TBODY></TABLE><P><I> The standard PCR Method was adopted to run the reaction </I></P><P><B> Figure 3: </B><I> All but ChlI1 and ChlI2 failed </I></P><P><B>Continued ChlH construction</B></P><P>At this stage, the ChlH gene construct was continued;</P><P> G1-P1-G2-G3-P2-G4-G5-G6</P><P>The ChlD gene was cut from the gel and extracted with another attempt to PCR.</P><P>To test for protein expression, the successful 3A gene was combined with lac creating a composite.</P><P>Continuing the construct of ChlH, a PCR reaction was performed to identify the successful or unsuccessful attempt in the composite build in addition to DVR1 identification.</P><P><B> Table 2: </B><I> The PCR reaction screening attempting to construct chlH failed </I></P><P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>Gene fragment</TH><TH>Primers</TH></TR><TR><TD>P1+G2</TD><TD>H1F+G2R</TD></TR><TR><TD>(G3+P2)+(G4-G5-G6)</TD><TD>GBF+G6R</TD></TR><TR><TD>DVR1</TD><TD>BBVF2+BBVR</TD></TR></TBODY></TABLE></P><P><B> Figure 4 </B><I> chlH and DVR1 Gibson assembly gel </I></P><B><P>Digest of DVR1</P></B><P>The next step was the insertion of DVR1 into the plasmid vector. The plasmid vector and the plasmid containing the gene of interest were ligated with EcoR1 and Pst1. The gene was introduced into the vector my means of 1 vector to 3 insert to maximise insertion efficiency.</P><P><B>ChlH construction by Gibson assembly</B></P><P>
The failure of the construction of the ChlH gene subjected the attempt in the construction of the gene using Gibson assembly. The provided gel image proved the construction also failed. </P><P><B> Figure 5 </B><I> PCR of ChlH Gibson Assembly </I></P></DIV><H3>January 2014</H3><DIV><P><B>Tuesday </B></P><P><U>ChlH construct PCR</U></P><P>In the attempt to yield a positive result in the construction of ChlH, each P1+G2, (G3+P2) and (G4-G5-G6) were PCR’d separately in the attempt to successfully join the individual components.</P><P><B>10/12/13</B></P><P><B> Table 3 </B><I> PCR of ChlH fragments </I></P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>Gene fragment</TH><TH>Primer</TH></TR><TR><TD>P1-G2</TD><TD>P1F + G2R</TD></TR><TR><TD>G3-P2</TD><TD>G3F + P2R</TD></TR><TR><TD>G4-G5-G6</TD><TD>G4F + G6R</TD></TR><TR><TD>(P1-G2) + (G3-P2)</TD><TD>P1F + P2R</TD></TR><TR><TD>(G3-P2) + (G4-G5-G6)</TD><TD>G3F + G6R</TD></TR><TR><TD>ChlD</TD><TD>ChlD F + ChlD R</TD></TR></TBODY></TABLE><P><B>Thursday</B>:
<B>09/01/14</B></P><P>Gel analysis of PCR gel of ChlH constructs and ChlD</P><P>
The results obtained would indicate the band to extract for Gibson assembly. The gel image showed positive results 
</P><P>
The marked were cut out and stored for gel extraction.

<B> Figure 6 </B><I> PCR gel analysis of ChlH constructs and ChlD. To compare the sizes, 25-500ng of plasmid were digested with and without lac. The expected size was approximately 200 bp.
The amplification of ChlD was faint indicating an issue with the construction of the gene.</I></P><P><B> Friday </B></P><P><B>10/01/14</B></P><P>ChlH screening</P><P>The bands on gel corresponding to ChlH were extracted to screen for the correct sizes. The result of the gel extraction showed low concentration indicating poor construction of gene.</P><P><B> Figure 7 </B><I> ChlD/ ChlI Plastocyanin screening: Digests were performed with enzymes EcoR1 and Pst1 to comment on the sizes of the inserts including ChlD, ChlI1 and Plastocyanin. These were also run against the corresponding components including lac.</I></P><P><B> Saturday</B></P><P><B>11/01/14</B></P><P><B> Table 4: </B><I> Continued construction for ChlH PCR </I></P><P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>Gene fragment</TH><TH>Primers</TH></TR><TR><TD>P1-G2</TD><TD>F1 + G2R</TD></TR><TR><TD>G3-P2</TD><TD>G3F + P2R</TD></TR><TR><TD>G4-G5-G6</TD><TD>G4F + G6R</TD></TR><TR><TD>CHlD</TD><TD>NF2 + NR2</TD></TR></TBODY></TABLE></P><P><B> Figure 8 </B><I> The results obtained from the gel yielded a successful result for P1-G2, responsible for the construction of CHlH and negative results for the remaining samples on the gel.</I></P><P><B> Thursday</B></P><P><B>30/01/14</B></P><P><U>PCR:</U> It is thought that the excess template in the previous PCR may have been responsible for the failure of PCR amplification. Template dilutions of 1/10 and 1/100 were tested by running another pcr. </P><P>The PCRs carried out were: </P><UL><LI>ChlD (new template), diluted </LI><LI> G3 -P2 PCR template</LI><LI>ChlH gel run + extracted template </LI><P><B> Figure 9 </B><I> 
ChlD and ChlH G3-P2 PCR template did not work, however, ChlH G3 -P2 PCR template was successful. </I></P><P><B> PCR for ChlD blocks: </B></P><P>G4 + (G5-G6) X3 = G4F + G6R  </P><P>P1- G2 (from the original templates) x3 = P1F +G2F.   </P><P><B> Figure 10 </B><I> 
G4 + (G5-G6) and P1- G2 PCRs worked. G4 + (G5-G6) showing a band of 1700 bp in length and P1-G2 showing 800 bp in length.  </I></P><P><B> PCR continuation: </B></P><P>(P1-G2) + (G3-P2)</P><P>G3-P2 + G4-G5-G6</P><P><B> Figure 11 </B><I> 
None of the PCRs from the ChlD blocks worked - clear, desired bands were not found  </I></P><H3>Feburary 2014</H3><DIV><P><B>Week 1</B></P><P><B>Monday</B></P><P><B>3/2/2014</B></P><P><B>Digestion of DVR1 was run.</B></P><P>Following the digest, a ligase reaction was conducted and transformation performed. Note, the concentration of DVR part in comparison to the concentration of the plasmid was 1.5 times more. Plates incubated overnight.</P><P><B>Protein expression of lac+plasto &amp; lac+ChlI</B></P><P>Expression of protein via lac promotor using 2uL of IPTG was done for each sample to amplify protein expression.SDS-PAGE was run according to methods. </P><P><B>Tuesday</B></P><P><B>4/2/14</B></P><P><B>SDS PAGE attempt #2</B></P><P>Here we conducted a second SDS PAGE for lac plasto and lac ChI1. </P><P>Lane order: 1-4 lac plasto, 5 is the ladder, 6-10 lac ChlI1</P><P>Expression of proteins was not visible by eye. No image of the gel was recorded. We think we need to do mass spec (MALDI/TOF/TOF) to identify proteins in bands. Discuss with APAF (Australian Proteomic Analysis Facility) at Macquarie University to ask if they can help us with performing mass spec. </P><P><B>Testing new ligase: New ligase purchased as concerns were that our ligase was old and the reason ligations were not successful</B></P><P>ChlH was digested with E+P restriction enzymes as per methods. To ligate, the ligation mixture comprised of 8.5uL DNA, 0.5uL ligase and 1uL of buffer. </P><P><B> Figure 12 </B><I> 
 The gel showed inconclusive results and requires further clarification. Further testing methods such as re-inserting the biobrick into another vector and growing it on a plate with the second vector antibiotic have been suggested.   </I></P><P>PCR:</P><P>Fragment 1 - (P1-G2) + (G3-P2) = P1F, P2R</P><P>Fragment 2- (G3-P2) + (G4-G5-G6) = G3F, G6R</P><P>Fragment 3- (P1-G2) + (G3-P2) + (G4-G5-G6) = P1F, G6R. </P><P>For such large fragments, the preliminary melting step was completed twice prior to the addition of the primers because of the long fragments. The rest of the process was continued on the regular loop as in other PCR protocol. </P><P><B>Wednesday</B></P><P><B>5/2/14</B></P><P><B>Western Blot: </B></P><P>Western blot for ChlI1 and plasto were carried out. </P><P>Results: The plasto lanes did not show any expression, however ChlI showed good expression in lanes 2,4 and 5. </P><P><B> Figure 13 </B><I> 
 The gel showed inconclusive results and requires further clarification. Further testing methods such as re-inserting the biobrick into another vector and growing it on a plate with the second vector antibiotic have been suggested. </I></P><P><B> Thursday 6/2/14: Gibson Assembly of ChlH: </B></P><P>G1: 3uL </P><P>P1-G2 exosap: 0.5uL </P><P>G3-P2 exosap: 4.8uL </P><P>G4-G5-G6 exosap: 1.0uL </P><P>Cam vector: 29uL with Gibson mix: 12.3uL or AMP vector 1.4uL with Gibson mix: 10.8 uL</P><P>- Plated out </P><P>PCR of Gibson Assembly product for ChlH: Standed PCR x2 using BBF/ BBR/ BBVF2, BBVR. </P><P> Results:</P><P><B>Week 2</B></P><P><B>Wednesday</B></P><P>12/2/14 </P><P>Nanodrop of ChlH fragments </P><P>ChlH Fragments: </P><P>P1-G2: </P><UL><LI><P>A= 19.3 ng/ml</P></LI><LI><P>B= 23.4 ng/m</P></LI><LI><P>C= 18.2 ng/m</P></LI></UL><P>G3-P2: </P><UL><LI><P>A = 27.8 A= 141ng/ml</P></LI><LI><P>B= 9.9 ng/ml</P></LI><LI><P>C= 47.5 ng/ml</P></LI></UL><P>G4-G5-G6: </P><UL><LI><P>A= 141ng/ml</P></LI><LI><P>B= 24.6 ng/ml</P></LI><LI><P>C= 62.5 ng/ml</P></LI></UL><P>ChlD Fragments: D1, D2, D3 </P><P><B>Thursday</B></P><P><B>13/2/14</B></P><P>Gel Electrophoresis for ChlH and ChlD fragments: </P><P>Top Gel Lane order: 1- Ladder, 2- G1, 3-5 - P1-G2, 6-8- G3-P2, 9-11- G4-G5-G6, 12-14- ChlD</P><P>Bottom Gel Lane Order: 1+ 2- Ladder, 3-5- P1-G2, 6-8- G3-P2, 9-11 G4-G5-G6</P><P>Results: ChlH fragments appear not have been digested. P1-G2 and G4-G5-G6 didn’t have plasmids on the gel so they did not digest. ChlD has digested with MLU and partially APAI. ChlD plasmids from lanes 12 and 13 were added together for further re-digestion. </P><P>New Digestion using E+P from previous gel electrophoresis for ChlH: </P><P>Lane Order: 1- G1, 2- G3-P2 A, 3- G3-P2 B, 4- G3-P2 C, 5- G4-G5-G6 A, 6- G4-G5-G6 C, 7- Plasto Control </P><P>New Digest for ChlD with APAI: The ChlD being redigested is a combination of lanes 12 and 13 from the previous electrophoresis gel. </P><P>ChlD Ligation : ChlD was ligased and transformed into E.coli. 2 colonies grew on the 300uL plate and 1 colony on the 30uL plate. </P><P><B>Friday 14/2/14</B></P><P><B>Plasmid nanodrop</B></P><P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>Lac</TH><TH>Gun</TH><TH>A</TH><TH>33.8 ng/µl</TH></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD>17.8 ng/µl</TD></TR><TR><TD>Lac</TD><TD>ChlM</TD><TD>A</TD><TD>33.5 ng/µl</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD>26.8 ng/µl</TD></TR><TR><TD>Lac</TD><TD>ChlP</TD><TD>A</TD><TD>32.2 ng/µl</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD>14.4 ng/µl</TD></TR><TR><TD>Lac</TD><TD>ChlI2</TD><TD>A</TD><TD>18.5 ng/µl</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD>15.2 ng/µl</TD></TR><TR><TD>Lac</TD><TD>POR</TD><TD>A</TD><TD>35.1 ng/µl</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD>24.5 ng/µl</TD></TR><TR><TD>Lac</TD><TD>ChlG</TD><TD>A</TD><TD>8.3 ng/µl</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD>13.2 ng/µl</TD></TR><TR><TD>Lac</TD><TD>YCF54</TD><TD>A</TD><TD>29.3 ng/µl</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD>33.9 ng/µl</TD></TR><TR><TD>Lac</TD><TD>CTH1</TD><TD>A</TD><TD>59.1 ng/µl</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD>51.6 ng/µl</TD></TR></TBODY></TABLE></P><P><B> Table 6 ; </B>: Biobrick concentrations </P><P><B>ChlH re-digest:  </B></P><P><B>Digests checking biobricks: </B></P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>With/without lac</TH><TH>Biobrick</TH><TH>Fragment </TH><TH>Expected weight</TH><TH>Measured </TH></TR><TR><TD>Lac</TD><TD>Gun</TD><TD>A</TD><TD>930</TD><TD>~930</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD> </TD><TD>~930</TD></TR><TR><TD>Lac</TD><TD>ChlM</TD><TD>A</TD><TD>873</TD><TD>~1050</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD> </TD><TD>~1050</TD></TR><TR><TD>Lac</TD><TD>ChlP</TD><TD>A</TD><TD>1299</TD><TD>~1500</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD> </TD><TD>~1500</TD></TR><TR><TD>Lac</TD><TD>ChlI2</TD><TD>A</TD><TD>1212</TD><TD>~1400</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD> </TD><TD>~1250</TD></TR><TR><TD>Lac</TD><TD>POR</TD><TD>A</TD><TD>1067</TD><TD>~1250</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD> </TD><TD>~1250</TD></TR><TR><TD>Lac</TD><TD>ChlG</TD><TD>A</TD><TD>1050</TD><TD>~1250</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD> </TD><TD>~1250</TD></TR><TR><TD>Lac</TD><TD>YCF54</TD><TD>A</TD><TD>471</TD><TD>~650</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD> </TD><TD>~650</TD></TR><TR><TD>Lac</TD><TD>CTH1</TD><TD>A</TD><TD>1152</TD><TD>~1350</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD> </TD><TD>~1350</TD></TR><TR><TD> </TD><TD>DVR1</TD><TD>A</TD><TD> </TD><TD>~1106</TD></TR><TR><TD> </TD><TD> </TD><TD>B</TD><TD> </TD><TD>~1106</TD></TR><TR><TD> </TD><TD> </TD><TD>C</TD><TD> </TD><TD>~1106</TD></TR><TR><TD> </TD><TD> </TD><TD>D</TD><TD> </TD><TD>~1106</TD></TR></TBODY></TABLE><P><U>Results:</U> Majority look as though they match the expected band length. All digests excluding DVR1 include lac. </P></DIV><H3>Winter Lab Session</H3><DIV><H4>AUGUST (SEMESTER 2)</H4><P><B>WEEK 1</B></P><P><B>Thursday</B></P><P><B>07/08/14 </B></P><P>Off and running with the whole team of 12 Biomolecular Major students. We sat through a full day of learning about iGEM; we had a discussion of project goals and aims; as well as a refresher course on how the Chlorophyll pathway works. Roles were assumed by our wiki-chiefs and those interested in gaining sponsorship and promotional roles were also filled. The wet lab group started to discuss the plan for the first wet lab next week as well as looking over the protocols.</P><H3>WET LAB WEEK 2 </H3><DIV><DIV>Thursday 14/08/14<DIV><P><U>Project Name:</U> After brainstorming many names, we decided on &quot;The Green Machine&quot; as our title and the slogan &quot;Follow the biobrick road&quot; as our theme to carry throughout our wiki page.</P><P><U>Stocks:</U> Made many stocks, plates and buffers as per methods.</P><P><U>Nanodrop:</U> leant how to use the nanodrop to quantitate all parts</P><P><U>Gene Info: </U>We did another stock-take of parts &amp; checked that we had enough to perform ligations to assemble or planned three Operons. We discussed the strategy for how we were going to make each of the three Operons. The parts we require to assemble our pathway are as follows:</P><P>ChlD - 2240bp</P><P>ChlI1 - 1202bp</P><P>ChlI2 - 1298bp </P><P>GUN4 - 782bp </P><P>ChlH - 4207bp</P><P>CTH1 - 1382bp</P><P>YCF54 - 556bp </P><P>Plasto - 410bp </P><P>ChlM - 959bp </P><P>POR - 1154bp </P><P>DVR1 - 1193bp</P><P>ChlP - 1385bp </P><P>ChlG - 11366bp</P><P><U>ChlD BioBrick Correction </U></P><P>As before, previous BioBrick (BB) from 2013 had a 50bp deletion/error within the ChlD (900bp) from using a single restriction digest. Our Aim is to excise the entire ChlD gene using Apa1 and Mlu1 restriction enzymes and to insert a complete ChlD gene into a BB.  We attempted this experiment again but did restriction enzyme digestions with the two enzymes separately to improve efficiency of cutting.</P><P>To prevent re-joining after digestion of our cut vector, we treated our samples with Alkaline Phosphatase (Fast A.P.). The DNA was then run on a 1% Agarose gel. 5 bands were identified and using a 1Kb ladder a complete ChlD (900bp) band was found and excised for ligation.</P><H3>WET LAB WEEK 3</H3><H3> Thursday</H3><DIV>21/08/14 </DIV><P>More plasmid prep was done, the following 6 genes were inserted into an Ampicillin backbone.  Cells were grown to extract more plasmid for stocks. </P><P>Chl1</P><P>Chl2</P><P>YCF54</P><P>ChlP</P><P>DVR1</P><P>POR</P><P>Composite Part Assembly: Trouble-shooting with the BioBrick assembly protocol, we found that if we ligated in a particular way then the plasmid linearises itself and then cannot be cut for the making of composite parts. We then started working on forming test composite parts. Our stocktake was also completed.</P><P>Composite parts were assembled of: lac + GUN4 + ChlI1 ; lac + GUN4 + ChlI2 ; lac + ChlI1 + GUN4. Digests were done with EcoRI &amp; SpeI on first gene in part (with lac), and separately with XbaI and PstI for the second gene. These were then ligated into a KAN backbone which was digested with EcoRI &amp; PstI. Digests and ligation steps ran at 37C for 1h and 80C for 20 minutes.</P><P>DH5-a electrocompetent E.coli were transformed via electroporation and plated out onto KAN LB-agar.</P><P>Transformations: Electroporation does not appear to be working well.  We changed to heat shock to transform our cells. There may be a problem with our electro-competent cells. Made more electro-competent cells to test. Also made chemical competent cells for heat-shock transformation</P><P><B>DRY LAB WEEK3 </B></P><P><B>Thursday</B></P><P><B>21/08/14</B></P><P>Decided on Outreach ideas. FINALLY. Online reality contest “So You Think You Can Synthesise”. Had discussions of framework for competition, making a trailer.</P><P>Met up with MQ Media Team later during the week</P><P>Started to put together the sponsorship package</P><H3>WET LAB WEEK 4 </H3><H3>Thursday </H3><H3><P><B>28/08/14</B></P><P><U>Digests</U></P>
Digests of GUN4+ChlI2 &amp; ChlD to check results from last week<P><U>Competent cells</U></P>
Electroporation does not seem to work and create viable competent cells ergo we shall stick to the heat shock methodology for further preps. More cells were made and used for plasmid preps, BB’s and composite parts.<P><U>Composite Parts </U></P><P>4:1 insert – vector ratio for Fast AP ligation steps to produce: </P><P><I>AMP backbone</I></P><P>CTH1 + YCF54</P><P>CTH1 + Plasto</P><P>ChlP + ChlG</P><P><I>CAM backbone</I></P><P>ChlD</P><P><I>KAN backbone</I></P><P>GUN4 + ChlI2</P><P>GUN4 + ChlI1</P><P>ChlI1 + GUN4</P></H3><H3>September 2014<DIV><P><B>Week 6</B></P><P><B>Thursday </B></P><P><B>04/09/14</B></P><P>A busy week!</P><P>Composite parts that had growth were digested with X &amp; P and run on 1% Agarose gel to check insert size. </P><P>PCR was also performed with BioBrick Forward and BioBrick Reverse primers to see if we could confirm correct assembly of composite parts. </P><P>further composite parts were assembled on the AMP backbone</P><P>ChlM + YCF54</P><P>POR + DVR1</P><P>POR + ChlP</P><P><I><B>Figure</B> Composite part RE digest images from plates that had growth</I></P><P><B>Friday</B></P><P><B>05/09/14</B></P><P>Liquid cultures of composite part transformants</P><P>Plasmid preps</P><P>RE digest of each part – into CAM BB</P><P>Competent cell prep: both chemical and electro-competent cells were made</P><P>New composite parts made:</P><P>/ChlM+YCF54/</P><P>/POR+DVR1/</P><P>/POR+ChlP/</P></DIV></H3><H3>September 2014</H3><DIV><P><B>Week 7</B></P><H3>WET LAB WEEK 6</H3><P>Wednesday 10/09/14</P><P>Ran PCR products from last week</P><P>ChlD</P><P>CTH1 + YCF54</P><P>CTH1 + Plasto</P><P>ChlI2 + GUN4</P><P>GUN4 + ChlI2</P><P>Plasmid preps done</P><P><B>Thursday 11/09/14&lt;</B></P><P><U>Sequencing</U></P><P>Plasmids were positive, sent to Macrogen for sequencing:</P><P>CTH1 + YCF54</P><P>ChlD</P><P>Gun4 + ChlI1</P><P>CTH1 + Plasto</P><P>Composite Checks</P><P>These composites were cut as a single digest and a double digest, and run on an agarose gel. Sizes were compared,</P><P>POR + ChlP</P><P>POR + DVR1</P><P>ChlM + YCF54</P><P>CTH1 + Plasto</P><P><I><B>Figure</B> Composite part screenings</I></P><P><U>Composite Parts</U></P><P>New composite parts were made, and built upon, transformed and plated out:</P><P>/CTH1+YCF54/ + Plasto</P><P>/CTH1+YCF54/ + ChlM</P><P>ChlI2 + ChlI1</P><P>ChlI2 + GUN4</P><U>Open Day</U><P>Saturday 13/09/14</P><P>Set up chromatography reactions in preparation for open day on Saturday.</P><P>Fluorescent plates drawn, grown, ready to go for Saturday.</P><P>Plasmid prep done for previous composite parts. CHlH has finally worked</P><B>WET LAB WEEK 7</B><P>Monday</P><P>15/09/14</P><P>Nanodrops of plasmid preps. </P><P>single and double digests of every second plasmid (a, c, e, f) run gel. Send for sequencing if successful.</P><P>Wednesday</P><P>17/09/14</P><P>gels re-labelled</P><P>transformations</P><P>CTH1 + YCFS4 + Plasto &lt;- ChlM</P><P>GUN4 + ChlD + CHlI2</P><P>&lt;&gt;Thursday 
18/09/14</P><P>New Composites</P><P>POR + ChlP + ChlG</P><P>POR + DVR1 + ChlG</P><P>POR + DVR1 + ChlP</P><P>ChlD</P><P>All transformed and plated out. </P><P><U>ChlD Fix</U></P><P>Apa1 &amp; Mlu1 digests with the backbone being treated with Fast AP in Mlu1 digest reaction. Ligations was as usual. </P><P>Then transformed and plated out onto CAM plates. Gel digests were run to resolve the 50bp difference </P><P>(850-900). </P><P>The resolution was seen.</P><P>^ Colony screen apaI mluI ChlD 850 and 900 + ChlH</P><P>^ pET ApaI MluI Digest Gel</P><P>ChlH PCR reaction was also run</P><H3>WET LAB – MIDSEM BREAK W1</H3><H3><P><B>Monday</B></P>
22/09/14<P>Liquid Cultures from Thursday plates. </P><P>POR + ChlP + ChlG</P><P>POR + DVR1 + ChlG</P><P>POR + DVR1 + ChlP</P><P>ChlD</P><P>Restriction Enzyme Digest of CTH1 + YCFS4 + Plasto + ChlM + ChlD composite part.</P><P>Gel run of ^ composite part and ChlD to compare with pET to confirm to presence of the 50bp. </P><P><B>Tuesday</B></P><P>23/09/14</P><P>EcoRI + X/P digests for plasmids from Mon. </P><P>ChlD A + M individual digests, compared against pET with same digest. </P><P>Re-screen CTH1 + YCFS4 + Plasto + ChlM colonies and grow in liquid culture</P><P><B>Wednesday
24/09/14</B></P><P>Recheck: ChlH in KAN and CAM both were not okay. </P><P>CTH...ChlM: not okay therefore re screen plates from liquid cultures. </P><P>Composite part-&gt; POR + DVR1 + ChlP + ChlG</P><P>Transform: ChlI2 + GUN4 + ChlD} new composite and plate out.  </P><P>Rescreen plated samples of CTHI + Plasto &amp; ChlM + YCFS4 all in AMP and put into liquid culture. </P><P><B>Thursday
25/09/14</B></P><P>Liquid culture of ChlI2 + GUN4 + ChlD</P><P>prep done in afternoon </P><P>POR ...ChlG transformed and plated out</P><P>CTHI ...ChlM gel resolved and excited for friday purification.</P><P>ChlI2 + GUN4, CTH1 + YCF54 and POR + DVR1 into CAM backbones. </P><P>prepare ChlI2 + GUN4, CTH1 + YCF54, ChlD for sequencing</P><P>Plasmid prep of CTHI + Plasto, ChlM + YC5S4. </P><P><B>Friday 26/09/14</B></P><P>CTHI ..ChlM gel band purified</P><P>Liquid cultures of new composite parts (ChlI2 + GUN4 + ChlD POR..ChlG) </P><P>Plasmid prep of CTHI + plasto (A-D) and ChlM + YCF54 (A-D) </P><P>Restriction enzyme screen plasmid prep from thursday. </P><P>Transformation of GEl extracted plasmids (linear [total of 10 for transformation] + circular + ligation) </P><P>*1ul ligase and 4.5 ul ligase buffer. </P><P>37oC for 1 hour and 80oC for 20mins.</P><P>5ul linear plasmid → 50ul competent cells (chemical) </P><P>→ 10ul circular plasmid → 50ul competent cells </P><P>Gel Run for ChIl2 + GUN4 + ChlD, ChlH1 + Plasto &amp; ChlM + YCF54 cuts. </P><P>LANE ORDER:</P><P>1.1….13.2-  ChIl2 + GUN4 + ChlD - ChlH1 + Plasto - ChlM + YCF54</P><P>*note: The wells for the last  3 lanes did not accept much of the sample. The sample would float to the surface when being inserted. </P><P>The transformants were plated out, the digest resolution wasn’t clear and needs to be re run on monday. 
liquid cultures of Sunday transformants 28/9 (CTH1...ChlM gel purified CTH1 + YCF54 (CAM), ChlI2 + GUN4 (CAM). </P></H3><H4>WET LAB – MIDSEM BREAK W2</H4><P><B>Monday
29/09/14</B></P><B><P>plasmid preps of:</P></B><PCHLM cam=""><P>CTH1..ChlM gel purified</P><P>CTH1..ChlM re-screen</P><P>POR..ChlG</P><P>POR + DVR AMP</P><P>CTH1 + YCF CAM</P><P>ChlI2 + GUN4 CAM</P><P><U>Re Digest and Gel which all results were good: </U></P><P>ChlI2 + GUN4 + ChlD</P><P>CTH1 + plasto AMP</P><P>ChlM + YCF54 AMP</P>

Gel: 

<P><I><B>Figure</B> ChlH linearized and gel purified. </I></P><P>Digest + gel: CTH1..ChlM and POR...ChlG</P><P>CTH..ChlM plasmid prep kept to H, I, J</P><P>POR ..ChlG plasmid prep kept to 1:1, 1:3, 1:6</P><P>ChlH upper and lower bands excised &amp; gel purified</P><P><B>Tuesday
30/09/14</B></P><P>Re-transformed and plated out:</P><P>CTH..ChlM x 3</P><P>POR..ChlG x 3</P><P>ChlH x 2</P><P>CTH..+ POR.. mixed x 2</P><P>Gel run on CAM transformants and re-screen of:</P><P>POR + DVR, ChlI2 + GUN4, CTH + YCF</P><P>Composite part created, transformed and plated out: ChlI2 + GUN4 + ChlD + ChlI1</P><P>ChlM cut into CAM Backbone, transformed and plated out. Tested  for registry </P><P>PCR reactions run on final constructs and checked for</P><P>F &amp;R from ends:</P><P>CTH...ChlM</P><P>POR...ChlG</P><H3>October 2014</H3></PCHLM><DIV><P><B>Week 9</B></P><P>Wednesday (1/10/14)</P>
PCR:
<P>1ul BB</P><P>5ul buffer (x10)</P><P>1ul F primer</P><P>1ul R primer</P><P>1ul dNTP</P><P>0.25 ul Taq</P><P>0.75 ul H2O (to Evel 50ul) </P><P><B>Master Mix:</B></P><P>50ul buffer </P><P>10ul dNTP</P><P>2.5ul Taq</P><P>407.5ul H2O</P><P>------ X ------</P><P>  C1                  +          C2          → CTH1 + ChlM (H)</P><P>(CTF + BBR)       (CMR + BBR) </P><P> C3                  +          C4          → CTH1 + ChlM (I)</P><P>(CTF + BBR)       (CMR + BBR) </P><P> C5                  +          C6          → CTH1 + ChlM (J)</P><P>(CTF + BBR)       (CMR + BBR) </P><P>PI → POR A        P2→ PORB         P3 → PORC</P><P>-----X-----</P><P>Setup for functional assays and plasmid preps of ChlM &amp; ChlI1 + ChlI2 </P><P>Thursday </P><P>2/10/14</P><P>Cyclase assay (100ul) </P><P>MPE - 8ul </P><P>1mM NADP - 10ul </P><P>10mM G-P-P - 10ul </P><P>Assay buff 1 x (50mM tricine, 2mM MgCl2, 1mM DTT 10% glycerol, pH 8.0) - 50ul 2x</P><P>0.5ul of G-6-P-dehydrogenase </P><P>total volume = 78ul allowing 22ul for other additions.</P><P>#1 Blank (water)</P><P>#2 22ul CTH1 part 1 </P><P>#3 11ul CTH1 part 2</P>
#4 11ul CTH<P>Bradford functional assays were done on induced cell pellets </P><P>10% Glycerol + 5mM Tricine NaOH ph8.0 + 2mM MgCl2 + 1mM DTT</P><P>Results:</P><P>            5µl         2.5 dilution</P><P>POR - ChlG   4         ~ 1.25        1
              3      ~ 1.75         1.5
              2        ~ 2        1.5
              1     ~ 2         1.5
 </P><P>CTH - ChlM   1        ~ 1        1
              2     ~ 1         1</P><P><B>Friday</B></P><P><B>3/10/14</B></P><P>Protein weight estimates:</P><P>ChlM - 30440 Da</P><P>CTH1 - 43873.3 Da</P><P>YCFS4 - 17073.7 Da</P><P>Plasto - 10339 Da</P><P>ChlI 1 - 39952 Da</P><P>GUN4 - 2450.6 Da</P><P>ChlD - 76420.1 Da</P><P>POR - 41871 Da</P><P>DVR1 - 37034 Da</P><P>ChlP - 47011 Da</P><P>ChlG - 36880 Da</P><P>Protein gels run of two complete composites. </P><P><I><B>Figure</B> ^CTH1-ChlM composite</I></P><P><I><B>Figure</B> ^POR-ChlG composite</I></P><P>The CTH1-ChlM composite showed good separation of products. The POR-ChlG composite had separation but only three parts were able to be easily identified. As Annotated the selected bands were cut-out for in-gel digestion &amp; analysis by MALDI- TOF/TOF.</P><P>WET LAB WEEK 8</P><P>Wednesday</P><P> 8/10/14</P><P>Plasmid preps were done on the ChlI1 - ChlD composite parts</P><P>Nanodrops:</P><P>Sample         nucleic acid conc (ng/µl)    260/280</P><P>ChlI1 + ChlID         179.6                 1.92</P><P>ChlI1 + ChlID         296.3                1.87</P><P>ChlI1 + ChlID         261.7                1.93</P><P>ChlI1 + ChlID         648                1.90</P><P>ChlI1 + ChlID         118.7                2.07</P><P>ChlI1 + ChlID         179.1                1.79</P><P>ChlI1 + ChlID         553.7                1.93 </P><P>ChlI1 + ChlID         321.6                1.92</P><P><I><B>Figure</B> All 8 were then digested and run on a gel, 5-10mL liquid cultures were also prepared and left to incubate overnight. </I></P><P>5-10mL liquid cultures were made of the following parts in an AMP backbone with a lac promoter.
ChlI2, YCFS4, ChlP, POR, GUN4, ChlM, ChlG, CTH1, CTH1-ChlM, POR-ChlG for large scale growth (50mL) for functional assays.</P><P>Thursday (9/10/14)</P><P>New composite ChlI1 + ChlD + GUN4 transformed and plated out. Intermediates and final parts prepped and sent for sequencing to confirm that the inserts are what they’re supposed to be. Gylcerol stocks were made of the current intermediates and final composites. Final parts in an AMP backbone were induced (OD600 = 0.4-5) for functional assays.</P><P>WET LAB WEEK 9</P><P>Monday</P><P> 13/10/14</P><P>Plasmid prep of ChlI1+ChlD+GUN4, All cultures were screened and prepped for sequencing.</P><P>Nanodrops:</P><P>Sample         nucleic acid conc (ng/µl)     260/280</P><P>ChlI1+ChlD+GUN4     539.2                1.86</P><P>ChlI1+ChlD+GUN4    315.4                1.91</P><P>ChlI1+ChlD+GUN4    493.7                1.90</P><P>ChlI1+ChlD+GUN4    164.0                1.91</P><P>ChlI1+ChlD+GUN4    217.5                1.90</P><P>ChlI1+ChlD+GUN4    447.0                1.90</P><P>ChlI1+ChlD+GUN4    499.2                1.90</P><P>All composites re-run on gels for results page, SDS-PAGE gels and MS/MS prep. </P><P>POR-ChlG and ChlI1-GUN4 parts were re-transformed for lysate harvesting.</P><P><I><B>Figure</B> ^ ChlI - ChlM Final gels</I></P><P><I><B>Figure</B> ^ POR-ChlG Final gel</I></P><P>Tuesday </P><P>14/10/14</P><P>POR assay’s were run with GUN4 activing as a negative control. We expected to see a major peak at ~630nm and a</P><P>secondary peak at ~670nm, but on both runs (a 20minute and 1.5hours) the second peak was still not visible.</P><P>Cyclase assays were also run mirroring experimentation from 2/10/14 on CTH1 and POR, these were run overnight </P><P>with an expected peak at ~590nm showing presence of the Mg - protoporphorin intermediate. The Mg is apparent but </P><P>no intermediates have been generated.</P><P><B>Wednesday</B></P><P><B>15/10/14</B></P><P>Liquid cultures were made from plate cultures from the previous day (POR-ChlG, CTH1, ChlI1-ChlD-GUN4, POR). The </P><P>liquid cultures from the POR-ChlG &amp; ChlI1+ChlD+GUN4 composites were then induced for growth as 50mL cultures for</P><P>French Pressing. The resulting proteins were then run on SDS-PAGE gels and used for functional assays; gels were</P><P>destained and bands cut for in-gel digestion and MALDI-TOF/TOF.</P><P>Thursday </P><P>16/10/14</P><P><I><B>Figure</B> ^ CTH1 - ChlM Final gel</I></P><P><I><B>Figure</B> ^ POR-ChlG Final Gel</I></P><H4>Thursday 16/10/14</H4><U>Assaying CTH1, YCF54 and Plastocyanin</U><P>Upon building the second operon including CTH1, YCF54 and Plasto, a functional assay testing the successful or unsuccessful expression of the yielding protoChlorophyllide was made. The assay mix included;</P>
Insert table code
Reagents
Volume
5µM magnesium-protoporphyrin IX monomethyl ester
8µl
1 µM NADPH
10µl
10 µM glucose 6 phosphate
10µl
1 µg/ml glucose 6 dehydrogenase
0.5µl
50 mM tricine
5 µl
2mM MgCl
5 µl
1mM DTT
5 µl
<P>Extract from Chlamydamonas was added to;</P><P>1. Supernatant + membrane (1:1)</P><P>2. Supernatant only</P><P>3. Membrane</P><P>9 CTH1 supernatant samples were assayed, followed by the addition of 5 µl of water and centrifuged and was run on </P><P>HPLC for 1.5 hrs.
</P><P>No significant protoChlorphyllide levels were detected using this assay, this technique however may be used in future assays,</P><H4> Friday 18/10/14 </H4><U>Assaying the second operon CTH1, YCF54, Plasto including ChlM</U><P>Including the final component necessary for the production of protoChlorophyllide in the second operon was the addition of ChlM.
</P><P>The assay mix was added to the pellet and was as follows;</P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>Reagents</TH><TH>Volume</TH></TR><TR><TD>Resuspension buffer</TD><TD>Made to 44uL</TD></TR><TR><TD>SAM 1mM</TD><TD>1uL</TD></TR>

Reagents
Volume
Resuspension buffer
Made to 44 µl
SAM 1mM
1 µl
MgP 20 µM
2 µl 
 <P>Figure x shows the MP standard elutes at 9.8 minutes, the precursor to the catalysis to MPE</P><P>Figure x shows the MPE standard shows a peak at 10.4 minutes </P><P>Figure x depicts the SAM control shows a significant peak at 9.8 minutes showing it does not result in the MPE peak at 10.4 minutes.</P><P>Therefore, figure x shows the assay presents a peak at 9.7 minutes. Presence of MP and the absence of MPE indicates the lack of conversion of MP to MPE. This may be due to a low level of expression or low ChlM activity. </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Macquarie_Australia/WetLab/Notebook">http://2014.igem.org/Team:Macquarie_Australia/WetLab/Notebook</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Macquarie_Australia/WetLab/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Macquarie_Australia/WetLab/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Macquarie_Australia/WetLab/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Macquarie_Australia/WetLab/Notebook&amp;oldid=400249" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>