<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Queens_Canada_ChimeriQ"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Queens_Canada/ChimeriQ">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Queens_Canada/ChimeriQ&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Queens_Canada/ChimeriQ&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Queens_Canada/ChimeriQ&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Queens_Canada/ChimeriQ" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Queens Canada/ChimeriQ</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="topbarcontainer"><DIV id="topbar"><DIV id="menu"><UL class="menulist"><LI class="menuitem1"><A href="https://2012.igem.org/Team:Queens_Canada">Home </A></LI><LI class="menuitem2"><A href="https://2012.igem.org/Team:Queens_Canada/Team">Team</A></LI><LI class="menuitem"><A href="https://2012.igem.org/Team:Queens_Canada/ChimeriQ">ChimeriQ</A></LI><LI id="desc">
Description
</LI><LI id="results">
Results
</LI><LI id="parts">
Parts
</LI><LI id="nblink">
Notebook
</LI></UL><LI class="menuitem"><A href="https://2012.igem.org/Team:Queens_Canada/Guide/DNA">Guide</A></LI><LI id="Guide_DNA">
DNA
</LI><LI id="Guide_mRNA">
mRNA
</LI><LI id="Guide_Protein">
Protein Structure
</LI><LI class="menuitem"><A href="https://2012.igem.org/Team:Queens_Canada/SynthetiQ">SynthetiQ</A></LI><LI id="whowhy">Who and Why?</LI><LI id="how">How?</LI><LI id="dyphd">Dance Your PhD
          </LI><LI class="menuitem"><A href="https://2012.igem.org/Team:Queens_Canada/Partners">Partners</A></LI><LI id="sponsors">
Sponsors
</LI><LI id="collaborators">
Attributions
</LI><LI class="menuitem"><A href="https://2012.igem.org/Team:Queens_Canada/Safety">Extras</A></LI><LI id="safety">
Safety
</LI><LI id="Flight">
Flight of the Flagellin
</LI></DIV></DIV></DIV><DIV id="controlbar">
Control
</DIV><DIV id="labcontent"><DIV id="NotebookTitle"><P>ChimeriQ - Description</P></DIV><DIV id="notebookmenu" style="border-bottom: 1px solid black"><UL id="gooeymenu1" class="gelbuttonmenu" style="z-index:99;"><LI><A href="#Overview"> Overview</A></LI><LI><A href="#Fluoresence"> Fluorescence</A></LI><LI><A href="#Catalysis"> Catalysis </A></LI><LI><A href="#CohDoc">Cohesin and Dockerin</A></LI><LI><A href="#Future">Future Applications</A></LI><LI><A href="#Future">References</A></LI></UL></DIV><DIV id="Overview" class="contenttitle"> 
Overview
</DIV><DIV class="contentbox"><P>
This year, our team is investigating new methods of increasing the efficiency of bioremediation and biosynthesis using modified bacteria. The development of the oil sands in Alberta, has resulted in the build up of toxic byproducts stored in massive tailings ponds. To help resolve these issues, our goals can be divided up into three main categories: the binding of pollutants, adhesion and aggregation of bacteria, and catalysis.
</P>
Most bacteria possess tail-like appendages called flagella, which can be genetically altered for novel functions. Each flagella is made up of a number of polymerizing proteins, often called flagellin. By making chimeric insertions in the variable domain of the flagellin, we can incorporate metal binding proteins, enzymes, adhesive proteins as well as scaffolding proteins to further extend the possible applications. To accomplish this, we can summarize the majority of our work into three main tasks: 
<UL><LI>
clone and modify the constant domains of the flagellin protein for making insertions using Biobricks and parts obtained from the wild.
</LI><LI>
design a flexible, compatible cloning method for efficientlymaking chimeric insertions using Biobricks and other parts
</LI><LI>
introduce binding and catalysis to the length of the flagella. 
</LI></UL></DIV><DIV id="Fluoresence" class="contenttitle">
Fluorescence
</DIV><DIV class="contentbox"><P> ChimeriQ, the project on tail-like appendages flagella, can be genetically altered for novel functions. As a proof of concept, we believed it would be most beneficially to use fluorescent proteins as a stepping stone to validated that proteins could indeed be expressed on the flagella.</P><P> Through the learning phase we researched intensively on the full flagella construct and what was required to incorporate protein infusions. As stated before, the D3 domain in the flagella construct has no real function, and can therefore be replaced with a protein of our choice. Through PCR overlap extension we would incorporate GFP and RFP into our constructs. To prove that it PCR overlap extension would worked, we would incorporate this PCR product containing the full fliC construct with no linkers with the GFP replacing the D3 domain (this was labelled nFFGFP - no linker Full Flic GFP construct) into the T7 promotor J04500 vector that contained the chloramphenicol resistance through digestion/ligation processes. We grew these ligations overnight and expected colonies. </P><P> As expected there were several colonies on each plate, to prove that the ligations worked only the colonies with green fluorescence were used for liquid colonies. Unfortunately we did not have access to an electron microscope, therefore we could not visually see the proteins on the flagella. Researching various flagellar isolation protocols , we developed two tested protocols to isolate the flagella containing GFP, which can be found in the protocols section. We also had a batch of J04500 controlled bacteria made to act as the control that would not contain protein infusions. </P><P> Essentially these two protocols would shear the flagella off of the cells using blender and heat techniques. The cell debris would be removed by centrifuge, and the sheared flagella would be collected as pellets. These isolated pellets when observed under the fluorescent microscope did glow green. Since there is a possibility that some cell debris would be present in the flagella pellets, we decided to run SDS-PAGE to see if there was a different between the controlled bacteria that would only contain flagella but not protein infusions. What we predicted was that at there should be a protein band at 70 kDa, which would be due to the GFP protein infused on the full fliC construct which would not be found on the control bacteria. Through running a few SDS- pages and comparing the control vs. fluorescent flagella bacteria, there was a distinct darker band at 70 kDa that was not found in the control. This means that it is likely that the flagella pellets in our nFFGFP (no linker Full Flic GFP construct) contained GFP proteins, which the J04500 bacteria did not. </P></DIV><DIV id="Catalysis" class="contenttitle">
Catalysis
</DIV><DIV class="contentbox"><P> A potential application of this project would be to externally catalyze useful chemical reactions. The only real practical limitation would be the size of the amino acid insert. Prior research in the field of chimeric flagella has shown that inserts of around 300 amino acids can be inserted into the FliC D3 domain without affecting the tertiary structure of the flagella. Thus, the search for feasible pathways was limited to reactions with steps that involved the use of a single enzyme and were sufficiently small. A potential parts in the biobrick registry that fit this criteria was XylE (catechol 2,3-dioxygenase). This enzyme converts catechol into 2-hydroxymuconate semialdehyde, which conveniently provides a bright yellow product. </P><P>
We also researched various pathways outside of the BioBrick registry for useful pathways and eventually settled on the enzyme Haloalkane dehalogenase. This enzyme is found in several organisms and refers to an enzyme the dehalogenates haloalkanes into alcohol and halides. Given that haloalkanes such as 1,2-dibromoethane are quite dangerous (Rated a 3 in the health section on the NFPA 704 scale [1]), this enzyme provides a useful means of cleaving these toxic products into safer products. We contacted various professors around the world, and were able to obtain the linB and the Rv2579 gene, which both encode the protein, but in Sphingomonas paucimobilis UT26 and Mycobacterium tuberculosis H37Rv, respectively. </P><P>
These enzymes were incorporated into E. coli in a similar manner to the fluorescent proteins. Using PCR overlap extension, we were able to create the plasmid for LinB, Rv2579, and XylE from an RFP plasmid template. The PCR overlap extension products were digested using DpnI and transformed into E. coli. The colonies were plated. As expected there were several colonies on each plate, and using the fluorescence of the E. coli we were able to isolate which colonies potentially contained the plasmid by picking the colonies without fluorescence.
</P></DIV><DIV id="CohDoc" class="contenttitle">
The Cohesin-Dockerin System
</DIV><DIV class="contentbox">
An exciting extension of our project included incorporating the cohesion/dockerin complex into our
flagella scaffold. Normally found in the cellulosome complex in the Clostridium bacteria, these proteins
interact with each other through a scaffoldin protein consisting of cohesins connected to dockerins
containing functional units such as the enzymes and carbohydrate binding modules. By incorporating
the cohesin protein in place of the D3 domain, we could have flagella that would be able to bind any
dockerins present. These dockerins would genetically modified to fuse with any enzymes or protein
of interest. The result would be a very flexible flagella scaffolding system that could bind enzymes
not natively expressed within its own flagellin sequence or genome. In theory we could have multiple
different dockerin/enzyme fusions bind to the same cohesion flagella scaffold; thereby giving our E.Coli
cell multiple functions or being able to express a full enzymatic pathway.
</DIV><DIV id="Future" class="contenttitle">
Future Applications
</DIV><DIV class="contentbox"><H3>Metal Binding</H3><P>Oil sands development in Canada is a very successful industry, however there are a number of environmental concerns associated with these developments. One result of these developments are significant amounts of heavy metals being released into the environment, including lead, mercury and cadmium [2]. The majority of heavy metal binding proteins are small in size, often under 200 amino acids, and should therefore insert comfortably within the flagellin protein, well within the size restriction [3]. There are a number of potential metal binding proteins found within the parts registry, including SmtA [4] and fMT [5], both of which are known to bind cadmium and are under 100 amino acids in length. We are hoping through the progress of our project future teams can successfully incorporate these proteins into flagella in order to remove metal contaminants from the environment.</P><H3>Nanotubes</H3><P>Nanotubes are microscopic cylindrical structures which have many applications due to their strength and flexibility, as well as their potential use in nanoscale circuits. The flagella of bacteria serve as naturally made nanotubes, and can be genetically modified to bind metals for the purpose of creating metal nanotubes [6]. Metal binding loops such as a histidine loop (four repeats of Gly-His-His-His-His-His-His) are able to bind metals including copper, gold and cobalt, and are small enough to fit easily within the flagellin protein [6]. By incorporating these loops within flagellin, it should be possible to create metal-bound flagellar nanotubes which may be used in the creation of circuits [6].</P></DIV><DIV id="References" class="contenttitle">
References
</DIV><DIV class="contentbox">
[1]1,2-dibromoethane MSDS.&quot; Sciencelab.com. Sciencelab.com, 9 Oct. 2005. Web. 3 Oct. 2012. <A href="http://www.sciencelab.com/msds.php?msdsId=9923711">http://www.sciencelab.com/msds.php?msdsId=9923711</A>.

[2]Kelly, E. N., Schindler, D. W., Hodson, P. V., Short, J. W., Radmanovich, R., and Nielsen, C. C. (2010) Oil sands development contributes elements toxic at low concentrations to the Athabasca River and its tributaries. PNAS, 107(37): 16178-16183.

[3]Westerlund-Wilkstrom, B., Tanskanen, J., Virkola, R., Hacker, J., Lindberg, M., Skurnik, M., and Korhonen, T. K. (1997) Functional expression of adhesive peptides as fusions to Escherichia coli flagellin. Protein Engineering, 10(11): 1319-1326.

[4]Miyake, K. (2011) SmtA. Registry of Standard Biological Parts.<A href="http://partsregistry.org/Part:BBa_K519010">http://partsregistry.org/Part:BBa_K519010</A>
[5]Kuipers, N. (2009) fMT. Registry of Standard Biological Parts. <A href="http://partsregistry.org/Part:BBa_K190019">http://partsregistry.org/Part:BBa_K190019</A>
[6]Kumara, M.T., Tripp, B.C., and Muralidharan, S. (2007) Self-assembly of metal nanoparticles and nanotubes on bioengineered flagella scaffolds. Chem. Mater. 19: 2056-2064.
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Queens_Canada/ChimeriQ">http://2012.igem.org/Team:Queens_Canada/ChimeriQ</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Queens_Canada/ChimeriQ" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Queens_Canada/ChimeriQ" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Queens_Canada/ChimeriQ&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Queens_Canada/ChimeriQ&amp;oldid=291107" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>