<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Queens_Canada_Guide_DNA"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Queens_Canada/Guide/DNA">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Queens_Canada/Guide/DNA&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Queens_Canada/Guide/DNA&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Queens_Canada/Guide/DNA&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Queens_Canada/Guide/DNA" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Queens Canada/Guide/DNA</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="topbarcontainer"><DIV id="topbar"><DIV id="menu"><UL class="menulist"><LI class="menuitem1"><A href="https://2012.igem.org/Team:Queens_Canada">Home </A></LI><LI class="menuitem2"><A href="https://2012.igem.org/Team:Queens_Canada/Team">Team</A></LI><LI class="menuitem"><A href="https://2012.igem.org/Team:Queens_Canada/ChimeriQ">ChimeriQ</A></LI><LI id="desc">
Description
</LI><LI id="results">
Results
</LI><LI id="parts">
Parts
</LI><LI id="nblink">
Notebook
</LI></UL><LI class="menuitem"><A href="https://2012.igem.org/Team:Queens_Canada/Guide/DNA">Guide</A></LI><LI id="Guide_DNA">
DNA
</LI><LI id="Guide_mRNA">
mRNA
</LI><LI id="Guide_Protein">
Protein Structure
</LI><LI class="menuitem"><A href="https://2012.igem.org/Team:Queens_Canada/SynthetiQ">SynthetiQ</A></LI><LI id="whowhy">Who and Why?</LI><LI id="how">How?</LI><LI id="dyphd">Dance Your PhD
          </LI><LI class="menuitem"><A href="https://2012.igem.org/Team:Queens_Canada/Partners">Partners</A></LI><LI id="sponsors">
Sponsors
</LI><LI id="collaborators">
Attributions
</LI><LI class="menuitem"><A href="https://2012.igem.org/Team:Queens_Canada/Safety">Extras</A></LI><LI id="safety">
Safety
</LI><LI id="Flight">
Flight of the Flagellin
</LI></DIV></DIV></DIV><DIV id="controlbar">
Control
</DIV><DIV id="labcontent"><DIV id="NotebookTitle"><P>DNA</P></DIV><DIV id="notebookmenu1" style="border-bottom: 1px solid black"><UL id="gooeymenu1" class="gelbuttonmenu" style="z-index:99;"><LI><A href="#Cut-sites"> Cut-sites </A></LI><LI><A href="#Bio_Fus"> Biobrick Fusion Standards</A></LI><LI><A href="#Cloning_methods"> Cloning methods</A></LI><LI><A href="#expression"> Parts for protein expression</A></LI></UL></DIV><DIV id="Cut-sites" class="contenttitle">
Cut Sites
</DIV><DIV class="contentbox">
Restriction Enzyme Cut sites are an essential element for creating protein chimers of any kind, not only for the creation of BioBrick Protein Chimers. As part of the protocol for creating these protein chimers, digestion and ligation will be involved. More often than not, digestion sites for restriction enzymes will be incorporated to the ends of the insert and the adjacent parts in order to ligate them together and into a plasmid. Therefore, it is critical to ensure that there are no cut sites present for the restriction enzymes that you will be using for digestion and ligation, otherwise it will be very difficult to get the full part ligated together with themselves and into the plasmid. For instance, if there is an EcoRI cut site present in the insert, then while you are digesting with the insert with EcoRI, the insert will be cut prematurely, and its function will likely be reduced. 
Removing restriction enzyme cut sites can be a very simple, or a very difficult process, depending on the location of the cut site in the desired part. If the part is close to the beginning or the end of the desired part, it may be simple to do a site directed mutagenesis. In order to do so, you would need to create a primer that overlaps the desired cut site, and change one nucleotide (preferably one that causes a silent mutation in the protein), such that the primer will still anneal to the template DNA, but will induce a single base pair nucleotide change that will remove the cut site from the PCR product. While the former is a very simple process, if the cut site is far enough away from the ends of the template DNA, this process is unlikely to work, as the primers will be too long. In this case, the simplest way to remove the cut site will be to have the DNA directly synthesized, although this can end up being a very costly process.

Below are the Restriction Enzyme cut sites for various BioBrick standards
<TABLE style="vertical-align: text-top;"><TBODY><TR><TD width="25%" text-align="center">
BB RFC [10]
<UL><LI>EcoRI site: GAATTC </LI><LI>XbaI site: TCTAGA </LI><LI>SpeI site: ACTAGT </LI><LI>PstI site: CTGCAG </LI><LI>NotI site: GCGGCCGC </LI></UL></TD><TD width="25%" text-align="center">
BioBrick BB-2 RFC[12]
<UL><LI>EcoRI site: GAATTC </LI><LI>SpeI site: ACTAGT </LI><LI>NheI site, GCTAGC </LI><LI>PstI site: CTGCAG </LI><LI>NotI site: GCGGCCGC </LI></UL></TD><TD width="25%" valign="top" text-align="center">
Berkeley RFC[21]
<UL><LI>EcoRI site: GAATTC </LI><LI>BglII site: AGATCT </LI><LI>BamHI site: GGATCC </LI><LI>XhoI site: CTCGAG </LI></UL></TD><TD width="25%" text-align="center">
Silver RFC[23]
<UL><LI>EcoRI site: GAATTC </LI><LI>XbaI site: TCTAGA </LI><LI>SpeI site: ACTAGT </LI><LI>PstI site: CTGCAG </LI><LI>NotI site: GCGGCCGC </LI></UL></TD></TR></TBODY></TABLE></DIV><DIV id="Cloning_methods" class="contenttitle">
Cloning Methods
</DIV><DIV class="contentbox"><H3>PCR Overlap Extension</H3><P>When cloning parts into our construct, we chose to use PCR overlap extension and digestion/ligation
techniques. In order to perform PCR Overlap extension (our preferred method of cloning) we were
required to fabricate a set of primers that were ligated onto the insert. These primer extensions served
to add overlap sites that match those of the parts we were adding. This allowed for PCR to extend
the part onto another part with a matching overlap site; with the primers on either end of the inserts
acting as the site of initiation for Taq polymerase. Our primary resource was this <A href="http://www.biotechniques.com/BiotechniquesJournal/2010/June/Overlap-extension-PCR-cloning-a-simple-and-reliable-way-to-create-recombinant-plasmids/biotechniques-280116.html">paper</A>. A picture
representation of PCR-Overlap Extension can be seen below:</P><H3>High Throughput Cloning</H3><P>The protocol for this method is standard.</P>
98C for 30s
60C for 30s
72C for 1.5/kb
- 18 cycles optimum
72C for 10min (final extension)
37C for 30min - Add Dpn1 (digest methylated DNA)
Tranform<P>
The choice of polyemerase is important! The article suggests using Phusion DNA Polymerase for the best results. We used Kapa Hi-Fi Hotsart Mix and confirm that it worked. The reference includes data for other polymerases as well.</P>
Obtained from the following reference, under the Supplementary Material:<A href="http://www.biotechniques.com/BiotechniquesJournal/2010/June/Overlap-extension-PCR-cloning-a-simple-and-reliable-way-to-create-recombinant-plasmids/biotechniques-280116.html">LINK</A> Anton V. Bryksin and Ichiro Matsumura. (2010) Overlap extension PCR cloning: a simple and reliable way to create recombinant plasmids. BioTechniques, Vol. 48, No. 6, pp. 463–465<P>Because this method is standard over all of our parts, we can use it for high throughput cloning. For example, if we had 96 parts in our standard, we can fill a 96 well PCR plate and run this program, digest with Dpn1, then transform all of these parts. This would about 3-4 hours, with the main amount of time being the PCR reaction. So in the case of chimeric protein design, if we wanted to screen a number of cloned structures configurations to find one that folded the way we wanted it to, we could use this method to rapidly clone and test a series of trials.</P><P>We can liken this to crystallography, which can be done at high throughput by robots.</P><P><B>An RFC is in the process of being submitted for this standard.</B></P><H3>Standard Cloning</H3><P>The other method of cloning made use of the cut sites found in our insert and a few restriction enzymes.
Early on we noted the BioBrick standard caused a very problematic frame-shifts within the scar when
ligating 2 or more parts. We tried to circumvent the problem by adding 1 or 2 nucleotides onto our DNA
sequence to fix the frame but found this to be tedious due to the fact that this varied based on the part
we wanted to ligate.</P><P>Therefore we started doing research into other assembly standards to see if any of them alleviated
the problem. We judged the merits and flaws of each one (a list of the assembly standards considered
can be found <A href="http://openwetware.org/wiki/The_BioBricks_Foundation:Standards/Technical/Formats">here</A>). What we searched for was a standard that would avoid frame-shifts and nonsense
mutations when doing protein fusion, avoid N-terminal destabilization signals and dam methylation sites
and preserve most of the restriction enzymes from the original BBa standard.</P><P>Considering these reasons, we chose to switch our prefixes and suffixes to Tom Knight’s BBa 2 standard
that allowed seamless in-frame ligations between multiple sequences of successive DNA. This change
resulted in only 1 restriction enzyme (Xba1 to Nhe1) that differed from original BBa standard therefore
providing the flexibility we needed to simultaneously work with both standards. For ligation, we used T4
ligase. A pictorial representation of the BBa 2 standard can be seen below:</P><DIV id="seqbox"><PRE>
      Prefix                      Suffix
    EcoRI    SpeI              NheI     PstI
5' GAATTC...ACTAGT ...part... GCTAGC...CTGCAG 3'

Fusing two parts would then leave the following scar:
5' ...part A... GCTAGT ...part B... 3'
                A  S
</PRE></DIV><P>We also considered making our own artificial restriction enzymes in order to cut DNA off-site. To do
so, we would have made zinc fingers to bind DNA and combined them with enzymes capable of cutting
away from the binding site. This would allow us to cut as close to the gene as possible.</P></DIV><DIV id="Bio_Fus" class="contenttitle">
Biobrick Fusion Standards
</DIV><DIV class="contentbox"><P>When designing our flagella construct we examined a number of different ways of creating fusion proteins. Using the original BioBrick assembly standard does not work for fusion proteins, as it results in a frame shift and stop codon in the scar. At first we worked on creating a new method of fusion protein production, however we ran into many of the same problems as the standards that were already present. We went through the various fusion standards present on the openwetware web page, analyzing their different merits, eventually settling on Tom Knight’s BB-2 standard. You can follow this <A href="http://openwetware.org/wiki/The_BioBricks_Foundation:Standards/Technical/Formats">link</A> for some information on the various standards we examined.</P><H3>Biofusion (Silver Lab)</H3><DIV id="seqbox"><PRE>
	  Prefix	                        Suffix
5’ GAATTC GCGGCCGC T TCTAGA ...part... ACTAGT A GCGGCCG CTGCAG 3’
   EcoRI  NotI        XbaI	        SpeI     NotI    PstI

This results in the following scar:

5’ ...part A...ACTAGA...part B...3’
               ThrArg
</PRE><P>The Biofusion standard slightly adjusted the original BioBrick standard such that the scar would now code for arginine instead of a stop codon, and the fusion would be in frame. Also, this standard uses the same enzymes as the original, so no new enzymes must be purchased. However, some issues do arise.These include the fact that the arginine present in the scar may be problematic in some cases due to its large size and the fact that it is coded by a rare codon. Also, Dam methylation may block cloning if the fused protein begins with Serine.</P></DIV><H3>Fusion Parts (Freiburg iGEM 2007)</H3><DIV id="seqbox"><PRE>
		 Prefix				            Suffix
5' GAATTC GCGGCCGC T TCTAGA TG GCCGGC...part...ACCGGT TAAT ACTAGT A GCGGCCG CTGCAG 3'
   EcoRI    NotI      XbaI  Met NgoMI           AgeI        SpeI     NotI    PstI

This results in the following scar:

5’ ...part A...ACCGGC...part B... 3’
	       ThrGly
</PRE><P>The Fusion Parts standard proposed by Freiburg serves to allow fusion proteins but remove some of the disadvantages of the Biofusion standard. It does this by adding two new restriction enzymes: NgoMI and AgeI. This standard allows a less disruptive scar of threonine and glycine, while maintaining in frame fusions. However, this does require the purchase of two new restriction enzymes.</P></DIV><H3>BglBricks (Berkeley)</H3><DIV id="seqbox"><PRE>
	 Prefix			   Suffix
5’ GAATTC ATG AGATCT...part...GGATCC TAA CTCGAG 3’
   EcoRI      BglII           BamHI      XhoI

This results in the following scar:

5’ ...part A...GGATCT...part B...3’
	       GlySer
</PRE><P>The BglBricks standard allows in frame production of fusion proteins with a benign scar of of glycine and serine. The prefix and suffix are also quite simple, containing only the 2 restriction sites connected by 3 bases. However, this method uses 3 new enzymes: BglII, BamHI and XhoI, of which BglII and BamHI cannot be heat-inactivated.</P></DIV><H3>BB-2 (Tom Knight)</H3><DIV id="seqbox"><PRE>
	Prefix		         Suffix
5’ GAATTC...ACTAGT...part...GCTAGC...CTGCAG 3’
   EcoRI     SpeI            NheI     PstI

This results in the following scar:

5’...part A...GCTAGT...part B... 3’
              AlaSer
</PRE><P>The BB-2 standard proposed by Tom Knight was the method we ended up following to produce our chimeric flagella. This standard requires the addition of only one new restriction enzyme (NheI), and produces in frame fusions with a benign scar of alanine and serine. Also, the NheI cut site is fairly rare in E. coli, decreasing the likelihood of having cut sites within proteins, and this enzyme can be heat-inactivated. The main concern with this method is that it has not yet been tested extensively to prove its value as a standard method of Biobrick assembly.</P></DIV></DIV><DIV id="expression" class="contenttitle">
Parts for protein expression
</DIV><DIV class="contentbox"><H3>Protein Expression</H3><DIV><P>As with any other gene sequences and parts, it was important to consider how to control the rate
of expression for the protein, i.e. induce transcription of the insert and subsequent translation.
Different promoter sequences are often incorporated upstream of the gene of interest to allow RNA
polymerase a site to bind to in which to start transcription. Likewise ribosomal binding sites (RBS) are
added downstream of the promoter to allow ribosomes to attach to the transcribe mRNA sequence.
Depending on the promoter and RBS used, the level of expression for the protein can change anywhere
from being produced in abundance or having little produced at all.</P><P>In our project we decided to use the <B>IPTG (Isopropyl β-D-1-thiogalactopyranoside) inducible promoter
with RBS (Part:BBa J04500)</B>. The complex is made up of the LacI regulated promoter found naturally in
E.coli and an rbs based off of a repressilator.</P></DIV><H3>Promoter</H3><DIV><P>The promoter used is the natural promoter from the LacZYA operon. It contains two protein binding
sites; the first is for the CAP protein which is found in E.coli and associated with cell health and glucose
supply. The second site binds LacI protein which is expressed to inhibit expression of the Lac operon
when no lactose is present. In the absence of the CAP protein and LacI, the promoter will increase
transcription. However, the promoter is naturally constitutive (meaning that by default it is in an ON
state) in E.coli strains that have low expression of LacI proteins (such as Top 10). Because of this fact
and prior experience with the part, we decided to keep using this promoter. In theory (and most likely
practice) any other constitutive promoter part would have sufficed for our project.</P><P>A crucial fact to note is that this part is incompatible in species containing active LacI coding regions
and environments containing lactose and lactose analogs. The former reason is because active LacI
expression will limit the efficiency of the promoter, the later because the presence of lactose would
continuously activate the promoter. Therefore these factors were considered before adoption of this
promoter.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Queens_Canada/Guide/DNA">http://2012.igem.org/Team:Queens_Canada/Guide/DNA</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Queens_Canada/Guide/DNA" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Queens_Canada/Guide/DNA" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Queens_Canada/Guide/DNA&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Queens_Canada/Guide/DNA&amp;oldid=297389" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>