"
Page
Discussion
View source
History
teams
Log in
 
Team:Queens Canada/Notebook/Week8
From 2012.igem.org
Home
Team
ChimeriQ
Description
Results
Parts
Notebook
Guide
DNA
mRNA
Protein Structure
SynthetiQ
Who and Why?
How?
Dance Your PhD
Partners
Sponsors
Attributions
Extras
Safety
Flight of the Flagellin
Control
Notebook - Week 8
Week
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17+
Project Diary
Protocols
Date
Protocol
People
DNA (if relevant)
Quantities and Parameters (if relevant)
Notes on Protocol/ Gel Lanes
Jun-17
Liquid Culture
Kevin, Beini
BL21 Cells
 
 
Jun-18
PCR
Beini, Victor, Andrew
Lead binding Protein, GFP w/ linkers, CFP
Mastermix
ddH2O 10 ul x 9 = 90 uL
Hotstart 12.5 uL x 9 = 112.5 uL
LP 0.75 uL
RP 0.75 uL
Template 1 uL
Total 25uL
Program:1. Denaturation (Initial): 95  for 3 mins 2. Denaturation: 95C for 20 seconds 3. Annealing: 60C for 15 seconds  4. Extension: 72 degrees, 20 seconds   5. Repeat step 2-4 for 20 cycles  6. Final extension 72 degrees, 5 minute  7. Cool down 4 degrees, one hour
 
 
Kapa Readymix PCR
David, Kevin, Dan
FliC C1 PCR Mastermix for 1/2 Colony PCR x 4
Mastermix
Kapa Readymix 50 uL
F primer: 3 uL
R primer: 3 uL
Colonies: 4 uL
Water: 40 uL
Total : 100 uL
Program
1. Denaturation (Initial): 95  for 5 mins 2. Denaturation: 95C for 20 seconds 3. Annealing: 60C for 15 seconds  4. Extension: 72 degrees, 20 seconds   5. Repeat step 2-4 for 30 cycles  6. Final extension 72 degrees, 5 minute  7. Cool down 4 degrees, one hour
 
 
 
Mastermix for 10 uL liquid colony x4
Mastermix
Kapa Readymix 50 uL
F primer: 3 uL
R primer: 3 uL
Liquid Culture: 40 uL
Water: 4 uL
Total : 100 uL
 
 
 
 
Mastermix for 5 uL liquid culture x 2
Mastermix
Kapa: 25 uL
F primer: 1.5 uL
R primer: 1.5 uL
Liquid Culture: 10 uL
Water: 12 uL
Total: 50 uL
 
 
 
 
Control Mastermix x2
Mastermix
Kapa: 25 uL
F primer: 1.5 uL
R primer: 1.5 uL
Water: 22 uL
Total: 50 uL
 
 
Double Gel Recipe
 
14 x 2 Lanes, a little bit bubbly
 
Gel:
1) Ladder, 2) PbrC, 3) Pbr1, 4) Pbr2, 5) GC, 6) G1, 7) G2
8) Ladder, 9) CFP C, 10) CFP 1, 11) CFP 2, 12) Control , 13) PC, 14) Blank, 15) Ladder, 16) P1, 17) P2, 18)P3, 19) P4, 20) Ladder, 21) L1, 22) L2, 23) L3, 24) L4, 25) L5, 26) L6, 27)Blank
Jun-19
Received biobrick parks
 
 
 
Got Biobrick ordered parts (31 of them)
had 27 LB plates so plated 24 part on
chloramphenicol using sterile loop and
agar stab plated 3 parts on ampicillinn
plates using sterile loop and agar stab
had 4 parts left over (3 amp, 1 specR)
placed the 27 streaked plates in the 37 degrees celcius incubator, placed the used agar stabs in the 4 degrees celcius fridge with the 4 unused agar stabs sitting at on the lab bench
 
LB Plates
Victor, Andrew
40 Plates were made, refer to Agar Plate Protocol
 
 
Jun-20
PCR
Victor, David, Faisal, Beini
8 Tubes Label
L1, L2 - Liquid colony normal left primer
C1, C2 - One colony normal left primer
LP1, LP2 - Liquid Colony w/ Promoter
CP1, CP2 - One colony w/ Promoter
Ctrl, Ctrl P - Control, Control w/ Promoter
PCR Mastermix
dd H20 10 uL
Hotstart 12.5 uL
LR 0.75 uL
RP 0.75 uL
Template 1 uL
Total 25 uL
Gel:
1) 1kB+, 2) CtrlP, 3) L1, 4) L2, 5) Lp1, 6) Lp2, 7) Blank 8) Blank, 9) Blank C, 10) Ctrl , 11) C1 , 12) C2, 13) Cp1, 14) Cp2, 15) Blank, 16) Blank,
Jun-21
Miniprep
Faisal, Beini, Phillip, Victor
 
 
 
 
Liquid Culture
 
 
 
 
 
PCR/Gel
Faisal, Beini, Phillip, Victor
FliC Parts ( 5 uL of template), PCR 14 trials total. Templates : mCitr (323 062)
mCer (323 023), N-split Citr (323 072), C-split Citr (323 055),
N-split ceru (323 011), C-split ceru (323 012)
PCR Mastermix
dd H20 10 uL
Hotstart 12.5 uL
LR 0.75 uL
RP 0.75 uL
Template 5 uL
Total 25 uL
Program:1. Denaturation (Initial): 95  for 2:30 mins 2. Denaturation: 98C for 20 seconds 3. Annealing: 60C for 15 seconds  4. Extension: 72 degrees, 15 seconds   5. Repeat step 2-4 for 15 cycles  6. Final extension 72 degrees, 3 minute  7. Cool down 4 degrees, one hour
Double Recipe Gel:
1) Ladder, 2)FliC Control,
3) Liquid Culture Trial 1, 4) Liquid Culture Trial 2, 5) Colony Trial 1, 6) Colony Trial 2, 7) Ladder, 8) C- split Control, 9) C-split Trial 1 (citrine), 10) C-split Trial 2 (citrine), 11) C-split Trial 1 (Cerulean), 12) C-split Trial 2 (cerulean), 13) Blank, 14) Blank, 15) Ladder, 16) Full Control, 17) Full Citrine Trial 1, 18) Full Citrine Trial 2, 19) Full Cerulean Trial 1, 20) Full Cerulean Trial 2, 21) Ladder, 22) N-split Control, 23) N-split Trial 1 (Citrine), 24) N-split trial 2 (Citrine), 25) N-split Trial 1 (cerulean), 26) N-split Trial 2 (cerulean) 27) Blank, 28) Blank
Jun-22
PCR/ Gel
David, Faisal, Beini
mCerulean K323023, mCitrine K323062, N-split mCer K323011,
C-split mCer K323012, N-split mCit K323072, CFP Series Control,
N-split series control.
Primers: CFP Series, Csplit FP_LP, Nsplit FP_RP, CFP_LP, N-split FP_RP
Mastermix
Kapa: 12.5 uL
F primer: 0.75 uL
R primer: 0.75 uL
Water: 10 uL
Template DNA: 1 uL
Total: 25 uL
Program:1. Denaturation (Initial): 95  for 5 mins 2. Denaturation: 98C for 30 seconds 3. Annealing: 55C for 15 seconds  4. Extension: 72 degrees, 35 seconds   5. Repeat step 2-4 for 30 cycles  6. Final extension 72 degrees, 5 minute  7. Cool down 4 degrees, one hour
Gel
1) Ladder, 2) C.C, 3) mCer1, 4) mCer2
5) mCit 1, 6) mCit 2, 7) NmCer1 8)NmCer2
9) Ladder, 10) NC, 11) CmCer1 12)CmCer2
13) NmCit 1, 14) NmCit 2.
 
Plate Streaking
Kevin
BL21 Competent Cells.
 
Monday June 18
In the morning, we looked at different protocols for colony PCR and from those, designed our own protocols, hoping at least one of them would work to amplify our fliC gene. Andrew, Victor, and Beini went into the lab to do PCR of fliC1, GFP, CFP, and PbrR. In the afternoon, Kevin brought in a delicately moist carrot cake topped with a smooth, rich, cream cheese-based icing. After everyone on the team had their share, we brought it over to the Chin-Sang lab for them to dig in. Aaron, David, Dan, and Kevin went into the lab to do colony PCR (which proved to be quite mind-numbing) and make a super gel of our latest PCR products. The rest of us spent the afternoon working on sponsorship and reading about chimeric restriction endonucleases.
Tuesday June 19
This morning we received our second shipment of BioBrick parts in the form of 31 agar stabs. David, Phillip, and Beini worked on inoculating all 27 of the plates we had left with our parts, while Victor and Andrew worked on making more LB plates. Dan and Kevin did more research on chimeric restriction enzymes. After lunch, we all sat down to put our brains together to evaluate the feasibility of making our own chimeric restriction enzymes. Seeing as our deliberation of BioBrick fusion standards has dragged on long enough, Kevin devised a way to decide which one we are ultimately going to use: everyone picks a standard, formulates a convincing argument for why we should use that standard, and then we show and tell! We only had time for David to present his case on the Silver fusion standard (RFC 23), which he did with no shortage of finesse, but we will continue this exciting debate tomorrow!
Wednesday June 20
This morning, we finished off the fusion standard mini-presentations. The final verdict? Tom Knight’s BB-2 standard (RFC 12) would be the most ideal for our purposes. It would be neat for us to test it out to see if it works, since it hasn’t been widely adopted by the synthetic biology community (yet). In the afternoon, Kevin and Maggie played around with Gromacs, a protein modelling software that Dr. Campbell recommended. In the lab, David, Victor, Faisal, and Beini had an assembly line going to prepare 31 liquid cultures. They also tried colony PCR using a different method lent to them by Jun (a Ph.D. student in the Chin-Sang lab).
Thursday June 21
In the morning, we searched the web for genomic DNA isolation protocols to use for our colony PCR. So far, we have not yet been successful in amplifying our fliC constant domain 1 gene, which is more than a little irksome. In the lab, we tried colony PCR with 5 ul of template DNA rather than our usual 1 ul. Faisal and Phillip spent ~2 hours laboriously labelling 31 Eppendorf tubes and spin columns in preparation for mini-preps galore. After lunch, we compiled a summary document of all the BioBricks we have ordered to date, to keep track of our progress for each part. We also discovered that the newspaper article about us got posted to the Kingston This Week website, featuring an excellent photo of Phillip, Andrew, and Kevin looking very happily down at a plate that spelled out QGEM in RFP bacteria. However, there was a typo with Andrew’s last name, spelled as “Paham” (now he can’t even show the article to his Phamily, har har har).
Friday June 22
Today, we ran a PCR and gel of five parts with similar annealing temperatures: mCerulean, mCitrine, and variations. We lowered the melting temperature and raised the number and length of cycles, hoping to increase the sensitivity of our products. These parameters seemed to work well, although we did get some unwanted products on the gel. While Faisal was making the Master Mix, he noticed a large particle floating around in our bottle of ddH2O, reminiscent of newt poo... time to get a fresh bottle. We made a few errors while we carried out the PCR, but always caught them in time... Friday labwork is the most fun! However, we are running out of Hot Start polymerase, so we can’t afford to make too many silly errors. Kevin had a meeting with Melissa Mahady Wilton from the Kingston School of Dance about our SynthetiQ project. She might be able to help us get an artist-in-residence, which would greatly facilitate the actual creation of our dances.
Retrieved from " http://2012.igem.org/Team:Queens_Canada/Notebook/Week8 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
