<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UBonn_HBRS_Description_Cloning skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UBonn HBRS/Description/Cloning</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><A href="//2016.igem.org/Team:UBonn_HBRS"><DIV id="logo" class="col-sm-2 col-xs-12"><DIV class="col-xs-4 visible-xs-block hidden-sm hidden-md hidden-lg"> </DIV></DIV><DIV id="title" class="col-sm-3 col-xs-12"> </DIV></A><NAV><UL id="mainNav"><LI><A href="//2016.igem.org/Team:UBonn_HBRS/Attributions">Attributions</A></LI><LI><A href="//2016.igem.org/Team:UBonn_HBRS/Attributions/Sponsors">Sponsors</A></LI></UL><LI><A href="//2016.igem.org/Team:UBonn_HBRS/HP/Silver">Human Practices</A></LI><LI><A href="//2016.igem.org/Team:UBonn_HBRS/Engagement">Education and Public Engagement</A></LI><LI><A href="//2016.igem.org/Team:UBonn_HBRS/Description">Description</A></LI><LI><A href="//2016.igem.org/Team:UBonn_HBRS/Description/B_Subtilis">B. Subtilis</A></LI><LI><A href="//2016.igem.org/Team:UBonn_HBRS/Description/Cloning">Cloning</A></LI><LI><A href="//2016.igem.org/Team:UBonn_HBRS/Description/Deinking">Deinking</A></LI><LI><A href="//2016.igem.org/Team:UBonn_HBRS/Description/Enzyme_Assays">Enzyme Assays</A></LI><LI><A href="//2016.igem.org/Team:UBonn_HBRS/Description/Notebook/2016">Lab Notebook</A></LI></NAV></DIV><DIV class="main container"><DIV class="main-content col-md-10 col-sm-12" role="main"><H1 class="blue-title">Cloning</H1><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#General_Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">General Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Genes_of_Interest"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Genes of Interest</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Cellulase_group"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Cellulase group</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Esterase-Laccase_group"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Esterase-Laccase group</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#General_Strategy"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">General Strategy</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#In_B._subtilis"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">In B. subtilis</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#In_E._coli"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">In E. coli</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Composite_.E2.80.9Cshuttle.E2.80.9D_plasmid_backbone"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Composite “shuttle” plasmid backbone</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Protocols_used"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Protocols used</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Ligation"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Transformation_of_E.Coli_cells"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Transformation of E.Coli cells</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#DNA_purification_.28.E2.80.9Cprep.E2.80.9D.29"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">DNA purification (“prep”)</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Gel_cleanup"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">Gel cleanup</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Dephosphorylation"><SPAN class="tocnumber">6.5</SPAN><SPAN class="toctext">Dephosphorylation</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="General_Abstract">General Abstract</SPAN></H2><P>The cloning aspect of this project is responsible for programming bacterial cells to secrete enzymes which can decouple ink particles from paper fibers.
</P><P>To this end, genes coding for enzymes were synthesized and ligated into a plasmid backbone, which is designed for use in both Escherichia coli and Bacillus subtilis.
</P><P>As expression of enzymes is intended for both <I>Bacillus subtilis</I> and <I>E.coli</I>, there are two separate tracks for cloning. For <I>B. subtilis</I>, the genes were assembled into pSBbs1AK1 along with secretory tags, and for <I>E.coli</I>, an inductive promoter (pLac) is placed upstream of the genes.
</P><H2><SPAN class="mw-headline" id="Genes_of_Interest">Genes of Interest</SPAN></H2><P>Within the scope of this project, genes of interest (GOIs) are those which catalyse either hydrolysis of paper fiber cellulose or ink particles.
</P><P>Synthesized at GenScript, each of these genes includes a sequence coding for His-tag, for ease of protein purification.
</P><H3><SPAN class="mw-headline" id="Cellulase_group">Cellulase group</SPAN></H3><P>Enzymes which break down cellulose molecules into monosaccharides, therefore partially disintegrate paper fibers and set ink particles free into the supernatant.
</P><DIV class="table-responsive panel panel-default with-caption"><TABLE class="table table-striped table-hover table-bordered"><THEAD><TR><TD>Gene name</TD><TD>Enzyme coded</TD><TD>Part number</TD><TD>Note</TD></TR></THEAD><TBODY><TR><TD>XynA</TD><TD>Xylanase</TD><TD>BBa_K2029002</TD><TD><A href="http://www.microbialcellfactories.com/content/11/1/66" target="_blank">http://www.microbialcellfactories.com/content/11/1/66</A></TD></TR><TR><TD>CelA</TD><TD>Cellulase</TD><TD>BBa_K2029004</TD><TD><A href="https://www.ncbi.nlm.nih.gov/pubmed/22101447" target="_blank">https://www.ncbi.nlm.nih.gov/pubmed/22101447</A></TD></TR><TR><TD>EngB</TD><TD>Endoglucanase</TD><TD>BBa_K2029005</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1785254/" target="_blank">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1785254/</A></TD></TR><TR><TD>BsCel5c</TD><TD>Cellulase</TD><TD>BBa_K2029006</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/21549854" target="_blank">http://www.ncbi.nlm.nih.gov/pubmed/21549854</A></TD></TR></TBODY></TABLE></DIV><DIV class="caption"><B>Table 1. Genes and Enzymes of Cellulase group.</B></DIV><H3><SPAN class="mw-headline" id="Esterase-Laccase_group">Esterase-Laccase group</SPAN></H3><P>Enzymes which hydrolyse esters or oxidize phenols and aromatic compounds found in commercial inks. Their ability to attack ink particles is not well studied.
</P><DIV class="table-responsive panel panel-default with-caption"><TABLE class="table table-striped table-hover table-bordered"><THEAD><TR><TD>Gene name</TD><TD>Enzyme coded</TD><TD>Part number</TD><TD>Note</TD></TR></THEAD><TBODY><TR><TD>EstC2</TD><TD>Esterase</TD><TD>BBa_K2029007</TD><TD><A href="https://www.ncbi.nlm.nih.gov/pubmed/21619698" target="_blank">https://www.ncbi.nlm.nih.gov/pubmed/21619698</A></TD></TR><TR><TD>LipA</TD><TD>Lipase</TD><TD>BBa_K2029008</TD><TD><A href="http://onlinelibrary.wiley.com/doi/10.1002/bit.10201/abstract" target="_blank">http://onlinelibrary.wiley.com/doi/10.1002/bit.10201/abstract</A></TD></TR><TR><TD>Bpul</TD><TD>Laccase</TD><TD>BBa_K2029009</TD><TD><A href="https://www.ncbi.nlm.nih.gov/nucleotide/388482909" target="_blank">https://www.ncbi.nlm.nih.gov/nucleotide/388482909</A></TD></TR></TBODY></TABLE></DIV><DIV class="caption"><B>Table 2. Genes and Enzymes of Esterase-Laccase group.</B></DIV><H2><SPAN class="mw-headline" id="General_Strategy">General Strategy</SPAN></H2><P>The primary target chassis organism is <I>B. subtilis</I>, for which a secretory tag system was designed. At first <I>E. coli</I> was only intended to be used as the chassis for cloning, but as the project went on, E.coli was decided on as the backup system in case gene expression in B.subtilis fails.
</P><H3><SPAN class="mw-headline" id="In_B._subtilis">In B. subtilis</SPAN></H3><P>As they express secretory proteins, GOIs require a signaling system in order to function in B.subtilis.
</P><DIV class="col-sm-offset-1 col-sm-10"><DIV class="img-100 borderless with-caption" style=""><DIV class="caption"><STRONG>Figure 1. Cloning strategy in B. subtilis</STRONG></DIV></DIV></DIV><P>
For this reason, two secretory tags are chosen to precede the genes: P34-sacB (Part: BBa_K2029001) and P34-nprb (BBa_K2029002). Both are constitutive in nature, which means they cannot be deactivated.
</P><H3><SPAN class="mw-headline" id="In_E._coli">In E. coli</SPAN></H3><P>E.coli is intended to be the “Plan B” host organism, in case protein expression in B.subtilis cannot be achieved.
</P><P>An inductive promoter – “pLac” – is coupled with a sequence coding for ribosome binding site (RBS). This whole sequence, so called pLac-RBS (Part: BBa_K2029014), was synthesized at Integrated DNA Technologies (IDT).
</P><DIV class="col-sm-offset-3 col-sm-6"><DIV class="img-100 borderless with-caption" style=""><DIV class="caption"><STRONG>Figure 2. Cloning strategy in E. coli</STRONG></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Composite_.E2.80.9Cshuttle.E2.80.9D_plasmid_backbone">Composite “shuttle” plasmid backbone</SPAN></H2><P>As the GOIs are intended to be put into both chassis – <I>E. coli</I> and <I>B. subtilis</I>, a so-called “shuttle” backbone was developed to ensure compatibility with both species. This plasmid backbone - now given the name pSBbs1AK1 in the iGEM part registry – contains two different origins of replication, as well as confers resistance to two different antibiotics — one for each organism: Kanamycin resistance for <I>B. subtilis</I> and Ampicillin resistance for <I>E. coli</I>.
</P><DIV class="col-sm-offset-2 col-sm-8"><DIV class="img-100 borderless with-caption" style=""><DIV class="caption"><STRONG>Figure 3. Gel electrophoresis of  pSBbs1AK1 with a mazF insert, showing linearisation of the plasmid when cut with the BioBrick sites E, P, X, S, as well as K and H, and two fragments in double digestions.</STRONG></DIV></DIV></DIV><P>
This backbone, however, has proven to be unsuitable for the gene assembly process due to its large size (appr. 5.5 kbp), hence all tag-gene constructs for B.subtilis and pLac-RBS-gene constructs for E.coli were assembled using existing iGEM backbones such as pSB1C3 and pSB4C5, and only transferred into pSBbs1AK1 in the final step.
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>A major limitation of this cloning project is the lack of success, up until this point, in either aforementioned tracks for <I>B.  subtilis</I> and <I>E. coli</I>.
</P><P>With the <I>B. subtilis</I> track, while a total of 3 tag-gene constructs (Parts: BBa_K2029011, BBa_K2029012, BBa_K2029013) have been obtained, none of which has been successfully cloned into <I>B. subtilis</I>.
</P><P>It is worth noting that all three constructs obtained contain the P34-nprb tag, and none with P34-SacB. Data showed there was little to no colony growth with pSB1C3 plasmids containing these tag-gene constructs. It is hypothesised that as these secretory tags are constitutive, protein expression overwhelms the <I>E. coli</I> cells used for assembly, and ultimately killed them. In a change of strategy, the tags were cloned into pSB4C5, a low-copy iGEM plasmid, which should reduce protein expression in theory. Results showed a noticeable improvement with P34-nprb, but no significant improvement in P34-SacB:
</P><DIV class="table-responsive panel panel-default with-caption"><TABLE class="table table-striped table-hover table-bordered"><THEAD><TR><TD>Sample</TD><TD>Plate 1</TD><TD>Plate 2</TD></TR></THEAD><TBODY><TR><TD>SacB + C3</TD><TD>0</TD><TD>0</TD></TR><TR><TD><STRONG>SacB + C3</STRONG></TD><TD><STRONG>4</STRONG></TD><TD><STRONG>4</STRONG></TD></TR><TR><TD>nprb + C3</TD><TD>0</TD><TD>0</TD></TR><TR><TD><STRONG>nprb + C5</STRONG></TD><TD><STRONG>700</STRONG></TD><TD><STRONG>300</STRONG></TD></TR><TR><TD>Controls</TD></TR><TR><TD>SacB</TD><TD>0</TD></TR><TR><TD>nprb</TD><TD>0</TD></TR><TR><TD>C3</TD><TD>0</TD></TR><TR><TD>C5</TD><TD>0</TD></TR></TBODY></TABLE></DIV><DIV class="caption"><B>Table 3. Data from an experiment showing growth of E.coli with P34-nprb and P34-SacB in pSB4C5 (“C5”), as opposed to in pSB1C3 (“C3”)</B></DIV><P>
With the <I>E. coli</I> track, ligation of the pLac-RBS construct with GOIs did not yield any colony until the very last week of work in the laboratory, so even though colonies have appeared, it has not been ascertained whether they hold the desired plasmids.
</P><P>Due to the lack of cloning success so far, no enzyme has been produced by bacterial cells, and the cloning sub-area has not been able to provide input to other aspects of the project, namely protein expression and purification in B.subtilis, deinking and enzyme assay.
</P><H2><SPAN class="mw-headline" id="Protocols_used">Protocols used</SPAN></H2><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><P>Masses of DNA needed are calculated to achieve as close as possible to an ideal ratio of 3:1 between molar amounts of insert and backbone. Volumes of DNA needed can then be calculated by dividing mass for concentration of sample. 
Volume of 10x T4 Ligase from New England BioLabs (NEB): 10% of total mixture 
Volume of Rapid 2x ligase buffer: 50% of total mixture 
Total volume reaches 5 μL. </P><H3><SPAN class="mw-headline" id="Transformation_of_E.Coli_cells">Transformation of E.Coli cells</SPAN></H3><P>5 μL of ligated DNA + 50 μL of chemocompetent E.coli, well mixed by resuspension with pipette 
Incubation on ice for 30-45 minutes 
Heat shock at 42°C for 42 seconds 
Incubation on ice for 5-10 minutes 
Addition of 250 μL of SOC medium 
Incubation at 37°C for 30-45 minutes 
Plating 100-150 μL of bacterial mixture on petri dish with correct antibiotics 
Incubation at 37°C overnight
</P><H3><SPAN class="mw-headline" id="DNA_purification_.28.E2.80.9Cprep.E2.80.9D.29">DNA purification (“prep”)</SPAN></H3><P>Macherey-Nagel’s NucleoSpin® Plasmid - Isolation of high- and low-copy plasmid DNA from <I>E. coli</I></P><H3><SPAN class="mw-headline" id="Gel_cleanup">Gel cleanup</SPAN></H3><P>Macherey-Nagel’s Gel Extraction
</P><H3><SPAN class="mw-headline" id="Dephosphorylation">Dephosphorylation</SPAN></H3><P>(to prevent re-ligation of backbones) 
Thermo Scientific Fast AP Thermosensitive Alkaline Phosphatase - Protocol for nucleic acid dephosphorylation
</P></DIV><DIV class="col-md-2 hidden-print hidden-sm hidden-xs" role="complementary"><NAV id="sideNav" class="affix-top "><STRONG>Contents</STRONG></NAV></DIV></DIV><FOOTER class="footer"><DIV class="container"><P class="text-muted">© iGEM Team Bonn 2016

				<A href="#" id="scrollTop" class="pull-right">
					Top
					</A></P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>