<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ESBS-Strasbourg_System_construction"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:ESBS-Strasbourg/System_construction">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ESBS-Strasbourg/System_construction&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/System_construction&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/System_construction&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ESBS-Strasbourg/System_construction" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ESBS-Strasbourg/System construction</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="header"><DIV><TABLE style="position:absolute; top:0; width:100%;" cellpadding="0"><TBODY><TR><TD><CENTER><DIV id="header" style="background-color:#FFFFFF;padding=0px;"><DIV align="center" id="nav"><UL><LI><A href="https://2008.igem.org/Team:ESBS-Strasbourg">home</A></LI><LI><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Team">team</A><DIV><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Team">Meet the Team</A><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Colaborations">CoLABorations</A><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Sponsors">Sponsors</A></DIV></LI><LI><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Project">project</A><DIV><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Project">Summary</A><SPAN><A>In Detail</A><DIV><A href="https://2008.igem.org/Team:ESBS-Strasbourg/System_construction">System</A><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Labeling"> Labeling</A><A href="https://2008.igem.org/Team:ESBS-Strasbourg/measurements">Measurements</A><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Results">Outcome</A></DIV></SPAN><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Modeling">Modeling</A><SPAN><A>Related</A><DIV><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Applications">Applications</A><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Project_idea_history">Brainstorming</A><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Essential_publications">Reading</A></DIV></SPAN></DIV></LI><LI><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Project">misc</A><DIV><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Links">Links</A><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Protocols">Protocols</A><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Paper">Tutorials + Papers</A></DIV></LI><LI><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Parts">parts</A><DIV><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Parts">Submitted Parts</A><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2008&amp;group=ESBS-Strasbourg">Sandbox</A></DIV></LI><LI><A href="https://2008.igem.org/Team:ESBS-Strasbourg/Notebook">notebook</A></LI></UL></DIV></DIV></CENTER></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="maincontent" style="margin-top:150px;"><H1><SPAN class="mw-headline" id="System_construction">System construction</SPAN></H1><H2><SPAN class="mw-headline" id="Goal">Goal</SPAN></H2><P>We intend to construct a cell-cycle dependent toggle switch in eucaryotic cells. This system can then be used as the basis for a binary cell division counter.
</P><H2><SPAN class="mw-headline" id="Organism">Organism</SPAN></H2><P><I>Saccharomyctes cerevisiae</I> is in respect to the cell cycle regulation the best understood model organism. This will give us the possibility to develop our project with high accuracy on a theoretical level.
Furthermore, generation times are in the right order to be able to express and also to degrade proteins within one generation time in a predictable manner. There is also the advantage of features like easy transformation and high rates of homologous recombination. In turn, the eucaryotic nature of yeast will allow an easy transfer of the system to more complex eucaryots as for example HeLa cells.
</P><H2><SPAN class="mw-headline" id="Counting">Counting</SPAN></H2><P>The mode of binary counting appeared more elegant to us than the continuous way of counting, because the former would need less reporter compounds. So we decided to implement a binary counting system. Thus, this system would allow us to count to a higher number because of the current limitations of reporter constructs. Secondly, the binary counting mechanism would allow us to construct each bit in a modular manner, hence all further bits could be easily attached through minor changes in this standard bit module, the cell cycle dependent toggle switch.
</P><H2><SPAN class="mw-headline" id="BioBrick_Standard">BioBrick Standard</SPAN></H2><P>In order to keep the interference between our system and the host metabolism to a minimum we are envisaging the use of yeast parts only where an interaction with the host is unavoidable. There are two unavoidable interfaces which are the dependence on a cell cycle specific signal as input and the use of the host transcriptional and translational machinery to ensure the expression of the system. For all other components we are planning to revert to the vast quantity of procaryotic parts from the Registry. Thus, our project will include various fusion proteins of mixed (eukaryotic, prokaryotic) nature.
By regarding each domain as a part on its own right a rapid construction of components (for example: TFs and Repressors) with different properties would be possible (see also <A href="https://2008.igem.org/Team:ESBS-Strasbourg/Essential_publications" class="external text" rel="nofollow">Dhar, PK; 2007</A>). However, it is not be possible to construct fusion proteins by using the BioBrick Standard developed by Knight (<A href="https://2008.igem.org/Team:ESBS-Strasbourg/Essential_publications" class="external text" rel="nofollow">Knight, T; 2003</A>) since the 8 base between two parts would lead to a frame shift. Therefore, we envisage the use of the standard developed by Silver (<A href="https://2008.igem.org/Team:ESBS-Strasbourg/Essential_publications" class="external text" rel="nofollow">Philips, IE; Silver, PA; 2007</A>) having only six base pairs between two parts. The transformation of existing parts into the Silver standard will therefore be an important part of our project.
In respect to the minimal interference between the system and the host one could even go further by introducing also the whole protein expression machinery from prokaryots. We termed such an approach the <A href="https://2008.igem.org/Team:ESBS-Strasbourg/Project_idea_history" class="external text" rel="nofollow">e-language</A>.
</P><H2><SPAN class="mw-headline" id="Architecture">Architecture</SPAN></H2><H3><SPAN class="mw-headline" id="System_level">System level</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:257px;"><DIV class="thumbcaption">System level</DIV></DIV></DIV><P>Technical spoken, our goal is to develop a system which, once activated through an external signal, gives us alternately a PoPS output, no output respectively (see figure System level). This system is equal to the first bit of the possible cell division counter, thus would allow us to count to one. According to the idea of synthetic biology, one can use this system by using the adequate PoPS signal without a further understanding of the system's functioning. Thus, the system is available whenever a cell cycle dependent switch is needed.
</P><H3><SPAN class="mw-headline" id="Device_level">Device level</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:257px;"><DIV class="thumbcaption">Device level</DIV></DIV></DIV><P>The figure on the right is showing our construct architecture on a device level. One can appreciate the functional units of our system. The two devices, odd device and even device, will be alternatively activated in successive generations. By coupling the PoPS outcome of the even device we will obtain the wished expression pattern. Additionally to these two devices, a starter device for the integration of the external start signal and a reporter device (not shown) will be necessary to get constructed.
</P><H3><SPAN class="mw-headline" id="Parts_level">Parts level</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:257px;"><DIV class="thumbcaption">Parts level</DIV></DIV></DIV><P>Both, the odd and the even device are constituted of the same set of three parts, which are in turn assembled parts, constituted from basic Biobrick parts. These are Repressor (R), Self-activator (sA) and Cross-activator (cA). 
The sA of each device ensures the expression of all parts of the device. R and cA in turn are both acting on the sA activator of the opposite device. As long as both parts are active (R and cA) the negative effect of R will overlay the positive effect of cA. An internal cell cycle dependent signal will have a negative effect on R, so that cA can induce the sA, thus the expression of the subsystem will be switched.
In order to realize the system, parts have to be assembled from basic Biobrick parts, which fulfil the requirements on each of these three levels. Special attention should be put to the following questions:</P><UL><LI>What cell cycle dependent signal can be used to inactivate R?</LI><LI>How can one ensure that the effect of R is stronger than those of cA?</LI></UL><P>The modeling has monitored that the system could theoretically work. Please take a closer look on the <A href="https://2008.igem.org/Team:ESBS-Strasbourg/Modeling" class="external text" rel="nofollow">Modeling</A> section if you want to know more.
</P><H2><SPAN class="mw-headline" id="Cell_cycle_dependent_Input">Cell cycle dependent Input</SPAN></H2><P>A cell cycle dependent input is required to initiate the switch of our system. Because of the good investigation of the cell cycle in yeast, various inputs which are whether absent or present throughout a time window of the cell cycle are available. This includes cyclins and CDKs (cyclin dependent kinase). When choosing such a cell cycle dependent factor, it is most important that the input is readable for the system in an effective manner. Cell cycle dependent degradation motifs as tags that target a protein to the Anaphase-Promoting Complex (APC) will allow us to authorise the degradation of proteins in a specific time frame.
</P><H2><SPAN class="mw-headline" id="Balance_of_R_and_cA">Balance of R and cA</SPAN></H2><P>The negative effect of the Repressor must overbalance the positive effect of Cross-activator on the self activator of the opposing device. Principally, one could imagine that either a difference in the concentration or in affinity for the promoter could fulfil these requirements. We decided to solve this problem by the use of promoters of different strength.
</P><H2><SPAN class="mw-headline" id="Parts">Parts</SPAN></H2><P>As outlined in the &quot;Architecture&quot; section our system is composed of 2 sets having 3 parts each. Following we will state the requirements for these parts and discuss possible choices.
</P><H3><SPAN class="mw-headline" id="Cross_Repressor">Cross Repressor</SPAN></H3><UL><LI>Requirements
</LI><LI>Negative effect on promoter of opposing construct.
</LI><LI>Degradation in during mitosis, when the APC is active.
</LI></UL><LI>Discussion:
</LI><P>We first of all thought about a pure physical blockade by using truncated transcription factors where only the binding domain will be kept, but soon came across with the TUP1 domain, a protein naturally used in budding yeast to repress the expression of genes. This protein is normally recruited by a set of transcription factors, but we found publications in which a direct fusion of the TUP1 region to a DNA binding domain had a repressive effect.
</P><H3><SPAN class="mw-headline" id="Cross-activator">Cross-activator</SPAN></H3><UL><LI>Requirements:
</LI><LI>Positive Effect on promoter of opposing construct.
</LI><LI>Needs to persist during APC activity, but must be degraded quite fast afterwards.
</LI><LI>Must be dominated by the Cross Repressor in its present
</LI></UL><LI>Discussion:
</LI><P>As we decided to use the same DNA-binding domain for both, the Cross-activator and the Cross-repressor, we needed to construct the Cross-activator in a similar fashion than the Cross-repressor. This is why we decided to use the VP16 activation domain, which is a domain recruiting Transcription Factors and leading finally to the initiation of Transcription, and fuse it to the same DNA-binding domain as the Cross-repressor.
</P><H3><SPAN class="mw-headline" id="Self-activator">Self-activator</SPAN></H3><UL><LI>Requirements:
</LI><LI>The Self-activator serves to keep the expression of the active construct at a high level, therefore preventing spontaneous switches. Furthermore we wanted that our system is controlled at every point of time.
</LI><LI>Degradation in APC window.
</LI></UL><LI>Discussion:
</LI><P>The Self-activator is exactly the same than the Cross-activator of the other system, with the difference that it is APC tagged.
</P><H2><SPAN class="mw-headline" id="Parts_Assembly">Parts Assembly</SPAN></H2>
Knowing the requirements for the individual parts we have to find the optimal assembly of the parts from the basic Biobricks. This process, explained in the <A href="https://2008.igem.org/Team:ESBS-Strasbourg/measurements" class="external text" rel="nofollow">Measurements</A> section will give pure system the final shape.</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:ESBS-Strasbourg/System_construction">http://2008.igem.org/Team:ESBS-Strasbourg/System_construction</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ESBS-Strasbourg/System_construction" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ESBS-Strasbourg/System_construction" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/System_construction&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ESBS-Strasbourg/System_construction&amp;oldid=99211" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>