<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TecMonterrey_ProjectDescription_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TecMonterrey/ProjectDescription.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TecMonterrey/ProjectDescription.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TecMonterrey/ProjectDescription.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TecMonterrey/ProjectDescription.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TecMonterrey/ProjectDescription.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TecMonterrey/ProjectDescription.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><H1>Modular, synthetic biology approach for the development of a bacterial cancer therapy in <EM>Escherichia coli.</EM></H1><P>By harnessing the inherent ability of facultative anaerobic bacteria to colonize and grow in tumoral environments, this project aims to prove the functionality of four different modules that would work together as a bacterial cancer therapy using <EM>Escherichia coli</EM> as chasis: Toxicity module, Secretion module, Localized induction module, and Internalization module. The expression of tumor specific therapeutic proteins, Apoptin and TRAIL, conforms the toxicity module. For these proteins to have their effect they need to be located in the extracellular matrix, therefore we are developing a module with a secretion function using hemolysin secretory mechanism. The hypoxic microenvironment present in tumors can be used for the localized induction module of tumor specific proteins, using the promoters HIP and nirB. Finally, Apoptin needs mechanisms to enter tumor cells’ cytoplasm. Proteins with this requirement could reach the cytoplasm when coupled with the internalization module, resulting in a fusion with the TAT peptide.</P><H2>Therapeutic Proteins Module</H2><P>The bacterial system expresses two therapeutic proteins and their fusions to a secretion sequence. The toxins used in this study were Apoptin and TRAIL, which are apoptosis-inducing proteins. The expression of these tumor specific therapeutic proteins conforms the toxicity module. </P><P>The contructs below are designed to express the therapeutic proteins by the use of IPTG.</P><P>Based in the Therapeutic proteins module we designed two new biobricks for the registry: </P><P><B><A href="“http://parts.igem.org/Part:BBa_K1166004”">BBa_K116604</A></B></P><P><B><A href="“http://parts.igem.org/Part:BBa_K1166005”"> BBa_K116605</A></B></P><H3>Apoptin</H3><P>Apoptin, also called VP3, is a protein from the Chicken Anemia Virus (CAV) known to cause p53-independent apoptosis in more than 70 human cancer cell lines including  hepatoma, osteosarcoma, melanoma, cholangiocarcinoma, colon carcinoma, lung cancer, breast cancer, prostate cancer, cervix cancer, gastric cancer while leaving normal cells unharmed. Apoptin contains a bipartite nuclear localization signal (NLS1: aa 82-88 and NLS2: aa 111-121) and a nuclear export signal (NES) (aa 97-105). In cancer cells, Apoptin localizes to the nucleus while in normal cells it's found in the cytoplasm; it is thought that the bipartite NLS is activated by phosphorylation at Threonine 108 which only happens in cancer cells. </P><P>Since Apoptin exerts its proapoptotic activity intracellularly, it is usually fused at the N-terminus with a cell-penetrating peptide such as the TAT peptide. </P><H3>TRAIL</H3><P>TRAIL (TNF Related Apoptosis Inducing Ligand) is known to cause apoptosis in various cancer cell lines with minimal cytotoxicity toward normal cells. TRAIL induces apoptosis by binding to two death receptor domains, TRAIL-R1(DR4) and TRAIL-R2(DR5). The binding of TRAIL triggers the trimerization of the death receptors that recruit and activate FADD a death domain-containing protein, FADD then recruits and activates caspase-8, leading to the formation of the death-inducing signaling complex (DISC).</P><P>Besides TRAIL-R1(DR4) and TRAIL-R2(DR5) TRAIL also binds to three antagonistic decoy receptors TRAIL-R3 (DcR1), TRAIL-R4 (DcR2) and osteoprotegerin, however, these receptors do not induce apoptosis but can protect cells from TRAIL-induced apoptosis.</P><H2>Secretion module</H2><P>In order for TRAIL and TAT-Apoptin to exert their anticancer activity, they must reach the extracellular matrix in the first place. This module uses the type I α-hemolysin secretion system of <EM>Escherichia coli</EM> to achieve that feat.</P><P>The construct designs for this module were:</P><P>The constructs above, each contain a Histidine6x tag, allowing purification and further detection by western blot using a mouse monoclonal anti-his antibody.</P><P>The first two fused with secretion sequence HlyA, required to be exported to the medium, while the other two do not contain it, in order to serve as negative controls.</P><P>The last two constructs contain a GFP, the first one tagged with HlyA and the second one left untagged. In order to be measured for fluorescence in both the interior of the cell, and in the media.</P><P>The last construct is required to be co-transformed with all of the constructs already mentioned in the secretion module, in order to provide the required secretion proteins.</P><P>Based in the secretion module we designed a new Device for the registry: </P><P><B><A href="“http://parts.igem.org/Part:BBa_K1166002”"> BBa_K1166002</A></B></P><P>This is a device that allows a protein to be secreted by means of the alpha-hemolysin secretion system in <EM>E. coli.</EM> It’s designed so that the only thing that you have to do is to assemble your protein part in the device via the biofusion standard (BBF RFC 23), (Cut the device with EcoRI and XbaI, cut your protein part with EcoRI and SpeI, mix &amp; ligate). This procedure will leave your part in frame with a signal peptide.</P><P>Note: It's important that your part doesn't contain a stop codon nor a terminator since your protein will be fused at the C-terminus.</P><H3>Detailed description</H3><P>In order for TRAIL and TAT-Apoptin to exert their anticancer activity they must reach the extracellular matrix in the first place. This module uses the type I alpha-hemolysin secretion system of <EM>E. coli</EM> to achieve that feat. </P><P>The mechanism of secretion is simple: In the cytosol, the substrate is recognized by means of a signal peptide and then translocated directly into the extracellular medium. The translocator consists of three proteins: HlyB, an ATP binding cassette; HlyD, a membrane fusion protein; and TolC, an outer membrane protein. Regarding the signal peptide, TAT-Apoptin and TRAIL were engineered to be fused C-terminally to the last 60 amino acids of HlyA (the natural substrate) since it has been shown that this C-terminal region of HlyA is both necessary and sufficient to direct secretion. While HlyB and HlyD are considered strain specific proteins, TolC is a component of multiple trans-membrane systems in many microorganisms, therefore we co-expressed only HlyB and HlyD.</P><H2>Internalization assay</H2><H3>Brief description</H3><P>Internalization of Apoptin into the cytosol of human cells was confirmed using a TAT-GFP as a proof of concept approach. </P><P>The internalization module was characterized through the use of TAT-GFP fusion protein purified using His-tag affinity columns. Internalization essay was performed on grown human cells (NIH-T3 and CACO-2) treated with 1-10 µg/mL TAT-GFP followed by a Western Blot analysis of the lysates and a fluorescence essay of the supernatant in a 96-well microplate. </P><P>The constructs required for this module were:</P><P>The first constructs produces a GFP tagged with both a histidine6x tag and TAT internalization peptide.</P><P>The second construct produces a GFP tagged with only a histidine6x tag serving as a negative control for the internalization assay.</P><P>Based in the internalization module we designed a new biobrick: </P><P><B><A href="“http://parts.igem.org/Part:BBa_K1166003”"> BBa_K1166003</A></B></P><P>The biobrick was designed in a way that the gene of interest is inserted in-frame at the 3’ end of TAT. This assures internalization of the protein of interest.</P><H3>Detailed description</H3><P>Internalization module relies on the ability of the HIV-1 trans-activator protein Tat transduction domain to deliver proteins across the cell membrane. Tat-fusion proteins internalization takes place in approximately 20 minutes and is dependent on the extracellular concentration of the Tat-fusion protein. (Vocero-Akbani et al., 1999).</P><P>In normal cells it stays in the cytoplasm where it has no significant effect, but in cancer cells it migrates into the nucleus where it triggers apoptosis. (Guelen, L. et al., 2004).</P><P><B>Localized expression</B></P><P>An important part of this Project is the localized expression of the therapeutic proteins. For more than sixty years it has been known that some bacteria grow preferably in hypoxic conditions, being <EM>Escherichia coli</EM> one of these organisms. Tumors are known to be regions with low oxygen concentrations (hypoxic). With these factors in favor, three different hypoxia promoters were characterized. </P><H2>HIP-1 promoter</H2><P>Based on the work of Mengesha, et al. (2006), a hypoxia-inducible promoter (HIP-1) was used to control the expression of green fluorescent protein (GFP). This promoter was originally obtained from a portion of <EM>Salmonella</EM> pepT promoter with the combination of two binding sites for the FNR regulon and a TATAAT sequence. </P><P>Experimental work demonstrated that protein production was succesfull under acute and chronic hypoxia, but not under normoxia. To the correct operation of the promoter and all the anoxic methabolic pathways, there is one important gene: the fumarate and nitrate reductase (FNR). It is a transcriptional factor that inactives with the interaction of its sensory domain with oxigen molecules and it is present in <EM>Salmonella</EM> and <EM>E. coli</EM>. </P><P>For this project it was designed a construct for the expresion of GFP under hipoxic conditions to be used in BL21 <EM>E. coli</EM> strains. This constrsuct contains the FNR promoter and CDS native form <EM>E. coli</EM>, followed by a double terminator, the HIP-1 promoter designed by Mengesha, et al. (2006) and all the sequences required for the GFP expression. This expression cassete was cloned into a pUC57 plasmid.</P><H2>Nir-B promoter</H2><P>According to Peakman, et al. (1990) the nirB gene is the required for the synthesis of the nitrite reductase apoprotein and cysG proteins required for the sirohaem synthesis.This protestetic group is fundamental in the anoxic methabolic pathways of <EM>E. coli</EM> and it is used by many enzymes to complete a six-electron.</P><P>Reduction of sulfur and nitrogen. Knowing that these genes are specifically activated in <EM>E. coli</EM> during anoxic growth, in this project it was used the nirB promoter to make a cassette of expression to measure the effect of hypoxic environment over the expression of GFP.</P><P>The cassette is composed by: the NirB promoter (BBa_K905000 sequence reported in iGEM) and all the sequences necessary to express GFP, all cloned in pUC57 vector and transformed in BL21 <EM>E. coli</EM> strain.</P><H2>Chimeric promoter</H2><P>According to the fundaments of the Hip-1 construct and the NirB promoter, it was designed a new cassette controlled by those parts over the expression of GFP:</P><P>The BioBricks produced by the Hipoxia promoter module were:</P><P><B><A href="“http://parts.igem.org/Part:BBa_K1166000”">BBa_K116600</A></B></P><P><B><A href="“http://parts.igem.org/Part:BBa_K1166001”">BBa_K116601</A></B></P><H2>References</H2><P>Mengesha, A., Dubois, L., Lambin, P., Landuyt, W., Chiu, R., Wouters, B., &amp; Theys, J. (2006). Development of a Flexible and Potent Hypoxia-Inducible Promoter for Tumor-Targeted Gene Expression in Attenuated Salmonella. <EM>Cancer Biology and Therapy</EM>, 1120-1128.</P><P>Peakman, T., Crouzet, J., Mayaux, J., Busby, S., Mohan, S., Harborne, N., . . . Cole, J. (1990). Nucleotide sequence, organisation and structural analysis of the products of genes in the nirB - cysG region of the Eschevichia coli K-12 chromosome. <EM>European Journal of Biochemistry Volume 191, Issue 2</EM>, 315-323.</P><P>Unden, G., &amp; Schirawski, J. (1997). The oxygen responsive transcriptional regulator FNR of <EM>Escherichia coli</EM>: The search for signals and reactions. <EM>Molecular Microbiology 25</EM>, 205-210.</P><P>Silhol, M., Tyagi, M., Giacca, M., Lebleu, B. &amp; Vivès, E. (2001). Different mechanisms for cellular internalization of the HIV-1 Tat-derived cell penetrating peptide and recombinant proteins fused to Tat. <EM>European journal of biochemistry</EM> / FEBS 269, 494–501.</P><P>Guelen, L. et al. (2004). TAT-apoptin is efficiently delivered and induces apoptosis in cancer cells. <EM>Oncogene 23</EM>, 1153–65.</P><P>Vocero-Akbani, A., Heyden, N., Lissy, N., Ratner, L. &amp; Dowdy, S. Killing. (1998). HIV-infected cells by transduction with an HIV protease-activated caspase-3 protein. <EM>Nature medicine 5</EM>, 29–33.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TecMonterrey/ProjectDescription.html">http://2013.igem.org/Team:TecMonterrey/ProjectDescription.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TecMonterrey/ProjectDescription.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TecMonterrey/ProjectDescription.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TecMonterrey/ProjectDescription.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TecMonterrey/ProjectDescription.html&amp;oldid=241698" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>