<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLouvain_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLouvain/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="site-header sticky-top py-1 navbar-center"><UL class="navbar-nav mr-auto container d-flex flex-column flex-md-row justify-content-between" id="list_menu"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
        Our project
      </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Context">Context</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Experiments">Experiments</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Improve">Perspectives</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
        In the lab
      </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Attributions">Attributions</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/InterLab">InterLab</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
        Parts
      </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Parts">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Basic_Part">Basic parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Composite_Part">Composite parts</A></DIV></LI><LI class="nav-item"><A class="nav-link smallcaps" href="https://2018.igem.org/Team:UCLouvain/Safety">Safety</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
        Human practices
      </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Human_Practices">Integrated Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Public_Engagement">Education &amp; Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
        Awards
      </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/awards/Best_therapeutic">Best therapeutic project</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Entrepreneurship">Entrepreneurship</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Public_Engagement">Public Engagement</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Human_Practices">Integrated Human Practices</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
        About us
      </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Team">Our Team</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/University">Our University</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Collaborations">iGEM Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Sponsor">Our Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:UCLouvain/Partners">Our Partners</A></DIV></LI><LI class="nav-item"><A class="nav-link smallcaps" href="https://igem.org/2018_Judging_Form?team=UCLouvain">Judging form</A></LI></UL></NAV></P><DIV style="margin-top: 2%"><P class="bordgauche" style="font-size:100px; color:#16A2B8;"> Results</P><H2>Reporter Plasmid Construction</H2><DIV class="container"><DIV class="row"><DIV class="col-12"><DIV style="text-align: center"><P class="legend" style="text-align:center;">Figure 1: Recipient bacteria carrying our reporter plasmid (AmpR and RFP)</P></DIV></DIV></DIV></DIV><H2>Killer Plasmid : design and construction</H2><DIV class="container"><DIV class="row"><DIV class="col-12"><H3>Design</H3>
          We received a mobilizable plasmid (pK18mob) carrying Kanamycin resistance gene and OriT. Our purpose in to combine its conjugative capabilities with the Crispr/Cas9 system, so the final killer plasmid will be able to conjugate and to cut
          a target gene (the RFP gene and Ampicillin resistance gene). So, from the pK18mob, we have inserted the Crispr/Cas9 system : the Cas9 gene, and a single guide RNA.
          We have design two single guide RNA, to target the RFP gene and the Ampicillin resistance gene by Crispr/Cas9 system. The sequence was design in Benchling, and carry : cut/ligation sites at extremities, promoter pJ23119, the guide RNA
          and the terminator. Three DAM site (GATC) was cleaned, to avoid methylation. This sequence was synthetized by IDT.

          <DIV style="text-align: center"><P class="legend" style="text-align:center;">Figure 2: Schematic representation of our system</P><H3>Construction Step 1: integration of Cas9 gene</H3>
            From the plasmid pCAS, we have cut the Cas9 gene (4469bp with its promoter and terminator) with NEB digestion enzyme, and we have inserted it by ligation (with T4 ligase) in the plasmid pK18mob (3793 bp, received from C.E.N). We verify
            the presence of Cas9 sequence insertion in our cloned bacteria by performing a Miniprep and an enzyme digestion to linearize the plasmid, and check its size.

            <DIV style="text-align: center"><P class="legend" style="text-align:center;">Figure 3: Linearizing digestion of the pK18mob vector with Cas9.</P></DIV>
            SL: SmartLadder, 1 - 2: empty vector, 3: vector with Cas9, 4: empty vector, 5: vector with Cas9, 6 - 7: empty vector, 8: uncut plasmid.

            <H3>Construction Step 2: Integration of the sgRNA</H3>

            After digestion of our sgRNA, the ligation was done with the plasmid pK18mob with the CAS9 sequence. The result was transformed in TOP10 strain. We verify the presence of our sgRNA insertion by performing PCR,
            and then, after Miniprep, by enzyme digestion for verification.
            <DIV style="text-align: center"><P class="legend" style="text-align:center;">Figure 4: PCR of the pK18mob vector with Cas9 with sgRNA.</P></DIV></DIV>
          SL: SmartLadder, 1 to 6: unpresent or empty vector, 7: vector with sgRNA, 8 - 9: unpresent or empty vector.

          <H3>Step 3: transformation of our plasmid in a conjugative strain
          </H3>
          We transformed our final killer plasmid in E. coli S17-1 strain. We used S17 electrocompetent cells.
          <UL><LI>We have now a conjugative strain (S17-1) carrying a killer plasmid, capable of conjugation and targeting the RFP gene, and a killer plasmid targeting the AmpR gene.</LI><LI>The inserted sequence of the Cas9 and of the sgRNA were sequenced by Macrogen, and are all correct.</LI><LI>To have a better biosafe system, we also have design the same killer plasmid, but in an other conjugative plasmid (pSW23T) which have de R6K Origin of Replication. (See Perspective / Improvement / Biosafety section)</LI></UL></DIV></DIV></DIV><H2>Efficiency of conjugation.</H2><DIV class="container"><DIV class="row"><DIV class="col-12"><H3>Test #1 conjugation efficiency: transfer of the mobilizable plasmid pK18mob</H3>
          To verify the efficiency of transfer of the mobilizable plasmid pK18mob, we conjugated the recipient strain DH5ɑ containing the plasmid pSB4A5_J04450 with the donor strain S17-1 containing the plasmid pK18mob (WT). The result of the
          conjugation was spread on LB/Xgal/IPTG box without selection pressure. The counting of strains is based on the expressed phenotype. Donors include lactose operon and are blue in appearance, recipient cells that have not conjugated
          express RFP. While the transfer of the plasmid pK18mob (WT) restores the lactose operon in DH5ɑ by transferring the intact LacZɑ gene. The simultaneous expression of the lactose operon and the rfp gene produces a purple phenotype with
          DH5ɑ bacteria that have received the plasmid that can be mobilized by conjugation.

          <DIV style="text-align: center"><P class="legend" style="text-align:center;">Figure 5: Differentiation of strains based on the expressed phenotype.</P></DIV>

          From left to right, &quot;red&quot; colony: DH5a with pSB4A5_J04450 (non-conjugated recipient), &quot;blue&quot; colony: S17-1 with pK18mob (donor) and &quot;purple&quot; colony: DH5a with pSB4A5_J04450 and pK18mob (recipient having conjugated).


          
          After 2 hours of conjugation on solid LB medium, the cells are collected and diluted in PBS 1X before being spread on LB/Xgal/IPTG plate per 100µl. Colony counts are carried out three times for each dilution.
          <DIV style="text-align: center"><P class="legend" style="text-align:center;">Table 1: Count after conjugation (transfer of mobilizable plasmid pK18mob).</P></DIV>

          The ratio of donor (D) to recipient (R) is 0.172, i. e. D:R ≈ 1:6. The effectiveness of the conjugation is calculated by the ratio between the number of conjugated recipients (R+) divided by the total number of recipients (R+ and R-),
          i.e. 61% effectiveness, of recipient bacteria having received the mobilizable plasmid.


        </DIV></DIV></DIV></DIV><H2>Efficiency of our Crispr/Cas9 system.</H2><DIV class="container"><DIV class="row"><DIV class="col-12"><H3>Test #1 of our Crispr/Cas9 efficiency </H3>

        To verify the efficiency of Crispr/Cas9 system of our plasmid, we have transformed the killer plasmid in a DH5ɑ strain carrying the RFP reporter plasmid (pSB4A5_J04450). If our CRISPR/Cas9 plasmid system work well, it will cut the RFP gene
        and the recipient bacteria (red and fluorescent) will appear white.

        <DIV style="text-align: center"><P class="legend" style="text-align:center;">Verification of our Crispr/Cas9 efficiency.
          </P></DIV>

        On the left : the recipient cells carrying the RFP reporter plasmid. On the right: the recipient cells carrying the RFP reporter plasmid and our Crispr/cas9 killer plasmid, targeting the RFP gene.

        <DIV style="text-align: center"><P class="legend" style="text-align:center;">Table 2: Count after transformation by pK18mob_CAS9 with or without sgRNA anti RFP.
          </P></DIV>

        The transformation of a donor bacterium by a plasmid containing Cas9 and its RNA anti RFP guide shows 100% suppression of the red phenotype in the transformed bacteria.
        <UL><LI>As we can see, all of our transformed bacteria appears in red. Our Crispr/Cas9 system seems to have an efficiency of 100%</LI></UL><H3>Biomolecular confirmation of our Crispr/Cas9 system</H3>

        An enzyme digestion (PstI) was performed on the same strains to check that the recipient plasmid was correctly cut after the transformation of our killer plasmid.

        <DIV style="text-align: center"><P class="legend" style="text-align:center;">Biomolecular confirmation of the effect of CRISPR/Cas9 on the reporter plasmid: miniprep on colony and linear digestion with PstI.</P></DIV>

        SL: SmartLadder, 1 to 3: killer plasmids: respectively, without sgRNA, with sgRNA anti-RFP and with sgRNA anti-TEM1, 4: reporter plasmid with TEM-1 and RFP in DH5ɑ, 5 to 7: transformation of DH5ɑ (with reporter plasmid) with the killer
        plasmid without sgRNA, 8 to 9: transformation of DH5ɑ (with reporter plasmid) with the killer plasmid with sgRNA anti TEM-1, 10 to 12: transformation of DH5ɑ (with reporter plasmid) with the killer plasmid with sgRNA anti RFP.

        <UL><LI>This plasmid digestion confirms that the reporter plasmid carrying the rfp and TEM-1 genes has been correctly cut by our Crispr/Cas9 system, both by the RNA guide targeting RFP and the one targeting the resistance gene.</LI></UL><H3>Tests of Conjugation
        </H3>
        Different test were performed to verify the conjugation and/or Crispr/Cas9 system efficiency.

        <H4>Conjugation “Cross” test</H4>

        the purpose of this test is the verify the conjugation and the efficiency of our system by crossing, in a agar plate, a recipient strain (carrying our RFP reporter plasmid) with both our control donor strain (S17-1 with our plasmid without
        sgRNA) and our donor strain (S17-1 with our killer plasmid with the sgRNA).

        <H5>Cross test 1 with RFP Killer plasmid
        </H5>

        Recipient strain: DH5ɑ strain (E. coli) with the RFP rapporter plasmid.
        Donor strain : S17-1 strain with our RFP killer plasmid.
        Control strain : S17-1 strain without our RFP sgRNA.

        <DIV style="text-align: center"><P class="legend" style="text-align:center;">Cross test 1.</P></DIV>

        A: Vertically, the recipient cell DH5ɑ (with RFP reporter plasmid pSB4A5_J04450). Horizontally, up: the control donor strain S17-1 (pK18mob_CAS9 without sgRNA), down: the donor strain S17-1 (pK18mob_CAS9 with sgRNA anti-RFP cutting
        reporter plasmid).

        <H5>Cross test 2 with AmpR killer plasmid
        </H5>

        Recipient cell: DS984 strain (E. coli CmR) with the RFP rapporter plasmid (AmpR).
        Donor strain : S17-1 strain with our Amp killer plasmid.
        Control strain : S17-1 strain without our AmpR sgRNA.

        <DIV style="text-align: center" class="container"><P class="legend" style="text-align:center;">Cross test 2. Vertically : the recipient cell (with RFP reporter plasmid). Horizontally : Up: the donor strain (cutting AMP gene) Down: the control donor strain (without cutting Amp gene)</P></DIV><H4>Conjugation “in plate” test</H4>

        Before conjugation test, our strain (donor, recipient and control) was precultured O/N from glycerol stock.
        The different strain used are:
        <UL><LI>Donor : S17-1 with our killer plasmid (targeting RFP or AmpR)
          </LI><LI>Control : S17-1 with our conjugative plasmid pK18mob, with Cas9 BUT without the sgRNA (no targeting)
          </LI><LI>Recipient : Dh5alpha, or DS984 (with CmR gene in chromosome) or TOP10 (with a TEM1 gene in chromosome). Each of the recipient strain have our reporter plasmid (with RFP or GFP)
          </LI></UL>
        The pre cultured strain were centrifuged, and resuspended in fresh LB, and keeped at 37°C for about one hour the regenerate the pili. Note: for first tests we used PBS to wash antibiotics of precultured strain, but we don’t use it anymore,
        to prevent bacteria from stress. Then, 5ul of each strain (donor, recipient and control) were spotted on a LB plate. For conjugation, 5ul of donor strain (and also control strain) were spotted over 5ul of the recipient strain. The plate
        was incubated at 37°C between 3 hours and 20 hours. After than, each spot (donor, recipîent, control, two conjugative) was resuspended on liquid LB and some dilution was done (from 10-2 to 10-6). Each dilution was put on LB plate with
        adequate antibiotics and putted at 37°C O/N, to check the result of conjugation and/or Crispr/Cas9 system.

        <DIV style="text-align: center"><P class="legend" style="text-align:center;">Classic conjugation assay used for our test</P></DIV><H4>Conjugation in plate test 1 (non quantitative) targeting AmpR (TEM1)
        </H4><UL><LI>Donor (D) : S17-1 with our killer plasmid (sgRNA targeting AmpR, resistance: KanR)
          </LI><LI>Control (CTRL) : S17-1 with our conjugative plasmid pK18mob, with Cas9 BUT without the sgRNA (no targeting). KanR
          </LI><LI>Recipient (R) : TOP10 (with a TEM1 gene in chromosome)
          </LI><LI>Conjugation time : 3 hours 30 min
          </LI><LI>D.O. (600 nm) = about 1 (for recipient strain), about 0,5 (for donor/control strain)
          </LI></UL><DIV style="text-align: center"><P class="legend" style="text-align:center;">Conjugation in plate test 1: On left : Donor and recipient strain after conjugation. On right : control and recipient strain after conjugation.
          </P></DIV><UL><LI> First result seems to show that conjugation append (at least in control strain) and that donor strain (if have conjugated) seems to kill the AmpR gene as expected. This first test is non quantitative, so we don’t know exactly how
            much bacteria we have, and more test are needed.</LI></UL><H4>Efficiency of Crispr/Cas9 system by conjugation.
        </H4>

        To verify the effectiveness of our system using Cas9 carried by a mobilizable plasmid, we contacted the number of events or a recipient cell (TOP10 PyrF::TEM-1) lost its red phenotype when losing its plasmid pSB4K5_bbaJ04450 after
        conjugation with a donor cell (S17-1) containing the mobilizable plasmid containing Cas9 and an anti-RFP RNA guide. A negative control was performed using a donor strain that did not include guide RNA. Donor cells were counted after
        dilution on LB kanamycin and recipients on LB ampicillin. Colony counts are carried out three times for each dilution. <B>The effectiveness of our system using Cas9 carried by a mobilizable plasmid give 27% for a D:R ratio 1:169.</B><DIV style="text-align: center"><P class="legend" style="text-align:center;">Table 3: Averages of the counts before and after conjugation and transfer of the mobilizable plasmid pK18mob_CAS9.
            Averages achieved over three spreads per sample. The interval is given by twice the standard deviation.
          </P></DIV><H4>Conjugation in fluorescence microscopy
        </H4>


        To visualize at the scale of a cell, the effect of the transfer of a mobilizable plasmid, containing Cas9 and its RNA anti RFP guide, by conjugation: we performed a 60-minute time lapse on a mixture containing the donor strain (S17-1 with
        pK18mob_CAS9_sgRNA anti RFP and pSB4C5_R0010_E0840 (GFP)) and the recipient strain (DH5a with pSB4A5_J04450 (RFP)).

        </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>