<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IIT-Madras_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IIT-Madras/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Collaborations Page</TITLE></P><DIV class="topnav" id="myTopnav"><DIV class="dropdown block"><DIV class="dropbtn">Project
    	  	</DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:IIT-Madras/Description">Description</A><A href="https://2018.igem.org/Team:IIT-Madras/Design">Design</A><A href="https://2018.igem.org/Team:IIT-Madras/Notebook">Notebook</A><A href="https://2018.igem.org/Team:IIT-Madras/Results">Results</A><A href="https://2018.igem.org/Team:IIT-Madras/Demonstrations">Demonstrations</A><A href="https://2018.igem.org/Team:IIT-Madras/Improvements">Improvements</A><A href="https://2018.igem.org/Team:IIT-Madras/InterLab">InterLab</A><A href="https://2018.igem.org/Team:IIT-Madras/Safety">Safety</A></DIV></DIV><A href="https://2018.igem.org/Team:IIT-Madras/Parts" class="block">Parts</A><A href="https://2018.igem.org/Team:IIT-Madras/Software" class="block">Software</A><DIV class="dropdown block"><DIV class="dropbtn">Human Practices
    	  	</DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:IIT-Madras/Human_Practices">Overview</A><A href="https://2018.igem.org/Team:IIT-Madras/Silver">Silver Human Practices</A><A href="https://2018.igem.org/Team:IIT-Madras/IntegratedHP">Integrated Human Practices</A></DIV></DIV><DIV class="dropdown block"><DIV class="dropbtn">Public Engagement
    	  	</DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:IIT-Madras/Public_Engagement">Overview</A><A href="https://2018.igem.org/Team:IIT-Madras/TheLanguageProject">The Language Project</A><A href="https://2018.igem.org/Team:IIT-Madras/SchoolVisits">School Visits</A><A href="https://2018.igem.org/Team:IIT-Madras/Synkranti">Research Magazine</A><A href="https://2018.igem.org/Team:IIT-Madras/Workshops">Workshop</A><A href="https://2018.igem.org/Team:IIT-Madras/SynBioJournalClub">SynBio Journal Club</A></DIV></DIV><DIV class="dropdown block"><DIV class="dropbtn">The Team
    	  	</DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:IIT-Madras/Team">The Team</A><A href="https://2018.igem.org/Team:IIT-Madras/Attributions">Attributions</A><A href="https://2018.igem.org/Team:IIT-Madras/Collaborations">Collaborations</A></DIV></DIV></DIV><DIV id="mySidenav" class="sidenav"><A href="javascript:void(0)" class="closebtn" onclick="closeNav()" style="margin-top: 4mm;">×</A><A href="https://2018.igem.org/Team:IIT-Madras">Home</A><A href="#" class="collapsible">Project</A><DIV class="coll_content"><A href="https://2018.igem.org/Team:IIT-Madras/Description">Description</A><A href="https://2018.igem.org/Team:IIT-Madras/Design">Design</A><A href="https://2018.igem.org/Team:IIT-Madras/Notebook">Notebook</A><A href="https://2018.igem.org/Team:IIT-Madras/Results">Results</A><A href="https://2018.igem.org/Team:IIT-Madras/Demonstrations">Demonstrations</A><A href="https://2018.igem.org/Team:IIT-Madras/Improvements">Improvements</A><A href="https://2018.igem.org/Team:IIT-Madras/InterLab">InterLab</A><A href="https://2018.igem.org/Team:IIT-Madras/Safety">Safety</A></DIV><A href="https://2018.igem.org/Team:IIT-Madras/Parts">Parts</A><A href="https://2018.igem.org/Team:IIT-Madras/Software">Software</A><A href="#" class="collapsible">Human Practices</A><DIV class="coll_content"><A href="https://2018.igem.org/Team:IIT-Madras/Human_Practices">Overview</A><A href="https://2018.igem.org/Team:IIT-Madras/Silver">Silver Human Practices</A><A href="" https:="">Integrated Human Practices</A></DIV><A href="#" class="collapsible">Public Engagement</A><DIV class="coll_content"><A href="https://2018.igem.org/Team:IIT-Madras/Public_Engagement">Overview</A><A href="https://2018.igem.org/Team:IIT-Madras/TheLanguageProject">Language Project</A><A href="https://2018.igem.org/Team:IIT-Madras/SchoolVisits">School Visits</A><A href="https://2018.igem.org/Team:IIT-Madras/Synkranti">Research Magazine</A><A href="https://2018.igem.org/Team:IIT-Madras/Workshops">Workshop</A><A href="https://2018.igem.org/Team:IIT-Madras/SynBioJournalClub">SynBio Journal Club</A></DIV><A href="#" class="collapsible">The Team</A><DIV class="coll_content"><A href="https://2018.igem.org/Team:IIT-Madras/Team">The Team</A><A href="https://2018.igem.org/Team:IIT-Madras/Attributions">Attributions</A><A href="https://2018.igem.org/Team:IIT-Madras/Collaborations">Collaborations</A></DIV></DIV><SPAN style="font-size:30px;cursor:pointer; margin-left: 4mm;" onclick="openNav()" class="nodesk">☰</SPAN><P><TITLE> Team: IIT-Madras/Results</TITLE></P><H1 style="font-family: 'title', sans-serif; font-size: 12mm; padding-left: 20%; color: #00b355;">Protocols</H1><CENTER><DIV class="box"><BUTTON class="collapsible2 font">Qiagen Plasmid Mini Kit</BUTTON><DIV class="content2"><OL type="I" style="font-size: 5.5mm; text-align: justify; " align="LEFT"><LI>Pick a single colony from a freshly streaked selective plate and inoculate a starter culture of 2–5 ml LB medium containing the appropriate selective antibiotic. Incubate for approximately 8 h at 37°C with vigorous shaking (approx. 300 rpm). Use a tube or flask with a volume of at least 4 times the volume of the culture. </LI><LI>Dilute the starter culture 1/500 to 1/1000 into 3 ml selective LB medium. Grow at 37°C for 12–16 h with vigorous shaking (approx. 300 rpm). Use a flask or vessel with a volume of at least 4 times the volume of the culture. The culture should reach a cell density of approximately 3–4 x 109 cells per milliliter, which typically corresponds to a pellet wet weight of approximately 3 g/liter medium.</LI><LI>Harvest the bacterial cells by centrifugation at 6000 x g for 15 min at 4°C. If you wish to stop the protocol and continue later, freeze the cell pellets at –20°C. </LI><LI> Resuspend the bacterial pellet in 0.3 ml of Buffer P1. Ensure that RNase A has been added to Buffer P1. If LyseBlue reagent has been added to Buffer P1, vigorously shake the buffer bottle before use to ensure LyseBlue particles are completely resuspended. The bacteria should be resuspended completely by vortexing or pipetting up and down until no cell clumps remain. </LI><LI>Add 0.3 ml of Buffer P2, mix thoroughly by vigorously inverting the sealed tube 4–6 times, and incubate at room temperature (15–25°C) for 5 min. Do not vortex, as this will result in shearing of genomic DNA. The lysate should appear viscous. Do not allow the lysis reaction to proceed for more than 5 min. After use, the bottle containing Buffer P2 should be closed immediately to avoid acidification from CO2 in the air. If LyseBlue has been added to Buffer P1, the cell suspension will turn blue after addition of Buffer P2. Mixing should result in a homogeneously colored suspension. If the suspension contains localized colorless regions or if brownish cell clumps are still visible, continue mixing the solution until a homogeneously colored suspension is achieved. 
</LI><LI>Add 0.3 ml of chilled Buffer P3, mix immediately and thoroughly by vigorously inverting 4–6 times, and incubate on ice for 5 min. Precipitation is enhanced by using chilled Buffer P3 and incubating on ice. After addition of Buffer P3, a fluffy white material forms and the lysate becomes less viscous. The precipitated material contains genomic DNA, proteins, cell debris, and KDS. The lysate should be mixed thoroughly to ensure even potassium dodecyl sulphate precipitation. If the mixture still appears viscous, more mixing is required to completely neutralize the solution. If LyseBlue reagent has been used, the suspension should be mixed until all trace of blue has gone and the suspension is colorless. A homogeneous colorless suspension indicates that the SDS has been effectively precipitated.</LI><LI>Centrifuge at maximum speed in a microcentrifuge for 10 min. Remove supernatant containing plasmid DNA promptly. Before loading the centrifuge, the sample should be mixed again. Centrifugation should be performed at maximum speed in 1.5 ml or 2 ml microcentrifuge tubes (e.g., 10,000–13,000 rpm in a microcentrifuge). Maximum speed corresponds to 14,000–18,000 x g for most microcentrifuges. After centrifugation, the supernatant should be clear. If the supernatant is not clear, a second,shorter centrifugation should be carried out to avoid applying any suspended or particulate material to the column. Suspended material (which causes the sample to appear turbid) will clog the column and reduce or eliminate flow. Optional: Remove a 50 µl sample from the cleared lysate and save it for an analytical gel (sample 1). </LI><LI>Equilibrate a QIAGEN-tip 20 by applying 1 ml Buffer QBT, and allow the column to empty by gravity flow. Place QIAGEN-tips into a QIArack over the waste tray or use the tip holders provided with each kit (see “Setup of QIAGEN-tips” page 13). Flow of buffer will begin automatically by reduction in surface tension due to the presence of detergent in the equilibration buffer. Allow the QIAGEN-tip to drain completely. QIAGEN-tips can be left unattended, since the flow of buffer will stop when the meniscus reaches the upper frit in the column. </LI><LI>Apply the supernatant from step 7 to the QIAGEN-tip 20 and allow it to enter the resin by gravity flow. The supernatant should be loaded onto the QIAGEN-tip promptly. If it is left too long and becomes cloudy due to further precipitation of protein, it must be centrifuged again before loading to prevent clogging of the QIAGEN-tip. Optional: Remove a 50 µl sample of the flow-through and save for an analytical gel (sample 2). </LI><LI>Wash the QIAGEN-tip 20 with 2 x 2 ml Buffer QC. Allow Buffer QC to move through the QIAGEN-tip by gravity flow. Optional: Remove a 220 µl sample of the combined wash fractions and save for an analytical gel (sample 3). </LI><LI>Elute DNA with 0.8 ml Buffer QF. Collect the eluate in a 1.5 ml or 2 ml microcentrifuge tubes (not supplied). Note: For constructs larger than 45–50 kb, prewarming the elution buffer to 65°C may help to increase yield. Optional: Remove a 45 µl sample of the eluate and save for an analytical gel (sample 4).</LI><LI>Precipitate DNA by adding 0.7 volumes (0.56 ml per 0.8 ml of elution volume) of room-temperature isopropanol to the eluted DNA. Mix and centrifuge immediately at ≥15,000 x g rpm for 30 min in a microcentrifuge. Carefully decant the supernatant. All solutions should be at room temperature to minimize salt precipitation. Isopropanol pellets have a glassy appearance and may be more difficult to see than the fluffy, salt-containing pellets that result from ethanol precipitation. Marking the outside of the tube before centrifugation allows the pellet to be easily located. Isopropanol pellets are also more loosely attached to the side of the tube, and care should be taken when removing the supernatant. </LI><LI>Wash DNA pellet with 1 ml of 70% ethanol and centrifuge at 15,000 x g for 10 min. Carefully decant the supernatant without disturbing the pellet. The 70% ethanol removes precipitated salt and replaces isopropanol with the more volatile ethanol, making the DNA easier to redissolve. </LI><LI>Air-dry the pellet for 5–10 min, and redissolve the DNA in a suitable volume of buffer (e.g., TE buffer, pH 8.0, or 10mM Tris·Cl, pH 8.5) Redissolve the DNA pellet by rinsing the walls to recover the DNA. Pipetting the DNA up and down to promote resuspension may cause shearing and should be avoided. Overdrying the pellet will make the DNA difficult to redissolve. DNA dissolves best under slightly alkaline conditions; it does not easily dissolve in acidic buffers.</LI></OL><P style="font-family: 'title', sans-serif; font-size: 5.5mm;">Protocol Source: Qiagen plasmid isolation kit</P></DIV><BUTTON class="collapsible2 font">Transformation of <EM>Acinetobacter baylyi</EM> ADP1</BUTTON><DIV class="content2"><OL type="I" style="font-size: 5.5mm; text-align: justify; " align="LEFT"><LI>Grow a fresh culture of AB overnight in rich media. Generally this is done by inoculating a small amount of frozen glycerol stock or previous culture into 10mL of LB in a sterile 50mL flask and allowing it to shake in a 30C incubator at 140RPM overnight. Alternatively, inoculate into 5mL of LB in a sterile test tube and shake at 30C and 200RPM.</LI><LI>In a sterile test tube, combine 1mL fresh LB, &gt;100ng of transforming DNA, and 70uL of overnight grown ADP1 culture.</LI><LI>Incubate the test tube in a shaking incubator at least 3 hours, ideally overnight.</LI><LI>Plate dilutions of the transformed culture onto selective and non-selective media plates using sterile glass roller beads. Incubate the plates at 30C overnight.</LI><LI>Pick colonies that grow on selective media and use PCR to assay for a successful transformation. Growth on non-selective but not selective plating suggests a failed transformation.</LI></OL><P style="font-family: 'title', sans-serif; font-size: 5.5mm;">Protocol Source: Barrick’s Lab, UT Austin</P></DIV><BUTTON class="collapsible2 font">Gel Electrophoresis</BUTTON><DIV class="content2"><OL type="I" style="font-size: 5.5mm; text-align: justify; " align="LEFT"><LI>Produce DNA agarose gel
Add proper amount of agarose in TAE/TBE buffer (we usually produce 0.5,
0.75, 1.5% gel) and use a microwave oven to make the agarose completely
dissolve in buffer (transparent).
 Let agarose solution cool down to about 50°C (about when you can
comfortably keep your hand on the flask), about 5 mins, and pour the agarose
into a gel tray with the well comb in place. Let sit at room temperature for 20-30 mins, until it has completely solidified.
</LI><LI>Add TAE/TBE buffer into gel electrophoresis tank</LI><LI>Add EtBr: 800~1000 μl.  Mix well.</LI><LI>Add dye into sample.
 Since the dye is 6x, so sample amount divide by 6.
 Prepare tape, mix sample with dye.</LI><LI>Prepare marker：5 μl.</LI><LI>Electrophoresis
 First well load marker
 Voltage 125V, 20 min
 Marker run to 3rd last row.</LI><LI>Photo-taking.
 Dry the gel before photo-taking.
 use UV light to observe the results and do photo-taking.</LI></OL><P style="font-family: 'title', sans-serif; font-size: 5.5mm;">Source: iGEM NCHU Taiwan</P></DIV><BUTTON class="collapsible2 font">Single Restriction Digestion</BUTTON><DIV class="content2"><OL type="I" style="font-size: 5.5mm; text-align: justify; " align="LEFT"><LI> Set up reaction as follows:
<TABLE><TBODY><TR><TH>Component</TH><TH>50 μl Reaction</TH></TR><TR><TH>DNA</TH><TH>1 μg </TH></TR><TR><TH>10X NEBuffer EcoRI </TH><TH>5 μl (1X)</TH></TR><TR><TH>EcoRI</TH><TH>1.0 μl (or 10 units)</TH></TR><TR><TH>Nuclease-free Water</TH><TH>to 50 μl</TH></TR></TBODY></TABLE></LI><LI> Incubate at 37°C for 5-15 minutes as EcoRI is Time-Saver qualified.</LI></OL></DIV><BUTTON class="collapsible2 font">Double Restriction Digestion</BUTTON><DIV class="content2"><OL type="I" style="font-size: 5.5mm; text-align: justify; " align="LEFT"><LI> Set up reaction as follows: 
<TABLE><TBODY><TR><TH>COMPONENT</TH><TH>50 μl Reaction</TH></TR><TR><TH>DNA</TH><TH>1 μg </TH></TR><TR><TH>10X NEBuffer EcoRI </TH><TH>7.5 μl (1X)</TH></TR><TR><TH>EcoRI</TH><TH>1.0 μl (or 10 units)</TH></TR><TR><TH>Pstl</TH><TH>1.0 μl (or 10 units)</TH></TR><TR><TH>Nuclease-free Water</TH><TH>to 50 μl</TH></TR></TBODY></TABLE></LI><LI>2. Incubate at 37°C for 5-15 minutes as both enzymes are Time-Saver qualified.</LI></OL></DIV><BUTTON class="collapsible2 font">Ligation Protocol</BUTTON><DIV class="content2"><OL type="I" style="font-size: 5.5mm; text-align: justify; " align="LEFT"><LI>Set up the following reaction in a microcentrifuge tube on ice.</LI><LI>(T4 DNA Ligase should be added last. Note that the table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.) Use NEBioCalculator to calculate molar ratios.
<TABLE><TBODY><TR><TH>COMPONENT</TH><TH>20 μl REACTION</TH></TR><TR><TH>T4 DNA Ligase Buffer (10X)*</TH><TH>2 μl</TH></TR><TR><TH>Vector DNA (4 kb)</TH><TH>50 ng (0.020 pmol)</TH></TR><TR><TH>Insert DNA (1 kb)</TH><TH>37.5 ng (0.060 pmol)</TH></TR><TR><TH>Nuclease-free water</TH><TH>to 20 μl</TH></TR><TR><TH>T4 DNA Ligase</TH><TH>1 μl</TH></TR></TBODY></TABLE></LI>
* The T4 DNA Ligase Buffer should be thawed and resuspended at room temperature.
<LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</LI><LI> For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours (alternatively, high concentration T4 DNA Ligase can be used in a 10 minute ligation).</LI><LI>Heat inactivate at 65°C for 10 minutes.</LI><LI>Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells.</LI><BUTTON class="collapsible2 font">Gel Elution using QIAquick Gel Extraction Kit</BUTTON><DIV class="content2"><OL type="I" style="font-size: 5.5mm; text-align: justify; " align="LEFT"><LI> Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose. </LI><LI>Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg, or approximately 100 μl). For example, add 300 μl of Buffer QG to each 100 mg of gel. For &gt;2% agarose gels, add 6 volumes of Buffer QG. The maximum amount of gel slice per QIAquick column is 400 mg; for gel slices &gt;400 mg, use more than one QIAquick column.
</LI><LI>Incubate at 50°C for 10 min or until the gel slice has completely dissolved. To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation. IMPORTANT: Solubilize agarose completely. For &gt;2% gels, increase incubation time.</LI><LI>After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn to yellow. The adsorption of DNA to the QIAquick membrane is efficient only at pH ≤7.5. Buffer QG contains a pH indicator that is yellow at pH ≤7.5 and orange or violet at higher pH, allowing easy determination of the optimal pH for DNA binding. </LI><LI>Add 1 gel volume of isopropanol to the sample and mix. For example, if the agarose gel slice is 100 mg, add 100 μl isopropanol. This step increases the yield of DNA fragments ≤500 bp and ≥4 kb. For DNA fragments between 500 bp and 4 kb, addition of isopropanol has no effect on yield. Do not centrifuge the sample at this stage. </LI><LI>Place a QIAquick spin column in a provided 2 ml collection tube. 
</LI><LI>To bind DNA, apply the sample to the QIAquick column, and then centrifuge for 1 min. The maximum volume of the column reservoir is 800 μl. For sample volumes &gt;800 μl, simply load the remainder and spin again. </LI><LI>Discard flow-through and place QIAquick column back into the same collection tube. Collection tubes are reused to reduce plastic waste. </LI><LI>Recommended: Add 0.5 ml of Buffer QG to QIAquick column and centrifuge for 1 min. This step will remove all traces of agarose. This is only required if the DNA will be used for direct sequencing, in vitro transcription or microinjection. </LI><LI>To wash, add 0.75 ml of Buffer PE into the QIAquick column and centrifuge for 1 min. Note: If the DNA will be used for salt-sensitive applications such as blunt-end ligation and direct sequencing, let the column stand 2–5 min after addition of Buffer PE before centrifuging.</LI><LI>Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at 7,900 x g (13,000 rpm). IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation step. </LI><LI>Place QIAquick column into a clean 1.5 ml microcentrifuge tube. </LI><LI>To elute DNA, add 50 μl of Buffer EB (10 mM Tris·Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane, and then centrifuge the column for 1 min. Alternatively, for increased DNA concentration, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min. After the addition of Buffer EB to the QIAquick membrane, increasing the incubation time to up to 4 min can increase the yield of purified DNA. IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volumes are 48 μl from 50 μl elution buffer volume and 28 μl from 30 μl. Elution efficiency is dependent on pH. The maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C because DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE buffer (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions.</LI><LI>If the purified DNA is to be analyzed on a gel, add 1 volume Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting it up and down before loading the gel. Loading Dye contains 3 marker dyes – bromophenol blue, xylene cyanol and orange G – that facilitate the estimation of DNA-migration distance and the optimization of the agarose gel run time. Identify the dyes according to migration distance and agarose gel percentage and type.</LI></OL></DIV><BUTTON class="collapsible2 font">PCR Purification using the QIAquick PCR Purification Kit</BUTTON><DIV class="content2"><OL type="I" style="font-size: 5.5mm; text-align: justify; " align="LEFT"><LI>Add 5 volumes of Buffer PB to 1 volume of the PCR sample, and then mix. It is not necessary to remove mineral oil or kerosene. For example, add 500 μl of Buffer PB to 100 μl PCR sample (not including oil). </LI><LI>If pH indicator I has been added to Buffer PB, check that the mixture’s color is yellow. If the color of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn yellow.</LI><LI>Place a QIAquick spin column in a provided 2 ml collection tube. </LI><LI>To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s. </LI><LI>Discard flow-through. Place the QIAquick column back into the same tube. Collection tubes are reused to reduce plastic waste. </LI><LI>To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s. </LI><LI>Discard flow-through and place the QIAquick column back into the same tube. Centrifuge the column for an additional 1 min. IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation. </LI><LI>Place QIAquick column in a clean 1.5 ml microcentrifuge tube.</LI><LI>To elute DNA, add 50 μl Buffer EB (10 mM Tris·Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min. Alternatively, for increased DNA concentration, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge. IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volumes are 48 μl from 50 μl elution buffer volume and 28 μl from 30 μl elution buffer. Elution efficiency is dependent on pH. Maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C because DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE buffer (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions. </LI><LI>If the purified DNA is to be analyzed on a gel, add 1 volume Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting it up and down before loading the gel. Loading Dye contains 3 marker dyes – bromophenol blue, xylene cyanol and orange G – that facilitate estimation of DNA-migration distance and optimization of the agarose gel run time. Identify the dyes according to migration distance and agarose gel percentage and type.</LI></OL></DIV><BUTTON class="collapsible2 font">Colony PCR using the Takara Bio EmeraldAmp GT PCR MasterMix</BUTTON><DIV class="content2"><P style="font-family: 'title', sans-serif; font-size: 5.5mm;">

General PCR Mixture (Total 50 μl） : 
EmeraldAmp GT PCR Master Mix (2X Premix) 25 μl 
Template &lt; 500 ng 
Forward Primer 0.2 μM (final conc.) 
Reverse Primer 0.2 μM (final conc.) 
dH2O (Sterile distilled water) up to 50 μl<B>Suggested PCR Conditions : </B>

3 Step (up to 6 kb) 
98℃ 10 sec. 
60℃ 30 sec.
72℃ 1 min./kb
for 30 cycles

2 Step (over 6 kb) 
98℃ 10 sec. 
68℃ 1 min./kb 
for 30 cycles

For optimal results, primers should have a Tm &gt;60℃ . The following formula is commonly used for estimating the Tm of the primers. 
Tm (℃ ) = 2(NA+NT) + 4(NG+NC) -5 
N : the number of adenine (A), thymidine (T), guanidine (G), or cytosine (C) bases in the primer 
(Note) Denaturation conditions vary depending on the thermal cycler and tubes used for PCR. Denaturation for 5 - 10 sec. at 98℃ or 20 - 30 sec. at 94℃ is recommended .  

PCR product : PCR products generated with EmeraldAmp GT PCR Master Mix have a single A at the 3'-termini, and PCR products can be directly cloned into a T-vector. It is also possible to clone products into blunt-end vectors after blunting and phosphorylation of the ends. 

Dye marker migration : When 5 μl of the reaction mixture is used for electrophoresis on a 1% Agarose L03 (Cat. #5003) gel, the blue dye front migrates near 3 - 5 kb and the yellow dye front is below 50 bp.</P></DIV><BUTTON class="collapsible2 font">Competent Cell Preparation (iGEM Protocol)</BUTTON><DIV class="content2"><OL type="I" style="font-size: 5.5mm; text-align: justify; " align="LEFT"><LI>Ethanol treat all working areas for sterility.</LI><LI>Inoculate 250 ml of SOB medium with 1 ml vial of seed stock and grow at 20°C to an OD600nm of 0.3. Use the &quot;cell culture&quot; function on the Nanodrop to determine OD value. OD value = 600nm Abs reading x 10</LI><UL><LI>This takes approximately 16 hours.</LI><LI>Controlling the temperature makes this a more reproducible process, but is not essential.</LI><LI>Room temperature will work. You can adjust this temperature somewhat to fit your schedule</LI><LI>Aim for lower, not higher OD if you can't hit this mark</LI></UL><LI>Fill an ice bucket halfway with ice. Use the ice to pre-chill as many flat bottom centrifuge bottles as needed.</LI><LI>Transfer the culture to the flat bottom centrifuge tubes. Weigh and balance the tubes using a scale</LI><UL><LI>Try to get the weights as close as possible, within 1 gram.</LI></UL><LI>Centrifuge at 3000g at 4°C for 10 minutes in a flat bottom centrifuge bottle.</LI><UL><LI>Flat bottom centrifuge tubes make the fragile cells much easier to resuspend</LI></UL><LI>Decant supernatant into waste receptacle, bleach before pouring down the drain.</LI><LI>Gently resuspend in 80 ml of ice cold CCMB80 buffer</LI><UL><LI>Pro tip: add 40ml first to resuspend the cells. When cells are in suspension, add another 40ml CCMB80 buffer for a total of 80ml</LI><LI>Pipet buffer against the wall of the centrifuge bottle to resuspend cells. Do not pipet directly into cell pellet!</LI><LI>After pipetting, there will still be some residual cells stuck to the bottom. Swirl the bottles gently to resuspend these remaining cells</LI></UL><LI>Incubate on ice for 20 minutes</LI><LI>Centrifuge again at 3000G at 4°C. Decant supernatant into waste receptacle, and bleach before pouring down the drain.</LI><LI>Resuspend cell pellet in 10 ml of ice cold CCMB80 buffer.</LI><UL><LI>If using multiple flat bottom centrifuge bottles, combine the cells post-resuspension</LI></UL><LI>Use Nanodrop to measure OD of a mixture of 200 μl SOC and 50 μl of the resuspended cells</LI><UL><LI>Use a mixture of 200 μl SOC and 50 μl CCMB80 buffer as the blank</LI></UL><LI>Add chilled CCMB80 to yield a final OD of 1.0-1.5 in this test.</LI><LI>Incubate on ice for 20 minutes. Prepare for aliquoting</LI><UL><LI>Make labels for aliquots. Use these to label storage microcentrifuge tubes/microtiter plates</LI><LI>Prepare dry ice in a separate ice bucket. Pre-chill tubes/plates on dry ice.</LI></UL><LI>Aliquot into chilled 2ml microcentrifuge tubes or 50 μl into chilled microtiter plates</LI><LI>Store at -80°C indefinitely.</LI><UL><LI>Flash freezing does not appear to be necessary</LI></UL><LI>Perform test transformations to calculate your <A href="http://parts.igem.org/Help:Competent_Cell_Test_Kit">competent cell efficiency</A></LI><UL><LI>Thawing and refreezing partially used cell aliquots dramatically reduces transformation efficiency by about 3x the first time, and about 6x total after several freeze/thaw cycles.</LI></UL><P style="font-family: 'title', sans-serif; font-size: 5.5mm;">Good cells should yield around 100 - 400 colonies
Transformation efficiency is (dilution factor=15) x colony count x 105/µgDNA
We expect that the transformation efficiency should be between 1.5x108 and 6x108 cfu/µgDNA</P><BUTTON class="collapsible2 font">PCR Protocol for Phusion High-Fidelity DNA Polymerase</BUTTON><DIV class="content2"><P style="font-family: 'title', sans-serif; font-size: 5.5mm;">
Reaction Setup: We recommend assembling all reaction components on ice and quickly transferring the reactions to a thermocycler preheated to the denaturation temperature (98°C). All components should be mixed and centrifuged prior to use. It is important to add Phusion DNA Polymerase last in order to prevent any primer degradation caused by the 3´→ 5´ exonuclease activity. Phusion DNA Polymerase may be diluted in 1X HF or GC Buffer just prior to use in order to reduce pipetting errors. Please note that protocols with Phusion DNA Polymerase may differ from protocols with other standard polymerases. As such, conditions recommended below should be used for optimal performance. 
Notes: Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary. Overlay the sample with mineral oil if using a PCR machine without a heated lid.

Transfer PCR tubes from ice to a PCR machine with the block preheated to 98°C and begin thermocycling:

Thermocycling conditions for a routine PCR:
General Guidelines:

Template: 
Use of high quality, purified DNA templates greatly enhances the success of PCR. Recommended amounts of DNA template for a 50 μl reaction are as follows:
f the template DNA is obtained from a cDNA synthesis reaction, the volume added should be less than 10% of the total reaction volume. 

Primers:
Oligonucleotide primers are generally 20–40 nucleotides in length and ideally have a GC content of 40–60%. Computer programs such as Primer3 can be used to design or analyze primers. The final concentration of each primer in a reaction using Phusion DNA Polymerase may be 0.2–1 μM, while 0.5 μM is recommended. 

Mg++ and additives:
Mg++ is critical to achieve optimal activity with Phusion DNA Polymerase. The final Mg++ concentration in 1X Phusion HF and GC Buffer is 1.5 mM. Excessive Mg++ can prevent full denaturation of DNA as well as cause non-specific binding of primers. The optimal Mg++ concentration is affected by dNTP concentration, the template being used and supplements that are added to the reaction. This can also be affected by the presence of chelators (e.g. EDTA). Mg++ can be optimized in 0.5 mM increments using the MgCl2provided.&gt;br&gt; 

Amplification of difficult targets, such as those with GC-rich sequences or secondary structure, may be improved by the presence of additives such as DMSO (included). A final concentration of 3% DMSO is recommended, although concentration can be optimized in 2% increments. It is important to note that if a high concentration of DMSO is used, the annealing temperature must be lowered as it decreases the primer Tm (2). Phusion DNA polymerase is also compatible with other additives such as formamide or glycerol. 

Deoxynucleotides:
The final concentration of dNTPs is typically 200 μM of each deoxynucleotide. Phusion cannot incorporate dUTP.

Phusion DNA Polymerase Concentration:
We generally recommend using Phusion DNA Polymerase at a concentration of 20 units/ml (1.0 units/50 μl reaction). However, the optimal concentration of Phusion DNA Polymerase may vary from 10–40 units/ml (0.5–2 units/50 μl reaction) depending on amplicon length and difficulty. Do not exceed 2 units/50 µl reaction, especially for amplicons longer than 5 kb.

Buffers:
5X Phusion HF Buffer and 5X Phusion GC Buffer are provided with the enzyme. HF buffer is recommended as the default buffer for high-fidelity amplification. For difficult templates, such as GC-rich templates or those with secondary structure, GC buffer can improve reaction performance. GC buffer should be used in experiments where HF buffer does not work. Detergent-free reaction buffers are also available for applications that do not tolerate detergents (e.g. microarray, DHPLC).

Denaturation: 
An initial denaturation of 30 seconds at 98°C is sufficient for most amplicons from pure DNA templates. Longer denaturation times can be used (up to 3 minutes) for templates that require it. 

During thermocycling, the denaturation step should be kept to a minimum. Typically, a 5–10 second denaturation at 98°C is recommended for most templates. 

Annealing: 
Annealing temperatures required for use with Phusion tend to be higher than with other PCR polymerases. The NEB Tm calculatorshould be used to determine the annealing temperature when using Phusion. Typically, primers greater than 20 nucleotides in length anneal for 10–30 seconds at 3°C above the Tm of the lower Tm primer. If the primer length is less than 20 nucleotides, an annealing temperature equivalent to the Tm of the lower primer should be used. A temperature gradient can also be used to optimize the annealing temperature for each primer pair. For two-step cycling, the gradient can be set as high as the extension temperature. 

For high Tm primer pairs, two-step cycling without a separate annealing step can be used. 

Extension:
The recommended extension temperature is 72°C. Extension times are dependent on amplicon length and complexity. Generally, an extension time of 15 seconds per kb can be used. For complex amplicons, such as genomic DNA, an extension time of 30 seconds per kb is recommended. Extension time can be increased to 40 seconds per kb for cDNA templates, if necessary.

Cycle number: 
Generally, 25–35 cycles yield sufficient product.

2-step PCR:
When primers with annealing temperatures ≥ 72°C are used, a 2-step thermocycling protocol is recommended.

Thermocycling conditions for a routine 2-step PCR:
PCR product:
The PCR products generated using Phusion DNA Polymerase have blunt ends; if cloning is the next step, then blunt-end cloning is recommended. If TA-cloning is preferred, then DNA should be purified prior to A-addition, as Phusion DNA Polymerase will degrade any overhangs generated.
Addition of an untemplated -dA can be done with Taq DNA Polymerase (NEB #M0267) or Klenow exo– (NEB #M0212 ).</P></DIV></OL></DIV></OL></DIV></DIV></CENTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>