Team:IIT-Madras/Results
iGEM Collaborations Page
Project
Description Design Notebook Results Demonstrations Improvements InterLab Safety
Parts Software
Human Practices
Overview Silver Human Practices Integrated Human Practices
Public Engagement
Overview The Language Project School Visits Research Magazine Workshop SynBio Journal Club
The Team
The Team Attributions Collaborations
×
Home Project
Description Design Notebook Results Demonstrations Improvements InterLab Safety
Parts Software Human Practices
Overview Silver Human Practices Integrated Human Practices
Public Engagement
Overview Language Project School Visits Research Magazine Workshop SynBio Journal Club
The Team
The Team Attributions Collaborations
☰
Team: IIT-Madras/Results
Results
Promoter results:
From the initial set of 20 promoters, we got successful cloning results for 15 promoters. We performed fluorometry experiments and obtained results for these 15 promoters. The promoter names are P1R, P3R, P4R, P5R, P6R, P70R, P8R, P9R, P71R, P1S, P2S, P5S, P6S, P8S, P71S.
We took pBAV1k-T5-GFP (BBa_K1321309) as the control, which we got from Addgene(https://www.addgene.org/267), and the wild-type as the negative control.
All promoters were cloned into a pBAV1k vector backbone with GFP. Fluorometry studies were then carried out with each clone.  From these 15 promoters, we got a spectrum of results showing varying expression. The R-Series promoters (promoters having the Salis lab calculated RBS) mostly exhibited better expression rates as compared to those of the S-series.
For each promoter, we had two biological replicates and three technical replicates. The errors and differences in each of these are depicted in the error bars.
Promoters P8R (BBa_K2857116), P8S (BBa_K2857118), P9R (BBa_K2857119), P1R (BBa_K2857111), P1S (BBa_K2857101), P8S (BBa_K2857012), and P3R (BBa_K2857113) showed higher expression than the T5 promoter.
Promoters P5R (BBa_K2857115), P5S (BBa_K2857105), P2S (BBa_K2857102), P71R (BBa_K2857120), P4R (BBa_K2857114), P6R (BBa_K2857116), P70R (BBa_K2857117),P71S (BBa_K2857) and P6S(BBa_K2857106) showed lower expression than the T5 promoter.
P8S and T5 promoter:
P8S showed tremendously high expression of GFP, 75-fold greater than the control.
Conclusion:
We have been successful in obtaining a spectrum of promoters which is desirable while generating a synthetic promoter library.
We have obtained promoters which show higher expression as well as ones showing lower expression than the T5 promoter.
We have designed a promoter P8S which shows about 75 fold higher expression than T5 promoter
Codon optimized GFP and Codon usage table results:
From the Acinetobacter baylyi ADP1 genome we obtained 1194 genes after removing hypothetical and putative sequences from the coding sequences annotation available from NCBI. The Codon Usage Table Enumerator (CUTE) tool from ChassiDex was used to generate the codon-usage table for the organism.
The occurrence of each codon is shown below (Fig1) in the genome of A. baylyi
The table attached below lists the percentage distribution of different codons amongst the analyzed genes.
The table attached below gives the probability of a particular amino acid being coded for by a particular codon.
These tables were then used to codon-optimize reporter proteins - GFP and mCherry - using the sequences available from NCBI. Codon-optimized GFP was restriction-cloned into a pBAV1K vector and transformed into A. baylyi. It showed promising results as illustrated below. The codon-optimized reporter protein showed considerably higher expression of fluorescence than the control.
The magnitude of the fluorescence readings on the y-axis is depicted in arbitrary units relative to pBAV1k-T5-GFP (BBa_K1321309) in Acinetobacter baylyi ADP1. WT is wild-type strain of Acinetobacter baylyi ADP1 while the last column shows fluorometry data for Acinetobacter baylyi ADP1 with the codon-optimized GFP in a pBAV1k vector under the T5 promoter. The error bars were calculated by taking into account two biological replicates and three technical replicates of each biological replicate.
Conclusion:
From the fluorometry results we can note that codon-optimized GFP shows higher expression in comparison to the control. This clearly demonstrates that the expression of codon-optimized GFP is greater than that of the non-codon optimized GFP.
This suggests that codon optimization has been successful.
This further indicates that the CUTE was successful in generating a valid codon-usage table.
