Team:IIT-Madras/Design
iGEM Collaborations Page
Project
Description Design Notebook Results Demonstrations Improvements InterLab Safety
Parts Software
Human Practices
Overview Silver Human Practices Integrated Human Practices
Public Engagement
Overview The Language Project School Visits Research Magazine Workshop SynBio Journal Club
The Team
The Team Attributions Collaborations
×
Home Project
Description Design Notebook Results Demonstrations Improvements InterLab Safety
Parts Software Human Practices
Overview Silver Human Practices Integrated Human Practices
Public Engagement
Overview Language Project School Visits Research Magazine Workshop SynBio Journal Club
The Team
The Team Attributions Collaborations
☰
Team: IIT-Madras/Design
Design
The rationale:
Introduction:
In synthetic biology and metabolic engineering studying the level of protein expression is very important. This may be phase dependent, inducible or constitutive. There are various factors influencing expression levels of proteins:(Glick et al 1987) [1]
The strength of the promoter [2]
The efficiency of the RBS [3]
Substrate and co-factor availability [4]
The half life of the mRNA [5]
The metabolic state of the cell
Stability of the foreign protein in the cell
The abundance of the specific tRNA/codon bias [6]
The copy number of the gene encoding the protein of interest [7]
Interaction of the protein with other proteins in the cell
Presence of an inducer or a signalling molecule for an inducible promoter
In our project, we focused on three factors - Promoter, RBS and codon bias.
Promoter: It is the region upstream of a gene where the RNA polymerase binds and initiates transcription. The binding of RNA polymerase to the promoter is often the rate-limiting step in a bacterial system as translation and transcription are coupled, unlike in an eukaryotic system where the mRNA undergoes post-transcriptional modifications. Hence, the promoter sequence is the major translational regulator of gene expression.
RBS: It is the region just upstream of the coding region where the ribosomal unit binds to initiate translation. The mRNA conformation at the RBS is extremely important in bringing both sub-units of the ribosome together to initiate translation.
Codon bias: The abundance of tRNA varies from species to species. Hence, with the genetic code being degenerate, each species prefers a different codon for each amino acid. The abundance of tRNA levels could also impact the rate of translation. By the process of codon optimization, gene expression can be increased.
Promoters (using random and rational approaches):
Promoters are usually structured similar to the construct shown below.
Region 1 is typically about 17 nucleotides long and contains the -35 conserved region which is recognized by RNA polymerase to initiate transcription. Following this is Region 2, which is about 17 nucleotides long. The -10 conserved region is present here which is typically a TATA box, a motif that is conserved for promoters. Following this we have a 6 nucleotide long region which we named Region 3 and then a transcription initiation site.
The regions flanking the -35 and -10 regions - regions 1, 2 and 3 - affect the strength of the promoter. [8] .
For our project we used a T5 promoter [BBa_K592008]. T5 is a constitutive promoter that is not under the influence of any protein. It is known to work in a broad range of microorganisms like E. coli, Acinetobacter baylyi ADP1 etc. [9] . An advantage of creating a T5 promoter-based library is that it might also work in other organisms like E. coli and industrially important chassis like Cornybacterium glutamicum. However, documentation and characterization of these promoters in these other chassis would be required to conclusively prove this. We have characterized these promoters in Acinetobacter baylyi ADP1.
We have classified the promoters we designed based on the method used into two categories:
i)P category: Randomizing the nucleotides flanking the -35 and -10 regions by conserving the GC content percentage wise in individual regions. Nucleotides flanking the -35 and -10 regions were also kept unchanged so as to not alter the interaction of RNA polymerase with the binding sites. Care was taken to prevent the accidental insertion of a biobrick restriction site or an Afl(II) restriction site.
We designed four promoters in this category. (BBa_K2857003, BBa_K2857004, BBa_K2857005, BBa_K2857006)
ii)Q Category:  In this approach, we introduced (in silico) point mutations in the T5 (BBa_K592008) promoter sequence . These promoters were selected based on the percentage of similarity they had with T5 (BBa_K592008) promoter. Care was taken to not create any restriction site or introduce mutations in the -35 and -10 regions. Based on this we selected promoters having 57% 66%, 79%, 85%, and 91% similarity. We selected two promoters showing 79% similarity with different sequences. This method was inspired by Mordaka P. M. et al 2018 [10] .
We designed six promoters using this approach (Q5:BBa_K2857007, Q6:BBa_K2857008, Q70:BBa_K2857009, Q71:BBa_K2857010, Q9:BBa_K2857011, Q8:BBa_K2857012)
Ribosome Binding Site
For each of these 10 promoters we used Salis lab RBS calculator to calculate the RBS specific for each promoter, GFP and Acinetobacter baylyi ADP1.(https://salislab.net/software/) (Salis, H M. “The Ribosome Binding Site Calculator.” Methods in Enzymology., U.S. National Library of Medicine, www.ncbi.nlm.nih.gov/pubmed/21601672.). These Ribosome Binding Sites (RBS) have been named as Biobricks BBa_K2857013-BBa_K2857022. We assembled the RBS with the promoter sequences and had this DNA sequence synthesized. The promoters with modified RBS are listed under BioBricks BBa_K2857111-BBa_K2857120.
The same promoters under the iGEM RBS are labelled as BioBricks BBa_K2857101-BBa_K2857110. These were also synthesized by IDT and submitted by us.
Promoter
Promoter BioBrick number
Corresponding Salis lab RBS BioBrick number
Complete Assembly(with iGEM RBS)S category
Complete Assembly (with Salis lab RBS) R category
P1
BBa_K2857003
BBa_K2857013
BBa_K2857101
BBa_K2857111
P2
BBa_K2857004
BBa_K2857014
BBa_K2857102
BBa_K2857112
P3
BBa_K2857005
BBa_K2857015
BBa_K2857103
BBa_K2857113
Q4
BBa_K2857006
BBa_K2857016
BBa_K2857104
BBa_K2857114
Q5
BBa_K2857007
BBa_K2857017
BBa_K2857105
BBa_K2857115
Q6
BBa_K2857008
BBa_K2857018
BBa_K2857106
BBa_K2857116
Q70
BBa_K2857009
BBa_K2857019
BBa_K2857107
BBa_K2857117
Q71
BBa_K2857010
BBa_K2857020
BBa_K2857108
BBa_K2857118
Q8
BBa_K2857012
BBa_K2857022
BBa_K2857109
BBa_K2857119
Q9
BBa_K2857011
BBa_K2857021
BBa_K2857110
BBa_K2857120
CUTE, A. baylyi and Codon Optimized-GFP:
We had contacted GenScript for codon optimized GFP for A. baylyi but they did not have reliable data on the codon usage table of Acinetobacter baylyi ADP1. There was one such table available on Kazusa , which was based on only two CDS.
We identified this industry based problem that codon usage tables were available mainly only for standard hosts which are widely used. This is a hindrance while using unconventional hosts for one’s studies. Hence, we created CUTE which is available on the ChassiDex website. It can be found on CUTE ChassiDex . This free-to-use online  tool can be used to generate the codon usage table of any organism as long as its CDS annotation is available.
We used CUTE to generate codon usage table data for A. baylyi ADP1 by taking into consideration the CDS annotation available on the NCBI site. This table can be found on the Results page of our wiki. From the Acinetobacter baylyi ADP1 genome we obtained 1194 genes after removing hypothetical and putative sequences from the coding sequences annotation available from NCBI.
Using this codon usage table, we codon optimized GFP and mCherry for Acinetobacter baylyi ADP1. We have submitted these BioBricks: GFP(BBa_K2857001) and mCherry (BBa_K2857002).
Assembly:
The iGEM standard vector pSB1C3 does not replicate in A. baylyi ADP1. From literature studies we found that pBAV1k can replicate in A. baylyi [11] . For our promoter studies, we amplified pBAV1k to get a promoter-less vector backbone into which our promoters could be cloned. Similarly, for our reporter studies, we amplified pBAV1k to get a reporter-less vector backbone into which the codon optimized GFP could be cloned.
pBAV1k could not be submitted due to the material-transfer agreement but it can be purchased from Addgene . This is also listed as a BioBrick (BBa_K1321309). This vector is a high copy, broad host range vector.
GFP was cloned in pBAV1k downstream of a T5 promoter and a RBS. Similarly, the promoters were cloned upstream of GFP in pBAV1k. To measure the codon-optimized GFP expression levels and the strengths of the promoters, fluorometry studies were carried out.
References
References
Glick, B.R. & Whitney, G.K. Journal of Industrial Microbiology (1987) 1: 277. https://doi.org/10.1007/BF01569305
Blazeck J., Alper H.S. Promoter engineering: recent advances in controlling transcription at the most fundamental level. Biotechnol. J. 2013;8:46–58. doi: 10.1002/biot.201200120.
Synthetic biology toolbox for controlling gene expression in the cyanobacterium Synechococcus sp. strain PCC 7002. Markley AL, Begemann MB, Clarke RE, Gordon GC, Pfleger BF ACS Synth Biol. 2015 May 15; 4(5):595-603.
(Metabolic pathway balancing and its role in the production of biofuels and chemicals. Jones JA, Toparlak ÖD, Koffas MA Curr Opin Biotechnol. 2015 Jun; 33():52-9.)
Use of expression-enhancing terminators in Saccharomyces cerevisiae to increase mRNA half-life and improve gene expression control for metabolic engineering applications. Curran KA, Karim AS, Gupta A, Alper HS Metab Eng. 2013 Sep; 19():88-97.
Codon usage: nature's roadmap to expression and folding of proteins. Angov E Biotechnol J. 2011 Jun; 6(6):650-9.)
Ajikumar P.K., Xiao W.H., Tyo K.E.J., Wang Y., Simeon F., Leonard E., Mucha O., Heng Phon T., Pfeifer B., Stephanopoulos G. Isoprenoid pathway optimization for Taxol precursor overproduction in Escherichia coli. Science. 2010;330:70–74. doi: 10.1126/science.1191652.
Gilman J, Love J. Synthetic promoter design for new microbial chassis. Biochemical Society Transactions. 2016;44(3):731-737. doi:10.1042/BST20160042
(Hermann Bujard, Reiner Gentz, Michael Lanzer, Dietrich Stueber, Michael Mueller, Ibrahim Ibrahimi, Marie-Therese aeuptle, Bernhard Dobberstein, [26] A T5 promoter-based transcription-translation system for the analysis of proteins in vitro and in vivo, Methods in Enzymology, Academic Press, Volume 155, 1987, Pages 416-433)*
Gilman J, Love J. Synthetic promoter design for new microbial chassis. Biochemical Society Transactions. 2016;44(3):731-737. doi:10.1042/BST20160042
(Murin, Charles Daniel, et al. Applied and Environmental Microbiology, American Society for Microbiology, Jan. 2012, )
Mordaka, Paweł M., and John T. Heap. “Stringency of Synthetic Promoter Sequences in Clostridium Revealed and Circumvented by Tuning Promoter Library Mutation Rates.” ACS Synthetic Biology, vol. 7, no. 2, 2018, pp. 672–681., doi:10.1021/acssynbio.7b00398.
Salis, H M. “The Ribosome Binding Site Calculator.” Methods in Enzymology., U.S. National Library of Medicine, www.ncbi.nlm.nih.gov/pubmed/21601672
