Team:IIT-Madras/Notebook
iGEM Collaborations Page
Project
Description Design Notebook Results Demonstrations Improvements InterLab Safety
Parts Software
Human Practices
Overview Silver Human Practices Integrated Human Practices
Public Engagement
Overview The Language Project School Visits Research Magazine Workshop SynBio Journal Club
The Team
The Team Attributions Collaborations
×
Home Project
Description Design Notebook Results Demonstrations Improvements InterLab Safety
Parts Software Human Practices
Overview Silver Human Practices Integrated Human Practices
Public Engagement
Overview Language Project School Visits Research Magazine Workshop SynBio Journal Club
The Team
The Team Attributions Collaborations
☰
Team: IIT-Madras/Results
Notebook
April 2018
13 April: We attempted making competent E. coli DH5alpha cells. We got the strain from Prof. Guhan Jayaraman's lab
16 April: We tried checking its competency by RFP transformation but did not get results.
20 April: We tried making competent cells again but we did not use liquid nitrogen for the flash freeze step. We used SOC media instead of LB this time.
22 April: We tried transformation but it failed in some plates. Much fewer colonies.
26 April:We met Guhan sir and Dr. Anantha Barathi Ma'am for guidance and help. Also, tried to analyze the feasibility of the project.
May 2018
8 May: Exams end and we discuss ideas with Anantha Ma'am (Instructor). One interesting study thought of was lignocellulosic mass conversion to other useful and economically highly valued products.
8-10 May: We do an extensive literature survey for this project and Acinetobacter baylyi ADP1 seems to be a good candidate.
14 May: We finalise work on lignin conversion to biofuels using Acinetobacter baylyi ADP1 as our project. We choose this organisms over others because it is non pathogenic and has amazing capabilities such as degrading aromatic compounds like caffeate, vaillate, etc.  Above all, it is naturally competent and Guhan sir's lab has expertise in working with this organism.
16 May: Acinetobacter baylyi has 6 copies of transposomal elements which were remove in 2017 to engineer a new strain called ADP-!Sx  (Suárez GA, Renda BA, Dasgupta A, Barrick JE. Reduced Mutation Rate  and Increased Transformability of Transposon-Free Acinetobacter baylyi ADP1-ISx. Kivisaar M, ed. Applied and Environmental  Microbiology. 2017;83(17):e01025-17. doi:10.1128/AEM.01025-17.) We request Prof. Barrick to send us the engineered strain.
18 May: We plan to compile all literature and decide on the direction of the project. Vanillin production seems to be potential candidate but experiments fail.
22 May: We realise that we need a proper toolkit and well established methods to work with A. baylyi. So we decide to shift the direction of our project towards expanding the toolbox of A. baylyi ADP1.
23 May: We finalise upon the things we want to try out: codon-optimized reporters and promoters. If time permits then terminators.
25 May: Anantha ma'am asks Sagar to place an order for CO-GFP and mCherry with GenScript. Meanwhile we do a lit survey for various approaches for promoter design.
29 May: GenScript informs us that there is no reliable codon usage table available. They send us link to the Kasuza website which has a codon table based on 2 coding regions.
30 May: We knew that coding region annotation was present on NCBI website. We discussed this problem with KD and idea of CUTE was born.
31 May: A workplan was drawn up to start working on promoter Ddsign, codon usage table for A. baylyi and codon optimization. We fixed upon using pBAV1K vector as it is known to work in A. baylyi and Sagar had preliminary results for it. KD, Burhan, Sathvik were assigned to work on the codon table of A. baylyi
Richa, Burhan and Sathvik get to work on promoter design.
June 2018
4 June: Codon generator was ready to be used on the ChassiDex website. We took the complete CDS annotation of A. baylyi and removed putative and hypothetical regions. We were left with 1194 CDS regions. We got the codon usage information. The data was then normalized and the codon table of A. baylyi was generated.
5 June: We now had enough data to have some parameters for designing promoters. Richa helped us to find regions upstream of the housekeeping genes in order to locate the -35 and -10 regions.
6 June: We tried talking to Mr. Anindhya of Medha Bioinformatics but all efforts to find conserved regions were fruitless. Meanwhile, we codon optimize the GFP and mCherry using the codon table. We replaced the less frequently used codons with frequently used codons and screened for unwanted restriction sites. Then, we sent the sequence to Anantha ma'am for another checkup.
8 June: We used various other tools such as ClustalW, MEME suite, etc but could not find the consensus regions upstream of the genes that we had selected.
10 June: We discussed the generic nature of CUTE and planned to implement the same in the promoter sequences. We also discussed the nature of the pBAV1K vector that allowed it to replicate in so many host organisms.
11 June: We discussed and agreed upon making a T5 promoter-based library for our project.
12 June: GFP and mCherry designs were corrected wherever necessary. Planned to place order for the finalized sequences.
13 June: A literature survey of T5 promoter and promoter-based library was started.
18 June: The lab training period for Burhan and Sathvik was begun. Anantha Barathi ma'am, Sagar, Velvizhi Devi and Suchi Smita helped out with the training. Various basic molecular biology and microbiology techniques were taught.This training made us familiar with lab safety and lab rules and familiarized us with lab equipment. In parallel, we also explored the various methods and approaches for building a promoter library.
19 June: We placed the order for the codon-optimized GFP and mCherry to IDT. We finalized upon the parameters and started generating sequences.
22-26 June: Sathvik leaves for home, lab training continues. Burhan generates promoter library containing over 300 sequences of the same design. Meanwhile, since the project abstract deadline was approaching, we decide to christen it with a nice name. Sathvik names it 'ADaPtat1on- Expanding the toolkit of Acinetobacter baylyi'.
29 June: Burhan leaves and the lab training period subsequently ends.
July 2018
9 July: We discuss over the phone about Clostridium Synthetic promoter library paper that was published recently.Seeing we also try to incorporate that approach in some of our promoters out of complete promoter library.
10 July Parameters taken into consideration by various authors were taken into consideration and similar lines were followed to generate the promoter library.
16 July We have the promoter sequences from both categories, with the former being called the P the and the latter the Q series. We planned to add RBS' calculated using the RBS calculator tool from the Salis Lab website. Codon-optimized reporters arrive.
18 July We assembled the parts in-silico and then screened for unwanted restriction sites and agreed on the selection of parts from each category. We agreed upon 4 parts from P category along with small spacer length of 6 between RBS and CDS (GFP) and 6 parts of Q category with  no additional spacer. For each part, the RBS was calculated individually using the Salis Lab calculator.
20 July The assembled parts were sent to Anantha Ma'am for screening to check if any unwanted restriction sites were created.
24 July We discussed the rationale again and made several corrections.
27-29 July Sathvik, Sai Guha and Sarvesh presented ADaPtat1on, ChassiDex and The Language Project at the All-India iGEM meet at the Indian Institute of Science, Bangalore.
30 July All team members return from vacations. Burhan gives an update on the work done on ADaPtat1on.
August 2018
2 August: Promoter sequences are finalized and the order for them is placed.
7 August: iGEM wetlab work starts- the first plan is to clone platform construct into the pBAV1K vector.
10 August: Platform construct ligation was improper and gave bands even at undesired region.
13 August: Troubleshooting of cloning of platform construct in pBAV1k using Afl(II) is done. All 20 promoters arrive.
17 August: We decide not to use the platfrom construct. Instead, we plan to PCR amplify the pBAV1K vector backbone using appropriate primers such that existing GFP can be excised out and codon-optimized GFP can be cloned.
20 August: We design appropriate primers for pBAV1k and place the order. We also design primers to clone the promoters.
27 August: Primers arrive. We amplify the backbone and GFP gblocks(>12cycles). In all our PCR amplification experiments, we used the Phusion Polymerase HF.
28 August: After checking length on gel, restriction digestion was done and overnight ligation was set up.
29 August: A. baylyi was transformed and we selected 5 colonies. We screened the colonies using Afl(II) as only the insert had that unique site.
30 August: Colonies 1, 4 and 5 gave positive results and so they were selected.
September 2018
12 September: Sankalpa joins labwork.
13 September: Cloning of the first 6 promoters in pSB1C3.
16 September: Cloning of the first 6 promoters fails, robably due to the low amount of DNA used.
17 September: PCR amplification of promoters and pSB1C3 is done. This time, we use only one promoter.
19 September: Colony PCR reveals successful transformation.
21 September: PCR amplification of pBAV1K is done such that T5 promoter remains unamplified. Other regions are amplified.
22 September: Cloning of the next 4 promoters in pBAV1k and pSB1C3 is done. Cloning of GFP into pSB1C3 is also carried out.
25 September: Contamination is suspected in our lab's plasmid isolation kit.
26 September: We did not get any colony on the P2R pBAV1K plate. The remaining transformations were all successful (as ascertained using colony PCR).
27 September: We attempt to clone the next 15 promoters and mCherry in the pSB1C3 vector for BioBrick submission.
October 2018
1 October: Fluorometry of CO-GFP was unsuccesful.
2 October: Set up a gradient PCR of the promoters
3 October: Amplification of promoters and cloning of promoters into pBAV1K.
4 October: Cloning of promoters into pBAV1K and transformation into A. baylyi
5 October: Transformation of promoters into A. baylyi
6 October: Plasmid isolation of pSB1C3 containing promoters from E. coli.
7 October: Fluorometry of CO-GFP and first set of promoters
8 October: Fluorometry of second set of promoters.
