Team:IIT-Madras/Software
iGEM Collaborations Page
Project
Description Design Notebook Results Demonstrations Improvements InterLab Safety
Parts Software
Human Practices
Overview Silver Human Practices Integrated Human Practices
Public Engagement
Overview The Language Project School Visits Research Magazine Workshop SynBio Journal Club
The Team
The Team Attributions Collaborations
×
Home Project
Description Design Notebook Results Demonstrations Improvements InterLab Safety
Parts Software Human Practices
Overview Silver Human Practices Integrated Human Practices
Public Engagement
Overview Language Project School Visits Research Magazine Workshop SynBio Journal Club
The Team
The Team Attributions Collaborations
☰
Team: IIT-Madras/ADaPtat1on
Software
ChassiDex
While iGEM teams and the synthetic biology community have mostly focused on developing DNA parts called ‘BioBricks’ for use in biological systems, the parts have to be ultimately used in an organism called a host or sometimes even ‘chassis’. For most projects, researchers try to stick with using a few familiar model organisms such as Escherichia coli for the host system requirements. Lack of experience is often a reason to avoid unfamiliar yet apt organisms for the project at hand. To explain this with an analogy, imagine Taxi-hailing apps being available only for desktop computers because the developers do not know how to develop applications that run on mobile phones. Hence, in the 2017 edition of iGEM, Team IIT Madras worked on developing a database of host organisms called ChassiDex, that aimed to curate different organisms along with all the information about them that is relevant in the context of synthetic biology. The idea of having such a database was very well received by the synthetic biology community that attended the iGEM Giant Jamboree in 2017. We have continued working on the project and revamped it with the suggestions we received, with the hope of improving this project and creating a team to manage the database. The website for the database is available at Chassidex.org.
Additionally, We also developed some independent tools to help make life easier for synthetic biologists.   These include CUTE, a codon table generator and ComicSyns, a font for synthetic biologists.
Code on GitHub
Chassidex Main Page
Sample Organism: A. Baylyi
Our Organism of Interest for the year
CUTE: Codon Usage TablE generator
Codon Usage Tables. Every host organism has its own codon bias: the biased usage of particular codons when two or more codons in the genetic code stand for the same amino acid. While expressing a protein heterologously in a host organism, the DNA sequence is optimized to match the codon bias of the host, while keeping the protein sequence constant. This process is known as codon optimization and is done to increase expression in the chosen host organism. With ChassiDex, we aimed to make working with novel hosts easier for biologists, and also realized the role of codon optimization in doing so. While trying to synthesize a GFP reporter for Acinetobacter baylyi, GenScript informed us that they only had access to a codon usage table from Kazusa [1] : a rather outdated database of codon tables that was generated in 2007 using the genome and coding sequence data available at that time. For A. baylyi, only two coding sequences were available and considered for generating the table, this wasn’t statistically significant. However, a large number of coding sequences have been identified from the sequenced genome of A. baylyi now and we decided to build a software tool to generate the codon table ourselves. We have integrated this tool with ChassiDex, hoping to help out synthetic biologists who might find themselves in a similar situation as ourselves. We used this tool to generate a codon usage table for Acinetobacter baylyi, which we then used to codon optimize our GFP protein. We found this codon optimized version of GFP to show a higher expression in A. baylyi compared to the unoptimized one. The tool is hosted at CUTE ChassiDex and has the following features designed for the end user who is not a geek in programming:
Completely Graphical User Interface (GUI), no coding or command line knowledge is required to use the tool.
Runs in the web browser and hence across all Operating Systems (Windows/MacOS/Linux)
No download/installation required.The tool can be used by simply visiting the website. Even on the website, no login or registration is required to use the tool.
It is written in JavaScript and the entire program is front-end, which means we do not have to manage server-side code. This reduces operating costs for us.
Code on GitHub
A Codon Bias Table generated for a FASTA Sequence taken from NCBI
ComicSyns
Draw SynBio circuits with ease! ComicSyns is an iconic icon font that lets you draw SynBio circuit symbols by simply typing in characters on your keyboard! The symbols are adapted from SBOL Visual [2]
Code on GitHub
ComicSyns Main Page
Link to our GitHub organisation
References
Nakamura, Y., Gojobori, T., & Ikemura, T. (2000). Codon usage tabulated from international DNA sequence databases: status for the year 2000. Nucleic Acids Research, 28(1), 292.
Cox et. al. 2018, Synthetic Biology Open Language Visual (SBOL Visual) Version 2.0, Journal of Integrative Bioinformatics, Volume 15, Issue 1, 20170074, ISSN (Online) 1613-4516, DOI: https://doi.org/10.1515/jib-2017-0074 .
