<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_INSA-Lyon_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:INSA-Lyon/Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM : Software</TITLE></P><DIV id="preloader"><DIV id="status"> </DIV></DIV><SECTION id="menu-area"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="nav-tabs navbar-collapse collapse"><UL id="top-menu" class="nav navbar-nav navbar-right main-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Composite_Part">Parts</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Wet Lab </A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Experiments">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Interlab">Interlab</A></LI><LI class="dropdown active"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Dry Lab </A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Software">Software</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Engagement">Public engagement</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:INSA-Lyon/Collaborations">Collaborations</A></LI></DIV></DIV></SECTION><DIV class="col-md-3 contents-sidebar"><UL id="sidebar" class="nav nav-stacked hidden-xs hidden-sm" data-spy="affix" data-target=".contents-sidebar" data-offset="100"><LI><A href="#softwhy">Why</A></LI><LI><A href="#softapplications">Applications</A></LI><LI><A href="#softfreetool">A not so fast but simple and free tool</A></LI><LI><A href="#softopensource">Is it open-source?</A></LI><LI><A href="#softexplained">How does it work?</A></LI><LI><A href="#softreliability">Reliability</A></LI><LI><A href="#softresults">Results</A></LI><LI><A href="#softreferences">References</A></LI></UL></DIV><SECTION id="topsoft"><DIV class="topsoft-overlay"><DIV class="container-fluid"><DIV class="row"><DIV class="topsoft-area"><DIV class="row"><DIV class="col-md-3 col-md-offset-4"><H5 class="wow fadeInUp" data-wow-duration="0.5s" data-wow-delay="0.5s">SOFTWARE</H5></DIV><DIV class="col-md-2 col-md-offset-3"><A class="read-more-btn wow fadeInUp" data-wow-duration="1s" data-wow-delay="1s" href="#software">Read More</A></DIV></DIV></DIV></DIV></DIV></DIV></SECTION></DIV><SECTION id="software"><DIV class="container"><DIV class="row"><DIV class="col-md-10 col-md-offset-2"><DIV class="software-content"><DIV class="row"><DIV class="title-area" id="softsummary"><H2 class="title">A SINGLE-STRAND DNA 3D-STRUCTURE PREDICTION TOOL</H2><H2>using ViennaRNA<SUP>[1]</SUP> and Rosetta<SUP>[2]</SUP> bioinformatics software</H2><P>
                                  We found a simple method to predict the structure of aptamers, by copying the protocol to compute RNA 3D-foldings with Rosetta. The trick is to convert the aptamer to RNA, then compute its structure, re-convert to DNA and minimize the free energy to adjust atom coordinates.
                              </P><DIV class="subtitle" id="softwhy"><H2 class="title">Why</H2><P>
                                      The best advantage of aptamers is their ability to be easily modified, improved, to increase their affinity or stability. This can be done by doing substitutions, insertions or deletions to the original sequence.
                                      To study the effect of the modification, you need to have the 3D structure of your aptamer. The goal of our tool is to predict the 5 most probable structures your aptamer would have in biological samples, and output them in PDB file format, from scratch with only primary sequence input.
                                  </P></DIV><DIV class="subtitle" id="softapplications"><H2 class="title">Applications</H2><P><STRONG>Want to see what the aptamer you designed looks like?</STRONG><STRONG>Want to prepare a PDB structure for docking with your target/biomarker?</STRONG><STRONG>Want to see how a polymorphism in the chain influences interaction with the target?</STRONG></P><P>
                                      Then use this computation procedure. Although it is not made to give precise energy computation results, it does return the most probable 3D structures.
                                  </P></DIV><DIV class="subtitle" id="softfreetool"><H2 class="title">A not so fast but simple and  free tool</H2><P>
                                  We are currently in the process of making the tool available online for non-commercial use. It will be hosted onto one of our school’s server so anyone will be able to use it directly online. (our team signed a custom licence of the Rosetta suite for the purpose of our web server).
                                  </P><A href="http://ia2c.heavydev.fr">http://ia2c.heavydev.fr</A><P>
                                  As we are not funded, our computation capacity is still limited: it may last several hours, and only 4 tasks can be treated at the same time.
                                  </P><P>
                                  You can also download it for your local server at <A href="https://github.com/ia2c/aptamers">https://github.com/ia2c/aptamers</A>, but you need to download Rosetta separately from rosettacommons.org. (see installation instructions)
                                  </P></DIV><DIV class="subtitle" id="softopensource"><H2 class="title">What did we create?  What did we take from others? Is it open-source?</H2><P>          The code we typed is an automatisation of a long, complex and hard command-line procedure with several different softwares<SUP>[3]</SUP>.
                                      Our code is only some Python script that runs Rosetta and ViennaRNA for you correctly.
                                      It makes the modelisation easy for you.
                                  </P><P>
                                      Our code is freely distributed, freely modifiable for any improvements, under the MIT licence ( <A href="https://opensource.org/licenses/MIT">https://opensource.org/licenses/MIT</A> ).                                  </P><P>
                                      ViennaRNA and Rosetta are both softwares distributed with their source-codes, modifiable, but only for non-commercial purpose (see links below).                    
                                      <A href="https://github.com/ViennaRNA/ViennaRNA/blob/master/COPYING">https://github.com/ViennaRNA/ViennaRNA/blob/master/COPYING</A><A href="https://els.comotion.uw.edu/licenses/86">https://els.comotion.uw.edu/licenses/86</A></P></DIV><DIV class="subtitle" id="softexplained"><H2 class="title">How does it work?</H2><P>
                                      Start with your primary DNA or RNA sequence as an input.
                                  </P><H2>Step 1 (only if the input is a DNA sequence): Transcription to RNA</H2><P>
                                  Our procedure was designed for RNA sequences. Therefore, if your input is a DNA sequence, we need to transcript it into a RNA sequence first. Prediction of the structure of the nucleic acid shows minor differences whether the input sequence is DNA or RNA<SUP>[4]</SUP>. Corrections can be implemented further in the process.
                                  Tech: We use the Biopython package to do so.
                                  </P><H2>Step 2: Prediction of the secondary structure</H2><P>
                                  If you did not provide the secondary structure of the sequence to the tool, we can compute it with the RNAFold algorithm from ViennaRNA, a dynamic algorithm of structure prediction.
                                  If the secondary structure of your RNA was described in an article, you should rely on it. This step is the most uncertain part of the procedure, because it is highly dependant on the environmental conditions: ionic concentrations, temperature, etc…
                                  </P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Fig. Example of a RNA secondary structure</FIGCAPTION></FIGURE><P><STRONG>Important remark:</STRONG> we are currently unable to predict pseudoknots or G-quadruplexes, which may be frequent in aptamers. Giving your own secondary structure may be a way to explicit those structures.</P><H2>Step 3: Computation of the 3D structure from the 2D</H2><P>We use Rosetta’s rna_denovo tool on the given 2D data, with flags -ignore_zero_occupancy and -no_minimize, and let it run for 20000 cycles (by default).
                                  </P><P>The algorithm used is Rosetta’s FARNA<SUP>[5]</SUP> (Fragment Assembly of RNA), a Monte-Carlo process, guided by a low-resolution knowledge-based energy function.
                                  </P><H2>Step 4: Minimization</H2><P>Rosetta will now score the structures and minimize only the 100 best ones
                                  (atoms are moved in space to reach a local minimum of potential energy).
                                  </P><P>Remark: it is a delicate step, because we selected the 100 best structures before having minimized them. This intermediate step is required in order to prevent missing out on better candidates.
                                  </P><H2>Step 5: Selection of the best ones</H2><P>We ask Rosetta to score the structures, so we can end up with the 5 best ones.
                                  This time, each structure corresponds to a local minimum of potential energy, so comparing structures really makes sense.
                                  </P><H2>Step 6: Saving the results in PDB files</H2><P>Rosetta exports the atom coordinates to write a PDB file.</P><H2>Step 7 (only is the input was a DNA sequence): Retro-transcription into DNA</H2><P>
                                      Our home-made python script can be used to modify the PDB files to change uracile bases into thymines and to deoxygenate riboses. Finally, structures are re-minimized to adapt atom coordinates from RNA to DNA.
                                  </P></DIV><DIV class="subtitle" id="softreliability"><H2 class="title">Reliability</H2><P>
                                      The structures you get as outputs may differ from one another. That is the reason why we keep the five most probable structures in the end instead of only one.
                                      Indeed, we can often observe variable domains and stable ones.
                                  </P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Fig. Superposition of the 20 most probable structures of the ATP aptamer</FIGCAPTION></FIGURE><P>
                                      We also made some analysis by comparing the models we obtained with the experimental PDB structures, obtained by X-ray crystallography, for the following single-strand nucleic acids:
                                      ATP Aptamer<SUP>[4]</SUP>HIV Reverse-Transcriptase (RT) Aptamer<SUP>[6]</SUP>Thrombin Aptamer (contains a G-quadruplex)<SUP>[7]</SUP>Human telomerase mRNA<SUP>[8]</SUP></P><P>      First, we superposed the 20 best structures to the native one.
                                  </P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Fig. ATP aptamer (left), HIV-RT aptamer (right). The green structures are native PDB files from X-ray crystallography, the coloured ones are our predicted structures.</FIGCAPTION></FIGURE><P>
                                  Then we used a measure of distance between the “real” native structure and the predicted one called RMSD (Root Mean Standard Deviation), the square-root of the sum of atom distances squared.
                                  In parallel, Rosetta gave us a potential energy value in its force-field unit (Rosetta Energy Score).
                                  </P><P>
                                  For all the obtained structures, the RMSD was under 10Å, which is the threshold between good and bad results, considering the flexibility of the single-strand nucleic acid molecules.
                                  </P><P>
                                  We are currently not able to predict G-quadruplexes or pseudoknots in the sequences, but we tried with some atom pair constraints to force them in the structures:
                                  </P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Fig. Thrombin aptamer, without and with constraints. </FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption">Fig. Scores without (blue) and with (orange) forced constraints for pseudoknots and G-quadruplexes. </FIGCAPTION></FIGURE><P>
                                      A Student’s test with a threshold of 5% to compare the two groups’ means of RMSD returns a p-value of 0.02, meaning that forcing the constraints is a treatment that gives significantly different (better) RMSDs than not forcing them.
                                  </P><FIGURE class="figure"><FIGCAPTION class="figure-caption">Fig. Human telomerase RNA without &amp; with forced constraints</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption">Fig. Scores without (blue) and with (orange) forced constraints for pseudoknots and G-quadruplexes. (Student’s p-value = 1.12 10<SUP>-6</SUP>).</FIGCAPTION></FIGURE><P>
                                      As shown in the two examples above, forcing bases pairing did not change the potential energy of the predicted molecule (no modification of its stability), but enabled the improvement of the models, with results closer to the reality. The user should have the possibility of using the secondary structure of his choosing (that would be well described in the literature).
                                  </P><P>
                                      As a conclusion, our method is relevant and gives good results.
                                  </P><P>
                                      Obviously, we cannot guarantee the accuracy of the results and we disclaim any responsability for or liability related to them.
                                      You can find more info about the scoring of structures in Rosetta’s documentation.
                                  </P></DIV><DIV class="subtitle" id="softresults"><H2 class="title">Results</H2><P>The structures computed by our tool look similar to PDB structures obtained by X-ray crystallography.
                                  </P><P>
                                      The shape of the molecule is very important, because the given structures are used for docking simulations with the aptamer’s target.
                                  </P><P>
                                      This is this procedure that helped us obtain the 3 PDB structures of the 3 aptamers against HBsAg<SUP>[9]</SUP>.
                                  </P><FIGCAPTION class="figure-caption" style="font-size:14px">Fig. The 3 HBV surface-antigen known aptamers, as predicted by our method</FIGCAPTION></DIV><DIV class="subtitle" id="softreferences"><H2 class="title">References</H2><P>
                                  [1]     <STRONG>ViennaRNA software </STRONG><A href="https://www.tbi.univie.ac.at/RNA/">https://www.tbi.univie.ac.at/RNA/</A></P><P>                   [2]     <STRONG>Rosetta’s documentation </STRONG><A href="http://rosettacommons.org/docs/latest/Home">http://rosettacommons.org/docs/latest/Home</A></P><P>
                                  [3]     Clarence Yu Cheng, Fang-Chieh Chou, Rhiju Das. <STRONG>Modeling Complex RNA Tertiary Folds with Rosetta</STRONG>. Methods in Enzymology, 553, 35-64, 2015, ISSN 15577988
                                  </P><P>
                                  [4]     Chin H Lin, Dinshaw J Patel. <STRONG>Structural basis of DNA folding and recognition in an AMP-DNA aptamer complex: distinct architectures but common recognition motifs for DNA and RNA aptamers complexed to AMP</STRONG>. Chemistry &amp; biology, 4, 817-832, 1997, ISSN 10745521.
                                  </P><P>
                                  [5]     <STRONG>FARNA algorithm </STRONG><A href="https://www.rosettacommons.org/manuals/archive/rosetta3.1_user_guide/app_RNA_denovo.html">https://www.rosettacommons.org/manuals/archive/rosetta3.1_user_guide/app_RNA_denovo.html</A></P><P>
                                  [6]         Miller M., Tuske S., Das K., DeStefano J. and Arnold E. (2015). <STRONG>Structure of HIV-1 reverse transcriptase bound to a novel 38-mer hairpin template-primer DNA aptamer</STRONG>. Protein Science, 25(1), pp.46-55.
                                  </P><P>
                                  [7]         Russo Krauss I., Spiridonova V., Pica A., Napolitano V. and Sica F. (2015). <STRONG>Different duplex/quadruplex junctions determine the properties of anti-thrombin aptamers with mixed folding</STRONG>. Nucleic Acids Res, 44(2), pp.983-991.
                                  </P><P>
                                  [8]         Theimer C., Blois C. and Feigon J. (2005). <STRONG>Structure of the Human Telomerase RNA Pseudoknot Reveals Conserved Tertiary Interactions Essential for Function</STRONG>. Molecular Cell, 17(5), pp.671-682.
                                  </P><P>
                                  [9]     Xi Z., Huang R., Li Z., et al. (2015). <STRONG>Selection of HBsAg-Specific DNA Aptamers Based on Carboxylated Magnetic Nanoparticles and Their Application in the Rapid and Simple Detection of Hepatitis B Virus Infection</STRONG>. ACS Appl. Mater. Interfaces, 7(21), pp.11215-11223.
                                  </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer"><DIV class="container"><DIV class="row"><DIV class="row"><DIV class="col-md-8 col-sm-8"><DIV class="footer-left"><P>Designed by Team iGEM INSA-LYON </P></DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>