Team:INSA-Lyon/Experiments
iGEM : Protocols
 
Toggle navigation
Project
Description
Design
Proof of Concept
Parts
Notebook
Achievements
Wet Lab
Protocols
Results
Safety
Interlab
Dry Lab
Modeling
Software
Human Practices
Human Practices
Silver
Gold
Integrated Practices
Public engagement
Team
Members
Attributions
Collaborations
DNA & Interactions
Cloning
Fluorescence detection
Protein purification
PROTOCOLS
Read More
DNA & Interactions
Getting Fluorescent DNA
You may want to get some fluorescent labelled DNA ? And can’t order it the way you want, here is a solution for you.
Material:
Amine modified DNA, here we chose /5AmMC6/ from IDT DNA.
FITC
Buffers: DMSO, 0.1 M Boron buffer adjusted to pH 10
Procedure:
Make a 1 mM of FITC solution in pure DMSO. Make sure to keep it away from light.
In a microcentrifuge tube gently mix 20 nmol DNA, 50 µL of 1 mM FITC solution and adjust  to 1 mL with boron buffer.
Incubate the mix overnight at 4°c, and you are good for purification step, see next paragraph.
How to purify DNA, a simple method
When you are in presence of a complex mix with some precious DNA you want to purify for further experiments, here is a solution.
Material:
The DNA mix to be purified/concentrated
LiCl 8M (for small DNA, under 100 nt) or Sodium Acetate 5M (classic DNA length)
Ethanol 100% and 70%, or Isopropanol 100% (for big volumes mixes)
Method:
Gently mix the DNA solution with 10% v/v LiCl or Na Acetate and 250% v/v ethanol 100% or isopropanol 100%.
Let the mix stand for 10 min.
Centrifugate at 13 000 rpm for at least 40 min at 4°C
Remove supernatant thoroughly, and add 1mL of ethanol 70%
Let the mix stand 10 min, then centrifugate in the sames conditions for 10 min.
Remove the supernatant very carefully and dry the pellet on air. You’re good to go !
Fluorescent ATP-Aptamer testing
To test  if the signal of the fluorescent aptamer could be specifically shut off in absence of the ATP target - the aptamer linked to fluorescein,  and the quencher fragment were hybridized in SSC 1X at 40°C for 20 min. That’s what we call the detection device. After a purification step, here’s the testing procedure:
Material:
The fluorescent detection device, dryed or in solution, 2.5 nmol is fine.
An 96 well plate, in black plastic.
Some PBS freshly made, pH = 7.1
ATP and GTP
Method:
Make a series of dilutions of ATP in PBS, we recommend to go from 10 mM to 1 µM. Store it on ice to prevent degradation.
Put 50 µL of each ATP dilutions per well, make triplicates. Don’t forget controls with plain PBS and a 10 mM GTP solution.
Dissolve the DNA in the appropriate volume of PBS, and quickly add 50 µL of this solution to each well with ATP or control solutions.
Measure fluorescence with a plate reader within 5 min to prevent bleaching.
Use the collected data !
Electrophoretic Mobility Shift Assay (EMSA)
EMSA are complicated experiments, with a lot of parameters to be adjusted.  Follow these general guidelines to adjust critical parameters (Temperature, pH, voltage, ionic composition).
Material:
An Aptamer and its proteic target.
TBE buffer 10X pH = 8.3
Folding buffer, according to the aptamer’s characteristics, often SSC 1X. Binding buffer (refer to the aptamer’s description for their compositions).
A native PAGE gel (12.5% Acrylamide, TBE 1X final, ions if needed for the interaction measured (Often Mg2+ or K+))
Method:
Dissolve the aptamer in the folding buffer, make sure to have a final concentration at least ten times lower than the expected Kd. Denature it at 99°C, then fold it 20 min at 42°C. Cool gently.
Make serials dilutions of target protein in the binding buffer.
Gently mix 16 µL of proteins with 4 µL of the folded aptamer. Let it stand for 20 min at room temperature.
Add loading dye and load 15 µL of the mixture in each well.
The gel migration should be done at 4°C and low voltage (100 V maximum).
Stain the gel with CYBR Green (50–100 times greater sensitivity than ethidium bromide) and photograph the gel.
Integrate the intensities of the bands thanks to an Imaging System. For each lane,calculate the ratio complexed DNA/free DNA. Plot  data and exploit them.
Latex beads experiments
Linking DNA and beads
Material
Dyed latex beads. They must have carboxylate groups in order to be reactive.
PBS buffer.
EDC, this carbodiimide is used as a crosslinking agent.
Sulfonic Acid Buffer 1 M, pH = 6.
DNA to be linked. It must have free amine group.
Method
Dilute the beads in PBS to get a 1 mL, 2% w/v solution. Centrifugate this at 15.000 g for 10 min.
Resuspend the beads in the sulfonic acid buffer, sonication (5 min at moderate power) may help. To this add quickly 20 mg of EDC, and let it stand for 15 min.
Centrifugate and resuspend in PBS buffer. Add  DNA, the amount needed is determined by the beads characteristics. It must be twice in excess. Incubate at least 2 hours.
Centrifugate the mixture and wash the pellet twice with PBS + Tween 0.01% w/v and BSA 0.1% w/v. The beads can be stored a few weeks in this buffer. Do not freeze them.
Make a control with a fluorescent DNA bearing an amine fonction.
Testing the beads with thrombin on multiplate
Material
Binding buffer: PBS buffer + 0,01 % Tween + 0.1 % BSA + 1 mM MgCl
Streptavidin coated multiplate.
Thrombin at different concentrations in the binding buffer.
Aptamers: 1) reverse complement of the aptamer targeting epitope 1 on the beads (RC),  2)  aptamer targeting a second epitope (A2). They must be biotinylated at one end (we chose 5’).
Latex beads with an aptamer fixed on them (A1).
Method
Thanks to the plate specifications estimate the amount of biotin it can link. Then put a two fold excess of RC and A2 in 100 µL PBS in each well, according to your experiment map. Don’t forget control wells which may be filled with PBS.
Let the mixture incubate 1 h at 30°C. Then wash the wells three times with the binding buffer. Your plate is ready.
In each well pour 20 µL of beads and 70 or 80 µL of binding buffer. Optionally add 10 µL Thrombin according to your experimental plan. The total volume in each well must be 100 µL.
Incubate this mixture on a rotating plate for 30 min at room temperature, then wash the wells with PBS.
Measure the OD600 of each wells. Because beads tend to be unequally spread, you must do at least triplicates. Don’t forget your control filled up with PBS.
This experiment is delicate because beads tends to flocculate. Ensure that they are well dispersed.
Preparing the papers strips for testing
Material
Nitrocellulose sheets, be sure to buy nitrocellulose with a high pore size
Streptavidin (you can use the purified parts from BBa_K1934030)
Aptamers, one for testing and one for control
Blocking buffer: 2% w/v milk powder + 0;01% Tween.
PBS
A plastifying machine (optional)
Method
Prepare a PBS solution containing 6 µM aptamer and 2 µM streptavidin. Incubate a room temperature for one hour.
Plastify the nitrocellulose sheet on one side only (optional)
Fill a crystal cone with the aptamers/streptavidin solution (10 µL), and draw bands on the paper as if using a pen. The faster you go the better the bands will be. Let it dry. You should depose around 10 µL of solution per cm (several passages may be needed).                                       Dry the nitrocellulose for two hours.
Incubate the strip in the blocking buffer for 20 min.
Dry the membrane again. Store in a cold, dry, dark place.
The paper pore size is a crucial factor. We were unsuccessful in further experiments because of this parameter.
Testing the beads on paper
Material
Nitrocellulose membrane with aptamers bands. Cut it in small strips (4mm width).
Latex beads with another aptamer linked
Buffer: PBS + Tween 0.01% + Triton 0.1% + MgCl 1 mM
Target protein to test.
Method
In an eppendorf tube mix 20 µL of the beads solution, 10 µL of the target and 30 µL of buffer. The target is optional if making a control. Let this stand for 3 min.
Dip 5 mm of the strip in this mixture and let the beads migrate.
Look for the bands.
cloning
Strains, Plasmids and Reagents
The plasmids used in the following manipulations were transformed into the strain NM522 of E. coli. This strain was grown in Luria-Bertrani (LB) meduim or in the SOC (Super Optimal Broth + glucose) medium or on agar Petri dishes - with the appropriate antibiotics (ampicillin [Amp] = 15 μg / μL, kanamycin [ Kan] = 15 μg / μL or chloramphenicol [Cm] = 15 μg / μL). The vectors used (pSB1C3, pSB1A3, pSB1K3), the parts referenced in Table 1 as well as the kit of competent cells (Competent Cell Test Kit) were provided by the iGEM Foundation. The restriction enzymes EcoRI, XbaI, SpeI and PstI were provided by the company New England Biolabs (NEB). The DNA concentrations were determined using the Nanodrop (ND-1000 Spectrophotometer UV-VIS, No. 5762 series LABTECH).
Preparing competent cells
Material
bacterial strains in liquid culture (37 ° C, 150 rpm) at OD 600> 0.2
cold TSS
ice
Method
Centrifuge 1 ml of culture at 8000 g for 2 minutes
Remove supernatant
Resuspend pellet in 100 uL of cold TSS
Incubate the tube on ice for 5 minutes
Use competent bacteria for transformation or for competent cell test
Note: All the operations of this protocol must be performed under sterile conditions.
Competent cell test
Material
70% ethanol
Ice
Competent cell aliquot(s)
Competent Cell Test Kit
Agar plates
Shaker
Incubator
SOC media
Method
Spin down the DNA tubes from the Competent Cell Test Kit to collect all of the DNA into the bottom of each tube prior to use.
Thaw competent cells on ice.
Pipet 1 µL of DNA into each microcentrifuge tube. For each concentration, use a separate tube.
Pipet 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes. Pre-heat waterbath now to 42°C.
Heat-shock the cells by placing into the waterbath for 1 minute.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes.
Add 200 µL of SOC media per tube, and incubate at 37°C for 2 hours
Pipet 20 µL from each tube onto the appropriate plate, and spread the mixture evenly across the plate. Do triplicates (3 each) of each tube. Incubate at 37°C overnight or approximately 16 hours. P
Count the number of colonies on a light field or a dark background and calculate competent cell’s efficiency using the following equation: (colonies on plate) / ng of DNA plated x 1000ng/µg
Note: The measurement "ng of DNA plated" refers to how much DNA was plated onto each agar plate, not the total amount of DNA used per transformation. You can calculate this number using the following equation: 1 µL x concentration of DNA (refer to vial) x (volume plated / total reaction volume)
Transformation
Material
Competent cells
Plasmid DNA
Ice
LB media
Petri dishes
Method
Throw competent cells on ice
Add 1 µl of resuspended DNA to 100 µl of competent cells in a microcentrifuge tube
Incubate tubes on ice for 30 minutes
Heat-shock tubes at 42°C for 50 seconds
Incubate on ice for 2 minutes
Add 900 µl of LB media (at room temperature) to each tube
Incubate at 37°C for 1 hour
Spread 100 µl of each transformation on a Petri dish
Centrifuge the rest of 900 µl, throw supernatant and resuspend the pellet in 100 µl of LB media; spread it on a Petri dish
Incubate the transformations over night at 37°C
Miniprep
Material
Plasmid DNA to be digested
Restriction enzymes
BSA
Digestion buffer
Method
Mix 100 ng of plasmid DNA (final volume of 4 µl) were mixed with 0.5 µl of BSA, 0.5 µl of each restriction enzyme, and 5 µl of the appropriate buffer (NEBuffer2)
Incubate the samples at 37°C for 30 minutes
Inactivate the enzymes at 80°C for 20 minutes
Ligation
Material
Linearized DNA backbone
DNA insert
T4 DNA ligase
Ligation buffer
Method
Mix 25 ng of the backbone with an equimolar amount of the fragment to be integrated, 0.5 µl of T4 DNA ligase and 1 µl of ligation buffer
Incubate at 16°C for 30 minutes
Inactivate the enzyme at 80°C for 20 minutes
IPTG screening
Material
Transformed bacteria
Agar plates
X-gal 2%
IPTG 100 mM
Method
Spread transformed bacteria on plates containing the corresponding antibiotic, 50 µl of X-gal 2% and 40 µl of IPTG 100 Mm
Incubate the transformations overnight at 37°C
Select white colonies for further analysis
Purification of DNA from agarose gel
Material
Agarose gel
TAE buffer
Ethidium bromide
UV lamp
Scalpel
QIAEX II Gel extraction kit
Method
Run sample on agarose gel in TAE buffer
Visualize DNA under UV using EtBr
Excise DNA with scalpel
Extract DNA using QIAEX II Gel extraction kit according to manufacturer’s protocol
PCR
PCR was performed using PrimeSTAR premix (Takara) or Taq DNA polymerase (NEB) according to each manufacturer’s protocol.
PCR purification
PCR was purified using QIAquick PCR Purification kit according to manufacturer’s protocol.
SLIC method
Material
Linearized DNA backbone
DNA insert
Oligomers
Appropriate buffer (NEBuffer2)
BSA 10X
T4 DNA polymerase
Ice
Competent cells
Method
Amplify insert by PCR using the oligomers (about 25 bp homologous to the ends of the vector)
Mix 100.5 ng of linearized vector, 7.5 ng of amplified insert (molar ratio 1: 2), 1 µl of NEBuffer2, 1 µl of 10X BSA and distilled water up to 10 µl
Add 0.5 µl of T4 DNA polymerase and incubate the solution at room temperature for 2.5 minutes, then on ice for 10 minutes
Transform 100 µl of competent cells with 5 µl of the previous solution
Computer-Aided Design
Computer-aided design in biology helps increasing the efficiency of biological manipulation and experimentation while easing the process of design and optimization. For our project we have used Serial Cloner, a free software. This software offers all the tools to manipulate and analyze DNA sequences (identification of restriction sites in a DNA sequence, simulation of DNA transcription to RNA and translation to protein, generating the inverse and/or complementary sequence, comparing two sequences together, etc.).
For each plasmid ordered, the following steps are followed:
Identification of parts of interest
Assembly of units and construction of the final plasmid on Serial Cloner
Check that the EcoRI, XbaI, SpeI PstI restriction sites are not present in the final sequence and if they are, remove them by silent mutation
Check that the spacing between the regulatory regions of the transcript and the TATAA box or between ribosome binding site and the initiation codon ATG are well respected
Codon optimization
Order
Fluorescence detection
Fluorescence range
A range of calibration of 9 samples of fluorescein with increasing concentrations (10-6, 10-5, 10-4, 10-3, 5.10-3, 10-2, 5.10-2, 10-1 and 1 mM) is prepared in three types of solvents: PBS, NaCl and distilled water. The use of these three solvents in parallel allows us to test the difference in fluorescence between an isotonic medium (NaCl, PBS) or a non-isotonic one (distilled water). In a 96 well plate, 25 µL of whole blood or serum were added to 25 µL of fluorescein in solution. Four calibration ranges assays were set:
Fluorescein (dissolved in NaCl) with whole blood,
Fluorescein (dissolved in distilled water) with whole blood,
Fluorescein (dissolved in PBS) with whole blood,
Fluorescein (dissolved in PBS) with serum.
Blanks are obtained by mixing 25 µL of whole blood/serum with 25 µL of PBS / NaCl / distilled water. Each concentration was tested in triplicate. The emitted fluorescence was measured using the plate reader Chameleon (Multilabel reader flat Chameleon - Hidex, No. 2,060,003 series SCIENCETEC) on 3 different times t = 0 min t = 10 min and t = 20 min in order to observe the kinetic of fluorescence.
Revelation with light filters
A calibration range consisting of 4 samples of fluorescein with increasing concentrations (10-3, 10-2, 10-1 and 1 mM) was prepared in distilled water. Blank corresponds to a solution of distilled water only. The solutions were placed into spectrophotometer 1 mL cuves and the emitted fluorescence was visualized using a smartphone with a combination of two filters. The first filter which has the transmitted wavelength corresponding to the maximum emission of fluorescein (512 nm), is positioned on the lens of the camera. The second filter, which has the transmitted wavelength corresponding to the absorption maximum of fluorescein (494 nm), is fixed on the flash. The picture is then taken under different conditions: combination of two filters, the filter on the single flash, the filter on the lens only and without any filter.
Protein purification
Purifying HisTagged proteins on NiNTA columns
Material
Quiagen NiNTA columns kit
NTI-10, NTI-20 and NTI-500 buffers
Expression culture of the protein
Method
Resuspend a pellet derived from 5 ml cell culture volume in 630 μl Lysis Buffer (NPI-10). Add 70 μl Lysozyme Stock Solution (10 mg/ml) and add 3 Units/ml culture volume Benzonase® Nuclease (i.e., for cell pellets from 5 ml cultures, add 15 Units Benzonase® Nuclease).
Incubate at room temperature for 15–30 min.
Centrifuge lysate at 12,000 x g for 15–30 min at 4°C. Collect supernatant.
Equilibrate the Ni-NTA spin column with 600 μl Buffer NPI-10. Centrifuge for 2 min at 890 x g . The spin columns should be centrifuged with an open lid to ensure that the centrifugation step is completed after 2 min.
Load up to 600 μl of the cleared lysate containing the 6xHis-tagged protein onto the pre-equilibrated Ni-NTA spin column. Centrifuge for 5 min at 270 x g (approx. 1600 rpm), and collect the flow-through.
Wash the Ni-NTA spin column twice with 600 μl Buffer NPI-20. Centrifuge for 2 min at 890 x g (approx. 2900 rpm).
Elute the protein twice with 300 μl Buffer NPI-500. Centrifuge for 2 min at 890 x g (approx. 2900 rpm), and collect the eluate
Purifying CBD-tagged proteins on cellulose
Material
Microcrystalline Cellulose (eg Avicell)
Wash buffer: Amonium Sulfate 1 M
Expression culture of the protein
Lysis buffer: 50 mM Tris, pH = 8 + 300 mM NaCl + 10% glycerol
Method
Wash the Cellulose five time in water. The equilibrate it in wash buffer.
Pellet the culture and resuspend it in 1 mL of lysis buffer. Then sonicate it, on ice, five times at moderate power. Centrifugate the lysate at 14.000 g for 10 min.
Pack the cellulose (10x10 mm) in small chromatography columns (we used syringes barrels).
Gently pour the lysate supernatant on the column. Once the liquid starts to flow out regularly measure the OD280 of the different fractions.
Continue by pouring wash buffer until the OD280 stabilizes around zero.
Change washing buffer to water. After a short moment the OD280 should rise, collect the hight OD 280 fractions. They should contain the protein.
If the previous step was not efficient, continue elution with 6M Urea.
Analyse collected fractions on an SDS PAGE.
Optionally proteins may be concentrated using ultrafiltration.
For more informations you can consult our full bibliography here
Designed by Team iGEM INSA-LYON
