Team:UCSC/Logs
Home
Team
Meet the Team
Breakdown
Fermentation
Field
Project
Human Practices
Outreach
Bioethics
Notebook
Logs
Protocols
Safety
Achievements
Parts
Medal Fulfillment
Results
Software
Daily Logs
To view our Protocols click here
Breakdown Team
June 2015
 
Wednesday, June 24
Discussed overall project goals for the research period
Need to test the efficiency of multiple cellulases
preferably those that are already present in halophiles and is compatible with Haloferax volcanii
identified the positions of the four different cellulases associated with Halorhabdus utahensis
Cellulosome construction possibility
 
Thursday, June 25
Looked through the halophile phylogenetic tree and found that Haloquadratum walsbyi (Hwa) is closer to Haloferax volcanii, and has cellulase that occurs in the natural metabolic pathway as well
Did a pairwise alignment between Halorhabdus utahensis (Hut) and Haloquadratum walsbyi (Hwa)
Concluded that it was probably not a good idea to risk using Haloquadratum walsbyi since it only contains a “probable” cellulase, and it does not contain any other beta-glucanase enzymes
Friday, June 26
Learned how to design primers for Gibson Assembly
Have two potential cellulase candidates
one from H.tiamaten
one from H.utahensis
Using pTA963 as an expression plasmid
Most of the cellulases seen need C-terminal His Tag since the N-terminus is found in the transmembrane
Need to use inverse PCR to linearize and amplify plasmid. This will also remove 6x His Tag and stop codon
His Tag and stop codon will be added using flagged primers
Introduced to idea of codon optimization
Worked on the pI Finder program, which takes in a FASTA amino acid sequence and calculates the theoretical isoelectric point
Narrowed criteria for usable cellulases:
Signal Peptide for excretion or transmembrane position
pI range between 4 and 5
Monday, June 29
Worked with different plasmid editors and decided on Geneious
Narrowed down the cellulase candidates from CAZy with respect to the signal peptide sequences
Introduction to PredSignal. Software that uses hidden markov models to predict the presence and purpose of archaeal signal peptides
Selection options for testing with cellulose:
Rayon
Filter paper that’s been bleached
Avicel microcrystalline cellulose
Tuesday, June 30
Continued the narrowing down of the cellulase candidates with respect to the presence of signal peptide
Found the sequence of the cellulase (Hu-CBH1) from H. utahensis that has been proven to work in the paper “Identification of a haloalkaliphilic and thermostable cellulase with improved ionic liquid tolerance” Zhang, Tao et al
 
July 2015
 
Wednesday, July 1
Have enzymes categories as beta glucanase and alpha amylase, but also looking for endo glucanase and exo glucanase
Discuss whether or not we can utilize the cellobiohydrolase Hu-CBH1 since it is patented
 
Thursday, July 2
Weekly team meeting (Moved to Thursday due to 4th of July Weekend)
Discussed a more narrow focus on Exoglucanases, Endoglucanases and B-glucosidases
We did not want to focus on Alpha amylases since they are used for breaking down starch
Monday, July 6
Design protocols for inverse PCR and Gibson Assembly
Performed inverse PCR to linearize and amplify our expression plasmid (pta963)
Protocols labelled as “ Polymerase Chain Reaction (PCR) for Amplification of pTA963 Expression Plasmids” and “Gibson Assembly Protocol”
 
Tuesday, July 7
Acquired Q5 2x Master mix for PCR reactions
H.tiamaten cellulase → homologs present, but still need to find the extension number
Gene is about 2,700 nucleotides long x 3 ~ 8,100
We have about 4 gene candidates + H.tiamaten cellulase that has already been tested
 
Wednesday, July 8
GOALS
Thursday 7/9 → Inverse PCR (2 experiments)
Friday 7/10 → Analysis - Gel/ plate prep
Wednesday 7/15 → Gibson/ PCR Analysis/ Gel, Transformation of E.coli
pick a strain of E.coli
chemically competent
Thursday 7/16 → Plate
Monday 7/20 Mini-Prep/ Sequencing Prep
 
Thursday, July 9
Need to prepare gene blocks for:
C-Terminus                            N-Terminus                                  ?
-Shewanella                           -Shewanella               -cellobiose phosphorylase
-tiamaten
-Hu-CBH1   
-b-glucosidase (hispanica)     -H.tiamaten               -B 1,4 glucanase (utahensis)
 
Friday, July 10
While choosing cellulase candidates
Checked for signal peptides
Searched through the archaeal genome browser
As of now we are deciding between: H.utahensis, H.hispanica, H.tiamaten
Found that we already have stock of H.hispanica in the freezer
Saturday, July 11
Meeting to design gene blocks for:
Shewenella fusion gene
Hu-CBH1 cellobiohydrolase
Cellobiose phosphorylase
Pyruvate decarboxylase
Note: Used IDT Gene Block Editor/ oligoanalyzer 3.1
Picture
Monday, July 13
TBE buffer was autoclaved this morning
Ran inverse PCR to amplify N-term pTA 963 and ran on a gel
Professor Bernick teaching PCR Fragment Assembly, Gibson Assembly and Flagged primers using sticks
Picture
Tuesday, July 14
Reran inverse PCR, changing annealing temperature and extension time
Grant Team Meeting:
UCSC Crowdfunding page
Reaching out to department chairs
Created program for codon optimizing proteins using the codon frequency table and amino acid sequence (Called optimizer.py)
 
Wednesday, July 15
Found protocol for making growth media for H. Hispanica in the Halohandbook pg. 14
23% Modified Growth Medium (MGM)
 
Thursday, July 16
Protocol for DNA isolation for H.hispanica (page 69 of Halohandbook)
Found that DNAWorks algorithm for codon optimization removes accuracy
It does not take into account “rare” codons
Friday, July 17
Weekly team meeting
Use of DNA works for codon optimization
Inverse PCR for amplification of C-term pta963 plasmid
Paused during extension phase because we forgot touchdown conditions, which may greatly affect results
 
Monday, July 20
N-term and 2 C-term PCRs completed  
EMERGENCY LAB MEETUP:
Breakthrough with codon optimization code (FOCUS)
Want to pay attention to conserved rare codons
Need to look for this pattern across organism of different domains
3 genes to test (All different forms of B. Glucosidase)
Wild type (H.hispanica) - control
optimized (DNA Works) + control
optimized (Jairo’s code) F.O.C.U.S
Assay
micro-crystalline cellulose and X-Glu plates to test for enzymatic activity
Professor Bernick explanation of more to test for Signal Peptide:
psortB
TM Pred
Introduction to the UCSC archaeal Genome Browser
Intro to Kasava Research Paper
 
Tuesday, July 21
EMERGENCY BREAKDOWN TEAM MEETUP:
Our hypothesis: Rare and conserved codons allows better folding of the protein structures = more efficient protein production
Obtain the secondary structures of the protein sequences through PSS Pred
Decided to codon optimize our beta glucosidase protein without optimizing the signal peptide since we know signal peptide still works based on the Hu-CBH1 paper
 
Wednesday, July 22
Looking for 2,565 bp band for beta-glucosidase gene in Haloarcula hispanica
Continued runs for N-term and C-term PCR
 
Thursday, July 23
Streaking in C-term (7/21) due to pause while running
too much primer for the N-term; we used 5 uL instead of 1.25 uL
codon bias for H.volcanii and H.hispanica are similar
completed another N-term and C-term reaction
working on perfecting conditions
Friday, July 24
General Meeting
Introducing entire team to F.O.C.U.S
Plan to test the protein expression levels of the beta-glucosidase from Haloarcula hispanica between the wild type, DNA Works Codon Optimized and F.O.C.U.S Codon optimized
Need to prove that the rate limiting step is the incorporation of a particular tRNA
Working out the kinks for isolation of wild type beta glucosidase
Smears may be due to a high concentration of genomic DNA during isolation
Use of DPN1 could get rid of non-methylated (i.e non-genomic DNA) which could mean more access to PCR product and amplification
 
Monday, July 27
Multiple sequence alignment of conserved proteins conserved in Archaea, Bacteria, and Eukaryotes
Alanyl-tRNA, DNA polymerase III subunit, rRNA dimethylase   
Further proteins amongst model organisms can be found in the paper “ Universal trees based on large combined protein” Brown, James R. et al
Tuesday, July 28
Searched for protein sequences and DNA sequences of conserved proteins in model organisms
Note, some protein sequences in certain organisms are not of equal length to others. Therefore, they may not be orthologs
Inverse PCR to amplify N-terminal version of plasmid pta963
Wednesday, July 29
Need to regrow Hispanica to the optimal OD (0.6 - 0.8) for wild type beta glucosidase isolation
 
August 2015
 
Monday, August 3
Gel for hispanica isolation did not work on Friday
Tested to see if the track dye and DNA ladder show up under UV light
Reran another gel to test if the results were due to track dye or gel
Redid another isolation since results were negative
 
Tuesday, August  4
Isolation of hispanica (2nd time)
50 uL Rxn (Iso, and negative control)
Turned attention towards using the Dali Server for protein structure information and structural alignment comparisons
Prepared protocol for E. coli cell electroporation
Wednesday, August 5
Isolated Hispanica from a different aliquot (Still with an OD of 1.4)
Multiple PCR reactions to find optimal conditions for isolating wild type b-glucosidase (Run at 71 degrees)
Negative control (No DNA)
Repeat of the same PCR protocol but NO PCR ENHANCER
2x Primers (No PCR Enhancer)
2x Template ( NO PCR Enhancer)
Amplification of the first isolation that worked (tube labelled ++)
 
Thursday, August 6
Redid PCR for gene isolation (Brought the annealing temperature to 66 degrees)
 
Friday, August 7
Performed isolation from a 3rd aliquot of Hispanica
Performed PCR after diluting concentration 100 fold for both aliquot 1 and 3
Meeting with Professor Bernick to discuss one method of measuring importance of rare codons for F.O.C.U.S
Encoding cost (Method of hyper weighting a codon that is more rare, especially if it is a rare codon in a group of more frequent codons)
Need to develop a training set and test set to measure consistency of rare codon idea
Prepare sequence information for engineered cellulase developed by 2014 UCSC iGEM team to perform site directed mutagenesis
 
Tuesday, August 11
Fragment assembly of codon optimized Beta-Glucosidase
Fragment concentrations from stock:
F1 → 0.88 uL,  F2 → 1.25 uL,  F3 → 1 uL
The Fw fixed Flag primer was diluted with 216 uL of TE buffer, vortexed, and stored
Primers used: Fw Flag Fixed5 and Bglu_ Rv flag
Nested PCR part 2 (adding flag primers to the isolated WT beta Glucosidas
Primers: Fw Flag fixed5 and Bglu_ WT_Rv flag
Proteins for F.O.C.U.S
EF-G, Ef-Tu, DNA Polymerase III, tRNA synthetase
Dali server search stored on Kerika
 
Wednesday, August 12
Working on getting the codon bias tables and nucleotide sequences to test out FOCUS
Reinoculated Hispanica to isolate it from a lower OD
Tested Fragment assembly with 1/10 dilution of fragments and non dilution
The lightbulb in the UV transilluminator broke, so weren’t able to capture a picture of the gel
 
Thursday, August 13
General Meeting
check 260/280 and 260/230 for the isolation
absorbance of nucleic acids/ protein
Run fragment assembly with 1/5 dilution, not 1/10
Length of the fragment assembly =  2641 bp
Length of the wild type = 2644 bp
Isolation of Hispanica from a culture with an OD of 0.4
concentration: 116.4 ng/uL
260/280: 1.64
260/230: 0.59
WT isolation and fragment assembly PCR
dilution (1:10) of new isolation
diluted fragments and non-diluted fragments
Annealing temp= 63.8 degrees
 
Friday, August 14
Isolation PCR reaction using isolation from OD 0.4 and 1.4 to see which is better
 
Sunday, August 16
Running the gel from Friday’s PCR reaction
The isolation with an OD of 0.4 worked the best!
dilute fragments reaction does not work
lower the annealing temperature
used an annealing temperature of 62.8 degrees
Elongation time changed to 45 seconds
Q5 reads 1000 bp every 15-45 seconds
PCR Reactions
0.4, 1.4, nd, d, i, F, …, _
Monday, August 17
Prepared 500 mL of LB agar in preparation for growing transformed E. coli cultures
Protocol: https://www.addgene.org/plasmid-protocols/bacterial-plates/
nanodrop for linearized PTA963
Conc. 260/280 260/230
C-term (7/21) 462.7 ng/uL 1.82 0.87
C-term (7/23) 561.9 ng/uL 1.83 0.93
N-term (7/23) 794.6 ng/uL 1.84 1.24
N-term (7/28) 335.3 ng/uL 1.84 0.74
More PCR reactions
0.4 flag, annealing temp @ 70 degrees → Today
1.4 re-isolation, run @ 71 degrees → Tomorrow
Fragment Assembly @ 65 degrees → Tomorrow
 
Tuesday, August 18
Bright band present for fragment assembly = SUCCESS!
Performed purification using Bernick Kit. Results:
         DNA       Conc.          260/280    260/230
N1        32.2ng/ul       1.58           0.14
N2        7.4ng/ul         2.07           0.03
C1        6.1ng/ul         1.76           0.02
C2        14.5 ng/ul      1.57           0.06
 
Wednesday, August 19
Re-running two isolation reactions with an annealing temp of 71 C and 42 sec extension
Gel extraction of fragment assembly to run Gibson assembly
 
Thursday, August 20
PCR reactions (Annealing temp. 65 degrees)
gel-extracted iso
gel-extracted frag (dil)
non gel-extracted iso
non-dil frag
 
Friday, August 21
To Do:
Confirm correct isolation band
redo fragment assembly
1 uL each fragment (10 uM)
correct concentrations (0.88 uL,1.25 uL,1uL)
65 degrees annealing temperature
Details for fragment assembly that was gel extracted:
19 uL at concentration of 66.4 ng/uL
260/280: 2.03
260/230: 0.61
Gibson Assembly
2.65 uL of C-term pta963
1.2 uL of Fragment Assembly
 
Saturday, August 22
Inverse PCR of N-Terminal pta963 for Ethanol Team
Transformed NEB 5-alpha competent E. coli with Gibson Assembly product
Attempted to isolate which band corresponded to the wild type Beta Glucosidase
Fw_Flg Fixed5 and Seq Rv3 primers, looking for band of 1000
 
Sunday, August 23
Checked transformation plate, put found no growth
Most likely due to cells having lost competence since they were not stored at -80 C for at least a week
Prepared reagents for making spheroplast
 
Monday, August 24
Made “Unbuffered spheroplasting solution”, “Buffered spheroplasting solution” and “Regeneration solution”
Protocol found on pgs. 59-62 of the Halohandbook
Redid inverse PCR of N-term pTA963
Elongation lowered to 2 min and 30 sec
 
Wednesday, August 26
Electroporation using electrocompetent E. coli cells
1800 volts
Used Electroporation protocol made by Fermentation team
Ag Tech Meet up Presentation
 
Thursday, August 27
Nested PCR Part 2
Used Seq/iso reaction product from 8/27 as template DNA since it has the brightest band corresponding to the wild type beta glucosidase
Concentration: 472.2 ng/uL ; 262/280: 1.79  ; 260/230: 0.68
Used Fw Flag Fixed5, Rv Flag and Rv Flag fixed as primers
 
Monday, August 31
Colony PCR from Transformation plate
Picked 6 colonies
Used miliq water to lyse cells (Tubes labelled 1P-6P)
Used Flagged primers: Fw Flag Fixed 5 and Rv Flag Fixed
Ran another reaction using sequencing primers: Seq Fw2 and Seq Rv2
 
September 2015
 
Tuesday, September 1
Meeting with the Dean of Students
Rerun a gel including fermentation team colony PCR samples and Breakdown sample (D - dilute, and ND- non-Dilute) of wild type isolation test
Made 236.25 uL of Q5 Master Mix
Enough for 27 reactions if we use 25 uL PCR reactions
Recipe (X10):
20 uL Q5 Buffer
2 uL of dNTP’s
0.5 uL of Q5 enzyme
11.25 uL of Miliq water
 
Wednesday, September 2
Colony PCR (4 reactions):
P2 with iso Primers and PCR enhancer (1:10 dilution)
P2 with iso Primers, No PCR enhancer (1:10 dilution)
P2 with Flagged Primers, PCR enhancer (1:10 dilution)
P2 with flagged Primers, No PCR enhancer (1:10 dilution)
Electroporation of Fermentation Gibson reactions (G2, G6, G7, and G8)
Colony PCR of P1 - P6 using Sequencing FW1 and Rv4 primers
Annealing Temp: 59 C
Extension time: 3 minutes
Need to prepare HVCA plates for Spheroplast analysis after transformation
Protocol for HVCA agar on Pg. 21 of Halohandbook
Transformants should grow on plates without Uracil
 
Friday, September 4
General Meeting
FOCUS Discussion
Transition Probability Modeling
Only need to produce a model with 2 states: fast and stall (excluding steady state)
John has a paper about “modeling ribosomal speed”
Colony PCR of P1 - P6
Dominic’s plasmid specific primers
Annealing Temp. 51
Inoculated H. volcanni for making spheroplast
Optimal absorbance reading at 600 between 0.8- 1.0 for late exponential phase
Nested PCR part 2
Template (0.4 isolation from 8/13, 8/14 and 8/17) to test which is best
Primers: Bglu_Fw_flagFixed5 and Bglu_wt_rv_fixedFlag
Annealing temp. : 69 C as suggested by TM Calculator
 
Saturday, September 5
Absorbance reading of inoculated H. volcanii
0.621 A
Plated Fermentation Team Electroporated Cells ( G2, G6, G7, and G8)
 
Monday, September 7
Colony PCR positive controls (using Gibson Assembly reaction as template)
Sequencing Primers Fw1 and Rv4 (Titaq)   Touchdown: 66 - 59 C
Sequencing Primers Fw1 and Rv4 (OneTaq)
Dominic’s Primers Aldy5Seq1F and Aldy5Seq5R (Titaq) Touchdown 66 - 59 C
Dominic’s Primers Aldy5Seq1F and Aldy5Seq5R (OneTaq)
Wednesday, September 9
Chose 8 new colony picks from original plate having Transformed E. coli cells with codon optimized B Glucosidase
Performed colony PCR using Dominic’s primers, Titaq and No PCR Enhancer
Thursday, September 10
Started Preparing BioBricks of Codon Optimized Beta Glucosidase, and Fermentation enzymes
Added Ampicilin to LB Agar plates and replated electroporated E. coli cells
Added 26.3 ul of 50 ng/uL ampicillin to each plate
Colony PCR using Q5 Master Mix
3 Reactions from fermentation and 2 from Breakdown
Friday, September 11
Professor Bernick Discussion
Dislikes homopolymer in both Breakdown and Dominic’s Primers
Prefers Dominic’s primers because they have a base change after the homopolymer
Suggests 10 cycle touchdown from 56 - 51 using Dominic’s Primers, using Titaq and PCR enhancer
Saturday, September 12
Verification that primers anneal to the C-term construct of pTA963
Aldy5Fw1 anneals from 485 - 508 (15 nucleotides from His Tag)
Aldy5Rv5 anneals from 569 - 587 (31 nucleotides from His Tag)
SeqFw1 anneals fromm 462 to 486
SeqRv4 anneals from 570 - 593 (Note, first G is supposed to be a C)
Might have to redo Gibson Assembly of fragment assembly based on positive control results
 
Sunday, September 13
Touchdown of Nested PCR Part 2
Annealing temp from 70 - 65 C
Fermentation: Butanol
June 2015
 
Wednesday, June 24
Outlined overall project goals
Used NCBI’s nucleotide best BLAST to find gene orthologs from the Garcia pathway in H. volcanii.
Found all genes in H. volcanii except for butyraldehyde dehydrogenase and butanol dehydrogenase.
Used BLAST to find other butyraldehyde and butanol dehydrogenases that could be used in H. volcanii.
Considered using an aldehyde/alcohol dehydrogenase fusion gene.
Found a promising fusion gene in Halomonas sp. S2151 called adhE2. Also had a pI near 5, which is ideal for H. volcanii.
Little was known about Halomonas and the gene was only recently discovered.
Considered using gas chromatography to confirm the presence of butyryl-CoA.
 
Thursday, June 25
Continued to pursue the idea of using gas chromatography to test for butyryl-CoA.
Rob Franks agreed to allow us to use the GC.
Searched for places to purchase butyryl-CoA to create a standard for GC, but it’s very expensive.
Started searching for protocols for gas chromatography of butyryl-CoA and more generally, fatty acids.
 
Friday, June 26
Created 10 L of 30% salt water stock to be used in producing media for H. volcanii.
Used protocol in pg 20 of Halohandbook.
Prepared Tris buffer (500 mL of pH 8.83 and 40 mL of pH 7.54).
 
Monday, June 29
Continued searching for gas chromatography protocols specific to butyryl-CoA.
Considered using qPCR, since finding a protocol is a challenge.
Sources suggest using HPLC to detect butyryl-CoA.
Rob Franks believes HPLC will be better suited for butyryl-CoA.
Found a site with potentially helpful primer design rules ( http://www.premierbiosoft.com/tech_notes/PCR_Primer_Design.html )
 
Tuesday, June 30
Decided to stop pursuing gas chromatography of butyryl-CoA. We are assuming that H. volcanii produces it at this point.
MGM plates have been prepared according to pg 14 of Halohandbook.
 
July 2015
 
Wednesday, July 1
Proposed the idea of using beads that bind to CoA groups to detect the presence of butyryl-CoA.
Trying to find a good promoter to increase the gene expression of the genes we decide to use.
Several searches yielded the promsing ptb promoter.
Considered using rational design to lower the pI of the fusion protein to near 4.5. Requires generating a model structure using SWISS-MODELLER.
Unable to find a 3D model of adhE2.
Jairo used his program to codon optimize the adhE2 gene for H. volcanii.
 
Thursday, July 2
Decided on the pTA963 plasmid for eventual expression of our fusion gene.
Professor Bernick urged us to focus on pI rather than percent identity of potential fusion genes.
Found yet another alternative fusion gene in Shewanella waksmanii ( http://www.ncbi.nlm.nih.gov/nuccore/655366186 ).
Found the nucleotide sequence for this fusion gene.
Discontinuing work on the Halomonas fusion gene adhE2.
 
Monday, July 6
Attempting to find a protocol for inserting the S. waksmanii fusion gene into the expression plasmid.
Dominic suggested using Gibson assembly.
Tried to codon optimize the S. waksmanii gene using Jairo’s program, but it ran into a syntax error which couldn’t be fixed.
Asked Professor Bernick to contact an expert in S. waksmanii (Chad Saltikov).
Reconsidered the ptb promoter as a backup.
Found the bgaH gene, which is used to assay the promoters of haloarchaeal genes.
Found a similar fusion gene in Shewanella sediminis.
Has a lower pI.
Unable to find the entire genome of S. waksmanii for codon optimization.
The full genome for S. sediminis is available, allowing codon optimization.
 
Tuesday, July 7
Professor Bernick approved the idea of using the S. sediminis fusion gene.
The fusion gene was codon optimized for H. volcanii.
Checked nucleotide sequence for the formation of structures that would affect Gibson assembly.
Split the S. sediminis fusion gene into smaller 500 - 700 bp fragments for ordering.
Fragments >1000 bp take longer and are more expensive.
 
Wednesday, July 8
Finished splitting the S. sediminis fusion gene into 3 gene fragments.
Began designing primers for the pTA963 plasmid and the His-tags.
 
Thursday, July 9
Plasmid primers were ordered.
Added overlaps on the S. sediminis fusion gene for both N- and C-terminal His tags.
 
Friday, July 10
Continuing to pursue the bgaH gene as a backup.
Started working on primers for fusion gene.
 
Saturday, July 11
Finished fusion gene primers.
 
Monday, July 13
Waited for approval of S. sediminis primers for ordering.
Ran inverse PCR on the pTA963 plasmid.
 
Tuesday, July 14
Checked optimized gene sequence to ensure that resulting amino acid sequence matches.
Did not match, so the program needs refinement.
 
Wednesday, July 15
Jairo attempted to fix codon optimization program.
 
Thursday, July 16
Jairo fixed his program, however Professor Bernick urged us to use DNAWorks for codon optimization.
Discontinued work on the ptb promoter, in the interest of time.
 
Friday, July 17
Plasmid N- and C-terminal protocols were accidentally swapped and were redone.
Continued to refine the S. sediminis gene fragments and primers.
Need to manually reduce the GC content for ordering.
 
Monday, July 20
Continued to work on the S. sediminis fragments and primers.
 
Tuesday, July 21
Still working on the S. sediminis fragments and primers.
According to Timpson et al., adh2 is an alcohol dehydrogenase in H. volcanii that shows high rates of butanol production.
Butanol production via reduction only occurs in acidic conditions.
The reverse oxidation reaction occurs in alkaline conditions.
Since H. volcanii is grown in alkaline conditions, oxidation is favored.
Considered growing H. volcanii and acidifying the environment.
 
H. volcanii also has two aldehyde dehydrogenases (aldY3 and aldY5).
 
It’s unclear which results in more butanol production.
Shifted the project to focus on developing two fusion genes consisting of adh2 and each of the aldehyde dehydrogenases.
Started researching linker sequences to connect the two dehydrogenase genes.
 
Wednesday, July 22
Finalized the S. sediminis fragments and primers.
Made minimal media for growing H. volcanii anaerobically.
Will give several days for colonies to grow, before acidifying the environment.
Reduced the number of rare codons in the S. sediminis gene.
Contacted Rob Franks about gas chromatography usage.
He said it should be able to handle slightly acidic conditions.
Need to prepare standards before running samples.
Looked up sequences of aldY3, aldY5, and adh2 on KEGG.
 
Thursday, July 23
Prepared the S. sediminis fragments and primers for ordering.
Started researching linker sequences to connect the two dehydrogenase genes.
Found a paper that describes characteristics of good linkers ( http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3726540/ )
Professor Bernick urged us to find natural linkers rather than designing our own.
 
Friday, July 24
??????
 
Monday, July 27
Designed primers with Dominic’s help for amplifying aldY3 and adh2.
Tried using Pfam to view the domains of fusion proteins to obtain natural linker sequences.
 
Tuesday, July 28
Used InterPro to visualize the linker region in natural fusion genes.
Met with Professor Bernick and established a long-term game plan:
Design primers for dehydrogenase genes such that they incorporate the linker region.
Use Gibson assembly to incorporate the dehydrogenase genes and linker into the pTA963 plasmid.
Transform the Gibson product into E. coli, and determine which colonies were successful in taking up the fusion gene by colony PCR.
Design sequencing primers and send product off for Sanger sequencing.
Use nested PCR on the colony PCR product.
Dominic confirmed that he also has adh2 primers, so we only designed the aldY3 primers.
Continued searching for linker sequences.
 
Wednesday, July 29
Several linker regions were designed from scratch. Started to incorporate overlap with dehydrogenase genes.
Other linkers were found to be conserved across several species.
Finalized aldY3 primers for ordering.
Settled on 4 linkers: 2 designed and 2 conserved.
 
Thursday, July 30
Received the S. sediminis primers and diluted them to 100 uM.
 
August 2015
 
Monday, August 3
Waiting for S. sediminis gene blocks.
The anaerobic cultures haven’t grown much - pH still at 7.5.
 
Tuesday, August 4
Received native gene primers. These primers were also diluted to 100 uM.
Genomic DNA isolated from H. volcanii
Followed protocol in pg 70 of Halohandbook.
 
Wednesday, August 5
Ran PCR on genomic DNA to isolate aldY5 (annealing temp 71℃).
Extension time should’ve been 1 min, but was only 30 sec.
Ran a gel on the PCR product - mostly primers showed up with little to no PCR product.
 
Thursday, August 6
Redid PCR of aldY5 and aldY3 using these conditions:
 
aldY3 and aldY5 were successful, adh2 was not.
 
Received the S. sediminis gene blocks.
 
Friday, August 7
Ran PCR again on aldY5 and aldY3 using these conditions:
Used a touchdown PCR on adh2 for more specific primer-binding (decrease annealing temperature by 0.5 ℃ each cycle).
The conditions were successful
Saturday, August 8
Re-ran PCR on adh2.
Ran PCR on the 8 gene/linker fragments (71℃ annealing, 1 min extension, 2 min final hold)
aldy3BFw & aldy3Blink1Rev = A
aldy5BFw & aldy5Blink1Rev = B
aldy3BFw & aldy3Blink2Rev = C
aldy5BFw & aldy5Blink2Rev = D
aldy3BFw & aldy3Blink3Rev = E
aldy5BFw & aldy5Blink3Rev = F
aldy3BFw & aldy3Blink4Rev = G
aldy5BFw & aldy5Blink4Rev = H
 
Monday, August 10
Unable to proceed due to a lack of supplies.
 
Tuesday, August 11
Still unable to proceed further due to a lack of supplies.
Anaerobic cultures appear to be growing.
Started to prepare minimal HEPES media rather than Tris (pH 6.55).
 
Wednesday, August 12
Continuing to make minimal media.
Considering eliminating potassium phosphate to avoid precipitate formation.
Professor Bernick suggested micro-aerobic cultures, in which a small amount of oxygen is injected into the culture vessel.
May help solve the redox imbalance from the lack of oxygen as a terminal electron acceptor.
Assembled S. sediminis gene fragments.
Ran a gel with aldY5/adh2 and S. sediminis fragments.
 
Thursday, August 13
The S. sediminis fragments were re-assembled (increased annealing temperature to 67℃).
Gene/linker sequences also ran again.
Key:
Lane1: aldy5BFw & aldy5Blink4Rev = H
Lane2: adh2Dlink1Fw & adh2DRev = I
Lane3: adh2Dlink2Fw & adh2DRev = J
Lane4: adh2Dlink3Fw & adh2DRev = K
Lane5: adh2Dlink4Fw & adh2DRev = L
PCR conditions:
 
Friday, August 14
Assembled the aldY and adh2 linker fragments.
aldy3BFw & aldy3Blink1Rev & adh2Dlink1Fw & adh2DRev = F1
aldy5BFw & aldy5Blink1Rev & adh2Dlink1Fw & adh2DRev = F2
aldy3BFw & aldy3Blink2Rev & adh2Dlink2Fw & adh2DRev = F3
aldy5BFw & aldy5Blink2Rev & adh2Dlink2Fw & adh2DRev = F4
aldy3BFw & aldy3Blink3Rev & adh2Dlink3Fw & adh2DRev = F5
aldy5BFw & aldy5Blink3Rev & adh2Dlink3Fw & adh2DRev = F6
aldy3BFw & aldy3Blink4Rev & adh2Dlink4Fw & adh2DRev = F7
aldy5BFw & aldy5Blink4Rev & adh2Dlink4Fw & adh2DRev = F8
PCR conditions for fragment assembly:
Samples were purified using a DNA cleanup kit, but purification was unsuccessful (see last lane).
 
Sunday, August 16
Attempted to determine why the cleanup kit didn’t work.
Tested the reagents in the kit.
Still unsuccessful.
Decided to not assemble fragments and then use Gibson assembly, but to go straight into Gibson assembly and see if the fragments assemble properly before annealing to the plasmid.
 
Monday, August 17
Continued to work on cleanup kit.
Sample G1 was successfully cleaned up.
Samples B, E, F, L need to be re-cleaned, from the kit mishap.
Despite cleanup, these samples had a low 260/230 value.
Unsure whether to use N-terminal or C-terminal plasmid for our Gibson reaction.
 
Tuesday, August 18
Confirmed that the N-terminal plasmid is required for Gibson of native fusion genes.
Considered the idea of testing cultures for fermentation products such as lactic acid.
Rob Franks suggested using HPLC, since lactic acid won’t run in the column.
HPLC also somewhat non-specific.
Ran Gibson assembly on A + I (F1) and B + I (F2)
 
Wednesday, August 19
Gibson assembly failed due to a bad plasmid. Need to redo plasmid before attempting Gibson assembly again.
Conditions used in generating the C-terminal and N-terminal plasmid:
 
Started to figure out the details surrounding micro-aerobic cultures and how much oxygen needs to be added.
 
Thursday, August 20
Recreated samples B, E, L, F.
Recreated both N- and C-terminal plasmid.
Results of plasmid (left) and samples B, E, L, F (right):
 
Ran Gibson assembly on both S. sediminis samples and A + I again.
Inoculated 3 micro-anaerobic cultures containing the following types of media:
HEPES buffer
DMSO
Unbuffered
Transformed the A + I Gibson product into E. coli using heat shock and plated on ampicillin plates.
 
Friday, August 21
Plates did not show growth, so transformation was unsuccessful.
Planned to re-plate and/or possibly rerun Gibson assembly using a different ratio of plasmid to fragment.
 
Monday, August 24
Created both buffered and unbuffered spheroplast solutions.
Considered switching to transformation by electroporation rather than heat shock.
 
Tuesday, August 25
Transformed E. coli with F3/G3 and C-terminal S. sediminis Gibson products using electroporation.
 
Wednesday, August 26
Ran Gibson assembly on all remaining gene/linker fragments.
 
Thursday, August 27
Ran colony PCR on G1 sample.
Prepared samples and standards for gas chromatography of micro-aerobic cultures (using ethyl acetate as solvent).
1% 1 mL butanol standard
1% 1 mL lactic acid standard
1 mL ethyl acetate blank
10% 1 mL samples of each culture (Unbuffered, HEPES, DMSO, and aerobic)
Generate more I fragment to rerun Gibson assembly on G2 (B + I).
Ran colony PCR on G3 (C + J).
 
Monday, August 31
Transformed and inoculated G4 and G5 into E. coli.
Prepared 18% MGM to be used in establishing a new aerobic culture.
 
September 2015
 
Tuesday, September 1
Ran colony PCR on G1, G4, and G5.
Ran Gibson assembly on G2.
Inoculated new aerobic cultures. Once grown, these cultures will be switched to anaerobic/micro-aerobic environments.
Fermentation: Ethanol
Week 3 (7/6/15 - 7/10/15)
Discussed the formation of the team
Used KEGG to view the metabolic pathway of Haloferax volcanii and identify genes involved with pyruvate breakdown in order to isolate potential genes to knockout
Lactate dehydrogenase & NADP-dependent malic enzyme
Found Zymomonas mobilis genome (2,056,363 bp) and Pyruvate decarboxylase gene, pdc, from Zymomonas mobilis subsp. ZM4 on NCBI
 
Week 4 (7/13/15 - 7/17/15)
 
We contacted Julie Maupin-Furlow from the University of Florida because of previous research she had done with H.vo and the Z.mo pdc gene
 
We optimized the Z.mo pdc for insertion into H.vo using DNAWorks
Designed gene blocks, an overlap region, and primers (flagged with our pTA963 plasmid) for gibson assembly of our H.vo pdc
 
Week 5 (7/20/15 - 7/24/15)
Began designing our knockouts
We used the UCSC archaea browser to retrieve genes before and after knockout gene and intergenic sequences
 
Week 6 (7/27/15 - 7/31/15)
Finalized our knockout sequences
Ran knockout genes through blastx and there are no protein matches in any organism
Designed primers for our knockout genes
Ordered our H.vo pdc gene blocks and primers along with our knockout gene blocks and primers
 
Week 7 (8/3/15 - 8/7/15)
Our gene blocks and primers arrived
We looked into making spheroplasts for transformation into H.vo
 
Week 8 (8/10/15 - 8/14/15)
Ran PCR for fragment assembly of our H.vo pdc gene
We ran PCR on or knockouts genes to amplify them
Purified H.vo pdc to be used for gibson assembly
 
Week 9 (8/17/15 - 8/21/15)
Did gibson assembly of our H.vo pdc into the N-term His-tagged plasmid pTA963
Transformed our product into chemically competent E. coli  by heat shocking them, then plated these onto ampicillin plates and let them grow overnight
Upon plating there was no growth and we determined that the problem was with the E. coli cells so we switched to top ten electro-competent E. coli
 
Week 10 (8/24/15 - 8/28/15)
Redid transformations
Electroporation was used to transform our Gibson assembly product into the electro-competent E. coli cells
Had significant growth on our ampicillin plate showing that our plasmid had been successfully transformed into the E. coli cells
 
Week 11 (8/31/15 - 9/4/15)
Ran colony PCR with the primers we used for our H.vo pdc fragment assembly
We chose 9 colonies from separate areas on the plate for a better chance of success
Results were inconclusive but we were able to select 5 candidates that were more likely to give us positive results
Reran colony PCR using plasmid specific primers
Again results were inconclusive
 
Week 12 (8/7/15 - 8/11/15)
Reran colony PCR using the same plasmid specific primers but doing a touchdown PCR
No results to examine
Reran colony PCR using a different pair of plasmid specific primer doing another touchdown PCR
No results to examine
Troubleshooting these errors
Field Team
Tuesday June 30
The field team went over the protocol and made sure all the equipment we needed was put together
Wednesday July 1
We collected 10 samples from varying locations in the Fremont salt flats
pH and salinity will have to be measured in lab due to the battery being knocked out of the portable meter
Initial plating of field samples was pushed to tomorrow due to time constraints
Thursday July 2
All of our samples were plated on 18% MGM agar plates (protocol found in the HaloHandbook)
Tuesday July 7
Substantial growth can be seen on our 18%MGM agar plates
Single cell isolations were started on colonies that showed the most robust growth from each field sample.
We decided on using a minimal media with cellulose as the only carbon source to screen for our cellulase producing halophile.
We decided on measuring turbidity on the UV spectrometer to determine sufficient growth on MCC.
Wednesday July 8
Some single cell isolations will need to be redone due to poor spreading technique
We characterized each of the single cell isolation colonies.
To prepare for 80% glycerol stocks we need to grow the single cell isolations ing 18%MGM to raise the cell count.
Thursday July 9
More single cell isolations are ready to incubate in MGM media for our cellulose screen and glycerol stocks.
The first minimal media we made was way to cloudy, we’re going to reduce it from 5g of MCC to 3g.
Friday July 10
Discussed what happens after we identify the haloarchaea after the cellulose screen. We could sequence the 16S region of the ribosomal DNA to identify the species and if there is a new species we could get the whole genome sequenced.
We would need to order a primer to do the proper PCR.
We also found a paper about solubilizing cellulose with 14%wt NaOH. It may break the chains of glucose but not substantially. (Kuo, Yen-Ning and Hong, Juan. "Investigation of Solubility of Microcrystalline Cellulose in Aqueous NaOH." (2004).)
Sunday July 12
We are going to make a soluble MCC minimal media by treating the MCC (2g) with NaOH (14.9%wt)
Monday July 13
We inoculated 5 more samples in MGM 18% from our previous single cell isolation.
We are re-isolated some of our previous single cell isolation due to smearing.
The protocol for the Micro-crystalline cellulose minimal media is finally complete for for solubilized and unsolubilized MCC.
We had to freeze the NaOH treated MCC in order to solubilize it.
Tuesday July 14
We made our solubilized MCC minimal media today.
We need to titrate the 5M TrisHCl slowly and we needed around 50mL to reach a pH of 7.5 for a 1L of solubilized MCC.
Thursday July 16
We made the insoluble MCC minal media and brought the pH down to 7.5.
We discussed what will happen after the 16S region PCR. We would use the NCBI BLASTN program to see.
The pH was raised overnight to a pH of 8.0 but it is acceptable because H. volcanii prefer a slightly alkaline environment
Tuesday July 21
We need to do more glycerol stocks for new single cell isolates.
Friday July 24
Any organism that can breakdown insoluble cellulose can theoretically breakdown insoluble MCC, so the insoluble MCC would tell us the most, but we are going to run both soluble and insoluble MCC screen just in case.
We are also considering using a glucose test to measure the concentration of glucose in the media as time goes on. We’re thinking of using a diabetes glucometer.
We are starting to prepare for designing the primers needed for the 16S ribosomal DNA PCR.
Monday July 27
We have 10 soluble and 10 insoluble MCC screens in both incubators.
The single cell isolate P5A is showing slight signs of pink in the insoluble test. Pink indicates a high cell count (if the cells are pink in color) or a high concentration of cellular metabolites, which both indicate cells can thrive of purified cellulose.
Tuesday July 28
There is growth in MCC minimal media cultures! We just need to isolate them and we’re thinking of making agar plates with the solubilized MCC minimal media.
We’re considering using the microscope to prove growth as opposed to optical density and turbidity.
Wednesday July 29
We looked at the cells that have been growing in our media under the microscope and we saw a heterogenous mixture of cell types including bacilli, cocci, and even a fast swimming species. Somewhere in the mix is our species producing cellulase and the other species may be feeding off the glucose being released from the MCC.
Friday July 31
We decided on which samples would move onto the soluble MCC agar plates to isolate our cellulase producing species.
We made MCC minimal media agar plates
Monday August 3
We’ve started plating on the soluble MCC agar plates.
More samples are showing growth in the cellulose screen
Friday August 7
We decided to go with universal archaeal primers for the 16S region of ribosomal DNA. The primers won’t work for bacteria which is ideal because we’re trying to find an archaea.
We decided to do a touchdown PCR reaction where the annealing temperature is lowered after every cycle to improve specificity of the amplification.
The annealing temperature will be set at 68℃ and will drop to 65℃ over 15 cycles
Friday August 21
The single cell isolation on the MCC plates are grown enough for identification. We need to grow them in MGM again to increase cell count for DNA extraction.
After the PCR we are going to send our PCR product to UC Berkeley to be sequenced. We’ll need controls for the PCR so we are also going to use E. coli and H. volcanii to prove our primers work only for archaea.
Tuesday August 25
We ran our sequences on the NCBI BLASTN program and we were getting hits all under the Haloferax genus
We going to show our findings at the Sierra System Synthetic Biology Symposium this weekend
Tuesday June 30
The field team went over the protocol and made sure all the equipment we needed was put together
Wednesday July 1
We collected 10 samples from varying locations in the Fremont salt flats
pH and salinity will have to be measured in lab due to the battery being knocked out of the portable meter
Initial plating of field samples was pushed to tomorrow due to time constraints
Thursday July 2
All of our samples were plated on 18% MGM agar plates (protocol found in the HaloHandbook)
Tuesday July 7
Substantial growth can be seen on our 18%MGM agar plates
Single cell isolations were started on colonies that showed the most robust growth from each field sample.
We decided on using a minimal media with cellulose as the only carbon source to screen for our cellulase producing halophile.
We decided on measuring turbidity on the UV spectrometer to determine sufficient growth on MCC.
Wednesday July 8
Some single cell isolations will need to be redone due to poor spreading technique
We characterized each of the single cell isolation colonies.
To prepare for 80% glycerol stocks we need to grow the single cell isolations ing 18%MGM to raise the cell count.
Thursday July 9
More single cell isolations are ready to incubate in MGM media for our cellulose screen and glycerol stocks.
The first minimal media we made was way to cloudy, we’re going to reduce it from 5g of MCC to 3g.
Friday July 10
Discussed what happens after we identify the haloarchaea after the cellulose screen. We could sequence the 16S region of the ribosomal DNA to identify the species and if there is a new species we could get the whole genome sequenced.
We would need to order a primer to do the proper PCR.
We also found a paper about solubilizing cellulose with 14%wt NaOH. It may break the chains of glucose but not substantially. (Kuo, Yen-Ning and Hong, Juan. "Investigation of Solubility of Microcrystalline Cellulose in Aqueous NaOH." (2004).)
Sunday July 12
We are going to make a soluble MCC minimal media by treating the MCC (2g) with NaOH (14.9%wt)
Monday July 13
We inoculated 5 more samples in MGM 18% from our previous single cell isolation.
We are re-isolated some of our previous single cell isolation due to smearing.
The protocol for the Micro-crystalline cellulose minimal media is finally complete for for solubilized and unsolubilized MCC.
We had to freeze the NaOH treated MCC in order to solubilize it.
Tuesday July 14
We made our solubilized MCC minimal media today.
We need to titrate the 5M TrisHCl slowly and we needed around 50mL to reach a pH of 7.5 for a 1L of solubilized MCC.
Thursday July 16
We made the insoluble MCC minal media and brought the pH down to 7.5.
We discussed what will happen after the 16S region PCR. We would use the NCBI BLASTN program to see.
The pH was raised overnight to a pH of 8.0 but it is acceptable because H. volcanii prefer a slightly alkaline environment
Tuesday July 21
We need to do more glycerol stocks for new single cell isolates.
Friday July 24
Any organism that can breakdown insoluble cellulose can theoretically breakdown insoluble MCC, so the insoluble MCC would tell us the most, but we are going to run both soluble and insoluble MCC screen just in case.
We are also considering using a glucose test to measure the concentration of glucose in the media as time goes on. We’re thinking of using a diabetes glucometer.
We are starting to prepare for designing the primers needed for the 16S ribosomal DNA PCR.
Monday July 27
We have 10 soluble and 10 insoluble MCC screens in both incubators.
The single cell isolate P5A is showing slight signs of pink in the insoluble test. Pink indicates a high cell count (if the cells are pink in color) or a high concentration of cellular metabolites, which both indicate cells can thrive of purified cellulose.
Tuesday July 28
There is growth in MCC minimal media cultures! We just need to isolate them and we’re thinking of making agar plates with the solubilized MCC minimal media.
We’re considering using the microscope to prove growth as opposed to optical density and turbidity.
Wednesday July 29
We looked at the cells that have been growing in our media under the microscope and we saw a heterogenous mixture of cell types including bacilli, cocci, and even a fast swimming species. Somewhere in the mix is our species producing cellulase and the other species may be feeding off the glucose being released from the MCC.
Friday July 31
We decided on which samples would move onto the soluble MCC agar plates to isolate our cellulase producing species.
We made MCC minimal media agar plates
Monday August 3
We’ve started plating on the soluble MCC agar plates.
More samples are showing growth in the cellulose screen
Friday August 7
We decided to go with universal archaeal primers for the 16S region of ribosomal DNA. The primers won’t work for bacteria which is ideal because we’re trying to find an archaea.
We decided to do a touchdown PCR reaction where the annealing temperature is lowered after every cycle to improve specificity of the amplification.
The annealing temperature will be set at 68℃ and will drop to 65℃ over 15 cycles
Friday August 21
The single cell isolation on the MCC plates are grown enough for identification. We need to grow them in MGM again to increase cell count for DNA extraction.
After the PCR we are going to send our PCR product to UC Berkeley to be sequenced. We’ll need controls for the PCR so we are also going to use E. coli and H. volcanii to prove our primers work only for archaea.
Tuesday August 25
We ran our sequences on the NCBI BLASTN program and we were getting hits all under the Haloferax genus
We going to show our findings at the Sierra System Synthetic Biology Symposium this weekend
   Home | Teams | Project | Human Practices | Notebook | Safety | Achievements | F.O.C.U.S   
