<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCSC_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCSC/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV><UL class="nav navbar-nav bernick_swag"><LI class="active"><A href="https://2015.igem.org/Team:UCSC">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Team </A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Breakdown">Breakdown</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Fermentation">Fermentation</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Field">Field</A></LI></UL><LI><A href="https://2015.igem.org/Team:UCSC/Project">Project</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Practices">Bioethics</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Logs">Logs</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Safety">Safety</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Medal_Fulfillment">Medal Fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Software">Software</A></LI></DIV></DIV><DIV id="our_content"><H1> Protocols </H1><P style="text-align: center;">
A compilation of the protocols used at UCSC: from Gibson to MGM.

To view our Notebook click <A href="https://2015.igem.org/Team:UCSC/Logs">here</A></P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
          Agarose Gel Protocol.
        </A></H4></DIV><DIV id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body"><P> Preparing DNA and Running in Gel Electrophoresis</P><P><STRONG>Reagents: </STRONG></P><OL><LI>10X TBE Buffer </LI><LI>Deionized Water </LI><LI>Agarose Powder</LI><LI>Gel Loading Dye Purple 6X spiked with SYBRGold </LI><LI>DNA solutions </LI><LI>DNA Ladder</LI></OL><P><STRONG>Equipment:</STRONG></P><OL><LI>Gel Doc-it UV imaging machine (Purple and grey with green tape) </LI><LI>Computer with Launch VisionWorksLS program open </LI><LI>Gel electrophoresis Box (with non-morphed electrodes) and Lid</LI><LI>Power Supply box </LI><LI>Microwave</LI><LI>Rotation Mixer</LI><LI>Weight</LI></OL><P><STRONG>Methods: </STRONG></P><UL><LI><STRONG><STRONG>Protocol 1: Preparing the Gel</STRONG></STRONG></LI></UL><OL><LI>Create<STRONG> 500 mL of 1X TBE buffer</STRONG> by mixing 50 mL of 10X TBE buffer and 450 mL of deionized H20</LI><LI>For a 1% (w/v) gel (for very large DNA), <STRONG>take 50 mL of 1X TBE</STRONG> buffer and mix with the appropriate percent mass of Agarose powder (Weigh out 0.5 g for a 1% gel)</LI><LI><STRONG>Mix</STRONG> thoroughly using the rotation mixer (optional)</LI><LI>Loosely <STRONG>plug flask</STRONG> with paper towel and heat in <STRONG>microwave</STRONG> for approximately 1 minute, or until the agarose is fully dissolved. The liquid should be completely clear.</LI><LI>Once completed, wait till the beaker cools and prepare to add solution to the gel tray. Put comb into the tray and make sure that the tray is oriented in the gel box so that its ends are blocked against the sides of the box. </LI><LI><STRONG>SLOWLY</STRONG> pour solution into gel tray to avoid creating bubbles. </LI><LI>Allow gel to solidify and <STRONG>cool for 20 - 30 minutes</STRONG> or until fully solidified. This can be tested by how easy it is to move the pick.</LI><LI>Once solidified, gently turn the gel tray with the gel so that the ends are exposed.</LI><LI><STRONG>Fill</STRONG> the gel box<STRONG> with 1X TBE buffer </STRONG>until gel is fully covered (0.5 - 1 cm above gel)</LI><LI>Slowly and carefully remove the comb so as not to tear the wells. </LI></OL><UL><LI><STRONG><STRONG>Protocol 2: Staining the DNA Samples and Running Gel</STRONG></STRONG></LI></UL><P><STRONG>Caution:</STRONG> Gel box has high voltage! Be sure to turn off power pack and unplug leads before removing lid of gel box.</P><OL><OL><LI>You will be staining the DNA using Professor Bernick’s Droplet Technique. Depending on how many samples you have (including controls and DNA ladder), place <STRONG>1 uL </STRONG>droplets of 6X Loading Dye on a piece of parafilm. </LI><LI>Take 5 uL of each DNA sample and add to an individual droplet. You will have a final volume of 6uL. Be sure to mix thoroughly up and down, then take the sample and inject into a well. </LI></OL></OL><UL><LI><STRONG><STRONG>Make sure to change pipet tips for each DNA sample so as to prevent contamination! Keep track of the positions of each sample</STRONG></STRONG></LI></UL><OL><LI>Once all samples have been loaded, close the gel box and attach all electrodes to the gel box and to the power pack. (Use color coding to be sure all electrodes are attached correctly. Black = negative electrode and Red = positive. <STRONG>The electrode closest to the samples will be negative while the further is positive. </STRONG></LI><LI>Turn on the power pack and set to 120 V. You should immediately see bubbles rising from the wires in the gel box. </LI><LI>Run continuously for <STRONG>75 minutes</STRONG>. Once complete, turn of the power pack. </LI></OL><UL><LI><STRONG><STRONG>Protocol 3: Visualizing the Gel</STRONG></STRONG></LI></UL><OL><LI>Remove the lid from the gel box (<STRONG>Be sure that the power pack is turned off!)</STRONG></LI><LI>Pour out the 1x TBE buffer into the used TBE buffer beaker </LI><LI>Open the Gel Doc it door and pull out the UV transilluminator. (<STRONG>Make sure it is off!) </STRONG></LI><LI>Transfer the gel to the UV transilluminator tray, push it back into the machine and flip the on switch on the lower right corner. </LI><LI>Close the door and make sure the knob is turned to Syber Gold. Go to the computer and open the Launch VisionWorks program. </LI><LI>Flick the UV light switch and press preview on the computer. When the gel is clearly visible and you can see the bands, press capture and save it to the folder IGEM2015. </LI></OL><P><STRONG>References: </STRONG></P><P>Ruben, Giulia, “Lab 2 Protocol #1, Gel Electrophoresis”, Biochemistry Laboratory, Winter 2015</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
          Colony PCR Protocols for Transformed <I>E.coli</I></A></H4></DIV><DIV id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><DIV class="panel-body"><P><STRONG>Colony PCR Protocol for Transformed </STRONG><STRONG><EM>E. coli</EM></STRONG></P><P><STRONG>Equipment</STRONG></P><OL><LI>Thermocycler</LI><LI>Micro-pipettes (P10, P20 &amp; P1000)</LI><LI>Thin-walled PCR Tubes</LI><LI>Eppendorf tubes</LI><LI>Microcentrifuge with PCR and Eppendorf tube adapters</LI><LI>Ice bucket and ice (Ice is found in the Autoclave room 202)</LI></OL><P><STRONG>Reagents</STRONG></P><OL><LI>Fresh Miliq water (Found in the Autoclave room 202)</LI><LI>SOC Media </LI><LI>Titaq 2x Master Mix</LI></OL><P><STRONG>Methods</STRONG></P><P>After plating your electroporated <EM>E. coli</EM> cells, choose about 6 large cultures with the least amount of micro satellites. </P><P>Take a micropipet tip and gently touch it to the colony (No need to scrape as this will be too many cells) </P><P>Place the micropipet into an eppendorf tube with 10 uL of water to create a 1:10 dilution of DNA, and slowly pipet up and down to lyse the cells. </P><P>Take 1uL of this dilution and use it as DNA template for your PCR reaction </P><UL><LI><STRONG><STRONG>Plasmid Specific Primer Options </STRONG></STRONG></LI></UL><UL><LI><STRONG>Breakdown Primers: </STRONG></LI></UL><UL><LI>bGlu_Seq_Fw1: 5’ CCCTCGATTTTCCGCCTGCCGATTA 3’</LI><LI>bGlu_Seq_Rv4: 5’ GCGCTCTAGAACTAGTGGATCCCCC 3’</LI></UL><UL><LI><STRONG><STRONG>Dominic Primers: </STRONG></STRONG></LI></UL><UL><LI>Aldy5Seq1F: 5’-TACTTCACATTCGCGGACCTATTG - 3’</LI><LI>Aldy5Seq5R: 5’-AGAACTAGTGGATCCCCCG - 3’</LI></UL><OL><LI>Professor Bernick Primers: </LI></OL><UL><LI><STRONG><STRONG>PCR Ingredients</STRONG></STRONG></LI></UL><TABLE><TBODY><TR><TD><P>Components </P></TD><TD><P>25 uL Reaction </P></TD></TR><TR><TD><P>Titaq 2x Master Mix </P></TD><TD><P>12.5 uL</P></TD></TR><TR><TD><P>Forward Primer</P></TD><TD><P>0.5 uL</P></TD></TR><TR><TD><P>Reverse Primer</P></TD><TD><P>0.5 uL</P></TD></TR><TR><TD><P>Template DNA </P></TD><TD><P>1 uL</P></TD></TR><TR><TD><P>PCR Enhancer</P></TD><TD><P>5 uL</P></TD></TR><TR><TD><P>Miliq Water </P></TD><TD><P>5.5 uL</P></TD></TR></TBODY></TABLE><P><STRONG>References</STRONG></P><P>Professor David Bernick, Lab Advisor </P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingThree"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
          Polymerase Chain Reaction (PCR) for Amplification of pTA963 Expression Plasmids
        </A></H4></DIV><DIV id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree"><DIV class="panel-body"><P><STRONG> Polymerase Chain Reaction (PCR) for Amplification of pTA963 Expression Plasmids</STRONG></P><P><STRONG>Equipment:</STRONG></P><OL><LI>Heat block or thermo-cycler</LI><LI>Micro-pipettes (P10 &amp; P20)</LI><LI>Thin-walled PCR Tubes</LI><LI>Microcentrifuge with PCR adapters</LI><LI>Small beaker and ice (Ice is found in the Autoclave room 202)</LI></OL><P><STRONG>Methods</STRONG></P><P>We will be amplifying <STRONG>two versions</STRONG> of the pTA963 expression plasmid: One which allows the insertions of a 6 x Histidine Tag on the N-Terminus region, and other which allows insertion in the C- Terminus region. </P><P>You will be using different primers depending on which plasmid you are amplifying, so be sure to grab the right ones!</P><P>If you have<STRONG> the N-Terminal</STRONG>, the primers you will be using are: </P><P><STRONG>pTA963Nterm Forward</STRONG></P><P><STRONG>pTA963Nterm Reverse</STRONG></P><P>If you have <STRONG>the C-Terminal</STRONG>, the primers you will be using are: </P><P><STRONG>pTA963Cterm Forward:</STRONG></P><P>CGATAAGCTTGATATCGAATTCCTG</P><P><STRONG>pTA963Cterm Reverse:</STRONG></P><P>ATGCGCAATAGGTCCGCG</P><P><STRONG>Before you begin</STRONG>, always prepare the thermocycler using the conditions below. The thermocycler takes time to warm up, so run the appropriate protocol as you are mixing so that you can place the PCR reaction in the machine <STRONG>as soon as possible</STRONG>. </P><P>In this experiment, you will be preparing the thermocycler for a Touchdown PCR. The annealing temperature for the first round of cycles will decrease by <STRONG>0.2</STRONG><STRONG>°C</STRONG> each cycle for <STRONG>a total temperature drop of 1°C/5 cycles. </STRONG>Use the appropriate parameters below depending on which reaction you are running.</P><TABLE><TBODY><TR><TD colspan="9"><P>C-terminal cycles (15 cycles, 10 cycles)</P></TD></TR><TR><TD><P>|&lt;-------</P></TD><TD><P>---------</P></TD><TD><P> x 15 |</P></TD><TD><P>|&lt;-------</P></TD><TD><P>---------</P></TD><TD><P>-- x 10| </P></TD></TR><TR><TD><P>98°C</P></TD><TD><P>98°C</P></TD><TD><P>66°C*</P></TD><TD><P>72°C</P></TD><TD><P>98°C</P></TD><TD><P>63°C</P></TD><TD><P>72°C</P></TD><TD><P>72°C</P></TD><TD><P>4°C</P></TD></TR><TR><TD><P>5:00</P></TD><TD><P>0:10</P></TD><TD><P>0:10</P></TD><TD><P>3:00</P></TD><TD><P>0:10</P></TD><TD><P>0:10</P></TD><TD><P>3:00</P></TD><TD><P>5:00</P></TD><TD><P>Hold</P></TD></TR></TBODY></TABLE><P>*Touchdown is a -0.2 degree per cycle for a total of 3 degree decrease.</P><TABLE><TBODY><TR><TD colspan="9"><P>N-terminal cycles (10 cycles, 15 cycles)</P></TD></TR><TR><TD><P>|&lt;-------</P></TD><TD><P>---------</P></TD><TD><P>-- x 10| </P></TD><TD><P>|&lt;-------</P></TD><TD><P>---------</P></TD><TD><P> x 5 |</P></TD></TR><TR><TD><P>98°C</P></TD><TD><P>98°C</P></TD><TD><P>64°C*</P></TD><TD><P>72°C</P></TD><TD><P>98°C</P></TD><TD><P>62°C</P></TD><TD><P>72°C</P></TD><TD><P>72°C</P></TD><TD><P>4°C</P></TD></TR><TR><TD><P>5:00</P></TD><TD><P>0:10</P></TD><TD><P>0:10</P></TD><TD><P>3:00</P></TD><TD><P>0:10</P></TD><TD><P>0:10</P></TD><TD><P>3:00</P></TD><TD><P>5:00</P></TD><TD><P>Hold</P></TD></TR></TBODY></TABLE><P>*Touchdown is -0.2 degrees per cycle for a total of 2 degree decrease</P><P>Use the following protocol if 2X Q5 <STRONG>Mastermix is not available </STRONG>:</P><TABLE><TBODY><TR><TD><P><STRONG>COMPONENTS</STRONG></P></TD><TD><P><STRONG>25 µL RXN</STRONG></P></TD><TD><P><STRONG>50 µL RXN</STRONG></P></TD><TD><P><STRONG>FINAL </STRONG><STRONG>CONC.</STRONG></P></TD></TR><TR><TD><P>5X Q5 Reaction Buffer</P></TD><TD><P>5µL</P></TD><TD><P>10 µL</P></TD><TD><P>1X</P></TD></TR><TR><TD><P>10 mM dNTPS</P></TD><TD><P>0.5 µL</P></TD><TD><P>1 µL</P></TD><TD><P>200 µM</P></TD></TR><TR><TD><P>10 uM Forward Primer</P></TD><TD><P>1.25 µL</P></TD><TD><P>2.5 µL</P></TD><TD><P>0.5 µM</P></TD></TR><TR><TD><P>10 uM Reverse Primer</P></TD><TD><P>1.25 µL</P></TD><TD><P>2.5 µL</P></TD><TD><P>0.5 µM</P></TD></TR><TR><TD><P>Template DNA</P></TD><TD><P>0.5 uL</P></TD><TD><P>1 uL</P></TD><TD><P>&lt;1,000 ng</P></TD></TR><TR><TD><P>Q5 High-Fidelity DNA Polymerase</P></TD><TD><P>0.25 µL</P></TD><TD><P>0.5 µL</P></TD><TD><P>0.02 U/µl</P></TD></TR><TR><TD><P>PCR Enhancer (optional)</P></TD><TD><P>(5 µL)</P></TD><TD><P>(10 µL)</P></TD><TD><P>(1X)</P></TD></TR><TR><TD><P>Nuclease-Free Water</P></TD><TD><P>11.25 ul</P></TD><TD><P>22.50</P></TD></TR></TBODY></TABLE><P><STRONG>If Mastermix is available, </STRONG>use the following protocol:</P><TABLE><TBODY><TR><TD><P><STRONG>COMPONENTS</STRONG></P></TD><TD><P><STRONG>25 µL RXN</STRONG></P></TD><TD><P><STRONG>50 uL RXN</STRONG></P></TD></TR><TR><TD><P>MiliQ H20</P></TD><TD><P>4.5 uL</P></TD><TD><P>9 uL</P></TD></TR><TR><TD><P>5X PCR Enhancer</P></TD><TD><P>5 uL</P></TD><TD><P>10 uL</P></TD></TR><TR><TD><P>10 uM Forward Primer</P></TD><TD><P>1.25 uL</P></TD><TD><P>2.5 uL</P></TD></TR><TR><TD><P>10 uM Reverse Primer</P></TD><TD><P>1.25 uL</P></TD><TD><P>2.5 uL</P></TD></TR><TR><TD><P>Diluted pTA963 Plasmid</P></TD><TD><P>0.5 uL</P></TD><TD><P>1 uL</P></TD></TR><TR><TD><P>2X Q5 MasterMix</P></TD><TD><P>12.5 uL</P></TD><TD><P>25 uL</P></TD></TR></TBODY></TABLE><P>Obtain thin-walled PCR tubes – be sure to wear gloves when removing them from the container. Then, distribute the appropriate volume of primers, plasmid and water into the tube. <STRONG>Do not add DNA Polymerase or Mastermix, this will go last and only when you are ready to run the reaction in the thermocycler. Place all reagent and reaction tubes on ice while you wait.</STRONG></P><P>It’s best to start with the appropriate volume of H20, then add all the other reagents. </P><P>Mix contents of each tube thoroughly, either by by pipetting up and down slowly, OR placing the tube on a vortex mixer and microcentrifuge for about a second. </P><P>Make sure your tube is well labelled with group members, the specific plasmid you are amplifying, and the date.</P><P><STRONG>Clean- Up</STRONG></P><P>Dump ice into the sink and place ice box in designated area on side bench </P><P>Place all remaining plasmid and primer samples back in the fridge. </P><P><STRONG>References: </STRONG></P><P>Ruben, Giulia, “Lab 4 Protocol #2, Polymerase Chain Reaction”, Biochemistry Laboratory, Winter 2015</P><P>Dominic Schenone, lab advisor </P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingFour"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
          Protocol for Fragment Assembly and Gene Isolation from <I>H. hispanica</I></A></H4></DIV><DIV id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour"><DIV class="panel-body"><P><STRONG>Protocol for Fragment Assembly and Gene Isolation from H. Hispanica</STRONG></P><P> </P><P><STRONG>Equipment:</STRONG></P><OL><LI>Heat block</LI><LI>Thermocycler</LI><LI>Micro-pipettes (P10, P20 &amp; P1000)</LI><LI>Thin-walled PCR Tubes</LI><LI>Eppendorf tubes</LI><LI>Microcentrifuge with PCR and Eppendorf tube adapters</LI><LI>Ice bucket and ice (Ice is found in the Autoclave room 202)</LI><LI>2 Liter Beaker</LI></OL><P> </P><P><STRONG>Reagents</STRONG></P><OL><LI>Fresh Miliq water (Also found in the Autoclave room 202)</LI><LI>23% Modified Growth Medium</LI></OL><P> </P><P><STRONG>Methods</STRONG></P><UL><LI><STRONG><STRONG>Fragment Assembly Protocol </STRONG></STRONG></LI></UL><P>Obtain the appropriate amounts of fragment 1, 2 and 3 that provide an equal molar concentration of each fragment. In this case, the concentrations are:</P><UL><LI>Fragment 1: 0.88 uL</LI><LI>Fragment 2: 1.25 uL</LI><LI>Fragment 3: 1 uL</LI></UL><P> </P><P>Use the following primers:</P><UL><LI>bGlu_Fw_flag:</LI></UL><P>5'CCTGCCGATTACTTCACATTCGCGGACC</P><P>TATTGCGCATATGGCACATGACACGACTGACG 3'</P><UL><LI>bGlu_Rv_flag:</LI></UL><P>5' TTCGATATCAAGCTTATCGTCAGTGGTG</P><P>GTGGTGGTGGTGCGAGCGACCGCGGACCTC 3'</P><P> </P><P>Set the thermocycler to the following conditions:</P><P> </P><TABLE><TBODY><TR><TD colspan="6"><P>Fragment Assembly cycles (25 cycles)</P></TD></TR><TR><TD><P>|&lt;-------------</P></TD><TD><P>----------------</P></TD><TD><P>x 25 |</P></TD></TR><TR><TD><P>98°C</P></TD><TD><P>98°C</P></TD><TD><P>65°C*</P></TD><TD><P>72°C</P></TD><TD><P>72°C</P></TD><TD><P>4°C</P></TD></TR><TR><TD><P>5:00</P></TD><TD><P>0:10</P></TD><TD><P>0:10</P></TD><TD><P>1:00</P></TD><TD><P>5:00</P></TD><TD><P>Hold</P></TD></TR></TBODY></TABLE><UL><LI><STRONG><STRONG>Gene Isolation Protocol </STRONG></STRONG></LI></UL><P> </P><UL><LI><STRONG><STRONG>Growth of Haloarcula Hispanica </STRONG></STRONG></LI></UL><P>Your cells need to be grown in 23% Modified Growth media till they have reached an A 600reading between 0.4 - 0.8.</P><P> </P><P>To make a liter of 23% Modified Growth media, use the following ingredients:</P><P> </P><TABLE><TBODY><TR><TD><P>Salt Water (30% Stock)</P></TD><TD><P>767</P></TD></TR><TR><TD><P>Pure Water</P></TD><TD><P>200</P></TD></TR><TR><TD><P>Peptone( Oxoid)</P></TD><TD><P>5</P></TD></TR><TR><TD><P>Yeast Extract</P></TD><TD><P>1</P></TD></TR></TBODY></TABLE><P>*Don’t use Difco Bacto-Peptone, as it was reported in 1988 that it contained bile salts that lyse halobacteria.</P><P> </P><UL><LI><STRONG><STRONG>Rapid Chromosomal DNA Isolation </STRONG></STRONG></LI></UL><P>Spin down 1 mL of a freshly grown Hbt. salinarum culture at exponential or early stationary</P><P>phase (1 minute at max. rpm in a microcentrifuge)</P><P> </P><P>Suck off the supernatant.</P><P> </P><P>Add 400 μL pure water and lyse the cells by gently pipetting. The cells disrupt immediately by</P><P>osmotic lysis.</P><P> </P><P>Inactivate the proteins by heating the DNA solution up to 70°C for 10 min.</P><P> </P><P>Comments:</P><OL><LI>a) DNA from freshly grown cultures (exponential or early stationary phase) give better results for</LI></OL><P>further reactions than old ones.</P><P> </P><OL><LI>b) use 1 – 3 μl as a template for amplifying a DNA fragment by PCR.</LI><LI>c) digest 30 μl (in 100 μl final volume) for southern blot analysis.</LI><LI>d) if there is a problem using the DNA as template for PCR it could be due to much DNA. Try</LI></OL><P>several concentrations or volumes e.g. 1, 5, 10 μl from stock solution.</P><P> </P><UL><LI><STRONG><STRONG>Nested PCR Part 1: Wild Type Beta Glucosidase Isolation</STRONG></STRONG></LI></UL><P>Use the following primers for the first step of nested PCR:</P><UL><LI>bGlu_iso_Fw: 5' CACTGTTTGGGTGCGCCTGTCATG 3'</LI><LI>bGlu_iso_Rv: 5' GTAGTGGGTATTCGGGGTCCGGTG 3'</LI></UL><P> </P><P>Set the thermocycler to the following conditions:</P><TABLE><TBODY><TR><TD colspan="6"><P>Gene Isolation cycles (25 cycles)</P></TD></TR><TR><TD><P>|&lt;-------------</P></TD><TD><P>----------------</P></TD><TD><P>x 25 |</P></TD></TR><TR><TD><P>98°C</P></TD><TD><P>98°C</P></TD><TD><P>71°C*</P></TD><TD><P>72°C</P></TD><TD><P>72°C</P></TD><TD><P>4°C</P></TD></TR><TR><TD><P>5:00</P></TD><TD><P>0:10</P></TD><TD><P>0:10</P></TD><TD><P>0:42 sec</P></TD><TD><P>5:00</P></TD><TD><P>Hold</P></TD></TR></TBODY></TABLE><P> </P><UL><LI><STRONG><STRONG>PCR Reaction Ingredients </STRONG></STRONG></LI></UL><P> </P><P>Use the following protocol if 2X Q5 <STRONG>Mastermix is not available </STRONG>:</P><TABLE><TBODY><TR><TD><P><STRONG>COMPONENTS</STRONG></P></TD><TD><P><STRONG>25 µL RXN</STRONG></P></TD><TD><P><STRONG>50 µL RXN</STRONG></P></TD><TD><P><STRONG>FINAL </STRONG><STRONG>CONC.</STRONG></P></TD></TR><TR><TD><P>5X Q5 Reaction Buffer</P></TD><TD><P>5µL</P></TD><TD><P>10 µL</P></TD><TD><P>1X</P></TD></TR><TR><TD><P>10 mM dNTP</P></TD><TD><P>0.5 µL</P></TD><TD><P>1 µL</P></TD><TD><P>200 µM</P></TD></TR><TR><TD><P>10 uM Forward Primer</P></TD><TD><P>1.25 µL</P></TD><TD><P>2.5 µL</P></TD><TD><P>0.5 µM</P></TD></TR><TR><TD><P>10 uM Reverse Primer</P></TD><TD><P>1.25 µL</P></TD><TD><P>2.5 µL</P></TD><TD><P>0.5 µM</P></TD></TR><TR><TD><P>Template DNA</P></TD><TD><P>0.5 uL</P></TD><TD><P>1 uL</P></TD><TD><P>&lt;1,000 ng</P></TD></TR><TR><TD><P>Q5 High-Fidelity DNA Polymerase</P></TD><TD><P>0.25 µL</P></TD><TD><P>0.5 µL</P></TD><TD><P>0.02 U/µl</P></TD></TR><TR><TD><P>PCR Enhancer (optional)</P></TD><TD><P>(5 µL)</P></TD><TD><P>(10 µL)</P></TD><TD><P>(1X)</P></TD></TR><TR><TD><P>Nuclease-Free Water</P></TD><TD><P>11.25 ul</P></TD><TD><P>22.50</P></TD></TR></TBODY></TABLE><P><STRONG>If Mastermix is available, </STRONG>use the following protocol:</P><P> </P><TABLE><TBODY><TR><TD><P><STRONG>COMPONENTS</STRONG></P></TD><TD><P><STRONG>25 µL RXN</STRONG></P></TD><TD><P><STRONG>50 uL RXN</STRONG></P></TD></TR><TR><TD><P>MiliQ H20</P></TD><TD><P>4.5 uL</P></TD><TD><P>9 uL</P></TD></TR><TR><TD><P>5X PCR Enhancer</P></TD><TD><P>5 uL</P></TD><TD><P>10 uL</P></TD></TR><TR><TD><P>10 uM Forward Primer</P></TD><TD><P>1.25 uL</P></TD><TD><P>2.5 uL</P></TD></TR><TR><TD><P>10 uM Reverse Primer</P></TD><TD><P>1.25 uL</P></TD><TD><P>2.5 uL</P></TD></TR><TR><TD><P>Diluted pTA963 Plasmid</P></TD><TD><P>0.5 uL</P></TD><TD><P>1 uL</P></TD></TR><TR><TD><P>2X Q5 MasterMix</P></TD><TD><P>12.5 uL</P></TD><TD><P>25 uL</P></TD></TR></TBODY></TABLE><P> </P><P>Obtain thin-walled PCR tubes – be sure to wear gloves when removing them from the container. Then, distribute the appropriate volume of primers, plasmid and water into the tube. <STRONG>Do not add DNA Polymerase or Mastermix, this will go last and only when you are ready to run the reaction in the thermocycler. Place all reagent and reaction tubes on ice while you wait.</STRONG></P><P> </P><P>It’s best to start with the appropriate volume of H20, then add all the other reagents.</P><P> </P><P>Mix contents of each tube thoroughly, either by by pipetting up and down slowly, OR placing the tube on a vortex mixer and microcentrifuge for about a second.</P><P> </P><P>Make sure your tube is well labelled with group members, the specific plasmid you are amplifying, and the date.</P><P> </P><P><STRONG>Clean- Up</STRONG></P><P>Dump ice into the sink and place ice box in designated area on side bench</P><P>Place all remaining plasmid and primer samples back in the fridge.</P><P> </P><P><STRONG>References: </STRONG></P><P>Ruben, Giulia, 2015 “Lab 4 Protocol #2, Polymerase Chain Reaction”, Biochemistry Laboratory</P><P> </P><P>Dyall-Smith, Mike., Rodriguez-Valera., 2008 “Modified Growth Medium (MGM) for Haloarchea”,<EM> The Halohandbook: Protocols for Haloarcheal Genetics </EM>(7th ed.)<EM>. </EM>14-15</P><P> </P><P>Dyall-Smith, Mike., Pfieffer, Matthias., 2008 “Alternative Rapid Chromosomal DNA Isolation”,<EM> The Halohandbook: Protocols for Haloarcheal Genetics </EM>(7th ed.)<EM>.</EM> 70</P><P>Protocol Taken from the Halohandbook Page 14 by Dr Mike Dyall-Smith</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingFive"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFive" aria-expanded="false" aria-controls="collapseFive">
          SOC Media
        </A></H4></DIV><DIV id="collapseFive" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFive"><DIV class="panel-body"><H1>Ingredients</H1><UL><LI><A href="http://openwetware.org/wiki/SOB">SOB</A></LI><LI>20 mM glucose</LI></UL><H1>Protocol</H1><OL><LI>Follow directions to make 1 liter of <A href="http://openwetware.org/wiki/SOB">SOB</A> media</LI><LI>After cooling medium to less than 50°C, add 20 ml filter sterilized 20% glucose solution</LI></OL><P> </P><H1>Summary</H1><P><STRONG>SOB Medium. Used in growing bacteria for preparing chemically competent cells</STRONG></P><P>Ingredients</P><UL><LI>0.5% (w/v) yeast extract</LI><LI>2% (w/v) tryptone</LI><LI>10 mM NaCl</LI><LI>2.5 mM KCl</LI><LI>20 mM MgSO4</LI></UL><P>Per liter:</P><UL><LI>5 g yeast extract</LI><LI>20 g tryptone</LI><LI>0.584 g NaCl</LI><LI>0.186 g KCl</LI><LI>2.4 g MgSO4</LI></UL><P><EM>Note:</EM> Some formulations of SOB use 10 mM MgCl2 and 10 mM MgSO4 instead of 20 mM MgSO4.</P><P>SOB medium is also available dry premixed from Difco, 0443-17.</P><P>Adjust to pH 7.5 prior to use. This requires approximately 25 ml of 1M NaOH per liter.</P><P> </P><P>15/10 medium</P><P>Growth of competent TOP10 cells in Example 2 of the Bloom05 patent is performed in 15/10 broth, which is similar to SOB:</P><UL><LI>1.5% yeast extract</LI><LI>1% Bacto-Tryptone</LI><LI>10mM NaCl</LI><LI>2mM KCl</LI><LI>10 mM MgCl2</LI><LI>10 mM MgSO4</LI></UL><H2><STRONG>Extended instructions</STRONG></H2><UL><LI>1 M MgCl2 stock: dissolve 20.33 g MgCl2 6H2O in 100 ml ddH2O (XXX), autoclave on liquid cycle @ XXX°C for 20 min (can be done at the same time as SOC pre-mix below)</LI><LI>250 mM KCl stock: dissolve 1.86 KCl in 100 ml ddH2O (XXX)</LI></UL><UL><LI>combine:</LI></UL><TABLE><TBODY><TR><TD><P><STRONG>Reagent</STRONG></P></TD><TD><P><STRONG>for 1 L</STRONG></P></TD><TD><P><STRONG>500 mL</STRONG></P></TD><TD><P><STRONG>100 mL</STRONG></P></TD></TR><TR><TD><P>tryptone</P></TD><TD><P>20 g</P></TD><TD><P>10 g</P></TD><TD><P>2g</P></TD></TR><TR><TD><P>yeast</P></TD><TD><P>5g</P></TD><TD><P>2.5 g</P></TD><TD><P>0.5 g</P></TD></TR><TR><TD><P>NaCl</P></TD><TD><P>0.5 g</P></TD><TD><P>0.25 g</P></TD><TD><P>0.05 g</P></TD></TR><TR><TD><P>250 nM KCl</P></TD><TD><P>10 mL</P></TD><TD><P>5mL</P></TD><TD><P>1 mL</P></TD></TR><TR><TD><P>ddH2O</P></TD><TD><P>980 ml</P></TD><TD><P>490 ml</P></TD><TD><P>98 ml</P></TD></TR></TBODY></TABLE><UL><LI>adjust pH to 7.0 w/ NaOH</LI><LI>bring to volume:</LI></UL><TABLE><TBODY><TR><TD><P><STRONG>Reagent</STRONG></P></TD><TD><P><STRONG>for 1 L</STRONG></P></TD><TD><P><STRONG>500 mL</STRONG></P></TD><TD><P><STRONG>100 mL</STRONG></P></TD></TR></TBODY></TABLE><UL><LI>autoclave on liquid cycle @ XXX°C for 20 min</LI></UL><UL><LI>add autoclaved 1 M MgCl2</LI></UL><TABLE><TBODY><TR><TD><P><STRONG>Reagent</STRONG></P></TD><TD><P><STRONG>for 1 L</STRONG></P></TD><TD><P><STRONG>500 mL</STRONG></P></TD><TD><P><STRONG>100 mL</STRONG></P></TD></TR><TR><TD><P>1 M MgCl2</P></TD><TD><P>10 ml</P></TD><TD><P>5 ml</P></TD><TD><P>1 ml</P></TD></TR></TBODY></TABLE><P> </P><H2><STRONG>Source</STRONG></H2><P>Adapted From:</P><OL><LI>Ausubel et al., <EM>Short Protocols in Molecular Biology</EM> (John Wiley &amp; Sons, ed. 4, 1999) pg. A1-36</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingSix"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseSix" aria-expanded="false" aria-controls="collapseSix">
          Typical Cycling Conditions for Q5 or Phusion
        </A></H4></DIV><DIV id="collapseSix" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingSix"><DIV class="panel-body"><H3>Typical Cycling Conditions for Q5 or Phusion</H3><P>Typical PCR protocol for a 1,000 bp amplicon</P><TABLE><TBODY><TR><TD><P>1 cycle</P></TD><TD><P>98°C</P></TD><TD><P>30 seconds</P></TD></TR><TR><TD rowspan="3"><P>25 cycles</P></TD><TD><P>98°C</P></TD><TD><P>10 seconds</P></TD></TR><TR><TD><P>55°C</P></TD><TD><P>15 seconds</P></TD></TR><TR><TD><P>72°C</P></TD><TD><P>15 seconds</P></TD></TR><TR><TD><P>1 cycle</P></TD><TD><P>72°C</P></TD><TD><P>5 minutes (to finish replication on all templates)</P></TD></TR><TR><TD><P>1 cycle</P></TD><TD><P>4-10°C</P></TD><TD><P>indefinite period (storing the sample prior to further analysis)</P></TD></TR></TBODY></TABLE><P><STRONG>Polymerase Chain Reaction (PCR) for Fragment Assembly of gblocks </STRONG></P><P><STRONG>Equipment:</STRONG></P><OL><LI>Heat block or thermo-cycler</LI><LI>Micro-pipettes (P10 &amp; P20)</LI><LI>Thin-walled PCR Tubes</LI><LI>Microcentrifuge with PCR adapters</LI><LI>Small beaker and ice (Ice is found in the Autoclave room 202)</LI></OL><P><STRONG>Methods</STRONG></P><P>We will be amplifying <STRONG>two versions</STRONG> of the gene blocks with a N terminus and C terminus His tag.</P><P>You will be using different primers depending on which protein you are creating, so be sure to grab the right ones!</P><P>If you have<STRONG> the N-Terminal</STRONG>, the primers you will be using are: </P><P><STRONG>Nterm Forward</STRONG></P><P>atgcaccaccaccaccaccacATGACGGTCACGAACGAG 3'Tm=62.4 3'Ta(annealing temp)=62.1</P><P><STRONG>Nterm Reverse</STRONG></P><P>tatcaagcttatcgacatTCACTTGTCCGAATAGGC 3'Tm=59.1 3'Ta(annealing temp)=62.1</P><P>If you have <STRONG>the C-Terminal</STRONG>, the primers you will be using are: </P><P><STRONG>Cterm Forward:</STRONG></P><P>cggacctattgcgcatATGACGGTCACGAACGAG 3'Tm=62.4 3'Ta(annealing temp)=65.4</P><P><STRONG>Cterm Reverse:</STRONG></P><P>tcagtggtggtggtggtggtgCTTGTCCGAATAGGCCGTG3'Tm=63.7 3'Ta(annealing temp)=65.4</P><P><STRONG>Before you begin</STRONG>, always prepare the thermocycler using the conditions below. The thermocycler takes time to warm up, so run the appropriate protocol as you are mixing so that you can place the PCR reaction in the machine <STRONG>as soon as possible</STRONG>. </P><P>In this experiment, you will be preparing the thermocycler for a Touchdown PCR. The annealing temperature for the first round of cycles will decrease by 0.2°C each cycle for <STRONG>a total temperature drop of 1°C/5 cycles. </STRONG>Use the appropriate parameters below depending on which reaction you are running.</P><TABLE><TBODY><TR><TD colspan="9"><P>C-terminal cycles (15 cycles, 10 cycles)</P></TD></TR><TR><TD><P>|&lt;-------</P></TD><TD><P>---------</P></TD><TD><P> x 15 |</P></TD><TD><P>|&lt;-------</P></TD><TD><P>---------</P></TD><TD><P>-- x 10| </P></TD></TR><TR><TD><P>98°C</P></TD><TD><P>98°C</P></TD><TD><P>65°C*</P></TD><TD><P>72°C</P></TD><TD><P>98°C</P></TD><TD><P>65°C</P></TD><TD><P>72°C</P></TD><TD><P>72°C</P></TD><TD><P>4°C</P></TD></TR><TR><TD><P>5:00</P></TD><TD><P>0:10</P></TD><TD><P>0:10</P></TD><TD><P>1:00</P></TD><TD><P>0:10</P></TD><TD><P>0:10</P></TD><TD><P>1:00</P></TD><TD><P>5:00</P></TD><TD><P>Hold</P></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD colspan="9"><P>N-terminal cycles (10 cycles, 15 cycles)</P></TD></TR><TR><TD><P>|&lt;-------</P></TD><TD><P>---------</P></TD><TD><P>-- x 10| </P></TD><TD><P>|&lt;-------</P></TD><TD><P>---------</P></TD><TD><P> x 5* |</P></TD></TR><TR><TD><P>98°C</P></TD><TD><P>98°C</P></TD><TD><P>62°C*</P></TD><TD><P>72°C</P></TD><TD><P>98°C</P></TD><TD><P>62°C</P></TD><TD><P>72°C</P></TD><TD><P>72°C</P></TD><TD><P>4°C</P></TD></TR><TR><TD><P>5:00</P></TD><TD><P>0:10</P></TD><TD><P>0:10</P></TD><TD><P>1:00</P></TD><TD><P>0:10</P></TD><TD><P>0:10</P></TD><TD><P>1:00</P></TD><TD><P>2:00**</P></TD><TD><P>Hold</P></TD></TR></TBODY></TABLE><P>Use the following protocol if 2X Q5 <STRONG>Mastermix is not available </STRONG>:</P><TABLE><TBODY><TR><TD><P><STRONG>COMPONENTS</STRONG></P></TD><TD><P><STRONG>25 µL RXN</STRONG></P></TD><TD><P><STRONG>50 µL RXN</STRONG></P></TD><TD><P><STRONG>FINAL </STRONG><STRONG>CONC.</STRONG></P></TD></TR><TR><TD><P>5X Q5 Reaction Buffer</P></TD><TD><P>5µL</P></TD><TD><P>10 µL</P></TD><TD><P>1X</P></TD></TR><TR><TD><P>10 mM dNTPS</P></TD><TD><P>0.5 µL</P></TD><TD><P>1 µL</P></TD><TD><P>200 µM</P></TD></TR><TR><TD><P>10 uM Forward Primer</P></TD><TD><P>1.25 µL</P></TD><TD><P>2.5 µL</P></TD><TD><P>0.5 µM</P></TD></TR><TR><TD><P>10 uM Reverse Primer</P></TD><TD><P>1.25 µL</P></TD><TD><P>2.5 µL</P></TD><TD><P>0.5 µM</P></TD></TR><TR><TD><P>Template DNA</P></TD><TD><P>0.5 uL</P></TD><TD><P>1 uL</P></TD><TD><P>&lt;1,000 ng</P></TD></TR><TR><TD><P>Q5 High-Fidelity DNA Polymerase</P></TD><TD><P>0.25 µL</P></TD><TD><P>0.5 µL</P></TD><TD><P>0.02 U/µl</P></TD></TR><TR><TD><P>5X Q5 High GC Enhancer (optional)</P></TD><TD><P>(5 µL)</P></TD><TD><P>(10 µL)</P></TD><TD><P>(1X)</P></TD></TR><TR><TD><P>Nuclease-Free Water</P></TD><TD><P>11.75 ul</P></TD><TD><P>23.50</P></TD></TR></TBODY></TABLE><P><STRONG>If Mastermix is available, </STRONG>use the following protocol:</P><TABLE><TBODY><TR><TD><P><STRONG>COMPONENTS</STRONG></P></TD><TD><P><STRONG>25 µL RXN</STRONG></P></TD></TR><TR><TD><P>MiliQ H20</P></TD><TD><P>4.5 uL</P></TD></TR><TR><TD><P>5X PCR Enhancer</P></TD><TD><P>5 uL</P></TD></TR><TR><TD><P>10 uM Forward Primer</P></TD><TD><P>1.25 uL</P></TD></TR><TR><TD><P>10 uM Reverse Primer</P></TD><TD><P>1.25 uL</P></TD></TR><TR><TD><P>Diluted pTA963 Plasmid</P></TD><TD><P>0.5 uL</P></TD></TR><TR><TD><P>2X Q5 MasterMix</P></TD><TD><P>12.5 uL</P></TD></TR></TBODY></TABLE><P><STRONG>For a 50 uL reaction, simply double the above amounts. </STRONG></P><P>Obtain thin-walled PCR tubes – be sure to wear gloves when removing them from the container. Then, distribute the appropriate volume of primers, plasmid and water into the tube. <STRONG>Do not add DNA Polymerase or Mastermix, this will go last and only when you are ready to run the reaction in the thermocycler. Place all reagent and reaction tubes on ice while you wait.</STRONG></P><P>It’s best to start with the appropriate volume of H20, then add all the other reagents. </P><P>Mix contents of each tube thoroughly, either by by pipetting up and down slowly, OR placing the tube on a vortex mixer and microcentrifuge for about a second. </P><P>Make sure your tube is well labelled with group members, the specific plasmid you are amplifying, and the date.</P><P><STRONG>Clean- Up</STRONG></P><P>Dump ice into the sink and place ice box in designated area on side bench </P><P>Place all remaining plasmid and primer samples back in the fridge. </P><P><STRONG>References: </STRONG></P><P>Ruben, Giulia, “Lab 4 Protocol #2, Polymerase Chain Reaction”, Biochemistry Laboratory, Winter 2015</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingSeven"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseSeven" aria-expanded="false" aria-controls="collapseSeven">
          Sample for PCR (ALDY3&amp;5, ADH2)Sample for PCR (ALDY3&amp;5, ADH2)
        </A></H4></DIV><DIV id="collapseSeven" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingSeven"><DIV class="panel-body"><P>Sample for PCR (ALDY3&amp;5, ADH2)</P><P>For a 50µl reaction:</P><OL><LI>10µl reaction buffer</LI><LI>1µl of 10mM dNTPs</LI><LI>2.5µl (10µM forward primer)</LI><LI>2.5µl (10µM reverse primer)</LI><LI>1µl template</LI><LI>add 10µl of PCR enhancer</LI><LI>22.75µl of miliQH2O (add first)</LI><LI>0.25µl of Q5 polymerase (add lasr)</LI></OL><H3> </H3><P>For the template:</P><UL><LI>example, if 140 ng/µl</LI><UL><LI>take 20µl of 140 ng/µl</LI><LI>add 20µl of miliQH2O</LI></UL><LI>then add 1µl of 70ng/µl template to 50µl reaction</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingEight"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseEight" aria-expanded="false" aria-controls="collapseEight">
          Gibson Assembly Protocol
        </A></H4></DIV><DIV id="collapseEight" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingEight"><DIV class="panel-body"><P><STRONG>Gibson Protocol (from board)</STRONG></P><UL><LI>10 μL of master mix</LI><LI>50 ng of plasmid and fragments (&gt; 3 μL of each)</LI><LI>Fill to 20 μL with milliQ</LI><LI>50℃ for 3 hours</LI><LI>4℃ hold</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingNine"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseNine" aria-expanded="false" aria-controls="collapseNine">
          PCR Clean-Up Kit Protocol
        </A></H4></DIV><DIV id="collapseNine" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingNine"><DIV class="panel-body"><P>Protocol for making the kit:</P><P>Membrane Binding Buffer:</P><OL><LI>26.5g of GITC</LI><LI>8.33ml of 3M potassium acetate </LI><LI>adjust volume to 50ml with H20</LI></OL><P>Membrane Wash Solution:</P><OL><LI>100X stock </LI><OL><LI>3.32ml of 3M potassium acetate </LI><LI>33.4µl of 0.5M EDTA </LI><LI>adjust volume to 100ml with H20</LI></OL><LI>10X working solution</LI><OL><LI>7.99ml of 100% ethanol</LI><LI>1ml of 100X stock </LI><LI>1.0ml of H20</LI></OL></OL><P>Protocol for using the kit:</P><OL><LI>add volume of sample to column </LI><LI>add 7X the amount of binding buffer to the sample in column </LI><LI>centrifuge at 16,000G for 1min</LI><OL><LI>discard flow through </LI></OL><LI>add 700µl of working solution wash </LI><LI>centrifuge at 16,000G for 1min</LI><OL><LI>discard flow through </LI></OL><LI>add 500µl of working solution wash</LI><LI>centrifuge at 16,000G for 5min</LI><OL><LI>discard flow through </LI></OL><LI>without adding any wash, centrifuge at 16,000G for 1min</LI><OL><LI>discard flow through </LI></OL></OL><OL start="9"><LI> transfer column to eppendorf tube </LI><LI> add 20µl of HOT miliq H20 to the column </LI><LI> centrifuge at 16,000G for 1min</LI><LI> nanodrop to check 260/230 is above 1 </LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTen"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseTen" aria-expanded="false" aria-controls="collapseTen">
          Field Sample Collection Protocol
        </A></H4></DIV><DIV id="collapseTen" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTen"><DIV class="panel-body"><P><STRONG>Materials</STRONG></P><UL><LI>gloves</LI><LI>disposable bags</LI><LI>50mL sterile capped sampling tubes </LI><LI>1 bottle of distilled water (from the milliQ) - about 500 ml</LI><LI>tape (labeling)</LI><LI>sharpie</LI><LI>pH probe</LI><LI>salinity meter</LI><LI>thermometer</LI><LI>styrofoam ice chest with ice</LI><LI>towel</LI><LI>camera</LI><LI>long-reach sampling pole and attaching tape (duct tape) </LI><LI>field notebook </LI><LI>spare clothes</LI></UL><P><STRONG>Prep Procedure:</STRONG></P><OL><LI>Obtain all materials. </LI><LI>Go through procedure with team</LI><LI>Fill ice chest with ice before leaving. </LI></OL><P><STRONG>Field Procedure:</STRONG></P><OL><LI>Distinguish varying sampling environments.</LI><LI>Look for sources of contamination to avoid (trash, oil, etc.).</LI><LI>Label falcon tube with identifier and initials of person sampling</LI><LI>Collect samples; fill 50 mL tube to the top and cap off immediately </LI><LI>For each sample, record time and date, explicit location with detail, and take photos.</LI><LI>Transport in styrofoam ice chest. </LI></OL><P><STRONG>Salinity and pH Procedure:</STRONG></P><OL><LI>Create 1:10 dilution of liquid sample with 5 mL of sample and 45 mL of DI water</LI><LI>Perform proper meter calibration</LI><LI>Measure pH and salinity using meter</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingEleven"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseEleven" aria-expanded="false" aria-controls="collapseEleven">
          Insoluble MCC Liquid Media Protocol
        </A></H4></DIV><DIV id="collapseEleven" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingEleven"><DIV class="panel-body"><P><STRONG>Materials:</STRONG></P><P>For 1 bottle of 1L Sol-MCC liquid media</P><OL><LI>Make up 30% SW</LI><LI>Into 1L Erlenmeyer flask</LI></OL><TABLE><TBODY><TR><TD><P>Distilled H20</P></TD><TD><P>330 ml</P></TD></TR><TR><TD><P>30% SW</P></TD><TD><P>600 ml</P></TD></TR></TBODY></TABLE><OL><LI>Add to mix</LI></OL><TABLE><TBODY><TR><TD><P>1 M Tris.HCl (pH 7.5)</P></TD><TD><P>30 ml</P></TD></TR></TBODY></TABLE><OL><LI>Add slowly to mix</LI></OL><TABLE><TBODY><TR><TD><P>Insoluble MCC </P></TD><TD><P>25.5 ml </P></TD></TR><TR><TD><P>Hv-Min Salts</P></TD><TD><P>12 ml</P></TD></TR><TR><TD><P>0.5 M KPO buffer (pH 7.5)</P></TD><TD><P>1950 µl</P></TD></TR><TR><TD><P>Thiamine &amp; Biotin</P></TD><TD><P>900 µl</P></TD></TR></TBODY></TABLE><OL><LI>Stir thoroughly for 15 minutes</LI><LI>Adjust pH with HCl/NaOH to 7.5 carefully</LI><LI>Pour mix into individuals flasks and top with foil</LI><LI>Autoclave for 30 minutes</LI></OL><P><STRONG>Medium Solutions and Components</STRONG></P><P><STRONG>Insoluble MCC Carbon Source</STRONG></P><OL><LI>Weigh out 3 g of MCC on scale in erlenmeyer flask</LI><LI>Pour 25.5 ml DI H2O into flask</LI></OL><P><STRONG>Trace Elements</STRONG></P><OL><LI>To 100 ml water add a few drops of concentrated HCl (be careful)</LI></OL><P>Dissolve the following one-by-one</P><TABLE><TBODY><TR><TD><P>MnCl2*4H20</P></TD><TD><P>36 mg</P></TD></TR><TR><TD><P>ZnSO4*7H20</P></TD><TD><P>44 mg</P></TD></TR><TR><TD><P>FeSO4*7H20</P></TD><TD><P>230 mg</P></TD></TR><TR><TD><P>CuSO4*5H20</P></TD><TD><P>0.4 ml </P></TD></TR></TBODY></TABLE><P><STRONG>Hv-Min Salts</STRONG></P><OL><LI>For 36 ml Hv-Min salts combine</LI></OL><TABLE><TBODY><TR><TD><P>1 M NH4Cl</P></TD><TD><P>30 ml</P></TD></TR><TR><TD><P>Trace elements</P></TD><TD><P>6 ml</P></TD></TR></TBODY></TABLE><OL><LI>Store in fridge</LI></OL><P><STRONG>Thiamine &amp; Biotin</STRONG></P><OL><LI>For 10.8 ml combine</LI></OL><TABLE><TBODY><TR><TD><P>Thiamine (1 mg/ml)</P></TD><TD><P>9.6 ml</P></TD></TR><TR><TD><P>Biotin (1 mg/ml)</P></TD><TD><P>1.2 ml</P></TD></TR></TBODY></TABLE><P><STRONG>0.5 M KPO</STRONG><STRONG>4</STRONG><STRONG> Buffer pH 7.5</STRONG></P><OL><LI>For 200 ml 0.5 M KPO4 buffer (pH 7.5)</LI></OL><TABLE><TBODY><TR><TD><P>1 M K2HPO4</P></TD><TD><P>83.4 ml</P></TD></TR><TR><TD><P>1 M KH2PO4</P></TD><TD><P>16.6 ml</P></TD></TR></TBODY></TABLE><OL><LI>Adjust pH to 7.5 by slowly adding HCl/NaOH. Record concentration of acid/base.</LI><LI>Add equal volume (100 ml) of DI H20</LI><LI>Store at room temperature</LI></OL><P>Edited with <A href="http://www.html-cleaner.com/" target="_blank">Online HTML Converter</A>.</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingThirteen"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseThirteen" aria-expanded="false" aria-controls="collapseThirteen">
          Soluble MCC Liquid Media Protocol
        </A></H4></DIV><DIV id="collapseThirteen" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThirteen"><DIV class="panel-body"><P>For 1 bottle of 1.05555 L Sol-MCC liquid media</P><OL><LI>Make up 30% SW</LI><LI>Into Erlenmeyer flask</LI></OL><TABLE><TBODY><TR><TD><P>Distilled H20</P></TD><TD><P>330 ml</P></TD></TR><TR><TD><P>30% SW</P></TD><TD><P>600 ml</P></TD></TR></TBODY></TABLE><OL><LI>Add to mix</LI></OL><TABLE><TBODY><TR><TD><P>1 M Tris.HCl (pH 7.5)</P></TD><TD><P>30 ml</P></TD></TR></TBODY></TABLE><OL><LI>Add slowly to mix</LI></OL><TABLE><TBODY><TR><TD><P>Thawed Solubilized MCC </P></TD><TD><P>80.7 ml </P></TD></TR><TR><TD><P>Hv-Min Salts</P></TD><TD><P>12 ml</P></TD></TR><TR><TD><P>0.5 M KPO4 buffer (pH 7.5)</P></TD><TD><P>1950 µl</P></TD></TR><TR><TD><P>Thiamine &amp; Biotin</P></TD><TD><P>900 µl</P></TD></TR></TBODY></TABLE><OL><LI>Stir thoroughly for 15 minutes</LI><LI>Adjust pH to ~7.5 carefully. If stored, pH may rise back to around 8.0 </LI><LI>Autoclave for 30 min</LI><LI>Pour mix into individuals labeled flasks and top with foil</LI></OL><P><STRONG>Medium Solutions and Components</STRONG></P><P><STRONG>Solubilized MCC Carbon Source</STRONG></P><OL><LI>Create 14.9 wt% NaOH Solution</LI><OL><LI>If solid NaOH is used</LI><OL><LI>Place erlenmeyer flask on scale and set to zero</LI><LI>Weigh out 14.9 g of NaOH in flask</LI><LI>Add DI H20 until solution reaches 100 g</LI></OL><LI>If 4M NaOH solution is used</LI><OL><LI>To create 100 mL of 14 wt% combine</LI></OL></OL></OL><TABLE><TBODY><TR><TD><P>4.0M NaOH </P></TD><TD><P> 93.1 ml</P></TD></TR><TR><TD><P>DI H20</P></TD><TD><P>6.9 ml</P></TD></TR></TBODY></TABLE><OL><LI>In 3 individual erlenmeyer flasks add 2.0 g of MCC</LI><LI>Add 26.9 ml of 14 wt% NaOH to each flask</LI><LI>Stir until solution swells</LI><LI>Freeze solution overnight</LI><LI>Thaw at room temperature until solution becomes gel-like</LI></OL><P><STRONG>Trace Elements</STRONG></P><OL><LI>To 100 ml water add a few drops of concentrated HCl (be careful)</LI></OL><P>Dissolve the following one-by-one</P><TABLE><TBODY><TR><TD><P>MnCl2*4H20</P></TD><TD><P>36 mg</P></TD></TR><TR><TD><P>ZnSO4*7H20</P></TD><TD><P>44 mg</P></TD></TR><TR><TD><P>FeSO4*7H20</P></TD><TD><P>230 mg</P></TD></TR><TR><TD><P>CuSO4*5H20</P></TD><TD><P>0.4 ml</P></TD></TR></TBODY></TABLE><P><STRONG>Hv-Min Salts</STRONG></P><OL><LI>For 72 ml Hv-Min salts combine</LI></OL><TABLE><TBODY><TR><TD><P>1 M NH4Cl</P></TD><TD><P>30 ml</P></TD></TR><TR><TD><P>Trace elements</P></TD><TD><P>6 ml</P></TD></TR><TR><TD><P>0.5 M CaCl2</P></TD><TD><P>36 ml</P></TD></TR></TBODY></TABLE><OL><LI>Store in fridge</LI></OL><P><STRONG>Thiamine &amp; Biotin</STRONG></P><OL><LI>For 10.8 ml combine</LI></OL><TABLE><TBODY><TR><TD><P>Thiamine (1 mg/ml)</P></TD><TD><P>9.6 ml</P></TD></TR><TR><TD><P>Biotin (1 mg/ml)</P></TD><TD><P>1.2 ml</P></TD></TR></TBODY></TABLE><P><STRONG>0.5 M KPO</STRONG><STRONG>4</STRONG><STRONG> Buffer pH 7.5</STRONG></P><OL><LI>For 200 ml 0.5 M KPO4 buffer (pH 7.5)</LI></OL><TABLE><TBODY><TR><TD><P>1 M K2HPO4</P></TD><TD><P>83.4 ml</P></TD></TR><TR><TD><P>1 M KH2PO4</P></TD><TD><P>16.6 ml</P></TD></TR></TBODY></TABLE><OL><LI>Adjust pH to 7.5 by slowly adding HCl/NaOH. Record concentration of acid/base.</LI><LI>Add equal volume (100 ml) of DI H20</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingFourteen"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFourteen" aria-expanded="false" aria-controls="collapseFourteen">
          Soluble MCC Agar Plate Protocol
        </A></H4></DIV><DIV id="collapseFourteen" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFourteen"><DIV class="panel-body"><P>For 1 bottle of 1.05555 L Sol-MCC liquid media</P><P> </P><OL><LI>Make up 30% SW</LI><LI>Into Erlenmeyer flask</LI></OL><TABLE><TBODY><TR><TD><P>Distilled H20</P></TD><TD><P>330 ml</P></TD></TR><TR><TD><P>30% SW</P></TD><TD><P>600 ml</P></TD></TR></TBODY></TABLE><P> </P><OL><LI>Add to mix</LI></OL><TABLE><TBODY><TR><TD><P>1 M Tris.HCl (pH 7.5)</P></TD><TD><P>30 ml</P></TD></TR></TBODY></TABLE><P> </P><OL><LI>Add slowly to mix</LI></OL><TABLE><TBODY><TR><TD><P>Thawed Solubilized MCC</P></TD><TD><P>80.7 ml</P></TD></TR><TR><TD><P>Hv-Min Salts</P></TD><TD><P>12 ml</P></TD></TR><TR><TD><P>0.5 M KPO4 buffer (pH 7.5)</P></TD><TD><P>1950 µl</P></TD></TR><TR><TD><P>Thiamine &amp; Biotin</P></TD><TD><P>900 µl</P></TD></TR></TBODY></TABLE><P> </P><OL><LI>Stir thoroughly for 15 minutes</LI><LI>Adjust pH to ~7.5 carefully. If stored, pH may rise back to around 8.0</LI><LI>Add 15g of Agar and mix</LI><LI>Autoclave for 30 min</LI><LI>Pour mix into individuals labeled plates and cool before storing</LI></OL><P><STRONG>Medium Solutions and Components</STRONG></P><P> </P><P><STRONG>Solubilized MCC Carbon Source</STRONG></P><P> </P><OL><LI>Create 14.9 wt% NaOH Solution</LI><OL><LI>If solid NaOH is used</LI><OL><LI>Place erlenmeyer flask on scale and set to zero</LI><LI>Weigh out 14.9 g of NaOH in flask</LI><LI>Add DI H20 until solution reaches 100 g</LI></OL><LI>If 4M NaOH solution is used</LI><OL><LI>To create 100 mL of 14 wt% combine</LI></OL></OL></OL><TABLE><TBODY><TR><TD><P>4.0M NaOH</P></TD><TD><P>93.1 ml</P></TD></TR><TR><TD><P>DI H20</P></TD><TD><P>6.9 ml</P></TD></TR></TBODY></TABLE><P> </P><OL><LI>In 3 individual erlenmeyer flasks add 2.0 g of MCC</LI><LI>Add 26.9 ml of 14 wt% NaOH to each flask</LI><LI>Stir until solution swells</LI><LI>Freeze solution overnight</LI><LI>Thaw at room temperature until solution becomes gel-like</LI></OL><P> </P><P><STRONG>Trace Elements</STRONG></P><P> </P><OL><LI>To 100 ml water add a few drops of concentrated HCl (be careful)</LI></OL><P>Dissolve the following one-by-one</P><TABLE><TBODY><TR><TD><P>MnCl2*4H20</P></TD><TD><P>36 mg</P></TD></TR><TR><TD><P>ZnSO4*7H20</P></TD><TD><P>44 mg</P></TD></TR><TR><TD><P>FeSO4*7H20</P></TD><TD><P>230 mg</P></TD></TR><TR><TD><P>CuSO4*5H20</P></TD><TD><P>0.4 ml</P></TD></TR></TBODY></TABLE><P><STRONG>Hv-Min Salts</STRONG></P><P> </P><OL><LI>For 72 ml Hv-Min salts combine</LI></OL><P> </P><TABLE><TBODY><TR><TD><P>1 M NH4Cl</P></TD><TD><P>30ml</P></TD></TR><TR><TD><P>Trace elements</P></TD><TD><P>6 ml</P></TD></TR><TR><TD><P>0.5 M CaCl2</P></TD><TD><P>36 ml</P></TD></TR></TBODY></TABLE><P> </P><OL><LI>Store in fridge</LI></OL><P><STRONG>Thiamine &amp; Biotin</STRONG></P><P> </P><OL><LI>For 10.8 ml combine</LI></OL><TABLE><TBODY><TR><TD><P>Thiamine (1 mg/ml)</P></TD><TD><P>9.6 ml</P></TD></TR><TR><TD><P>Biotin (1 mg/ml)</P></TD><TD><P>1.2 ml</P></TD></TR></TBODY></TABLE><P> </P><P><STRONG>0.5 M KPO</STRONG><STRONG>4</STRONG><STRONG> Buffer pH 7.5</STRONG></P><P> </P><OL><LI>For 200 ml 0.5 M KPO4 buffer (pH 7.5)</LI></OL><TABLE><TBODY><TR><TD><P>1 M K2HPO4</P></TD><TD><P>83.4 ml</P></TD></TR><TR><TD><P>1 M KH2PO4</P></TD><TD><P>16.6 ml</P></TD></TR></TBODY></TABLE><P> </P><OL><LI>Adjust pH to 7.5 by slowly adding HCl/NaOH. Record concentration of acid/base.</LI><LI>Add equal volume (100 ml) of DI H20</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingFifteen"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFifteen" aria-expanded="false" aria-controls="collapseFifteen">
          MGM Plate Protocol 
        </A></H4></DIV><DIV id="collapseFifteen" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFifteen"><DIV class="panel-body"><P>per Litre:</P><P> </P><OL><LI>Add the following ingredients (in a large beaker):</LI></OL><TABLE><TBODY><TR><TD><P>12 % MGM</P></TD><TD><P>18% MGM</P></TD><TD><P>23% MGM</P></TD><TD><P>25% MGM</P></TD></TR><TR><TD><P>Salt Water</P></TD><TD><P>400</P></TD><TD><P>600</P></TD><TD><P>767</P></TD><TD><P>833</P></TD></TR><TR><TD><P>Pure Water</P></TD><TD><P>567</P></TD><TD><P>367</P></TD><TD><P>200</P></TD><TD><P>134</P></TD></TR><TR><TD><P>Peptone (oxoid)*</P></TD><TD><P>5</P></TD><TD><P>5</P></TD><TD><P>5</P></TD><TD><P>5</P></TD></TR><TR><TD><P>Yeast Extract</P></TD><TD><P>1</P></TD><TD><P>1</P></TD><TD><P>1</P></TD><TD><P>1</P></TD></TR></TBODY></TABLE><P>*Don’t use Difco Bacto-Peptone, as it was reported in 1988 that it contained bile salts that lyse halobacteria (Kamekura et al., 1988). This was still the case in 2001, when tested by Thorsten Allers!</P><P> </P><OL start="2"><LI>Stir to dissolve (you may have to warm slightly). Adjust the pH up to 7.5 with 1M Tris.Cl, pH 7.5, using 5 ml per Litre. If this is not sufficient, then use 1M Tris base, drop-wise, to reach the final pH. In Melbourne, Australia, I had no problems adjusting and maintaining the pH, but in Germany the pH would drop to 5.5 after autoclaving, unless I used at least 10 mM Tris. Note that Tris.Cl is inhibitory if &gt; 45mM.</LI><LI>Adjust volume to 1000 ml with pure water (about another 30 ml water) For solid medium: Plates, add 15g Difco Bacto-agar; Top Layer Agar (for virus plaquing or transformations), add 7 g. It is better if the agar is largely dissolved before autoclaving, i.e. by heating to 100°C for 10-20 min. However with large volumes this can be difficult. The longer autoclaving time (30 min) should overcome this problem. The brand of agar is important, see note below at the end of section 2.4. Protocols for halobacterial genetics © Ver 7.2, March 2009 page 15</LI><LI>Sterilize (e.g. in a large, wide-mouthed 2 – 3 L flask) for 30 min, 101 kPa (15 lb). Cool to 55 – 60°C and pour into Petri dishes (see note below on pouring plates).</LI><LI>After autoclaving, make sure the agar is mixed before pouring as it tends to settle at the bottom. Pour relatively hot (about 60°C) as the high salt concentration makes the agar set easily. Bubbles are more difficult to get rid of (i.e. by flaming with a bunsen burner) than in normal media so be careful.</LI></OL><P><STRONG>6. Dry plates before use (30 – 60 min, 37°C, upside-down, with lids removed; or more conveniently, overnight at RT on the bench). Store plates at 4°C, wrapped in plastic film. </STRONG></P></DIV></DIV></DIV></DIV><DIV id="foot" class="center foot">  <A href="https://2015.igem.org/Team:UCSC">Home</A> | <A href="https://2015.igem.org/Team:UCSC/Team">Teams</A> | <A href="https://2015.igem.org/Team:UCSC/Project">Project</A> | <A href="https://2015.igem.org/Team:UCSC/Practices">Human Practices</A> | <A href="https://2015.igem.org/Team:UCSC/Notebook">Notebook</A> | <A href="https://2015.igem.org/Team:UCSC/Safety">Safety</A> | <A href="https://2015.igem.org/Team:UCSC/Parts">Achievements</A> | <A href="https://2015.igem.org/Team:UCSC/Software">F.O.C.U.S</A>  </DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>