Team:UCSC/Protocols
Home
Team
Meet the Team
Breakdown
Fermentation
Field
Project
Human Practices
Outreach
Bioethics
Notebook
Logs
Protocols
Safety
Achievements
Parts
Medal Fulfillment
Results
Software
Protocols
A compilation of the protocols used at UCSC: from Gibson to MGM.
To view our Notebook click here
Agarose Gel Protocol.
Preparing DNA and Running in Gel Electrophoresis
Reagents:
10X TBE Buffer
Deionized Water
Agarose Powder
Gel Loading Dye Purple 6X spiked with SYBRGold
DNA solutions
DNA Ladder
Equipment:
Gel Doc-it UV imaging machine (Purple and grey with green tape)
Computer with Launch VisionWorksLS program open
Gel electrophoresis Box (with non-morphed electrodes) and Lid
Power Supply box
Microwave
Rotation Mixer
Weight
Methods:
Protocol 1: Preparing the Gel
Create 500 mL of 1X TBE buffer by mixing 50 mL of 10X TBE buffer and 450 mL of deionized H20
For a 1% (w/v) gel (for very large DNA), take 50 mL of 1X TBE buffer and mix with the appropriate percent mass of Agarose powder (Weigh out 0.5 g for a 1% gel)
Mix thoroughly using the rotation mixer (optional)
Loosely plug flask with paper towel and heat in microwave for approximately 1 minute, or until the agarose is fully dissolved. The liquid should be completely clear.
Once completed, wait till the beaker cools and prepare to add solution to the gel tray. Put comb into the tray and make sure that the tray is oriented in the gel box so that its ends are blocked against the sides of the box.
SLOWLY pour solution into gel tray to avoid creating bubbles.
Allow gel to solidify and cool for 20 - 30 minutes or until fully solidified. This can be tested by how easy it is to move the pick.
Once solidified, gently turn the gel tray with the gel so that the ends are exposed.
Fill the gel box with 1X TBE buffer until gel is fully covered (0.5 - 1 cm above gel)
Slowly and carefully remove the comb so as not to tear the wells.
Protocol 2: Staining the DNA Samples and Running Gel
Caution: Gel box has high voltage! Be sure to turn off power pack and unplug leads before removing lid of gel box.
You will be staining the DNA using Professor Bernick’s Droplet Technique. Depending on how many samples you have (including controls and DNA ladder), place 1 uL droplets of 6X Loading Dye on a piece of parafilm.
Take 5 uL of each DNA sample and add to an individual droplet. You will have a final volume of 6uL. Be sure to mix thoroughly up and down, then take the sample and inject into a well.
Make sure to change pipet tips for each DNA sample so as to prevent contamination! Keep track of the positions of each sample
Once all samples have been loaded, close the gel box and attach all electrodes to the gel box and to the power pack. (Use color coding to be sure all electrodes are attached correctly. Black = negative electrode and Red = positive. The electrode closest to the samples will be negative while the further is positive.
Turn on the power pack and set to 120 V. You should immediately see bubbles rising from the wires in the gel box.
Run continuously for 75 minutes. Once complete, turn of the power pack.
Protocol 3: Visualizing the Gel
Remove the lid from the gel box (Be sure that the power pack is turned off!)
Pour out the 1x TBE buffer into the used TBE buffer beaker
Open the Gel Doc it door and pull out the UV transilluminator. (Make sure it is off!)
Transfer the gel to the UV transilluminator tray, push it back into the machine and flip the on switch on the lower right corner.
Close the door and make sure the knob is turned to Syber Gold. Go to the computer and open the Launch VisionWorks program.
Flick the UV light switch and press preview on the computer. When the gel is clearly visible and you can see the bands, press capture and save it to the folder IGEM2015.
References:
Ruben, Giulia, “Lab 2 Protocol #1, Gel Electrophoresis”, Biochemistry Laboratory, Winter 2015
Colony PCR Protocols for Transformed E.coli
Colony PCR Protocol for Transformed E. coli
Equipment
Thermocycler
Micro-pipettes (P10, P20 & P1000)
Thin-walled PCR Tubes
Eppendorf tubes
Microcentrifuge with PCR and Eppendorf tube adapters
Ice bucket and ice (Ice is found in the Autoclave room 202)
Reagents
Fresh Miliq water (Found in the Autoclave room 202)
SOC Media
Titaq 2x Master Mix
Methods
After plating your electroporated E. coli cells, choose about 6 large cultures with the least amount of micro satellites.
Take a micropipet tip and gently touch it to the colony (No need to scrape as this will be too many cells)
Place the micropipet into an eppendorf tube with 10 uL of water to create a 1:10 dilution of DNA, and slowly pipet up and down to lyse the cells.
Take 1uL of this dilution and use it as DNA template for your PCR reaction
Plasmid Specific Primer Options
Breakdown Primers:
bGlu_Seq_Fw1: 5’ CCCTCGATTTTCCGCCTGCCGATTA 3’
bGlu_Seq_Rv4: 5’ GCGCTCTAGAACTAGTGGATCCCCC 3’
Dominic Primers:
Aldy5Seq1F: 5’-TACTTCACATTCGCGGACCTATTG - 3’
Aldy5Seq5R: 5’-AGAACTAGTGGATCCCCCG - 3’
Professor Bernick Primers:
PCR Ingredients
Components
25 uL Reaction
Titaq 2x Master Mix
12.5 uL
Forward Primer
0.5 uL
Reverse Primer
0.5 uL
Template DNA
1 uL
PCR Enhancer
5 uL
Miliq Water
5.5 uL
References
Professor David Bernick, Lab Advisor
Polymerase Chain Reaction (PCR) for Amplification of pTA963 Expression Plasmids
Polymerase Chain Reaction (PCR) for Amplification of pTA963 Expression Plasmids
Equipment:
Heat block or thermo-cycler
Micro-pipettes (P10 & P20)
Thin-walled PCR Tubes
Microcentrifuge with PCR adapters
Small beaker and ice (Ice is found in the Autoclave room 202)
Methods
We will be amplifying two versions of the pTA963 expression plasmid: One which allows the insertions of a 6 x Histidine Tag on the N-Terminus region, and other which allows insertion in the C- Terminus region.
You will be using different primers depending on which plasmid you are amplifying, so be sure to grab the right ones!
If you have the N-Terminal, the primers you will be using are:
pTA963Nterm Forward
pTA963Nterm Reverse
If you have the C-Terminal, the primers you will be using are:
pTA963Cterm Forward:
CGATAAGCTTGATATCGAATTCCTG
pTA963Cterm Reverse:
ATGCGCAATAGGTCCGCG
Before you begin, always prepare the thermocycler using the conditions below. The thermocycler takes time to warm up, so run the appropriate protocol as you are mixing so that you can place the PCR reaction in the machine as soon as possible.
In this experiment, you will be preparing the thermocycler for a Touchdown PCR. The annealing temperature for the first round of cycles will decrease by 0.2°C each cycle for a total temperature drop of 1°C/5 cycles. Use the appropriate parameters below depending on which reaction you are running.
C-terminal cycles (15 cycles, 10 cycles)
|<-------
---------
x 15 |
|<-------
---------
-- x 10|
98°C
98°C
66°C*
72°C
98°C
63°C
72°C
72°C
4°C
5:00
0:10
0:10
3:00
0:10
0:10
3:00
5:00
Hold
*Touchdown is a -0.2 degree per cycle for a total of 3 degree decrease.
N-terminal cycles (10 cycles, 15 cycles)
|<-------
---------
-- x 10|
|<-------
---------
x 5 |
98°C
98°C
64°C*
72°C
98°C
62°C
72°C
72°C
4°C
5:00
0:10
0:10
3:00
0:10
0:10
3:00
5:00
Hold
*Touchdown is -0.2 degrees per cycle for a total of 2 degree decrease
Use the following protocol if 2X Q5 Mastermix is not available :
COMPONENTS
25 µL RXN
50 µL RXN
FINAL
CONC.
5X Q5 Reaction Buffer
5µL
10 µL
1X
10 mM dNTPS
0.5 µL
1 µL
200 µM
10 uM Forward Primer
1.25 µL
2.5 µL
0.5 µM
10 uM Reverse Primer
1.25 µL
2.5 µL
0.5 µM
Template DNA
0.5 uL
1 uL
<1,000 ng
Q5 High-Fidelity DNA Polymerase
0.25 µL
0.5 µL
0.02 U/µl
PCR Enhancer (optional)
(5 µL)
(10 µL)
(1X)
Nuclease-Free Water
11.25 ul
22.50
If Mastermix is available, use the following protocol:
COMPONENTS
25 µL RXN
50 uL RXN
MiliQ H20
4.5 uL
9 uL
5X PCR Enhancer
5 uL
10 uL
10 uM Forward Primer
1.25 uL
2.5 uL
10 uM Reverse Primer
1.25 uL
2.5 uL
Diluted pTA963 Plasmid
0.5 uL
1 uL
2X Q5 MasterMix
12.5 uL
25 uL
Obtain thin-walled PCR tubes – be sure to wear gloves when removing them from the container. Then, distribute the appropriate volume of primers, plasmid and water into the tube. Do not add DNA Polymerase or Mastermix, this will go last and only when you are ready to run the reaction in the thermocycler. Place all reagent and reaction tubes on ice while you wait.
It’s best to start with the appropriate volume of H20, then add all the other reagents.
Mix contents of each tube thoroughly, either by by pipetting up and down slowly, OR placing the tube on a vortex mixer and microcentrifuge for about a second.
Make sure your tube is well labelled with group members, the specific plasmid you are amplifying, and the date.
Clean- Up
Dump ice into the sink and place ice box in designated area on side bench
Place all remaining plasmid and primer samples back in the fridge.
References:
Ruben, Giulia, “Lab 4 Protocol #2, Polymerase Chain Reaction”, Biochemistry Laboratory, Winter 2015
Dominic Schenone, lab advisor
Protocol for Fragment Assembly and Gene Isolation from H. hispanica
Protocol for Fragment Assembly and Gene Isolation from H. Hispanica
 
Equipment:
Heat block
Thermocycler
Micro-pipettes (P10, P20 & P1000)
Thin-walled PCR Tubes
Eppendorf tubes
Microcentrifuge with PCR and Eppendorf tube adapters
Ice bucket and ice (Ice is found in the Autoclave room 202)
2 Liter Beaker
 
Reagents
Fresh Miliq water (Also found in the Autoclave room 202)
23% Modified Growth Medium
 
Methods
Fragment Assembly Protocol
Obtain the appropriate amounts of fragment 1, 2 and 3 that provide an equal molar concentration of each fragment. In this case, the concentrations are:
Fragment 1: 0.88 uL
Fragment 2: 1.25 uL
Fragment 3: 1 uL
 
Use the following primers:
bGlu_Fw_flag:
5'CCTGCCGATTACTTCACATTCGCGGACC
TATTGCGCATATGGCACATGACACGACTGACG 3'
bGlu_Rv_flag:
5' TTCGATATCAAGCTTATCGTCAGTGGTG
GTGGTGGTGGTGCGAGCGACCGCGGACCTC 3'
 
Set the thermocycler to the following conditions:
 
Fragment Assembly cycles (25 cycles)
|<-------------
----------------
x 25 |
98°C
98°C
65°C*
72°C
72°C
4°C
5:00
0:10
0:10
1:00
5:00
Hold
Gene Isolation Protocol
 
Growth of Haloarcula Hispanica
Your cells need to be grown in 23% Modified Growth media till they have reached an A 600reading between 0.4 - 0.8.
 
To make a liter of 23% Modified Growth media, use the following ingredients:
 
Salt Water (30% Stock)
767
Pure Water
200
Peptone( Oxoid)
5
Yeast Extract
1
*Don’t use Difco Bacto-Peptone, as it was reported in 1988 that it contained bile salts that lyse halobacteria.
 
Rapid Chromosomal DNA Isolation
Spin down 1 mL of a freshly grown Hbt. salinarum culture at exponential or early stationary
phase (1 minute at max. rpm in a microcentrifuge)
 
Suck off the supernatant.
 
Add 400 μL pure water and lyse the cells by gently pipetting. The cells disrupt immediately by
osmotic lysis.
 
Inactivate the proteins by heating the DNA solution up to 70°C for 10 min.
 
Comments:
a) DNA from freshly grown cultures (exponential or early stationary phase) give better results for
further reactions than old ones.
 
b) use 1 – 3 μl as a template for amplifying a DNA fragment by PCR.
c) digest 30 μl (in 100 μl final volume) for southern blot analysis.
d) if there is a problem using the DNA as template for PCR it could be due to much DNA. Try
several concentrations or volumes e.g. 1, 5, 10 μl from stock solution.
 
Nested PCR Part 1: Wild Type Beta Glucosidase Isolation
Use the following primers for the first step of nested PCR:
bGlu_iso_Fw: 5' CACTGTTTGGGTGCGCCTGTCATG 3'
bGlu_iso_Rv: 5' GTAGTGGGTATTCGGGGTCCGGTG 3'
 
Set the thermocycler to the following conditions:
Gene Isolation cycles (25 cycles)
|<-------------
----------------
x 25 |
98°C
98°C
71°C*
72°C
72°C
4°C
5:00
0:10
0:10
0:42 sec
5:00
Hold
 
PCR Reaction Ingredients
 
Use the following protocol if 2X Q5 Mastermix is not available :
COMPONENTS
25 µL RXN
50 µL RXN
FINAL
CONC.
5X Q5 Reaction Buffer
5µL
10 µL
1X
10 mM dNTP
0.5 µL
1 µL
200 µM
10 uM Forward Primer
1.25 µL
2.5 µL
0.5 µM
10 uM Reverse Primer
1.25 µL
2.5 µL
0.5 µM
Template DNA
0.5 uL
1 uL
<1,000 ng
Q5 High-Fidelity DNA Polymerase
0.25 µL
0.5 µL
0.02 U/µl
PCR Enhancer (optional)
(5 µL)
(10 µL)
(1X)
Nuclease-Free Water
11.25 ul
22.50
If Mastermix is available, use the following protocol:
 
COMPONENTS
25 µL RXN
50 uL RXN
MiliQ H20
4.5 uL
9 uL
5X PCR Enhancer
5 uL
10 uL
10 uM Forward Primer
1.25 uL
2.5 uL
10 uM Reverse Primer
1.25 uL
2.5 uL
Diluted pTA963 Plasmid
0.5 uL
1 uL
2X Q5 MasterMix
12.5 uL
25 uL
 
Obtain thin-walled PCR tubes – be sure to wear gloves when removing them from the container. Then, distribute the appropriate volume of primers, plasmid and water into the tube. Do not add DNA Polymerase or Mastermix, this will go last and only when you are ready to run the reaction in the thermocycler. Place all reagent and reaction tubes on ice while you wait.
 
It’s best to start with the appropriate volume of H20, then add all the other reagents.
 
Mix contents of each tube thoroughly, either by by pipetting up and down slowly, OR placing the tube on a vortex mixer and microcentrifuge for about a second.
 
Make sure your tube is well labelled with group members, the specific plasmid you are amplifying, and the date.
 
Clean- Up
Dump ice into the sink and place ice box in designated area on side bench
Place all remaining plasmid and primer samples back in the fridge.
 
References:
Ruben, Giulia, 2015 “Lab 4 Protocol #2, Polymerase Chain Reaction”, Biochemistry Laboratory
 
Dyall-Smith, Mike., Rodriguez-Valera., 2008 “Modified Growth Medium (MGM) for Haloarchea”, The Halohandbook: Protocols for Haloarcheal Genetics (7th ed.). 14-15
 
Dyall-Smith, Mike., Pfieffer, Matthias., 2008 “Alternative Rapid Chromosomal DNA Isolation”, The Halohandbook: Protocols for Haloarcheal Genetics (7th ed.). 70
Protocol Taken from the Halohandbook Page 14 by Dr Mike Dyall-Smith
SOC Media
Ingredients
SOB
20 mM glucose
Protocol
Follow directions to make 1 liter of SOB media
After cooling medium to less than 50°C, add 20 ml filter sterilized 20% glucose solution
 
Summary
SOB Medium. Used in growing bacteria for preparing chemically competent cells
Ingredients
0.5% (w/v) yeast extract
2% (w/v) tryptone
10 mM NaCl
2.5 mM KCl
20 mM MgSO4
Per liter:
5 g yeast extract
20 g tryptone
0.584 g NaCl
0.186 g KCl
2.4 g MgSO4
Note: Some formulations of SOB use 10 mM MgCl2 and 10 mM MgSO4 instead of 20 mM MgSO4.
SOB medium is also available dry premixed from Difco, 0443-17.
Adjust to pH 7.5 prior to use. This requires approximately 25 ml of 1M NaOH per liter.
 
15/10 medium
Growth of competent TOP10 cells in Example 2 of the Bloom05 patent is performed in 15/10 broth, which is similar to SOB:
1.5% yeast extract
1% Bacto-Tryptone
10mM NaCl
2mM KCl
10 mM MgCl2
10 mM MgSO4
Extended instructions
1 M MgCl2 stock: dissolve 20.33 g MgCl2 6H2O in 100 ml ddH2O (XXX), autoclave on liquid cycle @ XXX°C for 20 min (can be done at the same time as SOC pre-mix below)
250 mM KCl stock: dissolve 1.86 KCl in 100 ml ddH2O (XXX)
combine:
Reagent
for 1 L
500 mL
100 mL
tryptone
20 g
10 g
2g
yeast
5g
2.5 g
0.5 g
NaCl
0.5 g
0.25 g
0.05 g
250 nM KCl
10 mL
5mL
1 mL
ddH2O
980 ml
490 ml
98 ml
adjust pH to 7.0 w/ NaOH
bring to volume:
Reagent
for 1 L
500 mL
100 mL
autoclave on liquid cycle @ XXX°C for 20 min
add autoclaved 1 M MgCl2
Reagent
for 1 L
500 mL
100 mL
1 M MgCl2
10 ml
5 ml
1 ml
 
Source
Adapted From:
Ausubel et al., Short Protocols in Molecular Biology (John Wiley & Sons, ed. 4, 1999) pg. A1-36
Typical Cycling Conditions for Q5 or Phusion
Typical Cycling Conditions for Q5 or Phusion
Typical PCR protocol for a 1,000 bp amplicon
1 cycle
98°C
30 seconds
25 cycles
98°C
10 seconds
55°C
15 seconds
72°C
15 seconds
1 cycle
72°C
5 minutes (to finish replication on all templates)
1 cycle
4-10°C
indefinite period (storing the sample prior to further analysis)
Polymerase Chain Reaction (PCR) for Fragment Assembly of gblocks
Equipment:
Heat block or thermo-cycler
Micro-pipettes (P10 & P20)
Thin-walled PCR Tubes
Microcentrifuge with PCR adapters
Small beaker and ice (Ice is found in the Autoclave room 202)
Methods
We will be amplifying two versions of the gene blocks with a N terminus and C terminus His tag.
You will be using different primers depending on which protein you are creating, so be sure to grab the right ones!
If you have the N-Terminal, the primers you will be using are:
Nterm Forward
atgcaccaccaccaccaccacATGACGGTCACGAACGAG 3'Tm=62.4 3'Ta(annealing temp)=62.1
Nterm Reverse
tatcaagcttatcgacatTCACTTGTCCGAATAGGC 3'Tm=59.1 3'Ta(annealing temp)=62.1
If you have the C-Terminal, the primers you will be using are:
Cterm Forward:
cggacctattgcgcatATGACGGTCACGAACGAG 3'Tm=62.4 3'Ta(annealing temp)=65.4
Cterm Reverse:
tcagtggtggtggtggtggtgCTTGTCCGAATAGGCCGTG 3'Tm=63.7 3'Ta(annealing temp)=65.4
Before you begin, always prepare the thermocycler using the conditions below. The thermocycler takes time to warm up, so run the appropriate protocol as you are mixing so that you can place the PCR reaction in the machine as soon as possible.
In this experiment, you will be preparing the thermocycler for a Touchdown PCR. The annealing temperature for the first round of cycles will decrease by 0.2°C each cycle for a total temperature drop of 1°C/5 cycles. Use the appropriate parameters below depending on which reaction you are running.
C-terminal cycles (15 cycles, 10 cycles)
|<-------
---------
x 15 |
|<-------
---------
-- x 10|
98°C
98°C
65°C*
72°C
98°C
65°C
72°C
72°C
4°C
5:00
0:10
0:10
1:00
0:10
0:10
1:00
5:00
Hold
N-terminal cycles (10 cycles, 15 cycles)
|<-------
---------
-- x 10|
|<-------
---------
x 5* |
98°C
98°C
62°C*
72°C
98°C
62°C
72°C
72°C
4°C
5:00
0:10
0:10
1:00
0:10
0:10
1:00
2:00**
Hold
Use the following protocol if 2X Q5 Mastermix is not available :
COMPONENTS
25 µL RXN
50 µL RXN
FINAL
CONC.
5X Q5 Reaction Buffer
5µL
10 µL
1X
10 mM dNTPS
0.5 µL
1 µL
200 µM
10 uM Forward Primer
1.25 µL
2.5 µL
0.5 µM
10 uM Reverse Primer
1.25 µL
2.5 µL
0.5 µM
Template DNA
0.5 uL
1 uL
<1,000 ng
Q5 High-Fidelity DNA Polymerase
0.25 µL
0.5 µL
0.02 U/µl
5X Q5 High GC Enhancer (optional)
(5 µL)
(10 µL)
(1X)
Nuclease-Free Water
11.75 ul
23.50
If Mastermix is available, use the following protocol:
COMPONENTS
25 µL RXN
MiliQ H20
4.5 uL
5X PCR Enhancer
5 uL
10 uM Forward Primer
1.25 uL
10 uM Reverse Primer
1.25 uL
Diluted pTA963 Plasmid
0.5 uL
2X Q5 MasterMix
12.5 uL
For a 50 uL reaction, simply double the above amounts.
Obtain thin-walled PCR tubes – be sure to wear gloves when removing them from the container. Then, distribute the appropriate volume of primers, plasmid and water into the tube. Do not add DNA Polymerase or Mastermix, this will go last and only when you are ready to run the reaction in the thermocycler. Place all reagent and reaction tubes on ice while you wait.
It’s best to start with the appropriate volume of H20, then add all the other reagents.
Mix contents of each tube thoroughly, either by by pipetting up and down slowly, OR placing the tube on a vortex mixer and microcentrifuge for about a second.
Make sure your tube is well labelled with group members, the specific plasmid you are amplifying, and the date.
Clean- Up
Dump ice into the sink and place ice box in designated area on side bench
Place all remaining plasmid and primer samples back in the fridge.
References:
Ruben, Giulia, “Lab 4 Protocol #2, Polymerase Chain Reaction”, Biochemistry Laboratory, Winter 2015
Sample for PCR (ALDY3&5, ADH2)Sample for PCR (ALDY3&5, ADH2)
Sample for PCR (ALDY3&5, ADH2)
For a 50µl reaction:
10µl reaction buffer
1µl of 10mM dNTPs
2.5µl (10µM forward primer)
2.5µl (10µM reverse primer)
1µl template
add 10µl of PCR enhancer
22.75µl of miliQH2O (add first)
0.25µl of Q5 polymerase (add lasr)
 
For the template:
example, if 140 ng/µl
take 20µl of 140 ng/µl
add 20µl of miliQH2O
then add 1µl of 70ng/µl template to 50µl reaction
Gibson Assembly Protocol
Gibson Protocol (from board)
10 μL of master mix
50 ng of plasmid and fragments (> 3 μL of each)
Fill to 20 μL with milliQ
50℃ for 3 hours
4℃ hold
PCR Clean-Up Kit Protocol
Protocol for making the kit:
Membrane Binding Buffer:
26.5g of GITC
8.33ml of 3M potassium acetate
adjust volume to 50ml with H20
Membrane Wash Solution:
100X stock
3.32ml of 3M potassium acetate
33.4µl of 0.5M EDTA
adjust volume to 100ml with H20
10X working solution
7.99ml of 100% ethanol
1ml of 100X stock
1.0ml of H20
Protocol for using the kit:
add volume of sample to column
add 7X the amount of binding buffer to the sample in column
centrifuge at 16,000G for 1min
discard flow through
add 700µl of working solution wash
centrifuge at 16,000G for 1min
discard flow through
add 500µl of working solution wash
centrifuge at 16,000G for 5min
discard flow through
without adding any wash, centrifuge at 16,000G for 1min
discard flow through
transfer column to eppendorf tube
add 20µl of HOT miliq H20 to the column
centrifuge at 16,000G for 1min
nanodrop to check 260/230 is above 1
Field Sample Collection Protocol
Materials
gloves
disposable bags
50mL sterile capped sampling tubes
1 bottle of distilled water (from the milliQ) - about 500 ml
tape (labeling)
sharpie
pH probe
salinity meter
thermometer
styrofoam ice chest with ice
towel
camera
long-reach sampling pole and attaching tape (duct tape)
field notebook
spare clothes
Prep Procedure:
Obtain all materials.
Go through procedure with team
Fill ice chest with ice before leaving.
Field Procedure:
Distinguish varying sampling environments.
Look for sources of contamination to avoid (trash, oil, etc.).
Label falcon tube with identifier and initials of person sampling
Collect samples; fill 50 mL tube to the top and cap off immediately
For each sample, record time and date, explicit location with detail, and take photos.
Transport in styrofoam ice chest.
Salinity and pH Procedure:
Create 1:10 dilution of liquid sample with 5 mL of sample and 45 mL of DI water
Perform proper meter calibration
Measure pH and salinity using meter
Insoluble MCC Liquid Media Protocol
Materials:
For 1 bottle of 1L Sol-MCC liquid media
Make up 30% SW
Into 1L Erlenmeyer flask
Distilled H20
330 ml
30% SW
600 ml
Add to mix
1 M Tris.HCl (pH 7.5)
30 ml
Add slowly to mix
Insoluble MCC
25.5 ml
Hv-Min Salts
12 ml
0.5 M KPO buffer (pH 7.5)
1950 µl
Thiamine & Biotin
900 µl
Stir thoroughly for 15 minutes
Adjust pH with HCl/NaOH to 7.5 carefully
Pour mix into individuals flasks and top with foil
Autoclave for 30 minutes
Medium Solutions and Components
Insoluble MCC Carbon Source
Weigh out 3 g of MCC on scale in erlenmeyer flask
Pour 25.5 ml DI H2O into flask
Trace Elements
To 100 ml water add a few drops of concentrated HCl (be careful)
Dissolve the following one-by-one
MnCl2*4H20
36 mg
ZnSO4*7H20
44 mg
FeSO4*7H20
230 mg
CuSO4*5H20
0.4 ml
Hv-Min Salts
For 36 ml Hv-Min salts combine
1 M NH4Cl
30 ml
Trace elements
6 ml
Store in fridge
Thiamine & Biotin
For 10.8 ml combine
Thiamine (1 mg/ml)
9.6 ml
Biotin (1 mg/ml)
1.2 ml
0.5 M KPO4 Buffer pH 7.5
For 200 ml 0.5 M KPO4 buffer (pH 7.5)
1 M K2HPO4
83.4 ml
1 M KH2PO4
16.6 ml
Adjust pH to 7.5 by slowly adding HCl/NaOH. Record concentration of acid/base.
Add equal volume (100 ml) of DI H20
Store at room temperature
Edited with Online HTML Converter .
Soluble MCC Liquid Media Protocol
For 1 bottle of 1.05555 L Sol-MCC liquid media
Make up 30% SW
Into Erlenmeyer flask
Distilled H20
330 ml
30% SW
600 ml
Add to mix
1 M Tris.HCl (pH 7.5)
30 ml
Add slowly to mix
Thawed Solubilized MCC
80.7 ml
Hv-Min Salts
12 ml
0.5 M KPO4 buffer (pH 7.5)
1950 µl
Thiamine & Biotin
900 µl
Stir thoroughly for 15 minutes
Adjust pH to ~7.5 carefully. If stored, pH may rise back to around 8.0
Autoclave for 30 min
Pour mix into individuals labeled flasks and top with foil
Medium Solutions and Components
Solubilized MCC Carbon Source
Create 14.9 wt% NaOH Solution
If solid NaOH is used
Place erlenmeyer flask on scale and set to zero
Weigh out 14.9 g of NaOH in flask
Add DI H20 until solution reaches 100 g
If 4M NaOH solution is used
To create 100 mL of 14 wt% combine
4.0M NaOH
93.1 ml
DI H20
6.9 ml
In 3 individual erlenmeyer flasks add 2.0 g of MCC
Add 26.9 ml of 14 wt% NaOH to each flask
Stir until solution swells
Freeze solution overnight
Thaw at room temperature until solution becomes gel-like
Trace Elements
To 100 ml water add a few drops of concentrated HCl (be careful)
Dissolve the following one-by-one
MnCl2*4H20
36 mg
ZnSO4*7H20
44 mg
FeSO4*7H20
230 mg
CuSO4*5H20
0.4 ml
Hv-Min Salts
For 72 ml Hv-Min salts combine
1 M NH4Cl
30 ml
Trace elements
6 ml
0.5 M CaCl2
36 ml
Store in fridge
Thiamine & Biotin
For 10.8 ml combine
Thiamine (1 mg/ml)
9.6 ml
Biotin (1 mg/ml)
1.2 ml
0.5 M KPO4 Buffer pH 7.5
For 200 ml 0.5 M KPO4 buffer (pH 7.5)
1 M K2HPO4
83.4 ml
1 M KH2PO4
16.6 ml
Adjust pH to 7.5 by slowly adding HCl/NaOH. Record concentration of acid/base.
Add equal volume (100 ml) of DI H20
Soluble MCC Agar Plate Protocol
For 1 bottle of 1.05555 L Sol-MCC liquid media
 
Make up 30% SW
Into Erlenmeyer flask
Distilled H20
330 ml
30% SW
600 ml
 
Add to mix
1 M Tris.HCl (pH 7.5)
30 ml
 
Add slowly to mix
Thawed Solubilized MCC
80.7 ml
Hv-Min Salts
12 ml
0.5 M KPO4 buffer (pH 7.5)
1950 µl
Thiamine & Biotin
900 µl
 
Stir thoroughly for 15 minutes
Adjust pH to ~7.5 carefully. If stored, pH may rise back to around 8.0
Add 15g of Agar and mix
Autoclave for 30 min
Pour mix into individuals labeled plates and cool before storing
Medium Solutions and Components
 
Solubilized MCC Carbon Source
 
Create 14.9 wt% NaOH Solution
If solid NaOH is used
Place erlenmeyer flask on scale and set to zero
Weigh out 14.9 g of NaOH in flask
Add DI H20 until solution reaches 100 g
If 4M NaOH solution is used
To create 100 mL of 14 wt% combine
4.0M NaOH
93.1 ml
DI H20
6.9 ml
 
In 3 individual erlenmeyer flasks add 2.0 g of MCC
Add 26.9 ml of 14 wt% NaOH to each flask
Stir until solution swells
Freeze solution overnight
Thaw at room temperature until solution becomes gel-like
 
Trace Elements
 
To 100 ml water add a few drops of concentrated HCl (be careful)
Dissolve the following one-by-one
MnCl2*4H20
36 mg
ZnSO4*7H20
44 mg
FeSO4*7H20
230 mg
CuSO4*5H20
0.4 ml
Hv-Min Salts
 
For 72 ml Hv-Min salts combine
 
1 M NH4Cl
30ml
Trace elements
6 ml
0.5 M CaCl2
36 ml
 
Store in fridge
Thiamine & Biotin
 
For 10.8 ml combine
Thiamine (1 mg/ml)
9.6 ml
Biotin (1 mg/ml)
1.2 ml
 
0.5 M KPO4 Buffer pH 7.5
 
For 200 ml 0.5 M KPO4 buffer (pH 7.5)
1 M K2HPO4
83.4 ml
1 M KH2PO4
16.6 ml
 
Adjust pH to 7.5 by slowly adding HCl/NaOH. Record concentration of acid/base.
Add equal volume (100 ml) of DI H20
MGM Plate Protocol
per Litre:
 
Add the following ingredients (in a large beaker):
12 % MGM
18% MGM
23% MGM
25% MGM
Salt Water
400
600
767
833
Pure Water
567
367
200
134
Peptone (oxoid)*
5
5
5
5
Yeast Extract
1
1
1
1
*Don’t use Difco Bacto-Peptone, as it was reported in 1988 that it contained bile salts that lyse halobacteria (Kamekura et al., 1988). This was still the case in 2001, when tested by Thorsten Allers!
 
Stir to dissolve (you may have to warm slightly). Adjust the pH up to 7.5 with 1M Tris.Cl, pH 7.5, using 5 ml per Litre. If this is not sufficient, then use 1M Tris base, drop-wise, to reach the final pH. In Melbourne, Australia, I had no problems adjusting and maintaining the pH, but in Germany the pH would drop to 5.5 after autoclaving, unless I used at least 10 mM Tris. Note that Tris.Cl is inhibitory if > 45mM.
Adjust volume to 1000 ml with pure water (about another 30 ml water) For solid medium: Plates, add 15g Difco Bacto-agar; Top Layer Agar (for virus plaquing or transformations), add 7 g. It is better if the agar is largely dissolved before autoclaving, i.e. by heating to 100°C for 10-20 min. However with large volumes this can be difficult. The longer autoclaving time (30 min) should overcome this problem. The brand of agar is important, see note below at the end of section 2.4. Protocols for halobacterial genetics © Ver 7.2, March 2009 page 15
Sterilize (e.g. in a large, wide-mouthed 2 – 3 L flask) for 30 min, 101 kPa (15 lb). Cool to 55 – 60°C and pour into Petri dishes (see note below on pouring plates).
After autoclaving, make sure the agar is mixed before pouring as it tends to settle at the bottom. Pour relatively hot (about 60°C) as the high salt concentration makes the agar set easily. Bubbles are more difficult to get rid of (i.e. by flaming with a bunsen burner) than in normal media so be careful.
6. Dry plates before use (30 – 60 min, 37°C, upside-down, with lids removed; or more conveniently, overnight at RT on the bench). Store plates at 4°C, wrapped in plastic film.
   Home | Teams | Project | Human Practices | Notebook | Safety | Achievements | F.O.C.U.S   
