<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCSC_Logs skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCSC/Logs</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV><UL class="nav navbar-nav bernick_swag"><LI class="active"><A href="https://2015.igem.org/Team:UCSC">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Team </A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Team">Meet the Team</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Breakdown">Breakdown</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Fermentation">Fermentation</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Field">Field</A></LI></UL><LI><A href="https://2015.igem.org/Team:UCSC/Project">Project</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Practices">Bioethics</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Logs">Logs</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Safety">Safety</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Medal Fulfillment">Medal Fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:UCSC/Software">Software</A></LI></DIV></DIV><DIV id="our_content"><H1> Daily Logs </H1><P style="text-align: center;"> To view our Protocols click <A href="https://2015.igem.org/Team:UCSC/Protocols">here</A></P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
          Breakdown Team
        </A></H4></DIV><DIV id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body"><P><STRONG>June 2015</STRONG></P><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, June 24</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Discussed overall project goals for the research period</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Need to test the efficiency of multiple cellulases</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">preferably those that are already present in halophiles and is compatible with </SPAN><EM><SPAN style="font-weight: 400;">Haloferax volcanii</SPAN></EM></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">identified the positions of the four different cellulases associated with </SPAN><EM><SPAN style="font-weight: 400;">Halorhabdus utahensis</SPAN></EM></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Cellulosome construction possibility</SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, June 25</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Looked through the halophile phylogenetic tree and found that</SPAN><EM><SPAN style="font-weight: 400;"> Haloquadratum walsbyi </SPAN></EM><SPAN style="font-weight: 400;">(Hwa) is closer to</SPAN><EM><SPAN style="font-weight: 400;"> Haloferax volcanii</SPAN></EM><SPAN style="font-weight: 400;">, and has cellulase that occurs in the natural metabolic pathway as well </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Did a pairwise alignment between </SPAN><EM><SPAN style="font-weight: 400;">Halorhabdus utahensis</SPAN></EM><SPAN style="font-weight: 400;"> (Hut) and </SPAN><EM><SPAN style="font-weight: 400;">Haloquadratum walsbyi </SPAN></EM><SPAN style="font-weight: 400;">(Hwa)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Concluded that it was probably not a good idea to risk using Haloquadratum walsbyi since it only contains a “probable” cellulase, and it does not contain any other beta-glucanase enzymes</SPAN></LI></OL></OL><P><STRONG>Friday, June 26</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Learned how to design primers for Gibson Assembly</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Have two potential cellulase candidates </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">one from</SPAN><EM><SPAN style="font-weight: 400;"> H.tiamaten</SPAN></EM></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">one from </SPAN><EM><SPAN style="font-weight: 400;">H.utahensis</SPAN></EM></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Using pTA963 as an expression plasmid </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Most of the cellulases seen need C-terminal His Tag since the N-terminus is found in the transmembrane </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Need to use inverse PCR to linearize and amplify plasmid. This will also remove 6x His Tag and stop codon </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">His Tag and stop codon will be added using flagged primers </SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Introduced to idea of codon optimization</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Worked on the pI Finder program, which takes in a FASTA amino acid sequence and calculates the theoretical isoelectric point</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Narrowed criteria for usable cellulases: </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Signal Peptide for excretion or transmembrane position</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">pI range between 4 and 5</SPAN></LI></OL></OL><P><STRONG>Monday, June 29</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Worked with different plasmid editors and decided on Geneious </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Narrowed down the cellulase candidates from CAZy with respect to the signal peptide sequences </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Introduction to PredSignal. Software that uses hidden markov models to predict the presence and purpose of archaeal signal peptides </SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Selection options for testing with cellulose: </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Rayon</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Filter paper that’s been bleached</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Avicel microcrystalline cellulose </SPAN></LI></OL></OL><P><STRONG>Tuesday, June 30</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continued the narrowing down of the cellulase candidates with respect to the presence of signal peptide</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found the sequence of the cellulase (Hu-CBH1) from </SPAN><EM><SPAN style="font-weight: 400;">H. utahensis</SPAN></EM><SPAN style="font-weight: 400;"> that has been proven to work in the paper “Identification of a haloalkaliphilic and thermostable cellulase with improved ionic liquid tolerance” Zhang, Tao et al </SPAN></LI></OL><P><SPAN style="font-weight: 400;"> </SPAN></P><P><STRONG>July 2015</STRONG></P><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, July 1</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Have enzymes categories as beta glucanase and alpha amylase, but also looking for endo glucanase and exo glucanase</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Discuss whether or not we can utilize the cellobiohydrolase Hu-CBH1 since it is patented </SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, July 2 </STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Weekly team meeting (Moved to Thursday due to 4th of July Weekend) </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Discussed a more narrow focus on Exoglucanases, Endoglucanases and B-glucosidases </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">We did not want to focus on Alpha amylases since they are used for breaking down starch </SPAN></LI></OL></OL><P><STRONG>Monday, July 6</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Design protocols for inverse PCR and Gibson Assembly</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Performed inverse PCR to linearize and amplify our expression plasmid (pta963) </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Protocols labelled as “ Polymerase Chain Reaction (PCR) for Amplification of pTA963 Expression Plasmids” and “Gibson Assembly Protocol”</SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, July 7</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Acquired Q5 2x Master mix for PCR reactions</SPAN></LI><LI style="font-weight: 400;"><EM><SPAN style="font-weight: 400;">H.tiamaten</SPAN></EM><SPAN style="font-weight: 400;"> cellulase → homologs present, but still need to find the extension number</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Gene is about 2,700 nucleotides long x 3 ~ 8,100 </SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">We have about 4 gene candidates + </SPAN><EM><SPAN style="font-weight: 400;">H.tiamaten</SPAN></EM><SPAN style="font-weight: 400;"> cellulase that has already been tested</SPAN></LI></OL><P><SPAN style="font-weight: 400;"> </SPAN></P><P><STRONG>Wednesday, July 8</STRONG></P><P><SPAN style="font-weight: 400;">GOALS</SPAN></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Thursday 7/9 → Inverse PCR (2 experiments)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Friday 7/10 → Analysis - Gel/ plate prep</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Wednesday 7/15 → Gibson/ PCR Analysis/ Gel, Transformation of E.coli</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">pick a strain of E.coli</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">chemically competent</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Thursday 7/16 → Plate</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Monday 7/20 Mini-Prep/ Sequencing Prep</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, July 9</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Need to prepare gene blocks for: </SPAN></LI></OL><P><SPAN style="font-weight: 400;">C-Terminus</SPAN><SPAN style="font-weight: 400;">                           </SPAN><SPAN style="font-weight: 400;">N-Terminus</SPAN><SPAN style="font-weight: 400;">                                 </SPAN><SPAN style="font-weight: 400;">?</SPAN></P><P><SPAN style="font-weight: 400;">-</SPAN><EM><SPAN style="font-weight: 400;">Shewanella</SPAN></EM><SPAN style="font-weight: 400;">                          -</SPAN><EM><SPAN style="font-weight: 400;">Shewanella </SPAN></EM><SPAN style="font-weight: 400;">              -cellobiose phosphorylase </SPAN></P><P><SPAN style="font-weight: 400;">-</SPAN><EM><SPAN style="font-weight: 400;">tiamaten</SPAN></EM></P><P><SPAN style="font-weight: 400;">-Hu-CBH1   </SPAN></P><P><SPAN style="font-weight: 400;">-b-glucosidase (</SPAN><EM><SPAN style="font-weight: 400;">hispanica</SPAN></EM><SPAN style="font-weight: 400;">)     -</SPAN><EM><SPAN style="font-weight: 400;">H.tiamaten   </SPAN></EM><SPAN style="font-weight: 400;">    </SPAN><SPAN style="font-weight: 400;">       -B 1,4 glucanase (</SPAN><EM><SPAN style="font-weight: 400;">utahensis</SPAN></EM><SPAN style="font-weight: 400;">)</SPAN></P><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, July 10</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">While choosing cellulase candidates</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Checked for signal peptides</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Searched through the archaeal genome browser</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">As of now we are deciding between:</SPAN><EM><SPAN style="font-weight: 400;"> H.utahensis, H.hispanica, H.tiamaten</SPAN></EM></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found that we already have stock of </SPAN><EM><SPAN style="font-weight: 400;">H.hispanica</SPAN></EM><SPAN style="font-weight: 400;"> in the freezer</SPAN></LI></OL><P><STRONG>Saturday, July 11</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Meeting to design gene blocks for: </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Shewenella fusion gene</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Hu-CBH1 cellobiohydrolase </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Cellobiose phosphorylase </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Pyruvate decarboxylase </SPAN></LI></OL></OL><P><SPAN style="font-weight: 400;">Note: Used IDT Gene Block Editor/ oligoanalyzer 3.1</SPAN></P><P><SPAN style="font-weight: 400;">Picture </SPAN></P><P><STRONG>Monday, July 13</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">TBE buffer was autoclaved this morning</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran inverse PCR to amplify N-term pTA 963 and ran on a gel </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Professor Bernick teaching PCR Fragment Assembly, Gibson Assembly and Flagged primers using sticks</SPAN></LI></OL><P><SPAN style="font-weight: 400;">Picture</SPAN></P><P><STRONG>Tuesday, July 14</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Reran inverse PCR, changing annealing temperature and extension time </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Grant Team Meeting:</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">UCSC Crowdfunding page</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Reaching out to department chairs</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Created program for codon optimizing proteins using the codon frequency table and amino acid sequence (Called optimizer.py)</SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, July 15</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found protocol for making growth media for </SPAN><EM><SPAN style="font-weight: 400;">H. Hispanica </SPAN></EM><SPAN style="font-weight: 400;">in the Halohandbook pg. 14</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">23% Modified Growth Medium (MGM)</SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, July 16</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Protocol for DNA isolation for </SPAN><EM><SPAN style="font-weight: 400;">H.hispanica</SPAN></EM><SPAN style="font-weight: 400;"> (page 69 of Halohandbook)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found that DNAWorks algorithm for codon optimization removes accuracy </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">It does not take into account “rare” codons</SPAN></LI></OL></OL><P><STRONG>Friday, July 17</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Weekly team meeting</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Use of DNA works for codon optimization </SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Inverse PCR for amplification of C-term pta963 plasmid </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Paused during extension phase because we forgot touchdown conditions, which may greatly affect results </SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, July 20</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">N-term and 2 C-term PCRs completed  </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">EMERGENCY LAB MEETUP:</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Breakthrough with codon optimization code (FOCUS)</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Want to pay attention to conserved rare codons</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Need to look for this pattern across organism of different domains</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">3 genes to test (All different forms of B. Glucosidase)</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Wild type (</SPAN><EM><SPAN style="font-weight: 400;">H.hispanica</SPAN></EM><SPAN style="font-weight: 400;">) - control </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">optimized (DNA Works) + control</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">optimized (Jairo’s code) F.O.C.U.S</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Assay</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">micro-crystalline cellulose and X-Glu plates to test for enzymatic activity</SPAN></LI></OL></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Professor Bernick explanation of more to test for Signal Peptide: </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">psortB</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">TM Pred</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Introduction to the UCSC archaeal Genome Browser </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Intro to Kasava Research Paper </SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, July 21</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">EMERGENCY BREAKDOWN TEAM MEETUP:</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Our hypothesis: Rare and conserved codons allows better folding of the protein structures = more efficient protein production</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Obtain the secondary structures of the protein sequences through PSS Pred </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Decided to codon optimize our beta glucosidase protein without optimizing the signal peptide since we know signal peptide still works based on the Hu-CBH1 paper </SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, July 22</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Looking for 2,565 bp band for beta-glucosidase gene in </SPAN><EM><SPAN style="font-weight: 400;">Haloarcula hispanica</SPAN></EM></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continued runs for N-term and C-term PCR</SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, July 23</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Streaking in C-term (7/21) due to pause while running</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">too much primer for the N-term; we used 5 uL instead of 1.25 uL</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">codon bias for</SPAN><EM><SPAN style="font-weight: 400;"> H.volcanii </SPAN></EM><SPAN style="font-weight: 400;">and</SPAN><EM><SPAN style="font-weight: 400;"> H.hispanic</SPAN></EM><SPAN style="font-weight: 400;">a are similar</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">completed another N-term and C-term reaction </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">working on perfecting conditions</SPAN></LI></OL></OL><P><STRONG>Friday, July 24</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">General Meeting</SPAN></LI></UL><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Introducing entire team to F.O.C.U.S</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Plan to test the protein expression levels of the beta-glucosidase from Haloarcula hispanica between the wild type, DNA Works Codon Optimized and F.O.C.U.S Codon optimized</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Need to prove that the rate limiting step is the incorporation of a particular tRNA </SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Working out the kinks for isolation of wild type beta glucosidase </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Smears may be due to a high concentration of genomic DNA during isolation</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Use of DPN1 could get rid of non-methylated (i.e non-genomic DNA) which could mean more access to PCR product and amplification </SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, July 27</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Multiple sequence alignment of conserved proteins conserved in Archaea, Bacteria, and Eukaryotes </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Alanyl-tRNA, DNA polymerase III subunit, rRNA dimethylase</SPAN><SPAN style="font-weight: 400;">  </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Further proteins amongst model organisms can be found in the paper “ Universal trees based on large combined protein” Brown, James R. et al</SPAN></LI></OL></OL><P><STRONG>Tuesday, July 28</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Searched for protein sequences and DNA sequences of conserved proteins in model organisms</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Note, some protein sequences in certain organisms are not of equal length to others. Therefore, they may not be orthologs</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Inverse PCR to amplify N-terminal version of plasmid pta963</SPAN></LI></OL><P><STRONG>Wednesday, July 29</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Need to regrow Hispanica to the optimal OD (0.6 - 0.8) for wild type beta glucosidase isolation</SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>August 2015</STRONG></P><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, August 3</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Gel for hispanica isolation did not work on Friday</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Tested to see if the track dye and DNA ladder show up under UV light</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Reran another gel to test if the results were due to track dye or gel</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Redid another isolation since results were negative </SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, August  4</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Isolation of hispanica (2nd time)</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">50 uL Rxn (Iso, and negative control)</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Turned attention towards using the Dali Server for protein structure information and structural alignment comparisons</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Prepared protocol for </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM><SPAN style="font-weight: 400;"> cell electroporation</SPAN></LI></OL><P><STRONG>Wednesday, August 5</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Isolated Hispanica from a different aliquot (Still with an OD of 1.4)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Multiple PCR reactions to find optimal conditions for isolating wild type b-glucosidase (Run at 71 degrees) </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Negative control (No DNA) </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Repeat of the same PCR protocol but NO PCR ENHANCER</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">2x Primers (No PCR Enhancer) </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">2x Template ( NO PCR Enhancer) </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Amplification of the first isolation that worked (tube labelled ++)</SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG> Thursday, August 6</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Redid PCR for gene isolation (Brought the annealing temperature to 66 degrees)</SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, August 7</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Performed isolation from a 3rd aliquot of Hispanica</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Performed PCR after diluting concentration 100 fold for both aliquot 1 and 3</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Meeting with Professor Bernick to discuss one method of measuring importance of rare codons for F.O.C.U.S</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Encoding cost (Method of hyper weighting a codon that is more rare, especially if it is a rare codon in a group of more frequent codons)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Need to develop a training set and test set to measure consistency of rare codon idea</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Prepare sequence information for engineered cellulase developed by 2014 UCSC iGEM team to perform site directed mutagenesis </SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, August 11</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Fragment assembly of codon optimized Beta-Glucosidase </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Fragment concentrations from stock: </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">F1 → 0.88 uL,  F2 → 1.25 uL,  F3 → 1 uL</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">The Fw fixed Flag primer was diluted with 216 uL of TE buffer, vortexed, and stored </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Primers used: Fw Flag Fixed5 and Bglu_ Rv flag </SPAN></LI></OL></OL><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Nested PCR part 2 (adding flag primers to the isolated WT beta Glucosidas</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Primers: Fw Flag fixed5 and Bglu_ WT_Rv flag</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Proteins for F.O.C.U.S</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">EF-G, Ef-Tu, DNA Polymerase III, tRNA synthetase </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Dali server search stored on Kerika</SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, August 12</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Working on getting the codon bias tables and nucleotide sequences to test out FOCUS</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Reinoculated Hispanica to isolate it from a lower OD </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Tested Fragment assembly with 1/10 dilution of fragments and non dilution </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">The lightbulb in the UV transilluminator broke, so weren’t able to capture a picture of the gel </SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, August 13 </STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">General Meeting</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">check 260/280 and 260/230 for the isolation </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">absorbance of nucleic acids/ protein</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Run fragment assembly with 1/5 dilution, not 1/10</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Length of the fragment assembly =  2641 bp </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Length of the wild type = 2644 bp</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Isolation of Hispanica from a culture with an OD of 0.4</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">concentration: 116.4 ng/uL</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">260/280: 1.64</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">260/230: 0.59</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">WT isolation and fragment assembly PCR </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">dilution (1:10) of new isolation </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">diluted fragments and non-diluted fragments</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Annealing temp= 63.8 degrees</SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, August 14 </STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Isolation PCR reaction using isolation from OD 0.4 and 1.4 to see which is better</SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Sunday, August 16</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Running the gel from Friday’s PCR reaction </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">The isolation with an OD of 0.4 worked the best!</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">dilute fragments reaction does not work</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">lower the annealing temperature</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">used an annealing temperature of 62.8 degrees</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Elongation time changed to 45 seconds</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Q5 reads 1000 bp every 15-45 seconds</SPAN></LI></OL></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">PCR Reactions </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">0.4, 1.4, nd, d, i, F, …, _</SPAN></LI></OL></OL><P><STRONG>Monday, August 17</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Prepared 500 mL of LB agar in preparation for growing transformed </SPAN><EM><SPAN style="font-weight: 400;">E. coli </SPAN></EM><SPAN style="font-weight: 400;">cultures</SPAN></LI></OL><UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Protocol: https://www.addgene.org/plasmid-protocols/bacterial-plates/</SPAN></LI></UL></UL><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">nanodrop for linearized PTA963</SPAN></LI></OL><P><SPAN style="font-weight: 400;">Conc.</SPAN><SPAN style="font-weight: 400;">260/280</SPAN><SPAN style="font-weight: 400;">260/230</SPAN></P><UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">C-term (7/21)</SPAN><SPAN style="font-weight: 400;">462.7 ng/uL</SPAN><SPAN style="font-weight: 400;">1.82</SPAN><SPAN style="font-weight: 400;">0.87</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">C-term (7/23)</SPAN><SPAN style="font-weight: 400;">561.9 ng/uL</SPAN><SPAN style="font-weight: 400;">1.83</SPAN><SPAN style="font-weight: 400;">0.93</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">N-term (7/23)</SPAN><SPAN style="font-weight: 400;">794.6 ng/uL</SPAN><SPAN style="font-weight: 400;">1.84</SPAN><SPAN style="font-weight: 400;">1.24</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">N-term (7/28)</SPAN><SPAN style="font-weight: 400;">335.3 ng/uL</SPAN><SPAN style="font-weight: 400;">1.84</SPAN><SPAN style="font-weight: 400;">0.74</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">More PCR reactions</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">0.4 flag, annealing temp @ 70 degrees → Today</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">1.4 re-isolation, run @ 71 degrees → Tomorrow</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Fragment Assembly @ 65 degrees → Tomorrow</SPAN></LI></UL></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, August 18 </STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Bright band present for fragment assembly = SUCCESS!</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Performed purification using Bernick Kit. Results: </SPAN></LI></OL><P><SPAN style="font-weight: 400;">         </SPAN><SPAN style="font-weight: 400;">DNA</SPAN><SPAN style="font-weight: 400;">     </SPAN><SPAN style="font-weight: 400;"> Conc.</SPAN><SPAN style="font-weight: 400;">         </SPAN><SPAN style="font-weight: 400;">260/280</SPAN><SPAN style="font-weight: 400;">   </SPAN><SPAN style="font-weight: 400;">260/230</SPAN></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">N1        32.2ng/ul       1.58           0.14</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">N2        7.4ng/ul         2.07           0.03</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">C1        6.1ng/ul         1.76           0.02</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">C2        14.5 ng/ul      1.57           0.06</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, August 19 </STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Re-running two isolation reactions with an annealing temp of 71 C and 42 sec extension</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Gel extraction of fragment assembly to run Gibson assembly</SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, August 20 </STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">PCR reactions (Annealing temp. 65 degrees)</SPAN></LI></OL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">gel-extracted iso </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">gel-extracted frag (dil)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">non gel-extracted iso </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">non-dil frag</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, August 21</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">To Do:</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Confirm correct isolation band </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">redo fragment assembly</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">1 uL each fragment (10 uM)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">correct concentrations (0.88 uL,1.25 uL,1uL)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">65 degrees annealing temperature</SPAN></LI></OL></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Details for fragment assembly that was gel extracted: </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">19 uL at concentration of 66.4 ng/uL</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">260/280: 2.03</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">260/230: 0.61</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Gibson Assembly </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">2.65 uL of C-term pta963</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">1.2 uL of Fragment Assembly</SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Saturday, August 22</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Inverse PCR of N-Terminal pta963 for Ethanol Team</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Transformed NEB 5-alpha competent </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM><SPAN style="font-weight: 400;"> with Gibson Assembly product</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Attempted to isolate which band corresponded to the wild type Beta Glucosidase</SPAN></LI></OL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Fw_Flg Fixed5 and Seq Rv3 primers, looking for band of 1000</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Sunday, August 23</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Checked transformation plate, put found no growth </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Most likely due to cells having lost competence since they were not stored at -80 C for at least a week</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Prepared reagents for making spheroplast </SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, August 24 </STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Made “Unbuffered spheroplasting solution”, “Buffered spheroplasting solution” and “Regeneration solution”</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Protocol found on pgs. 59-62 of the Halohandbook </SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Redid inverse PCR of N-term pTA963</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Elongation lowered to 2 min and 30 sec </SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, August 26</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Electroporation using electrocompetent </SPAN><EM><SPAN style="font-weight: 400;">E. coli </SPAN></EM><SPAN style="font-weight: 400;">cells </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">1800 volts </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used Electroporation protocol made by Fermentation team </SPAN></LI></OL></OL><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ag Tech Meet up Presentation </SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, August 27</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Nested PCR Part 2 </SPAN></LI></OL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used Seq/iso reaction product from 8/27 as template DNA since it has the brightest band corresponding to the wild type beta glucosidase</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Concentration: 472.2 ng/uL ; 262/280: 1.79 ; 260/230: 0.68</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used Fw Flag Fixed5, Rv Flag and Rv Flag fixed as primers </SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, August 31</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Colony PCR from Transformation plate</SPAN></LI></OL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Picked 6 colonies</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used miliq water to lyse cells (Tubes labelled 1P-6P)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used Flagged primers: Fw Flag Fixed 5 and Rv Flag Fixed </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran another reaction using sequencing primers: Seq Fw2 and Seq Rv2</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>September 2015</STRONG></P><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, September 1</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Meeting with the Dean of Students</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Rerun a gel including fermentation team colony PCR samples and Breakdown sample (D - dilute, and ND- non-Dilute) of wild type isolation test </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Made 236.25 uL of Q5 Master Mix </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Enough for 27 reactions if we use 25 uL PCR reactions</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Recipe (X10): </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">20 uL Q5 Buffer</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">2 uL of dNTP’s</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">0.5 uL of Q5 enzyme</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">11.25 uL of Miliq water </SPAN></LI></OL></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, September 2</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Colony PCR (4 reactions): </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">P2 with iso Primers and PCR enhancer (1:10 dilution) </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">P2 with iso Primers, No PCR enhancer (1:10 dilution) </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">P2 with Flagged Primers, PCR enhancer (1:10 dilution)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">P2 with flagged Primers, No PCR enhancer (1:10 dilution) </SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Electroporation of Fermentation Gibson reactions (G2, G6, G7, and G8)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Colony PCR of P1 - P6 using Sequencing FW1 and Rv4 primers </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Annealing Temp: 59 C</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Extension time: 3 minutes </SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Need to prepare HVCA plates for Spheroplast analysis after transformation </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Protocol for HVCA agar on Pg. 21 of Halohandbook </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Transformants should grow on plates without Uracil </SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, September 4</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">General Meeting </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">FOCUS Discussion </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Transition Probability Modeling </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Only need to produce a model with 2 states: fast and stall (excluding steady state) </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">John has a paper about “modeling ribosomal speed”</SPAN></LI></OL></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Colony PCR of P1 - P6</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Dominic’s plasmid specific primers</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Annealing Temp. 51 </SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Inoculated </SPAN><EM><SPAN style="font-weight: 400;">H. volcanni </SPAN></EM><SPAN style="font-weight: 400;">for making spheroplast </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Optimal absorbance reading at 600 between 0.8- 1.0 for late exponential phase </SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Nested PCR part 2 </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Template (0.4 isolation from 8/13, 8/14 and 8/17) to test which is best </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Primers: Bglu_Fw_flagFixed5 and Bglu_wt_rv_fixedFlag</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Annealing temp. : 69 C as suggested by TM Calculator </SPAN></LI></OL></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Saturday, September 5</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Absorbance reading of inoculated </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii </SPAN></EM></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">0.621 A</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Plated Fermentation Team Electroporated Cells ( G2, G6, G7, and G8)</SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, September 7</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Colony PCR positive controls (using Gibson Assembly reaction as template)</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Sequencing Primers Fw1 and Rv4 (Titaq)   </SPAN><SPAN style="font-weight: 400;">Touchdown: 66 - 59 C</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Sequencing Primers Fw1 and Rv4 (OneTaq) </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Dominic’s Primers Aldy5Seq1F and Aldy5Seq5R (Titaq) </SPAN><SPAN style="font-weight: 400;">Touchdown 66 - 59 C</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Dominic’s Primers Aldy5Seq1F and Aldy5Seq5R (OneTaq)</SPAN></LI></OL></OL><P><STRONG>Wednesday, September 9</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Chose 8 new colony picks from original plate having Transformed E. coli cells with codon optimized B Glucosidase </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Performed colony PCR using Dominic’s primers, Titaq and No PCR Enhancer </SPAN></LI></OL></OL><P><STRONG>Thursday, September 10</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Started Preparing BioBricks of Codon Optimized Beta Glucosidase, and Fermentation enzymes </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Added Ampicilin to LB Agar plates and replated electroporated E. coli cells </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Added 26.3 ul of 50 ng/uL ampicillin to each plate</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Colony PCR using Q5 Master Mix </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">3 Reactions from fermentation and 2 from Breakdown </SPAN></LI></OL></OL><P><STRONG>Friday, September 11</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Professor Bernick Discussion </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Dislikes homopolymer in both Breakdown and Dominic’s Primers </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Prefers Dominic’s primers because they have a base change after the homopolymer </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Suggests 10 cycle touchdown from 56 - 51 using Dominic’s Primers, using Titaq and PCR enhancer </SPAN></LI></OL></OL><P><STRONG>Saturday, September 12</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Verification that primers anneal to the C-term construct of pTA963</SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Aldy5Fw1 anneals from 485 - 508 (15 nucleotides from His Tag)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Aldy5Rv5 anneals from 569 - 587 (31 nucleotides from His Tag)</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">SeqFw1 anneals fromm 462 to 486 </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">SeqRv4 anneals from 570 - 593 </SPAN><SPAN style="font-weight: 400;">(Note, first G is supposed to be a C)</SPAN></LI></OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Might have to redo Gibson Assembly of fragment assembly based on positive control results </SPAN></LI></OL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Sunday, September 13</STRONG></P><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Touchdown of Nested PCR Part 2 </SPAN></LI><OL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Annealing temp from 70 - 65 C</SPAN></LI></OL></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
          Fermentation: Butanol
        </A></H4></DIV><DIV id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><DIV class="panel-body"><P><STRONG>June 2015</STRONG></P><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, June 24</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Outlined overall project goals</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used NCBI’s nucleotide best BLAST to find gene orthologs from the Garcia pathway in </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found all genes in </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;"> except for butyraldehyde dehydrogenase and butanol dehydrogenase. </SPAN></LI></UL></UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used BLAST to find other butyraldehyde and butanol dehydrogenases that could be used in </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Considered using an aldehyde/alcohol dehydrogenase fusion gene.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found a promising fusion gene in </SPAN><EM><SPAN style="font-weight: 400;">Halomonas sp</SPAN></EM><SPAN style="font-weight: 400;">. S2151 called </SPAN><EM><SPAN style="font-weight: 400;">adhE2</SPAN></EM><SPAN style="font-weight: 400;">. Also had a pI near 5, which is ideal for </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Little was known about </SPAN><EM><SPAN style="font-weight: 400;">Halomonas</SPAN></EM><SPAN style="font-weight: 400;"> and the gene was only recently discovered.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Considered using gas chromatography to confirm the presence of butyryl-CoA.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, June 25</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continued to pursue the idea of using gas chromatography to test for butyryl-CoA.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Rob Franks agreed to allow us to use the GC.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Searched for places to purchase butyryl-CoA to create a standard for GC, but it’s very expensive.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Started searching for protocols for gas chromatography of butyryl-CoA and more generally, fatty acids.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, June 26</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Created 10 L of 30% salt water stock to be used in producing media for </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used protocol in pg 20 of Halohandbook.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Prepared Tris buffer (500 mL of pH 8.83 and 40 mL of pH 7.54).</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, June 29</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continued searching for gas chromatography protocols specific to butyryl-CoA.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Considered using qPCR, since finding a protocol is a challenge.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Sources suggest using HPLC to detect butyryl-CoA.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Rob Franks believes HPLC will be better suited for butyryl-CoA.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found a site with potentially helpful primer design rules (</SPAN><A href="http://www.premierbiosoft.com/tech_notes/PCR_Primer_Design.html"><SPAN style="font-weight: 400;">http://www.premierbiosoft.com/tech_notes/PCR_Primer_Design.html</SPAN></A><SPAN style="font-weight: 400;">)</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, June 30</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Decided to stop pursuing gas chromatography of butyryl-CoA. We are </SPAN><EM><SPAN style="font-weight: 400;">assuming</SPAN></EM><SPAN style="font-weight: 400;"> that </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;"> produces it at this point.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">MGM plates have been prepared according to pg 14 of Halohandbook.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>July 2015</STRONG></P><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, July 1</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Proposed the idea of using beads that bind to CoA groups to detect the presence of butyryl-CoA.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Trying to find a good promoter to increase the gene expression of the genes we decide to use.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Several searches yielded the promsing </SPAN><EM><SPAN style="font-weight: 400;">ptb</SPAN></EM><SPAN style="font-weight: 400;"> promoter.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Considered using rational design to lower the pI of the fusion protein to near 4.5. Requires generating a model structure using SWISS-MODELLER.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Unable to find a 3D model of </SPAN><EM><SPAN style="font-weight: 400;">adhE2</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Jairo used his program to codon optimize the </SPAN><EM><SPAN style="font-weight: 400;">adhE2</SPAN></EM><SPAN style="font-weight: 400;"> gene for </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, July 2</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Decided on the </SPAN><EM><SPAN style="font-weight: 400;">pTA963</SPAN></EM><SPAN style="font-weight: 400;"> plasmid for eventual expression of our fusion gene.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Professor Bernick urged us to focus on pI rather than percent identity of potential fusion genes.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found yet another alternative fusion gene in </SPAN><EM><SPAN style="font-weight: 400;">Shewanella waksmanii</SPAN></EM><SPAN style="font-weight: 400;"> (</SPAN><A href="http://www.ncbi.nlm.nih.gov/nuccore/655366186"><SPAN style="font-weight: 400;">http://www.ncbi.nlm.nih.gov/nuccore/655366186</SPAN></A><EM><SPAN style="font-weight: 400;">).</SPAN></EM></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found the nucleotide sequence for this fusion gene.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Discontinuing work on the </SPAN><EM><SPAN style="font-weight: 400;">Halomonas</SPAN></EM><SPAN style="font-weight: 400;"> fusion gene </SPAN><EM><SPAN style="font-weight: 400;">adhE2</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI></UL></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, July 6</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Attempting to find a protocol for inserting the </SPAN><EM><SPAN style="font-weight: 400;">S. waksmanii</SPAN></EM><SPAN style="font-weight: 400;"> fusion gene into the expression plasmid.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Dominic suggested using Gibson assembly.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Tried to codon optimize the </SPAN><EM><SPAN style="font-weight: 400;">S. waksmanii</SPAN></EM><SPAN style="font-weight: 400;"> gene using Jairo’s program, but it ran into a syntax error which couldn’t be fixed.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Asked Professor Bernick to contact an expert in </SPAN><EM><SPAN style="font-weight: 400;">S. waksmanii</SPAN></EM><SPAN style="font-weight: 400;"> (Chad Saltikov).</SPAN></LI></UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Reconsidered the </SPAN><EM><SPAN style="font-weight: 400;">ptb</SPAN></EM><SPAN style="font-weight: 400;"> promoter as a backup.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found the </SPAN><EM><SPAN style="font-weight: 400;">bgaH</SPAN></EM><SPAN style="font-weight: 400;">gene, which is used to assay the promoters of haloarchaeal genes.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found a similar fusion gene in </SPAN><EM><SPAN style="font-weight: 400;">Shewanella sediminis</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Has a lower pI.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Unable to find the entire genome of </SPAN><EM><SPAN style="font-weight: 400;">S. waksmanii</SPAN></EM><SPAN style="font-weight: 400;"> for codon optimization.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">The full genome for </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> is available, allowing codon optimization.</SPAN></LI></UL></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, July 7</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Professor Bernick approved the idea of using the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> fusion gene.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">The fusion gene was codon optimized for </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Checked nucleotide sequence for the formation of structures that would affect Gibson assembly.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Split the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> fusion gene into smaller 500 - 700 bp fragments for ordering.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Fragments &gt;1000 bp take longer and are more expensive.</SPAN></LI></UL></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, July 8</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Finished splitting the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> fusion gene into 3 gene fragments.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Began designing primers for the </SPAN><EM><SPAN style="font-weight: 400;">pTA963</SPAN></EM><SPAN style="font-weight: 400;"> plasmid and the His-tags.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, July 9</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Plasmid primers were ordered.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Added overlaps on the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> fusion gene for both N- and C-terminal His tags.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, July 10</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continuing to pursue the </SPAN><EM><SPAN style="font-weight: 400;">bgaH</SPAN></EM><SPAN style="font-weight: 400;"> gene as a backup.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Started working on primers for fusion gene.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Saturday, July 11</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Finished fusion gene primers.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, July 13</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Waited for approval of </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> primers for ordering.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran inverse PCR on the </SPAN><EM><SPAN style="font-weight: 400;">pTA963</SPAN></EM><SPAN style="font-weight: 400;"> plasmid.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, July 14</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Checked optimized gene sequence to ensure that resulting amino acid sequence matches.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Did not match, so the program needs refinement.</SPAN></LI></UL></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, July 15</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Jairo attempted to fix codon optimization program.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, July 16</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Jairo fixed his program, however Professor Bernick urged us to use DNAWorks for codon optimization.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Discontinued work on the </SPAN><EM><SPAN style="font-weight: 400;">ptb</SPAN></EM><SPAN style="font-weight: 400;"> promoter, in the interest of time.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, July 17</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Plasmid N- and C-terminal protocols were accidentally swapped and were redone.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continued to refine the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> gene fragments and primers.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Need to manually reduce the GC content for ordering.</SPAN></LI></UL></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, July 20</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continued to work on the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> fragments and primers.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, July 21</STRONG></P><UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Still working on the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> fragments and primers.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">According to Timpson </SPAN><EM><SPAN style="font-weight: 400;">et al</SPAN></EM><SPAN style="font-weight: 400;">., </SPAN><EM><SPAN style="font-weight: 400;">adh2</SPAN></EM><SPAN style="font-weight: 400;"> is an alcohol dehydrogenase in </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;"> that shows high rates of butanol production.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Butanol production via reduction only occurs in acidic conditions.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">The reverse oxidation reaction occurs in alkaline conditions.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Since </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii </SPAN></EM><SPAN style="font-weight: 400;">is grown in alkaline conditions, oxidation is favored.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Considered growing </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;"> and acidifying the environment.</SPAN></LI></UL></UL></UL><P> </P><UL><LI style="font-weight: 400;"><EM><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;"> also has two aldehyde dehydrogenases (</SPAN><EM><SPAN style="font-weight: 400;">aldY3</SPAN></EM><SPAN style="font-weight: 400;"> and </SPAN><EM><SPAN style="font-weight: 400;">aldY5</SPAN></EM><SPAN style="font-weight: 400;">).</SPAN></EM></LI></UL><P> </P><UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">It’s unclear which results in more butanol production.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Shifted the project to focus on developing two fusion genes consisting of </SPAN><EM><SPAN style="font-weight: 400;">adh2</SPAN></EM><SPAN style="font-weight: 400;"> and each of the aldehyde dehydrogenases.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Started researching linker sequences to connect the two dehydrogenase genes.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, July 22</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Finalized the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> fragments and primers.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Made minimal media for growing </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM><SPAN style="font-weight: 400;"> anaerobically.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Will give several days for colonies to grow, before acidifying the environment.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Reduced the number of rare codons in the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> gene.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Contacted Rob Franks about gas chromatography usage.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">He said it should be able to handle slightly acidic conditions.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Need to prepare standards before running samples.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Looked up sequences of </SPAN><EM><SPAN style="font-weight: 400;">aldY3</SPAN></EM><SPAN style="font-weight: 400;">, </SPAN><EM><SPAN style="font-weight: 400;">aldY5</SPAN></EM><SPAN style="font-weight: 400;">, and </SPAN><EM><SPAN style="font-weight: 400;">adh2</SPAN></EM><SPAN style="font-weight: 400;"> on KEGG.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, July 23</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Prepared the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> fragments and primers for ordering.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Started researching linker sequences to connect the two dehydrogenase genes.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found a paper that describes characteristics of good linkers (</SPAN><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3726540/"><SPAN style="font-weight: 400;">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3726540/</SPAN></A><SPAN style="font-weight: 400;">)</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Professor Bernick urged us to find natural linkers rather than designing our own.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, July 24</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">??????</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, July 27</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Designed primers with Dominic’s help for amplifying </SPAN><EM><SPAN style="font-weight: 400;">aldY3</SPAN></EM><SPAN style="font-weight: 400;"> and </SPAN><EM><SPAN style="font-weight: 400;">adh2</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Tried using Pfam to view the domains of fusion proteins to obtain natural linker sequences.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, July 28</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used InterPro to visualize the linker region in natural fusion genes.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Met with Professor Bernick and established a long-term game plan:</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Design primers for dehydrogenase genes such that they incorporate the linker region.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Use Gibson assembly to incorporate the dehydrogenase genes and linker into the </SPAN><EM><SPAN style="font-weight: 400;">pTA963</SPAN></EM><SPAN style="font-weight: 400;"> plasmid.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Transform the Gibson product into </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM><SPAN style="font-weight: 400;">, and determine which colonies were successful in taking up the fusion gene by colony PCR.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Design sequencing primers and send product off for Sanger sequencing.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Use nested PCR on the colony PCR product.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Dominic confirmed that he also has </SPAN><EM><SPAN style="font-weight: 400;">adh2</SPAN></EM><SPAN style="font-weight: 400;"> primers, so we only designed the </SPAN><EM><SPAN style="font-weight: 400;">aldY3</SPAN></EM><SPAN style="font-weight: 400;"> primers.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continued searching for linker sequences.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, July 29</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Several linker regions were designed from scratch. Started to incorporate overlap with dehydrogenase genes.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Other linkers were found to be conserved across several species.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Finalized </SPAN><EM><SPAN style="font-weight: 400;">aldY3</SPAN></EM><SPAN style="font-weight: 400;"> primers for ordering. </SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Settled on 4 linkers: 2 designed and 2 conserved.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, July 30</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Received the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> primers and diluted them to 100 uM.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>August 2015</STRONG></P><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, August 3</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Waiting for </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> gene blocks.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">The anaerobic cultures haven’t grown much - pH still at 7.5.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, August 4</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Received native gene primers. These primers were also diluted to 100 uM.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Genomic DNA isolated from </SPAN><EM><SPAN style="font-weight: 400;">H. volcanii</SPAN></EM></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Followed protocol in pg 70 of Halohandbook.</SPAN></LI></UL></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, August 5</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran PCR on genomic DNA to isolate </SPAN><EM><SPAN style="font-weight: 400;">aldY5</SPAN></EM><SPAN style="font-weight: 400;"> (annealing temp 71℃).</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Extension time should’ve been 1 min, but was only 30 sec.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran a gel on the PCR product - mostly primers showed up with little to no PCR product.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, August 6</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Redid PCR of </SPAN><EM><SPAN style="font-weight: 400;">aldY5</SPAN></EM><SPAN style="font-weight: 400;"> and </SPAN><EM><SPAN style="font-weight: 400;">aldY3</SPAN></EM><SPAN style="font-weight: 400;"> using these conditions:</SPAN></LI></UL><P> </P><UL><LI style="font-weight: 400;"><EM><EM><SPAN style="font-weight: 400;">aldY3</SPAN></EM><SPAN style="font-weight: 400;"> and </SPAN><EM><SPAN style="font-weight: 400;">aldY5</SPAN></EM><SPAN style="font-weight: 400;"> were successful, </SPAN><EM><SPAN style="font-weight: 400;">adh2</SPAN></EM><SPAN style="font-weight: 400;"> was not.</SPAN></EM></LI></UL><P> </P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Received the </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> gene blocks.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, August 7</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran PCR again on </SPAN><EM><SPAN style="font-weight: 400;">aldY5</SPAN></EM><SPAN style="font-weight: 400;"> and </SPAN><EM><SPAN style="font-weight: 400;">aldY3</SPAN></EM><SPAN style="font-weight: 400;"> using these conditions:</SPAN></LI></UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used a touchdown PCR on </SPAN><EM><SPAN style="font-weight: 400;">adh2</SPAN></EM><SPAN style="font-weight: 400;"> for more specific primer-binding (decrease annealing temperature by 0.5 ℃ each cycle).</SPAN></LI></UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">The conditions were successful</SPAN></LI></UL><P><STRONG>Saturday, August 8</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Re-ran PCR on </SPAN><EM><SPAN style="font-weight: 400;">adh2</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran PCR on the 8 gene/linker fragments (71℃ annealing, 1 min extension, 2 min final hold)</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy3BFw &amp; aldy3Blink1Rev = A</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy5BFw &amp; aldy5Blink1Rev = B</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy3BFw &amp; aldy3Blink2Rev = C</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy5BFw &amp; aldy5Blink2Rev = D</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy3BFw &amp; aldy3Blink3Rev = E</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy5BFw &amp; aldy5Blink3Rev = F</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy3BFw &amp; aldy3Blink4Rev = G</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy5BFw &amp; aldy5Blink4Rev = H</SPAN></LI></UL></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, August 10</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Unable to proceed due to a lack of supplies.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, August 11</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Still unable to proceed further due to a lack of supplies.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Anaerobic cultures appear to be growing.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Started to prepare minimal HEPES media rather than Tris (pH 6.55).</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, August 12</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continuing to make minimal media.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Considering eliminating potassium phosphate to avoid precipitate formation.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Professor Bernick suggested micro-aerobic cultures, in which a small amount of oxygen is injected into the culture vessel.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">May help solve the redox imbalance from the lack of oxygen as a terminal electron acceptor.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Assembled </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> gene fragments.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran a gel with </SPAN><EM><SPAN style="font-weight: 400;">aldY5</SPAN></EM><SPAN style="font-weight: 400;">/</SPAN><EM><SPAN style="font-weight: 400;">adh2</SPAN></EM><SPAN style="font-weight: 400;"> and </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> fragments.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, August 13</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">The </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> fragments were re-assembled (increased annealing temperature to 67℃).</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Gene/linker sequences also ran again.</SPAN></LI></UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Key: </SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Lane1: aldy5BFw &amp; aldy5Blink4Rev = H</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Lane2: adh2Dlink1Fw &amp; adh2DRev = I</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Lane3: adh2Dlink2Fw &amp; adh2DRev = J</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Lane4: adh2Dlink3Fw &amp; adh2DRev = K</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Lane5: adh2Dlink4Fw &amp; adh2DRev = L </SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">PCR conditions:</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, August 14</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Assembled the </SPAN><EM><SPAN style="font-weight: 400;">aldY</SPAN></EM><SPAN style="font-weight: 400;"> and </SPAN><EM><SPAN style="font-weight: 400;">adh2</SPAN></EM><SPAN style="font-weight: 400;"> linker fragments.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy3BFw &amp; aldy3Blink1Rev &amp; adh2Dlink1Fw &amp; adh2DRev = F1</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy5BFw &amp; aldy5Blink1Rev &amp; adh2Dlink1Fw &amp; adh2DRev = F2</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy3BFw &amp; aldy3Blink2Rev &amp; adh2Dlink2Fw &amp; adh2DRev = F3</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy5BFw &amp; aldy5Blink2Rev &amp; adh2Dlink2Fw &amp; adh2DRev = F4</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy3BFw &amp; aldy3Blink3Rev &amp; adh2Dlink3Fw &amp; adh2DRev = F5</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy5BFw &amp; aldy5Blink3Rev &amp; adh2Dlink3Fw &amp; adh2DRev = F6</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy3BFw &amp; aldy3Blink4Rev &amp; adh2Dlink4Fw &amp; adh2DRev = F7</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">aldy5BFw &amp; aldy5Blink4Rev &amp; adh2Dlink4Fw &amp; adh2DRev = F8</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">PCR conditions for fragment assembly:</SPAN></LI></UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Samples were purified using a DNA cleanup kit, but purification was unsuccessful (see last lane).</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Sunday, August 16</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Attempted to determine why the cleanup kit didn’t work.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Tested the reagents in the kit.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Still unsuccessful.</SPAN></LI></UL></UL><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Decided to not assemble fragments and then use Gibson assembly, but to go straight into Gibson assembly and see if the fragments assemble properly before annealing to the plasmid.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, August 17</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Continued to work on cleanup kit.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Sample G1 was successfully cleaned up.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Samples B, E, F, L need to be re-cleaned, from the kit mishap.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Despite cleanup, these samples had a low 260/230 value.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Unsure whether to use N-terminal or C-terminal plasmid for our Gibson reaction.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, August 18</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Confirmed that the N-terminal plasmid is required for Gibson of native fusion genes.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Considered the idea of testing cultures for fermentation products such as lactic acid.</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Rob Franks suggested using HPLC, since lactic acid won’t run in the column.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">HPLC also somewhat non-specific.</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran Gibson assembly on A + I (F1) and B + I (F2)</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, August 19</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Gibson assembly failed due to a bad plasmid. Need to redo plasmid before attempting Gibson assembly again.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Conditions used in generating the C-terminal and N-terminal plasmid:</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Started to figure out the details surrounding micro-aerobic cultures and how much oxygen needs to be added.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, August 20</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Recreated samples B, E, L, F.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Recreated both N- and C-terminal plasmid.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Results of plasmid (left) and samples B, E, L, F (right):</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran Gibson assembly on both </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> samples and A + I again.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Inoculated 3 micro-anaerobic cultures containing the following types of media:</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">HEPES buffer</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">DMSO</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Unbuffered</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Transformed the A + I Gibson product into </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM><SPAN style="font-weight: 400;"> using heat shock and plated on ampicillin plates.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Friday, August 21</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Plates did not show growth, so transformation was unsuccessful.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Planned to re-plate and/or possibly rerun Gibson assembly using a different ratio of plasmid to fragment.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, August 24</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Created both buffered and unbuffered spheroplast solutions.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Considered switching to transformation by electroporation rather than heat shock.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, August 25</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Transformed </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM><SPAN style="font-weight: 400;"> with F3/G3 and C-terminal </SPAN><EM><SPAN style="font-weight: 400;">S. sediminis</SPAN></EM><SPAN style="font-weight: 400;"> Gibson products using electroporation.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Wednesday, August 26</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran Gibson assembly on all remaining gene/linker fragments.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Thursday, August 27</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran colony PCR on G1 sample.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Prepared samples and standards for gas chromatography of micro-aerobic cultures (using ethyl acetate as solvent).</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">1% 1 mL butanol standard</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">1% 1 mL lactic acid standard</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">1 mL ethyl acetate blank</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">10% 1 mL samples of each culture (Unbuffered, HEPES, DMSO, and aerobic)</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Generate more I fragment to rerun Gibson assembly on G2 (B + I).</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran colony PCR on G3 (C + J).</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Monday, August 31</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Transformed and inoculated G4 and G5 into </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM><SPAN style="font-weight: 400;">.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Prepared 18% MGM to be used in establishing a new aerobic culture.</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>September 2015</STRONG></P><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Tuesday, September 1</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran colony PCR on G1, G4, and G5.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran Gibson assembly on G2.</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Inoculated new aerobic cultures. Once grown, these cultures will be switched to anaerobic/micro-aerobic environments.</SPAN></LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingThree"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
          Fermentation: Ethanol 
        </A></H4></DIV><DIV id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree"><DIV class="panel-body"><P><STRONG>Week 3 (7/6/15 - 7/10/15)</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Discussed the formation of the team</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Used KEGG to view the metabolic pathway of </SPAN><EM><SPAN style="font-weight: 400;">Haloferax volcanii </SPAN></EM><SPAN style="font-weight: 400;">and identify genes involved with pyruvate breakdown in order to isolate potential genes to knockout</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Lactate dehydrogenase &amp; NADP-dependent malic enzyme</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Found Zymomonas mobilis genome (</SPAN><SPAN style="font-weight: 400;">2,056,363 bp</SPAN><SPAN style="font-weight: 400;">) and Pyruvate decarboxylase gene, </SPAN><EM><SPAN style="font-weight: 400;">pdc</SPAN></EM><SPAN style="font-weight: 400;">, from Zymomonas mobilis subsp. ZM4 on NCBI</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Week 4 (7/13/15 - 7/17/15)</STRONG></P><P> </P><UL><LI><STRONG><SPAN style="font-weight: 400;">We contacted Julie Maupin-Furlow from the University of Florida because of previous research she had done with H.vo and the Z.mo </SPAN><EM><SPAN style="font-weight: 400;">pdc</SPAN></EM><SPAN style="font-weight: 400;"> gene</SPAN></STRONG></LI></UL><P> </P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">We optimized the Z.mo </SPAN><EM><SPAN style="font-weight: 400;">pdc</SPAN></EM><SPAN style="font-weight: 400;"> for insertion into H.vo using DNAWorks</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Designed gene blocks, an overlap region, and primers (flagged with our pTA963 plasmid) for gibson assembly of our H.vo </SPAN><EM><SPAN style="font-weight: 400;">pdc</SPAN></EM></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Week 5 (7/20/15 - 7/24/15)</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Began designing our knockouts</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">We used the UCSC archaea browser to retrieve genes before and after knockout gene and intergenic sequences</SPAN></LI></UL></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Week 6 (7/27/15 - 7/31/15)</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Finalized our knockout sequences</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran knockout genes through blastx and there are no protein matches in any organism</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Designed primers for our knockout genes</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ordered our H.vo </SPAN><EM><SPAN style="font-weight: 400;">pdc</SPAN></EM><SPAN style="font-weight: 400;"> gene blocks and primers along with our knockout gene blocks and primers</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Week 7 (8/3/15 - 8/7/15)</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Our gene blocks and primers arrived</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">We looked into making spheroplasts for transformation into H.vo</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Week 8 (8/10/15 - 8/14/15)</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran PCR for fragment assembly of our H.vo </SPAN><EM><SPAN style="font-weight: 400;">pdc</SPAN></EM><SPAN style="font-weight: 400;"> gene</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">We ran PCR on or knockouts genes to amplify them</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Purified H.vo </SPAN><EM><SPAN style="font-weight: 400;">pdc</SPAN></EM><SPAN style="font-weight: 400;"> to be used for gibson assembly</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Week 9 (8/17/15 - 8/21/15)</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Did gibson assembly of our H.vo </SPAN><EM><SPAN style="font-weight: 400;">pdc</SPAN></EM><SPAN style="font-weight: 400;"> into the N-term His-tagged plasmid pTA963</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Transformed our product into chemically competent </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM><SPAN style="font-weight: 400;"> by heat shocking them, then plated these onto ampicillin plates and let them grow overnight</SPAN></LI><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Upon plating there was no growth and we determined that the problem was with the </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM><SPAN style="font-weight: 400;"> cells so we switched to top ten electro-competent </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Week 10 (8/24/15 - 8/28/15)</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Redid transformations</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Electroporation was used to transform our Gibson assembly product into the electro-competent </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM><SPAN style="font-weight: 400;"> cells</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Had significant growth on our ampicillin plate showing that our plasmid had been successfully transformed into the </SPAN><EM><SPAN style="font-weight: 400;">E. coli</SPAN></EM><SPAN style="font-weight: 400;"> cells</SPAN></LI></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Week 11 (8/31/15 - 9/4/15)</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Ran colony PCR with the primers we used for our H.vo </SPAN><EM><SPAN style="font-weight: 400;">pdc</SPAN></EM><SPAN style="font-weight: 400;"> fragment assembly</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">We chose 9 colonies from separate areas on the plate for a better chance of success</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Results were inconclusive but we were able to select 5 candidates that were more likely to give us positive results</SPAN></LI></UL></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Reran colony PCR using plasmid specific primers</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Again results were inconclusive</SPAN></LI></UL></UL><P><STRONG><STRONG> </STRONG></STRONG></P><P><STRONG>Week 12 (8/7/15 - 8/11/15)</STRONG></P><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Reran colony PCR using the same plasmid specific primers but doing a touchdown PCR</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">No results to examine</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Reran colony PCR using a different pair of plasmid specific primer doing another touchdown PCR</SPAN></LI><UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">No results to examine</SPAN></LI></UL><LI style="font-weight: 400;"><SPAN style="font-weight: 400;">Troubleshooting these errors</SPAN></LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingFour"><H4 class="panel-title"><A class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
          Field Team
        </A></H4></DIV><DIV id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour"><DIV class="panel-body"><P>Tuesday June 30</P><UL><LI>The field team went over the protocol and made sure all the equipment we needed was put together </LI></UL><P>Wednesday July 1</P><UL><LI>We collected 10 samples from varying locations in the Fremont salt flats</LI><LI>pH and salinity will have to be measured in lab due to the battery being knocked out of the portable meter</LI><LI>Initial plating of field samples was pushed to tomorrow due to time constraints</LI></UL><P>Thursday July 2</P><UL><LI>All of our samples were plated on 18% MGM agar plates (protocol found in the HaloHandbook)</LI></UL><P>Tuesday July 7</P><UL><LI>Substantial growth can be seen on our 18%MGM agar plates</LI><LI>Single cell isolations were started on colonies that showed the most robust growth from each field sample.</LI><LI>We decided on using a minimal media with cellulose as the only carbon source to screen for our cellulase producing halophile.</LI><LI>We decided on measuring turbidity on the UV spectrometer to determine sufficient growth on MCC.</LI></UL><P>Wednesday July 8</P><UL><LI>Some single cell isolations will need to be redone due to poor spreading technique</LI><LI>We characterized each of the single cell isolation colonies. </LI><LI>To prepare for 80% glycerol stocks we need to grow the single cell isolations ing 18%MGM to raise the cell count.</LI></UL><P>Thursday July 9</P><UL><LI>More single cell isolations are ready to incubate in MGM media for our cellulose screen and glycerol stocks. </LI></UL><UL><LI>The first minimal media we made was way to cloudy, we’re going to reduce it from 5g of MCC to 3g. </LI></UL><P>Friday July 10</P><UL><LI>Discussed what happens after we identify the haloarchaea after the cellulose screen. We could sequence the 16S region of the ribosomal DNA to identify the species and if there is a new species we could get the whole genome sequenced. </LI></UL><UL><LI>We would need to order a primer to do the proper PCR. </LI><LI> We also found a paper about solubilizing cellulose with 14%wt NaOH. It may break the chains of glucose but not substantially. (Kuo, Yen-Ning and Hong, Juan. &quot;Investigation of Solubility of Microcrystalline Cellulose in Aqueous NaOH.&quot; (2004).)</LI></UL><P>Sunday July 12 </P><UL><LI>We are going to make a soluble MCC minimal media by treating the MCC (2g) with NaOH (14.9%wt)</LI></UL><P>Monday July 13</P><UL><LI>We inoculated 5 more samples in MGM 18% from our previous single cell isolation. </LI></UL><UL><LI>We are re-isolated some of our previous single cell isolation due to smearing. </LI></UL><UL><LI>The protocol for the Micro-crystalline cellulose minimal media is finally complete for for solubilized and unsolubilized MCC. </LI></UL><UL><LI>We had to freeze the NaOH treated MCC in order to solubilize it. </LI></UL><P>Tuesday July 14</P><UL><LI>We made our solubilized MCC minimal media today. </LI><LI>We need to titrate the 5M TrisHCl slowly and we needed around 50mL to reach a pH of 7.5 for a 1L of solubilized MCC. </LI></UL><P>Thursday July 16</P><UL><LI>We made the insoluble MCC minal media and brought the pH down to 7.5. </LI><LI>We discussed what will happen after the 16S region PCR. We would use the NCBI BLASTN program to see.</LI><LI>The pH was raised overnight to a pH of 8.0 but it is acceptable because <EM>H. volcanii</EM> prefer a slightly alkaline environment</LI></UL><P>Tuesday July 21</P><UL><LI>We need to do more glycerol stocks for new single cell isolates. </LI></UL><P>Friday July 24</P><UL><LI>Any organism that can breakdown insoluble cellulose can theoretically breakdown insoluble MCC, so the insoluble MCC would tell us the most, but we are going to run both soluble and insoluble MCC screen just in case. </LI></UL><UL><LI>We are also considering using a glucose test to measure the concentration of glucose in the media as time goes on. We’re thinking of using a diabetes glucometer. </LI><LI>We are starting to prepare for designing the primers needed for the 16S ribosomal DNA PCR. </LI></UL><P>Monday July 27</P><UL><LI>We have 10 soluble and 10 insoluble MCC screens in both incubators. </LI></UL><UL><LI>The single cell isolate P5A is showing slight signs of pink in the insoluble test. Pink indicates a high cell count (if the cells are pink in color) or a high concentration of cellular metabolites, which both indicate cells can thrive of purified cellulose. </LI></UL><P>Tuesday July 28</P><UL><LI>There is growth in MCC minimal media cultures! We just need to isolate them and we’re thinking of making agar plates with the solubilized MCC minimal media. </LI></UL><UL><LI>We’re considering using the microscope to prove growth as opposed to optical density and turbidity. </LI></UL><P>Wednesday July 29</P><UL><LI>We looked at the cells that have been growing in our media under the microscope and we saw a heterogenous mixture of cell types including bacilli, cocci, and even a fast swimming species. Somewhere in the mix is our species producing cellulase and the other species may be feeding off the glucose being released from the MCC. </LI></UL><P>Friday July 31</P><UL><LI>We decided on which samples would move onto the soluble MCC agar plates to isolate our cellulase producing species.</LI><LI>We made MCC minimal media agar plates</LI></UL><P>Monday August 3</P><UL><LI>We’ve started plating on the soluble MCC agar plates.</LI><LI>More samples are showing growth in the cellulose screen</LI></UL><P>Friday August 7 </P><UL><LI>We decided to go with universal archaeal primers for the 16S region of ribosomal DNA. The primers won’t work for bacteria which is ideal because we’re trying to find an archaea.</LI><LI>We decided to do a touchdown PCR reaction where the annealing temperature is lowered after every cycle to improve specificity of the amplification. </LI><LI>The annealing temperature will be set at 68℃ and will drop to 65℃ over 15 cycles</LI></UL><P>Friday August 21</P><UL><LI>The single cell isolation on the MCC plates are grown enough for identification. We need to grow them in MGM again to increase cell count for DNA extraction. </LI><LI>After the PCR we are going to send our PCR product to UC Berkeley to be sequenced. We’ll need controls for the PCR so we are also going to use E. coli and H. volcanii to prove our primers work only for archaea. </LI></UL><P>Tuesday August 25</P><UL><LI>We ran our sequences on the NCBI BLASTN program and we were getting hits all under the Haloferax genus </LI></UL><UL><LI>We going to show our findings at the Sierra System Synthetic Biology Symposium this weekend</LI></UL><P>Tuesday June 30</P><UL><LI>The field team went over the protocol and made sure all the equipment we needed was put together </LI></UL><P>Wednesday July 1</P><UL><LI>We collected 10 samples from varying locations in the Fremont salt flats</LI><LI>pH and salinity will have to be measured in lab due to the battery being knocked out of the portable meter</LI><LI>Initial plating of field samples was pushed to tomorrow due to time constraints</LI></UL><P>Thursday July 2</P><UL><LI>All of our samples were plated on 18% MGM agar plates (protocol found in the HaloHandbook)</LI></UL><P>Tuesday July 7</P><UL><LI>Substantial growth can be seen on our 18%MGM agar plates</LI><LI>Single cell isolations were started on colonies that showed the most robust growth from each field sample.</LI><LI>We decided on using a minimal media with cellulose as the only carbon source to screen for our cellulase producing halophile.</LI><LI>We decided on measuring turbidity on the UV spectrometer to determine sufficient growth on MCC.</LI></UL><P>Wednesday July 8</P><UL><LI>Some single cell isolations will need to be redone due to poor spreading technique</LI><LI>We characterized each of the single cell isolation colonies. </LI><LI>To prepare for 80% glycerol stocks we need to grow the single cell isolations ing 18%MGM to raise the cell count.</LI></UL><P>Thursday July 9</P><UL><LI>More single cell isolations are ready to incubate in MGM media for our cellulose screen and glycerol stocks. </LI></UL><UL><LI>The first minimal media we made was way to cloudy, we’re going to reduce it from 5g of MCC to 3g. </LI></UL><P>Friday July 10</P><UL><LI>Discussed what happens after we identify the haloarchaea after the cellulose screen. We could sequence the 16S region of the ribosomal DNA to identify the species and if there is a new species we could get the whole genome sequenced. </LI></UL><UL><LI>We would need to order a primer to do the proper PCR. </LI><LI> We also found a paper about solubilizing cellulose with 14%wt NaOH. It may break the chains of glucose but not substantially. (Kuo, Yen-Ning and Hong, Juan. &quot;Investigation of Solubility of Microcrystalline Cellulose in Aqueous NaOH.&quot; (2004).)</LI></UL><P>Sunday July 12 </P><UL><LI>We are going to make a soluble MCC minimal media by treating the MCC (2g) with NaOH (14.9%wt)</LI></UL><P>Monday July 13</P><UL><LI>We inoculated 5 more samples in MGM 18% from our previous single cell isolation. </LI></UL><UL><LI>We are re-isolated some of our previous single cell isolation due to smearing. </LI></UL><UL><LI>The protocol for the Micro-crystalline cellulose minimal media is finally complete for for solubilized and unsolubilized MCC. </LI></UL><UL><LI>We had to freeze the NaOH treated MCC in order to solubilize it. </LI></UL><P>Tuesday July 14</P><UL><LI>We made our solubilized MCC minimal media today. </LI><LI>We need to titrate the 5M TrisHCl slowly and we needed around 50mL to reach a pH of 7.5 for a 1L of solubilized MCC. </LI></UL><P>Thursday July 16</P><UL><LI>We made the insoluble MCC minal media and brought the pH down to 7.5. </LI><LI>We discussed what will happen after the 16S region PCR. We would use the NCBI BLASTN program to see.</LI><LI>The pH was raised overnight to a pH of 8.0 but it is acceptable because <EM>H. volcanii</EM> prefer a slightly alkaline environment</LI></UL><P>Tuesday July 21</P><UL><LI>We need to do more glycerol stocks for new single cell isolates. </LI></UL><P>Friday July 24</P><UL><LI>Any organism that can breakdown insoluble cellulose can theoretically breakdown insoluble MCC, so the insoluble MCC would tell us the most, but we are going to run both soluble and insoluble MCC screen just in case. </LI></UL><UL><LI>We are also considering using a glucose test to measure the concentration of glucose in the media as time goes on. We’re thinking of using a diabetes glucometer. </LI><LI>We are starting to prepare for designing the primers needed for the 16S ribosomal DNA PCR. </LI></UL><P>Monday July 27</P><UL><LI>We have 10 soluble and 10 insoluble MCC screens in both incubators. </LI></UL><UL><LI>The single cell isolate P5A is showing slight signs of pink in the insoluble test. Pink indicates a high cell count (if the cells are pink in color) or a high concentration of cellular metabolites, which both indicate cells can thrive of purified cellulose. </LI></UL><P>Tuesday July 28</P><UL><LI>There is growth in MCC minimal media cultures! We just need to isolate them and we’re thinking of making agar plates with the solubilized MCC minimal media. </LI></UL><UL><LI>We’re considering using the microscope to prove growth as opposed to optical density and turbidity. </LI></UL><P>Wednesday July 29</P><UL><LI>We looked at the cells that have been growing in our media under the microscope and we saw a heterogenous mixture of cell types including bacilli, cocci, and even a fast swimming species. Somewhere in the mix is our species producing cellulase and the other species may be feeding off the glucose being released from the MCC. </LI></UL><P>Friday July 31</P><UL><LI>We decided on which samples would move onto the soluble MCC agar plates to isolate our cellulase producing species.</LI><LI>We made MCC minimal media agar plates</LI></UL><P>Monday August 3</P><UL><LI>We’ve started plating on the soluble MCC agar plates.</LI><LI>More samples are showing growth in the cellulose screen</LI></UL><P>Friday August 7 </P><UL><LI>We decided to go with universal archaeal primers for the 16S region of ribosomal DNA. The primers won’t work for bacteria which is ideal because we’re trying to find an archaea.</LI><LI>We decided to do a touchdown PCR reaction where the annealing temperature is lowered after every cycle to improve specificity of the amplification. </LI><LI>The annealing temperature will be set at 68℃ and will drop to 65℃ over 15 cycles</LI></UL><P>Friday August 21</P><UL><LI>The single cell isolation on the MCC plates are grown enough for identification. We need to grow them in MGM again to increase cell count for DNA extraction. </LI><LI>After the PCR we are going to send our PCR product to UC Berkeley to be sequenced. We’ll need controls for the PCR so we are also going to use E. coli and H. volcanii to prove our primers work only for archaea. </LI></UL><P>Tuesday August 25</P><UL><LI>We ran our sequences on the NCBI BLASTN program and we were getting hits all under the Haloferax genus </LI></UL><UL><LI>We going to show our findings at the Sierra System Synthetic Biology Symposium this weekend</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV id="foot" class="center foot">  <A href="https://2015.igem.org/Team:UCSC">Home</A> | <A href="https://2015.igem.org/Team:UCSC/Team">Teams</A> | <A href="https://2015.igem.org/Team:UCSC/Project">Project</A> | <A href="https://2015.igem.org/Team:UCSC/Practices">Human Practices</A> | <A href="https://2015.igem.org/Team:UCSC/Notebook">Notebook</A> | <A href="https://2015.igem.org/Team:UCSC/Safety">Safety</A> | <A href="https://2015.igem.org/Team:UCSC/Parts">Achievements</A> | <A href="https://2015.igem.org/Team:UCSC/Software">F.O.C.U.S</A>  </DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>