Team:Rotterdam HR/Notebook
Notebook
Notebook
As a team with only one computer scientist (and one electrical engineer who can program),    we wanted a notebook that automatically updates whenever someone adds something to our lab journal.    Luckily, we gained a sponsor that gave use an electronic lab journal as sponsorship.    With this, we can use their API to download specific sections from our lab journal and display them on or wiki.    But, as a multidisciplinary team we have multiple kinds of notebook entries: Software, Hardware and Wetlab.    Therefore we also used the Google Drive RESTful API for downloading software and hardware entries from google sheets files.
More info about the notebook generator
show wetlab entries
show hardware entries
show software entries
Team page
May 10
×
Rens Boeser
Created the first prototype for the team page
01
Team page
Show more
May 10
Footer
May 20
×
Rens Boeser
Social media created
01
Footer
Show more
May 20
Footer
May 21
×
Rens Boeser
Social media buttons same color and fixed iGEM style issues
02
Footer
Show more
May 21
Notebook
May 24
×
Rens Boeser
Multiple week summary styles created for the notes
01
Notebook
Show more
May 24
Header
May 27
×
Rens Boeser
Added the header with mobile compatibility als resized logos and readded them to the header
01
Header
Show more
May 27
Main template
May 28
×
Rens Boeser
Added template header and footer with 'under construction' content
01
Main template
Show more
May 28
Header
May 28
×
Rens Boeser
Small fixes and menu adjustments
02
Header
Show more
May 28
Team
May 29
×
Rens Boeser
Added a PI list with the secondary PI
02
Team
Show more
May 29
Main page
June 7
×
Rens Boeser
Designed and added content holders to the index Page
01
Main page
Show more
June 7
Team
June 9
×
Rens Boeser
Added myself to the teammember Page and did some bugfixing
03
Team
Show more
June 9
Header
June 9
×
Rens Boeser
Started working on a better looking menubar
03
Header
Show more
June 9
Header
June 10
×
Rens Boeser
Added the ability to add sub-menus to items in the menu bar. Integration into the website and added indication for which Page you are on
04
Header
Show more
June 10
Construction pages
June 13
×
Rens Boeser
Updated all Pages to the new style with an under construction sign
01
Construction pages
Show more
June 13
Header
June 14
×
Rens Boeser
Added a way to recognise Pages under construction using the menubar
05
Header
Show more
June 14
Main page
June 14
×
Rens Boeser
Updated the content containers so that they are responsive
02
Main page
Show more
June 14
Architecture design [TC]
June 18
×
Tom van Dijk | Mike Vrieswijk
The first draft of the architecture design for the temperature controller was made.
01
Architecture design [TC]
Show more
June 18
Human Practices page
June 22
×
Rens Boeser
Content added to the human practices page
01
Human Practices page
Show more
June 22
Main page
June 23
×
Rens Boeser
Content created and added to the main page
03
Main page
Show more
June 23
Architecture design [TC]
June 24
×
Tom van Dijk | Mike Vrieswijk
Today a few changes have been made to the architecture design of the temperature controller. The choice was made to use a premade power supply as opposed to a self-made one, because this was considering the time left to finish this device the best choice. A few alterations were made to make the architecture design better readable.
02
Architecture design [TC]
Show more
June 24
First prototype [TC]
June 25
×
Tom van Dijk | Mike Vrieswijk
A 3d drawing was made for the casing that will hold the temperature controller. The first prototype was printed, but was determent to be inadequate due to the lack of enough space for the cooler block.
01
First prototype [TC]
Show more
June 25
Team
June 26
×
Rens Boeser
Added the remaining instructors to the team
04
Team
Show more
June 26
Second prototype [TC]
June 26
×
Tom van Dijk | Mike Vrieswijk
In the drawing space was added to fit the cooler block for the temperature controller. Slits were added to the drawing for the nuts to slide into. The altered drawing was printed. The newly printed parts, cooler block, fan and Peltier element were assembled. Everything fit together.
02
Second prototype [TC]
Show more
June 26
Assessing CooA Production
June 26
×
Dustin van der Meulen
To start testing the production of CooA, we firstly resuspended and transformed the BioBricks from the kit witch we wanted to use (Table 1). This has been done using the iGEM Kit plate instructions (Protocol 1, step 1 - 3) and the NEB 10 béta transformation protocol adapted New England Biolabs (Protocol 2), using bought cells with a reported transformation efficienty of 1-3 10^9 cfu/μg pUC19. Inucbation was at 37°C.
Table 1 : Resuspended BioBricks from the iGEM 2018 Distribution kit and their uses.
BBa_K592009
Blue Chromoprotein AmilCP
BBa_B0031
Ribosomal binding site (strong), derived from BBa_B0030
BBa_B0032
Ribosomal binding site (medium), derived from BBa_B0030
BBa_B0030
Ribosomal binding site (weak)
BBa_B0015
Double terminator (BBa_B0010 & BBa_B0012)
BBa_J23100
Strong Constitutive Anderson Promotor
BBa_J23105
Medium Constitutive Anderson Promotor
BBa_J23113
Weak Constitutive Anderson Promotor
BBa_J45199
Banana odor enzyme (ATF1) generator
BBa_K1184000
Killer Red
BBa_K352001
CooA
BBa_K352011
CooA responsive system
After checking the plates on the 28th of June, we recieved the following results (Table 2):
Table 2 : Kolonies found after overnight growth.
BBa_K592009
20 colonies
BBa_B0031
4 colonies
BBa_B0032
5 colonies
BBa_B0030
1 colonies
BBa_B0015
6 colonies
BBa_J23100
1 colonies
BBa_J23105
98 colonies
BBa_J23113
19 colonies
BBa_J45199
0 colonies
BBa_K1184000
4 colonies
BBa_K352001
2 colonies
BBa_K352011
4 colonies
01
Assessing CooA Production
Show more
June 26
Testrun [TC]
June 27
×
Tom van Dijk | Mike Vrieswijk
A short test run was done with the newly assembled cooler. The test concluded that the temperature controller could cool a water droplet to the point of freezing and heat it back up to its boiling point.
03
Testrun [TC]
Show more
June 27
Assessing CooA Production
June 29
×
Dustin van der Meulen
After making a 20 mL overnight culture of our NEB 10 béta cells containing our BioBricks, we performed a mini prep (Protocol 3). Using our nanodrop spectrophotometer  which gave us the following DNA concentrations:
 
BBa_K592009
72,5 ng/μL
BBa_B0031
118,9 ng/μL
BBa_B0032
52,5 ng/μL
BBa_B0030
488,6 ng/μL
BBa_B0015
53,5 ng/μL
BBa_J23100
148,3 ng/μL
BBa_J23105
96,3 ng/μL
BBa_J23113
151,8 ng/μL
BBa_K118400
140,4 ng/μL
BBa_K352001
77,7 ng/μL
BBa_K352011
76,8 ng/μL
After assessign our results, in the future we will perform the ethanol carry-over steps.
Also from the previously mentioned overnight culture 1 mL has been used to make Glycerol Stock (Protocol 4)
02
Assessing CooA Production
Show more
June 29
Draft decision
July 2
×
Tom van Dijk | Mike Vrieswijk
The choice to use a computer PSU as a power supply was made. The choice was made on the grounds, that computer PSU have a standard 12, 5 and 3.3V output. This is beneficial because these are the voltages necessary for the selected components.
01
Draft decision
Show more
July 2
Assessing CooA Production
July 2
×
Dustin van der Meulen
Today we performed a mini prep, digestion, dephosphorilation and ligation (following protocols 4 and 5) of a few promotor and RBS BioBricks. Because of administrative complications only the variants with J23113 and a RBS have been digested correctly.
03
Assessing CooA Production
Show more
July 2
Assessing CooA Production
July 2
×
Dustin van der Meulen
Today we transformed the previously made three combinations (BBa_J23113 + BBa_B0031 / B0032 / B0030) into chemically competent NEb 10 béta E.coli cells using the corresponding NEB protocol.
04
Assessing CooA Production
Show more
July 2
Architecture design [TC]
July 5
×
Tom van Dijk | Mike Vrieswijk
Changes were made to the architecture design of the temperature controller. Different voltage regulators were selected to fit the newly chosen power supply. To better readability of the architecture design a legend was added.
03
Architecture design [TC]
Show more
July 5
Assessing CooA Production
July 5
×
Dustin van der Meulen
After performing a mini prep and NotI digest on the 5th of July, and a gel electrophoresis today, we concluded that the cloning wasn't succesful.
 
05
Assessing CooA Production
Show more
July 5
Electrical circuit design [TC]
July 8
×
Tom van Dijk | Mike Vrieswijk
Custom footprints were made for multiple components and added to the parts library in the PCB design software.
01
Electrical circuit design [TC]
Show more
July 8
Electrical circuit design [TC]
July 9
×
Tom van Dijk | Mike Vrieswijk
The provisional electronic circuit design for the temperature controller was made.
02
Electrical circuit design [TC]
Show more
July 9
Assessing CooA Production
July 10
×
Dustin van der Meulen
Today we again performed a mini prep (overnight cultures from 10th of July) and digest. Sadly most digests did not give the expected results. Even though we had already progressed with the ligation. For now we've decided to progress with these ligation products.
 
06
Assessing CooA Production
Show more
July 10
Electrical circuit design [TC]
July 11
×
Tom van Dijk | Mike Vrieswijk
The electronic circuit design for the temperature controller was finished. Measuring point were added to the electronic circuit design.
03
Electrical circuit design [TC]
Show more
July 11
Stock up
July 11
×
Elise Grootscholten | Loraine Nelson | Paul Reusink
To have some basic biobricks ready, we stocked them by doing a plasmid isolation and freeze the DNA.
01
Stock up
Show more
July 11
Basic parts
July 11
×
Elise Grootscholten | Loraine Nelson | Paul Reusink
To start on a basic construct we did a digestion and ligation with the parts we isolated earlier. To prevent original constructs we did a defosphorylation after the digestion. We controlled the digestion by doing a gelelktroforeses.
01
Basic parts
Show more
July 11
Board layout [TC]
July 12
×
Tom van Dijk | Mike Vrieswijk
Started working on the board layout for the PCB of the temperature controller.
01
Board layout [TC]
Show more
July 12
Basic parts
July 12
×
Tom van Dijk | Randall de Waard
We continued with the ligated DNA by transformating it to Neb10Beta. This was plated onto agarplates and incubated for 1 day.
02
Basic parts
Show more
July 12
Board layout [TC]
July 13
×
Tom van Dijk | Mike Vrieswijk
Further work was done to the board layout for the temperature controller.
02
Board layout [TC]
Show more
July 13
Basic parts
July 13
×
Elise Grootscholten | Randall de Waard
The results of 7/12/2018 were collected, in action of this we decided to plate some ligated and transferred DNA from 7/12/2018 again. This we did because there wasn't a clear grow of colonies on the plates.
03
Basic parts
Show more
July 13
Blue White screening on paper
July 13
×
Elise Grootscholten | Randall de Waard
We tried to let bacteria grow on paper, therefore we used our own buisnesscards. We let the buisnesscards absorb some LB-medium and put them inside petridishes.
01
Blue White screening on paper
Show more
July 13
Board layout [TC]
July 15
×
Tom van Dijk | Mike Vrieswijk
Work has been done on the board layout for the temperature controller. The board layout is now finished.
03
Board layout [TC]
Show more
July 15
Assessing CooA Production
July 15
×
Dustin van der Meulen
After we transformed our ligation results from the 11th on the 12th, we recieved the following results on the 13th:
B0030+J23100: No growth
B0030+J23105: Unsure (small sports, maybe colonies)
B0031+J23100: Growth
B0031+J23105: Unsure (small sports, maybe colonies)
B0032+J23100: Growth
B0032+J23105: Unsure (small sports, maybe colonies)
K352001+B0015: Growth
On the 13th we retransformed the 4 ligations of which we didn't get conclusive results, which we looked at today:
B0030+J23100: No growth
B0030+J23105: Again unsure
B0031+J23105: No growth
B0032+J23105: No growth
After assessing the results we made overnight cultures of the following:
B0031+J23100
K352001+B0015
B0032+J23100
B0030+J23105 (1)
B0030+J23105 (2)
B0032+J23105
Also we retransformed the colonies we retransformed on the 13th again, and quickly spun our cells in a centrifuge, removing most of the supernatant and resuspending our cells before we spread them on Chloramphenicol plates.
07
Assessing CooA Production
Show more
July 15
Basic parts
July 16
×
Mei Ju Goemans | Elise Grootscholten | Dustin van der Meulen | Randall de Waard
To know if the ligation was succesfull we will controll the DNA, therefor we ented the bacteria's with DNA in LB-medium to grow overnight. We also plated the used bacteria's onto new plates to use for further experiments.
04
Basic parts
Show more
July 16
Blue White screening on paper
July 16
×
Mei Ju Goemans | Elise Grootscholten | Dustin van der Meulen | Randall de Waard
We did a blue white screening on our buisnesscards to see if bacteria had grew on them. We transferred the buisnesscards to new petrydishes and added ITPG and X-gal to the petrydishes.
02
Blue White screening on paper
Show more
July 16
Notebook
July 17
×
Rens Boeser
Frontend design for the notebook
02
Notebook
Show more
July 17
Basic parts
July 17
×
Rens Boeser | Dustin van der Meulen | Loraine Nelson  | Suzanne Romeijn | Randall de Waard
The resulst of 7/16/2018 showed that the DNA didnt ligate as planned.  To be able to digest enough DNA we did a plasmid isolation again. We digested this DNA and froze it for further use.
05
Basic parts
Show more
July 17
Stock up
July 18
×
Dustin van der Meulen | Loraine Nelson | Suzanne Romeijn | Mike Vrieswijk | Randall de Waard
We did another round of plasmid isolation for stocking.
02
Stock up
Show more
July 18
Notebook
July 19
×
Rens Boeser
Designed a way to follow experiments that take up multiple days and eventually designed a notebook entry page and made it a responsive page
03
Notebook
Show more
July 19
Basic parts
July 19
×
Dustin van der Meulen | Mei Ju Goemans| Randall de Waard
We digested all DNA we have in stock to test for our basic construct. We tested the DNA by doing a gelelktroforeses.
06
Basic parts
Show more
July 19
Notebook
July 21
×
Rens Boeser
Created a one page notebook. Made the notebook entry page more suited for the entry length
04
Notebook
Show more
July 21
Assessing Gas Production BioBricks in E.Coli
July 23
×
Elise Grootscholten | Randall de Waard
Transformations of the biobricks K133071, K173003 and I13453 are performed. K133071 will produce CO2 if there's pyruvate present and K173003 will produce CO2 if urea is present. I13453 is a promotor which will work if there's arabinose present.                                                                                                        
The biobricks were transformed first into electrocometent cells and later into chemical competent cells from the strain NEb10Beta. After transformations the culture was plated on agar plates with antibiotics. The first transformations didn't work out, but the second did, because then the right competent cells were used. 
Experiment 1: Transformation biobricks
Transformation of the biobricks K133071, K173003 and I13453 in NEB10Bèta.
Materials 
- Biobricks (Bba_K133071, Bba_K173003 and Bba_I3453) 
- Nuclease free water
- LB- agar
- Variable Volume Pipettes 
- Sterile pipetpoints
- Chemically competent cells ( NEB10Beta, 1x10^9) 
- Heater (42°C)  
- Incubator + shaker (37°C and 250 rpm) 
- Eppendorf tubes (1,5 and 2 ml) 
- Stable outgrow medium for NEB10Beta or SOC- outgrow medium 
-Chloramphenicol (40 mg/mL)
Methods
-K173003 --> pSB1C3 backbone, plate 6, well 15N
-K133071 --> pSB1C3 backbone, plate 6, wel 21A
-I13453 --> pSB1C3 backbone, plate 3, well 19O
To be able to use the DNA from the biobricks, they were first diluted in Nuclease free water. This was done by adding 10ul of nuclease free water to the DNA and incubated for 5 minutes at room temperature (18-24°C). After incubation the DNA is transferred to Eppendorf tubes and put on ice. 
For the transformation 1 ul of the DNA was added to 50 ul chemically competent cells strain NEB10Beta. The mixture was incubated for 30 minutes on ice. The mixture got a heat shock by putting the tube in a heater (42°C) for 30 seconds. After the heat shock the tube was immediately put on ice for 5 minutes. 
For good grow of the cells, 950 ul of outgrow medium was added to the mixture. After mixing the cells with outgrow medium, the culture was incubated for 1 hour at 37°C and 250 rpm.  
After incubation the culture was plated onto LB-agar plates. For specific growth of bacteria there was antibiotics added to the plates. The antibiotics that were used, depended on the plasmid. The biobricks used in this experiment were resistent to chloramphenicol (Can). We worked with a work concentration of 35 ug/ml for chloramphenicol. For good results we plated 100 ul undiluted culture onto a plate and centrifuged the rest of the culture. The supernatant was discarded and the pellet was resuspended and plated onto a LB-agar plate. The plates were then incubated by 37°C. 
Results
The first transformation was done in electrocompetent cells instead of chemocompetent cells so the first transformation did not succeed.
 
Conclusion
The second transformation from the biobricks was successful. The bacteria with the biobricks plasmids will be stored by 4°C and can be used for further experiments. 
The CFU from the negative controle will be plated on a plate with antibiotics to make sure there isn't a plasmid in the bacteria. The outgrow medium will be sterilized by filter again. 
 
01
Assessing Gas Production BioBricks in E.Coli
Show more
July 23
ATP sensor
July 24
×
Rens Boeser | Mei Ju Goemans | Elise Grootscholten | Dustin van der Meulen | Loraine Nelson | Suzanne Romeijn | Randall de Waard
New biobricks, used to make an ATP sensor, were transformed into NEB10Beta. The culture was than plated onto agar and incubated for 1 day.
01
ATP sensor
Show more
July 24
Gas output
July 24
×
Rens Boeser | Mei Ju Goemans | Elise Grootscholten | Dustin van der Meulen | Loraine Nelson | Suzanne Romeijn | Randall de Waard
New biobricks, which produces gasses, were transformed into NEB10Beta. The culture was than plated onto agar and incubated for 1 day.
01
Gas output
Show more
July 24
Notebook generator
July 24
×
Rens Boeser
Started working on an automatic notebook creator
01
Notebook generator
Show more
July 24
Assessing Gas Production BioBricks in E.Coli
July 24
×
Elise Grootscholten | Randall de Waard
To be able to always get the necessary biobricks, there have been made glycerolstocks of the transformed biobricks K133071, K173003 and I13453. The glycerolstocks are stored at -80 degrees Celsius. If needed, they can be retrieved from this storage to use for experiments. 
Experiment 2: Glycerolstocks
Making glycerol stocks of the biobricks K133071, K173003, I13453. 
Materials
- 87% glycerol 
- culture with biobrick plasmids
- LB-medium
- Variable Volume Pipette
- Sterile pipetpoints
Methods
The colonies from the transformation were used to make an overnight culture. The culture was made by putting a colony in LB-medium and then the culture was incubated for a night at 37°C.  
The next day the cultures had grown close and were ready for a glycerol stock. For making glycerol stocks there was added 250 ul 87% glycerol to 1 ml culture. This was frozen in -80°C.  
The stocks were stored at -80°C in drawer 3, tower 1, drawer 
02
Assessing Gas Production BioBricks in E.Coli
Show more
July 24
Assessing different ATP sensors
July 24
×
Loraine Nelson | Elise Grootscholten | Paul Reusink
Today we did a transformation with new biobricks. The biobricks were built into chemically competent cells from the strain NEb10Beta. After transformation the culture was plated on agar plates with antibiotics. The results were checked the next day. 
Materials: 
- Biobricks (Bba-K1499004 and Bba- K1614019) 
- Nuclease free water
- LB- agar
- Variable Volume Pipettes 
- Sterile pipetpoints
- Chemically competent cells ( NEB10Beta, 1x10^9) 
- Heater (42°C)  
- Incubator + shaker (37°C and 250 rpm) 
- Eppendorf tubes (1,5 and 2 ml) 
- Stable outgrow medium for NEB10Beta or SOC- outgrow medium 
Method: 
To be able to use the DNA from the biobricks, they were first diluted in Nuclease free water. This was done by adding 10ul of nuclease free water to the DNA and incubated for 5 minutes at room temperature (18-24°C). After incubation the DNA is transferred to Eppendorf tubes and put on ice. 
For the transformation 1 ul of the DNA was added to 50 ul chemically competent cells strain NEB10Beta. The mixture was incubated for 30 minutes on ice. The mixture got a heat shock by putting the tube in a heater (42°C) for 30 seconds. After the heat shock the tube was immediately put on ice for 5 minutes. 
For good grow of the cells, 950 ul of outgrow medium was added to the mixture. After mixing the cells with outgrow medium, the culture was incubated for 1 hour at 37°C and 250 rpm.  
After incubation the culture was plated onto LB-agar plates. For specific growth of bacteria there was antibiotics added to the plates. The antibiotics that were used, depended on the plasmid. The biobricks used in this experiment were resistent to chloramphenicol (Can). We worked with a work concentration of 35 ug/ml for chloramphenicol. For good results we plated 100 ul undiluted culture onto a plate and centrifuged the rest of the culture. The supernatant was discarded and the pellet was resuspended and plated onto a LB-agar plate. The plates were then incubated by 37°C. 
Results:
Tabel 1 shows the results of the plates which were incubated. 
Biobricks: 
Antibiotics: 
Dilution: 
Colonies:
CFU: 
Bba-K1614019
 Can 
0x 
109
1*10^3 /ml
Bba-K1614019
Can
Centrifuged
>300
   -
Bba-K1499004    
Can
0x
33
3*10^2 /ml
Bba-K1499004        
Can
Centrifuged
228
   -         
Positive controle: Bba- K1499004
None
0x
> 300
   -
Negative controle: outgrow medium
none
0x
18
   - 
Tabel 1: results from the transformation with the biobricks Bba-K1614019 and Bba-K1499004. 
The plates show CFU which can be used for further experiments.  The negative controle also has CFU, this shows that the outgrow medium was infected and not sterile. 
The plates are stored by 4°C. 
Conclusion: 
The transformation from the biobricks was successful. The bacteria with the biobricks plasmids will be stored by 4°C and can be used for further experiments. 
The CFU from the negative controle will be plated on a plate with antibiotics to make sure there isn't a plasmid in the bacteria. The outgrow medium will be sterilized by filter again. 
 
01
Assessing different ATP sensors
Show more
July 24
Notebook generator
July 25
×
Rens Boeser
Automatic generation of entries completed
02
Notebook generator
Show more
July 25
Google Drive API
July 25
×
Rens Boeser
Used the Google drive REST API to download the entries from drive
01
Google Drive API
Show more
July 25
Automatic uploader
July 25
×
Rens Boeser
Used the igemwiki API to update pages using software
01
Automatic uploader
Show more
July 25
Assessing Gas Production BioBricks in E.Coli
July 27
×
Elise Grootscholten | Randall de Waard
For further experiments there is isolated DNA needed of the biobricks J23100, K133071, K173003 and I134353. The DNA is isolated out of the bacteria with the help of a plasmid purifaction kit. After isolation this DNA can be used for digestion and ligation or other experiments. J23100 (from the glycerolstock): 325,59 ng/ul, K173003: 217,06 ng/ul, K133071: 186,79 ng/ul, I13453: 88,18 ng/ul
Experiment 3: Minipreps (DNA isolation)
27-07-18
Minipreps of the following biobricks in NEB10Bèta:
-J23100: Constitutive promoter
-K133071: Urea --> ammonia + CO2
-K173003: Pyruvate --> acetaldehyde + CO2
-I134353: Promoter (AraC protein binds with arabinose)
Materials 
- Mini prep, Plasmid Purification Kit, Machery Nagel 
- Eppendorf cups
- Variable Volume Pipets
- sterile pipet points
-Chloramphenicol (40 mg/mL)
Methods 
Miniprep protocol Plasmid Purification Kit, Machery Nagel
After the transformation we made an overnight culture from the colonies. The colonies were anted into 20 ml LB and incubated overnight at 37°C and 150 rpm. The culture was mini prepped the next day. For isolation about 6 mL overnight culture was used.
Plasmid DNA purification (NucleoSpin� Plasmid EasyPure)
Machery Nagel Mini preps
 
1 Cultivate
and harvest bacterial cells
12,000 x g, 30 s
 
2 Cell lysis 150 μL Buffer A1
250 μL Buffer A2
RT, up to 2 min
350 μL Buffer A3
3 Clarification of the lysate
> 12,000 x g, 3 min
4 Bind DNA
Load supernatant
1,000�2,000 x g, 30 s
5 Wash and dry silica
membrane
450 μL Buffer AQ
> 12,000 x g, 1 min
6 Elute DNA 50 μL Buffer AE
RT, 1 min
> 12,000 x g, 1 min
Results
J23100: 325,59 ng/ul
K173003: 217,06 ng/ul
K133071: 186,79 ng/ul
I13453: 88,18 ng/ul
Figure 1: Minipreps 27-07-18
03
Assessing Gas Production BioBricks in E.Coli
Show more
July 27
Assessing different ATP sensors
July 27
×
Loraine Nelson | Elise Grootscholten | Paul Reusink
To be able to always get the necessary biobricks, we made an glycerol stock after every transformation. The glycerol stocks are stored at -80 degrees Celsius. If needed, they can retrieved from this storage to use for experiments. 
Making glycerol stocks of the biobricks. 
Materials: 
- 87% glycerol 
- culture with biobrick plasmids
- LB-medium
- Variable Volume Pipette
- Sterile pipetpoints
Method: 
The colonies from the transformation were used to make an overnight culture. The culture was made by putting a colony in LB-medium and then the culture was incubated for a night at 37°C.  
The next day the cultures had grown close and were ready for a glycerol stock. For making glycerol stocks there was added 250 ul 87% glycerol to 1 ml culture. This was frozen in -80°C.  
The stocks were stored at -80°C in drawer 3, tower 1, drawer 
02
Assessing different ATP sensors
Show more
July 27
Automation program
July 29
×
Rens Boeser
Connected the page uploader to the notebook generator
01
Automation program
Show more
July 29
Automatic uploader
July 29
×
Rens Boeser
Created a program that uploads all updated pages
02
Automatic uploader
Show more
July 29
Assessing CooA Production
July 29
×
Dustin van der Meulen
After considering multiple tactics, we've decided to follow the standard iGEM 3A assembly in our following cloning steps. To this end we started with an overnight digest of all the different parts.
BioBrick BBa_ code
Restriction enzyme 1
Restriction enzyme 2
J23100
EcoRI
SpeI
J23105
EcoRI
SpeI
J23113
EcoRI
SpeI
B0030
EcoRI
XbaI
B0031
EcoRI
XbaI
B0032
EcoRI
XbaI
K352001
SpeI
PstI
B0015
XbaI
PstI
Used pipette scheme from protocol 7 to prepare the DNA to be digested with multiple combinations of EcoRI-HF, SpeI-HF, PstI and XbaI.
With this we started a digest of the following BioBricks using the following Restriction enzyme couples:
BioBrick BBa_ code
Restriction enzyme 1
Restriction enzyme 2
J23100
EcoRI
SpeI
J23105
EcoRI
SpeI
J23113
EcoRI
SpeI
B0030
EcoRI
XbaI
B0031
EcoRI
XbaI
B0032
EcoRI
XbaI
K352001
SpeI
PstI
B0015
XbaI
PstI
Digestion was started on the 30th of July, ~16:00, 37°C and 185 RPM.
Digestion was stopped on the 31st of July, ~10:00.
09
Assessing CooA Production
Show more
July 29
Making competent NEB10beta cells
July 30
×
Loraine Nelson | Jos Veldscholte
Made stock solution of MgSO4 and pure culture of NEB10 beta cells
Making 1 M sterile MgSO4
≈ 14.00: Dissolved 12,32 g MgSO4 in 50 mL demi water en filtered through 0.22 μm filter.
Making pure culture of NEB10 beta cells
≈15.00: 20 mL LB agar + 10 μL tetracyclin (50 mg/mL) to pour 1 plate.
16:30: streaked bacteria from glycerol stock: -80, section III, tower 1, shelf IV, box III, 6B
Left in 37°C room for o.n. growth.
01
Making competent NEB10beta cells
Show more
July 30
Page generator
July 31
×
Rens Boeser
Built a page builder that automatically adds the header and footer before uploading
01
Page generator
Show more
July 31
Google Drive API
July 31
×
Rens Boeser
Redone the google API handler for reusability and readability and made the google API handler generic
02
Google Drive API
Show more
July 31
Automation program
July 31
×
Rens Boeser
revamped the main program
02
Automation program
Show more
July 31
Assessing CooA Production
July 31
×
Dustin van der Meulen
Though we had started with our ligation using the rAPId dephos and ligation kit, we'd simultaniously put our DNA on gel, later revealing the digest to not have been succesful. Carefull examination of our work made us realise the used restriction enzyme dilutions to not have been made correctly.
Using the rAPid Dephos and Ligation Kit we performed a dephosphorilation and ligation of our DNA.
After checking our gel with our digestion products we later determined that the digestion didn't go as expected
 
10
Assessing CooA Production
Show more
July 31
Making competent NEB10beta cells
July 31
×
Loraine Nelson | Jos Veldscholte
Made Ksi broth (for competent cells) and started o.n. cultures
Make Ksi broth (for competent cells)
Dissolved 20 g Tryptone (Lab M limited) and 5 g Yeast extract (Lab M limited) in 1 liter demi water, added 10 ml 1 M NaCl and adjusted pH to 7.5 with KOH.
Autoclaved.
16:30
2 tubes with 2.5 mL Ksi inoculated with 1 colony NEB10beta from yesterday's plate. (Added 50 microliters of 1 M MgSO4 at 16:30 becauce I forgot to ad it to de autoclaved medium. 
Added 10 mL of 1 M MgSO4 to each half liter autoclaved medium (=Ksi).
02
Making competent NEB10beta cells
Show more
July 31
Automation program
August 1
×
Rens Boeser
Finished redoing the application for generating and uploading pages
03
Automation program
Show more
August 1
Making competent NEB10beta cells
August 1
×
Loraine Nelson | Jos Veldscholte
Made large NEB10 beta culture and aliquoted in 50 microliter portions and snap froze in dry ice/isopropanol
Making competent NEB10 beta cells
8:45:
Both cultures were fine.
Transfered one culture completely to 100 ml Erlenmeyer with 25 mL Ksi (RT). In shaking incubator at 37° C at 200 rpm.
Filled three one liter Erlenmeyers with 175 mL Ksi and put in shaking incubator in 37° C room to adjust temperature.
25 ml culture: at 8:40 OD600 was 0.64.
Transferred 8 ml to each 175 mL Erlenmeyer. Incubate at  37° C shaking rigorously (no speed indicator).
9:40: OD600 = 0.-9
10:30: OD600 = 0.30
10:50 OD600 = 0.46
10:58 OD600 = 0.55
Tranferred all cultures to 10 50 mL tubes and tare in pairs.
On ice 11:15
11:35: in Sorvall 4500 rpm, 15 min 0° C.
After the run, fluid was on the rotor and in the rotor holes.
Cleaned with lab desinfectant. Rinsed with water and dried with tissue.
From here on everything in the cold room with tubes on ice:
Resuspended pellets in 16,6 ml ice cold TFB1 (protocol "competente-cellen-maken"l) each with 10 mL plastic pipet and pipetboy. 
Combined into four 50 mL tubes. Centrifuged 15 min at 4500 rpm.
Resuspended pellets in 60 mL TFB1, kept on ice for 20 min and centrifuged 15 min 4500 rpm.
Respuspended pellets in 10,5 ml TFB2 (protocol "competente-cellen-maken"l). Left on Ice for 10 min.
Aliquotted into 50 μL portions in 2 mL conical tubes and snap froze in isopropanol on dry ice.
Transferred to -80°C Freezer.
03
Making competent NEB10beta cells
Show more
August 1
Notebook
August 2
×
Rens Boeser
Added month categorization to the page
05
Notebook
Show more
August 2
Assessing CooA Production
August 2
×
Dustin van der Meulen
We've redone the work mentioned in the previous CooA entry, though today our gel electrophoresis revealed we were only partially succesful, as only the digests with EcoRI and SpeI seem to be correct.
Experiment 9 and 10 have been repeated with a new enzyme dilution.
7th of august: review of the digestions on gel reveal that only the digestions with EcoRI-HF and SpeI-HF worked, while the PstI and XbaI digestion did not retrieve the expected results
11
Assessing CooA Production
Show more
August 2
Making competent NEB10beta cells
August 2
×
Loraine Nelson | Jos Veldscholte
Tested competence by transformation of pUC19
Test sample from NEB competent cells contains 50 pg pUC19 per μL.
Used 2 μL per transformation (= 100 ng).
Took three tubes from one of the two boxes.
A 100 ng pUC19
B 100 ng pUC19
C nothing
30 min on ice (11:00)
A 30 s heatshock in waterbath (11:30)
B 90 s heatshock in waterbath (11:30)
C was kept on ice still.
5 min on ice.
Added 1 ml of LB (11:40)
60 min shaking in 37 °C room (shaking slowly)
Poured LB agar plates with 50μg/ml Ampicillin and one plate without antibiotic.
12:40
Added 1 μL of culture A or B to 99 μL LB (A4 and B4)
Added 10 μL of culture A or B to 90 μL LB (A3 and C3)
Plated A3 and 4 and C3 and 4 on LB agar amp.
Plated 100 μL A, B and C (A2, B2 and C2) on LB agar amp.
Plated 100 μL C on plate without antibiotics (to test whether cells are alive)
Centrifuged A and B 6000 rpm in microcentrifuge for 2 min.
Aspirated supernatant.
Vortexed bacteria in remainder of the medium (>100 μL)
Plated on LB agar amp. plates (A1 and B1).
Consequently:
A was heat shocked for 30 s.
B was heat shocked for 90 s.
1 = 90% of the transformation, 2 = 10%, 3 = 1% and 4 = 0,1%
Put in automated incubator. Not sure about whether the sytem wil work. Is it on?
Colonies to be counted next monday by Dustin.
If there are no colonies visible, please extend incubation time!
04
Making competent NEB10beta cells
Show more
August 2
Making competent NEB10beta cells
August 3
×
Loraine Nelson | Jos Veldscholte
Counted colonies. Result: no transformation.
Colonies where counted at ~10:00. Only plate A1 showed signs of colonies (~15). All the others where empty.
After leaving the cells in the incubator for an extra half a day, there where no new colonies.
05
Making competent NEB10beta cells
Show more
August 3
Assessing CooA Production
August 4
×
Dustin van der Meulen
The previous two CooA entries have been repeated, though this time we remade the PstI and XbaI enzyme dilutions. This time the digests where as expected.
Experiment 11 has been repeated, but with new dilutions of PstI and XbaI.
After the incubation it was noted that the cup containging BBa_B0015 was almost empty, so it will have to be digested again. (did not happen) --> in the end we had enough
13th of august --> dephos and ligation was peformed on the obtained biobrick combinations following protocol 8
 
12
Assessing CooA Production
Show more
August 4
Making competent NEB10beta cells
August 7
×
Loraine Nelson | Jos Veldscholte
Repeated transformation.
Part 4 has been repeated, repeating the same steps but adding two extra positive controlls.
06
Making competent NEB10beta cells
Show more
August 7
Assessing different ATP sensors
August 9
×
Loraine Nelson | Elise Grootscholten | Paul Reusink
For further experiments we needed isolated DNA. The DNA is isolated out of the bacteria with the help of a plasmid purifaction kit. After isolation this DNA can be used for digestion and ligation or other experiments.
Materials: 
- Mini prep, Plasmid Purification Kit, Machery Michels 
- Eppendorf cups
- Variable Volume Pipets
- sterile pipet points
Method: 
After the transformation we made an overnight culture from the colonies. The colonies were anted into 20 ml LB and incubated overnight at 37°C and 150 rpm. The culture was mini prepped the next day. For isolation, 5,4 ml was added step wise in a 2ml Eppendorf cup. The culture was then centrifuged at >12000x g for 30 seconds. The supernatant was deposed, and the pellet was resupended with A1 buffer and vortexed. To the resuspended culture was then. 
03
Assessing different ATP sensors
Show more
August 9
Assessing Gas Production BioBricks in E.Coli
August 10
×
Elise Grootscholten | Randall de Waard
The biobricks J23100, K133071, K173003 and I134353 were sucessfully digested after the second time. After the digestions the biobricks K133071 and K173003 were dephosphorylated and ligated with the inserts J23100 and I13453. This was done in the original backbone of K133071 and K173003, and not another control backbone. To know if the biobricks were right ligated this was done by testing practically. See the experiments: Testing gas production. 
Experiment 4: Digestion, Defosforylation and ligation
Digestions
30-07-18 and 08-08-18
Biobricks: J23100, K133071, K173003 and I13453
Used pipette scheme (Table 1) to prepare the DNA to be cut with multiple combinations of EcoRI-HF, SpeI-HF, PstI and XbaI.
DNA
~ 2 μg
NEBbuffer 2.1
5 μL
Restriction enzyme 1 (1 U/μL)
2 μL
Restriction enzyme 2 (1 U/μL)
2 μL
Nuclease free water
Fill to 100 μL
(Table 1 ; Double Digest Pipette Scheme using :  https://nebcloner.neb.com/#!/redigest , and doubling the amount)
Biobricks:
J23100: Constitutive promotor
K133071: Urea --> ammonia + CO2
K173003: Pyruvate --> acetaldehyde + CO2
I134353: Promotor (AraC protein binds with arabinose)
With this we started a digest of the following BioBricks using the following Restriction enzyme couples (Table 2)
 
Number
BioBrick BBa_ code
Restriction enzyme 1
Restriction enzyme 2
9
J23100 (35 bp)
EcoRI
SpeI
11
K133071 (1707 bp)
EcoRI
XbaI
12
K173003 (3052 bp)
EcoRI
XbaI
10
I13453 (130 bp)
EcoRI
SpeI
Table 2: Biobricks and restriction enzymes
Digestion was started on the 30th of July, ~16:00, 37°C and 185 RPM.
Digestion was stopped on the 31st of July, ~10:00.
This has been done in tandem with the corresponding part from  Cloning Mulitple BioBricks to Assess CooA Production  Part X 
Dephosphorylation
10-08-18
K133071 and K173003 were dephosphoralized:
 
Volume
Compound
25 μL
Vector DNA (0,5 μg)
3 μL
phosphatase buffer (10x)
1 μL
milli Q
1 μL
phosphatase (1 U/μL)
30 μL
Total
Table 3: Pipetting scheme
The new DNA concentration of the reaction mix is 0,5 μg/30 μL
The reaction mix was incubated for 10 minutes at 37 °C
The phosphatase was inactivated for 2 minutes at 75 °C
 
 
 
 
For the ligation mix 100 ng of vector DNA was used from the dephosphorylation reaction mix.
The following vector-insert combinations were ligated:
Vector
Insert 1
Insert 2
K133071
J23100
I13453
K173003
J23100
I13453
 
 
 
Table 4: Biobrick combination
The original plasmids of the vector and inserts that were digested are about the same size in bp and the ratio vector DNA-insert DNA is determined to be 1:3. Therefore, the amount of insert DNA that was used for the ligation was 140 ng (rounded up to 7 μL of the digestion reaction mix). Because there was only 5 μL left of the K352001 insert DNA, the total 5 μL were used and an extra 2 μL of milli Q water were added to the pre-ligation mix.
Pre-Ligationmix:
Volume
Compound
6 μL
Vector DNA (0,5 μg/30 μL)
7 μL
Insert DNA (2 μg/100 μL)
4 μL
Delutionbuffer (5x)
3 μL
Milli Q
20 μL
Total
Table 5: Pre-ligationmix pipette scheme
Ligationmix:
incubation at room-temperature for 20 minutes.
Volume
Compound
10 μL
Pre-ligationmix
2 μL
10x T4 ligationbuffer
1 μL
T4 DNA ligase (1U/μL)
8 μL
Milli Q
20 μL
Total
Table 6: Ligationmix pipette scheme
 
All the vectors contain the CamR gene. 30 plates were poured with Cam (14 for the transformed cells (100μL), 14 for the transformed cells (900 μL centrifuged and resuspended in 100 μL and 1 positive and 1 negative control)
 
04
Assessing Gas Production BioBricks in E.Coli
Show more
August 10
Urea and sodium pyruvate test for resistance E.coli
August 10
×
Randall de Waard | Elise Grootscholten
Testing different amounts of urea and sodiumpyruvate to know which concentrations the bacteria survive.
Experiment 1: Testing different concentrations of Urea and Sodium pyruvate
08-08-18
Testing different amountsof urea and sodiumpyruvate to know which concentrations the bacteria survive.
-First let the LB agarose heat up (95 degrees Celcius)
-Second, weigh urea and sodium pyruvate in beakers (see table 1)
What
How much
Solution
Urea
1 g
1 mL purified water
Urea
2,4 g
2 mL purified water
Urea
4,8 g
4 mL purified water
Sodium pyruvate
5,7 mL
None
Table 1: weighing urea and sodium pyruvate for LB agarose plates.
-Purify these substances, so it is free of bacteria. Work beneath the flame and pour it in to plastic 50 mL tubes.
-Pour the LB agarose in the plastic tubes when it's about 45 degrees Celcius and mix gently.
-Pour the LB agarose mix in a petridish and wait until it's dry. Do this for all the plastic tubes.
-Also fill 2 petridishes with LB agarose without other substances (positive and negative control).
-prick a BL21(DE3) bacteria and put it in a 100 mL erlenmeyer with 20 mL culture medium.
09-08-18
Pipette 100 ul overnight culture on each plate, except on the negative control, and divide it. (24h 37 degrees Celcius)
 
Results
 
What
How much
Solution
Result
Urea
1 g
1 mL purified water
Grow
Urea
2,4 g
2 mL purified water
No grow
Urea
4,8 g
4 mL purified water
No grow
Sodium pyruvate
5,7 mL
-
Grow
Positive control
-
-
Grow
Negative control
-
-
No grow
Table 1: Results experiment 1
Conclusion
More tests will be done with urea and sodiumpyruvate because with 1 g urea the bacteria live and with 2,4 g urea the bacteria die.
The bacteria survive 5,7 mL sodiumpyruvate so we will test with higher concentrations.
 
01
Urea and sodium pyruvate test for resistance E.coli
Show more
August 10
Assessing Gas Production BioBricks in E.Coli
August 13
×
Elise Grootscholten | Randall de Waard
Transformations of the biobricks (K133071 + J23100), (K13071 + I1345), (K173003 + J23100) and (K173003 + I13453) in NEB10Bèta. There was no grow except for the biobrick combination K133071 + J23100.
Experiment 5: Transformation
13-08-2018
Transformations of the ligations of experiment 4
Materials
-Chloramphenicol (40 mg/mL)
-LB-agar
-Ligations
-Heat block,42 degrees Celcius
-Shaker, 37 degrees Celsius 
-Chemo competent cells NEB10bèta (New England Biolabs)
Methods
10 ul ligationmix is used for every transformation
Transformation Protocol
Overview
Quick Ligation products may be transformed by many different methods. The following protocol is recommended by New England Biolabs.
Protocol
Thaw competent cells on ice.
Chill approximately 5 ng (2 μl) of the ligation mixture in a 1.5 ml microcentrifuge tube.
Add 50 μl of competent cells to the DNA. Mix gently by pipetting up and down or flicking the tube 4�5 times to mix the cells and DNA. Do not vortex.
Place the mixture on ice for 30 minutes. Do not mix.
Heat shock at 42°C for 30 seconds*. Do not mix.
Add 950 μl of room temperature media* to the tube.
Place tube at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.
Warm selection plates to 37°C.
Spread 50�100 μl of the cells and ligation mixture onto the plates.
Incubate overnight at 37°C.
* Please note: For the duration and temperature of the heat shock step as well as for the media to be used during the recovery period, please follow the recommendations provided by the competent cells� manufacturer.
Instead of 2 ul of the ligation mix, use 10 ul ligation mix.
05
Assessing Gas Production BioBricks in E.Coli
Show more
August 13
Assessing Gas Production BioBricks in E.Coli
August 14
×
Elise Grootscholten | Randall de Waard
A colony PCR is done for the NEB10Bèta E.coli cells with expected the biobrick combination of K133071 with J23100. Nevertheless, on a gel the difference with and without promotor couldn't be seen. So there must be another way of proving the right biobricks are there.
Expermiment 6: Colony PCR K133071 + J23100
14-08-18
Mastermix
What
1 reaction
30 reactions
Taq buffer(10x)
5 ul
150 ul
10 mM dntp
1 ul
30 ul
10 uM Forward primer
1 ul
30 ul
10 uM Reverse primer
1 ul
30 ul
Taq polymerase
0,25 ul
7,5 ul
Nucease free hydrogen
41,75 ul
1252,5 ul
Total
50 ul
1500 ul
 
06
Assessing Gas Production BioBricks in E.Coli
Show more
August 14
Assessing Gas Production BioBricks in E.Coli
August 15
×
Elise Grootscholten | Randall de Waard
The ligations and transformations of the biobricks K173003 + J23100, K173003 + I13453, and K133071 + I13453 have been performed again. There was a lot of grow on the agar plates. To know if the biobricks were right ligated this was done by testing practically. See the experiments: Testing gas production. 
Experiment 7: Ligation of biobricks
15-08-18
K173003 + J23100
K173003 + I13453
K133071 + I13453
 
Volume
Compound
6 μL
vector DNA (0,5 μg/30 μL) (100 ng)
14 μL
insert DNA (2 μg/100 μL) (280 ng)
4 μL
dilutionbuffer (5x)
1 μL
milli Q water
25 μL
Total
Use 10 mL
Ligationmix:
Volume
Compound
10 μL
pre-ligationmix 
2 μL
10x T4 ligationbuffer
1 μL
T4 DNA ligase (1U/μL)
8 μL
milli Q water
20 μL
Total
incubation at room-temperature for 20 minutes.
Store at -20 degrees Celcius
07
Assessing Gas Production BioBricks in E.Coli
Show more
August 15
Assessing CooA Production
August 16
×
Dustin van der Meulen
We've performed a colony PCR on 15 random (though not the red/purple -esque colonies, which are collored only because they contain BBa_J04450) colonies from J23105+B0032 on the 13th. We also ligated the following biobricks anew:
B0031 + J23105
B0031 + J23113
B0032 + J23100
B0032 + J23105
We also transformed these biobricks into E.coli on the 16th, prepping and digesting them later on, though these and the PCR did not give us the result we'd hoped for.
Today a colony PCR will be performed to check our results from last friday, namely the transformation product of BBa_B0032 en BBa_J23105:
Mastermix
What
1 reaction
30 reactions
Taq buffer(10x)
5 ul
150 ul
10 mM dntp
1 ul
30 ul
10 uM Forward primer
1 ul
30 ul
10 uM Reverse primer
1 ul
30 ul
Taq polymerase
0,25 ul
7,5 ul
Nucease free hydrogen
41,75 ul
1252,5 ul
Total
50 ul
1500 ul
15 colonies have been randomly selected to be checked via PCR.
 
13
Assessing CooA Production
Show more
August 16
Testing gas production
August 16
×
Elise Grootscholten | Paul Reusink
Making a set up for the gas production testing and testing it with NEB10Bèta with the expected biobricks in it (K133071 + J23100) and a negative control.
Experiment 1: Gasproduction testing
15-08-18
Making a set up for the gas production testing and testing it with NEB10Bèta with the expected biobricks in it (K133071 + J23100) and a negative control.
Materials
-15 mL tubes
-Little glass tubes
-Cocktail prickers
-Culture medium
-Urea
-Chloramphenicol (40 mg/mL)
Methods
Stick tape on the little tube and cover the little tubes with tin foil. This can now enter the autoclave.
Make overnight cultures (20 mL)of the desired colonies in culture medium with 17,5 ul/ 20mL Chloramphenicol.
After a day, prepare the culture medium for the tests.
Put 1 g urea and 17,5 ul Chloramphenicol in 20 mL culture medium.
Pipette 10 mL of the overnight culture in the 15 mL tube en centrifuge 5000 rpm fot 5 minutes. Throw the supernatant away.
Fill the 15 mL tube and the little tube to the edge with culture medium. Work sterile.
Put a paper on the little tube and turn it around.
Then hold it above the 15 mL tube and pull the paper away and let the little tube fall into the 15 mL tube.
Try to do this so there's no gas in the little tube.
Now wait until gas is produced.
16-08-18
Making a set up for the gas production testing and testing it with NEB10Bèta with the expected biobricks in it (K133071 + J23100).
20 different colonies are tested.
Materials
Use the same materials as above.
Methods
Use the protocol above.
Make 700 mL culture medium with 35 g urea and 612,5 ul Chloramphenicol (40 mg/mL). This cannot be done sterile.
The culture medium can be used for the 20 different colonies with expected biobricks: K133071 + J23100.
 
Results
Gas was let into the little tubes so it was difficult to see whether there was gasproduction or not. Mainly in the negative control the tubes started with gas in it.
 
Figure 1: Little tube with cocktailpricker and 15 mL tube
Figure 2: Set up testing gas production
Figure 3. tube 1,2,3 and 4 starting amounts of air
Figure 4: After the weekend tube 1, 2, 3 and 4
Figure 5. tube 5,6,7 and 8 starting amounts of air
Figure 6: After the weekend tube 5, 6, 7 and 8
Figure 7. tube 9 and 10 strating amounts of air
Figure 8: After the weekend tube 9 and 10
Figure 9. tube 11,12,13,and 14 starting amounts of air
Figure 10: After the weekend tube 11, 12, 13 and 14
Figure 11. tube 15,16,17 and 18 starting amounts of air
Figure 12: After the weekend tube 15, 16, 17 and 18
Figure 13. tube 19 and 20 starting amounts of air
Figure 14: After the weekend tube 19 and 20
01
Testing gas production
Show more
August 16
Assessing Gas Production BioBricks in E.Coli
August 21
×
Elise Grootscholten | Randall de Waard
Minipreps are made of colonies 11 and 19 of biobrick combination K133071 + J23100. Results: 11. 270,57 ng/ul 19. 253,59 ng/ul.
Minipreps
21-08-18
Minipreps of colonies 11 and 19 (see Testing gasproduction experiment 1)
Materials
- Mini prep, Plasmid Purification Kit, Machery Michels 
- Eppendorf cups
- Variable Volume Pipets
- sterile pipet points
-Chloramphenicol (40 mg/mL)
Methods 
Miniprep protocol Plasmid Purification Kit, Machery Nagel
See experiment 3
Results
11. K133071 + J23100: 270,57 ng/ul
19. K133071 + J23100: 253,59 ng/ul
Figure 1: Results minipreps colonies 11 and 19
08
Assessing Gas Production BioBricks in E.Coli
Show more
August 21
Testing gas production
August 22
×
Elise Grootscholten | Paul Reusink
Making a set up for the gas production testing and testing it with NEB10Bèta with the expected biobricks in it (K133071 + I13453), (K173003 + J23100), (K173003 + I13453) and a negative control. Colonies 9 and 10 are chosen use for further tests because there was the biggest amount of gas production.
Experiment 2: Gasproduction testing
20-08-18
Making a set up for the gas production testing and testing it with NEB10Bèta with the expected biobricks in it (K133071 + I13453), (K173003 + J23100), (K173003 + I13453) and a negative control
Materials
See materials Experiment 1
Methods
See methods Experiment 1
850 mL culture medium is made with 21,5 g LB for 42 erlenmeyers (also for CooA production).
Overnight cultures are made of 10 colonies each of K173003 + I13453 and K173003 + J23100.
21-08-18
K173003 + J23100 are tested (see methods experiment 1)
Overnight cultures are made of  K133071 + I13453 and K173003 + I13453 are tested later.
Results
Colonies 9 and 10 are chosen use for further tests.
Figure 1. Starting amounts of air in tube 1,2,3 and 4.
Figure 2. Gas formation in tube 1,2,3 and 4. 30 minutes after adding      Sodium pyruvate.
Figure 3. Starting amounts of air in tube 5,6,7 and 8.
Figure 4. Gas formation in tube 5,6,7 and 8. 30 minutes after adding Sodium pyruvate.
Figure 5. Starting amounts of air in tube 9 and 10.
Figure 6. Gas formation in tube 9 and 10. 30 minutes after adding     Sodium pyruvate.
Figure 7: Gas formation in tube 1, 2, 3 and 4 after a dayr
Figure 8: Gas formation in tube 5, 6, 7 and 8 after a day
Figure 9: Gas formation in tube 9 and 10 after a day
02
Testing gas production
Show more
August 22
Testing gas production
August 22
×
Elise Grootscholten | Paul Reusink
Gasproduction testing for the biobricks (K173003 + I13453), (K133071 + I13453) and negative controls (B0015 and K133071 without urea and arabinose).
K173003 + I13453 is tested with sodiumpyruvate and arabinose for gasproduction and K133071 + I13453 is tested with urea and arabinose for gasproduction.
The negative control also produces a little bit gas.
Experiment 3: Gasproduction testing
Gasproduction testing for the biobricks K173003 + I13453, K133071 + I13453 and negative controls.
K173003 + I13453 is tested with sodiumpyruvate and arabinose for gasproduction and K133071 + I13453 is tested with urea and arabinose for gasproduction.
Materials
See experiment 1 and 2.
Methods
See experiment 1 and 2.
375 mL culture medium with CAM (already in it) + 18,75 g urea + 0,5 ul arabinose.
340 mL culture medium with CAM (already in it) + 51 mL natriumpyruvate (20 mM) + 0,5 ul arabinose.
Results and conclusion
Colonies 5 and 7 of the biobricks K173003 + I13453 (pyruvate)  and number 4 of the biobricks K133071 + I13453 (urea).
In both the negative controls there's been gas produced. Futher gasproduction test will be done.
Figure 1: Starting amounts in tube 1, 2, 3, 4 and 5 with Pyruvate and arabinose
Figure 2: Gasproduction in tube 1, 2, 3, 4 and 5 after a day with pyruvaat and arabinose
Figure 3: Starting amounts in tube 6, 7, 8, 9 and 10 with Pyruvate and arabinose
Figure 4: Gasproduction in tube 6, 7, 8, 9 and 10 after a day with pyruvate and arabinose
Figure 5: Starting amounts in tube 1, 2, 3, 4 and 5 with Urea and arabinose
Figure 6: Starting amounts in tube 6, 7, 8, 9 and 10 with Urea and arabinose
Figure 7: Gasproduction in tube 1, 2, 3, 4 and 5 after a day with urea and arabinose
Figure 8: Gasproduction in tube 6, 7, 8, 9 and 10 after a day with urea and arabinose
Figure 9: Starting amounts in the negative controls B0015 and K133071 + I13453 (without urea and arabinose)
Figure 10: Gasproduction in the negative controls B0015 and K133071 + I13453 after a day (without urea and arabinose)
03
Testing gas production
Show more
August 22
Testing gas production
August 24
×
Elise Grootscholten | Paul Reusink
Testing the gasproduction of the colonies 9 and 10 of biobricks (K173003 + J23100), colonies 5 and 7 of biobricks (K173003 + I13453) and colony 4 of biobricks (K133071 + I13453) with and without centrifuging the bacteria. There is also a negative control (J04450 pSB1K3) with Kanamycine. The negative contol started with a lot of gas inside the tube. We can not see wether there is produced more after a day or not. This have to be tested later.
Experiment 4: Gasproduction testing 
24-08-18
Testing the gasproduction with and without centrifuging the bacteria.
methods
See methods experiment 1
2 negative controls
colonies 9 and 10 of biobricks K173003 + J23100.
Colonies 5 and 7 of biobricks K173003 + I13453
Colony 4 of biobricks K133071 + I13453
Culture medium:
150 mL: Sodiumpyruvate (20 mM) 22,5 mL + Chloramphenicol (40 mg/mL) 131 ul
150 mL: Sodiumpyruvate (20mM) 22,5 mL +  Chloramphenicol (40 mg/mL) 131 ul + Arabinose (300 mg/mL) 0,5 ul
80 mL: Urea 4 g + Chloramphenicol (40 mg/mL) 70 ul + Arabinose (300 mg/mL) 0,5 ul
80 mL: Kanamycine (40 mg/mL) 70 ul
 
Figure 1: Starting amounts Nr. 9, 10 and 4 centrifuged
Figure 2. Gasproduction Nr 9. 10 and 4 after 4 hours centrifuged.
Figure 3: Starting amounts Nr. 5, 7 and negative control centrifuged
Figure 4. Gasproduction Nr 5,7 and negative control after 4 hours centrifuged.
Figure 5: Starting amounts Nr. 9, 10 and 4 not  centrifuged
Figure 6. Gasproduction Nr 9. 10 and 4 after 4 hours not centrifuged.
Figure 7: Starting amounts Nr. 5, 7 and negative control not centrifuged
Figure 8. Gasproduction Nr. 5, 7 and negative control after 4 hours, not centrifuged.
04
Testing gas production
Show more
August 24
Urea and sodium pyruvate test for resistance E.coli
August 28
×
Randall de Waard | Elise Grootscholten
Testing different amounts of urea and sodiumpyruvate to know which concentrations the bacteria survive.
Experiment 2: Testing different concentrations of Urea and Sodium pyruvate
13-08-18
Same testing like experiment 1 but with different concentrations.
What
How much
Solution
Urea
1,2 g
1 mL purified water
Urea
1,6 g
2 mL purified water
Sodium pyruvate (10 mM)
10 mL
None
Results
 
What
How much
Solution
Result
Urea
1,2 g
1 mL purified water
Little grow
Urea
1,6 g
2 mL purified water
No grow
Sodium pyruvate (10 mM)
10 mL
-
Grow
Positive control
-
-
Grow
Negative control
-
-
No Grow
Table 2: Results experiment 2
Conclusion
For urea is 1 g used/20 mL and for sodiumpyruvate is 6 mL (10mM) or 3 mL (20 mM) used.
 
02
Urea and sodium pyruvate test for resistance E.coli
Show more
August 28
Assessing Gas Production BioBricks in E.Coli
August 30
×
Elise Grootscholten | Randall de Waard
A miniprep of the biobrick K352002 is made. The concentration is 69,77 ng/ul.
Experiment 9: Minipreps
28-08-18
Materials
- Mini prep, Plasmid Purification Kit, Machery Nagel
- Eppendorf cups
- Variable Volume Pipets
- sterile pipet points
-Chloramphenicol (40 mg/mL)
Methods
Miniprep protocol Plasmid Purification Kit, Machery Nagel
See experiment 3
Make minipreps of the biobricks K352002, K352003
Results
The minipreps can not be used, because washing buffer without ethanol is used. The miniprep of K352002 has been done again.
29-08-18
Methods
Make a miniprep of the biobrick K352002
Results
K352002: 69,77 ng/μL
30-08-18
Methods
See experiment 3
Make minipreps of the biobricks K133116 and K173013
Figure 1: Miniprep K352002
09
Assessing Gas Production BioBricks in E.Coli
Show more
August 30
Chemo competent cells
August 30
×
None
Competent cells of NEB10 Beta and BL21 DE3 were made. These cells were made competent so new biobricks and ligation mixes could be transformed into these cells. BL21 DE3 and NEB10 beta had 80 cups of 50 ul each. 1 cup is needed for 1 transformation later. The cells were competent enough to use in the project. 
Making chemocompetent cells: 
Materials: 
- E. coli NEB10Beta strain 
- E. coli Bl21 (DE 3) strain
- Buffer 1 
30 mM natriumacetaat, 100mM rubidium chloride,10 mM calciumchloride, 50mM mangaan chloride, 15 % glycerol; pH 5,8 met verdunde azijnzuur, filter steriliseren.
Bewaren bij kamertemperatuur.
- Buffer 2 
10 mM MOPS, 75 mM calcium chloride, 10 mM rubidium chloride, 15% glycerol; pH 6,6 met NaOH, filter steriliseren.
Bewaren bij kamertemperatuur 
Method: 
The day before making competent cells the strains were ented into 20ml LB medium (100ml erlenmeyer) and incubated for 6 hours by 37 degrees at 200 rpm. The culture was then transmitted to 20 ml LB medium (100 ml erlenmeyer) in a dilution of 1:50 (400 ul/ 20 ML) this was incubated overnight at 37 degrees Celsius at 200 rpm. 
In the morning the culture was diluted 1:100 in 200 ml LB medium (1L erlenmeyer). This was then incubated by 37 degrees and 225 rpm till a OD600 level of 0,5. 
To make sure the right OD was used, the culture was measured every 20/30 minutes. The results of the measurements can be found in part 2. 
After hitting the OD600 of 0,5. The cultures were transmitted to 250 ml centrifuge tubes and cooled on ice. The cells are supposed to stay on ice as much as possible after this. After cooling down to 4 degrees, the cultures were centrifuged at 4500 rpm for 10 min at 4 degrees. The supernatant were disposed, the pellet was resuspended in 66,4 ml ice cold Buffer 1(see materials). This was centrifuged  at 5000 rpm for 10 min at 4 degrees. 
BL21DE3  80 cups on place: Drawer 3, tower 2, drawer 3, place 2. 
NEB10Beta 79 cups on place: Drawer 3, tower 2, drawer 3, place 3. 
01
Chemo competent cells
Show more
August 30
Assessing Gas Production BioBricks in E.Coli
August 31
×
Elise Grootscholten | Randall de Waard
Digestions, gelelectrophoresis, dephosphorylations and ligations of different biobrick combinations. These biobrick combinations are 4 different promoters (J23100, I13453, K352002, K352003)  with 4 different gasproduction biobricks (k173003, K173013, K133071, K133116). J23100 is about 1 kb to long. The rest seems likely to be right digested. The ligations will be transformed in NEB10bèta and digested again as control.
Experiment 10: Digestion, defosforylation and ligation
Materials
-Cutsmart buffer (10x)
-Restriction enzymes XbaI, PstI-HF, EcoRI-HF, SpeI
-Nuclease free water
-DNA
-Phosphatase buffer (10x)
-Phosphatase (1 U/μL)
-Delutionbuffer (5x)
-T4 ligationbuffer (10x)
-T4 DNA ligase (1U/μL)
Methods
 
DNA (J23100 325,59 ng/μl) ~ 2 μg
6,1
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
1 μL
Restriction enzyme 2 (1 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
Table 1: Digestion scheme 1
 
DNA (J23100 325,59 ng/μl) ~ 2 μg
6,1
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
Table 2: Digestion scheme 2
 
DNA (J23100 325,59 ng/μl) ~ 2 μg
6,1
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
Table 3: Digestion scheme 3
 
DNA (K352003 180,65 ng/μl) ~ 2 μg
11,1 μL
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
2 μL
Restriction enzyme 2 (1 U/μL)
2 μL
Nuclease free water
Fill to 50 μL
Table 4: Digestion scheme 4
 
DNA (K352002 69,77 ng/μl) ~ 2 μg
28,7 μL
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
2 μL
Restriction enzyme 2 (1 U/μL)
2 μL
Nuclease free water
Fill to 50 μL
Table 5: Digestion scheme 5
 
DNA (K173003 217,06 ng/μL) ~ 2 μg
9,2 μL
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
2 μL
Restriction enzyme 2 (1 U/μL)
2 μL
Nuclease free water
Fill to 50 μL
Table 6: Digestion scheme 6
 
DNA (I13453 88,18 ng/μL) ~ 2 μg
22,7 μL
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
1 μL
Restriction enzyme 2 (1 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 7: Digestion scheme 7
 
DNA (K133071 186,79 ng/μL) ~ 2 μg
10,7 μL
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
1 μL
Restriction enzyme 2 (1 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 8: Digestion scheme 8
 
DNA (pSB1K3 163,93 ng/μL) ~ 2 μg
 12,2 μL
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
2 μL
Restriction enzyme 2 (1 U/μL)
2 μL
Nuclease free water
Fill to 50 μL
Table 9: Digestion scheme 9
 
DNA (K173013 345,1 ng/μl) ~ 2 μg
 5,8 μL
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
1 μL
Restriction enzyme 2 (1 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 10: Digestion scheme 10
DNA (K133116 194,1 ng/μl) ~ 2 μg
10,3 μL
Cutsmart (10x)
5 μL
Restriction enzyme 1 (1 U/μL)
1 μL
Restriction enzyme 2 (1 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
Table 11: Digestion scheme 11
 
Biobrick
Restriction-enzyme 1
Restriction enzyme 2
K173003
XbaI
PstI-HF
K173013
XbaI
PstI-HF
K133071
XbaI
PstI-HF
K133116
XbaI
PstI-HF
J23100
EcoRI-HF
SpeI-HF
I13453
EcoRI-HF
SpeI-HF
K352002
EcoRI-HF
SpeI-HF
K352003
EcoRI-HF
SpeI-HF
pSB1K3
EcoRI-HF
PstI-HF
Table 12: Digestion scheme used enzymes
Gelelectrophoresis
Number
What
Amount
1.
Ladder
5 μL
2.
J23100 EcoRI-HF+ SPeI-HF
5 μL
3.
J23100 EcoRI-HF
5 μL
4.
J23100 SpeI-HF
5 μL
5.
pSB1K3 EcoRI-HF + PstI-HF
5 μL
6
K173013 XbaI + PstI-HF
5 μL
7.
K173003 XbaI + PstI-HF
5 μL
8.
K133071 XbaI + PstI-HF
5 μL
9.
K133116 XbaI + PstI-HF
5 μL
10.
K352002 EcoRI-HF + SpeI-HF
5 μL
11.
K352003 EcoRI-HF + SpeI-HF
5 μL
12.
I13453 EcoRI-HF + SpeI-HF
5 μL
13.
Ladder 
5 μL
14.
Ladder
10 μL
Table 13: Gelelectrophoresis order of digestions
dephosphorylation
 
Volume
Compound
12,5 μL
Vector DNA (0,5 μg)
2 μL
phosphatase buffer (10x)
4,5 μL
milli Q
1 μL
phosphatase (1 U/μL)
20 μL
Total
Table 14: Pipetting scheme dephosphorylation
 
Volume
Compound
37,5 μL
Vector DNA (0,5 μg)
6 μL
phosphatase buffer (10x)
13,5 μL
milli Q
3μL
phosphatase (1 U/μL)
60 μL
Total
Table 15: Pipetting scheme dephosphorylation
Pre-Ligationmix:
Volume
Compound
7,5 μL
Insert DNA (2 μg/50 μL) 300 ng
7,5 μL
Insert DNA (2 μg/50 μL) 300 ng
4 μL
Backbone (0,5 μg/ 20 μL) 100 ng
5 μL
Delutionbuffer (5x)
1 μL
Milli Q
25 μL
Total
Table 16: Pre-ligationmix pipette scheme
Ligationmix:
Volume
Compound
10 μL
Pre-ligationmix
2 μL
10x T4 ligationbuffer
1 μL
T4 DNA ligase (1U/μL)
8 μL
Milli Q
20 μL
Total
Table 17: Ligationmix pipette scheme
Ligation started at 15.45 (31-08-18) at room temperature.
end ligation around 11.00 (01-09-18)
Store at -20 degrees Celcius.
 
10
Assessing Gas Production BioBricks in E.Coli
Show more
August 31
Assessing Gas Production BioBricks in E.Coli
September 3
×
Elise Grootscholten | Randall de Waard
Transformations of the biobrick combinations in NEB10bèta. The transformations are plated on kanamycine agar plates, because all the ligations were done in pSB1K3 (kanamycine resistence) backbone. Pink and with colonies appeared after incubation by 37 degrees Celcius for about 12h. The white colonies will be used for further experiments.
Experiment 11: transformations of the biobrick ombinations in NEB10bèta.
03-09-18
Materials
See materials experiment 5.
-Kanamycin (40 mg/mL)
-Ampicillin (50 mg/mL)
Methods
See methods experiment 5
Make 19 agar plates with kanamycine (Kana) (17,5 ul/20mL) and 1 agar plate with ampicillin (Amp) (40 ul/mL). Plate all the transformations on agar plates with Kana , because all the ligations are done in the pSB1K3 (Kana resistence) backbone. 
Devide one Amp plate and one Kana plate in about 30 parts. Because the biobrick combinations with biobrick K133116 have originally Amp and Kana resistence, the right backbone plasmid (with hopefully the right ligation) is only growing on the agar plate with Kana.
Results
There is grown n every transformation plates, so the transformations are done well.
There are pink and white colonies present. Only the white colonies will be used for further experiments.
 
11
Assessing Gas Production BioBricks in E.Coli
Show more
September 3
Soldering PCB [TC]
September 4
×
Tom van Dijk | Mike Vrieswijk
Soldered the first temperature controller with the help of a sodlering stencil and reflow oven. All components were soldered onto the first temperature controller.
01
Soldering PCB [TC]
Show more
September 4
Soldering PCB [TC]
September 4
×
Tom van Dijk | Mike Vrieswijk
Soldered second temperature controller with the help of a sodlering stencil and reflow oven. All components were soldered onto the first temperature controller.
02
Soldering PCB [TC]
Show more
September 4
Assessing Gas Production BioBricks in E.Coli
September 6
×
Elise Grootscholten | Randall de Waard
Plasmid DNA was isolated of 20 different colonies. The first time something went wrong. The second time we had good concentrations of isolated plasmid DNA. After this we will digest the DNA to see if the plasmids all have the required biobricks. The tube have a code from now on, see table 1.
 
Number
Promotor
Gene
Backbone
A
K352002 (CooF)
K173013
pSB1K3
B
K352002 (CooF)
K173003
pSB1K3
C
K352002 (CooF)
K133071
pSB1K3
D
K352003 (CooM)
K133071
pSB1K3
E
K352003 (CooM)
K173003
pSB1K3
F
K352003 (CooM)
K173013
pSB1K3
H
I13453
K173013
pSB1K3
I
I13453
K173003
pSB1K3
J
I13453
K133071
pSB1K3
Table 1: list of biobricks of abbreviations
Experiment 12: Overnight cultures and minipreps
05-09-18/06-09-18
Materials
See materials experiment 3
Methods
See methods experiment 3
Number
Promotor
Gene
Backbone
A
K352002 (CooF)
K173013
pSB1K3
B
K352002 (CooF)
K173003
pSB1K3
C
K352002 (CooF)
K133071
pSB1K3
D
K352003 (CooM)
K133071
pSB1K3
E
K352003 (CooM)
K173003
pSB1K3
F
K352003 (CooM)
K173013
pSB1K3
G
B0032
J23105
pSB1A3
H
I13453
K173013
pSB1K3
I
I13453
K173003
pSB1K3
J
I13453
K133071
pSB1K3
Table 1: list of biobricks of abbreviations
 
Figure 1: Nanodrop results after DNA isolation. The biobrick combinations can be found in the methods (experiment 12)
12
Assessing Gas Production BioBricks in E.Coli
Show more
September 6
Assessing CooA Production
September 6
×
Dustin van der Meulen
After transforming the ligation product of J23105+B0032 and K352001+B0015 we had some growth of none red/pink colonies. Overnight cultures where made, and the DNA was prepped later on, checking them on gel.
Today a miniprep of pSB1K3 + J23100 + B0032 was performed. This gave concentrations of 51,3 and 89 ng/μL.
Of this, together with pSB1K3+ K352001 + B0015 and pSB1A3 a digest was performed:
 
Biobricks
Enzyme 1
Enzyme 2
1K3+J23+B32
EcoRI
SpeI
1K3+K35+B15
XbaI
PstI
pSB1A3
EcoRI
PstI
After two hours the digest was put on gel, though the rest was left for another two hours.
 
WhatsApp Image 2018-09-07 at 15.28.47.jpeg
 
 
CooA experiment 14
14
Assessing CooA Production
Show more
September 6
Assessing Gas Production BioBricks in E.Coli
September 10
×
Elise Grootscholten | Randall de Waard
The minipreps of experiment 12 are digested with the restriction enzymes: SmaI and ScaI. Only Sca and Sma are both incubated at 37 degrees Celcius. Sma has to be incubated at 25 degrees Celcius. This is done in experiment 15.
Experiment 12: Digestions of the minipreps
10-09-18
Materials
-SmaI (restriction-enzyme)
-ScaI (restriction-enzyme)
See materials experiment 10
Methods
See methods experiment 10
 
DNA (A1 396.7 ng/μl) ~ 2 μg
5.0 μL
Cutsmart (10x)
5 μL
Sca1-HF (2 U/μL)
1 μL
Sma1 (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
Table 1: Digestion scheme 1
 
DNA (B1 15.65 ng/μl) ~ 2 μg
127.8
Cutsmart (10x)
5 μL
Sca1-HF (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
Table 2: Digestion scheme 2
 
DNA (C1 186.6 ng/μl) ~ 2 μg
10.7 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 3: Digestion scheme 3
 
DNA (D1 79.0    ng/μl) ~ 2 μg
25.3 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 4: Digestion scheme 4
 
DNA (E1 97.7 ng/μl) ~ 2 μg
20.7 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 5: Digestion scheme 5
 
DNA (F1 106.7 ng/μl) ~ 2 μg
18.7 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 6: Digestion scheme 6
 
DNA (G1 328.9 ng/μl) ~ 2 μg
6.1 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 7: Digestion scheme 7
 
DNA (H1 121.0 ng/μl) ~ 2 μg
16.5 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 8: Digestion scheme 8
 
DNA (I1 1129.5 ng/μl) ~ 2 μg
 1.7 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 9: Digestion scheme 9
 
DNA (J1 1465.7 ng/μl) ~ 2 μg
 1.4 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 10: Digestion scheme 10
 
DNA (A2 310.4 ng/μl) ~ 2 μg
6.4 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 11: Digestion scheme 11
 
DNA (B2 364.9l) ~ 2 μg
5.5 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 12: Digestion scheme 12
 
DNA (C2 228.3 ng/μl) ~ 2 μg
8.8 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 13: Digestion scheme 13
 
DNA (D2 701.7 ng/μl) ~ 2 μg
2.9 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 14: Digestion scheme 14
 
DNA (E2 119.8 ng/μl) ~ 2 μg
16.7 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 15: Digestion scheme 15
 
DNA (F2 493.3 ng/μl) ~ 2 μg
4.1 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 16: Digestion scheme 16
 
DNA (G2 211.7 ng/μl) ~ 2 μg
9.5 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 17: Digestion scheme 17
DNA (H2 482.2 ng/μl) ~ 2 μg
4.2 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 18: Digestion scheme 18
 
DNA (I2 1364.7 ng/μl) ~ 2 μg
1.5 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
 Table 19: Digestion scheme 19
 
DNA (J2 1665.5 ng/μl) ~ 2 μg1.2
 μL
Cutsmart (10x)
5 μL
Sca1-HF 1 (2 U/μL)
1 μL
Sma1  (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
Table 20: Digestion scheme 20
Results
The results of the digestions can be found in figure 1 and 2 (Images experiment 13)
Ladder, A1, B1, C1, D1, E1, F1, G1, H1, I1, (J1), Ladder, A2, B2, C2, D2, E2, F2, G2, Ladder
Ladder, H2, I2, J2, Ladder, K3, J1, ladder
Something went wrong with the first (J1), so look at the second one.
Conclusion/discussion
Restriction-enzyme Sma was incubated at 37 degrees Celcius (same as Sca). This must be 25 degrees Celcius.
Therefore, Sma was added again and was incubated at 25 degrees Celcius. See results experiment 16.
Figure 1: Digestions with ScaI and SmaI
Figure 2: Digestions with ScaI and SmaI
13
Assessing Gas Production BioBricks in E.Coli
Show more
September 10
Soldering PCB [TC]
September 11
×
Tom van Dijk | Mike Vrieswijk
Testing first temperaturecontroller. Reworking surton components due to shorts on the PCB. Rework succesfull.
03
Soldering PCB [TC]
Show more
September 11
Soldering PCB [TC]
September 11
×
Tom van Dijk | Mike Vrieswijk
Testing second temperaturecontroller. Reworking surton components due to shorts on the PCB. Rework failed.
04
Soldering PCB [TC]
Show more
September 11
Assessing Gas Production BioBricks in E.Coli
September 13
×
Elise Grootscholten | Randall de Waard
Minipreps of the biobrick combinations: K352002+K133116+pSB1K3 (1) and K352003+K133116+pSB1K3 (2) are done. The DNA concentration is for 1:189,3 ng/μl and for 2: 355,3 ng/μl.
Experiment 14: Minipreps
13-09-18
Minipreps of the biobrick combinations: K352002+K133116+pSB1K3 and K352003+K133116+pSB1K3
Materials
- Mini prep, Plasmid Purification Kit, Machery Nagel
- Eppendorf cups
- Variable Volume Pipets
- sterile pipet points
-Chloramphenicol (40 mg/mL)
Methods
Machery Nagel Plasmid Purification
See experiment 3
Results
K352002+K133116+pSB1K3: 189,3 ng/μl
K352003+K133116+pSB1K3: 355,3 ng/μl
Figure 1: Minipreps 13-09-18
14
Assessing Gas Production BioBricks in E.Coli
Show more
September 13
Assessing Gas Production BioBricks in E.Coli
September 13
×
Elise Grootscholten | Randall de Waard
Digestions have been done of the biobrickcombinations K352002 +K133116 + pSB1K3 and K352003 + K133116 + pSB1K3 with restriction-enzymes ScaI and SmaI
Experiment 14: Digestions
13-09-18
Digestions of the biobrick combinations: K352002+K133116+pSB1K3 and K352003+K133116+pSB1K3
Materials
-SmaI (restriction-enzyme)
-ScaI (restricion-enzyme)
See experiment..
Methods
1 μL SmaI restriction-enzyme has been added to the digetions from experiment 13, except for pSB1K3.
After an our at room temperature the digestions were put in -20 degrees Celcius.
 
DNA (K352002+K133116+pSB1K3 189,3 ng/μl) ~ 2 μg
32,4 μL
Cutsmart (10x)
5 μL
Sca1-HF (2 U/μL)
1 μL
Sma1 (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
Table 1: Digestion scheme 1
DNA (K352003+K133116_pSB1K3 355,3 ng/μl) ~ 2 μg
5,6 μL
Cutsmart (10x)
5 μL
Sca1-HF (2 U/μL)
1 μL
Sma1 (2 U/μL)
1 μL
Nuclease free water
Fill to 50 μL
Table 2: Digestion scheme 2
 
 
15
Assessing Gas Production BioBricks in E.Coli
Show more
September 13
Assessing Gas Production BioBricks in E.Coli
September 14
×
Elise Grootscholten | Randall de Waard
All the digestions of experiment 13 have digested again with SmaI for an our at 25 degrees Celcius. Because earlier the digestions have been put immediately bt 37 degrees Celcius. It looks like H1, J1 and D2 include the right biobricks. 
Experiment 16: Gelelectroforesis
14-09-18
Materials
-Loading dye (6x)
-2 Log DNA ladder
-Digested DNA
-1% arabinose gel (1 g/100 mL TAE buffer)
Methods
Put 1 gram of arabinose with 100 mL TAE buffer.
Heat it in the microwave until the solution becomes clear.
Add 100 μL gelred when the solution is hand warm.
Pour the solution into the gel holder which you taped before.
Wait until the gel is solid then you can pipette your dna with Loading dye into the wells.
L: 3 μL
DNA: 2 μL loading dye + 5 μL DNA, 5 μL on gel.
Gel 1
L, A1, B1, C1, D1, E1, F1, G1, G1 with only smaI, H1, I1, J1, A2, B2, C2, D2, E2, F2, G2, L
Gel 2
L, H2, I2, J2, L, K352002 + K133116 + pSB1K3,  K352003 + K133116 + pSB1K3
Figure 1: Gel 1 (14-09-18), digested with ScaI and SmaI.
Figure 2: Gel 2, H2, I2, J2, L, K352002+K133116+pSB1K3, K352003+K133116+pSB1K3
16
Assessing Gas Production BioBricks in E.Coli
Show more
September 14
Testing gas production
September 17
×
Elise Grootscholten | Paul Reusink
Different strains of E.coli are tested as negative control. Those strains were: NEB10Bèta, BL21 (DE3), BL21, HB101, DH5alpha and JM109. NEB10Bèta produced the most gas. BL21, HB101 and JM101 produced none/almost none gas. Because BL21 isn't a K12 strain we can not use that one. That is why we will test further gasproduction in HB101 and JM109. And maybe those will be our final E.coli strains.
Experiment 5: Testing different strains of E.coli
Materials
All the following types of bacteria are E.coli:
-glycerolstock BL21
-glycerolstock HB101
-glycerolstock DH5alpha
-glycerolstock JM109
-agarplate with NEB10beta
-agarplat with BL21 (DE3) 
Methods
See experiment 1
Make overnight cultures of the following E.coli tribes: NEB10Bèta, BL21 (DE3), BL21, HB101, DH5alpha, JM109.
All the bacteria are de deluted in normal LB agar, LB agar with urea and LB agar with pyruvate. 
Results
 
Bacteria
LB
LB
Urea
Urea
Pyruvate
Pyruvate
NEB10beta
+
+++
+++
+++
+
++/+++
BL21 (DE3) 
-
++
-
-
-
-
BL21
-
-
-
-
-
-
DH5a
+++
+/-
-
-
+/-
+
HB101
-
-
-
-
+
-
JM109
-
-
-
-
-
-
05
Testing gas production
Show more
September 17
Designing peristaltic pump [PP]
September 18
×
Tom van Dijk | Mike Vrieswijk
Made a 3d-model of the parts for the peristaltic pump. Parts include: multiple gears, bearings, and casing.
01
Designing peristaltic pump [PP]
Show more
September 18
3d-printing peristaltic pump [PP]
September 18
×
Tom van Dijk | Mike Vrieswijk
3d-printed all parts of the peristaltic pump model.
02
3d-printing peristaltic pump [PP]
Show more
September 18
Animating peristaltic pump [PP]
September 18
×
Tom van Dijk | Mike Vrieswijk
An animation was made of the first peristaltic pump.
03
Animating peristaltic pump [PP]
Show more
September 18
Assessing CooA Production
September 20
×
Dustin van der Meulen
This week we peformed another cycle of cloning, starting with digesting J23105+B0032 with EcoRI and SpeI and K352001+B0015 with XbaI and PstI, checking these on gel (which seemed to be correct) and ligating them together on the pSB1A3 version of BBa_J04450. This was then grown overnight and prepped the following day.
Today a dephos and ligation has been performd using the rAPId dephos and ligation kit and protocol.
For the dehpos a total of 17 μL of pSB1A3 wasused, for an end concentration of 17 ng/μL
For the ligationthe endvolume was trippled as to have enough DNA.
For both inserts (K352001 & B0015 | J23100 + B0032) 7.5 μL was used (150 ng of DNA foreach insert)
For the veccor (pSB1A3) 3 μL was used (51 ng of DNA)
The ligation will be left overnigh (started at 15:30).
 
15
Assessing CooA Production
Show more
September 20
Assessing Gas Production BioBricks in E.Coli
September 24
×
Elise Grootscholten | Randall de Waard
32 minipreps have been performed. A lot of them have a low concentration nucleic acid and/or have a high 260/280 and 260/230 rate. Therefore those will be done again.
Experiment 17: Minipreps
20-09-18
Overnight cultures (35)
21-09-18
Minipreps (32)
Materials 
See experiment 3
-Kanamycin (40 mg/mL)
Methods
See experiment 3
Make 32 minipreps:
A: 6
B: 6
C:6
D:1
E:6
F:6
H:1
I: 1
J:2
Results/conclusion
There was no grow in B6, C2 and C5.
A lot of them have a low concentration nucleic acid and/or have a high 260/280 and 260/230 rate. Therefore those will be done again.
Thise are: A1, A4, A5, A6, B1, B5, B6 C1, C2, C3, C4, C5, C6, E1, E2, E5, E6, F4, F5
Figure 1: Minipreps, a total of (32) because number 5, 7 and 12 are wrong.
17
Assessing Gas Production BioBricks in E.Coli
Show more
September 24
Integration of silicon hose and peristaltic pump [PP]
September 25
×
Tom van Dijk | Mike Vrieswijk
Due to ordering issues a different silicon hose was implemented in the design. The new silicon tube was integrated into the 3d-printed peristaltic pump. A test was done with the new silicon tube in place. The test cocluded that the tube was not compatible with this peristaltic pump design. A new design must be made.
01
Integration of silicon hose and peristaltic pump [PP]
Show more
September 25
Designing peristaltic pump v2 [PP]
September 25
×
Tom van Dijk | Mike Vrieswijk
Peristaltic pump was redesigned and the model has been updated.
02
Designing peristaltic pump v2 [PP]
Show more
September 25
3d-printing peristaltic pump v2 [PP]
September 25
×
Tom van Dijk | Mike Vrieswijk
3d-printed all parts of the peristaltic pump model v2. All parts have been put together succefully.
03
3d-printing peristaltic pump v2 [PP]
Show more
September 25
Testing peristaltic pump v2 [PP]
September 25
×
Tom van Dijk | Mike Vrieswijk
The silicon tube was intergrated with the 3e-printed peristaltic pump v2 and tested. The test concluded that the motor had insufficient force. A new design must be made.
04
Testing peristaltic pump v2 [PP]
Show more
September 25
Designing peristaltic pump v3 [PP]
September 26
×
Tom van Dijk | Mike Vrieswijk
An editional planetary gearset was added to the first planetary gearset of the peristaltic pump v2 model to increase the power output by 4x.
01
Designing peristaltic pump v3 [PP]
Show more
September 26
3d-printing peristaltic pump v3 [PP]
September 26
×
Tom van Dijk | Mike Vrieswijk
3d-printed all parts of the peristaltic pump model v3. All parts have been put together succefully.
02
3d-printing peristaltic pump v3 [PP]
Show more
September 26
Testing peristaltic pump v3 [PP]
September 26
×
Tom van Dijk | Mike Vrieswijk
The newly pot together peristalticpump model v3 was tested. The test concluded a flaw in the design of the peristaltic pump v3. Failure: the silicon tube was pulled through the pump. A new design must be made.
03
Testing peristaltic pump v3 [PP]
Show more
September 26
Assessing CooA Production
September 26
×
Dustin van der Meulen
Digesting the CooA combinations with NotI we could check if the requiered biobricks where present in our cells. Out of 10 checked colonies, 3 appeared to be the right size. Because of an error the on plate location of these colonies was lost however, so another check will have to be performed.
Pipet 8 digestions
Nr. 1, 2, 4, 5, 7, 8, 9, and 10 of CooA. Use 5 μL of digest with 3 μL of loading dye.
Gel:
Ladder 5 ul 1, 2, 4, 5, 7, 8, 9, 10, pSB1K3 (Sca), G1 (Sma), Ladder 5 ul, ladder 3 ul
The CooA digestions are 5 ul
 
Gel CooA.jpg
 
 
CooA 18-09
16
Assessing CooA Production
Show more
September 26
Designing Test tube shaker [TTS]
September 27
×
Tom van Dijk | Mike Vrieswijk
A test tube shaker was designed to fit on top of a NEMU17 stepper motor. A spot for end switch was put in place to calibrate the test tube shaker. A cap for the test tube was designed.
01
Designing Test tube shaker [TTS]
Show more
September 27
3d-printing Test tube shaker [TTS]
September 27
×
Tom van Dijk | Mike Vrieswijk
The test tube shaker was 3d-printed and assembled. Nine caps were printed.
02
3d-printing Test tube shaker [TTS]
Show more
September 27
Testing Test tube shaker [TTS]
September 27
×
Tom van Dijk | Mike Vrieswijk
The newly assembled testtube shaker was tested for sufficient and controlled shakage. The test was deemed a succes. The calibration for the test tube shaker potition was tested and deemed in working order. The caps were tested for sufficient air permeability. The test concluded suficient air permeability.
03
Testing Test tube shaker [TTS]
Show more
September 27
Assessing Gas Production BioBricks in E.Coli
September 27
×
Elise Grootscholten | Randall de Waard
Making overnight cultures of the numbers: A1, A4, A5, A6, B1, B5, B6 C1, C2, C3, C4, C5, C6, E1, E2, E5, E6, F4, F5 with Kanamycin. These biobrick combinations can be found in experiment 12. Also the first digestions with XbaI, Eco0109I, HindIII and SspI-HF have been done.
Experiment 19: Making overnight cultures + first digestions
27-09-18
Materials
-Kanamycine 
-XbaI
-SspI-HF
-Eco019I
-HindIII
Methods
Make new overnight cultures of the numbers: A1, A4, A5, A6, B1, B5, B6 C1, C2, C3, C4, C5, C6, E1, E2, E5, E6, F4, F5 with Kanamycin. 
 
19
Assessing Gas Production BioBricks in E.Coli
Show more
September 27
Assessing Gas Production BioBricks in E.Coli
September 27
×
Elise Grootscholten | Randall de Waard
Minipreps have been made of the numbers: A1, A4, A5, A6, B1, B5, B6 C1, C2, C3, C4, C5, C6, E1, E2, E5, E6, F4, F5.
Experiment 20: Minipreps of gasproduction biobricks
27-09-18
Make minipreps of A1, A4, A5, A6, B1, B5, B6 C1, C2, C3, C4, C5, C6, E1, E2, E5, E6, F4, F5.
See experiment 3.
Also make master mixes for the control digestions and code the tubes.
 
20
Assessing Gas Production BioBricks in E.Coli
Show more
September 27
Designing peristaltic pump v4 [PP]
September 28
×
Tom van Dijk | Mike Vrieswijk
The innershape of the tube duct was rounded to increase friction on the tube. The model of the peristaltic pump v3 was updated.
01
Designing peristaltic pump v4 [PP]
Show more
September 28
3d-printing peristaltic pump v4 [PP]
September 28
×
Tom van Dijk | Mike Vrieswijk
The new parts for the tubeduct were 3d-printed and put together with the peritstaltic pump.
02
3d-printing peristaltic pump v4 [PP]
Show more
September 28
Testing peristaltic pump v4 [PP]
September 28
×
Tom van Dijk | Mike Vrieswijk
The new peristaltic pump v4 was tested. The test concluded a large improvement over the previous generation, but was deemed unreliable due tothe tube occasional being pulled through the pump. A new design must be made.
03
Testing peristaltic pump v4 [PP]
Show more
September 28
Designing peristaltic pump v4.1 [PP]
September 28
×
Tom van Dijk | Mike Vrieswijk
A clamp was designed to hold the tube into the peristaltic pump from both ends. the peristaltic pump model v4 was updated.
04
Designing peristaltic pump v4.1 [PP]
Show more
September 28
3d-printing peristaltic pump v4.1 [PP]
September 28
×
Tom van Dijk | Mike Vrieswijk
The new clamp was 3d-printed, put together and added to the epristaltic pump.
05
3d-printing peristaltic pump v4.1 [PP]
Show more
September 28
Testing peristaltic pump v4.1 [PP]
September 28
×
Tom van Dijk | Mike Vrieswijk
The new peristaltic pump v4.1 was tested. The test conclude no noticable difference to the behaviour of the tube. The tube was still occsionally pulled through the pump. A new design must be made.
06
Testing peristaltic pump v4.1 [PP]
Show more
September 28
Designing peristaltic pump v5 [PP]
September 30
×
Tom van Dijk | Mike Vrieswijk
The tube clamp was attached to the top piece of the model's casing. The model of the persitaltic pump v4.1 was updated.
01
Designing peristaltic pump v5 [PP]
Show more
September 30
3d-printing peristaltic pump v5 [PP]
September 30
×
Tom van Dijk | Mike Vrieswijk
The new integrated tube clamp was 3d-printed and added to the peristaltic pump.
02
3d-printing peristaltic pump v5 [PP]
Show more
September 30
Testing peristaltic pump v5 [PP]
September 30
×
Tom van Dijk | Mike Vrieswijk
The new peristaltic pump v5 was tested.  The new v5 was deemed a succes. No previous bad behaviour was observed.
03
Testing peristaltic pump v5 [PP]
Show more
September 30
Assessing CooA Production
October 1
×
Dustin van der Meulen
Today a colony PCR was performed of multiple CooA ligation products of J23100+B0032+K352001+B0015 on pSB1A3. This specific colony was later PCRed again with the gasproduction biobricks in preperation for sequencing. Only one of the colonies seemed to be the right length, though te lenght of the complete insert and BBa_J04450 is quite similair, so sequencing later this week will provide the final answer.
 
CooA 4-10
17
Assessing CooA Production
Show more
October 1
Assessing Gas Production BioBricks in E.Coli
October 4
×
Elise Grootscholten | Randall de Waard
From 27-09 until 04-10 more than 200 digestions and 17 gelelectrophoresis have been performed. This was done as a control for our composite parts. We digested with XbaI, Eco01019 and SspI-HF/HindIII. Every restriction-enzyme has an unique place to cut in our constructs. With this we could test if the promoter, backbone and gas production gene was present. However, the digestions didn't show us very good results. Only I1 and J1 looked good. After this we started with PCR.
Experiment 21: Digestions and gelelectrophoresis
01-10-18
Digestions of the gasproduction biobrick combinations (D, H, I and J) and a start with the gelelectrophoresis.
Methods
Digestions of D2, H1, I2 and J2 of experiment 12.
J2 will from now on be J3.
NEB 2.1 (10x)
4 μL
Eco0109I (20 U/μL)
1 μL
Nuclease free water
35 μL
Table 1: Master mix digestion 1
 
NEB 2.1 (10x)
4 μL
Eco0109I (20 U/μL)
1 μL
XbaI (20 U/μL)
1 μL
Nuclease free water
34 μL
 
Table 2: Master mix digestion 2
 
NEB 2.1 (10x)
2 μL
Eco0109I (20 U/μL)
1 μL
XbaI (20 U/μL)
1 μL
HindIII (20 U/μL)
1 μL
Nuclease free water
15 μL
Table 3: Master mix digestion 3
 
NEB 2.1 (10x)
2 μL
Eco0109I (20 U/μL)
1 μL
XbaI (20 U/μL)
1 μL
SspI-HF
1 μL
Nuclease free water
15  μL
Table 4: Master mix digestion 4
 
NEB 2.1 (10x)
2 μL
HindIII (20 U/μL)
1 μL
XbaI (20 U/μL)
1 μL
Nuclease free water
16 μL
 
Table 5: Master mix digestion 5
 
NEB 2.1 (10x)
2 μL
SspI-HF (20 U/μL)
1 μL
XbaI (20 U/μL)
1 μL
Nuclease free water
16 μL
 
Table 6: Master mix digestion 6
 
NEB 2.1 (10x)
4 μL
XbaI (20 U/μL)
1 μL
Nuclease free water
35 μL
 
Table 7: Master mix digestion 7
 
NEB 2.1 (10x)
2 μL
HindIII (20 U/μL)
1 μL
Nuclease free water
17 μL
 
Table 8: Master mix digestion 8
 
NEB 2.1 (10x)
2 μL
SspI-HF (20 U/μL)
1 μL
Nuclease free water
17 μL
 
Table 9: Master mix digestion 9
 
NEB 2.1 (10x)
2 μL
Eco0109I (20 U/μL)
1 μL
HindIII (20 U/μL)
1 μL
Nuclease free water
16 μL
 
Table 10: Master mix digestion 10
 
NEB 2.1 (10x)
2 μL
Eco0109I (20 U/μL)
1 μL
SspI-HF (20 U/μL)
1 μL
Nuclease free water
16 μL
 
Table 11: Master mix digestion 11
Pipette 9 μL of the mastermix with 2 μL of H1/ 1 μL of D2/ 1 μL of I2/ 1 μL of J23
D2 and J3 are digested with mastermix: 1, 2, 3, 5, 7, 8 and 10.
H1 and I2 are digested with mastermix: 1, 2, 4, 6, 7, 9, and 11.
L= 2 log DNA ladder
 
Gelelectrophoresis
01-10-18
Gel 1
L, B6 Eco01090I, B6 SspI-HF, B6 XbaI, B6 Eco0109I + SspI-HF, B6 Eco0109I + XbaI, B6 SspI-HF + XbaI, B6 Eco0109I + SspI-HF + XbaI, L, B2 SspI-HF, B2 XbaI, B2 Eco0109I + SspI-HF, B2 Eco0109I + XbaI, B2 SspI-HF + XbaI, B2 Eco0109I + SspI-HF + XbaI, L, H1 SspI-HF, H1 XbaI, H1Eco0109I + SspI-HF, H1 Eco0109I + XbaI
Gel 2
L, H1 SspI-HF + XbaI, H1 Eco0109I + SspI-HF + XbaI, I1 Eco0109I + SspI-HF, I1 Eco0109I + XbaI, I1 SspI-HF + XbaI, I1 Eco0109I + SspI-HF + XbaI, L, I1 SspI-HF, I1 XbaI, B3 SspI-HF, B3 XbaI, B3 Eco0109I + SspI-HF, B3 Eco0109I + XbaI, B3 SspI-HF + XbaI, B4 SspI-HF, B4 XbaI, B4 Eco0109I + SspI-HF, B4 Eco0109I + XbaI, B4 Eco0109I + SspI-HF + XbaI
Gel 3
L, F6 SspI-HF, F6 XbaI, F6 Eco0109I + SspI-HF, F6 Eco0109I + XbaI, F6 SspI-HF + XbaI, F6 Eco0109I + SspI-HF + XbaI, L, F5 SspI-HF, F5 XbaI, F5 Eco0109I + SspI-HF, F5 Eco0109I + XbaI, F5 SspI-HF + XbaI, F5 Eco0109I + SspI-HF + XbaI, L F4 XbaI, F4 Eco0109I + SspI-HF, F4 Eco0109I + XbaI, F4 sspI-HF + XbaI, L
Gel 4
L, F4 Eco0109 + XbaI + SspI-HF, F4 Eco0109, F3 XbaI, F3 Eco0109 + SspI-HF, F3 Eco0109 + XbaI, F3 SspI-HF + XbaI, F3 Eco0109 + XbaI + SspI-HF, F2 SspI-HF, F2 XbaI, L, F2 Eco0109 + SspI-HF, F2 Eco0109 + XbaI, F2 SspI-HF + XbaI, F2 Eco01090 + XbaI + SspI-HF, F1 XbaI, F1 Eco0109 + SspI-HF, F1 Eco0109 + XbaI, F1 SspI-HF + XbaI
Gel 5
L, F1 Eco0109 + XbaI + SspI-HF, E1 Eco0109, E1 SspI-HF, E1 XbaI, E1 Eco0109 + SspI-HF, E1 Eco0109 + XbaI, E1 SspI-HF + XbaI, E1 Eco0109 + XbaI + SspI-HF, L, E2 Eco0109, E2 SspI-HF, E1 XbaI, E2 Eco0109 + SspI-HF, E2 Eco0109 + XbaI, E2 SspI-HF + XbaI, E2 Eco0109 + XbaI + SspI-HF, L
Gel 6
L, E3 SspI-HF, E3 XbaI, E3 Eco0109 + SspI-HF, E3 Eco0109 + XbaI, E3 SspI-HF + XbaI, E3 Eco0109 + XbaI + SspI-HF, L, E6 Eco0109, E6 SspI-HF, E6 XbaI, E6 Eco0109 + SspI-HF, E6 Eco0109 + XbaI, E6 SspI-HF + XbaI, E6 Eco0109 + XbaI + SspI-HF, L
02-10-18
Gelelectrophoresis
Gel 1
L,- , A1 Eco0109 , A1 SspI-HF, A1 XbaI, A1 Eco0109 + SspI-HF, A1 Eco01019 + XbaI, A1 SspI-HF + XbaI, A1 Eco0109 + XbaI + SspI-HF, -, -, A2 Eco0109 , A2 SspI-HF, A2 XbaI, A2 Eco0109 + SspI-HF, A2 Eco01019 + XbaI, A2 SspI-HF + XbaI, A2 Eco0109 + XbaI + SspI-HF, -, L
Gel 2
L,- , A3 Eco0109 , A3 SspI-HF, A3 XbaI, A3 Eco0109 + SspI-HF, A3 Eco01019 + XbaI, A3 SspI-HF + XbaI, A3 Eco0109 + XbaI + SspI-HF, -, -, A4 Eco0109 , A4 SspI-HF, A4 XbaI, A4 Eco0109 + SspI-HF, A4 Eco01019 + XbaI, A4 SspI-HF + XbaI, A4 Eco0109 + XbaI + SspI-HF, -, L
03-10-18
Gelelectrophoresis
Gel 1
L, D2 Eco0109, D2 Eco0109 + XbaI, D2 Eco0109 +XbaI + HindIII, D2 HindII + XbaI, D2 XbaI, D2 HindIII, D2 Eco0109 + HindIII, D not digested, L, J3 Eco0109, J3 Eco0109 + XbaI, J3 Eco0109 + XbaI + HindIII, J3 XbaI, J3 HindIII, J3 Eco0109 + HindIII, J3.10, J not digested, L.
Gel 2
L, H2 Eco0109, H2 Eco0109 + XbaI, H2 Eco0109 + XbaI + SspI-HF, H2 SspI-HF + XbaI, H2 XbaI, H2 SspI-HF, H2 Eco0109 + SspI-HF, H2 not digested, L, I2 Eco0109, I2 Eco0109 + XbaI, I2 Eco0109 + XbaI + SspI-HF, I2 SspI-HF + XbaI, I2 XbaI, I2 SspI-HF, I2 Eco0109 +SspI-HF, I2 not digested, L
Gel 3
L, C1 Eco0109, C1 HindIII, C1 XbaI, C1 Eco0109 + XbaI, C1 Eco0109 + HindIII, C1 HindIII + XbaI, C1 Eco0109 + XbaI + HindIII, C1 not digested, L, C2 Eco0109, C2 HindIII, C2 XbaI, C2 Eco0109 + XbaI, C2 Eco0109 + HindIII, C2 HindIII + XbaI, C2 Eco0109 + XbaI + HindIII, C2 not dgested, L
Gel 4
L, C3 Eco0109, C3 HindIII, C3 XbaI, C3 Eco0109 + XbaI, C3 Eco0109 + HindIII, C3 HindIII + XbaI, C3 Eco0109 + XbaI + HindIII, C3 not digested, L, C4 Eco0109, C4 HindIII, C4 XbaI, C4 Eco0109 + XbaI, C4 Eco0109 + HindIII, C4 HindIII + XbaI, C4 Eco0109 + XbaI + HindIII, C4 not dgested, L
Gel 5
L, C6 Eco0109, C6 HindIII, C6 XbaI, C6 Eco0109 + XbaI, C6 Eco0109 + HindIII, C6 HindIII + XbaI, C6 Eco0109 + SspI-HF + XbaI, C6 not digested, L, B5 Eco0109, B5 SspI-HF, B5 XbaI, B5 XbaI (other tube), B5 Eco0109 + XbaI, B5 SspI-HF + Eco0109, B5 SspI-HF + XbaI, B5 Eco0109 + SspI-HF + XbaI, L
04-10-18
Gelelectrophoresis
Gel 1
L, J2 Eco0109, J2 HindIII, J2 XbaI, J2 Eco0109 + XbaI, J2 Eco0109 + HindIII, J2 HindIII + XbaI, J2 Eco0109 + HindIII + XbaI, J2 not digested, L, J1 Eco0109, J1 HindIII, J1 XbaI, J1 Eco0109 + XbaI, J1 Eco0109 + HindIII, J1 HindIII + XbaI, J1 Eco0109 + HindIII + XbaI, J1 not digested, L
Gel 2
L, B2 Eco0109, B2 HindIII, B3 Eco0109, B3 HindIII, B3 Eco0109 + SspI-HF + XbaI, B4 Eco0109, B4 HindIII, B4 Eco0109 + SspI-HF + XbaI, not digested B2, L, D1 Eco0109, D1 HindIII, D1 XbaI, D1 Eco0109 + XbaI, D1 Eco0109 + HindIII, D1 HindIII + XbaI, D1 Eco0109 + HindIII + XbaI, not digested D1, L
Gel 3
L, F1 Eco0109, F1 HindIII, F2 Eco0109, F3 Eco0109, F3 HindIII, F6 Eco0109, F6 HindIII not digested F6, L, A2 HindIII, A3 HindIII, not digested A1, E3 Eco0109, E3 HindIII, not digested E3, L
Gel 4
L, H1 Eco0109, H1 HindIII, H1 not digested, I1 Eco0109, I1 HindIII, L
Figure 1: Gel 1 01-10-18
Figure 2: Gel 2 01-10-18
Figure 3: Gel 3 01-10-18
Figure 4: Gel 4 01-10-18
iFigure 5: Gel 5 01-10-18
iFigure 6: Gel 6 01-10-18
Figure 7: Gel 1 02-10-18
Figure 8: Gel 2 02-10-18
Figure 9: Gel 1 03-10-18
Figure 10: Gel 2 03-10-18
Figure 11: Gel 3 03-10-18
Figure 12: Gel 4 03-10-18
Figure 13: Gel 5 03-10-18
Figure 14: Gel 1 04-10-18
Figure 15: Gel 2 04-10-18
Figure 16: Gel 3 04-10-18
Figure 17: Gel 4 04-10-18
21
Assessing Gas Production BioBricks in E.Coli
Show more
October 4
Preparing DNA for Submission
October 5
×
Dustin Vermeulen | Elise Grootscholten | Randall de Waard
Today we made colony PCRs of the biobricks we suspected that where correct of our gasproduction, and of colony 8 of CooA. These where then send to the Erasmus Medical Centre, who sequenced our parts for us.
Colony PCR
Materials: For our PCR (Both plasmid and colony) reactions we have used the standard neb protocols for a 25 μl reaction , with the following concentrations:
10X Standard Taq Reaction Buffer        2.5 μl
10 mM dNTPs                                        0.5 μl
10 μM forward Primer  (VF2)                 0.5 μl
10 μM reverse Primers  (VR)                 0.5 μl
Taq DNA Polymerase                             0.125 ul
Nuclease-free water                               to 25 μl
The primers VF2 and VR got delivered in powder form. So it was dissolved in Nuclease-free water till a concentration of 100 μm.
Methods: A master mix is made with the concentrations shown above. In case of a plasmid pcr, a volume of plasmid is added before the reaction is filled with Nuclease-free water. With the colony PCR we used a toothpick to touch on the desired colony. Before briefly submerging and shaking the tip of the toothpick in the PCR Eppendorfcup filled with the mastermix. Before pricking it into a clean Agar plate a couple of times to grow more colonies in case the PCR shows us the desired products
For our PCR reactions we use the Bio-Rad T100 thermal cycler. Afterwards the products of the PCR reaction were ran on a gel electrophoresis.
PCR Purification
8 μL PCR product + 2 μL ExoSAP-IT
37°C 20 minutes
80°C 15 minutes
 
Sequencing Reaction
4,0 μL
PCR product
0,15 μL
forward or reverse primer
0,5 μL
BigDye V3.1 (Applied Biosystems)
1,75 μL
Sequencing buffer (5x)
3,6 μL
Nuclease free water
10 μL
Total
96°C 1 minute
96°C 10 seconds
55°C 5 seconds
60°C 2 minutes
run for 35 cycli
10°C end
Sequencing reaction purification
Following protocol of CleanSeq Agencourt
Sequencing
ABI 3730 XL, 50 cm capillair
01
Preparing DNA for Submission
Show more
October 5
Preparing DNA for Submission
October 7
×
Dustin Vermeulen | Elise Grootscholten | Randall de Waard
Today we digested our succesfully sequenced biobricks (B1 / B6 (same biobrick), D1, I1 and J2), using EcoRI and PstI to get them into a pSB1C3 backbone. This was then transormed and plated. And the end of the day we made 6 overnight cultures of the barely visible colonies.
Materials
-Restriction enzymes EcoRI-HF and PstI
Methods
Digestion
Mastermix
Cutsmart (10x)
55 μL
EcoRI-HF (20 U/μL)
1 μL
PstI (20 U/μL)
1 μL
Nuclease free water
382,7 μL
50 μL mastermix - ... μL DNA (see below)
A2: 12,6 μL
B1: 15,7 μL
B5: 9,4 μL
B6: 9,6 μL
D1: 10,2 μL
E1: 15,7 μL (1 μg)
I1: 14,3 μL (1 μg)
J1.2: 17,4 μL
CooA (8): 5,4 μL
Dephosphorylation + ligation using rAPId dephos and ligation protocol and kit
02
Preparing DNA for Submission
Show more
October 7
Preparing DNA for Submission
October 9
×
Dustin Vermeulen | Elise Grootscholten | Randall de Waard
Today we mini prepped our DNA (~6 colonies per succesfully sequenced biobrick, overnight cultures made on the 8th of october) and digested this.
03
Preparing DNA for Submission
Show more
October 9
Testing gas production
October 11
×
Elise Grootscholten | Paul Reusink
Overnight cultures made of the following strains with the right plasmid:
Number
BBa_
HB101
JM109
J1
BBa_K2604000
1x
1x
I1
BBa_K2604001
3x
3x
D1
BBa_K2604003
2x
2x
B1
BBa_K2604006
1x
1x
B6
BBa_K2604006
1x
1x
Table 1: Overnight culture scheme.
I1, D1 and J1 are tested for the gas productio. The rest is only made for glycerolstocks
We can not say for sure if the PBad promoter and the gas production enzyme pyruvate decarboxylase work. Fot that more gas production test have to be done. Also the negative controls produce gas. This is a problem. The JM109 and HB101 E.coli strains didn't produce gas in experiment . 
Experiment 6: Testing gasproduction with our new biobricks.
Testing our new biobricks K260401 in JM109 and HB101. As a negative control we used the other new biobricks ...
Materials
-LB culture medium
-Kanamycin
-Bacteria on agar plates with the right plasmid
Methods
Make overnight cultures of the following strains with the right plasmid:
 
Number
BBa_
HB101
JM109
J1
BBa_K2604000
1x
1x
I1
BBa_K2604001
3x
3x
D1
BBa_K2604003
2x
2x
B1
BBa_K2604006
1x
1x
B6
BBa_K2604006
1x
1x
Table 1: Overnight culture scheme.
Only the numbers J1, I1 and D1 are tested. The rest is made for making glycerolstocks.
J1 and D1 are the negative controls in this experiment.
LB: 582 mL +18 mL 100 mM sodiumpyruvate and 525 μL kanamycin (40 mg/mL).
After this the LB is devided in 2. Number 1: 0,5 μL arabinose, number 2: 1,0 μL arabinose.
The rest is done just like earlier experiments with a total of 20 tubes because everthing is done twice.
Results
Some of the I1 (BBa_K2604001) JM109 and HB101 bacteria and the negative controls D1 and J1 (BBa_K2604003 and BBa_K2604000) have produced gas over the weekend.
We can not say for sure if the PBad promoter and the gas production enzyme pyruvate decarboxylase work. Fot that more gas production test have to be done. Also the negative controls produce gas. This is a problem. The JM109 and HB101 E.coli strains didn't produce gas in experiment . 
Figure 1: Gas testing before the weekend, II1.1 HB101 nr.1/  I1.2 HB101 nr.1/  I1.3, HB101 nr.1/  I1.1 JM109 nr.1
Figure 2: Gas testing after the weekend, II1.1 HB101 nr.1/  I1.2 HB101 nr.1/  I1.3, HB101 nr.1/  I1.1 JM109 nr.1
Figure 3: Gas testing before the weekend, I1.3 JM109 nr.1/ I1.2 JM109 nr.1/ J1 JM109 nr.1/ D1 HB101 nr.1
Figure 4: Gas testing after the weekend, I1.3 JM109 nr.1/ I1.2 JM109 nr.1/ J1 JM109 nr.1/ D1 HB101 nr.1
Figure 5: Gas testing before the weekend, D1 JM109 nr.1/ J1 HB101 nr.1
Figure 6: Gas testing after the weekend, D1 JM109 nr.1/ J1 HB101 nr.1
Figure 7: Gas production before the weekend, I1.3 JM109 nr.2/ I1.2 JM109 nr.2/ I1.1 JM109 nr.2/ I1.3 HB101 nr.2
Figure 8: Gas production after the weekend, I1.3 JM109 nr.2/ I1.2 JM109 nr.2/ I1.1 JM109 nr.2/ I1.3 HB101 nr.2
Figure 9: Gas production before the weekend, D1 JM109 nr.2/ D1 HB101 nr.2/ J1 HB101 nr.2/ J1 JM109 nr.2
Figure 10: Gas production after the weekend, D1 JM109 nr.2/ D1 HB101 nr.2/ J1 HB101 nr.2/ J1 JM109 nr.2
Figure 11: Gas production before the weekend, I1.2 HB101 nr.2/ I1.1 HB101 nr.2
Figure 12: Gas production after the weekend, I1.2 HB101 nr.2/ I1.1 HB101 nr.2
06
Testing gas production
Show more
October 11
Assessing different ATP sensors
October 15
×
Loraine Nelson | Elise Grootscholten | Paul Reusink
Biobricks from iGEM were transformed into NEB10Beta. A few of these are sensors that react to ATP and can be used to detect living cells. The Bacteria were plated on agar plates with the right antibiotics to select the bacteria with the right plasmid in them. Every transformation was successful, since there were over 300 colonies on every plate. These colonies could than be used for further experiments and can be easily used for DNA isolation. 
Materials: 
- Biobricks 
- Nuclease free water
- LB- agar
- Variable Volume Pipettes 
- Sterile pipetpoints
- Chemically competent cells ( NEB10Beta, 1x10^9) 
- Heater (42°C)  
- Incubator + shaker (37°C and 250 rpm) 
- Eppendorf tubes (1,5 and 2 ml) 
- Stable outgrow medium for NEB10Beta or SOC- outgrow medium 
Method: 
To be able to use the DNA from the biobricks, they were first diluted in Nuclease free water. This was done by adding 10ul of nuclease free water to the DNA and incubated for 5 minutes at room temperature (18-24°C). After incubation the DNA is transferred to Eppendorf tubes and put on ice. 
For the transformation 1 ul of the DNA was added to 50 ul chemically competent cells strain NEB10Beta. The mixture was incubated for 30 minutes on ice. The mixture got a heat shock by putting the tube in a heater (42°C) for 30 seconds. After the heat shock the tube was immediately put on ice for 5 minutes. 
For good grow of the cells, 950 ul of outgrow medium was added to the mixture. After mixing the cells with outgrow medium, the culture was incubated for 1 hour at 37°C and 250 rpm.  
After incubation the culture was plated onto LB-agar plates. For specific growth of bacteria there was antibiotics added to the plates. The antibiotics that were used, depended on the plasmid. The biobricks used in this experiment were resistent to chloramphenicol (Can). We worked with a work concentration of 35 ug/ml for chloramphenicol. For good results we plated 100 ul undiluted culture onto a plate and centrifuged the rest of the culture. The supernatant was discarded and the pellet was resuspended and plated onto a LB-agar plate. The plates were then incubated by 37°C. 
Results:
Tabel 1 shows the results of the plates which were incubated. 
Biobricks: 
Antibiotics: 
Dilution: 
Colonies:
CFU: 
Bba-K352002
Can 
0x 
>300
-
Bba-K352002
Can
Centrifuged
>300
-
Bba-K352003
Can
0x
>300
-
Bba-K352003
Can
Centrifuged
>300
 -      
Bba-K352004
Can
0x
>300
-
Bba-K352004
Can
Centrifuged
>300
-
Bba-K352005
Can
0x
>300
-
Bba-K352005
Can
Centrifuged
>300
-
Bba-K352006
Can
0x
>300
-
Bba-K352006
Can
Centrifuged
>300
-
Bba-K352007
Can
0x
>300
-
Bba-K352007
Can
Centrifuged
>300
-
Bba-K352008
Can
0x
>300
-
Bba-K352008
Can
Centrifuged
>300
-
Bba-K352009
Can
0x
>300
-
Bba-K352009
Can
Centrifuged
>300
-
Bba-K1390001
Can
0x
>300
-
Bba-K1390001
Can
Centrifuged
>300
-
Bba-K1390003
Can
0x
>300
-
Bba-K1390003
Can
Centrifuged
>300
-
Bba-K1023003
Can
0x
>300
-
Bba-K1023003
Can
Centrifuged
>300
-
Bba-K284001
amp
0x
>300
-
Bba-K284001
amp
Centrifuged
>300
-
Positive controle: Bba- K1023003
None
0x
>300
-   
Negative controle: outgrow medium
none
0x
0
-
Tabel 1: results from the transformation with the biobricks. 
Showed in the Tabel, the transformation gave  enough colonies on each plate, which can be used for further experiments. There is no contamination, as seen by the negative control. 
Conclusion: 
The transformation from the biobricks was successful. The bacteria with the biobricks plasmids will be stored by 4°C and can be used for further experiments. 
 
04
Assessing different ATP sensors
Show more
October 15
Assessing different ATP sensors
October 15
×
Loraine Nelson | Elise Grootscholten | Paul Reusink
The bacteria were mini prepped and than transformed into BL21 DE3. BL21 DE3 has a T7 promotor which is needed for expression of the biobricks. The transformation was successful and multiple colonies were formed on the agar plates with antibiotics. From this plates we grow a few colonies onto new plates for further experiments. 
Transformation Protocol NEB
Overview
Quick Ligation products may be transformed by many different methods. The following protocol is recommended by New England Biolabs.
Protocol
Thaw competent cells on ice.
Chill approximately 5 ng (2 μl) of the ligation mixture in a 1.5 ml microcentrifuge tube.
Add 50 μl of competent cells to the DNA. Mix gently by pipetting up and down or flicking the tube 4 to 5 times to mix the cells and DNA. Do not vortex.
Place the mixture on ice for 30 minutes. Do not mix.
Heat shock at 42°C for 30 seconds*. Do not mix.
Add 950 μl of room temperature media* to the tube.
Place tube at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.
Warm selection plates to 37°C.
Spread 50 to100 μl of the cells and ligation mixture onto the plates.
Incubate overnight at 37°C.
* Please note: For the duration and temperature of the heat shock step as well as for the media to be used during the recovery period, please follow the recommendations provided by the competent cells� manufacturer.
1 ul isolated K1023003 biobrick + 50 ul NEB10bèta 
Positive and negative control without antibiotics
 
05
Assessing different ATP sensors
Show more
October 15
Assessing Gas Production BioBricks in E.Coli
October 16
×
Elise Grootscholten | Randall de Waard
Glycerolstocks have been made of several bacteria with plasmids we created or from the iGEM kit.
22
Assessing Gas Production BioBricks in E.Coli
Show more
October 16
Assessing CooA Production
October 16
×
Dustin van der Meulen
After performing a digestion on the 16th we ran a gel electrophoresis, from which we could conlcude that only the combinations of J23100+B0031, J23100+B0032 and K352001+B0015 seemed to be correct, though because of the small insert size in the promoter and rbs combinations it is difficult to say. From these combinations overnight cultures where made, which we prepped today. 
J23100+B0031
125,9 ng/uL
J23100+B0032
334,1 ng/μL
K352001+B0015
240,3 ng/μL
After this we've discussed our progress with our PI's, 
08
Assessing CooA Production
Show more
October 16
Chemo competent cells
October 16
×
None
New competent cells were made. This time we used the strains JM109 and HB101, since these strains don't produce gas of themselfes. These strains can be used for measurement on our gas production system. The cells are competent enough to use. 
Experiment 2: Chemocompetent cells of E.coli strains JM109 and HB101
28-09-18
Methods
The methods are for JM109 and HB101 both apart.
2 ul pUC19 + 50 ul competent cells
With ampicilin: not diluted, 10^-1, 10^-2, 10^-3, 10^-4
Without ampicillin: not diluted, 10^-4, 10^-5, 10^-6
Negative control: SOC medium.
200 ul not diluted -->
30 ul + 270 ul SOC (10^-1) --> 
30 ul + 270 ul SOC (10^-2) --> 
30 ul + 270 ul SOC (10^-3) --> 
30 ul + 270 ul SOC (10^-4) --> 
30 ul + 270 ul SOC (10^-5) --> 
30 ul + 270 ul SOC (10^-6)
Results
JM109: 3,4*10^6 cfu/ug DNA
HB101: 4,4*10^6 cfu/ug DNA
02
Chemo competent cells
Show more
October 16
Preparing DNA for Submission
October 17
×
Dustin Vermeulen | Elise Grootscholten | Randall de Waard
Today we set our control digest from the 9th of october on gel (the gel was inconclusive because of poor quality), after which we put ~500 ng of DNA on our 96 wells submission plate (both the pSB1C3 and pSB1K3 (sequenced) versions of our biobricks). This was then dried in a 50°C stove for around 2 hours (until all wells were drie).
04
Preparing DNA for Submission
Show more
October 17
Relevant pages
Results
Hardware
Software
Experiment
