<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_XMU-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:XMU-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="jumbotron" style="background-image:url(https://static.igem.org/mediawiki/2016/e/e3/T--XMU-China--a_picture_for_results.jpg);"><DIV class="jumbotron-text"><H1 style="margin-bottom: 0px;padding-bottom:0;border-bottom: 1px solid #333;
 padding-left:16px; color:#333; font-family:" times="" new="" roman="">Results</H1><P style="padding-left:16px; padding-bottom: 15px;
    font-size: 21px;
    font-weight: 200;color:#333;text-shadow: 0 0 1px black;" id="section-1">All is well that ends well</P></DIV></DIV><DIV class="container"><NAV class="navbar  navbar-inverse navbar-fixed-top"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right" style="margin-top:.8em;"><LI class="dropdown" id="Project"><A style=" padding-top:20px;" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href=" https://2016.igem.org/Team:XMU-China/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Proof">Proof Of concept</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Results">Results</A></LI></UL><LI class="dropdown" id="Parts"><A style=" padding-top:20px;" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Composite_Part">Composite Part</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Interlab">Interlab</A></LI><LI class="dropdown" id="hp"><A style=" padding-top:20px;" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practice </A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Human_Practices">Overview</A></LI><LI><A href=" https://2016.igem.org/Team:XMU-China/HP/Silver">Practices Silver</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/HP/Gold">Practices Gold</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/HP/Newsletter">Newsletter</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Engagement">Engagement</A></LI><LI class="dropdown" id="Safety"><A style=" padding-top:20px;" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Safety </A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Safety">Safety</A></LI><LI><A href=" https://2016.igem.org/Team:XMU-China/Judging">Judging</A></LI><LI class="dropdown" id="Notebook"><A style=" padding-top:20px;" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Notebook">Notebook</A></LI><LI><A href=" https://2016.igem.org/Team:XMU-China/Protocols">Protocols</A></LI><LI class="dropdown" id="Team"><A style=" padding-top:20px;" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:XMU-China/Attributions">Attributions</A></LI></DIV></DIV></NAV><DIV class="row" style="margin-left:-2.7%;margin-right:-2.7%;"><DIV class="col-md-2 " id="myScrollspy"><DIV id="sidebar1"><UL class="nav nav-tabs nav-stacked" data-spy="affix" data-offset-top="300"><LI id="sidehead">Results</LI><LI class="active"><A href="#section-1">Entire circuits test</A></LI><LI><A href="#section-3">Responder test</A></LI><LI><A href="#section-4">Model</A></LI><LI><A href="#section-5">Discussion</A></LI><LI><A href="#section-6">Future Works</A></LI><LI><A href="#section-7">Improvement</A></LI><LI><A href="#section-8">Reference</A></LI></UL></DIV></DIV><DIV id="sidebar2"><UL style="color:#000;list-style:none;list-decoration:none;margin-left:3.4%;"><LI style="font-size:20px;color:#8968CD;border-bottom: 0.5px solid #aaa;width:95%;">Results</LI><LI><A style="color:#aaa;" href="#section-1">Entire circuits test</A></LI><LI><A style="color:#aaa;" href="#section-3">Responder test</A></LI><LI><A style="color:#aaa;" href="#section-4">Model</A></LI><LI><A style="color:#aaa;" href="#section-5">Discussion</A></LI><LI><A style="color:#aaa;" href="#section-6">Future Works</A></LI><LI><A style="color:#aaa;" href="#section-7">Improvement</A></LI><LI><A style="color:#aaa;" href="#section-8">Reference</A></LI></UL></DIV><DIV class="col-md-10"><H1 id="section-1">Entire circuits test
</H1><P>In our experiment, for testing our circuit more conveniently, we use <I>CFP</I> to replace the responder and use <I>GFP</I> to replace lysis. Because the fluorescence intensity is easier to be measured. Also, the intensity of the fluorescence can stand for the intensity of the corresponding genes' expression.</P><P>When the concentration of AHL is at low concentration, AHL cannot be detected by the quorum sensing system. And as a consequence, there is no enough inhibitor lacI to repress promoter P<SUB>lac</SUB>. So the <EM>cI</EM> gene is expressing at the wild-type levels. While lacI-2 and GFP are repressed since the CI protein is at high concentration. When AHL around the engineered bacteria reached a certain concentration or a threshold value, AHL could be detected by the sensing molecules Lux R. Then, Lux R interacted with AHL and form activating molecules to activate promoter Lux pR. CFP and lacI-1 express, and CFP protein could be detected by fluorescence measurement. And the lacI-1's product-lacR protein could inhibit the expression of <EM>cI</EM> gene. As a result, lacI-2 and GFP expressed. The lacI-2's product could further repress <EM>cI</EM> gene's expression, which was a negative feedback system. GFP could be measured by fluorescence detection to represent the level of lysis gene's expression. The time of initial expression of <SPAN id="section-2">CFP</SPAN> and GFP were different. As a result, the interval of time could represent the delay time which the switch give for responder expression.</P><H3>1. AHL Gradient Induction</H3><P>As we can see, quorum sensing system could be induced by AHL. As for our gene circuits, when AHL raised to its threshold value and was detected by the sensor part, the promoter Lux PR can be activated, CFP and GFP are expressed. In order to find out the relationship, we decided to make an AHL gradient induction for our gene circuits. </P><P>During our experiment, we first constructed our gene circuit successfully, then we transformed the plasmids into chemical competent cell (DH5α). And then our engineering bacteria were divided into four groups: experimental group 1, 10 and 20 (c(AHL) = 1 ng/μL, 10 ng/μL, and 20 ng/μL) and blank control group. Then we measured OD600 of the <I>E.coli</I> and the fluorescence intensity of CFP and GFP for 900 minutes. The OD600 showed the growth trend of the engineering bacteria. And the fluorescence intensity of each group showed the relationship between AHL and the expression of our circuits. </P><FIGURE><FIGCAPTION><CENTER><STRONG>Figure 1.1</STRONG> The OD600 of versus time of experimental group 1, 10, 20 and control</CENTER></FIGCAPTION></FIGURE><P>From figure 1.1, we found that the growth curves of each group don't show much difference. It suggested that they had the same growth trend, so the difference of fluorescence intensity was mainly from the expression of circuits or other factors.</P><FIGURE><FIGCAPTION><CENTER><STRONG>Figure 1.2</STRONG>The average cyan fluorescence intensity versus time of experimental group 1, 10, 20 and control.</CENTER></FIGCAPTION></FIGURE><P>So we did experiments at the same condition except the concentration of AHL. As showed in figure 1.2, with the increase of the AHL, the expression of the CFP showed an upper trend. In addition, by means of independent t-test, we compared the fluorescence intensity between the group 1, 10, 20 and control. We found the initial expression time of CFP was different. For group 10 and 20, CFP expressed initially and could be detected in the 240th minute. However, for group 1, CFP started to express in the 480th minute.</P><P><CENTER><STRONG> Figure 1.3 </STRONG>The average green fluorescence intensity versus time of experimental group 1, 10, 20 and control. </CENTER></P><P>T-test showed that AHL didn't influence the expression of GFP directly. Group 10 and 20 expressed initially in the 420th minute and green fluorescence intensity had no significant difference in expression level. However, as for group 1, green fluorescence intensity didn’t show much difference comparing with control group, which meant that 1 ng/μL of AHL was not enough to activated the sensor successfully. As a result, the repression on the P<SUB>lac</SUB> cannot be released, and the expression of <I>gfp</I> still stayed in a low level. So the sensor part is an specific and smart device. </P><H3>2. The Delayed Effect</H3><P><CENTER><STRONG> Figure 1.4 </STRONG>The average cyan and green fluorescence intensity versus time of experimental group 20 and control.</CENTER></P><P><CENTER><STRONG> Figure 1.5 </STRONG>The average cyan and green fluorescence intensity versus time of experimental group 10 and control.</CENTER></P><P>As mentioned above, we did analyze the fluorescence intensity data by means of t-test to show that if there was a significant variation between experiment groups and control. And then, we get the initial expression time of CFP/ GFP in experiment groups and control. For group 10 and 20, the initial expression time of CFP was the 240th minute and GFP was detected at the 420th minute. From the results, we could draw the conclusion that the expression time of GFP was delayed to 180mins post CFP, which was the same as our expectation. So the <STRONG>Sensor</STRONG> and <STRONG>switch</STRONG> expressed as expected and match with each other very well. </P><H1 style="border-bottom: 1px solid #aaa;color: #8968CD;text-shadow: 0 0 1px black;margin-bottom:.6em;padding-top: 0;
    padding-bottom: -5%;">Responder test
</H1><P>Against the backdrop of today's widely spread of drug-resistant bacteria, small regulatory RNA (sRNA) offers a new clinical therapy approach to repress the target drug-resistant genes and then it can enhance the effect of antibiotic in the clinical. Here we designed synthetic sRNA to target the antibiotic resistance gene chloramphenicol acetyltransferase (CAT) to generate representative data and to quantify knockdown effectiveness.</P><P>We designed and synthetized RNA silencing cassettes (<STRONG>Table 1</STRONG>) and constructed it into pSB1AC3, an official plasmid which contained ampicillin resistance gene and chloramphenicol resistance gene. Also, we used pSB1AC3+mRFP as control.</P><P><CENTER><STRONG> Figure 2.1 </STRONG>The diagram of synthetized RNA silencing cassettes and control.</CENTER></P><DIV><TABLE class="table table-hover table-striped" width="100%" style="color:#8968CD;font-size:14px"><THEAD><TR><TH>Part</TH><TH>Sequence</TH></TR></THEAD><TBODY><TR><TD>pR promoter</TD><TD>TAACACCGTGCGTGTTGACTATTTTACCTCTGGCGGTGATAATGGTTGC</TD></TR><TR><TD>GUIDE sequence</TD><TD>ATATCCAGTGATTTTTTTCTCCAT</TD></TR><TR><TD>Hfq binding domain</TD><TD>TTTCTGTTGGGCCATTGCATTGCCACTGATTTTCCAACATATAAAAAGACAAGCCCGAACAGTCGTCCGGGCTTTTTTTCTCGAG</TD></TR><TR><TD>T1/TE terminator</TD><TD>TTTCGTTTTATCTGTTTTTGTCGGTGAACGCTCTCTACTAGAGTCACACTGGCTCACCTTCGGGTGGGCCTTTCTGCGTTTATA</TD></TR></TBODY></TABLE></DIV><P><CENTER><STRONG>Table 1</STRONG> The sequence of synthetized RNA silencing cassettes</CENTER></P><P>Firstly, we designed and synthetized RNA silencing cassettes and construct it into pSB1AC3, an official plasmid which contains amp and cm resistance gene. Also, we used pSB1AC3+mRFP as control. Then we transfer the plasmids into chemical competent cell (DH5α) and add chloramphenicol of 50μg/ml to detect the knockdown effectiveness through the phenotype of survival index. We continuously measured the OD600 through the Bioscreen.</P><P><CENTER><STRONG> Figure 2.2 </STRONG>The growth curves of bacteria with the sRNA cassettes or mRFP on the chloramphenicol of 50ng/μL.</CENTER></P><P>The growth of chemical competent cells(DH5α) with pSB1AC3+sRNA is significant repressed by the chloramphenicol (50μg/mL) compared with the control with pSB1AC3+mRFP (The blue line). It indicated the sRNA's highly knockdown effectiveness.</P><P>From the result, we can found the OD600 of control group shows S-type growth trend. But the experimental group shows a low level all the time. It <SPAN id="section-4">suggests</SPAN> the growth of the bacteria which contain the sRNA cassettes significantly repressed by the chloramphenicol, which indicates the sRNA's highly knockdown effectiveness.</P><H1 style="border-bottom: 1px solid #aaa;color: #8968CD;text-shadow: 0 0 1px black;margin-bottom:.6em;padding-top: 0;
    padding-bottom: -5%;">Model
</H1><P><STRONG>Abstract</STRONG>: <EM>E.coli</EM> has been modified in order to produce small interfering RNA molecules (siRNA) to attack the resistance genes that become available for nearby bacteria, when a resistant strain goes through lysis. However, a regulation system should be applied so the productive cells do not become a problem. A circuit has been implemented, this circuit regulates the transcription of said siRNAs among the production of toxins that could lead to death of any nearby cell, including the siRNA producers themselves. Taking in consideration all these elements a mathematical model using differential equations has been designed to approach the number of viable cells within the toxic environment once the circuit has been activated.</P><P><EM><STRONG>Keywords</STRONG>: siRNA, toxin, mRNA, transcription, translation, exponential</EM></P><H3><STRONG>Rate of production of toxin mRNA and toxin molecules</STRONG></H3><P>The toxins are produced through a gene in the circuit, so the production of toxin depending on the translation of the mRNA produces, which also depends on transcription variables. A differential equation was taken in consideration in order to measure the mRNA rate production.</P><CENTER>$$\frac{dr}{dt} = K - D_{r}*r$$</CENTER><P>\(K\) represents the transcription rate, which is a constant parameter, in molecules of RNA per second, \(D_{r}\) is the degradation rate expressed in units per seconds as it depends on how much mRNA is actually present on the cell, expressed by the variable &quot;\(r\)&quot; in units of molecules of mRNA produced.</P><CENTER>$$r = \frac{K}{D_{r}}+Ce^{-D_{r}t}$$</CENTER><P>Afterwards, the original mRNA production equation was obtained through the integration of the differential equation by linear integration.</P><P>Establishing a new condition \(r(0)=R_{0}\) then we can complete the system. Substituting we obtain:</P><CENTER>$$R_{0} = \frac{K}{D_{r}}+C \rightarrow C = R_{0}-\frac{K}{D_{r}}$$</CENTER><P>We finally obtain the equation for mRNA synthesis:</P><CENTER>$$r(t)= \frac{K}{D_{r}}+(R_{0}-\frac{K}{D_{r}})e^{-D_{r}t}$$</CENTER><P>To simplify the model we assume that each mRNA molecule will be transcribed in order to produce one molecule of protein. This means that each molecule of mRNA produced is equivalent to one molecule of active toxin that in certain concentration can kill bacteria. We assume a fast degradation of mRNA inside the system and almost no significant dispersion of the molecules, so this ratio can be respected.</P><H3><STRONG>Productive cells inside the system</STRONG></H3><P>The number of <EM>E. coli</EM> cells within the system can not be considered as a constant, neither as a linear function due to the exponential nature of cellular division. Given normal growth of the cell culture, it is possible to describe the number of <EM>E.coli</EM> cells as a function of time. Nevertheless, the <EM>E. coli</EM> concerning the model produces toxins that alter the number of <EM>E.coli</EM> cells itself, also there is a natural decrease in the number of bacteria caused by cellular death, resulting in the addition of new terms to the traditional equation. An equation considering all the previous factors is shown below.</P><CENTER>$$C(t)=C_{0}e^{Gt}(1-D)-M*C$$</CENTER><CENTER>$$C(t)=C_{0}e^{Gt}(1-D)-M*C_{0}e^{Gt}(1-D)$$</CENTER><CENTER>$$C(t)=C_{0}e^{Gt}(1-D)(1-M)$$</CENTER><P>Where:
\(C_{0}\): Initial population [=] cells
\(C\): Total productive cells [=] cells
\(D\): Cell fraction ( \(C_{0}e^{Gt}(1-D)\) ) ​that dies naturally [=] adimensional &lt; 1
\(G\): Rate of cells that are born [=] 1/second
\(M\): Fraction of viable cells ( \(C_{0}e^{Gt}(1-D)\) ) that lyse due to toxins [=] adimensional &lt; 1 
\(D\) and \(M\) are not related, therefore they are independent. It is also important to notice that we assumed that the number of toxins equals the number of mRNAs transcribed, so our previous defined function \(r(t)\) can help us to determine the constant \(M\). </P><P><CENTER>\(r(t) &lt; X\): no cell death, \(M = 0\)</CENTER></P><P><CENTER>\(r(t) \geqslant X\): cell death occurs, \(M=\frac{T r(t)}{(1-D)C_{0}e^{Gt}}\) &lt; 1 </CENTER></P><P>where \(X\) is the critical toxin production rate. Also in order to determine the \(M\) term we need other variables.</P><P>\(M=\frac{T r(t)}{(1-D)C_{0}e^{Gt}}\), where:</P><P>\(r(t)\) = toxins produced in time \(t\) [=] (molecule of toxin)
\(T\) = cells that die by unit of toxin [=] (cells / molecule of toxin)
\(M\) = dead cells / total viable cells [=] adimensional</P><P><CENTER>for \(r(t) &lt; X\): \(C(t) = C_{0}e^{Gt}(1 − D)\) (3)</CENTER></P><P><CENTER>for \(r(t) \geqslant X\): \(C(t)= C_{0}e^{Gt}(1-D)(1-\frac{T r(t)}{(1-D)C_{0e^{Gt}}})\) (4) then</CENTER></P><P><CENTER>\(C(t) = C_{0}e^{Gt}(1 − D) − T r(t)\) (5)</CENTER></P><P id="section-5">The coupled equations yield the following final modelling:
</P><CENTER>$$C(t)=C_{0}e^{Gt}(1-D)- T[\frac{K}{D_{r}}+(R_{0}-\frac{K}{D_{r}})e^{-D_{r}t}]$$</CENTER><H1 style="border-bottom: 1px solid #aaa;color: #8968CD;text-shadow: 0 0 1px black;margin-bottom:.6em;padding-top: 0;
    padding-bottom: -5%;">Discussion
</H1><P> We got ideal experiment results. From the experiment results, we found that the entire gene circuits can express well. <STRONG>Sensor</STRONG> was a specific and smart device, which could detect the quorum sensing molecules AHL and activated the following gene circuits when AHL reached threshold value and stayed “off” state under the threshold value. <STRONG>Switch</STRONG> was a perfect device, which delay the expression of lysis(represented by <I>gfp</I>) so that responder(represented by <I>cfp</I>) had enough time to express. Also, <STRONG>responder</STRONG> test showed that sRNA is a dramatically weapon with high knockdown effectiveness. So if we connected these perfect parts into a synthetic biological device and transformed it into chemical competent cell (DH5α), we can get powerful bacterial soldier to fight and beat antibiotic resistant bacteria.</P><P> The data strongly supports our circuit design. Although we have chosen T4 endolysin gene as our lysis system, we didn’t achieved it for limiting of time. So there are some future works.</P><H1 style="border-bottom: 1px solid #aaa;color: #8968CD;text-shadow: 0 0 1px black;margin-bottom:.6em;padding-top: 0; padding-bottom: -5%;">Future Works
</H1><P>● Design the sRNA silencing cassettes according to acc(3)-Id gene and test its efficiency.
● Finish the M13 phage packaging experiment.
● Test the lysis gene: T4 endolysin.
● Expansion
We have stated that the four part of our gene circuit can be disassembled and assembled. The sensor, responder even the switch is changeable. <SPAN id="section-7">In</SPAN> the following time, we will search the alternative component. We believe more efficient gene circuit will be found and developed.
</P><H1 style="border-bottom: 1px solid #aaa;color: #8968CD;text-shadow: 0 0 1px black;margin-bottom:.6em;padding-top: 0;
    padding-bottom: -5%;">Improvement of project from WHU-2014
</H1><P>Because <A href="http://igem-whu.github.io/igem2014/">WHU-China</A> had tested their Lysis System by replacing the lysis gene with mRFP, and the system worked well, so we only tested the redesigned Street Cleaner System.</P><P>(1)Get sequence of formaldehyde dismutase from <I>Halomonas sp. JC-5B</I>.<I>Halomonas sp. JC-5B</I> is a formaldehyde tolerance bacteria in Pacific Ocean, first discovered by Third Institute Of Oceanography,State Oceanic Administration in 2014 [3].After designed a series of primers, we got the sequence of this enzyme by PCR.
</P><P id="section-2"><CENTER><STRONG>Figure 3.1</STRONG> The result of PCR (the length of FDM is 1200bp).</CENTER></P><P>(2)The expression of FDMWe cloned the sequence of FDM into plasmid pET-28a, a widely used expression vector system for prokaryote, transformed it into E.coli BL21(DE3), and induced using IPTG.</P><P id="section-2"><CENTER><STRONG id="section-8">Figure 3.2</STRONG> SDS-PAGE for FDM (FDM is about 44kDa).</CENTER></P><P>The redesigned system is <A href="http://parts.igem.org/Part:BBa_K1960101">BBa_K1960101</A>, which can replace the failed constructed parts <A href="http://parts.igem.org/Part:BBa_K1334015" target="_blank">BBa_K1334015</A>, <A href="http://parts.igem.org/Part:BBa_K1334016" target="_blank">BBa_K1334016</A> and <A href="http://parts.igem.org/Part:BBa_K1334017" target="_blank">BBa_K1334017</A>.</P><H1 style="border-bottom: 1px solid #aaa;color: #8968CD;text-shadow: 0 0 1px black;margin-bottom:.6em;padding-top: 0;
    padding-bottom: -5%;">Reference
</H1><P>[1] Bernheim, A. G., Libis, V. K., Lindner, A. B. &amp; Wintermute, E. H. Phage-mediated Delivery of Targeted sRNA Constructs to Knock Down Gene Expression in <I>E. coli</I>. <I>J. Vis. Exp.</I><STRONG>109</STRONG>, 1–10(2016). 
[2] Kobayashi, H., Kærn, M. &amp; Araki, M. et al. Programmable cells: Interfacing natural and engineered gene networks. <I>PNAS</I><STRONG>101</STRONG>, 8414-8419(2004). 
[3] Guilan Zhou(2014). The Diversity of Formaldehyde Resistant Bacteria in the Ocean Sediments and a Primary Study on a Formaldehyde Degrading Bacterium <EM>Halomonas sp.JC-5B</EM>. Master's dissertation, Xiamen University(2014)
</P></DIV></DIV></DIV><DIV class="foot" style="margin-top:5%;"><DIV style="width: 100%; height: 200px; background:#333; padding-top: 10px;" class="col-md-12"><DIV id="foot"><DIV style="display: inline-block; top: 3%; width: 35%; margin-left: 6%;"><P style="font-size: 16px; color: #fff; border-bottom: 1px solid #fff; padding-bottom: 3%; margin-bottom: 0px;">CONTACT US</P><A href="https://www.facebook.com/IGEM-XMU-China-1067638406629389" target="_blank" style="text-decoration:none;"><SPAN style=" padding-left:4%;font-size: 14px; color: #fff; margin-top: 10%; margin-bottom: 0px;">https://www.facebook.com/IGEM-XMU-China-1067638406629389</SPAN></A><A href="https://www.twitter.com/Amoy_igem" target="_blank" style="text-decoration:none;"><SPAN style="padding-left:3%;font-size: 14px; color: #fff; margin-top: 10%; margin-bottom: 0px;">https://www.twitter.com/Amoy_igem</SPAN></A><SPAN style="font-size: 14px; color: #fff; margin-top: 10%; margin-bottom: 0px;padding-left:3%;">igemxmu@gmail.com</SPAN></DIV></DIV></DIV><DIV style="background-color:#8968CD; color: #fff; height: 33%;text-align: center;padding-top: 7%;margin-bottom:0;"><P id="foottag" style=" margin-bottom: 0;margin-top: -2%;font-size: 12px;color:#FFF;line-height:12px;text-align:center;">Name: XMU-China   School: Xiamen University</P><P id="foottag" style="    margin-bottom: 0;margin-top: -1%;font-size: 12px;color:#FFF;line-height:12px;text-align:center;">Address: Xiamen University, No. 422, Siming South Road, Xiamen, Fujian, P. R. China 361005</P></DIV></DIV><DIV class="foot-2" style="margin-top:5%;z-index:0;"><DIV style="width: 100%; height:165px; background:#333; padding-top: 10px;" class="col-md-12"><DIV id="foot-2"><DIV style="display: inline-block; top: 3%; width: 94%; margin-left: 3%;"><P style="font-size: 13px; color: #fff; border-bottom: 1px solid #fff; padding-bottom: 3%; margin-bottom: 0px;">CONTACT US</P><A href="https://www.facebook.com/IGEM-XMU-China-1067638406629389" target="_blank" style="text-decoration:none;"><SPAN style=" padding-left:4%;font-size: 10px; color: #fff; margin-top: 10%; margin-bottom: 0px;">https://www.facebook.com/IGEM-XMU-China-1067638406629389</SPAN></A><A href="https://www.twitter.com/Amoy_igem" target="_blank" style="text-decoration:none;"><SPAN style="padding-left:3%;font-size:10px; color: #fff; margin-top: 10%; margin-bottom: 0px;">https://www.twitter.com/Amoy_igem</SPAN></A><SPAN style="font-size: 10px; color: #fff; margin-top: 10%; margin-bottom: 0px;padding-left:3%;">igemxmu@gmail.com</SPAN></DIV></DIV></DIV><DIV style="background-color:#8968CD; color: #fff; height: 10%;text-align: center;padding-top: 2%;padding-bottom: 2%;"><P id="foottag" style=" margin-bottom: 0;margin-top: 0;font-size: 9px;color:#FFF;line-height:9px;text-align:center;">Name: XMU-China   School: Xiamen University</P><P id="foottag" style="    margin-bottom: 0;margin-top: -4%;font-size: 9px;color:#FFF;line-height:9px;text-align:center;">Address: Xiamen University,  No. 422, Siming South Road, Xiamen, Fujian, P. R. China 361005</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>