<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IISc-Bangalore_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IISc-Bangalore/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="contentWrapper"><DIV style="position: fixed; width: 100%; top: 16px; left: 0; z-index: 1;"><DIV class="navbar"><DIV class="dropdown"><DIV class="dropbtn">COMMUNITY</DIV><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:IISc-Bangalore/HP/Silver">Human Practices</A><A href="https://2017.igem.org/Team:IISc-Bangalore/HP/Gold_Integrated">Integrated Practices</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Engagement">Public Engagement</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Safety">Safety</A></DIV></DIV><DIV class="dropdown"><DIV class="dropbtn">HARDWARE</DIV><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware">GCODe</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware/GCODe-Mini">Mini</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware/GCODe-Pro">Pro</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware/Results">Results</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware/Notebook">Hardware Evolution</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware/Documentation">Build Your Own</A></DIV></DIV><DIV class="dropdown"><DIV class="dropbtn">LAB</DIV><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:IISc-Bangalore/Assembly">Assembly</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Results">Results</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Demonstrate">Demonstrate</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Notebook">Notebook</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Protocols">Protocols</A><A href="https://2017.igem.org/Team:IISc-Bangalore/InterLab">InterLab</A></DIV></DIV><DIV class="dropdown"><DIV class="dropbtn">PROJECT</DIV><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:IISc-Bangalore/Description">iFLOAT</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Model">Model</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Experiments">Experiments</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Parts">Parts</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Design">Design</A></DIV></DIV><DIV class="dropdown"><DIV class="dropbtn">ABOUT</DIV><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:IISc-Bangalore/Team">Team</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Philosophy">Philosophy</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><A href="https://2017.igem.org/Team:IISc-Bangalore">HOME</A></DIV></DIV></DIV><DIV id="pagefooter" class="clearfix"><DIV id="bigfooter" class="clearfix"><DIV id="footersponsors"><TABLE style="margin: auto"><TBODY><TR><TH colspan="6">Sponsors</TH></TR></TBODY></TABLE></DIV><DIV id="footercontact"><TABLE style="margin: auto"><TBODY><TR><TH colspan="6">Contact</TH></TR><TR><TD> </TD></TR><TR><TD> </TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV id="inPageNav"><OL><LI><A href="#prep">Preparation</A></LI><LI><A href="#project">Project Work</A></LI></OL></DIV><DIV id="contentMain"><H1 id="prep">Preparation</H1><H2>16/3/17</H2><P> Autoclaved items and prepared media </P><H2>17/3/17</H2><P>Obtained <I>S. cerevisiae</I> BY4741 from Dr. Rajyaguru’s lab and incubated it at 30 degree celcius </P><P>Inoculated the above strain in 2 mL YPD [4.5 mL YPD media + 0.5 mL 20% glucose] using a sterile toothpick</P><H2>18/3/17</H2><P>Went to Prof. Dighe’s lab to get some chemicals, sans people</P><P>Yeast in the inoculated tube are growing fine but seem to have aggregated at the bottom.</P><H2>19/3/17</H2><P> Started 8 growth curves</P><P>Cuvette placement (orientation) was not right for the first few readings; path length was not 1 cm.</P><H2>21/3/17</H2><P>Autoclaved plates, falcons, media (YPDA and YPGA)</P><P>Got the plasmid pRS316 and HeLa cells from a lab</P><P>Decided to try shockwave transformation of yeast. We will be amplifying the plasmid pRS316 in <I>E. coli</I>, mini prepping it and taking it to Aerospace department.</P><H2>22/3/17</H2><P>Got DH5a competent cells from Prof. Chakravortty’s Lab. We have transformed the pRS316 plasmid in <I>E. coli</I>. We also made SOB media. Our selection marker for now is Ampicillin. </P><H2>23/3/17</H2><P>Plates (one streak and one spread) yeast culture and kept the remaining in cold room. We also observed plates of <I>E.coli</I> DH5a today</P><H2>24/3/17</H2><P>Transformants from the Amp+ plate were inoculated in SOB (3 test tubes)</P><P>WT from the amp- plate was inoculated in SOB (2 test tubes)</P><P>Observations: No GFP fluorescence was observed, possibly because plasmid is low copy (?). Note from Sai: In hindsight, we should have realized that the GFP was under a yeast promoter (hehe)</P><H2>27/3/17</H2><P>Step 8 of protocol failed: No pellet was obtained. Need to redo entire set of experiments :( </P><P> Miniprep was then carried out (protocol followed from Sambrook)</P><H2>29/3/17</H2><P>Plasmid (Yeast GFP) brought from Prof. Rajyaguru’s lab</P><P>pRS316 Amp/Ura. High conc, use 1 μL from Nupur’s protocol</P><P>Will show GFP expression in yeast only, not <I>E.coli</I></P><H2>31/3/17</H2><P>Buffers for electrophoresis were made and autoclaved</P><P>Electrophoresis showed presence of plasmid! :D </P><H2>02/4/17</H2><P>YPD media prepared and inoculation done.

</P><H2>03/4/17</H2><P>Shockwave transformation canceled by Akshay. Images of yeast cells were taken under the microscope for heamocytometry</P><P>At night comp cells prep was done. Gel was run for the transformants (terrible results, needs redoing)</P><H2>30/4/17</H2><P>Lab meeting done at 20:00. Plans were laid down for the whole project. As a whole, need to 
		<UL><LI>Overexpress hexose transporters
			</LI><LI>Overexpress invertase
			</LI><LI>ACE2 gene knockout to induce clumping
			</LI><LI>Create GFP-GVNP and GVNP-TEV biobricks
			</LI><LI>Hardware part
			</LI><LI>Fiji plugin for haemocytometer yeast count
		</LI></UL></P><P>Fresh LB, YPAD, YPAD (2x) was prepared</P><H2>01/5/17</H2><P>Heat shock transformation carried out in Dipa lab.
	
</P><H2>02/5/17</H2><P>(DC lab) Observation:- The colonies look good. Thus procedure shall be taken forward. Fresh media prepared, plates autoclaved.</P><H2>04/5/17</H2><P>Raj and Sharath carried out miniprep in DC lab.</P><P>Nanodrop readings too high. Ran, gel; too thick, not too clear. Repeated gel with lower concentration of DNA, presence of genomic DNA contamination and RNA contamination (probably non functional RNAse), need to repeat experiment</P><H2>05/5/17</H2><P>Raj and Sharath repeat miniprep, ran gel. Do get plasmid bands but with heavy DNA and RNA contamination (except for the third lane, for reasons unknown)</P><P>SOB plates were prepared, YPAD plates were prepared, and some required reagent solutions were prepared</P><H2>07/5/17</H2><P>Yeast cultures discarded, cant store them  properly without drastically affecting the efficiency. Fresh inoculum prepared</P><H2>08/5/17</H2><P>Bulk extraction commences from a 50 ml culture. Ran the gel and SUCCESS! :D . Realized some errors in our protocol which helped us prevent genomic DNA contamination. Nanodrop readings were positive and resuspension was done in TE and not MilliQ. </P><H2>09/5/17</H2><P>Met Nupur and asked for selection media, and the complete plasmid map we were given. Also shockwave transformation is a go, the first trial should be over tomorrow, 10th</P><H2>10/5/17</H2><P>Cleaned up the lab: hoods, drawers, trash. Rearranged equipment more sanely. Checked stocks of Eppendorfs, tips, etc.</P><H2>11/5/17</H2><P>LiAc transformation and shockwave transformation were carried out. Technical difficulties lead to LiAc transformation taking a LOT of time to carry out. </P><P>Media was prepared</P><P>After meeting with Dr Srinath, realized &quot;Why work with yeast? <I>E.coli</I> is easier to work with, protocols are easier and less time consuming&quot;</P><H2>12/5/17</H2><P>Made reagent stocks.</P><H2>15/5/17</H2><P>Media preparation. Different percentages of agarose gel prepared to check for fragility and suitability for low melting agar for cell density measurements. The burette idea was discarded, because vortexes formed and mixing occurs.</P><H2>16/5/17</H2><P>Inoculated DH5A (no plasmid) in LB media. Realized there are some contamination problems. Glycerol stocks were prepared and some transformants were plated</P><H2>18/5/17</H2><P>More inoculations done, growth curve of <I>E.coli</I> and yeast was carried out.</P><P>At 4:35 PM Srinath sir entered the lab and questioned everyone about the project. He pointed out various errors people were committing and asked for a lab meeting scheduled on 19th.</P><H2>30/5/17</H2><P>The lab notebook had been neglected for the past few days, sadly</P><P>Yesterday, five different transformants of pSB1A3 (ampicillin resistant) were plated. Today, we will miniprep the plasmid and ideally run a gel to confirm. To do this, we have prepared alkaline lysis solutions I and III beforehand.</P><P>Made working stocks of the required reagents</P><P>Ran a miniprep. Ran a gel, absolutely no bands were present. There was a minor screwup while loading. Will run gel again tomorrow.</P><H2>05/6/17</H2><P>Once again, a major ga in the lab notebook. 
	</P><P>Possible reason for failure in gel- maybe used 50x TAE rather than 1x. Maybe EtBr diffused too fast. The second gel ran on 31st failed because probably 20x SB was used. . Even the ladder was not seen so the problem lied with the gel. The third gel maybe ran for too long and thus no bands were seen. </P><P>On the plus side, the assembly plan is going well. Primers are being designed, to be finalized with Arunavo tomorrow and then with Srinath sir afterwards</P><P>Four primary inoculums made today. If they are dense enough then we'll carry out miniprep tomorrow, then run a gel and discuss primers.</P><H2>08/6/17</H2><P> Haemocytometry imaging was carried out. We tried out Nigrosin staining for contrast between cells. There was non uniform cell density due to smearing and dead cells take up the die,thus we need to counterstain with something else. We need to come up with aa reproducible protocol for collab with other teams</P><P>LB kits and SOB kits were made. Comp cells were transformed using pSB1C3 and pSB1A3.</P><H2>09/6/17</H2><P>Imaging was tried using the saturated <I>E.coli</I> and yeast primary inoculum (set up on 8.6.17 night). Nothing significant came out from the 1st sem lab compound microscope for safranin staining. </P><P>Phase contrast inverted microscope was used to take images. The cells and the grid of the haemocytometer do not come on the same focal plane. So superimposition of the cell and the grid image was suggested. Also <I>E.coli</I> cells doesn't come in 10X zoom and in 40X zoom the grids of the haemocytometer go out of the frame. ~Will try to take image from the inverted microscope on Monday (12.6) in 1st sem lab.</P><P>Plates containing transformed cells didnt show any colonies till night</P><H2>10/9/17</H2><P>SOB+chloramphenicol plates are made for transformation. Cells were transformed with 10, 50, 100 pg/microL concentrations of the plasmid and plated on SOB+ chloramphenicol plates in duplicates .Two controls were plates on -ve chloramphenicol and +ve chloramphenicol plates.</P><H2>11/6/17</H2><P>In the morning, imaging was done with safranin. Not going to use nigrosin anymore. Tried <I>E. coli</I> and yeast. Yeast looks good without dye, but planning to use it anyway to improve contrast for the software. Camera takes worse images compared to direct view through eyepiece. :( </P><P>Prepared more SOB plates with and without chloramphenicol. Went to Dighe’s lab, collected 3 aliquots of competent DH5alpha and two plasmids (NRR3GST and N3NRRCisT). Will collect <I>Pichia</I> culture/plate from Simna tomorrow (call 9535145489 to confirm)</P><P>Returned to lab, and immediately transformed their comp cells using comp cell test kit (for positive control). Used plates with IPTG (100 uL of 40 mM filter-sterilized IPTG spread over and dried). Right now, going to transform our comp cells (in duplicates) using the comp cell test kit. </P><H2>19/6/17</H2><P>Lab notebook has not been updated in quite a while again. We should have… Anyway, stuff last week failed. One miniprep+gel went well. Some transformants of J04450 were red while others on the same plate were not. Some of the red colonies were - “papillae,” apparently. We tried miniprepping them and got a good gel only once. WT DH5A and <I>Pichia</I> X33 plates were streaked. </P><P>Then, the transformation work started. 3*50 µL aliquots each of TSS- and CaCl2-competent cells were transformed with 10 pg/µL, 50 pg/µL and 100 pg/µL of J04450. From each transformant tube, three plates were made: 100 µL directly, 20 µL (with 80 µL PBS) and the remainder (pellet resuspended in 100 µL PBS). Spread plating was then done carefully. </P><H2>22/6/17</H2><P>A test PCR was done today. The results were disappointing as the band obtained was very weak, large an\mount of primer dimers too. Thus proper selectio0n of primers will be done and PCR shall be repeated tomorrow.</P><H2>Things learned at Lupin</H2><P>Look up stripping and redressing gas vesicles from P. rubescens and see if it has been done. Look up the protocols if so. Warning: P. rubescens toxicity (hepatotoxic microcystins). If not, think about using other gas vesicles... haloarchaeal/Anabaena?</P><UL><LI>Obtain <I>P. rubescens</I> (order from reputable online sources or obtain from nearby institutes). Look up protocols to culture and lyse the cells. Find someone who works on this to help us. Contact Walsby. Look at how gas vesicles are purified from them. Purify large numbers of gas vesicles, concentrate them, and store for future use. Assay their critical pressure, try centrifugation at various rpms, take images, try FACS (?), try SEM/TEM. 
		</LI><LI>Try overexpressing GvpA in <I>E. coli</I>. Decide on a promoter and RBS, and create an assembly plan for the GvpA plasmid. Order the necessary primers, assemble the plasmid. Transform into <I>E. coli</I>. Three possibilities: 1) nothing happens, 2) inclusion bodies are formed, 3) entire gas vesicles are formed. Look up a way to distinguish between these possibilities. If 1), give up. If 2), try denaturing, renaturing and see if gas vesicles are formed. Otherwise give up. 3) Lyse the cells, purify large numbers of gas vesicles, concentrate them, and store for future use.
		</LI><LI> Create an assembly plan for the GvpC-linker-DDDDK-sfGFP construct that can be used in both <I>E. coli</I> and <I>Pichia</I> by adding necessary promoters, RBSes, terminators and restriction sites using primers. Order the necessary primers for everything, assemble the plasmids. Transform into <I>E. coli</I> and <I>Pichia</I>. Overexpress the fusion protein. Create a rough protein extract.
		</LI><LI>Demonstrate iFLOAT by mixing gas vesicles purified in Steps 1/2 with the rough protein extract of Step 3 and performing serial purifications. At each stage, assay the quantity of protein purified. Perform protein purity assays (HPLC).
		</LI><LI>Design an adapter BioBrick for any protein someone might like to purify using the iFLOAT method. Create an assembly plan and execute it.
	</LI></UL><P>Fancy stuff we can try if we have time:  Try out acoustic sedimentation of cell lysate. Look into using gas vesicles for purifying other things that have protein binding domains (GvpC-BINDER) e.g. mammalian cells [a more general purification strategy].  Look into using it for rough enzyme industries. Try optimizing process with respect to temperature, reactor topography. Try designing a continuous flow device to execute iFLOAT, say with immobilized enterokinase beads in the final chamber. Think about reusing GvpA multiple times.</P><H2>01/9/17</H2><P>Orders for our biobricks placed, incase more problems occur with minipreps and other protocols. </P><H1 id="project">Project work</H1><H2>12/9/17</H2><P>Made primary inoculum of DH5A and TG1 for TSS comp cell prep</P><H2>13/9/17</H2><P>Gold plating for SEM, and SEM imaging of the gas vesicles was carried out. Vesicles of similar size were visible at low concentrations. At highe concentrations, salt crystals are predominant as the samples were stored in PBS. Removing salts for SEM is not viable as salt is required to maintain isotonicity.</P><P>Plates, eppendorfs, and spreaders were autoclaved. Chloramphenicol testing and comp cell prep was scheduled.. Subcultures of DH5A and TG1 were made</P><H2>14/9/17</H2><P>Plates check, cmp works. Comp cells prepared, DH5A: TSS protocol, TG1: CaCl2 protocol</P><H2>15/9/17</H2><P>Poured 19 cmp plates. Transformed five biobricks using NEB's comp cells. 
	</P><P>Biobricks transformed
		<UL><LI>BBa_K525998 T7 promoter+RBS 
			</LI><LI>BBa_K731721 T7 terminator 
			</LI><LI>BBa_K1321337 sfGFP CDS
			</LI><LI>BBa_K1650037 Spycatcher CDS
			</LI><LI>BBa_J18932 mCherry CDS
		</LI></UL></P><P>While the five plasmids were transformed, the eight Interlab biobricks were subjected to heat shock tranformation using DH5A comp cells.
	

</P><H2>16/9/17</H2><P> (12:30)Controls are good. All except 10M show colonies. Interlab plates checked after 14 hrs, biobrick plates checked after 16 hrs.</P><P>(15:30) Biobrick plates were restreaked onto Cmp plates made earlier. 10M was transformed again adn plate at 17:30</P><P>(18:00) Interlab plates restreaked, incubating now.
	
</P><H2>16/9/17</H2><P>LUDOX and Fluorescein measurements for the Interlab protocol were done and the readings noted down. Plans for DLS were layed down.</P><H2>18/9/17</H2><P>Images of the NRC-1 gas vesicle nanoparticles were retaken, taking salt crystals into account - 0.01 ug/ul concentration</P><P>miniprep of the transformed biobricks were carried out. Out of the four biobricks, T7 promoter + RBS did not give a prominent bacd, others were good. Nanodrop readings were also taken</P><H2>19/9/2017</H2><P>Prepared 
	<UL><LI> 500 ml LB
		</LI><LI>400+200 ml LB Agar
		</LI><LI>32 petri plates
		</LI><LI>2x LB plates
	</LI></UL>
	Inoculated
	<UL><LI>DH5a in  ml SOB (TSS comp cell prep)
	</LI></UL></P><H2>20/9/17</H2><P>Miniprep was carried out again. Again, the results fo the T7 promoter + RBS was not very prominent. Miniprep of the remaining colonies of the biobrick will be done.</P><P>Part 3 of the Interlab protocl was carried out (DC lab)</P><P>3M cultures were miniprepped and a gel was run</P><P>Biotinylation protocol was decided upon. We have 12.87 mg NHS-Biotin. </P><P>	1:30 AM
	<UL><LI>Set everything into autoclave
	</LI></UL>
	3:00 AM
	<UL><LI>Took everything out of the autoclave
		</LI><LI>Plates dried for 1 hr, agar stored in oven
	</LI></UL>
	4:00 AM
	<UL><LI>Plates dried in hood for 1 hr
	</LI></UL>
	5:00 AM to 7:00 AM
	<UL><LI>Plates poured
		</LI><LI>Cmp stock conc = 35 mg/ml, working conc = 35 ug/ml
	</LI></UL>
	21:30
	<UL><LI>TSS Protocol for comp cell prep
	</LI></UL>
	22:10
	<UL><LI>Set secondary culture 
	</LI></UL></P><H2>21/9/17</H2><P>VF2 and VR primers were optimized, PCR being carried out using <I>Taq</I>. The optimal temp being 56 degree C (DC lab)
</P><P>Optimizing PCR (set 1)
<UL><LI>Made reaction mixture of 200 µl per template , for splitting into 8 * 25 µl reactions
	</LI><LI> PCR reaction mixture
	</LI><LI>FP – 4µl
		</LI><LI>RP – 4µl
		</LI><LI>Stock template – 8 µl
		</LI><LI>2x MM – 100 µl
		</LI><LI>ddH2O – 84 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR 9:30 PM
	</LI><LI>Ran products on 1% agarose TAE gel (22/9, 12:20 AM)
</LI></P><H2>22/9/17</H2><P>Optimizing PCR (set 2)
<UL><LI>Made reaction mixture of 200 µl per template , for splitting into 8 * 25 µl reactions
	</LI><LI> PCR reaction mixture
	</LI><LI>FP – 4µl
		</LI><LI>RP – 4µl
		</LI><LI>Stock template – 8 µl
		</LI><LI>2x MM – 100 µl
		</LI><LI>ddH2O – 84 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR (23/9, 12 AM)
	</LI><LI>Ran products on 1% agarose SB gel (23/9, 11 PM)
</LI></P><H2>23/9/17</H2><P>Pooling PCR (set 2)
<UL><LI>Sai - PCR 4, Sharath PCR 3
	</LI><LI>Made reaction mixture of 1000 µl per template , for splitting into 20 * 50 µl reactions
	</LI><LI> PCR reaction mixture
	</LI><LI>FP – 20µl
		</LI><LI>RP – 20µl
		</LI><LI>Stock template – 40 µl
		</LI><LI>2x MM – 500 µl
		</LI><LI>ddH2O – 420 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR (24/9 , 2:00 AM) and pooled products
	</LI><LI>Ran products on 1% agarose SB gel (24/9, 4:00 AM)
</LI></P><H2>24/9/17</H2><P>Pooling PCR 1
<UL><LI>Made reaction mixture of 1000 µl per template , for splitting into 20 * 50 µl reactions
	</LI><LI> PCR reaction mixture
	</LI><LI>FP – 20µl
		</LI><LI>RP – 20µl
		</LI><LI>Stock template – 40 µl
		</LI><LI>2x MM – 500 µl
		</LI><LI>ddH2O – 420 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR (24/9 , 20:00) and pooled products
	</LI><LI>Ran products on 1% agarose SB gel (25/9, 00:30)
</LI></P><H2>26/9/17</H2><P> Phenol chloroform extraction of the PCR products 1,3,4 was done</P><P>4 transformants from the pathced paltes were inoculated to make glycerol stocks (3M3, 15B4, 16K8, 24K8 in 5 ml LB). 10M plasmid was transformed using heat shock.
</P><P>Optimizing PCR 2
<UL><LI>Made reaction mixture of 160 ul per template , for splitting into 8 * 20 ul reactions
	</LI><LI> PCR reaction mixture (Q5)
	</LI><LI>FP – 3.2µl
		</LI><LI>RP – 3.2µl
		</LI><LI>Stock template – 6.4 µl
		</LI><LI>2x MM – 80 µl
		</LI><LI>ddH2O – 67.2 µl
	</LI></UL><LI> PCR reaction mixture (Phu)
	</LI><LI>FP – 8µl
		</LI><LI>RP – 8µl
		</LI><LI>dNTP - 3.2µl
		</LI><LI>Stock template – 0.4 µl
		</LI><LI>Buffer - 32 µl
		</LI><LI> Phu pol - 1.6 µl
		</LI><LI>ddH2O – 106.8 µl
	</LI><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR (26/9, 11:50 PM)
	</LI><LI>Ran products on 1% agarose SB gel (27/9, 3 AM)
</LI></P><P>Interlab Day 1:
	<UL><LI>Prepare 8 plates of Cmp LB agar plates
		</LI><LI>Plate the following
		</LI><LI>Positive control –B
			</LI><LI>Negative control – D
			</LI><LI>Device 1 – F
			</LI><LI>Device 2 – H
			</LI><LI>Device 3 – J
			</LI><LI>Device 4 – L
			</LI><LI>Device 5 – N
			</LI><LI>Device 6 - P
		</LI></UL></P><H2>27/9/17</H2><P>Biotinylation of gas vesicles was carried out. 
</P><P>Pooling PCR 2
<UL><LI>Made reaction mixture of 1000 µl per template , for splitting into 20 * 50 µl reactions
	</LI><LI> PCR reaction mixture (Phu)
	</LI><LI>FP – 50µl
		</LI><LI>RP – 50µl
		</LI><LI>dNTP - 20µl
		</LI><LI>Stock template – 2.5 µl
		</LI><LI>Buffer - 200 µl
		</LI><LI> Phu pol - 10 µl
		</LI><LI>ddH2O – 667.5 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR ( 27/9, 9:30 AM)
	</LI><LI>Products were pooled (27/9 , 15:00)
	</LI><LI>No DNA pellet formed after purification and centrifugation. Thus PCR Failed
</LI></P><P> Interlab Day 2 protocols were carried out </P><UL><LI>Autoclave falcons + 500 LB broth
		</LI><LI>From each of the 8 plates , inoculate 2 colonies into 5 ml LB + cmp solutions
		</LI><LI>Incubate overnight
	</LI></UL><H2>28/9/17</H2><P>Pooling PCR 2 (attempt 2)
<UL><LI>Made reaction mixture of 1000 µl per template , for splitting into 20 * 50 µl reactions
	</LI><LI> PCR reaction mixture (Phu)
	</LI><LI>FP – 50µl
		</LI><LI>RP – 50µl
		</LI><LI>dNTP - 20µl
		</LI><LI>Stock template – 2 µl
		</LI><LI>Buffer - 200 µl
		</LI><LI> Phu pol - 10 µl
		</LI><LI>ddH2O – 668 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR at 7 PM
	</LI><LI>Ran PCR gel (1% agarose, SB) at 9:30 PM
</LI></P><P>Interlab day 3 protocol was carried out</P><UL><LI>Prepare 16 falcons with 12 ml LB+cmp media
		</LI><LI>Take OD600 of each of the above cultures. Calculate the volume required to make the OD600 of the 12 ml culture 0.02 (use the Interlab spreadsheet for that)
		</LI><LI>Aliquot required volumes of each culture into the falcons. Label properly.
		</LI><LI>At t=0 (1130), take out 500 ul from each culture and store on ice.
		</LI><LI>Incubate and shake the cultures at 37 degrees.
		</LI><LI>At t=2 (1330) , t=4 (1530) and t=6 (1730) aliquot 500 ul from the cultures. Store on ice.
	</LI></UL><H2>29/9/17</H2><P>Optimizing PCR 5
<UL><LI>Made reaction mixture of 200 ul per template , for splitting into 8 * 25 ul reactions
	</LI><LI> PCR reaction mixture (Q5)
	</LI><LI>FP – 4µl
		</LI><LI>RP – 4µl
		</LI><LI>Stock template – 80 µl
		</LI><LI> 2x MM - 100µl
		</LI><LI>ddH2O – 12 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR 6:10 PM
	</LI><LI>Ran products on 1.2% agarose TAE gel 8:20 PM (all temp gave bands)
</LI></P><P>Optimizing PCR 6
<UL><LI>Made reaction mixture of 160 µl per template , for splitting into 8 * 20 µl reactions
	</LI><LI> PCR reaction mixture (Phu)
	</LI><LI>FP – 8µl
		</LI><LI>RP5 – 7µl
		</LI><LI>RP6 - 1µl
		</LI><LI>dNTP - 3.2µl
		</LI><LI>Stock template – 0.4 µl
		</LI><LI>Buffer - 32 µl
		</LI><LI> Phu pol - 1.6 µl
		</LI><LI>ddH2O – 106.8 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR 6
	</LI><LI>Ran products on 1% agarose SB gel
</LI></P><P>Interlab Day 4 protocol was carried out</P><UL><LI>Transfer the collected aliquots into a 96 well plates
		</LI><LI>Take OD600 and fluorescence at 501 (excitation) and 511 (emission)
	</LI></UL><H2>30/9/17</H2><P>Restriction digestion of PCR product 1 was carried out.
	<UL><LI>Single digest with HindIII
		</LI><LI>Single digest with NcoI
		</LI><LI>Double digest with both
		</LI><LI>Negative control
		</LI><LI>Uncut plasmids</LI><LI>PCR product 1
	</LI></UL></P><P>Bulk restriction digestions were done
	<UL><LI> For PCR product 1: HindIII and NcoI
		</LI><LI> For PCR product 3: HindII and BamH1
		</LI><LI> For PCR product 4: NheI and BamHI
	</LI></UL></P><P>Gel electrophoresis of the digested products was done </P><H2>2/10/17</H2><P>Biotinylation and floatation assay experiments were carried out</P><P>The enzymes were tested out uning double digestion (NcoI + Nhei), (BamHI + NheI), (HindIII + NheI). Pooling of digested PCR product 3 was done (HindIII + BamHI)</P><P>Optimizing PCR 2
<UL><LI>Made reaction mixture of 160 µl per template , for splitting into 8 * 20 µl reactions
	</LI><LI> PCR reaction mixture (Phu)
	</LI><LI>FP – 8µl
		</LI><LI>RP – 8µl
		</LI><LI>dNTP - 3.2µl
		</LI><LI>Stock template – 60 µl
		</LI><LI>Buffer - 32 µl
		</LI><LI> Phu pol - 1.6 µl
		</LI><LI>ddH2O – 47.2 µl
	</LI></UL><LI> PCR reaction mixture (Control)
	</LI><LI>VF2 – 2µl
		</LI><LI>VR – 2µl
		</LI><LI>dNTP - 0.8µl
		</LI><LI>Stock template – 15 µl
		</LI><LI>Buffer - 8 µl
		</LI><LI> Phu pol - 0.4 µl
		</LI><LI>ddH2O – 11.8 µl
	</LI><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR at 6:00 PM
	</LI><LI>Ran products on 1% agarose SB gel at 10:30 PM
</LI></P><H2>05/10/17</H2><P> Purification of PCR products 2 and 6 was carried out. The pellets were small. Corresponding nanodrop readings were then taken</P><H2>06/10/17</H2><P>Checking restriction enzymes for 2,4,6. Purifying PCR2 product and amplifying was done. 
	<UL><LI> 2: Nco1 and NheI
		</LI><LI> 4: BamHI and NheI
		</LI><LI> 6: HindIII and NheI
	</LI></UL></P><H2>8/10/17</H2><P>Bulk digestions were carried out	
	<UL><LI>PCR product 1: HIndIII, NcoI (NEB 2-1)
		</LI><LI>PCR product 2: NcoI, NheI (NEB CutSmart)
		</LI><LI>PCR rpoduct 3: HindIII, BamHI (NEB 3.1)
		</LI><LI>PCR product 4: BamHI (HF), NheI (HF) (NEB CutSmart)
		</LI><LI>PCR product 6: HindIII, NheI (HF) (NEB 2.1)
	</LI></UL>
	all digestions were completed, gel run with 5ul after making upto 500ul. Nanodrop readings were then taken
</P><P> Ligation of (1,2) and (3,4) was done using T4 ligase.
	<UL><LI>1A+2C 1ug each
		</LI><LI>3C+4A 1ug each
	</LI></UL>
	The ligations were successful
</P><P>Optimizing PCR 8
<UL><LI>Made reaction mixture of 200 µl per template , for splitting into 8 * 25 µl reactions
	</LI><LI> PCR reaction mixture (Phu)
	</LI><LI>FP – 4µl
		</LI><LI>RP – 4µl
		</LI><LI>Stock template – 8 µl
		</LI><LI>2x MM - 100 µl
		</LI><LI>ddH2O – 84 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR 11:30 PM
	</LI><LI>Ran products on 1% TAE gel (1:30 AM, 9/10)
</LI></P><H2>9/10/17</H2><P> Ligation of (1A+2C) and (3C+4A) was carried out. Ligation of (1A+2C) and 6A was carried out. Both using T4 DNA ligase.
</P><P>Pooling PCR 8
<UL><LI>Made reaction mixture of 1000 µl per template , for splitting into 20 * 50 µl reactions
	</LI><LI> PCR reaction mixture (Phu)
	</LI><LI>FP – 50µl
		</LI><LI>RP – 50µl
		</LI><LI>dNTP - 20µl
		</LI><LI>Stock template – 40 µl
		</LI><LI>Buffer - 200 µl
		</LI><LI> Q5 pol - 10 µl
		</LI><LI>ddH2O – 630 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR 3:30 AM
	</LI><LI>Ran PCR gel (1% agarose, SB)
</LI></P><H2>10/10/17</H2><P>Growth curve STW wt-14028 was carried out</P><H2>11/10/17</H2><P>Colony PCR of the transformants was carried out (DC lab)</P><P>Growth curve wt-14028 was carried out</P><H2>12/10/17</H2><P>Nanodrop readings of all the digested products were taken annd noted down.</P><P> Digestions were setup
	<UL><LI> 4x D1 - PCR 1.2 - NEB 2.1, HINDIII, NcoI
		</LI><LI> 3x D2 - PCR 2 - NEB CutSmart, NcoI, NheI
		</LI><LI> 4x D3 - PCR 3 - NEB 3.1, HindIII, BamHI
		</LI><LI> 1x D1 - PCR 1 - NEB 2.1, HindIII, NcoI
		</LI><LI> 4x D4 - PCR 4.1 - NEB CutSmart, BamHi HF, NheI HF
		</LI><LI> 4x D6 - PCR 6 - NEB  2.1, HindIII, NehI
		</LI><LI> 4x DO1 - Oligo1 - NEB 2.1, HindIII, NdeI
		</LI><LI> 4x Do2 - Oligo2 - NEB CutSmart, BamH1 HF, Nhe1 HF
	</LI></UL></P><P>Pooling PCR 8
<UL><LI>Made reaction mixture of 500 µl per template , for splitting into 10 * 50 µl reactions
	</LI><LI> PCR reaction mixture (Phu)
	</LI><LI>FP – 25µl
		</LI><LI>RP – 25µl
		</LI><LI>dNTP - 10µl
		</LI><LI>Stock template – 20 µl
		</LI><LI>Buffer - 100 µl
		</LI><LI> Q5 pol - 5 µl
		</LI><LI>ddH2O – 315 µl
	</LI></UL><LI>Template stock – 100 ng/µl
	</LI><LI>Ran PCR at 5:00 PM
	</LI><LI>1% agarose TAE gel run at 10:00 PM
</LI></P><H2>13/10/17</H2><P> Colony PCR of the mCherry transformants was carried out</P><P> All negative. sfGFP gave a band at 1 kB. A, B, C gave bands at 350 bp</P><H2>14/10/17</H2><P>Gel purifications of the digests were carried out. The respective nanodrop readings were etaken and noted down. 
	
</P><H2>16/10/17</H2><P>Purified digested PCR 8 product but final concentration is only 25.1 ng/ul (9ul left) -&gt; cannot set up ligation </P><P>Transformed (1+2+3+4), (1+2+6)</P><P>Inoculated 3M3, 15B4, 23K for miniprep tomorrow</P><H2>17/10/17</H2><P>Tranformant colonies seen on (1+2+3+4) plate, none on (1+2+6)</P><P>Miniprepped 2M, 15B, and 24K. Performing colony PCR of tranformants in the other lab.Nanodrop readings were taken too. </P><P>Digestions of 3M (T7 promoter + RBS) - S and P, 15B (Spycatcher) - X and P, 24K (TT Terminator) - E and X were carried out.</P><P>Ligations were setup
		<UL><LI> L1 - D1g, D2g, D3g, D4g, T4 ligase buffer, T4 ligase
			</LI><LI> L2 - D1g, D2g, D6g, T4 buffer, T4 ligase
		</LI></UL></P><H2>18/10/17</H2><P> Repeating digestion
	<UL><LI> D3M - 3M, SpeI, PstI, CS, MQ
		</LI><LI> D15B - 15B, XbaI, PstI, CS, MQ
		</LI><LI> D24K - 24K, EcoRI HF, XbaI, CS, MQ
	</LI></UL></P><H2>19/10/17</H2><P> Ran 60 colony PCRs (30 for sfGFP-Spycatcher transformants, 30 for 6xHis-mCherry transformants); patched colonies onto fresh plates. Then ran gel to check bands (expected 1.56kB for sfGFP-SC. Incubated B11 and D26 in duplicate for plasmid isolation.</P><H2>20/10/17</H2><P>Plasmids were isolated from B11 and D26 according to miniprep protocol; nanodrop readings taken</P><P> Set up diagnostic digests to verify plasmid size annd ligation </P><P> Single digests
		<UL><LI>B11 plasmid 1 - NEB3.1, PstI
			</LI><LI>B11 plasmid 2, NEB 3.1, PstI
			</LI><LI>D26 plasmid, NEB 3.1, PstI
		</LI></UL></P><P> Diagnostic digests
		<UL><LI>B11 plasmid 1, NEB 3.1, PstI, BamHI, HindIII (2x)
			</LI><LI>B11 plasmid 2, NEB 3.1, PstI, BamHI, HandIII (2x) 
			</LI><LI>D26 plasmid 3, NEB 3.1, PstI, HindIII (2x), NcoI
		</LI></UL></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>