<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IISc-Bangalore_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IISc-Bangalore/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="contentWrapper"><DIV style="position: fixed; width: 100%; top: 16px; left: 0; z-index: 1;"><DIV class="navbar"><DIV class="dropdown"><DIV class="dropbtn">COMMUNITY</DIV><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:IISc-Bangalore/HP/Silver">Human Practices</A><A href="https://2017.igem.org/Team:IISc-Bangalore/HP/Gold_Integrated">Integrated Practices</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Engagement">Public Engagement</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Safety">Safety</A></DIV></DIV><DIV class="dropdown"><DIV class="dropbtn">HARDWARE</DIV><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware">GCODe</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware/GCODe-Mini">Mini</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware/GCODe-Pro">Pro</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware/Results">Results</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware/Notebook">Hardware Evolution</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Hardware/Documentation">Build Your Own</A></DIV></DIV><DIV class="dropdown"><DIV class="dropbtn">LAB</DIV><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:IISc-Bangalore/Assembly">Assembly</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Results">Results</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Demonstrate">Demonstrate</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Notebook">Notebook</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Protocols">Protocols</A><A href="https://2017.igem.org/Team:IISc-Bangalore/InterLab">InterLab</A></DIV></DIV><DIV class="dropdown"><DIV class="dropbtn">PROJECT</DIV><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:IISc-Bangalore/Description">iFLOAT</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Model">Model</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Experiments">Experiments</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Parts">Parts</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Design">Design</A></DIV></DIV><DIV class="dropdown"><DIV class="dropbtn">ABOUT</DIV><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:IISc-Bangalore/Team">Team</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Philosophy">Philosophy</A><A href="https://2017.igem.org/Team:IISc-Bangalore/Attributions">Attributions</A></DIV></DIV><DIV class="dropdown"><A href="https://2017.igem.org/Team:IISc-Bangalore">HOME</A></DIV></DIV></DIV><DIV id="pagefooter" class="clearfix"><DIV id="bigfooter" class="clearfix"><DIV id="footersponsors"><TABLE style="margin: auto"><TBODY><TR><TH colspan="6">Sponsors</TH></TR></TBODY></TABLE></DIV><DIV id="footercontact"><TABLE style="margin: auto"><TBODY><TR><TH colspan="6">Contact</TH></TR><TR><TD> </TD></TR><TR><TD> </TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><OL id="inPageNav"><LI><A href="#t7-expression-system">T7 expression system</A></LI><LI><A href="#sfgfp-spycatcher">sfGFP-SpyCatcher</A></LI><LI><A href="#mcherry-spytag">mCherry-SpyTag</A></LI><LI><A href="#gvpc">GvpC</A></LI><LI><A href="#other">Miscellaneous</A></LI></OL><DIV id="contentMain"><H1 id="t7-expression-system">T7 expression backbone</H1><P>Aggregating gas vesicles using SpyCatcher-SpyTag binding involves protein overexpression, and no system is better for protein expression than <I>E. coli</I> BL21 (DE3) as its <I>lon</I> and <I>ompT</I> protease deficiency yields a huge amount of protein.</P><P>BL21 (DE3) is a lysogenic strain that has the T7 RNA polymerase gene integrated into its genome under the lac operon; adding IPTG induces expression of T7 RNA polymerase, which recognizes the T7 promoter sequence. Any gene inserted downstream of the T7 promoter can thus be expressed.</P><P>Using <A href="http://parts.igem.org/Part:BBa_K525998">BBa_K525998</A> (T7 promoter+RBS) and <A href="http://parts.igem.org/Part:BBa_K731721">BBa_K731721</A> (T7 terminator), we have designed a T7 expression backbone that can be used to assemble and express fusion proteins easily.</P><H2>Choice of BioBricks</H2><P><A href="http://parts.igem.org/Part:BBa_K525998">BBa_K525998</A> (T7 promoter+RBS) was chosen as the strong RBS <A href="http://parts.igem.org/Part:BBa_B0034">B0034</A> used allows for maximal protein expression. <A href="http://parts.igem.org/Part:BBa_K731721">BBa_K731721</A> (T7 terminator) was chosen instead of the standard <A href="">B0015</A> double terminator as its <I>in vivo</I> termination efficiency is greater, as characterized by <A href="http://parts.igem.org/Part:BBa_K731700">BBa_K731700</A>.</P><H2>Our First Modification — <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2319001">BBa_K2319001</A> (HindIII+ATG+AgeI scar)</H2><P>A HindIII restriction site, a start codon and an AgeI restriction site (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2319001">BBa_K2319001</A>) are added immediately downstream of the T7 promoter+RBS. A number of design considerations motivated the choice of these restriction sites. The HindIII site (A\AGCTT) — sandwiched between the RBS and the start codon — has a sequence very similar to the optimal sequence predicted by the <A href="https://static.igem.org/mediawiki/parts/d/d1/Ribologo-small.gif">sequence logo</A> of <I>E. coli</I> ribosome binding sites. In fact, the HindIII sequence is closer to the optimal sequence than the typical 5'-TACTAG-3' mixed SpeI-XbaI restriction site formed by BioBrick assembly, improving the initial ribosome-mRNA binding and thereby increasing the rate of translation.</P><FIGURE><FIGURECAPTION><B>Figure 1</B>: the amino acids glycine (left), serine (center) and threonine (right)
</FIGURECAPTION></FIGURE><P>The AgeI site was chosen to simplify assembly of fusion proteins in this backbone: by inserting a protein coding sequence at the N-terminus of the existing protein using the HindIII and AgeI sites, a fusion protein can be formed with a benign scar. The AgeI site (A\CCGGT) is translated in-frame to Thr-Gly, amino acids commonly used in linker sequences for fusion proteins. Threonine's hydroxyl group makes it hydrophilic — allowing stabilizing interactions with the aqueous cellular environment — while glycine's small size makes the linker more flexible, allowing both protein domains to fold independently. In addition, the AgeI site is useful if the user wishes to transfer an RFC25-compatible fusion protein (Freiburg format) into our expression system.</P><H2>Our Second Modification — <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2319004">BBa_K2319004</A> (TAAG)</H2><P>A stop codon (TAA) and the nucleotide G are added immediately upstream of <A href="http://parts.igem.org/Part:BBa_K731721">BBa_K731721</A> (T7 terminator). This extra stop codon (assuming the fusion protein sequence has its own) ensures that translation is halted and prevents any translational read-through. The extra nucleotide G is added for a more subtle purpose: when placed just before the T7 terminator sequence (5'-CTAGC...TTTTG-3'), it forms the NheI restriction site (G\CTAGC). This allows any fusion protein to be inserted into our expression backbone using the HindIII and NheI sites.</P><H2>Using the T7 expression backbone</H2><FIGURE><FIGURECAPTION><B>Figure 2</B>: T7 expression backbone showing HindIII, AgeI and NheI sites
</FIGURECAPTION></FIGURE><H3>Expressing any protein of interest</H3><P>This T7 expression backbone can be used to express any protein if its coding sequence (with a start codon) is inserted using the HindIII and NheI sites. These sites can be added to the coding sequence using PCR with primers having 5'-overhangs.</P><H3>Fusing a protein domain at the N-terminus of an existing protein</H3><P>By inserting the coding sequence of a protein domain (including the start codon) using the HindIII and AgeI sites into the T7 expression backbone (which already contains a protein coding sequence), an N-terminal fusion can be performed.</P><H1 id="sfgfp-spycatcher">sfGFP-SpyCatcher</H1><P>Using <A href="http://parts.igem.org/Part:BBa_K1321337">BBa_K1321337</A> (sfGFP in Freiburg format) and <A href="http://parts.igem.org/Part:BBa_K731721">BBa_K1650037</A> (SpyCatcher), we make the fusion protein sfGFP-SpyCatcher to be expressed on the gas vesicle surface after fusion to GvpC.</P><H2>Choice of BioBricks</H2><P><A href="http://parts.igem.org/Part:BBa_K1321337">BBa_K1321337</A> (sfGFP in Freiburg format) was chosen because superfolder GFP exhibits intense bright green fluorescence (making it easy to assay) and folds easily into an extremely stable structure — its half-life for unfolding at room temperature is estimated at 28 years! <A href="http://parts.igem.org/Part:BBa_K1650037">BBa_K1650037</A> (SpyCatcher) was chosen as it possesses a 6xHis tag near the N-terminus, which allows the fusion protein to be easily purified using a Ni-NTA column.</P><H2>Our Modification — <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K23190032">BBa_K2319002</A> (GGSGSGSS linker)</H2><P>Between the sfGFP and SpyCatcher protein domains, we have inserted a short (8 aa), flexible, hydrophilic linker comprising Gly and Ser residues that allows sfGFP and SpyCatcher to fold independently of each other. Serine, like threonine, has a hydroxyl group that contributes to stability by interacting with the aqueous cellular environment. Glycine, again due to its small size, maintains flexibility of the linker.</P><P>This linker region also contains a BamHI restriction site (G\GATCC) whose in-frame translation is Gly-Ser. This BamHI site is used to link these two protein domains during assembly of the fusion construct.</P><H1 id="mcherry-spytag">mCherry-SpyTag</H1><P>Using <A href="http://parts.igem.org/Part:BBa_J18932">BBa_J18932</A> (mCherry RFP) and two well-designed oligos, we plan to produce mCherry-SpyTag.</P><H2>Choice of BioBricks</H2><P><A href="http://parts.igem.org/Part:BBa_J18932">BBa_J18932</A> (mCherry RFP) has an interesting flaw: an internal ATG near the N-terminus has a RBS-like sequence preceding it; this hidden translation start site leads to ~50% truncation of the produced mCherry protein!</P><H2>Our First Modification — Improved mCherry </H2><P>Using an <I>in silico</I> analysis of RBS strengths using an online RBS Calculator, we modified the nucleotide sequence preceding the translation start site to become a far weaker RBS while maintaining the same amino acid sequence. This inhibits translation initiation at that position by almost 75% (predicted) and so reduces the truncation of the protein.</P><PRE align="middle">BBa_J18932_mCherry      1 GTGAGCAAAGGCGAGGAAGATAACATG     27
                   |||...|||||.||.||||||||.|||
Improved_mCherry        1 GTGTCTAAAGGTGAAGAAGATAATATG     27</PRE><P>By exploiting a natural NdeI site (CA\TATG) occurring right after the modified sequence, we insert <A href="http://parts.igem.org/Part:BBa_K2319006">BBa_K2319006</A> comprising a HindIII site (for insertion into the T7 expression backbone), a start codon (ATG), and a 6xHis-tag (for easy Ni-NTA column-based protein purification), and the modified sequence preceding the hidden translation start site.</P><H2>Our Second Modification — SpyTag Linker</H2><P>Using another oligo, we insert <A href="http://parts.igem.org/Part:BBa_K2319008">BBa_K2319008</A> comprising a BamHI site (for insertion after the mCherry sequence), a GSGGGGS linker (for independent folding) and the SpyTag peptide sequence AHIVMVDAYKPTK. By doing this, we add SpyTag functionality to the mCherry protein, allowing it to bind to our sfGFP-SpyCatcher.</P><H1 id="gvpc">GvpC fusion</H1><P>After assembling our sfGFP-SpyCatcher and mCherry-SpyTag fusion proteins, we have to express them on the gas vesicle surface: this is where our AgeI sites come in handy — by using HindIII and AgeI, we can insert the <I>gvpC</I> sequence into this expression backbone to make the GvpC fusion proteins. Our project involves gas vesicles extracted from <I>Anabaena flos-aquae</I> and <I>Halobacterium salinarum</I> NRC-1 — we need the <I>gvpC</I> genes from both these organisms, but as always, there are design considerations to keep in mind.</P><H2>Our Modification — GvpC of <I>Halobacterium salinarum</I> NRC-1</H2><P><I>H. salinarum</I> is a halophile that tolerates hypersaline environments with ease — using a &quot;salt-in&quot; strategy, its cellular environment has evolved to be hypersaline itself! The vast majority of proteins synthesized by H. salinarum have negatively-charged acidic side chains and require 4-5 M salt concentrations to function. GvpC is one such protein. GvpC contains seven internal repeats of the domain that interlocks between GvpA &quot;ribs&quot; and strengthens the gas vesicle, followed by an acidic tail at the C-terminus which stabilizes the protein through effective solvation.</P><PRE><B>GvpC protein sequence from <I>Halobacterium salinarum</I> NRC-1, showing seven internal repeats and acidic tail</B>

                         MSVTDKRDEMSTARDKFAESQQEFESYADEFAADITAKQDDVSDLVDAITDFQAEMTNTT
                                                |  | |||||       |     | |   |       
                                              DAFHTYGDEFAAEVDHLRADIDAQRDVIREMQ       
                                              |||  | | ||        ||      |           
                                              DAFEAYADIFATDIADKQ-DIGNLLAAIEALRTEMNSTH
                                               ||||||| || | |    ||  | |||     |     
                                              GAFEAYADDFAADVAALR-DISDLVAAIDDFQEEFIAVQ
                                               ||  || || |        |  | ||| |    | |  
                                              DAFDNYAGDFDAE-------IDQLHAAIADQHDSFDATA
                                              |||  |   |                              
                                              DAFAEYRDEFYRIEVEALLEAINDFQQDIGDFRAEFETTE
                                              |||      ||  |  |   |                  |
                                              DAFVAFARDFYGHEITAEEGAAEAEAEPVEADADVEAEAE*
                                             #VSPDEAGGESAGTEEEETEPAEVETAAPEVEGSPADTADE
                                              AEDTEAEEETEEEAPEDMVQCRVCGEYYQAITEPHLQTHD
                                              MTIQEYRDEYGEDVPLRPDDKT

* denotes the truncation site (denoted C3 truncation site)
# denotes the start of the acidic tail (note the abundance of aspartate and glutamate residues)
Adapted from DasSarma et al (2013)
</PRE><P>For our proposed fusions, this is a problem: haloarchaeal gas vesicles &quot;shrug off&quot; their surface GvpC if high salt concentrations are not maintained. As a result, we removed this acidic tail that destabilizes the GvpC-gas vesicle binding at low salt concentrations, and used this truncated GvpC for our fusions.</P><H2>Our Modification — (GGGGS)2 linker + AgeI</H2><P>Our second modification to both <I>gvpC</I> sequences is an addition of a C-terminal (GGGGS)2 linker (for proper folding of the GvpC domain) and an AgeI site (for insertion into our sfGFP-SpyCatcher and mCherry-SpyTag backbones).</P><H1 id="other">Miscellaneous</H1><H2>Codon optimization</H2><P>As far as possible, we have manually codon-optimized the fusion proteins for the amino acids we are adding using our primers — His, Gly, Ser, Ala — using the codon table of <I>E. coli</I> and an online codon usage tool.</P><FIGURE><FIGURECAPTION><B>Figure 3</B>: Codon usage in E. coli genes
</FIGURECAPTION></FIGURE><H2>Choice of restriction enzymes</H2><P>Note that all the restriction sites we have chosen to insert are supplied by NEB as High Fidelity (HF) versions for optimal double digestions and downstream processes in NEB's CutSmart buffer.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>