<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Consort_Alberta_project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Consort Alberta/project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrapper_CON"><DIV id="home_CON"><DIV id="banner_CON"><DIV id="navigation_CON"><UL><LI class="navitaion_right_border_CON"><A href="https://2015.igem.org/Team:Consort_Alberta">Home</A></LI><LI class="navitaion_right_border_CON"><A href="https://2015.igem.org/Team:Consort_Alberta/project">Project</A></LI><LI><A href="https://2015.igem.org/Team:Consort_Alberta/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Consort_Alberta/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Consort_Alberta/Collaborations ">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Consort_Alberta/Modeling">Modelling</A></LI></UL></DIV><LI class="navitaion_right_border_CON"><A href="https://2015.igem.org/Team:Consort_Alberta/team">Team</A></LI><LI class="navitaion_right_border_CON"><A href="https://2015.igem.org/Team:Consort_Alberta/Practices">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Consort_Alberta/Practices">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Consort_Alberta/Practices">Practices</A></LI></DIV><LI class="navitaion_right_border_CON"><A href="https://2015.igem.org/Team:Consort_Alberta/notebook">Notebook</A></LI><LI class="navitaion_right_border_CON"><A href="https://2015.igem.org/Team:Consort_Alberta/safety">Safety</A></LI><LI class="navitaion_right_border_CON"><A href="https://2015.igem.org/Team:Consort_Alberta/Attributions">Attributions</A></LI><LI><A href="contact">Contact Us</A></LI></DIV></DIV><DIV id="con_CON"><H1 class="left_heading_CON">ECOS:</H1><DIV class="heading_to_left_CON"><P>ECOS is a BioBrick designed to detect xylene. Xylene is a carcinogen in crude oil that is closely associated with two other aromatic hydrocarbons, benzene and toluene, that are the more dangerous compounds found in crude oil. In Consort, and most of Alberta, our economy is based almost entirely on agriculture and the oil industry. These two portions of the economy coexist side by side; oil wells are drilled on land adjacent to crops and cattle. This poses an risk to consumer health and to our economy if oil spills occur. While oil companies have strict regulations and protocols that they follow, it is always important that we increase our environmental stewardship. Currently, there isn't a test that can check for contamination on site besides sending a sample off to a lab, which can be expensive, time-consuming, and sometimes inaccurate if the compounds in the sample are left to degrade for too long during transport. In order to properly monitor these sites, farmers and oil companies alike need the tools to do so. This is where we come in. ECOS will provide an on-site test that is cheap, efficient and easy. After talking to many community members that are intimately involved in agriculture  and the oilfield we believe that our project could be used for semi-annual testing of sites, water monitoring and post spill monitoring.</P></DIV><H1 class="left_heading_CON">Our BioBrick:</H1><DIV class="heading_to_left_CON"><P>Our resulting BioBrick consists of two main portions. The first portion of our plasmid will be responsible for producing the protein XylR. This is essential because the bonding of m-xylene undergoes a conformational change when bonded to XylR which then allows it to bind to, and positively regulate, the Pu promoter. At a 4:2 XylR to Xylene ratio the bonding starts the production of our reporter protein. The second portion of our plasmid expresses the indicator protein when in the presence of bonded XylR and m-xylene and essentially allow us to know if m-xylene is present in our soil sample or not. We have had ECOS synthesized by GenScript from pre-existing parts in the registry. ECOS consists of the following parts:</P><P>
J23100 - which is a constituent promoter.
B0034 - the RBS for our XylR gene.
I723017 - the XylR coding region which encodes for the transcriptional regulator XylR protein.
B0015 - the double stop codon for this sequence.
I723020 - this is the Pu promoter.
B0030 - RBS.1 strong.</P><P>We also attached the Reporter AmilCP in the backbone pCB1C3 to give us an output in correspondence to the level of xylene present. This was key to our lab work to ensure that ECOS worked properly and sucessfully detected xylene.</P></DIV><H1 class="left_heading_CON">Our Results:</H1><DIV class="heading_to_left_CON"><P>We were able to successfully create two parts. The first being just ECOS and the second having AmilCP attached. We did a lab on the part including AmilCP and did have positive results! We have successfully ligated AmilCP into the backbone of JM109 with ECOS. We are seeing production of AmilCP in the presence of xylene and in correlation to the concentration of xylene. During the second trial we used different concentrations of xylene to add to our 10mL of LB. We found that this time, leaving them in the incubator for shorter periods of time, we had much more uniform growth. Our output therefore did not have any correlation with the amount of cell growth. We did see very positive results. We had the darkest cells in the 50 uL tube and no dark cells in the negative control. We also were able to detect 50ppm based on volume. We grew cells in 10mL of LB and added different concentrations of xylene to the tubes. The lowest concentration we were able to see results with was .5 microlitres. Or 50ppm based on volume. We attempted to do trials with plates but they simply overgrew and gave us invalid results. We also did a trial with our prototype using a soil sample containing xylene. We detected 100ppm by volume with the prototype. We mixed 350mL of packed soil with 350uL of xylene, then we placed the soil in the upper chamber (the milk jug with a cardboard divider), making sure we didn't contaminate the sample with our hands/gloves. Then we placed our ECOS in the lower chamber (the coffee mug), and secured the entire unit to the shaker table and turning on our double fan system to circulate the air between the xylene in the soil sample and the ECOS in the lower chamber. We left it for 13 hours (4 p.m. to 10 a.m. the next morning).</P><P>For more details please view our <A href="https://static.igem.org/mediawiki/2015/c/ce/Xylenelab.pdf">Lab Write Up</A></P><P>We created a rudimentary prototype of this three-chambered system to test our E. coli and xylene soil samples with. The prototype consists of a 2L milk jug that is divided to contain our soil sample, a double fan system that keeps the air circulating between the milk jug and the mug with our E. coli in it. As outlined in our <A class="nav">Notebook</A>, in order to prepare for our &quot;real life&quot; experiments, we mixed 3.5mL of xylene for 350mL of soil. We then added the soil to our built prototype and added LB and cell cultures from yesterday to the chamber that holds ECOS. We then let it sit overnight, starting at 4 p.m. on the shaker table until 10 a.m. the next morning (a total of 13 hours). We spun down our cells and saw AmilCP being produced. Our prototype was succesful. In the future we will be working on creating more efficient, aesthetically pleasing and user friendly prototypes.</P></DIV><H1 class="left_heading_CON">Design:</H1><DIV class="heading_to_left_CON"><H2><B>Prototype 1:</B></H2><P>Our first prototype involves  trapping our E. coli containing ECOS in the matrix of alginate beads. Alginate is easy to manipulate, it is cheap, light and safe. It would also be extraordinarily simple to operate as you simply place the beads in the soil or water and wait. We did several trials with alginate; experimenting with different protocols to form the beads. We can manipulate the density of alginate's matrix to allow us to entrap our bacteria in the beads. The proteins will still be able to get through the pores so we can see an output. Unfortunately the beads ended up being totally opaque. With our trials with ECOS, we had to spin the cells down to see the AmilCP. You could not detect a color change when the cells were suspended in the LB. With this information, there is no way we would be able to tell if the protein was being produced or not in the alginate beads. However, the alginate's matrix would still allow the xylene to get in, which ensures our beads are safe. We also looked into making beads out of agar. Agar is too porous, though, and would not be able to hold the bacteria within the matrix.</P><H2><B>Prototype 2:</B></H2><P>While this prototype is slightly more complicated, at this point we may have to sacrifice simplicity for something that will give us results. We designed this prototype so that it has a positive pressure source which creates a current for our xylene to travel from our sample, which is in a heated container so that the xylene is vaporized, to our ECOS container in which we bubble it through the E. coli culture to get results. This was a plausible idea, as it would require fairly non-expensive materials and could be maintained by the average business person.</P></DIV><H1 class="left_heading_CON">Our Future Plans:</H1><DIV class="heading_to_left_CON"><P>As AmilCP is hard to see in our cloudy beads, so we plan to look into finding a different reporter which will be easier to work with. Protein Paintbox has many different and vibrant proteins that may work better with ECOS. This is key because we want to ensure our project is as user friendly as possible. We have also discussed improving our prototype to make it as efficient, as aesthetically pleasing and as simple possible. Creating alginate beads that we can safely and efficiently use is also a priority. In the meantime we are looking into improving our current prototype so it is a marketable product. We are also looking into FredSense, they are using electrical conductivity in liquid instead of using the traditional coloured protein. How there system will work is by using a special Enzyme that will break apart a spicific sugar that has been added to the broth, when cleaved this sugar becomes electrically conductive. when you run a voltage sweep through the broth a very large increase in voltage at a spicific point. these enzymes are produced faster and reqire less of them to show a posittive result. This will give us quantifiable evidence and there will be no question of human error involved. This would allow us to use an easier method of testing. We also intend to do more testing to quantify the limits of our detection system. ECOS is a product that can be utilized in our community and in our world. We want to make sure it's as easy as possible for everyday people to use.</P></DIV></DIV><H1 class="left_heading_CON"><DIV id="collaborations">Collaborations:</DIV></H1><DIV class="heading_to_left_CON"><P>This year, we collaborated with Canmore. Our goals was to get the opinion of another team in order to get an impartial and unbiased analysis of each others projects. We wanted to get another opinion on the usefulness of our biobrick along with any concerns about practicality or safety. Below are the write ups down by both teams.</P><P><B>Canmore:</B></P><P>Canmore’s project deals primarily with the issue of hair and feather build up in the water treatment systems. This is clearly a major issue and can use extensive amounts of money and time to remedy. There is an extreme amount of inconvenience to unclog the water treatment systems once they get to this point. Creating a BioBrick to produce keratinases could be an effective and cheap solution to the problem and also in preventing the problem. </P><P>Our concerns for Canmore lie in the prototype and safety. The specific engineering and prototyping of a bioreactor is extremely important. The conditions required to keep the bacteria alive, the amount of keratinases that would need to be produced, and the length of time this process will take are all aspects to look into when designing the bioreactor. Their plan to use the membrane bioreactor is a very good and easily applicable idea but the research as to optimum conditions concerning the production of the gene is incredibly important. Having a viable analysis of the cost, time and resources required to make this something we can implement is society is also an important consideration. The time frame of how long it takes to break down the keratin is also something to look into. The comparison of the time requirements to break down other pollutants in the bioreactor chamber compared with the time to break down keratin. Knowing this will allow a design maximizing potential. </P><P>Other thoughts for them to consider include: what types of bacteria are already being used in the bioreactor? What are they breaking down? Is there a way to genetically modify bacteria already in use to contain the kera or karus gene? This would eliminate the need to introduce another species and readjust conditions while allowing maximum productivity. 
Overall, I believe this project could introduce a valid and much needed solution to the issue of hair build up and could be quite effective, quick and safe. As the bioreactor is not connected to the main water line and is completely contained, contamination of water with E. coli would not be an issue. The proposed prototype also utilizes the current system and technologies used making it as easy as possible for implementation. Canmore has done an excellent job in looking at the practical portions of their project and with more research their project could do a lot of good in our world. (-Sam Davies)</P><P><B>Consort:</B></P><P>This year the iGem team from Consort is continuing with their ECOS project, bacteria designed to detect the presence of aromatic hydrocarbons in order to prevent any harm to the environment. Consort’s ongoing project is a great concept for many reasons. Firstly it is something that is a local problem in their community and especially hits home for some of the team members, which means that they devote themselves fully to the project. Secondly because the outcome of this project could be very helpful for surrounding communities and possibly could hold some wealth in it. Finally because this project could help save countless amounts of crops and livestock from what is a major threat to them.</P><P>The science behind the project is very intriguing as well. The indicator protein is atoned to xylene, which is a very volatile liquid hydrocarbon, which in itself is a good thing to find and remove but it is tied to other dangerous compounds. When these compounds are detected the proteins emit a color indication that would alert the user to the presence of said dangerous compounds. Whilst this all seems simple the BioBrick is not, and they have spent years developing this to perfection to start with their prototypes.</P><P>The prototype and models of prototypes look to be a very effective method of using ECOS. I was able to see one of the prototypes and it looked good even though I was unable to see it in action, this shows great promise for the team in their upcoming endeavors. I can see much success in the future if this team especially if they have a working prototype at the giant Jamboree, I personally wish them the best (-Joshua Lamb)</P></DIV><DIV id="con_CON"><H1 class="left_heading_CON">Modelling:</H1><DIV class="heading_to_left_CON"><P>Making a sensor involves more than just the creation of the basic biological circuit, as there could be many factors that might improve the performance of the system. An example of these could be the strength of the RBS, the spacing between the promoter and the reporter gene, or the copy number of the plasmid used to house the circuit. While we couldn't nearly address all of these points in our project, we did want to look into the optimization of our system. To do this, we turned to mathematical modelling to create a visual representation of our system.</P><P> We worked off of a basic framework published by Koutinas, M., <I>et al.</I>,<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1462-2920.2010.02245.x/abstract">[see here]</A> which was modelling the expression of the Ps promoter with XylR induction. The Ps promoter is the natural promoter found with the xylR gene, and the XylR protein can interact with both it and the Pu promoter we used in our project. Due to similarities between the Ps and Pu promoters, we assumed the deactivation rate of the two components were alike. For remaining values, we simply replaced the values of the Ps promoter with known values of the Pu promoter, keeping constants reported for XylR the same. In the end, we created five equations to represent the action of our system. Our model shows our BioBrick producing the protein XylR, binding with xylene and the relationship to the output of our respective proteins. Each of our formulas are constructed as a rate in which a concentration or output is given of a specific substance in respect to time- otherwise known as a derivative. We will now expand upon the equations we modified and used.</P><P>The first equation above represents the transcription of DNA into the XylR RNA transcript. RNA is the representation of the transcript, with P representing the number of plasmids inside a cell. J23100TC represents the activity of our promoter controlling XylR. ta is the transcription rate of the xylR into RNA, based on the transcription rate of E. coli and the size of XylR, and KRNAdeg represents the degradation rate in the E. coli.</P><P>The second and third equations represent the inactive and active forms of XylR- or the concentration of XylR that has not bonded to xylene (XylRi) and the concentration of XylR that has bonded to xylene (XylRa). RNA is the substitution of our first equation. tr is the translation rate of our RNA into XylR. rXylR is the oligomerization constant of XylR, rR,XylR is the dissociation constant of active XylR, and xyl in the total concentration of xylene. αXylRi accounts for degradation of XylR.</P><P>The fourth equation represents the concentration of xylene present. XylRi and XylRa refer to the concentrations of the inactive and active forms of XylR. rXylR is the oligomerization constant of XylR, rR,XylR is the dissociation constant of inactive XylR and xyl is the total concentration of xylene. αXylRi accounts for the degradation of XylRa, which would release the xylene it held. We multiplied by seconds per hour to get a domain more realistic to our needs.</P><P> The fifth equation represents the output of the pu promoter. PuTC represents the activity of the Pu promoter or the output of protein. αpu is the deactivation rate of the Pu promoter. KXylRa is the activation coefficients of the Pu due to binding and nps,ais the hill coefficient for the interaction. β0 and βPS are the basal and maximal expression, respectively, of the Pu promoter.</P><P> In order to create an output, we used the program <A href="http://www.scilab.org">Scilab</A>. By inputting these equations into code we were able to have the program calculate our estimates and graph the results. In the end our output was around 5.5 mPoPS. This is a logical output and tells us our biobrick will successfully over express the protein. We set the output of our model in PoPS due to our system's flexibility when it comes to our reporter protein. This allows us to quickly add in an extra equation to turn our current output into fluorescence, LacZ output, or pigment production without having to worry about editing other parts of our model. Below is a copy of our code that we had used in our program. The resulting graph is the output of our Pu promoter.</P><P>  While we have yet to apply our modelling framework to the optimization of our sensor, we have created the foundation of a model that can be easily tweaked to test the effect of different variables on sensor performance. Through this we aim to do a sensitivity analysis of the various factors involved in our model in the future so that we can predict what to change for version 2 of our sensor. From there we can return to our model, feeding new data into it to create many future versions of our system, improving performance each time.</P></DIV><DIV id="con_CON"><H1 class="left_heading_CON">References:</H1><DIV class="heading_to_left_CON"><P>-Kandyala, R., Raghavendra, S. and Rajasekharan, S. 2010. Xylene: An overview of its health hazards and preventive measures. Journal of Oral Maxillofacial Pathology. 14(1): 1-5.</P><P>Perez-Martin, J. and de Lorenzo, V. (1996). Identification of the Repressor Subdomain within the Signal Reception Module of the Prokaryotic Enhancer-binding Protein XylR of Pdeudomonas Putida. The Journal of Biological Chemistry.  271, 7899-7902.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>