<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Minnesota_Project_UV_Absorption"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Minnesota/Project/UV_Absorption">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Minnesota/Project/UV_Absorption&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota/Project/UV_Absorption&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota/Project/UV_Absorption&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Minnesota/Project/UV_Absorption" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Minnesota/Project/UV Absorption</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Team:Minnesota - Main Style Template</TITLE><TITLE>Team:Minnesota - Template</TITLE></P><DIV id="ContentAll"><DIV id="SideBarAll"><DIV class="sideNavBar2"><UL><LI><A href="https://2012.igem.org/Team:Minnesota">HOME</A></LI><LI><A href="https://2012.igem.org/Team:Minnesota/Team">TEAM</A></LI><LI><A class="current" href="https://2012.igem.org/Team:Minnesota/Project">PROJECT</A></LI><LI><A href="https://2012.igem.org/Team:Minnesota/Notebook">NOTEBOOK</A></LI><LI><A href="https://2012.igem.org/Team:Minnesota/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2012.igem.org/Team:Minnesota/Outreach">OUTREACH</A></LI><LI><A href="https://2012.igem.org/Team:Minnesota/Safety">SAFETY</A></LI><LI><A href="https://2012.igem.org/Team:Minnesota/Sponsors">SPONSORS</A></LI></UL></DIV><DIV style="position:absolute; top:420px; left:20px;font-family:Verdana;font-size:15;">
Like us on FB and follow us on Twitter!
</DIV></DIV><DIV id="MainBoxAll"><DIV id="MainBoxContent" style="background-color:white;"><DIV style="position:absolute;top:0px;margin-left:50px; width:600px; height:450px; overflow-y:scroll; text-align:left;overflow-x:hidden;"><H1>Synthesizing UV-Protective Compounds in Bacteria</H1><P>Our first idea was to engineer the production of the Micosprine-like Amino Acids (MAAs) in <I>E. coli</I>. These compounds confer UV protection in Anabaena variabilis and are currently used in sunscreen that protects against the entire UVA and UVB spectrum. To obtain these compounds for use in industry, the current method is direct extraction from A. variabilis. To develop an alternate way of producing these compounds, genes coding for enzymes in their production pathways were taken from A. variabilis and cloned into <I>E. coli</I> using BioBrickTM techniques. The result: E. coli that can produce UV protectant compounds, which has several implications. E. coli are cheap, easy to produce, grow quickly, are preferred for use in industry, and can be grown up in giant bioreactors. Therefore the speed and ease at which MAA-producing bacteria can be grown, as well as the amount that can be grown would drastically increase. Additionally, promoters and mechanisms of down-regulation in <I>E. coli</I> are well characterized, making modification and control of the pathway much simpler. These parts could easily be employed in future experiments where increased UV tolerance is desired (for example, in the characterization of a UV-sensitive promoter).  Alternatively, a skin-based, UV-protectant probiotic could eventually be developed if these parts are incorporated into a native, non-pathogenic skim bacterium chassis, such as <I>Staphylococcus epidermidis</I>.  Imagine a “sunscreen” that grows naturally on your skin!</P><P><B>Goal</B>
Clone multiple gene pathways from algal species into <I>E. coli</I> to produce natural UV-protective sunscreening compounds.</P><P><B>Background</B>
Our research has focused on two novel biosynthetic pathways found in two distinct algal species. A pathway ending in the production of two UV-protective compounds, shinorine and mycosporine-glycine, was cloned from <I>Anabaena varibalis</I>. A second pathway leading to the production of the unrelated UV-screening compound scytonemin was cloned from <I>Nostoc punctiforme</I>. Our objective is to develop novel and effective production platforms for these compounds, some of which are currently used in expensive sunscreens and lotions. These compounds are used both for the UV-absorptive properties as well as their role as potent antioxidants.</P><P><B>Figure 1. Biosynthetic pathways of MAAs.</B> a) Shinorine biosynthesis.  DHQS, dehydroquinate synthase; O-MT, O-methyltransferase; ATP-grasp; NRPS, non-ribosomal peptide synthase.  b) Scytonemin biosynthesis.   ScyA (NpR1276), acetolactate synthetase; ScyB (NpR1275), leucine dehydrogenase; ScyC (NpR1274), protein of unknown function.</P><P><B>Methods</B></P><P>Genomic DNA was obtained from the Brett Barney Lab (University of Minnesota) and ATCC for <I> A. variabilis</I> and <I>N. punctiforme</I>, respectively. From here, PCR primer extension was used to clone each individual gene (seven in total) with flanking restriction digest sites to be used for vector cloning. Each gene was cloned into an individual pUCBB BioBrick™ cloning vectors. From here, genes from each individual pathway were cloned onto one pUCBB BioBrick™ vector for each specific pathway.</P><P>Upon successful cloning of each pathway, high-performance liquid chromatography (HPLC) was used to analyze production of mycosporine-glycine, shinorine, and scytonemin. Methods for extraction and analysis of these compounds from liquid E. coli cultures have previously been developed by Balskus and Walsh (2011).</P><P>Additionally, UV-sensitivity screens were performed on E. coli cultures expressing target pathways. A preliminary screen was done using different exposure time points (t=5, 10, 15, 20 minutes) and dilutions. A secondary test was performed with a 104 fold dilution and an exposure time of 20 minutes.  Treated cells were plated and then CFUs were counted.</P><P><B>Parts List</B>
BBa_K814000      dehydroquinate synthase (DHQS) generator 
BBa_K814001    	ATP-grasp (ATPG) generator 
BBa_K814002    	dehydroquinate O-methyltrasferase (O-MT) generator 
BBa_K814003    	shinorine non-ribosomal peptide synthase (NRPS) generator 
BBa_K814004    	mycosporine-glycine biosynthetic pathway 
BBa_K814005    	shinorine biosynthetic pathway 
BBa_K814006    	negative control for mycosporine-like amino acid biosynthesis 
BBa_K814007    	ScyA (acetolactate synthase) generator 
BBa_K814008    	ScyB (leucine dehydrogenase) generator 
BBa_K814009    	ScyC generator 
BBa_K814010    	partial scytonemin biosynthetic pathway, scyCB 
BBa_K814011    	scytonemin biosynthetic pathway, scyBAC </P><P><B>Results</B></P><P><B>Figure 2.  HPLC analysis of E. coli expressing MAA biosynthetic genes.</B> Cell extracts of E. coli transformed with +DOA (DHQS, O-methyl transferase, and ATP-grasp) and +DOAN (DOA, and NRPS) showed production and 4-deoxygadusol (a) and mycosporine-glycine (b). Shinorine was not detected in the +DOAN expressing cells. Samples were compared against a +DA negative control (this construct lacks the ability to produce the 4-deoxygadusol intermediate). Cells expressing +ABC (ScyA, ScyB, ScyC) did not show scytonemin production. Data was compared with previous results from Balskus and Walsh, as a standard could not be obtained. </P><P><B>Figure 3. UV sensitivity screens of E. coli cultures producing MAAs.</B>  Preliminary results for the secondary screen showed increased survival in cells producing mycosporine-glycine when compared with +DO and +DA negative controls. Cells expressing +ABC did not show increased survivability.</P><P><B>Figure 4.  Visual inspection of E. coli cells containing the scytonemin pathway.</B>  Although HPLC and growth tests were negative, cells expressing +ABC were seen to be expressing a deep brown product that could easily be visualized (below). The control is a cell culture expressing a +DA plasmid vector. </P><P><B>Conclusion</B>
Successful transformation and expression of the mycosporine-glycine/shinorine pathway was achieved, and production of mycosporine-glycine was observed via HPLC analysis. This appears to incur resistance to UV-radiation to a certain degree. As we move forward, it will become critical to obtain expression of NRPS, the final gene in this pathway, and production of shinorine. Steps are being taken to optimize of this final step in the biosynthetic pathway is ongoing.</P><P>Although conclusive evidence regarding the production of scytonemin was not found, the initial results are promising. Previous studies on algae described a deep brown secretion in cells that were shown to be producing this compound. This pathway has not yet been fully characterized, and additional genes with unknown functions are present within the native <I>N. punctiforme</I> gene cluster. We are currently looking into cloning and expressing these additional genes, and looking for ways to isolate and analyze potential intermediate products within this pathway.</P><A href="https://2012.igem.org/Team:Minnesota/Project">Click here to return to Projects page.</A></DIV></DIV><DIV id="MainBoxOtherLinks"><P class="otherLinks"><A href="https://2012.igem.org/Main_Page">iGEM Home</A> | <A href="https://igem.org/Team.cgi?id=814">Team Minnesota Info</A> | 
<A href="http://www1.umn.edu/twincities/index.html">University of Minnesota Home</A> |  <A href="https://2012.igem.org/Team:Minnesota/Contact">Contact Us!</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Minnesota/Project/UV_Absorption">http://2012.igem.org/Team:Minnesota/Project/UV_Absorption</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Minnesota/Project/UV_Absorption" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Minnesota/Project/UV_Absorption" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Minnesota/Project/UV_Absorption&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Minnesota/Project/UV_Absorption&amp;oldid=268107" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>