<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUDT_CHINA_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUDT CHINA/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="myMainContainer"><DIV id="menu"><LI><A id="HOME" href="https://2015.igem.org/Team:NUDT_CHINA"><SPAN>HOME</SPAN></A></LI><LI><A id="TEAM" href="https://2015.igem.org/Team:NUDT_CHINA/Team"><SPAN>TEAM</SPAN></A></LI><LI><A id="PROJECT" href="#"><SPAN>PROJECT</SPAN></A></LI><LI><A id="Description" href="https://2015.igem.org/Team:NUDT_CHINA/Description" class="b"><SPAN>Description</SPAN></A></LI><LI><A id="Experiments" href="https://2015.igem.org/Team:NUDT_CHINA/Experiments" class="b"><SPAN>Experiments &amp; Protocols</SPAN></A></LI><LI><A id="Results" href="https://2015.igem.org/Team:NUDT_CHINA/Results" class="b"><SPAN>Results</SPAN></A></LI><LI><A id="Design" href="https://2015.igem.org/Team:NUDT_CHINA/Design" class="b"><SPAN>Design</SPAN></A></LI><LI><A id="PARTS" href="#"><SPAN>PARTS</SPAN></A></LI><LI><A id="Team" href="https://2015.igem.org/Team:NUDT_CHINA/Parts" class="b"><SPAN>Team Parts</SPAN></A></LI><LI><A id="Basic" href="https://2015.igem.org/Team:NUDT_CHINA/Basic_Part" class="b"><SPAN>Basic Parts</SPAN></A></LI><LI><A id="Composite" href="https://2015.igem.org/Team:NUDT_CHINA/Composite_Part" class="b"><SPAN>Composite Parts</SPAN></A></LI><LI><A id="Composite" href="https://2015.igem.org/Team:NUDT_CHINA/Part_Collection" class="b"><SPAN>Part Collection</SPAN></A></LI><LI><A id="NOTEBOOK" href="https://2015.igem.org/Team:NUDT_CHINA/Notebook"><SPAN>NOTEBOOK</SPAN></A></LI><LI><A id="ATTRIBUTIONS" href="https://2015.igem.org/Team:NUDT_CHINA/Attributions"><SPAN>ATTRIBUTIONS</SPAN></A></LI><LI><A id="COLLABORATIONS" href="https://2015.igem.org/Team:NUDT_CHINA/Collaborations"><SPAN>COLLABORATIONS</SPAN></A></LI><LI><A id="HUMAN" href="https://2015.igem.org/Team:NUDT_CHINA/Practices"><SPAN>HUMAN PRACTICES</SPAN></A></LI><LI><A id="SAFETY" href="https://2015.igem.org/Team:NUDT_CHINA/Safety"><SPAN>SAFETY</SPAN></A></LI><LI><A id="MODELING" href="https://2015.igem.org/Team:NUDT_CHINA/Modeling"><SPAN>MODELING</SPAN></A></LI></DIV><DIV id="NUDT_Container" class="clearStyle"><H1>RESULTS</H1><TABLE><TBODY><TR><P style="font-size: 12px; text-align: left; padding: 0px 50px 0px 100px;">TIPS1: The left <B style="color: #d82545">MENU :)</B> links to other NUDT_CHINA website.</P><P style="font-size: 12px; text-align: left; padding: 0px 50px 0px 100px;">TIPS2: The right <B style="color: #d82545">NAVIGATE :)</B> posites on content in this page. (Valid when <B style="color: #d82545">UNFOLDED</B>)</P><P style="font-size: 12px; text-align: left; padding: 0px 50px 0px 100px;">TIPS3: Click <B style="color: #d82545">SUBTITLE</B> to <B style="color: #d82545">FOLD/UNFOLD</B> related content. </P></TR><TR><BUTTON id="slideAll" style="margin: 0px 50px 0px 100px;"> CLICK TO FOLD/UNFOLD ALL</BUTTON></TR></TBODY></TABLE><H2 id="pos1">I.Evaluation of the binding ability of TALE-GFP1/2 to the DNA scaffold.<B style="color: #d82545">(CLICK)</B></H2><DIV class="BLANK_Container" id="slide1"><P>To evaluate whether TALE-GFP1/2 fusion protein can effectively target the binding motifs on plasmid DNA scaffold, the ChIP-PCR analysis was conducted. For this experiment, the plasmids of pSB1C3-Plac-RBS-TALE1-GFP1-Ter-Scaffold1, pSB1C3-Plac-RBS-TALE2-GFP2-Ter-Scaffold2 and pSB1C3-Plac-RBS-TALE3-GFP2-Ter-Scaffold1 were constructed, interpret into <I>E.coli</I> BL21 (DE3), and subsequently induced by IPTG. Bacterial lysis samples were cross-linked in 1% formaldehyde without ultrasonic treatment due to the small size of binding plasmid, and immunoprecipitated with anti-GFP polyclonal antibody. Because the binding motifs of TALEs are containing highly repeated sequences, and their flanking sequences are also homologous to the other parts of the harboring plasmid, the primers used for ChIP-PCR were forward P1 and reverse P2 for GFP1 amplification, and forward P3 and reverse P4 for GFP2 amplification, respectively (Fig. 1A). To optimize the culture temperature upon IPTG induction, TALE1-GFP1-Scaffold1 system was used as representative for the ChIP-PCR assay. The results showed that all the 20 oC, 25 oC and 30 oC culture temperature were suitable for TALE-GFP expression and binding to corresponding scaffold. Among them, we chose 25 oC culture temperature for the subsequent experiments (Fig. 1B). As shown in Fig 1C, a 471 bp and a 251 bp of DNA fragments were amplified from the precipitates of TALE1-GFP1-Scaffold1 and TALE2-GFP2-Scaffold2/TALE3-GFP2-Scaffold1 respectively using anti-GFP antibody. However, the negative control immunoprecipitations using no antibody (beads only) or normal rabbit IgG showed no amplification signal. The amplified fragment was confirmed by sequencing. These results indicate that TALE-GFP1/2 can specifically binds to the corresponding plasmid DNA binding motifs <I>in vivo</I>.</P><P>Figure 1. ChIP-PCR assay of TALE-GFP binding to the scaffold in <I>E.coli</I>. (A) A schematic showing the primers and the plasmid regions tested in ChIP assays. P1/P2 was designed for TALE1-GFP1 ChIP assay, and P3/P4 was used for TALE2-GFP2 and TALE3-GFP2 ChIP assay. (B) Exploration of the optimal temperature of <I>E.coli</I> culture for ChIP-PCR assay. (C) Determination of the binding abilities of TALE1-GFP1, TALE2-GFP2 and TALE3-GFP2 to corresponding DNA scaffolds. Input indicates an aliquot of total DNA. Antibodies used for immunoprecipitation are indicated above the lanes.</P></DIV><H2 id="pos2">II.Split GFP assay to determine the effectiveness of the TALE-DNA scaffold system.<B style="color: #d82545">(CLICK)</B></H2><DIV class="BLANK_Container" id="slide2"><P>The effect of TALE-DNA scaffold system on <I>in vivo</I> compartmentation was examined by the split GFP assay. The TALE1-GFP1/TALE3-GFP2-Scaffold1, TALE1-GFP1/TALE2-GFP2-Scaffold2, TALE1-GFP1/TALE2-GFP2-Scaffold3 (designed as shown in Fig. 2A), TALE1-GFP1/TALE3-GFP2 (no scaffold1 control) and TALE1-GFP1/TALE2-GFP2 (no scaffold2/3 control) plasmids transfected <I>E.coli</I> BL21 (DE3) were cultured and IPTG induced overnight. The groups without IPTG supplement were set as uninduced controls. Subsequently, bacterial samples were harvested and the fluorescence intensity (abbreviated as FI, Ex: 488 nm; Em: 538 nm) were determined by the Fluoroskan Ascent FL (Thermo Scientific). As shown in Fig. 2B, the value of FI/OD600 in TALE1-GFP1/TALE3-GFP2-Scaffold1 group was significantly higher than that of no scaffold1 control. In parallel, the data of FI/OD600 in TALE1-GFP1/TALE2-GFP2-Scaffold2 and TALE1-GFP1/TALE2-GFP2-Scaffold3 groups was significantly higher than that of no scaffold2/3 control. Notably, according to the RT-PCR analysis, the increase of the green fluorescence in scaffold system was not owing to the expression variation of split GFP (Fig. 2C). These findings suggest that TALE-DNA scaffold system might be an efficient device for the compartmentation and ordering of different proteins fused with TALE proteins.</P><P>Figure 2. Evaluation of the TALE-DNA scaffold system by split GFP assay. (A) Schematic representations of the integration manner of split GFP to TALE-DNA scaffold system and their hypothetical binding patterns to DNA motifs. (B) The green fluorescence (Ex: 488 nm; Em: 538 nm) of split GFP was detected after overnight culture of <I>E.coli</I> with or without IPTG induction. Relative fluorescence intensity was calculated with normalization of OD600 value. The relative fluorescence intensity of TALE1-GFP1/TALE3-GFP2 control group was set arbitrarily at 1.0, and the levels of other groups were adjusted correspondingly. This experiment was run in three parallel reactions, and the data represent results obtained from at least three independent experiments. **p &lt; 0.01. (C) RT-PCR analysis for determination of GFP1 and GFP2 expression in different TALE-GFP-scaffold groups. The cDNA sequence of 16S rRNA was amplified as standard.</P></DIV><H2 id="pos3">III.The function of TALE-DNA scaffold system on IAA production<B style="color: #d82545">(CLICK)</B></H2><DIV class="BLANK_Container" id="slide3"><P>To investigate the function of TALE-DNA scaffold system in heterologous metabolic pathway, we use a prototype of IAA synthetic pathway by fusing the IAAM and IAAH enzymes to two different TALEs (TALE1 and TALE2, Fig. 3A and 3B). The TALE1-IAAM/TALE2-IAAH-Scaffold2 and TALE1-IAAM/TALE2-IAAH-Scaffold3 plasmids were transfected to <I>E.coli</I> BL21 (DE3), cultured and then IPTG induced overnight. The TALE1-IAAM/TALE2-IAAH transfected group was used as no scaffold control. By sandwich ELISA analysis, both of the scaffold harboring groups showed significantly higher production of IAA than the no scaffold control group (&gt;3.2 fold of TALE1-IAAM/TALE2-IAAH-Scaffold2 and &gt;2.5 fold of TALE1-IAAM/TALE2-IAAH-Scaffold3, p &lt; 0.01, Fig. 3C). Notably, RT-PCR analysis showed that the increase of IAA production in scaffold system was not owing to the expression variation of IAAM and IAAH (Fig. 3D).  According to these results, it could be demonstrated that the TALE-DNA scaffold system can effectively increase the IAA production through an IAAM-IAAH metabolic pathway. Thus, TALE-DNA scaffold system might be an efficient accelerator to promote the rate of heterologous metabolic pathway in prokaryotic chassis.</P><P>Figure 3. Increase of IAA production by incorporating the IAAM and IAAH into TALE-DNA scaffold system. (A) The IAA production pathway converts L-Trp into IAA through IAAM and IAAH. (B) Schematic representations of the integration manner of IAAM and IAAH to TALE-DNA scaffold system and their hypothetical binding patterns to DNA motifs. (C) The production of IAA was determined by sandwich ELISA after overnight culture of <I>E.coli</I> with IPTG induction. After TMB coloring and reaction termination, the color was measured spectrophotometrically at a wavelength of 450 nm. The concentration of IAA in the samples is then determined by comparing their OD450 to the standard curves. The relative IAA concentration of TALE1-IAAM/TALE2-IAAH no scaffold control group was set arbitrarily at 1.0, and the levels of other groups were adjusted correspondingly. All of the data represent results obtained from at least three independent experiments. **p &lt; 0.01. (D) RT-PCR analysis for determination of IAAM and IAAH expression in different TALE-IAA-scaffold groups. The cDNA sequence of 16S rRNA was amplified as standard.</P></DIV></DIV><DIV id="navigation"><LI><A href="#myMainContainer" class="t"><SPAN>TOP/TIP</SPAN></A></LI><LI><A href="#pos1"><SPAN>I.</SPAN></A></LI><LI><A href="#pos2"><SPAN>II.</SPAN></A></LI><LI><A href="#pos3"><SPAN>III.</SPAN></A></LI></DIV><DIV id="NUDT_Footer"><CENTER><P>Copyright © 2015 NUDT_CHINA | Powered by Nianhao Xie</P></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>