<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUDT_CHINA_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUDT CHINA/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="myMainContainer"><DIV id="menu"><LI><A id="HOME" href="https://2015.igem.org/Team:NUDT_CHINA"><SPAN>HOME</SPAN></A></LI><LI><A id="TEAM" href="https://2015.igem.org/Team:NUDT_CHINA/Team"><SPAN>TEAM</SPAN></A></LI><LI><A id="PROJECT" href="#"><SPAN>PROJECT</SPAN></A></LI><LI><A id="Description" href="https://2015.igem.org/Team:NUDT_CHINA/Description" class="b"><SPAN>Description</SPAN></A></LI><LI><A id="Experiments" href="https://2015.igem.org/Team:NUDT_CHINA/Experiments" class="b"><SPAN>Experiments &amp; Protocols</SPAN></A></LI><LI><A id="Results" href="https://2015.igem.org/Team:NUDT_CHINA/Results" class="b"><SPAN>Results</SPAN></A></LI><LI><A id="Design" href="https://2015.igem.org/Team:NUDT_CHINA/Design" class="b"><SPAN>Design</SPAN></A></LI><LI><A id="PARTS" href="#"><SPAN>PARTS</SPAN></A></LI><LI><A id="Team" href="https://2015.igem.org/Team:NUDT_CHINA/Parts" class="b"><SPAN>Team Parts</SPAN></A></LI><LI><A id="Basic" href="https://2015.igem.org/Team:NUDT_CHINA/Basic_Part" class="b"><SPAN>Basic Parts</SPAN></A></LI><LI><A id="Composite" href="https://2015.igem.org/Team:NUDT_CHINA/Composite_Part" class="b"><SPAN>Composite Parts</SPAN></A></LI><LI><A id="Composite" href="https://2015.igem.org/Team:NUDT_CHINA/Part_Collection" class="b"><SPAN>Part Collection</SPAN></A></LI><LI><A id="NOTEBOOK" href="https://2015.igem.org/Team:NUDT_CHINA/Notebook"><SPAN>NOTEBOOK</SPAN></A></LI><LI><A id="ATTRIBUTIONS" href="https://2015.igem.org/Team:NUDT_CHINA/Attributions"><SPAN>ATTRIBUTIONS</SPAN></A></LI><LI><A id="COLLABORATIONS" href="https://2015.igem.org/Team:NUDT_CHINA/Collaborations"><SPAN>COLLABORATIONS</SPAN></A></LI><LI><A id="HUMAN" href="https://2015.igem.org/Team:NUDT_CHINA/Practices"><SPAN>HUMAN PRACTICES</SPAN></A></LI><LI><A id="SAFETY" href="https://2015.igem.org/Team:NUDT_CHINA/Safety"><SPAN>SAFETY</SPAN></A></LI><LI><A id="MODELING" href="https://2015.igem.org/Team:NUDT_CHINA/Modeling"><SPAN>MODELING</SPAN></A></LI></DIV><DIV id="NUDT_Container" class="clearStyle"><H1>Description: Engineering of a multi-enzymatic reaction accelerator in vivo based on TALE</H1><TABLE><TBODY><TR><P style="font-size: 12px; text-align: left; padding: 0px 50px 0px 100px;">TIPS1: The left <B style="color: #d82545">MENU :)</B> links to other NUDT_CHINA website.</P><P style="font-size: 12px; text-align: left; padding: 0px 50px 0px 100px;">TIPS2: The right <B style="color: #d82545">NAVIGATE :)</B> posites on content in this page. (Valid when <B style="color: #d82545">UNFOLDED</B>)</P><P style="font-size: 12px; text-align: left; padding: 0px 50px 0px 100px;">TIPS3: Click <B style="color: #d82545">SUBTITLE</B> to <B style="color: #d82545">FOLD/UNFOLD</B> related content. </P></TR><TR><BUTTON id="slideAll" style="margin: 0px 50px 0px 100px;"> CLICK TO FOLD/UNFOLD ALL</BUTTON></TR></TBODY></TABLE><H2 id="pos1">I.Abstract<B style="color: #d82545">(CLICK)</B></H2><DIV class="BLANK_Container" id="slide1"><P>For years, prokaryotic cells have been widely applied in synthetic biology and bio-engineering as the host organism. However, lacking of  <I>in vivo</I> compartmentation of the heterologous metabolic pathways, which results in a relevantly low concentration of substrate and enzyme, may cause a low production or productive efficacy, especially when producing through a complex multi-enzymatic cascade. In the current study, we developed a new method to accelerate a multi-enzymatic reaction by integration of a transcription activator-like effector (TALE)-based scaffold system into the bacteria chassis. In this system, we designed two different random DNA sequences as binding motifs, and then incorporated them into the plasmid. Meanwhile, the different TALE proteins, which could specifically target the corresponding binding motifs, were generated by the assembly of repeat-variable di-residue (RVD) part coding sequences. Subsequently, multi-enzymes were fused with these TALE proteins, which could efficiently guide the enzymes around the DNA scaffolds, to enrich the local enzyme concentration, and promote the rate of reaction. The effect of the TALE-based scaffold system were tested by the fragmented GFP study and validated by the heterologous production of ndole-3-acetic acid (IAA), is a plant hormone, through incorporating the TALE-fused tryptophan-2-mono-oxygenase (A.K.A. IAAM) and indoleacetimide hydrolase (A.K.A. IAAH)  in <I>E.coli</I>. To the best of our knowledge, this is the first report using TALE system as scaffolds for the spatial organization of bacterial metabolism.  This technique might provide a powerful way in synthesizing multi-enzymatic reaction programs in prokaryotic chassis for a wide range of application. Mo, its application might be extend to the eukaryotic biofactory as well.</P></DIV><H2 id="pos2">II.Introduction<B style="color: #d82545">(CLICK)</B></H2><DIV class="BLANK_Container" id="slide2"><P>Prokaryotic cells have been widely applied as the host cells in bio-engineering and relevant fields for years due to their great advantages in high-speed growth and high productivities against the Eukaryotic Cells. However, lacking of the compartmentation of the heterologous metabolic pathways, which result in a relevantly low concentration of substrate and enzyme, may cause a low production or productive efficacy, especially when producing through a complex multi-enzymatic cascade. For years, this problem has been bothering researchers in further enlarging the applications of prokaryotic cells in bio-engineering.</P><P>Hence, multiple manners have been designed to accelerate the multi-enzymatic reactions in prokaryotic cells by physically enriching the local concentration of enzymes and substrates in certain areas of the cell. In 2011, Camille et al. demonstrated a method to increase the hydrogen production of a heterogenous hydrogen-producing pathway <I>in vivo</I> with rationally designed RNA assemblies<SUP>[1]</SUP>. More recently, similar methods were applied <I>in vivo</I> on acyl-ACP reductase (AAR) and aldehyde deformylating oxygenase (ADO) <SUP>[2]</SUP>. Nevertheless, the limited corresponding ligadin as well as the fragility of RNA itself is challenging the expansibility and stability of such method, which may constrain its further application. Elsewhere, DNA and proteins have also been tried to be modified as scaffold for similar use <SUP>[3-5]</SUP> .  Unfortunately, it is difficult for the DNA scaffold to assemble as single-stranded loop-stem structure to bind the ligadin <I>in vivo</I>. Meanwhile, the complex structure of the scaffold proteins made it inconvenient for prokaryotic biofactory to correctly fold them, thus limits the application. Therefore, both of these two scaffold methods remained limited to in vitro applications.</P><P>Given the importance of developing an <I>in vivo</I> scaffold system that is natural, highly-stable, expandable and simple, a large body of work sought to establish a new method that provokes efficient interaction among different biomacromolecules <I>in vivo</I>. Notably, the recent developed genome editing technique (such as ZFN, TALEN, CRISPER-CAS9), can exactly meet this requirement and come up with clues for new scaffold systems. Accordingly, a recent report applied CRISPER-CAS9 technique as a scaffold system that concentrates specific enzymes for metaplastic functions<SUP>[6]</SUP>. However, since the binding between different dCas9-fused proteins and gRNA is relevantly nonspecific, the expandability of such method is still limited. TAL effector nuclease (TALEN) technique is based on the specific and direct combination between DNA and proteins, thus gives us a clue that the compartmentation of multi-enzymes can be achieved based on TALE fused enzymes and corresponding DNA scaffold.</P><P>TAL effector (TALE) is a family-III effector in <I>Xanthomonas</I> that helps when they infect various plant species <SUP>[7]</SUP>. Different TALEs share a similar domain structure that enables them to bind the genome of the host cell and act as transcriptional effectors. 1.5 to 33.5 tandem repeats (TAL repeats), each of which can recognize one specific DNA base pair, were determined in the central DNA binding domain of the TALEs  <SUP>[8,9]</SUP>. Each TAL repeat contains 33 to 35 highly conserved amino acids, among which, residues at positions 12 and 13 (also known as RVDs for repeat variable di-residues) confer DNA specificity. This structural characteristic allows the TAL effector being utilized in protein engineering applications. By physically fusing the TAL effector with the cleavage domain of FokI nucleases, TAL effector nucleases (TALENs) can be created. This nucleases were widely applied in Prokaryotic and Eukaryotic Cells. Other methods of engineering TAL effectors acting as transcriptional effectors were also reported.</P><CENTER><P>Fig.1 (A) The basic structure of TAL effectors. (B) 3D structure of the TAL effector bond with DNA (Deng et al., 2012)</P></CENTER><P>Using the DNA-binding characteristic of the TAL effector, we can generate a new method to increase the production of heterogenous multi-enzymatic reactions in Prokaryotic cells by rationally designed TALE proteins fused with specific enzymes and their corresponding DNA sequences (as known as Binding Motifs [BMs]). Here we used Escherichia coli as our chassis. A plasmid backbone was used as the scaffold for those BMs. The same plasmid was used to encode the fusion protein. Thus, enzymes fused with TALE proteins could be gathered around the DNA scaffolds, enrich the local enzyme concentration, and promote the rate of reaction.</P><CENTER><P>Fig.2 A diagrammatic sketch showing the TALE-fused enzymes combine with the scaffold.</P></CENTER></DIV><H2 id="pos3">III.Improvement we made<B style="color: #d82545">(CLICK)</B></H2><DIV class="BLANK_Container" id="slide3"><P>A. <B style="color: #d82545">BBa_I715019</B> and <B style="color: #d82545">BBa_I715020</B></P><H3>Two pairs of primers were designed to extend the use of these parts</H3><P style="padding: 0px 90px">1. Primers to add termination codons on BBa_I715019</P><P style="padding: 0px 90px">In our project, we plan to fuse the N-terminal of GFP (A.K.A. GFP1) to the C-terminal of TALE1 protein (you may visit our wiki for more details). However, the sample of BBa_I715019 provided by the 2015 DNA distribution does not have a termination codon on its 3’ terminal to stop the translation. To fix this, we designed a pair of primers to add a termination codon on the 3’ terminal of BBa_I715019 to further extend its usage.</P><P style="padding: 0px 130px">F-Prime: 5’- GAATTCGCGGCCGCTTCTAGAATGC-3’</P><P style="padding: 0px 130px">R-Prime: 5’- GGACTAGTATTATTGTTTGTCTGCC-3’</P><P style="padding: 0px 90px">2. Primers to remove the additional T base from the BBa_ I715020</P><P style="padding: 0px 90px">When fusing the GFP2 with our TALE2/3, we found the additional T base on the 5’ terminal of the BBa_I715020 annoying because it may cause frameshift mutation if we just use the standard Bio-brick assembly to fuse two enzymes. And that mutation may disable the whole reporter gene.</P><P style="padding: 0px 90px">In order to meet this challenge and make this part easier to use, we designed a pair of prime to remove that additional base and make this part easier to be used.</P><P style="padding: 0px 130px">F-Prime: 5’- CGGAATTCGCGGCCGCTTCTAGAAAGAATGGAATCC-3’</P><P style="padding: 0px 130px">R-Prime: 5’- GGACTAGTTTATTATTTGTATAGTTC -3</P><H3>We tested the function of a split-GFP system built by these parts</H3><P>In our project, after properly engineered those two parts, we fused them with the TAL effectors we designed and tested their function.</P><P>In our experiment, GFP1 was fused with TALE1, GFP2 was fused with TALE2; and SCAF2 (a scaffold where TALE protein can bind) was added into the plasmid. The plasmid was transformed into <I>E.coli</I> BL21(DE3) and cultured in LB with 30mg/ml Chloramphenicol to OD600=0.6, then inducted with 1mM IPTG overnight.</P><P>Fig. 1 Evaluation of the functions of split GFP. The green fluorescence (Ex: 488 nm; Em: 538 nm) of split GFP was detected after overnight culture of <I>E.coli</I> with or without GFP1/2 under the 1mM of IPTG induction. Relative fluorescence intensity was calculated with normalization of OD600 value. The relative fluorescence intensity of S2 control group was set arbitrarily at 1.0, and the levels of other groups were adjusted correspondingly. This experiment was run in three parallel reactions, and the data represent results obtained from at least three independent experiments. *0.01 &lt; p &lt; 0.05.</P><P>The results shows that the existence of GFP1 and GFP2 can observably increase the fluorescent intensity. Which then indicates that these two parts work as expected.</P><P>B. <B style="color: #d82545">BBa_K515100</B></P><P>The second functional prototype of our project this year is about the IAA synthesis pathway. In this prototype, enzyme IAAM and IAAH are fused with different TAL effector so that they can be concentrated around the scaffold. </P><P>Among the registry, BBa_K515100 is the only one available with those two enzymes, but unfortunately, we noticed that this part is a device that contains not only the CDS of those two enzymes but also other parts, and neither of the CDS of those two enzyme were submitted to the registry (they were only registered as BBa_K515000 and K515001). Thus, several improvements were made based on the part K515100 to provide an available version of IAAM and IAAH.</P><H3>Two pairs of primes were designed to decompose this device and gain the CDS of IAAM and IAAH</H3><P style="padding: 0px 90px">1. For IAAM</P><P style="padding: 0px 130px">F-Prime: 5’- GGAATTCGCGGCCGCTTCTAGAGATGTTTGGACCGG-3’</P><P style="padding: 0px 130px">R-Prime: 5’- GCGGCGGACTAGTCTTATTAGTCCCCCAGCG -3’</P><P style="padding: 0px 90px">2. For IAAH</P><P style="padding: 0px 130px">F-Prime: 5’- GGAATTCGCGGCCGCTTCTAGAGATGCGCGAAATG -3’</P><P style="padding: 0px 130px">R-Prime: 5’- GCGGGCGGCGGACTAGTCTTATTAGCCTTTTAACAC -3’</P><H3>Two new bio-bricks were designed based on this part using the primers above</H3><P style="padding: 0px 90px">1. BBa_K1789000</P><P style="padding: 0px 90px">2. BBa_K1789001</P><P>C. <B style="color: #d82545">Project improvement: ZJU_CHINA 2012</B></P><P>In the project of ZJU_CHINA 2012, the IAA producing pathway originates from <I>Pseudomonas savastanoi</I> was engineered into <I>E.coli</I>. They applied their RNA scaffold into that system in order to get a higher IAA productivity. Compared with strain without basic scaffold D0, IAA output with D0 resulted in a 1.4 fold increase (The result was permitted to be used by the team advisor of ZJU_CHINA 2012).</P><P>This year in our project, we made an improvement to that system by changing the RNA scaffold system into our TALE-based scaffold system.</P><P>The result showed a 3.2 fold increase for the TALE1-IAAM/TALE2-IAAH S2 system and a 2.5 fold increase for the TALE1-IAAM/TALE2-IAAH S3 system compared to strain without scaffold S2/S3.</P><P>Please visit our result and discussion page for details.</P></DIV><H2 id="pos4">IV.Reference<B style="color: #d82545">(CLICK)</B></H2><DIV class="BLANK_Container" id="slide4"><P style="padding: 0px auto">[1]C. J. Delebecque, A. B. Lindner, P. A. Silver, F. A. Aldaye, Organization of intracellular reactions with rationally designed RNA assemblies. <I>Science</I><B>333</B>, 470 (Jul 22, 2011).</P><P style="padding: 0px auto">[2]G. Sachdeva, A. Garg, D. Godding, J. C. Way, P. A. Silver, In vivo co-localization of enzymes on RNA scaffolds increases metabolic production in a geometrically dependent manner. <I>Nucleic acids research</I><B>42</B>, 9493 (Aug, 2014).</P><P style="padding: 0px auto">[3]J. Fu <I>et al</I>., Multi-enzyme complexes on DNA scaffolds capable of substrate channelling with an artificial swinging arm. <I>Nature nanotechnology</I><B>9</B>, 531 (Jul, 2014).</P><P style="padding: 0px auto">[4]N. Koga <I>et al.</I>, Principles for designing ideal protein structures. <I>Nature</I><B>491</B>, 222 (Nov 8, 2012).</P><P style="padding: 0px auto">[5]S. A. Heider, N. Wolf, A. Hofemeier, P. Peters-Wendisch, V. F. Wendisch, Optimization of the IPP Precursor Supply for the Production of Lycopene, Decaprenoxanthin and Astaxanthin by Corynebacterium glutamicum. <I>Frontiers in bioengineering and biotechnology</I><B>2</B>, 28 (2014).</P><P style="padding: 0px auto">[6]J. G. Zalatan <I>et al.</I>, Engineering complex synthetic transcriptional programs with CRISPR RNA scaffolds. <I>Cell</I><B>160</B>, 339 (Jan 15, 2015).</P><P style="padding: 0px auto">[7]J. Boch, U. Bonas, Xanthomonas AvrBs3 family-type III effectors: discovery and function. <I>Annual review of phytopathology</I><B>48</B>, 419 (2010).</P><P style="padding: 0px auto">[8]J. Boch <I>et al.</I>, Breaking the code of DNA binding specificity of TAL-type III effectors. <I>Science</I><B>326</B>, 1509 (Dec 11, 2009).</P><P style="padding: 0px auto">[9]M. J. Moscou, A. J. Bogdanove, A simple cipher governs DNA recognition by TAL effectors. <I>Science</I><B>326</B>, 1501 (Dec 11, 2009).</P></DIV></DIV><DIV id="navigation"><LI><A href="#myMainContainer" class="t" style="padding: 0px auto; font-size:12px;"><SPAN>TOP/TIP</SPAN></A></LI><LI><A href="#pos1"><SPAN>Abstract</SPAN></A></LI><LI><A href="#pos2"><SPAN>Introduction</SPAN></A></LI><LI><A href="#pos3"><SPAN>Improvement</SPAN></A></LI><LI><A href="#pos31" class="b"><SPAN>BBa_I715019 &amp; BBa_I715020</SPAN></A></LI><LI><A href="#pos32" class="b"><SPAN>BBa_K515100</SPAN></A></LI><LI><A href="#pos33" class="b"><SPAN>Improvement: ZJU_CHINA 2012</SPAN></A></LI><LI><A href="#pos4"><SPAN>Reference</SPAN></A></LI></DIV><DIV id="NUDT_Footer"><CENTER><P>Copyright © 2015 NUDT_CHINA | Powered by Nianhao Xie</P></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>