<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HBUT-China_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HBUT-China/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content-main"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 project-desc" id="contentbox"><DIV class="page-header"><H1>Notebook</H1></DIV><DIV class="tab"><DIV class="tab_menu"><UL><LI class="on">Molecular experiment log</LI><LI>Microbiological experiment log</LI></UL></DIV><DIV class="tab_box"><DIV><H3 align="center">Molecular experiment log</H3><SMALL class="bigtitle">Experimental Objectives:</SMALL><P>Construction of recombinant plasmids containing the pncrA promoter and the NcrB protein sequence And try to construct a recombinant plasmid containing the complete component.</P><P class="imgdesc">Figure 1 Plasmid map of bacteria</P><P class="imgdesc">Figure 2  Bacteria of biological bricks</P><P>7.21-7.29</P><P>Topic: PCR amplification amplification pncrA and repressor protein NcrB and agar making
             				</P><P>Materials:</P><P>2 × Buffer 100 μl</P><P>DD H2O 92 μl</P><P>pncrA forward primer 1 μl</P><P>pncrA reverse primer 1 μl</P><P>NcrB forward primer 1 μl</P><P>NcrB reverse primer 1 μl</P><P>The plasmid 4 μl</P><P>Agarose 0.3g</P><P>1 × TAE 30ml</P><P>1/10000 Bio Teke Dye 3 μl</P><P>2000bp maker4μl</P><P>10 × Buffer 8 μl</P><P>Procedure:</P><P>1. 5 μl of Buffer, 23 μl of DD H2O, 1 μl of plasmid, and 0.5 μl of forward primer and 0.5 μl reverse primer were mixed into the tube for a total of 50 μl. The mixture was divided and divided into two tubes and placed in a PCR instrument. PCR final system is the 25μl .There are 8 in total, of which 1, 2, 3, 4 marked N; 5,6,7,8 marked P.</P>
             				PCR parameters set: denaturation 94 ° C, refolding 60 ° C, extended 72 ° C, 30 cycles.<P>2. Take the 0.3 g agarose microwave oven and heat it in 30 ml of 1 × TAE (prepare 1% agarose gel).</P><P>3. Add 3μl of dye to the hot mix and Pour into the mold .If bubbles are generated, removeing them with plastic pipettes.</P><P>4. Let the gel solidify for 30 min.</P><P>5. After the gelation, suit remove the comb. Place it in the gel tank.</P><P>6. At the end of the PCR, take out 8 tubes and 8 μl were mixed with 1 μl of 10 × Buffer and added to each well. At the end of PCR, 8 tubes were removed and then 8 μl were mixed with 1 μl of 10 × Buffer and added to each well. When the point of time, the tip of the gun is perpendicular to the filling hole so that the corner just under the opening in the sample hole. Add the sample and the mark maker slowly to the sample hole</P><P>7. Run the gel electrophoresis at 114V for 30 minutes.</P><P>8. Press the glue</P><P>Results and analysis:</P><P class="imgdesc">Fig 3 .PCR products of  pncrA and NcrB in plasmid</P><P class="imgdesc">Note: M: DL2000 DNA marker</P><P>The promoter (pncrA) and repressor (NcrB) were successfully amplified.</P><P>1. The first system settings: 50μl system, the primer plus 1μl</P><P>Results: There are many non-specific amplification</P><P>Analysis: Because the primer concentration is too much, there are many non-specific amplifications</P><P>2. The second system settings: 25μl system, the primer plus 0.5μl</P><P>Results: As shown in the figure, the nonspecific bands are less, the target bands had no obvious tailing and are clearly visible.</P><P>Analysis: The product was successfully amplified and the next step was purified</P><P>Topic:DNA recovery</P><P>Materials:</P><P>Agarose 0.6g</P><P>1×TAE 60ml</P><P>1/10000 Bio Teke Dye 6 μl</P><P>2000bp maker 8μl</P><P>10×Buffer 8 μl</P><P>Amplified fragment promoter (pncrA)</P><P>Amplified fragment repressor protein (NcrB)</P><P>DNA purification recovery kit (TIANGEN company)</P><P>Procedure:</P><P>Use DNA purification recovery kit and then run the glue to see the recovery effect</P><P>Results and analysis:</P><DIV class="row"><DIV class="col-lg-6 col-xm-12"><P class="imgdesc">Fig4. Pecturing of pncrA and NcrB</P></DIV><DIV class="col-lg-6 col-xm-12"><P class="imgdesc">Fig5. Recovery of pncrA and NcrB</P></DIV></DIV><P>Glue recycling are successful and no other specific bands.</P><P>Topic: Digestion</P><P>Materials:</P><P>Plasmid vector PSB1C3 10 μl</P><P>Promoter (pncrA) 10 μl</P><P>Repressor protein (NcrB) 10 μl</P><P>restriction enzyme (Ecor1) 1 μl</P><P>restriction enzyme (Pst1) 1 μl </P><P>10 × Buffer O 2μ</P><P>Agarose 0.6 g</P><P>1×TAE 60ml</P><P>1/10000 Bio Teke Dye 6 μl</P><P>2000bp maker 8μl</P><P>10 × Buffer 3 μl</P><P>DNA purification recovery kit (TIANGEN company)</P><P>Procedure:</P><P>Digestion System:1 μl of Ecor1, 1 μl of Pst1, 10 μl of digested DNA, 10 × Buffer O 2 μ, 6 μl of distilled water, total 20μl.</P><P>Configure the mixture according to the system</P><P>37 ° C for 2 hours</P><P>65 ℃ fire for 20 minutes</P><P>Purification of the digestion product using a DNA purification recovery kit and running gel.</P><P>Results and analysis:</P><P>Digestion effect is not too good and the band is not bright enough.</P><P>Probably because the concentration of the target fragment we added was insufficient.</P><P class="imgdesc">Fig7. Digestion of Pncra and Ncrb </P><P>None. Results are expected after transformation and PCR.</P><P>7.29-8.4</P><P>Topic:Ligation</P><P>Materials:</P><P>Digested PSB1C3  2μl</P><P>Digested NcrB 5μl </P><P>Digested pncrA 8μl</P><P>T4 DNA ligase 1μl</P><P>10xT4 Buffer 2μl</P><P>DD H2O 19μl</P><P>Procedure:</P><P>Connection system: The total volume is 20μl</P><P>Linear plasmid vector: 1μl，about 30 ng (measured plasmid backbone concentration of 29.2 ng /μl)</P><P>Inserted DNA:plasmid backbone = 1: 1-5: 1 ，and we select ratio of 3: 1</P><P>Digested NcrB: 5μl (measured at a concentration of 16.7 ng /μl)</P><P>Or digested pncrA: 8μl (measured at a concentration of 11 ng /μl)</P><P>10xT4 Buffer: 2μl</P><P>T4 DNA ligase: 1μl (Thermo Corporation, No. EL0014, enzyme activity 5 weiss U /μl)</P><P>added DD H2O to 20 μl</P><P>Configure the mixture according to the system above</P><P>22 °C for 40 minutes</P><P>65 °C inactivated for 20 minutes</P><P>Results and analysis:</P><P>None. Results are expected after transformation and PCR.</P><P>Topic: Transformation</P><P>Materials:</P><P>Recombinant plasmid of ncrB DNA  2 μl </P><P>Recombinant plasmid of pncrA DNA  2 μl </P><P>DH5α bacterium 200 μl</P><P>SOC 800 μl</P><P>Chloramphenicol 200 μl</P><P>Agar Petri dish 8</P><P>Procedure:</P><P>1. Take a commercial competent cells (total 100μl), two self-made competent cells (each 80μl), put frozen on ice 15min</P><P>2. Prepare four sterilized 2mlEP tubes, put on ice to precool (This should be carried out at the same time with the first step )</P><P>3. Draw 50 μl of competent cell fluid added to the four precooled EP tube (two commercial, two self-made)</P><P>4. Add 1μl recombinant plasmid to each of the commercial competent cells and self-made competent cells. Do the same to the pncrA DNA.</P><P>5. Close the EP tube, and then put it in an ice bath for 30min</P><P>6. 42°C heat shock for 1min, put it in an ice bath for 5min</P><P>7. Add 200μl of SOC medium to each EP tube</P><P>8. 37 °C shaker culture 3h</P><P>9. Take two samples, one 50 μl and one 150 μl, of each EP tube, place onto the plate and label them.</P><P>10. place in incubator at 37C for 12 to 16 hours.</P><P class="bigtitle">Results and analysis:</P><DIV class="row"><DIV class="col-lg-6 col-xm-12"><P class="imgdesc">Fig 7.Transformation of pncrA</P></DIV><DIV class="col-lg-6 col-xm-12"><P class="imgdesc">Fig8.Transformation of NcrB</P></DIV></DIV><P>The recombinant plasmid with pncrA didn’t transform into E. coli, the recombinant plasmid with ncrB transformed into E. coli successfully. The DNA gel electrophoresis results showed digestion succeeded, thus there was no conoly due to the failure of ligation and transformation. </P><P>Topic: PCR of pncrA</P><P>Materials:</P><P>2 × taq enzyme 75 μl</P><P>pncrA forward primer 1.5 l</P><P>pncrA reverse primer 1.5 μl</P><P>DD H2O  72μl</P><P>Agarose 0.3g</P><P>1 × TAE 30ml</P><P>1/10000 Bio Teke Dye 3 μl</P><P>2000bp maker 4μl</P><P>10 × Buffer 6 μl</P><P>bacteria</P><P>Procedure:</P><P>PCR MIX system: the total volume is 25μl. 2×taq enzyme 25μl ( from Engine company), 0.5μl forward and 0.5μl reverse primer, DDH2O 24μl, the total is 50μl. Then divide them into two tubes. Repeat the operations for other two times, and there are 6 tubes totally.</P><P>A single colony was randomly selected at room temperature, and was picked with a sterile toothpick or a tip, and a copy was made on a solid plate and added to 500μl of liquid LB medium. The colonies were added to the PCR MIX system in the same manner. Do a colony PCR to verify the part size.</P><P>Results and analysis:</P><P class="imgdesc">Fig9. PCR of Pncra </P><P>There was a successful connection of pncrA DNA bands, which may have a small amount of specific amplification.  the recombinant plasmid with ncrB transformed into E. coli successfully, the problem may be either ligation or Transformation, but ligation is more likely.</P><P>8.4-8.11</P><P>Topic:Ligation</P><P>Materials:</P><P>Digested PSB1C3  4μl</P><P>Digested NcrB 6μl </P><P>Digested pncrA 6μl</P><P>T4 DNA ligase 2μl</P><P>10xT4 Buffer 2μl</P><P>Procedure:</P><P>Connection system: The total volume is 10μl.</P><P>Linear plasmid vector: 2μl</P><P>Interested DNA: plasmid backbone = 1: 1-5: 1 ，and we select ratio of 3: 1</P><P>Digested Ncrb: 6μl </P><P>Or digested Pncra: 6μl </P><P>10xT4 Buffer: 1μl</P><P>T4 DNA ligase: 1μl</P><P>added DD H2O to 20 μl</P><P>Configure the mixture according to the system</P><P>22 °C for 4 hours</P><P>65 ℃ inactivated for 30 minutes</P><P class="bigtitle">Results and analysis:</P><P>None. Results are expected after transformation and PCR.</P><P>Topic:Transformation</P><P>Materials:</P><P>ligated plasmid of ncrB DNA  2 μl </P><P>ligated plasmid of pncrA DNA  2 μl </P><P>DH5α bacterium 100 μl</P><P>Chloramphenicol 200 μl</P><P>LB medium 1 mL</P><P>LB solid medium 50 mL</P><P>Chloramphenicol 50 μl</P><P>Agar Petri dish 2</P><P class="bigtitle">Procedure:</P><P>1. Take two self-made competent cells (each 80μl) and put on ice to freeze - 15min</P><P>2. Prepare two sterilized 2ml EP tubes, put on ice to precool (This should be carried out at the same time as the first step )</P><P>3. Draw 50 μl of competent cell fluid and add to the two precooled EP tubes </P><P>4. Add 1μl recombinant plasmid to each of the commercial competent cells and self-made competent cells. Do the same to the pncrA DNA.</P><P>5. Close the EP tube, and then put it in an ice bath for 30min</P><P>6. 42°C heat shock for 90s, put it in an ice bath for 5min</P><P>7. Add 200 μl of SOC medium to each EP tube</P><P>8. 37 ℃ shaker culture 3h</P><P>9. Take 50 μl and 150 μl of each EP tube onto the plate and label them</P><P>10. 37 ° C incubating for 12 to 16 hours</P><P class="bigtitle">Results and analysis:</P><P class="imgdesc">Fig 10. Transformation of pncrA and ncrB </P><P>DH5α were successfully transformed and the colonies were grown on the medium.</P><P>Topic:PCR of pncrA and ncrB</P><P>Materials:</P><P>2xtaq enzyme 125μl</P><P>pncrA forward primer 2.5μl</P><P>pncrA reverse primer 2.5μl</P><P>ncrB forward primer 2.5μl</P><P>ncrB reverse primer 2.5μl</P><P>DD H2O 120μl</P><P>bacteria</P><P>Agarose 0.6g</P><P>1 × TAE 60ml</P><P>1/10000 Bio Teke Dye 6 μl</P><P>2000bp maker4μl</P><P>10 × Buffer 12 μl</P><P>Procedure:</P><P>PCR MIX system: the total volume is 25μl.</P><P>2×taq enzyme 25μl (from Engine company), </P><P>0.5μl forward and reverse primer, </P><P>DD H2O 24μl, the total is 50μl.</P><P>Then divide them into two tubes. Repeat the operations for other 5 times, and there are 10 tubes totally.</P><P>A single colony was randomly selected at room temperature, and was picked with a sterile toothpick or a tip, and a copy was made on a solid plate and added to 500μl of liquid LB medium. The colonies were added to the PCR MIX system in the same manner. Do a colony PCR to verify the part size.</P><P class="bigtitle">Results and analysis:</P><P class="imgdesc">Fig11. PCR of pncrA and ncrB</P><P>We improve the ligation system, extend the ligation time, optimize the transformation step. We try to make ncrB DNA, pncrA and plasmid vector connected and its successful transformation.</P><P>Topic:sequencing</P><P class="bigtitle">Procedure:</P><P>We will send the samples to company for further sequencing.</P><P>Results and analysis:</P><P class="imgdesc">Fig12.Comparison of ncrB</P><P class="imgdesc">Fig13.Comparison of pncrA</P><P>After sequencing, we confirmed that the DNA is indeed what we need.</P><P>8.12-8.23</P><P>Topic: Digestion，Ligation，Ligation</P><P>Procedure:</P><P>The method of digestion and ligation is the same as above.</P><P>Transformation: The system and method are the same as before, except that the colonies are not required to be coated with a solid culture medium and can be directly cultured on a liquid medium overnight to extract the plasmid. The transformed plasmids are E1010, B0010, B0012, B0032, B0034, J23102.</P><P class="bigtitle">Results and analysis:</P><P>B0010 failed to convert and there were no Colonies, but other plasmids successfully transformed.</P><P>Topic:3A assembly</P><P>Materials:</P><P>Linear plasmid vector (PSB1T3)  10μl</P><P>J23102  10μl</P><P>B0032  10μl</P><P>B0034  10μl</P><P>Digested of pncrA 5μl </P><P>T4 DNA ligase  4μl</P><P>10xT4 Buffer  3μl</P><P>restriction enzyme (EcoRI)  4μl</P><P>restriction enzyme (Pst1)  4μl </P><P>Restriction enzyme (XbaI) 2μl</P><P>restriction enzyme (SpeI)  2μl</P><P>10 × Buffer O  12μl</P><P>10 × Buffer T  6μl</P><P>DD H2O 18μl</P><P class="bigtitle">Procedure:</P><P>The method of digestion, ligation and transformation is the same as above. And then use the mention the plasmid kit.</P><P class="imgdesc">Fig14. Principle of 3A assembly</P><P>The clips are joined in this order:</P><P>J23102-B0032-ncrB-B0010-B0012-pncrA-B0034-E1010-B0012-B0034. </P><P>We first use the 3A method toconnect thesefragments:</P><P>J23102-B0032-ncrB; B0010-B0012-Pncra-B0034; E1010-B0012-B0034.</P><P>The &quot;three large fragments&quot; were then ligated using Gibson assembly technology (seamless splicing), and finally transformed into the PSB1C3 plasmid.</P><P class="bigtitle">Results and analysis:</P><P>Because of limited time, we regret that we have not been able to complete all the experiments. but we have successfully confirmed the J23102-B0032 connection to the feasibility of our idea.</P></DIV><DIV style="display: none;"><H3 align="center">Microbiological experiment log</H3><P class="bigtitle">2017/07/05</P><P>Topic: Bacteria Culturing</P><P class="bigtitle">Material:</P><P>LB 50mL</P><P>chloramphenicol</P><P>E.coli(K2304002)</P><P>5x test tube </P><P class="bigtitle">Procedure:</P><P>1.	Sterilize test tubes and LB media.</P><P>2.	Distribute the culture medium and chloramphenicol into the test tube. Each tube contained 5 ml of LB medium containing 5 μL of chloramphenicol.</P><P>3.	Inoculate 1% of E.coli(K2304002) in the LB medium.</P><P>4.	Culture the samples at 200rpm, 37 ℃overnight.</P><P class="bigtitle">2017/07/06</P><P>Topic: Test K2304002</P><P class="bigtitle">Material:</P><P>E.coli(K2304002) cultured on July 5</P><P>Nickel ion solution (0.5mol/L)</P><P class="bigtitle">Procedure:</P><P>1.	Add the nickel ion solution to the bacteria solution.
			            	</P><P>2.	Continue to culture bacteria for 2 hours.</P><P>3.	Observe whether the bacterial solution changes.</P><P>4.	Centrifuge the bacteria at 12,000ram for 10 minutes.</P><P>5.	Observe the solution after centrifugation.</P><P class="bigtitle">Result:</P><P>After 2 hours of culture, the bacteria solution did not change. We are very worried that K2304002 does not work. </P><P>But after centrifugation, The red precipitate appeared at the bottom of the centrifuge tube，while the blank control group produced very little, or none of the red precipitation. This indicates that KL2304002 can be induced by Ni ions to produce RFP.</P><P class="bigtitle">2017/07/17</P><P>Topic: Nickel ion tolerance</P><P class="bigtitle">Material:</P><P>LB 200mL</P><P>chloramphenicol</P><P>E.coli(K2304002)</P><P>30x test tube </P><P>ddH2O</P><P>Nickel ion solution (5x10<SUP>2</SUP>mol/L、5x10<SUP>1</SUP>mol/L、 5x10<SUP>0</SUP>mol/L、5x10<SUP>-1</SUP>mol/L、5x10<SUP>-2</SUP>mol/L、5x10<SUP>-3</SUP>mol/L、5x10<SUP>-4</SUP>mol/L)</P><P class="bigtitle">Procedure:</P><P>1.	Sterilize test tubes and LB media.</P><P>2.	Distribute the culture medium and chloramphenicol into the test tube. Each tube contained 5 ml of LB medium containing 5 μL of chloramphenicol.</P><P>3.	Each group of three tubes was added with the same concentration of nickel ions. One of test tube is blank, a total of eight test tubes.</P><P>4.	Inoculate 1% of E.coli(K2304002) in the LB medium.</P><P>5.	Culture the samples at 200rpm, 37 ℃overnight.</P><P>6.	Measure the OD600 and FI of each sample.</P><P class="bigtitle">Result:</P><P>By analyzing the OD value, we draw the relationship between cell growth and nickel ion concentration.</P><P>From the figure we can see that when the nickel ion concentration greater than 0.5mmol / L, the cell growth is inhibited.</P><P class="bigtitle">2017/07/28</P><P>Topic: Growth curve</P><P class="bigtitle">Material:</P><P>LB 1000mL</P><P>chloramphenicol</P><P>E.coli(K2304002)</P><P>8x beakerflask </P><P>Nickel ion solution (5x10<SUP>-2</SUP>mol/L、1x10<SUP>-2</SUP>mol/L、5x10<SUP>-3</SUP>mol/L、1x10<SUP>-3</SUP>mol/L、5x10<SUP>-4</SUP>mol/L</P>
			            	)
			            	<P class="bigtitle">Procedure:</P><P>1.	Sterilize beakerflask and LB media.</P><P>2.	Distribute the culture medium and chloramphenicol into the beakerflask. Each beakerflask contained 100 ml of LB medium containing 100 μL of chloramphenicol.</P><P>3.	Each beakerflask was added with different concentration of nickel ions. One of beakerflask is blank, a total of eight beakerflask.</P><P>4.	Inoculate 1% of E.coli(K2304002) in the LB medium.</P><P>5.	Culture the samples at 200rpm, 37 ℃.</P><P>6.	Detection of OD and fluorescence intensity every other hour.</P><P class="bigtitle">Result: </P><P>We find that the OD was close, but the fluorescence intensity was very different.</P><P>he graph below shows that there is a correlation between fluorescence intensity and nickel ion concentration.</P><P>After this experiment, we further measured the 24-hour growth cuve.</P><P class="bigtitle">2017/08/10</P><P>Topic: Specificity</P><P class="bigtitle">Material:</P><P>LB 100mL</P><P>chloramphenicol</P><P>E.coli(K2304002)</P><P>5x test tube </P><P>Cd<SUP>2 +</SUP>, Co<SUP>2 +</SUP>, Cu<SUP>2 +</SUP>, Fe<SUP>3 +</SUP>, and Ni<SUP>2 +</SUP>solution</P><P class="bigtitle">Procedure:</P><P>1.	Sterilize test tubes and LB media.</P><P>2.	Distribute the culture medium and chloramphenicol into the test tube. Each tube contained 5 ml of LB medium containing 5 μL of chloramphenicol.</P><P>3.	Each test book was added with different ions. One of test tube is blank, a total of six test tube.</P><P>4.	Inoculate 1% of E.coli(K2304002) in the LB medium.</P><P>5.	Culture the samples at 200rpm, 37 ℃ overnight.</P><P>6.	Observe them with a fluorescence microscope.</P><P class="bigtitle">Result: </P><P>Clearly, only the sample with Ni ions showed red fluorescence, the other groups had no red fluorescence. Note that the iron ions reacted with the substance in the culture medium，so that there is an impurity in the field of view.</P><P class="bigtitle">2017/08/15</P><P>Topic: Qualitative testing</P><P class="bigtitle">Material:</P><P>LB 100mL</P><P>chloramphenicol</P><P>E.coli(K2304002)</P><P>9x test tube </P><P>Nickel ion solution (1x10-5mol/L、5x10-6mol/L、1x10-6mol/L)</P><P class="bigtitle">Procedure: </P><P>1.	Sterilize test tubes and LB media.</P><P>2.	Dilute the cultures to a target OD of 0.8.</P><P>3.	Each test tube was added with different concentration of nickel ions.</P><P>4.	Inoculate 1% of E.coli(K2304002) in the LB medium.</P><P>5.	Culture the samples at 200rpm, 37 ℃ 2 hours.</P><P>6.	Detection of OD and fluorescence intensity every other hour.</P><P class="bigtitle">Result: </P><P>We found that this method can be uesd to find the linear relationship between nickel ions and fluorescence intensity.</P><P>This graph shows that after 2 hours, the nickle ion concentration is positively correlated with the fluorescence intensity. Continue to experiment, we can find a suitable means of detecting nickle ion.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>copyright ©  2017 IGEM-HBUT-China All rights reserved.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>