<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NKU_China_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NKU China/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="collapse navbar-collapse" id="navbar-menu"><UL class="nav navbar-nav navbar-right" data-in="fadeInDown" data-out="fadeOutUp"><LI><A href="https://2017.igem.org/Team:NKU_China">Home</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:NKU_China/Team" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Team" target="_blank">Team</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Collaborations" target="_blank">Collaborations</A></LI></UL><LI><A href="https://2017.igem.org/Team:NKU_China/Attributions" target="_blank">Attributions</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:NKU_China/Description" class="dropdown-toggle" data-toggle="dropdown">Projects</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Description" target="_blank">Description</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Experiments" target="_blank">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Model" target="_blank">Model</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/proof" target="_blank">Proof of Concept</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Demonstrate" target="_blank">Demonstrate</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:NKU_China/Human_Practices" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Human_Practices" target="_blank">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Silver" target="_blank">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Integrated and Gold" target="_blank">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Safety" target="_blank">Safety</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Parts" target="_blank">Parts</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/InterLab" target="_blank">InterLab</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:NKU_China/Notebook" class="dropdown-toggle" data-toggle="dropdown">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Notebook" target="_blank">Lab notes</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/protocols" target="_blank">Protocols</A></LI></DIV></DIV><DIV class="side"><DIV class="widget"><H6 class="title">Custom Pages</H6><UL class="link"><LI><A href="https://2017.igem.org/Team:NKU_China#">Home Page</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Description">Projects</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:NKU_China/Human_Practices">Human Practices</A></LI></UL></DIV><DIV class="widget"><H6 class="title">Additional Links</H6><UL class="link"><LI><A href="#">NKU_China</A></LI></UL></DIV></DIV><DIV class="card-holder"><DIV class="card-wrapper"><A href="#BE"><DIV class="card bg-01"><SPAN class="card-content"><I>BcsA</I>-knockout <I>Enterobacteria</I> sp. FY-07</SPAN></DIV></A></DIV><DIV class="card-wrapper"><A href="#Plasmids"><DIV class="card bg-02"><SPAN class="card-content">Plasmids</SPAN></DIV></A></DIV><DIV class="card-wrapper"><A href="#FI"><DIV class="card bg-03"><SPAN class="card-content">Functional Identification</SPAN></DIV></A></DIV><DIV class="card-wrapper"><A href="#BPC"><DIV class="card bg-04"><SPAN class="card-content">Basic Parts</SPAN></DIV></A></DIV></DIV><DIV id="note-head">Laboratory Notes</DIV><DIV class="note-part"><DIV class="note-header">1.	Construction of <I>BcsA</I>-knockout <I>Enterobacteria </I>sp. FY-07</DIV><DIV class="note-p">In order to express cellulose in <I>Enterobacteria</I> sp. FY-07 in a controllable way, we should first knock out the <I>BcsA</I> gene in <I>Enterobacteria</I> sp. FY-07. Therefore, <I>Enterobacteria</I> sp. FY-07 will not produce cellulose under natural environmental condition.</DIV><DIV class="note-subheader">Week 2 (2017.7.10-7.16)</DIV><DIV class="note-p">To begin with, we amplified the upstream and downstream of <I>BcsA</I> gene in <I>Enterobacteria</I> sp. FY-07 through PCR.
Next, we combined the upstream and downstream of <I>BcsA</I> gene by overlapping PCR and the overlapped PCR products were ligated into plasmid pTsk1.
Then pTsk1-Δ<I>BcsA</I> was transformed into S17 and positive clones were selected.</DIV><DIV class="note-subheader">Week 3 (2017.7.17-7.23)</DIV><DIV class="note-p">We screened the selected clones using PCR and introduced pTsk1-Δ<I>BcsA</I> to <I>Enterobacteria</I> sp. FY-07 through conjugative transfer.
The positive pTsk1-Δ<I>BcsA</I> transformants were cultured at 42℃ with chloramphenicol to select single-crossover clones.</DIV><DIV class="note-subheader">Week 4 (2017.7.24-7.30)</DIV><DIV class="note-p">We screened the single-crossover strains using PCR and then cultured right strains in LB medium and passaged every 12 hours for 4 generations.</DIV><DIV class="note-subheader">Week 5 (2017.7.31-8.6)</DIV><DIV class="note-p">We screened the selected clones using PCR and introduced pTsk1-Δ<I>BcsA</I> to <I>Enterobacteria</I> sp. FY-07 through conjugative transfer.
The positive pTsk1-Δ<I>BcsA</I> transformants were cultured at 42℃ with chloramphenicol to select single-crossover clones.</DIV><DIV class="note-subheader">Week 6-8 (2017.8.7-8.20)</DIV><DIV class="note-p">We reapeated the conjugative transfer and the following procedures, and successfully got double-crossover clones, which was verified using PCR. 
Finally, we obtained our aimed strain – <I>BcsA</I>-knockout <I>Enterobacteria</I> sp. FY-07.
The <I>BcsA</I>-knockout <I>Enterobacteria</I> sp. FY-07 strain was cultured and made competent for future use.
</DIV></DIV><DIV class="note-part"><DIV class="note-header">2.	Construction of Plasmids</DIV><DIV class="note-subheader">Week 1-3 (2017.7.3-7.23)</DIV><DIV class="note-p">Initially we planned to construct pBBR1-<I>rhlABC</I>-<I>fimS</I>-<I>lacI</I>-<I>lacO</I>-<I>BcsA</I> and pBad30-cm-<I>fimE</I>. Then first we cloned some single fragments by PCR for preparation.</DIV><DIV class="note-ps"><I>rhlABC</I></DIV><DIV class="note-p">
We cloned <I>rhlABC</I> gene from pET28a-<I>rhlABC</I> and gained the fragment through gel extraction purification. But the concentration was very low, just 22.314 ng/μL.
</DIV><DIV class="note-ps"><I>lacI</I>-<I>lacO</I></DIV><DIV class="note-p">We cloned <I>lacI</I> gene from <I>E.coli</I> K12 genome. After this we used the product as template and changed a new downstream primers to gain <I>lacI</I>-<I>lacO</I>.</DIV><DIV class="note-ps"><I>BcsA</I></DIV><DIV class="note-p"><I>BcsA</I> gene was cloned from FY-07 genome.</DIV><DIV class="note-ps"><I>fimS</I></DIV><DIV class="note-p"><I>fimS</I> was cloned from <I>BL</I>21 and was amplified by PCR.</DIV><DIV class="note-ps"><I>fimE</I></DIV><DIV class="note-p"><I>fimE</I> gene was cloned from <I>BL</I>21 and was amplified by PCR.</DIV><DIV class="note-subheader">Week 4-6 (2017.7.24-8.13)</DIV><DIV class="note-p">Because every time we cloned the <I>rhlABC</I> gene through PCR, the product is mixed with other non target products. In addition, the concentration of the fragment through gel extraction purification was always low. So this week we tried to find a way to improve this condition. </DIV><DIV class="note-ps">Using existing fragments as templates</DIV><DIV class="note-p">We tried to use existing <I>rhlABC</I> as templates, but sadly the products were only non-targeted products with 1000-2000bp.</DIV><DIV class="note-ps">Using temperature gradient to find the best Tm</DIV><DIV class="note-p">The PCR system and PCR reaction condition was the same as above. The best Tm we found is 50.7℃. However, we cloned it again using this Tm, the concentration of the product was still frustrating, just 18.699 ng/μL.</DIV><DIV class="note-ps">Nested PCR</DIV><DIV class="note-p">There was no aim product and the result of nucleic acid electrophoresis showed that the template could have some mistakes.</DIV><DIV class="note-ps">Verify the pET28a-<I>rhlABC</I></DIV><DIV class="note-p">Later we picked up several bacteria on the plate and cultured them, then we tried verify PCR with T7 primer. The result was that there were no aim strips. So we redesigned the primers and planned to get <I>rhlABC</I> through overlapping <I>rhlAB</I> and <I>rhlC</I>.</DIV><DIV class="note-subheader">Week 7-9 (2017.8.14-9.3)</DIV><DIV class="note-p">This week we tried to get some combined fragments using existing genes through overlap extention PCR(OE PCR).</DIV><DIV class="note-ps"><I>fimS</I>-<I>lacI</I>(<I>lacO</I>)</DIV><DIV class="note-p">We successfully got the aim products.</DIV><DIV class="note-p">① Tm=65℃  12cycles
②adding primers, Tm=61℃  18cycles</DIV><DIV class="note-ps"><I>rhlABC</I>-<I>fimS</I></DIV><DIV class="note-p">We successfully got the aim products.</DIV><DIV class="note-p">① Tm=60℃  12cycles
②adding primers, Tm=65℃  18cycles</DIV><DIV class="note-ps"><I>lacI</I>(<I>lacO</I>)-<I>BcsA</I></DIV><DIV class="note-p">We successfully got the aim products.</DIV><DIV class="note-p">① Tm=60℃  12cycles
②adding primers, Tm=61℃  35cycles</DIV><DIV class="note-ps">4.<I>rhlABC</I>-<I>fimS</I>-<I>lacI</I>-<I>lacO</I>-<I>BcsA</I></DIV><DIV class="note-p">In the same way, we got <I>fimS</I>-<I>lacI</I>-<I>lacO</I>-<I>BcsA</I>. Then, We have tried different combinations (<I>rhlABC</I>-<I>fimS</I> and <I>lacI</I>(<I>lacO</I>)-<I>BcsA</I>; <I>rhlABC</I> and <I>fimS</I>-<I>lacI</I>-<I>lacO</I>-<I>BcsA</I>) and different mix(STAR, LA and phanta mix), but didn’t get the aim product.</DIV><DIV class="note-subheader">Week 10 (2017.9.4-9.10)</DIV><DIV class="note-p">We continued OE PCR, simultaneously, we went on to construct pBad30-cm-<I>fimE</I>.</DIV><DIV class="note-p">Both pBad30 and cm segments were treated with restriction enzyme. Later cm gene was ligated to linear pBad30 and then we got pBad30-cm. In the same way, we got pBad30-cm-<I>fimE</I> and transformed it into DH5α. But when we use verify PCR to test the <I>fimE</I> gene, we didn’t get the aim strip.</DIV><DIV class="note-subheader">Week 11 (2017.9.11-9.17)</DIV><DIV class="note-p">To verify the feasibility of <I>fimS</I> first, we went on to construct pBBR1-<I>rfp</I>-<I>fimS</I>-<I>lacI</I>-<I>lacO</I>-<I>gfp</I>.We successfully got <I>rfp</I>, <I>gfp</I> and <I>rfp</I>-<I>fimS</I>-<I>lacI</I>-<I>lacO</I>, but failed to overlap the whole segment-- <I>rfp</I>-<I>fimS</I>-<I>lacI</I>-<I>lacO</I>-<I>gfp</I>.</DIV><DIV class="note-p">Sum up this result and previous experience, we desided to make some adjustments in our design. </DIV><DIV class="note-p">1. The distance between <I>lacI</I> and <I>lacO</I> is too close, which may affect the function. So we desided to construct pBBR1-<I>rhlABC</I>-<I>fimS</I>-<I>lacO</I>-<I>BcsA</I> and pBad30-<I>fimE</I>-cm-<I>lacI</I>. </DIV><DIV class="note-p">2. There was a solution that the <I>BcsA</I> and <I>gfp</I> genes were digested and ligated to pK18, and the <I>lacO</I>-<I>BcsA</I>/<I>GFP</I> could be obtained if PCR started from the <I>lacO</I> position before the pK18 cloning site.</DIV><DIV class="note-p">3. We redesigned the primers. </DIV><DIV class="note-subheader">Week 12 (2017.9.18-9.24)</DIV><DIV class="note-p">We redesigned the primers for Gibson Assembly and successfully constructed pBBR1-<I>rhlABC</I>-<I>fimS</I>-<I>lacO</I>-<I>BcsA</I>, pBBR1-<I>rfp</I>-<I>fimS</I>-<I>lacO</I>-<I>gfp</I> and pBad30-<I>fimE</I>-cm-<I>lacI</I>.</DIV></DIV><DIV class="note-part"><DIV class="note-header">3.	Functional Identification of Our Design</DIV><DIV class="note-subheader">Week 1-2 (2017.7.3-7.16)</DIV><DIV class="note-p">We constructed pBBR-<I>rhlABC</I> and introduced the plasmid into wild-type <I>Enterobacter</I> sp. FY-07. Through Diesel Emulsification Experiment，we verified that <I>Enterobacter</I> sp. FY-07 had the emulsification effect after overexpressing pBBR-<I>rhlABC</I></DIV><DIV class="note-subheader">Week 9 (2017.8.28-9.3)</DIV><DIV class="note-p">We verified that <I>BcsA</I>-knockout <I>Enterobacter</I> sp. FY-07 didn’t produce cellulose under general condition, while overexpression of <I>BcsA</I> can rescue the expression of cellulose in <I>BcsA</I>-knockout <I>Enterobacter</I>sp. FY-07.</DIV></DIV><DIV class="note-part"><DIV class="note-header">4.	Basic Parts Construction</DIV><DIV class="note-subheader">Week 13-16 (2017.9.25-10.22)</DIV><DIV class="note-p">1. The<I> lyase</I>, <I>fimS</I> and <I>fimE</I> genes are cloned in the same way as week 1. </DIV><DIV class="note-p">2. Then both these segments and pSB1C3 vector were treated with restriction enzymes. After that, the three gene were separately ligated to linear pSB1C3, and the ligation products were separately transformed into DH5α. </DIV><DIV class="note-p">3. Verification PCR was performed to select the positive clones. The positive strains were chosen to be cultured overnight and plasmids were extracted. After restriction enzyme digestion verification, the positive clones were sequenced.</DIV></DIV><FOOTER><DIV class="container"><DIV class="row "><DIV class="contact-info col-xs-4"><H2>Contacts</H2><H2>Address</H2><ADDRESS>
                    Nankai University
                    No.94 Weijin Road, Nankai District
                    Tianjin, P.R.China 300071</ADDRESS></DIV></DIV><DIV class="col-xs-12"><DIV class="copyright">
                    Copyright © Team NKU-China 2017
                </DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>