Team:NKU China/Notebook
Home
Team
Team
Collaborations
Attributions
Projects
Description
Experiments
Model
Proof of Concept
Demonstrate
Human Practices
Human Practices
Silver HP
Integrated and Gold
Safety
Parts
InterLab
Notebook
Lab notes
Protocols
Custom Pages
Home Page
Team
Attributions
Projects
Parts
Safety
InterLab
Human Practices
Additional Links
NKU_China
BcsA-knockout Enterobacteria sp. FY-07
Plasmids
Functional Identification
Basic Parts
Laboratory Notes
1. Construction of BcsA-knockout Enterobacteria sp. FY-07
In order to express cellulose in Enterobacteria sp. FY-07 in a controllable way, we should first knock out the BcsA gene in Enterobacteria sp. FY-07. Therefore, Enterobacteria sp. FY-07 will not produce cellulose under natural environmental condition.
Week 2 (2017.7.10-7.16)
To begin with, we amplified the upstream and downstream of BcsA gene in Enterobacteria sp. FY-07 through PCR. Next, we combined the upstream and downstream of BcsA gene by overlapping PCR and the overlapped PCR products were ligated into plasmid pTsk1. Then pTsk1-ΔBcsA was transformed into S17 and positive clones were selected.
Week 3 (2017.7.17-7.23)
We screened the selected clones using PCR and introduced pTsk1-ΔBcsA to Enterobacteria sp. FY-07 through conjugative transfer. The positive pTsk1-ΔBcsA transformants were cultured at 42℃ with chloramphenicol to select single-crossover clones.
Week 4 (2017.7.24-7.30)
We screened the single-crossover strains using PCR and then cultured right strains in LB medium and passaged every 12 hours for 4 generations.
Week 5 (2017.7.31-8.6)
We screened the selected clones using PCR and introduced pTsk1-ΔBcsA to Enterobacteria sp. FY-07 through conjugative transfer. The positive pTsk1-ΔBcsA transformants were cultured at 42℃ with chloramphenicol to select single-crossover clones.
Week 6-8 (2017.8.7-8.20)
We reapeated the conjugative transfer and the following procedures, and successfully got double-crossover clones, which was verified using PCR.  Finally, we obtained our aimed strain – BcsA-knockout Enterobacteria sp. FY-07. The BcsA-knockout Enterobacteria sp. FY-07 strain was cultured and made competent for future use.
2. Construction of Plasmids
Week 1-3 (2017.7.3-7.23)
Initially we planned to construct pBBR1-rhlABC-fimS-lacI-lacO-BcsA and pBad30-cm-fimE. Then first we cloned some single fragments by PCR for preparation.
rhlABC
We cloned rhlABC gene from pET28a-rhlABC and gained the fragment through gel extraction purification. But the concentration was very low, just 22.314 ng/μL.
lacI-lacO
We cloned lacI gene from E.coli K12 genome. After this we used the product as template and changed a new downstream primers to gain lacI-lacO.
BcsA
BcsA gene was cloned from FY-07 genome.
fimS
fimS was cloned from BL21 and was amplified by PCR.
fimE
fimE gene was cloned from BL21 and was amplified by PCR.
Week 4-6 (2017.7.24-8.13)
Because every time we cloned the rhlABC gene through PCR, the product is mixed with other non target products. In addition, the concentration of the fragment through gel extraction purification was always low. So this week we tried to find a way to improve this condition.
Using existing fragments as templates
We tried to use existing rhlABC as templates, but sadly the products were only non-targeted products with 1000-2000bp.
Using temperature gradient to find the best Tm
The PCR system and PCR reaction condition was the same as above. The best Tm we found is 50.7℃. However, we cloned it again using this Tm, the concentration of the product was still frustrating, just 18.699 ng/μL.
Nested PCR
There was no aim product and the result of nucleic acid electrophoresis showed that the template could have some mistakes.
Verify the pET28a-rhlABC
Later we picked up several bacteria on the plate and cultured them, then we tried verify PCR with T7 primer. The result was that there were no aim strips. So we redesigned the primers and planned to get rhlABC through overlapping rhlAB and rhlC.
Week 7-9 (2017.8.14-9.3)
This week we tried to get some combined fragments using existing genes through overlap extention PCR(OE PCR).
fimS-lacI(lacO)
We successfully got the aim products.
① Tm=65℃  12cycles ②adding primers, Tm=61℃  18cycles
rhlABC-fimS
We successfully got the aim products.
① Tm=60℃  12cycles ②adding primers, Tm=65℃  18cycles
lacI(lacO)-BcsA
We successfully got the aim products.
① Tm=60℃  12cycles ②adding primers, Tm=61℃  35cycles
4.rhlABC-fimS-lacI-lacO-BcsA
In the same way, we got fimS-lacI-lacO-BcsA. Then, We have tried different combinations (rhlABC-fimS and lacI(lacO)-BcsA; rhlABC and fimS-lacI-lacO-BcsA) and different mix(STAR, LA and phanta mix), but didn’t get the aim product.
Week 10 (2017.9.4-9.10)
We continued OE PCR, simultaneously, we went on to construct pBad30-cm-fimE.
Both pBad30 and cm segments were treated with restriction enzyme. Later cm gene was ligated to linear pBad30 and then we got pBad30-cm. In the same way, we got pBad30-cm-fimE and transformed it into DH5α. But when we use verify PCR to test the fimE gene, we didn’t get the aim strip.
Week 11 (2017.9.11-9.17)
To verify the feasibility of fimS first, we went on to construct pBBR1-rfp-fimS-lacI-lacO-gfp.We successfully got rfp, gfp and rfp-fimS-lacI-lacO, but failed to overlap the whole segment-- rfp-fimS-lacI-lacO-gfp.
Sum up this result and previous experience, we desided to make some adjustments in our design.
1. The distance between lacI and lacO is too close, which may affect the function. So we desided to construct pBBR1-rhlABC-fimS-lacO-BcsA and pBad30-fimE-cm-lacI.
2. There was a solution that the BcsA and gfp genes were digested and ligated to pK18, and the lacO-BcsA/GFP could be obtained if PCR started from the lacO position before the pK18 cloning site.
3. We redesigned the primers.
Week 12 (2017.9.18-9.24)
We redesigned the primers for Gibson Assembly and successfully constructed pBBR1-rhlABC-fimS-lacO-BcsA, pBBR1-rfp-fimS-lacO-gfp and pBad30-fimE-cm-lacI.
3. Functional Identification of Our Design
Week 1-2 (2017.7.3-7.16)
We constructed pBBR-rhlABC and introduced the plasmid into wild-type Enterobacter sp. FY-07. Through Diesel Emulsification Experiment，we verified that Enterobacter sp. FY-07 had the emulsification effect after overexpressing pBBR-rhlABC
Week 9 (2017.8.28-9.3)
We verified that BcsA-knockout Enterobacter sp. FY-07 didn’t produce cellulose under general condition, while overexpression of BcsA can rescue the expression of cellulose in BcsA-knockout Enterobacter sp. FY-07.
4. Basic Parts Construction
Week 13-16 (2017.9.25-10.22)
1. The lyase, fimS and fimE genes are cloned in the same way as week 1.
2. Then both these segments and pSB1C3 vector were treated with restriction enzymes. After that, the three gene were separately ligated to linear pSB1C3, and the ligation products were separately transformed into DH5α.
3. Verification PCR was performed to select the positive clones. The positive strains were chosen to be cultured overnight and plasmids were extracted. After restriction enzyme digestion verification, the positive clones were sequenced.
Contacts
Address
Nankai University No.94 Weijin Road, Nankai District Tianjin, P.R.China 300071
Copyright © Team NKU-China 2017
