<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Northwestern_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Northwestern/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Northwestern</TITLE></P><DIV id="content" class="container-fluid" id="index"><NAV><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#defaultNavbar1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="defaultNavbar1"><UL class="nav navbar-nav navbar-right"><LI class="active"><A href="https://2016.igem.org/Team:Northwestern">HOME<SPAN class="sr-only">(current)</SPAN></A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Team">About Us</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Collaborations">Collaboration</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Design">Design</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">LAB</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/InterLab">Interlab</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Parts">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Basic_Part">Basic</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Integrated_Practices">Integrated</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/Engagement">Engagement</A></LI></DIV></DIV></NAV><DIV class="parallax"> </DIV></DIV><TITLE>Experiments</TITLE><DIV class="container-fluid" id="experiments"><ARTICLE><H1><STRONG>EXPERIMENTS</STRONG></H1><P>Our project consists of two main experiments:</P><H3>Cas9 periplasm-translocation:</H3><P>In order to direct Cas9 to the periplasm for packaging into OMVs, we cloned and expressed multiple versions of Cas9 fused with different outer membrane proteins and Tat/Sec translocation signal peptides. We investigated whether Cas9 was expressed first in the periplasm and outer membrane of <EM>Escherichia coli</EM>, and then in <EM>E. coli</EM> outer membrane vesicles.</P><H3>Cas9 functionality test:</H3><P>We designed a customizable gRNA template compatible with <EM>Staphylococcus aureus</EM> Cas9 (saCas9) to validate the functionality of our saCas9 complex and for other iGEM teams to use.</P><H1>Constructs</H1><P>Our basic modified Cas9 part consists of a translocation signal sequence or periplasm-targeting protein fusion, a His6 tag to facilitate Western blotting or chromatography, and saCas9 codon-optimized for expression in <EM>E. coli</EM>. Because BioBricks assembly creates scar sequences containing stop codons, these coding sequence components do not have BioBricks restriction sites between them and are therefore meant to be used as simple parts. We initially tested the part in a plasmid with the <A href="http://parts.igem.org/Part:BBa_R0040" target="_blank">BBa_R0040</A> Tet promoter, the <A href="http://parts.igem.org/Part:BBa_J61100" target="_blank">BBa_J61100</A> RBS, and the pSB1C3 plasmid set of terminators. </P><P>We chose the Tat sequence TorA, the Sec sequence Ycdo, and the SRP sequence DsbA. TorA is the native Tat signal peptide in <EM>E. coli</EM>. The other signaling sequences were chosen at recommendation of our graduate student team or from prior work done characterizing signal sequences into the two respective translocation mechanisms<SUP><A href="http://www.jbc.org/content/282/11/8309" target="_blank">1</A></SUP>. We referenced these Tat, Sec, and SRP sequences from prior literature, especially from work done by Dr. Danielle Tullman-Ercek at Northwestern University, and had them synthesized by IDT as DNA ultramers.</P><P>Protein fusions of saCas9 were made with ice nucleation protein (INP) and cytolysin ClyA, both of which are proteins associated with the outer membrane whose use in chimeric protein membrane-translocation has been characterized in prior literature<SUP><A href="http://www.pnas.org/content/107/7/3099" target="_blank">2</A>,<A href="https://www.google.com/url?q=http://www.pnas.org/content/107/7/3099" target="_blank">3</A></SUP>. INP is an outer membrane protein that promotes ice formation at temperatures above freezing, allowing bacteria to generate frost damage on plant hosts <SUP><A href="http://www.ijbs.com/v08p1097.htm" target="_blank">4</A></SUP>. Chimeric INP fusions have previously been shown to transport proteins of up to 119 kDa into OMV membranes <SUP><A href="https://www.google.com/url?q=http://www.pnas.org/content/107/7/3099" target="_blank">3</A></SUP>. ClyA is a membrane porin that serves to increase infectious toxicity by puncturing the target cell’s membrane<SUP><A href="http://www.pnas.org/content/107/7/3099" target="_blank">2</A></SUP>. Both were used to direct saCas9 to the outer membrane, a fusion that in the case of ClyA decreases the efficacy of the native protein’s toxic function<SUP><A href="http://www.sciencedirect.com/science/article/pii/S0022283608004026" target="_blank">5</A></SUP>. A truncated form of INP was extracted from the existing Biobrick part (<A href="http://parts.igem.org/Part:BBa_K523013" target="_blank">BBa_K523013</A>) along with a short flexible linker, while the ClyA sequence was referenced from the similar fusion done by Dr. Matt DeLisa at Cornell University (GFP-ClyA)<SUP><A href="http://www.pnas.org/content/107/7/3099" target="_blank">2</A></SUP>, and ordered by the team as a g-Block. </P><DIV class="row images"><DIV class="col-sm-12"><P class="caption"><STRONG>Figure 1: </STRONG>Periplasm-directing parts in tet-SB1C3. Tat (TorA), Sec (DsbA), and SRP (YcdO) peptides are cloned onto the N-terminus of Cas9 in order to correctly traffic through the Tat and Sec pathways. A truncated form of INP is also cloned onto the N-terminus per the specifications of the existing part on which it is based (BBa_K523013).</P></DIV><DIV class="col-sm-12"><P class="caption"><STRONG>Figure 2:</STRONG> Cas9-ClyA in tet-SB1C3. The ClyA periplasm-directing device fuses ClyA onto the C-terminus of Cas9, which displays Cas9 on the inside of the outer membrane vesicle<SUP><A href="http://www.sciencedirect.com/science/article/pii/S0022283608004026" target="_blank">4</A></SUP>. The additional His6 tag on the C-terminus of ClyA decreases porin toxicity to the target cell. This part was designed but not constructed. </P></DIV></DIV><P>Once we cloned our parts into the BBa_R0040 Tet promoter, we became concerned about low levels of protein expression for our protein characterization steps. We hypothesized that using a stronger promoter such as T7 would increase protein expression, so we transferred the Cas9 cassettes into a pET28a vector in the BL21 <EM>E. coli</EM> strain to utilize a T7 polymerase system to increase expression for testing (Figure 3).</P><DIV class="row images"><DIV class="col-sm-12"><P class="caption"><STRONG>Figure 3:</STRONG> Periplasm-directing parts in pET28a vector. We used the T7 expression system to create high Cas9 yields in the event that expression in the periplasm was too faint to detect.</P></DIV></DIV><P>While we constructed parts to direct Cas9 to the periplasm, we concurrently wanted to test that saCas9 functions in E. coli. We built a device that synthesizes a corresponding guide RNA to direct saCas9. Our guide RNA device consists of mRFP device (BBa_J04450), and a scaffold and guide sequence targeting mRFP flanked by a BBa_J23119 constitutive promoter (Figure 4). This 20-base pair guide sequence is flanked by two BsaI sites and can be replaced with any desired sequence using Golden Gate assembly.</P><DIV class="row images"><DIV class="col-sm-12"><P class="caption"><STRONG>Figure 4:</STRONG> Cas9 and guide RNA devices.</P></DIV></DIV><P>While we constructed parts to direct Cas9 to the periplasm, we concurrently wanted to develop a procedure that would enable us to evaluate Cas9 activity. We built a plasmid containing a constitutive mRFP fluorescent protein (BBa_J04450), a BBa_J23119 constitutive promoter driving the production of a guide (gRNA) sequence and gRNA scaffold (Figure 4). This 20-base pair gRNA sequence is flanked by two BsaI sites to enable rapid construction of gRNA for any sequence of interest using Golden Gate assembly.</P><H1>Cloning Methods</H1><P>The periplasm-directed Cas9 device and individual parts were cloned using <A href="https://2016.igem.org/Team:Northwestern/gibson" target="_blank">Gibson assembly</A>. The guide RNA device was cloned with Gibson assembly, and the customizable guide sequence was swapped out using <A href="https://2016.igem.org/Team:Northwestern/golden_gate" target="_blank">Golden Gate assembly</A>. Signal sequences for Cas9 were synthesized by IDT with homology to the ribosome binding site in the tet backbone and to the His6 tag and first few base pairs of Cas9. </P><H1>Assays</H1><H2>Periplasm Fractioning</H2><P>To test whether the Tat/Sec system and outer membrane proteins successfully transport Cas9 across the inner membrane, we first separated the outer membrane and periplasm from the rest of the cell using a cold osmotic shock <A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#ColdOsmoticShock" target="_blank">procedure</A> from the 2014 Bielefeld iGEM team. This procedure sequentially applies two buffers of different osmotic pressures to burst the outer membrane.</P><H2>OMV Purification</H2><P>Dr. Matthew DeLisa at Cornell University, with the permission of Ronald Lloubes at the Institute of Microbiology of the Mediterranean, generously donated the hypervesicular strain JC8031. This NlpI-knockout strain of <EM>E. coli</EM> produces elevated levels of OMVs due to NlpI-mediated destabilization in the outer membrane attachment to the underlying peptidoglycan layer <SUP><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4475382/" target="_blank">5</A></SUP>.</P><P>Having obtained this cell line, we transformed our initial periplasm-directed Cas9 devices (Figure 1) into JC8031 and filtered the cultures with 100 kDa Millipore filters in order to concentrate OMVs for protein expression. </P><H2>Protein Expression</H2><P>Once we built our constructs and separated the periplasm and OMVs from the rest of the cells, we ran Western blots with antibodies against our His6 tag to look for Cas9 expression in OMVs, periplasmic, and cytoplasmic fractions. Total protein concentration in each sample was normalized to 5 µg using a <A href="http://www.amresco-inc.com/media.acux?path=/media/products/dfu/dfu-m172.pdf" target="_blank">Bradford assay</A>. We used a well-tested, his-tagged ClyA-GFP fusion, also supplied by Dr. DeLisa, as a positive control for expression in the periplasm and in OMVs.</P><H2>Nuclease Activity Assay</H2><P>To test the functionality of our saCas9-compatible guide RNA device, we co-transformed our saCas9 part in pSB1C3 and our gRNA device in pSB1T3 into TOP10 <EM>E. coli</EM> cells. Once the cells took up both plasmids, they would produce both Cas9 and a targeting guide RNA to make a double-stranded break in the mRFP gene and disrupt protein function. We tested this hypothesis by reading fluorescence of OD-standardized cultures with a plate reader at 555/584 excitation/emission, using a guide RNA device with a non-cutting template sequence and a Cas9-expressing device with no guide RNA as controls. Our hypothesis was that the culture containing guide RNA against mRFP would show decreased fluorescence.</P><H1>Protocols</H1><P>The protocols we used over the summer can be found below.</P><UL><LI><A href="https://2016.igem.org/Team:Northwestern/stock" target="_blank">Reagent Stocks</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/PCR" target="_blank">PCR </A></LI><LI><A href="https://www.qiagen.com/us/resources/resourcedetail?id=3987caa6-ef28-4abd-927e-d5759d986658&amp;lang=en" target="_blank">PCR Purification </A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/gels" target="_blank">Gel Electrophoresis</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/gel_ex" target="_blank">Gel Extraction </A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/gibson" target="_blank">Gibson Assembly</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/golden_gate" target="_blank">Golden Gate Assembly</A> </LI><LI><A href="https://www.neb.com/tools-and-resources/usage-guidelines/optimizing-restriction-endonuclease-reactions" target="_blank">Restriction Digest</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/ligation" target="_blank">Phosphorylation &amp; Ligation</A></LI><LI><A href="https://www.neb.com/protocols/1/01/01/vector-dephosphorylation-protocol" target="_blank">Dephosphorylation</A></LI><LI><A href="https://drive.google.com/file/d/0B9EAXnPGprpBNXFNVmJoV2FOUHc/view" target="_blank">Transformation</A></LI><LI><A href="http://parts.igem.org/Help:Competent_Cell_Test_Kit" target="_blank">Transformation Efficiency Test</A> </LI><LI><A href="http://www.epochlifescience.com/protocol/2160250%20miniprep.pdf" target="_blank">Miniprep </A></LI><LI><A href="https://www.neb.com/~/media/NebUs/Files/Application%20Notes/appNoteM0486.pdf" target="_blank">Colony PCR </A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/sequencing" target="_blank">Sequencing </A></LI><LI><A href="https://www.addgene.org/plasmid-protocols/create-glycerol-stock/" target="_blank">Glycerol Stocks </A></LI><LI><A href="http://www.amresco-inc.com/media.acux?path=/media/products/dfu/dfu-m172.pdf" target="_blank">Bradford Assay </A></LI><LI><A href="https://2014.igem.org/Team:Bielefeld-CeBiTec/Notebook/Protocols#ColdOsmoticShock" target="_blank">Periplasm Fractioning</A></LI><LI><A href="https://2016.igem.org/Team:Northwestern/OMV_purification" target="_blank">OMV Purification</A> </LI><LI><A href="https://drive.google.com/file/d/0B9EAXnPGprpBU3NNdzdRenRJSk0/view" target="_blank">Interlab </A></LI></UL></ARTICLE><FOOTER id="nav_foot"><DIV class="row"><DIV class="col-sm-4"><DIV><P><STRONG>Northwestern University</STRONG>
                Technological Institute
                2145 Sheridan Rd
                Evanston, IL 60208</P></DIV></DIV><DIV class="col-sm-4"><P> nuigem2016&lt;at&gt;gmail.com<A href="https://twitter.com/igem_nu" target="_blank">@iGEM_NU</A></P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>