Team:Northwestern/Results
Northwestern
Toggle navigation
HOME(current)
TEAM
About Us
Attributions
Collaboration
PROJECT
Description
Design
LAB
Experiments
Results
Notebook
Safety
Interlab
PARTS
Overview
Basic
HUMAN PRACTICES
Overview
Silver
Gold
Integrated
Engagement
 
Results
RESULTS
Cas9 Translocation to Periplasm
Bradford Assay:
Before we ran a Western blot to test all of our constructs, we performed a Bradford assay to normalize the total protein loaded into each well of the protein gel, as the whole cell lysis, periplasm fractionation, and OMV concentration involved different amounts of starting cell culture. A standard curve was created using known concentrations of bovine serum albumin ranging from 0.025-0.1 μg/μL, and this standard curve was used to calculate the protein concentration for each sample. The total protein concentration present in the OMV fractions of our hypervesiculating cells was less than that of the periplasm fraction, indicating that the OMV filtration procedure we obtained from UNSW-iGEM most likely does concentrate proteins secreted by cells in OMVs (Figure 1).
Figure 1: Average total protein concentration of Tat/Sec fusions in periplasm and OMV fractions in hypervesiculating strain JC8031.
 
 
Table 1: Total protein concentrations in ug/uL.
Western Blots:
We first tested expression conditions for our saCas9 assembled part under the pBAD promoter in a constitutively expressing line (TOP 10, a TetR negative strain) versus an inducible strain (TetR positive). We expressed overnight at 20°- 25° C, using an uninduced TetR+ strain as a negative control.
Whole cell lysates and periplasm fractions were collected from the expression cultures and run in a Western blot using an anti-His tag marker. A band appears at the expected size of saCas9 for our whole cell lysates, indicating successful expression of our saCas9, though many smaller products with our N-terminus His6 tag were also present. These may have been synthesis truncation products.
Figure 2: Our first Western blot confirmed Cas9 expression in the cytosol.
After verifying that our Cas9 was expressed, we proceeded to investigate the efficacy of our periplasm localizing parts under two expression systems, pBAD and T7 in a hypervesicular cell line (JC8301) and a T7 expression line (BL21 DE3), respectively. Expression was run overnight at 37° C, and whole cell lysates, periplasm fractions, and purified OMV fractions (only for the hypervesicular line) were run on in a Western using an anti-His tag antibody against a cytosolic Cas9 negative control (no signal sequence) and a literature verified periplasm and OMV localized ClyA-GFP positive control. A Bradford assay was run on each fraction sample to control for overall protein content, and the same amount of total protein was loaded in our Western.
Our gel image shows no band at the expected size for saCas9 expression in any of the fractions, including the whole cell Cas9 control. Our result indicates that our expression conditions, the temperature for example, may not have been suitable for saCas9 protein expression. Another possibility is that the hypervesicular and T7 cell lines may not be suitable for saCas9 expression at detectable levels, since our previous Western showed expression in the TOP 10 cell line. There is a strong band where our positive ClyA-GFP controls are expected, indicating that the anti-His antibody functioned correctly and that some amount of protein was localized to the periplasm and OMVs (very faint band) of the hypervesicular cell line.
Figure 3: Our second Western blot indicated that while smaller proteins (GFP) targeted to OMVs in previous work 1 are present in both periplasm fractions and in OMV lysates, our Cas9 was not successfully translocated by any of the signal sequences or protein-fusions constructed.
Cas9 Functionality
Nuclease assay:
Having shown that Cas9 was expressed in our cells, we next wanted to evaluate if the saCas9 we were expressing was functional. To do so, we conducted a standard cutting assay using a previously validated gRNA against the fluorescent protein mCherry. In order to prove that saCas9 was active in our cells, we co-transformed a plasmid containing saCas9 and a separate plasmid containing mRFP and a guide RNA (mRFP-gRNA) targeting 20 base pairs in the middle of the mRFP coding sequence into TOP10 E. coli cells. As a control, we also co-transformed our saCas9 plasmid and plasmid containing mRFP and a control guide RNA (control-gRNA) that did not target any sequence on either plasmid. All experiments were performed with minipreps that were verified by Sanger sequencing.If the Cas9 protein is active, it should bind to mRFP and create a double-stranded break in the gene. This DNA break will be repaired by the non-homologous end joining mechanism, which either accurately restores the break or imprecisely repairs the break through the addition or deletion of DNA bases, resulting in indels or frameshift mutations. Often these imprecise repairs result in knockdown of the target gene (Figure 4). Therefore, we would expect to see fewer red colonies and low levels of mCherry fluorescence in our experimental sample, and red colonies and high mRFP expression in the samples without the guide RNA targeting mCherry.
Figure 4: Cas9 and guide RNA devices. The plasmids for our cotransformation are intended to work together; the gRNA guide contains the sequence that directs the expressed Cas9 protein to cut mRFP, our reporter protein.
Figure 5 shows that the cells co-transfected with Cas9 and the control-gRNA transformed normally with reasonable efficiency and expressed RFP, resulting in pink colonies. In Figure 6, the transformation was much less efficient, and we hypothesize that one possible reason for this is that Cas9 activity could be stressful to the cell. It produced mostly white colonies, indicating that the Cas9 combined with the mRFP-gRNA were capable of knocking down RFP expression.  We did observe seven very large pink colonies, and we speculate that the mix of phenotypes is a result of differences in DNA repair among individual cells.
Figure 5: Cas9 and gRNA template cotransformation. The cells pictured were transformed with our Cas9 plasmid and a gRNA guide that did not target any gene. Most colonies on these plates were red, as expected.
Figure 6: Cas9 and gRNA guide cotransformation. The cells pictured were transformed with our Cas9 plasmid and our gRNA plasmid targetting mRFP. Most colonies on the plates were white, indicating that the mRFP gene had been prevented from expressing.
To verify further that Cas9 knocked out mRFP expression, we grew 15 mL cultures of each experimental condition and evaluated the mRFP fluorescence using a plate reader. We also included a negative, non-fluorescent control where we grew cultures of TOP10 cells that were only transduced with Cas9 . We previously observed severe inhibition of culture growth with the co-transformed cells, probably from the stress of two plasmids. Therefore, we grew up cultures for this test without antibiotic using careful sterile technique to a standard optical density of 0.5 absorbance units. We included six biological replicates for each guide RNA condition, three biological replicates of Cas9-only, and three technical replicates for all of the above. Our Cas9 positive control gRNA cells yielded a high fluorescence and our negative control cells without a gRNA showed almost no fluorescence. Importantly, our experimental condition, Cas9 and the mRFP gRNA showed a very low fluorescence, almost as low as the negative (Cas9-only) control, indicating that our saCas9 is active and able to knock down mRFP expression. This was true across all of our biological replicates. Figure 7 shows that gRNA with mRFP-targeting guide did not fluoresce, as we expected.
Going forward for this experiment, we would need to sequence the mRFP insert from the cells affected by Cas9 to confirm that Cas9 did in fact edit mRFP. This can be done by sequencing PCR fragments from colony PCR or by deep sequencing.
Figure 7: Fluorescence of TOP10 cultures with control-gRNA and Cas9, mRFP-gRNA and Cas9, and Cas9 only negative control. 100 uL of each sample was measured in triplicate using a BioTek Synergy plate reader.  Error bars represent one standard deviation, n represents number of biological replicates.
Northwestern University
Technological Institute
2145 Sheridan Rd
Evanston, IL 60208
nuigem2016<at>gmail.com
@iGEM_NU
