<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><UL class="navigation-menu" style=""><LI><A href="https://2013.igem.org/Team:Imperial_College">Project: Plasticity</A></LI><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Imperial_College/tour">At a glance</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Background">Waste Issue</A></LI><LI>M.A.P.L.E. Modules</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Waste_Degradation:_SRF">M1: Resource-full Waste</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/BioPlastic_Recycling:_PHB">M2: Plastic Fantastic</A></LI></UL><LI><A href="https://2013.igem.org/Team:Imperial_College/Applications">Bioplastic applications</A></LI><LI>Human Practices</LI><LI class="maple"><A href="https://2013.igem.org/Team:Imperial_College/MAPLE">M.A.P.L.E</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Industrial_Implementation">Industrial Implementation</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Managing Waste">Social Challenge</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Communication_work">Communicating our Project</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Project_Evolution">Project Evolution</A></LI><LI>Achievements</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Judging_Criteria">Judging Criteria</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/mainresults">Main Results</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Collaboration">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Biobricks">Biobricks</A></LI><LI>In the Lab</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/MSDS">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/labbook">Lab book</A></LI><LI><A href="#">Modelling Notes</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Modelling_Intro">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Modelling_beta-amyloid">Beta amyloid degradation</A></LI><LI><A href="#">Data</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Growth_Assays">Growth Assays</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Western Blots">Western Blot</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Enzyme Kinetics">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/PHB production">PHB production</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/3HB_assay">3HB assay</A></LI><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Imperial_College/Clearing_Zone_Assay">Clearing Assays</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Electron microscopy">Electron microscopy</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Protocols">Protocols</A></LI><LI>Team</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Meet_the_Team">Students</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Advisors">Advisors</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Attributions">Attributions</A></LI><DIV id="main-content"><DIV id="maincontent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Materials"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Waste_Conditioned_Media_.28WCM.29"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Waste Conditioned Media (WCM)</SPAN></A></LI><LI class="toclevel-2"><A href="#3.25_glucose_Nile_Red_plates"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">3% glucose Nile Red plates</SPAN></A></LI><LI class="toclevel-2"><A href="#M9_minimal_and_supplemented_media"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">M9 minimal  and supplemented media</SPAN></A></LI><LI class="toclevel-2"><A href="#Xylose_and_Arabinose"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Xylose and Arabinose</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Protocols_and_Assays"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Protocols and Assays</SPAN></A></LI><LI class="toclevel-2"><A href="#P3HB_production_and_extraction"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">P3HB production and extraction</SPAN></A></LI><LI class="toclevel-3"><A href="#Extracting_PHB"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Extracting PHB</SPAN></A></LI><LI class="toclevel-3"><A href="#LB-Agar_Plates_containing_Nile_Red_stain"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">LB-Agar Plates containing Nile Red stain</SPAN></A></LI><LI class="toclevel-3"><A href="#Nile_Red_Culture_staining"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Nile Red Culture staining</SPAN></A></LI><LI class="toclevel-3"><A href="#Fluorescent_Microscopy"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">Fluorescent Microscopy</SPAN></A></LI><LI class="toclevel-3"><A href="#Relationship_between_OD_and_dry_biomass_for_phaCAB_operon_containing_cells"><SPAN class="tocnumber">2.1.5</SPAN><SPAN class="toctext">Relationship between OD and dry biomass for phaCAB operon containing cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Cell_lysate_activity_assay"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Cell lysate activity assay</SPAN></A></LI><LI class="toclevel-2"><A href="#3HB_assay"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">3HB assay</SPAN></A></LI><LI class="toclevel-3"><A href="#Sample_Preparation:_3HB_from_P3HB"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Sample Preparation: 3HB from P3HB</SPAN></A></LI><LI class="toclevel-2"><A href="#Cloning"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Cloning</SPAN></A></LI><LI class="toclevel-2"><A href="#Waste_Growth_Assay"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Waste Growth Assay</SPAN></A></LI><LI class="toclevel-2"><A href="#Growth_assays"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Growth assays</SPAN></A></LI><LI class="toclevel-3"><A href="#3HB_and_Acetoacetate_LB_and_M9_growth_for_P3HB_synthesis"><SPAN class="tocnumber">2.6.1</SPAN><SPAN class="toctext">3HB and Acetoacetate LB and M9 growth for P3HB synthesis</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation_of_Poly_DEGA_plates."><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Preparation of Poly DEGA plates.</SPAN></A></LI><LI class="toclevel-2"><A href="#Western_blot"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Western blot</SPAN></A></LI><LI class="toclevel-2"><A href="#Clearing_zone_assay"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">Clearing zone assay</SPAN></A></LI><LI class="toclevel-2"><A href="#Protein_purification_by_metal_affinity_chromatography"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">Protein purification by metal affinity chromatography</SPAN></A></LI><LI class="toclevel-2"><A href="#Proteinase_K_kinetic_activity_assay"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">Proteinase K kinetic activity assay</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Materials">Materials</SPAN></H1><H3><SPAN class="mw-headline" id="Waste_Conditioned_Media_.28WCM.29">Waste Conditioned Media (WCM)</SPAN></H3><P>We added 1g <A href="http://en.wikipedia.org/wiki/Refuse-derived_fuel" class="external text" rel="nofollow">SRF</A> (Solid Recovered Fuel) /50 mL LB and then autoclaved the mixture. Once autoclaved, large waste chunks were removed through filter sterilisation (0.2 µm filters) before being used in growth assays as waste conditioned media (WCM).

SRF refers to the refuse from recycling facilities that has no value currently and is incinerated to produce power at a cost to the recycling facility, it is composed of 30% plastics while the rest is cellulosic waste</P><H3><SPAN class="mw-headline" id="3.25_glucose_Nile_Red_plates">3% glucose Nile Red plates</SPAN></H3><P>When making up LB agar media, add 3% glucose then autoclave. When autoclaving it is <B>essential</B> that you keep temperature &lt;125°C. Failure to do so will result in caramelisation (160°C) of the media, which will show as a darkening of the media. With this complete, 60 µL Nile Red was added in addition to 150 µL chloramphenicol in a 300 mL final volume. Plates were then poured and left to dry. Another successful mechanism to thwart caramelisation is to autoclave the LB-agar then add the 3% glucose after. Follow this with a 30 second microwave step and then an 80°C water bath for a couple hours to thoroughly sterilise.</P><H3><SPAN class="mw-headline" id="M9_minimal_and_supplemented_media">M9 minimal  and supplemented media</SPAN></H3><P>First made stocks of MgSO4 (246.48 g/mol), CaCl2 (147.02 g/mol), and thiamine hydrochloride (337.3 g/mol):
MgSO4: 100 mM - 2.46g in 100 mL ddH2O
CaCl2: 20 mM - 0.29g in 100 mL ddH2O
Thiamine hydrochloride: 50 mM - 0.84g in 50 mL ddH2O
The bottles used were pre-autoclaved. When all media was prepared, it was sterile filtrated, as autoclaving results in the formation of magnesium precipitate. See below for correct volumes and mass to add where [final] represents desired values.
</P><H3><SPAN class="mw-headline" id="Xylose_and_Arabinose">Xylose and Arabinose</SPAN></H3><P>For induction assays, 2% xylose was prepared, as was 6 µM arabinose.
</P><H1><SPAN class="mw-headline" id="Protocols_and_Assays">Protocols and Assays</SPAN></H1><H3><SPAN class="mw-headline" id="P3HB_production_and_extraction">P3HB production and extraction</SPAN></H3><H4><SPAN class="mw-headline" id="Extracting_PHB">Extracting PHB</SPAN></H4><UL><LI>Centrifuge settings: 4000RPM, 10mins.</LI><LI> Scale as appropriate.</LI><LI> After each centrifuge step the supernatant should be poured off. </LI><LI> This should provide PHB with 99% purity and a high molecular weight. </LI></UL><OL><LI>Centrifuge 50ml culture and resuspend in PBS.</LI><LI>Resuspend pellet in 5ml Triton X-100(1% v/v in PBS) for 30mins at room temp. </LI><LI>Centrifuge, resuspend in 5ml PBS.</LI><LI>Centrifuge, add 5ml sodium hyperchlorite solution and incubate at 30˚C for 1 hour.</LI><LI>Centrifuge, wash with 5ml distilled water and 70% EtOH several times.</LI><LI>Allow powder to dry.</LI></OL><P>Shahryar Shakeri, Comparison of intracellular polyhydroxybutyrate granules formation between different bacterial cell subpopulations by flow cytometry, Jundishapur Journal of Microbiology 2011</P><P><A href="http://jjmicrobiol.com/?page=article&amp;article_id=2342" class="external autonumber" rel="nofollow">[1]</A></P><H4><SPAN class="mw-headline" id="LB-Agar_Plates_containing_Nile_Red_stain">LB-Agar Plates containing Nile Red stain</SPAN></H4><P>Nile Red final concentration in media should be 0.5µg/ml</P><OL><LI>After autoclaving 300ml of LB-Agar, add 600µl Nile Red solution(0.25mg/ml DMSO) </LI><LI>Add 150µl chloramphenicol </LI><LI>Pour plates in a clean fume cupboard</LI></OL><H4><SPAN class="mw-headline" id="Nile_Red_Culture_staining">Nile Red Culture staining</SPAN></H4><P>Nile Red final concentration should be 20µg/ml media</P><P>Add 320µl to 4 ml overnight culture and incubate for 30mins at room temp. </P><H4><SPAN class="mw-headline" id="Fluorescent_Microscopy">Fluorescent Microscopy</SPAN></H4><P>Agar pads are first made</P><OL><LI>Small pads are made using 1% agarose in M9 minimal media between microscope slides.                             </LI><LI>2ul of the sample is added to each pad and imaged with the fluorescent microscope.                             </LI></OL><H4><SPAN class="mw-headline" id="Relationship_between_OD_and_dry_biomass_for_phaCAB_operon_containing_cells">Relationship between OD and dry biomass for phaCAB operon containing cells</SPAN></H4><OL><LI>Grow 3 large cultures in LB and 3% glucose - inoculate with 1ml OD 0.1 and grow for 24h.                             </LI><LI>Calculate OD of original culture and dilute until there is a range of 0.2 - 1.0 OD(try to have large volumes for low OD). </LI><LI>Weigh centrifuge containers.                                                                                              </LI><LI>Transfer solutions to these containers, recording what volume of each OD is transferred. .                                </LI><LI>Centrifuge these solutions down and dry the pellet out until constant weight in a 50°C oven.                                          </LI><LI>Dry cell mass = (container mass + dry cell mass)/container mass                                                           </LI><LI>Plot OD against dry cell mass.                                                                                            </LI></OL><H2><SPAN class="mw-headline" id="Cell_lysate_activity_assay">Cell lysate activity assay</SPAN></H2><OL><LI><B>Preparation of the cell lysate:</B></LI><P>5 mL cultures of EstC2 <A href="http://parts.igem.org/Part:BBa_K1149002" class="external text" rel="nofollow">BBa_K1149002</A>, PueB <A href="http://parts.igem.org/Part:BBa_K1149004" class="external text" rel="nofollow">BBa_K1149004</A> and PulA <A href="http://parts.igem.org/Part:BBa_K1149006" class="external text" rel="nofollow">BBa_K1149006</A> were incubated overnight. The next morning, these were diluted in 200 mL and incubated until the OD reached 0.5-0.6. 
</P>
The cultures were then induced with 6mM arabinose (final concentration)and left overnight. The following morning the cells were spun down, resuspended in 100 mM potassium phosphate buffer of pH 7.4 and probe-sonicated with 2 minute cycles of 2 seconds on, 2 seconds off, at 60% amplitude. The samples were kept on ice throughout the sonication process. The cell debris was pelleted by spinning and the supernatant was removed for utilising in the enzyme activity assays.<LI><B>Substrate preparation:</B></LI><P>The substrate was prepared by the addition of 8.8 µL of para-Nitrophenyl butyrate to 1 mL of Acetonitril. 
</P><LI><B>Incubation of the reagents:</B></LI><P>In 10mm cuvettes, 5 µL of the EstC2, PueB, PulA, and Empty vector were added to 1 mL of 100 mM Potassium Phosphate buffer of pH 7.4 and incubated for 2-3 minutes. For the Substrate alone assay, simply 1 mL of buffer was incubated.
</P><LI><B>Spectrometer programme:</B></LI><P>The Eppendorf BioSpectrometer was set up to run for 600 seconds, recording the absorbance at 405 nm every 30 seconds with 0 seconds delay, whilst incubating the reaction mixture at 37°C. For each run the spectrometer was first blanked with 1 mL of buffer and 5 µL of the corresponding cell lysate. For Substrate alone, the spectrometer was blanked with 1 mL of buffer.
</P><LI><B>Reaction mixture:</B></LI>
When the cuvettes were in place in the spectrometer, 4 µL of para-Nitrophenyl butyrate solution was added to each. The reaction mixture was mixed by pipetting up and down and the spectrometer programme was started immediately afterwards.</OL><H2><SPAN class="mw-headline" id="3HB_assay">3HB assay</SPAN></H2><P>We used a 3-hydroxybutyrate colorimetric assay kit. You can download the description of the product and the protocol  <A href="https://www.caymanchem.com/pdfs/700190.pdf" class="external text" rel="nofollow">here</A>. The chemical reaction behind the assay is illustrated on the image below:</P><H4><SPAN class="mw-headline" id="Sample_Preparation:_3HB_from_P3HB">Sample Preparation: 3HB from P3HB</SPAN></H4><UL><P>We treated around 0.1-0.5g of P3HB purified from various experiments (and from Sigma) with 10uL of <A href="http://parts.igem.org/Part:BBa_K1149010" class="external text" rel="nofollow">phaZ1</A> enzyme in 800uL tris buffer. The next day, we centrifuged the samples at 13 000rpm for 10 minutes and used the supernatant in different dilutions with the Assay Buffer (at 90%, 50% and, 10%).</P><LI><B>Sample 1:</B> Bioplastic made by <A href="http://parts.igem.org/Part:BBa_K1149051" class="external text" rel="nofollow">hybrid phaCAB</A> grown on 3% glucose LB</LI><LI><B>Sample 2:</B> &quot;the grey stuff that is directly from waste&quot;.  </LI><LI><B>Sample 3:</B> P(3HB) from Sigma (<A href="http://www.sigmaaldrich.com/catalog/product/aldrich/363502?lang=en&amp;region=GB" class="external text" rel="nofollow">CAS no. 29435-48-1</A>) treated with phaZ1</LI><LI><B>Sample 4:</B> P(3HB) from Sigma (<A href="http://www.sigmaaldrich.com/catalog/product/aldrich/363502?lang=en&amp;region=GB" class="external text" rel="nofollow">CAS no. 29435-48-1</A>) untreated, to determine signal from P(3HB)</LI><P>
We did all samples in a 96 well plate) in triplicate. Measurements were taken after 40 minutes incubation with Developer Solution at 20°C. plate setup:
</P></UL><H2><SPAN class="mw-headline" id="Cloning">Cloning</SPAN></H2><P><B>PCR</B></P><UL><LI>We used Pfu Ultra polymerase for all reactions where we use the DNA for anything afterwards. Download the exact manual from <A href="https://www.chem.agilent.com/Library/usermanuals/Public/600670.pdf" class="external text" rel="nofollow">here</A>.</LI><LI>We used MyTaq polymerase for colony PCR and checking plasmids. You can download the manual <A href="http://www.bioline.com/documents/product_inserts/MyTaq%E2%84%A2%20DNA%20Polymerase.pdf#zoom=130" class="external text" rel="nofollow">here</A>.</LI><LI>Our super Thermal Cycler was provided by our generous sponsor, Eppendorf. </LI></UL><P><B>Running gels</B></P><P>We used SybrSafe to stain the DNA and imaged in a Transilluminator. We cut out bands under a bluebox.
<B>Minipreps</B></P><UL>
We did minipreps from 4mL overnight cultures in LB, supplemented with antibiotics.
<LI>Chlo: 50 mg/mL</LI><LI>Amp:100 mg/mL</LI><LI>Kan: 100 mg/mL</LI><LI>We used <A href="http://www.qiagen.com/Products/Catalog/Sample-Technologies/DNA-Sample-Technologies/Plasmid-DNA/QIAprep-Spin-Miniprep-Kit" class="external text" rel="nofollow">Quiagen Miniprep kit</A>.</LI><P>We got an excellent tabletop centrifuge courtesy of Eppendorf, which has served us very well. 
</P></UL><P><B>Digests and Ligations</B></P><P> For restriction digests, we used <A href="https://www.neb.com/" class="external text" rel="nofollow">NEB</A> enzymes and used the companys protocols accordingly. For ligations, we used T4 ligase. We treated backbones with alkaline phosphatase before ligations and we have used Dpn1 treatment sometimes to degrade the original plasmid. </P><P><B>Sequencing</B></P><UL><LI>We used VF2, VR, G1004 and G1005 primers in order to verify the sequence of most of the biobrick parts. We also designed internal sequencing primers to verify the reads in long parts, such as phaCAB. </LI><LI><A href="http://parts.igem.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2 (BBa_G00100)</A>	tgccacctgacgtctaagaa	Tm=62</LI><LI><A href="http://parts.igem.org/Part:BBa_G00100" class="external text" rel="nofollow">VR (BBa_G00101)</A>	attaccgcctttgagtgagc	Tm=60</LI><LI><A href="http://parts.igem.org/Part:BBa_G1004" class="external text" rel="nofollow">BBa_G1004</A> (=prefix)      	gtttcttcgaattcgcggccgcttctag	Tm=63</LI><LI><A href="http://parts.igem.org/Part:BBa_G1004" class="external text" rel="nofollow">BBa_G1005</A> (=revcompl sufix)	gtttcttcctgcagcggccgctactagta	Tm=64</LI></UL><P><B>Transformation</B></P><P> We transformed our plasmids finally into MG1655. </P><P>For ligations we used mainly <A href="https://www.neb.com/products/c3019-neb-10-beta-competent-e-coli-high-efficiency" class="external text" rel="nofollow">NEB10</A> and some TOP10 and NEB5 cells for high efficiency. </P><H2><SPAN class="mw-headline" id="Waste_Growth_Assay">Waste Growth Assay</SPAN></H2><P>
Overnight (O/N) cultures of MG1655 transformed with (BBa_K639003) were diluted to OD 0.05 in either fresh LB or waste conditioned media and plated into 96 well plates (200 µl/well). OD600 was read at the indicated time points and media only (LB) was taken away as background signal. 

In addition to this a qualitative assay as performed. 9 mL mCherry bacteria were transferred into 300 mL of autoclaved SRF in LB, which were placed in a shaking incubator at 37°C. After 2 days 200 µL of this solution was plated out on chloramphenicol plates, this was then repeated a week later to gauge whether or not the bacteria were still alive.

To show that the MG1655 could grow on waste solely, they were grown in PBS (phosphate buffered saline) and autoclaved waste. In 100 mL PBS + waste, 1 mL MG1655 mCherry E.coli were added. They were then grown in a shaking incubator at 37°C and samples were plated after 2 days and 6 days.
</P><H2><SPAN class="mw-headline" id="Growth_assays">Growth assays</SPAN></H2><H3><SPAN class="mw-headline" id="3HB_and_Acetoacetate_LB_and_M9_growth_for_P3HB_synthesis">3HB and Acetoacetate LB and M9 growth for P3HB synthesis</SPAN></H3><P>
3HB solutions were prepared in advance. An initial stock of 100mM 3HB was made from 0.315g 3HB (FW:126) dissolved in 25 mL ddH2O. This was then diluted down to 10mM, 1mM, 100µM, 1µM working solutions in LB. Furthermore, 3% glucose in LB was prepared with solutions made from 3g glucose in 100 mL LB.

O/N cultures were prepared, these were diluted to OD=0.05 by measuring O/N OD0.05. With this complete the required dilutions of cells were made into 1.5 mL reaction tubes to give a total volume of 800 µL, this was then divided up into 4 wells for quadruplicate technical repeats. Cells used were phaABC, which is involved in P3HB synthesis from glucose, phaBC which synthesises P3HB from acetoacetate and EV empty vector which was used as a growth control.

With the plate loaded, it was then placed in a shaking incubator at 700 rpm, 37°C for 30 minutes. After this, we took readings using a robotic plate reader for t=0 at OD600, then repeated this over a further 7 hours.

A second plate was prepared to measure acetoacetate as a sole carbon source. Lithium acetoacetate (FW: 108.02) was prepared by placing 10 mg in 9.2 mL water to give a stock concentration of acetoacetate of 10 mM. Dilutions were made to 100 µM and 1µM. Ethylene glycol growth assays were prepared by diluting 1.11 g/L stock solution by 1000 and 500 to give final concentration of 100 mM and 200 mM respectively. Emulsions were prepared as described below. These were further diluted from stock solutions of 1.25 g/L to working solutions of 31 mg/L and 31 µg/L.
</P><H2><SPAN class="mw-headline" id="Preparation_of_Poly_DEGA_plates.">Preparation of Poly DEGA plates.</SPAN></H2><P>
For 320 mL final media, place 1.5 mL 0.5% poly DEGA in 18.5 mL ddH20. As poly DEGA is very viscous, the 20 mL solution was sonicated for 45 minutes. This complete, it was added to LB agar kept at 70°C along with 25 ug/mL chloramphenicol. The solution was then separated into 3 separate duran bottles of 100 mL volume. 0.21g xylose was added to a bottle, 0.21g arabinose to another and the other did not have an inducer added. The media was then plated out.
</P><H2><SPAN class="mw-headline" id="Western_blot">Western blot</SPAN></H2><P><B>
Sample preparation
</B></P><OL><LI>Prepare concentrated induction media. (Dissolve 0.134g Arabinose in 8ml pre-autoclaved LB, 1.52g xylose in 4ml pre-autoclaved LB). Filter sterilise induction media. </LI><LI> Make 5 mL overnight culture from the transformed cells.
</LI><LI> Next morning, dilute 0.5 ml overnight culture in 9.5 ml of LB media. The subsequent inducer concentrations are 6mM arabinose and 2% xylose. 
</LI><LI> Then grow the diluted culture at 37°C, and measure OD600 every hour until  the OD reaches 0.5-0.6.
</LI><LI> Then induce protein expression with the appropriate induction media, and grow the induced cells at 37°C for 5 hours.
</LI><LI> Spin 2ml of induced cells, collect 60ul of the supernatant with a 1.5ml tube, and add 20 ul western loading buffer to the supernatant. 
</LI><LI> Resuspend the pellet with 100 ul of lysis buffer. Wait for the cells to lyse (15-20 minutes) at room temperature. Vortex is allowed. 
</LI><LI> Add 33 ul of western loading buffer to the lysate.
</LI><LI> Boil the sample at 95°C for 5 minutes and freeze at -20°C.  
</LI></OL><P><B> Gel electrophoresis  </B></P><UL><LI>Place 15ul of the sample in each well. Run the gel at 60V for 1 hour and 20 minutes.</LI></UL><P><B> Transfer </B></P><UL><LI>Use <A href="http://www.lifetechnologies.com/uk/en/home/life-science/protein-expression-and-analysis/western-blotting/western-blot-transfer/iblot-dry-blotting-system.html" class="external text" rel="nofollow">iBlot® Dry Blotting System (Invitrogen)</A> to perform dry blotting.
</LI></UL><P><B> Antibody binding</B></P><P>All-in-one his antibody (from mouse) is used for proteins with his-tags. Primary and secondary Flag-antibodies are used for Flag-tagged proteins.
Mix 7 ml of 5% w/v non-fat dry milk with diluted antibodies, and place the membrane in the mixture.
Incubate for 1 hour at room temperature.
Wash 210 min with 15 ml 0.05% PHB-Tween.
Wash 25 min with 15 ml PBS.
&lt;/p&gt;</P><P><B>
Visualisation
</B></P><P>
Visualise the protein bands using <A href="http://www.lifetechnologies.com/1/1/28062-novex-ap-mouse-chemiluminescent-detection-kit.html" class="external text" rel="nofollow">Chemiluminescence detection kit (Invitrogen)</A>. Protein-specific signal is then captured by a chemiluminescent-compatible X-ray film.
</P><H2><SPAN class="mw-headline" id="Clearing_zone_assay">Clearing zone assay</SPAN></H2><OL><LI>Prepare 10 ml cell culture and induce phaZ1 expression using 2% xylose media. </LI><LI>Centrifuge the cell cultures at 4500 RCF (equivalent to 6221g) for 20 minutes at 4°C.</LI><LI>Add 2 ml of 50mM Tris buffer (pH 7.4) to the pellet and mix well by vortexing the tube. </LI><LI>Transfer the buffered pellet into 7ml tubes and keep the samples on ice.</LI><LI>Perform probe sonication with 2-minute cycles of 2 seconds on, 2 seconds off, at 60% amplitude. Keep the sample on ice while sonicating.</LI><LI>Centrifuge the samples again at 4000 g for 10 minutes and extract the supernatant, in order to separate cell debris and proteins.</LI><LI>Mix 0.5g of P3HB powder with 300 ml LB agar, make P3HB plates.</LI><LI>Use sterilised pipette tip to cut two wells, each has a 0.5 mm diameter. </LI><LI>Pipette 200ul of PhaZ1 cell lysate sample in one of the wells, and same amount of empty vector cell lysate sample in the other well.</LI><LI>Observe the plate every 24 hours.</LI></OL><H2><SPAN class="mw-headline" id="Protein_purification_by_metal_affinity_chromatography">Protein purification by metal affinity chromatography</SPAN></H2><OL><LI><B>Preparation:</B></LI><P>10 mL cultures were incubated overnight. These were diluted in 500 mL the next morning and incubated until the OD reached 0.5-0.6. The cultures were then induced with 6mM arabinose or 2% xylose (final concentration)and left overnight. The cells were then pelleted by centrifugation, 4,000 rpm for 10 minutes and kept in a -80ºC freezer.
</P><LI><B>Cell lysis</B></LI><P>The frozen pellets were allowed to thaw and resuspended in 10 mL of Tris-HCl binding buffer. The cells were lysed by probe-sonication at 60% amplitude with 2 minute cycles of 2 seconds on, 2 seconds off. The samples were kept on ice throughout the sonication process. The cell debris was pelleted by spinning and the supernatant removed for purification.
</P><LI><B>Purification with a Ni-NTA column:</B></LI><P>The column was washed with Buffer A (10 mM imidazole, 40 mM Tris pH 8, and 400 mM NaCl) to equilibrate the resin. The samples are added and the column is washed with Buffer A, and then eluted with increasing concentrations of Buffer B (500 mM imidazole, 40 mM Tris pH 8 and 400 mM NaCl). 
</P><LI><B>SDS PAGE:</B></LI><P>Samples were prepared for SDS PAGE by adding 7 µL of SDS running buffer to 20 µL of the fractions of interest. 8-20 µL of each sample were run on a polyacrylamide gel against a molecular weight marker.
</P><LI><B>Concentration:</B></LI><P><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1149010" class="external text" rel="nofollow">PHB depolymerase (phaz 1)</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1149002" class="external text" rel="nofollow">PUR esterase (EstC2)</A> were concentrated with 10 mM potassium phosphate buffer.
</P><H2><SPAN class="mw-headline" id="Proteinase_K_kinetic_activity_assay">Proteinase K kinetic activity assay</SPAN></H2><OL><LI><B>Tris-HCl buffer ingredients:</B></LI><P>1 litre of millicule water 
10.5 g of NaCl (180 mM)
0.65 g of CaCl2 (5 mM) 
1.7g of imidazole (25 mM)
6 g of Tris base (50 mM)
Add HCl to bring the pH to 7.4
</P><LI><B>Programme the spectrometer:</B></LI><P>Cuvette: 10 mm
Wavelength: 405 nm
Temperature: 37°C
Measuring time: 10:00 min:sec
Delay: 00:00 min:sec
Interval: 00:30 min:sec
</P><LI><B>Reagent preparation:</B></LI><P>Lyophilized powder proteinase K from Tritirachium album from Sigma-Aldrich was used, made into a 10 mg/ml stock solution with the buffer. The spectrometer was blanked with 1mL of buffer and 3 µL of proteinase K stock solution; this is equivalent to one unit of enzyme. 
</P><P>We used N-Succinyl Ala-Ala-Pro-Leu p-nitroanilide (AAPLp-NA) from Sigma-Aldrich as a substrate analog. Make a stock solution with buffer (10 mg/ml). The reagents were incubated at 37°C just before use. 
</P><P>The absorbance of each reaction mixture was measured at 405 nm, for 10 minutes at 30 second intervals. The reaction mixtures contained 3 µL of proteinase K stock solution and AAPLp-NA in Tris-HCl buffer to make final concentrations of 10 µM, 25 µM, 50 µM, 75 µM, 100 µM, 250 µM and 500 µM.
</P><DIV id="supporters" class="clear"><P><B>Our Sponsors</B></P><A href="/File:Logo-bioline.gif" title="File:Logo-bioline.gif">125px</A></DIV></OL></OL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Imperial_College/Protocols">http://2013.igem.org/Team:Imperial_College/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College/Protocols&amp;oldid=314973" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>