<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_BioPlastic_Recycling_PHB"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College/BioPlastic_Recycling:_PHB">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College/BioPlastic_Recycling:_PHB&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College/BioPlastic_Recycling:_PHB&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College/BioPlastic_Recycling:_PHB&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College/BioPlastic_Recycling:_PHB" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College/BioPlastic Recycling: PHB</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><UL class="navigation-menu" style=""><LI><A href="https://2013.igem.org/Team:Imperial_College">Project: Plasticity</A></LI><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Imperial_College/tour">At a glance</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Background">Waste Issue</A></LI><LI>M.A.P.L.E. Modules</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Waste_Degradation:_SRF">M1: Resource-full Waste</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/BioPlastic_Recycling:_PHB">M2: Plastic Fantastic</A></LI></UL><LI><A href="https://2013.igem.org/Team:Imperial_College/Applications">Bioplastic applications</A></LI><LI>Human Practices</LI><LI class="maple"><A href="https://2013.igem.org/Team:Imperial_College/MAPLE">M.A.P.L.E</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Industrial_Implementation">Industrial Implementation</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Managing Waste">Social Challenge</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Communication_work">Communicating our Project</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Project_Evolution">Project Evolution</A></LI><LI>Achievements</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Judging_Criteria">Judging Criteria</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/mainresults">Main Results</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Collaboration">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Biobricks">Biobricks</A></LI><LI>In the Lab</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/MSDS">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/labbook">Lab book</A></LI><LI><A href="#">Modelling Notes</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Modelling_Intro">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Modelling_beta-amyloid">Beta amyloid degradation</A></LI><LI><A href="#">Data</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Growth_Assays">Growth Assays</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Western Blots">Western Blot</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Enzyme Kinetics">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/PHB production">PHB production</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/3HB_assay">3HB assay</A></LI><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Imperial_College/Clearing_Zone_Assay">Clearing Assays</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Electron microscopy">Electron microscopy</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Protocols">Protocols</A></LI><LI>Team</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Meet_the_Team">Students</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Advisors">Advisors</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Attributions">Attributions</A></LI><DIV id="main-content"><H1><SPAN class="mw-headline" id="Module_2:_Plastic_Fantastic">Module 2: Plastic Fantastic</SPAN></H1><P><B>Plastic Fantastic is a complete P(3HB) bioplastic recycling platform, where P(3HB)  is degraded into monomeric form and then re-polymerised back into de novo P(3HB) for future applications.</B></P><DIV id="TabbedPanels1" class="TabbedPanels"><UL class="TabbedPanelsTabGroup"><LI class="TabbedPanelsTab" tabindex="0"><H4>Overview</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Specifications</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Design</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Modelling</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Testing &amp; results</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Future work</H4></LI></UL><DIV class="TabbedPanelsContentGroup"><DIV class="TabbedPanelsContent"><H3 id="overview">Overview</H3><P align="justify">We have developed a system by which PHB can be recycled when products made from it come to the end of their life. In order to do this, we have engineered <I>E.coli</I> to break down P3HB extracellularly to release the monomers of 3-hydroxybutyrate (3-HB). We have designed the metabolic connection between P3HB degradation and synthesis so that 3-HB can be used as a carbon source. We are testing a putative 3-HB permease to increase the rate of uptake of the molecule into the cell. An intracellular 3-HB dehydrogenase will then form acetoacetate from 3-HB which the cell can utilise. We have thoroughly characterised the phaZ1 PHB depolymerase enzyme and demonstrated that it degrades PHB. Our models predict that use of the enzyme on a bioreactor scale will be effective in degrading bioplastic. </P><H4><SPAN class="mw-headline" id="We_are_also_establishing_a_second_bioplastic_recycling_platform_for_polylactic_acid_.28PLA.29">We are also establishing a second bioplastic recycling platform for polylactic acid (PLA)</SPAN></H4><P>This aspect of our module is in collaboration with the <A href="https://2013.igem.org/Team:Imperial_College/Collaboration#Yale:_PLA-degradation" class="external text" rel="nofollow">Yale iGEM team.</A></P><P>Yale are working to biologically synthesise PLA. We have designed <I>E.coli</I> to secrete proteinase K to form lactic acid monomers, forming half of a second bioplastic recycling system. Lactic acid is used in the chemical synthesis of PLA and so this system could also be implemented in an industrial setting. 


</P></DIV><DIV class="TabbedPanelsContent"><H2 id="specification">Specification</H2><H4> Degradation of P(3HB)</H4><P><B>Our bacteria should resist any toxic effects that are associated with P(3HB) or 3HB</B></P><P><B>Our bacteria should secrete a functional (P3HB) degrading enzyme</B></P><H4> Synthesis of P(3HB) </H4><P><B>Our bacteria should take up and internalise 3HB from the surrounding media</B></P><P><B>Our bacteria should be able to utilise P(3HB) as a sole carbon source</B></P><H4> Degradation of PLA </H4><P><B>Our bacteria should secrete a functional PLA degrading enzyme</B></P><P><B>Our bacteria should be tolerant to lactic acid</B></P></DIV><DIV class="TabbedPanelsContent"><DIV id="CollapsiblePanel11" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>P(3HB) Degradation: Pathway<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent">The phaZ1 enzyme from <I>Ralstonia eutropha</I> can break down PHB. This bacterium uses PHB as storage molecule and expresses PhaZ1 when it needs to mobilise the nutrients from the polymer inside the cell. In our system, the enzyme will be secreted.
</DIV></DIV><DIV id="CollapsiblePanel12" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4> P(3HB) Degradation: Biobrick design<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P><A href="http://parts.igem.org/Part:BBa_K1149010" class="external text" rel="nofollow">BBa_K1149010</A>: <B>Extracellular expression of phaZ1, PHB depolymerase enzyme.</B> The construct design incorporates the RBS 0034 and is regulated by a strong xylose-inducible promoter <A href="http://parts.igem.org/Part:BBa_I741018" class="external text" rel="nofollow">BBa_I741018</A>. For extracellular secretion, we fused pelB secretion tag <A href="http://parts.igem.org/Part:BBa_J32015" class="external text" rel="nofollow">BBa_J32015</A> to the N terminus of the protein.  </P><P>
There are many strategies for secretion in <I>E.coli</I> and you can read about these on our <A href="https://2013.igem.org/Team:Imperial_College/SecretionHelp" class="external text" rel="nofollow">Secretion Guide page</A> and find the corresponding biobricks and iGEM projects. If you are currently in the process of designing a Genetically Engineered Machine and aim to secrete a protein in E.coli, this page provides a good starting point to look for information.
</P></DIV></DIV><DIV id="CollapsiblePanel13" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Synthesis of P(3HB) from 3HB: Pathway<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P>For the Plastic Fantastic recycling module, we designed a modifications to the PHB synthesis pathway of the Resourceful Waste module. The additional steps connect 3-Hydroxy-Butyrate to the downstream metabolite of the synthesis: acetoacetyl-CoA where from the next two steps are carried out by phaB and phaA enzymes, as described in Module1.  This pathway contains both new enzymes that we are introducing to E.coli and also endogenous enzymes, such as AtoA/D. Because of this, the pathway is in connection with the cells normal metabolism and our models were of great use for predicting the flux through the pathway and informing the design.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Plastic synthesis from 3HB monomers.</B> 3HB is taken up into the cell by the putative permease. The 3HB is then converted to P(3HB) by the pathways shown. The genes in <SPAN style="color:#006600"><B>green</B></SPAN> are those we have expressed within MG1655. Our system takes advantage of the endogenous ato operon.</DIV></DIV></DIV></DIV></DIV></DIV><DIV id="CollapsiblePanel14" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Synthesis of P(3HB) from 3HB: Biobrick design<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H4><SPAN class="mw-headline" id="3HB_Permease:">3HB Permease:</SPAN></H4><P>For 3HB import, we designed a biobrick for the expression of a Putative permease, found in the literature <A href="http://www.ncbi.nlm.nih.gov/nuccore/AB330992(1)" class="external autonumber" rel="nofollow">[1]</A>. We optimised the sequence for <I>E.coli</I> and planned experiments with the construct.</P><H4><SPAN class="mw-headline" id="Construct_for_bdh2_dehydrogenase_expression">Construct for bdh2 dehydrogenase expression</SPAN></H4><P><A href="http://parts.igem.org/Part:BBa_K1149050" class="external text" rel="nofollow">BBa_K1149050</A>: <B>Intracellular expression of bdh2, 3HB dehydrogenase enzyme. </B> We have used a strong arabinose inducible promoter <A href="http://parts.igem.org/Part:BBa_K206000" class="external text" rel="nofollow">BBa_K206000</A> in front of the operon for controlled expression of the enzyme; this enzyme is only required by the cell if 3HB is present.  We have included superfolder GFP <A href="http://parts.igem.org/Part:BBa_K515005" class="external text" rel="nofollow">BBa_K515005</A> in the operon with phaZ1 so that we can monitor gene expression from the promoter via fluorescence measurements. (see our corresponding data under results)</P></DIV></DIV><DIV id="CollapsiblePanel15" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>PLA degradation: pathway and biobrick design<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P>Poly(lactic acid) (PLA) depolymerases are categorized into protease-type and lipase-type (7). We examined the literature and found sequences for one enzyme from each type.
</P><H3><SPAN class="mw-headline" id="Protease-type">Protease-type</SPAN></H3><UL><LI>Proteinase K sequence from Tritirachium album, hydrolyzes poly(L-lactic acid) (PLLA) only.
</LI></UL><H3><SPAN class="mw-headline" id="Lipase-type">Lipase-type</SPAN></H3><UL><LI>Cutinase-like enzyme (CLE) sequence from Cryptococcus sp. strain S-2 preferentially hydrolyzes poly(D-lactic acid) (PDLA), but can also cleave poly(L-lactic acid) (PLLA).
</LI></UL><P>We considered the safety aspects of using these two enzymes, since their sequences were originally from Risk Group 2 (RG2) organisms. We submitted our safety information to iGEM for review. <B>Safety forms were approved on October 2nd, 2013 by the iGEM Safety Committee.</B></P><P>We did not use these organisms only the parts and the final sequences were codon optimised for expression in E. coli.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Proteinase K and CLE, Poly lactic acid (PLA) degradation mechanisms. Adapted from Kawai et. al 2011</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="TabbedPanelsContent"><DIV id="CollapsiblePanelMod1" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>1.0 P(3HB) Degradation Model<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P align="justify">
The P(3HB) degradation model was built and simulated in MATLAB Simbiology in the similar fashion as that seen in the modelling section of Module 1.  This model encompasses gene expression of PhaZ1 (P(3HB) depolymerase) controlled by a xylose inducible promoter<A href="http://parts.igem.org/Part:BBa_I741018" class="external autonumber" rel="nofollow">[2]</A>and the kinetics of PhaZ1 degrading the polymer P(3HB) into shorter oligomers or 3HB monomers.
</P><H4><SPAN class="mw-headline" id="1.1_Ordinary_differential_equations">1.1 Ordinary differential equations</SPAN></H4><H5><SPAN class="mw-headline" id="Genetic_expression_and_regulation_of_PhaZ1">Genetic expression and regulation of PhaZ1</SPAN></H5><UL><LI>Notice that the degradation terms are lacking because the cells are assumed to be in stationary growth.  Therefore, the assumed dominant process by which the mRNA and protein degrade - dilution - is absent.
</LI></UL><H6><SPAN class="mw-headline" id="Values.2C_sources_and_assumptions">Values, sources and assumptions</SPAN></H6><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH><TH>Assumptions/Notes</TH></TR><TR><TH>β</TH><TD>maximum rate of transcription</TD><TD>0.019</TD><TD>mM/min</TD><TD>Please see derivation 1 below.</TD><TD>Please see derivation 1 below.</TD></TR><TR><TH>n</TH><TD>Hill coefficient</TD><TD>2.0</TD><TD>dimensionless</TD><TD>Estimation.  Not on parts registry <A href="http://parts.igem.org/cgi/partsdb/part_info.cgi?part_name=BBa%20I741018" class="external autonumber" rel="nofollow">[3]</A> Didn't manage to find an appropriate source with such value.</TD><TD>Assume cooperativity of degree 2.</TD></TR><TR><TH>K</TH><TD>Activation coefficient</TD><TD>23.5</TD><TD>mM</TD><TD>Personal conversation with Christopher Hirst, Department of Bioengineering, Imperial College London</TD><TD>Taken from data for xylF in MOPS with 0.4% glucose with extra xylR on the plasmid, however the plasmid is low copy and also the presence of the xylR gene on the plasmid definitely alters the expression level.  Data fitted to Hill equation.  This value was the most accurate we managed to obtain, though a rough one as 3% glucose in LB was used in the expreiment.</TD></TR><TR><TH>k<SUB>2</SUB></TH><TD>Protein production rate</TD><TD>2.9</TD><TD>1/min</TD><TD>Please see derivation 2 below.</TD><TD>Please see derivation 2 below.</TD></TR><TR><TH>[Xylose]</TH><TD>Concentration of xylose in cell</TD><TD>constant: 0.01</TD><TD>mM</TD><TD>-</TD><TD>Manual input.</TD></TR><TR><TH>[mRNA]</TH><TD>Concentration of mRNA</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[PhaZ1]</TH><TD>Concentration of PhaZ1</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE><H6><SPAN class="mw-headline" id="Derivations">Derivations</SPAN></H6><P><B>1.Derivation of the maximal expression rate,β</B></P><UL><LI>Average molecular weight (Mw) of a base pair = 660g/mol<A href="http://www.geneinfinity.org/sp/sp_dnaprop.html" class="external autonumber" rel="nofollow">[4]</A><A href="http://www.lifetechnologies.com/uk/en/home/references/ambion-tech-support/rna-tools-and-calculators/dna-and-rna-molecular-weights-and-conversions.html" class="external autonumber" rel="nofollow">[5]</A></LI><LI>Average mass of a base pair = 660g/mol x 1.66x10<SUP>-24</SUP> = 1.1x10<SUP>-21</SUP>g
</LI><LI>Volume of an <I>E.coli</I> cell = 1µm<SUP>3</SUP><A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external autonumber" rel="nofollow">[6]</A> = 1x10<SUP>-15</SUP>L
</LI><LI>∴Mass concentration = </LI><LI>∴Molar concentration of 1 base pair in the volume of <I>E.coli</I> =  = 1.66x10<SUP>-6</SUP> mM
</LI></UL><LI>BioBrick assembly plasmid pSB1C3 is a high copy number plasmid (100-300 copies per cell)<A href="http://parts.igem.org/Part:pSB1C3?title=Part:pSB1C3" class="external autonumber" rel="nofollow">[7]</A></LI><LI>assume 200 copies per cell
</LI><LI>∴ concentration of the gene per cell = N x 200 x 1.66x10<SUP>-6</SUP>mM, where N = number of base pairs
</LI><LI>∴ concentration of the gene phaZ1 (N = 1245)<A href="http://bacteria.ensembl.org/cupriavidus_metallidurans_ch34/Gene/Summary?g=Rmet_1017;r=Chromosome:1106373-1107617;t=ABF07903" class="external autonumber" rel="nofollow">[8]</A> in the volume of an <I>E.coli</I> cell is = 0.41mM
</LI><LI>Transcription rate in <I>E.coli</I>= 80bp/s<A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external autonumber" rel="nofollow">[9]</A> = 80 x 1.66x10<SUP>-6</SUP>mM/s = 80 x 1.66x10<SUP>-6</SUP> x 60mM/min = 7.97x10<SUP>-3</SUP>mM/min
</LI><LI>∴ Rate of mRNA_phaZ1 production under the control of pBAD = 7.97x10<SUP>-3</SUP> ÷ 0.41 = <B>0.019/min</B></LI><P><B>2.Protein production rate of phaZ1, k<SUB>2</SUB></B></P><UL><LI>Average molecular weight(Mw) of an amino acid(aa)= 110g/mol<A href="http://www.genscript.com/conversion.html" class="external autonumber" rel="nofollow">[10]</A><A href="http://www.promega.com/~/media/Files/Resources/Technical%20References/Amino%20Acid%20Abbreviations%20and%20Molecular%20Weights.pdf" class="external autonumber" rel="nofollow">[11]</A></LI><LI>Average mass of an amino acid = 110g/mol x 1.66x10<SUP>-24</SUP>=1.83x10<SUP>-22</SUP>g/L
</LI><LI>∴Mass concentration of one aa in the volume of an <I>E.coli</I> =  = 1.83x10<SUP>-6</SUP>g/L
</LI><LI>∴Molar concentration of one aa =  = 1.66x10<SUP>-5</SUP>mM
</LI></UL><LI>Translation rate = 20aa/s = (20 x 1.66x10<SUP>-5</SUP> x 60)mM/min = 0.020mM/min
</LI><LI>PhaZ1 comprises of 414aa<A href="http://bacteria.ensembl.org/cupriavidus_metallidurans_ch34/Gene/Summary?g=Rmet_1017;r=Chromosome:1106373-1107617;t=ABF07903" class="external autonumber" rel="nofollow">[12]</A></LI><LI>∴concentration of PhaZ1's aa in the volume of an <I>E.coli</I>= 1.66x10<SUP>-5</SUP>mM x 414 = 6.87x10<SUP>-3</SUP>mM
</LI><LI>∴ Rate of protein production = 0.020 ÷ 6.87x10<SUP>-3</SUP> = <B>2.9/min</B></LI><H4><SPAN class="mw-headline" id="1.2_Enzyme_kinetics">1.2 Enzyme kinetics</SPAN></H4><H5><SPAN class="mw-headline" id="PhaZ1_reaction_with_P.283HB.29">PhaZ1 reaction with P(3HB)</SPAN></H5><H6><SPAN class="mw-headline" id="Values.2C_sources_and_assumptions_2">Values, sources and assumptions</SPAN></H6><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Source</TH><TH>Assumptions/Notes</TH></TR><TR><TH>k<SUB>cat,PhaZ1</SUB></TH><TD>Turnover number of PhaZ1 with P(3HB) as substrate</TD><TD>214</TD><TD>1/min</TD><TD>Please see &quot;Testing and Results&quot;</TD><TD>Vmax was obtained from enzymatic assay results, value = 29.59uM/s.  kcat = Vmax/[E], where Vmax is the maximum velocity of enzymatic reaction and [E] is the total enzyme concentration used.  [E] was measured as 360ng/uL.  The molar mass of PhaZ1 (43496 g/mol <A href="http://www.mybiosource.com/datasheet.php?products_id=1012640" class="external autonumber" rel="nofollow">[13]</A>)was used to convert the units of [E] into g/L.  After appropriate conversions and calculations, kcat was calculated to be 214/min</TD></TR><TR><TH>K<SUB>m</SUB></TH><TD>Michaelis constant</TD><TD>1.56</TD><TD>mM</TD><TD>Please see &quot;Testing and Results&quot;</TD><TD>-</TD></TR><TR><TH>[PhaZ1]</TH><TD>Concentration of PhaZ1</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[P(3HB)]</TH><TD>Extracellular concentration of P(3HB)</TD><TD>50</TD><TD>mM</TD><TD>-</TD><TD>user-defined initial value</TD></TR><TR><TH>Secretion</TH><TD>Secretion rate of PhaZ1 from cell to outside</TD><TD>0.5</TD><TD>1/minute</TD><TD>-</TD><TD>Appears as &quot;k&quot; in the Simbiology parameter list for that reaction object.  The resulting extracellular concentration was then scaled up by multiplying the concentration by 1.5x10<SUP>21</SUP> cells.</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="1.3_Simulation_results">1.3 Simulation results</SPAN></H4><P align="justify">
Please note that the result at this point was already scaled up (please refer to &quot;secretion&quot; from the table above.  The value 1.5x10<SUP>21</SUP> was chosen as the number of cells because this was the order of magnitude that would give the reasonable amount of enzyme for degradation.  Other parameters such as those involved in gene expression were the closest we could get them to and the enzyme kinetics data were taken from the our own enzyme assay experiment.  Therefore, we were left with the number of cells and the unknown secretion rate to modify.  After several runs of simulation, we decided to increase the number of cells to 1.5x10<SUP>21</SUP>, instead of the 1.5x10<SUP>12</SUP> assumed in the synthesis model. 
</P><P><B>(i)P(3HB) and 3HB levels over time (units: mM)</B></P><P><B>(ii)P(3HB) level over time (units: g/L)</B></P><H5><SPAN class="mw-headline" id="Graphs_.28i.29_to_.28ii.29_unit_conversion"> Graphs (i) to (ii) unit conversion</SPAN></H5><UL><LI>To convert from mM to g/L:
</LI><LI>Molecular weight(Mw) of P(3HB) = 1.8x10<SUP>12</SUP>g/mol<A href="http://webpages.charter.net/hvalentin/PDF-files/18.pdf" class="external autonumber" rel="nofollow">[14]</A></LI></UL><P align="justify">(Please note that this value was taken from a paper about the &quot;Production of poly(3-hydroxybutyrate-co-4-hydroxybutyrate) in recombinant Escherichia coli grown on glucose&quot; instead of the production of poly(3-hydroxybutyrate).  However, this is the closest we managed to get as far as species, substrate and type of plastic are concerned.</P><P align="justify">Also, it was noticed that the molecular weight would change depending on the cultivation time (as conditions such as growth and pH can also affect it) and it was hard to find a paper that contains the exact same conditions and the range of cultivation times for our simulations.  Furthermore, in reality there wouldn't be a single value of molecular weight obtained, but a range of values (known as polydispersity).<A href="http://aem.asm.org/content/78/9/3177.full" class="external autonumber" rel="nofollow">[15]</A>  Therefore, it was assumed that the molecular weight would remain constant during the time course of the simulation and that the value should be considered as an average.)</P><UL><LI>∴P(3HB)concentration in mM x 10<SUP>-3</SUP> x 1.8x10<SUP>12</SUP>g/mol = concentration in g/L
</LI></UL><H4>Future work</H4><P align="justify">With a more complete characterisation of the promoter under the experimental conditions used for this system, more accurate gene expression parameter values can be plugged into the model.  Also, the secretion rate was hard to determine and would be an important value to obtain experimentally because it is the only rate that controls how much of the enzyme is released to the extracellular space.</P><H4>P(3HB) degradation model:Download</H4><H4>Additional script:Download</H4></DIV></DIV><DIV id="CollapsiblePanelMod2" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>2.0 P(3HB) production from 3HB monomers<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P align="justify"> In module 2, the main objective is the bioplastic recycling. We have a detailed explanation of the P(3HB) synthesis model and the optimization in P(3HB) production yield. Our synthetic pathway of converting monomeric 3HB to polymeric P(3HB) is described in <A href="https://2013.igem.org/Team:Imperial_College/Waste_Degradation:_SRF" class="external text" rel="nofollow">module one</A>. In MAPLE system, both glucose from mixed waste and 3HB monomers are the inputs to produce P(3HB). In this section, we are going to introduce a customized metabolic model that consisted our synthetic pathway with a metabolic model built by Angela Dixon. <A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[16]</A> The combined metabolic model can predict the P(3HB) production with both 3HB and glucose as sources. </P><H4><SPAN class="mw-headline" id="Introduction_of_the_metabolic_model">Introduction of the metabolic model</SPAN></H4><P align="justify">
The metabolic model predicts the interaction of our pathway with endogenous pathways. In synthetic biology, an ideal system is fully orthogonal. However, if the synthetic pathway uses some metabolites from the metabolism pathways, the metabolic fluxes (the rate of conversion between metabolites) will be affected. If the cell is overloaded by losing too much metabolites, it would either reject the synthetic pathway or burn out. As for the optimization of PHB production in our MAPLE system, metabolic analysis need to be carried out to estimate the effects on the cell metabolism from increases in the PHB production. </P><H4><SPAN class="mw-headline" id="Methods">Methods</SPAN></H4><P align="justify">We altered the metabolic model by Angela Dixon in her paper “Predictive Mathematical Model for Polyhydroxybutyrate Synthesis in Escherichia coli”. <A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[17]</A> The metabolic model is built in Simbiology, which is an extension in MATLAB especially for biological system. The model consists two metabolic pathways: Glycolysis and TCA cycle. The reactions in the metabolic pathways are defined by ordinary differential equations and kinetic parameters. The model also contains a synthetic pathway that produce P(3HB) from acetyl-coA. The pathway consists three enzymes which are PhaA, PhaB and PhaC. 
The reasons we chose this model:</P><P>1. The model is based on ODEs instead of FBA (flux balance analysis), which is a common method of metabolic analysis. However, FBA cannot determine the real time concentrations of the metabolites whereas it can be done by models with ODEs.</P><P>2. Only single gene deletion or addition can be performed in FBA. More than that, the metabolic model here can perform modification on any single or multiple genes and reactions through Simbiology. </P><P>3. The FBA model is static, it cannot analyze the dynamic interactions between our synthetic pathways with the metabolic pathways. As both our P(3HB) synthesis model and the metabolic model are in the same platform. It was really easy to integrate our pathway into the metabolic model.</P><P>4. The strain in the metabolic model is E.coli K12, the strain MG1655 we are using is also included in K12 group. The models can therefore be more easily combined. </P><P><B>In our project, we have a different pathway:</B></P><P>1. We use 3HB as one of the input.</P><P>2. ato system is present in the E.coli MG1655 strain. </P><P>3. We removed phaA in our case because we use 3HB as the major source to produce P(3HB), it will avoid too much acetyl-coA be taken from the metabolic pathways.</P><P align="justify">Therefore, we substituted the original pathway with our synthetic pathway. Moreover, we added our gene expression models for the enzymes into the metabolic model as well. In terms of the consistency of the model, we changed the units of our synthetic pathway into micro molar and seconds. </P><P>Here is the combination of our model with the metabolic model</P><P><B>The pathway with the black border is our synthetic pathway. </B></P><P align="justify">We also carried out a parameter estimations of the initial concentrations of several substrates in our synthetic pathway. The reason is we have single-cell scaled magnitude for them originally (fM level), but there are much higher magnitudes in any of the species in the metabolic pathways (mM level). Therefore, we scaled up the pathway in order to maintain the consistency of the model. As for the metabolites which are involved in our pathway, we cloned the blocks of them from the metabolic pathway in Simbiology and deliberately put them into our pathway. That means our pathway dynamically interact with the metabolic pathways. </P><P>Here is the table for the updated initial concentrations for the species in our synthetic pathway:
All the kinetic data and initial concentrations of the metabolites in the metabolic pathways are referenced in the paper<A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[18]</A>: </P><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Substrates</TH><TH>Concentration</TH><TH>Units</TH><TH>Sources</TH><TH>Assumptions</TH></TR><TR><TH>3HB</TH><TD>5000</TD><TD>uM</TD><TD>-</TD><TD>The initial input to the system</TD></TR><TR><TH>Acetoacetate</TH><TD>1000</TD><TD>uM</TD><TD>-</TD><TD>The ato system in E.coli suggest a favourable conversion from acetoacetyl-coA to acetoacetate. We assumed the concentration of acetoacetate is 10 times larger than acetoacetyl-coA<A href="http://www.sciencedirect.com/science/article/pii/0003986177904969" class="external autonumber" rel="nofollow">[19]</A></TD></TR><TR><TH>Acetoacetyl-coA</TH><TD>100</TD><TD>uM</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/1103741" class="external autonumber" rel="nofollow">[20]</A></TD><TD>The ato system in E.coli suggest a favourable conversion from acetoacetyl-coA to acetoacetate. <A href="http://www.sciencedirect.com/science/article/pii/0003986177904969" class="external autonumber" rel="nofollow">[21]</A></TD></TR><TR><TH>NADH</TH><TD>200</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[22]</A></TD><TD>-</TD></TR><TR><TH>NAD+</TH><TD>1200</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[23]</A></TD><TD>-</TD></TR><TR><TH>Acetyl-coA</TH><TD>5000</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[24]</A></TD><TD>-</TD></TR><TR><TH>Coenzyme A</TH><TD>250</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[25]</A></TD><TD>-</TD></TR><TR><TH>Acetate</TH><TD>40000</TD><TD>uM</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/1103741" class="external autonumber" rel="nofollow">[26]</A></TD><TD>Assay data from the literature which the ato system efficiency was measured</TD></TR><TR><TH>3HB-coA</TH><TD>0</TD><TD>uM</TD><TD>-</TD><TD>One of the intermediate product of the pathway</TD></TR><TR><TH>NADPH</TH><TD>200</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[27]</A></TD><TD>-</TD></TR><TR><TH>NADP+</TH><TD>1200</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[28]</A></TD><TD>-</TD></TR><TR><TH>PHB</TH><TD>0</TD><TD>uM</TD><TD>-</TD><TD>The final product of the pathway</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="ATP_analysis:">ATP analysis:</SPAN></H4><P>In our optimization of P(3HB) production module, we discovered that increasing in the PhaB expression can efficiently increase the yield of P(3HB). It’s important to see whether increase in the PhaB expression would dramatically affects the metabolic pathways or not. As ATP is the essential energy carriers in the cell, we carried a parameter scan for the variation in ATP concentration under different concentrations of PhaB.</P><P>There is an approximately 5.3% drop in the concentration of ATP when increasing the phaB concentration from 50uM to 200uM. It suggests that increasing the flux in our pathway may reduce the energy transportation in the cell. However, it’s still unsure about the effect of 5.3% drop in ATP concentration on the cell growth.</P><H4><SPAN class="mw-headline" id="Large_scale_PHB_production_and_implementation_in_MAPLE">Large scale PHB production and implementation in MAPLE</SPAN></H4><P>In our MAPLE system, the bioplastic  P(3HB) is produced from both 3HB monomers and glucose. 3HB is the degradation product from the wasted P(3HB）plastic and glucose is the product from the degradation of mixed waste. Therefore, the overall PHB production rates need to be determined by the model.
</P><P>The result shows that the PHB yield can reach 70g/L after one day with both inputs whereas about 30g/L with glucose. It proves that using both glucose and 3HB can double the yield of P(3HB). The production rate 70g/L/Day is also relatively high for industrial scale of bioplastic production.
</P><P><B>Assumption in the production</B></P><UL><LI>Conditions and considerations accompanying this value are: 
</LI><LI>volume of the system is 1 litre
</LI><LI>growth medium: LB broth
</LI><LI>Cultivation temperature: 37°C
</LI><LI>wild-type E. coli K-12 strain MG1655 (NB: though not exactly our engineered one, it is the correct strain)
</LI></UL><LI>Assume stationary growth phase.
</LI><LI>Assume that the amount of P(3HB) accumulated during the time course of simulation doesn't exceed the cell's capacity or ability to contain it.
</LI><LI>Assume that all initial concentrations of the species in metabolic pathways and the synthetic pathways are in a large population level. The concentrations are maintained by approximately 1.5x10<SUP>12</SUP>cells in a 1L broth, calculated from value obtained from BioNumbers.  A range was given: 1-2x10<SUP>9</SUP> cells/ml, so 1.5x10<SUP>9</SUP> was taken. <A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?s=n&amp;id=104831&amp;ver=5" class="external autonumber" rel="nofollow">[29]</A></LI><H4><SPAN class="mw-headline" id="Video:_How_to_run_a_simulation_in_the_metabolic_model_.28please_refresh_if_it.27s_not_here.29">Video: How to run a simulation in the metabolic model (please refresh if it's not here)</SPAN></H4><H4>Metabolic model:Download</H4></DIV></DIV><DIV id="CollapsiblePanelMod3" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>3.0 PLA Degradation Model<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H4><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H4> 
The efficiency of degrading Polylactic acid (PLA) is important for the performance of MAPLE. Therefore, we chose a strong enzyme Proteinase K which is suggested by many literature as an efficient PLA degrading enzyme. We performed several assays in order to determine the kinetic properties of Proteinase K. The PLA degradation model was then built based on the experimental results. <H4><SPAN class="mw-headline" id="Objective_and_Design">Objective and Design</SPAN></H4><P>1. With the defined kinetic properties of Proteinase K. The model can predict the time needed to degrade a certain amount of PLA. It will produce useful information when degrading large amount of plastic. </P><P>2. The model also involves the gene expression model of the Proteinase K with an inducible promoter. Gene expression can be regulated by adjusting the inducer concentration. </P><P>3. A secretion module is contained in the model which assumes the efficiency of enzyme secretion to the culture. </P><H3><SPAN class="mw-headline" id="Specifications_of_the_model">Specifications of the model</SPAN></H3><P align="justify">Polylactic acid (PLA) degradation involves proteinase K:</P><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH width="150">enzyme</TH><TH>source organism</TH><TH>biobrick</TH><TH>reference</TH></TR><TR><TH>Proteinase K</TH><TD>Engyodontium album</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149002" class="external text" rel="nofollow">BBa_K1149002</A></TD><TD><A href="http://www.ncbi.nlm.nih.gov/nuccore/X14689.1" class="external autonumber" rel="nofollow">[30]</A></TD></TR></TBODY></TABLE><P>The overall PLA degradation model is shown as below:</P><P>There are two compartments which represents cells and the culture from left to right. The &quot;cell&quot; compartment contains the gene expression module whereas the &quot;culture&quot; compartment contains the degradation module. The &quot;secretion&quot; block that connects two compartments is the secretion module.</P><H4><SPAN class="mw-headline" id="Parameters_and_assumptions">Parameters and assumptions </SPAN></H4><H5><SPAN class="mw-headline" id="Gene_expression_module_of_Proteinase_K">Gene expression module of Proteinase K</SPAN></H5><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH><TH>Assumptions</TH></TR><TR><TH>β</TH><TD>maximum rate of transcription</TD><TD>0.020</TD><TD>mM/min</TD><TD>Please see derivation 1 below.</TD><TD>Please see derivation 1 below.</TD></TR><TR><TH>K</TH><TD>Activation coefficient</TD><TD>0.0031</TD><TD>mM</TD><TD><A href="http://parts.igem.org/Part:BBa_K206000:Characterization" class="external autonumber" rel="nofollow">[31]</A></TD><TD>Taking the &quot;switch point&quot; as the activation coefficient</TD></TR><TR><TH>d<SUB>mRNA</SUB></TH><TD>mRNA degradation rate</TD><TD>0.10</TD><TD>1/min</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC420366/pdf/07X.pdf" class="external autonumber" rel="nofollow">[32]</A></TD><TD>Taking the value of mRNA half-life in E.coli strain MG1655 as 6.8min.  rate = ln2/half-life = ln2/6.8 = 0.10</TD></TR><TR><TH>d<SUB>protein</SUB></TH><TD>Protein degradation rate</TD><TD>0.050</TD><TD>1/min</TD><TD><A href="http://jb.asm.org/content/189/23/8746.full" class="external autonumber" rel="nofollow">[33]</A></TD><TD>There is no active degradation pathway and that dilution is the dominant way by which the protein level decreases in a cell.  Rate = 1/doubling time, where doubling time = 20min.  Assuming steady-state growth in LB broth as presented in paper.</TD></TR><TR><TH>k<SUB>2</SUB></TH><TD>Protein production rate (Proteinase K)</TD><TD>3.0</TD><TD>1/min</TD><TD>Please see derivation 2 below.</TD><TD>Please see derivation 2 below.</TD></TR><TR><TH>[Arabinose]</TH><TD>Concentration of arabinose</TD><TD>Initial: 0.008</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[mRNA]</TH><TD>Concentration of mRNA</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[Proteinase K]</TH><TD>Concentration of Proteinase K</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE><P><B>1.Derivation of the maximal expression rate,β</B><UL><LI>Average molecular weight (Mw) of a base pair = 660g/mol<A href="http://www.geneinfinity.org/sp/sp_dnaprop.html" class="external autonumber" rel="nofollow">[34]</A><A href="http://www.lifetechnologies.com/uk/en/home/references/ambion-tech-support/rna-tools-and-calculators/dna-and-rna-molecular-weights-and-conversions.html" class="external autonumber" rel="nofollow">[35]</A></LI><LI>Average mass of a base pair = 660g/mol x 1.66x10<SUP>-24</SUP> = 1.1x10<SUP>-21</SUP>g
</LI><LI>Volume of an E.coli cell = 1µm<SUP>3</SUP><A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external autonumber" rel="nofollow">[36]</A> = 1x10<SUP>-15</SUP>L
</LI><LI>∴Mass concentration = </LI><LI>∴Molar concentration of 1 base pair in the volume of E.coli =  = 1.66x10<SUP>-6</SUP> mM
</LI></UL><LI>BioBrick assembly plasmid pSB1C3 is a high copy number plasmid (100-300 copies per cell)<A href="http://parts.igem.org/Part:pSB1C3?title=Part:pSB1C3" class="external autonumber" rel="nofollow">[37]</A></LI><LI>assume 200 copies per cell
</LI><LI>∴ concentration of the gene per cell = N x 200 x 1.66x10<SUP>-6</SUP>mM, where N = number of base pairs
</LI><LI>∴ concentration of the gene proteinase K (N = 1194) in the volume of an E.coli cell is = 0.40mM
</LI><LI>Transcription rate in E.coli = 80bp/s<A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external autonumber" rel="nofollow">[38]</A> = 80 x 1.66x10<SUP>-6</SUP>mM/s = 80 x 1.66x10<SUP>-6</SUP> x 60mM/min = 7.97x10<SUP>-3</SUP>mM/min
</LI><LI>∴ Rate of mRNA_Proteinase K production under the control of pBAD = 7.97x10<SUP>-3</SUP> ÷ 0.40 = <B>0.020/min</B></LI><B>2.Protein production rate of Proteinase K, k<SUB>2</SUB></B><UL><LI>Average molecular weight(Mw) of an amino acid(aa)= 110g/mol<A href="http://www.genscript.com/conversion.html" class="external autonumber" rel="nofollow">[39]</A><A href="http://www.promega.com/~/media/Files/Resources/Technical%20References/Amino%20Acid%20Abbreviations%20and%20Molecular%20Weights.pdf" class="external autonumber" rel="nofollow">[40]</A></LI><LI>Average mass of an amino acid = 110g/mol x 1.66x10<SUP>-24</SUP>=1.83x10<SUP>-22</SUP>g/L
</LI><LI>∴Mass concentration of one aa in the volume of an E.coli =  = 1.83x10<SUP>-6</SUP>g/L
</LI><LI>∴Molar concentration of one aa =  = 1.66x10<SUP>-5</SUP>mM
</LI></UL><LI>Translation rate = 20aa/s = (20 x 1.66x10<SUP>-5</SUP> x 60)mM/min = 0.020mM/min
</LI><LI>Proteinase K comprises of 398aa<A href="http://www.ncbi.nlm.nih.gov/nuccore/X14689.1" class="external autonumber" rel="nofollow">[41]</A></LI><LI>∴concentration of Proteinase K's aa in the volume of an E.coli = 1.66x10<SUP>-5</SUP>mM x 398 = 6.61x10<SUP>-3</SUP>mM
</LI><LI>∴ Rate of protein production = 0.020 ÷ 4.25x10<SUP>-3</SUP> = <B>3.0/min</B></LI></P><H4><SPAN class="mw-headline" id="Degradation">Degradation</SPAN></H4><P>The reaction equation of the PLA degradation is:</P><P>[Polylactic acid]+[Proteinase K]= 660 [lactic acid] + [Proteinase K]
</P><P><B>Assumptions:</B></P><P>1. We assumed 1 mole of polylactic acid  can produce 660 moles of lactic acid. </P><P>2. The molecular weight of a single polylactic acid monomer is 90 g/mol <A href="http://www.chemspider.com/Chemical-Structure.592.html" class="external autonumber" rel="nofollow">[42]</A>whereas the molecular weight of the solid polylactic acid is around 59500 g/mol.<A href="http://www.sciencedirect.com/science/article/pii/S0014305711003582" class="external autonumber" rel="nofollow">[43]</A>Therefore, the short-chain polylactic acid consists approximately 660 monomers. 660 molecules of lactic acid will be produced by degrading one chain of polymer.</P><P>3. Assumed the polylactic acid only contains one length of chain, however in reality the plastic chain has a range of lengths.</P><P>4. Considerations of temperature change and PH change are not included in the model. Therefore, the assumption is that they remain constant during the degradation </P><P>5. We also assumed a simple Michaelis-Menten mechanism for proteinase K. </P><P>&quot;[Proteinase K]&quot; here is the concentration of the enzyme Proteinase K in mM, whereas &quot;[Polylactic acid]&quot; represents the concentration of polylactic acid in mM. </P><P>The parameters for the kinetic equations are:</P><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH></TR><TR><TH>Km</TH><TD>Michaelis constant</TD><TD>0.032</TD><TD>mM</TD><TD>-</TD></TR><TR><TH>Vmax</TH><TD>Maximum velocity</TD><TD>2.472</TD><TD>mM/min</TD><TD>-</TD></TR><TR><TH>Proteinase K</TH><TD>Concentration of the enzyme in assays</TD><TD>30</TD><TD>mg/L</TD><TD>-</TD></TR><TR><TH>Kcat</TH><TD>turnover number</TD><TD>2348.4</TD><TD>1/min</TD><TD>See derivation below</TD></TR><TR><TH>Mw</TH><TD>Molecular weight of proteinase K</TD><TD>28.5</TD><TD>KDa</TD><TD><A href="http://www.sigmaaldrich.com/etc/medialib/docs/Sigma/Datasheet/2/p4850dat.Par.0001.File.tmp/p4850dat.pdf" class="external autonumber" rel="nofollow">[44]</A></TD></TR></TBODY></TABLE><P><B>Derivation:</B>Turnover number (Kcat) = Vmax*Mw/[proteinase K] = 2348.4</P><P>The efficiency of <B>Secretion</B> is assumed to be 90% secretion over 2 hours.<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1251600/" class="external autonumber" rel="nofollow">[45]</A> The rate of secretion in the model is therefore:</P><P>rate of secretion = 0.9[concentration of Proteinase K]/120 (mM/mins) </P><H2><SPAN class="mw-headline" id="Simulations_and_Results">Simulations and Results</SPAN></H2><P>The initial concentration of Polylactic acid is 40mM, which can be converted to 2380g/L. The derivation is: 40*59500/1000 = 2380g/L. There is approximately 2kg of polylactic acid in the 1 litre bioreactor,  which is a really large amount. <B>The simulation result is:</B></P><P><B>Results and explanations:</B>The simulation shows a very fast degradation of large amount of polylactic acid by proteinase K. 2kg/L of the plastic can be completely degraded in 6 minutes. The reason for fast degradation is that we have relative large kinetic parameters for the enzyme. Polylactic acid was not used as the substrate in our kinetic assays. Therefore, we made an assumption that polylactic acid in the model has similar chain length as that of the substrate in the assays. In reality, the PLA plastic is in solid form which enzymes can only degrade the plastic from the surface. The efficiency of degradation will be massively reduced due the 3D geometry. A 3D model is needed for future improvements of plastic degradation. The kinetic data of degrading solid form of PLA also need to be defined. </P><H4>PLA degradation model:Download</H4></DIV></DIV></DIV><DIV class="TabbedPanelsContent"><DIV id="CollapsiblePanel1" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Our chassis can grow with P(3HB) and 3HB<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H3><SPAN class="mw-headline" id="Summary">Summary</SPAN></H3><P>Our chassis can grow and survive in reasonable concentrations of P(3HB) and 3HB. To increase tolerance to 3HB we have re-designed our system to include expression of Bdh2 (BBa_K1149050, 3HB dehydrogenase), which breaks down 3HB to acetoacetate.  Acetoacetate toxicity was examined and these data are being utilised along with our metabolic models to determine the effects of differing levels of 3HB and acetoacetate on bioplastic (P3HB) production.
</P><P>MG1655 transformed with either native phaCAB or empty vector (control) plasmid were grown for 6h in LB containing 3-HB or P-3HB to assess any potential toxicities associated with these bioplasrics.
</P><H3><SPAN class="mw-headline" id="3-hydroxybutyrate_.283HB.29">3-hydroxybutyrate (3HB) </SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B> Toxicity of 3-hydroxybutyrate in LB with phaCAB and EV expression.</B> 3HB was tested at 5 concentrations for toxicity - 1 μM, 100 μM, 1 mM, 10 mM and 237 mM. Between 10 mM and 237 mM, there is a clear toxicity effect observed. Data points show final time point after 6h growth for each concentration. Growth was at 37°C with shaking over 6h. Error bars are SEM, n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Poly.283-hydroxybutyrate.29_P.283HB.29">Poly(3-hydroxybutyrate) P(3HB) </SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:565px;"><DIV class="thumbcaption"><B> Toxicity of Poly(3-hydroxybutyrate) in LB with phaCAB and EV expression.</B> P3HB toxicity was only tested at one concentration in LB - 31 μg/L, it was also tested at 31 mg/L however the resulting OD was over 1, so this data was not considered accurate. Growth was at 37°C with shaking over 6h. Error bars are SEM, n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV><P><B>Conclusion:</B>3HB is toxic above 10mM and P(3HB) had no effect on growth at 31 μg/L. These data can be utilised to determine the maximum level of 3HB which should be present in our final bioreactor design. We have shared these data with our modelling team in order to determine maximum P(3HB) production at or below 10mM (3HB). In light of these data we have modified our design to include the expression of bdh2 (BBa_K1149050)&lt;/b&gt;, which is a 3HB dehydrogenase that can convert 3HB into acetoacetate, a common metabolite in E.coli. Expression of bdh2 should overcome the toxicity limit of 3HB, since E. coli can metabolise acetoacetate&lt;/b&gt;To this end we tested whether an increase in acetoacetate would present any toxicities.
</P><H3><SPAN class="mw-headline" id="Acetoacetate">Acetoacetate </SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B> Toxicity of acetoacetate in LB with phaCAB and EV expression.</B> Acetoacetate was also tested for toxicity at 4 concentrations - 1 μM, 100 μM, 1mM and 287 mM. In phaCAB and EV this manifested itself between 1 mM and 287 mM. Data points show final time point after 6h growth for each concentration. Growth was at 37°C with shaking over 6h. Error bars are SEM, n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV><P><B>Conclusion:</B>Acetoacetate toxicity lies somewhere between 1mM and 287 mM.
</P></DIV></DIV><DIV id="CollapsiblePanel2" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Bioplastic degradation: Phaz1 enzymatic assays<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H2><SPAN class="mw-headline" id="Enzyme_activity_of_PHB_depolymerase_.28phaz1.29">Enzyme activity of PHB depolymerase (phaz1)</SPAN></H2><P align="justify">It can be seen from the <A href="https://2013.igem.org/Team:Imperial_College/Western_Blots" class="external text" rel="nofollow">Western Blot results</A> that phaz1 <A href="http://parts.igem.org/Part:BBa_K1149010" class="external text" rel="nofollow">BBa_K1149010</A> was being expressed. To show that this enzyme has esterase activity, colourimetric assays were performed using the substrate analog para-Nitrophenyl butyrate. When the ester bond in this substrate is cleaved, 4-Nitrophenol is released. This is accompanied by an increase in absorbance at the wavelength 405 nm and a colour change from colourless to yellow. The concentration of 4-Nitrophenol produced could then be calculated with the Beer-Lambert Law, as the extinction coefficient of 4-NP at 405 nm is 18,000 M-1 cm-1. This experiment was performed with both the crude cell lysate and purified PHB depolymerase. Our data shows that this enzyme is definitely active.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>para-Nitrophenyl butyrate is cleaved by PHB depolymerase to release 4-Nitrophenol. This is accompanied by an increase in absorbance at 405 nm. Figure adapted from <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/pNP_Assay#Enzyme" class="external text" rel="nofollow">TU Darmstadt 2012 iGEM</A></B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>When the substrate para-Nitrophenyl butyrate is added to the reaction mixture of purified PHB depolymerase (phaz1) in phosphate buffer, there is a colour change from colourless to yellow. Figure by Imperial College London iGEM 2013.</B></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Cell_lysate_assay">Cell lysate assay</SPAN></H2><P align="justify">After 48 hours of growing and inducing the phaz1 culture, the cells were lysed by probe-sonication, spun down and resuspended in 50mM Tris-HCl buffer. A reaction mixture containing 5 µL of crude phaz1 lysate, 4 µL of para-Nitrophenyl butyrate solution and 1 mL of 50 mM Tris-HCl pH 7.4 buffer was incubated in the <A href="http://www.eppendorf.com/int/index.php?sitemap=2.1&amp;action=products&amp;contentid=1&amp;catalognode=87236" class="external text" rel="nofollow">Eppendorf BioSpectrometer</A> for 570 seconds, whilst the absorbance at 405 nm was automatically recorded every 30 seconds.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:616px;"><DIV class="thumbcaption"><B>The increase in absorbance that accompanies the cleavage of para-Nitrophenyl butyrate by PHB depolymerase (phaz1). Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:856px;"><DIV class="thumbcaption"><B>The concentration of 4-Nitrophenol released by PHB depolymerase (phaz1) activity. Empty Vector and Substrate alone were used as negative controls. Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV><P align="justify">The above graph shows that greater esterase activity occurs when the phaz1 cell lysate is in the reaction mixture. The graph shows that there is also esterase activity occuring in the Empty Vector and Substrate alone reaction mixtures, but this is due to the imidazole present in the Tris-HCl buffer, which acts as a general base catalysis. <A href="http://pubs.acs.org/doi/pdf/10.1021/ja00874a035" class="external text" rel="nofollow">(6)</A></P><H2><SPAN class="mw-headline" id="Purified_enzyme_assay">Purified enzyme assay</SPAN></H2><P align="justify">Since the phaz1 <A href="http://parts.igem.org/Part:BBa_K1149010" class="external text" rel="nofollow">BBa_K1149010</A> construct contains a His tag, the protein could be purified by <A href="https://2013.igem.org/Team:Imperial_College/Protocols#Protein_purification_by_metal_affinity_chromatography" class="external text" rel="nofollow">metal affinity chromatography</A>. The colourimetric assay was carried out, as with the cell lysate. 100 mM potassium phosphate (pH 7.4) buffer was used instead of the Tris-HCl buffer, to eliminate the esterase activity caused by imidazole in the buffer used previously. Every 30 seconds for 30 minutes the absorbance at 405 nm of the reaction mixture was recorded. The reaction mixture contained 1 ml of phosphate buffer, 2.5 µL of purified PHB depolyermase (stock concentration 0.230 mg/ml) and para-Nitrophenyl butyrate to make final concentrations of 50 µM, 100 µM, 150 µM, 200 µM and 250 µM of substrate.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:834px;"><DIV class="thumbcaption"><B>The purified PHB depolyermase (phaz1) is definitely active. The graph shows that more 4-Nitrophenol is produced with greater concentrations of para-Nitrophenyl butyrate. Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV></DIV><P>The above graph shows how the concentration of 4-Nitrophenol produced is greater when the para-Nitrophenyl butyrate substrate concentration is greater. This data shows that we have <B>successfully purified active PHB depolyermase</B>.

</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>SDS PAGE gel shows that PHB depolymerase was purified from the crude cell extract. Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Michaelis-Menten_kinetics_of_phaZ1_PHB_depolymerase">Michaelis-Menten kinetics of phaZ1 PHB depolymerase</SPAN></H2><P>To gain the kinetic parameters for our P(3HB) degradation model, purified phaZ1 PHB depolymerase was used in the same activity assay as described above. The para-Nitrophenyl butyrate substrate concentrations used were 0.5 mM,1.0 mM, 2.5 mM and 5.0 mM and 5 µL of purified PHB depolymerase (stock concentration 360 ng/µL). From these preliminary results and using graphpad, we have predicted the kinetic parameters to be a Vmax of 29.59 µM/s and a Km 1.56 mM. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>The increase in absorbance that accompanies the cleavage of para-Nitrophenyl butyrate by PHB depolymerase (phaZ1). Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>The concentrations of 4-Nitrophenol produced from the breakdown of para-Nitrophenyl butyrate by phaZ1 PHB depolyermase. Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:588px;"><DIV class="thumbcaption"><B>Michaelis-Menten kinetic graph for phaZ1 PHB depolymerase. Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="CollapsiblePanel3" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Bioplastic degradation: Phaz1 degrades P(3HB)<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H2><SPAN class="mw-headline" id="Clearing_Zone_Assay">Clearing Zone Assay</SPAN></H2><P>Clearing zone experiment was conducted to semi-quantitatively test the ability of PhaZ1 to break down P3HB. PhaZ1 cell lysate was tested on a P3HB LB agar plate. The protocol can be found <A href="https://2013.igem.org/Team:Imperial_College/Protocols#Clear_zone_assay" class="external text" rel="nofollow">here</A>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:952px;"><DIV class="thumbcaption"><FONT size="2.7"><B>PhaZ1 showing P3HB degradation ability by generating a clear zone on a P3HB LB agar plate. A. No difference between empty vector cell lysate and PhaZ1 cell lysate right after they were pipetted into the wells. B. PhaZ1 in the cell lysate started to clear P3HB around the well after 1 day. C. PhaZ1 created a clear zone around the well after 3 days, whereas it was still cloudy around the empty vector cell lysate well. Figure by Imperial College London 2013 iGEM</B></FONT></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="3HB_assay">3HB assay</SPAN></H2><P>In order to show that phaZ1 is functional, we have measured the amount of 3HB monomers produced due to the degradation of the polymer, when incubated with the enzyme. We have repurposed a 3HB colorimetric assay kit that is used for 3HB detection from animal tissue and bodily fluid samples. Please see the [<A href="https://2013.igem.org/Team:Imperial_College/" class="external free" rel="nofollow">https://2013.igem.org/Team:Imperial_College/</A>
Protocols#3HB_assay protocol] and the <A href="https://2013.igem.org/Team:Imperial_College/3HB_assay" class="external text" rel="nofollow">data</A> sections for details.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><FONT size="2.7"><B>PhaZ1 treatment of P3HB sample increases the amount of 3HB monomer going into sulution. Both of the sample were incubated at 37°C  sahking for 15 hours. </B></FONT></DIV></DIV></DIV></DIV><P>It is clear from our results that enzyme treatment produces a lot more 3HB than the control. We do expect to see some background 3HB from the untreated sample since it was in the shaking incubator at 37°C in the Buffer solution. PHB is a bioplastic and degrades spontaneously as well as enzymatically. However, the spontaneous degradation occurs slowly. </P></DIV></DIV><DIV id="CollapsiblePanel4" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Our chassis can utilise 3HB<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H3><SPAN class="mw-headline" id="bdh2_improves_growth_on_3HB">bdh2 improves growth on 3HB</SPAN></H3><P>We have previously observed that MG1655 cells can survive in minimal media that contains 3HB as sole carbon source. This is evidence for that some uptake mechanism and a metabolic pathway is active at a low level in the cells. We hypothesised that the 3HB dehydrogenase could improve growth since it can convert 3HB into acetoacetate, a common metabolite in E.coli.
</P><P>In order to test this, we run a growth experiment with various 3HB concentrations where the growth of cells containing bdh2 (BBa_K1149050) and Empty vector was recorded on a 96 well plate. We have calculated the growth rates from this data and plot it on the graph below.
</P><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B> 3HB sole carbon source growth in minimal media with phaCAB and EV.</B> MG1655 phaCAB were grown in supplemented and minimal M9 media. Relative to the empty vector control, there was no significant difference in growth as t-test gave p = 0.8072 &gt; 0.05. However, an ANOVA of the data gave (<I>F</I><SUB>3,8</SUB> = 6.589, p &lt; 0.0149), thus the null hypothesis, there is no difference between M9M and M9S must be rejected. Indeed if we look closer, we see that both M9M EV and M9S EV are significantly different from another (p = 0.045), as are M9M phaCAB and M9S phaCAB (p = 0.0284). Data points show final time point after 6h growth for each concentration. Growth was at 37°C with shaking over 6h. Error bars are SEM, n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B> 3HB sole carbon source growth in M9S with P3HB synthesis pathway genes.</B> The 3 phaCAB promoter constructs are represented with native phaCAB, constitutive phaCAB and hybrid phaCAB. In addition bdh2 was tested to see whether its presence increased growth for the cells. This was completed with 3 3HB concentrations - 100 μM, 10 mM and 100 mM. ANOVA analysis shows that there is no significant difference between any of the constructs (<I>F</I><SUB>4,10</SUB> = 0.1875, p &lt; 0.9396). Data points show final time point after 6h growth for each concentration. Growth was at 37°C with shaking over 6h. Error bars are SEM, n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Bdh2 with no pelB secretion tag growth.</B> Bdh2 MG1655 cells were grown in M9S media to gauge growth, the growth does not differ from the control as p = 0.5543. Data points show final time point after 6h growth for each concentration. Growth was at 37°C with shaking over 6h. Error bars are SEM, n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Bdh2 with no pelB secretion tag fluorescence.</B> MG1655 cells with bdh2 were grown in <A href="https://2013.igem.org/Team:Imperial_College/Protocols#M9_minimal_and_supplemented_media" class="external text" rel="nofollow">M9S</A> and then tested for fluorescence by measuring their GFP output. There is significant fluorescence as p &lt; 0.0001. Data points show final time point after 6h growth for each concentration. Growth was at 37°C with shaking over 6h. Error bars are SEM, n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><SPAN style="color:#f00"><B>Conclusion:</B></SPAN>We can conclude that there is a slight increase in the growth rate of bdh2 containing cells at 10 000 uM 3HB. This suggests that bdh2 is able to function as expected and produces acetoacetate which is used by the cell`s central metabolic pathways for growth. At 100uM or 100 000 uM, we did not observe any differences in growth. This could be be because the rate limiting step at low concentrations could be the uptake of 3HB rather than it`s conversion to acetoacetate. We hope that we could observe an increase in growth if we added the putative permease we designed to the system. At higher level, there is a drop in growth rates in both bdh2 and control cells, probably because of toxicity issues. In addition, fluorescence is highly elevated in bdh2 compared to empty vector control. 
</P></DIV></DIV><DIV id="CollapsiblePanel5" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Completing the cycle<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H4><SPAN class="mw-headline" id="We_can_degrade_the_PHB_made_by_our_own_bacteria">We can degrade the PHB made by our own bacteria</SPAN></H4><P>With this experiment, we demonstrate that the <A href="https://2013.igem.org/Team:Imperial_College/PHB_production" class="external text" rel="nofollow">material that we purified in various experiments</A> is Poly-Hydroxy-Butyrate bioplastc. One of the samples analysed was extracted from bacteria growing on waste, therefore <B>we made PHB bioplastic directly from waste.</B></P><P><B>We can close the loop and degrade our own bioplastic, with our own enzyme, <A href="http://parts.igem.org/Part:BBa_K1149010" class="external text" rel="nofollow">PhaZ1</A>.</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><FONT size="2.7"><B>The bioplastic produced by our bacteria contains PHB and can be broken down into monomers by PhaZ1 enzyme (expressed by our bacteria)</B></FONT></DIV></DIV></DIV></DIV><P> The amount of 3HB monomers freed from the polymer is more in case of all PhZ1 treated samples, than the untreated sample. There is less 3HB produced from the samples that were purified from bacteria than from the pure polymer, purchased from Sigma. This could be because I put less amount in the assay to start with, because there was less available. I put about a 0.01-0.03 g of our own and 0.5 grams of Sigma-PHB.</P><P> Please see our <A href="https://2013.igem.org/Team:Imperial_College/Protocols#3HB_assay" class="external text" rel="nofollow">protocols</A> and <A href="https://2013.igem.org/Team:Imperial_College/3HB_assay" class="external text" rel="nofollow">data</A> pages for methods and details of the data.

</P></DIV></DIV><DIV id="CollapsiblePanel6" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Bioplastic degradation: Proteinase K enzymatic assays<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H2><SPAN class="mw-headline" id="Proteinase_K_kinetic_activity">Proteinase K kinetic activity</SPAN></H2>
According to the literature, protease K breaks down poly L-lactic acid as well as fibrous proteins, such as silk fibroin. In the active site of proteinase K is a catalytic triad containing serine, histidine and aspartic acid. L-lactic acid is analogous to L-alanine. Poly L-lactic acid, therefore, can bind to and be catalysed by the catalytic triad, as it mimics silk fibroin which is rich with L-alanine residues. Proteinase K cannot catalyse the breakdown of poly D-lactic acid, because the catalytic triad will not accommodate it. <A href="http://www.sciencedirect.com/science/article/pii/S014139101100142X" class="external autonumber" rel="nofollow">[46]</A><A href="http://www.biomedcentral.com/1472-6750/7/16" class="external autonumber" rel="nofollow">[47]</A><P align="justify">To obtain parameters to use in our <B>PLA degradation model</B> that kinetic activity of proteinase K was measured with a substrate analogue, N-succinyl Ala-Ala-Pro-Leu p-nitroanilide (AAPL p-NA). Cleavage of AAPL p-NA releases 4-nitroaniline (4-NA), which can be recorded by an increase in absorbance at wavelength 405nm. For this assay, <A href="http://www.sigmaaldrich.com/catalog/product/sial/p2308?lang=en&amp;region=GB" class="external text" rel="nofollow">proteinase K</A> and <A href="http://www.sigmaaldrich.com/catalog/product/sigma/s8511?lang=en&amp;region=GB" class="external text" rel="nofollow">AAPL p-NA</A> were purchased from Sigma.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:623px;"><DIV class="thumbcaption"><B>Molecular structure of N-succinyl Ala-Ala-Pro-Leu p-nitroanilide. Image by <A href="http://www.sigmaaldrich.com/catalog/product/sigma/s8511?lang=en&amp;region=GB" class="external text" rel="nofollow">Sigma</A></B></DIV></DIV></DIV></DIV>.
<P align="justify">The extinction coefficient of 4-nitroaniline is 9,500 M-1s-1 at wavelength 405 nm, according to <A href="http://www.sigmaaldrich.com/etc/medialib/docs/Sigma/Product_Information_Sheet/2/m4765pis.Par.0001.File.tmp/m4765pis.pdf" class="external text" rel="nofollow">Sigma</A>. Using the Beer-Lambert law, the concentrations of 4-NA were calculated and plotted against time.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:432px;"><DIV class="thumbcaption"><B>The absorbance change recorded during the catalysis of different concentrations of AAPLp-NA by Proteinase K. Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>The concentration of 4-NA produced from different starting concnetrations of AAPLp-NA. Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV><P align="justify">The initial velocities were calculated as the gradient between 0 and 30 seconds, when time = 0 seconds. These were plotted against the concentration AAPLp-NA and <A href="http://www.graphpad.com/" class="external text" rel="nofollow">graphpad</A> was used to estimate the Michaelis-Menten fit. It determined the Vmax as 46.31 µM/s and the Km as 19.28 µM.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:771px;"><DIV class="thumbcaption"><B>Michaelis-Menten plot for proteinase K activity cleaving AAPLp-NA at 37ºC and pH 7.4. Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="CollapsiblePanel10" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Bioplastic degradation: Proteinase K degrades PLA<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H3><SPAN class="mw-headline" id="Proteinase_K_degrades_PLA_cup">Proteinase K degrades PLA cup</SPAN></H3><P>Our bacteria are capable of growing on lactic acid, however the substance is toxic and results in a pronounced lag phase before growth commences:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">L-lactic acid in LB with Stress Response cells. MG1655 cells were grown on 0 mM and 5 mM L-Lactic Acid. Lactic acid is a byproduct of polylactic acid degradation, a key component to our project. Cells grown in 5mM L-lactic acid show a pronounced lag phase before commencing log growth. Growth was at 37°C with shaking over 6h. Error bars are SEM, n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV><P>We have demonstrated that Proteinase K can degrade PLA. The below images are Scanning electron micrographs from the surface of a PLA cup after incubation with the enzyme. Please see our <A href="https://2013.igem.org/Team:Imperial_College/Electron_microscopy" class="external text" rel="nofollow">SEM data page</A> for full description of expreiment and all SEM images. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Increased surface pitting and roughness indicated degradation of PLA surface after enzyme treatment. Figure by Imperial College London 2013 iGEM</B></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="CollapsiblePanel7" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>References<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H5>Papers Referenced</H5><OL class="reference"><LI>ANDERSON A, DAWES E. <CITE>Occurrence, Metabolism, Metabolic Role, and Industrial Uses of Bacterial Polyhydroxyalkanoates.</CITE> Microbiol Rev 1990 DEC;54(4):450-472.</LI><LI>Harding KG, Dennis JS, von Blottnitz H, Harrison STL. <CITE>Environmental analysis of plastic production processes: Comparing petroleum-based polypropylene and polyethylene with biologically-based poly-beta-hydroxybutyric acid using life cycle analysis.</CITE> J Biotechnol 2007 MAY 31;130(1):57-66.</LI><LI>Kim S, Dale BE. <CITE>Energy and Greenhouse Gas Profiles of Polyhydroxybutyrates Derived from Corn Grain: A Life Cycle Perspective.</CITE> Environ Sci Technol 2008 OCT 15;42(20):7690-7695.</LI><LI>Jendrossek D, Handrick R. <CITE>Microbial degradation of polyhydroxyalkanoates.</CITE> Annu Rev Microbiol 2002;56:403-432.</LI><LI>Philip S, Keshavarz T, Roy I. <CITE>Polyhydroxyalkanoates: biodegradable polymers with a range of applications.</CITE> Journal of Chemical Technology and Biotechnology 2007 MAR;82(3):233-247.</LI><LI>Thomas C. Bruice , Thomas H. Fife , John J. Bruno , Patricia. Benkovic<CITE>Hydroxyl Group (V)1 and Imidazole (X)2 Catalysis. The General Base Catalysis of Ester Hydrolysis by Imidazole and the Influence of a Neighboring Hydroxyl Group.</CITE>  J. Am. Chem. Soc., 1962, 84 (15), pp 3012–3018</LI><LI>Fusako Kawai, Kosuke Nakadai, Emiko Niskioka, Hajime Nakajima, Hitomi Ohara, Kazao Masaki, Haruyaki Lefuji.<CITE>Different enantioselectivity of two types of poly (lactic acid) depolymerases toward poly (L-lactic acid) and poly (D-lactic acid).</CITE>Polymer Degradation and Stability Volume 96, Issue 7, July 2011, Pages 1342–1348</LI><LI>Jun Liao, Manfred K Warmuth, Sridhar Govindarajan, Jon E Ness, Rebecca P Wang, Claes Gustafsson, Jeremy Minshull<CITE>Engineering proteinase K using machine learning and synthetic genes.</CITE>BMC Biotechnology 2007, 7:16 doi:10.1186/1472-6750-7-16</LI></OL></DIV></DIV><DIV id="CollapsiblePanel8" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Protocols<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H4>Clearing zone assay</H4><OL><LI>Prepare 10 ml cell culture and induce phaZ1 expression using 2% xylose media. </LI><LI>Centrifuge the cell cultures at 4500 RCF (equivalent to 6221g) for 20 minutes at 4°C.</LI><LI>Add 2 ml of 50mM Tris buffer (pH 7.4) to the pellet and mix well by vortexing the tube. </LI><LI>Transfer the buffered pellet into 7ml tubes and keep the samples on ice.</LI><LI>Perform probe sonication with 2-minute cycles of 2 seconds on, 2 seconds off, at 60% amplitude. Keep the sample on ice while sonicating.</LI><LI>Centrifuge the samples again at 4000 g for 10 minutes and extract the supernatant, in order to separate cell debris and proteins.</LI><LI>Each cuvette contains 700 ul of 0.5 mg/ml PHB emulsion and 300 ul of cell lysate. Use the 50mM Tris buffer as blank.</LI><LI>Measure the absorbance at OD650 every 24 hours. </LI><LI>Mix 0.5g of P3HB powder with 300 ml LB agar, make P3HB plates.</LI><LI>Use sterilised pipette tip to cut two wells, each has a 0.5 mm diameter. </LI><LI>Pipette 200ul of PhaZ1 cell lysate sample in one of the wells, and same amount of empty vector cell lysate sample in the other well.</LI><LI>Observe the plate every 24 hours.</LI></OL><H4>Protein purification by metal affinity chromatography</H4><OL><LI><B>Preparation:</B></LI>
10 mL cultures were incubated overnight. These were diluted in 500 mL the next morning and incubated until the OD reached 0.5-0.6. The cultures were then induced with 6mM arabinose or 2% xylose (final concentration)and left overnight. The cells were then pelleted by centrifugation, 4,000 rpm for 10 minutes and kept in a -80ºC freezer.

<LI><B>Cell lysis</B></LI>
The frozen pellets were allowed to thaw and resuspended in 10 mL of Tris-HCl binding buffer. The cells were lysed by probe-sonication at 60% amplitude with 2 minute cycles of 2 seconds on, 2 seconds off. The samples were kept on ice throughout the sonication process. The cell debris was pelleted by spinning and the supernatant removed for purification.

<LI><B>Purification with a Ni-NTA column:</B></LI>
The column was washed with Buffer A (10 mM imidazole, 40 mM Tris pH 8, and 400 mM NaCl) to equilibrate the resin. The samples are added and the column is washed with Buffer A, and then eluted with increasing concentrations of Buffer B (500 mM imidazole, 40 mM Tris pH 8 and 400 mM NaCl). 

<LI><B>SDS PAGE:</B></LI>
Samples were prepared for SDS PAGE by adding 7 µL of SDS running buffer to 20 µL of the fractions of interest. 8-20 µL of each sample were run on a polyacrylamide gel against a molecular weight marker.

<LI><B>Concentration:</B></LI>
PHB depolymerase (phaZ 1) and PUR esterase (EstC2) were concentrated with 10 mM potassium phosphate buffer.

<H2>Proteinase K kinetic activity assay</H2><OL><LI><B>Tris-HCl buffer ingredients:</B></LI>
1 litre of millicule water 
10.5 g of NaCl (180 mM)
0.65 g of CaCl2 (5 mM) 
1.7g of imidazole (25 mM)
6 g of Tris base (50 mM)
Add HCl to bring the pH to 7.4

<LI><B>Programme the spectrometer:</B></LI>
Cuvette: 10 mm
Wavelength: 405 nm
Temperature: 37°C
Measuring time: 10:00 min:sec
Delay: 00:00 min:sec
Interval: 00:30 min:sec

<LI><B>Reagent preparation:</B></LI>
Lyophilized powder proteinase K from Tritirachium album from Sigma-Aldrich was used, made into a 10 mg/ml stock solution with the buffer. The spectrometer was blanked with 1mL of buffer and 3 µL of proteinase K stock solution; this is equivalent to one unit of enzyme. 
 
We used N-Succinyl Ala-Ala-Pro-Leu p-nitroanilide (AAPLp-NA) from Sigma-Aldrich as a substrate analog. Make a stock solution with buffer (10 mg/ml). The reagents were incubated at 37°C just before use. 

The absorbance of each reaction mixture was measured at 405 nm, for 10 minutes at 30 second intervals. The reaction mixtures contained 3 µL of proteinase K stock solution and AAPLp-NA in Tris-HCl buffer to make final concentrations of 10 µM, 25 µM, 50 µM, 75 µM, 100 µM, 250 µM and 500 µM.<DIV id="CollapsiblePanel9" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>MSDS<FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H5><SPAN class="mw-headline" id="Chemical_reagents">Chemical reagents</SPAN></H5><P align="justify"> Some of the reagents we used were slightly toxic to human. In order to reduce the risks of using toxic reagents, we always wear gloves and labcoats. For some toxic reagents, we performed in fume cupboard. Click on the reagents to see the MSDS.</P><P>1. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=363502&amp;brand=ALDRICH&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Faldrich%2F363502%3Flang%3Den" class="external text" rel="nofollow">Poly(R)-3-hydroxybutyric acid</A>.</P><P> 2. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=GB&amp;brand=SIAL&amp;productNumber=P2308&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sial/p2308?lang=en&amp;region=GB" class="external text" rel="nofollow">Proteinase K</A>.</P><P> 3. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=54965&amp;brand=ALDRICH&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Faldrich%2F54965%3Flang%3Den" class="external text" rel="nofollow">Sodium 3-hydroxybutyrate</A>.</P><P>4. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=A3256&amp;brand=SIGMA&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Fsigma%2Fa3256%3Flang%3Den" class="external text" rel="nofollow">Arabinose</A>.</P><P> 5. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=A8509&amp;brand=ALDRICH&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Faldrich%2Fa8509%3Flang%3Den" class="external text" rel="nofollow">Acetoacetate</A>.</P></DIV></DIV><DIV class="TabbedPanelsContent"><P><B>We are changing the pelB secretion tag to OsmY-fusion partner for better secretion</B></P><P>As seen in the <A href="https://2013.igem.org/Team:Imperial_College/Western_Blots" class="external text" rel="nofollow">Western Blot</A> results, some of the enzymes designed for secretion were not secreted. These proteins may have been stuck in the periplasmic space. Thus we are optimising enzyme secretion by building a new construct with OsmY <A href="http://parts.igem.org/Part:BBa_K892008" class="external text" rel="nofollow">BBa_K892008</A> fusion for secretion. OsmY is naturally secreted from E.coli and will carry our PUR esterase enzyme with. Once in the culture medium, we think that the enzyme is going to be functional in the fusion form but we included a TEV site in between the domains that can be cleaved by TEV protease if needed.</P><P>The linker sequence we have designed between the domains is <B>T G S E N L Y F Q G S</B> (ACCGGCAGCGAGAACCTGTACTTCCAAGGCAGC) and includes the 7 amino acids of the TEV site as well as 4 other amino acids for flexibility. 
</P><P>This construct will be a substitute for <A href="http://parts.igem.org/Part:BBa_K1149010" class="external text" rel="nofollow">BBa_K1149010</A> and <A href="http://parts.igem.org/Part:BBa_K1149008" class="external text" rel="nofollow">BBa_K1149008</A> biobricks which are the PHB depolymerase PhaZ1 and ProteinaseK with the pelB seretion tag.
</P><P><B class="clear">Proteinase K activity</B></P><P>The Western Blot results show that proteinase K was expressed, however, no activity was detected in a cell lysate assay and no proteinase K could be purified from the cells. The proteinase K cultures were induced overnight with 6 mM (final concentration) of arabinose. According to the literature, proteinase K self-digests <A href="http://www.biomedcentral.com/1472-6750/7/16" class="external text" rel="nofollow">(8)</A>. Perhaps the induction strength and time was too long. Next time we will reduce the time and concentration of induction for proteinase K.</P></DIV><DIV id="supporters" class="clear"><P><B>Our Sponsors</B></P><A href="/File:Logo-bioline.gif" title="File:Logo-bioline.gif">125px</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Imperial_College/BioPlastic_Recycling:_PHB">http://2013.igem.org/Team:Imperial_College/BioPlastic_Recycling:_PHB</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College/BioPlastic_Recycling:_PHB" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College/BioPlastic_Recycling:_PHB" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College/BioPlastic_Recycling:_PHB&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College/BioPlastic_Recycling:_PHB&amp;oldid=361612" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></OL></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>