"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College/Modelling PLAdeg
From 2013.igem.org
Project: Plasticity
At a glance
Waste Issue
M.A.P.L.E. Modules
M1: Resource-full Waste
M2: Plastic Fantastic
Bioplastic applications
Human Practices
M.A.P.L.E
Industrial Implementation
Social Challenge
Communicating our Project
Project Evolution
Achievements
Judging Criteria
Main Results
Collaborations
Biobricks
In the Lab
Safety
Lab book
Modelling Notes
Introduction
Beta amyloid degradation
Data
Growth Assays
Western Blot
Enzyme Kinetics
PHB production
3HB assay
Clearing Assays
Electron microscopy
Protocols
Team
Students
Advisors
Sponsors
Attributions
Contents
1 Polyhydroxybutyrate (P(3HB)) Synthesis Module
1.1 Metabolic modelling
1.1.1 Introduction of the metabolic model
1.1.2 Methods
1.1.3 ATP analysis:
1.1.4 Large scale PHB production and implementation in MAPLE
Polyhydroxybutyrate (P(3HB)) Synthesis Module
Metabolic modelling
Introduction of the metabolic model
The metabolic model predicts the interaction of our pathway with endogenous pathways. In synthetic biology, an ideal system is fully orthogonal. However, if the synthetic pathway uses some metabolites from the metabolism pathways, the metabolic fluxes (the rate of conversion between metabolites) will be affected. If the cell is overloaded by losing too much metabolites, it would either reject the synthetic pathway or burn out. As for the optimization of PHB production in our MAPLE system, metabolic analysis need to be carried out to estimate the effects on the cell metabolism from increases in the PHB production.
Methods
We altered the metabolic model by Angela Dixon in her paper “Predictive Mathematical Model for Polyhydroxybutyrate Synthesis in Escherichia coli”. [1] The metabolic model is built in Simbiology, which is an extension in MATLAB especially for biological system analysis. The model consists two metabolic pathways: Glycolysis and TCA cycle. The reactions in the metabolic pathways are defined by ordinary differential equations and kinetic parameters. The model also contains a synthetic pathway that produce P(3HB) from acetyl-coA. The pathway consists three enzymes which are PhaA, PhaB and PhaC.  The reasons we chose this model:
1. The model is based on ODEs instead of FBA (flux balance analysis) method, which is a common methods of metabolic analysis. However, FBA cannot determine the real time concentration of the metabolites whereas it can be done by models of ODEs.
2. Only single gene deletion or addition can be performed in FBA. More than that, the metabolic model here can perform modification on any single or multiple genes and reactions through Simbiology.
3. The FBA model is static, it cannot analyze the dynamic interactions between our synthetic pathways with the metabolic pathways. As both our P(3HB) synthesis model and the metabolic model are in the same platform. It was really easy to integrate our pathway into the metabolic model.
In our project, we have a different pathway:
1. We use 3HB as one of the input.
2. Existence of ato system in the E.coli MG1655 strain.
3. We removed phaA in our case because we use 3HB as the major source to produce P(3HB) to avoid too much acetyl-coA be taken from the metabolic pathways.
Therefore, we substitute the original pathway with our synthetic pathway. Moreover, we added our gene expression models for the enzymes into the metabolic model as well. In terms of the consistency of the model, we changed the units of our synthetic pathway into micro molar and seconds.
Here is the combination of our model with the metabolic model
We also carried out a parameter estimations of the initial concentrations of several substrates in our synthetic pathway. The reason is we have single-cell scaled magnitude for them originally (fM level), but there are much higher magnitudes in any of the species in the metabolic pathways (mM level). Therefore, we scaled up the pathway in order to maintain the consistency of the model. As for the metabolites which are involved in our pathway, we cloned the blocks of them from the metabolic pathway in Simbiology and deliberately put them into our pathway. That means our pathway dynamically interact with the metabolic pathways.
Here is the table for the updated initial concentrations for the species in our synthetic pathway: All the kinetic data and initial concentrations of the metabolites in the metabolic pathways are referenced in the paper [2] :
Substrates
Concentration
Units
Sources
Assumptions
3HB
5000
uM
-
The initial input to the system
Acetoacetate
1000
uM
-
The ato system in E.coli suggest a favourable conversion from acetoacetyl-coA to acetoacetate. Therefore, the concentration of acetoacetate is 10 times larger than acetoacetyl-coA [3]
Acetoacetyl-coA
100
uM
[4]
The ato system in E.coli suggest a favourable conversion from acetoacetyl-coA to acetoacetate. [5]
NADH
200
uM
[6]
-
NAD+
1200
uM
[7]
-
Acetyl-coA
5000
uM
[8]
-
Coenzyme A
250
uM
[9]
-
Acetate
40000
uM
[10]
Assay data from the literature which the ato system efficiency was measured
3HB-coA
0
uM
-
One of the intermediate product of the pathway
NADPH
200
uM
[11]
-
NADP+
1200
uM
[12]
-
PHB
0
uM
-
The final product of the pathway
ATP analysis:
In our optimization of P(3HB) production module, we discovered that increasing in the PhaB expression can efficiently increase the yield of P(3HB). It’s important to see whether increase in the PhaB expression would dramatically change the fluxes in the metabolic pathways or not. As ATP is the essential energy carriers in the cell, we carried a parameter scan for the variation in ATP concentrations levels under different concentration of PhaB.
There is an approximately 5.3% drop in the concentration of ATP when increasing the phaB concentration from 50uM to 200uM. It suggests that increasing the efficiency in our pathway will reduce the efficiency of energy transports in the cell. However, it’s still unsure about  the effect of 5.3% drop in ATP concentration on the cell growth.
Large scale PHB production and implementation in MAPLE
In our MAPLE system, the bioplastic Polyhydroxybutyrate P(3HB) is produced from both 3HB monomers and glucose. 3HB is the degradation product from the wasted P(3HB）plastic and glucose is the product from the degradation of mixed waste in Module 1. Therefore, the overall PHB production rates need to be defined by considering different situations:
1. In module 1, we have glucose as the only source to produce PHB.
2. In module 2, we have both glucose and 3HB as resources.
Our Sponsors
125px
Retrieved from " http://2013.igem.org/Team:Imperial_College/Modelling_PLAdeg "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
