<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_Waste_Degradation_SRF"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College/Waste_Degradation:_SRF">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College/Waste_Degradation:_SRF&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College/Waste_Degradation:_SRF&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College/Waste_Degradation:_SRF&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College/Waste_Degradation:_SRF" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College/Waste Degradation: SRF</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><UL class="navigation-menu" style=""><LI><A href="https://2013.igem.org/Team:Imperial_College">Project: Plasticity</A></LI><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Imperial_College/tour">At a glance</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Background">Waste Issue</A></LI><LI>M.A.P.L.E. Modules</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Waste_Degradation:_SRF">M1: Resource-full Waste</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/BioPlastic_Recycling:_PHB">M2: Plastic Fantastic</A></LI></UL><LI><A href="https://2013.igem.org/Team:Imperial_College/Applications">Bioplastic applications</A></LI><LI>Human Practices</LI><LI class="maple"><A href="https://2013.igem.org/Team:Imperial_College/MAPLE">M.A.P.L.E</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Industrial_Implementation">Industrial Implementation</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Managing Waste">Social Challenge</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Communication_work">Communicating our Project</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Project_Evolution">Project Evolution</A></LI><LI>Achievements</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Judging_Criteria">Judging Criteria</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/mainresults">Main Results</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Collaboration">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Biobricks">Biobricks</A></LI><LI>In the Lab</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/MSDS">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/labbook">Lab book</A></LI><LI><A href="#">Modelling Notes</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Modelling_Intro">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Modelling_beta-amyloid">Beta amyloid degradation</A></LI><LI><A href="#">Data</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Growth_Assays">Growth Assays</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Western Blots">Western Blot</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Enzyme Kinetics">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/PHB production">PHB production</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/3HB_assay">3HB assay</A></LI><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Imperial_College/Clearing_Zone_Assay">Clearing Assays</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Electron microscopy">Electron microscopy</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Protocols">Protocols</A></LI><LI>Team</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Meet_the_Team">Students</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Advisors">Advisors</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Attributions">Attributions</A></LI><DIV id="main-content"><H1><SPAN class="mw-headline" id="Module_1:_Resource-full_Waste">Module 1: Resource-full Waste</SPAN></H1><P><B>Non-recyclable waste is sourced from a recycling centre, placed in a bioreactor with our M.A.P.L.E system which degrades the waste and synthesises the bioplastic P(3HB). Click the tabs to find out more</B></P><DIV id="TabbedPanels1" class="TabbedPanels"><UL class="TabbedPanelsTabGroup"><LI class="TabbedPanelsTab" tabindex="0"><H4>Overview</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Specifications</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Design</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Modelling</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Assembly</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Testing &amp; results</H4></LI><LI class="TabbedPanelsTab" tabindex="0"><H4>Future work</H4></LI></UL><DIV class="TabbedPanelsContentGroup"><DIV class="TabbedPanelsContent"><P>Waste from recovery facilities is a mixture of plastics and cellulosic rich biomass such as fibres and wood. The Resource-full waste module utilises the large variety of materials organisms can naturally degrade to recycle the waste and to provide food for the production of the bioplastic poly-3-hydroxybutyrate, P(3HB).  </P><H4>Separating polyurethane and selling its breakdown products</H4><P>Certain enzymes have the ability to degrade plastics as a result of their evolutionary history, dealing with tough and highly variable substrates such as lignin. Various enzymes have been biobricked to deal with most of the major petrochemical plastics, except for polyurethane. Polyurethane is one of the plastic constituents of this mixed waste. We have designed biobricks containing naturally occurring polyurethane esterases (also known as PUR esterases) to break polyurethane down into its constituent chemicals of ethylene glycol and poly(isocyanate). These valuable chemicals will be separated using specialised filters and sold back to industry.</P><H4>Using the residual material as a substrate for P(3HB) production</H4><P>The remaining organic portion of the mixed waste is used as a carbon source for P(3HB) production. P(3HB) is produced by organisms containing the pha CAB operon, originally found in soil bacteria such as <I>Ralstonia eutropha</I>. The operon contains three genes which are members of the pathway required to make P(3HB). P(3HB) is used as a storage molecule in the organisms which produce it. They do this when nutrients are limiting but when there is a plentiful carbon source.</P><P>P(3HB) is produced commercially by several companies. One of the crucial factors in whether production is economical or not is  the efficiency of bioplastic production. We have hugely increased the efficiency of P(3HB) production; an important step in making our system feasible.</P><P>To scale up the production of P(3HB), our system would require large amounts of purified sugars. We would, therefore, hydrolyse the long polysaccharide components to release the sugars. This is a current industrial pre-treatment technique to allow the fermentation of ligno-cellulosic biomass.</P><P>References:</P><OL class="reference"><LI>Harding KG, Dennis JS, von Blottnitz H, Harrison STL. <CITE>Environmental analysis of plastic production processes: Comparing petroleum-based polypropylene and polyethylene with biologically-based poly-beta-hydroxybutyric acid using life cycle analysis.</CITE> J Biotechnol 2007 MAY 31;130(1):57-66.</LI><LI>Kim S, Dale BE. <CITE>Energy and Greenhouse Gas Profiles of Polyhydroxybutyrates Derived from Corn Grain: A Life Cycle Perspective.</CITE> Environ Sci Technol 2008 OCT 15;42(20):7690-7695.</LI><LI>Kendall A. A life cycle assessment of biopolymer production from material recovery facility residuals. Resources, Conservation and Recycling 61 (2012) 69– 74 </LI>
conomou A. Following the leader: bacterial protein export through the Sec pathway. Trends in microbiology 1999;7(8) 315-320.
<LI>Chul-Hyung Kang, Ki-Hoon Oh, Mi-Hwa Lee, Tae-Kwang Oh, Bong Hee Kim, Jung- Hoon Yoon.<CITE> A novel family VII esterase with industrial potential from compost metagenomic library</CITE>Microbial Cell Factories 2011, 10:41 doi:10.1186/1475-2859-10-41</LI><LI>Da Almeida, Alejandra. <CITE>Effect of the granule associated protein phasin (PhaP) on cell growth and poly(3-hydroxybutyrate) (PHB) accumulation from glycerol in bioreactor cultures of recombinant E. coli.</CITE> Volume 131, Issue 2, Supplement, September 2007, Pages S167.</LI></OL></DIV><DIV class="TabbedPanelsContent"><H2 id="specification">Specification</H2><P><B>1. Our bacteria should survive and grow in mixed waste</B></P><P>In order for the bacteria to produce bioplastic from the mixed waste they first need to be able to use     
              it as a carbon source</P><P><B>2. Our bacteria should secrete a functional polyurethane esterase</B></P><P>To recover the resources from polyurethane in the mixed waste, the enzymes should secrete in an active form.</P><P><B>3. Our bacteria should be able to tolerate mixed waste degradation products, such as ethylene glycol</B></P><P>The products of polyurethane degradation are toxic and so any bacteria growing with them must be able to survive to a concentration which will allow economical production of ethylene glycol. We have chosen a strain of <I>E.coli</I>-MG1655, which is resistant to ethylene glycol toxicity.</P><P><B>4. Our bacteria should produce P(3HB)</B></P><P><B>5. Our bacteria should produce P(3HB) from the mixed waste</B></P></DIV><DIV class="TabbedPanelsContent"><DIV id="CollapsiblePaneldesign1" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>PUR Degradation: Pathway      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P>Lignin degrading microbial enzymes are capable of degrading plastics. Some previous iGEM teams have exploited this ability. We have built on their work and extended the plastic degradation capabilities of the synthetic biology community by improving polyurethane (PUR) degradation, as this has not been successfully achieved before. We have identified 5 PUR-esterase enzymes from the literature that are capable of catalyzing the below reaction.</P>
We have synthesised all of the genes in the below table and are testing them for expression in <I>E.coli</I>, secretion, activity and PUR degradation capabilities. Our ultimate design is to be able to control the relative levels of different enzymes in a waste-degrading bio-reactor, in order to adjust it to the composition of waste. We have, therefore, designed the expression constructs accordingly. Our models predict the degradation rate of polyurethane at the bioreactor scale. 
. 



</DIV></DIV><DIV id="CollapsiblePaneldesign2" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>PUR Degradation: BioBrick Designs      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH width="150">enzyme</TH><TH>source organism</TH><TH>biobrick</TH><TH>reference</TH></TR><TR><TH>EstCS2</TH><TD>uncultured unknown bacterium (GU256649.1)</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149002">BBa_K1149002</A></TD><TD><A href="http://www.microbialcellfactories.com/content/10/1/41">Kang et.al 2011</A></TD></TR><TR><TH>pueA</TH><TD>Pseudomonas chlororaphis</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149003">BBa_K1149003</A></TD><TD><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2000.tb09056.x/pdf">Stern et al., 2000</A></TD></TR><TR><TH>pueB</TH><TD>Pseudomonas chlororaphis</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149004">BBa_K1149004</A></TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830501000427">Howard et al., 2001</A></TD></TR><TR><TH>pudA</TH><TD>Comamonas acidovorans </TD><TD><A href="http://parts.igem.org/Part:BBa_K1149005">BBa_K1149005</A></TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830598000663">Allen et al. 1999</A></TD></TR><TR><TH>pulA</TH><TD>Pseudomonas fluorescens</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149006">BBa_K1149006</A></TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830598000687"> Vega et al.,  1999</A></TD></TR></TBODY></TABLE><P><B>Secretion Strategy:</B></P><P align="justify"><I>E.coli</I> is commonly used as a chassis in innovative iGEM projects that aim to prove a concept and make the case for a novel function in a biologically engineered machine.  In our case, we aim to degrade and synthesise plastic and the degradation part of our system needs to be extracellular. There are many strategies for secretion. You can read about these on our <A href="https://2013.igem.org/Team:Imperial_College/SecretionHelp" class="external text" rel="nofollow">Secretion Guide page</A> and find the corresponding biobricks and projects. If you are currently in the process of designing a Genetically Engineered Machine in E.coli, this page provides a good starting point to look for information.</P><P align="justify">We chose the pelB secretion tag as it has been demonstrated to work in many cases, sometimes with very high transport efficiency (1)(2). pelB has been used in iGEM projects for many years and is part of 50+ constructs. The UC-Davis team last year used it to secrete LC-Cutinase, a PET plastic degrading enzyme <A href="http://parts.igem.org/Part:BBa_K936013" class="external text" rel="nofollow">(BBa_K936013)</A> successfully, which is somewhat similar to our plastic-degrading enzymes.</P><P><B>Safety:</B></P><P>We considered the safety aspects of using the PudA enzyme, since its sequences were originally from a Risk Group 2 (RG2) organism. We submitted our safety information to iGEM for review. <B>Safety forms were approved on October 2nd, 2013 by the iGEM Safety Committee.</B> We did not use the organism only the part itself and the final sequences were codon optimised for expression in E. coli, which included the elimination of the forbidden restriction sites.</P><P>References</P><OL class="reference"><LI>Thanassi DG, Hultgren SJ. Multiple pathways allow protein secretion across the bacterial outer membrane. Current opinion in cell biology 2000;12(4) 420-430.</LI><LI>Sletta H, Tondervik A, Hakvag S, Aune TEV, Nedal A, Aune R, et al. The presence of N-terminal secretion signal sequences leads to strong stimulation of the total expression levels of three tested medically important proteins during high-cell-density cultivations of <I>E.coli</I>. Applied and Environmental Microbiology 2007;73(3) 906-912.</LI><DIV id="CollapsiblePaneldesign3" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Bioplastic Synthesis: Pathway      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P align="justify">Poly-3-hydroxybutyrate P(3HB) is a polyester which is naturally produced inside bacteria such as  <I>Ralstonia eutropha</I>, where it accumulates as granules inside the cell. In its native bacteria it is produced as an energy store (1).It's properties as a plastic has gained it the attention of researchers and industrialists. In our system, the bacteria will take up organic molecules from waste and use them for P(3HB) production. </P><P align="justify">We have made P(3HB) in <I>E.coli</I>, transferring three genes naturally found in  <I>Ralstonia eutropha</I> into <I>E.coli</I> MG1655. These encode the three enzymes necessary for P(3HB) production; polyhydroxyalkanoate synthase(phaC), 3-ketothiolase(phaA) and acetoacetyl coenzyme A reductase(phaB). These are encoded by the pha CAB operon. We have altered the expression of these three genes to maximise the production of P(3HB) as high yields are required for it to be economically viable.</P><P><B>3-ketothiolase:</B></P><P><B>Acetoacetyl coenzyme A reductase:</B></P><P><B>Polyhydroxyalkanoate synthase:</B></P></DIV></DIV><DIV id="CollapsiblePaneldesign4" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4> Bioplastic Synthesis: Biobrick Designs      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P>We used the pha CAB biobrick <A href="http://parts.igem.org/Part:BBa_K934001" class="external text" rel="nofollow">BBa_K934001</A> to produce P(3HB) which contains the native operon of <I>Ralstonia eutropha</I>. Inthe future we would like to see our system implemented at an industrial scale. This requires high yields to be economically viable. We created P(3HB) synthesis models to inform our experimental attempts to increase P(3HB) yield. We were especially interested to find out if up or down-regulation of any of the enzymes involved in the biosynthetic pathway could increase yields. To analyse this, we needed to know the reactions in the metabolic pathway and also the relative parameters of the enzymes and their dynamic relationships in order to identify bottlenecks in the flux of metabolites. The results from the metabolic model suggest that the amount of pha B is especially critical and increasing its level should give us more P(3HB). (Please see our modelling section for details.) To do this, we designed constructs with stronger promoters to up-regulate the expression of the operon. </P><P>We designed and constructed two new constructs for increased expression of the operon (including pha B). The first construct uses the constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23104" class="external text" rel="nofollow">J23104</A> and the <A href="http://parts.igem.org/Part:BBa_B0034" class="external text" rel="nofollow">RBS 0034</A> with a following scar site TACTAGAG in front of the ATG of the pha C gene. The second construct was designed as a hybrid promoter incorporating BBa_J23104 and the original promoter first, followed by the native promoter and RBS, due to recent results of high expression from a hybrid promoter<A href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0056321" class="external autonumber" rel="nofollow">[1]</A>. The region containing the native  promoter and RBS is 352 nucleotides long and might contain important regulatory elements and therefore we were interested to test the hybrid construct in comparison to both the native and constitutive promoter configurations.&lt;/</P></DIV></DIV><DIV id="CollapsiblePaneldesign5" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4> Chassis Design: Tolerance to Waste      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P align="justify">We chose the MG1655 <I>E.coli</I> strain as chassis because it constitutively expresses genes that make it resistant to toxic waste degradation products, such as Ethylene glycol. 
The <A href="http://parts.igem.org/Part:BBa_K892010" class="external text" rel="nofollow">aldA</A> and <A href="http://parts.igem.org/Part:BBa_K892009" class="external text" rel="nofollow">fucO</A> genes have an important role in decreasing the toxic effects of ethylene glycol by converting it into glycolaldehyde which is a link to the cell`s central metabolism. We have received these genes from the registry and future work could be to express these in <I>Bacillus</I> or cellulose-degrading organisms to make them tolerant as well. 
</P></DIV></DIV><DIV id="CollapsiblePaneldesign6" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Enzyme Secretion Strategy <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P align="justify"><I>E.coli</I> is commonly used as a chassis in innovative iGEM projects that aim to prove a concept and make the case for a novel function in a biologically engineered machine.  In our case, we aim to degrade and synthesise plastic and the degradation part of our system needs to be extracellular. There are many strategies for secretion. You can read about these on our <A href="https://2013.igem.org/Team:Imperial_College/SecretionHelp" class="external text" rel="nofollow">Secretion Guide page</A> and find the corresponding biobricks and projects. If you are currently in the process of designing a Genetically Engineered Machine in E.coli, this page provides a good starting point to look for information.</P><P align="justify">We chose the pelB secretion tag as it has been demonstrated to work in many cases, sometimes with very high transport efficiency (3). pelB has been used in iGEM projects for many years and is part of 50+ constructs. The UC-Davis team last year used it to secrete LC-Cutinase, a PET plastic degrading enzyme <A href="http://parts.igem.org/Part:BBa_K936013(BBa_K936013)" class="external autonumber" rel="nofollow">[2]</A> successfully, which is somewhat similar to our plastic-degrading enzymes.</P><P>References</P><OL class="reference"><LI>Economou A. Following the leader: bacterial protein export through the Sec pathway. Trends in microbiology 1999;7(8) 315-320.</LI><LI>Thanassi DG, Hultgren SJ. Multiple pathways allow protein secretion across the bacterial outer membrane. Current opinion in cell biology 2000;12(4) 420-430.</LI><LI>Sletta H, Tondervik A, Hakvag S, Aune TEV, Nedal A, Aune R, et al. The presence of N-terminal secretion signal sequences leads to strong stimulation of the total expression levels of three tested medically important proteins during high-cell-density cultivations of <I>E.coli</I>. Applied and Environmental Microbiology 2007;73(3) 906-912.</LI></OL></DIV></DIV><DIV id="CollapsiblePaneldesign7" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Safety      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P>Safety issues regarding our chassis can be found <A href="https://2013.igem.org/Team:Imperial_College/MSDS" class="external text" rel="nofollow">here</A>.</P></DIV></DIV><DIV class="TabbedPanelsContent"><H2 id="Modelling">Modelling</H2><H4><SPAN class="mw-headline" id="What_we_learnt_from_the_PUR_degradation_model:">What we learnt from the PUR degradation model:</SPAN></H4><UL><LI>The model assisted the design of enzyme assays, in particular:
</LI><LI>The toxicity of the PUR degradation product, Ethylene glycol was taken into account in the modelling and it successfully determined the maximum tolerance of the system.
</LI><LI>Ethylene glycol toxicity assays were carried out in the wet lab and were designed according to the results from our model.
</LI></UL><H4><SPAN class="mw-headline" id="What_we_learnt_from_the_P.283HB.29_synthesis_model:">What we learnt from the P(3HB) synthesis model:</SPAN></H4><UL><LI> Scanning concentration of pha B and sensitivity analysis showed that increasing the concentration of pha B would increase the production rate of P(3HB) in our engineered 'E.coli'.
</LI><LI> Simulations showed that the constitutive promoter J23104 could lead to a higher expression of pha B than the native promoter. As a result the Wet lab team designed and built  <A href="http://parts.igem.org/Part:BBa_K1149051" class="external text" rel="nofollow">BBa_K1149051</A>, which significantly increased bioplastic production.
</LI><LI> Results from the metabolic model suggested that the synthesis of P(3HB) would be accompanied by a drop of ATP over time and hence allowed us to identify potential limitations of our engineered system. This is important for the <A href="https://2013.igem.org/Team:Imperial_College/Industrial_Implementation" class="external text" rel="nofollow">Industrial Implementation of our system.</A></LI></UL><DIV id="CollapsiblePanelMod1" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H3>1.0 Polyurethane (PUR) degradation model      <FONT size="1">▼</FONT></H3><H4><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H4><P align="justify">
The efficiencies for polyurethane (PUR) degradation and ethylene glycol production are important for the performance of our system. We built a mathematical and deterministic model that is based on MATLAB extension Simbiology for polyurethane degradation. The model contains the kinetic properties of degradation enzymes that are helpful for the design of assays. As we scaled up the initial concentrations of all substrates to meet the conditions for a bio-reactor, the model can provide preliminary simulations and predictions for the <B>MAPLE</B> system. </P><H4><SPAN class="mw-headline" id="Design">Design</SPAN></H4><H5><SPAN class="mw-headline" id="Objective">Objective</SPAN></H5><P>Here are some specific objectives for the model to achieve:</P><P>1. The model should contain the gene expression model of the degradation enzymes because the enzyme concentration determines the rate of plastic degradation. In our case for PUR degradation, we used <A href="http://parts.igem.org/Part:BBa_K206000" class="external text" rel="nofollow">pBAD strong promoter K206000</A> for most enzymes. We built the gene expression model based on inducible pBAD promoter, which gene expression rate can be regulated by inducer concentration.</P><P>2. The model should show the efficiency of the enzyme secretion to the culture from the cells. It's also important because the enzyme concentration in the culture depends on it. Here we used pelB secretion tag for most enzymes in order to achieve a high efficiency.</P><P>3. The model basically predicts how long will take to degrade a known concentration of soluble polyurethane. It is assumed that the enzyme in our assays has the same kinetic properties as the enzyme used in the literature. The model can suggest a suitable concentration of the plastic to use in order to get good results from the assays.   </P><P>4. It is known that ethylene glycol is toxic to <I>E.coli</I>. It has, however, no clear effect on the growth of our MG1655 strain when the concentration of ethylene glycol is below 200 mM. The model, therefore, should suggest a safe range of polyurethane concentration to avoid a high concentration (&gt;200mM) of ethylene glycol produced. </P><H4><SPAN class="mw-headline" id="The_Model">The Model</SPAN></H4><P align="justify">Polyurethane (PUR) degradation involves 5 different degradation enzymes:</P><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH width="150">enzyme</TH><TH>source organism</TH><TH>biobrick</TH><TH>reference</TH></TR><TR><TH>EstCS2</TH><TD>uncultured unknown bacterium (GU256649.1)</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149002" class="external text" rel="nofollow">BBa_K1149002</A></TD><TD><A href="http://www.microbialcellfactories.com/content/10/1/41" class="external text" rel="nofollow">Kang et.al 2011</A></TD></TR><TR><TH>pueA</TH><TD>Pseudomonas chlororaphis</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149003" class="external text" rel="nofollow">BBa_K1149003</A></TD><TD><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2000.tb09056.x/pdf" class="external text" rel="nofollow">Stern et al., 2000</A></TD></TR><TR><TH>pueB</TH><TD>Pseudomonas chlororaphis</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149004" class="external text" rel="nofollow">BBa_K1149004</A></TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830501000427" class="external text" rel="nofollow">Howard et al., 2001</A></TD></TR><TR><TH>pudA</TH><TD>Comamonas acidovorans </TD><TD><A href="http://parts.igem.org/Part:BBa_K1149005" class="external text" rel="nofollow">BBa_K1149005</A></TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830598000663" class="external text" rel="nofollow">Allen et al. 1999</A></TD></TR><TR><TH>pulA</TH><TD>Pseudomonas fluorescens</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149006" class="external text" rel="nofollow">BBa_K1149006</A></TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830598000687" class="external text" rel="nofollow">Vega et al.,  1999</A></TD></TR></TBODY></TABLE><P>4 of the 5 enzymes are not well characterised before, so we could't find enough kinetic data from the literature. The only well-characterised PUR degradation enzyme PudA is used in the model as an illustration of all PUR degradation enzymes. The model will be more complete when the kinetic data of the other enzymes are defined. The finished PUR degradation model is shown as below: </P><P>There are two compartments which represents cells and the culture from left to right. The &quot;cell&quot; compartment contains the gene expression module whereas the &quot;culture&quot; compartment contains the degradation module. The &quot;secretion&quot; block that connects two compartments is the secretion module. </P><H4><SPAN class="mw-headline" id="Parameters_and_assumptions">Parameters and assumptions </SPAN></H4><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Source</TH><TH>Assumptions/Notes</TH></TR><TR><TH>β</TH><TD>maximum rate of transcription</TD><TD>0.015</TD><TD>mM/min</TD><TD>Please see derivation 1 below.</TD><TD>Please see derivation 1 below.</TD></TR><TR><TH>K</TH><TD>Activation coefficient</TD><TD>0.0031</TD><TD>mM</TD><TD><A href="http://parts.igem.org/Part:BBa_K206000:Characterization" class="external autonumber" rel="nofollow">[3]</A></TD><TD>Taking the &quot;switch point&quot; as the activation coefficient</TD></TR><TR><TH>d<SUB>mRNA</SUB></TH><TD>mRNA degradation rate</TD><TD>0.035</TD><TD>1/min</TD><TD><A href="http://jb.asm.org/content/189/23/8746.full" class="external autonumber" rel="nofollow">[4]</A></TD><TD>There is no active degradation pathway and that dilution is the dominant way by which it degrades.  Rate = ln2/doubling time, where doubling time of strain MG1655 = 20min.  Assuming steady-state growth in LB broth as presented in paper.  rate = ln2/20 = 0.035/min</TD></TR><TR><TH>d<SUB>protein</SUB></TH><TD>Protein degradation rate</TD><TD>0.035</TD><TD>1/min</TD><TD><A href="http://jb.asm.org/content/189/23/8746.full" class="external autonumber" rel="nofollow">[5]</A></TD><TD>There is no active degradation pathway and that dilution is the dominant way by which it degrades.  Rate = ln2/doubling time, where doubling time of strain MG1655 = 20min.  Assuming steady-state growth in LB broth as presented in paper.  rate = ln2/20 = 0.035/min</TD></TR><TR><TH>k<SUB>2</SUB></TH><TD>Protein production rate (PudA)</TD><TD>2.2</TD><TD>1/min</TD><TD>Please see derivation 2 below.</TD><TD>Please see derivation 2 below.</TD></TR><TR><TH>[Arabinose]</TH><TD>Concentration of arabinose</TD><TD>Initial: 0.008</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[mRNA]</TH><TD>Concentration of mRNA</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[PudA]</TH><TD>Concentration of PudA</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE><P><B>1.Derivation of the maximal expression rate,β</B><UL><LI>Average molecular weight (Mw) of a base pair = 660g/mol<A href="http://www.geneinfinity.org/sp/sp_dnaprop.html" class="external autonumber" rel="nofollow">[6]</A><A href="http://www.lifetechnologies.com/uk/en/home/references/ambion-tech-support/rna-tools-and-calculators/dna-and-rna-molecular-weights-and-conversions.html" class="external autonumber" rel="nofollow">[7]</A></LI><LI>Average mass of a base pair = 660g/mol x 1.66x10<SUP>-24</SUP> = 1.1x10<SUP>-21</SUP>g
</LI><LI>Volume of an E.coli cell = 1µm<SUP>3</SUP><A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external autonumber" rel="nofollow">[8]</A> = 1x10<SUP>-15</SUP>L
</LI><LI>∴Mass concentration = </LI><LI>∴Molar concentration of 1 base pair in the volume of E.coli =  = 1.66x10<SUP>-6</SUP> mM
</LI></UL><LI>BioBrick assembly plasmid pSB1C3 is a high copy number plasmid (100-300 copies per cell)<A href="http://parts.igem.org/Part:pSB1C3?title=Part:pSB1C3" class="external autonumber" rel="nofollow">[9]</A></LI><LI>assume 200 copies per cell
</LI><LI>∴ concentration of the gene per cell = N x 200 x 1.66x10<SUP>-6</SUP>mM, where N = number of base pairs
</LI><LI>∴ concentration of the gene PudA (N = 1644) in the volume of an E.coli cell is = 0.55mM
</LI><LI>Transcription rate in E.coli = 80bp/s<A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external autonumber" rel="nofollow">[10]</A> = 80 x 1.66x10<SUP>-6</SUP>mM/s = 80 x 1.66x10<SUP>-6</SUP> x 60mM/min = 7.97x10<SUP>-3</SUP>mM/min
</LI><LI>∴ Rate of mRNA_pudA production under the control of pBAD = 7.97x10<SUP>-3</SUP> ÷ 0.25 = <B>0.015/min</B></LI><B>2.Protein production rate of PudA, k<SUB>2</SUB></B><UL><LI>Average molecular weight(Mw) of an amino acid(aa)= 110g/mol<A href="http://www.genscript.com/conversion.html" class="external autonumber" rel="nofollow">[11]</A><A href="http://www.promega.com/~/media/Files/Resources/Technical%20References/Amino%20Acid%20Abbreviations%20and%20Molecular%20Weights.pdf" class="external autonumber" rel="nofollow">[12]</A></LI><LI>Average mass of an amino acid = 110g/mol x 1.66x10<SUP>-24</SUP>=1.83x10<SUP>-22</SUP>g/L
</LI><LI>∴Mass concentration of one aa in the volume of an E.coli =  = 1.83x10<SUP>-6</SUP>g/L
</LI><LI>∴Molar concentration of one aa =  = 1.66x10<SUP>-5</SUP>mM
</LI></UL><LI>Translation rate = 20aa/s = (20 x 1.66x10<SUP>-5</SUP> x 60)mM/min = 0.020mM/min
</LI><LI>PudA comprises of 548aa<A href="http://www.sciencedirect.com/science/article/pii/S0964830598000663" class="external autonumber" rel="nofollow">[13]</A></LI><LI>∴concentration of PudA's aa in the volume of an E.coli = 1.66x10<SUP>-5</SUP>mM x 548 = 9.10x10<SUP>-3</SUP>mM
</LI><LI>∴ Rate of protein production = 0.020 ÷ 4.25x10<SUP>-3</SUP> = <B>2.2/min</B></LI></P><H4><SPAN class="mw-headline" id="PUR_degradation_module">PUR degradation module</SPAN></H4><P>The reaction equation of the PUR degradation is:</P><P>[Polyurethane]+[PudA]= 4 [ethylene glycol] + 4 [polyisocyanate] + [PudA]
</P><P><B>Assumptions:</B></P><P>We assumed 1 mole of polyurethane dispersion can produce 4 moles of ethylene glycol. </P><P>The molecular weight of a single polyurethane monomer is 470 g/mol whereas the molecular weight of the polyurethane dispersion is around 2000 g/mol.<A href="http://www.polyurethanes.basf.de/pu/solutions/en/content/group/Arbeitsgebiete_und_Produkte/Grundprodukte/Lupraphen-Produktuebersicht_Gewicht" class="external autonumber" rel="nofollow">[14]</A>Therefore, the short-chain polyurethane consists approximately 4 monomers. 4 molecules of ethylene glycol will be produced by degrading one chain of polymer.</P><P>We also assumed a simple Michaelis-Menten mechanism for PudA</P><P>&quot;[PudA]&quot;is the concentration of the PudA enyme whereas &quot;[Polyurethane]&quot; is the concentration of the polyurethane. The kinetic parameters are defined as below:</P><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH><TH>Species</TH></TR><TR><TH>Km</TH><TD>Michaelis constant</TD><TD>51.5</TD><TD>mM</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830598000663" class="external autonumber" rel="nofollow">[15]</A></TD><TD>Comamonasacidovorans</TD></TR><TR><TH>Kcat</TH><TD>Turnover number</TD><TD>141.75</TD><TD>1/min</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830598000663" class="external autonumber" rel="nofollow">[16]</A></TD><TD>Comamonasacidovorans</TD></TR></TBODY></TABLE><P><B>Assumptions:</B></P><P>1. The kinetic parameters of PudA are not from the strain E.coli K12 MG1665 we are using. So the assumption here is that the kinetic property of pudA in E.coli is close to that in Comamonasacidovorans</P><P>2. Considerations of temperature change and PH change are not included in the model. Therefore, the assumption is that they remain constant during the degradation </P><H4><SPAN class="mw-headline" id="Secretion_module">Secretion module</SPAN></H4><P>The efficiency of <B>Secretion</B> is assumed to be 90% secretion over 2 hours.<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1251600/" class="external autonumber" rel="nofollow">[17]</A> The rate of secretion in the model is therefore:</P><P>rate of secretion = 0.9[concentration of PudA]/120 (mM/mins) </P><H4><SPAN class="mw-headline" id="Simulation_Results_and_Industrial_Implementation">Simulation Results and Industrial Implementation</SPAN></H4><P>The simulation result we worked out is especially related to the industrial degradation SRF which we aim to achieve a high degradation efficiency. According to the <A href="https://2013.igem.org/Team:Imperial_College/Industrial_Implementation" class="external text" rel="nofollow">data</A> we estimated for the percentage Polyurethane in SRF, we have about 5% Polyurethane contained in the waste. Therefore, in the model, we assumed 50g of polyurethane in 1kg of SRF based on the estimation. If we put 1kg of waste into a 5L bioreactor, the approximate concentration of polyurethane would be 5mM. Here is the simulation result: </P><P>The modelling result shows that we can efficiently degrade Polyurethane in SRF even in industrial scale. 5mM of polyurethane dispersion can be efficiently degraded in 15 mins. The final concentration of ethylene glycol we produced would be 20 mM that is under the toxicity limitation, this proves that our bio-reactor can maintain the efficiency without the toxicity effects. </P><P>As for the implementation of MAPLE system, the polyurethane degradation enzymes in our system are very efficient. Therefore, we need an efficient filter which removing ethylene glycol from the system. </P><H4>PUR degradation model:Download</H4></DIV></DIV><DIV id="CollapsiblePanelMod2" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H3>2.0 P(3HB) synthesis model      <FONT size="1">▼</FONT><DIV class="CollapsiblePanelContent"><P>This section is about modelling the plastic-synthesising <I>E.coli</I> developed in our project.  The plastic that this module synthesises is the bioplastic poly-3-hydroxybutyrate, also known as P(3HB).  The content below shows the process of building, designing and optimising the model.  Also, the considerations, assumptions and limitations will be discussed.
</P><P align="justify"> 
The model was built and simulated in Simbiology, a Matlab package designed for modelling biological systems.  In our model we have a compartment (labelled &quot;<I>E.coli</I>_1&quot;) that represents our engineered cell.  The compartment contains the reactions and species that interact with the polymer synthesis pathway.
</P><P>Here is our model for P(3HB) synthesis. It is part of module 1, this model uses glucose from waste as the input. Here we have the phaCAB operon, which convertsthe metabolite acetyl-coA into poly-3-hydroxybutyrate. We integrated our synthetic pathway into the metabolic pathway. This allowed us to quantitatively predict the P(3HB) production over time. More details about the metabolic model can be found in <A href="https://2013.igem.org/Team:Imperial_College/BioPlastic_Recycling:_PHB" class="external text" rel="nofollow">Module 2</A>.  </P><P>Firstly, consider the model concerning the synthetic pathway. This model does not take into account the existence of the metabolic pathway present within the bacteria. The synthetic pathway is mainly used for a sensitivity analysis by assuming minimum interference to the cell metabolism. The pathway is rearranged for clarity. </P><P align="justify">
The pathways on the right half of the cell are genetic expressions of the 3 enzymes involved in the plastic synthesis pathway.  The pathways on the left represent the actual P(3HB) production pathway.
</P><P align="justify">
As can be seen in the Key (above), the yellow, solid circle represents a reaction object and for each object parameter the following needs to be specified for the simulation to work:
</P><UL><LI>Rate equation or kinetic law (e.g. mass action, Michaelis-Menten etc.)
</LI><LI>Parameters in the rate equation
</LI><LI>Species involved in the reaction
</LI><LI>Value and units for each parameter
</LI></UL><P align="justify">
Simbiology will then use an ODE solver (ode15 or sundial) to solve these ODEs and give a plot of the specified output(s) (concentration level of a species over time, for instance).
</P><UL><LI>Note that 3HB deg, Acetoacetate deg, AcetoacetylCoA deg and 3HB-CoA deg are degradation rates (all = 0.035/min, please see assumptions below) of the corresponding species formed along the P(3HB) synthesis pathway.
</LI></UL></DIV></H3><H4>2.1 Ordinary Differential Equations      <FONT size="1">▼</FONT></H4><H5><SPAN class="mw-headline" id="Genetic_regulations_and_expressions">Genetic regulations and expressions</SPAN></H5><H6><SPAN class="mw-headline" id="BDH2_.283-hydroxybutyrate_dehydrogenase.29">BDH2 (3-hydroxybutyrate dehydrogenase)</SPAN></H6><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Schematic of the gene expression of BDH2 as seen in the model diagram</DIV></DIV></DIV></DIV><H6><SPAN class="mw-headline" id="PhaB_.28Acetoacetyl-CoA_reductase.29and_PhaC_.28P.283HB.29_synthase.29">PhaB (Acetoacetyl-CoA reductase)and PhaC (P(3HB) synthase)</SPAN></H6><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Schematic of the gene expression of PhaB and PhaC as seen in the model diagram</DIV></DIV></DIV></DIV></DIV></DIV><DIV id="CollapsiblePanelMod4" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H5>Values, sources and assumptions      <FONT size="1">▼</FONT></H5></DIV><DIV class="CollapsiblePanelContent"><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH><TH>Assumptions/Notes</TH></TR><TR><TH>β</TH><TD>maximum rate of transcription</TD><TD>0.032</TD><TD>mM/min</TD><TD>Please see derivation 1 below.</TD><TD>Please see derivation 1 below.</TD></TR><TR><TH>n</TH><TD>Hill coefficient</TD><TD>2.0</TD><TD>dimensionless</TD><TD><A href="http://parts.igem.org/Part:pSB1C3?title=Part:pSB1C3" class="external autonumber" rel="nofollow">[18]</A></TD><TD>For pBAD strong.  Taken from the parts registry page.  Rounded to 2.0 from 2.26 as Simbiology wouldn't allow a non-integer value for such parameter.</TD></TR><TR><TH>K</TH><TD>Activation coefficient</TD><TD>0.0031</TD><TD>mM</TD><TD><A href="http://parts.igem.org/Part:BBa_K206000:Characterization" class="external autonumber" rel="nofollow">[19]</A></TD><TD>For pBAD strong.  Taking the &quot;switch point&quot; (from the corresponding parts registry page) as the activation coefficient.</TD></TR><TR><TH>d<SUB>mRNA</SUB></TH><TD>mRNA degradation rate</TD><TD>0.035</TD><TD>1/min</TD><TD><A href="http://jb.asm.org/content/189/23/8746.full" class="external autonumber" rel="nofollow">[20]</A></TD><TD>There is no active degradation pathway and that dilution is the dominant way by which it degrades.  Rate = ln2/doubling time, where doubling time of strain MG1655 = 20min.  Assuming steady-state growth in LB broth as presented in paper.  rate = ln2/20 = 0.035/min</TD></TR><TR><TH>d<SUB>protein</SUB></TH><TD>Protein degradation rate</TD><TD>0.035</TD><TD>1/min</TD><TD><A href="http://jb.asm.org/content/189/23/8746.full" class="external autonumber" rel="nofollow">[21]</A></TD><TD>There is no active degradation pathway and that dilution is the dominant way by which it degrades.  Rate = ln2/doubling time, where doubling time of strain MG1655 = 20min.  Assuming steady-state growth in LB broth as presented in paper.  rate = ln2/20 = 0.035/min</TD></TR><TR><TH>k<SUB>2</SUB></TH><TD>Protein production rate (BDH2)</TD><TD>4.7</TD><TD>1/min</TD><TD>Please see derivation 2 below.</TD><TD>Please see derivation 2 below.</TD></TR><TR><TH>k<SUB>3</SUB></TH><TD>Protein production rate (PhaCB)</TD><TD>0.58</TD><TD>mM/min</TD><TD>Please see derivation 3 below.</TD><TD>Please see derivation 3 below.</TD></TR><TR><TH>[Arabinose]</TH><TD>Concentration of arabinose</TD><TD>Initial: 0.008</TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[mRNA]</TH><TD>Concentration of mRNA</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[BDH2]</TH><TD>Concentration of BDH2</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[PhaB]</TH><TD>Concentration of PhaB</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[PhaC]</TH><TD>Concentration of PhaC</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="CollapsiblePanelMod5" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H5>Derivations      <FONT size="1">▼</FONT></H5></DIV><DIV class="CollapsiblePanelContent"><B>1.Derivation of the maximal expression rate,β</B><UL><LI>Average molecular weight (Mw) of a base pair = 660g/mol<A href="http://www.geneinfinity.org/sp/sp_dnaprop.html" class="external autonumber" rel="nofollow">[22]</A><A href="http://www.lifetechnologies.com/uk/en/home/references/ambion-tech-support/rna-tools-and-calculators/dna-and-rna-molecular-weights-and-conversions.html" class="external autonumber" rel="nofollow">[23]</A></LI><LI>Average mass of a base pair = 660g/mol x 1.66x10<SUP>-24</SUP> = 1.1x10<SUP>-21</SUP>g
</LI><LI>Volume of an <I>E.coli</I> cell = 1µm<SUP>3</SUP><A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external autonumber" rel="nofollow">[24]</A> = 1x10<SUP>-15</SUP>L
</LI><LI>∴Mass concentration = </LI><LI>∴Molar concentration of 1 base pair in the volume of <I>E.coli</I> =  = 1.66x10<SUP>-6</SUP> mM
</LI></UL><LI>BioBrick assembly plasmid pSB1C3 is a high copy number plasmid (100-300 copies per cell)<A href="http://parts.igem.org/Part:pSB1C3?title=Part:pSB1C3" class="external autonumber" rel="nofollow">[25]</A></LI><LI>assume 200 copies per cell
</LI><LI>∴ concentration of the gene per cell = N x 200 x 1.66x10<SUP>-6</SUP>mM, where N = number of base pairs
</LI><LI>∴ concentration of the gene BDH2 (N = 768) in the volume of an <I>E.coli</I> cell is = 0.25mM
</LI><LI>Transcription rate in <I>E.coli</I>= 80bp/s<A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external autonumber" rel="nofollow">[26]</A> = 80 x 1.66x10<SUP>-6</SUP>mM/s = 80 x 1.66x10<SUP>-6</SUP> x 60mM/min = 7.97x10<SUP>-3</SUP>mM/min
</LI><LI>∴ Rate of mRNA_BDH2 production under the control of pBAD = 7.97x10<SUP>-3</SUP> ÷ 0.25 = <B>0.032/min</B></LI><P><B>2.Protein production rate of BDH2, k<SUB>2</SUB></B></P><UL><LI>Average molecular weight(Mw) of an amino acid(aa)= 110g/mol<A href="http://www.genscript.com/conversion.html" class="external autonumber" rel="nofollow">[27]</A><A href="http://www.promega.com/~/media/Files/Resources/Technical%20References/Amino%20Acid%20Abbreviations%20and%20Molecular%20Weights.pdf" class="external autonumber" rel="nofollow">[28]</A></LI><LI>Average mass of an amino acid = 110g/mol x 1.66x10<SUP>-24</SUP>=1.83x10<SUP>-22</SUP>g/L
</LI><LI>∴Mass concentration of one aa in the volume of an <I>E.coli</I> =  = 1.83x10<SUP>-6</SUP>g/L
</LI><LI>∴Molar concentration of one aa =  = 1.66x10<SUP>-5</SUP>mM
</LI></UL><LI>Translation rate = 20aa/s = (20 x 1.66x10<SUP>-5</SUP> x 60)mM/min = 0.020mM/min
</LI><LI>BDH2 comprises of 256aa<A href="http://www.uniprot.org/uniprot/Q2PEN2&amp;format=html" class="external autonumber" rel="nofollow">[29]</A></LI><LI>∴concentration of BDH2's aa in the volume of an <I>E.coli</I>= 1.66x10<SUP>-5</SUP>mM x 256 = 4.25x10<SUP>-3</SUP>mM
</LI><LI>∴ Rate of protein production = 0.020 ÷ 4.25x10<SUP>-3</SUP> = <B>4.7/min</B></LI><P><B>3.Protein production rate for PhaB and PhaC, k<SUB>3</SUB></B></P><UL><LI>Relative promoter strengths: J23104 = 1.3RPU, J23101 = 1.0RPU.<A href="http://sb6.biobricks.org/poster/automated-bioparts-characterisation-for-synthetic-biology/" class="external autonumber" rel="nofollow">[30]</A></LI><LI>∴ 104 is 1.3x stronger than 101.
</LI></UL><LI> In absolute units: take GFP synthesis rate (molecules per min per cell) and approximate that as a generic protein synthesis rate for the promoter.
</LI><LI> GFP synthesis rate of 101 = 2232 molecules per min per cell.<A href="http://sb6.biobricks.org/poster/automated-bioparts-characterisation-for-synthetic-biology/" class="external autonumber" rel="nofollow">[31]</A></LI><LI>∴ GFP synthesis rate of 104 = 1.3 x 2232 = 2902 molecules per min per cell
</LI><LI> Assume 1 molecule in an <I>E.coli</I> cell gives a concentration of 1nM.
</LI><LI> ∴ GFP synthesis rate of 104 = 2902 x 1nM = 2.9x10<SUP>-6</SUP>nM/min per cell
</LI><LI> Plasmid copy number assumed as 200 (as in derivation 1)
</LI><LI> ∴ GFP synthesis rate of 104 in our <I>E.coli</I> = 200 x 2.9x10<SUP>-6</SUP> = 0.00058nM/min = <B>0.58mM/min</B></LI></DIV></DIV><DIV id="CollapsiblePanelMod6" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>2.2 Enzyme kinetics      <FONT size="1">▼</FONT><DIV class="CollapsiblePanelContent"><UL><LI>Note:
</LI><LI>v = velocity of reaction
</LI><LI>Ac-CoA = acetyl-CoA
</LI><LI>AcAc-CoA = Acetoacetyl-CoA
</LI><LI>AcAc = acetoacetate
</LI></UL></DIV></H4><H4><SPAN class="mw-headline" id="BDH2_.283-hydroxybutyrate_dehydrogenase.29_2">BDH2 (3-hydroxybutyrate dehydrogenase)</SPAN></H4><P><B>Reaction R1:</B></P><P><B>Reaction R1r:</B></P><H5><SPAN class="mw-headline" id="Values.2C_sources_and_assumptions">Values, sources and assumptions</SPAN></H5><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Source</TH><TH>Assumptions/Notes</TH></TR><TR><TH>k<SUB>cat,BDH2_f</SUB></TH><TD>Turnover number of BDH2 in forward reaction</TD><TD>22200</TD><TD>1/min</TD><TD><A href="http://jb.oxfordjournals.org/content/early/2009/01/03/jb.mvn186.full.pdf" class="external autonumber" rel="nofollow">[32]</A></TD><TD>Sequences derived from <I>Pseudomonas fragi</I> but kinetic values from expression and purification of enzymes in <I>E. coli</I> XL1 Blue</TD></TR><TR><TH>k<SUB>cat,BDH2_r</SUB></TH><TD>Turnover number of BDH2 in reverse reaction</TD><TD>7200</TD><TD>1/min</TD><TD><A href="http://jb.oxfordjournals.org/content/early/2009/01/03/jb.mvn186.full.pdf" class="external autonumber" rel="nofollow">[33]</A></TD><TD>Sequences derived from <I>Pseudomonas fragi</I> but kinetic values from expression and purification of enzymes in <I>E. coli</I> XL1 Blue</TD></TR><TR><TH>K<SUB>i,NAD+</SUB></TH><TD>Inhibition constant of BDH2 with NAD+</TD><TD>2.5</TD><TD>mM</TD><TD><A href="http://www.hal.inserm.fr/hal-00374321/" class="external autonumber" rel="nofollow">[34]</A></TD><TD>Organism Tetrahymena pyriformis</TD></TR><TR><TH>K<SUB>i,NADH</SUB></TH><TD>Inhibition constant of BDH2 with NADH</TD><TD>1.1</TD><TD>mM</TD><TD><A href="http://www.hal.inserm.fr/hal-00374321/" class="external autonumber" rel="nofollow">[35]</A></TD><TD>Organism Tetrahymena pyriformis</TD></TR><TR><TH>K<SUB>m,AcAc</SUB></TH><TD>Michaelis constant for AcAc</TD><TD>0.37</TD><TD>mM</TD><TD><A href="http://jb.oxfordjournals.org/content/early/2009/01/03/jb.mvn186.full.pdf" class="external autonumber" rel="nofollow">[36]</A></TD><TD>Values in paper reference</TD></TR><TR><TH>K<SUB>m,NAD+</SUB></TH><TD>Michaelis constant for NAD+</TD><TD>0.24</TD><TD>mM</TD><TD><A href="http://jb.oxfordjournals.org/content/early/2009/01/03/jb.mvn186.full.pdf" class="external autonumber" rel="nofollow">[37]</A></TD><TD>Values in paper reference</TD></TR><TR><TH>K<SUB>m,NADH</SUB></TH><TD>Michaelis constant for NADH</TD><TD>0.010</TD><TD>mM</TD><TD><A href="http://jb.oxfordjournals.org/content/early/2009/01/03/jb.mvn186.full.pdf" class="external autonumber" rel="nofollow">[38]</A></TD><TD>Values in paper reference</TD></TR><TR><TH>K<SUB>m,3HB</SUB></TH><TD>Michaelis constant for 3HB</TD><TD>0.80</TD><TD>mM</TD><TD><A href="http://jb.oxfordjournals.org/content/early/2009/01/03/jb.mvn186.full.pdf" class="external autonumber" rel="nofollow">[39]</A></TD><TD>source organism <I>Pseudomonas fragi</I></TD></TR><TR><TH>[BDH2]</TH><TD>Concentration of BDH2</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[NAD+]</TH><TD>Concentration of NAD+</TD><TD>initial: 1.6x10<SUP>-13</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[3HB]</TH><TD>Intracellular concentration of 3HB</TD><TD>6.4x10<SUP>-14</SUP></TD><TD>mM</TD><TD>-</TD><TD>External concentration of 3HB assumed to be 0.01g.  This is shared by 1.5x10<SUP>12</SUP> cells.  Therefore, one cell has 0.01/1.5x10<SUP>12</SUP> = 6.7x10<SUP>-15</SUP> g.  To convert it into mol/L: mass of 3HB/molar mass of 3HB = 6.7x10<SUP>-15</SUP>/104.1 = 6.4x10<SUP>-17</SUP> M = 6.4x10<SUP>-14</SUP> mM</TD></TR><TR><TH>[AcAc]</TH><TD>Concentration of AcAc</TD><TD>initial: 1.0x10<SUP>-13</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[NADH]</TH><TD>Concentration of NADH</TD><TD>initial: 2.5x10<SUP>-14</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="atoAD_.28Acetyl-CoA:acetoacetyl-CoA_transferase_.28.CE.B1_and_.CE.B2_subunits.29.29">atoAD (Acetyl-CoA:acetoacetyl-CoA transferase (α and β subunits))</SPAN></H4><P><B>Reaction R2:</B></P><P><B>Reaction R2r:</B></P><H5><SPAN class="mw-headline" id="Values.2C_sources_and_assumptions_2">Values, sources and assumptions</SPAN></H5><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH><TH>Assumptions/Notes</TH></TR><TR><TH>V<SUB>max,atoAD_f</SUB></TH><TD>Maximum rate of atoAD in forward reaction</TD><TD>0.00244</TD><TD>mM/min</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/000398617590003X" class="external autonumber" rel="nofollow">[40]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>V<SUB>max,atoAD_r</SUB></TH><TD>Maximum rate of atoAD in reverse reaction</TD><TD>0.0108</TD><TD>mM/min</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/000398617590003X" class="external autonumber" rel="nofollow">[41]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>K<SUB>m,AcAc</SUB></TH><TD>Michaelis constant for AcAc</TD><TD>1.86</TD><TD>mM</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/000398617590003X" class="external autonumber" rel="nofollow">[42]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>K<SUB>m,Ac-CoA</SUB></TH><TD>Michaelis constant for Ac-CoA</TD><TD>0.26</TD><TD>mM</TD><TD><A href="http://www.brenda-enzymes.org/php/result_flat.php4?ecno=2.8.3.8" class="external autonumber" rel="nofollow">[43]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>K<SUB>m,Acetate</SUB></TH><TD>Michaelis constant for Acetate</TD><TD>53.1</TD><TD>mM</TD><TD><A href="http://aem.asm.org/content/73/24/7814.full.pdf" class="external autonumber" rel="nofollow">[44]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>K<SUB>m,AcAc-CoA</SUB></TH><TD>Michaelis constant for AcAc-CoA</TD><TD>0.035</TD><TD>mM</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/000398617590003X" class="external autonumber" rel="nofollow">[45]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>[AcAc]</TH><TD>Concentration of AcAc</TD><TD>initial: 1.0x10<SUP>-13</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[Acetate]</TH><TD>Concentration of Acetate</TD><TD>initial: 1.0x10<SUP>-13</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[AcAc-CoA]</TH><TD>Concentration of AcAc-CoA</TD><TD>initial: 1.0x10<SUP>-13</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[Ac-CoA]</TH><TD>Concentration of Ac-CoA</TD><TD>initial: 1.0x10<SUP>-14</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="phaB_.28Acetoacetyl-CoA_reductase.29">phaB (Acetoacetyl-CoA reductase)</SPAN></H4><P><B>Reaction R3:</B></P><P><B>Reaction R3r:</B></P><H5><SPAN class="mw-headline" id="Values.2C_sources_and_assumptions_3">Values, sources and assumptions</SPAN></H5><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH><TH>Assumptions/Notes</TH></TR><TR><TH>k<SUB>cat,PhaB_f</SUB></TH><TD>Turnover number of PhaB in forward reaction</TD><TD>6120</TD><TD>1/min</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/23913421" class="external autonumber" rel="nofollow">[46]</A></TD><TD>Gene in Ralstonia eutropha was engineered to put in E.coli and then purified</TD></TR><TR><TH>k<SUB>cat,PhaB_r</SUB></TH><TD>Turnover number of PhaB in reverse reaction</TD><TD>3600</TD><TD>1/min</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/3286259" class="external autonumber" rel="nofollow">[47]</A></TD><TD>originally in Zoogloea ramigera, expressed in E.coli and then purified</TD></TR><TR><TH>K<SUB>m,NADPH</SUB></TH><TD>Michaelis constant for NADPH</TD><TD>0.15</TD><TD>mM</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/23913421" class="external autonumber" rel="nofollow">[48]</A></TD><TD>Gene in Ralstonia eutropha was engineered to put in E.coli and then purified</TD></TR><TR><TH>K<SUB>m,AcAc-CoA</SUB></TH><TD>Michaelis constant for AcAc-CoA</TD><TD>0.0057</TD><TD>mM</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/23913421" class="external autonumber" rel="nofollow">[49]</A></TD><TD>Gene in Ralstonia eutropha was engineered to put in E.coli and then purified</TD></TR><TR><TH>K<SUB>m,NADP+</SUB></TH><TD>Michaelis constant for NADP+</TD><TD>0.0060</TD><TD>mM</TD><TD><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.1997.tb12740.x/pdf" class="external autonumber" rel="nofollow">[50]</A></TD><TD>purified from Methylobacterium extorquens</TD></TR><TR><TH>K<SUB>m,3HB-CoA</SUB></TH><TD>Michaelis constant for 3HB-CoA</TD><TD>0.026</TD><TD>mM</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/3286259" class="external autonumber" rel="nofollow">[51]</A></TD><TD>originally in Zoogloea ramigera, expressed in E.coli and then purified</TD></TR><TR><TH>[PhaB]</TH><TD>Concentration of PhaB</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[AcAc-CoA]</TH><TD>Concentration of AcAc-CoA</TD><TD>initial: 1.0x10<SUP>-13</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[NADPH]</TH><TD>Concentration of NADPH</TD><TD>initial:3.8x10<SUP>-14</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[NADP+]</TH><TD>Concentration of NADP+</TD><TD>initial:1.5x10<SUP>-13</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[3HB-CoA]</TH><TD>Concentration of 3HB-CoA</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="phaC_.28P.283HB.29_synthase.29">phaC (P(3HB) synthase)</SPAN></H4><P><B>Reaction R4:</B></P><H5><SPAN class="mw-headline" id="Values.2C_sources_and_assumptions_4">Values, sources and assumptions</SPAN></H5><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH><TH>Assumptions/Notes</TH></TR><TR><TH>k<SUB>cat,PhaC</SUB></TH><TD>Turnover number of PhaC</TD><TD>1680</TD><TD>1/min</TD><TD><A href="http://ac.els-cdn.com/S0003986101925226/1-s2.0-S0003986101925226-" class="external autonumber" rel="nofollow">[52]</A></TD><TD>-</TD></TR><TR><TH>K<SUB>m,PhaC</SUB></TH><TD>Michaelis constant of PhaC with 3HB-CoA</TD><TD>0.14</TD><TD>mM</TD><TD><A href="http://www.brenda-enzymes.org/php/result_flat.php4?ecno=2.3.1.B5" class="external autonumber" rel="nofollow">[53]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>[PhaC]</TH><TD>Concentration of PhaC</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[3HB-CoA]</TH><TD>Concentration of 3HB-CoA</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="atoB_.28acetyl-CoA_acetyltransferase.29">atoB (acetyl-CoA acetyltransferase)</SPAN></H4><P><B>Reaction R5:</B></P><P><B>Reaction R5r:</B></P><H5><SPAN class="mw-headline" id="Values.2C_sources_and_assumptions_5">Values, sources and assumptions</SPAN></H5><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH><TH>Assumptions/Notes</TH></TR><TR><TH>V<SUB>max,atoB_f</SUB></TH><TD>Maximum rate of atoB in forward reaction</TD><TD>3.8x10<SUP>-5</SUP></TD><TD>mM/min</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/9904" class="external autonumber" rel="nofollow">[54]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>V<SUB>max,atoB_r</SUB></TH><TD>Maximum rate of atoB in reverse reaction</TD><TD>8.5x10<SUP>-4</SUP></TD><TD>mM/min</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/9904" class="external autonumber" rel="nofollow">[55]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>K<SUB>m,atoB</SUB></TH><TD>Michaelis constant of atoB with Ac-CoA</TD><TD>0.47</TD><TD>mM</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/0003986176901521" class="external autonumber" rel="nofollow">[56]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>K<SUB>m,AcAc-CoA</SUB></TH><TD>Michaelis constant for AcAc-CoA</TD><TD>0.1</TD><TD>mM</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/0003986176901521" class="external autonumber" rel="nofollow">[57]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>K<SUB>m,CoA</SUB></TH><TD>Michaelis constant for CoA</TD><TD>0.25</TD><TD>mM</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/0003986176901521" class="external autonumber" rel="nofollow">[58]</A></TD><TD>Enzyme is naturally expressed in <I>E.coli</I>: assume the enzyme is stable during the time course of the simulation.</TD></TR><TR><TH>[Ac-CoA]</TH><TD>Concentration of Ac-CoA</TD><TD>initial: 1.0x10<SUP>-13</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[AcAc-CoA]</TH><TD>Concentration of AcAc-CoA</TD><TD>initial: 1.0x10<SUP>-13</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR><TR><TH>[CoA]</TH><TD>Concentration of CoA</TD><TD>initial: 5.5x10<SUP>-13</SUP></TD><TD>mM</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD><TD>see section 2.5 &quot;Initial concentrations of metabolites&quot;</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="CollapsiblePanelMod7" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H5>Reaction kinetics and mechanisms      <FONT size="1">▼</FONT></H5></DIV><DIV class="CollapsiblePanelContent"><P>List of enzymes and the corresponding mechanisms by which they work in the synthesis pathway</P><UL><LI>BDH2: Ordered-sequential bi-bi mechanism<A href="http://www.brenda-enzymes.org/php/result_flat.php4?ecno=1.1.1.30" class="external autonumber" rel="nofollow">[59]</A></LI><LI>atoAD: Ping-pong bi-bi mechanism<A href="http://www.sciencedirect.com/science/article/pii/000398617590003X" class="external autonumber" rel="nofollow">[60]</A></LI><LI>atoB: Ping-pong bi-bi mechanism<A href="http://www.sciencedirect.com/science/article/pii/000398617590003X" class="external autonumber" rel="nofollow">[61]</A></LI><LI>PhaB: Ordered-sequential bi-bi mechanism<A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[62]</A></LI><LI>PhaC: Michaelis-Menten<A href="http://connection.ebscohost.com/c/articles/85133798/new-insights-activation-substrate-recognition-polyhydroxyalkanoate-synthase-from-ralstonia-eutropha" class="external autonumber" rel="nofollow">[63]</A></LI></UL></DIV></DIV><DIV id="CollapsiblePanelMod8" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>2.3 Simulation results      <FONT size="1">▼</FONT><DIV class="CollapsiblePanelContent"><P align="justify">
Using the values presented in the tables above, the deterministic model was simulated to see how much P(3HB) could be formed inside the cell.  However, with the single cell model, it would be difficult to reflect how much of the plastic could actually be formed on a macroscopic scale.  Therefore, modifications to the simulation algorithm were needed to give scaled-up simulation results.  
</P><P align="justify">
In the scaled-up regime, though, new limitations were encountered.  For example, growth, cell division and the concentration gradient that each cell encounters and how much can be taken up by the cells.  Given the data available, what could be done was to assume that the bacteria were in a bioreactor, with constant nutrient and oxygen supply, and that they were in the phase of stationary growth (please see assumptions and explanations under the graphs below).
</P><P align="justify">
Here, three graphs are presented to show P(3HB)formation over time.  The first two graphs are (i)single-cell model results, concentration in mM and (ii) single-cell model results, concentration in g/L.  The first graph was the original graph produced from Simbiology as all concentrations were expressed in mM in the model.  However, to gain a more intuitive sense of how much could be produced g/L was used.  The third graph is the scaled-up simulation result.   
</P><P><B>(i) please see assumptions and explanations below</B></P><P><B>(ii) please see assumptions and explanations below</B></P><P><B>(iii) please see assumptions and explanations below</B></P><P><B>Graphs (i) and (ii): assumptions and explanations</B></P><UL><LI>To convert from mM to g/L: 
</LI><LI>Molecular weight(Mw) of P(3HB) = 1.8x10<SUP>12</SUP>g/mol<A href="http://webpages.charter.net/hvalentin/PDF-files/18.pdf" class="external autonumber" rel="nofollow">[64]</A></LI></UL><P align="justify">(Please note that this value was taken from a paper about the &quot;Production of poly(3-hydroxybutyrate-co-4-hydroxybutyrate) in recombinant Escherichia coli grown on glucose&quot; instead of the production of poly(3-hydroxybutyrate).  However, this is the closest we managed to get as far as species, substrate and type of plastic are concerned.</P><P align="justify">Also, it was noticed that the molecular weight would change depending on the cultivation time (as conditions such as growth and pH can also affect it) and it was hard to find a paper that contains the exact same conditions and the range of cultivation times for our simulations.  Furthermore, in reality there wouldn't be a single value of molecular weight obtained, but a range of values (known as polydispersity).<A href="http://aem.asm.org/content/78/9/3177.full" class="external autonumber" rel="nofollow">[65]</A>  Therefore, it was assumed that the molecular weight would remain constant during the time course of the simulation and that the value should be considered as an average.)</P><UL><LI>∴P(3HB)concentration in mM x 10<SUP>-3</SUP> x 1.8x10<SUP>12</SUP>g/mol = concentration in g/L
</LI></UL><P><B>Graph (iii): assumptions and explanations</B></P><UL><LI> Assume 1.5x10<SUP>12</SUP>cells in a 1L broth, calculated from value obtained from BioNumbers.  A range was given: 1-2x10<SUP>9</SUP> cells/ml, so 1.5x10<SUP>9</SUP> was taken. <A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?s=n&amp;id=104831&amp;ver=5" class="external autonumber" rel="nofollow">[66]</A></LI><LI>Conditions and considerations accompanying this value are: 
</LI><LI>growth medium: LB broth
</LI><LI>Cultivation temperature: 37°C
</LI><LI>wild-type E. coli K-12 strain MG1655 (NB: though not exactly our engineered one, it is the correct strain)
</LI></UL><LI>Assume stationary growth phase.
</LI><LI>Assume that the amount of P(3HB) accumulated during the time course of simulation doesn't exceed the cell's capacity or ability to contain it.
</LI><LI>Same metabolic considerations as above (single-cell results) 
</LI><LI>Simbiology single-cell simulation results multiplied by the number of cells stated above to give the red curve.  
</LI></DIV></H4><H4>2.4 Model-guided design and optimisation      <FONT size="1">▼</FONT></H4><H4><SPAN class="mw-headline" id="Sensitivity_analysis:_species_concentrations">Sensitivity analysis: species concentrations</SPAN></H4><P><B>What is sensitivity analysis and why do it?</B></P><P align="justify"> 
In order to look at how we could increase the production of P(3HB) we decided to run a sensitivity analysis (also in Simbiology) to identify what species in the model P(3HB) is sensitive to, given the specific conditions with which we set the model.  In other words, we calculated the time-dependent sensitivity of P(3HB) with respect to the initial conditions of other species (species formed along the synthesis pathway and enzymes involved).<A href="http://www.mathworks.co.uk/help/simbio/ug/calculating-sensitivities.html#brbumlr|ref" class="external autonumber" rel="nofollow">[67]</A></P><P align="justify"> 
For a species x, whose concentration depends on time, it can be expressed as x(t).  To calculate the sensitivity of x(t) with respect to another species y(t), the sensitivity with normalisation relative to the numerator x(t) can be calculated as:
</P><P align="justify"> 
Therefore, the underlying algorithm for this sensitivity analysis of P(3HB) is:
</P><H4><SPAN class="mw-headline" id="Sensitivity_analysis:_enzyme_concentrations">Sensitivity analysis: enzyme concentrations </SPAN></H4><P>The sensitivity analysis of the enzymes involved in the synthetic pathway is carried out, which determine the most sensitive enzyme in the pathway</P><P>We plot a time integral of the sensitivity analysis which has the algorithm as: </P><P>The senstivity result is:</P><P>According to the simulation result, we found that the PHB production is highly sensitive to the concentration of PhaB and PhaC. In contrast, the system is not sensitive to the concentration of atoAD and atoB. In theory, increase the concentration of any of the enzyme will increase the flux of the system. However, for the strain E.coli K12 MG1655, both atoAD (acetoacetate:acetoacetyl-coA transferase) and atoB （acetoacetyl-coA thiolase) favour the reverse reactions, it becomes the rate limiting step in our pathway. Acetoacetyl-coA intends to be converted to acetoacetate instead of 3HB-coA. <A href="http://www.sciencedirect.com/science/article/pii/0003986177904969" class="external autonumber" rel="nofollow">[68]</A>Therefore, the concentration of PhaB become really critical which push the system to flow in the forward direction.  </P><H4><SPAN class="mw-headline" id="Scan_with_different_levels_of_PhaB">Scan with different levels of PhaB</SPAN></H4><P align="justify"> The plot below is to show that increasing PhaB concentration within the engineered organism would have a positive effect on the concentration of the P(3HB) synthesised.  As so many parameters within the model were taken from different sources, it would be difficult to give a simulation plot that is accurate quantitatively. It was decided to show the single cell simulation for this because the scans were carried out up to the doubling time of 20mins, hence avoided the issue of scaling-up which would have otherwise subjected the simulation to more errors.  Therefore, this plot should be interpreted from a qualitative perspective and the trend should be observed.  
</P><H4><SPAN class="mw-headline" id="Difference_between_promoter_expressions_after_optimisation">Difference between promoter expressions after optimisation</SPAN></H4><P>In order to choose the strongest promoter that is available to increase the gene expression. We ran a parameter scan of enzyme expression rates of a range of Anderson's constitutive promoters. The simulations shows the PHB production under different Anderson's promoters. The promoters we tested are:</P><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH width="150">promoter</TH><TH>biobrick</TH><TH>GFP synthesis rate (mM/min)</TH></TR><TR><TH>J23100</TH><TD><A href="http://parts.igem.org/Part:BBa_J23100" class="external autonumber" rel="nofollow">[69]</A></TD><TD>0.41 </TD></TR><TR><TH>J23101</TH><TD><A href="http://parts.igem.org/Part:BBa_J23101" class="external autonumber" rel="nofollow">[70]</A></TD><TD>0.45</TD></TR><TR><TH>J23104</TH><TD><A href="http://parts.igem.org/Part:BBa_J23104" class="external autonumber" rel="nofollow">[71]</A></TD><TD>0.58</TD></TR><TR><TH>J23118</TH><TD><A href="http://parts.igem.org/Part:BBa_J23118" class="external autonumber" rel="nofollow">[72]</A></TD><TD>0.31</TD></TR></TBODY></TABLE><P>The derivations of the GFP synthesis rate are:</P><P>*Relative promoter strengths: J23104 = 1.3RPU, J23101 = 1.0RPU, J23100 = 0.92RPU, J23118 = 0.76RPU<A href="http://sb6.biobricks.org/poster/automated-bioparts-characterisation-for-synthetic-biology/" class="external autonumber" rel="nofollow">[73]</A><UL><LI> In absolute units: take GFP synthesis rate (molecules per min per cell) and approximate that as a generic protein synthesis rate for the promoter.
</LI><LI> GFP synthesis rate of 101 = 2232 molecules per min per cell.<A href="http://sb6.biobricks.org/poster/automated-bioparts-characterisation-for-synthetic-biology/" class="external autonumber" rel="nofollow">[74]</A></LI></UL><LI> Assume 1 molecule in an <I>E.coli</I> cell gives a concentration of 1nM.
</LI><LI> ∴ GFP synthesis rate of 101 = 2232 x 1nM = 2.2x10<SUP>-6</SUP>nM/min per cell
</LI><LI> Plasmid copy number assumed as 200 (as in derivation 1)
</LI><LI> ∴ GFP synthesis rate of 101 in our <I>E.coli</I> = 200 x 2.2x10<SUP>-6</SUP> = 0.00045nM/min = <B>0.45mM/min</B></LI><LI> ∴ The GFP synthesis rates of other promoters can be calculated by multiplying the RPU value with the GFP synthesis rate of J23101 promoter.</LI></P><P>The simulation result:</P><P>Although there is just a small increase the gene expression rate, J23104 promoter achieved a much higher PHB production than other constitutive promoters. Therefore, we decided to use J23104 promoter for our improved biobrick <A href="http://parts.igem.org/Part:BBa_K1149051" class="external text" rel="nofollow">BBa_K1149051</A>.</P><P><B>Here is our experimental results that show we have 11 times more PHB after the change of promoter.</B></P><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Comparison of P3HB production </B><B>  (left)</B> 1.5ml tube, natural phaCAB (BBa_K934001)  <B>(right)</B> 5ml tube, phaCAB expressed from the hybrid promoter, (BBa_K1149051).</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H4>P(3HB) synthesis model:Download</H4><H4>Additional script:Download</H4></DIV></DIV><DIV id="CollapsiblePanelMod10" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>2.5 Metabolic considerations      <FONT size="1">▼</FONT><DIV class="CollapsiblePanelContent"><P align="justify">In Module 1: Resourceful Plastic, glucose from the degradation is used as the input for the system that produces P(3HB). The P(3HB) synthetic pathway hence needs to be adapted for this purpose.  To do so, 3HB monomer would no longer be the source input to the engineered system.  So, BDH2 should be removed from the pathway. Acetyl-coA would then become the key resource to produce P(3HB). Also, a metabolic model was needed to show the production of Acetyl-coA from glucose. Furthermore, our synthetic pathway involves several key metabolites from the pathway such as NADPH, NADP+, NADH and NAD+. The initial concentration of those metabolites can be determined from the metabolic model by assuming they are in their steady states. More features and results of the metabolic model can be found in Module 2.   </P></DIV></H4><H4><SPAN class="mw-headline" id="Initial_concentrations_of_metabolites">Initial concentrations of metabolites</SPAN></H4><P align="justify">The model presented in Section 2.0 contains a synthesis pathway that interact metabolic species involved in so many other metabolic reactions elsewhere in the cell.  It was, therefore, difficult to determine how much of these metabolites would be left available for our plastic synthesis pathway to utilise.  So, a simple metabolic model of the cell (more detail in next paragraph) was needed to simulate and show insight into this problem.  We decided to use the steady state levels of these metabolites (without the engineered pathway) and plug the values into the P(3HB) synthesis model in Section 2.0 as the initial concentrations.
</P><P align="justify">The metabolic model involves two pathways - glycolysis pathway and the TCA cycle, where glucose is used as the sole carbon source. The reason is that glucose is directly taken by the glycolysis pathway and there are interactions between Acetyl-coA and TCA cycle. This Simbiology model is based on the one built by Angela Dixon in <B><I>Predictive Mathematical Model for Polyhydroxybutyrate Synthesis in Escherichia coli</I></B>.<A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[75]</A>. All reactions and pathways are based on ODEs and kinetics data presented in the original model. It can dynamically predict the concentration of all metabolites involved over a defined simulation time course. </P><P>The metabolic model for determining steady state concentrations is shown as below:</P><P><B>Keys:</B></P><P>The initial concentrations of all metabolites and kinetic data are referenced in the paper.<A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[76]</A> The simulation of the metabolites is:</P><P> Table of initial and steady-state concentrations of the metabolites:</P><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Substrates</TH><TH>Initial concentration</TH><TH>Steady state concentration </TH><TH>Units</TH><TH>Sources</TH></TR><TR><TH>NADH</TH><TD>200</TD><TD>250</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[77]</A></TD></TR><TR><TH>NAD+</TH><TD>1200</TD><TD>1600</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[78]</A></TD></TR><TR><TH>Acetyl-coA</TH><TD>1000</TD><TD>100</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[79]</A></TD></TR><TR><TH>Coenzyme A</TH><TD>250</TD><TD>5500</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[80]</A></TD></TR><TR><TH>NADPH</TH><TD>200</TD><TD>380</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[81]</A></TD></TR><TR><TH>NADP+</TH><TD>1200</TD><TD>1100</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[82]</A></TD></TR></TBODY></TABLE><P align="justify">&lt;p align=&quot;justify&quot;&gt;
It should also be noted that acetate, acetoacetate and acetoacetyl-CoA cannot be predicted by the metabolic model.  Therefore, their initial concentrations are rough estimates (value similar to other metabolites) such that a reasonable P(3HB) output can be produced.</P></DIV></DIV></DIV><DIV class="TabbedPanelsContent"><H3><SPAN class="mw-headline" id="Assembling_the_optimised_bioplastic_producing_operon">Assembling the optimised bioplastic producing operon</SPAN></H3><P><B>We made two new improved constructs by modifying the native operon:</B></P><P>The modellers in the drylab predicted that an increased level of pha B enzyme will increase the amount of PHB produced and therefore we decided to change the native <I>Ralstonia eutropha</I> promoter and RBS to a strong constitutive <I>E.coli</I> promoter (J23104) and RBS (0034) in front of the operon. We have succeeded with the swap and built the new biobrick <A href="http://parts.igem.org/Part:BBa_K1149052" class="external text" rel="nofollow">BBa_K1149052</A> which we refer to as constitutive phaCAB. In parallel, we have also assembled a different configuration of transcription initiation region: the &quot;hybrid&quot;. Surprisingly, our results show that our &quot;hybrid&quot; phaCAB construct <A href="http://parts.igem.org/Part:BBa_K1149051" class="external text" rel="nofollow">BBa_K1149051</A> significantly increased bioplastic production compared to native pha CAB. Please read on to discover more. &lt;/p&gt;</P><P>We tested the promoter and RBS by cloning them in front of amilCP blue chromoprotein gene  (<A href="http://parts.igem.org/Part:BBa_K1149020" class="external text" rel="nofollow">BBa_K1149020</A>). As you can see from the plate below, this expresses a bright blue protein, which became an intrinsic part of our <A href="https://2013.igem.org/Team:Imperial_College/Communication_work#" class="external text" rel="nofollow">BioNouveau</A> palette. 
</P><P>In order to change the promoter, we designed a forward primer that binds to the beginning of the first gene in the operon and contains a Xba restriction site: Pha_Fw (cgcttctagagatggctactgggaaaggagccg). We used pha_Fw and the suffix primer G1005 primer pain to amplify the insert out of the backbone.
The new promoter and RBS were J23104 and B0034 from the biobrick <A href="http://parts.igem.org/Part:BBa_K608002" class="external text" rel="nofollow">BBa_K608002</A> We digested the pSB1C3 backbone with this biobrick with SpeI and PstI.</P><P> Summary diagram of construction steps: </P><P>We performed DpnI digest on the gel purified PCR product (insert) in order to get rid of any contamination from the original plasmid. We did Alkaline Phosphatase treatment of the backbone to remove terminal phosphates in order to decrease self-ligation of plasmid. 
We transformed the ligation into NEB5 cells and we grew overnight cultures from single cell colonies. The next day, we purified the plasmids and digested 200 ng of them with Xba and PstI to screen for the right clones:</P><P>Colony #8 had the expected 4.2kB insert and therefore we sent this plasmid for sequencing. After having the sequenced it, we submitted the new part to registry as <A href="http://parts.igem.org/Part:BBa_K1149051" class="external text" rel="nofollow">BBa_K1149051</A>. This construct has double P+RBS since the native P+RBS stayed there in between the new P+RBS and the operon.  We have identified a clone (from colony #6) that contains the new constitutive promoter only and submitted it as <A href="http://parts.igem.org/Part:BBa_K1149052" class="external text" rel="nofollow">BBa_K1149052</A> biobrick.
We are testing both the hybrid and normal constitutive constructs for PHB bioplastic production. </P><H5><SPAN class="mw-headline" id="Partial_sequence_of_the_constitutive_phaCAB_construct_.28BBa_K1149052.29:"> Partial sequence of the constitutive phaCAB construct (<A href="http://parts.igem.org/Part:BBa_K1149052" class="external text" rel="nofollow">BBa_K1149052</A>): </SPAN></H5><H5><SPAN class="mw-headline" id="Partial_sequence_of_the_hybrid_phaCAB_construct_.28BBa_K1149051.29:"> Partial sequence of the hybrid phaCAB construct (<A href="http://parts.igem.org/Part:BBa_K1149051" class="external text" rel="nofollow">BBa_K1149051</A>): </SPAN></H5></DIV><DIV class="TabbedPanelsContent"><DIV id="CollapsiblePanel1" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Our chassis (MG1655) survives in mixed waste      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P><B>These assays were designed to test whether our chassis, <I>E.coli</I> (MG1655) could grow directly with waste over a long period of time. </B></P><H4><SPAN class="mw-headline" id="Waste_media">Waste media</SPAN></H4><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>LB-waste assay(A)</B> Waste media <B>(B)</B><I>E.coli</I> containing mCherry stress biosensor (BBa_K639003) were grown in mixed waste <B>(A)</B> over 3 days, then streaked in a qualitative assay to check for growth. <B>(C)</B> mCherry stress biosensor (BBa_K639003) transformed <I>E.coli</I> were streaked again after 7 days growth in SRF. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>PBS-waste assay(A)</B> waste media made up in PBS (phosphate buffered saline). <B>(B)</B><I>E. coli</I> expressing mCherry stress biosensor (BBa_K639003) grown in waste media <B>(A)</B> over 3 days, then streaked onto an antibiotic containing plate to qualitatively assess whether the <I>E.coli</I> had survived. <B>(C)</B> Streaked again after 6 days growth in SRF. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><B>Conclusion: MG1655 <I>E.coli</I> are viable and grow on mixed waste alone. Therefore we have established that our chassis could survive in a mixed waste bio-reactor context, which is validation of our concept to industrially implement our system.</B></P><H4><SPAN class="mw-headline" id="Waste_conditioned_media">Waste conditioned media</SPAN></H4><P>These assays were designed to test whether our chassis, <I>E.coli</I> MG1655 strain could grow with waste conditioned media (WCM) over a period of 24-48 hours.  Waste conditioned media is a filter sterilised version of the waste media and was designed for several reasons; Firstly we were unsure whether mixed waste would be toxic to <I>E.coli</I> and hence a less concentrated version may be more suitable and secondly large chunks of waste would prevent accurate OD600 measurements and therefore we decided to filter out the largest chunks.
</P><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD colspan="2"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Photograph of waste conditioned media cultures</B> mCherry stress biosensor (BBa_K639003) transformed MG1655 were grown with <A href="https://2013.igem.org/Team:Imperial_College/Protocols" class="external text" rel="nofollow">LB-WCM</A> at 37ºC, with shaking for 48 hours. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Growth assay in waste conditioned media (WCM)</B>.<I>E.coli</I> MG1655 strain transformed with <A href="http://parts.igem.org/Part:BBa_K639003" class="external text" rel="nofollow">mCherry stress biosensor.</A> were grown with LB-based waste conditioned media at 37ºC, with shaking. Error bars represent S.E.M., n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Growth assay in waste conditioned media (WCM).</B><I>E.coli</I> MG1655 strain transformed with either empty vector control (EV) or mCherry stress biosensor (BBa_K639003) were grown in WCM for 5 hours, with shaking at 37ºC. Error bars are SEM, n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><B> Conclusion: MG1655 transformed with either empty vector (EV) control or mCherry stress biosensor (BBa_K639003) vector are viable and can grow in waste conditioned media. Therefore waste conditioned media is an appropriate and novel experimental media with which to characterise biobricks within a mixed waste/landfill context. These data are also characterisation of an existing biobrick (BBa_K639003) </B></P></DIV></DIV><DIV id="CollapsiblePanel2" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Mixed waste degradation: Polyurethane (PUR)      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H4><SPAN class="mw-headline" id="PUR_Esterase_enzyme_activity"> PUR Esterase enzyme activity </SPAN></H4><H4><SPAN class="mw-headline" id="Cell_lysate_assay">Cell lysate assay</SPAN></H4><P align="justify">Since the PUR Esterases were not secreted, initially the cells were lysed to obtain crude cell extracts in order to test whether the enzymes are active. The <A href="https://2013.igem.org/Team:Imperial_College/Western_Blots" class="external text" rel="nofollow">Western Blot results</A> showed that the constructs EstCS2 <A href="http://parts.igem.org/Part:BBa_K1149002" class="external text" rel="nofollow">BBa_K1149002</A>, PueB <A href="http://parts.igem.org/Part:BBa_K1149004" class="external text" rel="nofollow">BBa_K1149004</A> and PulA <A href="http://parts.igem.org/Part:BBa_K1149006" class="external text" rel="nofollow">BBa_K1149006</A> were being expressed. The cultures expressing these three constructs were grown, lysed by sonication and utilised in a colourimetric assay with the substrate analog para-Nitrophenyl butyrate. The data shows that PUR Esterase EstCS2 <A href="http://parts.igem.org/Part:BBa_K1149002" class="external text" rel="nofollow">BBa_K1149002</A> is definitely active.</P><P align="justifiy"> para-Nitrophenyl butyrate (p-NP) is commonly used to indicate an enzyme’s esterase activity. The enzyme cleaves the ester bond and releases the 4-nitrophenol (4-NP), thus causing a colour change from colourless to yellow and an increased absorbance at wavelength 405 nm.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:809px;"><DIV class="thumbcaption"><B>para-Nitrophenyl butyrate is cleaved by Esterases to release 4-Nitrophenol. This is accompanied by an increase in absorbance at 405 nm. Figure adapted from <A href="https://2012.igem.org/Team:TU_Darmstadt/Protocols/pNP_Assay#Enzyme" class="external text" rel="nofollow">TU Darmstadt 2012 iGEM</A></B></DIV></DIV></DIV></DIV><P align="justify">The assay was run in the <A href="http://www.eppendorf.com/int/index.php?sitemap=2.1&amp;action=products&amp;contentid=1&amp;catalognode=87236" class="external text" rel="nofollow">Eppendorf BioSpectrometer</A> was used to automatically read the absorbance of the reaction mixture every 30 seconds. The concentration of 4-Nitrophenol produced from the reaction was calculated using the Beer-Lambert Law; the extinction coefficient of 4-NP is 18,000 M-1 cm-1 at 405 nm. </P><P align="justify">The results below show the concentration of 4-NP produced by the three different PUR esterases and compare them to the Empty Vector and Substrate alone as negative controls.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>The increase in absorbance that accompanies the cleavage of para-Nitrophenyl butyrate by PUR Esterase EstCS2. Figure by Imperial College London iGEM 2013</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:482px;"><DIV class="thumbcaption"><B>The concentration of 4-Nitrophenol released by PUR Esterase EstCS2 activity. Empty Vector and Substrate alone were used as negative controls. Figure by Imperial College London iGEM 2013</B></DIV></DIV></DIV><P>The above graphs clearly show that PUR Esterase EstCS2 is active and cleaving p-NP. The recorded concentrations of 4-NP, in the presence of this PUR Esterase, are much greater than with the Empty Vector or the Substrate alone. 
<B>Is PUR Esterase EstCS2 active? Yes!</B></P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>The enzyme activity of PUR Esterase PueB. The graph shows the concentration of the 4-Nitrophenol released during 10 minutes incubation of para-Nitrophenyl butyrate with PueB and Empty Vector, as well as the p-NP substrate by itself. Figure made by Imperial College London 2013 iGEM</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>The enzyme activity of PUR Esterase PulA. The graph shows the concentration of the 4-Nitrophenol released during 10 minutes incubation of para-Nitrophenyl butyrate with PulA and Empty Vector, as well as the p-NP substrate by itself.Figure made by imperial College London 2013 iGEM</B></DIV></DIV></DIV><P class="clear">The enzymes expressed by both PueB and PulA do not appear to have esterase activity for this substrate. There is a tiny increase in 4-NP concentration for PueB, PulA, Empty Vector and Substrate alone. This probably indicates that the substrate is slowly degrading by itself. Other constituents of the cell lysates are likely to be causing a slight increase in p-NP degradation, as PueB, PulA and Empty Vector show higher concentrations of 4-NP than the Substrate alone.</P><P align="justify"><B>Conclusion: PUR Esterase EstCS2 is active!</B></P></DIV></DIV><DIV id="CollapsiblePanel3" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Mixed waste degradation: Toxicity      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H4><SPAN class="mw-headline" id="Ethylene_glycol">Ethylene glycol </SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B> Ethylene glycol in LB with Stress Response cells.</B> MG1655 were grown in ethylene glycol, a byproduct of polyurethane degradation. Cells were grown in 0mM, 100mM or 200mM Ethylene Glycol. At 37°C, the concentrations of ethylene glycol used do not affect growth, however at 30°C, the increasing concentration results in halved growth. A two-tailed t-test addressing the null hypothesis, temperature does not affect growth with ethylene glycol shows that the null hypothesis must be rejected as p = 0.001. Data points show final time point after 6h growth for each concentration. Growth was at 37°C and 30°C with shaking over 6h. Error bars are SEM, n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></DIV><P>Reduced growth at 30°C likely due to decreased efficiency of MG1655 ethylene glycol break down enzymes. These enzymes (see UC Davis 2012) are endogenously expressed and detoxify Ethylene Glycol. 
</P></DIV></DIV><DIV id="CollapsiblePanel4" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Bioplastic production      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P align="justify"><I>E.coli</I> MG1655 containing the PhaCAB construct Part:BBa_K934001<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K934001" class="external autonumber" rel="nofollow">[83]</A> accumulates the bioplastic P3HB inside its cells. P3HB can be visualised through staining with Nile Red, which fluoresces strongly in a hydrophobic environment, such as when it binds to the P3HB. However staining will be much lower in cells where P3HB is not present. To see whether P3HB was being produced we grew <I>E.coli</I> containing the phaCAB operon under the native promoter from <I>Ralstonia eutropha</I>. This produced the results we hoped for as seen in the image below. </P><P>The P3HB production model indicated that upregulating atoB would lead to an increase in bioplastic production. We changed the promoter before the phaCAB operon to the strong promoter J23104 and a hybrid promoter we created.</P><P align="justify">The Nile Red plates also provide a preliminary way in which to investigate the effect of changing the promoter to a stronger version. The darker staining of the streaks of the constitutive and hybrid promoter containing constructs suggests the production of higher levels of P3HB in these strains. </P><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:698px;"><DIV class="thumbcaption"><B><I>E.coli</I> transformed with the phaCAB operon(native, hybrid and constitutive are different promoters expressing the operon) fluoresce when grown on LB-Agar plates with Nile Red stain. The P3HB produced  within them is stained. Those without the operon do not(EV- transformed with empty pSB1C3). When the promoter is changed from the native form to the hybrid or constitutive J23104 then the fluorescence is more intense, indicating increased P3HB production.</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P align="justify"> We imaged the cells using the fluorescent microscope to qualitatively understand how much P3HB is produced per cell. As indicated by other authors(1), we saw that P3HB is found in localised granules within the bacteria. Empty vector containing cells show little fluorescence. In those that do, the staining is qualitatively different than in the phaCAB containing strains. Nile Red is also used as a lipid stain and this is indicated by its staining of the membranes of EV cells. In contrast, in phaCAB containing cells, staining is restricted to points within the cells.</P><P align="justify">Each fluorescent image was created by exciting the stain at 530nm for 100ms. Despite this standard treatment of samples the intensity of fluorescence produced by the cells containing the hybrid promoter is greater than from the constitutive ones. This could indicate a difference in the strength of the promoters which lead to phaCAB expression and therefore the amount of P3HB produced. The images also suggest that not every single <I>E.coli</I> is stained. This may be due to the conditions in which the cells were grown or may be dependent upon the stage in the lifecycle of the <I>E.coli</I>.</P><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Fluorescent microscope images of <I>E.coli</I> stained with Nile Red. 1 -Empty vector, 2 - constitutive promoter phaCAB, 3 - hybrid promoter phaCAB. Images labelled a are fluorescent images excited at 530nm ovelaid on the bright field images shown in the b set of images. Images by Imperial College iGEM Team</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>We extracted the P3HB using a simple technique which maintains the chain length of the P3HB while releasing it from the cells. The amount of plastic extracted from the hybrid promoter-phaCAB(BBa_K1149051<A href="http://parts.igem.org/Part:BBa_K1149051" class="external autonumber" rel="nofollow">[84]</A>) was much larger compared to the native promoter-phaCAB. All other conditions were kept the same.</P><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:407px;"><DIV class="thumbcaption"><B>A comparison between the amount of plastic produced by bacteria containing an empty plasmid(EV), native promoter-phaCAB and hybrid promoter-phaCAB after plastic extraction. The EV does not produce P3HB while the hybrid produces much more than the native promoter.</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="CollapsiblePanel5" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Increasing Bioplastic Production      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P>We converted the data into the format with which industry compares P3HB production. These are mass of plastic per dry biomass and mass of plastic per litre of culture media. The strain containing the hybrid promoter had a ten fold higher ratio of plastic to biomass than that produced by the original part BBa_K934001</P><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:701px;"><DIV class="thumbcaption"><B>A summary of the improved production of P3HB by our hybrid promoter-phaCAB construct(BBa_K1149051) over the native promoter-phaCAB.</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Comparing the amount of plastic produced by the two constructs is visually striking.</P><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Comparison of P3HB production (left) 1.5ml tube, 90mg produced by native promoter-phaCAB (BBa_K934001), (right) 5ml tube, hybrid promoter-phaCAB produced 2.05g P3HB, (BBa_K1149051).</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="CollapsiblePanel6" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Bioplastic from mixed waste      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H2><SPAN class="mw-headline" id="P.283HB.29_From_Mixed_Waste">P(3HB) From Mixed Waste</SPAN></H2><P>E. coli (MG1655) transformed with native phaCAB (BBa_K934001) were grown with 600ml waste media (12g mixed waste in 600ml M9M- Minimal Media) overnight (16h) at 37oC with shaking. Prior to centrifugation large mixed waste chunks were filtered out. P(HB) was purified as described in <A href="https://2013.igem.org/Team:Imperial_College/Protocols" class="external text" rel="nofollow">Protocols.</A></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>P(3HB) purification:</B> The diagram summaries the process of P(3HB) purification from E. coli (MG1655) transformed with native phaCAB (BBa_K934001)</DIV></DIV></DIV></DIV><P class="clear">See <A href="https://www.youtube.com/watch?v=4msf6EMNIrY" class="external text" rel="nofollow">here</A> a video of the process.</P><H2><SPAN class="mw-headline" id="3HB_Assay:_Confirming_production_of_3HB">3HB Assay: Confirming production of 3HB</SPAN></H2><P>We have adapted a medical kit for 3HB detection for chemically analysing our samples. This provided a simple but accurate mean to measure the 3HB liberated from the polymer after phaZ1 depolymerase enzyme treatment. The monomer 3HB indicated the presence of P(3HB) in the material we purified from cells grown on waste as a sole carbon source. Please see the protocol <A href="https://2013.igem.org/Team:Imperial_College/Protocols#3HB_assay" class="external text" rel="nofollow">here</A> and our data and calculations <A href="https://2013.igem.org/Team:Imperial_College/3HB_assay" class="external text" rel="nofollow">here</A>.
</P><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>The chemical analysis of the produced bioplastic. The samples break break down to 3HB monomers after treatment with our PhaZ1 enzyme (BBa_K1149010). We synthesised P(3HB) using our improved Biobrick part (hybrid promoter phaCAB, BBa_K1149051). Our engineered bioplastic producing <I>E.coli</I> synthesised P(3HB) directly from waste. (Imperial iGEM data <A href="https://2013.igem.org/Team:Imperial_College/3HB_assay" class="external text" rel="nofollow">here</A>.)</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><FONT size="2.7"><B>The bioplastic produced by our bacteria contains PHB and can be broken down into monomers by PhaZ1 enzyme (expressed by our bacteria)Imperial iGEM data.</B></FONT></DIV></DIV></DIV></DIV><P>
We repeated the previously described experiment with <I>E.coli</I> cells containing all three P(3HB) production operons, grown on waste media. We also also tested the material left after the extraction protocol from cells containing the Empty vector only, grown on waste media. The yellow colour indicates presence of 3HB. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><FONT size="2.7"><B>Production of PHB from waste confirmed. Imperial iGEM data.</B></FONT></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="CollapsiblePanel7" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Extras      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H2><SPAN class="mw-headline" id="Growth_assays_with_different_experimental_media">Growth assays with different experimental media</SPAN></H2><P>In additional to standard LB and minimal media, several novel experimental media were developed in order to characterise Biobricks within a mixed waste/landfill setting. These media were characterised through an examination of pH and through an array of growth assays with the project chassis, <I>E.coli</I> MG1655 strain.
</P><TABLE class="wikitable" style="margin: 1em auto 1em auto;"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Media characterisation</B>. <I>E.coli</I> strain MG1655 were transformed with a control plasmid and grown in different experimental media over a period of 5 hours. LB media, minimal media (M9M), supplemented minimal media (M9S), as described <A href="https://2013.igem.org/Team:Imperial_College/Protocols#M9_minimal_and_supplemented_media" class="external text" rel="nofollow">here</A> or waste conditioned media (WCM), which is made from sterile filtrated mixed waste, see <A href="https://2013.igem.org/Team:Imperial_College/Protocols#Waste_Conditioned_Media_.28WCM.29" class="external text" rel="nofollow">here</A>. OD600 measured, error bars are S.E.M., n=4. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>pH of experimental media.</B> pH measurements of experimental media were made both before and after several experiments. The time periods refer to the duration MG1655 transformed <I>E.coli</I>were cultured in the media before a pH measurement was made. Figure made by Imperial College London 2013 iGEM.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><B>Conclusion: <I>E.coli</I> strain are viable and grow in all of our experimental medias. We have established a novel media that is optimised for characterisation of biobricks within a mixed waste/landfill context.</B></P></DIV></DIV><DIV id="CollapsiblePanel8" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>Protocols      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><H4>Waste Conditioned Media (WCM)</H4><P>We added 1g [http://en.wikipedia.org/wiki/Refuse-derived_fuel SRF] (Solid Recovered Fuel) /50 mL LB and then autoclaved the mixture. Once autoclaved, large waste chunks were removed through filter sterilisation (0.2 µm filters) before being used in growth assays as waste conditioned media (WCM).

SRF refers to the refuse from recycling facilities that has no value currently and is incinerated to produce power at a cost to the recycling facility, it is composed of 30% plastics while the rest is cellulosic waste</P><P class="clear">See here how we make up waste conditioned media and purify plastic from it.</P><H4>Waste Growth Assay</H4><P>
Overnight (O/N) cultures of MG1655 transformed with (BBa_K639003) were diluted to OD 0.05 in either fresh LB or waste conditioned media and plated into 96 well plates (200 µl/well). OD600 was read at the indicated time points and media only (LB) was taken away as background signal. 

In addition to this a qualitative assay as performed. 9 mL mCherry bacteria were transferred into 300 mL of autoclaved SRF in LB, which were placed in a shaking incubator at 37°C. After 2 days 200 µL of this solution was plated out on chloramphenicol plates, this was then repeated a week later to gauge whether or not the bacteria were still alive.

To show that the MG1655 could grow on waste solely, they were grown in PBS (phosphate buffered saline) and autoclaved waste. In 100 mL PBS + waste, 1 mL MG1655 mCherry E.coli were added. They were then grown in a shaking incubator at 37°C and samples were plated after 2 days and 6 days.
</P></DIV></DIV><DIV id="CollapsiblePanel9" class="CollapsiblePanel"><DIV class="CollapsiblePanelTab" tabindex="0"><H4>MSDS      <FONT size="1">▼</FONT></H4></DIV><DIV class="CollapsiblePanelContent"><P align="justify"> Some of the reagents we used were slightly toxic to human. In order to reduce the risks of using toxic reagents, we always wear gloves and labcoats. For some toxic reagents, we performed in fume cupboard. Click on the reagents to see the MSDS.</P><P>1. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=W222305&amp;brand=ALDRICH&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Faldrich%2Fw222305%3Flang%3Den" class="external text" rel="nofollow">Tributyrin</A></P><P>2. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=458392&amp;brand=ALDRICH&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Faldrich%2F458392%3Flang%3Den" class="external text" rel="nofollow">Poly(diethylene glycol adipate)</A>.</P><P> 3. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=19123&amp;brand=SIGMA&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Fsigma%2F19123%3Flang%3Den" class="external text" rel="nofollow">Nile red</A>.</P><P> 4. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=GB&amp;brand=SIGMA&amp;productNumber=A3256&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/a3256?lang=en&amp;region=GB" class="external text" rel="nofollow">Arabinose</A>.</P><P> 5. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=N9876&amp;brand=SIGMA&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Fsigma%2Fn9876%3Flang%3Den" class="external text" rel="nofollow">4-Nitrophenyl butyrate</A>.</P><P> 6. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=270717&amp;brand=ALDRICH&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Faldrich%2F270717%3Flang%3Den" class="external text" rel="nofollow">Acetonitrile</A>.</P><P> 7. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=S8511&amp;brand=SIGMA&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Fsigma%2Fs8511%3Flang%3Den" class="external text" rel="nofollow">N-Succinyl-Ala-Ala-Pro-Leu p-nitroanilide</A>.</P><P> 8. <A href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=GB&amp;language=en&amp;productNumber=L3126&amp;brand=SIGMA&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Fsigma%2Fl3126%3Flang%3Den" class="external text" rel="nofollow">Lipase from porcine pancreas</A>.</P></DIV></DIV></DIV><DIV class="TabbedPanelsContent"><H2 id="future work">Future work</H2><P><B>We are changing the pelB secretion tag to OsmY-fusion partner for better secretion</B></P><P>As seen in the <A href="https://2013.igem.org/Team:Imperial_College/Western_Blots" class="external text" rel="nofollow">Western Blot</A> results, some of the enzymes designed for secretion were not secreted. These proteins may have been stuck in the periplasmic space. Thus we are optimising enzyme secretion by building a new construct with OsmY <A href="http://parts.igem.org/Part:BBa_K892008" class="external text" rel="nofollow">BBa_K892008</A> fusion for secretion. OsmY is naturally secreted from <I>E.coli</I> and will carry our PUR esterase enzyme with. Once in the culture medium, we think that the enzyme is going to be functional in the fusion form but we included a TEV site in between the domains that can be cleaved by TEV protease if needed.</P><P>The linker sequence we have designed between the domains is <B>T G S E N L Y F Q G S</B> (ACCGGCAGCGAGAACCTGTACTTCCAAGGCAGC) and includes the 7 amino acids of the TEV site as well as 4 other amino acids for flexibility. 
</P><P>This construct will be a substitute for <A href="http://parts.igem.org/Part:BBa_K1149002" class="external text" rel="nofollow">BBa_K1149002</A> which is the PUR esterase EstCS2 with the pelB seretion tag.
</P><P>UPDATE 28-10-13:
</P><P>Whilst further optimisation is required a western blot analysis of our new construct OsmY-ESTC2 indicated that our enzyme maybe secreting into the supernatant.
</P><P><B>We will test our Hybrid phaCAB within a bioreactor setup</B></P><P>We will inoculate a 15L bioreactor at Imperial College London to perform a pilot scale test of our plastic producing system.
</P></DIV><DIV id="supporters" class="clear"><P><B>Our Sponsors</B></P><A href="/File:Logo-bioline.gif" title="File:Logo-bioline.gif">125px</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Imperial_College/Waste_Degradation:_SRF">http://2013.igem.org/Team:Imperial_College/Waste_Degradation:_SRF</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College/Waste_Degradation:_SRF" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College/Waste_Degradation:_SRF" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College/Waste_Degradation:_SRF&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College/Waste_Degradation:_SRF&amp;oldid=361323" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>