<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_Modelling_PLAdeg"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College/Modelling_PLAdeg">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College/Modelling_PLAdeg&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College/Modelling_PLAdeg&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College/Modelling_PLAdeg&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College/Modelling_PLAdeg" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College/Modelling PLAdeg</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><UL class="navigation-menu" style=""><LI><A href="https://2013.igem.org/Team:Imperial_College">Project: Plasticity</A></LI><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Imperial_College/tour">At a glance</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Background">Waste Issue</A></LI><LI>M.A.P.L.E. Modules</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Waste_Degradation:_SRF">M1: Resource-full Waste</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/BioPlastic_Recycling:_PHB">M2: Plastic Fantastic</A></LI></UL><LI><A href="https://2013.igem.org/Team:Imperial_College/Applications">Bioplastic applications</A></LI><LI>Human Practices</LI><LI class="maple"><A href="https://2013.igem.org/Team:Imperial_College/MAPLE">M.A.P.L.E</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Industrial_Implementation">Industrial Implementation</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Managing Waste">Social Challenge</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Communication_work">Communicating our Project</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Project_Evolution">Project Evolution</A></LI><LI>Achievements</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Judging_Criteria">Judging Criteria</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/mainresults">Main Results</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Collaboration">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Biobricks">Biobricks</A></LI><LI>In the Lab</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/MSDS">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/labbook">Lab book</A></LI><LI><A href="#">Modelling Notes</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Modelling_Intro">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Modelling_beta-amyloid">Beta amyloid degradation</A></LI><LI><A href="#">Data</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Growth_Assays">Growth Assays</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Western Blots">Western Blot</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Enzyme Kinetics">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/PHB production">PHB production</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/3HB_assay">3HB assay</A></LI><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Imperial_College/Clearing_Zone_Assay">Clearing Assays</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Electron microscopy">Electron microscopy</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Protocols">Protocols</A></LI><LI>Team</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Meet_the_Team">Students</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Advisors">Advisors</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Attributions">Attributions</A></LI><DIV id="main-content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Polyhydroxybutyrate_.28P.283HB.29.29_Synthesis_Module"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Polyhydroxybutyrate (P(3HB)) Synthesis Module</SPAN></A></LI><LI class="toclevel-2"><A href="#Metabolic_modelling"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Metabolic modelling</SPAN></A></LI><LI class="toclevel-3"><A href="#Introduction_of_the_metabolic_model"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Introduction of the metabolic model</SPAN></A></LI><LI class="toclevel-3"><A href="#Methods"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-3"><A href="#ATP_analysis:"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">ATP analysis:</SPAN></A></LI><LI class="toclevel-3"><A href="#Large_scale_PHB_production_and_implementation_in_MAPLE"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Large scale PHB production and implementation in MAPLE</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Polyhydroxybutyrate_.28P.283HB.29.29_Synthesis_Module">Polyhydroxybutyrate (P(3HB)) Synthesis Module </SPAN></H1><H2><SPAN class="mw-headline" id="Metabolic_modelling">Metabolic modelling</SPAN></H2><H3><SPAN class="mw-headline" id="Introduction_of_the_metabolic_model">Introduction of the metabolic model</SPAN></H3><P align="justify">
The metabolic model predicts the interaction of our pathway with endogenous pathways. In synthetic biology, an ideal system is fully orthogonal. However, if the synthetic pathway uses some metabolites from the metabolism pathways, the metabolic fluxes (the rate of conversion between metabolites) will be affected. If the cell is overloaded by losing too much metabolites, it would either reject the synthetic pathway or burn out. As for the optimization of PHB production in our MAPLE system, metabolic analysis need to be carried out to estimate the effects on the cell metabolism from increases in the PHB production. </P><H3><SPAN class="mw-headline" id="Methods">Methods</SPAN></H3><P align="justify">We altered the metabolic model by Angela Dixon in her paper “Predictive Mathematical Model for Polyhydroxybutyrate Synthesis in Escherichia coli”. <A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[1]</A> The metabolic model is built in Simbiology, which is an extension in MATLAB especially for biological system analysis. The model consists two metabolic pathways: Glycolysis and TCA cycle. The reactions in the metabolic pathways are defined by ordinary differential equations and kinetic parameters. The model also contains a synthetic pathway that produce P(3HB) from acetyl-coA. The pathway consists three enzymes which are PhaA, PhaB and PhaC. 
The reasons we chose this model:</P><P>1. The model is based on ODEs instead of FBA (flux balance analysis) method, which is a common methods of metabolic analysis. However, FBA cannot determine the real time concentration of the metabolites whereas it can be done by models of ODEs.</P><P>2. Only single gene deletion or addition can be performed in FBA. More than that, the metabolic model here can perform modification on any single or multiple genes and reactions through Simbiology. </P><P>3. The FBA model is static, it cannot analyze the dynamic interactions between our synthetic pathways with the metabolic pathways. As both our P(3HB) synthesis model and the metabolic model are in the same platform. It was really easy to integrate our pathway into the metabolic model.</P><P><B>In our project, we have a different pathway:</B></P><P>1. We use 3HB as one of the input.</P><P>2. Existence of ato system in the E.coli MG1655 strain. </P><P>3. We removed phaA in our case because we use 3HB as the major source to produce P(3HB) to avoid too much acetyl-coA be taken from the metabolic pathways.</P><P align="justify">Therefore, we substitute the original pathway with our synthetic pathway. Moreover, we added our gene expression models for the enzymes into the metabolic model as well. In terms of the consistency of the model, we changed the units of our synthetic pathway into micro molar and seconds. </P><P>Here is the combination of our model with the metabolic model</P><P align="justify">We also carried out a parameter estimations of the initial concentrations of several substrates in our synthetic pathway. The reason is we have single-cell scaled magnitude for them originally (fM level), but there are much higher magnitudes in any of the species in the metabolic pathways (mM level). Therefore, we scaled up the pathway in order to maintain the consistency of the model. As for the metabolites which are involved in our pathway, we cloned the blocks of them from the metabolic pathway in Simbiology and deliberately put them into our pathway. That means our pathway dynamically interact with the metabolic pathways. </P><P>Here is the table for the updated initial concentrations for the species in our synthetic pathway:
All the kinetic data and initial concentrations of the metabolites in the metabolic pathways are referenced in the paper<A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[2]</A>: </P><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Substrates</TH><TH>Concentration</TH><TH>Units</TH><TH>Sources</TH><TH>Assumptions</TH></TR><TR><TH>3HB</TH><TD>5000</TD><TD>uM</TD><TD>-</TD><TD>The initial input to the system</TD></TR><TR><TH>Acetoacetate</TH><TD>1000</TD><TD>uM</TD><TD>-</TD><TD>The ato system in E.coli suggest a favourable conversion from acetoacetyl-coA to acetoacetate. Therefore, the concentration of acetoacetate is 10 times larger than acetoacetyl-coA<A href="http://www.sciencedirect.com/science/article/pii/0003986177904969" class="external autonumber" rel="nofollow">[3]</A></TD></TR><TR><TH>Acetoacetyl-coA</TH><TD>100</TD><TD>uM</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/1103741" class="external autonumber" rel="nofollow">[4]</A></TD><TD>The ato system in E.coli suggest a favourable conversion from acetoacetyl-coA to acetoacetate. <A href="http://www.sciencedirect.com/science/article/pii/0003986177904969" class="external autonumber" rel="nofollow">[5]</A></TD></TR><TR><TH>NADH</TH><TD>200</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[6]</A></TD><TD>-</TD></TR><TR><TH>NAD+</TH><TD>1200</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[7]</A></TD><TD>-</TD></TR><TR><TH>Acetyl-coA</TH><TD>5000</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[8]</A></TD><TD>-</TD></TR><TR><TH>Coenzyme A</TH><TD>250</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[9]</A></TD><TD>-</TD></TR><TR><TH>Acetate</TH><TD>40000</TD><TD>uM</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/1103741" class="external autonumber" rel="nofollow">[10]</A></TD><TD>Assay data from the literature which the ato system efficiency was measured</TD></TR><TR><TH>3HB-coA</TH><TD>0</TD><TD>uM</TD><TD>-</TD><TD>One of the intermediate product of the pathway</TD></TR><TR><TH>NADPH</TH><TD>200</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[11]</A></TD><TD>-</TD></TR><TR><TH>NADP+</TH><TD>1200</TD><TD>uM</TD><TD><A href="http://digitalcommons.usu.edu/engineering_datasets/1/" class="external autonumber" rel="nofollow">[12]</A></TD><TD>-</TD></TR><TR><TH>PHB</TH><TD>0</TD><TD>uM</TD><TD>-</TD><TD>The final product of the pathway</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="ATP_analysis:">ATP analysis:</SPAN></H3><P>In our optimization of P(3HB) production module, we discovered that increasing in the PhaB expression can efficiently increase the yield of P(3HB). It’s important to see whether increase in the PhaB expression would dramatically change the fluxes in the metabolic pathways or not. As ATP is the essential energy carriers in the cell, we carried a parameter scan for the variation in ATP concentrations levels under different concentration of PhaB.</P><P>There is an approximately 5.3% drop in the concentration of ATP when increasing the phaB concentration from 50uM to 200uM. It suggests that increasing the efficiency in our pathway will reduce the efficiency of energy transports in the cell. However, it’s still unsure about  the effect of 5.3% drop in ATP concentration on the cell growth.</P><H3><SPAN class="mw-headline" id="Large_scale_PHB_production_and_implementation_in_MAPLE">Large scale PHB production and implementation in MAPLE</SPAN></H3><P>In our MAPLE system, the bioplastic Polyhydroxybutyrate P(3HB) is produced from both 3HB monomers and glucose. 3HB is the degradation product from the wasted P(3HB）plastic and glucose is the product from the degradation of mixed waste in Module 1. Therefore, the overall PHB production rates need to be defined by considering different situations:</P><P>1. In module 1, we have glucose as the only source to produce PHB.</P><P>2. In module 2, we have both glucose and 3HB as resources. </P></DIV><DIV id="supporters" class="clear"><P><B>Our Sponsors</B></P><A href="/File:Logo-bioline.gif" title="File:Logo-bioline.gif">125px</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Imperial_College/Modelling_PLAdeg">http://2013.igem.org/Team:Imperial_College/Modelling_PLAdeg</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College/Modelling_PLAdeg" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College/Modelling_PLAdeg" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College/Modelling_PLAdeg&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College/Modelling_PLAdeg&amp;oldid=292913" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>