"
Page
Discussion
View source
History
teams
Log in
 
Team:Imperial College/Modelling PURdeg
From 2013.igem.org
Project: Plasticity
At a glance
Waste Issue
M.A.P.L.E. Modules
M1: Resource-full Waste
M2: Plastic Fantastic
Bioplastic applications
Human Practices
M.A.P.L.E
Industrial Implementation
Social Challenge
Communicating our Project
Project Evolution
Achievements
Judging Criteria
Main Results
Collaborations
Biobricks
In the Lab
Safety
Lab book
Modelling Notes
Introduction
Beta amyloid degradation
Data
Growth Assays
Western Blot
Enzyme Kinetics
PHB production
3HB assay
Clearing Assays
Electron microscopy
Protocols
Team
Students
Advisors
Sponsors
Attributions
Contents
1 Polyurethane (PUR) Degradation Module
1.1 Introduction
1.2 Design
1.2.1 Objective
1.2.2 The Model
1.2.3 Parameters and assumptions
1.2.3.1 Gene expression module of PudA
1.2.3.2 PUR degradation module
1.3 Simulations and Results
Polyurethane (PUR) Degradation Module
Introduction
Both efficiencies of Polyurethane (PUR) degradation and ethylene glycol production are important for the performance of MAPLE system. We built a mathematical and deterministic model that is based on MATLAB extension Simbiology for Polyurethane degradation. The model contains the kinetic property of degradation enzymes that is helpful for the design of assays. As we scaled up the initial concentrations of all substrates to meet the conditions for a bio-reactor, the model can provide preliminary simulations and predictions for the MAPLE system.
Design
Objective
Here are some specific objectives for the model to achieve:
1. The model should contain the gene expression model of the degradation enzymes because the enzyme concentration determines the rate of plastic degradation. In our case for PUR degradation, we used pBAD strong promoter K206000 for most enzymes. We built the gene expression model based on inducible pBAD promoter, which gene expression rate can be regulated by inducer concentration.
2. The model should show the efficiency of the enzyme secretion to the culture from the cells. It's also important because the enzyme concentration in the culture depends on it. Here we used pelB secretion tag for most enzymes in order to achieve a high efficiency.
3. The model basically predict how long will take to degrade a known concentration of soluble polyurethane. It is assumed that the enzyme in our assays has the same kinetic properties as the enzyme used in the literature. The model can suggest a suitable concentration of the plastic to use in order to get good results from the assays.
4. It is known that ethylene glycol is toxic to E.coli. However, it has no clear effect on the growth of our MG1655 strain when the concentration of ethylene glycol is below 200mM. Therefore, the model should suggest a safe range of PUR concentration to avoid a high concentration (>200mM) of ethylene glycol produced.
The Model
Polyurethane (PUR) degradation involves 5 different degradation enzymes:
enzyme
source organism
biobrick
reference
EstCS2
uncultured unknown bacterium (GU256649.1)
BBa_K1149002
Kang et.al 2011
pueA
Pseudomonas chlororaphis
BBa_K1149003
Stern et al., 2000
pueB
Pseudomonas chlororaphis
BBa_K1149004
Howard et al., 2001
pudA
Comamonas acidovorans
BBa_K1149005
Allen et al. 1999
pulA
Pseudomonas fluorescens
BBa_K1149006
Vega et al.,  1999
However, 4 of the 5 enzymes are not well characterised before, so we could't find enough kinetic data from the literature. The only well-characterised PUR degradation enzyme PudA is used in the model as an illustration of all PUR degradation enzymes. The model will be more complete when the kinetic data of the other enzymes are defined. The finished PUR degradation model is shown as below:
Key
There are two compartments which represents cells and the culture from left to right. The "cell" compartment contains the gene expression module whereas the "culture" compartment contains the degradation module. The "secretion" block that connects two compartments is the secretion module.
Parameters and assumptions
Gene expression module of PudA
Parameter
Description
Value
Units
Sources
Assumptions
β
maximum rate of transcription
0.032
mM/min
Please see derivation 1 below.
Please see derivation 1 below.
K
Activation coefficient
0.0031
mM
[1]
Taking the "switch point" as the activation coefficient
dmRNA
mRNA degradation rate
0.10
1/min
[2]
Taking the value of mRNA half-life in E.coli strain MG1655 as 6.8min.  rate = ln2/half-life = ln2/6.8 = 0.10
dprotein
Protein degradation rate
0.050
1/min
[3]
There is no active degradation pathway and that dilution is the dominant way by which the protein level decreases in a cell.  Rate = 1/doubling time, where doubling time = 20min.  Assuming steady-state growth in LB broth as presented in paper.
k2
Protein production rate (PudA)
4.7
1/min
Please see derivation 2 below.
Please see derivation 2 below.
[Arabinose]
Concentration of arabinose
Initial: 0.008
mM
[mRNA]
Concentration of mRNA
-
mM
-
-
[PudA]
Concentration of PudA
-
mM
-
-
1.Derivation of the maximal expression rate,β
Average molecular weight (Mw) of a base pair = 660g/mol [4] [5]
Average mass of a base pair = 660g/mol x 1.66x10-24 = 1.1x10-21g
Volume of an E.coli cell = 1µm3 [6] = 1x10-15L
∴Mass concentration =
∴Molar concentration of 1 base pair in the volume of E.coli =
= 1.66x10-6 mM
BioBrick assembly plasmid pSB1C3 is a high copy number plasmid (100-300 copies per cell) [7]
assume 200 copies per cell
∴ concentration of the gene per cell = N x 200 x 1.66x10-6mM, where N = number of base pairs
∴ concentration of the gene pudA (N = 768) in the volume of an E.coli cell is = 0.25mM
Transcription rate in E.coli = 80bp/s [8] = 80 x 1.66x10-6mM/s = 80 x 1.66x10-6 x 60mM/min = 7.97x10-3mM/min
∴ Rate of mRNA_pudA production under the control of pBAD = 7.97x10-3 ÷ 0.25 = 0.032/min
2.Protein production rate of pudA, k2
Average molecular weight(Mw) of an amino acid(aa)= 110g/mol [9] [10]
Average mass of an amino acid = 110g/mol x 1.66x10-24=1.83x10-22g/L
∴Mass concentration of one aa in the volume of an E.coli =
= 1.83x10-6g/L
∴Molar concentration of one aa =
= 1.66x10-5mM
Translation rate = 20aa/s = (20 x 1.66x10-5 x 60)mM/min = 0.020mM/min
PudA comprises of 256aa [11]
∴concentration of pudA's aa in the volume of an E.coli = 1.66x10-5mM x 256 = 4.25x10-3mM
∴ Rate of protein production = 0.020 ÷ 4.25x10-3 = 4.7/min
PUR degradation module
The reaction equation of the PUR degradation is:
[Polyurethane]+[PudA]= 5 [ethylene glycol] + 5 [polyisocyanate] + [PudA]
Assumptions:
We assumed 1 mole of polyurethane dispersion can produce 5 moles of ethylene glycol.
The molecular weight of a single polyurethane monomer is 470 g/mol whereas the molecular weight of the polyurethane dispersion is around 2000 g/mol. [12] Therefore, the short-chain polyurethane consists approximately 5 monomers. 5 molecules of ethylene glycol will be produced by degrading one chain of polymer.
We also assumed a simple Michaelis-Menten mechanism for PudA
The parameters for the kinetic equations are:
Parameter
Description
Value
Units
Sources
Km
Michaelis constant
51.5
mM
[13]
Kcat
Turnover number
141.75
1/min
[14]
The efficiency of Secretion is assumed to be 90% secretion over 2 hours. [15] The rate of secretion in the model is therefore:
rate of secretion = 0.9[concentration of PudA]/120 (mM/mins)
Simulations and Results
In terms of ethylene glycol toxicity, if we can't let the concentration of ethylene glycol over than 200mM, the maximum initial concentration of polyurethane dispersion is 40mM because one mole of polyurethane dispersion can produce 5 moles of ethylene glycol. (from the derivation above) If we put 40mM as the initial concentration in the model, the simulation result is:
40mM of polyurethane dispersion can be efficiently degraded in 20 mins. In case if we keep the concentration of the plastic below 40mM. The ethylene glycol won't affect the cell growth in our assays. 40mM is converted to 80g/L which suggests no more than 80 grams of polyurethane to be put in the system in order to avoid toxicity.
As for the implementation of MAPLE system, the polyurethane degradation enzymes in our system are very efficient. Therefore, we need an efficient filter which removing ethylene glycol from the system.
Our Sponsors
125px
Retrieved from " http://2013.igem.org/Team:Imperial_College/Modelling_PURdeg "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
