<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Imperial_College_Modelling_PURdeg"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Imperial_College/Modelling_PURdeg">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Imperial_College/Modelling_PURdeg&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College/Modelling_PURdeg&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Imperial_College/Modelling_PURdeg&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Imperial_College/Modelling_PURdeg" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Imperial College/Modelling PURdeg</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><UL class="navigation-menu" style=""><LI><A href="https://2013.igem.org/Team:Imperial_College">Project: Plasticity</A></LI><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Imperial_College/tour">At a glance</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Background">Waste Issue</A></LI><LI>M.A.P.L.E. Modules</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Waste_Degradation:_SRF">M1: Resource-full Waste</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/BioPlastic_Recycling:_PHB">M2: Plastic Fantastic</A></LI></UL><LI><A href="https://2013.igem.org/Team:Imperial_College/Applications">Bioplastic applications</A></LI><LI>Human Practices</LI><LI class="maple"><A href="https://2013.igem.org/Team:Imperial_College/MAPLE">M.A.P.L.E</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Industrial_Implementation">Industrial Implementation</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Managing Waste">Social Challenge</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Communication_work">Communicating our Project</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Project_Evolution">Project Evolution</A></LI><LI>Achievements</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Judging_Criteria">Judging Criteria</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/mainresults">Main Results</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Collaboration">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Biobricks">Biobricks</A></LI><LI>In the Lab</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/MSDS">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/labbook">Lab book</A></LI><LI><A href="#">Modelling Notes</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Modelling_Intro">Introduction</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Modelling_beta-amyloid">Beta amyloid degradation</A></LI><LI><A href="#">Data</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Growth_Assays">Growth Assays</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Western Blots">Western Blot</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Enzyme Kinetics">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/PHB production">PHB production</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/3HB_assay">3HB assay</A></LI><LI><A href="https://2013.igem.org/wiki/index.php?title=Team:Imperial_College/Clearing_Zone_Assay">Clearing Assays</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Electron microscopy">Electron microscopy</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Protocols">Protocols</A></LI><LI>Team</LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Meet_the_Team">Students</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Advisors">Advisors</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Imperial_College/Attributions">Attributions</A></LI><DIV id="main-content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Polyurethane_.28PUR.29_Degradation_Module"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Polyurethane (PUR) Degradation Module</SPAN></A></LI><LI class="toclevel-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2"><A href="#Design"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-3"><A href="#Objective"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Objective</SPAN></A></LI><LI class="toclevel-3"><A href="#The_Model"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">The Model</SPAN></A></LI><LI class="toclevel-3"><A href="#Parameters_and_assumptions"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Parameters and assumptions</SPAN></A></LI><LI class="toclevel-4"><A href="#Gene_expression_module_of_PudA"><SPAN class="tocnumber">1.2.3.1</SPAN><SPAN class="toctext">Gene expression module of PudA</SPAN></A></LI><LI class="toclevel-4"><A href="#PUR_degradation_module"><SPAN class="tocnumber">1.2.3.2</SPAN><SPAN class="toctext">PUR degradation module</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#Simulations_and_Results"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Simulations and Results</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Polyurethane_.28PUR.29_Degradation_Module">Polyurethane (PUR) Degradation Module </SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P align="justify">
Both efficiencies of Polyurethane (PUR) degradation and ethylene glycol production are important for the performance of <B>MAPLE</B> system. We built a mathematical and deterministic model that is based on MATLAB extension Simbiology for Polyurethane degradation. The model contains the kinetic property of degradation enzymes that is helpful for the design of assays. As we scaled up the initial concentrations of all substrates to meet the conditions for a bio-reactor, the model can provide preliminary simulations and predictions for the <B>MAPLE</B> system. </P><H2><SPAN class="mw-headline" id="Design">Design</SPAN></H2><H3><SPAN class="mw-headline" id="Objective">Objective</SPAN></H3><P>Here are some specific objectives for the model to achieve:</P><P>1. The model should contain the gene expression model of the degradation enzymes because the enzyme concentration determines the rate of plastic degradation. In our case for PUR degradation, we used <A href="http://parts.igem.org/Part:BBa_K206000" class="external text" rel="nofollow">pBAD strong promoter K206000</A> for most enzymes. We built the gene expression model based on inducible pBAD promoter, which gene expression rate can be regulated by inducer concentration.</P><P>2. The model should show the efficiency of the enzyme secretion to the culture from the cells. It's also important because the enzyme concentration in the culture depends on it. Here we used pelB secretion tag for most enzymes in order to achieve a high efficiency.</P><P>3. The model basically predict how long will take to degrade a known concentration of soluble polyurethane. It is assumed that the enzyme in our assays has the same kinetic properties as the enzyme used in the literature. The model can suggest a suitable concentration of the plastic to use in order to get good results from the assays.   </P><P>4. It is known that ethylene glycol is toxic to E.coli. However, it has no clear effect on the growth of our MG1655 strain when the concentration of ethylene glycol is below 200mM. Therefore, the model should suggest a safe range of PUR concentration to avoid a high concentration (&gt;200mM) of ethylene glycol produced. </P><H3><SPAN class="mw-headline" id="The_Model">The Model</SPAN></H3><P align="justify">Polyurethane (PUR) degradation involves 5 different degradation enzymes:</P><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH width="150">enzyme</TH><TH>source organism</TH><TH>biobrick</TH><TH>reference</TH></TR><TR><TH>EstCS2</TH><TD>uncultured unknown bacterium (GU256649.1)</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149002" class="external text" rel="nofollow">BBa_K1149002</A></TD><TD><A href="http://www.microbialcellfactories.com/content/10/1/41" class="external text" rel="nofollow">Kang et.al 2011</A></TD></TR><TR><TH>pueA</TH><TD>Pseudomonas chlororaphis</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149003" class="external text" rel="nofollow">BBa_K1149003</A></TD><TD><A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2000.tb09056.x/pdf" class="external text" rel="nofollow">Stern et al., 2000</A></TD></TR><TR><TH>pueB</TH><TD>Pseudomonas chlororaphis</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149004" class="external text" rel="nofollow">BBa_K1149004</A></TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830501000427" class="external text" rel="nofollow">Howard et al., 2001</A></TD></TR><TR><TH>pudA</TH><TD>Comamonas acidovorans </TD><TD><A href="http://parts.igem.org/Part:BBa_K1149005" class="external text" rel="nofollow">BBa_K1149005</A></TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830598000663" class="external text" rel="nofollow">Allen et al. 1999</A></TD></TR><TR><TH>pulA</TH><TD>Pseudomonas fluorescens</TD><TD><A href="http://parts.igem.org/Part:BBa_K1149006" class="external text" rel="nofollow">BBa_K1149006</A></TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830598000687" class="external text" rel="nofollow">Vega et al.,  1999</A></TD></TR></TBODY></TABLE><P>However, 4 of the 5 enzymes are not well characterised before, so we could't find enough kinetic data from the literature. The only well-characterised PUR degradation enzyme PudA is used in the model as an illustration of all PUR degradation enzymes. The model will be more complete when the kinetic data of the other enzymes are defined. The finished PUR degradation model is shown as below: </P><P><B>Key</B></P><P>There are two compartments which represents cells and the culture from left to right. The &quot;cell&quot; compartment contains the gene expression module whereas the &quot;culture&quot; compartment contains the degradation module. The &quot;secretion&quot; block that connects two compartments is the secretion module. </P><H3><SPAN class="mw-headline" id="Parameters_and_assumptions">Parameters and assumptions </SPAN></H3><H4><SPAN class="mw-headline" id="Gene_expression_module_of_PudA">Gene expression module of PudA</SPAN></H4><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH><TH>Assumptions</TH></TR><TR><TH>Î²</TH><TD>maximum rate of transcription</TD><TD>0.032</TD><TD>mM/min</TD><TD>Please see derivation 1 below.</TD><TD>Please see derivation 1 below.</TD></TR><TR><TH>K</TH><TD>Activation coefficient</TD><TD>0.0031</TD><TD>mM</TD><TD><A href="http://parts.igem.org/Part:BBa_K206000:Characterization" class="external autonumber" rel="nofollow">[1]</A></TD><TD>Taking the &quot;switch point&quot; as the activation coefficient</TD></TR><TR><TH>d<SUB>mRNA</SUB></TH><TD>mRNA degradation rate</TD><TD>0.10</TD><TD>1/min</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC420366/pdf/07X.pdf" class="external autonumber" rel="nofollow">[2]</A></TD><TD>Taking the value of mRNA half-life in E.coli strain MG1655 as 6.8min.  rate = ln2/half-life = ln2/6.8 = 0.10</TD></TR><TR><TH>d<SUB>protein</SUB></TH><TD>Protein degradation rate</TD><TD>0.050</TD><TD>1/min</TD><TD><A href="http://jb.asm.org/content/189/23/8746.full" class="external autonumber" rel="nofollow">[3]</A></TD><TD>There is no active degradation pathway and that dilution is the dominant way by which the protein level decreases in a cell.  Rate = 1/doubling time, where doubling time = 20min.  Assuming steady-state growth in LB broth as presented in paper.</TD></TR><TR><TH>k<SUB>2</SUB></TH><TD>Protein production rate (PudA)</TD><TD>4.7</TD><TD>1/min</TD><TD>Please see derivation 2 below.</TD><TD>Please see derivation 2 below.</TD></TR><TR><TH>[Arabinose]</TH><TD>Concentration of arabinose</TD><TD>Initial: 0.008</TD><TD>mM</TD></TR><TR><TH>[mRNA]</TH><TD>Concentration of mRNA</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR><TR><TH>[PudA]</TH><TD>Concentration of PudA</TD><TD>-</TD><TD>mM</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE><P><B>1.Derivation of the maximal expression rate,Î²</B><UL><LI>Average molecular weight (Mw) of a base pair = 660g/mol<A href="http://www.geneinfinity.org/sp/sp_dnaprop.html" class="external autonumber" rel="nofollow">[4]</A><A href="http://www.lifetechnologies.com/uk/en/home/references/ambion-tech-support/rna-tools-and-calculators/dna-and-rna-molecular-weights-and-conversions.html" class="external autonumber" rel="nofollow">[5]</A></LI><LI>Average mass of a base pair = 660g/mol x 1.66x10<SUP>-24</SUP> = 1.1x10<SUP>-21</SUP>g
</LI><LI>Volume of an E.coli cell = 1Âµm<SUP>3</SUP><A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external autonumber" rel="nofollow">[6]</A> = 1x10<SUP>-15</SUP>L
</LI><LI>â´Mass concentration = </LI><LI>â´Molar concentration of 1 base pair in the volume of E.coli =  = 1.66x10<SUP>-6</SUP> mM
</LI></UL><LI>BioBrick assembly plasmid pSB1C3 is a high copy number plasmid (100-300 copies per cell)<A href="http://parts.igem.org/Part:pSB1C3?title=Part:pSB1C3" class="external autonumber" rel="nofollow">[7]</A></LI><LI>assume 200 copies per cell
</LI><LI>â´ concentration of the gene per cell = N x 200 x 1.66x10<SUP>-6</SUP>mM, where N = number of base pairs
</LI><LI>â´ concentration of the gene pudA (N = 768) in the volume of an E.coli cell is = 0.25mM
</LI><LI>Transcription rate in E.coli = 80bp/s<A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external autonumber" rel="nofollow">[8]</A> = 80 x 1.66x10<SUP>-6</SUP>mM/s = 80 x 1.66x10<SUP>-6</SUP> x 60mM/min = 7.97x10<SUP>-3</SUP>mM/min
</LI><LI>â´ Rate of mRNA_pudA production under the control of pBAD = 7.97x10<SUP>-3</SUP> Ã· 0.25 = <B>0.032/min</B></LI><B>2.Protein production rate of pudA, k<SUB>2</SUB></B><UL><LI>Average molecular weight(Mw) of an amino acid(aa)= 110g/mol<A href="http://www.genscript.com/conversion.html" class="external autonumber" rel="nofollow">[9]</A><A href="http://www.promega.com/~/media/Files/Resources/Technical%20References/Amino%20Acid%20Abbreviations%20and%20Molecular%20Weights.pdf" class="external autonumber" rel="nofollow">[10]</A></LI><LI>Average mass of an amino acid = 110g/mol x 1.66x10<SUP>-24</SUP>=1.83x10<SUP>-22</SUP>g/L
</LI><LI>â´Mass concentration of one aa in the volume of an E.coli =  = 1.83x10<SUP>-6</SUP>g/L
</LI><LI>â´Molar concentration of one aa =  = 1.66x10<SUP>-5</SUP>mM
</LI></UL><LI>Translation rate = 20aa/s = (20 x 1.66x10<SUP>-5</SUP> x 60)mM/min = 0.020mM/min
</LI><LI>PudA comprises of 256aa<A href="http://www.uniprot.org/uniprot/Q2PEN2&amp;format=html" class="external autonumber" rel="nofollow">[11]</A></LI><LI>â´concentration of pudA's aa in the volume of an E.coli = 1.66x10<SUP>-5</SUP>mM x 256 = 4.25x10<SUP>-3</SUP>mM
</LI><LI>â´ Rate of protein production = 0.020 Ã· 4.25x10<SUP>-3</SUP> = <B>4.7/min</B></LI></P><H4><SPAN class="mw-headline" id="PUR_degradation_module">PUR degradation module</SPAN></H4><P>The reaction equation of the PUR degradation is:</P><P>[Polyurethane]+[PudA]= 5 [ethylene glycol] + 5 [polyisocyanate] + [PudA]
</P><P><B>Assumptions:</B></P><P>We assumed 1 mole of polyurethane dispersion can produce 5 moles of ethylene glycol. </P><P>The molecular weight of a single polyurethane monomer is 470 g/mol whereas the molecular weight of the polyurethane dispersion is around 2000 g/mol.<A href="http://www.polyurethanes.basf.de/pu/solutions/en/content/group/Arbeitsgebiete_und_Produkte/Grundprodukte/Lupraphen-Produktuebersicht_Gewicht" class="external autonumber" rel="nofollow">[12]</A>Therefore, the short-chain polyurethane consists approximately 5 monomers. 5 molecules of ethylene glycol will be produced by degrading one chain of polymer.</P><P>We also assumed a simple Michaelis-Menten mechanism for PudA</P><P>The parameters for the kinetic equations are:</P><TABLE border="2" bgcolor="#efefef"><TBODY><TR><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Units</TH><TH>Sources</TH></TR><TR><TH>Km</TH><TD>Michaelis constant</TD><TD>51.5</TD><TD>mM</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830598000663" class="external autonumber" rel="nofollow">[13]</A></TD></TR><TR><TH>Kcat</TH><TD>Turnover number</TD><TD>141.75</TD><TD>1/min</TD><TD><A href="http://www.sciencedirect.com/science/article/pii/S0964830598000663" class="external autonumber" rel="nofollow">[14]</A></TD></TR></TBODY></TABLE><P>The efficiency of <B>Secretion</B> is assumed to be 90% secretion over 2 hours.<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1251600/" class="external autonumber" rel="nofollow">[15]</A> The rate of secretion in the model is therefore:</P><P>rate of secretion = 0.9[concentration of PudA]/120 (mM/mins) </P><H2><SPAN class="mw-headline" id="Simulations_and_Results">Simulations and Results</SPAN></H2><P>In terms of ethylene glycol toxicity, if we can't let the concentration of ethylene glycol over than 200mM, the maximum initial concentration of polyurethane dispersion is 40mM because one mole of polyurethane dispersion can produce 5 moles of ethylene glycol. (from the derivation above) If we put 40mM as the initial concentration in the model, the simulation result is:</P><P>40mM of polyurethane dispersion can be efficiently degraded in 20 mins. In case if we keep the concentration of the plastic below 40mM. The ethylene glycol won't affect the cell growth in our assays. 40mM is converted to 80g/L which suggests no more than 80 grams of polyurethane to be put in the system in order to avoid toxicity.   </P><P>As for the implementation of MAPLE system, the polyurethane degradation enzymes in our system are very efficient. Therefore, we need an efficient filter which removing ethylene glycol from the system. 


</P></DIV><DIV id="supporters" class="clear"><P><B>Our Sponsors</B></P><A href="/File:Logo-bioline.gif" title="File:Logo-bioline.gif">125px</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Imperial_College/Modelling_PURdeg">http://2013.igem.org/Team:Imperial_College/Modelling_PURdeg</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Imperial_College/Modelling_PURdeg" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Imperial_College/Modelling_PURdeg" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Imperial_College/Modelling_PURdeg&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Imperial_College/Modelling_PURdeg&amp;oldid=275146" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>