<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_IIT_Madras_DryLabAndModelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:IIT_Madras/DryLabAndModelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:IIT_Madras/DryLabAndModelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:IIT_Madras/DryLabAndModelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:IIT_Madras/DryLabAndModelling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:IIT_Madras/DryLabAndModelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:IIT Madras/DryLabAndModelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="header"><DIV class="container"><DIV class="heading visible-tablet visible-desktop" style="
    font-size: 30px;
    position: absolute;
    margin-left: 140px;
    margin-right: auto;
    top: 45px;
    font-family: 'Rammetto One', cursive;
">Indian Institute of Technology Madras</DIV></DIV></DIV><DIV class="navbar navbar-inverse"><DIV class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2013.igem.org/Team:IIT_Madras">
            Home
          </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
            Project
            </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/Background">
                Background




              </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/Approach">
                Approach
              </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/Results">
                Results
              </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/DryLabAndModelling">
                Dry lab and Modelling
              </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/Design">
                Design
              </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/Safety">
                Safety
              </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                Human Practices
                </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/HumanPractices">
                    Overview
                  </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/CodeRed">
                    Code Red
                  </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                Team
                </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/Team">
                    Bios
                  </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/Gallery">
                    Gallery
                  </A></LI><LI><A target="_blank" href="https://igem.org/Team.cgi?year=2013&amp;team_name=IIT_Madras">
                    Official Team Profile
                  </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
                Notebook
                </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/Protocol">
                    Protocols
                  </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/Weekly">
                    Weekly Summaries
                  </A></LI><LI><A href="https://2013.igem.org/Team:IIT_Madras/Attributions">
                Attributions
              </A></LI></DIV></DIV><DIV class="container"><H2 align="center">
      Dry Lab and Modelling
    </H2><DIV class="row featurette"><H3>
    BACKGROUND
</H3><P dir="ltr">
    Molecular docking is a widely-used computational tool for the study of molecular recognition, which aims to predict the binding mode and binding affinity
    of a complex formed by two or more constituent molecules with known structures. Protein-ligand docking in particular, is very important because of its
    therapeutic applications in modern structure-based drug design.
</P><P dir="ltr">
    Our team chose to do docking experiments in order to evaluate the binding characteristics of Shiga toxin and the peptide designed to neutralise it, as it
    is important to validate the results generated from experimental techniques.
</P><P dir="ltr">
    Since only the sequence of the peptide(WHWTWLSEY) was known, it was necessary to model the structure of the peptide. For this purpose, we used the online
    webserver, PEP-FOLD(Yimin SHEN et al.). PEP-FOLD is based on the concept of structural alphabet (SA) - i.e. a description of apolypeptidic conformation as
    a series of local canonical conformations, and uses a HMM (Hidden Markov Chain)-derived SA of 27 letters to describe proteins as series of overlapping
    fragments of four amino acids. PEP-FOLD is based on a two-step procedure:
</P><UL><LI dir="ltr"><P dir="ltr">
            prediction of a limited set of SA letters at each position from peptide amino acid sequence
        </P></LI><LI dir="ltr"><P dir="ltr">
            assembly of the prototype fragments associated with each SA letter using
        </P></LI></UL><P dir="ltr">
    a greedy algorithm and a generic protein coarse-grained force ﬁeld.
</P><P dir="ltr">
    As mentioned earlier, Shiga toxin binds to human Globotriaosylceramide (Gb3Cer) receptors which are present in the endothelium of mammalian cells. The
    crystal structure of Gb3 bound to Shiga toxin has already been determined by crystallization (PDB id - 1BOS) performed by Ling et.al. This structure was
    used in our experiments as a reference structure for carrying out docking studies.
</P><P dir="ltr">
    Further work done by Miura et.al., on Gb3 mimicking peptides revealed the Molecular dynamics and binding characteristics of the truncated peptide
    considering only the first five amino residues of our peptide (WHWTWLSEY). In order to compare the docking characteristics of the entire peptide as opposed
    to the truncated peptide that was used in the publication, we carried out our bioinformatics experiments.
</P><H3>
    TOOLS: HADDOCK
</H3><P dir="ltr"><A href="http://www.nmr.chem.uu.nl/haddock">HADDOCK</A>
    (High Ambiguity Driven protein-protein DOCKing) is an information-driven flexible docking approach for the modeling of biomolecular complexes. HADDOCK can
    deal with a large class of modelling problems including protein-protein, protein-nucleic acids and protein-ligand complexes. For the purpose of our
    studies, we used the Easy Interface of the HADDOCK server which requires several parameters including active and passive residues.
</P><P dir="ltr">
    Since our peptide (WHWTWLSEY) is a Gb3 receptor mimic, it is reasonable to assume that the residues in Shiga toxin to which the peptide binds are the same
    ones to which Gb3 binds .
</P><P dir="ltr">
    These active residues were obtained from PDB. These were found to be Lys(33), Asn(35),Asp(36), Asp(38), Thr(41), Glu(48), Thr(51), Asn(52), Arg(53),
    Trp(54), Asn(55), Thr(74), Asn(75), Gly(80), Gly(82), Phe(83).
</P><H3>
    RESULTS
</H3><P dir="ltr">
    HADDOCK uses the HADDOCK score to rank the clusters generated during docking. This is a weighted sum of:
</P><UL><LI dir="ltr"><P dir="ltr">
            Electrostatic energy (weight 0.2)
        </P></LI><LI dir="ltr"><P dir="ltr">
            Van der Waals energy (weight 1.0)
        </P></LI><LI dir="ltr"><P dir="ltr">
            Desolvation energy (weight 1.0)
        </P></LI><LI dir="ltr"><P dir="ltr">
            Restraints violation energies (distance, SANI) (weight 0.1)
        </P></LI></UL><P dir="ltr">
    According to this ranking score, we shortlisted the best models from the top 5 clusters.These were further analysed on the basis of low RMSD to the
    original structure of Gb3 bound to Shiga toxin.
</P><DIV dir="ltr"><TABLE><TBODY><TR><TD><P dir="ltr">
                        Clusters
                    </P></TD><TD><P dir="ltr">
                        HADDOCK score
                    </P></TD><TD><P dir="ltr">
                        RMSD from structure of lowest energy
                    </P></TD><TD><P dir="ltr">
                        Z Score
                    </P></TD></TR><TR><TD><P dir="ltr">
                        1
                    </P></TD><TD><P dir="ltr">
                        -110.4 +/- 6.1
                    </P></TD><TD><P dir="ltr">
                        0.4 +/- 0.2
                    </P></TD><TD><P dir="ltr">
                        -1.5
                    </P></TD></TR><TR><TD><P dir="ltr">
                        2
                    </P></TD><TD><P dir="ltr">
                        -100.8 +/- 6.2
                    </P></TD><TD><P dir="ltr">
                        1.6 +/- 0.3
                    </P></TD><TD><P dir="ltr">          -0.9
                    </P></TD></TR><TR><TD><P dir="ltr">
                        3
                    </P></TD><TD><P dir="ltr">
                        -93.6 +/- 10.2
                    </P></TD><TD><P dir="ltr">
                        1.0 +/- 0.3
                    </P></TD><TD><P dir="ltr">
                        -0.5
                    </P></TD></TR><TR><TD><P dir="ltr">
                        4
                    </P></TD><TD><P dir="ltr">
                        -91.2 +/- 3.9
                    </P></TD><TD><P dir="ltr">
                        2.4 +/- 0.0
                    </P></TD><TD><P dir="ltr">
                        -0.3
                    </P></TD></TR><TR><TD><P dir="ltr">
                        5
                    </P></TD><TD><P dir="ltr">
                        -70.9 +/- 7.4
                    </P></TD><TD><P dir="ltr">
                        2.2 +/- 0.2
                    </P></TD><TD><P dir="ltr">
                        0.9
                    </P></TD></TR><TR><TD><P dir="ltr">
                        6
                    </P></TD><TD><P dir="ltr">
                        -69.0 +/- 4.9
              </P></TD><TD><P dir="ltr">
                        2.0 +/- 0.1
                    </P></TD><TD><P dir="ltr">
                        1
                    </P></TD></TR><TR><TD><P dir="ltr">
                        7
                    </P></TD><TD><P dir="ltr">
                        -64.2 +/- 14.9
                    </P></TD><TD><P dir="ltr">
                        2.0 +/- 0.1
                    </P></TD><TD><P dir="ltr">
                        1.3
                    </P></TD></TR></TBODY></TABLE></DIV><P dir="ltr">
    Cluster 1
</P><P dir="ltr">
    Cluster 2
</P><P dir="ltr">
    Cluster 3
</P><P dir="ltr">
    Cluster 4
</P><P dir="ltr">
    Cluster 5
</P><P dir="ltr">
    Amongst the predicted results, Cluster 2 seems to be most suited as an interacting partner to Shiga toxin. This can be attributed to the optimum energy
    values assigned by HADDOCK as well as the high similarities of the binding modes of the peptide to the toxin as compared to the same of Gb3-Shiga toxin
    complex. This was analysed by the RMSD values as indicated by Pymol following structural alignment. The RMSD value obtained was 0.359.
</P><H3>
    SCOPE FOR FURTHER WORK
</H3><P dir="ltr">
    Our team intends to construct a peptide library using tools like SDM to create mutants of the peptide and analysing interactions of these with Shiga Toxin.
    We can also search for sequences homologous to the toxin which could be harmful in nature and test their interactions with peptide library.This is a
    high-throughput analysis and can be validated further by several wet lab techniques.
</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:IIT_Madras/DryLabAndModelling">http://2013.igem.org/Team:IIT_Madras/DryLabAndModelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:IIT_Madras/DryLabAndModelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:IIT_Madras/DryLabAndModelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:IIT_Madras/DryLabAndModelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:IIT_Madras/DryLabAndModelling&amp;oldid=236306" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>