Team:Edinburgh OG/Experiments/Cyanobacteria
-->
Toggle navigation
TEAM
About Us Attributions Collaborations
PROJECT
Description Design Method Proof of Concept Results
EXPERIMENTS
Team Cyanobacteria Team Rhodococcus Team Fungi Notebook
PARTS
SAFETY
INTERLAB
HUMAN PRACTICES
Silver Gold Integrated Practices
AWARDS
Introduction to Synechocystis sp.
Synthetic biology and the domestication of microorganisms
Despite the significant progress that has been achieved in manipulating genetic circuits and devices, the application of synthetic biology methods and techniques needs to be widened to a larger group of microorganisms in order to fully reach its potential. Rather than struggling with the limitations of the available commercial chassis, scientist should have the ability to truly select an organism for a specific task that best suits the user needs.  In order to allow researchers to edit, rewrite and repurpose genetic circuits, one must first standardize and characterize biological parts, such as promoters, ribosome binding sites, coding sequences, terminators as well as other regulatory elements but selecting an appropriate assembly standard is also part of doing it in a fast, efficient, and affordable manner (Ramey et al. 2015).
Synechocystis and its role in synthetic biology
Aside from the biofuels and added-value chemicals that cyanobacteria have been shown to produce, Synechocystis exhibits traits that could make it useful as an industrial biotechnology strain, as it prefers growth temperatures between 30–34 °C and has a doubling time of 7–10 h under intermediate light conditions (Stevan C. Albers et al. 2015). Several strategies have been applied to manage metabolic burden in heterologous systems. Accurate control of gene expression via promoter engineering, control over transcription factors turnover number and codon optimization have shown to be essential components when managing the adverse conditions generated during product formation (Albers et al. 2015). It wasn’t until 2011 that cyanobacteria engineering seemed to be resurging, when a broad-host-range cyanobacteria BioBrick shuttle vector, pPMQAK1, was made available. The latter seems to show that lack of well characterized parts and plasmids, as well as the community’s disarray in adopting standards seem to be among the causes preventing non-model organisms from being widely used (Huang et al. 2010).
Project aims
Taking into account what was mentioned before, the aims of this project were structured as follows. (1) Create a library of parts to be characterized and submitted to the iGEM competition. (2) In an effort to have a reporter protein working in all four organisms, generate and characterize two coding sequence that, regardless of codon bias work in all four of them. (3) Confirm the correct assembly and sequence of said parts. (4) Assemble the parts into expression cassettes for characterization. (5) Transform and characterize the parts using E. coli TOP10 cells. (6) Clone constructs into the Synechocystis sp. PCC 6803 shuttle vector (pPMQAK1) and finally (7) characterize the parts using Synechocystis sp. PCC 6803.
Promoter Design
Promoter studies in Synechocystis have demonstrated the mechanism of several native promoters. The promoters can be categorized into three different types (I, II, and III) depending on how the binding motifs arranged to interact with the respective sigma factor and its equivalent transcription factor. For example, some of these have been shown to be “light responsive (psa, psb, and secA), dark-inducible (lrtA), nitrate/nitrite-inducible (nirA), copper-ions responsive (petE), and heavy metal-ions inducible” (Huang et al. 2013a).
When using a Synthetic biology approach, genetic circuits are supposed to function as independently as possible from the endogenous cellular circuitry or even functionally replace the innate molecular machinery (Nandagopal & Elowitz 2011). In an effort to follow this approach, and avoid cross-talks with the host’s genetic background, the development of non-native promoters regulated by exogenous transcription factors seem to be a plausible option. In this scenario, the aim being an orthogonal promoter that can be fully repressed and highly inducible(Huang et al. 2013b). For this case the selected promoters were LacI-regulated and TetR-regulated, both inducible by Isopropyl β-D-1-thiogalactopyranoside (iPTG) and anhydrotetracycline (aTc) respectively.
L03
In order to initiate transcription the UP element, -35 element, extended -10 element, the -10 element, and a nucleotide in two positions downstream of the -10 element are considered critical for the correct interaction of a promoter with RNA polymerase (RNAP). One way to regulate transcription is to make use of transcription factors that bind to operator regions within promoters and disrupt or block the RNAP binding sites. Following that same principle, a promoter library was created by Huang et al, using the BBa_R0040 promoter. The L03 promoter was selected from this library of promoters because it showed a moderate repression and high induction rates in Synechocystis in the presence of the repressor protein TetR. The fact that TetR does not naturally exist in the cyanobacteria adds second layer of control over the independent behaviour of the promoter. Meaning that TetR will only interact with the operator regions in the L03 promoter(Huang et al. 2013b). For this research project, L03 was only intended to be used as a constitutive promoter in Synechocystis, meaning that the interactions with its repressor protein and inducer were not taken into account. However, since it will be characterized initially in E. coli, and the bacteria produces its own TetR, in order for it to work, aTc experiments will be required for gene induction.
Sca6-2
The Sca6-2 promoter, comes from Albers and co-workers, rational design approach towards the existing tac promoter. The tac promoter being a hybrid generated from the E. coli promoter regions Ptrp and Plac. Several iterations were generated by mutating nucleotides in a step-wise manner within the −10 and −35 cis-acting regions of the promoter. This produced a library of several promoters with a dynamic range of expression strength as well as its ability to be repressed, thanks to the lac operator sequence located after the -10 region of the promoter. These were further characterized in Synechocystis sp. PCC 6803. Showing that in the presence of the repressor protein LacI, the promoter was able to remain highly repressed. Subsequently, in the presence of the iPTG inducer, the promoter was able to express high concentrations of protein (Stevan C. Albers et al. 2015).
PA1lac0-1
One of the metabolite-inducible promoters that have been previously characterized in Synechocystis is PA1lac0-1 . Derived from the PA1 promoter of bacteriophage T7 (Lutz et al., 1997), PA1lac0-1 is a LacI-dependent promoter and Isopropyl β-D-1- thiogalactopyranoside (IPTG)-inducible promoter that has been characterized with ethylene biosynthesis as a reporter and has been shown to bring about an 8-fold induction in both E. coli and Synechocystis (Guerrero et al., 2012). PA1lac0-1 has been designed to include the A and B MoClo adaptor sites.
Pcpc560
Recently, the 560bp region found upstream of the cpcB gene encoding the beta subunit of c-phycocyanin in Synechocystis has been reported to act as a promoter that is able of producing heterologous proteins at a level up to 15% of total soluble protein in Synechocystis, which is comparable to levels achieved in E. coli (Zhou et al., 2014). In light of this, this genomic region has been termed as the Pcpc560 “super-strong” promoter. As this promoter already includes a native RBS. it has been designed to include the A and C fusion sites, which enables cloning directly in front of a reporter gene.
J23100_AB
Part J23100_AB is a constitutive promoter from the Anderson Laboratory collection. This promoter is regarded as the strongest promoter from that library. It was selected to serve as a positive promoter control in case the other two promoters were unable to express in E. coli.
Ribosome Binding Sites
In order to start translation, aside from the promoter and the coding sequence, a ribosome binding site (RBS) is required. Even though it is often referred as just a part of the mRNA sequence that binds to the ribosome, the surrounding sequence can also affect the translation initiation rate (Smit & Duin 2013). In the case of the Sca6-2 and L03 promoters, their respective RBS were included in their sequences, in an effort to avoid any negative results that could appear after disrupting the delicate area between the 3’ end of a promoter and the start codon of the coding sequence. Because these parts already included an RBS, the 3’ adapter region was designed to end the standard RBS overhang. The putative RBS are labeled in the promoter sequences in the supplementary data. For constructs related to the J23100 promoter, the strong RBS B0034_BC was used.
RBS*
BBa_B0034 is currently the strongest E. coli RBS present in the iGEM Registry, however, its activity drops by about 84% when it is used in Synechocystis (Heidorn et al., 2011). RBS* is an RBS that has been specifically designed to include the anti-Shine Dalgarno (SD) sequence of Synechocystis and has an optimal spacing of 9bp between the central base of the core SD sequence and the first base of the start codon (Heidorn et al., 2011). RBS* has been shown to be 5 times stronger than BBa_B0034 in Synechocystis (Heidorn et al., 2011).
Coding Sequences
Fluorescent reporters
Fluorescence is widely used as a proxy for promoter activity by expressing fluorescent proteins. While this remains an indirect measurement, it provides an insight into expression levels and provides a certain advantage by allowing for continuous quantification without disrupting the assessed cells (Lissemore et al. 2000).
In the past, other fluorescent proteins, including Cerulean, GFPmut3B and EYFP, have been demonstrated to work in both E. coli and Synechocystis. The reported excitation wavelenghts used were 433, 505 and 515 nm, respectively, where the maximum emission wavelengths registered for Cerulean, GFPmut3B and EYFP were 477, 514 and 529 nm, respectively (Fig. 4). This suggests that other fluorescent protein that work between those ranges might be suitable reporter proteins in cyanobacteria, in spite of the strong background of photosynthetic pigments (Huang et al. 2010).
For this project, the selected reporters were a monomeric Red fluorescent protein (mRFP), from Discosoma striata (coral) with a reported excitation peak at 584 nm and an emission peak at 607 nm. As a second reporter iLOV was selected. iLOV, a modified LOV2 domain from a blue light receptor phototropin (Briggs et al. 2007) has a reported  excitation peak around 476nm, with a measurable emission range between 510 and 550 nm.
mRFP and iLOV
The red fluorescent protein was selected because its bright red colour can be clearly seen when produced in E. coli, as well as its availability from the CIDAR ICE repository. On the other hand, iLOV was selected because of its many advantages that could make it a better option than the most widely used green fluorescent protein. Its gene size is only 336 base pairs; most of the other fluorescent proteins available are at least twice as big. Its size, its reported superior photostability as well as its ability to fluoresce under anaerobic conditions make it an interesting reporter to be used across multiple organisms (Christie et al. 2012).
LacI
The LacI protein is required for the adequate repression of Plac derived promoters. Without its binding to the operator region of these promoters, there is nothing blocking transcription and therefore gene expression. In an effort to improve the reaction time of the induction and have better control over induction, research groups have suggested the addition of a LVA degradation tag to the C-terminus of LacI. The LVA tag is a short peptide sequence that marks the intended protein sequence for degradation by cellular proteases (Karzai et al. 2000).  In E. coli, however, at least one study seems to show that LacI without the tag actually reacts quicker in response to inducers (Andreassen et al. 2014). In Synechocystis it has been suggested that the amount of LacI present is correlated to the leakiness of LacI dependent promoters (Albers et al. 2015). It has also been reported that the addition of the highly active LVA tag to proteins greatly decreases protein concentration (Wang et al. 2012). It is because of this that the synthesis of a LacI without an LVA tag was decided.
Predicting coding sequences for simultaneous use in multiple chassi
Before focusing on developing new synthetic biology tools for organisms, one must try to elucidate as much as possible on the mechanism of its existing genetic framework. Even when promoter sequences and the molecular machinery between E. coli  and Synechocystis are considered similar, DNA parts are not directly transferrable (Ramey et al. 2015). Aside from regulatory sequences, coding sequences can sometimes be troublesome to work with, especially if these are not codon optimized for the selected organism. For the work group this would have meant that all the reporter protein sequences would have required four codon optimizations (for E. coli, Rhodococcus, Penicillium, Synechocystis). As an alternative, and to avoid this situation for further projects, a different approach was discussed and implemented. By trying to design a sequence based on all four codon preference tables, one could generate a sequence able to express well in all four hosts. Perhaps this might not be the most ideal sequence, but, one that could potentially help avoid some DNA synthesis costs.
In order to predict the sequences, reliable codon usage tables were required. These were obtained from the CUTG database from the Kazusa DNA Research Institute in Japan. For the organisms present in the database a lists of the codon usage of genes as well as the number of coding sequences that were evaluated (Nakamura et al. 2000). As a proof of concept, two algorithms were designed in order to predict a sequence for the fluorescent protein using Kazusa’s codon usage datasets. The iLOV fluorescent reporter was selected because of its previously mentioned advantages over other fluorescent proteins. The result was a predicted sequence from each algorithm (more can be found in the materials and methods section). The resulting sequences were named “welov1” and “welov2”.
Terminators
While promoter strength is a determinant of gene expression levels, the terminator also plays an important role in RNA processing related to variability in RNA half-life. Therefore selecting an appropriate terminator is key if one expects positive gene expression. For this research project the terminators used were two double terminators (B0015_DE, BBa_B0015_DF) identical except for their 3’ adapter regions. This was required in order to design level 2 devices. Fortunately, this Rho-independent terminator has been shown to work in both E. coli and Synechocystis in the past (Huang et al. 2010).
iGEM plasmids and Phytobrick standard
In the case of the new parts, these are required to be submitted in a standard Universal Acceptor vector (P10500). In order to correctly clone a part into P10500 some guidelines must be followed based on the PhytoBrick standard.
Predicting coding sequences for simultaneous use in multiple chassi
The algorithms used for the predicting a de-optimzed coding sequence for the four organisms would start with an aminoacid sequence. For this method, the analysis relied on Kazusa’s codon usage datasets, the two approaches were generated. In order to simplify the calculations a script was developed with the assistance and cooperation of another student Diana Gamez, in the programing language PHP (program available in the supplementary data).
First Algorithm: sequence prediction based on similar abundance of codons.
Parting from the first aminoacid, the ideal codon was decided by 1) Obtaining the abundances for the respective coding codons in all four organisms. 2) Calculating the Euclidean distance between those abundances. 3) The codon with the minimum distance between abundance would then be selected as the appropriate triplet. This was done for all aminoacids from the iLOV protein. In an effort to mitigate selecting codons, that not only had the smallest Euclidean distance between abundances, but also a low abundance; values with higher abundance were preferred in the script.
Second Algorithm: sequence prediction is based on most abundant codon for the majority of microorganisms.
Parting from the first aminoacid, the ideal codon was decided by 1) Obtaining the abundances for the respective coding codons in all four organisms. 2) Evaluating all respective codons and select two with the highest abundance. 3) Calculating the Euclidean distance between those abundances. 4) The codon with the minimum distance between abundance would then be selected as the appropriate triplet. This was done for all the aminoacids in the iLOV sequence.
Strains, cell culture medium and growing conditions
E. coli TOP10 competent cells were kindly donated by Dr. Louise Horsfall’s laboratory (School of Biological Sciences, University of Edinburgh, UK) throughout the project. A second E. coli strain was also requested, the LacI deficient strain: lacI785 (del)::kan JW0336-1 was obtained from Yale’s “Coli Genetic Stock Center”. Synechocystis sp. PC 6803 cultures were made available by Dr. Annegret Honsbein from the Rosser Laboratory (School of Biological Sciences, University of Edinburgh, UK).
Transformation
Bacterial transformation was performed following a standard protocol provided by Dr. Heather Barker: 2 μL (~200ng) of purified plasmid were mixed with 100 μL of E. coli TOP10 competent cells (NEB), placed on ice for 30 minutes followed by 30 seconds of heat shock in a water bath at 42 °C. Cells were then placed back on ice for 30 seconds and 1 mL of SOC medium without antibiotics was added. Cells were grown for 1 hour in a 37 °C water bath. Finally, transformed cells were centrifuged for 1 minute at 4000 rpm, supernatant was discarded and cells were then re suspended in 100 μl of LB medium. 100 μl of cells were then plated on LB agar plates with their respective antibiotic, 100 μg/ml of ampicillin (for CIDAR ICE level 0, and level 2 Moclo assemblies), 50 μg/ml kanamycin (for level 1 Moclo assemblies), 35 ug/ml chloramphenicol (for level 0 gBlock/Moclo assemblies) and grown overnight (12-14 hours in a 37 °C incubator).
When transforming Moclo assemblies, in addition to the specified antibiotic, in order to allow for blue-white screening, LB agar plates were supplemented with iPTG and Xgal (0.1 mM and 40 μg/ml respectively).
Plasmid purification
Plasmid purification was carried out using the QIAprep Spin Miniprep Kit.  Colonies were selected and used to inoculate 10 ml LB overnight cultures of transformed TOP10 cells throughout the project. Elution was performed according to the standard protocol except for the initial overnight culture (10 mL), the volume in which the DNA was eluted (30-40 µl of nuclease free water) and the incubation period in the spin column (5 min incubation).
DNA quality and quantity measurement
Purified plasmid DNA samples were measured using the NanoDrop 100 spectrophotometer (Thermo Scientific). Samples were found to be between the Thermo Scientific suggested purity ranges (2.0-2.2 for 260/230 and 1.8-2.0 for 260/280 ratios).
Restriction enzyme digestion and agarose gel electrophoresis
Restriction enzyme digestions were carried out following New England Biolabs suggested reaction. Using 200-300 ng of DNA, 5 units of enzyme, 1× recommended buffer, and nuclease free water to a total volume of 20 µl, for 1-1.5 hours, reactions were then incubated at 80 °C for 10 minutes, in order to inactivate the enzymes. PCR products were analysed by agarose gel electrophoresis. Agarose gels were prepared using 0.8% (w/v) agarose, 1× TAE buffer (40mM Tris, 20mM acetic acid, and 1mM EDTA), 1× SYBR® Safe DNA gel stain and then run at 115 V in an electrophoresis chamber for 40-60 minutes. Samples were loaded into the gel with 1× of DNA gel loading dye (NEB). A 1kb DNA ladder (NEB) was used as molecular weight marker.
Moclo Assembly
Throughout the project two Moclo assembly protocols were evaluated, the first one in accordance to the CIDAR Moclo protocol consisting of: 10 fmol of each DNA component with up to six components total (for this case either double stranded synthetic gBlocks or previously made MoClo DNA parts as well as the appropriate destination vector), 10 U of BsmbI, BsaI or BbsI (NEB), 10 U of T4 DNA ligase (NEB), 1× T4 DNA ligase buffer (NEB), and nuclease free water to at total volume of 20 μL. Reactions were performed using the next parameters: 1 cycle (37 °C), 15 cycles (37 °C 1.5–3 min, 16 °C 3–5 min), followed by 50 °C for 5 min and 80 °C for 10 min and were then held at 4 °C, afterwards using 5 µl of the reaction for the transformation protocol (Iverson et al. 2016). The second protocol was a modified version of the previous one, its main difference consisting in using a greater concentration of DNA: 100 ng of each part was combined with 50 ng of destination plasmid, while following the same parameters from the previous reaction (Moore et al. 2016).
However, due to several negative results from the level 1 assemblies, even after trying both assembly protocols, another level 1 acceptor plasmid, pSRKM_AE, was used. This plasmid was designed and provided by the MSc. Student Matin Nuhamunada.
Sequencing
Sequencing was carried out by “Edinburgh Genomics” (The University of Edinburgh, UK). Samples were submitted in 0.2 ml strip tubes, containing 500 ng of purified plasmid as well as 3.2 mM of a primer as well as nuclease free water to a 10 µl total volume.
The primer used for all verifications consisted of:
VF2 Primer: tgccacctgacgtctaagaa
This primer works for all parts cloned in the CIDAR ICE plasmids as well as all parts cloned in any plasmids derived from iGEM’s “pSB1C3”.
Promoter characterization in live E. coli cells and Fluorescence Assessment
Promoter Sca6-2 was characterized in E. coli TOP10 cells. Colonies were picked and grown overnight at 37 °C, 250 RPM in 50 mL conical tubes in 10 mL of LB medium with its respective antibiotic (50 ug/mL kanamycin). Cell density was measured in a Jenway 7305 spectrophotometer, the obtained data was processed using Excel Data Processing tools. Cells were then subcultured into opaque-walled 96 well plates and adjusted to OD600 0.05 in 290 µl of LB medium supplemented with antibiotics as well as their respective inducer concentration (0, 0.005, 0.01, 0.05, 0.1 and 0.5 mM iPTG) in triplicates. Plates were then placed in the Fluostar omega microplate reader from BMG labtech, and grown at 37 °C for 20 hours at 300 RPM. Aside from monitoring OD600, fluorescence was determined by measuring emission wavelengths using three fixed excitation filters (584 and 544 nm) and three fixed emission filters (590 and 512 nm) every 30 minutes. Triplicate replicates were performed for each inducer concentration. All data is presented in relative fluorescence units, which represents the relative amount of the specific wavelengths of light collected by the plate reader at the conditions described above.
Experiments and Protocols
This is a record of protocol and optimization that we did
1. Competent cells
Competent cells were obtained following the iGEM protocol Help:Protocols/Competent Cells
2. Transformation
E. coli DH5α was transformed using the protocol Help:Protocols/Transformation
3. Fluorence Intensity Measurement
The Fluorescence Intensity was measured using the standardized protocol from iGEM Plate_Reader_Protocol_Update .
The Plate Reader (Fluostar omega, BMG LABTECH) was calibrated using the solutions included in the Interlab Measurement Kit.
4. Flow Cytometry
Fluorescence Intensity was measured using the Flow Cytometer (Attune NxT, Thermo Fisher Scientific) in cells grown in LB following guidelines from iGEM.                       The Flow Cytometer was calibrated using Sphero®Rainbow Calibration Particles (BD Bioscience), 8 peaks, calibrated for MEFL (Molecules of Equivalent Fluorescein). Four drops of calibration particles were dissolved in sheath fluid (1ml).                       Samples were prepared for measurement in the Flow Cytometer washing the culture media in filtered 1X PBS. Cells cultures were diluted 1:100, adding PSB in a 96-well microtiter plate (Thermofisher Scientific). The instrument was configured with a channel for GFP measurement with 488 nm laser and 530/30 filter
Thank you to our sponsors for their support
ABOUT US
We are the University of Edinburgh Overgraduate iGEM Team, competing in the new application track in iGEM 2016.
read more
LOCATION
School of Biological Sciences
The University of Edinburgh
King's Buildings
Edinburgh EH9 3JF,
United Kingdom
FOLLOW US ON
CONTACT US
Email: edigemmsc@ed.ac.uk
