<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_OG_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh OG/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>--&gt;
    </P><DIV class="container-fluid paddedNav"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse "><UL class="nav navbar-nav navbar-right"><LI class="sliding-middle-out dropdown"><A href="#" class="dropbtn">TEAM</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Edinburgh_OG/Team">About Us</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Team#attributions">Attributions</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Collaborations">Collaborations</A></DIV></LI><LI class="sliding-middle-out dropdown"><A href="#" class="dropbtn">PROJECT</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Edinburgh_OG/Description">Description</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Description#design">Design</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Experiments">Method</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Proof">Proof of Concept</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Results">Results</A></DIV></LI><LI class="sliding-middle-out dropdown"><A href="#" class="dropbtn">EXPERIMENTS</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Edinburgh_OG/Experiments/Cyanobacteria">Team Cyanobacteria</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Experiments/Rhodococcus">Team Rhodococcus</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Experiments/Fungi">Team Fungi</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Notebook">Notebook</A></DIV></LI><LI class="sliding-middle-out dropdown"><A href="https://2016.igem.org/Team:Edinburgh_OG/Parts" class="dropbtn">PARTS</A></LI><LI class="sliding-middle-out dropdown"><A href="https://2016.igem.org/Team:Edinburgh_OG/Safety" class="dropbtn">SAFETY</A></LI><LI class="sliding-middle-out dropdown"><A href="https://2016.igem.org/Team:Edinburgh_OG/Interlab" class="dropbtn">INTERLAB</A></LI><LI class="sliding-middle-out dropdown"><A href="https://2016.igem.org/Team:Edinburgh_OG/Human_Practices" class="dropbtn">HUMAN PRACTICES</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Edinburgh_OG/HP/Silver">Silver</A><A href="https://2016.igem.org/Team:Edinburgh_OG/HP/Gold">Gold</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Integrated_Practices">Integrated Practices</A></DIV></LI><LI class="sliding-middle-out dropdown"><A href="https://2016.igem.org/Team:Edinburgh_OG/Awards" class="dropbtn">AWARDS</A></LI></UL></DIV></DIV><SECTION class="bg-primary" id="summary"><DIV class="container"><DIV class="row"><DIV class="col-sm-12 text-center"><H2 class="section-heading">Description Summary</H2><DIV class="col-sm-4"><H3> Motivation</H3><H3><P class="text-faded" style="font-size: 15px">Earlier this summer, we have discussed to explore different kind of biotech applications using synthetic biology. Apparently, many of these applications were limited by the availability of suitable chassis for the particular process involved. Therefore, the Edinburgh Overgraduate iGEM team decided to expand the availability of chassis for applications which needs specialised host organism: (1) Sustainable production of biomaterials through photosynthesis, (2) degradation of various pollutants/toxic wastes, and (3) robust production of secondary metabolites.  
                      <MARK>Thus, we explore suitable organisms for the tasks and develop the means to do genetic manipulation and create a collection of necessary parts library.</MARK>In the process, we realized that by domesticating new organisms as chassis, we open up new risks and safety challenges towards users and environment. 
                      <MARK>Therefore, we also developed a tool (software) to screen for potential risk from the organismâ€™s database of toxic secondary metabolites.</MARK></P><CENTER><A href="https://static.igem.org/mediawiki/2016/0/08/T--Edinburgh_OG--Tom_Essay_WhyNeedMoreChassis.pdf" class="page-scroll btn btn-default btn-xl sr-button">An Essay on Expanding Chassis</A></CENTER></H3><H3> Our Chassis</H3><H3><DIV class="row"><DIV class="col-sm-4 text-center"><P><I>Synechocystis sp.</I></P></DIV><DIV class="col-sm-4 text-center"><P><I>Rhodococcus jostii</I> RHA1</P></DIV><DIV class="col-sm-4 text-center"><P><I>Penicillium roqueforti</I></P></DIV></DIV><P><B>Synechocystis sp.</B> is one of the most extensively studied cyanobacterial species, a unique class of microorganisms that are able to fix and metabolize carbon dioxide using the energy derived from sunlight. They are potential chassis for synthetic biology application in biomass production and carbon sequestration.</P><P><B>Rhodococcus jostii</B> is a gram positive bacteria with extensive catabolic pathway to degrade variety of chlorinated compounds, such as polychlorinated biphenyls (PCBs). They are potential chassis for synthetic biology applications in bioremediation.</P><P><B>Penicillium roqueforti</B> is a filamentous fungi used in the production of blue cheese. They are biotechnologically relevant for the industrial production of enzymes (cellulases, pectinases, lipases, proteases and amylases) and could be used to produce complex secondary metabolites.</P><CENTER><A href="https://2016.igem.org/Team:Edinburgh_OG/Experiments" class="page-scroll btn btn-default btn-xl sr-button">Find out more!</A></CENTER></H3><H3>Golden Gate MoClo</H3><H3><P class="text-faded" style="font-size: 15px">Central to synthetic biology is the use of systematic assembly and sharing of parts between community members. As working in new chassis will emphasizes on characterisation of suitable parts and trial-error approach, we chose to adopt Golden Gate MoClo as our assembly standard. This approach enables the assembly of multiple parts and transcriptional units by parallel approach via BpiI (BbsI) in different levels (Engler et al., 2008; Weber et al., 2011).
                      To utilise MoClo, a set of destination vectors need to be developed to accommodate assembly in different levels. Unfortunately, the chassis we were working on (R. jostii) cannot utilise the origin of replication from available destination vectors, which are designed for Eschericia coli. Therefore, we developed a set of MoClo destination vectors for assembly and transformation to R. jostii. We have shown that our destination vectors can be used for combinatorial assembly and transformed to the host organism. 
                      </P><CENTER><A href="https://2016.igem.org/Team:Edinburgh_OG/Description#design" class="page-scroll btn btn-default btn-xl sr-button">Vector Design</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Proof" class="page-scroll btn btn-default btn-xl sr-button">Proof of Concept</A></CENTER></H3><H3>Developed Parts (Phytobricks)</H3><H3><P class="text-faded" style="font-size: 15px">To utilise synthetic biology, a working parts (promoters, RBS, coding sequences) need to be developed as not all of them works orthogonally. We have developed a set of useful parts for expression in our chassis.                     
                      </P><CENTER><A href="https://2016.igem.org/Team:Edinburgh_OG/Parts" class="page-scroll btn btn-default btn-xl sr-button">Find out more!</A></CENTER></H3><H3>Human Practices</H3><H3><P class="text-faded" style="font-size: 15px">The domestication of non-model organisms raises both biosafety and biosecurity challenges (e.g. unknown pathogenicity and toxicity). Such issues, if are not properly addressed, can be a risk for both users and their surroundings. The Edinburgh OG team worked to develop an accessible, easy-to-use program to evaluate the toxicity of curated secondary metabolites from organisms used as chassis. We have incorporate this aspect in our project and communicate with other teams in order to give contribution towards current risk assessment procedures and as precautionary step for both experienced and non-experienced users.                   </P><CENTER><A href="https://2016.igem.org/Team:Edinburgh_OG/Description#design" class="page-scroll btn btn-default btn-xl sr-button">Find out more!</A></CENTER></H3></DIV></DIV></DIV></DIV></SECTION><SECTION id="introduction_2"><DIV class="container"><DIV class="row"><DIV class="col-sm-8" style="background-color:white"><H2> Project Description <P> The continuously growing field of industrial biotechnology has incited the replacement of non-renewable processes with more energy-efficient ones. Central to this is the use of genetically modified organisms modified with recombinant DNA technology for a wide variety of applications, ranging from the production of fine chemicals and pharmaceuticals to fuels and bioremediation [1]. These technologies and applications are largely dependent on the use of well-characterised industrial host organisms including Escherichia coli and Sacharomyces cerevisiae. However, such strains are not always the optimal choice for particular processes due to inherent metabolic limitations or a lack of consensus between the engineered components introduced into the host from a heterologous organism for the particular bioprocess.</P><P> While the well-characterised E. coli and S. cerevisiae have a host of molecular biology tools available for engineering them since they have been domesticated (they have been vastly changed from their wild-type and adapted to laboratory conditions), they come with distinct limitations, such as the inability to make the post-translational modifications found in eukaryotic proteins, with a high potential for protein misfolding, aggregation and degradation, besides incomplete translation due to different patterns of codon usage to eukaryotes [2]. Using native producer organisms in biotechnology may lead to higher productivities, but these non-model organisms often lack well-characterised genetic engineering tools, which may present difficulties in optimising and controlling metabolic pathways. This presents a major barrier to the widespread use of these organisms as biofactories.</P><P> The Edinburgh Overgraduate iGEM team will address those challenges by domesticating the desired host strains, applying synthetic biology (synbio) genetic engineering standards to the design of protein expression and metabolic engineering systems in a number of non-model organisms and characterising the behaviour of these standardised systems in a number of host strains with high potential for industrial applications. Furthermore, although it has its limitations in bacteria, we will include the CRISPR-Cas9 platform to assess its utility in these organisms as it is a potentially useful tool to have a better control of genomic perturbations and maximise those optimal expression systems and metabolic pathways. Through our project, the path is being paved so that other participants in the synbio community will more easily access these organisms in the laboratory, accelerating the understanding of these organisms and increasing the list of strains that are suitable for the manufacture of relevant products, including those that address global conservation challenges. </P></H2></DIV></DIV></DIV></SECTION><SECTION id="design"><DIV class="container"><DIV class="row"><DIV class="col-sm-8" style="background-color:white"><H2> Design of the Golden Gate Destination Vectors for Transformation in R. jostii </H2><P> We designed a set of Golden Gate destination vectors based on the MoClo standard to enable quick and easy way to create constructs and transformed it to our new Chassis, Rhodococcus jostii RHA1. We chose to use pSRKBB vector (Schmidt-dannert, 2014), which was derived from pSRK21 (Vesely et al., 2003) as the base of these destination vectors. This plasmid contains Corynebacterium replicon (pSR1) and has been successfully transformed to R. erythropolis, a closely related species of R. jostii (KnoppovÃ¡ et al., 2007; Vesely et al., 2003; Archer &amp; Sinskey, 1993). It has also been refined by the removal of illegal sites incompatible with BioBrick standard (Schmidt-dannert, 2014). We removed the illegal sites (BsaI and BbsI) and replace the cloning site of the plasmid with a standard BioBrick cloning site flanked by two verification primer binding site (VF2 and VR), enabling colony PCR for quick screening of insert, derived from pSB1C3. The resulting vector (pSRKM_Empty) can be used as the template to make Level-1 destination vectors by digestion and ligation of standardized LacZÎ± reporter flanked by fusion sites as described in Iverson et al. (2016).
</P><FIGURE><FIGCAPTION>Fig1. (A) Physical map of pSRKBB-Empty showing pSR1 replicon as a black band. The replicon contains per (a positive effector of replication) and rep (replicase) gene. pK19 backbone (white band) contains pBR322 ORI and Kanamycin resistance gene (KmR) regulated by the P45 promoter. Mutagenic primer binding sites are shown as red arrows (mut1, mut2, mut3, and mut4). The region in the red box was replaced by (B) a BioBrick cloning site flanked by standard verification primer binding sites, VF2 and VR, derived from Iverson et al. (2016), creating pSRKM_Empty. Destination vectors can be made by digestion and ligation with SpeI to insert standardized LacZÎ± flanked by the designated Type IIs restriction enzyme (recognition sites shown as bold) and four base overhangs for Level-1 (C) or Level-2 (D) assembly. (E) Schematic map of the level-1 destination vector pSRKM_NN and (F) the level-2 destination vector pSRCM_NN with chloramphenicol resistance (CmR). X refers to the fusion sites and NNNN to its four base overhangs: A (GGAG), B (TACT), C (AATG), D (AGGT), E (GCTT), F (CGCT), G (TGCC), and H (ACTA).</FIGCAPTION></FIGURE></DIV></DIV></DIV></SECTION><SECTION id="head"><DIV class="container" style="background-color:white"><DIV class="row"><DIV class="col-sm-10" style="background-color:white"><H2> References </H2><DIV class="newspaper"><P>[1] Nielsen, J., &amp; Jewett, M. C. (2008). Impact of systems biology on metabolic engineering of Saccharomyces cerevisiae. FEMS Yeast Research, 8(1), 122â€“131. http://doi.org/10.1111/j.1567-1364.2007.00302.x </P><P>[8] Weber, T., Blin, K., Duddela, S., Krug, D., Kim, H. U., Bruccoleri, R., â€¦ Medema, M. H. (2015). antiSMASH 3.0--a comprehensive resource for the genome mining of biosynthetic gene clusters. Nucleic Acids Research, 43(May), 1â€“7. http://doi.org/10.1093/nar/gkv437 </P><P>[9] Marles-Wright, J. (2016). Better by Design, Safer through Practice. Synbio LEAP Strategic Action Plan, (January), 1â€“5. </P><P> Archer, J. A. &amp; Sinskey, A. J. 1993. The DNA sequence and minimal replicon of the Corynebacterium glutamicum plasmid pSR1: evidence of a common ancestry with plasmids from C. diphtheriae. J Gen Microbiol, 139, 1753-9. </P><P> Iverson, S. V., Haddock, T. L., Beal, J. &amp; Densmore, D. M. 2016. CIDAR MoClo: Improved MoClo Assembly Standard and New E. coli Part Library Enable Rapid Combinatorial Design for Synthetic and Traditional Biology. ACS Synthetic Biology, 5, 99-103. </P><P> KnoppovÃ¡, M., Phensaijai, M., VeselÃ½, M., ZemanovÃ¡, M., NeÅ¡vera, J. &amp; PÃ¡tek, M. 2007. Plasmid Vectors for Testing In Vivo Promoter Activities in Corynebacterium glutamicum and Rhodococcus erythropolis. Current Microbiology, 55, 234-239. </P><P> Schmidt-dannert, C. 2014. pSRKBB-empty (Plasmid #59449). www.addgene.org. </P><P> Vesely, M., Patek, M., Nesvera, J., Cejkova, A., Masak, J. &amp; Jirku, V. 2003. Host-vector system for phenol-degrading 
Rhodococcus erythropolis based on Corynebacterium plasmids. Appl Microbiol Biotechnol, 61, 523-7. </P></DIV></DIV></DIV></DIV></SECTION><SECTION style="background-color: white" id="footer-sec2"><DIV style="padding: 10px;"><H3 style="text-align:center; font-size:20px;">Thank you to our sponsors for their support</H3></DIV></SECTION><SECTION id="footer-sec"><DIV class="container" style="margin-top:0px; background-color: #262626; color: #c6c6c6"><DIV class="row  pad-bottom"><DIV class="col-md-3"><H4><STRONG>ABOUT US</STRONG></H4><P style="text-align:center">
                    We are the University of Edinburgh Overgraduate iGEM Team, competing in the new application track in iGEM 2016.
                    <A href="//2016.igem.org/Team:Edinburgh_OG/Team">read more</A></P></DIV><DIV class="col-md-3"><H4><STRONG>LOCATION</STRONG></H4><P style="text-align:center">
                    School of Biological Sciences 
                    The University of Edinburgh 
                    King's Buildings 
                    Edinburgh EH9 3JF,  
                    United Kingdom </P></DIV><DIV class="col-md-3"><H4><STRONG>FOLLOW US ON</STRONG></H4></DIV><DIV class="col-md-3"><H4><STRONG>CONTACT US</STRONG></H4><P style="text-align:center">
                    Email: <A href="edigemmsc@ed.ac.uk" target="_blank">edigemmsc@ed.ac.uk</A></P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>