<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_OG_Experiments_Rhodococcus skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh OG/Experiments/Rhodococcus</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>--&gt;
    </P><DIV class="container-fluid paddedNav"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse "><UL class="nav navbar-nav navbar-right"><LI class="sliding-middle-out dropdown"><A href="#" class="dropbtn">TEAM</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Edinburgh_OG/Team">About Us</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Team#attributions">Attributions</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Collaborations">Collaborations</A></DIV></LI><LI class="sliding-middle-out dropdown"><A href="#" class="dropbtn">PROJECT</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Edinburgh_OG/Description">Description</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Description#design">Design</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Experiments">Method</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Proof">Proof of Concept</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Results">Results</A></DIV></LI><LI class="sliding-middle-out dropdown"><A href="#" class="dropbtn">EXPERIMENTS</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Edinburgh_OG/Experiments/Cyanobacteria">Team Cyanobacteria</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Experiments/Rhodococcus">Team Rhodococcus</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Experiments/Fungi">Team Fungi</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Notebook">Notebook</A></DIV></LI><LI class="sliding-middle-out dropdown"><A href="https://2016.igem.org/Team:Edinburgh_OG/Parts" class="dropbtn">PARTS</A></LI><LI class="sliding-middle-out dropdown"><A href="https://2016.igem.org/Team:Edinburgh_OG/Safety" class="dropbtn">SAFETY</A></LI><LI class="sliding-middle-out dropdown"><A href="https://2016.igem.org/Team:Edinburgh_OG/Interlab" class="dropbtn">INTERLAB</A></LI><LI class="sliding-middle-out dropdown"><A href="https://2016.igem.org/Team:Edinburgh_OG/Human_Practices" class="dropbtn">HUMAN PRACTICES</A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:Edinburgh_OG/HP/Silver">Silver</A><A href="https://2016.igem.org/Team:Edinburgh_OG/HP/Gold">Gold</A><A href="https://2016.igem.org/Team:Edinburgh_OG/Integrated_Practices">Integrated Practices</A></DIV></LI><LI class="sliding-middle-out dropdown"><A href="https://2016.igem.org/Team:Edinburgh_OG/Awards" class="dropbtn">AWARDS</A></LI></UL></DIV></DIV><SECTION class="bg-primary" id="about" style="background-color: #c92222"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2 text-center"><H2 class="section-heading">Introduction to <I>Rhodococcus sp.</I></H2><DIV class="col-sm-6"><P class="text-faded" style="font-size: 15px">

In the last three decades, there have been a number of patents and publications in the usage of several Rhodococcus strains as bioremediation and biotransformation agents.  These gram-positive actinobacteria are commonly found in contaminated soils and toxic environments (Larkin et al., 2005).  Along with the gram-negative Pseudomonas, they tolerate a range of organic solvents, exhibit unique enzymatic capabilities and have the ability to biodegrade several environmental pollutants.  They have been used to bioremediate soil contaminated with hydrocarbons (Warhurst and Fewson, 1994), pesticides (Larkin et al., 2005), nitriles (Brandao and Bull, 2003) and xenobiotics (Martinkova et al., 2009).  They have also been used to produce acrylamide (Hughes et al., 1998), triacylglycerols (Hernandez et al., 2008) and in fossil fuel desulfurization (Matsui et al., 2002).

                  </P></DIV></DIV><DIV class="row" style="color: white"><DIV class="col-lg-10 col-lg-offset-1 text-center"><DIV class="newspaper"><P> There are currently 58 different Rhodococcus species (http://www.bacterio.net/rhodococcus.html) and based on 16S rRNA data (McMinn et al., 2000) they are classified into three main subclades: R. erythropolis, R. equi and R. rhodochrous.  Of the R. erythropolis subclade, the three most characterised industrial species are R. erythropolis, R. opacus, and R. jostii (Goodfellow et al., 2012).   R. erythropolis was the first characterised bacteria in this subclade while R. jostii was the first to have its genome sequenced (McLeod et al., 2006).  R. opacus has the unique ability of accumulating up to 40% of its dry cell mass as triacylglycerols (Kurosawa et al., 2010).  All three species catabolise a wide range of oligosaccharides and organic compounds, but R. erythropolis has two industrial advantages of tolerating a wider range of temperatures and possessing a significantly smaller genome, corresponding to faster growth rate.  However, the larger genomes of R. jostii and R. opacus gives them the ability to metabolise a wider range of organic substances.
</P><P><B>Current genetic tools</B></P><P>Studies in the 1990s and 2000s showed that Rhodococcus could harbor cryptic or linear plasmids.  Linear plasmids tend to be relatively large and are of the inverton type (Stecker et al., 2003).  Several characteristic linear plasmids were linked to catabolic gene clusters.  Studies on R. erythropolis showed that pBD2 - a 210 kb linear plasmid - carries the genes responsible for the oxidation of isopropylbenzene (Stecker et al., 2003).  Another replicon, IGTS8 (150 kb) carries the genes responsible for fossil fuel desulfurization (Denis-Larose et al., 1997).  In R. jostii, three linear replicons - pRHL1 (1,100 kb), pRHL2 (450 kb), and pRHL3 (330 kb) – were predicted to carry the genes responsible for the catabolism of trichloroethene, naphthalene, alkylbenzene, biphenyl and choroaromatic compounds (McLeod et al., 2006).

Several cryptic plasmids were isolated from Rhodococci and their replication origins were used to construct plasmid-borne genetic tools.  Prominent examples include pKA22 and pRE8424.  pKA22 was the first sequenced cryptic plasmid from Rhodococcus (Kulakov et al., 1997).  It is a 5 kb circular plasmid with theta-type replication mechanism.  Its theta-type replication origin is unusual in being a broad host range replicon that replicates in Rhodococcus strains B264-1, I24 and erythropolis SQ1 (Lessard et al., 2004).  A very similar variant, pB264, was modified to create pAl298, a temperature sensitive shuttle vector tool for Rhodococcus.  The second cryptic plasmid, pRE8424 from R. erythropolis DSM8424 replicates through rolling-circle-type mechanism in a way that is similar to the pIJ101/pJV1 family of rolling circle plasmids (Khan 1997).  It was modified and used to create one of the most widely used Rhodococcus inducible expression vector, pTip.  Nakashima and Tamura (2004) cloned a thiostrepton-inducible promoter - Ptip from Streptomyces - and attached it upstream of a MCS.  Then, they have cloned in a thiostrepton-resistance cassette and Ptip transcription activator genes in the same pTip vector.  Afterwards, they have showed that recombinant proteins that cannot be expressed in E. coli could be inserted into this MCS and get expressed in Rhodococcus with yields up to 10 mg.  This class of vectors are the subject of a patent (US patent number 7709624) that gathered interest and showed the scientific community that Rhodococcus is a good expression system for proteins that do not express in E. coli or require lower expression temperatures (Nakashima and Tamura, 2010). 

Based on pTip vectors, several genetic tools were developed.  Sallam and colleagues (2006) created pTNR, a transposon vector for mutagenesis and random gene insertion applications.  Another system is pIcl and its derivative, pCpi.  pIcl is a methanol-induced expression system that is useful in industrial applications where the organic solvent is the inducer of expression.  pCpi is a mutated version of pIcl that shows a strong constitutive expression through of as yet an unknown mechanism (Kagawa et al., 2012).
Another plasmid class is pSRK21 and its derivatives.  pSRK21 is a hybrid of pSR1 and pGA1, two cryptic plasmids from Corynebacterium.  Its parental plasmids replicate through a specific rolling-circle-type mechanism (Osborn et al., 2000).  This construct was reported to replicate in E. coli, R. erythropolis, R. opacus (Vesely et al., 2003) and our study shows that it also replicates in R. jostii.  Claudia Schmidt-Dannert modified it to be BioBrick compatible (pSRKBB) and was used as the plasmid backbone in this study.  </P><P><B>ErmE Promoter Phytobrick</B></P><P> A widely used constitutive promoter, ErmE (promoter region of erythromycin-resistance gene) was isolated from Streptomycin (Bibb et al., 1985). It works with related gram positive actinobacteria, such as Rhodococcus. </P><P><B>P-45 Promoter Phytobrick</B></P><P> A strong constitutive promoter from Corynebacterium (Patek et al., 1996) that works with related gram positive actinobacteria, such as Rhodococcus. </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="Chassis"><DIV class="container" style="background-color: #f9ffa5"><DIV class="row"><DIV class="col-lg-8 text-center"><H2 class="section-heading">Experiments and Protocols</H2><H3 class="section-subheading text-muted">This is a record of protocol and optimization that we did</H3></DIV></DIV><DIV class="row text-center"><DIV class="col-md-3"><H4 class="service-heading">1. Competent cells</H4><P class="text-muted">Competent cells were obtained following the iGEM protocol
                      <A href="http://parts.igem.org/Help:Protocols/Competent_Cells">
                        Help:Protocols/Competent Cells </A></P></DIV><DIV class="col-md-3"><H4 class="service-heading">2. Transformation </H4><P class="text-muted">E. coli DH5α was transformed using the protocol
                      <A href="http://parts.igem.org/Help:Protocols/Transformation">
                        Help:Protocols/Transformation </A></P></DIV><DIV class="col-md-3"><H4 class="service-heading"><I>3. Fluorence Intensity Measurement</I></H4><P class="text-muted"> The Fluorescence Intensity was measured using the standardized protocol from iGEM
                      <A href="https://static.igem.org/mediawiki/2016/c/c5/InterLab_iGEM2016_Plate_Reader_Protocol_Updated_July.pdf">
                        Plate_Reader_Protocol_Update </A>.  The Plate Reader (Fluostar omega, BMG LABTECH) was calibrated using the solutions included in the Interlab Measurement Kit.
                  </P></DIV><DIV class="col-md-3"><H4 class="service-heading"><I>4. Flow Cytometry</I></H4><P class="text-muted">Fluorescence Intensity was measured using the Flow Cytometer (Attune NxT, Thermo Fisher Scientific) in cells grown in LB following guidelines from iGEM.
                      The Flow Cytometer was calibrated using Sphero®Rainbow Calibration Particles (BD Bioscience), 8 peaks, calibrated for MEFL (Molecules of Equivalent Fluorescein). Four drops of calibration particles were dissolved in sheath fluid (1ml).
                      Samples were prepared for measurement in the Flow Cytometer washing the culture media in filtered 1X PBS. Cells cultures were diluted 1:100, adding PSB in a 96-well microtiter plate (Thermofisher Scientific). The instrument was configured with a channel for GFP measurement with 488 nm laser and 530/30 filter
                </P></DIV></DIV></DIV></SECTION><SECTION style="background-color: white" id="footer-sec2"><DIV style="padding: 10px;"><H3 style="text-align:center; font-size:20px;">Thank you to our sponsors for their support</H3></DIV></SECTION><SECTION id="footer-sec"><DIV class="container" style="margin-top:0px; background-color: #262626; color: #c6c6c6"><DIV class="row  pad-bottom"><DIV class="col-md-3"><H4><STRONG>ABOUT US</STRONG></H4><P style="text-align:center">
                    We are the University of Edinburgh Overgraduate iGEM Team, competing in the new application track in iGEM 2016.
                    <A href="//2016.igem.org/Team:Edinburgh_OG/Team">read more</A></P></DIV><DIV class="col-md-3"><H4><STRONG>LOCATION</STRONG></H4><P style="text-align:center">
                    School of Biological Sciences 
                    The University of Edinburgh 
                    King's Buildings 
                    Edinburgh EH9 3JF,  
                    United Kingdom </P></DIV><DIV class="col-md-3"><H4><STRONG>FOLLOW US ON</STRONG></H4></DIV><DIV class="col-md-3"><H4><STRONG>CONTACT US</STRONG></H4><P style="text-align:center">
                    Email: <A href="edigemmsc@ed.ac.uk" target="_blank">edigemmsc@ed.ac.uk</A></P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>