Team:Edinburgh OG/Experiments/Fungi
-->
Toggle navigation
TEAM
About Us Attributions Collaborations
PROJECT
Description Design Method Proof of Concept Results
EXPERIMENTS
Team Cyanobacteria Team Rhodococcus Team Fungi Notebook
PARTS
SAFETY
INTERLAB
HUMAN PRACTICES
Silver Gold Integrated Practices
AWARDS
Introduction to Penicillium sp.
Penicillium roqueforti is perhaps the most widely recognised species of the Penicillium genus due its namesake cheese, Roquefort. P. roqueforti is globally employed in production of blue cheeses, including Gorgonzola, Stilton and Danish Blue (Goarin et al., 2015). Demand is large and increasing, with over 50,000 tons of blue veined cheeses produced each year in France alone (Ropars et al., 2012). Global market value has been predicted to reach 370 million USD by 2021 (Larsen et al., 1998). Unlike other cheese fungi, P. roqueforti is found across a wide range of environments. It is naturally resistant to cold temperatures, low oxygen levels and can thrive in a range of pH conditions (Goarin et al., 2015). These characteristics present P. roqueforti as a source of significant detrimental economic impact. It is a common spoilage agent of meat, silage, wheat, wood and refrigerated stored foods and the most widespread contaminant of baked goods (Ropars et al., 2012; Cheeseman et al., 2014; Lavermicocca et al., 2003).
The utility of Penicillium roqueforti as a starter culture for the production of many cheeses comes from an ability to effectively secrete a range of proteolytic and lipolytic enzymes. P. roqueforti also produces numerous bioactive secondary metabolites such as meroterpenoids mycophenolic acid (MPA) and andrastin A, which have potential as immunosuppressants and anti-cancer agents (Fontaine et al., 2015; Chang et al., 1993). P. roqueforti also produces mycotoxins such as the indole alkaloid and potent neurotoxin Roquefortine C, the PR-toxin and several other poorly characterised metabolites (Frisvad 2004). PR-toxin is potentially lethal and has been demonstrated to damage mammalian lungs, heart, liver and kidney tissue as well as inducing abortion (Matsuda et al., 2013; Chen et al., 1982; Hidalgo et al., 2014).
Despite its high economic importance and health implications P. roqueforti remains poorly understood. Cheeses ripened with P. roqueforti often contain bioactive metabolites and toxins, although at levels regarded as safe for the consumer (Fontaine et al., 2015; Finoli et al., 2001). Assumption of safety is primarily based on historical use and the European Food Safety Authority has determined P. roqueforti as unfit for Qualified Presumption of Safety due to a lack of validated analytical methods for the detection of mycotoxins under production conditions (Barlow et al., 2007). Further, the report cites a lack of data on possible long-term toxic effects and regards toxicological data for P. roqueforti metabolites as ‘insufficient to set a threshold for regulatory purposes’ (Barlow et al., 2007). Difficulties in determining safety also stem from poor understanding of mechanisms behind mycotoxin production and propensity of P. roqueforti to significantly alter secondary metabolite production in response to environmental changes (Barlow et al., 2007).
In 2014 the complete genome sequence of P. roqueforti FM164, became available allowing fundamental insights into the species (Cheeseman et al., 2014). Initial work has focused on elucidation of mechanisms involved in PR toxin production and other metabolites (Hidalgo et al., 2014). Comparative genomic analysis revealed multiple recent horizontal transfers of very large genomic islands (Cheeseman et al., 2014). Horizontal gene transfer (HGT) in eukaryotes is not commonly regarded as significant or frequent. Cheeseman et al (2014) add to the growing evidence of a wider impact of HGT in fungi, with implications in fundamental eukaryotic biology and practices within agriculture, biotechnology and the management of pathogens. Penicillia provide an ideal model for study of eukaryotic adaptation due to high selective pressure enforced by industrial use. Ropars et al (2016) suggest P. roqueforti as a particularly interesting candidate for study of intraspecies adaption due to presence in a large diversity of niches. Further, recent identification of sexual reproduction in P. roqueforti highlights a lack of fundamental understanding and opens potentially useful avenues for diversifying industrial uses (Ropars et al., 2012).
Genetic exploration of P. roqueforti will further understanding of fungal genomics and yield economic, environmental and health benefits. With the recent demonstration of gene silencing, availability of full genome data and transformation protocols several foundations to support further study are available (Durand et al., 1991; Goarin et al., 2015). Additionally, despite mycotoxin production P. roqueforti is ‘generally recognised as safe’ (GRAS) by the US Food and Drug Administration (2015). For these reasons it was chosen as one of the non-model organisms to explore as part of the ExpandED project.
Experiments and Protocols
This is a record of protocol and optimization that we did
1. Part design
2. Fungi culture
3. Golden Gate Assembly
4. Fungi Transformation
Methodology
We know that a main constraint for working with non-model organism is the lack of genetic tools but also sometimes basic data such as growth curves, standardized protocols and carbon requirements to mention some. For that reason, the methodology of this research was divided into two main stages. The first was the development of level 2 parts that consisted in the selection of parts from the iGEM distribution kit, obtaining the P444 and P480 plasmids, primer design and validation, gradient PCR to include MoClo overhangs and Golden Gate assembly for level 0, 1 and 2. The second stage was mostly done in P. roqueforti, including morphological and growth characterisation in different media, protoplast generation and assessing its viability in several conditions. Later, the designed level 2 parts from the first step were used to transform the fungus though two methods, PEG-mediated transformation and electroporation. Finally, the expression was measured with fluorescence-based reporters
1. Parts design
In order to expand the list of parts available for filamentous fungi a basic toolbox that allows rapid assembly of transcriptional units is proposed.
- The basic toolbox includes one constitutive and one inducible promoter. Glyceraldehyde-3-phosphate dehydrogenase (GPDA) is an enzyme responsible of catalysing important steps in glycolysis. The gene gpdA is constitutively expressed in several filamentous fungi but the sequence was first isolated in Aspergillus nidulans. This promoter has been previously used for expression of human interleukin-6 in Aspergillus niger.
- As inducible promoter we propose the PglaA. The Aspergillus α-glucoamylase (GlaA) enzyme is involved in the metabolism of starch. The glaA promoter is induced by starch, maltose, and low concentrations of glucose but repressed by xylose. Both promoters will be characterised through the expression of fluorescent proteins (GFP, RFP and iLOV).
-Also, as part of the 2016 Edinburgh OG iGEM team vision of facilitating the domestication of non-model organisms, two versions of a “universal reporter” sequence were generated. The sequence is based on the iLOV fluorescent protein gene and was codon-de-optimised with an algorithm considering the codon bias of each of our selected non-model organisms. The functionality of design of these “universal reporters” will be compared with the original iLOV from the registry.
-Filamentous fungi are naturally resistant to most of the common antibiotics used as selective markers for bacteria. Therefore, we propose to use phleomycin as selective marker. Phleomycin resistance is transferred by the ble gene from Streptomyces verticillus. The transcriptional unity will be designed with the PgdaA promoter and the Tcyc, a terminator commonly used in Saccharomyces cerevisiae
.
Primer design to include the new Phytobrick standard to the proposed parts.
The design also includes the BsmBI (Type IIS restriction enzyme) recognition site, two nucleotides on the 5’ end and one before the cutting site of the enzyme that can be modified by design to balance the GC content . To keep the coding sequences within the reading frame, the native ATG codon was removed since the “C” fusion sites already includes it. The annealing part of the primers was kept between 18-26 bp and the %GC and Tm of the whole primers were around 35-50% and 54-60 °C respectively. The PglaA promoter was send for synthesis to IDT with the restriction sites included as well as the respective fusion sites; E at the 5’ end and C at the 3’ end
.
PCR to add MoClo overhangs.
Gradient PCR was set up with in order to add the MoClo overhangs to PglaA, Tcyc, Phleomycin resistance cassette, GFP, RFP and iLOV with the designed primers (Table 3). PglaA, Tcyc and the Phleomycin resistance cassette were amplified directly from the P444 plasmid. PCR master mix was prepared with all the components excluding the forward and reverse primers, which were added lastly and directly to the 0.2 mL tube to avoid primer dimerization.
 
   
Component
   
Volume uL
 
Nuclease-Free water
13.5
5x Q5 Reaction Buffer
5.0
<
10 mM dNTPs
0.50
10 mM Forward primer
1.25
10 mM Reverse primer
1.25
DMSO
2.50
DNA template
0.50
Q5 High-Fidelity DNA polymerase
0.25
Total Volume
25
A 15°C temperature gradient (55-70 °C) with increments of 1.25 °C for a total gradient of 12 PCR tubes was implemented in the annealing step. The PCR conditions used were: 98°C initial denaturing step for 30 seconds followed by 30 cycles of denaturation at 98 °C for 10 seconds, annealing at 55-70 °C for 30 seconds and extension at 72 °C, time varied depending on the part; 25 seconds for Tcyc and iLOV, 35 seconds for GFP, RFP and PgdaA and 45 seconds for the Phleomycin resistance cassette. To finish, a final extension was conducted at 72 °C for 2 minutes
PCR samples were analysed by electrophoresis in a 0.8% agarose gel at 100V for 50 minutes. PCR reactions showing the expected bands were subjected to clean-up with the QIAquick PCR Purification Kit (Qiagen). DNA concentrations were determined with the NanoDrop 2000 (Thermofisher)
.
2. Fungi Culture
Penicillium shows viable growth on many mediums with several providing specific utility. Potato dextrose medium is commonly recommended for propagation and lower mycotoxin production levels have been observed on this medium compared to some alternatives in P. roqueforti culture (Hidalgo et al., 2014). Potato dextrose broth (PDB) and potato dextrose agar (PDA) were used for liquid and plate culture respectively and made to supplier recommendations.
M3 medium is a nutrient rich medium commonly used in insect culture and provides a suitable environment for regeneration of chitin rich cell walls following protoplast transformation procedures (Goarin et al., 2015). To generate selective M3 medium, phleomycin was added to 50 µg/mL, as recommended by Durand et al. (1991).
PDA   ....................................     M3
3. Golden Gate Assembly
Hierarchical assembly to generate LV2 constructs using MoClo.
To assemble our constructs we used the MoClo assembly method from LV0 parts until LV2 constructs. Each construct level was cloned into e.coli, digested for confirmation and sequenced.
This are the example of the LV2 plasmids that we intended to assembly, unfortunately we faced some setbacks due to the low efficiency and false positives in the Golden Gate Asembly (MoClo)
Golden gate assembly for LV0 parts.
The restriction-ligation of Level 0 was set up by adding in a single tube 30 fmol of each DNA component (PCR purified reactions and the PglaA promoter synthesised by IDT), 30 fmol of the destination vector (Universal Acceptor P10500), 10 units of BsmBI (NEB), 20 units of T4 ligase (NEB), 1X T4 ligase buffer (NEB) and water to a final volume of 25 µL. The reaction was incubated in a thermocycler with the following program: an initial incubation at 37 °C for 20 minutes followed by 20 cycles of 37°C for 90 seconds and 16°C for 3 minutes, one cycle of 50°C for 5 minutes and one at 80 °C for 10 minutes. The universal acceptor with no insert was used as negative control.
Ten microliters of the digestion-ligation reaction were transformed into 100 µL of chemically competent Top 10 cells by the heat-shock method. After 90 minutes’ recovery with 1 mL of SOC medium the cells were spin down, resuspended in 100 µL of media and plated into Chloramphenicol (34 µg/ml) + X-gal (200 µg/ml) + IPTG (1mM). The plates were incubated overnight at 37 °C. On the next day the number of white/blue colonies were counted and the efficiency of the protocol was determined.
Four single white colonies of each new LV0 assembly were used to inoculate 5 ml overnight liquid culture (LB + Chloramphenicol). All plasmids were purified with the modified QIAprep Spin Miniprep kit (Qiagen). DNA concentrations were measured with the NanoDrop 2000 (Thermofisher). The LV0 assembly was confirmed by digestion; 150-200 ng of each LV0 were incubated with 0.5 µL BsaI (NEB) with 1X CutSmart buffer (NEB) in a 20 µL reaction. The digestions were incubated at 37°C for 2 hours and the enzyme was heat inactivated at 65°C for 5 minutes. Digested and undigested samples were analysed by electrophoresis in a 0.8% agarose gel at 110V for 45 minutes.
We successfully developed LV0 parts!.
Despite the setbacks we successfully assemble LV0 parts that will contribute to the available tools for filamentous fungi. See our results!
.
Agarose Gel Electrophoresis of digested LV0 constructs.  Well: (1, 13) 1 kb ladder (molecular weight marker). (2) GFP in P10500 plasmid digested with BsaI. (3) Undigested GFP in P10500 plasmid. (4) RFP in P10500 plasmid digested with BsaI. (5) Undigested GFP in P10500 plasmid. (6) Tcyc terminator in P10500 plasmid digested with BsaI. (7) Undigested Tcyc terminator in P10500 plasmid. (8) PgdaA promoter in P10500 plasmid digested with BsaI. (9) Undigested PgdaA promoter in P10500 plasmid. (10) PglaA promoter in P10500 plasmid digested with BsaI. (11) Undigested PglaA promoter in P10500 plasmid. (12) Phleomycin resistance cassette in P10500 plasmid digested with BsaI.
Golden gate assembly for LV1 and LV2 parts.
The restriction-ligation of Level 1 was set up by adding in a single tube 60 fmol of each of the DNA component (purified DNA of the confirmed LV0 including the 2 de-optimized versions of iLOV), 30 fmol of the destination vector pSRKM_AE (provided by Matin Nahuada, iGEM ED OG) or DVK_EF, 10 units of BsaI (NEB), 20 units of T4 ligase (NEB), 1X T4 ligase buffer (NEB) and water to a final volume of 25 µL. The reaction was incubated in a thermocycler with the following program: 15 cycles of 37°C for 90 seconds and 16°C for 3 minutes followed by one cycle of 50°C for 5 minutes and one at 80 °C for 10 minutes. The pSRKM_AE and DVK_EF with no insert were used as negative control.
Fifteen microliters of the digestion-ligation reaction were transformed into 100 µL of chemically competent Top 10 cells by the heat-shock method. After 120 minutes’ recovery with 1 mL of SOC medium the cells were spin down, resuspended in 100 µL of medium and plated into LB + Kanamycin (50 µg/ml) + X-gal (200 µg/ml) + IPTG (1mM). The plates were incubated overnight at 37 °C, on the next day the number of white/blue colonies were counted and the efficiency of the protocol was determined.
Two single white colonies of each new LV0 assembly were used to inoculate 5 ml overnight liquid culture (LB + Kanamycin). All plasmids were purified with the modified QIAprep Spin Miniprep kit (Qiagen). DNA concentrations were measured with the NanoDrop 2000 (Thermofisher). The LV1 assembly was confirmed by digestion; 150-200 ng of each LV1 were incubated with 0.5 µL BbsI (NEB) with 1X 2.1 buffer (NEB) in a 20 µL reaction. The digestions were incubated at 37°C overnight and the enzyme was heat inactivated at 65°C for 10 minutes. Digested and undigested samples were analysed by electrophoresis in a 0.8% agarose gel at 100V for 50 minutes.
For the Level 2 assembly, the same protocol of the restriction-ligation was used. This time the DNA components were the LV1 confirmed parts (TU + Phleomycin resistance cassette) and the destination vector used was DVA_AF.  Fifteen microliters of the digestion-ligation reaction were transformed into 100 µL of chemically competent Top 10 cells by the heat-shock method and plated into LB + Ampicillin (100 μg/ml) + X-gal (200 µg/ml) + IPTG (1mM) plates after the recovery with SOC medium. White/blue screening was performed the next day after transformation.
All the white colonies of each new LV2 assembly were used to inoculate 5 ml overnight liquid culture [LB + Ampicillin (100 μg/ml)]. All plasmids were purified with the modified QIAprep Spin Miniprep kit (Qiagen). DNA concentrations were measured with the NanoDrop 2000 (Thermofisher). The LV2 assembly was confirmed by digestion; 150-200 ng of each LV2 were incubated with 0.5 µL BsaI (NEB) with 1X CutSmart buffer (NEB) in a 20 µL reaction. The digestions were incubated at 37°C overnight and the enzyme was heat inactivated at 65°C for 10 minutes. Digested and undigested samples were analysed by electrophoresis in a 0.8% agarose gel at 90V for 60 minutes.
4. Fungi Transformation
PEG-Mediated Transformation.
Our team adopted a modified version of the Goarin et al. (2016) PEG-mediated protoplast transformation protocol. Goarin et al. (2016) are the first to report efficient gene replacement in P. roqueforti and identify several improvements to prior protocols however our team were unable to replicate this.
The protocol required use of several protoplastisation solutions that were prepared in stock batches, autoclaved at 130°C for 20 minutes and stored at room temperature. TRF1 was prepared as a proto-solution, with 400 mg Glucanex (Sigma-Aldrich, L1412) and 20 mg bovine serum albumin (Sigma-Aldrich) was added to each 10 mL aliquot before passing through a sterile 0.2 µm low protein binding filter to sterilise (Merck Millipore, SLGP033RS).
Mono- (8.5%) and di-potassium (91.5%) orthophosphate was used to balance pH and were provided by the Marles-Wright Laboratory. Sorbitol and PEG4000 were also kindly provided by the Marles-Wright Laboratory. The pH of all solutions was measured using a calibrated 3510 pH meter (Jenway). Stirring and heating at 50°C for ~5 hours was required to dissolve PEG4000 at 60% and TRF5 was warmed at 50°C and stirred for 30 minutes prior to each use to ensure that PEG did not precipitate.
- TRF1: 1.2 M MgSO4, 10 mM orthophosphate pH 5.8, 400 mg Glucanex, 20 mg bovine serum albumin.
- TRF2: 0.6 M Sorbitol, 100 mM Tris–HCl pH 7.5.
- TRF3: 1 M sorbitol, 10 mM Tris–HCl pH 7.5.
- TRF4: 1 M sorbitol, 10 mM Tris–HCl pH 7.5, 10 mM CaCl2.
- TRF5: PEG4000 60 %, 1 M sorbitol, 10 mM Tris–HCl pH 7.5, 10 mM CaCl2.
For each attempt at transformation six 25 mL petri dishes of M3 medium were inoculated with a 100 µL from the PDB propagation stock. These were grown for 4 days at room temperature and harvested. After application of 10 mL of 0.5% Tween 20 (Scientific Laboratory Supplies, CHE3852) conidia were scraped using a sterile L-shaped cell spreader to loosen from the plate and plates were ‘vacuumed’ using a pipette to maximise yield. Cell suspensions were transferred to a 50 m: falcon tube and centrifuged for 5 minutes at 2,600 x g (room temperature) and the resultant conidia pellet was suspended in 1 mL of M3 medium.
This suspension was distributed evenly between five 100 mL sterile conical flasks holding 50 mL M3 liquid medium which were then incubated for ~18 hours at 30°C with shaking at 90 rpm. In attempt to address contamination issues kanamycin was added to incubation mediums at 50 μg/mL during one procedure. Light microscopy was performed to determine germination. All cultures were filtered through four layers of sterile Miracloth (Merck Millipore, 475855-1R). To condense the mycelium harvested on the cloth and remove residual media 50 mL of sterile water was washed through.
The flow through was discarded and the mycelia were blotted dry and then weighed. Where applicable 1 g of dry mycelium was suspended in 10 mL of TRF1 protoplastisation solution. When yields lower than 1g were obtained the full yield was suspended in 10 mL TRF1. This suspension was incubated at 30°C for 120 minutes with shaking at 90 rpm to allow lysis of cell walls. The solution was then overlaid with 10 mL of TRF2 solution or transferred to a 25 mL sterile rounded centrifuge tube and then overlaid with 10 mL TRF2 solution. Overlaying the solutions to leave an interface proved difficult and tedious but was achievable by trickling the TRF2 from a 10 mL serological pipette with the receptacle held as laterally as possible to minimise mixing.
The layered solution was then centrifuged for 10 minutes at 2,600 x g. To minimise mixing, centrifuge acceleration and deceleration parameters were set to minimum values. In principle, protoplasts should form a visible later at the interface that can be removed. This was not apparent in any attempt and so the full suspension was removed, discarding the pellet of cell debris. The 10 mL suspension was transferred to a sterile 50 mL falcon tube and washed with 10 ml and centrifuged at 2,600 x g for 10 minutes.
Following centrifugation, the solution was discarded and the resultant protoplast pellet was resuspended in 1 mL of TRF4. Ten micrograms of circular plasmid DNA (p444) and 50μl of TRF5 solution were added and the suspension was left on ice for 20 minutes. Following addition of 500 μL of TRF5 solution protoplasts were left in a waterbath at 34°C for 15 minutes in a waterbath at 34°C. After this heat shock the suspension was centrifuged for 8 minutes at 3,500 x g. The solution was discarded and the pellet resuspended in 200μl TRF4 solution. After addition of 13 mL of M3 supplemented with sorbitol at 182.12 g L−1 the suspension was incubated for 14 h at 23 °C with shaking at 80 rpm.
From this suspension 100 μL was plated onto selective M3 medium containing 50 µg/mL phleomycin and left to grow at room temperature for 5 days or refrigerated for 24 hours and then left at room temperature. Viability was tested by inoculating non-selective M3 plates with 100 μL of regenerating protoplasts. To provide negative control 100 μL of cell propagation culture was plated onto M3 plates with 50 µg/mL phleomycin.
Microscopy images of P. roqueforti. (A) Slide prepared with fungi in liquid M3 media before protoplast protocol. Stained with lactophenol blue and taken 100x with immersion oil. (B, C) Slides prepared after the enzymatic digestion of the cell wall. (D) Protoplast count before PEG-mediated transformation. Green arrow indicates an undigested conidia cell, orange indicates protoplast and red a germinated and undigested conidia cell.
Electroporation.
No reported account of successful electroporation in P. roqueforti could be found in the published literature and thus an adapted version of the general protocol for electroporation in filamentous fungi provided by Chakraborty et al. (1991) was employed.
Firstly electroporation buffer was prepared at 1 mM HEPES (Sigma, H3375) pH 7.5, 50 mM mannitol (kindly provided by Heather Barker). Under 1 mL of 10 mM NaOH was required to achieve desired pH. As described previously, for each attempt at transformation six M3 plates were inoculated with a 100 µL from the PDB propagation stock.
These were grown for 4 days at room temperature and harvested following application of 10 mL of 0.5% Tween 20 and then transferred to a 50 mL falcon tube and centrifuged for 5 minutes at 2,600 x g. The resultant conidia pellet was suspended in 1 mL of M3 medium and then distributed evenly between five 100 mL sterile conical flasks holding 50 mL M3 liquid medium.
The flasks were then incubated at 30°C with shaking at 90 rpm until germination was observed (~18 hours). After 16 hours of incubation 0-2mg/ml of Glucanex was added.
Germinated conidia were transferred to sterile 50 ml falcon tubes and washed free of media by centrifugation at 3900 x g for 5 minutes (room temperature). The conidia pellet was resuspended in electroporation buffer and centrifugation was repeated 3 times to wash the cells and remove salts.
Cells were resuspended in 1 mL of electroporation buffer and light microscopy was performed to determine conidia density. A 100 µL sample was then transferred to 2 mm electroporation cuvettes (Eurogentec, CE-0002-50) and circular plasmid DNA (p444 or level 1 phleomycin resistance cassette) was added at 5 µg per 6 x 106 conidia.
The cuvettes were left on ice for 15 minutes and then subjected to electroporation at room temperature (Bio-rad, MicroPulser) with voltage set to 2.5kv to give a field strength of 12.5 kV/cm. Immediately following electroporation 2 mL of M3 medium was added to each cuvette, mixed and the solution transferred to 25 mL falcon tubes containing an additional 10 mL M3 medium.
A 2 hour recovery incubation was performed at 30°C. Cells were then washed free of medium by centrifugation at for 5 minutes at 2,600 x g (room temperature) and resuspended in 5 mL sterile water. Transformants were selected by adding 100 µL to M3 plates containing 50 µg/mL phleomycin.
References
Barlow, S., Chesson, A., Collins, J.D., Dybing, E., Flynn, A., Fruijtier-Pölloth, C., Hardy, A., Knaap, A., Kuiper, H., Le Neindre, P. and Schans, J., 2007. Introduction of a Qualified Presumption of Safety (QPS) approach for assessment of selected microorganisms referred to EFSA. Opinion of the scientific committee. EFSA J, 587, pp.1-16.
Chakraborty, B.N., Patterson, N.A. and Kapoor, M. (1991) An electroporation-based system for high-efficiency transformation of germinated conidia of filamentous fungi. Canadian journal of microbiology, 37, pp.858-863.
Chang, S.C., Lu, K.L. and Yeh, S.F., 1993. Secondary metabolites resulting from degradation of PR toxin by Penicillium roqueforti. Applied and environmental microbiology, 59(4), pp.981-986.
Cheeseman, K., Ropars, J., Renault, P., Dupont, J., Gouzy, J., Branca, A., Abraham, A.L., Ceppi, M., Conseiller, E., Debuchy, R. and Malagnac, F., 2014. Multiple recent horizontal transfers of a large genomic region in cheese making fungi. Nature communications, 5. [Online] Available from:
[Accessed on July 20th 2016].
Finoli, C., Vecchio, A., Galli, A. and Dragoni, I., 2001. Roquefortine C occurrence in blue cheese. Journal of food protection, 64(2), pp.246-251.
Florea, M., Hagemann, H., Santosa, G., Abbott, J., Micklem, C.N., Spencer-Milnes, X., de Arroyo Garcia, L., Paschou, D., Lazenbatt, C., Kong, D. and Chughtai, H., 2016. Engineering control of bacterial cellulose production using a genetic toolkit and a new cellulose-producing strain. PNAS, p.201522985.
Fontaine, K., Passeró, E., Vallone, L., Hymery, N., Coton, M., Jany, J.L., Mounier, J. and Coton, E., 2015. Occurrence of roquefortine C, mycophenolic acid and aflatoxin M1 mycotoxins in blue-veined cheeses. Food control, 47, pp.634-640.
Frisvad, J. C., Smedsgaard, J., Larsen, T.O., Samson, R.A., 2004. Mycotoxins, drugs and other extrolites produced by species in Penicillium subgenus Penicillium. Studies in mycology, 49, pp.201-242.
Goarin, A., Silar, P. and Malagnac, F., 2015. Gene replacement in Penicillium roqueforti. Current genetics, 61(2), pp.203-210.
Hidalgo, P.I., Ullán, R.V., Albillos, S.M., Montero, O., Fernández-Bodega, M.Á., García-Estrada, C., Fernández-Aguado, M. and Martín, J.F., 2014. Molecular characterization of the PR-toxin gene cluster in Penicillium roqueforti and Penicillium chrysogenum: cross talk of secondary metabolite pathways. Fungal genetics and biology, 62, pp.11-24.
Kojima, R., Arai, T., Kasumi, T. and Ogihara, J., 2015.  Construction of transformation system in Penicillium purpurogenum. Journal of bioscience and bioengineering, 119(3), pp.314-316.
Larsen, M.D., Kristiansen, K.R. and Hansen, T.K., 1998. Characterization of the proteolytic activity of starter cultures of Penicillium roqueforti for production of blue veined cheeses. International journal of food microbiology, 43(3), pp.215-221.
Lavermicocca, P., Valerio, F. and Visconti, A., 2003. Antifungal activity of phenyllactic acid against molds isolated from bakery products. Applied and environmental microbiology, 69(1), pp.634-640.
Ropars, J., Dupont, J., Fontanillas, E., de La Vega, R.C.R., Malagnac, F., Coton, M., Giraud, T. and López-Villavicencio, M., 2012. Sex in cheese: evidence for sexuality in the fungus Penicillium roqueforti. PLoS One, 7(11), p.e49665.
Ropars, J., de La Vega, R.R., López-Villavicencio, M., Gouzy, J., Dupont, J., Swennen, D., Dumas, E., Giraud, T. and Branca, A., 2016. Diversity and Mechanisms of Genomic Adaptation in Penicillium. Aspergillus and Penicillium in the post-genomic era.  Norfolk: Caister Academic Press.
US Food and Drug Administration. 2015. Microorganisms & Microbial-Derived Ingredients Used in Food (Partial List). [Online] Available at:
[Accessed on August 11th 2016].
Thank you to our sponsors for their support
ABOUT US
We are the University of Edinburgh Overgraduate iGEM Team, competing in the new application track in iGEM 2016.
read more
LOCATION
School of Biological Sciences
The University of Edinburgh
King's Buildings
Edinburgh EH9 3JF,
United Kingdom
FOLLOW US ON
CONTACT US
Email: edigemmsc@ed.ac.uk
