<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_AHUT_China_parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:AHUT China/parts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="index_main"><DIV id="top_navi"><UL id="sec_top_navi"><LI><A href="https://2015.igem.org/Team:AHUT_China">Main</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/project">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/parts" style="color:#c34484; font-weight:600;">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/notebook">NOTEBOOK</A></LI></UL></DIV><DIV class="sec_block_top"><P style="width:800px; height:100px; padding:100px 80px; text-align:center; margin:0 auto; color:#f8f8f8; font-size:48px; font-family:'Trebuchet MS', Helvetica, sans-serif;">Answers to every question of biological science must be sought in the cell.</P><P style="height:100px; width:960px; line-height:100px; margin: 0 auto; text-align:right; color:#ffffff;font-size:24px; font-family:'Trebuchet MS', Helvetica, sans-serif;">
---E.B.Wilson
      </P></DIV><DIV class="sec_block" style="background-color:#fff;"><P name="Biobrick" id="Biobrick" class="sec_title">Biobrick</P><DIV style="width:960px; margin:0 auto; padding-top:20px;"><P class="sec_paragraph_title">Key Parts</P><P class="sec_p"><B>RgPAL1（<A href="http://parts.igem.org/Part:BBa_K1751000">BBa_K1751000</A>）</B></P><P class="sec_p">
        The gene RgPAL1 can express Phenylalanine ammonia lyase（PAL）, which is an enzyme that catalyzes a reaction converting L-phenylalanine to ammonia and trans-cinnamic acid.PAL is the first and committed step in the phenyl propanoid pathway and is therefore involved in the biosynthesis of the polyphenol compounds. In our project, PAL plays an important role in L-phenylalanine’s deaminization to produce trans-cinnamic acid. 
Phenylalanine ammonia lyase is specific for L-Phe, and to a lesser extent, L-Tyrosine. The reaction catalyzed by PAL is the spontaneous, non-oxidative deamination of L-phenylalanine to yield trans-cinnamic acid and ammonia.
      </P><P class="sec_p" style="text-align:center;">L-phenylalanine   trans-cinnamate + NH<SUB>3</SUB></P><P class="sec_p">
      The cofactor 3,5-dihydro-5-methyldiene-4H-imidazol-4-one (MIO) is involved in the reaction and sits atop the positive pole of three polar helices in the active site, which helps to increase its electrophilicity.MIO is reported to attack the aromatic ring of L-Phe in a Friedel-Crafts-type reaction, which activates the C-H bond and leads to cleavage of the bond. The carbanionintermediate formed by this mechanism is stabilized by partial positive regions in the active site. The mechanism of the reaction of PAL is thought to be similar to the mechanism of the related enzyme histidine ammonia lyase.PAL is inhibited by trans-cinnamic acid, and, in some species, may be inhibited trans-cinnamic acid derivatives.D-Phe and D-Tyr are competitive inhibitors.
      </P><P class="sec_paragraph_title">Promotor-RBS</P><P class="sec_p"><B>Promotor-RBS（<A href="http://parts.igem.org/Part:BBa_K847100">BBa_K847100</A>）</B></P><P class="sec_p">
      This part contains a constitutive promoter and strong RBS. The strength of this RBS is considered relative to BBa_B0031, BBa_B0032, BBa_B0033 and BBa_B0034. While the promotor is a member of a family of constitutive promotor parts labelled J23100 through J23119.
      </P><P class="sec_paragraph_title">Terminators</P><P class="sec_p"><B>Terminator(<A href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A>)</B></P><P class="sec_p"><B>Terminator(<A href="http://parts.igem.org/Part:BBa_B0012">BBa_B0012</A>)</B></P><P class="sec_paragraph_title">Plasmid</P><P class="sec_p"><B><A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A></B></P><P class="sec_p"><I>High copy BioBrick assembly plasmid</I></P><P class="sec_p">
        pSB1C3 is a high copy number plasmid (RFC [10]) carrying chloramphenicol resistance.
The replication origin is a pUC19-derived pMB1 (copy number of 100-300 per cell).
pSB1C3 has terminators bracketing its MCS which are designed to prevent transcription from *inside* the MCS from reading out into the vector. The efficiency of these terminators is known to be &lt; 100%. Ideally we would construct a future set of terminators for bracketing a MCS that were 100% efficient in terminating both into and out of the MCS region.
      </P><P class="sec_p"><I>Usage and Biology</I></P><P class="sec_p">
       In Spring 2011, pSB1C3 was sequenced using several primers: Pre-R, Suff-F, VF2, VR, and internal primers for the origin and resistance. The reference sequence below matches the most recent sequencing verification of pSB1C3 (w/ BBa_J04450, located in SP 4000 Well 2A).
      </P></DIV></DIV><DIV class="sec_block" style="background-color:#fff;"><P name="Protools" id="Protools" class="sec_title">Protools</P><DIV style="width:960px; margin:0 auto; padding-top:20px;"><P class="sec_paragraph_title">Total RNA Extraction from Rehmannia glutinosa Libosch</P><TABLE width="800" border="1" cellspacing="5" cellpadding="5" class="sec_table"><TBODY><TR><TD width="500"><B>Part Ⅰ -20℃ Preservation</B></TD><TD> </TD></TR><TR><TD>50×DTT Solution</TD><TD>700 µL</TD></TR><TR><TD>Recombinant DNase I (RNase-free； 5 U/µL)</TD><TD>1,000 U</TD></TR><TR><TD>10×DNase I Buffer</TD><TD>1 mL</TD></TR><TR><TD><B>Part Ⅱ Room Temperature(15℃～25℃)Preservation</B></TD><TD> </TD></TR><TR><TD>Buffer PE</TD><TD>28 mL</TD></TR><TR><TD>Buffer NB</TD><TD>1.5 mL×2</TD></TR><TR><TD>Buffer RL*1*2</TD><TD>28 mL</TD></TR><TR><TD>Buffer RWA*2</TD><TD>28mL</TD></TR><TR><TD>Buffer RWB*3</TD><TD>30mL</TD></TR><TR><TD>RNase Free dH<SUB>2</SUB>O</TD><TD>15mL</TD></TR><TR><TD>RNA Spin Column</TD><TD>50 series</TD></TR><TR><TD>RNase Free Collection Tube(1.5 mL)</TD><TD>50 tubes</TD></TR><TR><TD><B>Other Reagents</B></TD><TD> </TD></TR><TR><TD>Anhydrous Alcohol</TD><TD> </TD></TR><TR><TD>80%alcohol(0.1% DEPC prepared by treated water)</TD><TD> </TD></TR></TBODY></TABLE><P class="sec_paragraph_title">Specific Steps of Total RNA Extraction from Rehmannia glutinosa Libosch </P><P class="sec_p">
            1. Taking tender leaves of Rehmannia glutinosa Libosch after flash freezing by liquid nitrogen, then grounding them into powders by mortar.
  2. Adding 50mg ground powders into a 1.5mL sterile centrifuge tube of 450 µLBuffer RL to conduct schizolysis, then getting the lysate by pipettor’s repeating blowing to the degree that there is no obvious sign of precipitation in the centrifuge tube. 
  3. After 5 minutes’ centrifugation of the lysate under the conditions of 4℃、12000rpm, absorbing the supernatant to a new 1.5mL sterile centrifuge tube.
  4. Adding anhydrous alcohol into a 1.5mL sterile centrifuge tube of the supernatant to get alcohol supernatant after homogeneous mixing. The volume ratio of anhydrous alcohol to the supernatant is 1:2. 
  5. Shifting all the available alcohol supernatant and adding 500 µL Buffer RWA into RNA Spin Column. Then conducting centrifugation for 30 seconds under the condition of 12000 rpm and abandoning the filtrate.
  6. Adding 70 mL anhydrous alcohol into Buffer RWB. After homogeneous mixing, taking 600 µL of it into RNASpin Column centrifuged in the previous step. Then conducting centrifugation for 30 seconds under the condition of 12000 rpm and abandoning the filtrate.
  7. Adding 100 µL water of no RNA enzyme into the middle membrane of RNA Spin Column centrifugated in the previous step. After 5 minutes’ still standing, centrifuging elution RNA for 2 minutes under the condition of 12000 rpm. 
  8. Identifying the quality of the elution RNA. 
        </P><P class="sec_paragraph_title">Reverse Transcription of RNA</P><TABLE width="800" border="2" cellspacing="5" cellpadding="5" class="sec_table"><TBODY><TR><TD>Oligo dT Primer </TD><TD>1 µL</TD></TR><TR><TD>dNTP</TD><TD>1 µL</TD></TR><TR><TD>Templet RNA </TD><TD>4 µɡ</TD></TR><TR><TD>RNase free dH<SUB>2</SUB>O </TD><TD>Up to 10µL</TD></TR><TR><TD colspan="2">After 5 minutes’ heat preservation of 65 ℃, cooling on ice immediately</TD></TR></TBODY></TABLE><TABLE width="800" border="2" cellspacing="5" cellpadding="5" class="sec_table"><TBODY><TR><TD>Above Reaction Solutions After Denaturation</TD><TD>10µL</TD></TR><TR><TD>5×PrimeScript II Buffer</TD><TD>4µL</TD></TR><TR><TD>RNase Inhibitor (40 U/µL)</TD><TD>0.5 µL (20 U) </TD></TR><TR><TD>PrimeScript II RTase (200 U/µL)</TD><TD>1 µL (200 U)</TD></TR><TR><TD>RNase free dH<SUB>2</SUB>O </TD><TD>Up to 20µL</TD></TR><TR><TD colspan="2">Mixing softly and homogeneously </TD></TR><TR><TD colspan="2">42℃      30～60 min </TD></TR><TR><TD colspan="2">95℃      5 minutes later, cooling on ice</TD></TR></TBODY></TABLE><P class="sec_paragraph_title">Reaction Reagent of PCR</P><TABLE width="800" border="2" cellspacing="5" cellpadding="5" class="sec_table"><TBODY><TR><TD width="500">Template of Plasmid DNA</TD><TD>1 µL</TD></TR><TR><TD>10 ×PCR buffer solution</TD><TD>2 µL</TD></TR><TR><TD>primers 1</TD><TD>0.8 µL</TD></TR><TR><TD>primers 2</TD><TD>0.8 µL</TD></TR><TR><TD>10%DMSO</TD><TD>4 µL</TD></TR><TR><TD>dNTP (10 mmol/L)</TD><TD>1 µL</TD></TR><TR><TD>LA-Taq enzyme(5 U/µL)</TD><TD>0.2 µL</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>0.2 µL</TD></TR><TR><TD>25 mmol/L MgCl<SUB>2</SUB></TD><TD>5µL</TD></TR><TR><TD>Total</TD><TD>25 µL</TD></TR></TBODY></TABLE><P class="sec_paragraph_title">PCR Cycle</P><TABLE width="800" border="2" cellspacing="5" cellpadding="5" class="sec_table"><TBODY><TR><TD>Temperature</TD><TD>Duration</TD><TD>Times of Cycle</TD></TR><TR><TD>94℃</TD><TD>1 min</TD><TD>x1</TD></TR><TR><TD>94℃</TD><TD>30 sec</TD><TD>x30</TD></TR><TR><TD>60℃</TD><TD>30 sec</TD><TD>x30</TD></TR><TR><TD>72℃</TD><TD>2 min</TD><TD>x30</TD></TR><TR><TD>72℃</TD><TD>5 min</TD><TD>x1</TD></TR></TBODY></TABLE><P class="sec_paragraph_title">Target Gene’s Linking to T Vector</P><TABLE width="800" border="2" cellspacing="5" cellpadding="5" class="sec_table"><TBODY><TR><TD>Target Gene</TD><TD>4 µL</TD></TR><TR><TD>T Vector</TD><TD>1 µL</TD></TR><TR><TD>T4 Ligase</TD><TD>4 µL</TD></TR><TR><TD>10×Buffer Solution of Ligase</TD><TD>1 µL</TD></TR><TR><TD>Total</TD><TD>10 µL</TD></TR></TBODY></TABLE><P class="sec_paragraph_title">The Transformation of Colibacillus Made by Recombinational Plasmid of pMD<SUP>TM</SUP>19-T RgPAL1 and the Clone of this Recombinational Plasmid</P><P class="sec_p">
        1. The Cultivation of Colibacillus
Colibacillus is cultivated by LB medium in the shaker of 37℃ and 200 r/min rate. 

2. The Preparation of Competent Cells
A. Selecting single bacterial colony form the activated plate of E. coli DH5α and inoculating it in the medium tube of 5 mL LB liquid to conduct the shock culturing under the temperature of 37 ℃ for 2.5~3 hours and cool the culture by ice to 0 ℃. 
B. Pouring the culture into a 1.5 mL sterile centrifuge tube.
C. Centrifuging for 10 minutes under conditions of 4,000 r/min and 4 ℃.
D. Abandoning the supernatant and collecting the thallus.
E. Using pre-cooling 0.5 mL 0.1 mol/L CaCl<SUB>2</SUB> to re-suspend and centrifuge the thallus and abandoning the supernatant. 
F. Using pre-cooling 0.5 mL 0.1 mol/L CaCl<SUB>2</SUB> to re-suspend and ice-bath the thallus for 15 minutes and centrifuging to collect them.
G. Adding 200 µL pre-cooling 0.1mol/L CaCl<SUB>2</SUB> and being placed in ice-bath to get competent cells.

3. Transformation
A. Taking 5 µL overnight connected product into a tube of competent cells, and softly rotating the tube to homogeneously mix the contents and placing them on ice for 30 minutes. 
B. Placing the centrifuge tube in water bath of 42 ℃ for heat shock of 90 seconds. Do not shake the centrifuge tube. 
C. Immediately placing the centrifuge tube on ice to cool the cells for 5 minutes.
D. Adding 800 µL LB liquid medium to incubate under the conditions of 37 ℃ and 200r/min for 45 minutes. 
E. Taking 200 µL culture solution to be coated on LB plate of 100 µg/mL, Amp, 20 µL X-Gal and 4 µL IPTG and checking the transformed bacterial colonies after 12-16 hours’ cultivation under the temperature of 37 ℃.
F. Selecting white bacterial colonies and inoculating and cultivating them in the liquid medium of Amp. 
G. Extracting the plasmid of white bacterial colonies with the application of alkaline lysis. The testing method: conducting the analysis of agarose gel electrophoresis (AGE) on extracting plasmid and selecting relatively lagging ones to conduct double enzymes restriction test. 
H. Connecting the recycled product RgPAL1 and pCAMBIA1305.1 expression vector framework after double enzymes restriction under the temperature of 16℃overnight, and testing them by double enzymes restriction.</P><P class="sec_paragraph_title">Recycling the Product pMD<SUP>TM</SUP>19 T RgPAL1 in Colibacillus by Using Method of Alkaline Lysis</P><TABLE width="800" border="2" cellspacing="5" cellpadding="5" class="sec_table"><TBODY><TR><TD>E. coli bacterium suspension</TD><TD>1.5 mL</TD></TR><TR><TD>10000 r/min centrifugation (taking the sediment )</TD><TD>1 min</TD></TR><TR><TD>Glucose 50 mmol/L Tris-Cl (pH 8.0) 25 mmol/L</TD><TD>100µL</TD></TR><TR><TD>NaOH 0.2 mol/L 2%SDS</TD><TD>200µL</TD></TR><TR><TD>Potassium acetate 5 mol/L glacial acetic acid（pH 4.8）</TD><TD>150µL</TD></TR><TR><TD>12000 r/min centrifugation(taking the supernatant)</TD><TD>10min</TD></TR><TR><TD>1/10 volume NaAC(3 mol/L，pH5.2)anhydrous alcohol of two times’ volume(still standing)</TD><TD>5min</TD></TR><TR><TD>12000 r/min centrifugation, taking the supernatant(two times)</TD><TD>15min</TD></TR></TBODY></TABLE><P class="sec_paragraph_title">Reaction System of Double Enzymes Restriction</P><TABLE width="800" border="2" cellspacing="5" cellpadding="5" class="sec_table"><TBODY><TR><TD width="500">pMD<SUP>TM</SUP>19 T RgPAL1 cloning plasmid</TD><TD>2 µL</TD></TR><TR><TD>10 ×M Buffer solution of enzyme digestion</TD><TD>4 µL</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>15 µL</TD></TR><TR><TD>Enzyme NcoI</TD><TD>1 µL</TD></TR><TR><TD>Enzyme SpeI</TD><TD>1 µL</TD></TR><TR><TD>pCAMBIA1305.1 expression vector</TD><TD>2 µL</TD></TR><TR><TD>Total</TD><TD>25 µL</TD></TR><TR><TD colspan="2">Incubate at 37 °C incubator or heat bath for 0.25 to 2 hours. </TD></TR></TBODY></TABLE><P class="sec_paragraph_title">Construction of the Expression Vectors of RgPAL1 and pCAMBIA1305.1</P><TABLE width="800" border="2" cellspacing="5" cellpadding="5" class="sec_table"><TBODY><TR><TD width="500">10 ×M Buffer solution of enzyme digestion</TD><TD>2µL</TD></TR><TR><TD>RgPAL1 after recycle</TD><TD>4µL</TD></TR><TR><TD>pCAMBIA1305.1 Vector backbone</TD><TD>4µL</TD></TR><TR><TD>DNA ligase</TD><TD>2µL</TD></TR><TR><TD>Total</TD><TD>10µL</TD></TR></TBODY></TABLE><P class="sec_paragraph_title">The Transformation of Agrobacterium Tume Faciens</P><P class="sec_p">
        1. Pre-cultivation of explant: After two hours of washing the leaves of Rehmannia glutinosa Libosch by running water, letting them soak in mercuric chloride of 0.05%-0.1% mass concentration. Then washing them thoroughly with sterile water and letting them soak in 75% alcohol for 15 seconds. Next, washing them again with sterile water for 4-6 times. Cutting the leaves after washing into ones of 1 cm 2 size and drying the surface moisture by sterile filter paper. Inoculating them in MS solid induction medium of 1.5 mg·L<SUP>-1</SUP> 6-benzylamino adenine and 0.1 mg·L<SUP>-1</SUP> naphthalene acetic acid plant hormones. Then alternately cultivating them through the light and in the darkness for 16 hours and 8 hours respectively to get multiple shoots of Rehmannia glutinosa Libosch after a month. Shifting them into MS solid induction medium of no hormones to get aseptic seedlings. After their growing to 5cm, clipping and getting leaves of aseptic seedlings explant for the use of transformation. 
2. Preparation of agrobacterium EHA105 suspension which contains binary vectors: in the first three days of transformation, after building the target gene into binary vectors, shifting the recombinational binary vectors into agrobacterium EHA105 and using kanamycin to select bacterial colonies that contain the recombinational binary vectors. Inoculating agrobacterium in the LB fluid medium and conducting shock culturing of them to logarithmic growth phase under the condition of 28℃，180r/min. Preserving the bacterial strains in the temperatures of 4℃ and -20℃ respectively for standby. 
3. Co-cultivation of agrobacterium and explant: shifting the available explant leaves into the MS medium of 100 µmol·L<SUP>-1</SUP> acetosyringone. Then adding the activated MS suspension of root carcinoma agrobacterium engineering bacteria that contains plant expression vectors of the gene RgPAL1. Next, taking out the explant after 5 minutes’ full contact with bacterium suspension and absorbing bacterium suspension in the surface of explant by sterilization filter paper. Then co-cultivating the explant for 48 hours in MS induction medium of 1.5 mg·L<SUP>-1</SUP> 6-benzylamino adenine and 0.1 mg·L<SUP>-1</SUP> naphthalene acetic acid plant hormones. 
4. Selection of regeneration plants of resistance: immediately shifting the explant of co-cultivation into antibacterial MS solid medium of 500mg·L<SUP>-1</SUP> cefotaxime sodium. After the regeneration of multiple shoots given by the cultivation pathway of regeneration system, shifting them into absolute MS solid medium of 4 mg·L<SUP>-1</SUP> hygromycin to root until the plants are regenerated as well as transgenic Rehmannia glutinosa Libosch is gained.</P></DIV></DIV></DIV><DIV id="footer"><DIV class="conent"><UL id="foot_list"><LI>TEAM</LI><LI><A href="https://2015.igem.org/Team:AHUT_China/team#Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/team#Instructors">Instructors</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/team#Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/team#Acknowledgement">Acknowledgement</A></LI></UL><LI>PROJECT</LI><LI><A href="https://2015.igem.org/Team:AHUT_China/project#Pathway and Design">Pathway and Design</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/project#Result">Result</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/project#Future">Future Work</A></LI><LI>PARTS</LI><LI><A href="https://2015.igem.org/Team:AHUT_China/parts#Biobrick">Biobrick</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/parts#Protools">Protools</A></LI><LI>HUMAN PRACTICES</LI><LI><A href="https://2015.igem.org/Team:AHUT_China/Practices#Society">Society</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/Practices#NewPower">New Power</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/Practices#Questionnaire">Questionnaire</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/Practices#MOOC">MOOC</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/Practices#Art">Art</A></LI><LI>SAFETY</LI><LI><A href="https://2015.igem.org/Team:AHUT_China/Safety">Safety</A></LI><LI>NOTEBOOK</LI><LI><A href="https://2015.igem.org/Team:AHUT_China/notebook#Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:AHUT_China/notebook#Lab Pictures">Lab Pictures</A></LI><P id="foot_copyright">Copyright 2015 © AHUT iGEM Team , All rights reserved . </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>