<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Virginia_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Virginia/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Virginia/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Virginia/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Virginia/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Virginia/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Virginia/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>iGEM - Team Virginia - Project</TITLE></P><DIV class="container"><DIV class="bannerArea"><UL class="bannerNav"><LI><A href="https://2011.igem.org/Team:Virginia/Project">PROJECT</A></LI><LI><A href="https://2011.igem.org/Team:Virginia/Parts">PARTS</A></LI><LI><A href="https://2011.igem.org/Team:Virginia/Modeling">MODELING</A></LI><LI><A href="https://2011.igem.org/Team:Virginia/Notebook">NOTEBOOK</A></LI><LI><A href="https://2011.igem.org/Team:Virginia/Safety">SAFETY</A></LI><LI><A href="https://2011.igem.org/Team:Virginia/HumanPractices">HUMAN PRACTICES</A></LI><LI><A href="https://2011.igem.org/Team:Virginia/Resources">RESOURCES</A></LI><LI><A href="https://2011.igem.org/Team:Virginia/Team">TEAM</A></LI><LI><A href="https://2011.igem.org/Team:Virginia/Attributions">ATTRIBUTIONS</A></LI></UL></DIV><DIV class="container"><DIV class="content"><H1>Project</H1><H2>Abstract</H2><P>We use a synthetic biology approach to promote tissue regeneration at traumatic wound sites. Tissue regeneration is composed of three primary processes: the regrowth of functional parenchymal tissue, the regrowth of support tissues, and the regrowth of vasculature to sustain the nascent tissue formation (angiogenesis).</P><P>
Although tissue engineering has offered several effective solutions to address the first two processes, our project attempts to build upon these ideas and develop a more cost-effective and robust method to promote angiogenesis at traumatic wound sites.  We have devised a circuit to be incorporated in a yeast chassis that efficiently expresses two vital angiogenic proteins--vascular endothelial growth factor (VEGF) and platelet derived growth factor (PDGF-B)--in a sequential and time-dependent manner that approximates the natural cascade of growth factor release in the human body. We also intend to submit a Biobrick-compatible yeast plasmid backbone for future use.</P><H2>Context</H2><P>We're engineering a genetic circuit in yeast to accelerate human wound-healing with the goal of preventing dangerous infection which is often exacerbated in chronic or slow-healing wounds. We hope to accomplish this by locally expressing a set of growth factors associated with accelerated healing in a time-dependent fashion by microorganisms at the wound site.</P><P>Wound contamination is a double-edged sword. Virtually all wounds (~98%) are contaminated by aerobic organisms such as yeast and bacteria. The vast majority of these are benign, and many species actually aid the wound-healing process. On the other hand, the longer a wound remains unhealed, the greater the probability of dangerous infection by harmful pathogens. These dangerous infections can cause serious illness and even be life-threatening. In other words, microorganisms living at a wound-site are not necessarily bad, and often do good insofar as they accelerate the healing process or prevent serious infection.</P><P>The wound-healing process itself is a very complicated set of interdependent processes, but one bottleneck that can be alleviated is angiogenesis, the re-growth of vasculature. This can be accelerated by sequentially expressing vascular endothelial growth factor (VEGF) and platelet-derived growth factor (PDGF) in a time-dependent fashion in response to hypoxia-inducible factor (HIF-1). When expressed locally at appropriate levels, this has the potential to appreciably accelerate rates of wound-healing with few side-effects. The challenge is to design a genetic circuit that achieves optimal rates of HIF-linked growth factor expression without over-shooting target expression because over-expression of VEGF can have very serious side-effects.</P><P>After consulting our advisers, we decided to use a yeast chassis to propagate our circuit. Of the possible host organisms (yeast and bacteria), yeast are better suited to this application because they possess the complicated biomolecular machinery required to assemble the complex mammalian growth factors in question. We also had readier access toresources  to effectively undertake yeast lab protocol.   We understand that future applications of our circuit will most probably not utilize yeast as the shuttle organism, due to complications it may pose when  interacting with mammalian immune systems.  Therefore future applications of our project would ideally utilize more complex mammalian cells (such as monocytes)  that readily integrate within the human body (especially if extracted from the patient host), avoiding possible complications from an immune response.  </P><H2>Circuit Design</H2><P>Many complicated design factors influenced the overall engineering of the genetic circuit and its intended assembly. The design criteria for this circuit revolved around clinical requirements, economic restrictions, and iGEM competition standards.</P><P>The ultimate goal of this system is to improve upon an existing tissue engineering methodology for promoting rapid angiogenic processes at traumatic wound sites.  Angiogenesis is one of three general processes necessary for complete and efficient wound healing.  The other two components include the restoration of parenchymal cells (functional cells of interest at the wound site) and recruitment of support tissue such as fibroblasts into the wound site.  Tissue engineering thus far has provided adequate mechanisms to enhance the latter two processes.  Parenchymal cells may be seeded onto biodegradable and biointegratable polymer scaffolds, which subsequently may be inserted at a particular wound site.  Upon insertion of the scaffold, the parenchymal cells (or their precursors) may proliferate to fill the void at the wound site.  In addition, the scaffold may allow for the recruitment of endogenous fibroblasts to complement the proliferation of parenchymal cells, while restricting the fibroblasts from converting to their contractile phenotypes (myofibroblasts) thus initiating a tissue regeneration process rather than tissue repair via scar formation.</P><P dir="ltr">Our project proposes an improvement at the third stage of tissue regeneration: angiogenesis.  The growth of nascent vasculature and remodeling of existing ones is necessary for sustaining newly regenerated tissue.  Synthetic biology may provide a superior alternative that avoids many of the complexities inherent in the tissue engineering approach.  The tissue engineering approach involves the encapsulation of vital growth factors, namely vascular endothelial growth factor (VEGF) and platelet derived growth factor (PDGF), into the polymer scaffold, thus making the release of these growth factors dependent on the rate of degradation of the scaffold, which is itself dependent on a number of factors and may be prohibitively slow. Further, the use of a seeded polymer scaffold arbitrarily sets a terminal limit on the total amount of growth factor delivered because the growth factor is released from finite storage as opposed to being synthesized on site. In contrast, the release of VEGF and PDGF occurs naturally in a time dependent manner, with VEGF serving to kick start the growth of vasculature and PDGF used to stabilize the nascent vessels.  Both the efficiency of release and the time dependency may be answered by engineering genetic circuits within an appropriate host chassis.  This chassis may then be seeded to the polymer scaffold externally (much like is done with the parenchymal cells) through the use of a bioreactor.  Although in some situations the slow release of growth factors (based on the degradation of the polymer) may be ideal, it is important to introduce an alternate methodology to address the instances where a more rapid regeneration approach is favorable.  </P><P dir="ltr">Below is a diagram of our final genetic circuit with a brief description. </P><P dir="ltr"><B>Figure 1.</B><I>The circuit shows the expression of VEGF and LuxR governed by a hypoxia-inducible-factor-1 (HIF-1) promoter.  The LuxR/LuxI quorum sensing system was selected from Vibrio fischeri as an adequate mechanism to communicate between both components of the circuit.  Yeast also contains an acyl-homoserine lactone homologue to ensure that the system works effectively.  The LuxR in conjunction with the acyl-homoserine lactone produced by constitutively expressed LuxI, forms a dimer which activates the LuxR promoter, thus driving the expression of the PDGF growth factor and a negative control siRNA.  THe siRNA component will be complementary to a select random sequence preceding a kozak ribosomal binding site.  The random sequence ensures specificity within our circuit (thus not inhibiting host cell machinery) while also enabling a more modular circuit (in case the inhibited gene must be replaced.)  The siRNA offers negative control at the translational level, thus ceasing the expression of VEGF, while not inhibiting the expression of LuxR which ultimately drives the expression of PDGF.  The LuxR promter must be tuned to match a threshold concentration of LuxR, corresponding to the necessary expression levels of VEGF needed to activate angiogenic processes.</I></P><P dir="ltr">The chassis selected for the construction of our genetic circuit was yeast.  After much discussion with our advisers, we felt that for the time being this organism would suffice, as its machinery is complex enough to assemble the mammalian genes we were introducing, yet simple enough for us to culture and handle within a short time period.  </P><H2> Description of Components:</H2><B>HIF-1 Inducible Promoter: </B>HIF-1 (hypoxia inducible factor) is an endogenous growth factor released when oxygen reaches low levels (hypoxia) within a particular region in the body.  HIF-1 therefore is responsible for driving the expression of over 40 genes related to angiogenesis.  The presence of HIF-1 at a traumatic wound site under conditions of hypoxia is almost a guarantee, and thus it serves as an ideal component to govern the functionality of our circuit.  Two papers instrumental in supporting our reasons have been found.  The first ensures the capability of yeast to successfully assemble HIF-1 from its’ sub-components, HIF alpha and beta, expressed internally.  The second paper describes the construction of a series of novel bidirectional HIF-1 promoters, which then becomes applicable to the design scheme of our circuit.  The two papers are presented below in order. <A href="http://www.sciencedirect.com/science/article/pii/S0006291X06013428">http://www.sciencedirect.com/science/article/pii/S0006291X06013428</A><A href="http://www.nature.com/gt/journal/v8/n23/pdf/3301605a.pdf">http://www.nature.com/gt/journal/v8/n23/pdf/3301605a.pdf</A>
      There were several constraints in actually implementing and testing the HIF-1 bidirectional promoter.  First, although the first paper describes the ability of yeast to internally express and assemble the transcription factor, it does not address the capability of yeast to import HIF subunits through the cell wall when presented externally.  Second, the HIF-1 bidirectional promoter constructs were patented by Dr. Van Meir and Dr. Post, therefore eliminating the possibility of presenting them as potential “biobrick” parts.  To address these issues, we decided to utilize a GAL1/10 bidirectional promoter (naturally present in yeast) to govern the activity of our circuit.  The GAL1/10 bidirectional promoter is activated in the presence of galactose and thus allows for relatively easy expression of gene components since galactose can easily diffuse past the cell wall of yeast.  The promoter also does not contain a patent, is available on the registry of parts, and is also widely used and well characterized.  <B>VEGF sequence:</B> the VEGF sequence that we attempted to assemble represents the cDNA code for the protein.  We obtained this sequence from the website of Sinobiological Inc., which codes for the murine isoform of VEGF-A.  VEGF is the primary angiogenic growth factor responsible for kick-starting the growth of endothelial cells. <B>PDGF sequence:</B> the PDGF sequence that we attempted to assemble represents the cDNA code for the protein.  We obtained this sequence from the website of Sinobiological Inc., which codes for the murine isoform of PDGF-B.  PDGF-B is expressed later in the cascade of angiogenesis.  It serves to stabilize the nascent vessels.<B>LUXR/LUXI: </B>The LuxR/LuxI quorum sensing system was selected from Vibrio fischeri as an adequate mechanism to communicate between both components of the circuit.  Yeast also contains an acyl-homoserine lactone homologue to ensure that the system works effectively.  The LuxR in conjunction with the acyl-homoserine lactone produced by constitutively expressed LuxI, forms a dimer which activates the LuxR promoter, thus driving the expression of the PDGF growth factor and a negative control siRNA.  Since we are using a eukaryotic chassis, the yeast must contain a separate promoter for each gene that is expressed.  The alternate design accomodates this requirement by implementing two LUX R promoters, one in front of the PDGF and the other in front of the siRNA.  This way the second system is still entirely under the control of one activator (LUXR), while complying with the restrictions of the host chassis.  <B>siRNA: </B>The siRNA component will be complementary to a select random sequence preceding a kozak ribosomal binding site.  The random sequence ensures specificity within our circuit (thus not inhibiting host cell machinery) while also enabling a more modular circuit (in case the inhibited gene must be replaced.)  The siRNA offers negative control at the translational level, thus ceasing the expression of VEGF, while not inhibiting the expression of LuxR which ultimately drives the expression of PDGF.  The design of this negative control system was built upon a similar design for use in E.coli by the 2010 Stanford iGEM team.  Our system, however, is specifically tailored for use in yeast.<B>Padding sequences: </B>We generated random CGAT sequences and then screened for restriction sites that would interfere with our assemblies.  The padding sequences were necessary for optimal function of restriction enzymes.  Since our fragments were linear constructs, the enzymes require a physical space to rest on while cutting at the desired sites.  <B>Codon optimization:</B> GeneDesigner software was used to optimize the open reading frames for yeast expression.  Optimized sequences were also double checked on Serial Cloner 2.1 as mentioned below to determine presence of conflicting restriction sites.  <B>Restriction site avoidance:</B> We used SerialCloner 2.1 to determine whether and where restriction sites occurred within the engineered sequence at multiple stages within the design process. This step was first applied when generating neutral random padding sequences. It was then applied every time another part was added to the overall sequence. It was double and triple checked later.<B>Assembly overlap:</B> Because it would be cost-prohibitive to have the entire gene synthesized at once, we instead opted to split the sequence into smaller oligos, which we would then assemble in our lab. We initially planned to use the assembly method described by Gibson et al (affectionately known as Gibson assembly). Gibson assembly can be used to assemble genetic constructs of arbitrary length and essentially requires the use of phusion polyemerase, taq ligase, and t4 exonuclease to create complementary overhangs and enable binding of adjacent strands.  Due to complications with ordering, we ultimately utilized a similar assembly method known as polymerase cycling assembly. <B>Reverse-complementarity:</B> Due to an unfortunate misunderstanding with IDT’s online interface, we were mistakenly shipped single-stranded instead of double-stranded DNA. Without duplex DNA, none of the Gibson assembly reactions would work, and it took us several weeks of troubleshooting to discover the source of the problem. However, we were able to salvage half of the oligos we had synthesized by instead using PCA. We reordered every other oligo in the construct, reversed it and swapped all C’s with G’s and A’s with T’s and vice-versa to make the new oligos reverse-complimentary so they would bind to the top strand fragments.</DIV></DIV></DIV></DIV><DIV class="footerArea"><DIV class="copyright">Copyleft. All rights reversed.</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Virginia/Project">http://2011.igem.org/Team:Virginia/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Virginia/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Virginia/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Virginia/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Virginia/Project&amp;oldid=202718" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>