<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Amsterdam_Biobricks_Basic_Parts"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Amsterdam/Biobricks/Basic_Parts">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Amsterdam/Biobricks/Basic_Parts&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Amsterdam/Biobricks/Basic_Parts&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Amsterdam/Biobricks/Basic_Parts&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Amsterdam/Biobricks/Basic_Parts" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Amsterdam/Biobricks/Basic Parts</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="wrap"><DIV id="menu"><UL class="level1"><LI class="level1-li"><A class="level1-a" href="https://2011.igem.org/Team:Amsterdam">Home</A></LI><LI class="level1-li"><A class="level1-a drop" href="https://2011.igem.org/Team:Amsterdam/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Team/Students">Students</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Team/Advisors">Advisors</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Team/Lab_environment">Lab Environment</A></LI></UL><LI class="level1-li"><A class="level1-a drop" href="https://2011.igem.org/Team:Amsterdam/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Project/Description">Description</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Project/Modules">Modules</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Project/Data">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Project/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Project/Applications">Applications</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Project/Attributions">Attributions</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Project/Acknowledgements">Acknowledgements</A></LI><LI class="level1-li"><A class="level1-a drop" href="https://2011.igem.org/Team:Amsterdam/Biobricks">BioBricks</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Biobricks/Overview">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Biobricks/Basic_Parts">Basic Parts</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Biobricks/Composite_Parts">Composite Parts</A></LI><LI class="level1-li"><A class="level1-a drop" href="https://2011.igem.org/Team:Amsterdam/Labwork">Labwork</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Notebook/Overview">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Labwork/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Labwork/Characterization">Characterization</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Labwork/Data_Analysis">Data Analysis</A></LI><LI class="level1-li"><A class="level1-a drop" href="https://2011.igem.org/Team:Amsterdam/Human_Outreach">Human Outreach</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Human_Outreach/Collaboration">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Human_Outreach/Education">Education</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Human_Outreach/Media">Media</A></LI><LI class="level1-li"><A class="level1-a drop" href="https://2011.igem.org/Team:Amsterdam/Sponsors/Companies">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Sponsors/Companies">Companies</A></LI><LI><A href="https://2011.igem.org/Team:Amsterdam/Sponsors/Help_us">Help Us</A></LI></DIV><DIV id="contentgrid"><H1><SPAN class="mw-headline" id="Basic_Parts">Basic Parts</SPAN></H1><P>Three different types of basic parts were used to construct our CryoBricks: promoters, ribosome binding sites (RBSes) and coding regions. This page contains information on their purpose in the project. Emphasis is placed on the reason why we selected these particular parts, and in case of the coding regions, the biological function of the encoded protein. For more information on the constructs into which these parts were combined, refer to the <A href="/Team:Amsterdam/Biobricks/Composite_Parts" title="Team:Amsterdam/Biobricks/Composite Parts">composite parts</A> page.
</P><H2><SPAN class="mw-headline" id="Promoters">Promoters</SPAN></H2><P>Several criteria were considered in selecting promoters for use in our CryoBricks. An ideal promoter would be: available from the parts registry, so that we can easily obtain and incorporate it; active at low temperatures, so that it works when we need it to; and positively regulated, so it won't be active unless we want it to. A search for promoters that matched one or more of these criteria resulted in the following selection:
</P><H3><SPAN class="mw-headline" id="pLacI">pLacI</SPAN></H3><P>The Lac operon's promoter, which is often used to induce expression of proteins from plasmids artificially transformed into <I>E. coli</I>. Literature has numerous examples of this promoter being used to (over)express proteins at low temperatures, and it's available from the registry as <A href="http://partsregistry.org/Part:BBa_R0010" class="external text" rel="nofollow">BBa_R0010</A>. Unfortunately, it's not positively regulated, but its activity can be controlled to a certain extent. It's negatively regulated by the <I>LacI</I> protein, and as such can be stimulated by repressing <I>LacI</I> with IPTG or lactose. However, it will be active constitutively in strains that don't express <I>LacI</I>.
</P><H3><SPAN class="mw-headline" id="pBAD">pBAD</SPAN></H3><P>The promoter of the araBAD operon. Its biological function is similar to that of pLacI; in its natural context, it activates transcription of enzymes involved with the consumption of a carbon source, when this carbon source is available to the cell. Its activity is mediated by the <I>AraC</I> protein, which either activates of represses expression, depending on the concentration of L-arabinose.<A href="http://www.pnas.org/content/85/15/5444.full.pdf" class="external autonumber" rel="nofollow">[1]</A> The pBAD promoter is available from the registry as <A href="http://partsregistry.org/Part:BBa_I13453" class="external text" rel="nofollow">BBa_I13453</A>, but team <A href="https://2009.igem.org/Team:British_Columbia" class="external text" rel="nofollow">British Columbia 2009</A> derived two new promoters from this part, responding either more or less strongly to the presence of arabinose (Stronger response: <A href="http://partsregistry.org/Part:BBa_K206000" class="external text" rel="nofollow">BBa_K206000</A>, weaker response: <A href="http://partsregistry.org/Part:BBa_K206001" class="external text" rel="nofollow">BBa_K206001</A>). These pBAD derived promoters were selected because of their different arabinose sensitivity. This difference can facilitate finetuning the expression levels of proteins while using a single inducer, which can be further tweaked by combining the promoters with different RBSes.
</P><H3><SPAN class="mw-headline" id="hybB">hybB</SPAN></H3><P>The promoter of the hydrogenase II operon. It's also known as the coldshock promoter, because it's inactive at 37°C and activates at temperatures below 30°C. Using this promoter would allow us to only have our cells express CryoBricks when <I>E. coli</I> actually benefits from this, but makes inducing expression in a dose-dependent manner impossible, as it's not induced by a molecule we add. It enables CryoBrick expression to be directed on 'autopilot', which has pros (for example, in large-scale industrial applications) and cons (in case CryoBricks comprising such a promoter cross over to an uncontrolled environment, be it by accident or design). The registry submission of this brick, <A href="http://partsregistry.org/Part:BBa_J45503" class="external text" rel="nofollow">BBa_J45503</A>, never made it past the planning stage, but the brick <I>is</I> incorporated in several composite parts.
</P><H3><SPAN class="mw-headline" id="Final_selection">Final selection</SPAN></H3><P>All selected promoters fulfilled the availability criterion, being submitted to the registry, but hybB was only available as part of an assembled construct. The construct was ordered and preparations were made to PCR it out of this construct, but constraints on time and safety considerations caused us to decide against using it in the end.

The project was supposed to include a characterisation experiment to quantify the remaining promoters' strengths at low temperatures. Unfortunately, technical difficulties with cloning, and subsequent assembly of the reporter constructs required for this, made such characterisation infeasible. Even though we couldn't confirm pLacI and pBAD's activity at low temperatures, it was decided to construct our CryoBricks with <I>both</I> promoters. Of the available pBAD promoters, we selected the one that responded most strongly to arabinose (<A href="http://partsregistry.org/Part:BBa_K206000" class="external text" rel="nofollow">BBa_K206000</A>). It allows for the highest the maximum expression level, and any of the expression levels attainable with the related promoters can be achieved as well, given a finely tuned concentration of arabinose.
</P><H2><SPAN class="mw-headline" id="RBSes">RBSes</SPAN></H2><P>Similar to our promoter selection criteria, the ideal RBS for our CryoBricks should be active at low temperatures, and available from the registry. We suspect temperature may have quite a drastic impact on how an RBS functions; RBSes form structures in mRNA that mediate ribosome binding, but thermodynamics govern the dynamicity (and stability) of such structures. Literature study revealed only very little information about how temperature affects the activity of RBSes. To fill this gap in the knowledgebase, we planned to characterise some of the RBSes available from the registry, using reporter constructs to measure their strength at low temperatures. Unfortunately, technical difficulties and tight deadlines impaired these plans as they did with the plans to characterise promoter strengths.

Forced to omit characterising the strength of different RBSes at low temperatures, we decided to take our chances with the RBS team <A href="https://2010.igem.org/Team:Warsaw" class="external text" rel="nofollow">Warsaw 2010</A> defined as their standard against which to compare other RBS strengths (<A href="http://partsregistry.org/Part:BBa_B0034" class="external text" rel="nofollow">BBa_B0034</A>). Running on the crude assumption low temperatures affect all RBSes in an equivalent manner, we reasoned this RBS - which is relatively strong compared to other RBSes at 'normal' temperatures - should be relatively strong at low temperatures too.
</P><H2><SPAN class="mw-headline" id="Coding_regions">Coding regions</SPAN></H2><P>The novelty of the <I><B>icE. coli</B></I> project lies in its new submissions to the registry: 5 different coding regions, each encoding a protein reported to enhance cold resistance one way or another. Cold resistance, in this context, is defined as protein mediated mitigation of cold-induced systems failure. Such systems failure can be caused on many different levels, such as a decrease in the fluidity of the membrane; an excessive increase in structural rigidity of proteins, DNA and RNA; a drop in the rate of (bio)chemical reactions; and in extreme cases, mechanical damage caused by ice crystallisation. More often than not, low temperatures cause failure on several levels simultaneously, and mitigation of the effect on one level may be cancelled out by failure of another. In some cases, though, systems failure may be limited or prevented entirely by the expression of specific (combinations of) proteins.

Literature study reveals several proteins have been reported to decrease <I>E. coli</I> 's sensitivity to low temperatures. Team Amsterdam selected 5 of these proteins to create cold resistance BioBricks, or CryoBricks. They are all chaperones, originally found in cryophillic bacteria which thrive in cold environments, like the Arctic. The selected proteins are:
</P><UL><LI><I>Cpn10</I> and <I>Cpn60</I>, from <I>Oleispira antarctica</I></LI><LI><I>SheDnaK</I>, from <I>Shewanella</I> sp. AC10
</LI><LI><I>CspA</I> and <I>CspC</I>, from <I>Polaribacter irgensii</I></LI></UL><P>
All these proteins are, at least to a certain degree, homologous to chaperone proteins endogenous to <I>E. coli</I>. In addition, function loss of these endogenous chaperones is associated with cold-induced systems failure. Theoretically, expression of the cryophile chaperones listed above compensates for the loss of function endogenous chaperones experience at low temperatures, in doing so mitigating the impact of these temperatures on cellular activity.

Note that because of the many different levels at which cold-induced systems failure can occur, many different ways of characterising cold resistance are employed in different articles. Two prevalent ones used to characterise our CryoBricks are:
</P><UL><LI> the effect expression of a (combination of) protein(s) has on an organism's ability to survive freezing and thawing
</LI><LI> the effect expression of a (combination of) protein(s) has on an organism's specific growth rate at suboptimal temperatures
</LI></UL><P>
Especially the latter interpretation is of value to some of our <A href="https://2011.igem.org/Team:Amsterdam/Project/Applications" class="external text" rel="nofollow">applications</A>. Below, you'll find information on the biology of the selected chaperones, detailling how they contribute to cold resistance in <I>E. coli</I>. For more information on the methods and results of characterising them, refer to the <A href="https://2011.igem.org/Team:Amsterdam/Labwork/Characterisation" class="external text" rel="nofollow">CryoBrick characterisation</A> page.

Unfortunately, because transferring the parts to the pSB1C3 backbone proved too difficult in most cases, only <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K538000" class="external text" rel="nofollow">BBa_K538000</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K538004" class="external text" rel="nofollow">BBa_K538004</A> could be submitted to the registry as individual parts.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Cpn10-60_graphs.gif" title="File:Cpn10-60 graphs.gif">File:Cpn10-60 graphs.gif</A><DIV class="thumbcaption">Figure 1: (a) <I>E. coli</I> 's growth rate as a function of temperature, in strains that do (white circles) or do not (black circles) express <I>Cpn60/10</I>, and (b) <I>in vitro</I> refolding activities of <I>Cpn60/10</I> as a function of temperature (black circles), compared with that of <I>E. coli</I> 's endogenous <I>GroEL/ES</I> (white circles). <B>Adapted from Ferrer <I>et al.</I> (2003)</B></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Cpn10_and_Cpn60"><I>Cpn10</I> and <I>Cpn60</I></SPAN></H3><P><I>Cpn10</I> and <I>Cpn60</I> are homologous to <I>GroES</I> and <I>GroEL</I> of <I>E. coli</I>, respectively. The <I>GroEL/ES</I> chaperone system promotes the folding and/or assembly of over 30% of <I>E. coli'</I>s cellular proteins, is required for bacteriophage morphogenesis and has a role in protein secretion.<A href="http://www.nature.com/nature/journal/v355/n6355/abs/355033a0.html" class="external autonumber" rel="nofollow">[2]</A><A href="http://onlinelibrary.wiley.com/doi/10.1002/1521-3773%2820020402%2941:7%3C1098::AID-ANIE1098%3E3.0.CO;2-9/abstract" class="external autonumber" rel="nofollow">[3]</A> However, it rapidly loses its refolding activity at temperatures below 37°C.<A href="http://www.nature.com/nbt/journal/v21/n11/pdf/nbt1103-1266b.pdf" class="external autonumber" rel="nofollow">[4]</A> (Figure 1b) <I>Cpn60/10</I>, on the other hand, functions very well at these temperatures.

Ferrer <I>et al.</I> identified 22 housekeeping proteins involved with <I>E. coli</I> 's systems failure at low temperatures. They went on to demonstrate their interactions with chaperones are key determinants of activity at these temperatures. It was shown that the <I>Cpn60</I> protein from <I>O. antarctica</I> coprecipitates with many of the proteins found in the proteome of <I>E. coli</I>.<A href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.200500031/abstract" class="external autonumber" rel="nofollow">[5]</A> Also, by transforming <I>E. coli</I> with <I>Cpn10</I> and <I>Cpn60</I>, they've enabled it to grow even at freezing point. (Figure 1a) Their findings suggest inactivation of a few cold-sensitive key proteins causes systems failure in <I>E. coli</I>, and that cells may be 'rescued' by reactivating these genes.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><A href="/File:SheDnaK_graph.GIF" title="File:SheDnaK graph.GIF">File:SheDnaK graph.GIF</A><DIV class="thumbcaption">Figure 2: ATPase activity as a function of temperature, for the DnaK of <I>Shewanella</I> sp. Ac10 (SheDnaK, white squares) and <I>E. coli</I> (EcoDnaK, black circles). <B>Adapted from Yoshimune <I>et al.</I> (2005)</B></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="SheDnaK"><I>SheDnaK</I></SPAN></H3><P><I>DnaK</I> is a 70 kilodalton heatshock protein involved with many different processes. It's <I>E. coli'</I>s primary Hsp70 homolog<A href="http://www.jbc.org/content/270/18/10412.full" class="external autonumber" rel="nofollow">[6]</A>, and its function is mostly regulated by interactions with the smaller heatshock proteins <I>DnaJ</I> and <I>GrpE</I>. It's been debated whether or not <I>DnaK</I> is involved with assembling the ribosome's 30S subunit, but Maki, Southworth and Culver conclusively demonstrated this to be the case.<A href="http://rnajournal.cshlp.org/content/9/12/1418.full" class="external autonumber" rel="nofollow">[7]</A> It is also essential for lambda phage propagation, playing a key role in its replication by binding the <I>lambda P</I> protein, disassembling the <I>lambda P:DnaB</I> complex.<A href="http://www.jbc.org/content/268/7/4821.abstract" class="external autonumber" rel="nofollow">[8]</A>
Cold-induced inactivation of <I>EcoDnaK</I>, combined with the inability to refold it, is reported to be a causative agent of systems failure in <I>E. coli</I> at low temperatures.<A href="http://www.nature.com/nbt/journal/v21/n11/pdf/nbt1103-1266b.pdf" class="external text" rel="nofollow">[5</A>] Yoshimune <I>et al.</I> isolated <I>SheDnaK</I> from the Antarctic <I>Shewanella</I> bacterium, and observed it has a much higher ATPase activity at low temperatures than <I>EcoDnaK</I>, which is characteristic for cold-active enzymes.<A href="http://www.springerlink.com/content/f50n0gahppxacytb/" class="external autonumber" rel="nofollow">[9]</A> (Figure 2) They subsequently expressed <I>SheDnaK</I> in a <I>DnaK</I>-null mutant. This did <B>not</B> rescue <I>E. coli</I> 's growth at 43°C, nor did the strain support lambda phage propagation at 30°C, but it <B>did</B> allow their <I>SheDnaK</I> comprising <I>E. coli</I> to grow at temperatures as low as 15°C, which was impossible for the null-mutant. We use <I>SheDnaK</I> in our CryoBricks based on the speculation it may recover some of the functions <I>EcoDnaK</I> loses at low temperatures when they are expressed in parallel to eachother.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:296px;"><DIV class="thumbcaption">Figure 3: Freeze/thaw cycle survival rate of <I>E. coli</I> containing an empty vector (black circles), a vector carrying <I>P. irgensii</I> 's <I>CspA</I> gene (white circles), and a vector carrying <I>P. irgensii</I> 's <I>CspC</I> gene (white triangles). <B>Adapted from Uh <I>et al.</I> (2010)</B></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="CspA_and_CspC"><I>CspA</I> and <I>CspC</I></SPAN></H3><P>While exposure to low temperatures represses the expression of most proteins, cold shock protein (Csp) expression is actually induced by this. Jones and Inouye showed <I>E. coli'</I>s <I>CspA</I> concentration can increase over 200-fold within 1.5 hours, when exposed to cold shock.<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.1994.tb00359.x/pdf" class="external autonumber" rel="nofollow">[10]</A><I>CspA</I> functions as an RNA chaperone,<A href="http://www.jbc.org/content/272/1/196.full" class="external autonumber" rel="nofollow">[11]</A> capable of melting stable secundary structures in mRNA, and in doing so facilitating their translation at low temperatures.<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC207108/pdf/jbacter00084-0044.pdf" class="external autonumber" rel="nofollow">[12]</A><A href="http://www.jbc.org/content/274/6/3407.full" class="external autonumber" rel="nofollow">[13]</A> Its nucleic acid binding properties are mediated by a β-barrel structure, which is known as the cold shock domain and is common to all cold shock proteins whose structures have been clarified. In particular, two recurring motifs - RNP1 and RNP2 - are essential for cold shock proteins' ability to bind RNA.<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.1995.tb02431.x/abstract" class="external autonumber" rel="nofollow">[14]</A><I>E. coli</I> 's genome comprises a family of 9 highly similar RNA chaperones (<I>CspA</I> through <I>CspI</I>), 4 of which are cold-induced.<A href="http://www.springerlink.com/content/jtnbcavcdpg3xey4.pdf" class="external autonumber" rel="nofollow">[15]</A> Deletion of one, two or three of the cold-induced chaperones does not affect cell viability at low temperatures, but a quadruple knock-out strain exhibits a cold-sensitive phenotype. Xia, Ke and Inouye not only made and characterised these knock-out strains, but went on to show the cold-sensitive phenotype can be repressed by overexpressing any of the remaining members of the Csp family, with the exception of <I>CspD</I>.<A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.2001.02372.x/pdf" class="external autonumber" rel="nofollow">[16]</A> (Overexpressing <I>CspD</I> did, however, increase the cold resistance of the wild-type strain.) The apparent redundancy in the functioning of <I>E. coli</I> 's cold shock proteins suggests that they carry out a task which is key to cell survival.

Uh <I>et al.</I> identified a blind spot in the scientific community's understanding of cold shock proteins, stating the insights reported above are based mainly on studies of mesophilic bacteria. To fill this gap, they investigated a bacterium isolated from Arctic sea sediment (<I>Polaribacter irgensii</I>) and, following a homology-based selection procedure, cloned two of its genes (<I>CspA</I> and <I>CspC</I>) into <I>E. coli</I>.<A href="http://www.springerlink.com/content/qr2q5722j0t78653.pdf" class="external autonumber" rel="nofollow">[17]</A> They demonstrated expression of either of these genes can rescue the cold-sensitive phenotype Xia, Ke and Inouye observed in their quadruple knock-out. In addition to this, they show <I>P. irgensii</I> 's cold shock proteins drastically increase <I>E. coli</I> 's ability to survive freeze/thaw cycles. (Figure 3)
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL><LI><B>Huo, Martin &amp; Schleif</B> Alternative DNA loops regulate the arabinose operon in <I>Escherichia coli</I>, <I>Procl. Natl. Acad. Sci. USA 85, 5444-5448</I> (1988)
</LI><LI><B>Gething &amp; Sambrook</B> Protein folding in the cell, <I>Nature 355, 33–45</I> (1992)
</LI><LI><B>Walter &amp; Buchner</B> Molecular Chaperones — Cellular Machines for Protein Folding, <I>Angew. Chem. Int. Ed. Eng. 41, 1098–1113</I> (2002)
</LI><LI><B>Ferrer <I>et al.</I></B> Chaperonins govern growth of <I>Escherichia coli</I> at low temperatures, <I>Nat. Biotech. 21, 1266 - 1267</I> (2003)
</LI><LI><B>Strocchi, Ferrer, Timmis &amp; Golyshin</B> Low temperature-induced systems failure in <I>Escherichia coli</I>: Insights from rescue by cold-adapted chaperones, <I>Proteomics 6 (1), 193-206</I> (2005)
</LI><LI><B>Ziegelhoffer, Lopez-Buesa &amp; Craig</B> The Dissociation of ATP from hsp70 of <I>Saccharomyces cerevisiae</I> Is Stimulated by Both Ydj1p and Peptide Substrates, <I>J. Bio. Chem. 270, 10412-10419</I> (1995)
</LI><LI><B>Maki, Southworth &amp; Culver</B> Demonstration of the role of the DnaK chaperone system in assembly of 30S ribosomal subunits using a purified in vitro system, <I>RNA 9, 1418-1421</I> (2003)
</LI><LI><B>Osipiuk, Georgopoulos &amp; Zylicz</B> Initiation of lambda DNA replication - The Escherichia coli small heat shock proteins, DnaJ and GrpE, increase DnaK's affinity for the lambda P protein, <I>J. Bio. Chem. 268, 4821-4827</I> (1993)
</LI><LI><B>Yoshimune <I>et al.</I></B> Cold-active DnaK of an Antarctic psychrotroph <I>Shewanella</I> sp. Ac10 supporting the growth of <I>dnaK</I>-null mutant of <I>Escherichia coli</I> at cold temperatures <I>Extremophiles 9 (2), 145-150</I> (2005)
</LI><LI><B>Jones &amp; Inouye</B> The cold shock response - a hot topic. <I>Mol. Microbiol. 11, 811-818</I> (1994)
</LI><LI><B>Jiang, Hou &amp; Inouye</B> CspA, the major cold-shock protein of <I>Escherichia coli</I>, is an RNA chaperone. <I>J. Biol. Chem. 272, 196-202</I> (1997)
</LI><LI><B>Jones, Krah, Tafuri &amp; Wolffe</B> DNA gyrase, CS7.4, and the cold shock response in <I>Escherichia coli</I>. <I>J. Bacteriol. 174, 5798-5802</I> (1992)
</LI><LI><B>Schindler <I>et al.</I></B> The family of cold shock proteins of <I>Bacillus subtilis</I> - Stability and dynamics <I>in vitro</I> and <I>in vivo</I>. <I>J. Biol. Chem. 274, 3407-3413</I> (1999)
</LI><LI><B>Schröder <I>et al.</I></B> Mutational analysis of the putative nucleic acid-binding surface of the cold-shock domain, CspB, revealed an essential role of aromatic and basic residues in binding of single-stranded DNA containing the Y-box motif. <I>Mol. Microbiol. 16, 699-708</I> (1995)
</LI><LI><B>Ermolenko &amp; Makhatadze</B> Bacterial cold-shock proteins. <I>Cell. Mol. Life Sci. 59, 1902-1913</I> (2002)
</LI><LI><B>Xia, Ke &amp; Inouye</B> Acquirement of cold-sensitivity by quadruple deletion of the cspA family and its supression by PNPase S1 domain in <I>Escherichia coli</I>. <I>Mol. Microbiol. 40, 179-188</I> (2001)
</LI><LI><B>Uh <I>et al.</I></B> Rescue of a Cold-Sensitive Mutant at Low Temperatures by Cold Shock Proteins from <I>Polaribacter irgensii</I> KOPRI 22228 <I>J. Microbiol. 48, 798-802</I> (2010)
</LI></OL><DIV id="footer"><A href="https://2011.igem.org/Team:Amsterdam/Sponsors">iGEM Amsterdam 2011 is powered by:</A><A href="#top">Back to the Top</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Amsterdam/Biobricks/Basic_Parts">http://2011.igem.org/Team:Amsterdam/Biobricks/Basic_Parts</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Amsterdam/Biobricks/Basic_Parts" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Amsterdam/Biobricks/Basic_Parts" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Amsterdam/Biobricks/Basic_Parts&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Amsterdam/Biobricks/Basic_Parts&amp;oldid=152111" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>