<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_Hardware_Reagents skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/Hardware/Reagents</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header" class="sticky dark header-md noborder clearfix fixed"><HEADER id="topNav"><DIV class="container"><DIV class="navbar-collapse pull-right nav-main-collapse collapse submenu-dark"><NAV class="nav-main"><UL id="topMain" class="nav nav-pills nav-main"><LI><A href="https://2016.igem.org/Team:Valencia_UPV"> HOME </A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Team">MEET US</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Attributions">ATTRIBUTIONS</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="#"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Description">DESCRIPTION</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Software">DATA PROCESSING SOFTWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Proof" style="text-transform: none;">SPLIT Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Notebook">NOTEBOOK</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> LAB WARE </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware">HARDWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents">Reagents</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual">Manual</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Silver">SOCIAL ENGAGEMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Gold">SOLVING REAL PROBLEMS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Integrated_Practices">IMPACT ASSESSMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Safety">SAFETY</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> PARTS </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Part_Collection">BEST PART COLLECTION</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> MODEL</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Model">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/CP">Cas9:gRNA Complex Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Loop">R-Loop Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/ORF">Cleavage and Reading Frame Shift</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Conclusions">Conclusions</A></LI><LI><A class="active" href="https://2016.igem.org/Team:Valencia_UPV/Achievements"> ACHIEVEMENTS</A></LI></NAV></DIV></DIV></HEADER></DIV><DIV class="container"><H1>Reagents</H1></DIV><SECTION><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-2 col-sm-3"><DIV class="side-nav margin-bottom-60 margin-top-30"><DIV class="side-nav-head"><H4>Index</H4></DIV><UL class="list-group list-group-bordered list-group-noicon uppercase"><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents#Split-Cas9_id">Split-Cas9</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents#gRNATestingSystem_id">gRNA
                                Testing System</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents#Whywouldn%E2%80%99tourgRNAwork?_id">Why
                                wouldn’t our gRNA work?</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents#gRNAtestingmethods_id">gRNA
                                testing methods</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents#Ourdevice_id">Our
                                device</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents#Partsofthedevice(gRNATestingSystem1.0)_id">Parts of
                                the device (gRNA Testing System 1.0)</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents#Newpartsofthedevice(gRNATestingSystem2.0)_id">New
                                parts of the device (gRNA Testing System
                                2.0)</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents#References:_id">References:</A></LI></UL></DIV></DIV><DIV class="col-md-10 col-sm-9"><DIV class="blog-post-item" id="Split-Cas9_id"><H3>Split-Cas9</H3><P>We provide all the necessary constructions in order
                        to use autoreplicative vectors as Cas9 delivery system.
                        That means, providing both Cas9 halves fused with its
                        corresponding intein inserted into two different
                        vectors Tobacco Mosaic Virus (TMV) and Potato Virus X
                        (PVX).</P></DIV><DIV class="blog-post-item" id="gRNATestingSystem_id"><H3>gRNA Testing System</H3><P>Standardization and modularity are two of the
                        mainstays of Synthetic Biology. Bearing that in mind we
                        have design, built and tested a Phytobricks
                        standardized genetic circuit that allows to analyze the
                        efficiency of a given guide RNA (gRNA) in vivo.
                        We have designed a modular gRNA testing system that
                        allow us to determine the efficiency of any provided
                        gRNA in vivo. This system works as a genetic switch
                        that remains OFF until Cas9 endonuclease, guided by a
                        gRNA, performs a double stranded break inside the
                        target sequence. The device performance is based on the
                        presence of a luciferase reporter gene out of frame.
                        When a double strand break (DSB) occur due to Cas9
                        activity, a frameshift mutation may appear. Thus,
                        luciferase’s correct reading frame will be recovered
                        and bioluminescence will be detected through a
                        luciferase assay.
                        gRNA is a key element of CRISPR/Cas9 system. It leads
                        Cas9 endonuclease to the selected target in order to
                        produce a mutation. In order to maximize the mutation
                        efficiency in the desired crop variety gRNA must work
                        properly. Therefore gRNA design must be optimal.
                        Plant breeders could use the gRNA provided by our
                        software or any other one to directly edit their
                        desired crop. However, this process usually takes long
                        time and work turns time-consuming and in some cases
                        effortless. It would take a long time just to know if
                        the gRNA has worked or not. For instance, orange trees
                        need at least 1-2 years to exceed the first growth
                        stage, and 3-4 years to produce the first fruits.
                        Therefore, we encourage plant breeders to use our gRNA
                        testing system in order to determine the efficiency of
                        their RNA.</P></DIV><DIV class="blog-post-item" id="Whywouldn’tourgRNAwork?_id"><H3>Why wouldn’t our gRNA work?</H3><P>Our open-source database includes genes of interest,
                        which can be selected by plant breeders according to
                        their needs. It also directly provides the optimal gRNA
                        which, afterwards, must be ordered to synthesize. The
                        gRNA is obtained from gene consensus sequences acquired
                        from big databases such as NCBI or Sol Genomics.
                        However, the consensus sequence may match or not with
                        the specific variety to improve. Insertions and
                        deletions –Indels – and Single Nucleotide Polymorphisms
                        –SNPs – occur spontaneously and randomly in the genome
                        of plants in nature. That means the gRNA obtained by
                        our data processing software could not work optimally
                        on the desired variety. This makes almos mandatory to
                        test the gRNA in vivo before using it in the specific
                        variety. Furthermore, even if the gRNA matches
                        perfectly with the target sequence, mutagenesis may not
                        occur - or be less efficient - due to gRNA secondary
                        structure problems, as described by Summer et al.
                        (1).</P></DIV><DIV class="blog-post-item" id="gRNAtestingmethods_id"><H3>gRNA testing methods</H3><P>The current methods used to test the efficiency of
                        CRISPR/Cas9 consists in digesting amplified mutated
                        region with restriction enzymes or with T7
                        endonuclease. However, these methods are not suitable
                        if we want to be able to test any gRNA with a
                        reproducible method. Therefore, a new modular and
                        standard gRNA in vivo testing method claims to be
                        developed.</P><UL><LI>Digestion with restriction enzymes: the major
                            drawback of this method relies on the fact that
                            only targets that includes a restriction site in
                            DSB region are eligible. In order to check if the
                            mutation has occured, a digestion is performed with
                            the chosen restriction enzyme, and an
                            electrophoresis gel is carried out. If mutation
                            occured, the restriction site will be lost and the
                            enzyme won’t cut. Therefore, in the electrophoresis
                            gel, a band with higher molecular weight should be
                            observed. If mutation did not occur, two bands
                            should be observed, since the enzyme can recognize
                            the site and cut.</LI><LI>Problem: the range of possible targets
                                    is reduced, because they must contain a
                                    restriction site exactly in the place where
                                    the Cas9 cuts. Moreover, determining
                                    efficiency ratios through this method is
                                    difficult and not very accurate since it is
                                    based on band signal intensity
                                    comparison.</LI></UL><UL><LI>Digestion with T7 endonuclease: this strategy
                            is similar to the restriction enzymes one, yet it
                            uses the T7 endonuclease. This endonuclease cuts
                            where it finds heterodimers. When Cas9 cuts, due to
                            the non-homologous end joining DNA repair mechanism
                            - NHEJ -, plant cells introduce indels (2). After a
                            PCR amplification of the region, heterodimers can
                            be obtained from a denaturation and reannealing
                            step. When they are annealed, they might bind with
                            a strand which is not exactly complementary,
                            producing heterodimers that T7 can recognize and
                            cut. Therefore, in an electrophoresis gel, a high
                            molecular weight band will be observed if there is
                            not a cut, while, two shorter bands will appear if
                            T7 endonuclease cuts and so, Cas9 is well
                            working.</LI><LI>Problem: the T7 endonuclease is
                                    outrageously expensive. Moreover, this
                                    method does not allow to analyze the
                                    obtained mutation by sequencing since those
                                    frgaments with the mutation will be
                                    cut.</LI></UL><P>In order to provide an efficient solution to the
                        aforementioned drawbacks, we have engineered a gRNA
                        Testing System. In this strategy, we chose <I>Nicotiana
                        benthamiana</I> as chassis due to its fast growth and
                        all the benefits provided by a model plant. Our
                        methodology is Phytobricks standardized, thus turning
                        this system into a modular and standard one, two of the
                        mainstays of Synthetic Biology.</P></DIV><DIV class="blog-post-item" id="Ourdevice_id"><H3>Our device</H3><P>Any selected target can be inserted in the Testing
                        System since its Phytobricks standardized and its
                        overhang sequences are well defined. Based on
                        <I><I>Agrobacterium</I> tumefaciens</I> infection and
                        using luciferase as reporter gene, we will be able to
                        know how efficient the provided/designed gRNA is.
                        First of all, plant breeders must perform a genomic DNA
                        extraction of the plant they want to modify. Next, they
                        need to amplify the region they are going to target.
                        The primers needed to carry out this amplification are
                        provided by our Data Processing Software. The sequence
                        of the amplified target is then analyzed by sequencing.
                        Next step consists in ordering overlapping
                        oligonucleotides corresponding with the analyzed target
                        sequence. Finally the target is introduced in the
                        construction of our testing system, introduced in
                        A.tumefasciens and next in plant leaf cells by
                        agroinfiltration (Fig. 1).</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 1)
                            Schematic of the insertion of the target inside the
                            construction of gRNA Testing System circuit.</P></DIV><P>
                        This device is introduced in the plant along with the
                        corresponding gRNA and Cas9 construction. If the gRNA
                        leads Cas9 to the target and mutations are produced,
                        bioluminescence signal will be detected through a
                        luciferase assay.
                        The main advantages of this kind of assays are its
                        straightforward protocol (the analysis of each sample
                        only requires a few minutes) its enormous sensitivity
                        and its accuracy on the obtained results. Thus,
                        allowing us to obtain quantitative results.
                        The system is designed in order to be in OFF state
                        until the DSB and subsequent repair is produced. When
                        Cas9 cuts, indels appear, luciferase open reading frame
                        returns to the correct one and system turns ON (Fig. 2
                        and 3). Leaf samples are assayed for the presence of
                        the reporter gene by directly measuring the enzymatic
                        activity of luciferase enzyme in the presence of known
                        concentration of luciferin.</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 2)
                            Schematic of the function of the genetic circuit
                            that composes the gRNA Testing System. Only when
                            DSB and subsequent repair is produced the
                            luciferase’s open reading frame will shift to the
                            correct one (ON state).</P></DIV><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 3)
                            Device after endonuclease Cas9 cut and indels
                            production. a) The extra nucleotide is located
                            before the linker, so when indels change the frame,
                            the linker and the luciferase are in the correct
                            ORF. b) The extra nucleotide is located after the
                            linker. Thus linker’s frame will be different from
                            luciferase’s one after indel occurs.</P></DIV></DIV><DIV class="blog-post-item" id="Partsofthedevice(gRNATestingSystem1.0)_id"><H3>Parts of the device (gRNA Testing System 1.0)</H3><P>All the parts used in the construction of the gRNA
                        testing system 1.0 and following versions are described
                        <A href="https://2016.igem.org/Team:Valencia_UPV/Part_Collection">
                        here.</A></P><UL><LI>P35s: It is a strong constitutive promoter
                            derived from cauliflower mosaic virus (CaMV). It is
                            widely used in plants to improve the level of the
                            expression of foreign genes effectively in all
                            tissues (3).</LI><LI>5’ region: 5’ region from the <I>N.
                            benthamiana</I> polyubiquitin sequence (Accession
                            number: Nbv5.1tr6241949) is used due to its high
                            expression rate in every plant tissue (4). It can
                            help our gene to express itself and ensures that
                            the construction is well expressed. It contains
                            BsmbI and BsaI recognition sites with AATG in 3’ as
                            overhang that allows us to ligate with the
                            amplified target.</LI><LI>Target: Plant breeders will obtain the selected
                            gene from their original plant by PCR amplification
                            with the primers provided by our Data Processing
                            Software. These primers are designed in order to
                            obtain amplicons with the overhangs needed to
                            insert the target in our device, corresponding to
                            Phytobricks grammar (prefix: AATG, suffix: TTCG).
                            It’s mandatory to make sure that this region
                            doesn’t contain any stop codon in frame +1, but
                            neither in frame +2, so when an indel happens the
                            reading frame won’t be disrupted. The software
                            finds the optimal target with its corresponding
                            gRNA.</LI><LI>Linker SAGTI (Ser-Ala-Gly-Thr-Ile): it is a
                            flexible peptide linker between the target and the
                            luciferase. It allows the luciferase to acquire the
                            correct structure, avoiding interaction with the
                            target. Thus, luciferase assay will be carried out
                            in a successfully way. As it can be seen in Figure
                            3, the first part of the device is in ORF +1
                            whereas the luciferase is in ORF +2. The device is
                            designed so that there is an extra nucleotide
                            before the linker sequence that changes the reading
                            frame. If this nucleotide were after the linker
                            sequence, when Cas9 cut, the reading frame of the
                            linker will change, and the amino acids translated
                            won’t be the correct ones. Figure 3 shows how it
                            will be translated after the indels.</LI><LI>Luciferase: It is widely used as a reporter
                            enzyme. At the beginning, luciferase gene is out of
                            the reading frame due to the presence of an extra
                            nucleotide. After CRISPR/Cas9 acts indels occur.
                            These insertions and deletions of nucleotides
                            produce a frameshift and change luciferase’s frame
                            in the right frame +1, so it will be correctly
                            translated. In the experimental design, the first
                            methionine has been removed just to prevent the
                            unintended translation of the luciferase. Therefore
                            appearance of false positives will be avoided.</LI><LI>Tnos: Nopaline synthase terminator of the
                            nopaline synthase gene of <I><I>Agrobacterium</I>
                            tumefaciens</I>. It is used for gene transfection
                            (5).</LI></UL></DIV><DIV class="blog-post-item" id="Newpartsofthedevice(gRNATestingSystem2.0)_id"><H3>New parts of the device (gRNA Testing System
                        2.0)</H3><P>After building the device different luciferase were
                        performed in order to test its feasibility. Once the
                        obtained results were analyzed (See results <A href="https://2016.igem.org/Team:Valencia_UPV/Results">here</A>),
                        we determined that the system were working well but its
                        performance could be improved. Bearing that in mind, we
                        resolved to remove the 5’ region in order to obtain
                        higher levels of luciferase expression and also to test
                        different linkers and choose the most appropriate one.
                        The new selected linkers are:</P><UL><LI>EAAAK linker: rigid helical linker with the
                            sequence of A(EAAAK)nA (n =3) that effectively
                            separates protein domains as reported by Chen et
                            al. (6). An extra nucleotide was added following
                            the same strategy design that with SAGTI
                            linker.</LI><LI>RSIAT linker: short linker that separates
                            protein domains and that have been succesfully used
                            BiFC experiments (7). An extra nucleotide was added
                            following the same strategy design that the
                            aforementioned linkers.</LI><LI>RSIAT+TEV linker: RSIAT linker (7) followed by
                            a TEV protease recognition site. Therefore Tobacco
                            Etch Virus NIa protease will recognize the sequence
                            and cleave it, obtaining two different peptides as
                            result. On one side target plus linker peptide, and
                            on the other side luciferase protein. With this
                            linker we aim to obtain a more orthogonal design in
                            which the target sequence has minimal interference
                            with luciferase protein. Thus, luminescence signal
                            will not depend on the target.</LI></UL></DIV><DIV class="blog-post-item" id="References:_id"><H3>References:</H3><P>1. Thyme S, Akhmetova L, Montague T, Valen E, Schier
                        A. Internal guide RNA interactions interfere with
                        Cas9-mediated cleavage. Nature Communications.
                        2016;7:11750.
                        2. Biolabs N. Determining Genome Targeting Efficiency
                        using T7 Endonuclease I | NEB (Internet). Neb.com. 2016
                        (cited 27 July 2016). Available from:
                        https://www.neb.com/protocols/2014/08/11/determining-genome-targeting-efficiency-using-t7-endonuclease-i

                        3. Pauli S, Rothnie H, Chen G, He X, Hohn T. The
                        Cauliflower Mosaic Virus 35S Promoter Extends into the
                        Transcribed Region. Journal of Virology.
                        2004;78(22):12120-12128.
                        4. Benthamiana Atlas (Internet). Sefapps02.qut.edu.au.
                        2016 (cited 27 June 2016). Available from:
                        http://sefapps02.qut.edu.au/atlas/tREXXX2new.php?TrID=Nbv5.1tr6241949

                        5. Holden M, Levine M, Scholdberg T, Haynes R, Jenkins
                        G. The use of 35S and Tnos expression elements in the
                        measurement of genetically engineered plant materials.
                        Analytical and Bioanalytical Chemistry.
                        2009;396(6):2175-2187.
                        6. Chen X, Zaro J, Shen W. Fusion protein linkers:
                        Property, design and functionality. Advanced Drug
                        Delivery Reviews. 2013;65(10):1357-1369.
                        7. Hu CKerppola T. Simultaneous visualization of
                        multiple protein interactions in living cells using
                        multicolor fluorescence complementation analysis.
                        Nature Biotechnology. 2003;21(5):539-545.</P></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer" style="margin-bottom:-20px"><DIV class="container"><DIV class="row"><DIV class="col-md-5"><DIV class="row"><DIV class="col-md-6 col-sm-6 hidden-xs" style="padding:0;"><DIV class="margin-left-50 clearfix"><H4 class="margin-bottom-10" style="text-align:start;">Follow
                                Us</H4></DIV></DIV></DIV></DIV><DIV class="col-md-7"><H4 class="letter-spacing-1" style="text-align: start;">Sponsors</H4></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>