Team:Valencia UPV/Proof
HOME
TEAM
MEET US
COLLABORATIONS
ATTRIBUTIONS
PROJECT
DESCRIPTION
DATA PROCESSING SOFTWARE
SPLIT Cas9
RESULTS
NOTEBOOK
LAB WARE
HARDWARE
Reagents
Manual
HUMAN PRACTICES
OVERVIEW
SOCIAL ENGAGEMENT
SOLVING REAL PROBLEMS
IMPACT ASSESSMENT
SAFETY
PARTS
BEST PART COLLECTION
MODEL
OVERVIEW
Cas9:gRNA Complex Formation
R-Loop Formation
Cleavage and Reading Frame Shift
Conclusions
ACHIEVEMENTS
Split Cas9
Index
Abstract
Plant genome editing with CRISPR/Cas9
Technical requirements
Our solution
Superinfection exclusion
gRNA delivery
References:
Abstract
The usage of autoreplicative vectors as vehicle for Cas9 delivery entails a set of benefits. Of all these advantages the most remarkable one is the possibility of removing any trace of foreign DNA sequences once the mutation has occurred. Unfortunately, autoreplicative vectors cannot store SpCas9 massive coding sequence.
Regarding the last point, we resolve to divide SpCas9 into two halves. This novel split-Cas9 strategy allows us to deliver CRISPR/Cas9 system inside plant cells through autoreplicative vectors.
The utilization of split-Cas9 strategy along with these autoreplicative vehicles emerges as a new and revolutionary plant breeding technique.
Plant genome editing with CRISPR/Cas9
CRISPR/Cas9 technology has emerged as a novel and revolutionary genome editing technique that surpass conventional and mutation plant breeding techniques in terms of versatility and easy RNA programming. Thus, allowing us to modify a specific trait from one crop in order to obtain a brand new variety of known genotype.
This targeted plant genome editing strategy has a great potential for crop improvement to meet the world’s growing demand for food and to provide sustainable productive agriculture system simultaneously (1).
Many genes from different plant species has been successfully modified since the first application of CRISPR/Cas9 was reported in plants (2, 3 and 4). Thus, sparking off a whole new way of understanding and carrying out crop improvement while offering a whole new world of possibilities when it comes to targeted plant genome editing (Fig. 1).
Fig. 1 Number of genes reported edited by CRISPR/Cas9 to date, by plant species (1).
The CRISPR/Cas9 system, as plant genome editing technology, is classified inside the New Plant Breeding Techniques (NPBTs). NPBTs are faster than traditional breeding methods and can produce a null segregant line that notably lacks the transgenic insert (5, 6, 7 and 8). The plants developed by NPBTs are identical to the classically bred plants since both carry with the desired mutation but not with any transgene.
Technical requirements
Plant genomic editing with CRISPR/Cas9 system is usually harnessed to obtain new allelic series by producing indels due to Non Homolougus End Joining (NHEJ) repair. In order to generate those mutations its two main components – a guide RNA and Cas9 endonuclease – must be delivered inside the cell.
The first and most employed delivery method for CRISPR/Cas9 in plants is Agrobacterium-based transformation, which introduces Cas9 and gRNA coding sequences directly into the plant genome (2, 3 and 4). Nevertheless, this delivery method has a major drawback since the introduction of new coding sequences implies the generation of transgenic plants, with all the social and legal issues that they entail. Although a segregant line without transgenes can be obtained the process is expensive and time-consuming. The generation of new plants requires a first step of transformation to introduce foreign DNA, and the subsequent steps of segregation and genotyping up to obtain a homozygous line (Fig. 2).
Fig. 2 The generation of transgene free genetically edited crops. Transgene free homozygous mutants with the desired mutation could be selected by selfing plants of the generation zero and segregating the transgene in the next generation (1).
In this context the viral delivery of genes arises as a new tool, faster and cheaper, for crop improvement. Moreover, its higher infection and expression ratios make this system the most suitable for CRISPR/Cas9 delivery (9, 10 and 11). On the other hand, it is possible to remove viral particles from the plant through thermal or chemical treatments in order to eliminate the foreign coding sequences. Unfortunately, these viral approaches are chiefly limited by the size of their nucleic acid cargo, especially with regard to spCas9 4.2 kb coding sequence.
Our solution
Bearing in mind the aforementioned advantages and drawbacks of CRISPR/Cas9 system and the possible delivery methods, and aiming to maximize mutation efficiency, viral systems would be the perfect delivery strategy.
Moreover, the delivery of Cas9 and gRNA through viral systems will prevent new developed varieties from transgenic related legal and social issues. That is due to the possibility of treating modified plants in order to eliminate viral particles and therefore removing Cas9 and gRNA coding sequence from plant cells.
Following this strategy we are able to obtain new transgene-free modified crops. Unfortunately it is impossible to employ viral vectors as Cas9 delivery system due to the endonuclease’s massive coding sequence. In order to bypass this problem we followed a strategy developed by Truong et al. (12).
This approach is based on the division of SpCas9 endonuclease into two parts and the addition of an intein moiety to each half in order to allow the reconstitution of the protein. Thus, shortening the length of the coding sequences and making possible their introduction into viral vectors. The site between Lys637 and Thr638 was chosen as split-site as described by Truong et al. (12). The split-intein-Cas9 system was created by choosing the site between Lys637 and Thr638 as split-site, while fusing NpuDnaE N-intein to the C-terminus of SpCas9 and NpuDnaE N-terminus was fused to the N-terminus of the endonuclease (Fig. 3).
(Fig. 3) Schematic of split-intein-cas9 strategy.
Human codon optimized SpCas9 version was chosen since it has been reported to have the highest yield in Nicotiana benthamiana (13). Moreover, NpuDnaE inteins from Heidelberg 2014 team intein library were chosen ( Part:BBa_K1362400 , [Part:BBa_K1362401 path: http://parts.igem.org/wiki/index.php?title=Part:BBa_K1362401]). Therefore, we did not only test the feasibility of split-intein-Cas9 strategy but also check the activity of the inteins our chassis Nicotiana benthamiana.
See Split-Cas9 results here
Superinfection exclusion
Once we resolved to insert SpCas9 in two different pieces inside two different vectors, the next challenge was to solve the superinfection exclusion problem. Superinfection exclusion is a phenomenon in which a preexisting viral infection prevents a secondary infection with the same or a closely related virus. The mechanisms that prevent the infection of a determined virus into a previously infected cell are only partially understood (14). As shown in Figure 4 when a leaf is coinfiltrated with two different Agrobacterium tumefasciens cultures carrying two different TMV vectors with two different fluorescent proteins (YFP and DsRed) a mosaic appeared. However, when a leaf is coinfiltrated with two different Agrobacterium tumefasciens cultures carrying a TMV vector and a PVX one with different fluorescent proteins all the cells expressed both proteins.
(Fig. 4) A) Mosaic produced by the coinfection of the leaf with two different Agrobacterium tumesfasciens with two different TMV vectors carrying YFP and DsRED. B) YFP signal is shown in the whole leaf due to the coinfection with TMV viral system carrying GFP and PVX. C) The same leaf as in B) presenting also DsRed on its whole surface due to the absence of superinfection exlcusion.
gRNA delivery
Since CRISPR/Cas9 system is composed by two main elements, guide RNA delivery must be also considered. Bearing that in mind, we propposed two different solutions based on the strategy described by Gao et al. (15) Therefore, we propose to flank gRNA transcript by Hammerhead and Hepatitis Delta Virus self‐cleaving ribozymes (Fig. 5) and insert it along with one of the two halves of the Split-Cas9 into the same vector or using a third viral vector that does not show superinfection exclusion with the other two.
(Fig. 5) Structure of the pre-gRNA containing a Hammerhead ribozyme at the 5’ end, and a HDV ribozyme at the 3’ end (15).
References:
1.Khatodia S, Bhatotia K, Passricha N, Khurana S, Tuteja N. The CRISPR/Cas Genome-Editing Tool: Application in Improvement of Crops. Frontiers in Plant Science. 2016;7.
2. Li J, Norville J, Aach J, McCormack M, Zhang D, Bush J et al. Multiplex and homologous recombination–mediated genome editing in Arabidopsis and Nicotiana benthamiana using guide RNA and Cas9. Nature Biotechnology. 2013;31(8):688-691.
3. Nekrasov V, Staskawicz B, Weigel D, Jones J, Kamoun S. Targeted mutagenesis in the model plant Nicotiana benthamiana using Cas9 RNA-guided endonuclease. Nature Biotechnology. 2013;31(8):691-693.
4. Shan Q, Wang Y, Li J, Zhang Y, Chen K, Liang Z et al. Targeted genome modification of crop plants using a CRISPR-Cas system. Nature Biotechnology. 2013;31(8):686-688.
5. Podevin N, Devos Y, Davies H, Nielsen K. Transgenic or not? No simple answer!. EMBO reports. 2012;13(12):1057-1061.
6. Araki MIshii T. Towards social acceptance of plant breeding by genome editing. Trends in Plant Science. 2015;20(3):145-149.
7. Schaart J, van de Wiel C, Lotz L, Smulders M. Opportunities for Products of New Plant Breeding Techniques. Trends in Plant Science. 2016;21(5):438-449.
8. Woo J, Kim J, Kwon S, Corvalán C, Cho S, Kim H et al. DNA-free genome editing in plants with preassembled CRISPR-Cas9 ribonucleoproteins. Nat Biotechnol. 2015;33(11):1162-1164.
9. Ali Z, Abul-faraj A, Piatek M, Mahfouz M. Activity and specificity of TRV-mediated gene editing in plants. Plant Signaling & Behavior. 2015;10(10):e1044191.
10. Baltes N, Hummel A, Konecna E, Cegan R, Bruns A, Bisaro D et al. Conferring resistance to geminiviruses with the CRISPR–Cas prokaryotic immune system. NPLANTS. 2015;1(10):15145.
11. Yin K, Han T, Liu G, Chen T, Wang Y, Yu A et al. A geminivirus-based guide RNA delivery system for CRISPR/Cas9 mediated plant genome editing. Sci Rep. 2015;5:14926.
12. Truong D, Kühner K, Kühn R, Werfel S, Engelhardt S, Wurst W et al. Development of an intein-mediated split–Cas9 system for gene therapy. Nucleic Acids Res. 2015;43(13):6450-6458.
13. Vazquez-Vilar M, Bernabé-Orts J, Fernandez-del-Carmen A, Ziarsolo P, Blanca J, Granell A et al. A modular toolbox for gRNA–Cas9 genome engineering in plants based on the GoldenBraid standard. Plant Methods. 2016;12(1).
14. Julve J, Gandía A, Fernández-del-Carmen A, Sarrion-Perdigones A, Castelijns B, Granell A et al. A coat-independent superinfection exclusion rapidly imposed in Nicotiana benthamiana cells by tobacco mosaic virus is not prevented by depletion of the movement protein. Plant Molecular Biology. 2013;81(6):553-564.
15. Gao YZhao Y. Self-processing of ribozyme-flanked RNAs into guide RNAsin vitro and in vivo for CRISPR-mediated genome editing. Journal of Integrative Plant Biology. 2014;56(4):343-349.
Follow                                 Us
Sponsors
