<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_Hardware_Manual skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/Hardware/Manual</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header" class="sticky dark header-md noborder clearfix fixed"><HEADER id="topNav"><DIV class="container"><DIV class="navbar-collapse pull-right nav-main-collapse collapse submenu-dark"><NAV class="nav-main"><UL id="topMain" class="nav nav-pills nav-main"><LI><A href="https://2016.igem.org/Team:Valencia_UPV"> HOME </A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Team">MEET US</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Attributions">ATTRIBUTIONS</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="#"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Description">DESCRIPTION</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Software">DATA PROCESSING SOFTWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Proof" style="text-transform: none;">SPLIT Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Notebook">NOTEBOOK</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> LAB WARE </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware">HARDWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents">Reagents</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual">Manual</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Silver">SOCIAL ENGAGEMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Gold">SOLVING REAL PROBLEMS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Integrated_Practices">IMPACT ASSESSMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Safety">SAFETY</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> PARTS </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Part_Collection">BEST PART COLLECTION</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> MODEL</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Model">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/CP">Cas9:gRNA Complex Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Loop">R-Loop Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/ORF">Cleavage and Reading Frame Shift</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Conclusions">Conclusions</A></LI><LI><A class="active" href="https://2016.igem.org/Team:Valencia_UPV/Achievements"> ACHIEVEMENTS</A></LI></NAV></DIV></DIV></HEADER></DIV><DIV class="container"><H1>User manuals</H1></DIV><SECTION><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-2 col-sm-3"><DIV class="side-nav margin-bottom-60 margin-top-30"><DIV class="side-nav-head"><H4>Index</H4></DIV><UL class="list-group list-group-bordered list-group-noicon uppercase"><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Targetsearch_id">Target
                                search</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Identificationofthetarget_id">Identification
                                of the target</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#gRNAtestingsystem_id">gRNA
                                testing system</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Plantediting_id">Plant
                                editing</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Modifiedplantobtention_id">Modified
                                plant obtention</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#LabwareUserManual_id">Labware
                                User Manual</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Electroporator_id">Electroporator</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Colorimeter_id">Colorimeter</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Centrifuge_id">Centrifuge</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Gelelectrophoresisunit_id">Gel
                                electrophoresis unit</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Stirrerwithtemperaturecontrol_id">Stirrer
                                with temperature control</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Thermalcycler_id">Thermal
                                cycler</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Luminometer_id">Luminometer</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual#Phytotron_id">Phytotron</A></LI></UL></DIV></DIV><DIV class="col-md-10 col-sm-9"><DIV class="blog-post-item" id="HandbookforHYPE-ITediting_id"><H3>Handbook for HYPE-IT editing</H3><P>Here we show detailed instructions of how to edit a
                        plant using the HYPE-IT system based on Cas9. The
                        process is explained, with links to the specific
                        protocols.</P></DIV><DIV class="blog-post-item" id="Targetsearch_id"><H4>Target search</H4><P>Log in the Data Processing Software’s Web
                        http://hypeit.cloudno.de/
                        A) Unknown target</P><OL><LI>Go to “Target Search (DDBB)” to access the
                            search engine.</LI><LI>Search your phenotypic trait of interest to
                            know the gene to edit. You can choose it filling up
                            three fields: plant name or species, phenotypic
                            characteristic and/or the name of the gene.</LI><LI>You will be provided with a list of possible
                            options related to your search. To see details of
                            your choice, click on the left icon.</LI><LI>Here you have all the information about the
                            target gene. Click on “Show more” to access the
                            Data Processing Software and find the optimal gRNA
                            for your target. Move to step 5.</LI></OL><P>
                        B) Known target</P><OL><LI>Go to “Find optimal gRNA” to access the Data
                            Processing Software.</LI><LI>Paste the DNA sequence of the known gene in
                            FASTA format and click on “Find Targets”</LI></OL><P>
                        C) In both cases</P><OL><LI>The software shows a list of possible gRNAs
                            with their score. Select the highest one and you
                            will be provided with the optimal gRNA.
                            Additionally, the software provides the primers to
                            amplify the target of your plant with the overhangs
                            to use them with the Goldenbraid System with the
                            HYPE-IT strategy.</LI><LI>Order to synthesize to any synthesis company of
                            your choice the two primer oligonucleotides (DNA
                            sequences):</LI><LI>Primers + Goldenbraid overhangs
                                    Forward</LI><LI>Primers + Goldenbraid overhangs
                                    Reverse</LI><LI>The gRNA is synthetized after step 6 of the
                            gRNA testing system protocol.</LI><LI>RNA + Goldenbraid overhangs
                                    Forward</LI><LI>gRNA + Goldenbraid overhangs
                                    Reverse</LI><LI>Target consensus + Goldenbraid
                                    overhangs Forward</LI><LI>Target consensus + Goldenbraid
                                    overhangs Reverse</LI><LI>Target knockout + Goldenbraid overhangs
                                    Forward</LI><LI>Target knockout + Goldenbraid overhangs
                                    Reverse</LI></OL><P>
                        These overhangs will allow to insert the amplified gene
                        fragment of your variety in our gRNA Testing System.
                        The gRNA Goldenbraid overhangs are used to insert the
                        gRNA in a plasmid based in Goldenbraid for plant
                        infection.</P></DIV><DIV class="blog-post-item" id="Identificationofthetarget_id"><H4>Identification of the target</H4><OL><LI>Perform a genomic DNA extraction of the plant.
                            The protocol used will depend on the plant and
                            variety that you use. As an example, you can check
                            <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#OrangeDNAGenomeExtractionProtocol(AdaptedfromDelaportaextractionprotocol)_id">
                                here</A> the protocols we have used for orange
                                and rice extraction.
                            </LI><LI>Run electrophoresis gel (agarose 1%) of the DNA
                            extraction following <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Agarosegelelectrophoresis_id">
                                this protocol</A>. You should see a band higher
                                than the molecular weight ladder (above 10kb).
                            </LI><LI>Perform a PCR in the thermal cycler of the
                            genomic DNA with the ordered primers with
                            Goldenbraid overhangs in order to obtain the
                            fragment of the target gene. Depending on the
                            primers given by the Software and the polymerase
                            you use, the hybridization temperature for the PCR
                            will be different. You can use an online calculator
                            to know this temperature. It is usually provided by
                            the company where the primers were bought. When
                            using the HYPE-IT thermal cycler, it should be used
                            mineral oil to cover the liquid, in order to avoid
                            evaporation.</LI><LI>Run electrophoresis gel (agarose 1%) of the PCR
                            products following <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Agarosegelelectrophoresis_id">
                                this protocol</A>. You should see a band lower
                                than 100kb.
                            </LI><LI>Send the PCR product to sequence to a
                            sequencing service. The primers you can use to
                            sequence are the same used to amplify the gene
                            fragment.</LI><LI>Once you know the sequence of the gene you want
                            to edit, you have two possible paths to
                            follow:</LI><LI style="list-style-type: lower-alpha"> If the sequence is the same that the
                                    gRNA chosen in the Data Processing
                                    Software, you can order to synthetize the
                                    gRNA + Goldenbraid overhangs shown.
                                    Additionally, you will need to synthetize
                                    the target consensus + Goldenbraid
                                    overhangs sequences.</LI><LI>If the sequence is different that
                                    the gRNA chosen in the Data Processing
                                    Software, you can either choose a different
                                    gRNA in the software (repeating everything
                                    from step 1) or you can order to synthetize
                                    your own sequence adding these
                                    overhangs:</LI><LI>ATTG + sequence: gRNA
                                            forward</LI><LI>AAAC + sequence (reverse
                                            complementary): gRNA reverse</LI><LI>AATG + sequence: target
                                            consensus forward</LI><LI>CGAA + sequence (reverse
                                            complementary): target consensus
                                            reverse.</LI><LI>AATG + sequence removing the
                                            16th nucleotide: target knockout
                                            forward</LI><LI>CGAA + sequence removing the
                                            16th nucleotide (reverse
                                            complementary): target knockout
                                            reverse.</LI><LI>To obtain the functional gRNA, put in an
                            eppendorf the gRNA forward and gRNA reverse
                            oligonucleotides ordered in step 6. It is necessary
                            to put 18μL of water and 1μL of each
                            oligonucleotide. Leave 30 minutes at room
                            temperature.</LI><LI>Make a ligation reaction with the gRNA obtained
                            in step 6, U6 promoter and psgRNA (both provided in
                            the reagents kit). Follow the ligation protocol to
                            <A href="https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Ligationreaction_id">
                                ligate in α1</A>. You will obtain the device
                                U6:gRNA:psgRNA. Now you can directly edit your
                                plant or you can test if you wish the
                                efficiency of the gRNA in our gRNA testing
                                system.
                            </LI></OL></DIV><DIV class="blog-post-item" id="gRNAtestingsystem_id"><H3>gRNA testing system</H3><OL><LI>Ligation using Goldenbraid assembly system to
                            obtain the complete gRNA testing system device. The
                            ligation is made following the <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Ligationreaction_id">
                                ligation protocol</A>. The plasmids, linker and
                                promoters are provided in the reagents kit. The
                                ligations to be made are:
                            </LI><LI style="list-style-type: lower-alpha">In α1 plasmid: 35s or 35s:5’ + target
                                    consensus + linker (BBa_K2017003,
                                    BBa_K2017004, BBa_K2017005 or BBa_K2017006)
                                    + Tnos.</LI><LI style="list-style-type: lower-alpha">In α1 plasmid: 35s or 35s:5’ + target
                                    knockout + linker (BBa_K2017003,
                                    BBa_K2017004, BBa_K2017005 or BBa_K2017006)
                                    + Tnos.</LI><LI>Transform electrocompetent <I>E. coli</I> with
                            the ligation products of step 1 following the
                            <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#E.coliTransformation_id">
                                transformation protocol</A>.
                            </LI><LI>Culture <I>E. coli</I> in a Petri dish with
                            kanamycin resistance, as stated by <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Petridishculture_id">
                                petri dish culture protocol</A>.
                            </LI><LI>After waiting the time specified in the
                            protocol in point 3, pick white colonies in the
                            Petri dish and make a <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Liquidculture_id">
                                liquid culture</A>.
                            </LI><LI>Perform a miniprep to extract the plasmid of
                            <I>E. coli</I> and check if the selected colonies
                            have the desired plasmid. We use the E.Z.N.A ® ®.
                            Plasmid Mini Kit I, whose instructions are
                            explained in the <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Minipreps_id">
                                protocols</A></LI><LI>Digestion of minipreps with EcoRI (specific
                            enzyme for α1 digestion). Here you can find the
                            <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#DigestionProtocol_id">
                                digestion protocol</A></LI><LI>Run <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Agarosegelelectrophoresis_id">
                                electrophoresis gel</A> (agarose 1%) of the
                                digestion products. You should see two bands,
                                one at 6kb and other between 3 and 4kb. Steps 6
                                and 7 are made to check if the plasmid in the
                                picked colonies is the plasmid ligated in step
                                1.
                            </LI><LI>To obtain the functional target consensus, put
                            in an eppendorf the target consensus forward and
                            target consensus reverse oligonucleotides ordered
                            in step 6 of “Identification of the target”
                            section. It is necessary to put 18μL of water and
                            1μL of each oligonucleotide. Leave 30 minutes at
                            room temperature.</LI><LI>Repeat the same as in step 8 to obtain the
                            target knockout, mixing target knockout forward and
                            target knockout reverse.</LI><LI>As our experiments showed that the system was
                            more efficient in cis (Cas9 and gRNA in the same
                            plasmid as the testing system). So, ligate the
                            U6:gRNA:psgRNA with the plasmids obtained in step 5
                            (testing system target consensus and testing system
                            target knockout), in Ω2 plasmid, as in <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Ligationreaction_id">
                                ligation protocol</A>. Each testing system must
                                be ligated separately with the U6:gRNA:psgRNA
                                (only target consensus) and with
                                U6:XT1gRNA:psgRNA.
                            </LI><LI>Transform electrocompetent <I>E. coli</I> with
                            the ligation products of step 10 following the
                            <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#E.coliTransformation_id">
                                transformation protocol</A>.
                            </LI><LI>Repeat steps 3 to 7 for the transformed <I>E.
                            coli</I> in step 11.</LI><LI>Ligate the plasmids obtained in step 10 with
                            the 35s:Cas9:Tnos + SF (provided in the reagents
                            kit), in an α1 plasmid. Repeat steps 11 and
                            12.</LI><LI>Transform <I>Agrobacterium</I> C58 with the
                            extracted plasmids in the minipreps. Repeat the
                            same protocol as <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#E.coliTransformation_id"><I>E. coli</I> transformation</A>, but using
                                1440V. You should have three plasmids:
                            </LI><LI>Cas9 + gRNA testing system with
                                    Target consensus + gRNA</LI><LI>Cas9 + gRNA testing system with
                                    Target consensus + XT1gRNA</LI><LI>Cas9 + gRNA testing system with
                                    Target knockout + XT1gRNA</LI><LI>Repeat steps 3 to 7 for the transformed
                            <I>Agrobacterium</I> in step 14.</LI><LI>Agroinfiltrate <I>Nicotiana benthamiana</I>
                            with the three different <I>Agrobacterium</I>
                            obtained in step 15. Follow the <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Agrobacteriuminfiltration_id">
                                Agroinfiltration protocol</A>. The experimental
                                design is of your choice. We suggest to make
                                the following experiments (the devices not
                                created by you are provided in the reagents
                                kit). Each point represents one plant to be
                                agroinfiltrated with the stated agrobacterium
                                cultures.
                            </LI><LI>35s:Luciferase:Tnos (positive
                                    control)</LI><LI>pNOS:Luciferase:Tnos (negative
                                    control)</LI><LI>Cas9:gRNA testing system with Target
                                    consensus:gRNA + 35:Renilla:Tnos
                                    (sample)</LI><LI>Cas9:gRNA testing system with Target
                                    consensus: XT1gRNA + 35:Renilla:Tnos
                                    (negative control)</LI><LI>Cas9:gRNA testing system with Target
                                    knockout:XT1gRNA + 35:Renilla:Tnos
                                    (positive control)</LI><LI>Three days post-infiltration, perform a
                            luciferase assay to test the efficiency of the gRNA
                            you chose in the database or you designed after
                            sequencing. It is recommended to take at least
                            three replicates per agroinfiltrated leaf. You can
                            follow the <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Luciferaseassay_id">
                                luciferase assay protocol here</A>.
                            </LI></OL></DIV><DIV class="blog-post-item" id="Plantediting_id"><H4>Plant editing</H4><OL><LI>It is going to be used the U6:gRNA:psgRNA
                            device obtained in step 8 of “Identification of the
                            target” section. However, instead of using the
                            overhang ATTG in the forward brand, it will be used
                            the overhang AGCC.</LI><LI>Ligate the U6:gRNA:psgRNA with the TMV
                            split-Cas9 vector (provided in the reagents kit).
                            Follow the <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Ligationreaction_id">
                                ligation protocol</A>.
                            </LI><LI>Transform electrocompetent <I>E. coli</I> with
                            the ligation products of step 1 following the
                            <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#E.coliTransformation_id">
                                transformation protocol</A>.
                            </LI><LI>Culture <I>E. coli</I> in a Petri dish with
                            kanamycin resistance, as stated by <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Petridishculture_id">
                                petri dish culture protocol</A>.
                            </LI><LI>After waiting the time specified in the
                            protocol in point 3, pick white colonies in the
                            Petri dish and make a <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Liquidculture_id">
                                liquid culture</A>.
                            </LI><LI>Perform a miniprep to extract the plasmid of
                            <I>E. coli</I> and check if the selected colonies
                            have the desired plasmid. We use the E.Z.N.A ® ®.
                            Plasmid Mini Kit I, whose instructions are
                            explained in the <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Minipreps_id">
                                protocols</A></LI><LI>Digestion of minipreps with EcoRI (specific
                            enzyme for α1 digestion). Here you can find the
                            <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#DigestionProtocol_id">
                                digestion protocol</A></LI><LI>Run <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#Agarosegelelectrophoresis_id">
                                electrophoresis gel</A> (agarose 1%) of the
                                digestion products. You should see two bands,
                                one at 6kb and other between 3 and 4kb. Steps 6
                                and 7 are made to check if the plasmid in the
                                picked colonies is the plasmid ligated in step
                                2.
                            </LI><LI>Transform the plasmid obtained in step 6 in
              <I>Agrobacterium</I>, following the same steps from
                            3 to 7.</LI><LI>Agroinfiltrate <I>Nicotiana benthamiana</I>
                            with split-Cas9 + gRNA in TMV and split-Cas9 in PVX
                            (provided in the reagents kit). Follow the
                                <A href="%20https://2016.igem.org/Team:Valencia_UPV/Notebook/Protocol#E.coliinfiltration_id">
                                agroinfiltration protocol</A>. Take the
                                necessary preventive measures to avoid contact
                                of the agroinfiltrated plants with other
                                plants. The viral systems are safe and do not
                                have the coat protein, but it is still
                                advisable to manage it with caution.
                            </LI></OL></DIV><DIV class="blog-post-item" id="Modifiedplantobtention_id"><H4>Modified plant obtention</H4><OL><LI>Five days post-agroinfiltration, make
                            micropropagation of callus obtained from an
                            infected leave of the plant.</LI><LI>Make in vitro culture of the callus in a
                            phytotron to grow a plant with its cells edited.
                            The grown plant will have the knockout.</LI></OL></DIV><DIV class="blog-post-item" id="LabwareUserManual_id"><H3>Labware User Manual</H3><P>The labware created by HYPE-IT have multiple tools,
                        in this manual it will be explained how to operate
                        them.
                        All the tools, except the phytotron, are controlled
                        through the LCD display and the rotary encoder.
                        A interface with options will be display in the screen.
                        By turning the rotary encoder it is possible to change
                        the selected option. When the desired option is
                        selected, push the rotary encoder in order to access to
                        that option.</P></DIV><DIV class="blog-post-item" id="Electroporator_id"><H4>Electroporator</H4><P>Place an electroporator cuvette in the
                        electroporator. Make sure that the electrodes of the
                        cuvette contact with the terminal of the
                        electroporator.
                        Choose the option of “Electroporator” in the principal
                        menu in the menu, then the options of the
                        electroporator will be displayed:
                        “Set voltage”: this option is used to set the voltage
                        of electroporation. The typical value is 1500
                        volts.
                        “Set time”: this option is used to set the duration of
                        the electroporation in milliseconds. The typical value
                        of this parameter is 5 milliseconds.
                        “Run”: this option will start the electroporation. When
                        finished it will be displayed the real voltage of
                        electroporation.
                        “Go back”: this option will change the interface to the
                        principal menu.</P></DIV><DIV class="blog-post-item" id="Colorimeter_id"><H4>Colorimeter</H4><P>Place a cuvette in the colorimeter in the right way.
                        Avoid touching the walls of the cuvette in order to
                        have better measures.
                        Choose the option of “Colorimeter” in the principal
                        menu in the menu, then the options of the colorimeter
                        will be displayed:
                        “Blank”: this option is used to do the blank. This is
                        always needed to do before start doing measures. A
                        cuvette with the solvent has to be placed in order to
                        do the blank.
                        “Measure OD600”: this option is used to do the measure
                        of optical density. When the measure is done it will be
                        displayed the value of the optical density.
                        “Go back”: this option will change the interface to the
                        principal menu.</P></DIV><DIV class="blog-post-item" id="Centrifuge_id"><H4>Centrifuge</H4><P>Place the eppendorf tubes in the rotor of the
                        centrifuge. Do not desbalance the rotor of the
                        centrifuge, this is extremely dangerous.
                        Choose the option of “Centrifuge” in the principal menu
                        in the menu, then the options of the centrifuge will be
                        displayed:
                        “Set RPM”: this option is used the set the speed in
                        rpm. The max rpm is 13.000.
                        “Set time”: this option is used set the duration in
                        minutes will be activated the centrifuge.
                        “Status”: this option will shown in the screen the
                        actual speed and remaining time.        “Run”: this option will start the centrifuge.
                        “Go back”: this option will change the interface to the
                        principal menu.</P></DIV><DIV class="blog-post-item" id="Gelelectrophoresisunit_id"><H4>Gel electrophoresis unit</H4><P>Connect the electrophoresis cuvette to the terminals
                        in the labcase, make sure to respect the electrical
                        polarity.
                        Choose the option of “Electrophoresis” in the principal
                        menu in the menu, then the options of the
                        electrophoresis will be displayed:
                        “Set voltage”: this option is used to set the voltage
                        of the electrophoresis in volts. The typical value of
                        this parameter is 100 volts.
                        “Set time”: this option is used to set the duration of
                        the electrophoresis in minutes. The typical value of
                        this parameter is 30 minutes.
                        “Run”: this option will start the electrophoresis. The
                        remaining time of the electrophoresis will be displayed
                        in the screen. When finished the electrophoresis a
                        message will se shown in the screen.
                        “Go back”: this option will change the interface to the
                        principal menu.</P></DIV><DIV class="blog-post-item" id="Stirrerwithtemperaturecontrol_id"><H4>Stirrer with temperature control</H4><P>Place the sample tubes in the stirrer.
                        Choose the option of “Stirrer” in the principal menu in
                        the menu, then the options of the stirrer will be
                        displayed:
                        “Set temperature”: this option is used to set the
                        temperature of the stirrer in degrees. The typical
                        value of this parameter is 37 degrees.
                        “Run”: this option will begin to shake the stirrer and
                        start the temperature control.
                        “Stop”: this option will stop the shaking of stirrer
                        and stop the temperature control.
                        “Status”: this option will show the actual temperature
                        of the stirrer.
                        “Go back”: this option will change the interface to the
                        principal menu.</P></DIV><DIV class="blog-post-item" id="Thermalcycler_id"><H4>Thermal cycler</H4><P>Place the reaction tubes in the thermocycler. Avoid
                        touching the metallic surface, it might be hot.
                        94 62 72
                        Choose the option of “Thermal Cycler” in the principal
                        menu in the menu, then the options of the thermocycler
                        will be displayed:
                        “Set T1”: this option is used to set the temperature in
                        degrees of the first denaturalization. The typical
                        value of this parameter is 94 degrees.
                        “Set time1”: this option is used to set the duration in
                        seconds of the first denaturalization. The typical
                        value of this parameter is 600 seconds.
                        “Set T2”: this option is used to set the temperature in
                        degrees of the denaturalization. The typical value of
                        this parameter is 94 degrees.
                        “Set time2”: this option is used to set the duration in
                        seconds of the denaturalization. The typical value of
                        this parameter is 30 seconds.
                        “Set T3”: this option is used to set the temperature in
                        degrees of the annealing. The typical value of this
                        parameter is 62 degrees.
                        “Set time3”: this option is used to set the duration in
                        seconds of the annealing. The typical value of this
                        parameter is 30 seconds.
                        “Set T4”: this option is used to set the temperature in
                        degrees of the elongation. The typical value of this
                        parameter is 62 degrees.
                        “Set time4”: this option is used to set the duration in
                        seconds of the elongation. The typical value of this
                        parameter depends on the size of the amplified DNA
                        fragment.
                        “Set T5”: this option is used to set the temperature in
                        degrees of the hold. The typical value of this
                        parameter is 4 degrees. This temperature is used to
                        keep the PCR when it have finished.
                        “Run”: this option will begin the PCR.
                        “Status”: this option will show the actual cycle of the
                        PCR and the actual temperature.
                        “Go back”: this option will change the interface to the
                        principal menu.</P></DIV><DIV class="blog-post-item" id="Luminometer_id"><H4>Luminometer</H4><P>Place the three sample tubes in the luminometer and
                        make sure the cover is well placed. The idea about the
                        three sample tubes is to measure at the same time the
                        positive and negative control and the sample.
                        Choose the option of “Luminometer” in the principal
                        menu in the menu, then the options of the luminometer
                        will be displayed:
                        “Measure”: this option is used to do the measure of
                        light intensity When the measure is done it will be
                        displayed the light measured in the three samples.
                        “Go back”: this option will change the interface to the
                        principal menu.</P></DIV><DIV class="blog-post-item" id="Phytotron_id"><H4>Phytotron</H4><P>The phytotron does not have a physical interface so
                        to change the parameters it is necessary to reprogram
                        the Arduino.
                        In the Arduino code of the phytotron is explained how
                        to change parameters as the temperature reference or
                        the duration of the light cycles.</P></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer" style="margin-bottom:-20px"><DIV class="container"><DIV class="row"><DIV class="col-md-5"><DIV class="row"><DIV class="col-md-6 col-sm-6 hidden-xs" style="padding:0;"><DIV class="margin-left-50 clearfix"><H4 class="margin-bottom-10" style="text-align:start;">Follow
                                Us</H4></DIV></DIV></DIV></DIV><DIV class="col-md-7"><H4 class="letter-spacing-1" style="text-align: start;">Sponsors</H4></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>