Team:Valencia UPV/Hardware/Manual
HOME
TEAM
MEET US
COLLABORATIONS
ATTRIBUTIONS
PROJECT
DESCRIPTION
DATA PROCESSING SOFTWARE
SPLIT Cas9
RESULTS
NOTEBOOK
LAB WARE
HARDWARE
Reagents
Manual
HUMAN PRACTICES
OVERVIEW
SOCIAL ENGAGEMENT
SOLVING REAL PROBLEMS
IMPACT ASSESSMENT
SAFETY
PARTS
BEST PART COLLECTION
MODEL
OVERVIEW
Cas9:gRNA Complex Formation
R-Loop Formation
Cleavage and Reading Frame Shift
Conclusions
ACHIEVEMENTS
User manuals
Index
Target                                 search
Identification                                 of the target
gRNA                                 testing system
Plant                                 editing
Modified                                 plant obtention
Labware                                 User Manual
Electroporator
Colorimeter
Centrifuge
Gel                                 electrophoresis unit
Stirrer                                 with temperature control
Thermal                                 cycler
Luminometer
Phytotron
Handbook for HYPE-IT editing
Here we show detailed instructions of how to edit a                         plant using the HYPE-IT system based on Cas9. The                         process is explained, with links to the specific                         protocols.
Target search
Log in the Data Processing Software’s Web                         http://hypeit.cloudno.de/
A) Unknown target
Go to “Target Search (DDBB)” to access the                             search engine.
Search your phenotypic trait of interest to                             know the gene to edit. You can choose it filling up                             three fields: plant name or species, phenotypic                             characteristic and/or the name of the gene.
You will be provided with a list of possible                             options related to your search. To see details of                             your choice, click on the left icon.
Here you have all the information about the                             target gene. Click on “Show more” to access the                             Data Processing Software and find the optimal gRNA                             for your target. Move to step 5.
B) Known target
Go to “Find optimal gRNA” to access the Data                             Processing Software.
Paste the DNA sequence of the known gene in                             FASTA format and click on “Find Targets”
C) In both cases
The software shows a list of possible gRNAs                             with their score. Select the highest one and you                             will be provided with the optimal gRNA.                             Additionally, the software provides the primers to                             amplify the target of your plant with the overhangs                             to use them with the Goldenbraid System with the                             HYPE-IT strategy.
Order to synthesize to any synthesis company of                             your choice the two primer oligonucleotides (DNA                             sequences):
Primers + Goldenbraid overhangs                                     Forward
Primers + Goldenbraid overhangs                                     Reverse
The gRNA is synthetized after step 6 of the                             gRNA testing system protocol.
RNA + Goldenbraid overhangs Forward
gRNA + Goldenbraid overhangs                                     Reverse
Target consensus + Goldenbraid                                     overhangs Forward
Target consensus + Goldenbraid                                     overhangs Reverse
Target knockout + Goldenbraid overhangs                                     Forward
Target knockout + Goldenbraid overhangs                                     Reverse
These overhangs will allow to insert the amplified gene                         fragment of your variety in our gRNA Testing System.                         The gRNA Goldenbraid overhangs are used to insert the                         gRNA in a plasmid based in Goldenbraid for plant                         infection.
Identification of the target
Perform a genomic DNA extraction of the plant.                             The protocol used will depend on the plant and                             variety that you use. As an example, you can check here the protocols we have used for orange                                 and rice extraction.
Run electrophoresis gel (agarose 1%) of the DNA                             extraction following this protocol . You should see a band higher                                 than the molecular weight ladder (above 10kb).
Perform a PCR in the thermal cycler of the                             genomic DNA with the ordered primers with                             Goldenbraid overhangs in order to obtain the                             fragment of the target gene. Depending on the                             primers given by the Software and the polymerase                             you use, the hybridization temperature for the PCR                             will be different. You can use an online calculator                             to know this temperature. It is usually provided by                             the company where the primers were bought. When                             using the HYPE-IT thermal cycler, it should be used                             mineral oil to cover the liquid, in order to avoid                             evaporation.
Run electrophoresis gel (agarose 1%) of the PCR                             products following this protocol . You should see a band lower                                 than 100kb.
Send the PCR product to sequence to a                             sequencing service. The primers you can use to                             sequence are the same used to amplify the gene                             fragment.
Once you know the sequence of the gene you want                             to edit, you have two possible paths to                             follow:
If the sequence is the same that the                                     gRNA chosen in the Data Processing                                     Software, you can order to synthetize the                                     gRNA + Goldenbraid overhangs shown.                                     Additionally, you will need to synthetize                                     the target consensus + Goldenbraid                                     overhangs sequences.
If the sequence is different that                                     the gRNA chosen in the Data Processing                                     Software, you can either choose a different                                     gRNA in the software (repeating everything                                     from step 1) or you can order to synthetize                                     your own sequence adding these                                     overhangs:
ATTG + sequence: gRNA                                             forward
AAAC + sequence (reverse                                             complementary): gRNA reverse
AATG + sequence: target                                             consensus forward
CGAA + sequence (reverse                                             complementary): target consensus                                             reverse.
AATG + sequence removing the                                             16th nucleotide: target knockout                                             forward
CGAA + sequence removing the                                             16th nucleotide (reverse                                             complementary): target knockout                                             reverse.
To obtain the functional gRNA, put in an                             eppendorf the gRNA forward and gRNA reverse                             oligonucleotides ordered in step 6. It is necessary                             to put 18μL of water and 1μL of each                             oligonucleotide. Leave 30 minutes at room                             temperature.
Make a ligation reaction with the gRNA obtained                             in step 6, U6 promoter and psgRNA (both provided in                             the reagents kit). Follow the ligation protocol to ligate in α1 . You will obtain the device                                 U6:gRNA:psgRNA. Now you can directly edit your                                 plant or you can test if you wish the                                 efficiency of the gRNA in our gRNA testing                                 system.
gRNA testing system
Ligation using Goldenbraid assembly system to                             obtain the complete gRNA testing system device. The                             ligation is made following the ligation protocol . The plasmids, linker and                                 promoters are provided in the reagents kit. The                                 ligations to be made are:
In α1 plasmid: 35s or 35s:5’ + target                                     consensus + linker (BBa_K2017003,                                     BBa_K2017004, BBa_K2017005 or BBa_K2017006)                                     + Tnos.
In α1 plasmid: 35s or 35s:5’ + target                                     knockout + linker (BBa_K2017003,                                     BBa_K2017004, BBa_K2017005 or BBa_K2017006)                                     + Tnos.
Transform electrocompetent E. coli with                             the ligation products of step 1 following the transformation protocol .
Culture E. coli in a Petri dish with                             kanamycin resistance, as stated by petri dish culture protocol .
After waiting the time specified in the                             protocol in point 3, pick white colonies in the                             Petri dish and make a liquid culture .
Perform a miniprep to extract the plasmid of E. coli and check if the selected colonies                             have the desired plasmid. We use the E.Z.N.A ® ®.                             Plasmid Mini Kit I, whose instructions are                             explained in the protocols
Digestion of minipreps with EcoRI (specific                             enzyme for α1 digestion). Here you can find the digestion protocol
Run electrophoresis gel (agarose 1%) of the                                 digestion products. You should see two bands,                                 one at 6kb and other between 3 and 4kb. Steps 6                                 and 7 are made to check if the plasmid in the                                 picked colonies is the plasmid ligated in step                                 1.
To obtain the functional target consensus, put                             in an eppendorf the target consensus forward and                             target consensus reverse oligonucleotides ordered                             in step 6 of “Identification of the target”                             section. It is necessary to put 18μL of water and                             1μL of each oligonucleotide. Leave 30 minutes at                             room temperature.
Repeat the same as in step 8 to obtain the                             target knockout, mixing target knockout forward and                             target knockout reverse.
As our experiments showed that the system was                             more efficient in cis (Cas9 and gRNA in the same                             plasmid as the testing system). So, ligate the                             U6:gRNA:psgRNA with the plasmids obtained in step 5                             (testing system target consensus and testing system                             target knockout), in Ω2 plasmid, as in ligation protocol . Each testing system must                                 be ligated separately with the U6:gRNA:psgRNA                                 (only target consensus) and with                                 U6:XT1gRNA:psgRNA.
Transform electrocompetent E. coli with                             the ligation products of step 10 following the transformation protocol .
Repeat steps 3 to 7 for the transformed E.                             coli in step 11.
Ligate the plasmids obtained in step 10 with                             the 35s:Cas9:Tnos + SF (provided in the reagents                             kit), in an α1 plasmid. Repeat steps 11 and                             12.
Transform Agrobacterium C58 with the                             extracted plasmids in the minipreps. Repeat the                             same protocol as E. coli transformation , but using                                 1440V. You should have three plasmids:
Cas9 + gRNA testing system with                                     Target consensus + gRNA
Cas9 + gRNA testing system with                                     Target consensus + XT1gRNA
Cas9 + gRNA testing system with                                     Target knockout + XT1gRNA
Repeat steps 3 to 7 for the transformed Agrobacterium in step 14.
Agroinfiltrate Nicotiana benthamiana with the three different Agrobacterium obtained in step 15. Follow the Agroinfiltration protocol . The experimental                                 design is of your choice. We suggest to make                                 the following experiments (the devices not                                 created by you are provided in the reagents                                 kit). Each point represents one plant to be                                 agroinfiltrated with the stated agrobacterium                                 cultures.
35s:Luciferase:Tnos (positive                                     control)
pNOS:Luciferase:Tnos (negative                                     control)
Cas9:gRNA testing system with Target                                     consensus:gRNA + 35:Renilla:Tnos                                     (sample)
Cas9:gRNA testing system with Target                                     consensus: XT1gRNA + 35:Renilla:Tnos                                     (negative control)
Cas9:gRNA testing system with Target                                     knockout:XT1gRNA + 35:Renilla:Tnos                                     (positive control)
Three days post-infiltration, perform a                             luciferase assay to test the efficiency of the gRNA                             you chose in the database or you designed after                             sequencing. It is recommended to take at least                             three replicates per agroinfiltrated leaf. You can follow the luciferase assay protocol here .
Plant editing
It is going to be used the U6:gRNA:psgRNA                             device obtained in step 8 of “Identification of the                             target” section. However, instead of using the                             overhang ATTG in the forward brand, it will be used                             the overhang AGCC.
Ligate the U6:gRNA:psgRNA with the TMV                             split-Cas9 vector (provided in the reagents kit).                             Follow the ligation protocol .
Transform electrocompetent E. coli with                             the ligation products of step 1 following the transformation protocol .
Culture E. coli in a Petri dish with                             kanamycin resistance, as stated by petri dish culture protocol .
After waiting the time specified in the                             protocol in point 3, pick white colonies in the                             Petri dish and make a liquid culture .
Perform a miniprep to extract the plasmid of E. coli and check if the selected colonies                             have the desired plasmid. We use the E.Z.N.A ® ®.                             Plasmid Mini Kit I, whose instructions are                             explained in the protocols
Digestion of minipreps with EcoRI (specific                             enzyme for α1 digestion). Here you can find the digestion protocol
Run electrophoresis gel (agarose 1%) of the                                 digestion products. You should see two bands,                                 one at 6kb and other between 3 and 4kb. Steps 6                                 and 7 are made to check if the plasmid in the                                 picked colonies is the plasmid ligated in step                                 2.
Transform the plasmid obtained in step 6 in Agrobacterium, following the same steps from                             3 to 7.
Agroinfiltrate Nicotiana benthamiana with split-Cas9 + gRNA in TMV and split-Cas9 in PVX                             (provided in the reagents kit). Follow the agroinfiltration protocol . Take the                                 necessary preventive measures to avoid contact                                 of the agroinfiltrated plants with other                                 plants. The viral systems are safe and do not                                 have the coat protein, but it is still                                 advisable to manage it with caution.
Modified plant obtention
Five days post-agroinfiltration, make                             micropropagation of callus obtained from an                             infected leave of the plant.
Make in vitro culture of the callus in a                             phytotron to grow a plant with its cells edited.                             The grown plant will have the knockout.
Labware User Manual
The labware created by HYPE-IT have multiple tools,                         in this manual it will be explained how to operate                         them.
All the tools, except the phytotron, are controlled                         through the LCD display and the rotary encoder.
A interface with options will be display in the screen.                         By turning the rotary encoder it is possible to change                         the selected option. When the desired option is                         selected, push the rotary encoder in order to access to                         that option.
Electroporator
Place an electroporator cuvette in the                         electroporator. Make sure that the electrodes of the                         cuvette contact with the terminal of the                         electroporator.
Choose the option of “Electroporator” in the principal                         menu in the menu, then the options of the                         electroporator will be displayed:
“Set voltage”: this option is used to set the voltage                         of electroporation. The typical value is 1500                         volts.
“Set time”: this option is used to set the duration of                         the electroporation in milliseconds. The typical value                         of this parameter is 5 milliseconds.
“Run”: this option will start the electroporation. When                         finished it will be displayed the real voltage of                         electroporation.
“Go back”: this option will change the interface to the                         principal menu.
Colorimeter
Place a cuvette in the colorimeter in the right way.                         Avoid touching the walls of the cuvette in order to                         have better measures.
Choose the option of “Colorimeter” in the principal                         menu in the menu, then the options of the colorimeter                         will be displayed:
“Blank”: this option is used to do the blank. This is                         always needed to do before start doing measures. A                         cuvette with the solvent has to be placed in order to                         do the blank.
“Measure OD600”: this option is used to do the measure                         of optical density. When the measure is done it will be                         displayed the value of the optical density.
“Go back”: this option will change the interface to the                         principal menu.
Centrifuge
Place the eppendorf tubes in the rotor of the                         centrifuge. Do not desbalance the rotor of the                         centrifuge, this is extremely dangerous.
Choose the option of “Centrifuge” in the principal menu                         in the menu, then the options of the centrifuge will be                         displayed:
“Set RPM”: this option is used the set the speed in                         rpm. The max rpm is 13.000.
“Set time”: this option is used set the duration in                         minutes will be activated the centrifuge.
“Status”: this option will shown in the screen the                         actual speed and remaining time.
“Run”: this option will start the centrifuge.
“Go back”: this option will change the interface to the                         principal menu.
Gel electrophoresis unit
Connect the electrophoresis cuvette to the terminals                         in the labcase, make sure to respect the electrical                         polarity.
Choose the option of “Electrophoresis” in the principal                         menu in the menu, then the options of the                         electrophoresis will be displayed:
“Set voltage”: this option is used to set the voltage                         of the electrophoresis in volts. The typical value of                         this parameter is 100 volts.
“Set time”: this option is used to set the duration of                         the electrophoresis in minutes. The typical value of                         this parameter is 30 minutes.
“Run”: this option will start the electrophoresis. The                         remaining time of the electrophoresis will be displayed                         in the screen. When finished the electrophoresis a                         message will se shown in the screen.
“Go back”: this option will change the interface to the                         principal menu.
Stirrer with temperature control
Place the sample tubes in the stirrer.
Choose the option of “Stirrer” in the principal menu in                         the menu, then the options of the stirrer will be                         displayed:
“Set temperature”: this option is used to set the                         temperature of the stirrer in degrees. The typical                         value of this parameter is 37 degrees.
“Run”: this option will begin to shake the stirrer and                         start the temperature control.
“Stop”: this option will stop the shaking of stirrer                         and stop the temperature control.
“Status”: this option will show the actual temperature                         of the stirrer.
“Go back”: this option will change the interface to the                         principal menu.
Thermal cycler
Place the reaction tubes in the thermocycler. Avoid                         touching the metallic surface, it might be hot.
94 62 72
Choose the option of “Thermal Cycler” in the principal                         menu in the menu, then the options of the thermocycler                         will be displayed:
“Set T1”: this option is used to set the temperature in                         degrees of the first denaturalization. The typical                         value of this parameter is 94 degrees.
“Set time1”: this option is used to set the duration in                         seconds of the first denaturalization. The typical                         value of this parameter is 600 seconds.
“Set T2”: this option is used to set the temperature in                         degrees of the denaturalization. The typical value of                         this parameter is 94 degrees.
“Set time2”: this option is used to set the duration in                         seconds of the denaturalization. The typical value of                         this parameter is 30 seconds.
“Set T3”: this option is used to set the temperature in                         degrees of the annealing. The typical value of this                         parameter is 62 degrees.
“Set time3”: this option is used to set the duration in                         seconds of the annealing. The typical value of this                         parameter is 30 seconds.
“Set T4”: this option is used to set the temperature in                         degrees of the elongation. The typical value of this                         parameter is 62 degrees.
“Set time4”: this option is used to set the duration in                         seconds of the elongation. The typical value of this                         parameter depends on the size of the amplified DNA                         fragment.
“Set T5”: this option is used to set the temperature in                         degrees of the hold. The typical value of this                         parameter is 4 degrees. This temperature is used to                         keep the PCR when it have finished.
“Run”: this option will begin the PCR.
“Status”: this option will show the actual cycle of the                         PCR and the actual temperature.
“Go back”: this option will change the interface to the                         principal menu.
Luminometer
Place the three sample tubes in the luminometer and                         make sure the cover is well placed. The idea about the                         three sample tubes is to measure at the same time the                         positive and negative control and the sample.
Choose the option of “Luminometer” in the principal                         menu in the menu, then the options of the luminometer                         will be displayed:
“Measure”: this option is used to do the measure of                         light intensity When the measure is done it will be                         displayed the light measured in the three samples.
“Go back”: this option will change the interface to the                         principal menu.
Phytotron
The phytotron does not have a physical interface so                         to change the parameters it is necessary to reprogram                         the Arduino.
In the Arduino code of the phytotron is explained how                         to change parameters as the temperature reference or                         the duration of the light cycles.
Follow                                 Us
Sponsors
