<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_Loop skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/Loop</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header" class="sticky dark header-md noborder clearfix fixed"><HEADER id="topNav"><DIV class="container"><DIV class="navbar-collapse pull-right nav-main-collapse collapse submenu-dark"><NAV class="nav-main"><UL id="topMain" class="nav nav-pills nav-main"><LI><A href="https://2016.igem.org/Team:Valencia_UPV"> HOME </A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Team">MEET US</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Attributions">ATTRIBUTIONS</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="#"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Description">DESCRIPTION</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Software">DATA PROCESSING SOFTWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Proof" style="text-transform: none;">SPLIT Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Notebook">NOTEBOOK</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> LAB WARE </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware">HARDWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents">Reagents</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual">Manual</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Silver">SOCIAL ENGAGEMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Gold">SOLVING REAL PROBLEMS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Integrated_Practices">IMPACT ASSESSMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Safety">SAFETY</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> PARTS </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Part_Collection">BEST PART COLLECTION</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> MODEL</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Model">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/CP">Cas9:gRNA Complex Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Loop">R-Loop Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/ORF">Cleavage and Reading Frame Shift</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Conclusions">Conclusions</A></LI><LI><A class="active" href="https://2016.igem.org/Team:Valencia_UPV/Achievements"> ACHIEVEMENTS</A></LI></NAV></DIV></DIV></HEADER></DIV><SECTION class="page-header page-header-lg parallax parallax-3" style="background-image:url('https://static.igem.org/mediawiki/2016/b/b2/T--Valencia_UPV--modelTitleBack.png');"><DIV class="container"><H1>R-Loop Formation</H1></DIV></SECTION><SECTION><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-2 col-sm-3"><DIV class="side-nav margin-bottom-60 margin-top-30"><DIV class="side-nav-head"><H4>Index</H4></DIV><UL class="list-group list-group-bordered list-group-noicon uppercase"><LI class="list-group-item"><A href="NOTAREALURL#Overviewsec_id">Overview</A></LI><LI class="list-group-item"><A href="NOTAREALURL#Complexdiffusion._id">Complex
                                diffusion</A></LI><LI class="list-group-item"><A href="NOTAREALURL#ProbabilityofR-loopformation._id">Probability of R-loop formation</A></LI><LI class="list-group-item"><A href="NOTAREALURL#Off-targetsearchalgorithm._id">Off-target search algorithm</A></LI><LI class="list-group-item"><A href="NOTAREALURL#Freeenergyincrement%CE%94GSUBcomplex,targetSB1._id">Free
                                energy increment ΔG<SUB>complex,target</SUB></A></LI><LI class="list-group-item"><A href="NOTAREALURL#Parameters._id">Parameters</A></LI><LI class="list-group-item"><A href="NOTAREALURL#Mainremarks_id">Main
                                remarks</A></LI></UL></DIV></DIV><DIV class="col-md-10 col-sm-9"><DIV class="blog-post-item" id="Overviewsec_id"><H3>Overview</H3><P>After the formation of the Cas9:gRNA complex, it must <B>find</B> the <B>target</B> and knock it out. As soon as the complex is formed, it will wander around the nucleus describing a <B>random</B> pathway.

During this erratic trajectory, collisions with several regions of DNA will take place. If the union to those regions is <B>thermodynamically balanced</B> and <B>feasible</B> the R-loop will take place.This structure results on the <B>hybridization</B> of the <B>Cas9:gRNA</B> complex to a <B>DNA</B> sequence. This implies not only joining the target, but also to undesired regions similar to the target, named <B>off-targets.</B></P><P>In this step of the modeling, we used <B>Boltzmann</B> probability distribution and<B>Thermodynamics</B> in order to estimate the probability that the complex binds to the target, using a <B>search algorithm</B> based on the <B>transcriptional activity</B> and <B>target-similarity</B> provided by <B>local alignment</B> algorithms.</P></DIV><DIV class="blog-post-item" id="Complexdiffusion._id"><H3>Complex diffusion</H3><DIV class="blog-post-item" id="Modelandassumptions._id"><H4>Model and assumptions</H4><P>
The process of searching the target among all the genome is named <B>scanning</B>. Thus, we can express the contact rate between Cas9:gRNA complex and any other DNA region:</P><P>
                        Where parameters with values indicated in Table 5,
                        are:
                        <UL><LI>D is the compound diffusivity.</LI><LI>[Cas9:gRNA] is the concentration of the complex.</LI><LI>V is the compartment volume, i.e. the plant nuclear
                        volume.</LI><LI>λ<SUB>Cas9</SUB> is the characteristic length between
                        the place of production and binding.</LI></UL></P><P>
The picture below represents a spherical approach of the Cas9 shape, being λ=V<SUB>Cas9</SUB><SUP>1/3</SUP>. We can consider that next to the edge of Cas9 there will be DNA ready to hybridize gRNA if possible (2). Assumptions made to consider this situation are:<UL><LI>The complex can be considered as a macromolecule with
                        three-dimensional random diffusion around the
                        nucleus.</LI><LI>The net molar flow is presumably equal to zero, as the
                        compartment composition is well-mixed.</LI><LI>As DNA is dispersed among the nucleus, the complex will
                        be almost in permanent contact with it.</LI></UL></P></DIV><DIV class="blog-post-item" id="Simdiffusion._id"><H4>Simulations</H4><P>
 Varying the time of measurement and the number of Cas9
                        and gRNA copies introduced, we can expect different
                        results for this ratio:</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 10.
                            Comparison between values obtained for the
                            diffusion ratio of the Cas9:gRNA complex al time t
                            = 4320 minutes (3 days). All curves are relative to
                            the results obtained with 1 copy for Cas9.</P></DIV><P>
                        Simulations represented in the graphic above let us
                        check that the gain in the rate of contact between the
                        Cas9:gRNA complex, increases approximately in the same
                        way when so does the concentration of the complex. A
                        <B>minimum</B> of <B>10-15 gene copy number</B> for the gRNA
                        construction is necessary to achieve the plateau of the
                        k<SUB>r</SUB> ratio, independently of the gene copy
                        number for the Cas9 construction. Furthermore, the gain
                        in the number of gene copies encoding Cas9, is the same
                        produced in the random contact ratio.
                        As it can be inferred from this analysis, achieving
                        enough Cas9:gRNA concentration is critical to stablish
                        contact between the complex and the target. One
                        possibility to <B>increase repeatability</B> of the test,
                        <B>minimizing the randomness</B> of the complex diffusion, is
                        to <B>infiltrate the Testing System construction with the
                        gRNA sequence</B>. The expected result was that if the gRNA
                        is transcribed near the target, the aleatory of the
                        three-dimensional diffusion would be minimized. Joining
                        both pieces near to each other, it will be “easier” for
                        the complex to find the target. This suggestion was
                        implemented in <B>wet-lab experiments</B>, showing an <B>increase
                        of the light signal</B>.</P></DIV><DIV class="blog-post-item" id="ProbabilityofR-loopformation._id"><H3>Probability of R-loop formation</H3><P>
                        In order to knock out our genetic target, it must be
                        hybridized by the gRNA forming the R-loop. This
                        structure provides Cas9 with the necessary stability to
                        cut the DNA strand (2, 15). In order to get the
                        structure, it is necessary that <B>potential targets are
                        complementary enough to the gRNA</B>. Providing gRNA-DNA
                        complementarity means accomplishing the <B>thermodynamic
                        requirements</B> to let the knockout happen.</P><P>
                        Estimating the number of targets and off-targets, we
                        can obtain a distribution of the cleavage probability
                        in function of energy needed to cleave each DNA
                        location. Thus, M different energetic states are
                        considered as places where the R-loop could take place,
                        being our Testing System one of that states. This
                        scenario can fit to a <B>Boltzmann distribution</B> (2,5),
                        being the binding probability of the Cas9:gRNA complex
                        with the m-DNA region:</P><P>
                        This expression has information about the thermodynamic
                        balance after the R-loop formation. In order to obtain
                        it, we had to obtain previously the <B>free energy
                        increment for each DNA candidate</B>
                        (-ΔG<SUB>complex,target</SUB>), and the expected number
                        of those regions (N<SUB>target</SUB>).
                        Off-target regions were estimated using the off-target
                        search algorithm, getting <B>1 off-target for Ga20Ox and 5
                        for TFL</B>. The next section has the explanation of the
                        free energy increment, which ensures the thermodynamic
                        stability of the R-loop.</P></DIV><DIV class="blog-post-item" id="Off-targetsearchalgorithm._id"><H4>Off-target search algorithm.</H4><P>Off-targets are DNA regions where the R-loop could
                        take place because of the <B>high similarity</B> between that
                        region and the target. This means that off targets
                        steal Cas9 and gRNA supposed to knock out on-targets.
                        Most reliable off-target predictions are obtained by
                        experimental results, but in our model we must be able
                        to find off-targets quickly for all possible targets
                        (1,2) so we could not wait for experimental
                        results.
                        Alternatively, we have developed an off-target search
                        <B>based in transcriptional activities and local
                        alignments</B> between target and off-target candidates.
                        Our proposed strategy was the following one:</P><P>
                        The first step was to create a <B>gene library</B> with
                        sequences of the <B>most transcribed genes in <I>Nicotiana
                        benthamiana</I></B>. There is a clear relation between
                        transcriptional activity and relaxed state of chromatin
                        (13), letting us assume that those genes highly
                        transcribed will be more accessible to the gRNA.
                        This algorithm is implemented in the Matlab function
                        Nboffsearch.m. The search of potential off-targets for
                        each of our two targets, gave a result of 1 off-target
                        for Ga20Ox and 5 off-targets for the TFL.</P></DIV><DIV class="blog-post-item" id="FreeenergyincrementΔGSUBcomplex,targetSB1._id"><H4>Free energy increment ΔG<SUB>complex,target</SUB>.</H4><P>As there is <B>no energy supply catalyzing the R-loop</B>
                        (2), the process described above is a sequence of
                        reactions which in global, must accomplish the
                        thermodynamic law for this kind of processes:</P><P>
                        
                        Where the free energy is decomposed in the main stages
                        previously described (2,15):</P><P>
                        
                        Each one of the terms from the expression above, is
                        explained in following sections of the Modeling. We
                        determined these parameters for <B>two of the targets
                        contained in our Database</B>: ORYZA SATIVA JAPONICA GROUP
                        GIBBERELLIN 20 OXIDASE 2 (LOC4325003) and CITRUS
                        SINENSIS TERMINAL FLOWER (TFL). Those induce higher
                        grain yield in rice flowering in orange, respectively.
                        Values obtained for each of them were
                        ΔG<SUB>complex,Ga20Ox</SUB>= -10,37 kcal/mol and
                        ΔG<SUB>complex,TFL</SUB> =-11,78 kcal/mol.
                        These values were used to obtain the probability that
                        the Cas9:gRNA complex cleaves the target in the Testing
                        System, performing the desired knockout. To calculate
                        this probability, we had to account for <B>possible
                        off-targets</B>, as they could obtain
                        ΔG<SUB>complex,target</SUB>&lt;0, letting the R-loop
                        be formed at those regions as well.
                        After calculating the ΔG<SUB>complex,offtarget</SUB>
                        for off-targets suggested by the algorithm, only the
                        off-target for the Ga20Ox could be considered as a
                        final off-target, as TFL off-targets got free energy
                        increments higher than zero. Therefore, we calculated
                        the <B>P<SUB>complex,Ga20Ox</SUB></B>, obtaining a value of
                        0,9964.</P></DIV><DIV class="blog-post-item" id="PAMbindingenergy.._id"><H5>PAM binding energy</H5><P>Once the complex has been formed, it must find the
                        PAM sequence of the target included in the Testing
                        System. The PAM region has between <B>3 and 5 nucleotides</B>
                        which are recognized by Cas9, enabling the union of the
                        complex to the DNA. Each Cas9 specie binds better to
                        one type of PAM region. In our case, as we are working
                        with the human type Cas9, the PAM sequence will be
                        -NGGN-.</P><DIV style="text-align:left;"><P>
Bearing in mind that breaking <B>one hydrogen bond</B>
                        provides at least an energy supply of <B>1.2 kcal/mol</B>, the
                        binding of the PAM sequence must give a free energy
                        ΔG<SUB>PAM</SUB> at least of 9 kcal/mol.
                        Relying on the nucleotides arrangement,
                        ΔG<SUB>PAM</SUB> resulting from the PAM recognition
                        will vary. However, <B>it is possible that Cas9 interacts
                        with a PAM region even though there are mismatches</B>
                        between them (2). In order to know if this affinity for
                        regions with mismatches was significant, we studied the
                        ΔG<SUB>PAM</SUB> obtained for all possible PAM
                        combinations.</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 11.
                            Energies obtained by combining all possible RNA and DNA pairs with different PAM sequences. Units are kcal/mol.</P></DIV><P>White spaces in picture above represent PAM
                        alternatives which do not bind significantly to a -NGG-
                        PAM. The lower binding energy is clearly for PAMs with
                        the structure <B>-NGG-, achieving less than -9kcal/mol</B>.
                        However, there are other alternatives with affinity
                        enough to let the Cas9 bind to them. The potential of
                        these regions as possible off-targets relies also on
                        the Cas9 specie being used, as there are some more
                        “promiscuous” than others. We opted for <B>including this
                        potential off-target PAMs in our off-target search
                        algorithm</B>.</P><P>
                        This PAM-energy assignment is implemented in the Matlab
                        function energy_PAM.mat. The input is the target
                        sequence. Comparing the PAM extreme nucleotides of the
                        input sequence with a table containing Cas9-PAM binding
                        energies, it matches the information of the string
                        input with the corresponding energyΔG<SUB>PAM</SUB> .
                        In the particular case of targets implemented in our
                        testing system, the value of this parameter is in Table 2.</P><TABLE class="table table-bordered table-striped" align="right" text-align="center"><TBODY><TR><TH>Target</TH><TH>PAM</TH><TH>ΔG<SUB>PAM</SUB></TH></TR><TR><TD>Rice - Oryza sativa, Semi-dwarf; higher
                                    grain yield.</TD><TD>CGG</TD><TD>-9,600</TD></TR><TR><TD>Orange - Citrus sinensis, Induced
                                    flowering.</TD><TD>TGG</TD><TD>-9,700</TD></TR></TBODY></TABLE><P class="imgFooterP" style="text-align: center; font-style: italic;">
                            Table 2: Results of ΔG<SUB>PAM</SUB> for targets
                        implemented in Testing System</P><DIV class="blog-post-item" id="Cas9:gRNA:DNAhybridization._id"><H5>Cas9:gRNA:DNA hybridization</H5><P>Secondly, the release of the <B>energy from PAM binding
                        will be used to hybridize </B>the <B>gRNA</B> and nucleotides from
                        the <B>DNA</B> sequence. In order to know the energy
                        associated to different base pairs, we built a table
                        with the energy increment for all possible matching
                        duplexes. This table provides all necessary information to
                        calculate the ΔΔG<SUB>exchange,gRNA:target</SUB>
                        (8,9,10,11), as <B>there should not be mismatches between
                        both of them</B>. In the graphic below it can be
                        appreciated that there are two sources of variability
                        affecting the energetic balance of duplex
                        hybridization. On one hand, there is a clear dependence
                        of the nucleotide, and on the other hand, the type of
                        nucleic acid (RNA or DNA) also affects the free energy
                        increment.</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 12.
                           Comparison between the energy exchange resulting from hybridizations with all possible nucleotides of RNA or DNA</P></DIV><P>
                        Thus, we had to choose an approach which considered the
                        energy <B>differences between different nucleotides</B> and
                        <B>different nucleic acids</B>. Moreover, the model used to
                        estimate the energetic cost of forming the R-loop, had
                        to consider the <B>distance of base pairs to the PAM</B>
                        region.
                        This kind of <B>forward move</B> which considers all the
                        mentioned criteria, is known as <B>nearest neighbor model</B>.
                        The meaning of this model is that RNA:DNA union will
                        rely on the context. The energy used to bind a pair of
                        duplexes, uses the energy released by the previous
                        duplex union. In other words, it is assumed that the
                        energy from the k<SUP>th</SUP> hydrogen bond will be
                        used in the reaction of the nearest base pair, k +
                        1.
                        Thus, the hybridization of several nucleotides can be
                        represented as a sequence of binding reactions, leading
                        to a global difference between the hydrolysis of
                        DNA:DNA bounds, and the union of gRNA:DNA strands. The
                        term ΔΔG<SUB>exchange gRNA:target</SUB> reflects the
                        difference between the free energy used for the pair
                        DNA-DNA hydrolysis and the pair RNA-DNA hybridized.</P><DIV style="text-align:left;"><P>
                        However, the gRNA may have thermodynamically stable
                        unions with other regions which are not the target.
                        Those DNA regions, named off-targets, may have only few
                        mismatches that slightly affect its affinity towards
                        the gRNA. The half part of the gRNA close to the PAM,
                        is the most determining to form the R-loop. Therefore,
                        <B>if mismatches are</B> placed in the extreme <B>opposite</B> to the
                        PAM, they may <B>will not compromise</B> the off-target
                        knockout.</P></DIV><P>
                        In a similar way as we did with matching duplexes, our
                        first try was to find more information about energy
                        accounted when mismatches are produced. Nevertheless,
                        there is poor consensus among bibliography, not all
                        possible duplexes have been studied and we neither
                        could determine these energetic values empirically.
                        In order to solve this, we created a <B>penalty vector</B>
                        which adds a penalty to the match binding energy for
                        each mismatch: </P><P>The term d<SUB>k</SUB> in the equation above, refers to a
                        weight that decreases as k is increased, with k =
                        1,2,3…length gRNA (typically 20-23). We have estimated
                        values of those weights using criteria from our Scoring
                        System, as it had been validated comparing to other
                        target searchers available online. Those coefficients, collected in a vector which is represented on the right,
                        are multiplied by the <B>single mismatch average penalty</B>
                        of 0.78 kcal/mol, extracted from bibliography (2). The
                        implementation of the penalties is in the Matlab
                        function weights_exchange.m.</P><P>Using this strategy, we studied how would could vary
                        the ΔΔG<SUB>exchange gRNA:target</SUB> estimated for a
                        target with different number and positions of
                        mismatches. We implemented the obtention of
                        ΔΔG<SUB>exchange gRNA:target</SUB> in the Matlab
                        function energy_exchange.m. Results obtained for our
                        particular targets were:</P><TABLE class="table table-bordered table-striped" align="right"><TBODY><TR><TH>Target</TH><TH>ΔΔG<SUB>exchange gRNA:target</SUB></TH></TR><TR><TD>Rice - Oryza sativa, Semi-dwarf; higher
                                    grain yield.</TD><TD>-0.7710</TD></TR><TR><TD>Orange - Citrus sinensis, Induced
                                    flowering.</TD><TD>-2.0785</TD></TR></TBODY></TABLE><P class="imgFooterP" style="text-align: center; font-style: italic;">
                             Table 3: Results of ΔΔG<SUB>exchange gRNA:target</SUB>
                        for targets implemented in Testing System.</P><P>In order to know more about how do mismatches affect
                        the thermodynamic balance of the gRNA and DNA target
                        hybridization, we calculated values of ΔΔG<SUB>exchange
                        gRNA:target</SUB> for rice and orange, <B>varying the
                        position of a single mismatch</B>. Conditions of the
                        simulations are in Table 4.</P><TABLE class="table table-bordered table-striped"><TBODY><TR><TH>Original nucleotide</TH><TH>New nucleotide</TH><TH>position</TH><TH>ΔΔG<SUB>exchange gRNA:target</SUB></TH></TR><TR><TD>Rice - Oryza sativa, Semi-dwarf; higher
                                    grain yield.</TD></TR><TR><TD>G</TD><TD>A</TD><TD>23</TD><TD>-0,563</TD></TR><TR><TD>G</TD><TD>T</TD><TD>23</TD><TD>-0,603</TD></TR><TR><TD>G</TD><TD>C</TD><TD>23</TD><TD>-0,703</TD></TR><TR><TD>G</TD><TD>A</TD><TD>10</TD><TD>0,369</TD></TR><TR><TD>G</TD><TD>T</TD><TD>10</TD><TD>0,169</TD></TR><TR><TD>G</TD><TD>C</TD><TD>10</TD><TD>-0,381</TD></TR><TR><TD>C</TD><TD>A</TD><TD>7</TD><TD>4,989</TD></TR><TR><TD>C</TD><TD>G</TD><TD>7</TD><TD>2,389</TD></TR><TR><TD>C</TD><TD>T</TD><TD>7</TD><TD>4,589</TD></TR><TR><TD>Orange - Citrus sinensis, Induced
                                    flowering</TD></TR><TR><TD>G</TD><TD>A</TD><TD>23</TD><TD>-1,8805</TD></TR><TR><TD>G</TD><TD>T</TD><TD>23</TD><TD>-1,8905</TD></TR><TR><TD>G</TD><TD>C</TD><TD>23</TD><TD>-1,9705</TD></TR><TR><TD>T</TD><TD>A</TD><TD>12</TD><TD>-0,8885</TD></TR><TR><TD>T</TD><TD>G</TD><TD>12</TD><TD>-0,3885</TD></TR><TR><TD>T</TD><TD>C</TD><TD>12</TD><TD>-1,3385</TD></TR><TR><TD>C</TD><TD>A</TD><TD>4</TD><TD>2,6815</TD></TR><TR><TD>C</TD><TD>G</TD><TD>4</TD><TD>-0,3185</TD></TR><TR><TD>C</TD><TD>T</TD><TD>4</TD><TD>1,6815</TD></TR></TBODY></TABLE><P class="imgFooterP" style="text-align: center; font-style: italic;">
                              Table 4: Results of ΔΔG<SUB>exchange gRNA:target</SUB>
                        for targets implemented in Testing System.</P><P>
                        Results illustrated in the graphic below, show that
                        as expected, the R-loop formation is less likely as it
                        is reduced the distance between a mismatch and the PAM.
                        In general, it seems that our thermodynamics approach
                        emulates well the mechanism of a R-loop formation. With
                        both targets, the average result of changing one
                        nucleotide, is <B>decreased with higher PAM-distance</B>.
                        Mismatches placed <B>downstream the 20 <SUP>th</SUP>
                        nucleotide</B>, typically result in <B>positive free energy</B>
                        increments, avoiding the RNA:DNA hybridization. This
                        agrees with criteria found in bibliography (2), letting
                        us assume that the penalty system worked well as
                        representation of mismatch effects.</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 13.
Estimation of total energy exchange produced between the gRNA and the DNA target, using examples of TFL and Ga20ox.</P></DIV><P> The variability observed in each position is due to the
                        differences between three possible nucleotides.
                        However, there are some atypical results as well which
                        may be caused by unknown sources of variability. For
                        instance, the ΔΔG<SUB>exchange gRNA:target</SUB> is
                        overlapped for positions 23 and 10 in rice, while
                        energy difference for the position 23 was supposed to
                        be minor. This could be due to the necessity of
                        <B>training and improving</B> our function, using parameters
                        in the penalty function which are based on <B>empirical
                        evidence</B>.</P></DIV><DIV class="blog-post-item" id="DNAsupercoiling._id"><H5>DNA supercoiling</H5><P>Finally, the chromatin state is critical to let the
                        gRNA hybridize the DNA, and <B>some energy can be
                        extra-needed if the DNA is &quot;relaxed&quot;, i.e. positively
                        supercoiled</B>. Consequently, regions highly similar may
                        will not be able to join the gRNA because the chromatin
                        could be compressed. Having off-targets means that the
                        binding will be taking place in some unspecified
                        regions. The difference between the initial density of
                        a target σ<SUB>I</SUB> and a non-specific region
                        (σ<SUB>NS</SUB>) will affect to the difference in the
                        free energy needed to untwist the on-target DNA
                        region:</P><P>As we could not determine the chromatin state and its
                        evolution during the time that CRISPR/Cas9 was working
                        on the plant, we could not study this parameter. Nevertheless, information about the chromatin
                        supercoiling has been <B>indirectly introduced</B> in our
                        model. 
Since <B>high transcription activities</B> can be synonym of
                       <B> relaxed chromatin</B> (12, 13), we can assume that
                        <B>supercoiling will not be affecting to our Testing
                        System</B>, as it has the 35S promotor (a
                        constitutive promotor with high activity (4)). Moreover,
                        the difference in DNA supercoiling between the target
                        and off-targets, can be considered nearly zero because
                        potential off-targets will be those genes of
                        <I>Nicotiana benthamiana</I> which are highly
                        transcribed.</P></DIV><DIV class="blog-post-item" id="Parameters._id"><H3>Parameters</H3><DIV class="table-responsive" style="width:55%;overflow:inherit"><TABLE class="table table-bordered table-striped"><TBODY><TR><TH>Parameter</TH><TH>Value</TH><TH>Source</TH></TR><TR><TD>D</TD><TD>2700 µm<SUP>2</SUP>/min</TD><TD>Reference (6)</TD></TR><TR><TD>λ</TD><TD>0,015 µm</TD><TD>Reference (7)</TD></TR><TR><TD>V</TD><TD>14140 µm<SUP>3</SUP></TD><TD>Waterloo iGEM team 2015</TD></TR><TR><TD>[Cas9:gRNA](t)</TD><TD>t = 3 days</TD><TD>Model estimated.</TD></TR><TR><TD>ΔΔG <SUB>single mismatch
                                    penalty</SUB></TD><TD>0.078 kcal/mol</TD><TD>Reference (2)</TD></TR><TR><TD>k<SUB>r</SUB></TD><TD>0.0172⋅[Cas9:gRNA]</TD><TD>Model estimated</TD></TR><TR><TD>k<SUB>c</SUB></TD><TD>0,48 min<SUP>-1</SUP></TD><TD>Reference (2)</TD></TR><TR><TD>k<SUB>unbind</SUB></TD><TD>300 min<SUP>-1</SUP></TD><TD>Reference (2)</TD></TR><TR><TD>k<SUB>B</SUB></TD><TD>0.0019872041 kcal/(mol⋅K)</TD><TD>Reference (5)</TD></TR><TR><TD>N<SUB>(OFF-TARGETS)</SUB></TD><TD>1 for Ga20Ox</TD><TD>Model estimated</TD></TR><TR><TD>T</TD><TD>297 K</TD><TD>Experiment conditions</TD></TR><TR><TD>P<SUB>(complex,Ga20Ox)</SUB></TD><TD>0,9964</TD><TD>Model estimated</TD></TR></TBODY></TABLE><P class="imgFooterP" style="text-align: center; font-style: italic;">
                             Table 5: Parameters used calculation of the
                        k<SUB>cleavage TS</SUB> parameter.</P></DIV></DIV><DIV class="blog-post-item" id="Mainremarks_id"><H3>Main remarks</H3><P>In order to reduce the random influence of
                        three-dimensional diffusion of the complex, we
                        suggested introducing the gRNA and the Testing System
                        constructions one next to the other. Single mismatches
                        positioned downstream the 11<SUP>th</SUP>-10
                        <SUP>th</SUP> nucleotide of the gRNA, lead to a
                        positive free energy increment, i.e. they make unable
                        the R-loop between that gRNA and the DNA target.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer" style="margin-bottom:-20px"><DIV class="container"><DIV class="row"><DIV class="col-md-5"><DIV class="row"><DIV class="col-md-6 col-sm-6 hidden-xs" style="padding:0;"><DIV class="margin-left-50 clearfix"><H4 class="margin-bottom-10" style="text-align:start;">Follow
                                Us</H4></DIV></DIV></DIV></DIV><DIV class="col-md-7"><H4 class="letter-spacing-1" style="text-align: start;">Sponsors</H4></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>