<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header" class="sticky dark header-md noborder clearfix fixed"><HEADER id="topNav"><DIV class="container"><DIV class="navbar-collapse pull-right nav-main-collapse collapse submenu-dark"><NAV class="nav-main"><UL id="topMain" class="nav nav-pills nav-main"><LI><A href="https://2016.igem.org/Team:Valencia_UPV"> HOME </A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Team">MEET US</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Attributions">ATTRIBUTIONS</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="#"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Description">DESCRIPTION</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Software">DATA PROCESSING SOFTWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Proof" style="text-transform: none;">SPLIT Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Notebook">NOTEBOOK</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> LAB WARE </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware">HARDWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents">Reagents</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual">Manual</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Silver">SOCIAL ENGAGEMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Gold">SOLVING REAL PROBLEMS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Integrated_Practices">IMPACT ASSESSMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Safety">SAFETY</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> PARTS </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Part_Collection">BEST PART COLLECTION</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> MODEL</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Model">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/CP">Cas9:gRNA Complex Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Loop">R-Loop Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/ORF">Cleavage and Reading Frame Shift</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Conclusions">Conclusions</A></LI><LI><A class="active" href="https://2016.igem.org/Team:Valencia_UPV/Achievements"> ACHIEVEMENTS</A></LI></NAV></DIV></DIV></HEADER></DIV><SECTION class="page-header page-header-lg parallax parallax-3" style="background-image:url('https://static.igem.org/mediawiki/2016/f/fb/T--Valencia_UPV--resultsTitleBack.png');"><DIV class="container"><H1>Results</H1></DIV></SECTION><SECTION><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-2 col-sm-3"><DIV class="side-nav margin-bottom-60 margin-top-30"><DIV class="side-nav-head"><H4>Index</H4></DIV><UL class="list-group list-group-bordered list-group-noicon uppercase"><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Results#Split-Cas9_id">Split-Cas9</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/Results#gRNATestingSystem_id">gRNA Testing System</A></LI></UL></DIV></DIV><DIV class="col-md-10 col-sm-9"><DIV class="blog-post-item" id="Split-Cas9_id"><H3>Split-Cas9</H3><P>We were able to divide Cas9 endonuclease, insert it inside viral vectors and confirmed its activity in plants.In order to test the activity of Split-intein-Cas9 strategy delivered through autoreplicative viral vectors we followed a classical approach. Therefore, we chose <I>Nicotiana benthamiana</I> xylosyltransferase XT1 gene (Niben101Scf04205Ctg025) as targeted gene. Specifically, we selected a target sequence the mutation of which produces the disappearance of a EcoRI restriction site (Fig. 1).</P><P>((Fig. 1) Schematic representation of the structure of Niben101Scf04205Ctg025(XT1) with the sewuence of the target site.)Therefore, we introduce the two halves of Split-Cas9 inside two different viral vectors (TMV and PVX) into three different plants and introduce the corresponding gRNA at different times. In one no gRNA was introduced, which was treated as negative control. In the second one gRNA was introduced 3 days after Split-Cas9 system introduction or post infiltration (d.p.i.). In the third plant gRNA was introduced 6 d.p.i. Additionally, we introduced Cas9 endonuclease and gRNA through Ti plasmid into another plant, in order to use it as gene edition positive control.Time gaps between Split-Cas9 and gRNA introduction are due to difference of expression rates among time. Viral system gene expression peaks takes place later than genes introduced through A.tumefasciens Ti plasmid. Next, we extract genomic DNA from these four plants and carried out a PCR in order to amplify the mutated region. Afterwards, PCR reactions are purified and digested with EcoRI enzyme in order to determine if mutation has been occurred (Fig. 2).</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 2) Electrophoresis gel of XT1 amplicons digestion. Mutation efficiencies of Split-intein-Cas9 system and classical Cas9 system are compared by resistant bands signal intensity.</P></DIV><P>Once the presence of mutations was confirmed, resistant band was purified, and cloned into a vector. Finally, mutation sequences were analyzed (Fig. 2)</P><P>((Fig. 3) Alignment of XT1 sequences obtained from different clones of uncleaved bands XT1target site appears in green. Red letters and dashes indicate insertions and deletions respectively.)As can be seen in Figure 3, Split-intein-Cas9 system mutation activity delivered through viral vectors was confirmed. Furthermore, as shown in Figure 2, the mutation efficiency of Split-intein-Cas9 seems to be higher when the gRNA is introduced 6 d.p.i. Although its mutation efficiency seems lower than classical Cas9 delivery through Ti plasmid efficiency.</P></DIV><DIV class="blog-post-item" id="gRNATestingSystem_id"><H3>gRNA Testing System</H3><P>We have tested and proved the feasibility of different gRNA Testing System versions through different luciferase assays. In all these experiments different versions of the same target along different gRNAs has been used:</P><UL><LI>Consensus target (Target cons): Synthetic target extracted from databases and built from overlapping synthetic oligonucleotides.</LI><LI>Amplified target (Target PCR): DNA region amplified from desired crop that includes the selected target site.</LI><LI>Already-mutated target (Target KO): Synthetic target that simulates CRISPR/Cas9 mutation thanks to the removal of one base pair.</LI></UL><P>In all these experiments two kind of gRNA were used:</P><UL><LI>Target specific gRNA: gRNA designed to lead Cas9 endonuclease to the chosen target.</LI><LI>Non-targeting gRNA (NTgRNA): null gRNA used in order to lead Cas9 to another target site not studied in these assays. It is used as negative control.</LI></UL><P>As shown in Figure 4 gRNA Testing System 1.0 (with SAGTI linker) is able to turning ON from an initial OFF state since luminescence signal of tested samples is significantly higher than signal from negative control. However as Figure 5 indicates, the sensitivity of this version is low since positive control is one hundred times lower than internal positive control 35s:Luciferase:Tnos. </P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 4) Comparison of the different mutation efficiency ratios. Mutation ratios from Ga20PCR-Ga20gRNA and Ga20cons-Ga20gRNA are statistically higher than negative control Ga20PCR-NTgRNA, although lower than positive control Ga20KO-Ga20gRNA.</P></DIV><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 5) Comparison of the different mutation ratios. Luminescence ratios from luciferase constitutive expression regulated by pNOS and 35s promoters are added to the data shown in Figure 4</P></DIV><P>Regarding this results new gRNA Testing Systems versions were designed and tested.Once all the new versions with the new linkers were built, we performed an assay to determine which of them is the most suitable one. From Fig. 6 we can infer that the best version is the one with AEK linker due to its higher luciferase ratio.</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 6) Comparison of ratios between the new gRNA Testing versions. All of them present a higher signal than constitutively expressed luciferase under 35s and pNOS regulation.</P></DIV><P>After choosing the most appropriate version, we carried out an assay to test its capacity of turning ON. As figures 7 and 8 shown, luciferase ratios of Ga20PCR-Ga20gRNA and Ga20cons-Ga20gRNA are significantly higher than negative control. Thus, meaning the new version system works properly.</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 7) Comparison of the different mutation ratios of AEK linker version. Luminescence ratios from luciferase constitutive expression regulated by pNOS and 35s promoters are also shown.)</P></DIV><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig.8) Comparison of the different mutation efficiency ratios. Mutation ratios from Ga20PCR-Ga20gRNA and Ga20cons-Ga20gRNA are statistically higher than negative control Ga20PCR-NTgRNA.</P></DIV><P>Afterwards we also tested the feasibility and suitability of RSIAT linker version. As can be observed in figures 9 and 10, amplified targets from the desired crop seemed to be not working well in this Testing System version. Although, there is significant difference between the synthetic target circuit and its negative control.</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 9) Comparison of the different mutation ratios of RSIAT linker version. Luminescence ratios from luciferase constitutive expression regulated by pNOS and 35s promoters are also shown. As can be seen luciferase ratios from Ga20KO-NTgRNA are higher than constitutively expressed luciferase.</P></DIV><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 10) Comparison of the different mutation ratios of RSIAT linker version. As can be seen there is no statistical difference between Ga20PCR-Ga20gRNA and its negative  control Ga20PCR-NTgRNA. However Ga20cons-Ga20gRNA ratio is significantly higher than its negative control Ga20cons-NTgRNA.</P></DIV><P>In the pursuit of improving the sensitivity of our system we aim to enhance the CRISPR/Cas9 system efficiency. For our final assay we compared the efficiency of CRISPR/Cas9 system when introducing the target, gRNA and Cas9 endonuclease in the same plasmid inside the cell and the efficiency of the system when target and gRNA and Cas9 are delivered separately. As shown in figures 11 and 12 when all the elements are delivered inside the same plasmid mutation efficiency rises, probably due to the fact that there is more gRNA next to the target site. Thus facilitating the formation of complex Cas9-gRNA near the desired target.</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 11) Comparison of the different mutation ratios. Luminescence ratios from luciferase constitutive expression regulated by pNOS and 35s promoters are also shown.</P></DIV><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">(Fig. 12) Comparison of the different mutation ratios between cis (in the same plasmid) and trans (in different plasmids) construction.</P></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer" style="margin-bottom:-20px"><DIV class="container"><DIV class="row"><DIV class="col-md-5"><DIV class="row"><DIV class="col-md-6 col-sm-6 hidden-xs" style="padding:0;"><DIV class="margin-left-50 clearfix"><H4 class="margin-bottom-10" style="text-align:start;">Follow
                                Us</H4></DIV></DIV></DIV></DIV><DIV class="col-md-7"><H4 class="letter-spacing-1" style="text-align: start;">Sponsors</H4></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>