Team:Valencia UPV/Hardware/Reagents
HOME
TEAM
MEET US
COLLABORATIONS
ATTRIBUTIONS
PROJECT
DESCRIPTION
DATA PROCESSING SOFTWARE
SPLIT Cas9
RESULTS
NOTEBOOK
LAB WARE
HARDWARE
Reagents
Manual
HUMAN PRACTICES
OVERVIEW
SOCIAL ENGAGEMENT
SOLVING REAL PROBLEMS
IMPACT ASSESSMENT
SAFETY
PARTS
BEST PART COLLECTION
MODEL
OVERVIEW
Cas9:gRNA Complex Formation
R-Loop Formation
Cleavage and Reading Frame Shift
Conclusions
ACHIEVEMENTS
Reagents
Index
Split-Cas9
gRNA                                 Testing System
Why                                 wouldn’t our gRNA work?
gRNA                                 testing methods
Our                                 device
Parts of                                 the device (gRNA Testing System 1.0)
New                                 parts of the device (gRNA Testing System                                 2.0)
References:
Split-Cas9
We provide all the necessary constructions in order                         to use autoreplicative vectors as Cas9 delivery system.                         That means, providing both Cas9 halves fused with its                         corresponding intein inserted into two different                         vectors Tobacco Mosaic Virus (TMV) and Potato Virus X                         (PVX).
gRNA Testing System
Standardization and modularity are two of the                         mainstays of Synthetic Biology. Bearing that in mind we                         have design, built and tested a Phytobricks                         standardized genetic circuit that allows to analyze the                         efficiency of a given guide RNA (gRNA) in vivo.
We have designed a modular gRNA testing system that                         allow us to determine the efficiency of any provided                         gRNA in vivo. This system works as a genetic switch                         that remains OFF until Cas9 endonuclease, guided by a                         gRNA, performs a double stranded break inside the                         target sequence. The device performance is based on the                         presence of a luciferase reporter gene out of frame.                         When a double strand break (DSB) occur due to Cas9                         activity, a frameshift mutation may appear. Thus,                         luciferase’s correct reading frame will be recovered                         and bioluminescence will be detected through a                         luciferase assay.
gRNA is a key element of CRISPR/Cas9 system. It leads                         Cas9 endonuclease to the selected target in order to                         produce a mutation. In order to maximize the mutation                         efficiency in the desired crop variety gRNA must work                         properly. Therefore gRNA design must be optimal.
Plant breeders could use the gRNA provided by our                         software or any other one to directly edit their                         desired crop. However, this process usually takes long                         time and work turns time-consuming and in some cases                         effortless. It would take a long time just to know if                         the gRNA has worked or not. For instance, orange trees                         need at least 1-2 years to exceed the first growth                         stage, and 3-4 years to produce the first fruits.                         Therefore, we encourage plant breeders to use our gRNA                         testing system in order to determine the efficiency of                         their RNA.
Why wouldn’t our gRNA work?
Our open-source database includes genes of interest,                         which can be selected by plant breeders according to                         their needs. It also directly provides the optimal gRNA                         which, afterwards, must be ordered to synthesize. The                         gRNA is obtained from gene consensus sequences acquired                         from big databases such as NCBI or Sol Genomics.                         However, the consensus sequence may match or not with                         the specific variety to improve. Insertions and                         deletions –Indels – and Single Nucleotide Polymorphisms                         –SNPs – occur spontaneously and randomly in the genome                         of plants in nature. That means the gRNA obtained by                         our data processing software could not work optimally                         on the desired variety. This makes almos mandatory to                         test the gRNA in vivo before using it in the specific                         variety. Furthermore, even if the gRNA matches                         perfectly with the target sequence, mutagenesis may not                         occur - or be less efficient - due to gRNA secondary                         structure problems, as described by Summer et al.                         (1).
gRNA testing methods
The current methods used to test the efficiency of                         CRISPR/Cas9 consists in digesting amplified mutated                         region with restriction enzymes or with T7                         endonuclease. However, these methods are not suitable                         if we want to be able to test any gRNA with a                         reproducible method. Therefore, a new modular and                         standard gRNA in vivo testing method claims to be                         developed.
Digestion with restriction enzymes: the major                             drawback of this method relies on the fact that                             only targets that includes a restriction site in                             DSB region are eligible. In order to check if the                             mutation has occured, a digestion is performed with                             the chosen restriction enzyme, and an                             electrophoresis gel is carried out. If mutation                             occured, the restriction site will be lost and the                             enzyme won’t cut. Therefore, in the electrophoresis                             gel, a band with higher molecular weight should be                             observed. If mutation did not occur, two bands                             should be observed, since the enzyme can recognize                             the site and cut.
Problem: the range of possible targets                                     is reduced, because they must contain a                                     restriction site exactly in the place where                                     the Cas9 cuts. Moreover, determining                                     efficiency ratios through this method is                                     difficult and not very accurate since it is                                     based on band signal intensity                                     comparison.
Digestion with T7 endonuclease: this strategy                             is similar to the restriction enzymes one, yet it                             uses the T7 endonuclease. This endonuclease cuts                             where it finds heterodimers. When Cas9 cuts, due to                             the non-homologous end joining DNA repair mechanism                             - NHEJ -, plant cells introduce indels (2). After a                             PCR amplification of the region, heterodimers can                             be obtained from a denaturation and reannealing                             step. When they are annealed, they might bind with                             a strand which is not exactly complementary,                             producing heterodimers that T7 can recognize and                             cut. Therefore, in an electrophoresis gel, a high                             molecular weight band will be observed if there is                             not a cut, while, two shorter bands will appear if                             T7 endonuclease cuts and so, Cas9 is well                             working.
Problem: the T7 endonuclease is                                     outrageously expensive. Moreover, this                                     method does not allow to analyze the                                     obtained mutation by sequencing since those                                     frgaments with the mutation will be                                     cut.
In order to provide an efficient solution to the                         aforementioned drawbacks, we have engineered a gRNA                         Testing System. In this strategy, we chose Nicotiana                         benthamiana as chassis due to its fast growth and                         all the benefits provided by a model plant. Our                         methodology is Phytobricks standardized, thus turning                         this system into a modular and standard one, two of the                         mainstays of Synthetic Biology.
Our device
Any selected target can be inserted in the Testing                         System since its Phytobricks standardized and its                         overhang sequences are well defined. Based on Agrobacterium tumefaciens infection and                         using luciferase as reporter gene, we will be able to                         know how efficient the provided/designed gRNA is.
First of all, plant breeders must perform a genomic DNA                         extraction of the plant they want to modify. Next, they                         need to amplify the region they are going to target.                         The primers needed to carry out this amplification are                         provided by our Data Processing Software. The sequence                         of the amplified target is then analyzed by sequencing.                         Next step consists in ordering overlapping                         oligonucleotides corresponding with the analyzed target                         sequence. Finally the target is introduced in the                         construction of our testing system, introduced in                         A.tumefasciens and next in plant leaf cells by                         agroinfiltration (Fig. 1).
(Fig. 1)                             Schematic of the insertion of the target inside the                             construction of gRNA Testing System circuit.
This device is introduced in the plant along with the                         corresponding gRNA and Cas9 construction. If the gRNA                         leads Cas9 to the target and mutations are produced,                         bioluminescence signal will be detected through a luciferase assay.
The main advantages of this kind of assays are its                         straightforward protocol (the analysis of each sample                         only requires a few minutes) its enormous sensitivity                         and its accuracy on the obtained results. Thus,                         allowing us to obtain quantitative results.
The system is designed in order to be in OFF state                         until the DSB and subsequent repair is produced. When                         Cas9 cuts, indels appear, luciferase open reading frame                         returns to the correct one and system turns ON (Fig. 2                         and 3). Leaf samples are assayed for the presence of                         the reporter gene by directly measuring the enzymatic                         activity of luciferase enzyme in the presence of known                         concentration of luciferin.
(Fig. 2)                             Schematic of the function of the genetic circuit                             that composes the gRNA Testing System. Only when                             DSB and subsequent repair is produced the                             luciferase’s open reading frame will shift to the                             correct one (ON state).
(Fig. 3)                             Device after endonuclease Cas9 cut and indels                             production. a) The extra nucleotide is located                             before the linker, so when indels change the frame,                             the linker and the luciferase are in the correct                             ORF. b) The extra nucleotide is located after the                             linker. Thus linker’s frame will be different from                             luciferase’s one after indel occurs.
Parts of the device (gRNA Testing System 1.0)
All the parts used in the construction of the gRNA                         testing system 1.0 and following versions are described here.
P35s: It is a strong constitutive promoter                             derived from cauliflower mosaic virus (CaMV). It is                             widely used in plants to improve the level of the                             expression of foreign genes effectively in all                             tissues (3).
5’ region: 5’ region from the N.                             benthamiana polyubiquitin sequence (Accession                             number: Nbv5.1tr6241949) is used due to its high                             expression rate in every plant tissue (4). It can help our gene to express itself and ensures that                             the construction is well expressed. It contains                             BsmbI and BsaI recognition sites with AATG in 3’ as                             overhang that allows us to ligate with the                             amplified target.
Target: Plant breeders will obtain the selected                             gene from their original plant by PCR amplification                             with the primers provided by our Data Processing                             Software. These primers are designed in order to                             obtain amplicons with the overhangs needed to                             insert the target in our device, corresponding to                             Phytobricks grammar (prefix: AATG, suffix: TTCG).                             It’s mandatory to make sure that this region                             doesn’t contain any stop codon in frame +1, but                             neither in frame +2, so when an indel happens the                             reading frame won’t be disrupted. The software                             finds the optimal target with its corresponding                             gRNA.
Linker SAGTI (Ser-Ala-Gly-Thr-Ile): it is a                             flexible peptide linker between the target and the                             luciferase. It allows the luciferase to acquire the                             correct structure, avoiding interaction with the                             target. Thus, luciferase assay will be carried out                             in a successfully way. As it can be seen in Figure                             3, the first part of the device is in ORF +1                             whereas the luciferase is in ORF +2. The device is                             designed so that there is an extra nucleotide                             before the linker sequence that changes the reading                             frame. If this nucleotide were after the linker                             sequence, when Cas9 cut, the reading frame of the                             linker will change, and the amino acids translated                             won’t be the correct ones. Figure 3 shows how it                             will be translated after the indels.
Luciferase: It is widely used as a reporter                             enzyme. At the beginning, luciferase gene is out of                             the reading frame due to the presence of an extra                             nucleotide. After CRISPR/Cas9 acts indels occur.                             These insertions and deletions of nucleotides                             produce a frameshift and change luciferase’s frame                             in the right frame +1, so it will be correctly                             translated. In the experimental design, the first                             methionine has been removed just to prevent the                             unintended translation of the luciferase. Therefore                             appearance of false positives will be avoided.
Tnos: Nopaline synthase terminator of the                             nopaline synthase gene of Agrobacterium tumefaciens. It is used for gene transfection                             (5).
New parts of the device (gRNA Testing System                         2.0)
After building the device different luciferase were                         performed in order to test its feasibility. Once the                         obtained results were analyzed (See results here ),                         we determined that the system were working well but its                         performance could be improved. Bearing that in mind, we                         resolved to remove the 5’ region in order to obtain                         higher levels of luciferase expression and also to test                         different linkers and choose the most appropriate one.                         The new selected linkers are:
EAAAK linker: rigid helical linker with the                             sequence of A(EAAAK)nA (n =3) that effectively                             separates protein domains as reported by Chen et                             al. (6). An extra nucleotide was added following                             the same strategy design that with SAGTI                             linker.
RSIAT linker: short linker that separates                             protein domains and that have been succesfully used                             BiFC experiments (7). An extra nucleotide was added                             following the same strategy design that the                             aforementioned linkers.
RSIAT+TEV linker: RSIAT linker (7) followed by                             a TEV protease recognition site. Therefore Tobacco                             Etch Virus NIa protease will recognize the sequence                             and cleave it, obtaining two different peptides as                             result. On one side target plus linker peptide, and                             on the other side luciferase protein. With this                             linker we aim to obtain a more orthogonal design in                             which the target sequence has minimal interference                             with luciferase protein. Thus, luminescence signal                             will not depend on the target.
References:
1. Thyme S, Akhmetova L, Montague T, Valen E, Schier                         A. Internal guide RNA interactions interfere with                         Cas9-mediated cleavage. Nature Communications.                         2016;7:11750.
2. Biolabs N. Determining Genome Targeting Efficiency                         using T7 Endonuclease I | NEB (Internet). Neb.com. 2016                         (cited 27 July 2016). Available from:                         https://www.neb.com/protocols/2014/08/11/determining-genome-targeting-efficiency-using-t7-endonuclease-i
3. Pauli S, Rothnie H, Chen G, He X, Hohn T. The                         Cauliflower Mosaic Virus 35S Promoter Extends into the                         Transcribed Region. Journal of Virology.                         2004;78(22):12120-12128.
4. Benthamiana Atlas (Internet). Sefapps02.qut.edu.au.                         2016 (cited 27 June 2016). Available from:                         http://sefapps02.qut.edu.au/atlas/tREXXX2new.php?TrID=Nbv5.1tr6241949
5. Holden M, Levine M, Scholdberg T, Haynes R, Jenkins                         G. The use of 35S and Tnos expression elements in the                         measurement of genetically engineered plant materials.                         Analytical and Bioanalytical Chemistry.                         2009;396(6):2175-2187.
6. Chen X, Zaro J, Shen W. Fusion protein linkers:                         Property, design and functionality. Advanced Drug                         Delivery Reviews. 2013;65(10):1357-1369.
7. Hu CKerppola T. Simultaneous visualization of                         multiple protein interactions in living cells using                         multicolor fluorescence complementation analysis.                         Nature Biotechnology. 2003;21(5):539-545.
Follow                                 Us
Sponsors
