<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_Grna_testing_system skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/Grna testing system</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HYPE IT - Valencia UPV</TITLE></P><DIV id="wrapper"><DIV id="header" class="sticky clearfix"><HEADER id="topNav"><DIV class="container"><DIV class="navbar-collapse pull-right nav-main-collapse collapse submenu-dark"><NAV class="nav-main"><UL id="topMain" class="nav nav-pills nav-main"><LI><A href="https://2016.igem.org/Team:Valencia_UPV"> HOME </A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Team">OUR TEAM</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Attributions">ATTRIBUTIONS</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="#"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Project">OUR PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Design">DESIGN</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Software">DATA PROCESSING SOFTWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/proof">GRNA TESTING SYSTEM</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Split">SPLIT CAS9</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware">LAB CASE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/notebook">NOTEBOOK</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Human_Practices">OUR HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Silver">SOCIAL ENGAGEMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Gold">GOLD</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Integrated_Practices">INTEGRATED PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Safety">SAFETY</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> PARTS </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Parts">Our PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Basic_Part">BASIC PARTS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Composite_Part">COMPOSITEPARTS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Part_Collection">PART COLLECTION</A></LI><LI><A class="active" href="https://2016.igem.org/Team:Valencia_UPV/Model"> MODEL </A></LI></NAV></DIV></DIV></HEADER></DIV><SECTION class="page-header page-header-xs"><DIV class="container"><H1>GRNA TESTING SYSTEM</H1><SPAN class="font-lato size-18 hidden-xs">We believe in Simple &amp; Creative</SPAN></DIV></SECTION><SECTION><DIV class="container"><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><H1 class="blog-post-title">ABSTRACT</H1><P class="dropcap" style="text-align: justify">We have designed a modular gRNA testing system in order to check if the gRNA provided by the Data Processing Software works as expected on the plant variety to improve. This system works as a <STRONG>genetic switch</STRONG> that remains OFF if the gRNA does not work, and turns ON when the Cas9 does a double stranded break in the target. The device performance is based on the fact that the <STRONG>luciferase reporter gene</STRONG> is out of its reading frame. When CRISPR/Cas9 system works as expected, meaning that the <STRONG>gRNA is well designed</STRONG>, it will introduce indels in our device and the luciferase will be placed on its correct reading frame. Therefore, we will be able to detect bioluminescence with a luciferase assay.</P></DIV></DIV></DIV></SECTION><SECTION><DIV class="container"><DIV class="heading-title heading-border"><H3>Related <SPAN>Projects</SPAN></H3><P class="font-lato size-14">Lorem ipsum dolor sit amet.</P></DIV><DIV class="text-center"><DIV class="owl-carousel owl-padding-1 nomargin buttons-autohide controlls-over" data-plugin-options="{&quot;singleItem&quot;: false, &quot;items&quot;: &quot;4&quot;, &quot;autoPlay&quot;: 3500, &quot;navigation&quot;: true, &quot;pagination&quot;: false}"><DIV class="item-box"><DIV class="item-box-desc"><H3>Architect Project</H3><UL class="list-inline categories nomargin"><LI><A href="#">Architecture</A></LI><LI><A href="#">Design</A></LI></UL></DIV></DIV><DIV class="item-box"><DIV class="item-box-desc"><H3>Architect Project</H3><UL class="list-inline categories nomargin"><LI><A href="#">Architecture</A></LI><LI><A href="#">Design</A></LI></UL></DIV></DIV><DIV class="item-box"><DIV class="item-box-desc"><H3>Architect Project</H3><UL class="list-inline categories nomargin"><LI><A href="#">Architecture</A></LI><LI><A href="#">Design</A></LI></UL></DIV></DIV><DIV class="item-box"><DIV class="item-box-desc"><H3>Architect Project</H3><UL class="list-inline categories nomargin"><LI><A href="#">Architecture</A></LI><LI><A href="#">Design</A></LI></UL></DIV></DIV><DIV class="item-box"><DIV class="item-box-desc"><H3>Architect Project</H3><UL class="list-inline categories nomargin"><LI><A href="#">Architecture</A></LI><LI><A href="#">Design</A></LI></UL></DIV></DIV><DIV class="item-box"><DIV class="item-box-desc"><H3>Architect Project</H3><UL class="list-inline categories nomargin"><LI><A href="#">Architecture</A></LI><LI><A href="#">Design</A></LI></UL></DIV></DIV><DIV class="item-box"><DIV class="item-box-desc"><H3>Architect Project</H3><UL class="list-inline categories nomargin"><LI><A href="#">Architecture</A></LI><LI><A href="#">Design</A></LI></UL></DIV></DIV><DIV class="item-box"><DIV class="item-box-desc"><H3>Architect Project</H3><UL class="list-inline categories nomargin"><LI><A href="#">Architecture</A></LI><LI><A href="#">Design</A></LI></UL></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION><DIV class="container"><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><H1 class="blog-post-title">GRNA TESTING SYSTEM</H1><P class="dropcap" style="text-align: justify">GRNA is a key element in genome editing with CRISPR/Cas9 system, since it leads endonuclease Cas9 where the modification must be produced. That means gRNA must work properly in the plant variety that we want to improve.</P><P> The plant breeder could use the gRNA provided by our software directly in his plant. However, lots of plant species take a long time to grow. For example, orange trees need at least 1-2 years to exceed the first growth stage, and 3-4 years to produce the first fruits. If the gRNA didn’t work properly, the plant breeder would had lost a valuable time waiting to see a phenotypic improvement on his plant. It would take a long time just to know if the gRNA has worked or not. Due to that, it’s necessary to previously check the gRNA proper functioning.</P></DIV></DIV></DIV></SECTION><SECTION><DIV class="container"><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><H1 class="blog-post-title">WHY WOULDN’T OUR GRNA WORK?</H1><P class="dropcap" style="text-align: justify"> Our open-source database includes genes of interest, which can be selected by plant breeders according to their needs. It also directly provides the optimal gRNA which, afterwards, must be ordered to synthesize. The gRNA is obtained from gene consensus sequences acquired from big databases such as NCBI or Sol Genomic. However, the consensus sequence may match or not with the specific variety to improve. Insertions and deletions –Indels – and Single Nucleotide Polymorphisms –SNPs – occur spontaneously and randomly in the genome of different plant varieties. That means the gRNA obtained by our data processing software could not work optimally on the desired variety. This makes even more necessary to test the gRNA before using it in the specific variety. Furthermore, even if the gRNA matches perfectly with the target sequence, mutagenesis may not occur - or be less efficient - due to gRNA secondary structure problems.</P></DIV></DIV></DIV></SECTION><SECTION><DIV class="container"><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><H1 class="blog-post-title">GRNA TESTING METHODS</H1><P style="text-align: justify">The current methods used to test the efficiency of CRISPR/Cas9 -and therefore that can be used to test the gRNA- are digestion with restriction enzymes or digestion with T7 endonuclease and subsequent sequencing. However, these methods are not suitable if we want to make accessible and easy the testing of the gRNA.</P><UL><LI style="text-align: justify"><P style="text-align: justify">Digestion with restriction enzymes: when choosing the target within the gene, it is mandatory to choose a target including a restriction site where the Cas9 will make the DSB. Therefore, in order to check if the Cas9 produced the mutation, a digestion is performed with the restriction enzyme, and an electrophoresis gel is carried out. If the Cas9 cuts, the restriction site will be lost and the enzyme won’t cut. Therefore, in the electrophoresis gel, a band with higher molecular weight should be observed. If mutation did not occur, two bands should be observed, since the enzyme can recognize the site and cut.</P></LI><UL><LI style="text-align: justify"><P style="text-align: justify">Problem: the range of possible targets is reduced, because they must contain a restriction site exactly in the place where the Cas9 cuts. Additionally, when you find a target with a restriction site you might not have the needed enzyme. Buying it may imply a high cost, not affordable for everyone.</P></LI></UL><LI style="text-align: justify"><P style="text-align: justify">Digestion with T7 endonuclease: this strategy is similar to the restriction enzymes one, yet it uses the T7 endonuclease. This endonuclease cuts where it finds heterodimers. When Cas9 cuts, due to the non-homologous end joining DNA repair mechanism - NHEJ -, plant cells introduce indels. After a PCR amplification of the region, heterodimers can be obtained from a denaturation and reannealing step. When they are annealed, they might bind with a strand which is not exactly complementary, producing heterodimers that T7 can cut. Therefore, in an electrophoresis gel, a high molecular weight band will be observed if there is not a cut, while, two shorter bands will appear if T7 endonuclease cuts and so, Cas9 is well working.</P></LI><UL><LI style="text-align: justify"><P style="text-align: justify">Problem: the T7 endonuclease is outrageously expensive. Buying it may imply even a higher cost, not affordable for everyone.</P></LI></UL></UL><P style="text-align: justify">In order to provide an efficient solution to the drawbacks explained above, we have engineered a gRNA Testing System. In this strategy, we use Nicotiana benthamiana due to its fast growth and all the benefits provided by a model plant. Our methodology is based in the GoldenBraid Assembly System, that allows us to get a modular and standard system, two of the mainstays of Synthetic Biology field.</P></DIV></DIV></DIV></SECTION><SECTION><DIV class="container"><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><H1 class="blog-post-title">OUR DEVICE</H1><P class="dropcap" style="text-align: justify">The fragment of genome that we are going to target in our plant is inserted in the Testing System following the concept of modularity. Based on <EM>Agrobacterium tumefaciens</EM> infection and using a reporter gene in our device, we will be able to know how efficient the provided/designed gRNA is.</P><DIV class="blog-single-small-media"><DIV class="caption-primary"><STRONG>This is a simple caption.</STRONG> Lorem ipsum dolor sit amet adipiscium elit.</DIV></DIV><P style="text-align: justify">First of all, plant breeders have to carry out a genomic DNA extraction of the plant they want to modify. Next, they need to amplify the region they are going to target. The targets needed to this amplification are provided by our Data Processing Software. The amplified target sequence is introduced in <EM>N. benthamiana</EM> as part of the device with the luciferase reporter.</P><P style="text-align: justify">This device is introduced in the plant along with the corresponding gRNA and Cas9 construction. If the gRNA works on the desired variety, we will be able to detect it easily with a luciferase assay.</P><P style="text-align: justify">Our system is designed to allow the detection of luminescence. Thus, a luciferase assay is used to study gene expression rates, since it is fast and the analysis of each sample only requires a few minutes. Moreover, it is extremely sensitive and the results are very accurate, allowing us to obtain quantitative results. Originally the system is OFF since luciferase genetic sequence is not in the correct frame. <STRONG>When Cas9 cuts, indels appear, system turns ON, so luciferase gene is in the correct frame and it will be correctly translated</STRONG>. In that case, the plant breeder could check the genome editing in a simple way. Cells are assayed for the presence of the reporter by directly measuring the enzymatic activity of the reporter protein on luciferin substrate.</P><CENTER><DIV class="blog-single-small-media"><DIV class="caption-primary"><STRONG>This is a simple caption.</STRONG> Lorem ipsum dolor sit amet adipiscium elit.</DIV></DIV></CENTER></DIV></DIV></DIV></SECTION><FOOTER id="footer"><DIV class="container"><DIV class="row"><DIV class="col-md-3"><P>Integer posuere erat a ante venenatis dapibus posuere velit aliquet.</P><ADDRESS><UL class="list-unstyled"><LI class="footer-sprite address"> PO Box 21132 Here Weare St, Melbourne Vivas 2355 Australia</LI><LI class="footer-sprite phone"> Phone: 1-800-565-2390</LI><LI class="footer-sprite email"><A href="mailto:support@yourname.com">support@yourname.com</A></LI></UL></ADDRESS></DIV><DIV class="col-md-3"><H4 class="letter-spacing-1">LATEST NEWS</H4><UL class="footer-posts list-unstyled"><LI><A href="#">Donec sed odio dui. Nulla vitae elit libero, a pharetra augue</A><SMALL>29 June 2015</SMALL></LI><LI><A href="#">Nullam id dolor id nibh ultricies</A><SMALL>29 June 2015</SMALL></LI><LI><A href="#">Duis mollis, est non commodo luctus</A><SMALL>29 June 2015</SMALL></LI></UL></DIV><DIV class="col-md-2"><H4 class="letter-spacing-1">EXPLORE SMARTY</H4><UL class="footer-links list-unstyled"><LI><A href="#">Home</A></LI><LI><A href="#">About Us</A></LI><LI><A href="#">Our Services</A></LI><LI><A href="#">Our Clients</A></LI><LI><A href="#">Our Pricing</A></LI><LI><A href="#">Smarty Tour</A></LI><LI><A href="#">Contact Us</A></LI></UL></DIV><DIV class="col-md-4"><H4 class="letter-spacing-1">KEEP IN TOUCH</H4><P>Subscribe to Our Newsletter to get Important News &amp; Offers</P><FORM class="validate" action="php/newsletter.php" method="post" data-success="Subscribed! Thank you!" data-toastr-position="bottom-right"><DIV class="input-group"><SPAN class="input-group-btn"><BUTTON class="btn btn-success" type="submit">Subscribe</BUTTON></SPAN></DIV></FORM></DIV></DIV></DIV><DIV class="copyright"><DIV class="container"><UL class="pull-right nomargin list-inline mobile-block"><LI><A href="#">Terms &amp; Conditions</A></LI><LI>•</LI><LI><A href="#">Privacy</A></LI></UL>© All Rights Reserved, Company LTD</DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>