<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_CP skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/CP</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header" class="sticky dark header-md noborder clearfix fixed"><HEADER id="topNav"><DIV class="container"><DIV class="navbar-collapse pull-right nav-main-collapse collapse submenu-dark"><NAV class="nav-main"><UL id="topMain" class="nav nav-pills nav-main"><LI><A href="https://2016.igem.org/Team:Valencia_UPV"> HOME </A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Team">MEET US</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Attributions">ATTRIBUTIONS</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="#"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Description">DESCRIPTION</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Software">DATA PROCESSING SOFTWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Proof" style="text-transform: none;">SPLIT Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Notebook">NOTEBOOK</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> LAB WARE </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware">HARDWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents">Reagents</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual">Manual</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Silver">SOCIAL ENGAGEMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Gold">SOLVING REAL PROBLEMS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Integrated_Practices">IMPACT ASSESSMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Safety">SAFETY</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> PARTS </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Part_Collection">BEST PART COLLECTION</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> MODEL</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Model">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/CP">Cas9:gRNA Complex Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Loop">R-Loop Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/ORF">Cleavage and Reading Frame Shift</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Conclusions">Conclusions</A></LI><LI><A class="active" href="https://2016.igem.org/Team:Valencia_UPV/Achievements"> ACHIEVEMENTS</A></LI></NAV></DIV></DIV></HEADER></DIV><DIV class="container"><H1>Cas9:gRNA complex formation</H1></DIV><SECTION><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-2 col-sm-3"><DIV class="side-nav margin-bottom-60 margin-top-30"><DIV class="side-nav-head"><H4>Index</H4></DIV><UL class="list-group list-group-bordered list-group-noicon uppercase"><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/CP#Overview_id">Overview
									</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/CP#ODEs_id">ODEs
									</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/CP#Simulations._id">Simulations
									</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/CP#Parameters1_id">Parameters
									</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/CP#Mainremarks_id">Main remarks
									</A></LI></UL></DIV></DIV><DIV class="col-md-10 col-sm-9"><DIV class="blog-post-item" id="Overview_id"><H3>Overview</H3><P>Since Cas9 and gRNA constructions are delivered in the host, they will be transcribed (and translated in the case of Cas9) by the cell’s machinery. When Cas9 and grNA find each other in the plant nucleus, they will interact forming the Cas9:gRNA complex.</P><P>This complex will perform the knockout in the Testing System construction. Thus, there is a clear relation between the amount of this complex and the light signal produced. Optimizing the complex formation, we will be able to optimize the light measurements.

Studying this process, our aim was to analyze its stability under different conditions. Using Mass Action Kinetics, Quasi Steady State Analysis and Direct Lyapunov Method, we determined conditions to reach the steady state that provides the optimum amount of Cas9:gRNA complex. Firstly, we analyze the <B>interaction between Cas9 and gRNA</B>, following with a <B>study of agroinfiltration times</B> in the second section. 
</P></DIV><DIV class="blog-post-item" id="ODEs_id"><H3>ODEs</H3><P><B>Biochemical reactions</B></P><P>All the process takes place in the nucleus. The dynamics of these species can be represented by the following set of reactions:</P><P>The involved guide-RNA (gRNA) molecule has two main parts: the scaffold and the spacer. The first one is recognized by the inactive Cas9, resulting in an <B>intermediate complex</B> (Cas9 + gRNA) which is <B>unstable</B>. This reaction governed by the rate k<SUB>f</SUB>. Afterwards, it takes place the conformational change of the Cas9 endonuclease, leading to an <B>irreversible isomerization</B>, represented by the constant k<SUB>I</SUB>. Finally, the Cas9:gRNA complex is ready to scan the genome.</P><P><B>System of equations</B></P><P>Elements from reactions on the left are usually expressed in terms of <B>concentration</B> ([Cas9], [gRNA], [Cas9+gRNA], [Cas9:gRNA]). We rename those concentrations using the variables x<SUB>1</SUB>=[Cas9], x<SUB>2</SUB>= [gRNA], x<SUB>3</SUB>= [Cas9+gRNA], and x<SUB>4</SUB>=[Cas9:gRNA].</P><P>Then, using Mass Action Kinetics, we can build the system of Ordinary Differential Equations: </P><P>
This model represents the dynamics of two elements which interact affecting the steady-state condition of each other. Constants<B> α<SUB>1</SUB></B> and <B>β<SUB>1</SUB></B> reflect a <B>continuous input</B> of x<SUB>1</SUB><B>(Cas9)</B> and x<SUB>2</SUB><B>(gRNA)</B>. 

Initially, Cas9 and gRNA constructions get to the cell by viral infection. Thus, if the subject of study was a cell population, the flux of Cas9 and gRNA would vary according to the spread of the viral infection among cells. However, from an individual perspective, once a cell has been infected it constitutively produces Cas9 and gRNA. Thus, production of both elements in a <B>single cell</B>, fits the typical behavior of <B>constitutive production</B>, i.e. reaches the steady-state after certain time. This permanent production is equivalent to the constant input of Cas9 and gRNA in the cell, reflected in terms α<SUB>1</SUB> and β<SUB>1</SUB>, respectively.

Terms <B>α<SUB>2</SUB>⋅x<SUB>1</SUB>⋅x<SUB>2</SUB></B> and <B>β<SUB>2</SUB>⋅x<SUB>1</SUB>⋅x<SUB>2</SUB></B> describe the <B>exchange of Cas9</B> (x<SUB>1</SUB>) and <B>grNA</B> (x<SUB>2</SUB>) when they interact. Finally, <B>α<SUB>3</SUB>⋅x<SUB>1</SUB></B> and <B>β<SUB>3</SUB>⋅x<SUB>2</SUB></B> represent the <B>disappearance</B> of both species (x<SUB>1</SUB> and x<SUB>2</SUB>) from the system, by degradation or dillution. 
</P><P><B>Steady state analysis</B></P><P>Steady state conditions of the system will be determined by the stability of the interaction between Cas9 and gRNA. Thus, equations (eq. 1) and (eq. 2) expressing the variation of these elements, must be analyzed in order to characterize the behavior of the system. They fit to the <B>general expression</B> of the following model:
</P><P>
This model represents the dynamics of two elements which interact affecting the steady-state condition of each other.  As α<SUB>3</SUB> is the degradation rate of Cas9 and it is being compared to the degradation of the gRNA (β<SUB>3</SUB>), we can assume that α<SUB>3</SUB> is depreciable in relation to β<SUB>3</SUB>, which is the dominant degradation term. Therefore, the model is simplified, resulting in the <B>equilibrium positions</B>:
</P><P>
Using the Direct Lyapunov Method (3), we can get two expressions which can be used to know about the stability of the system given a solution. Further details about the mathematical discussion to demonstrate the use of the Direct Lyapunov Method are available in our Mathematical discussion. 

If conditions σ&lt;0 and Δ&gt;0 are provided, it can be assumed that the system will achieve a <B>local stability</B>. Replacing each model parameter (Table 1) by its value in our particular case, steady-state solutions are:
</P><P>
Therefore we can assume that stability conditions of the system (eq. 13) and (eq. 14) are achieved, considering that our system is locally stable around the equilibrium point:
</P></DIV><DIV class="blog-post-item" id="Parameters1_id"><H3>Parameters</H3><DIV class="table-responsive" style="text-align:center;width:55%;overflow:inherit"><TABLE class="table table-bordered table-striped"><TBODY><TR><TH>Parameter in general expression</TH><TH>Particular Value</TH><TH>Numeric Value (min<SUP>-1</SUP>)</TH><TH>Source</TH></TR><TR><TD>α<SUB>1</SUB></TD><TD>k<SUB>Cas9</SUB></TD><TD>0.000374737</TD><TD>(16)</TD></TR><TR><TD>α<SUB>2</SUB></TD><TD>k<SUB>f</SUB></TD><TD>0.00006</TD><TD>(16)</TD></TR><TR><TD>α<SUB>3</SUB></TD><TD>δ<SUB>Cas9</SUB></TD><TD>0.0000552</TD><TD>(16)</TD></TR><TR><TD>β<SUB>1</SUB></TD><TD>k<SUB>gRNA</SUB></TD><TD>0.0025284</TD><TD>(16)</TD></TR><TR><TD>β<SUB>2</SUB></TD><TD>k<SUB>f</SUB></TD><TD>0.00006</TD><TD>(16)</TD></TR><TR><TD>β<SUB>3</SUB></TD><TD>δ<SUB>gRNA</SUB></TD><TD>0.000252</TD><TD>(16)</TD></TR></TBODY></TABLE><P class="imgFooterP" style="text-align: center;font-style: italic;">Table 1. Parameters involved in Cas9 and gRNA dynamics.</P></DIV></DIV><DIV class="blog-post-item" id="Simulations._id"><H3>Simulations</H3></DIV><DIV class="blog-post-item" id="Cas9andgRNAinteraction._id"><H4>Cas9 and gRNA interaction</H4><P>Once we analytically determined that our solution was feasible and robust, we simulated in Matlab the same conditions in order to check if the steady-state values obtained analytically were the expected ones. As is can be seen in graphics below, the equilibrium values predicted analytically are very close to those obtained simulating in Matlab. There is a small difference between analytical and experimental results. This is probably due to the difference in the theoretical value of α<SUB>3</SUB>, which is presumably zero, and the value introduced in simulations, which is not null (value in Table 1).
</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 1. Simulation of Cas9 evolution using parameters from Table 1 and one gene copy number.</P></DIV><P>
Simulations performed let us check the equilibrium values obtained analytically. Analyzing Figure 1, it is observed that <B>Cas9 </B>endonuclease is produced reaching its maximum at the time of peak t<SUB>p</SUB> = 4877 minutes, i.e. 4 days approx. From day 4, there is enough of both compounds in order to form the Cas9:gRNA complex, explaining the decay in Cas9 after 4 days from infiltration. Four days later, at  t≈10 days (15000 minutes), Cas9 reaches its steady-state near the 0.7 nM predicted analytically. This behavior is similar to a Second Order System response after a step input. It reaches the equilibrium value with a time slope of t<SUB>s</SUB> = 2380 minutes, i.e. 1 day and 15 hours.
</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 2. Simulation of [gRNA] evolution using parameters from Table 1 and one gene copy number.</P></DIV><P>
Regarding the <B>gRNA</B> (Figure 2), it is produced without any oscillation. The maximum production (8.6 nM) is reached between around the 13th day, and the 63 % of this value at τ ≈3400 minutes. As <B>gRNA has a dynamic considerably faster than Cas9</B>, it keeps growing until the endonuclease is produced. As soon as this happens, Cas9 and gRNA are used to form the Cas9:gRNA complex. Thus, the peak in Figure 1 makes reference to this complex formation. In Figure 2 this consumption is not appreciated because there is plenty more gRNA disposable than gRNA used to form the complex with Cas9 produced.

However, using one gene copy number is not a current laboratory condition. As production of Cas9 and gRNA is directly related to the number of gene copies, higher amounts of them are a more realistic subject of study. Since terms K’<SUB>Cas9</SUB> and K’<SUB>gRNA</SUB> are a function of the number of copies, changing this parameter the equilibrium values will be affected. Following graphics represent different results <B>varying the gene copy number</B> only for Cas9, only for gRNA and finally for both of them.
</P><P><B>gRNA gene copy number</B></P><P>In the following simulations the gene copy number of the gRNA was increased gradually from 1 to 45. Results obtained each 5 copies were plotted to analyze the stability of Cas9 when the gRNA is produced faster and in higher quantities.
</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 3. Simulations of [Cas9] evolution using parameters form Table 1.</P></DIV><P>
From the simulations above, it is clear that <B>Cas9 efficacy is highly sensitive to the presence of gRNA in the nucleus</B>. The biggest difference among concentrations of Cas9 in the graphic above, is produced between the result with 1 copy and the result with 5. This means that even though Cas9 production is not increased, the endonuclease will be able to find the gRNA and form complexes with it. There is an indirect relation between the maximum Cas9 presence, the t<SUB>p</SUB> at which this maximum is reached, and the amount of gRNA. It seems quite logical, as the more gRNA present, the more likely it will be that all Cas9 produced is consumed to form the complex with those gRNAs.
</P><P>From the picture below, it can be confirmed that an increment in the Cas9 gene copy number, affects homogeneously to the quantity of gRNA. Qualitatively is has the same behavior as in the scenario with one copy. The only difference is reflected in a gain of the steady-state gRNA value, which increases in 5-fold from the previous simulation with 5 gene copies less.
</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 4. Simulations of [gRNA] evolution using parameters from Table 1.</P></DIV><P><B>Cas9 gene copy number</B></P><P>
When the gene copy number for Cas9 is increased, the production of the endonuclease behaves similarly to the gRNA in the picture above. However, the slope is considerably less. This difference in the dynamics was expected, as it is known that <B>RNA</B> reactions are <B>several orders faster</B> than <B>protein</B> ones.
</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 5. Simulations of [Cas9] evolution using parameters from Table 1.</P></DIV><P>
There is no decrease in the Cas9 evolution that let us know when does the interaction with the gRNA begin. However, the beginning of this reaction is well appreciated in the graphic with the gRNA evolution.
</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 6. Simulations of [gRNA] using parameters from Table 1.</P></DIV><P>
As it is mentioned above, the initial slope of gRNA is considerably faster than Cas9’s, as gRNA only means transcription but Cas9 must be translated as well, delaying its growing. This behavior is comparable to Cas9 variations when the gRNA copy number was increased. However, the difference between simulations with one copy number and 5 copies, are less remarkable in this case, reaching the half of the one-copy steady state value in this case. Again, this difference is smaller due to the fast production of gRNA. The time of peak changes as well among simulations with different number of Cas9 gene copies, being the maximum at approx. 5000 minutes, i.e. 3 and a half days. The minimum time peak is produced near 2 days after production.
</P><P><B>gRNA and Cas9 gene copy numbers</B></P><P>Simulations varying both gene copy numbers simultaneously were performed. This time the amount of intermediate complex produced was calculated. Results obtained are shown in the following graphic:
</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 7. Simulations of [Cas9:gRNA] intermediate complex production with different number of copies for Cas9 and gRNA.</P></DIV><P>
From this results it can be inferred that <B>the amount of complex being produced is very sensitive to the amount of Cas9</B> in the nucleus. This is reflected in the higher values of complex when Cas9 gene copy number has reached only the value of 5. Necessary amounts of gRNA to increase the intermediate presence, are higher than those needed for Cas9. This means that <B>gRNA will be the limiting reagent of this process</B> (14). Though this may refute the fact that gRNA is produced faster by cell’s machinery, its fast degradation explains why this element is limiting the Cas9:gRNA complex ready to perform the knockout. 

Therefore, <B>less amount of Cas9 will not be as determining as less amount of gRNA</B>. It will be critical to provide the system with the necessary gRNA in order to let enough complex be formed. Otherwise, less knockouts would be performed in the Testing System and light signal recorded during Luciferase assays would be less significant.
</P></DIV><DIV class="blog-post-item" id="Agroinfiltrationtimes._id"><H4>Agroinfiltration times</H4><P>
Experimentally Cas9 and gRNA can be infiltrated at the same or at different times in the plant. The difference between Cas9 and gRNA dynamics is known and it should be considered in order to improve the performance of our Testing System.
 
Next scenarios will assume that <B>Cas9 has been introduced few days earlier than gRNA</B>, letting time enough to be produced until reaching a certain concentration. Simulations now considered an existing amount of Cas9 when the gRNA is not present yet (point 0).
</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 8. Simulations of [Cas9:gRNA] intermediate complex obtained with different conditions relying initial amount of Cas9 available at the host nucleus.</P></DIV><P>
As expected, infecting plants previously with Cas9 to let it be produced, improves considerably the amount of complex obtained. Moreover, the t<SUB>p</SUB> does not change drastically in comparison with null [Cas9] initial conditions, being around 1000 minutes. The steady state is achieved as well without compromising the stability of the system. Therefore, greater amounts of the complex will be produced in less time, or in other words, the efficiency of the system will be increased if the Cas9 is already in the cell when the gRNA is infiltrated.
</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 9. Simulations of [Cas9:gRNA] intermediate complex obtained with different initial amounts of Cas9 and gRNA, representing different agroinfiltration times of each specie.</P></DIV><P>
In graphics above it can be appreciated how <B>the system becomes more robust as more Cas9 is available</B> in the cell nucleus. The advantage of reducing the importance of the gRNA presence, is that the amount of light signal will not be perturbed by a low gRNA concentration. Thus, the Testing System output will vary according to the efficiency of the gRNA to find the target, and not relying on other external factors such as the cell’s efficiency in the gRNA transcription.
</P></DIV><DIV class="blog-post-item" id="Mainremarks_id"><H3>Main remarks</H3><P>
The <B>usual lab protocol</B> considers the <B>simultaneous agroinfiltration</B> of the gRNA and the Cas9. However, <B>in simulations</B> we observed that the production of the <B>complex</B> could be <B>maximized</B> if <B>Cas9</B> was <B>agroinfiltrated in first place</B>.
</P></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer" style="margin-bottom:-20px"><DIV class="container"><DIV class="row"><DIV class="col-md-5"><DIV class="row"><DIV class="col-md-6 col-sm-6 hidden-xs" style="padding:0;"><DIV class="margin-left-50 clearfix"><H4 class="margin-bottom-10" style="text-align:start;">Follow
                                Us</H4></DIV></DIV></DIV></DIV><DIV class="col-md-7"><H4 class="letter-spacing-1" style="text-align: start;">Sponsors</H4></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>