Team:Valencia UPV/ORF
HOME
TEAM
MEET US
COLLABORATIONS
ATTRIBUTIONS
PROJECT
DESCRIPTION
DATA PROCESSING SOFTWARE
SPLIT Cas9
RESULTS
NOTEBOOK
LAB WARE
HARDWARE
Reagents
Manual
HUMAN PRACTICES
OVERVIEW
SOCIAL ENGAGEMENT
SOLVING REAL PROBLEMS
IMPACT ASSESSMENT
SAFETY
PARTS
BEST PART COLLECTION
MODEL
OVERVIEW
Cas9:gRNA Complex Formation
R-Loop Formation
Cleavage and Reading Frame Shift
Conclusions
ACHIEVEMENTS
Cleavage and Reading Frame Shift
Index
Overview
Knockouts                                 estimation
ORFs                                 probability distribution
Main                                 remarks
Overview
If the R-loop is completely established, then the                         complex Cas9:gRNA stays bounded to the DNA strand and                         Cas9 will cut the DNA sequence. This will lead in the loss of some base pairs that the plant cell will repair                         via Non Homologous End Joining (NHEJ).
Consequently, this loss may cause a change in the reading frame (TSOFF →                         TSON), and in the proteins produced by the cell from that moment, performing what is known as knockout.
Taking profit from this shift in                         the DNA reading frame, we included Luciferase after the                         genetic target.
Therefore, light signal emitted during Luciferase                         assays, will provide an output information about the                         whole process performed by CRISPR/Cas9 within the plant                         cells. As it relies on the target finding by the                         Cas9:gRNA complex, it is also a measure of the pair gRNA-target efficiency in order to knockout the undesired gene.
Knockouts estimation
While the Cas9:gRNA complex is produced, it produces                         knockouts among the host genome. In our case, these knockouts are expected to improve the desired plant                         trait. In this section of the modeling, we wanted to                         study in silico if the suggested pair target-gRNA would                         work in vivo, obtaining a predicted number of knockouts                         performed in the Testing System construction for the                         time at which the Luciferase assay is performed.
The number of knockouts produced at measurement time t m will be:
Where:
The concentration [Cas9:gRNA] (t) is the amount                             of complex produced at the time when the luciferase                             assay is performed. This can be estimated using the                             system of ODEs described in the previous section                             Cas9:gRNA complex formation.
The term NON makes reference to the                             quantity of on-targets in the nucleus, i.e. the gene copy number of the Testing System construction which has been agroinfiltrated.
The rate kcleavage has information                             about the cleavage frequency in the Testing System                             construction. It has information about the complex                             diffusion, the rate of Cas9 cleavage, and the                             thermodynamic stability of the R-loop formed before                             the cleavage. It is:
The rates in last expression mean:
The rate of collisions kr, produced                             between the Cas9:gRNA complex and the DNA in the                             nucleus.
The rate of cleavage kc at which Cas9 cuts DNA strands.
The occurrence of the Cas9:gRNA dissociation from a DNA region.
The probability that the R-loop is formed                             between the complex and the target used in the                             Testing System construction. This parameter has                             also information about potential off targets.
Values for parameters kr and                         Pcomplex,target are related to diffusion and                         thermodynamics, respectively.
ORFs probability distribution
From the moment that Cas9:gRNA complex is produced,                         it starts diffusing, colliding and binding to                         on-targets (ONs) and off-targets (OFFs). Those have                         known initial concentrations, since the number of                         on-targets is the gene copy number of the Testing                         System construction, and the number of off-targets has                         been obtained by our off-target search algorithm.
After the complex interacts with a target, it changes                         it sequence. The cut performed by Cas9 takes place in                         the PAM-proximal region, avoiding the possibility of a                         new zip-union with the gRNA to form the R-loop again.                         Therefore, knockouts cannot be represented as off                         targets or on targets anymore, as binding to the gRNA                         will not be thermodynamically feasible because of base                         pairs rearrangement.
In our Testing System strategy, we rely on the reading                         frame shift to let Luciferase be transcribed. However,                         this event will not always be likely to happen. There                         are three possible reading frames and the resulting one                         after the Cas9 cut, is unknown. This created the necessity of estimating the                         probability that the resulting frame after the Cas9                         cleavage and NHEJ reparation, was the correct for                         Luciferase transcription.
Consulting bibliography about frequency of indels in Nicotiana benthamiana, we                         could gather information about this phenomenon. Results                         showed barely any difference between the three possible                         results (ORF+1, ORF+2 and ORF+3). Therefore, the                         probability that our Testing System would express                         Luciferase was PON≈1/3.
Figure 14. Estimation                             of probability associated to each one of the                             possible resulting reading frames after Cas9                             knockout. Blue bars represent experiments performed only with Nicotiana, while orange bars depict results obtained working with various plants.
Main remarks
Analysis of bibliographic data determined that the                         probability that our Testing System would express                         Luciferase was PON≈1/3. Further experimental                         results in the future will allow to fully validate the                         model.
Follow                                 Us
Sponsors
