Team:Valencia UPV/Conclusions
HOME
TEAM
MEET US
COLLABORATIONS
ATTRIBUTIONS
PROJECT
DESCRIPTION
DATA PROCESSING SOFTWARE
SPLIT Cas9
RESULTS
NOTEBOOK
LAB WARE
HARDWARE
Reagents
Manual
HUMAN PRACTICES
OVERVIEW
SOCIAL ENGAGEMENT
SOLVING REAL PROBLEMS
IMPACT ASSESSMENT
SAFETY
PARTS
BEST PART COLLECTION
MODEL
OVERVIEW
Cas9:gRNA Complex Formation
R-Loop Formation
Cleavage and Reading Frame Shift
Conclusions
ACHIEVEMENTS
Model
Index
Conclusions
References
Downloads
Conclusions
In this Modeling task, we have developed a mathematical approach of the CRISPR/Cas9 mechanism and its performance in our Testing System. Our model has a set of ODEs representing Cas9:gRNA complex formation, a thermodynamic balance for the R-loop formation, a sequence-alignment algorithm to search off-targets and probabilities estimation to know the reading frame shift after the Cas9 cleavage.
With this in silico model, we have been able to study the behavior of the system under different conditions. For instance, proposals based on the modeling were implemented in wet lab, with some successful results. In particular we determined optimum times of agroinfiltration, and redefined lab protocols for genetic constructions design, as we described below..
Regarding the Cas9:gRNA complex formation, it has been observed in simulations that the fast dynamics of the gRNA causes its fast disappearance. In other words, the gRNA is the limiting reagent. The usual lab protocol considers the simultaneous agroinfiltration of the gRNA and the Cas9. However, in simulations we observed that the production of the complex could be maximized if Cas9 was agroinfiltrated in first place. This is because after 3 days approx. there will be enough Cas9 in the nucleus to sense small quantities of gRNA. This alternative protocol would make the system less susceptible to lower efficiency in the transcription mechanisms of the cell.
The three-dimension diffusion of the complex until it finds the target, describes an aleatory pathway. This implies more variability within the results of the Testing System, making it less robust and reducing its repeatability. In order to reduce this negative impact, we suggested introducing the gRNA and the Testing System constructions one next to the other. Thus, the probability that the complex “randomly” finds the target is increased and there is more control of intrinsic variability sources affecting the result. Wet lab experiments performed under these conditions showed favorable results when the gRNA was agroinfiltrated near to the target.
The analysis of the thermodynamic balance which regulates the R-loop formation, let us estimate that single mismatches positioned downstream the 11th-10 th nucleotide of the gRNA, lead to a positive free energy increment, i.e. they make unable the R-loop between that gRNA and the DNA target. This verifies bibliographic criteria about efficient design of gRNAs.
Another critical point is the chromatin state, which cannot be determined instantaneously nor continuously. We have tried to avoid this lack of information relating the DNA packaging with transcriptional activity. More information about this factor is necessary, letting us further study its relation with the CRISPR/Cas9 efficacy and efficiency.
In conclusion, though lack of data of our testing system under the simulated conditions has been an obstacle for the model validation, the model gives sensible semi-quantitative results. The large amount of time required when working with plants, has limited the quickly generation of data and test of hypothesis extracted from modeling results.  Further experimental results in the future will allow to fully validate the model. Nevertheless, results obtained have helped us to determine critical steps of the process, as well as to redesign laboratory protocols.
References
1. Bae S, Park J, Kim J. Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases. Bioinformatics. 2014;30(10):1473-1475.
2. Farasat ISalis H. A Biophysical Model of CRISPR/Cas9 Activity for Rational Design of Genome Editing and Gene Regulation. PLOS Computational Biology. 2016;12(1):e1004724.
3. Chattopadhyay J, Tapaswi P, Mukherjee D. Formation of a regular dissipative structure: a bifurcation and non-linear analysis. Biosystems. 1992;26(4):211-222.
4. Nakabayashi JSasaki A. A mathematical model of the intracellular replication and within host evolution of hepatitis type B virus: Understanding the long time course of chronic hepatitis. Journal of Theoretical Biology. 2011;269(1):318-329.
5. Kalinin MKononogov S. Boltzmann’s Constant, the Energy Meaning of Temperature, and Thermodynamic Irreversibility. Measurement Techniques. 2005;48(7):632-636.
6. Dill K, Ghosh K, Schmit J. Physical limits of cells and proteomes. Proceedings of the National Academy of Sciences. 2011;108(44):17876-17882.
7. Nishimasu H, Ran F, Hsu P, Konermann S, Shehata S, Dohmae N et al. Crystal Structure of Cas9 in Complex with Guide RNA and Target DNA. Cell. 2014;156(5):935-949.
8. SantaLucia, J, Allawi H, Seneviratne P. Improved Nearest-Neighbor Parameters for Predicting DNA Duplex Stability †. Biochemistry. 1996;35(11):3555-3562.
9. Watkins N, Kennelly W, Tsay M, Tuin A, Swenson L, Lee H et al. Thermodynamic contributions of single internal rA⋅dA, rC⋅dC, rG⋅dG and rU⋅ dT mismatches in RNA/DNA duplexes. Nucleic Acids Research. 2010;39(5):1894-1902.
10. Sugimoto N, Nakano M, Nakano S. Thermodynamics−Structure Relationship of Single Mismatches in RNA/DNA Duplexes †. Biochemistry. 2000;39(37):11270-11281.
11. Sugimoto N, Nakano S, Katoh M, Matsumura A, Nakamuta H, Ohmichi T et al. Thermodynamic Parameters To Predict Stability of RNA/DNA Hybrid Duplexes. Biochemistry. 1995;34(35):11211-11216.
12. Deng S, Stein R, Higgins N. Organization of supercoil domains and their reorganization by transcription. Molecular Microbiology. 2005;57(6):1511-1521.
13. Adrian J, Farrona S, Reimer J, Albani M, Coupland G, Turck F. cis-Regulatory Elements and Chromatin State Coordinately Control Temporal and Spatial Expression of FLOWERING LOCUS T in Arabidopsis. THE PLANT CELL ONLINE. 2010;22(5):1425-1440.
14. Ma X, Zhang Q, Zhu Q, Liu W, Chen Y, Qiu R et al. A Robust CRISPR/Cas9 System for Convenient, High-Efficiency Multiplex Genome Editing in Monocot and Dicot Plants. Molecular Plant. 2015;8(8):1274-1284.
15. Rutkauskas M, Sinkunas T, Songailiene I, Tikhomirova M, Siksnys V, Seidel R. Directional R-Loop Formation by the CRISPR-Cas Surveillance Complex Cascade Provides Efficient Off-Target Site Rejection. Cell Reports. 2015;10(9):1534-1543.
16. Moore R, Spinhirne A, Lai M, Preisser S, Li Y, Kang T et al. CRISPR-based self-cleaving mechanism for controllable gene delivery in human cells. Nucleic Acids Research. 2014;43(2):1297-1303.
Downloads
Mathematical discussion about Cas9 and gRNA complex formation
Energy_exchange
Energy_PAM
Nboffsearch
weight_exchange
Follow                                 Us
Sponsors
