<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Valencia_UPV_ORF skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Valencia UPV/ORF</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header" class="sticky dark header-md noborder clearfix fixed"><HEADER id="topNav"><DIV class="container"><DIV class="navbar-collapse pull-right nav-main-collapse collapse submenu-dark"><NAV class="nav-main"><UL id="topMain" class="nav nav-pills nav-main"><LI><A href="https://2016.igem.org/Team:Valencia_UPV"> HOME </A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> TEAM </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Team">MEET US</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Attributions">ATTRIBUTIONS</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="#"> PROJECT </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Description">DESCRIPTION</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Software">DATA PROCESSING SOFTWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Proof" style="text-transform: none;">SPLIT Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Notebook">NOTEBOOK</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> LAB WARE </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware">HARDWARE</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Reagents">Reagents</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Hardware/Manual">Manual</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> HUMAN PRACTICES </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Silver">SOCIAL ENGAGEMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/HP/Gold">SOLVING REAL PROBLEMS</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Integrated_Practices">IMPACT ASSESSMENT</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Safety">SAFETY</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> PARTS </A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Part_Collection">BEST PART COLLECTION</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="#"> MODEL</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Model">OVERVIEW</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/CP">Cas9:gRNA Complex Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Loop">R-Loop Formation</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/ORF">Cleavage and Reading Frame Shift</A></LI><LI><A href="https://2016.igem.org/Team:Valencia_UPV/Conclusions">Conclusions</A></LI><LI><A class="active" href="https://2016.igem.org/Team:Valencia_UPV/Achievements"> ACHIEVEMENTS</A></LI></NAV></DIV></DIV></HEADER></DIV><SECTION class="page-header page-header-lg parallax parallax-3" style="background-image:url('https://static.igem.org/mediawiki/2016/b/b2/T--Valencia_UPV--modelTitleBack.png');"><DIV class="container"><H1>Cleavage and Reading Frame Shift</H1></DIV></SECTION><SECTION><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-2 col-sm-3"><DIV class="side-nav margin-bottom-60 margin-top-30"><DIV class="side-nav-head"><H4>Index</H4></DIV><UL class="list-group list-group-bordered list-group-noicon uppercase"><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/ORF#Overview._id">Overview</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/ORF#Knockoutsestimation_id">Knockouts
                                estimation</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/ORF#ORFsprobabilitydistribution._id">ORFs
                                probability distribution</A></LI><LI class="list-group-item"><A href="https://2016.igem.org/Team:Valencia_UPV/ORF#Mainremarks_id">Main
                                remarks</A></LI></UL></DIV></DIV><DIV class="col-md-10 col-sm-9"><DIV class="blog-post-item" id="Overview._id"><H3>Overview</H3><P>If the R-loop is completely established, then the
                        complex Cas9:gRNA stays bounded to the DNA strand and
                        Cas9 will cut the DNA sequence. This will lead in the
                        <B>loss</B> of some <B>base pairs</B> that the plant cell will <B>repair
                        via Non Homologous End Joining</B> (NHEJ). Consequently, this loss may cause a <B>change</B>
                        in the <B>reading frame</B> (TS<SUB>OFF</SUB> â†’
                        TS<SUB>ON</SUB>), and in the <B>proteins</B>
                        produced by the cell from that moment, performing what is known as <B>knockout</B>. </P><P>
                        Taking profit from this shift in
                        the DNA reading frame, we included <B>Luciferase</B> after the
                        genetic target. Therefore, <B>light signal</B> emitted during Luciferase
                        assays, will provide an <B>output</B> information about the
                        whole process performed by CRISPR/Cas9 within the plant
                        cells. As it relies on the target finding by the
                        Cas9:gRNA complex, it is also a <B>measure</B> of the pair
                        <B>gRNA-target efficiency</B> in order to knockout the
                        undesired gene.</P></DIV><DIV class="blog-post-item" id="Knockoutsestimation_id"><H3>Knockouts estimation</H3><P>
                        While the Cas9:gRNA complex is produced, it produces
                        knockouts among the host genome. In our case, these
                        <B>knockouts are expected to improve the desired plant
                        trait</B>. In this section of the modeling, we wanted to
                        study <I>in silico</I> if the suggested pair target-gRNA would
                        work <I> in vivo</I>, obtaining a predicted number of knockouts
                        performed in the Testing System construction for the
                        time at which the Luciferase assay is performed.
                        The number of knockouts produced at <B>measurement time t
                        <SUB>m</SUB></B> will be:</P><P>Where:</P><UL><LI>The concentration [Cas9:gRNA] (t) is the amount
                            of <B>complex produced</B> at the time when the luciferase
                            assay is performed. This can be estimated using the
                            system of ODEs described in the previous section
                            Cas9:gRNA complex formation.</LI><LI>The term N<SUB>ON</SUB> makes reference to the
                            quantity of <B>on-targets</B> in the nucleus, i.e. the
                            <B>gene copy number</B> of the <B>Testing System construction</B>
                            which has been agroinfiltrated.</LI><LI>The rate k<SUB>cleavage</SUB> has information
                            about the <B>cleavage frequency</B> in the Testing System
                            construction. It has information about the complex
                            diffusion, the rate of Cas9 cleavage, and the
                            thermodynamic stability of the R-loop formed before
                            the cleavage. It is:</LI></UL><P>
                        The rates in last expression mean:</P><UL><LI>The rate of <B>collisions k<SUB>r</SUB></B>, produced
                            between the <B>Cas9:gRNA</B> complex and the <B>DNA</B> in the
                            nucleus.</LI><LI>The rate of cleavage k<SUB>c</SUB> at which
                            <B>Cas9 cuts DNA</B> strands.</LI><LI>The occurrence of the <B>Cas9:gRNA dissociation</B><B>from</B> a <B>DNA</B> region.</LI><LI>The probability that the <B>R-loop</B> is formed
                            between the <B>complex and the target</B> used in the
                            Testing System construction. This parameter has
                            also information about potential off targets.</LI></UL><P>Values for parameters k<SUB>r</SUB> and
                        P<SUB>complex,target</SUB> are related to diffusion and
                        thermodynamics, respectively.</P></DIV><DIV class="blog-post-item" id="ORFsprobabilitydistribution._id"><H3>ORFs probability distribution</H3><P>From the moment that Cas9:gRNA complex is produced,
                        it starts <B>diffusing, colliding</B> and <B>binding</B> to
                        on-targets (ONs) and off-targets (OFFs). Those have
                        known initial concentrations, since the number of
                        on-targets is the gene copy number of the Testing
                        System construction, and the number of off-targets has
                        been obtained by our off-target search algorithm.
                        After the complex interacts with a target, it changes
                        it sequence. The cut performed by Cas9 takes place in
                        the <B>PAM-proximal region</B>, avoiding the possibility of a
                        new zip-union with the gRNA to form the R-loop again.
                        Therefore, knockouts cannot be represented as off
                        targets or on targets anymore, as binding to the gRNA
                        will not be thermodynamically feasible because of base
                        pairs rearrangement.</P><P>In our Testing System strategy, we rely on the reading
                        frame shift to let Luciferase be transcribed. However,
                        this event will not always be likely to happen. There
                        are <B>three possible reading frames</B> and the resulting one
                        after the Cas9 cut, is unknown. This created the necessity of estimating the
                        probability that the resulting frame after the Cas9
                        cleavage and NHEJ reparation, was the correct for
                        Luciferase transcription.</P><P>
                        Consulting bibliography about
                        <B>frequency of indels</B> in <I>Nicotiana benthamiana</I>, we
                        could gather information about this phenomenon. Results
                        showed <B>barely any difference</B> between the three possible
                        results (ORF+1, ORF+2 and ORF+3). Therefore, the
                        probability that our Testing System would express
                        Luciferase was P<SUB>ON</SUB>â‰ˆ1/3.</P><DIV style="text-align:center;"><P class="imgFooterP" style="text-align: center;font-style: italic;">Figure 14. Estimation
                            of probability associated to each one of the
                            possible resulting reading frames after Cas9
                            knockout. Blue bars represent experiments performed only with Nicotiana, while orange bars depict results obtained working with various plants.</P></DIV></DIV><DIV class="blog-post-item" id="Mainremarks_id"><H3>Main remarks</H3><P>Analysis of bibliographic data determined that the
                        probability that our Testing System would express
                        Luciferase was P<SUB>ON</SUB>â‰ˆ1/3. Further experimental
                        results in the future will allow to fully validate the
                        model.</P></DIV></DIV></DIV></DIV></SECTION><FOOTER id="footer" style="margin-bottom:-20px"><DIV class="container"><DIV class="row"><DIV class="col-md-5"><DIV class="row"><DIV class="col-md-6 col-sm-6 hidden-xs" style="padding:0;"><DIV class="margin-left-50 clearfix"><H4 class="margin-bottom-10" style="text-align:start;">Follow
                                Us</H4></DIV></DIV></DIV></DIV><DIV class="col-md-7"><H4 class="letter-spacing-1" style="text-align: start;">Sponsors</H4></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>