<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WITS-CSIR_SA_Project_Concept"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WITS-CSIR_SA/Project/Concept">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WITS-CSIR_SA/Project/Concept&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WITS-CSIR_SA/Project/Concept&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WITS-CSIR_SA/Project/Concept&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WITS-CSIR_SA/Project/Concept" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WITS-CSIR SA/Project/Concept</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><A href="/wiki/images/c/c8/Example.ogg" class="internal" title="Example.ogg">Media:Example.ogg</A>&lt;!DOCTYPE html PUBLIC &quot;-//W3C//DTD XHTML 1.0 Transitional//EN&quot; &quot;<A href="http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd" class="external free" rel="nofollow">http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd</A>&quot;&gt;<TITLE>Biotweet - Collaboration</TITLE></P><DIV data-dojo-type="dijit.layout.BorderContainer" data-dojo-props="design: 'headline', gutters: false" class="container"><DIV style="position: absolute; width: 100%; top: 80px;"><UL class="dropdown dropdown-horizontal"><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA">Home</A></LI><LI><A href="#" class="dir">Project</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Project/Concept">Overview</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Characterization">Characterization</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Project/Modelling">Modelling</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Collaboration/Index">Collaboration</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Project/Notebook">Lab notebook</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Project/Applications">Potential applications</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Project/Achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Project/Protocols">Protocols</A></LI></UL><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Parts">Parts submitted</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Characterization">Data</A></LI><LI><A href="./" class="dir">Outreach</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Outreach/SciBono">Scibono experience</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Outreach/Forum">Synthetic biology forum</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Outreach/Survey">Survey</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Project/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Diary/Gallery">Gallery</A></LI><LI><A href="./" class="dir">About us</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/AboutUs/MeetTheTeam">Meet the team</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/Outreach/PublicRelations">Media attention</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/AboutUs/Attributions">Attributions</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/AboutUs/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/AboutUs/MeetOurBugs">Meet our bugs!</A></LI><LI><A href="https://2011.igem.org/Team:WITS-CSIR_SA/AboutUs/ContactUs">Contact us</A></LI></DIV><DIV data-dojo-type="dijit.layout.ContentPane" data-dojo-props="region: 'center'" class="center"><DIV style="width: 903px; margin: auto; height: 100%;"><DIV data-dojo-type="dijit.layout.BorderContainer" data-dojo-props="design: 'headline'" style="height: 100%;"><DIV data-dojo-type="dijit.layout.ContentPane" data-dojo-props="region: 'center'"><DIV style="height: 100%; overflow-y: auto;"><H1>Overview</H1><DIV data-dojo-type="dijit.TitlePane" data-dojo-props="title:'Can bacterial chemotaxis be reprogrammed for the redirecting of bacterial motility?'"><P>Analogous to the transfer of data as electronic packets in a computer network, the concept of engineering bacteria to transport packets of chemical signals between bacterial populations arose. In this way, the bacteria themselves can form network connections. 

In order to aspire towards this ultimate objective, we decided to focus on reprogramming bacterial motility. Each bacterium will travel in a stimulus-directed fashion to specific points of the network (the nodes), where they can communicate signals. Such a stimulus should be unique to the bacterium and not be part of the natural environment. With these specifications in mind, we needed a way of manipulating the chemotactic behaviour of bacteria. 

During chemotactic movement, the expression of CheZ induces counter-clockwise (CCW) flagellar activity - resulting in a “run and tumble” bacterial motion (Figure 1). For more on bacterial chemotaxis, <A href="https://2011.igem.org/Team:WITS-CSIR_SA/Project/Motility">see here</A>. 
</P><TABLE border="0" cellpadding="10" cellspacing="10"><TBODY><TR><TD><P><B>Figure 1.</B> The reprogramming of bacterial chemotactic response by using post-transcriptional activation of CheZ using an analyte-responsive riboswitch. CheZ expression results in counter-clockwise flagellar rotation and a “run and tumble” movement. (Click to enlarge)
</P></TD></TR></TBODY></TABLE></DIV><DIV data-dojo-type="dijit.TitlePane" data-dojo-props="title:'Project “Biotweet” – directed bacterial movement as a networking framework '"><P>The WITS iGEM 2011 team focused heavily on the key idea of guiding E.coli to move up a concentration gradient of a specific exogenous analyte – through controlled expression of this motility factor, CheZ. Our ambitious goal was to show that we can reprogram bacterial chemotaxis and thereby transfer products of any coding sequence to a desired location. 

 “Biotweet” is a theoretical concept of a framework for bacterial networks, using reengineered bacterial motility. Any network requires the establishment of connections between nodes of the network, allowing for the transport of node-to-node data packets. Reengineering the mechanisms of chemotaxis is the first step in the assembly of such a bacterial communication system. With “Biotweet”, we have attempted to construct such a network within a microbiological setting: a challenging task, considering that bacterial cells cannot be linked by wires.  
</P></DIV><DIV data-dojo-type="dijit.TitlePane" data-dojo-props="title:'How can the chemotaxis motility factor CheZ be exploited to reprogramme bacterial movement?'"><P>Riboswitches are ligand-inducible RNA protein expression regulators that are comprised of an aptamer domain and an expression platform (Gallivan, 2007). The aptamer is a sequence of nucleotides that is designed to specifically bind to ligands, while the expression platform consists of a ribosome binding site (RBS) and a downstream coding region (CDS). When the specific ligand or analyte binds to the aptamer domain, the riboswitch undergoes a structural change. This results in the exposure of the RBS and the expression of the downstream CDS (Figure 2, top left panel). 

By linking CheZ expression to post-transcriptional activation by a unique analyte (using an analyte-specific riboswitch), bacterial motility can be guided towards a specific stimulus. We chose to use artificial riboswitches to control the expression of the flagella rotation regulator protein CheZ. Using this approach, the expression of CheZ can be regulated in a ligand concentration-dependent manner, outside the control of the natural chemotaxis pathway (Figure 2, right panel).

We have decided to include two fluorescent reporters as fusion proteins (both N- and C-terminal CheZ fusions are well-tolerated, according to Gallivan, 2007). This allowed us to observe activation of the riboswitches in the context of our experiments.

We borrowed heavily from studies which have described riboswitch-controlled bacterial chemotaxis (Topps and Gallivan. 2007). Topps and Gallivan (2007) demonstrated that these riboswitches can be used to reprogram the chemotactic response, allowing bacteria to move up a concentration gradient of the analyte that activates the riboswitch (a process known as pseudotaxis). Bacteria can theoretically be reprogrammed to respond to any stimulus, if the appropriate riboswitch is made. This provides the versatility needed for biological network connections to be established in any application. Furthermore, this eliminates the need to engineer novel chemoreceptors.
</P><TABLE border="0" cellpadding="10" cellspacing="10"><TBODY><TR><TD><P><B>Figure 2. </B>Using ligand-responsive riboswitch activation of CheZ expression. CheZ can be fused to a reporter and direct bacterial motility in a ligand-dependent manner up a concentration gradient. Note: Fluorescent reporter proteins were fused to CheZ for observation of riboswitch activation during experimentation. (Click to enlarge)
</P></TD></TR></TBODY></TABLE></DIV><DIV data-dojo-type="dijit.TitlePane" data-dojo-props="title:'Riboswitch-mediated chemotaxis reprogramming'"><P>Using analyte-induced CheZ expression, the main objective of project “Biotweet” is to illustrate the ability to reengineer bacterial chemotactic behaviour through the activation of riboswitches. Construction of such a mechanism is done in a modular fashion, producing an opportunity for scientists to not only direct bacterial movement to a specific location, but additionally, the prospect of delivering products of any protein-encoding sequences to that same site. This can be done by fusing any translational unit to the CheZ sequence in the riboswitch.</P><TABLE border="0" cellpadding="10" cellspacing="10"><TBODY><TR><TD><P><B>Figure 3.</B>WITS iGEM 2011 riboswitch design. The riboswitch is linked to the motility activator CheZ protein, which can in turn be fused to any coding sequence. (Click to enlarge)</P></TD></TR></TBODY></TABLE></DIV><DIV data-dojo-type="dijit.TitlePane" data-dojo-props="title:'Bacteria function as the nodes in the “Biotweet” network – Programming an idealized “Send and Receive” motility module in &lt;i&gt;E.coli&lt;/i&gt;'"><P>The aim was to reprogram <I>E. coli</I> CheZ deletion mutants (which lack natural CheZ functionality) with riboswitch-activated CheZ – to search, localise to the riboswitch activator, and return to a set location in order to report. This, in principle, will show that bacteria can be programmed to move in a directed manner towards a new stimulus and establish a connection between two points in space. The generic design for this is depicted in Figure 4. The “send” module is essentially a riboswitch-activated CheZ expression system which is responsive to analyte A. The “receive” module works via the same expression principles, except that the riboswitch is responsive to analyte B (Figure 4).

To ensure that the “send” and “receive” modules act independently, analytes A and B need to function as riboswitch activators. Additionally, analyte A must also function as a de-repressor of the Repressor X, within a biphasic toggle (Figure 4). Repressor X will repress Cre-recombinase (as well as Repressor Y) expression and the toggle will only flip at a critical concentration of Analyte A. A constitutive promoter, flanked by lox sites, will control transcription of the riboswitches – which exist in opposite transcriptional orientation within a single construct. Cre recombination will allow for irreversible “flipping” of this constitutive promoter (by lox site recombination) and thereby switch the motility function (or “direction”) from “send” to “receive”, such that bacterial motility will then be dictated by a concentration gradient of analyte B (Figure 4). Ultimately, the bacterial chemotactic movement will switch from responsiveness to analyte A (“send”) to responsiveness to analyte B (“receive”), when the bacteria reach a critical concentration of analyte A.</P><TABLE border="0" cellpadding="10" cellspacing="10"><TBODY><TR><TD><P><B>Figure 4.</B> “Send” and “receive” riboswitch-controlled motility module. Featured is also an analyte A-responsive toggle switch which activates Cre-recombinase expression. Cre, in turn, causes “receive” motility activation by flipping a constitutively active promoter, and allowing motility to be controlled by analyte B. (Click to enlarge)
</P></TD></TR></TBODY></TABLE><P>Unfortunately, other than Tetracycline (Dox) riboswitches in yeast, at present, an engineered riboswitch which is sensitive to a known transcriptional de-repressor does not exist. For this reason, we could not put our theoretical (ideal) genetic circuit design into practice, since no analyte in literature can act as both a riboswitch activator and de-repressor in a biphasic toggle system. Therefore, we designed a system which allows the “receive” function (expression of Cre-recombinase) to rather be activated by IPTG (lacI) – see Figure 5. Also, we focused this design on the use of two known synthetic riboswitches which are responsive to the analytes theophylline and atrazine, respectively (see “Riboswitch Design Considerations” below).</P><TABLE border="0" cellpadding="10" cellspacing="10"><TBODY><TR><TD><B>Figure 5.</B> “Send” and “receive” riboswitch-controlled motility module. In this design, Cre-recombinase is activated by IPTG which is present only at the theophylline position (not present in the theophylline gradient - blue). As with Fig. 4, Cre, in turn, causes “receive” motility activation by flipping the constitutively active promoter, and allowing motility to be controlled by an Atrazine gradient (red). Note: analogous to the design above in Fig.4 – theophylline can be considered as analyte A, and atrazine as analyte B. (Click to enlarge)
</TD></TR></TBODY></TABLE></DIV><DIV data-dojo-type="dijit.TitlePane" data-dojo-props="title:'What riboswitch designs are currently available?'"><P>Our initial aim was to develop a theophylline-sensitive riboswitch which is capable of regulating CheZ (motility factor) translation. 

Using a high-throughput screen (Lynch et al., 2007) and a high-throughput selection (Topp and Gallivan, 2008) in E.coli, the Gallivan lab (Emory, USA) has been able to develop powerful theophylline-dependent riboswitches. It is worth noting that the motility-based screen (Topp and Gallivan, 2008) made use of a theophylline riboswitch library activating the expression of CheZ. Activation of the motility factor CheZ allows bacteria to move along the theophylline gradient. The best riboswitch identified in these earlier screens was the theophylline riboswitch clone 8.1, which displays 36-fold expression activation ratio (Figure 6).
</P><TABLE border="0" cellpadding="10" cellspacing="10"><TBODY><TR><TD><P><B>Figure 6. </B>Theophylline responsive riboswitch (clone 8.1). We have referred to this riboswitch as theophylline riboswitch 1. (Click to enlarge)
</P></TD></TR></TBODY></TABLE><P>In order to improve on their earlier riboswitches, Topp and Gallivan later developed an in vivo-based bacterial selection strategy based on fluorescence activated cell sorting (FACS) (Wieland et al., 2008; Lynch et al., 2009). By randomizing the sequence encoding the RBS (+ Shine Delgarno/SD), and by sorting a N12 library, clone 12.1 – which displays a 96-fold expression activation in the presence of theophylline (1 mM) – was identified (Figure 7). This is the largest activation ratio to date, for synthetic or natural riboswitches. This riboswitch contains a strong (and longer) UAAGG SD which is found in a tight (inaccessible) duplex in the “OFF” (i.e. no theophylline) state. Moreover, the UAAGG sequence is spaced optimally, with 6 nt separating the 5’ A of the anti-SD sequence and the start codon of the translation (AUG) (Figure 7).</P><TABLE border="0" cellpadding="10" cellspacing="10"><TBODY><TR><TD><P><B>Figure 7.</B>Theophylline responsive riboswitch (clone 12.1). We have referred to this riboswitch as theophylline riboswitch 2. (Click to enlarge)
</P></TD></TR></TBODY></TABLE><P>The atrazine riboswitch was developed recently by Sinha et al (2010). In order to make this riboswitch compatible with biobrick assembly, Pst1 site was removed by creating a G:U wobble base pair in lower stem region (Figure 8). </P><TABLE border="0" cellpadding="10" cellspacing="10"><TBODY><TR><TD><P><B>Figure 8. </B>Atrazine responsive riboswitch. We have referred to this riboswitch as atrazine riboswitch (atRS) (Sinha et al., 2010). (Click to enlarge)
</P></TD></TR></TBODY></TABLE></DIV><DIV data-dojo-type="dijit.TitlePane" data-dojo-props="title:'Can we improve on previous theophylline-responsive riboswitches developed by past iGEM teams?'"><P>We have identified that several theophylline riboswitches have been previously described by other iGEM teams. (For reference, see Lethbridge 2007, Lethbridge 2009 and NYMU Taipei 2010). One of our initial concerns was whether it was possible to isolate the riboswitch from the associated translational unit (coding region). Being able to do so would allow the riboswitch to be defined independently as a BioBrick, with the useful ability to act in a modular fashion for fusion to any adjacent translational unit. All previous teams have made this assumption. Our aim is to show that the riboswitch and its adjacent coding region should be considered together and should be cloned as one biobrick, or an alternative &quot;acanonical&quot; assembly standard should be used.</P><P>Team NYMU Taipei 2010 assumed that a created theophylline riboswitch (Gallivan lab clone 8.1) biobrick could be fused to a GFP coding region by standard biobrick assembly techniques. While it is possible to do this, we feel that the resulting 6 nt scar sequence insertion increases the distance between the RBS and the AUG start codon, a result which is likely to cause a decrease in riboswitch efficiency (see riboswitch construction in Figure 9:  inserted biobrick “scar” region highlighted in turquoise; AUG start underlined). In fact, Taipei did show data which showed only a 2.5 fold increase in gene activation at approx. 4 mM theophylline. Compared to the 36-fold activation at 1mM theophylline by Gallivan lab, this seems quite low. We therefore feel that this riboswitch is not behaving efficiently.</P><P><B>Figure 9.</B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K411001">BBa_K411001</A>,<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K411003">BBa_K411003</A> - Team NYMU-Taipei iGEM 2010. Team NYMU Taipei assumed that a created theophylline riboswitch (Gallivan lab clone 8.1) biobrick could be fused to a GFP coding region by standard biobrick assembly techniques. The resulting 6 nt scar sequence insertion increases the distance between the RBS and the AUG start codon, a result which is likely to cause a decrease in riboswitch efficiency. Inserted biobrick “scar” region highlighted in violet, AUG start underlined. </P><P>Team Lethbridge 2009, used the same theophylline riboswitch (clone 8.1) to generate a riboswitch-CheZ construct. However, this team created a 11 nt insert after the AUG start codon. This result created a UAG amber stop codon (in the 3rd amino acid position) as well as a frameshift mutation. Therefore it is unlikely that a functional CheZ could have been produced from this system. (see team Lethbridge 2009 below; Inserted region highlighted in violet, AUG start underlined) (Click to enlarge) </P><TABLE border="0" cellpadding="10" cellspacing="10"><TBODY><TR><TD><P><B>Figure 10.</B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K249028">BBa_K249028</A>, Team Lethbridge iGEM 2009. An 11 nt insertion after the AUG start codon results in a UAG amber stop codon (in the 3rd amino acid position) as well as a frameshift mutation. (Inserted region highlighted in violet, AUG start underlined) (Click to enlarge)
  </P></TD></TR></TBODY></TABLE><P>Due to the difficulties faced with previously submitted riboswitches we decided to make our own new BioBricks. An objective of our project was to show higher efficiency of activation and make these constructs available for future teams to use in a modular fashion. This way, any coding sequence can be controlled in a riboswitch-dependent manner for function in an unlimited number of applications (Figure 11). </P><TABLE border="0" cellpadding="10" cellspacing="10"><TBODY><TR><TD><P><B>Figure 11. </B>Previously submitted riboswitches versus WITS iGEM 2011 riboswitches. (Click to enlarge)
</P></TD></TR></TBODY></TABLE></DIV><DIV data-dojo-type="dijit.TitlePane" data-dojo-props="title:'References'"><P>Gallivan JP. Toward reprogramming bacteria with small molecules and RNA.CurrOpinChemBiol 2007;11:612-9</P><P>Topp S, Gallivan JP. Guiding bacteria with small molecules and RNA. J Am ChemSoc 2007;129:6807-11.</P><P>Lynch S.A. Desai S.K.,Sajja,H.K., and Gallivan J.P.A high-trhoughput screen for synthetic riboswitches reveals mechanistic insight into their function. 2007, ChemBiol 14:173-184</P><P>Topp S. and Gallivan J.P. Random walks to synthetic riboswitches – a high throughput selection based on cell motility. 2008, ChemBiochem 9:210-213</P><P>Wieland M., and Hartig, J.S. Improved aptazyme design and in vivo screening enable riboswitches in bacteria. 2008, Agnew.Chem.Iny.Ed.Engl 47:2604-2607</P><P>Lynch S.A. and Gallivan J.P., A flow cytometry based screen for synthetic riboswitches. 2009, Nucleic Acids Res. 37(1)184-192</P><P>Sinha J, Reyes SJ, Gallivan JP.Reprogramming bacteria to seek and destroy an herbicide.Nat Chem Biol. 2010 Jun;6(6):464-70. Epub 2010 May 9.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:WITS-CSIR_SA/Project/Concept">http://2011.igem.org/Team:WITS-CSIR_SA/Project/Concept</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WITS-CSIR_SA/Project/Concept" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WITS-CSIR_SA/Project/Concept" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WITS-CSIR_SA/Project/Concept&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WITS-CSIR_SA/Project/Concept&amp;oldid=257284" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>