<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="up_white_container"><DIV id="over_the_header_button"><DIV class="color"><A href="https://2011.igem.org/Team:Freiburg/Contact"> Contact </A></DIV></DIV></DIV><DIV id="up_line"><DIV id="up_line_header"><DIV style="float:right; position: absolute; bottom: 15px; right: 100px; padding-bottom:2px;">This is the wiki page of the Freiburger student team competing for iGEM 2011. Thank you for your interest! </DIV></DIV></DIV><DIV id="my_content"><DIV id="nav-body" style="width: 985px; float:left"><DIV id="navigation" style="width: 150px; float:left"><UL><LI><A id="home" href="https://2011.igem.org/Team:Freiburg" display="block"><SPAN class="blueSelect">&lt;</SPAN>home<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI><LI><A id="team" href="https://2011.igem.org/Team:Freiburg/Team" display="block"><SPAN class="blueSelect">&lt;</SPAN>team<SPAN class="blueSelect"> &gt; </SPAN></A></LI><LI id="members" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Members"><SPAN class="blue">→</SPAN>members</A></LI><LI id="attributions" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Attributions"><SPAN class="blue">→</SPAN>attributions</A></LI><LI id="freiburg" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Freiburg"><SPAN class="blue">→</SPAN>freiburg</A></LI></UL><LI><A id="project" href="https://2011.igem.org/Team:Freiburg/Project" display="block"><SPAN class="blueSelect">&lt;</SPAN>project<SPAN class="blueSelect"> &gt; </SPAN></A></LI><LI id="description" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Description"><SPAN class="blue">→</SPAN>description</A></LI><LI id="results" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Results"><SPAN class="blue">→</SPAN>results</A></LI><LI id="modelling" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Modelling"><SPAN class="blue">→</SPAN>modeling</A></LI><LI id="notebook" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Notebook"><SPAN class="blue">→</SPAN>notebook</A></LI><LI id="sampledata" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/SampleData"><SPAN class="blue">→</SPAN>sample data</A></LI><LI><A id="humanpractices" href="https://2011.igem.org/Team:Freiburg/Oath"><SPAN class="blueSelect">&lt;</SPAN>human practices<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI><LI id="oath" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Oath"><SPAN class="blue">→</SPAN>oath</A></LI><LI id="socioeconomicconsiderations" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/SocioeconomicConsiderations"><SPAN class="blue">→</SPAN>ethics</A></LI><LI id="safety" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Safety"><SPAN class="blue">→</SPAN>safety</A></LI><LI><A id="gallery" href="https://2011.igem.org/Team:Freiburg/Gallery"><SPAN class="blueSelect">&lt;</SPAN>gallery<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI><LI><A id="sponsors" href="https://2011.igem.org/Team:Freiburg/Sponsors" display="block"><SPAN class="blueSelect">&lt;</SPAN>sponsors<SPAN class="blueSelect"> &gt; </SPAN></A></LI><LI><A id="synbio oath" href="https://2011.igem.org/forum/Topic_List?n=368&amp;s=1"><SPAN class="blueSelect">&lt;</SPAN>to the forums<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI></DIV><DIV id="noNavigationContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#results"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">results</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Precipitator"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Precipitator</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Usage_and_Biology"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Usage and Biology</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Design_Notes"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Design Notes</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Source"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Source</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Mathematical_modeling"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Mathematical modeling</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-7"><A href="#References"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Precipitator_fused_with_GST_tag"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Precipitator fused with GST tag</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Results_2"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Methods"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Usage_and_Biology_2"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Usage and Biology</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Further_reading"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">Further reading</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Plastic_binding_domain"><SPAN class="tocnumber">1.2.6</SPAN><SPAN class="toctext">Plastic binding domain</SPAN></A></LI><LI class="toclevel-4 tocsection-14"><A href="#Experimental_setup"><SPAN class="tocnumber">1.2.6.1</SPAN><SPAN class="toctext">Experimental setup</SPAN></A></LI><LI class="toclevel-4 tocsection-15"><A href="#Results_3"><SPAN class="tocnumber">1.2.6.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-4 tocsection-16"><A href="#Discussion"><SPAN class="tocnumber">1.2.6.3</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Green_light_receptor"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Green light receptor</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#red_light_receptor"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">red light receptor</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#blue_light_receptor"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">blue light receptor</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Lysis_cassette"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Lysis cassette</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Parts_submitted_to_the_registry"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Parts submitted to the registry</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Precipitator_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Precipitator</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Green_light_receptor_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Green light receptor</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Lysis_cassette_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Lysis cassette</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#commons"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">commons</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="results">results</SPAN></H1><H2><SPAN class="mw-headline" id="Precipitator"><SPAN style="color:grey;">Precipitator</SPAN></SPAN></H2><P><A href="http://partsregistry.org/Part:BBa_K608406" class="external text" rel="nofollow">BBa_K608406</A><B>Precipitator</B></P><P>
Protein domain of Precipitator. Artificial Leucine Rich Repeat(LRR) with C and N-terminal hagfish domain fragments capping the artifical middle part. This part is one version of three different designed to bind nickel by histidines,
grouped together pointing away from the horseshoe shaped protein.
</P><P>Bacterial LRR Consensus of the central LRR fragment:
</P><P>LxxLxLxxNxLxxLPxxLPxx
</P><P>Protein code:
</P><P>CPSRCSCSGTEIRCNSKGLTSVPTGIPSS
</P><P>ATRLELESNKLQSLPHGVFDK
</P><P>LTQLTKSNNHLHSLPDNLPAS
</P><P>LEVLDVSNNHLHSLPDNLPAS
</P><P>LEVLDVSNNHLHSLPDNLPAS
</P><P>LEVLDVSNNHLHSLPDNLPAS
</P><P>LEVLDVSNNHLHSLPDNLPAS
</P><P>LEVLDVSNNHLHSLPDNLPAS
</P><P>LKELALDTNQLKSVPDGIFDR
</P><P>LTSLQKIWLHTNPWDCSCPRIDY
</P><P>LSRWLNKNSQKEQGSAKCSGSGKPVRSIICP
</P><P>This protein can be used to complex Nickel or Cobalt. 
The principal mechanism is comparable to Ni-NTA columns, as chelates the ions. Free binding sites of the ions are then exposed, so that a His-tagged protein can attach to them.
The design of the protein is of a particular interest, too. LRR are highly conserved motifs throughout evolution. They appear in all kingdoms of life in almost every thinkable role (Ligases, Receptors, Toxins etc.). Their core is highly conserved and provides a very stable backbone, while the intermediate, non-conserved aminoacids are almost freely interchangeable. 
</P><P>
This protein can be used to complex up to 4 Nickel or Cobalt. However the principal design oft he protein is of a particular interest, too. LRR are highly conserved motifs throughout evolution. They appear in all kingdoms of life in almost every thinkable role (Ligases, Receptors, Toxins etc.). Their core is highly conserved and provides a very stable backbone, while the intermediate, non-conserved aminoacids are almost freely interchangeable. 
</P><P>We only submitted one of the three versions, to reduce redundancy in the registry. Please contact us for any questions.
</P><H3><SPAN class="mw-headline" id="Usage_and_Biology">Usage and Biology</SPAN></H3><P>The Precipitator is a new artificially designed LRR protein. It is meant as protein that binds Nickel ions with Histidines grouped on its surface. The bound Nickel can then precipitate His-tagged proteins. In our Lab in a Cell it should function as an adaptor between the plastic surface of pipettes and the His-tagged protein. Please look at our detailed description of the design layout in our modeling section. The sequence was synthesised and cloned into the iGEm vector. The submitted sequence was fully confirmed by sequencing.
</P><P><A href="http://partsregistry.org/Part:BBa_K608407" class="external text" rel="nofollow">BBa_K608407</A></P><H3><SPAN class="mw-headline" id="Design_Notes">Design Notes</SPAN></H3><P>Here we investigated an optimal set of non-conserved aminoacids by analysing large sets of similar proteins and databases. You can use this piece of work as a template to design your own protein and give it any function you like, by simply interchanging aminoacids and fusing other domains on the N or C termini.
To guarantee proper folding and to shield off the hydrophobic core, a well studied fragment of an LRR protein coming from hagfish was used. This technique was investigated before 
</P><H3><SPAN class="mw-headline" id="Source">Source</SPAN></H3><P>The part is constructed from the following protein sequences 
(PDB: 2z66) N-Terminal hagfish
(PDB: 2z62) C-Terminal hagfish
(PDB: 3cvr) bacterial ligase
</P><H3><SPAN class="mw-headline" id="Mathematical_modeling"><A href="https://2011.igem.org/Team:Freiburg/Modelling#A_mathematical_model_to_determine_the_experimental_design" class="external text" rel="nofollow">Mathematical modeling</A></SPAN></H3><P>To determine the Affinity k_D, experiments to find out the binding affinity of the plastic binding domain are necessary. To get a direct access to these values, we cloned the plastic binding domain in front of a GFP. Then, dilution and washing assays could be performed on polystyrene microtiter plates, red out by a fluorescence plate reader. 
The desired parameters could be calculated by measuring dilution rows of GFP proteins and measuring the fluorescence signals at the different concentrations. C_total could be determined by a dilution row with subsequent washing steps, to find out at what [P] concentration there is a saturation. See description of the plastic binding subproject for more detailed explanation on the experimental setup.
</P><P>A qualitative experiment to prove that Nickel is binding the Precipitator is sufficient, since k_2 &gt;&gt; 1 and does not play a significant role in our setup. This experiment could have been done using a nanofilter that blocks protein but let through ions. The Nickel concentration of the flow through can then be measured. 
</P><P>Alternatively purification of the Precipitator by fusing it with a GST-tag could be done, to subsequently measure the absorbance of the protein, before and after adding Nickel to the solution. After Jordan 1974 a detectable change in the absorbance should be detectable after the complex is formed. A similar effect – a colorshift from white to blue -  is visible when one prepares a Ni-NTA column. For this purpose we cloned the GST domain in front of the Precipitator. However towards the end of the project there was no more time to perfom these experiments.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>Tiandi Wei et.al.; &quot;LRRML: a conformational database and an XML description of
leucine-rich repeats (LRRs)&quot;; BMC Structural Biology 2008 doi: 10.1186/1472-6807-8-47 
</P><P>
Schmidt, Marc et.al.; &quot;Crucial role for human Toll-like receptor 4 in the development of contact allergy to nickel&quot;;
Nature 2010
</P><P>
Letter, J.E. et.al.; &quot;Complexing of Nickel( 11) by Cysteine, Tyrosine,
and Related Ligands and Evidence for Zwitterion Reactivity&quot;;
Contribution from the Department of Chemistry, University of Alberta,
Edmonton, Alberta T6G 2G2, Canada.
</P><P>
Kajava, A.V.;&quot;Structural Diversity of Leucine-rich Repeat Proteins&quot;
J. Mol. Biol. (1998) 277, 519±527
</P><P>
Kim, Ho Min et. al.; &quot;Crystal Structure of the TLR4-MD-2 Complex with Bound Endotoxin Antagonist Eritoran&quot;
DOI 10.1016/j.cell.2007.08.002
</P><H3><SPAN class="mw-headline" id="Precipitator_fused_with_GST_tag"><B>Precipitator fused with GST tag</B></SPAN></H3><P>The Precipitator was fused with the C-terminus of our GST tag in order to extract and further test the construct for its Nickel binding affinity. We successfully cloned it together using the Gibson Assembly and then subsequently pasted it into the iGEM vector. The submitted sequence was partially confirmed by sequencing.
</P><P><A href="http://partsregistry.org/Part:BBa_K608408" class="external text" rel="nofollow">BBa_K608408</A><B>GST-tag</B></P><P>The GST-tag was PCR amplified from a pGEX vector with overhang primers including the iGEM restriction sites and then pasted into the iGEM vector. To verify the functionality of the construct we cloned it before a GFP sequence and expressed it with an IPTG inducible vector. Results see partsregistry page. The submitted sequence was partially confirmed by sequencing.
</P><H3><SPAN class="mw-headline" id="Results_2">Results</SPAN></H3><P>The non-purified, sonificated supernatant of a 50ml E.coli culture pellet was resuspended in 7.5ml PBS. This sample delivered 319,4 µg/µl of protein detected with a Bradford assay. A second identical sample was GST-purified with a Glutathion-Sepharose pull down assay and resulted in 1ml of sample with a  protein concentration of 1,6 µg/µl. 
However both concentrations were a little above the straight calibration line values. Therefore the results are not perfectly quantified, but the numbers are still significant enough to provide a good proof of the functionality of the assay.
</P><P>Then the GFP fluorescence of 100µl of supernatant of both samples was measured with a plate reader:
</P><P>Non-purified: 180630
</P><P>GST-purified:  243011
</P><P>Thus the ratio of GFP absorbance / 1µg protein is:
</P><P>Non-purified: 5642
</P><P>GST-purified: 1432257
</P><P>If the values of the Bradford assay are assumed as correct, the purified sample has a 254x higher GFP concentration than the non-purified sample. Due to 3 washing steps it can be assumed that there is barely any other protein left except GFP.
Furthermore it can be estimated that only 17% of the total GFP amount present in the 50ml cell culture, could be captured using this assay.
</P><H3><SPAN class="mw-headline" id="Methods">Methods</SPAN></H3><P>Plate reader:
</P><P>The fluorescence intensity and protein concentration were measured with the
FLUOstar Omega, which is a multi-mode microplate reader.
</P><P>Samples were pipetted into the microplate and analyzed via the plate
reader. In this experiment we focused on the protein concentration and the
fluorescence intensity of GFP.
</P><P>Bradford-assay: A method to determine the total protein concentration.
</P><P>To analyze the protein concentration of the samples, Coomassie Brillant Blue
was pippeted to each sample. With the binding of the dye to the proteins the
color changes from dark red to blue. The more protein in the solution the
more Coomassie dye can bind to proteins and the more the color changes into
blue. The absorption of bound Coomassie dye is 595nm. The absorbance is
proportional with the amount of bound dye. With a series of Bovine Serum
Albumin (BSA) measurements the exact protein concentration of the samples
can be determined. BSA acts like a “marker” because the concentration of BSA
is known and with a linear calibration line the exact protein concentration can
be detected.
</P><H3><SPAN class="mw-headline" id="Usage_and_Biology_2">Usage and Biology</SPAN></H3><P><B>Prepare construct</B></P><UL><LI>Cloning: Clone the GST Biobrick part BBa_K608408 beforte the N-terminus of the protein you wish to purify. You may want to put a Glycine linker inbetween the two parts, for better expression and folding proterties of the fusion protein. DO NOT USE THE RFC10 STANDARD, but another, which allows fusion proteins. We recommend the Gibson Assembly, it worked conviently for us.
</LI></UL><UL><LI>Vector: You may want to use a vector especially designed for protein expression. The iGEM vectors are all high copy plasmids, which can lead to an overload of protein in your E. coli cells. This can result in toxic effects or inclusion bodies.
</LI></UL><UL><LI>Transformation: you may want to chose a E. coli strain especially designed for protein expression. It is helpful for a successfull protein purification to use a protease deficient strain such as BL21.
</LI></UL><P><B>Cell culture and Lysis</B></P><UL><LI>Grow cells overnight after transformation in 2ml medium.
</LI></UL><UL><LI>Inoculate 250mL LB in a 1l flask 
</LI></UL><UL><LI>Optional: induce with IPTG (500 µL) 
</LI></UL><UL><LI>centrifuging cells in 50ml falcon tubes 
</LI></UL><UL><LI>resuspend pellets in 7,5µl ice cold PBS
</LI></UL><UL><LI>sonificatie cells (4x 1 min with pause, maximal power, 1 pulse per second)
</LI></UL><UL><LI>Add 1,5X Protease inhibitor and 25µl of 10mM Lysozyme and let rotate at room temperature for 30min
</LI></UL><UL><LI>Centrifuge lysate for 10min at 500xg
</LI></UL><UL><LI>Preparation of glutathione sepharose beads. You may add 1X protease inhibitor to all used PBS.
</LI></UL><UL><LI>Gently shake the bottle of sepharose to resuspend the matrix.
</LI></UL><UL><LI>Use a pipet to remove sufficient slurry for use and transfer to an 15 ml falcon tube. (Dispense 1.33 ml of original sepharose slurry per ml of final volume required.)
</LI></UL><UL><LI>Sendiment the matrix by centrifugation at 500xg for 5 min. Carefully decant the supernatant.
</LI></UL><UL><LI>Wash the sepharose by adding 10 ml of cold 1xPBS per 1.33 ml of the original slurry of glutathione sepharose dispensed. Invert to mix. (Sepharose must be thoroughly washed with PBS to remove the 20% ethanol storage solution. Residual ethanol may interfere with subsequent procedures).
</LI></UL><UL><LI>Sediment the matrix by centrifugation at 500xg for 5 minutes. Decant the supernatant.
</LI></UL><UL><LI>For each 1.33 ml of the original slurry, add 1 ml of 1xPBS. This produces a 50% slurry. Mix well prior to the subsequent pipetting steps. (Sepharose 4B equilibrated with PBS may be stored at 4°C for up to a month. 
</LI></UL><P><B>Purification of fusion proteins</B></P><UL><LI>Transfer supernatant from step 2)h) on top of slurry from step 4)f)
</LI></UL><UL><LI>Rotate falcon for 30 min
</LI></UL><UL><LI>Centrifuge at 500xg for 5min
</LI></UL><UL><LI>Collect supernatant in a separate 50ml falcon ( in case the protein does not attach well to the beads, this supernatant can be used for a second purification)
</LI></UL><UL><LI>Add 10ml of cold PBS with protease inhibitor to flask with the beads
</LI></UL><UL><LI>Rotate falcon for 10 min
</LI></UL><UL><LI>Repeat steps c) - f)
</LI></UL><UL><LI>Centrifuge at 500xg for 5min
</LI></UL><UL><LI>Collect supernatant in a separate 50ml falcon (in case the protein does not attach well to the beads, this supernatant can be used for a second purification)
</LI></UL><UL><LI>Eluate beads with 1ml elution buffer (50mM Tris pH 8, 10mM Glutathione) – transfer slurry into a 2ml eppi
</LI></UL><UL><LI>Centrifuge eppi at 13000rpm to make a stable pellet
</LI></UL><UL><LI>Transfer supernatant to a new eppi. Take care not to take up beads from the pellet.
</LI></UL><UL><LI>Optional: repeat j) – l) to wash the rest off the beads
</LI></UL><H3><SPAN class="mw-headline" id="Further_reading">Further reading</SPAN></H3><P>Sandra Harper, David W. Speicher  (2008); „Expression and Purification of GST Fusion Proteins“; John Wiley and Sons, Inc.; DOI: 10.1002/0471140864.ps0606s52
</P><H3><SPAN class="mw-headline" id="Plastic_binding_domain"><SPAN style="color:grey;">Plastic binding domain</SPAN></SPAN></H3><P><A href="http://partsregistry.org/Part:BBa_K608404" class="external text" rel="nofollow">BBa_K608404</A>
IPTG-inducible Promoter with plastic binding domain-tagged GFP
</P><H4><SPAN class="mw-headline" id="Experimental_setup">Experimental setup</SPAN></H4><P>According to <A href="https://2011.igem.org/Team:Freiburg/Description#Plastic_binding_domain" class="external text" rel="nofollow">Adey et al.</A> the plastic binding domain (pbd) binds to the polystyrene surface of micro titer plates (96 well plates). To investigate the binding properties of the plastic binding tag we started several spectroscopic assays using a plate reader (FLUOstar Omega) and polystyrene plates (Greiner bio one) To detect the fluorescence of the GFP tagged to the plastic binding domain we used black plates and a well scanning program measuring 10x10 spots in each well of the micro titer plate. We performed several washing steps to find out how much of the proteins can be found in the eluate and how much remains bound on the plate’s surface. To compare the pbd-tagged GFP to a normal GFP without special plastic binding ability we also measured GFP obtained via expression with our diverse PR (Promoter-Ribosome-binding-site constructs). To affirm the results obtained by fluorescence spectroscopy used the Bradford assay and screened for different protein concentrations in transparent polystyrene plates.
The calibration lines necessary for calculation of protein concentration can be found <A href="/wiki/images/1/16/Bradford%2BFluorescence_calibration.pdf" class="internal" title="Bradford+Fluorescence calibration.pdf">here</A>.
</P><H4><SPAN class="mw-headline" id="Results_3">Results</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Picture 1: % of tagged (red) or untagged (blue) GFP remaining in the well after washing compared to previous washing step / original concentration</DIV></DIV></DIV> After the first measurement of the basic fluorescence intensity we transferred the samples onto another well, refilled the exhausted well with PBS and measured both eluate and remaining protein. In a second washing step the liquid was taken out of the first well again and given to another well. This washing was performed a third time, resulting in three eluates and a triply washed well with more or less protein remaining on the walls.

As shown in picture 1, both pbd-tagged and untagged GFP fluorescence decreases after the first washing step. Only 0- 4% of the original concentrations of the untagged GFP and 2-7% of the pbd-tagged GFP remained in the well.<DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Picture 2: % of pbd-tagged GFP that can be eluted in the first washing step. The curve decreases with the amount of the start concentration. If the GFP-pbd solution is not oversaturated a lower amount of this protein can be washed away.</DIV></DIV></DIV> After a second washing step there were differences observable between the tagged and the untagged GFP.  While the percentage of the remaining “normal” GFP was scattered around zero, an average of 60% of the pbd-tagged GFP remained in the well. In a third washing step, this observation was confirmed. While, like before, the main part of the pbd-GFP remained in its original well, nearly all of the untagged GFP was washed off.
<P>
Taking a look at the pbd-bound GFP, it is striking that after the solution has already been diluted by one washing step, the percentage of protein that can be washed away diminishes. It comes to mind that the massive loss of pbd-tagged GFP after the first washing step might be due to an oversaturation of pbd-GFP in the solution.  In this case there would be too much pbd-GFP for the available plastic surface, so that most of the protein could be eluated.
</P>
As shown in picture 2 the percentage of eluted GFP diminishes when the used start concentration is lower. <DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Picture 3: if the start concentration is not oversaturated pbd-coupled is much more resistant to washing steps than GFP alone.</DIV></DIV></DIV><P>
To investigate this phenomenon  we compared different start concentrations of pbd-GFP concerning the amount of pbd-GFP that could be washed away.
polystyrene micro titer plates provide only a limited surface for the pbd to bind, the solution shouldn’t be oversaturated with plastic binding protein. In our case the diluted protein could only reach a surface of  91.5 mm2  per well. With a remaining protein amount of ~220 pg/well after three washing steps we estimate that 2,4 pg pbd-tagged protein can be bound per mm2.

In a range of 1-30ng/µL start concentration there remains about seven times more pbd-GFP after washing than &quot;normal&quot; GFP (see picture 3).
</P><H4><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H4><P>In the experiments described above we could show that in a certain range of start concentration the plastic binding domain (pbd)-coupled GFP was more resistant to elution steps than GFP alone. This result indicates that the pbd-tagged GFP bound stronger to the plastic surface of the microtiter plate than “normal” GFP. The fact that in case of a rather high start concentration the majority of the pbd-GFP was washed away can be explained by the limitation of available binding surface. It seems like the pbd-GFP binds seven times better to the used polystyrene material than GFP alone. The amount of pbd-GFP that can bind to 1 mm2 is about 2,4 pg. To prove this supposition further measurement and more washing steps should be performed.
</P><H2><SPAN class="mw-headline" id="Green_light_receptor"><SPAN style="color:green;">Green light receptor</SPAN></SPAN></H2><P>We could successfully amplify the gene sequences for <I>CcaS</I> and <I>CcaR</I> and the promotor region <I>PcpcG</I> via PCR from the whole <I>Synechocystis sp.</I> PCC6803 genome.
Later two parts (<A href="http://partsregistry.org/Part:BBa_K608101" class="external text" rel="nofollow">BBa_K608101</A> , <A href="http://partsregistry.org/Part:BBa_K608102" class="external text" rel="nofollow">BBa_K608102</A>), were inserted into the iGEM-Vector pSB1C3 and send to the registry.
Twice we tried to clone the promotor region <I>PcpcG</I> into a vector, but had no insert at all.</P><P>After a summer full of cloning we ran out of time to assemble 
the green light receptor cassette like we planned to.
For this reason we could not test the green light receptor and can not provide more information.</P><H2><SPAN class="mw-headline" id="red_light_receptor"><SPAN style="color:red;">red light receptor</SPAN></SPAN></H2><P>In August we decided to drop the red light receptor system.
The main reason beeing Jakob having to leave the team to start studying in Strasbourg.
So the FreiGEM team shrunk to five people.</P><P>Still we did submit one part for this system which is <A href="http://partsregistry.org/Part:BBa_K608151" class="external text" rel="nofollow">BBa_K608151</A>
the translational unit of pcyA
</P><P>We talked to some of the iGEM teams like TU Munich also working with this system
and we are sure that the red light receptor is not lost.</P><H2><SPAN class="mw-headline" id="blue_light_receptor"><SPAN style="color:blue;">blue light receptor</SPAN></SPAN></H2><P>The handling with the LOV domain and TrpR domain was very difficult.
On the one hand it was difficlut to digest the parts, because there was a SpeI restriction site in the tryptophan repressible promoter region. So we first had to mutate SpeI into NehI restriction site to digest our parts without digesting the promotor. In the beginning we also tried the Gibson assembly. Primer design was quite difficult because of the lenght, the usually very high annealing temperature and unspecifity of the primer sequences, combined to the mutations we had to include.  These hassel leaded to trouble to amplify the Lov domain.The 3A-assembly with the mutated parts wasn´t successful either. We only cloned the Not-Gate in the pSB1C3 vector. We tried this experiment with different parameters but we weren´t successful.
</P><H2><SPAN class="mw-headline" id="Lysis_cassette"><SPAN style="color:orange;">Lysis cassette</SPAN></SPAN></H2><P>
After a lot of mishappenings concerning the 2 parts making up the temperature sensitive lysis cassette, we made a last attempt to ligate them (link to workbook) and transform the novel composite over the last weekend before the Wiki-freeze. After putative positive colonies were picked, we managed to perform a last-minute, &quot;blind&quot; -ie not knowing if the sequence of the insert was correct-, and preliminary OD-measurement testing.
</P><P>We later verified the part sequence. Since we cannot upload .ab1 files, we made 2 base-call files from them without changing absolutely anything (that means that there are some false bases at the beginning and end of every file). 
</P><UL><LI><A href="/File:Freiburg11_S48%2BS66_Nr3-P8,fw.ab1_basecalls_.gb" title="File:Freiburg11 S48+S66 Nr3-P8,fw.ab1 basecalls .gb">File:Freiburg11 S48+S66 Nr3-P8,fw.ab1 basecalls .gb</A></LI></UL><DL><DD>Forward Primer
</DD></DL><UL><LI><A href="/File:Freiburg11_S48%2BS66_Nr3-P9,rev.ab1_basecalls_.gb" title="File:Freiburg11 S48+S66 Nr3-P9,rev.ab1 basecalls .gb">File:Freiburg11 S48+S66 Nr3-P9,rev.ab1 basecalls .gb</A></LI></UL><DL><DD>Reverse Primer
</DD></DL><P>The OD values of the measurement are shown in the chart (please do bear in mind that this was a measurement done under deadline-stress, without the possibility to plan and perform something more concrete before the Wiki-Freeze)
</P><DL><DD>LB with Cm (Chloramphenicol) was used as a blank before every (hourly) measurement
</DD></DL><P>Nr1: Insert is only part (<A href="http://partsregistry.org/Part:BBa_K608351" class="external text" rel="nofollow">BBa_K608351</A>)

Nr2: Insert is only part (<A href="http://partsregistry.org/Part:BBa_K608352" class="external text" rel="nofollow">BBa_K608352</A>)

Nr3: Both parts were correctly assembled (1:(<A href="http://partsregistry.org/Part:BBa_K608351" class="external text" rel="nofollow">BBa_K608351</A>) + 2:(<A href="http://partsregistry.org/Part:BBa_K608352" class="external text" rel="nofollow">BBa_K608352</A>))
</P><P>Therefore the temperature sensitive lysis cassette seems to be pretty temperature-sensitive but also functioning properly. More tests are planned if time allows (ie wiki-unfreeze ;) )
</P><H1><SPAN class="mw-headline" id="Parts_submitted_to_the_registry">Parts submitted to the registry</SPAN></H1><P>All our results are documented in our notebook and summarized in the partsregistry.
Here we shortly describe what happened to give you an idea about the current status of the different subprojects.
Please follow the links for more information.
</P><H2><SPAN class="mw-headline" id="Precipitator_2"><SPAN style="color:grey;">Precipitator</SPAN></SPAN></H2><P><A href="http://partsregistry.org/Part:BBa_K608404" class="external text" rel="nofollow">BBa_K608404</A>
IPTG-inducible Promoter with <B>plastic binding domain</B>-tagged GFP
</P><P><A href="http://partsregistry.org/Part:BBa_K608406" class="external text" rel="nofollow">BBa_K608406</A><B>Precipitator</B></P><P>The Precipitator is a new artificially designed LRR protein. It is meant as protein that binds Nickel ions with Histidines grouped on its surface. The bound Nickel can then precipitate His-tagged proteins. In our Lab in a Cell it should function as an adaptor between the plastic surface of pipettes and the His-tagged protein. Please look at our detailed description of the design layout in our modeling section. The sequence was synthesised and cloned into the iGEm vector. The submitted sequence was fully confirmed by sequencing.
</P><P><A href="http://partsregistry.org/Part:BBa_K608407" class="external text" rel="nofollow">BBa_K608407</A><B>Precipitator fused with GST tag</B></P><P>The Precipitator was fused with the C-terminus of our GST tag in order to extract and further test the construct for its Nickel binding affinity. We successfully cloned it together using the Gibson Assembly and then subsequently pasted it into the iGEM vector. The submitted sequence was partially confirmed by sequencing.
</P><P><A href="http://partsregistry.org/Part:BBa_K608408" class="external text" rel="nofollow">BBa_K608408</A><B>GST-tag</B></P><P>The GST-tag was PCR amplified from a pGEX vector with overhang primers including the iGEM restriction sites and then pasted into the iGEM vector. To verify the functionality of the construct we cloned it before a GFP sequence and expressed it with an IPTG inducible vector. Results see partsregistry page. The submitted sequence was partially confirmed by sequencing.
</P><H2><SPAN class="mw-headline" id="Green_light_receptor_2"><SPAN style="color:green;">Green light receptor</SPAN></SPAN></H2><P><A href="http://partsregistry.org/Part:BBa_K608101" class="external text" rel="nofollow">BBa_K608101</A><B>CcaR</B>, green light response regulator
</P><P>The green light response regulator CcaR was amplified via PCR from
the whole <I>Synechocystis sp. PCC 6803</I> genome and inserted into the pSB1C3 vector.
</P><P><A href="http://partsregistry.org/Part:BBa_K608102" class="external text" rel="nofollow">BBa_K608102</A></P><P><B>CcaS</B> , green light receptor
</P><P><A href="http://partsregistry.org/Part:BBa_K608151" class="external text" rel="nofollow">BBa_K608151</A>
translational unit of pcyA
</P><H2><SPAN class="mw-headline" id="Lysis_cassette_2"><SPAN style="color:orange;">Lysis cassette</SPAN></SPAN></H2><P><A href="http://partsregistry.org/Part:BBa_K608351" class="external text" rel="nofollow">BBa_K608351</A>
(correct) temperature sensitive promoter
</P><P><A href="http://partsregistry.org/Part:BBa_K608352" class="external text" rel="nofollow">BBa_K608352</A>
Bacteriophage Lysis Cassette with RBS
</P><H2><SPAN class="mw-headline" id="commons"><SPAN style="color:black;">commons</SPAN></SPAN></H2><P>At the beginning we had to assemble promoters and ribosome binding sites of different &quot;strengths&quot; (ie consensus identities) in order to ensure proper and controlled expression of our parts.
The cloning procedure was time-consuming (circa two weeks), so we decided to
send our constructs to the registry. We hope to help other teams saving some time.
We called these constructs <I>PR</I> to abbreviate &quot;Promoter and RBS&quot;
</P><P><A href="http://partsregistry.org/Part:BBa_K608002" class="external text" rel="nofollow">BBa_K608002</A>
strong Promoter and strong RBS (PR1)
</P><P><A href="http://partsregistry.org/Part:BBa_K608003" class="external text" rel="nofollow">BBa_K608003</A>
strong Promoter , medium RBS (PR2)
</P><P><A href="http://partsregistry.org/Part:BBa_K608004" class="external text" rel="nofollow">BBa_K608004</A>
strong Promoter , weak RBS (PR3)
</P><P><A href="http://partsregistry.org/Part:BBa_K608005" class="external text" rel="nofollow">BBa_K608005</A>
medium Promoter , strong RBS (PR4)
</P><P><A href="http://partsregistry.org/Part:BBa_K608006" class="external text" rel="nofollow">BBa_K608006</A>
medium Promoter , medium RBS (PR5)
</P><P><A href="http://partsregistry.org/Part:BBa_K608007" class="external text" rel="nofollow">BBa_K608007</A>
medium Promoter , weak RBS (PR6)
</P><P><B>Measurement of fluorescence intensity of PR-GFP and PR-RFP</B></P><P>
To quantify the strength of our PR-constructs we cloned GFP and RFP behind it
and quantified the protein output.</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">BSA calibration line</DIV></DIV></DIV><P>The fluorescence was measured with a plate reader:
The fluorescence intensity and protein concentration were measured with the FLUOstar Omega, 
which is a multi-mode microplate reader.
Samples were pipetted into the microplate and analyzed via the plate reader. In this experiment we focused on the protein concentration and the fluorescence intensity of RFP.
We measured the protein concentration with the bradford-assay. This is a method to determine the total protein concentration. To analyze the protein concentration of the samples, Coomassie Brillant Blue was pippeted to each sample. With the binding of the dye to the proteins the color changes from dark red to blue. The more protein in the solution the more Coomassie dye can bind to proteins and the more the color changes into blue. The absorption of bound Coomassie dye is 595nm. The absorbance is proportional with the amount of bound dye. With a series of Bovine Serum Albumin (BSA) measurements the exact protein concentration of the samples can be determined. BSA acts like a “marker” because the concentration of BSA is known and with a linear calibration line the exact protein concentration can be detected. 
</P><P>The values of the fluorescence intensity of RFP and GFP deviate from the expected values. We´ve expected PR1 to have the strongest GFP/RFP signal and PR6 with the lowest GFP/RFP signal. The data shows the differences. We could not repeat this measurement another time because of lack of time. We could not measure PR1-GFP because the sequencing was negative and there was no time for repeating. The measurement of PR3-RFP was an error and was excluded. 
</P><P><B>PR-GFP</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:364px;"><DIV class="thumbcaption">GFP fluorescence intensity dependent on the strenght of promoter and RBS.</DIV></DIV></DIV><P>GFP served as a reporter of expression. We wanted to know how strong the promoter and RBS activity is. With this reporter gene it was possible to analyze the expression via plate reader. GFP is excited at a wavelength of 509nm and has an emission of 520nm. The plate reader illuminates the samples with a high energy xenon flash lamp. Optical filters or monochromator create the exact wavelength. The more GFP in the sample the higher is the GFP fluorescence intensity. The intensity is collected with the second optical system and is detected with a side window photomultiplier tube.
</P><P><B>parts:</B><A href="http://partsregistry.org/Part:BBa_K608008" class="external text" rel="nofollow">BBa_K608008</A>
constitutive strong promoter with medium RBS and GFP (PR2-GFP)
</P><P><A href="http://partsregistry.org/Part:BBa_K608009" class="external text" rel="nofollow">BBa_K608009</A>
Strong promoter with weak RBS and GFP (PR3-GFP)
</P><P><A href="http://partsregistry.org/Part:BBa_K608010" class="external text" rel="nofollow">BBa_K608010</A>
Medium promoter with strong RBS and GFP (PR4-GFP)
</P><P><A href="http://partsregistry.org/Part:BBa_K608011" class="external text" rel="nofollow">BBa_K608011</A>
Medium promoter with medium RBS and GFP (PR5-GFP)
</P><P><A href="http://partsregistry.org/Part:BBa_K608012" class="external text" rel="nofollow">BBa_K608012</A>
Medium promoter with weak RBS and GFP (PR6-GFP)
</P><P><B>PR-RFP</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:364px;"><DIV class="thumbcaption">RFP fluorescence intensity dependent on the strength of promotor and RBS.</DIV></DIV></DIV><P>With RFP (Red Fluorescence Protein) the activity of promoter and RBS can be quantified.
RFP served like GFP as a reporter to determine the expression level. RFP is excited at a wavelength of 580nm. The more RFP in the solution the more is the RFP fluorescence intensity.The plate reader illuminates the samples with a high energy xenon flash lamp. Optical filters or monochromator create the exact wavelength. The more RFP in the sample the higher is the RFP fluorescence intensity. The intensity is collected with the second optical system and is detected with a side window photomultiplier tube.
</P><P><B>parts:</B></P><P><A href="http://partsregistry.org/Part:BBa_K608013" class="external text" rel="nofollow">BBa_K608013</A>
strong Promoter with strong RBS and RFP (PR1-RFP)
</P><P><A href="http://partsregistry.org/Part:BBa_K608014" class="external text" rel="nofollow">BBa_K608014 </A>
PR with RFP (PR2-RFP)
</P><P><A href="http://partsregistry.org/Part:BBa_K608015" class="external text" rel="nofollow">BBa_K608015</A>
PR with RFP (PR3-RFP)
</P><P><A href="http://partsregistry.org/Part:BBa_K608016" class="external text" rel="nofollow">BBa_K608016</A>
PR with RFP (PR4-RFP)
</P><P><A href="http://partsregistry.org/Part:BBa_K608017" class="external text" rel="nofollow">BBa_K608017</A>
PR with RFP (PR5-RFP)
</P><P><A href="http://partsregistry.org/Part:BBa_K608018" class="external text" rel="nofollow">BBa_K608018</A>
PR with RFP (PR6-RFP)
</P></DIV></DIV><DIV align="center">                  </DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Freiburg/Results">http://2011.igem.org/Team:Freiburg/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Results&amp;oldid=168984" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>