<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Description"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Description">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Description&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Description&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Description&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Description" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Description</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="up_white_container"><DIV id="over_the_header_button"><DIV class="color"><A href="https://2011.igem.org/Team:Freiburg/Contact"> Contact </A></DIV></DIV></DIV><DIV id="up_line"><DIV id="up_line_header"><DIV style="float:right; position: absolute; bottom: 15px; right: 100px; padding-bottom:2px;">This is the wiki page of the Freiburger student team competing for iGEM 2011. Thank you for your interest! </DIV></DIV></DIV><DIV id="my_content"><DIV id="nav-body" style="width: 985px; float:left"><DIV id="navigation" style="width: 150px; float:left"><UL><LI><A id="home" href="https://2011.igem.org/Team:Freiburg" display="block"><SPAN class="blueSelect">&lt;</SPAN>home<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI><LI><A id="team" href="https://2011.igem.org/Team:Freiburg/Team" display="block"><SPAN class="blueSelect">&lt;</SPAN>team<SPAN class="blueSelect"> &gt; </SPAN></A></LI><LI id="members" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Members"><SPAN class="blue">→</SPAN>members</A></LI><LI id="attributions" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Attributions"><SPAN class="blue">→</SPAN>attributions</A></LI><LI id="freiburg" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Freiburg"><SPAN class="blue">→</SPAN>freiburg</A></LI></UL><LI><A id="project" href="https://2011.igem.org/Team:Freiburg/Project" display="block"><SPAN class="blueSelect">&lt;</SPAN>project<SPAN class="blueSelect"> &gt; </SPAN></A></LI><LI id="description" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Description"><SPAN class="blue">→</SPAN>description</A></LI><LI id="results" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Results"><SPAN class="blue">→</SPAN>results</A></LI><LI id="modelling" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Modelling"><SPAN class="blue">→</SPAN>modeling</A></LI><LI id="notebook" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Notebook"><SPAN class="blue">→</SPAN>notebook</A></LI><LI id="sampledata" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/SampleData"><SPAN class="blue">→</SPAN>sample data</A></LI><LI><A id="humanpractices" href="https://2011.igem.org/Team:Freiburg/Oath"><SPAN class="blueSelect">&lt;</SPAN>human practices<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI><LI id="oath" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Oath"><SPAN class="blue">→</SPAN>oath</A></LI><LI id="socioeconomicconsiderations" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/SocioeconomicConsiderations"><SPAN class="blue">→</SPAN>ethics</A></LI><LI id="safety" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Safety"><SPAN class="blue">→</SPAN>safety</A></LI><LI><A id="gallery" href="https://2011.igem.org/Team:Freiburg/Gallery"><SPAN class="blueSelect">&lt;</SPAN>gallery<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI><LI><A id="sponsors" href="https://2011.igem.org/Team:Freiburg/Sponsors" display="block"><SPAN class="blueSelect">&lt;</SPAN>sponsors<SPAN class="blueSelect"> &gt; </SPAN></A></LI><LI><A id="synbio oath" href="https://2011.igem.org/forum/Topic_List?n=368&amp;s=1"><SPAN class="blueSelect">&lt;</SPAN>to the forums<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI></DIV><DIV id="noNavigationContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Precipitator"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Precipitator</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_Concept"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The Concept</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Part_design"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Part design</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Plastic_binding_domain"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Plastic binding domain</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Green_light_receptor"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Green light receptor</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Blue_light_receptor"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Blue light receptor</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Red_light_receptor"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Red light receptor</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Lysis_cassette"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Lysis cassette</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#The_Concept_2"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">The Concept</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Part_Design_2"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Part Design</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#The_temperature_sensitive_promoter_.28BBa_K608351.29"><SPAN class="tocnumber">6.2.1</SPAN><SPAN class="toctext">The temperature sensitive promoter (BBa_K608351)</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#The_.CE.BB_lysis_genes_.28BBa_K608352.29"><SPAN class="tocnumber">6.2.2</SPAN><SPAN class="toctext">The λ lysis genes (BBa_K608352)</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#The_temperature_sensitive_lysis_cassette"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">The temperature sensitive lysis cassette</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Bacterial_artificial_chromosome"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Bacterial artificial chromosome</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Precipitator"><SPAN style="color:grey;">Precipitator</SPAN></SPAN></H2><H3><SPAN class="mw-headline" id="The_Concept">The Concept</SPAN></H3><P>We created a cellular, self-replicating purification device for His-tagged proteins. It is a completely artificial fusion protein, which consists of a repeating LRRNT motif domain, coordinating Ni2+ Ions on its surface capped on N and C terminal ends by a hagfish sequence of a similar LRRNT motif. We named this construct “THE PRECIPITATOR”. A second domain, a short hydrophobic peptide stretch, binds a polystyrene surface, called the plastic binding domain.
After expression of the Precipitator in a light inducible <I>E. coli</I> strain, the cells are lysed and the lysate is taken up with a serological pipette, in preparation of the actual protein purification steps.
The underlying mechanism is comparable to Ni-NTA columns. Our Precipitator protein binds on the surface of the pipette, presenting the chelated Nickel ions. Free coordination sites of the Nickel ions are exposed, so that a His-tagged protein can attach to them. Cells expressing a His-tagged protein can be dissolved by the heat inducible lysis-device. Subsequently, when the lysate is taken up with a serological pipette coated with the Precipitator protein, the His-tagged proteins bind to it. Cell debris is then washed off, while the His-tagged protein stays and is eluted afterwards, in the same fashion as done in Ni-NTA columns with imidazole solutions, increasing in concentration. The His-tagged protein is finally captured in a distinct fraction.
</P><H3><SPAN class="mw-headline" id="Part_design">Part design</SPAN></H3><P>The Precipitator <A href="http://partsregistry.org/Part:BBa_K608406" class="external text" rel="nofollow">BBa_K608406</A> protein is made of an artificial Leucine Rich Repeat (LRR) as the middle part of our own design, capped by C and N-terminal hagfish domain fragments. This part is one version of three different designed to bind nickel by histidines, grouped together pointing away from the horseshoe shaped protein. Please see modeling for more details
</P><P>Bacterial LRR Consensus of the central LRR fragment: 
LxxLxLxxNxLxxLPxxLPxx 
</P><TABLE border="1" align="center" style="color:black; background-color:#ffffcc;" width="100%" cellpadding="10%" cellspacing="0"><TBODY><TR><TD style="width: 50%;background-color:lightgrey;"><B>Protein sequence</B><SPAN style="color:violet;"><P><B>CPSRCSCSGTEIRCNSKGLTSVPTGIPSS</B><B>ATRLELESNKLQSLPHGVFDK</B></P></SPAN><SPAN style="color:green;"><B>LTQLTKSNN</B><SPAN style="color:red;"><B>H</B></SPAN>L<SPAN style="color:red;"><B>H</B></SPAN><B>SLPDNLPAS</B><B>LEVLDVSNN</B><SPAN style="color:red;"><B>H</B></SPAN>L<SPAN style="color:red;"><B>H</B></SPAN><B>SLPDNLPAS</B><B>LEVLDVSNN</B><SPAN style="color:red;"><B>H</B></SPAN>L<SPAN style="color:red;"><B>H</B></SPAN><B>SLPDNLPAS</B><B>LEVLDVSNN</B><SPAN style="color:red;"><B>H</B></SPAN>L<SPAN style="color:red;"><B>H</B></SPAN><B>SLPDNLPAS</B><B>LEVLDVSNN</B><SPAN style="color:red;"><B>H</B></SPAN>L<SPAN style="color:red;"><B>H</B></SPAN><B>SLPDNLPAS</B><B>LEVLDVSNN</B><SPAN style="color:red;"><B>H</B></SPAN>L<SPAN style="color:red;"><B>H</B></SPAN><B>SLPDNLPAS</B></SPAN><SPAN style="color:yellow;"><B>LKELALDTNQLKSVPDGIFDR</B><B>LTSLQKIWLHTNPWDCSCPRIDY</B><B>LSRWLNKNSQKEQGSAKCSGSGKPVRSIICP</B></SPAN></TD></TR><TR><TD colspan="2" style="width: 50%;background-color:lightgrey;"><P><B>Colorcode (same in the sequence and video)</B><SPAN style="color:#83D630;"><B>green spheres: conserved Aminoacids Leu, Pro, Asn</B></SPAN><SPAN style="color:red;"><B>Histidin</B></SPAN><SPAN style="color:violet;"><B>N-terminal Hagfish cap</B>cap</SPAN><SPAN style="color:yellow;"><B>C-terminal Hagfish cap</B></SPAN><SPAN style="color:green;"><B>artifical LRR domain fragment</B></SPAN></P></TD></TR></TBODY></TABLE><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" width="20%" align="right"><TBODY><TR><TD><A href="/File:HURRRRGH.png" title="File:HURRRRGH.png">200px</A><P>Precipitator binding a polystyrene surface with the plastic binding domain and a His-tagged Protein via Nickel ions
</P></TD></TR></TBODY></TABLE><P>This protein can be used to complex Nickel or Cobalt. Histidines are positioned in such a way, that they can coordinate the ions
from two to four orthogonally oriented directions. Free binding sites of the ions are then exposed, so that a His-tagged protein can attach to them. This protein can be used to complex up to 4 nickel or cobalt ions. 
The underlying design of the protein is of a particular interest, too. 
LRRs are highly conserved motifs throughout evolution. They appear in all kingdoms of life in almost every thinkable role (ligases, receptors, toxins etc.). Their core is highly conserved and provides a very stable backbone, while the non-conserved aminoacids are almost freely interchangeable. 
Here we investigated an optimal set of non-conserved aminoacids by analysing large sets of similar proteins and databases. You can use this piece of work as a template to design your own protein and give it any function you like, by simply interchanging aminoacids and fusing other domains on the N or C termini. To guarantee proper folding and to shield off the hydrophobic core, a well studied fragment of an LRR protein coming from hagfish was used. The efficiency of this technique was proven before (Schmidt 2010). To find out the most likely folding, we designed many different protein sequences, trying out a variety of sets of non coding aminoacids for the LRR and submitted these to the I-TASSER structure prediction 
</P><P>We only submitted one of the three versions to the registry to reduce redundancy. Please contact us for any questions.
</P><H2><SPAN class="mw-headline" id="Plastic_binding_domain">Plastic binding domain</SPAN></H2><P>One of the issues of our project „lab in a cell“ was to use endogenous proteins produced by the cell itself for specific purification and hereby to replace expensive columns. The „precipitator“ designed by our team contains a protein binding domain which complexes nickel and thus enables the binding of His-tagged proteins. After cell lysis the “precipitator” is freely dissolved in the cell lysate. To be able to isolate the “precipitator”-His-tagged-protein-complex from the other cell components it has to be immobilized by another protein domain. The part we designed for this function is the so-called plastic binding domain.
</P><P>During routine phage display of random peptide libraries, phages were found that bound directly to the plastic surface of the used plastic micro titer plates. The number of plastic binding phages obtained during the phage display experiments depended on the saturation of the plastic micro titer plates with target protein for the antibody-binding phages and could be reduced by the use of blocking proteins as BSA or non-fat milk. Plastic binding phages were resistant to washing steps with PBS alone as well as to PBS in combination with BSA or non-fat dry milk. It was shown that plastic binding phages were even more difficult to recover by acid elution than the “normal” antibody binding phages (Adey et al., 1995). 
</P><P>The binding strength of the plastic binding protein was best on polystyrene plastic surfaces and also observed on PVC-plates (Adey et al., 1995). 
</P><P>The mechanisms by which the phage surface proteins bind to plastic are not well understood. The plastic binding amino acid sequences showed no obvious sequence similarity but where generally enriched by Tyr and Trp residues and are completely devoid of Cys residues. It is possible that the binding comes off non-specific hydrophobic interactions due to partial denaturation of the protein (Cantanero et al., 1980) and potentially due to interactions of the Tyr and Trp residues with the aromatic moieties of the polystyrene plastic surface of micro titer plates.
</P><P>As well as high hydrophobicity does not necessarily imply plastic binding quality (Menendez et al., 2005) the observed plastic binding phages showed hydrophobic peptide sequences on their surfaces. Expressing such hydrophobic proteins in our host organism <I>E. coli</I> can lead to problems with inclusion bodies or decreased vitality up to cell death.
</P><P>During our experiments we couldn’t obtain any clones containing both the gene for the plastic binding protein and a constitutive promoter-RBS-construct. We finally succeeded in expressing the plastic binding domain using an inducible promoter. To test the plastic binding domain and get data for our modeling we hereby used an IPTG-inducible promoter. In our completed “Lab in a cell” model the plastic binding domain, as a part of the “precipitator” would be expressed by one of our light inducible promoters.
</P><H2><SPAN class="mw-headline" id="Green_light_receptor"><SPAN style="color:green;">Green light receptor</SPAN></SPAN></H2><P>
Sometimes a regulated and coordinated gene expression and therefore protein production is needed.
We decided to use light-controlled gene expression, because light is everywhere and always available.
The green light receptor is a light-sensing system from the cyanobactrium  <I>Synechocystis sp.</I> PCC6803. It consists of three parts interacting with each other in order to start regulated gene expression. These parts are the following:
The main receptor is  CcaS(1), a cyanobacteriochrome, which shows increased autophosphorylation after exposure to green light. The protein CcaS is made up of a N-terminal transmembrane helix, a cyanobactreria specific  GAF domain, two PAS domains and a C-terminal histidine kinase.(Yuu Hirose et al.) 
To be fully functional CcaS has to bind the chromophore Phycocyanobilin (PCB) with its GAF-domain. 
The GAF domain in this system has the ligation motif Cys-Leu, instead of the usual plant GAF-domain with Cys-His.
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="right"><TBODY><TR><TD><P>Green light system
</P></TD></TR></TBODY></TABLE><P>Its response regulator is CcaR (2)which belongs to the family of OmpR regulators. CcaR consists of an N-terminal receiver domain that can be phosphorylated by CcaS and a C-terminal DNA-binding domain that binds directly to the promoter region of cpcG2 (3). CpcG2 is an atypical phycobilisome which is playing a role in the energy transfer to photosystem I. (Yuu Hirose et al.)
</P><P>After light of the wavelength of 532 nm is exposed to the CcaS receptor, it changes its conformation. It undergoes autophosphorylation and the phosphate is transfered to the response regulator CcaR. Once phosphorylated, CcaR can bind to the specific promoter region of cpcG and activate gene expression.
</P><P>As the green light sensing system from the cyanobacteria <I>Synechocystis sp.</I> PCC6803  was proven by J. J. Tabor to work also in <I>E. coli</I>, our plan is to integrate the genes for CcaS and CcaR into <I>E. coli</I> genome with a BAC (bacterial artificial chromosome). The researchers gene of interest just needs to be inserted behind the cpcG2 promoter region and transferred into &quot;our&quot; <I>E .coli</I> strain to become green light inducible.
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="left"><TBODY><TR><TD><P>Green light system
</P></TD></TR></TBODY></TABLE><P>References:</P><P>Yuu Hirose et al ( 2008 )&quot;Cyanobacteriochrome CcaS is the green light receptor that induces the expression of phycobilisome linker protein.&quot;
Proc Natl Acad Sci U S A. 105: 9528–9533.
</P><P>Tabor, J. J., Levskaya, A., &amp; Voigt, C. A. (2011). Multichromatic control of gene expression in Escherichia coli. Journal of Molecular Biology, 405(2), 315-324. Elsevier Ltd. Retrieved from <A href="http://www.ncbi.nlm.nih.gov/pubmed/21035461" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/21035461</A></P><H2><SPAN class="mw-headline" id="Blue_light_receptor"><SPAN style="color:mediumblue;"><A href="http://www.youtube.com/watch?v=SEz70ClCetM|" class="external text" rel="nofollow">Blue light</A> receptor</SPAN></SPAN></H2><P><B>LOV- and tryptophan-activated protein (LovTAP)</B></P><P>We used the LovTAP protein designed by Strickland et al. to induce protein expression using blue light (470 nm). The LovTAP consists of the photoactive LOV2 domain of <I>Avena sativa</I> phototropin1 and the TrpR domain of <I>E. coli</I> as an output module. LOV2 is ligated via its carboxyl-terminal to the amino-terminal of TrpR with an α-helical domain linker. This shared helix couples the two functions of these proteins. The photoactive LOV domain (light, oxygen, voltage) absorbs photons leading to a formation of a covalent adduct between flavin mononucleotide (FMN) cofactor and a conserved cysteine residue (Strickland et al., 2008). The TrpR domain inhibits DNA digestion by nuclease. 
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="right"><TBODY><TR><TD><P>blue light system
</P></TD></TR></TBODY></TABLE><P>The shared helix acts as a rigid lever arm and assoociates with the domains dependent of the light/dark state. This causes a steric overlap. Photoexcitation changes the conformational ensemble of the proteins and also changes the stability of the helix-domain contacts. This change shifts the relative affinity of the shared helix for each of the two domains. Photoexcitation sensed by one domain allows allosterical propagation to the other domain (Strickland et al., 2008).
</P><P>In the light state the LovTAP binds to the operator DNA and inhibits the gene expression.  To avoid this inhibition, we cloned a Not-Gate after the LovTAP.  Not-Gate is like an inverter, because it changes the input to its opposite. The Not-Gate consists of a tetracycline repressor and a TetR repressible promoter . In the light state the LovTAP binds to operator DNA and inhibits the expression of tetracycline repressor  and the TetR promoter is active. In the dark state the tryptophan repressor is expressed and inhibits the expression of the protein of interest.
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="left"><TBODY><TR><TD><P>LovTAP with Not-Gate
</P></TD></TR></TBODY></TABLE><P>Problems arose during the cloning of the LovTAP and Not-Gate in pSB1C3 vector. A reason might be that the mutation of the SpeI restriction site to NehI in the suffix wasn´t succesful and therefore the digestion with NehI did not work. Somehow we could only clone the Not-Gate successful in the vector without the LovTAP. 
</P><P>References:</P><P>Strickland et al. 2008
“Light-activated DNA binding in a designed allosteric protein”
National Academy of Sciences (2008) 10709-10714</P><H2><SPAN class="mw-headline" id="Red_light_receptor"><SPAN style="color:red;">Red light receptor</SPAN></SPAN></H2><P>The red light sensing system was used by various iGEM teams in the past, that’s why we thought it shouldn’t be too difficult to integrate it into our project.</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="right"><TBODY><TR><TD><P>the red light receptor system
</P></TD></TR></TBODY></TABLE><P>It consists of a transmembrane part called cph8 , its response regulator OmpR and a co-responding promoter region PompC. Like the green light receptor it needs a chromophore like Phycocyanobilin (PCB) to be fully functional.</P><P>More detailed:
</P><P>Cph8 is a fusion protein of the phytochrome cph1 from  Synechocystis sp. PCC6803 and the <I>E. coli</I> histidine kinase EnvZ.  When cph8 is phosphorylated it  passes a phosphoryl group to OmpR which binds to and activates transcription of promoter region<I> PompC</I> . (Tabor, J.) . OmpR is the response regulator for osmoregulation and as it is part of an important regulatory pathway in <I>E. coli</I>, you have to use an <I>E. coli</I>-strain deficient  of EnvZ to make your system light dependent. In the paper we are refering to cph8 is expressed from the promoter P(LTetO-1)in a phosphorylated ground state. Light of 650 nm wavelenght unphosphorylates cph8 and light of 705 nm wavelenght leads to the phosphorylated state again. So the red light receptor produces a positive output in darkness.
In order to have a positive output there  has to be a genetic inverter like a NOT-gate.
The question remains if cph8 would still be expressed with a different promoter in the phosphorylated ground state. 
</P><P>To establish a red light receptor you need a part design like the one of  <A href="http://partsregistry.org/Part:BBa_M30109" class="external text" rel="nofollow">BBa_M30109</A> , but like other iGEM teams we had difficulties with this part and decided to assemble it ourselves.
</P><P>References:</P><P>Tabor, J.J., Levskaya, A., &amp; Voigt, C. A. (2010). Multichromatic control of gene expression in Escherichia coli. Journal of Molecular Biology, 405(2), 315-324.
</P><H2><SPAN class="mw-headline" id="Lysis_cassette"><SPAN style="color:orange;">Lysis cassette</SPAN></SPAN></H2><H3><SPAN class="mw-headline" id="The_Concept_2">The Concept</SPAN></H3><P>The idea behind our lysis cassette was to be able to achieve cell lysis by simply heating the bacteria to 42°C for a short period of time. This was accomplished by combining the biobricks <A href="http://partsregistry.org/Part:BBa_K098995" class="external text" rel="nofollow">BBa_K098995</A> (temperature sensitive promoter) and <A href="http://partsregistry.org/Part:BBa_K124017" class="external text" rel="nofollow">BBa_K124017</A> (Bacteriophage λ lysis genes). Sequencing of these parts (carried out by GATC Biotech GmbH) showed some fundamental inconsistencies leading to a lot of time spent on fixing them. 
The sequences were corrected and the new biobricks <A href="http://partsregistry.org/Part:BBa_K608351" class="external text" rel="nofollow">BBa_K608351</A> and <A href="http://partsregistry.org/Part:BBa_K608352" class="external text" rel="nofollow">BBa_K608352</A> were sent to the registry, although the novel temperature sensitive lysis cassette composite's sequence was verified some 18 hours before the Wiki-freeze, so that it could not be sent to the registry on time.
Nevertheless some preliminary OD-measurement results did hint at the expected functionality of the newly developed composite part.</P><H3><SPAN class="mw-headline" id="Part_Design_2">Part Design</SPAN></H3><H4><SPAN class="mw-headline" id="The_temperature_sensitive_promoter_.28BBa_K608351.29">The temperature sensitive promoter (<A href="http://partsregistry.org/Part:BBa_K608351" class="external text" rel="nofollow">BBa_K608351</A>)</SPAN></H4><P>It consists of a constitutive promoter (<A href="http://partsregistry.org/Part:BBa_J23114" class="external text" rel="nofollow">BBa_J23114</A>), the temperature sensitive λ cI repressor protein which has a cI857 mutation that results in denaturation of the repressor when the temperature is raised from 30°C to 42°C, and a cI regulated promoter based on the λ pR promoter (<A href="http://partsregistry.org/Part:BBa_R0051" class="external text" rel="nofollow">BBa_R0051</A>).
</P><DL><DD>Troubleshooting: <A href="http://partsregistry.org/Part:BBa_K098995:Experience" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_K098995:Experience</A></DD></DL><TABLE width="610px" cellpadding="15%" cellspacing="0" border="1" align="center"><TBODY><TR><TD style="width: 50%;background-color:lightgrey;"><P>The temperature sensitive promoter functioning at 28-30°C
</P></TD></TR><TR><TD style="width: 50%;background-color:lightgrey;"><P>The temperature sensitive promoter functioning at 42°C
</P></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="The_.CE.BB_lysis_genes_.28BBa_K608352.29">The λ lysis genes (<A href="http://partsregistry.org/Part:BBa_K608352" class="external text" rel="nofollow">BBa_K608352</A>)</SPAN></H4><P>It consists of the standard bacteriophage λ lysis genes, ie S/S’, R, Rz/Rz1, coded in overlapping sequences with phase 1 and 2 frameshifts.
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="right"><TBODY><TR><TD><P>Bacteriophage λ lysis genes at work
</P></TD></TR></TBODY></TABLE><UL><LI>S: λ Anti-Holin
</LI><LI>S’: λ Holin
</LI><LI>R: λ Endolysin
</LI><LI>Rz: Putative type-II signal anchor protein 
</LI><LI>Rz1: Outer membrane lipoprotein
</LI></UL><P>The phage λ lysozyme (R Endolysin) hydrolyses the beta-1,4-glycosidic bond between N-acetylmuramic acid (MurNAc) and N-acetylglucosamine (GlcNAc), but in order to degrade the cell wall it requires a small transmembrane protein called holin which perforates the membrane for endolysin to gain access to the murein [Ry Young et al].
The auxiliary lysis proteins Rz and Rz1 have long been known to play a vital role but have yet to be assigned a specific function stemming from functional and structural analysis. The assumption is that they form a complex spanning the periplasm and fuse the outer and inner membranes, removing the last physical barrier for cell lysis [Joel Berry et al].  
</P><DL><DD>Troubleshooting: The biobrick did not have a RBS upstream of the CDS, so we had to clone the RBS <A href="http://partsregistry.org/Part:BBa_B0034" class="external text" rel="nofollow">BBa_B0034</A> in order to make a translational unit for cloning with the temperature sensitive promoter
</DD></DL><H3><SPAN class="mw-headline" id="The_temperature_sensitive_lysis_cassette">The temperature sensitive lysis cassette</SPAN></H3><P>The much anticipated part that we frustratingly could not clone in time to send to the registry.</P><P>Theoretically 28°C - 30°C incubation should repress the lysis genes, while a shift to 42°C would lead to their expression and finally cell lysis.
</P><TABLE width="50%" cellpadding="15%" cellspacing="0" border="1" align="center"><TBODY><TR><TD style="width: 50%;background-color:lightgrey;"><P>The temperature sensitive lysis cassette at 28-30°C
</P></TD><TD style="width: 50%;background-color:lightgrey;"><P>
The temperature sensitive lysis cassette at 42°C
</P></TD></TR></TBODY></TABLE><P><B>References:</B></P><P>Nils B. Adey et al. 1995 “Characterization of phage that bind plastic from phage-displayed random peptide libraries”
Gene 156 (1995) 27-31 </P><P>Alfredo Menendez &amp; Jamie K. Scott 2005 “The nature of target-unrelated peptides recovered in the screening of phage-displayed random peptide libraries with  antibodies”
Anal. Biochem. 336 (2005) 145-157 </P><P>L. A. Cantarero et al. 1980
“The absorptive characteristics of proteins for polystyrene and their significance in solid phase immunoassays” 
Anal. Biochem. 105 (1980) 375-382 </P><P>Ry Young et al. 2000
“Phages will out: strategies of host cell lysis”
Trends in Microbiology Vol 8, Issue 3 (2000) 120-128</P><P>Joel Berry et al. 2008
“The final step in the phage infection cycle: the Rz and Rz1 lysis proteins link the inner and outer membranes”
Molecular Microbiology 70 (2008) 341–351</P><H2><SPAN class="mw-headline" id="Bacterial_artificial_chromosome">Bacterial artificial chromosome</SPAN></H2><P>
Bacterial artificial chromosome (BAC) is a vector based on the single copy plasmid F-factor from <I>Escherichia coli</I>, which can host inserts from bacteria or other sources between 150 and 350 kilo base pairs (kbp) but up to 700 kbp, and hold them for more then 100 generations. The BAC vector has some common gene components that are: the oriS and repE which are responsible for the not bidirectional replication. The parA and parB genes control the copy number of the BAC. In addition there is a selection marker commonly an antibiotic resistance and finally the cloning segment with the restriction sites. This cloning segment will be flanked by a T7 and SP6 promoter. Usually BACs are used to build up libraries or genetically screening, through its stability.

We want to put most of our parts in a BAC so it can be used as a “Lab in a cell” system. This is very easy to handle and is a persistent transformation. Combined with our BAC vector we also have a transient plasmid with promoters for the expression of the His-tagged protein you want to purify. This protein can be cloned after these promoters and can be expressed with the influence of light (for example blue light). With this single cloning step, you can express your protein of interest. With the BAC vector all the other proteins are expressed dependent on light exposure and heat.
</P><P>References:
</P><P>O'Connor M, Peifer M, Bender W (1989). &quot;Construction of large DNA segments in Escherichia coli&quot;. Science 244 (4910): 1307–1312.
</P><P>Shizuya H, Birren B, Kim U-J, Mancino V, Slepak T, Tachiiri Y, Simon M (1992). &quot;Cloning and stable maintenance of 300-kilobase-pair fragments of human DNA in Escherichia coli using an F-factor-based vector&quot;. Proc Natl Acad Sci USA 89 (18): 8794–8797. 
</P></DIV></DIV><DIV align="center">                  </DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Freiburg/Description">http://2011.igem.org/Team:Freiburg/Description</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Description" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Description" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Description&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Description&amp;oldid=168886" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>