<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="up_white_container"><DIV id="over_the_header_button"><DIV class="color"><A href="https://2011.igem.org/Team:Freiburg/Contact"> Contact </A></DIV></DIV></DIV><DIV id="up_line"><DIV id="up_line_header"><DIV style="float:right; position: absolute; bottom: 15px; right: 100px; padding-bottom:2px;">This is the wiki page of the Freiburger student team competing for iGEM 2011. Thank you for your interest! </DIV></DIV></DIV><DIV id="my_content"><DIV id="nav-body" style="width: 985px; float:left"><DIV id="navigation" style="width: 150px; float:left"><UL><LI><A id="home" href="https://2011.igem.org/Team:Freiburg" display="block"><SPAN class="blueSelect">&lt;</SPAN>home<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI><LI><A id="team" href="https://2011.igem.org/Team:Freiburg/Team" display="block"><SPAN class="blueSelect">&lt;</SPAN>team<SPAN class="blueSelect"> &gt; </SPAN></A></LI><LI id="members" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Members"><SPAN class="blue">→</SPAN>members</A></LI><LI id="attributions" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Attributions"><SPAN class="blue">→</SPAN>attributions</A></LI><LI id="freiburg" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Freiburg"><SPAN class="blue">→</SPAN>freiburg</A></LI></UL><LI><A id="project" href="https://2011.igem.org/Team:Freiburg/Project" display="block"><SPAN class="blueSelect">&lt;</SPAN>project<SPAN class="blueSelect"> &gt; </SPAN></A></LI><LI id="description" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Description"><SPAN class="blue">→</SPAN>description</A></LI><LI id="results" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Results"><SPAN class="blue">→</SPAN>results</A></LI><LI id="modelling" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Modelling"><SPAN class="blue">→</SPAN>modeling</A></LI><LI id="notebook" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Notebook"><SPAN class="blue">→</SPAN>notebook</A></LI><LI id="sampledata" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/SampleData"><SPAN class="blue">→</SPAN>sample data</A></LI><LI><A id="humanpractices" href="https://2011.igem.org/Team:Freiburg/Oath"><SPAN class="blueSelect">&lt;</SPAN>human practices<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI><LI id="oath" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Oath"><SPAN class="blue">→</SPAN>oath</A></LI><LI id="socioeconomicconsiderations" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/SocioeconomicConsiderations"><SPAN class="blue">→</SPAN>ethics</A></LI><LI id="safety" class="sub2"><A href="https://2011.igem.org/Team:Freiburg/Safety"><SPAN class="blue">→</SPAN>safety</A></LI><LI><A id="gallery" href="https://2011.igem.org/Team:Freiburg/Gallery"><SPAN class="blueSelect">&lt;</SPAN>gallery<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI><LI><A id="sponsors" href="https://2011.igem.org/Team:Freiburg/Sponsors" display="block"><SPAN class="blueSelect">&lt;</SPAN>sponsors<SPAN class="blueSelect"> &gt; </SPAN></A></LI><LI><A id="synbio oath" href="https://2011.igem.org/forum/Topic_List?n=368&amp;s=1"><SPAN class="blueSelect">&lt;</SPAN>to the forums<SPAN class="blueSelect" display="none"> &gt; </SPAN></A></LI></DIV><DIV id="noNavigationContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Modelling:_Rational_protein_design"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Modelling: Rational protein design</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_Idea"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The Idea</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#The_core_structure"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">The core structure</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#imposing_desired_functions"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">imposing desired functions</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#stabilizing_the_structure"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">stabilizing the structure</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#optimzing_the_sequence"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">optimzing the sequence</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#A_mathematical_model_to_determine_the_experimental_design"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">A mathematical model to determine the experimental design</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Conclusions"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Conclusions</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Modelling:_Rational_protein_design">Modelling: Rational protein design</SPAN></H2><H3><SPAN class="mw-headline" id="The_Idea">The Idea</SPAN></H3><P>First there was the idea to replace the chemically produced Ni-NTA columns by some biological, reproducible material. It seemed unreasonable to produce an ion precipitating substance in a complicated, energy consuming way – completely artificial, while so many proteins in nature are capable of the same task – and much more complicated things. But how to harness that? 
The second milestone in the generation of the idea came up in a lecture of Prof. Martin, about Nickel allergy. He found a mutation in the TLR-4 receptor that introduces Histidines in the LRR motif of the protein, which are capable of binding nickel and thus forming complexes of receptors on the cell surface of immune cells, triggering the inflammatory reaction. 
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1"><TBODY><TR><TD><P>Schmidt 2010 a) Two sequences of a TLR-4 from mice, one with mutations, a phenotype showing Nickel allergy(Histidines marked red), one wild-type without a Nickel allergy. b) assumed superimposition of Nickel ions binding on the LeucineRichRepeat structure of the TLR-4
</P></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="The_core_structure">The core structure</SPAN></H3><P>This was the necessary clue that gave answer to the question.
In the following work, a lot of investigation about the perfect Nickel precipitating protein was done. It seemed that there is no optimal natural protein which would serves our needs. 
Furthermore, many of the natural LRR proteins – of course – have their own biological function, we we were afraid could interfere in the cellular system we wanted to express them in. Luckily there are over a thousand already known structures of LRR proteins, what gave us a broad choice of motifs to choose and compare.
The Construction of a new protein
After a long and detailed search, we found out that it is most reasonable to use bacterial LRR motifs, since they seemed very well conserved in sequence, and they are the shortest – with only 21 amino acids per LRR repeat. (Wei 2008, Kajava 1998). We wanted to have a protein as simple and as predictable in behavior and structure as possible.
Several search inquiries led to the most conserved and shortest of all bacterial LRR (PDB: 3CVR), unluckily this protein is a toxin derived from <I>Yersinia pestis</I>. We of course did not want to mess around with toxins,
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="right"><TBODY><TR><TD><P>The aminoacid sequence of a bacterial 
</P><P>ligase we chose for our design (PDB:3CVR)
</P></TD></TR></TBODY></TABLE><P>especially not with one of the black plague. We also did not know what amino acid pattern on the LRR motif would cause the toxic effect. Probably, there would have been no harm in using this protein alone, even if we did clone and express it, but why should we do that when we have a huge choice freely available anyway. Finally, we settled on the bacterial ligase PDB:3CVR. A ligase would be definitively harmless, especially since we planned to restructure it entirely.
</P><P>To do this we needed to understand how the structure of the Ligase was made up.
Our desired protein LRR motif was only a part of a bigger protein “factory” which included several domains that seemed to serve distinct functions – more than we had use for.
So we exclusively used the LRR domain. It seemed to us that this domain had mostly a structural role in this “factory”, but we could still not exactly tell. To avoid any unspecific biological function it was necessary to rearrange the amino acid composition.
We fed the protein sequence in a free online consensus sequence generator called weblogo from the Berkeley University.
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" width="75%"><TBODY><TR><TD><P>Consensus sequences derived from the repeating loops of the 3CVR ligase to work out the conserved aminoacids and shared chemical properties of the different positions in the loops.
</P></TD></TR></TBODY></TABLE><P>
By analyzing the logo it was obvious which positions of the LRR motif were conserved and which not – AND: which of the non-conserved amino acids appeared in what kind of pattern. Were there positions in the protein that required a polar amino acid? Or a non-polar, hydrophobic /-philic, charged, non-charged one?. We compared the consensus sequence with the 3D structure, using PYMOL, to extract as much information as possible, and then came up with this ideal consensus sequence:
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1"><TBODY><TR><TD><P>ideal consensus sequence extracted from the consensus motif above
</P></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="imposing_desired_functions">imposing desired functions</SPAN></H3><P>The next consideration we had to do was: how many Nickel do we need on the surface of our ideal Nickel binding protein, in what pattern, with what distances between, and at which angles toward each other to allow proper ion complexion?
Nickel can be complexed by imidazole structures from 4 planar orthogonal directions, as well as to axial positions. It can, however, only take four ligands at once - preferably in a planar orientation. Cobalt has a bipyramidal setup for ligand-binding, too, but can take up to six ligands. The distances from an N-atom in the Imidazole ring to the ion had to be between 3 and 6 Angström. We found a crystal structure of a different protein which was resolved with three Histidines complexing a Nickel ion for a comparison, as well as some old publications that analyzed peptide ion bonds that taught us how the complex should look like. (Jordan 1974)
We wanted our protein to be short. First in order to cause as little unspecific binding as possible and further to allow an easy expression as well as for keeping costs low for gene synthesis. For a proper purification of His-tagged proteins in Ni-NTA columns we knew that there are up to three Nickel involved in the interaction between the His tag (which consists of 6 or 7 Histidines) and the column. These have to be in close proximity to one another. There are always two Histidines of the tag binding to one ion.
</P><P>This was what we had to implement into our LRR backbone.
</P><P>To get a first impression how Histidines are positioned in a bacterial Ligase, we replaced all non conserved amino acids by Histidines and sent the sequence for a 3D sequence prediction to I-TASSER, an online structure prediction software tool.
</P><TABLE style="color:black; background-color:lightgrey;" width="85%" cellpadding="15%" cellspacing="0" border="1"><TBODY><TR><TD colspan="2"><FONT size="5">I-Tasser: Structure prediction</FONT></TD></TR><TR><TD style="width: 100%;background-color:lightgrey;"><DL><DT> I-Tasser is a 3-dimensional protein prediction method which can be used via a downloadable program or a online server. It is provided by the Yang-Zhang lab of the University of Kansas. 
</DT></DL><P>The community-wide Critical Assessment of Structure Prediction (CASP) ranked the I-Tasser as the best protein prediction method in the server section. 
The method of the I-Tasser can be divided into 4 stages:
Stage 1 is called threading. In this stage the sequence or parts of the sequence or motifs are compared with already known protein structures to identify evolutionary relatives and predict secondary structures. This gives a sequence profile and together with the query sequence it is threaded by LOMETS, a server combining seven threading programs which evaluate the sequence respectively the templates with an individual score . The top templates are selected for further consideration. 
Stage 2 is responsible for structural assembly. The structure is modeled ab initio by aligning different templates from the threading. This is a very complex process and different programs are involved.
Stage 3 is a model selection of the achievements of stage 2 with a following refinement of the predicted models. 
Stage 4 announces the function of the predicted molecule by its structural conformation comparing with already known proteins of PDB database. 
The predicted models are all evaluated by the C-Score, which assesses the quality of the prediction. It has a value from -5 to 2; the more positive the C-Score the merrier and more plausible the predicted structure is. 
</P><P>
Caption
</P><P>1.Ambrish Roy, Alper Kucukural, Yang Zhang. I-TASSER: a unified platform for automated protein structure and function prediction. Nature Protocols, vol 5, 725-738 (2010). 
</P><P>2.Yang Zhang. Template-based modeling and free modeling by I-TASSER in CASP7. Proteins, vol 69 (Suppl 8), 108-117 (2007).
</P></TD></TR></TBODY></TABLE><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="left"><TBODY><TR><TD><P>first aminoacid sequence submitted to I-TASSER to determine 
</P><P>proper Histidine patterns on the LRR backbone's surface
</P></TD></TR></TBODY></TABLE><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="right"><TBODY><TR><TD><P>first predicted structure used to determine proper Histidine 
</P><P>patterns on the LRR backbone's surface
</P></TD></TR></TBODY></TABLE><P>We received this pdb file with a C-score of 0.53. Obviously this was not a realistic model, since the folding would obviously be impaired by so many Histidines everywhere. But is was useful for finding several good locations where the Nickel could fit in in respect to the requirements it has for binding ligands. We manually fit in a Nickel ion, as it was crystallized in the mentioned PDB file, and measured the distances and evaluated the 3dimensional orientation of the Histidines towards the Nickel. Histidines can coordinate ligands with their free electron pair pointing planar away from the imidazole ring. What we further realized from the structure file was, that the ends of the LRR segments were “open”, meaning the hydrophobic core of the protein was exposed and as it is visible in the prediction, curled in on one end into a sort of helix. This means the protein folding is not reliable and the structure needs some sor of cap on either end in order to stabilize the LRR core motif.
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="right"><TBODY><TR><TD><P>superimposition of a Nickel ion into our first I-TASSER structure.
</P><P>Dotted lines show the distances between the atoms, angles were 
</P><P>analysed to see whether binding is likely to happen or not.
</P><P>Histidine residues were analysed for flexibility in the backbone
</P><P>and their potential to orient towards optimal binding.
</P></TD></TR></TBODY></TABLE><P>
A solution to this was shown by Schmidt et. al. 2010, who crystallized the TLR-4 receptor. 
In this very useful work he dissected the TLR4(PDB: 3FXI) into 3 parts, since it was too large and unhandy to be crystallized at once..
</P><H3><SPAN class="mw-headline" id="stabilizing_the_structure">stabilizing the structure</SPAN></H3><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" align="right"><TBODY><TR><TD><P>Kim, Ho Min 2007: TLR-4 receptor in 3 pieces for crystallisation. Each piece was capped by Hagfish domains. Afterwards all three pieces were assembled by superimposition.
</P></TD></TR></TBODY></TABLE><P>To overcome this problem of an exposed hydrophobic core, he used the N- and C-terminal protein fragments of a LRR protein derived from hagfish. They tried a variety of different versions of how to glue together his fragments and these N- and C-terminal caps until they found a working one.
</P><P>We used this knowledge for our purpose, applied the same sequences and attached them to our protein sequence. These caps do still in parts show the typical LRR consensus sequences (which luckily is highly conserved in all kingdoms of life!), which made it possible to fit them onto our stack of LRR loops in the right position. 
</P><P>To the outer end the protein has a helix on the N-terminal side, shielding off the core and the C-terminus while the LRR disappears slowly turn by turn with more and more hydrophilic amino acids replacing the LRR consensus sequence.
</P><H3><SPAN class="mw-headline" id="optimzing_the_sequence">optimzing the sequence</SPAN></H3><P>After combining these caps with our structural analysis for possible Nickel binding sites and filling the empty positions of the consensus sequence with the ideal sequence derived from the ligase sequence (see above), we were still unsure whether this would work out. To increase our chances of success we designed six different versions with distinct Histidine patterns on the LRR core.
</P><TABLE style="color:black; background-color:lightgrey;" cellpadding="15%" cellspacing="0" border="1" width="85%"><TBODY><TR><TD style="width: 100%;background-color:lightgrey;"><P>Six different Histidine patterns inserted into our ideal LRR sequence, capped by Hagfish domain fragments were sent for I-TASSER structure prediction. C-Score shown under the sequence.
</P></TD></TR></TBODY></TABLE><P>After evaluating the C-Scores of I-TASSER we saw that our favorite versions 1,2 and 4 gave acceptable scores. We then reverse translated them with a codon usage optimized for <I>E. coli</I>, using <A href="http://www.bioinformatics.org/sms2/rev_trans.html" class="external free" rel="nofollow">http://www.bioinformatics.org/sms2/rev_trans.html</A>
The sequence was handed over to our sponsor ATG:biosynthetics, who ran their own analysis on it, to optimize the sequence for RNA trafficking and secondary structures. In pictures below the alignment of our reverse translated sequence and the returned sequence is shown: ours above, ATG:biosynthetics sequence below. Underlined in red are the mutations the company introduced to optimize the expression. They also were so kind as to synthesize the three genes for us as a gift.
</P><TABLE style="color:black; background-color:lightgrey;" width="620px" cellpadding="15%" cellspacing="0" border="1" align="center"><TBODY><TR><TD style="width=200px;"><FONT size="4">Rational 1</FONT></TD><TD style="width=200px;"><FONT size="4">Rational 2</FONT></TD><TD style="width=200px;"><FONT size="4">Rational 4</FONT></TD></TR><TR><TD><FONT size="0.1">ATGtgcccgagccgttgcagctgtagcgg
<P>caccgaaattcgctgcaacagcaaaggcc
tgaccagcgtgccgaccggcattccga
gcagcgcgacccgcctggaactggaaag
caacaaactgcaaagcctgccggatggcgt
gtttgataaactgaccgcgctgcataaaag
caacaaccagctgaccagcctgccggataac
ctgccggcgagcctggaacatctggcggtg
agcaacaaccagctgaccagcctgccggat
aacctgccggcgagcctggaagcgctgcat
gtgagcaacaaccagctgaccagcctgccg
gataacctgccggcgagcctggaacatctg
gcggtgagcaacaaccagctgaccagcctg
ccggataacctgccggcgagcctggaagcg
ctgcatgtgagcaacaaccagctgaccagc
ctgccggataacctgccggcgagcctggaa
catctggcggtgagcaacaaccagctgacc
agcctgccggataacctgccggcgagcctg
aaagcgctgcatctggataccaaccagctg
aaaagcgtgccggatggcatttttgatcgc
ctgaccagcctgcaaaaaatttggctgcaa
accaacccgtgggattgcagctgcccgcgc
attgattatctgagccgctggctgaacaaa
aacagccagaaagaacagggcagcgcgaa
atgcagcggcagcggcaaaccggtgcgca
gcattatttgcccgTAGTAA</P></FONT></TD><TD style="width=200px;"><P><FONT size="0.1">ATGtgcccgagccgttgcagctgtagcgg
caccgaaattcgctgcaacagcaaaggcct
gaccagcgtgccgaccggcattccgagcag
cgcgacccgcctggaactggaaagcaacaa
actgcaaagcctgccggatggcgtgtttga
taaactgacccagctgaccaaaagcaacaa
ccatctgcatagcctgccggataacctgcc
ggcgagcctggaagtgctggatgtgagcaa
caaccatctgcatagcctgccggataacct
gccggcgagcctggaagtgctggatgtgag
caacaaccatctgcatagcctgccggataa
cctgccggcgagcctggaagtgctggatgt
gagcaacaaccatctgcatagcctgccgga
taacctgccggcgagcctggaagtgctgga
tgtgagcaacaaccatctgcatagcctgcc
ggataacctgccggcgagcctggaagtgct
ggatgtgagcaacaaccatctgcatagc
ctgccggataacctgccggcgagcctga
aagaactggcgctggataccaaccatctgca
tagcgtgccggatggcatttttgatcgcctg
accagcctgcaaaaaatttggctgcaaacca
acccgtgggattgcagctgcccgcgcattga
ttatctgagccgctggctgaacaaaaacagc
cagaaagaacagggcagcgcgaaatgcagcg
gcagcggcaaaccggtgcgcagcattatttg
cccgTAGTAA</FONT></P></TD><TD style="width=200px;"><P><FONT size="0.1">ATGtgcccgagccgttgcagctgtagc
ggcaccgaaattcgctgcaacagcaa
aggcctgaccagcgtgccgaccggcat
tccgagcagcgcgacccgcctggaactg
gaaagcaacaaactgcaaagcctgccgg
atggcgtgtttgataaactgacccagct
gaccaaaagcaacaaccagctgaccagc
ctgccggatcatctgccgcatagcctgg
aagtgctggatgtgagcaacaaccagct
gaccagcctgccggatcatctgccgcat
agcctggaagtgctggatgtgagcaaca
accagctgaccagcctgccggatcatct
gccgcatagcctggaagtgctggatgtg
agcaacaaccagctgaccagcctgccgg
atcatctgccgcatagcctggaagtgct
ggatgtgagcaacaaccagctgaccagc
ctgccggatcatctgccgcatagcctgg
aagtgctggatgtgagcaacaaccagct
gaccagcctgccggatcatctgccgcat
agcctgaaagaactggcgctggatacca
accagctgaaaagcgtgccggatcatat
ttttcatcgcctgaccagcctgcaaaaa
atttggctgcaaaccaacccgtgggatt
gcagctgcccgcgcattgattatctgag
ccgctggctgaacaaaaacagccagaaa
gaacagggcagcgcgaaatgcagcggca
gcggcaaaccggtgcgcagcattatttg
cccgTAGTAA</FONT></P></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="A_mathematical_model_to_determine_the_experimental_design">A mathematical model to determine the experimental design</SPAN></H3><P>The modeling was done in order to realize what parameters are crucial and which experiments need to be conducted to determine those.
</P><P>We asked the following questions:
</P><P>How much Precipitator protein will bind to the Plastic surface? How stable is the binding of the plastic binding tag to polystyrene depending on the size of the column? How much His-tagged protein will the Precipitator be able to bind? Which parameters do we have to determine? How often can one column be reused until the efficiency goes too low due to unbinding of the Precipitator protein from the plastic surface? How can we find out the affinity constants?
We developed a Flow-chart diagram to make the reactions that happen before, during and after the washing steps visible.
Subsequently we developed chemical reaction equations and formulate differential equations after them. Here appeared the parameters we had to determine. After carefully considering these equations, we introduce several simplifications, either because the expected information was marginal, or because the information was experimentally not accessible.
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Math101.gif" title="File:Math101.gif">File:Math101.gif</A></DIV></DIV><DIV class="center"><DIV class="floatnone"><A href="/File:Math002.gif" title="File:Math002.gif">File:Math002.gif</A></DIV></DIV><DIV class="center"><DIV class="floatnone"><A href="/File:Math003.gif" title="File:Math003.gif">File:Math003.gif</A></DIV></DIV><DIV class="center"><DIV class="floatnone"><A href="/File:Math009.gif" title="File:Math009.gif">File:Math009.gif</A></DIV></DIV><DIV class="center"><DIV class="floatnone"><A href="/File:Math010.gif" title="File:Math010.gif">File:Math010.gif</A></DIV></DIV><DIV class="center"><DIV class="floatnone"><A href="/File:Math011.gif" title="File:Math011.gif">File:Math011.gif</A></DIV></DIV><DIV class="center"><DIV class="floatnone"><A href="/File:Math012.gif" title="File:Math012.gif">File:Math012.gif</A></DIV></DIV><DIV class="center"><DIV class="floatnone"><A href="/File:Math013.gif" title="File:Math013.gif">File:Math013.gif</A></DIV></DIV><DIV class="center"><DIV class="floatnone"><A href="/File:Math014.gif" title="File:Math014.gif">File:Math014.gif</A></DIV></DIV><DIV class="center"><DIV class="floatnone"><A href="/File:Math015.gif" title="File:Math015.gif">File:Math015.gif</A></DIV></DIV><H3><SPAN class="mw-headline" id="Conclusions">Conclusions</SPAN></H3><P>To determine the Affinity k_D, experiments to find out the binding affinity of the plastic binding domain are necessary. To get a direct access to these values, we cloned the plastic binding domain in front of a GFP. Then, dilution and washing assays could be performed on polystyrene microtiter plates, red out by a fluorescence plate reader. 
The desired parameters could be calculated by measuring dilution rows of GFP proteins and measuring the fluorescence signals at the different concentrations. C_total could be determined by a dilution row with subsequent washing steps, to find out at what [P] concentration there is a saturation. See description of the plastic binding subproject for more detailed explanation on the experimental setup.
</P><P>A qualitative experiment to prove that Nickel is binding the Precipitator is sufficient, since k_2 &gt;&gt; 1 and does not play a significant role in our setup. This experiment could have been done using a nanofilter that blocks protein but let through ions. The Nickel concentration of the flow through can then be measured. 
</P><P>Alternatively purification of the Precipitator by fusing it with a GST-tag could be done, to subsequently measure the absorbance of the protein, before and after adding Nickel to the solution. After Jordan 1974 a detectable change in the absorbance should be detectable after the complex is formed. A similar effect – a colorshift from white to blue -  is visible when one prepares a Ni-NTA column. For this purpose we cloned the GST domain in front of the Precipitator. However towards the end of the project there was no more time to perfom these experiments.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>Tiandi Wei et.al.; &quot;LRRML: a conformational database and an XML description of
leucine-rich repeats (LRRs)&quot;; BMC Structural Biology 2008 doi: 10.1186/1472-6807-8-47 
</P><P>
Schmidt, Marc et.al.; &quot;Crucial role for human Toll-like receptor 4 in the development of contact allergy to nickel&quot;;
Nature 2010
</P><P>
Letter, J.E. et.al.; &quot;Complexing of Nickel( 11) by Cysteine, Tyrosine,
and Related Ligands and Evidence for Zwitterion Reactivity&quot;;
Contribution from the Department of Chemistry, University of Alberta,
Edmonton, Alberta T6G 2G2, Canada.
</P><P>
Kajava, A.V.;&quot;Structural Diversity of Leucine-rich Repeat Proteins&quot;
J. Mol. Biol. (1998) 277, 519±527
</P><P>
Kim, Ho Min et. al.; &quot;Crystal Structure of the TLR4-MD-2 Complex with Bound Endotoxin Antagonist Eritoran&quot;
DOI 10.1016/j.cell.2007.08.002
</P><P>
Nieba, L. et al BIACORE analysis of histidine-tagged proteins using a chelating NTA sensor chip. Analytical Biochemistry 252: 217-228; (1997).
</P></DIV></DIV><DIV align="center">                  </DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Freiburg/Modelling">http://2011.igem.org/Team:Freiburg/Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Modelling&amp;oldid=169000" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>