"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/Notebook/7 September
From 2011.igem.org
Contact
This is the wiki page of the Freiburger student team competing for iGEM 2011. Thank you for your interest!
< home >
< team >
→ members
→ attributions
→ freiburg
< project >
→ description
→ results
→ modeling
→ notebook
→ sample data
< human practices >
→ oath
→ ethics
→ safety
< gallery >
< sponsors >
< to the forums >
Previous entry
7 September
Next entry
Contents
1 Commons
1.1 Ligation
1.2 Trafo
1.3 PCR of psB1C3
2 Green light receptor
3 blue light receptor
3.1 no colonies on the plates
3.2 Miniprep
3.3 PCR
4 Lysis cassette
4.1 Troubleshooting of the modified Lysis genes K124017
4.1.1 PCR Purification and Gel documentation
4.1.2 Ligation and Transformation
5 Precipitator
5.1 Trafo
5.2 Ligation
Commons
Ligation
Investigators: Sandra
Ligation of PR1-PR6 with RFP (S1b and S2a). PR1-PR6 were digested with antarctica before start of ligation. Start of ligation at 12:40. Incubation at room temperature for at least 3 hours.
Trafo
Investigators: Sandra
Trafo of:
PR1+GFP
PR2+GFP
PR3+GFP
PR4+GFP
PR5+GFP
PR6+GFP
PR1+50b
PR2+50b
PCR of psB1C3
Investigators: Julia
Green light receptor
PCR
Name: Julia
 
Date: 7.09.2011
Project Name: PcpcG
 
PCR-Mixture for one Reaction:
For a 50 µl reaction use
32,5µl
H20
10µl
5x Phusion Buffer
2.5µl
Primer     up
 tatgaattcgcggccgcttctagaCCATTGTGCTTTTCTCTATCAACC
2.5µl
Primer     dw
 tatctgcagcggccgctactagtaACTTAAAAGTTGTTTAATGTCCAGCC
1µl
dNTPs
1µl
DNA-Template
 Synechocystis genome
0.5 µl
Phusion (add in the end)
 
 
What program do you use?
  Temperature Time ( min)
94 C
5:00
94 C
00:30
30x
56 C
00:30
72 C
1:00
72 C
7:00
To confirm the PCR-Product has the correct size, load 2 µl of the sample onto an agarose-gel.
 Size of expected gene : 238 bp
blue light receptor
no colonies on the plates
There were no colonies on the plates with ligated parts of the 2A-assembly.
Miniprep
Investigators: Sophie
yesterday two tubes with LB were inoculated with cells from our S35 glycerol stock(BBa_K322999). Today minipreps were performed.
PCR
Investigators: Sophie
Name: Sophie
Date: 07.09.11
Continue from Experiment (Date)
(Name)
Project Name: Gibson of ♥ and NOT
PCR-Mixture for one Reaction:
For a 50 µl reaction use
32,5µl
H20
Name
10µl
5x Phusion Buffer
of Primer
2.5µl
Primer fw
♥: P97
NOT: P99
2.5µl
Primer dw
♥: P98
NOT: P100
1µl
dNTPs
of Template DNA
1µl
DNA-Template
M45a,b,c (BBa_Q0400)
Miniprep of M35 (1x) and M35b (2x) (BBa_K322999)
0.5 µl
Phusion (add in the end)
Digestion with Dpn I and purification with PCR purification kit What program do you use?
First 20 cycles touchdown 69°C -0.4/cycle
next 10 cycles touchdown 72°C -0.2 per cycle
PCR products were loaded onto a gel
Lysis cassette
Troubleshooting of the modified Lysis genes K124017
Investigators:Theo
S4+S15 Nr1 and 7 were digested with XbaI and PstI, loaded on a 1% agarose gel and extracted on the 6th of September.
Today I digested the quickchanged-temperature sensitive promotor (K098995) with SpeI and PstI for ca 3 hours, incubated it with antarctic phosphatase for 1 hour and PCR-purified it.
PCR Purification and Gel documentation
PCR Purification Kit
Qiagen Kit
Name: Theo
Date: 7.9.2011
Continue from Experiment : Troubleshooting of the modified Lysis genes K12401 (7.9.2011, Theo)
Project Name: Troubleshooting of the modified Lysis genes K12401
Procedure
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix. It is not necessary
to remove mineral oil or kerosene.
For example, add 500 μl of Buffer PB to 100 μl PCR sample (not including oil).
If pH indicator I has beein added to Buffer PB, check that the color of the mixture is yellow.
If the color of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH
5.0, and mix. The color of the mixture will turn to yellow.
Place a QIAquick spin column in a provided 2 ml collection tube.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube.
Collection tubes are re-used to reduce plastic waste.
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless
the flow-through is discarded before this additional centrifugation.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB (10 mM Tris·Cl, pH 8.5) or water (pH 7.0–8.5) to
the center of the QIAquick membrane and centrifuge the column for 1 min. Alternatively,
for increased DNA concentration, add 30 μl elution buffer to the center of the QIAquick
membrane, let the column stand for 1 min, and then centrifuge.
IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volume is 48 μl from 50 μl elution buffer volume, and 28 μl from 30 μl elution buffer. Elution efficiency is dependent on pH. The maximum elution efficiency is achieved
between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C as DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE buffer (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions.
If the purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to
5 volumes of purified DNA. Mix the solution by pipetting up and down before
loading the gel. Loading dye contains 3 marker dyes (bromophenol blue, xylene cyanol, and orange G) that facilitate estimation of DNA migration distance and optimization of agarose gel run time. Refer to Table 2 (page 15) to identify the dyes according to migration distance and agarose gel percentage and type.
Documentation:
Why are you doing this experiment? Name the parts which you cut out.
To get rid of the small fragment cut with SpeI and PstI from the M48 part (quickchanged-temperature sensitive promotor (K098995)), so that it doesn’t religate with the vector.
Insert Gel Picture here and mark the bands you will excise. Describe the used ladder and sizes of the bands.
Describe your results and mistakes and measure the DNA concentration with the Nanodrop and note the results. Also make a note of the ratio between A260/A280 for each sample.
M48: Concentration= 13ng/mikrol and A260/280=2,13
Ligation and Transformation
The amount used for the gel was 5mikrol, so this corresponds to about 60ng DNA from the M48 sample.
One can see that the concentration of M66 cut with EcoRI and PstI is about 2-3 times higher (cannot be measured because of the agarose from the gel extraction).
The concentration of S4+S15 Nr1 should be about 1/2 of that of M48, so I am going to use 2x and 6x the amount of M48 for the ligation.
There is no band to be seen for S4+S15 Nr7, so the concentration is lower than 5ng/mikrol, so I am going to use all of it for the ligation (ca 25mikrol) and change the reaction concentrations accordingly.
Procedure
 
PCR tube:
total volume 20 μl
 
1. add H2O                                 (17 μl -X-Y-Z)
2. add 2 μl Ligase Buffer 10x
3. add Insert   
4. add Vector                           (20ng needed)
5. Add 1 μl T4-DNA Ligase
6. Incubate  10-30 min at room temperature
 
Precipitator
Trafo
Investigators: Sandra
Trafo of the already ligated parts of the pbd + Gst-vector
Ligation
Investigators: Sophie
For pbd in iGEM C3-vector: again Ligation (see 31.08.11) because I don't find the last ligation product that produced positive clones. I have to redo the Trafo because the last positive clones seem to have mutated over the last generations, probably because the pbd is toxic.
labelled: L S54-ε5-C3 1:1 / 1:3
stored in "Ligationsreaktionen"-box
   
   
   
   
   
   
Retrieved from " http://2011.igem.org/Team:Freiburg/Notebook/7_September "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
