<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_MIT_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:MIT/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:MIT/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:MIT/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:MIT/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="project_title" style="text-align: center; margin-top: -0.8em; float:left; z-index:1; position:relative;
margin-left:3em;"><A href="https://2012.igem.org/Team:MIT"><H1 style="border: none; line-height: 1.2em; margin: 0; padding: 0; padding-bottom: 0.5em;">

RNA Strand Displacement for Sensing, Information Processing,and Actuation in Mammalian Cells
</H1></A></DIV></DIV><UL class="nav"><LI id="nav_motivation"><A href="https://2012.igem.org/Team:MIT/Motivation">MOTIVATION</A></LI><LI id="nav_results"><A href="https://2012.igem.org/Team:MIT/ResultsOverview">RESULTS</A></LI><LI><A href="https://2012.igem.org/Team:MIT/ResultsOverview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:MIT/CircuitProduction">Circuit Production</A></LI><LI><A href="https://2012.igem.org/Team:MIT/NOTGate">NOT Gate</A></LI><LI><A href="https://2012.igem.org/Team:MIT/Sensing">Sensing</A></LI><LI><A href="https://2012.igem.org/Team:MIT/Actuation">Actuation</A></LI><LI><A style="background: #993333;" href="https://2012.igem.org/Team:MIT/TheKeyReaction"><EM>Enabling The Key Reaction</EM></A></LI><LI><A href="https://2012.igem.org/Team:MIT/ResultsBiobricks">BioBricks</A></LI><LI><A href="https://2012.igem.org/Team:MIT/ResultsAttributions">Attributions</A></LI></UL><LI id="nav_methods"><A href="https://2012.igem.org/Team:MIT/MaterialsAndMethods">METHODS</A></LI><LI id="nav_notebook"><A href="https://2012.igem.org/Team:MIT/Notebook">NOTEBOOK</A></LI><LI><A href="https://2012.igem.org/Team:MIT/Notebook#constructionbio">Construction &amp;<EM>In Vitro</EM></A></LI><LI><A href="https://2012.igem.org/Team:MIT/Notebook#springtcbio">Spring Tissue Culture</A></LI><LI><A href="https://2012.igem.org/Team:MIT/Notebook#summertcbio">Summer Tissue Culture</A></LI><LI><A href="https://2012.igem.org/Team:MIT/Notebook#falltcbio">Fall Tissue Culture</A></LI><LI id="nav_human_practices"><A href="https://2012.igem.org/Team:MIT/HumanPractices">HUMAN PRACTICES</A></LI><LI id="nav_safety"><A href="https://2012.igem.org/Team:MIT/Safety">SAFETY</A></LI><LI id="nav_team"><A href="https://2012.igem.org/Team:MIT/Team">TEAM</A></LI><LI id="nav_partners"><A href="https://2012.igem.org/Team:MIT/Partners">SPONSORS</A></LI><LI id="nav_contact"><A href="https://2012.igem.org/Team:MIT/Contact">CONTACT US</A></LI><DIV id="col_nav"><DIV id="accordion"><H3 id="construction_h3"><A href="#">Construction and <EM>In Vitro</EM></A></H3><DIV class="col_list"><UL><LI id="construction">Construction and <EM>In Vitro</EM> Studies</LI></UL></DIV><H3><A href="#">Tissue Culture</A></H3><DIV class="col_list"><UL><LI id="springtc">Spring</LI><LI id="summertc">Summer</LI><LI id="falltc">Fall</LI></UL></DIV></DIV></DIV><DIV id="col_left"><DIV class="section" id="overviewbio"><H1> Notebook Overview </H1>

Our team has been working hard on our project for many months, and we are proud to share our exciting results with the iGEM community! Here we present a summary of our daily progress and experiments. 
Starting in the 2012 spring semester, our team learned laboratory technique and began experiments. Over the summer, the full 16 member team - including one undergraduate student from another university - worked full-time on the project. Currently, during the 2012 fall semester, many members of the team are continuing to invest time and conduct experiments for the project through the Undergraduate Research Opportunities Program at MIT.

</DIV><DIV class="section" id="constructionbio"><H1>Construction and <EM>In Vitro</EM> Lab Notebook</H1><TABLE border="0" cellpadding="0" cellspacing="0" id="notebook"><TBODY><TR><TH class="date">Date</TH><TH class="name">Team member</TH><TH class="experimentns">Experiments</TH></TR><TR><TD class="date">3/5/2012</TD><TD class="name">RL</TD><TD class="experiments">Restriction mapping pBACe with EcoR1</TD></TR><TR><TD class="date">3/7/2012</TD><TD class="name">RL</TD><TD class="experiments">Made 1% agarose Gels</TD></TR><TR><TD class="date">3/7/2012</TD><TD class="name">RL</TD><TD class="experiments">Ran gel of pBACe digest. NB. Make sure gel is in correct orientation</TD></TR><TR><TD class="date">3/12/2012</TD><TD class="name">RL</TD><TD class="experiments">Ran gel again</TD></TR><TR><TD class="date">3/14/2012</TD><TD class="name">RL</TD><TD class="experiments">Made antibiotic plates</TD></TR><TR><TD class="date">3/20/2012</TD><TD class="name">RL</TD><TD class="experiments">Cultured competent Cells</TD></TR><TR><TD class="date">3/27/2012</TD><TD class="name">RL</TD><TD class="experiments">pBACe 3,6 Gibson, electroporated into cells.</TD></TR><TR><TD class="date">3/29/2012</TD><TD class="name">RL</TD><TD class="experiments">created primers for new pBACe3.6 experiment</TD></TR><TR><TD class="date">3/30/2012</TD><TD class="name">RL</TD><TD class="experiments">grew cultures for miniprep</TD></TR><TR><TD class="date">3/30/2012</TD><TD class="name">RL</TD><TD class="experiments">annealed andrew's oligions in PCR machine</TD></TR><TR><TD class="date">4/2/2012</TD><TD class="name">RL</TD><TD class="experiments">Made and aliquoted SOC</TD></TR><TR><TD class="date">4/4/2012</TD><TD class="name">RL</TD><TD class="experiments">Ran an analytic gel of 224 and 226.Hyperladder I, empty well, 226 G, 226 PI-SceI, 224 G, 224 PI-SceI, started gel at 100V at 9:40, run for 25 mins, finish at 10:05</TD></TR><TR><TD class="date">4/10/2012</TD><TD class="name">RL</TD><TD class="experiments">Inoculated culture 1 from -80C iGem box using 1000:1 Amp (10 ul for 10 ml), split into 2 x 5ml tubes. Put in 37 C shaker at 9:00 PM</TD></TR><TR><TD class="date">4/10/2012</TD><TD class="name">EA</TD><TD class="experiments">Made 1% agarose Gels</TD></TR><TR><TD class="date">4/11/2012</TD><TD class="name">RL</TD><TD class="experiments">miniprep- CMV:Amcyan-miFF4 LacI colony 1. Practiced running a gel.</TD></TR><TR><TD class="date">4/23/2012</TD><TD class="name">RL</TD><TD class="experiments">Used genious to model pBACe3.6 + J85224/226 forward and backwards to check for useful cut sites. For 224: Decided on NdeI to easily determine whether the part was inserted correctly or reversed.</TD></TR><TR><TD class="date">4/25/2012</TD><TD class="name">RL</TD><TD class="experiments">For 226 it looks like either NcoI or NdeI would work - Probably go with NdeI for consistency with 224. Did them by hand in Genious for pBACe3.6 + 224. They do not form any hairpins and have Tms of 50 and 58 CBecause the only variation between pBace3.6 + 224, 226, both forwards and reversed is in the section we're trying to sequence, these two primers should be sufficient to sequence each strand. </TD></TR><TR><TD class="date">5/1/2012</TD><TD class="name">RL</TD><TD class="experiments">FACS training </TD></TR><TR><TD class="date">5/7/2012</TD><TD class="name">RL</TD><TD class="experiments">Finished designing primers to check the insertions for the pBACe 3.6 gene. Some slight adjustments were made to correct for self annealing in the backwards primer as well as the melting temperature of the first. Starting work continuing the sequencing of pJAZZ. Reading Felix's notes and going over his sequencing progress before beginning.</TD></TR><TR><TD class="date">5/12/2012</TD><TD class="name">KG</TD><TD class="experiments">Made 1% agarose Gels</TD></TR><TR><TD class="date">5/15/2012</TD><TD class="name">RL</TD><TD class="experiments">More pJazz sequencing. </TD></TR><TR><TD class="date">3/27/2012</TD><TD class="name">AS</TD><TD class="experiments">Diluted pUC 19 DNA to 50 pg/uL. Transfromed. Pepared gels.</TD></TR><TR><TD class="date">4/1/2012</TD><TD class="name">AS</TD><TD class="experiments">Calculated competency of bacteria for transformation. Gel extractions. Made gels. Introduction to cell culture.</TD></TR><TR><TD class="date">3/21/2012</TD><TD class="name">NK</TD><TD class="experiments">Prepared Z-competent 10G E.Coli using the Zymo protocol</TD></TR><TR><TD class="date">4/4/2012</TD><TD class="name">NK</TD><TD class="experiments">Prepared LB + Agar for culture plates. Made 1% gels. Set up 4 PCR reactions for - PCR of J85224, J85226 with J85419, J85418 (To Gibson into PI-SceI site)- PCR of J85224, J85226 and J85430, J85431 (to add PI-SceI cutsites) .</TD></TR><TR><TD class="date">4/6/2012</TD><TD class="name">NK</TD><TD class="experiments">DNA minipreps on qiacube.</TD></TR><TR><TD class="date">4/9/2012</TD><TD class="name">NK</TD><TD class="experiments">Tissue Culture refresher with Ken.</TD></TR><TR><TD class="date">3/13/2012</TD><TD class="name">KB</TD><TD class="experiments">Inoculate/Grow cells</TD></TR><TR><TD class="date">3/14/2012</TD><TD class="name">KB</TD><TD class="experiments">Miniprep using QiaCube robot.</TD></TR><TR><TD class="date">3/15/2012</TD><TD class="name">KB</TD><TD class="experiments">Gateway reaction of pENTR_L4_pCMV_R1:AmCyan-mirFF4-L2; Transformation</TD></TR><TR><TD class="date">3/16/2012</TD><TD class="name">KB</TD><TD class="experiments">morning: check for colonies</TD></TR><TR><TD class="date">3/19/2012</TD><TD class="name">KB</TD><TD class="experiments">Design restriction map and restriction digest the Input CMV and AM CYAN ; Start Culture for Gateway Products Miniprep </TD></TR><TR><TD class="date">3/20/2012</TD><TD class="name">KB</TD><TD class="experiments">Gateway Product Miniprep ; Nanodrop</TD></TR><TR><TD class="date">3/21/2012</TD><TD class="name">KB</TD><TD class="experiments">Restriction Digest of Gateway Product ; run gel of Gateway input and product</TD></TR><TR><TD class="date">4/2/2012</TD><TD class="name">KB</TD><TD class="experiments">Restriction Digest of CMV promoter using Lincoln Lab's protocol ; Run gel ; Begin Gateway Reaction with new inputs</TD></TR><TR><TD class="date">4/4/2012</TD><TD class="name">KB</TD><TD class="experiments">Making plates (3:00-3:40), Transformation of Gateway Reaction 2 (3:40-5:40)</TD></TR><TR><TD class="date">4/5/2012</TD><TD class="name">KB</TD><TD class="experiments">morning: check for colonies (11:36 AM), afternoon: inoculate cells (4:11 PM)</TD></TR><TR><TD class="date">4/9/2012</TD><TD class="name">KB</TD><TD class="experiments">Restriction Digest of (gateway input) pENTR_L1_AmCyan_mirFF4-2A-LacI_L2, Restriction Digest of (gateway output) pZDonor 1-GTW-2r-pCMV-AmCyan-mirFF4-2A-LacI (4:50 PM to 5:40 PM) ; gel electrophoresis (5:49 PM to 6:09 PM)</TD></TR><TR><TD class="date">4/10/2012</TD><TD class="name">KB</TD><TD class="experiments">Redo Restriction digest of gateway input and 6 outputs (let digest sit for 3 hours) (3:30-7:00 PM) , gel extraction for Jon (4:00-5:00 PM)</TD></TR><TR><TD class="date">4/11/2012</TD><TD class="name">KB</TD><TD class="experiments">Run gel 2X for gateway expression vector / AmCyan-LacI Input, 5 part Gibson Assembly, transformation, making gels </TD></TR><TR><TD class="date">4/16/2012</TD><TD class="name">KB</TD><TD class="experiments">(3:00 - 6:00 PM) - Finish Restriction Digest of Minipreps of Culture 1 from Gateway of CMV (3-3:30): AmCyan-miRFF4-LacI for Keren, Observe Ken passage cells (3:30 - 4), Meeting with Giulio and Jameel (5-6)</TD></TR><TR><TD class="date">4/17/2012</TD><TD class="name">KB</TD><TD class="experiments">Run Gel of CMV:AmCyan-MirFF4-LacI Digest (3:05-3:30 PM), Make Gels (5:20-5:45)</TD></TR><TR><TD class="date">4/18/2012</TD><TD class="name">KB</TD><TD class="experiments">Inoculate pBACe3.6 colonies </TD></TR><TR><TD class="date">4/20/2012</TD><TD class="name">KB</TD><TD class="experiments">Split HEK cells 1:3</TD></TR><TR><TD class="date">4/23/2012</TD><TD class="name">KB</TD><TD class="experiments">Split HEK Cells 1:3 (3-3:30 PM)</TD></TR><TR><TD class="date">4/24/2012</TD><TD class="name">KB</TD><TD class="experiments">Transfection of ROX+RNA gate and pEXPR Hef1a:Cerulean  (5:00 PM - 8:00 PM), Split HEK Cells 1:6 (6-6:15 PM)</TD></TR><TR><TD class="date">4/25/2012</TD><TD class="name">KB</TD><TD class="experiments"> Split experimental culture 1:6, Microscopy of 4 Transfections from April 24th</TD></TR><TR><TD class="date">4/26/2012</TD><TD class="name">KB</TD><TD class="experiments">(3:30 - 7:30 PM) - FACS of 4 transfections from April 24th, 5 new transfections, some microscopy, Split Culture</TD></TR><TR><TD class="date">4/27/2012</TD><TD class="name">KB</TD><TD class="experiments">(4:15-4:40 PM) - Split HEK cells 1:12 </TD></TR><TR><TD class="date">4/20/2012</TD><TD class="name">KB</TD><TD class="experiments">Split HEK cells 1:6</TD></TR><TR><TD class="date">5/2/2012</TD><TD class="name">KB</TD><TD class="experiments">Split HEK cells 1:3</TD></TR><TR><TD class="date">5/3/2012</TD><TD class="name">KB</TD><TD class="experiments">Transfection of ROX with Ala'a, Split HEK cells 1:3</TD></TR><TR><TD class="date">5/4/2012</TD><TD class="name">KB</TD><TD class="experiments">Split HEK Cells 1:12 </TD></TR><TR><TD class="date">5/7/2012</TD><TD class="name">KB</TD><TD class="experiments">Split HEK Cells 1:3</TD></TR><TR><TD class="date">5/8/2012</TD><TD class="name">KB</TD><TD class="experiments">Repeat of 5 Transfections from April 26th, Split HEK Cells 1:12 </TD></TR><TR><TD class="date">5/10/2012</TD><TD class="name">KB</TD><TD class="experiments">Prepared Transfected Cells for FACS; Ran FACS with Deepak</TD></TR><TR><TD class="date">5/15/2012</TD><TD class="name">KB</TD><TD class="experiments">Ran 5 Transfections again but with cells from Nathan, metafectine and a new annealed gate:output</TD></TR><TR><TD class="date">6/11/2012</TD><TD class="name">KB</TD><TD class="experiments">Split cells 1:3 and 1:6</TD></TR><TR><TD class="date">4/10/2012</TD><TD class="name">KEK</TD><TD class="experiments">Prepared for experiments by looking up details and becoming aquainted with machines available (plate reader). Resuspended Oligos</TD></TR><TR><TD class="date">4/12/2012</TD><TD class="name">KEK</TD><TD class="experiments">Made serial dilutions of ROX part (1 uM - 1pM).  Used Lu Lab Plate reader to measure fluorescence. </TD></TR><TR><TD class="date">4/17/2012</TD><TD class="name">KEK</TD><TD class="experiments">Made 50 uL 100 uM aliquots of the RNA parts (S1, S6, Gate, Output). Annealed gate.</TD></TR><TR><TD class="date">4/18/2012</TD><TD class="name">KEK</TD><TD class="experiments">Determined detection limit of plate reader via serial dilutions of DOX. Prepared 1x TAE 12.5 mM Mg2+ (filtered). We have quenching!</TD></TR><TR><TD class="date">4/19/2012</TD><TD class="name">KEK</TD><TD class="experiments">Appear to have in vitro RNA strand-displacement. </TD></TR><TR><TD class="date">4/24/2012</TD><TD class="name">KEK</TD><TD class="experiments">Plate reader in use</TD></TR><TR><TD class="date">5/1/2012</TD><TD class="name">KEK</TD><TD class="experiments">FACS orientation, made 100 nM gate, secondary analysis of 4/26/2012 results. Performed a new kinetic experiment</TD></TR><TR><TD class="date">5/3/2012</TD><TD class="name">KEK</TD><TD class="experiments">Took endpoint measurement for experiments from 5/01/2012. Tried strand displacement at 37 &amp;deg;C, but temperature dropped and stopped experiment. Made more 100 nM stock of gate:output. Set up ladder experiment protocol for plate reader. </TD></TR><TR><TD class="date">5/8/2012</TD><TD class="name">KEK</TD><TD class="experiments">Annealed gate:output at RT (final concentration 45.45 uM) Annealed gate:input_sg at RT (final concentratin 45.45 uM). Annealed gate:input_s6 using the heatblock. Having issues observing strand displacement.</TD></TR><TR><TD class="date">5/10/2012</TD><TD class="name">KEK</TD><TD class="experiments">Quantified gate:input anneals by measuring fluoresence vs. ROX. Didn't see any decrease in fluorescence -&gt; the annealing protocol shouldn't affect fluorescence of ROX.Quantified RT-annealed gate:output by doing strand displacement.</TD></TR><TR><TD class="date">5/13/2012</TD><TD class="name">KEK</TD><TD class="experiments">Set up annealing reactions on a plate reader for ROX concentrations of 2 nM, 5 nM and 10 nM. Measurements taken every 5 seconds for 30 minutes, then every 1 minute for &lt;12 hours. Some datapoints for the latter will be missing -- another UROP needed to do OD measuremens every 30 minutes so I let him do that.</TD></TR><TR><TD class="date">5/15/2012</TD><TD class="name">KEK</TD><TD class="experiments">Did a serial dilution of the wells from the strand displacement experiment to see if there a concentration effect to quenching (to answer the question whether we see increase in fluorescence when we dilute). Used ~10 nM sample from the plate, well G5. Made 5 nM, 1 nM, 100 pM, 10 pM dilutions with end-volume 100 µL.</TD></TR><TR><TD class="date">5/17/2012</TD><TD class="name">KEK</TD><TD class="experiments">Plan: Do strand displacement reactions with the various 100+101 tubes that we have to make sure they work.Refiltered the 1X TAE buffer stock just in case - it's been a month since it was made.</TD></TR><TR><TD class="date">6/8/2012</TD><TD class="name">GA</TD><TD class="experiments">LR reaction to make eYFP 4*luc expression vector </TD></TR><TR><TD class="date">6/9/2012</TD><TD class="name">GA</TD><TD class="experiments">Transformed E.coli cells with EYFP 4*luc expression vector</TD></TR><TR><TD class="date">6/10/2012</TD><TD class="name">GA</TD><TD class="experiments">Observed no colonies of transformed cells. Redo LR reaction to make eYFP 4*luc expression vector </TD></TR><TR><TD class="date">6/11/2012</TD><TD class="name">GA</TD><TD class="experiments">Transformed E.coli cells with EYFP 4*luc expression vector made via the second LR reaction</TD></TR><TR><TD class="date">6/12/2012</TD><TD class="name">GA</TD><TD class="experiments">Observed colonies. Stored in cold room</TD></TR><TR><TD rowspan="12" class="s2">7/11/2012</TD><TD class="name">RL, JE</TD><TD class="experiments">LR Expression Vectors  (Hef1A-eYFP-4xFF1) : Aligned sequence</TD></TR><TR><TD class="experiments">JK, WL</TD><TD class="experiments">LR Expression Vectors  (Hef1A-eYFP-4xFF1) : MIDIprep. Aligned sequence</TD></TR><TR><TD class="experiments">KG, JE</TD><TD class="experiments">1. Hef1A/CMV-TetR: Transformed vectors into E.coli cells2. FF4 construct: Transformed into E.coli cells</TD></TR><TR><TD class="experiments">JE,WL, </TD><TD class="experiments">RNA Expression (U6-TetO-FF1): Performed Sequencing Alignment</TD></TR><TR><TD class="experiments">JK, RL</TD><TD class="experiments">RNA Expression (U6-TetO-FF1): Inoculate for MIDIprep</TD></TR><TR><TD class="experiments">DI, LX</TD><TD class="experiments">Pour Plates (IPTG/XGAL)</TD></TR><TR><TD class="experiments">AS</TD><TD class="experiments">Miniprep GG donor vectors. Make cell stock</TD></TR><TR><TD class="experiments">DI, LX</TD><TD class="experiments">Transform GG donor vectors</TD></TR><TR><TD class="experiments">RL, CV</TD><TD class="experiments">Resuspend and anneal oligos</TD></TR><TR><TD class="experiments">LV, NS</TD><TD class="experiments">PCR golden gate entry vectors</TD></TR><TR><TD class="experiments">NS, DI</TD><TD class="experiments">PCR eBFP2 SP6</TD></TR><TR><TD class="experiments">KEK, WL</TD><TD class="experiments">Plate reader study using Alexa- Em/Ex DNA Gate</TD></TR><TR><TD rowspan="7" class="s2">7/12/2012</TD><TD class="name">JK, KMG</TD><TD class="experiments">Hef1A/CMV-TetR: Inoculate transformed cells</TD></TR><TR><TD class="experiments">JK</TD><TD class="experiments">RNA Expression (U6-TetO-FF1): MIDI</TD></TR><TR><TD class="experiments">AS, EA</TD><TD class="experiments">RNA Expression (U6-TetO-FF1): Restriction map/ send for sequencing</TD></TR><TR><TD class="experiments">WL</TD><TD class="experiments">Pour more Plates (IPTG/XGAL)</TD></TR><TR><TD class="experiments">WL, DI, NS</TD><TD class="experiments">Re-transform (DONR)Inoculate 50 mL (DONR)</TD></TR><TR><TD class="experiments">DI, NS</TD><TD class="experiments">SP6: Gel Check and Gel purify</TD></TR><TR><TD class="experiments">KEK, WL</TD><TD class="experiments">Plate reader study using Alexa- Em/Ex DNA Gate. Incubate with mRNA gates</TD></TR><TR><TD rowspan="5" class="s2">7/13/2012</TD><TD class="name">KMG</TD><TD class="experiments">LR Expression Vectors  (Hef1A-eYFP-4xFF1) : Take from cell stock and miniprep. Restriction map.</TD></TR><TR><TD class="experiments">RL</TD><TD class="experiments">Send for sequencing Hef1A/CMV-TetR</TD></TR><TR><TD class="experiments">JK, AS</TD><TD class="experiments">MIDIprep GG entry vectors</TD></TR><TR><TD class="experiments">DI</TD><TD class="experiments">Transform GG entry vectors</TD></TR><TR><TD class="experiments">DI, NS</TD><TD class="experiments">Use RNEasy kit to purify SP6</TD></TR><TR><TD rowspan="2" class="s2">7/14/2012</TD><TD class="name">JK</TD><TD class="experiments">Inoculate for MIDIprep of Hef1A/CMV-TetR</TD></TR><TR><TD class="experiments">DI</TD><TD class="experiments">Inoculate for Miniprep of GG entry vectors</TD></TR><TR><TD rowspan="2" class="s2">7/16/2012</TD><TD class="name">JK, DI</TD><TD class="experiments">1.Design new LRs expression vectors2. Hef1A/CMV-TetR ___Restriction Map (miniprep from 7/13), Send for Sequencing, Inoculate MIDI3. Actuation: FF4 Decoys/TuDs:  Design Restriction Maps (for all GG). Restriction Map (minipreps from 7/15). Send for Sequencing 4. eBFP2 1x-4x: Miniprep (inoculations from 7/15). Make Cell Stocks (of correctly restriction mapped samples)</TD></TR><TR><TD class="experiments">KEK, WL</TD><TD class="experiments">Plate reader study of Alexa- Em/Ex DNA Gate</TD></TR><TR><TD rowspan="4" class="s2">7/17/2012</TD><TD class="name">DI, KEK</TD><TD class="experiments">1. Design U6-S1, U6-S6 Expression vectors2. Hef1A/CMV TetR: Restriction Map (miniprep from 7/13), Send for Sequencing3. FF4: Restriction Map</TD></TR><TR><TD class="experiments">1. Re-design Restriction Maps (for all GG) 2. FF4 Decoys/TuDs: Restriction Map (minipreps from 7/16), Send for Sequencing  3. Make Cell Stocks (of correctly restriction mapped samples)</TD></TR><TR><TD class="experiments">DI, JK</TD><TD class="experiments">Re-invitro transcribe SP6 contruct</TD></TR><TR><TD class="experiments">KEK, WL</TD><TD class="experiments">Plate reader study: Alexa- Em/Ex DNA Gate. Incubate with mRNA gates</TD></TR><TR><TD rowspan="3" class="s2">7/18/2012</TD><TD class="name">KMG, JE</TD><TD class="experiments">Restriction maps ( CMV tetR, FF4 constructs). </TD></TR><TR><TD class="experiments">CV, JK</TD><TD class="experiments">In-vitro transcription of mRNA/ Redilute tetR entry vector. Set up new LR (CMV/Hef1A- TetR)</TD></TR><TR><TD class="experiments">DI, JK</TD><TD class="experiments"> Re IVT/Purify SP6-eBFP2</TD></TR><TR><TD rowspan="2" class="s2">7/19/2012</TD><TD class="name">KMG, JE, RL</TD><TD class="experiments">U6-TetO - S1/S6: Get in Primers U6-TetO-S1/S6. PCR + RCR purify U6. GG + Transform U6 </TD></TR><TR><TD class="experiments">CV, JK</TD><TD class="experiments">CMV/Hef1A tetR: transformed into E.coli</TD></TR><TR><TD rowspan="2" class="s2">7/20/2012</TD><TD class="name">KMG, JE, RL</TD><TD class="experiments">U6-TetO - S1/S6: Inoculate </TD></TR><TR><TD class="experiments">CV, JK</TD><TD class="experiments">CMV/Hef1A tetR: inoculate transformed cells</TD></TR><TR><TD rowspan="4" class="s2">7/21/2012</TD><TD class="name">KMG, JE, RL</TD><TD class="experiments">U6-TetO - S1/S6: Miniprep. Made cell stock</TD></TR><TR><TD class="experiments">CV, JK</TD><TD class="experiments">CMV/Hef1A tetR: miniprep. made cell stock</TD></TR><TR><TD class="experiments">JK, LX</TD><TD class="experiments">Inoculate (new GG) eBFP2 1x, eBFP2 2x, eBFP2 3x ,eBFP2 4x constructs</TD></TR><TR><TD class="experiments">DI, JK</TD><TD class="experiments">Transform mKate-intronic FF4 (entry vector)</TD></TR><TR><TD rowspan="3" class="s2">7/22/2012</TD><TD class="name">JK, LX</TD><TD class="experiments">Performed gel check of U6-TetO-S1 and U6-TetO-S6 to see whether we have the right constructs made</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">Preliminary fuel studies</TD></TR><TR><TD class="experiments">DI,JK</TD><TD class="experiments">1.        Hef1A-TetR__ Miniprep2.        Decoy FF4-3p-4ntin __Miniprep, restriction map and sent for sequencing. Made cell stock. Inoculate transformed cells. Midiprep3.        TuD FF4-3p and TuD FF4-3p-4ntin __Aligned sequence and midiprep correctly aligned constructs4.        pEXPR_12_Hef1A:mKate-IntronicFF4__ transformed new LR5.        Transform mKate-intronic FF4 (entry vector)</TD></TR><TR><TD rowspan="5" class="s2">7/23/2012</TD><TD class="name">JK, LX</TD><TD class="experiments">Miniprep eBFP2-1x, eBFP2-2x, eBFP2-3x, eBFP2-4x. Restriction mapped the constructs and sent them for sequencing. </TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">In-vitro study of fuel strand in the dynamics of strand displacement</TD></TR><TR><TD class="experiments">GA, FS</TD><TD class="experiments">Checked nucleotide sequences for NOT gate </TD></TR><TR><TD class="experiments">DI, JK</TD><TD class="experiments">1.        Hef1A-TetR __ miniprep and map2.        Decoy FF4-3p-4ntin __miniprep. Map and sent for sequencing3.        TuD FF4-3p and TuD FF4-3p-4ntin __ miniprep4.        Q1_U6-TetO-FF4_QX        __ obtained primers5.        mKate-IntronicFF4  (Entry Vector)__ miniprep from cell stock</TD></TR><TR><TD rowspan="5" class="s2">7/24/2012</TD><TD class="name">JK,LX</TD><TD class="experiments">Align the sequencing results with our designs for eBFP2-1x, eBFP2-2x, eBFP2-3x, eBFP2-4x. Midiprep correctly aligned constructs</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">Fuel studies. Measure and discard old plates</TD></TR><TR><TD class="experiments">GA, FS</TD><TD class="experiments">Checked that domain X do not cause unwanted displacement. Ordered oligos and NOT gates. </TD></TR><TR><TD class="experiments">JE, KMG</TD><TD class="experiments">Obtained HHR G-blocks</TD></TR><TR><TD class="experiments">DI, JK</TD><TD class="experiments">1.        Hef1A-TetR__ LR troubleshooting2.        Decoy FF4-3p-4ntin __ Aligned sequencing results. Midiprep3.        TuD FF4-3p and TuD FF4-3p-4ntin __ Aligned the sequencing results. Midiprep sequences correctly aligned </TD></TR><TR><TD rowspan="8" class="s2">7/25/2012</TD><TD class="name">DI, JK</TD><TD class="experiments">1. Gradient PCR to check 2.        Re-PCR Q1-eBFP2-QX constructs and purified the PCR products.2. Re-annealed oligos. Performed gel check. Carried out GG cloning to assemble the parts for the constructs. Transformed the constructs into E.coli the identity of the constructs for U6-TetO-S1 and U6-TetO-S6. Gel check. 3. In vitro transcribed SP6-eBFP2 mRNA</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">In-vitro study: mRNA as input (mN domains)</TD></TR><TR><TD class="experiments">DI, JK</TD><TD class="experiments">1. Resuspended G-blocks. GG assembly and transformed the constructs2. Re-anneal ALL oligos. Run gel to check3. Discard all TetR Entry Vectors EXCEPT #4  </TD></TR><TR><TD class="experiments">DI, JK</TD><TD class="experiments">1.        Hef1A-TetR__ carried out new LR reactions. Transformed. 2.        Decoy FF4-3p__Inoculate and midi3.        TuD FF4-3p and TuD FF4-3p-4ntin __ Re-PCR Q1-U6-TetO-Q3, PCR Purify. Re-anneal oligos. Gel Check and GG annealed. Transformed.4.        Q1_U6-TetO-FF4_QX        __ Gradient PCR. Gel check5.        pEXPR_12_Hef1A:mKate-IntronicFF4 __ new LR reactions. Transformed the products of the LR reactions. 6.        pEXPR_12_Hef1A:eYFP-4xFF4__ new LR reactions. Transformed the products of the LR reactions.7.        Make new 5 uM primer stocks</TD></TR><TR><TD class="experiments">WL, AS</TD><TD class="experiments">Re-dilute ALL canonical LR Entry Vectors to 5fmol stock solutions </TD></TR><TR><TD class="experiments">RL, LX</TD><TD class="experiments">1. Send GG donor vectors for sequencing with M13 F &amp; R. Send eYFP-4xFF4 from Box 2 for sequencing2. Pour plates</TD></TR><TR><TD class="experiments">RL, LX, JK</TD><TD class="experiments">Design an organized Workflow 0.2 for all of our correctly sequenced parts (Part name, concentration, DNA location, cell stock)</TD></TR><TR><TD class="experiments">EA, GA</TD><TD class="experiments">Check Qiagen Inventory: Low on Wash Buffer PE for Minipreps. Low on Buffer N3 for PCR Purification. Enough tubes and buffers for this week, possibly next week. The kits should most likely be ordered beginning of next week</TD></TR><TR><TD rowspan="4" class="s2">7/26/2012</TD><TD class="name">JK,LX</TD><TD class="experiments">1.        PCR troubleshoot for U6-TetO-S1 and U6-TetO-S6 constructs.2.        Inoculate the transformed E.coli to amplify the constructs for eBFP2-1x, eBFP2-2x, eBFP2-3x, eBFP2-4x</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">In-vitro study: NOT sensor design stock check. mRNA as input V2</TD></TR><TR><TD class="experiments">JE, KMG</TD><TD class="experiments">Inoculate cells transformed with the GG constructs </TD></TR><TR><TD class="experiments">DI, JK</TD><TD class="experiments">1.        Hef1A-TetR__ Inoculate the transformed cells. New LR2.        TuD-FF4-3p and TuD-FF4-3p-4ntin- if GG/ anneals fail, order new primers3.        Decoy FF4-3p__ midiprep4.        TuD FF4-3p and TuD FF4-3p-4ntin __inoculate the transformed cells5.        Q1_U6-TetO-FF4_QX        __PCR troubleshoot and ordered new primers6.        pEXPR_12_Hef1A:mKate-IntronicFF4 __ inoculate the transformed cells. New LR7.        pEXPR_12_Hef1A:eYFP-4xFF4__inoculate the transformed cells</TD></TR><TR><TD rowspan="5" class="s2">7/27/2012</TD><TD class="name">JK, LX</TD><TD class="experiments">1.        Ordered new primers for U6-TetO-S1 and U6-TetO-S6 constructs. 2.        PNK treating of oligos (to add 5’P). Annealed oligos and GG transformed</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">NOT sensor sequences. mRNA as input (appended sequence). PAGE purification of gate: outputs</TD></TR><TR><TD class="experiments">GA,FS</TD><TD class="experiments">Troubleshoot NOT gate contructions</TD></TR><TR><TD class="experiments">JE, KMG</TD><TD class="experiments">Miniprep the plasmids of cells transformed with GG constructs (HHR G-blocks. Mapped and send for sequencing. PCR (add SP6). Checked PCR on Gel. Inoculate and Midiprep</TD></TR><TR><TD class="experiments">DI, JK</TD><TD class="experiments">1.        Hef1A-TetR__ Transformed new LR2.        TuD FF4-3p and TuD FF4-3p-4ntin __PNK (to add 5'P) and GG annealed. Transformed.3.        pEXPR_12_Hef1A:mKate-IntronicFF4__ transformed new LR</TD></TR><TR><TD rowspan="6" class="s2">7/28/2012</TD><TD class="name">JE, KMG</TD><TD class="experiments">Aligned the sequence of correct constructs. Midiprep. RNA gel</TD></TR><TR><TD class="experiments">JK, LX</TD><TD class="experiments">Inoculate cells transformed with eBFP2 1x, eBFP2 2x, eBFP2 3x, eBFP2 4x constructs</TD></TR><TR><TD class="experiments">JE</TD><TD class="experiments">Aligned sequence for HHR G-blocks </TD></TR><TR><TD class="experiments">AS</TD><TD class="experiments">Inoculate cells transformed with Hef1A-TetR constructs +Variants (Ubc promoter)</TD></TR><TR><TD class="experiments">DI, WL</TD><TD class="experiments">Inoculate cells transformed with pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using old primers)</TD></TR><TR><TD class="experiments">RL, EA</TD><TD class="experiments">1. Hef1A:mKateIntFF4 +Variants (Ubc, CAG promoter): Inoculate transformed cells2. Hef1A:mKateIntFF5FF4: inoculate transformed cells3. pEXPR_23_Hef1A-LacO:eYFP-4xFF4 AND pEXPR_23_TRE-tight-LacO:eYFP-4xFF4: inoculate transformed cells</TD></TR><TR><TD rowspan="6" class="s2">7/29/2012</TD><TD class="name">JK, LX</TD><TD class="experiments">eBFP2 1x, eBFP2 2x, eBFP2 3x, eBFP2 4x constructs: miniprep, restriction map and make cell stock</TD></TR><TR><TD class="experiments">LX</TD><TD class="experiments">SP6-eBFP2 DNA construct: check PCR on gel</TD></TR><TR><TD class="experiments">JE</TD><TD class="experiments">HHR G-blocks: check PCR on gel</TD></TR><TR><TD class="experiments">AS</TD><TD class="experiments">Mini prep and restriction map Hef1A-TetR constructs +Variants (Ubc promoter). Make cell stock</TD></TR><TR><TD class="experiments">DI, WL</TD><TD class="experiments">Mini prep and restriction map  pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using old primers). Make cell stock</TD></TR><TR><TD class="experiments">RL, EA</TD><TD class="experiments">1. Hef1A:mKateIntFF4 +Variants (Ubc, CAG promoter):Mini prep and restriction map. Make cell stock2. Hef1A:mKateIntFF5FF4: Mini prep and restriction map. Make cell stock3. pEXPR_23_Hef1A-LacO:eYFP-4xFF4 AND pEXPR_23_TRE-tight-LacO:eYFP-4xFF4:Mini prep and restriction map. Make cell stock</TD></TR><TR><TD rowspan="9" class="s2">7/30/2012</TD><TD class="name">KEK, CV</TD><TD class="experiments">Send for sequencing NOT sensor</TD></TR><TR><TD class="experiments">JK, LX</TD><TD class="experiments">eBFP2 1x, eBFP2 2x, eBFP2 3x, eBFP2 4x constructs: Send for sequencing</TD></TR><TR><TD class="experiments">LX</TD><TD class="experiments">Gel purify SP6-eBFP2 DNA/ verify eBFP2 mRNA folding at room temperature</TD></TR><TR><TD class="experiments">GA, FS</TD><TD class="experiments">NOT Gate: do gate annealing</TD></TR><TR><TD class="experiments">JE, KMG</TD><TD class="experiments">Obtained NOT gate primers</TD></TR><TR><TD class="experiments">JE</TD><TD class="experiments">Redo Gradient PCR of HH3 G-blocks. Check PCR on gel and gel purify</TD></TR><TR><TD class="experiments">AS</TD><TD class="experiments">Hef1A-TetR constructs +Variants (Ubc promoter): Send for sequencing and inoculate for more colonies</TD></TR><TR><TD class="experiments">DI, WL</TD><TD class="experiments">pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using old primers): send for sequencing</TD></TR><TR><TD class="experiments">RL, EA</TD><TD class="experiments">1. Hef1A:mKateIntFF4 +Variants (Ubc, CAG promoter): send for sequencing and inoculate for more colonies2. Hef1A:mKateIntFF5FF4: Send for sequencing</TD></TR><TR><TD rowspan="10" class="s2">7/31/2012</TD><TD class="name">KEK, CV</TD><TD class="experiments">experimental design of unmodified eBFP2 sensor</TD></TR><TR><TD class="experiments">JK, LX</TD><TD class="experiments">U6-TetO-S1 and U6-TetO-S6 constructs: received new primers. Resuspend primers and PCR</TD></TR><TR><TD class="experiments">JK, LX</TD><TD class="experiments">eBFP2 1x, eBFP2 2x, eBFP2 3x, eBFP2 4x constructs: Aligned sequencing</TD></TR><TR><TD class="experiments">LX</TD><TD class="experiments">Purification of invitro transcribed SP6-eBFP2 mRNA </TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">tested new RNA G:O anneal, study mRNA sensor transfer curve and fuel transfer curve </TD></TR><TR><TD class="experiments">GA, FS</TD><TD class="experiments">in vitro experiment to check the transfer function of NOT gate</TD></TR><TR><TD class="experiments">JE, KMG</TD><TD class="experiments">Re-do PCR (HH3 G-blocks). Check PCR on gel and PCR troubleshoot</TD></TR><TR><TD class="experiments">AS</TD><TD class="experiments">Hef1A-TetR constructs +Variants (Ubc promoter): aligned sequencing results. Miniprep (more colonies). Restriction map to check the constructs are correct. Make cell stock. Send for sequencing and inoculate cells for MIDIprep</TD></TR><TR><TD class="experiments">DI, WL</TD><TD class="experiments">1. pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using old primers): aligned sequencing data 2. pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): obtained new primers/ resuspend primers. Anneal oligos-GG (with T4 kinase). Transformed cells </TD></TR><TR><TD class="experiments">RL, EA</TD><TD class="experiments">1. Q1_U6-TetO-FF4_QX: obtained new primers, resuspend primers and PCR2. Hef1A:mKateIntFF4 +Variants (Ubc, CAG promoter): Aligned sequencing results. Miniprep for more colonies, restriction map. 3. Hef1A:mKateIntFF5FF4: Aligned sequencing results</TD></TR><TR><TD rowspan="8" class="s2">8/1/2012</TD><TD class="name">JK, LX</TD><TD class="experiments">Ordered Gblocks for U6-TetO-S1 and U6-TetO-S6</TD></TR><TR><TD class="experiments">LX</TD><TD class="experiments">Invitro transcribed SP6-eBFP2 mRNA </TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">update old plate info and clean plate reader computer data </TD></TR><TR><TD class="experiments">GA,FS</TD><TD class="experiments">planned invitro experiment to check the transfer function of NOT gate </TD></TR><TR><TD class="experiments">JE, KMG</TD><TD class="experiments">HH3 G-blocks:Re-do PCR (gradient w/ Betaine), Check PCR on gel </TD></TR><TR><TD class="experiments">AS</TD><TD class="experiments">Hef1A-TetR constructs +Variants (Ubc promoter): aligned sequencing results. MIDIprep and pass along to the transfection team</TD></TR><TR><TD class="experiments">DI, WL</TD><TD class="experiments">pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): inoculate transformed cells</TD></TR><TR><TD class="experiments">RL, EA</TD><TD class="experiments">1. Q1_U6-TetO-FF4_QX:Grad PCR w/ Betaine, Check on gel and ordered gBlock         2. Hef1A:mKateIntFF4 +Variants (Ubc, CAG promoter): Miniprep (more colonies). Do restriction map to check the constructs are correct. Make cell Stock and send for sequencing. Inoculate the cels for Midiprep later on</TD></TR><TR><TD rowspan="5" class="s2">8/2/2012</TD><TD class="name">LX</TD><TD class="experiments">PCR amplify SP6-eBFP2 DNA and do RNA purification of SP6-eBFP2 mRNA </TD></TR><TR><TD class="experiments">JE, KMG</TD><TD class="experiments">HH4 G-blocks:Re-do PCR (gradient w/ Betaine), Check PCR on gel </TD></TR><TR><TD class="experiments">DI, WL</TD><TD class="experiments">pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): Miniprep. Restriction map to check the constructs. Make cell stock and send for sequencing. Inoculate transformed cells for MIDIprep later on</TD></TR><TR><TD class="experiments">RL, EA</TD><TD class="experiments">Hef1A:mKateIntFF4 +Variants (Ubc, CAG promoter): Miniprep (more colonies). Aligned sequencing data and MIDIprep. Pass along MIDIprep samples for the transfection team</TD></TR><TR><TD class="experiments">DI</TD><TD class="experiments">Designed new T7 IVT Primers (eBFP2,LacI,LacZa, HH's) and ordered primers</TD></TR><TR><TD class="date">8/3/2012</TD><TD class="name">DI, WL</TD><TD class="experiments">pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): Align Sequencing results and Order gBlocks</TD></TR><TR><TD rowspan="5" class="s2">8/6/2012</TD><TD class="name">KEK, CV</TD><TD class="experiments">1. NOT Sensor: ordered sequences and simulate2. unmodified eBFP2 sensor: obtained new fuels, anneal domain 1 G:O and run experiment to simulate</TD></TR><TR><TD class="experiments">LX, JK</TD><TD class="experiments">eBFP2 1x, eBFP2 2x, eBFP2 3x, eBFP2 4x constructs: Ordered new primers and Gblocks</TD></TR><TR><TD class="experiments">LX</TD><TD class="experiments">SP6-eBFP2 DNA: check products on gel. Gel purify</TD></TR><TR><TD class="experiments">KB</TD><TD class="experiments">Ordered 2' O-me RNA intermediate gates</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">Obtain new primers, Resuspend primers and PCR the respective templates</TD></TR><TR><TD rowspan="6" class="s2">8/7/2012</TD><TD class="name">KEK, CV</TD><TD class="experiments">unmodified eBFP2 sensor: troubleshoot, look at overnight data, revise plans. get more mRNA. compare domains (run experiment)</TD></TR><TR><TD class="experiments">LX</TD><TD class="experiments">1. U6-TetO-S6: Resuspend new Gblocks. GG anneal. Transformed cells with constructs2. SP6-eBFP2 mRNA: IVT RNA purification</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments"> design gate:outputs to be transcribed in vivo </TD></TR><TR><TD class="experiments">GA, FS</TD><TD class="experiments">NOT Gate: experiment to test transfer function at higher concentration </TD></TR><TR><TD class="experiments">JE, KMG</TD><TD class="experiments">Gradient PCR HH2 G-blocks</TD></TR><TR><TD class="experiments">RL, AS</TD><TD class="experiments">1. Hef1A:TetR-KRAB Expression Vector: transformed cells with vector2. Q1_U6-TetO-FF4_QX: resuspend new g-blocks. GG anneal and transformed cells with constructs. </TD></TR><TR><TD rowspan="4" class="s2">8/8/2012</TD><TD class="name">KEK, CV</TD><TD class="experiments">1. NOT Sensor: anneal 2. unmodified eBFP2 sensor:  look at overnight results of debug run and overnight results of domain comparison. Rerun experiment/Make new plan</TD></TR><TR><TD class="experiments">LX</TD><TD class="experiments">1. U6-TetO-S6: inoculate transformed cells 2. U6-TetO-S1: resuspend new Gblocks. GG anneal and transformed cells with constructs3. SP6-eBFP2 mRNA: IVT RNA purification4. PCR T7-eBFP2 DNA</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">in vitro test of 2'O-Me parts</TD></TR><TR><TD class="experiments">RL, AS</TD><TD class="experiments">1. Hef1A:TetR-KRAB Expression Vector: inoculate cells (TE buffer). Miniprep. Make cell stocks. Restriction map and pass long constructs to transfection team2. Q1_U6-TetO-FF4_QX: inoculate transformed cells</TD></TR><TR><TD rowspan="5" class="s2">8/9/2012</TD><TD class="name">KEK, CV</TD><TD class="experiments">1. NOT Sensor: run experiment2. unmodified eBFP2 sensor:  look at overnight results and analyze</TD></TR><TR><TD class="experiments">LX</TD><TD class="experiments">1. U6-TetO-S6: miniprep. restriction map. send for sequencing. make cell stock and inoculate more cells for MIDIprep2. U6-TetO-S1: inoculate transformed cells3. T7-eBFP2 DNA: check product on gel and gel purify</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">discuss about bulges and transcribing gate:output as a single strand</TD></TR><TR><TD class="experiments">RL, AS</TD><TD class="experiments">1. pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): obtained and resuspend new G blocks. GG anneal and transformed cells2. Q1_U6-TetO-FF4_QX: miniprep. restriction map. send constructs for sequencing. Make cell stock and Inoculate more transformed cells for MIDI</TD></TR><TR><TD class="experiments">JE</TD><TD class="experiments">Ordered new reverse primers and Inoculated new minipreps         </TD></TR><TR><TD rowspan="4" class="s2">8/10/2012</TD><TD class="name">LX</TD><TD class="experiments">1. U6-TetO-S6: aligned sequencing results. MIDIprep2. U6-TetO-S1: Miniprep. Restriction map. Send for sequencing. Inoculate transformed cells for MIDIprep3. T7-eBFP2 mRNA: invitro transcribed and RNA purification</TD></TR><TR><TD class="experiments">JE</TD><TD class="experiments">miniprep HH2 G-blocks</TD></TR><TR><TD class="experiments">RL, AS</TD><TD class="experiments">1. pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): inoculate transformed cells2. Q1_U6-TetO-FF4_QX: Aligned sequencing results. MIDIprep</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">Testing new anneal protocols</TD></TR><TR><TD rowspan="2" class="s2">8/11/2012</TD><TD class="name">RL, AS</TD><TD class="experiments">pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): miniprep, restriction map. Send for sequencing and inoculate cells for MIDI prep</TD></TR><TR><TD class="experiments">LX</TD><TD class="experiments">U6-TetO-S1: Aligned sequencing results. MIDIprep</TD></TR><TR><TD class="date">8/12/2012</TD><TD class="name">RL, AS</TD><TD class="experiments">pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): aligned sequencing results. Make cell stocks. Inoculate cells for MIDIprep</TD></TR><TR><TD rowspan="6" class="s2">8/13/2012</TD><TD class="name">JK, LX</TD><TD class="experiments">Ordered new primers/ gBlocks for eBFP2 1x , eBFP2 2x, eBFP2 3x and eBFP2 4x </TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">New RNA anneals </TD></TR><TR><TD class="experiments">GA, FS</TD><TD class="experiments">Plan in-votro experiments for strand displacement using NOT gate</TD></TR><TR><TD class="experiments">JE</TD><TD class="experiments">HHR G-blocks: check PCR on gel</TD></TR><TR><TD class="experiments">RL, AS</TD><TD class="experiments">pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): send for sequencing. </TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">Testing new rRep6, dsROX</TD></TR><TR><TD rowspan="5" class="s2">8/14/2012</TD><TD class="name">KEK, CV</TD><TD class="experiments">debug annealing problems and carried out transfer function experiment for NOT sensor</TD></TR><TR><TD class="experiments">LX</TD><TD class="experiments">In-vitro transcription of T7-eBFP2 mRNA </TD></TR><TR><TD class="experiments">JE</TD><TD class="experiments">HHR gBlocks: Check other 3 hammerheads on gel. Purification showed low concentration. Redo PCR amplification with T7 promoter</TD></TR><TR><TD class="experiments">RL, AS </TD><TD class="experiments">pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): Aligned sequencing results, MIDIprep. Re-inoculate cells for MIDIprep</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">Testing dRep6 consistency with respect to their positions on the plates, observed dRep6 coming apart, carried out positional analysis</TD></TR><TR><TD rowspan="5" class="s2">8/15/2012</TD><TD class="name">KEK, CV</TD><TD class="experiments">1. Do literature search for mRNA target: how does temperature affect dRep6/ Tested 2'OMe RNA for mRNA sensing (at 37C)2. Test with mRNA for NOT sensor3. Do transfer function experiment for unmodified eBFP2 sensor </TD></TR><TR><TD class="experiments">JK, LX</TD><TD class="experiments">Make template for T7/SP6 lacI/lacZ DNA, mRNA </TD></TR><TR><TD class="experiments">JE, KMG</TD><TD class="experiments">1. Retry purification of HHR gBlocks 2. Design new primers for HHR with Stem and mKate</TD></TR><TR><TD class="experiments">RL, AS</TD><TD class="experiments">pENTR_U6-TetO:TuD-FF4-3p and pENTR_U6-TetO:TuD-FF4-3p-4ntin (using new primers): Midiprep and pass long to transfection team</TD></TR><TR><TD class="experiments">KEK</TD><TD class="experiments">Testing fuels, addition of inputs in a step-wise manner</TD></TR><TR><TD rowspan="2" class="s2">8/16/2012</TD><TD class="name">KEK, CV</TD><TD class="experiments">Design new reporter/ test with mRNA for NOT sensor and carried out transfer function experiment for unmodified eBFP2 sensor </TD></TR><TR><TD class="experiments">JE, KMG</TD><TD class="experiments">Ordered new primers/ gBlocks for HHR with stem and mKate</TD></TR><TR><TD class="date">8/17/2012</TD><TD class="name">KEK</TD><TD class="experiments">Testing Rox on 384 plate/ Running reactions in test tubes and taking samples from tubes at different time intervals </TD></TR><TR><TD class="date">8/18/2012</TD><TD class="name">KEK</TD><TD class="experiments">Anneals/ dsROX vs rRep6, testing the effect of carrier RNA</TD></TR><TR><TD class="date">8/19/2012</TD><TD class="name">KEK</TD><TD class="experiments">Testing positional variants on 384, newest PCR anneal</TD></TR><TR><TD class="date">8/20/2012</TD><TD class="name">KEK</TD><TD class="experiments">Dose response for S1 RNA carrier, dRep6 PCR anneal testing. Testing PCR Gate-Outputs, mRNA sensing </TD></TR><TR><TD class="date">9/7/2012</TD><TD class="name">KEK</TD><TD class="experiments">annealed reporter</TD></TR><TR><TD class="date">9/20/2012</TD><TD class="name">KEK</TD><TD class="experiments">new long anneals</TD></TR><TR><TD class="date">9/27/2012</TD><TD class="name">KEK</TD><TD class="experiments">annealed and tested the newest RNA strand design</TD></TR></TBODY></TABLE></DIV><DIV class="section" id="springtcbio"><H1> Spring Tissue Culture Lab Notebook </H1><B>April 9th, 2012</B>We are using HEK cells with constitutive EYFP.
Typically, they need to be passaged 2-3 days, today we passaged at 1:6 ratio. <B>April 11th, 2012</B>Split at a 1:6 ratio. They will need splitting again by Friday.
Also, iGEM now has P1000 tips and culture dishes in the bottom drawer. All of our goods are labeled &quot;iGEM.&quot;
The P1000 tips are to be used with the aspirator when aspirating liquids in the hood. Be sure to EtOH before and after use.
Last night (4/11/12) Kenneth metafected cells at 11 pm. 
Movies are here: http://www.youtube.com/playlist?list=PL07C09A5037691E54  
<B>April 13th, 2012</B>Split at 1:12 ratio, so they will be able to last 3 days (till Monday).
Made 10 Aliquots of Typsin (10 mL). They are on the bottom shelf of the freezer, labeled with iGEM on them.
When you need to passage cells, take an aliquot and thaw in the hot water bath. For now, we won't be needing all 10 mL in the aliquot so I need to check if we should freeze it or put in the fridge. 
<B>April 16th, 2012.</B>Made media according to the formula above.
Katie split cells today at 1:6. They need splitting on Wednesday.
Also, materials for making media (except DMEM, and essential AA's), will be stored in aliquots in the freezer, labeled.  Essential AA's will be aliquotted and stored in the +4.
To make media, you will simply need remove the 50 mL of DMEM, and place that into a labeled conical tube. Then add thawed frozen components and essential AAs.
Frozen components are on the bottom door shelf of the freezer. Essential Amino Acids are in the fridge on the door as well. DMEM can be found in the cold room, some is stocked in the metal min-fridge in the back too.
<B>April 18th, 2012</B>Split cells at 1:6 ratio. Needs to be split again Friday.
Also expanded what we have in culture.
A back up dish will be maintained by myself only.
Another back up dish will be maintained by Katie only.
The plate labeled for experiments can have cells taken from it for transfections etc. When this is used, please notify me, so I can replace it. 
<B>April 20th, 2012.</B>Katie and I (Nathan) split cells today at a 1:13 ratio. They need to be split again on Monday. 
April 23rd, 2012.
Nathan split his cells 1:6 (the petri dish labeled NK and the one labeled experiment) and will split again on Wednesday
Katie split her cells 1:3 to prepare for transfection tomorrow
Followed same protocol as above. 
<B>April 24th, 2012.</B>Katie used 2 mL of her cells that she did not use for transfection to split in a 1:6 ratio (2 mL cells, 11 mL of media) following standard protocol
Katie also performed two transfections:
(1) ROX+RNA(Gate) + CFP into her HEK293 cells
(2) just CFP into her HEK cells
with the help of Kenneth. Deepak is assisting with the confocal, the Zeiss and FACS.
<B>April 25th, 2012.</B> Katie split the experimental plate 1:6 at 7:30 PM. Need to be split again on Friday. 
Nathan split a backup plate around 3 pm. Need to be split again on Friday.
<B>April 26th, 2012.</B>Katie performed 5 transfections (using her culture). 
Katie also split her cells 1:3. They need splitting tomorrow as well as do Nathan K's and the experimental plates. 
<B>April 27th, 2012.</B>Nathan split a Back up and experimental plate were both split at 1:12
Katie split her plate 1:12. 
<B>April 30th, 2012.</B>Katie split her plate 1:6 in preparation for Ala's Transfections on Wednesday.
<B>May 2nd, 2012.</B>Katie split her plate 1:3 in preparation for Ala's Transfections on Thursday.
<B>May 3rd, 2012.</B> Katie performed 3 transfections of ROX with Ala'a. One with 0.1 ug ROX, one with 1 ug ROX and one with 3 ug ROX. Each also with 5 uL of CFP. 
Katie also split HEK cells (labeled KB) 1:3. Will split again on Friday 
<B>May 4th, 2012.</B>Nathan split cells 1:12. Will be ready by Monday. Will probably need to make another bottle of media by Monday. 
Katie split cells 1:12. Will be ready by Monday. 
<B>May 7th, 2012.</B> Nathan split cells 1:6. Will be ready by Wednesday. Also made a new bottle of media. Its labeled iGEM, Fully supplemented (with L-glu, Non-Ess. AA, FBS, P/S), and 5/7/12
Katie split cells 1:3. Will be ready tomorrow for transfection 
<B>May 8th, 2012.</B>Katie split cells 1:12. Katie also did 5 transfections (repeated from April 26th) 
<B>May 9th, 2012.</B>Nathan disposed of Katie's cells in culture. Split two plates 1:6.
<B>May 11th, 2012.</B>Reducing what we have in culture to one plate. Split 1:12, will be ready on Monday.
<B>May 14th, 2012.</B>Cells were quite confluent (95%), and split on the lower side of ~1:6 (2 mL from 13 mL total). Will be ready by Wednesday Also seeded a plate 1:3 for Katie, which will be ready by Tuesday to use in transfections. Also made more trypsin aliquots. 
<B>May 15th, 2012.</B>Katie used Nathan's cells that were seeded 1:3 for 5 transfections. All Details on [RNA &amp; ROX Lifetime Experiment|igem2012:RNA &amp; ROX Lifetime Experiment] page
<B>May 16th, 2012.</B>Cells in dish marked NK were underconfluent. Possible that they were either split too hard, over trypsinized, or over confluency was no good.  Media color is fine (Red), so will come back tomorrow and see if they need to be split or not. Also need to get them back onto a MWF splitting schedule. New aim is to re suspend in 12 mL total to make 1:6, 1:12 splits more straightforward. 
Plan is to split 1:3 Thursday, which means they would be confluent by Friday. Then split 1:12 for over the weekend. This of course depends on the confluency reached by tomorrow.?Could also do a split that is 1:24 (which would last 4 days roughly before splitting), and see how that turns out by Monday. 
<B>May 17th, 2012.</B>Cells in NK dish were properly confluent (80%). Seeded two plates, one at 1:3, which will then be split tomorrow at 1:12. The other 1:24, and to be safe/relatively consistent with the other plate, will do a media change tomorrow.  By Monday, going to check which looks more viable in terms of media colour, confluency etc. Then going to decide which to keep. (Serial splits, or one large split) 
<B>May 18th, 2012.</B>Cells in 1:3 were confluent... Pipetted up and down to detach from the plate instead of trypsin (don't want to loose healthy cells that may not have adhered). Final volume was 10 mL, so split 1:10. I checked under the scope before placing in the incubator, and they look similar to the 1:12 splits, so they should last until monday. To be safe, will check on Sunday their confluency.
1:24, changed media. But it is possible that some cells were washed away in the process (especially if they didn't adhere). Will check sunday as well, but the populations looked incredibly low.
It may make sense for the summer to bring up a new stock from the freezer (depending on how high passage numbers are getting from my plate, which is all we have in culture now). 
<B>May 19th, 2012</B>Both cell populations looked incredibly low. The 1:24 split obviously has less cells than the serial split. However, neither were even 50% confluent, and the media still looks rather healthy. Will probably check on Monday for media color and see if a media change is necessary. 
<B>May 21st, 2012</B>Both cell populations looked low, but the serial split plate looked better than the 1:24. Changed media in the 1:24. Trypsinized the serial split to break up the large clumps of cells to spread more evenly in the plate, but did not split at a ratio (13 mL of cell solution went back into the dish). Will check on Wednesday. 
<B>May 23rd, 2012</B>Serial split plate was destroyed, none of the cells adhered/recovered from trypsinization. 1:24 plate has low populations, but looks OK. Did a media change, will check again on either Thursday afternoon or Friday.
<B>May 25th, 2012</B>Media color is normal, but not changing at all. Cells are alive,  but they are not spreading out on the plate. More or less forming &quot;colonies.&quot; Today, they were trypsinized to break up the colonies, spun down at 260g (according to this Penn iGEM protocol ) for 5 minutes and then resuspended in fresh media. The entire solution was placed into the dish. Will check tomorrow afternoon their confluency. At this point, if these cant come back it would be wise to get a fresh stock of 293Ts. We also should make frozen DMSO stocks so that when mistakes occur, we can at least work from the same passage. 
<B>May 26th, 2012</B>Cells in culture look good! Growing more as a monolayer and not colonies. Looking about 70%-75% confluent. Will split 1:6 today (out of 12 ml) and check again on Monday. Hopefully we can get them back on the MWF schedule. Also as an extra precaution, cells were centrifuged out of the trypsin/media mixture and resuspended in 12 mL fresh media prior to seeding two tissue culture dishes.  
<B>May 28th, 2012</B>Cells look good. Both dishes were split 1:6 (out of 12 mL). Followed the normal protocol (no centrifuging). During the protocol, there was a problem with the pipette aid. It would aspirtate, but not dispel fluid. Had to grab a pipette aid from the hood next to the microscope (washed with EtOH when transferring to and from hoods). Emailed Patrick about it. 
<B>May 30th, 2012</B>Patrick emailed over info about what was going on with the pipettor in the hood. 
Cells are growing, but a bit underconfluent (50%-60%). Since I can not come into lab tomorrow, going to split 1:6 anyways, check on Friday. If still underconfluent a bit on Friday, will let go to Saturday. I would rather have the cells be underconfluent and in log phase than overconfluent and in a lag phase. 
<B>June 1st, 2012</B>One plate was allowed to grow for an extra day. The other was split 1:12, into two dishes to last until Monday (which they will be back at ~60% confluency). This gives us a backup plate in the event that letting a plate go an extra day means the cells are over confluent.
<B>June 2nd, 2012</B>The plate that was allowed to go the extra day looks good, 80%-90% confluent, and not overgrown at all. Will Split 1:6 today, which will put back on MWF splitting schedule.
Cells split yesterday will be checked on Monday... Could do either a 1:3 split from the backup plates on Mon. to get the 80-90% confluency by Wed. or simply expand our cultures from the one plate that looks good (80-90% confluent as of today). Expanded cultures from the properly confluent plate.
<B>June 4th, 2012</B>Split cells 1:6. Will check on Wednesday.
<B>June 6th, 2012</B>Made a new bottle of media. Split cells 1:6. Will check on Friday.
<B>June 8th, 2012</B>Cells were underconfluent and allowed to grow an extra day.
<B>June 9th, 2012</B>Cells are still underconfluent. Seeded a dish 1:3, the other 1:6. To ensure an accurate 12 mL final volume, cells in the trypsin/media mixture were centrifuged at 260g for 5 mins, and then resuspended in fresh media prior to seeding. Also, I dropped our aspirating tool on accident, and the glass portion to it broke. Put it in the pickle jar sharps container. Will need to make another one, or simply use pastuer pipettes. 
Also received an important email from Kenneth regarding the incubators. Information from said email is below.
The water tray in the bottom of the incubator needs to be checked and filled when empty. Without water, the incubator will not be at the proper humidity and the CO2 sensor will not work properly, which means no CO2 and the media becomes too basic (purple) and cells will die/grow slowly. You can use the DI water from the white tap on the sink, no need to add anything to the water. Also, use a graduated cylinder to make easier! (need to check about antibacterial blue stuff)
</DIV><DIV class="section" id="summertcbio"><H1>Summer Tissue Culture Lab Notebook</H1><H2><B> Week One: 6/11/12 - 6/17/12 </B></H2><B>June 11th, 2012</B>One plate of cells was underconfluent, will be ready by tomorrow (the 1:6 from 6/9/12).
The 1:3 plate was 90% confluent. Seeded two plates at 1:3, two plates at 1:6, to have cells ready by tomorrow and Wednesday.
The plan is to potentially use one plate tomorrow to seed more plates for Thursday and Friday, and expand what we have in culture.
We need to have a conversation with Deepak to have a concrete plan on how much we need to expand our cultures.
<B>June 12th, 2012</B>To Do: 1cm plate for Deepak/ other students 
both 1:6 plates appeared way under confluent on the scope (about 10% confluence)
used 1:3 plate from 6/11/12 for transfections of pEXPR Hef1a:Cerulean and IGEM-BBa_K779100 (the gate i.e. the fluorophore + RNA) 
<B>June 13th, 2012</B>Cells are still underconfluent, and have reached a pretty high passage number (at least 30). So, we are talking about getting fresh cells from the summer, and this time making a DMSO stock so we can return to a low passage. 
We performed 6 transfections in duplicate using both 1:6 plates. 
<B>June 14th, 2012</B>Bleached and threw out all remaining 1:3 and 1:6 plates which we were maintaining
Received a frozen DMSO stock of HEK293 (without constitutive YFP) from Kenneth (1.5x 10^6 cells). Going to make more DMSO stocks on Monday.: 
We thawed out the cell. Added them to 12 mL of DMEM with phenol red. Spun down at 1180 RPM for 4 minutes. Aspirated off the media / DMSO. Resuspended the cells in 12 mL of fresh DMEM. Transferred all 12 mL of culture onto a new dish. 
We also observed the Zeiss data from the transfections. 
<B>June 16th, 2012</B>As of today, we don’t have HEK+EYFP, but we may not need them
Split HEK293 today 1:6 for monday. Seeded 5 plates, so that we have enough for transfections and to create DMSO stocks. 
Also added DIH20 to the tray at the bottom of the incubator, to keep humidity and CO2 levels constant. 
Also created more FBS stocks. One for Freezing Media, another for making media for normal culture.
<H2><B> Week Two: 6/18/12 - 6/24/12 </B></H2><B>June 18th, 2012</B>We split one plate of cells and seeded two dishes 1:6. One is back up, the other to potentially use for transfections on Wednesday. We also seeded a dish 1:3 to use in transfections tomorrow for imaging.
We used three plates to make cell stocks of HEK 293T. 3 cryovials contain 1.2x10^6 cells. 3 cryovials contain 1.5x10^6 cells.
One plate was then used for transfections for FACs. A static image was also taken.
<B>June 19th, 2012</B>We seeded one plate 1:6 to be ready by Thursday.
We also repeated the transfections for FACs analysis and Continuous imaging.
<B>June 20th, 2012</B>Prepared samples for FACs analysis. We also split one plate 1:6 to have two dishes ready by friday for transfections. Today we also transfected eCFP to get a quantitative measurement of our transfection efficiency on Friday.
<B>June 21th, 2012</B>Split 1:6 plate from 6/19  1:6 to be ready for Saturday.
<B>June 22nd, 2012</B>
split 1:6 two plates for Sunday (possibly Monday)
<B>June 24th, 2012</B>
There were HEK293+EYFP in our incubator, dated 6/22. As a result, the cells were very confluent, almost 100%. Cells were seeded at 1:3, 1:6 and 1:12. Hopefully, they are still viable. The 1:3 plate, if totally confluent again tomorrow can serve for cells tocks, the rest for transfections if necessary through the week. Or, making more cell stocks... Today I am also bringing up one of the cryovials of HEK293 to test cell viability. Another bottle of fresh media was made. The concentrations are slightly off, by around 1.25%, but according to Deepak it should be OK. Also autoclaved more pastuer pipettes.
<H2><B>Week Three: 6/25/12 - 7/1/12</B></H2><B>June 25th, 2012</B>Made 5 cryovials of HEK293+EYFP.
<B>June 27th, 2012</B>corrected media today. FBS was at 8.8%, so we created another bottle of media at 11.2%, then combined and filter sterilized together.
Made FBS aliquots.
<B>June 28th, 2012</B>Cells more confluent with new 10% media! Split six 1:6 plates for Saturday. 
Need a P1000 in the hood and need P20 calibrated 
<B>June 29th, 2012</B>Split two 1:6 plates of HEK293 for Sunday and two 1:6 plates of HEK293 eYFP for Sunday.
Aliquoted more FBS.  Will need more media soon ( we have another 350 mL bottle...) Got a new calibrated P20! Also aliquoted more trypsin.  
<B>June 30th, 2012</B>Split two 1:6 plates of HEK293 for Monday and two 1:6 plates of HEK293 eYFP for Monday. Also seeded a 6 well plate for FACs training. Should have 200,000 cells for each sample by Monday. Also did a media change on the ROX transfection, and took some images using the microscope in the TC room. Saved on the flashdrive there. Accidentally spilled bleach while cleaning the vaccum trap. Contained with paper towels, and cleaned up. (only spilled around 5-10mL). Also, our box of short pastuer pipettes is low, will need to autoclave more soon.
<B>July 1st, 2012</B>Split two 1:6 plates of HEK293 for Tuesday and two 1:6 plates of HEK293 eYFP for Tuesday. Also seeded another 6 well plate for FACs training, but this time a higher density. Will probably pool samples for tomorrow. Should have ~1,000,000 cells for between the two dishes for tomorrow. Disposed of old (turning basic) media. Did FACS analysis and imaging of the ROX transfection that compares RNAiMax and Lipo2k.
<H2><B>Week Four: 7/2/12 - 7/8/12</B></H2><B>July 2nd, 2012</B>Split two 1:6 plates of HEK293+eYFP for Wednesday. Split two 1:6 plates of HEK293 for Wednesday.
Made a new 500 mL bottle of DMEM with phenol red with 10% FBS, 1% Nonessential Amino Acids, 1% L-Glutamine, 1% Pen/Strep
<B>July 3rd, 2012</B>Split two 1:6 plates of HEK293 for Thursday. HEK293+EYFP were a bit under confluent. Also brought up a frozen stock of HEK293+EYFP to test cell viability.
<B>July 4th, 2012</B>Split two 1:6 plates of HEK293 and HEK293+EYFP for friday.. HEK293+EYFP were a bit under confluent, let go till tomorrow. Did a media change on the transfections. Transfections in phenol red media were not as intensely red as before, only slightly tinted. Imaged and ran FACS analysis on transfections. Seeded a 6 well plate with ~ 500,000 cells per well for FACS training.
<B>July 5th, 2012</B>Split two 1:6 plates of HEK293 and HEK293+EYFP for Saturday. Cells brought up from frozen stock on 7/3/12 about 70% confluent. Will also check again on 7/6/12.
Ran FACS Analysis on Transfections from 7/3/12. Received Alexa tagged output for our Rep6. Diluted and annealed with the ROX tagged gate. Will transfect tomorrow. Also expecting LRs to transfect tomorrow. 
<H2><B>Week Seven: 7/22/12 - 7/29/12</B></H2><B>Sunday July 22nd:</B>Maintenance:
We froze down 10 stocks of HEK293's at P3, which are in the -140.
We have one plate of HEK293+eYFP split 1:6 today to be ready Tuesday.
We have four plates of HEK293's split 1:6 for Tuesday.
We have four plates of HEK293's split 1:3 for Monday.
We have one plate of HEK293's split Friday ~1:6, which were under confluent, which should be ready tomorrow.
Need to bring up one stock of 293's on Thursday to check for viability.
Also need to remember to seed HEK293+eYFP's for Jonathan for the 26th. 
Also need to figure out the proper amount of L-Glutamine to add to the clear media. 
Prepared KB's Optimizing efficiency of dsRNA + DNA samples for FACS.
<B>Monday July 23rd:</B>Split 4 plates of HEK293s 1:6 
Ran FACS on KB's ALEXA, ROX, ALEXA:ROX, +S1, +S6 experiments (following preparation protocol from above)
<B>Tuesday July 24th:</B> Took one plate of HEK293's at P4 (that were split 1:3) to freeze down into 10 cryovials, with ~1,000,000 cells each
Transfections:
KB repeated ALEXA/ROX transfections with 24 well plates (details on expt page)
NK repeated the rTTA3 system, this time with a DOX curve
Passaging:
Split four plates of HEK293s at 1:6 for Thursday
Split one plate of 293+eYFP for Thursday
FACS Prep:
Seeded 250,000 cells per well in a 6 well dish for Jon and whoever else wants FACS training on Thursday 
Seeded 2 wells of HEK293s, 2 wells of HEK293+eYFP, 2 wells of half 293s &amp; half eYFPs. Will prep for FACS on Thursday
<B>Wednesday July 25th:</B>1.	Moved frozen down cells from -80 to -140 (at 9:00 A.M.) 
	Note: Next Tuesday we should bring up a vial to test viability - KB - 7/26
2.	Performed a media change on 9 wells of KB's Alexa, Tag transfections from 7/24 (at 10:30 A.M)
3.	Supplemented Red Media (12:00 PM):
4.	Supplemented Clear Media (12:00 PM):
5.	Supplementing Media Notes:
	Keep all components in small aliquots - easy to thaw
	Always aliquot the entire FBS bottle - it is bad to keep thawing/freezing FBS
6.	Eta transfected constitutive colors at 2:00 PM for Friday Microscopy Training 
7.	Katie transfected Alexa:ROX into her Tag well at 2:30 P.M. to optimize dsRNA + DNA transfection 
8.	Linh retransfected the eYFP knockdown system with FF1 to show that her efficiency and results are repeatable 
9.	Nathan induced a DOX ladder to his rtTA3/DOX system and also retransfected the entire system (due to volume issues the previous day)
10.	Split 5 plates of HEK293's 1:6 
<B>Thursday July 26th:</B> 7:40 - 8:40 A.M. - Prepare Katie's ALEXA FACS samples and Nathan's DOX FACS Samples
10:45 A.M. - 12:00  PM - FACS KB and NK samples 
2:00 PM - NK: Prepare FACS samples for JE + AS 
We prepared 2 wells of 250k HEK293s. 2 wells of HEK293s+EYFP. 2 wells of mixed both 293 and 293+EYFP
4:30 PM - KB: Transfect S6 Input annealed to gate with Hef1a:eYFP Marker
4:45 PM - KB: Split four plates of HEK293's 1:6. Split one plate of HEK293+eYFP 1:6
5:00 PM -   Nathan induced DOX ladder on his rtTA3 transfections
<B>Friday July 27th:</B>8:00 AM - LV + NK prepare FACS samples
11 AM  - FACS NK's rtTA3 DOX, LV's FF1 KD
2:00 PM - KB Transfect Alexa:ROX, +S1, +S6 system with 2X input. Transfect S6:Gate to well transfected with Hef1a:eYFP yesterday. 
•	Protocol - Transfection72712.xlsx 
3:30 PM - LV Transfect miRFF1 - eYFP knockdown system 
4:30 PM  -  NK transfect three plasmid / four plasmid constitutive colors to up efficiency. Let's break 90%! Also retransfect DOX curve system? 
6:00 PM - KB passaged four plates of HEK293's 1:6 (to be ready Sunday) 
<B>Saturday July 28th:</B>FACS KB Optimization / 2x Input Experiments
FACS EA's Single Plasmid Constitutive Colors Expt.
Induce DOX (NK)
Transfect multi plasmid efficiency (NK) 
Passage 
<B>Sunday July 29th:</B>
* NK DOX FACS
* LV Knockdown FACS 
* EA transfect constitutive colors for scope / FACS training
* Passage 
<H2><B>Week Eight: 7/30/12 - 8/5/12 </B></H2><B>Monday July 30th:</B>* FACS NK multi plasmid efficiency

* LV transfect FF1 KD with different parameters

* KB transfect ALEXA/ROX system with 3X input. Also try dsRNA + ssRNA optimization as suggested by Eerik.

* Passage
<B>Tuesday July 31st:</B>* Confocal KB's transfection

* NK transfect efficiency &amp; multiple dox conditions

* LV FACS practice from EA's single color transfections

* Passage

* Make new clear DMEM! 
<B>Wednesday August 1st:</B>* Confocal KB's transfections 

* KB transfect eYFP 

* NK induce DOX ladder

* LV transfect KD with Hef1A:TetR

* make red supplemented DMEM

* FACS LV's KD with reduced parameters, KB's two ALEXA:ROX expts (one with 15 pmol rep6 and 45 pmol of either input) (one with 30 pmol rep6 and 75 pmol of either input)
<B>Thursday August 2nd:</B>* KB transfect in 35 mm dishes at 7:45 AM

* KB confocal with DM at 10:30 AM

* NK FACS DOX Ladder, efficiency

* NK transfect FF4 system, FF1 system

* Make new clear supplemented DMEM (Note, we only have one stock bottle left!)

* Split 4 plates of HEK293 at 1:6 
<B>Friday August 3rd:</B>* NK transfect : 

* KB transfect 15,20,25,30 ALEXA:ROX 

* LV transfect FF4 system?

* Split 4 plates of HEK293 at 1:6
<B>Saturday August 4th:</B>* KB FACS with DM 

* NK FACS 
<H2><B>Week Nine: 8/6/12 - 8/12/12</B></H2><B>Monday 8/6/12:</B>NK transfect DOX curve with 16 data points
NK transfect FF4 system with varying amounts of decoy / miRFF4, and the Lac system
KB transfect FF1 system
Split 4 plates of HEK293s
<B>Tuesday 8/7/12:</B>KB transfect 15,20,25,30 pmol of annealed reporter along with 3 positive controls - Transfection of Rep6+Alexa (July 5th)
NK redo DOX transfection (with 16 data points?)
NK induce DOX, IPTG
Split 4 plates of HEK293s at 1:6 with Jenna
Bring Giulio into TC - bring up a new vial of eYFPs and start passaging 
<B>Wednesday 8/8/12:</B>FACS KB FF1 and NK FF4 at 9:00 A.M.
Supplemented red DMEM with Giulio
FACS NK DOX Ladder and KB Alexa:ROX
Jenna and Giulio split five plates of HEK293+eYFPs at 1:6 
Eta retransfect FF4
Nathan induce DOX on yesterday's transfection
KB retransfect ALEXA:ROX
KB retransfect FF1 system (with TetRKrab this time)
KB split four plates of HEK293s at 1:6 
<B>Thursday 8/9/12:</B>main culture underconfluent - only performed media change 
Eta and Nathan retransfected FF4 system
FACS NK DOX ladder
made supplemented clear media 
<B>Friday 8/10/12:</B>FACS KB A:R + 3X Inputs
FACS KB FF1 Knockdown with TetR, TetR-Krab
KB transfect A:R + U6:TetO inputs
KB transfect FF1 with TetR, TetR-Krab and DOX in media
NK transfect LacI? FF4, 16 data point DOX
GA transfect constitutive colors
Passage - 5 plates 1:6 for Sunday
<B>Saturday 8/11/12:</B>Scope KB A:R with DNA inputs
FACS EA FF4
NK induce DOX ladder
Design meeting! 
Passage - 5 plates 1:6 for Monday
<H2><B>Week Eleven: 8/20/12 - 8/26/12</B></H2><B>Monday 8/20/12:</B>Split 2 plates of 293s at 1:6
Split 1 plate of eYFPs at 1:6 
Giulio split 293s 1:6
<B>Wednesday 8/22/12:</B>Split 1 plate of eYPFs 1:12 for Saturday
Split 1 plate of 293s 1:3 for Thursday
Split 2 plates of 293s 1:12 for Saturday
Seeded a 6 well plate for Jameel. One well of 1,000,000 293s and one well of 1,000,000 eYFPs 
Giulio split 293s
Things to do:
make media (thaw FBS!)
bring up cells from frozen
check if there is lab stock of FBS
<B>Thursday 8/23/12:</B>
giulio microinjection expt
<B>Saturday 8/25/12:</B>
made supplemented clear DMEM, supplemented red DMEM
Split one plate of 293s 1:3, two plates 1:6 and one plate 1:12
Split one plate of eYPFs at 1:12
Thawing a bottle of FBS to aliquot
<B>Sunday 8/26/12:</B>
made 50 mL aliquots of FBS for iGEM
split cells (on P20) 1:3, 1:6 and 1:12
transfected constitutive colors for imaging
<H2><B>Week Twelve: 8/27/12 - 9/2/12 </B></H2><B>Tuesday 8/28/12:</B>Split cells (on P21) 1:3, 1:6 and 1:12
seeded cells for Giulio for FACS training on Thursday
<B>Wednesday 8/29/12:</B>Giulio : microinjection experiment with HeLa cells
image the constitutive color transfection
transfect A:R in triplicate
image A:R (2 - 9 PM)
FACS at 5 PM
make media
bring cells up from frozen stock
plate HeLa cells 
</DIV><DIV class="section" id="falltcbio"><H1>Fall Tissue Culture Lab Notebook</H1><B> 9/20/12 </B> NK transfect FF1 titration expt 
 Split 3 plates of 293s at 1:6
<B> 9/21/12 </B> NK transfect FF1 titration expt 
 Split 3 plates of 293s at 1:6 
 GA and KB nucleofection of long reporter, long reporter + bulge, with either the correct or incorrrect input
 KB transfect U6-TetO-S1 and U6-TetO-S6
<B> 9/22/12 </B> KB transfect new longer RNA reporters and new inputs. FACS. Data looks good! 
 KB split two plates of 293s at 1:6 and one plate at 1:4. Supplemented clear media
 NK FACS FF1 titration
 NK transfect FF1 system
<B> 9/23/12 </B> NK FACS FF1 Titration
NK split cells 1:6 into three plates
<B> 9/24/12 </B> NK FACS FF1 Titration and retransfect FF1 system
 NK split cells 1:6 into 3 plates
<B> 9/25/12 </B> NK transfect FF1 system, HH_mKate and mKate_HH
 NK Took out FBS to thaw to make more red media
 NK Took out trypsin to thaw to replenish aliquots. 
 KB Split cells 1:6
<B> 9/26/12</B> NK made more red media
 FAC FACS FF1 System
 NK transfect FF1 System, HH_mKate and mKate_HH
 NK Split cells 1:6 into three plates
<B> 9/27/12 </B> NK FACS FF1 System, HH_mKate and mKate_HH
 NK Split cells 1:6 into three plates
NK re-transfect FF1 System, HH_mKate and mKate_HH
<B> 9/28/12 </B> NK FACS FF1 System, HH_mKate and mKate_HH.
 NK Split cells 1:6 into three plates.
<B> 9/29/12 </B> NK re-transfect HH_mKate, mKate_HH, Hef1a:mKate and Hef1a:mKate-int-FF4 as a back up.
<B> 10/1/12 </B> NK FACS HH_mKate, mKate_HH, Hef1a:mKate and Hef1a:mKate-int-FF4.
 Gave HEK293 Cells to Giulio. Will refresh working cultures tomorrow.
<B> 10/3/12 </B> NK- brought up new cells from the -140.
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:MIT/Notebook">http://2012.igem.org/Team:MIT/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:MIT/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:MIT/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:MIT/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:MIT/Notebook&amp;oldid=291457" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>