"
Page
Discussion
View source
History
teams
Log in
 
Team:MIT/Actuation
From 2012.igem.org
RNA Strand Displacement for Sensing, Information Processing,
and Actuation in Mammalian Cells
MOTIVATION
RESULTS
Overview
Circuit Production
NOT Gate
Sensing
Actuation
Enabling The Key Reaction
BioBricks
Attributions
METHODS
NOTEBOOK
Construction & In Vitro
Spring Tissue Culture
Summer Tissue Culture
Fall Tissue Culture
HUMAN PRACTICES
SAFETY
TEAM
SPONSORS
CONTACT US
Overview
Results Overview
Circuit Production
Short RNA Production
Circuit Production: Hammerhead Ribozymes
NOT Gate
Design
Modeling
In Vitro Results
Sensing
Design
Modeling
In Vitro Results
Actuation
TuD and Decoy RNAs
Modulating Hammerheads
The Key Reaction
Design
Nucleic Acid Delivery
Experimental Strand Displacement
Our BioBricks
Favorites
All BioBricks
Attributions
Attributions
Decoys and Tough Decoy (TuD) RNAs
Background
Decoy and Tough Decoy (TuD) RNAs are novel technologies for regulating RNA interference ( Haraguchi et al. , Nucleic Acids Res. 2009; Xie et al. , Nature 2012). In short, they are single strands of RNA with one antisense microRNA binding domain (Decoy) or a stabilized stem-loop with two microRNA binding domains (TuD).
In accordance with the central dogma of biology, transcribed mRNA strands are translated by cellular machinery into proteins. Mammalian cells incorporate a layer of expression control, RNA interference, to control mRNA levels. RNA interference uses microRNAs (miRNAs) that are processed by protein complexes to knock down mRNA levels in the cell, reducing protein expression. Decoys and TuDs are artificial strands of RNA with miRNA-binding domains that are thought to sequester the miRNA into stable complexes throuh complementary basepairing, disabling a particular RNA interference pathway.
While miRNAs act as repressors, TuDs and Decoys act as double-repressors such that the presence of the TuDs/Decoys increases protein output.
Thus miRNAs, Decoys and TuDs can potentially be incorporated in RNA strand-displacement circuits to control protein expression levels. As these are short RNAs, we imagine that these strands can be outputs of RNA strand displacement circuits. This will allow strand-displacement circuits to connect to traditional protein-based synthetic biology circuits.
Diagram depicting the interactions between Decoys, TuDs, miRNAs, mRNAs and proteins.
miRNAs are used in the RNA interference pathway to knock down mRNAs, which are usually translated into proteins. Decoys and TuDs repress this pathway, resulting in protein production.
Experimental Design
We designed a proof-of-concept double-repression circuit to test the in vivo actuation ability of Decoy and TuD RNAs as indicated by fluorescent protein expression.
We observed from the Haraguchi et al. study that a particular TuD RNA structure, one with a stabilizing stem-loop structure, yielded increased levels of activity compared to Decoy RNA, so we set out to assess the functionalities of both Decoy and TuD designs. Furthermore, the studies showed that a small 4-nucleotide bulge sequence in the miRNA-binding domains of Decoy and TuD RNAs increased their abilities to relieve repression of protein expression. The bulge in the miRNA-binding sequence likely protected the Decoy and TuD RNA from the RISC protein complex. This characteristic informed our decision to test two different Decoy and TuD RNA species – one that lacked a 4-nucleotide bulge in the miRNA-binding domain, and one that had the bulge sequence.
We designed DNA plasmids containing the mammalian U6 promoter (see Circuit Production for more information on the U6 promoter) that could be used to drive expression of our RNAs in vivo upon transfection into mammalian cells.
Circuit diagram for testing TuD and Decoy-based actuation.
Our circuit begins with a constitutively (always; from the Hef1A promoter) expressed yellow fluorescent protein (eYFP), with the mRNA transcript containing several binding domains for a particular miRNA (FF4). We then introduce the miRNA (from the Hef1A-LacO:mKate-Intronic miR-FF4 construct) into the system, resulting in repression of yellow protein expression. Finally, upon introducing Decoy and TuD RNA into the system (the U6-TetO:Decoy FF4/TuD FF4 construct), the repression is subsequently relieved, allowing for expression of the yellow fluorescent protein. Note that the miR-FF4 miRNA is produced as a spliced-out intron from mKate, a red fluorescent protein. Thus red fluorescence indicates presence of miR-FF4.
Results
Flow cytometry data of our experiments with Decoy RNA.
We tested two different ratios of eYFP, mKate-Intronic miRFF4, and Decoy. The first experiment had a ratio of 1:1:1 and the second 1:1:2. In the instance with more Decoy the RNAi-induced repression was more relieved as indicated by a shift towards higher yellow fluorescence values in the FITC channel.
Ultimately in increasing the relative concentration of Decoy that was transfected, we observed a slight relief of repression of the system. We are currently working on better optimizing our transfection ratios so that our system can be tightly regulated and have a defined switching behavior between the on and off states for protein expression. We also plan to use Decoys or TuDs as conditional outputs of upstream strand displacement that could be used to trigger protein expression in specific cases.
Using Hammerhead Ribozymes to Control Protein Expression
Motivation
Using the Hammerhead ribozymes introduced in Circuit Production , we plan on controlling protein expression in vivo. We intend to use the self-cleaving action of Hammerheads to control mRNA levels, and thus protein levels. Hammerheads are RNA constructs, so they should be able to interface with the RNA strands used in strand displacement.
Designs for Testing Hammerheads In Vivo
We tested this idea by appending or prepending a Hammerhead structure to the mKate fluorescent protein. The Hammerhead sequence should cleave and destabilize the mKate transcript, preventing fluorescent protein production. We used the Hammerhead sequences by Yen et al. (in particular, N117) and designed a family of Hammerhead-containing constructs using the red fluorescent protein mKate.
The designs include putting the Hammerhead motif in the 5' or 3' UTR of mKate, producing either Hammerhead-mKate (HH-mKate) or mKate-Hammerhead (mKate-HH) constructs. For HH-mKate, the Hammerhead was placed before the Kozak sequence and a spacer was included to ensure that the Hammerhead forms correctly (validated using NUPACK ). For mKate-HH, the Hammerhead sequence was placed after the stop codon. Again, the folding of the Hammerhead structure was confirmed in simulations.
Circuits to test the Hammerhead ribozyme function in vivo.
Left: constitutively expressed mKate (red fluorescent protin); middle: Hammerhead-mKate, where the Hammerhead is in the 5' UTR; right: Hammerhead in the 3' UTR. The Hammerhead in the middle and right circuits should self-cleave post transcription, destabilizing the mKate mRNA and preventing its expression.
In Vivo Results for Hammerhead-mKate Constructs
Key: Red: mKate, Blue: mKate-Hammerhead, Green: Hammerhead-mKate.
100,000 HEK293 cells were transfected with equimolar amounts of Hef1a:TagBFP, as a transfection marker, and one of the following: Hef1a:mKate, Hef1a:Hammmerhead-mKate, and Hef1a:mKate-Hammerhead. 48 hrs later, cells were harvested and analyzed by flow cytometry until 10,000 events were recorded. The histograms are gated on the blue population of cells, meaning that we are examining red fluorescence (mKate signal) in cells that also received the TagBFP DNA. There is less red fluorescence in the Hammerhead constructs compared to the Hef1a:mKate. This suggests that the Hammerhead ribozymes could be cleaving the mRNA.
We are currently working on producing a mutant Hammerhead to validate our preliminary results of Hammerhead-mediated mRNA regulation.
Designs for Modulating Hammerhead Activity Using Strand Displacement
We aim to modulate the Hammerhead activity using inputs from strand displacement reactions.
Our design disrupts the Hammerhead structure from forming by introducing a stem that is complementary to the Hammerhead sequence. A toehold on the stem allows for an input strand from an upstream strand displacement reaction to disrupt this strand, reforming the Hammerhead structure.
A method for controlling the Hammerhead structure formation using input from strand displacement. On the left we depict the 3D predicted structure of the Hammerhead with a stem. This Hammerhead is inactive due to the stem. Once an input strand is added, it can bind via a toehold and displace the stem. This allows the Hammerhead structure the form (right). Compare this resulting structure to the regular active Hammerhead, shown in the bottom-right corner. The similarities between the strand-displacement activated Hammerhead and regular Hammerhead suggest that the active catalytic Hammerhead can indeed be formed.
Retrieved from " http://2012.igem.org/Team:MIT/Actuation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
