<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_MIT_ResultsTest"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:MIT/ResultsTest">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:MIT/ResultsTest&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/ResultsTest&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/ResultsTest&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:MIT/ResultsTest" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:MIT/ResultsTest</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P>
DEPRECATED.  DO NOT USE OR EDIT.  If a page uses this template, relink with MIT-results2.

</P><DIV id="col_left"><DIV class="section" id="b1bio"><H1> RNA Strand Displacement In Vitro </H1><P><B> Previously: </B></P><P>
In 2011, Lulu Qian and Erik Winfree, researchers at Caltech, published a paper entitled &quot;Scaling Up Digital Circuit Computation with DNA Strand Displacement Cascades.&quot; This paper demonstrated how scalable logic circuits based on DNA strand displacement are capable of processes as complicated as the square root function. See our <A href="https://2012.igem.org/Team:MIT/Motivation">motivation</A> page for more details. 
</P><P><B> MIT iGEM 2012: </B></P><P>
Before our team attempted to bring the mechanism of strand displacement into an in vivo context, we first decided to assay strand displacement in vitro using RNA. We used 2'-O-methylated RNA strands, which had not been shown to undergo strand displacement in vitro. Before creating our own constructs, we adapted sequences from the Qian/Winfree paper to RNA.
</P><P><B>MIT iGEM Foundational Experiment: </B></P><P>
Figure A shows a foundational in vitro RNA strand displacement experiment that was performed on a plate reader. The negative control, in black, is a well that received only an annealed reporter complex. The bottom strand of this complex is the gate strand, T*-S6*, with the 3' end tagged with the ROX fluorophore. The top strand of the complex is the output strand, S6. This is complementary to the S6* domain of the gate strand. The 5' end is tagged with the Iowa Black RQ quencher, which absorbs the ROX fluorescence; thus, when the two strands of the reporter are annealed, no fluorescence should be observed.  The positive control, in red, is the input strand, T-S6, annealed to the gate strand, T*-S6* tagged with ROX. This is what we would expect the product of a strand displacement reaction to look like. We can see that in the experimental well, when the input is present, it can bind to the exposed T* domain of the reporter and displace the output strand, yielding a fluorescent complex and a waste strand.</P><P><I> Figure A </I></P></DIV><DIV class="section" id="b2bio"><H1> Nucleic Acid Delivery </H1><P>In order to implement RNA strand displacement cascades in vivo, we first demonstrated our ability to deliver nucleic acids to mammalian cells. We have achieved the delivery of  plasmid DNA, single-stranded modified RNA and double-stranded modified RNA to mammalian cells through both lipofection and nucleofection. </P><P><B> (1) Delivery of Plasmid DNA to Mammalian Cells </B></P><P> Through the Gateway method, we have assembled many promoter-gene constructs as detailed on our Parts Page. After construction of the plasmid, we deliver the plasmid DNA to Mammalian Cells through the use of transient transfection, lipofection with Lipofectamine 2000 reage
<B> Images of eYFp/mKate transfection from Confocal </B></P><I> This figure shows HEK293 cells which were transiently transfected with Hef1A:eYFP and Hef1A:mKate using Lipofectamine 2000. Equimole amounts of DNA were delivered, 500 ng total per 1.65 uL of reagent. Images were taken on a Zeiss microscope at 10X.  </I><P><B> (2) Delivery of 2'-O-Me RNA to Mammalian Cells </B></P><P> 
For the purpose of our experiments, RNA oligos with chemical modifications that confer significant overall stability and  increase in Tm is necessary to prevent spontaneous dissociation and rapid degradation by nucleases under in vivo conditions. One form of chemically modified RNA, 2’O-Methyl RNA, is a naturally occurring and nontoxic RNA variant found in mammalian ribosomal RNAs and transfer RNAs. These modified oligos are in most respects similar to RNA, but the 2' O-Methyl modification increases overall stability as the -OH functional group at the 2' position is replaced with a -OMe group, which can't perform cleavage of the RNA backbone. In addition to significant nuclease stability, the modification seem to confer increases in Tm, which minimizes the chance of the RNA strands dissociating upon introduction to a cellular environment. (CITATION NEEDED!)

Therefore, we need the ability to deliver 2’-O-Methyl RNA to mammalian cells. 
<I>The movie below shows HEK293 cells expressing constitutive eYFP with a 2'-O-Methyl RNA strand labeled with ROX (5-carboxy-x-rhodamine) on the 3' end. As time passes, the complex/vesicles are uptaken by the cell, releasing their payload resulting in whole cell fluorescence. Each frame is 5 minutes, movie encompasses 200 minutes in 9 seconds.</I></P><P><I>Delivery of ROX-labeled 2'-O-Methyl RNA into HEK cells.</I></P><P><I>Time point images taken at t = 0, 2, 3, and 4 hours post-transfection. Images taken at 10X on Zeiss microscope.</I></P><P>
Once we demonstrated ability to deliver 2'-O-Me RNA to mammalian cells, we ran optimization experiments to optimize the ratio of 2'-O-Me RNA delivered to RNAiMAX (transfection reagent used).  
</P><P><B> Figure of transfection efficiency with dsALEXA </B><I>Caption</I></P><P><B> (3) Inducible Control of Protein Expression </B></P><P><I>The microscopy images above show a brightfield view, the blue filter and the red filter. TagBFP serves as our transfection marker, indicating that cells have taken in foreign DNA. In the red channel, we show that as you increase the concentration of DOX, more cells fluoresce red.</I></P><P><I> The figure above was generated by transfecting the inducible expression system and varying the concentration of DOX across 16 different data points, and then analyzing using flow cytometry. We demonstrate that as you increase the concentration of DOX, the mean fluorescence increases. At high concentrations of DOX, we eventually see saturation of signal. </I></P><P><B> Delivery of Plasmid DNA which transcribes short RNA Inputs </B></P></DIV><DIV class="section" id="b3bio"><H1>In Vivo RNA Strand Displacement </H1><P>We believe in RNA strand displacement as the ultimate processing medium for mammalian cellular circuits. In order to achieve strand displacement in vivo, we went through five different experimental designs after confirming the ability to deliver and produce many different types of oligos in vivo. 
</P><P><B> Strategy 1: Lipofectamine 2000 Transfection of RNA version of Reporter from Winfree/QIan 2011 Paper</B></P><P>Our first strategy to implement RNA strand displacement in vivo was to adapt the DNA sequences of inputs, gates and reporters from the Qian/Winfree,  &quot;Scaling Up Digital Circuit Computation with DNA Strand Displacement Cascades,&quot; 2011 Science Paper  to 2’-O-Methyl RNA strands to transfect into mammalian cells. See our Motivation page for more details.

In the first foundational experiment, HEK293 (Human Embryonic Kidney cells) were used that constitutively expressed a yellow fluorescent protein (eYFP) in order to be easily visible in microscopy images. 200,000 HEK293 cells were seeded into four wells of a 24 well plate in supplemented DMEM without phenol red pH indicator. The negative control well did not receive any RNA. As a transfection reagent, each well received 1 uL of Lipofectamine 2000. The positive control well received 5 pmol of a gate strand tagged with a ROX fluorophore annealed to an input strand, to act as a product of a strand displacement reaction. The scrambled input well received 5 pmol annealed double stranded reporter with quenched ROX along with a 5 pmol of an input strand containing the correct toehold domain but the incorrect binding domain. Therefore, when both constructs are inside the cell, a strand displacement reaction should not occur, and the fluorophore remains quenched. In the final well, correct input, the cells received 5 pmol of double stranded reporter as well as 5 pmol of an input strand with the correct toehold domain and hybridization domain. Accordingly, we should expect that the toehold of the input strand binds to the complementary exposed toehold on the double stranded reporter, and will branch migrate and effectively kick off the output strand of the reporter that is tagged with a quencher. Therefore, the fluorophore will no longer be quenched, yielding red fluorescence.
<I> Refer to this diagram to identify labeled strands </I><I>In the negative control well, 200,000 HEK293+eYFP cells are healthy and adherent. In the positive control well, we see localized red fluorescence in the form of vesicles as well as distributed, whole cell red fluorescence. In the scrambled input well, we see red vesicles as well as red whole cell fluorescence. In the correct input well, we see only whole cell red fluorescence. </I></P><P><B>Strategy 2: Switch Transfection reagent to RNAiMAX </B></P> From the first foundational experiment, we observed localized red fluorescence in vesicles as well as whole cell fluorescence. This indicates that our reporter complex is either melting, being degraded, being recognized by a specific enzyme etc as well as the reporter is coming apart inside of the lipofectamine vesicles. We researched better transfection reagents for double stranded RNA, and found that Lipofectamine RNAiMAX is designed specifically for the delivery of double stranded RNA, whereas Lipofectamine 200 is specifically designed for the delivery of plasmids.
 Once we received the new transfection reagent, we set up experiments similar to the initial experiment but with an optimized protocol for RNAiMAX (See Materials and Methods). 
<I> Caption for images from 6/14 experiment with old reporter and RNAiMAX where we do not see fluorescent vesicles anymore however we do still see fluorescent cells in the control and experimental wells </I><B> Strategy 3: Tag RNA strand with an Alexa Fluorophore to act as a transfection marker </B><B> Strategy 4: Create DNA plasmids driving transcription of RNA inputs, while transfecting RNA Reporter </B><B> Strategy 4: Nucleofect RNA reporter, RNA inputs </B><B> [Strategy 5]: Redesign RNA Reporter </B></DIV><DIV class="section" id="iv1bio"><H1>Sensing Overview</H1><P>
We designed, modeled and tested an mRNA sensor that interfaces with RNA-based strand displacement circuitry.
</P><H1>Sensing Design</H1><P>
We imposed the following criteria on our mRNA sensor design:
	<UL><LI>orthogonality</LI><LI>easy integration with strand displacement circuits</LI><LI>ability to amplify a signal</LI><LI>ease of sensor generation</LI></UL></P><P>
The way we implemented this sensor is by considering the mRNA sequence and choosing within it two consecutive abstract domains: the <EM>toehold</EM> and the <EM>sensing</EM> domains. These domains mirror the toehold and recognition domains in strand displacement, so provide an interface to any strand displacement circuit. Specifically, the chosen toehold and sensing domain will then directly interface with an intermediate gate:output complex using toehold-mediated strand displacement, with signal amplification achieved using a fuel strand.
</P><P><EM>Illustration of an mRNA sensor.</EM> A toehold region in the mRNA can bind to the toehold on a gate:output complex. Branch migration can then occur, resulting in a free output signal. Using fuel, the mRNA can be released from the gate, enabling it to react with another gate:output complex.
</P><P>
However, this abstraction needs to take into account the secondary structure of mRNA: some regions are more accessible (less basepairing), while others are strongly basepaired and thus unavailable to be directly sensed by mechanisms that involve basepairing to the mRNA (<A href="http://www.nature.com/ng/journal/v39/n10/abs/ng2135.html">Kertesz <EM>et al.</EM> 2007</A>).</P><P><EM><A href="http://nupack.org">Software</A> rendered secondary structure of eBFP2 (BBa_K779300) showing various secondary structure elements (e.g. stems and loops)</EM></P><P>
Much work has been done to generate these secondary structures computationally (see <A href="http://nupack.org">nupack.org</A>), and to find accessible regions predicted to be miRNA binding sites within mRNAs (e.g. <A href="http://genie.weizmann.ac.il/pubs/mir07/mir07_prediction.html">PITA</A>). We leveraged these algorithms to identify potential toehold and sensing domains within mRNAs by looking for regions with different levels of accessibility.
</P><P>
Once suitable domains have been chosen using modeling of the secondary structure, we can rank them by their orthogonality to other transcribed RNAs. These sequences are available in online databases (e.g. <A href="http://webserver.mbi.ufl.edu/~shaw/293.html">mRNA data for HEK 293 cells</A>). For strand displacement, the rate limiting step is the binding of a strand to a gate:output complex using the toehold. After this step, the process of strand displacement is sensitive to nucleotide mismatches, with early mismatches being more disruptive than later mismatches (Qian <EM>et al.</EM> 2011). Thus, we can use a weighted <A>Hamming distance</A> between a candidate domain and every subsequence in the transcriptome. This method is similar to identifying orthogonal sequences for protein-DNA interactions as previously published (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3424557/">Silver <EM>et al.</EM> 2012</A>).
Care has to be taken to make sure that sensors in a circuit are orthogonal as well, but the same method can be applied to any sequences that are introduced into a cell.
</P><P><EM>Four highlighted domains of eBFP2 with various secondary structures we chose to sense.</EM> Blue-filled circles represent toehold nucleotides, green-filled nucleotides represent sensing domain nucleootides. The border of a nucleotide's circle represents the probability of the nucleotide being in the given state (red - very likely, blue - very unlikely). These are closeups of the mRNA structure above. 
</P></DIV><DIV class="section" id="iv2bio"><H1>Sensing Modeling</H1><P>
Modeling of mRNA secondary structure was done using <A href="http://nupack.org">NUPACK</A>. Strand displacement reaction kinetics were simulated using <A href="http://lepton.research.microsoft.com/webgec/">Visual GEC</A>.
</P></DIV><DIV class="section" id="iv3bio"><H1>Sensing <EM>In vitro</EM> studies</H1><P>
We chose 4 domains in eBFP2 with various predicted secondary structure properties to test out <EM>in vitro</EM>. eBFP2 (BBa_K779300) mRNA was produced by PCRing on a T7 promoter and terminator (TODO: link to primers?). The resulting template was then used for in vitro transcription (see Protocols TODO: link). After purification and quantification on a NanoDrop 1000, the transcribed mRNA, corresponding gate:outputs and fuel strands along with a fluorescent reporter were added to wells in a 96-well plate and the fluorescence was measured on a plate reader (See TODO: link to protocol).

For proof-of-concept studies we chose DNA as nucleic acid for the gate:output, fuel, and reporter complexes, as these results will mirror results with RNA-based strands, as shown by foundational experiments. However, the thermodynamics, kinetics and steady states will be different between DNA and RNA strands. We expect mRNA to produce less output than a corresponding DNA input mimic (comprising of just the toehold and sensing domains) in the same amount.
</P><P>
Results indicate that there is a difference between the 4 domains we chose, and that the output signal from mRNA is less than the signal from DNA mimics.

In vitro <EM>results from a DNA-based mRNA sensor.</EM>  Graphed here is the fold increase in RFU - comparing the pre-input fluorescence to the fluorescence after (TODO) hours. Baseline (light gray) reflects no input. DNA input mimic (dark gray) is a DNA oligonucleotide with the same sequence as the toehold and sensing domains in mRNA (black).
<EM>Comparing mRNA as an input to DNA as an input.</EM> Graphed here is the ratio of fold increase fluorescence comparing mRNA inputs to DNA inputs. This indicates the possible completion level due to mRNA inputs. As described above, we expect this to be &lt;1.
</P></DIV><DIV class="section" id="m2bio"><H1>Not Gate in vitro</H1><P><I>Figure 1 - DNA molecules that constitute the NOT GATE </I></P><P>
The original strand displacement paper demonstrated AND and OR gates, but did not include NOT gates.  We successfully designed and built a strand displacement NOT gate in vitro, expanding the computational structures possible with strand displacement.
</P><P>
The design of our NOT gate is in Figure 1 above, where a letter with a '*' depicts a complementary domain to the one denoted by the letter alone. We arrived to this design after iterating through numerous other ideas, trying each time to reduce the number of molecules involved and their complexity.
</P><P>
To understand the behavior of this NOT gate, it can be useful to consider two extreme cases: no input and saturation-level input.
</P><P>
When the input is not present, molecule B can bind reversibly with A (by partially displacing a1) and reversibly with C (by partially displacing c2). When B binds with C, molecule D finishes the job and fully kicks c2 off of c.  c2 then triggers the readout E by irreversibly displacing e2 from e1.  Meanwhile, D also frees B from c1, making B catalytic and allowing it to react with more C molecules, amplifying the output.  Therefore we will see high fluorescence.
</P><P>
When the input is present in high concentration, B binds to a2, partially displacing a2 from a1.  The input then binds to a1, completing what B started by fully and irreversibly separating a2 and a1.  This step was inspired by the mechanism of the cooperative hybridization (Cooperative Hybridization of Oligonucleotides,David Yu Zhang,JACS 2011).  Since B is stuck with a2, it can no longer displace c2 from C, and the readout pathway described above cannot continue. Consequently e2 cannot be displaced from the readout. Therefore we will see no fluorescence.
</P><P><I> Experimental result for the in vitro NOT GATE where the output fluorescence is normalized to the highest value of the NOT GATE transfer function and the total volume for each level of input was 100ul.</I></P><P>
The relative concentration of A with respect of input and B is extremely important. Indeed if the concentration of A is too low the cooperative hybridization between A , B  and a high concentration of input   can be slow, consequently B can displace c2 from C, that is, we would have a high level of output although the input level is high. On the other hand if the concentration of A is too high, even without the presence of input, B will continuously reversibly bind with A. Consequently B will not displace c2 from C and therefore we would not see a high level of output when the level of input is low. 
</P><P>
In addition to the relative concentration of the different components another important point is the absolute concentration of them. This is mainly due to how the cooperative hybridization works. Indeed the reactants are three and the products two, consequently at low concentration the reactants are more favorable in the reaction whereas at high concentration  the products will be more favorable. 
</P><P>
Our strategy  consisted first in finding the right concentration to let the cooperative hybridization works and then we tuned the concentration of A to find the right trade of between the interaction of A, input and B when the input is high and the interaction of A and B when the input is low.
</P><H1>Not Gate modeling</H1><P><I>Figure 2 - NOT GATE transfer function in vitro and by simulation using the software Visual DSD</I></P><P>
The NOT GATE modeling was performed using Visual DSD, in Fig2 the overlay of the simulated transfer function and the in vitro one where the basal fluorescence this time was subtracted. 
 The simulation helped us to find the right trade off, as mentioned before, in the choice  of  the relative concentration of A with to respect of B and input. 
For the rate constant we used the one from  the article &quot;Scaling Up Digital Circuit Computation with DNA Strand Displacement Cascades, Lulu Qian  and Erik Winfree, Science, 2011&quot;.
</P></DIV><DIV class="section" id="m3bio"><H1>Hammerhead Ribozymes</H1><I> Red: mKate, Blue: mKate-Hammerhead, Cyan: Hammerhead-mKate. Overall, inserting a hammerhead ribozyme sequence into mRNA decreases the red fluorescent output. </I><DIV class="section" id="m40bio"><H1>Decoys and Tough Decoys (TuDs)</H1><P>
We wanted something that would provide a tight double repression system with a very distinct change between on and off. The TuDs and Decoys design were originally inspired by the “Vectors expressing efficient RNA decoys achieve the long-term suppression of specific microRNA activity in mammalian cells,” paper. We copied their designs and wanted to reproduce the results in our lab. To do so, we ordered TuDs and decoys both with and without bulges. The bulges are designed to disrupt RISC complex activity; something which degrades short RNA like our decoys in the cell. 

Sources: http://nar.oxfordjournals.org/content/early/2009/02/17/nar.gkp040.abstract 
http://www.ncbi.nlm.nih.gov/pubmed/9695408
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:MIT/ResultsTest">http://2012.igem.org/Team:MIT/ResultsTest</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:MIT/ResultsTest" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:MIT/ResultsTest" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:MIT/ResultsTest&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:MIT/ResultsTest&amp;oldid=245583" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>