<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_MIT_MaterialsAndMethods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:MIT/MaterialsAndMethods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:MIT/MaterialsAndMethods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/MaterialsAndMethods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/MaterialsAndMethods&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:MIT/MaterialsAndMethods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:MIT/MaterialsAndMethods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="project_title" style="text-align: center; margin-top: -0.8em; float:left; z-index:1; position:relative;
margin-left:3em;"><A href="https://2012.igem.org/Team:MIT"><H1 style="border: none; line-height: 1.2em; margin: 0; padding: 0; padding-bottom: 0.5em;">

RNA Strand Displacement for Sensing, Information Processing,and Actuation in Mammalian Cells
</H1></A></DIV></DIV><UL class="nav"><LI id="nav_motivation"><A href="https://2012.igem.org/Team:MIT/Motivation">MOTIVATION</A></LI><LI id="nav_results"><A href="https://2012.igem.org/Team:MIT/ResultsOverview">RESULTS</A></LI><LI><A href="https://2012.igem.org/Team:MIT/ResultsOverview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:MIT/CircuitProduction">Circuit Production</A></LI><LI><A href="https://2012.igem.org/Team:MIT/NOTGate">NOT Gate</A></LI><LI><A href="https://2012.igem.org/Team:MIT/Sensing">Sensing</A></LI><LI><A href="https://2012.igem.org/Team:MIT/Actuation">Actuation</A></LI><LI><A style="background: #993333;" href="https://2012.igem.org/Team:MIT/TheKeyReaction"><EM>Enabling The Key Reaction</EM></A></LI><LI><A href="https://2012.igem.org/Team:MIT/ResultsBiobricks">BioBricks</A></LI><LI><A href="https://2012.igem.org/Team:MIT/ResultsAttributions">Attributions</A></LI></UL><LI id="nav_methods"><A href="https://2012.igem.org/Team:MIT/MaterialsAndMethods">METHODS</A></LI><LI id="nav_notebook"><A href="https://2012.igem.org/Team:MIT/Notebook">NOTEBOOK</A></LI><LI><A href="https://2012.igem.org/Team:MIT/Notebook#constructionbio">Construction &amp;<EM>In Vitro</EM></A></LI><LI><A href="https://2012.igem.org/Team:MIT/Notebook#springtcbio">Spring Tissue Culture</A></LI><LI><A href="https://2012.igem.org/Team:MIT/Notebook#summertcbio">Summer Tissue Culture</A></LI><LI><A href="https://2012.igem.org/Team:MIT/Notebook#falltcbio">Fall Tissue Culture</A></LI><LI id="nav_human_practices"><A href="https://2012.igem.org/Team:MIT/HumanPractices">HUMAN PRACTICES</A></LI><LI id="nav_safety"><A href="https://2012.igem.org/Team:MIT/Safety">SAFETY</A></LI><LI id="nav_team"><A href="https://2012.igem.org/Team:MIT/Team">TEAM</A></LI><LI id="nav_partners"><A href="https://2012.igem.org/Team:MIT/Partners">SPONSORS</A></LI><LI id="nav_contact"><A href="https://2012.igem.org/Team:MIT/Contact">CONTACT US</A></LI><DIV id="col_nav"><DIV id="accordion"><H3><A href="#">Bacterial</A></H3><DIV class="col_list"><UL><LI id="b1">Annealing Oligos</LI><LI id="b2">BP Reaction</LI><LI id="b3"> Capped Transcription Reaction Assembly</LI><LI id="b4"> Cell Stock </LI><LI id="b5"> Gel Extraction - QIAGEN</LI><LI id="b6"> Gel Preparation </LI><LI id="b7"> Golden Gate Reaction </LI><LI id="b8"> Inoculation - Midiprep </LI><LI id="b9"> Inoculation - Miniprep </LI><LI id="b10"> LR Reaction </LI><LI id="b11"> Midiprep QIAGEN </LI><LI id="b12"> Miniprep QIAGEN </LI><LI id="b13"> Nanodrop </LI><LI id="b14"> PCR - Using PFX </LI><LI id="b15"> Pouring LB Agar Plates </LI><LI id="b16"> Restriction Mapping </LI><LI id="b17"> Sequencing </LI><LI id="b18"> Transformation </LI></UL></DIV><H3><A href="#">In Vitro</A></H3><DIV class="col_list"><UL><LI id="iv1">Gate Anneals</LI><LI id="iv2">Plate Reader Studies</LI><LI id="iv3">In Vitro Transcription - New England BioLabs </LI></UL></DIV><H3><A href="#">Mammalian</A></H3><DIV class="col_list"><UL><LI id="m1"> DNA Transfection </LI><LI id="m2"> RNA Transfection </LI><LI id="m4"> Gelatin Pretreatment </LI><LI id="m5"> Passaging Cells </LI><LI id="m6"> Preparing Samples for Flow Cytometry </LI><LI id="m7"> Cryopreservation </LI></UL></DIV><H3><A href="#">Instrumentation</A></H3><DIV class="col_list"><UL><LI id="i1"> Fluorescent Microscopy </LI><LI id="i2"> Confocal Microscopy </LI><LI id="i3"> Flow Cytometry </LI></UL></DIV><H3><A href="#">Automation</A></H3><DIV class="col_list"><UL><LI id="robot"> Automation</LI></UL></DIV></DIV></DIV><DIV id="col_left"><DIV class="section" id="headerbio"><H1> Methods and Protocols Overview</H1>
In order to assess the functionality of RNA strand displacement <EM>in vivo</EM>, as well as to design and implement our sensing, processing, and actuating modules for our proposed genetic circuit, our team employed a variety of protocols and methods in our experimentation. 
The experiments that we conducted to produce our results included the generation of nucleic-acid circuitry components in <EM>E. coli</EM> bacteria, the <EM>in vitro</EM> construction and testing of strand displacement components, the <EM>in vivo</EM> testing of various circuit components in HEK293 mammalian cells, and the use of laboratory instrumentation for data generation and analysis. 
</DIV><DIV class="section" id="b1bio"><H1> Annealing Oligos </H1><OL><LI> Add the following to a 1.5 mL tube at room temperature and mix:  20 uL of each primer, 5 uL sH20, 5 uL T4 Buffer. </LI><LI> Set a heat block to 98°C, place tube in heat block, turn off heat block.</LI><LI>Allow block to cool down to room temperature, oligos should be annealed. </LI></OL></DIV><DIV class="section" id="b2bio"><H1> BP Reaction </H1><OL><LI> Add the following components to a 1.5 mL tube at room temperature and mix: attB-PCR product (=10 ng/uL; final amount ~15-150 ng) 1-7 uL , Donor vector (150 ng/uL) 1 uL ,TE buffer, pH 8.0 to 8 uL. </LI><LI> Thaw on ice the BP Clonase II enzyme mix for about 2 minutes. Vortex the BP Clonase II enzyme mix briefly twice (2 seconds each time). </LI><LI> To each sample (Step 1, above), add 2 uL of BP Clonase II enzyme mix to the reaction and mix well by vortexing briefly twice. Microcentrifuge briefly. </LI><LI> Return BP Clonase II enzyme mix to -20°C or -80°C storage. </LI><LI> Incluate reactions at 25°C for 1 hour. </LI><LI> Add 1 uL of the Proteinase K solution to each sample to terminate the reaction. Vortex briefly. Incubate samples at 37°C for 10 minutes. </LI><LI> Transform 1 uL of each BP reaction into competent cells (see transformation protocol). Plate 20 uL and 100 uL of each on LB plates containing 100 ug/mL kanamycin. </LI></OL></DIV><DIV class="section" id="b3bio"><H1> Capped Transcription Reaction Assembly </H1><OL><LI> Use mMESSAGE mMACHINE Kit: thaw the frozen reagents [on ice]. Place the RNA Polymerase Enzyme Mix on ice, it is stored in glycerol and will not be frozen at -20°C. Vortex the 10X Reaction Buffer and the 2X NTP/CAP until they are completely in solution. Once thawed, store the ribonucleotides (2X NTP/CAP) on ice, but keep the 10X Reaction Buffer at room temperature while assembling the reaction. All reagents should be microfuged briefly before opening to prevent loss and/or contamination of material that may be present around the rim of the tube. </LI><LI>Assemble transcription reaction at room temp. The spermidine in the 10X Reaction Buffer can coprecipitate the template DNA if the reaction is assembled on ice. Add the 10X Reaction Buffer after the water and the ribonucleotides are already in the tube. Add the 10X Reaction Buffer after the water and the ribonucleotides are already in the tube. The following amounts are for a single 20 uL reaction. Reactions may be scaled up or down if desired.  </LI><LI> IMPORTANT: the following reaction setup is recommended when the RNA produced will be 300 bp - 5 kb in length. For transcripts longer or shorter than this, see optimization protocols. 

</LI><LI> To 20 uL Nuclease-free Water add: L 2x NTP/CAP, 2 uL 10X Reaction Buffer,  0.1-1 ug Linear template DNA (use 0.1-0.2 ug PCR-product template or 1 ug linearized plasmid template), 2 uL Enzyme Mix. </LI><LI> Mix thoroughly. Gently flick the tube or pipette the mixture up and down gently, and then microfuge tube briefly to collect the reaction mixture at the bottom of the tube. </LI><LI>Incubate at 37C, 1 hr. Typically 80% yield is achieved after a 1 hr incubation. For maximum yield, we recommend a 2 hr incubation. Since SP6 reactions are somewhat slower than T3 and T7 reactions, they especially may benefit from the second hour of incubation. A second hour of incubation is recommended for synthesis of &lt;300 base transcripts and for inefficiently transcribed templates. If the reaction is trace-labelled: after the incubation (before or after TURBO DNase treatment), remove an aliquot of trace-radiolabeled reactions to assess yield by TCA precipitation. </LI><LI> (Optional) Add 1 uL TURBO DNase, mix well and incubate 15 min at 37 C. This DNase treatment removes the template DNA. For many applications it may not be necessary because the template DNA will be present at a very low concentration relative to the RNA. </LI></OL></DIV><DIV class="section" id="b4bio"><H1> Cell Stock </H1><OL><LI> Add 500 uL of overnight LB liquid culture (sequence verified) to a new cryo-vial. </LI><LI> Add 500 uL of sterilized 50% glycerol in water to vial. </LI><LI> Label tube, store at -80C.  </LI></OL></DIV><DIV class="section" id="b5bio"><H1> Gel Extraction – QIAGEN </H1><OL><LI> Prepare agarose gel (1%), use thick-welled comb. </LI><LI> Run desired sample on gel. </LI><LI> Excise desired band from gel using a razor blade. Place in 2mL Eppendorf tube. </LI><LI> Weigh slice. </LI><LI> Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 uL) </LI><LI> Incubate at 50°C for 10 min or until gel is dissolved; vortex every 2-3 min. </LI><LI> Confirm that the color of the mixture is yellow (if not, add 10 uL of 3M sodium acetate, pH 5.0) </LI><LI> Add 1 gel volume of isopropanol. </LI><LI> Apply 750 uL of mixture to QIAquick column and centrifuge for 1 min (max speed, ~13000 rpm, RT). Discard flow-through and place column back in tube. Repeat until all of the mixture has been applied to the column. </LI><LI> Add 500 uL Buffer QG to column, centrifuge for 1 min, discard flow-through. </LI><LI> Add 750 uL Buffer PE to column, centrifuge for 1 min, discard flow-through. </LI><LI> Dry spin by centrifuging for 1 min. </LI><LI> Place column in a fresh 1.5 mL Eppendorf tube. </LI><LI> Add 50 uL Buffer EB to the center of the membrane to elute DNA, centrifuge for 1 min. </LI><LI> Measure the concentration of DNA using Nanodrop (see Nanodrop protocol). </LI></OL></DIV><DIV class="section" id="b6bio"><H1>Gel Preparation</H1><OL><LI> To make a 1% agarose gel: dissolve UltraPure agarose to a final concentration of 1% in 1X TAE buffer in a glass bottle. For a small gel, measure ~ .5 g agarose, add 1X TAE buffer until 50 mL total volume. For larger gel, scale up as necessary. </LI><LI> Heat the solution in the microwave with frequent stirring to dissolve the agarose homogeneously. ~ 1 minute/200 mL solution. </LI><LI> Add 10 uL SYBRSafe (1:10000) per 100 mL of the solution and mix well. </LI><LI> Pour the solution into a clean small gel tray with comb added for wells. </LI><LI> Wait ~ 15-20 mins for gel to solidify. </LI><LI> If stored, place in Ziploc bag and store at 4°C. </LI><DIV class="section" id="b7bio"><H1> Golden Gate Reaction </H1><OL><LI> Add the following to a PCR tube (0.6 mL) at room temperature and mix: 1 uL (~ 100 ng/uL) donor vector, 2-4 uL DNA (equimolar ratio), 2 uL T4 Ligase Buffer, 1 uL T4 Ligase, 1 uL BsaI, To 20 uL sH2O </LI><LI> Place tube in thermocycler, run standard golden gate protocol: Loop (20-30 cycles), 5 min 37C, 2 min 16 C. </LI><LI> Transform 1 uL of each GG reaction into competent cells (see transformation protocol). Plate 20 uL and 100 uL of each on LB plates containing 100 ug/mL kanamycin with IPTG and X-gal. Incubate at 30°C for at least 16 hrs, select correct colonies through blue/white screening (white = correct, blue = incorrect). </LI></OL></DIV><DIV class="section" id="b8bio"><H1> Inoculation – Midiprep </H1><OL><LI> Add 50 mL of fresh, sterile LB broth to a clean 250 mL Erlenmeyer flask. </LI><LI> Add 50 uL appropriate antibiotic, mix well. </LI><LI> Pick a colony from plate using sterile inoculation loop or pipette tip, and gently swirl in the solution. </LI><LI> Cap flask, place securely in shaking incubator, grow at 37C overnight (~ 16 hours). </LI></OL></DIV><DIV class="section" id="b9bio"><H1> Inoculation – Miniprep </H1><OL><LI> Add 5 mL of fresh, sterile LB broth to a clean 15 mL tube. </LI><LI> Add 5 uL appropriate antibiotic, mix well. </LI><LI> Pick a colony from plate using sterile inoculation loop or pipette tip, and gently swirl in the solution. </LI><LI> Cap tube, place in shaking incubator, grow at 37°C overnight (~ 16 hours). </LI></OL></DIV><DIV class="section" id="b10bio"><H1> LR Reaction </H1><OL><LI> Add the following components to a 1.5 mL tube at room temperature and mix: Entry clone (50-150 ng) 1-7 uL, Destination vector (150 ng/uL) 1 uL, TE buffer, pH 8.0 to 8 uL. </LI><LI> Thaw on ice the LR Clonase II enzyme mix for about 2 minutes. Vortex the LR Clonase II enzyme mix briefly twice (2 seconds each time). </LI><LI> To each sample (Step 1, above), add 2 uL of LR Clonase II enzyme mix to the reaction and mix well by vortexing briefly twice. Microcentrifuge briefly. </LI><LI> Return LR Clonase II enzyme mix to -20°C or -80°C storage. </LI><LI> Incubate reactions at 25°C for 1 hour. </LI><LI> Transform 1 uL of each LR reaction into competent cells (see transformation protocol). Plate 20 uL and 100 uL of each on LB plates containing 100 ug/mL ampicillin or carbenicillin. </LI></OL></DIV><DIV class="section" id="b11bio"><H1> Midiprep – QIAGEN </H1><OL><LI> Transfer the overnight culture of plasmid cells into a 50 mL tube. </LI><LI> Centrifuge for 15 min at 6000g and 4°C. Pour out supernatant. </LI><LI> Add 4 mL Buffer P1 and vortex (setting #7) to fully resuspend cells. </LI><LI> Add 4 mL Buffer P2 and mix by inverting the tube 4-6 times. The cell suspension should turn uniformly blue. Incubate 5 min at RT. </LI><LI> Add 4mL Buffer P3 and mix immediately by inverting tube 4-6 times. The cell suspension should turn white. Incubate on ice 15 min. </LI><LI> Centrifuge for 30 min at 20,000g. If supernatant still cloudy, centrifuge an additional 15 min. Promptly remove supernatant containing plasmid DNA. </LI><LI> Equilibrate QIAGEN-tip 100 by applying 4mL of buffer QBT and allowing the column to empty by gravity flow. </LI><LI> Apply the supernatant from step 6 to the QIAGEN-tip. Allow liquid to drain out. Plasmid DNA is retained on filter. </LI><LI> Wash the filter twice by applying 10 mL buffer QC. </LI><LI> Place the QIAGEN-tip over a 15 mL tube and add 5 mL Buffer QF. DNA is now eluted into 15 mL tube. </LI><LI> Precipitate DNA by adding 3.5 mL room temperature isopropanol. Mix and centrifuge at 15,000g and 4C for 30 min. Remove the supernatant, being careful not to dump out the pellet containing the plasmid DNA </LI><LI> Wash the pellet by adding 2 mL 70% EtOH at RT. Remove the supernatant, being careful not to dump out the pellet containing the plasmid DNA </LI><LI> Air-dry the pellet for 5-10 min. </LI><LI> Resuspend pellet in ??uL TE buffer </LI><LI> Measure the concentration of DNA using Nanodrop (see Nanodrop protocol). </LI><DIV class="section" id="b12bio"><H1> Miniprep – QIAGEN </H1><OL><LI> Before starting: Make sure provided RNase A solution has been added to Buffer P1 before use. One vial of RNase A per bottle of Buffer P1 to give final concentration of 100ug/mL. Add the provided LyseBlue reagent to Buffer P1 and mix before use. Use one vial LyseBlue per bottle of P1 to achieve 1:1000 dilution. Finally, make sure ethanol has been added to Buffer PE. </LI><LI> Transfer 1-5mL of overnight culture of plasmid cells into 2ml microcentrifuge collection tubes (1 per try) provided in the kit. Pellet for 1 min. Decant all the liquid and add 1 ml of the culture into the corresponding tube. Make sure not to mix up the tries. </LI><LI> Resuspend pelleted cells in 250 uL Buffer P1 and transfer to microcentrifuge tube. </LI><LI> Add 250uL Buffer P2 and mix thoroughly by inverting tube 4-6 times. Do NOT vortex. Mixture turns blue. </LI><LI> Add 350uL of Buffer N3 and mix IMMEDIATELY by inverting tube 4-6 times. Do NOT vortex. Mixture is no longer blue. </LI><LI> Centrifuge 10min at 13,000 rpm in table-top centrifuge. </LI><LI> Apply the supernatant to a QIAprep spin column by decanting. Do NOT get any of the sticky precipitate. </LI><LI> Centrifuge for 30 - 60s at 13000rpm. Discard flowthrough. </LI><LI> Wash the QIAprep column by adding 0.5 mL Buffer PB. </LI><LI> Centrifuge for 30 - 60s at 13000rpm. Discard flowthrough. </LI><LI> Wash the QIAprep column by adding 0.75 mL Buffer PE. </LI><LI> Centrifuge for 30 - 60s at 13000rpm. Discard flowthrough. </LI><LI> Place the QIAprep column in a clean 1.5mL microcentrifuge tube. To elute DNA, add 50uL Buffer EB to center of each column. Be careful NOT to pierce column. </LI><LI> Let stand for 1 minute. </LI><LI> Centrifuge for 60s at 13000rpm. </LI><LI> Remove column and discard, tube now contains DNA. </LI><LI> Measure DNA concentration using Nanodrop (see Nanodrop protocol). </LI></OL></DIV><DIV class="section" id="b13bio"><H1> Nanodrop </H1><OL><LI> Open program, select Nucleic Acid, make sure sample type is set to DNA-50. </LI><LI> Make sure Nanodrop pedestal is wiped clean using a Kim wipe. </LI><LI> Blank the spec using 1 uL Buffer EB or sH2O. </LI><LI> Add 1 uL of sample to pedestal, measure concentration. </LI></OL></DIV><DIV class="section" id="b14bio"><H1> PCR – Using PFX </H1><OL><LI> Thaw PFX supermix, primers, template DNA on ice. </LI><LI> To a new PCR tube (0.6 mL tube), add: 22.5 uL PFX supermix, 1 uL 5 uM Forward primer, 1 uL 5 uM Reverse, primer ,0.5 uL (~ 100 ng/uL) Template DNA </LI><LI> Mix solution well. </LI><LI> Place tube in PCR thermocycler. Set thermocycler program: Initial Denaturation: 5 min @ 95°C ; Loop (35 cycles), Denaturation: 30s @ 95°C , Annealing: 30s @ *(see below), Elongation: ** (see below) @ 68°C; Final Elongation: 15 min @ 68°C. Store: 16°C. </LI><LI> *: Calculate Tm based on annealing region: (4(#G or C)+2(#A or T))C </LI><LI> **: Calculate Extension Time based on length of product: 30s/500bp </LI></OL></DIV><DIV class="section" id="b15bio"><H1> Pouring LB Agar Plates </H1><OL><LI> Weigh 17.5 g of LB Agar, add into a sterilized Pyrex container. </LI><LI> Fill with sterile water up to 500 mL mark. </LI><LI> Cap loosely and tape with autoclave tape. </LI><LI> Autoclave for 40 mins. </LI><LI> Let sit at room temperature so solution cools sufficiently – should not hurt to touch bottle. </LI><LI> Add 500 uL of appropriate antibiotic(s) (1000X), gently swirl to mix. </LI><LI> Transfer 20 mL of solution into a new plate using a pipette. Use flame to get rid of any bubbles. </LI><LI> Cap plate and let sit at room temperature until the agar cools and solidifies. </LI><LI> Label plate with date and antibiotic(s) added. </LI></OL></DIV><DIV class="section" id="b16bio"><H1> Restriction Mapping </H1><OL><LI> Use Geneious, Vector NTI, or related program to select an appropriate restriction enzyme for mapping. A good enzyme should cut both the wildtype backbone vector and the correct vector with insert to yield band patterns that can be distinguished reasonably on a gel. Follow appropriate protocol based on enzyme properties (see NEB website) </LI><LI> Add the following to a PCR tube (0.6 uL) and mix: 10 uL (~ 100 ng/uL) DNA, 2 uL (10X) appropriate NEB Buffer, 0.5 uL Enzyme, To 20 uL sH2O. </LI><LI> Incubate at 37°C for 1-2 hours or appropriate enzyme digestion conditions. </LI><LI> Run sample on agarose gel to visualize. </LI></OL></DIV><DIV class="section" id="b17bio"><H1> Sequencing </H1><OL><LI> Fill out Genewiz sequencing form. </LI><LI> To a fresh tube add the following and mix: 10 uL Template DNA 5 uL 5 uM appropriate Sequencing primer. </LI><LI> Cap tube, place in Ziploc bag and attach to bag to sequencing form. Submit to Genewiz. </LI><LI> When results are obtained, align sequence with predicted sequence in Geneious program. </LI></OL></DIV><DIV class="section" id="b18bio"><H1> Transformation </H1><OL><LI> Make sure that the incubator (37°C) and a heat block (42°C) are on. Make sure SOC medium is clear and uncontaminated. </LI><LI> Allow DNA to thaw on ice. </LI><LI> Thaw competent (10G) cells on ice for 5 mins. </LI><LI> Add 1 uL DNA to cells, mix by gently tapping tube. </LI><LI> Incubate cells on ice for 30 min. </LI><LI> Heat shock cells at 42°C in heat block for 30 sec. </LI><LI> Immediately place cells back on ice for 2 min. </LI><LI> Add 900 uL SOC medium to cells, place at RT. </LI><LI> Place tube in shaking incubator at 37°C for 1 hour. </LI><LI> Plate 100 uL cells onto a new plate with appropriate antibiotic. </LI><LI> Place plate upside down overnight and let incubate at 37°C (~ 16 hrs). Should see colonies if successful. </LI></OL></DIV><DIV class="section" id="iv1bio"><H1> Gate Anneals </H1><EM>Annealing oligonucleotides for strand displacement</EM>
One reaction results in 100 µL of annealed product at 10 µM (calculated based on the bottom strand) in 1X TAE buffer (12.5 mM Mg<SUP>2+</SUP>).<EM>Note</EM>: The top strand refers to an output strand that can act as a downstream input, whereas the bottom strand refers to the gate strand, which is always complexed with another strand in reactions.<OL><LI>In a PCR tube mix:
</LI><LI>12 µL of top strand (from a 100 µM stock)</LI><LI>10 µL of bottom strand (from a 100 µM stock)</LI><LI>5 µL of 10x TAE buffer (125 mM Mg<SUP>2+</SUP>)</LI><LI>23 µL of nuclease free water</LI><LI>Run a PCR program that starts at holds at 92°C for 2 minutes and then decreases the temperature by 0.1°C over (every) 6 seconds until reaching 22°C.</LI><LI>Add 50 µL of 1X TAE buffer (12.5 mM Mg<SUP>2+</SUP>).</LI><LI>Store at -20°C.</LI></OL><EM>Note</EM>: If the products are fluorescent, use an opaque tube or cover the tube in foil in order to protect it from being bleached by light.
Adapted from Qian L, Winfree E. <EM>Scaling up digital circuit computation with DNA displacement cascades.</EM>, Science. 2011 Jun 3; 332 (6034):1196-201.</DIV><DIV class="section" id="iv2bio"><H1> Plate Reader Studies </H1><H2><B> Kinetic Studies</B></H2><P><EM>Note</EM>: Buffer is 1X TAE buffer (12.5 mM Mg<SUP>2+</SUP>). Carrier is TTT-TTT-TTT-TTT-TTT-TTT-TT (DNA), in buffer. Amount of reporter is determined by the amount of fluorescent strand. 1X reactive species concentration = 10 nM = 1 pmol in 100 µL.
</P><OL><LI>In a 96-well plate add 500 pmol carrier to each well being used.
Suggested: 50 µL of 10 µM stock (in 1X buffer).
</LI><LI>Let wells incubate with carrier for a few minutes.
Suggested: 10 minutes.
</LI><LI>Prepare the following wells by adding:
</LI><LI>Blank: 50 µL of buffer.</LI><LI>Positive control: 1 pmol RNA-ROX (reporter bottom strand) or dsROX (reporter bottom strand annealed to an input strand).<EM>Suggested</EM>: 1 µL of 1 µM stock, 49 µL buffer.</LI><LI>Baseline (optional): 1.2 pmol reporter complex.<EM>Suggested</EM>: 1.2 µL of 1 µM stock, 48.8 µL buffer.</LI>
Blank, positive control and baseline wells should be at 100 µL volume.<UL><LI>Negative control: 1.2 pmol reporter complex.</LI><LI>Reaction well: 1.2 pmol reporter complex.<EM>Suggested</EM>: add 1.2 µL of 1 µM stock, 47.8 µL buffer.
Negative and reaction wells should be at 99 µL volume.</LI></UL><LI>Measure the baseline fluorescence (10-20 datapoints as desired).</LI><LI>Add inputs.<EM>Suggested</EM>: 1X = 1 µL at 1 µM.</LI><LI>Mix with “magical” pipette tip:
</LI><LI>Rinse pipette in 10 µM carrier solution by pipetting up and down.</LI><LI>Use this empty tip to pipette up and down in the well.</LI><LI>Use a new tip for each well.</LI></OL><LI>Measure fluorescence<EM>Suggested</EM>: For experiments lasting hours, 3 minutes between each sampling is sufficient. For larger concentrations decrease this time to see faster kinetics. Avoid oversampling as this can cause photobleaching.</LI><P><EM>Note</EM>: If there are more components than just input and reporter, add necessary controls (+/- input). Adjust volumes of buffer accordingly.
</P><P>
The reporter is added in excess (1.2x) to guarantee that the reporting reaction is driven to maximum completion, resulting in 1x fluorescent output from 1x input.</P><P>
Carrier is added to prevent non-specific sticking of RNA or DNA to plastic, such as pipette tips or plate wells (Zhang, D., Turberfield, A., Yurke, B., Winfree, E. <EM>Engineering Entropy-Driven Reactions and Networks Catalyzed by DNA.</EM>, Science 2007 Nov; 318 (5853):1121-1125).
</P><P>
Data was gathered using the Tecan Safire II Microplate Reader with Magellan 5.03 software.
Settings for ROX fluorophore: excitation 588 nm, emission 611 nm, bandwith (ex/em) 10 nm, gain 160, 5 flashes per read, 40 µs integration time, 0 µs lag time, Z-position 6286 nm (top, 96 well plate), 100 ms between move and flash.
Settings for Alexa fluorophore: excitation 495 nm, emission 517 nm, bandwith (ex/em) 10 nm, gain 120, 5 flashes per read, 40 µs integration time, 0 µs lag time, Z-position 6286 nm (top, 96 well plate), 100 ms between move and flash.
Data analysis was performed using standard spreadsheet software.
</P></DIV><DIV class="section" id="iv3bio"><H1> In Vitro Transcription - New England BioLabs </H1><OL><LI> Thaw the 10X transcription buffer and 20X NTP mix at room temperature for the minimum amount of time required for complete thawing. Do not thaw at 37°C. If precipitant is evident following thawing, vortex briefly to resuspend. Keep the 20X High Molecular Weight Component (HMW) Mix and T7 RNA Polymerase at –20°C until needed. </LI><LI> Combine the following, in order, taking caution to avoid ribonuclease contamination. Set up separate reactions for each plasmid of interest:  RNase-Free Water to 40 uL, 4 uL10X Transcription Buffer 4, 2uL 20X Ribonucleotide Solution Mix, 2uL Template(s) (1–2 µg) X µl, 20X HMW Mix, 2 uL  T7 RNA Polymerase (500 units/µl) </LI><LI> Incubate at 37 °C for 2-4hrs. </LI><LI> For analysis of IVT run 1uL of transcription reaction in a 1% agarose gel. </LI>

Adapted from New Englands Biolabs protocol, which can be found at http://www.neb.com/nebecomm/ManualFiles/manualE2030.pdf.
</OL></DIV><DIV class="section" id="m1bio"><H1> DNA Transfection </H1><OL><LI>Label enough eppendorf tubes for the number of conditions/wells you are transfecting plus one for a master mix of transfection reagent and unsupplemented DMEM. If possible, try to pool conditions/wells, i.e, if doing a small molecule induction ladder.</LI><LI> Aliquot 50 uL of unsupplemented DMEM to every tube. If you are pooling n conditions/wells, aliquot 50*n. For your mastermix, aliquot 50* total number of conditions/wells of unsupplemented DMEM. </LI><LI> Dilute plasmid DNA in tubes containing 50 (or 50*n) unsupplemented DMEM. Be sure to mix well. For our transfections, we transfect at most 500 ng of plasmid DNA. Be sure that DNA is of high purity and concentrated enough that it can be pipetted in small volumes (from .5 uL- 1 uL). Anything larger will dilute the final concentration. </LI><LI> Dilute transfection reagent into unsupplemented DMEM to create the mastermix. We use 1.65 uL Lipofectamine per 500 ng of plasmid DNA per well. So, if transfecting n conditions/wells, dilute 1.65 * n transfection reagent. </LI><LI> Add 51.65 uL of master mix per tube and pipette gently to mix.  If pooling, mix 51.65 * # of conditions pooled. Let complexes form for 20 minutes in the hood. </LI><LI> While complexes are forming, cells must be seeded into a 24 well plate at 100,000 cells in 500 uL. <B> Note. </B> If transfecting in <B> glass </B> bottom well plates, you must pretreat with gelatin (see menu).</LI><LI> Trypsinize and count confluent cells. Centrifuge at 1180 rpm for 4 minutes and resuspend in fresh media. Dilute to the 100,000 cells/ 500 uL. Add 500 uL of cells to each well. </LI><LI> When complexes are done forming, add dropwise to each well. Swirl the entire plate to mix, and then place into an incubator. Small molecule induction can be done 24 hrs later, and gene expression can be assayed 48 hrs later. </LI> *Adapted from Invitrogen, <I> Lipofectaime® 2000 Reagent Product Manual.</I><DIV class="section" id="m2bio"><H1>RNA Transfection</H1><OL><LI> Label enough eppendorf tubes for the number of conditions/wells you are transfecting plus one for a master mix of transfection reagent for single stranded RNA diluted in serum-free media plus one for a master mix of transfection reagent for double stranded RNA in serum-free media. If possible, try to pool conditions/wells, if doing multiple wells of the same condition. </LI><LI> Aliquot 50 uL of unsupplemented DMEM to every tube. If you are pooling n conditions/wells, aliquot 50*n uL. For your mastermix(s), aliquot 50* total number of conditions/wells of unsupplemented DMEM. If some wells receive single stranded RNA and some receive double stranded RNA, create a mastermix for the amount of conditions for each RNA type. </LI><LI> Dilute RNA in tubes containing 50 (or 50*n) uL unsupplemented DMEM. Be sure to mix well. Be sure that RNA is concentrated enough that it can be pipetted in small volumes (from .5 uL- 1 uL). Anything larger will dilute the final concentration. For our transfections, we standardize around transfection of 25 pmol of labeled RNA. When transfecting one well with a double stranded RNA (gate:output) and a single stranded RNA (input), we make the vesicles in separate tubes, and for example, 1X input would be 25 pmol of input diluted in 50 (or 50*n) uL unsupplemented DMEM. </LI><LI> Dilute transfection reagent into unsupplemented DMEM to create the mastermix. We use 1.33 uL of Lipofectamine RNAiMAX per 25 pmol of dsRNA per well. So if transfecting n conditions/wells, dilute 1.33 * n uL transfection reagent in mastermix. </LI><LI> For single stranded RNA, we use Lipofectamine 2000 transfection reagent. We use 1.65 uL Lipofectamine 2000 per 25 pmol of ssRNA per well. So if transfecting n conditions/wells, dilute 1.65* n uL transfection reagent in mastermix. </LI><LI> Add 51.33 uL of Lipofectamine RNAiMAX mastermix to each tube containing double stranded RNA for complex formation. Add 51.65 uL of Lipofectamine 2000 mastermix to each tube containing single stranded RNA for complex formation. </LI><LI> While complexes are forming, cells must be seeded into a 24 well plate at 100,000 cells in 500 uL plating volume. Note. If transfecting in glass bottom well plates, you must pretreat with gelatin (see menu) </LI><LI> Trypsinize and count confluent cells. Centrifuge at 1180 rpm for 4 minutes and resuspend in fresh supplemented media. Dilute to the 100,000 cells/ 500 uL. Add 500 uL of cells to each well. </LI><LI> When complexes are done forming, add dropwise to each well. Add single stranded RNA complexes and double stranded RNA complexes separately to a well receiving both types of RNA. Swirl the entire plate to mix, and then place into an incubator. </LI><LI> Since we use fluorescently labeled RNAs, microscopy and flow cytometry can be performed immediately following transfection, keeping in mind that the cells take a few hours to become adherent to the plate. </LI></OL> *Adapted from Invitrogen, <I> Transfecting Stealth™ or siRNA into Mammalian Cells using Lipofectamine™ RNAiMAX. </I></DIV><DIV class="section" id="m4bio"><H1>Gelatin Pretreatment</H1>
This protocol is for use in 24 well, glass bottom, plate format. It can be scaled up or down depending on what size dish you are using.
<OL><LI>Pipette  x uL  of .1 (w/v) % sterile, ultrapure gelatin into each well. </LI><LI> Let the well plate incubate for at least 20 mins before aspirating off the gelatin. </LI><LI> Plates can then be used for transfection, in particular those that will be used for microscopy. </LI></OL></DIV><DIV class="section" id="m5bio"><H1> Passaging Cells</H1>
Note. This protocol is for culturing cells in 100mm^2 round dishes. For other culture volumes, the amounts of reagents required will be different. 

Before moving cells from the incubator into the hood,
<OL><LI> Wipe down hood with 70% EtOH. </LI><LI> Pre-warm reagents in a 37 °C water bath. For a full 500 mL bottle of media this can take up to 20 mins to fully warm. </LI><LI> Anything moving into the hood, especially from the water bath, must be washed off with 70% EtOH. </LI></OL>
Once your hood and reagents are ready,
<OL><LI> Move cells from incubator to hood. </LI><LI> Aspirate off media using a sterilized (autoclaved), glass pastuer pipette, or a disposable aspirating pipette. </LI><LI> Optional: Wash cells gently using 2.5 mL PBS, then aspirate. </LI><LI> Add 2.5 mL of trypsin to cells. Incubate at most 1 min at 37°C. Work quickly, as over trypsinization will damage the cells.</LI><LI> Add 9.5 mL of supplemented media to inactivate trypsin. Pipette to get cells into a single cell suspension. </LI><LI> Pipette cells into a conical tube and take note of the final volume (which should be 12mL). This will dictate how much is transferred to a new cell culture dish. If splitting 1:6, then you need 2 mL of cell solution. </LI><LI> After transferring cells to new culture dishes, add supplemented media to a final volume of 12-13 mL. </LI><LI> Label dishes with Name, Date, Cell type, splitting ratio, and passage number. </LI><LI> Place cells into incubator. </LI><LI> Destroy unused cells by aspirating into the vacuum trap, or by adding bleach. </LI><LI> Wash out of hood with 70% EtOH, close the sash, and turn on UV Light. </LI></OL></DIV><DIV class="section" id="m6bio"><H1> Preparing Samples for Flow Cytometry </H1>
Note, this protocol is for use with 24 well plates. Trypsin volumes will need to scale according to the plate format you use.
<OL><LI> Take well plate from the incubator. </LI><LI> Aspirate off the supernatant in each well. If cells have not adhered, pipette the supernatant into a conical tube. </LI><LI> Add 500 uL of trypsin to each well. </LI><LI> Incubate at 37°C for at most a minute. </LI><LI> Add 1 mL of supplemented media to stop the trypsinization. </LI><LI> Transfer cells to the appropriate conical tube. </LI><LI> Centrifuge for 1180 rpm for 4-5 minutes. </LI><LI> Apsirate supernatant carefully, not disturbing the cell pellet (it will be rather small). </LI><LI> Resuspend  in 1 mL of PBS, aiming for a single cell suspension. Any clumps of cells could clog the flow cytometer. </LI><LI> Transfer from the conical tube to the 5 mL polystyrene tube for FACS. Any other tube (such as polypropylene) will not work. </LI></OL></DIV><DIV class="section" id="m7bio"><H1> Cryopreservation and Cell Thawing </H1><H2><B> Cryopreservation</B></H2>
This protocol requires the use of Dimethyl Sulfoxide (DMSO), and isopropyl alcohol (IPA). Before using these reagents, you should read a materials safety data sheet (MSDS) for proper handling.
Also, for cryopreservation, you ideally want to preserve cells that are at a low passage number (P4 or less). 

<OL><LI> Label as many cryovials as you need (if aiming to freeze down 10*10<SUP>6</SUP> cells at 1*10<SUP>6</SUP> cells/mL, you need 10 cryovials) with the the passage number of the cells, concentration, initials, date, and cell type. Be sure to also label the caps with something to be able to pick out cryovials easily. </LI><LI> Prepare freezing medium by mixing unsupplemented media (DMEM) (50% v/v),  FBS (40% v/v) , and  DMSO (10% v.v)  DMSO is hygroscopic, so you must work quickly when transferring from stocks. Be sure to seal the bottle tightly. </LI><LI> Count cells using a hemocytometer (see menu) or an automated cell counter. Dilute cells to a concentration of 1.0*10<SUP>6</SUP>- 1.5*10<SUP>6</SUP> cells/mL. </LI><LI> Centrifuge cells at 1180 rpm for 4-5 minutes. Aspirate supernatant and resuspend in freezing media, to a single cell suspension. </LI><LI> While cells are in the centrifuge, be sure to check the IPA in the Mr. Frosty freezing chamber. It must be changed every 5th use. Be sure to dispose of IPA properly. </LI><LI> Aliquot 1 mL cells to cryovials. </LI><LI> Place croyvials in the Mr. Frosty chamber. Freeze at -80°C from 6-24 hrs. Then transfer to either liquid nitrogen tanks, or a -140°C freezer. </LI><LI> A few days later, thaw a vial to check cell viability. </LI><H2><B> Thawing Mammalian Cells </B></H2>
Mamallian cells must be thawed quickly for optimal viability. Since you are removing cells from the -140°C, you need to move quickly and consider extra protection such as a face shield (cryovials can explode). 
<OL><LI> Have 12 mL of media pre-aliquotted and warmed. In addition, warm the stock bottle of media. </LI><LI> Move media into hood. </LI><LI> Working quickly, retrieve cells from -140°C freezer. Slightly untwist cap, to reduce the chance of the cryovial exploding. </LI><LI> Carefully dip the bottom of the cryvial into the 37°C waterbath. DO NOT SUBMERGE or you will contaminate your cells. </LI><LI> When there is a small ice-chunk, bring cells to hood (cells will continue to thaw in transport). </LI><LI> Pipette cells into pre-warmed aliquot of media. Centrifige at 1180 rpm for 5 minutes. </LI><LI> Aspirate supernatant carefully, to not disturb the cell pellet. Resuspend in 12 mL fresh, warmed media. </LI><LI> Place into culture dish, and observe for confluency over the next few days, changing media after about two days. In our experience, 1-1.2*10<SUP>6</SUP> cells reaches confluency in 2-3 days.</LI></OL><DIV class="section" id="i1bio"><H1>Fluorescent Microscopy</H1><P>
All fluorescent microscopy observations were carried out using a Zeiss Axiovert 200M epifluorescence microscope equipped with a 1344 x 1024 pixel cooled ORCA-ER CCD camera (Hamamatsu Corporation) and a Zeiss 10X objective. False coloring was performed by capturing images with the appropriate colors and assembling them into larger mosaics using custom software (GFP filters: 470/40 excitation &amp; 525/50 emission, DsRedExpress: 565/30 &amp; 620/60). Cells were live-cell imaged in 24-well Tissue Culture treated plastic plates (Corning Corporation) in DMEM media with appropriate supplements as detailed in cell culture protocols.
</P></DIV><DIV class="section" id="i2bio"><H1>Confocal Microscopy</H1><P>
All confocal microscopy observations were carried out using a Leica TCS SP5 within a 37C humidity controlled chamber supplemented with 5% CO<SUB>2</SUB> and a Leica 10x objective with 3.0 zoom enabled. PMT wavelength ranges were set using default spectra for GFP, YFP, Alexa, ROX, mKate, BFP spectrums in Leica AFS Software with laser excitation as follows:
GFP - 488nm, 30%; YFP - 488nm, 30%, Alexa - 488nm, 30%; ROX - 630nm, 40%, mKate - 565nm, 30%, BFP - 405nm, 12%.Cells were live-cell imaged in 24-well glass bottom plates (MatTEK Corporation) treated with 1% gelatin in DMEM media with appropriate supplements as detailed in cell culture protocols.
</P></DIV><DIV class="section" id="i3bio"><H1>Flow Cytometry</H1><P>
All flow cytometry observations were carried out using a LSRFortessa  flow analyzers (BD Biosciences) with the following sets of setting. Alexa and GFP/YFP were measured using a 488 nm Laser, a 505 nm Longpass filter and a 530/30 emission filter in the linear range of PMT values adjusted such that negative cells fell at 10<SUP>2</SUP>. BFP was measured with a 405 nm Laser, a 460 nm Longpass filter and a 480/40 emission filter. mKate and ROX were measured using a 561 nm laser and a 582/15 emission filter. For each sample,  cells were trypsinized according to details in cell culture protocols and approximately 1 x 10<SUP>4</SUP> to ~  1 x 10<SUP>5</SUP> cell events  were  collected. In parallel, Rainbow Calibration Particles (BD Biosciences) were measured in order to equalize the data between different instruments and settings.
</P></DIV><DIV class="section" id="robotbio"><H1>Robotic Automation</H1><P>
Members of our team worked with a TECAN EVO 150 liquid handling robot to automate common biology tasks for iGEM and for others in our labspace.  We worked on a set of Python modules that offer more advanced control of the robot than the default TECAN programming language.  Along the way, we automated the Miniprep DNA separation protocol, set up automated transfections, and wrote a program to compile spreadsheet input into robot code.  The video below shows the robot running one of our protocols.
</P><P><I>Robot at work.  Video is playing at 8x speed.</I></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:MIT/MaterialsAndMethods">http://2012.igem.org/Team:MIT/MaterialsAndMethods</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:MIT/MaterialsAndMethods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:MIT/MaterialsAndMethods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:MIT/MaterialsAndMethods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:MIT/MaterialsAndMethods&amp;oldid=266454" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></OL></DIV></OL></DIV></OL></DIV></OL></DIV></DIV></DIV></DIV></BODY></HTML>