"
Page
Discussion
View source
History
teams
Log in
 
Team:HKU-Hong Kong/Description
From 2011.igem.org
Project
Lab Work
Team
Data
Gratitude
Super Silencer  |  Prepare to be silenced
Overview
Description
Results
BioBricks
Modelling
Attributions
Lab Diaries
Lab Protocol
Safety
Profile
Human Practice
Social Networks
Professors & Advisors
Video
Registry Parts
Sponsors
Acknowledgments
Super Silencer
A. Promoter
Our Team has planned to utilize four different constitutive promoters first. We have selected BBa_J23103, BBa_J23106, BBa_J23109 and BBa_J23116 in the constitutive promoter family. In our project, we would like to select some of the suitable promoters to fine-tune our fusion protein expression so that a better repression activity can be achieved. We have tried to characterize the selected promoters by measuring the mRFP intensity in the plasmid J61002 as a confirmation of promoter activity. The confirmation data of the promoter activity have been shown as follow and input into the Experience page of these promoters. From the experimental data for promoter in this project, we believed that BBa_J23109 and BBa_J23116 are some of the suitable promoters to be used and further investigate.
Comparison of red fluorescence intensity among 4 different promoters
B. Ribosome binding sites
There have been various characterizations of various ribosome binding sites by a number of iGEM teams. The characterization of the ribosome binding sites is as important as the characterization of the promoters to fine tune our fusion protein expression. Unfortunately, due to time constraints and a number of experimental reasons, our team has failed to characterize the ribosome binding sites. As a result, our team finally decided to use BBa_B0034 in our project as its strength is one of the highest in the Registry collection.
C. DNA-binding domain
DNA-binding domain (DBD) is a protein domain that contains at least one motif that recognizes double- or single-stranded DNA. DBD can recognize and bind on a specific DNA sequence, which is called recognition site, with different affinity.
The Tet repressor (tetR) we used in the experiment is a transcription factor for regulating gene expression. It binds to the operator site (tetO) by using a multi-helical DBD of the N-terminal of the proteins, while the C-terminal is for regulation of DNA binding which depends on the co-factors.
Advantage:
high affinity between tetR and tetO... "most efficiently inducible system of transcriptional regulation known so far"
can sensitively switch the affinity of tetR for tetO when [MgTc]+ binds
The binding of tetR on the tetO is regulated by the antibiotic tetracycline. When tetracycline is absent, the DBD of tetR binds to the tetO and transcription halt. If tetracycline enters the cell, it binds magnesium cations (Mg++) to form the [MgTc]+ complex. This complex acts as an inducer to bind the TetR, causing conformational changes of the DBD, causing tetR dissociates from the tetO site.
In structural basis, tetR is a homodimer, which composed of 2 polypeptide chains. Each chain consists of 10 alpha helices.α1 - α3 are the DBD, in which α2 and 3 constitude a classical helix-turn-helix motif. α4 contributes to the hydrophobic center of the DBD and links the DBD to the regulatory domain. α5- α10 are the core for regulation of the binding between tetR of [MgTc]+.
D. Polymerization domain
H-NS proteins are a type of histone-like proteins found in abundance in Gram-negative bacteria (such as E.coli). Phylogenetic studies have shown there are some varieties of H-NS proteins in different bacteria species. Nonetheless, H-NS are involved in regulating transcription by repression, and the structuring of the nucleoid. The regulation of H-NS expression is complex. Its expression is like to be influenced by the external environment as well. However, it is also capable of autoregulating its own transcriptions.
H-NS regulates gene expression negatively; whilst a complete understanding of its mechanism is yet to be established due to limitations of current lab techniques, research has identified a number of common themes: first, H-NS proteins acts on (non-specifically) intrinsically curved DNA sites near the promoter of the gene to be repressed. Intrinsically curved DNA sites near the promoter enable upstream and downstream DNA to be brought in close proximity after the binding of the RNA polymerase on the promoter. Bound H-NS on these regions interacts to form a nucleoprotein complex that traps RNA polymerase, thus effectively stopping transcription. An obvious advantage of mechanisms in which RNA polymerase is trapped on the promoter is that it needs not be recruited from “solution” once repression is relieved, which allows a rapid response to changes in environmental conditions.
E. Achieving Repression
By expressing the tetR:H-NS fusion proteins, the tetR part of the proteins wound recognize and specifically bind to the tetR binding region (tetO2). There are two tet-R binding sites as tet-R form a dimer as in the tet repressor- operator system. As the fusion proteins bound to tet-O, it is expected that the H-NS part of the fusion protein would attract and oligomerize with other native H-NS proteins inside the cell.  With the oligomerization of the fusion proteins and native H-NS, the DNA covered by the oligomers is expected to trap the RNA polymerase during transcription, thus gene (GFP) repression is achieved.
Finally, we have successfully created the following biobricks and submitted to the registry,
Constitutive promoter + tetR
BBa_K544001
BBa_K544002
BBa_K544003
BBa_K544012
Constitutive promoter + tetR-HNS (2-90)
BBa_K544011
BBa_K544013
BBa_K544014
Dkeima Red fluorescent Protein
BBa_K544021
F. What have been completed?
Construct fusion protein (tetR-HNS)
tetR (as a control)
HNS:tetR (Prediction: no significant repression activity)
tetR-HNS
tetR-HNS; HNS (to express more native H-NS)
Adding tetO sites into pEGFP or psfGFP plasmid
Different positions of teto2 sites are designed
Upstream promoter (Site 0, 1), Between promoter and RBS(Site 2), Downstream EGFP (Site 3,4), as shown:
Recombination of tetO2-X GFP from plasmids into E.Coli MG1655 bacterial genome (attTn7 site)
Transform fusion protein carrying plasmid into the MG1655-EGFP strain
Test for GFP expression
References
Rimsky S. Structure of the histone-like protein H-NS and its role in regulation and genome superstructure. Current Opinion in Microbiology [online] 2004 Apr;7(2):109-14. Available: http://www.sciencedirect.com/science/article/pii/S1369527404000141
Dorman CJ. H-NS: A universal regulator for a dynamic genome. Nature Reviews Microbiology [online] 2004 May;2(5):391-400. Available: http://www.nature.com/nrmicro/journal/v2/n5/abs/nrmicro883.html
Dame RT, Wyman C, Wurm R, Wagner R, Goosen N. Structural Basis for H-NS-mediated Trapping of RNA Polymerase in the Open Initiation Complex at the rrnB P1. The Journal of biological chemistry [online] 2002 Jan 18;277(3):2146-50. Available: http://www.jbc.org/content/277/3/2146.full
Retrieved from " http://2011.igem.org/Team:HKU-Hong_Kong/Description "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
