Team:BIT-China/Protocols
PROJECT
Background
Description
Idea & Design
EXPERIMENTS
Regulator
Feedback
Output
Results
MODELING
Overview
Fluorescent Probe Model
H2O2 Decomposition Model
roGFP2-Orp1                         Michaelis equation Model
HUMAN PRACTICES
Overview
Integrated Human Practices
Education & Public Engagement
Collaborations
NOTEBOOK
Lab Book
Methodology / Protocols
Material & Equipment
Measurement / InterLab
Safety
ACHIEVEMENTS
Judging Form
Parts
Improve
Applied Design
Demonstrate
TEAM
Members
Attributions
Gallery
PROTOCOLS
Basic Protocol
1.Protocol for PCR
a. Protocol for PCR with PrimeSTAR (Premix)
1) Prepare the reaction mix in PCR tube:
Template DNA
1µl
Forward primer
2µl
Reverse primer
2µl
PrimeSTAR (Premix)
25µl
ddH2O
20µl
Total volume
50µl
When we ligate two fragments by OE-PCR, we usually add two kinds of fragment DNA 1µl                                 respectively and add ddH20 up to 50µl.
2) Set the program of thermal cycler:
1X
98℃
2min
30X
98℃
10s
30X
55℃
5s
30X
72℃
1000bp/min
1X
72℃
5min
1X
16℃
∞
b. Protocol for PCR with Primer Star HS
1) Prepare the reaction mix in PCR tube:
Template DNA
3µL
Forward primer
3µL
Reverse primer
3µL
Primer Star HS DNA Polymerase
0.5µL
dNTP Mixture
4µL
2 X GC buffer
25µL
ddH2O
11.5µL
Total volume
50µL
When we ligate two fragments by OE-PCR, we usually add two kinds of fragment DNA 1µl                                 respectively and add ddH20 up to 50µl.
2) Set the program of thermal cycler:
1X
98℃
2min
30X
98℃
10s
30X
55℃
5s
30X
72℃
1000bp/min
1X
72℃
5min
1X
16℃
∞
c. Fast pfu fly
1) Prepare the reaction mix in PCR tube:
Template DNA
1µl
Forward primer
2µl
Reverse primer
2µl
dNTP
2µl
5X FastPfu Fly buffer
5µl
FastPfu Fly DNA polymerase
0.5µl
ddH2O
12.5µl
Total volume
25µl
When we ligate two fragments by OE-PCR, we usually add two kinds of fragment DNA 1µl                                 respectively and add ddH2O up to 25µl.
2) Set the program of thermal cycler:
1X
94℃
5min
30X
94℃
30s
30X
55℃
30s
30X
72℃
1500bp/min
1X
72℃
10min
1X
16℃
∞
d. Taq mix
1) Prepare the reaction mix in PCR tube:
ddH2O
6.5µl
Forward primer
0.5µl
Reverse primer
0.5µl
2×Taq PCR Master Mix
7.5µl
Total volume
15µl
2) Select positive clones and put them in the PCR tubes.
3) Make sure the bacteria are in the cPCR tubes.
4) Set the program of thermal cycler:
1X
94℃
10min
25X
94℃
30s
25X
55℃
30s
25X
72℃
1000bp/min
1X
72℃
10min
1X
16℃
∞
2. Agarose gel electrophoresis
1) Measure TAE buffer.
2) Weigh agarose powder and add agarose powder to the buffer.
3) Melt the mixture in a microwave until the solution becomes clear.
4) Let the solution cool to about 40-50℃ and add nucleic acid dye.
5) Pour the solution into the gel tray with corresponding comb.
6) Let the gel cool until it becomes solid.
7) Carefully pull out the comb and place the gel in the electrophoresis chamber.
8) Add enough TAE Buffer.
Pipette DNA samples mixed with appropriate amount of loading buffer and dye (Gene                                 Finder) into wells on the gel.
10) Run the gel at 60V for 20 minutes.
3. Gel purification
Protocol for Gel Purification is based on the introduction of Tiangen Gel Extraction                                 Kit.
4. Digestion
1) Prepare reaction mix according to the following table
Enzyme digestion system (20μL)
Fragment or plasmid need to digest
16μL
corresponding enzymes of upstream restriction Enzyme cutting site
1μL
corresponding enzymes of downstream restriction Enzyme cutting site
1μL
Enzyme buffer (Buffer)
2μL
2) incubate at recommended temperature (37℃) for at least 1 hour.
3) Purify the digestion product by inactivating at 80°C.
5.Ligation
1) Prepare reaction mix according to the following table
T4 DNA Ligase system (10μL)
digested fragment/vector
8.5μL
T4 DNA ligase
0.5μL
10x T4 DNA ligase buffer
1μL
2) Ligate the mix overnight at 16℃.
Note: when 3A assembly is performed, we always adjust the dose of each fragment                                 according to their concentration to make sure the amounts of them are same to each                                 other. Vector can be added less than fragment but no less than 1.5μL.
6. Transformation
a. Yeast
1) The cells are added into 2 to 5 ml of liquid YPD medium and incubate overnight at                                 30°C with a shaker.
2) The cultured cells are seeded in fresh YPD medium at a volume of 10% by volume.
3) Cultured at 30°C and 200 rpm for 4-5 hours. Note: To make the cells complete at                                 least twice mitosis, the conversion efficiency will remain unchanged after 3-4 mitosis.
4) Add the cell culture solution to a 1.5 ml centrifuge tube, centrifuge at 4000 rpm                                 for 5 minutes or 5000 rpm for 2mins.
5) Discard the supernatant and resuspend the cells with 1 ml of sterile water and                                 centrifuge again.
6) The supernatant is discarded and the cells are resuspended with 1 ml of 100mM LiAc                                 and allowed to stand for five minutes.
7) 4000 rpm for three minutes, discard the supernatant.
8) The SS-DNA is boiled for five minutes and rapidly cooled on ice.
9) Add the conversion mixture in sequence: 240ul PEG(50% w/v), 36ul 1.0M LiAc; 10ul                                 SS-DNA（10.0mg/ml); x ul DNA（1~5ul); (74-x) ul Sterile water; Total:360ul;
10) Centrifuge the tube for one minute to allow the ingredients to mix well.
11) Incubated in a 30 degree incubator for 30 minutes; 20 to 30 minutes in a 42 degree                                 water bath.
Note: The time of thermal shock and the type of bacteria related to the specific                                 circumstances can be optimized to improve the conversion efficiency.
12) 4000 rpm for three minutes, and the supernatant is removed with a micropipette.
13) 4000 rpm for five minutes, discard the medium and wash twice with sterile water.
14) 100ul of sterile water to resuspend the cells (action to be gentle), coated plate,                                 and then cultured 2 to 4 days.
b. E.coli
1) Take the competent cells (BMTOP10/DH5α) out of the Ultra-low temperature freezer.
2) Put the competent cells in the ice for 5 minutes.
3) Add 10ul of the ligation product to the competent cells and put them in the ice for                                 30 minutes.
4) Next put the competent cells in water about 42℃ for 90 seconds. Then put the                                 competent cells in ice for 5 minutes immediately.
5) Add 600ul LB-media to the competent cells and put them into the Incubator shaker for                                 1 hour in 37℃.
6) Take 100ul of the competent cells into the LB plate containing corresponding                                 antibiotics and spread them on the plate evenly.
7) Put the LB plate to the Incubator for 18 hours.
7. Protocol for mini-prep
Protocol for mini-prep is based on the introduction of Tiangen Mini-prep Kit8.
8. Yeast Genome extraction
Protocol for Yeast Genome extraction is based on the introduction of TIANamp Yeast DNA                                 Kit.
Special protocol
1. fluorescence measurement
Protocol for fluorescence measurement is based on the introduction of Reactive oxygen                                 species (ROS) testing Kit.
a. Yeast induced culture
1) add the yeast into 2 to 5 mL of liquid medium and incubated overnight in a shaker.
2) The cultured cells are seeded in fresh medium at a volume of 10% by volume.
b. ROS detection by DCFH-DA staining
For the detection of accumulated ROS, incubate DCFH-DA with yeast cells for 30 min                                 (37℃), then wash the cells with PBS twice and analyzed by flow cytometry and                                 Fluorescent microplate reader.
2. Yeast Survival Rate Test
Spotting assays and viability tests
For spotting assays, yeast strains recovered from −80°C vials were shaken in SGR-URA                                 liquid medium overnight for revitalization. Then they were adjusted to the identical                                 OD600 of ∼0.5 and serially diluted to 10−1, 10−2, and 10−3. A 3-μl amount of each                                 diluted yeast culture was spotted on SD-URA or other plates. SGR-URA plates served as                                 the spotting control. Generally, pictures were taken after 2 or 3 d, depending on the                                 growth.
For survival tests, yeast cells were shaken in test tubes and grown until they reached                                 the exponential phase, and then different apoptotic inducers were added to induce                                 apoptosis. After the cells were treated by Mn for 12 h or H2O2 for 100 min, the number                                 of surviving colonies was determined by plating a small aliquot of the treated cultures                                 on yeast extract/peptone/dextrose plates or selective plates (SD or SGR dropout                                 plates).
3. Yeast Growth Measurement
1) The cells are added into 2 to 5 ml of liquid YPD medium and incubate overnight at                                 30°C with a shaker.
2) The cultured cells are seeded in fresh YPD medium at a volume of 10% by volume.
3) When it grow to the right time, take some of it to do the test through                                 Fluorescent microplate reader.
4. dCas9
1) PCR of a plasmid containing dCas9 to obtain a gene fragment of dCas9 protein.
2) Synthesis of g RNA by chemical synthesis.
3) Construct the skeleton of the plasmid by OE-PCR.
4) Using dCas9 plasmid as template, PCR was used to recover the fragment and the vector                                 backbone was added to a 6.5 μl mix system at a molar ratio of 2:1, and then 1 μl of taq                                 Ligase was added, 50 degree reaction for 1 h, PCR instrument Temperature control, after                                 the reaction is completed, the large intestine transformation is coated with the                                 ampicillin plate.
5) Colony PCR, get the target plasmid
5. qPCR
1) RNA extraction
1. Take 2mL of cells cultured to log phase for centrifugation at 1000g for 5 minutes                                 at 4°C
2. Discard the supernatant, add 2mL BufferSE/LyticaseMixture, and keep it at 30 degrees                                 for half an hour.
3. Centrifuge at 3400g for 5 minutes, discard the supernatant, add 350u                                 LBufferYRL/2-mercaptoethanol (20uL/mL) and 50g glass beads, beat at maximum speed for 5                                 minutes, centrifuge at 13000g for 3 minutes, transfer the supernatant to a new                                 centrifuge tube.
4. Add an equal amount of 70% ethanol, transfer the sample to the adsorption column,                                 centrifuge at 13000g for 1 minute, discard the supernatant.
5. Add 700uL WashBufferI, centrifuge for 1 minute, discard the supernatant
6. Add 500uLWashBufferII wash, centrifuge at 13000g for 1 minute, discard the                                 supernatant, repeat once, centrifuge again for 2 minutes.
7. Pack the adsorption column with a clean 1.5mL centrifuge tube, collect the RNA with                                 30-50uLDEP-treated water, and centrifuge for 1 minute to get the RNA solution.
2) RT-PCR
System
ingredient
volume（μL)
ddH2O
6
5×Primrscript RT Master Mix
2
TotalRNA
2
Total
10
Program
37℃
15min
85℃
5sec
3) qPCRr
System
ingredient
volume（μL)
2×SYBR Green qPCR MasterMix
10
T
1
F
1
R
1
ROX Reference Dye
1
DdH2O
6
Total
20
Program
