<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Demonstration</TITLE></P><DIV id="app"><DIV id="main"><DIV id="top-zone"><DIV id="top-description"><H1>Demonstration</H1><P>
                    In our biological part, we aimed to design a biosensor that can specifically detect biomarkers in samples. Through investigation, we know that the aptamer could well convert the protein into other more detecable signals.
                </P></DIV></DIV><DIV id="main-content"><DIV id="main-word"><DIV id="third-box" class="hidden-sm-and-down"><DIV id="third-menu" style="width: 150px"><A class="third-menu-item" href="javascript:void(0);" onclick="jump(1)">Coupling</A><A class="third-menu-item" href="javascript:void(0);" onclick="jump(2)">Decoupling</A><A class="third-menu-item" href="javascript:void(0);" onclick="jump(3)">Probe</A><A class="third-menu-item" href="javascript:void(0);" onclick="jump(4)">Test</A></DIV></DIV><DIV id="detail"><DIV class="mid"><H2>Coupling</H2><P>
                            We obtained some aptamers of CK-MB protein using the SELEX method, so we decided to use CK-MB protein to construct a system to prove our design.</P><P>
                            We chose Apt.21 to perform the aptamer and CK-MB protein affinity experiments first. Its principle is similar to ELISA. The gradient concentration protein was coated in a 96-well plate, and the bovine serum albumin (BSA) was blocked and then ligated with biotin. After that, horseradish peroxidase (HRP) with streptavidin was added to connect with the aptamer. Then ,add TMB developer to react with HRP in blue. Finally , measure the light absorption value, if the aptamer has a good affinity for the protein, the OD value will increase as the protein concentration increases.
                        </P><DIV class="shadow"><DIV class="half"><DIV class="half1"><P>Fig 1. OD value increases as protein concentration increases</P></DIV><DIV class="half2"><P>
                                        Fig 2. According to the data on the left, we calculate the dissociation constant Kd of the aptamer.
                                    </P></DIV></DIV></DIV><P>
                            According to the experimental results, the dissociation constant is small enough, which means that the ligand binds strongly enough with the CK-MB protein to conduct the competition experiment (protein couple aptamer, complementary chain decoupling).
                        </P></DIV><DIV class="mid"><H2>Decoupling</H2><P>
                            Before the competition experiment, we carried out some preliminary experiments, to explore the relevant factors that affect the competition experiment. We use a complementary strand modified with a fluorophore to couple with an aptamer, decoupling complementary strands with a certain concentration of protein, then measuring the decoupled fluorescence intensity. Change the condition of an experiment and explore its effect on protein decoupling by comparing the fluorescence intensity.
                        </P><P>
                            We designed a comparative experiment of complementary strand lengths, comparing the effects of 15 bp and 25 bp complementary strands on protein decoupling, and the reaction temperatures at room temperature and 37 ℃.
                        </P><DIV class="shadow"><DIV class="half"><DIV class="half1"><P>
                                        Fig 3. The effect of the length of the complementary strand on the coupling
                                    </P></DIV><DIV class="half2"><P>
                                        Fig 4. The effect of the temperature on the coupling
                                    </P></DIV></DIV></DIV><P>
                            The following are some of the most suitable reaction conditions we have explored.
                            Complementary chain binding length: 15 bp
                            Reaction temperature: 37℃
                            Number of washings at the end of the reaction: 5 times
                            Aptamer complementary chain reaction time: 1h
                            CK-MB reaction time: 1.5h
                            There are still many conditions that may affect the reaction. We have not explored it. We will continue to optimize it in the future.
                        </P><P>
                            Based on the previously defined complementary chain sequences, we performed an aptamer complementary chain ligation experiment.
                        </P><DIV class="shadow"><P>
                                Figure 5. The concentration of the agarose gel is 3%. Lane 1 is the aptamer, lane 2 is the complementary chain, lane 3 is the maker, lane 4 is the product after the ligation reaction, and lane 4 is shown. Bringing the lanes on lanes 1 and 2 indicates that the aptamer and the complementary chain are well connected.
                            </P></DIV><P>
                            The competition experiment is to detect the fluorescence value of the supernatant by reacting different concentrations of CK-MB protein with an equal amount of magnetic bead-aptamer-fluorescent complementary chain complex.
                        </P><DIV class="shadow"><P>
                                Figure 6. Within a certain concentration range, as the protein concentration increases, the fluorescence value increases significantly. At a certain concentration, the fluorescence value tends to be flat.
                            </P></DIV><P>
                            The experimental result data proves that the function of the aptamer in the system is normal.
                        </P></DIV><DIV class="mid"><H2>Probe</H2><P>
                            As for the design of microorganism probe, we first carried out the design of engineering bacteria, and we constructed plasmids that could express the fluorescence of green, red and yellow.
                        </P><P>
                            We selected the green fluorescent engineering bacteria for the subsequent probe connection experiment.We used the heterogeneous dual-function crosslinker n-succinimidyl 4-(p-maleimidophenyl) butyrate (SMPB)cross-linking mercaptoyl and amino,to connect engineered bacteria with the modified fluorescence complementary chain, which was detected by flow cytometry.We also explore the results of different dosage of Dimethyl sulfoxide (DMSO), using to dissolve the SMPB.
                        </P><DIV class="shadow"><P>
                                Figure 7. A is the negative control, B is the ligation product using 10 μl SMPB solution and 20 μl coupling buffer, C is the ligation product using 15 μl SMPB solution and 15 μl coupling buffer, and D is the ligation product using 20 μl SMPB solution and 10 μl coupling buffer. The closer the peak is to the left, the lower the fluorescence intensity.
                            </P></DIV><P>
                            The results indicate that the engineered bacteria and complementary chains are successfully linked, and the amount of DMSO in Condition B is the best of the three. Our overall system can be obtained by mixing the engineered bacteria attached to the complementary chain with the magnetic beads attached to the aptamer (the aptamer and magnetic beads are linked by biotin and streptavidin).
                        </P></DIV><DIV class="mid"><H2>Test</H2><P>
                            Before conducting the overall system function test, we conducted experiments on the growth curve of different inoculum concentrations.
                        </P><P>
                            The whole system test was carried out by reacting different concentrations of protein with an equal amount of the system to obtain the supernatant containing the engineered bacteria, inoculated into 20 ml of chlorinase-resistant LB medium at 37℃ shaker.Then ,we measured the growth curve and recorded the fluorescence value.
                        </P><P>
                            It can be seen that different concentrations of the target protein have obvious differences after sensory expression, indicating that the overall system function is normal.
                        </P><DIV class="shadow"><P>
                                Fig 8. We selected the data for the 21st hour, taking the logarithm of the protein concentration as the abscissa and the fluorescence intensity as the ordinate.
                            </P></DIV><P>
                            The above test results demonstrate that our system can effectively perform quantitative detection of disease markers.
                        </P><P>
                            References:
                            [1] Yan Y, Sha W, Sun J, et al. DNA diagnostics with a bacterial reporter probe.[J]. Chemical Communications, 2011, 47(26):7470-7472.
                            [2] Toriello N M, Douglas E S, Thaitrong N, et al. Integrated microfluidic bioprocessor for single-cell gene expression analysis[J]. Proc Natl Acad Sci U S A, 2008, 105(51):20173-20178.
                            [3] Irwin P L, Nguyen L H, Paoli G C, et al. Evidence for a bimodal distribution of Escherichia coli doubling times below a threshold initial cell concentration[J]. Bmc Microbiology, 2010, 10(1):1-14.
                            [4] Hsiao S C, Shum B J, Onoe H, et al. Direct Cell Surface Modification with DNA for the Capture of Primary Cells and the Investigation of Myotube Formation on Defined Patterns[J]. Langmuir, 2009, 25(12):6985-6991.
                            [5] Rebecca Notman †, Massimo Noro ‡, Brendan O'Malley ‡ A, et al. Molecular Basis for Dimethylsulfoxide (DMSO) Action on Lipid Membranes[J]. Journal of the American Chemical Society, 2006, 128(43):13982-13983.
                            [6] Yamashige R, Kimoto M, Takezawa Y, et al. Highly specific unnatural base pair systems as a third base pair for PCR amplification[J]. Nucleic Acids Research, 2012, 40(6):2793-2806.
                            [7] M. Svobodová, A. Pinto, P. Nadal, et al. Comparison of different methods for generation of single-stranded DNA for SELEX processes[J]. Analytical &amp; Bioanalytical Chemistry, 2012, 404(3):835-842.
                        </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>