<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tokyo-NoKoGen_Project_lux_operon"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tokyo-NoKoGen/Project/lux_operon">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tokyo-NoKoGen/Project/lux_operon&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo-NoKoGen/Project/lux_operon&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo-NoKoGen/Project/lux_operon&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tokyo-NoKoGen/Project/lux_operon" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tokyo-NoKoGen/Project/lux operon</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navi"><DIV id="title"><H1><P id="t1">Tokyo-NoKoGen</P><P id="t2">2012</P></H1></DIV><DIV id="menu"><UL><LI><DIV class="mitem"><A class="mlink" href="https://2012.igem.org/Team:Tokyo-NoKoGen">home</A><DIV class="submenu">
welcome to our team wiki! 
				</DIV></DIV></LI><LI><DIV class="mitem"><A class="mlink" href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/overview">project[+]</A><DIV class="submenu"><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/overview">overview</A><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/lux_operon">lux operon</A><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/rhodopsin">rhodopsin</A><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/modelling">modelling</A><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/Future_work">Future work</A></DIV></DIV></LI><LI><DIV class="mitem"><A class="mlink" href="https://2012.igem.org/Team:Tokyo-NoKoGen/team">team[+]</A><DIV class="submenu"><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/team">Members</A><A href="http://www.tuat.ac.jp/">Our university</A><A href="https://igem.org/Team.cgi">Official profile</A><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Team/Photogalary">Photogallery</A></DIV></DIV></LI><LI><DIV class="mitem"><A class="mlink" href="https://2012.igem.org/Team:Tokyo-NoKoGen/Parts">biobrick[+]</A><DIV class="submenu"><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Parts">Parts submitted</A><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Biobrick/attributions">attributions</A></DIV></DIV></LI><LI><DIV class="mitem"><A class="mlink" href="https://2012.igem.org/Team:Tokyo-NoKoGen/Notebook/diary">notebook[+]</A><DIV class="submenu"><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Notebook/diary">diary</A><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Notebook/protocol">protocol</A><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Notebook/safety">safety</A></DIV></DIV></LI><LI><DIV class="mitemL"><A class="mlink" href="https://2012.igem.org/Team:Tokyo-NoKoGen/Humanpractice">humanpractice</A></DIV></LI><LI><DIV class="mitemL"><A class="mlink" href="https://2012.igem.org/Team:Tokyo-NoKoGen/Achievement">achievement</A></DIV></LI></UL></DIV></DIV><DIV id="index"><UL><LI><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/overview">overview</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/lux_operon">lux operon</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/rhodopsin">rhodopsin</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/modelling">modelling</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo-NoKoGen/Project/Future_work">Future work</A></LI><LI><DIV id="ind">project</DIV></LI></UL></DIV><DIV id="container"><DIV id="bg1"><DIV id="bg2"><DIV id="contents"><FONT size="27">lux operon</FONT><H2>Introduction</H2><H3>What is bioluminescence ?</H3>
　Bioluminescence is visible light emission in living organisms mediated by an enzyme catalyst. The bioluminescence has been observed in many different organisms including bacteria, fungi, fish, insects, algae, and squid.(1)

<H3>lux system</H3>
　Lux operon codes for bacterial luciferase (<I>luxAB</I>) and the fatty acid reductase complex (<I>luxCDE</I>) that is responsible for the biosynthesis of the aldehyde substrate for luminescent reaction.



　The <I>luxA</I> and <I>luxB</I> code for α and β subunit of luciferase. Bacterial luciferase catalyzes the oxidation of reduced flavin mononucleotide (FMNH<SUB>2</SUB>) and long-chain fatty aldehyde resulting in the emission of a blue-green light.(1)



FMNH<SUB>2</SUB> +O<SUB>2</SUB> → FMN + RCOOH + H<SUB>2</SUB>O + light



The <I>luxCDE</I> are required for conversion of fatty acid into long chain aldehyde.

<I>luxC</I> : reductase 
<I>luxD</I> : transferase 
<I>luxE</I> :  synthase



the <I>luxG</I> which codes for flavin reductase is very closely linked to the <I>luxE</I> gene 

<H3>advantage of lux system</H3>
Adding substrate is NOT necessary



Light emission occurs AUTOMATICALLY

<H2>Objective</H2><H3>1. Cloning lux operon from <I>Photobacterium phosphoreum</I></H3>
Why we chose lux gene from <I>Photobacterium phosphoreum</I>?
<I>P. phosphoreum</I> (NCMB844) is known that the luminescence intensity is brightest of other strains. (2)

　But maximum luminescence levels require lower temperatures than the other luminescent species, because this species live in the deeper waters of the ocean. (2)
<H4><U>Method</U></H4><H4>Construction of biobrick</H4>
Original lux gene from <I>P. phosphoreum</I> has three illegal restriction sites.


In order to remove restriction site, we chose overlap extension PCR method and designed four primer set.

Lux gene fragment was amplified and four PCR products were connected.

First, we tried to construct arabinose inducible lux gene biobrick.
<H4>Evaluation</H4>
　We first investigate the best culturing condition.
Transformed <I>E coli</I>. TOP10 was precultured into LB medium and incubated at 37℃. After incubating for 12 hour, precultured solution was inoculated into 100 mL LB medium. When OD660 of the culture became 0.6, arabinose(0.25% f.c.) was added to induce and expressed the lux proteins at different temperature(20 ℃ and 30 ℃). After induction, the 500 μL of culture was harvested at certain time and measured light intensity and OD by plate reader.

Fig. Investigation of culturing condition

<H4>Result and discussion</H4>
　Here, we show the result. It is suggested that at lower temperature(20 ℃), lux operon from <I>P. phosphoreum kishitanii</I> can emit brighter light  than at 30 ℃.
　However, since we first measured the intensity of the light at 3 hours after induction, there is still possibility that the <I>E. coli</I> cultured at 30 ℃ emitted brighter light within 3 hours.  
To check the change of light intensity within 3 hours from induction, we operated another experiment. In this time, we measured the intensity every 30 minutes until 3 hours.
 

　The figure below shows the result of second experiment. As we supposed, the intensity of the light emitted by <I>E. coli</I> cultured at 30 ℃ came to its peak within 3 hours. Yet, the peak intensity was much lower than that of 20 ℃. Then we concluded that 20 ℃ is better to express this lux operon.
<H4>Reference</H4>
(1) Meighen EA. (1993) Bacterial bioluminescence: organization, regulation, and application of the lux genes The FASEB Journal vol. 7 no. 11 1016-1022 
(2) Edward A. Meighen Autoinduction of light emission in different species of bioluminescent bacteria Luminescence 1999;14:3–9 

<H3>2. Improvement of lux luminescence</H3>
　We succeeded in constructing arabinose inducible lux gene biobric and found out the suitable culturing condition to express, so we next tried to improve the coli express system by changing the color, strengthening the light intensity, and shortening the reaction time.
<H4>Change the color</H4>
We used LumP, YFP, and GFP proteins to change the color.


What is LumP and YFP ?

　Bacterial luciferase derived from <I>Vibrio fischeri</I>, which is a strain of luminous bacteria, shows brue-green light emission (wavelength is 495 nm). 

　However, <I>Vibrio fischeri</I> strain Y-1 emits longer wavelength light than normal <I>Vibrio fischeri</I> (535 nm, yellow)(3). The yellow shift of light emission wavelength is caused by YFP, which is a secondary emitter protein. YFP does not emit light itself, but it interacts with the luciferase, resulting in a shift of wavelength of light emission. YFP has FMN as a chromophore, and its size is about 28 kDa.

　Furthermore, in luminous bacteria, there are strains that emit light with shorter wavelength than normal <I>Vibrio fischeri</I> (475 nm,blue), such as <I>Photobacterium phosphoreum, Photobacterium kishitanii</I> and <I>Photobacterium leiognathi</I>.

　It is known that blue shift of these luminous bacterium is caused by lumazine protein (LumP). Similar to YFP, LumP shifts the wavelength by interacting with the luciferase. As a chromophore, it has 6,7-dimethyl-8-(1'-D-ribityl)lumazine (DMRL), and its size is about 21kDa.


Learning from previous reports, we decided to use these proteins for changing the color.
<H4>Reference</H4>
(3) Daubner SC, Astorga AM, Leisman GB, Baldwin TO., 1987, Proc Natl Acad Sci U S A.

Yellow light emission of <I>Vibrio fischeri</I> strain Y-1: purification and characterization of the energy-accepting yellow fluorescent protein.
<H4><U>Method</U></H4><H4>Construction of biobrick</H4>
First, we produced the constitutive promoter + lumP + double terminator construct.

Then, we combined the construct to the construct of cloned Lux operon with Pbad promoter.

The construct of Lux operon with <I>luxY</I> is produced by joining the Lux operon and <I>luxY</I> biobrick part.


***BRET***
　Bacterial luciferase from Lux operon emits light by catalyzing oxidization of luciferin. Oxidized luciferin, oxiluciferine, has high energy. When the oxiluciferin transits from high energy state to low energy state, light is emitted.
　On the other hand, fluorescence substance, such as GFP, absorbs the light and becomes high energy state. And when the fluorescence substance transits from high energy state to low energy state, light is emitted.
　If the luciferase exists near the fluorescence substance, Bioluminescence Resonance Energy Transfer (BRET) occurs. BRET is a phenomenon that can be observed when a fluorescence substance is excited by obtaining energy from the bioluminescence. This only happens when the wavelength of bioluminescence is the same as the wavelength that excites the fluorescence substance.

　The wavelength of bacterial luciferase from Lux operon (497 nm) is close to the wavelength that GFP excites(501 nm).
We considered that BRET might occur between the bacterial luciferase and the GFP, so we designed the construct of Lux operon with GFP.<H4><U>Method</U></H4><H4>Construction of biobrick</H4>
The construct of Lux operon with GFP is produced by joining the Lux operon and GFP biobrick part.<H4>Evaluation</H4>
　Transformed <I>E coli.</I> TOP10 was precultured into LB medium and incubated at 37℃. After incubating for 12 hour, precultured solution was diluted with LB medium to the point that OD660 became 0.6, then applied into 96well microtiter plate. After adding arabinose(0.25% f.c.), the plate was shaken at 20 ℃ to express the lux proteins. The spectrum of the light and intensity was measured by plate reader once in an hour.

<H4>Result</H4><U>1.lux+LumP</U>
　We could observe the shift of wavelength from 488 nm to 478 nm (Figure.a). The changing color can clearly recognize by the photograph(Figure.e). And we also achieved to enhance the light intensity. Maximal absorption of Lux+LumP is about 3-fold, comparing to that of lux(cloned).
Furthermore, we demonstrated that the lumP part(Edinburgh 2009, BBa_K216007) could work with the Lux operon from <I>Photobacterium phosphoreum kishitanii</I> strain.
<B>[Figure.a]</B><U>2.lux+GFP</U>
　We observed the maximal absorption of Lux+GFP at 512 nm (Figure.b). We thought that the shift of emitting light wavelength was caused by BRET. After 1 hour culture, peak could not be observed at 512 nm, and we performed the culture in dark condition. So it is possible to occur that GFP absorbes the lux luminescence and emits fluorescence.<B>[Figure.b]</B><U>3.lux+YFP</U>
　We could not observe the yellow light from lux+YFP construct (Figure.c). We assumed that the yellow shift didn’t occur due to difference of luminous bacteria strain (YFP is from <I>Vivrio fischeri</I>.).<B>[Figure.c]</B><U>4.Comparing to lux from BioBrick part(Cambridge 2010, BBa_K325909)</U>
　We also evaluated the difference of the light emission property between our cloned lux operon and lux operon from BioBrick part(BBa_K325909) (Figure.d).
Sequentially, it was revealed that our lux and Cambridge lux had almost same maximal absorption peak and the light intensity.
<B>[Figure.d]</B><B>[Figure.e]</B>Left is lux+GFP, middle is lux(cloned), right is lux+LumP.
<H3>Strengthen the intensity</H3><H4>Rib operon</H4><I>P. phosphoreum</I> has rib operon(<I>ribEBHA</I>) downstream of <I>luxG</I>. Their gene productshave been identified as riboflavin synthase(RibE), DHBP synthase(RibB), lumazine synthase(RibH) and GTP cyclohydrolaseII(RibA).

　Riboflavin is the key substrate for lux operon because riboflavin is the precursor of riboflavin 5'- monophosphate (FMN). FMNH2 is the substrate for emitting light. Synthesis of riboflavin is important to enhance the luminescent. Instead of rib operon from <I>P. phosphoreum</I>, we cloned the rib genes from <I>E. coli</I> BL21(DE3) and constructed rib operon biobrick.
<H4><U>Method</U></H4><H4>Construction of biobrick</H4>
We cloned <I>ribA, ribB, ribC, ribF,</I> and <I>ribDH</I> from <I>E.coli </I>BL21(DE3).
Original rib genes, <I>ribA</I> and <I>ribDH</I>, have illegal restriction site in each genes.


In order to remove restriction site, we chose inverse extension PCR method.

After sequencing of each gene, we started to construct rib operon. First, ribosome binding site (RBS) was ligated with each gene. Second, each genes are ligated. Finally, promoter and double terminator were ligated. We constructed rib operon biobrick.

<H4>Result</H4>
coming soom...
<H4>Reference</H4>
(4) Nack-Do Sung and Chan Yong Lee. The lux Genes and Riboflavin Genes in Bioluminescent System of <I>Photobacterium leiognathi</I> are under Common Regulation Journal of Photoscience (2004), Vol. 11(1), pp. 41-45


<H3>shorten the reaction time</H3>
We would like to make the bioluminescent reaction time faster than BBa_K769011 by constitutively expressing all <I>lux</I> genes except the <I>luxA</I> or <I>luxD</I> gene.

<H4><U>Method</U></H4><H4>Construction of biobrick</H4>
Pconst-<I>lux CDBFEG</I> –DT (deletion of<I> luxA</I>) was constructed by inverse PCR.
 
Primers with restriction site <I>Eco</I>RI and <I>Xba</I>I at 5’ end and <I>Spe</I>I and <I>Pst</I>I at 3’ end was used to clone <I>luxA</I>
Arabinose inducible promoter was added in front of <I>luxA</I>. 
These two parts were connected.

The same process was performed to construct this part.
<H4>Evaluation method</H4>
　Transformed <I>E coli</I>. TOP10 was precultured into LB medium and incubated at 37℃. After incubating for 12 hour, precultured solution was inoculated into 100 mL LB medium. When OD660 of the culture became 0.6, arabinose(0.25% f.c.) was added to induce and expressed the proteins at 20 ℃. After induction, the 500 μL of culture was harvested at every 30 minutes and measured light intensity and OD by plate reader.
<H4>Result</H4>

　As we expected, when only <I>luxA</I> or <I>luxD</I> were  induced, the light intensity reached its peak faster than wild type. However, the maximum intensity of these two construct far lower than wild type. It may be caused by loss of mass balance of lux proteins. We could improve the light emission by expressing <I>luxAB</I> or <I>luxCDE</I> together not only <I>luxA</I> or <I>luxD</I>. 

<DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Tokyo-NoKoGen/Project/lux_operon">http://2012.igem.org/Team:Tokyo-NoKoGen/Project/lux_operon</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tokyo-NoKoGen/Project/lux_operon" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tokyo-NoKoGen/Project/lux_operon" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tokyo-NoKoGen/Project/lux_operon&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tokyo-NoKoGen/Project/lux_operon&amp;oldid=295254" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>