<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Signalling_Lab_Work"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Cambridge/Signalling/Lab_Work">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Signalling/Lab_Work&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Signalling/Lab_Work&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Signalling/Lab_Work&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Signalling/Lab_Work" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Signalling/Lab Work</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="bodytable"><DIV class="menubar"><CENTER><A href="/Team:Cambridge" title="Team:Cambridge"><FONT face="verdana" style="color:#ffffff"><B>Home</B></FONT></A> <A href="/Team:Cambridge/Voltage" title="Team:Cambridge/Voltage"><FONT face="verdana" style="color:#b3c6fe"><B>Voltage</B></FONT></A> <A href="/Team:Cambridge/Signalling" title="Team:Cambridge/Signalling"><FONT face="verdana" style="color:#b3c6fe"><B>Signalling</B></FONT></A> <A href="/Team:Cambridge/Bacillus" title="Team:Cambridge/Bacillus"><FONT face="verdana" style="color:#b3c6fe"><B>Bacillus</B></FONT></A> <A href="/Team:Cambridge/Modelling" title="Team:Cambridge/Modelling"><FONT face="verdana" style="color:#b3c6fe"><B>Modelling</B></FONT></A> <A href="/Team:Cambridge/Protocols" title="Team:Cambridge/Protocols"><FONT face="verdana" style="color:#ffffff"><B>Protocols</B></FONT></A> <A href="/Team:Cambridge/People" title="Team:Cambridge/People"><FONT face="verdana" style="color:#ffffff"><B>People</B></FONT></A> </CENTER></DIV><DIV class="submenu"><CENTER><A href="/Team:Cambridge/Signalling" title="Team:Cambridge/Signalling"><FONT face="verdana" style="color:#444444"><B>Signalling Overview</B></FONT></A> <A href="/Team:Cambridge/Signalling/Vectors" title="Team:Cambridge/Signalling/Vectors"><FONT face="verdana" style="color:#444444"><B>Vectors</B></FONT></A> <A href="/Team:Cambridge/Signalling/Constructs" title="Team:Cambridge/Signalling/Constructs"><FONT face="verdana" style="color:#444444"><B>Constructs</B></FONT></A> <A href="/Team:Cambridge/Signalling/Primers" title="Team:Cambridge/Signalling/Primers"><FONT face="verdana" style="color:#444444"><B>Primers</B></FONT></A> <A href="/Team:Cambridge/Signalling/Experiments" title="Team:Cambridge/Signalling/Experiments"><FONT face="verdana" style="color:#444444"><B>Experiments</B></FONT></A> <STRONG class="selflink"><FONT face="verdana" style="color:#444444"><B>Lab Notebook</B></FONT></STRONG> </CENTER></DIV><TABLE style="background:#444444; padding:15px;"><TABLE align="left" width="border=&quot;0&quot;" style="background:#444444; padding:5px;"><TBODY><TR><TD style="width:100%; height: 400; padding-left: 15px;"><DIV class="contentf"><DIV style="height: 400; background: white; line-height:170% padding: 5px;"><DIV style="color: white; font: 2px;">x</DIV></DIV></DIV></TD></TR><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#July.2C_22nd"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">July, 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Dry_work"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Dry work</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#July.2C_24th"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">July, 24th</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Single_Colony"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Single Colony</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#July.2C_25th"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">July, 25th</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Results_from_Yesterday"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results from Yesterday</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#July.2C_28th"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">July, 28th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Results_from_previous_days"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Results from previous days</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Wet_Work"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Wet Work</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#July.2C_29th"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">July, 29th</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Result_from_yesterday"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Result from yesterday</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#July.2C_30th"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">July, 30th</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Wet_Work_2"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Wet Work</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#July.2C_31st"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">July, 31st</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Wet_work_3"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Wet work</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#August.2C_1st"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">August, 1st</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Wet_Work_4"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Wet Work</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#August.2C_5th"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">August, 5th</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Testing_AHL_degradation_by_aiiA"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Testing AHL degradation by aiiA</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#August.2C_6th"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">August, 6th</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#PCR_parts_of_Agr_operon"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">PCR parts of Agr operon</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#August.2C_9th"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">August, 9th</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Plans_for_Monday"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Plans for Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#Lab_Work"><SPAN class="tocnumber">12.1.1</SPAN><SPAN class="toctext">Lab Work</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Book.2FComputer_Work"><SPAN class="tocnumber">12.1.2</SPAN><SPAN class="toctext">Book/Computer Work</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Longer_Term_Plans"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">Longer Term Plans</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#August.2C_11th"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">August, 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#PCR_Bacillus_RBS"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">PCR Bacillus RBS</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#August.2C_13th"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">August, 13th</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#PCR_Agr_A.2C_B.2C_C.2C_D"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">PCR Agr A, B, C, D</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#August.2C_15th"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">August, 15th</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#PCRing_out_promoters"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">PCRing out promoters</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Digest_and_Ligate_Agr_A_and_C_to_pSB4C5"><SPAN class="tocnumber">15.2</SPAN><SPAN class="toctext">Digest and Ligate Agr A and C to pSB4C5</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#August.2C_16th"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">August, 16th</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#Agr_A_and_C_ligation_to_pSB4C5"><SPAN class="tocnumber">16.1</SPAN><SPAN class="toctext">Agr A and C ligation to pSB4C5</SPAN></A></LI><LI class="toclevel-1 tocsection-36"><A href="#August.2C_18th"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">August, 18th</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Check_promoters"><SPAN class="tocnumber">17.1</SPAN><SPAN class="toctext">Check promoters</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#Transformation_of_AgrA_and_AgrC"><SPAN class="tocnumber">17.2</SPAN><SPAN class="toctext">Transformation of AgrA and AgrC</SPAN></A></LI><LI class="toclevel-1 tocsection-39"><A href="#August.2C_19th"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">August, 19th</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Results_of_AgrA_and_C_transformation"><SPAN class="tocnumber">18.1</SPAN><SPAN class="toctext">Results of AgrA and C transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Lux_parts"><SPAN class="tocnumber">18.2</SPAN><SPAN class="toctext">Lux parts</SPAN></A></LI><LI class="toclevel-1 tocsection-42"><A href="#August.2C_20th"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">August, 20th</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#Check_Lux_components"><SPAN class="tocnumber">19.1</SPAN><SPAN class="toctext">Check Lux components</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#Ligation"><SPAN class="tocnumber">19.2</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-45"><A href="#August.2C_21st"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">August, 21st</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#Transformation_of_ligation_products"><SPAN class="tocnumber">20.1</SPAN><SPAN class="toctext">Transformation of ligation products</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#J04630"><SPAN class="tocnumber">20.2</SPAN><SPAN class="toctext">J04630</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#Plasmid_stocks"><SPAN class="tocnumber">20.3</SPAN><SPAN class="toctext">Plasmid stocks</SPAN></A></LI><LI class="toclevel-1 tocsection-49"><A href="#August.2C_22nd"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">August, 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-50"><A href="#New_PCR_of_promoters"><SPAN class="tocnumber">21.1</SPAN><SPAN class="toctext">New PCR of promoters</SPAN></A></LI><LI class="toclevel-1 tocsection-51"><A href="#August.2C_23rd"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">August, 23rd</SPAN></A></LI><LI class="toclevel-2 tocsection-52"><A href="#Check_promoters_.28after_PCR_from_bacillus_vectors.29"><SPAN class="tocnumber">22.1</SPAN><SPAN class="toctext">Check promoters (after PCR from bacillus vectors)</SPAN></A></LI><LI class="toclevel-1 tocsection-53"><A href="#August.2C_27th"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">August, 27th</SPAN></A></LI><LI class="toclevel-2 tocsection-54"><A href="#New_transformation_of_ligation_products"><SPAN class="tocnumber">23.1</SPAN><SPAN class="toctext">New transformation of ligation products</SPAN></A></LI><LI class="toclevel-1 tocsection-55"><A href="#August.2C_28th"><SPAN class="tocnumber">24</SPAN><SPAN class="toctext">August, 28th</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#Results_from_transformation_of_ligation_products"><SPAN class="tocnumber">24.1</SPAN><SPAN class="toctext">Results from transformation of ligation products</SPAN></A></LI><LI class="toclevel-2 tocsection-57"><A href="#Transformation_of_ligation_products_.28new.29"><SPAN class="tocnumber">24.2</SPAN><SPAN class="toctext">Transformation of ligation products (new)</SPAN></A></LI><LI class="toclevel-1 tocsection-58"><A href="#August.2C_29th"><SPAN class="tocnumber">25</SPAN><SPAN class="toctext">August, 29th</SPAN></A></LI><LI class="toclevel-2 tocsection-59"><A href="#Results_of_transformation_with_our_ligation_products"><SPAN class="tocnumber">25.1</SPAN><SPAN class="toctext">Results of transformation with our ligation products</SPAN></A></LI><LI class="toclevel-2 tocsection-60"><A href="#Single_colony_PCR_to_check_our_transformation"><SPAN class="tocnumber">25.2</SPAN><SPAN class="toctext">Single colony PCR to check our transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#Plate_biobricks_from_MIT"><SPAN class="tocnumber">25.3</SPAN><SPAN class="toctext">Plate biobricks from MIT</SPAN></A></LI><LI class="toclevel-1 tocsection-62"><A href="#September.2C_2nd"><SPAN class="tocnumber">26</SPAN><SPAN class="toctext">September, 2nd</SPAN></A></LI><LI class="toclevel-2 tocsection-63"><A href="#Check_transformation_in_E.coli_of_our_ligated_products"><SPAN class="tocnumber">26.1</SPAN><SPAN class="toctext">Check transformation in E.coli of our ligated products</SPAN></A></LI><LI class="toclevel-1 tocsection-64"><A href="#September.2C_3rd"><SPAN class="tocnumber">27</SPAN><SPAN class="toctext">September, 3rd</SPAN></A></LI><LI class="toclevel-2 tocsection-65"><A href="#Checking_the_insert_of_promoters"><SPAN class="tocnumber">27.1</SPAN><SPAN class="toctext">Checking the insert of promoters</SPAN></A></LI><LI class="toclevel-2 tocsection-66"><A href="#Make_some_stock_of_our_new_biobricks"><SPAN class="tocnumber">27.2</SPAN><SPAN class="toctext">Make some stock of our new biobricks</SPAN></A></LI><LI class="toclevel-2 tocsection-67"><A href="#Transformation_of_new_.22biobricks.22"><SPAN class="tocnumber">27.3</SPAN><SPAN class="toctext">Transformation of new &quot;biobricks&quot;</SPAN></A></LI><LI class="toclevel-1 tocsection-68"><A href="#September.2C_4th"><SPAN class="tocnumber">28</SPAN><SPAN class="toctext">September, 4th</SPAN></A></LI><LI class="toclevel-2 tocsection-69"><A href="#Results_of_new_.22biobricks.22transformation"><SPAN class="tocnumber">28.1</SPAN><SPAN class="toctext">Results of new &quot;biobricks&quot;transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-70"><A href="#New_attempt_of_new_.22biobricks.22_transformation"><SPAN class="tocnumber">28.2</SPAN><SPAN class="toctext">New attempt of new &quot;biobricks&quot; transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-71"><A href="#September.2C_7th"><SPAN class="tocnumber">29</SPAN><SPAN class="toctext">September, 7th</SPAN></A></LI><LI class="toclevel-2 tocsection-72"><A href="#New_PCR"><SPAN class="tocnumber">29.1</SPAN><SPAN class="toctext">New PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-73"><A href="#September.2C_8th"><SPAN class="tocnumber">30</SPAN><SPAN class="toctext">September, 8th</SPAN></A></LI><LI class="toclevel-2 tocsection-74"><A href="#Check_product_of_PCR_from_yesterday"><SPAN class="tocnumber">30.1</SPAN><SPAN class="toctext">Check product of PCR from yesterday</SPAN></A></LI><LI class="toclevel-2 tocsection-75"><A href="#Check_transformation_from_03.2F09"><SPAN class="tocnumber">30.2</SPAN><SPAN class="toctext">Check transformation from 03/09</SPAN></A></LI><LI class="toclevel-1 tocsection-76"><A href="#September.2C_10th"><SPAN class="tocnumber">31</SPAN><SPAN class="toctext">September, 10th</SPAN></A></LI><LI class="toclevel-2 tocsection-77"><A href="#PCR_of_GFP.2BRBS_and_Promoter.2BRBS"><SPAN class="tocnumber">31.1</SPAN><SPAN class="toctext">PCR of GFP+RBS and Promoter+RBS</SPAN></A></LI><LI class="toclevel-2 tocsection-78"><A href="#RBS_screening"><SPAN class="tocnumber">31.2</SPAN><SPAN class="toctext">RBS screening</SPAN></A></LI><LI class="toclevel-1 tocsection-79"><A href="#September.2C_11th"><SPAN class="tocnumber">32</SPAN><SPAN class="toctext">September, 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-80"><A href="#RBS_screening_2"><SPAN class="tocnumber">32.1</SPAN><SPAN class="toctext">RBS screening</SPAN></A></LI><LI class="toclevel-2 tocsection-81"><A href="#RBS_screning_but_single_digest"><SPAN class="tocnumber">32.2</SPAN><SPAN class="toctext">RBS screning but single digest</SPAN></A></LI><LI class="toclevel-2 tocsection-82"><A href="#Backbone_for_ligation"><SPAN class="tocnumber">32.3</SPAN><SPAN class="toctext">Backbone for ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-83"><A href="#September.2C_12th"><SPAN class="tocnumber">33</SPAN><SPAN class="toctext">September, 12th</SPAN></A></LI><LI class="toclevel-2 tocsection-84"><A href="#RBS_screening_.28single_digest.29"><SPAN class="tocnumber">33.1</SPAN><SPAN class="toctext">RBS screening (single digest)</SPAN></A></LI><LI class="toclevel-2 tocsection-85"><A href="#Double_check_of_RBS_S6_and_stock"><SPAN class="tocnumber">33.2</SPAN><SPAN class="toctext">Double check of RBS S6 and stock</SPAN></A></LI><LI class="toclevel-2 tocsection-86"><A href="#Check_more_RBS_W"><SPAN class="tocnumber">33.3</SPAN><SPAN class="toctext">Check more RBS W</SPAN></A></LI><LI class="toclevel-2 tocsection-87"><A href="#Check_PCR_products"><SPAN class="tocnumber">33.4</SPAN><SPAN class="toctext">Check PCR products</SPAN></A></LI><LI class="toclevel-1 tocsection-88"><A href="#September.2C_13th"><SPAN class="tocnumber">34</SPAN><SPAN class="toctext">September, 13th</SPAN></A></LI><LI class="toclevel-2 tocsection-89"><A href="#Grow_RBS_S6"><SPAN class="tocnumber">34.1</SPAN><SPAN class="toctext">Grow RBS S6</SPAN></A></LI><LI class="toclevel-2 tocsection-90"><A href="#RBS_W_screening"><SPAN class="tocnumber">34.2</SPAN><SPAN class="toctext">RBS W screening</SPAN></A></LI><LI class="toclevel-2 tocsection-91"><A href="#Double_check_of_RBS_W6_and_stock"><SPAN class="tocnumber">34.3</SPAN><SPAN class="toctext">Double check of RBS W6 and stock</SPAN></A></LI><LI class="toclevel-1 tocsection-92"><A href="#September.2C_14th"><SPAN class="tocnumber">35</SPAN><SPAN class="toctext">September, 14th</SPAN></A></LI><LI class="toclevel-2 tocsection-93"><A href="#RBS_S6_and_RBS_W6"><SPAN class="tocnumber">35.1</SPAN><SPAN class="toctext">RBS S6 and RBS W6</SPAN></A></LI><LI class="toclevel-1 tocsection-94"><A href="#September.2C_15th"><SPAN class="tocnumber">36</SPAN><SPAN class="toctext">September, 15th</SPAN></A></LI><LI class="toclevel-2 tocsection-95"><A href="#Check_big_stock_of_RBS_S_and_W"><SPAN class="tocnumber">36.1</SPAN><SPAN class="toctext">Check big stock of RBS S and W</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="July.2C_22nd">July, 22nd</SPAN></H1><H2><SPAN class="mw-headline" id="Dry_work">Dry work</SPAN></H2><UL><LI> found assay to test for AIP 
</LI><LI> found shuttle vector for constitutive expression of GFP
</LI><LI> found protocol for Bacillus subtilis transformation
</LI></UL><H1><SPAN class="mw-headline" id="July.2C_24th">July, 24th</SPAN></H1><H2><SPAN class="mw-headline" id="Single_Colony">Single Colony</SPAN></H2><UL><LI> Single colony of I746001 and I746101 tranfered into 10mL LB [with 100μg/mL conc. of Amp.]
</LI><LI> Single colony of I746001 and I746101 streaked from Amp 100 plate to Kan 25 palte (to test for Kan resistance as well)
</LI></UL><P>Incubate overnight at 37°C
</P><H1><SPAN class="mw-headline" id="July.2C_25th">July, 25th</SPAN></H1><H1><SPAN class="mw-headline" id="Results_from_Yesterday">Results from Yesterday</SPAN></H1><UL><LI> Kan resistance of biobricks
</LI></UL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Biobrick
</TH><TH> Antibiotic
</TH><TH> Contration (μg/mL)
</TH><TH> Observations
</TH><TH> Conclusion
</TH></TR><TR><TD> I746001 </TD><TD> Kan </TD><TD> 25 </TD><TD> Many colonies </TD><TD> Kan Resistance OK
</TD></TR><TR><TD> I746101 </TD><TD> Kan </TD><TD> 25 </TD><TD> Many colonies </TD><TD> Kan Resistance OK
</TD></TR><TR><TD> Control PUC19 </TD><TD> Kan </TD><TD> 25 </TD><TD> No colony </TD><TD> No Kan Resistance
</TD></TR></TBODY></TABLE><P><B>Results</B></P><P>Nothing! 1A1 cells were kept in the fridge! B.S. can not be kept in the fridge, low temperatures kill them!
</P><UL><LI>Test if biobrick of last year are really the one we want
</LI></UL><UL><LI> Yesterday : Single colony of I746001 and I746101 transferred in 10mL of LB + Amp (final concentration 100μg/mL)
</LI><LI> Take 8mL of I746001 (AIP sender) and of I746101
</LI><LI> Centrifuge them
</LI><LI> Saved 2ml of each in tubes to regrow and add LB
</LI><LI> Miniprep I746001 and I746101 pellets (plasmid mniprep kit) : same protocol but 60μL of Zyppy Glution Buffer in step 8
</LI></UL><UL><LI> Restriction Digest : use EcoI and Spel restriction enzymes
</LI><LI> Add sequentially : 15μL of SDW + 2μL of 10X Fast Digest Buffer + 2μL of DNA + 1μL of EcoI + 1μL of Spel
</LI><LI> Microfuge tubes
</LI><LI> Incubate for 10min at 37°C
</LI><LI> Heat inactivation of enzymes at 80°C for 5min
</LI></UL><UL><LI> GEL - 20μLof samples + 4μL of dye
</LI></UL><UL><LI> Grow 1A1
</LI><LI> 10mL of LB + 15μL of 1A1
</LI><LI> Grow at 37°C
</LI></UL><H1><SPAN class="mw-headline" id="July.2C_28th">July, 28th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_from_previous_days">Results from previous days</SPAN></H2><UL><LI><B>Result of the gel</B></LI></UL><UL><LI> Lane3 : hyperladder1
</LI><LI> Lane4 : I746101
</LI><LI> Lane5 : I746001
</LI></UL><P>We wanted to check the size of our biobricks. 
For I746101, we have a band of about 3000kb, which is the size of the vectorm and a band of about 2000kb. The size of the biobrick is 2057kp, so it should be ok.
</P><P>For I746001, we hqve the same band of 3000kb (the vector) and a band of about 1000kb. The size of the biobrick is 896kb. 
</P><P>So, the size of our biobricks are ok. 
</P><H2><SPAN class="mw-headline" id="Wet_Work">Wet Work</SPAN></H2><P>- Digest
</P><OL><LI> For biobricks I746001 and I746101, use digest enzymes EcoRI and SpeI, and 2μL of DNA
<DL><DD> Add 15μL of SDW, 2μL of 10X Fast Digest Buffer, 2μL of DNA, 1μL of EcoRI and 1μL of SpeI
</DD></DL></LI><LI> For ppL82 (from plate and tube), use digest enzymes BamHI and PstI and add 15μL of DNA
<DL><DD> Add 2μL of SDW, 2μL of 10X Fast Digest Buffer, 15μL of DNA, 1μL of BamHI and 1μL of PstI
</DD></DL></LI><LI> Microfuge tubes
</LI><LI> Incubate 10 min at 37°C
</LI><LI> Heat shock for 5min at 80°C
</LI></OL><P>- Gel : add 4μL of dye and 21μL of samples
</P><UL><LI><B>New plates</B></LI></UL><P>-Plate I746001 and I746101 on Kn plates from Amp plates from 24/07/2008.
</P><H1><SPAN class="mw-headline" id="July.2C_29th">July, 29th</SPAN></H1><H2><SPAN class="mw-headline" id="Result_from_yesterday">Result from yesterday</SPAN></H2><UL><LI><B>Result from the gel (29/07/2008)</B></LI></UL><UL><LI> Lane8 : ppL82 (plate)
</LI><LI> Lane9 : ppL82 (tube)
</LI><LI> Lane 10 : I746001
</LI><LI> Lane 11 : I746101
</LI><LI> Lane 12 : hyperladderI
</LI></UL><P>The ladder seems to be wrong. So it was really difficult to ckeck the size of our plasmid ppL82. To make, that, we assume that the size of our biobricks was ok, and we estimate the size of our plasmid. The plasmid ppL82 seems to have the right size, but we will have to check again to be able to use a ladder.
</P><H1><SPAN class="mw-headline" id="July.2C_30th">July, 30th</SPAN></H1><H2><SPAN class="mw-headline" id="Wet_Work_2">Wet Work</SPAN></H2><UL><LI><B>Checking our big stock of biobricks and PNZ plasmid</B></LI></UL><P>
We did big culture of biobricks and PNZ8901 plasmid to be able to make stocks. We want to check them before making stocks.
</P><P>- Plamsid miniprep (for I746001, I746101 and PNZ8901 from big flasks
</P><P>- Measure DNA concentration in our samples to decide the volume of DNA we have to add in our preparation
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> 260/280
</TH><TH> DNA concentration (ng/nl)
</TH></TR><TR><TD> I746001 </TD><TD> 1.79 </TD><TD> 50.5
</TD></TR><TR><TD> I746101 </TD><TD> 1.87 </TD><TD> 54.3
</TD></TR><TR><TD> PNZ8901 </TD><TD> 1.84 </TD><TD> 87
</TD></TR></TBODY></TABLE><P>
- Digest
</P><TABLE class="wikitable"><TBODY><TR><TH> For I746001 and I746101
</TH><TH> For PNZ8901 plasmid
</TH></TR><TR><TD> 14μL of SDW
</TD><TD> 14μL of SDW
</TD></TR><TR><TD> 2μL of 10X Fast Digest Buffer
</TD><TD> 2μL of Buffer 3 (Biolabs)
</TD></TR><TR><TD> 2μL of DNA of biobricks
</TD><TD> 2μL of DNA of PNZ8901
</TD></TR><TR><TD> 1μL of EcoRI
</TD><TD> 1μL of PstI
</TD></TR><TR><TD> 1μL of SpeI
</TD><TD> 1μL of SalI
</TD></TR></TBODY></TABLE><P>
- Gel
</P><UL><LI><B>Results from this gel</B></LI></UL><UL><LI> Lane 9 : I746001
</LI><LI> Lane 10 : I746101
</LI><LI> Lane 11 : PNZ8901
</LI><LI> Lane 12 : HyperladderI
</LI></UL><P>Everthing is really too big! There is a problem, either dimerization, either contamination, either a problem in our work. So we are going to run a new gel.
</P><UL><LI><B>New gel to check</B></LI></UL><P>- Miniprep plasmid from growth bottles (I746001, I746101 and PNZ8901); from plates (I746001, I746101 and PNZ8901).
</P><P>- Do single digest for PNZ8901 (one with PstI, one with SalI)
</P><P>- Run a gel with : PNZ8901 from friday, PNZ8901 digest with PstI, PNZ8901 digest with SalI, 3 samples from growth bottles, 3 samples from plates (to check the size of the uncut vectors)
</P><UL><LI><B>Result from this second gel</B></LI></UL><P>The ladders are really bad! But the size of our biobricks and plasmid are too big! So we can not trust these big cultures. They is a problem.  With tests from the first week, we know that biobricks are right, so we are going to grow new cultures from these first cultures, to make stocks. 
</P><P>Concerning the vectors, they are from last year, so we are not sure of what they are. Since we ordered new well defined vectors (we should receive them on friday), we will use them in the next steps to be sure of our work.
</P><H1><SPAN class="mw-headline" id="July.2C_31st">July, 31st</SPAN></H1><H2><SPAN class="mw-headline" id="Wet_work_3">Wet work</SPAN></H2><P>We want to make stocks of biobricks I746001 and I746101, since our big cultures have a problem,  we want to make some stocks from our first plates (which are good).
</P><P>We used original Amp plates from 23/07/08 of I746001 and I7461001. We took 5 single colonies and spotted on new Amp plates and inoculated in 10mL of LB with Amp.
</P><H1><SPAN class="mw-headline" id="August.2C_1st">August, 1st</SPAN></H1><H2><SPAN class="mw-headline" id="Wet_Work_4">Wet Work</SPAN></H2><UL><LI><B>Check biobricks I746001 and I746101</B></LI></UL><P>
- <B>Single colony PCR</B></P><P>We had 5 single colonies for each biobrick fro; yestrday. We are going to PCR each single colony. 
</P><P>Add 1μL of cells, 7 μL of SDW, 10μL of Phusion Flash master mix, 1μL of VF2 primer and 1μL of VR primer.
</P><P>
-<B>Gel</B></P><P>
Add 5μL of PCR products, 14μL of SDW and 1μL of dye.
</P><P>
-<B>Result</B></P><UL><LI> Lane1 : HyperladderI
</LI><LI> Lane2 : I746001 colony 1
</LI><LI> Lane3 : I746001 colony 2
</LI><LI> Lane4 : I746001 colony 3
</LI><LI> Lane5 : I746001 colony 4
</LI><LI> Lane6 : I746101 colony 1
</LI><LI> Lane7 : I746101 colony 2
</LI><LI> Lane8 : I746101 colony 3
</LI><LI> Lane9 : I746101 colony 4
</LI></UL><P>The result is good. Each colony of I746001 is about 1000kb (all the same size), for I746101, the size is good (about 2000kb), except for one single colony for which there is nothing. It could be a problem during the manipulation.
</P><P>
- <B>Check colony 1</B></P><P>We have colony1 for each biobrick. We want to check them.
</P><P>- Plasmid miniprep
</P><P>- Digest with EcoRI and SpeI
</P><P>- Run a gel (for cut plasmid, add 19μL of DNA and 1μL of dye; for uncut plasmids, add 2μL of DNA, 17μL of SDW and 1μL of dye; for PCR product, add 5μL of DNA, 14μL of SDW and 1μL of dye)
</P><UL><LI> Lane1 : Hyperladder1
</LI><LI> Lane2 : I746001 double digest (miniprep product)
</LI><LI> Lane3 : I746101 double digest (miniprep product)
</LI><LI> Lane4 : I746001 uncut (miniprep product)
</LI><LI> Lane5 : I746101 uncut (miniprep product)
</LI><LI> Lane6 : I746001 colony5 (PCR product)
</LI><LI> Lane7 : I746101 colony5 (PCR product)
</LI></UL><P>- Result : everything is ok except for biobrick I746001, we have only one band
</P><P>- New double digest for biobrick I746001 (lane9), new gel : ok!
</P><P>We are sure to have some good biobricks for agr (I746001 and I746101)
</P><H1><SPAN class="mw-headline" id="August.2C_5th">August, 5th</SPAN></H1><H2><SPAN class="mw-headline" id="Testing_AHL_degradation_by_aiiA">Testing AHL degradation by aiiA </SPAN></H2><UL><LI>The product of the aiiA gene AHL-lactonase degrades AHL in many strains of Bacillus, however, it is unclear to what extent (if any) B. subtilis strain 168 degrades AHL. If 168 is good at degrading AHL this will be a problem when our signaling system is eventually moved into strain 168. 
</LI></UL><P>We are testing B. subtilis strain 168’s ability to degrade AHL by exposing synthetic AHL to the strain and then testing its ability to be recognized by the receiver.
</P><UL><LI><B>Materials</B></LI></UL><P>Bacillus subtilis - protease deficient (NPR and APR) derived from strain 168 - grown up in an over night culture of LB. 
</P><P>Synthetic AHL – Sigma K3007 – AHL powder was dissolved in EA buffer and diluted to 10uM concentration
</P><P>E. coli with T9002 AHL receiver grown up in overnight culture of LB
</P><P>All plates contain soft agar with 200uL of E. coli AHL receiver mixed-in. Plates 1,2 and 4 contain Bacillus in fresh LB medium, while plate 5 contains Bacillus in it’s overnight LB. Plate 5 was used to test if an excreted product of Bacillus is responsible for the degradation of AHL. The product of the aiiA gene in Bacillus subtilis is most likely not exported. 
</P><UL><LI>Plate 1: 10uL Bacillus + 10uL AHL
</LI><LI>Plate 2: 10uL Bacillus + 10uL EA buffer (negative control + does EA buffer kill bacillus?)
</LI><LI>Plate 3: 10uL LB + 10uL AHL (positive control)
</LI><LI>Plate 4: 10uL Bacillus + 10uL AHL incubated together for 1.5 hours prior to plating
</LI><LI>Plate 5: 10uL Bacillus in overnight LB + 10uL AHL
</LI></UL><UL><LI><B>Results</B></LI></UL><P>plates visualized 3 hours after AHL/Bacillus inoculation.....
</P><H1><SPAN class="mw-headline" id="August.2C_6th">August, 6th</SPAN></H1><H2><SPAN class="mw-headline" id="PCR_parts_of_Agr_operon">PCR parts of Agr operon</SPAN></H2><P>We received primers for Agr A,B,C, and D today - <A href="http://openwetware.org/wiki/IGEM:Cambridge/2008/Turing_Pattern_Formation/Primers" class="external free" rel="nofollow">http://openwetware.org/wiki/IGEM:Cambridge/2008/Turing_Pattern_Formation/Primers</A></P><P>We must PCR each individual part out of the previous BioBricks (I746001 &amp;I746101) in order to put our bacillus RBS on them. 
</P><P>I was concerned that the region of most homology would be the BioBrick prefix and suffix itself, so I first digested the DNA with Xbal and SpeI. 
</P><P>For some reason the digestion didn't work that well, so I've decided to go ahead and use the uncut BioBrick DNA as template for my reactions.
</P><H1><SPAN class="mw-headline" id="August.2C_9th">August, 9th</SPAN></H1><H2><SPAN class="mw-headline" id="Plans_for_Monday">Plans for Monday</SPAN></H2><H3><SPAN class="mw-headline" id="Lab_Work">Lab Work</SPAN></H3><UL><LI>Be sure that the Spc50 plates are working, as the control for Spc50 grew and shouldn't have. We already know that things can die at Spc100 (the double crossover test from earlier this week) so it is unlikely that the stock is bad. We should test Spc50 again with regular bacillus and with E. coli at 50-100 ug/ml and hope that they die.
</LI><LI>Finish all the agr PCRs, but especially the RBS. 
</LI><LI>Confirm that amyE insertion works, using the amylase/ery tests.
</LI><LI>Confirm that with ECE153 inserted,  we get detectable fluoresence in a microscope.
</LI><LI>Confirm that ECE166 transformation works, using plasmid miniprepping. 
</LI><LI>Find the qiagen miniprep kit or adapt the protocol.
</LI></UL><LI>Confirm that when ECE166 is transformed, we get detectable fluoresence in a microscope.
</LI><H3><SPAN class="mw-headline" id="Book.2FComputer_Work">Book/Computer Work</SPAN></H3><UL><LI>Find good promoters in vectors and order PCR primers to get them out
</LI><LI>Find out how to assay for/make AIP
</LI><LI>Find/Contact Jim Ajioka
</LI></UL><LI>Find out how to do a c-terminal GFP fusion for membrane proteins agrB and agrC
</LI><LI>Can we fuse GFP and still expect them to localize to the membrane?
</LI><LI>AgrB transmembrane topology (<A href="http://www.jbc.org/cgi/content/full/277/38/34736#F1" class="external free" rel="nofollow">http://www.jbc.org/cgi/content/full/277/38/34736#F1</A>).  The signal peptide is not on the N terminus, and both the N and C terminus are intracellular.  
</LI><LI>AgrC transmembrane topology (<A href="http://openurl.ingenta.com/content?genre=article&amp;issn=0950-382X&amp;volume=28&amp;issue=3&amp;spage=655&amp;epage=662" class="external free" rel="nofollow">http://openurl.ingenta.com/content?genre=article&amp;issn=0950-382X&amp;volume=28&amp;issue=3&amp;spage=655&amp;epage=662</A> (see figure 2 B)), the N terminus is probably extracellular.  The C terminus is probably intracellular.  The article doesn't say where the signal peptide is.
</LI><LI>Should we do a antibody tag instead?
</LI><H2><SPAN class="mw-headline" id="Longer_Term_Plans">Longer Term Plans</SPAN></H2><UL><LI>Ligate the RBS into a BB vector so we can build a RBS/GFP construct.
</LI><LI>Put the RBS/GFP construct into ECE166 and ECE 153. 
</LI><LI>Compare the 'out of the box' ECE153 with ECE153 with RBS/GFP biobrick inserts
</LI><LI>Compare the 'out of the box' ECE166 with ECE166 with RBS/GFP biobrick inserts
</LI></UL><H1><SPAN class="mw-headline" id="August.2C_11th">August, 11th</SPAN></H1><H2><SPAN class="mw-headline" id="PCR_Bacillus_RBS">PCR Bacillus RBS</SPAN></H2><P>We created 2 different Bacillus RBS sites simply by primer annealing and extension. Each product includes an 8bp RBS plus the BioBrick prefix and suffix for a total of 54bp. 
The products were visualized on a 3.5% Agarose gel with bioline's hyperladder V. 
</P><P>RBSs is the consensus RBS sequence for Bacillus subtilis, and thus we believe it to be a very strong binding site -   AAAGGAGG
</P><P>RBSw has a 2bp modification from RBSs, which we believe will weaken it -  AGAGGTGG
</P><P>
The amplification of RBSs yielded only one band on the gel. The product was purified using microclean and contained 14.8ng/uL after clean-up.
</P><P>RBSw, however produced 2 bands on the gel. The correct size was gel-extracted and purified. After purification the yield was 12.3 ng/uL.
</P><H1><SPAN class="mw-headline" id="August.2C_13th">August, 13th</SPAN></H1><H2><SPAN class="mw-headline" id="PCR_Agr_A.2C_B.2C_C.2C_D">PCR Agr A, B, C, D</SPAN></H2><P>Doing a PCR directly from the old Biobricks has worked for all parts of the operon. No prior digestion seems to be needed.
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Part </TH><TH> 260/280 </TH><TH> μg/mL
</TH></TR><TR><TD> A </TD><TD> 1.78 </TD><TD> 92.5
</TD></TR><TR><TD> B </TD><TD> 1.87</TD><TD> 96.5
</TD></TR><TR><TD> C </TD><TD> 1.9 </TD><TD> 127.7
</TD></TR><TR><TD> D </TD><TD> 1.82 </TD><TD> 26.3
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="August.2C_15th">August, 15th</SPAN></H1><H2><SPAN class="mw-headline" id="PCRing_out_promoters">PCRing out promoters</SPAN></H2><P>- 4 different promoters : Pxyl, Pspc, Ppac, Pupp
</P><P>- Make master mix enough for 4RXNs
</P><H2><SPAN class="mw-headline" id="Digest_and_Ligate_Agr_A_and_C_to_pSB4C5">Digest and Ligate Agr A and C to pSB4C5</SPAN></H2><P>Previous PCR products of Agr A and C were digested using Fermentas Fast Digest - EcoRI and PstI according to their protocol except a longer time was given to digest. 
</P><P>3uL of AgrA was used and 4.4 uL of AgrC was used and incubated for 40minutes. 
</P><P>10uL totaling to 1ug of pSB4C5 plasmid was also digested and incubated for 10 minutes. 
</P><P>The digestion was visualized on a 0.8% Agarose gel and produced expected sizes:
</P><P>Plasmid: approx 3bk
</P><P>Agr A: approx 700bp
</P><P>Agr C: approx 1300bp
</P><H1><SPAN class="mw-headline" id="August.2C_16th">August, 16th</SPAN></H1><H2><SPAN class="mw-headline" id="Agr_A_and_C_ligation_to_pSB4C5">Agr A and C ligation to pSB4C5</SPAN></H2><P>Ligation was performed using Fermentas Rapid Ligation kit according to their protocol. 
</P><P>After ligation, samples were visualized on a gel, but little product of the correct size was seen. 
</P><P>For agrA we should have gotten a band of size 3700 (3000plasmid + 700 insert)
</P><P>For agrC we should have gotten a band of size 4300 (3000plasmid + 1300insert)
</P><P>The bands of appropriate sizes were extracted for transformation.
</P><H1><SPAN class="mw-headline" id="August.2C_18th">August, 18th</SPAN></H1><H2><SPAN class="mw-headline" id="Check_promoters">Check promoters</SPAN></H2><P>On Friday, PCR out promoters, we want to check them.
</P><P>- Run a gel
</P><P>- Results : good size of bands!!!
</P><H2><SPAN class="mw-headline" id="Transformation_of_AgrA_and_AgrC">Transformation of AgrA and AgrC</SPAN></H2><P>AgrA and C gell-extracted ligation was transformed into Top 10 cells using standard protocol. Puc9 was used as a positive control. 
</P><H1><SPAN class="mw-headline" id="August.2C_19th">August, 19th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_of_AgrA_and_C_transformation">Results of AgrA and C transformation</SPAN></H2><P>Transformation has failed. No colonies were visible for the plates of Agr A or C. The Puc9 positive control grew. 
We believe that the problem was in the gel-extraction between ligation and transformation. Most of our plasmid was probably lost in this step. Next time we will directly use the results from the ligation reaction to transform. Although many bands were seen on the previous gel of the ligation reaction, we will check our transformation growth by single colony PCR to confirm transformation of the plasmid with our correct insert.
</P><H2><SPAN class="mw-headline" id="Lux_parts">Lux parts</SPAN></H2><P>To make the Lux Receiver, we need 4 different parts ;
</P><UL><LI> R0040, TetR repressible promoter
</LI></UL><UL><LI> SO168, luxR + double terminator
</LI></UL><UL><LI> R0062, promoter activated by luxR
</LI></UL><UL><LI> JO4630 (GFP + double terminator)
</LI></UL><P>All these parts have been transformed into E.coli. We want to test them. R004, R0062 and JO4630 have already been tested, it should be fine. 
We received from the MIT R0040, R0062 and S068 already transformed into E.coli, so we want to check these stocks (which are certainly fine) and use them.
For JO4630, we want to double check our transformation.
</P><P>- Plate on antibiotic plates and do LB stocks of single colony from the MIT stock (R0040, R0062 and S0168).
</P><P>- Put on Kan plates 4 different colonies from J04630 (transformation Amp plate) and also incubate these colonies into LB
</P><H1><SPAN class="mw-headline" id="August.2C_20th">August, 20th</SPAN></H1><H2><SPAN class="mw-headline" id="Check_Lux_components">Check Lux components</SPAN></H2><P>- Single colony PCR for :
</P><UL><LI>R0040 (MIT stuff)
</LI></UL><UL><LI>R0062 (MIT stuff)
</LI></UL><UL><LI>4 different colonies of S0168 (from a transformation plate from 12/08)
</LI></UL><UL><LI>4 different colonies of J04630 (from a transformation plate from 12/08)
</LI></UL><P>- Protocol : add 1μL of cells (diluted in water), 10μL of Master Mis, 7μL of SDW, 1μL of VF primer and 1μL of VR primer
</P><P>- Gel PCR products
</P><P>Gel 1
</P><UL><LI> Lane2 : Hyperladder1
</LI><LI> Lane3 : JO4630, colony 1
</LI><LI> Lane4 : JO4630, colony 2
</LI><LI> Lane5 : JO4630, colony 3
</LI><LI> Lane6 : JO4630, colony 4
</LI><LI> Lane7 : HyperladderI
</LI></UL><P>Gel 2
</P><UL><LI> Lane2 : HyperladderI
</LI><LI> Lane3 : ECE190 double digest
</LI><LI> Lane4 : S0168, colony 1
</LI><LI> Lane5 : S0168, colony 2
</LI><LI> Lane6 : S0168, colony 3
</LI><LI> Lane7 : S0168, colony 4
</LI><LI> Lane8 : HyperladderI
</LI><LI> Lane9 : R0040
</LI><LI> Lane10 : R0062
</LI><LI> Lane11 : Ladder 100bp
</LI></UL><DIV class="center"><DIV class="floatnone"><A href="/File:Photoo.gif" title="File:Photoo.gif">300px</A></DIV></DIV><P>- <B>Results</B></P><UL><LI> R0040 and R0062 : one big band of about 300b (expected size 293), OK!
</LI></UL><UL><LI> S0168 : one band of about 400b for the 4 different colonies (expected size 1234!), bad! This plate does not contain S0168
</LI></UL><UL><LI> J04630 (colonies 2 and 4) : one band of about 1100b (expected size 1173), OK!
</LI></UL><UL><LI> J04630 (colony 1) : one good band plus another band... 
</LI></UL><UL><LI> J04630 (colony 3) : one band of about 600b, bad!
</LI></UL><H2><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H2><P>- Materials :
</P><UL><LI>AgrA
</LI><LI>AgrB
</LI><LI>AgrC
</LI><LI>AgrD
</LI><LI>Pupp
</LI><LI>Pspac
</LI><LI>Ppac
</LI><LI>Pxyl
</LI><LI>RBS S
</LI><LI>RBS W
</LI><LI>psB4C5
</LI></UL><P>- Double digest of PCR products
</P><P>- Run vector, AgrA and AgrD on a gel
</P><P>- DNA clean and concentrator for AgrA, B,C and D, promoters
</P><P>- Microclean for both RBS
</P><P>- Nanodrop
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> 260/280 </TH><TH> ng/μL
</TH></TR><TR><TD> AgrA </TD><TD> 1.66 </TD><TD> 16.4
</TD></TR><TR><TD> AgrB </TD><TD> 1.91 </TD><TD> 23.5
</TD></TR><TR><TD> AgrC </TD><TD> 1.99 </TD><TD> 35.9
</TD></TR><TR><TD> AgrD </TD><TD> 2.13 </TD><TD> 4.9
</TD></TR><TR><TD> Pxyl </TD><TD> 1.54 </TD><TD> 5.6
</TD></TR><TR><TD> Ppac </TD><TD> 1.49 </TD><TD> 4.6
</TD></TR><TR><TD> Pspc </TD><TD> 1.62 </TD><TD> 9.6
</TD></TR><TR><TD> Pupp </TD><TD> 1.88 </TD><TD> 8.5
</TD></TR><TR><TD> RBS S </TD><TD> 2.44 </TD><TD> 29
</TD></TR><TR><TD> RBS W </TD><TD> 1.44 </TD><TD> 10.7
</TD></TR></TBODY></TABLE><P>- Extract plasmid annd Agr from gel and clean
</P><P>- Ligation
</P><H1><SPAN class="mw-headline" id="August.2C_21st">August, 21st</SPAN></H1><H2><SPAN class="mw-headline" id="Transformation_of_ligation_products">Transformation of ligation products</SPAN></H2><P>- Spin chemically competent TOP10, add 100μL of CaCl2 solution
</P><P>- Add 5μL of DNA (ligation products), and 1μL of PUC9
</P><P>- Continue the protocol of transformation
</P><H2><SPAN class="mw-headline" id="J04630">J04630</SPAN></H2><P>- Plate good colonies from yesterday on Kan25 and put in LB (for plasmid stock for tomorrow)
</P><H2><SPAN class="mw-headline" id="Plasmid_stocks">Plasmid stocks</SPAN></H2><P>- Plasmid miniprep R0040 and R0062
</P><H1><SPAN class="mw-headline" id="August.2C_22nd">August, 22nd</SPAN></H1><H2><SPAN class="mw-headline" id="New_PCR_of_promoters">New PCR of promoters</SPAN></H2><H1><SPAN class="mw-headline" id="August.2C_23rd">August, 23rd</SPAN></H1><H2><SPAN class="mw-headline" id="Check_promoters_.28after_PCR_from_bacillus_vectors.29">Check promoters (after PCR from bacillus vectors)</SPAN></H2><P>- Gel
</P><UL><LI> Lane1 : Hyperladder 5
</LI><LI> Lane2 : Pxyl
</LI><LI> Lane3 : Pspac
</LI><LI> lane4 : Ppac
</LI><LI> Lane5 : Pupp
</LI></UL><P>Size is ok.
</P><H1><SPAN class="mw-headline" id="August.2C_27th">August, 27th</SPAN></H1><H2><SPAN class="mw-headline" id="New_transformation_of_ligation_products">New transformation of ligation products</SPAN></H2><UL><LI> Products : agrD, Pupp, Pspac, RBS W, RBS S (from ligation with biolabs kit)
</LI></UL><P>- Competent top 10 from the freezer
</P><P>- Add 5μL of ligation products, and 1.2μL of PUC9
</P><H1><SPAN class="mw-headline" id="August.2C_28th">August, 28th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_from_transformation_of_ligation_products">Results from transformation of ligation products</SPAN></H2><P>- nothing on plate, even on control plate
</P><UL><LI>Reasons ? 
</LI></UL><P>- cells non competent anymore (try with fresh competent cells)
</P><P>- not enough DNA (try with 10μL of DNA)
</P><P>- very short ligation products
</P><H2><SPAN class="mw-headline" id="Transformation_of_ligation_products_.28new.29">Transformation of ligation products (new)</SPAN></H2><UL><LI> Products to ligate : Pupp, Pspac, agrD, RBS S and RBS W
</LI></UL><P>- new fresh competent TOP 10
</P><P>- 5μL of DNA (1.5μL of PUC9)
</P><P>- 2h30 in the incubator
</P><H1><SPAN class="mw-headline" id="August.2C_29th">August, 29th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_of_transformation_with_our_ligation_products">Results of transformation with our ligation products</SPAN></H2><UL><LI> Everything grew! Better efficiency with electrop. than with chemical protocol
</LI></UL><H2><SPAN class="mw-headline" id="Single_colony_PCR_to_check_our_transformation">Single colony PCR to check our transformation</SPAN></H2><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>   Transformed products </TH><TH> number of picked colonies from chemical transformation </TH><TH> number of picked colonies from electrop. transformation (neat) </TH><TH> number of picked colonies from electrop. transformation (1:10)
</TH></TR><TR><TD> Pupp </TD><TD> 2 </TD><TD> 2 </TD><TD> 0
</TD></TR><TR><TD> Pspac </TD><TD> 2 </TD><TD> 2 </TD><TD> 0
</TD></TR><TR><TD> RBS S </TD><TD> 3 </TD><TD> 0 </TD><TD> 2
</TD></TR><TR><TD> RBS W </TD><TD> 3 </TD><TD> 0 </TD><TD> 2
</TD></TR><TR><TD> agrD </TD><TD> 3 </TD><TD> 2 </TD><TD> 0
</TD></TR></TBODY></TABLE><P>- Single colony PCR : 13μL of SDW+cells, 5μL of Master Mix, 1μL of VF and 1μL of VR (and plate each single colony)
</P><P>- Load a gel (1.3% agarose) : 5μL of PCR products + 1μL of dye (only 1μL of 100b ladder)
</P><P>In the death plasmid, the VF-VR size is about 280b.
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>   Transformed products </TH><TH> size of the product (with cutting sites) </TH><TH> expected size after PCR (about)
</TH></TR><TR><TD> Pupp </TD><TD> 255 </TD><TD> 480
</TD></TR><TR><TD> Pspac </TD><TD> 125 </TD><TD> 350
</TD></TR><TR><TD> RBS S </TD><TD> 56 </TD><TD> 280
</TD></TR><TR><TD> RBS W </TD><TD> 56 </TD><TD> 280
</TD></TR><TR><TD> agrD </TD><TD> 200 </TD><TD> 430
</TD></TR></TBODY></TABLE><UL><LI> Result : nothing, even no ladder, problem with the gel!
</LI></UL><P>- run again on a e-gel
</P><UL><LI> Lane1 :ladder 100bp
</LI><LI> Lane2 : Pupp colony 1
</LI><LI> Lane3 : Pupp colony 3
</LI><LI> Lane4 : Pspac colony 1
</LI><LI> Lane5 : Pspac colony 3
</LI><LI> Lane6 : RBS S colony 1
</LI><LI> Lane7 : RBS S colony 4
</LI><LI> Lane8 : RBS W colony 1
</LI><LI> Lane9 : RBS W colony 4
</LI><LI> Lane10 : agrD colony 1
</LI><LI> Lane11 : agrD colony 4
</LI><LI> Lane12 : HyperladderI
</LI></UL><UL><LI>Result : nothing, just the primers! Problem with our PCR
</LI></UL><H2><SPAN class="mw-headline" id="Plate_biobricks_from_MIT">Plate biobricks from MIT</SPAN></H2><UL><LI>E0840
</LI><LI>B0014
</LI><LI>I712007
</LI><LI>C0012
</LI><LI>B0015
</LI><LI>C0061
</LI><LI>R0063
</LI></UL><H1><SPAN class="mw-headline" id="September.2C_2nd">September, 2nd</SPAN></H1><H2><SPAN class="mw-headline" id="Check_transformation_in_E.coli_of_our_ligated_products">Check transformation in E.coli of our ligated products</SPAN></H2><P>- Single colony PCR with VF and VR : add 10μL of SDW, 5μL of MM, 1μL of VF, 1μL of VR and 3μL of cells
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> inserted products </TH><TH> number of colonies to check (chemical transf.) </TH><TH> number of colonies to check (electrop. transf.)
</TH></TR><TR><TD> RBS S </TD><TD> 3 </TD><TD> 2
</TD></TR><TR><TD> RBS W </TD><TD> 3 </TD><TD> 2
</TD></TR><TR><TD> Pspac </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> Pupp </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> agrD </TD><TD> 3 </TD><TD> 2
</TD></TR></TBODY></TABLE><P>- Gel (3% of agarose)
</P><UL><LI> Result
</LI></UL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> gel </TH><TH> lane </TH><TH> inserted product (name of the colony) </TH><TH> observation </TH><TH> conclusion
</TH></TR><TR><TD> top </TD><TD> 3 </TD><TD> RBS S (1) </TD><TD> one band (260bp) </TD><TD> size of the insert, RBS too small to see on a gel
</TD></TR><TR><TD> top </TD><TD> 4 </TD><TD> RBS S (2) </TD><TD> one band (260bp) </TD><TD> size of the insert, RBS too small to see on a gel
</TD></TR><TR><TD> top </TD><TD> 5 </TD><TD> RBS S (4) </TD><TD> nothing </TD><TD> ?
</TD></TR><TR><TD> top </TD><TD> 6 </TD><TD> RBS S (5) </TD><TD> nothing </TD><TD> ?
</TD></TR><TR><TD> top </TD><TD> 7 </TD><TD> RBS W (1) </TD><TD> 2 band (260bp + 350bp) </TD><TD> size of the insert, RBS too small to see on a gel + something else?
</TD></TR><TR><TD> top </TD><TD> 8 </TD><TD> RBS W (2) </TD><TD> one band (350bp) </TD><TD> too big
</TD></TR><TR><TD> top </TD><TD> 9 </TD><TD> RBS W (4) </TD><TD> one band (260bp) </TD><TD> size of the insert, RBS too small to see on a gel
</TD></TR><TR><TD> top </TD><TD> 10 </TD><TD> RBS W (5) </TD><TD> one band (260bp) </TD><TD> size of the insert, RBS too small to see on a gel
</TD></TR><TR><TD> top </TD><TD> 11 </TD><TD> Pspac (1) </TD><TD> 2 band (260bp + 350bp) </TD><TD> maybe problem with products loaded on gel (exactly the same bands than for RBS W)
</TD></TR><TR><TD> top </TD><TD> 12 </TD><TD> Pspac (2) </TD><TD> one band (450bp) </TD><TD> size of Pupp?
</TD></TR><TR><TD> top </TD><TD> 13 </TD><TD> Pspac (3) </TD><TD> one band (450bp) </TD><TD> size of Pupp?
</TD></TR><TR><TD> bottom </TD><TD> 3 </TD><TD> Pspac (4) </TD><TD> nothing </TD><TD> ?
</TD></TR><TR><TD> bottom </TD><TD> 4 </TD><TD> Pupp (1) </TD><TD> one band (400bp) </TD><TD> size of Pspac?
</TD></TR><TR><TD> bottom </TD><TD> 5 </TD><TD> Pupp (2) </TD><TD> one band (400bp) </TD><TD> size of Pspac?
</TD></TR><TR><TD> bottom </TD><TD> 6 </TD><TD> Pupp (3) </TD><TD> one band (slightly lower than 400bp) </TD><TD> size of Pspac?
</TD></TR><TR><TD> bottom </TD><TD> 7 </TD><TD> Pupp (4) </TD><TD> one band (400bp) </TD><TD> size of Pspac?
</TD></TR><TR><TD> bottom </TD><TD> 8 </TD><TD> agrD (1) </TD><TD> 2 bands (380 and 450bp) </TD><TD> big band is ok
</TD></TR><TR><TD> bottom </TD><TD> 9 </TD><TD> agrD (2) </TD><TD> 2 bands (380 and 450bp) </TD><TD> big band is ok
</TD></TR><TR><TD> bottom </TD><TD> 10 </TD><TD> agrD (3) </TD><TD> no bands </TD><TD> ?
</TD></TR><TR><TD> bottom </TD><TD> 11 </TD><TD>agrD (4) </TD><TD> strong band (450bp) </TD><TD> ok
</TD></TR><TR><TD> bottom </TD><TD> 12 </TD><TD> P2 </TD><TD> one band (350bp) </TD><TD> ok
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="September.2C_3rd">September, 3rd</SPAN></H1><H2><SPAN class="mw-headline" id="Checking_the_insert_of_promoters">Checking the insert of promoters</SPAN></H2><P>We ligated promoters into death vector, and transforned into Top 10. Our transformation worked, but it was hard to say with the result of yesterday if we had the good insert. Moreover, it is possible that we inverted Pupp and Pspac. So we are going to make a new single colony PCR with the primers of promoters (we used the same single colonies than yesterday).
</P><P>We also want to PCR again all promoters (Pspac, Pxyl, Pupp and Ppac) from plasmids to have more.
</P><P>Picture of the gel with the different promoters
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> promoter </TH><TH> colony </TH><TH> primers </TH><TH> name
</TH></TR><TR><TD> Pspac </TD><TD> 1 </TD><TD> Pspac primers </TD><TD> A
</TD></TR><TR><TD> Pspac </TD><TD> 1 </TD><TD> Pupp primers </TD><TD> B
</TD></TR><TR><TD> Pspac </TD><TD> 2 </TD><TD> Pspac primers </TD><TD> C
</TD></TR><TR><TD> Pspac </TD><TD> 2 </TD><TD> Pupp primers </TD><TD> D
</TD></TR><TR><TD> Pupp </TD><TD> 1 </TD><TD> Pupp primers </TD><TD> E
</TD></TR><TR><TD> Pupp </TD><TD> 1 </TD><TD> Pspac primers </TD><TD> F
</TD></TR><TR><TD> Pupp </TD><TD> 3 </TD><TD> Pupp primers </TD><TD> G
</TD></TR><TR><TD> Pupp </TD><TD> 3 </TD><TD> Pspac primers </TD><TD> H
</TD></TR></TBODY></TABLE><P>- Single colony PCR : add 10μL of SDW, 5μL of MM, 1μL + 1μL of primers, 3μL of cells
</P><P>- Gel
</P><UL><LI> lane 3 : hyperladderI
</LI><LI> lane 4 : Pspac (colony1) with Pspac primers
</LI><LI> lane 5 : Pspac (colony1) with Pupp primers
</LI><LI> lane 6 : Pspac (colony2) with Pspac primers
</LI><LI> lane 7 : Pspac (colony2) with Pupp primers
</LI><LI> lane 8 : Pupp (colony1) with Pupp primers
</LI><LI> lane 9 : Pupp (colony1) with Pspac primers
</LI><LI> lane 10 : Pupp (colony3) with Pupp primers
</LI><LI> lane 11 : Pupp (colony3) with Pspac primers
</LI><LI> lane 12 : HyperladderI
</LI></UL><P>- Results
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> promoter </TH><TH> colony </TH><TH> primers </TH><TH> observation </TH><TH> Conclusion
</TH></TR><TR><TD> Pspac </TD><TD> 1 </TD><TD> Pspac primers </TD><TD> nothing </TD><TD> it is in fact Pupp
</TD></TR><TR><TD> Pspac </TD><TD> 1 </TD><TD> Pupp primers </TD><TD> band (about 280bp) </TD><TD> Pupp OK
</TD></TR><TR><TD> Pspac </TD><TD> 2 </TD><TD> Pspac primers </TD><TD> nothing </TD><TD> it is in fact Pupp
</TD></TR><TR><TD> Pspac </TD><TD> 2 </TD><TD> Pupp primers </TD><TD> band (about 280bp) </TD><TD> Pupp OK
</TD></TR><TR><TD> Pupp </TD><TD> 1 </TD><TD> Pupp primers </TD><TD> nothing </TD><TD> it is in fact Pspac
</TD></TR><TR><TD> Pupp </TD><TD> 1 </TD><TD> Pspac primers </TD><TD> band (about 180bp) </TD><TD> Pspac OK
</TD></TR><TR><TD> Pupp </TD><TD> 3 </TD><TD> Pupp primers</TD><TD> nothing </TD><TD> it is in fact Pspac
</TD></TR><TR><TD> Pupp </TD><TD> 3 </TD><TD> Pspac primers </TD><TD> band (about 180bp) </TD><TD> Pspac OK
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Make_some_stock_of_our_new_biobricks">Make some stock of our new biobricks</SPAN></H2><UL><LI> Pupp (colony1) into the death vector, transformed into TOP10
</LI><LI> Pspac (colony1) into the death vector, transformed into TOP10
</LI><LI> agrD (colony5) into the death vector, transformed into TOP10
</LI></UL><P>- Grow this single colony into 10mL of LB (Cm35)
</P><H2><SPAN class="mw-headline" id="Transformation_of_new_.22biobricks.22">Transformation of new &quot;biobricks&quot;</SPAN></H2><P>- transform into E.coli :
</P><UL><LI>agrA
</LI><LI>agrB
</LI><LI>agrC
</LI><LI>Pxyl
</LI></UL><H1><SPAN class="mw-headline" id="September.2C_4th">September, 4th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_of_new_.22biobricks.22transformation">Results of new &quot;biobricks&quot;transformation</SPAN></H2><P>- Nothing on our plate, except for the control plate. 
</P><H2><SPAN class="mw-headline" id="New_attempt_of_new_.22biobricks.22_transformation">New attempt of new &quot;biobricks&quot; transformation</SPAN></H2><P>- transform into E.coli these ligated products:
</P><UL><LI>agrA
</LI><LI>agrB
</LI><LI>agrC
</LI><LI>Pxyl
</LI></UL><H1><SPAN class="mw-headline" id="September.2C_7th">September, 7th</SPAN></H1><H2><SPAN class="mw-headline" id="New_PCR">New PCR</SPAN></H2><P>- New stocks of Master Mix
</P><P>- PCR agrA, agrB, Pxyl, Pspac, Ppac and Pupp
</P><H1><SPAN class="mw-headline" id="September.2C_8th">September, 8th</SPAN></H1><H2><SPAN class="mw-headline" id="Check_product_of_PCR_from_yesterday">Check product of PCR from yesterday</SPAN></H2><UL><LI> Lane3 : Hyperladder I
</LI><LI> Lane4 : agrA
</LI><LI> Lane5 : agrB
</LI><LI> Lane6 : Pxyl
</LI><LI> Lane7 : Pspac
</LI><LI> Lane8 : Ppac
</LI><LI> Lane9 : Pupp
</LI><LI> Lane10 Hyperladder IV
</LI></UL><UL><LI> Results : everything is ok
</LI></UL><H2><SPAN class="mw-headline" id="Check_transformation_from_03.2F09">Check transformation from 03/09</SPAN></H2><P>After several days of incubation, we have a few colonies on our transformed plates (agrA, B and C).
</P><UL><LI> Single colony PCR 
</LI></UL><P>- Add 13μL of cells diluted into SDW, 5μL of MM, 1μL of VF and 1μL of VR
</P><UL><LI> Gel
</LI></UL><H1><SPAN class="mw-headline" id="September.2C_10th">September, 10th</SPAN></H1><H2><SPAN class="mw-headline" id="PCR_of_GFP.2BRBS_and_Promoter.2BRBS">PCR of GFP+RBS and Promoter+RBS</SPAN></H2><P>- PCR
</P><P>- Run a 1.2% agarose gel with 1μL of sample
</P><UL><LI>Lane 3 : HyperladderI
</LI><LI>Lane 4 : GFP + RBS 1A
</LI><LI>Lane 5 : GFP + RBS1B
</LI><LI>Lane 6 : GFP + RBS 2A
</LI><LI>Lane 7 : GFP + RBS 2B
</LI><LI>Lane 8 : Pupp + RBS 1
</LI><LI>Lane 9 : Pupp + RBS 2 
</LI><LI>Lane 10: HyperladderIV
</LI></UL><UL><LI> Result
</LI></UL><H2><SPAN class="mw-headline" id="RBS_screening">RBS screening</SPAN></H2><UL><LI>5 colonies from chemical transformation + 6 colonies from electroporation transformation for RBS S
</LI><LI>5 colonies from chemical transformation + 6 colonies from electroporation transformation for RBS W
</LI></UL><P>- PCR : 11μL of SDW + 5μL of MM + 1+1μL of primers (RBS detect + VR) + 2μL of cells (program iGEM34)
</P><P>- Gel1
</P><UL><LI>Lane2 : RBSS1
</LI><LI>Lane3 : RBS S2
</LI><LI>Lane4 : RBS S3
</LI><LI>Lane5 : RBS S4
</LI><LI>Lane6 : RBS S5
</LI><LI>Lane7 : RBS S6
</LI><LI>Lane8 : HyperladderIV
</LI><LI>Lane9 : RBS S7
</LI><LI>Lane10 : RBS S8
</LI><LI>Lane11 : RBS S9
</LI><LI>Lane12 : RBS S10
</LI><LI>Lane13 : RBS S6 with VF and VR primers
</LI><LI>Lane14 : RBS S11
</LI></UL><P>- Gel2
</P><UL><LI>Lane2 : RBSW1
</LI><LI>Lane3 : RBS W2
</LI><LI>Lane4 : RBS W3
</LI><LI>Lane5 : RBS W4
</LI><LI>Lane6 : RBS W5
</LI><LI>Lane7 : RBS W6
</LI><LI>Lane8 : HyperladderIV
</LI><LI>Lane9 : RBS W7
</LI><LI>Lane10 : RBS W8
</LI><LI>Lane11 : RBS W9
</LI><LI>Lane12 : RBS W10
</LI><LI>Lane13 : RBS W6 with VF and VR primers
</LI><LI>Lane14 : RBS W11
</LI></UL><UL><LI>Result
</LI></UL><P>- Nothing, except for the PCR with VF and VR primers, either our primers are not right, either we have no insert. But the gel is not good enough to see very well, we will try to run a new gel.
</P><H1><SPAN class="mw-headline" id="September.2C_11th">September, 11th</SPAN></H1><H2><SPAN class="mw-headline" id="RBS_screening_2">RBS screening</SPAN></H2><P>- We run our PCR products from yesterday on a new 1.8% agarose gel 
</P><P>- Gel1
</P><UL><LI>Lane2 : RBSS1
</LI><LI>Lane3 : RBS S2
</LI><LI>Lane4 : RBS S3
</LI><LI>Lane5 : RBS S4
</LI><LI>Lane6 : RBS S5
</LI><LI>Lane7 : RBS S6
</LI><LI>Lane8 : HyperladderIV
</LI><LI>Lane9 : RBS S7
</LI><LI>Lane10 : RBS S8
</LI><LI>Lane11 : RBS S9
</LI><LI>Lane12 : RBS S10
</LI><LI>Lane13 : RBS S6 with VF and VR primers
</LI><LI>Lane14 : RBS S11
</LI></UL><P>- Gel2
</P><UL><LI>Lane2 : RBSW1
</LI><LI>Lane3 : RBS W2
</LI><LI>Lane4 : RBS W3
</LI><LI>Lane5 : RBS W4
</LI><LI>Lane6 : RBS W5
</LI><LI>Lane7 : RBS W6
</LI><LI>Lane8 : HyperladderIV
</LI><LI>Lane9 : RBS W7
</LI><LI>Lane10 : RBS W8
</LI><LI>Lane11 : RBS W9
</LI><LI>Lane12 : RBS W10
</LI><LI>Lane13 : RBS W6 with VF and VR primers
</LI><LI>Lane14 : RBS W11
</LI></UL><UL><LI>Result
</LI></UL><P>- Nothing, except for the PCR with VF and VR primers, either our primers are not right, either we have no insert.
</P><H2><SPAN class="mw-headline" id="RBS_screning_but_single_digest">RBS screning but single digest</SPAN></H2><P>- Grow some colonies of Top10 transformed with RBS into 10mL of LB with antibiotic
</P><P>We want to check our transformation with single digest. If we have self ligation in our transformation, we will loose the XbaI cutting site.
</P><H2><SPAN class="mw-headline" id="Backbone_for_ligation">Backbone for ligation</SPAN></H2><P>- New pellets in 600μL of SDW
</P><P>- Plasmid miniprep
</P><H1><SPAN class="mw-headline" id="September.2C_12th">September, 12th</SPAN></H1><H2><SPAN class="mw-headline" id="RBS_screening_.28single_digest.29">RBS screening (single digest)</SPAN></H2><P>- Plasmid miniprep 6 colonies of RBS S and 6 colonies of RBS W (without endo wash buffer)
</P><P>- Nanodrop
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> product </TH><TH> concentration (ng/μL) </TH><TH> 260/280 </TH><TH> quantity of DNA to add to have about 300ng (μL) </TH><TH> added SDW for single digest (μL)
</TH></TR><TR><TD> RBS W1 </TD><TD> 57.1 </TD><TD> 1.57 </TD><TD> 6 </TD><TD> 11
</TD></TR><TR><TD> RBS W3 </TD><TD> 54.1 </TD><TD> 1.53 </TD><TD> 6 </TD><TD> 11
</TD></TR><TR><TD> RBS W5 </TD><TD> 15.1 </TD><TD> 1.85 </TD><TD> 17 </TD><TD> 0
</TD></TR><TR><TD> RBS W 7</TD><TD> 21.8 </TD><TD> 1.66 </TD><TD> 15 </TD><TD> 2
</TD></TR><TR><TD> RBS W9</TD><TD> 21.8 </TD><TD> 1.78 </TD><TD> 15 </TD><TD> 2
</TD></TR><TR><TD> RBS W11 </TD><TD> 13.8 </TD><TD> 1.73 </TD><TD> 17 </TD><TD> 0
</TD></TR><TR><TD> RBS S2 </TD><TD> 19.5 </TD><TD> 1.7 </TD><TD> 15 </TD><TD> 2
</TD></TR><TR><TD> RBS S4 </TD><TD> 82.13 </TD><TD> 2.57 </TD><TD> 4 </TD><TD> 13
</TD></TR><TR><TD> RBS S6 </TD><TD> 44.7 </TD><TD> 1.64 </TD><TD> 7 </TD><TD> 10
</TD></TR><TR><TD> RBS S8 </TD><TD> 104.0 </TD><TD> 1.48 </TD><TD> 3 </TD><TD> 14
</TD></TR><TR><TD> RBS S10 </TD><TD> 13.1 </TD><TD> 1.82 </TD><TD> 17 </TD><TD> 0
</TD></TR><TR><TD> RBS S11 </TD><TD> 20.1 </TD><TD> 1.73 </TD><TD> 15 </TD><TD> 2
</TD></TR></TBODY></TABLE><P>- Single digest : with EcoRI and XbaI, add SDW and DNA (according to the previous table, 300mg of DNA), 2μL of fast digest buffer, and 1μL of enzyme
</P><P>- Gel1
</P><UL><LI>Lane2 : HyperladderI
</LI><LI>Lane3 : RBS W1 cut with EcoRI
</LI><LI>Lane4 : RBS W1 cut with XbaI
</LI><LI>Lane5 : RBS W3 cut with EcoRI
</LI><LI>Lane6 : RBS W3 cut with XbaI
</LI><LI>Lane7 : RBS W7 cut with EcoRI
</LI><LI>Lane8 : RBS W7 cut with XbaI
</LI><LI>Lane9 : RBS W9 cut with EcoRI
</LI><LI>Lane10 : RBS W9 cut with XbaI
</LI><LI>Lane11 : RBS W9 uncut 
</LI><LI>Lane12 : supercoiled ladder
</LI></UL><P>
- Gel2
</P><UL><LI>Lane2 : HyperladderI
</LI><LI>Lane3 : RBS S4 cut with EcoRI
</LI><LI>Lane4 : RBS S4 cut with XbaI
</LI><LI>Lane5 : RBS S6 cut with EcoRI
</LI><LI>Lane6 : RBS S6 cut with XbaI
</LI><LI>Lane7 : RBS S8 cut with EcoRI
</LI><LI>Lane8 : RBS S8 cut with XbaI
</LI><LI>Lane9 : RBS S11 cut with EcoRI
</LI><LI>Lane10 : RBS S11 cut with XbaI
</LI><LI>Lane11 : RBS S4 uncut 
</LI><LI>Lane12 : supercoiled ladder
</LI></UL><UL><LI>Result
</LI></UL><P>The size of our plasmid is about 3000bp (ok on the gel)
</P><UL><LI>RBS W
</LI></UL><P>- W1 with EcoRI : cut
</P><P>- W1 with XbaI : uncut, self ligation
</P><P>- W3 with EcoRI : cut
</P><P>- W3 with XbaI : uncut, self ligation
</P><P>- W7 with EcoRI : impossible to see
</P><P>- W7 with XbaI : uncut, self ligation
</P><P>- W9 with EcoRI : cut
</P><P>- W9 with XbaI : uncut, self ligation
</P><UL><LI>RBS S
</LI></UL><P>- S4 with EcoRI : cut
</P><P>- S4 with XbaI : uncut, self ligation
</P><P>- S6 with EcoRI : cut
</P><P>- S6 with XbaI : 2 band, this plasmid is partially cut!!! Transformation with RBS S
</P><P>- S8 with EcoRI : cut
</P><P>- S8 with XbaI : uncut, self ligation
</P><P>- S11 with EcoRI : cut
</P><P>- S11 with XbaI : uncut, self ligation
</P><H2><SPAN class="mw-headline" id="Double_check_of_RBS_S6_and_stock">Double check of RBS S6 and stock</SPAN></H2><P>- Grow RBS S6 in LB with Cm35
</P><H2><SPAN class="mw-headline" id="Check_more_RBS_W">Check more RBS W</SPAN></H2><P>- Grow RBS W2, 4, 6, 8, 10 in 10mL of LB with antibiotic
</P><H2><SPAN class="mw-headline" id="Check_PCR_products">Check PCR products</SPAN></H2><P>- Gel (2μL of each product)
</P><UL><LI>Lane1 : λ ladder
</LI><LI>Lane2 : agrA
</LI><LI>Lane3 : agrB
</LI><LI>Lane4 : agrC
</LI><LI>Lane5 : rep
</LI><LI>Lane6 : pSB4C5 1
</LI><LI>Lane7 : pSB4C5 2
</LI><LI>Lane8 : Pxyl
</LI><LI>Lane9 : Ppac
</LI><LI>Lane10 : RBS S
</LI><LI>Lane11 : RBS W
</LI><LI>Lane12 : HyperladderI
</LI></UL><UL><LI>Results
</LI></UL><P>- agr B : 2 bands?
</P><P>- agrC : ok
</P><P>- rep : small band but good size
</P><P>- backbones : ok
</P><P>- promoters : ok
</P><P>- RBS : nothing
</P><H1><SPAN class="mw-headline" id="September.2C_13th">September, 13th</SPAN></H1><H2><SPAN class="mw-headline" id="Grow_RBS_S6">Grow RBS S6</SPAN></H2><P>- Put the 10mL of LB with RBS S6 from yesterday into 200mL of LB with antibiotic
</P><P>- Incubate at 37°C with shaking
</P><H2><SPAN class="mw-headline" id="RBS_W_screening">RBS W screening</SPAN></H2><P>- Plasmid miniprep of RBS W2, 4, 6, 8, 10 9witouh endo wash buffer)
</P><P>- Nanodrop
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> product </TH><TH> concentration (ng/μL) </TH><TH> 260/280 </TH><TH> quantity of DNA to add to have about 300ng (μL) </TH><TH> added SDW for single digest (μL)
</TH></TR><TR><TD> RBS W2 </TD><TD> 42.4 </TD><TD> 1.62 </TD><TD> 8 </TD><TD> 9
</TD></TR><TR><TD> RBS W4 </TD><TD> 35.1 </TD><TD> 1.61 </TD><TD> 10 </TD><TD> 7
</TD></TR><TR><TD> RBS W6 </TD><TD> 41.0 </TD><TD> 1.67 </TD><TD> 8 </TD><TD> 9
</TD></TR><TR><TD> RBS W8 </TD><TD> 72.8 </TD><TD> 1.61 </TD><TD> 5 </TD><TD> 12
</TD></TR><TR><TD> RBS W10 </TD><TD> 51.0 </TD><TD> 1.60 </TD><TD> 6 </TD><TD> 11
</TD></TR></TBODY></TABLE><P>- Single digest with EcoRI and XbaI
</P><P>- Gel1
</P><UL><LI>Lane2 : HyperladderI
</LI><LI>Lane3 : RBS W2 cut with EcoRI
</LI><LI>Lane4 : RBS W2 cut with XbaI
</LI><LI>Lane5 : RBS W4 cut with EcoRI
</LI><LI>Lane6 : RBS W4 cut with XbaI
</LI><LI>Lane7 : RBS W5 cut with EcoRI
</LI><LI>Lane8 : RBS W5 cut with XbaI
</LI><LI>Lane9 : RBS W6 cut with EcoRI
</LI><LI>Lane10 : RBS W6 cut with XbaI
</LI><LI>Lane11 : RBS W8 uncut 
</LI><LI>Lane12 : supercoiled ladder
</LI></UL><P>
- Gel2
</P><UL><LI>Lane2 : HyperladderI
</LI><LI>Lane3 : RBS W8 cut with EcoRI
</LI><LI>Lane4 : RBS W8 cut with XbaI
</LI><LI>Lane5 : RBS W10 cut with EcoRI
</LI><LI>Lane6 : RBS W10 cut with XbaI
</LI><LI>Lane7 : RBS W11 cut with EcoRI
</LI><LI>Lane8 : RBS W11 cut with XbaI
</LI><LI>Lane9 : RBS W8 uncut 
</LI><LI>Lane10 : supercoiled ladder
</LI></UL><UL><LI>Result
</LI></UL><P>- W2 with EcoRI : cut
</P><P>- W2 with XbaI : uncut, self ligation
</P><P>- W4 with EcoRI : cut
</P><P>- W4 with XbaI : uncut, self ligation
</P><P>- W5 with EcoRI : cut
</P><P>- W5 with XbaI : uncut, self ligation
</P><P>- W6 with EcoRI : cut
</P><P>- W6 with XbaI : cut (2 bands), transformation ok for RBS W6
</P><P>- W8 with EcoRI : impossible to see
</P><P>- W8 with XbaI : impossible to see
</P><P>- W10 with EcoRI : cut
</P><P>- W10 with XbaI : maybe cut, it seems to be 2 bands, but quite difficult to see
</P><P>- W11 with EcoRI : cut
</P><P>- W11 with XbaI : uncut, self ligation
</P><H2><SPAN class="mw-headline" id="Double_check_of_RBS_W6_and_stock">Double check of RBS W6 and stock</SPAN></H2><P>- Grow RBS W6 in 10mL of LB with antibiotic
</P><H1><SPAN class="mw-headline" id="September.2C_14th">September, 14th</SPAN></H1><H2><SPAN class="mw-headline" id="RBS_S6_and_RBS_W6">RBS S6 and RBS W6</SPAN></H2><P>- Put the 10mL of LB with OBS W6 in 200mL of LB with antibiotic
</P><P>- Aliquot the 200mL of LB with RBS S6, spin, throw out the supernatant and freeze the pellet
</P><H1><SPAN class="mw-headline" id="September.2C_15th">September, 15th</SPAN></H1><H2><SPAN class="mw-headline" id="Check_big_stock_of_RBS_S_and_W">Check big stock of RBS S and W</SPAN></H2><UL><LI> We want to check our big stock of RBS S and W, with all digest enzymes
</LI></UL><P>- Plasmid miniprep RBS S6 and RBS W6 (from 200mL flask)
</P><P>- Nanodrop
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> product </TH><TH> concentration (ng/μL) </TH><TH> 260/280
</TH></TR><TR><TD> RBS W2 </TD><TD> 90.3 </TD><TD> 1.70
</TD></TR><TR><TD> RBS W4 </TD><TD> 124.0 </TD><TD> 1.80
</TD></TR></TBODY></TABLE><P>- Single digest with EcoRI, XbaI, SpeI and PstI
</P><P>- Gel 
</P><UL><LI>Lane1 : HyperladderI
</LI><LI>Lane2 : RBS S6 cut with EcoRI
</LI><LI>Lane3 : RBS S6 cut with SpeI
</LI><LI>Lane4 : RBS S6 cut with PstI
</LI><LI>Lane5 : RBS S6 cut with XbaI
</LI><LI>Lane6 : RBS S6 uncut
</LI><LI>Lane7 : RBS W6 cut with EcoRI
</LI><LI>Lane8 : RBS W6 cut with SpeI
</LI><LI>Lane9 : RBS W6 cut with PstI
</LI><LI>Lane10 : RBS W6 cut with XbaI
</LI><LI>Lane11 : RBS W6 uncut
</LI><LI>Lane12 : supercoiled ladder
</LI></UL><UL><LI>Result
</LI></UL><P>- Even no band for the uncut plasmid. There is a problem with plasmid miniprep, we will try again with less (more diluted) cells for plasmid miniprep.
</P><P>- New plasmid miniprep (the pellets from 20mL of LB is diluted into 1.2ml instead of 0.6)
</P><P>- New single digest
</P><P>- New gel (same lanes than before)
</P><UL><LI>Result
</LI></UL><P>- One band for everything, but the same size than the uncut plasmid. Moreover, this uncut plasmid should be 3000bp, and here, it is more. So, we have a problem in the growth of cells, we will try again and check the 10ml tube which is used to inoculate the big flask and also the big flask.
</P><DIV style="color: white; font: 2px;">x</DIV></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Cambridge/Signalling/Lab_Work">http://2008.igem.org/Team:Cambridge/Signalling/Lab_Work</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Signalling/Lab_Work" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Signalling/Lab_Work" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Signalling/Lab_Work&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Signalling/Lab_Work&amp;oldid=104578" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>