"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Signalling/Lab Work
From 2008.igem.org
Home   Voltage   Signalling   Bacillus   Modelling   Protocols   People  
Signalling Overview   Vectors   Constructs   Primers   Experiments   Lab Notebook  
x
Contents
1 July, 22nd
1.1 Dry work
2 July, 24th
2.1 Single Colony
3 July, 25th
4 Results from Yesterday
5 July, 28th
5.1 Results from previous days
5.2 Wet Work
6 July, 29th
6.1 Result from yesterday
7 July, 30th
7.1 Wet Work
8 July, 31st
8.1 Wet work
9 August, 1st
9.1 Wet Work
10 August, 5th
10.1 Testing AHL degradation by aiiA
11 August, 6th
11.1 PCR parts of Agr operon
12 August, 9th
12.1 Plans for Monday
12.1.1 Lab Work
12.1.2 Book/Computer Work
12.2 Longer Term Plans
13 August, 11th
13.1 PCR Bacillus RBS
14 August, 13th
14.1 PCR Agr A, B, C, D
15 August, 15th
15.1 PCRing out promoters
15.2 Digest and Ligate Agr A and C to pSB4C5
16 August, 16th
16.1 Agr A and C ligation to pSB4C5
17 August, 18th
17.1 Check promoters
17.2 Transformation of AgrA and AgrC
18 August, 19th
18.1 Results of AgrA and C transformation
18.2 Lux parts
19 August, 20th
19.1 Check Lux components
19.2 Ligation
20 August, 21st
20.1 Transformation of ligation products
20.2 J04630
20.3 Plasmid stocks
21 August, 22nd
21.1 New PCR of promoters
22 August, 23rd
22.1 Check promoters (after PCR from bacillus vectors)
23 August, 27th
23.1 New transformation of ligation products
24 August, 28th
24.1 Results from transformation of ligation products
24.2 Transformation of ligation products (new)
25 August, 29th
25.1 Results of transformation with our ligation products
25.2 Single colony PCR to check our transformation
25.3 Plate biobricks from MIT
26 September, 2nd
26.1 Check transformation in E.coli of our ligated products
27 September, 3rd
27.1 Checking the insert of promoters
27.2 Make some stock of our new biobricks
27.3 Transformation of new "biobricks"
28 September, 4th
28.1 Results of new "biobricks"transformation
28.2 New attempt of new "biobricks" transformation
29 September, 7th
29.1 New PCR
30 September, 8th
30.1 Check product of PCR from yesterday
30.2 Check transformation from 03/09
31 September, 10th
31.1 PCR of GFP+RBS and Promoter+RBS
31.2 RBS screening
32 September, 11th
32.1 RBS screening
32.2 RBS screning but single digest
32.3 Backbone for ligation
33 September, 12th
33.1 RBS screening (single digest)
33.2 Double check of RBS S6 and stock
33.3 Check more RBS W
33.4 Check PCR products
34 September, 13th
34.1 Grow RBS S6
34.2 RBS W screening
34.3 Double check of RBS W6 and stock
35 September, 14th
35.1 RBS S6 and RBS W6
36 September, 15th
36.1 Check big stock of RBS S and W
July, 22nd
Dry work
found assay to test for AIP
found shuttle vector for constitutive expression of GFP
found protocol for Bacillus subtilis transformation
July, 24th
Single Colony
Single colony of I746001 and I746101 tranfered into 10mL LB [with 100μg/mL conc. of Amp.]
Single colony of I746001 and I746101 streaked from Amp 100 plate to Kan 25 palte (to test for Kan resistance as well)
Incubate overnight at 37°C
July, 25th
Results from Yesterday
Kan resistance of biobricks
Biobrick
Antibiotic
Contration (μg/mL)
Observations
Conclusion
I746001
Kan
25
Many colonies
Kan Resistance OK
I746101
Kan
25
Many colonies
Kan Resistance OK
Control PUC19
Kan
25
No colony
No Kan Resistance
Results
Nothing! 1A1 cells were kept in the fridge! B.S. can not be kept in the fridge, low temperatures kill them!
Test if biobrick of last year are really the one we want
Yesterday : Single colony of I746001 and I746101 transferred in 10mL of LB + Amp (final concentration 100μg/mL)
Take 8mL of I746001 (AIP sender) and of I746101
Centrifuge them
Saved 2ml of each in tubes to regrow and add LB
Miniprep I746001 and I746101 pellets (plasmid mniprep kit) : same protocol but 60μL of Zyppy Glution Buffer in step 8
Restriction Digest : use EcoI and Spel restriction enzymes
Add sequentially : 15μL of SDW + 2μL of 10X Fast Digest Buffer + 2μL of DNA + 1μL of EcoI + 1μL of Spel
Microfuge tubes
Incubate for 10min at 37°C
Heat inactivation of enzymes at 80°C for 5min
GEL - 20μLof samples + 4μL of dye
Grow 1A1
10mL of LB + 15μL of 1A1
Grow at 37°C
July, 28th
Results from previous days
Result of the gel
Lane3 : hyperladder1
Lane4 : I746101
Lane5 : I746001
We wanted to check the size of our biobricks.  For I746101, we have a band of about 3000kb, which is the size of the vectorm and a band of about 2000kb. The size of the biobrick is 2057kp, so it should be ok.
For I746001, we hqve the same band of 3000kb (the vector) and a band of about 1000kb. The size of the biobrick is 896kb.
So, the size of our biobricks are ok.
Wet Work
- Digest
For biobricks I746001 and I746101, use digest enzymes EcoRI and SpeI, and 2μL of DNA
Add 15μL of SDW, 2μL of 10X Fast Digest Buffer, 2μL of DNA, 1μL of EcoRI and 1μL of SpeI
For ppL82 (from plate and tube), use digest enzymes BamHI and PstI and add 15μL of DNA
Add 2μL of SDW, 2μL of 10X Fast Digest Buffer, 15μL of DNA, 1μL of BamHI and 1μL of PstI
Microfuge tubes
Incubate 10 min at 37°C
Heat shock for 5min at 80°C
- Gel : add 4μL of dye and 21μL of samples
New plates
-Plate I746001 and I746101 on Kn plates from Amp plates from 24/07/2008.
July, 29th
Result from yesterday
Result from the gel (29/07/2008)
Lane8 : ppL82 (plate)
Lane9 : ppL82 (tube)
Lane 10 : I746001
Lane 11 : I746101
Lane 12 : hyperladderI
The ladder seems to be wrong. So it was really difficult to ckeck the size of our plasmid ppL82. To make, that, we assume that the size of our biobricks was ok, and we estimate the size of our plasmid. The plasmid ppL82 seems to have the right size, but we will have to check again to be able to use a ladder.
July, 30th
Wet Work
Checking our big stock of biobricks and PNZ plasmid
We did big culture of biobricks and PNZ8901 plasmid to be able to make stocks. We want to check them before making stocks.
- Plamsid miniprep (for I746001, I746101 and PNZ8901 from big flasks
- Measure DNA concentration in our samples to decide the volume of DNA we have to add in our preparation
260/280
DNA concentration (ng/nl)
I746001
1.79
50.5
I746101
1.87
54.3
PNZ8901
1.84
87
- Digest
For I746001 and I746101
For PNZ8901 plasmid
14μL of SDW
14μL of SDW
2μL of 10X Fast Digest Buffer
2μL of Buffer 3 (Biolabs)
2μL of DNA of biobricks
2μL of DNA of PNZ8901
1μL of EcoRI
1μL of PstI
1μL of SpeI
1μL of SalI
- Gel
Results from this gel
Lane 9 : I746001
Lane 10 : I746101
Lane 11 : PNZ8901
Lane 12 : HyperladderI
Everthing is really too big! There is a problem, either dimerization, either contamination, either a problem in our work. So we are going to run a new gel.
New gel to check
- Miniprep plasmid from growth bottles (I746001, I746101 and PNZ8901); from plates (I746001, I746101 and PNZ8901).
- Do single digest for PNZ8901 (one with PstI, one with SalI)
- Run a gel with : PNZ8901 from friday, PNZ8901 digest with PstI, PNZ8901 digest with SalI, 3 samples from growth bottles, 3 samples from plates (to check the size of the uncut vectors)
Result from this second gel
The ladders are really bad! But the size of our biobricks and plasmid are too big! So we can not trust these big cultures. They is a problem.  With tests from the first week, we know that biobricks are right, so we are going to grow new cultures from these first cultures, to make stocks.
Concerning the vectors, they are from last year, so we are not sure of what they are. Since we ordered new well defined vectors (we should receive them on friday), we will use them in the next steps to be sure of our work.
July, 31st
Wet work
We want to make stocks of biobricks I746001 and I746101, since our big cultures have a problem,  we want to make some stocks from our first plates (which are good).
We used original Amp plates from 23/07/08 of I746001 and I7461001. We took 5 single colonies and spotted on new Amp plates and inoculated in 10mL of LB with Amp.
August, 1st
Wet Work
Check biobricks I746001 and I746101
- Single colony PCR
We had 5 single colonies for each biobrick fro; yestrday. We are going to PCR each single colony.
Add 1μL of cells, 7 μL of SDW, 10μL of Phusion Flash master mix, 1μL of VF2 primer and 1μL of VR primer.
-Gel
Add 5μL of PCR products, 14μL of SDW and 1μL of dye.
-Result
Lane1 : HyperladderI
Lane2 : I746001 colony 1
Lane3 : I746001 colony 2
Lane4 : I746001 colony 3
Lane5 : I746001 colony 4
Lane6 : I746101 colony 1
Lane7 : I746101 colony 2
Lane8 : I746101 colony 3
Lane9 : I746101 colony 4
The result is good. Each colony of I746001 is about 1000kb (all the same size), for I746101, the size is good (about 2000kb), except for one single colony for which there is nothing. It could be a problem during the manipulation.
- Check colony 1
We have colony1 for each biobrick. We want to check them.
- Plasmid miniprep
- Digest with EcoRI and SpeI
- Run a gel (for cut plasmid, add 19μL of DNA and 1μL of dye; for uncut plasmids, add 2μL of DNA, 17μL of SDW and 1μL of dye; for PCR product, add 5μL of DNA, 14μL of SDW and 1μL of dye)
Lane1 : Hyperladder1
Lane2 : I746001 double digest (miniprep product)
Lane3 : I746101 double digest (miniprep product)
Lane4 : I746001 uncut (miniprep product)
Lane5 : I746101 uncut (miniprep product)
Lane6 : I746001 colony5 (PCR product)
Lane7 : I746101 colony5 (PCR product)
- Result : everything is ok except for biobrick I746001, we have only one band
- New double digest for biobrick I746001 (lane9), new gel : ok!
We are sure to have some good biobricks for agr (I746001 and I746101)
August, 5th
Testing AHL degradation by aiiA
The product of the aiiA gene AHL-lactonase degrades AHL in many strains of Bacillus, however, it is unclear to what extent (if any) B. subtilis strain 168 degrades AHL. If 168 is good at degrading AHL this will be a problem when our signaling system is eventually moved into strain 168.
We are testing B. subtilis strain 168’s ability to degrade AHL by exposing synthetic AHL to the strain and then testing its ability to be recognized by the receiver.
Materials
Bacillus subtilis - protease deficient (NPR and APR) derived from strain 168 - grown up in an over night culture of LB.
Synthetic AHL – Sigma K3007 – AHL powder was dissolved in EA buffer and diluted to 10uM concentration
E. coli with T9002 AHL receiver grown up in overnight culture of LB
All plates contain soft agar with 200uL of E. coli AHL receiver mixed-in. Plates 1,2 and 4 contain Bacillus in fresh LB medium, while plate 5 contains Bacillus in it’s overnight LB. Plate 5 was used to test if an excreted product of Bacillus is responsible for the degradation of AHL. The product of the aiiA gene in Bacillus subtilis is most likely not exported.
Plate 1: 10uL Bacillus + 10uL AHL
Plate 2: 10uL Bacillus + 10uL EA buffer (negative control + does EA buffer kill bacillus?)
Plate 3: 10uL LB + 10uL AHL (positive control)
Plate 4: 10uL Bacillus + 10uL AHL incubated together for 1.5 hours prior to plating
Plate 5: 10uL Bacillus in overnight LB + 10uL AHL
Results
plates visualized 3 hours after AHL/Bacillus inoculation.....
August, 6th
PCR parts of Agr operon
We received primers for Agr A,B,C, and D today - http://openwetware.org/wiki/IGEM:Cambridge/2008/Turing_Pattern_Formation/Primers
We must PCR each individual part out of the previous BioBricks (I746001 &I746101) in order to put our bacillus RBS on them.
I was concerned that the region of most homology would be the BioBrick prefix and suffix itself, so I first digested the DNA with Xbal and SpeI.
For some reason the digestion didn't work that well, so I've decided to go ahead and use the uncut BioBrick DNA as template for my reactions.
August, 9th
Plans for Monday
Lab Work
Be sure that the Spc50 plates are working, as the control for Spc50 grew and shouldn't have. We already know that things can die at Spc100 (the double crossover test from earlier this week) so it is unlikely that the stock is bad. We should test Spc50 again with regular bacillus and with E. coli at 50-100 ug/ml and hope that they die.
Finish all the agr PCRs, but especially the RBS.
Confirm that amyE insertion works, using the amylase/ery tests.
Confirm that with ECE153 inserted,  we get detectable fluoresence in a microscope.
Confirm that ECE166 transformation works, using plasmid miniprepping.
Find the qiagen miniprep kit or adapt the protocol.
Confirm that when ECE166 is transformed, we get detectable fluoresence in a microscope.
Book/Computer Work
Find good promoters in vectors and order PCR primers to get them out
Find out how to assay for/make AIP
Find/Contact Jim Ajioka
Find out how to do a c-terminal GFP fusion for membrane proteins agrB and agrC
Can we fuse GFP and still expect them to localize to the membrane?
AgrB transmembrane topology ( http://www.jbc.org/cgi/content/full/277/38/34736#F1 ).  The signal peptide is not on the N terminus, and both the N and C terminus are intracellular.
AgrC transmembrane topology ( http://openurl.ingenta.com/content?genre=article&issn=0950-382X&volume=28&issue=3&spage=655&epage=662 (see figure 2 B)), the N terminus is probably extracellular.  The C terminus is probably intracellular.  The article doesn't say where the signal peptide is.
Should we do a antibody tag instead?
Longer Term Plans
Ligate the RBS into a BB vector so we can build a RBS/GFP construct.
Put the RBS/GFP construct into ECE166 and ECE 153.
Compare the 'out of the box' ECE153 with ECE153 with RBS/GFP biobrick inserts
Compare the 'out of the box' ECE166 with ECE166 with RBS/GFP biobrick inserts
August, 11th
PCR Bacillus RBS
We created 2 different Bacillus RBS sites simply by primer annealing and extension. Each product includes an 8bp RBS plus the BioBrick prefix and suffix for a total of 54bp.  The products were visualized on a 3.5% Agarose gel with bioline's hyperladder V.
RBSs is the consensus RBS sequence for Bacillus subtilis, and thus we believe it to be a very strong binding site -   AAAGGAGG
RBSw has a 2bp modification from RBSs, which we believe will weaken it -  AGAGGTGG
The amplification of RBSs yielded only one band on the gel. The product was purified using microclean and contained 14.8ng/uL after clean-up.
RBSw, however produced 2 bands on the gel. The correct size was gel-extracted and purified. After purification the yield was 12.3 ng/uL.
August, 13th
PCR Agr A, B, C, D
Doing a PCR directly from the old Biobricks has worked for all parts of the operon. No prior digestion seems to be needed.
Part
260/280
μg/mL
A
1.78
92.5
B
1.87
96.5
C
1.9
127.7
D
1.82
26.3
August, 15th
PCRing out promoters
- 4 different promoters : Pxyl, Pspc, Ppac, Pupp
- Make master mix enough for 4RXNs
Digest and Ligate Agr A and C to pSB4C5
Previous PCR products of Agr A and C were digested using Fermentas Fast Digest - EcoRI and PstI according to their protocol except a longer time was given to digest.
3uL of AgrA was used and 4.4 uL of AgrC was used and incubated for 40minutes.
10uL totaling to 1ug of pSB4C5 plasmid was also digested and incubated for 10 minutes.
The digestion was visualized on a 0.8% Agarose gel and produced expected sizes:
Plasmid: approx 3bk
Agr A: approx 700bp
Agr C: approx 1300bp
August, 16th
Agr A and C ligation to pSB4C5
Ligation was performed using Fermentas Rapid Ligation kit according to their protocol.
After ligation, samples were visualized on a gel, but little product of the correct size was seen.
For agrA we should have gotten a band of size 3700 (3000plasmid + 700 insert)
For agrC we should have gotten a band of size 4300 (3000plasmid + 1300insert)
The bands of appropriate sizes were extracted for transformation.
August, 18th
Check promoters
On Friday, PCR out promoters, we want to check them.
- Run a gel
- Results : good size of bands!!!
Transformation of AgrA and AgrC
AgrA and C gell-extracted ligation was transformed into Top 10 cells using standard protocol. Puc9 was used as a positive control.
August, 19th
Results of AgrA and C transformation
Transformation has failed. No colonies were visible for the plates of Agr A or C. The Puc9 positive control grew.  We believe that the problem was in the gel-extraction between ligation and transformation. Most of our plasmid was probably lost in this step. Next time we will directly use the results from the ligation reaction to transform. Although many bands were seen on the previous gel of the ligation reaction, we will check our transformation growth by single colony PCR to confirm transformation of the plasmid with our correct insert.
Lux parts
To make the Lux Receiver, we need 4 different parts ;
R0040, TetR repressible promoter
SO168, luxR + double terminator
R0062, promoter activated by luxR
JO4630 (GFP + double terminator)
All these parts have been transformed into E.coli. We want to test them. R004, R0062 and JO4630 have already been tested, it should be fine.  We received from the MIT R0040, R0062 and S068 already transformed into E.coli, so we want to check these stocks (which are certainly fine) and use them. For JO4630, we want to double check our transformation.
- Plate on antibiotic plates and do LB stocks of single colony from the MIT stock (R0040, R0062 and S0168).
- Put on Kan plates 4 different colonies from J04630 (transformation Amp plate) and also incubate these colonies into LB
August, 20th
Check Lux components
- Single colony PCR for :
R0040 (MIT stuff)
R0062 (MIT stuff)
4 different colonies of S0168 (from a transformation plate from 12/08)
4 different colonies of J04630 (from a transformation plate from 12/08)
- Protocol : add 1μL of cells (diluted in water), 10μL of Master Mis, 7μL of SDW, 1μL of VF primer and 1μL of VR primer
- Gel PCR products
Gel 1
Lane2 : Hyperladder1
Lane3 : JO4630, colony 1
Lane4 : JO4630, colony 2
Lane5 : JO4630, colony 3
Lane6 : JO4630, colony 4
Lane7 : HyperladderI
Gel 2
Lane2 : HyperladderI
Lane3 : ECE190 double digest
Lane4 : S0168, colony 1
Lane5 : S0168, colony 2
Lane6 : S0168, colony 3
Lane7 : S0168, colony 4
Lane8 : HyperladderI
Lane9 : R0040
Lane10 : R0062
Lane11 : Ladder 100bp
300px
- Results
R0040 and R0062 : one big band of about 300b (expected size 293), OK!
S0168 : one band of about 400b for the 4 different colonies (expected size 1234!), bad! This plate does not contain S0168
J04630 (colonies 2 and 4) : one band of about 1100b (expected size 1173), OK!
J04630 (colony 1) : one good band plus another band...
J04630 (colony 3) : one band of about 600b, bad!
Ligation
- Materials :
AgrA
AgrB
AgrC
AgrD
Pupp
Pspac
Ppac
Pxyl
RBS S
RBS W
psB4C5
- Double digest of PCR products
- Run vector, AgrA and AgrD on a gel
- DNA clean and concentrator for AgrA, B,C and D, promoters
- Microclean for both RBS
- Nanodrop
260/280
ng/μL
AgrA
1.66
16.4
AgrB
1.91
23.5
AgrC
1.99
35.9
AgrD
2.13
4.9
Pxyl
1.54
5.6
Ppac
1.49
4.6
Pspc
1.62
9.6
Pupp
1.88
8.5
RBS S
2.44
29
RBS W
1.44
10.7
- Extract plasmid annd Agr from gel and clean
- Ligation
August, 21st
Transformation of ligation products
- Spin chemically competent TOP10, add 100μL of CaCl2 solution
- Add 5μL of DNA (ligation products), and 1μL of PUC9
- Continue the protocol of transformation
J04630
- Plate good colonies from yesterday on Kan25 and put in LB (for plasmid stock for tomorrow)
Plasmid stocks
- Plasmid miniprep R0040 and R0062
August, 22nd
New PCR of promoters
August, 23rd
Check promoters (after PCR from bacillus vectors)
- Gel
Lane1 : Hyperladder 5
Lane2 : Pxyl
Lane3 : Pspac
lane4 : Ppac
Lane5 : Pupp
Size is ok.
August, 27th
New transformation of ligation products
Products : agrD, Pupp, Pspac, RBS W, RBS S (from ligation with biolabs kit)
- Competent top 10 from the freezer
- Add 5μL of ligation products, and 1.2μL of PUC9
August, 28th
Results from transformation of ligation products
- nothing on plate, even on control plate
Reasons ?
- cells non competent anymore (try with fresh competent cells)
- not enough DNA (try with 10μL of DNA)
- very short ligation products
Transformation of ligation products (new)
Products to ligate : Pupp, Pspac, agrD, RBS S and RBS W
- new fresh competent TOP 10
- 5μL of DNA (1.5μL of PUC9)
- 2h30 in the incubator
August, 29th
Results of transformation with our ligation products
Everything grew! Better efficiency with electrop. than with chemical protocol
Single colony PCR to check our transformation
Transformed products
number of picked colonies from chemical transformation
number of picked colonies from electrop. transformation (neat)
number of picked colonies from electrop. transformation (1:10)
Pupp
2
2
0
Pspac
2
2
0
RBS S
3
0
2
RBS W
3
0
2
agrD
3
2
0
- Single colony PCR : 13μL of SDW+cells, 5μL of Master Mix, 1μL of VF and 1μL of VR (and plate each single colony)
- Load a gel (1.3% agarose) : 5μL of PCR products + 1μL of dye (only 1μL of 100b ladder)
In the death plasmid, the VF-VR size is about 280b.
Transformed products
size of the product (with cutting sites)
expected size after PCR (about)
Pupp
255
480
Pspac
125
350
RBS S
56
280
RBS W
56
280
agrD
200
430
Result : nothing, even no ladder, problem with the gel!
- run again on a e-gel
Lane1 :ladder 100bp
Lane2 : Pupp colony 1
Lane3 : Pupp colony 3
Lane4 : Pspac colony 1
Lane5 : Pspac colony 3
Lane6 : RBS S colony 1
Lane7 : RBS S colony 4
Lane8 : RBS W colony 1
Lane9 : RBS W colony 4
Lane10 : agrD colony 1
Lane11 : agrD colony 4
Lane12 : HyperladderI
Result : nothing, just the primers! Problem with our PCR
Plate biobricks from MIT
E0840
B0014
I712007
C0012
B0015
C0061
R0063
September, 2nd
Check transformation in E.coli of our ligated products
- Single colony PCR with VF and VR : add 10μL of SDW, 5μL of MM, 1μL of VF, 1μL of VR and 3μL of cells
inserted products
number of colonies to check (chemical transf.)
number of colonies to check (electrop. transf.)
RBS S
3
2
RBS W
3
2
Pspac
2
2
Pupp
2
2
agrD
3
2
- Gel (3% of agarose)
Result
gel
lane
inserted product (name of the colony)
observation
conclusion
top
3
RBS S (1)
one band (260bp)
size of the insert, RBS too small to see on a gel
top
4
RBS S (2)
one band (260bp)
size of the insert, RBS too small to see on a gel
top
5
RBS S (4)
nothing
 ?
top
6
RBS S (5)
nothing
 ?
top
7
RBS W (1)
2 band (260bp + 350bp)
size of the insert, RBS too small to see on a gel + something else?
top
8
RBS W (2)
one band (350bp)
too big
top
9
RBS W (4)
one band (260bp)
size of the insert, RBS too small to see on a gel
top
10
RBS W (5)
one band (260bp)
size of the insert, RBS too small to see on a gel
top
11
Pspac (1)
2 band (260bp + 350bp)
maybe problem with products loaded on gel (exactly the same bands than for RBS W)
top
12
Pspac (2)
one band (450bp)
size of Pupp?
top
13
Pspac (3)
one band (450bp)
size of Pupp?
bottom
3
Pspac (4)
nothing
 ?
bottom
4
Pupp (1)
one band (400bp)
size of Pspac?
bottom
5
Pupp (2)
one band (400bp)
size of Pspac?
bottom
6
Pupp (3)
one band (slightly lower than 400bp)
size of Pspac?
bottom
7
Pupp (4)
one band (400bp)
size of Pspac?
bottom
8
agrD (1)
2 bands (380 and 450bp)
big band is ok
bottom
9
agrD (2)
2 bands (380 and 450bp)
big band is ok
bottom
10
agrD (3)
no bands
 ?
bottom
11
agrD (4)
strong band (450bp)
ok
bottom
12
P2
one band (350bp)
ok
September, 3rd
Checking the insert of promoters
We ligated promoters into death vector, and transforned into Top 10. Our transformation worked, but it was hard to say with the result of yesterday if we had the good insert. Moreover, it is possible that we inverted Pupp and Pspac. So we are going to make a new single colony PCR with the primers of promoters (we used the same single colonies than yesterday).
We also want to PCR again all promoters (Pspac, Pxyl, Pupp and Ppac) from plasmids to have more.
Picture of the gel with the different promoters
promoter
colony
primers
name
Pspac
1
Pspac primers
A
Pspac
1
Pupp primers
B
Pspac
2
Pspac primers
C
Pspac
2
Pupp primers
D
Pupp
1
Pupp primers
E
Pupp
1
Pspac primers
F
Pupp
3
Pupp primers
G
Pupp
3
Pspac primers
H
- Single colony PCR : add 10μL of SDW, 5μL of MM, 1μL + 1μL of primers, 3μL of cells
- Gel
lane 3 : hyperladderI
lane 4 : Pspac (colony1) with Pspac primers
lane 5 : Pspac (colony1) with Pupp primers
lane 6 : Pspac (colony2) with Pspac primers
lane 7 : Pspac (colony2) with Pupp primers
lane 8 : Pupp (colony1) with Pupp primers
lane 9 : Pupp (colony1) with Pspac primers
lane 10 : Pupp (colony3) with Pupp primers
lane 11 : Pupp (colony3) with Pspac primers
lane 12 : HyperladderI
- Results
promoter
colony
primers
observation
Conclusion
Pspac
1
Pspac primers
nothing
it is in fact Pupp
Pspac
1
Pupp primers
band (about 280bp)
Pupp OK
Pspac
2
Pspac primers
nothing
it is in fact Pupp
Pspac
2
Pupp primers
band (about 280bp)
Pupp OK
Pupp
1
Pupp primers
nothing
it is in fact Pspac
Pupp
1
Pspac primers
band (about 180bp)
Pspac OK
Pupp
3
Pupp primers
nothing
it is in fact Pspac
Pupp
3
Pspac primers
band (about 180bp)
Pspac OK
Make some stock of our new biobricks
Pupp (colony1) into the death vector, transformed into TOP10
Pspac (colony1) into the death vector, transformed into TOP10
agrD (colony5) into the death vector, transformed into TOP10
- Grow this single colony into 10mL of LB (Cm35)
Transformation of new "biobricks"
- transform into E.coli :
agrA
agrB
agrC
Pxyl
September, 4th
Results of new "biobricks"transformation
- Nothing on our plate, except for the control plate.
New attempt of new "biobricks" transformation
- transform into E.coli these ligated products:
agrA
agrB
agrC
Pxyl
September, 7th
New PCR
- New stocks of Master Mix
- PCR agrA, agrB, Pxyl, Pspac, Ppac and Pupp
September, 8th
Check product of PCR from yesterday
Lane3 : Hyperladder I
Lane4 : agrA
Lane5 : agrB
Lane6 : Pxyl
Lane7 : Pspac
Lane8 : Ppac
Lane9 : Pupp
Lane10 Hyperladder IV
Results : everything is ok
Check transformation from 03/09
After several days of incubation, we have a few colonies on our transformed plates (agrA, B and C).
Single colony PCR
- Add 13μL of cells diluted into SDW, 5μL of MM, 1μL of VF and 1μL of VR
Gel
September, 10th
PCR of GFP+RBS and Promoter+RBS
- PCR
- Run a 1.2% agarose gel with 1μL of sample
Lane 3 : HyperladderI
Lane 4 : GFP + RBS 1A
Lane 5 : GFP + RBS1B
Lane 6 : GFP + RBS 2A
Lane 7 : GFP + RBS 2B
Lane 8 : Pupp + RBS 1
Lane 9 : Pupp + RBS 2
Lane 10: HyperladderIV
Result
RBS screening
5 colonies from chemical transformation + 6 colonies from electroporation transformation for RBS S
5 colonies from chemical transformation + 6 colonies from electroporation transformation for RBS W
- PCR : 11μL of SDW + 5μL of MM + 1+1μL of primers (RBS detect + VR) + 2μL of cells (program iGEM34)
- Gel1
Lane2 : RBSS1
Lane3 : RBS S2
Lane4 : RBS S3
Lane5 : RBS S4
Lane6 : RBS S5
Lane7 : RBS S6
Lane8 : HyperladderIV
Lane9 : RBS S7
Lane10 : RBS S8
Lane11 : RBS S9
Lane12 : RBS S10
Lane13 : RBS S6 with VF and VR primers
Lane14 : RBS S11
- Gel2
Lane2 : RBSW1
Lane3 : RBS W2
Lane4 : RBS W3
Lane5 : RBS W4
Lane6 : RBS W5
Lane7 : RBS W6
Lane8 : HyperladderIV
Lane9 : RBS W7
Lane10 : RBS W8
Lane11 : RBS W9
Lane12 : RBS W10
Lane13 : RBS W6 with VF and VR primers
Lane14 : RBS W11
Result
- Nothing, except for the PCR with VF and VR primers, either our primers are not right, either we have no insert. But the gel is not good enough to see very well, we will try to run a new gel.
September, 11th
RBS screening
- We run our PCR products from yesterday on a new 1.8% agarose gel
- Gel1
Lane2 : RBSS1
Lane3 : RBS S2
Lane4 : RBS S3
Lane5 : RBS S4
Lane6 : RBS S5
Lane7 : RBS S6
Lane8 : HyperladderIV
Lane9 : RBS S7
Lane10 : RBS S8
Lane11 : RBS S9
Lane12 : RBS S10
Lane13 : RBS S6 with VF and VR primers
Lane14 : RBS S11
- Gel2
Lane2 : RBSW1
Lane3 : RBS W2
Lane4 : RBS W3
Lane5 : RBS W4
Lane6 : RBS W5
Lane7 : RBS W6
Lane8 : HyperladderIV
Lane9 : RBS W7
Lane10 : RBS W8
Lane11 : RBS W9
Lane12 : RBS W10
Lane13 : RBS W6 with VF and VR primers
Lane14 : RBS W11
Result
- Nothing, except for the PCR with VF and VR primers, either our primers are not right, either we have no insert.
RBS screning but single digest
- Grow some colonies of Top10 transformed with RBS into 10mL of LB with antibiotic
We want to check our transformation with single digest. If we have self ligation in our transformation, we will loose the XbaI cutting site.
Backbone for ligation
- New pellets in 600μL of SDW
- Plasmid miniprep
September, 12th
RBS screening (single digest)
- Plasmid miniprep 6 colonies of RBS S and 6 colonies of RBS W (without endo wash buffer)
- Nanodrop
product
concentration (ng/μL)
260/280
quantity of DNA to add to have about 300ng (μL)
added SDW for single digest (μL)
RBS W1
57.1
1.57
6
11
RBS W3
54.1
1.53
6
11
RBS W5
15.1
1.85
17
0
RBS W 7
21.8
1.66
15
2
RBS W9
21.8
1.78
15
2
RBS W11
13.8
1.73
17
0
RBS S2
19.5
1.7
15
2
RBS S4
82.13
2.57
4
13
RBS S6
44.7
1.64
7
10
RBS S8
104.0
1.48
3
14
RBS S10
13.1
1.82
17
0
RBS S11
20.1
1.73
15
2
- Single digest : with EcoRI and XbaI, add SDW and DNA (according to the previous table, 300mg of DNA), 2μL of fast digest buffer, and 1μL of enzyme
- Gel1
Lane2 : HyperladderI
Lane3 : RBS W1 cut with EcoRI
Lane4 : RBS W1 cut with XbaI
Lane5 : RBS W3 cut with EcoRI
Lane6 : RBS W3 cut with XbaI
Lane7 : RBS W7 cut with EcoRI
Lane8 : RBS W7 cut with XbaI
Lane9 : RBS W9 cut with EcoRI
Lane10 : RBS W9 cut with XbaI
Lane11 : RBS W9 uncut
Lane12 : supercoiled ladder
- Gel2
Lane2 : HyperladderI
Lane3 : RBS S4 cut with EcoRI
Lane4 : RBS S4 cut with XbaI
Lane5 : RBS S6 cut with EcoRI
Lane6 : RBS S6 cut with XbaI
Lane7 : RBS S8 cut with EcoRI
Lane8 : RBS S8 cut with XbaI
Lane9 : RBS S11 cut with EcoRI
Lane10 : RBS S11 cut with XbaI
Lane11 : RBS S4 uncut
Lane12 : supercoiled ladder
Result
The size of our plasmid is about 3000bp (ok on the gel)
RBS W
- W1 with EcoRI : cut
- W1 with XbaI : uncut, self ligation
- W3 with EcoRI : cut
- W3 with XbaI : uncut, self ligation
- W7 with EcoRI : impossible to see
- W7 with XbaI : uncut, self ligation
- W9 with EcoRI : cut
- W9 with XbaI : uncut, self ligation
RBS S
- S4 with EcoRI : cut
- S4 with XbaI : uncut, self ligation
- S6 with EcoRI : cut
- S6 with XbaI : 2 band, this plasmid is partially cut!!! Transformation with RBS S
- S8 with EcoRI : cut
- S8 with XbaI : uncut, self ligation
- S11 with EcoRI : cut
- S11 with XbaI : uncut, self ligation
Double check of RBS S6 and stock
- Grow RBS S6 in LB with Cm35
Check more RBS W
- Grow RBS W2, 4, 6, 8, 10 in 10mL of LB with antibiotic
Check PCR products
- Gel (2μL of each product)
Lane1 : λ ladder
Lane2 : agrA
Lane3 : agrB
Lane4 : agrC
Lane5 : rep
Lane6 : pSB4C5 1
Lane7 : pSB4C5 2
Lane8 : Pxyl
Lane9 : Ppac
Lane10 : RBS S
Lane11 : RBS W
Lane12 : HyperladderI
Results
- agr B : 2 bands?
- agrC : ok
- rep : small band but good size
- backbones : ok
- promoters : ok
- RBS : nothing
September, 13th
Grow RBS S6
- Put the 10mL of LB with RBS S6 from yesterday into 200mL of LB with antibiotic
- Incubate at 37°C with shaking
RBS W screening
- Plasmid miniprep of RBS W2, 4, 6, 8, 10 9witouh endo wash buffer)
- Nanodrop
product
concentration (ng/μL)
260/280
quantity of DNA to add to have about 300ng (μL)
added SDW for single digest (μL)
RBS W2
42.4
1.62
8
9
RBS W4
35.1
1.61
10
7
RBS W6
41.0
1.67
8
9
RBS W8
72.8
1.61
5
12
RBS W10
51.0
1.60
6
11
- Single digest with EcoRI and XbaI
- Gel1
Lane2 : HyperladderI
Lane3 : RBS W2 cut with EcoRI
Lane4 : RBS W2 cut with XbaI
Lane5 : RBS W4 cut with EcoRI
Lane6 : RBS W4 cut with XbaI
Lane7 : RBS W5 cut with EcoRI
Lane8 : RBS W5 cut with XbaI
Lane9 : RBS W6 cut with EcoRI
Lane10 : RBS W6 cut with XbaI
Lane11 : RBS W8 uncut
Lane12 : supercoiled ladder
- Gel2
Lane2 : HyperladderI
Lane3 : RBS W8 cut with EcoRI
Lane4 : RBS W8 cut with XbaI
Lane5 : RBS W10 cut with EcoRI
Lane6 : RBS W10 cut with XbaI
Lane7 : RBS W11 cut with EcoRI
Lane8 : RBS W11 cut with XbaI
Lane9 : RBS W8 uncut
Lane10 : supercoiled ladder
Result
- W2 with EcoRI : cut
- W2 with XbaI : uncut, self ligation
- W4 with EcoRI : cut
- W4 with XbaI : uncut, self ligation
- W5 with EcoRI : cut
- W5 with XbaI : uncut, self ligation
- W6 with EcoRI : cut
- W6 with XbaI : cut (2 bands), transformation ok for RBS W6
- W8 with EcoRI : impossible to see
- W8 with XbaI : impossible to see
- W10 with EcoRI : cut
- W10 with XbaI : maybe cut, it seems to be 2 bands, but quite difficult to see
- W11 with EcoRI : cut
- W11 with XbaI : uncut, self ligation
Double check of RBS W6 and stock
- Grow RBS W6 in 10mL of LB with antibiotic
September, 14th
RBS S6 and RBS W6
- Put the 10mL of LB with OBS W6 in 200mL of LB with antibiotic
- Aliquot the 200mL of LB with RBS S6, spin, throw out the supernatant and freeze the pellet
September, 15th
Check big stock of RBS S and W
We want to check our big stock of RBS S and W, with all digest enzymes
- Plasmid miniprep RBS S6 and RBS W6 (from 200mL flask)
- Nanodrop
product
concentration (ng/μL)
260/280
RBS W2
90.3
1.70
RBS W4
124.0
1.80
- Single digest with EcoRI, XbaI, SpeI and PstI
- Gel
Lane1 : HyperladderI
Lane2 : RBS S6 cut with EcoRI
Lane3 : RBS S6 cut with SpeI
Lane4 : RBS S6 cut with PstI
Lane5 : RBS S6 cut with XbaI
Lane6 : RBS S6 uncut
Lane7 : RBS W6 cut with EcoRI
Lane8 : RBS W6 cut with SpeI
Lane9 : RBS W6 cut with PstI
Lane10 : RBS W6 cut with XbaI
Lane11 : RBS W6 uncut
Lane12 : supercoiled ladder
Result
- Even no band for the uncut plasmid. There is a problem with plasmid miniprep, we will try again with less (more diluted) cells for plasmid miniprep.
- New plasmid miniprep (the pellets from 20mL of LB is diluted into 1.2ml instead of 0.6)
- New single digest
- New gel (same lanes than before)
Result
- One band for everything, but the same size than the uncut plasmid. Moreover, this uncut plasmid should be 3000bp, and here, it is more. So, we have a problem in the growth of cells, we will try again and check the 10ml tube which is used to inoculate the big flask and also the big flask.
x
Retrieved from " http://2008.igem.org/Team:Cambridge/Signalling/Lab_Work "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
