<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Bacillus_Lab_Work"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Cambridge/Bacillus/Lab_Work">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Bacillus/Lab_Work&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Bacillus/Lab_Work&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Bacillus/Lab_Work&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Bacillus/Lab_Work" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Bacillus/Lab Work</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="bodytable"><DIV class="menubar"><CENTER><A href="/Team:Cambridge" title="Team:Cambridge"><FONT face="verdana" style="color:#ffffff"><B>Home</B></FONT></A> <A href="/Team:Cambridge/Voltage" title="Team:Cambridge/Voltage"><FONT face="verdana" style="color:#b3c6fe"><B>Voltage</B></FONT></A> <A href="/Team:Cambridge/Signalling" title="Team:Cambridge/Signalling"><FONT face="verdana" style="color:#b3c6fe"><B>Signalling</B></FONT></A> <A href="/Team:Cambridge/Bacillus" title="Team:Cambridge/Bacillus"><FONT face="verdana" style="color:#b3c6fe"><B>Bacillus</B></FONT></A> <A href="/Team:Cambridge/Modelling" title="Team:Cambridge/Modelling"><FONT face="verdana" style="color:#b3c6fe"><B>Modelling</B></FONT></A> <A href="/Team:Cambridge/Protocols" title="Team:Cambridge/Protocols"><FONT face="verdana" style="color:#ffffff"><B>Protocols</B></FONT></A> <A href="/Team:Cambridge/People" title="Team:Cambridge/People"><FONT face="verdana" style="color:#ffffff"><B>People</B></FONT></A> </CENTER></DIV><DIV class="submenu"><CENTER><A href="/Team:Cambridge/Bacillus" title="Team:Cambridge/Bacillus"><FONT face="verdana" style="color:#444444"><B>Bacillus Overview</B></FONT></A> <A href="/Team:Cambridge/Bacillus_subtilis_transformation" title="Team:Cambridge/Bacillus subtilis transformation"><FONT face="verdana" style="color:#444444"><B>Transformation</B></FONT></A> <A href="/Team:Cambridge/Signalling/Constructs" title="Team:Cambridge/Signalling/Constructs"><FONT face="verdana" style="color:#444444"><B>RBS &amp; Promoters</B></FONT></A> <A href="/Team:Cambridge/Bacillus/Primers" title="Team:Cambridge/Bacillus/Primers"><FONT face="verdana" style="color:#444444"><B>Primers</B></FONT></A> <A href="/Team:Cambridge/Bacillus_subtilis_transformation" title="Team:Cambridge/Bacillus subtilis transformation"><FONT face="verdana" style="color:#444444"><B>InFusion</B></FONT></A> <A href="/Team:Cambridge/Improved_GFP" title="Team:Cambridge/Improved GFP"><FONT face="verdana" style="color:#444444"><B>Superfolder GFP</B></FONT></A> <STRONG class="selflink"><FONT face="verdana" style="color:#444444"><B>Lab Notebook</B></FONT></STRONG> </CENTER></DIV><TABLE style="background:#444444; padding:15px;"><TABLE align="left" width="border=&quot;0&quot;" style="background:#444444; padding:5px;"><TBODY><TR><TD style="width:100%; height: 400; padding-left: 15px;"><DIV class="contentf"><DIV style="height: 400; background: white; line-height:170% padding: 5px;"><DIV style="color: white; font: 2px;">x</DIV></DIV></DIV></TD></TR><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#July_22nd"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">July 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Checking_antibiotic_resistance"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Checking antibiotic resistance</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#July_23rd"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">July 23rd</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#July_24th"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">July 24th</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Antibiotic_test_for_Bacillus_resistance"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Antibiotic test for Bacillus resistance</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Salts_for_making_Bacillus_Competent"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Salts for making Bacillus Competent</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#July_25th"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">July 25th</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Results_from_Yesterday"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Results from Yesterday</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Wet_Work"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Wet Work</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#July_28th"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">July 28th</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Wet_Work_2"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Wet Work</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#July_29th"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">July 29th</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Wet_Work_3"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Wet Work</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Results_from_the_gel"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Results from the gel</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#July_30th"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">July 30th</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#August_1st"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">August 1st</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#August_2nd"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">August 2nd</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Yesterday.27s_Results"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Yesterday's Results</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Lab_Work"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Lab Work</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#August_4th"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">August 4th</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Wet_Work_4"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Wet Work</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#August_5th"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">August 5th</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Checking_vectors"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Checking vectors</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Transformation_of_Bacillus"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Transformation of Bacillus</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#New_stocks"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">New stocks</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#August_6th"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">August 6th</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Result_from_transformation"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Result from transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Check_Vectors"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">Check Vectors</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#August_7th"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">August 7th</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Transformation_of_B.S._IA771"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">Transformation of B.S. IA771</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Check_Vectors_2"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">Check Vectors</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Test_ECE112_transformation"><SPAN class="tocnumber">13.3</SPAN><SPAN class="toctext">Test ECE112 transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#New_stock"><SPAN class="tocnumber">13.4</SPAN><SPAN class="toctext">New stock</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#August_8th"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">August 8th</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#Result_from_yesterday_transformation"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Result from yesterday transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Result_for_test_of_ECE112_transformation"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">Result for test of ECE112 transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Control_with_erythromycin"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">Control with erythromycin</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#Amylase_production_screening"><SPAN class="tocnumber">14.4</SPAN><SPAN class="toctext">Amylase production screening</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#New_stocks_2"><SPAN class="tocnumber">14.5</SPAN><SPAN class="toctext">New stocks</SPAN></A></LI><LI class="toclevel-1 tocsection-40"><A href="#August_11th"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">August 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Erytromycin_Experiment"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">Erytromycin Experiment</SPAN></A></LI><LI class="toclevel-2 tocsection-42"><A href="#Amylase_screening_experiment"><SPAN class="tocnumber">15.2</SPAN><SPAN class="toctext">Amylase screening experiment</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#Test_our_stock_of_ECE171"><SPAN class="tocnumber">15.3</SPAN><SPAN class="toctext">Test our stock of ECE171</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#Control_of_Spc_resistance_of_Bacillus"><SPAN class="tocnumber">15.4</SPAN><SPAN class="toctext">Control of Spc resistance of Bacillus</SPAN></A></LI><LI class="toclevel-1 tocsection-45"><A href="#August_12th"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">August 12th</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#Results_from_yesterday_2"><SPAN class="tocnumber">16.1</SPAN><SPAN class="toctext">Results from yesterday</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#New_plates_and_LB_stocks"><SPAN class="tocnumber">16.2</SPAN><SPAN class="toctext">New plates and LB stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#New_test_for_amylase"><SPAN class="tocnumber">16.3</SPAN><SPAN class="toctext">New test for amylase</SPAN></A></LI><LI class="toclevel-1 tocsection-49"><A href="#August_13th"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">August 13th</SPAN></A></LI><LI class="toclevel-2 tocsection-50"><A href="#Results_for_Starch_plates"><SPAN class="tocnumber">17.1</SPAN><SPAN class="toctext">Results for Starch plates</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#Transformation_of_IA751"><SPAN class="tocnumber">17.2</SPAN><SPAN class="toctext">Transformation of IA751</SPAN></A></LI><LI class="toclevel-2 tocsection-52"><A href="#Check_our_stock_of_ECE171"><SPAN class="tocnumber">17.3</SPAN><SPAN class="toctext">Check our stock of ECE171</SPAN></A></LI><LI class="toclevel-2 tocsection-53"><A href="#Glycerol_stocks"><SPAN class="tocnumber">17.4</SPAN><SPAN class="toctext">Glycerol stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-54"><A href="#Xylose_experiment"><SPAN class="tocnumber">17.5</SPAN><SPAN class="toctext">Xylose experiment</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#Plasmid_miniprep_for_B.S."><SPAN class="tocnumber">17.6</SPAN><SPAN class="toctext">Plasmid miniprep for B.S.</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#Transformation_of_ECE188.2C_189.2C_190_in_E.coli"><SPAN class="tocnumber">17.7</SPAN><SPAN class="toctext">Transformation of ECE188, 189, 190 in E.coli</SPAN></A></LI><LI class="toclevel-2 tocsection-57"><A href="#In_Preparation_of_Beta-galactosidase_Assay_.28Promoter_Assay_PA.29"><SPAN class="tocnumber">17.8</SPAN><SPAN class="toctext">In Preparation of Beta-galactosidase Assay (Promoter Assay PA)</SPAN></A></LI><LI class="toclevel-1 tocsection-58"><A href="#August_14th"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">August 14th</SPAN></A></LI><LI class="toclevel-2 tocsection-59"><A href="#Results_of_transformation"><SPAN class="tocnumber">18.1</SPAN><SPAN class="toctext">Results of transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-60"><A href="#BioBrick_extraction_for_testing_promoters_and_RBS_in_B.subtillis"><SPAN class="tocnumber">18.2</SPAN><SPAN class="toctext">BioBrick extraction for testing promoters and RBS in B.subtillis</SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#Transformation_of_vectors_188.2C_189.2C_190"><SPAN class="tocnumber">18.3</SPAN><SPAN class="toctext">Transformation of vectors 188, 189, 190</SPAN></A></LI><LI class="toclevel-2 tocsection-62"><A href="#Double_digest_of_ECE166_.28extracted_from_transformed_Bacillus_yesterday.29"><SPAN class="tocnumber">18.4</SPAN><SPAN class="toctext">Double digest of ECE166 (extracted from transformed Bacillus yesterday)</SPAN></A></LI><LI class="toclevel-2 tocsection-63"><A href="#Erythromycin_plate"><SPAN class="tocnumber">18.5</SPAN><SPAN class="toctext">Erythromycin plate</SPAN></A></LI><LI class="toclevel-2 tocsection-64"><A href="#New_stocks_3"><SPAN class="tocnumber">18.6</SPAN><SPAN class="toctext">New stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-65"><A href="#Check_fluorescence_with_microscope"><SPAN class="tocnumber">18.7</SPAN><SPAN class="toctext">Check fluorescence with microscope</SPAN></A></LI><LI class="toclevel-2 tocsection-66"><A href="#Plates_from_yesterday_for_Biobrick_Extraction_.28PA.29"><SPAN class="tocnumber">18.8</SPAN><SPAN class="toctext">Plates from yesterday for Biobrick Extraction (PA)</SPAN></A></LI><LI class="toclevel-3 tocsection-67"><A href="#Single_Colony_PCR_of_I13522_and_E0040"><SPAN class="tocnumber">18.8.1</SPAN><SPAN class="toctext">Single Colony PCR of I13522 and E0040</SPAN></A></LI><LI class="toclevel-1 tocsection-68"><A href="#August_15th"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">August 15th</SPAN></A></LI><LI class="toclevel-2 tocsection-69"><A href="#Transformation_ECE188.2C_189.2C_190_into_E.coli"><SPAN class="tocnumber">19.1</SPAN><SPAN class="toctext">Transformation ECE188, 189, 190 into E.coli</SPAN></A></LI><LI class="toclevel-2 tocsection-70"><A href="#Plasmid_miniprep_ECE_151_.28x2.29.2C_ECE_153.2C_ECE_166"><SPAN class="tocnumber">19.2</SPAN><SPAN class="toctext">Plasmid miniprep ECE 151 (x2), ECE 153, ECE 166</SPAN></A></LI><LI class="toclevel-2 tocsection-71"><A href="#Xylose_Induction_with_ECE_153"><SPAN class="tocnumber">19.3</SPAN><SPAN class="toctext">Xylose Induction with ECE 153</SPAN></A></LI><LI class="toclevel-2 tocsection-72"><A href="#Glycerol_Stock_Transformation"><SPAN class="tocnumber">19.4</SPAN><SPAN class="toctext">Glycerol Stock Transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-73"><A href="#Transforming_ECE_188.2C_189.2C_190_.283rd_try.29"><SPAN class="tocnumber">19.5</SPAN><SPAN class="toctext">Transforming ECE 188, 189, 190 (3rd try)</SPAN></A></LI><LI class="toclevel-2 tocsection-74"><A href="#Biobrick_E0040_and_I13522_.28PA.29"><SPAN class="tocnumber">19.6</SPAN><SPAN class="toctext">Biobrick E0040 and I13522 (PA)</SPAN></A></LI><LI class="toclevel-1 tocsection-75"><A href="#August_16th"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">August 16th</SPAN></A></LI><LI class="toclevel-2 tocsection-76"><A href="#Result_from_the_days_before"><SPAN class="tocnumber">20.1</SPAN><SPAN class="toctext">Result from the days before</SPAN></A></LI><LI class="toclevel-1 tocsection-77"><A href="#August_18th"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">August 18th</SPAN></A></LI><LI class="toclevel-2 tocsection-78"><A href="#Test_Ery_efficiency"><SPAN class="tocnumber">21.1</SPAN><SPAN class="toctext">Test Ery efficiency</SPAN></A></LI><LI class="toclevel-2 tocsection-79"><A href="#Single_colony_plates"><SPAN class="tocnumber">21.2</SPAN><SPAN class="toctext">Single colony plates</SPAN></A></LI><LI class="toclevel-2 tocsection-80"><A href="#Transformation_of_glycerol_stocks_of_B.S."><SPAN class="tocnumber">21.3</SPAN><SPAN class="toctext">Transformation of glycerol stocks of B.S.</SPAN></A></LI><LI class="toclevel-2 tocsection-81"><A href="#PA_with_I13522.2C_E0040_and_B0034_Preparation"><SPAN class="tocnumber">21.4</SPAN><SPAN class="toctext">PA with I13522, E0040 and B0034 Preparation</SPAN></A></LI><LI class="toclevel-1 tocsection-82"><A href="#August_19th"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">August 19th</SPAN></A></LI><LI class="toclevel-2 tocsection-83"><A href="#Results_from_the_transformation_with_glycerol_stocks"><SPAN class="tocnumber">22.1</SPAN><SPAN class="toctext">Results from the transformation with glycerol stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-84"><A href="#Test_our_Erythromycin_stock"><SPAN class="tocnumber">22.2</SPAN><SPAN class="toctext">Test our Erythromycin stock</SPAN></A></LI><LI class="toclevel-2 tocsection-85"><A href="#Plasmid_miniprep_of_ECE153.2C_ECE166_and_ECE171"><SPAN class="tocnumber">22.3</SPAN><SPAN class="toctext">Plasmid miniprep of ECE153, ECE166 and ECE171</SPAN></A></LI><LI class="toclevel-2 tocsection-86"><A href="#Transformation_of_IA771"><SPAN class="tocnumber">22.4</SPAN><SPAN class="toctext">Transformation of IA771</SPAN></A></LI><LI class="toclevel-2 tocsection-87"><A href="#Dealing_with_R0040.2C_R0062.2C_S0168.2C_I13522_for_PA"><SPAN class="tocnumber">22.5</SPAN><SPAN class="toctext">Dealing with R0040, R0062, S0168, I13522 for PA</SPAN></A></LI><LI class="toclevel-1 tocsection-88"><A href="#August_20th"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">August 20th</SPAN></A></LI><LI class="toclevel-2 tocsection-89"><A href="#Check_vector_ECE190"><SPAN class="tocnumber">23.1</SPAN><SPAN class="toctext">Check vector ECE190</SPAN></A></LI><LI class="toclevel-1 tocsection-90"><A href="#August_21st"><SPAN class="tocnumber">24</SPAN><SPAN class="toctext">August 21st</SPAN></A></LI><LI class="toclevel-2 tocsection-91"><A href="#Check_ECE188"><SPAN class="tocnumber">24.1</SPAN><SPAN class="toctext">Check ECE188</SPAN></A></LI><LI class="toclevel-1 tocsection-92"><A href="#August_22nd"><SPAN class="tocnumber">25</SPAN><SPAN class="toctext">August 22nd</SPAN></A></LI><LI class="toclevel-2 tocsection-93"><A href="#Stock_ECE112.2C_166.2C_153"><SPAN class="tocnumber">25.1</SPAN><SPAN class="toctext">Stock ECE112, 166, 153</SPAN></A></LI><LI class="toclevel-2 tocsection-94"><A href="#Transformation_of_Bacillus_2"><SPAN class="tocnumber">25.2</SPAN><SPAN class="toctext">Transformation of Bacillus</SPAN></A></LI><LI class="toclevel-1 tocsection-95"><A href="#August_23rd"><SPAN class="tocnumber">26</SPAN><SPAN class="toctext">August 23rd</SPAN></A></LI><LI class="toclevel-2 tocsection-96"><A href="#Results_of_transformation_from_23.2F08.2F2008"><SPAN class="tocnumber">26.1</SPAN><SPAN class="toctext">Results of transformation from 23/08/2008</SPAN></A></LI><LI class="toclevel-1 tocsection-97"><A href="#August_26th"><SPAN class="tocnumber">27</SPAN><SPAN class="toctext">August 26th</SPAN></A></LI><LI class="toclevel-2 tocsection-98"><A href="#Xylose_test"><SPAN class="tocnumber">27.1</SPAN><SPAN class="toctext">Xylose test</SPAN></A></LI><LI class="toclevel-2 tocsection-99"><A href="#Starch_plate"><SPAN class="tocnumber">27.2</SPAN><SPAN class="toctext">Starch plate</SPAN></A></LI><LI class="toclevel-2 tocsection-100"><A href="#Ery_plate"><SPAN class="tocnumber">27.3</SPAN><SPAN class="toctext">Ery plate</SPAN></A></LI><LI class="toclevel-1 tocsection-101"><A href="#August_27th"><SPAN class="tocnumber">28</SPAN><SPAN class="toctext">August 27th</SPAN></A></LI><LI class="toclevel-2 tocsection-102"><A href="#Xylose_test_2"><SPAN class="tocnumber">28.1</SPAN><SPAN class="toctext">Xylose test</SPAN></A></LI><LI class="toclevel-2 tocsection-103"><A href="#Starch_plate_2"><SPAN class="tocnumber">28.2</SPAN><SPAN class="toctext">Starch plate</SPAN></A></LI><LI class="toclevel-2 tocsection-104"><A href="#Ery_test"><SPAN class="tocnumber">28.3</SPAN><SPAN class="toctext">Ery test</SPAN></A></LI><LI class="toclevel-2 tocsection-105"><A href="#Test_glycerol_stocks_of_competent_bacillus"><SPAN class="tocnumber">28.4</SPAN><SPAN class="toctext">Test glycerol stocks of competent bacillus</SPAN></A></LI><LI class="toclevel-1 tocsection-106"><A href="#August_28th"><SPAN class="tocnumber">29</SPAN><SPAN class="toctext">August 28th</SPAN></A></LI><LI class="toclevel-2 tocsection-107"><A href="#Results_from_Bacillus_transformation_with_glycerol_stocks"><SPAN class="tocnumber">29.1</SPAN><SPAN class="toctext">Results from Bacillus transformation with glycerol stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-108"><A href="#Test_AmyE_insertion"><SPAN class="tocnumber">29.2</SPAN><SPAN class="toctext">Test AmyE insertion</SPAN></A></LI><LI class="toclevel-1 tocsection-109"><A href="#August_29th"><SPAN class="tocnumber">30</SPAN><SPAN class="toctext">August 29th</SPAN></A></LI><LI class="toclevel-2 tocsection-110"><A href="#Stock_of_ECE112"><SPAN class="tocnumber">30.1</SPAN><SPAN class="toctext">Stock of ECE112</SPAN></A></LI><LI class="toclevel-2 tocsection-111"><A href="#Transformation_of_glycerol_stock"><SPAN class="tocnumber">30.2</SPAN><SPAN class="toctext">Transformation of glycerol stock</SPAN></A></LI><LI class="toclevel-1 tocsection-112"><A href="#September_1st"><SPAN class="tocnumber">31</SPAN><SPAN class="toctext">September 1st</SPAN></A></LI><LI class="toclevel-2 tocsection-113"><A href="#Result_for_Bacillus_transformation_.2827.2F08.29"><SPAN class="tocnumber">31.1</SPAN><SPAN class="toctext">Result for Bacillus transformation (27/08)</SPAN></A></LI><LI class="toclevel-2 tocsection-114"><A href="#New_Bacillus_transformation_.28to_make_glycerol_stocks.29"><SPAN class="tocnumber">31.2</SPAN><SPAN class="toctext">New Bacillus transformation (to make glycerol stocks)</SPAN></A></LI><LI class="toclevel-1 tocsection-115"><A href="#September_2nd"><SPAN class="tocnumber">32</SPAN><SPAN class="toctext">September 2nd</SPAN></A></LI><LI class="toclevel-2 tocsection-116"><A href="#Transformation_of_Bacillus_3"><SPAN class="tocnumber">32.1</SPAN><SPAN class="toctext">Transformation of Bacillus</SPAN></A></LI><LI class="toclevel-2 tocsection-117"><A href="#Check_ECE112_stock"><SPAN class="tocnumber">32.2</SPAN><SPAN class="toctext">Check ECE112 stock</SPAN></A></LI><LI class="toclevel-1 tocsection-118"><A href="#September_3rd"><SPAN class="tocnumber">33</SPAN><SPAN class="toctext">September 3rd</SPAN></A></LI><LI class="toclevel-2 tocsection-119"><A href="#Result_from_bacillus_transformation_with_fresh_competent_cells_.2802.2F09.29"><SPAN class="tocnumber">33.1</SPAN><SPAN class="toctext">Result from bacillus transformation with fresh competent cells (02/09)</SPAN></A></LI><LI class="toclevel-2 tocsection-120"><A href="#Check_ECE112_stock_2"><SPAN class="tocnumber">33.2</SPAN><SPAN class="toctext">Check ECE112 stock</SPAN></A></LI><LI class="toclevel-2 tocsection-121"><A href="#Test_glycerol_stocks_.28from_02.2F09.29_of_competent_bacillus"><SPAN class="tocnumber">33.3</SPAN><SPAN class="toctext">Test glycerol stocks (from 02/09) of competent bacillus</SPAN></A></LI><LI class="toclevel-2 tocsection-122"><A href="#Starch_plates"><SPAN class="tocnumber">33.4</SPAN><SPAN class="toctext">Starch plates</SPAN></A></LI><LI class="toclevel-1 tocsection-123"><A href="#September_4th"><SPAN class="tocnumber">34</SPAN><SPAN class="toctext">September 4th</SPAN></A></LI><LI class="toclevel-2 tocsection-124"><A href="#Result_of_Bacillus_transformation_.28glycerol_stock_of_competent_cells.29"><SPAN class="tocnumber">34.1</SPAN><SPAN class="toctext">Result of Bacillus transformation (glycerol stock of competent cells)</SPAN></A></LI><LI class="toclevel-2 tocsection-125"><A href="#Result_of_starch_plates"><SPAN class="tocnumber">34.2</SPAN><SPAN class="toctext">Result of starch plates</SPAN></A></LI><LI class="toclevel-2 tocsection-126"><A href="#Colony_PCR_from_bacillus_subtilis_colonies"><SPAN class="tocnumber">34.3</SPAN><SPAN class="toctext">Colony PCR from bacillus subtilis colonies</SPAN></A></LI><LI class="toclevel-1 tocsection-127"><A href="#September_5th"><SPAN class="tocnumber">35</SPAN><SPAN class="toctext">September 5th</SPAN></A></LI><LI class="toclevel-2 tocsection-128"><A href="#Starch_plates_2"><SPAN class="tocnumber">35.1</SPAN><SPAN class="toctext">Starch plates</SPAN></A></LI><LI class="toclevel-1 tocsection-129"><A href="#September_6th"><SPAN class="tocnumber">36</SPAN><SPAN class="toctext">September 6th</SPAN></A></LI><LI class="toclevel-2 tocsection-130"><A href="#Results_of_starch_plates"><SPAN class="tocnumber">36.1</SPAN><SPAN class="toctext">Results of starch plates</SPAN></A></LI><LI class="toclevel-1 tocsection-131"><A href="#September_8th"><SPAN class="tocnumber">37</SPAN><SPAN class="toctext">September 8th</SPAN></A></LI><LI class="toclevel-2 tocsection-132"><A href="#Prepare_chromosomal_DNA_extraction_from_Bacillus"><SPAN class="tocnumber">37.1</SPAN><SPAN class="toctext">Prepare chromosomal DNA extraction from Bacillus</SPAN></A></LI><LI class="toclevel-2 tocsection-133"><A href="#PCR_AmyE_from_ECE112"><SPAN class="tocnumber">37.2</SPAN><SPAN class="toctext">PCR AmyE from ECE112</SPAN></A></LI><LI class="toclevel-1 tocsection-134"><A href="#September_9th"><SPAN class="tocnumber">38</SPAN><SPAN class="toctext">September 9th</SPAN></A></LI><LI class="toclevel-2 tocsection-135"><A href="#Chromosomal_DNA_extraction"><SPAN class="tocnumber">38.1</SPAN><SPAN class="toctext">Chromosomal DNA extraction</SPAN></A></LI><LI class="toclevel-2 tocsection-136"><A href="#PCR_AmyE_again"><SPAN class="tocnumber">38.2</SPAN><SPAN class="toctext">PCR AmyE again</SPAN></A></LI><LI class="toclevel-1 tocsection-137"><A href="#September_10th"><SPAN class="tocnumber">39</SPAN><SPAN class="toctext">September 10th</SPAN></A></LI><LI class="toclevel-2 tocsection-138"><A href="#Check_PCR_for_pSBINT1_vector"><SPAN class="tocnumber">39.1</SPAN><SPAN class="toctext">Check PCR for pSBINT1 vector</SPAN></A></LI><LI class="toclevel-1 tocsection-139"><A href="#September_11th"><SPAN class="tocnumber">40</SPAN><SPAN class="toctext">September 11th</SPAN></A></LI><LI class="toclevel-2 tocsection-140"><A href="#PCR_for_pBSINT1_biobrick_vector"><SPAN class="tocnumber">40.1</SPAN><SPAN class="toctext">PCR for pBSINT1 biobrick vector</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="July_22nd">July 22nd</SPAN></H1><P>We have 4 tubes from last year. These strains are frozen in glycerol. 
</P><UL><LI>PNZ8901 plasmid in E.coli MC1061 Strain
</LI><LI>Chloroamphenicol resistant
</LI><LI>&quot;sure&quot; vector
</LI><LI>does not integrate
</LI></UL><UL><LI>B. subtilis strain 1A1
</LI><LI>No resistance
</LI><LI>Same as strain 168 but tryptophan deficient
</LI></UL><UL><LI>E.coli strain MC1061, no plasmid
</LI></UL><UL><LI>PP182 plasmid in E.coli strain DH5alpha
</LI><LI>Ampicillin resistance
</LI><LI>integrates
</LI></UL><H2><SPAN class="mw-headline" id="Checking_antibiotic_resistance">Checking antibiotic resistance</SPAN></H2><P><B>Purpose</B> : Grow each of them on a plate to test their antibiotic resistance
</P><P><B>Protocol</B></P><UL><LI>Warm up frozen tubes
</LI><LI>Take 15uL of each and put it on plates without antibiotic
</LI><LI>Incubate at 37°C
</LI></UL><H1><SPAN class="mw-headline" id="July_23rd">July 23rd</SPAN></H1><UL><LI> Take one colony from each of the plates grown from the tubes yesterday and plated them again on LA without antibiotics
</LI></UL><P><B>1/ Purpose</B> Check antibiotic resistance of our different strains
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Strain
</TH><TH> Plasmid
</TH><TH> Extracted from
</TH><TH> Antibiotic use
</TH><TH> Concentration of antibiotic
</TH></TR><TR><TD> DH5alpha </TD><TD> PP182 </TD><TD> tube </TD><TD> Amp </TD><TD> 100
</TD></TR><TR><TD> MC1061 </TD><TD> NONE </TD><TD> tube </TD><TD> Cm </TD><TD> 35
</TD></TR><TR><TD> MC1061 </TD><TD> PNZ8901 </TD><TD> tube </TD><TD> Cm </TD><TD> 35
</TD></TR><TR><TD> DH5alpha </TD><TD> PP182 </TD><TD> plate </TD><TD> Amp </TD><TD> 100
</TD></TR><TR><TD> MC1061 </TD><TD> NONE </TD><TD> plate </TD><TD> Cm </TD><TD> 35
</TD></TR><TR><TD> MC1061 </TD><TD> PNZ8901 </TD><TD> plate </TD><TD> Cm </TD><TD> 35
</TD></TR></TBODY></TABLE><P><B>2/ Purpose</B> : Find the right concentration of antibiotic so that B. subtilis survive
</P><P>- Grow 15uL of B. 1A1 (frozen tube) in 5mL of LB
</P><P>- Incubate at 37°C
</P><P><B>3/ Purpose </B>: Grow plasmids in TOP10, transformation
</P><P>4 plasmids :
</P><UL><LI> I746000
</LI><LI> I746100
</LI><LI> I746101
</LI><LI> I746001
</LI></UL><P>1 control : PUC19
</P><P>- Add 20uL of TOP10 competent cells and 0.5uL of Plasmid in an eppendorf
</P><P>- 30min on ice
</P><P>- Heat shock : 60s at 42°C
</P><P>- 2min on ice
</P><P>- Add 89.5uL of SOC
</P><P>- 60min at 37°C
</P><P>- Put the mix on plate with ampicillin resistance
</P><P>- Incubate at 37°C
</P><H1><SPAN class="mw-headline" id="July_24th">July 24th</SPAN></H1><UL><LI> Test of antibiotic resistances of strains of last year
</LI></UL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Strain
</TH><TH> Plasmid
</TH><TH> Extracted from
</TH><TH> Antibiotic use
</TH><TH> Observations
</TH><TH> Conclusion
</TH></TR><TR><TD> DH5alpha </TD><TD> PP182 </TD><TD> tube </TD><TD> Amp </TD><TD> Many colonies </TD><TD> Amp Resistance OK
</TD></TR><TR><TD> DH5alpha </TD><TD> PP182 </TD><TD> plate </TD><TD> Amp </TD><TD> Many colonies </TD><TD> Idem
</TD></TR><TR><TD> MC1061 </TD><TD> NONE </TD><TD> tube </TD><TD> Cm </TD><TD> Nothing </TD><TD> As expecting, no Cm Resist.
</TD></TR><TR><TD> MC1061 </TD><TD> NONE </TD><TD> plate </TD><TD> Cm </TD><TD> Nothing </TD><TD> Idem
</TD></TR><TR><TD> MC1061 </TD><TD> PNZ8901 </TD><TD> plate </TD><TD> Cm </TD><TD> Maybe a few colonies </TD><TD> Contamination??
</TD></TR><TR><TD> MC1061 </TD><TD> PNZ8901 </TD><TD> tube </TD><TD> Cm </TD><TD> No colonies </TD><TD> no Cm Resist.
</TD></TR></TBODY></TABLE><UL><LI> Transformation of E.coli with different plasmids from last year
</LI></UL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Strain
</TH><TH> Inserted plasmid
</TH><TH> Antibiotic
</TH><TH> Observation
</TH><TH> Conclusion
</TH></TR><TR><TD> E.coli </TD><TD> I746000 </TD><TD> Amp </TD><TD> No colonies </TD><TD> Antibiotic resistance unknown, no Amp Resist., Pb : no terminator
</TD></TR><TR><TD> E.coli </TD><TD> I746100 </TD><TD> Amp </TD><TD> No colonies </TD><TD> Antibiotic resistance unknown, no Amp Resist., Pb : no terminator
</TD></TR><TR><TD> E.coli </TD><TD> I746001 </TD><TD> Amp </TD><TD> Many colonies </TD><TD> Transformation OK
</TD></TR><TR><TD> E.coli </TD><TD> I746101 </TD><TD> Amp </TD><TD> Many colonies </TD><TD> Transformation OK
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Antibiotic_test_for_Bacillus_resistance">Antibiotic test for Bacillus resistance</SPAN></H3><P>We want to find the lowest concentration of antibiotic which kills Bacillus.
</P><P><B>Dilution of Amp.</B> [ 1:1000 means 1 part stock to 1000 part Sterile Distilled Water ]
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Concentration (μg/mL) </TH><TH> 100 </TH><TH> 75 </TH><TH> 50 </TH><TH> 25 </TH><TH> 10
</TH></TR><TR><TD> 100 mg/mL Stock </TD><TD> 1:1000 </TD><TD> 3:4000 </TD><TD> 1:2000 </TD><TD> 1:4000 </TD><TD> 1:10000
</TD></TR></TBODY></TABLE><P><B>Dilution of Cm.</B> [ 1:1000 means 1 part stock to 1000 part Sterile Distilled Water ]
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Concentration (μg/mL) </TH><TH> 35 </TH><TH> 25 </TH><TH> 15 </TH><TH> 10 </TH><TH> 5
</TH></TR><TR><TD> 100 mg/mL Stock </TD><TD> 1:1000 </TD><TD> 1:1400 </TD><TD> 3:7000 </TD><TD> 1:3500 </TD><TD> 1:7000
</TD></TR></TBODY></TABLE><UL><LI>Add Disks in antibiotic solution for 5 mins [ Protocol to sterilize tweezers : Wipe with Kimwipes, Ethanol, Flame ]
</LI><LI>Melt Soft Agar
</LI><LI>3mL SA + 10μL Cells 1A1 from LB prepared on 23/7/08
</LI><LI>Pour SA+Cells over blank hard agar plates
</LI><LI>put disks with different concentration of Amp/Cm above SA
</LI><LI>Incubate at 37°C
</LI></UL><P><B>Results</B> [Plates were prepared before lunch, at 5pm there were visible growth]
</P><P>-Amp Plates: Huge rings of no growth around 100, 75, 50, 25, and 10.
</P><P>-Cm PLates: Tiny rings of no growth around 5, 10, 15 and 25. Small ring of no growth around 35 but not well defined. (No clear zones)
</P><H2><SPAN class="mw-headline" id="Salts_for_making_Bacillus_Competent">Salts for making Bacillus Competent</SPAN></H2><P><B>10x Medium A Base</B></P><UL><LI> Yeast Extract 10g
</LI><LI> Casamino Acid 2g
</LI><LI> add Distilled water to 900mL
</LI></UL><P>-Aliquot into 5 different bottles. [180mL each]
</P><P><B>10x Bacillus Salts</B></P><UL><LI>NH4)2SO4 20g
</LI><LI>K2PO4 anhydrous 139.66g
</LI><LI>KH2PO4 60g
</LI><LI>Tri-Sodium Citrate 10g
</LI><LI>MgSO4.7H2O 2g
</LI><LI>Add Distilled water to 1000mL
</LI></UL><P>-Aliguot into 5 different bottles. [200mL each]
</P><P><B>Medium B</B></P><UL><LI>Preparing 50mM CaCl2.2H2O
</LI><LI>CaCl2.2H2O 1.470g
</LI><LI>Add Distilled water to 20mL [Final conc. 500mM]
</LI><LI>Take 10mL of 500mM
</LI><LI>Add 90mL of distilled water [Final conc. 50mM]
</LI></UL><UL><LI>Preparing 250nM MgCl2.6H2O
</LI><LI>MgCl2.6H2O 0.508g
</LI><LI>Add Distilled water to 10mL [Final conc. 250mM]
</LI><LI>Take 1mL of 250mM
</LI><LI>Add 99mL of Distilled water [Final conc. 250μM]
</LI><LI>Take 1mL of 250μM
</LI><LI>Add 99mL of Distilled water [Final conc. 250nM]
</LI></UL><UL><LI>Add 100mL of 50mM CaCl2.2H2O
</LI><LI>Add 100mL of 250nM MgCl2.6H2O
</LI></UL><P>-Total volume 200mL
</P><P><B>NOTE: To complete Medium B, Take 0.2mL of this solution</B></P><H1><SPAN class="mw-headline" id="July_25th">July 25th</SPAN></H1><H3><SPAN class="mw-headline" id="Results_from_Yesterday">Results from Yesterday</SPAN></H3><UL><LI>Antibiotic test
</LI></UL><P><B>Results for Amp</B></P><P>Even with a concentration of 10μg/mL, Amp kills Bacillus. Since we want to know which is the lowest concentration of Amp which kills B.S, we are going to test with some lower concentrations.
</P><P><B>Results for Cm</B></P><P>Even with a concentration of 35μg/mL, ther is not a clear area around the antibiotic disk. So we have to test some higher concentration of Cm.
</P><H2><SPAN class="mw-headline" id="Wet_Work">Wet Work</SPAN></H2><UL><LI> New antibiotic tests
</LI></UL><P><B>Dilution of Amp</B></P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Concentration (μg/mL) </TH><TH> 10 </TH><TH> 7.5 </TH><TH> 5 </TH><TH> 2.5 </TH><TH> 1
</TH></TR><TR><TD> 10 μg/mL Stock (from yesterday) </TD><TD> 1:1 </TD><TD> 3:4 </TD><TD> 1:2 </TD><TD> 1:4 </TD><TD> 1:10
</TD></TR></TBODY></TABLE><P><B>Dilution of Cm</B></P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Concentration (μg/mL) </TH><TH> 35 </TH><TH> 37.5 </TH><TH> 40 </TH><TH> 42.5 </TH><TH> 45
</TH></TR><TR><TD> 35 μg/mL Stock (from yesterday) </TD><TD> 1:1 </TD><TD> 5:6 (from 45μg/mL) </TD><TD> 1:875 </TD><TD> 17:18 (from 45μg/mL) </TD><TD> 9:7000
</TD></TR></TBODY></TABLE><P>- Melt Soft Agar
- 3mL SA + 10μL Cells 1A1 from LB prepared on 23/7/08
- Pour SA+Cells over blank hard agar plates
- Put 5 blank disks on each agar plate
- Add 40μL of antibiotic on each disk
- Incubate at 37°C
</P><P><B>Results</B></P><P>Nothing! 1A1 cells were kept in the freedge! B.S. can not be kept in the fridge, low temperatures kill them!
</P><H1><SPAN class="mw-headline" id="July_28th">July 28th</SPAN></H1><UL><LI><B>Results of antibiotic plates from yesterday</B></LI></UL><P>For Cm, 35μg/mL should be enough to kill B.S.
</P><P>For Amp, nothing can be concluded!
</P><H2><SPAN class="mw-headline" id="Wet_Work_2">Wet Work</SPAN></H2><UL><LI><B>Check plasmid ppL82</B></LI></UL><P>We had 2 samples of ppL82 in DH5α in LB solution (~4mL), one from the frozen glycerol tube and one from a colony picked on a plate.
Normally plasmid and cells should not be kept in freezer. So, we want to extract this plasmid, check its size and then keep it in the fridge.
We do that for the 2 different samples.
</P><P>
- Plasmid Miniprep (standard protocol)
</P><P>- Test the concentration of DNA in each tube
</P><OL><LI> ppL82 plate : 89.8ng/mL
</LI><LI> ppL82 tube : 71ng/mL
</LI></OL><UL><LI><B>Preparation of different stocks of strains and plasmids</B></LI></UL><P><B>PNZ8901</B></P><P>- Plate a single colony (from 23/07/08) on a Cm plate
</P><P>- Incubate at 37°C
</P><P>
- Grow the entire frozen glycerol tube in 20mL of LB without antibiotic to check if this stock is still good (we will check the plasmid on a gel tomorrow)
</P><P>- Incubate at 37°C
</P><P><B>MC1061</B></P><P>-Pick e single colony of MC1061 (from 22/07/08) and put it in 10mL of LB (to make glycerol stocks)
</P><P>-Incubate at 37°C
</P><P><B>1A1</B></P><P>- Add 100μL of LB+1A (mix of friday) and 10mL of LB
</P><P>- Incubate at 37°C
</P><UL><LI><B>Make B.S. 1A1 competent and transform them</B></LI></UL><P>- Prepare Medium A
</P><P>Add 81mL of SDW, 10mL of 10X Medium A base and 9mL of 10X Bacillus salts
</P><P>
- Make B.S. 1A1 competent
</P><OL><LI>In 10mL of medium A and add about 10 colonies of B.S.
</LI><LI>Check the OD650, you should have an OD between 0.1 and 0.2. t0 for OD = 0.1876
</LI><LI>Check the OD650 every 20min and plot OD against time on semi-log paper. When the point at the culture leaves log growth, it is ok
</LI></OL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Time (min) </TH><TH> 0 </TH><TH> 20 </TH><TH> 40 </TH><TH> 60 </TH><TH> 70 </TH><TH> 80
</TH></TR><TR><TD> OD650 </TD><TD> 0.1876 </TD><TD> 0.2074 </TD><TD> 0.3282 </TD><TD> 0.4545 </TD><TD> 0.4895 </TD><TD> 0.5040
</TD></TR></TBODY></TABLE><P>At, t0 = 70min, log growth seems to stop.
</P><OL><LI>Incubate at 37°C 90min after t0
</LI><LI>Warn 10 Ependorf tubes with 0.45mL of Medium B
</LI><LI>Add 50μL of culture in each tube of Medium B at t90
</LI><LI>Incubate the diluted culture at 37°C with vigorous aeration for 90min
</LI></OL><P>- Storage : we want to store 7 tubes
</P><P>6 tubes in the freezer (with 60μL of glycerol)
</P><P>1 tube on the bench
</P><P>- Transformation
</P><OL><LI>We make ppL82 from plate, ppL82 from tube, and a control
</LI><LI>Add 0.5μg of DNA (15μL)
</LI><LI>Incubate at 37°C for 30min
</LI><LI>Plate on Cm plates
</LI></OL><H1><SPAN class="mw-headline" id="July_29th">July 29th</SPAN></H1><UL><LI><B>Result from the gel (29/07/2008)</B></LI></UL><UL><LI> Lane8 : ppL82 (plate)
</LI><LI> Lane9 : ppL82 (tube)
</LI><LI> Lane 10 : I746001
</LI><LI> Lane 11 : I746101
</LI><LI> Lane 12 : hyperladderI
</LI></UL><P>The ladder seems to be wrong. So it was really difficult to ckeck the size of our plasmid ppL82. To make, that, we assume that the size of our biobricks was ok, and we estimate the size of our plasmid. The plasmid ppL82 seems to have the right size, but we will have to check again to be able to use a ladder.
</P><UL><LI><B>Competence of B.S. kept on the bench</B></LI></UL><P>We observed B.S. with a microscope. Problem, all B.S. seems to be dead!
</P><P><I>Possible reasons of this problem</I> : 
</P><P>- Problem with the vector (it could be a wrong vector, we are not really sure)
</P><P>- Quantity of DNA, we added 0.5μg. The protocol was 1μg, but normally it should be ok.
</P><P>- B.S. may not be competent (we have to test motility with microscope)
</P><P>- Cells could be dead : replate them
</P><P>- Try to add tryptophan in the medium
</P><P><I>Probable reason</I> : we forgot to dilute the medium B, that's why it did not work
</P><UL><LI><B>Results from transformation plates (B.S.)</B></LI></UL><P>
There are 2 colonies on a plate, but it does not look like Bacillus. Moreover, there is also a colony on our negative control plate. SO it must be contamination (resistant to CM!). We will do the transformation again.
</P><H2><SPAN class="mw-headline" id="Wet_Work_3">Wet Work</SPAN></H2><UL><LI><B>Prepare medium for transformation of B.S.</B></LI></UL><P>
- Preparation of medium B: 10mL of medium A and 0.2mL of 50mMCaCl22(H2O) + 250mMgCl26(H2O)
</P><P>- Preparation of medium A with tryptophan : 81mL of SDW, 9mL of 10X Bacillus salts, 10mL of 10X Medium A base and 0.1mL of Tryptophan (11mg/mL)
</P><UL><LI><B>Check  plasmid PNZ8901</B></LI></UL><P>
- Plasmid miniprep (same protocol with 60μL of elution buffer)
</P><P>- Digest : with PstI and SalI from Biolabs, and Buffer 3 (add 15μL of DNA)
</P><P>- Gel (17μL of sample and 3μL of dye)
</P><H2><SPAN class="mw-headline" id="Results_from_the_gel">Results from the gel</SPAN></H2><UL><LI> Lane9 : PNZ8901
</LI><LI> Lane10 : HyperladderI
</LI></UL><P>The sizes we expected were about 1100kb and 2100kb. The sizes should be ok!
</P><H1><SPAN class="mw-headline" id="July_30th">July 30th</SPAN></H1><UL><LI><B>Transforming Bacillus Subtilis with medium A and medium A + tryptophan</B></LI></UL><P>Medium A
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Time (min) </TH><TH> 0 </TH><TH> 20 </TH><TH> 40 </TH><TH> 60 </TH><TH> 73 </TH><TH> 85 </TH><TH> 95 </TH><TH> 105
</TH></TR><TR><TD> OD650 </TD><TD> 0.1394 </TD><TD> 0.1370 </TD><TD> 0.1660 </TD><TD> 0.2304 </TD><TD> 0.2778 </TD><TD> 0.3138 </TD><TD> 0.3373 </TD><TD> 0.3583
</TD></TR></TBODY></TABLE><P>For this medium, the curb stop to increase logarithmically at 100min.
</P><P>
Medium A + tryptophan
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Time (min) </TH><TH> 0 </TH><TH> 20 </TH><TH> 40 </TH><TH> 60 </TH><TH> 73 </TH><TH> 85 </TH><TH> 95
</TH></TR><TR><TD> OD650 </TD><TD> 0.1493 </TD><TD> 0.1636 </TD><TD> 0.2080 </TD><TD> 0.2909 </TD><TD> 0.3512 </TD><TD> 0.4002 </TD><TD> 0.4165
</TD></TR></TBODY></TABLE><P>For this medium, the curb stop to increase logarithmically at 90min.
</P><P>
- Incubate 90min at 37°C by shaking
</P><P>- Prepare 7 tubes for medium A and 7 tubes for medium A + tryptophan : add 0.45mL of prewarmed medium B and 0.05mL of culture and incubate 90min at 37°C by shaking.
</P><P>- Transform B.S. by adding DNA and incubate 30min at 37°C
</P><P>We have one transformation with ppL82 (add 7.5μL of DNA), one with PNZ8901 (add 10μL of DNA) and one negative control without DNA.
</P><P>- Plate 500μL on Cm plate (35μg/mL)
</P><P>
- For the remaining tubes, in one, add 60μL of glycerol and keep it in the freezer and keep the other one on the bench (for each medium) 
</P><UL><LI><B>Checking our big stock of biobricks and PNZ plasmid</B></LI></UL><P>
We did big culture of biobricks and PNZ8901 plasmid to be able to make stocks. We want to check them before making stocks.
</P><P>- Plamsid miniprep (for I746001, I746101 and PNZ8901 from big flasks
</P><P>- Measure DNA concentration in our samples to decide the volume of DNA we have to add in our preparation
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> 260/280
</TH><TH> DNA concentration (ng/nl)
</TH></TR><TR><TD> I746001 </TD><TD> 1.79 </TD><TD> 50.5
</TD></TR><TR><TD> I746101 </TD><TD> 1.87 </TD><TD> 54.3
</TD></TR><TR><TD> PNZ8901 </TD><TD> 1.84 </TD><TD> 87
</TD></TR></TBODY></TABLE><P>
- Digest
</P><TABLE class="wikitable"><TBODY><TR><TH> For I746001 and I746101
</TH><TH> For PNZ8901 plasmid
</TH></TR><TR><TD> 14μL of SDW
</TD><TD> 14μL of SDW
</TD></TR><TR><TD> 2μL of 10X Fast Digest Buffer
</TD><TD> 2μL of Buffer 3 (Biolabs)
</TD></TR><TR><TD> 2μL of DNA of biobricks
</TD><TD> 2μL of DNA of PNZ8901
</TD></TR><TR><TD> 1μL of EcoRI
</TD><TD> 1μL of PstI
</TD></TR><TR><TD> 1μL of SpeI
</TD><TD> 1μL of SalI
</TD></TR></TBODY></TABLE><P>
- Gel
</P><UL><LI><B>Results from this gel</B></LI></UL><UL><LI> Lane 9 : I746001
</LI><LI> Lane 10 : I746101
</LI><LI> Lane 11 : PNZ8901
</LI><LI> Lane 12 : HyperladderI
</LI></UL><P>Everthing is really too big! There is a problem, either dimerization, either contamination, either a problem in our work. So we are going to run a new gel.
</P><UL><LI><B>New gel to check</B></LI></UL><P>- Miniprep plasmid from growth bottles (I746001, I746101 and PNZ8901); from plates (I746001, I746101 and PNZ8901).
</P><P>- Do single digest for PNZ8901 (one with PstI, one with SalI)
</P><P>- Run a gel with : PNZ8901 from friday, PNZ8901 digest with PstI, PNZ8901 digest with SalI, 3 samples from growth bottles, 3 samples from plates (to check the size of the uncut vectors)
</P><UL><LI><B>Result from this second gel</B></LI></UL><P>The ladders are really bad! But the size of our biobricks and plasmid are too big! So we can not trust these big cultures. They is a problem.  With tests from the first week, we know that biobricks are right, so we are going to grow new cultures from these first cultures, to make stocks. 
</P><P>Concerning the vectors, they are from last year, so we are not sure of what they are. Since we ordered new well defined vectors (we should receive them on friday), we will use them in the next steps to be sure of our work.
</P><H1><SPAN class="mw-headline" id="August_1st">August 1st</SPAN></H1><UL><LI><B>Grow received vectors</B></LI></UL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Vectors </TH><TH> AmpR (μg/μL) </TH><TH> CmR (μg/μL) </TH><TH> KanR (μg/μL)
</TH></TR><TR><TD> ECE112</TD><TD> 100 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> ECE147</TD><TD> 50 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> ECE149</TD><TD> 50 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> ECE150</TD><TD> 50 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> ECE151</TD><TD> 50 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> ECE153</TD><TD> 100 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> ECE162</TD><TD> 100 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> ECE165</TD><TD> 100 </TD><TD> 10 </TD><TD> 0
</TD></TR><TR><TD> ECE166</TD><TD> 100 </TD><TD> 10 </TD><TD> 0
</TD></TR><TR><TD> ECE171</TD><TD> 50 </TD><TD> 0 </TD><TD> 10
</TD></TR><TR><TD> ECE172</TD><TD> 100 </TD><TD> 10 </TD><TD> 0
</TD></TR><TR><TD> ECE176</TD><TD> 100 </TD><TD> 5 </TD><TD> 0
</TD></TR></TBODY></TABLE><P>- <B>Preparation of Agar plates</B></P><P>We have bottles of 200mL.
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Type </TH><TH> Amp (μL) </TH><TH> Cm (μL) </TH><TH> Kan (μL)
</TH></TR><TR><TD> Amp100 </TD><TD> 200 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> Amp50 </TD><TD> 100 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> Amp100 + Cm10</TD><TD> 200 </TD><TD> 57.1 </TD><TD> 0
</TD></TR><TR><TD> Amp100 + Cm5</TD><TD> 200 </TD><TD> 28.6 </TD><TD> 0
</TD></TR><TR><TD> Amp50 + Kan10</TD><TD> 50 </TD><TD> 0 </TD><TD> 80
</TD></TR></TBODY></TABLE><P>
- <B>Preparation of LB tubes</B></P><P>We prepare 10mL of LB in which we add a single colony.
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Type </TH><TH> Amp (μL) </TH><TH> Cm (μL) </TH><TH> Kan (μL)
</TH></TR><TR><TD> Amp100 </TD><TD> 10 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> Amp50 </TD><TD> 5 </TD><TD> 0 </TD><TD> 0
</TD></TR><TR><TD> Amp100 + Cm10</TD><TD> 10 </TD><TD> 2.9 </TD><TD> 0
</TD></TR><TR><TD> Amp100 + Cm5</TD><TD> 10 </TD><TD> 1.4 </TD><TD> 0
</TD></TR><TR><TD> Amp50 + Kan10</TD><TD> 5 </TD><TD> 0 </TD><TD> 4
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="August_2nd">August 2nd</SPAN></H1><H2><SPAN class="mw-headline" id="Yesterday.27s_Results">Yesterday's Results</SPAN></H2><UL><LI> All Plates grew!
</LI><LI>ECE 172 seemed to have trouble growing on the plate (no single colonies, just one big lump at the start of the streak)
</LI><LI>ECE 150 grew too well! (possibly hard to pick out single colony)
</LI></UL><UL><LI>Tubes:
</LI><LI>ECE 176, 172, 153 seem to be quite clear!!
</LI><LI>ECE 150, 151 had strange floating 'colonies' in the tube
</LI><LI>All other tubes were quite cloudy
</LI></UL><H2><SPAN class="mw-headline" id="Lab_Work">Lab Work</SPAN></H2><P>Fridged all plates except ECE 172 ==&gt; Benched.
</P><P>All tubes palleted and placed in freezer although ECE 176, 172, 153 doesn't seem to have any pallets
</P><P>All red plates does not seem to have any extra growth despite the air conditioning had been turned off when I entered the lab this afternoon.
</P><H1><SPAN class="mw-headline" id="August_4th">August 4th</SPAN></H1><H2><SPAN class="mw-headline" id="Wet_Work_4">Wet Work</SPAN></H2><UL><LI><B>Check vectors</B></LI></UL><P>- for each vector, plasmid miniprep from frozen pellets (step 7 only once, with 60μL of Elution Buffer for ECE112, ECE147, ECE149, ECE 150 and only 30μL for the others)
</P><P>- Single digest (15μL of SDW, 2μL of Buffer, 2μL of DNA, 1μL of enzyme, then incubate 10min at 37°C and 30 min for the one using HindIII with EcoRI buffer)
</P><P>- Double digest (mix 7.5μL of each single digest, and then incubate 10min (or 30min for EcoRI + HindIII and EcoRI buffer) at 37°C, then heat shock 5min at 80°C
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Vectors </TH><TH> Enzyme 1 </TH><TH> Enzyme 2 </TH><TH> Buffer </TH><TH> Entire size </TH><TH> Size of part1 </TH><TH> Size of part2
</TH></TR><TR><TD> ECE112</TD><TD> XbaI </TD><TD> EcoRI </TD><TD> Fast Digest </TD><TD> 10156 </TD><TD> 6900 </TD><TD> 3295
</TD></TR><TR><TD> ECE147</TD><TD> EcoRI </TD><TD> HindIII </TD><TD> EcoRI </TD><TD> 5482 </TD><TD> 4913 </TD><TD> 600
</TD></TR><TR><TD> ECE149</TD><TD> SpeI </TD><TD> PstI </TD><TD> Fast Digest </TD><TD> 6192 </TD><TD> 4919 </TD><TD> 1300
</TD></TR><TR><TD> ECE150</TD><TD> SpeI </TD><TD> HimdIII </TD><TD> Buffer 2 </TD><TD> 6166 </TD><TD> 5300 </TD><TD> 778
</TD></TR><TR><TD> ECE151</TD><TD> SpeI </TD><TD> EcoRI </TD><TD> Fast Digest </TD><TD> 6166 </TD><TD> 4825 </TD><TD> 1300
</TD></TR><TR><TD> ECE153</TD><TD> SalI </TD><TD> Buffer 3 </TD></TR><TR><TD> ECE162</TD><TD> SalI </TD><TD> Buffer 3 </TD><TD> 7600 </TD><TD> 6000 </TD><TD> 1600
</TD></TR><TR><TD> ECE165</TD><TD> EcoRI </TD><TD> HindIII </TD><TD> EcoRI </TD><TD> 5952 </TD><TD> 5200 </TD><TD> 793
</TD></TR><TR><TD> ECE166</TD><TD> EcoRI </TD><TD> HindIII </TD><TD> EcoRI </TD><TD> 7262 </TD><TD> 5100 </TD><TD> 2103
</TD></TR><TR><TD> ECE171</TD><TD> EcoRI </TD><TD> PstI </TD><TD> Fast Digest </TD><TD> 5444 </TD><TD> 3600 </TD><TD> 1826
</TD></TR><TR><TD> ECE172</TD><TD> HindIII </TD><TD> Buffer 2 </TD><TD> 6462 </TD><TD> 3900 </TD><TD> 2540
</TD></TR><TR><TD> ECE176</TD><TD> EcoRI </TD><TD> XbaI </TD><TD> Fast Digest </TD><TD> 866 </TD><TD> 4250 </TD><TD> 5128
</TD></TR></TBODY></TABLE><UL><LI><B>Results</B></LI></UL><UL><LI> Lane1 : HyperladderI
</LI><LI> Lane2 : ECE112
</LI><LI> Lane3 : ECE147
</LI><LI> Lane4 : ECE149
</LI><LI> Lane5 : ECE150
</LI><LI> Lane6 : ECE151
</LI><LI> Lane7 : ECE162
</LI><LI> Lane8 : ECE165
</LI><LI> Lane9 : ECE166
</LI><LI> Lane10 : ECE171
</LI><LI> Lane11 : ECE172
</LI><LI> Lane12 : HyperladderI
</LI></UL><P>- ECE112, ECE165, ECE166, ECE171 : right bands, ok
</P><P>- ECE172 : no band, not enough DNA
</P><P>- ECE147, ECE150, ECE162 : only one band, maybe not enough DNA
</P><P>- ECE149 : wrong size!
</P><P>- ECE151 : not sure, check again
</P><UL><LI><B>Plates</B></LI></UL><P>- Streak new plates with different strains of Bacillus : 1A1, IA751 and IA771
</P><H1><SPAN class="mw-headline" id="August_5th">August 5th</SPAN></H1><H2><SPAN class="mw-headline" id="Checking_vectors">Checking vectors</SPAN></H2><P>- Plasmid miniprep for ECE166 (for stock), ECE172 and ECE153
</P><P>- Digest of ECE153, ECE172 and ECE162 (with plasmid miniprep from yesterday)
</P><P>- Run on a gel single digest for ECE147 (from yesterday with more DNA), ECE149 (from yesterday with more DNA), ECE150 (from yesterday with more DNA) and ECE 153, ECE162, ECE172
</P><UL><LI><B>Results</B></LI></UL><UL><LI> Lane3 : HyperladderI
</LI><LI> Lane4 : ECE147
</LI><LI> Lane5 : ECE149
</LI><LI> Lane6 : ECE150
</LI><LI> Lane7 : ECE151
</LI><LI> Lane8 : ECE1162
</LI><LI> Lane9 : ECE172
</LI><LI> Lane10 : hyperladderI
</LI></UL><P>Very low bands : not enough DNA!!!
</P><H2><SPAN class="mw-headline" id="Transformation_of_Bacillus">Transformation of Bacillus</SPAN></H2><UL><LI> Result of Nanodrop
</LI></UL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Vector </TH><TH> 260/280 </TH><TH> ng/μL
</TH></TR><TR><TD> ECE112 </TD><TD> 1.75 </TD><TD> 64.6
</TD></TR><TR><TD> ECE166 </TD><TD> 172 </TD><TD> 138.6
</TD></TR></TBODY></TABLE><P>- Prepare medium A with tryptophan, and medium B
</P><P>- add 5mL of medium A in 3 different tubes, inoculate each tube with colonies (1A1, IA751 ans IA771)
</P><P>- Check OD every 20min
</P><P>- Incubate 90min
</P><P>- Add 0.45mL of medium B and 0.05mL of culture in Ependorf tubes
</P><P>- Incubate 90min
</P><P>- Transform : add 1μg of DNA (some with ECE112, some with ECE166) (so you need to nanodrop samples before!)
</P><P>- Incubate 30min
</P><P>- Pipette 200μL of solution, spread it on aech plate, wait 10min, and do it again
</P><P>- Incubate 24hours
</P><P>- A few glycerol tubes to stock cells : add 2/7 glycerol to cell tubes
</P><P>
- Transformation from glycerol stock from 30/07/2008
</P><P>- Spin glycerol stocks, pipette out glycerol
</P><P>- Add 0.5mL of medium B, incubate for 1 hour
</P><P>- Add 10μL of ECE112 (640ng)
</P><P>- Incubate 2hours
</P><P>- Plate 200μL, and 10min later, still 200μL
</P><H2><SPAN class="mw-headline" id="New_stocks">New stocks</SPAN></H2><P>- Do glycerol stock of I746001 and I746101 (no sterile glycerol)
</P><P>- Put IA751, IA771 in 10mL LB
</P><P>- Put ECE 176 in 10mL LB + antibiotic
</P><P>- Reinoculate the tube of LB from yesterday with ECE166 plate
</P><P>- ECE176 replated onto Amp100 + Cm5
</P><H1><SPAN class="mw-headline" id="August_6th">August 6th</SPAN></H1><H2><SPAN class="mw-headline" id="Result_from_transformation">Result from transformation</SPAN></H2><P>PLATES HAD BEEN BINNED!! OOPS
</P><H2><SPAN class="mw-headline" id="Check_Vectors">Check Vectors</SPAN></H2><P>- Plasmid miniprep for ECE176 (from 05/08 LB stock and from 04/08 LB stock with a second inoculation)
</P><P>We want to check uncut plasmid, single and double digest for ECE147, ECE149, ECE150, ECE153, ECE162, ECE172, ECE176. In order to have enough DNA (to see the bands), we will add 1μg of DNA to make single digest, and wr will incubate during 1h (instead of only 10min).
</P><UL><LI> Nanodrop
</LI></UL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Vetor </TH><TH> 260/280 </TH><TH> μg/mL
</TH></TR><TR><TD> ECE147 </TD><TD> 1.67 </TD><TD> 50.8
</TD></TR><TR><TD> ECE149 </TD><TD> 1.66 </TD><TD> 51.9
</TD></TR><TR><TD> ECE150 </TD><TD> 1.78 </TD><TD> 107.0
</TD></TR><TR><TD> ECE153 </TD><TD> 1.68 </TD><TD> 31.2
</TD></TR><TR><TD> ECE162</TD><TD> 1.56 </TD><TD> 33.1
</TD></TR><TR><TD> ECE172 (04/08)</TD><TD> 1.64 </TD><TD> 22.8
</TD></TR><TR><TD> ECE172 (05/08) </TD><TD> 1.89 </TD><TD> 213.1
</TD></TR><TR><TD> ECE176 (04/08)</TD><TD> 1.89 </TD><TD> 111.1
</TD></TR><TR><TD> ECE176 (05/08) </TD><TD> 1.83 </TD><TD> 96.2
</TD></TR></TBODY></TABLE><P>We made singe digest for the samples from 05/08 only.
</P><P>- Double digest (1 hour of incubation in water bath at 37°C)
</P><P>- Gel
</P><P>Gel1
</P><UL><LI> Lane1 : HyperladderI
</LI><LI> Lane2 : ECE147 with EcoRI
</LI><LI> Lane3 : ECE147 with HindIII
</LI><LI> Lane4 : ECE149 with SpeI
</LI><LI> Lane5 : ECE149 with PstI
</LI><LI> Lane6 : ECE150 with SpeI
</LI><LI> Lane7 : ECE150 with HindIII
</LI><LI> Lane8 : ECE153 with SalI
</LI><LI> Lane9 : ECE162 with SalI
</LI><LI> Lane10 : ECE172 with HindIII
</LI><LI> Lane11 : ECE176 with EcoRI
</LI><LI> Lane12 : HyperladderI
</LI></UL><P>Gel2
</P><UL><LI> Lane1 : HyperladderI
</LI><LI> Lane2 : ECE176 with XbaI
</LI><LI> Lane3 : ECE176 double digest
</LI><LI> Lane4 : ECE150 double digest
</LI><LI> Lane5 : ECE149 double digest
</LI><LI> Lane6 : ECE147 double digest
</LI><LI> Lane7 : HyperladderI
</LI><LI> Lane8 : ECE149 uncut
</LI><LI> Lane9 : ECE153 uncut
</LI><LI> Lane10 : ECE172 uncut
</LI><LI> Lane11 : supercoiled ladder
</LI></UL><UL><LI> Results
</LI></UL><P>Our supercoiled ladder was very bad, so it was impossible to conclude for uncut vectors.
</P><P>- ECE147, ECE 150, ECE153, ECE 176 single digest : ok
</P><P>- ECE149 : problem, 2 cutting sites for PstI (only one in the sequence)
</P><P>- ECE162 : only one band whereas SalI should have 2 cutting sites
</P><P>- ECE172 : wrong sizes!!!
</P><P>- Double digest for ECE176 : ok!
</P><P>- Double digest for ECE 150, 149 and 147 : only one band!!!
</P><P>There may be a problem when we do the double digest by adding two single digest. We will try again with a direct double digest.
</P><H1><SPAN class="mw-headline" id="August_7th">August 7th</SPAN></H1><H2><SPAN class="mw-headline" id="Transformation_of_B.S._IA771">Transformation of B.S. IA771</SPAN></H2><P>New transformation with the same protocol than 2 days ago. We will try to add less liquid on plates (to avoid growing colonies in liquid)
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Tube </TH><TH> Plate 1 </TH><TH> Plate 2 </TH><TH> Plate 3 </TH><TH> Plate 4
</TH></TR><TR><TD> ECE166 </TD><TD> Cm5 + 200μL of cells </TD><TD> Cm5 + 100μL of cells </TD><TD> Cm5 + 50μL of cells </TD><TD> Cm10 + 100μL of cells
</TD></TR><TR><TD> ECE153 </TD><TD> Spc50 + 200μL of cells </TD><TD> Spc50 + 100μL of cells </TD><TD> Spc50 + 50μL of cells </TD></TR><TR><TD> ECE166 </TD><TD> Cm5 + 100μL of cells </TD><TD> Cm10 + 100μL of cells </TD></TR><TR><TD> ECE153 </TD><TD> Spc50 + 100μL of cells </TD></TR><TR><TD> no DNA </TD><TD> Cm5 + 100μL of cells </TD><TD> Cm10 + 100μL of cells </TD><TD> Spc50 + 100μL of cells </TD></TR><TR><TD> ECE166 (glycerol stock)</TD><TD> Cm5 + 100μL of cells </TD><TD> Cm10 + 100μL of cells </TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Check_Vectors_2">Check Vectors</SPAN></H2><P>We want to make a final check for ECE147, 149, 150, 153 and 162.
</P><P>- Double digest + gel
</P><UL><LI> Results
</LI></UL><UL><LI> Lane3 : HyperladderI
</LI><LI> Lane4 : ECE147
</LI><LI> Lane5 : ECE149
</LI><LI> Lane6 : ECE150
</LI><LI> Lane7 : ECE153
</LI><LI> Lane8 : ECE162
</LI><LI> Lane9 : HyperladderI
</LI></UL><P>- ECE 147, ECE150, ECE153 :ok!
</P><P>- ECE 149 : 3 bands! not the right vector
</P><P>- ECE162 : ony one band! It has been uncut... maybe the good vector (according to precedent gel), but not sure
</P><H2><SPAN class="mw-headline" id="Test_ECE112_transformation">Test ECE112 transformation</SPAN></H2><P>- for the three different strain (1A1, IA771, IA751), make 12 spots
</P><P>- take one colony with a loop, and put it on a Cm5 plate and on a Cm5 + Spc100 plate (do that for the 12 spots)
</P><P>- incubate
</P><H2><SPAN class="mw-headline" id="New_stock">New stock</SPAN></H2><P>- Put ECE171 in 10mL of LB
</P><P>- Incubate
</P><H1><SPAN class="mw-headline" id="August_8th">August 8th</SPAN></H1><H2><SPAN class="mw-headline" id="Result_from_yesterday_transformation">Result from yesterday transformation</SPAN></H2><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Vector </TH><TH> Antibiotic </TH><TH> Quantity of cells added </TH><TH> number of colonies
</TH></TR><TR><TD> ECE166 </TD><TD> Cm5 </TD><TD> 200μL </TD><TD> 0 : problem!!!
</TD></TR><TR><TD> ECE166 </TD><TD> Cm5 </TD><TD> 100μL </TD><TD> 2
</TD></TR><TR><TD> ECE166 </TD><TD> Cm5 </TD><TD> 50μL </TD><TD> 1
</TD></TR><TR><TD> ECE166 </TD><TD> Cm10 </TD><TD> 100μL </TD><TD> 0 : no resistance to Cm10!
</TD></TR><TR><TD> ECE166 (spin) </TD><TD> Cm5 </TD><TD> 100μL </TD><TD> a lot, confluent
</TD></TR><TR><TD> ECE166 (spin) </TD><TD> Cm10 </TD><TD> 100μL </TD><TD> 0
</TD></TR><TR><TD> ECE166 (glycerol + spin) </TD><TD> Cm5 </TD><TD> 100μL </TD><TD> about 150, very small
</TD></TR><TR><TD> ECE166 (spin) </TD><TD> Cm10 </TD><TD> 100μL </TD><TD> 0
</TD></TR><TR><TD> ECE153 </TD><TD> Spc50 </TD><TD> 200μL </TD><TD> 21 + a lot of confluent colonies
</TD></TR><TR><TD>  ECE153 </TD><TD> Spc50 </TD><TD> 100μL </TD><TD> 7 + confluent (a few)
</TD></TR><TR><TD>  ECE153 </TD><TD> Spc50 </TD><TD> 50μL </TD><TD> 4
</TD></TR><TR><TD>  ECE153 (spin) </TD><TD> Spc50 </TD><TD> 100μL </TD><TD> 25 + about 200 (small and almost confluent)
</TD></TR><TR><TD>  No DNA </TD><TD> Cm5 </TD><TD> 100μL </TD><TD> 0
</TD></TR><TR><TD>  No DNA </TD><TD> Cm10 </TD><TD> 100μL </TD><TD> 0
</TD></TR><TR><TD>  No DNA </TD><TD> Spc50 </TD><TD> 100μL </TD><TD> about 12, maybe more : problem!!!!!
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Result_for_test_of_ECE112_transformation">Result for test of ECE112 transformation</SPAN></H2><P>On Cm5 plates, we have some colonies for each strains, on Cm5 + Spc 100 plates, no colonies on each plates. This result seems good, however, we forgot to make a control. Since, we had some colonies on control plates, it is possible that our bacillus are resistant to Cm5, even if they are not transformed. 
We addded some control colonies.
</P><H2><SPAN class="mw-headline" id="Control_with_erythromycin">Control with erythromycin</SPAN></H2><P>- Prepare erythromycin, stock 5mg/mL : 0.05g in 10mL of SDW
</P><P>- Prepare plates : add 20μL of Ery (5mg/mL) into 200mL of agar (final concentration : 0.5μg/mL)
</P><P>- Do several spots on each plate
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Plate </TH><TH> Colonies added from
</TH></TR><TR><TD>1 </TD><TD> ECE166 + 100μL of cells (2 different plates)
</TD></TR><TR><TD>2</TD><TD> ECE153 + 100μL of cells (2 different plates)
</TD></TR><TR><TD>3 </TD><TD> ECE166 (from glycerol stock)
</TD></TR><TR><TD>4 </TD><TD> Control (from a plate of IA771 without antibiotic)
</TD></TR><TR><TD>5 </TD><TD> IA771 + ECE112 (05/08)
</TD></TR><TR><TD>6 </TD><TD> IA771 + ECE166 (05/08)
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Amylase_production_screening">Amylase production screening</SPAN></H2><UL><LI> Prepare agar plate with starch
</LI></UL><P>- Starch : 1g of starch in 10mL of SDW
</P><P>- Add 2mL of Starch solution into 200mL of agar, mix
</P><P>-Inoculate on different plates : 1A1 +ECE112 (plate from 05/08), IA751 + ECE112 (plate from 05/08) and IA771 + ECE153
</P><H2><SPAN class="mw-headline" id="New_stocks_2">New stocks</SPAN></H2><P>- Take 1mL from LB stock oh ECE171 (from 07/08) and put it into 99mL of LB
</P><H1><SPAN class="mw-headline" id="August_11th">August 11th</SPAN></H1><H2><SPAN class="mw-headline" id="Erytromycin_Experiment">Erytromycin Experiment</SPAN></H2><P>- New preparation of Ery : 0.05g into 10mL of ETHANOL
</P><P>- We plated again our plates (same than 08/08/08)
</P><H2><SPAN class="mw-headline" id="Amylase_screening_experiment">Amylase screening experiment</SPAN></H2><P>- New method : add 2g of starch powder into 200mL of Agar, shake, pipette to plate (and avoid bubbles)
</P><P>- Same plates than 08/08/08
</P><H2><SPAN class="mw-headline" id="Test_our_stock_of_ECE171">Test our stock of ECE171</SPAN></H2><P>-Plasmid miniprep 9from the samples of 10mL and the one of 100mL)
</P><P>-Nanodrop both DNA
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Vetor </TH><TH> 260/280 </TH><TH> μg/mL
</TH></TR><TR><TD> ECE171 (10mL) </TD><TD> 1.73 </TD><TD> 136.3
</TD></TR><TR><TD> ECE171(100mL) </TD><TD> 1.82 </TD><TD> 99.5
</TD></TR></TBODY></TABLE><P>- Double digest (EcoRI and PstI) with 10μL of DNA and 1h of incubation
</P><P>- Samples have been frozen, they should be run onto a gel
</P><H2><SPAN class="mw-headline" id="Control_of_Spc_resistance_of_Bacillus">Control of Spc resistance of Bacillus</SPAN></H2><P>- Spc50 plate with transformed  IA771 (ECE153, which is Spc resistant) and with IA771 9which should not be Spc resistant)
</P><H1><SPAN class="mw-headline" id="August_12th">August 12th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_from_yesterday_2">Results from yesterday</SPAN></H2><UL><LI> Ery plates
</LI></UL><P>- IA751 (control) : most of colonies did not grow, which is good, but 2 or 3 grew...
</P><P>- IA7771 + ECE166 : all colonies grew : ok 9non integration vector)
</P><P>- IA771 + ECE153 : growth of all colonies... problem!!! if IA771 is transformed with this integration vector, it should lose its EryR
</P><P>- IA771 + ECE112 : growth of all colonies... problem!!! if IA771 is transformed with this integration vector, it should lose its EryR
</P><UL><LI> Spc resistance
</LI></UL><P>- IA771 : no growth on Spc50 plate, so no natural resistance
</P><P>- transformed cells grew
</P><UL><LI> Amylase plates
</LI></UL><P>- Add iodine solution for 1min
</P><P>- Problem : no aparent blue, no diffusion of iodine into LA agar + a lot of contamination
</P><H2><SPAN class="mw-headline" id="New_plates_and_LB_stocks">New plates and LB stocks</SPAN></H2><P>- In order to try to make plasmid miniprep at the end of the day, grow IA771 transformed with ECE166 in 10mL LB + Cm5
</P><P>7hours is not enough to reach exponential phase!!! We will try again tomorrow by incubating longer
</P><P>- Grow IA771 and IA751 ( first plate from sterile disks of strain) imto 10mL of LB
</P><P>- Grow ECE166, 171, 153 in E.coli with approriate antibiotics 9to transform tomorrow)
</P><P>- Grow IA771 transformed with ECE153 into LB+Spc50 (to check fluorescence with xylose induction)
</P><H2><SPAN class="mw-headline" id="New_test_for_amylase">New test for amylase</SPAN></H2><P>We tried 2 different methods.
</P><P>- Dilute 1g of starch into 100mL of agar and try to dilute it and boil it to sterilize. The problem is that it is very difficult to dilute...
</P><P>- The second method seem to be better. We diluted 1g of starch into 100mL of Soft Agar (it dilutes very well). Then plate blank agar plate (LA) and then add a thin layer of SA ith 1% of starch. Poke the plates.
</P><P>- We plated IAI+ECE112, IA751+ECE112 and also IA751 and 1A1 for control.
</P><H1><SPAN class="mw-headline" id="August_13th">August 13th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_for_Starch_plates">Results for Starch plates</SPAN></H2><P>- Add 5mL of iodine
</P><P>- 1A1 or IA751 : big zones of clearance
</P><P>- IA751 + ECE112 : no zones of clearance (photos), just small white points on colonies : the gene AmyE seems to be knocked out, transformation ok!
</P><P>- 1A1 + ECE112 : problem, soft agar melted... impossible to observe!
</P><H2><SPAN class="mw-headline" id="Transformation_of_IA751">Transformation of IA751</SPAN></H2><P>- We used IA751 plate from 12/08/08, vectors ECE 153, 166, 171
</P><P>- Spectrophotometer : blank made with medium A
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Time (min) </TH><TH> 0 </TH><TH> 20 </TH><TH> 40 </TH><TH> 60 </TH><TH> 80 </TH><TH> 100 </TH><TH> 120
</TH></TR><TR><TD> OD650 </TD><TD> 0.1487 </TD><TD> 0.1541 </TD><TD> 0.1642 </TD><TD> 0.1980 </TD><TD> 0.2470 </TD><TD> 0.3205 </TD><TD> 0.3643
</TD></TR></TBODY></TABLE><P>- t0 = 120min
</P><P>- Follow the protocol of transformation
</P><P>- Plasmid miniprep of ECE153, 166, 171 (to make the transformation)
</P><P>- Nanodrop ( to add 0.5μg of DNA for transformation)
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Vetor </TH><TH> 260/280 </TH><TH> μg/mL </TH><TH> quantity of DNA to add for transformation (μL)
</TH></TR><TR><TD> ECE153 </TD><TD> 1.5 </TD><TD> 18.4 </TD><TD> 27.2
</TD></TR><TR><TD> ECE166 </TD><TD> 1.83 </TD><TD> 39.5 </TD><TD> 12.7
</TD></TR><TR><TD> ECE171 </TD><TD> 1.83 </TD><TD> 115.8 </TD><TD> 4.4
</TD></TR></TBODY></TABLE><P>- Plate (with appropriate antibiotics)
</P><UL><LI> ECE166 : Cm5
</LI><LI> ECE153 : Spc50
</LI><LI> ECE171 : Kan5 (to prepare agar plate, 40μL of Kan 25mg/mL into 200mL of agar)
</LI></UL><H2><SPAN class="mw-headline" id="Check_our_stock_of_ECE171">Check our stock of ECE171</SPAN></H2><P>- We made big stocks of ECE171, before keeping it, we wanted to check it, so we run double digest on a gel!!
</P><P>- Results : problem!!
</P><P>- 2 possible causes : Double digest was made &quot; days before and kept in the freezer (possible degradation) + too much DNA on the gel
</P><P>- New double digest (from pellets in the freezer) + new gel
</P><P>- Results : ok!
</P><H2><SPAN class="mw-headline" id="Glycerol_stocks">Glycerol stocks</SPAN></H2><P>- for IA751 and IA771, add 100μL of culture and 500μL of glycerol (60%)
</P><H2><SPAN class="mw-headline" id="Xylose_experiment">Xylose experiment</SPAN></H2><P>- To test the transformation of ECE153, we want to induce the promoter Pxyl, and we will have green fluorescence
</P><P>- Add 1mL of culture IA751 + ECE153 (from yesterday), 8mL of LB and 45μL of Spc50, 1mL of xylose (1g into 10mL)
</P><H2><SPAN class="mw-headline" id="Plasmid_miniprep_for_B.S.">Plasmid miniprep for B.S.</SPAN></H2><P>- We first tried to use the same protocol than for E.coli (Zyppy kit) for ECE166 transformed in IA751
</P><P>- Nanodrop to see the result
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Vetor </TH><TH> 260/280 </TH><TH> μg/mL
</TH></TR><TR><TD> ECE166, colony 1 </TD><TD> 1.69 </TD><TD> 11.2
</TD></TR><TR><TD> 1.41</TD><TD> 14.5
</TD></TR><TR><TD> ECE166, colony 2</TD><TD> 1.34 </TD><TD> 7.0
</TD></TR></TBODY></TABLE><P>- We have very low concentration of DNA, we will digest that plasmid tomorrow morning to see if it is ECE166, and if it does work, we will try to modify our protocole tomorrow
</P><H2><SPAN class="mw-headline" id="Transformation_of_ECE188.2C_189.2C_190_in_E.coli">Transformation of ECE188, 189, 190 in E.coli</SPAN></H2><UL><LI> We received this vector in DNA stocks, so we have to transform them to test them 9because we do not have enough DNA)
</LI><LI> Use TOP10 competent cells
</LI><LI> Pellet, add 100μL of CaCl2 solution, for each transformation, use 50μL of cells
</LI><LI> Add 2μL of DNA, and 1μL of PUC19 for control
</LI><LI> 30min on ice
</LI><LI> 2min at 42°C, 2min on ice
</LI><LI> 2h at 37°C
</LI><LI> Plate on Amp100 plates
</LI></UL><H2><SPAN class="mw-headline" id="In_Preparation_of_Beta-galactosidase_Assay_.28Promoter_Assay_PA.29">In Preparation of Beta-galactosidase Assay (Promoter Assay PA)</SPAN></H2><P>Biobrick Extraction:
</P><P>E0040
</P><UL><LI> GFP (Amp resistance)
</LI><LI> Extracted from 2007 Plate 1 Well 5H
</LI></UL><P>I13522
</P><UL><LI> E.coli constitutive promoter with RBS and GFP (Amp resistance)
</LI><LI> Extracted from 2007 Plate 3 Well 13C
</LI></UL><P>B0034
</P><UL><LI> E. coli RBS (Amp resistance)
</LI><LI> Extracted from 2008 Plate 1000 Well 2E
</LI></UL><P>R0040
</P><UL><LI> E. coli promoter (Amp resistance)
</LI><LI> Extracted from 2008 Plate 1000 Well 4C
</LI></UL><P>Transformation of competent TOP10 with the 4 biobricks above along with pUC19 control
</P><UL><LI> 20μL TOP10 used for each transformation with 2μL DNA
</LI><LI> Followed standard protocol
</LI><LI> Plated out neat and 1/10 on Amp100 plates after 2 hours of incubation, then plates are kept at 37°C overnight
</LI></UL><H1><SPAN class="mw-headline" id="August_14th">August 14th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_of_transformation"> Results of transformation</SPAN></H2><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Vector </TH><TH> Antibiotic </TH><TH> number of colonies </TH><TH> Transformation efficiency
</TH></TR><TR><TD> ECE166 </TD><TD> Cm5 </TD><TD> 31 </TD><TD> 62
</TD></TR><TR><TD> Control </TD><TD> Cm5 </TD><TD> 0 </TD></TR><TR><TD> ECE171 </TD><TD> Kan5 </TD><TD> 5 + a lot of small colonies </TD><TD> ?
</TD></TR><TR><TD> Control </TD><TD> Kan5 </TD><TD> 1 + 5 very small </TD></TR><TR><TD> ECE153 </TD><TD> Spc50 </TD><TD> about 25 (on a side) </TD><TD> 50
</TD></TR><TR><TD> Control </TD><TD> Spc50 </TD><TD> 6 + 2 very small  </TD></TR></TBODY></TABLE><P>- Problem with our control, maybe we should check the resistance of competent cells (before transformation)
</P><H2><SPAN class="mw-headline" id="BioBrick_extraction_for_testing_promoters_and_RBS_in_B.subtillis">BioBrick extraction for testing promoters and RBS in <I>B.subtillis</I></SPAN></H2><P>The primers for inducible <I>B.subtillis</I> promoters have been ordered. Meanwhile, we would like to be able to compare the RBS and promoter strengths in <I>E.coli</I> and <I>B.subtillis</I>, using GFP fluorescence to quantify gene expression.
</P><P>We attempted to isolate 4 BioBricks: I13522 (GFP under constitutive promoter), E0040 (GFP only), R0040 (a promoter), &amp; B0034 (an <I>E.coli</I> RBS).
So far, only the former two, extracted from 2007 wells, grew after transformation. Single-colony PCR was used to test the transformants.
</P><P>Expected VF2-VR fragment sizes:
</P><P>I13522 -  ~2.4 kb
E0040 -  958 bp
R0040 -  292 bp
B0034 -  250 bp
</P><H2><SPAN class="mw-headline" id="Transformation_of_vectors_188.2C_189.2C_190">Transformation of vectors 188, 189, 190</SPAN></H2><P>- Transformation of yesterday did not work! there was nothing on plates, even on the control plate
</P><P>- Try again, same protocol, 2μL of DNA, control : PUC9
</P><P>- Plate on Amp100 for control and Amp100 + Cm5 for vectors
</P><H2><SPAN class="mw-headline" id="Double_digest_of_ECE166_.28extracted_from_transformed_Bacillus_yesterday.29">Double digest of ECE166 (extracted from transformed Bacillus yesterday)</SPAN></H2><P>- Transformation from 07/08
</P><P>- 16μL of DNA, 2μL of Buffer EcoRI, 1μL of EcoRI (Biolab) and 1μL of HindIII
</P><P>- Incubator at 37°C for 35min, heat shock at 80°C for 5min
</P><P>- Run on a gel with 4μL of DNA
</P><UL><LI> Result : no bands! not enough DNA!
</LI></UL><P>- Run on a new gel with 16μL of DNA
</P><UL><LI> Result : &quot; bands, one of about 2000b, and one of about 5000b&gt; The big one seem to be too small...
</LI></UL><P>We are going to check this transformation with fluorescence too!
</P><H2><SPAN class="mw-headline" id="Erythromycin_plate">Erythromycin plate</SPAN></H2><P>- Erythromycin plate for the transformation 13/08/08
</P><P>- IA771 (control, they should survive)
</P><P>- IA771 + ECE166 (non integration vector, so colonies should survive)
</P><P>- IA771 + ECE171 (integration vector but in another locus, should survive)
</P><P>- IA771 + 153 9integration vector, they should die if they are transformed)
</P><H2><SPAN class="mw-headline" id="New_stocks_3">New stocks</SPAN></H2><P>- LB stocks with antibiotic of ECE151, 153, 166
</P><P>- LB stock of ECE153 with Spc50 (one from colonies from LB stock from 11/08 and one from 13/08 transformation plates)
</P><H2><SPAN class="mw-headline" id="Check_fluorescence_with_microscope">Check fluorescence with microscope</SPAN></H2><UL><LI> ECE166 from the plate from 13/08
</LI></UL><P>We diluted one colony from this plate into SDW and observed with microscope.
</P><P>Result : Bacillus is fluorescent! There are some bacteria which are brighter than others, but it is because it is not an integration vector. This transformation worked!!!
</P><UL><LI> ECE153
</LI></UL><P>We added xylose, but we did not incubate our sample after that, so it did not work&gt; We will try again tomorrow with a period of incubation&gt;</P><P>We will have to 
</P><H2><SPAN class="mw-headline" id="Plates_from_yesterday_for_Biobrick_Extraction_.28PA.29">Plates from yesterday for Biobrick Extraction (PA)</SPAN></H2><UL><LI> Growth observed on Amp100 plates for I13522 and E0040 only but not for R0040, B0034 and pUC9 control
</LI><LI> Re-plated overnight culture neat on Amp 75 plates and incubate at 37°C overnight
</LI></UL><H4><SPAN class="mw-headline" id="Single_Colony_PCR_of_I13522_and_E0040"> Single Colony PCR of I13522 and E0040</SPAN></H4><UL><LI> 5 colonies picked from each neat agar plate onto Amp75 plates
</LI><LI> PCR following standard protocols
</LI><LI> Run PCR product on 1.2% agarose gel at 70V which is later soaked in EtBr
</LI></UL><P>First Row:
</P><UL><LI> Lane 2 - Hyperladder IV
</LI><LI> Lane 3-7 - I13522 Colonies 1-5 PCR
</LI><LI> Lane 9-13 - E0040 Colonies 1-5 PCR
</LI></UL><P>Second Row:
</P><UL><LI> Lane 2 - Hyperladder IV
</LI><LI> Lane 3-6, 9 - R0010 Colonies 1-5 PCR
</LI></UL><P>Result:
</P><UL><LI> Expected size of band: I13522 - 2375bp; E0040 - 958bp; R0010 - 292bp
</LI><LI> Picked colonies 4 and 5 for E0040 into 10ml LB with Amp100 and incubate overnight at 37°C as the band corresponds to about 958bp
</LI><LI> Band of R0010 colonies 3 and 5 correspond to 292bp and will incubate in LB tomorrow when colonies have grown on agar plates
</LI></UL><H1><SPAN class="mw-headline" id="August_15th">August 15th</SPAN></H1><H2><SPAN class="mw-headline" id="Transformation_ECE188.2C_189.2C_190_into_E.coli">Transformation ECE188, 189, 190 into E.coli</SPAN></H2><P>- Results : only our control plate grew!!
</P><P>- Maybe we did not add enough DNA, or there is another problem. We will e mail the guy from Bacillus center before trying again, because we don't have enough DNA.
</P><H2><SPAN class="mw-headline" id="Plasmid_miniprep_ECE_151_.28x2.29.2C_ECE_153.2C_ECE_166">Plasmid miniprep ECE 151 (x2), ECE 153, ECE 166</SPAN></H2><UL><LI><B>Nanodrop results</B> :
</LI></UL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> 260/280 </TH><TH> ng/μL
</TH></TR><TR><TD> ECE 151 (3) </TD><TD> 1.79 </TD><TD> 86.8
</TD></TR><TR><TD> ECE 151 </TD><TD> 1.88 </TD><TD> 106.3
</TD></TR><TR><TD> ECE 153 </TD><TD> 1.80 </TD><TD> 32.2
</TD></TR><TR><TD> ECE 166 </TD><TD> 1.87 </TD><TD> 36.2
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Xylose_Induction_with_ECE_153">Xylose Induction with ECE 153</SPAN></H2><P>- 4 Tubes
</P><P>- No fluorescence! We have to think about the transformation of integration vectors into Bacillus.
</P><H2><SPAN class="mw-headline" id="Glycerol_Stock_Transformation">Glycerol Stock Transformation</SPAN></H2><P>- Spin down cells (competent) from 5/8 and 13/8 stocks
</P><P>- Remove liquid, resuspend in Medium B
</P><P>- Incubate for 60 mins at 37°C
</P><P>- Add ECE 166 into tubes
</P><P>- Incubate for 30 mins at 37°C
</P><P>- Plate out on CM 5 plates 
</P><P>- 4 plates 
</P><UL><LI> 771 (5/8) + ECE 166
</LI></UL><UL><LI> 771 (5/8) Control
</LI></UL><UL><LI> 771 (13/8) + ECE 166
</LI></UL><UL><LI> 771 (13/8) Control
</LI></UL><H2><SPAN class="mw-headline" id="Transforming_ECE_188.2C_189.2C_190_.283rd_try.29">Transforming ECE 188, 189, 190 (3rd try)</SPAN></H2><P>- 2 tubes of chemically competent top 10
</P><P>- Spin down, remove liquid
</P><P>- Resuspend cells in 100μL of CaCl2 50mM
</P><P>- 4 tubes with 50μL of cells
</P><UL><LI> ECE 190 (all)
</LI></UL><UL><LI> ECE 189 (all)
</LI></UL><UL><LI> ECE 188 (all)
</LI></UL><UL><LI> PUC 9 (5μL) [ Control ]
</LI></UL><P>- Ice for 30 mins
</P><P>- 42°C for 2 mins
</P><P>- Ice for 2 mins
</P><P>- Incubate at 37°C for 2 hours
</P><P>- Plate out on Amp 100 (Neat)
</P><H2><SPAN class="mw-headline" id="Biobrick_E0040_and_I13522_.28PA.29">Biobrick E0040 and I13522 (PA)</SPAN></H2><UL><LI> PCR E0040 colonies 4 and 5 after miniprep of plasmid to ensure plasmid identity again
</LI><LI> Protocol:
</LI><LI>SDW 7.5μL
</LI><LI>MasterMix 10μL
</LI><LI>VR and VF2 1x2μL
</LI><LI>DNA sample 0.5μL
</LI></UL><LI> Run on 1.2% SyBr E-gel
</LI><LI> Lane 2 - 100bp ladder
</LI><LI> Lane 3 - E0040 colony 4
</LI><LI> Lane 4 - E0040 colony 5
</LI><LI> Result: Bands at around 1000bp on both lanes 3 and 4 with a brighter band for lane 3.  Correct band as E0040 VF-VR is 958bp
</LI><LI> Minprep of plasmid E0040 from colony 4 is kept
</LI><UL><LI> Transform chemically compotent TOP10 with I13522 biobrick extract again
</LI><LI> Transformation following standard protocols
</LI><LI> TOP10 with I13522 plated neat and 1/10 on Amp. 100 plates and incubate at 37°C overnight
</LI><LI> Will pick single colonies tomorrow for PCR analysis and into LB with Amp100
</LI></UL><H1><SPAN class="mw-headline" id="August_16th">August 16th</SPAN></H1><H2><SPAN class="mw-headline" id="Result_from_the_days_before">Result from the days before</SPAN></H2><UL><LI>For Ery plates
</LI></UL><P>- Everything grew, we will try again to see if there is a problem with the antibiotic (and we will do a negative control with the strain IA751)
</P><UL><LI> Transformation of E.coli with ECE188, 189, 190
</LI></UL><P>- We have a result from our 3 transformation in E.coli. To transform E.coli with these vectors we have to wait for 2 days before seeing cells!! We will check the vectors next week.
</P><UL><LI> Transformation of B.S. IA771 (glycerol stocks)
</LI></UL><P>- Nothing! Problem with the storage of competent IA771???
</P><H1><SPAN class="mw-headline" id="August_18th">August 18th</SPAN></H1><H2><SPAN class="mw-headline" id="Test_Ery_efficiency">Test Ery efficiency</SPAN></H2><P>Since all Ery plates grew, we want to check the antibiotic stock.
</P><P>- New Ery plate of IA751 (they should die)
</P><P>- New LB stock (Ery5) with IA751
</P><H2><SPAN class="mw-headline" id="Single_colony_plates">Single colony plates</SPAN></H2><P>- New plates (Cm5 + Amp100) with single colony of ECE188 (from 14/08), ECE189 (from 13/08) and ECE190 (from 14/08)
</P><P>- Grow the same single colony into LB with antibiotic
</P><H2><SPAN class="mw-headline" id="Transformation_of_glycerol_stocks_of_B.S.">Transformation of glycerol stocks of B.S.</SPAN></H2><P>The result from the last transformation of glycerol stocks was negative&gt; We tried again with two different protocols. 
</P><UL><LI><B>First protocol</B></LI></UL><P>- Spin the glycerol stock, throw out the supernatant (just keep about 100μL), add 0.5μL of Medium B
</P><P>- Incubate about 1h15
</P><P>- Add 0.6μg of ECE166 (because we managed to transform BS with this vector)
</P><P>- Incubate 1h30
</P><P>- Plate on Cm5 plates (DNA less control and transformed cells)
</P><UL><LI><B>Second protocol</B></LI></UL><P>- Spin the glycerol stock, throw out the supernatant (just keep about 100μL), add 0.5μL of Medium B
</P><P>- Add 0.6μg of ECE166 (because we managed to transform BS with this vector)
</P><P>- Incubate 30min
</P><P>- Plate on Cm5 plates (transformed cells)
</P><P>
- Plate cells after 10min of incubation (without DNA) on blank plates (to see if they are alive)
</P><H2><SPAN class="mw-headline" id="PA_with_I13522.2C_E0040_and_B0034_Preparation"> PA with I13522, E0040 and B0034 Preparation</SPAN></H2><UL><LI> R0040 TOP10 miniprep from LB culture done by Kevin
</LI><LI> 5 single colonies of I13522 picked onto Amp100 plate and LB with Amp100 done on Sunday
</LI><LI> Miniprep of the 5 colonies with sinigle colony PCR
</LI><LI> PCR followed standard protocol and product is run on 0.8% EtBr E-gel
</LI><LI> iGEM34 porgramme is used for PCR
</LI></UL><P>Gel:
</P><UL><LI> 4μL dye with 20μL PCR product
</LI><LI> 5μL Hyperladder I with 15μL SDW into lane 2
</LI><LI> Lanes 3-7: I13522 Colonies 1-5
</LI><LI> Lane 8: R0040
</LI><LI> Lane 9: Hyperladder I
</LI></UL><P>Result:
</P><UL><LI> Size of I13522 is not right as the bands are all around 4500bp but it should be 2375bp
</LI><LI> R0040 is of the correct size
</LI></UL><UL><LI> Transformed chemically compotent TOP10 with B0034 biobrick extract and pUC9 control
</LI><LI> Plate on Amp100 plates after standard transformation
</LI></UL><H1><SPAN class="mw-headline" id="August_19th">August 19th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_from_the_transformation_with_glycerol_stocks">Results from the transformation with glycerol stocks</SPAN></H2><P>We used two different glycerol stocks, one from 07/08 and one from 13/08. We also had 2 different protocols (cf 18/08 page).
</P><P>We tested our stocks on blank plates, cells are alive, the problem is to know if they are still competent or not.
</P><UL><LI><B>Protocol 1 (with incubation time before adding DNA)</B></LI></UL><P>- Stock from 07/08 : nothing on the control and nothing on the transformation plate.
</P><P>- Stock from 13/08 : nothing on the control and nothing on the transformation plate.
</P><UL><LI><B>Protocol 2 (without incubation time before adding DNA)</B></LI></UL><P>- Stock from 07/08 : nothing on the control and nothing on the transformation plate.
</P><P>- Stock from 13/08 : nothing on the control, and 1 colony on the transformed plate. We checked the fluorescence of bacteria into this colony (vector ECE166 with promoter and GFP). We have fluorescence, so our transformation is ok.
</P><P>The result of this confirm that our stock of competent cells in glycerol is still competnet since we managed to transform one colony. The problem is certainly not the storage, but our protocol when we use frozen competent cells&gt; We need to improve the efficiency of this protocol. 
</P><H2><SPAN class="mw-headline" id="Test_our_Erythromycin_stock">Test our Erythromycin stock</SPAN></H2><P>- We put some colonies on a Ery 0.5 plate. They grow, but in a very small amount, so our antibiotic seems to be fine. 
</P><P>- Cells in LB with antibiotic : no growth!
</P><P>So our antibiotic is ok, either we did not transform our cells and that is why everything grew on Ery plates, either there is a problem in our protocol for Ery test.
</P><H2><SPAN class="mw-headline" id="Plasmid_miniprep_of_ECE153.2C_ECE166_and_ECE171">Plasmid miniprep of ECE153, ECE166 and ECE171</SPAN></H2><P>- Plasmid miniprep LB stocks from yesterday
</P><P>- Nanodrop
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> 260/280 </TH><TH> ng/μL
</TH></TR><TR><TD> ECE 153 </TD><TD> 1.63 </TD><TD> 32.6
</TD></TR><TR><TD> ECE 166 (1) </TD><TD> 1.73 </TD><TD> 56.2
</TD></TR><TR><TD> ECE 166 (2) </TD><TD> 1.67 </TD><TD> 57.7
</TD></TR><TR><TD> ECE 171 </TD><TD> 1.81 </TD><TD> 185.5
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Transformation_of_IA771">Transformation of IA771</SPAN></H2><P>We used IA751 plate from 12/08/08, vectors ECE 153, 166, 171.
</P><P>Spectrophotometer : blank made with medium A
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Time (min) </TH><TH> 0 </TH><TH> 20 </TH><TH> 40 </TH><TH> 80
</TH></TR><TR><TD> OD650 </TD><TD> 0.1512 </TD><TD> 0.1498 </TD><TD> 0.1476 </TD><TD> 0.1535
</TD></TR></TBODY></TABLE><P>There was a problem with our colonies. At t=80min, we inoculated again our tube of Medium A.
</P><P>- We used IA751 plate from 12/08/08, vectors ECE 153, 166, 171
</P><P>- Spectrophotometer : blank made with medium A
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Time (min) </TH><TH> 0 </TH><TH> 20 </TH><TH> 40 </TH><TH> 60 </TH><TH> 80 </TH><TH> 100
</TH></TR><TR><TD> OD650 </TD><TD> 0.2393 </TD><TD> 0.2504 </TD><TD> 0.2663 </TD><TD> 0.2807 </TD><TD> 0.3184 </TD><TD> 0.3672
</TD></TR></TBODY></TABLE><P>- Continue the protocol, add DNA (ECE153, 166, 171)
</P><P>- Plate on antibiotic plates
</P><UL><LI> New glycerol stocks
</LI></UL><P>- 8 tubes : spin cells, throw away the supernatant, add 500μL of glycerol 60%
</P><P>- 8 tubes : spin cells, keep only 100μL of medium B, resuspend cells and add 500μL of glycerol 60%
</P><P>- Put them in the freezer at -80°C
</P><H2><SPAN class="mw-headline" id="Dealing_with_R0040.2C_R0062.2C_S0168.2C_I13522_for_PA">Dealing with R0040, R0062, S0168, I13522 for PA</SPAN></H2><UL><LI> Plated R0040, R0062, S0168 received from MIT onto Amp100 plates and incubate at 37°C
</LI><LI> Pciked 4 single colonies for each after colonies have grown and incubate overnight
</LI></UL><P>Check I13522 biobrick DNA
</P><UL><LI> PCR biobrick DNA from 2007 wells
</LI><LI> Run 0.8% EtBr E-gel
</LI><LI> Lane 5: I13522
</LI><LI> Lane 7: Hyperladder I
</LI></UL><P>Result:
</P><UL><LI> Size of band is too small under 1000bp should be 2375bp instead
</LI><LI> Transform TOP10 again but with I13522 from 2008 wells
</LI></UL><H1><SPAN class="mw-headline" id="August_20th">August 20th</SPAN></H1><H2><SPAN class="mw-headline" id="Check_vector_ECE190">Check vector ECE190</SPAN></H2><P>- Plasmid miniprep
</P><P>- Nanodrop
</P><P>- Double digest : 6μL of SDW, 8μL of DNA, 1μL of buffer, 1μL of XbaI and 1μL of PstI
</P><P>- Run on a gel
</P><P>- <B>Result</B> : 2 bands (lane3, a little bit more than 3000b and a little bit more than 5000b) :ok!
</P><H1><SPAN class="mw-headline" id="August_21st">August 21st</SPAN></H1><H2><SPAN class="mw-headline" id="Check_ECE188">Check ECE188</SPAN></H2><P>- Plasmid miniprep
</P><P>- Double digest with XbaI and PstI
</P><P>- Gel (lane6)
</P><P>- Result : nothing! not enough DNA
</P><H1><SPAN class="mw-headline" id="August_22nd">August 22nd</SPAN></H1><H2><SPAN class="mw-headline" id="Stock_ECE112.2C_166.2C_153">Stock ECE112, 166, 153</SPAN></H2><P>- Plasmid miniprep (2 tubes of ECE112, 2 tubes of ECE166 and 1 tube of ECE153)
</P><P>- Nanodrop
</P><H2><SPAN class="mw-headline" id="Transformation_of_Bacillus_2">Transformation of Bacillus</SPAN></H2><UL><LI> For strain IA751 : transform with ECE166, 171, 153 and 112
</LI></UL><UL><LI> For strain IA771 : transform with ECE166, 171
</LI></UL><UL><LI> for glycerol stocks (one with 100μL of medium B and glycerol, and one with only glycerol) : transform with ECE166
</LI></UL><P>
- We used the same protocol than before with a few changes :
</P><UL><LI> add sterile 50% glucose in medium A
</LI><LI> don't keep samples after using them for spectrophotometer
</LI><LI> spin each tubes and keep only 100μL of medium A (to concentrate cells) before adding DNA
</LI></UL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Time (min) </TH><TH> 0 </TH><TH> 20 </TH><TH> 40 </TH><TH> 60 </TH><TH> 80 </TH><TH> 100 </TH><TH> 120 </TH><TH> 130 </TH><TH> 140
</TH></TR><TR><TD> OD650 for IA751 </TD><TD> 0.1817 </TD><TD> 0.1870 </TD><TD> 0.1929 </TD><TD> 0.2115 </TD><TD> 0.2543 </TD><TD> 0.3291 </TD><TD> 0.3695 </TD><TD> 0.4188 </TD><TD>0.4224
</TD></TR><TR><TD> OD650 for IA751 </TD><TD>0.1818 </TD><TD> 0.1874 </TD><TD> 0.1907 </TD><TD> 0.2088 </TD><TD> 0.2452 </TD><TD> 0.3182 </TD><TD> 0.3575 </TD><TD> 0.4228 </TD><TD> 0.4314
</TD></TR></TBODY></TABLE><P>- For glycerol stocks, resuspend cells into 100μL of medium B, add DNA and wait for 30min, end plate them
</P><H1><SPAN class="mw-headline" id="August_23rd">August 23rd</SPAN></H1><H2><SPAN class="mw-headline" id="Results_of_transformation_from_23.2F08.2F2008">Results of transformation from 23/08/2008</SPAN></H2><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Vector </TH><TH> Strain </TH><TH> Antibiotic </TH><TH> number of colonies </TH><TH> Conclusion
</TH></TR><TR><TD> ECE166 </TD><TD> IA771, glycerol stock with medium B </TD><TD> Cm5 </TD><TD> one large colony and a lot of very small colonies </TD><TD> Contamination?
</TD></TR><TR><TD> Control </TD><TD> IA771, glycerol stock with medium B </TD><TD> Cm5 </TD><TD> a lot of very small colonies </TD><TD> Contamination?
</TD></TR><TR><TD> ECE166 </TD><TD> IA771, glycerol stock without medium B </TD><TD> Cm5 </TD><TD> many very small colonies </TD><TD> Contamination?
</TD></TR><TR><TD> Control </TD><TD> IA771, glycerol stock without medium B </TD><TD> Cm5 </TD><TD> a few tiny colonies </TD><TD> Contamination?
</TD></TR><TR><TD> ECE166 </TD><TD> IA751, fresh </TD><TD> Cm5 </TD><TD> 9 large colonies, no small ones </TD><TD> to test
</TD></TR><TR><TD> Control </TD><TD> IA751, fresh </TD><TD> Cm5 </TD><TD> no growth, scratches </TD><TD> ok
</TD></TR><TR><TD> ECE166 </TD><TD> IA771, fresh </TD><TD> Cm5 </TD><TD> 1 large colony, many small worm-like growth, scratches? </TD><TD> to test
</TD></TR><TR><TD> Control </TD><TD> IA771, fresh </TD><TD> Cm5 </TD><TD> no colonies at all </TD><TD> ok
</TD></TR><TR><TD> ECE171 </TD><TD> IA771, fresh </TD><TD> Kan5 </TD><TD> no colonies </TD><TD> ??
</TD></TR><TR><TD> Control </TD><TD> IA771, fresh </TD><TD> Cm5 </TD><TD> no colonies  </TD><TD> ok
</TD></TR><TR><TD> ECE171 </TD><TD> IA751, fresh </TD><TD> Kan5 </TD><TD> about 30 large colonies and 100 small colonies </TD><TD> to test?
</TD></TR><TR><TD> Control </TD><TD> IA751, fresh </TD><TD> Cm5 </TD><TD> 3 large colonies, about 20 small to medium colonies  </TD><TD> contamination?
</TD></TR><TR><TD> ECE112 </TD><TD> IA751, fresh </TD><TD> Cm5 + Spc100 </TD><TD> no growth </TD><TD> ??
</TD></TR><TR><TD> Control </TD><TD> IA751, fresh </TD><TD> Cm5 + Spc100 </TD><TD> no growth </TD></TR><TR><TD> ECE153 </TD><TD> IA751, fresh </TD><TD> Spc50 </TD><TD> about 500 medium to large colonies </TD><TD> to test
</TD></TR><TR><TD> Control </TD><TD> IA751, fresh </TD><TD> Spc50 </TD><TD> no growth </TD><TD> ok
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="August_26th">August 26th</SPAN></H1><H2><SPAN class="mw-headline" id="Xylose_test">Xylose test</SPAN></H2><UL><LI> for ECE153
</LI></UL><P>- Prepare 4 tubes with 10mL LB and Spc50, inoculate each tube with a colony from IA751 + ECE153
</P><H2><SPAN class="mw-headline" id="Starch_plate">Starch plate</SPAN></H2><P>- prepare some blank agar plates
</P><P>- add 1g of starch into 100mL of soft agar, mix, boil
</P><P>- put a thin layer of soft agar with starch
</P><P>- plate with IA751, IA771, IA751+ECE153 and IA751 (control plate, to see if we mixed some plates)
</P><P>- Incubate at 37°C
</P><H2><SPAN class="mw-headline" id="Ery_plate">Ery plate</SPAN></H2><P>- Plate IA751, IA771, IA751 (control plate) and IA751+ECE153
</P><H1><SPAN class="mw-headline" id="August_27th">August 27th</SPAN></H1><H2><SPAN class="mw-headline" id="Xylose_test_2">Xylose test</SPAN></H2><UL><LI> Preparation of xylose tube
</LI></UL><P>- Add 1mL of culture from yesterday, 1mL of xylose (0.1g/mL) and 8mL of LB and 45μL of Spc (do that for the 4 different colonies)
</P><P>- Incubate 5hours
</P><P>- Observation with microscope : some green task, but they do not really match with bacillus cells, not conclusive!
</P><H2><SPAN class="mw-headline" id="Starch_plate_2">Starch plate</SPAN></H2><P>- Add 5mL of iodine solution on starch plate, wait for 1min, throw the liquid
</P><P>- Result
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Strain </TH><TH> added vector </TH><TH> Observation </TH><TH> Conclusion
</TH></TR><TR><TD> IA771 </TD><TD> none </TD><TD> no clearance zones </TD><TD> ok
</TD></TR><TR><TD> IA751 </TD><TD> none </TD><TD> big clearance zones </TD><TD> ok
</TD></TR><TR><TD> IA751 </TD><TD> ECE153 </TD><TD> no clearance zones </TD><TD> trnsformation worked!
</TD></TR><TR><TD> IA751 or IA771 </TD><TD> none or ECE171 </TD><TD> big clearance zones </TD><TD> we have some colonues on control plate with IA751, contamination? or antibiotic resistance?
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Ery_test">Ery test</SPAN></H2><P>- IA771 alive and IA751 dead, confirmation that we did not mix plates (same conclusion than with starch plate, last line)
</P><H2><SPAN class="mw-headline" id="Test_glycerol_stocks_of_competent_bacillus">Test glycerol stocks of competent bacillus</SPAN></H2><P>- Pellet bacillus cells, add 100μL of prewarmed mediumB, add 0.6μg of DNA, incubate 30min at 37°C by shaking, plate
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Strain </TH><TH> added vector </TH><TH> Antibiotic
</TH></TR><TR><TD> IA771 </TD><TD> none </TD><TD> blank plate (to check they are alive)
</TD></TR><TR><TD> IA771 </TD><TD> none </TD><TD> Cm5 (negative control)
</TD></TR><TR><TD> IA771 </TD><TD> ECE166 </TD><TD> Cm5
</TD></TR><TR><TD> IA751 </TD><TD> none </TD><TD> blank plate (to check they are alive)
</TD></TR><TR><TD> IA751 </TD><TD> none </TD><TD> Cm5 (negative control)
</TD></TR><TR><TD> IA751 </TD><TD> none </TD><TD> Spc50 (negative control)
</TD></TR><TR><TD> IA751 </TD><TD> ECE112 </TD><TD> Cm5
</TD></TR><TR><TD> IA751 </TD><TD> ECE153 </TD><TD> Spc50
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="August_28th">August 28th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_from_Bacillus_transformation_with_glycerol_stocks">Results from Bacillus transformation with glycerol stocks</SPAN></H2><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>  Strain </TH><TH> added vector </TH><TH> Antibiotic </TH><TH> number of colonies </TH><TH> Conclusion
</TH></TR><TR><TD> IA771 </TD><TD> none </TD><TD> blank plate </TD><TD> confluent colonies </TD><TD> cells are alive
</TD></TR><TR><TD> IA771 </TD><TD> none </TD><TD> Cm5 </TD><TD> 0 </TD><TD> no contamination
</TD></TR><TR><TD> IA771 </TD><TD> ECE166 </TD><TD> Cm5 </TD><TD> 2 </TD><TD> transformation ok, very low efficiency
</TD></TR><TR><TD> IA751 </TD><TD> none </TD><TD> blank plate </TD><TD> confluent colonies </TD><TD> cells are alive
</TD></TR><TR><TD> IA751 </TD><TD> none </TD><TD> Cm5 </TD><TD> 0 </TD><TD> no contamination
</TD></TR><TR><TD> IA751 </TD><TD> none </TD><TD> Spc50 </TD><TD> 0 </TD><TD> no contamination
</TD></TR><TR><TD> IA751 </TD><TD> ECE112 </TD><TD> Cm5 </TD><TD> 0 </TD><TD> problem, we can not manage to transform this vector
</TD></TR><TR><TD> IA751 </TD><TD> ECE153 </TD><TD> Spc50 </TD><TD> 156 </TD><TD> transformation ok, good efficiency
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Test_AmyE_insertion">Test AmyE insertion</SPAN></H2><UL><LI> Colonies PCR for Bacillus (IA751 transformed with ECE153, IA751, and IA771)
</LI></UL><P>- 20μL of lysozyme, a few colonies for each
</P><P>- Cycle : 15min qt 37°C, 15min at 99°C, 1min at 4°C, 1min at 99°C and 1min at 4°C
</P><P>- for each PCR : 12μL of cells, 1μL of cells, 1μL of primer1, 1μL of primer2, 5μL of master mix
</P><P>- PCR (iGEM program)
</P><P>- run a gel
</P><UL><LI> Result : nothing on the gel!!! But we can not manage to PCR anything... there is a problem in our PCR protocol, or products!
</LI></UL><H1><SPAN class="mw-headline" id="August_29th">August 29th</SPAN></H1><H2><SPAN class="mw-headline" id="Stock_of_ECE112">Stock of ECE112</SPAN></H2><P>- Bulk up ECE112 (Amp100) rfom 10mL LB (28/08), take 1mL and add to 100mL of LB in big flask
</P><P>- Grow overnight in 37°C incubator with vigorous shaking
</P><P>- Aliquot in 10mL tubes, pellet and freeze cell pellets
</P><H2><SPAN class="mw-headline" id="Transformation_of_glycerol_stock">Transformation of glycerol stock</SPAN></H2><UL><LI> Strian : IA751
</LI></UL><P>- Transfer into Eppendorf tubes and spin down for 30min
</P><P>-Remove glycerol and add medium B
</P><P>- Add ECE112 and ECE153, and one control tube
</P><P>- incubate for 30min in 37°C
</P><P>- Plate out ECE112 (Cm5), ECE153 (Spc50), DNA less (blank, Cm5, Spc50)
</P><H1><SPAN class="mw-headline" id="September_1st">September 1st</SPAN></H1><H2><SPAN class="mw-headline" id="Result_for_Bacillus_transformation_.2827.2F08.29">Result for Bacillus transformation (27/08)</SPAN></H2><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH>   Strain </TH><TH> Vector </TH><TH> Antibiotic </TH><TH> number of colonies </TH><TH> Conclusion
</TH></TR><TR><TD> IA751 </TD><TD> None </TD><TD> Spc50 </TD><TD> 0 </TD><TD> ok
</TD></TR><TR><TD> IA751 </TD><TD> ECE153 </TD><TD> Spc50 </TD><TD> about 200 </TD><TD> very good efficiency
</TD></TR><TR><TD> IA751 </TD><TD> None </TD><TD> Blank </TD><TD> confluent </TD><TD> cells are alive
</TD></TR><TR><TD> IA751 </TD><TD> None </TD><TD> Cm5 </TD><TD> a lot </TD><TD> problem of contamination (bacillus??? or E.coli???)
</TD></TR><TR><TD> IA751 </TD><TD> ECE112 </TD><TD> Cm5 </TD><TD> a lot </TD><TD> to check
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="New_Bacillus_transformation_.28to_make_glycerol_stocks.29">New Bacillus transformation (to make glycerol stocks)</SPAN></H2><UL><LI>2 attempts with plates from 12/08 and 18/08, no growth... Cells may be dead.
</LI></UL><UL><LI>New plate of IA771 and IA751 (with glycerol stocks)
</LI></UL><H1><SPAN class="mw-headline" id="September_2nd">September 2nd</SPAN></H1><H2><SPAN class="mw-headline" id="Transformation_of_Bacillus_3">Transformation of Bacillus</SPAN></H2><UL><LI> with fresh plates from yesterday, last protocol
</LI></UL><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> Time (min) </TH><TH> 0 </TH><TH> 20 </TH><TH> 40 </TH><TH> 60 </TH><TH> 85 </TH><TH> 105 </TH><TH> 125
</TH></TR><TR><TD> OD650 for IA751 </TD><TD> 0.1192 </TD><TD> 0.1330 </TD><TD> 0.1736 </TD><TD> 0.2238 </TD><TD> 0.3124 </TD><TD> 0.4523 </TD><TD> 0.5113
</TD></TR><TR><TD> OD650 for IA771 </TD><TD>0.1144 </TD><TD> 0.1216 </TD><TD> 0.1593 </TD><TD> 0.2296  </TD><TD> 0.3476 </TD><TD> 0.4302 </TD><TD> 0.5565
</TD></TR></TBODY></TABLE><UL><LI>t0 (IA751) = 100min
</LI><LI>t0 (IA771) = 110min
</LI></UL><UL><LI>Transformation
</LI></UL><P>- IA751 on Cm5 and Spc 50 (control DNA less plates)
</P><P>- IA751 + ECE112 (1μg of DNA) on Cm5
</P><P>- IA751 + ECE153 (0.6μg of DNA) on Scp50
</P><P>- IA771 on Cm5 (control)
</P><P>- IA771 + ECE166 on Cm5
</P><UL><LI> Glycerol stocks (about 20 tubes of each)
</LI></UL><H2><SPAN class="mw-headline" id="Check_ECE112_stock">Check ECE112 stock</SPAN></H2><P>- Double digest with XbaI and EcoRI
</P><P>- Gel : problem only one band (single digest???)
</P><H1><SPAN class="mw-headline" id="September_3rd">September 3rd</SPAN></H1><H2><SPAN class="mw-headline" id="Result_from_bacillus_transformation_with_fresh_competent_cells_.2802.2F09.29">Result from bacillus transformation with fresh competent cells (02/09)</SPAN></H2><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> strain </TH><TH> antibiotic </TH><TH> added vector </TH><TH> observation </TH><TH> conclusion
</TH></TR><TR><TD> IA751 </TD><TD> none </TD><TD> none </TD><TD> confluent colonies </TD><TD> cells are alive
</TD></TR><TR><TD> IA751 </TD><TD> Spc50 </TD><TD> none </TD><TD> no colonies </TD><TD> no contamination
</TD></TR><TR><TD> IA751 </TD><TD> Spc50 </TD><TD> ECE153 </TD><TD> 70 colonies </TD><TD> transformation to check, efficiency : about 115 colonies/μg of DNA
</TD></TR><TR><TD> IA771 </TD><TD> none </TD><TD> none </TD><TD> confluent colonies </TD><TD> cells are alive
</TD></TR><TR><TD> IA771 </TD><TD> Cm5 </TD><TD> none </TD><TD> no colonies </TD><TD> no contamination
</TD></TR><TR><TD> IA771 </TD><TD> Cm5 </TD><TD> ECE166 </TD><TD> 16 colonies </TD><TD> transformation to check, efficiency : about 27 colonies/μg of DNA
</TD></TR><TR><TD> IA771 </TD><TD> Cm5 </TD><TD> ECE112 </TD><TD> 41 small colonies </TD><TD> transformation to check, efficiency : about 41 colonies/μg of DNA
</TD></TR></TBODY></TABLE><P>With 1μg of DNA, we manage to obtain colonies with ECE112. We will check our transformation with starch plates.
</P><H2><SPAN class="mw-headline" id="Check_ECE112_stock_2">Check ECE112 stock</SPAN></H2><P>- Single and double digest with XbaI and EcoRI
</P><P>- Gel
</P><UL><LI> Lane 3 : HyperladderI
</LI><LI> Lane 4 : ECE112, 100mL flask, single digest with XbaI (expected size about 10,000bp)
</LI><LI> Lane 5 : ECE112, 100mL flask, single digest with EcoRI (expected size about 10,000bp)
</LI><LI> Lane 6 : ECE112, 100mL flask, double digest (expected size about 3200bp and 6,900bp)
</LI><LI> Lane 7 : ECE112, 10mL flask, single digest with XbaI (expected size about 10,000bp)
</LI><LI> Lane 8 : ECE112, 10mL flask, single digest with EcoRI (expected size about 10,000bp)
</LI><LI> Lane 9 : ECE112, 10mL flask, double digest (expected size about 3200bp and 6,900bp)
</LI><LI> Lane 10 : HyperladderI
</LI></UL><P>- Results : ok
</P><H2><SPAN class="mw-headline" id="Test_glycerol_stocks_.28from_02.2F09.29_of_competent_bacillus">Test glycerol stocks (from 02/09) of competent bacillus</SPAN></H2><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> strain </TH><TH> antibiotic </TH><TH> added vector </TH><TH> quantity of DNA
</TH></TR><TR><TD> IA751 </TD><TD> none </TD><TD> none </TD><TD> 0
</TD></TR><TR><TD> IA751 </TD><TD> Spc50 </TD><TD> none </TD><TD> 0
</TD></TR><TR><TD> IA751 </TD><TD> Spc50 </TD><TD> ECE153 </TD><TD> 0.6
</TD></TR><TR><TD> IA771 </TD><TD> none </TD><TD> none </TD><TD> 0
</TD></TR><TR><TD> IA771 </TD><TD> Cm5 </TD><TD> none </TD><TD> 0
</TD></TR><TR><TD> IA771 </TD><TD> Cm5 </TD><TD> ECE166 </TD><TD> 0.6
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Starch_plates">Starch plates</SPAN></H2><P>- Use blank plates
</P><P>- Melt 100mL of Soft Agar, and add slowly 1g of starch
</P><P>- Mix, boil to sterilize
</P><P>- Pour on blank plate
</P><P>- Put 4 or 5 colonies on each plate
</P><UL><LI> IA751 from blank plate (positive control)
</LI><LI> IA771 from blank plate (negative control)
</LI><LI> IA751 transformed with ECE112 (Cm5, 29/08)
</LI><LI> IA751 control (Cm5, 29/08)
</LI><LI> IA751 + ECE112 (Cm5, 02/09)
</LI><LI> IA751 + ECE153 (Cm5, 02/09)
</LI><LI> IA751 + ECE153 (Cm5, 29/08)
</LI></UL><H1><SPAN class="mw-headline" id="September_4th">September 4th</SPAN></H1><H2><SPAN class="mw-headline" id="Result_of_Bacillus_transformation_.28glycerol_stock_of_competent_cells.29">Result of Bacillus transformation (glycerol stock of competent cells)</SPAN></H2><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> strain </TH><TH> antibiotic </TH><TH> added vector </TH><TH> observation </TH><TH> conclusion
</TH></TR><TR><TD> IA751 </TD><TD> none </TD><TD> none </TD><TD> confluent colonies </TD><TD> cells alive
</TD></TR><TR><TD> IA751 </TD><TD> Spc50 </TD><TD> none </TD><TD> nothing </TD><TD> no contamination
</TD></TR><TR><TD> IA751 </TD><TD> Spc50 </TD><TD> ECE153 </TD><TD> 15 </TD><TD> ok, efficiency 25 colonies/μg of DNA
</TD></TR><TR><TD> IA771 </TD><TD> none </TD><TD> none </TD><TD> confluent colonies </TD><TD> cells alive
</TD></TR><TR><TD> IA771 </TD><TD> Cm5 </TD><TD> none </TD><TD> nothing </TD><TD> no contamination
</TD></TR><TR><TD> IA771 </TD><TD> Cm5 </TD><TD> ECE166 </TD><TD> 15 </TD><TD> ok, efficiency 5 colonies/μg of DNA
</TD></TR></TBODY></TABLE><P>We have no contamination in our glycerol stocks. However, the efficiency of glycerol stocks seems to be lower than these of fresh competent cells.
</P><H2><SPAN class="mw-headline" id="Result_of_starch_plates">Result of starch plates</SPAN></H2><P>We had 5mL of iodine solution, wait for 1min and observe.
</P><P>Controls (the small zone of clearing are contamination, we have not yet manage to find how to avoid them)
</P><P>Negative control : IA771, no zone of clearing
</P><P>Positive control : IA751, big zones of clearing
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> strain </TH><TH> vector </TH><TH> photo </TH><TH> observation </TH><TH> conclusion
</TH></TR><TR><TD> IA751 </TD><TD> none </TD><TD> big zones of clearing for each colony </TD><TD> ok
</TD></TR><TR><TD> IA771 </TD><TD> none </TD><TD> no zone of clearing for each colony </TD><TD> ok
</TD></TR><TR><TD> IA751 </TD><TD> ECE112/none (29/08) </TD><TD> clearing zones for control, so no trnasformation, it is maybe E.coli, 1 clearing zone out of 3 colonies, it is certainly contamination, but that means that some colonies are transformed </TD><TD> ok
</TD></TR><TR><TD> IA751 </TD><TD> ECE112 (02/09) </TD><TD> 1 zone of clearing out of 4 colonies </TD><TD> some colonies are not transformed and have a resistance to Cm5 (contamination?)
</TD></TR><TR><TD> IA751 </TD><TD> ECE153 (28/08) </TD><TD> no zone of clearing  </TD><TD> 5 transformed colonies
</TD></TR><TR><TD> IA751 </TD><TD> ECE153 (02/09) </TD><TD> 2 zone of clearing out of 5 colonies </TD><TD> some colonies are not transformed and have a resistance to Spc50 (contamination?)
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Colony_PCR_from_bacillus_subtilis_colonies">Colony PCR from bacillus subtilis colonies</SPAN></H2><P>We want to check the size of the insert in the AmyE region(IA751) after transformation with an integration vector (to check if we have a single cross over or not). 
</P><P>We are going to test our transformation of ECE112 and 153.
</P><P>We already tried this protocol, but our PCR did not work, so we tried again, with one sample of chromosomal DNA from bacillus to check if our transformation works. 
</P><P>- add 20μL of 0.05mg/mL lysozyme into a PCR tube
</P><P>- add a few colonies into this tube until you obtain a dense solution. We test :
</P><UL><LI> IA751
</LI><LI> IA751 + ECE153 (29/08, 02/09, 03/09)
</LI><LI> IA751 + ECE112 (02/09)
</LI></UL><P>- 15min in 37°C, then 15min in 99°C, 1min in 4°C, 1min in 99°C, 1min in 4°C
</P><P>- For IA751, IA751 + ECE112, IA751 + ECE153, and  bacillus chromosomal DNA, add 12μL of SDW, 5μL of MM, 1+1μL of primers (amyE primers), 1μL of the solution prepared befor with cells (or DNA)
</P><P>- PCR
</P><P>- Gel
</P><UL><LI> Lane1 : hyperladderI
</LI><LI> Lane2 : IA751 + ECE153 (29/08)
</LI><LI> Lane3 : IA751 + ECE153 (02/09)
</LI><LI> Lane4 : IA751 + ECE153 (03/09)
</LI><LI> Lane5 : IA751 + ECE112 (02/09)
</LI><LI> Lane6 : IA751
</LI><LI> Lane7 : Chromosomal DNA
</LI></UL><P>- Result : only chromosomal DNA worked. That means that our primers are ok, but we have to find a protocol for chromosomal DNA miniprep of bacillus to be able to PCR something into the genome of bacillus.
</P><UL><LI>New attempt without the lyse process (only protocol of single colony PCR, like for E.coli)
</LI></UL><P>- Gel
</P><UL><LI> Lane1 : hyperladderI
</LI><LI> Lane2 : IA751 + ECE153 (29/08)
</LI><LI> Lane3 : IA751 + ECE153 (02/09)
</LI><LI> Lane4 : IA751 + ECE153 (03/09)
</LI><LI> Lane5 : IA751 + ECE112 (02/09)
</LI><LI> Lane6 : IA751
</LI><LI> Lane7 : Chromosomal DNA
</LI></UL><P>- Result : Lane4, we have something but too small.
</P><H1><SPAN class="mw-headline" id="September_5th">September 5th</SPAN></H1><H2><SPAN class="mw-headline" id="Starch_plates_2">Starch plates</SPAN></H2><P>Prepare some starch plates. We want to test our transformation but we can not manage to PCR Bacillus. So we are going to try to extract chromosomal DNA from Bacillus. We make these starch plates to test it with colonies which have positive control with amylase test.
</P><P>- Prepare starch plates (you can find the protocol <A href="/IGEM:Cambridge/2008/Turing_Pattern_Formation/Experiments/Bacillus_subtilis_transfomation#ECE153" title="IGEM:Cambridge/2008/Turing Pattern Formation/Experiments/Bacillus subtilis transfomation">here</A>)
</P><P>- Plate 4 different plates :
</P><UL><LI> 2 colonies of IA75 (positive control) and 3 colonies of IA771 (negative control) 
</LI><LI> 5 colonies of IA751 transformed with ECE153 from 29/08
</LI><LI> 3 colonies of IA751 transformed with ECE153 from 02/09 and 3 colonies from 03/09
</LI><LI> 5 colonies of IA751 transformed with ECE112 from 02/09
</LI></UL><H1><SPAN class="mw-headline" id="September_6th">September 6th</SPAN></H1><H2><SPAN class="mw-headline" id="Results_of_starch_plates">Results of starch plates</SPAN></H2><P>We had 5mL of iodine solution, wait for 1min and observe.
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> strain </TH><TH> vector </TH><TH> photo </TH><TH> observation </TH><TH> conclusion
</TH></TR><TR><TD> IA751 </TD><TD> none </TD><TD><DIV class="center"><DIV class="floatnone"><A href="/File:Photo06/09controls.gif" title="File:Photo06/09controls.gif">100px</A></DIV></DIV></TD><TD> big zones of clearing for IA751 (right), no zone of clearing for IA771 (left) </TD><TD> IA751 : positive control and IA771 : negative control
</TD></TR><TR><TD> IA751 </TD><TD> ECE153 from 02/09 (on left) </TD><TD><DIV class="center"><DIV class="floatnone"><A href="/File:Photo06/091530203.gif" title="File:Photo06/091530203.gif">100px</A></DIV></DIV></TD><TD> colony 1 : zone of clearing, no transformation; colony 2 and 3 : no zone of clearing, transformation ok</TD><TD> 2 transformed colonies out of 3
</TD></TR><TR><TD> IA751 </TD><TD> ECE153 from 03/09 (on right) </TD><TD><DIV class="center"><DIV class="floatnone"><A href="/File:Photo06/091530203.gif" title="File:Photo06/091530203.gif">100px</A></DIV></DIV></TD><TD> colony 1,2 and 3 : no zone of clearing, transformation ok</TD><TD> transformation ok
</TD></TR><TR><TD> IA751 </TD><TD> ECE153 (28/08) </TD><TD><DIV class="center"><DIV class="floatnone"><A href="/File:Photo06/0915328.gif" title="File:Photo06/0915328.gif">100px</A></DIV></DIV></TD><TD> no zone of clearing for colonies 1, 2 and 3, impossible to say for colonies 4 and 5 because big contamination (so big zone of clearing) </TD><TD> colony 1,2,3 : ok, no result for the others
</TD></TR><TR><TD> IA751 </TD><TD> ECE112 (02/09) </TD><TD><DIV class="center"><DIV class="floatnone"><A href="/File:Photo06/0911202.gif" title="File:Photo06/0911202.gif">100px</A></DIV></DIV></TD><TD> no zones of clearing </TD><TD> transformation ok
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="September_8th">September 8th</SPAN></H1><H2><SPAN class="mw-headline" id="Prepare_chromosomal_DNA_extraction_from_Bacillus">Prepare chromosomal DNA extraction from Bacillus</SPAN></H2><P>- Grow 2 tubes into LB for
</P><UL><LI> IA751
</LI><LI> IA751 + ECE153 from 03/09 (for tissue kit)
</LI><LI> IA751 + ECE153 from 02/09
</LI><LI> IA751 + ECE112 from 05/09
</LI></UL><H2><SPAN class="mw-headline" id="PCR_AmyE_from_ECE112">PCR AmyE from ECE112</SPAN></H2><P>- Add 12μL of SDW, 5 μL of MM, 1μL + 1μL of each primers (for AmyE back and AmyE front) and 1μL of ECE112
</P><UL><LI> Lane1 : HyperladderI
</LI><LI> Lane4 : AmyE front
</LI><LI> Lane5 : AmyE front
</LI><LI> Lane6 : AmyE back
</LI><LI> Lane7 : AmyE back
</LI></UL><UL><LI> Results : good band for AmyE front, very low band for AmyE back
</LI></UL><H1><SPAN class="mw-headline" id="September_9th">September 9th</SPAN></H1><H2><SPAN class="mw-headline" id="Chromosomal_DNA_extraction">Chromosomal DNA extraction</SPAN></H2><P>- ZR soil microbe DNA kit for
</P><UL><LI>IA751
</LI><LI>IA751+ECE112
</LI><LI>IA751+ECE153
</LI></UL><P>- Nanodrop
</P><TABLE class="wikitable" style="text-align:center" border="1"><TBODY><TR><TH> product (chromosomal DNA) </TH><TH> concentration (ng/μL)
</TH></TR><TR><TD> IA751 </TD><TD> 75
</TD></TR><TR><TD> IA751 + ECE112</TD><TD> 48.6
</TD></TR><TR><TD> IA751 +ECE153 </TD><TD> 74.2
</TD></TR></TBODY></TABLE><P>
- PCR with AmyE detect primers (12μL of SDW, 5μL of MM, 1+1μL of each primer, 1μL of chromosomal DNA or plasmid)
</P><P>- Gel
</P><UL><LI> Lane1 : HyperladderI
</LI><LI> Lane2 : AmyE Back
</LI><LI> Lane3 : AmyE Front
</LI><LI> Lane4 : Bacillus chromosomal DNA (given by Duncan to check)
</LI><LI> Lane5 : ECE112 (plasmid)
</LI><LI> Lane6 : ECE153 (plasmid)
</LI><LI> Lane7 : chromosomal DNA IA751 + ECE112
</LI><LI> Lane8 : chromosomal DNA IA751 + ECE153
</LI><LI> Lane9 : chromosomal DNA IA751
</LI></UL><UL><LI> Results
</LI></UL><P>- Only something for IA751, and ECE153 (pure plasmid)???
</P><P>- New PCR with 4μL of chromosomal DNA and new PCR program
</P><P>- New gel
</P><UL><LI> Lane1 : IA751 (chromosomal DNA)
</LI><LI> Lane2 : ECE153 (chromosomal DNA)
</LI><LI> Lane3 : ECE112 (chromosomal DNA)
</LI><LI> Lane4 : ECE112 (plasmid)
</LI><LI> Lane5 : ECE153 (plasmid)
</LI><LI> Lane6 : Bacillus chromosomal DNA (given by Duncan)
</LI><LI> Lane7 : HyperladderI
</LI><LI> Lane8 : agrA
</LI><LI> Lane9 : agrB
</LI></UL><UL><LI>Result
</LI></UL><P>- No insert in IA751 transformed with ECE153 and ECE112 (same size than control!) and however the amylase test is positive for this colony....
</P><H2><SPAN class="mw-headline" id="PCR_AmyE_again">PCR AmyE again</SPAN></H2><P>- 24μL of SDW, 10μL of MM, 2μL of each primers, 2μL of ECE112
</P><UL><LI> Results (cf gel, lane 2 and 3)
</LI></UL><P>- Very good result for amyE front, nothing for AmyE back
</P><H1><SPAN class="mw-headline" id="September_10th">September 10th</SPAN></H1><H2><SPAN class="mw-headline" id="Check_PCR_for_pSBINT1_vector">Check PCR for pSBINT1 vector</SPAN></H2><P>- Gel
</P><UL><LI>Lane2 : HyperladderI
</LI><LI>Lane3 : rep+bla
</LI><LI>Lane4 : PCR of I714062
</LI><LI>Lane5 : PCD of J04630
</LI></UL><UL><LI> Result
</LI></UL><P>- One band for rep, but not very clear, we should try again
</P><P>- for the other parts, we have a good band but the wrong size, these biobricks seem to be bad, we will order them from the MIT
</P><H1><SPAN class="mw-headline" id="September_11th">September 11th</SPAN></H1><H2><SPAN class="mw-headline" id="PCR_for_pBSINT1_biobrick_vector">PCR for pBSINT1 biobrick vector</SPAN></H2><UL><LI> PCR rep+bla part from I732007 (pSB1A3 backbone)
</LI></UL><P>- Add 12μL of SDW, 5μL of MM, 1μL of rep fwd primer, 1μL of rep rev primer and 1μL of I732007 (program iGEM new)
</P><UL><LI> PCR I714062 and J04630 with VF and VR. We want to extract the cutting sites of the biobrick vector with GFP inside (in the future we would prefer to make our biobrick vector with a RSB for B.S. inside)
</LI></UL><P>- Add 9μL of SDW, 5μL of MM, 1μL of VF, 1μL VR and 4μL of each biobrick (program iGEM new)
</P><DIV style="color: white; font: 2px;">x</DIV></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Cambridge/Bacillus/Lab_Work">http://2008.igem.org/Team:Cambridge/Bacillus/Lab_Work</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Bacillus/Lab_Work" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Bacillus/Lab_Work" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Bacillus/Lab_Work&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Bacillus/Lab_Work&amp;oldid=104118" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>