<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Cambridge/Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Modelling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="bodytable"><DIV class="menubar"><CENTER><A href="/Team:Cambridge" title="Team:Cambridge"><FONT face="verdana" style="color:#ffffff"><B>Home</B></FONT></A> <A href="/Team:Cambridge/Voltage" title="Team:Cambridge/Voltage"><FONT face="verdana" style="color:#b3c6fe"><B>Voltage</B></FONT></A> <A href="/Team:Cambridge/Signalling" title="Team:Cambridge/Signalling"><FONT face="verdana" style="color:#b3c6fe"><B>Signalling</B></FONT></A> <A href="/Team:Cambridge/Bacillus" title="Team:Cambridge/Bacillus"><FONT face="verdana" style="color:#b3c6fe"><B>Bacillus</B></FONT></A> <STRONG class="selflink"><FONT face="verdana" style="color:#b3c6fe"><B>Modelling</B></FONT></STRONG> <A href="/Team:Cambridge/Protocols" title="Team:Cambridge/Protocols"><FONT face="verdana" style="color:#ffffff"><B>Protocols</B></FONT></A> <A href="/Team:Cambridge/People" title="Team:Cambridge/People"><FONT face="verdana" style="color:#ffffff"><B>People</B></FONT></A> </CENTER></DIV><DIV class="submenu"><CENTER><STRONG class="selflink"><FONT face="verdana" style="color:#444444"><B>Introduction</B></FONT></STRONG> <A href="/Team:Cambridge/Modelling#Modelling_Bacterial_Quorum_Sensing_Systems_-_Parameters" title="Team:Cambridge/Modelling"><FONT face="verdana" style="color:#444444"><B>Parameters</B></FONT></A> <A href="/Team:Cambridge/Modelling#The_agr-Quorum_Sensing_System_of_S.aureus" title="Team:Cambridge/Modelling"><FONT face="verdana" style="color:#444444"><B>Components</B></FONT></A> <A href="/Team:Cambridge/Modelling#Modelling_the_complete_agr-quorum_sensing_system" title="Team:Cambridge/Modelling"><FONT face="verdana" style="color:#444444"><B>Complete System</B></FONT></A> <A href="/Team:Cambridge/Modelling#Pattern_formation" title="Team:Cambridge/Modelling"><FONT face="verdana" style="color:#444444"><B>Pattern Formation</B></FONT></A> <A href="/Team:Cambridge/Modelling#Further_Questions" title="Team:Cambridge/Modelling"><FONT face="verdana" style="color:#444444"><B>Further Questions</B></FONT></A> </CENTER></DIV><TABLE style="background:#444444; padding:15px;"><TABLE align="left" width="border=&quot;0&quot;" style="background:#444444; padding:5px;"><TBODY><TR><TD style="width:100%; height: 400; padding-left: 15px;"><DIV class="contentf"><DIV style="height: 400; background: white; line-height:170% padding: 5px;"><DIV style="color: white; font: 2px;">x</DIV><B>We are interested in using peptide-based quorum-sensing systems from gram-positive bacteria to create self-organising biological systems. We would like to demonstrate how to engineer multicellular biological systems that are capable of expressing spatial patterns of gene expression, such as with GFP, on a bacterial lawn. This would be a first step in the direction of engineering multicellular behaviour that will become immensely useful when thinking about further applications: Instead of using GFP in our patterns, we could use genes that increase cellular electric conductivity to arrive at self-assembling systems responding to electric input and output.</B><P><B>In the following article, we would like to introduce a model of the agr-QS of <I>S.aureus</I> to illustrate how a typical QS system works. Furthermore, the model will predict theoretical values for biological parameters (such as the threshold cell density) that will be verified and we will be also be able to predict how the system behaves if we change a number crucial parameters. This can be extremely useful in informing design decisions when building a synthetic device. Finally, we would like to use the model in a hypothetical setup with a second parallel agr-system to investigate how to engineer a biological patterning systems.</B></P><P><I>A .pdf download of this article can be found by clicking here: <A href="https://static.igem.org/mediawiki/2008/f/fa/Cambridge_2008_Modelling.pdf" class="external autonumber" rel="nofollow">[1]</A></I></P><H1><SPAN class="mw-headline" id="Modelling_Bacterial_Quorum_Sensing_Systems_-_Parameters">Modelling Bacterial Quorum Sensing Systems - Parameters</SPAN></H1><P>Modelling biological systems involves finding relations between a great number of variables and parameters. Usually, these systems cannot be solved in an analytical way as might be accomplished with simpler systems. The problem is compounded by the fact that we only have limited knowledge of the specific parameters involved (production rates, decay rates, etc.). In this article, we shall make a number of assumptions:
</P><P>1) we work with enzyme concentrations [A] etc. and assume enzymes are well-mixed
</P><P>2) we use standard enzyme kinetics to infer relations between enzyme concentrations
</P><P>3) and we scale rate parameters relative to a fixed value.
</P><P>To expand on the last point, we note that all parameters we will be working with denote rates and thus have the same dimensions. Therefore, we can work with scalar values only. Secondly, we can scale these parameters w.r.t one of them: We shall fix the maximum rate of promoter expression at 1 and determine the other parameters in relation to this rate.
</P><P>For example, basal expression rates must be significantly lower than the maximum rate of promoter expression. In fact, in the agr-QS system of <I>Staphylococcus aureus</I> that we are going to model the p2 promoter seems to exhibit a significant increase<SUP><A href="https://2008.igem.org/Team:Cambridge/Modeling#References|1" class="external autonumber" rel="nofollow">[2]</A></SUP> upon activation in wild-types. We shall take basal rates of 0.05, implying a 20-fold increase of promoter activity translating into even a greater increase in AIP production (which is 50-fold according to Gustaffson<SUP><A href="https://2008.igem.org/Team:Cambridge/Modeling#References|2" class="external autonumber" rel="nofollow">[3]</A></SUP> who has also modelled part of the agr-system). Furthermore, we assume that all enzymes decay at the same rate of 0.2. 
</P><P>In addition, phosphorylation and enzyme complex formation rates must be significantly higher than protein expression rates. We shall take phosphorylation rates at 50 (and dephosphorylation rates at 5) and enzyme complex formation rates at 20 (deformation rates at 2). The important thing to remember is that these rates are much higher than the previous ones and will thus allow us to use quasi-steady state assumptions in our model. An example of this method can be found here<SUP><A href="https://2008.igem.org/Team:Cambridge/Modeling#References|3" class="external autonumber" rel="nofollow">[4]</A></SUP>. To some degree, the quasi-steady state assumption allows us to ignore actual parameter values. This comes in quite handy when modelling biological systems.
</P><P>To summarise, the parameter values we shall be using are:
</P><P>maximum promoter expression rate - 1,
</P><P>basal expression rate - β=0.02,
</P><P>enzyme decay rate - δ=0.2, 
</P><P>enzyme complex formation and deformation rates - c<SUB>+</SUB>=20 and c<SUB>-</SUB>=2, 
</P><P>enzyme phosphorylation and dephosphorylation rates - p<SUB>+</SUB>=50 and p<SUB>-</SUB>=5.
</P><H1><SPAN class="mw-headline" id="The_agr-Quorum_Sensing_System_of_S.aureus">The agr-Quorum Sensing System of <I>S.aureus</I></SPAN></H1><P>
The agr-QS system is a quorum-sensing system in <I>S. aureus</I> and is typical for Gram-positive bacteria. It relies on the signalling peptide AIP (auto-inducing peptide) that is produced by the cell and activates the quorum-sensing system. The QS response is quite tightly regulated as expression occurs only once the local bacterial density passes a fixed threshold value, similar unlike a switch. In a high-density environment the bacteria will exhibit novel behaviour such as virulence or sporulation (both biologically costly but critical and advantageous).
</P><H2><SPAN class="mw-headline" id="Synthesis_Equation">Synthesis Equation</SPAN></H2><P>Let us at first look at protein synthesis. The RNA Polymerase binds to the promoter and transcribes mRNA, [m], at a specific rate (assuming Michaelis-Menten kinetics, which we can deduce from simple chemical kinetics theory) depending on the promoter strength and the inducer concentration, [A]. It is then translated at a rate depending on the strength of the ribosome binding site and folds into enzyme [E]. mRNA decay rates are much greater than enzyme decay rates so we can assume that [m] is in a quasi-steady state. The rate equations (after scaling) can be simplified into a single equation (3) as follows:
</P><P>and further scaling of [E] will leave us with the rate equation,
</P><P>where p=1 following previous assumptions of maximal promoter expression. By assuming quasi-steady state behaviour, we have gotten rid of the mRNA dynamics. We shall use this synthesis equation in the following sections.
</P><H2><SPAN class="mw-headline" id="Recasting_the_agr-receiver_part_as_a_biobrick">Recasting the agr-receiver part as a biobrick</SPAN></H2><P>We would like to have the receiver part of the agr-QS system in form of a biobrick. It becomes useful to think about inputs and outputs to and from such a biobrick. We shall use PoPS as a universal signal to &quot;connect&quot; biobricks to each other. A constitutive promoter biobrick for instance has no inputs but only PoPS output. Our agr-receiver device can be thought of as follows:
</P><P>In designing the biobrick we use two principles introduced in a recent paper and well-known in iGEM circles<SUP><A href="https://2008.igem.org/Team:Cambridge/Modeling#References|4" class="external autonumber" rel="nofollow">[5]</A></SUP>: Standardisation and characterisation. Here, our biobrick part is modular in the sense that the &quot;interface&quot; between biobricks consists of PoPS in- and output. In essence, this demonstrates a higher-level analogue to the biobrick restriction site standard that is used in assembling the biobrick parts. We will also look at the output, or transfer function, of the biobrick. 
</P><H2><SPAN class="mw-headline" id="Modelling_the_agr-quorum_sensing_response.2Freceiver">Modelling the agr-quorum sensing response/receiver</SPAN></H2><P>To model and test the agr-receiver, we need to put a p2 promoter (and rbs) upstream of the device we just introduced. Let [P] denote the concentration of AIP, [A] denote the concentration of AgrA and similarly for the other proteins, [CP] denote the obvious enzyme complex and [A<SUP>+</SUP>] be AgrA phosphorylated.
</P><P>We also need to consider concentrations in extra- and intracellular space carefully. AIP exists (or is only a chemical player) in the extracellular environment whereas cell enzymes (such as AgrA and AgrC) exist only within a cell. Assuming that the local density (i.e. the volume fraction) of cells is ρ then 1-ρ will be occupied by AIP with a certain concentration. We assume that concentrations have values relative to extra- or intracellular space i.e. if [A] = 0.2 when ρ=0.1 then doubling intracelullar space to ρ=0.2 while holding the total amount of AgrA constant, will result in [A]=0.1.
</P><P>The equations of this model read: 
</P><P>Assuming that concentration of [CP] is in a quasi-steady state we arrive at [CP] = c<SUB>+</SUB>/(c<SUB>-</SUB> + δ} [C] [P] and can substitute it back into the system to simplify it into a 4-component system:
</P><P>Since A and C are surface enzymes, we can also assume that their levels change much slower than the levels AIP or A<SUP>+</SUP> implying that A and C are in a quasi-steady state. We would then end up with a 2-component system in P and A<SUP>+</SUP> and could easily determine the dynamics of this system (and show that the systems has two stable steady states (low- and high-density behaviour) and an unstable steady state (the 'switch' density). In the following sections we shall, however, only use minimal assumptions and use the 4-component system in modelling only.
</P><H2><SPAN class="mw-headline" id="Characterising_the_agr-receiver_device">Characterising the agr-receiver device</SPAN></H2><P>Let us now look at how the receiver behaves under a number of conditions. In all cases, we assume that [P] is held constant (last rate equation would thus fall away).
</P><P>- In the following, we can observe PoPS output at a fixed point in time while varying AIP levels. This is also called a transfer function (between input and output). Incidentally, PoPS output is identical to PoPS input in this case (both are using the same promoter). The step-like shape of the curve agrees with previously published results<SUP><A href="https://2008.igem.org/Team:Cambridge/Modeling#References|1" class="external autonumber" rel="nofollow">[6]</A></SUP>. 
</P><P>- We can also look at a surface plot of transfer functions, adding an additional time dimension. In this case, we can also clearly see static dynamic performance - PoPS output plotted against time while holding AIP levels constant. This is important in determining the timing of the system output, which can be crucial when using this part in more complex settings.
</P><P>Throughout the plots we observe step-like behaviour w.r.t AIP concentrations - this is typical for quorum-sensing systems that react sensitively to local cell densities. Our model seems to work. It is interesting to note that even when we change phosphorylation and complex formation rates to different scalar values (we still require them to be large), the transfer function will only change negligibly. No qualitative change is observed.
</P><H2><SPAN class="mw-headline" id="Modelling_the_agr-sender_device">Modelling the agr-sender device</SPAN></H2><P>To complete the agr-system, let us look at the sender that works as follows: AgrB and AgrD lie downstream of a promoter, AgrB is a membrane protein that forms a complex with AgrD and produces AIP.  We assume the reaction is B + D &lt;=&gt; BD -&gt; B + P. A corresponding biobrick would consist of agrB and agrD with a terminator and would take PoPS input, but would not generate PoPS ouput since the biobrick would only change AIP levels. Again, we assume quasi-steady state behaviour for BD. The model reads:
</P><P>where p<SUB>AIP</SUB> is the production rate of AIP (we take it to be 1). After substituting BD into the rate equations, we end up with a three-component system again. In the wild-type system, AIP synthesis lies downstream of the p2 promoter (1st figure). There is again a term involving ρ to account for extra- and intracellular medium.
</P><H1><SPAN class="mw-headline" id="Modelling_the_complete_agr-quorum_sensing_system">Modelling the complete agr-quorum sensing system</SPAN></H1><P>Let us now consider the complete agr-QS system as illustrated in the 1st figure. Clearly, this will be described by solving the equations for the agr-receiver and -sender simultaneously (where PoPS in the sender part will now be the p2 synthesis term from before). The system of equations to be solved will consist of the rate equations of [A]-[D] and [P]. For the rate equation involving [P] we need to add both contributions from the sender and receiver systems and only take a single decay and basal expression term so that the resulting equation reads: 
</P><P>In the lab one can &quot;simply&quot; (Biologists clearly agree with this statement) put the sender and receiver device downstream of a p2 promoter to recreate the agr-QS system with biobricks. For completeness, the entire model reads:
</P><P>and we substitute in the appropriate terms that we arrived at using quasi-steady state assumptions. Remember we took δ=0.05. We solve these equations and plot PoPS output w.r.t. the volumefraction of intracellular space in the following figure. In the case of <I>S. aureus</I>, PoPS output drives a virulence gene. 
</P><P>While the curve is surprisingly robust when changing initial conditions and most parameters, it is extremely sensitive when varying basal rates of expression (assuming maximum expression is fixed at 1). The shape of the curve and particularly the threshold density at which the system switches behaviour, is therefore largely dependent on the basal rate of expression. The QS-system switches from low- into high-density behaviour at a local density of roughly 0.005 per unit volume.
</P><P>While we were not able to find data that would verify this estimate, we can consider the gram-negative <I>E.coli</I>. In ideal conditions, <I>E.coli</I> grows to roughly (2-5) x 10<SUP>9</SUP> cells/ml in LB until it reaches stationary phase. Assuming that a single <I>E.coli</I> cell has a volume on the order of 1μm<SUP>3</SUP>, the cell density at stationary phase will be on the order of 10<SUP>-3</SUP> per unit volume. Assuming that growth in ideal conditions implies higher resulting cell densities, this number will be an upper limit to cell densities, and the <I>E.coli</I>-QS system must be activated at roughly those densities (or slightly below). This validates that our guess of 5x10<SUP>-3</SUP> for the threshold density in agr-systems is in the correct order of magnitude.
</P><P>Finally, it is interesting to note that we have arrived at this result by purely theoretical considerations. We have seen that the basal rate of expression plays a critical role in the system response. This in turn will be useful in informing possible design choices for engineered applications: To change the threshold density one can simply increase basal rates of expression, e.g. by constitutively expressing the agr-system in parallel to the agr-genes that are regulated by p2. As an example, if one was to increase basal expression rates (i.e. β) the threshold value would decrease. This makes it extremely interesting to engineered applications in Synthetic Biology.
</P><H1><SPAN class="mw-headline" id="Pattern_formation">Pattern formation</SPAN></H1><P>Is it possible to engineer bacteria that exhibit multicellular behaviour and can express spatial patterns (e.g. on a bacterial lawn)? 
</P><P>We would like to use AIP quorum-sensing systems to demonstrate how this might be accomplished. AIP-based quorum-sensing systems have the advantage that there exist numerous derivatives in gram-positive bacteria that are mutually exclusive. This stands in constrast with OHL-based systems (in gram-negative bacteria) that exhibit a high occurance of interference between OHL derivatives.
</P><P>Before we try to model such behaviour, let us look at a simple mathematical model for pattern formation.
</P><H2><SPAN class="mw-headline" id="Reaction-diffusion_systems">Reaction-diffusion systems</SPAN></H2><P>The theory of patterns is vast and it would (and does) extend beyond the scope of this project. However, we will look at a very simple mathematical system that is capable of generating patterns. This might give us some intuition in how patterns are formed even though in our case there might be no direct application. 
</P><P><B>Turing Patterns</B> Turing's idea was to look at systems in the absence of diffusion. He required such a system to have a stable steady-state and upon addition of diffusion, required this to become unstable. A simple example is the Schnakenberg system that is analysed in Mathematical Biology by J. Murray. 
</P><P>This is a simple system that is capable of creating diffusion-driven patterns. The mathematical analysis goes as follows: One considers the system in the absence of diffusion and looks for a stable steady-state. Next, one considers the same system with the diffusion term added and substitutes in a trial solution that is composed of a Fourier series while requiring the steady-state to be unstable. This will result in a number of further conditions (inequalities) and it is easy to find the (Turing) space of parameters that will satisfy all conditions for pattern generation. A gallery of patterns produced by Schnakenberg system can be found on the next page. 
</P><P>We can see that the type of patterns created is highly dependent on our parameter values, particularly α and β. Can we arrive at similiar patterns with our AIP-QS system?
</P><H2><SPAN class="mw-headline" id="Pattern_formation_in_Synthetic_Biology">Pattern formation in Synthetic Biology</SPAN></H2><P>How can we enginner bacteria to express spatial patterns? (It is interesting to note that many bacteria do form patterns, particularly when exposed to different nutrient levels) We would like to express patterns on a bacterial lawn with constant bacterial density everywhere and we can think of patterns as spatial distributions of PoPS output (downstream of which we can put GFP) or cell differentiation. Since we have modelled the agr-system and know that non-interfering AIP-based systems exist, we would like to construct a pattern generating system as follows: Both the AIP-A and AIP-B systems inhibit production of the &quot;competitor&quot; molecule. Furthermore, both systems are self-activating. This would be a more complex analogue to the simple reaction-diffusion system introduced previously. 
</P><P>In the following part, we assume that nutrients are abundant (we do not model nutrient consumption) and that the bacterial density throughout the lawn is constant and high by setting ρ=0.01. Note that we are disregarding the fact that these bacteria now live on a 2d surface. We would arrive at a more detailed estimate for bacterial densities in a bacterial lawn if we introduced a diffusion term to account for signalling molecules diffusing across the nutrient layer. 
</P><P>Furthermore, the synthesis terms in this model are bounded in our system. Meinhardt mentioned<SUP><A href="https://2008.igem.org/Team:Cambridge/Modeling#References|5" class="external autonumber" rel="nofollow">[7]</A></SUP> stripe-like patterns can be generated by such systems where activator production is bounded. While he was analysing a simple two-component system, we would like to generate patterns by using two agr-systems in parallel, both self-activating and inhibiting the parallel system. 
</P><P>The rate equations for one of the parallel agr-systems (with peptide P<SUB>1</SUB>) read as follows:
</P><P>where f(x)=40/(0.1+x). The equations for the the second agr-derivative system with peptide P<SUB>2</SUB> read analogously with P<SUB>1</SUB> and P<SUB>2</SUB> interchanged (and independent variables for intracellular enzymes). Note that we have added a diffusion term to account for spatial diffusion across the plane.
</P><P>Solving those equations with random initial conditions in levels of P<SUB>i</SUB> gives us:
</P><P>Some regularity in the patches of differentiation can be observed although these &quot;patterns&quot; depend to a greater degree on initial conditions compared to the patterns seen previously in the Schnakenberg system. However, this is a promising indicator that demonstrates how coordinated multicellular behaviour might originate from an engineered system.
</P><H1><SPAN class="mw-headline" id="Further_Questions">Further Questions</SPAN></H1><H2><SPAN class="mw-headline" id="A_concise_model">A concise model</SPAN></H2><P>We have just tried to go from intra- to intercellular behaviour by naively applying the primarily intracellular agr-QS model to the intercellular case. We cannot say how accurate this approach is since we did not run experiments. However, we can consider a more concise and simpler version of our pattern-generating system. The most important feature of our envisioned systems is that both &quot;populations&quot; inhibit or compete against each other and that the auto-synthesis term is bounded. Assuming that competion can be modelled by simple mass-action, a simplified version of our model would read:
</P><P>Of course, one can also try other systems that incorporate the main features of our system. Further work can be done on the dynamics that these systems exhibit. 
</P><H2><SPAN class="mw-headline" id="Other_pattern-generating_systems">Other pattern-generating systems</SPAN></H2><P>There is a vast number of pattern-generating systems. While we trying to model two competitively inhibiting populations, patterns might be created by a completely different system. In particular, we are interested in the question how pattern generation might occur using a single signalling molecule only.  
</P><H2><SPAN class="mw-headline" id="Bounded_patterns">Bounded patterns</SPAN></H2><P>Patterning is everywhere in biology - one can think of the whole development process as &quot;pattern formation&quot;. Here, patterns generation is switched on by higher-level mechanisms and patterns are expressed in bounded domains (e.g. if we regard organs as patterns, these are clearly seperated from other organs). How does this work? How can we generate patterns that are constraint to a domain? Can we engineer a system that might exhibit this kind of behaviour?    
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>All equations have been solved numerically with a simple difference method - this method is easy to implement and gives accurate results, even though one must be careful that stability does not become an issue.
</P><UL><LI>[1] [Novick et al. 1995] Cell density control of staphylococcal virulence mediated by an octapeptide pheromone
</LI><LI>[2] [Gustafsson, 2004] Characterizing the Dynamics of the Quorum-Sensing System in <I>Staphylococcus aureus</I></LI><LI>[3] [Dockery et al. 2001] A Mathematical Model for Quorum Sensing in <I>Pseudomonas aeruginosa</I></LI><LI>[4] [Canton+Labno, 2008] Refinement and standardization of synthetic biological parts and devices
</LI><LI>[5] [Meinhard, 1992] Rep. Prog. Phys., Pattern formation in biology: a comparison of models and experiments
</LI></UL><DIV style="color: white; font: 2px;">x</DIV></DIV></DIV></TD></TR></TBODY></TABLE></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Cambridge/Modelling">http://2008.igem.org/Team:Cambridge/Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Modelling&amp;oldid=94176" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>