<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Signalling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Cambridge/Signalling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Signalling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Signalling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Signalling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Signalling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Signalling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="bodytable"><DIV class="menubar"><CENTER><A href="/Team:Cambridge" title="Team:Cambridge"><FONT face="verdana" style="color:#ffffff"><B>Home</B></FONT></A> <A href="/Team:Cambridge/Voltage" title="Team:Cambridge/Voltage"><FONT face="verdana" style="color:#b3c6fe"><B>Voltage</B></FONT></A> <STRONG class="selflink"><FONT face="verdana" style="color:#b3c6fe"><B>Signalling</B></FONT></STRONG> <A href="/Team:Cambridge/Bacillus" title="Team:Cambridge/Bacillus"><FONT face="verdana" style="color:#b3c6fe"><B>Bacillus</B></FONT></A> <A href="/Team:Cambridge/Modelling" title="Team:Cambridge/Modelling"><FONT face="verdana" style="color:#b3c6fe"><B>Modelling</B></FONT></A> <A href="/Team:Cambridge/Protocols" title="Team:Cambridge/Protocols"><FONT face="verdana" style="color:#ffffff"><B>Protocols</B></FONT></A> <A href="/Team:Cambridge/People" title="Team:Cambridge/People"><FONT face="verdana" style="color:#ffffff"><B>People</B></FONT></A> </CENTER></DIV><DIV class="submenu"><CENTER><STRONG class="selflink"><FONT face="verdana" style="color:#444444"><B>Signalling Overview</B></FONT></STRONG> <A href="/Team:Cambridge/Signalling/Vectors" title="Team:Cambridge/Signalling/Vectors"><FONT face="verdana" style="color:#444444"><B>Vectors</B></FONT></A> <A href="/Team:Cambridge/Signalling/Constructs" title="Team:Cambridge/Signalling/Constructs"><FONT face="verdana" style="color:#444444"><B>Constructs</B></FONT></A> <A href="/Team:Cambridge/Signalling/Primers" title="Team:Cambridge/Signalling/Primers"><FONT face="verdana" style="color:#444444"><B>Primers</B></FONT></A> <A href="/Team:Cambridge/Signalling/Experiments" title="Team:Cambridge/Signalling/Experiments"><FONT face="verdana" style="color:#444444"><B>Experiments</B></FONT></A> <A href="/Team:Cambridge/Signalling/Lab_Work" title="Team:Cambridge/Signalling/Lab Work"><FONT face="verdana" style="color:#444444"><B>Lab Notebook</B></FONT></A> </CENTER></DIV><P><TABLE style="background:#444444; padding:15px;"><TABLE align="left" width="border=&quot;0&quot;" style="background:#444444; padding:5px;"><TBODY><TR><TD style="width:100%; height: 400; padding-left: 15px;"><DIV class="contentf"><DIV style="height: 400; background: white; line-height:170% padding: 5px;"><DIV style="color: white; font: 2px;">x</DIV><B>Using peptide-based signalling systems from gram-positive bacteria, we have laid the foundations for a self-organising biological system, capable of expressing spatial patterns of GFP expression on a bacterial lawn. The focus of our investigation was on a simple two-component Reaction-Diffusion system, allowing for simple spatial 'patterning' of gene expression. The simplest of these patterns, which <A href="/Team:Cambridge/Modelling" title="Team:Cambridge/Modelling">we have modelled in silico</A>, mimic the spots and stripes seen on animal coats. In 1952, Alan Turing famously described this Reaction-Diffusion system and suggested it as the basis for self-organization and pattern formation in biological systems. This is a first step in the direction of engineering multicellular behaviour.</B></DIV></DIV></TD></TR></TBODY></TABLE></TABLE></P><P>(A) The model consists of two diffusible signals secreted by every cell. The activator, which is controlled by a stochastic bistable switch, turns on itself and its own inhibitor. (B) A field of cells can be stably patterned into two different zones, so long as the inhibitor diffuses faster than the activator. The activator and inhibitor are synthesized in the source at the center, and turned off by accumulation of the inhibitor in the periphery.
</P><P>Our system uses two well-characterized bacterial communication systems to generate this behavior.  The agr peptide signalling system from S. aureus serves as our activatory signal (pictured), while the lux system from V. fischeri serves as our inhibitor. Bacillus subtilis serves as an excellent chassis for this project because of the ease with which chromosomal integration can be performed.
</P><H2><SPAN class="mw-headline" id="Grasshopper_Example">Grasshopper Example</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Moving flame fronts</DIV></DIV></DIV><P>The reaction-diffusion system depends on an activator and inhibitory signal that spread throughout the medium. The &quot;grasshopper&quot; example is quite intuitive: Imagine it is hot and there is a field of dry grass with grasshoppers. Suddenly, a fire starts burning at some point and spreads (the activator signal) so that the grasshoppers move away from that point to avoid the fire. However, the grasshoppers also generate moisture (the inhibitory signal) thus preventing the areas of dry grass the grasshoppers move to of catching fire. The initial patch will have burnt down. However, the surrounded area will be saturated by moisture generated by the grasshoppers thus preventing the fire from spreading. Imagine now that at the beginning, not a single patch, but numerous randomly distributed patches (resembling noise) of dry grass catch fire. The resulting patterning of charred grass and grasshoppers is called a Turing Pattern. It is important to note that the inhibitory signal (grasshoppers) must travel faster than the activation signal (fire) as to prevent the whole field from burning down which would result in no patterning at all. 
</P><H1><SPAN class="mw-headline" id="Objective">Objective</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Schematic of Activator/Inhibitor system in B. subtilis</DIV></DIV></DIV><P>This sub-project seeks to generate <A href="http://en.wikipedia.org/wiki/Turing_Patterns" class="extiw" title="wikipedia:Turing Patterns">Turing Patterns</A> by creating a <A href="http://en.wikipedia.org/wiki/Reaction-Diffusion_System" class="extiw" title="wikipedia:Reaction-Diffusion System">Reaction-Diffusion system</A> in the <A href="http://en.wikipedia.org/wiki/gram-positive_bacteria" class="extiw" title="wikipedia:gram-positive bacteria">gram-positive bacteria</A><A href="http://en.wikipedia.org/wiki/Bacillus_subtilis" class="extiw" title="wikipedia:Bacillus subtilis">Bacillus subtilis</A>. Our goal was to integrate two signalling systems into this bacterium and use an autofeedback mechanism to generate self-organizing patterns from random noise. We planned to incorporate the <I>agr</I> peptide signalling system from <I>S. aureus</I> and the <I>lux</I> AHL system from <I>V. fisheri</I>.
</P><H2><SPAN class="mw-headline" id="Materials"> Materials </SPAN></H2><H3><SPAN class="mw-headline" id="Bacillus_Strains"> Bacillus Strains </SPAN></H3><P>Bacillus strain 1A1 (derivative of standard strain 168)
</P><UL><LI> deficient in tryptophan, have to add to media
</LI><LI> keep at room temp, aren't freezable
</LI></UL><P>Bacillus strain 1A771 (derivative of standard strain 168)
</P><UL><LI> deficient at tryptophan, have to add to media
</LI><LI> keep at room temp, aren't freezable
</LI><LI> contains erm insertion at amyE locus, so transformants at amyE locus can be screened for erythromycin resistance
</LI></UL><P>Bacillus strain 1A751 (derivative of standard strain 168)
</P><UL><LI> deficient at tryptophan, have to add to media
</LI><LI> keep at room temp, aren't freezable
</LI><LI> knockout of Apr and Npr, two major extracellular proteases 
</LI><LI><A href="http://mic.sgmjournals.org/cgi/content/abstract/141/2/281" class="external text" rel="nofollow">Original Publication</A></LI></UL><H3><SPAN class="mw-headline" id="Vectors"><A href="/Team:Cambridge/Signalling/Vectors" title="Team:Cambridge/Signalling/Vectors"> Vectors</A></SPAN></H3><UL><LI> Details of vectors we used, including maps, sequences, and testing
</LI></UL><H3><SPAN class="mw-headline" id="Primers"><A href="/Team:Cambridge/Signalling/Primers" title="Team:Cambridge/Signalling/Primers"> Primers</A></SPAN></H3><UL><LI> List of primers we have ordered
</LI></UL><H3><SPAN class="mw-headline" id="Constructs"><A href="/Team:Cambridge/Signalling/Constructs" title="Team:Cambridge/Signalling/Constructs"> Constructs</A></SPAN></H3><UL><LI> Planning and organization of biobrick assemblies and recombinant plasmids
</LI></UL><H3><SPAN class="mw-headline" id="Lab_Work_Page"><A href="/Team:Cambridge/Signalling/Lab_Work" title="Team:Cambridge/Signalling/Lab Work"> Lab Work Page</A></SPAN></H3><UL><LI> Main page detailing our day-to-day research activities
</LI></UL><H3><SPAN class="mw-headline" id="Experiments"><A href="/Team:Cambridge/Signalling/Experiments" title="Team:Cambridge/Signalling/Experiments"> Experiments</A></SPAN></H3><UL><LI> Main page detailing our planned and completed experiments
</LI></UL><DIV style="color: white; font: 2px;">x</DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Cambridge/Signalling">http://2008.igem.org/Team:Cambridge/Signalling</A>&quot;</DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Signalling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Signalling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Signalling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Signalling&amp;oldid=104726" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>