<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Voltage_Lab_Work"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Cambridge/Voltage/Lab_Work">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Voltage/Lab_Work&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Voltage/Lab_Work&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Voltage/Lab_Work&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Voltage/Lab_Work" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Voltage/Lab Work</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="bodytable"><DIV class="menubar"><CENTER><A href="/Team:Cambridge" title="Team:Cambridge"><FONT face="verdana" style="color:#ffffff"><B>Home</B></FONT></A> <A href="/Team:Cambridge/Voltage" title="Team:Cambridge/Voltage"><FONT face="verdana" style="color:#b3c6fe"><B>Voltage</B></FONT></A> <A href="/Team:Cambridge/Signalling" title="Team:Cambridge/Signalling"><FONT face="verdana" style="color:#b3c6fe"><B>Signalling</B></FONT></A> <A href="/Team:Cambridge/Bacillus" title="Team:Cambridge/Bacillus"><FONT face="verdana" style="color:#b3c6fe"><B>Bacillus</B></FONT></A> <A href="/Team:Cambridge/Modelling" title="Team:Cambridge/Modelling"><FONT face="verdana" style="color:#b3c6fe"><B>Modelling</B></FONT></A> <A href="/Team:Cambridge/Protocols" title="Team:Cambridge/Protocols"><FONT face="verdana" style="color:#ffffff"><B>Protocols</B></FONT></A> <A href="/Team:Cambridge/People" title="Team:Cambridge/People"><FONT face="verdana" style="color:#ffffff"><B>People</B></FONT></A> </CENTER></DIV><DIV class="submenu"><CENTER><A href="/Team:Cambridge/Voltage" title="Team:Cambridge/Voltage"><FONT face="verdana" style="color:#444444"><B>Overview</B></FONT></A> <A href="/IGEM:Cambridge/2008/Notebook/Voltage/Gene_Design" title="IGEM:Cambridge/2008/Notebook/Voltage/Gene Design"><FONT face="verdana" style="color:#444444"><B>Constructs</B></FONT></A> <A href="/IGEM:Cambridge/2008/Notebook/Voltage/Mutant_Strains" title="IGEM:Cambridge/2008/Notebook/Voltage/Mutant Strains"><FONT face="verdana" style="color:#444444"><B>Mutants</B></FONT></A> <A href="/IGEM:Cambridge/2008/Notebook/Voltage/K%2B_Concentrations" title="IGEM:Cambridge/2008/Notebook/Voltage/K+ Concentrations"><FONT face="verdana" style="color:#444444"><B>Potassium Assay</B></FONT></A> <A href="/IGEM:Cambridge/2008/Notebook/Voltage/Output" title="IGEM:Cambridge/2008/Notebook/Voltage/Output"><FONT face="verdana" style="color:#444444"><B>Voltage Output</B></FONT></A> <A href="/IGEM:Cambridge/2008/Notebook/Voltage/Future_Work" title="IGEM:Cambridge/2008/Notebook/Voltage/Future Work"><FONT face="verdana" style="color:#444444"><B>Future Work</B></FONT></A> </CENTER></DIV><TABLE style="background:#444444; padding:15px;"><TABLE align="left" width="border=&quot;0&quot;" style="background:#444444; padding:5px;"><TBODY><TR><TD style="width:100%; height: 400; padding-left: 15px;"><DIV class="contentf"><DIV style="height: 400; background: white; line-height:170% padding: 5px;"><DIV style="color: white; font: 2px;">x</DIV><H1><SPAN class="mw-headline" id="15th_July">15th July</SPAN></H1><H2><SPAN class="mw-headline" id="Conference_in_Edinburgh">Conference in Edinburgh</SPAN></H2><P>Meet with the other UK iGEM teams.
</P><H1><SPAN class="mw-headline" id="16th_July">16th July</SPAN></H1><H2><SPAN class="mw-headline" id="Conference_in_Edinburgh_2">Conference in Edinburgh</SPAN></H2><P>Presentation of projects to the other teams and to the attendees of the SynBioStandards network meeting.
</P><H1><SPAN class="mw-headline" id="17th_July">17th July</SPAN></H1><H2><SPAN class="mw-headline" id="Progress">Progress</SPAN></H2><P>-Ordered mutant E.coli chassis to test: Kch<SUP>-</SUP>, KdpD<SUP>-</SUP>, KdpE<SUP>-</SUP>, KdpF-E<SUP>-</SUP></P><P>-Designed two plasmids:
</P><UL><LI><A href="/IGEM:Cambridge/2008/Notebook/Voltage/Gene_Design#KdpF-C_Biobrick-_BBa_K090003.3D" title="IGEM:Cambridge/2008/Notebook/Voltage/Gene Design">Kdp Plasmid</A></LI></UL><UL><LI><A href="/IGEM:Cambridge/2008/Notebook/Voltage/Gene_Design#GluR0_Biobrick-_BBa_K090002" title="IGEM:Cambridge/2008/Notebook/Voltage/Gene Design">Glu Gated channel Plasmid</A></LI></UL><P>-Sourced V.harveyi BB170 culture to obtain glutamate-gated potassium channel gene
</P><P>-Modelled protein structure
</P><P>-Designed primers for extraction of Kdp Operon from E.coli and Glu Gated channel gene from V.harveyi
</P><P>-Meeting with Julia Davies
</P><P>-Discussed feasibility
</P><P>-Osmotic problems with ion sequestration, possible solution - &quot;inert&quot; sugars such as sorbitol and manitol
</P><P>-Glutamate/glutamine specificity problems, may need glutamine- as well as glutamate-free growth medium
</P><P>-Need to test efficacy of voltage creation over range of osmolarities, use flame photometer to plot total K<SUP>+</SUP> as f<SUP>n</SUP> of growth curve
</P><P>-Discussed Kch<SUP>-</SUP> mutants, generally benign
</P><P>-Potential PD measurement methods: high resistance input voltmeter to minimise noise, glass capillary microelectrode, oxygen electrode
</P><P>-Measurement issue of cells acting as charged particles, neutralising charge around electrode
</P><P>-Decided on oxygen electrode for small voltage change measurements
</P><P>
-Extracted BioBricks from registry to use in plasmids:
</P><UL><LI> Promoter BBa_J23100
</LI></UL><UL><LI> RBS BBa_B0030
</LI></UL><UL><LI> Terminators BBa_B1002 and BBa_B1006
</LI></UL><H1><SPAN class="mw-headline" id="18th_July">18th July</SPAN></H1><H2><SPAN class="mw-headline" id="Wet_Work">Wet Work</SPAN></H2><P>There was no growth on the plates spread yesterday. There has been a few adjustments to the protocol.
</P><P>Biobrick Extraction
</P></DIV></DIV></TD></TR><TR><TH> Class
</TH><TH> Registry
</TH><TH> Part Kit Location
</TH><TH> Tube Label
</TH></TR><TR><TD> Promoter </TD><TD> BBa_J23100 </TD><TD> 2008 1002-10E </TD><TD> F
</TD></TR><TR><TD> RBS </TD><TD> BBa_B0030 </TD><TD> 2008 1002-5G </TD><TD> G
</TD></TR><TR><TD> Stop </TD><TD> BBa_B1002 </TD><TD> 2008 1016-8A </TD><TD> H
</TD></TR><TR><TD> Stop </TD><TD> BBa_B1006 </TD><TD> 2008 1016-8D </TD><TD> I
</TD></TR><TR><TD> Promoter </TD><TD> BBa_J23113 </TD><TD> 2008 1002-12B </TD><TD> J
</TD></TR><TR><TD> Promoter </TD><TD> BBa_J23114 </TD><TD> 2008 1002-12C </TD><TD> K
</TD></TR><TR><TD> Control </TD><TD> pUC19 </TD><TD> L
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Bigger.2C_better.2C_faster.2C_stronger">Bigger, better, faster, stronger</SPAN></H3><P>We found we could not extract enough DNA from the registry, so we are upgrading the extraction to the &quot;bigger, better, faster, stronger&quot; method.
</P><P>Warm 50μL of EB in Eppendorf tubes at 50°C and add 4 punched spots. Keep it warming at 50°C for 20mins and spin down for 3 minutes at 15,000 g. Warm agin for 10mins and spin down again for 3mins. Pipette off the liquid which should have the DNA in.
We then confirmed with PCR.
</P><P>
Each sample except the control was then put through 34 cycles of PCR.
The following were added to eppendorf tubes:
</P><P>5μL of DNA in EB buffer
</P><P>2.5μL of each primer for Biobrick vectors
</P><P>25μL of Finnzymes mastermix
</P><P>15μL of sterile distilled water
</P><P>And all 50μL were then run through the PCR reaction and 17μL of product run on an E-Gel with 3μL dye.
</P><P>The results gave large amounts of DNA in each lane for all Biobricks except F.
The original extracts of G,H,I,J,K and L were then used for the following transformations.
</P><P>To chilled tubes add 5μL of DNA + EB solution to 5μL of Cells ( Chemically competent Top 10 ) 
</P><P>Ice for 30 minutes 
Heat shock at 42°C for 60 seconds 
Ice for 2 minutes 
Add 500μL of SOC 
Incubate at 37°C overnight
</P><P>Prepare Agar plates : Add 200μL of Amp (100mg/mL) into 200mL of LA ( Final conc. of 100μg/mL ) and add 100μL of Amp (100mg/mL) into 200mL of LA ( Final conc. of 50μg/mL )
Plate neat samples of G,H,I,J,K,L onto both types of plate and incubate overnight.
</P><H1><SPAN class="mw-headline" id="21st_July">21st July</SPAN></H1><H2><SPAN class="mw-headline" id="Wet_Work_2">Wet Work</SPAN></H2><H3><SPAN class="mw-headline" id="Aims_for_Today">Aims for Today</SPAN></H3><P>The results of yesterday's modified protocol still yielded no growth on the plates. Using yesterday's PCR product our aims were:
</P><UL><LI>Cut promoters with spe1
</LI><LI>Cut RBS with Xba1
</LI><LI>Ligate these together and perform further PCR amplification of linear fragments. 
</LI></UL><UL><LI>Cut first 'stop' with spe1
</LI><LI>Cut second 'stop with xba1
</LI><LI>Ligate these together and perform further PCR amplification of linear fragments. 
</LI></UL><UL><LI>Finally, put all the above elements into a vector plasmid.
</LI></UL><UL><LI>End of day: Aims completed
</LI></UL><H3><SPAN class="mw-headline" id="Length">Length</SPAN></H3><UL><LI>Promoter = 35 bases
</LI><LI>RBS = 16 bases
</LI><LI>Stop (B1002) = 35 bases and Stop (B1006) = 39 bases
</LI></UL><H1><SPAN class="mw-headline" id="22nd_July">22nd July</SPAN></H1><H2><SPAN class="mw-headline" id="Wet_Work_3">Wet Work</SPAN></H2><UL><LI> Inoculated 10mL LB with E.coli strain MG1655
</LI><LI> Inoculated 10mL LB with E.coli strain TOP10 transformed with BioBrick J5311
</LI></UL><P>16 LA plates were poured with 8 having 25μg/mL Kanamycin, ready for mutant strains to be grown on.
</P><P>The gel extracts from the ligation of primer and rbs were extracted using Zymoclean gel extraction kit. Hopefully these now contain linear DNA of BBa_J23113 and BBa_B0030, and BBa_J23114 and BBa_B0030.
</P><P>BioBricks J45996, J45995 and J45250 were also extracted from the registry, using the &quot;bigger, better, faster, stronger, method&quot;, in order to provide template to extracted the OsmY promoter.
</P><H1><SPAN class="mw-headline" id="23rd_July">23rd July</SPAN></H1><H2><SPAN class="mw-headline" id="Wet_Work_4">Wet Work</SPAN></H2><UL><LI> Recovered Biobrick plasmid YFP from previous culture.
</LI><LI>1.6 ml of cells according to Zippy Plasmid Miniprep Kit protocol (60μL Elution Buffer used)
</LI></UL><UL><LI>Added 20ml LB to 10ml innoculation of E Coli MG1655 (see 22/07/08) to increase volume in preparation for experiments tomorrow on K+ tolerance in media.
</LI></UL><H1><SPAN class="mw-headline" id="24th_July">24th July</SPAN></H1><H2><SPAN class="mw-headline" id="Double_Terminator">Double Terminator</SPAN></H2><P>The double terminator was made from BBa_B1006 and BBa_B1002, combined together.
</P><P>First the two Biobricks were cut using the fast digest enzymes, at either Xba or Spe. They were then run on an agarose gel,  cut out and extracted, using Zymoclean gel extraction. The 2006 was not cut succesfully with Spe, so was not extracted. The samples were then ligated, using fast ligation, such that 1002-1002 and 1002-1006 were hopefully produced. This was done by combining the 1002 cut with Spe with the 1002 cut with Xba and the 1006 cut with Xba. These were then run on an agarose gel and cut out, then frozen overnight.
</P><H1><SPAN class="mw-headline" id="25th_July">25th July</SPAN></H1><H2><SPAN class="mw-headline" id="Double_Terminator_2">Double Terminator</SPAN></H2><P>The DNA was extracted from lanes 3&amp;4 of the gel and extracted using the Zyppy Gel Extraction kit.
The extracted DNA was then PCRed using 34 cycles, and then 10μL was frozen at -20°C.
8μL of the PCR product was cut with both EcoR1 and Pst1, as was 5μL of the death plasmid, PSB4C5, using the fast digest protocol.
The gene was then ligated into the plasmid backbone, using the fast ligation kit, from Finnzymes. 
</P><P>500μL of normal TOP10 cells were made competent, and then transformed with the 4μL of the new plasmid, and grown on chloramphenicol overnight. 1μL of Puc19 was used as a control.
</P><H2><SPAN class="mw-headline" id="K.2B_assay_for_E.coli">K+ assay for E.coli</SPAN></H2><P>Solutions of potassium chlotide were made up from 0mM to 500mM. 100μL of cells, 100μL of 10x KCl solution and 800μL of SDW were mixed and the OD600 measured every 30 mins, after agitation to ensure homogenaity. After 4 hours 800μL of the cell solution were spun down and resuspended in 1mL SDW, lysed using the freeze-thaw method, and the K+ concentration measured using the Flame Photometer.
</P><H1><SPAN class="mw-headline" id="28th_July">28th July</SPAN></H1><P>The flame photometer was also calibrated using μM concentrations of potassium.
</P><P>
The mutants were streaked out from the innoculation discs onto both kanamycin and non antibiotic plates for the resistant strains, and just onto the normal plates for non-resistant strains. They were then incubated overnight at 37°C. 
</P><H3><SPAN class="mw-headline" id="Mutant_Plates">Mutant Plates</SPAN></H3><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>JW142
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>TK3D11
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>TMNT
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>JW0045
</P></DIV></DIV></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="29th_July">29th July</SPAN></H1><UL><LI> Mutant <I>E.coli</I> strains had grown on plates.
</LI><LI> Transferred mutants to liquid culture.
</LI><LI> Primers arrived! Prepped as described in <A href="/Protocols" title="Protocols">Protocols</A> page
</LI><LI> PCR started with OsmY (from Biobrick plasmids) and KDP (from E. Coli MG1655 Cell Culture)
</LI><LI> E-Gel (SYBR) to confirm OsmY
</LI><LI> E-Gel (EtBr) to confirm Kdp and OsmY
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:278px;"><DIV class="thumbcaption">Gel showing KDP (2 thick bands on the left and OsmY (dimmer bands to the right)</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="30th_July">30th July</SPAN></H1><H1><SPAN class="mw-headline" id="Competence_test">Competence test</SPAN></H1><UL><LI> Transformed PUK plasmid and YFP plasmid into mutant <I>E.coli</I> strains.
</LI></UL><H1><SPAN class="mw-headline" id="osmY_and_Kdp">osmY and Kdp</SPAN></H1><UL><LI>digested PCR products, and death gene vector with EcoR1 and Pst1
</LI><LI>ligated new BioBricks into vector
</LI><LI>Transformed into competent TOP10 cells
</LI><LI>Plate out onto Chloramphenicol plates
</LI></UL><H1><SPAN class="mw-headline" id="31st_July">31st July</SPAN></H1><H2><SPAN class="mw-headline" id="Mutants">Mutants</SPAN></H2><UL><LI> Mutant strains were grown in different potassium levels: 1mM, 50mM and 200mM.
</LI><LI> Growth curves were plotted, using OD600 readings every 30 minutes for 3 hours.
</LI></UL><H2><SPAN class="mw-headline" id="Kdp">Kdp</SPAN></H2><UL><LI> Ligated Kdp into death vector PSB4C5 again.
</LI><LI> More PCR of gene from MG1655.
</LI></UL><H2><SPAN class="mw-headline" id="OsmY">OsmY</SPAN></H2><UL><LI> TOP10 containing osmY-containing plasmid were grown in LB.
</LI></UL><H1><SPAN class="mw-headline" id="1st_August">1st August</SPAN></H1><H2><SPAN class="mw-headline" id="Mutants_2">Mutants</SPAN></H2><UL><LI> The OD600 of the mutants grown in different K+ concentrations was taken again.
</LI></UL><H2><SPAN class="mw-headline" id="osmY_2">osmY</SPAN></H2><UL><LI> Plasmid mini prep was done.
</LI><LI> Gel was run, showing clear band at around 450bp, indicating osmY with BioBrick primer prefix and suffix.
</LI><LI> Glycerol stocks were made of the TOP10 cultures with the osmY-containing plasmid.
</LI><LI> More LB was added to the TOP10 cultures and they were put in the incubator.
</LI></UL><H2><SPAN class="mw-headline" id="Plasmid">Plasmid</SPAN></H2><UL><LI> PSB4C5 was extracted from DB31, into 100μl SDW - final conc. 106.5ng/μl.
</LI></UL><H1><SPAN class="mw-headline" id="4th_August">4th August</SPAN></H1><H2><SPAN class="mw-headline" id="Biobricks">Biobricks</SPAN></H2><UL><LI> The liagtion reaction from last week was drop dialised and then 5μL used to transform highly competent TOP10, before plating on chloramphenicol plates.
</LI><LI> The Biobrick combination of promoter-RBS were then cut with Pst1 and EcoR1 and ligated into pSB4C5 and then transformed into TOP10 and incubated overnight at room temperature in 600μL SOC.
</LI></UL><H1><SPAN class="mw-headline" id="5th_August">5th August</SPAN></H1><H2><SPAN class="mw-headline" id="Kdp.3F">Kdp?</SPAN></H2><UL><LI> There were transformants from yesterday's transformation
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:153px;"><DIV class="thumbcaption">James celebrates apparent success</DIV></DIV></DIV></DIV><UL><LI> After Biobrick PCR the gel showed short fragments, suggesting self ligation of the vector.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Ellis realises the sad truth</DIV></DIV></DIV></DIV><UL><LI> Repeated digest and ligation protocol from the weekend, with EcoR1 and Pst1 instead of Xba1 and Spe.
</LI></UL><H2><SPAN class="mw-headline" id="Biobricks_2">Biobricks</SPAN></H2><UL><LI> Extracted two GFP-only sequences (J04630 and I714062) from the registry, for use as reporters for osmY.
</LI><LI> PCR on these and the RBS B0030. Ran E-gel to check PCR.
</LI><LI> Cut osmY plasmid with Spe, and RBS B0030 with Xba, and ligated the two.
</LI><LI> Transformed cells with this construct. Then realised this was silly.
</LI></UL><H1><SPAN class="mw-headline" id="6th_August">6th August</SPAN></H1><H2><SPAN class="mw-headline" id="Kdp.3F_2">Kdp?</SPAN></H2><UL><LI> Transformants!
</LI><LI> But... PCR showed that none contained the Kdp insert.
</LI><LI> Gelled cut vector and extracted backbone band to use for ligation.
</LI><LI> Ligated Kdp gene into backbone and transformed cells. Again.
</LI></UL><H2><SPAN class="mw-headline" id="Mutants_3">Mutants</SPAN></H2><UL><LI> Grew up 5 mutants in LB, 1mM, 50mM and 200mM KCl.
</LI></UL><H1><SPAN class="mw-headline" id="7th_August">7th August</SPAN></H1><H2><SPAN class="mw-headline" id="Flame_Photometry">Flame Photometry</SPAN></H2><UL><LI> Various concentrations of KCl were made up in LB to calibrate for LB instead of water.
</LI><LI> Took OD600 readings of mutants grown in KCl concentrations in shaking incubator overnight
</LI><LI> They were then spun down, resuspended in SDW and spun again to wash, then resuspended.
</LI><LI> The resuspended cells were put through the flame photometer to discern internal cell K+ concentration.
</LI><LI> The supernatant K+ concentration was also measured.
</LI></UL><H2><SPAN class="mw-headline" id="Kdp_2">Kdp</SPAN></H2><UL><LI> Ran ligation mixtures on agarose gel and extracted 8kb (supercoiled plasmid) bands.
</LI><LI> Transformed cells with cleaned up plasmid.
</LI></UL><H2><SPAN class="mw-headline" id="Mutants_4">Mutants</SPAN></H2><H1><SPAN class="mw-headline" id="8th_August">8th August</SPAN></H1><H2><SPAN class="mw-headline" id="Kdp.3F_3">Kdp?</SPAN></H2><UL><LI> Plates from yesterday showed no transformants. Others having difficulty with transformation too.
</LI><LI> Made up new Ampicillin stock and got new SOC and CaCl<SUB>2</SUB> to try control (PUC) transformation in TOP10 next week.
</LI></UL><H2><SPAN class="mw-headline" id="Mutants_5">Mutants</SPAN></H2><UL><LI> Grew up 5 mutant strains in 5 different KCl concentrations in LB:
</LI></UL><BLOCKQUOTE style="background: white; border: 1px solid black; padding: 1em;"><TABLE border="1" cellpadding="2"><TBODY><TR valign="top"><TD width="20%"><B>Strain and Mutation</B></TD><TD width="20%"><B>KCl conc (mM)</B></TD><TD width="20%"><B>OD Reading</B></TD></TR><TR valign="top"><TD width="20%"><B>BW25113</B></TD><TD width="20%">10
</TD></TR><TR valign="top"><TD width="20%">20
</TD></TR><TR valign="top"><TD width="20%">50
</TD></TR><TR valign="top"><TD width="20%">100
</TD></TR><TR valign="top"><TD width="20%">200
</TD></TR><TR valign="top"><TD width="20%"><B>BW30166</B></TD><TD width="20%">10
</TD></TR><TR valign="top"><TD width="20%">20
</TD></TR><TR valign="top"><TD width="20%">50
</TD></TR><TR valign="top"><TD width="20%">100
</TD></TR><TR valign="top"><TD width="20%">200
</TD></TR><TR valign="top"><TD width="20%"><B>JW0045</B></TD><TD width="20%">10
</TD></TR><TR valign="top"><TD width="20%">20
</TD></TR><TR valign="top"><TD width="20%">50
</TD></TR><TR valign="top"><TD width="20%">100
</TD></TR><TR valign="top"><TD width="20%">200
</TD></TR><TR valign="top"><TD width="20%"><B>JW1242</B></TD><TD width="20%">10
</TD></TR><TR valign="top"><TD width="20%">20
</TD></TR><TR valign="top"><TD width="20%">50
</TD></TR><TR valign="top"><TD width="20%">100
</TD></TR><TR valign="top"><TD width="20%">200
</TD></TR><TR valign="top"><TD width="20%"><B>JW3314</B></TD><TD width="20%">10
</TD></TR><TR valign="top"><TD width="20%">20
</TD></TR><TR valign="top"><TD width="20%">50
</TD></TR><TR valign="top"><TD width="20%">100
</TD></TR><TR valign="top"><TD width="20%">200
</TD></TR></TBODY></TABLE></BLOCKQUOTE><H1><SPAN class="mw-headline" id="11th_August">11th August</SPAN></H1><H2><SPAN class="mw-headline" id="Kdp.3F_4">Kdp?</SPAN></H2><UL><LI>PCR colonies on plates - no insert seen.
</LI><LI> Transform overweekend ligation.
</LI></UL><H2><SPAN class="mw-headline" id="OsmY_and_RBS">OsmY and RBS</SPAN></H2><UL><LI> PCR B0030 and restrict 10μL with Xba.
</LI><LI> Plasmid extract OsmY and restriction digest with Xba.
</LI><LI> Ligate and transform cells with 5μL
</LI></UL><H1><SPAN class="mw-headline" id="12th_August">12th August</SPAN></H1><H2><SPAN class="mw-headline" id="Kdp_3">Kdp</SPAN></H2><UL><LI> Again try ligation of Kdp and Vector backbone
</LI></UL><H2><SPAN class="mw-headline" id="OsmY_3">OsmY</SPAN></H2><UL><LI> Again try ligation of OsmY and B0030 backbone
</LI><LI> This time try using Xba for B0030 and ligate. Gel again and cut out linear fragment. Cut with Spe and gel - cut out circular fragment.
</LI></UL><H2><SPAN class="mw-headline" id="Make_competent_stocks">Make competent stocks</SPAN></H2><P>Chemically competent TOP10 cells were made by growing overnight in 200ml LB in a shaking incubater at 37°C. 
200ml were spun down at 3800rpm for 8 mins, resuspended in 20ml CaCl<SUB>2</SUB>, spun down again and resuspended in 4ml CaCl<SUB>2</SUB>.
Again they were spun down (6500 rpm for 5 mins)and resuspended in 4ml 60%glycerol, spun down and resuspended in 2ml 60%glycerol. Finally they were aliquoted (100μL) into stock tubes and flash frozen in liquid nitrogen.
</P><P>Electroporation competent TOP10 and DH5α cells were made by growing overnight in 200ml LB in a shaking incubater at 37°C.
100ml were spun down at 3800rpm for 8 mins, resuspended in 25ml SDW, spun down again and resuspended in 2ml SDW.
Again they were spun down (6500 rpm for 5 mins)and resuspended in 2ml 60%glycerol, spun down and resuspended in 2ml 60%glycerol. Finally they were aliquoted (100μL) into stock tubes and flash frozen in liquid nitrogen.
However lost half of the DH5α when centrifuge tube broke.
</P><H1><SPAN class="mw-headline" id="13th_August">13th August</SPAN></H1><H2><SPAN class="mw-headline" id="Kdp_4">Kdp</SPAN></H2><UL><LI> Same ole, same ole.
</LI></UL><H2><SPAN class="mw-headline" id="Recovering_frozen_cells">Recovering frozen cells</SPAN></H2><P>To recover frozen cells they were thawed on ice and spun down at6500rpm for 5mins. They were then resuspended in 100μL CaCl<SUB>2</SUB>.
They were then transformed with 2μL pUC9 and grown on 100μg/mL Amp plates.
</P><H1><SPAN class="mw-headline" id="14th_August">14th August</SPAN></H1><H2><SPAN class="mw-headline" id="Kdp_5">Kdp</SPAN></H2><UL><LI> Cut Kdp with EcoR1 and Pst1 and ran on gel.
</LI></UL><H2><SPAN class="mw-headline" id="OsmY_and_RBS_2">OsmY and RBS</SPAN></H2><UL><LI> Cut with Spe and Xba respectively and ligated, using cooling ligation steps (42 - 35°C) and run on gel.
</LI></UL><H1><SPAN class="mw-headline" id="15th_August">15th August</SPAN></H1><H2><SPAN class="mw-headline" id="Kdp_6">Kdp</SPAN></H2><UL><LI> Nothing worked. Again. Query gremlins? Or possible sabotage by people in 106.
</LI></UL><H2><SPAN class="mw-headline" id="MEGA_Mutant_Assay">MEGA Mutant Assay</SPAN></H2><UL><LI> THAT'S NUMBERWANG! (Actual Data)
</LI></UL><H1><SPAN class="mw-headline" id="16th_August">16th August</SPAN></H1><P>Bank Holiday. Apparently no-one was quite that dedicated to come into work.
</P><H1><SPAN class="mw-headline" id="19th_August">19th August</SPAN></H1><H2><SPAN class="mw-headline" id="Ligation_Diagnostic">Ligation Diagnostic</SPAN></H2><UL><LI>Due to lack of kdp insert following scPCR of transformed colonies, the ligation procedure was tested, involving Kdp &amp; Vector biobricks together and separately, dirty ligations, negative controls where buffer was added but no ligase etc.
</LI><LI>Results of gel seemed inconclusive as hyperladder I ran very badly, but there seemed to be no DNA in the 'vector only lane' on both occasions.
</LI><LI>Conclusion: investigate our supply of pSB4C5 vector.
</LI></UL><H2><SPAN class="mw-headline" id="Kdp_7">Kdp</SPAN></H2><UL><LI>YPF biobrick plasmid used to provide vector backbone and a Kdp digest/ligation was attempted.
</LI></UL><H1><SPAN class="mw-headline" id="20th_August">20th August</SPAN></H1><H2><SPAN class="mw-headline" id="pSB4C5_.26_RBS">pSB4C5 &amp; RBS</SPAN></H2><UL><LI>scPCR perfomed on 5 colonies of DB31 &amp; TOP10 to determine whether they contain pSB4C5 and B0030 respectively.
</LI><LI>Sucessful colonies grown up in LB, and pSB4C5 glycerol stocks were created.
</LI><LI>Large quantities of pSB4C5 were also harvested. 
</LI></UL><H2><SPAN class="mw-headline" id="Kdp_8">Kdp</SPAN></H2><UL><LI>Electroporation of cells with the Kdp-YFP backbone.
</LI><LI>This failed (Even control, Jim!)
</LI></UL><H1><SPAN class="mw-headline" id="21st_August">21st August</SPAN></H1><H2><SPAN class="mw-headline" id="OsmY_RBS">OsmY RBS</SPAN></H2><UL><LI>B0030 harvested from LB stocks.
</LI><LI>Cut OsmY with Spe1 &amp; B0030 with Xba1
</LI><LI>Attempted to ligate promoter and RBS sequences together again.
</LI></UL><H1><SPAN class="mw-headline" id="22nd_August">22nd August</SPAN></H1><H2><SPAN class="mw-headline" id="Kdp_.26_OsmY-RBS">Kdp &amp; OsmY-RBS </SPAN></H2><UL><LI> Digestion/ligation &amp; drop dialysis prior to transformation into TOP10.
</LI></UL><H1><SPAN class="mw-headline" id="26th_August">26th August</SPAN></H1><H2><SPAN class="mw-headline" id=".3DKdp_screening">=Kdp screening</SPAN></H2><UL><LI> Many transformant colonies were found on chloramphenicol plates.
</LI><LI> Single colony PCR was perfomed over a selection of 44 colonies using forward and reverse biobrick primers in order to gauge the size of the vector insert by gel visualisation.
</LI><LI>All screened colonies were negative for Kdp - concluded most likely to be blunt ended self ligation of pSB4C5 vector (without death gene). 
</LI></UL><H1><SPAN class="mw-headline" id="27th_August">27th August</SPAN></H1><H2><SPAN class="mw-headline" id="OsmY_RBS_2">OsmY RBS</SPAN></H2><UL><LI> Primers containing RBS for osmY Biobrick were received
</LI><LI> PCR was performed from biobrick J45250
</LI><LI> Ligated into dephosphorylated vector backbone
</LI><LI> Colonies transformed
</LI></UL><H2><SPAN class="mw-headline" id="Restriction_Digest_.26_Ligation_Test">Restriction Digest &amp; Ligation Test</SPAN></H2><UL><LI> Various combinations of restriction digests &amp; T4 phage genome ligation test.
</LI><LI> All results positive :-)
</LI></UL><H1><SPAN class="mw-headline" id="29th_August">29th August</SPAN></H1><H2><SPAN class="mw-headline" id="GluR0">GluR0</SPAN></H2><UL><LI> We received our synthesised plasmid from DNA 2.0 today!
</LI><LI>Transformed into TOP10.
</LI></UL><P><A href="/File:Glur0.tif" title="File:Glur0.tif">File:Glur0.tif</A></P><H1><SPAN class="mw-headline" id="2nd_September">2nd September</SPAN></H1><H2><SPAN class="mw-headline" id="WE_HAVE_KDP.21">WE HAVE KDP!</SPAN></H2><UL><LI> It has successfully been ligated into the death plasmid, transformed into TOP10 and plasmid prepped.
</LI><LI> The beautiful gel (note lanes 2-4 of right-hand gel, showing 7kb supercoiled plasmid):
</LI></UL><H1><SPAN class="mw-headline" id="5th_September">5th September</SPAN></H1><H2><SPAN class="mw-headline" id="STOP">STOP</SPAN></H2><UL><LI> Transform B1002 and B1006 into TOP10
</LI></UL><H2><SPAN class="mw-headline" id="GluR0_2">GluR0</SPAN></H2><UL><LI>Cut with E+P or E+S
</LI><LI>Cut pSB4C5 with same
</LI><LI>Dephosphorylate pSB4C5
</LI><LI>Gel Extract, Ligate, Transform.
</LI></UL><H1><SPAN class="mw-headline" id="6th_September">6th September</SPAN></H1><H2><SPAN class="mw-headline" id="Ligations">Ligations</SPAN></H2><UL><LI> KDP and OsmY into pSB4C5 using E+S or X+P respectively
</LI><LI> Attempt to double stop with B1006 and B1002
</LI></UL><H1><SPAN class="mw-headline" id="9th_September">9th September</SPAN></H1><H2><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H2><UL><LI> Ligations from previous day transformed into TOP10
</LI></UL><H1><SPAN class="mw-headline" id="10th_September">10th September</SPAN></H1><H2><SPAN class="mw-headline" id="Analysis">Analysis</SPAN></H2><UL><LI> Analyse transformants
</LI></UL><H2><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H2><UL><LI>Re-attempt KDP/OsmY ligation
</LI></UL><H1><SPAN class="mw-headline" id="11th_September">11th September</SPAN></H1><H2><SPAN class="mw-headline" id="Ligation_2">Ligation</SPAN></H2><UL><LI> GluR0 + pSB4T5
</LI></UL><H1><SPAN class="mw-headline" id="12th_September">12th September</SPAN></H1><H2><SPAN class="mw-headline" id="Analysis_2">Analysis</SPAN></H2><UL><LI> Analyse KDP, OsmY, Glur0 ligations
</LI></UL><H1><SPAN class="mw-headline" id="15th_September">15th September</SPAN></H1><H2><SPAN class="mw-headline" id="Plasmid_Prep">Plasmid Prep</SPAN></H2><UL><LI> KDP, OsmY and GluR0
</LI></UL><H2><SPAN class="mw-headline" id="Restriction_Digest">Restriction Digest</SPAN></H2><UL><LI>Digested prepped plasmids and run on gel.
</LI></UL><H1><SPAN class="mw-headline" id="16th_September">16th September</SPAN></H1><H2><SPAN class="mw-headline" id="Ligations_2">Ligations</SPAN></H2><UL><LI> GluR0 + pSB4T5 or pSB3T5
</LI><LI> B1002 + B1006
</LI></UL><H2><SPAN class="mw-headline" id="Gel">Gel</SPAN></H2><UL><LI> B1002 + B1006 ligation product run with GluR0 unsupercoiled
</LI><LI> GluR0 was lightly irradiated to induce sing strand breaks to allow it to de-supercoil and act as a 4kb template.
</LI></UL><H1><SPAN class="mw-headline" id="18th_September">18th September</SPAN></H1><H2><SPAN class="mw-headline" id="Applications">Applications</SPAN></H2><UL><LI> Designed standard infusion primers for biobrick assembly. Abandoned due to unfeasibly large scar sites between Biobricks, more effective to use custom designed primers for each reaction.
</LI></UL><H1><SPAN class="mw-headline" id="19th_September">19th September</SPAN></H1><H2><SPAN class="mw-headline" id="GluR0_Testing">GluR0 Testing</SPAN></H2><UL><LI> Tested GluR0 expression cells with shiny/expensive/confusing/very, very, very old electrical equipment. Engineers left. Damn (and blast) them. The filthy varmints.
</LI><LI>Initial results promising.
</LI></UL><H1><SPAN class="mw-headline" id="22nd_September">22nd September</SPAN></H1><H2><SPAN class="mw-headline" id="PCR">PCR</SPAN></H2><UL><LI> Single colony PCR of GluR0, OsmY, KDP.
</LI><LI> Results were ambiguous or negative.
</LI></UL><H1><SPAN class="mw-headline" id="23rd_September">23rd September</SPAN></H1><H2><SPAN class="mw-headline" id="Plasmid_Prep_2">Plasmid Prep</SPAN></H2><UL><LI> Minipreps of some cells from the previous day
</LI></UL><H2><SPAN class="mw-headline" id="Gel_2">Gel</SPAN></H2><UL><LI> Plasmids from miniprep run on gel.
</LI><LI> Results negative or ambiguous.
</LI></UL><H1><SPAN class="mw-headline" id="24th_September">24th September</SPAN></H1><H2><SPAN class="mw-headline" id="Restriction_Digest_2">Restriction Digest</SPAN></H2><UL><LI> GluR0 plasmid cut (E or E+P) and run on gel.
</LI><LI> Results negative
</LI></UL><H2><SPAN class="mw-headline" id="Plasmid_Prep_3">Plasmid Prep</SPAN></H2><UL><LI> KDP and some OsmY colonies. Run on gel. Best colony selected for continued work.
</LI></UL><H1><SPAN class="mw-headline" id="25th_September">25th September</SPAN></H1><H2><SPAN class="mw-headline" id="Plasmid_Prep_4">Plasmid Prep</SPAN></H2><UL><LI> Miniprep on best cells containing KDP, OsmY and GluR0. 
</LI></UL><H2><SPAN class="mw-headline" id="Restriction_Digest_3">Restriction Digest</SPAN></H2><UL><LI> Plasmids cut with various combinations of enzymes.
</LI><LI> Run on gel, results ambiguous.
</LI></UL><H1><SPAN class="mw-headline" id="26th_September">26th September</SPAN></H1><H2><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H2><UL><LI> Samples PCRed and cleaned, prepared for sequencing.
</LI></UL><DIV style="color: white; font: 2px;">x</DIV></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Cambridge/Voltage/Lab_Work">http://2008.igem.org/Team:Cambridge/Voltage/Lab_Work</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Voltage/Lab_Work" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Voltage/Lab_Work" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Voltage/Lab_Work&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Voltage/Lab_Work&amp;oldid=97882" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>