"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Signalling
From 2008.igem.org
Home   Voltage   Signalling   Bacillus   Modelling   Protocols   People  
Signalling Overview   Vectors   Constructs   Primers   Experiments   Lab Notebook  
x
Using peptide-based signalling systems from gram-positive bacteria, we have laid the foundations for a self-organising biological system, capable of expressing spatial patterns of GFP expression on a bacterial lawn. The focus of our investigation was on a simple two-component Reaction-Diffusion system, allowing for simple spatial 'patterning' of gene expression. The simplest of these patterns, which we have modelled in silico , mimic the spots and stripes seen on animal coats. In 1952, Alan Turing famously described this Reaction-Diffusion system and suggested it as the basis for self-organization and pattern formation in biological systems. This is a first step in the direction of engineering multicellular behaviour.
(A) The model consists of two diffusible signals secreted by every cell. The activator, which is controlled by a stochastic bistable switch, turns on itself and its own inhibitor. (B) A field of cells can be stably patterned into two different zones, so long as the inhibitor diffuses faster than the activator. The activator and inhibitor are synthesized in the source at the center, and turned off by accumulation of the inhibitor in the periphery.
Our system uses two well-characterized bacterial communication systems to generate this behavior.  The agr peptide signalling system from S. aureus serves as our activatory signal (pictured), while the lux system from V. fischeri serves as our inhibitor. Bacillus subtilis serves as an excellent chassis for this project because of the ease with which chromosomal integration can be performed.
Grasshopper Example
Moving flame fronts
The reaction-diffusion system depends on an activator and inhibitory signal that spread throughout the medium. The "grasshopper" example is quite intuitive: Imagine it is hot and there is a field of dry grass with grasshoppers. Suddenly, a fire starts burning at some point and spreads (the activator signal) so that the grasshoppers move away from that point to avoid the fire. However, the grasshoppers also generate moisture (the inhibitory signal) thus preventing the areas of dry grass the grasshoppers move to of catching fire. The initial patch will have burnt down. However, the surrounded area will be saturated by moisture generated by the grasshoppers thus preventing the fire from spreading. Imagine now that at the beginning, not a single patch, but numerous randomly distributed patches (resembling noise) of dry grass catch fire. The resulting patterning of charred grass and grasshoppers is called a Turing Pattern. It is important to note that the inhibitory signal (grasshoppers) must travel faster than the activation signal (fire) as to prevent the whole field from burning down which would result in no patterning at all.
Objective
Schematic of Activator/Inhibitor system in B. subtilis
This sub-project seeks to generate Turing Patterns by creating a Reaction-Diffusion system in the gram-positive bacteria Bacillus subtilis . Our goal was to integrate two signalling systems into this bacterium and use an autofeedback mechanism to generate self-organizing patterns from random noise. We planned to incorporate the agr peptide signalling system from S. aureus and the lux AHL system from V. fisheri.
Materials
Bacillus Strains
Bacillus strain 1A1 (derivative of standard strain 168)
deficient in tryptophan, have to add to media
keep at room temp, aren't freezable
Bacillus strain 1A771 (derivative of standard strain 168)
deficient at tryptophan, have to add to media
keep at room temp, aren't freezable
contains erm insertion at amyE locus, so transformants at amyE locus can be screened for erythromycin resistance
Bacillus strain 1A751 (derivative of standard strain 168)
deficient at tryptophan, have to add to media
keep at room temp, aren't freezable
knockout of Apr and Npr, two major extracellular proteases
Original Publication
Vectors
Details of vectors we used, including maps, sequences, and testing
Primers
List of primers we have ordered
Constructs
Planning and organization of biobrick assemblies and recombinant plasmids
Lab Work Page
Main page detailing our day-to-day research activities
Experiments
Main page detailing our planned and completed experiments
x
Retrieved from " http://2008.igem.org/Team:Cambridge/Signalling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
