"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Protocols
From 2008.igem.org
Home   Voltage   Signalling   Bacillus   Modelling   Protocols   People  
General Protocols   B.subtilis Protocols  
x
Contents
1 Agarose Gel
2 Biobrick Extraction
3 Competent Cells Stocks
4 Dephosphorylation
5 E-Gel
6 Fast Digestion
7 Fast Ligation
8 Flame Photometry
9 Gel DNA Recovery (Zymoclean)
10 Glycerol Stock (Modified Version)
11 Media Preparation
11.1 NOK 2.0 (No potassium nutrient broth)
11.2 Bacillus media
12 Oxygen Electrode
13 Plasmid Miniprep (Zyppy)
14 Primer Preparation and Storage
15 PCR
15.1 Making your own 4x Master Mix
16 E.coli transformation
17 Qiagen QIAquick PCR Purification Kit
18 Bacillus subtilis Transformation
Agarose Gel
Agarose gel
Used for Electrophoresis when extracting any product afterwards.
Making the Gel
Agarose Gel is made by adding the appropriate mass of Ultrapure Agarose (usually 0.8g) to a duran bottle, and then adding 100ml of TAE buffer to the bottle. 10μL of SyBr Green (10000x) can be added if necessary.
The whole is then microwaved at high for 1 minute, stood for 3 minutes and again heated for 1 minute. Boiling should be avoided and the powder should fully dissolve. It should then be left to cool.
The ends of the gel tray must be taped with autoclave tape and firmly sealed.
When the solution is hand hot the gel can be poured. Pipette around 1ml of the solution up each of the taped ends of the gel tray to prevent leakage. Insert the comb and pour the gel to around 6ml.
Leave to set and remove the comb and tape.
Loading the Gel
Mix around 17μL of DNA sample with 3μL of loading dye. The exact amount of DNA varies according to solution strength, but should be around 80ng
We've come to discover that adding 19μL DNA with 1μL loading dye yields crisper bands.
Place the gel tray with gel into the running bay (wells at black end) and add enough TAE buffer to just cover the gel.
Carefully load around 20μL of sample into wells, including some ladder as reference. Normal Agarose gels do not need to have every well filled - there is TAE in them already.
Running the Gel
Connect up the red lead to the red terminals and the black lead to the black terminals of the power supply and the running bay.
Make sure the wells are at the black end.
Set the power supply to 80 volts and switch it on.
Check the gel every 20 mins or so to make sure that the gel is running the right way and that the yellow dye (equivalent to 60bp) has not run to the end of the gel.
When the front dye has reached about 3/4 of the way to the end of the gel turn the power supply off and disconnect the leads.
Visualising the Gel
If no SyBr green has been added the gel needs to be placed in the EtBr solution for 20 mins.
Drain the gel and place in the UV visualiser.
If extracting make sure the prep UV button is pressed, and take a picture of the gel.
To extract the gel slide the light tray out, ensure shielding for all who are involved, and carefully cut out the gel fragment, usind prep UV.
SyBr Green gels can also be visualised using special visualisers.
Biobrick Extraction
Link to list of extracted parts
We found we could not extract enough DNA from the registry, so we are upgrading the extraction to the "bigger, better, faster, stronger" method.
Warm 50μL of EB in Eppendorf tubes at 50°C and add 4 punched spots.
Keep it at 50°C for 20mins
Spin down for 3 minutes at 13,000 g.
Warm for 10mins
Spin down again for 3mins.
Pipette off the supernatant which should contain DNA.
We then confirmed with PCR.
Competent Cells Stocks
Chemically competent cells are made by growing overnight in 200ml LB in a shaking incubator at 37°C. 200ml are spun down at 3800rpm for 8 mins, resuspended in 20ml CaCl2, spun down again and resuspended in 4ml CaCl2. Again spun down (6500 rpm for 5 mins)and resuspended in 4ml 60%glycerol, spun down and resuspended in 2ml 60%glycerol. Finally they are aliquoted (100μL) into stock tubes and flash frozen in liquid nitrogen.
To recover cells thaw the tubes on ice, spin down and resucpend in 100μL ice cold CaCl2.
Electroporation competent TOP10 and DH5α cells were made by growing overnight in 200ml LB in a shaking incubater at 37°C. 100ml were spun down at 3800rpm for 8 mins, resuspended in 25ml SDW, spun down again and resuspended in 2ml SDW. Again they were spun down (6500 rpm for 5 mins)and resuspended in 2ml 60%glycerol, spun down and resuspended in 2ml 60%glycerol. Finally they were aliquoted (100μL) into stock tubes and flash frozen in liquid nitrogen.
To recover cells thaw the tubes on ice, spin down and resucpend in 100μL ice cold SDW.
Dephosphorylation
Add appropriate volume of 10x SAP buffer to restricted fragments.
Add 1ul of Shrimp Alkaline Phosphatase for every pmol of DNA ends. For example, the cut plasmid pSB4C5, which is about 3kb, requires 1 uL of SAP for every ug of vector.
Incubate for 1 hour at 37°C.
Heatkill enzyme for 15 mins at 80°C
E-Gel
Our E-gel machine from invitrogen
E-Gel is used to confirm size of products, but not for extraction.
The E-Gel must be prepared by placing inside the Gel Dock and running the Pre-run for 2 minutes.
The Gel can then be loaded. 19μL of DNA, made up with SDW as necessary, is added to each well, and 1μL of loading dye is also added. Adding less dye allows a more sharply defined band.
All unloaded lanes must be filled with SDW, to ensure correct running.
The gel is then run, using the Run program, for 26 mins.
The gel can then be visualised in the UV visualiser, for EtBr, or using the E-Gel visualiser, for SyBr Green.
Fast Digestion
The following are added to a 1.5μL eppendorf:
2μL plasmid DNA
2μL 10x fast digest buffer
enough water to make up to 19μL (18μL if double digest)
1μL of each fast digest enzyme
If using PCR product the whole is made up to 30μL, using 10μL DNA and 3μL buffer.
The tubes are then incubated at 37°C for 5mins and then the enzyme inactivated at 80°C for 15mins
Fast Ligation
To calculate the amount of insert you need, use the equation:
2CB/AD
A = the length of the vector backbone
B = the length of the insert
C = ng of the backbone you will use (concentration*volume) -> usually between 500ng to 1ug
D = concentration of the insert
The result will be the volume of insert you should add to achieve a 2:1 molar ratio of insert to backbone.
The following are added to a 1.5μL eppendorf:
xμL vector DNA
xμL gene DNA
1.5μL 10x fast digest buffer
1.5μL 10mM ATP
enough water to make up to 15μL
1μL of fast ligase enzyme
The tubes are then incubated at room temperature for 5mins and then the enzyme inactivated at 75°C for 15mins
We have now got a different fast ligation kit.
Now the following are added to a 1.5μL eppendorf:
xμL vector DNA
xμL gene DNA
4μL 15x fast digest buffer
enough water to make up to 19μL
1μL of T4 fast ligase enzyme
The tubes are then incubated at room temperature for 5mins.
Flame Photometry
Chris attempts to unravel the secrets of the Flame Photometer
Set Up
Make sure discharge pipe (on right hand side) is placed in something to collect discharge, and valve is not completely shut.
Check clear tube is connected to compressor OUTLET, and securely attached.
Turn on compressor.
Gauge on back of photometer should read 12psi.
Check gas pipe is attached at tap.
Turn on gas.
Wait 1 minute.
Turn on photometer. It should click, and orange "flame on" light should come on.
Place beaker of distilled water in the photometer tray, and make sure thin tube reaches water.
Wait 15 minutes before taking any readings.
Use "blank" dial to adjust reading to 0 for distilled water.
Taking readings
Place thin tube into sample, far in, as the liquid will be taken up quite rapidly.
Wait for reading to stabilise (reading may continue to fluctuate at high values) and take reading.
Place tube in distilled water beaker between each sample, waiting until reading returns to 0.
In case of compressor blow-out
Turn off gas at tap.
Turn off photometer.
Turn off compressor.
Redo set up steps, with 15 minute waiting time reduced to 5, as the equipment will already be almost up to temperature.
In case of wildly fluctuating "zero" readings
Tap digital display.
Allow tube to dry (take out of SDW) before loading next sample.
Gel DNA Recovery (Zymoclean)
Add 3 parts ADB Buffer to 1 part Gel volume
Incubate 55°C for 5-10 minutes, until gel melts
Place solution into Zymo-Spin column and then into a collection tube
Spin for 30 seconds. Empty collection tube when necessary
Add 200μL Wash Buffer. Spin for 30 seconds. Repeat wash step.
Place column into Eppendorf. Add 6-10μL SDW to elute DNA. Add more SDW to elute more DNA - normally 20μL.
Glycerol Stock (Modified Version)
Inoculate single colony growing on plate into 10 ml LB with appropriate selection.  Grow overnight.
Add 100μL culture to 500μL sterile 80% glycerol stock (need to sterilize using sterile .2μm filter and syringe).
Vortex briefly
Freeze at -80 degrees.
Media Preparation
NOK 2.0 (No potassium nutrient broth)
Measure out:
6.0g    Na2HPO4
3.0g    NaH2PO4
0.5g    NaCl
1.0g    NH4Cl
0.0147g CaCl2
0.246g  MgSO4.7H2O
2.0g    Glucose
1.0g    NOK Amino Acid Powder (see below)
Add to 1dm3 (1l) of SDW and shake well. Note heating 40-50°C may be necessary to achieve full dissolution of solutes. There may be a small amount of indeterminate crud remaining at the end of this process; this is entirely normal and can be ignored. This mixture should then be filter sterilised into 50ml separate tubes (reduces contamination risk of whole batch). Keep refrigerated or frozen until needed.
NOK Amino Acid Powder composition
This is a solid homogenized mixture of all solid amino acids (except possible GluR0 channel agonists alanine, threonine, serine, glycine, glutamine & glutamate.)
Current batch is stored at room temperature on the chemical shelf near the gel area. To make a new batch, measure out 0.5g of each of the following (L-forms ONLY):
Arginine
Asparagine
Aspartic Acid
Cystein
Histidine
Isoleucine
Leucine
Lysine
Methionine
Phenylalanine
Proline
Tryptophan
Tyrosine
Valine
Bacillus media
10X Medium A base:
Yeast extract 10g
Casamino acids 2g
Distilled water to 900mL
Autoclave, then add :
50% glucose, filter sterilized 100mL
10X Bacillus salts:
(NH4)2SO4 20g
Anhydrous K2HPO4 139.7g
KH2PO4 60g
Tri-sodium citrate 10g
MgSO4•7H2O 2g
Water to 1000mL
Medium A
Sterile distilled water 81mL
10X Medium A base 10mL
10X Bacillus salts 9mL
L-Tryptophan (11mg/mL) 0.1mL
Medium B
Medium A 10mL
50mM CaCl2•2H2O 0.1mL
250nM MgCl2•6H2O 0.1mL
Important: Autoclave Medium A base before adding glucose, and autoclave Bacillus salts. Store aliquots of 10X Medium A base 10mL and 10X Bacillus salts 9mL and keep them in the fridge, never use them twice to avoid contamination
Oxygen Electrode
For the protocol see http://www.rankbrothers.co.uk/download/digioxy.pdf
Plasmid Miniprep (Zyppy)
Plasmid Miniprep kit
Add 100μL 7x Lysis Buffer to 600μL Cell Culture. Invert 4 to 6 times
Add 350μL cold Neutralization Buffer. Invert 4 to 6 times
Centrifuge 13,000g for 2 minutes
Transfer supernatant to Zymo-Spin column
Place column in a collection tube and spin for 15 seconds. Discard flow through
Add 200μL Endo-Wash Buffer. Spin for 15 seconds
Add 400μL Zyppy Wash. Spin for 30 seconds.  Repeat this step one more time for better purity.
Transfer column to clean Eppendorf. Add 30 to 100μL Elution Buffer to the column and spin for 15 seconds to elute DNA
Primer Preparation and Storage
Sigma Primers arrived as tubes of dried oligos
Suspend in sterile distilled water (SDW) to form a 100μM stock solution - stored at -20°C
10μM aliquots taken using 10μL stock and 90μL SDW - stored at 4°C
PCR
Our Piko PCR machine provided by Finnzymes
To make 50μL of PCR mix the following were added to Eppendorf tubes:
5μL of DNA in EB buffer
2.5μL of each primer
25μL of 2x Finnzymes mastermix
15μL of sterile distilled water
These values can be adjusted for any volumes of final mixture.
The total final volume is transferred to thin walled tubes (may need to be spun down) and then placed in a PCR machine and the appropriate program run. When using primers designed to produce longer product than original gene touchdown cycles should be run with annealing at the initial binding Tm and then normal cycles at the full length Tm.
Making your own 4x Master Mix
Per 20μl Reaction, Add in this order:
Sterile Distilled Water
20μL - 5.2μL - volume of template and primer DNA (12.8 μL if 2μL of DNA)
4μL of 5x HF buffer
0.4μL of 10mM dNTPs (0.1μL of each)
0.6μL  of 100% DMSO (3% rxn volume)
0.2μL of Phusion Polymerase
E.coli transformation
Plates with transformed cells
First chemically competent cells are made
Cells are grown for 1-2 hours in LB and pelleted (at 4000rpm for 8 min.)
Pellet is resuspended in 1mL 50mM CaCl
Again cells are pelleted.
Pellet is resuspended in 200μL 50mM CaCl
To transform cells
To chilled tubes add 5μL of plasmid DNA + EB solution to 50μL of Cells (Chemically competent cells)
Chill on ice for 30 minutes
Heat shock at 42°C for 60 seconds
Chill on ice for 2 minutes
Add 200μL of SOC
Incubate at 37°C for 2 hours
After transformation cells need to be grown under selection conditions.
Qiagen QIAquick PCR Purification Kit
(from Qiagen QIAquickspin Handbook)
This protocol is designed to purify single- or double-stranded DNA fragments from PCR  and other enzymatic reactions (see page 8). For cleanup of other enzymatic reactions,  follow the protocol as described for PCR samples or use the new MinElute Reaction  Cleanup Kit. Fragments ranging from 100 bp to 10 kb are purified from primers,  nucleotides, polymerases, and salts using QIAquick spin columns in a microcentrifuge.  Notes:
Add ethanol (96–100%) to Buffer PE before use (see bottle label for volume).
All centrifuge steps are at 13,000 rpm (~17,900 x g) in a conventional
tabletop microcentrifuge.
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix. It is not necessary to remove mineral oil or kerosene.
For example, add 500 µl of Buffer PB to 100 µl PCR sample (not including oil).
Place a QIAquick spin column in a provided 2 ml collection tube.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube. Collection tubes are re-used to reduce plastic waste.
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube. Centrifuge the column for an additional 1 min.
IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 µl Buffer EB (10 mM Tris·Cl, pH 8.5) or H2O to the center of the QIAquick membrane and centrifuge the column for 1 min. Alternatively, for increased DNA concentration, add 30 µl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge.
IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volume is 48 µl from 50 µl elution buffer volume, and 28 µl from 30 µl elution buffer.
Elution efficiency is dependent on pH. The maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C as DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions.
Bacillus subtilis Transformation
Transformation Procotols for B. subtilis can be found here: Bacillus subtilis Transformation Protocols
x
Retrieved from " http://2008.igem.org/Team:Cambridge/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
