<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Birkbeck_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Birkbeck/Composite Part</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="cssmenu"><UL><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/TeamMembers"><SPAN>Members</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Attributions"><SPAN>Attributions</SPAN></A></LI></UL><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Research"><SPAN>Research</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Description"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/pp"><SPAN>Background</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Experiments"><SPAN>Experiments</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Measurement"><SPAN>Measurements</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Modeling"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Results"><SPAN>Results</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Conclusion"><SPAN>Conclusion</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/BioBricks"><SPAN>BioBricks</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Basic_Parts"><SPAN>Basic Parts</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Composite_Part"><SPAN>Composite Parts</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Outreach"><SPAN>Outreach</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Practices"><SPAN>Human Practices</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Collaborations"><SPAN>Collaborations</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/InterLab Study"><SPAN>InterLab</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Notebook"><SPAN>Journal</SPAN></A></LI></DIV><DIV id="maincontainer"><H1>Our BioBricks</H1><H2>Composite Parts</H2><H3><B>tfa (tail fibre assembly) gene circuit (BBa_K1846001)</B></H3><P>This part provides the gene sequence for tfa (tail fibre assembly) protein, together with a TetR repressible promoter (<A href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A>), ribosome binding site (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034) </A>and an rrNB T1 terminator (<A href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A>).
</P><P>The tfa (tail fibre assembly) protein of bacteriophage lambda assists in the assembly of the stf (short tail fibre) protein into a functional short tail fibre [1],[2]. Tfa gene displays a high level of homology (~40%) with the gp38 of bacteriophage T4 [2],[3],[4].

</P><P>This tfa circuit was synthesised removing the illegal restriction sites to make the sequence BioBrick-compatible. The sequence was cloned into a pSB1C3 and the success of the cloning procedure was confirmed by restriction with <I>EcoRI</I> and <I>SpeI</I> restriction enzymes and followed by agarose gel electrophoresis (Figure 1). The construct was also confirmed by sequencing. This BioBrick was registered as part <A href="http://parts.igem.org/Part:BBa_K1846001">BBa_K1846001</A>.</P><H3><B>TetR circuit (BBa_K1846003)</B></H3><P>This is a constitutive part for the production of the tetracyline repressor TetR, using a constitutive, medium-copy chloramphenicol promoter (<A href="http://parts.igem.org/Part:BBa_I14033">BBa_I14033</A>), ribosome binding sequence (<A href="http://parts.igem.org/Part:BBa_B0031">BBa_B0031</A>) and double terminator (rrnb T1 terminator followed by a T7Te terminator, <A href="http://parts.igem.org/Part:BBa_B0010">BBa_B0010</A>).</P><P>TetR generator cassette (<A href="http://parts.igem.org/Part:BBa_P0140">BBa_P0140</A>) and P(Cat) promoter (<A href="http://parts.igem.org/Part:BBa_I14033">BBa_I14033</A>) were obtained from 2015 iGEM distribution kit  and cloned into a single pSB1C3 vector. Correct cloning of the parts into the pSB1C3 shipping vector was confirmed by agarose gel electrophoresis (Figure 2) and Sanger sequencing. This BioBrick has been registered as part <A href="http://parts.igem.org/Part:BBa_K1846003">BBa_K1846003</A>.

</P><H3><B>TetR-regulated tfa (tail fibre assembly) circuit (BBa_K1846007)</B></H3><P>A circuit controlling the production of the tail fibre assembly (tfa) protein of bacteriophage lambda to prevent toxicity to the cell. The tfa gene operates under a TetR repressible promoter, while a second circuit (<A href="http://parts.igem.org/Part:BBa_K1846003">BBa_K1846003</A>) produces TetR (tetracycline repressor) under control of a P(Cat) promoter. With the production of the tfa protein under control of a Tet-repressible promoter, the tfa coding sequence will not be expressed due to P(TetR). However, on addition of anhydrous tetracycline (aTc), TetR will preferably bind to aTc molecule, alleviating repression of P(TetR) and allowing the initiation of transcription. The correct cloning of this BioBrick was confirmed through Sanger sequencing and agarose gel electrophoresis (Figure 3).
</P><P>We characterised this construct by analysing the soluble protein fraction of the cell lysate (Figures 4 and 5). Under the TetR regulation, <I>tfa</I> gene is expressed under a tight control, and can be visualised on the gel (tfa protein = 194 aa, MW: 21602.2 Da), (Figure 4, samples 7, 8, 9 and 11, 12 and 13; Figure 5, samples 4, 5 and 6). <I>tfa</I> gene circuit on its own does not appear to produce tfa protein bands (Figure 4, samples 1, 2 and 3). Although the literature suggests that the protein is soluble even with high expression [1], we hypothesise the lack of bands in these samples is due to the protein being highly acidic, and thus toxic to the cell. It may be that due to the toxicity the cells that express less of the protein are selected for in these experimental conditions. Under the regulation of TetR, even when induced, the expression of the <I>tfa</I> gene is considerably lower than when unregulated and hence, we hypothesise, a considerable amount of soluble material can be recovered. The TetR repressor contains an LVA tag for rapid degradation and hence in the uninduced samples 7 and 11 in Figure 4, and sample 4 in Figure 5, tfa protein bands can still be observed.</P><H3><B>cI-Cro circuit (BBa_K1846005)</B></H3><P>The cI-Cro construct contains a circuit for the production - via a constitutive promoter - of the cI repressor protein. The circuit uses a T7 promoter to drive the expression of the <I>cro</I> gene in the opposite direction to the <I>cI</I> gene, essentially silencing the expression of the <I>cI</I> gene. The strength of the promoter used means that in the presence of T7 DNA polymerase, production of the Cro repressor protein will incapacitate the production of the cI repressor protein. Thus, the presence of the T7 RNA polymerase in the system enables the switch from the lysogenic cycle to the lytic cycle.</P><P>The cI repressor protein (also known as lambda repressor) is responsible for keeping bacteriophage lambda in the lysogenic cycle through the cooperative binding of two repressor dimers to the DNA, repressing the expression of the <I>cro</I> gene [5],[6]. Cro repressor protein, on the other hand, controls the late stage of the lytic development. When the concentration of the Cro repressor is greater than that of the cI repressor, the lytic cycle is initiated [5],[6].
</P><P>This cI-Cro circuit was synthesised removing the illegal restriction sites to make the sequence BioBrick-compatible. The sequence was cloned into a pSB1C3 and the success of the cloning procedure was confirmed by restriction with <I>EcoRI</I> and <I>SpeI</I> restriction enzymes and followed by agarose gel electrophoresis (Figure 6). The construct was also confirmed by sequencing. This BioBrick was registered as part <A href="http://parts.igem.org/Part:BBa_K1846005">BBa_K1846005</A>.
</P><P>We characterised this construct by analysing the soluble protein fraction of the cell lysate (Figure 7). As cI repressor protein is produced constitutively in our circuit, we expected to see a cI repressor protein band at ca 26 kDa (cI repressor protein = 237 aa, MW: 26211.8 Da). Our control <I>E.coli</I> 10Î² cells containing the cI-Cro construct have shown the expected bands at ca 26 kDa (Figure 7, samples 7, 8 and 9). We attribute the lack of cI protein bands in our T7 Express cell lysate (Figure 7, sample 4) to T7 RNAP leakage, which would silence the expression of the <I>cI</I> gene through expression of <I>cro</I> gene in the opposite direction. Absence of Cro repressor protein bands (Cro repressor protein = 66 aa; MW: 7363.4 Da) could be due to protein overexpression and aggregation into insoluble fraction (due to the strength of the T7 promoter). This hypothesis requires further investigation, however our preliminary results suggest the circuit would be functional.

Previously existing BioBricks or parts containing either cI or Cro repressors include: <A href="http://parts.igem.org/Part:BBa_K150001">BBa_K150001</A>; <A href="http://parts.igem.org/Part:BBa_K150004">BBa_K150004</A>; <A href="http://parts.igem.org/Part:BBa_K1195004">BBa_K1195004</A>; <A href="http://parts.igem.org/Part:BBa_I741111">BBa_I741111</A>; <A href="http://parts.igem.org/Part:BBa_K177009">BBa_K177009</A>; <A href="http://parts.igem.org/Part:BBa_K648028">BBa_K648028</A>; <A href="http://parts.igem.org/Part:BBa_K648128">BBa_K648128</A>; <A href="http://parts.igem.org/Part:BBa_K648132">BBa_K648132</A>.
</P><P>[1] Hashemolhosseini, S., Stierhof, Y. D., Hindennach, I., &amp; Henning, U. (1996). Characterization of the helper proteins for the assembly of tail fibers of coliphages T4 and Î». Journal of Bacteriology, 178(21), 6258â6265.</P><P>[2] Montag, D., Schwarz, H., &amp; Henning, U. (1989). A component of the side tail fiber of E. coli bacteriophage Î» can functionally replace the receptor-recognizing part of a long tail fiber protein of the unrelated bacteriophage T4. Journal of Bacteriology, 171(8), 4378â4384.</P><P>[3] Hendrix, R. W., &amp; Duda, R. L. (1992). Bacteriophage lambda PaPa: not the mother of all lambda phages. Science (New York, N.Y.), 258(5085), 1145â1148. doi:10.1126/science.1439823</P><P>[4] Haggard-Ljungquist, E., Halling, C., &amp; Calendar, R. (1992). DNA sequences of the tail fiber genes of bacteriophage P2: Evidence for horizontal transfer of tail fiber genes among unrelated bacteriophages. Journal of Bacteriology, 174(5), 1462â1477.</P><P>[5] Echols, H., &amp; Murialdo, H. (1978). Genetic map of bacteriophage lambda. Microbiological Reviews, 42(3), 577â591.</P><P>[6] Friedman, D. I., &amp; Court, D. L. (2001). Bacteriophage lambda: Alive and well and still doing its thing. Current Opinion in Microbiology, 4(2), 201â207. doi:10.1016/S1369-5274(00)00189-2</P><LI><A href="https://2015.igem.org/Team:Birkbeck/BioBricks">Return to BioBricks overview</A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Basic_Parts">Have a look at our Basic Parts</A></LI></DIV></DIV></DIV></DIV></DIV></BODY></HTML>