Team:Birkbeck/Experiments
Team
Members
Attributions
Research
Overview
Background
Experiments
Measurements
Modelling
Results
Conclusion
Safety
BioBricks
Basic Parts
Composite Parts
Outreach
Human Practices
Collaborations
InterLab
Journal
jQuery UI Accordion - Collapse content
Experiments and Protocols
Growth Curve Protocol
50 mL Culture Flasks
In order to investigate the best time to infect E. coli with a recombinant λ-bacteriophage containing a detectable signal, the growth kinetics of E. coli in LB media must be characterized. In this set of experiments, the main aim is to gage when cultures are entering particular stages of the growth curve (refer to figure 1). In conjunction with measuring the optical density (OD) of cultures at a wavelength of 600 nm (OD600), the level of background noise of potential signal molecules was also investigated. For instance GFP & RFP have excitation peaks at 395/475/501 nm and 555 nm respectively. We investigated if we could detect the expression of gfp by absorption at low culture densities.
Figure 1: Schematic of Microbial Growth Curve. After inoculation, microbial cells adapt to the new environment (lag phase). After ample time to adapt, the microbe starts to divide rapidly with the shortest doubling time (log phase). As nutrients become depleted, cells are unable to divide and hence the culture remains "stationary" with regards to growth.
The data from the growth curves also gives scope for when to induce the “payload” circuit. At different time points in the growth curve, anhydrotetracycline can be added in order to relieve repression of mrfp expression. Interesting time points for investigating signal intensity & detection limits are; the lag, mid log & stationary phase. The time point which yields the greatest signal & lowest detection limit will be considered for infection of E. coli cultures to engineered recombinant λ-bacteriophage.
Day 1
The glycerol stock of E. coli cells was streaked out on a LB agar plate. Glycerol stock cells that bear a plasmid had to be selected for by supplementing the LB agar plates with the appropriate antibiotic at the MIC.  Plates were incubated overnight at 37oC overnight.
Day 2
Three single colony were picked from each plate and used to inoculate three 10 mL of LB broth. LB broth was supplemented with antibiotics if required. Cultures were grown overnight shaking at a rate of 180 rpm at 37oC.
Day 3
Cultures were diluted back to an OD600 of ~ 0.4 in sterile and preheated LB broth. The diluted culture was used as a seed inoculum. A 1% inoculum was carried out in 50 mL of preheated LB broth. The initial OD600 was taken after inoculation to represent the starting point of the growth curve.
The cultures were left to grow for 1 hour with the OD600, OD475, OD395 & OD555. After 1 hour, culture ODs were measured at 20 minute intervals.  At time points 60 minutes & 175 minutes, a viable count was taken. A dilution series ranging from 10-1 – 10-7 was carried out. 3 Χ 20 μL of each dilution was spotted on separate quadrants of the LB agar plate.
The OD of the cultures was monitored for 5 hours. A final reading of the culture OD was carried out at 24 hours. If delays in growth of cells under certain treatments are observed, the phenomenon can be further investigated by ethanol fixing cells and assessing the cellular lengths. Results can be reviewed InterLab Study Additional Results .
96-well Microtitre Plate Growth Curve.
Exactly the same preparation was carried out for the first 2 days with exception to 9 colonies being picked instead of 3. When it came to inoculation, an overnight culture was used to inoculate 200 μL of LB broth (1% inoculum). The general set up of the microtitre plate is displayed in Fig. 2.
Fig. 2: 96-Well Microtitre Plate Set Up for Growth Curve. The "BLANK" in row D was averaged & subtracted from each culture at different time points throughout the growth curve. The contents of each well is labelled; DH1-9 - E. coli DH5α, Pos1-9 - E. coli DH5α containing interlab study positive control gfp expression device, P1-1-9 - E. coli DH5α containing P1-gfp expression device & P2-1-9 E. coli DH5α containing P2-gfp expression device.
Microtitre plates were sealed with adhesive tabe and placed in a BMG-LabTech SPECTROstar. An initial reading was taken with readings being take at intervals of 20 minutes for 580 minutes. Shaking was set at 200 rpm with an incubation temperature of 37oC.
BioBrick Assembly 1: Assembly of the stf gene Protocol
For the assembly of the bacteriophage Lambda tail fibre protein (stf gene), our initial step was the design of ORF314. This sequence was optimised to remove any illegal restriction sites before having the entire gene synthesised. This sequence was initially cloned into both the linearised shipping plasmid pSB1C3 to create our first BioBrick (see details here ) and into the linearised assembly plasmid pSB1K3 for further processing, using the standard iGEM restriction protocol with EcoRI and PstI enzymes, followed by ligation using T4 DNA ligase as per the standard iGEM protocol. Following purification of the plasmids using the Thermo Scientific GeneJET plasmid miniprep kit, the part size was confirmed by setting up a 10 μL restriction with EcoRI and PstI, followed by agarose gel electrophoresis (15 W, 85 mA, 150 V (limiting factor)) (Fig 1).
Fig 1. Agarose gel electrophoresis to confirm presence of BioBrick in shipping plasmid.
A 6x His-tag was subsequently added to the construct within the assembly backbone (Fig. 2) by carrying out site-directed mutagenesis to introduce restriction sites for the Type 2 restriction enzyme BsaI before the stop codon of the ORF314 sequence (Fig. 3). The construct was then restricted with this enzyme before being annealed and ligated with a pair of oligonucleotides with compatible overhangs, which contained the His-tag sequence replacement BioBrick suffix sequences.
Fig 2. Synthesised oligonucleotides containing a His-tag and BioBrick suffix.
Fig 3. Construct containing Assembly plasmid pSB1K3 and ORF314 (a) showing location of primers creating BsaI cut sites, and (b) after restriction-ligation.
After addition of the His-tag, both the recombinant pSB1K3 plasmid and our synthesised ORF401 gene were restricted with the blunt-end restriction enzyme EcoRV and the sticky-end enzyme EcoRI as per the standard iGEM protocol, in order to create incompatible ends. The two products were combined and ligated with T4 DNA ligase – again according to standard iGEM protocol – before agarose gel electrophoresis was carried out to determine the success of the restriction-ligation. Results can be viewed here .
What's next
To create the complete stf gene, we will now carry out 3A assembly. We will restrict the above construct with EcoRV and PstI restriction enzymes to remove the ORF314 gene including the 6x His-tag. Consecutively, we will restrict the linearised backbone pSB1C3 with EcoRI and PstI and the previously synthesised sequence ORF401 will be restricted with EcoRI and EcoRV. As such, we will create three fragments that can only be combined in a single combination, which we will anneal and ligate using T4 DNA ligase as per the approved iGEM protocol. We will then transform these recombinant plasmids into 10β competent cells and carry out antibiotic selection to exclude any recombinants containing the pSB1K3 backbone. Successful colonies will be grown in overnight cultures, before a plasmid miniprep (Thermo Scientific GeneJET) will be carried out to purify the plasmids. The correctness of the construct will be confirmed both by carrying out single and double restriction of the plasmid before agarose gel electrophoresis, and through Sanger sequencing. On confirmation of the sequence, the construct will be submitted as a BioBrick.
Additionally, the stf construct will be cloned into a previously generated vector containing a promoter, ribosome binding site and terminator to facilitate characterisation of the part, as well as to create a new composite part.
Preparing Chemically Competent Cells
1.  Grow 5 mL overnight culture in LB broth.
2.  Sub-culture (1% inoculum) in 25 mL of LB.
3.  Grow until OD600 = 0.3-0.5 (DH5a, this is around 2 hours).
4.  Put on ice for 20 mins (can be longer, not an essential step).
5.  Spin cells down at 4oC at 4,500 rpm for 20 mins.
6.  Resuspend in 2.5 mL of ice cold TSB.
7.  Leave on ice for at least 10 mins.
8.  Snap freeze in 50 uL aliquots to -80oC or use immediately.
Chemical Transformation Protocol
1.  Thaw out the 50 µL cell suspension aliquot on ice.
2.  Add 100pg-1µg of plasmid DNA to the cell suspension (approximately 1-5 µL of plasmid solution).
3.  Mix plasmid and cells thoroughly and incubate on ice for 30 minutes.
4.  Induce heat shock by incubating cells at 42oC for 30 seconds.
5.  Incubate on ice for 5 minutes.
6.  Add 450 µL of LB and incubate at 37oC for 1 hour.
7.  Plate out 50 µL of straight transformant cells & also carry out a 10-fold dilution (plate out 50µL). Note that the plate should contain the appropriate antibiotic at the MBC in order to select for transformants which contain the desired plasmid.
8.  Incubate overnnight at 37oC in a static incubator.
Creating Electrocompetent Cells
Step 1 - Overnight Culture Preparation
1.  Transfer 10 mL of sterile LB to a 50 mL Falcon Tube.
2.  Inoculate media with a single colony of E. coli cells.
3.  Incubate overnight at 37°C, in a shaker at 230 rpm.
Step 2 - Preparing Electrocompetent E. coli Cells for Storage.
1.  Add 150-200 mL of LB media to the autoclaved 2 L conical (NB/ culture volume must be 1/10 of the total flask volume).
2.  Use a 0.5% inoculum from step 1.
3.  Incubate at 30°C until OD600 ~0.6 (approximately 4 hours).
4.  When culture reaches OD600 = 0.6, cool to 4°C
Step 3 (after culture OD600 ~ 0.6 has been achieved)
1.  Aliquot 50 mL of culture into Falcon tubes. Incubate on ice
2.  Pre-cool centrifuge to 4°C
3.  Centrifuge the tubes for 10 mins at 4°C at 4000 rpm.
4.  Decant supernatant.
5.  Wash 1: resuspend pellet in 10 mL of 1 mM Hepes buffer (pH 7). Incubate on ice for 5 mins. Gently disturb tube every 30 seconds to facilitate resuspension of the pellet.
6.  Centrifuge for 10 mins at 4°C at 4000 rpm
7.  Decant supernatant & resuspend pellet in 10 mL of 1 mM Hepes buffer (pH 7). Incubate on ice for 5 minutes. Gently disturb tube every 30 seconds to facilitate resuspension of the pellet.
8.  Add two of the Falcon tubes together when cells are fully resuspended (giving two 20 mL cell suspensions). Top each cell suspension up to 50 mL with 1 mM Hepes (pH 7).
9.  centrifuge again for 10 mins at 4°C at 4000 rpm.
10. Decant the supernatant & resuspend the pellet in 10 mL Hepes (pH 7). Incubate on ice for 5 minutes.
11. centrifuge again for 10 mins at 4°C at 4000 rpm.
12. Decant the supernatant & resuspend the pellet in 2 mL of Hepes (pH 7). Incubate on ice for 5 minutes.
13. Add glycerol to a final concentration of 12.5% (vol/vol). Mix thoroughly (but GENTLY) & incubate on ice for 5 minutes.
14. Aliquot 50 µL of cells suspension into 500 µL tubes and store at -80oC.
Back to experiments and protocols
Transformation of Plasmids into Electrocompetent Cells
For more details about making electrocompetent cells, please see our Making electrocompetent E.coli cells protocol .
Electroporation protocol.
1.  Thaw out the 50 µL cell suspension aliquot on ice for ca. 30 minutes. Concurrently, cool electroporation tubes on ice.
2.  Add 100pg-1µg of plasmid DNA to the cell suspension (approximately 1-5 µL of plasmid solution).
3.  Mix plasmid and cells thoroughly and continue to incubate on ice for 5-10 minutes.
4.  Transfer the mixture of cells and plasmids to an electroporation tube.
5.  Electroporate cells; for E.coli, this is done at 2.5 kV, 200 Ohms, 25 uF.
6.  Immediately add 450-950 uL of LB broth.
7.  Incubate tubes at 37oC for 60-90 minutes in a statis incubator.
8.  Plate 100 uL of cells onto agar plates with appropriate antibiotic resistance.
9.  Incubate overnnight at 37oC in a static incubator.
To find measurements & statistical analysis
Back to main Research page
