<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Birkbeck_Discussion skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Birkbeck/Discussion</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="cssmenu"><UL><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/TeamMembers"><SPAN>Members</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Attributions"><SPAN>Attributions</SPAN></A></LI></UL><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Research"><SPAN>Research</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Description"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/pp"><SPAN>Background</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Experiments"><SPAN>Experiments</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Measurement"><SPAN>Measurements</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Modeling"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Results"><SPAN>Results</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Conclusion"><SPAN>Conclusion</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/BioBricks"><SPAN>BioBricks</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Basic_Parts"><SPAN>Basic Parts</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Composite_Part"><SPAN>Composite Parts</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Outreach"><SPAN>Outreach</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Practices"><SPAN>Human Practices</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Collaborations"><SPAN>Collaborations</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/InterLab Study"><SPAN>InterLab</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Notebook"><SPAN>Journal</SPAN></A></LI></DIV><DIV id="maincontainer"><H2><U><B>Discussion</B></U></H2><H3><B><U>Signal Detection</U></B></H3><P>Considering our results, at time 0 minutes, there is a significant signal for P1-<I>gfp</I> expression device compared to the <I>E. coli</I> DH5α (P=0.005, with reference to <B>Fig. 9</B> in results section). This result cannot be regarded as this is probably carry over fluorescence from sub-culturing.  Photo-bleaching prior to the first reading may yield more accurate readings when considering the expression of <I>gfp</I> from promoters. Also the error bars in each of the expression devices do appear excessive. A means of reducing this may be to sub-culture the cells &amp; grow to an OD<SUB>600</SUB> of 0.2 and sub-culturing into experimental cultures in order to minimise the error bars.</P><P>One technique which has been developed is transcription-reverse transcription concerted reaction (TCR) (Ishiguro <I>et al</I>., 1996). TCR monitors transcription in a sequence-dependant manner in <I>in-vitro</I> systems (Ishiguro <I>et al</I>., 1996). The TCR protocol was adapted in order to improve sensitivity for detecting <I>M. tuberculosis</I> from clinical sputum samples (Drouillon <I>et al</I>., 2009 [TCR-2]). Considering P1-<I>gfp</I> culture at 20 minutes, there is a significant fluorescent signal (P=0.002) with no difference between the OD<SUB>600</SUB> growth curves at this point (P=1).  This is significantly less than the hours taken for results to be generated by TCR-2 (Drouillon <I>et al</I>., 2009). Cells cannot be quantified at the 20 minutes time point in our study as no viable count was carried out, therefore the detection limit of the <I>E. coli</I> DH5α with regards to fluorescence cannot be determined.</P><P>With regards to <A href="https://catalog.hardydiagnostics.com/cp_prod/Content/hugo/TBPrepKit.htm">Decontamination Kits</A>, samples would take just under one hour to process. Of note, some bacteria (such as <I>Staphylococcus aureus</I>) can be resistant to decontamination (refer to the kit previously mentioned). A key point to note here is that the decontamination step must not be carried out for <EM>longer than 15 minutes</EM> due to the reduction in cell viability of <I>Mycobacterium</I> cells.</P><P>As transduction was not characterised in this study, it is difficult to assess exactly how long this step may take. Considering the application of D29 mycobacteriophage could be spotted directly onto soft agar plates giving rise to plaques on both <I>M. smegmatis</I>&amp;<I>M. tuberculosis</I> (Sampson <I>et al</I>., 2009). A mycobacteriophage assay for detection of live <I>Mycobacterium</I> cells has previously been carried out using a plaque assay (Alcaide <I>et al</I>., 2003). A 1 mL decontaminated sample was washed and grown overnight in the standard <I>Mycobacterium</I> media <B>Middlebrook 7H9</B> supplemented with 10% (vol/vol) oleic acid-albumin-dextrose catalase (OADC) (Alcaide <I>et al</I>., 2003). 100 μL of mycobacteriophage was added to the overnight cultures and incubated at 37<SUP>o</SUP>C for 1 hour before adding 100 μL of viricidal solution (Alcaide <I>et al</I>., 2003). A soft agar lawn was poured and incubated for 24 hours before counting plaques. The use of a fluorescent protein as a tag would perhaps remove the need for overnight incubation (in fact a 48 hour protocol).</P><P>Alcaide &amp; co-workers did demonstrate that smear positive samples were extremely sensitive to the phage assay and claim that the detection limit was 10 cfu/mL with smear negative samples showing much higher detection limits. Another study using mycobacteriophages as a diagnostic tool used the cheaper medium Mueller–Hinton instead of Middlebrook 7H9 (Hemvani <I>et al</I>., 2012). The Hemvani study showed similar results to the Alcaide study. The change in media did however result in 3.5% of sputum samples being un-interpretable due to high levels of contamination even after treatment with vancomycin &amp; polymyxin B (Hemvani <I>et al</I>., 2012).</P><P>Considering the presorption phase of the assay developed by Alcaide <I>et al</I>, this would add an hour onto the protocol. Given that transduction is 100% efficient (unlikely) and that the kinetics of <I>gfp</I> expression behaves the same in <I>E. coli</I> DH5α as does a <I>Mycobacterium</I> sp. cell (another wild assumption), we are looking at from sputum sample collection until signal detection at approximately 2-4 hours (given that decontaminated sputum samples could be directly exposed to recombinant bacteriophages). It must be noted that in our approach, the signal is produced by transduction of genetic material as opposed to amplification of phage particles being the signal. The use of transduction hass been successfully applied in genetic engineering of <I>Mycobacterium</I> sp. (Jain <I>et al</I>., 2014; Tufariello <I>et al</I>., 2014) which is promising for the development of a diagnostic tool in introducing a detectable &quot;payload&quot; for diagnostic purposes.</P><P>Considering our results, at 1 hour of culturing <I>E. coli</I> DH5α, there is a significant signal observed from the culture expressing the P1-<I>gfp</I> expression device (P&lt;0.001). Considering the growth, there was no significant difference between the OD<SUB>600</SUB> of the P1-<I>gfp</I> expression device &amp; the <I>E. coli</I> without plasmid (P=1 for both 50 mL cultures &amp; 200 μL, with reference to <B>Fig. 1</B>&amp;<B>Fig. 5</B> respectively). At the 1 hour time point, there is a significant difference between the 50 mL &amp; 200 μL cultures (P=0.017) with the 200 μL cultures showing a higher OD<SUB>600</SUB>. This entails that the fluorescence of the 200 μL cultures at 1 hour corresponds to a higher culture density than 2.57×10<SUP>6</SUP> cfu/mL.</P><P>Considering the TCR-2 method of detection, 2.57×10<SUP>6</SUP> cfu/mL is an increase in detection limit by a magnitude of 10<SUP>5</SUP> (30 cfu of <I>M. tuberculosis</I>/mL of sputum [Drouillon <I>et al</I>., 2009]) which is clearly an undesirable due to the slow growth of <I>M. tuberculosis</I>. There appears to be a reduction in fluorescence from the P1-<I>gfp</I> expression device after 20 minutes which might correspond to the depletion of mRNA/GFP. The increase in fluorescence when comparing the 20 &amp; 40 minute time points may be the expression of <I>gfp</I> from each of the promoter increasing to a detectable level.</P><P>An interesting observation was made on the growth kinetics of the <I>E. coli</I> DH5α containing P1-<I>gfp</I> expression device &amp;<I>E. coli</I> DH5α not containing any plasmid. Between 100-400 minutes, the <I>E. coli</I> DH5α containing P1-<I>gfp</I> expression device has a significantly lower OD<SUB>601</SUB>&amp; higher fluorescence when compared to <I>E. coli</I> DH5α. The significant difference is not observed in the positive control or P2-<I>gfp</I> expression device with limited detection of fluorescence. This data suggests that compromising cellular growth with regard to expression of a signal may yield a quicker detection of a signal with the product.</P><P>It is obvious that the P1 promoter will not be able to drive gene expression in all kinds of bacteria. The proposed T7-RNAP-driven <I>rfp</I> expression was not characterised as the circuit was not constructed due to cloning issues. However, the growth kinetics would be expected to be slower than that of the P1-<I>gfp</I> expression device. <B>Fig. 1</B> is a diagramatic representation of the proposed genetic circuit for the characterization of a signal.</P><P><B>Fig. 1: TetR <I>rfp</I> Expression device</B>.</P><P>The idea behind the circuit displayed in <B>Fig. 1</B> is to have the amplification of <I>rfp</I> expression inducible. The induced expression of the amplification step in this circuit would mimic the infection of the <I>E. coli</I> cells by a recombinant λ-bacteriophage. This would allow different time points in the growth curve to be stimulated and signal intensities could be compared relative to the cell dose &amp; physiological state of cells.</P><P>In the first part of the circuit displayed in <B>Fig. 1</B>, a strong constituitive promoter (such as <A href="http://parts.igem.org/Part:BBa_I14033">pCAT) will drive the expression of </A><A href="http://parts.igem.org/Part:BBa_P0140"><I>tetR</I></A>. The TetR will bind to <A href="http://parts.igem.org/Part:BBa_I14015"><I>tetO</I>&amp; repress the expression of <I>T7-RNAP</I>. The T7-RNAP would then drive the exmpreesion of </A><A href="http://parts.igem.org/Part:BBa_E1010">rfp</A> via <A href="http://parts.igem.org/Part:BBa_K525998">T7 promoter</A>.</P><P>The <A href="http://parts.igem.org/Part:BBa_E1010">highly mutated <I>rfp</I></A> was chosen as a marker due to the ability of the protein to be observed on a plate as well as liquid culture without the use of fluorescence. The absorption of GFP could not be detected accurately (see <B>Fig. 2</B>&amp;<B>Fig. 6-8</B> in the additional <A href="https://2015.igem.org/Team:Birkbeck/Results">Interlab Results</A>. This approach can't be applied to detecting an early signal however, left on a plate can yield very suggestive &amp; qualitative results.</P><P>Another proposed way in which to characterize this circuit was to clone into a <I>gt 11</I> λ-bacteriophage vector. with reference to <B>Fig. 1</B>, the right hand side of the line down the circuit (i.e. from the <I>tetO</I>) would be cloned into the <I>gt 11</I> vector. There could be many possible time points to assess what part of the growth curve would yield the highest signal. </P><H4><U>Reference</U></H4><H5><U>Signal Detection</U></H5><LI><B>Alcaide F., Gali N., Dominguez J., Berlanga P., Blanco S., Orus P. &amp; Martin R.</B> (2003). Usefulness of a New Mycobacteriophage-Based Technique for Rapid Diagnosis of Pulmonary Tuberculosis. <I>Journal of Clinical Microbiology</I> Vol. 41, No. 7, p2867-2871.</LI><LI><B>Drouillon V., Deloogu G., Dettori G., Lagrange P.H., Benecchi M., Houriez F., Baroli K., Ardito F., Torelli R., Chezzi C., Fadda G. &amp; Herrman J-L.</B> (2009). Multicenter Evaluation of a Transcription-Reverse Transcription Concerted Assay for Rapid Detection of <I>Mycobacterium tuberculosis</I> Complex in Clinical Specimens. <I>Journal of Clinical Microbiology</I>, Vol. 47, p3461-3465.</LI><LI><B>Hemvani N., Patidar V. &amp;  Chitnis D.S.</B> (2012). A simple and economical in-house phage technique for the rapid detection of rifampin, isoniazid, ethambutol, streptomycin, and ciprofloxacin drug resistance in <I>Mycobacterium tuberculosis</I>, directly on decontaminated sputum samples. <I>International Journal of Infectious Diseases</I>, Vol. 16, p232-236</LI><LI><B>Ishiguro T., Saitoh J., Yawata H., Otsuka M., Inoue T. &amp; Sugiura Y.</B> (1996). Fluorescence Detection of Specific Sequence of Nucleic Acids by Oxazole Yellow-Linked Oligonucleotides. Homogeneous Quantitative Monitoring of <I>in-vitro</I> Transcription. <I>Nucleic Acids Research</I>, Vol. 24, No. 24, p4992-4997</LI><LI><B>Jain P., Hsu T., Arai M., Biermann K., Thaler D.S., Nguyen A., González P.A., Tufariello J.M., Kriakov J., Chen B., Larsen M.H. &amp; Jacobs W.R. Jr.</B> (2014). Specialized Transduction Designed for Precise High-Throughput Unmarked Deletions in <I>Mycobacterium tuberculosis</I>. <I>American Society for Microbiology</I>, Vol. 5, No. 3</LI><LI><B>Sampson T., Broussard G.W., Marinelli L.J., Jacobs-Sera D., Ray M., Ko C-C., Russell D., Hendrix R.W. &amp; and Graham F. Hatfull G.F.</B> (2009). Mycobacteriophages BPs, Angel and Halo: Comparative Genomics Reveals a Novel Class of Ultra-Small Mobile Genetic Elements. <I>Microbiology</I>, Vol. 155, p2962-2977</LI><LI><B>Tufarielloa J.M., Maleka A.A., Vilchèzea C., Colea L.E., Ratnera H.K, Pablo A. Gonzáleza P.A., Paras Jaina P., Hatfulle G.F., Larsena M.H. &amp; William R. Jacobs W.</B>(2014). Enhanced Specialized Transduction Using Recombineering in <I>Mycobacterium tuberculosis</I>. <I>American Society for Microbiology</I>, Vol. 5, No. 3, p1-7</LI><P><B>Sean Ross Craig</B> (data analysis, cloning, restriction diagnostics, measurements &amp; uploading content to the wiki), <B>Elliott Parris</B> (measurements &amp; restriction diagnostics), <B>Rachel Wellman</B> (restriction diagnostics &amp; measurements) &amp;<B>Ariana Mirzarafie-Ahi</B> (cloning).</P><P>With thanks to <B>Dr. Vitor Pinheiro</B>, <B>Dr. Jane Nicklin</B>, <B>Bilkis Kazi</B>, <B>Barbara &quot;<I>Babz</I>&quot; Steijl</B>, <B>Luba &quot;<I>Aunty</I>&quot; Prout</B>.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>