Team:Birkbeck/Modeling
Team
Members
Attributions
Research
Overview
Background
Experiments
Measurements
Modelling
Results
Conclusion
Safety
BioBricks
Basic Parts
Composite Parts
Outreach
Human Practices
Collaborations
InterLab
Journal
jQuery UI Accordion - Collapse content
Modelling
Modelling the directed evolution of bacteriophage lambda
To accomplish the goal of altering the host specificity of bacteriophage lambda, we took a step-by-step directed evolution approach, working according to the 'design - build - model - test - repeat' principle. This started with the design of several mutations both to alter the phage phenotype and to make segments of its genotype BioBrick-compatible. We then proceeded to 'build' individual genes through synthesis and cloning. The next steps are to recombine these genes into a recombinant bacteriophage genome, before carrying out testing and selection, and repeating this process.
The process is as follows:
Short tail fibre (stf) and reporter genes are cloned into a gt11 bacteriophage lambda vector.
Regulatory circuits are created to control the production of the tail fibre assembly (tfa) protein and the switch between lytic/lysogenic cycles of the bacteriophage. While AraC is produced, production of T7 RNA polymerase (RNAP) is inhibited, resulting in production of the cI repressor dominating and the bacteriophage remaining in lysogenic phase. On the addition of arabinose, AraC preferentially binds to to this molecule, dissociating from the promoter controlling T7 RNAP. As a result, T7 RNAP is produced, which initiates production of the Cro repressor, overriding cI production and placing the bacteriophage into the lytic phase.
The circularised gt11 vector and plasmids containing the regulatory circuits are transformed into E.coli host cells, and the host machinery is used to create a recombinant bacteriophage containing our desired mutations and additions.
On addition of arabinose, the recombinant phage progeny goes into lytic phase and emerges from the lysed host cells.
Double knock-out strains of E.coli (containing neither lamB nor ompC receptor proteins) are exposed to the recombinant phages to select for baceriophage lambda mutants with widened host range.
The added reporter gene will facilitate easy identification of phage progeny that has successfully infected the E.coli double knock-out strains. These bacteriophages are purified and cultured, and the process is repeated to further improve infectivity.
Back to main Research page
To find out more about our results
