<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Birkbeck_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Birkbeck/Measurement</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="cssmenu"><UL><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/TeamMembers"><SPAN>Members</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Attributions"><SPAN>Attributions</SPAN></A></LI></UL><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Research"><SPAN>Research</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Description"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/pp"><SPAN>Background</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Experiments"><SPAN>Experiments</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Measurement"><SPAN>Measurements</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Modeling"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Results"><SPAN>Results</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Conclusion"><SPAN>Conclusion</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/BioBricks"><SPAN>BioBricks</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Basic_Parts"><SPAN>Basic Parts</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Composite_Part"><SPAN>Composite Parts</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Outreach"><SPAN>Outreach</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Practices"><SPAN>Human Practices</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Collaborations"><SPAN>Collaborations</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/InterLab Study"><SPAN>InterLab</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Notebook"><SPAN>Journal</SPAN></A></LI></DIV><TITLE>jQuery UI Accordion - Collapse content</TITLE><DIV id="maincontainer"><H2><B>Measurement</B></H2><DIV class="accordion"><CENTER><H3><A href="#">InterLab Study
            </A></H3><P>Click here to see our contribution to iGEM's InterLab Measurement study.
                    </P></CENTER><DIV><H3><U><B>Introduction</B></U></H3><P>The InterLab Study is aimed at investigating the expression levels of three promoters (J23101, J23106 &amp; J23117). Grren fluorescent protein (<I>gfp</I>-mut3b – E0040) was used as a reporter for expression and the level of gene expression was determined by measuring the fluorescence of each <I>E. coli</I> DH5α strain generated. The promoters under investigation were designed by John Anderson and submitted by the Berkeley iGEM 2006 group (with reference to <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_J23100">Part description</A>&amp;<A href="http://openwetware.org/wiki/IGEM:UC_Berkeley/2006">UC Berkeley/2006</A>).</P><P>The previous InterLab Study (2014), described by Dr. Jake Beal (iGEM Foundation) at the 2014 Jamboree in the following <A href="https://vimeo.com/120088076">link</A>, was composed of 45 teams representing 18 countries globally. As part of our project in designing a phage-based diagnostic tool, detection of signals is fundamental to our end goal. Studying the expression levels of different promoters in a standardized fashion maybe one way in which an output can be achieved with our product.</P><H3><U><B>Cloning</B></U></H3><P>In order to investigate the promoter expression, <I>gfp</I> had to be expressed by each of the promoters. The BioBrick <A href="http://parts.igem.org/Part:BBa_I13504"> BBa_I3504</A> contains a ribosome binding site (RBS – <A href="http://parts.igem.org/Part:BBa_B0034">B0034</A>), <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_E0040">gfp-mut3b (E0040)</A> and two terminators (<A href="http://parts.igem.org/Part:BBa_B0010">B0010</A>&amp;<A href="http://parts.igem.org/Part:BBa_B0012">B0012</A>). Cloning this BioBrick into different promoters will result in the expression of <I>gfp</I> and therefore detectable fluorescence from cells. The level of fluorescence detected from each of the promoters can be used to assess promoter activity.</P><P>The positive control in this study was; <A href="http://parts.igem.org/Part:BBa_I20270">pSB1C3-Bba_I20270</A>. Two negative controls were used; <A href="http://parts.igem.org/Part:BBa_R0040">pSB1C3-BBa_R0040</A>&amp;<I>E. coli</I> DH5α containing no plasmid</P><P>2 A assembly was carried out to make all 3 devices. The <A href="http://parts.igem.org/Part:BBa_I13504"> BBa_I3504 BioBrick</A> was in a plasmid backbone containing an ampicillin resistance cassette with each of the promoter devices in a plasmid backbone containing a chloramphenicol resistance cassette (P1 [<A href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_J23101"> BBa_J23101</A>], P2 [<A href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_J23106"> BBa_J23106</A>] &amp; P3 [<A href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_J23117"> BBa_J23117</A>]). </P><P>The BBa_I3504 plasmid was digested with <I>XbaI</I>&amp;<I>PstI</I> in order to liberate the BBa_I3504 BioBrick. Each of the promoters were restricted with <I>SpeI</I>&amp;<I>PstI</I> in order to open up the plasmid and allow digested BBa_I3504 to anneal and be ligated into each of the promoters. The cloning process is diagrammatically shown in <B>Fig. 1</B>. 
</P><P><B>Fig. 1: Cloning Strategy for Device Construction</B>.</P><P>Three colonies from each transformation were picked and grown in 5 mL LB broth (supplemented with 35 μg/mL chloramphenicol). 500 μL of overnight culture was used to make glycerol stocks with the remaining culture used for miniprepping &amp; restriction diagnostics with <I>EcoRI</I>&amp;<I>PstI</I>. The expected restriction pattern is displayed in <B>Fig. 2</B> with the restriction diagnostic results being displayed in <B>Fig. 3</B>. Plasmid maps can be visualised in <A href="https://2015.igem.org/Team:Birkbeck/InterLab_Study/Interlab_plasmid_maps"><B>Fig. S1-S5</B></A>.</P><P><B>Fig. 2: Expected Restriction Pattern</B>. In the lane MW is a 2Log ladder. All substrate plasmids were digested with <I>EcoRI</I>&amp;<I>PstI</I>. Lanes 1-3 are restriction diagnostics of P1, P2 &amp; P3 <I>gfp</I> Expression devices respectively. Lane 4 is the restriction diagnostic of pSB1C3-BBa_I3504. Lane 5 is the restriction digest of the positive control used (Bba_I20270). Lane 6 is the negative control plasmid restriction (pSB1C3-BBa_R0040)</P><P><B>Fig. 3: Restriction Diagnostic (<I>EcoRI</I>&amp;<I>PstI</I> Digestion) of Interlab <I>gfp</I> Expression Devices</B>. In Lane 1 is 5 μL of 2-Log NEB ladder. In lanes 2-4 are 5 μL of P1-3-<I>gfp</I> expression device digests respectively. Lane 5 contains 5 μL of the <I>gfp</I> part source digest. Lane 6 contains 5 μL of digested <I>gfp</I>-positive control. Lane 7 contains 5 μL of digested <I>gfp</I>-negative control expression device.</P><H3><U><B>Experimental Procedure</B></U></H3><P>After screening plasmids for the BBa_I3504 insert into the three promoter devices, successful transformants were streaked from glycerol stocks on LB agar (1.5%) plates were supplemented with chloramphenicol to a final concentration of 35 μg/mL chloramphenicol. Plates were incubated at 37<SUP>o</SUP>C overnight in a static incubator.</P><P>3 discrete single colonies were used to inoculate 10 mL of LB broth supplemented with 35 μg/mL of chloramphenicol. Cultures were grown at 37<SUP>o</SUP>C shaking at 250 rpm for 18 hours.</P><P>After 18 hours of incubation, the optical density (at a wavelength of 600 nm – OD<SUB>600</SUB>) of each culture was taken by aliquoting 200 μL of overnight culture into a 96-well plate and measuring the absorbance of each culture at OD<SUB>600</SUB>. Each culture was diluted back to an OD<SUB>600</SUB> = 0.5 by calculating the required volume of culture to be diluted in sterile LB broth; Volume used in inoculation = (Desired OD/Actual OD) × Volume to be inoculated. The diluted OD600 was measured and adjusted where required. When an OD<SUB>600</SUB> range between 0.475-0.525 was achieved, fluorescence of each culture was then measured after standardization.</P><H3><U><B>Results</B></U></H3><P>The differences in means of each triplicate cultures were analysed. <B>Fig. 4</B> displays the mean of each of the standardize OD<SUB>600</SUB> with error bars representing 95% confidence intervals. All data was analysed using IBM SPSS statistics software package.</P><P><B>Fig. 4: Means of Standardized Triplicate Devices OD<SUB>600</SUB></B>.</P><P>A <A href="https://2015.igem.org/Team:Birkbeck/InterLab_Study/ungrouped_standardized_OD600">one-way ANOVA</A> conducted between each of the triplicates showed there was no statistical differences between the triplicates of each expression device showed that there was no significant difference (P=0.123). Inspecting the multiple comparison table, there is no significant difference between the means of the <I>E. coli</I> DH5α strains standardized. This data shows that each culture was successfully standardized by dilution to an OD<SUB>600</SUB> range between 0.475-0.525.</P><P><B>Fig. 5: Means of Standardized Devices OD<SUB>600</SUB></B>.</P><P>Each of the OD<SUB>600</SUB> = 0.5 dilutions were grouped and averaged. The data is displayed in <B>Fig. 4</B>. A <A href="https://2015.igem.org/Team:Birkbeck/InterLab_Study/grouped_standardized_OD600">one-way ANOVA</A> between the different expression devices showed that there was no statistical differences between each expression device with regard to OD<SUB>600</SUB> standardization (P=0.284).</P><P><B>Fig. 6: Means of Standardized Triplicate Devices Fluorescence</B>.</P><P>The mean measurement of fluorescence for each triplicate is displayed in <B>Fig. 5</B>. There was <A href="https://2015.igem.org/Team:Birkbeck/InterLab_Study/fluorescence_ANOVA_ungrouped">very highly statistically significant</A> differences the between all the triplicates (P&lt;0.001). However when comparing the same <I>E. coli</I> DH5α strain, there was no significant difference between the triplicates. This data shows that there is a difference between the different promoters under investigation &amp; the control <I>E. coli</I> DH5α strains with no difference between each of the triplicates.</P><P><B>Fig. 7: Means of Standardized Devices Fluorescence (au)</B>.</P><P>The data from <B>Fig. 5</B> was grouped according to expression device and is displayed in <B>Fig. 6</B>. Conducting a <A href="https://2015.igem.org/Team:Birkbeck/InterLab_Study/fluorescence_ANOVA">one-way ANOVA</A> on the data; P1, P2 &amp; positive control expression devices shows a very highly significant difference between all other expression devices (P&lt;0.001). Comparing the mean of P3 device with P1, P2 &amp; positive control, there appears to be a very highly significant difference (P&lt;0.001) with P3 showing the lowest level of fluorescence. The P3 device shows no statistical significant difference between the B0012 &amp;<I>E. coli</I> DH5α negative controls (P=0.704 &amp; P=0.243 respectively). There is no statistically significant difference observed between the two negative controls (P=0.967).</P><H3><U><B>Conclusion</B></U></H3><P>Results from this study indicate that the P1-<I>gfp</I> expression device yields the highest expression levels after 18 hours of growth. P2-<I>gfp</I> expression device has a higher expression level than the positive control but a lower level of expression when compared to the P1-<I>gfp</I> expression device. Te P3-<I>gfp</I> expression device has a fluorescence level which is comparible to the negative controls used in this study. Each of the <I>gfp</I> expression devices P1, P2 &amp; positive control from this study were further characterised in our InterLab Study Additional Results <B>Fig. 9: Growth Curves of Different Strains of E. coli DH5α Following Culture Fluorescence</B>, which is in the tab below. The data from this experiment also shows the same pattern of expression from each of the promoters under investigation.</P><H3><U><B>Participants</B></U></H3><P><B>Sean Ross Craig</B> (data analysis, cloning, restriction diagnostics &amp; measurements), <B>Elliott Parris</B> (measurements &amp; restriction diagnostics), <B>Rachel Wellman</B> (restriction diagnostics &amp; measurements) &amp;<B>Ariana Mirzarafie-Ahi</B> (cloning &amp; measurements).</P><P>With thanks to <B>Dr. Vitor Pinheiro</B>, <B>Dr. Jane Nicklin</B>, <B>Bilkis Kazi</B>, <B>Barbara Steijl</B>, <B>Luba Prout</B>.</P></DIV></DIV><DIV id="maincontainer"><DIV class="accordion"><CENTER><H3><A href="#">InterLab Study Additional Results
            </A></H3><P>Click here to see the results of additional experiments that support our research in contribution to iGEM's InterLab Study. We conducted these experiments in order to assess the time frames in which fluorescent signals could be detected from cultures.
                    </P></CENTER><DIV><P><B><U>Fig. 1: Growth Curve of <I>E. coli</I> DH5α Strains Following Culture Optical Density of 600 nm</U></B>.</P><P>Growth kinetics was initially studied using 50 mL cultures. <B>Fig. 1</B> shows the growth kinetics of <I>E. coli</I> DH5α &amp; derivative strains containing plasmids from the <A href="https://2015.igem.org/Team:Birkbeck/InterLab_Study">InterLab</A> study.  The growth curve shows that the <I>E. coli</I> strain that contains the P1-<I>gfp</I> expression device grows at a slower rate than the other strains investigated. At 220 minutes the <I>E. coli</I> DH5α P1 strain has a significantly lower OD<SUB>600</SUB> than the <I>E. coli</I> DH5α (P=0.023). <I>E. coli</I> DH5α remains significantly higher in OD<SUB>600</SUB> than <I>E. coli</I> DH5α with the P1-<I>gfp</I> expression device (P=&lt;0.001). The only difference between the <I>E. coli</I> DH5α &amp;<I>E. coli</I> DH5α positive control device is observed at 280 minutes into the growth curve (P=0.016) where the positive control has a higher OD<SUB>600</SUB>. The multiple comparison table showing P values can be viewed in <A href="https://2015.igem.org/Team:Birkbeck/Results/Table_S1"><B>Table S1</B>.</A></P>.
<P><B><U>Fig. 2: Growth Curve of <I>E. coli</I> DH5α Strains Following Culture Optical Density of 395 nm</U></B>.</P><P>In order to investigate if there could be a point in the <I>E. coli</I> DH5α growth curve in which a signal from the GFP could be detected by absorbance, the growth curves were also conducted using the major absorption peak of GFP (wavelength 395 nm). The growth curve data for the culture optical density is displayed in <B>Fig. 2</B>.</P><P>When comparing the data point of <I>E. coli</I> DH5α strains, there appears to be a significant increase in culture OD<SUB>395</SUB> in the <I>E. coli</I> cells with the P1-<I>gfp</I> expression device (P&lt;0.001). This apparent signal is only present between 60-100 minutes of growth. When comparing the positive control &amp;<I>E. coli</I> DH5α1, there is no significant difference between the data sets at 60 or 100 minutes (P=1 for both time points). A small potential signal is observed from the oositive control <I>gfp</I> expression device at 280 minutes (P=0.012) &amp; 300 minutes (P=0.006). This significance is lost after 300 minutes (P=0.262). See <A href="https://2015.igem.org/Team:Birkbeck/Results/Table_S2"><B>Table S2</B></A> for more details. In order to verify these results &amp; to test for the feasibility of scaling down to a 96-well microtitre plate assay, a 10 hour growth curve was conducted in a 96-well microtitre plate (see <B>Fig. 6-8</B>).</P><P><B><U>Fig. 3: Viable Count of <I>E. coli</I> DH5α After 60 mins</U></B>.</P><P>In order to assess how many viable cells correspond to different optical densities, a viable count was conducted on <I>E. coli</I> DH5α1 at 60 minutes (<B>Fig. 3</B>&amp;<B>Table 1</B>), 175 minutes (<B>Fig. 5</B>&amp;<B>Table 2</B>) &amp; 225 minutes (data not shown due to high level of contamination).</P><P>Considering the OD<SUB>600</SUB> of the <I>E. coli</I> DH5α1 cultures at 60 mins, triplicate cultures were OD<SUB>600</SUB> = 0.029, 0.01 &amp; 0.025. The viable count of each of the cultures gave a mean of 2.57×10<SUP>6</SUP> cfu/mL (<B>Table 1</B>). It can therefore be concluded that an <I>E. coli</I> DH5α culture at an OD<SUB>600</SUB> = 0.021 corresponds to approximately 2.57×10<SUP>6</SUP> cfu/mL.</P><CENTER><DIV class="itemBody"><TABLE class="Default PivotTable" data-lookname="Default" data-rowlabelwidth="1" data-collabelheight="1" data-numrows="2" data-numcols="5" data-rowlabels="Corner" data-decimal-char="." data-col-labels="N;Minimum;Maximum;Mean;Std. Deviation" data-row-labels="Viable.Count;Valid N (listwise)"><CAPTION class="title" style="width:522px">Descriptive Statistics of 1 hour Viable Count of <I>E. coli</I> DH5α.</CAPTION><TBODY class="PivotBody defaultLayer innerFrameBd"><TR><TD class="cornerLabels"> </TD><TD class="columnLabels dataAreaLeft vCC role3">N</TD><TD class="columnLabels vCC role3">Minimum</TD><TD class="columnLabels vCC role3">Maximum</TD><TD class="columnLabels vCC role3">Mean</TD><TD class="columnLabels vCC role3">Std. Deviation</TD></TR><TR><TD class="rowLabels dataAreaTop role3">Viable Count</TD><TD class=" e dataAreaTop dataAreaLeft vCC">9</TD><TD class=" e dataAreaTop vCC">1600000</TD><TD class=" e dataAreaTop vCC">3750000</TD><TD class=" e dataAreaTop vCC">2566666.67</TD><TD class=" e dataAreaTop vCC">627495.020</TD></TR><TR><TD class="rowLabels hCR role3">Valid N (listwise)</TD><TD class=" o dataAreaLeft hCR vCC">9</TD><TD class=" o hCR vCC"> </TD><TD class=" o hCR vCC"> </TD><TD class=" o hCR vCC"> </TD><TD class=" o hCR vCC"> </TD></TR></TBODY></TABLE></DIV></CENTER><P><B><U>Table 1: Descriptive Statistics of 60 minutes Viable Count of <I>E. coli</I> DH5α.</U></B>.</P><P><B><U>Fig. 4: Viable Count of <I>E. coli</I> DH5α After 175 mins</U></B>.</P><P>At 175 minutes into growth, the <I>E. coli</I> DH5α1 cultures had OD<SUB>600</SUB> of; 0.255, 0.216 &amp; 0.262. The viable count of each of the cultures gave a mean of 1.33×10<SUP>8</SUP> cfu/mL (<B>Table 2</B>). It can therefore be concluded that an <I>E. coli</I> DH5α culture at an OD<SUB>600</SUB> = 0.244 corresponds to approximately 1.33×10<SUP>8</SUP> cfu/mL. Considering the previous OD<SUB>600</SUB> (0.021), there is approximately a 10-fold increase in OD<SUB>600</SUB> which corresponds to nearly a 100-fold increase in viable cells.</P><CENTER><DIV class="itemBody"><TABLE class="Default PivotTable" data-lookname="Default" data-rowlabelwidth="1" data-collabelheight="1" data-numrows="2" data-numcols="5" data-rowlabels="Corner" data-decimal-char="." data-col-labels="N;Minimum;Maximum;Mean;Std. Deviation" data-row-labels="Viable.Count;Valid N (listwise)"><CAPTION class="title" style="width:547px"><SPAN class="details">Descriptive Statistics of 175 Minutes Viable Count of <I>E. coli</I> DH5α.</SPAN></CAPTION><TBODY class="PivotBody defaultLayer innerFrameBd"><TR><TD class="cornerLabels"> </TD><TD class="columnLabels dataAreaLeft vCC role3">N</TD><TD class="columnLabels vCC role3">Minimum</TD><TD class="columnLabels vCC role3">Maximum</TD><TD class="columnLabels vCC role3">Mean</TD><TD class="columnLabels vCC role3">Std. Deviation</TD></TR><TR><TD class="rowLabels dataAreaTop role3">Viable Count</TD><TD class=" e dataAreaTop dataAreaLeft vCC">9</TD><TD class=" e dataAreaTop vCC">90000000</TD><TD class=" e dataAreaTop vCC">175000000</TD><TD class=" e dataAreaTop vCC">133333333.33</TD><TD class=" e dataAreaTop vCC">29154759.474</TD></TR><TR><TD class="rowLabels hCR role3">Valid N (listwise)</TD><TD class=" o dataAreaLeft hCR vCC">9</TD><TD class=" o hCR vCC"> </TD><TD class=" o hCR vCC"> </TD><TD class=" o hCR vCC"> </TD><TD class=" o hCR vCC"> </TD></TR></TBODY></TABLE></DIV></CENTER><P><B><U>Table 2: Descriptive Statistics of 175 minutes Viable Count of <I>E. coli</I> DH5α.</U></B></P><P><B><U>Fig. 5: Growth Curves of Different Strains of <I>E. coli</I> DH5α Following Culture Optical Density at 601 nm</U></B>.</P><P>All data analysis tables for the OD<SUB>601</SUB> growth curves are in; <A href="https://2015.igem.org/Team:Birkbeck/table_S3">Table S3</A>, <A href="https://2015.igem.org/Team:Birkbeck/table_S5">Table S5</A>&amp;<A href="https://2015.igem.org/Team:Birkbeck/table_S6">Table S6</A> for 0-200 minutes, 220-420 minutes &amp; 440-580 minutes respectively. Comparing the growth of <I>E. coli</I> DH5α containing P1-<I>gfp</I> expression device with the <I>E. coli</I> DH5α, there is no significant difference between the two cultures at 80 minutes (P=0.943), however, at 100 minutes there is a very highly significant difference between the 2 strains of <I>E. coli</I> DH5α (P&lt;0.001). The OD<SUB>601</SUB> of <I>E. coli</I> DH5α containg the P1-<I>gfp</I> expression device remains at a significantly lower that the <I>E. coli</I> DH5α untill 400 minutes (P=0.441).</P><P>Comparing the <I>E. coli</I> DH5α P1-<I>gfp</I> expression device with the positive control <I>gfp</I><I>E. coli</I> DH5α cultures, the OD<SUB>601</SUB> remains insignificantly different at 100 minutes (P=0.848) with significance being observed at 120 minutes (P&lt;0.001). The significance is observed until 280 minutes into the growth curves (P=0.065). Interestingly, this is a shorter window of significance compared to <I>E. coli</I> P1-<I>gfp</I> expression device &amp;<I>E. coli</I> DH5α (160 minute Vs 300 minutes respectively).</P><P>Comparing P1 &amp; P2-<I>gfp</I> expression devices in <I>E. coli</I> DH5α, no significance is observed at 100 minutes (P=0.132). P1-<I>gfp</I> expression device is very highly significantly lower (P&lt;0.001) than P2-<I>gfp</I> expression device after 120 minutes of culturing. This significance is observed until 320 minutes of culturing (P=0.075).</P><P>When comparing the positive control, P2-<I>gfp</I>&amp;<I>E. coli</I> DH5α, there is no significant difference throughout the growth curve. After 420 minutes, there is no significant difference between any of the <I>E. coli</I> DH5α cultures OD<SUB>601</SUB> (P=0.81).</P><P><B><U>Fig. 6: Growth Curves of Different Strains of <I>E. coli</I> DH5α Following Culture Optical Density at 501 nm</U></B>.</P><P>All data analysis tables for the OD<SUB>501</SUB> growth curves are in; <A href="https://2015.igem.org/Team:Birkbeck/table_S7">Table S7</A>, <A href="https://2015.igem.org/Team:Birkbeck/table_S8">Table S8</A>&amp;<A href="https://2015.igem.org/Team:Birkbeck/table_S9">Table S9</A> for 0-200 minutes, 220-420 &amp; 440-580 minutes respectively. Comparing the growth of <I>E. coli</I> DH5α containing P1-<I>gfp</I> expression device with the <I>E. coli</I> DH5α, there is no significant difference between the two cultures at 80 minutes (P=0.987). After 100 minutes there is a highly significant difference between the two cultures (P=0.003). This difference remains significant until 400 minutes into culturing (P=0.093).</P><P>Comparing the <I>E. coli</I> DH5α P1-<I>gfp</I> expression device with the positive control <I>gfp</I><I>E. coli</I> DH5α cultures, the OD<SUB>501</SUB> remains insignificantly different at 100 minutes (P=0.85) with significance being observed at 120 minutes (P&lt;0.001). The P1-<I>gfp</I> expression device culture remains significantly lower than the positive control <I>gfp</I> expression device until 320 minutes (P=0.053).</P><P>Comparing P1 &amp; P2-<I>gfp</I> expression devices in <I>E. coli</I> DH5α, no significance is observed at 100 minutes (P=0.403). P1-<I>gfp</I> expression device is very highly significantly lower (P&lt;0.001) than P2-<I>gfp</I> expression device after 120 minutes of culturing. This significance is observed until 320 minutes of culturing (P=0.096).</P><P>When comparing the positive control, P2-<I>gfp</I>&amp;<I>E. coli</I> DH5α, there is no significant difference throughout the growth curve. After 420 minutes, there is no significant difference between any of the <I>E. coli</I> DH5α cultures OD<SUB>501</SUB> (P=0.09). These results are identical to the results obtained in th OD<SUB>501</SUB> absorption of the <I>E. coli</I> DH5α cultures.</P><P><B><U>Fig. 7: Growth Curves of Different Strains of <I>E. coli</I> DH5α Following Culture Optical Density at 475 nm</U></B>.</P><P>All data analysis tables for the OD<SUB>475</SUB> growth curves are in; <A href="https://2015.igem.org/Team:Birkbeck/Table_S10">Table S10</A>, <A href="https://2015.igem.org/Team:Birkbeck/table_S11">Table S11</A>&amp;<A href="https://2015.igem.org/Team:Birkbeck/table_S12">Table S12</A> for 0-200 minutes, 220-420 &amp; 440-580 minutes respectively. Comparing the growth of <I>E. coli</I> DH5α containing P1-<I>gfp</I> expression device with the <I>E. coli</I> DH5α, there is no significant difference between the two cultures at 100 minutes (P=0.982). After 120 minutes there is a highly significant difference between the two cultures (P=0.003). This difference remains significant until 440 minutes into culturing (P=0.745).</P><P>Comparing the <I>E. coli</I> DH5α P1-<I>gfp</I> expression device with the positive control <I>gfp</I><I>E. coli</I> DH5α cultures, the OD<SUB>475</SUB> remains insignificantly different at 100 minutes (P=0.844) with significance being observed at 120 minutes (P&lt;0.001). The P1-<I>gfp</I> expression device culture remains significantly lower than the positive control <I>gfp</I> expression device until 340 minutes (P=0.19).</P><P>Comparing P1 &amp; P2-<I>gfp</I> expression devices in <I>E. coli</I> DH5α, no significance is observed at 100 minutes (P=0.464). P1-<I>gfp</I> expression device is very highly significantly lower (P&lt;0.001) than P2-<I>gfp</I> expression device after 120 minutes of culturing. This significance is observed until 340 minutes of culturing (P=0.163).</P><P>When comparing the positive control, P2-<I>gfp</I>&amp;<I>E. coli</I> DH5α, there is no significant difference throughout the growth curve. Conducting a one-way ANOVA of the culture density at 440+ minutes shows there is a highly significant difference between the means (P=0.003). The multiple comparisons table shows no significant difference between any of the individual data point (data not shown).</P><P><B><U>Fig. 8: Growth Curves of Different Strains of <I>E. coli</I> DH5α Following Culture Optical Density at 395 nm</U></B>.</P><P>All data analysis tables for the OD<SUB>395</SUB> growth curves are in; <A href="https://2015.igem.org/Team:Birkbeck/Table_S13">Table S13</A>, <A href="https://2015.igem.org/Team:Birkbeck/table_S14">Table S14</A>&amp;<A href="https://2015.igem.org/Team:Birkbeck/table_S15">Table S15</A> for 0-200 minutes, 220-420 &amp; 440-580 minutes respectively. Comparing the growth of <I>E. coli</I> DH5α containing P1-<I>gfp</I> expression device with the <I>E. coli</I> DH5α, there is no significant difference between the two cultures at 80 minutes (P=0.937). After 100 minutes there is a highly significant difference between the two cultures (P&lt;0.001). This difference remains significant until 440 minutes into culturing (P=0.083)</P><P>Comparing the <I>E. coli</I> DH5α P1-<I>gfp</I> expression device with the positive control <I>gfp</I><I>E. coli</I> DH5α cultures, the OD<SUB>395</SUB> remains insignificantly different at 80 minutes (P=0.992) with significance being observed at 100 minutes (P=0.011). The P1-<I>gfp</I> expression device culture remains significantly lower than the positive control <I>gfp</I> expression device until 440 minutes (P=0.325).</P><P>Comparing P1 &amp; P2-<I>gfp</I> expression devices in <I>E. coli</I> DH5α, no significance is observed at 100 minutes (P=0.358). P1-<I>gfp</I> expression device is very highly significantly lower (P&lt;0.001) than P2-<I>gfp</I> expression device after 120 minutes of culturing. This significance is observed until 340 minutes of culturing (P=0.171).</P><P>When comparing the positive control, P2-<I>gfp</I>&amp;<I>E. coli</I> DH5α, there is no significant difference throughout the growth curve. Conducting a one-way ANOVA of the culture density at 440+ minutes shows there is a highly significant difference between the means (P&lt;0.001). The multiple comparisons table shows no significant difference between any of the individual data point (data not shown).</P><P><B><U>Fig. 9: Growth Curves of Different Strains of <I>E. coli</I> DH5α Following Culture Fluorescence</U></B>.</P><P>All data analysis tables for the culture fluorescence growth curves are; <A href="https://2015.igem.org/Team:Birkbeck/table_S16">Table S16</A>, <A href="https://2015.igem.org/Team:Birkbeck/Table_S17 ">Table S17</A>&amp;<A href="https://2015.igem.org/Team:Birkbeck/table_S18">Table S18</A> for 0-200 minutes, 220-420 &amp; 440-580 minutes respectively. Comparing the P1-<I>gfp</I> expression device with <I>E. coli</I> DH5α, at 0 minutes there is a highly significant difference between the two cultures fluorescence (P=0.005). This significance is observed until 100 minutes (P=0.427). There is an insignificant difference between the two cultures until 160 minutes (P&lt;0.001). At 220 minutes into the growth curve, the fluorescence between P1<I>gfp</I> expression device is insignificant (P=0.277) with significance being observed at 240 minutes (P=0.001). The fluorescence of P1-<I>gfp</I> expression device remains higher than the <I>E. coli</I> DH5α culture.</P><P>Comparing the positive control <I>gfp</I> expression device with <I>E. coli</I> DH5α, at no point in the growth curve does the fluorescence of the positive control <I>gfp</I> expression device increase above that of the <I>E. coli</I> DH5α culture. Even at 580 minutes of culturing the positive control does not increase in fluorescence above the <I>E. coli</I><I>E. coli</I> DH5α (P=0.677).</P><P>Comparing the P2-<I>gfp</I> expression device with <I>E. coli</I> DH5α, Initially there is no significant difference between the two cultures in fluorescence (P=1). There is a highly significant Increase in fluorescence from P2 at 180 minutes (P=0.007). At 200 minutes, there is no significant difference in fluorescence observed between P2 + <I>E. coli</I> DH5α (P=0.146). Insignificance of fluorescence between the two cultures is observed until 500 minutes of culturing (P=0.038). The fluorescence from the P2-<I>gfp</I> expression device remains significantly higher than <I>E. coli</I> DH5α throughout the rest of the growth curve.</P><P>Comparing the P1-<I>gfp</I> expression device with the positive control <I>gfp</I> expression device there is a significantly higher fluorescence from 0 minutes (P=0.034) &amp; 20 minutes (P=0.038). At 40 minutes of culturing there is no significant difference in fluorescence (P=0.164). At 60 minutes the P1-<I>gfp</I> expression device has a significantly higher fluorescence than the positive control (P=0.017). The significance is not observed at 80 minutes (P=0.849). There is no significant fluorescence from P1 compared to the positive control until 180 minutes (P=0.009). No significance is observed between the two promoters until 340 minutes where P1 has a higher fluorescence (P=0.019). The P1-<I>gfp</I> expression device has a higher fluorescence compared to the positive control at 360 minutes (P=0.024) and becomes insignificant at 380 minutes (P=0.245) where it remains insignificant for the result of the culturing time.</P><P>Comparing P2-<I>gfp</I> expression device with the positive control, there is no time point where the P2-<I>gfp</I> expression device has a statistically higher fluorescence than the positive control <I>gfp</I> expression device.</P><P>When comparing the data sets for the P1 &amp; P2 <I>gfp</I> expression devices, there is no significant difference until 60 minutes (P=0.021). At 60 minutes, the P1-<I>gfp</I> expression device has a higher fluorescence. There is no significance between P1 &amp; P2 <I>gfp</I> expression devices at 80 minutes (P=0.646) &amp; no other time points show significance between these two cultures.</P><P>Discussion of these results &amp; credit for the work are shown in the tab below.</P></DIV></DIV><DIV class="accordion"><CENTER><H3><A href="#">Discussion of Additional InterLab Study Results
            </A></H3><P>Click here to see a discussion of our results additional to iGEM's InterLab Measurement study, and importantly how they they bare relevance to our research.
                    </P></CENTER><DIV><H2><B><U>Signal Detection</U></B></H2><P>Considering our results, at time 0 minutes, there is a significant signal for P1-<I>gfp</I> expression device compared to the <I>E. coli</I> DH5α (P=0.005, with reference to <B>Fig. 9</B> in results section). This result cannot be regarded as this is probably carry over fluorescence from sub-culturing.  Photo-bleaching prior to the first reading may yield more accurate readings when considering the expression of <I>gfp</I> from promoters. Also the error bars in each of the expression devices do appear excessive. A means of reducing this may be to sub-culture the cells &amp; grow to an OD<SUB>600</SUB> of 0.2 and sub-culturing into experimental cultures in order to minimise the error bars.</P><P>One technique which has been developed is transcription-reverse transcription concerted reaction (TCR) (Ishiguro <I>et al</I>., 1996). TCR monitors transcription in a sequence-dependant manner in <I>in-vitro</I> systems (Ishiguro <I>et al</I>., 1996). The TCR protocol was adapted in order to improve sensitivity for detecting <I>M. tuberculosis</I> from clinical sputum samples (Drouillon <I>et al</I>., 2009 [TCR-2]). Considering P1-<I>gfp</I> culture at 20 minutes, there is a significant fluorescent signal (P=0.002) with no difference between the OD<SUB>600</SUB> growth curves at this point (P=1).  This is significantly less than the hours taken for results to be generated by TCR-2 (Drouillon <I>et al</I>., 2009). Cells cannot be quantified at the 20 minutes time point in our study as no viable count was carried out, therefore the detection limit of the <I>E. coli</I> DH5α with regards to fluorescence cannot be determined.</P><P>With regards to <A href="https://catalog.hardydiagnostics.com/cp_prod/Content/hugo/TBPrepKit.htm">Decontamination Kits</A>, samples would take just under one hour to process. Of note, some bacteria (such as <I>Staphylococcus aureus</I>) can be resistant to decontamination (refer to the kit previously mentioned). A key point to note here is that the decontamination step must not be carried out for <EM>longer than 15 minutes</EM> due to the reduction in cell viability of <I>Mycobacterium</I> cells.</P><P>As transduction was not characterised in this study, it is difficult to assess exactly how long this step may take. Considering the application of D29 mycobacteriophage could be spotted directly onto soft agar plates giving rise to plaques on both <I>M. smegmatis</I>&amp;<I>M. tuberculosis</I> (Sampson <I>et al</I>., 2009). A mycobacteriophage assay for detection of live <I>Mycobacterium</I> cells has previously been carried out using a plaque assay (Alcaide <I>et al</I>., 2003). A 1 mL decontaminated sample was washed and grown overnight in the standard <I>Mycobacterium</I> media <B>Middlebrook 7H9</B> supplemented with 10% (vol/vol) oleic acid-albumin-dextrose catalase (OADC) (Alcaide <I>et al</I>., 2003). 100 μL of mycobacteriophage was added to the overnight cultures and incubated at 37<SUP>o</SUP>C for 1 hour before adding 100 μL of viricidal solution (Alcaide <I>et al</I>., 2003). A soft agar lawn was poured and incubated for 24 hours before counting plaques. The use of a fluorescent protein as a tag would perhaps remove the need for overnight incubation (in fact a 48 hour protocol).</P><P>Alcaide &amp; co-workers did demonstrate that smear positive samples were extremely sensitive to the phage assay and claim that the detection limit was 10 cfu/mL with smear negative samples showing much higher detection limits. Another study using mycobacteriophages as a diagnostic tool used the cheaper medium Mueller–Hinton instead of Middlebrook 7H9 (Hemvani <I>et al</I>., 2012). The Hemvani study showed similar results to the Alcaide study. The change in media did however result in 3.5% of sputum samples being un-interpretable due to high levels of contamination even after treatment with vancomycin &amp; polymyxin B (Hemvani <I>et al</I>., 2012).</P><P>Considering the presorption phase of the assay developed by Alcaide <I>et al</I>, this would add an hour onto the protocol. Given that transduction is 100% efficient (unlikely) and that the kinetics of <I>gfp</I> expression behaves the same in <I>E. coli</I> DH5α as does a <I>Mycobacterium</I> sp. cell (another wild assumption), we are looking at from sputum sample collection until signal detection at approximately 2-4 hours (given that decontaminated sputum samples could be directly exposed to recombinant bacteriophages). It must be noted that in our approach, the signal is produced by transduction of genetic material as opposed to amplification of phage particles being the signal. The use of transduction hass been successfully applied in genetic engineering of <I>Mycobacterium</I> sp. (Jain <I>et al</I>., 2014; Tufariello <I>et al</I>., 2014) which is promising for the development of a diagnostic tool in introducing a detectable &quot;payload&quot; for diagnostic purposes.</P><P>Considering our results, at 1 hour of culturing <I>E. coli</I> DH5α, there is a significant signal observed from the culture expressing the P1-<I>gfp</I> expression device (P&lt;0.001). Considering the growth, there was no significant difference between the OD<SUB>600</SUB> of the P1-<I>gfp</I> expression device &amp; the <I>E. coli</I> without plasmid (P=1 for both 50 mL cultures &amp; 200 μL, with reference to <B>Fig. 1</B>&amp;<B>Fig. 5</B> respectively). At the 1 hour time point, there is a significant difference between the 50 mL &amp; 200 μL cultures (P=0.017) with the 200 μL cultures showing a higher OD<SUB>600</SUB>. This entails that the fluorescence of the 200 μL cultures at 1 hour corresponds to a higher culture density than 2.57×10<SUP>6</SUP> cfu/mL.</P><P>Considering the TCR-2 method of detection, 2.57×10<SUP>6</SUP> cfu/mL is an increase in detection limit by a magnitude of 10<SUP>5</SUP> (30 cfu of <I>M. tuberculosis</I>/mL of sputum [Drouillon <I>et al</I>., 2009]) which is clearly an undesirable due to the slow growth of <I>M. tuberculosis</I>. There appears to be a reduction in fluorescence from the P1-<I>gfp</I> expression device after 20 minutes which might correspond to the depletion of mRNA/GFP. The increase in fluorescence when comparing the 20 &amp; 40 minute time points may be the expression of <I>gfp</I> from each of the promoter increasing to a detectable level.</P><P>An interesting observation was made on the growth kinetics of the <I>E. coli</I> DH5α containing P1-<I>gfp</I> expression device &amp;<I>E. coli</I> DH5α not containing any plasmid. Between 100-400 minutes, the <I>E. coli</I> DH5α containing P1-<I>gfp</I> expression device has a significantly lower OD<SUB>601</SUB>&amp; higher fluorescence when compared to <I>E. coli</I> DH5α. The significant difference is not observed in the positive control or P2-<I>gfp</I> expression device with limited detection of fluorescence. This data suggests that compromising cellular growth with regard to expression of a signal may yield a quicker detection of a signal with the product.</P><P>It is obvious that the P1 promoter will not be able to drive gene expression in all kinds of bacteria. The proposed T7-RNAP-driven <I>rfp</I> expression was not characterised as the circuit was not constructed due to cloning issues. However, the growth kinetics would be expected to be slower than that of the P1-<I>gfp</I> expression device. <B>Fig. 1</B> is a diagramatic representation of the proposed genetic circuit for the characterization of a signal.</P><P><B>Fig. 1: TetR <I>rfp</I> Expression device</B>.</P><P>The idea behind the circuit displayed in <B>Fig. 1</B> is to have the amplification of <I>rfp</I> expression inducible. The induced expression of the amplification step in this circuit would mimic the infection of the <I>E. coli</I> cells by a recombinant λ-bacteriophage. This would allow different time points in the growth curve to be stimulated and signal intensities could be compared relative to the cell dose &amp; physiological state of cells.</P><P>In the first part of the circuit displayed in <B>Fig. 1</B>, a strong constituitive promoter (such as <A href="http://parts.igem.org/Part:BBa_I14033">pCAT) will drive the expression of </A><A href="http://parts.igem.org/Part:BBa_P0140"><I>tetR</I></A>. The TetR will bind to <A href="http://parts.igem.org/Part:BBa_I14015"><I>tetO</I>&amp; repress the expression of <I>T7-RNAP</I>. The T7-RNAP would then drive the exmpreesion of </A><A href="http://parts.igem.org/Part:BBa_E1010">rfp</A> via <A href="http://parts.igem.org/Part:BBa_K525998">T7 promoter</A>.</P><P>The <A href="http://parts.igem.org/Part:BBa_E1010">highly mutated <I>rfp</I></A> was chosen as a marker due to the ability of the protein to be observed on a plate as well as liquid culture without the use of fluorescence. The absorption of GFP could not be detected accurately (see <B>Fig. 2</B>&amp;<B>Fig. 6-8</B> in the additional <A href="https://2015.igem.org/Team:Birkbeck/Results">Interlab Results</A>. This approach can't be applied to detecting an early signal however, left on a plate can yield very suggestive &amp; qualitative results.</P><P>Another proposed way in which to characterize this circuit was to clone into a <I>gt 11</I> λ-bacteriophage vector. with reference to <B>Fig. 1</B>, the right hand side of the line down the circuit (i.e. from the <I>tetO</I>) would be cloned into the <I>gt 11</I> vector. There could be many possible time points to assess what part of the growth curve would yield the highest signal. </P><H2><U>References</U></H2><LI><B>Alcaide F., Gali N., Dominguez J., Berlanga P., Blanco S., Orus P. &amp; Martin R.</B> (2003). Usefulness of a New Mycobacteriophage-Based Technique for Rapid Diagnosis of Pulmonary Tuberculosis. <I>Journal of Clinical Microbiology</I> Vol. 41, No. 7, p2867-2871.</LI><LI><B>Drouillon V., Deloogu G., Dettori G., Lagrange P.H., Benecchi M., Houriez F., Baroli K., Ardito F., Torelli R., Chezzi C., Fadda G. &amp; Herrman J-L.</B> (2009). Multicenter Evaluation of a Transcription-Reverse Transcription Concerted Assay for Rapid Detection of <I>Mycobacterium tuberculosis</I> Complex in Clinical Specimens. <I>Journal of Clinical Microbiology</I>, Vol. 47, p3461-3465.</LI><LI><B>Hemvani N., Patidar V. &amp;  Chitnis D.S.</B> (2012). A simple and economical in-house phage technique for the rapid detection of rifampin, isoniazid, ethambutol, streptomycin, and ciprofloxacin drug resistance in <I>Mycobacterium tuberculosis</I>, directly on decontaminated sputum samples. <I>International Journal of Infectious Diseases</I>, Vol. 16, p232-236</LI><LI><B>Ishiguro T., Saitoh J., Yawata H., Otsuka M., Inoue T. &amp; Sugiura Y.</B> (1996). Fluorescence Detection of Specific Sequence of Nucleic Acids by Oxazole Yellow-Linked Oligonucleotides. Homogeneous Quantitative Monitoring of <I>in-vitro</I> Transcription. <I>Nucleic Acids Research</I>, Vol. 24, No. 24, p4992-4997</LI><LI><B>Jain P., Hsu T., Arai M., Biermann K., Thaler D.S., Nguyen A., González P.A., Tufariello J.M., Kriakov J., Chen B., Larsen M.H. &amp; Jacobs W.R. Jr.</B> (2014). Specialized Transduction Designed for Precise High-Throughput Unmarked Deletions in <I>Mycobacterium tuberculosis</I>. <I>American Society for Microbiology</I>, Vol. 5, No. 3</LI><LI><B>Sampson T., Broussard G.W., Marinelli L.J., Jacobs-Sera D., Ray M., Ko C-C., Russell D., Hendrix R.W. &amp; and Graham F. Hatfull G.F.</B> (2009). Mycobacteriophages BPs, Angel and Halo: Comparative Genomics Reveals a Novel Class of Ultra-Small Mobile Genetic Elements. <I>Microbiology</I>, Vol. 155, p2962-2977</LI><LI><B>Tufarielloa J.M., Maleka A.A., Vilchèzea C., Colea L.E., Ratnera H.K, Pablo A. Gonzáleza P.A., Paras Jaina P., Hatfulle G.F., Larsena M.H. &amp; William R. Jacobs W.</B>(2014). Enhanced Specialized Transduction Using Recombineering in <I>Mycobacterium tuberculosis</I>. <I>American Society for Microbiology</I>, Vol. 5, No. 3, p1-7</LI><P><B>Sean Ross Craig</B> (data analysis, cloning, restriction diagnostics, measurements &amp; uploading content to the wiki), <B>Elliott Parris</B> (measurements &amp; restriction diagnostics), <B>Rachel Wellman</B> (restriction diagnostics &amp; measurements) &amp;<B>Ariana Mirzarafie-Ahi</B> (cloning).</P><P>With thanks to <B>Dr. Vitor Pinheiro</B>, <B>Dr. Jane Nicklin</B>, <B>Bilkis Kazi</B>, <B>Barbara Steijl</B>, <B>Luba Prout</B>.</P></DIV></DIV><P>Back to main Research page</P><P>To find out more about our modeling</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>