<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Birkbeck_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Birkbeck/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="cssmenu"><UL><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/TeamMembers"><SPAN>Members</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Attributions"><SPAN>Attributions</SPAN></A></LI></UL><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Research"><SPAN>Research</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Description"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/pp"><SPAN>Background</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Experiments"><SPAN>Experiments</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Measurement"><SPAN>Measurements</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Modeling"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Results"><SPAN>Results</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Conclusion"><SPAN>Conclusion</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/BioBricks"><SPAN>BioBricks</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Basic_Parts"><SPAN>Basic Parts</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Composite_Part"><SPAN>Composite Parts</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Outreach"><SPAN>Outreach</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Practices"><SPAN>Human Practices</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Collaborations"><SPAN>Collaborations</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/InterLab Study"><SPAN>InterLab</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Notebook"><SPAN>Journal</SPAN></A></LI></DIV><DIV id="maincontainer"><H2>Modular Construction of Bacteriophage for Diagnostic Systems</H2><H3> Introduction and Aim </H3><P>Personal medicine and non-invasive biological measurements are the fastest growing areas in biotechnology. Viruses and bacteriophages are central to a number of biotechnology applications spanning diverse fields, from adenovirus for gene therapy in humans to M13 phages for use in lasers and batteries. These simple organisms have also provided us with much of our understanding of gene regulation and replication. The specific interaction that enables a virus to infect a host and carry out its multiplication process is at the heart of many new emerging technologies that have the capacity of providing breakthroughs in multiple fields of research.</P><P>Lambda phage is a well-characterised bacteriophage where all steps in the infection and life cycles are known. Multiple biotechnological tools based on it have also been developed (such as recombineering and Gateway® cloning). Our aim is to use the lambda bacteriophage to explore the avenue of producing a modular chassis. For example, broad host phage could be used for biosensing and biotechnological applications and high specificity could be explored for therapy. Host broadening and host reassignment can both (and have been shown to) be achievable by fusing genes from related phages (T4 / lambda) and by directed evolution.</P><H3> Experimental Approach </H3><P>Using available data on how different phages infect cells and provided the time restrictions, we will assess the modularity of the infection strategies as follows:</P><P>•	Bioinformatics analysis of key lambda genes involved in host specificity and infection mechanism will be used to analyse relating phylogeny to known mechanisms, to explore modularity / host reassignment or broadening host range.</P><P>•	Host specificity factors (i.e. tail fibre protein [<I>stf</I> gene, loci:lambdap27/lambdap28] and tip attachment protein J [J gene, locus: lambdap21]) will be targeted for engineering with a view to increase host infectivity (against a LamB / OmpC E. coli negative mutants) or to alter host specificity (against a second type species – potentially, <I>Photorhabdus luminescens</I> or a <I>Rhizobium</I> species as target). Another component of the λ-bacteriophage gene product which may facilitate in changing host specificity is <A href="https://2015.igem.org/Team:Birkbeck/Composite_Part">tail fibre assembly (TFA) protein</A>.</P><P>•	Engineering of phage cycle regulators will be performed to allow exogenous control of lytic / lysogenic cycle and to allow introduction of a biosensor.</P><H3> Impact of Findings </H3><P>We hope to establish lambda phage modularity as a proof-of-principle showing that phages can be engineered as biosensors that are cheap to make, stable for transportation and not requiring specialist knowledge to use. We see this as a potential route towards developing affordable point-of-care assays for pathogen detection.</P><P>Previous efforts so far have yielded great results regarding the proof of concept for quickly detecting bacteria using phages, and we would like to take the technology one step further by producing a single robust lambda phage-based multi-purpose chassis.</P><P>To find out more about our research</P><P>To find an overview on our project</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>