<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Birkbeck_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Birkbeck/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="cssmenu"><UL><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/TeamMembers"><SPAN>Members</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Attributions"><SPAN>Attributions</SPAN></A></LI></UL><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Research"><SPAN>Research</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Description"><SPAN>Overview</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/pp"><SPAN>Background</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Experiments"><SPAN>Experiments</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Measurement"><SPAN>Measurements</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Modeling"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Results"><SPAN>Results</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Conclusion"><SPAN>Conclusion</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/BioBricks"><SPAN>BioBricks</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Basic_Parts"><SPAN>Basic Parts</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Composite_Part"><SPAN>Composite Parts</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Outreach"><SPAN>Outreach</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Birkbeck/Practices"><SPAN>Human Practices</SPAN></A></LI><LI class="last"><A href="https://2015.igem.org/Team:Birkbeck/Collaborations"><SPAN>Collaborations</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/InterLab Study"><SPAN>InterLab</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:Birkbeck/Notebook"><SPAN>Journal</SPAN></A></LI></DIV><TITLE>jQuery UI Accordion - Collapse content</TITLE><DIV id="maincontainer"><H2>Notebook</H2><DIV class="accordion"><H3><A href="#">Week 1</A></H3><DIV><P>A joint collaboration between the UCL, Biohackspace &amp; Birkbeck iGEM teams were held in order to train team members in certain roles within each team. The organised boot camp took place over a week. Teams all met in mornings to carry out basic lab training. In afternoons, 3 specific training routes were followed; DIYbio, Software/Automation &amp; extra lab. Members from each formed afternoon groups.</P><H4>Monday 15<SUP>th</SUP> June</H4><H5>Morning Lab</H5><P>Monday morning consisted of a Lab safety induction &amp; a discussion on the standardised cloning strategy to be applied throughout the iGEM competition. The common major feature of each plasmid backbone has a prefix (to the 5' end of the biobrick) &amp; suffix (3' end of the biobrick). The prefix and suffix restriction sites are highlighted in <I>Figure 1</I>. The standardised engineering procedure works by cutting the vector backbone in the suffix region while inserts (if already cloned into a vector) are cut in the prefix and suffix (cartoon representation of reactions are displayed in <I>Figure 2</I>). The ligation of insert and vector backbone will yield a recombinant plasmid with a scar as the <I>XbaI</I>&amp;<I>SpeI</I> sites anneal.</P><P><I><B>Figure 1: Prefix &amp; Suffix Restriction Sites</B></I>.</P><P><I><B>Figure 2: Cloning Strategy</B></I>.</P><P> On the first day of the lab, each of the iGEM teams were split into 3 groups. Each sub team was investigating different promoters used in expressing <I>mrfp</I>; 
                    <UL><LI><A href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K823005"> Group 1 used pSB1C3 - BBa_J23101 as a vector.</A></LI><LI><A href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K823008"> Group 2 Used  pSB1C3 - BBa_J23106 as a vector.</A></LI><LI><A href="http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K823013"> Group 3 Used  pSB1C3 - Bba_J23117 as a vector.</A></LI></UL></P><P>This interlab study aims at using the same protocol in expression of <I>rfp</I> in <I>E. coli</I> cells. With reference to <I>Figure 3</I>, the 2006 Berkeley iGEM team characterised each of the promoters. iGEM teams across the globe will quantitatively measure the fluorescence of RFP &amp; GFP with respect to each promoter listed. This will generate a large data set and therefore a statistically more reliable conclusion of the original experiments.</P><P><LI><I><B><A href="http://parts.igem.org/Part:BBa_J23101">Figure 3: Results of the Berkeley 2006 iGEM Team</A></B></I>.</LI></P><P>On day 1, the main goal in the morning lab was to make a ligation reaction ready for transformation on day 2. The backbone (pSB1C3 based) had to be cut with <I>SpeI</I>&amp;<I>PstI</I> which linearised the vector backbone by cutting into the suffix(as highlighted in <I>Figures 1</I>&amp;<I>Figures 2</I>). The insert dervived from pSB1A2 by a double restriction digest using <I>XbaI</I>&amp;<I>PstI</I>. In order to verify reactions had worked, each of the double digests were ran on a 1% (wt/vol) agarose gel (100 V for 1 hr), visualised by ethidium bromide staining and visualising bands using a U.V. light source.</P><P>After reaction verification, a ligation was performed by using ~75 ng of insert to 25 ng of vector. The reaction was carried out at room temperature for 30 minutes and the heat killed at 80°C for 20 mins. Reactions were stored at 4°C overnight for transformation.</P><H5>Meet &amp; Greets</H5><P>All team members were invited to the anatomy building of UCL to meet each other. After a brief 30 minutes of introductions, A skype talk was held with Randy Rettberg (the president of the iGEM foundation). The talk consisted of Randy Rettbergs general background, interest in synthetic biology &amp; the origins of the iGEM competition. A Q &amp; A session was held after the talk.</P><P>A previous iGEM team member (UCL 2014, Georgia Bondy) held an interactive talk on the division of labour within groups. Emphasis was placed on the need for team members to take responsibility for <B>one aspect</B> of the project. The &quot;babies&quot; had to be nurtured by the team member who chooses to be responsible for each task.</P><H4>Tuesday 16<SUP>th</SUP> June</H4><H5>Morning Lab</H5><P>On tuesday morning, groups were divided into sub-groups with the different subgroups using different methods in transformation. For electroporation, the ligation reaction had to be de-salted prior to electroporation. The chemical transformation protocol did not require this step and 2 μL of ligation reaction was pipetted straight into the compotent cells. <I>E. coli</I> cells were out grown in SOC media for 1 hour before plating on LB media containing chloramphenicol (20 μg/mL) to select for the transformants. A negative control of cells treated with 2 μL of distilled water was carried out.</P><H5>Software</H5><P>The first talk in the software route was on &quot;automated synthetic biology&quot; by Chris Grant. The main issues in this talk were the streamlining of high-throughput processes by the use of robotics. The next talks were on computer modelling of metabolic pathways/genetic circuits (by Miriam Leon &amp; Rob Stanley). An interactive workshop took place after the talk with the repressilator system being used as an example.</P><H5>DIY Bio</H5><P>The overall aim of this set of workshops was to construct an open souce spectrophotometer. Software for data collection/manipulation was downloaded. The basic principles of 3D printing was explained. Components of the open source we constructed using 3D printing.</P><H5>Extra Lab</H5><P>In the extra lab sessions, the tricks of the synthetic biology trade were put to use. The use of molecular crowding agents (namely PEG) was used in order to spead up the ligation reactions. The use of PEG reduced the reaction times by 50%!</P><H4>Wednesday 17<SUP>th</SUP> June</H4><H5>Morning Lab</H5><P>The plates that were inoculated on Tuesday were checked for any colonies. <B>no growth was observed on the negative control plate</B>. The chloramphenicol provided the selective pressure that would promote the growth of <I>E. coli</I> cells that were transformed with plasmid. Single colonies were picked and grown in liquid media (5 mL LB containing 20 μg/mL chloramphenicol) for plasmid harvesting. It was noted that there was significantly more transformants in the electroporation method of transformation. It was difficult to pick single colonies off the electroporation plates.</P><H5>Software</H5><P>The principles of wiki design was discussed by Lewis Moffat (UCL 2014 iGEM team). The logic behind html coding was explained. As a workshop a spoof page was set up and images &amp; text were uploaded to the page. Everyone involved in the workshop got hands-on experience in directly uploading &amp; editing a wiki page.</P><H5>DIY Bio</H5><P>A tour of London Biohackspace was given to people involved in this group. A talk on DIYbio was given by Ilya Levantis. The parts previously constructed for the open source spectrophotometer were put together.</P><H5>Extra Lab</H5><P>Plates that were inoculated on Tuesday were checked for any transformants. The practical was followed by a talk on advanced assembly techniques (by Dr. Vitor Pinheiro).</P><H4>Thursday 18<SUP>th</SUP> June</H4><H5>Morning Lab</H5><P>Overnight cultures of the transformants were ready to harvest! A glycerol cell stock of <I>E. coli</I> transformants was made by harvesting 0.5 mL of cells and mixing with 0.5 mL of 50% (vol/vol) sterile glycerole (25% final [glycerol]). 1 mL of culture was lysed and used to harvest plasmid DNA.</P><P>Purified plasmids were screened for the presence of the insert by a diagnostic restriction digest. This restriction digest effectively liberates the insert from the vector and should yield 2 clear bands. <I>EcoRI</I>&amp;<I>PstI</I> were used to liberate the insert (note that where ligation occurs between <I>XbaI</I>&amp;<I>SpeI</I> in the suffix forms a scar, therefore the <I>EcoRI</I> restriction site in the suffix must be used as a diagnostic - refer to <I>Figure 1</I>). Agarose gels were ran of the digests. Varying degrees of success was observed.</P><H5>Software &amp; Extra Lab</H5><P>The extra lab &amp; Software groups had a skype interview with Jacob Beal about the interlab study. He laid out the main aims of the interlab study and answered questions that teams members had over the project.</P><H5>DIY Bio</H5><P>The open source spectrophotomoeter was tested. A strain of <I>E. coli</I> expressing <I>rfp</I> from a plasmid was used to test the spectrophotometer.</P><H4>Friday 19<SUP>th</SUP> June (<B><EM>THE MINI JAMBOREE!</EM></B>)</H4><P>The time had came to showcase the skills which we had learned over the week. The iGEM teams met up in the morning in order to prepare for the presentations in the afternoon. Each of the iGEM teams had to also discuss their biobrick of choice related to their project. Each presentation lasted 5-10 minutes.</P><P>The teams that were split into different afternoon workshops also had to present what they have learned over the course of the week. Each teams presentation lasted 5-10 minutes (with exception to the software team lasting ~3 minutes - short &amp; sweet!).</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 2</A></H3><DIV><H4>Wednesday 24<SUP>th</SUP> June</H4><P>The Birkbeck 2015 iGEM team visited William Tyndale Primary School in order to carry out a “scientific” busk. The Birkeck team and other current students/graduates of Birkbeck College presented on a wide variety of scientific topics.</P><P>There were 3 talks:</P><OL><LI><P>The importance of cleanliness and application of microbes in industry.</P><P>a.   Kids took part in an activity using glitter to illustrate the epidemiology of bacteria being passed in a population through contact. Clapping &amp; shouting was involved in this activity which the kids animatedly took part in.</P><P>b.   It was also explained that not all microbes are bad &amp; that in fact some are actually used in the production of food.</P></LI><LI><P>Imaging, Geology &amp; Planetary Science.</P><P>a.   Kids interacted enthusiastically with an activity involving piecing together an image of a planet. </P><P>b.   Real geological samples were on display, with a Birkbeck Geology graduate describing the fossil&amp; rock samples on show.</P></LI><LI><P>Genetic Inheritance, DNA Structure &amp; DNA function.</P><P>a.   The use of coloured beans (to represent genes) and paper flowers (to represent the phenotype) was used in an activity to show the nature of inheritance.</P><P>b.   Kids took part in forming a human DNA double helix.</P><P>c.   Kids took the role of detectives in the “Break the Code” DNA game. Paper models of a DNA with a secret code was used to illustrate the different functions of DNA based on the sequence and reinforce the structure of DNA.</P></LI></OL><P>Along with teaching the kids a little science (although the level of understanding &amp; knowledge the kids had was grossly under estimated) an important message was conveyed by each group. “What actually interested each individual in science?”, was used to try to inspire the kids into perhaps studying science at degree level. Many people presenting were from all walks of life, ages and reasons for being enthusiastic about their specialized subject area.</P><H4>Thursday 25<SUP>th</SUP> June</H4><P>Team Birkbeck were inducted into Lab303/307 at the main building at Birkbeck College (Mallet Street). The induction involved training with lab equipment (such as centrifuges, pH meter, plate readers &amp; PCR machines), the disposal of waste materials, procedures to follow in emergencies,  staff members to contact for particular problems &amp; location of consumables (including logging consumable usage). It was agreed that work in the lab would commence Monday 6<SUP>th</SUP> July.</P><P>Team meeting to disuss roles in individuals are playing in the team. People with previous tasks updated the team to the progress of their work.</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 3</A></H3><DIV><H4>Friday 3<SUP>rd</SUP>July</H4><P>Team meeting was had in order to see where everyone was. The main focus was on sponsorship. A sponsorship package has to be made by Monday morning &amp; student profiles finished off. It was agreed that Wayne constructed the sponsorship package &amp; everyone made their own profiles. The profiles were to be sent to Sean (who will upload to the appropriate wiki page).</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 4</A></H3><DIV><H4>Monday 6<SUP>th</SUP>July</H4><P>All the ground work for the lab work was set up. An untold amount of all pipette tips were autoclaved. 2 L of LB agar was prepared was prepared for later use. The only antibiotic available at the moment is chloramphicol.</P><H4>Tuesday 7<SUP>th</SUP>July</H4><P>The media prepared yesterday had to melted. 500 mL was used as plain LB plates. Another 500 mL batch of LB was supplemented with 25 μg/mL of chloramphenicol.</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 5</A></H3><DIV><H4>Monday 13<SUP>th</SUP>July</H4><H5>Pinheiro Lab</H5><P>Having finally got the lab prepared, it was time to start creating some BioBricks. First up, the most basic one: ORF314, (a piece of) tail fibre protein of lambda phage. After cloning the gene into iGEM standard backbones, the recombinant plasmids were transformed into competent cells…</P><H5>Lab 307</H5><P>A prelimenary growth curve of <I>E. coli</I> 10β &amp;<I>E. coli</I> DH5α. The culture was measured at 2 different wavelengths (395 nm &amp; 600 nm). This was mainly to characterise the growth curve for any downstream experiments involving the exposure of bacteriophages to <I>E. coli</I>&amp; characterising the negative control for the <A href="https://2015.igem.org/Team:Birkbeck/InterLab_Study">Interlab Study</A>.</P><P>The experiment took 3 hours to reach exponential growth. The experimental procedure was adapted in order to minimise the time taken for cells to reach the exponential phase of growth.</P><H4>Tuesday 14<SUP>th</SUP>July</H4><H5>Pinheiro Lab</H5><P>One day in and time for the first disappointment! None of yesterday’s transformations worked, and an agarose gel didn’t show a hint of our DNA. So back to the start – another attempt at restriction-ligation of our gene into the standard shipping and assembly backbones, and another transformation. And then a nervous wait overnight to see if the second attempt would be successful!</P><H5>Lab 307</H5><P>Lab work was halted for 1 week. </P><H4>Wednesday 15<SUP>th</SUP>July</H4><H5>Pinheiro Lab</H5><P>Success! Well, a little of it, anyway. Some beautiful little clones have appeared on the kanamycin plates (containing cells with the assembly backbone); not so much luck for the chloramphenicol plates (which contain the shipping backbone). So for now, it’s time to set up some overnight cultures – we’ll just have to clone our gene over from the assembly to the shipping backbone for submission later.</P><P>The parts required for the payload were amplified by electroporation into <I>E. coli</I> 10β. Transformed <I>E. coli</I> 10β were plated on LB agar containing the appropriate antibiotic concentration and incubated at 37<SUP>o</SUP>C overnight.</P><H5>Lab 307</H5><P>Nothing practical happend in this lab. </P><H4>Thursday 16<SUP>th</SUP>July</H4><H5>Pinheiro Lab</H5><P>With all except one overnight culture showing clear signs of cell growth, it’s time to purify some plasmids. Easy-peasy, and by the end of the day, we have a good amount of purified BioBrick, so now it’s time to move on to the more difficult part of the process.</P><P>All the transformants for the payload showed signs of growth. For each component of the circuit, 3 colonies were pick and grown overnight for miniprepping.</P><H5>Lab 307</H5><P>Nothing practical happend in this lab. </P><H4>Friday 17<SUP>th</SUP>July</H4><H5>Pinheiro Lab</H5><P>After spending much of the week designing and redesigning a whole collection of primers and other DNA sequences to amplify our ORF314 gene (after all, we don’t want to run out, so the easiest way to amplify seemed through PCR) and to introduce a His-tag into the sequence. By the end of the day, we’re finally confident we’ve cracked it, and the order is put in. Now all we can do is wait until our order arrives!</P><P>Plasmids of overnight cultures of the payload circuit were harvested. Restriction diagnostics must be carried out prior to use.</P><H5>Lab 307</H5><P>Nothing practical happend in this lab. </P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 6</A></H3><DIV><H4>Monday 20<SUP>th</SUP>July</H4><H5>Pinheiro Lab</H5><P>Today, part of our team (Luba, Elliott, Ariana and Barbara) got a chance to learn how to make a bacteriophage preparation. Professor John Ward at UCL invited us to his lab to learn a bit more about how to culture phages in the lab – something that will no doubt come in very useful when we get to the phage directed evolution stage of our project!</P><H5>Lab 307</H5><P>The <I>gfp</I> part for the interlab study was picked up from the UCL team &amp; transferred to Lab 307. Nothing practical happend in this lab. </P><H4>Tuesday 21<SUP>st</SUP>July</H4><H5>Pinheiro Lab</H5><P>Our primers arrived! Time to get back into the lab for some hard graft. The first step was to introduce a pair of new restriction sites into our plasmid using PCR – which now contains the ORF314 gene – so we can carry out restriction to create suitable overhangs. The next step will be to introduce a His-tag into the gene.</P><H5>Lab 307</H5><P>The hard graft in Lab 307 begins! <I>E. coli</I> DH5α was streaked onto an LB plate for a growth curve on wednesday. at the same time, the transformation efficiency of the cells were determined  along with the amplification of the plasmid containing the <I>gfp</I> part for the interlab study.</P><P>The restriction diagnostic of each of the components for the payload was carried out. It appears that all parts are correct. The plasmid containing <I>tetR</I> did not fully digest with <I>EcoRI</I>&amp;<I>PstI</I>. On the gel it appeared that there was a high concentration of the <I>tetR</I> plasmid which was confirmed by nanodrop.</P><H4>Wednesday 22<SUP>nd</SUP>July</H4><H5>Pinheiro Lab</H5><P>Another delivery we received yesterday was a pair of DNA oligos we designed containing a His-tag. Today we annealed them, creating a short strand of dsDNA with overhangs compatible with those we’ve just created in our plasmid. Next step: ligation of the oligo and the plasmid, and just like that, we have a BioBrick containing a His-tag… Or so we hope. After purification it was time to transform this new plasmid into some more competent cells, so hopefully tomorrow we’ll have some brand new recombinant colonies.</P><H5>Lab 307</H5><P>Three separate colonies were picked from the <I>E. coli</I> DH5α streak. The three colonies are going to be used for the first negative control measurements for the <A href="https://2015.igem.org/Team:Birkbeck/InterLab_Study">Interlab Study</A>. One of the cultures will also provide the basis for a growth curve.</P><P>Three separate colonies from the <I>gfp</I> plasmid transformation were picked and grown overnight in 5 mL LB broth containing 100 μg/mL of ampicillin. The three cultures will be harvested after approximately 18 hours of growth. This plasmid is essential in generating the devices required for the <A href="https://2015.igem.org/Team:Birkbeck/InterLab_Study">Interlab Study</A>.</P><H4>Thursday 23<SUP>rd</SUP>July</H4><H5>Pinheiro Lab</H5><P>Colonies aplenty! Time to set up some overnight cultures; tomorrow we should be able to purify the plasmids from the cultures, and then we’ll have a second BioBrick (almost) ready for submission. Things are looking up ☺.</P><H5>Lab 307</H5><P>The optical densities at wavelengths of 395 nm, 475 nm &amp; 600 nm was measured from the triplicate cultures. Each of the triplicates was then diluted to an OD<SUB>600</SUB> = 0.5. One culture was of <I>E. coli</I> DH5α was diluted 1/10 in LB broth and used to inoculate triplicate flasks for a growth curve. The wavelengths investigated were; 395 nm, 475 nm &amp; 600 nm. Two viable counts were also conducted at times of; 1 hour &amp; 2.9 hours. The optical density was measured for 4 hours &amp; 40 minutes with a 24 hour measurement..</P><H4>Friday 24<SUP>th</SUP>July</H4><H5>Lab 307</H5><P>The final 24 hour ODs were taken &amp; viable count data tabulated.</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 7</A></H3><DIV><H4>Friday 31 July</H4><P>Birkbeck hosted the first-ever London iGEM Meetup! The teams of UCL, London Biohackers, University of Kent and University of Westminster joined us in central London for an afternoon of presentations. Each team got 20 minutes to present their project to the audience, explaining both their goals and their accomplishments to far. It proved excellent practice for the Giant Jamboree! Afterwards, there was plenty of time to admire each other's posters, ask questions and of course enjoy a drink and nibble or two.</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 8</A></H3><DIV><H4>3-7 August</H4><P>Having prepared our ORF314 for further processing, this week we planned to insert a His-tag into the plasmid, after which the plan was to add the ORF401 sequence so by the end of the week we would have a complete bacteriophage lambda stf (tail fibre protein) gene with a His-tag for easy purification. But alas! In typical scientific fashion, our brilliant plan utterly failed. Although the insertion of the His-tag through the clever use of oligos and a type 2 restriction enzyme appears to have worked well, the next step – excising a ~200 bp section of the plasmid and replacing it with the ORF401 using one blunt-end and one sticky-end restriction enzyme – appears to have resulted in… well, nothing. At least, on optimistically running an agarose gel, the only products appeared to be our original plasmid (containing ORF314 and the His-tag), the pSB1K3 backbone on its own, and ORF314 (with His-tag) on its own. Time to order some primers to amplify our ORF401 gene then – for now, it’s just a case of waiting until the primers arrive so we don’t run out of DNA!</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 9</A></H3><DIV><H4>10-14 August</H4><P>Our primers have finally arrived and we amplified our synthetic genes.</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 10</A></H3><DIV><H4>17-21 August</H4><P>Having amplified our synthetic sequences and our linearised backbones, we proceed to attempt a serious run of 3A assembly. Things were looking up... Which made it all the more disappointing to find out that just one of our five BioBrick plates grew any colonies! (we were attempting ORF314 with His-tag, the stf gene with and without His-tag, and tfa both as a standalone gene and in a circuit) The stf gene with His-tag was the only apparent success, but on running the product on a gel, we discovered that in fact we were dealing with empty vectors, with our gene nowhere in sight! Time for some troubleshooting...</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 11</A></H3><DIV><H4>24-28 August</H4><P>With the Giant Jamboree rapidly approaching, we finally have some great developments this week! On close examination of our gene sequences, it turned out the amplification of the various sequences had resulted in incompatible ends between the genes and the standard backbones. Annoying, but at least now we know and can address the issue... Luckily, this was not a problem for our tail fibre assembly (tfa) circuit, which we managed to clone into the shipping vector this week. Gel electrophoresis and sequencing confirmed it was really there, so we now have a brand-spanking-new composite part to our name! This was followed by a successful sequencing result for our cloning of a promoter into a previously-existing tetracycline repressor (TetR) circuit, so things are looking up with two composite parts and a basic part registered and ready. More to come next week!</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 12</A></H3><DIV><H4>1-4 September</H4><P>While our lab efforts continued determinedly, the Friday and Saturday saw us attend the first ever UK iGEM meet-up in London, organised by the University of Westminster. The two-day event allowed us to practice giving our presentation ahead of the 'real thing' at the Jamboree, unveil our final poster to the other iGEM teams in attendance, as well as offering helpful talks by several academics such as Paul Freemont from Imperial College London, and those having success in synthetic biology industry, such as Sean Ward from Synthace and Edward Perello from Desktop Genetics. Last but not least the free food provided by Westminster was incredible, as were the vodka chocolates. Altogether, it proved both a fun and very useful experience and we extend our thanks to Westminster for organising such a successful event!</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 13</A></H3><DIV><H4>7-11 September</H4><P>Things are moving forward, with several more BioBricks approaching submission readiness, including the tfa gene and the cI-cro lytic cycle regulation circuit. Meanwhile, our struggle to clone the completed stf gene into a vector continues...</P></DIV></DIV><DIV class="accordion"><H3><A href="#">Week 14</A></H3><DIV><H4>14-18 September</H4><P>The last week of lab and wiki work! As you can see, Sean and Elliott have been doing an admirable job making our wiki look pretty fantastic, and they're now putting the final touches on all their hard work. Meanwhile, Ariana has been editing our video interviews, designing our Jamboree T-shirts (how else will you recognise us there!) and helping Luba, who has been working like a machine to get all our BioBricks ready for submission before the deadline...</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>