Team:Birkbeck/Discussion
Team
Members
Attributions
Research
Overview
Background
Experiments
Measurements
Modelling
Results
Conclusion
Safety
BioBricks
Basic Parts
Composite Parts
Outreach
Human Practices
Collaborations
InterLab
Journal
Discussion
Signal Detection
Considering our results, at time 0 minutes, there is a significant signal for P1-gfp expression device compared to the E. coli DH5α (P=0.005, with reference to Fig. 9 in results section). This result cannot be regarded as this is probably carry over fluorescence from sub-culturing.  Photo-bleaching prior to the first reading may yield more accurate readings when considering the expression of gfp from promoters. Also the error bars in each of the expression devices do appear excessive. A means of reducing this may be to sub-culture the cells & grow to an OD600 of 0.2 and sub-culturing into experimental cultures in order to minimise the error bars.
One technique which has been developed is transcription-reverse transcription concerted reaction (TCR) (Ishiguro et al., 1996). TCR monitors transcription in a sequence-dependant manner in in-vitro systems (Ishiguro et al., 1996). The TCR protocol was adapted in order to improve sensitivity for detecting M. tuberculosis from clinical sputum samples (Drouillon et al., 2009 [TCR-2]). Considering P1-gfp culture at 20 minutes, there is a significant fluorescent signal (P=0.002) with no difference between the OD600 growth curves at this point (P=1).  This is significantly less than the hours taken for results to be generated by TCR-2 (Drouillon et al., 2009). Cells cannot be quantified at the 20 minutes time point in our study as no viable count was carried out, therefore the detection limit of the E. coli DH5α with regards to fluorescence cannot be determined.
With regards to Decontamination Kits , samples would take just under one hour to process. Of note, some bacteria (such as Staphylococcus aureus) can be resistant to decontamination (refer to the kit previously mentioned). A key point to note here is that the decontamination step must not be carried out for longer than 15 minutes due to the reduction in cell viability of Mycobacterium cells.
As transduction was not characterised in this study, it is difficult to assess exactly how long this step may take. Considering the application of D29 mycobacteriophage could be spotted directly onto soft agar plates giving rise to plaques on both M. smegmatis & M. tuberculosis (Sampson et al., 2009). A mycobacteriophage assay for detection of live Mycobacterium cells has previously been carried out using a plaque assay (Alcaide et al., 2003). A 1 mL decontaminated sample was washed and grown overnight in the standard Mycobacterium media Middlebrook 7H9 supplemented with 10% (vol/vol) oleic acid-albumin-dextrose catalase (OADC) (Alcaide et al., 2003). 100 μL of mycobacteriophage was added to the overnight cultures and incubated at 37oC for 1 hour before adding 100 μL of viricidal solution (Alcaide et al., 2003). A soft agar lawn was poured and incubated for 24 hours before counting plaques. The use of a fluorescent protein as a tag would perhaps remove the need for overnight incubation (in fact a 48 hour protocol).
Alcaide & co-workers did demonstrate that smear positive samples were extremely sensitive to the phage assay and claim that the detection limit was 10 cfu/mL with smear negative samples showing much higher detection limits. Another study using mycobacteriophages as a diagnostic tool used the cheaper medium Mueller–Hinton instead of Middlebrook 7H9 (Hemvani et al., 2012). The Hemvani study showed similar results to the Alcaide study. The change in media did however result in 3.5% of sputum samples being un-interpretable due to high levels of contamination even after treatment with vancomycin & polymyxin B (Hemvani et al., 2012).
Considering the presorption phase of the assay developed by Alcaide et al, this would add an hour onto the protocol. Given that transduction is 100% efficient (unlikely) and that the kinetics of gfp expression behaves the same in E. coli DH5α as does a Mycobacterium sp. cell (another wild assumption), we are looking at from sputum sample collection until signal detection at approximately 2-4 hours (given that decontaminated sputum samples could be directly exposed to recombinant bacteriophages). It must be noted that in our approach, the signal is produced by transduction of genetic material as opposed to amplification of phage particles being the signal. The use of transduction hass been successfully applied in genetic engineering of Mycobacterium sp. (Jain et al., 2014; Tufariello et al., 2014) which is promising for the development of a diagnostic tool in introducing a detectable "payload" for diagnostic purposes.
Considering our results, at 1 hour of culturing E. coli DH5α, there is a significant signal observed from the culture expressing the P1-gfp expression device (P<0.001). Considering the growth, there was no significant difference between the OD600 of the P1-gfp expression device & the E. coli without plasmid (P=1 for both 50 mL cultures & 200 μL, with reference to Fig. 1 & Fig. 5 respectively). At the 1 hour time point, there is a significant difference between the 50 mL & 200 μL cultures (P=0.017) with the 200 μL cultures showing a higher OD600. This entails that the fluorescence of the 200 μL cultures at 1 hour corresponds to a higher culture density than 2.57×106 cfu/mL.
Considering the TCR-2 method of detection, 2.57×106 cfu/mL is an increase in detection limit by a magnitude of 105 (30 cfu of M. tuberculosis/mL of sputum [Drouillon et al., 2009]) which is clearly an undesirable due to the slow growth of M. tuberculosis. There appears to be a reduction in fluorescence from the P1-gfp expression device after 20 minutes which might correspond to the depletion of mRNA/GFP. The increase in fluorescence when comparing the 20 & 40 minute time points may be the expression of gfp from each of the promoter increasing to a detectable level.
An interesting observation was made on the growth kinetics of the E. coli DH5α containing P1-gfp expression device & E. coli DH5α not containing any plasmid. Between 100-400 minutes, the E. coli DH5α containing P1-gfp expression device has a significantly lower OD601 & higher fluorescence when compared to E. coli DH5α. The significant difference is not observed in the positive control or P2-gfp expression device with limited detection of fluorescence. This data suggests that compromising cellular growth with regard to expression of a signal may yield a quicker detection of a signal with the product.
It is obvious that the P1 promoter will not be able to drive gene expression in all kinds of bacteria. The proposed T7-RNAP-driven rfp expression was not characterised as the circuit was not constructed due to cloning issues. However, the growth kinetics would be expected to be slower than that of the P1-gfp expression device. Fig. 1 is a diagramatic representation of the proposed genetic circuit for the characterization of a signal.
Fig. 1: TetR rfp Expression device.
The idea behind the circuit displayed in Fig. 1 is to have the amplification of rfp expression inducible. The induced expression of the amplification step in this circuit would mimic the infection of the E. coli cells by a recombinant λ-bacteriophage. This would allow different time points in the growth curve to be stimulated and signal intensities could be compared relative to the cell dose & physiological state of cells.
In the first part of the circuit displayed in Fig. 1, a strong constituitive promoter (such as pCAT) will drive the expression of tetR . The TetR will bind to tetO & repress the expression of T7-RNAP. The T7-RNAP would then drive the exmpreesion of rfp via T7 promoter .
The highly mutated rfp was chosen as a marker due to the ability of the protein to be observed on a plate as well as liquid culture without the use of fluorescence. The absorption of GFP could not be detected accurately (see Fig. 2 & Fig. 6-8 in the additional Interlab Results . This approach can't be applied to detecting an early signal however, left on a plate can yield very suggestive & qualitative results.
Another proposed way in which to characterize this circuit was to clone into a gt 11 λ-bacteriophage vector. with reference to Fig. 1, the right hand side of the line down the circuit (i.e. from the tetO) would be cloned into the gt 11 vector. There could be many possible time points to assess what part of the growth curve would yield the highest signal.
Reference
Signal Detection
Alcaide F., Gali N., Dominguez J., Berlanga P., Blanco S., Orus P. & Martin R. (2003). Usefulness of a New Mycobacteriophage-Based Technique for Rapid Diagnosis of Pulmonary Tuberculosis. Journal of Clinical Microbiology Vol. 41, No. 7, p2867-2871.
Drouillon V., Deloogu G., Dettori G., Lagrange P.H., Benecchi M., Houriez F., Baroli K., Ardito F., Torelli R., Chezzi C., Fadda G. & Herrman J-L. (2009). Multicenter Evaluation of a Transcription-Reverse Transcription Concerted Assay for Rapid Detection of Mycobacterium tuberculosis Complex in Clinical Specimens. Journal of Clinical Microbiology, Vol. 47, p3461-3465.
Hemvani N., Patidar V. & Chitnis D.S. (2012). A simple and economical in-house phage technique for the rapid detection of rifampin, isoniazid, ethambutol, streptomycin, and ciprofloxacin drug resistance in Mycobacterium tuberculosis, directly on decontaminated sputum samples. International Journal of Infectious Diseases, Vol. 16, p232-236
Ishiguro T., Saitoh J., Yawata H., Otsuka M., Inoue T. & Sugiura Y. (1996). Fluorescence Detection of Specific Sequence of Nucleic Acids by Oxazole Yellow-Linked Oligonucleotides. Homogeneous Quantitative Monitoring of in-vitro Transcription. Nucleic Acids Research, Vol. 24, No. 24, p4992-4997
Jain P., Hsu T., Arai M., Biermann K., Thaler D.S., Nguyen A., González P.A., Tufariello J.M., Kriakov J., Chen B., Larsen M.H. & Jacobs W.R. Jr. (2014). Specialized Transduction Designed for Precise High-Throughput Unmarked Deletions in Mycobacterium tuberculosis. American Society for Microbiology, Vol. 5, No. 3
Sampson T., Broussard G.W., Marinelli L.J., Jacobs-Sera D., Ray M., Ko C-C., Russell D., Hendrix R.W. & and Graham F. Hatfull G.F. (2009). Mycobacteriophages BPs, Angel and Halo: Comparative Genomics Reveals a Novel Class of Ultra-Small Mobile Genetic Elements. Microbiology, Vol. 155, p2962-2977
Tufarielloa J.M., Maleka A.A., Vilchèzea C., Colea L.E., Ratnera H.K, Pablo A. Gonzáleza P.A., Paras Jaina P., Hatfulle G.F., Larsena M.H. & William R. Jacobs W.(2014). Enhanced Specialized Transduction Using Recombineering in Mycobacterium tuberculosis. American Society for Microbiology, Vol. 5, No. 3, p1-7
Sean Ross Craig (data analysis, cloning, restriction diagnostics, measurements & uploading content to the wiki), Elliott Parris (measurements & restriction diagnostics), Rachel Wellman (restriction diagnostics & measurements) & Ariana Mirzarafie-Ahi (cloning).
With thanks to Dr. Vitor Pinheiro, Dr. Jane Nicklin, Bilkis Kazi, Barbara "Babz" Steijl, Luba "Aunty" Prout.
