<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_RHIT_GeneticsModel skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:RHIT/GeneticsModel</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="globalWrapper"><DIV id="bannerContainer"><DIV id="mainContainer"><DIV id="contentContainer"><DIV class="footer"><DIV class="column footer_size"><CENTER><H6> Connect with us on Facebook! </H6></CENTER></DIV><DIV class="column footer_size"><CENTER><H6> Contact Us! </H6><P style="text-align: center"> patekm@rose-hulman.edu  5500 Wabash Ave.  Terre Haute, IN 47803 </P></CENTER></DIV></DIV><DIV class="column third_size"><CENTER><H4 style="cursor:pointer" id="gen">Genetics Model </H4></CENTER></DIV><DIV class="column third_size"><DIV class="popup" onclick="PopUp()"><SPAN class="popuptext" id="myPopup" style="position:absolute; top:20px; left:300px; width:500px">  Click the &quot;Genetics Model&quot; title to toggle our visual
</SPAN></DIV></DIV><DIV class="column full_size"><P>The DNA coding for the 6 enzymes required for breakdown and assimilation of PET was too long to fit on one plasmid. To rectify this and to be able to test smaller subsystems, the PETase and MHETase genes were placed on Backbone 1, Plasmid 1. The Glycolaldehyde Reductase, Glycolaldehyde Dehydrogenase, Glycolate Oxidase, and Malate Synthase were placed in sequence on Backbone 2, Plasmid 2.</P></DIV><DIV class="column full_size"><H1>Plasmid 1</H1></DIV><DIV class="column half_size"><DIV class="container"><DIV class="overlay"><DIV class="text"> Repressed System </DIV></DIV></DIV></DIV><DIV class="column half_size"><DIV class="container"><DIV class="overlay"><DIV class="text"> Activated System </DIV></DIV></DIV></DIV><DIV class="column full_size"><P>Plasmid 1 uses an AraC and pBAD promoter to regulate the expression of PETase and MHETase. The transcription factor made from AraC usually binds and represses the pBAD promoter, halting transcription of the plasmid. The inducer, arabinose, can be added to the media, and this molecule binds to the AraC transcription factor on the DNA strand and changes its conformation so that transcription can occur [1]. The reaction scheme on the left explains a more complete mechanism of the transcription/translation of these proteins. The creation of AraC protein is related to a constitutive promoter which we assume enters the system as a constant rate, K. This method was also used by the UC Davis iGEM team in 2012. We assumed a fast-equilibrium hypothesis on the formation of the dimer and an essentially constant pool of arabinose in the environment. We can also streamline the binding of the two arabinose inducers to the AraC dimer into one reaction determined by the rate parameters k3+ and k3-. Since the amount of RNA polymerase does not change relative to these molecules, and since the frequent assumption used in literature is to group the transcription and translation rate into one overall rate of protein production, we simplify the system further. From these assumptions, we can simplify the system down into the system shown on the right.</P><H3>Model Equations:</H3><H3> Simulations: </H3><CENTER>
Table 1. List of values used in simulation parameters</CENTER><P>Figure 1 maps out the expression level of the free repressor protein AraC and Plasmid 1 genes, PETase and MHETase over 12 hours. The population of cells in the simulation was 1e+08. While free AraC protein, which would normally suppress expression, is relatively low there is a high expression of the objective proteins. The MHETase and PETase plots both follow the same trend, although, MHETase does decay slower than PETase and also reaches a higher maximum concentration. </P><CENTER>
Figure 1. Expression of AraC promoter in presence of Arabinose in Plasmid 1</CENTER><H3> Sensitivity Analysis: </H3><P>In Table 1, there are three values that say they are estimated in the calculated column. This is due to the lack of comprehensive material on them, and that these values create realistic behavior of the system. There is a small range in each value where the system remains stable and biologically relevant, so these values were pulled from that range. The behavior of the system throughout the range is consistent with only slight variation on amounts. </P><H3>References:</H3><UL><LI>[1] D. Madar, E. Dekel, A. Bren and U. Alon. “Negative auto-regulation increases the input dynamic-range of the arabinose system of Escherichia coli,” BMC Systems Biology. 2011. [Online]. https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-5-111 </LI><LI>[2] L. Wang, Y. J. Zhou, D. Ji, and Z.K. Zhao, “An accurate method for estimation of the intracellular aqueous volume of Escherichia coli cells,” Journal of Microbiological Methods, 2013, p. 8. [Online].http://bionumbers.hms.harvard.edu/bionumber.aspx?id=108813&amp;ver=3&amp;trm=e%20coli%20cell%20volume&amp;org</LI><LI>[3] H. Bremer and P. P. Dennis, “Modulation of chemical composition and other parameters of the cell by growth rate.” Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd ed. [Online]. Neidhardt, et al. eds. chapter 97, 1991, p. 1559. [Online]. http://bionumbers.hms.harvard.edu/bionumber.aspx?&amp;id=100059&amp;ver=39</LI><LI>[4] X. Zhang, T. Reeder, and R. Schleif. Transcription Activation Parameters at ara pBAD.” Journal of Molecular Biology. Vol 258, 1996, p. 14-24. [Online]. http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.417.6037&amp;rep=rep1&amp;type=pdf</LI><LI>[5] D. Kolodrubetz and R. Schleif. “Identifical of araC protein on two-dimensional gels, its in vivo instability and normal level.” Journal of Molecular Biology. Vol 149, issue 1, pp.133-139, 1981. [Online].  https://www.sciencedirect.com/science/article/pii/0022283681902655 </LI><LI>[6] J.A. Mergerle, F. Georg, et al. “Timing and Dynamics of Single Cell Gene Expression in the Arabinose Utilization System.” Biophysical Journal. Vol. 95, Issue 4, pp.2103-2115, Aug 2008. [Online]. https://www.sciencedirect.com/science/article/pii/S0006349508701681 </LI><LI>[7] R. Schleif. “AraC protein, regulation of the L-arabinose operon in Escherichia coli, and the light switch mechanism of AraC action.” FEMS Microbiology Review. vol. 34, issue 5, September 2010, pp. 779-796. [Online]. https://academic.oup.com/femsre/article/34/5/779/797770</LI><LI>[8] K. Martin, L. Huo, and R.Schleif. “The DNA Loop model for ara repression: AraC Protein occupies the proposed loop sites in vivo and repression-negative mutations lie in these same sites.” In USA Proceedings National Academy of Science. vol. 82, June 1986, pp. 3654-3658. [Available: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC323581/pdf/pnas00315-0095.pdf]</LI><LI>[9] “Part: pSB1C3.”iGEM Registry of Standard Biological Parts. Sept 2008. [Online]. http://parts.igem.org/Part:pSB1C3</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>