<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WHU-China_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WHU-China/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WHU-China/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WHU-China/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WHU-China/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WHU-China/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WHU-China/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="center"><DIV class="middle"><DIV class="main"><DIV class="passage divcell0"><H3>
								Project Description
							</H3><P><I> The above figure is the overview of our design</I></P><P align="justify">
								Utilizing human microbiota to tackle diseases has long been the keen desire of scientists. This year, we WHU-China team engineered a kind of probiotics &quot;<I>E. coslim</I>&quot; from <I>Escherichia coli</I>, hoping to provide a new approach for treating obesity. Specifically, three genetic devices were designed to help people lose weight. The first two devices were assembled to metabolize the excessive energy in human intesinte. To achieve this goal, promoters activated by high concentration of glucose and fatty acids were devised and synthesized respectively. When functional genes were placed downstream of these promoters appropriately, the first two devices were supposed to degrade fatty acids and convert glucose into cellulose respectively, thus preventing excessive calorie intake as well as producing prebiotics. Meanwhile, the third device was designed to regulate human gut microbiota. Also, xylose inducing death device was planned to terminate the effects of &quot;<I>E.coslim</I>&quot; at will to prevent horizontal gene transfer and control the density of &quot;<I>E. coslim</I>&quot; in future applications. As a whole, we are developing &quot;<I>E. coslim</I>&quot; to metabolize the excessive energy and regulate the microbiome composition in intestine to control obesity.
							</P></DIV><DIV class="passage divcell1"><H3>
									Obesity: a severe global problem
						  </H3><P align="justify">
									Obesity refers to a health condition that body fat is accumulated to some
									extent. According to WHO, body mass index (BMI) is an index of weight-for-height
									that is commonly used to classify obesity in adults. It is a risk factor
									for various diseases, such as cardiovascular diseases (mainly heart disease
									and stroke), type 2 diabetes, musculoskeletal disorders (especially osteoarthritis),
									some cancers (endometrial, breast, and colon).
								</P><P align="justify">
									As it is shown in figure 1 and 2, a large amount of people from all over
									the world are overweight in both developed and developing countries
									and it is and will become more and more severe.
								</P><I>
									Figure 1(from reference [4]): Past and projected prevalence of overweight
									(BMI ≥25 kg/m2)
									<I>
										Figure 2: Prevalence of obesity in different countries.
										
										(Picture from The Wellington Grey blog)
									</I></I><H4>
									The Cause of Obesity
						  </H4><P align="justify">
									Obesity is most commonly caused by a combination of different factors such as excessive food energy
									intake, lack of physical activity, and genetic susceptibility, although
									a few cases are caused primarily by genes, endocrine disorders, medications
									or psychiatric illness.
								</P><P align="justify">
									However, the problem of obesity emerged globally only several decades
									ago. Since the change of genome of a species requires a long time, the
									outbreak of obesity is unlikely to be caused by changes in human genome.
									For most individuals, controlling food intake and doing physical activity
									in a proper way are effective strategies to lose weight. But for some people
									whose health condition or current life pace keep them away from systemic
									and regular exercise and diet, modulating the composition of microorganisms
									in intestine might act as an alternative.								</P><P align="justify">
									Reports by Gordon have shown that, apart from human genome, the collective
									genome of microorganisms (microbiome) in human intestine is associated
									with our obesity [1]. Furthermore, microbiome is able to be changed through
									control of food intake [1].
								</P><P align="justify">
									Two groups of beneficial bacteria are dominant in the human gut, the Bacteroidetes
									and the Firmicutes. The relative proportion of Firmicutes increases
									in obese people by comparison with lean people [2].								</P><I>
									Figure 3: How excess of energy contributes to obesity
								</I><P align="justify">
								Pertinent study by Gordon attested their initial hypothesis that changes
								in microbial component have a causal relationship with obesity, thus might
								have potential therapeutic implications [2] [3]. Colonization of germ-free
								mice with an ‘obese microbiota’ results in a significantly greater increase
								in total body fat than colonization with a ‘lean microbiota’ [3].
								<I>
									Figure from reference [3]
								</I></P><H4>
									Present strategies to lose weight
						  </H4><P align="justify">
									Diet, exercise, drug and surgery are major ways to lose weight. However,
									they all have many drawbacks. Diet may cause nutritional imbalance and
									can be a heavy mental burden since the people may not be able to enjoy
									the food they want. Exercise requires regular time and is ineffective in
									many cases. Drugs and surgery may have many side effects and are many times
									more costly.
								</P><H4>
									Our idea
								</H4><P align="justify">
									Previous situations and insights construct our theoretical fundament.
									We tried to utilize synthetic biology to provide a cheap, convenient, effective
									and safe approach for treating obesity. Instead of passive alternation
									of microbiota, we tried to construct an engineered E.coli----- E.coslim
									to positively change microbiota in intestine. As Figure 3 shown, we place
									E.coslim in the role of sensing and consuming excessive energy, which thus leads
									to the double effects: lowering the proportion of Firmicutes and increasing
									that of Bacteroidetes, and decreasing the energy available in one’s intestine.
									</P><P align="justify">
									To achieve these two goals, we designed four devices, fatty acids consumption,
									cellulose synthesis, colonization and death device of E.coslim.
								</P><H4>
										References
						  </H4>
									[1] Ruth E. Ley1, Peter J. Turnbaugh1, Samuel Klein1 &amp; Jeffrey I. Gordon1
									Microbial ecology: Human gut microbes associated with obesity. Nature 444,
									1022-1023 (21 December 2006)
									
									[2] Peter J. Turnbaugh1, Ruth E. Ley1, Michael A. Mahowald1, Vincent Magrini2,
									Elaine R. Mardis1,2 &amp; Jeffrey I. Gordon1 An obesity-associated gut microbiome
									with increased capacity for energy harvest. Vol 444|21/28 December 2006|
									doi: 10.1038/nature05414
									
									[3] Ley RE. Obesity and the human microbiome. Curr Opin Gastroenterol.
									2010 Jan; 26(1):5-11.
									
									[4] Y Claire Wang et.al. Health and economic burden of the projected obesity
									trends in the USA and the UK. Lancet. 2011
									</DIV><DIV class="passage divcell2"><A name="Introduction"><H3>
									Introduction
								</H3></A><P align="justify">  To metabolize the excessive energy, fatty acids and sugar should be the primary targets for our E.coslim to help people lose weight. 
								For E.coslim to degrade fatty acids and convert glucose into cellulose, they must be able to sense and get regulated by high concentration of glucose and fatty acids.
								Otherwise, the system may not only be inefficient but also may cause serious problems. However, there is no promoter available in nature
								that can be solely regulated by glucose or fatty acids.
								Therefore, to achieve our goals, we designed an indirect pathway and a direct synthetic promoter
								to sense and be regulated by glucose concentration. Also another synthetic promoter was designed to sense and be regulated by fatty acids.
								</P><A name="Indirect Pathway Design"><H3>
									Indirect Pathway Design
								</H3></A><P align="justify">In a cell, the total amount of ATP, ADP and  AMP molecules remains constant. Low glucose concentration results in high activity of adenylate cyclase converting ATP into cAMP, who binds to and converts cAMP receptor protein (abbreviated as CRP) to DNA-binding configuration. Conversely, when glucose concentration gets high, more ATP and less cAMP will be produced, resulting in low DNA-binding activity of CRP.</P><P align="justify">We embeded gene cI of lambda phage(<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_P0451">BBa_P0451</A>) downstream promoter PcstA (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K118011">BBa_K118011</A>), which can be activated by the binding of CRP, and genes of red fluorescence protein(RFP, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I13507">BBa_I13507</A>)  respectively downstream the promoter BBa_R0051 repressed by protein cI. In this way we constructed an indirect regulation pathway with glucose sensors, transcription activator CRP and transcription repressor cI. If the device works as design, output of RFP will increase following the elevation of glucose concentration, and vice versa.</P><P align="center"> The indirect regulatory pathway</P><P align="justify"><STRONG>Method</STRONG></P><P align="justify"><STRONG>Construction of plasmid for indirect  regulation pathway</STRONG></P><P align="justify">In this experiment, RFP reported the  function of the indirect regulation pathway.</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861173">BBa_K861173</A>: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I13507">BBa_I13507</A>, an mRFP generator with RBS and terminator, was  embedded after CRP activated promoter <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K118011">BBa_K118011</A>.</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861172">BBa_K861172</A>: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_P0451">BBa_P0451</A>, a cI generator with RBS and terminator,  was embedded after promoter <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K118011">BBa_K118011</A> activated by CRP.</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861169">BBa_K861169</A>: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861172">BBa_K861172</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I763007">BBa_I763007</A>, a cI repressed RFP generator, were assembled .</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861174">BBa_K861174</A>: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K137115">BBa_K137115</A>, constitutively  expressing cI generator with promoter, RBS and terminator, was assembled  to <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I763007">BBa_I763007</A></P><P align="justify">All new composite parts mentioned above  were transformed   to competent cells  of Escherichia coli str. DH5a. All positive clones were validated using PCR, restriction enzyme  digestion and DNA sequencing.</P><P align="justify"><STRONG>Cell culture fluorescence measurement</STRONG></P><P align="justify">Minimal medium with different concentrations  of glucose(1mM, 4mM, 10 mM , 20 mM , 50 mM ,100 mM) was transferred into a 96-well plate, 200 μL for each well. Then each well was inoculated with 2 μL of seed liquor which was activated overnight in M9 minimal medium with 50mM  glucose at 37℃. The wells without inoculation were regarded as blank controls to revise the results. Under each condition, three parallel samples were set. The plate was incubated at 37℃, 150rpm. Cell culture fluorescence was recorded on a SpectraMax M2 plate reader (Molecular Devices). Excitation at 584 nm and emission at 607 nm were used. All fluorescence results were normalized with cell density by measuring the absorbance at 600 nm.</P><P align="justify"><STRONG>Capturing  fluorescence  image </STRONG></P><P align="justify">Cell morphology was observed  through fluorescence microscope, and the images of bacteria  with different  glucose concentrations were captured.</P><P align="justify"><STRONG>Fluorescence analysis of cyto-imaging</STRONG></P><P align="justify">A program named FANCY was  designed to recognize single cell and calculate the fluorescence strength according to the images. For more information, please click  <A href="https://2012.igem.org/Team:WHU-China/Modeling?catalog=3">Here</A>.</P><P align="justify"><STRONG>Results</STRONG></P><P align="justify">Purified  plasmids  constructed before were digested with XbaI and PstI for confirmation. The agarose gel electrophoresis showed that the lengths were correct. At last, the plasmids were sent for sequencing. Results showed no mutation.</P><P align="center">The result for fluorescence measurement of indirect device</P><P align="justify">In the cell culture fluorescence measurement experiment, fluorescence of BBa_K861173 decreased  coordinating with glucose concentration, while BBa_K861169 increased. The fluorescence of BBa_K861174 was too low to record, so we do not show it here. All of the results coincided with expected results indicating that we have successfully constructed the promoter which was activated by high concentration of glucose.</P><P align="center">The fluorescence image of indirect device in different concentration of glucose</P><P align="justify">Fluorescence images indicated that all cells were growing normally, because the size and morphology were both the same as that of the cells in LB medium. The fluorescence of the cells in the images showed the same discipline as results from the fluorescence measurement experiments. 
						      The results of FANCY are showed as bellow, fluorescence intensity of PcstA+cI+RFP increased with the glucose concentration, while that of the PcstA+RFP decreased with glucose concentration. It conformed well with the results that showed above.</P><P align="center">The fluorescence intensity of indirect device from program FANCY</P><P align="justify"> </P><P align="justify"><STRONG>Discussion </STRONG>
						      All results of the three experiments  indicated the device worked as expected. Next, RFP will be replaced with genes of  cellulose synthesis. So the excess glucose can  be transformed into cellulose. 
						      Although the indirect regulation pathway was tested effective,it  worked through an intermediate product, protein cI. This determines that the  device will be less sensitive to glucose than a direct regulation pathway  without intermediate.</P><P align="left"> </P><A name="Direct Regulatory Promoter Design"><H3>
									Direct Regulatory Promoter Design								</H3></A><P align="justify"><STRONG>1. Glucose sensor</STRONG></P><P align="justify">To turn CRP into a repressor, we firstly consulted papers about CRP binding site and found out that 22-bp palindromic consensus site of the sequence AAATGTGATCT*AGATCACATTT. Unfortunately, this consensus sequence contained XbaI restriction site. We then designed the binding site by changing the most frequent bases in the binding site into second most frequent bases. Specifically, two binding sites were designed: AAATGTGATTTAAATCACATTT, AAATGTGATTATAATCACATTT.</P><P align="justify">Firstly, to construct a promoter that can be directly repressed by CRP, we inserted modified CRP binding site between -35 and -10 region of promoter BBa_ J23110 (TTTACGGCTAGCTCAGTCCTAGGTACAATGCTAGC). However, it did not work out as we expected. In our reporter assay, this promoter failed to express higher level of RFP when bacteria grew in M9 medium with higher glucose concentration.</P><P align="justify">Then, we tried a different strategy. We designed another promoter overlapping with CRP binding site to satisfy the sequence of -10 region of the promoter (TTGACAGCTAGCTCAAATGTGATTTAAATCACATTT). The promoter also failed to show the desired function, the expression of RFP showed that the promoter was unstable.</P><P align="justify">Finally, we designed a promoter with modified CRP binding site overlapping with the -10 region of the promoter. To satisfy better CRP binding, we firstly changed the sequence in -10 region to meet the CRP consensus sequence (TTGACAGCTAGCTCAAATGTGATTATAATCACATTT). We named it <I>Pcar</I>, which was short for <I>Promoter of CRP As a Repressor</I> . This time, Pcar showed exciting properties as we expected.</P><P> The above figure shows the design of the Pcar</P><P align="justify"><STRONG>2. Fatty acids sensor</STRONG></P><P align="justify">In fact, fatty acids sensor had already been tried by iGEM2006_Tokyo_Alliance (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J54220">BBa_J54220</A>).Yet their design failed to give desirable function.  To design a promoter that is under the sole regulation of fatty acids concentration, double FadR binding sites in promoter of FadL were placed and overlapping with the downstream of constitutive promoter J23110 (TTTACGGCTAGCTCAGTCCTAGGTACAATGCTAGCTGGTCCGACCTATACTCTCGCC ACTGGTCTGATTTCTAAGA). We named the promoter PfadR. Also, we planned to overexpress FadR to prevent leaky expression of the promoter.</P><P> The above figure shows the design of the PfadR</P><A name="Pcar"><H3><I>Pcar</I></H3></A><P align="justify">Although the indirect regulation pathway was tested  effective, we went on attempting a more compact and widely useful direct  regulation design. Hence we altered a  constitutive promoter (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23119">BBa_J23119</A>) into a CRP repressible one. We also established a new technical standard for our strategy of repressible promoter design (for more information, click on <A href="https://2012.igem.org/Team:WHU-China/Project?catalog=2#Standard">Standard</A>),  but we shall focus on the design itself now.</P><P align="justify">We designed promoter Pcar based on promoter  BBa_J23119, inserting CRP-binding site and overlaping it with six base pairs in the -10 region of promoter. Since steric hindrance of CRP dimer blocks the function of -10 region, gene downstream will be repressed when glucose concentration is low. That is, most CRP appears in DNA-binding configuration. The repressive effect is undermined when glucose concentration increases. Accordingly,  we changed CRP from an activator to a repressor, simplifying the device with potential advantages of higher sensibility and efficiency. As experimental results showed, promoter Pcar worked as we expected.						  </P><P align="justify"> </P><P align="center">The direct regulatory pathway</P><P align="justify"><STRONG>Methods</STRONG></P><P align="justify"><STRONG>Design of the promoter Pcar which is  activated by glucose </STRONG></P><P align="justify">Promoter Pcar, glucose biosensor plasmid, is derived from a constitutive promoter (BBa_J23119) by adding a CRP binding site upstream the promoter where has several base pairs overlapping with RNA polymerase binding site. The sequence was synthesized with restriction  enzyme cutting site for EcoRI and XbaI at the 5' terminal and SpeI at 3'  terminal. The sequence of promoter Pcar had cohesive terminus at both ends, so it was very convenient for us to construct the plasmid for functional  detection.The sequence of Pcar is as followed:</P><P align="center">The design concept of promoter <I>Pcar</I></P><P align="justify"><STRONG>Construction of plasmid for direct  regulation pathway</STRONG></P><P align="justify">In this experiment, RFP reported the  function of the indirect regulation pathway.</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861179">BBa_K861179</A>: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I13507">BBa_I13507</A>, an mRFP generator with RBS and terminator was embedded downstream the constitutive promoter BBa_J23119</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861176">BBa_K861176</A>: BBa_I13507 was embedded downstream the artificial promoter Pcar.</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861178">BBa_K861178</A>: a constitutive expressed CRP(<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23116">BBa_J23116</A>+<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861161">BBa_K861161</A>) was assembled with K861176.</P><P align="justify">All new composite parts mentioned above  were transformed to competent cells of <I>Escherichia coli</I> str. DH5a. All positive  clones were validated using PCR, restriction enzyme digestion and DNA  sequencing.</P><P align="justify"><STRONG>Functional detection</STRONG></P><P align="justify">The same methods with that of the indirect regulation pathway were used  to confirm that the promoter worked as expected. For details, please click <A href="https://2012.igem.org/Team:WHU-China/Project?catalog=2#Indirect Pathway Design">Here</A>.</P><P align="justify"><STRONG>Results</STRONG></P><P align="justify"><STRONG>Construction  of the plasmid for functional detection</STRONG></P><P align="justify">The sizes of the Promoters Pcar and J23119  are less than 100 bp.   And they were proved to be  correct by  agarose gel electrophoresis. Restriction Digestion of the  plasmid BBa_I13507 only have one lad on the agarose gel, it told that the  plasmid was digested well. After transformation, competent cells were cultured on agar plate with 50 μg/L  of ampicillin. Both red and white bacterial colonies emerged on one plate. The red ones were  the correct clones revealing promoter embedded  successfully, while the white ones were negative. The red clones were picked out and cultured in LB medium for plasmid extraction.  Purified plasmids were digested with XbaI and PstI for confirmation. The bands  of 2000bp and 1000bp showed that the promoter had been embedded successfully.  At last, the plasmids we acquired were sent for sequencing, and the result showed no mutation existed. </P><P align="center">The agarose gel for digestion comfirmation</P><P align="justify"><STRONG>Cell culture fluorescence measurement. </STRONG></P><P align="justify">The  correct clones were cultured in 96-well plate at 37℃ for 24 hours, then the fluorescence  and absorbance at 600 nm were recorded on a SpectraMax M2 plate reader. All fluorescence results were normalized with absorbance at 600 nm. The results represented  the fluorescence of every well.
							  The fluorescence  of K861179 was about 10000 Relative Light Units. It did not vary with  concentrations of glucose. However we found a positive correlation between the  fluorescence of K861176 and concentration of glucose. At a glucose concentration  lower than 4mM, the fluorescence was very low, but at high concentration of  glucose like 100mM, the fluorescence was much less than that of K861179. </P><P align="center">The result of promoter Pcar from fluorescence measurement</P><P align="justify"><STRONG>Capturing of the fluorescence image </STRONG></P><P align="justify">Fluorescence images indicated that all the cells were growing normally, because the size and  morphology were both the same with cells in LB medium. The fluorescence of the  cells in the images showed the same discipline with results from the fluorescence  measurement experiments. Fluorescence of K861179 was very strong but it didn't  change with the glucose concentration. On the contrary, fluorescence of K861176  was relatively weak but increased with  concentration of glucose.</P><P align="center">Fluorescence images of Pcar and J23119 in different glucose concentration</P><P align="justify"><STRONG>Fluorescent analysis of cyto-imaging</STRONG></P><P align="justify">The results of FANCY are showed as bellow, single cell was recognized from fluorescence images and fluorescence intensity was calculated. In the table, data showed that RFP expression was activated in high glucose concentration, which conformed well with results above.For more information about FANCY,click <A href="https://2012.igem.org/Team:WHU-China/Modeling?catalog=3">Here</A>.</P><P align="center">Fluorescent intensity of Pcar from program FANCY</P><P align="justify"><STRONG> </STRONG></P><P align="justify"><STRONG>Discussion</STRONG></P><P align="justify">The promoter  Pcar is a promoter designed for the <EM>Escherichia  coli</EM> and it is derived from a constitutive promoter BBa_J23119. Pcar includes the CRP-binding site and the RNA polymerase-binding site which overlap each other several base pairs. Therefore, because of the steric hindrance between CRP and RNA polymerase, gene downstream of the promoter will be repressed at high concentration of CRP. In the cells, low glucose concentration results in increasing activity by adenylate cyclase. cAMP binds to the cAMP receptor protein, which, in its bound form, is able to bind tightly to the specific DNA site in the promoter and to repress the  gene downstream. On the contrary, high  glucose concentration will result in the activation of the promoter.</P><P align="justify"> </P><A name="PfadR"><H3><I>PfadR</I></H3></A><H4>Outline</H4><P align="justify"> The design of 
<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861071"> Pcar</A> had sparkled the design of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861060">PfadR</A>. In nature, beta oxidation needs these five genes, and is regulated by many factors besides fatty acids concentration. Therefore, there is no promoter that can meet our demand to be solely regulated by fatty acids. Therefore, we tried to develop a synthetic promoter that would be regulated solely by fatty acids. We found that the last 3 base pairs of constitutive promoter BBa_J23110 are the same as the first three base pairs of FadR binding site of FadL. Therefore, we thought that maybe by preventing the initiation of transcription, we can achieve our goal. </P><H4>Design of the Promoter PfadR Repressed by Fatty Acids</H4><P align="justify"> Promoter PfadR, is derived from BBa_J23110. Specifically, FadR binding site of FadL gene was placed overlapping with the last 3 base pairs of BBa_J23110. The sequence was synthesized with restriction sites for EcoRI and XbaI at the 5' terminal and SpeI at 3' terminal. We used overlapping PCR to get the double strand DNA. The sequence design of PfadR is as followed: </P><P align="justify"> Forward: GGAATTCTCTAGATTTACGGCTAGCTCAGTCCTAGGTACAATGCTAGCTGGTCCGACCT</P><P align="justify"> Reverse： GACTAGTTCTTAGAAATCAGACCAGTGGCGAGAGTATAGGTCGGACCAGCTAGCATTGT</P><H4>Procedures</H4><P align="justify"> For M9 medium using oleic acids as sole carbon source, oleic acids was first emulsified 10% Triton X-100. M9 medium was then slowly added with constant vortex. M9 medium with high concentration of oleic acids was diluted by M9 medium with triton to form various concentrations. </P><P align="justify">For M9 medium using glucose as sole carbon source, M9 medium with high concentration of glucose was diluted by M9 medium to form various concentrations. </P><P align="justify"> After 24h of incubation in 24 well plates in 37°C, bacteria culture was centrifuged at 3000rmp for 5min, washed and resuspended in PBS. We detected the OD600 and fluorescence of using SpectraMax M2 plate reader (Molecular Devices) .Excitation at 584 nm and emission at 607 nm were used. All fluorescence was normalized with cell density by measuring the absorbance at 600 nm. </P><H4>Result</H4><P align="justify"> Normalized using Fluorescence/0D600</P><P align="justify"> Blue: Constitutive promoter J23110</P><P align="justify"> Red: PfadR</P><P align="justify"> Glucose Concentration gradient: 0.5, 1, 5, 10 mM</P><P align="justify"> Fatty acid Concentration gradient: 0.025, 0.05, 0.1, 0.25, 0.5, 1, 1.5 mM</P><P><I> PfadR and BBa_J23110 promoter strength at different glucose and fatty acids concentration</I></P><P align="justify"> As shown from the results, the promoter showed about 3 times induction from glucose to 1.5umol/L fatty acids medium and the fluorescence of PfadR was about one sixth of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23110">BBa_J23110</A>. This may because the tandem FadR binding site had made it more difficult for Polymerase to start gene transcription. Also, in medium that used glucose as sole carbon source, PfadR seemed to be leaky. However, since our bacteria was wild type <I>E.coli</I>, Fab genes was not mutated, meaning that bacteria can synthesis fatty acids. Therefore, there may be a basal level fatty acids concentration inside the cell, making the transcription not being totally repressed. </P><P align="justify"> It should also be noticed that, from fatty acids concentration 0.025umol/L to 1.5umol/L, the induction was not very obvious. For fatty acids concentration F0.25, 0.5 and 1, it seemed to have similar fluorescence strength. The promoter was not sensitive enough. To further improve the function of PfadR, we are planning to modify the sequence of FadR binding sites to make FadR overlap more willing promoter region. Also, we will try to overexpressed FadR protein to see its effects on PfadR.</P><P align="justify"> </P><A name="Standard"><H3>Standard</H3></A><P align="justify">To address the problem of sensing substrates in our metabolic devices, we have developed a few promoters following the strategy of modifying unconserved regions to protein binding sites. As steric hindrance can stop RNA polymerase as well as other transcription factors from correctly binding and functioning, the binding protein become the repressor of our synthetic promoters.</P><P align="justify">For different promoters, the details of modification come in different ways. For instance, the PfadR (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J861060">BBa_J861060</A>) was modified downstream -10 region, while Pcar (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J861171">BBa_J861171</A>) was modified between -35 and -10 region. Indeed we have tried some other modifications (for more information, click on design of promoters) but without desirable functions and we still do not know why. However, all modifications aimed at the same goal, to introduce overlapping sequences between binding site of the designed repressor and the promoter.</P><P align="justify">To test the function of these promoters, we used fluorescence measurements. First, both the modified promoter and the unmodified one (as control) were assembled with mRFP gene as a reporter. Hence, after culturing and testing fluorescence intensities using different methods (see <A href="https://2012.igem.org/Team:WHU-China/Project?catalog=2#Indirect Pathway Design">indirect regulation</A> and <A href="https://2012.igem.org/Team:WHU-China/Modeling?catalog=3">FANCY</A>), we can learn if the synthetic promoter is regulated as expected.</P><P align="justify">After a few successes, we are endeavoring setting up a new technical standard on design, construction and functional test of promoters. </P><P align="justify"> </P></DIV><DIV class="passage divcell3"><P>The above is the video introduction of fatty acid degradation device. For Chinese mainland visitor, please visit <A href="http://v.youku.com/v_show/id_XNDY2MTM3Mjc2.html">here</A> for the video</P><A name="Purpose"><H3>Purpose</H3></A><P align="justify">
             To help people lose weight without the need of food restriction, we designed a genetically modified 

<I>E.coli</I> that can sense and degrade excessive fatty acid intake by the host. We hope that, together with other two 

devices we designed, we can introduce our <I>E.coslim</I> as a resident in intestine to consume the excessive calories intake 

by the host and regulate intestinal microbiota.
      </P><A name="Outline"><H3>Outline</H3></A><P align="justify">

Genes that are responsible for degradation and transportation of fatty acids (FAs) from <I>E.coli K12</I> and from 

<I>Salmonella enterica LT2</I> were cloned. Also, a promoter named PfadR that can be regulated solely by fatty acids was also 

designed. By placing those fatty acids degradation genes downstream of the artificially designed promoter PfadR (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861060">BBa_K861060</A>), we hope to create a device that degrades FAs only when the concentration of FAs is high.
 </P><P align="justify">
Long chain fatty acids are firstly imported by the transmembrane protein <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861500">FadL</A>. After FAs get into cells, a CoA will 

be added by inner membrane-associated <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861010">FadD</A> (acyl-CoA synthase). β-oxidation is initiated by <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861020">FadE</A>(acyl-CoA dehydrogenase), 

which will convert acyl-CoA into enoyl-CoA. The following cycles of hydration, oxidation, and thiolytic cleavage are carried 

out by tetrameric complex consisting of two <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861400">FadA</A> and two <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861030">FadB</A> proteins or two <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861041">FadI</A> and two <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861031">FadJ</A> in anaerobic condition. 

<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861050">FadR</A> is a transcriptional regulator that, when not binds to acyl-CoA, can either serve as an activator for fatty acid 

synthesis genes like FabA, FabB and etc., or a repressor for fatty acid degradation genes like FadA, FadB, FadD FadE, FadL, 

FadI, FadJ and etc. After long chain fatty acids are converted to fatty acyl- CoA by FadD, it can bind to FadR. The binding 

will alter the conformation of FadR, making FadR unable to bind to the DNA sequence it recognizes to fulfill its function. 

Therefore, FadR can no longer activate or repress the transcription of genes downstream FadR binding sites. However, to our 

knowledge, there is no promoter available in nature that can respond solely to FadR since those promoters are often also regulated 

by glucose concentration or oxidative stress and many other factors.
In our design, FadL, FadD, FadE, FadA, FadB, FadI and FadJ from <I>Escherichia coli K12</I>, and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861042">FadA</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861032">FadB</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861021">FadE</A> 

from <I>Salmonella enterica LT2</I> were placed downstream a synthetic promoter PfadR to make them solely regulated by fatty acids concentration.</P><A name="Progress"><H3>Progress</H3></A><H4>
Cloning of the gene
</H4><P align="justify">
First, the genome of <I>Escherichia coli K12 str. DH5ɑ</I> and <I>Salmonella enterica LT2</I> (symbolized as S-) were extracted 

and amplified by PCR using primers for each gene. The sequences of the primers used are as bellow (5’---3’).<EM>fadR</EM>  Forward: GGAATTCTCTAGAATGGTCATTAAGGCGCAAAG
      Reverse: GACTAGTCTTATCGCCCCTGAATGGCTAAATC<EM>fadA</EM>  Forward: GGAATTCTCTAGAATGGAACAGGTTGTCATTGTCG
      Reverse: GACTAGTTTAAACCCGCTCAAACACCGT<EM>fadB</EM>  Forward: GGAATTCTCTAGAATGCTTTACAAAGGCGACACC
      Reverse: GACTAGTTTAAGCCGTTTTCAGGTCGCC<EM>fadD</EM>  Forward: GGAATTC TCTAGATTGAAGAAGGTTTGGCTTAACCG
      Reverse: GACTAGTTCAGGCTTTATTGTCCACTTTGC<EM>fadE</EM>  Forward: GGAATTC TCTAGAATGATGATTTTGAGTATTCTCG
      Reverse: GACTAGTTTACGCGGCTTCAACTTTCCG<EM>fadL</EM>  Forward: GGAATTC TCTAGAATGAGCCAGAAAACCCTG
	Reverse: GACTAGTTAGAACGCGTAGTTAAAGTTAG<EM>fadI</EM>  Forward: GGAATTC TCTAGA ATGGGTCAGGTTTTACC
    Reverse: GACTAGTTTATTCCGCCTCCAGAACCA<EM>fadJ</EM>  Forward: GGAATTCTCTAGAATGGAAATGACATCAGC
    Reverse: GACTAGTTTATTGCAGGTCAGTTGCAGTTG<EM>s-fadA</EM>  Forward: GGAATTCTCTAGAATGGTCATTAAGGCGCAAAG
      Reverse: GACTAGTCTTATCGCCCCTGAATGGCTAAATC<EM>s-fadB</EM>  Forward: GGAATTCTCTAGAATGCTTTATAAAGGCGACACC
        Reverse: GACTAGTTAAGCCGTTTTCAGAGAACC<EM>s-fadE</EM>  Forward: GGAATTCTCTAGAATGATGATTTTGAGTATTATCG
Reverse: GACTAGTTATGCGGCTTCGACTTTACGC</P><H4>
Design of the Promoter PfadR Repressed by Fatty Acids 
</H4><P align="justify">
Promoter PfadR, was derived from <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23110">BBa_J23110</A>. Specifically, FadR binding site of <EM>fadL</EM> gene was placed overlapping with the last 

3 base pairs of BBa_J23110. The sequence was synthesized with restriction sites for EcoRI and XbaI at the 5' terminal and SpeI at 

3' terminal. We used overlapping PCR to get the double strand DNA. The primer design of PfadR is as followed:
Forward:
GGAATTCTCTAGATTTACGGCTAGCTCAGTCCTAGGTACAATGCTAGCTGGTCCGACCT
Reverse：
GACTAGTTCTTAGAAATCAGACCAGTGGCGAGAGTATAGGTCGGACCAGCTAGCATTGT</P><H4>
Construction of Biobricks
</H4><P align="justify">
Fatty acid degradation project is divided into two parts: promoter, and gene function.

To discover the optimal combination of those fatty acid genes :

1. PCR was used to clone those genes in <I>E.coli K12</I> and <I>Salmonella enterica LT2</I>,
2. those genes were restriction digested and ligated into pSB1C3,
3. those genes were restriction digested and ligated with RBS(<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0030">B0030</A>),
4. RBS-FadA, RBS-FadI, and RBS-S-FadA was ligated with both <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_R0011">BBa_R0011</A> promoter and our PfadR
   RBS-FadR, RBS-FadB, RBS-FadJ, RBS-FadE, RBS-FadD, RBS-FadL, RBS-S-FadB, and RBS-S-FadE were ligated with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0034">B0034</A>,
5. Promoter-RBS-FadA was ligated with RBS-FadB-Terminator, Promoter-RBS-FadI was ligated with RBS-FadJ-Terminator and 

Promoter-RBS-S-FadA was ligated with RBS-S-FadB-Terminator. RBS-FadE-Terminator, RBS-FadD-Terminator, RBS-FadL-Terminator, 

and RBS-S-FadE-Terminator, were ligated with BBa_R0011 promoter, PfadR and various constitutive promoters.


For Promoter PfadR
(1) promoter PfadR was synthesized using overlapping PCR
(2) RFP reporter was ligated downstream the promoter and ligted into pSB6A1
(3) J23116+ RBS+ FadR+ Terminator was ligated to PfadR+ RFP in pSB6A1</P><A name="Experimental Procedure"><H3>Experimental Procedure</H3></A><P align="center"><I>Cupric acetate-pyridine reaction</I></P><P>
We used cupric acetate-pyridine as a color developing reagent to determine fatty acids consumption of genetically modified bacteria. We have modified existing methods to extract free fatty acid in M9 medium. Also, we used IPTG induced promoter BBa_R0011 to see the expression of those proteins and extract proteins from cells. For more details, please see protocol: <A href="https://2012.igem.org/Team:WHU-China/Notebooks?catalog=3#Protocols of Cupric-Soap Reaction">Cupric-Soap Reaction</A> for more details.
</P><P align="justify">We also conducted <EM>in vitro</EM> experienments in which we characterized fatty acid degradation capabilities of combination of enzymes using a cell free system. Please see protocol: <A href="https://2012.igem.org/Team:WHU-China/Notebooks?catalog=3#In vitro experiment"><I>In vitro</I> Experiment</A> for more details.
</P><H3>Results</H3><H4>
Characterization of each gene
</H4><P align="justify">
In this experiment, we wanted to test whether the ability of degrading fatty acids of our genetically modified bacteria was enhanced as expected by transforming plasmids constitutively expressing related genes in the β-oxidation pathway. The effects of the genes we tested is listed in the following chart I. The ability was reflected by the change of the concentration of the fatty acids in the medium. It was measured by cupric-acetate soap reaction as described in Protocols section. Each time we inoculated 50mg bacteria into 30ml M9 medium using fatty acids as sole carbon source, collecting the sample at the time as shown in the picture. Then the analysis of the free fatty acids was performed.
</P><P align="justify">
The following figures showed the effects on degrading fatty acids by expressing different genes in β-oxidation pathway in <EM>E.coli</EM>. They were under the regulation of promoters with different kinds of strength. J23107 and J23114 are constitutive promoters provided by the committee. PfadR is the promoter designed by ourselves. It consists of the sequence of a constitutive promoter and the binding sequence of the transcription factor, FadR, which is the sensor of the fatty acids. FadE is the acyl-CoA dehydrogenase, which have been proved as performing the rate limiting reaction in the pathway. S-FadE is the counterpart of FadE in the bacteria Samonella. FadD is the acyl-CoA synthase. FadL, a transmembrane protein, is responsible for transporting fatty acids into the bacteria. The control we used was the E.coli expressing galU, a gene responsible for synthesize cellulose.</P><P><I>Fatty acid degradation at 6h</I></P><P><I>Fatty acid degradation at 12h</I></P><P><I>Fatty acid degradation at 18h</I></P><P><I>Fatty acid degradation at 24h</I></P><P><I>Fatty acid degradation of bacteria overexpressing each gene in 24h</I></P><P align="justify">Based on the measurements of the consumption at given time, we concluded that overexpressing FadL increased the metabolizing ability no matter under the regulation of J23114 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861002">BBa_K861002</A>) or our designed promoter, PfadR(<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861003">BBa_K861003</A>). The advantage was more obvious when the time expanded (consumption at 18h and 24h). It's plausible because more fadL may transport more fatty acids into the bacteria. The increased inner fatty acids concentration was quite favorable. We also noticed that the later one's consumption was lower. It may attribute to the fact that our promoter was weaker than the J23114. If the copy number of FadL was less, its metabolizing rate would be slower. And the fact that the PfadR needs to be induced may also make the time needed to synthesize protein longer, which may make it less competitive. These data opposed to our assumption that overexpressing the rate limiting enzyme FadE ((<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861025">BBa_K861025</A> and (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861026">BBa_K861026</A>) wouldn't have obvious effect. It may because the original level of FadE was enough, thus overexpression was not needed. The strength of the promoter didn't affects the rate much, which partially suggested the reason above.</P><P align="justify">We found that the slope between 12h and 18h and between 18 and 24h were less than the others. It may because the bacteria have entered static status, the amount of bacteria became consistent. Also, after the first death phase, they entered logarithmic phase again. Since our inoculation was relatively large and the oleate was excessive, the situation that the bacteria has experience two life cycle was possible. A growth curve in the future can test the theory.
</P><H4><EM>In vitro</EM> Experiment</H4><P align="justify">The <EM>in vitro</EM> experiment was designed to make up for the limitation of time to test the combination of expressing different genes together (We were short of time assembling the genes together). So we used the cell extracts to do the enzyme assay. The advantage was that we can easily mix the enzyme we want together. Since the purpose of this experiment was to test whether overexpressing multiple genes were superior to a single gene, we set the amount of every gene the same in the combination for simplicity. In the future, we can test more combinations to find the best ratio.</P><P><I>Fatty acids remaining after 6 hours of reaction</I></P><P align="justify">The result was that the cell extract of bacteria overexpressing FadE (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861024">BBa_K861024</A>), FadD (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861013">BBa_K861013</A>), S- FadA S- FadB(<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861038">BBa_K861038</A>) (a regulon), FadI FadJ(<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861037">BBa_K861037</A>) (a regulon) separately all degraded oleate obviously faster than the control. This may be somewhat contrasting to the result of our <EM>in vivo</EM> assay, in which overexpressing the FadD and FadE did not have obvious results. However, it can be explained. The promoter was not that strong in the <EM>in vivo</EM> assay, otherwise the growth of the bacteria would be inhibited. However, in the <EM>in vitro</EM> assay, this consideration was not necessary. Also, the concentration in the final reaction system was quite high by collecting 1L medium, which may improve the metabolizing rate. </P><P align="justify">It also showed that simultaneously increasing the concentration of FadE, FadD, S- FadA and S- FadB significantly improved the degrading ability. Increasing the doze of the FadI and FadJ on the basis of above did not make any difference, while increasing FadA and FadB may be indispensible because only expressing FadD and FadE was worse than expressing the gene alone. </P></DIV><DIV class="passage divcell4"><P>The above is the video introduction of cellulose synthesis degradation device. For Chinese mainland visitor, please visit <A href="http://v.youku.com/v_show/id_XNDY2MTQwNTI0.html">here</A> for the video.</P><A name="Outline1"><H3>Outline</H3></A><P align="justify">
Cellulose is an essential material for keeping intestine peristalsis without producing energy, as prebiotics, feeding vegetarian bacteria flora (including Bacteroides, whose appropriate amount has proved important to prevent obesity) of intestine as well. Thus, cellulose helps people keep slim and healthy.</P><P align="justify">
The developing device aims at transforming glucose into cellulose, thus producing cellulose as well as reducing energy intake. To achieve this goal, we cloned genes of enzymes responding to cellulose synthesis from the <I>Escherichia coli</I> str. DH5α, constructing functional expressional elements with these genes respectively downstream of promoter activated by glucose. In this way, cellulose synthetase complex was built artificially under regulation of glucose, repressed under low concentration of glucose and activated under high concentration of glucose.</P><P align="justify">
In the future, this device can be integrated to the assembled <EM>E. coslim</EM>, activated when excess glucose is sensed in intestine, converting it to cellulose.</P><P align="justify">
The same as device I (fatty acid degradation), on one hand, we divide our work into two parallel sections.<EM> Function</EM> section includes a series of molecular biological manipulation on four genes of the cellulose synthetase complex and another two genes responding to produce substrates for cellulose synthesis. On the other hand, the design, construction and function tests of glucose-activated promoter belong to <EM>regulation</EM> section.</P><A name="Design1"><H3>
Design</H3></A><H5 align="justify">Genes to be Cloned</H5><P align="justify">
4 genes, <EM>bcsA</EM>, <EM>bcsB</EM>, <EM>bcsZ</EM> and <EM>bcsC</EM>, from the rdar morphotype bacterium, are involved in cellulose biosynthesis.
</P><P align="justify">	BcsA is considered to be the catalytic subunit.
</P><P align="justify">BcsB can be activated the soon it binds to c-di-GMP.
</P><P align="justify">BcsZ encodes endo-1,4-D-glucanase which belongs to glycosyl hydrolase family Ⅷ. Activation of BcsZ is required for optimal synthesis and membrane translocation of cellulose.
</P><P align="justify">Although <EM>BcsC</EM> is transcribed constitutively, cellulose synthesis occurs only in the circumstances of  AdrA.
</P><P align="justify">AdrA ,a diguanylate cyclase (DGC), cyclizestwo GTPs into c-di-GMP. In turn, the activity of cellulose synthase can be increased when binds to c-di-GMP.
</P><P align="justify">GalU catalyzes the addition of UTP to α-D-glucose 1-phosphate to yield UDP-D-glucose, which is the substrate for cellulose synthase complex
</P><P align="justify">GalF is a predicted subunit of a GalU/GalF protein complex involved in colanic acid building blocks biosynthesis
</P><H5 align="justify">Plasmid construct concept</H5><P align="justify">
After the indirect regulatory pathway and promoter Pcar being tested to be effective,we can embed the genes which are involved in cellulose synthesis downstream any one of them.So cellulase can only be expressed when glucose concentration is high, and also the expression will increase with glucose concentration. In this way, our E.coslim will have the ability to convert excess glucose into cellulose.
</P><A name="Progress1"><H3>Progress</H3></A><H5 align="justify">Clone of genes</H5><P align="justify">
As for the genes that we cloned, there was no difference between <I>E. Coli</I> str. K12 MG1655 and more available DH5α. we purified and amplified these genes from genome of <I>Escherichia coli</I> str. DH5α using PCR. The primers contained standard restriction enzyme cutting sites. The sequences of the primers used are as below.
</P><P align="justify"><EM>bcsA</EM>  Antisense CCTGCAGTACTAGTATCATTGTTGAGCCAAAGCCTG 
  　　Sense      CGAATTCTTCTAGAGATGAGTATCCTGACCCGGTGG
</P><P align="justify"><EM>bcsB</EM>  Antisense  CCTGCAGTACTAGTATTACTCGTTATCCGGGTTAAGAC 
  　　Sense       CGAATTCTTCTAGAGATGAAAAGAAAACTATTCTGGATTTG
</P><P align="justify"><EM>bcsZ</EM>  Antisense  CCTGCAGTACTAGTATTAGTGTGAATTTGCGCATTCCTGG 
  　　Sense       CGAATTCTTCTAGAGATGAATGTGTTGCGTAGTGGAATCG
</P><P align="justify"><EM>bcsC</EM>  Antisense  CCTGCAGTACTAGTATTACCAGTCGGCGTAAGGTATCA 
  　　Sense       CGAATTCTTCTAGAGATGCGCAAATTCACACTAAACATATTC
</P><P align="justify"><EM>galF</EM> Antisense  CCTGCAGTACTAGTATTATTCGCTTAACAGCTTCTCG 
  　　Sense      CGAATTCTTCTAGAGATGACGAATTTAAAAGCAGTTATACC
</P><P align="justify"><EM>galU</EM> Antisense  CCTGCAGTACTAGTATTACTTCTTAATGCCCATCTCTTCT 
  　　Sense       CGAATTCTTCTAGAGATGGCTGCCATTAATACGAAAG

</P><P> </P><P align="justify">Then the genes were digested with restriction enzymes and assembled to RBS (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0030">BBa_B0030</A>) and terminator (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0024">BBa_B0024</A>).
  </P><H5 align="justify">Construction of the plasmid expressing cellulose synthetase controlled by promoter we designed</H5><P align="justify">
All coding sequences were assembled to RBS and terminator, afterwards, they were embedded downstream the promoter Pcar, which can be activated at high glucose concentration.
The biobricks constructed were showed as bellow:
</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861102">BBa_K861102</A>: Pcar+RBS+<I>bcsA</I>+terminator
</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861112">BBa_K861112</A>: Pcar+RBS+<I>bcsB</I>+terminator
</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861122">BBa_K861122</A>: Pcar+RBS+<I>bcsZ</I>+terminator
</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861132">BBa_K861132</A>: Pcar+RBS+<I>bcsC</I>+terminator
</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861142">BBa_K861142</A>: Pcar+RBS+<I>galU</I>+terminator
</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861152">BBa_K861152</A>: Pcar+RBS+<I>galF</I>+terminator
</P><P align="justify"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861074">BBa_K861074</A>: Pcar+RBS+<I>adrA</I>+terminator
</P><P align="justify">
All new composite parts mentioned above were transformed to competent cells of <I>Escherichia coli</I> str. DH5α. All positive clones were validated using PCR, restriction enzyme digestion and DNA sequencing.</P><H5 align="justify">Detection of Cellulose Synthesis</H5><P align="justify">
To detect the cellulose synthesis, we used cellulase to degrade cellulose in the cell culture. Then total reducing sugar in the culture was measured. So the difference of total reducing sugar between culture before and after treated with cellulase represented the total cellulose synthesized by the cell. For detailed information, please click <A href="https://2012.igem.org/Team:WHU-China/Notebooks?catalog=3#Cellulose Detection">Here</A>.</P><A name="Results1"><H3>Results</H3></A><H5 align="justify">
Clone of genes</H5><P align="justify">
The gene <I>bcsA</I> is 2619bp, <I>bcsB</I> is 2340bp, <I>bcsZ</I> is 1107 bp, <I>bcsC</I> is 3474bp, <I>galU</I> is 909bp and <I>galF</I> is 894 bp. After PCR amplification, DNA fragments were examined by agarose gel electrophoresis. All genes proved to be correct. Then the genes were digested with restriction enzymes and embedded into plasmid backbone pSB1A2. To confirm the accuracy of sequences, positive clones were sent for sequencing after transformation. And the results showed that no mutation existed in genes.</P><H5 align="justify">Detection of Cellulose Synthesis</H5><P align="justify">After treating with cellulase, total reducing sugar in supernatant and deposits were measured by methods described in our protocol. Colors in the tubes 
becoming darker  meant that reducing sugar increased with time. Amount of reducing sugar was calculated according to standard curve of glucose.</P><P align="justify">The formula of standard curve is as bellow:</P><P align="justify"><I>y</I>=1.3795<I>x</I>+0.0373</P><P align="justify">In our experiment, cells that expressed protein which has nothing to do with cellulose synthesis was used as a control.We transformed the seven genes involved in cellulose synthesis into <EM>E.coli</EM> and measured cellulose production with method mentioned above. In the step of measuring total cellulose production, excessive cellulase was appended and incubated at 50 ℃ for 1 hour, and results showed that some of the seven genes help increase the ability of cellulose synthesis.</P><P align="justify">In addition,AdrA is responsible for the production of a small molecule,c-di-GMP, which is known as an activator of cellulase. So we co-overexpress adrA with bcsA. Cellulose output in both supernatant and deposit were showed in the following figure. The cellulose production in <EM>adrA</EM>+<EM>bcsA</EM> was more than 0.6 mg/mL, which was almost two times higher than that of control.</P><P align="justify"> </P><A name="Discussion1"><H3><STRONG>Disscusion</STRONG></H3></A><P align="justify">In our experiment, results showed that cellulose yield in <EM>adrA</EM>+<EM>bcsA</EM> was higher than that of the single gene. So it indicated that the co-expression of some genes would increase cellulose production. In the next stage, we will try more combination of these genes and find the most efficient one</P><P align="justify">Actually, BcsB can be activated by c-di-GMP. But till  the deadline, we have not successfully assembled Pcar and <EM>bcsB</EM>. In the following work, we are going to assemble a plasmid including all of the seven genes. Maybe in this way, cellulose production will increase greatly.</P><P align="justify">Other than application in the project of <EM>E.coslim</EM>, the cellulose device can also be used in many other fields, such as papermaking industry, biomedical materials, audio equipment and so on.</P></DIV><DIV class="passage divcell5"><P>The above is the video introduction of colonization device. For Chinese mainland visitor, please visit <A href="http://v.youku.com/v_show/id_XNDY2MTQwNDc2.html">here</A> for the video</P><A name="Purpose2"><H3>Purpose</H3></A><P align="justify">
             One big challenge of probiotics is their survival in intestine. We respond to this challenge by expressing gene <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861070">adrA</A> responsible for manufacturing the second messager c-di-GMP, a magic molecule that leads to inhibition of motility and increase of adhesion and division of <I>E.coli</I>.           </P><A name="Outline2"><H3>
            Outline
</H3></A>
AdrA protein can convert GTP into c-di-GMP, a magic second massager that, besides promoting the production of cellulose(for more details, see our celluose synthesis device), can reduce the expression of flagella and acute virulence gene simultaneously. In the same time, c-di-GMP can facilitate the synthesis of various adhesins and exopolysaccharides and can promote the proteolysis of replication inhibitors. As a result, AdrA makes cells become adhesive and promote the formation of biofilm, making the bacteria gain advantages to survive in the hostile environment of intestine.
<A name="Experimental Procedure2"><H3>
Experimental Procedure
</H3></A><P align="justify">
We tested the function of AdrA gene by plate assay. Formore details, please visit our protocol: <A href="https://2012.igem.org/Team:WHU-China/Notebooks?catalog=3#Plate Assay">Plate Assay</A>.
</P><A name="Results2"><H3>
Results
</H3></A><P align="justify">As can be seen from the plate. Clone with AdrA constitutively expressed using <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23107">BBa_J23107</A> was much more smaller compared to the control that only contain <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0030">the plasmids of RBS</A>, though the amount of bacteria was similar. This result showed that the AdrA was successfully expressed, elevating c-di-GMP level, leading to the increase expression of adhesins and inhibition of motility.</P></DIV><DIV class="passage divcell6"><P>The above is the video introduction of death device. For Chinese mainland visitor, please visit <A href="http://v.youku.com/v_show/id_XNDY2MTQwNDIw.html">here</A> for the video</P><A name="Outline4"><H3> Outline </H3></A><P align="justify">
We plan to place endonuclease and YhjH gene downstream the xylR repressed promoter<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861200"> (BBa_K861200)</A> to make bacteria die and lose their adhesion when exposed to xylose.
</P><A name="Description"><H3>Description</H3></A><P align="justify">
In order to terminate the overreaction of fatty acids degradation and cellulose synthesis, we designed a death device. Firstly, we determined to use endonuclease YhjH<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861090"> (BBa_K861090)</A> responsible for c-di-GMP degradation to counteract the impact of AdrA and BglII(<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K112106"> (BBa_K112106)</A>), a protein that can kill the microbe without inducing cell lysis. Since AdrA but not YhjH contains this restriction site, our E.coslim will lose their adhesion more easily. To control the expression of endonuclease and YhjH, a regulator is 

necessary. We selected xylose as the inducer because this monomer hardly exists in nature, so the GMOs will not be killed under natural condition. Also, it is healthy and hard to be absorbed by human beings so the concentration of xylose in the gut can keep high for a long time. We are planning to device another synthetic promoter repressed by XylR from <I>B.subtilis</I>. We hope that eating xylose will 

subsequently derepress the expression of endonuclease and YhjH, ending up with the loss of adhesion and the death of our E.coslim, therefore efficiently terminate the process of lossing weight. For more information and for a brief introduction of our two-capsule design, please visit our <A href="https://2012.igem.org/Team:WHU-China/Modeling?catalog=0">Future Perspective </A>.

</P><A name="Prevention of HGT"><H3> Prevention of HGT </H3></A><P align="justify">
In human gut, there are billions of microbes. Horizontal gene transfer of those high efficient metabolic genes between GMOs 

and normal intestine residents may get out of control and cause disastrous effects. Therefore, a mechanism to prevent HGT 

is highly necessary.
To prevent horizontal gene transfer, we designed a two-plasmids system. In one plasmid, xlyR repressor encoding gene<A href="http://partsregistry.org/wiki/index.php/Part:BBa_K143036"> (BBa_K143036)</A>  from 

<I>Bacillus subtilis</I> will be assembled with constitutively expressed promoter. On the other plasmid, metabolic gene will be coupled with xylR repressed death system. In 

our GMOs, since death system is suppressed by xylR in low xylose concentration, the bacteria will not die. However, when 

the HGT of those high efficient metabolic genes happens, the death system will not be suppressed, and the recipient will 

be killed.
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:WHU-China/Project">http://2012.igem.org/Team:WHU-China/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WHU-China/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WHU-China/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WHU-China/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WHU-China/Project&amp;oldid=298724" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>