<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WHU-China_Notebooks"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WHU-China/Notebooks">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WHU-China/Notebooks&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WHU-China/Notebooks&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WHU-China/Notebooks&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WHU-China/Notebooks" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WHU-China/Notebooks</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="center"><DIV class="middle"><DIV class="main"><DIV class="passage divcell0"><H3>Achievements</H3><H4>Three devices</H4><P align="justify">-&gt; 66 new biobricks parts submitted and 39 parts proved to be effective. 
Please visit <A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=WHU-China">here</A> for more information.</P><P align="justify">-&gt; <I>In vivo</I> and <I>in vitro</I> experiments on fatty acid degradation.
Please visit <A href="https://2012.igem.org/Team:WHU-China/Project?catalog=3">here</A> for more information.</P><P align="justify">-&gt; Cloning and testing the function of cellulose synthetase genes. 
Please visit <A href="https://2012.igem.org/Team:WHU-China/Project?catalog=4">here</A> for more information.</P><P align="justify">-&gt; Using AdrA to <A href="https://2012.igem.org/Team:WHU-China/Project?catalog=4">activate cellulose synthetase</A> and <A href="https://2012.igem.org/Team:WHU-China/Project?catalog=5">facilitate colonization</A>. 
</P><P align="justify">-&gt; Modeling on <A href="https://2012.igem.org/Team:WHU-China/Modeling?catalog=2">gut microbiota</A> and <A href="https://2012.igem.org/Team:WHU-China/Modeling?catalog=1">FadR regulation</A>.</P><H4><A href="https://2012.igem.org/Team:WHU-China/Project?catalog=2">Promoter design</A></H4><P align="justify">-&gt; Novel promoter design: Pcar (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861171">BBa_K861171</A>)</P><P align="justify">-&gt; Novel promoter design: PfadR(<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861060">BBa_K861060</A>)</P><P align="justify">-&gt; Setting up a <A href="https://2012.igem.org/Team:WHU-China/Project?catalog=2">standard</A> of promoter design</P><H4>Methods we improved</H4><P align="justify">-&gt; Expanding the cupric soap method to quantify fatty acid in medium. 
Please visit <A href="https://2012.igem.org/Team:WHU-China/Notebooks?catalog=2">here</A> for more information.</P><P align="justify">-&gt; A new method to measure the cellulose production. 
Please visit <A href="https://2012.igem.org/Team:WHU-China/Notebooks?catalog=2">here</A> for more information.</P><P align="justify">-&gt; An open software <A href="https://2012.igem.org/Team:WHU-China/Modeling?catalog=3">Fancy</A> to measure the fluorescence of single bacterium</P><H4>The biobricks we improved</H4><P align="justify">-&gt; PfadR (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861060">BBa_K861060</A>), an improved design of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J54222">BBa_J54222</A></P><P align="justify">-&gt; Improved function of two promoters <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K118011">BBa_K118011</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_R0051">BBa_R0051</A></P><H4>Human practice</H4><P align="justify">-&gt; Help HUST iGEM team to debug its project and modeling.</P><P align="justify">-&gt; <A href="https://2012.igem.org/Team:WHU-China/HumanPractice">Human practice</A> on survey of GMOs and publicizing iGEM to the public.</P></DIV><DIV class="passage divcell1"><H1>Favourite Parts</H1><TABLE border="5" cellpadding="0" style="border:2px solid #760044;"><TBODY><TR><TH style="background:#f680c3;"> </TH><TH style="background:#f680c4;"> </TH><TH style="background:#f680c4;">Name</TH><TH style="background:#f680c4;">Type</TH><TH style="background:#f680c4;">Description</TH><TH style="background:#f680c4;">Designer</TH><TH style="background:#f680c4;">Length</TH></TR><TR><TD>1</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:
BBa_K861171">BBa_K861171</A></TD><TD>Regulatory</TD><TD>Pcar, synthetic promoter repressed by CRP</TD><TD width="2px">Kuanwei Sheng</TD><TD align="right">36</TD></TR><TR><TD>2</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861070">BBa_K861070</A></TD><TD>Coding</TD><TD>AdrA gene from E.coli K12</TD><TD width="2px">Kuanwei Sheng</TD><TD align="right">1167</TD></TR><TR><TD>3</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861060">BBa_K861060</A></TD><TD>Regulatory</TD><TD>PfadR, synthetic promoter with tandem FadR binding site</TD><TD width="2px">Kuanwei Sheng</TD><TD align="right">76</TD></TR></TBODY></TABLE><H1>Part Related to Promoter Design</H1><TABLE border="5" cellpadding="0" style="border:2px solid #760044;"><TBODY><TR><TH style="background:#f680c3;"> </TH><TH style="background:#f680c4;"> </TH><TH style="background:#f680c4;">Name</TH><TH style="background:#f680c4;">Type</TH><TH style="background:#f680c4;">Description</TH><TH style="background:#f680c4;">Designer</TH><TH style="background:#f680c4;">Length</TH></TR><TR><TD>1</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861160">BBa_K861160</A></TD><TD>Coding</TD><TD>CRP, cAMP receptor protein</TD><TD width="2px">Kuanwei Sheng</TD><TD align="right">633</TD></TR><TR><TD>2</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861161">BBa_K861161</A></TD><TD>Generator</TD><TD>CRP with a RBS and a terminator</TD><TD width="2px">Kuanwei Sheng</TD><TD align="right">757</TD></TR><TR><TD>3</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861162">BBa_K861162</A></TD><TD>Generator</TD><TD>Constitutively expresses Crp</TD><TD width="2px">Xian Xia, Kuanwei Sheng</TD><TD align="right">800</TD></TR><TR><TD>4</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861163">BBa_K861163</A></TD><TD>Generator</TD><TD>Constitutively expresses Crp</TD><TD width="2px">Xian Xia</TD><TD align="right">800</TD></TR><TR><TD>5</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861169">BBa_K861169</A></TD><TD>Regulatory</TD><TD>Indirect regulatory device, activated at high glucose concentration</TD><TD width="2px">Kuanwei Sheng</TD><TD align="right">1961</TD></TR><TR><TD>6</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861170">BBa_K861170</A></TD><TD>Regulatory</TD><TD>PI ,Glucose-activated promoter</TD><TD width="2px">Kuanwei Sheng</TD><TD align="right">36</TD></TR><TR><TD>7</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861172">BBa_K861172</A></TD><TD>Generator</TD><TD> lambda cI generator controlled by PcstA (glucose-repressible promoter)</TD><TD width="2px">Xian Xia</TD><TD align="right">1069</TD></TR><TR><TD>8</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861173">BBa_K861173</A></TD><TD>Reporter</TD><TD>mRFP generator controlled by  PcstA (glucose-repressible promoter)</TD><TD width="2px">Kuanwei Sheng, Xian Xia</TD><TD align="right">500</TD></TR><TR><TD>9</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861174">BBa_K861174</A></TD><TD>Regulatory</TD><TD>A control device for BBa_K861169</TD><TD width="2px">Xian Xia</TD><TD align="right">1899</TD></TR><TR><TD>10</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861175">BBa_K861175</A></TD><TD>Reporter</TD><TD>Efficacy testing RFP generator of BBa_K861170 (PI)</TD><TD width="2px">Kuanwei Shneg, Xian Xia</TD><TD align="right">905</TD></TR><TR><TD>11</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861176">BBa_K861176</A></TD><TD>Reporter</TD><TD>mRFP expressed after Pcar</TD><TD width="2px">Kuanwei Sheng, Xian Xia</TD><TD align="right">905</TD></TR><TR><TD>12</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861177">BBa_K861177</A></TD><TD>Generator</TD><TD>K861162+ K861175 </TD><TD width="2px">Kuanwei Sheng</TD><TD align="right">1713</TD></TR><TR><TD>13</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861178">BBa_K861178</A></TD><TD>Device</TD><TD>Pcar efficacy testing device with CRP overexpressed</TD><TD width="2px">Kuanwei Sheng, Xian Xia</TD><TD align="right">1713</TD></TR><TR><TD>14</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861179">BBa_K861179</A></TD><TD>Generator</TD><TD>K861162+J23119</TD><TD width="2px">Chang Liu</TD><TD align="right">843</TD></TR><TR><TD>15</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861061">BBa_K861061</A></TD><TD>Reporter</TD><TD>Efficacy testing RFP generator of BBa_K861060 (PfadR)</TD><TD width="100px">Kuanwei Sheng</TD><TD align="right">945</TD></TR><TR><TD>16</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861062">BBa_K861062</A></TD><TD>Regulatory</TD><TD>PfadR with FadR overexpressed</TD><TD width="100px">Kuanwei Sheng</TD><TD align="right">1840</TD></TR></TBODY></TABLE><H1>Parts Related to Fatty Acid Degradtion</H1><TABLE border="0" cellpadding="5" style="border:2px solid #760044;"><TBODY><TR><TH style="background:#f680c3;"> </TH><TH style="background:#f680c4;"> </TH><TH style="background:#f680c4;">Name</TH><TH style="background:#f680c4;">Type</TH><TH style="background:#f680c4;">Description</TH><TH style="background:#f680c4;">Designer</TH><TH style="background:#f680c4;">Length</TH></TR><TR><TD>1</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861500">BBa_K861500</A></TD><TD>Coding</TD><TD>FadL, Long-chain fatty acids transporter from E.coli str. K12</TD><TD>Kuanwei Sheng</TD><TD>1465</TD></TR><TR><TD>2</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861001">BBa_K861001</A></TD><TD>Generator</TD><TD>FadL with a strong RBS and a double terminator</TD><TD>Kuanwei Sheng</TD><TD>1465</TD></TR><TR><TD>3</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861002">BBa_K861002</A></TD><TD>Generator</TD><TD>Constitutively expressed FadL</TD><TD>Kuanwei Sheng</TD><TD>1508</TD></TR><TR><TD>4</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861003">BBa_K861003</A></TD><TD>Generator</TD><TD>PfadR regulated FadL</TD><TD>Kuanwei Sheng</TD><TD>1549</TD></TR><TR><TD>5</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861003">BBa_K861003</A></TD><TD>Generator</TD><TD>PfadR regulated FadL</TD><TD>Kuanwei Sheng</TD><TD>1549</TD></TR><TR><TD>6</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861010">BBa_K861010</A></TD><TD>Coding</TD><TD>FadD, an inner membrane-associated acyl-CoA synthase</TD><TD>Kuanwei Sheng</TD><TD>1605</TD></TR><TR><TD>7</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861011">BBa_K861011</A></TD><TD>Generator</TD><TD>FadD with RBS and terminator</TD><TD>Kuanwei Sheng</TD><TD>1729</TD></TR><TR><TD>8</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861013">BBa_K861013</A></TD><TD>Generator</TD><TD>IPTG induced FadD</TD><TD>Kuanwei Sheng</TD><TD>1792</TD></TR><TR><TD>9</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861014">BBa_K861014</A></TD><TD>Generator</TD><TD>Constitutively expressed FadD</TD><TD>Kuanwei Sheng</TD><TD>1772</TD></TR><TR><TD>10</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861015">BBa_K861015</A></TD><TD>Generator</TD><TD>Constitutively expressed FadD</TD><TD>Kuanwei Sheng</TD><TD>1772</TD></TR><TR><TD>11</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861016">BBa_K861016</A></TD><TD>Generator</TD><TD>PfadR regulated FadD</TD><TD>Kuanwei Sheng</TD><TD>1813</TD></TR><TR><TD>12</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861020">BBa_K861020</A></TD><TD>Coding</TD><TD>FadE ,acyl-CoA dehydrogenase</TD><TD>Kuanwei Sheng</TD><TD>2445</TD></TR><TR><TD>13</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861021">BBa_K861021</A></TD><TD>Coding</TD><TD>FadE gene for fatty acid degradation from Salmonella enterica LT2</TD><TD>Kuanwei Sheng</TD><TD>2592</TD></TR><TR><TD>14</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861022">BBa_K861022</A></TD><TD>Generator</TD><TD>FadE with RBS and terminator</TD><TD>Kuanwei Sheng</TD><TD>2569</TD></TR><TR><TD>15</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861023">BBa_K861023</A></TD><TD>Generator</TD><TD>S-FadE with a RBS and a terminator</TD><TD>Kuanwei Sheng</TD><TD>2716</TD></TR><TR><TD>16</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861024">BBa_K861024</A></TD><TD>Generator</TD><TD>IPTG induced FadE</TD><TD>Kuanwei Sheng</TD><TD>2632</TD></TR><TR><TD>17</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861025">BBa_K861025</A></TD><TD>Generator</TD><TD>Constitutively expressed FadE</TD><TD>Kuanwei Sheng</TD><TD>2612</TD></TR><TR><TD>18</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861026">BBa_K861026</A></TD><TD>Generator</TD><TD>Constitutively expressed FadE</TD><TD>Kuanwei Sheng</TD><TD>2612</TD></TR><TR><TD>19</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861027">BBa_K861027</A></TD><TD>Generator</TD><TD>Constitutively expressed FadE</TD><TD>Kuanwei Sheng</TD><TD>2612</TD></TR><TR><TD>20</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861028">BBa_K861028</A></TD><TD>Generator</TD><TD>PfadR regulated FadE</TD><TD>Kuanwei Sheng</TD><TD>2653</TD></TR><TR><TD>21</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861029">BBa_K861029</A></TD><TD>Generator</TD><TD>PfadR regulated S-FadE</TD><TD>Kuanwei Sheng</TD><TD>2800</TD></TR><TR><TD>22</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861030">BBa_K861030</A></TD><TD>Coding</TD><TD>FadB, gene for fatty acid degradation from E.coli K12</TD><TD>Kuanwei Sheng</TD><TD>2190</TD></TR><TR><TD>23</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861031">BBa_K861031</A></TD><TD>Coding</TD><TD>FadJ, gene for fatty acid degradation from E.coli str. K12</TD><TD>Kuanwei Sheng</TD><TD>2145</TD></TR><TR><TD>24</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861032">BBa_K861032</A></TD><TD>Coding</TD><TD>S-FadB gene for fatty acid degradation from Salmonella enterica LT2</TD><TD>Kuanwei Sheng</TD><TD>2190</TD></TR><TR><TD>25</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861033">BBa_K861033</A></TD><TD>Generator</TD><TD>FadB with a RBS and a terminator</TD><TD>Kuanwei Sheng</TD><TD>2314</TD></TR><TR><TD>26</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861034">BBa_K861034</A></TD><TD>Generator</TD><TD>FadJ gene with a RBS and a terminator</TD><TD>Kuanwei Sheng</TD><TD>2269</TD></TR><TR><TD>27</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861035">BBa_K861035</A></TD><TD>Generator</TD><TD>S-FadB with a RBS and a terminator</TD><TD>Kuanwei Sheng</TD><TD>2314</TD></TR><TR><TD>28</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861036">BBa_K861036</A></TD><TD>Generator</TD><TD>IPTG induced FadA and FadB</TD><TD>Kuanwei Sheng</TD><TD>3570</TD></TR><TR><TD>29</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861037">BBa_K861037</A></TD><TD>Generator</TD><TD>IPTG induced FadI and FadJ</TD><TD>Kuanwei Sheng</TD><TD>3672</TD></TR><TR><TD>30</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861038">BBa_K861038</A></TD><TD>Generator</TD><TD>IPTG induced S-FadA and S-FadB</TD><TD>Kuanwei Sheng</TD><TD>3673</TD></TR><TR><TD>30</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861038">BBa_K861038</A></TD><TD>Generator</TD><TD>IPTG induced S-FadA and S-FadB</TD><TD>Kuanwei Sheng</TD><TD>3673</TD></TR><TR><TD>30</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861400">BBa_K861400</A></TD><TD>Coding</TD><TD>FadA protein coding sequence for fatty acid degradation</TD><TD>Kuanwei Sheng</TD><TD>1164</TD></TR><TR><TD>31</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861041">BBa_K861041</A></TD><TD>Coding</TD><TD>FadI, gene for fatty acid degradation from E.coli str. K12</TD><TD>Kuanwei Sheng</TD><TD>1311</TD></TR><TR><TD>32</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861042">BBa_K861042</A></TD><TD>Coding</TD><TD>FadA, gene for fatty acid degradation from Salmonella enterica LT2</TD><TD>Kuanwei Sheng</TD><TD>1164</TD></TR><TR><TD>33</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861043">BBa_K861043</A></TD><TD>Translational_Unit</TD><TD>FadA with a RBS</TD><TD>Kuanwei Sheng</TD><TD>1185</TD></TR><TR><TD>34</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861044">BBa_K861044</A></TD><TD>Translational_Unit</TD><TD>FadI with a RBS</TD><TD>Kuanwei Sheng</TD><TD>1332</TD></TR><TR><TD>35</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861045">BBa_K861045</A></TD><TD>Generator</TD><TD>FadI with a RBS</TD><TD>Kuanwei Sheng</TD><TD>1288</TD></TR><TR><TD>36</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861046">BBa_K861046</A></TD><TD>Generator</TD><TD>PfadR regulated FadA and FadB</TD><TD>Kuanwei Sheng</TD><TD>3590</TD></TR><TR><TD>37</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861047">BBa_K861047</A></TD><TD>Generator</TD><TD>PfadR regulated FadI and FadJ</TD><TD>Kuanwei Sheng</TD><TD>3693</TD></TR><TR><TD>38</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861048">BBa_K861048</A></TD><TD>Generator</TD><TD>PfadR regulated S-FadA and S-FadB</TD><TD>Kuanwei Sheng</TD><TD>3591</TD></TR><TR><TD>39</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861050">BBa_K861050</A></TD><TD>Coding</TD><TD>FadR, fatty acid sensor from E.coli str. K12</TD><TD>Kuanwei Sheng</TD><TD>720</TD></TR><TR><TD>40</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861051">BBa_K861051</A></TD><TD>Generator</TD><TD>FadR with a RBS and a terminator</TD><TD>Kuanwei Sheng</TD><TD>844</TD></TR><TR><TD>41</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861052">BBa_K861052</A></TD><TD>Generator</TD><TD>Constitutive expressed FadR</TD><TD>Kuanwei Sheng</TD><TD>887</TD></TR><TR><TD>42</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861345">BBa_K861345</A></TD><TD>Generator</TD><TD>IPTG induced S-fadE</TD><TD>Kuanwei Sheng</TD><TD>2779</TD></TR><TR><TD>43</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861346">BBa_K861346</A></TD><TD>Intermediate</TD><TD>IPTG induced promoter with FadA, intermediate part for BBa_K861036</TD><TD>Kuanwei Sheng</TD><TD>1248</TD></TR><TR><TD>44</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861347">BBa_K861347</A></TD><TD>Intermediate</TD><TD>IPTG induced promoter with FadI, intermediate part for BBa_K861037</TD><TD>Kuanwei Sheng</TD><TD>1395</TD></TR><TR><TD>45</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861348">BBa_K861348</A></TD><TD>Intermediate</TD><TD>IPTG induced promoter with S-FadA, intermediate part for BBa_K861038</TD><TD>Kuanwei Sheng</TD><TD>1351</TD></TR><TR><TD>46</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861349">BBa_K861349</A></TD><TD>Intermediate</TD><TD>PfadR with FadA, intermediate part for BBa_K861046</TD><TD>Kuanwei Sheng</TD><TD>1269</TD></TR><TR><TD>47</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861350">BBa_K861350</A></TD><TD>Composite</TD><TD>PfadR with FadI, intermediate part for BBa_K861047</TD><TD>Kuanwei Sheng</TD><TD>1416</TD></TR><TR><TD>48</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861351">BBa_K861351</A></TD><TD>Composite</TD><TD>PfadR with S-FadA, intermediate part for BBa_K861048</TD><TD>Kuanwei Sheng</TD><TD>1372</TD></TR></TBODY></TABLE><H1>Parts Related to Cellulose</H1><TABLE border="0" cellpadding="5" style="border:2px solid #760044;"><TBODY><TR><TH style="background:#f680c3;"> </TH><TH style="background:#f680c4;"> </TH><TH style="background:#f680c4;">Name</TH><TH style="background:#f680c4;">Type</TH><TH style="background:#f680c4;">Description</TH><TH style="background:#f680c4;">Designer</TH><TH style="background:#f680c4;">Length</TH></TR><TR><TD>1</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K86150">BBa_K86150</A></TD><TD>Coding</TD><TD>BcsA,cellulose synthetase, catalytic subunit</TD><TD>Xian Xia</TD><TD>2619</TD></TR><TR><TD>2</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861101">BBa_K861101</A></TD><TD>Generator</TD><TD>BcsA with RBS and terminator</TD><TD>Xian Xia</TD><TD>2743</TD></TR><TR><TD>3</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861102">BBa_K861102</A></TD><TD>Generator</TD><TD>BcsA controlled by promoter repressed by CRP</TD><TD>Xian Xia</TD><TD>2787</TD></TR><TR><TD>4</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861110">BBa_K861110</A></TD><TD>Coding</TD><TD>BcsB,regulator of cellulose synthetase</TD><TD>Xian Xia</TD><TD>2340</TD></TR><TR><TD>5</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861111">BBa_K861111</A></TD><TD>Generator</TD><TD>BcsB with RBS and terminator</TD><TD>Xian Xia</TD><TD>2464</TD></TR><TR><TD>6</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861112">BBa_K861112</A></TD><TD>Generator</TD><TD>BcsB controlled by promoter repressed by CRP</TD><TD>Xian Xia</TD><TD>2508</TD></TR><TR><TD>7</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861120">BBa_K861120</A></TD><TD>Coding</TD><TD>BcsZ, endo-1,4-D-glucanase</TD><TD>Xian Xia</TD><TD>1107</TD></TR><TR><TD>8</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861121">BBa_K861121</A></TD><TD>Generator</TD><TD>BcsZ with RBS and terminator</TD><TD>Xian Xia</TD><TD>1231</TD></TR><TR><TD>9</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861122">BBa_K861122</A></TD><TD>Generator</TD><TD>BcsZ generator repressed by CRP (directly)</TD><TD>Xian Xia</TD><TD>1275</TD></TR><TR><TD>10</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861130">BBa_K861130</A></TD><TD>Coding</TD><TD>BcsC,cellulose synthetase subunit</TD><TD>Xian Xia</TD><TD>3474</TD></TR><TR><TD>11</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861131">BBa_K861131</A></TD><TD>Generator</TD><TD>BcsC with RBS and terminator</TD><TD>Xian Xia</TD><TD>3598</TD></TR><TR><TD>12</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861132">BBa_K861132</A></TD><TD>Generator</TD><TD>BcsC controlled by promoter repressed by CRP</TD><TD>Xian Xia</TD><TD>3642</TD></TR><TR><TD>13</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861140">BBa_K861140</A></TD><TD>Coding</TD><TD>GalU,glucose-1-phosphate uridylyltransferase</TD><TD>Xian Xia</TD><TD>909</TD></TR><TR><TD>14</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861141">BBa_K861141</A></TD><TD>Generator</TD><TD>GalU with a RBS and a terminator</TD><TD>Xian Xia</TD><TD>1033</TD></TR><TR><TD>15</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861142">BBa_K861142</A></TD><TD>Generator</TD><TD>GalU generator repressed by CRP (directly)</TD><TD>Xian Xia</TD><TD>1077</TD></TR><TR><TD>16</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861150">BBa_K861150</A></TD><TD>Coding</TD><TD>GalF, putative regulatory subunit for GalU</TD><TD>Xian Xia</TD><TD>894</TD></TR><TR><TD>17</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861151">BBa_K861151</A></TD><TD>Generator</TD><TD>GalF with a RBS and a terminator</TD><TD>Xian Xia</TD><TD>1018</TD></TR><TR><TD>18</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861152">BBa_K861152</A></TD><TD>Generator</TD><TD>GalF generator repressed by CRP (directly)</TD><TD>Xian Xia</TD><TD>1062</TD></TR></TBODY></TABLE><H1>Parts Related to Colonization</H1><TABLE border="0" cellpadding="5" style="border:2px solid #760044;"><TBODY><TR><TH style="background:#f680c3;"> </TH><TH style="background:#f680c4;"> </TH><TH style="background:#f680c4;">Name</TH><TH style="background:#f680c4;">Type</TH><TH style="background:#f680c4;">Description</TH><TH style="background:#f680c4;">Designer</TH><TH style="background:#f680c4;">Length</TH></TR><TR><TD>1</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861071">BBa_K861071</A></TD><TD>Generator</TD><TD>AdrA gene with a RBS and a terminator</TD><TD>Kuanwei Sheng</TD><TD>1293</TD></TR><TR><TD>2</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861072">BBa_K861072</A></TD><TD>Generator</TD><TD>Constitutively expressed AdrA</TD><TD>Kuanwei Sheng</TD><TD>1336</TD></TR><TR><TD>3</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861073">BBa_K861073</A></TD><TD>Generator</TD><TD>Constitutively expressed AdrA</TD><TD>Kuanwei Sheng</TD><TD>1336</TD></TR><TR><TD>4</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861074">BBa_K861074</A></TD><TD>Generator</TD><TD>Pcar regulated AdrA</TD><TD>Kuanwei Sheng</TD><TD>1337</TD></TR><TR><TD>5</TD><TD>-</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861090">BBa_K861090</A></TD><TD>Coding</TD><TD>YhjH Gene From E.coli str. K12</TD><TD>Kuanwei Sheng</TD><TD>768</TD></TR><TR><TD>6</TD><TD>W</TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K861091">BBa_K861091</A></TD><TD>Generator</TD><TD>YhjH gene with a RBS and a Terminator</TD><TD>Kuanwei Sheng</TD><TD>892</TD></TR></TBODY></TABLE></DIV><DIV class="passage divcell2"><A name="Brief Answers to Safety Questions"><H3>Brief Answers to Safety Questions</H3></A><P align="justify"> Welcome to our Safety Page. We will first answer the safety questions asked by iGEM headquarters briefly, and then discuss safety issues associated with our project in detail. In addition, we will provide our ideas and practice on guaranteeing and developing biosafety.
            </P><P align="justify"><STRONG>Q1. Would any of your project ideas raise safety issues in terms of: Researcher safety, public safety, or environmental safety?</STRONG>
              No. Our design is based on the commonly used nonpathogenic <I>E. coli K.12</I> strain and genes we manipulated are original genes in <I>E. coli</I>. The protein products, at least from current understanding, will cause no harm to researchers, the public and environment. In addition, strict lab practice is executed to further ensure safety.
            </P><P align="justify"><STRONG>Q2. Do any of the new BioBrick parts (or devices) that you made this year raise any safety issues? If yes, 
Did you document these issues in the Registry?
How did you manage to handle the safety issue?
How could other teams learn from your experience?</STRONG>
              Yes, we will discuss this question in latter part of this page (Safety Considerations of Our Biobrick parts and Our Project).
            </P><P align="justify"><STRONG>Q3. Is there a local biosafety group, committee, or review board at your institution? 
              If yes, what does your local biosafety group think about your project?
              If no, which specific biosafety rules or guidelines do you have to consider in your country?</STRONG>
              Yes. All materials obtained have received the approvals from the department's laboratory management committees. We are also obliged to observe the regulations of Teaching Centre of Experimental Biology and apply for approval for materials before we start our project.
            </P><P align="justify"><STRONG>Q4. Do you have any other ideas how to deal with safety issues that could be useful for future iGEM competitions? How could parts, devices and systems be made even safer through biosafety engineering?</STRONG>
              Some classified measures should be taken according to the safety of the material. For example, plasmids that may harm the safety, when submitted, should receive more attention and have a stricter package. Some harmful byproducts during experiments should be eliminated properly.
              We have done our human practice aiming at understanding attitude of the public towards genetically modified baceria and publicizing biosafety ideas to the public, which we think should be popularized to other teams in future iGEM competitions.
            </P><A name="General Safety Issues"><H3>General Safety Issues</H3></A><P align="justify">
              In this part, we will illustrate organisms, reagents and equipments we use that may cause safety problems, and introduce our operation standard, management and trains protecting the team members, public and environment.
              
              As above, all the organisms and DNA hosts are not of high individual or community risk. Until now, the only used organism is <I>E. coli K.12</I> (and some of its varieties, for example, DH5α, a RecA mutated strain). Our current lab of basic-biosafety level 1 is safe enough to manipulate this strain. Only the genome (a generous present from University of Dundee iGEM team), but not the living Salmonella enterica was manipulated in the lab, and the genes from it are homogeneous of <I>E. coli K.12</I>, not associated with pathogenesis. We will not implement our future experimental plan with microorganisms of risk group 2 or vertebrates in our current lab, for too low is the biosafety level and no animal facilities.
              
              To manipulate microorganisms, an ultra clean cabinet is used and strict aseptic technique is followed. All experimenters have been trained on foundation microbiology technique and biosafety. All microorganism contacting vessels are sterilized before and after experiments in appropriate protocols. Also, all microorganism materials will be sterilized before discarded. In this way, we believe that no public or environmental harm will be caused by the experimental organisms. In addition, no one in our team will be hurt by the experimental organisms.
              
              The genetic modifications we make will change metabolism of bacteria. However, there is no evidence both theoretically and experimentally that these modifications will improve the pathogenicity of the bacteria or cause damage to environment even taking the risk of horizontal gene transfer into consideration. The effects of expressing gene <I>adrA</I> regarding infectivity and pathogenicity on <I>E. coli</I> is not clear now, but it still can be easily controlled in the lab environment without human ingestion. Also, we will insert a death device into the bacteria cell when we construct the whole system (still far from now) to avoid the proliferation out of control. For more information about gene safety, please read the next section and documents of our relevant parts on registry.
            </P><P align="justify">
              We have considered the potential harmful chemicals and equipments, as listed below:
              </P><P align="justify"> Basic molecular experiments: NaOH, HCl, SDS, acrylamide, TEMED, ethanol, IPTG, liquid nitrogen, β-mercaptoethanol, xylene cyanol FF.
              </P><P align="justify">  Bacteria culture: ampicillin, kanamycin, chloramphenicol.
              </P><P align="justify">        Chemical analysis and measurements: acetone, cupric acetate, Sudan III, Congo red, Coomassie Brilliant Blue G-250, Coomassie Brilliant Blue R-250.
              </P><P align="justify">              Equipments: UV lamp, supercentrifuge, heating equipments (alcohol burner, PCR amplifier, water bath, dry bath), electrophoresis apparatus, -80℃ refrigerator, ultrasonic cell disruptor.
              </P><P align="justify">              All of these are regular reagents and apparatus in a molecular biology laboratory. The risks of them come from inflammability, explosibility, irritation, corrosivity, toxicity, carcinogenicity and physical injury. But none of them raises special safety issue, with chemical hood, emergency shower, normal personal protection, safety management and safety training.
              Only trace amount of antibiotics are used. Inactivated will they be before discared.
            </P><P><STRONG>Emergency Shower</STRONG></P><A name="Safety Considerations of Our Biobrick parts and Our Project"><H3>Safety Considerations of Our Biobrick parts and Our Project</H3></A><P>
              In this section, we are going to answer safety question 2 in detail, taking the potential risk in the future into concern.
            
              The main safety challenge we must face is that as a practical bacteria therapy our direction is, we shall demonstrate that the “<I>E. coslim</I>” will not harm its host when it is developed completely. As few experiments we can do in such limited time, we have done a series of theoretical work to solve problems in this field.
              Intestine-colonized pathogenic <I>E. coli</I> may cause immune responses, following by diarrhea, inflammation and fever (although the strain we use is considered non-pathogen). Thus we plan to transplant the whole synthetic system to another organism (for example, <I>Bacillus subtilis</I>, has proved safe in human intestine). We know that it is hard as the two organisms are very different on transcriptional mechanisms, but we believe the work of establishing model system (that is what we are doing) in <I>E. coli</I> will make it easier. Also, as the rapid development of synthetic biology and gut microbiology, we hope in the near future, we can modify the genome of <I>E. coli</I> to change it a safe intestine microbe.
            
              Another risk is that when the engineered bacteria get higher efficiency on energy production, they could proliferate out of control. To forestall this situation, we design a “death” device (for more information, click on <A href="https://2012.igem.org/Team:WHU-China/Death">Death</A>) using D-xylose as inducer. It means if you want to stop weight losing, what you have to do is to eat some D-xylose, and the “<I>E. coslim</I>” will die, shed from the intestinal wall and be poured out. What is more, designed as a two-plasmid system, it prevents all possible horizontal gene transfers.
            
              Further safety issues will be raised and discussed in future experiments, including those operates in intestinal model and animals (For more information, click on Future Perspectives).
            </P><A name="Safety Management and Practice"><H3>Safety Management and Practice</H3></A><P align="justify">
              Our project has past a review of an expert committee, of which members are professors or associate professors of microbiology, genetics and bioengineering. Safety issues are considered seriously before they approved our design. 
              We have consulted a few experts for safety questions about both artificial intestinal bacteria and experiments. Their advices help us improve the safety of the whole planning system. For that, we thank Dr. Yulan Wang and Dr. Tiangang Liu so much.
              Our lab is supervised by Teaching Centre of Experimental Biology, Wuhan University (TCEB, whose leader is our instructor <A href="https://2012.igem.org/Team:WHU-China/Team#Zhixiong Xie">Dr. Zhixiong Xie</A>). An expert group of this centre formulates safety guidelines and guarantees their performance in all teaching labs. All experiments we do past its review.
Our safety management system includes the followings:
Management of Chemicals, Equipments and Experimenters: Chemicals and equipments are registered in TCEB before they are available for us. Equipments are checked and maintained regular. And an entrance guard system is used to ensure only the admitted could enter the lab.
                  <UL><LI>
                  Responsibility distribution: All members worked in the lab are divided to three groups. The group leaders arrange schedules of experiments after assessing safety issues in the group (for example, the safety train of the experimental executer). Every member records his/her  experiments in detail in the notebook of the group. Each group takes responses for cleaning and checking risks in the lab in turns. The group leader takes responses to the team leader. The team leader reports regular on safety to engineer Miss Long Yan, who is authorized by TCEB and manages the lab.
                </LI><LI>
                  Management of Chemicals, Equipments and Experimenters: Chemicals and equipments are registered in TCEB before they are available for us. Equipments are checked and maintained regular. And an entrance guard system is used to ensure only the admitted could enter the lab.
                </LI><LI>
                  Training: Our team members have been trained after Guidance of Student Experiments formulated by TCEB.                
                </LI></UL></P><P><STRONG>Guidence of Student Experiments</STRONG></P><A name="Biosafety and Publicity"><H3>Biosafety and Publicity</H3></A><P align="justify">
              We believe that biosafety is not an issue which should be only considered inside biological lab, but also around people and communities. Spreading knowledge of biosafety to the public will help eliminate misunderstanding and prejudice, from which the science and all the people will benefit. We hope that sharing this idea with all iGEM teams can be useful to take advantage on biosafety.
            
              To investigate public attitude towards our project and to publicize our biosafety ideas are what we do for our human practice. As “eating bacteria” is somehow an unacceptable concept now, we would like to find whether people know the truth about biosafety issues raised by bioengineered bacteria, or they are just panicked by their imaginary “bacteria”. And then, after initiating basic knowledge of microbiology, gene engineering and synthetic biology, we shall take a look on if people’s opinion to biosafety issues will change. In this way, we will estimate the value of our scientific publicity.
            
              The rational discussions of biosafety we take with the public do not limited on our project, but also hotspot issues such as transgenics and stem cell therapies. For more information, please click on <A href="https://2012.igem.org/Team:WHU-China/Human_Practice">Human Practice</A>.
            </P><P><STRONG>Introducing a Simplified Intestinal Model</STRONG></P></DIV><DIV class="passage divcell3"><A name="Protocol for fluorescence measurement"><H3>Protocol for Fluorescence Measurement</H3></A><P align="justify">Minimal Medium</P><P align="justify">M9: </P><P align="justify"> For 1L Medium add</P><P align="justify">Na<SUB>2</SUB>HPO<SUB>4</SUB>·12H<SUB>2</SUB>O                  15g </P><P align="justify">KH<SUB>2</SUB>PO<SUB>4</SUB>                           3g</P><P align="justify">NaCl                              1g </P><P align="justify">NH<SUB>4</SUB>Cl                            0.5g</P><P align="justify">After autoclaving, add: </P><P align="justify">MgSO<SUB>4</SUB>(1mol/L)                    2mL</P><P align="justify">Bacteria strain:CaCl<SUB>2</SUB>                            100μL </P><P align="justify">Triton X-100 (as emulsifier)           2uL</P><P align="justify"><STRONG>Note</STRONG></P><P align="justify">1.For M9 medium using oleic acid as sole carbon source, various amount of oleic acid was first emulsified 1:1 with 10% Triton X-100. M9 medium was then slowly added with constant vortex. M9 medium with high concentration of oleic acid was diluted by M9 medium with triton to form various concentrations；</P><P align="justify">2. For M9 medium using glucose as sole carbon source, M9 medium with high concentration of glucose was diluted by M9 medium to form various concentrations.</P><P align="justify"><STRONG>Step</STRONG></P><P align="justify">1. Seed liquor which was activated over night was inoculated into M9 medium which contains different concentration of oleic acid. And it was then incubated at 37℃ for 24 hours;</P><P align="justify">2. After 24h of incubation in 24 well plates in 37℃, bacteria culture was centrifuged at 3000rmp for 5min, washed and resuspended in PBS;</P><P align="justify">3. We detected the OD600 and fluorescence of using SpectraMax M2 plate reader (Molecular Devices) .Excitation at 584 nm and emission at 607 nm were used. All fluorescence was normalized with cell density by measuring the absorbance at 600 nm.</P><A name="Protocols of Cupric-Soap Reaction"><H3>Protocols of Cupric-Soap Reaction</H3></A><P align="justify">To test metabolism of long fatty acids, we used oleic acid as sole carbon source in mediums, and used cupric-soap reaction to determinate oleic acid concentration preliminarily. </P><P align="justify">Modified M9 minimal medium with emulsified oleic acid as sole carbon</P><P align="justify"> Minimal medium was the same with that in Materials and methods for PfadR </P><P align="justify">What's Important:</P><P align="justify">Slowly pour M9 minimal medium into mixture of oleic acid and Triton X-100 to get more homogeneous solution.</P><P align="justify">Analysis the concentration of the oleic acid in the medium by cupric-acetate method</P><P align="justify"><STRONG>Step</STRONG></P><P align="justify">1.Collect 5 ml medium which has been used to cultivate bacteria. Then centrifuge the medium at 3000rpm for 10 min to separate bacteria and medium;</P><P align="justify"> 2.Decant 3ml supernatant liquid into a 10ml EP. Add 3ml acetone to the liquid, 1ml at a time, shaking 10-20 times before adding another 1 ml in order to avoid the effect of the ions of the liquid during the extraction progress;</P><P align="justify"> 3.Add 3 ml isooctane once ,shaking for at least 90s, stand for 2min or longer until layering completely;</P><P align="justify" p="">4.Collect 3 ml clear isooctane in a 5 ml EP, Add 800?l cupric-acetate (5% m/v, adjust pH to 6.8 with pyridine), Shaking for 90 seconds, stand for 2min or longer until layering completely;</P><P align="justify">5.Detect OD of organic phase in a spectrometry at 715nm.</P><P align="justify"><STRONG>The Standard Curve</STRONG></P><P align="justify">We add 1.6ml, 3.2ml, 4.8ml, 6.4ml, 9.6ml, 11.2ml oleic acid into 3ml M9 medium to generate a gradient. The absorbency is measured as described in the protocol.</P><P align="justify">We slightly modified the methods provided by Kwon and Rhee, 1986, adding the extraction step, for it is indispensable when bacteria are in the medium. The standard curve is still very close to that of the paper's, which proves the applicability and accuracy of our modified protocol.</P><A name="Krebs-Ringer Phosphate（KRPD Buffer Solution）"><H3>Krebs-Ringer Phosphate（KRPD Buffer Solution）</H3></A><P align="justify"><STRONG>Solutions</STRONG></P><P align="justify"> Solution A：</P><P align="justify">For 100 mL: </P><P align="justify">NaCl  7.5985g ; KCl  0.3727g ; CaCl<SUB>2</SUB>  0.1054g; glucose-H<SUB>2</SUB>O  0.9495g, </P><P align="justify"> Solution A：</P><P align="justify">For 100 mL:  </P><P align="justify"> NaH<SUB>2</SUB>PO<SUB>4</SUB>·2H<SUB>2</SUB>O 0 .2964g ;Na<SUB>2</SUB>HPO<SUB>4</SUB>·12H<SUB>2</SUB>O  2.9011g，</P><P align="justify">After dissolving, add MgSO<SUB>4</SUB>·7H<SUB>2</SUB>O  0.3130g</P><P align="justify">Mix A solution and B solution, add 790ml dd H<SUB>2</SUB>O, then regulate pH to 7.2~7.4 using 1M NaOH/1M HCl, finally add dd H<SUB>2</SUB>O to a final total volume of 1L.  </P><P align="justify"><STRONG>Procedures</STRONG></P><P align="justify">1. cultivate cells in 1 liter of medium to late exponential phase;</P><P align="justify">2. harvest by centrifugation at 35℃;</P><P align="justify">3. wash twice with 2 liters of warm(35℃)0.05M potassium phosphate buffer (pH 7.4) containing 1%(v/v) Triton X-100;</P><P align="justify">4. resuspend in 0.05M potassium phosphate buffer (pH 7.4) containing mercaptoethanol;</P><P align="justify">5. add sufficient volume of buffer to give a concentration of about 50mg(dry weight) of cells/ml;</P><P align="justify">6. disrupt cells with a Bransor Sonifier for 1min. The treatment was applied for 15s intervals, under 4 ℃;</P><P align="justify"> 7. centrifuge at10,000×g for 30min; </P><P align="justify">8. decant supernatant liquid for enzymatic assay, the protein concentration was determined by the biuret method;</P><P align="justify">9.analyze fatty acid oxidation in cell-free extracts</P><P align="justify">Note: little oxidation was observed when less than 1mg.</P><P align="justify"><STRONG>Reaction Mixture</STRONG></P><P align="justify">1.0ml of freshly oxygenated Krebs-Ringer phosphate(pH 7.4)</P><P align="justify">Palmitate-1-14C, 20nmole (80,000 counts/min)</P><P align="justify">CoA, 1 μ mole</P><P align="justify">NAD, 1μ mole</P><P align="justify">ATP, 1μ mole</P><P align="justify">Succinate, 10 n moles</P><P align="justify">Supernatant protein, 1~5 mg</P><P align="justify">dd H<SUB>2</SUB>O to a final total volume of 2.0ml </P><A name="Cellulose Detection"><H3>Cellulose Detection</H3></A><P align="justify">1. Inoculate the bacteria into liquid LB medium and then incubate it for 24 hours at 37℃;</P><P align="justify">2. After centrifugation, supernatant is preserved for use, while deposits are resuspended with PBS and adjust to OD600 1.0;</P><P align="justify">3. Exceed cellulase was used to digest cellulose in supernatant and deposits. After the cellulase is added, 1mL solution was sampled every 3 min and cellulose of the sample was inactivated immediately;</P><P align="justify">4. Ten minutes later, the reduce sugar is detected in all the samples with or without cellulose;</P><P align="justify">5. 2mL DNS solution is mixed with the sample, and incubated in boiling water for two minutes, then it is cooled rapidly;</P><P align="justify">6. After 9 mL ddH<SUB>2</SUB>O being added into the solution, record absorbance at 540nm;</P><P align="justify">7.The Standard Curve of glucose concentration</P><P align="justify">Then all tubes was treated in the same way with that in step 5 and 6.</P><P align="justify">8.Calculate the amount of cellulose in cell culture.</P><A name="In vitro Experiment"><H3><I>In vitro</I> Experiment</H3></A><P align="justify">1. inoculate 100μl bacteria into 100ml LB , shaking at  37C overnight .</P><P align="justify">2. inoculate 1L LB using the culture above at the next morning , on the scale of 1:20 .</P><P align="justify">3. Separate above medium into 250ml flask , shaking at 37C for 3-4h .</P><P align="justify">4. 8000 rpm , 6min to collect the bacteria .</P><P align="justify">5. Using PBS to resuspend the bacteria to 50mg / ml .</P><P align="justify">6. Ultrasonication 1min , every 15s having an interval to cool on the ice .</P><P align="justify">7. Using Coomassie blue staining to measure the concentration of the total protein .</P><P align="justify">8. In 2ml reaction system , add 5ul oleate ,40 ul Triton 10% , 1ml Krebs-Ringer phosphate buffer , 10nmol succinate , 1 umol COA , 1umol ATP , 1 umol NAD . </P><P align="justify">9. react at 37C for 6h</P><P align="justify">In this experiment , the concentration of total protein in the cell extracts of the five sample are almost the same , so we assume that adding the same volume guarantee the same amount of protein is added . </P><P align="justify">The reaction system for the control BBa_R0011+(R0011), BBa_R0011+fadE , BBa_R0011 + fadD , BBa_R0011 + Samonella A Samonella B , BBa_R0011 + fadI fadJ : 1ml Krebs-Ringer phosphate buffer , 10nmol succinate , 1 umol COA , 1umol ATP , 1 umol NAD , 1ml cell extract .</P><P align="justify">The reaction system for BBa_R0011+fadD and BBa_R0011+fadE mixture : 2ml Krebs-Ringer phosphate buffer , 20 nmol succinate , 2 umol COA , 2 umol ATP , 2 umol NAD , 1ml cell extract for each gene.</P>

<P align="justify">The reaction system for BBa_R0011+fadD , BBa_R0011+fadE , and BBa_R0011 + Samonella fadA Samonella fadB mixture :
3ml Krebs-Ringer phosphate buffer , 30nmol succinate , 3umol COA , 3umol ATP , 3 umol NAD , 1ml cell extract for each gene.</P><P align="justify">The reaction system for BBa_R0011+fadD , BBa_R0011+fadE , BBa_R0011+Samonella A Samonella B , and BBa_R0011+fad I fad J mixture :4ml Krebs-Ringer phosphate buffer , 40nmol succinate , 4umol COA , 4umol ATP , 4 umol NAD , 1ml cell extract for each gene </P><A name="Plate Assay"><H3>Plate Assay</H3></A><P align="justify">1. J12107- AdrA and control RBS was incubate and vortex in LB medium with Ampicillin overnight</P><P align="justify">2. OD600 was measured; LB was then added to make the two test tubes had the same OD600</P><P align="justify">3. Bacteria were inoculated by sterilized needle piercing a pre-labeled 0.3% agar LB semisolid plate for 4-5 minutes in super clean bench.</P><P align="justify"> 4. Carefully placed the plate horizontally in a 37 degree incubator overnight. Avoid shaking of the plate.</P><P align="justify">5. Clones on the plate were observed. </P><A name="Materials and Methods of SDS-PAGE"><H3>Materials and Methods of SDS-PAGE</H3></A><P align="justify">Coomassie Blue Stain</P><P align="justify">-Coomassie brilliant blue G-250 50mg</P><P align="justify">-95% ethanol                25ml</P><P align="justify">-85%H<SUB>3</SUB>PO<SUB>4</SUB>                 50ml</P><P align="justify">-H<SUB>2</SUB>O, adjust to 500ml</P><P align="justify">-filter</P><P align="justify">Destain solution </P><P align="justify">-methanol        250ml</P><P align="justify">-acetic acid         50ml</P><P align="justify">-H<SUB>2</SUB>O     adjust to 500ml</P><P align="justify">2x SDS loading buffer</P><P align="justify">-0.5mol/l  Tirs-HCl(PH6.8) 25ml</P><P align="justify">-10%SDS   8ml</P><P align="justify">-50%glycerol   20ml</P><P align="justify">-2-mercaptoethanol  2ml</P><P align="justify">-1%Bromphenol Blue  4ml</P><P align="justify">-H<SUB>2</SUB>O        adjust to 100ml</P><P align="justify">10xSDS-PAGE running buffer </P><P align="justify">Tris base,               30.3 g </P><P align="justify">M glycine                144.1g</P><P align="justify">SDS                      10 g</P><P align="justify">-H<SUB>2</SUB>O                 adjust to 1L</P><P align="justify">Steps</P><P align="justify">Protein expression</P><P align="justify">1.inoculate the liquid strains into LB medium supplement with 50μg/mL ampicillin, incubate the medium at at 37℃ until OD600 reaches 0.6;</P><P align="justify">2. Separate the culture into two test tubes. Add IPTG into one of two tubes at a final concentration of 1mM to induce the expression of </P><P align="justify">3. 1 mL of the culture is sampled every hour. After centrifugation, deposits was suspended with 300 μL PBS and 200 μL 2x SDS loading buffer, then incubated in boiling water for 15 min.</P><P align="justify">4. 10μl of samples was load in lanes, run the SDS-PAGE</P><P align="justify">5. Stained the SDS-PAGE 5 hours</P><P align="justify">6. Destain the SDS-PAGE until you can see the protein band.</P></DIV><DIV class="passage divcell4"><H3>team history</H3><P><STRONG>Dec. 2011</STRONG></P><P align="justify">WHU iGEM team was established</P><P><STRONG>Dec. 2011 to Feb. 2012</STRONG></P><P align="justify">Every one presented their own idea, then we discussed the feasibility of these ideas.</P><P><STRONG>Feb. 2012 </STRONG></P><P align="justify">The final project was determined, named E.coslim</P><P><STRONG>Mar. 2012</STRONG></P><P align="justify">We finished an outstanding presentation. Our reply was approved by the leaders of college of life sciences, Wuhan University.</P><P><STRONG>Apr. 2012 </STRONG></P><P align="justify">Our iGEM team was divided into 3 groups—group of Sheng, group of Xia and group of Mei. </P><P><STRONG>Apr. 2012 to prsent</STRONG></P><P align="justify">Experiments started….</P><P><STRONG>Apr. 2012</STRONG></P><P align="justify">Group of Sheng:  FADR was connected to pSB1A2 plasmid carrier</P><P align="justify">               FADR was connected to RFP gene</P><P align="justify">Group of Xia: starting to connect genes of CI control system </P><P align="justify">            Testing the function of CI control system</P><P align="justify">Group of Mei: YhjH/ FadE/ FadD/ FadL gene was connected to pSB1A2 plasmid carrier</P><P><STRONG>May. 2012 </STRONG></P><P align="justify">Group of Sheng: FadB/ FadJ gene was connected to pSB1A2 plasmid carrier, then BBa_B0030(RBS) as well</P><P align="justify">Group of Xia:  They devised two promoters p110 and p101, which were expected to be controlled by glucose. However, they failed.</P><P align="justify">Group of Mei:  YhjH/ FadE/ FadD/ FadL gene was connected to BBa_B0030(RBS)</P><P><STRONG>June 2012 </STRONG></P><P align="justify">G of S: FadR was connected to low-copied plasmid carrier.</P><P align="justify">       sFadB and sFadA genes were connected to BBa_B0030 (RBS)</P><P align="justify">       sFadE gene was connected to pSB1A2 plasmid carrier</P><P align="justify">       G of X: the connection of genes, which were relative with cellulose, was finished.</P><P align="justify">       Another two promoter were devised, p1 and p2</P><P align="justify">G of M: YhjH/ FadE/ FadD/ FadL gene was connected to BBa_B0024 (terminator)</P><P><STRONG>July 2012 </STRONG></P><P align="justify">G of S: FadJ was connected to BBa_B0024 (terminator), sFadE gene was copied by PCR technology</P><P align="justify">      On the front half of this month, they conducted several pre-experiments of oleic acid test</P><P align="justify">      On the next half month, they started normal experiments of oleic acid test</P><P align="justify">G of X: started to test the function of p1 and p2</P><P align="justify">G of M: YhjH/ FadE gene was connected to BBa_J23100 (promoter)</P><P align="justify">      Finish the standard curve of oleic acid test</P><P><STRONG>Aug. 2012 </STRONG></P><P align="justify">G of S: sFadE was induced to point mutation, then it was connected to BBa_B0030 (RBS) and BBa_B0024 (terminator) respectively</P><P align="justify">        sFadB was connected to BBa_B0030 (RBS) and BBa_B0024 (terminator) respectively.</P><P align="justify">G of X: test the function of cellulose-controlled genes, having got satisfying results</P><P align="justify">G of M: copied Adra gene and finished the connection of BBa_B0030 (RBS), BBa_B0024 (terminator) and BBa_J23107 (promoter), BBa_J23114 (promoter) respectively. FadE/YhjH/FadD/fadL were connected to BBa_J23107 (promoter), BBa_J23114 (promoter). </P><P><STRONG>Sep 2012</STRONG></P><P align="justify">G of S: Transferred all the parts in pSB1A2 into pSB1C3, Tested the function of PfadR, Characterized the effect of each gene on fatty acid consumption, started to set up the platform for in vitro experiments.</P><P align="justify">G of X: Transferred all the parts in pSB1A2 into pSB1C3, tested the function of cellulose system. Repeat the test of the function of cellulose-controlled genes.</P><P align="justify">G of M: Transferred all the parts in pSB1A2 into pSB1C3, test the function of Adra/YhjH.</P></DIV><DIV class="passage divcell5"><H3>Brainstorming</H3><H4>About E.coslim by Kuanwei Sheng</H4><P align="justify">In order to help people lose weight, besides our three devices, we also have come up with many other creative ways. </P><P align="justify"><STRONG>Ⅰ.Short chain peptides synthesis</STRONG></P><P align="justify">Recent researches have indicated that some short chain peptides in intestine have effect on inhibiting appetites, therefore decrease the food intake. We thus formed the idea that we could synthesis a DNA chain that encodes those short peptides.</P><P align="justify"><STRONG>Ⅱ.Biosynthesis of L-carnitine</STRONG></P><P align="justify">L-carnitine is a molecule that facilitates the progress of transporting fatty acids into mitochondria where these fatty acids will be disintegrated. We once considered use L-carnitine to help the host metabolize fatty acid better. However, the biosynthesis of L-carnitine has too many derivatives or the pathway is patented by others.</P><P align="justify"><STRONG>Ⅲ.Xylose isomerase</STRONG></P><P align="justify">Xylose is a prebiotics that can hardly be absorbed by human. We consulted many papers and find that glucose can be converted into xylose by xylose isomerase. Therefore, we thought maybe we could lower the glucose available in intestine by expressing xylose isomerase. However, this process is shown to be reversible latter. Mutated the sequence of the protein may generate high converting rate, yet it is too laborious and risky for a short time project.</P><H4>Other novel ideas</H4><P align="justify"><STRONG>Tackle Water bloom by Tong Wang and Kuanwei Sheng</STRONG></P><P align="justify">Since the detrimental effects caused by cyanobacteria to the environment such as making water carcinogenic have become a serve global problem, we therefore tried to employ E.coli as an expression system to eliminate these detrimental effects. When we first took over this project, we thought about limiting the growth of cyanobacteria. Along with the process we got to read a large amount of relevant papers about cyanobacteria, we found that not only the main detrimental effects caused by cyanobacteria is attributed to its product which is a cyclic peptide called microcystin, but also microcystin can regulate the population density. Then we came to realize the importance of microcystin and began to search information about it. Through over this process, we discovered a gene cluster which is responsible for the microcystin degradation pathway. It encodes four enzymes——MlrA、MlrB、MlrC and MlrD. Also, we found that some non-toxic cyclic peptides produced by cyanobacteria such as Anabaenopeptin B and Anabaenopeptin F can induce lysis of cyanobacteria. The latter finding can be utilized as an effective cell population density control mechanism. Thus we thought about constructing two independent systems to eliminate cyanobacteria, one about inducing lysis of cyanobacteria and the other about degrading microcystin. Finally we gave up this project because of the reasons that these gene clusters are too large to be expressed in E.coli and that E.coli cannot survive in the sea, however, we still feel proud of these fancy ideas.</P><P align="justify"><STRONG>Desalination of sea water by Kuanwei Sheng</STRONG></P><P align="justify">We came up with the thought that engineering bacteria can intake ions like Na+, cl-, Mg2+ and etc. under special stimulus. Also, under another certain stimulus, the bacteria can export those salts out of cells for reuse. Therefore, we can use these bacteria to desalinize sea water and extract the salt the same time. However, there is no such ion channel that can meet our needs.</P><P align="justify"><STRONG>Sense the earthquake By Min Ye</STRONG></P><P align="justify">We once tried to find proteins that can sense vibration and construct a pathway to report that vibration. However, we are not able to find the protein that can meet our needs.</P><P align="justify"><STRONG>Auto plasmid preps By Kuanwei Sheng</STRONG></P><P align="justify">We thought about constructing a synthetic protein that combines zinc finger protein which can recognize the specific sequence of DNA and signal peptide that can make proteins be exported out of the bacteria by secretion pathway. Therefore, it is possible that plasmid can be exported out the cells together with the zinc finger protein.</P><P align="justify"><STRONG>Outer Membrane Vesicle (OMV) to treat cancer By Kuanwei Sheng</STRONG></P><P align="justify">We thought to use Outer Membrane Vesicle (OMV) to treat cancer. Specifically, we thought about localizing antibody that can recognize certain cancer on the surface of OMVs via signal peptides. Also, we thought we may localize cell division inhibitor protein or protein that lead to cell death inside the OMVs. Therefore, we hoped that the OMV excreted by the bacteria can recognize and kill cancer cells.</P><P align="justify"><STRONG>Multicell Yeast By Wenxiong Zhou</STRONG></P><P align="justify">Transform the yeast from a single-cell microbe to a multi-cell organism by setting bistability of gene expression among cells of yeasts adhered in amalgamation.</P><P align="justify"><STRONG>To kill superbacteria</STRONG></P><P align="justify">Now with the spread usage of the antibiotics, many bacteria adopt ability to fight against the antibiotics. And now it’s a big problem that antibiotics are no longer useful as before. So to kill these bacteria, Jing and her group thinks they can device some proteins or artificial micromachine to detect DNA that code proteins contributing to the ability to degenerate antibiotics. And then by transferring these plasmids coding artificial proteins or micromachines, they can inhibit the expression of enzymes or proteins degenerating antibiotics. </P></DIV><DIV class="passage divcell6"><A name="team"><H3>Team</H3></A><H3><STRONG>Presentation</STRONG></H3><A name="human practice"><H3><STRONG>Human Practice</STRONG></H3></A><A name="doodle"><H3><STRONG>Doodle</STRONG></H3></A><A name="funny pictures"><H3><STRONG>Funny Pictures</STRONG></H3></A></DIV><DIV class="passage divcell7"><H3><STRONG>A Letter from Fancy</STRONG></H3><P align="left">There is a very meaningful sentence in the novel The Lord of the Ring: when the respected old Billbo was going to leave the village in which he had lived for many years, he gave a lecture--”The people among you whom I recognize haven’t reached half whom I should do, besides, among those whom I love are less than the half I should do, either.</P><P align="left">None of us need to fight in a war or fight with monsters. However, all of us have indeed fought and stuck to the very end, regardless of whatever obstacles to overcome and whatever precious to sacrifice.</P><P align="left">The greatness lies in the persistence to do the ordinary things perfectly. We are all inspired by our design. However, the gap between the repetitive, seemingly endless molecular cloning and the last magic probiotic can easily wreck the passion. To the opposite, most of us have successfully endured the monotonous life. Sometimes the enzymes just don’t function well. Other times the PCR never get right for some unclear reasons. We just survived those really despair moments and made one step closer to our goal.</P><P align="left">And those heart-touching scenes are clear as if they happened yesterday. Regardless of our persuation to wait for the rain to stop, our captain determined to fetch the induction cooker in his dormitory to perform an reaction as soon as possible (We don’t have an appropriate heater in our lab). When his figure disappeared in the dark and rain, words failed me; when I got a message at 3 o’ clock in the midnight, the excited tune claiming that we no longer needed to worry about our competent cells overwhelmed me with tears; when I heard that our “artist” having caught a cold but promised that he would complete all the pictures of the web site in time, I fell into silence again; every time when Xian Xia stayed overnight in the lab, the room would be quite tidy and all the reagents would got replenished. When I saw this in the next morning, I was greatly touched.</P><P align="left">Most of us has sacrificed much time to pursue personal interest. We just decrease the time for required exams for going abroad, for working in our own labs, and for some courses we are really interested in, let alone our hobbies. No time for travelling, reading lasted paper on interested topics, shopping, chatting with friends and so on. However, after the complaints and anguish, we finally reached such a peaceful and simple spiritual state that we just want to try our best to get the best result.</P><P align="left">A man who never experienced iGEM in WHU can never really understand the complex of feelings: glory and dream, perseverance and persistence, joy and tears. You can never image how brave and respected my teammates are, how many efforts and love they have put in the team, and how strong the relationship we have built since the team has been organised.
  
 </P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:WHU-China/Notebooks">http://2012.igem.org/Team:WHU-China/Notebooks</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WHU-China/Notebooks" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WHU-China/Notebooks" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WHU-China/Notebooks&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WHU-China/Notebooks&amp;oldid=298735" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>