"
Page
Discussion
View source
History
teams
Log in
 
Team:WHU-China/Project/fatty acid degradation
From 2012.igem.org
Fatty Acid Degradation Device
Purpose
To help people lose weight without the need of food restriction, we designed a genetically modified E.coli that can sense and degrade excessive fatty acids intake by the host. We hope that, together with other two   devices we designed, we can introduce our E.coslim as resident in intestine to consume the excessive calories intake   by the host and regulate intestinal microbiota.
Outline
Genes that are responsible for degradation and transportation of fatty acids (FAs) from E.coli K12 and from Salmonella enterica LT2 were cloned. Also, a promoter that can that can be regulated soley by fatty acids was also   designed. By placing those fatty acids degradation genes downstream of the artificially designed promoter that can sense   the concentration of FAs, we hope to create a device that to degrade FAs only when the concentration of FAs is high.
Long chain fatty acids are firstly being imported by the transmembrane protein FadL. After FAs get into cells, a CoA will   be added by inner membrane-associated FadD (acyl-CoA synthase). β-oxidation is initiated by FadE(acyl-CoA dehydrogenase),   which will convert acyl-CoA into enoyl-CoA. The following cycles of hydration, oxidation, and thiolytic cleavage are carried   out by tetrameric complex consisting of two FadA and two FadB proteins or two FadI and two FadJ in anaerobic condition.   FadR is a transcriptional regulator that, when not binds to acyl-CoA, can either serve as an activator for fatty acid   synthesis gene like FabA, FabB and etc. or a repressor for fatty acid degradation gene like FadA, FadB, FadD FadE, FadL,   FadI, FadJ and etc. After long chain fatty acids are converted to fatty acyl- CoA by FadD, it can bind to FadR. The binding   will alter the conformation of FadR, making FadR unable to bind to the DNA sequence it recognizes to fulfill its function.   Therefore, FadR can no longer activate or repress the transcription of genes downstream FadR binding sites. However, to our   knowledge, there is no promoter exists in nature that can respond solely to FadR since those promoters are often regulated   by glucose concentration or oxidative stress and many other factors. In our design, FadL, FadD, FadE, FadA, FadB FadI, FadJ and FadA from Escherichia coli K12, and FadA, FadB and FadE   from Salmonella enterica LT2 are placed downstream a synthetic promoter PfadR (BBa_K861060) to make them under the   sole regulation of fatty acids concentration.
Progress
Cloning of the gene
First, the genome of Escherichia coli K12 str. DH5ɑ and Salmonella enterica LT2 (symbolized as S-) were got   and amplified in PCR using primers for each gene. The sequences of the primers used are as bellow (5’---3’). FadR  Forward: GGAATTCTCTAGAATGGTCATTAAGGCGCAAAG Reverse: GACTAGTCTTATCGCCCCTGAATGGCTAAATC FadA  Forward: GGAATTCTCTAGAATGGAACAGGTTGTCATTGTCG Reverse: GACTAGTTTAAACCCGCTCAAACACCGT FadB  Forward: GGAATTCTCTAGAATGCTTTACAAAGGCGACACC Reverse: GACTAGTTTAAGCCGTTTTCAGGTCGCC FadD  Forward: GGAATTC TCTAGATTGAAGAAGGTTTGGCTTAACCG Reverse: GACTAGTTCAGGCTTTATTGTCCACTTTGC FadE  Forward: GGAATTC TCTAGAATGATGATTTTGAGTATTCTCG Reverse: GACTAGTTTACGCGGCTTCAACTTTCCG FadL  Forward: GGAATTC TCTAGAATGAGCCAGAAAACCCTG Reverse: GACTAGTTAGAACGCGTAGTTAAAGTTAG FadI  Forward: GGAATTC TCTAGA ATGGGTCAGGTTTTACC Reverse: GACTAGTTTATTCCGCCTCCAGAACCA FadJ  Forward: GGAATTCTCTAGAATGGAAATGACATCAGC Reverse: GACTAGTTTATTGCAGGTCAGTTGCAGTTG S-FadA  Forward: GGAATTCTCTAGAATGGTCATTAAGGCGCAAAG Reverse: GACTAGTCTTATCGCCCCTGAATGGCTAAATC S-FadB  Forward: GGAATTCTCTAGAATGCTTTATAAAGGCGACACC Reverse: GACTAGTTAAGCCGTTTTCAGAGAACC S-FadE  Forward: GGAATTCTCTAGAATGATGATTTTGAGTATTATCG Reverse: GACTAGTTATGCGGCTTCGACTTTACGC
Design of the Promoter PfadR Repressed by Fatty Acids
Promoter PfadR, is derived from BBa_J23110. Specifically, FadR binding site of FadL gene is placed overlapping with the last   3 base pairs of BBa_J23110 The sequence was synthesized with restriction sites for EcoRI and XbaI at the 5' terminal and SpeI at   3' terminal. We use overlapping PCR to get the double strand DNA. The sequence design of PfadR is as followed: Forward: GGAATTCTCTAGATTTACGGCTAGCTCAGTCCTAGGTACAATGCTAGCTGGTCCGACCT Reverse： GACTAGTTCTTAGAAATCAGACCAGTGGCGAGAGTATAGGTCGGACCAGCTAGCATTGT
Construction of Biobricks
Fatty acid degradation project is divided into two parts: Promoter, and gene function To discover the optimal combination of those fatty acid genes, we: 1. PCR to clone those genes in E.coli K12 and Salmonella enterica LT2 2. Restriction digest and ligate those gene into pSB1C3 3. Restriction digest and ligate those gene with RBS(B0030) 4. RBS-FadA, RBS-FadI, and RBS-S- FadA is ligated with both BBa_R0011 promoter and our PfadR RBS-FadR, RBS-FadB, RBS-FadJ, RBS-FadE, RBS-FadD, RBS-FadL, RBS-S-FadB, and RBS-S-FadE are ligated with B0034 5.PROMOTER-RBS-FadA is ligated with RBS-FadB-Terminator, PROMOTER-RBS-FadI is ligated with RBS-FadJ-Terminator and   PROMOTER-RBS-S-FadA is ligated with RBS-S-FadB-Terminator. RBS-FadE-Terminator, RBS-FadD-Terminator, RBS-FadL-Terminator,   and RBS-S-FadE-Terminator, are ligated with BBa_R0011 promoter, PfadR and various constitutive promoters.   For Promoter PfadR 1. promoter PfadR was synthesized using overlapping PCR 2. RFP reporter was ligated downstream the promoter and ligted into pSB6A1 3. J23116+ RBS+ FadR+ Terminator was ligated to PfadR+ RFP in pSB6A1
Experimental Procedure
We used cupric acetate-pyridine as a color developing reagent to determine fatty acid consumption of genetically modified bacteria. We had modified existing methods to extract free fatty acid in M9 medium. Also, we used IPTG induced promoter BBa_R0011 to see the expression of those proteins and extract proteins from cells. For more details, please see Protocol page.
Result
Characterization of each gene
In this experiment, we wanted to test whether the ability of degrading fatty acids of our genetically modified bacteria was enhanced as expected by transforming plasmids constitutively expressing related genes in the beta oxidation pathway. The effects of the genes we have tested is listed in the following chart I. The ability was reflected by the change of the concentration of the fatty acids in the medium. It was measured by cupric-acetate soap reaction as described Protocols section.(此处可给一个链接) Each time we inoculated 50mg bacteria into 30ml M9 medium using fatty acid as sole carbon source, collecting the sample at the time as shown in the picture. Then the analysis of the free fatty acids was performed.
图 As our data suggests, PfadR FadL and J23114 FadL can degrade the fatty acids better than the control. They are also faster than the control, bacteria express protein irrelevant to fatty acid degradaton of similar size.Others are not that obvious. The column figure is present as below. The result of the analysis of the variance is present in chart II.
Retrieved from " http://2012.igem.org/Team:WHU-China/Project/fatty_acid_degradation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
