"
Page
Discussion
View source
History
teams
Log in
 
Team:WHU-China/Notebooks
From 2012.igem.org
Achievements
Three devices
-> 66 new biobricks parts submitted and 39 parts proved to be effective.  Please visit here for more information.
-> In vivo and in vitro experiments on fatty acid degradation. Please visit here for more information.
-> Cloning and testing the function of cellulose synthetase genes.  Please visit here for more information.
-> Using AdrA to activate cellulose synthetase and facilitate colonization .
-> Modeling on gut microbiota and FadR regulation .
Promoter design
-> Novel promoter design: Pcar ( BBa_K861171 )
-> Novel promoter design: PfadR( BBa_K861060 )
-> Setting up a standard of promoter design
Methods we improved
-> Expanding the cupric soap method to quantify fatty acid in medium.  Please visit here for more information.
-> A new method to measure the cellulose production.  Please visit here for more information.
-> An open software Fancy to measure the fluorescence of single bacterium
The biobricks we improved
-> PfadR ( BBa_K861060 ), an improved design of BBa_J54222
-> Improved function of two promoters BBa_K118011 and BBa_R0051
Human practice
-> Help HUST iGEM team to debug its project and modeling.
-> Human practice on survey of GMOs and publicizing iGEM to the public.
Favourite Parts
 
 
Name
Type
Description
Designer
Length
1
W
BBa_K861171
Regulatory
Pcar, synthetic promoter repressed by CRP
Kuanwei Sheng
36
2
W
BBa_K861070
Coding
AdrA gene from E.coli K12
Kuanwei Sheng
1167
3
W
BBa_K861060
Regulatory
PfadR, synthetic promoter with tandem FadR binding site
Kuanwei Sheng
76
Part Related to Promoter Design
 
 
Name
Type
Description
Designer
Length
1
W
BBa_K861160
Coding
CRP, cAMP receptor protein
Kuanwei Sheng
633
2
W
BBa_K861161
Generator
CRP with a RBS and a terminator
Kuanwei Sheng
757
3
-
BBa_K861162
Generator
Constitutively expresses Crp
Xian Xia, Kuanwei Sheng
800
4
-
BBa_K861163
Generator
Constitutively expresses Crp
Xian Xia
800
5
W
BBa_K861169
Regulatory
Indirect regulatory device, activated at high glucose concentration
Kuanwei Sheng
1961
6
-
BBa_K861170
Regulatory
PI ,Glucose-activated promoter
Kuanwei Sheng
36
7
W
BBa_K861172
Generator
lambda cI generator controlled by PcstA (glucose-repressible promoter)
Xian Xia
1069
8
W
BBa_K861173
Reporter
mRFP generator controlled by  PcstA (glucose-repressible promoter)
Kuanwei Sheng, Xian Xia
500
9
W
BBa_K861174
Regulatory
A control device for BBa_K861169
Xian Xia
1899
10
-
BBa_K861175
Reporter
Efficacy testing RFP generator of BBa_K861170 (PI)
Kuanwei Shneg, Xian Xia
905
11
W
BBa_K861176
Reporter
mRFP expressed after Pcar
Kuanwei Sheng, Xian Xia
905
12
-
BBa_K861177
Generator
K861162+ K861175
Kuanwei Sheng
1713
13
-
BBa_K861178
Device
Pcar efficacy testing device with CRP overexpressed
Kuanwei Sheng, Xian Xia
1713
14
-
BBa_K861179
Generator
K861162+J23119
Chang Liu
843
15
W
BBa_K861061
Reporter
Efficacy testing RFP generator of BBa_K861060 (PfadR)
Kuanwei Sheng
945
16
W
BBa_K861062
Regulatory
PfadR with FadR overexpressed
Kuanwei Sheng
1840
Parts Related to Fatty Acid Degradtion
 
 
Name
Type
Description
Designer
Length
1
W
BBa_K861500
Coding
FadL, Long-chain fatty acids transporter from E.coli str. K12
Kuanwei Sheng
1465
2
W
BBa_K861001
Generator
FadL with a strong RBS and a double terminator
Kuanwei Sheng
1465
3
W
BBa_K861002
Generator
Constitutively expressed FadL
Kuanwei Sheng
1508
4
W
BBa_K861003
Generator
PfadR regulated FadL
Kuanwei Sheng
1549
5
W
BBa_K861003
Generator
PfadR regulated FadL
Kuanwei Sheng
1549
6
W
BBa_K861010
Coding
FadD, an inner membrane-associated acyl-CoA synthase
Kuanwei Sheng
1605
7
W
BBa_K861011
Generator
FadD with RBS and terminator
Kuanwei Sheng
1729
8
W
BBa_K861013
Generator
IPTG induced FadD
Kuanwei Sheng
1792
9
-
BBa_K861014
Generator
Constitutively expressed FadD
Kuanwei Sheng
1772
10
-
BBa_K861015
Generator
Constitutively expressed FadD
Kuanwei Sheng
1772
11
-
BBa_K861016
Generator
PfadR regulated FadD
Kuanwei Sheng
1813
12
W
BBa_K861020
Coding
FadE ,acyl-CoA dehydrogenase
Kuanwei Sheng
2445
13
-
BBa_K861021
Coding
FadE gene for fatty acid degradation from Salmonella enterica LT2
Kuanwei Sheng
2592
14
W
BBa_K861022
Generator
FadE with RBS and terminator
Kuanwei Sheng
2569
15
-
BBa_K861023
Generator
S-FadE with a RBS and a terminator
Kuanwei Sheng
2716
16
W
BBa_K861024
Generator
IPTG induced FadE
Kuanwei Sheng
2632
17
-
BBa_K861025
Generator
Constitutively expressed FadE
Kuanwei Sheng
2612
18
-
BBa_K861026
Generator
Constitutively expressed FadE
Kuanwei Sheng
2612
19
-
BBa_K861027
Generator
Constitutively expressed FadE
Kuanwei Sheng
2612
20
-
BBa_K861028
Generator
PfadR regulated FadE
Kuanwei Sheng
2653
21
-
BBa_K861029
Generator
PfadR regulated S-FadE
Kuanwei Sheng
2800
22
-
BBa_K861030
Coding
FadB, gene for fatty acid degradation from E.coli K12
Kuanwei Sheng
2190
23
W
BBa_K861031
Coding
FadJ, gene for fatty acid degradation from E.coli str. K12
Kuanwei Sheng
2145
24
W
BBa_K861032
Coding
S-FadB gene for fatty acid degradation from Salmonella enterica LT2
Kuanwei Sheng
2190
25
W
BBa_K861033
Generator
FadB with a RBS and a terminator
Kuanwei Sheng
2314
26
W
BBa_K861034
Generator
FadJ gene with a RBS and a terminator
Kuanwei Sheng
2269
27
-
BBa_K861035
Generator
S-FadB with a RBS and a terminator
Kuanwei Sheng
2314
28
-
BBa_K861036
Generator
IPTG induced FadA and FadB
Kuanwei Sheng
3570
29
W
BBa_K861037
Generator
IPTG induced FadI and FadJ
Kuanwei Sheng
3672
30
W
BBa_K861038
Generator
IPTG induced S-FadA and S-FadB
Kuanwei Sheng
3673
30
W
BBa_K861038
Generator
IPTG induced S-FadA and S-FadB
Kuanwei Sheng
3673
30
-
BBa_K861400
Coding
FadA protein coding sequence for fatty acid degradation
Kuanwei Sheng
1164
31
W
BBa_K861041
Coding
FadI, gene for fatty acid degradation from E.coli str. K12
Kuanwei Sheng
1311
32
W
BBa_K861042
Coding
FadA, gene for fatty acid degradation from Salmonella enterica LT2
Kuanwei Sheng
1164
33
-
BBa_K861043
Translational_Unit
FadA with a RBS
Kuanwei Sheng
1185
34
W
BBa_K861044
Translational_Unit
FadI with a RBS
Kuanwei Sheng
1332
35
W
BBa_K861045
Generator
FadI with a RBS
Kuanwei Sheng
1288
36
-
BBa_K861046
Generator
PfadR regulated FadA and FadB
Kuanwei Sheng
3590
37
-
BBa_K861047
Generator
PfadR regulated FadI and FadJ
Kuanwei Sheng
3693
38
-
BBa_K861048
Generator
PfadR regulated S-FadA and S-FadB
Kuanwei Sheng
3591
39
-
BBa_K861050
Coding
FadR, fatty acid sensor from E.coli str. K12
Kuanwei Sheng
720
40
-
BBa_K861051
Generator
FadR with a RBS and a terminator
Kuanwei Sheng
844
41
-
BBa_K861052
Generator
Constitutive expressed FadR
Kuanwei Sheng
887
42
-
BBa_K861345
Generator
IPTG induced S-fadE
Kuanwei Sheng
2779
43
-
BBa_K861346
Intermediate
IPTG induced promoter with FadA, intermediate part for BBa_K861036
Kuanwei Sheng
1248
44
W
BBa_K861347
Intermediate
IPTG induced promoter with FadI, intermediate part for BBa_K861037
Kuanwei Sheng
1395
45
W
BBa_K861348
Intermediate
IPTG induced promoter with S-FadA, intermediate part for BBa_K861038
Kuanwei Sheng
1351
46
-
BBa_K861349
Intermediate
PfadR with FadA, intermediate part for BBa_K861046
Kuanwei Sheng
1269
47
-
BBa_K861350
Composite
PfadR with FadI, intermediate part for BBa_K861047
Kuanwei Sheng
1416
48
-
BBa_K861351
Composite
PfadR with S-FadA, intermediate part for BBa_K861048
Kuanwei Sheng
1372
Parts Related to Cellulose
 
 
Name
Type
Description
Designer
Length
1
W
BBa_K86150
Coding
BcsA,cellulose synthetase, catalytic subunit
Xian Xia
2619
2
W
BBa_K861101
Generator
BcsA with RBS and terminator
Xian Xia
2743
3
W
BBa_K861102
Generator
BcsA controlled by promoter repressed by CRP
Xian Xia
2787
4
-
BBa_K861110
Coding
BcsB,regulator of cellulose synthetase
Xian Xia
2340
5
-
BBa_K861111
Generator
BcsB with RBS and terminator
Xian Xia
2464
6
-
BBa_K861112
Generator
BcsB controlled by promoter repressed by CRP
Xian Xia
2508
7
-
BBa_K861120
Coding
BcsZ, endo-1,4-D-glucanase
Xian Xia
1107
8
-
BBa_K861121
Generator
BcsZ with RBS and terminator
Xian Xia
1231
9
-
BBa_K861122
Generator
BcsZ generator repressed by CRP (directly)
Xian Xia
1275
10
-
BBa_K861130
Coding
BcsC,cellulose synthetase subunit
Xian Xia
3474
11
-
BBa_K861131
Generator
BcsC with RBS and terminator
Xian Xia
3598
12
-
BBa_K861132
Generator
BcsC controlled by promoter repressed by CRP
Xian Xia
3642
13
W
BBa_K861140
Coding
GalU,glucose-1-phosphate uridylyltransferase
Xian Xia
909
14
W
BBa_K861141
Generator
GalU with a RBS and a terminator
Xian Xia
1033
15
W
BBa_K861142
Generator
GalU generator repressed by CRP (directly)
Xian Xia
1077
16
W
BBa_K861150
Coding
GalF, putative regulatory subunit for GalU
Xian Xia
894
17
W
BBa_K861151
Generator
GalF with a RBS and a terminator
Xian Xia
1018
18
W
BBa_K861152
Generator
GalF generator repressed by CRP (directly)
Xian Xia
1062
Parts Related to Colonization
 
 
Name
Type
Description
Designer
Length
1
W
BBa_K861071
Generator
AdrA gene with a RBS and a terminator
Kuanwei Sheng
1293
2
W
BBa_K861072
Generator
Constitutively expressed AdrA
Kuanwei Sheng
1336
3
W
BBa_K861073
Generator
Constitutively expressed AdrA
Kuanwei Sheng
1336
4
W
BBa_K861074
Generator
Pcar regulated AdrA
Kuanwei Sheng
1337
5
-
BBa_K861090
Coding
YhjH Gene From E.coli str. K12
Kuanwei Sheng
768
6
W
BBa_K861091
Generator
YhjH gene with a RBS and a Terminator
Kuanwei Sheng
892
Brief Answers to Safety Questions
Welcome to our Safety Page. We will first answer the safety questions asked by iGEM headquarters briefly, and then discuss safety issues associated with our project in detail. In addition, we will provide our ideas and practice on guaranteeing and developing biosafety.
Q1. Would any of your project ideas raise safety issues in terms of: Researcher safety, public safety, or environmental safety? No. Our design is based on the commonly used nonpathogenic E. coli K.12 strain and genes we manipulated are original genes in E. coli. The protein products, at least from current understanding, will cause no harm to researchers, the public and environment. In addition, strict lab practice is executed to further ensure safety.
Q2. Do any of the new BioBrick parts (or devices) that you made this year raise any safety issues? If yes, Did you document these issues in the Registry? How did you manage to handle the safety issue? How could other teams learn from your experience? Yes, we will discuss this question in latter part of this page (Safety Considerations of Our Biobrick parts and Our Project).
Q3. Is there a local biosafety group, committee, or review board at your institution? If yes, what does your local biosafety group think about your project? If no, which specific biosafety rules or guidelines do you have to consider in your country? Yes. All materials obtained have received the approvals from the department's laboratory management committees. We are also obliged to observe the regulations of Teaching Centre of Experimental Biology and apply for approval for materials before we start our project.
Q4. Do you have any other ideas how to deal with safety issues that could be useful for future iGEM competitions? How could parts, devices and systems be made even safer through biosafety engineering? Some classified measures should be taken according to the safety of the material. For example, plasmids that may harm the safety, when submitted, should receive more attention and have a stricter package. Some harmful byproducts during experiments should be eliminated properly. We have done our human practice aiming at understanding attitude of the public towards genetically modified baceria and publicizing biosafety ideas to the public, which we think should be popularized to other teams in future iGEM competitions.
General Safety Issues
In this part, we will illustrate organisms, reagents and equipments we use that may cause safety problems, and introduce our operation standard, management and trains protecting the team members, public and environment.
As above, all the organisms and DNA hosts are not of high individual or community risk. Until now, the only used organism is E. coli K.12 (and some of its varieties, for example, DH5α, a RecA mutated strain). Our current lab of basic-biosafety level 1 is safe enough to manipulate this strain. Only the genome (a generous present from University of Dundee iGEM team), but not the living Salmonella enterica was manipulated in the lab, and the genes from it are homogeneous of E. coli K.12, not associated with pathogenesis. We will not implement our future experimental plan with microorganisms of risk group 2 or vertebrates in our current lab, for too low is the biosafety level and no animal facilities.
To manipulate microorganisms, an ultra clean cabinet is used and strict aseptic technique is followed. All experimenters have been trained on foundation microbiology technique and biosafety. All microorganism contacting vessels are sterilized before and after experiments in appropriate protocols. Also, all microorganism materials will be sterilized before discarded. In this way, we believe that no public or environmental harm will be caused by the experimental organisms. In addition, no one in our team will be hurt by the experimental organisms.
The genetic modifications we make will change metabolism of bacteria. However, there is no evidence both theoretically and experimentally that these modifications will improve the pathogenicity of the bacteria or cause damage to environment even taking the risk of horizontal gene transfer into consideration. The effects of expressing gene adrA regarding infectivity and pathogenicity on E. coli is not clear now, but it still can be easily controlled in the lab environment without human ingestion. Also, we will insert a death device into the bacteria cell when we construct the whole system (still far from now) to avoid the proliferation out of control. For more information about gene safety, please read the next section and documents of our relevant parts on registry.
We have considered the potential harmful chemicals and equipments, as listed below:
Basic molecular experiments: NaOH, HCl, SDS, acrylamide, TEMED, ethanol, IPTG, liquid nitrogen, β-mercaptoethanol, xylene cyanol FF.
Bacteria culture: ampicillin, kanamycin, chloramphenicol.
Chemical analysis and measurements: acetone, cupric acetate, Sudan III, Congo red, Coomassie Brilliant Blue G-250, Coomassie Brilliant Blue R-250.
Equipments: UV lamp, supercentrifuge, heating equipments (alcohol burner, PCR amplifier, water bath, dry bath), electrophoresis apparatus, -80℃ refrigerator, ultrasonic cell disruptor.
All of these are regular reagents and apparatus in a molecular biology laboratory. The risks of them come from inflammability, explosibility, irritation, corrosivity, toxicity, carcinogenicity and physical injury. But none of them raises special safety issue, with chemical hood, emergency shower, normal personal protection, safety management and safety training. Only trace amount of antibiotics are used. Inactivated will they be before discared.
Emergency Shower
Safety Considerations of Our Biobrick parts and Our Project
In this section, we are going to answer safety question 2 in detail, taking the potential risk in the future into concern.
The main safety challenge we must face is that as a practical bacteria therapy our direction is, we shall demonstrate that the “E. coslim” will not harm its host when it is developed completely. As few experiments we can do in such limited time, we have done a series of theoretical work to solve problems in this field.               Intestine-colonized pathogenic E. coli may cause immune responses, following by diarrhea, inflammation and fever (although the strain we use is considered non-pathogen). Thus we plan to transplant the whole synthetic system to another organism (for example, Bacillus subtilis, has proved safe in human intestine). We know that it is hard as the two organisms are very different on transcriptional mechanisms, but we believe the work of establishing model system (that is what we are doing) in E. coli will make it easier. Also, as the rapid development of synthetic biology and gut microbiology, we hope in the near future, we can modify the genome of E. coli to change it a safe intestine microbe.
Another risk is that when the engineered bacteria get higher efficiency on energy production, they could proliferate out of control. To forestall this situation, we design a “death” device (for more information, click on Death ) using D-xylose as inducer. It means if you want to stop weight losing, what you have to do is to eat some D-xylose, and the “E. coslim” will die, shed from the intestinal wall and be poured out. What is more, designed as a two-plasmid system, it prevents all possible horizontal gene transfers.
Further safety issues will be raised and discussed in future experiments, including those operates in intestinal model and animals (For more information, click on Future Perspectives).
Safety Management and Practice
Our project has past a review of an expert committee, of which members are professors or associate professors of microbiology, genetics and bioengineering. Safety issues are considered seriously before they approved our design.                We have consulted a few experts for safety questions about both artificial intestinal bacteria and experiments. Their advices help us improve the safety of the whole planning system. For that, we thank Dr. Yulan Wang and Dr. Tiangang Liu so much.               Our lab is supervised by Teaching Centre of Experimental Biology, Wuhan University (TCEB, whose leader is our instructor Dr. Zhixiong Xie ). An expert group of this centre formulates safety guidelines and guarantees their performance in all teaching labs. All experiments we do past its review. Our safety management system includes the followings: Management of Chemicals, Equipments and Experimenters: Chemicals and equipments are registered in TCEB before they are available for us. Equipments are checked and maintained regular. And an entrance guard system is used to ensure only the admitted could enter the lab.
Responsibility distribution: All members worked in the lab are divided to three groups. The group leaders arrange schedules of experiments after assessing safety issues in the group (for example, the safety train of the experimental executer). Every member records his/her  experiments in detail in the notebook of the group. Each group takes responses for cleaning and checking risks in the lab in turns. The group leader takes responses to the team leader. The team leader reports regular on safety to engineer Miss Long Yan, who is authorized by TCEB and manages the lab.
Management of Chemicals, Equipments and Experimenters: Chemicals and equipments are registered in TCEB before they are available for us. Equipments are checked and maintained regular. And an entrance guard system is used to ensure only the admitted could enter the lab.
Training: Our team members have been trained after Guidance of Student Experiments formulated by TCEB.
Guidence of Student Experiments
Biosafety and Publicity
We believe that biosafety is not an issue which should be only considered inside biological lab, but also around people and communities. Spreading knowledge of biosafety to the public will help eliminate misunderstanding and prejudice, from which the science and all the people will benefit. We hope that sharing this idea with all iGEM teams can be useful to take advantage on biosafety.
To investigate public attitude towards our project and to publicize our biosafety ideas are what we do for our human practice. As “eating bacteria” is somehow an unacceptable concept now, we would like to find whether people know the truth about biosafety issues raised by bioengineered bacteria, or they are just panicked by their imaginary “bacteria”. And then, after initiating basic knowledge of microbiology, gene engineering and synthetic biology, we shall take a look on if people’s opinion to biosafety issues will change. In this way, we will estimate the value of our scientific publicity.
The rational discussions of biosafety we take with the public do not limited on our project, but also hotspot issues such as transgenics and stem cell therapies. For more information, please click on Human Practice .
Introducing a Simplified Intestinal Model
Protocol for Fluorescence Measurement
Minimal Medium
M9:
For 1L Medium add
Na2HPO4·12H2O                  15g
KH2PO4 3g
NaCl                              1g
NH4Cl                            0.5g
After autoclaving, add:
MgSO4(1mol/L)                    2mL
Bacteria strain:CaCl2 100μL
Triton X-100 (as emulsifier)           2uL
Note
1.For M9 medium using oleic acid as sole carbon source, various amount of oleic acid was first emulsified 1:1 with 10% Triton X-100. M9 medium was then slowly added with constant vortex. M9 medium with high concentration of oleic acid was diluted by M9 medium with triton to form various concentrations；
2. For M9 medium using glucose as sole carbon source, M9 medium with high concentration of glucose was diluted by M9 medium to form various concentrations.
Step
1. Seed liquor which was activated over night was inoculated into M9 medium which contains different concentration of oleic acid. And it was then incubated at 37℃ for 24 hours;
2. After 24h of incubation in 24 well plates in 37℃, bacteria culture was centrifuged at 3000rmp for 5min, washed and resuspended in PBS;
3. We detected the OD600 and fluorescence of using SpectraMax M2 plate reader (Molecular Devices) .Excitation at 584 nm and emission at 607 nm were used. All fluorescence was normalized with cell density by measuring the absorbance at 600 nm.
Protocols of Cupric-Soap Reaction
To test metabolism of long fatty acids, we used oleic acid as sole carbon source in mediums, and used cupric-soap reaction to determinate oleic acid concentration preliminarily.
Modified M9 minimal medium with emulsified oleic acid as sole carbon
Minimal medium was the same with that in Materials and methods for PfadR
What's Important:
Slowly pour M9 minimal medium into mixture of oleic acid and Triton X-100 to get more homogeneous solution.
Analysis the concentration of the oleic acid in the medium by cupric-acetate method
Step
1.Collect 5 ml medium which has been used to cultivate bacteria. Then centrifuge the medium at 3000rpm for 10 min to separate bacteria and medium;
2.Decant 3ml supernatant liquid into a 10ml EP. Add 3ml acetone to the liquid, 1ml at a time, shaking 10-20 times before adding another 1 ml in order to avoid the effect of the ions of the liquid during the extraction progress;
3.Add 3 ml isooctane once ,shaking for at least 90s, stand for 2min or longer until layering completely;
4.Collect 3 ml clear isooctane in a 5 ml EP, Add 800?l cupric-acetate (5% m/v, adjust pH to 6.8 with pyridine), Shaking for 90 seconds, stand for 2min or longer until layering completely;
5.Detect OD of organic phase in a spectrometry at 715nm.
The Standard Curve
We add 1.6ml, 3.2ml, 4.8ml, 6.4ml, 9.6ml, 11.2ml oleic acid into 3ml M9 medium to generate a gradient. The absorbency is measured as described in the protocol.
We slightly modified the methods provided by Kwon and Rhee, 1986, adding the extraction step, for it is indispensable when bacteria are in the medium. The standard curve is still very close to that of the paper's, which proves the applicability and accuracy of our modified protocol.
Krebs-Ringer Phosphate（KRPD Buffer Solution）
Solutions
Solution A：
For 100 mL:
NaCl  7.5985g ; KCl  0.3727g ; CaCl2 0.1054g; glucose-H2O  0.9495g,
Solution A：
For 100 mL:
NaH2PO4·2H2O 0 .2964g ;Na2HPO4·12H2O  2.9011g，
After dissolving, add MgSO4·7H2O  0.3130g
Mix A solution and B solution, add 790ml dd H2O, then regulate pH to 7.2~7.4 using 1M NaOH/1M HCl, finally add dd H2O to a final total volume of 1L.
Procedures
1. cultivate cells in 1 liter of medium to late exponential phase;
2. harvest by centrifugation at 35℃;
3. wash twice with 2 liters of warm(35℃)0.05M potassium phosphate buffer (pH 7.4) containing 1%(v/v) Triton X-100;
4. resuspend in 0.05M potassium phosphate buffer (pH 7.4) containing mercaptoethanol;
5. add sufficient volume of buffer to give a concentration of about 50mg(dry weight) of cells/ml;
6. disrupt cells with a Bransor Sonifier for 1min. The treatment was applied for 15s intervals, under 4 ℃;
7. centrifuge at10,000×g for 30min;
8. decant supernatant liquid for enzymatic assay, the protein concentration was determined by the biuret method;
9.analyze fatty acid oxidation in cell-free extracts
Note: little oxidation was observed when less than 1mg.
Reaction Mixture
1.0ml of freshly oxygenated Krebs-Ringer phosphate(pH 7.4)
Palmitate-1-14C, 20nmole (80,000 counts/min)
CoA, 1 μ mole
NAD, 1μ mole
ATP, 1μ mole
Succinate, 10 n moles
Supernatant protein, 1~5 mg
dd H2O to a final total volume of 2.0ml
Cellulose Detection
1. Inoculate the bacteria into liquid LB medium and then incubate it for 24 hours at 37℃;
2. After centrifugation, supernatant is preserved for use, while deposits are resuspended with PBS and adjust to OD600 1.0;
3. Exceed cellulase was used to digest cellulose in supernatant and deposits. After the cellulase is added, 1mL solution was sampled every 3 min and cellulose of the sample was inactivated immediately;
4. Ten minutes later, the reduce sugar is detected in all the samples with or without cellulose;
5. 2mL DNS solution is mixed with the sample, and incubated in boiling water for two minutes, then it is cooled rapidly;
6. After 9 mL ddH2O being added into the solution, record absorbance at 540nm;
7.The Standard Curve of glucose concentration
Then all tubes was treated in the same way with that in step 5 and 6.
8.Calculate the amount of cellulose in cell culture.
In vitro Experiment
1. inoculate 100μl bacteria into 100ml LB , shaking at  37C overnight .
2. inoculate 1L LB using the culture above at the next morning , on the scale of 1:20 .
3. Separate above medium into 250ml flask , shaking at 37C for 3-4h .
4. 8000 rpm , 6min to collect the bacteria .
5. Using PBS to resuspend the bacteria to 50mg / ml .
6. Ultrasonication 1min , every 15s having an interval to cool on the ice .
7. Using Coomassie blue staining to measure the concentration of the total protein .
8. In 2ml reaction system , add 5ul oleate ,40 ul Triton 10% , 1ml Krebs-Ringer phosphate buffer , 10nmol succinate , 1 umol COA , 1umol ATP , 1 umol NAD .
9. react at 37C for 6h
In this experiment , the concentration of total protein in the cell extracts of the five sample are almost the same , so we assume that adding the same volume guarantee the same amount of protein is added .
The reaction system for the control BBa_R0011+(R0011), BBa_R0011+fadE , BBa_R0011 + fadD , BBa_R0011 + Samonella A Samonella B , BBa_R0011 + fadI fadJ : 1ml Krebs-Ringer phosphate buffer , 10nmol succinate , 1 umol COA , 1umol ATP , 1 umol NAD , 1ml cell extract .
The reaction system for BBa_R0011+fadD and BBa_R0011+fadE mixture : 2ml Krebs-Ringer phosphate buffer , 20 nmol succinate , 2 umol COA , 2 umol ATP , 2 umol NAD , 1ml cell extract for each gene.

The reaction system for BBa_R0011+fadD , BBa_R0011+fadE , and BBa_R0011 + Samonella fadA Samonella fadB mixture : 3ml Krebs-Ringer phosphate buffer , 30nmol succinate , 3umol COA , 3umol ATP , 3 umol NAD , 1ml cell extract for each gene.
The reaction system for BBa_R0011+fadD , BBa_R0011+fadE , BBa_R0011+Samonella A Samonella B , and BBa_R0011+fad I fad J mixture :4ml Krebs-Ringer phosphate buffer , 40nmol succinate , 4umol COA , 4umol ATP , 4 umol NAD , 1ml cell extract for each gene
Plate Assay
1. J12107- AdrA and control RBS was incubate and vortex in LB medium with Ampicillin overnight
2. OD600 was measured; LB was then added to make the two test tubes had the same OD600
3. Bacteria were inoculated by sterilized needle piercing a pre-labeled 0.3% agar LB semisolid plate for 4-5 minutes in super clean bench.
4. Carefully placed the plate horizontally in a 37 degree incubator overnight. Avoid shaking of the plate.
5. Clones on the plate were observed.
Materials and Methods of SDS-PAGE
Coomassie Blue Stain
-Coomassie brilliant blue G-250 50mg
-95% ethanol                25ml
-85%H3PO4 50ml
-H2O, adjust to 500ml
-filter
Destain solution
-methanol        250ml
-acetic acid         50ml
-H2O     adjust to 500ml
2x SDS loading buffer
-0.5mol/l  Tirs-HCl(PH6.8) 25ml
-10%SDS   8ml
-50%glycerol   20ml
-2-mercaptoethanol  2ml
-1%Bromphenol Blue  4ml
-H2O        adjust to 100ml
10xSDS-PAGE running buffer
Tris base,               30.3 g
M glycine                144.1g
SDS                      10 g
-H2O                 adjust to 1L
Steps
Protein expression
1.inoculate the liquid strains into LB medium supplement with 50μg/mL ampicillin, incubate the medium at at 37℃ until OD600 reaches 0.6;
2. Separate the culture into two test tubes. Add IPTG into one of two tubes at a final concentration of 1mM to induce the expression of
3. 1 mL of the culture is sampled every hour. After centrifugation, deposits was suspended with 300 μL PBS and 200 μL 2x SDS loading buffer, then incubated in boiling water for 15 min.
4. 10μl of samples was load in lanes, run the SDS-PAGE
5. Stained the SDS-PAGE 5 hours
6. Destain the SDS-PAGE until you can see the protein band.
team history
Dec. 2011
WHU iGEM team was established
Dec. 2011 to Feb. 2012
Every one presented their own idea, then we discussed the feasibility of these ideas.
Feb. 2012
The final project was determined, named E.coslim
Mar. 2012
We finished an outstanding presentation. Our reply was approved by the leaders of college of life sciences, Wuhan University.
Apr. 2012
Our iGEM team was divided into 3 groups—group of Sheng, group of Xia and group of Mei.
Apr. 2012 to prsent
Experiments started….
Apr. 2012
Group of Sheng:  FADR was connected to pSB1A2 plasmid carrier
FADR was connected to RFP gene
Group of Xia: starting to connect genes of CI control system
Testing the function of CI control system
Group of Mei: YhjH/ FadE/ FadD/ FadL gene was connected to pSB1A2 plasmid carrier
May. 2012
Group of Sheng: FadB/ FadJ gene was connected to pSB1A2 plasmid carrier, then BBa_B0030(RBS) as well
Group of Xia:  They devised two promoters p110 and p101, which were expected to be controlled by glucose. However, they failed.
Group of Mei:  YhjH/ FadE/ FadD/ FadL gene was connected to BBa_B0030(RBS)
June 2012
G of S: FadR was connected to low-copied plasmid carrier.
sFadB and sFadA genes were connected to BBa_B0030 (RBS)
sFadE gene was connected to pSB1A2 plasmid carrier
G of X: the connection of genes, which were relative with cellulose, was finished.
Another two promoter were devised, p1 and p2
G of M: YhjH/ FadE/ FadD/ FadL gene was connected to BBa_B0024 (terminator)
July 2012
G of S: FadJ was connected to BBa_B0024 (terminator), sFadE gene was copied by PCR technology
On the front half of this month, they conducted several pre-experiments of oleic acid test
On the next half month, they started normal experiments of oleic acid test
G of X: started to test the function of p1 and p2
G of M: YhjH/ FadE gene was connected to BBa_J23100 (promoter)
Finish the standard curve of oleic acid test
Aug. 2012
G of S: sFadE was induced to point mutation, then it was connected to BBa_B0030 (RBS) and BBa_B0024 (terminator) respectively
sFadB was connected to BBa_B0030 (RBS) and BBa_B0024 (terminator) respectively.
G of X: test the function of cellulose-controlled genes, having got satisfying results
G of M: copied Adra gene and finished the connection of BBa_B0030 (RBS), BBa_B0024 (terminator) and BBa_J23107 (promoter), BBa_J23114 (promoter) respectively. FadE/YhjH/FadD/fadL were connected to BBa_J23107 (promoter), BBa_J23114 (promoter).
Sep 2012
G of S: Transferred all the parts in pSB1A2 into pSB1C3, Tested the function of PfadR, Characterized the effect of each gene on fatty acid consumption, started to set up the platform for in vitro experiments.
G of X: Transferred all the parts in pSB1A2 into pSB1C3, tested the function of cellulose system. Repeat the test of the function of cellulose-controlled genes.
G of M: Transferred all the parts in pSB1A2 into pSB1C3, test the function of Adra/YhjH.
Brainstorming
About E.coslim by Kuanwei Sheng
In order to help people lose weight, besides our three devices, we also have come up with many other creative ways.
Ⅰ.Short chain peptides synthesis
Recent researches have indicated that some short chain peptides in intestine have effect on inhibiting appetites, therefore decrease the food intake. We thus formed the idea that we could synthesis a DNA chain that encodes those short peptides.
Ⅱ.Biosynthesis of L-carnitine
L-carnitine is a molecule that facilitates the progress of transporting fatty acids into mitochondria where these fatty acids will be disintegrated. We once considered use L-carnitine to help the host metabolize fatty acid better. However, the biosynthesis of L-carnitine has too many derivatives or the pathway is patented by others.
Ⅲ.Xylose isomerase
Xylose is a prebiotics that can hardly be absorbed by human. We consulted many papers and find that glucose can be converted into xylose by xylose isomerase. Therefore, we thought maybe we could lower the glucose available in intestine by expressing xylose isomerase. However, this process is shown to be reversible latter. Mutated the sequence of the protein may generate high converting rate, yet it is too laborious and risky for a short time project.
Other novel ideas
Tackle Water bloom by Tong Wang and Kuanwei Sheng
Since the detrimental effects caused by cyanobacteria to the environment such as making water carcinogenic have become a serve global problem, we therefore tried to employ E.coli as an expression system to eliminate these detrimental effects. When we first took over this project, we thought about limiting the growth of cyanobacteria. Along with the process we got to read a large amount of relevant papers about cyanobacteria, we found that not only the main detrimental effects caused by cyanobacteria is attributed to its product which is a cyclic peptide called microcystin, but also microcystin can regulate the population density. Then we came to realize the importance of microcystin and began to search information about it. Through over this process, we discovered a gene cluster which is responsible for the microcystin degradation pathway. It encodes four enzymes——MlrA、MlrB、MlrC and MlrD. Also, we found that some non-toxic cyclic peptides produced by cyanobacteria such as Anabaenopeptin B and Anabaenopeptin F can induce lysis of cyanobacteria. The latter finding can be utilized as an effective cell population density control mechanism. Thus we thought about constructing two independent systems to eliminate cyanobacteria, one about inducing lysis of cyanobacteria and the other about degrading microcystin. Finally we gave up this project because of the reasons that these gene clusters are too large to be expressed in E.coli and that E.coli cannot survive in the sea, however, we still feel proud of these fancy ideas.
Desalination of sea water by Kuanwei Sheng
We came up with the thought that engineering bacteria can intake ions like Na+, cl-, Mg2+ and etc. under special stimulus. Also, under another certain stimulus, the bacteria can export those salts out of cells for reuse. Therefore, we can use these bacteria to desalinize sea water and extract the salt the same time. However, there is no such ion channel that can meet our needs.
Sense the earthquake By Min Ye
We once tried to find proteins that can sense vibration and construct a pathway to report that vibration. However, we are not able to find the protein that can meet our needs.
Auto plasmid preps By Kuanwei Sheng
We thought about constructing a synthetic protein that combines zinc finger protein which can recognize the specific sequence of DNA and signal peptide that can make proteins be exported out of the bacteria by secretion pathway. Therefore, it is possible that plasmid can be exported out the cells together with the zinc finger protein.
Outer Membrane Vesicle (OMV) to treat cancer By Kuanwei Sheng
We thought to use Outer Membrane Vesicle (OMV) to treat cancer. Specifically, we thought about localizing antibody that can recognize certain cancer on the surface of OMVs via signal peptides. Also, we thought we may localize cell division inhibitor protein or protein that lead to cell death inside the OMVs. Therefore, we hoped that the OMV excreted by the bacteria can recognize and kill cancer cells.
Multicell Yeast By Wenxiong Zhou
Transform the yeast from a single-cell microbe to a multi-cell organism by setting bistability of gene expression among cells of yeasts adhered in amalgamation.
To kill superbacteria
Now with the spread usage of the antibiotics, many bacteria adopt ability to fight against the antibiotics. And now it’s a big problem that antibiotics are no longer useful as before. So to kill these bacteria, Jing and her group thinks they can device some proteins or artificial micromachine to detect DNA that code proteins contributing to the ability to degenerate antibiotics. And then by transferring these plasmids coding artificial proteins or micromachines, they can inhibit the expression of enzymes or proteins degenerating antibiotics.
Team
Presentation
Human Practice
Doodle
Funny Pictures
A Letter from Fancy
There is a very meaningful sentence in the novel The Lord of the Ring: when the respected old Billbo was going to leave the village in which he had lived for many years, he gave a lecture--”The people among you whom I recognize haven’t reached half whom I should do, besides, among those whom I love are less than the half I should do, either.
None of us need to fight in a war or fight with monsters. However, all of us have indeed fought and stuck to the very end, regardless of whatever obstacles to overcome and whatever precious to sacrifice.
The greatness lies in the persistence to do the ordinary things perfectly. We are all inspired by our design. However, the gap between the repetitive, seemingly endless molecular cloning and the last magic probiotic can easily wreck the passion. To the opposite, most of us have successfully endured the monotonous life. Sometimes the enzymes just don’t function well. Other times the PCR never get right for some unclear reasons. We just survived those really despair moments and made one step closer to our goal.
And those heart-touching scenes are clear as if they happened yesterday. Regardless of our persuation to wait for the rain to stop, our captain determined to fetch the induction cooker in his dormitory to perform an reaction as soon as possible (We don’t have an appropriate heater in our lab). When his figure disappeared in the dark and rain, words failed me; when I got a message at 3 o’ clock in the midnight, the excited tune claiming that we no longer needed to worry about our competent cells overwhelmed me with tears; when I heard that our “artist” having caught a cold but promised that he would complete all the pictures of the web site in time, I fell into silence again; every time when Xian Xia stayed overnight in the lab, the room would be quite tidy and all the reagents would got replenished. When I saw this in the next morning, I was greatly touched.
Most of us has sacrificed much time to pursue personal interest. We just decrease the time for required exams for going abroad, for working in our own labs, and for some courses we are really interested in, let alone our hobbies. No time for travelling, reading lasted paper on interested topics, shopping, chatting with friends and so on. However, after the complaints and anguish, we finally reached such a peaceful and simple spiritual state that we just want to try our best to get the best result.
A man who never experienced iGEM in WHU can never really understand the complex of feelings: glory and dream, perseverance and persistence, joy and tears. You can never image how brave and respected my teammates are, how many efforts and love they have put in the team, and how strong the relationship we have built since the team has been organised.
Retrieved from " http://2012.igem.org/Team:WHU-China/Notebooks "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
