<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WHU-China_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WHU-China/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WHU-China/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WHU-China/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WHU-China/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WHU-China/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WHU-China/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="center"><DIV class="middle"><DIV class="main"><DIV class="passage divcell0"><A name="0"><H3 align="center">Future Perspective</H3></A><P align="left">To date, we have built all our three devices and tested the function of many biobrick parts. However, our aim is far more than just making a toy in the laboratory. We are trying to create a product that has clinical applications or can be utilized other areas.   </P><P align="left">First, we will amalgamate the three devices, namely, the <I>Fatty Acid Degradation</I>, the <I>Cellulose Synthesis</I>, and the <I>Colonization</I>, into one whole system. Since there are tens of genes and regulation elements in all and their total length is more than 30kb, a larger vector like λ phage rather than a plasmid is likely to be adopted. It may also be a good choice to integrate the three devices into the chromosome of bacteria.   </P><P align="left">Second, the <I>Escherichia coli</I> may be a good model for molecule cloning operations, but not suit for pharmacy since there may be risk in infection and diseases. We propose <I>Lactobacillus</I> as a better model because not only its safety has been well demonstrated in food industry, but also the yogurt made from the genetically modified <I>Lactobacillus</I> will possess the property of making you slim!  </P><P align="left">The function of the whole system in <I>Escherichia coli</I> or <I>Lactobacillus</I> will be tested both <I>in vitro</I> and <I>in vivo</I> to confirm the effect of our project. The gut microbiota and E.coslim will be inoculated in a glass tube, through which plasma made from different kinds of food will flow. We will test the changes of the microbe community inoculated. Furthermore, a gut microbiota transplantation experiment on mice may also be conducted for further confirmation.   </P><P align="left">Our product will be finally packaged into two capsules. Capsule A is E.coslim, which is able to influent the human body's absorption to high energy contained nutrients, regulate the gut microbiota and make you slimmer day by day. And Capsule B is xylose, which can induce the <I>Death Device</I> in E.coslim and avoid people from malnutrition after taking in E.coslim for a long enough time.   </P><P align="left">Last but not least, we not only create a new microbe that can make people slim, but also provide tools to sense the fatty acid, the glucose and the xylose in other circumstances. So the new biobricks we submit may be applied in a variety of areas, such as the degradation of waste oil, urine sugar test for diabetes patients, safety control of genetic engineering, and so on. We are looking forward to the day when E.coslim is truly applied in people's life, in clinical and more widely in other areas beyond our imagination.   </P></DIV><DIV class="passage divcell1"><H3 align="center">Model I: Fatty Acid Degradation</H3><P align="left">The Fatty Acid Degradation Device may be the most complicated part in our project, along with its great importance. The antagonistic relationship between gene fadR and other genes related to β oxidation -- the fadL, fadD, etc, -- makes it regulatable to the concentration of fatty acid in the environment. Thus, it is necessary to explore the quantitative response corresponding to the concentration change of fatty acid. We build an ordinary differential equations-based mathematical model to describe the device and find a proper set of parameters under which the proportion of the steady expression level of fadL to fadR changes broadly from 0.2 to 3.5. The model mathematically demonstrates the effect of the Fatty Acid Degradation Device and also provides meaningful clues for the optimization of the device in experiments.  </P><A name="The Ordinary Differential Equations of the Model"><H3 align="center">The Ordinary Differential Equations of the Model</H3></A><P align="left">We conduct an evaluation by mathematical modeling and build the ordinary differential equations (ODE) as follows: </P><P>①</P><P> </P><P align="left">For simplicity, all genes with a promoter PfadR and equally regulated by FadR are deemed as a whole and represented as FadX, i.e., FadX refers to FadL, FadD, FadE, FadA, FadB, FadI, FadJ. And the Complex, or variable <I>x<SUB>7</SUB></I>, refers to the Fatty Acyl-CoA-FadR Complex.   </P><P align="left">Parameters in the ODEs:  </P><P align="left">
　　① <I>E</I> denotes the constitutive expression rate of FadR, and D the degradation rates of FadR, FadX and Complex, which is assumed equal.   </P><P align="justify">
　　② <I>a</I> denotes the affinity of FadR to the promoter PfadR, and <I>V</I> denotes the background expression rate of related genes.   </P><P align="justify">
　　③ <I>k<SUB>1</SUB></I> and <I>k<SUB>2</SUB></I> denote the forward and reverse reaction rate coefficients, respectively. <I>k<SUB>3</SUB></I> to <I>k<SUB>6</SUB></I> are parameters related to enzyme-catalyzed reactions based on the Michaelis-Menten Equation. Specially,  </P><P align="justify"> </P><P>②</P><P align="left">while <I>f</I> denotes the concentration of fatty acid outside the bacteria, <I>K<SUB>L</SUB></I> the Michaelis constant of FadL, and <I>k<SUB>L</SUB></I> the maximal activity of FadL. </P><P align="left">Details for the ODE can be illustrated in Fig 1.  </P><P align="justify"> </P><P><STRONG>Fig 1</STRONG> Illustration of the meaning of the ODE</P><A name="Analysis on the Steady State of the ODE"><H3>Analysis on the Steady State of the ODE</H3></A><P align="left">By setting the right side of the equations to zeros, we get algebra equations about the five variables at the steady state. And after elimination we obtain the cubit equation  </P><P>(ak<SUB>1</SUB>D<SUP>3</SUP>)x<SUP>3</SUP> + (D<SUP>3</SUP>k<SUB>1</SUB> - ak<SUB>1</SUB>D<SUP>2</SUP>E)x<SUP>2</SUP> + (k<SUB>6</SUB>D<SUP>2</SUP>V - k<SUB>1</SUB>D<SUP>2</SUP>E + k<SUB>1</SUB>k<SUB>3</SUB>DV + k<SUB>2</SUB>k<SUB>6</SUB>DV)x - k<SUB>6</SUB>DEV - k<SUB>2</SUB>k<SUB>6</SUB>EV = 0 ③  </P><P align="left">And the value of each variable in its steady state (the balanced point) is  </P><P><CENTER>④
</CENTER></P><P align="left">The ODE (1) is highly complicated and we adopt numerical methods to analyze its properties. First, we generate 100000 sets of parameters stochastically (all in interval [0,10], and this setting keeps unchanged without special statement) to see the root distribution of equation (3). The results show that there is only one real positive root in 99890 cases and 3 in the rest cases. No cases when the real positive root doesn't exist are found.   </P><P align="left">Then note that the balanced point in (4) may not be authentic when <I>k<SUB>4</SUB></I>＜<I>k<SUB>3</SUB></I> and <I>x<SUB>3</SUB><SUP>*</SUP></I> becomes negative, which is impossible to occur. So we stochastically generate 100 parameters in which <I>k<SUB>4</SUB></I>＜<I>k<SUB>3</SUB></I>, and after solve the ODE numerically we find that all variables tend to an asymptotic steady state except for <I>x<SUB>3</SUB></I>, which approaches +∞ as t →+∞ (Fig 2). Interestingly, we find that the authentic balanced point of <I>x<SUB>1</SUB></I> and <I>x<SUB>2</SUB></I> calculated by directly solving the ODE (1) numerically is very <I>close</I> to that calculated by formula (4). For example, when <I>E</I>=4.5249, <I>a</I>=8.0649, <I>V</I>=2.5906, <I>D</I>=1.6831, <I>k<SUB>1</SUB></I>=5.2315, <I>k<SUB>2</SUB></I>=8.6560, <I>k<SUB>3</SUB></I>=8.7696, <I>k<SUB>4</SUB></I>=1.0092, <I>k<SUB>5</SUB></I>=6.9635, <I>k<SUB>6</SUB></I>=9.3253, <I>x<SUB>1</SUB></I> and <I>x<SUB>2</SUB></I> finally approach to 2.6599 and 0.0686, respectively, while <I>x<SUB>1</SUB><SUP>*</SUP></I>=2.3952 and <I>x<SUB>2</SUB><SUP>*</SUP></I>=0.0758 (Fig 2). The term <I>close</I> may not be mathematically strict, but it plays an important role in the later discussion.  </P><P><STRONG>Fig 2</STRONG> Numerical simulation when <I>k<SUB>4</SUB></I>＜<I>k<SUB>3</SUB></I>.  </P><P align="left">Besides, <I>x<SUB>4</SUB><SUP>*</SUP></I> and <I>x<SUB>5</SUB><SUP>*</SUP></I> may also be negative. We also generate 100000 sets of parameters stochastically (without the limitation of <I>k<SUB>4</SUB></I>＜<I>k<SUB>3</SUB></I>) to see how frequently <I>x<SUB>4</SUB><SUP>*</SUP></I> or <I>x<SUB>5</SUB><SUP>*</SUP></I> will be negative and what will it be like. However, it turns out that in no case will <I>x<SUB>4</SUB><SUP>*</SUP></I> or<I>x<SUB>5</SUB><SUP>*</SUP></I> be negative. So we may draw a <I>fuzzy</I> conclusion according to all the results above that under most conditions (99.89%), we can obtain a balanced point of ODE (1) which may not be authentic by formula (4). Fuzzy as the conclusion is, it is still useful to serve as an indicator for searching for a proper set of parameters, under which the Fatty Acid Degradation Device is highly regulatable.  </P><A name="Parameter Screening"><H3>Parameter Screening</H3></A><P align="left">It is expected that when the concentration of fatty acid in the environment is high, the expression level of gene fadR is relatively low while that of gene fadX is relatively high, and vice versa. And among the 10 parameters in ODE (1), <I>k<SUB>3</SUB></I> is positively related to the concentration of fatty acid outside the bacteria according to formula (2). So we assume that the device is thought to be <I>regulatable</I> when <I>k<SUB>3</SUB></I> is equal to 0.5 and 1.5, the ratio of the expression level of fadX to fadR at steady state rises but is still lower than 0.5, and when <I>k<SUB>3</SUB></I> is equal to 8.5 and 9.5, the ratio also rises and is both greater than 2.0.   </P><P align="left">We take advantage of the simplicity in calculating complexity of formula (4) to calculate the steady expression levels of fadR and fadX despite its possible errors. 10000 random parameters (<I>k<SUB>3</SUB></I> excluded) are generated and for each <I>k<SUB>3</SUB></I> in [0.5, 1.5, 8.5, 9.5], the balanced point of ODE (1) is calculated according to formula (4), respectively. Then compare the ratio of the expression levels of fadX to fadR at the balanced point and save it if it meets the condition above. We verify all the parameters saved by directly solving the ODE (1) numerically to see if it really meets the condition. 182 out of 10000 sets of parameters are saved and 57 of them remain after the verification. A typical example below (Fig 3) illustrates the change of the expression levels of fadR and fadX and their ratio corresponding to <I>k<SUB>3</SUB></I>. As <I>k<SUB>3</SUB></I> increases, the ratio rises smoothly from 0.2 to 3.5, while the expression level of fadX rises from 0.6 to 2.0, and that of fadR decreases from 2.6 to 0.6.    </P><CENTER><P><STRONG>Fig 3</STRONG> The change of the expression levels of fadR and fadX and their ratio corresponding to <I>k<SUB>3</SUB></I>.</P></CENTER><A name="Conclusion"><H3>Conclusion</H3></A><P align="left">To evaluate the response of gene expression levels to the concentration of fatty acid in the environment quantitatively, we build a mathematical model based on ODE and demonstrate that the antagonistic relationship between fadR and fadX serves as a linear regulator to the gene expression. This is important for the function of Fatty Acid Degradation Device because the model suggests that the Device can adjust itself to an appropriate state when induced by fatty acid and function properly rather than changes drastically. So the Device is implied mathematically to possess a great potential of applications in human being.</P></DIV><DIV class="passage divcell2"><A name="Gut Microbiota Regulation"><H3 align="center">Gut Microbiota Regulation</H3></A><P align="left">Perhaps the most challenging part of our idea is that the E.coslim will make a difference not only by influence the metabolism but also interact with our inner ecosystem --- the Human Gut Microbiota. Collectively, the microbial associates that reside in and on the human body constitute our microbiota, and the genes they encode is known as our microbiome. Containing at least 100 trillion of cells, the human gut microbiota is of high complexity and diversity, with taxa across the tree of life, bacteria, eukaryotes, viruses and archaeons. As sometimes referred to our <EM>forgotten organ</EM>, it plays a major role in health and diseases in human, including obesity and diabetes. And maybe even more importantly, it interacts with the immune system, providing signals to promote the maturation of immune cells and the normal development of immune functions. </P><P align="left">Though highly complicated, the gut microbiota is typically dominated by bacteria and specifically by members of the divisions Bacteriodetes and Firmicutes. And interestingly, it has been found that in animal model of obesity, the interplay of the two phyla is shifted with a significant reduction of Bacteriodetes and a corresponding increase of Firmicutes. This results in an increased capacity for harvesting energy from food and produces low-level inflammation. And more importantly, the imbalanced microbiota may be related to the recurrence of obesity after the treatment (whatever drugs, diets or exercises) stops. </P><P align="left">Let us refocus on our versatile E.coslim. When establishing itself in the gut, E.coslim will interact with the microbiota and thus regulate them. Considering the importance of microbiota in human health, it is necessary to know what effects the E.coslim has on the gut microbiota. On one hand, it may be a great disaster for people if the establishment of E.coslim leads to the extinction of some major bacteria taxa. And on the other hand, it may be proven noneffective if E.coslim died gradually after intake. </P><CENTER><P> We hope that our E. coslim can modulate gut microbiota therefore have long- term effect to help people lose weight</P><P align="left">However, the metagenomic sequencing may be required to conduct the experiment to see the change of microbiota caused by E.coslim. Due to the limitation of time and funds, we as undergraduate students are not able to perform such experiments. Instead, we use a mathematical model to predict the results of the interaction between E.coslim and the gut microbiota. While only a concise description is presented in this page, <STRONG>a detailed support information of this mathematical model can be downloaded <A href="https://static.igem.org/mediawiki/2012/b/b6/Support_Information.pdf">here</A></STRONG>.</P><P align="left">Consistent with previous discoveries and without particularity, we assume that the gut microbiota is constituted by two populations of bacteria, the Firmicutes (x<SUB>1</SUB>) and the Bacteriodetes (x<SUB>2</SUB>). Bacterium within each population has the same properties, which is reflected in the equality of the corresponding mathematical parameters. And it is also assumed that there are two kinds of resources, glucose (S) and fatty acid (R), which are perfectly substitutable for both populations. We simplify this situation as an exploitative competition in a chemostat, and the ODEs are:</P><P> </P><P align="justify">　　1. <EM>S(t)</EM> and <EM>R(t)</EM>: concentrations of glucose and fatty acid, respectively. </P><P align="justify">
　　2. <EM>x<SUB>i</SUB></EM>: biomass of the competing populations at time t   </P><P align="justify">
　　3. <EM>S<SUP>0</SUP></EM> and <EM>R<SUP>0</SUP></EM>: concentrations of resource <EM>S</EM> and <EM>R</EM> in the feed bottle   </P><P align="justify">
　　4. D: dilution rate   </P><P align="justify">
　　[The specific death rates of the microorganisms are assumed to be insignificant compared to this dilution rate D]   </P><P align="justify">
　　5.<EM>S<SUB>i</SUB></EM> and <EM>R<SUB>i</SUB></EM>: the rate of conversion of nutrient <EM>S</EM> to biomass of population <EM>x<SUB>i</SUB></EM></P><P align="justify">
　　[if the conversion of nutrient to biomass is proportional to the amount of nutrient consumed, the consumption rate of resource <EM>S</EM> per unit of　competitor <EM>x<SUB>i</SUB></EM> is denoted <EM>S</EM><SUB>i</SUB>[S(t),R(t)]/ξ<SUB>i</SUB>  where ξ<SUB>i</SUB> is the respective growth yield constant. ]   </P><P align="justify">
　　6. <EM>G<SUB>i</SUB></EM>: the rate of conversion of nutrient to biomass of population <EM>x<SUB>i</SUB></EM></P><P align="justify">
　　[Since perfectly substitutable resources are alternative sources of the same essential nutrient, the rate of conversion of nutrient to biomass of population <EM>x<SUB>i</SUB></EM> is made up of a contribution from the consumption of resource <EM>S</EM> as well as <EM>R</EM>: </P><P align="justify">
　　Here we choose    </P><P align="left">　　And let   </P><P align="left">They denote the maximal growth rate of population <EM>x<SUB>i</SUB></EM> on resource <EM>S(R)</EM> when none of the other resource is available.   </P><A name="Situation Before the Establishment of &lt;i&gt;E. coslim&lt;/i&gt; Flora"><H3>Situation Before the Establishment of <I>E. coslim</I> Flora</H3></A><P align="left">A model is built to describe the quantitative relationship between Firmicutes and Bacteriodetes in obese people's intestines.   </P><P align="left">Parameters <EM>m<SUB>Si</SUB>(m<SUB>Si</SUB>)</EM> can be assigned values to Firmicutes and Bacteriodetes so as to simulate the ability to utilize glucose and fatty acid. If their ability to use nutrient are given as follow:   </P><P><CENTER><TABLE border="1" bordercolor="#999999" cellpadding="10"><TBODY><TR><TD align="center">  Firmicutes   </TD><TD align="center">  Bacteriodetes  </TD></TR><TR><TD align="center"> Glucose    </TD><TD align="center">  +++ (<EM>m<SUB>S1</SUB></EM>)  </TD><TD align="center">  ++(<EM>m<SUB>S2</SUB></EM>)         </TD></TR><TR><TD align="center"> Fatty acid </TD><TD align="center">  + (<EM>m<SUB>R1</SUB></EM>)       </TD><TD align="center">  ++(<EM>m<SUB>R2</SUB></EM>)         </TD></TR></TBODY></TABLE></CENTER></P><P align="left">Then we can set <EM>m<SUB>S1</SUB></EM>=2.25, <EM>m<SUB>R1</SUB></EM>=0.5, <EM>m<SUB>S2</SUB></EM>=2.1, <EM>m<SUB>R2</SUB></EM>=2.1. In order to make easier ODEs, we set <EM>S<SUP>0</SUP></EM>=<EM>R<SUP>0</SUP></EM>=D=1 and ξ<SUB>i</SUB>/η<SUB>i</SUB>=100. Simulation result is shown in Fig 1.  </P></CENTER><P><STRONG>Fig 1   </STRONG></P><P align="left">In this situation, the ratio N(Firmicutes)/N(Bacteriodetes) is rather high, usually achieving a value 8.0, and each of their absolute number, or concentration, is stable.   </P><H3>Situation After the Establishment of <I>E. coslim</I> Flora</H3><P align="left">We try to add the E.coslim into system. E.coslilm consumes glucose as well as fatty acid, thus makes itself a competitor to Firmicutes and Bacteriodetes. While it is reproducing in intestines, the competition among these three types of bacteria makes the number change gradually. And we want to know the results of the establishment of the GEB in the gut. </P><P align="left">The key point is to find out how competitive our E.coslim should be. In other words, we have to point out its ability to consume glucose and fatty acid---to study new parameters (<EM>m<SUB>S3 </SUB></EM>,<EM>m<SUB>R3</SUB></EM>). For example, if <EM>m<SUB>S3</SUB></EM>＞<EM>m<SUB>S1</SUB></EM>, then we conclude that E.coslim has stronger ability to consume glucose than Firmicutes. We try to find out an appropriate pair of (<EM>m<SUB>S3 </SUB></EM>,<EM>m<SUB>R3</SUB></EM>). And next we will discuss different situations of (<EM>m<SUB>S3 </SUB></EM>,<EM>m<SUB>R3</SUB></EM>), respectively. </P><P align="justify"><STRONG>

Situation 1. (<EM>m<SUB>S3 </SUB></EM>,<EM>m<SUB>R3</SUB></EM>)=(2.5, 2.1) </STRONG></P><P align="left">The number of &quot;+&quot; in the table below qulitatively represents the ability of the corresponding bacteria to consume the corresponding resource. &quot;+&quot; represents &quot;low&quot;, &quot;++&quot; moderate, and &quot;+++&quot; strong, etc. 
<CENTER><TABLE border="1" bordercolor="#999999" cellpadding="10"><TBODY><TR><TD align="center">  Firmicutes  </TD><TD align="center">  Bacteriodetes</TD><TD align="center">E.coslim</TD></TR><TR><TD align="center"> Glucose    </TD><TD align="center">  +++ (<EM>m<SUB>S1</SUB></EM>) </TD><TD align="center">  ++(<EM>m<SUB>S2</SUB></EM>)       </TD><TD align="center">++++(<EM>m<SUB>S3</SUB></EM>)</TD></TR><TR><TD align="center"> Fatty acid </TD><TD align="center">  + (<EM>m<SUB>R1</SUB></EM>)      </TD><TD align="center">  ++(<EM>m<SUB>R2</SUB></EM>)         </TD><TD align="center">++(<EM>m<SUB>R3</SUB></EM>)</TD></TR></TBODY></TABLE></CENTER></P><P align="justify">　　The simulation result is shown in Fig 2. </P><P><STRONG>Fig 2</STRONG></P><P align="justify"><STRONG>Situation 1'. (<EM>m<SUB>S3 </SUB></EM>,<EM>m<SUB>R3</SUB></EM>)=(2.1,2.5) </STRONG></P><P><CENTER><TABLE border="1" bordercolor="#999999" cellpadding="10"><TBODY><TR><TD align="center">  Firmicutes  </TD><TD align="center">  Bacteriodetes</TD><TD align="center">E.coslim</TD></TR><TR><TD align="center"> Glucose    </TD><TD align="center">  +++ (<EM>m<SUB>S1</SUB></EM>) </TD><TD align="center">  ++(<EM>m<SUB>S2</SUB></EM>)       </TD><TD align="center">++(<EM>m<SUB>S3</SUB></EM>)</TD></TR><TR><TD align="center"> Fatty acid </TD><TD align="center">  + (<EM>m<SUB>R1</SUB></EM>)      </TD><TD align="center">  ++(<EM>m<SUB>R2</SUB></EM>)         </TD><TD align="center">+++(<EM>m<SUB>R3</SUB></EM>)</TD></TR></TBODY></TABLE></CENTER></P><P><STRONG>Fig 3</STRONG></P><P> </P><P align="left">Situation 1 and 1' show that E.coslim are so competitive that others die out. This may be detrimental to human health since the importance of the gut microbiota. 
    </P><P align="justify"><STRONG>


Situation 2. (<EM>m<SUB>S3 </SUB></EM>,<EM>m<SUB>R3</SUB></EM>)=(2.1,0.4) </STRONG></P><P><CENTER><TABLE border="1" bordercolor="#999999" cellpadding="10"><TBODY><TR><TD align="center">  Firmicutes  </TD><TD align="center">  Bacteriodetes</TD><TD align="center">E.coslim</TD></TR><TR><TD align="center"> Glucose    </TD><TD align="center">  +++ (<EM>m<SUB>S1</SUB></EM>) </TD><TD align="center">  ++(<EM>m<SUB>S2</SUB></EM>)       </TD><TD align="center">++(<EM>m<SUB>S3</SUB></EM>)</TD></TR><TR><TD align="center"> Fatty acid </TD><TD align="center">  + (<EM>m<SUB>R1</SUB></EM>)      </TD><TD align="center">  ++(<EM>m<SUB>R2</SUB></EM>)         </TD><TD align="center">＜+(<EM>m<SUB>R3</SUB></EM>)</TD></TR></TBODY></TABLE></CENTER></P><P><STRONG>Fig 4 </STRONG></P><P> </P><P align="left">In this situation E.coslim is too weak to survive in the gut, which refers to as noneffective. 
    </P><P align="justify"><STRONG>


Situation 3. (<EM>m<SUB>S3 </SUB></EM>,<EM>m<SUB>R3</SUB></EM>)=(2.1,2.11) </STRONG></P><P><CENTER><TABLE border="1" bordercolor="#999999" cellpadding="10"><TBODY><TR><TD align="center">  Firmicutes  </TD><TD align="center">  Bacteriodetes</TD><TD align="center">E.coslim</TD></TR><TR><TD align="center"> Glucose    </TD><TD align="center">  +++ (<EM>m<SUB>S1</SUB></EM>) </TD><TD align="center">  ++(<EM>m<SUB>S2</SUB></EM>)       </TD><TD align="center">++(<EM>m<SUB>S3</SUB></EM>)</TD></TR><TR><TD align="center"> Fatty acid </TD><TD align="center">  + (<EM>m<SUB>R1</SUB></EM>)      </TD><TD align="center">  ++(<EM>m<SUB>R2</SUB></EM>)         </TD><TD align="center">++(<EM>m<SUB>R3</SUB></EM>)</TD></TR></TBODY></TABLE></CENTER></P><P><STRONG>Fig 5 </STRONG></P><P> </P><P align="justify"><STRONG>Situation 4. (<EM>m<SUB>S3 </SUB></EM>,<EM>m<SUB>R3</SUB></EM>)=(2.25,0.5) </STRONG></P><P><CENTER><TABLE border="1" bordercolor="#999999" cellpadding="10"><TBODY><TR><TD align="center">  Firmicutes  </TD><TD align="center">  Bacteriodetes</TD><TD align="center">E.coslim</TD></TR><TR><TD align="center"> Glucose    </TD><TD align="center">  +++ (<EM>m<SUB>S1</SUB></EM>) </TD><TD align="center">  ++(<EM>m<SUB>S2</SUB></EM>)       </TD><TD align="center">+++(<EM>m<SUB>S3</SUB></EM>)</TD></TR><TR><TD align="center"> Fatty acid </TD><TD align="center">  + (<EM>m<SUB>R1</SUB></EM>)      </TD><TD align="center">  ++(<EM>m<SUB>R2</SUB></EM>)         </TD><TD align="center">+(<EM>m<SUB>R3</SUB></EM>)</TD></TR></TBODY></TABLE></CENTER></P><P><STRONG>Fig 6</STRONG></P><P> </P><P align="left">The simulation result shows that only in this situation will the E.coslim establish itself in the gut successfully without leading to the extinction of Firmicutes or Bacteriodetes. And surprisingly, the proportion of Firmicutes to Bacteriodetes also declines, as consistent to the condition in normal people. We may predict that E.coslim which suits Situation 4 can not only influence the human metabolism but also regulate the gut microbiota constitution, thus preventing the recurrence of obesity. </P><P align="left">We also map the change of the consuming rate on each resource between the situation before and after the establishment of E.coslim (Situation 4 is used), as shown in Fig 7. And it is illustrated that the consuming rates of Bacteriodetes on both of the resources decline slightly, while that of Firmicutes is down-regulated acompanied with the increase of that of E.coslim. </P><P align="center"><STRONG>Fig 7 </STRONG></P><P align="justify"> </P><A name="Conclusion1"><H3 align="center">Conclusion</H3></A><P align="justify">
By carefully assigning values to parameters in the mathematical model of the microbiota regulation, we found that the metabolic capacity of E.coslim should be controlled to a proper level, rather than as great as possible. Although it seems that a greater ability to degrade fatty acid and synthesize cellulose may further reduce the calorie people absorbed, it may also lead to potential healthy risk since the microbiota consititution may change drastically. The metabolic capacity indicated by related parameters also helps us in designing a perfect-functioning E.coslim. </P><P align="justify"> </P></DIV><DIV class="passage divcell3"><H3>Gene Expression Measurement by <I>FANCY</I></H3><H3>Abstract</H3><P align="left">The quantitative measurement of gene expression levels is of great importance in the research of molecule biology, but existing measuring methods are either semi-quantitative (Western Blotting) or require costly instruments and agents (Flow Cytometry). We invent a novel quantitative and inexpensive method, <I>FANCY</I>, which is short for <I>Fluorescent ANalysis of CYto-imaging</I> (also in the honor of a wise and pretty girl <A href="https://2012.igem.org/Team:WHU-China/Team?catalog=1#Fan Cheng">Fan Cheng</A> in our team) and based on cell fluorescence imaging and Support Vector Machine, to measure the gene expression in our project. <I>FANCY</I> contains three sub-programme: <I>FANCYSelector</I> to select single cells manually and calculate their properties, <I>FANCYTrainer</I> to train an SVM, and <I>FANCYScanner</I> to identify single cells by SVM trained by <I>FANCYTrainer</I>. The application of <I>FANCY</I> to 5 typical images proves the effectiveness and the error rate is acceptable. All our programs are freely available online and we suggest <I>FANCY</I> more widely applied in iGEM and other further researches.</P><A name="Protocol"><H3>Protocol</H3></A><P align="justify"><STRONG>Preparation of fluorescent images </STRONG></P><P align="justify">
　　1. Connect a red fluorescent protein gene (gene <I>rfp</I>, or biobrick 1-22O) with the objective gene under the same promoter in a plasmid by molecule cloning, i.e., make a polycistron which can encode both the objective gene product and RFP, thus making the expression levels of the two genes the same. Other fluorescent genes such as <I>gfp</I> or <I>yfp</I> may be also applicable.  </P><P align="justify">
　　2. Transform the plasmid into E.coli and incubate it until a proper density.   </P><P align="justify">
　　3. Prepare a slide sample of the transformed E.coli and dry it in the air. Cover the bacteria with anti-fluorescence quencher to maintain the fluorescence. The bacteria may be diluted to avoid cell mass formation on the slide during the preparation.   </P><P align="justify">
　　4. Take photos of the slide by fluorescent microscopy. Select visual fields with as many single cells and little cell masses as possible, and avoid photographing for too long a time since the fluorescent density may decrease under the exciting light.   </P><P align="justify"><STRONG>Support vector machine training (FANCYSelector and FANCYTrainer)  </STRONG></P><P align="justify">
　　1. 10 typical images are selected as the training images. </P><P align="justify">
　　2. Transfer the original RGB images to gray images <I>G</I> and binary images <I>B</I>. No image enhancement was processed in our project, but it is suggested to be done in other conditions if needed.   </P><P align="justify">
　　3. For each object detected in the binary image <I>B</I>, ask the user to classify whether it is a single cell or a cell mass, and after that the object will be marked TRUE or FALSE, respectively. Then calculate the area <I>A</I>, perimeter <I>P</I>, Euler number <I>E</I>, the maximal length of any two points in the object <I>L</I> and the fluorescence <I>F</I>. The fluorescence <I>F</I> is calculated by the numerical double integral of the gray level of the image <I>G</I> on the region identified by the current object in the image <I>B</I>. </P><P align="justify">
　　4. Record all the data obtained in Step 3 and save them in an ASCII or .mat file (optional).   </P><P align="justify">
　　5. Use the data <I>A</I>, <I>P</I>, <I>E</I> and <I>L</I> as the training data and the user's classification result as the group data to train the support vector machine (SVM). The Gaussian radial basis function with a scaling factor of 1 is selected as the kennel function of the SVM.   </P><P align="justify"><STRONG>Single cell identification and fluorescence calculation (FANCYScanner)  </STRONG></P><P align="justify">
　　1. For all the images taken from the slide, transfer them to gray and binary images respectively, calculate the <I>A</I>, <I>P</I>, <I>E</I>, <I>L</I> and <I>F</I> of each object in the binary images (the same as the former part) and classify them as single cells or cell masses using the SVM trained. All objects classified as single cells will be marked with * in the binary image <I>B</I>. And <I>B</I> will be saved for possible performance evaluation.   </P><P align="justify">
   2. All data from objects classified as single cells will be recorded in a ASCII file. And the average and the standard variance of the fluorescent density (fluorescence <I>F</I> divided by area <I>A</I>) is calculated as the quantitative description of the objective gene expression level.
</P><A name="Performance Evaluation"><H3>Performance Evaluation</H3></A><P align="left">We randomly selected and applied both <I>FANCY</I> and manual identification to 5 other images to evaluate the performance of <I>FANCY</I>. The false positive and false negative rate of <I>FANCY</I> are calculated in comparison to manual identification, which defaults to be absolutely correct. 179 positive and 122 negative objects in all are identified by <I>FANCY</I> and the evaluation result is shown in Table 1.</P><P align="justify"><CENTER><STRONG>Table 1</STRONG></CENTER></P><P align="left"><SMALL><STRONG>Fig 1</STRONG>. Typical false positive and false negative results of <I>FANCY</I> identification. <STRONG>A</STRONG>. The identification result. <STRONG>B</STRONG>. The original fluorescent image (some cells in the original image are not able to be detected after image transformation). </SMALL></P><P align="justify"><SMALL><STRONG>Fig 2.</STRONG> Comparison of a cell fluorescent image and its identification result by <EM>FANCY</EM>. <STRONG>A.</STRONG> The cell fluorescent image. <STRONG>B.</STRONG> The identification result, in which objects identified as single cells are dotted with a *. </SMALL></P><P align="left">The false positive and negative rate may decrease with more training data and proper image enhancement. </P><A name="Source Code"><H3>Source Code</H3></A><P align="justify"> 　 　 All source code programmed in Matlab can be freely downloaded <A href="https://static.igem.org/mediawiki/2012/8/80/FANCY.zip">here</A>. For detailed usage please refer to the annotation in the code.</P><P> </P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:WHU-China/Modeling">http://2012.igem.org/Team:WHU-China/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WHU-China/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WHU-China/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WHU-China/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WHU-China/Modeling&amp;oldid=298339" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>