<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SDU-Denmark_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:SDU-Denmark/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SDU-Denmark/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SDU-Denmark/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SDU-Denmark/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SDU-Denmark/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SDU-Denmark/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="container"><DIV class="container"><UL id="topnav"><LI><A href="https://2009.igem.org/Team:SDU-Denmark">Home</A></LI><LI><A href="https://2009.igem.org/Team:SDU-Denmark/Background">Background</A><SPAN><A href="https://2009.igem.org/Team:SDU-Denmark/Background#Description_of_S._aureus">S. aureus</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Background#Biofilm_Formation">Biofilm</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Background#Quorum-sensing">Quorum-sensing</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Background#Quorum-quenching_using_RIP">Quenrum-quenching</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Brainstorm">Brainstorm</A></SPAN></LI><LI><A href="https://2009.igem.org/Team:SDU-Denmark/Project">Project</A><SPAN><A href="https://2009.igem.org/Team:SDU-Denmark/Project#Concept">Concept</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Project#Methods">Methods</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Project#Results">Results</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Project#Implications">Implications</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Protocols">Protocols</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Tools">Tools</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Safety">Safety</A></SPAN></LI><LI><A href="https://2009.igem.org/Team:SDU-Denmark/Parts">Parts</A><SPAN><A href="https://2009.igem.org/Team:SDU-Denmark/Parts#Our_submitted_parts">Submitted parts</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark | /Parts#Contributions_and_improvements_to_existing_parts">Contributions</A> | 
                <A href="https://2009.igem.org/Team:SDU-Denmark/Parts/backbone_troubles">pSB1A3 characterization</A></SPAN></LI><LI><A href="https://2009.igem.org/Team:SDU-Denmark/Team">Team</A><SPAN><A href="https://2009.igem.org/Team:SDU-Denmark/Team#Students">Students</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Team#Advisors">Advisors</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Team#About_SDU">About SDU</A> |
                <A href="https://2009.igem.org/Team:SDU-Denmark/Helping">What we did to help others</A></SPAN></LI></UL></DIV><DIV id="leftcontent"><H1><SPAN class="mw-headline" id="Concept">Concept</SPAN></H1><H3><SPAN class="mw-headline" id="Bacto-Bandage">Bacto-Bandage</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Different kinds of plastic adhesive drapes are already in use today.</DIV></DIV></DIV><P>The main goal of our project was to assemble biobricks in to a plasmid and in this way make <I>E. coli</I> produce and excrete the quorum-quenching protein <I>RIP</I>.
</P><P>We propose making a bandage that contains our engineered bacteria behind a semipermeable membrane, allowing only small peptides such as <I>RIP</I> to pass through, into the wound.
</P><P>We have <A href="https://2009.igem.org/Team:SDU-Denmark/Background#Quorum-quenching_using_RIP" class="external text" rel="nofollow">reason to hope</A> that this will weaken the biofilm formation, thereby making multiresistant bacteria easier to treat with conventional antibiotics. This should also increase the effectiveness of the antibiotics and thereby decrease the amount and range of antibiotics that are needed today, to fight the infections.
</P><P>Since our system doesn't kill <I>S. aureus</I> but only prevents it from becoming pathogenic, it is less likely to promote the development of resistant strains.
</P><H3><SPAN class="mw-headline" id="Parts">Parts</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Our biological machine, composed of a <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J13002" class="external text" rel="nofollow">promoter and RBS</A>, our novel <I><A href="http://partsregistry.org/Part:BBa_K237002" class="external text" rel="nofollow">RIP</A></I> part and a <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0015" class="external text" rel="nofollow">double terminator</A>.</DIV></DIV></DIV><P>Our idea is to compose a plasmid containing a promoter, which should either be constitutive or inducible. After the promoter sequence we have a RBS, and then our <I>RIP</I> sequence, which we based on the amino acid sequence of the protein and made in PCR via two flanking primers. We are working with a clean <I>RIP</I> sequence and a <I>RIP</I> sequence with an export sequence on it, called <I>sRIP</I>. Following the <I>RIP</I> sequence we have a terminator. 
</P><P>All the four parts were incorporated into an ampicillin resistant plasmid backbone.
</P><P>We designed our system with two different promotors, since we didn't know whether a high concentration of <I>RIP</I> will quorum quench the <I>E. coli</I> itself.  It will therefore be practical for us to control the rate of transcription - hence the inducible promoter.
</P><P>On the other hand we know that a higher concentration of <I>RIP</I> is more efficient in quorum-quenching <I>S. aureus</I>, and it will therefore be more efficient if the plasmid is transcribed at all times. Furthermore if the export of the protein out of the cell is weak, an overproduction of the protein might puncture the cell and <I>RIP</I> will leak out. This will of course kill the cells, but we don't see this as a big problem as they easily can be replaced. Our goal is to get as much <I>RIP</I> exported from the bandage to the wound one way or another.
</P><H1><SPAN class="mw-headline" id="Methods">Methods</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Flowchart detailing our project.</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Finding_parts">Finding parts</SPAN></H2><P>We used the <A href="http://partsregistry.org" class="external text" rel="nofollow">Registry of Standard Biological Parts</A> (Parts Registry in short) to find a suitable <A href="http://partsregistry.org/Part:pSB1A2" class="external text" rel="nofollow">backbone</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J13002" class="external text" rel="nofollow">promoter, RBS</A> and <A href="http://partsregistry.org/Part:BBa_B0015" class="external text" rel="nofollow">terminators</A>. All the parts where checked for compatibility with the standard BioBrick assembly method (<A href="http://partsregistry.org/Assembly:Standard_assembly" class="external text" rel="nofollow">RFC 10</A>).
</P><P>We also wanted to test our system with an <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_R0011" class="external text" rel="nofollow">inducible promoter</A>, but sadly never had the time.
</P><H2><SPAN class="mw-headline" id="Ligations">Ligations</SPAN></H2><P>We ligated our parts together using a slightly different <A href="https://2009.igem.org/Team:SDU-Denmark/Protocols#Ligation_-_Protocol_3" class="external text" rel="nofollow">protocol</A> then suggested in the <A href="http://partsregistry.org/Help:BioBrick_Assembly" class="external text" rel="nofollow">Parts Registry help pages</A>, before inserting them into our backbone:
</P><P>1. Amplification by PCR using <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A> and our modified <A href="http://partsregistry.org/Part:BBa_K237007" class="external text" rel="nofollow">VR</A> primer (<A href="https://2009.igem.org/Team:SDU-Denmark/Protocols#Protocol_for_Primer_PCR" class="external text" rel="nofollow">Protocol</A>).
</P><P>2. Digest with fast digest restriction enzymes, upstream part with SpeI and downstream part with XbaI (<A href="https://2009.igem.org/Team:SDU-Denmark/Protocols#Restriction_digest_-_Protocol_2" class="external text" rel="nofollow">Protocol</A>).
</P><P>3. Ligate with T4 DNA Ligase, resulting in a mixed site (<A href="https://2009.igem.org/Team:SDU-Denmark/Protocols#Ligation_-_Protocol_1" class="external text" rel="nofollow">Protocol</A>), of either 6 or 8 bp length.
</P><H2><SPAN class="mw-headline" id="Inserting_parts_in_backbone">Inserting parts in backbone</SPAN></H2><P>The steps was as follows:
</P><P>To skip a bacerial growth step, and thereby save some time, our parts where amplified by PCR, as suggested in <A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf" class="external text" rel="nofollow">Ginkgo Bioworks assembly manual</A>.
</P><P>1. Amplification the ligated products by PCR using <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2 primer</A> and our modified <A href="http://partsregistry.org/Part:BBa_K237007" class="external text" rel="nofollow">VR2 primer</A> (<A href="https://2009.igem.org/Team:SDU-Denmark/Protocols#Protocol_for_Primer_PCR" class="external text" rel="nofollow">Protocol</A>).
</P><P>2. Digesting with fast digest restriction enzymes EcoRI-HFâ¢ and PstI (<A href="https://2009.igem.org/Team:SDU-Denmark/Protocols#Restriction_digest_-_Protocol_2" class="external text" rel="nofollow">Protocol</A>).
</P><P>We used fast digest enzymes, since normal enzymes where either slow or ineffective in cutting our bricks. 
</P><P>3. Ligated with T4 DNA Ligase (<A href="https://2009.igem.org/Team:SDU-Denmark/Protocols#Ligation_-_Protocol_1" class="external text" rel="nofollow">Protocol</A>).
</P><P>4. Transformation into competent <I>E.coli</I> (<A href="https://2009.igem.org/Team:SDU-Denmark/Protocols#Electroporation" class="external text" rel="nofollow">Protocol</A>).
</P><H2><SPAN class="mw-headline" id="Testing">Testing</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Flowchart concerning our testing phase.</DIV></DIV></DIV><P>To test if our 4 part brick was actually inserted into <I>E.coli</I> and functional, we prepared three different test systems: gene transcription by measuring <I>RIP</I> mRNA expression (Northern Blot), MALDI to measure protein expression and a actual test of quorum-quenching.  
</P><H3><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:52px;"><DIV class="thumbcaption">Gel picture of Colony PCR.</DIV></DIV></DIV><P>We started by checking our part lengths by colony PCR (<A href="https://2009.igem.org/Team:SDU-Denmark/Protocols#Protocol_for_colony_PCR" class="external text" rel="nofollow">Protocol</A>).
</P><H3><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H3><P>We sent our parts to sequencing to be sure that we had exactly the right DNA in our bacteria. We used the Plasmid editor <A href="/Team:SDU-Denmark/Tools" title="Team:SDU-Denmark/Tools">APE</A>, to see our sequencing data.
</P><H3><SPAN class="mw-headline" id="RNA_expression">RNA expression</SPAN></H3><P>To measure if RNA was being expressed, we used northern blotting (<A href="https://2009.igem.org/Team:SDU-Denmark/Protocols#Protocol_for_Northern_Blotting" class="external text" rel="nofollow">Protocol</A>). We used a radioactive probe (32P) that binds to <I>RIP</I> mRNA and ran it through a membrane, to separate mRNA's into different sizes. The membrane was then put on a film, and scanned so we could visualize if there was any <I>RIP</I> mRNA in our cells.
</P><H3><SPAN class="mw-headline" id="Biofilm_assay">Biofilm assay</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Biofilm assay.</DIV></DIV></DIV><P>To test whether or not our <I>RIP</I> part was being secreted out of our <I>E. coli</I> biomachine, we designed a biofilm assay.
</P><P>The assay was made on a 4*6-well plate. <I>S.aureus</I> were grown in the wells, thereby making biofilms. 
</P><P>We then added different <I>E. coli</I> components to the wells: supernatant, crushed pellet and living <I>E. coli</I>. After 24 hours the wells were washed three times with sterile 1x PBS solution to remove planctonic bacteria. Cells remaining adhered to the wells are subsequently stained with violet blue, the solution stands for 15 minutes and was washed twice.
</P><P>1 ml of 96% ethanol was added to each stained well to loosen the biofilm. The biofilm was tested by measuring the absorbance by transferring the crystal violet/ethanol solution from each well to a cuvette, and measure the optical density (OD) of each of these samples at a wavelength of 595 nm.
</P><H3><SPAN class="mw-headline" id="MALDI">MALDI</SPAN></H3><P>Matrix-assisted laser desorption/ionization (MALDI) is a technique used in mass spectrometry, allowing the analysis of for example proteins. 
</P><P>We did a MALDI on the supernatant of our 4 part brick, to see whether or not <I>RIP</I> was being exported out of the bacteria.
</P><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Sequencing confirmed that we have the right DNA in our bacteria. The red parts are restriction-sites, green is the composite promoter and RBS, blue is <I>RIP</I> and orange is our double terminator.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="New_parts_and_primers">New parts and primers</SPAN></H3><P>We have successfully constructed the <I>RIP</I> and <I>sRIP</I><A href="/Team:SDU-Denmark/Parts" title="Team:SDU-Denmark/Parts">parts</A>, and used these with older parts, all of which have been added to the registry. We have also added a slightly modified VR primer to the registry.
</P><H3><SPAN class="mw-headline" id="RIP_expression_in_E._coli"><I>RIP</I> expression in <I>E. coli</I></SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Northern blot of our 4 brick part () and our RIP-Terminator part (). As seen here only the 4 brick part (left) is transcribed into mRNA and hopefully further translated into peptide.</DIV></DIV></DIV><P>We have successfully assembled a 4 brick part containing a constitutive promotor with RBS, our novel <I>RIP</I> part and a terminator. 
</P><P>Sequencing confirmed the 4 brick part DNA sequence and a northern blot analysis confirmed that our <I>E. coli</I> indeed transcribes the <I>RIP</I> gene. To our knowledge this is the first recombinant organism to harbour and transcribe the <I>RIP</I> gene. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Mass spectrometry of supernatant from our 4 part <I>RIP</I> colony (A and C) compared with a reference colony (B and D). No conclusive results could be gathered from this analysis.</DIV></DIV></DIV><P>We also did protein MALDI mass spectrometry, which didn't show anything conclusive, perhaps as a consequence of the <I>RIP</I> peptides small size, which makes it hard to distinguish from fragmentation peaks of larger proteins. Another possible explanation could be that our bacteria is not transporting <I>RIP</I> out of the cell.
</P><H3><SPAN class="mw-headline" id="Quorum-quenching_and_biofilm_inhibition">Quorum-quenching and biofilm inhibition</SPAN></H3><P>We are currently testing the quorum-quenching effect on <I>S. aureus</I> biofilm formation and the results are promising. The big question is whether or not, <I>RIP</I> gets exported out of <I>E. coli</I> and past the peptideglycan layer and capsule. The final results will be presented at the Jamboree.
</P><H3><SPAN class="mw-headline" id="Problems">Problems</SPAN></H3><P>Throughout most of the summer, we've had problems getting our ligations product sizes to match. What ever we tried nothing seemed to work, since it was always the wrong size. Then a few weeks ago, it was discovered, that the <A href="/Team:SDU-Denmark/Parts/backbone_troubles" title="Team:SDU-Denmark/Parts/backbone troubles">problem</A> was with our backbone (1K from plate 1, 2009 spring distribution), not our work. 1K was suppose to be , but instead it was .
</P><P>Sadly we didn't discover this till the very last week, and so we didn't get to do all the experiments we wanted, including one with an inducible promoter and one with our signaling peptide coupled with <I>RIP</I>. 
</P><H1><SPAN class="mw-headline" id="Implications">Implications</SPAN></H1><P>If indeed our <I>E. coli</I> can inhibit biofilm formation by producing <I>RIP</I>, this generates many different opportunities for further development.
</P><P>Our machine opens up for the possibility to combat infections more intelligently. 
</P><P>As mentioned earlier we considered using a inducible promotor to control the expression of <I>RIP</I>. Optimally the promotor should react to the level of autoinducers in the environment by use of quorum-signaling receptors. 
The concept could also be implicated on other strains of bacteria performing quorum sensing with other types of autoinducers.
</P><P>A logical next step would be to couple several kinds of quorum-quenching molecules to our machine, together with different quorum-signaling receptors. This would allow for inhibition of only the specific bacteria that are currently infecting the wound.
</P><P>Essentially the Bacto-Bandage would sense if any kind of bacteria was trying to colonize the wound and inhibit this by means of producing the most effective quorum-quenching molecules.
</P></DIV><DIV id="rightcontent"><H2><SPAN class="mw-headline" id="Our_project_in_1_minute">Our project in 1 minute</SPAN></H2><P>We want to fight hospital infections, using a biological machine.
</P><P>For this purpose we have constructed a recombinant <I>E. coli</I>, which produces the <I>RIP</I> peptide. <I>RIP</I> inhibits biofilm formation in the bacteria <I>S. aureus</I>.
</P><P>This peptide will make it easier to fight infections and kill <I>S. aureus</I> with conventional antibiotics.
</P><H2><SPAN class="mw-headline" id="Other_stuff">Other stuff</SPAN></H2><UL><LI> Check out all our <A href="/Team:SDU-Denmark/Protocols" title="Team:SDU-Denmark/Protocols">protocols</A>.
</LI></UL><UL><LI> See what <A href="/Team:SDU-Denmark/Tools" title="Team:SDU-Denmark/Tools">tools</A> we used.
</LI></UL><UL><LI> Read our <A href="/Team:SDU-Denmark/Safety" title="Team:SDU-Denmark/Safety">safety</A> page.
</LI></UL><H2><SPAN class="mw-headline" id="Content">Content</SPAN></H2><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Concept"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Concept</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Bacto-Bandage"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Bacto-Bandage</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Parts"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Parts</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Methods"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Finding_parts"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Finding parts</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Ligations"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Ligations</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Inserting_parts_in_backbone"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Inserting parts in backbone</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Testing"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Testing</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Colony_PCR"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Sequencing"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#RNA_expression"><SPAN class="tocnumber">2.4.3</SPAN><SPAN class="toctext">RNA expression</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Biofilm_assay"><SPAN class="tocnumber">2.4.4</SPAN><SPAN class="toctext">Biofilm assay</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#MALDI"><SPAN class="tocnumber">2.4.5</SPAN><SPAN class="toctext">MALDI</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#New_parts_and_primers"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">New parts and primers</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#RIP_expression_in_E._coli"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">RIP expression in E. coli</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Quorum-quenching_and_biofilm_inhibition"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Quorum-quenching and biofilm inhibition</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Problems"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Problems</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Implications"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Implications</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Our_project_in_1_minute"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Our project in 1 minute</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Other_stuff"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Other stuff</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Content"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Content</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:SDU-Denmark/Project">http://2009.igem.org/Team:SDU-Denmark/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SDU-Denmark/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SDU-Denmark/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SDU-Denmark/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SDU-Denmark/Project&amp;oldid=161154" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>